<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/256239-a-method-of-generating-analogues-of-fkbp-ligands-which-incorporate-a-non-natural-starter-unit by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:46:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 256239:A METHOD OF GENERATING ANALOGUES OF FKBP-LIGANDS WHICH INCORPORATE A NON-NATURAL STARTER UNIT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF GENERATING ANALOGUES OF FKBP-LIGANDS WHICH INCORPORATE A NON-NATURAL STARTER UNIT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to production of polyketides and other natural products and to libraries of compounds and individual novel compounds. One important area is the isolation and potential use of novel FKBP-ligand analogues and host cells that produce these compounds. The invention is particularly concerned with methods for the efficient transformation of strains that produce FKBP analogues and recombinant cells in which cloned genes or gene cassettes are expressed to generate novel compounds such as polyketide (especially rapamycin) FKBP-ligand analogues, and to processes foT their preparation, and to means employed therein (e.g. nucleic acids, vectors, gene cassettes and genetically modified strains).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The instant application is divided out of Indian Patent Application No.<br>
1971/KOLNP/2004.<br>
Field of the Invention<br>
The present invention relates to novel polyketide (rapamycin) fkbp-ligand<br>
analogues and method for production thereof. One<br>
important area is the isolation and potential use of novel FKBP-ligand analogues and<br>
host cells that produce these compounds. The invention is particularly concerned<br>
with methods for the efficient transformation of strains that produce FKBP analogues<br>
and recombinant cells in which cloned genes or gene cassettes are expressed to<br>
generate novel compounds such as polyketide (especially rapamycin) FKBP-ligand<br>
analogues, and to processes for their preparation, and to means employed therein<br>
(e.g. nucleic acids, vectors, gene cassettes and genetically modified strains).<br>
Background of the invention<br>
Rapamycin (sirolimus) (Figure 1) is a lipophilic macrolide produced by<br>
Streptomyces hygroscopicus NRRL 5491 (Sehgal et al., 1975; Vezina et al., 1975;<br>
U.S. Patent No. 3,929,992; U.S. Patent No. 3,993,749) with a 1,2,3-tricarbonyl<br>
moiety linked to a pipecolic acid lactone (Paiva et al., 1991). Other related macrolides<br>
(Figure 2) include FK506 (tacrolimus) (Schreiber and Crabtree, 1992), FK520<br>
(ascomycin or immunomycin) (Wu et al., 2000), FK525 (Hatanaka H, et al., 1989,<br>
FK523 (Hatanaka, H., et al, 1988), antascomicins (Fehr, T., et al., 1996) and<br>
meridamycin (Salituro et al., 1995). For the purpose of this invention rapamycin is<br>
described by the numbering convention of McAlpine et al. (1991) in preference to the<br>
numbering conventions of Findlay et al. (1980) or Chemical Abstracts (11th-<br>
Cumulative Index, 1982-1986 p60719CS).<br>
The versatile mode of action of rapamycin demonstrates the pharmacological<br>
value ofthe compound and emphasizes the necessity to isolate novel derivatives of<br>
the drug. Rapamycin shows moderate antifungal activity, mainly against Candida<br>
species but also against filamentous fungi (Baker ef al., 1978; Sehgal et al., 1975;<br>
Vezina et a!., 1975; U.S. Patent No. 3,929,992; U.S. Patent No. 3,993,749).<br>
Rapamycin inhibits cell proliferation by targeting signal transduction pathways in a<br>
variety of cell types, e.g. by inhibiting signalling pathways that allow progression from<br>
the G1 to the S-phase of the cell cycle (Kuo ef al., 1992). In T cells rapamycin inhibits<br>
signalling from the IL-2 receptor and subsequent autoproliferation of the T cells<br>
resulting in immunosuppression. The inhibitory effects of rapamycin are not limited to<br><br>
T cells, since rapamycin inhibits the proliferation of many mammalian cell types<br>
(Brunn ef al, 1996). Rapamycin is, therefore, a potent immunosuppressant with<br>
established or predicted therapeutic applications in the prevention of organ allograft<br>
rejection and in the treatment of autoimmune diseases (Kahan et al., 1991). It<br>
appears to cause fewer side effects than the standard anti-rejection treatments<br>
(Navia, 1996). 40-O-(2-hydroxy)ethyl-rapamycin (SDZ RAD, Certican, Everolimus) is<br>
a semi-synthetic analogue of rapamycin that shows immunosuppressive<br>
pharmacological effects (Sedrani, R. et al., 1996; U.S. 5,655,772).The clinical<br>
efficacy of the drug is presently under investigation in Phase III clinical trials (Kirchner<br>
et aL, 2000). The rapamycin ester CCI-779 (Wyeth-Ayerst) inhibits cell growth in vitro<br>
and inhibits tumour growth in vivo (Yu et al., 2001). The drug is currently in Phase 111<br>
clinical trials. The value of rapamycin in the treatment of chronic plaque psoriasis<br>
(Kirby and Griffiths, 2001), the potential use of effects such as the stimulation of<br>
neurite outgrowth in PC12 cells (Lyons et al., 1994), the block of the proliferative<br>
responses to cytokines by vascular and smooth muscle cells after mechanical injury<br>
(Gregory et a/., 1993) and its role in prevention of allograft fibrosis (Waller and<br>
Nicholson, 2001) are areas of intense research (Kahan and Camardo, 2001). Recent<br>
reports reveal that rapamycin is associated with lower incidence of cancer in organ<br>
allograft patients on long-term immunosuppressive therapy than those on other<br>
immunosuppressive regimes, and that this redueed cancer incidence is due to<br>
inhibition of angiogenesis (Guba et al., 2002). It has been reported that the<br>
neurotrophic activities of immunophifin ligands are independent of their<br>
immunosuppressive activity (Steiner et al., 1997) and that nerve growth stimulation is<br>
promoted by disruption of the mature steroid receptor complex as outlined in the<br>
patent application WO01/03692. Side effects such as hyperiipidemia and<br>
thrombocytopenia as well as potential teratogenic effects have been reported<br>
(Hentges et a/., 2001; Kahan and Camardo, 2001).<br>
. The polyketide backbone of rapamycin is synthesised by head-to-tail<br>
condensation of a total of seven propionate and seven acetate units to a shikimate<br>
derived cyclohexane carboxylic acid starter unit (Paiva et al., 1991). The L-lysine<br>
derived imino acid, pipecolic acid, is condensed via an amide linkage onto the last<br>
acetate of the polyketide backbone (Paiva et el, 1993) and is followed by<br>
lactonisation to form the macrocycle. A107 kb genomic region containing the<br>
biosynthetic gene cluster has been sequenced (Schwecke ef at., 1995). Analysis of<br>
the open reading frames revealed three large genes encoding the modular polyketide<br><br>
synthase (PKS) (Aparicio et al., 1996; Schwecke et al., 1995). Embedded between<br>
the PKS genes lies the rapP gene encoding a protein with sequence similarity to<br>
activation domains of nonribosomal peptide synthetases and it is thought to act<br>
analogously (Konig et al., 1997). The region encoding the PKS genes is flanked on<br>
both sides by 24 additional open reading frames encoding enzymes believed to be<br>
required for the biosynthesis of rapamycin (Molnar et al., 1996). These include the<br>
following post-polyketide modification enzymes: two cytochrome P-450<br>
monooxygenases, designated as RapJ and RapN, an associated ferredoxin RapO,<br>
and three potential SAM-dependent O-methyltransferases Rapl, RapM and RapQ.<br>
Other adjacent genes have putative roles in the regulation and the export of<br>
rapamycin (Molnar et al., 1996). The cluster also contains the gene rapL whose<br>
product RapL is proposed to catalyse the formation of the rapamycin precursor L-<br>
pipecolic acid through the cyclodeamination of L-lysine (Khaw et al., 1998; Paiva et<br>
al., 1993). The introduction of a frameshift mutation into rapL gave rise to a mutant<br>
unable to produce significant amounts of rapamycin and feeding of L-pipecolic acid to<br>
the growth medium restored wild-type levels of rapamycin production (Khaw et al.,<br>
1998). The biosynthetic precursors to the cyclohexane ring of rapamycin originate<br>
from the shikimic acid pathway (Lowden et al., 1996; Lowden et al., 2001). Other<br>
closely-related macrolides such as FK506 (tacrolimus) (Schreiber and Crabtree,<br>
1992), FK520 (ascomycin or immunomycin) (Wu et al., 2000), antascomicin (Fehr,<br>
T., et al., 1996) and meridamycin (Salituro et al., 1995) share a common<br>
pharmacophore that interacts with FK506-binding proteins (FKBPs) (Figure 2). Thus<br>
rapamycin and related compounds for example, but without limitation, FK506, FK520,<br>
'hyg', FK523, meridamycin, antascomicin, FK525 and tsukubamycin can be<br>
considered "FKBP-ligands". The partial sequence of the FK506 gene cluster<br>
(Motamedi et al., 1996; Motamedi et a/., 1997; Motamedi and Shafiee, 1998), the<br>
'hyg' cluster (Ruan et al., 1997) and the complete sequence of the FK520 gene<br>
cluster have been published (Wu et al., 2000; U.S. Patent No. 6,150,513). There is<br>
significant homology between genes within these clusters and the rapamycin<br>
biosynthetic gene cluster and similarity in enzyme function (Motamedi et al., 1996).<br>
The pharmacologic actions of rapamycin characterised to date are believed to<br>
be mediated by the interaction with cytosolic receptors termed FKBPs or<br>
immunophilins. lmmunophilins (this term is used to denote immunosuppressant<br>
binding proteins) catalyse the isomerisation of cis and trans peptidyl-proline bonds<br>
and belong to a highly conserved family of enzymes found in a wide variety of<br><br>
organisms (Rosen and Schreiber, 1992). Two large groups of enzymes belonging to<br>
the family of immunophilins are represented by FKBPs and cyclophilins (Schreiber<br>
and Crabtree, 1992). The major intracellular rapamycin receptor in eukaryotic T-cells<br>
is FKBP12 (DiLella and Craig, 1991) and the resulting complex interacts specifically<br>
with target proteins to inhibit the signal transduction cascade of the cell. FK506, an<br>
immunosuppressive agent structurally related to rapamycin, also specifically binds to<br>
FKBP12 but it effects immunosuppression through a different mechanism (Chang et <br>
al., 1991; Sigal and Dumont, 1992). Rapamycin and FK506 compete for the same<br>
binding site, thus FK506 can have an antagonistic effect with rapamycin when the<br>
two drugs are used together (Cao et al., 1995). Analysis of the crystal structure of<br>
the FKBP12-rapamycin complex has identified a rapamycin-binding pharmacophore<br>
termed the 'binding domain' (Van Duyne et al., 1993) (see Figure 1). The 'binding<br>
domain' is required for the interaction with the immunophilin and consists, for both<br>
FK506 and rapamycin, of the C-1 to C-14 region including the ester linkage, the<br>
pipecolinyl ring, the dicarbonyl and the hemiketal ring (see Figure 2). The interaction<br>
is characterised by many hydrophobic contacts and some hydrogen bonds including<br>
one to the hydroxyl group on the cyclohexane ring. The pipecolinyl ring (C2 to N7)<br>
makes the deepest penetration into the protein where it is surrounded by highly<br>
conserved aromatic amino acid, residues lining the hydrophobic binding cavity. Both<br>
the C1 and the CB carbonyl groups are involved in hydrogen bonding and the C9<br>
carbonyl group protrudes into a pocket formed by three completely conserved<br>
aromatic amino acid residues (one tyrosine and two phenylalanine acid residues) in<br>
FKBP12. The domain of the immunophilin-ligand complex interacting with the target<br>
protein projects away from FKBP.<br>
The target of the rapamycin-FKBP12 complex has been identified in yeast as<br>
TOR (target of rapamycin) (Alarcon et a/., 1999) and the mammalian protein is known<br>
as FRAP (FKBP-rapamycin associated protein) or mTOR (mammalian target of<br>
rapamycin) (Brown et a/., 1994). These proteins show significant similarity to the<br>
phosphotransferase domains of phosphatidylinositol 3- kinases and the observation<br>
that a point mutation in the FKBP12-rapamycin binding domain (FRB) of mTOR<br>
abolishes mTOR kinase activity provides evidence for the involvement of FRB in the<br>
function of the kinase domain (Vilella-Bach et al., 1999). The crystal structure of<br>
FKBP12-rapamycin with a truncated form of mTOR containing the FRB domain<br>
(Chen ef at., 1995) has been obtained thus defining the 'effector' domain of<br>
rapamycin (Choi et al., 1996; Liang et al., 1999). The analysis of the crystal structure<br><br>
revealed that protein-protein contacts are relatively limited compared to the<br>
interaction between rapamycin and each protein. No hydrogen bonds between<br>
rapamycin and FRB were identified. Interaction is concentrated in a series of<br>
hydrophobic contacts between the triene region of rapamycin and mainly aromatic<br>
residues of FRB (Liang et al., 1999). The most deeply buried atom of rapamycin is<br>
the methyl attached to C23 (see Figure 2). The C23 to C34 region and the<br>
cyclohexyl ring of rapamycin make superficial hydrophobic contacts with FRB. A<br>
small conformational change in rapamycin was evident between the binary and the<br>
ternary complexes (Liang et al., 1999).<br>
Divergences between the biological effects of C16 methcxy group rapamycin<br>
analogues and their ability to bind FKBP12 were detected and the location of the C16<br>
subsfrtuents at the interfacial space between FKBP12 and mTOR was postulated<br>
(Luengo et al., 1995). The analysis of the crystal structure of FKBP12 with the non-<br>
immunosuppressive 28-Omethyl rapamycin revealed a significant difference in the<br>
orientation of the cyclohexyl ring which may result in disruption of mTOR binding<br>
(Kallen et al., 1996).<br>
Rapamycin impacts signalling cascades within the cell through the inhibition<br>
of the p70S6k kinase, a serine/threonine kinase in higher eukaryotes which<br>
phosphorylates the ribosomal protein S6 (Ferrari et al., 1993; Kuo et al., 1992). The<br>
S6 protein is located in the ribosomal 40S subunit and it is believed to be an<br>
important functional site involved in tRNA and mRNA binding. A regulatory function<br>
for mRNA translation through S6 phosphorylation by p70S6k has been postulated<br>
(Kawasome et al., 1998). Rapamycin inhibits protein synthesis through its effect on<br>
other growth related events, including the activity of cyclin-dependent kinases,<br>
phosphorylation of cAMP-responsive element modulator (CREM) and<br>
phosphorylation of the elongation factor binding protein 4E-BB1 (PHAS1) (Hung et <br>
al., 1996). The drug induces the accumulation of the dephosphorylated species of<br>
4E-BP1 that binds to the translation initiation factor elF-4E, thus, suppressing<br>
translation initiation of cap-dependent mRNAs (Hara et al., 1997; Raught et al.,<br>
2001).<br>
A link between mTOR signalling and localized protein synthesis in neurons;<br>
the effect on the phosphorylation state of proteins involved in translational control;<br>
the abundance of components of the translation machinery at the transcriptional and<br>
translational levels; control of amino acid permease activity and the coordination of<br>
the transcription of many enzymes involved in metabolic pathways have been<br><br>
described (Raught et al., 2001). Rapamycin sensitive signalling pathways also<br>
appear to play an important role in embryonic brain development, learning and<br>
memory formation (Tang et al., 2002). Research on TOR proteins in yeast also<br>
revealed their roles in modulating nutrient-sensitive signalling pathways (Hardwick et <br>
al., 1999). Similarly, mTOR has been identified as a direct target for the action of<br>
protein kinase B and of having a key role in insulin signalling (Shepherd et al., 1998;<br>
Nave et al., 1999). Mammalian TOR has also been implicated in the polarization of<br>
the actin cytoskeleton and the regulation of translational initiation (Alarcon et al.,<br>
1999). Phophatidylinositol 3-kinases, such as mTOR, are functional in several<br>
aspects of the pathogenesis of tumours such as cell-cycle progression, adhesion, cell<br>
survival and angiogenesis (Roymans and Siegers, 2001).<br>
Most immunophilins do not appear to be directly involved in<br>
immunosuppressive activities and relatively little is known concerning their natural<br>
ligands although candidates for natural ligands of the FKBPs termed FKBP-<br>
associated proteins (FAP) such as FAP48 and FAP1 have been reported. The<br>
specific interaction of FAPs with FKBPs during the formation of complexes was<br>
prevented by rapamycin in a dose-dependent manner (Chambraud et al., 1996; Kunz<br>
et al., 2000). Immunophilins appear to function in a wide range of cellular activities<br>
such as protein folding; assembly and trafficking of proteins; co- regulation of<br>
molecular complexes including heat shock proteins; steroid receptors; ion channels;<br>
cell-to-cell interactions and transcription and translation of genes (Galat 2000;<br>
Hamilton and Steiner 1998). All immunophilins possess the protein folding property<br>
of peptidyl-prolyl cis-trans isomerisation and several immunophilins are found located<br>
in the endoplasmic reticulum, a principal site of protein synthesis in the cell. In<br>
addition to FKBP12 (U.S. 5,109,112) other immunophilins include FKBP12.6 (U.S.<br>
5,457,182), FKBP13 (Hendrickson et al., 1993; U.S. 5,498,597), FKBP25 (Hung and<br>
Schreiber, 1992; Jin et al., 1992), FKBP14.6 (U.S. 5,354,845), FKBP52 (U.S.<br>
5,763,590), FKBP60 (Yem et al., 1992) and FKBP65 (Patterson et al., 2000).<br>
The multitude of the FKBP's which are present in different cell types also<br>
underline the utility of isolating novel FKBP-ligand analogues with potentially<br>
changed binding and/or effector domains.<br>
Pharmacokinetic studies of rapamycin and rapamycin analogues have<br>
demonstrated the need for the development of novel rapamycin compounds that may<br>
be more stable in solution, more resistant to metabolic attack and have improved bio-<br>
availability. Modification using chemically available positions on the molecule has<br><br>
been addressed, however, this approach has limited utility as the sites available for<br>
chemical modification are limited and there is less ability to selectively modify a<br>
particular position. Biological approaches to producing novel rapamycin analogues<br>
have been less successful due to the difficulties encountered in working with the<br>
organism (Lomovskaya et al., 1997; Kieser et al., 2000) despite the availability of the<br>
sequence of the biosynthetic gene cluster of rapamycin from S. hygroscopicus<br>
(Schwecke et al., 1995).<br>
A range of synthesised rapamycin analogues using the chemically available<br>
sites of the molecule has been reported. The description of the following compounds<br>
Was adapted to the numbering system of the rapamycin molecule described in Figure<br>
1. Chemically available sites on the molecule for derivatisation or replacement<br>
include C40 and C28 hydroxyl groups (e.g. U.S. 5,665,772; U.S. 5,362,718), C39<br>
and C16 methoxy groups (e.g. WO96/41807; U.S. 5,728,710), C32, C26 and C9 keto<br>
groups (e.g. U.S. 5,378,836; U.S. 5,138,051; U.S. 5,665,772). Hydrogenation at<br>
C17, C19 and/or C21, targeting the triene, resulted in retention of antifungal activity<br>
but loss of immunosuppression (e.g. U.S. 5,391,730; U.S. 5,023,262). Significant<br>
improvements in the stability of the molecule (e.g. formation of oximes at C32, C40<br>
and/or C28, U.S. 5,563,145, U.S. 5,446,048), resistance to metabolic attack (e.g.<br>
U.S. 5,912,253), bioavailability (e.g. U.S. 5,221,670; U.S. 5,955,457; WO98/04279)<br>
and the production of prodrugs (e.g. U.S. 6,015,815; U.S. 5,432,183) have been<br>
achieved through derivatisation. However, chemical modification requires significant<br>
quantities of rapamycin template and, as a base and acid labile compound, it is<br>
difficult to work with. Where chemical derivatisation can be group selective, it is often<br>
difficult to be site selective. Consequently, chemical modification invariably requires<br>
multiple protective and deprotecive steps and produces mixed products in variable<br>
yields.	<br>
The isolation of rapamycin analogues using biological methods such as<br>
biotransformation and phage-based genetic modification has also been described,<br>
isolation of minor metabolites from both mutant strains and rapamycin producing<br>
strains has provided small quantities of a number of rapamycin analogues. These<br>
strains are often low yielding and produce mixtures of rapamycin analogues. The<br>
isolation of 27-O-desmethylrapamycin and 27-desmethoxyrapamycin was reported<br>
from the culture supernatant of S. hygroscopicus NCIMB 40319 (Box et al., 1995).<br>
The antifungal activity of 27-O-desmethylrapamycin was lower than that of rapamycin<br>
but the inhibition of FKBP12 PPIase activity seemed to be increased. The inhibition<br><br>
of ConA-stimuiated proliferation of murine splenic T cells and the inhibition of UPS-<br>
stimulated proliferation of murine splenic B cells was decreased when compared to<br>
rapamycin (Box et al, 1995). Similarly, antifungal activities of the rapamycin<br>
derivatives prolylrapamycin, 27-O-desmethylrapamycin and 27-<br>
desmethoxyrapamycin were lower than that of rapamycin (Wong et al., 1998).<br>
Rapamycin analogues (16-O-desmethylrapamycin, 27-O-desmethylrapamycin, 39-O-<br>
desmethylrapamycin, 16,27-O-bisdesmethylrapamycin, prolylrapamycin, 26-O-<br>
desmethylprolylrapamycin, 9-deoxorapamycin, 27-desmethoxyrapamycin, 27-<br>
desmethoxy-39-O-desmethylrapamycin, 9-deoxo-27-desmethoxyrapamycin, 28-<br>
dehydrorapamycin, 9-deoxo-27-desmethoxy-39-O-desmethylrapamycin) were also<br>
isolated from Actinoplanes sp N902-109 after the addition of cytochrome P450<br>
inhibitors and/or precursor feeding to the culture or after biotransformation of isolated<br>
rapamycin (Nishida et al., 1995). The use of such inhibitors, however, only allows<br>
the targeting of a particular enzyme function and is not site selective. Rational<br>
production of a single selected analogue is not possible via this method. The<br>
resulting production of mixtures of rapamycin analogues rather than a single desired<br>
product also impacts yield. The mixed lymphocyte reaction (MLR) inhibitory activity<br>
of the compounds was assessed and little effect on the activity was detected after the<br>
loss of the methyl group at C27 or/and C16. In addition, 9-deoxorapamycin showed<br>
a more significant decrease in activity and the loss of the methoxy group at C27, the<br>
hydroxy group at C28 and the substitution of a pipecolinyl group for a prolyl group<br>
resulted in a reduction in potency (Nishida et al., 1995). Similarly, biotransformation<br>
of rapamycin and the isolation of 16,39-O-bisdesmethylrapamycin have been<br>
reported (WO 94/09010). The retention of inhibitory activity in cell proliferation<br>
assays with compounds modified in the cyclohexyl ring, e.g. 39-O-<br>
desmethylrapamycin and C40 modifications such as SDZ RAD, identify this region of<br>
the molecule as a target for the generation of novel rapamycin analogues. Novel<br>
rapamycin analogues were reported after feeding cyclohexanecarboxylic acid,<br>
cycloheptanecarboxylic acid, cyclohex-1-enecarboxylic acid, 3-<br>
methylcyclohexanecarboxylic acid, cyclohex-3-enecarboxylic acid, 3-<br>
hydroxycyclohex-4-enecarboxylic acid and cyclohept-1-enecarboxylic acid to cultures<br>
of S. hygmscopicus thus demonstrating the flexibility in the loading module of the<br>
rapamycin polyketide synthase (PAS. Lowden, PhD dissertation, University of<br>
Cambridge, 1997). These novel rapamycin analogues were produced in competition<br><br>
with the natural starter, 4, 5-dihydroxycyclohex-1-enecarboxylic acid, resulting in<br>
reduced yields and mixed products.<br>
The isolation of recombinant S. hygroscopicus strains producing various<br>
rapamycin analogues, using biological methods mediated by phage technology<br>
(Lomovskaya et al., 1997), has been reported. In the presence of added proline<br>
derivatives, a S. hygroscopicus rapL deletion mutant synthesized the novel<br>
rapamycin analogues prolylrapamycin, 4-hydroxyprolylrapamycin and 4-<br>
hydroxyprolyl-26-desmethoxy-rapamycin (Khaw et al., 1998). Similarly, the novel<br>
rapamycins 3-hydroxy-prolyl-rapamycin, 3-hydroxy-pro!yl-26-desmethoxy-rapamycin,<br>
and trans-3-aza-bicyclof3,1,0]hexane-2-carboxylic acid rapamycin have been<br>
identified as described in WO98/54308. The activity of prolylrapamycin and 4-<br>
hydroxyprolyl-26-desmethoxy-rapamycin was assessed in proliferation assays and<br>
the inhibitory activity of the latter compound was significantly less than that of<br>
rapamycin (Khaw et al., 1998). The deletion of five contiguous genes, rapQONML<br>
(responsible for post-polyketide modifications at C16, C27 and production of L-<br>
pipecolic acid) and their replacement with a neomycin resistance marker in S.<br>
hygroscopicus ATCC29253 using phage-based methology resulted in the production<br>
of 16-O-desmethyl-27-desmethoxyrapamycin when fed with pipecolic acid (Chung et <br>
al., 2001). No complementation of this deletion mutant has been demonstrated using<br>
this technology. Furthermore, the site-specific functionality of rapM and rapQ<br>
remains unclear, therefore, rational design of rapamycin analogues requiring<br>
methylation at C16-OH or C27-OH has not been enabled. The phage-based<br>
methodology suffers from a number of drawbacks as described in more det all below.<br>
It offers a difficult and protracted process of obtaining engineered strains and has a<br>
reduced versatility in comparison to the methodology disclosed within this current<br>
patent.	<br>
Conventional approaches to manipulate rapamycin modifying genes using<br>
biological methods comprise the mutation or deletion of individual genes in the<br>
chromosome of a host strain or/and the insertion of individual genes as extra copies<br>
of homologous or heterologous genes either individually or as gene cassettes<br>
(WO01/79520, WO 03/048375). However, the isolation of novel rapamycin<br>
analogues using such biological methods has been limited due to the difficulties in<br>
transforming the rapamycin-producing organism S. hygroscopicus. It has been<br>
reported that the commonly used methods of transformation with plasmid DNA or<br>
conjugal transfer were Unsuccessful with the rapamycin producing strain (Lomovskya<br><br>
et al., 1997, Schweke et al., 1995, Kieser et al., 2000). The current state of the art<br>
uses the methodology of Lomovskya et al. (1997), a work intensive phage based<br>
method that is severely limited by the size of the cloned DNA fragments transferred<br>
into S. hygroscopicus (Kieser et al., 2000). This technology is limited to the transfer<br>
of a maximum of 6.4 kb of cloned DNA. Thus, when complementing a deletion<br>
mutant using this technology the artisan is limited to the inclusion of -2 functional<br>
genes in addition to desired promoter, regions of homology and resistance marker.<br>
The genetic information for the rapamycin biosynthetic gene cluster has been<br>
available since 1995 (Schwecke et al., 1995), however, limited progress in this area<br>
has been made (Khaw et al., 1998; Chung et al., 2001; WO01/34816).<br>
Summary of the Invention<br>
The present invention provides recombinant methods for the efficient<br>
transformation of strains that contain a biosynthetic cluster encoding an FKBP tigand,<br>
for example but without limitation Streptomyces hygroscopicus subsp. hygroscopicus<br>
NRRL 5491, Actinoplanes sp. N902-109 FERM BP-3832, Streptomyces sp. AA6554,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6475 ATCC14891,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6678 ATCC 55087,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6674, Streptomyces<br>
hygroscopicus var. ascomyceticus ATCC 55276, Streptomyces tsukubaensis<br>
No.9993 FERM BP-927, Streptomyces hygroscopicus subsp. yakushimaensis,<br>
Streptomyces sp. DSM 4137, Streptomyces sp. DSM 7348, Micromonospora n.sp.<br>
A92-306401 DSM 8429, Steptomyces sp. MA 685B ATCC 55098, Steptomyces sp.<br>
MA 6848, said methods comprising:<br>
(a) constructing a conjugative deletion plasmid in an £ coli strain that is<br>
dam , dcm or dam and dcm .	-<br>
(b) generation of spores from said strain suitable for conjugation wherein<br>
said strain is grown at a humidity of between 10% and 40% and the spores<br>
are harvested at between 5 and 30 days;<br>
(c) . conjugating the E. co// strain of step (a) with the spores from step (b)<br>
on a medium that comprises per litre:<br>
i) 0.5g to 5g corn steep powder,<br>
ii) 0.1 g to 5g Yeast extract,<br>
iii) 0.1 g to 10g calcium carbonate; and<br>
iv) 0.01 g to 0.5 g iron sulphate;<br>
said media additionally containing BACTO-agar and starch and having been<br><br>
dried to result in 1-20% weight loss; and<br>
(d) optionally culturing the strain under conditions suitable for polyketide<br>
production.<br>
In a preferred embodiment the methods are used for the transformation of<br>
Streptomyces hygroscopicus subsp. hygroscopicus (e.g. NRRL 5491), Actinoplanes<br>
sp. N9Q2-109 (e.g. FERM BP-3832), Streptomyces sp. AA6554, Streptomyces<br>
hygroscopicus var. ascomyceticus {e.g. MA 6475 ATCC 14891), Streptomyces<br>
hygroscopicus var. ascomyceticus {e.g. MA 6678 ATCC 55087), Streptomyces<br>
hygroscopicus var. ascomyceticus (e.g.MA 6674), Streptomyces hygroscopicus var.<br>
ascomyceticus {e.g. ATCC 55276), Streptomyces tsukubaensis No.9993 (e.g. FERM<br>
BP-927), Streptomyces hygroscopicus subsp. yakushimaensis, Streptomyces sp.<br>
{e.g. DSM 4137), Streptomyces sp. (e.g. DSM 7348), Micromonospora n.sp. A92-<br>
306401 (e.g. DSM 8429) or Streptomyces sp. (e.g. MA 6858 ATCC 55098). In a<br>
more preferred embodiment the methods are used for the transformation of: S.<br>
hygroscopicus subsp. hygroscopicus (e.g. NRRL 5491) or S. hygroscopicus var.<br>
ascomyceticus (e.g. ATCC 14891). In a still more highly preferred embodiment the<br>
methods are used for the transformation of the rapamycin producer S hygroscopicus<br>
subsp. hygroscopicus (e.g. NRRL 5491).<br>
Therefore the present invention also provides a recombinant strain that<br>
contains biosynthetic clusters that encode FKBP-ligands where one or more auxiliary<br>
genes have been deleted or inactivated using the methods as described herein.<br>
In a further aspect, the present invention provides recombinant methods and<br>
materials for expressing combinations of polyketide modification enzymes so as to<br>
produce novel polyketide analogues. In a specific embodiment, the present invention<br>
provides recombinant methods and materials for expressing the combinations of<br>
enzymes responsible for post-PKS modification and/or precursor supply from<br>
biosynthetic clusters that encode FKBP-ligands for example but without limitation<br>
rapamycin, FK506, FK520, FK523, FK525, antascomicin, meridamycin,<br>
tsukubamycin and analogues therof and methods for the production of analogues in<br>
recombinant host cells. In a preferred embodiment the recombinant methods and<br>
materials are used for expressing the combinations of enzymes responsible for post-<br>
PKS modification and/or precursor supply in the biosynthesis of rapamycin, FK520,<br>
FK506 and 'hyg' and methods for the production of rapamycin, FK520, FK506 and<br>
'hyg' analogues in recombinant host cells. In a more highly preferred embodiment<br>
the recombinant methods and materials are used for expressing the combinations of<br><br>
enzymes responsible for post-PKS modification and/or precursor supply in the<br>
biosynthesis of rapamycin and methods for the production of rapamycin analogues in<br>
recombinant host cells.<br>
Broadly, the present invention is concerned with the alteration of a gene<br>
system which has a core portion responsible for the production of a basic product,<br>
and a multiplicity of modifying genes responsible for effecting relatively small<br>
modifications to the basic product- e.g. effecting glycosylation, oxidation, reduction,<br>
alkylation, dealkylation, acylation or cyclisation of the basic product, and a multiplicity<br>
of precursor supply genes which are involved in the production of particular precursor<br>
compounds (e.g. pipecolate; 4,5 dihydroxycyclohex-1-ene carboxyiic acid). Thus the<br>
basic product may be a modular polyketide and the modifying genes may be<br>
concerned with glycosylation and/or other modifications of a polyketide chain, and<br>
the precursor supply genes may be involved in the production and/or incorporation of<br>
natural or non-natural precursors (e.g. pipecolate and/or 4,5 dihydroxycyclohex-1-<br>
ene carboxyiic acid in the rapamycin system).<br>
The core portion may not function properly or even at all in the absence of a<br>
precursor supply gene (unless a natural or unnatural precursor compound is supplied<br>
or is otherwise available).<br>
In one aspect the invention provides methods for the alteration of a gene<br>
system with a core portion that cannot function due to a deletion or inactivation of a<br>
precursor supply gene. Suitable gene systems include, but are not limited to, the<br>
rapamycin, antascomicin, FK520, FK506, 'hyg', FK523, meridamycin, FK525 and<br>
tsukubamycin biosynthetic clusters. In this aspect of the invention, the precursor<br>
supply gene lacking is preferably rapK or a homologue of rapK (e.g. fkbO in the<br>
FK506 or FK520 gene clusters). The gene system is preferably the rapamycin<br>
cluster. The precursor supply gene lacking is more preferably rapK. This aspect of<br>
the invention provides methods for the efficient production of a multiplicity of basic<br>
products through the incorporation of natural or non-natural precursors (e.g. 4,5-<br>
dihydroxycyclohex-1-ene carboxyiic acid). Methods may also embody further<br>
aspects as set out below.<br>
Another type of system is a non-ribosomal peptide ("NRP") system where the<br>
basic product is a peptide and the modifying genes are genes responsible for<br>
modifications to a peptide (glycosylation, reduction etc), and the precursor supply<br>
genes are genes involved in the production of unusual amino acid residues to be<br>
incorporated in the peptide. Systems can also be of mixed type, e.g. having a<br><br>
polyketide part and a part with a different biosynthetic origin, e.g. NRP. Indeed,<br>
rapamycin can be regarded as an example of this since the pipecolate residue is an<br>
amino acid residue added by an enzyme similar to ones found in NRP systems.<br>
These modifying genes and precursor supply genes may be regarded as<br>
"auxiliary genes" for polyketide synthesis and the term "auxiliary genes" as used<br>
herein may refer to modifying genes, precursor supply genes or both.<br>
The alteration of the gene system involves the creation of a functioning<br>
altered system in which the set of auxiliary genes has been altered. Thus one or<br>
more auxiliary genes (and preferably two or more, three or more, four or more, five or<br>
more, six or more or seven or more) may have been deleted (or rendered non-<br>
functional) and/or replaced by different genes.<br>
This may involve a "deletion system" comprising nucleic acid encoding a<br>
gene system lacking a multiplicity of functional auxiliary genes. This deletion system<br>
can then be complemented with one or more functional auxiliary genes (which may<br>
be the same as or different from the genes they replace). This can be carried out<br>
combinatorially, a deletion system being complemented by a multiplicity of different<br>
genes and sets of genes.<br>
An altered system which differs from the natural system in lacking one or<br>
more modifying functions could be produced (a) by producing a deletion system and<br>
restoring by complementation less than all of the deleted genes; or (b) by selectively<br>
deleting or inactivating genes of an existing system. In an altered system produced<br>
according to (b) genes may be inactivated by site-directed mutagenesis of an active<br>
site important in the protein function (active site point mutation), by truncation of the<br>
gene through a frameshift mutation, by an in-frame deletion of a section of the gene<br>
important to its function, such as an active site; partial deletion or inactivation by point<br>
mutation. These could all be carried out by double recombination and selecting for<br>
the mutant genotype, or by single recombination. In a preferred embodiment the<br>
altered system is produced by method (a). Such methods could also be used in<br>
producing a deletion system. The "complementation" approach (a) is preferably<br>
homologous, in that the "restored" genes are from the same gene cluster, however,<br>
heterologous complementation, wherein the "restored" genes are sslected from a<br>
different biosynthetic cluster that encodes FKBP-ligands, is also contemplated by the<br>
present invention. In a preferred embodiment the "restored" genes are essentially<br>
the same as the deleted genes, or are variants thereof, which perform similar<br>
functions.<br><br>
In a further aspect of the invention, an altered system with a deleted (or non-<br>
functional) precursor supply gene can be fed with alternative precursors so that it<br>
produces variant products.<br>
As applied to a polyketide synthase ("PKS") system, one preferred type of<br>
embodiment is a method for producing polyketides comprising: (a) providing a strain<br>
of an organism which contains one or more PKS genes expressible to produce a<br>
functioning PKS which can generate a polyketide in the organism, for example PKS<br>
genes that encode a FKBP-ligand, the organism lacking one or more (and preferably<br>
a plurality) of functional auxiliary genes naturally associated with said PKS genes<br>
which encode gene products capable of effecting respective modifications of the<br>
polyketide; and (b) effecting complementation by causing said organism to express<br>
one or more auxiliary genes, the expressed modifying genes constituting an<br>
incomplete set of auxiliary genes naturally associated with said PKS genes and/or<br>
comprising one or more variant auxiliary genes; and (c) culturing said strain and<br>
optionally isolating the polyketide analogues produced.<br>
The step of providing a strain of an organism containing one or more PKS<br>
genes may include a step of providing nucleic acid encoding a gene cluster<br>
comprising said one or more PKS genes and lacking said one or more auxiliary<br>
genes; and introducing said nucleic acid into the organism.<br>
The PKS genes are preferably rapamycin genes. The auxiliary genes which<br>
are lacking are preferably one or more of rapK, rap], rapQ, rapM, the contiguous<br>
genes rapN and O (herein designated as rapN/O), rapL and rapJ. In specific<br>
embodiments contemplated by the present invention:<br>
i) one auxiliary gene is lacking, for example rapK; rapl; rapQ; rapM; rapL,<br>
rapN/O or rapJ is lacking; preferably where one auxiliary gene is lacking it<br>
is selected from the group consisting of rapK; rapl; fapQ; rapM; rapN/O and rapJ;<br>
ii) two auxiliary genes are lacking for example: rapKrapl; rapKrapQ;<br>
rapKrapM; rapKrapN/O; rapKrapL; rapKrapJ; rapklrapQ; raplrapM;<br>
rapIrapNAD; raplrapL; raplrapJ; rapQrapM; rapQrapN/O; rapQrapL;<br>
rapQrapJ; rapMrapN/O; rapMrapL; rapMrapJ; rapN/OrapL; rapN/OrapJ or<br>
rapLrapJ are lacking;<br>
iii) three auxiliary genes are lacking for example: rapKraplrapQ;<br>
rapKraplrapM; rapKrapl rapN/O; rapKraplrapl; rapKraplrapJ;<br>
rapKrapQrapM; rapKrapQRapN/O; rapKrapQrapL; rapKrapQrapJ;<br><br>
rapKrapMrapN/O; rapKrapMrapL; rapKrapMrapJ; rapKrapN/OrapL;<br>
rapKrapN/OrapJ; rapKrapLrapJ; rap'mpQrapM; raplrspQrepN/O;<br>
raplrapQrapL; raplrapQrapJ; rapIrapMrapN/O; raplrapMrapL; rapl<br>
rapMrapJ; raplrapN/OrapL; raplrapN/OrapJ; raplrapLrapJ;<br>
rapQrapMrapN/O; rapQrapMrapL; rapQrapMrapJ; rapQrapN/OrapL;<br>
rapQrapN/OrapJ; rapQrapLrapJ; rapMrapN/OrapL; rapMrapN/OrapJ;<br>
rapMrapLrap or rapN/OrapLrapJ are lacking<br>
iv) four auxiliary genes are lacking, for example: rapKraplrapQrapM;<br>
rapKraplrapQrapN/O; rapKraplrapQrapL; rapKraplrapQrapJ;<br>
rapKrapIrapMmpN/O; rapKraplrapMrapL; rapKraplrapMrapJ;<br>
rapKraplrapN/OrapL; rapKraplrapN/OrapJ; rapKraplrapLrapJ;<br>
rapKrapQrapMrapN/O; rapKrapQrapMrapL; rapKrapQrapMrapJ;<br>
rapKrapQrapN/OrapL; rapK, rapQ, rapN/O, rapJ; rapKrapQrapLrapJ;<br>
rapKrapMrapN/OrapL; rapKrapMrapN/OrapJ; rapKrapMrapLrapJ;<br>
rapKrapN/OrapLrapJ; raplrapQrapMrapN/O; raplrapQrapMrapL; rapl<br>
rapQrapMrapJ; raplrapQrapN/OrapL; raplrapQrapN/OrapJ;<br>
raplrapQrapLrapJ; raplrapMrapN/OrapL; raplrapMrapN/OrapJ;<br>
raplrapMrapLrapJ; raplrapN/OrapLrapJ; rapQrapMrapN/OrapL;<br>
rapQrapMrapN/OrapJ; rapQrapMrapLrapJ; rapQrapN/OrapLrapJ or<br>
rapMrapN/OrapLrapJ are lacking;<br>
v) five auxiliary genes are lacking, for example: rapKraplrapQrapMrapN/O;<br>
rapKraptrapQrapMrapL; rapKraplrapQrapMrapJ;<br>
. rapKraplrapQrapN/OrapL; rapKraplrapQrap N/OrapJ;<br>
rapKraplrapQrapLrapJ;rapKraplrapMrapN/OrapL;<br>
rapKraplrapMrapN/OrapJ;rapKraplrapMrapLrapJ;<br>
rapKraplrapN/OrapLrapJ; rapKrapQrapMrapN/OrapL;<br>
- rapKrapQrapMrapN/OrapJ; rapKrapQrapMrapLrapJ;<br>
rapKrapQrapN/OrapLrapJ;rapKrapMrapN/OrapLrapJ;<br>
raplrapQrapMrapN/OrapL;raplrapQrapMrapN/OrapJ;<br>
raplrapQrapN/OrapLrapJ; raplrapMrapN/OrapLrapJ;<br>
rapQrapMrapN/OrapLrapJ or raplrapQrapMrapLrapJ are lacking;<br>
vi) six auxiliary genes are lacking for example: rapKraplrapQrapMrapN/OrapL;<br>
rapKraplrapQrapMrapN/OrapJ; rapKrapirapQrapMrapLrapJ;<br>
rapKraplrapQrapN/OrapLrapJ; rapKraplrapMrapN/OrapLrapJ;<br><br>
rapKrapQrapMrapN/OrapLrapJ or raplrapQrapMrapN/OrapLrapJ are<br>
lacking; or<br>
vii) seven auxiliary genes are lacking, e.g. rapKraplrapQrapMrapN/OrapLrapJ<br>
are lacking.<br>
The expression "lacking one or more functional auxiliary genes" covers both<br>
the lack of a gene and the presence of a gene but in a non-functioning state, e.g.<br>
because it has been specifically disabled.<br>
In one aspect, the invention provides a novel and expeditious route to the<br>
efficient incorporation of natural or non-natural precursors into FKBP-ligands. These<br>
include, but are not limited to, the rapamycin, antascomicin, FK520, FK506, hyg',<br>
FK523, meridamycin, FK525 and tsukubamycin polyketide synthase/non-ribosomal<br>
peptide synthase systems, the invention thus provides novel analogues of their<br>
respective natural products. In specific aspect, the invention provides a novel and<br>
expeditious route to the efficient incorporation of natural or non-natural precursors<br>
providing novel rapamycin analogues.<br>
Therefore in one-aspect the present invention provides a method of<br>
generating analogues of. FKBP-ligands which incorporate a non-natural starter unit, .<br>
said method comprising:<br>
(a)	generating a recombinant strain in which at least the rapK homologue has<br>
been deleted or inactivated; and<br>
(b)	feeding a non-natural starter unit to said strain<br>
In a preferred embodiment the recombinant strain is generated using the<br>
methods of the present invention.<br>
In further aspects the invention provides libraries of compounds and individual<br>
compounds available using such systems. Thus a typical compound is a variant of a<br>
compound naturally produced by a gene system which has a eore portion<br>
responsible for the production of a basic product, and a multiplicity of auxiliary genes<br>
responsible for effecting relatively small modifications to the basic product, the<br>
variant being producible by a system altered so that one or more of the auxiliary<br>
genes are absent, non-functional, or replaced by functional variants. A preferred<br>
class of compounds is rapamycin analogues corresponding to products of a<br>
rapamycin system wherein one or more of the genes selected from the group<br>
consisting of rapK, rapl, rapQ, rapM, rapN, rapO, rapL and rapJ genes are absent,<br>
non-functional or variant.<br><br>
In a further aspect, the present invention provides novel FKBP-analogues, in<br>
a preferred embodiment the present invention provides novel rapamycin analogues.<br>
Such compounds may have one or more useful properties, for example but without<br>
[imitation, utility as immunosuppressants, antifungal agents, anticancer agents,<br>
neuroregenerative agents, or agents for the treatment of psoriasis, rheumatoid<br>
arthritis, fibrosis and other hyperproliferative diseases.<br>
Definitions:<br>
As used herein the term "modifying gene(s)" includes the genes required for<br>
post-polyketide synthase modifications of the polyketide, for example but without<br>
limitation cytochrome P-450 monooxygenases, ferredoxins and SAM-dependent O-<br>
methyltransferases. In the rapamycin system these modifying genes include rapN/O,<br>
rapM, rapl, rapQ, and rapJ but a person of skill in the art will appreciate that PKS<br>
systems related to rapamycin (for example but without limitation: FK506, FK520,<br>
antascomicin, 'hyg', FK523, meridamycin, FK525 and tsukubamycin) will have<br>
homoiogues of at least a subset of these genes, some of which are discussed further<br>
below.<br>
As used herein the term "precursor supply gene(s)" includes the genes<br>
required for the supply of the natural or non-natural precursors, the genes required<br>
for the synthesis of any naturally or non-naturally incorporated precursors and the<br>
genes required for the incorporation of any naturally or non-naturally incorporated<br>
precursors. For example but without limitation in the rapamycin system these genes<br>
include rapL, rapK and rapP but a person of skill in the art will appreciate that PKS<br>
systems related to rapamycin (for example but without limitation: FK506, FK520,<br>
antascomicin, 'hyg', FK523, meridamycin, FK525 and tsukubamycin) will have<br>
homoiogues of these genes, some of which are discussed further below.<br>
As used herein, the term "auxiliary gene(s)" includes references to modifying<br>
genes, precursor supply genes or both modifying genes and precursor supply genes.<br>
As used herein, the term "precursor" includes the natural starter units (i.e.<br>
4,5-dihydroxycyclohex-1-ene carboxylic acid), non-natural starter units, and naturally<br>
incorporated amino acids (i.e. pipecolic acid) and non-naturally incorporated amino<br>
acids<br>
As used herein the term "non-natural starter unif refers to any compounds<br>
which can be incorporated as a starter unit in polyketide synthesis that are not the<br>
starter unit usually chosen by that PKS.<br><br>
As used herein, the term "FKBP-ligands" refers to compounds that bind to<br>
the immunophilin FKBP, such compounds preferentially contains an a, B-diketo<br>
amide where the B-keto is masked as an hemi-acet al. Such compounds include,<br>
without limitation, rapamycin, FK520, FK506, antascomicin, hyg', FK523,<br>
meridamycin, FK525 and tsukubamycin,<br>
As used herein, the term "biosynthetic clusters that encode FKBP-<br>
ligands" includes but is not limited to the gene clusters which direct the synthesis of<br>
rapamycin, FK506, FK520, 'hyg', FK523, antascomicin, meridamycin, FK525 and<br>
tsukubamycin.<br>
As used herein the term "strains that contain biosynthetic clusters that<br>
encode FKBP-ligands" includes but is not limited to: Streptomyces hygrvscopicus<br>
subsp. hygrvscopicus (e.g. NRRL 5491), Actinoplanes sp. N902-109 (e.g. FERM BP-<br>
3832), Streptomyces sp. AA6554, Streptomyces hygroscophus var. ascomyceticus<br>
MA 6475 (e.g. ATCC 14891), Streptomyces hygrvscopicus var. ascomyceticus MA<br>
6678 (e.g. ATCC 55087), Streptomyces hygroscopicus var. ascomyceticus MA 6674,<br>
Streptomyces hygroscopicus var. ascomyceticus (e.g. ATCC 55276), Streptomyces<br>
tsukubaensis No.9993 (e.g. FERM BP-927), Streptomyces hygrvscopicus subsp.<br>
yakushimaensis, Streptomyces sp. (e.g. DSM4137), Streptomyces sp. (e.g. DSM<br>
7348), Micromonospora n.sp. A92-306401 (e.g. DSM 8429) or Streptomyces sp. MA<br>
6858 (e.g. ATCC 55098).<br>
As used herein, the term urapK homologue" refers to homologues of the<br>
rapamycin gene rapKfrom other biosynthetic clusters that encode FKBP-ligands, for<br>
example but without limitation: the fkbO gene from the FK520 cluster, the fkbO gene<br>
from the FK506 cluster and the Orf5 in the 'hyg' cluster. Such rapK homologues perform the same function as rapK in the synthesis of these related FKBP-ligands,<br>
namely they are essential for the supply of the natural starter unit. Preferably, such<br>
rapK homologues have at least 40% sequence identity, preferably at least 60%, at<br>
least 70%, at least 80%, at least 90% or at least 95% sequence identity to the<br>
sequence of rapK as shown in Figure 27 (SEQ ID NO: 13).<br>
Det alled Description of the Invention<br>
In one aspect, the present invention provides a novel and expeditious method<br>
for the transformation of S. hygroscopicus. The use of phage technology for the<br>
isolation of genetically modified strains of S. hygroscopicus has previously been<br>
described (Khaw et al., 1998; Lomovskaya et al., 1997). However, no method other<br><br>
than transection has ever been reported for the introduction of DNA into the<br>
rapamycin producing strain S. hygroscopicus. Indeed, it has been stated previously<br>
that the commonly used methods of transformation with plasmid DNA or conjugal<br>
transfer were unsuccessful with the rapamycin-producing strain (Lomovskaya et al.,<br>
1997, Kieser et al., 2000; Schweke et al., 1995).<br>
In the present invention, surprisingly a conjugation protocol to successfully<br>
transform S. hygroscopicus was established as described in Example 1. The<br>
methodology was exemplified by the isolation of the deletion mutant in S.<br>
hygroscopicus MG2-10 (Example 2) and by the expression of genes and gene<br>
combinations as described in Examples 3, 5 and 15.<br>
Therefore, in one aspect the present invention provides a method for<br>
producing a recombinant strain that contains biosynthetic clusters that encode FKBP-<br>
ligands where one or more auxiliary genes have been deleted or inactivated said<br>
method comprising:<br>
(a)	construction of a conjugative plasmid in an E coli strain that is dam <br>
dcm or dam and dcm ;<br>
(b)	generation of spores from said strain suitable for conjugation wherein<br>
said strain is grown at a humidity of between 10% and 40% and the spores<br>
are harvested at between 5 and 30 days;<br>
(c)	conjugating the E. coli strain of step (a) with the spores from step (b)<br>
in a medium that comprises per litre:<br>
i) 0.5g to 5g corn steep powder,<br>
ii) 0.1g to 5g Yeast extract,<br>
iii) 0.1g to 10g calcium carbonate; and<br>
iv) 0.01 g to 0.5 g iron sulphate;<br>
said media additionally containing BACTO-agar and starch and having been<br>
dried to result in 1-20% weight loss; and<br>
(d)	optionally culturing the strain under conditions suitable for polyketide<br>
production.<br>
Preferably the E coli strain of step (a) is dam' and dcm".<br>
Preferably, in step (b) the spores are harvested at between 10 and 25 days or<br>
at between 14 and 21 days. In another embodiment, in step (b) the strain is grown<br>
at a humidity of between 10 and 20%.<br>
In a specific embodiment the starch in the media in step (c) used is wheat<br>
starch.<br><br>
In preferred embodiments the media used in step (c) comprises 1g to 4g corn<br>
steep powder, 1g to 4g Yeast extract, 1g to 5g calcium carbonate; and 0.2g to 0.4 g<br>
iron sulphate per litre. In a more preferred embodiment the media comprises per<br>
litre: 2.5g com steep powder, 3g Yeast extract, 3g calcium carbonate; and 0.3g iron<br>
sulphate;<br>
The complementation strategy disclosed in this invention provides an<br>
expeditious method to assess and identify the function of each auxiliary gene i.e.<br>
rapK, rapQ, rapN/O, rapM, rapL, rapJ and/or rapl in rapamycin biosynthesis. The<br>
gene product RapK has previously been identified as an interesting candidate for a<br>
pteridine-dependent dioxygenase that could also catalyse an oxidative step in the<br>
biosynthesis of rapamycin (Molnar et al., 1996). The homologous gene fkbO was<br>
identified in the biosynthetic gene cluster of FK506 and due to the structural similarity<br>
of rapamycin and FK506 a role for rapKin the oxidation of the C9 OH group was<br>
postulated (Motamedi et al., 1996). The findings in Examples 3,4 and 6, describing<br>
the rapK-dependent production of pre-rapamycin by S. hygroscopicus MG2-<br>
10[pSGsetrapK] suggests that RapK has at least an additional function in rapamycin<br>
biosynthesis.<br>
In another aspect, therefore, the methods of the present invention led to the<br>
elucidation of the function of RapK, namely that the expression of the rapK gene is<br>
essential for the accumulation of any cyclised macrolide product. In a further aspect,<br>
the present invention describes the complementation of S. hygroscopicus MG2-10<br>
with fkbO, the homologue of rapK from the FK520 cluster, with the surprising<br>
observation of fkbO dependent production of pre-rapamycin by S. hygroscopicus<br>
MG2-10[pMG169-1] (Example 11). It can be seen by one skilled in the art that fkbO<br>
fulfils a similar function in the production of FK520 as rapK and fkbO in the<br>
production of pre-rapamycin. Further, one skilled in the art will appreciate that other<br>
homologues of rapK, including but not limited to, fkbO in the FK506 cluster, fkbO in<br>
the FK520 cluster and Orf5 in the 'hyg' cluster also fulfil the same function. In a<br>
further aspect of the invention, homologues of rapK in biosynthetic clusters that<br>
encode FKBP-ligands, including, but not limited to, FK506, FK520, FK525,<br>
antascomicin, FK523, tsukubamycin, and 'hyg' can be deleted or inactivated,<br>
providing strains unable to make their respective known natural products. Similarly,<br>
the complementation strategy outlined above provides an expeditious method to<br>
investigate the function, specificity and order for the expressed products of auxiliary<br>
genes in the biosynthesis of other polyketides or non-ribosomal peptides.<br><br>
In a preferred class of embodiment, the present invention provides a method<br>
for the production of a recombinant host strain capable of producing rapamycin<br>
analogues, further involving the construction of genomic delations, including but not<br>
limited to rapQONMLKJl introduced into S. hygroscopicus and complementation or<br>
partial complementation by expressing single genes or combinations of genes,<br>
including but not limited to rapK, rap/, rapQ, rapM, the contiguous genes rapN and O<br>
(herein designated as rapN/O), rapL and rapJ, in gene cassettes. Further, the<br>
invention provides a method of producing said rapamycin analogues by culturing said<br>
recombinant host strain, and optionally isolating the rapamycin analogues produced.<br>
Thus, the recombinant strain MG2-10lpSGsetrapK], produced by complementation of<br>
the genomic deletion strain S. hygroscopicus MG2-10, with rapK, was cultured to<br>
produce 9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin (pre-<br>
rapamycin).<br>
In a further aspect of this class of the invention, the strategy involves the<br>
integration of a vector comprising a sub-set of genes including, but rot limited to,<br>
rapK, rapl, rapQ, rapM, rapN, rapO, rapL and rapJ into the S. hygroscopicus deletion<br>
mutant above. Such.integration may be performed using a variety of available<br>
integration functions including but not limited to: OC31 -based vectors, vectors based<br>
on pSAM2 integrase (e.g. in pPM927 (Smovkina et al., 1990)), R4 integrase (e.g. in<br>
pAT98 (Matsuura et al., 1996)), $VWB integrase (e.g. in pKT02 (Van Mellaert et al.,<br>
1998)), 4&gt;BT1 integrase ((e.g. pRT801) Gregory et al., in press) and L5 integrase<br>
(e.g. Lee et at., 1991). In some cases this may need alteration of the host strain by<br>
addition of the specific attB site for the integrase to enable high efficiency integration.<br>
Replicating vectors could also be used, either as replacements to, or in addition to<br>
0C31-based vectors. These include, but are not limited to, vectors based on plJ101<br>
(e.g. plJ487, Kieser et al., 2000), pSG5 (e.g. pKC1139, Bierman et al., 1992) and<br>
SCP2*te.g. plJ698, Kieser et al., 2000). This methodology has been exemplified<br>
herein by the use of the ΦBT1 and ΦC31 site-specific integration functions.<br>
Although the introduction of gene cassettes into S. hygroscopicus has been<br>
exemplified in the present invention using the ΦBT1 and the ΦC31 site-specific<br>
integration functions, those skilled in the art will appreciate that there are a number of<br>
different strategies described in the literature, including those mentioned above that<br>
could also be used to introduce such gene cassettes into prokaryotic, or more<br>
preferably actinomycete, host strains. These include the use of alternative site-<br>
specific integration vectors as described above and in the following articles (Kieser et<br><br>
al., 2000; Van Mellaert et al., 1998; Lee ef at., 1991; Smovkina et al., 1990; Matsuura<br>
et al., 1996). Alternatively, plasmids containing the gene cassettes may be<br>
integrated into a neutral site on the chromosome using homologous recombination<br>
sites. Further, for a number of actinomycete host strains, including S. hygroscopicus,<br>
the gene cassettes may be introduced on self-replicating plasmids (Kieser et al.,<br>
2000;WO98/01571).<br>
In a further aspect of this class, the invention provides gene cassettes for the<br>
complementation of the recombinant S. hygroscopicus deletion strains. Methods of<br>
constructing gene cassettes and their heterologous use to produce hybrid<br>
glycosylated macrolides have been previously described (Gaisser et al., 2002;<br>
WO01/79520, WO 03/048375). The cloning method used to isolate the gene<br>
cassettes of the present invention differs significantly from the approach previously<br>
described in that the gene cassette is assembled directly in an expression vector<br>
rather than pre-assembling the genes in pUC18/19- plasmids, thus providing a more<br>
rapid cloning procedure. The approach is exemplified as described in Example 3,4,<br>
5, 9 and 15. As described herein, a suitable vector (for example but without limitation<br>
pSGLitl) can be constructed for use in the construction of said gene cassettes,<br>
where a suitable restriction site (for example but without limitation Xba\), sensitive to<br>
dam methylation is inserted 5' to the gene(s) of interest and a second restriction site<br>
(for example Xba\) can be inserted 3' to the genes of interest The skilled artisan will<br>
appreciate that other restriction sites may be used as an alternative to Xba\ and that<br>
the methylation sensitive site may be 5' or 3' of the gene(s) of interest.<br>
The use of gene cassettes enables the rapid and parallel generation of<br>
multiple recombinant strains deleted in any combination of modifying genes from a<br>
single S. hygroscopicus deletion strain. The cloning strategy facilitates the assembly<br>
of a library of gene cassettes in either a directed or random manner, and is therefore<br>
a powerful tool for the combinatorial production of novel rapamycin analogues<br>
including but not exclusively limited to 9-deoxo-16-O-desrnethyl-27-desrnethoxy-39-<br>
O-desmethyl-rapamycin (pre-rapamycin), 9-deoxo-16-O-desmethyl-27-O-desmethyl-<br>
39-O-desmethyl-rapamycin, 16-O-desmethyl-27-desmethoxy-39-O-desmethyl-<br>
rapamycin, 9-deoxo-16-O-desmethyl-39-O-desmethyl-rapamycin, 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-rapamycin, 16-O-desmethyl-27-O-desmethyl-39-O-<br>
desmethyl-rapamycin , 9-deoxoi27-O-desmethyl-39-O-desmethyl-rapamycin , 9-<br>
deoxo-16-O-desmethyl-27-O-desmethyl-rapamycin , 27-O-desmethyl-39-O-<br>
desmethyl-rapamycin , 9-deoxo-16-O-desmethyl-rapamycin , 9-deoxo-39-O-<br><br>
desmethyl-rapamycin , 8-deoxo-15-O-desmethyl-26-desmethoxy-38-O-desmethyl-<br>
prolylrapamycin (pre-prolylrapamycin), 8-deoxo-1 &amp;-O-desmethyl-26-O-desmethyl-38-<br>
O-desmethyl-prolylrapamycin, 15-O-desmethyl-26-desmethoxy-38-O-desmethyl-<br>
prolylrapamycin, 8-deoxo-26-desmethoxy-38-O-desmethyi-prolylrapamycin, 8-<br>
deoxo-15-O-desmethyl-38-O-desmethyl-prolylrapamycin , 8-deoxo-15-O-desmethyl-<br>
26-desmethoxy-proIylrapamycin, 15-O-desmethyl-26-O-desmethyl-3B-O-desmethyi-<br>
prolylrapamycin , 8-deoxo-26-O-desmethyI-38-O-desmethyl-prolylrapamycin , 8-<br>
deoxo-15-O-desmethyl-26-O-desmethyl-prolylrapamycin , 15-O-desmethyl-38-O-<br>
desmethyl-prolylrapamycin , 15-O-desmethyl-26-O-desmethyl-prolylrapamycin , 15-<br>
O-desmethyi-26-desmethoxy-prolylrapamycir., 26-desmethoxy-38-O-desmethyl-<br>
prolylrapamycin , 26-O-desmethyl-38-O-desmethyl-prolylrapamycin , 8-deoxo-15-O-<br>
desmethyl-prolylrapamycin , 8-deoxo-26-O-desmethyl-prolylrapamycin , 8-deoxo-38-<br>
O-desmethyl-prolylrapamycin , 15-O-desmethyl-prolylrapamycin , 38-O-desmethyl-<br>
prolylrapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-39-desmethoxy-<br>
rapamycin, 9-deoxo-16-O-desmethyl-27-O-desmethyI-39-desmethoxy-rapamycin,<br>
16-O-desmethyl-27-desmethoxy-39-desmethoxy-rapamycin, 9-deoxo-27-<br>
desmethoxy-39-desmethoxy-rapamycin, 9-deoxo-16-O-desmethyl-39-desmethoxy-<br>
rapamycin , 16-O-desmethyl-27-O-desmethyl-39-desmethoxy-rapamycin , 9-deoxo-<br>
27-O-desmethyl-39-desmethoxy-rapamycin, 16-O-desmethy!-39-desmethoxy-<br>
rapamycin , 27-desmethoxy-39-desmethoxy-rapamycin , 27-O-desmethyl-39-<br>
desmethoxy-rapamycin, 9-deoxo-39-desmethoxy-rapamycin , 8-deoxo-15-O-<br>
desmethyl-26-desmethoxy-38-desmethoxy-pro!ylrapamycin, 8-deoxo-15-O-<br>
desmethyi-26-O-desmethyl-38-desmethoxy-proIylrapamycin, 15-O-desmethyl-26-<br>
desmethoxy-38-desmethoxy-prolylrapamycin, 8-deoxo-26-desmethoxy-38-<br>
desmethoxy-prolylrapamycin, 8-deoxo-15-O-desmethyl-38-desmethoxy-<br>
prolylrapamycin, 15-O-desmethyl-26-O-desmethyl-38-desmeftioxy-prolylrapamycin ,<br>
8-deox6-26-O-desmethyl-38-desmethoxy-pro)ylrapamycin, 15-O-desmethyI-38-<br>
desmethoxy-prolylrapamycin , 26-desmethoxyr38-desmethoxy-prolylrapamycin , 26-<br>
O-desmethyl-38-desmethoxy-prolylrapamycin, 8-deoxo-38-desmethoxy-<br>
prolylrapamyctn, 38-desmethoxy-prolylrapamycin, 9-deoxo-16-O-desmethyl-27-<br>
desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-<br>
(hydroxycyclohexenyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-<br>
cis-methoxy-4-trans-hydroxycyclohexyl)-36-(dihydroxy cyciohexyl) rapamycin, 9-<br>
deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-<br>
hydroxycyclohexyI)-36-(hydroxynorbomyl) rapamycin, 9-deoxo-16-O-desmethyl-27-<br><br>
desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-methyl-4-<br>
hydroxycyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-<br>
cis-methoxy-4-trans-hydroxycyclohexyl)-36-(4-methylhydroxycyclohexyl) rapamycin,<br>
9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-<br>
hydroxycyclohexyl)-36-(3-fluoro-4-hydroxycyclohexyl) rapamycin, 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-<br>
hydroxy-4-fluorocyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-<br>
de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-chloro-4-hydroxycyciohexyl)<br>
rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-<br>
hydroxycyclohexyl)-36-(3-hydroxy-4-chlorocyclohexylJ rapamycin, 9-deoxc~16-O-<br>
desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-<br>
cis-4-cis-dihydroxycyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-<br>
36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-trans-4-trans-<br>
dihydroxycyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-O-desmethyl-39-O-<br>
desmethyl rapamycin, 9-deoxo-16-O-desmethyl-270-desmethyl-36-de(3-cis-<br>
methoxy-4-trans-hydroxycyc]ohexyl)-36-(hydroxycyclohexenyl) rapamycin, 9-deoxo-<br>
16-O-desmethyl-27-O-desmethyl-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-<br>
36-(hydroxynorbomyl) rapamycin, 9-deoxo-16-O-desmethyl-27-O-desmethyl-36-<br>
de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(4-methyl hydroxycyclohexyl)<br>
rapamycin.<br>
in a further aspect of this class, the present invention provides a system for<br>
the combinatorial production of recombinant host strains capable of producing rapamycin analogues, involving construction of a genomic deletion rapQONMLKJI<br>
introduced into S. hygroscopicus and its partial complementation by a combinatorial<br>
library of gene cassettes comprising one or a plurality of the deleted auxiliary genes<br>
rapQ, rapN/O, rapM, rapL, rapK, rapJ, and rap/.	-<br>
The approach outlined comprises as a part the cloning strategy to combine<br>
genes including but not exclusively limited to rapK, rapl, rapQ, rapM, rapN/O, rapL<br>
and rapJ, and / or genes with similar gene functions, in any possible gene<br>
combination and gene order.<br>
Another aspect of the invention allows the enhancement of gene expression<br>
by changing the order of genes in a gene cassette. As applied to the preferred class,<br>
the genes may comprise one or more of rapK, rapl, rapQ, rapM, rapN/O, rapL and<br>
rapJ and / or genes with similar functions, allowing the arrangement of the genes in a<br>
multitude of permutations as outlined in Example 5.<br><br>
The cloning strategy outlined in this invention also allows the introduction of a<br>
histidine tag in combination with a terminator sequence 3' of the gene cassette to<br>
enhance gene expression. Those skilled in the art will appreciate other terminator<br>
sequences could be used.<br>
Another aspect of the invention describes the multiple uses of promotor<br>
sequences in the assembled gene cassette to optimise gene expression.<br>
It will now be obvious to one skilled in the art that S. hygroscopicus deletion<br>
strains, the deletion comprising, but not limited to, a gene or a sub-set of the genes<br>
rapQ, rapN/O, rapM, rapL, rapK, rapJ and rapl could be constructed. In this case,<br>
gene cassettes for complementation or partial complementation would generally<br>
comprise single genes or a plurality of genes selected from the sub-set of the genes<br>
deleted.<br>
It is well known to those skilled in the art that there are homologues to several<br>
of the rapamycin modifylng and precursor supply genes in the gene clusters of<br>
closely related systems including FK506 (Motamedi et al, 1996; Motamedi et al,<br>
1997; Motamedi &amp; Shafiee, 1998) and FK520 (Wu et al, 2000). These include the<br>
following as described in Table I below:<br><br>
Although the gene clusters of other closely related systems, including but not<br>
limited to those for the biosynthesis of FK523, meridamycin, FK525, antascomicin<br>
and tsukubamycin have not yet been sequenced, it can be anticipated that these will<br>
be shown to bear a close resemblance to those whose sequences have been<br>
determined, and, in particular, that these gene clusters will contain close homologues<br>
of several of the rapamycin modifylng and precursor supply genes. Therefore, in a<br><br>
further aspect of the invention, genes from heterologous gene clusters from such<br>
closely related systems, including but not limited to FK506, FK520, FK523,<br>
antascomicin, meridamycin, FK525, 'hyg' and tsukubamycin can be included in gene<br>
cassettes in place of or in addition to their rapamycin homologues for<br>
complementation and/or partial complementation of a rapamycin producer strain<br>
containing a gene deletion or deletions including but not limited to the genes rapK,<br>
rapl, rapQ, rapM, rapN/O, rapL and rapJ.<br>
It is well known to those skilled in the art that polyketide gene clusters may be<br>
expressed in heterologous hosts (Pfeifer and Khosla, 2001). Accordingly, the present<br>
invention includes the transfer of the rapamycin biosynthetic gene cluster with or<br>
without resistance and regulatory genes, either complete or containing deletions, for<br>
complementation in heterologous hosts. Methods and vectors for the transfer as<br>
defined above of such large pieces of DNA are well known in the art (Rawlings, 2001;<br>
Staunton and Weissman, 2001) or are provided herein in the methods disclosed. In<br>
this context a preferred host cell strain is a prokaryote, more preferably an<br>
actinomycete or Escherichia coli, still more preferably include, but are not limited to<br>
S. hygroscopicus, S. hygroscopicus sp., S. hygroscopicus var. ascomyceticus,<br>
Streptomyces tsukubaensis, Streptomyces coeficolor, Streptomyces Jividans,<br>
Saccharopolyspora erythraea, Streptomyces fradiae, Streptomyces avermitilis,<br>
Streptomyces cinnamonensis, Streptomyces rimosus, Streptomyces albus,<br>
Streptomyces griseofuscus,. Streptomyces longisporoflavus, Streptomyces<br>
venezuelae, Micromonospora griseorubida, Amycolatopsis mediterranei or<br>
Actinoplanes sp. N902-109.<br>
In another aspect, the rapamycin analogues of the invention may be obtained by<br>
a process comprising the steps of:<br>
a)	constructing a deletion strain, by the methods of the invention; the deletion<br>
including, but not limited to, the genes rapK, rapQ, rapN/O, rapM, rapL, rapJ<br>
and rapl, or a sub-set thereof,<br>
b)	culturing the strain under conditions suitable for polyketide production;<br>
c)	optionally, isolating the rapamycin analogue intermediate produced;<br>
d)	constructing a biotransformation strain containing a gene cassette comprising<br>
all or a sub-set of the genes deleted;<br>
e)	feeding the rapamycin analogue intermediate in culture supernatant or<br>
isolated as in step c) to a culture of the biotransformation strain under suitable<br>
biotransformation conditions<br><br>
f) optionally isolating the rapamycin analogue produced.<br>
Suitable host strains for the construction of the biotransformation strain<br>
include the native host strain in which the rapamycin biosynthetic gene cluster has<br>
been deleted, or substantially deleted or inactivated, so as to abolish polyketide<br>
synthesis, or a heterologous host strain. Methods for the expressing of gene<br>
cassettes comprising one or a plurality of modifylng or precursor supply genes in<br>
heterologous hosts are described in WO 01/79520. In this context heterologous<br>
hosts suitable for biotransformation of the said FKBP-ligand analogue intermediates<br>
include, but are not limited to, S. hygroscopicus, S. hygroscopicus sp., S.<br>
hygroscopicus van ascomyceticus, Streptomyces tsukubaensis, Streptomyces<br>
coelicolor, Streptomyces IMdans, Saccharopolyspora erythraea, Strsptomyces<br>
fradiae, Streptomyces avermitilis, Streptomyces cinnamonensis, Streptomyces<br>
rimosus, Streptomyces albus, Streptomyces griseofuscus, Streptomyces<br>
iortgisporoHavus, Streptomyces venezuelae, Micromonospora griseorubida,<br>
Amycolatopsis mediterranei, Escherichia coli and Actinoplanes sp. N902-109.<br>
The close structural relationship between rapamycin and FK506, FK520,<br>
FK523, 'hyg', meridamycin, antascomicin, FK525 and tsukubamycin, among others,<br>
and tine established homologies between genes involved in the biosynthesis of<br>
rapamycin and FK506 and FK520 (vide supra), renders obvious the application of the<br>
methods of the present invention to these closely related systems. In a further<br>
aspect, therefore, the invention includes the construction of deletion strains of the<br>
producer strains of closely related compounds, including but not limited to FK506,<br>
FK520, FK523, 'hyg', antascomicin, meridamycin, FK525 and tsukubamycin<br>
containing a gene deletion or deletions of modifylng and/or precursor supply genes,<br>
and more particularly including but not limited to genes with similar functions as rapK,<br>
rapl, rapQ, rapM, rapN/O, rapL and rapJ, and their complementation or partial<br>
complementation with a gene or gene cassettes comprising all or a sub-set of the<br>
deleted homologous genes, or their functional homologues from heterologous gene<br>
clusters, including but not limited to rapK, rapl, rapQ, rapM, rapN/O, rapL and rapJ to<br>
produce recombinant strains capable of producing polyketide analogues varylng from<br>
the parent polyketide in the incorporation of alternative precursors and/or the extent<br>
of post-PKS modification. Further, the invention provides a method of producing said<br>
polyketide analogues by culturing said recombinant host strains, and optionally<br>
isolating the polyketide analogues produced.<br><br>
In a further aspect, the invention provides a method for the production of<br>
recombinant host strains capable of producing polyketide FKBP-ligand analogues<br>
(other than rapamycin) varylng from the parent polyketide in the incorporation of<br>
alternative precursors and/or the extent of post-PKS modification, comprising the<br>
construction of a genomic deletion strain from which all or a portion of the auxiliary<br>
genes have been removed, and its partial complementation by a gene cassette<br>
comprising one or a plurality of the deleted genes and/or their homologues, and<br>
further a method of producing said polyketide analogues by culturing said<br>
recombinant host strain, and optionally isolating the polyketide analogues produced.<br>
It is well known in the art that in most cases that auxiliary genes are co-located with<br>
polyketide synthase genes in a gene cluster (Hopwood, 1997; Motamedi and<br>
Shafiee, 1998; Wu et al., 2000) thus facilitating creation of the deletion strain. The<br>
auxiliary genes to be deleted may or may not naturally form a contiguous sequence,<br>
however, once the deletion strain has been created the partial complementation by<br>
gene cassettes provides an expeditious approach to the production of recombinant<br>
strains in which one or a plurality of the said genes have been deleted. Therefore, in<br>
a further aspect, the invention provides a method for the combinatorial production of<br>
recombinant host strains capable of producing polyketide FKBP-ligand analogues<br>
(other than rapamycin) varylng from the parent polyketide in the incorporation of<br>
alternative precursors and/or the extent of post-PKS modification, comprising the<br>
partial complementation of the said genomic deletion strain by a combinatorial library<br>
of gene cassettes comprising one or a plurality of the deleted genes, and further a<br>
method of producing said polyketide analogues by culturing said recombinant host<br>
strains under conditions suitable for polyketide production, and optionally isolating<br>
the polyketide analogues produced. In this context a preferred recombinant host cell<br>
strain is a prokaryote, more preferably an actinomycete, still more preferably a strain<br>
selected from S. hygroscopicus, S. hygroscopicus sp., S. hygroscopicus var.<br>
ascomyceticus, Stneptomyces tsukubaensis, Streptomyces coelicolor, Streptomyces<br>
lividans, Saccharopolyspora erythraea, Streptomyces fradiae, Streptomyces<br>
avermitilis, Streptomyces cinnamonensis, Steptomyces rimosus, Streptomyces<br>
afbus, Streptomyces griseofuscus, Streptomyces longisporoflavus, Streptomyces<br>
venezuelae, Micromonospora griseorubida, Amycolatopsis mediterranei or<br>
Actinoplanes sp. N902-109.<br>
Those skilled in the art will appreciate that the methods of the present<br>
invention could be applied to recombinant host strains in which the polyketide<br><br>
synthase (PKS) has been altered by genetic engineering to express a modified<br>
rapamycin or other polyketide analogue. The prior art describes several methods for<br>
the production of novel polyketides by the deletion or inactivation of individual<br>
domains (W093/13663, W097/92358), construction of hybrid polyketide synthases<br>
(WO98/01546, WO00/00618, WO00/01827) or alteration of domain specificity by<br>
site-directed mutagenesis (WO02/14482).<br>
It is well known in the art that non-ribosomal peptides are biosynthesised by<br>
Non-Ribosomal Peptide Synthases (NRPSs) via the stepwise condensation of<br>
successive amino acid building blocks, in a process analogous to that of polyketide<br>
biosynthesis (for review see Marahiel et al., 1997; Schwarzer and Marahiel, 2001). It<br>
is well known that several non-ribosomal peptides include unusual amino-acid<br>
residues (modified, proteinogenic amino acids and / or non-proteinogenic amino<br>
acids) and carboxy acids, the biosynthetic genes for which are co-located with the<br>
non-ribosomal peptide synthase genes in the non-ribosomal peptide gene cluster<br>
(Marahiel et al., 1997; Konz and Marahiel, 1999; Blanc et al., 1997). In several<br>
cases, the non-ribosomal peptide product initially released from the NRPS is further<br>
modified by a set of enzymes, including but not limited to glycosyl transferases,<br>
reductases, acylation or heterocyclic ring formation (Konz and Marahiel, 1999; Blanc<br>
et ah, 1995). These include the antibiotics chloroeremomycin, pristinamycin,<br>
vancomycin and bleomycin (Konz and Marahiel, 1999; Du et al., 2000). The genes<br>
for these post-NRPS enzymes are also typically co-located in the biosynthetic gene<br>
cluster (Marahiel et al., 1997; Schwarzer and Marahiel, 2001). Therefore, in a further<br>
aspect, the invention includes a method for the production of non-ribosomal peptide<br>
analogues, varylng from the parent non-ribosomal peptide in the incorporation of<br>
alternative precursor amino-acids and/or the extent of post-NRPS modification,<br>
comprising the construction of a genomic deletion strain from Which all or a portion of<br>
the genes encoding the native amino-acid precursor synthesis and/or post-NRPS<br>
enzymes have been removed, and its partial complementation by a gene cassette<br>
comprising one or a plurality of the deleted genes and/or their homologues, and<br>
further a method of producing said non-ribosomal peptide analogues by culturing<br>
said recombinant host strain, and optionally isolating the non-ribosomal peptide<br>
analogues produced. The post-NRPS and precursor biosynthesis genes to be<br>
deleted may or may not naturally form a contiguous sequence, however, once the<br>
deletion strain has been created the partial complementation by gene cassettes<br>
provides an expeditious approach to the production of recombinant strains in which<br><br>
one or a plurality of the said genes have been deleted. Therefore, in a further<br>
aspect, the invention provides a method for the combinatorial production of<br>
recombinant host strains capable of producing non-ribosomal peptide analogues<br>
varylng from the parent non-ribosomal peptide in the incorporation of alternative<br>
precursors and/or the extent of post-NRPS modification, comprising the partial<br>
complementation of the said genomic deletion strain by a combinatorial library of<br>
gene cassettes comprising one or a plurality of the deleted genes, and further a<br>
method of producing said non-ribosomal peptide analogues by culturing said<br>
recombinant host strains under conditions suitable for non-ribosomal peptide<br>
production, arid optionally isolating the non-ribosomal peptide analogues produced.<br>
In this context a preferred recombinant host cell strain is a prokaryote, more<br>
preferably an actinomycete, still more preferably a strain selected from S.<br>
hygroscopicus, S. hygroscopicus sp., S. hygroscopicus var. ascomyceticus,<br>
Streptomyces tsukubaensis, Streptomyces coelicolor, Streptomyces fividans,<br>
Saccharopolyspora erythraea, Streptomyces fradiae, Streptomyces avermitilis,<br>
Streptomyces cinnamonensis, Streptomyces rimosus, Streptomyces albus,<br>
Streptomyces griseofuscus, Streptomyces longisporoflavus, Streptomyces<br>
venezuelae, Micromonospora griseorubida, Amycolatopsis mediterranei or<br>
Actinoplanes sp. N902-109.<br>
It is well known that many actinomycetes contain multiple biosynthetic gene<br>
clusters for different secondary metabolites, including polyketides and non-<br>
ribosomally synthesised peptides. Specifically, it has been demonstrated that strains<br>
of S. hygroscopicus produce a variety of polyketides and non-ribosomally<br>
synthesised peptides in addition to rapamycin, FK506, FK520, FK523, meridamycin,<br>
FK525, antascomicin and tsukubamycin. These include, but are not limited to,<br>
elaiophylin, bialaphos, hygromycin, augustmycin, endomycin (ft, B), glebomycin,<br>
hygroscopin, ossamycin and nigericin. These additional biosynthetic gene clusters<br>
represent a competing requirement for.biosynthetic precursors and an additional<br>
metabolic demand on the host strain. In order to enhance production of the desired<br>
rapamycin, or other polyketide, analogues, it may therefore be advantageous to<br>
delete or inactivate any other biosynthetic gene clusters present in the host strain.<br>
Methods for the deletion or inactivation of biosynthetic gene clusters are well known<br>
in the art.<br>
In a further aspect of this class, the invention provides a mutasynthesis<br>
methodology for the complementation of recombinant deletion strains<br><br>
In a further aspect, S. hygroscopicus strains of the present invention<br>
containing a deletion of rapL may be fed with analogues of the naturally incorporated<br>
amino acid, L-pipecolic acid, to produce new analogues of rapamycin in which the<br>
pipecolyl residue is replaced. Prior art describes that a rapL mutant can be<br>
complemented by the addition of L-pipecolic acid to the culture (Khaw et al., 1998).<br>
Similarly, it was demonstrated that rapamycin analogues were isolated after the<br>
feeding and incorporation of L-pipecolic acid analogues, L-proline, L-trans-4-<br>
hydroxyproline, L-cis-4-hydroxyproline, L-cis-3-hydroxyproline, trans-3-aza-bicyclo[3,<br>
1, 0]hexane-2-carboxylic acid (WO98/54308). Using S. hygroscopicus MG2-10 as<br>
strain background to express genes or gene cassettes encoding for post-PKS<br>
modifylng steps not including rapL or rapL homologues, a library of S. hygroscopicus<br>
strains is generated, capable of producing a plurality of modified products on feeding<br>
with L-pipecolic acid analogues. Suitable L-pipecolic acid analogues include alkyl-,<br>
halo-, hydroxy-, and amino-substituted pipecolic acids and prolines, and more<br>
particularly L-proline, L-trans-4-hydroxyproline, L-cis-4-hydroxyproline, L-cis-3-<br>
hydroxyproline, frans-3-aza-bicyclo[3,1, 0]hexane-2-carboxylic acid and L-pipecolic<br>
acid analogues demonstrated to catalyse PP-ATP exchange measured by a<br>
modification of Lipmann's method (Nielsen et al., 1991) including L-4-hydroxyproline,<br>
1- hydroxyproline, 2-hydroxyproline, 3- hydroxyproline, frans-3-methyl-L-proline, cis-<br>
3-methylproline, cis-3-methyl-DL-proline, c/s,fral7S-4-methylproline, cis-4-methyl-DL-<br>
proline, l/al7s-4-methy)-DL-proline, fral7s-4-aminoproline, cis-4-chloro-L-proIine, 5-<br>
iminoproline hydrochloride, cis-5-methyl-DL-proline, (+)-piperazic acid, 5-<br>
chloropipecolic acid, 5-hydroxypipecolic acid, cis-4-hydroxy-L-pipecolic acid, trans-4-<br>
hydroxy-D-pipecolic acid, 4-hydroxyallopipecolic acid, thiazolidine-4-carboxylic acid<br>
(Nielsen et al., 1991). This approach is exemplified in Example 7.<br>
The production of a limited number of novel rapamycin-analogues after<br>
feeding-close structural analogues of the natural 4,5-dihydroxycyclohex-1-<br>
enecarboxylic acid.starter.unit to cultures of S. hygroscopicus has previously been<br>
described, thus demonstrating that the loading module of the rapamycin polyketide<br>
synthase has some flexibility with respect to the starter acid (PAS. Lowden, PhD<br>
dissertation, University of Cambridge, 1997). However, these methods led to the<br>
production of a mixture of products. In a further aspect, the present invention allows<br>
for the production of rapamycin and related FKBP-ligand analogues by feeding<br>
strains of the present invention with analogues of the naturally incorporated 4,5-<br>
dihydroxycyclohex-1-enecarboxylic acid starter unit to produce rapamycin analogues<br><br>
incorporating alternative starter units including, but not limited to, cyclohexane<br>
carboxylic acid, 3-cis,4-trans-dihydroxycyclohexane carboxylic acid, 1-cyclohexene<br>
carboxylic acid, 3-cyclohexene carboxylic acid, cycloheptane carboxylic acid, 2-<br>
norbornane carboxylic acid, 3-hydroxycycIohexane carboxylic acid, 4-<br>
hydroxycyclohexane carboxylic acid, 3-methylcyclohexane carboxylic acid, 4-<br>
methylcyclohexane carboxylic acid, 3-(cis/trans)methoxycyclohexane carboxylic acid,<br>
4-(cis/trans)methoxycyclohexane carboxylic acid, 4-oxo cyclohexane carboxylic acid,<br>
3-fluoro-4-hydroxycarboxylic acid and 4-fluoro-3-hydroxycarboxylic acid, 3-<br>
cyclohexane oxide carboxylic acid, 3,4-cis-dihydroxycyclohexane carboxylic acid, 3-<br>
chloro-4-hydroxycarboxylic acid and 4-chloro-3-hydroxycarboxylic acid (and the pair<br>
of opposite diastereomers), cyclohexylpropionic acid, 4-tert-Butylcyclohexane<br>
carboxylic acid and simple esters and salts thereof. This approach is exemplified in<br>
Examples 8,19 and 20.<br>
Additionally, structural analogues of biosynthetic precursors of the 4,5-<br>
dihydroxycyclohex-1-enecarboxylic acid starter unit may be fed (Lowden et al.,<br>
2001), leading to production of novel rapamycin analogues incorporating alternative<br>
starter units.<br>
However, these methods can lead to the production of mixed groups of<br>
products; therefore, the present invention additionally provides a method for<br>
removing the competition between the endogenously produced starter unit and the<br>
alternative starter acid analogues that are fed in order to improve the efficiency of<br>
production of novel rapamycin analogues.<br>
In order to remove the competition between the endogenously produced<br>
natural starter unit and the alternative starter acid analogues fed, it is preferable to<br>
disrupt the biosynthesis of the natural 4,5-dihydroxycyclohex-1-enecarboxylic acid<br>
starter unit. This may be achieved by deletion or inactivation ef one or more of the<br>
genes involved in the biosynthesis of the natural 4,5-dihydroxycyclohex-1-<br>
enecarboxylic acid starter unit from shikimic acid (Lowden et al., 2001) or the<br>
biosynthesis of shikimic acid itself. In the latter case, it may be necessary to<br>
supplement cultures with aromatic amino acids (phenyl alanine, tyrosine, tryptophan).<br>
Alternatively, endogenous production of the natural 4,5-dihydroxycyclohex-1-ene<br>
carboxylic acid starter unit may be suppressed by the addition of a chemical inhibitor<br>
of shikimic acid biosynthesis. Such inhibitors are well known in the literature.<br>
In a further aspect the invention makes use of the surprising discovery that<br>
rapK is involved in the supply of the biosynthetic precursors), e.g. 4,5-<br><br>
dihydroxycyclohex-1-ene carboxylic acid starter unit of rapamycin and therefore that<br>
deletion or inactivation of rapK or a rapK homologue provides a strain lacking in<br>
competition between the natural starter unit and fed non-natural starter units. In<br>
another aspect, the invention provides, a method for the efficient incorporation of fed<br>
acids including, but not limited to those described below.<br>
Therefore in one aspect of the invention the method comprises feeding starter<br>
units of the formula where X = bond or CH2 and R1, R2, R3l R4, R5<br>
and R6 may be the same or different and may independently be CI, F, OH, SH, H,<br>
alkyl, CN, Br, R7l OR7, C(0)R7 or HNR7 where R7 is a C1-C4 alkyl; R-, and R3, R2 and<br>
R4, R3 and R5, R4 and R6, R1 and R6, or R2 and R6 may be joined as either a<br>
substituted or unsubstituted methylene link, an ether link, a thia link or an amino link,<br>
R, and R2, R3 and R4 or R5 and R6 may be taken together as a ketone; provided that<br>
no more than 4 of R1f R2, Rs, R4, R5 or R5 may be CI; no more than 2 of R1, R2, R3,<br>
R4, R5 or R6 may be HNR7; no more than 2 of R1, R2, R3, R4, R5 or R6 may be SH and<br>
both R groups from one carbon on the ring are not OH.<br>
In a preferred embodiment the starter unit is not selected from the group<br>
consisting of: cyclohexane carboxylic acid, 3-cis,4-trans-dihydroxycyciohexane<br>
carboxylic acid, cycloheptane carboxylic acid and 3-(cis/frans)-methylcyclohexane<br>
carboxylic acid<br>
In preferred embodiments: where R1,R2, R3, R4, R5 or R6 are a combination of<br>
F and OH substitution no more than 3 of R1-6 are substituted and the remainder are<br>
H. Where R1, R2, R3, R4, R5 or R6 are a combination of CI and OH substitution no<br>
more than 3 of R1-6 are substituted and the remainder are H. Where any two of R1,<br>
R2, R3, R4, R5 or R6 are OH and any two remaining R groups are F on one carbon the<br>
remainder are H. Where two of R1 ,R2, R3l R4, R6 or R6 are CI the remainder are H.<br>
Where two of R1 ,R2, R3, R4, R5 or R6are CI, not originating from the same carbon,<br>
and a further R is OH the remainder are H. Where one of R1, R2, R3, R4, R5 or R6 is<br>
alkyl and the remainder are H; the alkyl group shall have a linear length of no greater<br>
than 3 carbons. Where one of R1, R2, R3, R4, R5 or R6 is NHR7 the remainder are H.<br>
In more highly preferred embodiments: where two of Ri, R2, R3, R4, R5 or R6<br>
are OH and a third R group is F, the remainder are H. Where two of Rn, R2, R3, R4, <br>
R5 or R6 are F the remainder are H. Where two of R1, R2, R3l R4, R5 or R6 are OH the<br>
remainder are H. Where two of R1, R2, R3, R4, R5 or R6 are OH and a third R group is<br><br>
CI the remainder are H. Where two of R1, R2, R3, R4, Re or R6 are F, and a third R<br>
group is OH the remainder are H. Where one of R1, R2, R3, R4, R5 or R6 is SH the<br>
remainder are H. Where one of R1, R2, R3, R4, R5 or R6 is SH and a second R group<br>
is OH (not originating from the same carbon) the remainder are H.<br>
In still more higly preferred embodiments: where one of R-,, R2, R3, R4, R5 or<br>
R6 is F the remainder are H. Where of R1, R2, R3, R4, R5 or R6 are CI the remainder<br>
are H. Where one of R1, R2, R3, R4, R5 or R6, are F and a second R group is OH (not<br>
originating from the same carbon) the remainder are H. Where one of R1, R2, R3, R4,<br>
R5 or R6 is CI and a second R group is OH (not originating from the same carbon) the<br>
remainder are H. Where one of R1, R2, R3 ,R4, R5 or R6 is alkyl and the remainder<br>
are H; the alkyl group shall contain no more than 4 carbons and have a linear length<br>
of no greater than 3 carbons. Where one of R1, R2| R3, R4, R5 or R6 is alkyl and a<br>
second R group is OH (not originating from the same carbon) and remainder are H;<br>
the alkyl group shall contain no more than 4 carbons and have a linear length of no<br>
greater than 3 carbons.<br>
A further aspect of the invention comprises feeding starter units of the formula<br>
, where X = bond or CH2 and R1 ,R2l R3, R4, R6 and R6 may be the<br>
same or different and may independently be CI, F, OH, SH, H, alkyl, CN, Br, R7, OR7,<br>
C(O)R7 or HNR7 where R7 is a C1-C4 alkyl; R1 and R3, R2 and R4, R3 and R5, R4 and<br>
R6, R1 and R5, or R2 and R6 may be joined as either a substituted or unsubstituted<br>
methylene link, an ether link, a thia link or an amino link, R1 and R2, R3 and R4 or R5<br>
and R6 may be taken together as a ketone; provided that no more than 4 of R1, R2,<br>
R3, R4, R5 or R6 may be CI; no more than 2 of R1 R2, R3, R4, R5 or R6 may be HNR7;<br>
no morethan 2 of R1 R2, R3, R4, R5 or R6 may be SH and both R groups from one<br>
carbon on the ring are not OH.<br>
In a preferred embodiment the starter unit is not selected from the group<br>
consisting of: 1-cyclohexene carboxylic acid and 1-cycloheptene carboxylic acid<br>
In preferred embodiments, where R1 R2, R3, R4, R5 or R6 are a combination of<br>
F and OH substitution no more than 3 of R^ are substituted and the remainder are<br>
H. Where R1, R2, R3, R4, R5 or R6 are a combination of CI and OH substitution no<br>
more than 3 of R1-6 are substituted and the remainder are H. Where any two of R1,<br>
R2. R3. R4, R5 or R6 are OH and two of the remaining R groups are F on the same<br>
carbon the remainder are H. Where two of R1, R2, R3, R4, R5 or R6 are CI the<br><br>
remainder are H. Where two of R1, R2, R3 R4 R5 or R6 are CI, not originating from<br>
the same carbon, and a further R group is OH the remainder are H. Where one of<br>
-R2. R3, R4, R5 or Re is alkyl and the remainder are H; the alkyl group shall have a<br>
linear length of no greater than 3 carbons. Where one of R1, R2, R3, R4, R5 or R6 is<br>
NHR7 the remainder are H.<br>
In more highly preferred embodiments: where two of R1, R2, R3, R4, R5 or R6<br>
are OH and a third R group is F, the remainder are H. Where two of R1, R2, R3, R4,<br>
R5 or R6 are F the remainder are H. Where two of R1( R2, R3, R4, R5 or R6 are OH the<br>
remainder are H. Where two of R1, R2, R3, R4, R5 or R6 are OH and a third R group is<br>
CI the remainder are H. Where two of R1, R2, R3, R4, R6 or R6 are F, and a third R<br>
group is OH the remainder are H. Where one of R1, R2, R3, R^ R5 or Re is SH the<br>
remainder are H. Where one of R1, R2, R3, R4, R5 or R6 is SH and a second R group<br>
is OH (not originating from the same carbon) the remainder are H.<br>
In still more higly preferred embodiments: where one of R1, R2, R3, R4, R5 or<br>
R6 is F the remainder are H. Where of R^ R2, R3, R4, R5 or R6 are CI the remainder<br>
are H. Where one of R1, R2, R3l R4, R5 or R6, are F and a second R group is OH (not<br>
originating from the same carbon) the remainder are H. Where one of Rit R2, R3, R4, R5 or Re is CI, a second R group is OH (not originating from the same carbon) the<br>
remainder are H. Where one of R1( R2, R3) R1, R5 or Re is alkyl and the remainder<br>
are H; the alkyl group shall contain no more than 4 carbons and have a linear length<br>
of no greater than 3 carbons. Where one of R1, R2, R3, R4, R5 or R6 is alkyl and a<br>
second R group is OH (not originating from the same carbon) the remainder are H;<br>
and the alkyl group shall contain no more than 4 carbons and have a linear length of<br>
no greater than 3 carbons.<br>
A further aspect of the invention comprises feeding starter units of the<br>
formula':<br>
where X = bond or CH2, R., and R2, may be the same or different<br><br>
and may independently be F, CI, OH, SH, H, CN, OR7l C(O)R7, or NHR7 wherein R7<br>
is a C1-C4 alkyl, R^ and R2 may also be taken together to form a ketone, a<br>
spirocyclopropyl group or with -OCH2-, -CH20-, -SCH2- or-CH2S-; furthermore R3,<br>
and R4 may be the same or different and may independently be be F, CI, Br, OR7, H<br>
or CN; provided that both R groups from one carbon on the ring are not OH.<br><br>
In a preferred embodiment the starter unit shall not be 5-cis-hydroxyl-3-<br>
cyclohexene carboxylic acid.<br>
In preferred embodiments:-Where two of R1, R2, Rs, or R4 are F the remainder<br>
are H. Where-one of R1( R2, R3, or R4 is Ci the remainder are H. Where one of R3, or<br>
R4 is F and one of R1 or R2 is OH the remainder are H. Where one of R3 or R4 is CI<br>
and one of R1 or R2 is OH the remainder are H. Where one of R1 or R2 is SH the<br>
remainder are H. Where one of R^ R2, R3, or R4 is alkyl and the remainder are H;<br>
the alkyl group shall contain no more than 4 carbons and have a linear length of no<br>
greater than 3 carbons. Where one of R3 or R* is alkyl and R1 or R2 is OH the<br>
remainder are H; and the alkyl group shall contain no more than 4 carbons and have<br>
a linear length of no greater than 3 carbons.<br>
In more highly preferred embodiment where one of R1, R2, R3, or R4 is F the<br>
remainder are H. Where one of R1( R2l R3, or R4 is CI the remainder are H<br>
A further aspect of the invention comprises feeding starter units of the formula<br>
where Rn, R2, R3, R*, R5 or R6 may be the same or different and<br><br>
may independently be be be CI, F, OH, SH, H, alkyl, CN, Br, R7, OR7, C(O)R7 or<br>
HNR7 where R7 is a C1-C4 alkyl; R1 and R3, R2 and R4, R3 and R5, R4 and R6, Rn and<br>
R5, or R2 and R6 may be joined as either a substituted or unsubstituted methylene<br>
link, an ether link, a thia link or an amino link, R3 and R» or R5 and R6 may be taken<br>
together as a ketone;provided that both R groups from one carbon en the ring are not<br>
OH.<br>
In preferred embodiments: Where two of R1f R2, Rs, R4, R5 or R6 are F the<br>
remainder are H. Where two of R-,, R2, R3, R1, R5 or R6 are OH, the remainder are H.<br>
Where two of R^ R2, R3, R+, R5 or R6 are OH, and a third R group is F the remainder<br>
are H. Where two of RL R2, R3, R4, R5 or R6 are OH, and a third R group is CI the<br>
remainder are H. Where two of R^ R2, R3, R4, R5 or R6 are F and a third R group is<br>
OH the remainder are H. Where one of R1, R2, R3, R+, R5 or R6 is Br the remainder<br>
are H. Where one of Ru R2| R$, R*, R6 or R6 is Br and a second R group is OH the<br>
remainder are H<br>
In more preferred embodiments: Where one of R1, R2, R5, R4, R5 or R6 is F<br>
the remainder are H. Where one of R1( R2, Rs, R1, R5 or R6 are CI the remainder are<br>
H. Where one of R^ R2, R3, R4l R5 or Re is F and a second R group is OH (not<br><br>
originating from the same carbon) the remainder are H. Where one of R1, R2, R3, R4,<br>
R6 or Re is CI and a second R group is OH (not originating from the same carbon) the<br>
remainder are H, Where one of R1f R2&gt; R3, R4, R5 or Re is SH the remainder are H.<br>
Where one R1, R2, R3, R4, Re or Re is SH and a second R group is OH (not originating<br>
from the same carbon) the remainder are H. Where one of R1, R2, R3, R4, R5 or R6 is<br>
alkyl and the remainder are H; the alkyl group shall contain no more than 4 carbons<br>
and have a linear length of no greater than 3 carbons. Where one of R^ R2, R3, R4, R5 or Re alkyl and a second R group is OH (not originating from the same carbon) the<br>
remainder are H; and the alkyl group shall contain no more than 4 carbons and have<br>
a linear length of no greater than 3 carbons<br>
A further aspect of the invention comprises feeding starter units of the formula<br>
where R1, R2, R3, R4, R5 or R6 may be the same or different and<br><br>
may independently be be be CI, F, OH, SH, H, alkyl, CN, Br, R7, OR7, C(O)R7 or<br>
HNR7 where R7 is a C1-C4 alkyl; R1 and R3, R2 and R4, R3 and R5, R4 and R6l R1 and<br>
R5, or R2 and R6 may be joined as either a substituted or unsubstituted methylene<br>
link, an ether link, a thia link or an amino link, R3 and R4 or R5 and R6 may be taken<br>
together as a ketone;provided that both R groups from one carbon on the ring are not<br>
OH.<br>
In preferred embodiments: where R1F R2, R3, R4, R5 or R6 are a combination of<br>
F and OH substitution no more than 3 of R^ are substituted and the remainder are<br>
H. Where R1, R2, R3, R4, R5 or R6 are a combination of CI and OH substitution no<br>
more than 3 of R^ are substituted and the remainder are H. Where two of R1, R2,<br>
R3, R4, R5 or R6 are OH and two of the remaining R groups are F on one carbon the<br>
remainder are H. Where two of R1( R2l R3, R4, R5 or R6 are CI the remainder are H.<br>
Where two of R1, R2l R3, R4, R5 or R6 are CI (not originating from the same carbon)<br>
and a third R group is OH, the remainder are H. Where one of R1, R2l R3, R4, R5 or<br>
R6 is alkyl and the remainder are H; the alkyl group shall have a linear length of no<br>
greater than 3 carbons. Where two of R1( R2, R3, R4, R5 or R6 are SH the remainder<br>
are H. Where one of R1, R2) R3, R4, R5 or R6is HNR7 the remainder are H.<br>
In more preferred embodiments: Where two of R^ R2, R3l R4, R5 or R6 are F<br>
the remainder are H. Where two of R1, R2, R3, R4, R5 or R6 are OH the remainder are<br><br>
H. Where two of R1r R2, R3, R4, R5 or R6 are OH and a third R group is F, the<br>
remainder are H. Where two of R1, R2, R3, R4, R5 or R6 are OH and a third R group is<br>
Ci the remainder are H. Where two of R1, R2, R3, R4, R5 or RBare F, and a third R<br>
groups is OH the remainder are H. Where one of R1( R2, R3, R^ R5 or R6 is Br the<br>
remainder are H, Where one R1( R2, R3, R4, R5 or Re is Br and a second R group is<br>
OH (not originating from the same carbon) the remainder are H. Where one of R1, Ra, Rs, R4, R5 or Re is SH the remainder are H. Where one of R-,, R2, R3, R4, R5 or R6<br>
is SH and a second R groups is OH (not originating from the same carbon) the<br>
remainder are H.<br>
In more preferred embodiments: Where one of R^ R2, R3, R4, R5 or R6 is F<br>
the remainder are H. Where one of R1( R2l R3, R4, R5 or Re is CI the remainder are<br>
H. Where one of R1f R2, R3, R4, R5 or R6 is F and a second R group is OH (not<br>
originating from the same carbon) the remainder are H. Where one of R1, R2, R3) R4,<br>
R5 or R6 is CI and a second R group is OH (not originating from the same carbon) the<br>
remainder are H. Where one of R1, R2, R3, R4, R5 or R6 is alkyl and the remainder<br>
are H; the alkyl group shall contain no more than 4 carbons and have a linear length<br>
of no greater than 3 carbons. Where one of R1; R2, R3, R^ R6 or R6 is alkyl and a<br>
second R group is OH (not originating from the same carbon) the remainder are H;<br>
and the alkyl group shall contain no more than 4 carbons and have a linear length of<br>
no greater than 3 carbons.<br>
A further aspect of the invention comprises feeding starter units of the formula<br>
where R^ and R2, may be the same or different and may<br><br>
independently be F, CI, OH, SH, H, CN, OR7, C(O)R7, or NHR7 wherein R7 is a C1-<br>
C4 alkyl, R1 and R2 may also be taken together to form a ketone, a spirocyclopropyl<br>
group or with -OCH2-, -CH20-, -SCHr or -CH2S-; furthermore R3, and R4 may be the<br>
same or different and may independently be be F, CI, Br, OR7, H or CN; provided<br>
that both R groups from one carbon on the ring are not OH..<br>
In preferred embodiments: Where one of R^ R2, R3 and R4 is F the remainder<br>
are H. Where one of R1, R2, R3 and R4 is CI the remainder are H. Where one of R1,<br>
R2, R3 and R4 is F and a second R groups is OH (not originating from the same<br>
carbon) the remainder are H. Where one of R1f R2, R3 and R« is CI and a second R<br>
group is OH (not originating from the same carbon) the remainder are H. Where one<br><br>
of R1, R2l R3 and R* is SH the remainder are H. Where one of R1, R2l R3 and R4 is<br>
alkyl the remainder are H; and the alkyl group shall contain no more than 4 carbons<br>
and have a linear length of no greater than 3 carbons. Where one of R1F R2, R3 and<br>
R4 is alkyl and a second R groups is OH (not originating from the same carbon) the<br>
remainder are H; and the alkyl group shall contain no more than 4 carbons and have<br>
a linear length of no greater than 3 carbons. Where two of R1, R2, R3 and R4 are F<br>
the remainder are H.<br>
An additional aspect of the invention comprises feeding starter units of the<br>
formula<br>
where X = bond or CH2;and R1, R2, R3) R4 or R5 may be the same or<br><br>
different and may independently be be CI, F, OH, SH, H, alkyl, CN, Br, R7, OR7,<br>
C(O)R7 or HNR7 where R7 is a C1-C4 alkyl, R1 and Rs, R2 and R4, may be taken<br>
together as a ketone or linked as either a substituted or unsubstituted methylene link,<br>
an ether link, a thia link or an amino link where R1 and R2 or R3 and R4 are linked as<br>
a spiro-cyclopropyl group or with -OCHr or -CH20- or -SCH^ or -CH2S-, R5 may be<br>
F, CL, OR7, H or CN; provided that no more than two of R1, R2, R3, R4 or R5 are SH<br>
and that both R groups attached to one carbon are not OH.<br>
In preferred embodiments: where R1, R2, R^ R4 or R5 are a combination of F<br>
and OH no more than 3 of R1, R2, R3) R4 or R5 are substituted and the remainder are<br>
H. Where R1, R2, R3, R4 or R5 are a combination of CI and OH no more than 3 of R^<br>
are substituted and the remainder are H. Where R1( R2, R3, R4 or R5 are a<br>
combination of two are OH (not on the same carbon) and two are F on one carbon<br>
the remainder are H. Where two of R1f R2, R3, R4 or R5 are CI the remainder are H.<br>
Where two of R1, R2, R3, R4 or R5 are CI (not originating from the same carbon) and a<br>
third R group is OH the remainder are H. Where one of Rf1, R2, R3, R4 or R5 is alkyl<br>
the remainder are H; and the alkyl group shall have a linear length of no greater than<br>
3 carbons. Where two of R1( R2, R3, R4 or R5 are SH the remainder are H. Where<br>
one of R1( R2, R3, R4 or R5 is NHR7 the remainder are H. Where one of R1, R2) R3) R4<br>
or R5 is SH the remainder are H.<br>
In more highly preferred embodiments: where one of R^ R2, R3, R4 or R5 is<br>
OH the remainder are H. Where one of R^ R2, R3, R4 or R5 is F the remainder are H.<br>
Where one of R1, R2, R3, R4 or R5 is CI the remainder are H. Where one of R1, R2,<br>
R3, R* or R5 is F and a second R group is OH (not originating from the same carbon)<br><br>
the remainder are H. Where one of R1, R2, R3l R* or R5 is CI and a second R groups<br>
is OH (not originating from the same carbon) the remainder are H. Where one of R-,,<br>
R2, R3l RA or R6 is SH and a second R group is OH (not originating from the same<br>
carbon) the remainder are H. Where one of R1, R2, Ra, R* or R5 is alkyl the<br>
remainder are H; and the alkyl group shall contain no more than 4 carbons and have<br>
a linear length of no greater than 3 carbons. Where one of R-\, R2l R3, R4 or R6 is<br>
alkyl and a second R group is OH (not originating from the same carbon) the<br>
remainder are H; and the alkyl group shall contain no more than 4 carbons and have<br>
a linear length of no greater than 3 carbons. Where two of R1, R2, R3, R4 or R5 are F<br>
the remainder are H. Where two of R1, R2, R3, R4 or R5 are OH the remainder are H.<br>
Where two of R-,, R2) R3, R4 or R5 are OH and a third R group is F the remainder are<br>
H. Where two of R1( R2, R3, R4 or R5 are OH and a third R groups is CI the remainder<br>
are H. Where two of R1, R2l R3, R« or R5 are F and a third R group is OH the<br>
remainder are H.<br>
An additional aspect of the invention comprises feeding starter units of the<br>
formula where R1, R2l R3 and R4 may be the same or different and<br>
may independently be CI, F, OH, SH, H, alkyl. CN, Br, R7f OR7, C(O)R7 or HNR7<br>
where R7 is a C1-C4 alkyl, R^ and R2 or R5 and R4 may be taken together to form a<br>
ketone, provided that two R groups attached to the same carbon are not both OH.<br>
In preferred embodiments: Where one of R1, Rz, R3 or R4 is F the remainder<br>
are H. Where one of R1, R2, R3 or R4 is CI the remainder are H. Where one of R1f<br>
R2, R3 or R4 is Br the remainder are H. Where one of R1f R2, R3 or Rj is OH the<br>
remainder are H. Where one of R1, R2, R3 or R4 is F and a second R group is OH<br>
(not originating from the same carbon) the remainder are H. Where one of R1, R2l R3<br>
or R4 is CI and a second R groups is OH (not originating from the same carbon) the<br>
remainder are H. Where one of R1( R2, R3 or R4 is SH the remainder are H. Where<br>
one of R1, R2, R3 or R4 is SH and a second R groups is OH (not originating from the<br>
same carbon) the remainder are H. Where one of R1, R2, R3 or R4 is alkyl the<br>
remainder are H; and the alkyl group shall contain no more than 4 carbons and have<br>
a linear length of no greater than 3 carbons. Where one of R1, R2, R3 or FM is alkyl<br>
and a second R groups is OH (not originating from the same carbon) the remainder<br>
are H; and the alkyl group shall contain no more than 4 carbons and have a linear<br>
length of no greater than 3 carbons. Where two of R1, R2, R3 or R4 are F the<br><br>
remainder are H. Where two of R-,, R2) R3 or R4 are OH the remainder are H. Where<br>
two of R1, R2l R3 or R4 are OH and a third R group is F the remainder are H. Where<br>
two of Rv, R2, R3 or R4 are OH and a third R group is CI the remainder are H. Where<br>
two of R1, R2l R3 or R4 are F and a third R group is OH the remainder are H.<br>
In a preferred embodiment the present invention provides a method for the<br>
efficient incorporation of. 2-norbomane carboxylic acid; 2-(cis/trans)-<br>
hydroxycyclohexane carboxylic acid; 3-(cis/frans)-hydroxycyclohexane carboxylic<br>
acid; 4-(o/s/frans)-hydroxycyclohexane carboxylic acid; 2-(cisrfrans)-<br>
methylcyclohexane carboxylic acid; 4-(cis/trans)-methylcyclohexane carboxylic acid;<br>
3-(cis/!rans)-methoxycyclohexane carboxylic acid; 4-(c;s/frans)-methoxycyclohexane<br>
carboxylic acid; 4-oxocyclohexane carboxylic acid; ethyl 2-oxocyclohexane carboxylic<br>
acid; 4-trans-n-pentylcyclohexane carboxylic acid; 2-trans-aminocyclohexane<br>
carboxylic acid; 4-cis-aminocyclohexane carboxylic acid; 4-(cis/lrans)-<br>
aminomethylcyclohexane carboxylic acid; cyclopentane carboxylic acid; cyclobutane<br>
carboxylic acid; 1-methylcyclohexane carboxylic acid; 3-trans-hydroxy-4-cis-<br>
fluorocyclohexane carboxylic acid and 4-trans-hydroxy-3-cis-fluorocyclohexane<br>
carboxylic acid; 3-cis-hydroxy-4-trans-fluorocyclohexane carboxylic acid and 4-cis-<br>
hydroxy-3-trans-fluorocyclohexane carboxylic acid; 3-cis-hydroxy-4-trans-<br>
chlorocyclohexane carboxylic acid and 4-cis-hydroxy-3-trans-chlorocycIohexane<br>
carboxylic acid; 3-fral7S-hydroxy-4-cis-chlorocyclohexane carboxylic acid and 4-trans-<br>
hydroxy-3-cis-chlorocyclohexane carboxylic acid; 3-trans-cyclohexeneoxide<br>
carboxylic acid; 3-cis-cyclohexeneoxide carboxylic acid; 3,4-cis-<br>
dihydroxycyclohexane carboxylic acid and 3,4-tral7s-dihydroxycyclohexane carboxylic<br>
acid; cyclohexaneacetic acid; cyclohexanepropionic acid or 4-cistoans-tert-<br>
butylcyciohexane carboxylic acid or simple esters or salts thereof into FKBP-ligand<br>
analogQes by a strain with rapK or a rapK homologue deleted or inactivated, in a<br>
more preferred embodiment the present invention provides a method for the efficient<br>
incorporation of: 3-(cis#ral?s)-hydroxycyclohexane carboxylic acid; 4-(cistfrans)-<br>
hydroxycyclohexane carboxylic acid; 3-(cis/?rai)s)-methoxycyclohexane carboxylic<br>
acid; 4-(cisrtrans)-methoxycyclohexane carboxylic acid; 4-oxo cyclohexane<br>
carboxylic acid; cyclobutane carboxylic acid; 3-trans-hydroxy-4-cis-ftuorocyclohexane<br>
carboxylic acid and 4-trans-hydroxy-3-cis-fluorocyclohexane carboxylic acid; 3-cis-<br>
hydroxy-4-trans-fluorocyclohexane carboxylic acid and 4-cis-hydroxy-3-trans-<br>
fluorocyclohexane carboxylic acid; 3-cis-hydroxy-4-trans-chlorocyclohexane<br><br>
carboxylic acid and 4-cis-hydroxy-3-frBns-chlorocycIohexane carboxylic acid; 3-trans-<br>
hydroxy-4-cis-chlorocyclohexane carboxylic acid and 4-<ralis-hydroxy-3-cis-></ralis-hydroxy-3-cis->
chlorocyclohexane carboxylic acid; 3-trans-cyclohexeneoxide carboxylic acid; 3-cis-<br>
cydohexeneoxide carboxylic acid; 3,4-cis-dihydroxycyclohexane carboxylic acid and<br>
3,4-trans-dihydroxycyclohexane carboxylic acid; cyclohexanepropionic acid; 4-<br>
ds/frans-tert-butylcyclohexane carboxylic acid or simple esters or salts thereof into<br>
FKBP-ligand analogues by a strain with rapK or a rapK homologue deleted or<br>
inactivated.<br>
In a specific embodiment of the present invention the fed starter units are not<br>
cyclohexane carboxylic acid, 3-cis,4-trans-dihydroxycyclohexane carboxylic acid, 1-<br>
cyclohexene carboxylic acid, 3-cydohexene carboxylic acid, cycioheptane carboxylic<br>
acid, 3-(cis/trans)-methylcyclohexane carboxylic acid, 4-(cis/transy<br>
methylcyclohexane carboxylic acid, 1-cycloheptene carboxylic acid or 5-cis-hydroxyl-<br>
3-cyclohexene carboxylic acid.<br>
The strains for use in the embodiments described above are selected from<br>
the group comprising: Streptomyces hygroscopicus subsp. hygroscopicus NRRL<br>
5491, Actinoplanes sp. N902-109 FERM BP-3832, Streptomyces sp. AA6554,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6475 ATCC14891,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6678 ATCC 55087,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6674, Streptomyces<br>
hygroscopicus var. ascomyceticus ATCC 55276, Streptomyces hygroscopicus<br>
subsp. ascomyceticus ATCC 14891, Streptomyces tsukubaensis No.9993 FERM BP-<br>
927, Streptomyces hygroscopicus subsp. yakushimaensis, Streptomyces sp. DSM<br>
4137, Streptomyces sp. DSM 7348, Micromonospora n.sp. A92-306401 DSM 8429,<br>
Steptomyces sp. MA 6858 ATCC 55098, Steptomyces sp. MA 6848. In a preferred<br>
embodiment said strain is selected from the group consisting ef: Steptomyces<br>
hygroscopicus subsp. hygroscopicus NRRL 5491, Actinoplanes sp. N902-109 FERM<br>
BP-3832, Streptomyces sp. AA6554, Streptomyces hygroscopicus var.<br>
ascomyceticus MA 6475 ATCC 14891, Streptomyces hygroscopicus var.<br>
ascomyceticus MA 6678 ATCC 55087, Streptomyces hygroscopicus var.<br>
ascomyceticus MA 6674, Streptomyces hygroscopicus var. ascomyceticus ATCC<br>
55276, Streptomyces hygroscopicus subsp. ascomyceticus ATCC 14891,<br>
Streptomyces tsukubaensis No.9993 FERM BP-927, Streptomyces hygroscopicus<br>
subsp. yakushimaensis, Streptomyces sp. DSM 4137, Streptomyces sp. DSM 7348,<br>
Micromonospora n.sp. A92-306401 DSM 8429 or Streptomyces sp. MA 6858 ATCC<br><br>
55098. In a more highly preferred embodiment the strain is the rapamycin producer<br>
S. hygroscopicussubsp. hygroscopicus.<br>
In the methods for.the efficient incorporation of fed carboxylic acids described<br>
above the compounds produced are analogues of the FKBP-ligards as described<br>
herein, for example but without limitation: rapamycin, FK506, FK520, FK523, FK525,<br>
antascomicin, meridamycin and tsukubamycin. In a preferred embodiment the<br>
compounds produced are analogues of rapamycin, FK506 or FK520. In a more<br>
highly preferred embodiment the compounds produced are analogues of rapamycin;<br>
these compounds correspond to Formula II or Formula III as described below.<br>
Additionally, the methods described above may be used to generate novel<br>
FK506 and FK520 analogues which correspond to Formula I below:<br><br>
R2 = H, alkyl, halo, hydroxyl, thiol R5 = H, alkyl, halo, hydroxyl, thiol<br>
R4 = H, alkyl, halo, hydroxyl, thiol<br>
R5 = OMe, Me or H<br>
R6 = OMe, Me or H<br><br>
R7 = CH2CH3 or CH2CH=CH2<br>
Z = keto or CH2<br>
X=X'=bond; X=bond and X'- CH2l S, O or X= OH2, S, O, fused cyclopropyl unit and<br>
X'=bond<br>
In a preferred embodiment, where R6 = OH<br>
and Ro= H, OH, halo, alkyl or thiol.<br>
In a further preferred embodiment <br>
  where Rs= OH and R9 = halo.<br>
  where Re = 4-trans-OH, R9 = 3-cis-OCH3, and R2 = R3 = R4 = H,<br>
X = CH2l X' = bond, Z = keto, R6 = Re = OCH3 and R7 = CH2CH3<br>
  where Re = 4-trans-OH, R6 = 3-cis-OCH3, and R2 = R3 = R4 = H,<br>
X = CH2) X" = bond, Z - keto, R5 = R6 = OCH3 and R7 = CH2CH=CH2<br>
Thus, for example, the recombinant strain S. hygroscopicus MG2-10 can be<br>
cultured in the presence of cyclohexane carboxylic acid to produce 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-39-desmethoxy-rapamycin (Example 12). It can be seen<br>
by one skilled in the art that homologues to rapK in other biosynthetic clusters that<br>
encode FKBP-ligands, including, but not limited to, FK506, FK520, FK523, FK525,<br>
meridamycin, tsukubamycin, antascomicin and 'hyg' can also be deleted or<br>
inactivated allowing efficient feeding of starter unit carboxylic acids leading to the<br>
production of novel analogues.<br>
In another aspect, S. hygroscopicus strains of the invention (including rapL or<br>
rapL homologues or not including rapL or rapL homologues and/or including rapK or<br>
rapK homologues or not including rapK or rapK homologues) may be fed with<br>
analogues of L-pipecolic acid, as described above, in combination with analogues of<br>
the natural 4,5-dihydroxycyclohex-1-enecarboxylic acid starter unit, as described<br><br>
above, to produce rapamycin analogues in which both the starter unit and the<br>
pipecolyl residue have been replaced. This approach is exemplified in Examples 10,<br>
11 and 12.<br>
The present invention provides a process for producing FKBP-ligand<br>
analogues varylng in the extent of post-PKS modification and/or in which the<br>
pipecolic acid residue has been replaced, and optionally the starter 4,5-<br>
dihydroxycyclohex-1-enecarboxylic acid residue has been replaced. This process<br>
comprises the step of deleting or inactivating one or more genes in the<br>
microorganism host cell involved in the production of the precursor compound, L-<br>
pipecolic acid and/or 4,5-dihydroxycyclohex-1-ene carboxylic acid, required for<br>
biosynthesis of the rapamycin polyketide/NRPS template and/or in its subsequent<br>
post-PKS modification, thereby to suppress the production of the natural product.<br>
The process further comprises transforming the microorganism host cells with nucleic<br>
acid encoding polyketide-modifylng genes to restore polyketide production, culturing<br>
the transformed host cells under conditions suitable for polyketide production and<br>
optionally isolating the rapamycin analogues produced.<br>
The present invention provides a process for the production of FKBP-ligand .<br>
analogues including, but not limited to FK506, FK520, FK523, FK525, tsukubamycin,<br>
antascomicin, meridamycin and 'hyg', varylng in the extent of post-PKS modification<br>
and/or in which the amino acid residue has been replaced, and optionally the starter<br>
unit has been replaced. This process comprises the step of deleting or inactivating<br>
one or more genes in the microorganism host cell involved in the production of the<br>
precursor amino acid residue and/or starter unit, required for the biosynthesis of the<br>
polyketide/NRPS template and/or in its subsequent post-PKS modification, thereby to<br>
suppress the production of the natural product The process further comprises<br>
transforming the microorganism host cells with nucleic acid encoding polyketide-<br>
modifylng genes to restore polyketide production, culturing the transformed host cells<br>
under conditions suitable for polyketide production and optionally isolating polyketide<br>
analogues produced.<br>
The present invention provides novel FKBP-Figand analogues.<br>
In a further aspect the present invention provides the following FK520<br>
analogues: 31-desmethoxy-FK520, 31-desmethoxy-31-cis-hydroxy-32-fr^ns-hydroxy-<br>
FK520, 31 -desmethoxy-31 -cis-hydroxy-32-cis-hydroxy-FK520, 31 -desmethoxy-31 -<br>
frans-hydroxy-32-trans-hydroxy-FK520, 31 -O-desmethyf-32-dehydroxy-FK520, 31 -O-<br>
desmethyl-FK520,31-desmethoxy-31-methyl-FK520,31-O-desmethyl-32-dehydroxy-<br><br>
32-methyl-FK520,31-O-desmethyl-32-dehydrQxy-32-fluoro-FK520l31-desmethoxy-<br>
31 -fluoro-FK520,31 -O-desmethyl-32-dehydroxy-32-chloro-FK520, 31 -desmethoxy-<br>
31-chloro-FK520, 31-O-desmethy»-32-dehydroxy-32-fe/to^<br>
methoxy-4-hydroxy-cydohexyl)-29-(hydroxy-cycloheptyl)-FK520, 29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norborny))-FK520, 9-deoxo-31-desmethoxy-FK520,<br>
9-deoxo-31-desmethoxy-31-cfe-hydroxy-32-trans-hydroxy-FK520, 9-deoxo-31-<br>
desmethoxy-31 -cis-hydroxy-32-cis-hydroxy-FK520,9-deoxo-31 -desmethoxy-31 -<br>
frans-hydroxy-32-trans-hydroxy-FK520,9-deoxo-31 -O-desmethyl-32-dehydroxy-<br>
FK520, 9-deoxo-31-Odesmethyl-FK520I9-deoxo-31-desmethoxy-31-methyl-FK520,<br>
9-deoxo-31 -O-desmethyl-32-dehydroxy-32-methyl-FK520, 9-deoxo-31 -O-desmethyl-<br>
32-dehydroxy-32-fluoro-FK520, 9-deoxo-31-desmethoxy-31 -fluoro-FK520, 9-deoxo-<br>
31 -O-desmethyl-32-dehydroxy-32-chloro-FK520, 9-deoxo-31 -desmethoxy-31 -chloro-<br>
FK520, 9-deoxo-31 -Odesmethyl-32-dehydroxy-32-ferf-butyl-FK520, 9-deoxo-29-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-cycloheptyl)-FK5201 9-deoxo-29-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-FK520, 30-desmethoxy-<br>
prolyl-FK520l30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-prolyl-FK520, 30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-prolyl-FK520, 30-desmethoxy-30-trans-<br>
hydroxy-31-trans-hydroxy-prolyl-FK520, 30-O-desmethyl-31-dehydroxy-prolyl-FK520,<br>
30-O-desmethyl-prolyl-FK520, 30-desmethoxy-30-methyl-prolyl-FK520, 30-O-<br>
desmethyl-31-dehydroxy-31-methyl-proiyl-FK520, 30-Odesmethyl-31-dehydroxy-31-<br>
fluoro-prolyl-FK520, 30-desmethoxy-30-fluoro-prolyl-FK520, 30-O-desmethyl-31 -<br>
dehydroxy-31-di)oro-prolyl-FK520) 30-desmethoxy-30-chloro-prolyl-FK520, 30-<br>
desmethyl-31-dehydroxy-31-terf-butyl-prolyl-FK520,28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-cycloheptyl)-FK520, 28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbornyl)-FK520I 8-deoxo-30-desmethoxy-31-hydroxy-<br>
prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-fra^-hydroxy-prolyl-<br>
FK520,,:8-deoxo-30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-prolyl-FK520, 8-<br>
deoxo-30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
O-desmethyl-31 -dehydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-prolyl-FK520, 8-<br>
deoxo-30-desmethoxy-30-methyl-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -<br>
dehydroxy-31-methyl-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-<br>
fluoro-prolyl-FK520, 8-deoxo-30-desmethoxy-30-fluoro-prolyl-FK520, 8-deoxo-30-O-<br>
desmethyl-31 -dehydroxy-31 -chloro-prolyl-FK520,8-deoxo-30-desmethoxy-30-chloro<br>
prolyl-FK520t8-deoxo-30-O-desmethyl-31-dehydroxy-31-tert-butyl-prolyl-FK520, 8-<br>
deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-prolyl-<br><br>
FK520,8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbomyl)-<br>
prolyl-FK520, 30-desmethoxy-3-hydroxy-prolyl-FK520,30-desmethoxy-30-cis-<br>
hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-<br>
31-cis-hydroxy-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-trans-hydroxy-31-trans-<br>
hydroxy-3-hydroxy-prolyl-FK520, 30-O-desmethyl-31rdehydroxy-3-hydroxy-prolyl-<br>
FK520, 30-O-desmethyl-3-hydroxy-prolyl-FK52Q, 30-desmethoxy-30-methyl-3-<br>
hydroxy-prolyl-FK520, 30-O-desmethyl-31 -dehydroxy-31 -methyl-3-hydroxy-prolyJ-<br>
FK520, 30-O-desmethyl-31 -dehydroxy-31 -fluoro-3-hydroxy-prolyl-FK520, 30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK5201 30~O-desmethyl-31 -dehydroxy-31 -<br>
chloro-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-chloro-34iydroxy-prolyl-FK520,<br>
30-O-desmethyl-31 -dehydroxy-31 -tert-butyl-3-hydroxy-prolyl-FK520, 28-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-3-hydroxy-prDlyl-FK520, 28-<br>
de(3-mettioxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbomyl)-3-hydroxy-prolyl-FK520,<br>
8-deoxo-30-desmethoxy-31-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-trans-hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
O-desmethyl-31-dehydroxy-3-hydroxy-prolyl-FK520l 8-deoxo-30-O-desmethyl-3-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-methyl-3-hydroxy-prolyl-FK520, 8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-methyl-3-hydroxy-prolyl-FK520, 8-deoxo-<br>
30-O-desmethyl-31 -dehydroxy-31 -fluoro-3-hydroxy-pro)yl-FK520, 8-deoxo-30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK520, B-deoxo-30-O-desmethyl-31 -<br>
dehydroxy-31-chloro-3-hydroxy-pro!yl-FK520, 8-deoxo-30-desmethoxy-30-chloro-3-<br>
hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -tert-<br>
butyl-3-hydroxy-pro!yl-FK520, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-cycloheptyl)-3-hydroxy-proIyl-FK520, 8-deoxo-28-de^-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbornyl)-3-hydroxy-prolyl-FK520, 30-desmethoxy-4-<br>
hydroxy-prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-31 -trans-hydroxy-4-hydroxy-<br>
prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-hydroxy-prolyl-FK520,<br>
30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-4-hydroxy-prolyl-FK520, 30-O-<br>
desmethyl-31-dehydroxy-4-hydroxy-prolyl-FK520, 30-O-desmethyl-4-hydroxy-prolyl-<br>
FK520,30-desmethoxy-30-methyl-4-hydroxy-prolyl-FK520,30-O-desmethyl-31 -<br>
dehydroxy-31-methyl-4-hydroxy-prolyl-FK520, 30-O-desmethyl-31 -dehydroxy-31 -<br>
fluoro-4-hydroxy-prolyl-FK520, 30-desmethoxy-30-fiuoro-4-hydroxy-prolyl-FK520, 30-<br>
O-desmethyl-31 -dehydroxy-3" -chloro-4-hydroxy-prolyl-FK520, 30-desmethoxy-30-<br><br>
chloro-4-hydroxy-pro!yl-FK520,30-O-desmethyl-31-dehydroxy-31 -terf-butyl-4-<br>
hydroxy-prolyl-FK520,28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
cycloheptyl)-4-hydroxy-prolyl-FK520, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbomyl)-4-hydroxy-prolyl-FK520, 8-deoxo-30-desrnethoxy-31-hydroxy-4-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK520, 8-deoxo30-desmethoxy-30-cis-hydroxy-31-c;s-hydroxy-4-<br>
hydroxy-prolyl-FK520&gt;8-deoxo-30-desmethoxy-30-&amp;ans-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-4-hydroxy-prolyl-<br>
FK520, 8-deoxo-30-O&gt;desmethyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-<br>
30-methyM-hydroxy-prolyl-FK520) 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -<br>
methyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -fluoro-4-<br>
hydroxy-prolyl-FK520,8-deoxo-30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK520, 8-<br>
deoxo-30-O-desmethyl-31-deriydroxy-31-chlor4-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-chloro-3-hydroxy-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-<br>
31 -dehydroxy-31 -ferf-butyl-4-hydroxy-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-4-hydroxy-prolyl-FK520, 8-deoxo-28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbomyl)-4-hydroxy-prolyl-FK520)<br>
31-desmethoxy-trans-3-bicyclo[3.1.0.]FK520, 31-desmethoxy-31-cis-hydroxy-32-<br>
frans-hydroxy-trans-3-bicyclo[3.1.0.]FK520,31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK520, 31 -desmethoxy-31 -trans-hydroxy-32-trans-<br>
hydroxy-trans-3-bicyclo[3.1.0.JFK520, 31 -O-desmethyl-32-dehydroxy-trans-3-<br>
bicyclo[3.1.0.JFK520, 31-O-desmethyl-trans-3-bicyclo[3.1.0.JFK520, 31-desmethoxy-<br>
31 -methyl-trans-3-bicyclo[3.1.0.JFK520, 31 -O-desmethyl-32-dehydroxy-32-methyl-<br>
trans~3-b)cyc\o[3.1.O.JFK520, 31-O-desmethyl-32-dehydroxy-32-fluoro-frBns-3-<br>
bicyclo[3.1.0.JFK520, 31-desmethoxy-31-fIuoro-trans-3-bicyclo[3.1.0.]FK520, 31-O-<br>
desmethyl-32-dehydroxy-32-chloro-trans-3-bicyclo[3.1.0.]FK520, 31-desmethoxy-31-<br>
chloro-trans-3-bicyclo[3.1.0.]FK520, 31-O-desmethyl-32-dehydroxy-32-fe/t-butyl-<br>
frans-3-bicyclo[3.1.0.JFK520, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
cycloheptyl)-trans-3-bicyclo[3.1.0.JFK520,29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-norbomylH/Tans-3-bicyclo[3.1.0.]FK520l 9-deoxo-31-desmethoxy-fral7S-<br>
3-bicyclo[3.1.0.]FK520, 9-deoxo-31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-<br>
frans-3-bicyclo[3.1.0.]FK520,9-deoxo-31 -desmethoxy-31 -cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-desmethoxy-31-trans-hydroxy-32-<br>
frans-hydroxy-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-O-desmethyl-32-dehydroxy-<br>
frans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31 -O-desmethy Wrans-3-bicyclo[3.1.0.JFK520,<br><br>
9-deoxo-31-desmethoxy-31-methyW/ans-3-bicyclot3.1.0.]FK520,9-deoxo-31-O-<br>
desme%l-32-dehydroxy-32-methyl-trans-3-bicyclo[3.1.0.]FK520l9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-fluoro-tnans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-<br>
desmethoxy-31-fluoro-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-O-desmethyl-32-<br>
dehydroxy-32-chloro-fcrans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-desmethoxy-31-<br>
chloro-trans-3-bicyclo[3.1.0.]FK520,9-deoxo-31-O-desmethyl-32-dehydroxy-32-tert-<br>
butyl-*rans-3-bicyclo[3.1.0.JFK520, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyciohexyl)-<br>
29-(hydroxy-cycloheptyl)-trans-3-bicyclo[3.1.0.]FK520I9-deoxo-29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-trans-3-bicyclo[3.1.0.JFK520,,<br>
In a preferred embodiment, the present invention provides the following<br>
FK520 analogues: 31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-FK520, 31-<br>
desmethoxy-31-cis-hydroxy-32-cis-hydroxy-FK520, 31-desmethoxy-31-trans-<br>
hydroxy-32-£rans-hydroxy-FK520, 31 -desmethoxy-31 -methyl-FK520, 31 -desmethoxy-<br>
31-fluoro-FK520, 31-desmethoxy-31-chloro-FK520, 31-O-desmethyl-32-dehydroxy-<br>
32-tert-butyl-FK520, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
cycloheptyl)-FK520,29-de(3-methoxy-4-hydroxy-cyclohexy0-29-(hydroxy-norbornyl)-<br>
FK520, 9-deoxo-31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-FK520, 9-deoxo-<br>
31 -desmethoxy-31 -cis-hydroxy-32-cis-hydroxy-FK520, 9-deoxo-31 -desmethoxy-31 -<br>
/rans-hydroxy-32-fraAJS-hydroxy-FK520, 9-deoxo-31-desmethoxy-31-methyl-FK520,<br>
9-deoxo-31-desmethoxy-31-f!uoro-FK520, 9-deoxo-31-desmethoxy-31-chloro-FK520,<br>
9-deoxo-31-O-desmethyl-32-dehydroxy-32-fert-butyl-FK520, 9-deoxo-29-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-cycloheptyl)-FK520, 9-deoxo-29-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-norbomyl)-FK520,, 30-desmethoxy-30-<br>
cis-hydroxy-31-trans-hydroxy-prolyl-FK520) 30-desmethoxy-30-cis-hydroxy-31-cis-<br>
hydroxy-prolyl-FK520, 30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-prolyl-<br>
FK520, 30-desmethoxy-30-methyl-prolyl-FK520, 30-desmethoxy-30-fluoro-prolyl-<br>
FK520,%0-desmethoxy-30-chloro-prolyl-FK520,30-desmethyl-31-dehydroxy-31-tert-<br>
butyl-prolyl-FK520,28-de(3-methoxy-4-hydroxy-cyclohexy0-28-(hydroxy-<br>
cycloheptyl)-FK520128-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-<br>
FK520, 8-deoxo-30-desmethoxy-31 -hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-<br>
30-c/'s-hydroxy-31-trans-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-<br>
hydroxy-31-cis-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-fcrans-hydroxy-31-<br>
frans-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-prolyl-FK520, 8-<br>
deoxo-30-O-desmethyl-prolyl-FK520, 8-deoxo-30-desmethoxy-30-methyl-prolyl-<br>
FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-methyl-prolyl-FK520, 8-deoxo-30-<br><br>
O-desmethyl-31-dehydroxy-31-fluoro-proiyl-FK520,8-deoxo-30-desmethoxy-30-<br>
fluoro-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -chloro-prolyl-FK520,<br>
8-deoxo-30-desmethoxy-30-chloro-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-<br>
dehydroxy-31-tert-butyl-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-cycloheptyl)-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-{hydroxy-norbomyl)-prolyl-FK520,.30-desmethoxy-3-hydroxy-<br>
prolyl-FK520, 30-desmethoxy-30-cfe-hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-<br>
FK520,30-desmethoxy-30-cis-hydroxy-31 -cis-hydroxy-3-hydroxy-prolyl-FK520, 30-<br>
desmethoxy-30-trans-hydrbxy-31 -trans-hydroxy-3-hydroxy-prolyl-FK520, 30-O-<br>
desmethyl-31-dehydroxy-3-hydroxy-prolyl-FK520,30-O-desmethyl-3-hydroxy-prolyl-<br>
FK520,30-desmethoxy-30-methyl-3-hydroxy-prolyl-FK520, 30-O-desmethyl-31-<br>
dehydroxy-31-methyl-3-hydroxy-prolyl-FK520&gt;30-O-desmethyl-31-dehydroxy-31-<br>
fluoro-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-fluoro-3-hydrow-prolyl-FK520, 30-<br>
O-desmethyl-31-dehydroxy-31-chloro-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-<br>
chloro-3-hydroxy-prolyl-FK520, 30-O-desmethyl-31 -dehydroxy-31 -ferf-butyl-3-<br>
hydroxy-prolyl-FK520,28-de(3-methoxy-4-hydroxy-cyclohexy])-28-(hydroxy-<br>
cycloheptyl)-3-hydroxy-prolyl-FK520, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbornyl)-3-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-31-hydroxy-3-<br>
hydroxy-prolyl-FK520,8-deoxo-30-desmethoxy-30-c;s-hydroxy-31-trans-hydroxy-3-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31 -cis-hydroxy-3-<br>
hydroxy-prolyl-FK520,8-debxo-30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-3-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-3-hydroxy-prolyl-<br>
FK520, 8-deoxo-30-O-desmethyl-3-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-<br>
30-methyl-3-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -<br>
methyl-3-hydroxy-prolyl-FK520) 8-deoxo-30-O-desmethyl-31-dehydroxy-31-fluoro-3-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-fluoro-3-hydroxy-proiyl-FK520, 8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-chloro-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-chloro-3-hydroxy-3-hydroxy-pro!yl-FK520, 8-deoxo-30-O-desmethyl-<br>
31 -dehydroxy-31 -terf-butyl-3-hydroxy-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK520, 8-deoxo-28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-3-hydroxy-prolyl-FK520,<br>
30-desmethoxy-4-hydroxy-prolyl-FK520,30-desmethoxy-30-cis-hydroxy-31-trans-<br>
hydroxy-4-hydroxy-prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-<br>
hydroxy-prolyl-FK520,30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-4-hydroxy-<br>
prolyl-FK520, 30-O-desmethyl-31-dehydroxy-4-hydroxy-prolyl-FK520, 30-O-<br><br>
desmetbyM-hydroxy-prolyl-FK520, 30-desmethoxy-30-methyl-4-hydroxy-prolyl-<br>
FK520, 30-O-desmethyl-31-dehydroxy-31-methyl-4-hydroxy-prolyl-FK520, 30-O-<br>
desmethyl-31 -dehydroxy-31 -fluoro-4-hydroxy-prolyl-FK520, 30-desmethoxy-30-<br>
tluoro-4-hydroxy-prolyl-FK520, 30-O-desmethyl-31 -dehydroxy-31 -chloro-4-hydroxy-<br>
prolyl-FK520, 30-desmethoxy-30-chloro-4-hydroxy-prolyl-FK520, 30-O-desmethyl-31 -<br>
dehydroxy-31 -tert-butyl-4-hydroxy-prolyl-FK5201 28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-cycloheptyl)-4-hydroxy-prolyl-FK520,28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl&gt;-28-(hydroxy-norbomyl)-4-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-31-hydroxy-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-<br>
hydroxy-31-irans-hydroxy-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-<br>
hydroxy-31-cis-hydroxy-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-trans-<br>
hydroxy-31-trans-hydroxy-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-<br>
dehydroxy-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-4-hydroxy-prolyl-.<br>
FK520, 8-deoxo-30-desmethoxy-30-methyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-<br>
desmethyl-31 -dehydroxy-31 -methyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-<br>
desmethyl-31-dehydroxy-31:tluoro-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-<br>
30-fluoro-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -chloro-<br>
4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-chloro-3-hydroxy-4-hydroxy-<br>
prolyl-FK520, 8-deoxo-30-O-desmethy^1 -dehydroxy-31 -ferf-butyl-4-hydroxy-prolyl-<br>
FK520, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyO-28-(hydroxy-cycIoheptyl)-4-<br>
hydroxy-prolyf-FK520, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
norbornyl)-4-hydroxy-prolyl-FK520,31-desmethoxy-fral7s-3-bicycloI3.1.0.JFK520, 31-<br>
desmethoxy-31-cis-hydroxy-32-trans-hydroxy-trans-3-bicyclo[3.1.0.]FK520, 31-<br>
desmethoxy-31-cis-hydroxy-32-cis-hydroxy-trans-3-bicyclo[3.1.0.]FK520, 31 -<br>
desmethoxy-31-trans-hydroxy-32-trans-hydroxy-<rans-3-bicyclo></rans-3-bicyclo>
desmethyl-32-dehydroxy-£rans-3-bicyclo[3.1.0.]FK520, 31-O-desmethyl-trans-3-<br>
bicyclop.1.0.]FK520&gt;31-desmethoxy-31-methyWrans-3-bicycto[3.1.0.]FK520, 31-O-<br>
desmethyl-32-dehydroxy-32-methykrans-3-bicyclo[3.1.0.]FK520, 31-O-desmethyl-<br>
32-dehydroxy-32-fluoro-trans-3-bicyclo[3.1.0.]FK520, 31-desmethoxy-31-fluoro-trans-<br>
3-bicyclo[3.1.0.]FK520, 31 -O-desmethyl-32-dehydroxy-32-chloro-trans-3-<br>
bicyclot3.1.0.]FK520,31-desmethoxy-31-chloro-trans-3-bicyclo[3.1.0.]FK520, 31-O-<br>
desmethyl-32-dehydroxy-32-tert-butyl-trans-3-bicyclo[3.1.0.]FK520, 29-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-cycloheptyl)-trans-3-<br>
bicyclo[3.1.0.]FK520, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
norbornyl)-trans-3-bicyclo[3.1.0.]FK520,9-deoxo-31 -desmethoxy-trans-3-<br><br>
bicyclo[3.1.0.]FK520, 9-deoxo-31-desmethoxy-31-cis-hydroxy-32-&amp;ial7S-hydroxy-<br>
frans-3-bicyclo[3.1.0.]FK520,9-deoxo-31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.JFK520, 9-deoxo-31-desmethoxy-31 -trans-hydroxy-32-<br>
frans-hydroxy-trans-3-bicyclo[3.1.0.]FK520I9-deoxo-31-O-desm6thy]-32-dehydroxy-<br>
frans-3-bicyclo[3.1.0.]FK520,9-deoxo-31-O-desmethyl-trans-3-bicyclo[3.1.0.]FK520,<br>
9-deoxo-31-desmethoxy-31-methyl-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-methyl-tans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31 -O-<br>
desmethyl-32-dehydroxy-32-fluoro-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31 -<br>
desmethoxy-31 -fluoro-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31 -O-desmethyl-32-<br>
dehydroxy-32-chloro-trans-3-bicyclo[3.1.0.]FK520,9-deoxo-31-desmethoxy-31-<br>
chloro-trans-3-bicyclo[3.1.0.]FK520,9-deoxo-31-O-desmethyl-32-dehydroxy-32-feff-<br>
butyl-trans-3-bicyclo[3.1.0.JFK520, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-cycloheptyl)-fral7s-3-bicyclo[3.1.0.JFK520, 9-deoxo-29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norbomyl)-trans-3-bicyclo[3.1.0.]FK520,<br>
In a more highly preferred embodiment, the present invention provides the<br>
following novel FK520 analogues: 31 -desmethoxy-31 -methyl-FK520, 31-<br>
desmethoxy-31.-fluon&gt;FK520, 31-desmethoxy-31-chlon&gt;FK520, 31-O-desmethyl-32-<br>
dehydroxy-32-ferf-butyl-FK520,29-de(3-methoxy-4-hydroxy-cycIohexyl)-29-(hydroxy-<br>
cycloheptyl)-FK520, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-<br>
FK520, 9-deoxo-31 -desmethoxy-31 -methyl-FK520, 9-deoxo-31 -desmethoxy-31 -<br>
fluoro-FK520,9-deoxo-31-desmethoxy-31-chloro-FK520, 9-deoxo-31-O-desmethyt-<br>
32-dehydroxy-32-fert-butyl-FK520, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-cycloheptyl)-FK520, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-norbornyl)-FK520, 30-desmethoxy-30-methyl-prolyl-FK520, 30-<br>
desmethoxy-30-fluoro-prolyl-FK520, 30-desmethoxy-30-chloro-prolyl-FK520, 30-<br>
desmethyl-31-dehydroxy-31-ferf-butyl-prolyl-FK520, 28-de(3-rnethoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-cycloheptyl)-FK520, 28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbornyr)-FK520)8-deoxo-30-desmethoxii'-31-hydroxy-<br>
prolyl-FK520, 8-deoxo-3f&gt;desmethoxy-30-cis-hydroxy-31-trans-hydroxy-prolyl-<br>
FK520, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31 -cis-hydroxy-prolyl-FK520, 8-<br>
deoxo-30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
O-desmethyl-31 -dehydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-prolyl-FK520, 8-<br>
deoxo-30-desmethoxy-30-methyl-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -<br>
dehydroxy-31-methyl-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-<br>
fluoro-prolyl-FK520, 8-deoxo-30-desmethoxy-30-fluoro-prolyl-FK520, 8-deoxo-30-O-<br><br>
desmethyl-31 -dehydroxy-31 -chloro-proly)-FK520, 8-deoxo-30-desmethoxy-30-chloro-<br>
prolyl-FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-terf-butyl-prolyl-FK520, 8-<br>
deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-prolyl-<br>
FK520,8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-<br>
prolyl-FK520,30-desmethoxy-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-cis-<br>
hydroxy-31 -trans-hydroxy-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-<br>
31 -cis-hydroxy-3-hydroxy-prolyl-FK520, 30-desmethoxy-30-<rans-hydroxy-31></rans-hydroxy-31>
hydroxy-3-hydroxy-prolyl-FK520, 30-O-desmethyl-31-dehydroxy-3-hydroxy-prolyl-<br>
FK520, 30-O-desmethyl-3-hydroxy-prolyl-FK520,30-desmethoxy-30-methyl-3-<br>
hydroxy-prolyl-FK520, 30-O-desmethyl-31 -dehydroxy-31 -methyl-3-hydroxy-prolyl-<br>
FK520, 30-O-desmethyl-31 -dehydroxy-31 -fluoro-3-hydroxy-pro!yl-FK520, 30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK520, 30-O-desmethyl-31 -dehydroxy-31 -<br>
chIoro-3-hydroxy-prolyl-FK520I30-desmethoxy-30-chloro-3-hydroxy-prolyl-FK520,<br>
30-O-desmethyl-31 -dehydroxy-31 -fe/t-butyl-3-hydroxy-prolyl-FK520,28-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-2S-(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK520, 28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbomyl)-3-hydroxy-prolyl-FK520,<br>
8-deoxp-30rdesmethoxy-31-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31 -4rans-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-irans-hydroxy-31 -frar/s-hydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
O-desmethyl-31 -dehydroxy-3-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-3-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-methyl-3-hydroxy-prolyl-FK5201 8-<br>
deoxo-30-O-desmethyl-31 -dehydroxy-31 -methyl-3-hydroxy-prolyl-FK520, 8-deoxo-<br>
30-O-desmethyl-31 -dehydroxy-31 -fluoro-3-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-fluoro-3-hydroxy-proly)-FK520, 8-deoxo-30-O-desmethyl-31 -<br>
dehydroxy-31-chloro-3-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-ch!oro-3-<br>
hydroxy^3-hydroxy-prolyl-FK520, 8-deoxo-3D-O-desmethyl-3lHJehydroxy-31-fe/t-<br>
butyl-3-hydroxy-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbornyl)-3-hydroxy-prolyl-FK520) 30-desmethoxy-4-<br>
hydroxy-prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-4-hydroxy-<br>
prolyl-FK520, 30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-hydroxy-prolyl-FK520,<br>
30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-4-hydroxy-prolyl-FK520, 30-O-<br>
desmethyl-31-dehydroxy-4-hydroxy-pro)yl-FK520, 3l&gt;0-desmethyl-4-hydroxy-pro^<br>
FK520, 30-desmethoxy-30-methyl-4-hydroxy-prolyl-FK520,30-O-desmethyl-31 -<br><br>
dehydroxy-31-methyl-4-hydroxy-proIyl-FK520,30-O-desmethyl-31-dehydroxy-31-<br>
fluoro-4-hydroxy-prolyl-FK520,30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK520, 30-<br>
O-desmethyl-31-dehydroxy-31-chloro-4-hydroxy-prolyl-FK520l 30-desmethoxy-30-<br>
chloro-4-hydroxy-prolyl-FK520,30-O-desmethyl-31-dehydroxy-31-fert-butyl-4-<br>
hydroxy-prolyl-FK520,28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
cycloheptyl)-4-hydroxy-prolyl-FK520, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbornyl)-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-31 -hydroxy-4-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK520) B-deoxo-30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-30-trans-hydroxy-31-frartS-hydroxy-4-<br>
hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31 -dehydroxy-4-hydroxy-prolyl-<br>
. FK520, 8-deoxo-30-O-desmethyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-desmethoxy-<br>
30-methyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-<br>
methyl-4-hydroxy-prolyl-FK520, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-fluoro-4-<br>
hydroxy-pro)yl-FK520, 8-deoxo-30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK520, 8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-chloro-4-hydroxy-prolyl-FK520, 8-deoxo-30-<br>
desmethoxy-30-chloro-3-hydroxy-4-hydroxy-prolyl-FK520,8-deoxo-30-O-desmethyl-<br>
31-dehydroxy-31~ferf-butyl-4-hydroxy-prolyl-FK520, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-4-hydroxy-prolyl-FK520, 8-deoxo-28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-4-hydroxy-prDlyl-FK520r<br>
31-desmethoxy-trans-3-bicyclo[3.1.0.]FK520, 31-desmethoxy-31-cis-hydroxy-32-<br>
frans-hydroxy-£ral7S-3-bicyclo[3.1.0.]FK520,31 -desmethoxy-31 -cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK520,31 -desmethoxy-31-trans-hydroxy-32-trans-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK520,31-O-desmethyl-32-dehydroxy-trans-3-<br>
bicyclo[3.1.0.]FK520, 31-O-desmethyMrans-3-bicyclo[3.1.0.]FK520, 31-desmethoxy-<br>
31-methyMrans-3-bicyclo[3.1.0.]FK520, 31-O-desmethyl-32-dehydroxy-32-methyl-<br>
Zrans-34)icyclo[3.1.0.)FK520, 31-O-desmethyl-32-dehydroxy-32-fluoro-foal7S-3-<br>
bicyclo[3.1.0.JFK520, 31-desmethoxy-31-fluorchfrans-3-bicyclo[3.1.0.JFK520, 31-O-<br>
desmethyl-32-dehydroxy-32-chloro-trans-3-b»cyclo[3.1.0.]FK520, 31-desmethoxy-31-<br>
chloro-trans-3-bicycio[3.1.0.]FK520, 31-O-desmethyl-32-dehydroxy-32-te/f-butyl-<br>
fral7S-3-bicyclo[3.1.0.]FK520, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
cycloheptyl)-£rans-3-bicyclo[3.1.0.JFK520, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-norbornyl)-trans-3-bicycio[3,1.0.JFK520, 9-deoxo-31 -desmethoxy-trans-<br>
3-bicyclo[3.1.0.JFK52O, 9-deoxo-31 -desmethoxy-31 -c;s-hydroxy-32-trans-hydroxy-<br>
frans-3-bicyclo[3.1.0.]FK520, 9^deoxo-31-desmethoxy-31-cis-hydroxy-32-c/5-<br><br>
hydroxy-trans-3-bicyclo[3.1.0.]FK520I9-deoxo-31-desmethoxy-31-trans-hydroxy-32-<br>
frBns-hydroxy-<rans-3-bicyclo></rans-3-bicyclo>
trans-3-b)cyc\o[3.1.0.]FK520, 9-deoxo-31-O-desmethyl-trans-3-bicyclo[3.1.0.JFK520,<br>
9-deoxo-31-desmethoxy-31-methyWrans-3-bicyclo[3.1.0.JFK520,9-deoxo-31 -O-<br>
desmethyl-32-dehydroxy-32-methyl-trans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-fluoro-irans-3-bicyclo{3.1.0JFK520, 9-deoxo-31 -<br>
desmethoxy-31-fluoro-trans-3-bicyclo[3.1.0.JFK520, 9-deoxo-31-O-desmethyl-32-<br>
dehydroxy-32-chloro-irans-3-bicyclo[3.1.0.]FK520, 9-deoxo-31-desmethoxy-31-<br>
chioro-trans-3-bicyclo[3.1.0.]FK520,9-deoxo-31-O-desmethyl-32-dehydroxy-32-terf-<br>
butyMral?s-3-bicyclo[3.1.0.]FK520, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-
hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-trans-3-bicyclo[3.1.0.]FK520.<br>
In a further aspect the present invention provides the following FK506<br>
analogues: 31-desmethoxy-FK506,31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-<br>
FK506, 31-desmethoxy-31-cis-hydroxy-32-cis-hydroxy-FK506, 31-desmethoxy-31-<br>
frans-hydroxy-32-&amp;-ans-hydroxy-FK506, 31-O-desmethyl-32-dehydroxy-FK506, 31-O-<br>
desmethyl-FK506, 31-desmethoxy-31-methyl-FK506,31-O-desmethyl-32-dehydroxy-<br>
32-methyl-FK506,31-O-desmethyl-32-dehydroxy-32-fluoro-FK506&gt;3l-desmethoxy-<br>
31-fluoro-FK506, 31-O-desmethyl-32-dehydroxy-32-chloro-FK506, 31-desmethoxy-<br>
31-chloro-FK506,31-O-desmethyl-32-dehydroxy-32-terf-butyl-FK506,29-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-cycloheptyl)-FK506, 29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-FK506, 9-deoxo-31-desmethoxy-FK506,<br>
9-deoxo-31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-FK506, 9-deoxo-31-<br>
desmethoxy-31-cis-hydroxy-32-cis-hydroxy-FK506, 9-deoxo-31-desmethoxy-31-<br>
frans-hydroxy-32-trans-hydroxy-FK506, 9-deoxo-31-O-desmethyl-32-dehydroxy-<br>
FK506,9-deoxo-31-O-desmethyl-FK566,9-deoxo-31-desmethoxy-31-methyl-FK506,<br>
9-deoxo^31-O-desmethyl-32-dehydroxy-32-methyl-FK506, 9-deoxo-31-O-desmethyl-<br>
32-dehydroxy-32-fluoro-FK506, 9-deoxo-31-desmethoxy-31-fluoro-FK506, 9-deoxo-<br>
31-O-desmethyl-32-dehydroxy-32-chIoro-FK506,9-deoxo-31-desmethoxy-31-chloro-<br>
FK506, 9-deoxo-31-O-desmethyl-32-dehydroxy-32-fert-butyl-FK506, 9-deoxo-29-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-cycloheptyl)-FK506, 9-deoxo-29-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-norbomyO-FK506,30-desmethoxy-<br>
prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-prolyl-FK506, 30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-prolyl-FK506, 30-desmethoxy-30-trans-<br>
hydroxy-31-franJ^hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-prolyl-FK506,<br><br>
30-O-desmethyl-prolyl-FK506, 30-desmethoxy-30-methyl-prolyl-FK506, 30-O-<br>
desmethyl-31-dehydroxy-31-methyl-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-31-<br>
fluoro-prolyl-FK506, 30-desmethoxy-30-fluoro-prolyl-FK506, 30-O-desmethyl-31-<br>
dehydroxy-31 -chloro-prolyl-FK506, 30-desmethoxy-30-chloro-prolyl-FK506, 30-O-<br>
desmethyl-31 -dehydroxy-31 -te/f-butyl-prolyl-FK506,28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-cycloheptyl)-FK506, 28-de(3-methoxy-4-hydroxy-<br>
cycIohexyl)-28-(hydroxy-norbomyl)-FK506, 8-deoxo-30-desmethoxy-31-hydroxy-<br>
prolyl-FK506, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-prolyl-<br>
FK506, 8-deoxo-30ntesmethoxy-30-cis-hydroxy-31 -cis-hydroxy-prolyl-FK506, 8-<br>
deoxo-30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
O-desmethyl-31 -dehydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-pro!yl-FK506, 8-<br>
deoxo-30-desmethoxy-30-methyl-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -<br>
dehydroxy-31 -methyl-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -<br>
fluoro-prolyl-FK506, 8-deoxo-30-desmethoxy-30-fluoro-prolyl-FK506, 8-deoxo-30-O-<br>
desmethyl-31-dehydroxy-31-chloro-prolyl-FK506, 8-deoxo-30-desmethoxy-30-chloro-<br>
prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -ferf-butyl-prolyl-FK506, 8-<br>
deoxo-28-de(3-methoxyr4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyO-prolyl-<br>
FK506, S-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-<br>
prolyl-FK506, 30-desmethoxy-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-<br>
hydroxy-31 -trans-hydroxy-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-<br>
31 -cis4iydroxy-3-hydroxy-prolyl-FK506, 3f&gt;desmethoxy-30-£rans-hydroxy-31 -trans-<br>
hydroxy-3-hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-3-hydroxy-prolyl-<br>
FK506, 30-O-desmethyl-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-methyl-3-<br>
hydroxy-prolyl-FK506,30-O-desmethyl-31 -dehydroxy-31 -methyl-3-hydroxy-prolyl-<br>
FK506, SO-O-desmethyl-SI-dehydroxy-SI-fluoro-S-hydroxy-prolyl-FKSOB, 30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK506,30-O-desmethyl-3lKiehydroxy-31-<br>
chloro-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-chloro-3-hydroxy-proIyl-FK50B,<br>
30-O-desmethyl-31-dehydroxyT31-terf-butyl-3-hydroxy-prolyl-FK506, 28-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK506, 2B-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-3-hydroxy-pro!yl-FK506,<br>
8-deoxo-30-desmethoxy-31 -hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31-irans-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-trans-hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
O-desmethyl-3l-dehydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-3-<br><br>
hydroxy-prolyl-FK506,8-deoxo-30-desmethoxy-30-methyl-3-hydroxy-prolyl-FK506, 8-<br>
deoxo-SO-O-desmethyl-SI-dehydroxy-SI-methyl-S-hydroxy-prolyl-FKSOe.S-deoxo-<br>
30-O-desmeth^31-dehydroxy-31-fluoro-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK506, 8-deoxo-30-Odesmethyl-31-<br>
dehydroxy-31-chloro-3-hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-chloro-3-<br>
hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-terf-<br>
butyl-3-hydroxy-proly)-FK506, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK506, 8-deoxo-28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbomyl)-3-hydroxy-prolyl-FK506, 30-desmethoxy-4-<br>
hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-4-hydroxy-<br>
prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31-cfe-hydroxy-4-hydroxy-prolyl-FK506,<br>
30-desmethoxy-30-trans-hydroxy-31 -trans-hydroxy-4-hydroxy-prolyl-FK506,30-O-<br>
desmethyl-31-dehydroxy-4-hydroxy-proIyl-FK506, 30-O-desmethyl-4-hydroxy-prolyl-<br>
FK506, 30-desmethoxy-30-methyl-4-hydroxy-prolyl-FK506, 30-O-desmethyl-31-<br>
dehydroxy-31 -methyl-4-hydroxy-prolyl-FK506, 30-O-desmethjri-31 -dehydroxy-31 -<br>
fluoro-4-hydroxy-prolyl-FK506, 30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK506, 30-<br>
0-desmethyl-31 -dehydroxy-31 -chloro-4-hydroxy-prolyl-FK5D6, 30-desmethoxy-30-<br>
chloro-4-hydroxy-proIyl-FK506, 30-O-desmethyl-31 -dehydroxy-31 -terf-butyl-4-<br>
hydroxy-pro)yl-FK506l28-de(3-metboxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
cycloheptyl)-4-hydroxy-prolyl-FK506, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbornyl)-4-hydroxy-proIyl-FK506, 8-deoxo-30-desmethoxy-31-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31 -lrans-hydroxy-4-<br>
hydroxy-prolyl-FK506,8-deoxo-30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31-dehydroxy-4-hydroxy-prolyl-<br>
FK506, 8-deoxo-30-O-desmethyM-hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-<br>
30-meth~yl-4-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -<br>
methyl-4-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-fluoro-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK506, 8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-chloro-4-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-chloro-3-hydroxy-4-hydroxy-proiyl-FK506, 8-deoxo-3r&gt;0-desmethyl-<br>
SI-dehydroxy-SI-tert-butyl-^hydroxy-prolyl-FKSOe, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-4-hydroxy-prolyl-FK506, 8-deoxo-28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-4-hydroxy-prolyl-FK506,<br>
31-desmethoxy-fral7S-3-bicyclo[3.1.0.]FK506,31-desmethoxy-31-cis-hydroxy-32-<br><br>
frans-hydroxy-trans-3-bicyclo[3.1.0.]FK506,31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK506, 31-desmethoxy-31-trans-hydroxy-32-frBns-<br>
hydroxy-trans-3-bicyclo[3.1.0.JFK506, 31-O-desmethyl-32-dehydroxy-trans-3-<br>
bicyclo[3.1.0.JFK506, 31-O-desmethyWrans-3-bicyclo[3.1.0.]FK506, 31-desmethoxy-<br>
31 -methyl-trans-3-bicyclo[3.1.0.JFK506,31-O-desmethyl-32-dehydroxy-32-methyl-<br>
fral7s-3-bicyclo[3.1.0.JFK506, 31 -O-desmethyl-32-dehydroxy-32-fiuoro-fral7s-3-<br>
bicyclo[3.1.0.JFK506, 31-desmethoxy-31-fluoro-trans-3-bicyclo[3.1.0.JFK506, 31-O-<br>
desmethyl-32-dehydroxy-32-chloro-trans-3-bicyc)o[3.1.0.]FK506, 31 -desmethoxy-31 -<br>
chloro-trans-3-bicyclo[3.1.0.]FK506, 31-O-desmethyl-32-dehydroxy-32-terf-butyl-<br>
frans-3-bicyclo[3.1.0.]FK506, 29-de(3-methoxy-4-hydroxy-cycIohexyJ)-29-(hydroxy-<br>
cycloheptyl)-trans-3-bicyclo[3.1.0.]FK506,29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-norbornyl)-f/ans-3-bicyclo[3.1.0.JFK506, 9-deoxo-31 -desmethoxy-trans-<br>
3-bicyclo[3.1.0.]FK506,9-deoxo-31-desmethoxy-31 -cis-hydroxy-32-trans-hydroxy-<br>
frans-3-bicyclo[3.1.0.]FK506,9-deoxo-31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK506,9-deoxo-31-desmethoxy-31-irans-hydroxy-32-<br>
frans-hydroxy-fral?s-3-bicyclo[3.1.0JFK505, 9-deoxo-31 -O-desmethyl-32-dehydroxy-<br>
frans-3-bicyclo[3.1.0.JFK506,9-deoxo-31-O-desmethyl-trans-3-bicyclo[3.1.0JFK506,<br>
9-deoxo-31-desmethoxy-31-methyWrans-3-bicyclo[3.1.0.3FK506,9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-methyl-trans-3-bicyclo[3.1.0.]FK506,9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-fluoro-trans-3-bicyclo[3.1.0.]FK506,9-deoxo-31-<br>
desmethoxy-31-fluoro-f/al7S-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-desmethyl-32-<br>
dehydroxy-32-chIoro-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31 -desmethoxy-31 -<br>
chloro-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-desmethyl-32-dehydroxy-32-fert-<br>
butyl-trans-3-bicyclo[3.1.0.JFK506, 9-deoxo-29-de(3-methaxy-4~hydroxy-cyclohexyl)-<br>
29-(hydroxy-cycloheptyl)-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norbomyl)-trans-3-bicyclo[3.3.0.]FK506,<br>
In a preferred embodiment, the present invention provides the following FK506<br>
analogues: 31 -desmethoxy-31 -cis-hydroxy-3,2-trans-hydroxy-FK506, 31 -desmethoxy-<br>
31-cis-hydroxy-32-cis-hydroxy-FK506,31-desmethoxy-31-trans-hydroxy-32-trans-<br>
hydroxy-FK506, 31-desmethoxy-31 -methyl-FK506, 31-O-desmethyl-32-dehydroxy-<br>
32-terf-butyl-FK506, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
cycloheptyl)-FK506, 29-de(3-methoxy-4-hydroxy-cycJohexyl)-29-(hydroxy-norbornyl)-<br>
FK506, 9-deoxo-31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-FK506,9-deoxo-<br>
31-desmethoxy-31-cis-hydroxy-32-cis-hydroxy-FK506, 9-deoxo-31-desmethoxy-31-<br>
frans-hydroxy-32-trans-hydroxy-FK506,9-deoxo-31-desmethoxy-31-methyl-FK506,<br><br>
9-deoxo-31 -O-desmethyl-Sa-dehydroxy-Sa-te/f-butyl-FKSOe, 9-deoxo-29-de(3-<br>
methoxy-^-hydroxy-cyclohexyl^Q-^ydroxy-cycloheptylJ-FKSOB, 9-deoxo-29-de(3-<br>
methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-FK506,.30-desmethoxy-30-<br>
cis-hydroxy-31 -fral?s-hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31 -cis-<br>
hydroxy-prolyl-FK506, 30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-prolyl-<br>
FK506,30-desmethoxy-30-methyl-prolyl-FK506,30-O-desmethyl-31-dehydroxy-31-<br>
fert-butyl-prolyl-FK506128-de(3-methoxy-4-hydroxy-c^clohexyl)-28^(hydroxy-<br>
cycloheptyl)-FK506,28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-<br>
FK506, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-prolyl-FK506, 8-<br>
deoxo-30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-proIyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-trans-hydroxy-31-trans-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-methyl-protyl-FK506, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-terf-<br>
butyl-prolyl-FK506,8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
cycloheptyl)-pro!yl-FK506, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbornyl)-prolyl-FK506, 30-desmBthoxy-3-hydroxy-prolyl-FK506, 30-<br>
desmethoxy-30-cis-hydroxy-31-trans-hydroxy-3-hydroxy-pro)yl-FK506, 30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-3-hydroxy-prolyl-FK506, 30-desmethoxy-<br>
30-trans-hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-FK506, 30-O-desmethyl-31-<br>
dehydroxy-S-hydroxy-prolyl-FKSOe.SO-O-desmethyl-S-hydroxy-proIyl-FKSOe, 30-<br>
desmethoxy-30-methyl-3-hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-31-<br>
methyl-3-hydroxy-prolyl-FK506,30-O-desmethyJ-31-dehydroxy-31-fluoro-3-hydroxy-<br>
prolyl-FK506, 30-desmethoxy-30-f!uoro-3-hydroxy-prolyl-FK506, 30-O-desmethyl-31 -<br>
dehydroxy-31 -chloro-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-chloro-3-hydroxy-<br>
pro!yl-FK506, 30-O-desmethyl-31-dehydroxy-31-ferf-butyl-3-hydroxy-proiyl-FK506,<br>
28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-3-hydroxy-prolyl-<br>
FK506, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-Qorbornyl)-3-hydroxy-<br>
prolyl-FK506, 8-deoxo-30-desmethoxy-31 -hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-<br>
30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-<br>
30-desmethoxy-30-cfe-hydroxy-31-cis-hydroxy-3-hydroxy-proIyl-FK506, 8-deoxo-30-<br>
desmethoxy-SO-trans-hydroxy-SI-trans-hydroxy-S-hydroxy-prolyl-FKSOB, 8-deoxo-30-<br>
O-desmethyl-SI-dehydroxy-S-hydroxy-prolyl-FKSOe, 8-deoxo-30-O-desmethyl-3-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-methyl-3-hydroxy-prolyl-FK506, 8-<br>
deoxo-30-O-desmethyl-31 -dehydroxy-31 -methyl-3-hydroxy-prolyl-FK506, 8-deoxo-<br>
30-O-desmethyl-31-dehydroxy-31-fluoro-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -<br><br>
dehydroxy-31-chloro-3-hydroxy-prolyl-FK506t 8-deoxo-30-desmethoxy-30-chloro-3-<br>
hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -tert-<br>
butyl-S-hydroxy-prolyl-FKSOe, 8-deoxo-28-de(3-methpxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK506, 8-deoxo-28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbornyl)-3-hydroxy-prolyl-FK506, 30-desmethoxy-4-<br>
hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-4-hydroxy-<br>
prolyl-FK506, 30&gt;desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-hydroxy-prolyl-FK506,<br>
30-desmethoxy-30-fral7S-hydroxy-31-trans-hydroxy-4-hydroxy-prolyl-FK506,30-O-<br>
desmethyl-31-dehydroxy-4-hydroxy-prolyl-FK506, 30-O-desmethyl-4-hydroxy-prolyl-<br>
FK506, 30-desmethoxy-30-methyl-4-hydroxy-proiyl-FK506,30-O-desmethyl-31-<br>
dehydroxy-31-methyl-4-hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-31-<br>
fluoro-4-hydroxy-prolyl-FK506, 30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK506, 30-<br>
0-desmethyl-31-dehydroxy-31-chIoro-4-hydroxy-prolyl-FK506, 30-desmethoxy-30-<br>
chloro^-hydroxy-prolyl-FKSOe, 30-O-desmethyl-31 -dehydroxy-31 -ferf-butyl-4-<br>
hydroxy-prolyl-FK506, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
cycloheptyl)-4-hydroxy-prolyl-FK506, 28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbornyl)-4^hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-31 -hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-3f&gt;cis-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-ffans-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31-dehydroxy-4-hydroxy-prolyl-<br>
FK506,8-deoxo-30-O-desmethyl-4-hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-<br>
30-methyf-4-hydroxy-prolyl-FK506,8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -<br>
methyl-4-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -fluoro-4-<br>
hydroxy-proIyl-FK506, 8-deoxo-30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK506, 8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-chloro-4-hydroxy-prQlyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-chloro-3-hydroxy-4-hydroxy-prolyl-FK506,8-deoxo-30-O-desmethyl-<br>
31-dehydroxy-31-fert-butyl-4-hydroxy-prolyl-FK506, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-4-hydroxy-prolyl-FK506, 8-deoxo-28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbomyl)-4-hydroxy-prolyl-FK5061<br>
31 -desmethoxy-trans-3-bicyclo[3.1.0.JFK506, 31 -desmethoxy-31 -cis-hydroxy-32-<br>
frans-hydroxy-trans-3-bicycio[3.1.0.]FK506, 31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicycio[3.1.0.]FK506, 31-desmethoxy-31-trans-hydroxy-32-trans-<br>
hydroxy-trans-3-bicyclo[3.1.0.JFK506,31 -O-desmethyl-32-dehydroxy-trans-3-<br>
bicyc)o[3.1.0.]FK506, 31-O-desmethyWrans-3-bicyclo[3.1.i).]FK50B, 31-desmethoxy-<br><br>
31-me%kran^3-bicyclo[3.1.0JFK506,31-O-desmethyl-32-dehydroxy-32-methyl-<br>
frans-3-bicyclo[3.1.0.]FK506, 31-O-desmethyh32-dehydroxy-32-fluoro-trans-3-<br>
bicyclo[3.1.0.]FK506, 31-desmethoxy-31-fIuoro-trans-3-bicyclo[3.1.0.JFK506, 31TO-<br>
desmethyl-32-dehydroxy-32-chloro-trans-3-bicyclo[3.1.0.]FK506, 31-desmethoxy-31-<br>
chloro-trans-3-bicycio[3.1.0.]FK506, 31-O-desmethyl-32-dehydroxy-32-terf-butyl-<br>
frans-3-bicyclo[3.1.0.JFK506, 29-de(3-methoxy-4-hydroxy-K;yclohexyl)-29-(hydroxy-<br>
cycloheptyl)-trans-3-bicyclo[3.1.0.]FK506,29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-norbornyl)-frBns-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-desmethoxy-frBns-<br>
3-bicyclo[3.1.0.]FK506, 9-deoxo-31-desmethoxy-31-cis-hydroxy-32-trans-hydroxy-<br>
frans-3-bicydo[3.1.0.]FK506, 9-deoxo-31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-fral7S-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-desmethoxy-31-trans-hydroxy-32-<br>
frans-hydroxy-trans-3-bicyclo[3.1.0.]FK506,9-deoxo-31-O-desmethyl-32-dehydroxy-<br>
frans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-desmethyl-trans-3-bicyclo[3.1.0.]FK506,<br>
9-deoxo-31-desmethoxy-31-methyl-trans-3-bicyclo[3.1.0JFK506,9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-methyl-ffBns-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-fluoro-fral7S-3-bicyclo[3.1.0.JFK506, 9-deoxo-31 -<br>
desmethoxy-31-fluoro-trans-3-bicyclo[3.lO.]FK506, 9-dBoxo-31-O-desmethy^32- .<br>
dehydroxy-32-chloro-tral7S-3-bicyclot3.1.0.]FK506, 9-deoxo-31-desmethoxy-31-<br>
chloro-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-desmethyl-32-dehydroxy-32-terf-<br>
butyl-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-cycloheptyl)-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norbornyO-trans-3-bicyclo[3.1,0.]FK506.<br>
In a more highly preferred embodiment, the present invention provides the<br>
following FK506 analogues: 31-desmethoxy-31-methyl-FK506,31-O-desmethyl-32-<br>
dehydroxy-32-terf-butyl-FK506,29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
cycloheptyl)-FK506, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-norbornyl)-<br>
FK506,!9-deoxo-31-desmethoxy-31-methyl-FK506,9-deoxo-3i-O-desmethyl-32-<br>
dehydroxy-32-tert-butyl-FK506,9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-<br>
(hydroxy-cycloheptyl)-FK506, 9-deoxo*29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-<br>
(hydroxy-norbornyl)-FK506,30-desmethoxy-30-methyl-prolyl-FK506, 30-O-<br>
desmethyl-31 -dehydroxy-31 -tert-butyl-prolyl-FK506, 28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-cycloheptyO-FK506, 28-de(3-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbomyl)-FK506, 8-deoxo-30-desmethoxy-30-methyl-<br>
prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -ferf-butyl-prolyl-FK506, 8-<br>
deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-prolyl-<br><br>
FK506,8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbomyl)-<br>
prolyl-FK506, 30-desmethoxy-3-hydroxy-prolyl-FK506,30-desmethoxy-30-cis-<br>
hydroxy-31-frans*hydroxy-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-<br>
31-cfe-hydroxy-3-hydroxy-prolyl-FK506,30-desmethoxy-30-trans-hydroxy-31-trans-<br>
hydroxy-3-hydroxy-prolyl-FK506, 30-Odesmethyl-31-dehydroxy-3-hydroxy-prolyl-<br>
FK506,30-O-desmethyl-3-hydroxy-prolyl-FK506,30-desmethoxy-30-methyl-3-<br>
hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-31-methyl-3-hydroxy-prolyl-<br>
FK506, 3f&gt;CVdesmethyl-31-dehydroxy-31-fluoro-3-hydroxy-prolyl-FK506, 30-<br>
desmethoxy-30-fluoro-3-hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-31-<br>
chloro-3-hydroxy-prolyl-FK506, 30-desmethoxy-30-chloro-3-hydroxy-prolyl-FK506,<br>
30-O-desmethyl-31 -dehydroxy-31 -tert-birtyl-3-hydroxy-prolyl-FK506, 28-de(3-<br>
methoxy-4-hydroxy-cyclohexy()-28-(hydroxy-cycloheptyl)-3-hydroxy-prolyl-FK506, 28-<br>
de(3-methoxy-4-hydroxy-cydohexyl)-28-(hydroxy-norbomyl)-3-hydroxy-prolyl-FK506,<br>
8-deoxo-30-desmethoxy-31-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-cis-hydroxy-31-fcnans-hydroxy-3-hydroxy-prolyl-FK506,8-deoxo30-<br>
desmethoxy-30-cis-hydroxy-31-cis-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desrnethoxy-30-trans-hydroxy-31 -trans-hydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
O-desmethyl-31-dehydroxy-3-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-3-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-methyl-3-hydroxy-prolyl-FK506, 8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-methyl-3-hydroxy-prolyl-FK506, 8-deoxo-<br>
30-O-desmethyl-31-dehydroxy-31-fluoro-3-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-fluoro-3-hydroxy-protyl-FK506,8-deoxo-30-O-desmethyl-31 -<br>
dehydroxy-31-chloro-3-hydroxy-profyl-FK506, 8-deoxo-30-desmethoxy-30-chloro-3-<br>
hydroxy-3-hydroxy-prolyl-FK506, B-deoxo-30-O-desmethyl-31 -dehydroxy-31 -tert-<br>
butyl-3-hydroxy-prolyl-FK506, 8-deoxo-28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-<br>
(hydroxy-cycloheptyO-S-hydroxy-prolyl-FKSOB, 8-deoxo-28-de^-methoxy-4-hydroxy-<br>
cyclohexyl)-28-(hydroxy-norbornyl)-3-hydroxy-prolyl-FK506, 30-desmethoxy-4-<br>
hydroxy-prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31-trans-hydroxy-4-hydroxy-<br>
prolyl-FK506, 30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-hydroxy-prolyl-FK506,<br>
30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-4-hydroxy-prolyl-FK506, 30-O-<br>
desmethyl-31-dehydroxy-4-hydroxy-prolyl-FK506, 30-O-desmethyl-4-hydroxy-prolyl-<br>
FK506,30-desm6thoxy-30-methyl-4-hydroxy-prolyl-FK506, 30-O-desmethyl-31 -<br>
dehydroxy-31 -methyl-4-hydroxy-prolyl-FK506, 30-O-desmethyl-31 -dehydroxy-31 -<br>
fluoro-4-hydroxy-prolyl-FK506,30-desmethoxy-30-fluoro-4-hydroxy-proiyl-FK506, 30-<br>
O-desmethyl-31 -dehydroxy-31 -chloro-4-hydroxy-prolyl-FK506, 30-desmethoxy-30-<br><br>
chloro-4-hydroxy-prolyl-FK506, 30-O-desmethyl-31-dehydroxy-31-terf-butyl-4-<br>
hydroxy-prolyl-FK506,28-de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-<br>
cycloheptylH^hydroxy-prolyl-FKSOB, 28-de(3-methoxy-4-:hydroxy-cyclohexyl)-28-<br>
(hydroxy-norbomyl)-4-hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-31 -hydroxy-4-<br>
hydroxy-prolyl^K506,8-deoxo-3Cklesmethoxy-30-cis-hydroxy-31-frahs-hydroxy-4-<br>
hydroxy-proly(-FK506, 8-deoxo-30-desmethoxy-30-cis-hydroxy-31-cis-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-30-trans-hydroxy-31-trans-hydroxy-4-<br>
hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-4-hydroxy-prolyl-<br>
FK506, 8-deoxo-30-O-desmethyl-4-hydroxy-prolyl-FK506, 8-deoxo-30-desmethoxy-<br>
30-methyM-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31-dehydroxy-31-<br>
methyl-4-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-31 -dehydroxy-31 -fluoro-4-<br>
hydroxy-prolyl-FK506,8-deoxo-30-desmethoxy-30-fluoro-4-hydroxy-prolyl-FK506,8-<br>
deoxo-30-O-desmethyl-31-dehydroxy-31-chloro-4-hydroxy-prolyl-FK506, 8-deoxo-30-<br>
desmethoxy-30-chloro-3-hydroxyU-hydroxy-prolyl-FK506, 8-deoxo-30-O-desmethyl-<br>
31-dehydroxy-31-terf-butyl-4-hydroxy-prolyl-FK506, 8-deoxo-28-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-28-(hydroxy-cycloheptyl)-4-hydroxy-prolyl-FK506,8-deoxo-28-<br>
de(3-methoxy-4-hydroxy-cyclohexyl)-28-(hydroxy-norbornyl)-4-hydroxy-prolyl-FK506,<br>
31 -desmethoxy-trans-3-bicyclo[3.1.0.JFK506,31 -desmethoxy-31 -cis-hydroxy-32-<br>
frans-hydroxy-fral7S-3-bicyc!o[3.1:0.]FK506, 31-desmethoxy-31-cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK506, 31-desmethoxy-31-trans-hydroxy-32-f/ans-<br>
hydroxy-trans-3-bicyclo[3.1.0.JFK506,31 -O-desmethyl-32-dehydroxy-trans-3-<br>
bicyclo[3.1.0.IFK506,31-O-desmethyMrans-3-bicyclo[3.1.0.JFK5O6, 31-desmethoxy-<br>
31-methyl-trans-3-bicyclo[3.1.0.]FK506,31-O-desmethyl-32-dehydroxy-32-methyl-<br>
frans-3-bicyclo[3.1.0.]FK506,31-C&gt;-desmethyl-32-dehydroxy-32-f[uoro-fr3ns-3-<br>
bicyclo[3.1.0.]FK506, 31-desmethoxy-31-fluoro-trans-3-bicyclo[3.1.0.JFK506, 31-O-<br>
desmethyl-32-dehydroxy-32-chloro-iral)s-3-bicyclo[3.1.0.JFK506, 31 -desmethoxy-31 -<br>
chloro-&amp;ans-3-bicyclo[3.1.0.]FK506)31-O-desmethyl-32-dehydroxy-32-tert-butyl-<br>
frans-3-bicyclo[3.1.0.]FK506, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-29-(hydroxy-<br>
cyc)oheptyl)^frans-3-bicyclo[3.1.0.JFK506, 29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-norbornyl)-^rans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-desmethoxy-trans-<br>
3-bicycIo[3.1.0.]FK506,9-deoxo-31-desmethoxy-31-cis-hydroxy-32-fral7s-hydroxy-<br>
frans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31 -desmethoxy-31 -cis-hydroxy-32-cis-<br>
hydroxy-trans-3-bicyclo[3.1.0.]FK506,9-deoxo-31 -desmethoxy-31 -trans-hydroxy-32-<br>
fral7S-hydroxy-trans-3-bicyclo[3.1.0.]FK506,9-deoxo-31 -O-desmethyl-32-dehydroxy-<br>
frans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-desmethyl-trans-3-bicyclo[3.1.0.JFK506,<br><br>
9-deoxo-31 -desmethoxy-31 -methyMrans-3~bicyclo[3.1.0.JFK506, 9-deoxo-31 -O-<br>
desmethyl-32-dehydroxy-32-methyl-irans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-<br>
desmethyl-32-dehydroxy-32-fluoro-trans-3-bicyclo[3.1.0JFK506,9-deoxo-31-<br>
desmethoxy-31-fluoroHfrans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-O-desmethyl-32-<br>
dehydroxy-32-ch!oro-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-31-desmethoxy-31-<br>
chloro-&amp;ans-3-bicyclot3.1.0.]FK506,9-deoxo-31-O-desmethyl-32-dehydroxy-32-fert-<br>
butyl-trans-3-bicyclo[3.1.0.JFK506, 9-deoxo-29-de(3-methoxy-4-hydroxy-cyclohexyl)-<br>
29-(hydroxy-cycloheptyl)-trans-3-bicyclo[3.1.0.]FK506, 9-deoxo-29-de(3-methoxy-4-<br>
hydroxy-cyclohexyl)-29-(hydroxy-norbomyl)-fralJs-3-bicyclo[3.1.0.JFK506.<br>
In further aspects the invention provides:<br>
A: Compounds of the formula:<br><br>
where:<br>
x = bond or CHR1,, or -CHFVx-CHrV is<br><br>
R15 =<br><br><br>
R1 = OH, OCH3<br>
R2 = H, OH, OCH3<br>
R3 = H, OH, CH3, F, CI, OCH3<br>
R4 = H, OH, CH3, F, CI<br>
R5 = H, OH<br>
R6 = H, OH<br>
R7 = H<br>
R8=H, keto.<br>
R9 = H, keto<br>
R10 = H<br>
R11 = H<br>
R13 = H<br>
R14 = H<br>
R16 = OH, OCH3<br>
R17 = H, OH, CI, F and<br>
y = bond, CH2<br>
with the-proviso that the compounds do not include the following::<br>
i) where R1 = OCH3 in combination with R2= H, R« = C, RIB = cis-3-OH, Rn7<br>
= trans-4-OH, R5= H, R6= H, R7= H, R8= H, R9= H, Ri0= H, R„ = H, x =<br>
CHRn;<br>
ii) where R1 - OH in combination with R2= OCH3, R1E = C, R16 = cis-3-OH,<br>
R17= trans-A-OH, R5= H, R6= H, R7= H, RB,R9 = keto, R10= H Rin = H, x =<br>
CHRn,'<br>
iii) where Rt = OH in combination with R2 = OH, R16 = C, R16 = cis-3-OCH3,<br>
R17= trans-A-OH, R5= H, R6= H, R7= H, RB,RB =keto, R10= H, Rn = H, x =<br>
CHR11;<br><br>
iv) where Ri = OH in combination with R2 = H, RiS = C, R16 = cis-3-OCH3, R17<br>
= trans-4-OH, R6= H, Re= H, R7= H, R8lR9 = keto, R10= H, Rn = H, x =<br>
CHR11;<br>
v) where R1 = OCH3 in combination with R2 = H, R15 = C, R16 = cis-3-OH, R17<br>
= frans-4-OH, RB= H, Rs= H, R7= H, R8lR9 = keto, R10 = H, Rn = H, x =<br>
CHR11;<br>
vi) where R1 = OCH3 in combination with R2= H, R15 = C, Ri6 = cis-3-OCH3,<br>
Ri7= trans-4-OH, R6 = H, Re = H, R7 = H, R8 = H, R9 = H, R10 = H, R„ =<br>
H, x = CHRni<br>
vii) except where R1 = OCH3 in combination with R2 = OH, R15 = C, Ri6 = cis-<br>
3-OCH3l R17= frans-4-OH, R5 = H, Re = H, R7 = H, R8 = H, R9 = H, R10 =<br>
H,R11 = H,x = CHRi1;<br>
vii*0 where RA - OCH3 in combination with R2 = OCH3, R« = C, Ri6 - cisr3-<br>
OCH3, R17 = frans-4-OH, R5 = H, Re = H, R7 = H, R6 = H, R9 = H, R10 = H,<br>
R11 = H,x = CHR11;<br>
be) where R-, = OH in combination with R2 = OCH3, R16 = C, R16 = cis-3-OCH3,<br>
Ri7= trans-40H, R5 = H, R6 = H, R7 = H, R8|R8 = keto, R10 = H, RM = H, x<br>
= CHR11;<br>
x) where R., = OCH3 in combination with R2 = OH, R15 = C, Ri6 = cis-3-OCH3,<br>
R17= trans-4-OH, R5 = H, R6 = H, R7 = H, RB,R9 = keto, R10 = H, R1n = H, x<br>
= CHR„;<br>
xi) where R1 = OCH3 in combination with R2 = H, R15 = C, R16 = cis-3-OCH3,<br>
R17= frans-4-OH, R5 = H, R6 = H, R7 = H, R8,R9 = keto, R10 = H, Ru = H, x<br>
= CHRni<br>
xii) where R-, = OCH3 in combination with R2 = OCH3, Rn5 = C, R16 = cis-3-OH,<br>
R17 = frans-4-OH, R5 = H, Re = H, R7 = H, R8,R9 = keto, R10 = H, R„ = H, x<br>
= CHRu;<br>
xiii) where R1 = OCH3 in combination with R2 = H, Ri5 = C, R16 = cis-3-OCH3,<br>
R17= trans-4-OH, R6 = H, R6 = H, R7 = H, Re = H, R9 = H, R10 = H, x =<br>
bond;<br>
xiv) where R1 = OCH3 in combination with R2 = OCH3, R15 = C, RiB = cis-3-<br>
OCH3, R17 = trans-4-OH, R5 = H, Re = H, R7 = H, R8 = H. R9 = H, R10 = H,<br>
x = bond;<br>
xv) where R1 = OCH3 in combination with R2 = OH, R15 = C, R16 = cis-3-OCH3)<br>
R17 = trans-A-OH, R5 = H, R6 = H, R7 = H, RB,R9 = keto, R1D = H, x = bond;<br><br>
xvi) where Rt = OCH3 in combination with R2 = H, R« = C, R16 = cis-3-OCH3,<br>
Ri7a trans-4-OH, R5 = H, Re = H, R7 = H, RB.RO = keto, R10 = H, x = bond;<br>
xvii) where Rn = OCH3 in combination with R2 = OCH3, R15 = C, R1e = H, R17 =<br>
OH, R5 = H, Re = H, R7 = H, R8|R8 = keto, R10 = H, Rn = H, x = CHRu;<br>
xviii) where -CHRe-x-CHRu- is and R^ = H, R13 = H, R14 = H, in<br>
combination with Ri = OCH3, R2 = OCH3) R16 = C, R16 = cis-3-OCH3, R17 =<br>
trans-A-OH, R7 = H, R8, R9 = keto, R10 = H;<br>
xix) where R16 = G, R16 = c;s-3-OCH3, Ri7 = trans-A-OH, y = bond, in<br>
combination with Ri = OCH3, R2 = H, R5 = H, Re = OH, R7 = H, Rin = H, x<br>
= bond, RB,R9 = keto, R10 = H<br>
xx) where R16 = G, R3 = H, R4 = frans-OH, y = bond, in combination with Ri =<br>
OCH3, R2 = OCH3, R5 = H, Re = H, R7 = H, R-n = H, x = CHRu, RBIRS =<br>
keto, R10 = H<br>
xxi) where R15 = G, R3 = H, R4 = OH, y = CH2 in combination with R1 = OCH3,<br>
R2 = OCH3; R5 = H, R6 = H, R7 = H, R-n = H, x = CHRu, RB.RS = keto, R10<br>
= H<br>
xxii) where Ri5 = G, R3 = cis-OH, R4 = H, y = bond, in combination with Ri =<br>
OCH3, R2 = OCH3, R5 = H, Re= H, R7 = H, Rn = H, x = CHRu, Re,Re =<br>
keto, R10 = H<br>
xxiii) where R15 = G, R3 = CH3, R4 = OH, y = bond, in combination with Rn =<br>
OCH3, R2 = OCH3, R5= H, Re= H, R7 = H, R-n = H, x = CHR11, RB,R9 =<br>
keto, R10 = H<br>
xxiv) where R15 = G, R3 = H, R4 = OH, y = CH2, in combination with Ri = OH, R2<br>
= OH, R5 = H, Re = H, R7 = H, R^ = H, x = CHRu, RB=RS = H, R10 = H<br>
xxvjr where R16 = G, R3 = H, R4 = OH, y = CH2, in combination with R-, = OCH3,<br>
R2 = OCH3, Rg = H, Re = H, R7 = H, R11 = H, X = CHRu, R8=R9 = H, R10 =<br>
H<br>
xxvi) where R16 = G, R3 = H, R1 = OH, y - CH2, in combination with R1 = OH, R2<br>
= OCH3, R5 = H, Re = H, R7 = H, Rn = H, x = CHR1,, R8=R9 = H, R10 = H<br>
xxvii) where R15 = G, R3 = H, R4 = OH, y = CH2, in combination with Ri = OH, R2<br>
= H, R5 = H, Re = H, R7 = H, R^ = H, x = CHRu, Re=Rs = H, R10 = Hj<br>
xxviii) where R15 = G, R3 = H, R4 = OH, y = CH2, in combination with Ri = OH, R2<br>
= OCH3, R5 = H, R6 = H, R7 = H, R„ = H, x = CHRu, R8,R9 = keto, R10 = H<br><br>
xxix) where R15= G, R3 = H, R4 = OH, y = CH2, in combination with R1 = OCH3,<br>
R2 = H, R5 = H, Re = H, R7 = H, R;i = H, x = CHR„, RBjR9 = keto, R10 = H<br>
B. Compounds according to the formula below<br><br>
where<br>
R, = OH, OCH3<br>
R2 = H, OH, OCH3<br>
R3 = H, OH, CH3, OCH3<br>
R4 = H,OH<br>
R5 = H<br>
Re = H, OH<br>
R7 = H<br>
RB = H, keto<br>
R9 = H, keto<br>
Rio = H<br>
x = bonct, CH2 or -CHRe-x-CHRs- is<br><br>
Rn = H<br>
R13 = H<br>
Ri4 = H<br>
y = bond, CH2<br>
with the proviso that the compounds do not include the following:<br><br>
0	where R3 = H, R4 = trans-OH, y = bond, in combination with Rt =<br>
OCH3) R2 = OCH3l R6 = H, Re = H, R7 = H, x = CH2, R8)R9 = keto, R10<br>
= H<br>
ii) where R3 = H, R4 = OH, y = CH2 in combination with R1 = OCH3, R2 =<br>
OCH3, R6 = H, R6 = H, R7 = H, x = CH2,<r8 keto r10="H&lt;br/">
iii) where R3 = cis-OH, R4 = H, y = bond, in combination with R-, = OCH3,<br>
R2 = OCH3, R5 = H, R6 = H, R7 = H, x = CH2, RB,R8 = keto, R10 = H<br>
iv) where R3 = CHS, R4 = OH, y = bond, in combination with Rn = OCH3,<br>
R2 = OCH3, R6 = H, R6 = H, R7 = H, x = CH2, R8,RB = keto, R10 = H<br>
v) where R3 = H, R» = OH, y = CH2, in combination with R1 = OH, R2 =<br>
OH, R5 = H, Re = H, R7 = H, x= CH2l R8=R9 = H, R10 = H<br>
vi) where R3 = H, R4 = OH, y = CH2, in combination with R^ = OCH3, R2 =<br>
OCH3, R5 = H, Re = H, R7 = H, x = CH2, RB=R9 ~ H, R10 = H<br>
vii) where R3 = H, R4 = OH, y = CH2, in combination with R., = OH, R2 =<br>
OCH3, R5 = H, Re = H, R7 = H, x = CH2l R8=R9 — H, R10 = H<br>
viii) where R3 = H, R4 = OH, y = CH2, in combination with R^ = OH, R2 = H,<br>
R5= H, Re = H, R7 = H, x = CH2l R8=R9= H, R10 — H;<br>
ix) where R3 = H, R4 = OH, y = CH2, in combination with R1 = OH, R2 =<br>
OCH3, R5 = H, R6 = H, R7 = H, x = CH2, R8,R9 = keto, R10 = H<br>
x) where R3 = H, R4 = OH, y = CH2, in combination with R1 = OCH3, R2 =<br>
H, R5 = H, Re = H, R7 = H, x = CH2, RB,RB = keto, R10 = H<br>
xi) where R3 = OCH3, R4 = OH, y = bond, in combination with R1 = OCH3,<br>
R2 = H, R5 = H, R6 = OH, R7 = H, x = bond, RB,R9 = keto, R10 = H<br>
xii) where -CHRe-x-CHRs- is and Rn = H, R13 = H, Ru = H, in<br>
combination with Rn = OCH3, R2 = OCH3, R3 = OCH3, R4 = OH, R7 =<br>
H, RB,R9 = keto, R10 = H<br>
xiii) where Ri = OCH3 in combination with R2 = H, R3 = OCH3, R4= OH, R6<br>
= H, Re = H, R7 = H, R8 = H, R6 = H, R10 = H, x = bond, y = bond<br>
xiv) where Ri = OCH3 in combination with R2 = OCH3, R3 = OCH3, R4=<br>
OH, R5 = H, R6 = H, R7 = H, R8 = H, R9 = H, R10 = H, x = bond, y =<br>
bond<br>
xv) where R1 = OCH3 in combination with R2 = OH, R3 = OCH3, R4 = OH,<br>
Rs = H, R6 = H, R7 = H, R8,R9 = keto, R10 = H, x = bond, y = bond<br><br>
xvi)	where Ri = OCH3 in combination with R2 = H, R3 = OCH3, R4 = OH, Rs<br>
= H, R5 = H, R7 = H, R^Rg = keto, R10 = H, x = bond, y = bond<br>
xvii)	where R1 = OCH3, R2 = H, R5 = OH, R4 = OH, R5 = H, R9 = H<br>
xviii)	where R1 = OCH3, R2 = H, Rg = OCH3&gt; R4 = OH, R8 = H, R9 = H<br>
xix)	where Ri = OCH3) R2 = H, R3 = OH, R» = OH, RB,R9 = keto<br>
xx)	where R1 = OH, R2 = OH, R5 = OCH3, R4 = OH, RB,R9 = keto<br>
xxi)	where R^ = OCH3, R2 = OCH3, R3 = OH, R4 = OH, R8,R9 = keto<br>
xxii)	where R1 = OCH3, R2 = OH, R5 = OCH3, R4 = OH, R„,R9 = keto<br>
xxiii)	where R1 = OCH3, R2 = OCH3, R3 = OCH3, R4 = OH, R5 = H, R8 = H<br>
C. A compound selected from the group consisting of. 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin (pre-rapamycin), 9-deoxo-16-<br>
0-desmethyl-27-O-desmethyl-39-O-desmethyl-rapamycin, 16-O-desmethyl-27-<br>
desmethoxy-39-O-desmethyl-rapamycin, 9-deoxo-16-O-desmethyl-39-O-desmethyl-<br>
rapamycin , 9-deoxo-16-O-desmethyl-27-desmethoxy-raparnycin , 16-O-desmethyl-<br>
27-O-desmethyl-39-O-desmethyl-rapamycin, 9-deoxo-27-O-desmethyl-39-O-<br>
desmethyl-rapamycin, 9-deoxo-16-O-desmethyl-27-O-desmethyl-rapamycin , 27-O-<br>
desmethyl-39-O-desmethyl-rapamycin, 9-deoxo-16-O-desmethyl-rapamycin , 9-<br>
deoxo-39-O-desmethyl-rapamycin , 8-deoxo-15-O-desmethyl-26-desmethoxy-38-O-<br>
desmethyl-prolylrapamycin (pre-prolylrapamycin), 8-deoxo-15-O-desmethyl-26-O-<br>
desmethyl-38-O-desmethyl-prolylrapamycin, 15-O-desmethyl-26-desmethoxy-38-O-<br>
desmethyl-prolylrapamycin, 8-deoxo-26-desmethoxy-38-O-desmethyl-<br>
prolylrapamycin , 8-deoxo-15-O-desmethyl-38-O-desmethyl-prolylrapamycin , 8-<br>
deoxo-15-O-desmethyl-26-desmethoxy-prolylrapamycin, 15-O-desmethyl-26-O-<br>
desmethyl-38-O-desmethyl-prolylrapamycin, 8-deoxo-26-O-desmethyl-38-O-<br>
desmethyl-prolylrapamycin, 8-deoxo-15-O-desmethyl-26-O-desmethyl-<br>
prolylrapamycin , 15-O-desmethyl-38-O-desmethyl-prolylrapamycin , 15-O-<br>
desmethyl-26-O-desmethyl-prolylrapamycin, 15-O-desmethyl-26-desmethoxy-<br>
prolylrapamycin, 26-desmethoxy-38-O-desmethyl-prolylrapamycin, 26-O-desmethyl-<br>
38-O-desmethyl-prolylrapamycin , 8-deoxo-15-O-desmethyl-prolylrapamycin , 8-<br>
deoxo-26-O-desmethyl-prolylrapamycin, 8-deoxo-38-O-desmethyl-prolylrapamycin,<br>
15-O-desmethyl-prolylrapamycin , 38-O-desmethyl-prolylrapamycin , 9-deoxo-16-O-<br>
desmethyl-27-desmetiioxy-39-desmethoxy-rapamycin, 9-deoxo-16-O-desmethyl-27-<br>
O-desmethyl-39-desmethoxy-rapamycin, 16-O-desmethyl-27-desmethoxy-39-<br>
desmethoxy-rapamycin, 9-deoxo-27-desrnethoxy-39-desmethoxy-rapamycin, 9-<br><br>
deoxo-16-O-desmethyl-39-desmethoxy-rapamycin, 16-O-desmethyl-27-O-<br>
desmethyl-39-clesmethoxy-rapamycin, 9-deoxo-27-O-desmethyl-39-desmethoxy-<br>
rapamycin , 16-O-desmethyl-39-desmethoxy-rapamycin , 27-desmethoxy-39-<br>
desmethoxy-rapamycin, 27-O-desmethyl-39-desmethoxy-rapamycin , 9-deoxo-39-<br>
desmethoxy-rapamycin, B-deoxo-15-O-desmethyl-26-desmethoxy-38-desmethoxy-<br>
prolylrapamycin, 8-deoxo-15-O-desmethyl-26-O-desmethyl-38-desmethoxy-<br>
prolylrapamycin, 15-O-desmethyl-26-desmethoxy-38-desmethoxy-prolylrapamycin,<br>
8-deoxo-26-desmethoxy-38-desmethoxy-prolylrapamycin, 8-deoxo-15-O-desmethyl-<br>
38-desmethoxy-proly)rapamycin, 15-O-desmethyl-26-O-desmethyl-38-desmethoxy-<br>
proiylrapamycin , 8-deoxo-26-O-desmethyl-38-desmethoxy-proIylrapamycin , 15-O-<br>
desmethyl-38-desmethoxy-prolylrapamycin, 26-desmethoxy-38-desmethoxy-<br>
prolylrapamycin , 26-O-desmethyl-38-desmethoxy-prolylrapamycin , 8-deoxo-38-<br>
desmethoxy-prolylrapamycin, 38-desmethoxy-prolylrapamycin, 9-deoxo-16-0<br>
desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-<br>
(hydroxycyclohexenyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-<br>
cis-methoxy-4-trans-hydroxycyclohexyl)-36-(dihydroxy cyclohexyl) rapamycin, 9-<br>
deoxo-16-O-desmethyl-27-desm6thoxy-36-de(3-cis-methoxy-4-trans-<br>
hydroxycyclohexyl)-36-(hydroxynorbornyl) rapamycin, 9-deoxo-16-O-desmethyl-27-<br>
desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-methyl-4-<br>
hydroxycyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-<br>
cis-methoxy-4-trans-hydroxycyclohexyl)-36-(4-methylhydroxycyclohexyl) rapamycin,<br>
9-deoxo-16-O-desme&amp;iyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-<br>
hydroxycyclohexyl)-36-{3-fluoro-4-hydroxycyclohexyl) rapamycin, 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-<br>
hydroxy-4-fluorocyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-<br>
de(3^is-methoxy-4-trans-hydroxycyclohexyl)-36-(3-chloro-4-hydroxycyclohexyl)<br>
rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-<br>
hydroxycyclohexyl)-36-(3-hydroxy-4-chlorocycloh6xyl) rapamycin, 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-<br>
cis-4-cis-dihydroxycyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-desmethoxy-<br>
36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-36-(3-trans-4-trans-<br>
dihydroxycyclohexyl) rapamycin, 9-deoxo-16-O-desmethyl-27-O-desmethyl-39-O-<br>
desmethyl rapamycin, 9-deoxo-16-O-desmethyl-270-desmethyl-36-de(3-cis-<br>
methoxy-4-trans-hydroxycyclohexyl)-36-(hydroxycyclohexenyl) rapamycin, 9-deoxo-<br>
16-O-desmethy)-27-O-desmethyl-36-de(3-cis-methoxy-4-trans-hydroxycyclohexyl)-<br><br>
36-(hydroxynorbomyl) rapamycin, 9-deoxo-16-O-desmethyl-27-O-desmethyl-36-<br>
de(3-cis-methoxy-4-trans-hydroxycyciohexyl)-36-(4-methylhydroxycyclohexyl)<br>
rapamycin.<br>
In a specific embodiment the present invention describes methods to produce<br>
and optionally isolate the following compounds (Figure 10, Figure 11, Figure 12,<br>
Figure 13, and Figures 14,15,16 and Figure 17):<br><br><br><br><br><br><br><br><br><br><br><br><br><br>
In a further aspect, the invention provides novel rapamycin analogues of<br><br>
where<br>
x = bond or CHRu, or -CHRs-x-CHRr is<br><br><br>
y =	bond or CHR12<br>
Ri=	OH, OCH3<br>
R2=	H, OH, OCH3<br>
Rr=	H, OH, OCH3, alkyl-, halo-, amino-, thiol- residue<br>
R*=	H, OH, OCH3, alkyh halo-, amino-, thiol- residue<br>
R5=	H, alkyl-, halo-, hydroxy- residue<br>
RB=	H, alkyl-, halo-, hydroxy- residue<br>
R7=	H, alkyl-, halo-, hydroxy- residue<br>
Rs,Rfi= =OorH,H<br>
Rio=	H, alkyl-, halo-, hydroxy- residue<br>
Rn=	H, alkyl-, halo-, hydroxy- residue<br>
R12=	H, alkyl-, halo-, hydroxy-residue<br>
Ri3=	H, alkyl-, halo-, hydroxy- residue<br>
R14=	H, alkyl-, halo-, hydroxy- residue<br>
Additionally, the present invention also provides novel rapamycin analogues of<br><br>
where:<br>
x = bond or CHRu, or -CHRrx-CHRjr is<br><br><br>
R,=	OH, OCH3<br>
R2=	H, OH, OCH3<br>
R6=	H, alkyl-, halo-, hydroxy- residue<br>
Re=	H, alkyl-, halo-, hydroxy- residue<br>
R7=	H, alkyl-, halo-, hydroxy- residue<br>
R8,R9= =OorH,H<br>
R10=	H, alkyl-, halo-, hydroxy- residue<br>
Rn=	H, alkyl-, halo-, hydroxy- residue<br>
R12=	H, alkyl-, halo-, hydroxy-residue<br>
R13=	H, alkyl-, halo-, hydroxy- residue<br>
R14=	H, alkyl-, halo-, hydroxy- residue<br><br>
The novel rapamycin analogues are useful directly, and as templates for<br>
further semi-synthesis or bioconversion to produce compounds useful, as<br>
immunosuppressants, antifungal agents, anticancer agents, neuroregenerative<br>
agents or agents for the treatment of psoriasis, rheumatoid arthritis, fibrosis and other<br>
hyperproliferative diseases.<br>
Therefore in a further aspect, the present invention provides use of the FKBP-<br>
iigand analogues generated in the manufacture of a medicament for the treatment of<br>
cancer, the treatment of fungal infections, the treatment of autoimmune,<br>
inflammatory, proliferative and hyperproliferative diseases or the maintenance of<br>
immunosuppression.<br><br>
One skilled in the art would be able by routine experimentation to determine<br>
the ability of these compounds to inhibit fungal growth {e.g. Baker, H., et al., 1978;<br>
NCCLS Reference method for broth dilution antifungal susceptibility testing for<br>
yeasts: Approved standard M27-A, 17(9). 1997), and for example but without<br>
limitation using the methods described in Example 19. Additionally, one skilled in the<br>
art would be able by routine experimentation to determine the ability of these<br>
compounds to inhibit tumour cell growth, for example but without limitation using the<br>
methods described in Example 19, (also see Dudkin, L, et al., 2001; Yu et at. 2001).<br>
In a further aspect the compounds of this invention are useful for inducing<br>
immunosuppression and therefore relate to methods of therapeutically or<br>
prophylactically inducing a suppression of a human's or an animal's immune system<br>
for the treatment or prevention of rejection of transplanted organs or tissue, the<br>
treatment of autoimmune, inflammatory, proliferative and hyperproliferative diseases<br>
(examples include but are not inclusively limited to autoimmune diseases, diabetes<br>
type I, acute or chronic rejection of an organ or tissue transplant, asthma, tumours or<br>
hyperprolific disorders, psoriasis, eczema, rheumatoid arthritis, fibrosis, allergies and<br>
food related allergies). Such assays are well known to those of skill in the art, for<br>
example but without limitation: Immunosuppressant activity - Warner, LM.,et al.,<br>
1992, Kahan et al. (1991) &amp; Kahan &amp; Camardo, 2001); Allografts - Fishbein, T.M., et<br>
al., 2002, Kirchner et al. 2000; Autoimmune / Inflammatory / Asthma - Carlson, R.P.<br>
et al., 1993, Powell, N. et al., 2001; Diabetes I - Rabinovitch, A. et al., 2002; Psoriasis<br>
- Reitamo, S. et al., 2001; Rheumatoid arthritis - Foey, A., et al., 2002; Fibrosis -<br>
Zhu, J. et al., 1999, Jain, S., et al., 2001, Gregory et al. 1993<br>
The ability of the compounds of this invention to induce immunosuppression<br>
may be demonstrated in standard tests used for this purpose, for example but<br>
without limitation using the methods described in example 19.-In a further aspect the<br>
compounds of this invention are useful in relation to antifibrotic, neuroregenerative<br>
and anti-angiogenic mechanisms, one skilled in the art would be able by routine<br>
experimentation to determine the ability of these compounds to prevent angiogenesis<br>
(e.g. Guba, M.,et al., 2002,). One of skill in the art would be able by routine<br>
experimentation to determine the utility of these compounds in stents (e.g. Morice,<br>
M.C., et al., 2002). Additionally, one of skill in the art would be able by routine<br>
experimentation to determine the neuroregenerative ability of these compounds (e.g.<br>
Myckatyn, T.M., et al., 2002, Steiner et al. 1997)<br><br>
Brief description of the/Figures	' \J	\N v<br>
Figure 1	Structure of rapamycin, the sections to the left of the line represent the<br>
binding domain and those to the right indicate the effector domain.<br>
Figure 2	Structure of rapamycin (A), FK-506 (B), FK-520 (C) and meridamycin<br>
(D)<br>
Figure 3	Plasmid map of pMG55, a double recombination vector with RpsL<br>
positive selection and oriT for conjugation.<br>
Figure 4	A flow chart demonstrating the cloning strategy for the isolation of<br>
pMAG144-16 to create MG2-10.<br>
Figure 5	Overview over the gene cassettes<br>
Figure 6	Structure of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl<br>
rapamycin<br>
Figure 7	Structure of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl<br>
prolylrapamycin<br>
Figure 8	Structure of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-desmethoxy<br>
rapamycin<br>
Figure 9	Structure of 16-O-desmethyl-27-desmethoxy rapamycin<br>
Figure 10 Structures of compounds 1,2,4, 5,6, 8, 9,15,16,17,18 and 19<br>
Figure 11 Structures of compounds 3,7,10,11,12,13,14, 20, 21, 22 and 23<br>
Figure 12 Structures of compounds 24,25, 27, 28,29, 31, 32, 38, 39,40,41 and<br>
42<br>
Figure 13 Structures of compounds 26,30, 33,34,35,36, 37,43, 44, 45, and 46<br>
Figure 14 Structures of compounds 47,48, 50, 51, 53 and 57<br>
Figure 15 Structures of compounds 49, 52, 54, 55, 56, and 58<br>
Figure 16 Structure of compounds 61, 64,66, 67, 68, and 70<br>
Figure 17 Structure of compounds 59, 60,62,63, 65, and 69<br>
Figure 18 Pre-rapamycin heteronuclear multiple bond coherence HMBC<br>
Figure 19 Pre-rapamycin heteronuclear multiple quantum coherence HMQC<br>
Figure 20 Pre-rapamycin correlation spectroscopy (COSY) indicated by solid<br>
arrows, Pre-rapamycin total correlation spectroscopy (TOCSY)<br>
indicated by dotted arrows.<br>
Figure 21 Corrections in the DNA sequence of rapN, the corrected sequence is<br>
shown on top (SEQ ID NO: 1) and the published sequence (ace no:<br>
X86780, nt 91764-92978) is shown underneath (SEQ ID NO: 2).<br><br>
Figure 22 Corrections in the amino acid sequence of RapN, the corrected<br>
sequence is shown on top (SEQ ID NO: 3) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 4).<br>
Figure 23 Corrections in the DNA sequence of rapM, the corrected sequence is<br>
shown on top (SEQ ID NO: 5) and the published sequence (ace no:<br>
X86780, nt 92992-93945 complement) is shown underneath (SEQ ID<br>
NO: 6).<br>
Figure 24 Corrections in the amino acid sequence of RapM, the corrected<br>
sequence is shown on top (SEQ ID NO: 7) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 8).<br>
Figure 25 Corrections "m the DNA sequence of rapL, the corrected sequence is<br>
shown on top (SEQ ID NO: 9), the published sequence (ace no:<br>
X86780, nt 94047-95078 complement) is shown at the bottom (SEQ<br>
ID NO: 10).<br>
Figure 26 Corrections in the amino acid sequence of RapL, the corrected<br>
sequence is shown at the top (SEQ ID NO: 11) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 12)<br>
Figure 27 Corrections in the DNA sequence of rapK, the corrected sequence is<br>
shown at the top (SEQ ID NO: 13) and the published sequence (ace<br>
no: X86780, nt 95430-96434) is shown at the bottom (SEQ ID NO:<br>
14).<br>
Figure 28 Corrections in the amino acid sequence of RapK, the corrected<br>
sequence is shown at the top (SEQ ID NO: 15) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 16).<br>
Figure 29 Corrections in the DNA sequence of rapJ, the corrected sequence is<br>
shown at the top (SEQ ID NO: 17) and the published sequence (ace<br>
no: X86780, nt 96465-97625) is shown at the bottom (SEQ ID NO:<br>
18).<br>
Figure 30 Corrections in the amino acid sequence of RapJ, the corrected<br>
sequence is shown at the top (SEQ ID NO: 19) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 20).<br>
Figure 31 Corrections in the DNA sequence of rapl, the corrected sequence is<br>
shown at the top (SEQ ID NO: 21) and the published sequence (ace<br>
no: X86780, nt 97622-98404) is shown at the bottom (SEQ ID NO:<br>
22).<br><br>
Figure 32  Corrections in the amino acid sequence of Rapl, the corrected<br>
sequence is shown at the top (SEQ ID NO: 23) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 24).<br>
Figure 33  Corrections in the DNA sequence of rapQ, the corrected sequence is<br>
shown at the top (SEQ ID NO: 25) and the published sequence (ace<br>
no: X86780, nt 90798-91433) is shown at the bottom (SEQ ID NO:<br>
26).<br>
Figure 34  Corrections in the amino acid sequence of RapQ, the corrected<br>
sequence is shown at the top (SEQ ID NO: 27) and the published<br>
sequence (ace no: X86780) is shown underneath (SEQ ID NO: 28).<br>
Figure 35  A flow chart demonstrating the cloning strategy for the isolation of<br>
pMG278-1 to create MG3.<br>
Figure 36  A flow chart demonstrating the cloning strategy for the isolation of<br>
pMG267-1 to create MG4.<br>
Materials and Methods<br>
Materials<br>
All molecular biology enzymes and reagents were from commercial sources.<br>
D/L pipecolic acid was obtained from Sigma.<br>
Starter materials<br>
Table IV summarises the sources of the acids used in the feeding<br>
experiments described in the Examples section. For those compounds that were<br>
purchased det alls of the source are given. A brief synthetic method is given for those<br>
starter acids that were synthesised in house. A person of skill in the art will<br>
appreciate that variations on the methods described are routine and are within the<br>
scope of the present invention.<br><br><br><br><br>
WO 2004/007709<br><br>
PCT/GB2003/003230<br><br><br>
Acid	Company	Stock<br>
number	synthesis<br>
3-cis-cyclohexeneoxide carboxylic<br>
acid			in house, Method<br>
A<br>
Mixture of 3,4-cis-<br>
dihydroxycyclohexane carboxylic<br>
acid and 3,4-trans-<br>
dihydroxycyclohexane carboxylic<br>
acid			in house, Method<br>
D<br>
Cyclohexaneacetic acid	Aldrich	C10,450-7	<br>
Cyclohexanepropionic acid	Aldrich	16,147	<br>
4-cis/trans-fe/f-butylcyclohexane<br>
carboxylic acid	Aldrich	37,493-8	<br><br><br>
OH<br>
Synthesis of 3-cis,4-trans-dihydroxycyclohexane carboxylic acid<br>
Q	_  J&gt;	o<br><br>
HQ<br>
KOH<br>
HO°<br>
THF/H20<br><br>
Racemic 3-cis,4-trans-dihydroxycyclohexane carboxylic acid was readily attainable<br>
from commercially available racemic 3-cyclohexene carboxylic acid. This acid was<br>
epoxidised through treatment with /wete-chloroperbenzoic acid and converted to the<br>
lactone in situ by the addition of base (triethylamine), thus setting up the relative<br>
stereochemistries. This lactone was then hydrolysed by the action of aqueous<br>
potassium hydroxide, and the final product purified over ion exchange resin, (see<br>
PAS Lowden Thesis 1997, Corey, E. J. and Huang, H., 1989).<br><br><br>
Method A:<br>
0H Et3N- IBCF<br>
HOCH2CH2SiMe3<br><br>
A 9<br><br>
Epoxides A and B were synthesised by standard steps. Cyclohex-3-ene carboxylic<br>
acid was protected with 2-trimethylsilylethanol following activation with<br>
isobutylchloroformate and triethylamine. The resultant ester was treated with meta-<br>
chloroperbenzoic acid and the resultant racemic mix of diastereomers separated on<br>
normal phase silica. The epoxides were either reacted on (see below) or<br>
deprotected directly by the treatment of trifluoroacetic acid, to liberate the respective<br>
free acids.<br>
86<br><br>
! <br>
A protected epoxide was treated with anhydrous HF-pyridine to effect the ring<br>
opening to produce a pair of racemic regiomers, containing F and OH in a trans<br>
arrangement (as previously demonstrated for cyclohexene oxide). The esters were<br>
then deprotected with trifluoroacetic acid to liberate the free acids, (see Welch, J. T.<br>
and Seper, K., W., 1988)<br><br>
A protected epoxide was treated with concentrated hydrochloric acid suspended<br>
organic solvent to affect the ring opening to produce a pair of racemic regiomers,<br>
containing CI and OH in a trans arrangement (as previously demonstrated for<br>
cyclohexene oxide). The esters were then deprotected with trifluoroacetic acid to<br>
liberate the free acids, (see Chini, M., Crotti, P., et al., 1992)<br><br>
cis-dihydroxylcyclocarboxylic acids were generated by treating protected epoxides<br>
with a catalytic amount of osmium tetraoxide together with a co-oxidant. The esters<br>
were then deprotected with trifluoroacetic acid to liberate the free acids.<br><br>
Bacterial strains and growth conditions<br>
Escherichia coli DH10B (GibcoBRL) was grown in 2xTY medium as described<br>
by Sambrook et al. (1989) and E. coli ET12567(pUB307) as described in MacNeil et al. (1992) and E. coli ET12567(pUZ8002) as described in Paget et al. (1999) in 2xTY<br>
medium with kanamycin (25 ng/ml). The vectors pUC18 and Litmus28 were obtained<br>
from New England Biolabs. Vector pSET152 is described in Bierman et al., (1992a).<br>
E. coli transfbrmants were selected for with 100 jig/ml ampicillin or 50 ng/ml<br>
apramycin.<br>
The rapamycin producer S. hygroscopicus ATCC29253 and its derivatives<br>
were maintained on medium 1 agar plates (see below) at 26°C, and cultivated in<br>
TSBGM (Tryptic Soy Broth with 1.0 % glucose and 100 mM MES, pH 6.0) as<br>
described in (Khaw et al., 1998), supplemented with 100 pg/ml apramycin when<br>
required.<br>
Liquid cultures were grown at 25°C in side-baffled Erlenmeyer flasks with<br>
shaking at 300 rpm.<br>
The streptomycin resistant mutant S. hygroscopicus MG1C was selected<br>
using standard procedures and maintained on medium 1 with streptomycin<br>
(50|ig/ml).<br>
Feeding methods:<br>
Spore stocks of all strains were prepared after growth on medium 1,<br>
preserved in 20% w/v glycerols 0% w/v lactose in distilled water and stored at -80<br>
°C. Vegetative cultures were prepared by inoculating 100^ of frozen stock into 50ml<br>
medium 6 in 250ml flask. The culture was incubated for 36 to 48 hours at 28 °C,<br>
250rpm.<br>
Feeding procedure: Vegetative cultures were inoculated at 0.5 ml into 7ml medium 7<br>
in 50ml tubes. Cultivation was carried out for 7 days, 26°C, 250rpm. The<br>
feeding/addition of the selected carboxylic acids ("non-natural starters" or "natural<br>
starters") were carried out at 24 and 48 hours after inoculation and were fed at 1mM<br>
or 3mM.<br><br><br><br><br><br><br><br><br><br>
Description of Strains<br>
All strains shared the wild type morphology, with cream vegetative mycelia,<br>
white aerial hyphae, developing grey spores turning black and characteristically<br>
hygroscopic.<br>
Preferably spores for use in the generation of the recombinant strains as<br>
described herein were dark grey in colour, as defined in Fan 4, 202 C to B, more<br>
preferably they are as defined in Fan 4, 202 B (Royal Horticultural Society Colour<br>
Chart 2001, available from The Royal Horticultural Society, 80 Vincent Square,<br>
London, SW1P2PE).<br>
DAM manipulation and sequencing<br>
DNA manipulations, PCR and electroporation procedures were carried out as<br>
described in Sambrook et al. (1989). Southern hybridisations were carried out with<br>
probes labelled with digoxigenin using the DIG DNA labelling kit as described by the<br>
manufacturer (Boehringer Mannheim). DNA sequencing was performed as described<br>
previously (Gaisser et al., 2000).<br>
Fermentation of Streptomyces hygroscopicus strains.<br>
Streptomyces hygroscopicus strains were cultured from a frozen spore stock<br>
in cryopreservative (20% glycerol; 10% lactose w/v in distilled water) on Medium 1<br>
(see Materials and Methods) and spores were harvested after 10-20 days growth at<br>
29°C. Alternatively, spores from frozen working stocks were inoculated directly into<br>
pre-culture medium. A primary pre-culture was inoculated with the harvested spores<br>
and cultured in 250 ml Erlenmeyer flasks containing 50 ml Medium 6 (see Materials<br>
and Methods), shaken at 250 rpm with a two-inch throw, at 30°C, for two days. The<br>
primary pre-culture was used to inoculate secondary pre-cultures of Medium 6 (see<br><br>
Materials and Methods), at 10% v/v, which was shaken at 300 rpm with a one-inch<br>
throw, at 28°C, for a further 24h. Secondary pre-cultures were used to inoculate, at<br>
10% v/v, production Medium 8 (see Materials and Methods) containing 0.01% v/v<br>
SAG 417 antifoam and allowed to ferment in a stirred bioreactor for five to seven<br>
days at 26°C. Airflow was set to 0.75 wm, over pressure at 0.5 bar and the impeller<br>
tip speed was controlled between 0.98 ms*1 and 2.67 ms"1. Additional SAG 417 was<br>
added on demand. pH was controlled at 6 - 7 with ammonium (10% v/v) or sulphuric<br>
acid (1 M) and glucose solution (40% w/v) was drip fed on initiation of ammonium<br>
demand.<br>
Extraction and High Performance Liquid Chromatography (HPLC) analysis Method<br>
Centrifugation was carried out on 50 ml of the fermentation broth and the<br>
supernatant and the mycelium were extracted separately as follows. The mycelia<br>
were washed with H20 and extracted with 50 ml of methanol for 16 hours at 4°C. The<br>
cell debris was removed by centrifugation, the methanol evaporated to dryness then<br>
dissolved in 200 yl methanol. The supernatant of the fermentation broth was<br>
extracted twice with an equal volume of ethyl acetate. The organic layer was dried<br>
over Na2S04, evaporated to dryness and then dissolved in 200 µl methanol. HPLC<br>
analysis was performed on a Hewlett Packard HP1100 liquid chromatograph with<br>
variable wavelength detector or a Finnigan MAT LCQ (Finnigan, CA) instrument.<br>
High-resolution spectra were obtained on a Bruker BioApex IIA J T Fourier<br>
Transform-Ion Cyclotron Resonance (FT-ICR) mass spectrometer (Bruker, Bremen,<br>
FRG).<br>
For NMR analysis, the bacterial broth was centrifuged, the supernatant<br>
extracted with three equal volumes of ethylacetate and the myWia extracted with<br>
methanol as described above. The extracts were combined, dried (Na2S04) and<br>
evaporated under reduced pressure to yleld a white solid.<br>
Proton detected NMR spectra (1H, DQF-COSY, TOCSY, HMQC, HMBC,<br>
NOESY) were recorded on a Bruker Advance DRX500 spectrometer which operated<br>
at 500 MHz at 27 °C, with the exception of example 6, where the Bruker Advance<br>
DRX500 spectrometer was operated at 500 MHz at 10°C. Chemical shifts are<br>
described in parts per million (ppm) on the 8 scale and are referenced to CHCI3 at 5H<br>
7.26 (1H) and CHCI3 at 8C 77.0 (13C). J values are given in Hertz (Hz).<br><br>
Extraction, isolation and analysis protocols (B).<br>
Extraction and purification protocol:<br>
The fermentation broth was clarified by centrifugation to provide supernatant and<br>
cells. The supernatant was applied to a column (16x15 cm) of Diaion* HP20 resin<br>
(Supelco), washed with water followed by 75% MeOH/H20 and then eluted with<br>
MeOH. The cells were mixed to homogeneity with an equal volume of acetone. After<br>
at least 30 minutes the acetone slurry was clarified by centrifugation and the<br>
supernatant decanted. The pelleted cells were similarly extracted twice more with<br>
acetone. The acetone extract was combined with the MeOH from the HP20 column<br>
and the solvent was removed in vacuo to give an aqueous concentrate. The aqueous<br>
(typically 1-2 L) was extracted with EtOAc (3x1-2 L) and the solvent removed in<br>
vacuo to give an oily crude extract (typically 20 g). The oily residue was dissolved in<br>
a minimal volume of EtOAc and dried onto silica. The coated silica was applied to a<br>
silica column (400g, 36 x 6 cm) that was eluted sequentially with acetone/hexane<br>
mixtures ranging from 25% acetone initially to 100% acetone. The fractions<br>
containing rapamycin analogues were identified by HPLC (280 nm) using conditions<br>
described within.<br>
The rapamycin analogue-containing fractions were combined and the solvent was<br>
removed in vacuo. The residue was further chromatographed over Sephadex LH20,<br>
eluting with 10:10:1 chloroform/heptane/ethanol. The semipurified rapamycin<br>
analogues were purified by reverse phase (C18) high performance liquid<br>
chromatography using a Gilson HPLC, eluting a Phenomenex 21.2 x 250 mm Luna 5<br>
urn C18 BDS column at 21 mL/min, isocratic elution with 50% to 70% CH3CN/H20<br>
mixtures depending on the polarity of the rapamycin analogue.<br>
Analysis of culture broths	'.<br>
An aliquot of whole broth (1 mL) was shaken with CH3CN (1 mL) for 30 minutes. The<br>
mixture was clarified by centrifugation and the supernatant analysed by HPLC with<br>
diode array detection. The HPLC system comprised an Agilent HP1100 equipped<br>
with a BDS HYPERSIL C18 3 urn 4.6 x 150 mm column CThermoHypersil-Keystone)<br>
heated to 40°C. The gradient elution was from 55% mobile phase B to 95% mobile<br>
phase B over 10 minutes followed by an isocratic hold at 95% mobile phase B for 2<br>
minutes with a flow rate of 1 mL/min. Mobile phase A was 10% acetonitrile:90%<br>
water, containing 10 mM ammonium acetate and 0.1% trifluoroacetic acid, mobile<br>
phase B was 90% acetonitrile:10% water, containing 10 mM ammonium acetate and<br><br>
0.1% trifluoroacetic acid. Rapamycin analogues were identified by the presence of<br>
the characteristic rapamycin triene, centred on 278 nm. FK506 and FK520<br>
analogues are identified by LC-MS analysis.<br>
Analysis by LCMS<br>
The HPLC system described above was coupled to a Bruker Daltonics Esquire3000<br>
electrospray mass spectrometer. The same column and gradient elution scheme<br>
were used as described above. Mobile phase A was water, mobile phase B was<br>
acetonitrile. Positive negative switching was used over a scan range of 500 to 1000<br>
Dalton.<br>
Example 1<br>
Conjugation ofS. hygroscopicus<br>
The plasmid to be conjugated into S. hygroscopicus was transformed by<br>
electroporation into the dam' dcm' ET12567 £ coli strain containing either pUB307<br>
as described in MacNeil et al. (1992) or pUZP002 as described in Paget et at. (1999).<br>
A preculture was used (over night culture, 30°C) to inoculate fresh 2xTY (with 50<br>
fj.g/ml apramycin and 25 ug/ml kanamycin) at a dilution of 1/25 and grown with<br>
shaking at 37°C to an optical density at 595 nm of 0.25-0.6. The cells from this broth<br>
were washed twice with 2xTY, then resuspended with 0.5 ml of 2xTY per 25 ml<br>
original culture. The quality of the spore stock used is critical for the success of this<br>
method. In this context the age of the spores when harvested and the use of medium<br>
1 are crucial for the isolation of high-quality spore suspension. To isolate high- quality<br>
spore suspensions of S. hygroscopicus, pre-dried plates of medium 1 agar (see<br>
Materials and Methods section) were spread with S. hygroscopicus spores or mycelia<br>
using standard microbiological techniques followed by incubation at 26°-28°C for 14-<br>
21 days. Spores were harvested by addition of 1-2 ml of sterile 20 % w/v glycerol or<br>
water by standard, techniques. An aliquot of 200 pi of the S. hygroscopicus spore<br>
suspension was washed in 500 |xl of 2xTY, resuspended in 500 µl of 2xTY, subjected<br>
to heat shock at 50°C for 10 minutes then cooled on ice. An aliquot of 0.5 ml of the £<br>
coli suspension was mixed with the heat-shocked spores and this mixture plated on<br>
medium 1 agar plates. These plates were incubated at 26°-280C for 16 hours before<br>
overlaylng with 1 mg of nalidixic acid and 1 mg of apramycin per plate. Exconjugant<br>
colonies usually appeared after 3-7 days.<br><br>
Use in S.hygroscopicus MG2-10 of an alternative integrating vector, pRT801<br>
Conjugation was also carried out using the 4&gt;BT1-based integrating vector<br>
pRT801 into S.hygroscopicus MG2-10 as described above. Exconjugants were<br>
patched on to medium .1 containing 50ug/ml apramycin and 50ug/ml nalidixic acid,<br>
and shown to be apramycin resistant<br>
Example 2<br>
Isolation of the S. hygroscopicus mutant MG2-10 carrylng the chromosomal deletion<br>
of rapQONMLKJI (Figure 4).<br>
An S. hygroscopicus mutant (MG2-10) in which the rapamycin modifylng<br>
genes rapQ, rapO/N, rapM, rapL, rapK, rapJ and rap! were deleted was constructed<br>
as described below.<br>
Isolation of the streptomycin resistant mutant MG1C:<br>
S.hygroscopicus NRRL5491 mycelia were spread onto plates of medium 1<br>
containing 50mg/ml streptomycin. Three colonies were isolated and labelled MG1A,<br>
MG1B and MG1C. These were conjugated as in example 1 with the plasmid pMG49,<br>
a derivative of pSET152 containing the rpsL gene from S.lividans TK24.<br>
Exconjugants from each of these conjugations were patched onto a plate if medium 1<br>
containing 50mg/ml apramycin and 50mg/ml nalidixic acid, to confirm the presence of<br>
the plasmid pMG49. They were then streaked, along with the original strains MG1A,<br>
MG1B and MG1C, onto a both a plate of medium 1 containing no antibiotic and a<br>
plate of.medium 1 containing 50mg/ml streptomycin. Growth was seen in all cases<br>
except fhe streaks of MG1A [pMG49], MG1B [pMG49] and MG1C [pMG49] on<br>
streptomycin, indicating that the w.t. rpsL gene from S.lividans TK24 conferred<br>
dominant streptomycin sensitivity on these strains. The production of pre-rapamycin<br>
was measured in MG1A, MG1B and MG1C and the best producer, MG1C, was kept<br>
for further work.<br>
Conjugation ofS. hygroscopicus MG1C<br>
Conjugations were carried out as described in example 1 using the<br>
streptomycin resistant S. hygroscopicus MG1C and vector pMG55 derived<br><br>
constructs.<br>
Construction of conjugative double recombination vector pMG55 (Figure 3)<br>
The primers MAG47 5'-GCAAGCTTGGTACCGACACGCTCGCCGAACAGG-<br>
3' (SEQ ID NO: 29) and MAG48 5'-GCGCATGCCCTAGGGTGTACATTACTTCTCC-<br>
3' (SEQ ID NO: 30) were used to amplify the S.lividans rpsL gene using the plasmid<br>
pRPSL21 (Shima et al., 1996) as a template. The PCR fragment was digested with<br>
Sp/jl and H/ndlll, isolated and ligated with the 3.2 kb fragment of pSET152 (Bierman<br>
et al., 1992b), which had been digested with Sph\ and H/ndlll. After transformation<br>
into £ coli DH1 OB, plasmid pMG55 was isolated. This plasmid was confirmed by<br>
sequencing. Plasmid pMG55 contains the rpsL gene to allow selection for double<br>
recombinants (Hosted and Baltz, 1997).<br>
Isolation of the S. hygroscopicus mutant MG2-10 carrylng the chromosomal deletion<br>
of rapQONMLKJI (Figure 4)<br>
The primers MAG23 5'-TATCTAGACTTCGCACGTGCCTGGGACA-3' (SEQ<br>
ID NO: 31) and MAG24 5^AGMGCTTACCCMTTCCAACATCACCT-3' (SEQ ID<br>
NO: 32) were used to amplify the left region of homology (from nt 89298 to nt 90798<br>
in the rapamycin cluster as described in Schwecke et al.(Schwecke et al., 1995)<br>
using genomic DNA prepared from S. hygroscopicus NRRL5491 as a template. The<br>
1.5 kb PCR product was digested with Xba\ and H/ndlll and ligated into pUC18 cut<br>
with Xbal and H/ndlll. After transformation into E. coli DM OB, the plasmid pMAG127-<br>
8 was isolated. The primers MAG25 5'-GGAAGCTTTGACCACACGCCGCCCGTTC-<br>
3* (SEQ ID NO: 33) and MAG26 5*-ATGCATGCCCGCCGCAACCCGCTGGCCT-3'<br>
(SEQ ID NO: 34) were used to amplify the right region of homology (from nt 98404 to<br>
nt 99904 in the rapamycin cluster as described in Schwecke et al. (1995)) using<br>
genomic DNA prepared from S. hygroscopicus NRRL5491 as a template. The 1.5 kb<br>
product of PCR was digested with H/ndlll and Sph\ and ligated into pUC18 cut with<br>
H/ndlll and Sphl After transformation into £ co//DH10B, the plasmid pMAG128-2<br>
was isolated (Figure 4). Both plasmids were checked by sequence analysis. The<br>
plasmid pMAG127-8 was digested with Sph\ and H/ndlll, the plasmid pMAG128-2<br>
was digested with Xoal and H/ndlll and the 1.5 kb fragments were isolated from both<br>
plasmids. These fragments were ligated into pUC18 cut with Sph\ and Xbal and used<br>
to transform £ coli DH10B. The plasmid pMAG131-1 was isolated. This plasmid was<br>
digested with Sph\ and Xba\, the 3 kb fragment was isolated and ligated into pMG55<br><br>
cut with Sph\ and Avr\\ and the DNA was used to transform £ coli DH10B. The<br>
plasmid pMAG144-16 was isolated and used to conjugate S. hygroscopicus MG1C.<br>
An apramycin resistant S. hygroscopicus colony was isolated, grown for 24 hours in<br>
TSBGM with shaking at 26°C, and spread onto medium 1 agar plates containing 50<br>
ng/l streptomycin. Streptomycin resistant colonies were isolated and shown to be<br>
apramycin sensitive. The 7606 nt chromosomal deletion of the rapQONMLKJI region<br>
of the rapamycin cluster was verified in the mutant MG2-10 by using the 1.5 kb PCR<br>
product of MAG23 and MAG24 to probe EcoRI- and SamHI-digested chromosomal<br>
DNA. Analysis of the wild type S. hygroscopicus showed the expected 5.8 kb EcoRI<br>
and 5.9 kb BamHI band after hybridisation. When chromosomal DNA of MG2-10 was<br>
treated similarly, 9.6 kb EcoRI and 7.6 kb BamH\ bands were detected, indicating<br>
that rapQONMLKJI had been removed.<br>
Example 3<br>
Expression of rapK in the S. hygroscopicus mutant MG2-10 carrylng the<br>
chromosomal deletion of rapQONMLKJI (Figure 4)<br>
Construction of expression vector pSGsetl<br>
The pSET152 (Bierman et al., 1992a) derived vector pCJR336 (kindly<br>
provided by Christine Martin and Corinne Squire) was created by cloning the primer<br>
dimerofCR347 5'-<br>
TAAACTAGTCCATCTGAGAGTTTCATATGGCCCTATTCTGCCCAGCCGCTCTAG<br>
AAAT-3' (SEQ ID NO: 35) and CR348 5'-<br>
ATTTCTAGAGCGGCTGGGCAGAATAGGGCCATATGAAACTCTCAGATGGACTAG<br>
TTTA -3' (SEQ ID NO: 36) into PvuW digested pSET152 using standard molecular<br>
biological techniques, thus introducing sites for the restriction enzymes Spel, aldel,<br>
and Xbal into pSET152. The orientation of the insert was confirmed by sequencing.<br>
Plasmid pCJR336 was digested using the restriction enzymes Wdel/Spel and vector<br>
pSG142 (Gaisser et ah, 2000) was digested identically. The resulting DNA bands of<br>
about 5.4 kb for pCJR336 and 1.2 kb for pSG142 were isolated followed by a ligation<br>
which was used to transform E. coli DH10B. The vector construct containing the<br>
acf//-ORF4 regulator region was isolated and digested using the restriction enzyme<br>
Xba\ followed by an alkaline phosphatase treatment according to standard protocols.<br>
The isolated DNA was ligated with a fragment of about 200 bp from plasmid<br>
pEXo/eG2cas (pSG142 derivative containing the ca. 1.2kb Ndel/BglW fragment of<br><br>
pSGcasOleG2 (WO01/79520) digested with the restriction enzymes Xba\ and Nhe\.<br>
Vector pSGsetl was isolated and the correct orientation of the insert was verified<br>
using restriction digests and sequence analysis. Plasmid pSGsetl contains the actll-<br>
ORF4 regulator, the Pacu promoter and the 6xHis-tag coding sequence as well as the<br>
lambda t0 transcriptional termination region (originating from plasmid pQE-16) and it<br>
can integrate site-specifically at the OC31 attachment site.<br>
Cloning of rapK<br>
The gene rapK was amplified by PCR using the primers BIOSG8 5'-<br>
GGGCATATGAGGCAATTGACTCCGCCGGTCACGGCACCGTACTGCC -3' (SEQ<br>
ID NO: 37) and BIOSG9 5'-<br>
GGGGTCTAGAGGTCACGCCACCACACCCTCGATCTCGACC -3' (SEQ ID NO: 38),<br>
which introduce a Afcfel site at the 5' end and a Xba\ site at the 3' end of rapK.<br>
Plasmid pR19 (Schwecke et al., 1995) was used as a template. After treatment with<br>
T4 polynucleotide kinase using standard techniques the PCR product was ligated<br>
with Smal-cut pUC18 and used to transform E. co//DH10B. The DNA sequence of<br>
rapK in the isolated plasmid pUCrapK was verified by sequence analysis. The .<br>
differences in the DNA sequence compared to the published sequence (ace. no.<br>
X86780) are shown in Figure 27. The resulting changes in RapK are shown in Figure<br>
28.<br>
Isolation ofpSGsetrapK<br>
Plasmid pUCrapK was digested with aldel and Xba\ and the insert fragments<br>
were isolated and ligated into identically digested pSGsetl. The ligation was used to<br>
transform £ coli DH10B using standard procedures and the transformants were<br>
analysed. Plasmid pSGsetrapK, was isolated and the construct was verified using<br>
restrictiSn digests and sequence analysis.<br>
Example 4 ...<br>
Identification of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmBthyl-rapamycin<br>
(pre-rapamycin, Figure 6)	.<br>
9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin (pre-<br>
rapamycin) was obtained by conjugating the S. hygroscopicus strain MG2-10 as<br>
described in Example 1 with pSGsetrapK and isolating the products produced on<br><br>
fermentation. This demonstrates that it is possible to complement the deletion of<br>
rapK in the MG2-10 strain and that if the strain is fed with pipecolic acid, pre-<br>
rapamycin is produced, an analogue which is lacking the post-PKS modifications.<br>
The plasmid pSGsetrapK was conjugated into S. hygrvscopicus MG2-10 and<br>
the strain grown in TSBGM fed with 2mg/l pipecolic acid at 25°C with shaking. The<br>
mycelia were extracted with methanol and the culture broth was extracted with ethyl<br>
acetate as described previously.<br>
Analysis of the culture broth of the pipecolic acid-fed S. hygrvscopicus mutant<br>
MG2-10[pSGsetrapK] by HPLC with UV detection at 280nm revealed the presence of<br>
two major new peaks with retention times of 4.0 and 5.1 minutes. Electrospray mass<br>
spectroscopy of these peaks revealed that both contained ions corresponding to a<br>
compound with a MW of 841.5. Neither of these peaks was seen in the culture<br>
extractions of the S. hygrvscopicus NRRL 5491 strain or the mutant strain MG2-10<br>
without the rapK expression plasmid pSGsetrapK. MS/MS analysis of the ion with<br>
m/z of 864 (corresponding to the sodium adduct of pre-rapamycin) revealed that it<br>
fragmented into an ion with m/z of 735 corresponding to the loss of m/z 129<br>
(pipecolic acid), or an ion with m/z of 556 corresponding to the loss of m/z 308 (C28-<br>
C42 of pre- rapamycin). This ion itself fragmented further to an ion with m/z 306,<br>
corresponding to the loss of m/z 250 (C14 to C27 of pre- rapamycin). This<br>
fragmentation pattern was identical to the pattern seen for rapamycin but with the<br>
second loss of m/z (-308) reduced by 14, corresponding to the absence of the C39<br>
O-methyl group, the third loss of m/z (-250) reduced by 44, corresponding to the<br>
absence of the C27 methbxy and C16 O-methyl groups and the final ion (306) having<br>
a mass reduced by 14 corresponding to the absence of the C9 ketone group. This<br>
was evidence that the compound with MW 841.5 represents 9tdeoxo-16-O-<br>
desmetriyl-27-desmethoxy-39-O-desmethyl-rapamycin (pre- rapamycin).<br>
Example 5.<br>
Preparation of gene cassettes for expression in S. hygrvscopicus MG2-10<br>
Gene cassettes able to direct the expression of a variety of rapamycin<br>
modifylng genes and combinations of modifylng genes were constructed as<br>
described below.<br><br>
Cloning of rapN/O<br>
The contiguous genes rapN and rapO, hereafter designated rapN/O were<br>
amplified by PCR using the primers BIOSG2 5'-<br>
GGGCATATGTCGACGACCGATCAGGGTGAGACCGGAAAGGCCTG -3' (SEQ ID<br>
NO: 39) and.BIOSG3 5'- .<br>
GGGGTCTAGAGGTCAGTCCTGGGGTTCGAGAAGCTCGCCGGTCTCCTT-3'<br>
(SEQ ID NO: 40), which introduce a aldel site at the 5' end and a Xba\ site at the 3'<br>
end of rapN/O. Plasmid pR19 (Schwecke et al., 1995) was used as a template. After<br>
treatment with T4 polynucleotide kinase using standard techniques the PCR product<br>
was ligated into Smal-cut pUC18 and used to transform £ coli DH10B. The DNA<br>
sequence of rapWO in the isolated plasmid pUCrapN/O was verified by sequence<br>
analysis. The differences in the DNA sequence compared to the published sequence<br>
(ace. no. X86780) are shown in Fig 21. The resulting changes in RapN are shown in<br>
Fig 22.<br>
Cloning ofrapM<br>
The gene rapMwas amplified by PCR using the primers BIOSG4 5'-<br>
GGGCATATGATCCAACCCGACGTCGTGACCGCCTTCACAGCGG -3' (SEQ ID<br>
NO:41)andBIOSG5 5'-<br>
GGGGTCTAGAGGTCACACGCGGACGGCGATCTGGTGCCGATAGG -3* (SEQ ID<br>
NO: 42), which introduce a aldel site at the 5" end and a Xba\ site at the 3' end of<br>
rapM. Plasmid pR19 (Schwecke et ai, 1995) was used as a template. After treatment<br>
with T4 polynucleotide kinase using standard techniques the PCR product was<br>
ligated into Smal-cut pUC18 and used to transform £ coli DH10B. The DNA<br>
sequence of rapM in the isolated plasmid pUCrapM was verified by sequence<br>
analysis. The differences in the DNA sequence compared to the published sequence<br>
(ace. no. X86780) are shown in Fig 23. The resulting changes in RapM are shown in<br>
Fig 24.<br>
Cloning of rapL<br>
The gene rapL was amplified by PCR using the primers BIOSG6 5'-<br>
GGGCATATGCAGACCAAGGTTCTGTGCCAGCGTGACATCAAG -3' (SEQ ID NO:<br>
43) and BIOSG7 5'-<br>
GGGGTCTAGAGGTCACTACAGCGAGTACGGATCGAGGACGTCCTCGGGCG-3'<br><br>
(SEQ ID NO: 44), which introduce a aldel site at the 5' end and a Xba\ site at the 3'<br>
end of rapL Plasmid pR19 (Schwecke et al„ 1995) was used as a template. After<br>
treatment with T4 polynucleotide kinase using standard techniques the PCR product<br>
was ligated into Smal-cut pUC18 and used to transform E coli DH10B. The DNA<br>
sequence of rapL in the isolated plasmid pUCrapL was verified by sequence<br>
analysis. The differences in the DNA sequence compared to the published sequence<br>
(ace. no. X86780) are shown in Fig 25. The resulting changes in RapL are shown in<br>
Fig 26.<br>
Cloning of raphes<br>
The gene rapL was amplified by PCR using the primers BIOSG6 5'-<br>
GGGCATATGCAGACCAAGGTTCTGTGCCAGCGTGACATCAAG -3' (SEQ ID NO:<br>
43) and BIOSG45 5'-<br>
GGAGATCTCAGCGAGTACGGATCGAGGACGTCCTCGGGCG -3' (SEQ ID NO:<br>
45), which introduce a aldel site at the 5' end and a BgR\ site at the 3' end of rapL<br>
Plasmid pR19 (Schwecke et al., 1995) was used as a template. After treatment with<br>
T4 polynucleotide kinase using standard techniques the PCR product was ligated<br>
into Smal-cut pUC19 and used to transform E coli DH10B. The DNA sequence of<br>
rapL in the isolated plasmid pUC19rapLhfe was verified by sequence analysis.<br>
Cloning of rapK<br>
The gene rapKwas amplified by PCR using the primers BIOSG8 5"-<br>
GGGCATATGAGGCAATTGACTCCGCCGGTCACGGCACCGTACTGCC -3" (SEQ<br>
ID NO: 37) and BIOSG9 5'-<br>
GGGGTCTAGAGGTCACGCCACCACACCCTCGATCTCGACC -3' (SEQ ID NO: 38),<br>
which introduce a Afdel site at the 5' end and a Xbal site at thet3' end of rapK.<br>
PlasmidPpR19 (Schwecke et al., 1995) was used as a template. After treatment with<br>
T4 polynucleotide kinase using standard techniques the PCR product was ligated<br>
with Smal-cut pUC18 and used to transform £ coli DH10B. The DNA sequence of<br>
rapK in the isolated plasmid pUCrapK was verified by sequence analysis. The<br>
differences in the DNA sequence compared to the published sequence (ace. no.<br>
X86780) are shown in Fig 27. The resulting changes in RapK are shown Fig 28.<br>
Isolation of pSGsetrpaN/O, pSGsetrapJ, pSGsetrapM, pSGsetrapQ, pSGsetrapl,<br>
pSGsetrapK, and pSGsetrapL<br><br>
Plasmids pUCrapN/O, pUCrapJ, pUCrapM, pUCrapl, pUCrapL, pUCrapK and<br>
pAHL42 were digested w'rth Nde\ and Xbal and the insert fragments, ranging in size<br>
from about 1.3 kb to 0.7 kb, were isolated and ligated into identically digested<br>
pSGsetl The ligations were used to transform E coli DH10B using standard<br>
procedures and the transformants were analysed. Plasmids pSGsetrapN/O,<br>
i<br>
pSGsetrapJ, pSGsetrapM, pSGsetrapQ, pSGsetrapl, pSGsetrapK, and pSGsetrapL<br>
were isolated and the constructs were verified using restriction digests and sequence<br>
analysis.<br>
Cloning of rap J<br>
The gene rapJ was amplified by PCR using the primers BIOSG10 5'-<br>
GGGCATATGAGCACCGMGCTCAGCAAGAGAGCACGCCCACCGCACGCT-3'<br>
(SEQ ID NO: 46) and B10SG11 5'-<br>
GGGGTCTAGAGGTCACTCCGCTCCCCAGGTGACCCGGAGCTCGGC -3' (SEQ ID<br>
NO: 47), which introduce a Wdel site at the 5' end and a Xbal site at the 3' end of<br>
rapJ. Plasmid pR19 (Schwecke et al., 1995) was used as a template. After treatment<br>
with T4 polynucleotide kinase using standard techniques the PCR product was<br>
ligated with Smal-cut pUC18 and used to transform E coli DH10B. The DNA<br>
sequence of rapJ in the isolated plasmid pUCrapJ was verified by sequence analysis.<br>
The differences in the DNA sequence compared to the published sequence (ace. no.<br>
X86780) are shown in Fig 29. The resulting changes in RapJ are shown in Fig 30.<br>
Cioning of rapl<br>
The gene rapl was amplified by PCR using the primers B10SG12 5'-<br>
GGGCATATGAGCGCGTCCGTGCAGACCATCAAGCTGCC -3' (SEQ ID NO: 48)<br>
and BIQSG13 5'-GGGGTCTAGAGGTCAGGCGTCCCCGCGGCGGGCGACGACCT.<br>
-3' (SEQ ID NO: 49), which introduce a aldel site at the 5' end and a Xbal site at the<br>
3' end of rapl. Plasmid pAHL2 (kindly provided by Huai-Lo Lee) is derived from.<br>
pUC18 containing the rapl gene and was used as a template. After treatment with T4<br>
polynucleotide kinase using standard techniques the PCR product was ligated with<br>
Smal-cut pUC18 and used to transform E coli DH10B. The DNA sequence of rapl in<br>
the isolated plasmid pUCrapl was verified by sequence analysis. The differences in<br>
the DNA sequence compared to the published sequence (ace. no. X86780) are<br>
shown in Fig 31. The resulting changes in Rapl are shown in Fig 32.<br><br>
Cloning ofrapQ<br>
The gene rapQ was amplified by PCR using the primers AHL21 5'-<br>
CATATGTTGGAATTGGGTACCCGCCTG -3' (SEQ ID NO: 50) and AHL22 5'-<br>
TCTAGACGCTCACGCCTCCAGGGTG -3' (SEQ ID NO: 51), which introduce a aldel<br>
site at the 5' end and a Xbal site at the 3' end of rapQ. Plasmid pR19 (Schwecke et al., 1995) was used as a template. After treatment with T4 polynucleotide kinase<br>
using standard techniques the PCR product was ligated with Smal-cut pUCl 8 and<br>
used to transform E. coli DH10B. The DNA sequence of rapQ in the isolated plasmid<br>
pAHL42 was verified by sequence analysis. The differences in the DNA sequence<br>
compared to the published sequence (ace. no. X86780) are shown in Fig 33. The<br>
resulting changes in RapQ are shown in Fig 34.<br>
Isolation ofpUCIBeryBVcas<br>
The gene eryBV was amplified by PCR using the primers casOleG21<br>
(WO01/79520) and 7966 5'-<br>
GGGGAATTCAGATCTGGTCTAGAGGTCAGCCGGCGTGGCGGCGCGTG<br>
AGTTCCTCCAGTCGCGGGACGATCT -3' (SEQ ID NO: 52) and pSG142 (Gaisser et al., 2000) as template. The PCR fragment was cloned using standard procedures<br>
and plasmid pUC18eryBVcas was isolated with an aldel site overlapping the start<br>
codon of eryBV and an Xba\ and Bglll site following the stop codon. The construct<br>
was verified by sequence analysis.<br>
Isolation of vector pSGLitl<br>
The gene eryBV was amplified by PCR using the primers BIOSG1 5'-<br>
GGGTCTAGATCCGGACGAACGCATCGATTAATTAAGGAGGACACATA -3' (SEQ<br>
ID NO: 53) and 7966 5'-<br>
GGGGAATTCAGATCTGGTCTAGAGGTCAGCCGGCGTGGCGGCGCGTGAGTTC<br>
CTCCAGTCGCGGGACGATCT -3' (SEQ ID NO: 52), which introduce a Xba) site<br>
sensitive to Dam methylation at the 5' end and a Xbal site and a Bgl\\ site at 3' end of<br>
eryBV. Plasmid pUC18eryBVcas was used as a template. After treatment with T4<br>
polynucleotide kinase using standard techniques the PCR product was ligated with<br>
Smal-cut pUC18 and used to transform E. coli DH10B. The construct was then<br>
digested using BamHI/Bgfll and an about 1.3 kb DNA band was isolated from an<br>
agarose gel followed by the ligation with BamHl/Bg/ll digested Litmus 28 vector DNA<br><br>
using standard procedures. The vector pSGLitl was isolated and the DNA sequence<br>
of the insert was verified by-sequence analysis.<br>
Isolation of pSGsetrpaN/O, pSGsetrapJ, pSGsetrapM, pSGsetrapQ, pSGsetrapl,<br>
pSGsetrapK, and pSGsetrapL<br>
Plasmids pUCrapN/O, pUCrapJ, pUCrapM, pUCrapl, pUCrapL, pUCrapK and<br>
pAHL42 were digested with Nde\ and Xba\ and the insert fragments ranging in size<br>
from about 1.3 kb to 0.7 kb were isolated and ligated into identically digested<br>
pSGsetl. The ligations were used to transform E coli DH10B using standard<br>
procedures and the transformants were analysed. Plasmids pSGsetrapN/O,<br>
pSGsetrapJ, pSGsetrapM, pSGsetrapQ, pSGsetrapl, pSGsetrapK, and pSGsetrapL<br>
were isolated and the constructs were verified using restriction digests and sequence<br>
analysis.<br>
Isolation of pSGLitrapN/O, pSGLitrapJ, pSGLitrapM, pSGLitrapQ, pSGLifrapt,<br>
pSGLitrapK, pSGLitrapL and pSGUtrapL^s<br>
Plasmids pSGsetrpaN/O, pSGsetrapJ, pSGsetrapM, pSGsetrapQ,<br>
pSGsetrapl, pSGsetrapK, pSGsetrapL, and pUCIQrapLhfe were digested using<br>
NdeMBgR] restriction enzymes and the bands ranging from about 0.7 to 1.3 kb were<br>
isolated followed by ligations with pSGLitl digested with Nde\/Bglll The ligations<br>
were used to transform E coli ET12567 and the transformants were analysed.<br>
Plasmids pSGLitrapN/O, pSGLitrapJ, pSGLitrapM, pSGLitrapQ, pSGLitrapl,<br>
pSGLitrapK, pSGLitrapL and pSGLitrapUis were isolated.<br>
Isolation of plasmids pSGsetapKI, pSGsetrapKM, pSGsetrapKN/O, pSGsetrapKL,<br>
pSGsetrapKQ and pSGrapKJ	*<br>
The plasmids pSGLitrapN/O, pSGLitrapJ, pSGLitrapM, pSGLitrapQ,<br>
pSGLitrapl, and pSGLitrapL were digested using Xba\ and the fragments ranging<br>
from about 0.8 to 1.3 kb were isolated followed by ligations with pSGsetrapK .<br>
digested with Xba\ and treated with alkaline phosphatase using standard molecular<br>
biological techniques. The ligations were used to transform E. coli DH10B and the<br>
transformants were analysed. Plasmids pSGsetrapKI, pSGsetrapKM,<br>
pSGsetrapKN/O, pSGsetrapKL, pSGsetrapKQ and pSGrapKJ were isolated and the<br>
orientation of the insert was verified by restriction digest analysis. For the addition of<br>
rapLtfe these constructs were either digested with Bglll/Xbal followed by partial digest<br><br>
with Bglll as appropriate and the isolated vector fragments were ligated with the ~<br>
1kb Xbal/Bglll fragment of pSGLitrapLfe.<br>
Isolation ofplasmids pSGsetrapKIJ, pSGsetrapKIM and pSGsetrapKIQ<br>
The plasmids pSGLitrapJ, pSGLitrapWI, and pSGLitrapQ were digested using<br>
Xbal and the fragments ranging from about 0.8 to 1.3 were isolated followed by<br>
ligations with pSGsetrapKI digested with Xbal and treated with alkaline phosphatase<br>
using standard molecular biological techniques. The ligations were used to transform<br>
£ co// DM OB and the transformants were analysed. Plasmids pSGsetrapKIJ,<br>
pSGsetrapKIM, and pSGrapKIQ were isolated and the orientation of the insert was<br>
verified by restriction digest analysis. For the addition of rapUis these constructs were<br>
either digested with Bglll/Xbal followed by partial digest with Bglll as appropriate and<br>
the isolated vector fragments were ligated with the ~ 1kb Xbal/Bglll fragment of<br>
pSGLitrapUis.<br>
Isolation of plasmids pSGsetrapKN/OI, pSGsetrapKN/OQ, pSGsetrapKN/OM and<br>
pSGsetrapKN/OJ. ......<br>
The plasmids pSGLitrapl, pSGLitrapM, pSGLitrapJ, and pSGLitrapQ were<br>
digested using Xbal and the fragments ranging from about 0.8 to 1.3 were isolated<br>
followed by ligations with pSGsetrapKN/O digested with Xbal and treated with<br>
alkaline phosphatase using standard molecular biological techniques. The ligations<br>
were used to transform £ co// DH10B and the transformants were analysed.<br>
Plasmids pSGsetrapKN/OI, pSGsetrapKN/OQ, pSGsetrapKN/OM and pSGrapKN/OJ<br>
were isolated and the orientation of the insert was verified by restriction digest<br>
analysis. For the addition of rapLhis these constructs were either digested with<br>
Bglll/Xbal followed by partial digest with Bglll as appropriate and the isolated vector<br>
fragments were ligated with the ~ 1kb Xbal/Bglll fragment of pSGL"rtrapLhl8.<br>
Isolation ofplasmids pSGsetrapKJM and pSGsetrapKJQ<br>
The plasmids pSGLitrapM and pSGLitrapQ were digested using Xbal and the<br>
fragments ranging from about 0.8 to 1.1 were isolated followed by a ligation with<br>
pSGsetrapKJ digested with Xbal and treated with alkaline phosphatase using<br>
standard molecular biological techniques. The ligations were used to transform £<br>
coli DM OB and the transformants were analysed. Plasmids pSGsetrapKJM and<br>
pSGrapKJQ were isolated and the orientation of the insert was verified by restriction<br><br>
digest analysis. For the addition of rapUs these constructs were either digested with<br>
Bglll/Xbal followed by partial digest with Bglll as appropriate and the isolated vector<br>
fragments were ligated with the ~ 1kb Xbal/Bglll fragment of pSGUtrapUis-<br>
Using the same strategy outlined above, the following gene cassettes were isolated:<br>
pSGsetrapKIJM  pSGsetrapKN/OJI  pSGsetrapKIQN/OM<br>
pSGsetrapKIJQ  pSGsetrapKJMN/O pSGsetrapKJMN/OQ<br>
pSGsetrapKIJN/O pSGsetrapKJQN/O pSGsetrapKIJN/OMQ<br>
pSGsetrapKIMN/O pSGsetrapKIJN/OM pSGsetrapN/OQ<br>
pSGsetrapKIQN/O pSGsetrapKIJN/OQ pSGsetrapKIJMN/OQ<br>
pSGsetrapKN/OMQ pSGsetrapKIMN/OQ<br>
An overview is given in Figure 5.<br>
For the addition of rapUt these cassette constructs were either digested with<br>
BgR\lXba\ or with Xba\ followed by partial digest with Bgl\\ as appropriate and the<br>
isolated vector fragments were ligated with the about 1 kb Xba\IBgl\\ fragment of<br>
pSGLJtrapUb-<br>
Example 6<br>
Isolation of 9-deoxo-16-O-desmethyl-2 7-desmethoxy-39-O-desmethyl-rapamycin<br>
(pre-rapamycin, Figure 6)<br>
9-Deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin (pre-<br>
rapamycin) was obtained by conjugating the S. hygroscopicus strain MG2-10 with<br>
pSGsetrapKL and isolating the products generated as described below. This<br>
demonstrates that it is possible to complement the deletion of rapK and rapi. in the<br>
MG2-1Q strain and that pre-rapamycin is produced, an analogue which is lacking<br>
post-PKS modification. The feeding of pipecolic acid is not required when rapL is<br>
complemented confirming that rapL plays a role in the provision of pipecolic acid in<br>
the production of rapamycin.<br>
4<br>
S. hygroscopicus MG2-10[pSGsetrapKL] was cultured from a frozen working<br>
spore stock in cryopreservative (20% glycerol, 10% lactose w/v in distilled water) on<br>
Medium 1 (see Materials and Methods) and spores were harvested after 14 days<br>
growth at 29°C. A primary pre-culture was inoculated with the harvested spores and<br>
cultured in two 250 ml Erlenmeyer flasks containing 50 ml Medium 3 (see Materials<br><br>
and Methods), shaken at 250 rpm with a two-inch throw, at 30°C, for two days. The<br>
primary pre-culture was used to inoculate two secondary prs-cultures of Medium 2<br>
(see Materials and Methods) and Medium 3, at 10% v/v, which was shaken at 300<br>
rpm with a one-inch throw, at 25°C, for a further 24h. Four litres of Medium 4 (see<br>
Materials and Methods) and Medium 5 (see Materials and Methods) were prepared<br>
containing 0.01% v/v Pluronic L101 antifoam (BASF). Production Medium 4 was<br>
inoculated with the secondary pre-culture in Medium 2 and Production Medium 5 was<br>
inoculated with the secondary pre-culture in Medium 3 at 10% v/v and allowed to<br>
ferment in a 7 L stirred bioreactor for five to seven days at 25°C. Airflow was set to<br>
0.75 wm and the impeller tip speed was controlled between 0.98 ms"1 and 2.67 ms"1.<br>
Additional Pluronic L101 was added on demand.<br>
To confirm the structure of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-<br>
desmethyl-rapamycin (pre-rapamycin), broths from Medium 4 and Medium 5 were<br>
extracted with ethyl acetate and reduced to a crude extract by evaporation. The<br>
extracts were defatted on partition with hexane:methanot:water and flashed through a<br>
70 g silica cartridge starting with hexane and finishing with acetone. Pre-rapamycin<br>
fractions from each fermentation were pooled and flashed through a C18 cartridge<br>
starting with water and finishing with methanol. Pre-rapamycin (8.5 mg) was isolated<br>
after chromatography on Sephadex LH20 using heptane:chloroform:ethanol as the<br>
mobile phase. This compound was analysed and the structure fully confirmed by<br>
NMR (Figure 18-20). The 1H and 13C NMR data are given in Table V below.<br><br><br><br><br><br>
Example 7<br>
Isolation of 8-deoxo-15-O-desmethyl-26-desmethoxy-38-O-desmethyl-<br>
prolylrapamycin (pre-prolylrapamycin, Figure 7)<br>
Feeding of S. hygroscopicus MG2-10[pSEGrapK] with proline acid resulted in<br>
the production pre-prolylrapamycin as described below. This demonstrated that in<br>
the absence of rapL alternative pipecolic acid analogues are incorporated.<br>
S. hygroscopicus MG2-10[pSGsetrapK] was grown in TSBGM fed with 1 mg/l<br>
proline at 25°C with shaking. The mycelia were extracted with methanol and the<br>
culture broth was extracted with ethyl acetate as described previously.<br>
Analysis of the culture broth of the proline-fed S. hygroscopicus mutant MG2-<br>
10[pSG,setrapKJ by HPLC with UV detection at 280nm revealed the presence of two<br>
major new peaks with retention times of 4.5 and 4.6 minutes. Electrospray mass<br>
spectroscopy of these peaks revealed that both contained ions corresponding to a<br>
compound with a MW of 827.5. Neither of these peaks were seen in the cultures of<br>
S. hygroscopicus NRRL 5491, S. hygroscopicus MG1C or S. hygroscopicus MG2-10<br>
without the rapK expression plasmid pSGsetrapK. MS/MS analysis of the ion with<br>
m/z of 850 (corresponding to the sodium adduct of pre-prolylrapamycin) revealed that<br>
it fragmented into an ion with m/z of 735 corresponding to the loss of m/z 115<br>
(proline), or an ion with m/z of 542 corresponding to the loss of m/z 308 (C27-C41 of<br>
pre-prolylrapamycin). This ion itself fragmented further to an ion with m/z 292,<br><br>
corresponding to the loss of m/z 250 (C13 to C26 of pre-prolylrapamycin). This<br>
fragmentation pattern was identical to the pattern seen for rapamycin but with the first<br>
loss of m/z (-115) reduced by 14 corresponding to the change from pipecolic acid to<br>
proline for the amino acid, the second loss of mfe (-308) reduced by 14,<br>
corresponding to the absence of the. C38 O-methyl group, the third loss of m/z (-250)<br>
reduced by 44, corresponding to the absence of the C26 methoxy.and C15 O-methyl<br>
groups and the final ion (306) having a mass reduced by 14 corresponding to the<br>
absence of the C8 ketone group and the change from pipecolic acid to proline. This<br>
was evidence that the compound with MW of 827.5 represents 8-deoxo-15-O-<br>
desmethyl-26-desmethoxy-38-O-desmethyl-prolylrapamycin (pre-prolylrapamycin).<br>
Example 8<br>
Isolation of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-desmethoxy-rapamycin (39-<br>
dehydroxy pre- rapamycin, Figure 8)<br>
Feeding of S. hygroscopicus MG2-10[pSGsetrapK] with pipecolic acid and<br>
cyclohexane carboxylic acid resulted in the production of two major compounds, pre-<br>
rapamycin which corresponds to the incorporation of the natural starter unit and 39-<br>
dehydroxy pre-rapamycin, which corresponds to the incorporation of the fed starter<br>
unit<br>
S. hygroscopicus MG2-10[pSGsetrapK| was grown in TSBGM fed with 2mg/l<br>
pipecolic acid and 1mM cyclohexane carboxylic acid at 25°C with shaking. The<br>
culture broth was extracted with ethyl acetate as described previously.<br>
Analysis of the culture broth of the cyclohexane carboxylic acid-fed S.<br>
hygroscopicus mutant MG2-10[pSGsetrapK] by HPLC with UV detection at 280nm<br>
revealed the presence of one major new peak with a retention time of 5.8 minutes.<br>
Electrospray mass spectroscopy of this peak revealed that it contained ions<br>
corresponding to a compound with a MW of 825.5. This peak was not seen in the<br>
cultures of S. hygroscopicus NRRL5491, S. hygroscopicus MG1C or S.<br>
hygroscopicus MG2-10 without the rapK expression plasmid pSGsetrapK. MS/MS .<br>
analysis of the ion with m/z of 848 (corresponding to the sodium adduct of 39-<br>
dehydroxy pre-rapamycin) revealed that it fragmented into an ion with m/z of 719<br>
corresponding to the loss of m/z 129 (pipecolic acid), or an ion with m/z of 556<br>
corresponding to the loss of m/z 292 (C28-C42 of 39-dehydroxy pre-rapamycin). This<br><br>
ion itself fragmented further to an ion with m/z 306, corresponding to the loss of m/z<br>
250 (C14 to C27 of 39-dehydroxy pre-rapamycin). This fragmentation pattern was<br>
identical to the pattern seen for pre- rapamycin but with the second loss of m/z (-292)<br>
reduced by 16, corresponding to the absence of the C39 hydroxy group. This was<br>
evidence that the compound with MW 825.5 represents 9-deoxo-15-O-desmethyl-27-<br>
desmethoxy-39-desmethoxy-rapamycin(39-dehydroxy-pre-rapamycin).<br>
Example 9<br>
Isolation of 16-O-desmethyl-27-desmethoxy-rapamycin (Figure 9)<br>
The S hygroscopicus strain MG2-10 was conjugated with pSGsetrapKIJ as<br>
described in Example 1. Feeding of this strain with pipecolic acid and isolation of the<br>
products produced on fermentation resulted in the production of 16-O-desmethyl-27-<br>
desmethoxy-rapamycin.<br>
The plasmid pSGsetrapKIJ (Figure 5) was conjugated into S. hygroscopicus<br>
MG2-10 and the strain grown inTSB GM fed with 2mg/l pipecolic acid at 25°C with<br>
shaking. The mycelia were extracted with methanol and the culture broth extracted<br>
with ethyl acetate as described previously.<br>
Analysis of the extracts of the S. hygroscopicus mutant MG2-<br>
10[pSGsetrapKIJ] by electrospray mass spectroscopy revealed one major new peak<br>
of retention time 4.3 minutes which contained ions corresponding to a compound with<br>
a MW of 869. This peak was not seen in the cultures of S. hygroscopicus NRRL<br>
5491, S. hygroscopicus MG1C S. hygroscopicus MG2-10 with or without the rapK<br>
expression plasmid pSGsetrapK. MS/MS analysis of the ion with m/z of 892<br>
(corresponding to the sodium adduct of 16-O-desmethyl-27-desrnethoxy-rapamycin)<br>
reveale9 that it fragmented into an ion with m/z of 763 corresponding to the loss of<br>
m/z 129 (pipecolic acid), or an ion with m/z of 570 corresponding to the loss of m/z<br>
322 (C28-C42 of 16-O-desmethyl-27-desmethoxy-rapamycin). This ion itself<br>
fragmented further to an ion with m/z 320, corresponding to the loss of m/z 250 (C14<br>
to C27 of 16-O-desmethyl-27-desmethoxy-rapamycin). This fragmentation pattern<br>
was identical to the pattern seen for rapamycin but with the third loss of m/z (-250)<br>
reduced by 44, corresponding to the absence of the C16 methyl and C27 methoxy<br>
groups. This was evidence that the compound with MW 869 was 16-O-desmethyl-27-<br>
desmethoxy-rapamycin.<br><br>
Example 10<br>
Army feeding<br>
S. hygroscopicus MG2-10[pSGsetrapKI] was used to cany out an array<br>
;  feeding. Primary vegetative cultures were prepared by innocculating medium with<br>
spore stock as described in the Materials and Methods. TSB GM medium was<br>
inoculated at 10% v/v using methods described in the materials and methods section.<br>
The following compounds were added as indicated in Table VI below<br><br>
The cultures were incubated, extracted and measured using techniques<br>
described in the Material and Method section. Table VII shows the results of the<br>
analysis showing the ion (m/z) observed for each combination of starter carboxylic<br>
acid and amino acid:<br><br>
These data demonstrate incorporation of the fed compounds.<br>
Example 11<br>
Complementation of S.hygroscopicus MG2-10 with fkbQ<br>
To assess whether rapK homologous genes such as fkbO in S.hygroscopicus<br>
var. ascomyceticus and S.tsukubaensis, and orf5 in the partially sequenced 'hyg'<br><br>
cluster (Ruan et al., 1997) fulfil similar functions, complementation assays were<br>
carried out using fkbO as described below.<br>
Isolation ofpMG169-1<br>
The gene fkbO from Strepomyces hygroscopicus var. ascomyceticus (ATCC<br>
14891), an FK520 producer, was amplified by PCR using the primers fkbof 5'-<br>
GGGCATATGACCGATGCCGGACGCCA 3' (SEQ ID NO: 54) and fkbor 5'<br>
GGGGTCTAGATCACGCCACCATGCCTTCGA 3" (SEQ ID NO: 55), introducing a<br>
aldel site at the 5'end and a Xbal site at the 3'end of fkbO. Genomic DNA isolated<br>
from S.hygroscopicusvar. ascomyceticus (ATCC 14891) was used as a template.<br>
The amplified PCR product was subjected to digestion with aldel and Xbal and<br>
ligated with WdekXbal cut pSGsetl. The ligation was used to transform Ecoli DH10B<br>
and the transformants were analysed using methods described in the Materials and<br>
Methods section. Plasmid pMG169-1 was isolated and verified by restriction<br>
digestion and S.hygivscopicus MG2-10 was transformed using methods described in<br>
the Materials and Methods section.<br>
Heterologous complementation of rapK by fkbO<br>
S.hygroscopicus MG2-10[pMG169-1] was grown in TSBGM fed with 2mg/l<br>
pipecolic acid at 25°C with shaking. The culture broth and mycelia were extracted<br>
using methods described in the Materials and Methods section (Method A). Analysis<br>
of the extract with UV detection at 280nm revealed the presence of two major new<br>
peaks with retention times of 4.5 and 4.6 minutes. Electrospray mass spectroscopy<br>
of these peaks revealed that both contained ions with a MW of 827.5 corresponding<br>
to two isomers of pre-rapamycin (Example 7).<br>
Example 12<br>
Efficient production of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-desmethoxy-<br>
rapamycin (39-dehyroxy pre-rapamycin, Figure 8) in the absence of competition by<br>
endogenous starter unit by feeding to a rapK knockout mutant<br>
The ability of S. hygroscopicus strains MG2-10 and MG2-10[pSGsetrapK] to<br>
incorporate a different starter unit, cyclohexane carboxylic acid, was compared as<br>
described below. When fed cyclohexane carboxylic acid and pipecolic acid MG2-10<br>
produced only one compound (39-dehydroxy pre-rapamycin) corresponding to<br><br>
incorporation of the fed starter unit only, whereas MG2-10[pSGsetrapK] produced<br>
two compounds in a 1:1 ratio, 39-dehydroxy pre-rapamycin and pre-rapamycin. This<br>
demonstrated that rapK is required for the incorporation of the natural endogenous<br>
starter unit and a rapK knock-out strain had no competition of the endogenous starter<br>
unit with the fed starter unit<br>
S. hygroscopicus MG2-10 was grown on TSBGM fed with 2 mg/L pipecolic<br>
acid and 1 mM cyclohexane carboxylic acid at 25°C with shaking. The culture broth<br>
was extracted with ethyl acetate as described previously. Analysis of the extracts by<br>
HPLC with UV detection at 280 nm revealed the presence of one new major peak<br>
with a retention time of 5.8 min. However, S. hygroscopicus MG2-10[pSGsetrapK|<br>
(Example 4), produced pre-rapamycin (Figure 6) in addition to 39-dehydroxy pre-<br>
rapamycin in a ratio of ~1:1 when fed with cyclohexane carboxylic acid (Example 8,<br>
Figure 8). Surprisingly, feeding of cyclohexane carboxylic acid to S. hygroscopicus<br>
MG2-10 resulted in a single product, 39-dehydroxy pre-rapamycin. The endogenous<br>
starter, 4,5-dihydroxycyclohex-1-ene carboxylic acid, was not incorporated in the<br>
absence of rapK. There was therefore no competition between the incorporation of<br>
the fed carboxylic acid and the endogenous starter.<br>
Example 13<br>
Elucidation of the function ofRapM<br>
Cultures of Str&amp;ptomyces IMdans TK24, S. lividans TK24[pSGsetrapM] and<br>
S. IMdans TK24[pSGsetrapQ] were grown in TSBGM with shaking at 30°C and fed<br>
with 20 ng/ml of pre-rapamycin. Controls remained unfed. After a further 5 days<br>
incubation, the cultures were extracted with ethylacetate and brought to dryness.<br>
Reconst'rtution and analysis by LC-MS identified no productiorf of rapamycin<br>
analogues in the unfed controls. Two major new peaks were identified in the extract<br>
of S. //V/dansTK24[pSGsetrapM] fed pre-rapamycin, one at 2.5 min and one at 7.9<br>
min. Electrospray mass spectroscopy of these peaks revealed that both contained<br>
ions corresponding to a compound with a MW of 855.6, consistent with 9-deoxo-16-<br>
0-methyl-27-desmethoxy-39-O-desmethyl-rapamycin(16-O-methyl-pre-rapamycin)..<br>
Two isomers were commonly observed when extracts were analysed by LC-MS in<br>
the absence of TFA. No new peaks were identified in the extracts of S. lividans TK24<br>
or S. lividans TK24[pSGsetrapQ]. Unmodified pre-rapamycin was clearly evident.<br><br>
RapM was clearly responsible for methylation at the C16 hyclroxyl, RapQ was not<br>
specific for this site.<br>
Example 14<br>
Elucidation of the function of RapJ<br>
Cultures of S. IMdans TK24, S. Uvidans TK24[pSGsetrapK], S. IMdans<br>
TK24[pSGsetrapJ] and S. Uvidans TK24[pSGsetrapKJ] were grown in TSBGM with<br>
shaking at 30°C and fed with 40 jig/ml of pre-rapamycin. Controls remained unfed.<br>
After a further 5 days incubation, the cultures were extracted with ethylacetate and<br>
brought to dryness. Reconstitution and analysis by LC-MS identified no production of<br>
rapamycin analogues in the unfed controls. One major new peak at 4.9 min was<br>
identified in the extracts of S. IMdans TK24[pSGsetrapKJ] and S. IMdans<br>
TK24[pSGsetrapJ] fed pre-rapamycin. Electrospray mass spectroscopy of this peak<br>
revealed that it contained ions corresponding to a compound with a MW of 855.5,<br>
consistent with 16-O-desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin (C9 oxo-<br>
pre-rapamycin). In extracts of S. IMdans TK24 and S. IMdans TK24[pSGsetrapK] fed<br>
with pre-rapamycin, no new peaks were identified. Unmodified pre-rapamycin was<br>
clearly evident.<br>
Due to the homology of RapJ with FkbD of the FK506 and FK520 cluster,<br>
Rap J has been postulated to oxidise pre-rapamycin at C9 to 9-hydroxy-16-O-<br>
desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin (C9 OH-pre-rapamycin).<br>
RapK has been postulated to be responsible for the further conversion to the ketone.<br>
Surprisingly, in the presence of RapJ, but in the absence of RapK, 16-O-desmethyl-<br>
27-desmethoxy-39-O-desmethyl-rapamycin (C9 keto-pre-rapamycin) was formed.<br>
RapJ clearly has an oxidative function at C9, complete conversion to the ketone was<br>
observed. RapK does not have an oxidative function at C9.<br>
Example 15<br>
Plasmids containing the following combinations of rapamycin modifylng genes<br>
were constructed as described below: pMG260 (rapl, rapJ, rapN, rapO, and rapL),<br>
pMG261 (rapl, rapJ, rapN, rapO, rapM and rapL), pMG262 (rapl, rapJ, rapN, rapO,.<br>
rapM, rapQ and rapL) pMG236 (rapN, rapO, rapQ and rapL) and pMG238 (rapJ and<br>
rapL).<br>
Isolation of plasmids pMG236 andpMG238<br><br>
The plasmids pSGsetrapNOQ and pSGsetrapJ were digested using Bglll/Xbal and<br>
the isolated vector fragments were ligated with the 1kb Xbai/Bglll fragment of<br>
pSGLitrapUte. Plasmids pMG236 (expressing rapN, rapO, rapQ and rapL) and<br>
pMG238 (expressing rapJ and rapL) respectively, were isolated.<br>
Isolation of plasmids pMG260, pMG261 andpMG262<br>
The plasmids pSGSetrapKIJNOL, pSGSetrapKIJMNOL, and pSGSetrapKUMNOQL<br>
were digested using Bglll and the isolated insert fragments (containing the rapamycin<br>
cluster genes from the Bglll site in rapl to the Bglll site after rapL) were ligated with<br>
the vector-containing fragment from pSGSetrapl digested with Bglll. Plasmids<br>
pMG260 (expressing rapl, rapJ, rapN, rapO, and rapL), pMG261 (expressing rapl,<br>
rapJ, rapN, rapO, rapM and rapL), and pMG262 (expressing rapl, rapJ, rapN, rapO,<br>
rapM, rapQ and rapL) were isolated.<br>
Example 16<br>
An S.hygroscopicus mutant (MG3) carrylng the chromosomal deletion of rapK<br>
was constructed as described below. Heterologous complementation of rapK with<br>
fkbO can then be performed as described and will result in the restoration of<br>
rapamycin production demonstrating that fkbO"\s able to complement the function of<br>
rapK'm S. hygroscopicus.<br>
Isolation of the S.hygroscopicus mutant MG3 canylng the chromosomal deletion of<br>
rapK<br>
The primers RAPKF1 5'-CAAAGCTTCCTGGCGCGGTTCGGCCGGCA-3'<br>
(SEQ ID NO: 56) and RAPKF2 5,-TGGCATGCCCTTCCCCGCCGTTCCCTGGC-3,<br>
(SEQ ID.NO: 57) were used to amplify the left region of homology outside the gene<br>
rapK (from nt94403 to nt95429 in the rapamycin cluster as described in Schwecke et<br>
al., 1995) using genomic DNA prepared from S.hygroscopicus NRRL5491 as a<br>
template. The 1kb PCR product was phosphorylated using T4 polynucleotide kinase<br>
and ligated into dephosphorylated Smal cut pUC18. After transformation into Eco//<br>
DH10B, the plasmid pMG233-7 was isolated. The primers RAPKR1 5'-<br>
TGGCATGCCCCCGCCGAGCTGACCTGGAA-3' (SEQ ID NO: 58) and RAPKR2 5'-<br>
GTTCTAGAGCTTACGCGTGATGTCGAACG-3' (SEQ ID NO: 59) were used to<br>
amplify the right region of homology outside the gene rapK (from nt96435 to nt97428<br>
in the rapamycin cluster as described in Schwecke et al., 1995) using genomic DNA<br><br>
prepared from S.hygroscopicus NRRL5491 as a template: The 1kb PCR product was<br>
phosphorylated using T4 polynucleotide kinase and ligated into dephosphorylated<br>
Smal cut pUC18. After transformation into Eco// DH10B, the plasmid pMG257-7 was<br>
isolated. Both plasmids were checked by sequence analysis. The plasmid pMG233-7<br>
was digested with SphlZXbal and the 3.7kb fragment was isolated, pMG257-7 was<br>
digested with Sphl/Xbal and the 1kb fragment isolated. These fragments were ligated<br>
and used to transform Eco// DH10B. The plasmid pMG268-12 was isolated. This<br>
plasmid was digested with Hindlll/Xbal and the 2kb fragment isolated and ligated into<br>
pMG55 cut with Hindlll/Xbal and the DNA was used to transform E.coli DH10B. The<br>
plasmid pMG278-1 was isolated and used to conjugate S.hygroscopicus MG1C.<br>
An apramycin resistant colony is isolated, and is grown for 24 hours in<br>
TSBGM with shaking at 30°C and spread onto medium 1 agar plates containing<br>
50ug/l streptomycin. Streptomycin resistant colonies are isolated and shown to be<br>
apramycin sensitive. The 1004nt chromosomal deletion of rapK can be verified in the<br>
mutant MG3 by Southern blotting. An overview is given in Figure 35.<br>
S.hygroscopicus MG3 is grown in TSBGM at 26°C with shaking. The culture<br>
broth and mycelia are extracted using methods as described in the Materials and<br>
Methods section. Analysis of the extract with UV detection reveals the presence of no<br>
peaks with the characteristic rapamycin triene.<br>
Expression offkbO in the S.hygroscopicus mutant MG3 carrylng the chromosomal<br>
deletion ofrapK<br>
Plasmid pMG 169-1 (described in example 11) is transformed into<br>
S.hygroscopicus mutant MG3 using methods as described in the Materials and<br>
Methods section.<br>
Heterologous complementation of rapK by fkbO<br>
S.hygroscopicus MG3pMG169-1 is grown in TSBGM at 26°C with shaking.<br>
The culture broth and mycelia arere extracted using methods as described in the<br>
Materials and Methods section. Analysis of the extract with UV detection at 280nm.<br>
reveals the presence of two major new peaks. Electrospray mass spectroscopy of<br>
these peaks reveals that these contain ions with a MW of 913 corresponding to<br>
rapamycin.<br><br>
Example 17<br>
Isolation and heterologous complementation of the S.hygroscopicus var<br>
ascomyceticus mutant MG4 canylng the chromosomal deletion of fkbO<br>
Isolation of the S.hygroscopicus var ascomyceticus mutant MG4 carrylng the<br>
chromosomal deletion of fkbO<br>
The primers FKOF1 5'-GCTCTAGAGCCCGCGGCTCGCCGGACACG-3'<br>
(SEQ ID NO: 60) and FKOF2 5'-CCCCTGCAGGCGTCCGGCATCGGTCATCAG-3'<br>
(SEQ ID NO: 61) were used to amplify the left region of homology (from nt45750 to<br>
nt46751 in the ascomycin cluster as described in Wu et al., 2000) using genomic<br>
DNA prepared from S.hygroscopicus var ascomyceticus ATCC14891 as a template.<br>
The 1kb PCR product was phosphorylated using T4 polynucleotide kinase and<br>
ligated into dephosphorylated Smal cut pUC18. After transformation into E.coli.<br>
DH10B, the plasmid pMG258-4 was isolated. The primers FKOR1 5'-<br>
CGCCTGCAGGGATACGGTCCGCCGGGTCTGC-3' (SEQ ID NO: 62) and FKOR2<br>
5XCMGCTTGTACGGTTCGCCACGGGCGTGC-3' (SEQ ID NO: 63) were used to<br>
amplify the right region of homology (from nt47785. to. nt48781 in the rapamycin<br>
cluster as described in Wu et al., 2000) using genomic DNA prepared from<br>
S.hygroscopicus var ascomyceticus ATCC14891 as a template. The 1kb PCR<br>
product was phosphorylated using T4 polynucleotide kinase and ligated into<br>
dephosphorylated Smal cut pUC18. After transformation into E.coli DH10B, the<br>
plasmid pMG259-5 was isolated. Both plasmids were checked by sequence analysis.<br>
The plasmid pMG258-4 was digested with Sbfl/Hindlll and the 3.7kb fragment was<br>
isolated, pMG259-5 was digested with Sbfl/Hindlll and the 1kb fragment isolated.<br>
These fragments were ligated and used to transform E.coli DH10B. The plasmid<br>
pMG265-1 was isolated. This plasmid was digested with HindHI/EcoRI and the 2kb<br>
fragment isolated and ligated into pMG55 cut with HindHI/EcoRI and the DNA was<br>
used to transform Ecoli DH10B, The plasmid pMG267-1 was isolated and used to<br>
conjugate S.hygroscopicus var ascomyceticus ATCC14891.<br>
An apramycin resistant colony is isolated and is grown for 24 hours in<br>
TSBGM with shaking at 30°C and spread onto medium 1 agar plates containing<br>
50ug/l streptomycin. Streptomycin resistant colonies are isolated and shown to be<br>
apramycin sensitive. The 1034nt chromosomal deletion of fkbO can be verified in the<br>
mutant MG4 by Southern blotting. An overview is given in Figure 36.<br><br>
Expression ofRapKin the S.hygroscopicus var ascomyceticus mutant MG4 carrylng<br>
the chromosomal deletion of fkbO<br>
Plasmid pSGsetRapK is transformed into S.hygroscopicus mutant MG4 as<br>
described in the Materials and Methods section.<br>
Heterologous complementation of fkbO by rapK<br>
S.hygroscopicus var ascomyceticus MG4 pSGSetRapK is grown in TSBGM<br>
at 26°C with shaking. The culture broth and mycelia are extracted using methods as<br>
described in the Materials and Methods section. The extract is analysed by LC-MS to<br>
reveal the presence of a major new peak and to reveal that this contains ions that<br>
correspond to FK520 (ascomycin).<br>
Example 18<br>
It is obvious to those skilled in the art that other biosynthetic clusters that<br>
encode FKBP-ligands for example, FK506, can be modified such that the rapK<br>
homologue is deleted or inactivated using the methods as described herein. In<br>
FK506, for example; this could be done by amplifylng PGR products against the<br>
regions either side of the fkbO gene (sequence accession number AF082099,<br>
AF082100), ligating these together in a vector such as pMG55, transforming the<br>
FK506-producing strain, selecting for the double crossover and confirming the<br>
removal of the fkbO gene by southern blotting.<br>
Example 19<br>
Incorporation of non-natural starter units by the rapK deletion strain, S.<br>
hygroscopicus MG2-10, into rapamycin analogues in the absence of competition by<br>
endogenous natural starter unit.<br>
As demonstrated in examples 10 and 12, the rapamycin PKS has a high<br>
degree of flexibility for non-natural starter units and in the absence of rapK, the<br>
system is free of competition from the natural starter. In this example, the degree of<br>
flexibility is further demonstrated.<br>
S. hygroscopicus MG2-10 was grown, fed and extracted according to the<br>
feeding, extraction and analysis methods outlined in Materials and Methods (Method<br>
B). The range of carboxylic acids fed along with the compounds generated are listed<br>
below. Surprisingly, all of the carboxylic acids listed were incorporated as<br>
determined by observing the characteristic UV chromophore at 278 nm and<br><br>
electrospray mass spectrometry and resulted in the production of rapamycin<br>
analogues.<br>
The rapamycin analogues generated corresponded to the formula below as<br>
described in Table VIII:<br><br><br><br><br><br>
Example 20<br>
Incorporation of non-natural starter units by the rapK deletion strain, S.<br>
hygroscopicus MG2-■/0[pSGsetrapN/OQLhiS], into rapamycin analogues in the<br>
absence of competition by endogenous natural starter unit.<br>
As demonstrated in examples 10,12 and 19, the rapamycin PKS has a high<br>
degree of flexibility for non-natural starter units and in the absence of rapK, the<br>
system is free of competition from the natural starter. In this example, the degree of<br>
flexibility is further demonstrated.<br>
S. hygroscopicus MG2-10[pSGsetrapN/OQLhis] was grown, fed and extracted<br>
according to the feeding, extraction and analysis methods outlined in Materials and<br>
Methods (Method B). The range of carboxylic acids fed along with the compounds<br>
generated are listed below. Surprisingly, all of the carboxylic acids listed were<br>
incorporated as determined by observing the characteristic UV chromophore at 278<br><br>
nm and electrospray mass spectrometry and resulted in the production of rapamycin<br>
analogues.<br>
The rapamycin analogues generated corresponded to the formula below as<br>
described in Table IX:<br><br><br>
Carboxylic acid	I M-H  [M+K] I Compound generated<br>
starter unit fed.<br>
1-cyclohexene	824.4 864.4 RIB = E, R« = 3-OH, y = bond, in<br>
carboxvlic acid	combination with R1 = OH, R2 = H, R6 = H,<br>
carboxylic ac»o	Rs = H, R7 = H, R6 = H, R6 = H, R10 = H, x =<br>
	CHg	<br>
840.4 880.4 Ris = E, R16 = 3-OH, y = bond, in<br>
combination with R1 = OH, R2 = OH, R6 =<br>
H, Re = H, R7 = H, Re = H, R5 = H, Rn&gt; — H,<br>
x = CH2<br>
3-cycIohexene	840.4 880.4 R15 = C, R™ = OH, R17 = OH, in<br>
.  ,.  ..	combination with R1 = OH, R2 = H, R5 - H,<br>
carboxylic acid	R _H R _Up iu R-M P - H v -<br>
r%6 - n, K7 — n, (\B ~ n, K9 — n, rvio — n, x —<br>
	CH5;	<br>
822.4 862.4 R15 = A, Ri6 = OH, R17~= H, in combination<br>
with R! = OH, R2 = H, R6 = H, Re = H, R7 =<br>
	H, R6 = H, R9 = H, Rip = H, x = CH2	<br>
840.4 880.4 R15 = A, R16 = OH, R17 = H, in combination<br>
with R1 = OH, R2 = OH, R5 = H, R6 = H, R7<br>
= H, R6 = H, Rg = H, R10 = H, x = CH2<br>
cycloheptane	854.4 894.4 R15 = E, R1B = OH, y = CH2) in combination<br>
.  ..  ..	with R-, = OH, R2 = OH, R5 = H, R6 = H, R7<br>
carboxyhc acd	= H&gt; R? = ^ ^ = H ^ = H x J ^<br>
methyl-2-norbornane  852.4 892.4 R15 = B, Ri6 s OH, R17 = H, in combination<br>
cartoxylic acid	!* * Z™A% °* *" ■* £.= »■ *<br>
— n, KB - n, Kg — n, K10 — n, x — on2<br>
3-hydroxycyclohexane  824.4 864.4 R16 = E, R16 = 3-OH, y = bond, in<br>
carboxylic acid	^fl Jf1*! 7, 0RH'_RH =RH' %\"l<br>
	_CH^	<br>
4-hydroxycyclohexane  840.4 880.4 R16 = E, R16 = 4-OH, y = bond, in<br>
carboxylic acid	^fl *** " °RH-_RH =H' *" *<br>
r\e ~ n, r\7 — n, r\s ~ n, t\g — n, rvio — n, A -<br>
	Cji	2	<br>
824.4 864.4 R1S = E, R16 = 4-OH, y = bond, in<br>
combination with Ri = OH, R2 = OH, R5 =<br>
H, Rg = H, R7 = H, Rg = H, Rg = H, R10 = H,<br>
	x = CH2	<br>
4-methylcyclohexane  838.4 878.4 R15 = D, R17 = OH, in combination with R1 =<br>
carboxylic acid	?H1Rz Z H- *« = "• *«= H, R7 = H, R8 = H,<br>
_;	Kg — n, K10 = n, X — Un2	<br>
854.4 894.4 R15= D, R17 a OH, in combination with R1 —<br>
OH, R2 = OH, R5 = H, R6 = H, R7 = H, Re =<br>
	;	H, Rg = H, Rip = H, x = CH2	<br><br>
Example 20<br>
Incorporation of non-natural starter units by the rapK deletion strain, S.<br>
hygroscopicus MG3, into rapamycin analogues in the absence of competition by<br>
endogenous natural starter unit.<br>
As demonstrated in examples 10,12 and 19, the rapamycin PKS has a high<br>
degree of flexibility for non-natural starter units and in the absence of rapK, the<br>
system is free of competition from the natural starter. In this example, the degree of<br>
flexibility is further demonstrated.<br>
S. hygroscopicus MG3 is grown, fed and extracted according to the feeding,<br>
extraction and analysis methods outlined in Materials and Methods (Method B). The<br>
range of carboxylic acids fed that can be fed is listed below. Incorporation of the<br>
carboxylic acids listed and production of rapamycin analogues is determined by<br>
observing the characteristic UV chromophore at 278 nm and electrospray mass<br>
spectrometry.<br>
Carboxylic acid starter units that can be fed include, cyclohexane carboxylic<br>
acid, 3-cis,4-trans-dihydroxycyclohexane carboxylic acid, 1-cyclohexene carboxylic<br>
acid, 3-cyclohexene carboxylic acid, cycloheptane carboxylic acid, methyl 2-<br>
norbornane carboxylate, 3-hydroxycyclohexane carboxylic acid, 4-<br>
hydroxycyclohexane carboxylic acid, 3-methylcyclohexane carboxylic acid, 4-<br>
methylcyclohexane carboxylic acid, 3-(cis/trans)methoxycyclohexane carboxylic acid,<br>
4-(cis/trans)methoxycyclohexane carboxylic acid, ethyl 4-cyclohexanone carboxylate,<br>
3-fluoro-4-hydroxycarboxylic acid and 4-fluoro-3-hydroxycarboxylic acid, 3-<br>
cyclohexane oxide carboxylic acid, 3,4-cis-dihydroxycyclohexane carboxylic acid, 3-<br>
chloro-4-hydroxycarboxylic acid and 4-chloro-3-hydroxycarboxylic acid (and the pair<br>
of opposite diastereomers), cyclohexylpropionic acid and 4-tert-Butylcyclohexane<br>
carboxylic acid<br>
Example 21<br>
Incorporation of non-natural starter units by the fkbO deletion strain, S.<br>
hygroscopicus var. ascomyceticus MG4, into FK520 analogues in the absence of<br>
competition by endogenous natural starter unit<br>
As demonstrated in examples 10,12,19 and 20, the rapamycin PKS has a<br>
high degree of flexibility for non-natural starter units. In the absence of fkbO, the<br>
FK520 system is free of competition from the natural starter. In this example, the<br><br>
degree of flexibility of the FK520 PKS is investigated, free of competition from the<br>
natural starter.<br>
S. hygrvscopicus van ascomyceticus MG4 is grown, fed and extracted<br>
according to the feeding, extraction and analysis methods outlined in Materials and<br>
Methods (Method 6). Examples of the range of carboxylic acids that can be fed are<br>
given in Table IV. Incorporation of the carboxylic acids listed and production of<br>
FK520 analogues is determined by electrospray mass spectrometry.<br>
Example 22<br>
Incorporation of non-natural starter acids into FK506 analogues by an fkbO deletion<br>
mutant ofS. tsukubaensis in absence of competition from the natural starter.<br>
An fkbO deletion mutant of S. tsukubaensis is grown and fed according to the<br>
feeding methods outlined in Materials and Methods. A sub-set of the carboxylic<br>
acids listed in Table IV in Materials and Methods is fed. Analysis is performed as<br>
described in Method (B) of Materials and Methods.<br>
Example 23<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10[pSGsetrapKILJ<br>
9-deoxo-16-O-desmethyl-27-desmethoxy-rapamycin was obtained by<br>
conjugating the S. hygroscopicus strain MG2-10 with pSGsetrapKIU and isolating<br>
the fermentation products generated as described below. This demonstrates that it is<br>
possible to complement the deletion of rapK, rapl and rapL in the MG2-10 strain and<br>
that 9-deoxo-16-Odesmethyl-27-desmethoxy-rapamycin is produced, an analogue<br>
which is lacking the post-PKS modifications. The feeding of pipecolic acid is not<br>
required when rapL is complemented confirming that rapL plays a role in the<br>
provision of pipecolic acid in the production of rapamycin.<br>
S. hygroscopicus MG2-10 [pSGsetKILhfe] was fermented (see Materials and<br>
Methods), extracted and isolated using the method (B) as outlined in Materials and<br>
Methods. The isocratic solvent system used for preparative HPLC was 60%<br>
CH3CN/H20.<br>
9-Deoxo-16-O-desmethyl-27-desmethoxy rapamycin (Compound 6) has the following<br>
characteristics:<br>
Isolated yleld: 22 mg<br>
Molecular weight 856<br><br>
Molecular formula: C49H77NOH<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa* = 878, m/z for M-H = 854<br>
Table X below summarises the 1H and 13C NMR data for 9-deoxo-16-O-desmethyl-<br>
27-desmethoxy rapamycin in CDCI3.<br><br><br><br><br>
b: tentative assignment<br>
c: the assignment may be interchanged<br>
d: the assignment may be interchanged<br>
Compound 6 exists as a 1:1 mixture of conformers in CDCI3. The data above<br>
is for both conformers. Where a dotted line has been drawn across the table it was<br>
not possible to determine connectivity between spin systems, hence the assignment<br>
of data to a particular conformer is not possible.<br>
Example 24<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10[pSGsetrapKIMLh]<br>
9-Deoxo-27-desmethoxy-rapamycin was obtained by conjugating S.<br>
hygroscopicus MG2-10 strain with pSGsetKIMLhte as described in example 1 and<br>
isolating the products produced on fermentation. This demonstrated that it was<br>
possible to complement the deletion of rapK, rapl, rapM and rapL in the MG2-10<br>
strain with the production of a rapamycin analogue lacking some post-PKS<br>
modification.<br>
S. hygroscopicus MG2-10 [pSGsetKIMLhte] was fermented (see Materials and<br>
Methods), extracted and isolated using the method (B) as outlined in Materials and<br>
Methods.<br>
The isocratic solvent system used for preparative HPLC was 75% CH3CN/H20.<br>
9-Deoxo-27-desmethoxy rapamycin (Compound 16) has the following characteristics:<br>
isolated yleld: 24 mg<br>
Molecular weight: 870<br>
Molecular formula: C50H79NOH<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
•nm	"<br>
Electrospray MS: m/z for MNa+ = 892, m/z for M-H = 868<br>
Table XI below summarises the 1H and 13C NMR data for 9-deoxo-27-desmethoxy<br>
rapamycin in CDCI3.<br><br><br><br><br><br>
Example 25<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10[pSGsetKIN/OLh]<br>
9-Deoxo-16-O-desmethyl-27-O-desmethyl-rapamycin was obtained by<br>
conjugating S. hygroscopicus MG2-10 strain with pSGsetKIN/OLhis as described in<br>
Example 1 and isolating the products produced on fermentation. This demonstrated<br>
that it was possible to complement the deletion of rapK, rapl, eapN/O and rapL in the<br>
MG2-1© strain with the production of a rapamycin analogue lacking some post-PKS<br>
modification.<br>
S. hygroscopicus MG2-10 [pSGsetKIN/OUia] was fermented (see Materials<br>
and Methods), extracted and isolated using the method (B) as outlined in Materials<br>
and Methods.<br>
The isocratic solvent system used for preparative HPLC was 60% CH3CN/H20.<br>
9-Deoxo-16-O-desmethyl-27-O-desmethylrapamycin (Compound 9) has the following<br>
characteristics:<br>
Isolated yleld: 77 mg<br>
Molecular weight 872<br><br>
Molecular formula: C49H77NO12<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa+ = 894, m/z for M-H = 870<br>
Table XII below summarises the 1H and 13C NMR data for 9-deoxo-16-O-desmethyl-<br>
27-O-desmethylrapamycin in CDCI3.<br><br><br><br>
Example 26<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10[pSGsetKJLh]<br>
16-O-Desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin was obtained by<br>
conjugating S. hygroscopicus MG2-10 strain with pSGsetKJLWB as described in<br>
Example 1 and isolating the products produced on fermentation. This demonstrated<br><br>
that it was possible to complement the deletion of rapK, rapJ and rapL in the MG2-10<br>
strain with the production of a rapamycin analogue lacking some post-PKS<br>
modification.<br>
S. hygrvscopicus MG2-10 [pSGsetKJLhj6] was fermented (see Materials and<br>
Methods), extracted and isolated using the method (B) as outlined in Materials and<br>
Methods.<br>
The isocratic solvent system used for preparative HPLC was 55% CH3CN/H20.<br>
16-O-Desmethyl-27-desmethoxy-39-O-desmethyl rapamycin (Compound 3) has the<br>
following characteristics:<br>
Isolated yleld: 176 mg (mixture of 2 interconverting isomers)<br>
Molecular weight 856<br>
Molecular formula: C4BH73N012<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa+ = 878, m/z for M-H = 854<br>
MS fragmentation: The sodiated adduct (m/z 878) was fragmented to provide<br>
three fragments: C8-C42, m/z MNa+ 749; C1-C27, m/z MNa+ 570; C28-C42+C1-<br>
C14, m/z MNa+ 628. The fragment ions 628 and 570 were fragmented further to give<br>
the same fragment: C1-C14, m/z MNa* 320. The mass of this C1-C14 fragment is<br>
14 mass units greater than the equivalent fragment from the fragmentation of the<br>
sodiated adduct of 9-deoxo-16-O-desmethyl-27-desmethoxy-39-Odesmethyl<br>
rapamycin (Compound 1) consistent with oxidation at C9.<br>
Example 27<br>
Isolation of product from fermentation of S. hygroscopicus MG2-lO[pSGsetKMNOLh]<br>
9-Deoxo-27-O-desmethyl-39-O-desmethyl-rapamycin was obtained by<br>
conjugating S. hygroscopicus MG2-10 strain with pSGsetKMN/OLhs as described in<br>
example 1 and isolating the products produced on fermentation. This demonstrated<br>
that it was possible to complement the deletion of rapK, rapM, rapN/O and rapL in<br>
the MG2-10 strain with the production of a rapamycin analogue lacking some post-<br>
PKS modification.<br>
S. hygroscopicus MG2-10 [pSGsetKMN/OLws] was fermented (see Materials<br>
and Methods), extracted and isolated using the method (B) as outlined in Materials<br>
and Methods.<br>
The isocratic solvent system used for preparative HPLC was 60% CH3CN/H20.<br><br>
9-Deoxo-27-O-desmethyl-39-O-desmethyl rapamycin (Compound 8) has the<br>
following characteristics:<br>
Isolated yleld: 6 mg<br>
Molecular weight: 872<br>
Molecular formula: C^HTTNO^<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa* = 894, m/z for M-H = 870<br>
MS fragmentation: The sodiated adduct (m/z 894) was fragmented to provide<br>
three fragments: C8-C42, m/z MNa+ 765; C1-C27, m/z MNa+ 586; C28-C42+C1-<br>
C14, m/z MNa+ 614. The fragment ions 614 and 586 were fragmented further to give<br>
the same fragment: C1-C14, m/z MNa* 306. The C1-C14 is identical to that<br>
obtained from fragmentation of the sodiated adduct of 9-deoxo-16-O-desmethyl-27-<br>
desmethoxy-39-O-desmethyl rapamycin; the compound is 9-deoxo. The C1-C27<br>
fragment is 30 mass units greater than the equivalent fragment from 9-deoxo-16-O-<br>
desmethyl-27-desmethoxy-39-O-desmethyl rapamycin, consistent with one<br>
hydroxylation and one methylation; RapM methylates the hydroxy group at C-16 (see<br>
Example 22 for pSGsetKIUis together with Example 23 pSGsetKIMUis) and RapN in<br>
combination with RapO hydroxylates C27 so the data is consistent with the<br>
compound being 9-deoxo-27-O-desmethyl-39-O-desmethyl rapamycin (Compound<br>
8).<br>
Example 28<br>
Isolation of product from fermentation ofS. hygroscopicus MG2-lO[pSGsetKIJLh]<br>
16-O-desmethyl-27-desmethoxy-rapamycin was obtained by conjugating S.<br>
hygroscopicus MG2-10 strain with pSGsetKIJLws as described] in Example 1 and<br>
isolating the products produced on fermentation. This demonstrated that it was<br>
possible to complement the deletion ofrapK, rapl, rapJ and rapL in the MG2-10<br>
strain with the production of a rapamycin analogue lacking some post-PKS<br>
modification.<br>
S. hygroscopicus MG2-10 [pSGsetKIJUiJ was fermented (see Materials and<br>
Methods), extracted and isolated using the method (B) as outlined in Materials and<br>
Methods.<br>
The isocratic solvent system used for preparative HPLC was 60% CH3CN/H20.<br><br>
16-ODesmethyl-27-desmethoxy rapamycin (Compound 12) has the following<br>
characteristics:<br>
isolated yleld: 11 mg<br>
Molecular weight: 870<br>
Molecular formula: C49H75N012<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa* = 892, m/z for M-H = 868<br>
Example 29<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10fpSGsetKL\^<br>
9-Deoxo-16-O-desmefriyl-27-desmethoxy-39-O-desmethyl-rapamycin was<br>
obtained by conjugating S. hygroscopicus MG2-10 strain with pSGsetKLws as<br>
described in example 1 and isolating the products produced on fermentation. This<br>
demonstrated that it was possible to complement the deletion of rapK and rapL in the<br>
MG2-10 strain with the production of a rapamycin analogue lacking post-PKS<br>
modification (pre-rapamycin).<br>
S. hygroscopicus MG2-10 [pSGsetKUiJ was fermented, extracted and<br>
isolated using the methods outlined in Materials and Methods.<br>
The isocratic solvent system used for preparative HPLC was 60%<br>
CH3CN/H20.<br>
9-Deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl rapamycin<br>
(Compound 1) has the following characteristics:<br>
Isolated yleld: 24 mg<br>
Molecular weight 842<br>
..Molecular formula: C48H75NOH<br>
" UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa+ = 864, m/z for M-H = 840<br>
MS fragmentation: The sodiated adduct (m/z 864.5) was fragmented to<br>
provide four fragments: C8-C42, m/z MNa* 735; C1-C27, m/z MNa* 556; C28-<br>
C42+C1-C14, m/z MNa* 614, C1-C14, m/z MNa+ 306. The expected m/z for these<br>
fragments were determined by comparison to the reported fragmentation of<br>
rapamycin (J. A. Reather, Ph.D. Dissertation, University of Cambridge, 2000). These<br><br>
fragments have the same m/z as the predicted m/z for the fragmentation of 9-deoxo-<br>
16-O-desmethyl-27-desmethoxy-39-O-desmethyl rapamycin.<br>
Example 30<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10 fed with<br>
cyclohexane carboxylic acid<br>
9-Deoxo-16-O-desmethyl-27-desmethoxy-39-desmethoxy-rapamycin was<br>
obtained on feeding cyclohexane carboxylic acid to S. hygroscopicus MG2-10 and<br>
isolating the products produced on fermentation. The resulting mutasynthesis<br>
demonstrated that it was possible to chemically complement the deletion of rapK in<br>
the MG2-10 strain, in the absence of natural endogenous starter, with the resulting<br>
production of a rapamycin analogue lacking post-PKS modification.<br>
S. hygroscopicus MG2-10 was fermented (see Materials and Methods), fed<br>
(see Materials and Methods), extracted and isolated using the method (B) as outlined<br>
in Materials and Methods.<br>
The tsocratic solvent system used for preparative HPLC was 50%<br>
CH3CW/H20.<br>
9-Deoxo-16-Odesmethyl-27-desmethoxy-39-desmethoxy rapamycin<br>
(Compound AT) has the following characteristics:<br>
Isolated yleld: 12 mg<br>
Molecular weight 826<br>
Molecular formula: C48H75NO10<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa+ = 848.5, m/z for M-H = 825<br>
MS fragmentation: The sodiated adduct (m/z 848.5) was fragmented to<br>
provideCfour fragments: C8-C42, m/z MNa+ 719; C1-C27, m/z MNa+ 556; C28-<br>
C42+C1-C14, m/z MNa+ 598, C1-C14, m/z MNa+ 306. These data illustrate that the<br>
difference between Compound 47 and 9-deoxo-16-O-desmethyl-27-desmethoxy-39-<br>
O-desmethyl rapamycin (Compound 1) is located in the region of C28-C42. This<br>
fragment is 16 mass units less for Compound 47 than it is for Compound 1,<br>
consistent with Compound 47 being 9-deoxo-16-O-desmethyl-27-desmethoxy-39-<br>
desmethoxy rapamycin.<br>
Example 31<br>
Isolation of product from fermentation of S. hygroscopicus MG2-10[pSGsetKNOLh]<br><br>
9-Deoxo-16-Odesmethyl-27-O-desmethyl-39-O-desmethyl-rapamycin is<br>
obtained by conjugating S. hygroscopicus MG2-10 strain with pSGsetKN/OUs as<br>
described in Example 1 and isolating the products produced on fermentation. This<br>
demonstrates that it is possible to complement the deletion of rapK, rapN/O and rapL<br>
in the MG2-10 strain with the production of a rapamycin analogue lacking some post-<br>
PKS modification.<br>
S. hygroscopicus MG2-10 [pSGsetKN/OLhls] is fermented (see Materials and<br>
Methods), extracted and isolated using the method (B) as outlined in Materials and<br>
Methods.<br>
The isocratic solvent system used for preparative HPLC is 60% CH3CN/H20.<br>
9-Deoxo-16-O-desmethyl-27-O-desmethyl-39-O-desmethyl rapamycin<br>
(Compound 2) has the following characteristics:<br>
Molecular weight 858<br>
Molecular formula: C4BH75NO12<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MK* = 896, m/z for M-H = 856<br>
Example 32<br>
identification of product from fennentation ofS. hygroscopicus MG2-<br>
10[pSGsetKJNOLhJ<br>
16-O-Desmethyl-27-O-desmethyl-39-O-desmethyl-rapamycin was obtained<br>
by conjugating S. hygroscopicus MG2-10 strain with pSGsetKJN/OLhte as described<br>
in example 1 and analysing the products produced on fermentation. This<br>
demonstrated that it was possible to complement the deletion of rapK, rapJ, rapN/O<br>
and rapL in the MG2-10 strain with the production of a rapamycin analogue lacking<br>
some post-PKS modification.<br>
The fermentation broth (1 mL) was treated as described in the extraction,<br>
isolation and analysis Method (B) described in Materials and Methods. The HPLC<br>
chromatogram (280 nm) contained a peak that had the characteristic rapamycin<br>
triene (268 nm, 278 nm, 288 nm). This peak was not observed in the chromatogram<br>
of the control sample extracted from S. hygroscopicus MG2-10 in the absence of the<br>
cassette. LCMS (see Materials and Methods, Method B) of the novel rapamycin<br>
analogue peak gave ions m/z 895 (MNa*) and 871 (M-H). These ions confirm that the<br>
molecular weight of the novel rapamycin analogue is 872, 30 mass units greater than<br><br>
9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl rapamycin (Compound 1),<br>
consistent with oxidation at C9 (rapJ) and hydroxylation at C27 (rapN/O). These data<br>
are consistent with the compound being 16-O-desmethyl-27-O-desmethyl-39-O<br>
desmethyl rapamycin (Compound 7).<br>
Example 33<br>
Isolation of product from fermentation ofS. hygroscopicus MG2-10[pSGsetKJNOLh]<br>
16-O-Desmethyl-27-O-desmethyl-39-O-desmethyl-rapamycin is obtained by<br>
conjugating S. hygroscopicus MG2-10 strain with pSGsetKJN/OLws as described in<br>
Example 1 and isolating the products produced on fermentation. This demonstrates<br>
that it is possible to complement the deletion of rapK, rapJ, rapN/O and rapL in the<br>
MG2-10 strain with the production of a rapamycin analogue lacking some post-PKS<br>
modification.<br>
S. hygroscopicus MG2-10 [pSGsetKJN/OUis] is fermented (see Materials and<br>
Methods), extracted and isolated using the method (B) as outlined in Materials and<br>
Methods.<br>
The isocratic solvent system used for preparative HPLC is 60% CH3CN/H20,<br>
16-O-Desmethyl-27-O-desmethyl-39-O-desmethyl rapamycin (Compound 7)<br>
has the following characteristics:<br>
Molecular weight 872<br>
Molecular formula: C^H^NO-is<br>
UV (by diode array detection during HPLC analysis): 268 nm, 278 nm, 288<br>
nm<br>
Electrospray MS: m/z for MNa* = 895, m/z for M-H = 871<br>
Example 34<br>
Identification of product from fermentation of S. hygroscopicus MG2-10<br>
[pSGsetKIJNOQLh]<br>
16-O-Desmethyl-rapamycin was obtained by conjugating S. hygroscopicus<br>
MG2-10 strain with pSGsetKUN/OQLKs as described in example 1 and analysing the<br>
products produced on fermentation. This demonstrated that it was possible to<br>
complement the deletion of rapK, rapl, rap J, rapN/O, rapQ and rapL in the MG2-10<br>
strain with the production of a rapamycin analogue lacking methylation at C16-OH.<br>
In addition, it clearly identified RapQ as the SAM-dependent O-methyltransferase<br>
responsible for methylation of C27-OH.<br><br>
S. hygrvscopicus MG2-10 [pSGsetKIJN/OQUiJ was fermented (see Materials<br>
and Methods), extracted and analysed using the method (B) as outlined in Materials<br>
and Methods.<br>
The fermentation broth (1 mL) was treated as described in Materials and<br>
Methods. The HPLC chromatogram (280 nm) contained a peak that had the<br>
characteristic rapamycin triene (268 nm, 278 nm, 288 nm). This peak was not<br>
observed in the chromatogram of the control sample extracted from S. hygroscopicus<br>
MG2-10 in the absence of the cassette. LCMS (see Materials and Methods) of the<br>
novel rapamycin analogue peak gave ions m/z 923 (MNa*) and 899 (M-H). These<br>
ions confirm that the molecular weight of the novel rapamycin analogue is 900,14<br>
mass units less than rapamycin. It has already been established that the only post-<br>
PKS gene not included in the cassette, rapM, acts to methylate the C16-OH, hence<br>
the novel rapamycin analogue is 16-O-desmethyl rapamycin (Compound 20) and<br>
rapQ is shown to be functional and acting to O-methylate at C27.<br>
Example 35<br>
Bioassay of rapamycin analogues:<br>
(1) = 9-deoxo-16-O-desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin<br>
(pre-rapamycin)<br>
(6) = 9-deoxc-16-O-desmethyl-27-desmethoxy-rapamycin<br>
(16)= 9-deoxo-27-desmethoxy-rapamycin,<br>
(3) = 16-O-desmethyl-27-desmethoxy-39-O-desmethyl-rapamycin<br>
(9) = 9-deoxo-16-Odesmethyl-27-O-desmethyl-rapamycin<br>
(8) = 9-deoxo-27-O-desmethyl-39-O-desmethyl-rapamycin.<br>
Cancer Cell Lines:<br>
-Growth inhibition of adherent human tumour cell lines of solid malignancies<br>
HT29 (colon) and MCF-7 (breast) was tested in vitro using an MTT (3-(4,5-<br>
dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide) assay using micro-titre plates<br>
(Sieuwerts, A.M., et al., 1995). All cell lines were obtained from either the ATCC<br>
(American Type Culture Collection) or ECACC (European Collection of Cell<br>
Cultures). All cell lines were grown from frozen stocks and passaged at least once<br>
prior to use in RPM11640. Cells were harvested from sub-confluent cultures using<br>
minimal trypsinization. Cells were diluted to the appropriate density for each cell line<br>
(dependent on cell doubling time) in RPM11640, and seeded in 60 wells of a 96 well<br>
plate in a volume of 100 µl per well (i.e. outside wells of the plate were not used).<br><br>
Plates were incubated at 37°C overnight Following this incubation, log scale<br>
dilutions of reference and test substances were added in 100 µl per well, 6 replicates<br>
were used to test all test compounds, reference compounds and medium controls.<br>
Plates were incubated for a further 72 h prior to analysis. MTT (5 mg/ml) was added<br>
to each well and plates were re-incubated for 3-4 h. Unreacted MTT was removed<br>
from the wells and formazan crystals formed from the MTT were dissolved in DMSO<br>
and characteristic absorbance read at 570 nm. The concentration (nM) of each test<br>
compound and reference compound, which resulted in 50% of maximum inhibition<br>
(ICso),, was calculated for each cell line and quoted along with the maximum<br>
percentage of inhibition observed (lm), see Table XIII. For reference, rapamycin has<br>
an ICsoOf 200nM and an lmof 40% in the HT-29 cell line and an IC50of 0.03nM and an<br>
!m of 56% in the MCF-7 cell line.<br><br>
Mixed Lymphocyte Reaction (MLR):<br>
Originally developed to assess tissue compatibility prior to allografts, MLR<br>
offers an established model for immune reaction in vitro (SOULILLOU, J.P., et al.<br>
(1975); T. Meo. "Immunological Methods", L Lefkovits and B.Pernis, Eds., Academic<br>
Press, N.Y. pp. 227-239 (1979). MLR was performed by mixing splenic lymphocytes<br>
isolated from C57BU6 mice (5x10s cells) with inhibited splenic lymphocytes from<br>
CBA mice (2.5x105 cells). The inhibited CBA lymphocytes induced a proliferative<br>
response in C57BL/6 lymphoctes and this was determined by [3H] thymidine<br>
incorporation into DNA as a measure of proliferation of splenic lymphocytes isolated<br>
from C57BL/6 mice. The anti-proliferative effect was assayed for in the presence of<br>
log scale dilutions of reference compounds, test compounds and media controls over<br>
a 72 h period at 37 °C. The concentration of each test compound and reference<br>
compound, which inhibited lymphocyte proliferation by 50% (IC50), compared to<br>
control proliferation, was calculated for each cell line and quoted as a ratio of the<br><br>
concentration of rapamycin required to inhibit lymphocyte proliferation by 50% (rlC50),<br>
see Table XIV.<br><br>
Anti-fungal Assay:<br>
The comparative anti-fungal activities of reference and test compounds were<br>
determined against pathogenic fungi Candida albicans DSM 5816, Candida albicans<br>
DSM 1386 and Candida glabrata DSM 11226. This was achieved using a microtitre<br>
plate adaption of the NCCLS Reference Method for Broth Dilution Antifungal<br>
Susceptibility Testing for Yeasts: Approved Standard (M27-A, vol. 17 No. 9. (1997)).<br>
Yeast strains were inoculated (104 cfu/ml) to RPM11640 media containing 0.165 mM<br>
MOPS, pH 7. Growth was determined in the presence of log scale dilutions of<br>
reference compounds, test compounds and media controls after incubation with<br>
shaking at 37 °C, 24 h. Minimum inhibitory concentration (MIC) and minimum<br>
fungicidal activity (MFC) were determined for test compounds and expressed as a<br>
ratio of the rapamycin minimum inhibitory concentration (rMIC respectively), see<br>
Table XV.<br><br>
References<br>
Alarcon, CM., Heitman, J., and Cardenas, M.E. (1999) Protein kinase activity and<br>
identification of a toxic effector domain of the target of rapamycin TOR<br><br>
proteins in yeast Molecular Biology of the Cell 10:2531-2546.<br>
Aparicio, J.F., Molnar, I., Schwecke, T., Konig, A., Haydock, S.F., Khaw, L.E.,<br>
Staunton, J., and Leadlay, P.F. (1996) Organization of the biosynthetic gene<br>
cluster for rapamycin in Streptomyces hygroscopicus: analysis of the<br>
enzymatic domains in the modular polyketide synthase. Gene 169: 9-16.<br>
Baker, H., Sidorowicz, A., Sehgal, S.N., and Vezina, C. (1978) Rapamycin (AY-<br>
22,989), a new antifungal antibiotic. III. In vitro and in vivo evaluation.<br>
Journal of Antibiotics 31: 539-545.<br>
Bierman, M., Logan, R., O'Brien, K., Seno, E.T., Nagaraja Rao, R., and Schoner,<br>
B.E. (1992) Plasmid cloning vectors for the conjugal transfer of DNA from<br>
Escherichia colito Streptomyces spp. Gene 116:43-49.<br>
Blanc, V., Lagneaux, D., Didier, P., Gil, P., Lacroix, P., and Crouzet, J. (1995)<br>
Cloning and analysis of structural genes from Streptomyces pristinaespiralis<br>
encoding enzymes involved in the conversion of pristinamycin llB to<br>
pristinamycin llA (PHA): PHA synthase and NADH:riboflavin 5'-phosphate<br>
oxidoreductase. Journal of Bacteriology ATI: 5206-5214.<br>
Blanc, V., Gil, P., Bamas-Jacques, N., Lorenzon, S., Zagorec, M., Schleuniger, J.,<br>
Bisch, D., Blanche, F., Debussche, L, Crouzet, J., and Thibaut, D. (1997)<br>
Identification and analysis of genes from Streptomyces pristinaespiralis<br>
encoding enzymes involved in the biosynthesis of the 4-dimethylamino-L-<br>
phenylalanine precursor of pristinamycin I. Molecular Microbiology 23:191-<br>
202.<br>
Box, S.J., Shelley, P.R., Tyler, J.W., Verrall, M.S., Warr, S:R.C, Badger, A.M., Levy,<br>
M.A., and Banks, R.M. (1995) 27-O-Demethylrapamycin, an<br>
immunosuppressant compound produced by a new strain of Streptomyces<br>
hygroscopicus. Journal of Antibiotics 48:1347-1349. -<br>
Brown, E.J., Albers, M.W., Shin, T.B., Ichikawa, K., Keith, C.T., Lane, W.S., and<br>
Schreiber, S.L (1994) A mammalian protein targeted by G1-arresting<br>
rapamycin-receptor complex. Nature 369:756-758.<br>
Brunn, G.J., Williams, J., Sabers, C, Wiederrecht G., Lawrence, J.C., and Abraham,<br>
R.T. (1996) Direct inhibition of the signaling functions of the mammalian<br>
target of rapamycin by the phosphoinosttide 3-kinase inhibitors, wortmannin<br>
and LY294002. EMBO Journal 15: 5256-5267.<br>
Cao, W., Mohacsi, P., Shorthouse, R., Pratt, R. and Morris, R.E. (1995). Effects of<br>
rapamycin on growth factor-stimulated vascular smooth muscle cell DNA<br><br>
synthesis. Inhibition of basic fibroblast growth factor and platelet-derived<br>
growth factor action and antagonism of rapamycin by FK506. Transplantation<br>
59(3): 390-395.<br>
Carlson, R.P., Hartman, DA, Tomchek, LA, Walter, T.L, Lugay, J.R., Calhoun, W.,<br>
Sehgal, Si.N., Chang, J.Y. (1993). Rapamycin, a potential disease-modifylng<br>
antiarthritic drug. J. Pharmacol. Exp. Ther. 266(2): 1125-38.<br>
Chambraud, B., Radanyl, C, Camonis, J.H., Shazand, K., Rajkowskt, K., and<br>
Baulieu, E.E. (1996) FAP48, a new protein that forms specific complexes both<br>
immunophilins FKBP59 and FKBP12. Prevention by the immunosuppressant<br>
drugs FK506 and rapamycin. Journal of Biological Chemistry 271: 32923-<br>
32929.<br>
Chang, J.Y., Sehgal, S.N., and Bansbach, C.C. (1991) FK506 and rapamycin: novel<br>
pharmacological probes of the immune response. Trends in Pharmacological<br>
Sciences 12:218-223.<br>
Chen, J., Zheng, X.F., Brown, E.J., and Schreiber, S.L (1995) Identification of an 11-<br>
kDa FKBP12-rapamycin-binding domain within the 289-kDa FKBP12-<br>
rapamycin-associated protein and characterization of a critical serine residue.<br>
Proceedings of the National Academy of Sciences of the United States of<br>
America 92:4947-4951.<br>
Chini, M., Crotti, P., Gardelli, C, and Macchia, F., (1992), Tetrahedron, 48, 3805-<br>
3812<br>
Choi, J.W., Chen, J., Schreiber, S.L, and Clardy, J. (1996) Structure of the FKBP12-<br>
rapamycin complex interacting with the binding domain of human FRAP.<br>
Science 273: 239-242.<br>
Chung, L, Liu, L, Patel, S., Carney, J.R., and Reeves, CD. (2001) Deletion of<br>
rapQNML from the rapamycin gene cluster of Streptomyces hygroscopicus<br>
gives production of the 16-O-desmethyl-27-desmethoxy analog. Journal of<br>
Antibiotics 54:250-256.<br>
Corey, E, J. and Huang, H., (1989) Tetrahedron Lett., 30, 5235-5238<br>
DiLella, AG., and Craig, R.J. (1991) Exon organization of the human FKBP-12 gene:<br>
correlation with structural and functional protein domains. Biochemistry 30:<br>
8512-8517.<br>
Du, L.C., Sanchez, C, Chen, M., Edwards, D.J., and Shen, B. (2000) The<br>
biosynthetic gene cluster for the antitumor drug bleomycin from Streptomyces<br>
verticiilus ATCC15003 supporting functional interactions between<br><br>
nonribosomal peptide synthetases and a polyketide synthase. Chemistry &amp;<br>
Biology 7: 623-642.<br>
Dudkin, L, Dilling, M.B., Cheshire, P.J., Harwood, F.C., Hollingshead, M., Arbuck,<br>
S.G., Travis, R., Sausville, E.A., Houghton, P.J. (2001). Biochemical<br>
correlates of mTOR inhibition by the rapamycin ester CCI-779 and tumor<br>
growth inhibition. Clin. Cancer Res. 7(6):1758-64<br>
Fehr, T., Sanglier, J-J., Schuler, W., Gschwind, L, Ponelle, M., Schilling, W.,<br>
Wioland, C. (1996). Antascomicinc A, B, C, D and E: Novel FKBP12 binding<br>
compounds from a Micromonospora strain. J. Antibiot. 49(3): 230-233.<br>
Ferrari, S., Pearson, R.B., Siegmann, M., Kozma, S.C., and Thomas, G. (1993) The<br>
immunosuppressant rapamycin induces inactivation of P70s6k through<br>
dephosphorylation of a novel set of sites. Journal of Biological Chemistry 268:<br>
16091-16094.<br>
Findiay J A, and Radios, L (1980) Canadian Journal of Chemistry 58:579.<br>
Fishbein, T.M., Florman, S., Gondolesi, G., Schiano, T., LeLeiko, N., Tschernia, A.,<br>
Kaufman, S. (2002). Intestinal transplantation before and after the introduction<br>
of sirolimus.	—<br>
Transplantation. 73(10): 1538-42.<br>
Foey, A., Green, P., Foxwell, B., Feldmann, M., Brennan, F. (2002). Cytokine-<br>
stimulated T cells induce macrophage IL-10 production dependent on<br>
phosphatidylinositol 3-kinase and p70S6K: implications for rheumatoid<br>
arthritis. Arthritis Res. 4(1):64-70. Epub 2001 Oct 10.<br>
Gaisser, S., Reather, J., Wirtz, G., Kellenberger, L., Staunton, J., and Leadlay, P.F.<br>
(2000) A defined system for hybrid macrolide biosynthesis in<br>
Saccharopolyspora erythraea. Molecular Microbiology 36: 391-401.<br>
Gaisser,.S., Lill, R., Staunton, J., Mendez, C, Salas, J., Leadlay, PF. (2002) Parallel<br>
pathways for oxidation of 14-membered polyketide macrolactones in<br>
Saccharopolyspora erythraea. Mol Microbiol 44:771-81.<br>
Galat, A. (2000) Sequence diversification of the FK506-binding proteins in several<br>
different genomes. European Journal of Biochemistry 267: 4945-4959.<br>
Gregory, C.R., Huie, P., Billingham, M.E. and Morris, R.E. (1993). Rapamycin<br>
inhibits arterial intimal thickening caused by both alloimmune and mechanical<br>
injury. Its effect on cellular, growth factor and cytokine response in injured<br>
vessels. Transplantation 55(6): 1409-1418.<br>
Gregory MA, Till R1 Smith, MCM. (in Press) integration site for Streptomyces phage<br><br>
c|&gt;BT1 and the development of site-specific integrating vectors. J Bacteriol.<br>
Guba, M., von Breitenbuch, P., Steinbauer, M., Koehl, G., Flegel, S., Hornung, M.,<br>
Bruns, C.J., Zuelke, C, Farkas, S., Anthuber, M., Jauch, K.W., and Geissler,<br>
E.K. (2002) Rapamycin inhibits primary and metastatic tumor growth by<br>
antiangiogenesis: involvement of vascular endothelial growth factor. Nature<br>
Medicine 8:128-135.<br>
Hamilton, G.S., and Steiner, J. P. (1998) Immunophilins: Beyond<br>
immunosuppression. Journal of Medicinal Chemistry 41: 5119-5143.<br>
Hara, K., Yonezawa, K., Kozlowski, M.T., Sugimoto, T., Andrabi, K., Weng, Q.P.,<br>
Kasuga, M., Nishimoto, I., and Avruch, J. (1997) Regulation of elF-4E BP1<br>
phosphorylation by mTOR. Journal of Biological Chemistry 272: 26457-<br>
26463.<br>
Hardwick, J.S., Kuruvilla, F.G., Tong, J.K., Shamji, A.F., and Schreiber, S.L. (1999)<br>
Rapamycin-modulated transcription defines the subset of nutrient-sensitive<br>
signaling pathways directly controlled by the Tor proteins. Proceedings of the<br>
National Academy of Sciences of the United States of America 96:14866-<br>
14870.<br>
Hatanaka, H., Kino, T., Miyata, S., Inamura, N., Kuroda, A., Goto, T., Tanaka, H.,<br>
Okuhara, M. (1988). FR-900520 and FR-900523, novel immunosuppressants<br>
isolated from a Streptomyces. II. Fermentation, isolation and physico-<br>
chemical and biological characteristics.i/. Antibiot. (Tokyo). 41(11): 1592-601.<br>
Hatanaka H, Kino T, Asano M, Goto T, Tanaka H, Okuhara M. (1989). FK-506<br>
related compounds produced by Streptomyces tsukubaensis No. 9993. J.<br>
Antibiot (Tokyo). 42(4):620-2.<br>
Hendrickson, BA, Zhang, W., Craig, R.J., Jin, Y.J., Bierer, B.E., Burakoff, S., and<br>
DiLella, A.G. (1993) Structural organization of the genes encoding human and<br>
murine FK506-binding protein (FKBP)13 and comparison to FKBP1. Gene<br>
134: 271-275.<br>
Hentges, K.E., Sirry, B., Gingeras, A.C., Sarbassov, D., Sonenberg, N., Sabatini, D.,<br>
and Peterson, A.S. (2001) FRAP/mTOR is required for proliferation and<br>
patterning during embryonic development in the mouse. Proceedings of the<br>
National Academy of Sciences of the United States of America 98:13796-<br>
13801.<br>
Hopwood, D.A. (1997) Genetic contributions to understanding polyketide synthases.<br><br>
Hosted, T.J., and Baltz, R.H. (1997) Use of rpsL for dominance selection and gene<br>
replacement in Streptomyces roseosporus. Journal of Bacteriology MS: 180-<br>
186.<br>
Hung, D.T.,-and Schreiber, S.L (1992) cDNA cloning of a human 25 kDa FK506 and<br>
rapamycin binding protein. Biochemical and Biophysical Research<br>
Communications 184:733-738.<br>
Hung, D.T., Jamison, T.F., and Schreiber, S.L. (1996) Understanding and controlling<br>
the cell cycle with natural products. Chemistry &amp; Biology 3: 623-639.<br>
Jain, S., Bickneli, G.R., Whiting, P.H., Nicholson, M.L (2001). Rapamycin reduces<br>
expression of fibrosis-associated genes in an experimental model of renal<br>
ischaemia reperfusion injury. Transplant Proc. 33(1-2):556-8.<br>
Jin, Y.J., Burakoff, S.J., and Bierer, B.E. (1992) Molecular cloning of a 25-kDa high<br>
affinity rapamycin binding protein, FKBP25. Journal of Biological Chemistry<br>
287:10942-10945.<br>
Kahan, B.D., Chang, J.Y., and Sehgal, S.N. (1991) Preclinical evaluation of a new<br>
potent immunosuppressive agent, rapamycin. Transplantation 52:185-191.<br>
Kahan, B.D., and Camardo, J.S. (2001) Rapamycin: Clinical results and future<br>
opportunities. Transplantation 72:1181 -1193.<br>
Kallen, J. A., Sedrani, R., and Cottens S. (1996) X-ray crystal structure of 28-O-<br>
methylrapamycin complexed with FKBP12: Is the cyclohexyl moiety part of<br>
the effector domain of rapamycin? Journal of the American Chemical Society<br>
118: 5857-5861.<br>
Kawasome, H., Papst, P., Webb, S., Keller, G.M., Johnson, G.L, Gelfand, E.W., and<br>
Terada, N. (1998) Targeted disruption of p70s8k defines its role in protein<br>
synthesis and rapamycin sensitivity. Proceedings of the National Academy of<br>
Sciences of the United States of America 95: 5033-5038.<br>
Khaw, C.E., Bohm, GA, Metcalfe, S., Staunton, J., and Leadlay, P.F. (1998)<br>
Mutational biosynthesis of novel rapamycins by a strain of Streptomyces<br>
hygroscopicus NRRL 5491 disrupted in rapL, encoding a putative lysine<br>
cyclodeaminase. Journal of Bacteriology 180: 809-814.<br>
Kieser, T., Bibb, M.J., Buttner, M.J., Chater, K.F., and Hopwood, DA (2000)<br>
Practical Streptomyces Genetics, John Innes Foundation, Norwich.<br>
Kirby, B., and GrifRths,C.E.M. (2001) Psoriasis: the future. British Journal of<br>
Dermatology 144:37-43.<br>
Kirchner, G.I., Winkler, M., Mueller L, Vidal, C, Jacobsen, W., Franzke, A., Wagner,<br><br>
S., Blick, S., Manns MP., and Sewing K.-F.(2000) Pharmacokinetics of SDZ<br>
RAD and cyclosporin including their metabolites in seven kidney graft patients<br>
after the first dose of SDZ RAD. British Journal of Clinical Pharmacology<br>
50:449-454.-<br>
Konig, A., Schwecke, T., Molnar, I., Bohm, G., Lowden, PAS., Staunton, J., and<br>
Leadlay, P.F. (1997) The pipecolate-incorporating enzyme for the<br>
biosynthesis of the immunosuppressant rapamycin. Nucleotide sequence<br>
analysis, disruption and heterologus expression of rapPfrom Streptomyces<br>
hygroscopicus. European Journal of Biochemistry 247: 526-534.<br>
Kunz, J., Loeschmann, A, Deuter-Reinhard, M., and Hall, M.N. (2000) FAP1, a<br>
homologue of human transcription factor NF-X1, competes with rapamycin for<br>
binding to FKBP12 in yeast. Molecular Microbiology 37:1480-1493.<br>
Kuo, C.J., Chung, J.K., Fiorentino, D.F., Flanagan, W.M., Blenis, J., and Crabtree,<br>
G.R. (1992) Rapamycin selectively inhibits interleukin-2 activation of p70 S6<br>
kinase. Nature 358:70-73.<br>
Lee, M.H. Pascopella, L, Jacobs, W.R., Jr and Hatfull, G.F (1991). Site specific<br>
integration of mycobacteriophage L5: integration-proficient vectors for<br>
Mycobacterium smegmatis, Mycobacterium tuberculosis and Bacille<br>
Calmette-Guerin: Prpc. Natl. Acad. Sci. USA, 88:3111-3115.<br>
Lee MH, Pascopella L, Jacobs WR Jr, Hatfull GF. (1991), Site-specific integration of<br>
mycobacteriophage L5: integration-proficient vectors for Mycobacterium<br>
smegmatis, Mycobacterium tuberculosis, and bacille Calmette-Guerin. Proc<br>
Natl Acad Sci USA; 88:3111-5.<br>
Liang, J., Choi, J., and Clardy, J. (1999) Refined structure of the FKBP12-rapamycin-<br>
FRB ternary complex at 2.2 A resolution. Acta Crystallographica Section D-<br>
Biological Crystallography 55:736-744.	-<br>
Lomovskaya, N., Fonstein, L., Ruan, X., Stassi, D., Katz, L, and Hutchinson, C.R.<br>
(1997) Gene disruption and replacement in the rapamycin-producing<br>
Streptomyces hygroscopicus strain ATCC 29253. Microbiology-Uk 143: 875-<br>
883.<br>
Lowden, P.A.S., Bohm, G., Staunton, J., and Leadlay, P.F. (1996) The nature of the<br>
starter unit for the rapamycin polyketide sythase. Angewandte Chemie 35:<br>
2249-2251.<br>
Lowden, P. A. S., (1997) Ph.D. Dissertation, University of Cambridge. "Studies on the<br>
biosynthesis of rapamycin".<br><br>
Lowden. PAS., Wilkinson, B., Bohm, GA, Handa, S., Floss, H.G., Leadlay, P.F.,<br>
and Staunton, J. (2001) Origin and true nature of the starter unit for the<br>
rapamycin polyketide synthase. Angewandte Chemie-lnternational Edition 40:<br>
777-779.<br>
Luengo J.!., Yamashita,.D.S., Dunnington, D., Beck, AK., Rozamus, L.W., Yen,<br>
H.K., Bossard, M.J., Levy, M.A, Hand, A, Newmantarr, T., Badger, A.,<br>
Faucette, L, Johnson, R.K., Dalessio, K., Porter, T., Shu, A.Y.L, Heys, R.,<br>
Choi, J.W., Kongsaeree, P., Clardy, J., and Holt, DA (1995) Structure-<br>
Activity Studies of Rapamycin Analogs - Evidence That the C-7 Methoxy<br>
Group Is Part of the Effector Domain and Positioned at the Fkbp12-Frap<br>
Interface. Chemistry &amp; Biology 2:471-481.<br>
Lyons, W.E., George, E.B., Dawson, T.M., Steiner, J.P., and Snyder, S.H. (1994)<br>
Immunosuppressant FK506 promotes neurite outgrowth in cultures of RC12<br>
cells and sensory ganglia. Proceedings of the National Academy of Sciences<br>
of the United States of America 91:3191-3195.<br>
MacNeil, D.J., Gewain, K.M., Ruby, C.L, Dezeny, G., Gibbons, P.H., and MacNeil, T.<br>
(1992) Analysis of Steptomyces avermitilis genes required for avermectin<br>
biosynthesis utilizing a novel integration vector. Gene 111: 61-68.<br>
Marahiel, MA, Stachelhaus, T., and Mootz, H.D. (1997) Modular peptide<br>
. synthetases involved in nonribosomal peptide synthesis. Chemical Reviews<br>
97: 2651-2673.<br>
Matsuura, M., Noguchi, T., Yamaguchi, D., Aida, T., Asayama, M., Takahashi, H. and<br>
Shirai, M. (1996). The sre gene (ORF469) encodes a site-specific<br>
recombinase responsible for integration of the R4 phage genome. J Bact.<br>
178(11):3374-3376.<br>
McAlpine, J. B,.Swanson S. J., Jackson, M., Whittern, D.N. (1991). Revised NMR<br>
"assignments for rapamycin. Journal of Antibiotics 44: 688-690.<br>
Meo, T. in "Immunological Methods", L Lefkovits and B. Pernis, Eds., Academic<br>
Press, N.Y. pp. 227-239 (1979).<br>
Molnar, I., Aparicio, J.F., Haydock, S.F., Khaw, L.E., Schwecke, T., Konig, A,<br>
Staunton, J., and Leadlay, P.F. (1996) Organisation of the biosynthetic gene<br>
cluster for rapamycin in Streptomyces hygroscopicus: analysis of genes<br>
flanking the polyketide synthase. Gene 169:1-7.<br>
Morice, M.C., Serruys, P.W., Sousa, J.E., Fajadet, J., Ban Hayashi, E., Perin, M.,<br>
Colombo, A, Schuler, G., Barragan, P., Guagliumi, G., Molnar, F., Falotico,<br><br>
R. (2002). RAVEL Study Group. Randomized Study with the Siroiimus-<br>
Coated Bx Velocity Balloon-Expandable Stent in the Treatment of Patients<br>
with de Novo Native Coronary Artery Lesions. A randomized comparison of a<br>
sirolimus-eluting stent with a standard stent for coronary revascularization. N.<br>
EngJJ. Med. 346(23): 1773-80.<br>
Motamedi, H., Shafiee, A., Cai, S.J., Streicher, S.L., Arison, B.H., and Miller, R.R.<br>
(1996) Characterization of methyltransferase and hydroxylase genes involved<br>
in the biosynthesis of the immunosuppressants FK506 and FK520. Journal of<br>
Bacteriology 178: 5243-5248.<br>
Motamedi, H., Cai, S.J., Shafiee, A., and Elliston, K.O. (1997) Structural organization<br>
of a multifunctional polyketide synthase involved in the biosynthesis of the<br>
macrolide immunosuppressant FK506. European Journal of Biochemistry<br>
244: 74-80.<br>
Motamedi, H., and Shafiee, A (1998) The biosynthetic gene cluster for the<br>
macrolactone ring of the immunosuppressant FK506. European Journal of<br>
Biochemistry 256: 528-534.<br>
Myckatyn, T.M., Ellis, RA, Grand, A.G., Sen, S.K., Lowe, J.B. 3rd, Hunter, DA,<br>
Mackinnon, S.E. (2002). The effects of rapamycin in murine peripheral nerve<br>
isografts and allografts.<br>
Plast. Reconstr. Surg. 109(7):2405-17.<br>
Nave, B.T., Ouwens, D.M., Withers, D.J., Alessi, D.R., and Sheperd, P.R. (1999)<br>
Mammalian target of rapamycin is a direct target for protein kinase B:<br>
identification of a convergence point for opposing effects of insulin and amino-<br>
acid deficiency on protein translation. Biochemical Journal 344:427-431.<br>
Navia, MA (1996) Protein-drug complexes important for immunoregulation and<br>
organ transplantation. Current Opinion in Structural Biology 6: 838-847.<br>
NCCLSTReference Method for Broth Dilution Antifungal Susceptibility Testing for<br>
Yeasts: Approved Standard M27-A, vol. 17 No. 9. (1997).<br>
Nishida, H., Sakakibara, T., Aoki, F., Sa'rto, T., Ichikawa, K., Inagaki, T., Kojima, Y.,<br>
Yamauchi, Y., Huang, L.H., Guadliana, MA, Kaneko, T., and Kojima, N.<br>
(1995) Generation of novel rapamycin structures by microbial manipulations.<br>
Journal of Antibiotics 48: 657-666.<br>
Nielsen, J.B., Hsu, M.J., Byrne, K.M., and Kaplan, L (1991) Biosynthesis of the<br>
immunosuppressant immunomycin: the enzymology of pipecolate<br>
incorporation. Biochemistry 30: 5789-5796.<br><br>
Paget, M.S.B., Chamberlin, L, Atrih, A., Foster, S.J., and Buttner, M.J. (1999)<br>
Evidence that the extracytoplasmic function sigma factor c^ is required for<br>
normal cell wall structure in Streptomyces coelicolor A3(2). Journal of<br>
Bacteriology 181:204-211)<br>
Paiva, N.L, Demain, A.L, and Roberts, M.F. (1991) Incorporation of acetate,<br>
propionate, and methionine into rapamycin By Streptomyces hygroscopicus.<br>
Journal of Natural Products 54:167-177.<br>
Paiva, N.L., Demain, A.L, and Roberts, M.F. (1993) The immediate precursor of the<br>
nitrogen-containing ring of rapamycin is free pipecolic acid. Enzyme and<br>
Microbial Technology 15:581-585.<br>
Patterson, C.E., Schaub, T., Coleman, E.J., and Davies E.C. (2000) Developmental<br>
regulation of FKBP65. An ER-localized extracellular matrix binding-protein.<br>
Molecular Biology of the Cell 11:3925-3935.<br>
Pfeifer, BA, Admiraal, S.J., Gramajo, H., Cane, D.E., and Khosla, C. (2001)<br>
Biosynthesis of complex polyketides in a metabolically engineered strain of £.<br>
co//. Science 291:1790-1792.<br>
Powell, N., Till, S., Bungre, J., Corrigan, C. (2001). The immunomodulatory drugs<br>
cyclosporin A, mycophenolate mofetil, and sirolimus (rapamycin) inhibit<br>
allergen-induced proliferation and IL-5 production by PBMCs from atopic<br>
asthmatic patients.<br>
J. Allergy Clin. Immunol. 108(6):915-7<br>
Rabinovitch, A., Suarez-Pinzon, W.L., Shapiro, A.M., Rajotte, R.V., Power, R. (2002).<br>
Combination therapy with sirolimus and interleukin-2 prevents spontaneous<br>
and recurrent autoimmune diabetes in NOD mice.Diabetes. 51(3):638-45.<br>
Raught, B., Gingras, A.C., and Sonenberg, N. (2001) The target of rapamycin (TOR)<br>
proteins. Proceedings of the National Academy of Sciences of the United<br>
States of America 98:7037-7044.<br>
Rawlings, B.J. (2001) Type I polyketide. biosynthesis in bacteria (Part A). Natural<br>
Product Reports 18:190-227.<br>
Reather, J. A., (2000), Ph.D. Dissertation, University of Cambridge. "Late steps in<br>
the biosynthesis of macrocyclic lactones".<br>
Re'rtamo, S., Spuls, P., Sassolas, B., Lahfa, M., Claudy, A., Griffiths, C.E.; Sirolimus<br>
European Psoriasis Study Group. (2001). Efficacy of sirolimus (rapamycin)<br>
administered concomitantly with a subtherapeutic dose of cyclosporin in the<br>
treatment of severe psoriasis: a randomized controlled trial. Br. J. Dermatol.<br><br>
145(3):438-45.<br>
Rosen, M.K., and Schreiber, S.L (1992) Natural products as probes of cellular<br>
function: studies of immunophilins. Angewandte Chemie-lnternational Edition<br>
in English 31: 384-400.<br>
Roymans, D., and Siegers, H. (2001) Phosphaditidylinositol 3-kinases in tumor<br>
progression. European Journal of Biochemistry 268:487-498.<br>
Ruan, XA, Stass, D., Lax, SA, and Katz, L (1997) A second type-l PKS gene<br>
cluster isolated from Streptomyces hygroscopicus ATCC 29253, a rapamycin-<br>
producing strain. Gene 203:1-9.<br>
Salituro, G.M., Zink, D.L, Dahl, A., Nielsen, J., Wu, E., Huang, L, Kastner C,<br>
Dumont, F. (1995) Meridamycin: a novel nonimmunosuppressive FKBP12<br>
ligandfrom Streptomyces hygroscopicus. Tetrahydron letters 36: 997-1000.<br>
Schwarzer, D., and Marahiel, MA. (2001) Multimodular biocatalysts for natural<br>
product assembly. Naturwissenschaften 88: 93-101.<br>
Sambrook, J., Fritsch. E.F., and Maniatis, T. (1989) Molecular cloning: a laboratory<br>
manual, 2nd ed. Cold Spring Harbor Laboratory Press, N.Y.<br>
Schreiber, S.L, and Crabtree, G.R. (1992) The mechanism of action of cyclosporine<br>
A and FK506. Immunology Today 13:136-142.<br>
Schwecke, T., Aparicio, J.F., Molnar, l., K6nig, A., Khaw, L.E., Haydock, S.F.,<br>
Oliynyk, M., Caffrey, P., Cortes, J., Lester, J.B., Bohm, G.A., Staunton, J.,<br>
• and Leadlay, P.F. (1995) The biosynthetic gene cluster for the polyketide<br>
immunosuppressant rapamycin. Proceedings of the National Academy of<br>
Sciences of the United States of America 92: 7839-7843.<br>
Sedrani, R., Cottens, S., Kallen, J., and Schuler, W. (1998) Chemical modifications of<br>
rapamycin: the discovery of SDZ RAD. Transplantation Proceedings 30:<br>
2192-2194.	'	£	•	■<br>
Sehgal,"S.N., Baker, H., and Vezina, C. (1975) Rapamycin (AY-22,989), a new<br>
antifungal antibiotic II. Fermentation, isolation and characterization. The<br>
Journal of Antibiotics 28:727-733.<br>
Shepherd, P.R, Withers, D.J., and Siddle K. (1998) Phosphoinositide 3-kinase: the<br>
key switch mechanism in insulin signalling. Biochemical Journal 333:471-<br>
490.<br>
Shima, J., Hesketh, A., Okamoto, S., Kawamoto, S., and Ochi, K. (1996) Induction of<br>
actinorhodin production by rpsL (encoding ribosomal protein S12) mutations<br>
that confer streptomycin resistance in Streptomyces lividans and<br><br>
Streptomyces coelicolor A3(2). Journal of Bacteriology 178: 7276-7284.<br>
Sigal, N.H., and Dumont, F.J. (1992) Cyclosporin A, FK-506, and rapamycin:<br>
pharmacological probes of lymphocyte signal transduction. Annual Review of<br>
Immunology 10:519-560.<br>
Sieuwerts, A.M., Klijn, J.G., Peters, H.A., Foekens, J.A. (1995). The MTT tetrazolium<br>
salt assay scrutinized: how to use this assay reliably to measure metabolic<br>
activity of cell cultures in vitro for the assessment of growth characteristics,<br>
IC50-values and cell survival. Eur. J Clin. Chem. Clin. Biochem. 33(11):813-<br>
23.<br>
Smovkina, T., Mazodier, P., Boccard, F., Thompson, CJ. and Guerineau, M. (1990)<br>
Construction of a series of pSAM2-based integrative vectors for use in<br>
actinomycetes. Gene 94:53-59.<br>
SOULILLOU, J.P., CARPENTER, C.B., LUNDIN, A.P. and STROM, T.B. (1975)<br>
Augmentation of proliferation and in vitro production of cytotoxic cells by 2-ME<br>
in the rat J Immunol. 115(6): 1566-71.<br>
Staunton, J., and Weissman, K.J. (2001) Polyketide biosynthesis: a millennium<br>
review. Natural Product Reports 18:380-416.<br>
Steiner, J.P., Hamilton, G.S., Ross, D. T., Valentine, H.L, Guo, H., Connolly, M.A.,<br>
Liang, S., Ramsey, C, Li, J.-H.J., Huang, W., Howorth, P., Soni, R., Fuller,<br>
M., Sauer, H., Nowotnik, A.C., and Suzdak, P.D. (1997) Neutrophic<br>
immunophiiin ligands stimulate structural and functional recovery in<br>
neurodegenerative animal models. Proceedings of the National Academy of<br>
Sciences of the United States of America 94:2019-2024.<br>
Tang, S.J., Reis, G., Kang, H., Gingras, A-C., Sonenberg, N., and Schuman, E.M.<br>
(2002) A rapamycin-sensitive signaling pathway contributes to long-term<br>
synaptic plasticity in the hippocampus. Proceedings ofthe National Academy<br>
of Sciences of the United States of America 1:467-472.<br>
Van Duyne, G.D., Standaert, R.F., Karplus, PA, Schreiber, S.L,.and Clardy, J.<br>
(1993) Atomic structures of the human immunophiiin FKBP-12 complexes<br>
with FK506 and rapamycin. Journal of Molecular Biology 229:105-124.<br>
Van Mellaert L, Mei, L, Lammertyn, E., Schacht, S., and Anne, J. (1998) Site-<br>
specific integration of bacteriophage VWB genome into Streptomyces<br>
venezuelae and construction of a VWB-based integrative vector. Microbiology<br>
144:3351-3358.<br>
Vezina, C, Kudelski, A., and Sehgal, S.N. (1975) Rapamycin (AY-22,989), a new<br><br>
antifungal antibiotic. I. Taxonomy of the producing streptomycete and<br>
isolation of the active principle. 777e Journal of Antibiotics 28: 721-726.<br>
Vilella-Bach, M., Nuzzi, P., Fang, Y.M., and Chen, J. (1999) The FKBP12-rapamycin-<br>
binding domain is required for FKBP12-rapamycin-associated protein kinase<br>
activity.and Gi progression. Journal of Biological Chemistry 274:4266-4272.<br>
Waller, J.R., and Nicholson, M.L (2001) Molecular mechanisms of renal allograft<br>
fibrosis. British Journal of Surgery 88:1429-1441.<br>
Warner, L.M., Adams, L.M., Chang, J.Y., Sehgal, S.N. (1992). A modification of the<br>
in vivo mixed lymphocyte reaction and rapamycin's effect in this model.Clin.<br>
Immunol. Immunopathol. 64(3):242-7.<br>
Weber, T., and Marahiel, M.A. (2001) Exploring the domain structure of modular<br>
nonribosomal peptide synthetases. Structure 9: R3-R9<br>
Welch, J. T. and Seper, K., W., (1988), J. Org. Chem., 53,2991-2999<br>
Wilkinson, B., Foster, G., Rudd, BAM., Taylor, N.L., Blackaby, A.P., Sidebottom,<br>
P.J., Cooper, D.J., Dawson, M.J., Buss, A.D., Gaisser, S., Bohm, I.U., Rowe,<br>
C.J., Cortes, J., Leadlay, P.F. and Staunton, J. (2000). Novel octaketide<br>
macrofides related to 6-deoxoerythronolide B provide evidence for iterative<br>
operation of the erythromycin polyketide synthase. Chemistry &amp; Biology 7:<br>
111-117.<br>
Wong, G.K., Griffith, S., Kojima, I., and Demain, A.L. (1998) Antifungal activities of<br>
rapamycin and its derivatives, prolylrapamycin, 32-desmethylrapamycin, and<br>
32-desmethoxyrapamycin. Journal of Antibiotics 51:487-491.<br>
Wu, K., Chung, L., Revill, W.P., Katz, L, and Reeves, CD. (2000) The FK520 gene<br>
cluster of Streptomyces hygroscopicus var. ascomyceticus (ATCC 14891)<br>
contains genes for biosynthesis of unusual polyketide extender units. Gene<br>
251: 81-90.<br>
Yem, A!W., Tomasselli, A.G., Heinrikson, R.L., Zurcher-Neely, H., Ruff, VA.,<br>
Johnson, RA, and Deibel, M.R. (1992) The Hsp56 component of steroid<br>
receptor complexes binds to immobilized FK506 and shows homology to<br>
FKBP-12 and FKBP-13. Journal of Biological Chemistry 267: 2868-2871.<br>
Yu, K., Toral-Barza, L, Discafani, C, Zhang, W.G., Skotnicki, J., Frost, P., Gibbons,<br>
J.J. (2001) mTOR, a novel target in breast cancer the effect of CCI-779, an<br>
mTOR inhibitor, in preclinical models of breast cancer. Endocrine-Related<br>
Cancer 8:249-258.<br>
Zhu, J., Wu J., Frizell, E., Liu, S.L, Bashey, R., Rubin, R., Norton, P., Zern, MA<br><br>
(1999). Rapamycin inhibits hepatic stellate cell proliferation in vitro and limits<br>
fibrogenesis in an in vivo model of liver fibrosis.<br>
Gastroenterology. 117(5):1198-204.<br><br>
WE CLAIM :<br>
1.	A method of generating analogues of FKBP-ligands which incorporate a non-natural<br>
starter unit, said method comprising:<br>
(a)	generating a recombinant strain in which at least the rapK homologue which has<br>
at least 40% sequence identity to the sequence of rapK as shown in SEQ ID NO: 13<br>
has been deleted or inactivated; and<br>
(b)	feeding a non-natural starter unit to said strain.<br><br>
2.	The method as claimed in claim 1, which additionally comprises deleting one or more<br>
further auxiliary genes.<br>
3.	The method as claimed in any one of claims 1 or 2, which additionally comprises<br>
restoring by complementation one or more of the deleted genes.<br>
4.	The method as claimed in any one of claims 1 to 3, which additionally comprises the<br>
step of isolating and purifylng the FKBP-ligand analogues generated.<br>
5.	The method as claimed in any one of claims 1 to 4, wherein said non-natural starter<br>
unit is selected from the group consisting of: 2-norbornane carboxylic acid; 2-<br>
(cis/trans)-hydroxycyclohexane carboxylic acid; 3-(cis/trans)-hydroxycyclohexane<br>
carboxylic acid; 4-(cis/trans)-hydroxycyclohexane carboxylic acid; 2-(cis/trans)-<br>
methylcyclohexane carboxylic acid; 4-(cis/trans)-methylcyclohexane carboxylic acid;<br>
3-(cis/trans)-methoxycyclohexane carboxylic acid; 4-(cis/trans)-methoxycyclohexane<br>
carboxylic acid; 4-oxo cyclohexane carboxylic acid; 2-oxo cyclohexane carboxylic<br>
acid; 4-trans-n-pentylcyclohexane carboxylic acid; 2-trans-aminocyclohexane<br>
carboxylic acid; 4-cis-aminocyclohexane carboxylic acid; 4-(cis/trans)-<br>
aminomethylcyclohexane carboxylic acid; cyclopentane carboxylic acid; cyclobutane<br>
carboxylic acid; 1-methylcyclohexane carboxylic acid; 3-trans-hydroxy-4-cis-<br>
fluorocyclohexane carboxylic acid and 4-trans-hydroxy-3-cis-fluorocyclohexane<br>
carboxylic acid; 3-cis-hydroxy-4-trans-fluorocyclohexane carboxylic acid and 4-cis-<br>
hydroxy-3-trans-fluorocyclohexane carboxylic acid; 3-c/'s-hydroxy-4-trans-<br>
chlorocyclohexane carboxylic acid and 4-cis-hydroxy-3-trans-chlorocyclohexane<br>
carboxylic acid; 3-trans-hydroxy-4-cis-chlorocyclohexane carboxylic acid and 4-trans-<br>
hydroxy-3-cis-chlorocyclohexane carboxylic acid; 3-trans-cyclohexeneoxide<br>
carboxylic acid; 3-cis-cyclohexeneoxide carboxylic acid; 3,4-cis-<br>
dihydroxycyclohexane carboxylic acid and 3,4-trans-dihydroxycyclohexane carboxylic<br><br>
acid; cyclohexaneacetic acid; cyclohexanepropionic acid and 4-cis/trans-tert-<br>
butylcyclohexane carboxylic acid or simple esters or salts thereof.<br>
6.	The method as claimed in claim 5, wherein said non-natural starter unit is selected<br>
from the group consisting of: 3-(cis/trans)-hydroxycyclohexane carboxylic acid; 4-<br>
(cis/trans)-hydroxycyclohexane carboxylic acid; 3-(cis/trans)-methoxycyclohexane<br>
carboxylic acid; 4-(cis/trans)-methoxycyclohexane carboxylic acid; 4-oxo<br>
cyclohexane carboxylic acid; cyclobutane carboxylic acid; 3-trans-hydroxy-4-cis-<br>
fluorocyclohexane carboxylic acid and 4-trans-hydroxy-3-cis-fluorocyclohexane<br>
carboxylic acid; 3-cis-hydroxy-4-trans-fluorocyclohexane carboxylic acid and 4-cis-<br>
hydroxy-3-trans-fluorocyclohexane carboxylic acid; 3-cis-hydroxy-4-trans-<br>
chlorocyclohexane carboxylic acid and 4-cis-hydroxy-3-trans-chlorocyclohexane<br>
carboxylic acid; 3-trans-hydroxy-4-cis-chlorocyclohexane carboxylic acid and 4-trans-<br>
hydroxy-3-cis-chlorocyclohexane carboxylic acid; 3-trans-cyclohexeneoxide<br>
carboxylic acid; 3-cis-cyclohexeneoxide carboxylic acid; 3,4-cis-<br>
dihydroxycyclohexane carboxylic acid and 3,4-trans-dihydroxycyclohexane carboxylic<br>
acid; cyclohexanepropionic acid; 4-cis/trans-tert-butylcyclohexane carboxylic acid or<br>
simple esters or salts thereof.<br>
7.	The method as claimed in claim 5 provided that the non-natural starter fed to said<br>
recombinant strain is not: cyclohexane carboxylic acid, 3-cis,A-trans-<br>
dihydroxycyclohexane carboxylic acid, 1-cyclohexene carboxylic acid, 3-cyclohexene<br>
carboxylic acid, cycloheptane carboxylic acid, 3-(cis/trans)-methylcyclohexane<br>
carboxylic acid, 4-(cis/frans)-methylcyclohexane carboxylic acid, 1-cycloheptene<br>
carboxylic acid or 5-cis-hydroxyl-3-cyclohexene carboxylic acid.<br>
8.	The method as claimed in any one of claims 1 to 7, wherein the recombinant strain of<br>
part (a) is made according to a method comprising:<br><br>
(a)	constructing a conjugative deletion plasmid in an E. coli strain that is dam",<br>
dcm" or dam" and dcm.<br>
(b)	generation of spores from said host strain suitable for conjugation wherein<br>
said strain is grown at a humidity of between 10% and 40% and the spores are<br>
harvested at between 5 and 30 days;<br>
(c)	conjugating the E. coli strain of step (a) with the spores from step (b) on a<br>
medium that comprises per litre:<br>
i) 0.5g to 5g corn steep powder,<br>
ii)  0.1g to 5g Yeast extract,<br><br>
iii) 0.1g to 10g calcium carbonate; and<br>
iv) 0.01g to 0.5 g iron sulphate;<br>
said medium additionally containing BACTO-agar and starch and having been dried<br>
to result in 1-20% weight loss; and<br>
(d)  optionally culturing the recombinant strain under conditions suitable for<br>
polyketide production.<br>
9.	The method as claimed in any one of claims 1 to 8, wherein the strain is selected<br>
from the group consisting of Streptomyces hygroscopicus subsp. hygroscopicus<br>
NRRL 5491, Actinoplanes sp. N902-109 FERM BP-3832, Streptomyces sp. AA6554,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6475 ATCC 14891,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6678 ATCC 55087,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6674, Streptomyces<br>
hygroscopicus var. ascomyceticus ATCC 55276, Streptomyces tsukubaensis<br>
No.9993 FERM BP-927, Streptomyces hygroscopicus subsp. yakushimaensis,<br>
Streptomyces sp. DSM 4137, Streptomyces sp. DSM 7348, Micromonospora n.sp.<br>
A92-306401 DSM 8429 and Streptomyces sp. MA 6858 ATCC 55098.<br>
10.	The method as claimed in claim 9, wherein the strains are selected from the group<br>
consisting of: S. hygroscopicus subsp. hygroscopicus NRRL 5491 and S.<br>
hygroscopicus var. ascomyceticus ATCC 14891.<br>
11.	The method as claimed in claim 10, wherein the strain is the rapamycin producer S.<br>
hygroscopicus subsp. hygroscopicus NRRL 5491.<br>
12.	The method as claimed in claim 3, wherein the process of restoring the deleted<br>
genes comprises:<br>
(a)	constructing a gene cassette which contains one or more of the deleted genes<br>
and<br>
(b)	transforming said recombinant strain that contains biosynthetic clusters that<br>
encode FKBP-ligands with said gene cassette.<br>
13.	The method as claimed in claim 12, wherein said gene cassette is assembled directly<br>
in an expression vector.<br>
14.	The method as claimed in claim 12 or 13, wherein the complementation is<br><br>
homologous.<br>
15.	The method as claimed in claim 12 or 13, wherein the complementation is<br>
heterologous.<br>
16.	The method as claimed in any one of claims 2 to 15, wherein said one or more<br>
deleted or inactivated auxiliary genes are selected from the group consisting of<br>
starter unit supply genes, amino acid precursor supply genes, cytochrome P-450<br>
monooxygenases, ferredoxins and SAM-dependent O-methyltransferases.<br>
17.	The method as claimed in claim 16, wherein the deleted or inactivated genes are<br>
selected from the group consisting of rapL, rapN, rapO, rapM, rapQ, rapl and rapJ.<br>
18.	A recombinant strain that contains biosynthetic clusters that encode FKBP-ligands,<br>
wherein at least the rapK homologue has been deleted or inactivated.<br>
19.	The recombinant strain as claimed in claim 18, wherein the strain is selected from<br>
the group consisting of Streptomyces hygroscopicus subsp. hygroscopicus NRRL<br>
5491, Actinoplanes sp. N902-109 FERM BP-3832, Streptomyces sp. AA6554,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6475 ATCC 14891,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6678 ATCC 55087,<br>
Streptomyces hygroscopicus var. ascomyceticus MA 6674, Streptomyces<br>
hygroscopicus var. ascomyceticus ATCC 55276, Streptomyces tsukubaensis<br>
No.9993 FERM BP-927, Streptomyces hygroscopicus subsp. yakushimaensis,<br>
Streptomyces sp. DSM 4137, Streptomyces sp. DSM 7348, Micromonospora n.sp.<br>
A92-306401 DSM 8429 and Streptomyces sp. MA 6858 ATCC 55098.<br>
20.	The recombinant strain as claimed in claim 19, wherein the strains are selected from<br>
the group consisting of S. hygroscopicus subsp. hygroscopicus NRRL 5491 and S.<br>
hygroscopicus var. ascomyceticus ATCC 14891.<br>
21.	The recombinant strain as claimed in claim 20, wherein the strain is the rapamycin<br>
producer S. hygroscopicus subsp. hygroscopicus NRRL 5491.<br>
22.	The recombinant strain as claimed in any one of claims 18 to 21, wherein the<br>
additionally deleted or inactivated auxiliary genes are selected from the group<br>
consisting of: rapL, rapN/O, rapQ, rapM, rapl and rapJ have been deleted, wherein<br><br>
said deletion is not rapQrapN/OrapMrapL<br><br><br><br>
A METHOD OF GENERATING ANALOGUES OF FKBP-LIGANDS WHICH<br>
INCORPORATE A NON-NATURAL STARTER UNIT<br><br><br>
ABSTRACT	<br><br><br>
The present invention relates to production of polyketides and other natural products and to libraries of compounds<br>
and individual novel compounds. One important area is the isolation and potential use of novel FKBP-ligand analogues and host cells<br>
that produce these compounds. The invention is particularly concerned with methods for the efficient transformation of strains that<br>
produce FKBP analogues and recombinant cells in which cloned genes or gene cassettes are expressed to generate novel compounds<br>
such as polyketide (especially rapamycin) FKBP-ligand analogues, and to processes foT their preparation, and to means employed<br>
therein (e.g. nucleic acids, vectors, gene cassettes and genetically modified strains).</r8></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI2NzQta29sbnAtMjAwNy1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">02674-kolnp-2007-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgwMy0xMC0yMDEyKS1BTk5FWFVSRSBUTyBGT1JNIDMucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(03-10-2012)-ANNEXURE TO FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgwMy0xMC0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(03-10-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgwOS0wNC0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(09-04-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgwOS0wNC0yMDEyKS1GT1JNIDMucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(09-04-2012)-FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1BTUFOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04ucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-AMANDED PAGES OF SPECIFICATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1FWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-EXAMINATION REPORT REPLY RECEIVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1GT1JNLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-FORM-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1GT1JNLTIucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-FORM-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1PVEhFUiBQQVRFTlQgRE9DVU1FTlQucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-OTHER PATENT DOCUMENT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LSgxNS0xMS0yMDExKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-(15-11-2011)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUNBTkNFTExFRCBQQUdFUy5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-CANCELLED PAGES.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFIDEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUVYQU1JTkFUSU9OIFJFUE9SVC5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-EXAMINATION REPORT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUZPUk0gMTMucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUZPUk0gMTgucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdQQS5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GPA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtQUJTVFJBQ1QucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtQ0xBSU1TLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtREVTQ1JJUFRJT04gKENPTVBMRVRFKS5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtRFJBV0lOR1MucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-DRAWINGS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtRk9STSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-FORM 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtRk9STSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtRk9STSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtRk9STSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-FORM 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtU1BFQ0lGSUNBVElPTi1DT01QTEVURS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-SPECIFICATION-COMPLETE-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LUdSQU5URUQtU1BFQ0lGSUNBVElPTi1DT01QTEVURS5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-GRANTED-SPECIFICATION-COMPLETE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LU1BUktFRCBVUCBDT1BZLTEucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-MARKED UP COPY-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LU1BUktFRCBVUCBDT1BZLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-MARKED UP COPY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LU9USEVSUy5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LVBSSU9SSVRZIERPQ1VNRU5ULnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-PRIORITY DOCUMENT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVC0xLnBkZg==" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-REPLY TO EXAMINATION REPORT-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVC5wZGY=" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-REPLY TO EXAMINATION REPORT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY3NC1LT0xOUC0yMDA3LVNFUVVFTkNFIExJU1RJTkcucGRm" target="_blank" style="word-wrap:break-word;">2674-KOLNP-2007-SEQUENCE LISTING.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="256238-activation-of-a-glass-surface.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="256240-catalyst-compositions-comprising-metal-phosphate-bound-zeolite-and-methods-of-using-same-to-catalytically-crack-hydrocarbons.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>256239</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2674/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-May-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-May-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIOTICA TECHNOLOGY LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHESTERFORD RESEARCH PARK, LITTLE CHESTERFORD NR SAFFRON WALDEN, ESSEX CB10 1XL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GAISSER SABINE</td>
											<td>CHESTERFORD RESEARCH PARK, LITTLE CHESTERFORD NR SAFFRON WALDEN, ESSEX CB10 1XL</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GREGORY MATTHEW ALAN</td>
											<td>CHESTERFORD RESEARCH PARK, LITTLE CHESTERFORD NR SAFFRON WALDEN, ESSEX CB10 1XL</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PETKOVIC HRVOJE</td>
											<td>CHESTERFORD RESEARCH PARK, LITTLE CHESTERFORD NR SAFFRON WALDEN, ESSEX CB10 1XL</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MOSS STEVEN</td>
											<td>CHESTERFORD RESEARCH PARK, LITTLE CHESTERFORD NR SAFFRON WALDEN, ESSEX CB10 1XL</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/36</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2003/003230</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-07-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0216509.0</td>
									<td>2002-07-16</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0224922.5</td>
									<td>2002-10-25</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/256239-a-method-of-generating-analogues-of-fkbp-ligands-which-incorporate-a-non-natural-starter-unit by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:46:19 GMT -->
</html>
