<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/256541-recombinant-nucleic-acid-molecule-vector-comprising-it-and-polypeptide-encoded-by-it by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:26:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 256541:&quot;RECOMBINANT NUCLEIC ACID MOLECULE, VECTOR COMPRISING IT AND POLYPEPTIDE ENCODED BY IT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;RECOMBINANT NUCLEIC ACID MOLECULE, VECTOR COMPRISING IT AND POLYPEPTIDE ENCODED BY IT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT IT AND POLYPEPTIDE ENCODED BY IT&quot; The invention discloses a recombinant nucleic acid molecule encoding a 5- enolpyruvylshikimate-3-phosphate synthase enzyme selected from the group consisting of: a) a nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO:1 or 2, or a complement thereof; b) a nucleic acid molecule comprising a nucleotide sequence having at least 95% sequence identity to the nucleotide sequence of SEQ ID NO: 1 or 2, or a complement thereof, wherein said nucleotide sequence encodes a polypeptide having herbicide resistance activity; c) the herbicide resistance nucleotide sequence of the DNA insert of the plasmid deposited as Accession No. NRRL B-30804, or a complement thereof; d) a nucleic acid molecule which encodes a polypeptide comprising the amino acid sequence of SEQ ID NO:3; and, e) a nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide having at least 95% amino acid sequence identity to the amino acid sequence of SEQ ID NO:3, wherein said polypeptide has herbicide resistance activity. The invention is also for a vector comprising said molecule and a polypeptide encoded by it having herbicide resistance.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>WO 2006/110188	PCT/US2005/046608<br>
"RECOMBINANT NUCLEIC ACID MOLECULE, VECTOR COMPRISING<br>
IT AND POLYPEPTIDE ENCODED BY IT"<br><br>
This invention provides a novel gene encoding a 5-enolpyruvylshikimate-3-<br>
phosphate (EPSP) synthase that provides herbicide resistance, and variants thereof.<br>
This gene and its variants are useful in plant biology, crop breeding, and plant cell<br>
culture.<br>
BACKGROUND OF THE INVENTION<br>
N-phosphonomethylglycine, commonly referred to as glyphosate, is an<br>
important agronomic chemical. Glyphosate inhibits the enzyme that converts<br>
phosphoenolpyruvic acid (PEP) and 3-phosphoshikimic acid to 5-enolpyruvyl-3-<br>
phosphoshikrrnic acid. Inhibition of this enzyme (5-enolpyruvylshikimate-3-<br>
phosphate synthase; referred to herein as "EPSP synthase") kills plant cells by<br>
shutting down the shikimate pathway, thereby inhibiting aromatic amino acid<br>
biosynthesis.<br>
Since glyphosate-class herbicides inhibit arpmatic amino acid biosynthesis,<br>
they not only kill plant cells, but are also toxic to bacienal cells. Glyphosate inhibits<br>
many bacterial EPSP synthases, and thus is toxic to these bacteria. However, certain<br>
bacterial EPSP synthases have a high tolerance to glyphosate.<br>
Plant cells resistant to glyphosate toxicity can be produced by transforming<br>
plant cells to express glyphosate-resistant bacterial EPSP synthases. Notably, the<br>
bacterial gene from Agrobacterium tumefaciens strain CP4 has been used to confer<br>
herbicide resistance on plant cells following expression in plants. A mutated EPSP<br>
synthase from Salmonella typhimurium strain CT7 confers giyphosate resistance in<br>
bacterial cells, and confers glyphosate resistance on plant cells (U.S. Patent Nos.<br>
4,535,060; 4,769,061; and 5,094,945). However, there is a need for other herbicide<br>
resistance genes.<br>
BRIEF SUMMARY OF THE INVENTION<br>
Compositions and methods for conferring herbicide resistance to bacteria,<br>
plants, plant cells, tissues and seeds are provided. Compositions include nucleic acid<br>
l<br><br>
WO 2006/110188	PCT/US2005/046608<br>
molecules encoding sequences for herbicide resistance polypeptides, vectors<br>
comprising those nucleic acid molecules, and host cells comprising the vectors.<br>
Compositions comprising a coding sequence for a polypeptide that confers resistance<br>
or tolerance to glyphosate herbicides are provided, as well as antibodies to the<br>
polypeptides. Compositions of the present invention include synthetic nucleic acid<br>
molecules encoding herbicide resistance polypeptides. The coding sequences can be<br>
used in DNA constructs or expression cassettes for transformation and expression in<br>
organisms, including microorganisms and plants. Compositions also comprise<br>
transformed bacteria, plants, plant cells, tissues, and seeds. In addition, methods are<br>
provided for producing the polypeptides encoded by the synthetic nucleotides of the<br>
invention.<br>
In particular, isolated nucleic acid molecules corresponding to herbicide<br>
resistance-conferring nucleic acid sequences are provided. Additionally, amino acid<br>
sequences corresponding to the polynucleotides are encompassed. In particular, the<br>
present invention provides for isolated nucleic acid molecules comprising the<br>
nucleotide sequence set forth in SEQ ID NO:1 or 2, a nucleotide sequence encoding<br>
the amino acid sequence shown in SEQ ID NO:3, the herbicide resistance nucleotide<br>
sequence deposited in a hacterial host as Accession No. B-30804, as well as variants<br>
and fragments thereof. Nucleotide sequences that are complementary to a nucleotide<br>
sequence of the invention, or that hybridize to a sequence of the invention are also<br>
encompassed.<br>
BRIEF DESCRIPTION OF FIGURES<br>
Figure 1 shows the nucleotide sequence of grg8 (SEQ ID NO:1). The open<br>
reading frame starts at nucleotide 296 and ends at nucleotide 1555 (SEQ ID NO:2).<br>
Figure 2 shows the amino acid sequence of GRG8 protein (SEQ ID NO:3).<br>
Figure 3 shows an alignment of the amino acid sequences of GRG8 (SEQ ID<br>
NO:3), GRG8ml_(C22) (SEQ ID NO:5), GRG8m3_(N24) (SEQ ID NO:9),<br>
GRG8m2_(N15) (SEQ ID NO:7), GRG8m4_(Al) (SEQ ID NO: 11), GRG8m5_(B2)<br>
(SEQ ID NO: 13), GRG8m6_(B7) (SEQ ID NO: 15), GRG8m7_(B11) (SEQ ID<br>
NO.17), GRG8m8_(Cll) (SEQ ID NO: 19), GRG8m9_(E3) (SEQ ID NO:21), and<br>
GRG8ml0_(E4) (SEQ ID NO:23).<br>
2<br><br>
WO 2006/110188	PCT/US2005/046608<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present inventions now will be described more fully hereinafter with<br>
reference to the accompanying drawings, in which some, but not all embodiments of<br>
the inventions are shown. Indeed, these inventions may be embodied in many<br>
different forms and should not be construed as limited to the embodiments set forth<br>
herein; rather, these embodiments are provided so that this disclosure will satisfy<br>
applicable legal requirements. Like numbers refer to like elements throughout.<br>
Many modifications and other embodiments of the inventions set forth herein<br>
will come to mind to one skilled in the art to which these inventions pertain having<br>
the benefit of the teachings presented in the foregoing descriptions and the associated<br>
drawings. Therefore, it is to be understood that the inventions are not to be limited to<br>
the specific embodiments disclosed and that modifications and other embodiments are<br>
intended to be included within the scope of the appended claims. Although specific<br>
terms are employed herein, they are used in a generic and descriptive sense only and<br>
not for purposes of limitation.<br>
The present invention is drawn to compositions and methods for regulating<br>
herbicide resistance in organisms, particularly in plants or plant cells. The methods<br>
involve, transforming organisms with nucleotide sequences encodingiheglyphosate.<br>
resistance gene of the invention. The nucieotide sequences of the invention arc useful<br>
for preparing plants that show increased tolerance to the herbicide glyphosate. Thus,<br>
transformed bacteria, plants, plant cells, plant tissues and seeds are provided.<br>
Compositions include nucleic acids and proteins relating to herbicide tolerance in<br>
microorganisms and plants as well as transformed bacteria, plants, plant tissues and<br>
seeds. Nucieotide sequences of the glyphosate resistance gene (grg8) and the amino<br>
acid sequences of the proteins encoded thereby are disclosed. The sequences find use<br>
in the construction of expression vectors for subsequent transformation into plants of<br>
interest, as probes for the isolation of other glyphosate resistance genes, as selectable<br>
markers, and the like.<br>
Plasmids containing the herbicide resistance nucieotide sequences of the<br>
invention were deposited in the permanent collection of the Agricultural Research<br>
Service Culture Collection, Northern Regional Research Laboratory (NRRL) on<br>
December 21,2004, and assigned Accession No. NRRL B-30804. This deposit will<br>
3<br><br>
WO 2006/110188	PCT/US2005/046608<br>
be maintained under the terms of the Budapest Treaty on the International<br>
Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure.<br>
This deposit was made merely as a convenience for those of skill in the art and is not<br>
an admission that a deposit is required under 35 U.S.C. §112.<br>
By "glyphosate" is intended any herbicidal form of N-<br>
phosphonomethylglycine (including any salt thereof) and other forms that result in the<br>
production of the glyphosate anion in planta. An "herbicide resistance protein" or a<br>
protein resulting from expression of an "herbicide resistance-encoding nucleic acid<br>
molecule" includes proteins that confer upon a cell the ability to tolerate a higher<br>
concentration of an herbicide than cells that do not express the protein, or to tolerate a<br>
certain concentration of an herbicide for a longer time than cells that do not express<br>
the protein. A "glyphosate resistance protein" includes a protein that confers upon a<br>
cell the ability to tolerate a higher concentration of glyphosate than cells that do not<br>
express the protein, or to tolerate a certain concentration of glyphosate for a longer<br>
period of time than cells that do not express the protein. By "tolerate" or "tolerance"<br>
is intended either to survive, or to carry out essential cellular functions such as protein<br>
synthesis and respiration in a manner that is not readily discemable from untreated<br>
cells.<br>
Isolated Nucleic Acid Molecules, and Variants and Fragments Thereof<br>
One aspect of the invention pertains to isolated nucleic acid molecules<br>
comprising nucleotide sequences encoding herbicide resistance proteins and<br>
polypeptides or biologically active portions thereof, as well as nucleic acid molecules<br>
sufficient for use as hybridization probes to identify herbicide resistance-encoding<br>
nucleic acids. As used herein, the term "nucleic acid molecule" is intended to include<br>
DNA molecules (e.g., cDNA or genomic DNA) and RNA molecules (e.g., mRNA)<br>
and analogs of the DNA or RNA generated using nucleotide analogs. The nucleic<br>
acid molecule can be single-stranded or double-stranded.<br>
Nucleotide sequences encoding the proteins of the present invention include<br>
the sequences set forth in SEQ ID NOS: 1 and 2, the herbicide resistance nucleotide<br>
sequence deposited in a bacterial host as Accession No. NRRL B-30804, and variants,<br>
fragments, and complements thereof. By "complement" is intended a nucleotide<br>
4<br><br>
WO 2006/110188	PCT/US2005/046608<br>
sequence that is sufficiently complementary to a given nucleotide sequence such that<br>
it can hybridize to the given nucleotide sequence to thereby form a stable duplex. The<br>
corresponding amino acid sequence for the herbicide resistance protein encoded by<br>
these nucleotide sequences is set forth in SEQ ID NO:3. The invention also<br>
encompasses nucleic acid molecules comprising nucleotide sequences encoding<br>
partial-length herbicide resistance proteins, and complements thereof.<br>
An "isolated" or "purified" nucleic acid molecule or protein, or biologically<br>
active portion thereof, is substantially free of other cellular material, or culture<br>
medium when produced by recombinant techniques, or substantially free of chemical<br>
precursors or other chemicals when chemically synthesized. Preferably, an "isolated"<br>
nucleic acid is free of sequences (preferably protein encoding sequences) that<br>
naturally flank the nucleic acid (i.e., sequences located at the 5' and 3' ends of the<br>
nucleic acid) in the genomic DNA of the organism from which the nucleic acid is<br>
derived. For purposes of the invention, "isolated" when used to refer to nucleic acid<br>
molecules excludes isolated chromosomes. For example, in various embodiments, the<br>
isolated glyphosate resistance-encoding nucleic acid molecule can contain less than<br>
about 5 kb, 4 kb, 3 kb, 2 kb, 1 kb, 0.5 kb, or 0.1 kb of nucleotide sequence that<br>
naturally flanks the nucleic acid molecule in genomic DNA of the cell from which the<br>
nucleic acid is derived. An herbicide resistance protein that is substantially free of<br>
cellular material includes preparations of protein having less than about 30%, 20%,<br>
10%, or 5% (by dry weight) of non-herbicide resistance protein (also referred to<br>
herein as a "contaminating protein").<br>
Nucleic acid molecules that are fragments of these herbicide resistance-<br>
encoding nucleotide sequences are also encompassed by the present invention. By<br>
"fragment" is intended a portion of the nucleotide sequence encoding an herbicide<br>
resistance protein. A fragment of a nucleotide sequence may encode a biologically<br>
active portion of an herbicide resistance protein, or it may be a fragment that can be<br>
used as a hybridization probe or PCR primer using methods disclosed below. Nucleic<br>
acid molecules that are fragments of an herbicide resistance nucleotide sequence<br>
comprise at least about 15,20, 50, 75,100,200, 300,350,400,450,500, 550, 600,<br>
650, 700, 750, 800, 850, 900, 950, 1000,1050,1100,1150,1200,1250,1300,1350,<br>
1400,1450,1500,1550,1600,1650,1700, 1750,1800,1850,1900,1950 contiguous<br>
5<br><br>
WO 2006/110188	PCT/US2005/046608<br>
nucleotides, or up to the number of nucleotides present in a full-length herbicide<br>
resistance-encoding nucleotide sequence disclosed herein (for example, 2000<br>
nucleotides for SEQ ID NO: 1, and 1257 nucleotides for SEQ ID NO:2). By<br>
"contiguous" nucleotides is intended nucleotide residues that are immediately<br>
adjacent to one another.<br>
Fragments of the nucleotide sequences of the present invention generally will<br>
encode protein fragments that retain the biological activity of the full-length<br>
glyphosate resistance protein; i.e., herbicide-resistance activity. By "retains herbicide<br>
resistance activity" is intended that the fragment will have at least about 30%, at least<br>
about 50%, at least about 70%, or at least about 80% of the herbicide resistance<br>
activity of the full-length glyphosate resistance proteins disclosed herein as SEQ ID<br>
NO:3. Methods for measuring herbicide resistance activity are well known in the art.<br>
See, for example, U.S. Patent Nos. 4,535,060, and 5,188,642, each of which are<br>
herein incorporated by reference in their entirety.<br>
A fragment of an herbicide resistance-encoding nucleotide sequence that<br>
encodes a biologically active portion of a protein of the invention will encode at least<br>
about 15, 25, 30, 50, 75,100, 125,150,175, 200, 250,300, 350, 400 contiguous<br>
amino acids, or up to the total number of ammo acids present in a full-length<br>
herbicide resistance protein of the invention (for example, 419 amino acids for the<br>
protein of the invention).<br>
Herbicide resistance proteins of the present invention are encoded by a<br>
nucleotide sequence sufficiently identical to the nucleotide sequence of SEQ ID NO: 1<br>
or 2. The term "sufficiently identical" is intended an amino acid or nucleotide<br>
sequence that has at least about 60% or 65% sequence identity, about 70% or 75%<br>
sequence identity, about 80% or 85% sequence identity, about 90%, 91%, 92%, 93%,<br>
94%, 95%, 96%, 97%, 98% or 99% sequence identity compared to a reference<br>
sequence using one of the alignment programs described herein using standard<br>
parameters. One of skill in the art will recognize that these values can be<br>
appropriately adjusted to determine corresponding identity of proteins encoded by two<br>
nucleotide sequences by taking into account codon degeneracy, amino acid similarity,<br>
reading frame positioning, and the like.<br>
6<br><br>
WO 2006/110188	PCT/US2005/046608<br>
To determine the percent identity of two amino acid sequences or of two<br>
nucleic acids, the sequences are aligned for optimal comparison purposes. The<br>
percent identity between the two sequences is a function of the number of identical<br>
positions shared by the sequences (i.e., percent identity = number of identical<br>
positions/total number of positions (e.g., overlapping positions) x 100). In one<br>
embodiment, the two sequences are the same length. The percent identity between<br>
two sequences can be determined using techniques similar to those described below,<br>
with or without allowing gaps. In calculating percent identity, typically exact matches<br>
are counted.<br>
The determination of percent identity between two sequences can be<br>
accomplished using a mathematical algorithm. A non-limiting example of a<br>
mathematical algorithm utilized for the comparison of two sequences is the algorithm<br>
of Karlin and Altschul (1990) Proc. Natl. Acad. Sci. USA 87:2264-2268, modified as<br>
in Karlin and Altschul (1993) Proc. Natl. Acad. Sci. USA 90:5873-5877. Such an<br>
algorithm is incorporated into the BLASTN and BLASTX programs of Altschul et al.<br>
(1990) J. Mol. Biol. 215:403-410. BLAST nucleotide searches can be performed with<br>
the BLASTN program, score = 100, wordlength = 12, to obtain nucleotide sequences<br>
homologous to GDC-like nucleic acid molecules of the invention. BLASTprotein<br>
searches can be performed with the BLASTX program, score = 50, wordlength = 3, to<br>
obtain amino acid sequences homologous to herbicide resistance protein molecules of<br>
the invention. To obtain gapped alignments for comparison purposes, Gapped<br>
BLAST can be utilized as described in Altschul et al. (1997) Nucleic Acids Res.<br>
25:3389-3402. Alternatively, PSI-Blast can be used to perform an iterated search that<br>
detects distant relationships between molecules. See Altschul et al. (1997) supra.<br>
When utilizing BLAST, Gapped BLAST, and PSI-Blast programs, the default<br>
parameters of the respective programs (e.g., BLASTX and BLASTN) can be used.<br>
See www.ncbi.nhn.nih.gov. Another non-limiting example of a mathematical<br>
algorithm utilized for the comparison of sequences is the ClustalW algorithm<br>
(Higgins et al.(1994) Nucleic Acids Res. 22:4673-4680). ClustalW compares<br>
sequences and aligns the entirety of the amino acid or DNA sequence, and thus can<br>
provide data about the sequence conservation of the entire amino acid sequence. The<br>
ClustalW algorithm is used in several commercially available DNA/amino acid<br>
7<br><br>
WO 2006/110188	PCT/US2005/046608<br>
analysis software packages, such as the ALIGNX module of the Vector NTI Program<br>
Suite (Invitrogen Corporation, Carlsbad, CA). After alignment of amino acid<br>
sequences with ClustalW, the percent amino acid identity can be assessed. A non-<br>
limiting example of a software program useful for analysis of ClustalW alignments is<br>
GeneDoc™. Genedoc™ (Karl Nicholas) allows assessment of amino acid (or DNA)<br>
similarity and identity between multiple proteins. Another non-limiting example of a<br>
mathematical algorithm utilized for the comparison of sequences is the algorithm of<br>
Myers and Miller (1988) CABIOS 4:11-17. Such an algorithm is incorporated into the<br>
ALIGN program (version 2.0), which is part of the GCG sequence alignment software<br>
package (available from Accelrys, Inc., San Diego, CA). When utilizing the ALIGN<br>
program for comparing amino acid sequences, a PAM120 weight residue table, a gap<br>
length penalty of 12, and a gap penalty of 4 can be used.<br>
'.        Unless otherwise stated, GAP Version 10, which uses the algorithm of<br>
Needleman and Wunsch (1970) J. Mol Biol 48(3):443-453, will be used to determine<br>
sequence identity or similarity using the following parameters: % identity and %<br>
similarity for a nucleotide sequence using GAP Weight of 50 and Length Weight of 3,<br>
and the nwsgapdna.cmp scoring matrix; % identity or % similarity for an amino acid<br>
sequence using GAP weight of 8 and length weight of 2, and the BLOSUM62 scoring.<br>
program. Equivalent programs may also be used. By "equivalent program" is<br>
intended any sequence comparison program that, for any two sequences in question,<br>
generates an alignment having identical nucleotide residue matches and an identical<br>
percent sequence identity when compared to the corresponding alignment generated<br>
by GAP Version 10.<br>
The invention also encompasses variant nucleic acid molecules. "Variants" of<br>
the herbicide resistance-encoding nucleotide sequences include those sequences that<br>
encode the herbicide resistance protein disclosed herein but that differ conservatively<br>
because of the degeneracy of the genetic code, as well as those that are sufficiently<br>
identical as discussed above. Naturally occurring allelic variants can be identified<br>
with the use of well-known molecular biology techniques, such as polymerase chain<br>
reaction (PCR) and hybridization techniques as outlined below. Variant nucleotide<br>
sequences also include synthetically derived nucleotide sequences that have been<br>
generated, for example, by using site-directed mutagenesis but which still encode the<br>
8<br><br>
WO 2006/110188	PCT/US2005/046608<br>
herbicide resistance proteins disclosed in the present invention as discussed below.<br>
Variant proteins encompassed by the present invention are biologically active, that is<br>
they retain the desired biological activity of the native protein, that is, herbicide<br>
resistance activity. By "retains herbicide resistance activity" is intended that the<br>
variant will have at least about 30%, at least about 50%, at least about 70%, or at least<br>
about 80% of the herbicide resistance activity of the native protein. Methods for<br>
measuring herbicide resistance activity are well known in the art. See, for example,<br>
U.S. Patent Nos. 4,535,060, and 5,188,642, each of which are herein incorporated by<br>
reference in their entirety.<br>
The skilled artisan will further appreciate that changes can be introduced by<br>
mutation into the nucleotide sequences of the invention thereby leading to changes in<br>
the amino acid sequence of the encoded herbicide resistance proteins, without altering<br>
the biological activity of the proteins. Thus, variant isolated nucleic acid molecules<br>
cart be created by introducing one or more nucleotide substitutions, additions, or<br>
deletions into the corresponding nucleotide sequence disclosed herein, such that one<br>
or more amino acid substitutions, additions or deletions are introduced into the<br>
encoded protein. Mutations can be introduced by standard techniques, such as site-<br>
directed mutagenesis and PCR mediated mutagenesis. Such variant nucleotide<br>
sequences are aiso encompassed by the present invention.<br>
For example, conservative amino acid substitutions may be made at one or<br>
more predicted, nonessential amino acid residues. A "nonessential" amino acid<br>
residue is a residue that can be altered from the wild-type sequence of an herbicide<br>
resistance protein without altering the biological activity, whereas an "essential"<br>
amino acid residue is required for biological activity. A "conservative amino acid<br>
substitution" is one in which the amino acid residue is replaced with an amino acid<br>
residue having a similar side chain. Families of amino acid residues having similar<br>
side chains have been defined in the art. These families include amino acids with<br>
basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic<br>
acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine,<br>
serine, threonine, tyrosine, cysteine), nonpolar side chains (e.g., alanine, valine,<br>
leucine, isoleucine, proline, phenylalanine, methionine, tryptophan), beta-branched<br>
side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g.,<br>
9<br><br>
WO 2006/110188	PCT/US2005/046608<br>
tyrosine, phenylalanine, tryptophan, histidine). Amino acid substitutions may be<br>
made in non-conserved regions that retain function. In general, such substitutions<br>
would not be made for conserved amino acid residues, or for amino acid residues<br>
residing within a conserved motif, where such residues are essential for protein<br>
activity. However, one of skill in the art would understand that functional variants<br>
may have minor conserved or non-conserved alterations in the conserved residues.<br>
Alternatively, variant nucleotide sequences can be made by introducing<br>
mutations randomly along all or part of the coding sequence, such as by saturation<br>
mutagenesis, and the resultant mutants can be screened for the ability to confer<br>
herbicide resistance activity to identify mutants that retain activity. Following<br>
mutagenesis, the encoded protein can be expressed in a cell, and the activity of the<br>
protein can be determined using standard assay techniques.<br>
Using methods such as PCR, hybridization, and the like corresponding<br>
herbicide resistance sequences can be identified, such sequences having substantial<br>
identity to the sequences of the invention. See, for example, Sambrook and Russell<br>
(2001) Molecular Cloning: A Laboratory Manual (Cold Spring Harbor Laboratory<br>
Press, Cold Spring Harbor, NY) and Innis, et al. (1990) PCR Protocols: A Guide to<br>
Methods and Applications (Academic Press, Sl..Louis, MO).<br>
In a hybridization method, ail or part of the herbicide resistance nucleotide<br>
sequence can be used to screen cDNA or genomic libraries. Methods for construction<br>
of such cDNA and genomic libraries are generally known in the art and are disclosed<br>
in Sambrook and Russell (2001) supra. The so-called hybridization probes may be<br>
genomic DNA fragments, cDNA fragments, RNA fragments, or other<br>
oligonucleotides, and may be labeled with a detectable group such as 32P, or any other<br>
detectable marker, such as other radioisotopes, a fluorescent compound, an enzyme,<br>
or an enzyme co-factor. Probes for hybridization can be made by labeling synthetic<br>
oligonucleotides based on the known herbicide resistance-encoding nucleotide<br>
sequence(s) disclosed herein. Degenerate primers designed on the basis of conserved<br>
nucleotides or amino acid residues in the nucleotide sequence or encoded amino acid<br>
sequence can additionally be used. The probe typically comprises a region of<br>
nucleotide sequence that hybridizes under stringent conditions to at least about 12, at<br>
least about 25, at least about 50, 75,100,125,150,175, 200,250, 300, 350,400, 500,<br>
10<br><br>
WO 2006/110188	PCT/US2005/046608<br>
600, 700, 800, 900,1000,1200, 1400, 1600, or 1800 consecutive nucleotides of<br>
herbicide resistance-encoding nucleotide sequence(s) of the invention or a fragment or<br>
variant thereof. Methods for the preparation of probes for hybridization are generally<br>
known in the art and are disclosed in Sambrook and Russell (2001) supra, and<br>
Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual (2d ed., Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, NY), both of which are herein<br>
incorporated by reference.<br>
For example, an entire herbicide resistance sequence disclosed herein, or one<br>
or more portions thereof, may be used as a probe capable of specifically hybridizing<br>
to corresponding herbicide resistance sequences and messenger KNAs. To achieve<br>
specific hybridization under a variety of conditions, such probes include sequences<br>
that are unique and are at least about 10 nucleotides in length, and at least about 20<br>
nucleotides in length. Such probes may be used to amplify corresponding herbicide<br>
resistance sequences from a chosen organism by PCR. This technique may be used to<br>
isolate additional coding sequences from a desired organism or as a diagnostic assay<br>
to determine the presence of coding sequences in an organism. Hybridization<br>
techniques include hybridization screening of plated DNA libraries (either plaques or<br>
colonies; see, for example, Sambrook et al. (1989) Molecular Cloning: A Laboratory<br>
Manual (2d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY).<br>
Hybridization of such sequences may be carried out under stringent<br>
conditions. By "stringent conditions" or "stringent hybridization conditions" is<br>
intended conditions under which a probe will hybridize to its target sequence to a<br>
detectably greater degree than to other sequences (e.g., at least 2-fold over<br>
background). Stringent conditions are sequence-dependent and will be different in<br>
different circumstances. By controlling the stringency of the hybridization and/or<br>
washing conditions, target sequences that are 100% complementary to the probe can<br>
be identified (homologous probing). Alternatively, stringency conditions can be<br>
adjusted to allow some mismatching in sequences so that lower degrees of similarity<br>
are detected (heterologous probing). Generally, a probe is less than about 1000<br>
nucleotides in length, or less than about 500 nucleotides in length.<br>
Typically, stringent conditions will be those in which the salt concentration is<br>
less than about 1.5 M Na ion, typically about 0.01 to 1.0 M Na ion concentration (or<br>
11<br><br>
WO 2006/110188	PCT/US2005/046608<br>
other salts) at pH 7.0 to 8.3 and the temperature is at least about 30°C for short probes<br>
(e.g., 10 to 50 nucleotides) and at least about 60°C for long probes (e.g., greater than<br>
50 nucleotides). Stringent conditions may also be achieved with the addition of<br>
destabilizing agents such as formamide. Exemplary low stringency conditions<br>
include hybridization with a buffer solution of 30 to 35% formamide, 1 M NaCl, 1%<br>
SDS (sodium dodecyl sulfate) at 37°C, and a wash in IX to 2X SSC (20X SSC = 3.0<br>
M NaCl/0.3 M trisodium citrate) at 50 to 55°C. Exemplary moderate stringency<br>
conditions include hybridization in 40 to 45% formamide, 1.0 M NaCl, 1% SDS at<br>
37°C, and a wash in 0.5X to IX SSC at 55 to 60°C. Exemplary high stringency<br>
conditions include hybridization in 50% formamide, 1 M NaCl, 1% SDS at 37°C, and<br>
a wash in 0.1X SSC at 60 to 65°C. Optionally, wash buffers may comprise about<br>
0.1% to about 1% SDS. Duration of hybridization is generally less than about 24<br>
hours, usually about 4 to about 12 hours.<br>
Specificity is typically the function of post-hybridization washes, the critical<br>
factors being the ionic strength and temperature of the final wash solution. For DNA-<br>
DNA hybrids, the Tm can be approximated from the equation of Meinkoth and Wahl<br>
(1984) Anal. Biochem. 138:267-284: Tm= 81.5°C +16.6 (log M) + 0.41 (%GC) -<br>
0.61 (% form) - 500/L; where M is the molarity of monovalent cations, %GC is the<br>
percentage of guanosine and cytosine nucleotides in the DNA, % form is the<br>
percentage of formamide in the hybridization solution, and L is the length of the<br>
hybrid in base pairs. The Tm is the temperature (under defined ionic strength and pH)<br>
at which 50% of a complementary target sequence hybridizes to a perfectly matched<br>
probe. Tm is reduced by about 1°C for each 1% of mismatching; thus, Tm,<br>
hybridization, and/or wash conditions can be adjusted to hybridize to sequences of the<br>
desired identity. For example, if sequences with ≥90% identity are sought, the Tm can<br>
be decreased 10°C. Generally, stringent conditions are selected to be about 5°C lower<br>
than the thermal melting point (Tm) for the specific sequence and its complement at a<br>
defined ionic strength and pH. However, severely stringent conditions can utilize a<br>
hybridization and/or wash at 1,2, 3, or 4°C lower than the thermal melting point (Tm);<br>
moderately stringent conditions can utilize a hybridization and/or wash at 6,7, 8, 9, or<br>
10°C lower than the thermal melting point (Tm); low stringency conditions can utilize<br>
a hybridization and/or wash at 11, 12,13, 14,15, or 20°C lower than the thermal<br>
12<br><br>
WO 2006/110188	PCT/US2005/046608<br>
melting point (Tm). Using the equation, hybridization and wash compositions, and<br>
desired Tm, those of ordinary skill will understand that variations in the stringency of<br>
hybridization and/or wash solutions are inherently described. If the desired degree of<br>
mismatching results in a Tm of less than 45 °C (aqueous solution) or 32°C (formamide<br>
solution), it is preferred to increase the SSC concentration so that a higher temperature<br>
can be used. An extensive guide to the hybridization of nucleic acids is found in<br>
Tijssen (1993) Laboratory Techniques in Biochemistry and Molecular Biology—<br>
Hybridization with Nucleic Acid Probes, Part I, Chapter 2 (Elsevier, New York); and<br>
Ausubel et al, eds. (1995) Current Protocols in Molecular Biology, Chapter 2<br>
(Greene Publishing and Wiley-Interscience, New York). See Sambrook et al, (1989)<br>
Molecular Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor Laboratory<br>
Press, Cold Spring Harbor, NY).<br>
Isolated Proteins and Variants and Fragments Thereof<br>
Herbicide resistance proteins are also encompassed within the present<br>
invention. By "herbicide resistance protein" is intended a protein having the amino<br>
acid sequence set forth in SEQ ID NO:3. Fragments, biologically active portions, and<br>
variants thereof are also provided, and may be used to practice the methods of the<br>
present invention.<br>
"Fragments" or "biologically active portions" include polypeptide fragments<br>
comprising a portion of an amino acid sequence encoding an herbicide resistance<br>
protein as set forth in SEQ ID NO:3 and that retains herbicide resistance activity. A<br>
biologically active portion of an herbicide resistance protein can be a polypeptide that<br>
is, for example, 10, 25, 50,100 or more amino acids in length. Such biologically<br>
active portions can be prepared by recombinant techniques and evaluated for<br>
herbicide resistance activity. Methods for measuring herbicide resistance activity are<br>
well known in the art. See, for example, U.S. Patent Nos. 4,535,060, and 5,188,642,<br>
each of which are herein incorporated by reference in their entirety. As used here, a<br>
fragment comprises at least 8 contiguous ammo acids of SEQ ID NO:3. The<br>
invention encompasses other fragments, however, such as any fragment in the protein<br>
greater than about 10, 20, 30, 50,100,150, 200,250, 300, 350, or 400 amino acids.<br>
13<br><br>
WO 2006/110188	PCT/US2005/046608<br>
By "variants" is intended proteins or polypeptides having an amino acid<br>
sequence that is at least about 60%, 65%, at least about 70%, 75%, at least about 80%,<br>
85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the<br>
amino acid sequence of SEQ ID NO:3. Variants also include polypeptides encoded by<br>
a nucleic acid molecule that hybridizes to the nucleic acid molecule of SEQ ID NOS:1<br>
or 2, or a complement thereof, under stringent conditions. Variants include<br>
polypeptides that differ in amino acid sequence due to mutagenesis. Variant proteins<br>
encompassed by the present invention are biologically active, that is they continue to<br>
possess the desired biological activity of the native protein, that is, retaining herbicide<br>
resistance activity. Methods for measuring herbicide resistance activity are well<br>
known in the art. See, for example, U.S. Patent Nos. 4,535,060, and 5,188,642, each<br>
of which are herein incorporated by reference in their entirety.<br>
Bacterial genes, such as the grg8 gene of this invention, quite often possess<br>
multiple methionine initiation codons in proximity to the start of the open reading<br>
frame. Often, translation initiation at one or more of these, start codons will lead to<br>
generation of a functional protein. These start codons can include ATG codons.<br>
However, bacteria such as Bacillus sp. also recognize the codon GTG as a start codon,<br>
and proteins that initiate translation at GTX codons contain a methionine at theiirst<br>
amino acid, furthermore, it is not often determined a priori which of these codons are<br>
used naturally in the bacterium. Thus, it is understood that use of one of the alternate<br>
methionine codons may lead to generation of variants of grg8 that confer herbicide<br>
resistance. These herbicide resistance proteins are encompassed in the present<br>
invention and may be used in the methods of the present invention.<br>
Antibodies to the polypeptides of the present invention, or to variants or<br>
fragments thereof, are also encompassed. Methods for producing antibodies are well<br>
known in the art (see, for example, Harlow and Lane (1988) Antibodies: A Laboratory<br>
Manual (Cold Spring Harbor Laboratory, Cold Spring Harbor, NY); U.S. Patent No.<br>
4,196,265).<br>
Altered or Improved Variants<br>
It is recognized that the DNA sequence of grg8 may be altered by various<br>
methods, and that these alterations may result in DNA sequences encoding proteins<br>
14<br><br>
WO 2006/110188	PCT/US2005/046608<br>
with amino acid sequences different than that encoded by grg8. This protein may be<br>
altered in various ways including amino acid substitutions, deletions, truncations, and<br>
insertions. Methods for such manipulations are generally known in the art. For<br>
example, amino acid sequence variants of the GRG8 protein can be prepared by<br>
mutations in the DNA. This may also be accomplished by one of several forms of<br>
mutagenesis and/or by directed evolution. In some aspects, the changes encoded in<br>
the amino acid sequence will not substantially affect the function of the protein. Such<br>
variants will possess the desired herbicide resistance activity. However, it is<br>
understood that the ability of GRG8 to confer herbicide resistance may be improved<br>
by use of such techniques upon the compositions of the present invention. For<br>
example, GRG8 may be expressed in host cells that exhibit high rates of base<br>
misincorporation during DNA replication, such as XL-1 Red (Stratagene, La Jolla,<br>
CA). After propagation in such strains, grg8 DNA can be isolated (for example by<br>
preparing plasmid DNA, or by amplifying by PCR and cloning the resulting PCR<br>
fragment into a vector) and cultured in non-mutagenic strains. Clones containing<br>
mutations in grg8 can be identified by measuring improved resistance to an herbicide<br>
such as glyphosate, for example by growing cells in increasing concentrations of<br>
glyphosatc and testing for clones that confer tolcerance to increasing concentrations of<br>
glyphosate.<br>
Alternatively, alterations may be made to the protein sequence of many<br>
proteins at the amino or carboxy terminus without substantially affecting activity.<br>
These alterations can include insertions, deletions, or alterations introduced by<br>
modern molecular methods, such as PCR, including PCR amplifications that alter or<br>
extend the protein coding sequence by virtue of inclusion of amino acid encoding<br>
sequences in the oligonucleotides utilized in the PCR amplification. Alternatively, the<br>
protein sequences added can include entire protein-coding sequences, such as those<br>
used commonly in the art to generate protein fusions. Such fusion proteins are often<br>
used to (1) increase expression of a protein of interest; (2) introduce a binding<br>
domain, enzymatic activity, or epitope to facilitate either protein purification, protein<br>
detection, or other experimental uses known in the art; or, (3) target secretion or<br>
translation of a protein to a subcellular organelle, such as the periplasmic space of<br>
15<br><br>
WO 2006/110188	PCT/US2005/046608<br>
gram-negative bacteria, or the endoplasmic reticulum of eukaryotic cells, the latter of<br>
which often results in glycosylation of the protein.<br>
Variant nucleotide and amino acid sequences of the present invention also<br>
encompass sequences derived from mutagenic and recombinogenic procedures such<br>
as DNA shuffling. With such a procedure, one or more different herbicide resistance<br>
protein coding regions can be used to create a new herbicide resistance protein<br>
possessing the desired properties. In this manner, libraries of recombinant<br>
polynucleotides are generated from a population of related sequence polynucleotides<br>
comprising sequence regions that have substantial sequence identity and can be<br>
homologously recombined in vitro or in vivo. For example, using this approach,<br>
sequence motifs encoding a domain of interest may be shuffled between the herbicide<br>
resistance gene of the invention and other known herbicide resistance genes to obtain<br>
a new gene coding for a protein with an improved property of interest, such as an<br>
increased glyphosate resistance activity. Strategies for such DNA shuffling are<br>
known in the art. See, for example, Stemmer (1994) Proc. Natl. Acad. Sci. USA<br>
91:10747-10751; Stemmer (1994) Nature 370:389-391; Crameri et al. (1997) Nature<br>
Biotech. 15:436-438; Moore et al. (1997) J. Mol. Biol 272:336-347; Zhang et al.<br>
(1927) Proc. Natl, Acad. Sci. USA 94:451)44509; Crameri et al (1998)Nature<br>
391:288-291; and U.S. Patent Nos. 5,605,793 and 5,837,458.<br>
Transformation of Bacterial or Plant Cells<br>
Provided herein are novel isolated genes that confer resistance to an herbicide.<br>
Also provided is an amino acid sequence of the GRG8 protein. The protein resulting<br>
from translation of this gene allows cells to function in the presence of concentrations<br>
of an herbicide that are otherwise toxic to cells including plant cells and bacterial<br>
cells. In one aspect of the invention, the grg8 gene is useful as a marker to assess<br>
transformation of bacterial or plant cells.<br>
By engineering grg8 to be (1) expressed from a bacterial promoter known to<br>
stimulate transcription in the organism to be tested, (2) properly translated to generate<br>
an intact GRG8 peptide, and (3) placing the cells in an otherwise toxic concentration<br>
of herbicide, cells that have been transformed with DNA by virtue of their resistance<br>
to herbicide can be identified. By "promoter" is intended a nucleic acid sequence that<br>
16<br><br>
WO 2006/110188	PCT/US2005/046608<br>
functions to direct transcription of a downstream coding sequence. The promoter,<br>
together with other transcriptional and translational regulatory nucleic acid sequences,<br>
(also termed "control sequences") are necessary for the expression of a DNA<br>
sequence of interest.<br>
Transformation of bacterial cells is accomplished by one of several techniques<br>
known in the art, including but not limited to, electroporation or chemical<br>
transformation (See, for example, Ausubel (ed.) (1994) Current Protocols in<br>
Molecular Biology (John Wiley and Sons, Inc., Indianapolis, IN)). Markers<br>
conferring resistance to toxic substances are useful in identifying transformed cells<br>
(having taken up and expressed the test DNA) from non-transformed cells (those not<br>
containing or not expressing the test DNA). In one aspect of the invention, the grg8<br>
gene is useful as a marker to assess transformation of bacterial or plant cells.<br>
Transformation of plant cells can be accomplished in a similar fashion. By<br>
"plant" is intended whole plants, plant organs (e.g., leaves, stems, roots, etc.), seeds,<br>
plant cells, propagules, embryos and progeny of the same. Plant cells can be<br>
differentiated or undifferentiated (e.g. callus, suspension culture cells, protoplasts, leaf<br>
cells, root cells, phloem cells, pollen). "Transgenic plants" or "transformed plants" or<br>
"stably transformed" plants, cells or tissues, refer to plants that haye incorporated or<br>
integrated exogenous nucleic acid sequences or DNA fragments into the plant ceil.<br>
By "stable transformation" is intended that the nucleotide construct introduced into a<br>
plant integrates into the genome of the plant and is capable of being inherited by<br>
progeny thereof.<br>
The grg8 gene of the invention may be modified to obtain or enhance<br>
expression in plant cells. The herbicide resistance sequences of the invention may be<br>
provided in expression cassettes for expression in the plant of interest. "Plant<br>
expression cassette" includes DNA constructs that are capable of resulting in the<br>
expression of a protein from an open reading frame in a plant cell. The cassette will<br>
include in the 5 '-3' direction of transcription, a transcriptional initiation region (i.e.,<br>
promoter) operably-linked to a DNA sequence of the invention, and a translational<br>
and transcriptional termination region (i.e., termination region) functional in plants.<br>
The cassette may additionally contain at least one additional gene to be co-<br>
transformed into the organism, such as a selectable marker gene. Alternatively, the<br>
17<br><br>
WO 2006/110188	PCT/US2005/046608<br>
additional gene(s) can be provided on multiple expression cassettes. Such an<br>
expression cassette is provided with a plurality of restriction sites for insertion of the<br>
herbicide resistance sequence to be under the transcriptionai regulation of the<br>
regulatory regions.<br>
The promoter may be native or analogous, or foreign or heterologous, to the<br>
plant host and/or to the DNA sequence of the invention. Additionally, the promoter<br>
may be the natural sequence or alternatively a synthetic sequence. Where the<br>
promoter is "native" or "homologous" to the plant host, it is intended that the<br>
promoter is found in the native plant into which the promoter is introduced. Where<br>
the promoter is "foreign" or "heterologous" to the DNA sequence of the invention, it<br>
is intended that the promoter is not the native or naturally occurring promoter for the<br>
operably linked DNA sequence of the invention. "Heterologous" generally refers to<br>
the nucleic acid sequences that are not endogenous to the cell or part of the native<br>
genome in which they are present, and have been added to the cell by infection,<br>
transfection, microinjection, electroporation, microprojection, or the like. By<br>
"operably linked" is intended a functional linkage between a promoter and a second<br>
sequence, wherein the promoter sequence initiates and mediates transcription of the<br>
DNA sequence corresponding to the second sequence, Generally, operably  linked<br>
means that the nucleic acid sequences being linked are contiguous and, where<br>
necessary to join two protein coding regions, contiguous and in the same reading<br>
frame.<br>
Often, such constructs will also contain 5' and 3' untranslated regions. Such<br>
constructs may contain a "signal sequence" or "leader sequence" to facilitate co-<br>
translational or post-translational transport of the peptide of interest to certain<br>
intracellular structures such as the chloroplast (or other plastid), endoplasmic<br>
reticulum, or Golgi apparatus, or to be secreted. For example, the gene can be<br>
engineered to contain a signal peptide to facilitate transfer of the peptide to the<br>
endoplasmic reticulum. By "signal sequence" is intended a sequence that is known or<br>
suspected to result in cotranslational or post-translational peptide transport across the<br>
cell membrane. In eukaryotes, this transport typically involves secretion into the<br>
Golgi apparatus, with some resulting glycosylation. By "leader sequence" is intended<br>
any sequence that when translated, results in an amino acid sequence sufficient to<br>
18<br><br>
WO 2006/110188	PCT/US2005/046608<br>
trigger co-translational transport of the peptide chain to a sub-cellular organelle. Thus,<br>
this includes leader sequences targeting transport and/or glycosylation by passage into<br>
the endoplasmic reticulum, passage to vacuoles, plastids including chloroplasts,<br>
mitochondria, and the like. The plant expression cassette can also be engineered to<br>
contain an intron, such that mRNA processing of the intron is required for expression.<br>
By "3' untranslated region" is intended a nucleotide sequence located<br>
downstream of a coding sequence. Polyadenylation signal sequences and other<br>
sequences encoding regulatory signals capable of affecting the addition of<br>
polyadenylic acid tracts to the 3' end of the mRNA precursor are 3' untranslated<br>
regions. By "5' untranslated region" is intended a nucleotide sequence located<br>
upstream of a coding sequence.<br>
Other upstream or downstream untranslated elements include enhancers.<br>
Enhancers are nucleotide sequences that act to increase the expression of a promoter<br>
region. Enhancers are well known in the art and include, but are not limited to, the<br>
SV40 enhancer region and the 35S enhancer element.<br>
The termination region may be native with the transcriptional initiation region,<br>
may be native with the herbicide resistance sequence of the present invention, or may<br>
be derived from another source. Convenient termination regions are available from<br>
the Ti-plasmid of A. tumefaciens, such as the octopine synthase and nopaline synthase<br>
termination regions. See also Guerineau et al. (1991) Mol. Gen. Genet. 262:141-144;<br>
Proudfoot (1991) Cell 64:611-674; Sanfacon et al. (1991) Genes Dev. 5:141-149;<br>
Mogen et al. (1990) Plant Cell 2:1261-1272; Munroe et al. (1990) Gene 91:151-158;<br>
Ballas et al. (1989) Nucleic Acids Res. 17:7891-7903; and Joshi et al. (1987) Nucleic<br>
AcidRes. 15:9627-9639.<br>
Where appropriate, the gene(s) may be optimized for increased expression in<br>
the transformed host cell. That is, the genes can be synthesized using host cell-<br>
preferred codons for improved expression, or may be synthesized using codons at a<br>
host-preferred codon usage frequency. Generally, the GC content of the gene will be<br>
increased. See, for example, Campbell and Gowri (1990) Plant Physiol. 92:1-11 for a<br>
discussion of host-preferred codon usage. Methods are known in the art for<br>
synthesizing host-preferred genes. See, for example, U.S. Patent Nos. 6,320,100;<br>
6,075,185; 5,380,831; and 5,436,391, U.S. Published Application Nos. 20040005600<br>
19<br><br>
WO 2006/110188	PCT/US2005/046608<br>
and 20010003849, and Murray et al. (1989) Nucleic Acids Res. 17:477-498, herein<br>
incorporated by reference.<br>
In one embodiment, the nucleic acids of interest are targeted to the chioropiast<br>
for expression. In this manner, where the nucleic acid of interest is not directly<br>
inserted into the chioropiast, the expression cassette will additionally contain a nucleic<br>
acid encoding a transit peptide to direct the gene product of interest to the<br>
chloroplasts. Such transit peptides are known in the art. See, for example, Von<br>
Heijne et al. (1991) Plant Mol. Biol. Rep. 9:104-126; Clark et al. (1989) J. Biol.<br>
Chem. 264:17544-17550; Della-Cioppa et al. (1987) Plant Physiol. 84:965-968;<br>
Romer et al. (1993) Biochem. Biophys. Res. Commun. 196:1414-1421; and Shah et al.<br>
(1986) Science 233:478-481.<br>
The nucleic acids of interest to be targeted to the chioropiast may be optimized<br>
for expression in the chioropiast to account for differences in codon usage between<br>
the plant nucleus and this organclle. In this manner, the nucleic acids of interest may<br>
be synthesized using chloroplast-preferred codons. See, for example, U.S. Patent No.<br>
5,380,831, herein incorporated by reference.<br>
Typically this "plant expression cassette" will be inserted into a "plant<br>
transformation vector." By "transformation vector" is intendeds DNA molecule that<br>
is necessary for efficient transformation of a ceil. Such a molecule may consist of one<br>
or more expression cassettes, and may be organized into more than one "vector" DNA<br>
molecule. For example, binary vectors are plant transformation vectors that utilize<br>
two non-contiguous DNA vectors to encode all requisite cis- and trans-acting<br>
functions for transformation of plant cells (Hellens and Mullineaux (2000) Trends in<br>
Plant Science 5:446-451). "Vector" refers to a nucleic acid construct designed for<br>
transfer between different host cells. "Expression vector" refers to a vector that has<br>
the ability to incorporate, integrate and express heterologous DNA sequences or<br>
fragments in a foreign cell.<br>
This plant transfonnation vector may be comprised of one or more DNA<br>
vectors needed for achieving plant transformation. For example, it is a common<br>
practice in the art to utilize plant transformation vectors that are comprised of more<br>
than one contiguous DNA segment. These vectors are often referred to in the art as<br>
"binary vectors." Binary vectors as well as vectors with helper plasmids are most<br>
20<br><br>
WO 2006/110188	PCT/US2005/046608<br>
often used for Agrobacterium-mediated transformation, where the size and<br>
complexity of DNA segments needed to achieve efficient transformation is quite<br>
large, and it is advantageous to separate functions onto separate DNA molecules.<br>
Binary vectors typically contain a plasmid vector that contains the cis-acting<br>
sequences required for T-DNA transfer (such as left border and right border), a<br>
selectable marker that is engineered to be capable of expression in a plant cell, and a<br>
"gene of interest" (a gene engineered to be capable of expression in a plant cell for<br>
which generation of transgenic plants is desired). Also present on this plasmid vector<br>
are sequences required for bacterial replication. The cis-acting sequences are arranged<br>
in a fashion to allow efficient transfer into plant cells and expression therein. For<br>
example, the selectable marker gene and the gene of interest are located between the<br>
left and right borders. Often a second plasmid vector contains the trans-acting factors<br>
that mediate T-DNA transfer from Agrobacterium to plant cells. This plasmid often<br>
contains the virulence functions (Vir genes) that allow infection of plant cells by<br>
Agrobacterium, and transfer of DNA by cleavage at border sequences and vir-<br>
mediated DNA transfer, as in understood in the art (Hellens and Mullineaux (2000)<br>
Trends in Plant Science, 5:446-451). Several types of Agrobacterium strains (e.g.<br>
LBA4404, GV3101, EHAlOl, EHA105, etc.)  can be used  for plant transformation<br>
The second plasmid vector is not necessary for transforming the plants by other<br>
methods such as microprojection, microinjection, electroporation, polyethylene<br>
glycol, etc.<br>
Plant Transformation<br>
Methods of the invention involve introducing a nucleotide construct into a<br>
plant. By "introducing" is intended to present to the plant the nucleotide construct in<br>
such a manner that the construct gains access to the interior of a cell of the plant. The<br>
methods of the invention do not require that a particular method for introducing a<br>
nucleotide construct to a plant is used, only that the nucleotide construct gains access<br>
to the interior of at least one cell of the plant. Methods for introducing nucleotide<br>
constructs into plants are known in the art including, but not limited to, stable<br>
transformation methods, transient transformation methods, and virus-mediated<br>
methods.<br>
21<br><br>
WO 2006/110188	PCT/US2005/046608<br>
In general, plant transformation methods involve transferring heterologous<br>
DNA into target plant cells (e.g. immature or mature embryos, suspension cultures,<br>
undifferentiated cailus, protoplasts, etc.), followed by applying a maximum threshold<br>
level of appropriate selection (depending on the selectable marker gene and in this<br>
case "glyphosate") to recover the transformed plant cells from a group of<br>
untransformed cell mass. Explants are typically transferred to a fresh supply of the<br>
same medium and cultured routinely. Subsequently, the transformed cells are<br>
differentiated into shoots after placing on regeneration medium supplemented with a<br>
maximum threshold level of selecting agent (e.g. "glyphosate"). The shoots are then<br>
transferred to a selective rooting medium for recovering rooted shoot or plantlet. The<br>
transgenic plantlet then grow into mature plants and produce fertile seeds (e.g. Hiei et<br>
al. (1994) The Plant Journal 6:271-282; Ishida et al. (1996) Nature Biotechnology<br>
14:745-750). Explants are typically transferred to a fresh supply of the same medium<br>
and cultured routinely. A general description of the techniques and methods for<br>
generating transgenic plants are found in Ayres and Park (1994) Critical Reviews in<br>
Plant Science 13:219-239 and Bomanineni and Jauhar (1997) Maydica 42:107-120.<br>
Since the transformed material contains many cells; both transformed and non-<br>
transformed cells are present in any piece ofsubjected target callusor tissue or group<br>
of cells. The ability to kill non-transformed cells and allow transformed cells to<br>
proliferate results in transformed plant cultures. Often, the ability to remove non-<br>
transformed cells is a limitation to rapid recovery of transformed plant cells and<br>
successful generation of transgenic plants. Molecular and biochemical methods can be<br>
used to confirm the presence of the integrated heterologous gene of interest in the<br>
genome of transgenic plant.<br>
Generation of transgenic plants may be performed by one of several methods,<br>
including, but not limited to, introduction of heterologous DNA by Agrobacterium<br>
into plant cells (Agrobacterium-mediated transformation), bombardment of plant cells<br>
with heterologous foreign DNA adhered to particles, and various other non-particle<br>
direct-mediated methods (e.g. Hiei et al. (1994) The Plant Journal 6:271-282; Ishida<br>
et al. (1996) Nature Biotechnology 14:745-750; Ayres and Park (1994) Critical<br>
Reviews in Plant Science 13:219-239; Bommineni and Jauhar (1997) Maydica<br>
42:107-120) to transfer DNA.<br>
22<br><br>
WO 2006/110188	PCT/US2005/046608<br>
Methods for transformation of chloroplasts are known in the art. See, for<br>
example, Svab et al. (1990) Proc. Natl. Acad. Sci. USA 87:8526-8530; Svab and<br>
Maliga (1993) Proc. Natl. Acad. Sci. USA 90:913-917; Svab and Maliga (1993)<br>
EMBO J. 12:601-606. The method relies on particle gun delivery of DNA containing<br>
a selectable marker and targeting of the DNA to the plastid genome through<br>
homologous recombination. Additionally, plastid transformation can be<br>
accomplished by transactivation of a silent plastid-borne transgene by tissue-preferred<br>
expression of a nuclear-encoded and plastid-directed RNA polymerase. Such a<br>
system has been reported in McBride et al. (1994) Proc. Natl. Acad. Sci. USA<br>
91:7301-7305.<br>
The cells that have been transformed may be grown into plants in accordance<br>
with conventional ways. See, for example, McCormick et al. (1986) Plant Cell<br>
Reports 5:81-84. These plants may then be grown, and either pollinated with the<br>
same transformed strain or different strains, and the resulting hybrid having<br>
constitutive expression of the desired phenotypic characteristic identified. Two or<br>
more generations may be grown to ensure that expression of the desired phenotypic<br>
characteristic is stably maintained and inherited and then seeds harvested to ensure<br>
expression of the desired phenotypic characteristic has been a chieved. In this manner,<br>
the present invention provides transformed seed (also referred to as "transgenic seed")<br>
having a nucleotide construct of the invention, for example, an expression cassette of<br>
the invention, stably incorporated into their genome.<br>
Plants<br>
The present invention may be used for transformation of any plant species,<br>
including, but not limited to, monocots and dicots. Examples of plants of interest<br>
include, but are not limited to, corn (maize), sorghum, wheat, sunflower, tomato,<br>
crucifers, peppers, potato, cotton, rice, soybean, sugarbeet, sugarcane, tobacco, barley,<br>
and oilseed rape, Brassica sp., alfalfa, rye, millet, safflower, peanuts, sweet potato,<br>
cassava, coffee, coconut, pineapple, citrus trees, cocoa, tea, banana, avocado, fig, guava,<br>
mango, olive, papaya, cashew, macadamia, almond, oats, vegetables, ornamentals, and<br>
conifers.<br>
23<br><br>
WO 2006/110188	PCT/US2005/046608<br>
Vegetables include, but are not limited to, tomatoes, lettuce, green beans, lima<br>
beans, peas, and members of the genus Curcumis such as cucumber, cantaloupe, and<br>
musk melon. Ornamentals include, but are not limited to, azalea, hydrangea, hibiscus,<br>
roses, tulips, daffodils, petunias, carnation, poinsettia, and chrysanthemum. Preferably,<br>
plants of the present invention are crop plants (for example, maize, sorghum, wheat,<br>
sunflower, tomato, crucifers, peppers, potato, cotton, rice, soybean, sugarbeet,<br>
sugarcane, tobacco, barley, oilseed rape, etc.).<br>
This invention is particularly suitable for any member of the monocot plant<br>
family including, but not limited to, maize, rice, barley, oats, wheat, sorghum, rye,<br>
sugarcane, pineapple, yams, onion, banana, coconut, and dates.<br>
Evaluation of Plant Transformation<br>
Following introduction of heterologous foreign DNA into plant cells, the<br>
transformation or integration of the heterologous gene in the plant genome is<br>
confirmed by various methods such as analysis of nucleic acids, proteins and<br>
metabolites associated with the integrated gene.<br>
PCR analysis is a rapid method to screen transformed cells, tissue or shoots for<br>
the presence  of incorporated gene at the earlier slage before transplanting into the soil<br>
(Sambrook and Russeli (2001) Molecular Cloning: A Laboratory Manual (Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, NY)). PCR is carried out using<br>
oligonucleotide primers specific to the gene of interest or Agrobacterium vector<br>
background, etc.<br>
Plant transformation may be confirmed by Southern blot analysis of genomic<br>
DNA (Sambrook and Russell (2001) supra). In general, total DNA is extracted from<br>
the transformant, digested with appropriate restriction enzymes, fractionated in an<br>
agarose gel and transferred to a nitrocellulose or nylon membrane. The membrane or<br>
"blot" can then be probed with, for example, radiolabeled 32P target DNA fragment to<br>
confirm the integration of the introduced gene in the plant genome according to<br>
standard techniques (Sambrook and Russell, 2001, supra).<br>
In Northern analysis, RNA is isolated from specific tissues of transformant,<br>
fractionated in a formaldehyde agarose gel, and blotted onto a nylon filter according<br>
to standard procedures that are routinely used in the art (Sambrook and Russell (2001)<br>
24<br><br>
WO 2006/110188	PCT/US2005/046608<br>
supra). Expression of RNA encoded by the grg8 is then tested by hybridizing the<br>
filter to a radioactive probe derived from a GDC by methods known in the art<br>
(Sambrook and Russell (2001) supra)<br>
Western blot and biochemical assays and the like may be carried out on the<br>
transgenic plants to determine the presence of protein encoded by the herbicide<br>
resistance gene by standard procedures (Sambrook and Russell (2001) supra) using<br>
antibodies that bind to one or more epitopes present on the herbicide resistance<br>
protein.<br>
The following examples are offered by way of illustration and not by way of<br>
limitation.<br>
EXPERIMENTAL<br>
Example 1: Isolation of ATX4145<br>
ATX4145 was isolated by plating samples of soil on Enriched Minimal Media<br>
(EMM) containing glyphosate as the sole source of phosphorus. Since EMM contains<br>
no aromatic amino acids, a strain must be resistant to glyphosate in order to grow on<br>
this media.<br>
Two grams of soil were suspended in approximately 30 ml of water, and<br>
sonicated for 30 seconds in a sonicating water bath. The sample was vortexed for 5<br>
seconds and permitted to settle for 60 seconds. This process was repeated 3 times.<br>
100 ml of this suspension was added to 3 ml of EMM supplemented with 4 mM<br>
glyphosate (pH 6.0). EMM contains (per 900 mls): 10 g sucrose, 2 g NaNO3, 1.0 ml<br>
0.8 M MgSO4,1.0 ml 0.1 M CaCl2,1.0 ml Trace Elements Solution (In 100 ml of<br>
l000x solution: 0.1 g FeSO4 7H2O,0.5 mg CuSO4-5H2O,1.0 mg H3BO3,1.0 mg<br>
MnSO4-5H2O,7.0 mg ZnSO4-7H20,1.0 mg MoO3,4.0 g KC1). The culture was<br>
shaken on a tissue culture roller drum for sixteen days at 21°C and then 100 ml was<br>
used to inoculate 3 ml of fresh EMM containing 4 mM glyphosate as the only<br>
phosphorus source. After five days, 100 ml was used to inoculate another fresh 3 ml<br>
culture. After sufficient growth, the culture was plated onto solid media by streaking<br>
a 1 ml loop onto the surface of agar plate containing EMM agar containing 5 mM<br>
glyphosate as the sole phosphorus source and stored at 21°C. The culture was then<br>
replated for isolation. One particular strain, designated ATX4145, was selected due<br>
25<br><br>
WO 2006/110188	PCT/US2005/046608<br>
to its ability to grow in the presence of high glyphosate concentrations. On Luria<br>
Bertani (LB) agar, colonies are white, circular, pinsize to 1 mm and stain Gram<br>
negative.<br>
Example 2. Preparation and Screening of Cosmid Libraries<br>
Total DNA was extracted from a culture of ATX4145 using methods<br>
commonly known in the art. The DNA was partially digested with restriction enzyme<br>
Sau3AI and ligated with SuperCos (Stratagene) vector fragment according to the<br>
manufacturer's directions. Ligation products were packaged into phage particles<br>
using GigaPack III XL packaging extract (Stratagene), transfected into E. coli cells,<br>
and plated on LB Agar containing 50 mg/ml kanamycin to select for colonies<br>
containing cosmids. Approximately 1100 colonies were picked for screening.<br>
Colonies were grown in rich liquid medium containing 50 mg/ml kanamycin,<br>
then pinned onto M63 agar medium containing 50 mg/rnl kanamycin and 7 mM<br>
glyphosate. M63 agar medium contains 100 mM KH2PO4, 15 mM (NH4SO4, 50<br>
mM CaCl2, 1 mM FeSO4, 50 mM MgCl2, 55 mM glucose, 25 mg/liter L-proline, 10<br>
mg/liter thiamine HC1, sufficient NaOH to adjust the pH to 7.0, and 15 g/liter agar.<br>
Several colonies which grewin the presence of 7 mM glyphosate were identified.<br>
Cosmid DNA was prepared from each of these colonies and re-transformed into E.<br>
coli XL1 Blue MRF cells. In each case cells retransformed with cosmid DNA grew<br>
on M63 medium in the presence of 5 mM glyphosate while cells containing empty<br>
SuperCos vector did not. One cosmid, designated 3-M5, was selected for further<br>
characterization. This cosmid was later renamed pAX298.<br>
Example 3. Identification of grg8 in Cosmid pAX298<br>
To identify the gene(s) responsible for the glyphosate-resistance shown by<br>
cosmid pAX298, DNA from this clone was mutagenized with transposable elements.<br>
In this method, one identifies clones that have suffered transposon insertions, and<br>
have lost the ability to confer glyphosate resistance. The location of the transposon<br>
insertions identifies the open reading frame responsible for the glyphosate resistance<br>
phenotype.<br>
26<br><br>
WO 2006/110188	PCT/US2005/046608<br>
Cosmid pAX298 was subjected to in vitro transposon mutagenesis using an<br>
EZ::TN Insertion Kit (Epicentre, Madison, WT) and the manufacturer's protocol. This<br>
process randomly inserts a transposon fragment into the cosmid DNA and thus<br>
randomly disrupts the function of genes in the cosmid. This particular transposon<br>
contains a gene encoding resistance to trimethoprim, so transposon insertion clones<br>
may be selected by the ability to grow in the presence of that antibiotic. The locations<br>
of the transposon insertions may be determined by restriction fragment mapping or by<br>
sequencing with primers which anneal in the transposon. Transposon insertion clones<br>
of pAX298 were plated on M63 medium containing glyphosate. Three clones were<br>
found which had lost the ability to grow in the presence of glyphosate, indicating that<br>
the transposon had disrupted the gene responsible for resistance.<br>
The DNA sequence was determined for the region of pAX298 containing the<br>
transposon insertions using sequencing methods well known in the art and is<br>
presented below. An open reading frame (ORF, bases 296 through 1555 of SEQ ID<br>
NO: 1) was identified. Analysis of sequence data from four transposon insertion picks<br>
that had lost resistance to glyphosate revealed that all four insertions were within the<br>
ORF. This indicates that the ORF encodes the resistance conferred by the cosmid.<br>
Example 4. Homology of GRGS with Other Proteins<br>
The deduced amino acid sequence of this ORF has homology to EPSP<br>
synthase enzymes, indicating that the ORF encodes an EPSP synthase. Cosmid<br>
pAX298 was transformed into E. coli aroA-, a strain in which the native aroA gene,<br>
encoding EPSP synthase, has been deleted. This strain cannot grow on M63 medium<br>
because it requires exogenously supplied aromatic amino acids. The presence of<br>
cosmid pAX298 complemented the aroA- phenotype, that is, it allowed the strain to<br>
grow on M63 medium without exogenously supplied aromatic amino acids. This is<br>
further evidence that the cosmid contains an EPSP synthase gene. This gene was<br>
named grg8.<br>
Examination of the deduced amino acid sequence (SEQ ID NO:3) revealed<br>
that it does not contain the four domains typical of Class II EPSP synthase enzymes.<br>
Thus it is a novel, non-Class II, glyphosate-resistant EPSP synthase.<br>
27<br><br>
WO 2006/110188	PCT/US2005/046608<br>
Example 5. Engineering of grg8 for Expression of GRG8 Protein in E. coli<br>
The grg8 open reading frame (ORF) was amplified by PCR, cloned into the<br>
plasmid vector pCR-Blunt-II from Invitrogen, and transformed into E. coli strain<br>
DH5a. Plasmid DNA was prepared and the presence and orientation of the inserts<br>
were determined by restriction digest. One clone contained the ORF in the forward<br>
orientation with respect to the lac promoter in the vector. This plasmid was named<br>
pAX299. The insert was sequenced and the plasmid was tested for the ability to<br>
confer resistance to glyphosate.<br>
Plasmid pAX299 containing the grg8 ORF was deposited at the Agricultural<br>
Research Service Culture Collection (NRRL) on December 21, 2004, and assigned<br>
Accession No. NRRL B-30804.<br>
Example 6. grg8 Confers Resistance to High Levels of Glyphosate<br>
pAX299 was tested for the ability to grow on M63 medium in the presence of<br>
glyphosate using cells containing empty vector as a control. The plasmid pAX296 in<br>
host strain DH5a was used as a negative control. This plasmid contains a fragment of<br>
tobacco genomic DNA cloned into the vector pCR-Blunt-II. The results are<br>
summarized in Table 1.  Starter cultures of the two strains were grown in M63 broth<br>
with 50 mg/ml kanamycin to maintain the plasmid. The starter cultures were diluted<br>
to approximately equal cell densities, as determined by measuring OD600, then<br>
diluted 1 to 200 into 3 ml of M63 broth containing 0 to 200 mM glyphosate.<br>
Triplicate 3 ml cultures of each strain in each concentration were grown. Cultures<br>
were grown at 37°C with constant shaking. After about 46 hours, 0.3 ml aliquots<br>
were withdrawn and the OD600 was measured. The results are presented in Table 1.<br>
Values represent means + standard deviations. These results demonstrate that GRG8<br>
confers resistance to high levels of glyphosate.<br>
28<br><br>
WO 2006/110188	PCT/US2005/046608<br>
Table 1. Glyphosate resistance of pAX299 containing grg8<br><br>
Glyphosateconcentration (mM)		Negativecontrol<br>
0	1.18 + 0.029	1.09 ±0.024<br>
10	1.15 ±0.018	0.04 + 0.0008<br>
20	1.10 + 0.043	0.04 + 0.0004<br>
50	1.20+ 0.037	0.04 + 0.0004<br>
100	1.31+0.024	0.04 ± 0.0005<br>
200	0.05 + 0.004	0.04 + 0.0004<br>
Example 7. Purification of GRG8 Expressed as a 6xHis-tagged Protein in E. coli<br>
The grg8 coding region was amplified by PCR using PfuUltra™ DNA<br>
polymerase (Stratagene). Oligonucleotides used to prime PCR are designed to<br>
introduce restriction enzyme recognition sites near the 5' and 3' ends of the resulting<br>
PCR product. The resulting PCR product is digested with Sal I. The digested product<br>
is cloned into the 6xHis-tag expression vector pRSFlb (Novagen), prepared by<br>
digestion with Sal I. The resulting clone contains GRG8 in the same translational<br>
reading frame as, and immediately C-terminal to, the 6xIIis tag. General strategies<br>
for generating such clones, and for expressing proteins containing 6xHis-tag are well<br>
known in the art.<br>
The ability of this clone to confer glyphosate resistance is confirmed by<br>
plating cells onto M63 media containing 5 mM glyphosate. The ability the clone<br>
containing grgS to grow at this concentration of glyphosate is determined. The level<br>
of expression of GRG8 protein may be determined on an SDS-PAGE protein gel.<br>
GRG8 protein can be isolated by purification of the induced GRG8 protein by<br>
chromatography on, for example, Ni-NTA Superflow Resin (Qiagcn), as per<br>
manufacturer's instructions.<br>
Example 8. Mutagenesis of grg8 and isolation of functional variants<br>
The grg8 open reading frame was mutagenized using PCR-based mutagenesis<br>
as known in the art. Plasmid pAX700, containing grg8, was amplified using the<br>
Genemorph® mutagenesis kit (Stratagene) using varying amounts of input DNA. The<br>
resulting PCR products were purified and digested with Bam HI and Hind III,<br>
repurified, and ligated into a modified pRSFlb vector (Novagen). The resulting<br>
libraries were transformed into E. coli, picked into 384 well plates, and grown to<br>
29<br><br>
WO 2006/110188	PCT/US2005/046608<br>
saturation at 37°C. The resulting 384 stock plates were stamped onto minimal media<br>
(M63 plus) supplemented with kanamycin, and with varying concentrations of<br>
giyphosate. Mutagenized clones that conferred resistance to 50 mM glyphosate<br>
(shown in Table 2) were selected. For a subset of these clones, DNA was prepared,<br>
and the DNA sequence of the grg8 variant was determined.<br>
Table 2. Growth of E. coli expressing grg8 variants in the presence of<br>
giyphosate<br><br>
Mutation	Original Name	Growth on 50mM giyphosate<br>
Vector control	-	-<br>
grg8	grg8	+++<br>
grg8m1	C22	+++<br>
grg8m2	N15	+++<br>
grg8m3	N24	+++<br>
grg8m4	Al	+++<br>
grg8m5	B2	+++<br>
grg8m6	B7	+++<br>
grg8m7	B11	+++<br>
grg8m8	C11	+++<br>
grg8m9	E3	+++<br>
grg8m10	F4	+++<br>
Example 9. Engineering grg8 for Plant Transformation<br>
The grg8 open reading frame (ORF) is amplified by polymerase chain<br>
reactions from a full-length cDNA template. HindIII restriction sites are added to<br>
each end of the ORF during PCR. Additionally, the nucleotide sequence ACC is<br>
added immediately 5' to the start codon of the gene to increase translational efficiency<br>
(Kozak (1987) 15:8125-8148; Joshi (1987) Nucleic Acids Research 15:6643-6653).<br>
The PCR product is cloned and sequenced, using techniques well known in the art, to<br>
ensure that no mutations are introduced during PCR.<br>
The plasmid containing the grg8 PCR product is digested with, for example,<br>
Hind III, and the fragment containing the intact ORF is isolated. In this example, the<br>
fragment is cloned into the Hind III site of a plasmid such as pAX200, a plant<br>
expression vector containing the rice actin promoter (McElroy et al. (1991) Molec.<br>
Gen. Genet. 231:150-160) and the PinII terminator (An et al. (1989) The Plant Cell<br>
1:115-122). The promoter - gene - terminator fragment from this intermediate<br>
30<br><br>
WO 2006/110188	PCT/US2005/046608<br>
plasmid is then subcloned into a plasmid such as pSB 11 (Japan Tobacco, Inc.) to form<br>
a final plasmid, referred to herein as pSBHGRG8. pSBHGRG8 is organized such<br>
that the DNA fragment containing the promoter - grg8 — terminator construct may be<br>
excised by double digestion with appropriate restriction enzymes and also used for<br>
transformation into plants by, for example, aerosol beam injection. The structure of<br>
pSBl 1GRG8 is verified by restriction digests and gel electrophoresis and by<br>
sequencing across the various cloning junctions.<br>
The plasmid is mobilized into Agrobacterium tumefaciens strain LBA4404<br>
which also harbors the plasmid pSBl (Japan Tobacco, Inc.), using triparental mating<br>
procedures well known in the art, and plating on media containing spectinomycin.<br>
Plasmid pSBl 1GRG8 carries antibiotic resistance but is a narrow host range plasmid<br>
and cannot replicate in Agrobacterium. Antibiotic-resistant colonies arise when<br>
pSBl 1GRG8 integrates into the broad host range plasmid, such as pSBl, through<br>
homologous recombination. The resulting cointegrate product is verified by Southern<br>
hybridization. The Agrobacterium strain harboring the cointegrate can then be used<br>
to transform maize, for example by the Purelntro method (Japan Tobacco).<br>
Example 10. Transformation of grs8 into Plant Cells<br>
Maize ears are best collected 8-12 days after pollination. Embryos are isolated<br>
from the ears, and embryos 0.8-1.5 mm in size are preferred for use in transformation.<br>
Embryos are plated scutellum side-up on a suitable incubation media, such as<br>
DN62A5S media (3.98 g/L N6 Salts; 1 mL/L (of l000x Stock) N6 Vitamins; 800<br>
mg/L L-Asparagine; 100 mg/L Myo-inositol; 1.4 g/L L-Proline; 100 mg/L casamino<br>
acids; 50 g/L sucrose; 1 mL/L (of 1 mg/mL Stock) 2,4-D). However, media and salts<br>
other than DN62A5S are suitable and are known in the art. Embryos are incubated<br>
overnight at 25°C in the dark. However, it is not necessary per se to incubate the<br>
embryos overnight.<br>
The resulting explants are transferred to mesh squares (30-40 per plate),<br>
transferred onto osmotic media for about 30-45 minutes, then transferred to a beaming<br>
plate (see, for example, PCT Publication No. WO/0138514 and U.S. Patent No.<br>
5,240,842).<br>
31<br><br>
WO 2006/110188	PCT/US2005/046608<br>
DNA constructs designed to express GRG8 in plant cells are accelerated into<br>
plant tissue using an aerosol beam accelerator, using conditions essentially as<br>
described in PCT Publication No. WO/0138514.   After beaming, embryos are<br>
incubated for about 30 min on osmotic media, and placed onto incubation media<br>
overnight at 25°C in the dark. To avoid unduly damaging beamed explants, they are<br>
incubated for at least 24 hours prior to transfer to recovery media. Embryos are then<br>
spread onto recovery period media, for about 5 days, 25°C in the dark, then<br>
transferred to a selection media, Explants are incubated in selection media for up to<br>
eight weeks, depending on the nature and characteristics of the particular selection<br>
utilized. After the selection period, the resulting callus is transferred to embryo<br>
maturation media, until the formation of mature somatic embryos is observed. The<br>
resulting mature somatic embryos are then placed under low light, and the process of<br>
regeneration is initiated by methods known in the art. The resulting shoots are<br>
allowed to root on rooting media, and the resulting plants are transferred to nursery<br>
pots and propagated as transgenic plants.<br>
Materials<br>
Components	per liter	source<br>
Chu'sN6 BasalSalt Mixture(Prod. No. C416)	3.98 g/L	Phytotechnology Labs<br>
Chu'sN6VitaminSolution (Prod.No. C 149)	l mL/L (of l000x Stock)	Phytotechnology Labs<br>
L-Asparagine	800 mg/L	Phytotechnology Labs<br>
Myo-inositol	100 mg/L	Sigma<br>
L-Proline	1.4 g/L	Phytotechnology Labs<br>
Casamino acids	100 mg/L	Fisher Scientific<br>
Sucrose	50 g/L	Phytotechnology Labs<br>
2,4-D (Prod. No.D-7299)	1 mL/L (of 1 mg/mL Stock)	Sigma<br>
DN62A5S Media<br>
Adjust the pH of the solution to pH 5.8 with IN KOH/1N KC1, add Gelrite<br>
(Sigma) to 3 g/L, and autoclave. After cooling to 50°C, add 2 ml/L of a 5 mg/ml<br>
stock solution of Silver Nitrate (Phytotechnology Labs). Recipe yields about 20<br>
plates.<br>
32<br><br>
WO 2006/110188	PCT/US2005/046608<br>
Example 11. Transformation of grg8 into Maize Plant Cells by Azrobacterium-<br>
Mediated Transformation<br>
Ears are best collected 8-12 days after pollination. Embryos are isolated from<br>
the ears, and embryos 0.8-1.5 mm in size are preferred to be used for transformation.<br>
Embryos are plated scutellum side-up on a suitable incubation media, and incubated<br>
overnight at 25°C in the dark. However, it is not necessary per se to incubate the<br>
embryos overnight. Embryos are contacted with an Agrobacterium strain containing<br>
the appropriate vectors for Ti plasmid mediated transfer for about 5-10 min, and then<br>
plated onto co-cultivation media for about 3 days (25°C in the dark). After co-<br>
cultivation, explants are transferred to recovery period media for about five days (at<br>
25°C in the dark). Explants are incubated in selection media for up to eight weeks,<br>
depending on the nature and characteristics of the particular selection utilized. After<br>
the selection period, the resulting callus is transferred to embryo maturation media,<br>
until the formation of mature somatic embryos is observed. The resulting mature<br>
somatic embryos are then placed under low light, and the process of regeneration is<br>
initiated as known in the art. The resulting shoots are allowed to root on rooting<br>
media, and the resulting plants are transferred to nursery pots and propagated as<br>
transgenic plants.<br>
All publications and patent applications mentioned in the specification are<br>
indicative of the level of skill of those skilled in the art to which this invention<br>
pertains. All publications and patent applications are herein incorporated by reference<br>
to the same extent as if each individual publication or patent application was<br>
specifically and individually indicated to be incorporated by reference.<br>
Although the foregoing invention has been described in some detail by way of<br>
illustration and example for purposes of clarity of understanding, it will be obvious<br>
that certain changes and modifications may be practiced within the scope of the<br>
appended claims.<br>
33<br><br>
WO 2006/110188	PCT/US2005/046608<br><br>
WE CLAIM:<br>
1.	A recombinant nucleic acid molecule encoding a 5-enolpyruvylshikimate-3-phosphate<br>
synthase enzyme selected from the group consisting of:<br>
a)	a nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO: 1 or 2, or<br>
a complement thereof;<br>
b)	a nucleic acid molecule comprising a nucleotide sequence having at least 95%<br>
sequence identity to the nucleotide sequence of SEQ ID NO: 1 or 2, or a complement thereof, wherein<br>
said nucleotide sequence encodes a polypeptide having herbicide resistance activity;<br>
c)	the herbicide resistance nucleotide sequence of the DNA insert of the plasmid<br>
deposited as Accession No. NRRL B-30804, or a complement thereof;<br>
d)	a nucleic acid molecule which encodes a polypeptide comprising the amino acid<br>
sequence of SEQ ID NO:3; and,<br>
e)	a nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide<br>
having at least 95% amino acid sequence identity to the amino acid sequence of SEQ ID NO:3,<br>
wherein said polypeptide has herbicide resistance activity.<br><br>
2.	The recombinant nucleic acid molecule as claimed in claim 1, wherein said nucleotide<br>
sequence is a synthetic sequence that has been designed for expression in a plant.<br>
3.	The recombinant nucleic acid molecule as claimed in claim 1, wherein said nucleic acid<br>
molecule is operably linked to a promoter that drives expression of said nucleic acid molecule in a<br>
plant cell.<br>
4.	A vector comprising the nucleic acid molecule as claimed in claim 1, 2, or 3.<br>
5.	The vector as claimed in claim 4, further comprising a nucleic acid molecule encoding a<br>
heterologous polypeptide.<br>
6.	A host cell that contains the vector as claimed in claim 4, wherein said host cell is a<br>
bacterial host cell.<br>
7.	A recombinant polypeptide selected from the group consisting of:<br><br>
a)	a polypeptide comprising the amino acid sequence of SEQ ID NO:3;<br>
b)	a polypeptide encoded by the nucleotide sequence of SEQ ID NO: 1 or 2;<br>
c)	a polypeptide comprising an amino acid sequence having at least 95% sequence<br>
identity to the amino acid sequence of SEQ ID NO:3, wherein said polypeptide has herbicide<br>
resistance activity;<br>
d)	a polypeptide that is encoded by a nucleotide sequence that is at least 95% identical to<br>
the nucleotide sequence of SEQ ID NO: 1 or 2, wherein said polypeptide has herbicide resistance<br>
activity; and,<br><br>
e) a polypeptide that is encoded by the herbicide resistance nucleotide sequence of the<br>
DNA insert of the plasmid deposited as Accession No. NRRL B-30804.<br>
8.	The polypeptide as claimed in claim 7 further comprising a heterologous amino acid<br>
sequence.<br>
9.	The polypeptide as claimed in claim 8, wherein said heterologous amino acid sequence<br>
further comprises one or more of the following:<br><br>
a)	An aspartic acid substituted for a glycine at amino acid 4 of SEQ ID NO:3;<br>
b)	A serine substituted for a proline at amino acid 12 of SEQ ID NO:3;<br>
c)	An alanine substituted for the threonine residue at amino acid 47 of SEQ ID NO:3;<br>
d)	An aspartic acid substituted for a glycine at amino acid 66 of SEQ ID NO:3;<br>
e)	A methionine substituted for a threonine at amino acid 68 of SEQ ID NO:3;<br>
f)	A tyrosine substituted for the aspartic acid residue at amino acid 73 of SEQ ID NO:3;<br>
g)	A methionine substituted for the threonine residue at amino acid 76 of SEQ ID NO:3;<br>
h) A cysteine substituted for the glycine residue at amino acid 81 of SEQ ID NO:3;<br>
i) A leucine substituted for the proline residue at amino acid 87 of SEQ ID NO:3;<br>
j) A valine substituted for the alanine residue at amino acid 89 of SEQ ID NO:3;<br>
k) A leucine substituted for the proline residue at amino acid 127 of SEQ ID NO:3;<br>
1) A glutamic acid substituted for an aspartic acid residue at amino acid 133 of SEQ ID<br>
NO:3;<br>
m) An aspartic acid substituted for a glycine residue at amino acid 139 of SEQ ID NO:3;<br>
n) A cysteine substituted for a glycine residue to amino acid 157 of SEQ ID NO:3;<br>
o) A serine substituted for a glycine residue at amino acid 169 of SEQ ID NO:3;<br>
p) A threonine substituted for an alanine residue at amino acid 188 of SEQ ID NO:3;<br>
q) A phenylalanine substituted for a leucine residue at amino acid 194 of SEQ ID NO:3;<br>
r) A serine substituted for a glycine residue at amino acid 195 of SEQ ID NO:3;<br>
s) A tyrosine substituted for an aspartic acid residue at amino acid 205 of SEQ ID NO:3;<br>
t) A valine substituted for an alanine residue at amino acid 209 of SEQ ID NO:3;<br>
u) A threonine substituted for an alanine residue at amino acid 225 of SEQ ID NO:3;<br>
v) A valine substituted for an alanine residue at amino acid 235 of SEQ ID NO:3;<br>
w) An alanine substituted for a valine residue at amino acid 239 of SEQ ID NO:3;<br>
x)A threonine substituted for a lysine residue at amino acid 260 of SEQ ID NO:3;<br>
y) A methionine substituted for a threonine residue at amino acid 265 of SEQ ID NO:3;<br>
z) A tyrosine substituted for an aspartic acid residue at amino acid 292 of SEQ ID NO:3;<br>
aa) A serine substituted for a glycine residue at amino acid 293 of SEQ ID NO:3;<br>
bb) An alanine substituted for a valine residue at amino acid 317 of SEQ ID NO:3;<br><br>
cc) An asparagine substituted for a serine residue at amino acid 335 of SEQ ID NO:3;<br>
dd) A serine substituted for an alanine residue at amino acid 355 of SEQ ID NO:3;<br>
ee) An asparagine substituted for an isoleucine residue at amino acid 364 of SEQ ID<br>
NO:3; or,<br>
ff) A valine substituted for an alanine residue at amino acid 382 of SEQ ID NO:3.<br>
10	A recombinant nucleic acid encoding the polypeptide as claimed in claim 1, wherein said<br>
nucleic acid further encodes one or more of the following:<br>
a)	An aspartic acid substituted for a glycine at amino acid 4 of SEQ ID NO:3;<br>
b)	A serine substituted for a proline at amino acid 12 of SEQ ID NO:3;<br>
c)	An alanine substituted for the threonine residue at amino acid 47 of SEQ ID NO:3;<br>
d)	An aspartic acid substituted for a glycine at amino acid 66 of SEQ ID NO:3;<br>
e)	A methionine substituted for a threonine at amino acid 68 of SEQ ID NO:3;<br>
f)	A tyrosine substituted for the aspartic acid residue at amino acid 73 of SEQ ID NO:3;<br>
g)	A methionine substituted for the threonine residue at amino acid 76 of SEQ ID NO:3;<br>
h) A cysteine substituted for the glycine residue at amino acid 81 of SEQ ID NO: 3;<br>
i) A leucine substituted for the proline residue at amino acid 87 of SEQ ID NO:3;<br>
j) A valine substituted for the alanine residue at amino acid 89 of SEQ ID NO:3;<br>
k) A leucine substituted for the proline residue at amino acid 127 of SEQ ID NO:3;<br>
1) A glutamic acid substituted for an aspartic acid residue at amino acid 133 of SEQ ID<br>
NO:3;<br>
m) An aspartic acid substituted for a glycine residue at amino acid 139 of SEQ ID NO:3;<br>
n) A cysteine substituted for a glycine residue to amino acid 157 of SEQ ID NO:3;<br>
o) A serine substituted for a glycine residue at amino acid 169 of SEQ ID NO:3;<br>
p) A threonine substituted for an alanine residue at amino acid 188 of SEQ ID NO:3;<br>
q) A phenylalanine substituted for a leucine residue at amino acid 194 of SEQ ID NO:3;<br>
r) A serine substituted for a glycine residue at amino acid 195 of SEQ ID NO:3;<br>
s) A tyrosine substituted for an aspartic acid residue at amino acid 205 of SEQ ID NO:3;<br>
t) A valine substituted for an alanine residue at amino acid 209 of SEQ ID NO:3;<br>
u) A threonine substituted for an alanine residue at amino acid 225 of SEQ ID NO:3;<br>
v) A valine substituted for an alanine residue at amino acid 235 of SEQ ID NO:3;<br>
w) An alanine substituted for a valine residue at amino acid 239 of SEQ ID NO:3;<br>
x) A threonine substituted for a lysine residue at amino acid 260 of SEQ ID NO:3;<br>
y) A methionine substituted for a threonine residue at amino acid 265 of SEQ ID NO:3;<br>
z) A tyrosine substituted for an aspartic acid residue at amino acid 292 of SEQ ID NO:3;<br><br>
aa) A serine substituted for a glycine residue at amino acid 293 of SEQ ID NO:3;<br>
bb) An alanine substituted for a valine residue at amino acid 317 of SEQ ID NO:3;<br>
cc) An asparagine substituted for a serine residue at amino acid 335 of SEQ ID NO:3;<br>
dd) A serine substituted for an alanine residue at amino acid 355 of SEQ ID NO:3;<br>
ee) An asparagine substituted for an isoleucine residue at amino acid 364 of SEQ ID<br>
NO:3; or,<br>
ff) A valine substituted for an alanine residue at amino acid 382 of SEQ ID NO:3.<br>
11.	A method for producing a recombinant polypeptide with herbicide resistance activity,<br>
comprising culturing the host cell as claimed in claim 6 under conditions in which a nucleic acid<br>
molecule encoding the polypeptide is expressed, said polypeptide being selected from the group<br>
consisting of:<br>
a)	a polypeptide comprising the amino acid sequence of SEQ ID NO:3;<br>
b)	a polypeptide encoded by the nucleic acid sequence of SEQ ID NO: 1 or 2;<br>
c)	a polypeptide comprising an amino acid sequence having at least 95% sequence<br>
identity to a polypeptide with the amino acid sequence of SEQ ID NO:3, wherein said polypeptide has<br>
herbicide resistance activity; and<br>
d)	a polypeptide encoded by a nucleic acid molecule comprising a nucleotide sequence<br>
having at least 95% sequence identity to the nucleic acid sequence of SEQ ID NO:1 or 2, wherein said<br>
polypeptide has herbicide resistance activity; and,<br>
e)	a polypeptide that is encoded by the herbicide resistance nucleotide sequence of the<br>
DNA insert of the plasmid deposited as Accession No. NRRL B-30804.<br><br>
12.	A method for conferring resistance to an herbicide in a plant, said method comprising<br>
transforming said plant with a DNA construct, said construct comprising a promoter that drives<br>
expression in a plant cell operably linked with a nucleotide sequence as claimed in claim 1, and<br>
regenerating a transformed plant.<br>
13.	The method as claimed in claim 12, wherein said herbicide is a glyphosate.<br><br>
ABSTRACT<br><br>
The invention discloses a recombinant nucleic acid molecule encoding a 5-<br>
enolpyruvylshikimate-3-phosphate synthase enzyme selected from the group consisting of: a)<br>
a nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO:1 or 2, or a<br>
complement thereof; b) a nucleic acid molecule comprising a nucleotide sequence having at<br>
least 95% sequence identity to the nucleotide sequence of SEQ ID NO: 1 or 2, or a<br>
complement thereof, wherein said nucleotide sequence encodes a polypeptide having<br>
herbicide resistance activity; c) the herbicide resistance nucleotide sequence of the DNA<br>
insert of the plasmid deposited as Accession No. NRRL B-30804, or a complement thereof;<br>
d) a nucleic acid molecule which encodes a polypeptide comprising the amino acid sequence<br>
of SEQ ID NO:3; and, e) a nucleic acid molecule comprising a nucleotide sequence encoding<br>
a polypeptide having at least 95% amino acid sequence identity to the amino acid sequence of<br>
SEQ ID NO:3, wherein said polypeptide has herbicide resistance activity.<br>
The invention is also for a vector comprising said molecule and a polypeptide encoded<br>
by it having herbicide resistance.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-correspondence others-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1mb3JtIDMtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-form 3-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MDAta29sbnAtMjAwNy1wcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">02500-kolnp-2007-priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA1LSgwNC0wNi0yMDEzKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2005-(04-06-2013)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA1LSgwNC0wNi0yMDEzKS1DTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2005-(04-06-2013)-CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA1LSgwNC0wNi0yMDEzKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2005-(04-06-2013)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA1LSgwNC0wNi0yMDEzKS1GT1JNIDEucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2005-(04-06-2013)-FORM 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA1LSgwNC0wNi0yMDEzKS1GT1JNIDIucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2005-(04-06-2013)-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgwMS0wNi0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(01-06-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgwNC0wNy0yMDEyKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(04-07-2012)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgwNC0wNy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(04-07-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgwNC0wNy0yMDEyKS1GT1JNLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(04-07-2012)-FORM-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgwNC0wNy0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(04-07-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1BTUFOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04ucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-AMANDED PAGES OF SPECIFICATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1ERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1EUkFXSU5HUy5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-DRAWINGS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1FWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-EXAMINATION REPORT REPLY RECEIVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1GT1JNLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-FORM-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1GT1JNLTIucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-FORM-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1GT1JNLTMucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-FORM-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1GT1JNLTUucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-FORM-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1QRVRJVElPTiBVTkRFUiBSVUxFIDEzNy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-PETITION UNDER RULE 137-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1LT0xOUC0yMDA3LSgyMC0wMy0yMDEyKS1QRVRJVElPTiBVTkRFUiBSVUxFIDEzNy5wZGY=" target="_blank" style="word-wrap:break-word;">2500-KOLNP-2007-(20-03-2012)-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUwMC1rb2xucC0yMDA3LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">2500-kolnp-2007-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QtMDI1MDAta29sbnAtMjAwNy5qcGc=" target="_blank" style="word-wrap:break-word;">abstract-02500-kolnp-2007.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="256540-method-to-enable-combination-of-network-controlled-mobility-and-user-equipment-controlled-mobility-between-different-internet-protocol-versions.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="256542-polypropylene-film-based-sheets.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>256541</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2500/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>27/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Jul-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Jun-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ATHENIX CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2202 ELLIS ROAD, SUITE B DURHAM, NC</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HAMMER, PHILIP,E</td>
											<td>15 COVINGTON SQUARE DRIVE, CARY, NC 27513</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HINSON, TODD, K</td>
											<td>3510 MORIAH ROAD, ROUGEMONT, NC 27572</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82,A01H 5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/046608</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/640195</td>
									<td>2004-12-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/256541-recombinant-nucleic-acid-molecule-vector-comprising-it-and-polypeptide-encoded-by-it by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:26:26 GMT -->
</html>
