<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/256784-process-for-producing-heterologous-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:10:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 256784:&quot;PROCESS FOR PRODUCING HETEROLOGOUS POLYPEPTIDES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PROCESS FOR PRODUCING HETEROLOGOUS POLYPEPTIDES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A process is described for producing a polypeptide heterologous to E. coli wherein E. coli cells comprising nucleic acid encoding the polypeptide are cultured in a culture medium while feeding to the culture medium a transportable organophosphate, such that the nucleic acid is expressed. The polypeptide is then recovered from the cells.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BACKGROUND OF THE INVENTION<br>
Related Applications<br>
This application claims priority to U.S. Provisional Application No.: 60/552,678<br>
filed March 11,2004, to which U.S. Provisional Application this application claims<br>
priority under 35 U.S.C. ยง119, the contents of which are incorporated herein by reference.<br>
1. Field of die Invention<br>
The invention relates to a process for producing a polypeptide heterologous to E.<br>
coll. More particularly, the invention is directed to using organophosphate to improve<br>
yield of such polypeptides.<br>
2. Description of Related Art<br>
Expression of heterologous proteins by Escherichia coli, aided by the wellunderstood<br>
molecular biology and relative ease in genetic manipulation of the<br>
microorganism, has been very productive in both laboratory and industry. Typically, an<br>
inducible promoter (for example, the alkaline phosphatase promoter, the tac promoter, the<br>
arabinose promoter, etc.) is employed for the regulation of heterologous protein<br>
expression. The requirement of an induction event provides the researcher the<br>
opportunity to manage the timing of expression of the target protein. This ability is<br>
especially important for those heterologous proteins that are not well tolerated at high<br>
concentrations by the host. By achieving desirable cell density prior to the induction of<br>
expression, the volumetric yield of the desired protein may be maximized.<br>
Cells cease to grow when the microorganism is deprived of a required nutrient.<br>
The limiting component may be carbon, nitrogen, phosphate, oxygen or any of the<br>
elements required by the cell. Under such conditions, the cells exit from the growth phase.<br>
A way to alleviate the culture of the stress responses caused by the nutrient limitation is to<br>
provide a feed of the lacking component. Common feeds introduced into fed-batch<br>
fermentation processes include glucose, amino acids, oxygen, etc.<br>
In the case of cellular phosphorus (P), the requirement for phosphate supply is not<br>
surprising given that P is the fifth most abundant element in a cell behind carbon, oxygen,<br>
nitrogen, and hydrogen. Slanier, Adelberg and Ingraham, The Microbial World. 4th ed.<br>
(Prentice Hall, NJ 1976), p. 1357. Phosphorus is an essential component in numerous<br>
macromolecules such as nucleic acids, liposaccharides and membrane lipids. Furthermore,<br>
its role in the high-energy phosphoanhydride bonds makes it especially important in energy<br>
metabolism. E. coli is capable of utilizing inorganic phosphate (Pi), organophosphate or<br>
phosphonate as the primary P source. The uptake of Pi from the environment can be<br>
achieved through two transporter systems, the Pit and the Pst systems. For the<br>
organophosphates, most are non-transportable and they first need to be hydrolyzed<br>
enzymatically in the periplasm before the released Pi can be taken up by the Pi transport<br>
system(s). Only a few organophosphates are transportable, and glycerol-3-phosphate<br>
(G3P) is one such example. G3P and glycerophosphate-1-phosphate (G1P) are known as<br>
alpha-glycerophosphates. In response to Pi-limitation and carbon-limitation, E. coli is<br>
capable of taking up available intact G3P from the external environment into the<br>
intracellular compartment, where G3P is metabolized to yield needed phosphate or carbon.<br>
Wanner, "Phosphorus Assimulation and Control of the Phosphate Regulon", in Escherichia<br>
coli and Salmonella Cellular and Molecular Biology, Neidhardt, ed., (second edition),<br>
American Society for Microbiology Press (1996), pp. 1357-1365.<br>
Further references on G3P are Silhavy et al., J. Bacteriol. 126: 951-958 (1976) on<br>
the periplasmic protein related to the sn-glycerol-3-phosphate transport system of E. coli;<br>
Argast et al, J. Bacteriol. 136: 1070-1083 (1978) on a second transport system for snglycerol-<br>
3-phosphate in E. coli; Elvin et al., J. Bacteriol. 161: 1054-1058 (1985) on Pi<br>
exchange mediated by the g/pr-dependent G3P transport system; Rao et al., J. Bacteriol.<br>
175: 74-79 (1993) on the effect ofglpT and glpD mutations on expression of \lnephoA gene<br>
in E. coli; and Elashvili et al, Appl. Environ. Microbiol. 64: 2601-2608 (1998) onphnE<br>
and glpT genes enhancing utilization of organophosphates in E. coli K-l 2. Further,<br>
Vergeles et al, Eur. J. Biochem.. 233: 442-447 (1995) disclose the high efficiency of<br>
glycerol-2-phosphate (G2P), otherwise known as beta-glycerophosphate, and G3P as<br>
nucleotidyl acceptors in snake venom phosphodiesterase esterifications.<br>
The current understanding of the two transport systems for the uptake of<br>
exogenous G3P in E. coli, the Ugp and GlpT transport systems, has been well<br>
summarized in the book Escherichia coli and Salmonella, Cellular and Molecular Biology<br>
edited by Neidhardt et. al. (second edition), supra, pp. 1364 referring to references 13 and<br>
81. The Ugp operon belongs to ihepho regulon. It is induced by phosphate limitation<br>
and positively regulated byphoB protein. The Ugp system is a periplasmic binding<br>
protein-dependent multi-component transport system, with ugpB encoding the<br>
periplasmic binding protein, ugpA and ugpC encoding integral membrane channel<br>
proteins, and ugpC encoding ATPase. GlpT is part of the glp system that mediates the<br>
uptake and metabolism of glycerol, G3P, and glycerol phosphoryl phosphodiesters (Lin et<br>
al, Annu. Rev. Microbiol. 30- 535-578 (1976); Chapter 20; pg 307-342 Dissimilatory<br>
Pathways for sugars, polyols and carboxylates. Escherichia coli and Salmonella, Cellular<br>
and Molecular Biology, second edition). This transport system is an anion exchanger that<br>
is known to mediate the efflux of Pi from the cytoplasm by exchange with external G3P.<br>
In a wild-type strain growing on G3P, while little Pi is released by cells taking up G3P<br>
via the Ugp system, Pi can be released into the periplasm when G3P is taken up via the<br>
GlpT system. If a repressive amount of Pi is released as a result of g/pr-permeasemediated<br>
efflux, the to regulon activity, the Ugp system included, will be shut off.<br>
Under certain conditions, GlpT is the only route for the exit of Pi from the cell by<br>
exchange with external G3P. Elvin et al, J. Bacteriol. 161. 1054-1058 (1985);<br>
Rosenberg, "Phosphate transport in prokaryotes," p. 205-248. In B. P. Rosen and S.<br>
Silver (ed.), Ion Transport in Prokaryotes (Academic Press, Inc., New York, 1987).<br>
When the capacities of the Ugp and the GlpT systems are compared to transport<br>
G3P, the maximal velocities of the two systems are similar. The apparent affinity for<br>
G3P is higher with the Ugp system than with the GlpT system. Likely, both systems will<br>
be able to supply enough G3P for cell growth if available in the growth medium.<br>
However, G3P transported exclusively via the Ugp system can serve as the sole source<br>
only of phosphate but not of carbon, while GlpT-transported G3P can serve as the sole<br>
source for both (Schweizer et al, J Bacteriol. ISO: 1154-1163 (1982)). The two ugp<br>
genes coding for the 2o-regulon-dependent G3P transport system have been mapped<br>
(Schweizer et al, J. Bacteriol. 150: 1164-1171 (1982)), the ugp region containing these<br>
genes has been characterized (Schweizer et al, Mol and Gen. Genetics. 197: 161-168<br>
(1984)), and the regulation of ugp operon studied (Schweizer et al., J. Bacteriol, 163:<br>
392-394 (1985); Kasahara et al, J. Bacteriol. 173: 549-558 (1991); Su et al, Molecular<br>
&amp; General Genetics, 230: 28-32 (1991); Brzoska et al, "wgp-dependent transport system<br>
for sn-glycerol 3-phosphate ofEscherichia coli" p. 170-177 in A. Torriani-Gorini, F. G.<br>
Rothman, S. Silver, A. Wright, and E. Yagil (ed.), Phosphate Metabolism and Cellular<br>
Regulation in Microorganisms (American Society for Microbiology, Washington, D.C.,<br>
1987); Brzoska et al, J. Bacteriol. 176: 15-20 (1994); and Xavier etal, J. Bacteriol.<br>
177: 699-704 (1995)).<br>
In wild-type strains, there exists a stable intracellular pool of G3P and it is<br>
maintained at approximately 200 uM. Internally, G3P can be synthesized by the enzymatic<br>
conversion of glycerol by glycerol kinase (encoded by glpK) to G3P when grown on<br>
glycerol as the sole carbon source, or from the reduction of the glycolytic intermediate,<br>
dihydroxyactone phosphate, by G3P synthase, the gene product of the gpsA gene, during<br>
growth on carbon sources other than glycerol. Since G3P is an important intermediate that<br>
forms the scaffold of all phospholipid molecules, internal glycerol phosphates may also be<br>
generated from the breakdown of phospholipids and triacylglycerol. As a metabolite,<br>
internal G3P may be channeled into the phospholipid biosynthetic pathway or be oxidized<br>
by G3P dehydrogenase to form dihydroxyacetone phosphate and fed into the glycolytic<br>
pathway.<br>
In situations where the AP promoter is employed for regulating heterologous<br>
protein expression in E. coli, since induction occurs only after the medium is depleted of<br>
Pi, cells induced for AP promoter activity are typically starved for phosphate and in a<br>
declining state of health. They may have to scavenge for phosphate needed for cellular<br>
functions. Possible consequences of such phosphate scavenging may include turnover of<br>
ribosomes, lower cell energetics, and increased protease expression and proteolysis (St.<br>
John and Goldberg, J. Bacteriol. 143: 1223-1233 (1980)), potentially leading to less<br>
healthy cells with reduced capacity for protein accumulation.<br>
Improving the metabolic state of E. coll may conceivably increase the capacity of<br>
the cell to synthesize proteins. If phosphate is fed slowly, the cells may only sense low Pi<br>
concentration in the periplasm, thereby inducing ihepho regulon without being starved<br>
intracellularly for the P atom (see U.S. Pat. No. 5,304,472). There is a need for providing<br>
further methods of producing heterologous polypeptides in E. coli.<br>
SUMMARY OF THE INVENTION<br>
In the invention herein, a process is provided for improving the expression of<br>
heterologous polypeptides in E. coli. The feeding of transportable organophosphate such<br>
as an alpha-glycerophosphate to various E. coli hosts, including those with and without<br>
the wild-type glpT gene and those with and without the wild-type phoA gene, such as, for<br>
example, (ugp+ kglpT phoA-) E. coli, is shown to improve the expression of<br>
heterologous protein at both shake-flask and 10-L-fermentor scale, and is expected to<br>
perform similarly at larger scale such as 10,OOOL. Product yield benefit was observed<br>
across multiple model systems that employed a variety of promoters, including inducible<br>
promoters such as the tac, T7 or AP promoter, for the expression of the heterologous<br>
proteins. A further advantage is that the product can be obtained earlier in the active<br>
Figure 2 shows expression of a cytoplasmic Apo2L in aHMS174 E. coli host using<br>
the T7 promoter in a shake-flask culture, utilizing either water or 200 mM G3P as a<br>
supplement in CRAP medium.<br>
Figure 3 shows the effect of feeding of G3P during fermentation on secreted IGF-<br>
1 accumulation over time. This uses a wild-type E. coli host, the AP promoter, and<br>
continuously fed glucose.<br>
Figure 4 shows the effect of a glpT mutation and G3P feeding during fermentation<br>
on secreted IGF-1 accumulation over time. This uses a hglpTE. coli host, the AP<br>
promoter, and varying G3P feed rate.<br>
Figure 5 shows the plasmid diagram for pAPApo2-P2RU.<br>
Figure 6 shows the nucleotide sequence of human Apo-2 ligand cDNA (SEQ ID<br>
NO:1) and its derived amino acid sequence (SEQ ID NO:2). The "N" at nucleotide<br>
position 447 (in SEQ ID NO:1) is used to indicate the nucleotide base may be a "T" or<br>
Figure 7 shows the effect of G3P feeding on specific accumulation of Apo2L in<br>
the Ag/p J E. coli (43F6) host, with three different feed rates and a control with no G3P<br>
feed.<br>
Figure 8 shows the benefit on the specific total accumulation of Apo2L of feeding<br>
glycerophosphate over inorganic phosphate to the wild-type glpT host (43E7),, wherein<br>
the cell density increases to over 200 OD550.<br>
Figure 9 shows the effect on specific total accumulation of Apo2L of replacement<br>
of inorganic phosphate with glycerophosphate in the wild-type glpT E. coli host (43E7)<br>
and kglpTE. coli (43F6) host.<br>
Figure 10 shows the effect on total Apo2L accumulation of replacement of alphaglycerophosphate<br>
with a 50:50 mixture of alpha- and beta-glycerophosphate as a feed,<br>
versus a no-feed control, in a AglpTE. coli (61G1) host.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
Definitions<br>
As used herein, "polypeptide" refers generally to peptides and proteins having<br>
more than about ten amino acids. "Heterologous" polypeptides are those polypeptides<br>
foreign to the host cell being utilized, such as a human protein produced by E. coli. While<br>
the polypeptide may be prokaryotic or eukaryotic, preferably it is eukaryotic, more<br>
preferably mammalian, and most preferably human.<br>
Examples of mammalian polypeptides include molecules such as, e.g., rennin; a<br>
growth hormone, including human growth hormone or bovine growth hormone; growthhormone<br>
releasing factor; parathyroid hormone; thyroid-stimulating hormone;<br>
lipoproteins; 1-antitrypsin; insulin A-chain; insulin B-chain; proinsulin; thrombopoietin;<br>
follicle-stimulating hormone; calcitonin; luteinizing hormone; glucagon; clotting factors<br>
such as factor VIIIC, factor IX, tissue factor, and von Willebrands factor; anti-clotting<br>
factors such as Protein C; atrial naturietic factor; lung surfactant; a plasminogen activator,<br>
such as urokinase or human urine or tissue-type plasminogen activator (t-PA); bombesin;<br>
thrombin; hemopoietic growth factor; tumor necrosis factor-alpha and -beta; antibodies to<br>
ErbB2 domain(s) such as 2C4 (WO 01/00245; hybridoma ATCC HB-12697), which binds<br>
to a region in the extracellular domain of ErbB2 (e.g., any one or more residues in the<br>
region from about residue 22 to about residue 584 of ErbB2, inclusive); enkephalinase;<br>
mullerian-inhibiting substance; relaxin A-chain; relaxin B-chain; prorelaxin; mouse<br>
gonadotropin-associated peptide; a microbial protein, such as beta-lactamase; DNase;<br>
inhibin; activin; vascular endothelial growth factor (VEGF); receptors for hormones or<br>
growth factors; integrin; protein A or D; rheumatoid factors; a neurotrophic factor such as<br>
brain-derived neurotrophic factor (BDNF), neurotrophin-3, -4, -5, or -6 (NT-3, NT-4, NT-<br>
5, or NT-6), or a nerve growth factor such as NGF; cardiotrophins (cardiac hypertrophy<br>
factor) such as cardiotrophin-1 (CT-1); platelet-derived growth factor (PDGF); fibroblast<br>
growth factor such as aFGF and bFGF; epidermal growth factor (EGF); transforming<br>
growth factor (TGF) such as TGF-alpha and TGF-beta, including TGF- 1, TGF- 2, TGF-<br>
3, TGF- 4, or TGF- 5; insulin-like growth factor-I and -II (IGF-I and IGF-II); des(l-3)-<br>
IGF-I (brain IGF-I); insulin-like growth factor binding proteins; CD proteins such as CD-<br>
3, CD-4, CD-8, and CD-I9; erythropoietm; osteoinductive factors; immunotoxins; a bone<br>
morphogenetic protein (BMP); an interferon such as interferon-alpha, -beta, and -gamma;<br>
a serum albumin, such as human serum albumin (HSA) or bovine serum albumin (BSA);<br>
colony stimulating factors (CSFs), e.g., M-CSF, GM-CSF, and G-CSF; interleukins (ILs),<br>
e.g., IL-1 to IL-10; anti-HER-2 antibody; Apo2 ligand (Apo2L); superoxide dismutase; Tcell<br>
receptors; surface-membrane proteins; decay-accelerating factor; viral antigens such<br>
as, for example, a portion of the AIDS envelope; transport proteins; homing receptors;<br>
addressins; regulatory proteins; antibodies; and fragments of any of the above-listed<br>
polypeptides.<br>
The preferred polypeptides of interest include polypeptides such as HSA, BSA,<br>
anti-lgE, anti-CD20, anti-IgG, t-PA, gp!20, anti-CDlla, anti-CD18, 2C4, anti-VEGF,<br>
VEGF, TGF-beta, activin, inhibin, anti-HER-2, DNase, IGF-I, IGF-II, brain IGF-I, growth<br>
hormone, relaxin chains, growth- hormone releasing factor, insulin chains or pro-insulin,<br>
antibodies and antibody fragments, NGF, NT-3, BDNF, Apo2L, and urokinase. The<br>
polypeptide is most preferably IGF-1 or Apo2L.<br>
The terms "Apo2 ligand," "Apo2L," and "TRAIL" are used herein interchangeably<br>
to refer to a polypeptide sequence that includes amino acid residues 114-281, inclusive,<br>
residues 95-281, inclusive, residues 92-281, inclusive, residues 91-281, inclusive, residues<br>
41-281, inclusive, residues 15-281, inclusive, or residues 1-281, inclusive, of the amino<br>
acid sequence shown in Figure 6 (SEQ ID NO:2), as well as biologically active fragments,<br>
and deletional, insertional, or substitutional variants of the above sequences. In one<br>
embodiment, the polypeptide sequence comprises residues 114-281 of Figure 6 (SEQ ID<br>
NO:2). Optionally, the polypeptide sequence comprises residues 92-281 or residues 91-<br>
281 of Figure 6 (SEQ ID NO:2). The Apo2L polypeptides may be encoded by the native<br>
nucleotide sequence shown in Figure 6 (SEQ ID NO: 1). Optionally, the codon that<br>
encodes residue Prol 19 (Figure 6; SEQ ID NO:1) may be "CCT" or "CCG." In another<br>
preferred embodiment, the fragments or variants are biologically active and have at least<br>
about 80% amino acid sequence identity, more preferably at least about 90% sequence<br>
identity, and even more preferably, at least 95%, 96%, 97%, 98%, or 99% sequence<br>
identity, with any one of the above sequences. The definition encompasses substitutional<br>
variants of Apo2 ligand in which at least one of its native amino acids is substituted by an<br>
alanine residue. The definition also encompasses a native-sequence Apo2 ligand isolated<br>
from an Apo2 ligand source or prepared by recombinant or synthetic methods. The Apo2<br>
ligand of the invention includes the polypeptides referred to as Apo2 ligand or TRAIL<br>
disclosed in WO 97/01633, WO 97/25428, and WO 01/00832. The terms "Apo2 ligand"<br>
and "Apo2L" are used to refer generally to forms of the Apo2 ligand that include<br>
monomer, dimer, or trimer forms of the polypeptide. All numbering of amino acid<br>
residues referred to in the Apo2L sequence uses the numbering according to Figure 6<br>
(SEQ ID NO:2) unless specifically stated otherwise. For instance, "D203" or "Asp203"<br>
refers to the aspartic acid residue at position 203 in the sequence provided in Figure 6<br>
(SEQ ID NO:2).<br>
The term "Apo-2 ligand extracellular domain" or "Apo2 ligand BCD" refers to a<br>
form of Apo2 ligand that is essentially free of transmembrane and cytoplasmic domains.<br>
Ordinarily, the BCD will have less than 1% of such transmembrane and cytoplasmic<br>
domains, and preferably will have less than 0.5% of such domains. "Biologically active"<br>
or "biological activity," as it relates to Apo2L, refers to (a) having the ability to induce or<br>
stimulate apoptosis in at least one type of mammalian cancer cell or virally infected cell in<br>
vivo or ex vivo; (b) capable of raising an antibody (i.e., immunogenic), (c) capable of<br>
binding and/or stimulating a receptor for Apo2L; or (d) retaining the activity of a native or<br>
naturally occurring Apo2L polypeptide.<br>
The expression "control sequences" refers to DNA sequences necessary for the<br>
expression of an operably linked coding sequence in a particular host organism. The<br>
control sequences that are suitable for prokaryotes include a promoter, optionally an<br>
operator sequence, and a ribosome-binding site.<br>
Nucleic acid is "operably linked" when it is placed into a functional relationship<br>
with another nucleic acid sequence. For example, DNA for a presequence or secretory<br>
leader is operably linked to DNA for a polypeptide if it is expressed as a pre-protein that<br>
participates in the secretion of the polypeptide; a promoter is operably linked to a coding<br>
sequence if it affects the transcription of the sequence; or a ribosome-binding site is<br>
operably linked to a coding sequence if it is positioned so as to facilitate translation.<br>
Generally, "operably linked" means that the DNA sequences being linked are contiguous,<br>
and, in me case of a secretory leader, contiguous and in reading phase. Linking is<br>
accomplished by ligation at convenient restriction sites. If such sites do not exist, the<br>
synthetic oligonucleotide adaptors or linkers may be used in accordance with conventional<br>
practice.<br>
As used herein, the expressions "cell," "cell line," and "cell culture" are used<br>
interchangeably and all such designations include progeny. Thus, the words<br>
"transformants" and "transformed cells" include the primary subject cell and cultures<br>
derived therefrom without regard for the number of transfers. It is also understood that all<br>
progeny may not be precisely identical in DNA content, due to deliberate or inadvertent<br>
mutations. Mutant progeny that have the same function or biological activity as screened<br>
for in the originally transformed cell are included. Where distinct designations are<br>
intended, it will be clear from the context.<br>
The term "organophosphate" as used herein refers to a phosphate compound<br>
containing<br>
one or more carbon atoms, which can also contain halide atoms. Such phosphate<br>
compound must be such that it can be fed to and utilized by a cell culture. These<br>
compounds are often used as pesticides. "Transportable" organophosphates can be<br>
transported from the external environment of the cell into the cell without having to be<br>
pre-hydrolyzed in any way. If an E. coli strain does not grow well with an<br>
organophosphate, the utilization of such organophosphate can be enhanced by<br>
overexpressing in E. coli thsphnE gene product. Such gene confers the spontaneous<br>
organophosphate utilization phenotype to the E. coli strain upon transformation. See<br>
Elashvili et al, supra. Examples of suitable organophosphates include alkyl<br>
haiophosphates such as diisopropyl fluorophosphate, alkyl phosphates such as diisopropyl<br>
phosphate and 3,4-dihydroxybutyl-l-phosphate, as well as sugar- or alkanol-containing<br>
phosphates such as hexose-6-phosphate and glycerol-3-phosphate. Glucose-1-phosphate,<br>
hexose-6-phosphate and glycerophosphates such as glucose-1-glycerophosphate, fructose-<br>
6-glycerophosphate, alpha-glycerophosphates such as glycerol-1-phosphate and glycerol-<br>
3-phosphate, and beta-gtycero phosphate (glycerol-2-phosphate) are preferred, with<br>
glycerophosphates more preferred, alpha- and/or beta-glycerophosphates still more<br>
preferred, and glycerol-2-phosphate and/or glycerol-3-phosphate still more preferred, and<br>
a mixture of glycerol-2- and glycerol-3-phosphate or glycerol-3-phosphate most<br>
particularly preferred herein for use. As used herein, the term "G3P" without being in a<br>
mixture or "G3P alone" refers to a composition containing at least about 80% glycerol-3-<br>
phosphate; it may contain up to about 20% impurities such as G2P. A mixture of G3P and<br>
G2P would contain less than about 80% G3P.<br>
An inorganic phosphate is a phosphate compound that does not contain any carbon<br>
atoms, with the phosphate typically being associated with an alkali or alkali earth metal<br>
such as potassium, calcium, magnesium, or sodium phosphate.<br>
"Active growth phase" refers to the phase of the culturing step wherein the cells are<br>
actively growing and not severely nutrient-limited cells such as those that are in stationary<br>
phase.<br>
Modes for Carrying Out the Invention<br>
The present invention provides a method for producing polypeptides heterologous<br>
to E. coli. In this method E. coli cells comprising nucleic acid encoding the polypeptide<br>
are cultured in a culture medium while feeding to the culture medium a transportable<br>
organophosphate, such that the nucleic acid is expressed. The polypeptide is then<br>
recovered from the cells. The recovery may be from the cytoplasm, periplasm, or culture<br>
medium of the cells. The culturing may take place in any suitable vessel, preferably a<br>
shake flask or fermentor, more preferably, in a fermentor.<br>
Culturing parameters are used and polypeptide production may be conducted in a<br>
conventional manner, such as those procedures described below.<br>
A. Selection of Nucleic Acid and Modifications Thereof<br>
The nucleic acid encoding the polypeptide of interest is suitably RNA, cDNA, or<br>
genomic DNA from any source, provided it encodes the polypeptide(s) of interest.<br>
Methods are well known for selecting the appropriate nucleic acid for expression of<br>
heterologous polypeptides (including variants thereof) in E. coli.<br>
If monoclonal antibodies are being produced, DNA encoding the monoclonal<br>
antibodies is readily isolated and sequenced using conventional procedures (e.g., by using<br>
oligonucleotide probes that are capable of binding specifically to genes encoding the<br>
heavy and light chains of murine antibodies). The hybridoma cells serve as a preferred<br>
source of such DNA. Once isolated, the DNA may be placed into expression vectors,<br>
which are then transformed into the bacterial host cells herein to obtain the synthesis of<br>
monoclonal antibodies in the recombinant host cells. Review articles on recombinant<br>
expression in bacteria of DNA encoding the antibody include Skerra et al., Curr. Opinion<br>
in Immunol. 5: 256-262 (1993) and Pliickthun, Immunol. Revs.. 130: 151-188 (1992).<br>
Methods for humanizing non-human antibodies have been described in the art.<br>
Preferably, a humanized antibody has one or more amino acid residues introduced into it<br>
from a source that is non-human. These non-human amino acid residues are often referred<br>
to as "import" residues, which are typically taken from an "import" variable domain.<br>
Humanization can be essentially performed following the method of Winter and coworkers<br>
(Jones etal, Nature, 321: 522-525 (1986); Riechmannef a/., Nature. 332: 323-<br>
327 (1988); Verhoeyenef a/., Science. 239: 1534-1536 (1988)), by substituting<br>
hypervariable region sequences for the corresponding sequences of a human antibody.<br>
Accordingly, such "humanized" antibodies are chimeric antibodies (U.S. Pat. No.<br>
4,816,567) wherein substantially less than an intact human variable domain has been<br>
substituted by the corresponding sequence from a non-human species. In practice,<br>
humanized antibodies are typically human antibodies in which some hypervariable region<br>
residues and possibly some FR residues are substituted by residues from analogous sites in<br>
rodent antibodies.<br>
The choice of human variable domains, both light and heavy, to be used in making<br>
the humanized antibodies is very important to reduce antigenicity. According to the socalled<br>
"best-fit" method, the sequence of the variable domain of a rodent antibody is<br>
screened against the entire library of known human variable-domain sequences. The<br>
human sequence that is closest to that of the rodent is then accepted as the human<br>
framework region (FR) for the humanized antibody (Sims et al., J. Immunol. 151: 2296<br>
(1993); Chothia et al., 3. Mol. Biol. 196: 901 (1987)). Another method uses a particular<br>
framework region derived from the consensus sequence of all human antibodies of a<br>
particular subgroup of light or heavy chains. The same framework may be used for<br>
several different humanized antibodies (Carter et al, Proc. Natl. Acad. Sci. USA. 89: 4285<br>
(1992); Prestaera/., J. Immunol. 151: 2623 (1993)).<br>
It is further important that antibodies be humanized with retention of high affinity<br>
for the antigen and other favorable biological properties. To achieve this goal, according<br>
to a preferred method, humanized antibodies are prepared by a process of analysis of the<br>
parental sequences and various conceptual humanized products using three-dimensional<br>
models of the parental and humanized sequences. Three-dimensional immunoglobulin<br>
models are commonly available and are familiar to those skilled in the art. Computer<br>
programs are available that illustrate and display probable three-dimensional<br>
conformational structures of selected candidate immunoglobulin sequences. Inspection of<br>
these displays permits analysis of the likely role of the residues in the functioning of the<br>
candidate immunoglobulin sequence, i.e., the analysis of residues that influence the ability<br>
of the candidate immunoglobulin to bind its antigen. In this way, FR residues can be<br>
selected and combined from the recipient and import sequences so that the desired<br>
antibody characteristic, such as increased affinity for the target antigen(s), is achieved. In<br>
general, the hypervariable region residues are directly and most substantially involved in<br>
influencing antigen binding.<br>
Various forms of the humanized antibody or affinity-matured antibody are<br>
contemplated. For example, the humanized antibody or affinity-matured antibody may be<br>
an antibody fragment, such as a Fab, that is optionally conjugated with one or more<br>
targeting agent(s) in order to generate an immunoconjugate. Alternatively, the humanized<br>
antibody or affinity-matured antibody may be an intact antibody, such as an intact IgGl<br>
antibody.<br>
Fab'-SH fragments can be directly recovered from E. coli and chemically coupled<br>
to form F(ab')2 fragments (Carter et a/., Bio/Technology. 10: 163-167 (1992)). According<br>
to another approach, F(ab')2 fragments can be isolated directly from recombinant host cell<br>
culture. Other techniques for the production of antibody fragments will be apparent to the<br>
skilled practitioner. In other embodiments, the antibody of choice is a single-chain Fv<br>
fragment (scFv) (WO 93/16185; U.S. Pat Nos. 5,571,894 and 5,587,458). The antibody<br>
fragment may also be a "linear antibody", e.g., as described in U.S. Pat. No. 5,641,870.<br>
Such linear antibody fragments may be monospecific or bispecific.<br>
Bispecific antibodies are antibodies that have binding specificities for at least two<br>
different epitopes. Exemplary bispecific antibodies may bind to two different epitopes of<br>
the same protein. Bispecific antibodies can be prepared as full-length antibodies or<br>
antibody fragments (e.g., F(ab')2 bispecific antibodies). These may be as fusions of<br>
various antibody chains or can be one chain. One heavy chain can be competent by itself.<br>
In one approach to producing bispecific antibodies, a bispecific immunoadhesin is<br>
prepared by introducing into a host cell DNA sequences encoding a first fusion comprising<br>
a first binding domain fused to an immunoglobulin heavy-chain constant domain sequence<br>
lacking a light-chain binding site; a second fusion comprising a second binding domain<br>
fused to an immunoglobulin heavy-chain constant domain sequence retaining a light-chain<br>
binding site; and an immunoglobulin light-chain, respectively. The host cells are then<br>
cultured so as to express the DNA sequences to produce a mixture of (i) a heterotrimer<br>
comprising the first fusion covalently linked with a second fusion-immunoglobulin lightchain<br>
pair; (ii) a heterotetramer comprising two covalently linked second fusionimmunoglobulin<br>
light-chain pairs; and (iii) a homodimer comprising two covalently linked<br>
molecules of the first fusion. The mixture of products is removed from the cell culture and<br>
the heterotrimer is isolated from the other products. This approach is disclosed in WO<br>
94/04690. For further details of generating bispecific antibodies see, for example, Suresh<br>
era/., Methods in Enzvmologv. 121: 210 (1986).<br>
According to another approach described in U.S. Pat. No. 5,731,168, the interface<br>
between a pair of antibody molecules can be engineered to maximize the percentage of<br>
heterodimers that are recovered from recombinant cell culture. The preferred interface<br>
comprises at least a part of the Cn3 domain of an antibody constant domain. In this<br>
method, one or more small amino acid side chains from the interface of the first antibody<br>
molecule are replaced with larger side chains (e.g., tyrosine or tryptophan). Compensatory<br>
"cavities" of identical or similar size to the large side chain(s) are created on the interface<br>
of the second antibody molecule by replacing large amino acid side chains with smaller<br>
ones (e.g., alanine or threonine). This provides a mechanism for increasing the yield of<br>
the heterodimer over other unwanted end-products such as homodimers.<br>
Bispecific antibodies include cross-linked or "heteroconjugate" antibodies. For<br>
example, one of the antibodies in the heteroconjugate can be coupled to avidin, the other<br>
to biotin. Such antibodies have, for example, been proposed to target immune system cells<br>
to unwanted cells (U.S. Pat. No. 4,676,980), and for treatment of fflV infection (WO<br>
91/00360, WO 92/200373, and EP 03089). Heteroconjugate antibodies may be made<br>
using any convenient cross-linking methods. Suitable cross-linking agents are well known<br>
in the art, and are disclosed, for example, in U.S. Pat. No. 4,676,980, along with a number<br>
of cross-linking techniques.<br>
Techniques for generating bispecific antibodies from antibody fragments have also<br>
been described in the literature. For example, bispecific antibodies can be prepared using<br>
chemical linkage. Brennan et al, Science, 229: 81 (1985) describe a procedure wherein<br>
intact antibodies are proteolytically cleaved to generate F(ab')2 fragments. These<br>
fragments are reduced in the presence of the dithiol complexing agent sodium arsenite to<br>
stabilize vicinal dithiols and prevent intermolecular disulfide formation. The Fab'<br>
fragments generated are then converted to thionitrobenzoate (TNB) derivatives. One of<br>
the Fab'-TNB derivatives is then reconverted to the Fab'-thiol by reduction with<br>
mercaptoethylamine and is mixed with an equimolar amount of the other Fab'-TNB<br>
derivative to form the bispecific antibody. The bispecific antibodies produced can be used<br>
as agents for the selective immobilization of enzymes.<br>
Additionally, Fab'-SH fragments can be directly recovered from E. coli and<br>
chemically coupled to form bispecific antibodies (Shalaby et al, J. Exp. Med.. 175: 217-<br>
225 (1992)).<br>
Various techniques for making and isolating bispecific antibody fragments directly<br>
from recombinant cell culture have also been described. For example, bispecific<br>
antibodies have been produced using leucine zippers (Kostelny et al, J. Immunol, 148.<br>
1547-1553 (1992)). The leucine zipper peptides from the Fos and Jun proteins are linked<br>
to the Fab' portions of two different antibodies by gene fusion. The antibody homodimers<br>
are reduced at the hinge region to form monomers and then re-oxidized to form the<br>
antibody heterodimers. This method can also be utilized for the production of antibody<br>
homodimers. The "diabody" technology described by Hollinger et al, Proc. Natl. Acad.<br>
Sci. USA. 90: 6444-6448 (1993) has provided an alternative mechanism for making<br>
bispecific antibody fragments. The fragments comprise a heavy-chain variable domain<br>
(VH) connected to a light-chain variable domain (VL) by a linker that is too short to allow<br>
pairing between the two domains on the same chain. Accordingly, the VH and VL domains<br>
of one fragment are forced to pair with the complementary VL and VH domains of another<br>
fragment, thereby forming two antigen-binding sites. Another strategy for making<br>
bispecific antibody fragments by the use of single-chain Fv (sFv) dimers has also been<br>
reported (Gruber et al, J. Immunol. 152: 5368 (1994)).<br>
Antibodies with more than two valencies are contemplated. For example,<br>
trispecific antibodies can be prepared (Tutt et al., J. Immunol.. 147: 60 (1991)).<br>
Nucleic acid molecules encoding polypeptide variants are prepared by a variety of<br>
methods known in the art. These methods include, but are not limited to, isolation from a<br>
natural source (in the case of naturally occurring amino acid sequence variants) or<br>
preparation by oligonucleotide-mediated (or site-directed) mutagenesis, PCR mutagenesis,<br>
or cassette mutagenesis of an earlier prepared variant or a non-variant version of the<br>
polypeptide.<br>
It may be desirable to modify the antibody of the invention with respect to effector<br>
function, e.g., so as to enhance Fc receptor binding. This may be achieved by introducing<br>
one or more amino acid substitutions into an Fc region of the antibody. Alternatively or<br>
additionally, cysteine residues) may be introduced in the Fc region, thereby allowing<br>
interchain disulfide bond formation in this region.<br>
To increase the serum half-life of the antibody, one may incorporate a salvage<br>
receptor binding epitope into the antibody (especially an antibody fragment) as described<br>
in U.S. Pat. 5,739,277, for example. As used herein, the term "salvage receptor binding<br>
epitope" refers to an epitope of the Fc region of an IgG molecule (e.g., IgGi, IgG2, IgGs,<br>
or IgG4) that is responsible for increasing the in vivo serum half-life of the IgG molecule.<br>
Other modifications of the antibody are contemplated herein. For example, the<br>
antibody may be linked to one of a variety of non-proteinaceous polymers, e.g.,<br>
polyethylene grycol, polypropylene glycol, polyoxyalkylenes, or copolymers of<br>
polyethylene glycol and polypropylene glycol.<br>
B. Insertion of Nucleic Acid Into a Replicable Vector<br>
The heterologous nucleic acid (e.g., cDNA or genomic DNA) is suitably inserted<br>
into a replicable vector for expression in the E. coli under the control of a suitable<br>
promoter. Many vectors are available for this purpose, and selection of the appropriate<br>
vector will depend mainly on the size of the nucleic acid to be inserted into the vector and<br>
the particular host cell to be transformed with the vector. Each vector contains various<br>
components depending on the particular host cell with which it is compatible. Depending<br>
on the particular type of host, the vector components generally include, but are not limited<br>
to, one or more of the following: a signal sequence, an origin of replication, one or more<br>
marker genes, a promoter, and a transcription termination sequence.<br>
In general, plasmid vectors containing replicon and control sequences that are<br>
derived from species compatible with the host cell are used in connection with E. coli<br>
hosts. The vector ordinarily carries a replication site, as well as marking sequences mat<br>
are capable of providing phenotypic selection in transformed cells. For example, E. coli is<br>
typically transformed using pBR322, a plasmid derived from an E. coli species (see, e.g.,<br>
Bolivar etal., Gene, 2: 95 (1977)). pBR322 contains genes for ampicillin and tetracycline<br>
resistance and thus provides easy means for identifying transformed cells. The pBR322<br>
plasmid, or other bacterial plasmid or phage, also generally contains, or is modified to<br>
contain, promoters that can be used by the E. coli host for expression of the selectable<br>
marker genes.<br>
(i) Signal Sequence Component<br>
The DNA encoding the polypeptide of interest herein may be expressed not only<br>
directly, but also as a fusion with another polypeptide, preferably a signal sequence or<br>
other polypeptide having a specific cleavage site at the N-terminus of the mature<br>
polypeptide. In general, the signal sequence may be a component of the vector, or it may<br>
be a part of the polypeptide-encoding DNA that is inserted into the vector. The<br>
heterologous signal sequence selected should be one that is recognized and processed (i.e.,<br>
cleaved by a signal peptidase) by the host cell.<br>
For prokaiyotic host cells that do not recognize and process the native or a<br>
eukaryotic polyp eptide signal sequence, the signal sequence is substituted by a prokaryotic<br>
signal sequence, selected, for example, from the group consisting of the alkaline<br>
phosphatase, penicillinase, Ipp, or heat-stable enterotoxin II leaders.<br>
(ii) Origin of Replication Component<br>
Expression vectors contain a nucleic acid sequence that enables the vector to<br>
replicate in one or more selected host cells. Such sequences are well known for a variety<br>
of bacteria. The origin of replication from the plasmid pBR322 is suitable for most Gramnegative<br>
bacteria such as E. coli.<br>
(iii) Selection Gene Component<br>
Expression vectors generally contain a selection gene, also termed a selectable<br>
marker. This gene encodes a protein necessary for the survival or growth of transformed<br>
host cells grown in a selective culture medium. Host cells not transformed with the vector<br>
containing the selection gene will not survive in the culture medium. This selectable<br>
marker is separate from the genetic markers as utilized and defined by this invention.<br>
Typical selection genes encode proteins that (a) confer resistance to antibiotics or other<br>
toxins, e.g., ampicillin, neomycin, methotrexate, or tetracycline, (b) complement<br>
auxotrophic deficiencies other than those caused by the presence of the genetic marker(s),<br>
or (c) supply critical nutrients not available from complex media, e.g., the gene encoding<br>
D-alanine racemase for Bacilli.<br>
One example of a selection scheme utilizes a drug to arrest growth of a host cell.<br>
In this case, those cells that are successfully transformed with the nucleic acid of interest<br>
produce a polypeptide conferring drug resistance and thus survive the selection regimen.<br>
Examples of such dominant selection use the drugs neomycin (Southern et a/., J. Molec.<br>
Appl. Genet. I: 327 (1982)), mycophenolic acid (Mulligan et a/., Science. 209: 1422<br>
(1980)), or hygromycin (Sugden et a/., Mol. Cell. Biol.. 5: 410-413 (1985)). The three<br>
examples given above employ bacterial genes under eukaryotic control to convey<br>
resistance to the appropriate drug G418 or neomycin (geneticin), xgpt (mycophenolic<br>
acid), or hygromycin, respectively.<br>
(iv) Promoter Component<br>
The expression vector for producing the polypeptide of interest contains a suitable<br>
promoter that is recognized by E. coli and is operably linked to the nucleic acid encoding<br>
the polypeptide of interest. Promoters suitable for use with E. coli hosts include the beta-<br>
lactamase and lactose promoter systems (Chang etal. Nature, 275: 615 (1978); Goeddel<br>
et al., Nature, 281: 544 (1979)), the arabinose promoter system (Guzman et al, I<br>
Bacteriol.. 174: 7716-7728 (1992)), alkaline phosphatase, the T7 promoter, atryptophan<br>
(trp) promoter system (Goeddel, Nucleic Acids Res.. 8: 4057 (1980) and EP 36,776) and<br>
hybrid promoters such as the tac promoter (deBoer et al, Proc. Natl. Acad. Sci. USA, 80:<br>
21-25 (1983)). However, other known bacterial promoters are suitable. Their nucleotide<br>
sequences have been published, thereby enabling a skilled worker operably to ligate them<br>
to DNA encoding the polypeptide of interest (Siebenlist et al., Cell. 20: 269 (1980)) using<br>
linkers or adaptors to supply any required restriction sites.<br>
Preferably, the promoter employed herein is an inducible promoter, i.e., one that is<br>
activated by an inducing agent or condition (such as periplasmic phosphate depletion).<br>
Preferred such inducible promoters herein are the alkaline phosphatase promoter, the tac<br>
promoter, or the T7 promoter.<br>
Promoters for use in bacterial systems also generally contain a Shine-Dalgarno<br>
(S.D.) sequence operably linked to the DNA encoding the polypeptide of interest. The<br>
promoter can be removed from the bacterial source DNA by restriction enzyme digestion<br>
and inserted into the vector containing the desired DNA.<br>
(v) Construction and Analysis of Vectors<br>
Construction of suitable vectors containing one or more of the above-listed<br>
components employs standard ligation techniques. Isolated plasmids or DNA fragments<br>
are cleaved, tailored, and re-ligated in the form desired to generate the plasmids required.<br>
For analysis to confirm correct sequences in plasmids constructed, the ligation<br>
mixtures are used to transform E. coli K.12 strain 294 (ATCC 31,446) or other strains, and<br>
successful transformants are selected by ampicillin or tetracycline resistance where<br>
appropriate. Plasmids from the transformants are prepared, analyzed by restriction<br>
endonuclease digestion, and/or sequenced by the method of Sanger et al., Proc. Natl.<br>
Acad. Sci. USA. 7j4: 5463-5467 (1977) or Messing et al. Nucleic Acids Res.. 9: 309<br>
(1981), or by the method of Maxam et al, Methods in Enzvmology. 65: 499 (1980).<br>
C. Selection and Transformation of Host Cells<br>
E. coli hosts suitable as parental hosts for expression plasmids herein include E.<br>
coli W3110 (ATCC 27,325), E. coli 294 (ATCC 31,446), E. coli B, and E. coli X1776<br>
(ATCC 31,537). These examples are illustrative rather than limiting. Mutant cells of any<br>
of the above-mentioned strains may also be employed as the starting hosts that are then<br>
further mutated to contain at least the minimum genotype required herein. E. coli<br>
strain W3110 is a preferred parental host because it is a common host strain for<br>
recombinant DNA product fermentations. Examples of starting E. coli hosts to be used as<br>
parent hosts, along with their genotypes, are included in the table below:<br>
Strain<br>
W3110<br>
1A2<br>
9E4<br>
27A7<br>
27C6<br>
27C7<br>
33D3<br>
36F8<br>
43D3<br>
43E7<br>
43F6<br>
44D6<br>
45F8<br>
45F9<br>
61G1<br>
Genotype<br>
K-12 F lambda lN(trnD-rmE) 1<br>
AjhuA (btonA)<br>
kfhuA (&amp;tonA)ptr3<br>
A/huA (AtonA)ptr3phoA&amp;E15 (argF-lac)169<br>
&amp;JhuA (ktonA) phoA&amp;E15 (argF-lac)169 ptr3 ompT A(nmpc-fepE)<br>
A/huA (A.tonA)phoAAE15 (argF-lac)169ptr3 degP41::kanR ompT k(nmpc-fepE)<br>
&amp;fhuA (htonA) ptr3 laclq lacLS ompT &amp;(nmpc-fepE) degP::kanR<br>
kfhuA (&amp;tonA)phoAt\E15 (argF-lac) 1 69 ptr 3 degP41::kanR ilvG+<br>
AJhuA (MonA) phoAAElS (argF-lac)169 ptr3 degP4 1 : :kanR ompT &amp;(nmpc-fepE)<br>
ilvG+<br>
hjhuA (MonA)phoAAE15 (argF-lac) 169 ptr3 degP41 ompT &amp;(mnpc-fepE) ilvG+<br>
A/huA (A.tonA) phaA&amp;ElS (argF-lac) 169 ptr3 degP41;:kanR ompT ^nmpc-fepE)<br>
A(rbs7) ilvG+ kglpT596<br>
AfliuA (MonA) (argF-lac) 169 ptr3 degP41::kanR ompT &amp;(nmpc-fepE) ilvG+<br>
AfltuA (MonA) (argF-lac) 1 69 ptr3 degP41 ompT k(nmpc-fepE) ilvG+<br>
phoS(T10Y)<br>
AfhuA (MonA) (argF-lac) 1 69 ptr3 degP41 ompT &amp;(nmpc-fepE) i/vG+<br>
phoS(T10Y) cyo::kanR<br>
bfliuA Aptr AompTAdegP \phoA ilvG+ kglpTQ<br>
Also suitable are the intermediates in making strain 36F8, i.e., 27B4 (U.S. Pat. No.<br>
5,304,472) and 35E7 (a spontaneous temperature-resistant colony isolate growing better<br>
than 27B4). An additional suitable strain is the E. coli strain having the mutant<br>
periplasmic protease(s) disclosed in U.S. Pat. No. 4,946,783 issued August 7,1990.<br>
In one embodiment, the E. coli host cell employed is wild type with respect to or in<br>
reference to the g/pJgene, such as 43E7, or is deficient in the g/p^gene, such as 43F6 or<br>
61G1. In another embodiment, the E. coli host cell employed is wild type with respect to<br>
or in reference to ihephoA gene. In a preferred embodiment, the E. coli is deficient in<br>
chromosomal phoA. In another preferred embodiment, the E. coli is deficient in<br>
chromosomal glpT and in chromosomal phoA. In a more preferred embodiment, the E.<br>
coli is deficient in chromosomal glpT and in chromosomal phoA, but not in chromosomal<br>
ugp. The most preferred such mutant E. coli host is 43F6 or 61 Gl, the genotypes of which<br>
are given in the above table. As used herein, "wild type with respect to glpT' refers to E.<br>
coli hosts that areglpT+ or glpl competent cells, i.e., those that are not deficient in<br>
chromosomal glpT. Similarly, as used herein, "wild type with respect lophoA" refers to<br>
E. coli hosts that aiephoA+ orphoA competent cells, i.e., those that are not deficient in<br>
chromosomal phoA.<br>
The strains of this invention may be produced by chromosomal integration of the<br>
parental strain or other techniques, including those set forth in the Examples below.<br>
The nucleic acid encoding the polypeptide is inserted into the host cells.<br>
Preferably, this is accomplished by transforming the host cells with the above-described<br>
expression vectors and culturing in conventional nutrient media modified as appropriate<br>
for inducing the various promoters.<br>
Transformation means introducing DNA into an organism so that the DNA is<br>
replicable, either as an extrachromosomal element or by chromosomal integrant.<br>
Depending on the host cell used, transformation is done using standard techniques<br>
appropriate to such cells. The calcium treatment employing calcium chloride, as described<br>
in section 1.82 of Sambrook et al, Molecular Cloning: A Laboratory Manual (New York:<br>
Cold Spring Harbor Laboratory Press, 1989), is generally used for prokaryotic cells or<br>
other cells that contain substantial cell-wall barriers. Another method for transformation<br>
employs polyethylene glycol/DMSO, as described in Chung and Miller, Nucleic Acids<br>
Res.. 16: 3580 (1988). Yet another method is the use of the technique termed<br>
electroporation.<br>
D. Culturing the Host Cells<br>
E. coli cells used to produce the polypeptide of interest are cultured in suitable<br>
media as described generally in Sambrook et al, supra. The culture conditions, such as<br>
temperature, pH, and the like, are those previously used with the host cell selected for<br>
expression, and will be apparent to the ordinarily skilled artisan.<br>
The cells are cultured while the culture medium is fed with a transportable<br>
organophosphate such as a glycerophosphate, e.g., alpha-glycerophosphate and/or betaglycerophosphate,<br>
and especially glycerol-2-phosphate and/or glycerol-3-phosphate. The<br>
culturing may take place in a shake flask or a fermentor, preferably a fermentor. The<br>
polypeptide is preferably recovered from the cytoplasm, periplasm, or culture medium of<br>
the cells.<br>
In the process of this invention, expression of the nucleic acid can begin at any<br>
phase of the culturing step. However, preferably expression of the nucleic acid begins<br>
while cell density is still increasing. This can be accomplished by the inducement of the<br>
promoter with the appropriate inducer or inducing condition before cell growth ceases.<br>
The feed rate of the organophosphate into the culture medium to be employed for<br>
maximum production of the polypeptide depends on many factors, including the type of<br>
organophosphate, the concentration of organophosphate, the type of polypeptide being<br>
produced, the type of promoter, the host cell strain employed, and the cell density in the<br>
broth. If the polypeptide is IGF-I and the organophosphate is glycerol-3-phosphate<br>
intended to extend the production duration, under the culture conditions described and<br>
using a 10-L process, the feed rate of the organophosphate is preferably from about 1 to 7<br>
mmoles/hour per about 8-10 liters (see Fig. 4), more preferably from about 1 to 6<br>
mmoles/hour, and still more preferably from about 2 to 6 mmoles/hour, yet still more<br>
preferably from about 2 to 5 mmoles/hour, and most preferably from about 3 to 4<br>
mmoles/hour. The optimal feed rate is dependent on the process, the cell density, the<br>
respiration rate, etc.<br>
Also, in a preferred embodiment, where the polypeptide is Apo2L and the<br>
organophosphate is glycerol-3-phosphate intending to shift product expression to concur<br>
with the active growth phase and using a 10-L process, the feed rate of the<br>
organophosphate is from about 4 to 17 mmoles/hour per about 8-10 liters (see Fig. 7),<br>
more preferably from about 6 to 16 mmoles/hour, still more preferably from about 8 to 15<br>
mmoles/hour, and most preferably from about 10 to 14 mmoles/hour. The optimal feed<br>
rate of the organophosphate needs to be determined for the individual process employed<br>
for the expression of the specific heterologous protein.<br>
Any other necessary media ingredients besides carbon, nitrogen, and inorganic<br>
phosphate sources may also be included at appropriate concentrations introduced alone or<br>
as a mixture with another ingredient or medium such as a complex nitrogen source.<br>
Preferably, an inorganic phosphate is also present in the culture medium at the start of the<br>
culturing step. If such inorganic phosphate, preferably sodium and/or potassium<br>
phosphate, is present, the ratio of inorganic phosphate to organophosphate depends on<br>
such factors as the type of polypeptide expressed and organophosphate employed. This<br>
ratio can be any proportion, as determined readily by those skilled in the art, ranging<br>
typically from about 1:10 (one part of Pi to 10 parts of organophosphate) to 1:0.25. For<br>
Apo2 ligand, preferably it ranges from about 1:4 to 1:0.25, and more preferably about 1:3<br>
to 1:0.5, and yet more preferably about 1:3 to 1:1, and still more preferably about 1:2 to<br>
1:1, and most preferably about 1:1. Such ratios allow earlier induction of protein<br>
expression, and in some cases allow more product to be produced earlier. The pH of the<br>
medium may be any pH from about 5-9, depending mainly on the host organism.<br>
If the promoter is an inducible promoter, for induction to occur, typically the cells<br>
are cultured until a certain optical density is achieved, e.g., a Asso of about 200 for a highcell-<br>
density process, at which point induction is initiated (e.g., by addition of an inducer,<br>
by depletion of a medium component, etc.), to induce expression of the gene encoding the<br>
polypeptide of interest.<br>
Where the alkaline phosphatase promoter is employed, E. coli cells used to<br>
produce the polypeptide of interest of this invention are cultured in suitable media in<br>
which the alkaline phosphatase promoter can be induced as described generally, e.g., in<br>
Sambrook et at., supra. At first, the medium may contain inorganic phosphate for the<br>
growth of the bacterium in an amount sufficiently large to support significant cell growth<br>
and avoid induction of synthesis of target heterologous polypeptide under the promoter<br>
control. As the cells grow and utilize phosphate, they decrease the level of inorganic<br>
phosphate in the medium, thereby causing induction of synthesis of the polypeptide when<br>
the inorganic phosphate is exhausted. By adding, for example, a feed constituting a<br>
mixture of G2P and G3P or a G3P feed, further growth to a higher cell density, such as up<br>
to 200 OD550 or higher, takes place in the absence of inorganic phosphate or at starvation<br>
levels of inorganic phosphate in the periplasm and supporting culture medium, resulting in<br>
an increase or an extension of product accumulation.<br>
E. Detecting Expression<br>
Gene expression may be measured in a sample directly, for example, by<br>
conventional northern blotting to quantitate the transcription of mRNA (Thomas, Proc.<br>
Nad. Acad. Sci. USA. 77: 5201-5205 (1980)), dot blotting (RNA analysis), or in situ<br>
hybridization, using an appropriately labeled probe, based on the sequences that encode<br>
the polypeptide. Various labels may be employed, most commonly radioisotopes,<br>
particularly 32P. However, other techniques may also be employed, such as using biotinmodified<br>
nucleotides for introduction into a polynucleotide. The biotin then serves as the<br>
site for binding to avidin or antibodies, which may be labeled with a wide variety of<br>
labels, such as radionuclides, fluorescers, enzymes, or the like. Alternatively, assays or<br>
gels may be employed for detection of protein.<br>
For secretion of an expressed gene product, the host cell is cultured under<br>
conditions sufficient for secretion of the gene product. Such conditions include, e.g.,<br>
temperature, nutrient, and cell density conditions that permit secretion by the cell.<br>
Moreover, such conditions are those under which the cell can perform the basic cellular<br>
functions of transcription, translation, and passage of proteins from one cellular<br>
compartment to another, as are known to those skilled in the art.<br>
F. Purification of Polypeptides<br>
The following procedures, individually or in combination, are exemplary of<br>
suitable purification procedures, with the specific method(s) used being dependent on the<br>
type of polypeptide: fractionation on immunoaffinity or ion-exchange columns; ethanol<br>
precipitation; reversed-phase HPLC; hydrophobic-interaction chromatography;<br>
chromatography on silica; chromatography on an ion-exchange resin such as SSEPHAROSE<br>
โข and DEAE; chromatofocusing; SDS-PAGE; ammonium-sulfate<br>
precipitation; and gel filtration using, for example, SEPHADEXโข G-75 medium. The<br>
monoclonal antibodies may be suitably separated from the culture medium by<br>
conventional antibody purification procedures such as, for example, protein ASEPHAROSE<br>
โข medium, hydroxyapatite chromatography, gel electrophoresis, dialysis,<br>
or affinity chromatography.<br>
The invention will be more fully understood by reference to the following<br>
examples. They should not, however, be construed as limiting the scope of the invention.<br>
All literature and patent citations herein are incorporated by reference.<br>
EXAMPLE 1<br>
Feeding of G3P to Shake Flask Culture for the Production of Llama<br>
Antibody Fragment (Heavy Chain) and Apo2L<br>
Background:<br>
The inclusion of 200 mM G3P (final concentration) in either low-phosphate<br>
(CRAP) or high-phosphate culture medium (THCD) was compared to the respective<br>
control addition (water) for the expression of a heterologous protein in shake-flask<br>
culture. In the first part of this Example, the target heterologous protein is a 13kD llama<br>
anti-HCG camelid monobody. Camelid antibodies have been previously shown to have 2<br>
species, a classic IgG molecule consisting of two heavy plus two light chains and a<br>
heavy-chain IgG molecule lacking a light chain referred to as monobody. The camelid<br>
monobody was expressed by BL21, an E. coli B strain, using a tac promoter in either a<br>
low-phosphate (CRAP)- or a high-phosphate (THCD) -rich media. The malE binding<br>
protein signal sequence preceding the antibody-fragment-encoding sequence directed the<br>
secretion of the expression protein into the periplasm of the host. In the second part of<br>
this Example, a T7 promoter was used to regulate the expression of Apo2 ligand in<br>
HMS174, an E. coli K12 strain, in G3P-supplemented and unsupplemented CRAP<br>
medium. Production of heterologous protein in both experiments was induced with the<br>
addition of IPTG upon reaching the desired cell density.<br>
Materials &amp; Methods:<br>
pCB36624 86.RIG Plasmid Construction:<br>
pCB36624_86.RIG.was constructed by modifying vector pL1602 (Sidhu et al, I<br>
Mol. Biol. 296:487-495 (2000)). Vector pS1602, which has pTac promoter sequence and<br>
malE secretion signal sequence, contained a sequence of human growth hormone fused to<br>
the C-terminal domain of the gene-3 minor coat protein (p3) of phage mu. The sequence<br>
encoding hGH was removed and the resulting vector sequence served as the vector<br>
backbone for the insertion of a synthetic DNA fragment encoding the llama anti-HCG<br>
antibody (Spinelli et al, Nat. Struct. Biol. 3(9): 752-757 (1996)). The resulting phagemid<br>
(pCB36624) encoded the fusion product under the control of the IPTG-inducible Ptac<br>
promoter (Amman and Brosius, Gene. 40:183-190 (1985)). The expressed polypeptide<br>
included the maltose-binding protein signal peptide, followed by the anti-HCG coding<br>
region, followed by a FLAG epitope tag, followed by a Gly/Ser-rich linker peptide<br>
containing a suppressible stop codon, followed by P3C (the C-terminal domain of the<br>
phage coat protein).<br>
Phage-displayed libraries were constructed using the method of Sidhu et al, JL<br>
Mol. Biol, 296: 487-495 (2000) with appropriately designed "stop template" phagemids.<br>
For library NNS17, a derivative of pCB36624 that contained TAA stop codons in place of<br>
codons 93, 94,100 and 101 was used as the template for the Kunkel mutagenesis method<br>
(Kunkel et al, Methods Enzymol. 154: 367-382 (1987)), with mutagenic oligonucleotide<br>
NNS17 designed to simultaneously repair the stop codons and introduce 17 NNK<br>
degenerate codons between the codons encoding Gly95 and Trpl03.<br>
Like all monobodies, the llama anti-HCG is a Vh3 family member and as such is<br>
recognized by Protein A. The Protein A binding interaction was used as a surrogate for<br>
CDR3-mediated stability. The resulting phage libraries were sorted by multiple rounds<br>
against Protein A as readout of scaffold stability and expression. The sorted libraries were<br>
analyzed for selection bias in the distribution of amino acids in the NNS library. Scaffold<br>
RIG, as named by the sequence at positions 96, 97 and 98, turned out to be the most<br>
dominant clone based on the sequenced residues. The 17-amino-acid-long CDR3 sequence<br>
for Scaffold RIG was determined to be RIGRSVFNLRRESWVTW (SEQ ID NO:4). The<br>
phagemid with Scaffold RIG is renamed pCB36624_86.RIG, with the DNA sequence:<br>
petl9b.nohis Plasmid Construction<br>
Using standard molecular biology techniques, Apo2L codons 114-281 were<br>
amplified by polymerase chain reaction from a full-length Apo2L clone isolated from<br>
human placental cDNA. Additional nucleotides containing restriction sites to facilitate<br>
cloning are added to the 5' and 3' sequences, respectively. The 5' oligonucleotide primer<br>
has the sequence:<br>
containing the underlined Nde I restriction site. The 3' oligonucleotide primer has the<br>
sequence:<br>
containing the underlined BamH I restriction site. The resulting fragment was subcloned<br>
using the restriction sites Nde I to BamH I into a modified baculovirus expression vector<br>
pVL1392 (Pharmingen) in frame and downstream of a sequence containing a His 10 tag<br>
and an enterokinase cleavage site (Pitti era/., J. Biol. Chem.. 271:12687-12690 (1997)).<br>
pVL1392-Apo2L was digested with Nde I and BamH I and theMfe I-to-BamHI fragment<br>
generated was subcloned into pET-19b (Novagen), also digested with Nde I and BamH I.<br>
The resultant plasmid was named pet!9b.nohis.<br>
Bacterial Strains:<br>
Competent cells of BL21 (Stratagene) and HMS174 (Merck) were transformed<br>
with pCB36624_86.RIG and pet!9b.nohis, respectively, using standard procedures.<br>
Transformants were picked after growth on an LB plate containing 50ug/mL carbenicillin<br>
(LB+CARB50โข carbenicillin), streak-purified, and grown in LB broth with 50 jag/mL<br>
CARB50โข carbenicillin in a 30ยฐC incubator. pCB36624_86.RIG conferred carbenicillin<br>
resistance to the production host BL21/ pCB36624_86.RIG and petl 9b.nohis to<br>
HMS174/petl9b.nohis, allowing the transformed hosts to grow in the presence of the<br>
antibiotic.<br>
Fermentation Medium:<br>
Both low-phosphate (CRAP) culture medium and high-phosphate (THCD) culture<br>
medium were used for the evaluation of production of llama antibody fragment and Apo2<br>
ligand. The media composition (with the quantities of each component utilized per liter<br>
of initial medium) is listed below:<br>
Ingredient Low-PQ4 Medium High-PCXi Medium<br>
Glucose<br>
Ammonium Sulfate<br>
Na2HPO4<br>
NaH2P04-H20<br>
Sodium Citrate, Dihydrate<br>
Potassium Chloride<br>
1 M Magnesium Sulfate<br>
Hycase SF<br>
Yeast Extract<br>
Ouantitv/L<br>
5.5 g<br>
3.57 g<br>
-<br>
-<br>
0.71 g<br>
1.07g<br>
7ml<br>
5.36 g<br>
5.36 g<br>
Ouantitv/L<br>
5-5 g<br>
3.57 g<br>
1.86g<br>
0.93 g<br>
0.71 g<br>
1.07g<br>
7ml<br>
-<br>
5.36 g<br>
Casamino Acids - 5.36g<br>
1M MOPS, 0i 7.3 110ml 110ml<br>
KOH for pH adjustment to pH 7.3 as needed as needed<br>
To prepare 200 mM of G3P-supplemented medium, 5 ml of 1 M DL-alphaglycerophosphate<br>
(G3P) (Sigma Chem. Co.) was added to 20 ml of low-PC^ medium<br>
with 50 ug/ml of carbenicillin (low-PO4 medium+CARBSOโข carbenicillin) or high-PO4<br>
medium with 50 ug/ml of carbencillin (high-PO4 medium+CARBSOโข carbenicillin)<br>
prior to inoculation. For the unsupplemented (control) medium, 5 ml of water was used<br>
in place of G3P.<br>
Shake-Flask Fermentation:<br>
Shake-flask fermentation was conducted in a 125-ml baffled flask containing 25<br>
ml of control or G3P-supplemented medium. An overnight culture of<br>
BL21/pCB36624_86.RIG or HMS 174/petl9b.nohis grown in LB+CARB50โข<br>
carbenicillin was back-diluted at approx. 1:100 for inoculation into the control or G3 Psupplemented<br>
media. Cultures were incubated at 30ยฐC on a shaker at 250 RPM and<br>
product expression was induced by the addition of 1 mM of IPTG when cell density<br>
reached approximately 50-60% of the potential cell growth supported by the medium.<br>
Cell pellets from 1 ml of broth culture, taken just before the addition of the inducer and at<br>
approximately 24 hrs post-inoculation, were prepared and stored at -20ยฐC.<br>
Llama Antibody Fragment Accumulation Analyzed by PAGE and Densitometry:<br>
Frozen (-20ยฐC) cell pellet prepared from 1 ml of culture sample was thawed and<br>
resuspended in sufficient quantity of lOmM TRIS, pH 7.6 + 1 mM EDTA, pH 8.0 (TE) to<br>
bring the cell suspension to 1 OD/25 ul concentration. 25 ul of the TE-cell suspension<br>
was mixed with 25 ul of 2X sample buffer containing beta-mercaptoethanol. The mixture<br>
was heated at 95ยฐC for 5 mins before 10 ul (equivalent to 0.2 OD) was loaded per well<br>
onto NU-PAGEโข precasted 10% Bis-Tris gel (Novex). Electrophoresis was performed in<br>
MES buffer (2-(N-morpholino) ethanesulphonic acid in deionised water adjusted to the<br>
appropriate pH, such as with 1 N NaOH). The resolved gel was stained with<br>
COOMASSIE BLUE R250โข stain and then destained. The band intensity of the 13-kD<br>
antibody fragment was determined using Kodak DIGITAL SCIENCE IDโข imaging<br>
software after scanning the wet gel with the Kodak imaging system.<br>
Apo2 Ligand Accumulation Analyzed by Reversed-Phase HPLC:<br>
Frozen (-20ยฐC) cell pellet prepared from 1 ml of culture sample was resuspended<br>
in sufficient quantity of TE buffer to bring the cell suspension to 1 OD/25 ul<br>
concentration. 20 ul of the cell suspension was mixed into 480 ul of 6 M guanidine HC1,<br>
pH 9.0 + lOOmM dithiothreitol (DTT), and was allowed to incubate at room temperature<br>
for an hour before being centrifuged at 13,000 rpm for 15 mins to recover the<br>
supernatant/extract. The extract was filtered through a MILLIPOREโข spin-filter before<br>
20 ul was loaded onto an HPLC column (PerSeptive Biosystems POROSยฎ Rl/10<br>
medium) for reverse-phase chromatography. The HPLC separation was conducted at<br>
80ยฐC with the mobile phases flowing at 1.0 ml/min and employed a gradient of 28% to<br>
35% of acetonitrile with 0.1% TFA over 20 minutes for the resolution of the Apo2L away<br>
from the contaminating proteins. Peak detection was at 280 nm wavelength. The amount<br>
of monomer present in samples was calculated using an average response factor<br>
(mAU/ug) derived from the area under the peak associated with 5-20 ug of purified<br>
standards analyzed by the same method.<br>
Results:<br>
Figure 1 shows that the antibody is expressed to higher levels in both high-PO4<br>
(THCD) and low-PO4 (CRAP) medium supplemented with 200 mM G3P versus the<br>
control.<br>
Figure 2 shows that the Apo2L protein is expressed to higher levels in low-PCu<br>
(CRAP) medium supplemented with 200 mM G3P versus the control.<br>
EXAMPLE!<br>
Feeding of G3P to 10-L Fermentor Culture of Wild-type or (AglpTphoA- ugp+) Host for<br>
Production of 1GF-I Regulated by Alkaline Phosphatase Promoter<br>
Materials &amp; Methods:<br>
pBKIGF-2B Plasmid for Expression of IGF-I:<br>
The plasmid pBKIGF-2, used for the expression of IGF-I herein, was constructed<br>
as detailed in U.S. Pat. No. 5,342,763. This plasmid was constructed from a basic<br>
backbone of pBR322. The transcriptional and translational sequences required for<br>
expression of the IGF-I gene in E. coli are provided by the alkaline phosphatase promoter<br>
and the trp Shine-Dalgamo sequences. The lambda transcriptional terminator is situated<br>
adjacent to the IGF-I termination codon. Secretion of the protein from the cytoplasm is<br>
directed by the lamB signal sequence. The majority of rhIGF-I is found in the cell<br>
periplasmic space. Plasmid pBKIGF-2B confers tetracycline resistance upon the<br>
transformed host.<br>
Bacterial Strains and Growth Conditions:<br>
The hosts used in the IGF-I fermentation are derivatives of E. coli W3110<br>
(Bachmann, Cellular and Molecular Biology, vol. 2 (Washington, D.C.: American<br>
Society for Microbiology, 1987), pp. 1190-1219). Experiments concerning a host with<br>
wild-typeglpTwere carried out with strain 43E7 (E. coli V*/3llQjhuA(tonA) &amp;(argF-lac)<br>
ptr3 degP41 &amp;ompT&amp;(nmpc-fepE) ilvG+ phoA), and experiments concerning a host with<br>
a Ag/pJ mutation were carried out with strain 43F6 (E. coli W31 IQJhuAftonA) A(argFlac)<br>
ptr3 degP41 i\ompT&amp;(nmpc-fepE) ilvG+ phoA AglpT). Competent cells of 43E7 or<br>
43F6 were transformed with pBKIGF-2B using standard procedures. Transformants<br>
were picked after growth on an LB plate containing 20ng/mL tetracycline (LB +<br>
TET20โข tetracycline), streak-purified, and grown in LB broth with 20 ^ig/mL TET20โข<br>
tetracycline in a 37ยฐC shaker/incubator before being tested in the fermentor. pBKIGF-2B<br>
confers tetracycline resistance to the production host and allows the transformed host to<br>
grow in the presence of the antibiotic.<br>
10-L Fermentation Process:<br>
The fermentation medium composition and run protocol used for the expression<br>
of IGF-I were somewhat similar to those used in the IGF-I process described in U.S. Pat.<br>
No. 5,342,763. Briefly, a shake-flask seed culture of 43E7/pBKIGF-2 or 43F6/pBKIGF-<br>
2 was used to inoculate the rich production medium. The composition of the medium<br>
(with the quantities of each component utilized per liter of initial medium) is described<br>
below:<br>
Ingredient Quantity/L<br>
Glucose200-500 g<br>
Ammonium Sulfate 2-10 g<br>
Sodium Phosphate, Monobasic Dihydrate 1-5 g<br>
Potassium Phosphate, Dibasic 1 -5 g<br>
Sodium Citrate, Dihydrate 0.5-5 g<br>
Potassium Chloride 0.5-5 g<br>
Magnesium Sulfate, Heptahydrate 0.5-5 g<br>
PLURONICโข Polyol, L61 0.1 -5 mL<br>
Ferric Chloride, Heptahydrate 10-100 mg<br>
Zinc Sulfate, Heptahydrate 0.1 -10 mg<br>
Cobalt Chloride, Hexahydrate 0.1-10 mg<br>
Sodium Molybdate, Dihydrate 0.1 -10 mg<br>
Cupric Sulfate, Pentahydrate 0.1 -10 mg<br>
Boric Acid 0.1-lOmg<br>
Manganese Sulfate, Monohydrate 0.1-lOmg<br>
Hydrochloric Acid 10-100 mg<br>
Tetracycline 4-30 mg<br>
Yeast Extract 5-25 g<br>
NZ Amine AS 5-25 g<br>
Methionine 0-5 g<br>
Ammonium Hydroxide as required to<br>
control pH<br>
Sulfuric Acid as required to<br>
control pH<br>
A portion of the glucose, yeast extract, methionine, and NZ Amine AS is added to the<br>
medium initially, with the remainder being fed throughout the fermentation.<br>
The 10-liter fermentation was a fed-batch process with fermentation parameters<br>
set as follows:<br>
Agitation: 1000RPM<br>
Aeration: 10.0 slpm<br>
pH control: 7.3<br>
Temp.: 37 ยฐC<br>
Back pressure: 0.3 bar<br>
Glucose feed: computer-controlled using an algorithm to<br>
maintain the dissolved oxygen<br>
concentration (DC2) at 30% of air<br>
saturation after the DO2 drops to 30%.<br>
Complex nitrogen feed: constant feed rate of 0.2 mL/min<br>
beginning when ODsso reaches 40 and<br>
maintained for the remaining time of the<br>
run<br>
Run Duration. 40 to 50 hours<br>
In experiments involving glycerol-3-phosphate (G3P) feeding, the appropriate<br>
amount of 1 M G3P stock solution was spiked into the complex nitrogen feed and the<br>
subsequent supplemented feed feed-rate increased to deliver the desired amount of<br>
complex nitrogen plus G3P to the culture.<br>
The impact of the &amp;glpT mutation with or without the G3P feeding was assessed<br>
by the difference in the IGF-I accumulation. The total amount of IGF-I in a sample<br>
solubilized in 6M guanidine+100 mM DTT was measured by a reversed-phase HPLC<br>
method as described in U.S. Pat. No. 6,559,122.<br>
Results:<br>
Figure 3 shows that with the wild-type host (43E7) and AP promoter and<br>
continuously fed glucose, the amount of secreted IGF-I was distinctly higher when G3P<br>
was fed to the medium than when G3P was not added.<br>
Figure 4 shows that with the Ag/prhost (43F6) and AP promoter, the amount of<br>
secreted IGF-I was distinctly higher when G3P was fed to the culture at 1.18 or 3.28<br>
mmoles/hour, per approximately 8 liters, than when G3P was not added, but was not<br>
higher when 8.22 mmoles/hour, per approximately 8 liters, of G3P was fed. The<br>
optimum feed rate will be readily determined by one skilled in the art based on the<br>
product, type of organophosphate, etc. Under the conditions of the fermentation process<br>
described, culturing in a 10-liter fermentor to produce IGF-I, there is an optimal G3P feed<br>
rate, per approximately 8-10 liters, in the preferred range of about 1-7 mmoles/hour, more<br>
preferably about 1-6 mmoles/hour, still more preferably about 2-6 mmoles/hour, yet more<br>
preferably about 2-5 mmoles/hour, and most preferably about 3-4 mmoles/hour. Not<br>
only does this range of feed rates increase the amount of product over control, but also it<br>
extends the duration of production relative to the control.<br>
EXAMPLE 3<br>
Feeding of Glycero-3-phosphate to Improve Apo2 Ligand Accumulation in the 10-L<br>
Process<br>
Background on Apo2 Ligand<br>
Apoptosis-inducing ligand 2 (Apo2L) (Pitti et a/., J. Biol. Chem.. 271: 12687-<br>
12690 (1996)), also known as tumor necrosis factor-related apoptosis inducing ligand<br>
 (TRAIL) (Wiley et al., Immunity. 3: 673-682 (1995)), is a type II membrane protein and<br>
a member of the TNF family of ligands. Apo2L/TRAIL triggers apoptosis in a wide<br>
variety of cancer cells, but not in most normal cells, through binding to its cognate death<br>
receptors (WO 99/00423; Ashkenaa, FASEB J.. 13: (7) A1336 (April 23, 1999);<br>
Ashkenazi, Nature Reviews - Cancer, 2: 420-430 (2002)). A soluble fragment of the<br>
extracellular domain of Apo2 ligand, corresponding to amino acid residues 114-281<br>
(from here on referred to as Apo2L/TRAIL), is currently under investigation for potential<br>
clinical studies and has been successfully expressed in E. coli.<br>
General Description of the Fermentation Process:<br>
The expression vector encodes for the use of the alkaline phosphatase (AP)<br>
promoter to regulate the production of the approximately 19.5-kDa polypeptide. The<br>
expressed nascent polypeptides, upon release from the ribosomes, fold into monomers in<br>
the cytoplasm and further associate to become the biologically active homotrimer. During<br>
fermentation, the process parameters are set such that cellular activities are conducted at<br>
peak oxygen uptake rates of approximately 3.0 mmoles/L-min. After broth harvest, the<br>
cytoplasmically trapped heterologous protein is released by mechanical cell disruption<br>
into the cell lysate from which it may be recovered.<br>
Materials and Methods:<br>
pAPApo2-P2RU Plasmid Construction:<br>
pAPApo2-P2RU is described in WO 01/00832 published January 4, 2001.<br>
Briefly, this plasmid, the construct of which is shown in Figure 5, encodes the coexpression<br>
of Apo-2L (amino acid residues 114-281) and the rare-codon tRNA's encoded<br>
bypro2 and argU, which co-expression is regulated by the alkaline phosphatase<br>
promoter. The pBR322-based plasmid (Sutcliffe, Cold Spring Harbor Symp. Quant.<br>
Biol. 43:77-90 (1978)) pAPApo2-P2RU was used to produce the Apo-2L in E. coli. The<br>
transcriptional and translational sequences required for the expression of Apo-2L are<br>
provided by the alkaline phosphatase promoter and the trp Shine-Dalgamo sequence, as<br>
described for the plasmid phGHl (Chang et al, Gene, 55:189-196 (1987)). The coding<br>
sequence for Apo-2L (from 114-281) is located downstream of the promoter and Shine-<br>
Dalgarno sequences and is preceded by an initiation methionine. The coding sequence<br>
includes nucleotides (shown in Figure 6) encoding residues 114-281 of Apo-2L (Figure 6<br>
- SEQ ID NOS:1 and 2, respectively, for nucleotide and amino acid sequences) except<br>
that the codon encoding residue Proll9 is changed to "CCG" instead of "CCT" in order<br>
to eliminate potential secondary structure. The sequence encoding the lambda to<br>
transcriptional terminator (Scholtissek et al, Nucleic Acids Res.. 15: 3185 (1987))<br>
follows the Apo-2L coding sequence.<br>
Additionally, this plasmid also includes sequences for the expression of the<br>
tRNA's/wo2 (Komine etal, J. Mol. Biol. 212:579-598 (1990)) andargU/dnaY(Garciaet<br>
al, Cell 45:453-459 (1986)). These genes were cloned by PCR from K coli W3110 and<br>
placed downstream of the lambda t0 transcriptional-terminator sequence. This plasmid<br>
confers both tetracycline and ampicillin resistance upon the production host.<br>
Bacterial Strains and Growth Conditions:<br>
Strain 43E7 (E. coli W3110 QiuA(tonA) phoA l\(argF-lac) ptr3 degP ompT<br>
ilvG+)) was used as the wild-type production host for comparison to 43F6, the glpTmutated<br>
host for the expression of Apo2 ligand and the rare codon tRNA's. Competent<br>
cells of 43E7 or 43F6 were prepared and transformed with pAPApo2-P2RU using<br>
standard procedures. Transformants were picked from LB plates containing 20 ug/ml<br>
tetracycline (LB+Tet20), streak-purified, and grown in LB broth with 20 ug/ml<br>
tetracycline in a 30ยฐC shaker/incubator before being stored in DMSO at -80ยฐC.<br>
Fermentation Process for Apo2L Production:<br>
A shake-flask inoculum was prepared by inoculating sterile LB medium containing<br>
4-6 mM sodium phosphate with a freshly thawed stock culture vial. Appropriate<br>
antibiotics were included in the medium to provide selective pressure to ensure retention<br>
of the plasmid. Flask cultures were incubated with shaking at about 30ยฐC (28ยฐC-32ยฐC) for<br>
14-18 hours. This culture was then used to inoculate the production fermentation vessel.<br>
The inoculation volume was between 0.1% and 10% of the initial volume of medium.<br>
Production of Apo2L was carried out in the production medium given in Table 1 to<br>
achieve a final culture volume of approximately 10 liters. The fermentation process was<br>
conducted at about 30ยฐC (28-32ยฐC) and pH controlled at approximately 7.0 (6.5- 7.5).<br>
The aeration rate and the agitation rate were set to provide adequate transfer of oxygen to<br>
the culture. Just prior to depletion of the batched phosphate (at approximately 75-85 OD),<br>
a DL-alpha-glycerophosphate feed (vendor product specification shows product purity at<br>
80-90%, with beta-glycerophosphate listed as the main impurity) was initiated and fed at<br>
the desired feed rate. Throughout the fermentation process, the cell culture was fed<br>
glucose as the primary carbon source based on a computer algorithm while ensuring<br>
aerobic conditions.<br>
Two batch additions of approximately 50-150 uM (final concentration) ZnSC&gt;4<br>
were made during the fermentation process, one just prior to the induction of product<br>
expression, the other at approximately the mid-point of the production period for improved<br>
homotrimer assembly. In this example, the additions occurred at a culture optical density<br>
of about 80-120 ODsso and at about 28 hours post-inoculation.<br>
The fermentation was allowed to proceed for about 34-45 hours before being<br>
harvested.<br>
 A portion of these ingredients may be fed to the culture during the fermentation.<br>
Ammonium hydroxide was added as required to control pH.<br>
Assessment of Soluble Product Accumulation During Fermentation Process by Ion-<br>
Exchange HPLC Chromatography Method:<br>
Broth samples were taken over the time course of the fermentation process. Cells<br>
from 1 milliliter of broth samples diluted to a cell density of 20 ODsso were collected by<br>
centrifugation and the resultant cell pellets were stored at -20ยฐC until analysis. The cell<br>
pellets were thawed and resuspended in 0.5 ml of extraction buffer (50mM HEPES, pH<br>
8.0, 50 mM EDTA and 0.2 mg/ml hen egg-white lysozyme) and mechanically disrupted to<br>
release the product from the cytoplasm. Solids were removed from the cell lysates by<br>
centrifugation before the clarified lysates were loaded onto an HPLC column (DIONEX<br>
PROP ACโข IEX medium) for trimer quantisation. The HPLC assay method resolved the<br>
product away from the contaminating E. coli proteins by use of a 5%-22% gradient of 1M<br>
NaCl in a 25-mM phosphate (pH 7.5) buffer over 25 minutes at a flow rate of 0.5 ml/min.<br>
Assessment of Total Monomeric Apo2L Expression During Fermentation Process by<br>
Reversed-Phase HPLC Chromatography:<br>
Fresh culture broth or previously frozen and then thawed samples were used for<br>
the quantitation of total monomer production. 20 ul of sample was mixed into 480 ul of<br>
6M guanidine HC1, pH 9.0 with 100 mM DTT and was allowed to incubate at room<br>
36<br>
temperature for an hour before being centrifuged at 13,000 rpm for 15 mins to recover the<br>
extract. The extract was filtered through a spin-filter before 20 ul was loaded onto an<br>
HPLC column (PerSeptive Biosystems POROSยฎ Rl/10 medium) for reverse-phase<br>
chromatography. The HPLC separation was conducted at 80ยฐC with the mobile phases<br>
flowing at 1.0 ml/min and employed a gradient of 28% to 35% of acetonitrile with 0.1%<br>
TFA over 20 minutes for the resolution of the Apo2L away from the contaminating<br>
proteins. Peak detection was at 280-nm wavelength. The amount of monomer present in<br>
samples was calculated using an average response factor (mAU/ug) derived from the area<br>
under the peak associated with 5-20 ug of purified standards analyzed by the same<br>
method.<br>
Results:<br>
Figure 7 shows an improved specific product titer (referred to as specific titer in<br>
ug/OD-ml in the graph) with an optimum G3P feed rate to the AglpT host (43F6). All of<br>
the G3P-fed runs performed better than the no-feed control. In this example, as the feed<br>
rate for an approximately 8-liter culture increased from 6 to 12 mmole/hour, the specific<br>
product titer improved, but as the rate increased above 12 mmole/hour to 18 mmole/hour,<br>
the specific titer was lower. The optimum feed rate of G3P will be readily determined by<br>
one skilled in the art based on the product, type of organophosphate, etc. Under these<br>
particular conditions, culturing in a 10-liter fermentor cells for producing this specific<br>
product, Apo2L, the preferred feed rate of G3P, per approximately 8-10 liters, is<br>
preferably in the range of about 4 to 17 mmole/hour, more preferably about 6 to 16<br>
mmole/hour, still more preferably about 8 to 15 mmole/hour, and most preferably about<br>
10 to 14 mmole/hour.<br>
Figure 8 shows an improved specific product titer (referred to as specific total<br>
accumulation in ug/OD-ml in the graph) feeding G3P over feeding inorganic phosphate to<br>
the wild-type glpT host (43E7). While glycerophosphate feeding increased specific total<br>
accumulation of Apo2L, feeding inorganic phosphate negatively impacted specific total<br>
accumulation compared to the no-feed control. Similar trends would be expected using a<br>
lower gh/cerophosphate feed than was employed. The results here are intended to, and do,<br>
show that a high level of expression can be obtained by feeding glycerophosphate to a<br>
wild-type glpThost. Further, in this particular experiment, similar to the inorganic<br>
phosphate feed case, the culture cell density increased to over 200 OD550 when the<br>
glycerophosphate was fed, but not for the no-feed situation.<br>
EXAMPLE 4<br>
Expression of AP Promoter-Driven Apo2L Product during Active Growth Phase<br>
The same plasmid construction, production host strain, medium composition,<br>
fermentation process and product assay methods were used as described in Example 3<br>
except for the phosphate batching and the G3P addition. A portion of the inorganic<br>
phosphate typically included in the salt batching in a control process was replaced with an<br>
equivalent number of moles of G3P, either added immediately after inoculation or a few<br>
hours prior to the depletion of the batched inorganic phosphate. In these examples, the<br>
added G3P was expected to be the source of phosphate for a significant fraction of the<br>
cell growth subsequent to the addition.<br>
Fermentation Process for Apo2L Production during Active Growth Phase:<br>
The inoculum preparation protocol was the same as that described in Example 3.<br>
Production of Apo2L was carried out in the production medium given in Table 1 except<br>
that either 75% or 50% of the phosphate salts was eliminated from the initial batching<br>
and replaced with an equivalent number of moles of G3P added back as a batch addition<br>
post inoculation. The fermentation was conducted at about 30ยฐC (28-32ยฐC) and pH was<br>
controlled at approximately 7.0 (6.5-7.5) as per standard protocol. The aeration rate and<br>
the agitation rate were as described in Example 3. For the case where 50% of the<br>
inorganic phosphate was replaced with G3P, the inorganic phosphate was batched in prior<br>
to medium sterilization while the glycerol-3-phosphate replacement was made<br>
approximately 1-2 hours before the batched phosphate was expected to run out (at<br>
approximately 30-40 ODsso). For the case where 75% of the inorganic phosphate was<br>
replaced with G3P, both the inorganic phosphate and the G3P were added immediately<br>
after the inoculation of the fermentor. Throughout the fermentation process, the cell<br>
culture was fed glucose as the primary carbon source based on a computer algorithm<br>
while ensuring aerobic conditions. Zn additions were made during the fermentation<br>
process as described in the earlier section. The fermentation was allowed to proceed for<br>
about 34-45 hours.<br>
Results:<br>
Figure 9 shows the induction of heterologous protein expression occurring<br>
significantly earlier in the active growth phase when 50% - 75% of the PO4 batching was<br>
replaced with G3P addition for both the wild-type and g/p!T-mutated hosts, shifting the<br>
specific total accumulation curve to the left of that for the duplicate control cases<br>
conducted with the wild-type host with no G3P substitution. This indicates an advantage<br>
of this invention in that the product can be obtained earlier during the fermentation<br>
process.<br>
While all ratios of Pi to G3P tested herein achieved this advantage regardless of<br>
the host type, Table 2 shows that using the 1:1 or 1.3 ratio of Pi to G3P for the glpTmutated<br>
host 43F6 produced the highest volumetric Apo2L productivity rate (an average<br>
of about 0,34 versus an average of about 0.24 mg/ml-hr for the control host). Further,<br>
using either ratio and the wild-type or mutated host achieved the peak specific<br>
accumulation (in ug/OD-ml) earlier (22 to 26 hours versus 28 to 30 hours). This shows<br>
that in certain preferred embodiments, the invention can achieve similar, if not higher,<br>
amounts of monomeric Apo2L in approximately 10% to 25% less fermentation time than<br>
otherwise to improve process productivity significantly.<br>
(Table Removed) Expression of AP Promoter-Driven Apo2L Product Using a 50/50 Mixture of<br>
Alpha- and Beta- Glycerophosphate<br>
A procedure similar to that described in Example 3 was followed except that a<br>
cheaper grade of approximately 50:50 mix of alpha- and beta-glycerophosphate was<br>
employed instead of G3P as the feed using strain 61 Gl (glpT mutant host).<br>
Results<br>
Figure 10 shows that similar yield improvement over the no-feed control was<br>
obtained using the mixture or the higher grade G3P material. Use of the alpha/beta<br>
mixture would lessen the cost of raw material without compromising the production<br>
results.<br><br><br><br><br><br><br><br><br>
We claim:<br>
1.	A process for producing and increasing yields of a polypeptide heterologous to E. coli<br>
comprising:<br>
(a)	culturing E. coli cells comprising a nucleic acid encoding the polypeptide in a culture<br>
medium while feeding to the culture medium a transportable organophosphate, that is a sugar<br>
phosphate or a glycerophosphate, such that the nucleic acid is expressed, and<br>
(b)	recovering the polypeptide from the cells, so that the yields of the polypeptide are<br>
increased, wherein an inorganic phosphate is fed to the culture medium during the culturing<br>
step.<br>
2.	The process as claimed in claim 1 wherein the organophosphate is a glycerophosphate.<br>
3.	The process as claimed in claim 2 wherein the glycerophosphate is an alpha-glycerophosphate or beta-glycerophosphate, or a mixture thereof.<br>
4.	The process as claimed in claim 3 wherein the glycerophosphate is a mixture of glycerol-2-phosphate and glycerol-3-phosphate or is glycerol-3-phosphate.<br>
5.	The process as claimed in claim 1 wherein the culturing takes place in a shake flask or fermentor.<br>
6.	The process as claimed in claim 1 wherein the polypeptide is recovered from the cytoplasm, periplasm or culture medium of the cells.<br>
7.	The process as claimed in claim 1 wherein expression of the nucleic acid is regulated by an inducible promoter.<br>
8.	The process as claimed in claim 7 wherein the inducible promoter is the alkaline phosphatase promoter.<br>
9.	The process as claimed in claim 7 wherein the inducible promoter is the tac promoter.<br>
10.	The process as claimed in claim 7 wherein the inducible promoter is the T7 promoter.<br>
11.	The process as claimed in claim 7 wherein expression of the nucleic acid begins while in the active growth phase of the culturing step.<br>
12.	The process as claimed in claim 1 wherein the E. coli is deficient in chromosomal phoA.<br>
13.	The process as claimed in claim 1 wherein the E. coli is wild type with respect to chromosomal glpT.<br>
14.	The process as claimed in claim 1 wherein the E. coli is deficient in chromosomal glpT.<br>
15.	The process as claimed in claim 1 wherein the E. coli is deficient in chromosomal phoA and<br>
glpT.<br>
16.	The process as claimed in claim 15 wherein the E. coli is not deficient in chromosomal ugp.<br>
17.	The process as claimed in claim 1 wherein the polypeptide is a eukaryotic polypeptide.<br>
18.	The process as claimed in claim 1 wherein the polypeptide is a mammalian polypeptide.<br>
19.	The process as claimed in claim 1 wherein the polypeptide is insulin-like growth factor-1.<br>
20.	The process as claimed in claim 19 wherein the feed rate of the organophosphate is from about 1 to 7 mmoles/hour per about 8-10 liters and the culturing takes place in a 10-liter fermentor.<br>
21.	The process as claimed in claim 20 wherein the feed rate of the organophosphate is from about 2 to 6 mmoles/hour per about 8-10 liters.<br>
22.	The process as claimed in claim 21 wherein the feed rate of the organophosphate is from about 3 to 4 mmoles/hour per about 8-10 liters.<br>
23.	The process as claimed in claim 1 wherein the ratio of added inorganic phosphate to organophosphate ranges from about 1:10 to 1:0.25.<br>
24.	The process as claimed in claim 1 wherein the organophosphate is a hexose- 6 phosphate.<br>
25.	The process as claimed in claim 1 wherein the polypeptide is Apo2L.<br>
26.	The process as claimed in claim 25, wherein the feed rate of the organophosphate is from about 4 to 17 mmoles/hour per about 8-10 liters and the culturing takes place in a 10-liter fermentor.<br>
27.	The process as claimed in claim 26, wherein the feed rate is from about 6 to 16 mmole/hour per about 8-10 liters.<br>
28.	The process as claimed in claim 27, wherein the feed rate is from about 8 to 15 mmole/hour per about 8-10 liters.<br>
29.	The process as claimed in claim 28, wherein the feed rate is from about 10 to 14 mmole/hour per about 8-10 liters.<br>
30.	Process for producing polypeptides as claimed in any of the above claims substantially as described in the specification and illustrated in the drawings and sequence listings.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUFic3RyYWN0LSgwMy0wNC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Abstract-(03-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUFic3RyYWN0LSgxNC0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Abstract-(14-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUFzc2lnbm1lbnQtKDI4LTA3LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Assignment-(28-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWFzc2lnbm1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-assignments.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNsYWltcy0oMDMtMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Claims-(03-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1ERUxOUC0yMDA2LUNsYWltcy0oMDQtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-DELNP-2006-Claims-(04-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNsYWltcy0oMTQtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Claims-(14-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence Others-(02-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence Others-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDgtMjAxMSktLnBkZg==" target="_blank" style="word-wrap:break-word;">4760-DELNP-2006-Correspondence Others-(04-08-2011)-.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDgtMjAxMSkuLnBkZg==" target="_blank" style="word-wrap:break-word;">4760-DELNP-2006-Correspondence Others-(04-08-2011)..pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-DELNP-2006-Correspondence Others-(04-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence Others-(11-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence Others-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence Others-(28-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMjEtMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence-others (21-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycyAxLnBkZg==" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-correspondence-others 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDMtMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence-Others-(03-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLW90aGVycy0oMTQtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence-others-(14-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Correspondence-Others-(29-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LURyYXdpbmdzLSgwMi0wOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Drawings-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUZvcm0tMS0oMDItMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Form-1-(02-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUZvcm0tMS0oMDItMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Form-1-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUZvcm0tMS0oMTQtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Form-1-(14-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUZvcm0tMi0oMTQtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-Form-2-(14-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1ERUxOUC0yMDA2LUZvcm0tMy0oMDQtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-DELNP-2006-Form-3-(04-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LUdQQS0oMDItMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-GPA-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LXBjdC0zMjYucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-pct-326.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1kZWxucC0yMDA2LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">4760-delnp-2006-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc2MC1ERUxOUC0yMDA2LVBldGl0aW8tMTM3LSgwNC0wOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">4760-DELNP-2006-Petitio-137-(04-08-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="256783-metal-polymer-composite-a-method-for-its-extrusion-and-shaped-articles-made-therefrom.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="256785-method-to-delete-mail-attachments.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>256784</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4760/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Aug-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Jul-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Aug-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENENTECH, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1 DNA WAY, SOUTH SAN FRANCISCO, CALIFORNIA 94080-4990 (US),</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WOON-LAM, SUSAN LEUNG</td>
											<td>609 CUESTA AVENUE, SAN MATEO, CA 94403, US</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SWARTZ, JAMES R.</td>
											<td>1860 WHITE OAK DRIVE, MENLO PARK, CA 94025, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/70</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/007880</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-03-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/552,678</td>
									<td>2004-03-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/256784-process-for-producing-heterologous-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:10:23 GMT -->
</html>
