<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/256854-a-gram-negative-bacterium-producing-3-0-deacylated-lipopolysaccharide-lps by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:05:42 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 256854:&quot;A GRAM NEGATIVE BACTERIUM PRODUCING 3-0-DEACYLATED LIPOPOLYSACCHARIDE (LPS)&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A GRAM NEGATIVE BACTERIUM PRODUCING 3-0-DEACYLATED LIPOPOLYSACCHARIDE (LPS)&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The current invention provides new Gram negative polypeptides exhibiting lipid A 3-0-deaeylasc activity and arc capable of modifying and/or detoxifying gram negative LPS. The present invention also provides Gram negative bacteria, Gram negative bacterial lipopolysaccharides (LPS) and compositions comprising LPS, which are provided with or treated with a 3-O-deacylase activity according to the invention and which may be used for pharmaceutical and/or veterinary purposes, in particular for the preparation of whole cell or acellular vaccines against pathogenic Gram negatives such as Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention<br>
The current invention relates to the field of microbiology, in particular the biology of Gram negative LPS synthesis and modification. The invention also relates to the field of medicine, in particular to the field of vaccination against bacterial pathogens. The present invention further relates to Gram negative bacteria, Gram negative bacterial lipopolysaccharides (LPS) and compositions comprising LPS, which may be used for pharmaceutical and/or veterinary purposes, in particular for the preparation of vaccines against Gram negatives such as Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica. The invention further provides vaccines containing deacylated LPS, and to the use of modified and detoxified LPS in the preparation of whole cell and acellular vaccines.<br>
Background of the invention<br>
Bordetella pertussis infection is causative agent of whooping cough, with an estimated number of 60 millions cases each year, killing approximately 355,000 people worldwide annually (WHO), in particular children and immune compromised individuals. Although treatment with antibiotics is available (erythromycin), by the time the disease is diagnosed, bacterial toxins have often caused severe damage. Prevention of the disease is therefore of great importance. The prime means of control remains vaccination. Conventionally, vaccines against pertussis ("whooping-cough") infections have been based on whole cells of B. pertussis. Whole cell Bordetella pertussis vaccines, comprising whole bacteria that have been killed by heat treatment, formalin or other means, have been included in general vaccination programs since the early 1950's.<br>
Immunization with the whole-cell pertussis vaccine, while effective at preventing whooping cough in infants, has been associated with local, systemic and neurological reactions, including fevers, convulsions and encephalopathy in children. LPS is responsible for the major part of the adverse reactions in children following pertussis immunization. During bacterial infections of animals, LPS or its lipid A moiety activates the innate immune system through interaction with Toll-like receptors, primarily TLR-4. The host response to lipid A includes the production of cationic<br>
antimicrobial        peptides,        cytokines, chemokines	and	additional<br>
immunostimulatory molecules. In limited infections, the response to lipid A helps to clear the bacteria, but in overwhelming sepsis, high levels of circulating cytokines and procoagulant activity may damage the microvasculature and precipitate the syndrome of Gram-negative septic shock with disseminated intravascular coagulation.<br>
No conclusive evidence for a protective role of LPS in pertussis vaccines is available, although passive immunization experiments in mice have demonstrated that antibodies against LPS can confer a level of protection. In addition and more importantly, the presence of LPS in a vaccine however does provide adjuvant activity by enhancing the immune response against other antigens (K. Mills: Immunity to Bordetellapertussis. Microbes and Infection 3: 655-677 (2001).<br>
Concerns about safety have adversely affected vaccine uptake and have motivated the development of acellular pertussis vaccines, prepared with highly purified antigens from B. pertussis. In recent years, besides the so-called "whole cell vaccines" or "WCV's", also acellular vaccines or "ACVs" have now been introduced in several countries.<br>
Acellular vaccines normally comprise of 1 to 3 or more antigens of the pathogenic organism. In the case of B. pertussis antigens commonly used are: pertussis toxin (PT, normally treated to destroy its toxicity while retaining immunogenicity), filamentous hemagglutinin (FHA), fimbriae, and the 69 kD protein or pertactin (Prn). In general the reactogenicity of acellular vaccine is much lower than the reactogenicity of whole cell vaccine. Acellular vaccine is associated with a significantly reduced frequency of systemic reactions (fever, vomiting, fretfulness, anorexia) and local reactions (swelling, redness, warmth, tenderness, stiffness, pain). However, the clinical data are still controversial whether the protective immunity of acellular vaccines matches the protective effect of whole cell vaccine. In many studies the protective effect of whole cell vaccines is superior and a debate is ongoing whether this outweighs the risk of rare but serious adverse effects of whole cell vaccines in infants. Currently various immunizations schemes are being tested, wherein up to six doses of acellular vaccine are given. The whole cell vaccine was initially given 5 times, incorporated with the routine vaccines schedule with the last booster given between 4-6 years of age. The acellular pertussis vaccine is now recommended to be given 6 times including a last dose (combined with the diphtheria-tetanus vaccine) during the teenage years. The<br>
acellular vaccine appears to be safer than the whole cell-based vaccine, but both should not be given to children with a previous allergic reaction to the pertussis vaccine The adverse side effects of pertussis whole cell vaccines have been well documented in the art (review: S.H. Yeh: Pertussis: persistent pathogen, imperfect vaccines. Expert Rev. Vaccines 2: 113-127 (2003). Although currently used acellular vaccines in part overcome these adverse side effects, the protective immunity provided by these vaccines is still controversial and leaves much room for improvement. Importantly, in a mouse model superior long-term protection was found with whole-cell as compared to acellular vaccines (K. Mills: Immunity to Bordetella pertussis. Microbes and Infection 3: 655-677 (2001)). Moreover, acellular vaccines are more costly and difficult to produce, requiring isolation, extensive purification and quality control of various antigens and mixing and formulating them in optimal / desired quantities. There is clearly a long felt need for better B. pertussis, B. parapertussis, B. bronchiseptica and other Gram negative vaccines.<br>
Detailed description of the invention<br>
The current invention provides methods and means for the preparation of improved pertussis vaccines. The invention discloses novel Bordetella proteins. These novel B. pertussis, B. parapertussis and B. bronchiseptica proteins and DNA molecules encoding these proteins are used according to the invention to modify lipid A and thereby provide new B. pertussis, B. parapertussis and B. bronchiseptica bacterial strains and other Gram negative bacterial cells, comprising at least partially 3-0-deacylated and detoxified LPS. The current invention also provides improved compositions for vaccination, comprising Bordetella species bacterial cells comprising partially 3-0-deacylated LPS, pharmaceutical compositions comprising isolated and at least partially 3-0-deacylated LPS or in vitro 3-O-deacylated LPS. The invention further provides antibodies raised against and specific for 3-O-deacylated lipid A and/or LPS molecules.<br>
Lipopolysaccharide (LPS), a major component of the Gram-negative bacterial outer membrane, is known to be important for the functioning of this membrane as a permeability barrier and for the resistance against complement-mediated cell lysis (reviewed in 1). It consists of three covalently linked domains: lipid A, the core, and the O-antigen. Lipid A forms the hydrophobic membrane anchor and is responsible for<br>
the endotoxic activity of LPS. In Escherichia coli, it consists of a 1, 4'-bisphosphorylated p-l,6-linked glucosamine disaccharide, which is substituted with R-3-hydroxymyristic acid residues at positions 2, 3, 2', and 3' via ester or amide linkage. Secondary lauroyl and myristoyl groups substitute the hydroxyl group of R-3-hydroxymyristoyl at the 2'- and 3'-positions, respectively (Fig. 1A). Previous studies have shown that the phosphate groups, the glucosamine disaccharide, and the correct number and length of the acyl chains are important for the biological activity of lipid A (1,2,3).<br>
The basic structure of lipid A is reasonably well conserved among Gram-negative bacteria, although slight variations in the pattern of the substitutions of the two phosphates and the acyl-chain number and length are observed (4, 5). Additional modifications of lipid A (Fig. IB) are regulated in Salmonella enterica serovar Typhimurium (S. Typhimurium) by the two-component regulatory system PhoP/PhoQ (6, 7). In response to low Mg2+ levels, the sensor kinase PhoQ phosphorylates and thereby activates the transcriptional activator PhoP, which leads to the activation or repression of 40 different genes (6, 8). A second regulatory system involved in lipid A modification is the PmrA/PmrB two-component system, which itself is PhoP/PhoQ regulated (9, 10). Mutants with alterations in the PhoP/PhoQ system exhibit reduced virulence and an increased susceptibility to anti-microbial peptides (11, 12). Homologs of the PhoP/PhoQ and PmrA/PmrB systems have been identified in other Gram-negative bacteria, including E. coli, Yersiniapestis, and Pseudomonas aeruginosa (13, 14).<br>
Up till now, several lipid A-modifying enzymes have been identified. Substitution of the 1 and 4' phosphate groups with one or two 4-amino-4-deoxy-L-arabinose (L-Ara4N) moieties in S. Typhimurium was found to be dependent on the enzyme ArnT (15). Recently, the PmrC protein was identified to mediate the addition of phosphoethanolamine (pEtN) to lipid A in Salmonella enterica (16). Another enzyme, designated LpxO, catalyzes the C-dependent hydroxylation of lipid A (17), and a lipid A 1-phosphatase was identified in Rhizobium leguminosarum (18). All these enzymes are thought to reside within the inner membrane or periplasmic space (15, 16, 17, 18). Recently, a new class of outer membrane-localized lipid A-modifying enzymes was discovered. One of them is the palmitoyl transferase PagP (19). Palmitoylation of lipid A leads to an increased resistance to cationic anti-microbial peptides (7).<br>
Furthermore, palmitoylated lipid A antagonizes LPS-induced activation of human cells (20). Homologs of PagP are found, amongst others, in S. Typhimurium, Bordetella pertussis, Bordetella bronchiseptica, Bordetella parapertussis, Legionella pneumophila, E. coli, and Y. pestis (19, 21).<br>
Another outer membrane-localized lipid A-modifying enzyme is the 3-0-deacylase PagL (22). This enzyme was discovered in S. Typhimurium and shown to hydrolyze the ester bond at the 3 position of lipid A, thereby releasing the primary 3-hydroxymyristoyl moiety (22). Thus far, no obvious homologs of pagL could be found in the nonredundant or unfinished microbial databases, except in the closely related species Salmonella typhi and Salmonella paratyphi (22). Nevertheless, some other Gram-negative bacteria, including P. aeruginosa (14), R. leguminosarum (23), Helicobacter pylori (24), and Porhyromonas gingivalis (25) contain 3-O-deacylated lipid A species, suggesting that these organisms contain enzymes with a similar activity as PagL.<br>
The current invention discloses the identification of pagL homologs in a variety of Gram-negative bacteria. Limited sequence similarity between the various proteins and advanced bioinformatics tools were used to identify these homologs and their active-site residues. In this specification, we describe the presence and use of pagL homologs for heterologous expression in a variety of Gram-negative bacteria. Although the overall sequence similarity with known pagL genes from Salmonella spp. is rather low, a conserved PagL domain could be distinguished in the C-terminal region.<br>
The prior art only describes PagL proteins from Salmonella spp. and discloses heterologous expression of pagL only in E. coli (22), resulting in deacylated LPS. No data are available in the art about the presence of pagL homologs in other Gram negatives. Heterologous pagL expression in other Gram negatives, whether PagL is functional in other Gram negatives, the effect of PagL on lipid A / LPS composition, bacterial viability, toxicity and immunogenicity in other Gram negatives are all unknown factors. Only limited data for heterologous Salmonella pagL expression in E.coli is available, where a TLR response was measured in cells which express recombinant human TLR4, which does not reflect a natural situation of Gram negative infections (Kawasaki et al., J Biol Chem. 2004).<br>
The specification of the current invention discloses activity of the Pseudomonas aeruginosa and Bordetella bronchiseptica pagL homologs, which was confirmed upon<br>
heterologous expression in Escherichia coli and Bordetella spp., which resulted in the removal of a /?-3-hydroxymyristoyl group from lipid A. The effect on biological activity of LPS was assayed with human macrophage cells. Upon deacylation by PagL, E. coli lipid A (but not B. pertussis Lipid A) underwent another modification, which was the result of the activity of the endogenous palmitoyl transferase PagP. Furthermore, a conserved histidine-serine couple as active-site residues was identified, suggesting a catalytic mechanism similar to serine hydrolases. Finally, in vitro activity of PagL on LPS substrates is demonstrated. The biological function of PagL may be applied according to the invention to modify Gram negative pathogenicity, toxicity and immunogenicity. This modification may take place on whole bacterial cells or parts, fractions or compounds derivable thereof. The invention ultimately provides novel vaccines against Gram negative bacterial infections, comprising whole cells of Gram negative bacteria according to the invention or modified lipid A / LPS obtainable and/or isolated from these bacteria, or in vitro modified LPS / lipid A molecules.<br>
Detailed description<br>
Definitions:<br>
"Sequence identity" is herein defined as a relationship between two or more amino acid (polypeptide or protein) sequences or two or more nucleic acid (polynucleotide) sequences, as determined by comparing the sequences. In the art, "identity" also means the degree of sequence relatedness between amino acid or nucleic acid sequences, as the case may be, as determined by the match between strings of such sequences. "Similarity" between two amino acid sequences is determined by comparing the amino acid sequence and its conserved amino acid substitutes of one polypeptide to the sequence of a second polypeptide. "Identity" and "similarity" can be readily calculated by known methods, including but not limited to those described in (Computational Molecular Biology, Lesk, A. M., ed., Oxford University Press, New York, 1988; Biocomputing: Informatics and Genome Projects, Smith, D. W., ed., Academic Press, New York, 1993; Computer Analysis of Sequence Data, Part I, Griffin, A. M., and Griffin, H. G., eds., Humana Press, New Jersey, 1994; Sequence Analysis in Molecular Biology, von Heine, G., Academic Press, 1987; and Sequence Analysis Primer, Gribskov, M. and Devereux, J., eds., M Stockton Press, New York, 1991; and Carillo, H., and Lipman, D., SIAM J. Applied Math., 48:1073 (1988).<br>
Preferred methods to determine identity are designed to give the largest match between the sequences tested. Methods to determine identity and similarity are codified in publicly available computer programs. Preferred computer program methods to determine identity and similarity between two sequences include e.g. the GCG program package (Devereux, J., et al., Nucleic Acids Research 12 (1): 387 (1984)), BestFit, BLASTP, BLASTN, and FASTA (Altschul, S. F. et al., J. Mol. Biol. 215:403-410 (1990). The BLAST X program is publicly available from NCBI and other sources (BLAST Manual, Altschul, S., et al., NCBI NLM NIH Bethesda, MD 20894; Altschul, S., et al., J. Mol. Biol. 215:403-410 (1990). The well-known Smith Waterman algorithm may also be used to determine identity.<br>
Preferred parameters for polypeptide sequence comparison include the following: Algorithm: Needleman and Wunsch, J. Mol. Biol. 48:443-453 (1970); Comparison matrix: BLOSSUM62 from Hentikoff and Hentikoff, Proc. Natl. Acad. Sci. USA. 89:10915-10919 (1992); Gap Penalty: 12; and Gap Length Penalty: 4. A program useful with these parameters is publicly available as the "Ogap" program from Genetics Computer Group, located in Madison, WI. The aforementioned parameters are the default parameters for amino acid comparisons (along with no penalty for end gaps). Preferred parameters for nucleic acid comparison include the following: Algorithm: Needleman and Wunsch, J. Mol. Biol. 48:443-453 (1970); Comparison matrix: matches=+10, mismatch=0; Gap Penalty: 50; Gap Length Penalty: 3. Available as the Gap program from Genetics Computer Group, located in Madison, Wisconsin. Given above are the default parameters for nucleic acid comparisons.<br>
Optionally, in determining the degree of amino acid similarity, the skilled person may also take into account so-called "conservative" amino acid substitutions, as will be clear to the skilled person. Conservative amino acid substitutions refer to the interchangeability of residues having similar side chains. For example, a group of amino acids having aliphatic side chains is glycine, alanine, valine, leucine, and isoleucine; a group of amino acids having aliphatic-hydroxyl side chains is serine and threonine; a group of amino acids having amide-containing side chains is asparagine and glutamine; a group of amino acids having aromatic side chains is phenylalanine, tyrosine, and tryptophan; a group of amino acids having basic side chains is lysine, arginine, and histidine; a group of amino acids having acidic side chains is aspartic acid and glutamic acid and a group of amino acids having sulphur-containing side chains is<br>
cysteine and methionine. Preferred conservative amino acids substitution groups are: valine-leucine-isoleucine, phenylalanine-tyrosine, lysine-arginine, alanine-valine, and asparagine-glutamine. Substitutional variants of the amino acid sequence disclosed herein are those in which at least one residue in the disclosed sequences has been removed and a different residue inserted in its place. Preferably, the amino acid change is conservative. Preferred conservative substitutions for each of the naturally occurring amino acids are as follows: Ala to ser; Arg to lys; Asn to gin or his; Asp to glu; Cys to ser or ala; Gin to asn; Glu to asp; Gly to pro; His to asn or gin; lie to leu or val; Leu to ile or val; Lys to arg; gin or glu; Met to leu or ile; Phe to met, leu or tyr; Ser to thr; Thr to ser; Tip to tyr; Tyr to trp or phe; and, Val to ile or leu.<br>
A DNA segment according to the invention is "operably linked" when it is placed into a functional relationship with another DNA segment. For example, a promoter or enhancer is operably linked to a coding sequence if it stimulates the transcription of the sequence. DNA for a signal sequence is operably linked to DNA encoding a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide. Generally, DNA sequences that are operably linked are contiguous, and, in the case of a signal sequence, both contiguous and in reading phase. However, enhancers need not be contiguous with the coding sequences whose transcription they control. Linking is accomplished by ligation at convenient restriction sites or at adapters or linkers inserted in lieu thereof.<br>
The selection of an appropriate promoter sequence generally depends upon the host cell selected for the expression of the DNA segment. Examples of suitable promoter sequences include prokaryotic, and eukaryotic promoters well known in the art (see, e.g. Sambrook and Russell, 2001, supra). The transcriptional regulatory sequences typically include a heterologous enhancer or promoter that is recognised by the host. The selection of an appropriate promoter depends upon the host, but promoters such as the trp, lac and phage promoters, tRNA promoters and glycolytic enzyme promoters are known and available (see, e.g. Sambrook and Russell, 2001, supra). Expression vectors include the replication system and transcriptional and translational regulatory sequences together with the insertion site for the polypeptide encoding segment can be employed. Examples of workable combinations of cell lines and expression vectors are described in Sambrook and Russell (2001, supra) and in Metzger et al. (1988) Nature 334: 31-36. For example, suitable expression vectors can<br>
be expressed in, yeast, e.g. S.cerevisiae,    insect cells, e.g., Sf9 cells, mammalian cells, e.g., CHO cells and bacterial cells, e.g., E. coli or Bordetella spp.<br>
In a first embodiment, the current invention provides new polypeptides comprising lipid A 3-0-deacylase activity, whereby the polypeptide exhibits at least 25, 30, 40, 50, 60, 70, 80, 90, 95, 98 or 99 % amino acid identity with SEQ ID No. 1 and the polypeptide exhibits lipid A 3-O-deacylase activity as determined by the assays described in this specification, in vivo as exemplified in example 3 or in vitro according to example 9. Preferably the polypeptide having lipid A 3-0-deacylase activity is the polypeptide according to SEQ ID No. 1, the PagL protein of Bordetella bronchiseptica and Bordetella parapertussis, or a part thereof, a mutant thereof, or a fusion protein comprising at least a part of SEQ ID No. 1 comprising the lipid A 3-0-deacylase activity.<br>
In another embodiment the current invention comprises a nucleic acid sequence encoding the polypeptide exhibiting at least 25, 30, 40, 50, 60, 70, 80, 90, 95, 98 or 99 % amino acid identity with SEQ ID No. 1. Preferably, the nucleic acid sequence according to the invention exhibits at least 50, 60, 70, 80, 90, 95, 98 or 99% identity with the nucleic acid sequence according to SEQ ID No's 2 or SEQ ID No. 3, the pagL genes from B. bronchiseptica and B. parapertussis, respectively. The nucleic acid sequence may be a full length coding sequence or may be coding or non-coding ( or complementary) parts, fragments or even oligonucleotides derived thereof.<br>
The invention further comprises DNA vectors comprising the nucleic acid sequences according to the invention and/or encoding polypeptides exhibiting at least 25, 30, 40, 50, 60, 70, 80, 90, 95, 98 or 99 % amino acid identity with SEQ ID No. 1. DNA vectors according to the invention may be any vector known in the art, such as, but not limited to: plasmids, phages, phagemids, cosmids, artificial chromosomes, vectors for (homologous) genomic integration. The vectors may contain markers, such as selectable markers, providing antibiotic resistance, fluorescent labels, molecular tags etc. Methods for cloning nucleic acids and expression of encoded proteins according the invention are known to the skilled artisan and may for instanced be found in Sambrook et al., Molecular Cloning, Cold Spring Harbor Laboratory Press, NY 1989 and Ausubel F. et al., ed., Current Protocols in Molecular Biology, Wiley Interscience, 2004. Preferably the vector according to the current invention is a vector wherein the<br>
nucleic acid sequence is operably linked to regulatory sequences such as promoters, enhancers and terminators, providing expression of the gene and translation of the messenger into the lipid A 3-0-deacylase protein. Most preferably the vector is capable of conferring expression and lipid A 3-0-deacylase activity to a Gram negative bacterial host cell, optionally in an inducible fashion, for instance by the inducible tac promoter on plasmid pMMB67.<br>
The invention also provides antibodies capable of binding to the polypeptide according to SEQ ID No.l. Antibodies according to the invention may be monoclonal antibodies or polyclonal antibodies, raised in a host by injecting polypeptides according to the invention, as shown in the examples. Antibodies may be used for diagnostic purposes, for instance for analyzing expression of PagL proteins and mutants or homologs thereof in Gram negative bacteria. Antibodies may also be used for isolation and/or purification of proteins exhibiting lipid A 3-O-deacylase activity.<br>
In another aspect the invention pertains to Gram negative bacteria comprising a nucleic acid molecule according to the invention and/or encoding a polypeptide molecule according to the invention. Preferably the nucleic acid molecule is comprised within a DNA vector according to the invention, providing expression of the encoded protein in Gram negative bacterial cells and providing a source of lipid A 3-0-deacylase activity to the cell. Preferably said Gram negative bacterium is a bacterium which does not comprise in its genome a gene encoding a functional protein exhibiting lipid A 3-(9-deacylase activity such as a protein having significant ()40 percent) identity with a PagL protein as in SEQ ID No. 1. Most preferably, providing a source of lipid A 3-0-deacylase activity will alter the composition of the LPS in the outer membrane of the cell wall of the Gram negative bacterial cell. The Gram negative bacterium to be provided with a source of lipid A 3-O-deacylase activity may also be a bacterium comprising a non functional gene, having significant homology with a nucleic acid sequence as provided in SEQ ID No's 2 or 3, for instance by a mutation, frame shift or deletion, such as Bordetella pertussis.<br>
However, also a Gram negative bacterium that does comprises a (partly) functional gene in its genome encoding a protein having lipid A 3-O-deacylase activity, may be provided with an additional source for this activity within the scope of this invention. Gram negative bacteria may have a certain level of lipid A 3-O-deacylase activity but said activity may be enhanced by providing additional and/or enhanced<br>
expression of a polypeptide according to the invention. Preferably this will result in a temporary or permanent increase in lipid A 3-0-deacylase activity in the bacterium to such an extent that the lipid A and/or LPS composition of the bacterium is temporary or permanently altered or modified, as compared to the wildtype bacterium. Such a Gram negative bacterium may for instance be a Bordetella parapertussis or a Bordetella bronchiseptica bacterium, but any other Gram negative bacterium, preferably a pathogenic Gram negative bacterium, may be chosen, for example Neisseria spp., such as N.meningitidis, N.gonorrhoeae, N.lactamica.<br>
A Gram negative bacterium according to the invention comprising lipid A 3-0-deacylase activity or elevated levels of lipid A 3-0-deacylase activity preferably comprises at least partially 3-O-deacylated lipid A and/or LPS species in the outer membrane of the bacterial cell wall. Alternatively the Gram negative bacterium according to the invention may comprise LPS or lipid A species carrying a secondary modification after the 3-0-deacylation of lipid A, such as palmitoylation, dephosphorylation or any other secondary modification after 3-O-deacylation of lipid A. The bacterial cell according to the invention may comprise at least 10, 20, 30, 40, 50, 60, 70, 80 or 90 percent of its total LPS/lipid A in 3-O-deacylated form, or may alternatively comprise at least 10, 20, 30, 40, 50, 60, 70, 80 or 90 percent of its lipid A/LPS in a form carrying a secondary modification, such as for example, but not limited to, palmitoylation or dephosphorylation.<br>
In another aspect the current invention provides methods for producing partially 3-O-deacylated LPS. In a first embodiment, such a method comprises the step of culturing the Gram negative bacterium according to the invention under conditions conducive to synthesis of the deactylated LPS, and optionally, recovery of the deacylated LPS. Methods for culturing various Gram negative bacteria are known in the art and may for instance be found in Methods for General and Molecular Bacteriology. P. Gerhardt et al., Eds. American Society for Microbiology, Washington DC, 1994. Methods for recovery, isolation and/or purification of LPS are also known in the art (Meningococcal Vaccines, Methods and Protocols. A.J. Pollard and M.C.J. Maiden, Eds. Chapter 12: Construction of LPS mutants, pp.155-165. Humana Press, Totowa, New Jersey, 2001) and may for instance be carried out according to the examples provided in this specification.<br>
Alternatively the current invention provides a method for producing at least partially 3-O-deacylated LPS or lipid A in vitro, the method comprising the steps of providing a composition comprising LPS or lipid A in crude or (partially) purified form and bringing this composition into contact with a polypeptide or protein according to the invention under conditions conducive to enzymatic 3-Odeacylation in vitro. Such conditions can be found in the current specification, in example 9 and in the methods section.<br>
In yet another embodiment the current invention provides compositions comprising at least partially 3-Odeacylated LPS and/or lipid A, preferably comprising at least 10, 20, 30, 40, 50, 60, 70, 80, 90, 95, 98 or 99 percent of the total LPS or lipid A in its 3-0-deacylated form or in another form carrying a secondary modification after 3-0-deacylation, such as a palmitoylated form.<br>
Compositions according to the invention, comprising partially 3-O-deacylated LPS and/or lipid A and optionally carrying secondary modifications, either comprised in the outer membrane of the cell wall of bacterial cells, or in crude or purified forms, may be used for the manufacture of pharmaceutical compositions. In a particularly preferred embodiment, such pharmaceutical compositions according to the invention may be compositions suitable for vaccination purposes. Such pharmaceutical compositions are capable of eliciting an immune response in a host organism, preferably a mammal, more preferably a human, against a Gram negative bacterium. The presence of at least partially 3-0-deacylated LPS and/or lipid A or alternatively LPS carrying secondary modifications after 3-O-deacylation, provides several advantages, such as the advantage of a reduced toxicity, a reduced number and reduced severity of side effects in the subject and a higher tolerated dose for the composition in the subject to be treated or vaccinated. The pharmaceutical composition may contain 1 or more excipients and/or adjuvants. Pharmaceutically acceptable excipients and adjuvants are known in the art and may be freely chosen by the skilled person, for instance from: Current protocols in Immunology, Wiley Interscience 2003 or Remmington's Pharmaceutical Sciences, 18th ed., Mack Publishing Company, 1990.<br>
In a first embodiment the pharmaceutical composition may be a whole cell vaccine, comprising live or live attenuated bacterial cells or non-viable bacterial cells, which may have been inactivated by freezing, heat treatment, mechanical disruption, chemical treatment or other methods known in the art of pharmacy and vaccination<br>
(J.L. Pace, H.A. Rossi, V.M. Esposito, S.M. Frey, K.D. Tucker, R.I. Walker. Inactivated whole-cell bacterial vaccines: current status and novel strategies. Vaccine 16: 1563-1574 (1998)). Preferably the bacterial cell is a Gram negative, pathogenic bacterial cell, more preferably the bacterial cell is of the genera Bordetella, Salmonella, Shigella, Neisseria, Klebsiella, Pseudomonas, Haemophilus, Escherichia, Proteus and most preferably is Bordetella pertussis, Bordetella parapertussis or Bordetella bronchiseptica.<br>
In an second preferred embodiment, the pharmaceutical composition according to the invention may be an a-cellular vaccine, comprising of 1, 2, 3 or more immunogenic components of the Gram negative pathogenic bacterium and comprising at least partially 3-0-deacylated LPS or lipid A, or said LPS carrying secondary modifications after 3-0-deacylation. Preferably the partially 3-0-deacylated lipid A and/or LPS is obtained from a Gram negative, pathogenic bacterial cell according to the invention, wherein preferably the bacterial cell is of the genus Bordetella, and most preferably is Bordetella pertussis, Bordetella parapertussis or Bordetella bronchiseptica. The at least partially 3-0-deacylated lipid A and/or LPS, optionally carrying secondary modification after deacylation, may be used for eliciting a protective immune response against the bacterium producing it, but alternatively may also be used and admixed to other compositions for use as a suitable adjuvant substance. LPS is known in the art to be a suitable adjuvant for vaccination purposes, activating Toll like receptors and stimulating an innate immune response. Partially 3-0-deacylated and at least partially detoxified LPS and/or lipid A according to the invention largely retains this immune stimulating (adjuvant) activity, while causing less toxicity related adverse side effects, such as local swelling, redness, pain and fever.<br>
Pharmaceutically acceptable composition and vaccines according to the invention may be used in methods of treatment of subjects suffering from or at risk of acquiring a pathogenic, Gram negative bacterial infection, comprising administering the pharmaceutical composition, a whole cell or an a-cellular vaccine according to the invention. The use of specific adjuvants, the relative and absolute amounts of substances in the compositions and the doses regimen for the administration are known or may be determined by the skilled person and may be adapted for the circumstances such as the particular pathogenic infection or the status of the particular subject to be treated. The doses regimen may comprise a single dose but may also comprise multiple<br>
doses, for instance booster doses and may be administered orally, intranasally or parenterally. Various doses regimens for vaccination purposes are known in the art and and may be suitably adapted by the skilled person.<br>
Brief Description of Figures<br>
Fig. 1. Lipid A architecture. A, E. coli lipid A consists of a bisphosphorylated glucosamine disaccharide substituted with four /?-3-hydroxymyristoyl moieties, of which the 2' and 3' fatty-acyl chains are esterified with laurate and myristate, respectively. B, Regulated modifications of Salmonella lipid A. Substitution of the phosphate moieties with L-Ara4N or pEtN is mediated by ArnT and PmrC, respectively, the formation of a 2-hydroxymyristate-modified lipid A by LpxO, the addition of a secondary palmitoyl chain at the 2-position by PagP, and the removal of the 3-hydroxymyristoyl moiety at the 3-position by PagL.<br>
Fig. 2. Multiple sequence alignment of the PagL proteins. Sequences were aligned using ClustalW (http://www.ch.embnet.org/software/ClustalW.html). Hyphens indicate gaps introduced for optimal alignment. Absolutely conserved residues are marked with asterisks. Indicated by colons and dots are strongly and weakly conserved residues, respectively. The pagL ORF in B. pertussis is disrupted by a frame shift, which was restored for this alignment by adding two nucleotides in codon 33. The GenBank protein accession numbers for the PagL homologs are: S. Typhimurium AAL21147, B. bronchiseptica NP_890306, B. parapertussis NP_885487, B. pertussis BX470248§, P. aeruginosa NP_253350, P. fluorescens NZ_AAAT03000006§, P. putida NC_002947§, P. syringae ZP_00125465, B. fungorum NZ_AAAJ03000003§, B. mallei NC_002970§, B. pseudomallei NC_002930§, R. metallidurans ZP_00274744, R. solanacearum NP_522762, and A. vinelandii ZP_00089534. The symbol § indicates GenBank Accession Numbers of whole (unfinished) genomes, in which the PagL homologs were manually identified.<br>
Fig. 3. Expression and membrane localization of PagL in E. coli BL21 Star™ (DE3). Membranes from E. coli BL21 Star™ (DE3) containing empty pET-1 la or the pPagL plasmids were isolated and analyzed by SDS-PAGE. Proteins were stained with Coomassie Brilliant Blue. Asterisks indicate the bands that were subjected to microsequencing and were found to correspond to the mature PagL proteins. The band indicated by the double asterisk corresponds to the PagL(Bb) precursor protein. Molecular weight standard proteins are present on the left side.<br>
Fig. 4. Analysis by Tricine-SDS-PAGE of LPS modification in vivo. Exponentially growing E. coli BL21 Star™ (DE3) cells containing pET-lla or the pPagL constructs were induced with IPTG for the indicated time, after which 1 ODeoo unit culture samples were collected and analyzed by Tricine-SDS-PAGE.<br>
Fig. 5. GC/MS analysis of wild-type and PagL-modified E. coli BL21 Star™ (DE3)<br>
LPS. GC/MS analysis of purified E. coli BL21 Star™ (DE3) wild-type LPS (WT), PagL(St)-modified LPS (L(St)), PagL(Bb)-modified LPS (L(Bb)), and PagL(Pa)-modified LPS (L(Pa)) (t= time after induction). Indicated are the normalized C14/C14-3OH ratios with wild-type LPS set at 100 (values shown above bars).<br>
Fig. 6. Structural analysis by ESI-MS of wild-type and PagL-modified E. coli BL21 Star™ (DE3) LPS. Lipid A species from wild-type E. coli BL21 Star™ (DE3) containing empty pET-lla (A), and lipid A species modified by PagL(st) (B), PagL(pa) (C), and PagL(Bb) (D) were analyzed by ESI-MS. Major peaks at m/z 1797, 1928, 1622, and 1490 were interpreted as the characteristic hexa-acylated to-phosphate species that is typically found in E. coli, a hexa-acylated to-phosphate species substituted with an L-Ara4N moiety, a 3-0-deacylated wono-phosphate species substituted with an L-Ara4N moiety, and a 3-O-deacylated wono-phosphate species, respectively. The major peaks at m/z 1716 and 1847 probably represent fragment ions of the species at m/z 1797 and 1928.<br>
Fig. 7. In vivo re-modification of deacylated LPS and the role of endogenous PagP.<br>
A, Exponentially growing E. coli BL21 Star™ (DE3) cells containing the empty pET-lla vector or the pPagL(Bb) plasmid were induced with IPTG for the indicated time<br>
period. Samples corresponding to 1 ODeoo unit were collected and analyzed by Tricine-SDS-PAGE. B and C, The fatty acid content of purified E. coli BL21 Star™ (DE3) wild-type LPS (WT) and PagL(Bb)-modified LPS (L(Bb)), isolated at the indicated time after induction ofpagL expression, was analyzed by GC/MS. Indicated are the normalised C14/C14-3OH (B) and C16/C14 (C) ratios with wild-type LPS set at 100 (values shown above bars). D, Exponentially growing wild-type E. coli BL21 Star™ (DE3) or E. coli BL21 Star™ (DE3) and its pagP mutant derivative JG101, containing pPagL(pa), were induced with IPTG for the indicated time period, after which 1 ODgoo unit culture samples were collected and analyzed on Tricine-SDS-PAGE gel.<br>
Fig. 8. Topology model for PagL from P, aeruginosa. A model for the topology of PagL(pa) was constructed using the general rules of outer membrane protein architecture as described in (44). The proposed model consists of an eight-stranded p-barrel with four loops (LI-4) extending into the external environment. Residues in the postulated p-strands are shown in diamonds, which are shaded for residues that are exposed to the lipid bilayers. Hisi49 and Serisi (marked in red; position in the PagL(pa) precursor) are absolutely conserved (Fig. 2) and are suggested to be part of a 'classical' catalytic triad of a serine hydrolase. Potential candidates for the acidic residue of the catalytic triad are indicated in yellow. Numbers refer to the position of the residues in the precursor sequence.<br>
Fig. 9. Identification of PagL(Pa) active-site residues by amino acid substitution.<br>
Exponentially growing E. coli BL21 Star™ (DE3) cells containing the empty pET-1 la vector, the pPagL(pa) plasmid, or the mutant pPagL(pa) plasmids were induced with IPTG for 75 min, after which 1 ODeoo unit culture samples were collected and analyzed by SDS-PAGE followed by immunoblotting with primary antibodies against PagL(pa) (A) and by Tricine-SDS-PAGE to visualize LPS (B).<br>
Fig. 10. In vivo modification of B. pertussis LPS. A, LPS from wild-type B. pertussis strain Tohama or B. pertussis strain Tohama carrying the pMMB67EH-PagL(Bb) plasmid was isolated and analyzed by Tricine-SDS-PAGE. B, The fatty acid content of purified B. pertussis strain Tohama wild-type LPS (WT), and PagL(Bb)-modified LPS<br>
(PagL) was analyzed by GC/MS. Indicated is the normalised C14-3OH/C10-3OH ratio with wild-type LPS set at 100 (values shown above bars).<br>
Fig. 11. Biological activity of isolated LPS. IL-6 (A) or IL-10 (B) induction in MM6 cells by purified LPS. The horizontal axes give the LPS concentration in mg/ml and the vertical axes give the ELISA-read out at 450 nm.<br>
Fig. 12. Heat-modifiability of purified, refolded PagL(pa)(-) analysed by semi-native<br>
SDS-PAGE. Coomassie Brilliant Blue stained semi-native SDS-PAGE gel showing the heath-modifiability of purified, refolded PagL(pa)(-). Samples were treated in sample buffer containing 0.1% SDS at room temperature (RT) or 2% SDS at 100°C (15 min), prior to electrophoresis. Molecular weight standard proteins are present on the left side.<br>
Fig. 13. In vitro LPS modification by membrane-bound or in vitro refolded PagL.<br>
Silver-stained Tricine-SDS-PAGE gels showing in vitro PagL activity. A, Purified N. meningitidis L3-LPS was incubated in a detergent-containing buffer for 18 h at 37°C with or without cell envelopes prepared from E. coli BL21 Star™ (DE3) containing empty pET-lla, or the pPagL plasmids. B, Purified N. meningitidis L3-LPS was incubated in a detergent-containing buffer in the absence or presence of 5 mM EDTA for 18 h at 37°C with or without 4 ug in vitro refolded PagL(pa) without its signal sequence (PagL(pa)(-)). Similar amounts of assay mixes were loaded in all lanes.<br>
Fig 14. Analysis by Tricine-SDS-PAGE of in vivo LPS modification. LPS was<br>
isolated from wild-type and PagP/PagL-expressing B. pertussis strain Tohama by hot phenol/water extraction and analysed by Tricine-SDS-PAGE.<br>
Fig. 15. Structural analysis by ESI-MS of wild-type and PagL/PagP-modified B. pertussis LPS. Lipid A species from wild-type B. pertussis strain Tohama (A), and lipid A species modified by PagLpb) (B), PagP(EC) (C), and PagP(ep) (D) were analysed by ESI-MS. Major peaks at m/z 1557, 1477, 1387, 1307, 1251, and 1081 were interpreted as the characteristic penta-acylated to-phosphate species that is typically found in B. pertussis, the corresponding penta-acylated wooo-phosphate species, the deacylated lipid A species of the molecular ion at m/z 1557 missing the primary 3-<br>
hydroxydecanoic acid residue at the 3 position, the deacylated lipid A species of the molecular ion at m/z 1477 missing the primary 3-hydroxydecanoic acid residue at the 3 position, the deacylated lipid A species of the molecular ion at m/z 1477 missing a primary 3-hydroxytetradecanoic acid residue, and the deacylated lipid A species of the molecular ion at m/z 1477, missing both the primary 3-hydroxydecanoic acid residue at the 3 position and a primary 3-hydroxytetradecanoic acid residue, respectively. The peaks at m/z 1320, 1490, 1545, 1625, 1715, and 1796 correspond to the PagP-mediated palmitoylation of the molecular ions present at m/z 1081, 1251, 1307, 1387, 1477, and 1557, respectively.<br>
Examples<br>
Experimental procedures<br>
Bacterial Strains and Growth Conditions<br>
All bacterial strains used in this study are described in Table I. Typically, the E. coli and P. aeruginosa strains were grown at 37°C on modified Luria-Bertani broth agar, designated LB agar (26), or in LB broth, while shaking at 200 rpm. For E. coli, the medium was supplemented with 0.2% glucose. When appropriate, bacteria were grown in the presence of 100 ug/ml ampicillin, 50 p.g/ml kanamycin, 50 ng/ml naladixic acid, or 100 ug/ml streptomycin, for plasmid maintenance or strain selection. S. Typhimurium SRI 1 was grown on LB agar plates at 37°C. B. bronchiseptica and B. pertussis strains were grown at 35°C on Borduet-Gengou agar (Difco) supplemented with 15% defibrinated sheep blood. To induce the expression of the pagLb) gene in B. pertussis, the bacteria were grown in synthetic Thijs medium (48) supplemented with 1 mM isopropyl-1-thio-p-D-galactopyranoside (IPTG) (end concentration) at 35°C, while shaking (180 rpm).<br>
TABLE 1: Bacterial strains andplasmids used in this study(Table Removed)<br><br>
Recombinant DNA Techniques<br>
Plasmid DNA was isolated using the Promega Wizard®P/«.s SV Minipreps system. Calf-intestine alkaline phosphatase and restriction endonucleases were used according to the instructions of the manufacturer (Fermentas). DNA fragments were isolated from agarose gels using the Qiagen quick gel extraction kit. Ligations were performed by using the rapid DNA ligation kit (Roche).<br>
The pagL genes from S. Typhimurium SRI 1 (pagL\), B. bronchiseptica B505 (pagL(Bb)), and the pagL gene, with or without its signal sequence-encoding part, from<br>
P. aeruginosaPAO25 (pagL(pa)ypagL(pa)(- )) were cloned into pET-lla (Novagen)<br>
behind the T7 promoter. The genes were amplified by PCR using chromosomal DNA<br>
as template. Template DNA was prepared by resuspending ~109 bacteria in 50 ul<br>
distilled water, after which the suspension was heated for 15 min at 95°C. The<br>
suspension was then centrifuged for 1 min at 16,100x g, after which the supernatant<br>
was used as template DNA. The sequences of the forward primers, which contained an<br>
Ndel site (underlined), including an ATG start codon, were 5'-<br>
AAC ATATGAAGAGAATATTTATATATC-3'	(pagL(SQ),	5'-<br>
AACATATGAAGAAACTACTTCCGCTGG-3'	(pagL),	5'-<br>
AACATATGGCGGACGTCTCGGCCGCCG-3'	(pagZ(Pa)(-)),	and         5'-<br>
AACATATGCAATTTCTCAAGAAAAACA-3' (pagL(Eb)). The sequences of the reverse primers, which contained an BamHl site (underlined) and included a stop codon, were 5' - AAGGATCCTC AGAAATTATA ACTAATT-3' (pagL(St)), 5'-AAGGATCCCTAGATCGGGATCTTGTAG-3' (pagL(P&amp;), pagL(Pa)(-)), and 5'-AAGGATCCTCAGAACTGGTACGTATAG-3' (pagL(Bb)). The PCRs were done under the following conditions: 50 ul total reaction volume, 25 pmol of each primer, 0.2 mM dNTPs, 3 ul template DNA solution, 1.5% dimethylsulfoxide, 1.75 units of Expand High Fidelity enzyme mix with buffer supplied by the manufacturer (Roche). The temperature program was as follows: 95°C for 3 min, a cycle of 1 min at 95°C, 1 min at 60°C, and 1 min 30 s at 72°C repeated 30 times, followed by 10 min at 72°C and subsequent cooling to 4°C. The PCR products were purified from agarose gel and subsequently cloned into pCRII-TOPO. Plasmid DNA from correct clones was digested with Ndel and BamHl, and the PagL-encoding fragments were ligated into Ndel/BamHl-digested pET-lla. The ligation-mixture was used to transform E. coli DH5a using the CaCb method (27). Plasmid DNA from transformants was checked for presence of the correct PagL-encoding insert by digestion with Ndel and BamHl. Plasmids that gave a correct digestion profile were designated pPagL(pa), pPagL(pa)(-), pPagL(Bb), and pPagL(st) (Table I). The correct coding sequences of the cloned pagL genes were confirmed by nucleotide sequencing in both directions. To subclone the pagL(Qb) gene into the broad-host-range, low-copy pMMB67EH vector, pPagL(Bb) plasmid DNA was digested with Xbal and HinDlll, and the PagL(Bb)-encoding fragment was ligated into .YZjalAWwDIII-digested pMMB67EH. The ligation mixture was used to transform E. coli DHSoc. Plasmid DNA from transformants was checked<br>
for presence of the correct PagL-encoding insert by digestion with Xbal and HinDlll. A plasmid that gave a correct digestion profile was designated pMMB67EH-PagL(Bb) (Table I). The latter plasmid was used to transform transform E. coli SM10, which allowed subsequent transfer of pMMB67EH-PagL(Bb) to B. pertussis by conjugation on solid medium as described by Stibitz et al (52). Mutations were introduced inpagL by using the QuikChange® Site-Directed Mutagenesis Kit (Stratagene) and the primers listed in Table II. Plasmid pPagL(pa) was used as the template in which the mutations were created. The presence of the correct mutations was confirmed by nucleotide sequencing in both directions. TABLE II<br>
Primers used for site-directed mutagenesis<br>
Name"	Sequence (5'-3')b<br>
H81A_FW	GAAGGCGCCGGCAAGGCGTCGCTGTCGTTCGCT<br>
H81A_REV	AGCGAACGACAGCGACGCCTTGCCGGCGCCTTC<br>
H81N_FW	GAAGGCGCCGGCAAGAACTCGCTGTCGTTCGCT<br>
H81N_REV	AGCGAACGACAGCGAGTTCTTGCCGGCGCCTTC<br>
S 8 4A_FW	GGCAAGCATTCGCTGGCGTTCGCTCCGGTATTC<br>
S8 4A_REV	GAATACCGGAGCGAACGCCAGCGAATGCTTGCC<br>
S84C_FW	GGCAAGCATTCGCTGTGCTTCGCTCCGGTATTC<br>
S 8 4 C_REV	GAATACCGGAGCGAAGCACAGCGAATGCTTGCC<br>
HI4 9A_FW	GGCGTTCGGGCGATCGCGTATTCCAACGCCGGC<br>
H14 9A_REV	GCCGGCGTTGGAATACGCGATCGCCCGAACGCC<br>
H14 9N_FW	GGCGTTCGGGCGATCAACTATTCCAACGCCGGC<br>
H14 9N_REV	GCCGGCGTTGGAATAGTTGATCGCCCGAACGCC<br>
S151A_FW	CGGGCGATCCACTATGCGAACGCCGGCCTGAAA<br>
S151A_REV	TTTCAGGCCGGCGTTCGCATAGTGGATCGCCCG<br>
S151C_FW	CGGGCGATCCACTATTGCAACGCCGGCCTGAAA<br>
SI51C REV	TTTCAGGCCGGCGTTGCAATAGTGGATCGCCCG<br>
aThe primer name gives the amino acid substitution, e.g. H81A_FW indicates that the oligonucleotide shown was used as the forward primer in a site-directed mutagenesis procedure to substitute the histidine at position 81 of the precursor PagL(Pa) by an alanine. b Introduced mutations are underlined.<br>
SDS-PAGE and Immunoblotting<br>
Proteins   were   analyzed   by   sodium   dodecyl   sulfate-polyacrylamide   gel electrophoresis (SDS-PAGE) (28), with 0.2% SDS in the running gel, by using the Bio-<br>
Rad Mini-PROTEAN®3 apparatus. Samples were applied to a 13% polyacrylamide gel with a 4% stacking gel and subjected to electrophoresis at 150 V. Proteins were stained with Coomassie Brilliant Blue. Prestained or unstained Precision<br>
TM<br>
Plus Protein Standard from Bio-Rad was used to determine the relative molecular mass (Mr). For Western blotting, proteins were transferred from SDS-PAGE gels onto nitrocellulose membranes. The membranes were blocked overnight in phosphate-buffered saline (PBS) (pH 7.6), 0.5% non-fat dried milk, 0.1% Tween-20 and incubated with primary antibodies directed against PagL(pa) in blocking buffer, followed by an incubation with horse-radish peroxidase-conjugated rabbit anti-guinea pig IgG antibodies (Sigma) in blocking buffer. Blots were developed using SuperSignal® WestPico Chemiluminescent Substrate (Pierce).<br>
Semi-Native SDS-PAGE<br>
Proteins were analysed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (28), with 0.2% SDS in the running gel, by using the Bio-Rad Mini-PROTEAN®3 apparatus. For semi-native SDS-PAGE, no SDS was added to the running and stacking gel, and the samples were not heated prior to electrophoresis. Samples were applied to a 13% polyacrylamide gel with a 4% stacking gel and subjected to electrophoresis at 150 V. For semi-native SDS-PAGE, electrophoresis was performed at a constant current of 15 mA on ice. Proteins were stained with Coomassie Brilliant Blue. Prestained or unstained Precision Plus Protein Standard from Bio-Rad was used to determine the relative molecular mass (Mr).<br>
Tricine-SDS-PAGE<br>
To LPS-containing samples 0.5 mg/ml proteinase K (end concentration) was added to the sample buffer (28). The samples were incubated for 60 min at 55°C, followed by 10 min at 95°C to inactivate proteinase K. The samples was then diluted 10 fold by adding sample buffer, after which 2 ul of the sample were applied to a Tricine-SDS-PAGE gel (30). The bromophenol blue was allowed to run into the separating gel at 35 V, after which the voltage was increased to 105 V. After the front reached the bottom of the gel, the samples were left running for another 45 min. The gels were fixed overnight in water/ethanol/acetic acid 11:8:1 (v/v/v) and subsequently stained with silver as described (31).<br>
Polyclonal Antibodies<br>
For antibody production, pPagL(Pa)(-), was used to transform E. coli BL21 Star™ (DE3) to allow for expression of the truncated pagL gene. The PagL(pa) protein, accumulating in inclusion bodies, was isolated (29), purified from a preparative SDS-PAGE gel, and used for immunization of guinea pigs at Eurogentec.<br>
Microsequencing<br>
Proteins were transferred from SDS-PAGE gels to an Immobilon™-P polyvinylidene difluoride membrane (Millipore Corp.) in 192 mM glycine, 25 mM Tris (pH 8.3), 10% methanol (v/v) at 100 V for 1 h using the Bio-Rad Mini-PROTEAN®2 blotting apparatus. After transfer, the membrane was washed 3 times for 15 min with distilled water. Transferred proteins were stained with Coomassie Brilliant Blue. The membrane was dried in the air, and the putative PagL bands were excised and subjected to microsequencing at the Sequencing Center Facility, Utrecht University, the Netherlands.<br>
Isolation of IPS and analysis by Gas Chromatography-Mass Spectrometry (GC/MS)<br>
LPS was isolated using the hot phenol/water extraction method (3). In short, B. pertussis strain Tohama, with or without plasmid pMMB67EH-PagL(Bt)), was grown in 3 liters Thijs medium (48) in the presence of 1 mM IPTG (end concentration). Cells were harvested by centrifugation and resuspended in 40 mM sodiumphosphate buffer (pH 7;0) containing 5 mM EDTA. The cells were treated over night with lysozyme at 4°C, after which an equal volume of phenol was added. The suspension was heated to 70°C and incubated for 30 minutes while shaking. The suspension was cooled to 10°C, after which phases were separated by centrifugation. The upper phase was collected and the extraction was repeated by adding an equal volume of distilled water to the lower phase. After subsequent incubation at 70°C, cooling, and centrifugation, the two upper phases were mixed and dialysed against tap water until the phenol odour disappeared. After freeze-drying the dialysed fractions, LPS was dissolved in phosphate-buffered saline (pH 7.2) at a concentration of 1 mg/ml. For fatty acid analysis by GC/MS, a five-fold (v/v) excess of acetone was added to an aliquot of the isolated LPS, after which the solution was dried at 60°C under a nitrogen flow. Subsequently, 10 ug of C12:0(2OH) (1 mg/ml in ethanoft was added as an internal<br>
standard, as well as 100 fj.1 of acetylchloride/ethanol 1:9 (v/v), after which the samples were derivatized for 1 h at 90°C. After cooling, the reaction was stopped by adding 200 ul of 1 M K2HPO4 (pH 8.0), followed by extraction of the acyl-ethyl esters with 200 fal ethyl acetate. A 1-ul volume of the upper phase was used for analysis by GC/MS on a Finnigan MAT SSQ in the electron-impact mode.<br>
Biological activity ofLPS<br>
IL-6 and IL-10 induction by wild type and PagL-modified B. pertussis Tohama LPS was tested with the human macrophage cell line MM6 (49). MM6 cells were seeded in microtiter plates (2.105/well) in 400 jxl of IMDM (Gibco BRL) supplemented with 10% fetal calf serum (Gibco BRL) and stimulated with 200 il of serial dilutions of the LPS stock solution, for 16-18 h at 37°C in a humid atmosphere containing 5% CC)2. IL-6 and IL-10 levels in the culture supernatants were quantified with an ELISA against human IL-6 or IL-10 according to the instructions of the manufacturer (PeliPair™ reagent set, Sanquin Reagents, Amsterdam, The Netherlands).<br>
Isolation of Cell Envelopes<br>
Cells were harvested by centrifugation for 10 min at 1,500x g, and washed once in 50 ml of cold 0.9% sodium chloride solution. The cell pellets were frozen for at least 15 min at -80°C, and then suspended in 20 ml of 3 mM EDTA, 10 mM Tris-HCl (pH 8.0) containing Complete Protease inhibitor cocktail (Roche). The cells were disrupted by sonication, after which unbroken cells were removed by centrifugation for 10 min at 1,500x g. The cell envelopes were pelleted from the supernatant by centrifugation for 1.5 h at 150,000x g and resuspended in 2 mM Tris-HCl (pH 7.4). The cell envelopes were stored at -80°C in aliquots.<br>
Isolation of inclusion bodies<br>
For inclusion body isolation, PagL(pa)(-) was expressed in E. coli BL21 Star™ (DE3) from pPagL(pa)(-) (Table 1). A Two-liter culture was grown at 37°C in LB medium supplemented with ampicillin till an ODeoo between 0.4 and 0.6. Then, 1 mM IPTG (end concentration) was added to the culture to induce expression of the recombinant gene, after which the culture was incubated further at 37°C, while shaking.<br>
After approximately 4 hours, cells were harvested by centrifugation (15 min at 4,000 rpm (4°C)). Harvested cells were washed once in 400 ml 0.9% NaCl and then resuspended in 80 ml TE 50:40 (50 mM Tris-HCl (pH 8.0), 40 mM EDTA). Sucrose (0.25 g/ml (end concentration)) and lysozyme (0.2 mg/ml (end concentration)) were added, after which the suspension was incubated for 30 min at RT, while shaking. The suspension was sonicated three times on ice (1.5 min, with 2 min pauses in-between) using a Branson 250 Sonfier with macrotip (output 9, duty cycle 50%). Following sonication, 0.13% (w/v) Brij-35P (Fluka) was added, and the suspension was sonicated for an additional 2 min. Dense material (inclusion bodies) was collected by centrifugation for 2 hrs at 4,000 rpm (4°C), after which the pellet was washed once in 40 ml TE 50:40, followed by another washing step using 40 ml 10 mM Tris-HCl (pH 8.3). The obtained inclusion bodies were solubilized in 8 M urea supplemented with 10 mM glycine (pH 8.3) and precipitated with TCA. Finally, the obtained proteins were solubilized in 8 M urea supplemented with 10 mM glycine (pH 8.3) at a protein concentration of 10 mg/ml. This mixture was centrifugated for 2 hrs at 13,000 rpm to remove residual insoluble material and membranes.<br>
Refolding and purification of PagL(pa)(-)<br>
PagL(pa)(-) was refolded in vitro by two-fold dilution of the 10 mg/ml protein solution (see above) in 10% (w/v) lauryldimethylamine oxide (LDAO) and subsequent sonication for 10 min. Refolded PagL(pa)(-) was purified by Fast Protein Liquid Chromatography (FPLC) using a 1 ml MonoQ (Amersham Biosiences) ion-exchange column. The protein solution was diluted 4 times in buffer A (20 mM Tris-HCl (pH 8.0), 0.08 % (w/v) CioEs). The solution was loaded onto the column, which was pre-equilibrated with buffer A, and washed once with buffer A, and the proteins were eluted with a linear gradient of 0-1 M NaCl in buffer A. Fractions were analysed by SDS-PAGE for the presence of the refolded PagL(pa)(-) protein. Those containing the protein were pooled and concentrated to a protein concentration of 10 mg/ml using Centricon concentrators with a molecular mass cut-off of 3 kDa (Amicon). The protein solution was then dialyzed three times overnight against 10 ml 2 mM Tris-HCl (pH 8.0), 0.06% (w/v) CioEs using a membrane with a molecular mass cut-off of 3.5 kDa.<br>
In vitro Modification Assay<br>
Refolded PagL(pa)(-) (10 mg/ml) or cell envelopes isolated from E. coli BL21 Star™ (DE3) containing the empty vector pET-1 la or the pPagL plasmids were diluted 10 fold in double distilled water. 4 ul of the diluted refolded protein or cell envelope solution was incubated in 50 mM Hepes (pH 8.0), 0.1% Triton X-100, 0.5 M NaCl, and 0.75 nmol N. meningitidis L3-LPS in a final volume of 10 ul at 37°C for 16 h. To test whether the reaction was dependent on divalent cations, 5 mM EDTA was added into the reaction with the refoled PagL(pa)(-). The reactions were terminated by boiling in sample buffer (28), after which the samples were treated with 0.5 mg/ml proteinase K for 1 hour at 55°C, followed by 10 min incubation at 95°C. The samples were diluted 25 fold by adding sample buffer, after which 2 jl of the samples were analysed by Tricine-SDS-PAGE (see above).<br>
Isolation of LPS and analysis by Electrospray lonisation-Mass Spectrometry (ES1-MS)<br>
LPS was isolated using the hot phenol/water extraction method (Westphal and Jann, Methods Carbohydr. Chem. 5; 83-91,1965) with slight modifications. In short, bacteria were grown in THUS medium in the presence of 1 mM IPTG (end concentration) for 64 h. Cells were harvested by centrifugation and resuspended in 40 mM sodium phosphate buffer (pH 7.0) containing 5 mM EDTA. The cells were treated overnight with lysozyme at 4°C, after which an equal volume of phenol was added. The suspension was heated to 70°C, incubated for 30 min while shaking, and subsequently cooled to 10°C, after which phases were separated by centrifugation for 10 min at 8,000 x g. The upper phase was collected and the extraction was repeated after adding an equal volume of distilled water to the lower phase. The two upper phases were combined, dialysed against tap water until the phenol odour disappeared, freeze-dried, and subsequently taken up in distilled water. The LPS was subsequently pelleted by centrifugation for 3 h at 150,000 x g and dissolved in distilled water, after which the LPS concentration was determined by analysing the 3-hydroxytetradecanoic acid content, using a 6890 Agilent gas chromatograph, as described (Welch, Clin. Microbiol. Rev. 1991). For ESI-MS, a 200 jil aliquot of isolated LPS (50 nmol/ml) was freeze-dried and taken up in 0.1 ml 2% acetic acid. The mixture was heated for 2 h at 95°C to hydrolyse the LPS and release the lipid A moiety. Subsequently, the mixture was cooled to room temperature and centrifuged for 10 min at 16,100 x g. The pellet<br>
was washed twice in 0.1 ml double- distilled water, taken up in 0.1 ml double-distilled water, and 0.3 ml chloroform/methanol (2:1, v/v) was added. After vigorous vortexing, phases were separated by centrifugation for 10 min at 16,100 x g. The upper phase was then used for structural analysis of purified lipid A by nanoelectrospray tandem MS on a Finnigan LCQ in the negative ion mode (Wilm and Mann, Anal. Chem. 1996).<br>
Example 1: Identification ofPagL Homologs in various Gram-negative Bacteria<br>
The 187-amino acid sequence of the 5*. Typhimurium PagL precursor protein (GenBank Accession Number AAL21147, SEQ ID No. 17) was used as a lead to identify putative PagL homologs in other Gram-negative bacteria, by searching all completed and unfinished genomes of Gram-negative bacteria present in the NCBI database (http://www.ncbi.nlm.nih.gov/sutils/genom_table.cgi). BLAST search (34) revealed the presence of putative homologs in the Bordetella spp. B. pertussis, B. bronchiseptica, and B. parapertussis (Fig. 2). The PagL homologs of B. bronchiseptica and B. parapertussis are two mutually identical 178-amino acid polypeptides (Fig. 2) with, as predicted by the signalP server (35), a 25-amino acid N-terminal signal peptide. A gene for a PagL homolog was also found in the genome of the B. pertussis Tohama I strain (36), but this open reading frame (ORF) was disrupted by a frame shift (SEQ ID No. 4), which could be restored as in SEQ ID No. 5 to encode a protein as in SEQ ID No.l. Nucleotide sequencing of the PagL ORFs from B. pertussis strains B509 and B134 also showed the presence of the same frame shift2, which indicates that disruption of the PagL ORF might be a common feature in B. pertussis strains. By using the newly identified B. bronchiseptica PagL homolog as a probe for further BLAST analysis, additional putative pagL homologs could be identified in the genomes of P. aeruginosa (SEQ ID No 6, 30% identity), Pseudomonas fluorescens (SEQ ID No 7, 29% identity), Pseudomonas syringae (SEQ ID No 8, 31% identity), Pseudomonas putida, 2x (SEQ ID No 9 + 10, 32/33%), Ralstonia metallidurans (SEQ ID No 15, 28%), Ralstonia solanacearum (SEQ ID No 16, 29%), Burkholderia mallei (SEQ ID No 12, 28%), Burkholderia pseudomallei (SEQ ID No 13, 28%), Burkholderia fungorum (SEQ ID No 11, 29%), and Azotobacter vinelandii (SEQ ID No 14, 27%) Alignments are shown in Fig. 2. Together, all PagL homologs exhibited a low overall mutual sequence identity, albeit higher than with S. typhimurium (24% identity), but contained a clear homologous domain near the C terminus. Our finding of this<br>
conserved motif allows identification of PagL homologs in other (bacterial) species and allows the use of a suitable PagL homolog for any host bacterium and/or any LPS to be 3-0-deacylated.<br>
Example 2: Cloning ofpagL and Heterologous Expression in E. coli<br>
To verify their putative lipid A-deacylase activity, we cloned the pagL homologs of P. aeruginosa (pagZ,(pa)) and B. bronchiseptica (pagL)). We included in these studies pagL(si) as a reference. These pagL genes were amplified from the chromosomes by PCR and eventually cloned in pET-lla under the control of the T7 promoter, resulting in plasmids, pPagL(pa), pPagL(Bb), and pPagL(st).<br>
To investigate expression and membrane localization of PagL in E. coli, E. coli BL21 Star™ (DE3) containing the empty vector pET-lla or the pPagL plasmids were grown overnight in LB, after which cell envelopes were isolated. Analysis by SDS-PAGE revealed the presence of prominent additional bands with Mrs of 15000-18000 in the cell envelopes of the cells expressing PagL (Fig. 3). This was consistent with the expected molecular masses of the mature PagL proteins, i.e. PagL(pa) 16.1 kDa, PagL(Bb) 17.2 kDa, and PagL(st) 18.2 kDa. To identify the additional protein bands, they were subjected to microsequencing. The sequences of the first 5 amino acid residues of PagL(pa), PagL(Bb), and PagL(St) were ADVSA, QPTQG, and NDNVF, respectively, indicating that cleavage of the signal peptide by leader peptidase occurs between amino acid residues 23 and 24 (AQA-ADV), 25 and 26 (AQA-QPT), and between 20 and 21 (CSA-NDN), respectively. Particularly in the case of expression of PagL(Bb), an additional band with a higher Mr was visible on the gel (Fig. 2). The N-terminal sequence of this band, MQFLK, corresponded with that of the precursor<br>
Example 3 : In vivo Modification ofE. coli LPS by PagL<br>
To study whether the cloned PagL homologs were active on E. coli LPS, IPTG was added to exponentially growing E. coli BL21 Star™ (DE3) cells containing the empty vector pET-lla or the pPagL plasmids, and after various incubation periods, samples equivalent to one OD600 unit were collected and their LPS content was analyzed by Tricine-SDS-PAGE. In accordance with the expected hydrolysis of the R-3-hydroxymyristate at the 3 position of lipid A, expression of any of the three pagL homologs converted the LPS into a form with a higher electrophoretic mobility (Fig. 4).<br>
The   conversion   was   almost   complete    within 75 min after PagL(pa) or PagL(Bb) were induced, but took somewhat longer in the case of PagL(st).<br>
Structural Analysis of PagL-Modified LPS: to determine its fatty acid content, LPS was isolated from bacteria that were grown in the presence of 10 mM MgCb to suppress PhoP/PhoQ-regulated modifications of lipid A and analyzed by GC/MS. The C14:0/C14:0(3OH) ratio in the PagL-modified LPS samples was increased as compared with that in the wild-type LPS (Fig. 5), consistent with the expected removal of a C14-3OH from lipid A. To confirm these data, the lipid A moieties were isolated and analyzed by ESI-MS in the positive ion mode, which revealed the presence of four major lipid A species in wild-type LPS (Fig. 6A). The peak at m/z 1797 represents the characteristic hexa-acylated to-phosphate species that is typically found in E. coli, whereas the peak at m/z 1928 corresponds to a hexa-acylated to-phosphate species substituted with an L-Ara4N moiety. The two remaining peaks at m/z 1716 and m/z 1847 most likely represent fragment ions of the two former species missing a phosphate group. Upon expression of PagL(St) (Fig. 6B), PagL(Pa) (Fig. 6C), or PagL(Bb) (Fig. 6D), the major lipid A species were present at m/z 1622 and m/z 1490, which correspond to the loss of one p-hydroxymyristate residue and one phosphate group from the major species at m/z 1928 and m/z 1797 present in the empty vector control, respectively. Also here, the loss of the phosphate group is probably an artefact of the ionisation procedure. Based upon the GC/MS and ESI-MS data, it can be concluded that the identified PagL homologs of P. aeruginosa and B. bronchiseptica, like that of S. Typhimurium, are active lipid A deacylases. Furthermore, the data suggest that the deacylation is not dependent upon the absence or presence of an L-Ara4N moiety, since both species were deacylated efficiently.<br>
Example 4: Subsequent In Vivo Modification ofPagL-deacylatedLPS<br>
In the course of these experiments, it was observed that after prolonged PagL expression, PagL-modified LPS was no longer detectable on Tricine-SDS-PAGE gels, and that the LPS migrated again at the position of wild-type LPS, as illustrated for the strain expressing PagL(Bb) (Fig. 7A). The PagL protein was still abundantly present at this time point, as revealed on SDS-PAGE gels (data not shown). Furthermore, analysis by GC/MS revealed that the C14:0/C14:0(3OH) ratio was not decreased again for the LPS isolated after 5 h induction of PagL(Bb) (Fig. 7B). Thus, the secondary modification<br>
bserved on the Tricine-SDS-PAGE gel (Fig. 7A) was not the consequence of restoration of the PagL modification, but the result of (an) additional modification(s) that restored the electrophoretic mobility to that of wild-type LPS. Therefore, other fatty acid ratios were compared. A striking increase in the C16:0/C14:0 ratio was found in the LPS of cells induced 5 h for PagL production (Fig. 7C), suggesting that the PagL-deacylated LPS was subsequently palmitoylated.<br>
A protein that adds palmitate to lipid A is the outer membrane protein PagP (19) (Fig. 1). Therefore, we hypothesized that the secondary modification of PagL-modified LPS might have been the result of endogenous PagP activity. To investigate this possibility, we transformed wild-type E. coli BL21 Star™ (DE3) and its pagP mutant derivative JG101 with the pPagL(pa) plasmid. The secondary modification of PagL-modified LPS was again observed in the case of the wild-type strain, but not in that of the mutant strain (Fig. 7D). This result strongly suggests that the secondary modification of PagL-modified LPS (Fig. 7A) was indeed the consequence of endogenous PagP activity.<br>
Example 5: Identification of PagL Active-Site Residues<br>
The mutual sequence identity between the identified PagL homologs is very low (Fig. 2). Among the few totally conserved residues are a histidine and a serine, which, we hypothesize, might be part of a 'classical' Asp/Glu-His-Ser catalytic triad of serine hydrolases. These putative active-site residues are located at the lipid-exposed side near the top of a p-strand in a topology model we propose (Fig. 8). Interestingly, in the outer membrane phospholipase A, the active-site His and Ser are located in a similar position (37). To test whether these residues, located at positions 149 and 151 of the PagL(pa) precursor protein, respectively, are indeed important for catalytic activity, they were substituted by alanine or asparagine, and by alanine or cysteine, respectively. As a control, the same substitutions were made for a non-conserved histidine and serine residue, located at positions 81 and 84 of the PagL(Pa) precursor, respectively. The protein and LPS profiles of E. coli BL21 Star™ (DE3) cells carrying the relevant plasmids and induced for 75 min with IPTG were analyzed by immunoblotting (Fig. 9A) and Tricine-SDS-PAGE (Fig. 9B), respectively. Whereas substitution of the non-conserved His81 and Ser84 did not affect LPS deacylation, deacylation of LPS was no longer observed when the conserved His 149 and SerlSl were substituted (Fig. 9B),<br>
even though the expression of these mutant proteins was not affected (Fig. 9A). These results strongly support the hypothesis that the conserved histidine at position 149 and serine at position 151 of the precursor PagL(pa) protein are active-site residues and that PagL mechanistically functions as a serine hydrolase.<br>
Example 6: Cloning ofpagL(Bt,) and Heterologous Expression in B. pertussis<br>
To modify B. pertussis LPS in vivo, we cloned the pagL gene of B. bronchiseptica (pagL(wj). The pagL gene was amplified from the chromosome by PCR and eventually cloned in pMMB67EH under the control of the Tac promoter, resulting in plasmid, pMMB67EH-PagL(Bb), which was transferred to B. pertussis strain Tohama by conjugation.<br>
To address the modification of B. pertussis LPS in vivo, wild-type B. pertussis strain Tohama, or B. pertussis strain Tohama containing the pMMB67EH-PagL(Bb) plasmid were grown in Thijs medium supplemented with 1 mM IPTG (end concentration). LPS was isolated by the hot phenol-water extraction method and analysed by Tricine-SDS-PAGE (Fig. 10A) and GC-MS (Fig. 10B). Analysis on Tricine-SDS-PAGE gel showed that the LPS isolated from the PagL(Bb) expressing strain migrated slightly faster as compared to the wild type B. pertussis LPS. Strikingly, the LPS isolated from the PagL(Bb) expressing strain showed two distinct LPS populations. One population migrating around the height of the wild type Tohama LPS, and one population migrating faster. This latter LPS population can also be seen in the wild type LPS preparation; however the abundance of it is much lower. To verify that the LPS from the PagL expressing strain was indeed deacylated at its 3 position, it was analysed by GC-MS. The C14:0(3OH)/C10:0(3OH) ratio in the PagL-modified LPS sample was increased as compared with that in the wild-type LPS (Fig. 10B), consistent with the expected removal of a C10-3OH from the 3 position of B. pertussis lipid A.<br>
Example 7: Biological activity of PagL-modified LPS<br>
To assess the endotoxic activity of the PagL-modified and wild-type B. pertussis LPS, their ability to stimulate the production of IL-6 and IL-10 in the human macrophage cell line MM6 was measured. As can be seen in figure 2, for wild-type LPS, the production of both IL-6 (Fig. 11A) and IL-10 (Fig. 1 IB) by the MM6 cells is increased as compared to when the cells were stimulated with an equal amount of<br>
PagL-modified   LPS.   Thus,   it   can   be    concluded that the in vivo deacylation of B. pertussis LPS by PagL results in a reduction in endotoxic activity of this LPS.<br>
Example 8: Cloning, Expression, Purification, and Refolding of PagL (pa)<br>
The pagL gene from P. aeruginosa PAO25 without its signal sequence-encoding part was cloned into pET-lla, resulting in plasmid pPagL(pa)(-). To obtain inclusion bodies, PagL without its signal sequence was expressed in E. coli BL21 Star™ (DE3). Inclusion bodies were isolated and solubilized in urea, after which the protein was refolded by diluting two-fold in 10% lauryldimethylamine oxide (LDAO) and further purified by Fast Protein Liquid Chromatography (FPLC). Correct refolding was confirmed by SDS-PAGE (Fig. 12) and circular dichroism (CD) measurements (data not shown). On SDS-PAGE gel, the refolded protein had a lower electrophoretic mobility as compared to the denatured form, whereas CD measurements showed that the refolded protein predominantly had a p-sheet conformation.<br>
Example 9: In vitro LPS Modification by Membrane-Localized and Refolded PagL<br>
To test whether membrane-localized or in vitro refolded PagL was capable of modifying externally added LPS in vitro, we incubated refolded PagL(pa)(-), or isolated cell envelopes from E. coli BL21 Star™ (DE3) containing the empty vector pET-1 la, or the pPagL plasmids, together with purified LPS of N. meningitidis. Modification of LPS was assessed by Tricine-SDS-PAGE (Fig. 13). In accordance with the expected hydrolysis of the .ft-3-hydroxymyristate at the 3 position of lipid A, LPS was converted into a form with a higher electrophoretic mobility when membrane-localized PagL (Fig 13A) or refolded PagL(pa)(-) (Fig 13B) was present. The reaction with the refolded PagL(pa)(-) was independent on the presence of divalent cations, as deacylation of LPS was still observed in the presence of 5 mM EDTA (Fig. 13B).<br>
Example 10: Altered lipid A structure after expression ofPagP and PagL in B.pertussis To express PagP and PagL in B. pertussis strain Tohama, the pagL gene of B. bronchiseptica (pagL(Bvj) and the pagP gene of B. pertussis (pagPp)) were expressed from the broad-host range low-copy number expression vector pMMB67EH. As a control, a strain expressing the pagP gene of E. coli (pagP(EC)) was also constructed. LPS was isolated from wild-type, PagP-expressing, or PagL-expressing B. pertussis<br>
strain Tohama and analysed by Tricine- SDS-PAGE. LPS isolated from the PagL(Bb)-expressing strain appeared unaffected on the gel, whereas that from the PagP-expressing appeared potentially modified, since a band with a lower electrophoretic mobility than that of wild-type B. pertussis LPS was detected (Fig. 14). Furthermore, as compared to the PagP(Bp)-expressing strain, the modification-efficiency appeared higher in the PagP(EC)-expressing strain (Fig. 14). To evaluate the possible LPS modifications in further detail, the lipid A moieties of the strains were analysed by ESI-MS in the negative-ion mode. This analysis revealed the presence of four major lipid A species in wild-type LPS (Fig. ISA). The peak at m/z 1557 represents the characteristic penta-acylated to-phosphate species that is typically found in B. pertussis (Caroff et al., Microbes. Infect., 1994), whereas the peak at m/z 1477 corresponds to a penta-acylated /no«o-phosphate species. The two remaining peaks at m/z 1307 and 1251 represent deacylated lipid A species of the molecular ion at m/z 1477, which miss the primary 3-hydroxydecanoic acid residue at the 3 position or a primary 3-hydroxytetradecanoic acid residue (either at the 2 or the 3' position), respectively. These results indicate a high heterogeneity among the lipid A species in wild-type B. pertussis, which was apparently not resolved in the gel analysis (Fig. 14). Interestingly, calculation of the relative amounts of the individual lipid A species from the corresponding peak heights revealed that in wild-type B. pertussis LPS, a large quantity of lipid A species (-50%) consists of tetra-acylated forms. Furthermore, the large majority of lipid A species are /Motto-phosphate forms (-80%). To exclude the possibility that the high abundancy of under-acylated and hypo-phosphorylated lipid A species was an artefact of the hydrolysation procedure used to isolate lipid A, we tested whether shorter or longer periods of hydrolysation (varying between 1 and 4 h) influenced the relative abundance of the lipid A species, which was, however, not the case (data not shown). Furthermore, the total phosphate content of a solution with a known concentration of purified wild-type B. pertussis LPS was determined. Consistent with the high prevalence of mono-phosphate lipid A species detected by ESI-MS, only slightly more than half of the phosphate content expected, when LPS would have been fully phosphorylated, was detected (data not shown).<br>
Upon expression of PagL(Bb) (Fig. 15B), three lipid A species, at m/z 1081, 1307, and 1387, respectively, were present. The major peak at m/z 1307 corresponds to the 7?70«o-phosphate deacylated form missing the 3-hydroxydecanoic acid residue at the<br>
3 position, whereas the peak at m/z 1387 corresponds to the Z?w-phosphorylated form of the molecular ion at m/z 1307. The peak at m/z 1081 corresponds to a mono-phosphate form missing both a 3-hydroxydecanoic and a 3-hydroxytetradecanoic acid residue. The relative content of lipid A species that miss the 3-hydroxydecanoic acid residue at their 3 position was increased from about 37 percent in wild-type B. pertussis LPS to more than 92 percent in the strain expressing PagL(Bb)- Thus, even though the electrophoretic mobility of the LPS was not affected (Fig. 14, lane 2), the pagL(Bb)-encoded lipid A 3-O-deacylase was active in B. pertussis.<br>
Upon expression of PagP(Ec) (Fig. 15C) and PagPp) (Fig. 15D), several new lipid A species were detected (Table III). The peaks at m/z 1320, 1490, 1545, 1625, 1715, and 1796 correspond to the expected PagP-mediated palmitoylation of the molecular ions present at m/z 1081, 1251, 1307, 1387, 1477, and 1557, respectively. The difference in modification-efficiency between E. coli and B. pertussis PagP, which was seen after analysis by Tricine-SDS-PAGE (Fig. 14), was also revealed in the mass spectrometrical analysis. In the strain expressing E. coli PagP, -47% of the total lipid A population was palmitoylated, in contrast to only -9% in the strain expressing PagP(Bp). Interestingly, in the strain expressing PagP(ep), in contrast to that expressing PagP(£C), lipid A species missing a 3-hydroxytetradecanoic acid residue were not found to be palmitoylated. A possible explanation for this discrepancy is the difference in specificity of the two PagP enzymes. Whereas E. coli PagP adds an acyl chain at the 2 position of lipid A, B. pertussis PagP adds a palmitate at the 3' position (Bishop et al, EMBO J., 2000; Preston et al, Mol. Microbiol., 2003). Thus, the complete absence of palmitoylated lipid A species that miss one 3-hydroxytetradecanoic acid residue in the strain expressing B. pertussis PagP suggests that the lipid A molecules missing a 3-hydroxytetradecanoic acid residue miss it specifically at their 3' position. This could then partially explain the difference in modification efficiency that was observed between the two PagP enzymes, as the substrate pool for E. coli PagP would be larger than that for B. pertussis PagP. Furthermore, this hypothesis is consistent with the presence of hypo-acylated lipid A species in vivo.<br>
Table III: Relative abundance of lipid A molecular ions as determined by ESI-MS<br>
(Table Removed)<br>
References<br>
1.	Raetz, C. R. H., and Whitfield, C. (2002) Annu. Rev. Biochem. 71, 635-700<br>
2.	Loppnow, H.,  Brade, H., Durrbaum, I., Dinarello, C. A., Kusumoto, S.,<br>
Rietschel, E. T., and Flad, H. D. (1989) J, Immunol. 142, 3229-3238<br>
3.	Steeghs, L., Berns, M., ten Hove, J., de Jong, A., Roholl, P., van Alphen, L.,<br>
Tommassen, J., and van der Ley, P. (2002) Cell. Microbiol. 4, 599-611<br>
4.	Nikaido, H., and Vaara, M. (1987) in Escherichia coli and Salmonella: Cellular<br>
and Molecular Biology (Neidhardt, F. C., ed) Vol.  1, pp. 7-22, American<br>
Society for Microbiology, Washington, D. C.<br>
5.	Caroff, M., Karibian, D., Cavaillon, J-M, and Haeffner-Cavaillon, N. (2002)<br>
Microbes Infect. 4, 915-926<br>
6.	Guo, L., Lim, K. B., Gunn, J. S., Bainbridge, B., Darveau, R. P., Hackett, M.,<br>
and Miller, S. I. (1997) Science 276, 250-253<br>
7.	Guo, L., Lim, K. B., Poduje, C. M., Daniel, M., Gunn, J. S., Hackett, M., and<br>
Miller, S. I. (1998) Cell 95,189-198<br>
8.	Gunn, J. S., Belden, W. J., Miller, S. I. (1998) Microb. Pathog. 25, 77-90<br>
9.	Gunn, J. S., Lim, K. B., Krueger, J., Kim, K., Guo, L., Hackett, M., and Miller,<br>
S. I. (1998) Mo/. Microbiol. 27, 1171-1182<br>
10.	Gunn, J. S., Ryan, S. S., Van Velkinburgh, J. C., Ernst, R. K., and Miller, S. I.<br>
(2000) Infect. Immun. 68, 6139-6146<br>
11.	Miller, S. I., Kukral, A. M., and Mekalanos, J. J. (1989) Proc. Natl. Acad. Sci.<br>
U.S.A. 86, 5054-5058<br>
12.	Gunn, J. S., and Miller, S. I. (1996) J. Bacterial. 178, 6857-6864<br>
13.	Ernst, R. K., Guina, T., and Miller, S. I. (1999) J. Infect. Dis. 179, Suppl. 2,<br>
326-330<br>
14.	Ernst, R. K., Yi, E. C., Guo, L., Lim, K. B., Burns, J. L., Hackett, M., and<br>
Miller, S. I. (1999) Science 286, 1561-1565<br>
15.	Trent, M. S., Ribeiro, A. A., Lin, S., Cotter, R. J., and Raetz, C. R. H. (2001) J.<br>
Biol. Chem. 276,43122-43131<br>
16.	Lee, H., Hsu, F. F., Turk, J., and Groisman, E. A. (2004) J. Bacterial. 186,<br>
4124-4133<br>
17.	Gibbons, H. S., Lin, S., Cotter, R.    J., and Raetz C. R. H. (2000) J. Biol.<br>
Chem. 275, 32940-32949<br>
18.	Karbarz, M. J., Kalb, S. R., Cotter, R. J., and Raetz,  C. R. H. (2003) J. Biol.<br>
Chem. 278, 39269-39279<br>
19.	Bishop, R. E., Gibbons, H. S., Guina, T., Trent, M. S., Miller, S. I., and Raetz,<br>
C. R. H. (2000) EMBOJ. 19, 5071-5080<br>
20.	Tanamoto, K., and Azumi, S. (2000) J. Immunol. 164, 3149-3156<br>
21.	Robey, M., O'Connell, W., and Cianciotto, N. P. (2001) Infect. Immun. 69,<br>
4276-4286<br>
22.	Trent, M. S., Pabich, W., Raetz, C. R. H., and Miller, S. I. (2001) J. Biol. Chem.<br>
276, 9083-9092<br>
23.	Bhat, U. R., Forsberg, L. S., and Carlson, R. W. (1994) J. Biol Chem. 269,<br>
14402-14410<br>
24.	Moran, A. P., Lindner, B., and Walsh, E. J. (1997) J. Bacterial. 179, 6453-6463<br>
25.	Kumada,  H.,  Haishima,  Y.,  Umemoto,  T.,  and Tanamoto,  K.  (1995) J.<br>
Bacterial. 177,2098-2106<br>
26.	Tommassen, J., van Tol, H., and Lugtenberg, B. (1983) EMBOJ. 2, 1275-1279<br>
27.	Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) in Molecular Cloning: A<br>
Laboratory Manual.  Cold Spring Harbour Laboratory Press, Cold Spring<br>
Harbour (NY)<br>
28.	Laemmli, U. K. (1970) Nature 227, 680-685<br>
29.	Dekker, N., Merck, K., Tommassen, J., and Verheij, H. M. (1995) Eur. J.<br>
Biochem. 232, 214-219<br>
30.	Lesse, A. J., Campagnari, A. A., Bittner, W. E., and Apicella, M. A. (1990) J.<br>
Immunol Methods 126, 109-117<br>
31.	Tsai, C. M., andFrasch, C. E. (\9Vl)Anal. Biochem. 119, 115-119<br>
32.	Westphal, O., and Jann, J.K. (1965) Methods Carbohydr. Chem. 5, 83-91<br>
33.	Wilm, M., and Mann, M. (1996) Anal. Chem. 68, 1-8<br>
34.	Altschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990) J.<br>
Mol.Biol 215,403-410<br>
35.	Nielsen, H., Brunak, S., and von Heijne, G. (1999) Protein Eng. 12, 3-9<br>
36.	Parkhill, J., Sebaihia, M., Preston, A., Murphy, L. D., Thomson, N., Harris, D.<br>
E., Holden, M. T., Churcher, C. M., Bentley, S. D., Mungall, K. L., Cerdeno-<br>
Tarraga, A. M, Temple, L., James, K., Harris, B., Quail, M. A., Achtman, M., Atkin, R., Baker, S., Basham, D., Bason, N., Cherevach, I., Chillingworth, T., Collins, M., Cronin, A., Davis, P., Doggett, J., Feltwell, T., Goble, A., Hamlin, N., Hauser, H., Holroyd, S., Jagels, K., Leather, S., Moule, S., Norberczak, H., O'Neil, S., Ormond, D., Price, C., Rabbinowitsch, E., Rutter, S., Sanders, M., Saunders, D., Seeger, K., Sharp, S., Simmonds, M., Skelton, J., Squares, R., Squares, S., Stevens, K., Unwin, L., Whitehead, S., Barrell, B. G., and Maskell, D. J. (2003) Nat. Genet. 35, 32-40<br>
37.	Snijder, H. J., Ubarretxena-Belandia, I., Blaauw, M., Kalk, K. H., Verheij, H.<br>
M., Egmond, M. R., Dekker, N., and Dijkstra, B. W. (1999) Nature 401, 717-<br>
721<br>
38.	McClelland, M., Sanderson, K. E., Spieth, J., Clifton, S. W., Latreille, P.,<br>
Courtney, L., Porwollik, S., Ali, J., Dante, M., Du, F., Hou, S., Layman, D.,<br>
Leonard, S., Nguyen, C., Scott, K., Holmes, A., Grewal, N., Mulvaney, E.,<br>
Ryan, E., Sun, H., Florea, L., Miller, W., Stoneking, T., Nhan, M., Waterston,<br>
R., and Wilson, R. K. (2001) Nature 413, 852-856<br>
39.	Basu, S. S., White, K. A., Que, N. L., and Raetz, C. R. H. (1999) J. Biol. Chem.<br>
274, 11150-11158<br>
40.	Kulshin, V. A., Zahringer, U., Lindner, B., Jager, K. E., Dmitriev, B. A., and<br>
Rietschel, E. T. (1991) Eur. J. Biochem. 198, 697-704<br>
41.	Hwang, P. M., Choy, W. Y., Lo, E. I., Chen, L., Forman-Kay, J. D., Raetz, C.<br>
R., Prive, G. G., Bishop, R. E., and Kay, L. E. (2002) Proc. Natl. Acad. Sci.<br>
USA. 99,13560-13565<br>
42.	Kol, M. A., van Dalen, A., de Kroon, A. L, and de Kruijff, B. (2003; J. Biol.<br>
Chem. 278, 24586-24593<br>
43.	von Heijne, G. (1983) Eur. J. Biochem. 133,17-21<br>
44.	Tommassen, J. (1988) in Membrane Biogenesis (Op den Kamp, J.A.F., ed)<br>
NATO ASI series, Vol. H16, pp. 351-373. Springer Verlag, Berlin, Heidelberg,<br>
New York.<br>
45.	Haas, D., and Holloway, B.W. (1976) Mol. Gen. Genet. 144, 243-251<br>
46.	Pace, J., Hayman, M. J., and Galan, J. E. (1993) Cell 72, 505-514<br>
47.	Hanahan, D. (1983) J. Mol. Biol. 166, 557-580<br>
48.	Thalen, M., van den Ussel, J., Jiskoot, W., Zomer, B., Roholl, P., de Gooijer,<br>
40 C, Beuvery, C., and Trampen, J.    (\999)J.Biotechnol. 75, 147-159.<br>
49.	Ziegler-Heitbrock, H. W., Thiel, E., Futterer, A., Herzog, V., Wirtz, A., and<br>
Riethmuller, G. (1988) J. Cancer 41, 456-461<br>
50.	Simon, R., Priefer, U., and Puhler, A. A. (1983) Bio/Technol. 1, 784-791<br>
51.	Furste, J. P., Pansegrau, W., Frank, R., Blocker, H., Scholz, P., Bagdasarian,<br>
M, and Lanka, E. (1986) Gene 48,119-131<br>
52.	Stibitz, S., Black, W., and Falkow, S. (1986) Gene 50, 133-140<br><br><br><br><br><br><br><br>
We claim<br>
1.	A gram negative bacterium for producing 3-O-deacylated lipopolysaccharide(LPS) comprising a nucleic acid sequence encoding a polypeptide with minimum 25% amino acid sequence identity with SEQ ID NO: 1, wherein the nucleic acid sequence confers an increase in lipid A 3-O-deacylase activity to the bacterium.<br>
2.	The gram negative bacterium as claimed in claim 1, wherein the bacterium is selected from a group comprising of Bordetella pertussis, Bordetella parapertussis, Bordetella bronchiseptica, Neisseria meningitis, Neisseria gonorroheae and Neisseria lactamica.<br>
3.	The gram negative bacterium as claimed in claim 2, wherein the bacterium is Bordetella pertussis.<br>
4.	A method for producing 3-Odeacylated lipopolysaccharide (LPS) of claim 1, comprising the steps of culturing the bacterium as claimed in any one of claims 1 to 3 under conditions conducive to synthesis of the 3-O-deacylated LPS, and recovering the 3-O-deacylated LPS.<br>
5.	The method as claimed in claim 4, optionally comprising the steps of bringing the LPS in crude form in contact with a nucleic acid sequence encoding the polypeptide as described in claim 1 under conditions conducive the synthesis of the 3-O-deacylated LPS.<br>
6.	A composition comprising the 3-O-deacylated LPS as obtained in any one of claims 4 and 5 along with a pharmaceutically acceptable excipients and/or adjuvants.<br>
7.	The composition as claimed in claim 6, in the form of an acellular vaccine.<br>
8.	The gram negative bacterium as claimed in any one of the claims 1 to 3, in the form of a whole cell vaccine.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUFic3RyYWN0LSgwNS0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Abstract-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1ERUxOUC0yMDA3LUFzc2lnbm1lbnQtKDI5LTA3LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4857-DELNP-2007-Assignment-(29-07-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUNsYWltcy0oMDUtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Claims-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Correspondence Others-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Correspondence Others-(13-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTUtMDktMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4857-DELNP-2007-Correspondence-Others-(15-09-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjQtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Correspondence-Others-(24-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMDctMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4857-DELNP-2007-Correspondence-Others-(29-07-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMzEtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Correspondence-Others-(31-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LURyYXdpbmdzLSgwNS0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Drawings-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUZvcm0tMS0oMDUtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Form-1-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1ERUxOUC0yMDA3LUZvcm0tMS0oMjktMDctMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4857-DELNP-2007-Form-1-(29-07-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUZvcm0tMi0oMDUtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Form-2-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1ERUxOUC0yMDA3LUZvcm0tMjYtKDE1LTA5LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4857-DELNP-2007-Form-26-(15-09-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUZvcm0tMy0oMDUtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Form-3-(05-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LUZvcm0tMy0oMjQtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Form-3-(24-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LWZvcm0tNi0oMjktMDctMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-form-6-(29-07-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1Ny1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMDUtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4857-delnp-2007-Petition-137-(05-07-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="256853-a-method-and-system-for-controlling-a-user-controllable-telecommunication-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="256855-a-method-of-seaming-an-on-machine-seamable-multiaxial-papermaker-s-fabrics-and-a-papermaker-s-fabric.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>256854</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4857/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>32/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>09-Aug-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Aug-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Jun-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DE STATT DER NEDERLANDEN,VERT DOOR DE MINISTER VAN VWS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>POSTBUS 20350,2500EJ DEN HAAG, THE NETHERLANDS</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TOMMASSEN, JOHANNES, PETRUS, MARIA</td>
											<td>HAMAKERLAAN 7, NL-3571 ZE UTRECHT, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN DERLEY, PETER, ANDRE</td>
											<td>VEEARTSENIJSTRAAT 211, NL-3572 DJ UTRECHT, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>GEURTSEN, JEROEN, JOHANNES, GERARDUS</td>
											<td>MATTENBIESLAAN 110, NL-3452 AG VLEUTEN, THE NETHERLANDS.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/16;A61K 35/74;A61K 39/10;</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL2005/050081</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04078445.6</td>
									<td>2004-12-17</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/256854-a-gram-negative-bacterium-producing-3-0-deacylated-lipopolysaccharide-lps by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:05:43 GMT -->
</html>
