<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/257602-ice-recrystallisation-inhibition-protein-or-antifreeze-proteins-from-deschampsia-lolium-and-festuca-species-of-grass by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:58:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 257602:&quot;ICE RECRYSTALLISATION INHIBITION PROTEIN OR ANTIFREEZE PROTEINS FROM DESCHAMPSIA, LOLIUM AND FESTUCA SPECIES OF GRASS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ICE RECRYSTALLISATION INHIBITION PROTEIN OR ANTIFREEZE PROTEINS FROM DESCHAMPSIA, LOLIUM AND FESTUCA SPECIES OF GRASS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to nucleic acids or nucleic acid fragments encoding amino acid sequences for polypeptides involved in tolerance to freezing and/or low temperature stress in plants. More particularly, the present invention relates to nucleic acids or nucleic acid fragments encoding amino acid sequences for ice recrystallisation inhibition proteins (IRIPs) in plants, and the use thereof for the modification of plant response to freezing and/or low temperature stress. Even more particularly, the present invention relates to polypeptides involved in tolerance to freezing and/or low temperature stress in Deschampsia and Lolium species.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to nucleic acids or nucleic acid fragments encoding amino acid sequences for polypeptides involved in tolerance to freezing and/or low temperature stress in plants. More particularly, the present invention relates to nucleic acids or nucleic acid fragments encoding amino acid sequences for ice recrystallisation inhibition proteins (IRIPs) in plants, and the use thereof for the modification of plant response to freezing and/or low temperature stress. Even more particularly, the present invention relates to polypeptides involved in tolerance to freezing and/or low temperature stress in Deschampsia and Lolium species.<br>
Plants have evolved a range of physiological and biochemical responses to freezing and low temperature stress. As a consequence of poikilothermy many plant species are tolerant of temperature extremes, including exposure to sub-zero temperatures. Subzero temperatures negatively impact on plant cells in many ways. As temperatures drop below freezing ice crystal formation initially takes place extracellularly, in the apoplasm. This leads to an elevation of intracellular solute concentration as water is lost by osmosis to the extracellular ice, resulting in severe dehydration. Desiccation, whereby as much as 90% of intracellular water can be lost at -10°C, induces multiple forms of membrane damage. Furthermore, extracellular ice obstructs gas and solute exchange, and growing ice crystals cause plasmolysis.<br>
Plants and other organisms that are exposed to subzero temperatures have evolved varied mechanisms to confer tolerance to freezing stress including deployment of variant isozymes, synthesis of osmoprotectants and compatible solutes, and modification of membrane lipid composition. A particular characteristic of tolerance to freezing, and to temperature stresses in general, is the phenomenon of acquired tolerance. For freezing stress this is termed cold acclimation, whereby a transition to low, non-freezing temperature can confer tolerance of subsequent exposure to otherwise lethal subzero temperatures.<br>
A common response of plant and other species with tolerance to subzero temperatures and freezing is the expression of anti-freeze proteins (AFPs). AFPs have an affinity for<br><br>
ice, by virtue of structural complementarity, thereby inhibiting its growth. Adsorbtion of AFPs onto ice surfaces has two distinct effects: thermal hysteresis (TH) and recrystallisation inhibition (Rl). TH results from a noncolligative freezing point depression as ice front growth becomes restricted to sterically unfavourable spaces between AFPs. This broadens the gap between the melting and freezing points of ice, and this range is the measure of TH. AFPs mediate the effect of Rl by interfering with the migration of ice boundaries which normally thermodynamically favour the creation of large, ice crystals at the expense of smaller ones. Thus Rl activity limits the growth of large ice crystals that have the potential to puncture cell walls and membranes and cause plasmolysis. Rl activity has been identified in extracts from a limited number of plant species, and the nucleotide sequence of one ice recrystallisation inhibition protein (IRIP) conferring such activity has been reported from Lolium perenne.<br>
Antarctic hair grass Deschampsia antarctica is one of only two angiosperms to have overcome the geographical and environmental impediments to colonising the Antarctic continent. It grows in favourable locations along the western coast of the Antarctic Peninsula. D. antarctica is an over-wintering species with a short growing season that at Palmer Station (64°47'S), is typically November to March. In respect of low temperature stress, on Leonie^ Island in northern Marguerite Bay (67°36'S) towards the southern limit of distribution of D. antarctica, air temperatures below -30°C have been recorded during the austral winter. During the growing season, when plants are most vulnerable to freezing stress, episodes of temperatures down to -15°C can occur early in the growing season. D. antarctica has a well developed cold-acclimation response, and significant cellular damage only occurs in plants exposed to temperatures substantially below those at which they freeze.<br>
Despite D. antarctica's well developed freezing tolerance no biochemical or physiological mechanisms have been identified that can coherently account for this capacity.<br>
There is a need for materials useful in modifying the tolerance to freezing and low temperature stress in a wide range of plants, and for methods for their use.<br><br>
It is an object of the present invention to overcome, or at least alleviate, one or more of the difficulties or deficiencies associated with the prior art.<br>
In one aspect, the present invention provides substantially purified or isolated nucleic acids or nucleic acid fragments encoding IRIPs from a Deschampsia species, preferably Antarctic hair-grass, Deschamps/a antarcfica, or functionally active fragments or variants thereof.<br>
In a preferred embodiment of this aspect of the invention, the substantially purified or isolated nucleic acid or nucleic acid fragment includes a nucleotide sequence selected from the group consisting of (a) sequences shown in Figures 8, 9, 11,12, 14, 15, 17, 18, 20, 21, 23 and 24 hereto; (b) complements of the sequences recited in (a); (c) sequences antisense to the sequences recited in (a) and (b); (d) functionally active fragments and variants of the sequences recited in (a), (b) and (c); and (e) RNA sequences corresponding to the sequences recited in (a), (b), (c) and (d).<br>
In another aspect, the present invention provides substantially purified or isolated nucleic acids or nucleic acid fragments encoding IRIPs from a ryegrass (Lolium) or fescue (Festuca) species. These species may be of any suitable type, including Italian or annual ryegrass, perennial ryegrass, tall fescue, meadow fescue and red fescue. Preferably the species is a ryegrass, more preferably perennial ryegrass (L perenne).<br>
In a preferred embodiment of this aspect of the invention, the substantially purified or isolated nucleic acid or nucleic acid fragment includes a nucleotide sequence selected from the group consisting of (a) sequences shown in Figures 26, 27, 29 and 30 hereto; (b) complements of the sequences recited in (a); (c) sequences antisense to the sequences recited in (a) and (b); (d) functionally active fragments and variants of the sequences recited in (a), (b) and (c); and (e) RNA sequences corresponding to the sequences recited in (a), (b), (c) and (d).<br>
The present invention provides substantially purified or isolated nucleic acids or nucleic acid fragments encoding amino acid sequences for a class of polypeptides which are related to IRIP or functionally active fragments or variants thereof.   Such proteins are<br><br>
More preferably the regulatory element includes a nucleotide sequence selected from the groups consisting of (a) sequences shown in Figures 32 and 33 hereto; (b) complements of the sequences recited in (a); and (c) functionally active fragments and variants of the sequences recited in (a) and (b).<br>
in a further aspect of the present invention there is provided a substantially purified or isolated regulatory element from a Lolium or Festuca species, preferably Lolium perenne, said regulatory element being capable of modifying plant response to freezing and/or low temperature stress.<br>
More preferably the regulatory element includes a nucleotide sequence selected from the group consisting of (a) sequence shown in Figure 34 hereto; (b) complement of the sequence recited in (a) and (c) functionally active fragments and variants of the sequences recited in (a) and (b).<br>
Preferably the regulatory element is a promoter.<br>
Preferably the regulatory element is isolated from an IRIP nucleic acid or nucleic acid. fragment.<br>
As used herein, the term IRIP-like relates to polypeptides that are produced in the plant in substantially the same organs and at substantially the same developmental stages as IRIP.<br>
The term "isolated" means that the material is removed from its original environment (eg. the natural environment if it is naturally occurring). For example, a naturally occurring nucleic acid or polypeptide present in a living plant is not isolated, but the same nucleic acid or polypeptide separated from some or all of the coexisting materials in the natural system, is isolated. Such nucleic acids could be part of a vector and/or such nucleic acids could be part of a composition, and still be isolated in that such a vector or composition is not part of its natural environment.<br><br>
As used herein, the term "consensus contig" refers to a nucleotide sequence that is assembled from two or more constituent nucleotide sequences that share common or overlapping regions of sequence homology. For example, the nucleotide sequence of two or more nucleic acid fragments can be compared and aligned in order to identify common or overlapping sequences. Where common or overlapping sequences exist between two or more nucleic acid fragments, the sequences (and thus their corresponding nucleic acid fragments) can be assembled into a single contiguous nucleotide sequence.<br>
The term "purified" means that the nucleic acid or polypeptide is substantially free of other nucleic acids or polypeptides.<br>
By "functionally active" in respect of a nucleic acid it is meant that the fragment or variant (such as an analogue, derivative or mutant) is capable of modifying the tolerance of freezing and/or low temperature stress in a plant. Such variants include naturally occurring allelic variants and non-naturally occurring variants. Additions, deletions, substitutions and derivatizations of one or more of the nucleotides are contemplated so long as the modifications do not result in loss of functional activity of the fragment or variant. Preferably the functionally active fragment or variant has at least approximately 80% identity to the relevant part of the above mentioned sequence, more preferably at least approximately 90% identity, most preferably at least approximately 95% identity. Such functionally active variants and fragments include, for example, those having nucleic acid changes which result in conservative amino acid substitutions of one or more residues in the corresponding amino acid sequence. Preferably the fragment has a size of at least 30 nucleotides, more preferably at least 45 nucleotides, most preferably at least 60 nucleotides.<br>
By "functionally active" in respect of a polypeptide is meant that the fragment or variant has one or more of the biological properties of an IRIP or IRIP-like protein. Additions, deletions, substitutions and derivatizations of one or more of the amino acids are contemplated so long as the modifications do not result in loss of functional activity of the fragment or variant. Preferably the functionally active fragment or variant has at least approximately 60% identity to the relevant part of the above mentioned sequence,<br><br>
more preferably at least approximately 80% identity, most preferably at least approximately 90% identity. Such functionally active variants and fragments include, for example, those having conservative amino acid substitutions of one or more residues in the corresponding amino acid sequence. Preferably the fragment has a size of at least 10 amino acids, more preferably at least 15 amino acids, most preferably at least 20 amino acids.<br>
The term "construct" as used herein refers to an artificially assembled or isolated nucleic acid molecule which includes the gene of interest. In general a construct may include the gene or genes of interest, a marker gene which in some cases can also be the gene of interest and appropriate regulatory sequences. It should be appreciated that the inclusion of regulatory sequences in a construct is optional, for example, such sequences may not be required in situations where the regulatory sequences of a host cell are to be used. The term construct includes vectors but should not be seen as being limited thereto.<br>
The term "vector" as used herein encompasses both cloning and expression vectors. Vectors are often recombinant molecules containing nucleic acid molecules from several sources.<br>
By "operatively linked" is meant that said regulatory element is capable of causing expression of said nucleic acid or nucleic acid fragment in a plant cell and said terminator is capable of terminating expression of said nucleic acid or nucleic acid fragment in a plant cell. Preferably, said regulatory element is upstream of said nucleic acid or nucleic acid fragment and said terminator js downstream of said nucleic acid or nucleic acid fragment.<br>
By "an effective amount" it is meant an amount sufficient to result in an identifiable phenotypic trait in said plant, or a plant, plant seed or other plant part derived therefrom. Such amounts can be readily determined by an appropriately skilled person, taking into account the type of plant, the route of administration and other relevant factors. Such a person will readily be able to determine a suitable amount and method of administration. See, for example, Maniatis et a\, Molecular Cloning: A Laboratory Manual, Cold Spring<br><br>
Harbor Laboratory, Cold Spring Harbor, the relevant disclosure of which is incorporated herein by reference.<br>
It will also be understood that the term "comprises" (or its grammatical variants) as used in this specification is equivalent to the term "includes" and should not be taken as excluding the presence of other elements or features.<br>
Genes encoding other IRIP or IRIP-like proteins for modifying the tolerance of plants to freezing and/or low temperature stress, either as cDNAs or genomic DNAs, may be isolated directly by using all or a portion of the nucleic acids or nucleic acid fragments of the present invention as hybridisation probes to screen libraries from the desired plant employing the methodology well known to those skilled in the art. Specific oligonucleotide probes based upon the nucleic acid sequences of the present invention may be designed and synthesized by methods known in the art. Moreover, the entire sequences may be used directly to synthesize DNA probes by methods known to the skilled artisan, such as random primer DNA labelling, nick translation, or end-labelling techniques, or RNA probes using available in vitro transcription systems. In addition, specific primers may be designed and used to amplify a part or all of the sequences of the present invention. The resulting amplification products may be labelled directly during amplification reactions or labelled after amplification reactions, and used as probes to isolate full length cDNA or genomic fragments under conditions of appropriate stringency.<br>
In addition, short segments of the nucleic acids or nucleic acid fragments of the present invention may be used in protocols to amplify longer nucleic acid fragments encoding homologous genes from DNA or RNA. For example, polymerase chain reaction may be performed on a library of cloned nucleic acid fragments wherein the sequence of one primer is derived from the nucleic acid sequences of the present invention, and the sequence of the other primer takes advantage of the presence of the polyadenylic acid tracts to the 3' end of the mRNA precursor encoding plant genes. Alternatively, the second primer sequence may be based upon sequences derived from the cloning vector. For example, those skilled in the art can follow the RACE protocol [Frohman et al. (1988) Proc. Natl. Acad Sci.  USA 85:8998, the entire disclosure of which is<br><br><br>
incorporated herein by reference] to generate cDNAs by using PCR to amplify copies of the region between a single point in the transcript and the 3' or 5' end. Using commercially available 3' RACE and 5' RACE systems (BRL), specific 3' or 5' cDNA fragments may be isolated [Ohara et al. (1989) Proc. Natl. Acad Sci USA 86:5673; 5 Loh et al. (1989) Science 243:217, the entire disclosures of which are incorporated herein by reference]. Products generated by the 3' and 5' RACE procedures may be combined to generate full-length cDNAs.<br>
In a further aspect of the present invention there is provided an isolated ice recrystallisation inhibition protein derived from a plant wherein said protein includes: an IRIP domain including 3 to 32 repeats of the amino acid sequence set forth in SEQ ID NO: 141; and an amino acid residue motif as set forth in SEQ ID NO: 145.<br>
In a further aspect, the present invention also provides isolated nucleic acids which encode the ice recrystallisation inhibition proteins described herein.<br>
In a preferred embodiment of this aspect of the invention, the substantially purified or isolated polypeptide includes an amino acid sequence selected from the group consisting of sequences shown in Figures 10, 13, 16,19, 22 and 25 hereto; and functionally active fragments and variants thereof.<br>
In a still further aspect of the present invention there is provided a substantially purified or isolated IRIP or IRIP-like polypeptide from a ryegrass (Lolium) or fescue (Festuca) species; and functionally active fragments and variants thereof.<br>
The ryegrass (Lolium) or fescue (Festuca) species may be of any suitable type, including Italian or annual ryegrass, perennial ryegrass, tall fescue, meadow fescue and red fescue.   Preferably the species is a ryegrass, more preferably perennial ryegrass (L. perenne).<br>
In a preferred embodiment of this aspect of the invention, the substantially purified or isolated polypeptide includes an amino acid sequence selected from the group<br><br><br>
consisting of sequences shown in Figures 28 and 31 hereto; and functionally active fragments and variants thereof.<br>
The Applicant has found that the polypeptides of the present invention include relatively few leucine rich repeat (LRR) motifs.<br>
Preferably said LLR motifs from a Deschampsia species include the consensus sequencer set forth in SEQ ID NO: 128, or the consensus sequence set forth in SEQ ID NO: 143.<br>
Preferably said LRR motifs from a Lolium or Festuca species include the consensus sequencer set forth in SEQ ID NO: 129, or the consensus sequence set forth in SEQ ID NO: 144.<br>
Applicant has found that polypeptides of the present invention including relatively few LRR motifs, preferably 3 or fewer LRR motifs, more preferably 1 or fewer LRR motifs, may be more efficient at modifying tolerance of freezing and/or low temperature 20 stress in a plant than nucleic acids or nucleic acid fragments having relatively more LRR motifs, for example approximately 9 or more LRR motifs. Similarly, the nucleic acids or nucleic acid fragments encoding such polypeptides may be more efficient at modifying tolerance of freezing and/or low temperature stress in a plant.<br>
In a further embodiment of this aspect of the invention, there is provided a polypeptide recombinantly produced from a nucleic acid or nucleic acid fragment according to the present invention. Techniques for recombinantly producing polypeptides are known to those skilled in the art.<br><br>
Availability of the nucleotide sequences of the present invention and deduced amino acid sequences facilitates immunological screening of cDNA expression libraries. Synthetic peptides representing portions of the instant amino acid sequences may be synthesized. These peptides may be used to immunise animals to produce polyclonal or monoclonal antibodies with specificity for peptides and/or proteins comprising the amino acid sequences. These antibodies may be then used to screen cDNA expression libraries to isolate full-length cDNA clones of interest.<br>
A genotype is the genetic constitution of an individual or group. Variations in genotype are essential in commercial breeding programs, in determining parentage, in diagnostics and fingerprinting, and the like. Genotypes can be readily described in terms of genetic markers. A genetic marker identifies a specific region or locus in the genome. The more genetic markers, the finer defined is the genotype. A genetic marker becomes particularly useful when it is allelic between organisms because it then may serve to unambiguously identify an individual. Furthermore, a genetic marker becomes particularly useful when it is based on nucleic acid sequence information that can unambiguously establish a genotype of an individual and when the function encoded by such nucleic acid is known and is associated with a specific trait. Such nucleic acids and/or nucleotide sequence information including single nucleotide polymorphisms (SNP's), variations in single nucleotides between allelic forms of such nucleotide sequence, can be used as perfect markers or candidate genes for the given trait.<br>
Applicants have identified a number of SNPs of the nucleic acids or nucleic acid fragments of the present invention. These are indicated (marked with grey on the black background) in the figures that show multiple alignments of nucleotide sequences of nucleic acid fragments contributing to consensus contig sequences. See for example, Figures 8,11,14,17,20, 23, 26 and 29 hereto.<br>
Accordingly, in a further aspect of the present invention, there is provided a substantially purified or isolated nucleic acid or nucleic acid fragment including a single nucleotide polymorphism (SNP) from a nucleic acid fragment shown in Figures 8, 9,11,12,14,15, 17, 18, 20, 21, 23, 24, 26, 27, 29 and 30 hereto, or complements or sequences antisense thereto.<br><br>
In a still further aspect of the present invention there is provided a method of isolating a nucleic acid or nucleic acid fragment of the present invention including a single nucleotide polymorphism (SNP), said method including sequencing nucleic acid fragments from a nucleic acid library.<br>
The nucleic acid library may be of any suitable type and is preferably a cDNA library.<br>
The nucleic acid fragments may be isolated from recombinant plasmids or may be amplified, for example using polymerase chain reaction.<br>
The sequencing may be performed by techniques known to those skilled in the art.<br>
In a still further aspect of the present invention, there is provided use of nucleic acids or nucleic acid fragments of the present invention including SNP's, and/or nucleotide sequence information thereof, as molecular genetic markers.<br>
In a still further aspect of the present invention there is provided use of a nucleic acid or nucleic acid fragment according to the present invention, and/or nucleotide sequence information thereof, as a molecular genetic marker.<br>
More particularly, nucleic acids or nucleic acid fragments according to the present invention and/or nucleotide sequence information thereof may be used as a molecular genetic marker for quantitative trait loci (QTL) tagging, QTL mapping, DNA fingerprinting and in marker assisted selection, particularly in grasses and cereals. Even more particularly, nucleic acids or nucleic acid fragments according to the present invention and/or nucleotide sequence information thereof may be used as molecular genetic markers in grass and cereal improvement, e.g. tagging QTLs for tolerance to freezing and/or low temperature stress. Even more particularly, sequence information revealing SNPs in allelic variants of the nucleic acids or nucleic acid fragments of the present invention and/or nucleotide sequence information thereof may be used as molecular genetic markers for QTL tagging and mapping and in marker assisted selection, particularly in grasses and cereals.<br><br>
In a still further aspect of the present invention there is provided a construct including a nucleic acid or nucleic acid fragment according to the present invention. The construct may be a vector.<br>
In a preferred embodiment of this aspect of the invention, the vector may include a regulatory element such as a promoter, a nucleic acid or nucleic acid fragment according to the present invention and a terminator; said regulatory element, nucleic acid or nucleic acid fragment and terminator being operatively linked.<br>
In a further preferred embodiment of this aspect of the invention, the vector may include a regulatory element according to the present invention, a further nucleic acid molecule and a terminator; said regulatory element, further nucleic acid molecule and terminator being operatively linked.<br>
In a still further preferred embodiment of this aspect of the invention, the vector may include a regulatory element according to the present invention, a nucleic acid or nucleic acid fragment according to the present invention and a terminator, said regulatory element, nucleic acid or nucleic acid fragment and terminator being operatively linked.<br>
The vector may be of any suitable type and may be viral or non-viral. The vector may be an expression vector. Such vectors include chromosomal, non-chromosomal and synthetic nucleic acid sequences, eg. derivatives of plant viruses; bacterial plasmids; derivatives of the Ti plasmid from Agrobacterium tumefaciens, derivatives of the Ri plasmid from Agrobacterium rhizogenes; phage DNA; yeast artificial chromosomes; bacterial artificial chromosomes; binary bacterial artificial chromosomes; vectors derived from combinations of plasmids and phage DNA. However, any other vector may be used as long as it is replicable, or integrative or viable in the plant cell.<br>
The regulatory element and terminator may be of any suitable type and may be endogenous to the target plant cell or may be exogenous, provided that they are functional in the target plant cell.<br><br>
In another embodiment, the construct or vector may include more than one nucleic acid. The nucleic acids within the same construct or vector may have identical or differing sequences. In one preferred embodiment, the construct or vector has at least two nucleic acids encoding functionally similar enzymes. In a particularly preferred embodiment, each further nucleic acid molecule has one or more upstream regulatory elements and one or more downstream terminators, although expression of more than one further nucleic acid molecule from an upstream regulatory element or termination of more than one further nucleic acid molecule from a downstream terminator(s) is not precluded.<br>
Preferably the regulatory element is a promoter. A variety of promoters which may be employed in the constructs and vectors of the present invention are well known to those skilled in the art. Factors influencing the choice of promoter include the desired tissue specificity of the vector, and whether constitutive or inducible expression is desired and the nature of the plant cell to be transformed (eg. monocotyledon or dicotyledon). Particularly suitable promoters include but are not limited to the constitutive Cauliflower Mosaic Virus 35S (CaMV 35S) promoter and derivatives thereof, the maize Ubiquitin promoter, the rice Actin promoter, and the tissue-specific Arabidopsis small subunit (ASSU) promoter. Alternatively, the regulatory element may be a regulatory element according to the present invention.<br>
A variety of terminators which may be employed in the vectors and constructs of the present invention are also well known to those skilled in the art. The terminator may be from the same gene as the promoter sequence or a different gene. Particularly suitable terminators are polyadenylation signals, such as the CaMV 35S polyA and other terminators from the nopaline synthase (nos), the octopine synthase (ocs) and the rbcS genes.<br>
The further nucleic acid molecule may be a sequence, for example a gene or fragment thereof, or sequence antisense thereto, which is capable of modifying plant response to freezing and/or low temperature stress. It may be a nucleic acid or nucleic acid fragment according to the present invention, but is not limited thereto.<br><br>
The vector, in addition to the regulatory element, the nucleic acid or nucleic acid fragment of the present invention and the terminator, may include further elements necessary for expression of the nucleic acid or nucleic acid fragment, in different combinations, for example vector backbone, origin of replication (on), multiple cloning sites, recognition sites for recombination events, spacer sequences, enhancers, introns (such as the maize Ubiquitin (Ubi) intron), antibiotic resistance genes and other selectable marker genes [such as the neomycin phosphotransferase (npt2) gene, the hygromycin phosphotransferase (hph) gene, the phosphinothricin acetyltransferase (bar or par) gene and the gentamycin acetyl transferase (aaacC1) gene], and reporter genes (such as beta-glucuronidase (GUS) gene (gusA) and green fluorescent protein (gfp)]. The vector may also contain a ribosome binding site for translation initiation. The vector may also include appropriate sequences for amplifying expression.<br>
As an alternative to use of a selectable marker gene to provide a phenotypic trait for selection of transformed host cells, the presence of the construct or vector in transformed cells may be determined by other techniques well known in the art, such as PCR (polymerase chain reaction). Southern blot hybridisation analysis, histochemical GUS assays, visual examination including microscopic examination of fluorescence emitted by gfp, northern and Western blot hybridisation analyses.<br>
Those skilled in the art will appreciate that the various components of the construct or vector are operatively linked, so as to result in expression of said nucleic acid or nucleic acid fragment. Techniques for operatively linking the components of the vector of the present invention are well known to those skilled in the art. Such techniques include the use of linkers, such as synthetic linkers, for example including one or more restriction enzyme sites.<br>
The constructs and vectors of the present invention may be incorporated into a variety of plants, including monocotyledons (such as grasses from the genera Deschampsia, Lolium, Festuca, Paspalum, Pennisetum, Panicum and other forage and turf grasses, com, oat, sugarcane, wheat and barley), dicotyledons (such as Arabidopsis, tobacco, white clover, red clover, subterranean clover, alfalfa, eucalyptus, potato, sugarbeet, canola, soybean, chickpea) and gymnosperms.<br><br>
Techniques for incorporating the constructs and vectors of the present invention into plant cells (for example by transduction, transfection or transformation) are well known to those skilled in the art. Such techniques include Agrobacterium-mediated introduction, electroporation to tissues, cells and protoplasts, protoplast fusion, Injection into reproductive organs, injection into immature embryos and high velocity projectile introduction to cells, tissues, calli, immature and mature embryos. The choice of technique will depend largely on the type of plant to be transformed.<br>
Cells incorporating the constructs and vectors of the present invention may be selected, as described above, and then cultured in an appropriate medium to regenerate transformed plants, using techniques well known in the art. The culture conditions, such as temperature, pH and the like, will be apparent to the person skilled in the art. The resulting plants may be reproduced, either sexually or asexually, using methods well known in the art, to produce successive generations of transformed plants.<br>
In a further aspect of the present invention there is provided a plant cell, plant, plant seed or other plant part, including, e.g. transformed with, a construct or vector of the present invention.<br>
The plant cell, plant, plant seed or other plant part may be from any suitable species, including monocotyledons, dicotyledons and gymnosperms.<br>
The present invention also provides a plant plant seed or other plant part, or a plant extract, derived from a plant cell of the present invention.<br>
The present invention also provides a plant, plant seed or other plant part, or a plant extract, derived from a plant of the present invention.<br>
In a further aspect of the present invention there is provided a method of modifying tolerance of freezing and/or low temperature stress in a plant, said method including introducing into said plant an effective amount of a nucleic acid or nucleic acid fragment, construct and/or a vector according to the present invention.<br><br>
Using the methods and materials of the present invention, the tolerance of freezing and/or low temperature stress in a plant may be increased or decreased or otherwise modified. For example, the tolerance of freezing and/or low temperature stress may be increased or otherwise altered. They may be increased, for example, by incorporating additional copies of a sense nucleic acid or nucleic acid fragment of the present invention. They may be decreased, for example, by incorporating an antisense nucleic acid or nucleic acid fragment of the present invention.<br>
In a further aspect of the present invention there is provided a preparation for transforming a plant comprising at least one nucleic acid according to the present invention. The preparation may contain vectors or other constructs to facilitate administration to and/or transformation of the plant with the nucleic acid.<br>
The present, invention will now be more fully described with reference to the accompanying Examples and drawings. It should be understood, however, that the description following is illustrative only and should not be taken in any way as a restriction on the generality of the invention described above.<br>
In the Figures<br>
Figure 1. Rl assay on total extracts of leaves from non-acclimated (grown at 22°C) and cold acclimated (5°C) D. antarctica. A, Initial ice crystal structure following snap freezing. B, Ice crystal structure after 16h incubation at -3°C. Capillary B contains extraction buffer; capillaries 1-7:1000, 250, 62.5,15.6, 3.91, 0.977 and 0.244 µg mL-1 respectively of total leaf protein. Extracts were either untreated or subject to incubation at 95°C for 5 min as indicated. Endpoint of Rl activity defined as the lowest protein concentration (µg mL-1) at which ice crystal structure in B remains unchanged from that in A.<br>
Figure 2. A, Repeat structures of representative IRIP orthologues HvlRIP (SEQ ID NO. 126) and DalRIPd (SEQ ID NO. 31). Four cysteine residues conserved in LRR proteins, and predicted to form two disulphide bridges are shown in bold, connected by lines to show predicted bridges. A highly conserved 9 amino acid<br><br>
motif including 3 of these cysteine residues is underlined. Consensus sequences for plant LRRs (SEQ ID NOs. 128 and 143) (Kobe and Kajava (2001) Curr Opin Struct Biol 11:725), and the IRIP repeat (SEQ ID NO. 141) (this work) are shown in bold below the tandem repeat alignments, and identical residues highlighted by grey boxes. B, Sequence alignment of IRIP orthologues and a putative PSKR orthologue from Oryza sativa. Sequences include LplRIP (Sidebottom, et al. (2000 Nature 406:256)) (SEQ ID NO. 124), predicted IRIP orthologue TalRIP derived from assembly of T. aestivum sequences in the NCBI EST database (SEQ ID NO. 125), predicted HvlRIP derived from assembly of H. vulgare sequences in the NCBI EST database (SEQ ID NO. 126), predicted LmlRIP derived from assembly of L multiflorum sequences in the NCBI EST database (SEQ ID NO. 142), and OsPSKR a putative PSKR orthologue from Oryza sativa (NP_911036) (SEQ ID NO. 127). Sequences of the present invention are DalRlPa (SEQ ID NO. 17), DalRIPd (SEQ ID NO. 31) DalRIPe.7 (SEQ ID NO. 38), LplRlPa (SEQ ID NO. 102) and LplRIPb (SEQ ID NO. 120). Identical and conserved residues are highlighted by black and grey boxes respectively. Four invariantly conserved cysteine residues are marked with arrowheads. C, Schematic of domain organisation in IRIP orthologues and OsPSKR. SP: signal peptide; 2x S-S; domain predicted to form 2 disulphide bridges; LRRs numbered; lsland/lsla./ls.: island domain; IRIP repeats unlabelled; TM: transmembrane domain. D, Phylogenetic tree of IRIP orthologues and LRR proteins. Branch lengths are proportional to the number of amino acid substitutions per site (indicated by scale bar). LRR proteins include accession number as suffix.<br>
Figure 3. Structural modelling of IRIPs. A, Theoretical structure of DalRlPa aligned along the prism face of ice (parallel to the a-axis). B, Theoretical structure of LplRlPa aligned along the prism face of ice (parallel to the a-axis). Cysteine residues at positions 120 and 143 have been modelled to participate in an additional disulphide bond, relative to DalRlPa. C, Ribbon backbone diagram of DalRlPa highlighting the amino acid residue composition of the two putative ice binding surfaces, side "A" and side "B".   D, Cross-sectional view of ribbon<br><br>
backbone of 2 ß-roll loops of DalRlPa showing positions of amino acid residue side chains.<br>
Figure 4. Genomic organisation of IRIP genes. A, D. antarctica genomic Southern probed with DalRIPe. B, L perenne polygenic genomic Southern probed with LplRIPa. C, L perenne isogenic genomic Southern probed with LplRIPa. 1: Undigested; 2: Sphl; 3: Kpnl; 4: Hindlll; 5: BamHI.<br>
Figure 5. SNP-based genetic mapping. A, Distribution of locus-specific amplification primers, genomic amplicons and putative SNP loci across the components of the LplRIPa transcriptional unit. LplRIP SNPs are indicated using the nomenclature na (number), indicating that the SNP was identified between NA6 parental haplotypes, and the relevant base pair coordinate. SNPs derived from single gene copies that segregate in the F1 progeny are shown in bold, while SNPs that potentially discriminate between paralogous gene copies are shown in normal text. B, Genetic map order in the upper part of the LG1 of the NA6 parental genetic map, showing the LplRIPNA476-detected SNP locus.<br>
Figure 6. Analysis of IRIP gene expression in response to temperature. A, Northern blot of RNA from D. antarctica leaves and roots grown at 22, 4, and -16°C, probed with DalRIPe. B, Northern blot of RNA from L perenne leaves and roots grown at 22 and 4°C, probed with LplRIPa.<br>
Figure 7. Rl assay on total extracts of E. coli expressing a putative orthologue of histone H3.2 and DalRIPe. A, Initial ice crystal structure following snap freezing. B, Ice crystal structure after 16h incubation at -3°C. Capillary EB contains extraction buffer; capillary BSA 1000 µg mL-1 bovine serum albumin; capillary PC cold acclimated D. antarctica leaf extract as positive control; capillaries 1-4: 400, 100, 25 and 6.25 µg mL-1 respectively of total extracts of E. coli.<br>
Figure 8. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence DalRlPa (SEQ ID NOs. 1-15)<br><br>
Figure 9. Consensus nucleotide sequence of DalRlPa (SEQ ID NO. 16)<br>
Figure 10. Deduced amino acid sequence of DalRlPa (SEQ ID NO. 17)<br>
Figure 11. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence DalRIPb (SEQ ID NOs. 18-20)<br>
Figure 12. Consensus nucleotide sequence of DalRIPb (SEQ ID NO. 21)<br>
Figure 13. Deduced amino acid sequence of DalRIPb (SEQ ID NO. 22)<br>
Figure 14. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence DalRIPd (SEQ ID NOs. 23-29)<br>
Figure 15. Consensus nucleotide sequence of DalRIPd (SEQ ID NO. 30)<br>
Figure 16. Deduced amino acid sequence of DalRIPd (SEQ ID NO. 31)<br>
Figure 17. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence DalRIPe7 (SEQ ID NOs. 32-36)<br>
Figure 18. Consensus nucleotide sequence of DalRIPe7 (SEQ ID NO. 37)<br>
Figure 19. Deduced amino acid sequence of DalRIPe7 (SEQ ID NO. 38)<br>
Figure 20. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence DalRIPe8 (SEQ ID NOs. 39-44)<br>
Figure 21. Nucleotide sequence of DalRIPe8 (SEQ ID NO. 45)<br>
Figure 22. Deduced amino acid sequence of DalRIPe8 (SEQ ID NO. 46)<br>
Figure 23. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence DalRIPf (SEQ ID NOs. 47-52)<br><br>
Figure 24. Consensus nucleotide sequence of DalRIPf (SEQ ID NO. 53)<br>
Figure 25. Deduced amino acid sequence of DalRIPf (SEQ ID NO. 54)<br>
Figure 26. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence LplRlPa (SEQ ID NOs. 55-100)<br>
Figure 27. Consensus nucleotide sequence of LplRlPa (SEQ ID NO. 101)<br>
Figure 28. Deduced amino acid sequence of LplRlPa (SEQ ID NO. 102)<br>
Figure 29. Nucleotide sequences of the nucleic acid fragments contributing to the consensus contig sequence LplRIPb (SEQ ID NOs. 103-118)<br>
Figure 30. Consensus nucleotide sequence of LplRIPb (SEQ ID NO. 119)<br>
Figure 31. Deduced amino acid sequence of LplRIPb (SEQ ID NO. 120)<br>
Figure 32. Nucleotide sequence of promoter region of DalRlPa extending to the initiating ATG (underlined) (SEQ ID NO. 121)<br>
Figure 33. Nucleotide sequence of promoter region of DalRIPd extending to the initiating ATG (underlined) (SEQ ID NO. 122)<br>
Figure 34. Nucleotide sequence of promoter region of LplRlPa extending to the initiating ATG (underlined) (SEQ ID NO. 123)<br>
Figure 35. Plasmid map of vector used for DalRlPa gain of function biolistic transformation of wheat.<br>
Figure 36. Plasmid map of vector used for DalRIPd gain of function biolistic transformation of wheat.<br><br>
Figure 37. Plasmid map of vector used for DalRIPe7 gain of function bioiistic transformation of wheat.<br>
Figure 38. Plasmid map of vector used for DalRIPe8 gain of function bioiistic transformation of wheat.<br>
Figure 39. Plasmid map of vector used for LplRlPa gain of function bioiistic transformation of ryegrass.<br>
Figure 40. Plasmid map of vector used for LplRIPb gain of function bioiistic transformation of ryegrass.<br>
Figure 41. Plasmid map of vector used for LplRlPa loss of function bioiistic transformation of ryegrass.<br>
Figure 42. Plasmid map of vector used for LplRIPb loss of function bioiistic transformation of ryegrass.<br>
Figure  43.   Plasmid   map  of vector   used  for  DatRlPa   gain  of  function Agrobacterium mediated transformation of wheat and barley.<br>
Figure  44.   Plasmid   map of vector  used  for DalRIPd  gain   of function Agrobacterium mediated transformation of wheat and barley.<br>
Figure  45.   Plasmid   map of vector  used  for DalRIPe7 gain  of function Agrobacterium mediated transformation of wheat and barley.<br>
Figure 46.  Plasmid   map of vector  used for DalRIPe8  gain  of function Agrobacterium mediated transformation of wheat and barley.<br>
Figure   47.   Plasmid   map  of vector   used  for  DalRlPa   gain  of  function Agrobacterium mediated transformation of Arabidopsis.<br><br>
Figure  48.   Plasmid   map  of vector used  for DalRIPd gain  of function Agrobacterium mediated transformation of Arabidopsis.<br>
Figure 49.  Plasmid  map of vector used for DalRIPe7 gain  of function Agrobacterium mediated transformation of Arabidopsis.<br>
Figure  50.  Plasmid   map  of vector used  for DalRIPe8  gain  of function Agrobacterium mediated transformation of Arabidopsis.<br>
Figure   51-  Plasmid   map  of vector  used  for LplRlPa   gain   of function Agrobacterium mediated transformation of Arabidopsis.<br>
Figure   52.   Plasmid   map  of vector  used  for LplRIPb  gain   of function Agrobacterium mediated transformation of Arabidopsis.<br>
Figure 53. Plasmid map of vector used for DalRIPd promoter::GUS reporter gene Agrobacterium mediated transformation of Arabidopsis.<br>
Example 1<br>
Materials and Methods<br>
Plant propagation, stress treatments, extraction of Rl activity, DNA and RNA<br>
D. antarctica material was collected in the vicinity of Jubany station on King George Island (62°14'S 58°40'W). Plants were germinated from seeds in the soil seed bank and thereafter were propagated vegetatively. Lolium perenne plants were of cultivar Impact Doubled haploid L perenne plants, where isolate DH297 of cultivar Vema.<br>
Individual plants were grown in soil at the indicated temperatures under a 16/8h light/dark regime and photosynthetic photon flux intensity of 400 µmol m-2 s-1 in<br><br>
Enconair (Winnipeg, Canada) GC-20 plant growth chambers. Plants were cold acclimated by growth at 5°C for 2 weeks. Plants were divided into aerial and subterranean parts and snap frozen in liquid N2.<br>
Total cellular extracts were prepared after (Doucet et al (2000) Cryobiology 40:218) by grinding plant material under liquid N2 and resuspending the powder in 2 mL g-1 of freshly prepared extraction buffer (50mM Tris pH 7.4, 20mM ascorbate, 10mM EDTA). The extract was filtered through miracloth (Calbiochem, La Jolla, USA). Apoplastic extracts were based on the method of Chun et al (1998) Euphytica 102:219. Leaf material was vacuum infiltrated in extraction buffer for 30 min, excess liquid removed from the leaves, and extracts collected by centrifugation at 800g for 30 min. All extracts were aliquoted, frozen in liquid N2, and stored at -80°C.<br>
RNA and DNA were extracted using RNeasy and DNeasy Plant Mini kits (QIAGEN, Germany) respectively.<br>
Ice recrystallisation inhibition (Rl) assays<br>
Extracts were thawed and insoluble material pelleted by centrifugation at 16,060g for 5 min. Supernatants were collected and for total cellular extracts protein content quantified using the Bio-Rad protein assay (Bio-Rad, Mississauga, ON, Canada), according to the manufacturer's instructions. Unless otherwise stated all extracts were incubated at 95°C for 5 min. The supernatants from both heat treated and untreated extracts were collected following centrifugation at 16,060g for 2 min. Serial 2 or 4-fold dilutions into extraction buffer were prepared.<br>
The capillary method for the Rl assay was modified from that of Tomczak et al (2003) Biochem. Biophys. Res. Commun. 311:1041. Briefly, extracts were loaded into 10µl glass capillaries (Drummond Scientific, Broomall, PA, USA), heat sealed and arrayed on a glass slide secured with adhesive tape. Extraction buffer, and BSA (Bio-Rad, Mississauga, ON, Canada) dissolved in extraction buffer to a final concentration of 1000 µg mL-1 were included as negative controls. The capillary array was snap-frozen in an ethanol/dry ice bath and immersed in a reservoir of motor vehicle coolant diluted to a<br><br>
final concentration of 10% monoethylene glycol, atop a jacketed stage through which the same solution at -3°C was circulated using a refrigerated water bath (PolyScience Model 910, Niles.IL, USA).<br>
Samples were scored after overnight (-16 h) incubation at -3°C. The endpoint of Rl activity in total leaf extracts is defined as the lowest protein concentration (µg mL-1) at which ice crystal structure following incubation at -3°C for 16h, remained unchanged from that initially induced by snap freezing. For apoplastic extracts, because of the low yields of protein from non-acclimated plants, the endpoint of Rl activity was expressed as the equivalent wet weight of starting plant material per volume of extract.<br>
Digital images were captured with a Leica DFC 300 F camera mounted on a Leica MZFL III stereoscopic zoom microscope using Leica FireCam software (Leica, Heerbrugg, Switzerland). Polarising light filters mounted perpendicularly on the microscope objective lens and beneath the stage enhanced visualisation of ice crystal morphology.<br>
Preparation of cDNA libraries, isolation and sequencing of cDNAs coding for IRIPs from Antarctic hair-grass, Deschampsia antarctica.<br>
cDNA libraries representing mRNAs from various organs and tissues from Antarctic hair-grass, Deschampsia antarctica were prepared. The characteristics of the libraries are described below (Table 1).<br>
TABLE 1 cDNA libraries from Antarctic hair-grass, Deschampsia antarctica.<br>
 (Table Removed)<br><br>
The cDNA libraries may be prepared by any of many methods available. For example, total RNA may be isolated using the Trizol method (Gibco-BRL, USA) or the RNeasy Plant Mini kit (Qiagen,Germany), following the manufacturers' instructions. cDNAs may be generated using the SMART PCR cDNA synthesis kit (Clontech, USA), cDNAs may be amplified by long distance polymerase chain reaction using the Advantage 2 PCR Enzyme system (Clontech, USA), cDNAs may be cleaned using the GeneClean spin column (Bio 101, USA), tailed and size fractionated, according to the protocol provided by Clontech. The cDNAs may be introduced into the pGEM-T Easy Vector system 1 (Promega, USA) according to the protocol provided by Promega. The cDNAs in the pGEM-T Easy plasmid vector are transfected into Escherichia coli Epicurian coli XL10-Gold ultra competent cells (Stratagene, USA) according to the protocol provided by Stratagene.<br>
Alternatively, the cDNAs may be introduced into plasmid vectors for first preparing the cDNA libraries in Uni-ZAP XR vectors according to the manufacturer's protocol (Stratagene Cloning Systems, La Jolla, CA, USA). The Uni-ZAP XR libraries are converted into plasmid libraries according to the protocol provided by Stratagene. Upon conversion, cDNA inserts will be contained in the plasmid vector pBluescript. In addition, the cDNAs may be introduced directly into precut pBluescript II SK(+) vectors (Stratagene) using T4 DNA ligase (New England Biolabs), followed by transfection into E. coli DH10B cells according to the manufacturer's protocol (GIBCO BRL Products).<br><br>
cDNA clones encoding putative IRIP orthologues from D. antarctica came from 5 libraries derived from either shoots or roots grown at 4°C or -15°C, and from shoots transferred from -15°C to 25°C for 24 hours. One of these variants (DalRIPd) was isolated from both root and shoot libraries, whilst the other four forms were derived from shoot libraries only.<br>
Molecular cloning of genomic sequences<br>
All but one of D. antarctica cDNAs encoding IRIP orthologues (DalRlPa), encoded N-<br>
terminally truncated hypothetical IRIP forms. Therefore full length genomic sequences<br>
where isolated using the GenomeWalker kit (BO Biosciences, Palo Alto, USA) and<br>
nested gene specific primers. The 3' UTR primers for DalRIPd were:<br>
5' GACATCGCGATTGGTCCCACCAAGTG 3' (SEQ ID NO. 130), and<br>
5' GCATCCTGCACGGACATATCATTA 3' (SEQ ID NO. 131);<br>
and DalRIPe:<br>
5' GTTACATAAGACGATTGGCCCCACCAAG 3' (SEQ ID NO. 132), and<br>
5' CAATCCACTCACTGATCATTAACCACC 3' (SEQ ID NO. 133). For the isolation of<br>
LplRlPa nested primers<br>
5' GATGGTATATCCACGAAGTTACAT 3' (SEQ ID NO. 134), and<br>
5' ATTGGCCCCACCAAGTGA 3' (SEQ ID NO. 135) complementary to conserved<br>
regions within the 3' UTR of the D. antarctica IRIP forms were employed. LplRlPa was<br>
also obtained from the L perenne North African6 x Aurorae genetic mapping population<br>
(see below). PCR products were purified using QIAquick gel extraction kit (QIAGEN,<br>
Germany) and molecularly cloned into pGEM-T Easy as above.<br>
DNA template preparation, sequencing and analysis<br>
Templates for sequencing of cDNA and genomic clones plasmid DNA was purified using a QIAprep spin miniprep kit (QIAGEN, Germany). Sequencing reactions primed with a modified SMART primer (5' AAGCAGTGGTAACAACGCAGAGTGGG 3') (SEQ ID NO. 136), M13F or M13R primers were performed either using BigDye Terminators or ET Terminators, and the reaction products resolved on an ABI Prism 3700, or 3730x1 DNA Analyser (PE Biosystems, Foster City, USA), or a MegaBACE 4000 (Amersham<br><br>
Biosciences, UK) respectively. Sequence files were used as queries for BLASTX, BLASTN and TBLASTN Altschul et al 1987 Nucleic Acids Res. 25:3389 searches of the SWISS-PROT, GenBank Main and GenBank EST databases respectively. BLASTX searches of the NCBI database of GenBank+EMBL+DDBJ sequences from EST divisions with full length IRIP gene sequences reveal the presence of many IRIP gene related sequences in grasses including Lolium multiflorum (Italian rye grass), Leymus chinensis, Puccinellia tenuiflora, and from in-house EST data Agrostis tenuis. We have also identified in the NCBI EST database orthologous sequences in the cereals Hordeum vulgare (barley), S. cereale (winter rye), and Triticum monococcum, T. turgidum and T. aestivum (diploid, tetraploid and hexaploid forms of wheat), frequently associated with cDNA libraries derived from cold-stressed or vernalised material. We are not yet aware of any species of the Pooideae without IRIP orthologues. EST sequences were also assembled to derive representative IRIP orthologues. Sequence assembly was performed using Sequencher (Gene Codes, Ann Arbor, Ml, USA). Potential signal sequences were identified by analysis with SignalP (http://www.cbs.dtu.dk/services/SiqnalP/). Subcellular localisation was predicted using TMHMM (http://www.cbs.dtu.dk/services/TMHMM/) and PSORT (http://psort.nibb.ac.ipl Phylogenlc analysis was performed using Vector NTl (Invitrogen, Carlsbad, CA, USA) using the Align X algorithm with default parameters.<br>
Structural modelling<br>
Homology modelling was performed using Schrodinger molecular modelling software (version macromodel 8.6, Portland, OR, USA) using the Prime homology modelling module. A homology model of the DalRlPa and LplRlPa proteins was build using the N terminus of the crystal structure of Phaseolus vulgaris polygalacturonase-inhibiting protein (PGIP) (pdb entry: 10GQ) and a right-handed version of a theoretical Lolium perenne AFP model (Kuiper et al (2001) Biophys. J. 81:3560) as templates. The model was geometrically optimised with distance constraints holding optimal hydrogen bond distances between beta sheet regions for 10,000 iterations using an OPLS2001 forcefield and Generalised Bom (GB) solvation. This was followed by an additional 5000 iterations minimisation without constraints applied. Images were generated using Swiss Pdb Viewer and Pov-Ray3.5.<br><br>
Southern and Northern analysis<br>
For Southerns 10 µg of DNA was digested with restriction enzymes, separated on a 1% agarose TAE gel and stained with ethidium bromide before transfer to Hybond-N membrane (Amersham Biosciences). For Northerns 10 µg of RNA was separated on a 1% agarose formaldehyde gel, transferred to Hybond- N membrane (Amersham Biosciences) and stained with methylene blue. Isolated fragments of DalRIPe7 and LplRlPa were radio-labelled with α-32P-dCTP using the Rediprime II Random Prime Labelling System (Amersham Biosciences) and purified with MicroSpin™ S-400 HR Columns (Amersham Biosciences) according to the manufacturer's instructions. The blotted membranes were hybridised with radio-labelled probe as described by Maniatis et al, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor. Hybridisation patterns were visualised and quantified on a Typhoon 8600 Variable Mode Imager (Amersham Biosciences) according to the manufacturers instructions.<br>
SNP-based genetic mapping<br>
The perennial ryegrass population used for SNP discovery and genetic mapping was an F1 progeny set derived from a reciprocal pair-cross between the heterozygous parental genotypes North African3 (NA6) and Aurora6 (AU6) (Faville et al..Theor Appl Genet, in press).<br>
The procedure for targeted in vitro SNP discovery in perennial ryegrass was described by Forster et al. (2004) Molecular marker-based genetic analysis of pasture and turf grasses, in: Molecular Breeding of Forage and Turf, Hopkins et al (eds) Kluwer Academic Press pp 197-239. The partial cDNA sequence of the IRIP reported from L perenne (Sidebottom et al (2000) Nature 406:256), along with the sequence of LplRlPa, were assembled into a single contig and locus amplification primers (LAPs) were designed to generate 3 amplicons located at various positions within the gene unit (Fig. 5A), to cover the 5'-untranslated region (UTR), single exon and 3'-UTR regions. Genomic amplicons were generated using standard PCR conditions from each parental genotype of the F1(NAs x AU6) population.   PCR fragments were cloned using the<br><br>
TOPO TA system (Invitrogen K4575-40) and DNA sequences were derived through cycle-sequencing. Sequences were assembled in contigs using the Sequencher ver. 4.1.4 application (Gene Codes) and putative SNPs were identified. A total of 1072 bp from the 1495 bp gene length was represented by high quality sequence. In all 26 SNPs were identified following alignment of sequences from the parental genotypes, at an average incidence of 1 per 40 bp, which is higher than the global frequency of 1 per 60 bp observed through resequencing of 87 genes and c. 76 kb of perennial ryegrass genomic DNA {N.O.I. Cogan, unpublished data). This high SNP frequency, along with the observation of more than two haplotype structures in the NAs parent, suggests that paralogous DNA sequences may have been clustered, due to cross-amplification between gene copies. Polymorphic SNP loci were validated using the single nucleotide primer extension (SNuPe) assay system followed by capillary electrophoresis on the MegaBACE1000 platform (Amersham Biosciences).<br>
Genotypic variation for the SNP locus LplRIPHA476 was determined using a sub-set (96) genotypes of the F1(NA6 x AU6) population. The corresponding genomic locus was integrated within the framework of the existing genetic map as previously described (Faville et al., Theor Appl Genet in press). Comparative genomics analysis of IRIP genes was performed using the wEST SQL database in the GrainGenes resource (http://wheat.pw.usda.gov/wEST/). The nucleotide sequences were used for BLASTN and TBLASTX analysis in the GrainGenes BLAST page with the search restricted to ESTs that have been assigned to wheat deletion bins Qi et al (2003) Functional and Integrative Genomics 3:39) The highest matching ESTs were then used to detect the relevant deletion bins using the Mapped Loci query function in wEST SQL.<br>
Genetic map information for the LplRIPb gene was obtained from the second generation perennial ryegrass genetic mapping population derived from a pair-cross between the genotypes North African6 (NA6) and Aurora6 (AU6). LplRlPa specific locus amplification primers (LAPs) were employed to generate 3 genomic amplicons (Fig. 5A) from each parental genotype. Sequence assembly and analysis revealed 2 distinct LplRIP paralogues LplRIP a and b, and a total of 26 SNPs. An assayed SNP locus in LplRlPa showing an AB x SB segregation structure produced only AB-type F1 progeny (data not shown), which is diagnostic of multiple gene structure. A total of 8 SNP loci in<br><br>
LplRIPb showed structures consistent with a single gene copy, but of these 6 showed AA x BB patterns, and could not be genetically mapped in the F1(NA6 x AU6) sib-ship. Of the two loci revealing polymorphism within the NAe parental genotype (LplR/PNA476 and Lp/R/PNA694) (Fig. 5A), Lp/R/PNA476 was used to genotype the mapping population. The corresponding genomic locus was located on NA6 LG1 in the terminal location c. 7 cM from locus xLpesi3f (Fig. 5B).<br>
Expression profiling: RT-PCR?<br>
Expression in E. coli<br>
The sequence encoding DalRIPe and a putative orthologue of histone H3.2 was PCR adapted with the primer pairs<br>
5' CAGCTTGGATCCATGGCGAACTGCTGTCTGCTA3' (SEQ ID NO. 137) and 5' ACTCACAAGCTTAACCTCCTGTCACGACTTTGT 3' (SEQ ID NO. 138); and 5' AGGAGAGGATCCATGGCGCGTACCAAACAGACC 3' (SEQ ID NO. 139) and 5' TMTTGAAGCTTTTAGGCGCGTTCGCCACGGAT 3' (SEQ ID NO. 140) respectively. They were molecularly cloned into BamHI and Hindlll restricted pQE-30 (QIAGEN, Germany) and transformed into M15[pREP4]. To induce expression a culture was grown in the presence of ampicillin and kanamycin to mid-log phase, where upon IPTG to 1 mM was added and incubation continued for a further 4 h. Cells were harvested, resuspended in 1 ml extraction buffer and lysed by sonication. The lysate was incubated at 95°C for 10 min., spun at 16,060g for 5 min, the supernatant aliquoted, frozen in liquid N2, and stored at -80°C.<br><br>
Example 2<br>
Results<br>
Rl activity in D. antarctica and L. perenne is induced by cold acclimation and present in the apoplasm<br>
Rl assays reveal that D. antarctica has activity, induced by cold acclimation, to inhibit further growth of ice crystals following freezing. Given that total leaf extracts from plants grown at 22°C containing 1000 µg mL-1 of protein possess no Rl activity (Fig. 1) transfer of plants to 5°C for 2 weeks induces Rl activity by greater than 64 fold. Furthermore this activity is unaffected by incubation at 95°C for 5 min (Fig. 1). Rl activity is also induced more than 8 fold in the roots of D. antarctica in response to cold acclimation (Table 2). Similarly Rl activity in L perenne is below the threshold of detection in the leaves and roots of non-acclimated plants but is induced in excess of 16 and 4 fold respectively following cold acclimation (Table 2).<br>
Table 2. Rl activity3 in leaves and roots of non-acclimated (grown at 22°C) and cold acclimated (5°C) 0. antarctica and L. perenne.<br>
(Table Removed)<br>
aExpressed as lowest concentration of tota[ protein extract (µg mL-1) at which activity retained.<br>
b No activity detectable at 1000 and 800 µg mL-1 for leaves and roots respectively.<br><br>
Moreover Rl activity is present in the extracellular spaces of D. antarctica and L perenne. Leaf apoplastic extracts from plants of both species grown at 22°C possess no Rl activity whilst activity is induced in response to acclimation at least 73 fold in D. antarctica and 1.7 fold in L perenne (Table 3). These correspond to apoplastic protein concentrations, in cold acclimated plants, of 0.31 and 14 µg mL-1 respectively.<br>
Table 3. Rl activity3 in apoplastic extracts of leaves from non-acclimated (grown at 22°C) and cold acclimated (5°C) D. antarctica and L perenne.<br>
(Table Removed)<br>
3 Expressed as the lowest equivalent wet weight of starting plant material per volume of extract (mg mL-1) at which activity retained. b No activity detectable at 6,550 and 6,590 mg mL-1 for D. antarctica and L perenne respectively.<br>
Therefore activity to inhibit the consolidation of ice crystals by recrystallisation is induced in response to cold acclimation in both leaves and roots of D. antarctica, and to a significantly lesser extent in L perenne. Moreover a significant proportion of this Rl activity, particularly in D. antarctica, is localised to apoplastic spaces.<br>
IRIP orthologues from D. antarctica are predicted to be secreted proteins and contain two types of repeat motif.<br>
Full length clones of the putative IRIP orthologues DalRlPa, and DalRIPd, e7 and e8 were obtained from D. antarctica cDNA and genomic resources respectively. Two genomic clones encoding putative IRIP paralogues LplRIP a and b were also obtained from L perenne. In addition many IRIP related sequences have been identified in EST collections from other cereals and grasses.  The repeat structures of the longest IRIP<br><br><br>
orthologue, HvIRIP from Hordeum vulgare (barley), and the shortest, DalRIPd, are shown in Figure 2A. In all predicted IRIP orthologues (Fig. 2B, C), the C-terminal approximate 120 residues consist entirely of 16 tandem repeats of a degenerate 7-8 5 amino acid residue motif (the "IRIP repeat") (Fig. 2A). The consensus sequence for the IRIP repeat is (shown in SEQ ID NO: 141), with the glycine residue being most conserved (91.9% identity) across all forms. A multiple sequence alignment (Fig. 2B) indicates that relative to IRIP forms in other species IRIPs from L. perenne lack the 14th IRIP repeat but have an additional highly degenerate repeat 10 immediately N-terminal to the usual start position of the IRIP domain (Fig. 2B, C).<br>
Database sequence similarity searches with IRIP sequences reveal that the region N-terminal to the IRIP domain is related to proteins with leucine rich repeat (LRR) motifs. Most closely related is a putative Oryza sativa orthologue of a phytohormone receptor, the phytosulfokine receptor (PSKR) (NP_911036). Regions of significant sequence similarity with IRIPs extend in a discontinuous fashion through the first 17 LRRs of the putative PSKR orthologue and approximately 22 residues into a 36 amino acid residue "island" domain (Li and Chory (1997) Cell 90:929), where similarity ceases with the advent of the IRIP domain (Fig. 2B, C). With reference to the organisation of LRRs in the O. sativa PSKR orthologue IRIPs lack between 8 and 16 of the 17 LRRs (Fig. 2B, C).<br>
Phylogenetic analysis on the sequences of IRIP orthologues outside of the IRIP domain, together with representative LRR containing proteins reveal that IRIPs fall into a highly robust and distinct clade (Fig. 2D). The sister group to the IRIP clade includes PSKR orthologues and contains LRR receptor-like kinases (LRR-RLKs) of 25 both monocot and dicot origin (Fig. 2D). The most distant clade in this analysis includes polygalacturonase-inhibiting protein (PGIP) orthologues, and a LRR containing AFP from D. carota (Fig. 2D).<br>
Immediately N-terminal to the LRRs in the predicted IRIPs is a highly conserved 10 amino acid residue motif (shown in SEQ ID NO: 145) containing 3 30 invariant cysteine residues (Fig. 2A, B). An additional invariant cysteine residue occurs a further 31-32 residues proximal to the N-terminus (Fig. 2A, B). The corresponding cysteine<br><br>
residues have been shown to form 2 conformationally critical disulphide bridges in the structural determination of the LRR-RLK PGIP of Phaseolus vulgaris (Di Matteo et al (2003) Proc. Natl. Acad Sci. USA 100:10124). It is surmised based on their conservation in all IRIP forms that the orthologous residues also participate in structurally important disulphide bonds.<br>
At the N-terminus of all IRIP orthologues is a 20 or 21 amino acid residue region predicted to function as a signal peptide, with a cleavage site between conserved alanine, and threonine or valine residues (Fig. 2B). Consistent with this the mature versions of all full length IRIP forms are predicted to be extraceliularly localised. Thus it is likely that IRIPs, are predominantly apoplastic.<br>
Structural modelling of IRIPs predict conformations that are lattice matched to ice surfaces<br>
Three-dimensional structures of DalRlPa and LplRlPa (Fig. 3) were constructed by comparative homology modelling. The structural model of DalRlPa has three main regions, the double disulphide bonded N-terminal domain, one LRR loop and the IRIP domain (Fig. 3A). LplRlPa is similar to the DalRIPe in overall structure except for its three LRRs, and additional predicted disulphide bond (Fig. 3B).<br>
The extended ß-roll structure of the IRIP domain is predicted to form two surfaces complementary to the prism face of ice, on alternate sides of the domain. Adjacent parallel ß-loops are spaced approximately 4.5 A apart (Fig. 3A, B), whilst threonine and other solvent-accessible residues are arrayed in two ranks on the ß-strand faces, spaced 2 residues, or approximately 7.4 A apart (Fig. 3C). This almost exactly matches the prism ice surface that has repeating structures 4.5 A parallel to the a-axis, and 7.35 A parallel to the c-axis of ice. The putative ice-binding surfaces are stabilised by valine residues that allow tight regular hydrophobic packing of the central core of the ß-roll region, and by asparagine residues that participate in hydrogen bonds between adjacent ß-strands (Fig. 3D). The highly conserved glycine residues in the IRIP repeat<br><br>
are structurally important as they form the turns between the upper and lower ß-strand faces of the ice-binding ß-roll.<br>
Although the conformation of the LRR regions of the DalRlPa and LplRlPa models recapitulate the right-handed ß-roll of the IRIP repeat they do not engender a flat ß-sheet roll. On one side of the ß-roll adjacent parallel ß-loops are spaced approximately 4.5 A apart, but on the other, adjacently α-helical strands cannot pack as closely (Fig. 3A, B). Therefore with each iteration of the LRR the ß-roll structure becomes increasingly curved, displaying a concave ß-sheet region. As a consequence relative to DalRlPa the predicted LplRlPa structure with three LRRs appears to display a less than optimal surface lattice match to the ice prism face (Fig. 3B).<br>
Genomic organisation of IRIP related sequences in D. antarctica and L. perenne<br>
Consistent with the multiple IRIP gene variants isolated from them, the genomes of D. antarctica and L perenne both appear to harbour multiple IRIP-related sequences. Up to 5 hybridising bands are detected by interrogation of a Southern blot of D. antarctica genomic DNA with DalRIPe (Fig. 4A). The occurrence of as few as 2 hybridising bands (tracks 3 and 5, Fig. 4A), is evidence that IRIP related sequences may be physically linked on 2 genomic fragments totalling 20kbp. At least 4 LplRlPa sequence related restriction fragments are detected in a sample genome from a heterogeneous breeding population of L. perenne (Fig. 4B). The isogenic genomic DNA from a doubled haploid plant, however, exhibits only one strongly hybridising band, with a background of less intense bands (Fig. 4C).<br>
SNP-based genetic mapping of LplRIPb<br>
The genetic map position of LplRIPb was determined using single nucleotide polymorphism (SNP). Genetic map information for the LplRIPb gene was obtained from the second generation perennial ryegrass genetic mapping population derived from a pair-cross between the genotypes North Africans (NA6) and Auroras (AU6). A total of 8 SNP loci in LplRIPb (Fig. 5A) showed structures consistent with a single gene copy, but<br><br>
of these 6 showed AA x BB patterns, and could not be genetically mapped In the F1(NA6 x AU6) sib-ship. Of the two loci revealing polymorphism within the NA6 parental genotype (Lp/R/PNA476 and Lp/R/PNA694) (Fig. 5A), Lp/R/PNA476 was used to genotype the mapping population. The corresponding genomic locus was located on NAe LG1 in the terminal location c. 7 cM from locus xLpesi3f (Fig. 5B).<br>
The DNA sequence from LplRIPb was used to detect putatively orthologous wheat ESTs that had been assigned to the wheat deletion map (Endo and Gill (1996) Journal of Heredity 87:95; Qi et al (2003) Functional and Integrative Genomics 3:39). The three highest matching ESTs based on TBLASTX analysis were all assigned to deletion bins on chromosomes 4AL, 5BL and 5DL (BE48991: E = 7 x 10-71; BF200590: E = 5 x 10-58; BG314423: E = 2 x 10-41). The next lowest matching wheat EST (BG607348: E = 2 x 10-40) detected a deletion bin on chromosome 1BL, as well as 5BL.<br>
Expression analysis of LplRIPs and DalRIPs<br>
The modulation of steady state levels of IRIP gene transcripts in response to temperature in D. antarctica and L perenne, were investigated. A Northern blot. comprising RNA samples extracted from the roots and leaves of D. antarctica plants grown at 22°C, 4°C and -16°C probed with full length DalRIPe7 detects appreciable levels of transcript only in leaves of cold-acclimated plants (Fig. 6A). Quantitative analysis reveal that transfer of plants to 4°C for 2 weeks increases steady state IRIP mRNA levels approximately 47 fold relative to those grown at 22°C. By contrast LplRlPa transcript levels in L perenne are below the threshold level of detection in leaves, but elevated approximately 4 fold in the roots of cold-acclimated plants relative to those grown at 22°C (Fig. 6B). Thus steady state levels of IRIP transcripts are greatly elaborated in leaves of D. antarctica in response to cold-acclimation, but only moderately so in the roots of L. perenne.<br>
Heterologously expressed DalRIPe7 possesses Rl activity<br>
Extract from E. coli expressing DalRIPe was assayed for Rl activity.   Whilst extracts from cells expressing a putative D. antarctica orthologue of histone H3.2 at a<br><br>
concentration of 400 µg mL-1 of protein possess no Rl activity, those expressing DalRIPe7 retain activity down to 100 µg mL-1 (Fig. 7). Therefore activity to inhibit further growth of ice crystals following freezing is specifically conferred by DalRIPe7 and can account, in some part, for this activity in pianta.<br>
Example 3<br>
Discussion<br>
A physiological and functional genomics study in D. antarctica has resulted in the identification and characterisation of a gene family encoding IRIPs, the actions of which can account for its tolerance of freezing. D. antarctica has activity induced by cold acclimation, and present in the apoplasm, to inhibit ice recrystallisation thereby minimising the catastrophic plasmolytic consequences of uncontrolled ice crystal growth. This capacity is correlated with the expression of IRIP genes, and the primary structure, conformation, localisation, and most significantly, the activity of their products.<br>
IRIP genes encode proteins with two types of potential ice binding domains<br>
We have isolated and characterised putative full length IRIP genes from both D. antarctica and L. perenne. The form reported previously from L perenne (Sidebottom et al (2000) Nature 406:256), isolated as a protein associated with Rl activity, lacked an N-terminal methionine, and was comprised solely of 16 IRIP repeats. The IRIP forms reported here, unlike any other known AFPs, include 2 ice binding domains, the IRIP and LRR domains. Ten LRRs are also the predominant feature of an unrelated (Fig. 2D) ice recrystallisation inhibiting AFP from D. carota (Worrall et al. (1998) Science 282:115; Meyer et al. (1999) FEBS Lett. 447:171).<br>
Although the LRR domain has the potential to function in ice binding, in the various IRIP forms described here, there is a wide range of variation in its relative contribution to the overall primary structure. Thus whilst all IRIPs contain 16 IRIP repeats, LRRs are present from as many as 9 iterations in the H. vulgare form HvIRIP to as few as one in the D. antarctica forms, and in the extreme case of DalRIPd the residues with similarity<br><br>
to the LRR number only 17 of the usual complement of 24 or 25 residues (Fig. 2A, B, C).<br>
Despite the apparent plasticity in the number of LRRs, and even their dispensability, other features commonly found in LRR proteins, including the region predicted to participate in 2 disulphide bridges, and the probable signal sequence are invariant in IRIPs, suggesting that they are important for structure/function and/or localisation.<br>
What is evolutionary origin of IRIP genes?<br>
All plant (and animal) AFPs characterised to date appear to have arisen relatively recently in evolutionary terms by the co-option of existing protein structures (Logson and Doolittte (1997) Proc Natl Acad Sci U S A. 94:3485). This is also true of the IRIPs characterised in this study. Outside the IRIP domain itself IRIPs are structurally related to LRR-RLKs, having greatest affinity with orthologues of PSKR, a receptor for the plant hormone phytosulfokine. PSKR first isolated and characterised in Daucus carota (carrot) consists of an extracellular domain containing 21 LRRs, a single pass transmembrane domain, and a cytoplasmic serine-threonine kinase domain (Matsubayashi ef a/. (2002) Science. 296:1470). Phytosulfokine js a secreted 5 residue sulfated peptide with a key role in cellular de-differentiation and redifferentiation (Matsubayashi and Sakagami (1996) Proc Natl Acad Sci U S A. 93:7623). It is difficult to conceive the mechanistic connection between hormonal regulation of cell fate determination and antifreeze activity. A more likely scenario is that a PSKR related protein was co-opted as an AFP, either because of intrinsic structural complementarity to ice crystals, but more credibly as a vehicle to target the "hitch hiking" IRIP domain to the cellular compartment where Rl activity is critical, the apoplast. It is possible to envisage an evolutionary scenario whereby a PSKR-like LRR-RLK protein has acquired a novel domain, the IRIP repeat domain, in the process losing its transmembrane and intracellular kinase domains, thereby becoming an untethered apoplastic protein with a novel function. Moreover PSKR-like genes might have been predisposed for such a role because of their expression in organ primordia, cells of which are particularly vulnerable to freezing induced damage. Like the IRIPs, the majority of known plant AFPs are derived from secreted proteins, many being orthologues of pathogenesis-<br><br>
related proteins (Griffith and Yaish (2004) Trends Plant Sci. 9:399), and one, the AFP from D. carota (Worrall et al. (1998) Science 282:115; Meyer et al. (1999) FEBS Lett. 447:171) also being related to LRR-RLKs.<br>
IRIPs exhibit plasticity in the number and arrangement of LRRs, more so when compared to their presumed nearest relatives the PSKRs. Thus, there are 17 LRRs in D. carota PSKR and its presumed homologue in O. sativa, 9 in HvlRIP, 3 in the L perenne and T. aestivum IRIP versions, and one, or part of one, in the D. antarctica forms (Fig. 2B, C). Such evolutionary plasticity in number and arrangement of LRRs has been noted in analyses of LRR-RLKs (see for example Dixon et al. (1998) Plant Cell. 10:1915)). All the LRRs in extant IRIP forms have high levels of identity and conservation with LRRs in the PSKRs, with no evidence for the addition of any other sequences, LRR or otherwise. Therefore in respect of the principle of maximum parsimony it is most likely that the evolution of IRIPs has resulted from the progressive loss of LRRs rather than their acquisition or rearrangement.<br>
Because of their evolutionarily recent co-option as AFPs all known plant AFPs exhibit in their sequences clear affinities to particular classes of protein. Although this is true for the PSKR affinities of the LRR.related N-terminal part of the IRIPs, the other potential ice binding motif in IRIPs, the IRIP repeat, exhibits no sequence similarity to any reported nucleotide or amino acid sequences. For this reason its origin is unknown. Because of its length, the shortest repeat known in an AFP, the IRIP repeat could conceivably be derived from a simple repetitive element of as few as 21 nucleotides in length, such as those found in intergenic DNA. However to date BLASTN searches have failed to identify any closely related genomic sequences.<br>
Genes encoding IRIP orthologues appear to be monophyletic in origin and confined to the sub-family Pooideae. No sequences related to the IRIP domain have been found in sequence similarity searches of any dicots including Arabidopis thaliana, in the genome of O. sativa, or the extensive EST resources derived from Zea mays or Sorghum bicolor. Furthermore the IRIP clade (Fig. 3B) is highly distinct and deeply rooted supporting the notion that IRIPs arose once early in the evolutionary history of the Pooideae and have subsequently diverged in both copy number and structure. On this<br><br>
basis IRIP genes are predicted to have arisen sometime after the divergence of the Pooideae and Panicoideae 60 mya, but before that of the Triticodae and Poodae 35 mya (Huang et a/, (2002) Plant Mol Biol. 48:805).<br>
Structural modelling predicts that IRIP repeat has greater affinity for ice than LRR<br>
We have used comparative homology modelling to devise a theoretical 3-D structure for full length IRIPs. A truncated version of LplRIP modelled previously (Kuiper et a/ (2001) Biophys. J. 81: 3560) did not include the LRR domain, nor the twin disulphide bond-forming N-terminal domain. The structural model demonstrates that both the IRIP and LRR domains can contribute to a common, structurally complementary ice-binding domain.<br>
The predominant ice-binding region is predicted to be the IRIP domain, which presents two ice-binding faces, on either side of the ß-roll domain.   The putative ice binding surfaces however are not as regular.as the stereotypical threonine-X-threonine motifs in beta-roll configurations observed in two unrelated insect AFPs with high TH activity (Graether et al (2000) Nature. 406:325; Liou et al, (2000) Nafure.406:322)..   The D. antarctica and L. perenne IRIPs exhibit only 30 to 40% threonine at the analogous positions. This is likely due to the differences in the primary function of the proteins. Insect AFPs must provide appreciable TH activity as most insects are not freeze-tolerant. The regularity of the threonine residues on the presenting ice binding surfaces has been implicated in their high TH activity, TH activity having been shown to rapidly decrease with increasing mutational substitution of residues in the ice binding surface (Marshall et al (2002) FEBS Lett. 529:261).   By contrast since D. antarctica is freeze tolerant the primary purpose of AFPs in this organism would be to provide Rl activity, to avoid the plasmolytic consequences of continued ice crystal growth in already frozen   tissue.<br>
In fact, IRIPs may have evolved to have low TH activity as high activity may prove detrimental during the inevitable seasonal freezing of these plants. If a plant were to deploy an IRIP with a relatively high TH activity, the apoplastic fluid of the plant would remain liquid until the temperature dropped below the lower end of the TH gap.<br><br>
Freezing would then occur much more rapidly than if initiated close to the freezing equilibrium point, and would do so with the spicular dendritic growth observed with other AFPs, potentially doing much mechanical damage to cells.<br>
The LRR domains of full-length IRIPs are also predicted to contribute to ice binding surfaces but not with the inherent structural complementary to the prism face of ice of the IRIP domain. Whilst solved crystal structures of LRRs in proteins form parallel ß-sheets on one side of a ß-roll, the other side is made up of adjacently packed α-helical strands (Di Matteo et al (2003) Proc. Natl. Acad Sci. USA 100:10124). As the α-helical regions cannot pack as closely as the ß-sheet regions the p roll structure will curve, displaying a concave ß-sheet region, proportional to the numbers of LRRs. Extended curved ß-sheet surfaces of LRR regions do not present an optimal surface lattice match to ice, although the AFP from D. carota consists predominantly of 10 LRRs. Indeed globular type III fish AFP also does not have an obvious regular ice binding surface and yet displays reasonable TH and Rl activity (Baardsnes and Davies, (2002) Biochim Biophys Acta. 1601:49).<br>
Genetic mapping of LplRIP and relationship to syntenic cold tolerance and vernalisation QTLs  <br>
In order to determine the location of IRIP genes in the genome of L. perenne and their proximity to endogenous or syntenic cold tolerance and vernalisation quantitative trait loci (QTLs) LplRIPb was genetically mapped using single nucleotide polymorphism (SNP).<br>
The analysis of SNP variation in LplRIP gene(s) revealed a high level of variation, even compared to average values observed over a large sample of perennial ryegrass. This observation, along with the excess of recovered haplotype structures, is strongly suggestive of multiple gene structure. Although Southern hybridisation analysis indicated a relatively simple genome organisation (Fig. 5C), a number of minor bands were observed, which may correspond to paralogues of the LplRlPa gene. The segregating Lp/R/PNA476 SNP locus may identify such a paralogous sequence, based<br><br>
on the results of comparative genetics and genomics analysis. Macrosynteny based on heterologous RFLP markers has demonstrated a broad correspondence between each of the perennial ryegrass linkage groups and each of the homoeologous groups of wheat (Jones et al. (2002)T/7eoref/ca/ and Applied Genetics 105: 577). On this basis, the SNP locus location on perennial ryegrass LG1 should correspond to a region of conserved synteny with the wheat homoeologous 1S chromosomes.     However, comparative genomics has identified wheat sequences assigned to the group 4L and 5L chromosomes, which are related through evolutionary translocations (Devos et al., (1995) Theoretical and Applied Genetics 91:282). The location of LplRIP ortholoci on Triticeae group 5 chromosomes is also consistent with the detection of QTLs for winter hardiness and frost tolerance on these chromosomes in wheat (Sutka (1994) Euphytica 77:277; Galiba et al., (1995). Theoretical and Applied Genetics 90:1174; Galiba et al., (1997) Theoretical and Applied Genetics 95:265; Toth et al. (2003) TheorAppI Genet 107:509) and barley (Pan ef al. (1994) Theoretical and Applied Genetics 89: 900; Francia et al. (2004) Theoretical and Applied Genetics 108: 670; Reinheimer et al, (2004) Theoretical and Applied Genetics 109: 1267), in the same region as the vernalisation response genes that control heading date. Based on conserved synteny, this would predict a location in perennial ryegrass on the upper part of LG4 (Yamada et al.  (2004) Crop Science 44: 925).    The detection of wheat IRIP paralogues on chromosome 1BL, however, indicates the complexity of this gene family in wheat, and suggests that paralogous sequences may be located on other LGs in perennial ryegrass as well. In this interpretation, the non-segregating SNPs may identify variation between LG1  and LG4-located paralogues.    In order to test this hypothesis, it would be necessary to identify polymorphic SNPs for the second gene copy in other germplasm. Another possibility is that the LG1 -located xLpiripna476 locus identifies a non-syntenic region.   The ends of each LG in perennial ryegrass were enriched for non-syntenic markers, as previously shown for other Poaceae species (Jones et al (2002) Theoretical and Applied Genetics 105:577). The closest marker to xLpiripna476 is an EST-RFLP marker, xLpesi3f (Faville ef al., Theor Appl Genet, in press), which preferentially detected wheat ESTs allocated to deletion bins on chromosome 4A. Finally, it is to be understood that various alterations, modifications and/or additions may be made without departing from the spirit of the present invention as outlined herein.<br><br>
Documents cited in this specification are for reference purposes only and their inclusion is not acknowledgment that they form part of the common general knowledge in the relevant art.<br><br><br><br><br><br><br><br><br>
We claim:<br>
1.	An isolated ice recrystallisation inhibition protein derived from a plant wherein said protein includes:<br>
an IRIP domain including 3 to 32 repeats of the amino acid sequence set forth in SEQ ID NO: 141;<br>
and<br>
an amino acid residue motif as set forth in SEQ ID NO: 145.<br>
2.	The isolated ice recrystallisation inhibition protein as claimed in claim 1 wherein the IRIP domain includes 15 or 16 tandem repeats of the amino acid sequence set forth in SEQ ID NO: 141.<br>
3.	The isolated ice recrystallisation inhibition protein as claimed in claim 1 or 2 wherein the protein further includes 3 or fewer Leucine Rich Repeat (LRR) motifs.<br>
4.	The isolated ice recrystallisation inhibition protein as claimed in any one of claims 1 to 3 wherein the protein includes one or fewer LRR motifs.<br>
5.	The isolated ice recrystallisation inhibition protein as claimed in any one of claims 1 to 4 wherein an LRR motif includes the amino acid sequence set forth in SEQ ID NO: 128 or SEQ ID NO: 143.<br>
6.	The isolated ice recrystallisation inhibition protein as claimed in any one of claims 3 to 5 wherein the LRR motif(s) are positioned N-terminal to the IRIP domain.<br>
7.	The isolated ice recrystallisation inhibition protein as claimed in any one of claims 3 to 6 wherein the amino acid residue motif as set forth in SEQ ID NO: 145 is positioned N-terminal to the LRR motif(s).<br>
8.	The isolated ice recrystallisation inhibition protein as claimed in any one of claims 1 to 7 wherein the protein includes an amino acid sequence selected from the list consisting of: SEQ ID NO: 17, SEQ ID NO: 22, SEQ ID NO: 31, SEQ ID NO: 38, SEQ ID NO: 46 and SEQ ID NO: 54.<br>
9.	The isolated ice recrystallisation inhibition protein as claimed in any one of claims 1 to 8 wherein the plant is a Deschampsia species.<br><br>
10.        The isolated ice recrystallisation inhibition protein as claimed in any one of claims 1 to 9 wherein the plant is Deschampsia Antarctica.<br>
11.	A recombinant nucleic acid including a nucleotide sequence which encodes an isolated ice recrystallisation inhibition protein as claimed in any one of claims 1 to 10.<br>
12.	The recombinant nucleic acid as claimed in claim 11 wherein the nucleic acid comprises a nucleotide sequence selected from the list consisting of: SEQ ID NOs: 1-16,18-21, 23-30, 32-37, 39-45, 47-53.<br>
13.	An isolated or substantially purified nucleic acid including a single nucleotide polymorphism (SNP) from a nucleic acid as claimed in claim 11 or 12.<br>
14.	A nucleic acid construct including one or more nucleic acids as claimed in any one of claims 11 to 13.<br>
15.	A method of modifying tolerance of freezing and/or low temperature stress in a plant, said method including introducing into said plant an effective amount of a nucleic acid as claimed in any one of claims 11 to 13, and/or a nucleic acid construct as claimed in claim 14.<br>
16.	A preparation for transforming a plant, the preparation comprising a nucleic acid as claimed in any one of claims 11 to 13, and/or a nucleic acid construct as claimed in claim 14.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUFic3RyYWN0LSgxMS0wMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Abstract-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1ERUxOUC0yMDA2LUFzc2lnbm1lbnQtKDE2LTAxLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-DELNP-2006-Assignment-(16-01-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNsYWltcy0oMDktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Claims-(09-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNsYWltcy0oMTEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Claims-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1ERUxOUC0yMDA2LUNsYWltcy0oMTQtMTEtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">3560-DELNP-2006-Claims-(14-11-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNsYWltcy0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Claims-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDYtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Correspondence Others-(06-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Correspondence Others-(09-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycyAxLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-correspondence-others 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Correspondence-Others-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Correspondence-Others-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDExLTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Description (Complete)-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUZvcm0tMS0oMDktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Form-1-(09-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUZvcm0tMTMtKDExLTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Form-13-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1ERUxOUC0yMDA2LUZvcm0tMTMtKDE0LTExLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">3560-DELNP-2006-Form-13-(14-11-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUZvcm0tMi0oMDktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Form-2-(09-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUZvcm0tMy0oMDYtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Form-3-(06-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUZvcm0tMy0oMDktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Form-3-(09-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUZvcm0tMy0oMTEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Form-3-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LUdQQS0oMTEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-GPA-(11-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU2MC1kZWxucC0yMDA2LVBldGl0aW9uIDEzNy0oMTEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3560-delnp-2006-Petition 137-(11-03-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="257601-an-adsorbent-for-an-oral-administration.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="257603-process-for-production-of-living-radical-polymers-and-polymers.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>257602</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3560/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Oct-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Oct-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AGRICULTURE VICTORIA SERVICES PTY. LTD.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>475 MICKLEHAM ROAD, ATTWOOD, VICTORIA 3049 AUSTRALIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JOHN ULRIK PETER</td>
											<td>19 RUCKER STREET, WESTGARTH VICTORIA 3070 AUSTRALIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SPANGENBERG GERMAN</td>
											<td>56 ARTHUR STREET, BUNDOORA VICTORIA 3083 AUSTRALIA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>POLOTONIANKA RENATA MARTINA</td>
											<td>117 EASY STREET, COLLINGWOOD VICTORIA 3066, AUSTRALIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/29</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU04/001633</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-11-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2003906477</td>
									<td>2003-11-24</td>
								    <td>Australia</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/257602-ice-recrystallisation-inhibition-protein-or-antifreeze-proteins-from-deschampsia-lolium-and-festuca-species-of-grass by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:58:39 GMT -->
</html>
