<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/257847-mutant-expandases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:17:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 257847:MUTANT EXPANDASES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MUTANT EXPANDASES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a mutant expandase that is a variant of a model polypeptide with expandase activity whereby the mutant expandase has an at least 2-fold improved in vitro expandase activity towards adipyl-6-APA in comparison with the model polypeptide with expandase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MUTANT EXPANDASES<br>
Field of the invention<br>
The present invention relates to mutant expandase enzymes, polynucleotides<br>
encoding such enzymes, to microorganisms transformed with said polynucleotides<br>
encoding said mutant expandase enzymes and with the use of such mutant expandase<br>
enzymes or such microorganisms in the ring expansion of 5-carboxypentanoyl-6-<br>
aminopenicillanic acid (adipyl-6-APA = ad-6-APA).<br>
Background of the invention<br>
Beta-lactam antibiotics constitute the most important group of antibiotic<br>
compounds with a long history of clinical use. Among this group, the prominent ones<br>
are the penicillins and cephalosporins. Penicillins are naturally produced by various<br>
filamentous fungi such as Penicillium (e.g. P. chrysogenum). Cephalosporins are<br>
naturally produced by various microorganisms such as Acremonium (e.g. A.<br>
chrysogenum) and Streptomyces (e.g. Streptomyces clavuligerus)<br>
As a result of classical strain improvement techniques, the production levels of<br>
the antibiotics in P. chrysogenum and A. chrysogenum have increased remarkably over<br>
the past decades. With the increasing knowledge of the biosynthetic pathways leading<br>
to penicillins and cephalosporins, and the advent of recombinant DMA technology, new<br>
tools for the improvement of production strains have become available.<br>
Most enzymes involved in p-lactam biosynthesis have been identified and their<br>
corresponding genes have been cloned, as can be found in Ingolia and Queener, Med<br>
Res Rev (1989) 9:245-264 (biosynthesis route and enzymes), and Aharonowitz,<br>
Cohen, and Martin, Ann Rev Microbiol (1992) 46:461-495 (gene cloning).<br>
The first two steps in the biosynthesis of penicillin in P. chrysogenum are the<br>
condensation of the three amino acids L-5-amino-5-carboxypentanoic acid (L-ccaminoadipic<br>
acid) (A), L-cystein (C) and L-valine (V) into the tripeptide LLD-ACV,<br>
followed by cyclization of this tripeptide to form isopenicillin N. This compound contains<br>
the typical (3-lactam structure.<br>
The third step involves the replacement of the hydrophilic side chain of L-5-<br>
amino-5-carboxypentanoic acid by a hydrophobic side chain by the action of the<br>
enzyme acyltransferase (AT),<br>
In EP-A-0448180 has been described that the enzymatic exchange reaction<br>
mediated by AT takes place inside a cellular organelle, the microbody. The observation<br>
that substantial quantities of deacetoxycephalosporin C (DAOC) can be formed by nonprecursed<br>
P. chrysogenum transformants expressing deacetoxycephalosporin C<br>
synthase (EC 1.14.20.1 - DAOCS, further indicated herein as expandase) implies the<br>
presence of significant amounts of penicillin N, the natural substrate for expandase, in<br>
P. chrysogenum (AM et al., J Antibiot (1995) 48:338-340). However, the D-a-aminoadipyl<br>
side chains of DAOC cannot be easily removed.<br>
Cephaiosporins are much more expensive than penicillins. One reason is that<br>
some cephalosporins (e.g., cephalexin) are made from penicillins by a number of<br>
chemical conversions. Another reason is that, so far, only cephaiosporins with a D-aamino-<br>
adipyl side chain could be tormented. Cephalosporin C, by far the most<br>
important starting material in this respect, is very soluble in water at any pH, thus<br>
implying lengthy and costly isolation processes using cumbersome and expensive<br>
column technology. Cephalosporin C obtained in this way has to be converted into<br>
therapeutically used cephaiosporins by a number of chemical and enzymatic<br>
conversions.<br>
The methods currently favored in industry to prepare the intermediate 7-<br>
amino-deacetoxycephaloporanic acid (7-ADCA) involve complex chemical steps<br>
leading to the expansion and derivfltization of penicillin G. One of the necessary<br>
chemical steps to produce 7-ADCA involves the expansion of the 5-membered<br>
penicillin ring structure to a 6-membered Cephalosporin ring structure (see for instance<br>
US 4,003,894). This complex chemical processing is both expensive and noxious to<br>
the environment.<br>
Consequently, there is a great desire to replace such chemical processes with<br>
enzymatic reactions such as enzymatic catalysis, preferably during fermentation. A key<br>
to the replacement of the chemical expansion process by a biological process is the<br>
central enzyme in the Cephalosporin biosynthetic pathway, expandase.<br>
The expandase enzyme from the bacterium Streptomyces clavuligerus (S.<br>
clavuligerus) was found to carry out, in some cases, penicillin ring expansions. When<br>
introduced into P. chrysogenum, it can convert the penicillin ring structure into the<br>
Cephalosporin ring structure, as described in Cantwell et al., Proc R Soc Lond B (1992)<br>
248:283-289. The expandase enzyme has been well characterized (EP-A-0366354)<br>
both biochemical and functional, as has its corresponding gene. Both physical maps of<br>
the cefE gene (the gene encoding the expandase enzyme of S. clavuligerus - EP-A-<br>
0341892), DNA sequence and transformation studies in P. chrysogenum with cefE<br>
have been described. The DNA and amino acid sequence of the S. clavuligerus<br>
expandase enzyme are represented in SEQ ID NO 1.<br>
Another source for an expandase enzyme is the bacterium Nocardia<br>
lactamdurans (N. lactamdurans, formerly S. lactamdurans). Both the biochemical<br>
properties of the enzyme and the DNA sequence of the gene encoding the enzyme<br>
have been described - see Cortes et al., J Gen Microbiol (1987) 133:3165-3174 and<br>
Coque et al., Mol Gen Genet (1993) 236:453-458, respectively. The DNA and amino<br>
acid sequence of the N. lactamdurans expandase enzyme are represented in SEQ ID<br>
NO 2.<br>
Recently, novel expandase (cefE) genes and enzymes have been found in<br>
Streptomyces jumonjinensis, Streptomyces ambofaciens and Streptomyces<br>
chartreuses - see Hsu et al. (2004), Appl. and Environm. Microbiol. 70, 6257-6263.<br>
Table 1 summarizes the amino acid sequence identities from several expandases and<br>
shows that the sequence identities range from 67 to 85%, depending on the source of<br>
the enzyme.<br>
Table 1 : Amino acid sequence identities of several expandases (data taken from Hsu et al).<br>
(Table Removed)<br>
In the biosynthesis of cephatosporins, which takes mainly place in prokaryotic<br>
cells, the deacetoxycephalosporin-C is subsequently converted to deacetylcephalosporin-<br>
C by the enzyme deacetylcephalosporin C synthase also named<br>
deacetoxycephalosporin-C hydroxylase or hydroxylase (EC 1.14.11.26 - DACS).<br>
Genes encoding such hydroxylases are named cefF-genes (e.g. see Hsu et al.) The<br>
expandase found in eukaryotic cells, e.g. Acremonium chrysogenurn can catalyze the<br>
direct conversion of penicillin N to deacetoxycephalosporin-C due to possession of<br>
both expandase and hydrolyase activity, hence the encoded gene is termed cefEF(see<br>
Hsu et al.),<br>
As defined herein, the term expandase relates to expandase enzymes (EC<br>
1.14.20.1, encoded by cefE genes) as well as expandases also possessing in addition<br>
the hydroxylase activity (EC 1.14.20.1 + EC 1.14.11.26 encoded by cefEF genes)<br>
Since the expandase enzyme catalyses the expansion of the 5-membered<br>
thiazolidine ring of penicillin N to the 6-membered dihydrothiazine ring of DAOC, this<br>
enzyme would be of course a logical candidate to replace the ring expansion steps of<br>
the chemical process. Unfortunately, the enzyme works on the penicillin N intermediate<br>
of the cephalosporin biosynthetic pathway, but not or very inefficiently on the readily<br>
available inexpensive penicillins as produced by P, chrysogenurn, like penicillin V or<br>
penicillin G. Penicillin N is commercially not available and even when expanded, its Da-<br>
amino-adipyl side chain cannot be easily removed by penicillin acylases.<br>
It has been reported that the expandase enzyme is capable of expanding<br>
penicillins with particular side chains to the corresponding 7-ADCA derivative. This<br>
feature of the expandase has been exploited in the technology as disclosed in EP-A-<br>
0532341, W095/04148 and W095/04149, In these disclosures the conventional<br>
chemical in vitro conversion of penicillin G to 7-ADCA has been replaced by the in vivo<br>
conversion of certain 6-aminopenicillanic acid (6-APA) derivatives in recombinant<br>
Penidllium chrysogenurn strains transformed with an expandase gene.<br>
More particularly, EP-A-0532341 teaches the in vivo use of the expandase<br>
enzyme in P. chrysogenurn, in combination with a adipyl side chain (further referred to<br>
as adipyl) as a feedstock, which is a substrate for the acyltransferase enzyme in P.<br>
chrysogenurn. This leads to the formation of adipyl-6-APA, which is converted by an<br>
expandase enzyme introduced into the P. chrysogenurn strain to yield adipyl-7-ADCA.<br>
Finally, the removal of the adipyl side chain is described, yielding 7-ADCA as a final<br>
product. Furthermore, EP-A-540210 teaches the similar production of adipyl-7-ADAC<br>
and adipyl-7-ACA.<br>
As an alternative approach, expansion of penicillin G using P. chrysogenurn<br>
transformed with cefE has been proposed in EP0828850. Furthermore, in order to<br>
increase the expansion of penicillin G the use of mutant cefE gene of S. clavuligerus<br>
has been described. US 5,919,680, WO 98/02551, WO 99/33994, WO 01/85951 and<br>
EP 1348759 all disclose mutant cefE genes allegedly coding for enzymes having a<br>
higher expandase activity on penicillin G.<br>
Nevertheless, these mutant expandases still do not provide for a commercially<br>
attractive process for the production of cephalosporins.<br>
Therefore, there is a need for further improving the process wherein<br>
cephalosporins are prepared from expansion of adipyl-6-APA. This process is<br>
characterized by an efficient removal of the side chain material from the cephalosporin<br>
core material. One step to further improve this process is improvement of the<br>
expandase activity on adipyl-6-APA.<br>
Description of the Figures<br>
1/3. A&amp; B Amino acid alignment of improved expandase mutants; SCLAVEIW<br>
represents amino acid sequence of expandase of S. clavuligerus; NOCAEIW<br>
represents amino acid sequence of expandase of N. lactamdurans; For the<br>
other sequences the notation is as follows: 309EIWIT represents amino acid<br>
sequence of expandase of mutant expandase H309, and so son.<br>
2/3. Schematic representation of vector plATWAn<br>
3/3. HPLC analysis of expandase assay mixture according to Example 3<br>
A. at t = 0 minutes<br>
B. at t = 30 minutes<br>
Detailed description of the invention<br>
In a first aspect, the invention provides a mutant expandase that is a variant of<br>
a model polypeptide with expandase activity. Preferably, the invention provides a<br>
mutant expandase, whereby the mutant expandase has an at least 2-fold improved in<br>
vitro expandase activity towards adipyl-6-APA in comparison with the model<br>
polypeptide with expandase activity. The determination of the in vitro expandase<br>
activity towards adipyl-6-APA is described in detail in the Materials and Methods<br>
(Assay (1) - (3)). More preferably the in vitro expandase activity towards adipyl-6-APA<br>
of the mutant expandase is improved at least 2.5-fold, more preferably at least 3-fold,<br>
more preferably at least 4-fold, more preferably at least 5-fold, more preferably at least<br>
6-fold, more preferably at least 7-fold, more preferably at least 8-fold, more preferably<br>
at least 9-fold, more preferably at least 10-fold, more preferably at least 11-fold.<br>
In some of the mutant expandase enzymes according to the invention parts of<br>
the amino acid sequence of the expandase of S. clavuligerus have been replaced by<br>
corresponding parts of the amino acid sequence of expandase of N. lactamdurans. The<br>
amino acid sequence of the Ce/E gene of N. lactamdurans strain ATCC27382 is<br>
represented in SEQ ID NO: 2.<br>
In the description of these mutant expandases the amino acids are numbered<br>
according to the numbering of the homologous parts of the amino acid sequence of the<br>
expandase of S. clavuligerus. The hotnology of the expandase amino acid sequences<br>
of S. clavuligerus and N. lactamdurans is to be derived from Figure 1 and Table 1,<br>
With "altered or mutant expandase" in the context of the present invention is<br>
meant any enzyme having expandase activity, which has not been obtained from a<br>
natural source and for which the amino acid sequence differs from the complete amino<br>
acid sequences of the natural expandase enzymes of S. clavuligerus and N.<br>
lactamdurans.<br>
The invention also provides a mutant expandase which is preferably being<br>
modified at least at an amino acid position selected from group 1 consisting of positions<br>
2, 18, 59, 73, 74, 89, 90, 99, 101, 105, 112, 113, 155, 170, 177, 209, 213, 217, 244,<br>
249, 251, 277, 278, 280, 281, 284, 293, 300, 307 and 311 using the amino acid<br>
position numbering of the amino acid sequence of the expandase enzyme encoded by<br>
the cefE gene of Streptomyces clavuligerus. The nucleotide sequence of the cefE gene<br>
of Streptomyces clavuligerus as well as the amino acid sequence encoded by said<br>
cefE gene are depicted in SEQ ID NO: 1. More preferably, the mutant expandase is<br>
modified at least at an amino acid position selected from group 2 consisting of positions<br>
2,18,59,89,90,99, 101, 105, 112, 113,170,177,209,213,217,249,251,278, 280,<br>
284 and 293. The mutant expandase may also have been modified at least at an amino<br>
acid position selected from the group 1 which does not form part of group 2, together<br>
with at least an amino acid position selected from the group 2.<br>
A highly preferred mutant expandase is modified at least at amino acid<br>
position 89, in combination with any of the amino acid positions of the expandase,<br>
preferably with an amino acid position selected from group 1 or an amino acid position<br>
selected from group 2 or an amino acid position selected from group 1 which does not<br>
form part of group 2, together with at least an amino acid position selected from the<br>
group 2.<br>
Most preferably, the invention provides a mutant expandase whereby the<br>
mutant expandase has an at least 2-fold improved in vitro expandase activity towards<br>
adipyl-6-APA in comparison with the model polypeptide with expandase activity, more<br>
preferably at least 2.5-fold, more preferably at least 3-fold, more preferably at least 4-<br>
fold, more preferably at least 5-fold, more preferably at least 6-fold, more preferably at<br>
least 7-fold, more preferably at least 8-fold, more preferably at least 9-fold, more<br>
preferably at least 10-fold, more preferably at least 11-fold and whereby the mutant<br>
expandase is preferably being modified at at least an amino acid position selected from<br>
the group 1 defined above using the amino acid position numbering of the amino acid<br>
sequence of the expandase enzyme encoded by the cefE gene of Streptomyces<br>
clavuligerus. More preferably, the mutant expandase is modified at least at an amino<br>
acid position selected from the group 2. The mutant expandase with the improved<br>
expandase activity may also have been modified at least at an amino acid position<br>
selected from the group 1 together with at least at an amino acid position selected from<br>
the group 2,<br>
A highly preferred mutant expandase with improved expandase activity is<br>
modified at least at amino acid position 89, in combination with any of the amino acid<br>
positions of the expandase, preferably with an amino acid position selected from group<br>
1 or an amino acid position selected from group 2 or an amino acid position selected<br>
from group 1 together with at least at an amino acid position selected from group 2.<br>
The modification at an amino acid position may comprise a substitution by<br>
another amino acid, selected from the group of 20 L-amino acids that occur in Nature -<br>
see Table 2. Alternatively, the modification at an amino acid position may comprise a<br>
deletion of the amino acid at said position. Furthermore, the modification at an amino<br>
acid position may comprise a substitution of one or more amino acids at the C-terminal<br>
or N-terminal side of said amino acid.<br>
(Table Removed)<br>
The model polypeptide with expandase activity as used in the present<br>
invention is selected from the group consisting of a polypeptide with expandase<br>
activity, preferably having an amino acid sequence according to SEQ ID NO: 1 or<br>
having an amino acid sequence according to SEQ ID NO: 2 and polypeptides with<br>
expandase activity having an amino acid sequence with a percentage identity with SEQ<br>
ID NO: 1 of at least 70%, preferably at least 75%, more preferably at least 80%, more<br>
preferably at least 85%, more preferably at least 90%, most preferably at least 95%, or<br>
with SEQ ID NO: 2 of at least 70%, preferably at least 75%, more preferably at least<br>
80%, more preferably at least 85%, more preferably at least 90%, most preferably at<br>
least 95%, such as the expandase enzymes that are summarized in Table 1. Most<br>
preferred as model polypeptide with expandase activity as used in the present<br>
invention is a polypeptide with expandase activity, having the amino acid sequence<br>
according to SEQ ID NO: 1 or having the amino acid sequence according to SEQ ID<br>
NO: 2.<br>
The present invention preferably provides mutant expandases that have<br>
modifications at least at<br>
o 2 or more amino acid positions selected from the group consisting of 2, 89, 277,<br>
281 and 300, more preferably at positions 2+281 or 89+281 or 277+300.<br>
o 3 or more amino acid positions selected from the group consisting of 2, 89, 281,<br>
293 and 311, more preferably at positions 2+89+281 or 89+281+311 or<br>
89+281+293.<br>
o 4 or more amino acid positions selected from the group consisting of 2, 73, 89,<br>
90, 217, 244, 277, 280, 281, 306, 307 and 311, more preferably at positions<br>
2+277+280+281 or 2+281+307+311 or 73+89+281+311 or 89+217+281+311 or<br>
89+244+281+311 or 89+281+307+311 or 277+281+306+311 or<br>
89+281+306+311 or 90+281+306+311.<br>
o 5 or more amino acid positions selected from the group consisting of 2, 73, 89,<br>
90,155, 213, 249, 278, 281, 293, 300, 306, 307 and 311, more preferably at<br>
positions 2+89+281+306+311 or 2+155+281+306+311 or 73+89+213+281+311<br>
or 89+78+218+307+311, 89+281 +293+300+311 or 89+249+281 +307+311.<br>
o 6 amino acid positions selected from the group consisting of<br>
90+105+113+281+306+311.<br>
o 7 or more amino acid positions selected from the group consisting of 2, 73, 89,<br>
105,113, 155, 170,177, 251, 277, 280, 281, 293, 300, 306, 307 and 311, more<br>
preferably 73+89+281+293+300+307+311 or<br>
105+113+155+177+281 +306+311 or 155+177+277+280+281 +306+ 311.<br>
o 8 or more amino acid positions selected from the group consisting of 2, 89, 90,<br>
99,105,113,155,177, 277, 281, 307 and 311 more preferably at<br>
2+90+99+105+113+281 +306+311 or 2+90+105+113+155+177+277+281<br>
o 9 amino acid positions selected from the group consisting of<br>
2+90+105+113+155+177+281+306+311.<br>
o 10 amino acid positions selected from the group consisting of 2, 59, 90,101,<br>
105,113,155,177, 209, 277, 281, 307 and 311 more preferably at<br>
2+90+105+113+155+177+277+281 +306+311.<br>
o 11 amino acid positions selected from the group consisting of 2, 90,105,113,<br>
155,177,277,281,293,307,311.<br>
The present invention relates to substitutions of certain of the amino acids of<br>
expandase. Preferably such substitutions involve replacement of:<br>
• histidine at position 18 according to SEQ ID NO 1 by cystein, serine, threonine,<br>
asparagine, glutamine, tyrosine, lysine and arginine; more preferably serine,<br>
threonine, asparagine, glutamine, lysine and arginine; most preferably arginine;<br>
• methionine at position 74 according to SEQ ID NO 4 by threonine or by a<br>
hydrophobic amino acid such as valine, leucine, isoleucine and phenylalanine or by<br>
an amphiphilic amino acid such as tyrosine, tryptophan, histidine, glutamine and<br>
asparagine, more preferably threonine or isoleucin;<br>
• Threonine at position 89 according to SEQ ID NO 1 preferably by a charged amino<br>
acid, preferably a negatively charged amino acid such as aspartic acid or glutamic<br>
acid, most preferably a positively charged amino acid such as lysine, arginine or<br>
hystidine; most preferred being lysine.<br>
• serine at position 112 according to SEQ ID NO 1 preferably by threonine;<br>
• histidine at position 244 according to SEQ ID NO 1 by glutamine or asparagine,<br>
more preferably glutamine;<br>
• aspartic acid at position 284 according to SEQ ID NO 1 and at position 285 in SEQ<br>
ID NO 4 by an amino acid with a polar side chain such as serine, three-nine,<br>
asparagine, glutamine, glutamic acid, histidine, lysine, arginine and tyrosine, more<br>
preferably asparagine, glutamine, glutamic acid, lysine and arginine; most preferred<br>
being asparagine<br>
• glycine at position 300 according to SEQ ID NO 1 by an amino acid with a small<br>
residue, such as alanine, serine, threonine, cystein, valine, isoleucine, leucine,<br>
asparagine and aspartic acid, more preferably valine.<br>
The invention furthermore provides mutant expandases as defined<br>
hereinbefore that are variants of the group consisting of the wild type expandases of S.<br>
clavuligerus, S. jumonjinensis, S. ambofaciens, S. chartreuses and N. lactamdurans.<br>
Most preferred are mutant expandases as defined hereinbefore that are variants of the<br>
wild type expandase of S. clavuligerus or N. lactamdurans. Most preferred are mutant<br>
expandases as defined hereinbefore that are variants of the wild type expandase of S.<br>
clavuligerus. Preferred mutant expandases that are mutants of the expandase of<br>
Streptomyces olavuligems have modifications at least at<br>
o 2 or more amino acid positions selected from the group consisting of D2, T89,<br>
1277, C281 and G300.<br>
o 3 or more amino acid positions selected from the group consisting of D2, T89,<br>
C281,T293andA311<br>
o 4 or more amino acid positions selected from the group consisting of D2, M73,<br>
T89, N90, Y217, H244,1277, E280, C281, R306, R307 and A311<br>
o 5 or more amino acid positions selected from the group consisting of D2, M73,<br>
T89, N90, C155, T213, R249, A278, C281, T293, R306, R307 and A311<br>
o 6 amino acid positions selected from the group consisting of<br>
N90+T105+G113+C281+R306+A311.<br>
o 7 or more amino acid positions selected from the group consisting of D2, M73,<br>
T89, T105, G113, C155, H170, P177, D251, L277, E280, C281, T293, G300,<br>
R306, R307 and A311<br>
o 8 or more amino acid positions selected from the group consisting of D2, T89,<br>
N90, M99, T105, G113, 0155, P177, L277, 0281, R306, R307 and A311.<br>
o 9 amino acid positions selected from the group consisting of<br>
D2+N90+T105+G113+0155+P177+0281+R306+A311.<br>
o 10 amino acid positions selected from the group consisting of D2, S59, N90,<br>
T105, G113, T105, G113, C155, P177, G209,1277, C281, R306, R307, A311.<br>
o 11 amino acid positions selected from the group consisting of D2, N90, T105,<br>
G113, C155, P177, L277, C281, R306, R307, A311.<br>
Preferred modifications at the respective positions are D2N, D2H, D2Y, S59G,<br>
M73H, M73I, T89A, T89K, T89V, N90W, IM90S, M99T, T105I, Y101F, G113D, C155W,<br>
H170Y, P177L, G209A, T213A, Y217H, H244R, R249C, D251G, L277Q, L277T,<br>
A278V, E280G, C281Y, T293K, G300S, R306deletion, R307deletion, R370T and<br>
A311D. In this notation, the letter following the number represents the amino acid (one<br>
letter code) present in the mutant expandase. The deletion at position R306 or R307<br>
mean that the original arginine at position 306 or 307 is no longer present in the mutant<br>
expandase. Highly preferred mutant expandases are the mutant expandases that are<br>
summarized in Table 3.<br>
Preferably, the mutant expandases provided by the present invention have an<br>
improved expandase activity on adipyl-6-APA as defined herein before, the mutant<br>
expandases provided by the present invention have likewise, an improved expandase<br>
activity on Pen-G. Preferably, the mutant expandases provided by the present<br>
invention have an improved expandase on both adipyl-6-APA as well as Pen-G. Mutant<br>
expandases with an improved activity on Pen-G can be used advantageously in a<br>
process for the production of phenylacetyl-7-ADCA as described further below.<br>
The present invention also provides mutant expandases with a decreased or<br>
even absent expandase activity with iso-penicillin N (iPN). Preferably the expandase<br>
activity with either adipyl-6-APA or Pen-G is not affected, but more preferably the<br>
expandase activity with either adipyl-6-APA or Pen-G or both is improved as defined<br>
hereinbefore. The advantage of these more preferred mutant expandases is that the<br>
decreased or even absent expandase activity with iso-penicillin N (iPN) results in less<br>
byproduct in a fermentation process to produce ad-7-ADCA or phenylacetyl-7-ADCA.<br>
Preferred mutant expandases have a decreased or even absent expandase<br>
activity with iso-penicillin N (iPN) as a substrate optionally combined with an improved<br>
expandase activity on ad-6-APA as a substrate and have been modified at position 89<br>
according to the amino acid numbering of SEQ ID No 1. whereby the naturally<br>
occurring amino acid has been replaced, preferably by a charged amino acid,<br>
preferably a negatively charged amino acid such as aspartic acid or glutamic acid,<br>
most preferably a positively charged amino acid such as lysine, arginine or hystidine;<br>
most preferred being lysine.<br>
A highly preferred mutant expandase is selected from the group consisting of<br>
H101, H106, H111, H122, H127, H262, H301, H305, H308, H309, H401, H402, H403,<br>
H501, H502, H503, H504, H505, H506, H507, H508, H601, H602, H603, H604, H605,<br>
H606, H607, H608, H609, H650, H651, H652, H653, H654, H655, H656, H657, H658,<br>
H659, H660, H661, H662, G601, G602, G603, G604, G605, G606, G607, G608, G609,<br>
G610, G611, G613 and G614 (see Table 3).<br>
In a second aspect, the invention provides a polynucleotide encoding the<br>
mutant expandase of the present invention. The polynucleotide encoding the mutant<br>
expandase according to the present invention can be any polynucleotide that encodes<br>
the proper amino acid sequence according to the invention. This implies that it may<br>
correspond to the ce/Egene of S. clavuligerus except for the nucleotides encoding the<br>
modifications at the respective amino acid positions. Alternatively, it may correspond to<br>
the cefE gene of S. clavuligerus except for the polynucleotide fragments encoding the<br>
parts of the amino acid sequence corresponding with sequences of N. lactamdurans<br>
expandase. Alternatively, the polynucleotide of the invention may comprise a coding<br>
sequence in which the codon usage for the various amino acids deviates from the<br>
codon usage in S. clavuligerus and/or in N. lactamdurans. For example, the codon<br>
usage may be adapted to the codon usage of a particular host cell, which will or has<br>
been transformed with the DNA fragment encoding the altered expandase.<br>
In a third aspect, the invention provides an expression vector or expression<br>
cassette comprising the polynucleotide of the invention as defined hereinbefore.<br>
In a fourth aspect, the invention provides a transformed host cell, transformed<br>
with the polynucleotide of the invention or the expression vector or expression cassette<br>
of the invention. The transformed host cell may be used for the production of the<br>
mutant expandase of the invention or the host cell may be used for the production of a<br>
beta-lactam compound of interest.<br>
Host cells for the production of the mutant expandase of the invention are<br>
preferably host cells which are known in the art for their efficient protein or enzyme<br>
production, either extracellular or intracellularly, for example microorganisms such as<br>
fungi, yeast and bacteria. Examples of preferred host cells comprise, but are not limited<br>
to, the following genera: Aspergillus (e.g. A. niger, A. oryzea), Penicillium (e.g. P.<br>
emersonii, P. chrysogenum), Saccharomyces (e.g. S. cerevisiae), Kluyvemmyces (e.g.<br>
K. lactis), Bacillus (e.g. B. subtilis, B. licheniformis, B. amyloliquefadens). Escherichia,<br>
(E. coif), Streptomyces (e.g. S. clavuligerus).<br>
Host cells for the production of a beta-lactam compound of interest are<br>
preferably host cells that are known In the art for their efficient beta-lactam compound<br>
production. Examples of preferred host cells comprise, but are not limited to, to the<br>
following genera: Penicillium (e.g. P. chrysogenum), Acremonium (e.g. A.<br>
chrysogenum), Streptomyces (e.g. S. clavuligerus), Nocardia (e.g. N. lactamdurans),<br>
Lysobacter (e.g. L. lactamgenus) and Flavobacterium species.<br>
In a fifth aspect, the invention provides a process for the production pf the<br>
mutant expandase of the invention comprising cultivating the transformed host cell<br>
according to the invention under conditions conducive to the production of the mutant<br>
expandase and, optionally, recovering the mutant expandase. The recovered mutant<br>
expandase may be used advantageously in an in vitro process to produce a desired<br>
cephalosporin from a corresponding penicillin, for instance the recovered mutant<br>
expandase may be used in a process to produce phenylacetyl-7-ADCA from Pen-G or<br>
adipyl-7-ADCA from adipyl -6-APA.<br>
In a sixth aspect, the invention provides a process for the production of a betalactam<br>
compound of interest comprising cultivating the transformed host cell according<br>
to the invention under conditions conducive to the production of the beta-lactam<br>
compound of interest and, optionally, recovering the beta-lactam compound. Preferred<br>
beta-lactam compounds belong to the group of cephaiosporins such as phenylacetyl-7-<br>
ADCA, adipyl-7-ADCA, adipyl-7-ADAC and adipyl-7-ACA. In a preferred embodiment,<br>
the invention provides a process for the production of phenylacetyl-7-ADCA or adipyl-7-<br>
ADCA by cultivating a selected strain of Penicillium chrysogenum, that has been<br>
transformed with a selected polynucleotide of the invention that encodes a mutant<br>
expandase of the invention. For the production of phenylacetyl-7-ADCA, a mutant<br>
expandase is selected that has a high improvement factor on Pen-G as a substrate.<br>
For the production of adipyl-7-ADCA, a mutant expandase is selected that has a high<br>
improvement factor on ad-6-APA as a substrate.<br>
In another preferred embodiment, the invention provides a process for the<br>
production of 7-ADCA comprising the process for the production of phenylacetyl-7-<br>
ADCA or adipyl-7-ADCA as described herein before, followed by, after optional<br>
purification of said 7-ADCA-derivatives, a process step in which the phenylacetyl or<br>
adipyl side chain of phenylacetyl-7-ADCA and adipyl-7-ADCA respectively is cleaved<br>
off thereby generating 7-ADCA and the liberated side chains acid. Said cleavage can<br>
be obtained by chemical means or, more preferably, enzymatically using an acylase<br>
enzyme. Suitable acylases for the cleavage of the adipyl side chain are obtainable from<br>
various Pseudomonas species such as Pseudomonas SY-77 or Pseudomonas SE-83.<br>
Suitable acylases fro the cleavage of the phenylacetyl side chains are the penicillin<br>
acylases from Escherichia coli or Alcaligenes feacalis.<br>
In a seventh aspect, the invention provides a process for the production of a<br>
cephalosporin from a corresponding penicillin whereby the expansion of the 5-<br>
membered thiazolidine ring of the penicillin to the 6-membered dihydrothiazine ring of<br>
the cephalosporin occurs in an in vitro process, catalyzed by a mutant expandase of<br>
the invention. In a preferred process, adipyl-6-APA is expanded to the corresponding<br>
adipyl-7-ADCA by a mutant expandase of the invention, preferably a mutant<br>
expandase that has a high improvement factor adipyl-6-APA as a substrate. In another<br>
preferred process, Pen-G is expanded to the corresponding phenylacetyl-7-ADCA by a<br>
mutant expandase of the invention, preferably a mutant expandase that has a high<br>
improvement factor on Pen-G as a substrate.<br>
The process may be followed by, after optional purification of said 7-ADCAderivatives,<br>
a process step in which the side chains of adipyl-7-ADCA and<br>
phenylacetyl-7-ADCA are cleaved off thereby generating 7-ADCA and the liberated<br>
side chains acid - supra vide.<br>
The desired end product 7-ADCA can be recovered according to methods<br>
known in the art involving chemical or enzymatic cleavage of the side chain of adipyl-7-<br>
ADCA or phenylacetyl-7-ADCA and optionally the resulting 7-ADCA can be further<br>
purified and/or crystallized.<br>
In an eight aspect, the invention provides a method to obtain the mutant<br>
expandases of the invention whereby the process comprises the following steps:<br>
1. Mutagenesis of a cloned gene encoding a model polypeptide with expandase<br>
activity thus obtaining a collection of mutagenised genes encoding the mutant<br>
expandases;<br>
2. Expression of the collection of mutagenised genes encoding the mutant<br>
expandases in a suitable host and screening the collection of mutant expandases<br>
for an improved activity with a suitable substrate;<br>
3. Optionally repeating steps 1 and 2 one or several times using either the gene<br>
encoding the model polypeptide with expandase activity or one or more of the<br>
mutagenised genes encoding mutant expandases with an improved activity on the<br>
suitable substrate.<br>
Cloning of the gene encoding a model polypeptide with expandase activity can<br>
be carried out according to methods known in the art.<br>
Preferred model polypeptides with expandase activity are selected from the<br>
group consisting of a polypeptide with expandase activity obtainable from<br>
Streptomyces davuligerus, preferably having an amino acid sequence according to<br>
SEQ ID NO: 1 and polypeptides with expandase activity having an amino acid<br>
sequence with a percentage identity with SEQ ID NO: 1 of at least 70%, preferably at<br>
least 75%, more preferably at least 80%, more preferably at least 85%, more preferably<br>
at least 90%, most preferably at least 95%, such as the expandase enzymes that are<br>
summarized in Table 1.<br>
Any mutagenesis technique can be employed which results in mutations over<br>
the entire gene. Suitable techniques are error prone (EP) PCR (Polymerase Chain<br>
Reaction) and/or by using saturated Mutation Primer PCR (sMPP) exactly according to<br>
WO 03/010183.<br>
Materials and Methods<br>
7. General<br>
Oligonucleotides were synthesized by Invitrogen (Carlsbad CA, US).<br>
DNA sequencing was carried out by SEQLAB (Gb'ttingen, Germany) or by Baseclear<br>
(Leiden, The Netherlands).<br>
Restriction enzymes were purchased from Invitrogen.<br>
The protein assays were carried out according to the method described by Bradford,<br>
MM (1976) Anal Biochem. 72:248-54<br>
Escherichia co//DH10B electromax competent cells were obtained from Invitrogen. The<br>
protocol was delivered by the manufacturer,<br>
SDS-PAGE electrophoresis was carried out in the system supplied by Invitrogen.<br>
NuPAGE Novex Bis-Tris Gels (Invitrogen). SimplyBlue SafeStain Microwave protocol<br>
Overview of used constructs<br>
Plasmid<br>
PBAD-MH-Zeo-MBP-ScEwt<br>
PBAD-MH-Zeo-MBP-H127<br>
PBAD-MH-Z80-MBP-H406<br>
PBAD-MH-Zeo-MBP-H402<br>
PBAD-MH-Zeo-MBP-H403<br>
Description<br>
Arabinose inducible E.coli expression construct with<br>
maltose binding protein (MBP) fused to wild type<br>
S.clavuligerus expandase. Zeocine marker<br>
Idem but fused to H127 expandase (see also Table 3x)<br>
Idem but fused to H406 expandase (see also Table 3x)<br>
Idem but fused to H402 expandase (see also Table 3x)<br>
Idem but fused to H403 expandase (see also Table 3x)<br>
1. Cloning of the E. coli expression vectors<br>
The fusion product is ligated in the pBAD/MH MBP-ScEwt Dest zeo vector by<br>
an EcoRI/Sa/l digestion. This replaces the wild type expandase gene by the library<br>
genes. To prevent significant numbers of wild type constructs, the ligation constructs<br>
were digested with Sma\.<br>
2. Construction of the libraries<br>
Transformation of E. coll with the ligation mix yielded libraries of roughly<br>
12,000-13,000 colonies. To test the quality of the libraries, a random set of colonies<br>
was selected and the constructs were digested with restriction enzymes. This showed<br>
that approximately 26% of the library revealed an aberrant pattern. Next, sequence<br>
analysis was performed on 10 regular clones to determine mutation frequencies.<br>
Sequences showed that both saturation and error prone mutations were present at a<br>
satisfactory level<br>
3. Expression and screening of mutant expandases<br>
3.1. Screening of the cefE library for improved activity<br>
E. coli TG1 cells containing the library A were plated and grown on agar<br>
plates. Colonies are picked robotically into 96-wells microtiter plates containing liquid<br>
medium (2YT, 30jig/ml chloramphenicol, 05% glucose), and grown to saturation by<br>
shaking at 37°C. Cells are subcultured in 96 well megatiter plates; 1:40 inoculum into<br>
2YT, 30 (ig/ml chloramphenicol and induced by 0.2 mM IPTG shaking at 28°C for 24 h.<br>
Cells are pelleted and washed with an equal volume of buffer (5 mM morpholine<br>
pH7.5). After spinning, the cells are resuspended in lysis mix for primary screening<br>
Expandase assay (1)).<br>
3.2. Primary screening in microtitre plates (MTP)<br>
Growth and induction<br>
The expandase E. coli library is plated on Luria-Bertani (LB) medium plates low<br>
salt agar + 25 u.g/ml Zeocine and incubated overnight at 37 °C, Microtitre plates (MTP)<br>
with 150 pi LB low salt medium and 25 u.g/ml Zeocine are inoculated from the plates<br>
and incubated 36 hrs at 25 °C and 550 rpm.<br>
From the MTP, 5 u.l is inoculated in deep well plate with 1 ml 2*TY (tryptone and<br>
yeast extract) medium + 50 u.g/ml Zeocine + arabinose. To the remaining culture in the<br>
MTP, 100 ul 50% glycerol is added and frozen at -80 °C as glycerol stock.<br>
The deep well plate is incubated for 30 hrs at 25°C and 550 rpm. The deep well<br>
plate cultures are centrifuged at 2750 rpm (Heraeus multifuge 4 kr.) for 10 min. The<br>
supernatant is discarded and the pellet is frozen at -20 "C.<br>
Cell free extract (CFE)<br>
The frozen pellets obtained in the previous section are thawed in 200u.l<br>
extraction buffer (50 mM Tris/HCI pH=7.5; 5 mM DTT; 0.1 mg/ml DNAse; 5 mM<br>
MgSCyH20 and 0.5 mg/ml lysozyme). To resuspend pellets, the plates are shaken.<br>
The plates are incubated for 30 minutes at room temperature and centrifuged for 10<br>
minutes at 2750 rpm.<br>
3.3 Secondary screening in shake flasks<br>
Growth and induction<br>
Selected colonies are inoculated from plate in 10 ml LB low salt + Zeocin 25<br>
ug/ml and incubated overnight at 37°C, 280 rpm. The grown culture is inoculated in<br>
100 ml LB low salt with 25 ug/ml Zeocine at an optical density (600 nm) between 0.010<br>
and 0.050 (Biochrom Ultraspec 2000). After growth at 37°C, 280 rpm, the cells are<br>
harvested at an optical density at 600 nm between 0.4-0.6. Then, arabinose is added<br>
(final concentration 0.2%) and induced overnight at 27°C at 220 rpm.<br>
4. Expandase assays<br>
4.1 Expandase assay (1) - Screening assay (primary assay):<br>
Expandase assay buffer: 5 mM morpholine buffer pH 7,5 and cofactor mix<br>
(final concentrations 50 jiM ATP, 1 mM DTT, 60 u.M FeS04, 2,7 mM ascorbate), 500<br>
|iM ad-6-APA, 1/10th volume of E. coli cell lysate and 2.4 mM a-KG. Each cofactor was<br>
made in buffer pH 7.5, which was critical for activity.<br>
The expandase reaction was found to be linear for up to 3 hours with respect<br>
to adipyi-7ADCA formation/detection. The optimal temperature for the in vitro<br>
expandase reaction was determined to be approximately 11°C, probably due to<br>
instability of the enzyme or production at higher temperatures. Nevertheless, actual<br>
screening was performed at a temperature between room temperature and 25°C,<br>
which is more reflective of the desired reaction condition temperature.<br>
Screening of the family shuffled libraries for improved 7-ADCA formation was<br>
determined by Flow Injection Analysis-MS (FIA-MS, negative mode, Mass Transition<br>
[M-H]'341&gt;295). Final analysis: LC column: using solvent conditions between 0.1%<br>
formic acid in 40% MeOH to 100% MeOH. Two channels (two mass transitions [M-H~])<br>
are used to monitor for each reaction product. Samples are analyzed after 30 minutes<br>
of reaction.<br>
4.2 Expandase assay (2)- Secondary exoandase assay<br>
Expandase assay buffer: 5 mM morpholine buffer pH 7,5 and cofactor mix<br>
(final concentrations: 50 \iM ATP, 1 mM DTT, 60 \iM FeS04, 2,7 mM ascorbate), 2 mM<br>
adipyl-6-APA, 1/10tn volume of £ co//cell lysate and 2.4 mM a-ketoglutarate. Each<br>
cofactor was made in buffer pH 7.5, which was critical for activity. The conversion was<br>
performed at 29°C and samples were taken between 0 and 30 minutes. The reaction<br>
was stopped with 60% (v/v) acetonitrile final concentration. The formed adipyl-7-ADCA<br>
was analysed on HPLC. (Mobile phase: 10% acetonitrile in 10 mM potassium<br>
phosphate buffer pH 3.0) Column: X-Terra™ RP18 3.5u.. 3.9x100 mm detected at 214<br>
and 254 nm (254 nm to analyse expanded antibiotics). HPLC data analysis: software:<br>
Waters Millennium32® 3.20.<br>
4.3 Expandase assay (3) - Expandase assay<br>
A total ot 300 uj of a reaction mixture consisting of 50 mM Tris/HCI pH 7.5; 1<br>
mM DTT; 2.7 mM Ascorbate; 0,05 mM ATP; 24 mM a-ketoglutarate; 0.06 mM<br>
FeS04.7H20; 4 mM adipyl-6-amino-penicillanic acid (ad-6-APA) was added to 75 uJ<br>
CFE and incubated at 29^ for the desired time. Adding 60 uJ maleic acid with 10 g/l<br>
EDTA stopped the reaction. The formation of adipyl-7-amino-desacetoxycephalosporinic<br>
acid (ad-7-ADCA) was detected using 1H-NMR.<br>
The adipyl-6-APA was obtained from 6-APA and adipic acid catalyzed by penicillin<br>
acylase (EC 3.5.1.11). Alternatively, adipyl-6-APA can be obtained by culturing for<br>
instance a suitable Penicillium chrysogenum strain in the presence of adipic acid as<br>
precursor.<br>
EXAMPLES<br>
Example 1<br>
Cloning of wild type expandase<br>
The cefE gene was PCR cloned from S. clavuligerus genomic DMA and<br>
ligated in GFP-fusion vectors, the so-called pCK series, and used to transform E. coli<br>
TG1 (Amersham Pharmacia Biotech, Inc. NJ USA). Several colonies from the plated<br>
transformations were selected and checked for an active clone. Several active clones<br>
were identified. Clone '778' appeared to be a wild type gene but interestingly, one other<br>
clone, clone "779", revealed strongly improved activity on adipyl-6-APA expansion. The<br>
product of clone "779" was sequenced and revealed three amino acid mutations<br>
 (H18R, D284N and G300V). The adipyl-7-ADCA production of the clones analysed by<br>
Flow Injection Analysis-Mass Spectrometry (FIA/MS) showed that the Sc-cefE mutant<br>
"779" was superior to the wild-type parents. Measurement of GFP fluorescence<br>
suggests that the improvement of the expandase enzyme product of clone "779" is not<br>
due to improved expression of cefE therein. Hence, it was concluded that the<br>
improvement of the expandase enzyme product of clone "779" is an improvement of<br>
the activity as a result of the amino acid substitutions.<br>
pCK derived GFP fusion vectors<br>
Fusion tag vector series (called pCK series) containing a His tag fused to the<br>
3' end of expandase with green fluorescent protein (GFP) fusion at the 3' end of the<br>
HIS tag were constructed. The GFP containing vectors enable high throughput<br>
normalisation of expandase activity data in subsequent assays with respect to protein<br>
concentration. The vectors contain the p15A origin of replication (low copy number),<br>
and a lac promoter that is repressed in the presence of glucose.<br>
Example 2<br>
Family shuffling and Screening of wild-type N. lactamdurans and<br>
S. clavuligerus cefE genes - Library A<br>
The expandase genes were shuffled according to the process described in<br>
EP752008. Expandase mutant libraries were made using the GFP-fusion tag vectors<br>
(pCK). Only the Ce/E ORF was shuffled, while other sequences where not changed.<br>
After moving the shuffled pool of expandases into the pCK vectors, one library set<br>
(Library A) was made using the two parental wild type expandase genes (S.<br>
clavuligerus and N. lactamdurans) plus clone "779".<br>
Initial plating of a sample of the library showed that the libraries contained at<br>
least 70% green fluorescent colonies. Due to this, it was not considered necessary to<br>
pre-screen out non-fluorescent colonies before proceeding to the first FIA/MS screens.<br>
DNA sequencing of randomly picked colonies form library set A indicated that<br>
all parents were represented in the library, so this library was selected for initial<br>
screening.<br>
720 clones from library A were screened and 14 clones (mutant expandases<br>
H101, H102, H104, H106, H108, H111, H112, H114, H115, H117, H120, H122, H125,<br>
H127) were retested and the preliminary results showed an expandase activity<br>
improvement compared to the S. clavuligerus expandase (clone "778") as judged by in<br>
vitro expandase assay (1). These clones were chosen for further testing. Normalization<br>
of the expandase activity with respect to expandase-GFP concentration (i.e. correcting<br>
for differences in the expandase protein concentration) confirmed that these clones<br>
were improved in specific activity.<br>
Example 3<br>
Family shuffling of wild-type N. lactamdurans and S. clavuligerus cefE genes.<br>
Library B<br>
Library B was constructed shuffling wild type N. lactamdurans and S.<br>
clavuligerus cefE as parent genes, The "779" hit sequence was not used in<br>
construction of this library. This expandase library showed to have a significant level of<br>
inactive or very poorly active clones in the library. In order to remove inactive clones<br>
from further testing, a FACS-based pre-screen was used for the library B screening to<br>
improve the percentage of active clones that enter the expandase assays.<br>
After FAGS sorting based on their fluorescence by GFP, selected cells were recovered<br>
under repressing conditions. DNA prepared from these cultures was used to transform<br>
coll TG1. These expandase constructs used are also fusion proteins with GFP as<br>
described for library A.<br>
70-80 percent of the library was GFP-active. In total, 2200 clones from this<br>
library were then screened using the in vitro expandase assay and FIA-MS detection of<br>
adipyl-7-ADCA formation.<br>
Fourteen possible hits were selected (H200 series). Four of these hits,<br>
amongst them H262, were confirmed through retesting procedure including<br>
^transformation and in vitro testing and analysis MS.<br>
Example 4<br>
Construction of chimeric expandase sequences using shuffled expandases<br>
Library C<br>
Using traditional molecular biological techniques and bioinformatics tools, the<br>
hit sequences from the two above described libraries (libraries A and B) were<br>
recombined, resulting in library C (H300 series). Three resulting chimeras were tested<br>
for activity, amongst them expandase H301.<br>
Six remaining clones (amongst them: H305, H307, H308, H309) were not<br>
tested as GFP fusion product nor analysed by FIA-MS. These clones were directly<br>
recloned in the pSJ vector series fused with maltose binding protein (MBP) upstream of<br>
the CefcL mutant genes and analysed, after the In vitro assay, on HPLC.<br>
Example 5<br>
Recjoning of genes encoding improved expandases using pSJ based vector series and<br>
enzymatic activity of the improved expandases<br>
pSJO.8 was used as cloning vector for the expandase libraries. This plasmid<br>
was constructed by replacing the expandase gene (as Ndel-Nsil fragment) by a small<br>
oligonucleotide introducing a A/of/ restriction site. S. davuligerus cefE was cloned as<br>
Nde\-Nsi\ fragment in pSJOS, resulting in pSJOS.<br>
pSJ05 contained an additional Nde\ site just upstream of the Nsi\ site. This<br>
site was removed to prevent possible interference with further cloning of the shuffled<br>
genes in the Nde\-Nsi\ sites, resulting in pSJ11. Next, maltose binding protein (MBP)<br>
was fused to the 5' end of cefE resulting in pSJ10. A plasmid containing MBP-cefE<br>
fusion genes for Nocardia lactambdurans (pSJ01) was also constructed. DNA<br>
sequence analysis confirmed all sequences.<br>
Improved clones from the A, B and C library were recloned from the pCKvectors<br>
into the pSJ10 vector (MBP-cefE behind the tac promoter). This was done by<br>
amplifying the improved expandases by PCR. The forward (5'-end) primer introduces a<br>
A/del and the reverse (3'-0nd) primer introduces a Nsil site. After digestion with Ndeand Nsil the hit was cloned in the E. coli vector (pSJ10) and in the Penicillium<br>
chrysogenum vector (plATWAn- Figure 2/3) Table 2.<br>
Finally, a few clones of each were sequenced and one of each, containing the<br>
verified correct sequence, was used for further analysis.<br>
The DNA and amino acid sequences of the best expandase hits are included<br>
in the sequence listing - see Table 2 for an overview.<br>
Induction of expression<br>
E. coli NM554 was grown overnight on 2xTY medium with chloramphenicol<br>
(30 ug/ml) at 37°C. The cells were diluted to an optical density at 540 nm (OD540) of<br>
0.010-0.020 in fresh 2xTY with chloramphenicol (30 ng/ml). When cells reached an<br>
optical density at 540 nm of 0.4-0.6, IPTG was added (final concentration 0.5 mM) and<br>
expression was induced overnight at 27°C.<br>
The cells were washed and resuspended in extraction buffer (5 mM DTT, 50<br>
mM Tris/HCI pH 7.5 with lysozym) and sonicated (on ice water). After centrifugation the<br>
supernatant is used for the secondary expandase assay (2).<br>
Expression of the expandase in the cell free extract was determined by SDS-PAGE.<br>
Table 2. H100, H200 and H300 series expandase hits, their mutation(s). their E. coli<br>
and Penidllium expression vectors and their expandase activity (assay 2).<br>
Expandase<br>
(Table Removed)<br>
Specific activity (nmol 7-ADCA.min'1.mg~1 protein)<br>
I.F. = Improvement factor; equal the ratio of the specific activity of the mutant<br>
expandase and the specific activity of the expandase from S. clavuligeris<br>
Example 6<br>
Expression of expandase H122 and H127..in Penicillium chrvsoaenum<br>
Expandase variants H122 and H127 were cloned into vectors for Penicillium<br>
chrysogenum expression (plAT series plasmids, Table 2), Shuffled expandase genes<br>
cloned into a plasmid vector with an E. coli replicon and selectable marker. A promoter<br>
compatible with expression in P. chrysogenum, i.e., the IPNS promoter, was used to<br>
express the expandase gene. Additionally a selectable marker for Penicillium<br>
chrysogenum is present (amdS).<br>
The expression constructs were obtained by digesting the expression<br>
construct, plAT127 (expandase H127) and plAT122 (expandase H122), with A/of/. A<br>
high producing Penicillium chrysogenum is co-transformed with the linear fragments<br>
together with the amdS expression cassette (H/nc/lll digest).<br>
By precursing with adipate, adipyl -7-ADCA production can be tested in strains<br>
expressing active expandase genes.<br>
The cephalosporin production of the strain expression expandase H127 was<br>
compared to the strain expressing wild type expandase in batch regime with lactose as<br>
carbon source. The adipyl-7-ADCA production is increased between 10-50% and the<br>
total p-lactam production was increased by 0-30%, depending on the lactose and the<br>
adipate concentration.<br>
Additionally, expandase H122 was expressed in Penicillium chrysogenum.<br>
Comparison of the cephalosporin production with the strain expressing wild type<br>
expandase showed that the adipyl -7-ADCA production is increased between 0 to 50%<br>
and the total beta-lactam production varied between -20 to 0% depending on the<br>
lactose and the adipate concentration in the media.<br>
Example 7<br>
1st generation improved expandases (H400 series).<br>
The first generation expandase library was constructed by performing error<br>
prone (EP) PCR (Polymerase Chain Reaction) on wild type Streptomyces clavuligerus<br>
CefE gene (SEQ ID No.1), After screening this library for improved conversion of ad-6-<br>
APA to ad-7-ADCA, 3 different mutant genes were selected. The mutants exhibited an<br>
improved ad-6-APA expansion activity up to 2.5-fold (H401, H402, H403: see Table 3).<br>
Example 8<br>
2nd generation ImprQved expandases (H500 series)<br>
The second-generation expandase library was constructed by using saturated<br>
Mutation Primer PCR (sMPP) exactly according to WO 03/010183. The selected<br>
mutant genes from the first generation (H401, H402 and H403) were used as templates<br>
for the construction of the 2nd generation library.<br>
The sMPP was performed by using Taq polymerase, thereby introducing<br>
additional mutations (random) and increasing the variation of the library. Designed<br>
primers annealed at the mutation positions and were saturated at the mutations found<br>
in the 1st generation expandase hits. Additionally, a universal forward and reverse<br>
primer was designed in which an Ndel and Nsil site were introduced respectively. This<br>
facilitated cloning into Penicillium expression vector for testing of the mutant<br>
expandases in vivo.<br>
The library was grown and expression of the expandase was induced as<br>
described in the materials and methods section. The formation of ad-7-ADCA was<br>
determined using NMR as described in the materials and methods section.<br>
Approximately 150 improved mutant expandases were selected and retested for<br>
their ad-6-APA expandase activity. Based upon the results of this retest, 17 mutants<br>
were selected for final tests in shake flasks and analysis by HPLC as described in the<br>
Materials and Methods.<br>
In total, 15 of the 17 selected hits showed a significant improved expandase<br>
activity with ad-6-APA as a substrate. To exclude the false positive selection of<br>
expression mutants, protein levels were quantified on SDS-PAGE. This confirmed that<br>
the improved activity was not due to stronger expression of expandase in E. coll, but<br>
that the improvement could be attributed to an improved specific activity of the<br>
expandase mutant.<br>
In total, 8 mutant expandases were identified which exhibited an improvement<br>
factor of their ad-6-APA expandase up to 4,5-fold (compared to the wild type<br>
expandase of S. clavuligerus - SEQ ID No 1.). These mutants are designated H501 -<br>
H508 (see Table 3).<br>
Example 9<br>
3rd generation of improved expandases (H600 series)<br>
The 8 mutants obtained in the 2nd generation Example 2 (H501 - H508) were<br>
used as templates for the construction of the 3rd generation expandase library. This<br>
library was constructed in the same way as the 2nd generation expandase library.<br>
Additionally, the SKA signaling sequence at the C-terminus of the expandases was<br>
changed to SKD, causing expression of expandase solely in the cytosol when<br>
expressed in Penicillium.<br>
The screening of this library yielded 22 different expandase mutant genes that<br>
were significantly improved between 5-fold and 11-fold compared to S. clavuligerus<br>
expandase (Table 3, H600 series).<br>
Screening of the same library for a second time yielded 13 additional mutant<br>
expandases (G601 - G611 and G613 - G614; see Table 3)<br>
Example 10<br>
Activity of improved mutant expandases with iso-penicillin N (iPN) and Penicillin-G<br>
(Pen-G) as a substrate.<br>
The activity of several mutant expandases was measured using iPN and Pen-<br>
G as a substrate. The assay with iPN as a substrate was carried out as described in<br>
the materials and methods section except that ad-6-APA was replaced by the same<br>
concentration of iPN (4 mM). The assay with Pen-G as a substrate was carried out as<br>
described in the materials and methods section except that ad-6-APA was replaced by<br>
7 mM Pen-G. Table 3 summarizes the activities of the various expandase mutants for<br>
iPN and Pen-G.<br>
Whereas most of the expandase mutants tested exhibited a 5-fold<br>
improvement of their expandase activity with iPN, mutants that carry the T89K mutation<br>
lost virtually all activity towards iPN.<br>
The activity towards Pen-G of the various expandase mutants tested was<br>
either the same as the wild type expandase (improvement factor is 1) or improved up to<br>
8-fold. The data further show that there is no correlation whatsoever between the<br>
improvement factors obtained for a single mutant with ad-6-APA and Pen-G as a<br>
substrate. The ratio between the respective improvement factors for ad-6-APA and<br>
Pen-G vary in a range from 0.1 (e.g. H654) to 1,2 (e.g. H503).<br>
Table 3. Mutant expandases and their expandase activity toward ad-6-APA, Pen-G and<br>
iPN. Each mutant is identified by its code; the mutations are introduced in the model<br>
expandase from S. clavuligerus (SEQ ID NO 1). The expandase activity is expressed<br>
as improvement factor compared to the model expandase from S. clavuligerus<br>
(expandase activity = 1 by definition) according to the in vitro expandase assays<br>
described in the Materials and Methods and Example 4.<br>
(Table Removed)<br>
Table 3 shows that mutant expandases have been obtained which have improvement<br>
factors in the range of 1.5 to 10.6-fold,<br><br><br><br><br>
CLAIMS<br>
1. A mutant expandase that is a variant of a model polypeptide with expandase<br>
activity whereby the mutant expandase has an at least 2-fold improved in vitro<br>
expandase activity towards adipyl-6-APA in comparison with the model<br>
polypeptide with expandase activity.<br>
2. A mutant expandase that is a variant of a model polypeptide with expandase<br>
activity whereby the mutant expandase has been modified at at least an amino<br>
acid position selected from group 1 consisting of positions 2, 18, 59, 73, 74, 89,<br>
90, 99, 101, 105, 112, 113, 155, 170, 177, 209, 213, 217, 244, 249, 251, 277,<br>
278, 280, 281, 284, 293, 300, 307 and 311 using the amino acid position<br>
numbering of the amino acid sequence of the expandase enzyme encoded by<br>
the cefE gene of Slreptomyces clavuligerus (SEQ ID NO: 1).<br>
3. A mutant expandase according to claim 2 whereby the mutant expandase has<br>
been modified at at least an amino acid position selected from the group 2<br>
consisting of positions 2, 18, 59, 89, 90, 99, 101, 105, 112, 113, 170, 177, 209,<br>
213, 217, 249, 251, 278, 280, 284 and 293, optionally in combination with at<br>
least an amino acid position selected from the group 1 and which does not<br>
belong to group 2.<br>
4. A mutant expandase according to claim 1 whereby the mutant expandase has<br>
been modified at at least an amino acid position selected from group 1 as<br>
defined in claim 2.<br>
5. A mutant expandase according to claim 4 whereby the mutant expandase has<br>
been modified at at least an amino acid position selected from group 2 as<br>
defined in claim 3 optionally in combination with at least an amino acid position<br>
selected from the group 1 and which does not belong to group 2.<br>
6. A mutant expandase according to anyone of the preceding claims wherein the<br>
model peptide is the expandase of Streptomyces clavuligerus encoded by the<br>
cefE gene as depicted in SEQ ID No. 1 or wherein model peptide is the<br>
expandase of Nocardia lactamdurans encoded by the cefE gene as depicted in<br>
SEQ ID No. 2<br>
7. A mutant expandase having an at least 2-fold improved in vitro expandase<br>
activity towards adipyl-6-APA in comparison with the expandase from<br>
Streptomyces clavuligerus and which is selected from the group consisting of<br>
H101, H106, H111, H122, H127, H262, H301, H305, H308, H309, H402, H501,<br>
H502, H503, H504, H505, H506, H507, H508, H601, H602, H603, H604, H605,<br>
H606, H607, H608, H609, H650, H651, H652, H653, H654, H655, H656, H657,<br>
H658, H659, H660, H661, H662, G601, G602, G603, G604, G605, G606,<br>
G607, G608, G609, G610, G611, G613 and G614.<br>
8. A polynucleotide encoding the mutant expandase of anyone of the preceding<br>
claims,<br>
9. An expression vector or cassette comprising the polynucleotide of claim 8.<br>
10. A host cell transformed with the polynucleotide of claim 8 or the vector or<br>
cassette of claim 9.<br>
11. A method of producing the mutant expandase of claims 1-7 comprising<br>
cultivating a host cell according to claim 10 under conditions conducive to the<br>
production of the mutant expandase and, optionally, recovering the polypeptide.<br>
12. A method of producing a (J-lactam compound of interest comprising cultivating a<br>
host cell according to claim 10 under conditions conducive to the production of<br>
the p-lactam compound and, optionally, recovering the (3-lactam compound,<br>
13. A method according to claim 12, further comprising N-deacylating the p-lactam<br>
compound to produce an N-deacylated (3-lactam compound.<br>
14. A method to obtain the mutant expandases according to anyone of claims 1-7,<br>
whereby the method comprises the following steps:<br>
a. Mutagenesis of a cloned gene encoding a model polypeptide with<br>
expandase activity, preferably cefE encoding the expandase from S.<br>
clavuligerus, thus obtaining a collection of mutagenised genes encoding<br>
the mutant expandases;<br>
b. Expression of the collection of mutagenised genes encoding the mutant<br>
expandases in a suitable host and screening the collection of mutant<br>
expandases for an improved activity with a suitable substrate, preferably<br>
ad-6-APA;<br>
c. Optionally repeating steps 1 and 2 one or several times using thereby<br>
mutagenising the gene encoding the model polypeptide with expandase<br>
activity and/or one or more of the mutagenised genes encoding mutant<br>
expandases with an improved activity on the suitable substrate.<br>
15. Use of the mutant expandase polypeptide according to claim 1 to 7 in the<br>
expansion of the penam ring into a ceph-3-em ring.<br>
16. Use of the mutant expandase polypeptide according to claim 1 to 7 in the<br>
expansion of the penam ring of adipyl-6-APA into a ceph-3-em ring.<br>
17. Use according to claim 15 in the production of adipyl-7-ADCA, adipyl-7-ADAC<br>
or adipyl-7-ACA.<br>
18. An adipyl-6-APA producing microorganism transformed with a DNA fragment<br>
encoding an altered expandase polypeptide according to claim 1 to 7.<br>
19. Method for the fermentative production of adipyl-7-ADCA comprising<br>
a. Culturing a microorganism capable of producing adipyl-6-APA and<br>
transformed with a DNA fragment encoding an altered expandase<br>
according to claim 1 to 3 under adipyl-6-APA-producing conditions<br>
b. In situ converting the adipyl-6-APA into adipyl-7-ADCA using the altered<br>
expandase; and<br>
c. Isolating the desired adipyl-7-ADCA.<br>
20. Method for the production of 7-ADCA comprising<br>
a. Culturing a microorganism capable of producing adipyl-6-APA and<br>
transformed with a DNA fragment encoding an altered expandase<br>
according to claim 1 to 3 under adipyl-6-APA-producing conditions<br>
b. In situ converting the adipyl-6-APA into adipyl-7-ADCA using the altered<br>
expandase;<br>
c. Isolating the desired adipyl-7-ADCA.<br>
d. Removal of the adipyl side chain to yield 7-ADCA; and<br>
e. Recovery of 7-ADCA.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUFic3RyYWN0LSgyNy0wNS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Abstract-(27-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUFzc2lnbm1lbnQtKDA5LTA4LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Assignment-(09-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUNsYWltcy0oMjctMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Claims-(27-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Correspondence Others-(02-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Correspondence Others-(09-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Correspondence Others-(20-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Correspondence-Others-(27-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LURyYXdpbmdzLSgyNy0wNS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Drawings-(27-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUZvcm0tMS0oMDktMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Form-1-(09-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUZvcm0tMTgtKDI1LTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Form-18-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUZvcm0tMi0oMDktMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Form-2-(09-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUZvcm0tMy0oMDItMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Form-3-(02-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUZvcm0tMy0oMjAtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-Form-3-(20-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LUdQQS0oMDktMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-GPA-(09-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyOC1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2828-delnp-2007-pct-304.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="257846-a-process-for-producing-carboxylic-acid-having-a-carbon-number-of-n-1.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="257848-a-method-for-making-a-recombinant-protein.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>257847</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2828/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>46/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-Nov-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Nov-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Apr-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM SINOCHEM PHARMACEUTICALS NETHERLANDS B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Alexander Fleminlaan 1, 2613 AX DELFT, THE NETHERLANDS</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RAAMSDONK, LOURINA MADELEINE</td>
											<td>KNOBBELZWAANSINGEL 15, NL-2496 LN DEN HAAG, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JENNE, STEPHANE J.</td>
											<td>859 CABOT LANE, FOSTER CITY, CALIFORNIA 94404, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KREBBER, CLAUS</td>
											<td>3500 LOUIS ROAD, PALO ALTO, CALIFORNIA 94303, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SCHIPPER, DICK</td>
											<td>OOSTSINGEL 205, NL-2612 HL DELFT, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KERKMAN, RICHARD</td>
											<td>KONINGINNEWEG 12, NL-2496 LN DEN ZANDVOORT, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>6</td>
											<td>BOVENBERG, ROELOF ARY LANS</td>
											<td>KRALINGSE PLASLAAN 10, NL-3062 DA ROTTERDAM, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>7</td>
											<td>CHEN, YAN</td>
											<td>1765 BOWERS AVENUE, SANTA CLARA, CALIFORNIA 95051, USA</td>
										</tr>
										<tr>
											<td>8</td>
											<td>CHEN, YONG HONG</td>
											<td>828 VEGA CIRCLE, FOSTER CITY, CALIFORNIA 94404, USA</td>
										</tr>
										<tr>
											<td>9</td>
											<td>GAO, XIAODONG</td>
											<td>1005-M BRYANT WAY, SUNNYVALE, CALIFORNIA 94087, USA</td>
										</tr>
										<tr>
											<td>10</td>
											<td>KREBBER, ANKE</td>
											<td>3500 LOUIS ROAD, PALO ALTO, CALIFORNIA 94303, USA</td>
										</tr>
										<tr>
											<td>11</td>
											<td>TO LA CHING, CHARLENE</td>
											<td>4987 KENSON DRIVE, SAN JOSE, CALIFORNIA 95124, USA</td>
										</tr>
										<tr>
											<td>12</td>
											<td>TOBIN, MATTHEW</td>
											<td>2072 EATON AVENUE, SAN CARLOS, CALIFORNIA 94070, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/055148</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04105022.0</td>
									<td>2004-10-13</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>05106347.7</td>
									<td>2005-07-12</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/257847-mutant-expandases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:17:51 GMT -->
</html>
