<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/257903-engineering-single-gene-controlled-staygreen-potential-into-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:08:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 257903:&quot;ENGINEERING SINGLE-GENE-CONTROLLED STAYGREEN POTENTIAL INTO PLANTS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ENGINEERING SINGLE-GENE-CONTROLLED STAYGREEN POTENTIAL INTO PLANTS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The enzymes of the ACC synthase family are used in producing ethylene. Nucleotide and polypeptide sequences of ACC synthases are provided along with knockout plant cells having inhibition in expression and/or activity in an ACC synthase and knockout plants displaying a staygreen phenotype, a male sterility phenotype, or an inhibition in ethylene production. Methods for modulating staygreen potential in plants, methods for modulating sterility in plants, and methods for inhibiting ethylene production in plants are also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ENGINEERING SINGLE-GENE-CONTROLLED STAYGREEN<br>
POTENTIAL INTO PLANTS<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
[0001]	This application is a non-provisional utility patent application claiming<br>
priority to and benefit of the following prior provisional patent application-. USSN 60/480,861, filed June 23,2003, entitled "Engineering single-gene-controlled staygreen potential into plants" by Gallic et al., which is incorporated herein by reference in its entirety for all purposes.<br>
STATEMENT AS TO RIGHTS TO INVENTIONS MADE UNDER FEDERALLY SPONSORED RESEARCH AND DEVELOPMENT<br>
[0002]	This invention was made with government support under Grant No. 95-<br>
35304-4657 from USDA/CREES and support under Grant MCB-0076434-002 from the National Science Foundation. The government may have certain rights to this invention.<br>
FIELD OF THE INVENTION<br>
[0003]	This invention relates to modulating staygreen potential in plants, inhibiting<br>
ethylene production in plants, and modulating sterility in plants. The invention also provides knockout plant cells, e.g., where the knockout plant cells are disrupted in ACC synthase expression and/or activity, or knockout plants, e.g., which display a staygreen phenotype or a male sterility phenotype. Nucleic acid sequences and amino acid sequences encoding various ACC synthases are also included.<br>
BACKGROUND OF THE INVENTION<br>
[0004]	Stay-green is a term used to describe a plant phenotype, e.g., whereby leaf<br>
senescence (most easily distinguished by yellowing of the leaf associated with chlorophyll degradation) is delayed compared to a standard reference. See, Thomas H and Howarth CJ (2000) "Five ways to stay green" Journal of Experimental Botany 51: 329-337. In sorghum, several stay-green genotypes have been identified which exhibit a delay in leaf senescence during grain filling and maturation. See, Duncan RR, et al. (1981) "Descriptive comparison of senescent and non-senescent sorghum genotypes." Agronomy Journal 73: 849-853. Moreover, under conditions of limited water availability, which normally hastens leaf senescence (see, e.g., Rosenow DT, and Clark LE (1981) Drought tolerance in sorghum. In:<br>
Loden HD, Wilkinson D, eds. Proceedings of the 36th annual corn and sorghum industry research conference. 18-31), these genotypes retain more green leaf area and continue to fill grain normally (see, e.g., McBee GG, Waskom RM, Miller FR, Creelman RA (1983) Effect of senescence and non-senescence on carbohydrates in sorghum during late kernel maturity states. Crop Science 23: 372-377; Rosenow DT, Quisenberry JE, Wendt CW, Clark LE (1983) Drought-tolerant sorghum and cotton germplasm. Agricultural Water Management 7: 207-222; and, Bon-ell AK, Douglas ACL (1996) Maintaining green leaf area in grain sorghum increases yield in a water-limited environment. In: Foale MA, Henzell RG, Kneipp JF, eds. Proceedings of the third Australian sorghum conference. Melbourne: Australian Institute of Agricultural Science, Occasional Publication No. 93). The stay-green phenotype has also been used as a selection criterion for the development of improved varieties of com, particularly with regard to the development of drought-tolerance. See, e.g., Russell WA (1991) Genetic improvement of maize yields. Advances in Agronomy 46: 245-298; and, Bruce et al. (2002), "Molecular and physiological approaches to maize improvement for drought tolerance" Journal of Experimental Botany. 53 (366): 13-25.<br>
[0005]	Five fundamentally distinct types of stay-green have been described, which<br>
are Types A, B, C, D and E (see e.g., Thomas H, Smart CM (1993) Crops that stay green. Annals of Applied Biology 123: 193-219; and, Thomas and Howarth, supra). In Type A stay-green, initiation of the senescence program is delayed, but then proceeds at a normal rate. In Type B stay-green, while initiation of the senescence program is unchanged, the progression is comparatively slower. In Type C stay-green, chlorophyll is retained even though senescence (as determined through measurements of physiological function such as phoiosyiithetic capacity) proceeds at a normal rate. Type D stay-green is more artificial in that killing of the leaf (i.e. by freezing, boiling or drying) prevents initiation of the senescence program, thereby stopping the degradation of chlorophyll. In Type E stay-green, initial levels of chlorophyll are higher, while initiation and progression of leaf senescence are unchanged, thereby giving the illusion of a relatively slower progression rate. Type A and B are functional stay-greens, as photosynthetic capacity is maintained along with chlorophyll content, and these are the types associated with increased yield and drought tolerance in sorghum. Despite the potential importance of this trait, in particular the benefits associated with increasing yield and drought tolerance, very little progress has<br>
been made in understanding the biochemical, physiological or molecular basis for genetically determined stay-green (Thomas and Howarth, supra).<br>
[0006]	This invention solves these and other problems. The invention relates to the<br>
identification of ACC synthase genes associated with staygreen potential phenotype in plants and modulation of staygreen potential and/or ethylene production. Polypeptides encoded by these genes, methods for modulating staygreen potential in plants, methods for inhibiting ethylene production in plants, methods for modulating sterility in plants, and knockout plant cells and plants, as well as other features, will become apparent upon review of the following materials.<br>
SUMMARY OF THE INVENTION<br>
[0007]	This invention provides methods and compositions for modulating staygreen<br>
potential and sterility in plants and modulating (e.g., inhibiting) ethylene synthesis and/or production in plants. This invention also relates to ACC synthase nucleic acid sequences in plants, exemplified by, e.g., SEQ ID NO:1 through SEQ ID NO:6 and SEQ ID NO: 10, and a set of polypeptide sequences, e.g., SEQ ID NO:7 through SEQ ID NO:9 and SEQ ED NO: 11, which can modulate these activities.<br>
[0008]	In a first aspect, the invention provides for an isolated or-recombinant<br>
knockout plant cell comprising at least one disruption in at least one endogenous ACC synthase gene (e.g., a nucleic acid sequence, or complement thereof, comprising, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ TD NO:3 (gACS7)). The -disruption inhibits expression or activity of at least one ACC synthase protein compared to a corresponding control plant cell lacking the disruption. In one embodiment, the at least one endogenous ACC synthase gene comprises two or more endogenous ACC synthase genes (e.g., any two or more of ACS2, ACS6, and ACS7, e.g., ACS2 and ACS6). Similarly, in another embodiment, the at least one endogenous ACC synthase gene comprises three or more endogenous ACC synthase genes.   In certain embodiments, the disruption results in reduced ethylene production by the knockout plant cell as compared to the control plant cell.<br>
[0009]	In one embodiment, the at least one disruption in the knockout plant cell is<br>
produced by introducing at least one polynucleotide sequence comprising an ACC synthase nucleic acid sequence, or subsequence thereof, into a plant cell, such that the at least one polynucleotide sequence is linked to a promoter in a sense or antisense orientation, and where the at least one polynucleotide sequence comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more sequence identity to SEQ ID NO:1 (gACS2), SEQ ID N0:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7), or SEQ ID NO: 10 (CCRA178R) or a subsequence thereof, or a complement thereof. In another embodiment, the disruption is introduced into the plant cell by introducing at least one polynucleotide sequence comprising one or more subsequences of an ACC synthase nucleic acid sequence configured for RNA silencing or interference.<br>
[0010]	In another embodiment, the disruption comprises insertion of one or more<br>
transposons, where the one or more transposons are in the at least one endogenous ACC synthase gene. In yet another embodiment, the disruption comprises one or more point mutations in the at least one endogenous ACC synthase gene. The disruption can be a homozygous disruption in the at least one ACC synthase gene. Alternatively, the disruption is a heterozygous disruption in the at least one ACC synthase gene. In certain embodiments, when more than one ACC synthase gene is involved, there is more than one disruption, which can include homozygous disruptions, heterozygous disruptions or a combination of homozygous disruptions and heterozygous disruptions.<br>
L0011J	In certain embodiments, a plant cell of the invention is from a dicot or<br>
monocot. In one aspect, the plant cell is in a hybrid plant comprising a staygreen potential phenotype. In another aspect, the plant cell is in a plant comprising a sterility phenotype, e.g., a male sterility phenotype. Plants regenerated from the plant cells of the invention are also a feature of the invention.<br>
[0012]	The invention also provides for knockout plants that comprise a staygreen<br>
potential phenotype. For example, the invention provides for a knockout plant that comprises a staygreen potential phenotype, where the staygreen potential phenotype results from a disruption in at least one endogenous ACC synthase gene. In one embodiment, the disruption includes one or more transposons, and inhibits expression or activity of at least<br>
one ACC synthase protein compared to a corresponding control plant. In another embodiment, the disruption includes one or more point mutations in the endogenous ACC synthase gene and inhibits expression or activity of at least one ACC synthase protein compared to a corresponding control plant.   In certain embodiments, the at least one endogenous ACC synthase gene comprises a nucleic acid sequence comprising, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ED NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ ID NO:3 (gACS7), or a complement thereof. In certain embodiments, the knockout plant is a hybrid plant. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0013]	In another embodiment, a knockout plant includes a transgenic plant that<br>
comprises a staygreen potential phenotype. For example, a transgenic plant of the invention includes a staygreen potential phenotype resulting from at least one introduced transgene that inhibits ethylene synthesis. The introduced transgene includes a nucleic acid sequence encoding at least one ACC synthase or subsequence thereof, which nucleic acid sequence comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more sequence identity to SEQ ID NO:1 (gACS2), SEQ ID N0:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7) or SEQ ID NO: 10 (CCRA17SR), or a subsequence thereof, or a complement thereof, and is in a configuration that modifies a level of expression or activity of the at least one ACC synthase (e.g., a sense, antisense, RNA silencing or interference configuration). Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0014]	A transgenic plant of the invention can also include a staygreen potential<br>
phenotype resulting from at least one introduced transgene which inhibits ethylene synthesis, where said at least one introduced transgene comprises a nucleic acid sequence encoding a subsequence(s) of at least one ACC synthase, which at least one ACC synthase comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more sequence identity to SEQ ID NO:7 (pACS2), SEQ ID NO:8 (pACS6), SEQ ID NO.:9 (pACS7), or SEQ ID NO.: 11 (pCCRA178R), or a conservative variation thereof. The nucleic acid sequence is typically in an RNA silencing or interference configuration (or,<br>
e.g., a sense or antisense configuration), and modifies a level of expression or activity of the at least one ACC synthase. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0015]	The staygreen potential of a plant of the invention includes, but is not limited<br>
to, e.g., (a) a reduction in the production of at least one ACC synthase specific mRNA; (b) a reduction in the production of an ACC synthase; (c) a reduction in the production of ethylene; (d) a delay in leaf senescence; (e) an increase of drought resistance; (f) an increased time in maintaining photosynthetic activity; (g) an increased transpiration; (h) an increased stomatal conductance; (i) an increased CO^ assimilation; (j) an increased maintenance of CC&gt;2 assimilation; or (k) any combination of (a)-(j); compared to a corresponding control plant, and the like.<br>
[0016]	One aspect of the invention provides knockout or transgenic plants including<br>
sterility phenotypes, e.g., a male or female sterility phenotype. Thus, one class of embodiments provides a knockout plant comprising a male sterility phenotype (e.g., reduced pollen shedding) which results from at least one disruption in at least one endogenous ACC synthase gene. The disruption inhibits expression or activity of at least one ACC synthase protein compared to a corresponding control plant. Jn one embodiment, the at least one disruption results in reduced ethylene production by the knockout plant as compared to the control plant. In one embodiment, the at least one disruption includes one or more transposons, wherein the one or more transposons are in the at least one endogenous ACC synthase gene. In another embodiment, the at least one disruption comprises one or more point mutations, wherein the one or more point mutations are in the at least one endogenous ACC synthase gene. In yet another embodiment, the at least one disruption is introduced into the knockout plant by introducing at least one polynucleotide sequence comprising one or more subsequences of an ACC synthase nucleic acid sequence configured for RNA silencing or interference. In certain embodiments, the at least one endogenous ACC synthase gene comprises a nucleic acid sequence comprising, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ ID NO:3 (gACS7), or a complement thereof. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0017]	Another class of embodiments provides a transgenic knockout plant<br>
comprising a male sterility phenotype resulting from at least one introduced transgene which inhibits ethylene synthesis. The at least one introduced transgene comprises a nucleic acid sequence encoding at least one ACC synthase, which nucleic acid sequence comprises at least about 85% sequence identity to SEQ ID NO: 1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID N0:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7) or SEQ ID NO: 10 (CCRA178R), or a subsequence thereof, or a complement thereof, and is in a configuration that modifies a level of expression or activity of the at least one ACC synthase (e.g., an antisense, sense or RNA silencing or interference configuration). In certain embodiments, the transgene includes a tissue-specific promoter or an inducible promoter. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0018]	Polynucleotides are also a feature of the invention. In certain embodiments,<br>
an isolated or recombinant polynucleotide comprises a member selected from the group consisting of: (a) a polynucleotide, or a complement thereof, comprising, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7) or SEQ ID NO: 10 (CCRA17SR), or a subsequence thereof, or a conservative variation thereof; (b) a polynucleotide, or a complement thereof, encoding a polypeptide sequence of SEQ ID NO:7 (pACS2), SEQ ID NO:8 (pACS6), SEQ ID NO.:9 (pACS7), or SEQ ID NO: 11 (pCCRAHSR), or a subsequence thereof, or a conservative variation thereof; (c) a polynucleotide, or a complement thereof, that hybridizes under stringent conditions over substantially the entire length of a polynucleotide subsequence comprising at least 100 contiguous nucleotides of SEQ ID N0:l (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7), or SEQ ID NO: 10 (CCRA178R), or that hybridizes to a polynucleotide sequence of (a) or (b); and, (d) a polynucleotide that is at least about 85% identical to a polynucleotide sequence of (a), (b) or (c). In certain embodiments, the polynucleotide inhibits ethylene production when expressed in a plant.<br>
[0019]	The polynucleotides of the invention can comprise or be contained within an<br>
expression cassette or a vector (e.g., a viral vector). The vector or expression cassette can comprise a promoter (e.g., a constitutive, tissue-specific, or inducible promoter) operably linked to the polynucleotide. A polynucleotide of the invention can be linked to the promoter in an antisense orientation or a sense orientation, be configured for RNA silencing or interference, or the like.<br>
[0020]	The invention also provides methods for inhibiting ethylene production in a<br>
plant (and plants produced by such methods). For example, a method of inhibiting ethylene production comprises inactivating one or more ACC synthase genes in the plant, wherein the one or more ACC synthase genes encode one or more ACC synthases, wherein at least one of the one or more ACC synthases comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more identity to SEQ ID NO:7 (pACS2), SEQ ED NO:8 CpACS6), SEQ ED NO:9 (pAC7) or SEQ ID NO: 11 (pCCRA178R).<br>
[0021]	In one embodiment, the inactivating step comprises introducing one or more<br>
mutations into an ACC synthase gene sequence, wherein the one or more mutations in the ACC synthase gene sequence comprise one or more transposons, thereby inactivating the one or more ACC synthase genes compared to a corresponding control plant. In another embodiment, the inactivating step comprises introducing one or more mutations into an ACC synthase gene sequence, wherein the one or more mutations in the ACC synthase gene sequence comprise one or more point mutations, thereby inactivating the one or more ACC synthase genes compared to a corresponding control plant. The one or more mutations can comprise, for example, a homozygous disruption in the one or more ACC synthase genes, a heterozygous disruption in the one or more ACC synthase genes, or a combination of both homozygous disruptions and heterozygous disruptions if more than one ACC synthase gene is disrupted. In certain embodiments, the one or more mutations are introduced by a sexual cross. In certain embodiments, at least one of the one or more ACC synthase genes is, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, identical to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6) or SEQ ED NO:3 (gAC7), or a complement thereof).<br>
[0022]	In another embodiment, the inactivating step comprises: (a) introducing into<br>
w<br>
the plant at least one polynucleotide sequence, wherein the at least one polynucleotide<br>
sequence comprises a nucleic acid encoding one or more ACC synthases, or a subsequence thereof, and a promoter, which promoter functions in plants to produce an RNA sequence; and, (b) expressing the at least one polynucleotide sequence, thereby inactivating the one or more ACC synthase genes compared to a corresponding control plant (e.g., its non-transgenic parent or a non-transgenic plant of the same species). For example, the at least one polynucleotide sequence can be introduced by techniques including, but not limited to, electroporation, micro-projectile bombardment, Agrobacterium-mediated transfer, and the like. In certain aspects of the invention, the polynucleotide is linked to the promoter in a sense orientation or an antisense orientation, oris configured for RNA silencing or interference. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0023]	Methods for modulating staygreen potential in plants are also a feature of the<br>
invention (as are plants produced by such methods). For example, a method of modulating staygreen potential comprises: a) selecting at least one ACC synthase gene (e.g., encoding an ACC synthase, for example, SEQ ID NO:7 (pACS2), SEQ ID NO:8 (pACS6), SEQ ID NO:9 (pAC7) or SEQ ID NO: 11 (pCCRA178R)) to mutate, thereby providing at least one desired ACC synthase gene; b) introducing a mutant form (e.g., an antisense or sense configuration of at least one ACC synthase gene or subsequence thereof, an RNA silencing configuration of at least one ACC synthase gene or subsequence thereof, a heterozygous mutation in the at least one ACC synthase gene, a homozygous mutation in the at least one ACC synthase gene or a combination of homozygous mutation ana heterozygous mutation if more than one ACC synthase gene is selected, and the like) of the at least one desired ACC synthase gene into the plant; and, c) expressing the mutant form, thereby modulating staygreen potential in the plant. In one embodiment, selecting the at least one ACC synthase gene comprises determining a degree (e.g., weak, moderate or strong) of staygreen potential desired. In certain embodiments, the mutant gene is introduced by Agrobacterium-mediated transfer, electroporation, micro-projectile bombardment, a sexual cross, or the like. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0024]	Detection of expression products is performed either qualitatively (by<br>
detecting presence or absence of one or more product of interest) or quantitatively (by monitoring the level of expression of one or more product of interest). In one embodiment, the expression product is an RNA expression product. Aspects of the invention optionally include monitoring an expression level of a nucleic acid, polypeptide or chemical (e.g., ACC, ethylene, etc.) as noted herein for detection of ACC synthase, ethylene production, staygreen potential, etc. in a plant or in a population of plants.<br>
[0025]	The compositions and methods of the invention can include a variety of<br>
plants, e.g., a plant of the Poaceae (Gramineae) family. Examples of members of the Poaceae family include, by are not limited to, Acamptoclados, Achnatherum, Achnella, Acroceras, Aegilops, Aegopgon, Agroelymus, Agrohordeum, Agropogon, Agropyron, Agrositanion, Agrostis, Aira, Allolepis, Alloteropsis, Alopecurus, Amblyopyrum, Ammophila, Ampelodesmos, Amphibromus, Amphicarpum, Amphilophis, Anastrophus, Anatherum, Andropogron, Anemathele, Aneurolepidiwn, Anisantha, Anthaenantia, Anthephora, Anthochloa, Anthoxanthum, Apera, Apluda, Archtagrostis, Arctophila, Argillochloa, Aristida, Arrhenatherum, Arthraxon, Arthrostylidium, Arundinaria, Anmdinella, Arundo, Aspris, Atheropogon, Avena, Avenella, Avenochloa, Avenula, Axonopus, Bambusa, Beckmannia, Blepharidachne, Blepharoneiiron, Bothriochloa, Bouteloua, Brachiaria, Brachyelytnun, Brachypodium, Briza, Brizopynan, Bromelica, Bromopsis, Bromus, Buchloe, Bulbilis, Calamagrostis, Calamovilfa, Campulosus, Capriola, Catabrosa, Catapodiwn, Calhestecum, Cenchropsis, Cenchms, Centotheca, Ceratochloa, Chaetochloa, Chasinanthium, Chimonobambusa, Chionochloa, Chloris, Chondrosum, Chrysopon, C^iusqi-'ea.-Cinna, Cladoraphis, Codorachis, Coix, Coleanthus, Colpodhuu, Coridochloa, Cornucopiae, Cortaderia, Corynephorus, Cottea, Critesion, Crypsis, Ctenium, Cutandia, Cylindropyrum, Cymbopogon, Cynodon, Cynosurus, Cytrococcwn, Dactylis, Dacryloctenium, Danthonia, Dasyochloa, Dasypmm, Davyella, Dendrocalamus,<br>
r<br>
Deschampsia, Desmazeria, Deyeuxia, Diarina, Diarrhena, Dichanthelium, Dichanthiuin, Dichelachne, Diectomus, Digjtaria, Dimeria, Dimorpostachys, Dinebra, Diplachne, Dissanthelium, Dissochondrus, Distichlis, Drepanostachyum, Dupoa, Dupontia, Echinochloa, Ectospenna, Ehrharta, Eleusine, Elyhordeum, Elyleymus, Elymordeum, Elymus, Elyonurus, Elysitanion, Elytesion, Elytrigia, Enneapogon, Enteropogon, Epicampes, Eragrostis, Eremochloa, Eremopoa, Eremopyrum, Erianthus, Ericoma,<br>
Erichloa, Eriochrysis, Erioneuron, Euchlaena, Eudasta, Eulalia, Eulaliopsis, Eiistachys, Fargesia, Festuca, Festulolium, Fingerhuthia, Fluminia, Garnotia, Gastridium, Gaudinia, Gigantochloa, Glyceria, Graphephomm, Gymnopogon, Gynerium, Hackelochloa, Hainardia, Hakonechloa, Haynaldia, Heleochloa, Helictotrichon, Hemarthria, Flesperochloa, Hesperostipa, Heteropogon, Hibanobambusa, Hierochloe, Hilaria, Holcus, -Honmlocenchrus, Hordeum, Hydrochloa, Hymenachne, Hyparrhenia, Hypogynium, Hystrix, Ichnanthus, Imperata, Indocalamus, Isachne, Ischaemum, Ixophorus, Koeleria, Koiycarpus, Lagunis, Lamarckia, Lasiacis, Leersia, Leptochloa, Leptochloopsis, Leptocoryphium, Leptoloma, Leptogon, Lepturus, Lerchenfeldia, Leucopoa, Leymostachys, Leymus, Limnodea, Lithachne, Lollwn, Lophochlaena, Lophochloa, Lophopyruin, Ludolfia, Luziola, Lycums, Lygewn, Maltea, Manisuris, Megastachya, Melica, Melinis, Mibora, Microchloa, Microlaena, Microstegium, Milium, Miscanthus, Mnesithea, Molinia, Monanthochloe, Monerma, Monroa, Muhlenbergia, Nardus, Nassella, Nazia, Neeragrostis, Neoschischkinia, Neostapfia, Neyraudia, Nothoholcus, Olyra, Opizia, Oplismenus, Orcuttia, Oryza, Oryzopsis, Otatea, Oxytenanthera, Panicularia, Panicitm, Pappopliontm, Parapholis, Pascopyrum, Paspalidium, Paspalum, Pennisetum, Phalaris, Phalaroides, Phanopyrum, Pliarus, Phippsia, Phleurn, Pholiurus, Phragmites, Phyllostachys, Piptatherum, Piptochaetium, Pleioblastus, Pleopogon, Pleuraphis, Pleuropogon, Poa, Podagrostis, Polypogon, Polytrias, Psathyrostachys, Pseudelymus, Pseudoroegneria, Pseudosasa, Ptilagrostis, Puccinellia, Pucciphippsia, Redfleldia, Reimaria, Reimarochloa, Rhaphis, Rhombolytrum, Rhynchelytrum, Roegneria, Rostraria, Rottboellia, Rytilix, Saccharum, Sacciolepis, Sasa, Sasaella, Sasamorpha, Savastana, Schedonnardus, •Sc.hi.swus. Schirachne. Schizachyrium, Schizostachywn, Sclerochloa. Scleropoa. Scleropogon, Scolochloa, Scnbneria, Secale, Semiarundinaria, Sesleria, Setaria, Shibataea, Sieglingia, Sinanindinaria, Sinobambusa, Sinocalannis, Sitanion, Sorghastnun, Sorghum, Spartina, Sphenopholis, Spodiopogon, Sporobolus, Stapfia, Steinchisma, Stenotaphrum, Stipa, Stlpagrostis, Stiporyzopsis, Swallenia, Syntherisma, Taeniatherum, Terrellia, Terrelymus, Thamnocalamus, Themeda, Thinopyriim, Thuarea, Thysanolaena, Torresia, Torreyochloa, Trachynia, Trachypogon, Tragus, Trichachne, Trichloris, Tricholaena, Trichoneura, Tridens, Triodia, Triplasis, Tripogon, Tripsacum, Trisetobromus, Trisetum, Triticosecale, Triticum, Tuctoria, Uniola, Urachne, Uralepis, Urochloa, Vahlodea, Valota, Vaseyochloa, Ventenata, Vetiveria, Vilfa, Vulpia, Willkommia, Yushania, Zea, Zizania, Zizaniopsis, and Zoysia. In one embodiment, the plant is Zea mays, wheat, rice, sorghum,<br><br>
oarley, oat, lawn grass, rye, soybean, tomato, potato, pepper, broccoli, cabbage, a commercial com line, or the like.<br>
[0026]	Kits which incorporate one or more of the nucleic acids or polypeptides<br>
noted above are also a feature of the invention. Such kits can include any of the above noted components and further include, e.g., instructions for use of the components in any of the methods noted herein, packaging materials, containers for holding the components, and/or the like. For example, a kit for modulating staygreen potential in a plant includes a container containing at least one polynucleotide sequence comprising a nucleic acid sequence, where the nucleic acid sequence is, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, identical to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cAC7) or SEQ ID NO: 10 (CCRA178R), or a subsequence thereof, or a complement thereof. In a further embodiment, the kit includes instructional materials for the use of the at least one polynucleotide sequence to control staygreen potential in a plant. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
[0027]	As another example, a kit for modulating sterility, e.g., male sterility, in a<br>
plant includes a container containing at least one polynucleotide sequence comprising a nucleic acid sequence, wherein the nucleic acid sequence is, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, identical to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cAC7) or SEQ ID NO: 10 (CCRA17SR), or a subsequence thereof, or a complement thereof. The kit optionally also includes instructional materials for the use of the at least one polynucleotide sequence to control sterility, e.g., male sterility, in a plant. Essentially all of the features noted above apply to this embodiment as well, as relevant.<br>
DEFINITIONS<br>
[0028]	Before describing the invention in detail, it is to be understood that this<br>
invention is not limited to particular devices or biological systems, which can, of course, vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only, and is not intended to be limiting. As used in this<br><br>
specification and the appended claims, the singular forms "a", "an" and "the" include plural referents unless the content clearly dictates otherwise. Thus, for example, reference to "a cell" includes a combination of two or more cells, and the like.<br>
[0029]	Unless defined otherwise, all technical and scientific terms used herein have<br>
the same meaning as commonly understood by one of ordinary skill in the art to which the invention pertains. In describing and claiming the present invention, the following terminology will be used in accordance with the definitions set out below.<br>
[0030]	The term "plant" refers generically to any of: whole plants, plant parts or<br>
organs (e.g., leaves, stems, roots, etc.), shoot vegetative organs/structures (e.g. leaves, stems and tubers), roots, flowers and floral organs/structures (e.g. bracts, sepals, petals, stamens, carpels, anthers and ovules), seed (including embryo, endosperm, and seed coat), fruit (the mature ovary), plant tissue (e.g. vascular tissue, ground tissue, and the like), tissue culture callus, and plant cells (e.g. guard cells, egg cells, trichomes and the like), and progeny of same. Plant cell, as used herein, further includes, without limitation, cells obtained from or found in: seeds, cultures, suspension cultures, embryos, meristematic regions, callus tissue, leaves, roots, shoots, gametophytes, sporophytes, pollen, and microspores. Plant cells can also be understood to include modified cells, such as protoplasts, obtained from the aforementioned tissues.<br>
[0031]	The term "dicot" refers to a dicotyledonous plant. Dicotyledonous plants<br>
belong to large subclass of Angiosperms that have two seed-leaves (cotyledon).<br>
[0032]	The term "monocot" refers to a monocotyledonous plant, which in the<br>
developing plant has unly one cotyledon.<br>
[0033]	The term "knockout plant cell" refers to a plant cell having a disruption in at<br>
least one ACC synthase gene in the cell, where the disruption results in a reduced expression or activity of the ACC synthase encoded by that gene compared to a control cell. The knockout can be the result of, e.g., antisense constructs, sense constructs, RNA silencing constructs, RNA interference, genomic disruptions (e.g., transposons, tilling, lomologous recombination, etc.), and the like. The term "knockout plant" refers to a plant :hat has a disruption in at least one of its ACC synthase genss in at least one cell.<br>
[0034]	The term "transgenic" refers to a plant that has incorporated nucleic acid<br>
sequences, including but not limited to genes, polynucleotides, DNA, RNA, etc., which have been introduced into a plant compared to a non-introduced plant.<br>
[0035]	The term "endogenous" relates to any gene or nucleic acid sequence that is<br>
already present in a cell.<br>
[0036]	A "transposable element" (TE) or "transposable genetic element" is a DNA<br>
sequence that can move from one location to another in a cell. Movement of a transposable element can occur from episome to episome, from episome to chromosome, from chromosome to chromosome, or from chromosome to episome. Transposable elements are characterized by the presence of inverted repeat sequences at their termini. Mobilization is mediated enzymatically by a "transposase." Structurally, a transposable element is categorized as a "transposon," ("TN") or an "insertion sequence element," (IS element) based on the presence or absence, respectively, of genetic sequences in addition to those necessary for mobilization of the element. A hiini-transposon or mini-IS element typically lacks sequences encoding a transposase.<br>
[0037]	The term "nucleic acid" or "polynucleotide" is generally used in its art-<br>
recognized meaning to refer to a ribose nucleic acid (RNA) or deoxyribose nucleic acid (DNA) polymer, or analog thereof, e.g., a nucleotide polymer comprising modifications of the nucleotides, a peptide nucleic acid, or the like. In certain applications, the nucleic acid can be a polymer that includes multiple monomer types, e.g., both RNA and DNA subunits. A nucleic acid can be, e.g., a chromosome or chromosomal segment, a vector (e.g., an expression vector), an expression cassette, a naked DNA or RNA polymer, the product of a polymerase chain reaction (PCR), an oligonucleotide, a probe, etc. A nucleic acid can be e.g., single-stranded and/or double-stranded. Unless otherwise indicated, a particular nucleic acid sequence of this invention optionally comprises or encodes complementary sequences, in addition to any sequence explicitly indicated.<br>
[0038]	The term "polynucleotide sequence" or "nucleotide sequence" refers to a<br>
contiguous sequence of nucleotides in a single nucleic acid or to a representation, e.g., a character string, thereof. That is, a "polynucleotide sequence" is a polymer of nucleotides (an oligonucleotide, a DNA, a nucleic acid, etc.) or a character string representing a nucleotide polymer, depending on context. From any specified polynucleotide sequence,<br>
either the given nucleic acid or the complementary polynucleotide sequence (e.g., the complementary nucleic acid) can be determined.<br>
[0039]	The term "subsequence" or "fragment" is any portion of an entire sequence.<br>
[0040]	A "phenotype" is the display of a trait in an individual plant resulting from<br>
the interaction of gene expression and the environment.<br>
[0041]	An "expression cassette" is a nucleic acid construct, e.g., vector, such as a<br>
plasmid, a viral vector, etc., capable of producing transcripts and, potentially, polypeptides encoded by a polynucleotide sequence. An expression vector is capable of producing transcripts in an exogenous cell, e.g., a bacterial cell, or a plant cell, in vivo or in vitro, e.g., a cultured plant protoplast. Expression of a product can be either constitutive or inducible depending, e.g., on the promoter selected. Antisense, sense or RNA interference or silencing configurations that are not or cannot be translated are expressly included by this definition. In the context of an expression vector, a promoter is said to be "operably linked" to a polynucleotide sequence if it is capable of regulating expression of the associated polynucleotide sequence. The term also applies to alternative exogenous gene constructs, such as expressed or integrated transgenes. Similarly, the term operably linked applies equally to alternative or additional transcriptional regulatory sequences such as enhancers, associated with a polynucleotide sequence.<br>
[0042]	A polynucleotide sequence is said to "encode" a sense or antisense RNA<br>
molecule, or RNA silencing or interference molecule or a polypeptide, if the polynucleotide<br>
sequence can be transcribed (in spliced or unspliced form) and/or translated into the RNA or<br>
polyp^ptide, or a subsequence thereof.	.,	...<br>
[0043]	"Expression of a gene" or "expression of a nucleic acid" means transcription<br>
of DNA into RNA (optionally including modification of the RNA, e.g., splicing), translation of RNA into a polypeptide (possibly including subsequent modification of the polypeptide, e.g., posttranslational modification), or both transcription and translation, as indicated by the context.<br>
[0044]	The term "gene" is used broadly to refer to any nucleic acid associated with a<br>
biological function. Genes typically include coding sequences and/or the regulatory sequences required for expression of such coding sequences. The term "gene" applies to a specific genomic sequence, as well as to a cDNA or an mRNA encoded by that genomic<br>
sequence. Genes also include non-expressed nucleic acid segments that, for example, form recognition sequences for other proteins. Non-expressed regulatory sequences include promoters and enhancers, to which regulatory proteins such as transcription factors bind, resulting in transcription of adjacent or nearby sequences.<br>
[0045]	A "polypeptide" is a polymer comprising two or more ammo acid residues<br>
(e.g., a peptide or a protein). The polymer can additionally comprise non-amino acid elements such as labels, quenchers, blocking groups, or the like and can optionally comprise modifications such as glycosylation or the like. The amino acid residues of the polypeptide can be natural or non-natural and can be unsubstituted, unmodified, substituted or modified.<br>
[0046]	The term "recombinant" indicates that the material (e.g., a cell, a nucleic<br>
acid, or a protein) has been artificially or synthetically (non-naturally) altered by human intervention. The alteration can be performed on the material within, or removed from, its natural environment or state. For example, a "recombinant nucleic acid" is one that is made by recombining nucleic acids, e.g., during cloning, DNA shuffling or other procedures; a "recombinant polypeptide" or "recombinant protein" is a polypeptide or protein which is produced by expression of a recombinant nucleic acid. Examples of recombinant cells include cells containing recombinant nucleic acids and/or recombinant polypeptides.<br>
[0047]	The term "vector" refers to the means by which a nucleic acid can be<br>
propagated and/or transferred between organisms, cells, or cellular components. Vectors include plasmids, viruses, bacteriophage, pro-viruses, phagemids, transposons, and artificial chromosomes, and the like, that replicate autonomously or can integrate into a chromosome of a host cell. A vector can also be a naked RNA polynucleotide, a naked DNA polynucleotide, a polynucleotide composed of both DNA and RNA within the same strand, a poly-lysine-conjugated DNA or RNA, a peptide-conjugated DNA or RNA, a liposome-conjugated DNA, or the like, that are not autonomously replicating.<br>
[0048]	In the context of the invention, the term "isolated" refers to a biological<br>
material, such as a nucleic acid or a protein, which is substantially free from components that normally accompany or interact with it in its naturally occurring environment. The isolated material optionally comprises material not found with the material in its natural environment, e.g., a cell. For example, if the material is in its natural environment, such as a cell, the material has been placed at a location in the cell (e.g., genome or genetic element)<br>
not native to a material found in that environment. For example, a naturally occurring nucleic acid (e.g., a coding sequence, a promoter, an enhancer, etc.) becomes isolated if it is introduced by non-naturally occuningmeans to a locus of the genome (e.g., a vector, such as a plasmid or virus vector, or amplicon) not native to that nucleic acid. An isolated plant cell, for example, can be in an environment (e.g., a cell culture system, or purified from cell culture) other than the native environment of wild-type plant cells (e.g., a whole plant).<br>
[0049]	The term "variant" with respect to a polypeptide refers to an amino acid<br>
sequence that is altered by one or more amino acids with respect to a reference sequence. The variant can have "conservative" changes, wherein a substituted amino acid has similar structural or chemical properties, e.g., replacement of leucine with isoleucine. Alternatively, a variant can have "nonconservative" changes, e.g., replacement of a glycine with a tryptophan. Analogous minor variation can also include amino acid deletion or insertion, or both. Guidance in determining which amino acid residues can be substituted, inserted,.or deleted without eliminating biological or immunological activity can be found using computer programs well known in the art, for example, DNASTAR software. Examples of conservative substitutions are also described below.<br>
[0050]	A "host cell", as used herein, is a cell which has been transformed or<br>
transfected, or is capable of transformation or transfection, by an exogenous polynucleotide sequence. "Exogenous polynucleotide sequence" is defined to mean a sequence not naturally in the cell, or which is naturally present in the cell but at a different genetic locus, in different copy number, or under direction of a different regulatory element.<br>
[0051]	A "promoter", as used herein, includes reference to a region of DNA<br>
upstream from the start of transcription and involved in recognition and binding of RNA polymerase and other proteins to initiate transcription. A "plant promoter" is a promoter capable of initiating transcription in plant cells. Exemplary plant promoters include, but are not limited to, those that are obtained from plants, plant viruses, and bacteria which comprise genes expressed in plant cells, such as Agrobacterium or Rhizobiwn. Examples of promoters under developmental control include promoters that preferentially initiate transcription in certain tissues, such as leaves, roots, or seeds or spatially in regions such as endosperm, embryo, or meristematic regions. Such promoters are referred to as "tissue-preferred" or "tissue-specific". A temporally regulated promoter drives expression at particular times, such as between 0-25 days after pollination. A "cell-type-preferred"<br>
promoter primarily drives expression in certain cell types in one or more organs, for &gt; xample, vascular cells in roots or leaves. An "inducible" promoter is a promoter that is under environmental control and may be inducible or de-repressible. Examples of environmental conditions that may effect transcription by inducible promoters include anaerobic conditions or the presence of light. Tissue-specific, cell-type-specific, and inducible promoters constitute the class of "non-constitutive" promoters. A "constitutive" promoter is a promoter that is active under most environmental conditions and in all or nearly all tissues, at all or nearly all stages of development.<br>
[0052]	"Transformation", as used herein, is the process by which a ceil is<br>
"transformed" by exogenous DNA when such exogenous DNA has been introduced inside the cell membrane. Exogenous DNA may or may not be integrated (covalently linked) into chromosomal DNA making up the genome of the cell. In prokaryotes and yeasts, for example, the exogenous DNA may be maintained on an episomal element, such as a plasmid. With respect to higher eukaryotic cells, a stably transformed or transfected cell is one in which the exogenous DNA has become integrated into the chromosome so that it is inherited by daughter cells through chromosome replication. This stability is demonstrated by the ability of the eukaryotic cell to establish cell lines or clones comprised of a population of daughter cells containing the exogenous DNA.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0053]	Figure 1 schematically illustrates ethylene biosynthetic and signaling genes<br>
in plants, e.g., Arabidopsis.<br>
[0054]	Figure 2 schematically illustrates isolated and mapped ACC synthase genes<br>
and Mu insertion mutations. ACC6 is also known as ACS6, ACC2 is also known as ACS2, and ACC7 is also known as ACS7.<br>
[0055]	Figure 3, Panels A, B, C, and D illustrate heterozygous ACC synthase<br>
knockouts in plants, e.g., maize. Panel A and Panel B illustrate heterozygous ACC synthase knockout plants in a field of wild-type plants. Panel C and Panel D illustrate leaves from a heterozygous ACC synthase knockout plant, left side of panel, compared to ' leaves from an ACC synthase wild-type plant, right side of panel.<br>
J0056]	Figure 4 illustrates an enhanced staygreen trait observed in leaves of plants<br>
that are homozygous ACC synthase knockouts (right) compared to wild type leaves (left) and heterozygous knockout leaves (middle).<br>
[0057]	Figure 5, Panels A, B, C, D, E, and F illustrate leaf transpiration (Panels A<br>
and D), stomatal conductance (Panels B and E) and COs assimilation (Panels C and F) for wild-type (B73, +/+) and ACS6 null (15,O/O) mutant leaves under control conditions (Panels A, B, and C) or drought conditions (Panels D, E, and F). For control conditions, plants were grown under well-watered conditions and each leaf on a plant was measured at forty days after pollination (dap). For drought conditions, plants were grown under limited water conditions and each leaf on a plant was measured at forty days after pollination. Values represent a mean of six determinations.<br>
[0058]	Figure 6, Panels A, B and C illustrate leaf transpiration (Panel A), stomatal<br>
conductance (Panel B) and CO2 assimilation (Panel C) for wild-type (B73, +/+), ACS2 null (7, O/O) and ACS6 null (15, O/O) mutant leaves. Plants were grown under limited water conditions and each leaf on a plant was measured at forty days after pollination. Values represent a mean of six determinations.<br>
[0059]	Figure 7 schematically illustrates phylogenetic analysis of ACC synthase<br>
gene sequences, where maize sequences are indicated by {A47 (also known as ACS2 or ACC2 herein), A50 (also known as ACS7 or ACC7 herein), A65 (also known as ACS6 or ACC6 herein)), Arabidopsis sequences are indicated by (AtACS...), tomato sequences are indicated by (LeACS...), rice sequences are indicated by (indica (OsiACS...),&amp; japonica (OsjACS...)), wheat sequences are indicated by (TaACS...), and banana sequences are indicated by (MaACS...).<br>
[0060]	Figure 8 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS2 or ACC2), A50 (also known as ACS7 or ACC7), and A65 (also known as ACS6 or ACC6) with both dicot (AtACS, LeACS) and monocot (OsiACS, OsjACS, TaACS, and MaACS) species. The alignment is done with a most stringent criteria (identical amino acids) and the plurality is 26.00, the threshold is 4, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25. SEQ ID NOs are as follows: A47pep SEQ ID NO:25; ASOpep SEQ ID NO:26; osiacslpep SEQ ID NO:27; osjacslpep SEQ ID NO:28; TaACS2pep SEQ ID NO:29; AtACSlpep SEQ ID NO:30; AtACS2pep SEQ ID NO:31; LeACS2pep SEQ ID NO:32; LeACS4pep SEQ ID<br>
NO:33; MaACSlpep SEQ ID NO:34; MaACSSpep SEQ ID NO:35; LeACSl Apep SEQ ID "NO:36; LeACSlBpep SEQ ID NO:37; LeACS6pep SEQ ED NO:38; AtACS6pep SEQ ID NO:39; A65pep SEQ ID NO:40; osiacs2pep SEQ ID NO:41; AtACSSpep SEQ ID NO:42; AtACS9 SEQ ID NO:43; AtACS4pep SEQ ID N0:44; AtACSS SEQ ED NO:45; MaACS2pep SEQ ID NO:46; MaACS3pep SEQ ID NO:47; LeACSSpep SEQ ID NO:4S; LeACS7pep SEQ ID NO:49; OsjACS2pep SEQ ID NO.50; OsjACS3 SEQ ID NO:51; OsiACS3pep SEQ ID NO:52; and AtACS? SEQ ID NO:53.<br>
[0061]	Figure 9 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS2 or ACC2), A50 (also known as ACS7 or ACC7), and A65 (also known as ACS6 or ACC6) with both dicot (AtACS, LeACS) and monocot (OsiACS, OsjACS, TaACS, MaACS) species (SEQ ID NOs:25-53). The alignment is done with a stringent criteria (similar amino acid residues) and the plurality is 26.00, the threshold is 2, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25.<br>
[0062]	Figure 10 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS2 or ACC2), A50 (also known as ACS7 or ACC7), and A65 (also known as ACS6 or ACC6) with both dicot (AtACS, LeACS) and monocot (OsiACS, OsjACS, TaACS, MaACS) species (SEQ ID NOs:25-53). The alignment is done with a less stringent criteria (somewhat similar amino acid residues) and the plurality is 26.00, the threshold is 0, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25.<br>
[0063]	Figure 11 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS 2 or ACC2), and A50 (also known as ACS7 or ACC7) with sequences that are most similar to ACS2 and ACS7 (SEQ ID NOs:25-39). The alignment is done with most stringent criteria (identical amino acids) and the plurality is 15.00, the threshold is 4, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25.<br>
[0064]	Figure 12 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS2 or ACC2), and A50 (also known as ACS7 or ACC7) with sequences that are most similar to ACS2 and ACS7 (SEQ ID NOs:25-39). The alignment is done with stringent criteria (similar amino acid residues) and the plurality<br>
is 15.00, the threshold is 2, the AveWeight is 1.00, the AveMatch is 2.78 and the \vMisMatch is -2.25.<br>
[0065]	Figure 13 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A65 (also known as ACS6 or ACC6) with sequences that are most similar to ACS6 (SEQ ID NOs:40-53). The alignment is done with most stringent criteria "(identical amino acids) 'and the plurality is 14.00, the threshold is 4, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25.<br>
[0066]	Figure 14 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A65 (also known as ACS6 or ACC6) with sequences that are most similar to ACS6 (SEQ ED NOs:40-53). The alignment is done with stringent criteria (similar amino acid residues) and the plurality is 14.00, the threshold is 2, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25.<br>
[0067]	Figure 15 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS2 or ACC2), A50 (also known as ACS7 or<br>
ACC7), and A65 (also known as ACS6 or ACC6) (SEQ ID NOs:25-26 and 40). The<br>
alignment is done with most stringent criteria (identical amino acids) and the plurality is<br>
3.00, the threshold is 4, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch<br>
is-2.25.	'<br>
[0068]	Figure 16 illustrates a peptide consensus sequence alignment with ACC<br>
synthase sequences of A47 (also known as ACS2 or ACC2), A50 (also known as ACS7 or ACC7), and A65 (also known as ACS6 or ACC6) (SEQ ID NOs:25-26 and 40). The alignment is done with stringent criteria (similar amino acid residues) and the plurality is 3.00, the threshold is 2, the AveWeight is 1.00, the AveMatch is 2.78 and the AvMisMatch is -2.25.<br>
[0069]	Figure 17 Panels A, B, C, and D illustrate total chlorophyll data for wild-<br>
type and ACC synthase knockout plants. Panels A and B illustrate total chlorophyll data for wild-type (B73, +/+), ACS2 null (0/0), and ACS6 null (0/0) plants 40 days after pollination for plants grown under normal conditions (Panel A) or drought conditions (Panel B). Panel C compares total chlorophyll for wild-type (B73, +/+) and ACS6 null (0/0) plants 40 days after pollination under normal and drought conditions. Panel D<br>
illustrates a comparison of total chlorophyll for B73 (wild-type) plants collected at 30 and 40 days after pollination.<br>
[0070]	Figure 18 Panels A, B, C, and D illustrate soluble protein data for wild-type<br>
and ACC synthase knockout plants. Panels A and B illustrate soluble protein data for wild-type (B73, +/+), ACS2 null (0/0), and ACS6 null (0/0) plants 40 days after pollination for plants grown under normal conditions (Panel A) or drought conditions (Panel B). Panel C compares soluble protein for wild-type (B73, +/+) and ACS6 null (0/0) plants 40 days after pollination under normal and drought conditions. Panel D illustrates a comparison of soluble protein for B73 (wild-type) plants collected at 30 and 40 days after pollination.<br>
[0071]	Figure 19, Panels A and B illustrate ethylene production in seedling leaves.<br>
Panel A illustrates various lines. In Panel B, the seedling leaves are averaged by genotype. In Panel C, ethylene production was determined for every leaf of wild-type (i.e., B73) plants at 20, 30, and 40 DAP. Leaf 1 represents the oldest surviving leaf and leaf 11 the youngest. Three replicates were measured and the average and standard deviation reported.<br>
[0072]	Figure 20 Panel A illustrates chlorophyll data, Panel B soluble protein, and<br>
Panel C Rubisco expression. The level of chlorophyll a+b (Panel A) and soluble protein (Panel B) was measured in the third oldest leaf (Leaf 3), sixth oldest leaf (Leaf 6), and ninth oldest leaf (Leaf 9) of adult wild-type (i.e., ACS6/ACS6), acs2/acs2, and acs6lacs6 plants following dark treatment for 7 days. Plants were watered daily. Additional acs6lacs6 plants were watered daily with 100 uM ACC during the treatment. acs6lacs6 leaves watered with 100 j.iM ACC but kept unsheathed are also shown. The average and standard deviation of leaves from three individual plants is shown. (Panel C) Western analysis of the same leaves was performed using rice anti-Rubisco antiserum. Soluble protein from leaf samples of equal fresh weight was used.<br>
[0073]	Figure 21 Panels A-C illustrate the ACS2 hairpin construct. Panel A is a<br>
schematic diagram of PHP20600 containing a ubiquitin promoter (UBI1ZM PRO) driving expression of the ACS2 hairpin (a terminal repeat consisting of TR1 and TR2). RB represents the Agrobacterium right border sequence. A 4126 bp fragment of the 49682 bp cassette is illustrated. Panel B presents the sequence of ZM-ACS2 TR1 (SEQ ID NO:54), and Panel C presents the sequence of ZM-ACS2 TR2 (SEQ ID NO:55).<br>
J0074]	Figure 22 Panels A-C illustrate the ACS6 hairpin construct. Panel A is a<br>
' schematic diagram of PHP20323 containing a ubiquitin promoter (UBI1ZM PRO) driving expression of the ACS6 hairpin (a terminal repeat consisting of TR1 and TR2). RB represents the Agrobacterium right border sequence. A 3564 bp fragment of the 49108 bp cassette is illustrated. Panel B presents the sequence of ZM-ACS6 TR1 (SEQ ID NO:56), - and Panel C presents the sequence of ZM-ACS6 TR2 (SEQ ID NO:57).<br>
[0075]	Figure 23 Panels A and B illustrate events generated for ACS2- and ACS6-<br>
hairpin constructs. Panel A presents a diagram showing the number of individual events for ACS2 hairpin (PHP20600) and the associated transgene copy number per event. Panel B presents a diagram showing the number of individual events for ACS6 hairpin (PHP20323) and the associated transgene copy number per event.<br>
DETAILED DESCRIPTION<br>
[0076]	"Stay-green" is a term commonly used to describe a plant phenotype.<br>
Staygreen is a desirable trait in commercial agriculture, e.g., a desirable trait associated with grain filling. As described herein, five fundamentally distinct types of stay-green have been described, including Types A, B, C, D, and E (see, e.g., Thomas H and Smart CM (1993) Crops that stay green. Annals of Applied Biology 123: 193-219; and Thomas H and Howarth CJ (2000) Five ways to stay green. Journal of Experimental Botany 51: 329-337). However, there is very little description of the biochemical, physiological or molecular basis for genetically determined stay-green. See, e.g., Thomas and Howarth, supra. This invention provides a molecular/biochemical basis for staygreen potential.<br>
[0077]	A number of environmental and physiological conditions have been shown<br>
to significantly alter the timing and progression of leaf senescence and can provide some insight into the basis for this trait. Among environmental factors, light is probably the most significant, and it has long been established that leaf senescence can be induced in many plant species by placing detached leaves in darkness. See, e.g., Weaver LM and Amasino RM (2001) Senescence is induced in individually darkened Arabidopsis leaves, but inhibited in whole darkened plants. Plant Physiology 127: 876-886. Limited nutrient and water availability have also been shown to induce leaf senescence prematurely. See, e.g., Rosenow DT, et al. (1983) Drought-tolerant sorghum and cotton gennplasm. Agricultural Water Management 7: 207-222. Among physiological determinants, growth regulators play<br>
a key role in directing the leaf senescence program. Modification of cytokinin levels can significantly delay leaf senescence. For example, plants transformed with isopentenyl transferase (ipt), an Agrobacterium gene encoding a rate-limiting step in cytokinin biosynthesis, when placed under the control of a senescence inducible promoter, resulted in autoregulated cytokinin production and a strong stay-green phenotype. See, e.g., Can S and Amasino RM (1995) Inhibition of leaf senescence by autoregulated production of cytokinin. Science 270: 1986-1988. However, there are other factors that are involved with this trait.<br>
[0078]	For example, ethylene has also been implicated in controlling leaf<br>
senescence (see, e.g., Davis KM and Grierson D (1989) Identification ofcDNA clones for tomato (Lycopersicon esculentum Mill.) mRNAs that accumulate during fruit ripening and leaf senescence in response to ethylene. Planta 179: 73-80) and some dicot plants impaired in ethylene production or perception also show a delay in leaf senescence (see, e.g., Picton S, et al. (1993) Altered fruit ripening and leaf senescence in tomatoes expressing an antisense ethylene-forming enzyme transgene. The Plant Journal 3: 469-481; Grbic V and Bleeker AB (1995) Ethylene regulates the timing of leaf senescence inArabidopsis. The Plant Journal 8: 95-102; and, John I, et al. (1995) Delayed leaf senescence in ethylene-deficient ACC-oxidase antisense tomato plants: molecular and physiological analysis. The Plant Journal 7: 483-490), which can be phenocopied by exogenous application of inhibitors of ethylene biosynthesis and action (see, e.g., Abeles FB, et al. (1992) Ethylene in Plant Biology. Academic Press, San Diego, CA).<br>
[0079]	Ethylene perception involves membrane-localized receptors that, e.g., in<br>
Arabidopsis include ETR1, ERS1, ETR2, ERS2, and EIN4 (see Figure 1). ETR1, ETR2, and EIN4 are composed of three domains, an N-terminal ethylene binding domain, a putative histidine protein kinase domain, and a C-terminal receiver domain, whereas ERS1 and ERS2 lack the receiver domain. These genes have been grouped into two subfamilies based on homology, where ETR1 and ERS1 comprise one subfamily and ETR2, ERS2 and EIN4 comprise the other. In Arabidopsis, analysis of loss-of-function mutants has revealed that ethylene inhibits the signaling activity of these receptors and subsequently their ability to activate CTR1, a negative regulator of ethylene responses that is related to mammalian RAF-type serine/threonine kinases. Ethylene signal transduction pathway suggests that ethylene binding to the receptor inhibits its own kinase activity, resulting in decreased activity of CTR1, and consequently, an increase in EIN2 activity (which acts downstream of<br>
CTR1) ultimately leads to increases in ethylene responsiveness. Differential expression of members of the ethylene receptor family has been observed, both developmentally and in response to ethylene.<br>
[0080]	The identification and analysis of mutants in Arabidopsis and tomato that are<br>
deficient in ethylene biosynthesis and perception are valuable in establishing the role that ethylene plays in plant growth and development. Mutant analysis has also been instrumental in identifying and characterizing the ethylene signal transduction pathway. While many ethylene mutants have been identified in dicot plants (e.g., Arabidopsis and tomato), no such mutants have been identified in monocots (e.g., rice, wheat, and com). Herein are described, e.g., ethylene mutants (e.g., in a monocot) deficient in ACC synthase, the first enzyme in the ethylene biosynthetic pathway.<br>
[0081]	This invention provides ACC synthase polynucleotide sequences from<br>
plants, which modulate staygreen potential in plants and ethylene production, exemplified by, e.g., SEQ ID NO:1 through SEQ ID NO:6 and SEQ ID NO: 10 and, e.g., a set of polypeptide sequences which modulate staygreen potential in plants and/or ethylene production, e.g., SEQ ID NO:7 through SEQ ID NO:9 and SEQ ID NO: 11. The invention also provides knockout plant cells deficient in ACC synthase and knockout plants having a staygreen potential phenotype, as well as knockout plants having a male sterility phenotype. The plants of the invention can have the characteristic of regulating responses to environmental stress better than control plants, e.g., higher tolerance to drought stress. Plants of the invention can also have a higher tolerance for other stresses (e.g., crowding in, e.g., maize) compared to a control plants. Thus, plants of the invention can be planted at higher densities than currently practiced by fanners. In addition, plants of the invention are critical in elucidating the regulatory roles that ethylene plays throughout plant development as well as its role in regulating responses to stress, e.g., drought, crowding, etc.<br>
ETHYLENE IN PLANTS<br>
[0082]	Ethylene (€2^4) is a gaseous plant hormone. It has a varied spectrum of<br>
effects that can be tissue and/or species specific. For example, physiological activities include, but are not limited to, promotion of food ripening, abscission of leaves and fruit of dicotyledonous species, flower senescence, stem extension of aquatic plants, gas space (aerenchyma) development in roots, leaf epinastic curvatures, stem and shoot swelling (in<br>
association with stunting), femaleness in curcubits, fruit growth in certain species, apical hook closure in etiolated shoots, root hair formation, flowering in the Bromeliaceae, diageotropism of etiolated shoots, and increased gene expression (e.g., of polygalacturonase, cellulase, chitinases, (3-1,3-glucanases, etc.). Ethylene is released naturally by ripening fruit and is also produced by most plant tissues, e.g., in response to ' stress (e.g., drought, crowding, disease or pathogen attack, temperature (cold or heat) stress, wounding, etc.), and in maturing and senescing organs.<br>
[0083]	Ethylene is generated from methionine by a well-defined pathway involving<br>
the conversion of S-adenosyl-L-methionine (SAM or Ado Met) to the cyclic amino acid 1-aminocyclopropane-1-carboxylic acid (ACC) which is facilitated by ACC synthase (see, e.g., Figure 1). Sulphur is conserved in the process by recycling 5'-methythioadenosine.<br>
[0084]	ACC synthase is an aminotransferase which catalyzes the rate limiting step<br>
in the formation of ethylene by converting S-adenosylmethionine to ACC. Typically, the enzyme requires pyridoxal phosphate as a cofactor. ACC synthase is typically encoded in multigene families. Examples include SEQ ED NOs:l-3, described herein. Individual members can exhibit tissue-specific regulation and/or are induced in response to environmental and chemical stimuli. Features of the invention include ACC synthase sequences and subsequences. See the section herein entitled "Polynucleotides and Polypeptides of the Invention."<br>
[0085]	Ethylene is then produced from the oxidation of ACC through the action of<br>
ACC oxidase (also known as the ethylene forming enzyme) with hydrogen cyanide as a secondary product that is detoxified by p-cyanoalanine synthase. ACC oxidase is encoded by multigene families in which individual members exhibit tissue-specific regulation and/or are induced in response to environmental and chemical stimuli. Activity of ACC oxidase can be inhibited by anoxia and cobalt ions. The ACC oxidase enzyme is stereospecific and uses cof actors, e.g., Fe+2, 02, ascorbate, etc. Finally, ethylene is metabolized by oxidation to or to ethylene oxide and ethylene glycol.<br>
POLYNUCLEOTIDES AND PQLYPEPTIDES OF THE INVENTION<br>
[0086]	The invention features the identification of gene sequences, coding nucleic<br>
acid sequences, and amino acid sequences of ACC synthase, which are associated, e.g., with staygreen potential in plants and/or ethylene production. The sequences of the<br>
invention can influence staygreen potential in plants by modulating the production of Jthylene.<br>
[0087]	Polynucleotide sequences of the invention include, e.g., the polynucleotide<br>
sequences represented by SEQ ID NO:1 through SEQ ID NO:6 and SEQ ED NO: 10, and subsequences thereof. In addition to the sequences expressly provided in the accompanying sequence listing, the invention includes polynucleotide sequences that are highly related structurally and/or functionally. For example, polynucleotides encoding polypeptide sequences represented by SEQ ID NO:7 through SEQ ID NO:9 and SEQ DO NO: 11, or subsequences thereof, are one embodiment of the invention. In addition, polynucleotide sequences of the invention include polynucleotide sequences that hybridize under stringent conditions to a polynucleotide sequence comprising any of SEQ ID NO:1-SEQ ID NO:6 and SEQ ID NO: 10, or a subsequence thereof (e.g., a subsequence comprising at least 100 contiguous nucleotides). Polynucleotides of the invention also include ACC synthase sequences and/or subsequences configured for RNA production, e.g., mRNA, antisense RNA, sense RNA, RNA silencing and interference configurations, etc.<br>
[0088]	In addition to the polynucleotide sequences of the invention, e.g.,<br>
enumerated in SEQ ID NO:1 to SEQ ED NO:6 and SEQ DD NO: 10, polynucleotide sequences that are substantially identical to a polynucleotide of the invention can be used in the compositions and methods of the invention. Substantially identical or substantially similar polynucleotide sequences are defined as polynucleotide sequences that are identical, on a nucleotide by nucleotide bases, with at least a subsequence of a reference polynucleotide, e.g., selected from SEQ ID NOs:l-6 and 10. Such polynucleotides can include, e.g., insertions, deletions, and substitutions relative to any of SEQ ID NOs:l-6 and 10. For example, such polynucleotides are typically at least about 70% identical to a reference polynucleotide selected from among SEQ ID NO:1 through SEQ ED NO:6 and SEQ ID NO: 10, or subsequence thereof. For example, at least 7 out of 10 nucleotides within a window of comparison are identical to the reference sequence selected, e.g., from SEQ ID NO: 1-6 and 10. Frequently, such sequences are at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 98%, at least about 99%, or at least about 99.5%, identical to the reference sequence, e.g., at least one of SEQ ID NO:1 to SEQ ID NO:6 or SEQ ID NO: 10. Subsequences of the polynucleotides of the invention described above, e.g., SEQ ID NOs: 1-6 and 10, including,<br>
e.g., at least about 5, at least about 10, at least about 15, at least about 20, at least about 25, 1 at least about 50, at least about 75, at least about 100, at least about 500, about 1000 or more, contiguous nucleotides or complementary subsequences thereof are also a feature of the invention. Such subsequences can be, e.g., oligonucleotides, such as synthetic oligonucleotides, isolated oligonucleotides, or full-length genes or cDNAs.<br>
[0089]	In addition, polynucleotide sequences complementary to any of the above<br>
described sequences are included among the polynucleotides of the invention.<br>
[0090]	Polypeptide sequences of the invention include, e.g., the amino acid<br>
sequences represented by SEQ ID NO:7 through SEQ ID NO:9 and SEQ ID NO: 11, and subsequences thereof. In addition to the sequences expressly provided in the accompanying sequence listing, the invention includes amino acid sequences that are highly related structurally and/or functionally. For example, in addition to the amino acid sequences of the invention, e.g., enumerated in SEQ ID NO:7 to SEQ ID NO:9 and SEQ ID NO: 11, amino acid sequences that are substantially identical to a polypeptide of the invention can be used in the compositions and methods of the invention. Substantially identical or substantially similar amino acid sequences are defined as amino acid sequences that are identical, on an amino acid by amino acid bases, with at least a subsequence of a reference polypeptide, e.g., selected from SEQ ID NOs:7-9 and 11'. Such polypeptides.can include, e.g., insertions, deletions, and substitutions relative to any of SEQ ID NOs:7-9 and 11. For example, such polypeptides are typically at least about 70% identical to a reference polypeptide selected from among SEQ ID NO:7 through SEQ ID NO:9 and SEQ ID NO: 11, or a subsequence thereof. For example, at least 7 out of 10 amino acids within a window of comparison are identical to the reference sequence selected, e.g., from SEQ ID NO:7-9 and 11. Frequently, such sequences are at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 98%, at least about 99%, or at least about 99.5%, identical to the reference sequence, e.g., at least one of SEQ ID NO:7 to SEQ ID NO:9 or SEQ ID NO: 11. Subsequences of the polypeptides of the invention described above, e.g., SEQ ID NOs:7-9 and 11, including, e.g., at least about 5, at least about 10, at least about 15, at least about 20, at least about 25, at least about 50, at least about 75, at least about 100, at least about 500, about 1000 or more, contiguous amino acids are also a feature of the invention. Conservative variants of amino acid sequences or subsequences of the invention are also amino acid sequences of the invention. Polypeptides<br>
of the invention are optionally immunogenic, enzymatically active, enzymatically inactive, and the like.<br>
[0091]	Where the polynucleotide sequences of the invention are translated to form a<br>
polypeptide or subsequence of a polypeptide, nucleotide changes can result in either conservative or non-conservative amino acid substitutions. Conservative amino acid substitutions refer to the interchangeability of residues having functionally similar side chains. Conservative substitution tables providing functionally similar amino acids are well known in the art. Table 1 sets forth six groups which contain amino acids that are "conservative substitutions" for one another. Other conservative substitution charts are available in the art, and can be used in a similar manner.<br>
Table 1: Conservative Substitution Group<br>
(Table Removed)<br>
[0092]	One of skill in the art will appreciate that many conservative substitutions of<br>
the nucleic acid constructs which are disclosed yield functionally identical constructs. For example, as discussed above, owing to the degeneracy of the genetic code, "silent substitutions" (i.e., substitutions in a nucleic acid sequence which do not result in an alteration in an encoded polypeptide) are an implied feature of every nucleic acid sequence which encodes an amino acid. Similarly, "conservative amino acid substitutions," in one or a few amino acids in an amino acid sequence (e.g., about 1%, 2%, 3%, 4%, 5%, 6%, 7%, 8%, 9%, 10% or more) are substituted with different amino acids with highly similar properties, are also readily identified as being highly similar to a disclosed construct. Such conservative variations of each disclosed sequence are a feature of the invention.<br>
[0093]	Methods for obtaining conservative variants, as well as more divergent<br>
versions of the nucleic acids and polypeptides of the invention, are widely known in the art. In addition to naturally occurring homologues which can be obtained, e.g., by screening<br>
enomic or expression libraries according to any of a variety of well-established protocols, see, e.g., Ausubel et al. Current Protocols in Molecular Biology (supplemented through 2004) John Wiley &amp; Sons, New York ("Ausubel"); Sambrook et al. Molecular Cloning - A Laboratory Manual (2nd Ed.), Vol. 1-3, Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989 ("Sambrook"), and Berger and Kimmel Guide to Molecular Cloning Techniques.-Methods in Enzymology volume 152 Academic Press, Inc., San Diego, CA ("Berger"), additional variants can be produced by any of a variety of mutagenesis procedures. Many such procedures are known in the art, including site directed mutagenesis, oligonucleotide-directed mutagenesis, and many others. For example, site directed mutagenesis is described, e.g., in Smith (1985) "In vitro mutagenesis" Ann. Rev. Genet. 19:423-462, and references therein, Botstein &amp; Shortle (1985) "Strategies and applications of in vitro imitagenesis" Science 229:1193-1201; and Carter (1986) "Site-directed mutagenesis" Biochem. J. 237:1-7. Oligonucleotide-directed mutagenesis is described, e.g., in Zoller &amp; Smith (1982) "Oligonucleotide-directed mutagenesis using M13'-derived vectors: an efficient and general procedure for the production of point mutations in any DNA fragment" Nucleic Acids Res. 10:6487-6500). Mutagenesis using modified bases is described e.g., in Kunkel (1985) "Rapid and efficient site-specific mutagenesis without phenotypic selection" Proc. Natl. Acad. Sci. USA "82:488-492, and Taylor et al. (1985) "T7z rapid generation of oligonucleotide-directed mutations at high frequency usingphosphorothioate-modifiedDNA" Nucl. Acids Res. 13: 8765-8787. Mutagenesis using gapped duplex DNA is described, e.g., in Kramer et al. (1984) "The gapped duplex DNA approach to oligonucleotide-directed mutation construction' Nucl. Acids Res. 12: 9441-9460). Point mismatch mutagenesis is described, e.g., by Kramer et al. (1984) "Point Mismatch Repair" Cell 38:879-887). Double-strand break mutagenesis is described, e.g., in Mandecki (1986) "Oligonucleotide-directed double-strand break repair in plasmids of Escherichia coli: a method for site-specific mutagenesis" Proc. Natl. Acad. Sci. USA. 83:7177-7181, and in Arnold (1993) "Protein engineering for unusual environments" Current Opinion in Biotechnology 4:450-455). Mutagenesis using repair-deficient host strains is described, e.g., in Carter et al. (1985) "Improved oligonucleotide site-directed mutagenesis using M13 vectors" Nucl. Acids Res. 13: 4431-4443. Mutagenesis by total gene synthesis is described e.g., by Nambiar et al. (1984) "Total synthesis and cloning of a gene coding for the ribonudease S protein" Science 223: 1299-1301. DNA shuffling is described, e.g., by Stemmer (1994) "Rapid evolution of a protein in<br>
vitro by DNA shuffling" Nature 370:389-391, and Stemmer (1994) "DNA shuffling by random fragmentation and reassembly: In vitro recombination for molecular evolution." Proc. Natl. Acad. Sci. USA 91:10747-10751.<br>
[0094]	Many of the above methods are further described in Methods in Enzymology<br>
Volume 154, which also describes useful controls for trouble-shooting problems with various mutagenesis methods. Kits for mutagenesis, library construction and other diversity generation methods are also commercially available. For example, kits are available from, e.g., Amersham International pic (Piscataway, NJ) (e.g., using the Eckstein method above), Bio/Can Scientific (Mississauga, Ontario, CANADA), Bio-Rad (Hercules, CA) (e.g., using the Kunkel method described above), Boehringer Mannheim Corp. (Ridgefield, Connecticut), Clonetech Laboratories of BD Bidsciences (Palo Alto, CA), DNA Technologies (Gaithersburg, MD), Epicentre Technologies (Madison, Wl) (e.g., the 5 prime 3 prime kit); Genpak Inc. (Stony Brook, New York), Lemargo Inc (Toronto, CANADA), Invitrogen Life Technologies (Carlsbad, California), New England Biolabs (Beverly, MA), Pharmacia Biotech (Peapack, New Jersey), Prornega Corp. (Madison, WI), QBiogene (Carlsbad, CA), and Stratagene (La Jolla, CA) (e.g., QuickChange™ site-directed mutagenesis kit and Chameleon™ double-stranded, site-directed mutagenesis kit).<br>
Determining Sequence Relationships<br>
[0095]	The nucleic acid and amino acid sequences of the invention include, e.g.,<br>
those provided in SEQ ID N0:l to SEQ ID NO: 11 and subsequences thereof, as well as similar sequences. Similar sequences are objectively determined by any number of methods, e.g., percent identity, hybridization, immunologically, and the like. A variety of methods for determining relationships between two or more sequences (e.g., identity, similarity and/or homology) are available and well known in the art. The methods include manual alignment, computer assisted sequence alignment and combinations thereof, for example. A number of algorithms (which are generally computer implemented) for performing sequence alignment are widely available or can be produced by one of skill. These methods include, e.g., the local homology algorithm of Smith and Waterman (1981) Adv. Appl. Math. 2:482; the homology alignment algorithm of Needleman and Wunsch (1970) J. Mol. Biol. 48:443; the search for similarity method of Pearson and Lipman (1988) Proc. Natl. Acad. Sci. (USA) 85:2444; and/or by computerized implementations of these<br>
algorithms (e.g., GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package Release 7.0, Genetics Computer Group, 575 Science Dr., Madison, WI).<br>
[0096]	For example, software for performing sequence identity (and sequence<br>
similarity) analysis using the BLAST algorithm is described in Altschul et al. (1990) J. Mol. Biol. 215:403-410. This software is publicly available, e.g., through the National Center for Biotechnology Information on the world wide web at ncbi.nlm.nih.gov. This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold. These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are then extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Cumulative scores are calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always &gt; 0) and N (penalty score for mismatching residues; always 
[0097]	Additionally, the BLAST algorithm performs a statistical analysis of the<br>
similarity between two sequences (see, e.g., Karlin &amp; Altschul (1993) Proc. Nat'l. Acad. Sci. USA 90:5873-5787). One measure of similarity provided by the BLAST algorithm is the smallest sum probability (p(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar to a reference sequence (and, therefore, in this context, homologous) if the smallest sum probability in a comparison of the test nucleic<br>
acid to the reference nucleic acid is less than about 0.1, or less than about 0.01, and or even less than about 0.001.<br>
[0098]	Another example of a useful sequence alignment algorithm is PILEUP.<br>
PILEUP creates a multiple sequence alignment from a group of related sequences using progressive, pairwise alignments. It can also plot a tree showing the clustering relationships used to create the alignment. PILEUP uses a simplification of the progressive alignment method of Feng &amp; Doolittle (1987) J. Mol. Evol. 35:351-360. The method used is similar to the method described by Higgins &amp; Sharp (1989) CABIOS5:151-153. The program can align, e.g., up to 300 sequences of a maximum length of 5,000 letters. The multiple alignment procedure begins with the pairwise alignment of the two most similar sequences, producing a cluster of two aligned sequences. This cluster can then be aligned to the next most related sequence or cluster of aligned sequences. Two clusters of sequences can be aligned by a simple extension of the pairwise alignment, of two individual sequences. The final alignment is achieved by a series of progressive, pairwise alignments. The program can also be used to plot a dendogram or tree representation of clustering relationships. The program is run by designating specific sequences and their amino acid or nucleotide coordinates for regions of sequence comparison.<br>
[0099]	An additional example of an algorithm that is suitable for multiple DNA, or<br>
amino acid, sequence alignments is the CLUSTALW program (Thompson, J. D. et al. (1994) Nucl. Acids. Res. 22: 4673-4680). CLUSTALW performs multiple pairwise comparisons between groups of sequences and assembles them into a multiple alignment based on hornology. Gap open and Gap extension penalties can be, e.g., 10 and 0.05 respectively. For amino acid alignments, the BLOSUM algorithm can be used as a protein weight matrix. See, e.g., Henikoff and Henikoff (1992) Proc. Natl. Acad. Sci. USA 89: 10915-10919.<br>
Nucleic Acid Hybridization<br>
[0100]	Similarity between ACC synthase nucleic acids of the invention can also be<br>
evaluated by "hybridization" between single stranded (or single stranded regions of) nucleic acids with complementary or partially complementary polynucleotide sequences.<br>
[0101]	Hybridization is a measure of the physical association between nucleic acids,<br>
typically, in solution, or with one of the nucleic acid strands immobilized on a solid support,<br>
e.g., a membrane, a bead, a chip, a filter, etc. Nucleic acid hybridization occurs based on a variety of well characterized physico-chemical forces, such as hydrogen bonding, solvent exclusion, base stacking, and the like. Numerous protocols for nucleic acid hybridization are well known in the art. An extensive guide to the hybridization of nucleic acids is found in Tijssen (1993) Laboratory Techniques in Biochemistry and Molecular Biology-Hybridization with Nucleic Acid Probes, part I, chapter 2, "Overview of principles of hybridization and the strategy of nucleic acid probe assays," (Elsevier, New York), as well as in Ausubel et al. Current Protocols in Molecular Biology (supplemented through 2004) John Wiley &amp; Sons, New York ("Ausubel"); Sambrook et al. Molecular Cloning - A Laboratory Manual (2nd Ed.), Vol. 1-3, Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989 ("Sambrook"), and Berger and Kimniel Guide to Molecular Cloning Techniques, Methods in Enzymologv volume 152 Academic Press, Inc., San Diego, CA ("Berger"). Hames and Higgins (1995) Gene Probes 1. IRL Press at Oxford University Press, Oxford, England (Hames and Higgins 1) and Hames and Higgins (1995) Gene Probes 2. IRL Press at Oxford University Press, Oxford, England (Hames and Higgins 2) provide details on the synthesis, labeling, detection and quantification of DNA and RNA, including oligonucleotides.<br>
[0102]	Conditions suitable for obtaining hybridization, including differential<br>
hybridization, are selected according to the theoretical melting temperature (Tm) between complementary and partially complementary nucleic acids. Under a given set of conditions, e.g., solvent composition, ionic strength, etc., the Tm is the temperature at which the duplex between the hybridizing nucleic acid strands is 50% denatured. That is, the Tm corresponds to the temperature corresponding to the midpoint in transition from helix to random coii; it depends on the length of the polynucleotides, nucleotide composition, and ionic strength, for long stretches of nucleotides.<br>
[0103]	After hybridization, unhybridized nucleic acids can be removed by a series<br>
of washes, the stringency of which can be adjusted depending upon the desired results. Low stringency washing conditions (e.g., using higher salt and lower temperature) increase sensitivity, but can product nonspecific hybridization signals and high background signals. Higher stringency conditions (e.g., using lower salt and higher temperature that is closer to the Tm) lower the background signal, typically with primarily the specific signal remaining.<br>
See, also, Rapley, R. and Walker, J.M. eds., Molecular Biomethods Handbook (Humana Press, Inc. 1998).<br>
[0104]	"Stringent hybridization wash conditions" or "stringent conditions" in the<br>
context of nucleic acid hybridization experiments, such as Southern and northern hybridizations, are sequence dependent, and are different under different environmental parameters. An extensive guide to the hybridization of nucleic acids is found in Tijssen (1993), supra, and in Hames and Higgins 1 and Hames and Higgins 2, supra.<br>
[0105]	An example of stringent hybridization conditions for hybridization of<br>
complementary nucleic acids which have more than 100 complementary residues on a filter in a Southern or northern blot is 2x SSC, 50% fprmamide at 42°C, with the hybridization being carried out overnight (e.g., for approximately 20 hours). An example of stringent wash conditions is a 0.2x SSC wash at 65°C for 15 minutes (see Sambrook, supra for a description of SSC buffer). Often, the wash determining the stringency is preceded by a low stringency wash to remove signal due to residual unhybridized probe. An example low stringency wash is 2x SSC at room temperature (e.g., 20°C for 15 minutes).<br>
[0106]	In general, a signal to noise ratio of at least 2.5x-5x (and typically higher)<br>
than that observed for an unrelated probe in the particular hybridization "assay indicates detection of a specific hybridization. Detection of at least stringent hybridization between two sequences in the context of the invention indicates relatively strong structural similarity to, e.g., the nucleic acids of ACC synthases provided in the sequence listings herein.<br>
[0107]	For purposes of the invention, generally, "highly stringent" hybridization and<br>
wash conditions are'selected to be about 5° C or less lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength and pH (as noted below, highly stringent conditions can also be referred to in comparative terms). Target sequences that are closely related or identical to the nucleotide sequence of interest (e.g., "probe") can be identified under stringent or highly stringent conditions. Lower stringency conditions are appropriate for sequences that are less complementary.<br>
[0108]	For example, in determining stringent or highly stringent hybridization (or<br>
even more stringent hybridization) and wash conditions, the stringency of the hybridization and wash conditions is gradually increased (e.g., by increasing temperature, decreasing salt concentration, increasing detergent concentration, and/or increasing the concentration of<br>
organic solvents, such as formamide, in the hybridization or wash), until a selected set of criteria are met. For example, the stringency of the hybridization and wash conditions is gradually increased until a probe comprising one or more polynucleotide sequences of the invention, e.g., selected from SEQ ID NO:1 to SEQ ID NO:6 and SEQ ID NO: 10, or a subsequence thereof, and/or complementary polynucleotide sequences thereof, binds to a perfectly matched corhplementary target (again, a nucleic acid comprising one or more nucleic acid sequences or subsequences selected from SEQ ID NO:1 to SEQ ID NO:6 and SEQ ID NO: 10, and complementary polynucleotide sequences thereof), with a signal to noise ratio that is at least 2.5x, and optionally 5x, or lOx, or lOOx or more, as high as that observed for hybridization of the probe to an unmatched target, as desired.<br>
[0109]	Using subsequences derived from the nucleic acids encoding the ACC<br>
synthase polypeptides of the invention, novel target nucleic acids can be obtained; such target nucleic acids are also a feature of the invention. For example, such target nucleic acids include sequences that hybridize under stringent conditions to an oligonucleotide probe that corresponds to a unique subsequence of any of the polynucleotides of the invention, e.g., SEQ ID NOs: 1-6, 10, or a complementary sequence thereof; the probe optionally encodes a unique subsequence in any of the polypeptides of the invention, e.g., SEQ ID NOs: 7-9 and 11.<br>
[0110]	For example, hybridization conditions are chosen under which a target<br>
oligonucleotide that is perfectly complementary to the oligonucleotide probe hybridizes to the probe with at least about a 5-10x higher signal to noise ratio than for hybridization of the target oligonucleotide to a negative control non-complimentary nucleic acid.<br>
[0111]	Higher ratios of signal to noise can be achieved by increasing the stringency<br>
of the hybridization conditions such that ratios of about 15x, 20x, 30x, 50x or more are obtained. The particular signal will depend on the label used in the relevant assay, e.g., a fluorescent label, a colorimetric label, a radioactive label, or the like.<br>
Vectors. Promoters and Expression Systems<br>
[0112]	Nucleic acids of the invention can be in any of a variety of forms, e.g.,<br>
expression cassettes, vectors, plasmids, or linear nucleic acid sequences. For example, vectors, plasmids, cosmids, bacterial artificial chromosomes (BACs), YACs (yeast artificial chromosomes), phage, viruses and nucleic acid segments can comprise an ACC synthase<br>
nucleic acid sequence or subsequence thereof which one desires to introduce into cells. These nucleic acid constructs can further include promoters, enhancers, polylinkers, regulatory genes, etc.<br>
[0113]	Thus, the present invention also relates, e.g., to vectors comprising the<br>
polynucleotides of the present invention, host cells that incorporate the vectors of the invention, and the production of polypeptides of the invention by recombinant techniques.<br>
[0114]	In accordance with this aspect of the invention, the vector may be, for<br>
example, a plasmid vector, a single or double-stranded phage vector, or a single or double-stranded RNA or DNA viral vector. Such vectors may be introduced into cells as polynucleotides, preferably DNA, by well known techniques for introducing DNA and RNA into cells. The vectors, in the case of phage and viral vectors, also may be and preferably are introduced into cells as packaged or encapsidated virus by well known techniques for infection and transduction. Viral vectors may be replication competent or replication defective. In the latter case, viral propagation generally will occur only in complementing host cells.<br>
[0115]	Preferred among vectors, in certain respects, are those for expression of<br>
polynucleotides and polypeptides of the present invention. Generally, such vectors comprise cis-acting control regions effective for expression in a host, operably linked to the polynucleotide to be expressed. Appropriate trans-acting factors are supplied by the host, supplied by a complementing vector or supplied by the vector itself upon introduction into the host.<br>
[0116]	In certain preferred embodiments in this regard, the vectors provide for<br>
preferred expression. Such preferred expression may be inducible expression, temporally limited expression, or expression restricted to predominantly certain types of cells, or any combination of the above. Particularly preferred among inducible vectors are vectors that can be induced for expression by environmental factors that are easy to manipulate, such as temperature and nutrient additives. A variety of vectors suitable to this aspect of the invention, including constitutive and inducible expression vectors for use in prokaryotic and eukaryotic hosts, are well known and employed routinely by those of skill in the art. Such vectors include, among others, chromosomal, episomal and virus-derived vectors, e.g., vectors derived from bacterial plasmids, from bacteriophage, from transposons, from yeast<br>
episomes, from insertion elements, from yeast chromosomal elements, from viruses such as baculoviruses, papova viruses, such as SV40, vaccinia viruses, adenoviruses, fowl pox viruses, pseudorabies viruses and retroviruses, and vectors derived from combinations thereof, such as those derived from plasmid and bacteriophage genetic elements, such as cosmids and phagemids and binaries used for Agrobacterium-medialed transformations. All "may be used for expression in accordance with this aspect of the present invention.<br>
[0117]	Vectors that are functional in plants can be binary plasmids derived from<br>
Agrobcicterium. Such vectors are capable of transforming plant cells. These vectors contain left and right border sequences that are required for integration into the host (plant) chromosome. At minimum, between these border sequences is the gene (or other polynucleotide sequence of the present invention) to be expressed, typically under control of regulatory elements. In one embodiment, a selectable marker and a reporter gene are also included. For ease of obtaining sufficient quantities of vector, a bacterial origin that allows replication in E. coll can be used.<br>
[0118]	The following vectors, which are commercially available, are provided by<br>
way of example. Among vectors preferred for use in bacteria are pQE70, pQE60 and pQE-9, available from Qiagen; pBS vectors, Phagescript vectors, Bluescript vectors. pNHSA, pNH16a, pNHlSA, pNH46A, available from Stratagene^and ptrc99a, pKK223-3r pKK233-3, pDR540, pRIT5 available from Pharmacia. Among preferred eukaryotic vectors are pWLNEO, pSV2CAT, pOG44, pXTl and pSG available from Stratagene; and pSVK3, pBPV, pMSG and pSVL available from Pharmacia. Useful plant binary vectors include BIN19 and its derivatives available from Clontech. These vectors are listed solely by way of illustration of the many commercially available and well-known vectors that are available to those of skill in the art for use in accordance with this aspect of the present invention. It will be appreciated that any other plasmid or vector suitable for, for example, introduction, maintenance, propagation or expression of a polynucleotide or polypeptide of the invention in a host may be used in this aspect of the invention, several of which are disclosed in more detail below.<br>
[0119]	In general, expression constructs will contain sites for transcription initiation<br>
and termination, and, in the transcribed region, a ribosome-binding site for translation when the construct encodes a polypeptide. The coding portion of the mature transcripts expressed<br>
by the constructs will include a translation-initiating AUG at the beginning and a termination codon appropriately positioned at the end of the polypeptide to be translated.<br>
[0120]	In addition, the constructs may contain control regions that regulate as well<br>
as engender expression. Generally, in accordance with many commonly practiced procedures, such regions will operate by controlling transcription, such as transcription factors, represser binding sites and termination signals, among others. For secretion of a translated protein into the lumen of the endoplasmic reticulum, into the periplasmic space or into the extracellular environment, appropriate secretion signals may be incorporated into the expressed polypeptide. These signals may be endogenous to the polypeptide or they may be heterologous signals.<br>
[0121]	Transcription of the DNA (e.g., encoding the polypeptides) of the present<br>
invention by higher eukaryotes may be increased by inserting an enhancer sequence into the vector. Enhancers are cis-acting elements of DNA, usually about from 10 to 300 bp that act to increase transcriptional activity of a promoter in a given host cell-type. Examples of enhancers include the SV40 enhancer, which is located on the late side of the replication origin at bp 100 to 270, the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers. Additional enhancers useful in the invention to increase transcription of the introduced DNA segment, include, inter alia, viral enhancers like those within the 35S promoter, as shown by Odell et al, Plant Mol. Biol. 10:263-72 (1988), and an enhancer from an opine gene as described by Fromm et al., Plant Cell 1:977 (1989). The enhancer may affect the tissue-specificity and/or temporal specificity of expression of sequences included in the vector.<br>
[0122]	Termination regions also facilitate effective expression by ending<br>
transcription at appropriate points. Useful terminators for practicing this invention include, but are not limited to, pinll (see An et al., Plant Cell 1(1): 115-122 (1989)), glbl (see Genbank Accession #L22345), gz (see gzw64a terminator, Genbank Accession #878780), and the nos terminator from Agrobacterium. The termination region can be native with the promoter nucleotide sequence, can be native with the DNA sequence of interest, or can be derived from another source. For example, other convenient termination regions are available from the Ti-plasmid of A. tumefaciens, such as the octopine synthase and nopaline synthase termination regions. See also: Guerineau etal. (1991) Mol. Gen. Genet. 262:141-144; Proudfoot (1991) Cell 64:671-674; Sanfacon et al (1991) Genes Dev. 5:141-149;<br><br>
Mogen et al (1990) Plant Cell 2:1261-1272; Munroe et al. (1990) Gene 91:151-158; Ballas . et al 1989) Nucleic Acids Res. 17:7891-7903; and Joshi et al. (1987) Nucleic Acid Res. 15:9627-9639.<br>
[0123]	Among known eukaryotic promoters suitable for generalized expression are the<br>
CMV immediate early promoter, the HSV thymidine kinase promoter, the early and late SV40 promoters, the promoters of retroviral LTRs, such as those of the Rous sarcoma virus ("RSV"), metallothionein promoters, such as the mouse metallothionein-I promoter and various plant promoters, such as globulin-1. When available, the native promoters of the ACC synthase genes may be used. Representatives of prokaryotic promoters include the phage lambda PL promoter, the E. coli lac, tip and tac promoters to name just a few of the well-known promoters.<br>
[0124]	Isolated or recombinant plants, or plant cells, incorporating the ACC<br>
synthase nucleic acids are a feature of the invention. The transformation of plant cells and protoplasts can be earned out in essentially any of the various ways known to those skilled in the art of plant molecular biology, including, but not limited to, the methods described herein. See, in general, Methods in Enzymology, Vol. 153 (Recombinant DNA Part D) Wu and Grossman (eds.) 1987, Academic Press, incorporated herein by reference. As used herein, the term "transformation" means alteration of the1 genotype of a host plant by the introduction of a nucleic acid sequence, e.g., a "heterologous", "exogenous" or "foreign" nucleic acid sequence. The heterologous nucleic acid sequence need not necessarily originate from a different source but it will, at some point, have been external to the cell into which is introduced.<br>
[0125]	In addition to Berger, Ausubel and Sambrook, useful general references for<br>
plant cell cloning, culture and regeneration include Jones (ed) (1995) Plant Gene Transfer and Expression Protocols-- Methods in Molecular Biology. Volume 49 Humana Press Towata NJ; Payne et al. (1992) Plant Cell and Tissue Culture in Liquid Systems John Wiley &amp; Sons, Inc. New York, NY (Payne); and Gamborg and Phillips (eds) (1995) Plant Cell. Tissue_and Organ Culture; Fundamental Methods Springer Lab Manual, Springer-Verlag (Berlin Heidelberg New York) (Gamborg). A variety of cell culture media are described in Atlas and Parks (eds) The Handbook of Microbiological Media (1993) CRC Press, Boca Raton, FL (Atlas). Additional information for plant cell culture is found in available commercial literature such as the Life Science Research Cell Culture Catalogue (1998) from<br><br>
§gma- Aldrich, Inc (St Louis, MO) (Sigma-LSRCCC) and, e.g., the Plant culture t Catalogue and supplement (1997) also from Sigma-Aldrich, Inc (St Louis, MO) (Sigma-<br>
PCCS). Additional details regarding plant cell culture are found in Croy, (ed.) (1993) Plant Molecular Biology Bios Scientific Publishers, Oxford, U.K. See also the section herein entitled "Plant transformations."<br>
[0126]	In an embodiment of this invention, recombinant vectors including one or<br>
more of the ACC synthase nucleic acids or a subsequence thereof, e.g., selected from SEQ ID NO: 1 to SEQ ID NO:6, or SEQ ID NO: 10, suitable for the transformation of plant cells are prepared. In another embodiment, a nucleic acid sequence encoding for the desired ACC synthase RNA or protein or subsequence thereof, e.g., selected from among SEQ ID NO:7 to SEQ ID NO:9, or SEQ ID NO: 11, is conveniently used to construct a recombinant expression cassette which can be introduced into the desired plant. In the context of the invention, an expression cassette will typically comprise a selected ACC synthase nucleic acid sequence or subsequence in an RNA configuration (e.g., antisense., sense, RNA silencing or interference configuration, and/or the like) operably linked to a promoter sequence and other transcriptional and translational initiation regulatory sequences which are sufficient to direct the transcription of the ACC synthase RNA configuration sequence in the intended tissues (e.g., entire plant, leaves, anthers, roots, etc.) of the transformed plant.<br>
[0127]	In general, the particular promoter used in the expression cassette in plants<br>
depends on the intended application. Any of a number of promoters can be suitable. For example, the nucleic acids can be combined with constitutive, inducible, tissue-specific (tissue-preferred), or other promoters for expression in plants. For example, a strongly or weakly constitutive plant promoter that directs expression of an ACC synthase RNA configuration sequence in all tissues of a plant can be favorably employed. Such promoters are active under most environmental conditions and states of development or cell differentiation. Examples of constitutive promoters include the 1'- or 2'- promoter of Agrobacteriwn tumefaciens (see, e.g., O'Grady (1995) Plant Mol. Biol. 29:99-108). Other plant promoters include the ribulose-l,3-bisphosphate carboxylase small subunit promoter, the phaseolin promoter, alcohol dehydrogenase (Adh) gene promoters (see, e.g., Millar (1996) Plant Mol. Biol. 31:897-904), sucrose synthase promoters, a-tubulin promoters, actin promoters, such as the Arabidopsis actin gene promoter (see, e.g., Huang (1997) Plant<br><br>
WO 2005/016504<br>
Mol. Biol. 1997 33:125-139), cab, PEPCase, R gene complex, ACT11 from Arabidopsis "(Huang et al. Plant Mol. Biol. 33:125-139 (1996)), Cat3 from Arabidopsis (Zhong et al., Mol. Gen. Genet. 251:196-203 (1996)), the gene encoding stearoyl-acyl carrier protein desaturase from Brassica napus (Solocombe et al. (1994) Plant Phvsiol. 104:1167-1176), GPcl from maize (Martinez et al. (1989) J. Mol. Biol 208:551-565), Gpc2 from maize (Manjunath et al. (1997), Plant Mol. Biol. 33:97-112), and other transcription initiation regions from various plant genes known to those of skill. See also Holtorf (1995) "Comparison of different constitutive and indudble promoters for the overexpression of transgenes in Arabidopsis thaliana" Plant Mol. Biol. 29:637-646. The promoter sequence from the E8 gene (see, Deikman and Fischer (1988) EMBOJ 7:3315) and other genes can also be used, along with promoters specific for-monocotyledonous species (e.g., McElroy D., et al. (1994.) Foreign gene expression in transgenic cereals. Trends Biotech.. 12:62-68). Other constitutive promoters include, for example, the core promoter of the Rsyn7 promoter and other constitutive promoters disclosed in WO 99/43838 and U.S. Patent No. 6,072,050; the core CaMV 35S promoter (Odell et al. (1985) Nature 313:810-812); rice actin (McElroy et al. (1990) Plant Cell 2:163-171); ubiquitin (Christensen et al (1989) Plant Mol. Biol. 12:619-632 and Christensen et al (1992) Plant Mol. Biol. 18:675-689V. pEMU (Last et al (1991) Theor. Appl. Genet. 81:581-588); MAS (Velten et al (1984) EMBOJ. 3:2723-2730); ALS promoter (U.S. Patent No.'5,659,026), and the like. Yet, other constitutive promoters include, for example, U.S. Patent Nos. 5,608,149; 5,608,144; 5,604,121; 5,569,597; 5,466,785; 5,399,680; 5,268,463; 5,608,142; and 6,177,611.<br>
[0128]	In addition to the promoters noted herein, promoters of bacterial origin<br>
which operate in plants can be used in the inversion. They 'nclude, e.g., the ^top'ie synthase promoter, the nopaline synthase promoter and other promoters derived from Ti plasmids. See, Herrera-Estrella et al. (1983) Nature 303:209. Viral promoters can also be used. Examples of viral promoters include the 35S and 19S RNA promoters of cauliflower mosaic virus (CaMV). See, Odell et al., (1985) Nature 313:810; and, Dagless (1997) Arch. Virol. 142:183-191. Other examples of constitutive promoters from viruses which infect plants include the promoter of the tobacco mosaic virus; cauliflower mosaic virus (CaMV) 19S and 35S promoters or the promoter of Figwort mosaic virus, e.g., the figwort mosaic virus 35S promoter (see, e.g., Maiti (1997) Transgenic Res. 6:143-156), etc. Alternatively, novel promoters with useful characteristics can be identified from any viral, bacterial, or<br>
plant source by methods, including sequence analysis, enhancer or promoter trapping, and<br>
J the like, known in the art.<br>
[0129]	Tissue-preferred (tissue-specific) promoters and enhancers can be utilized to<br>
target enhanced gene expression within a particular plant tissue. Tissue-preferred (tissue-specific) promoters include, e.g., those described in Yamamoto et al. (1997) Plant J. ~12(2):255-265; Kawamata et al (1997) Plant Cell Phvsiol. 3S(7):792-S03; Hansen et al (1997) Mol. Gen Genet. 254(3):337-343; Russell et al. (1997) Transgenic Res. 6(2): 157-168; Rinehart etal (1996) Plant Phvsiol. 112(3):1331-1341; Van Camp et al (1996) Plant Phvsiol. 112(2):525-535; Canevascini etal. (1996) Plant Phvsiol. 112(21:513-524: Yamamoto et al. (1994) Plant Cell Phvsiol. 35(5):773-778; Lam (1994) Results Probl. Cell Differ. 20:181-196; Orozcoetal. (19931 Plant Mol Biol. 23(6): 1129-1138: Matsuoka 
(1993)	Proc Natl. Acad. Sci. USA 90(20):9586-9590; and Guevara-Garcia et al. (1993)<br>
Plant J. 4(3):495-505. Such promoters can be modified, if necessary, for weak expression.<br>
[0130]	In certain embodiments, leaf specific promoters can be used, e.g., pyruvate,<br>
orthophosphate dikinase (PPDK) promoter from C4 plant (maize), cab-mi Ca+2 promoter from maize, the Arabidopsis thaliana myb-related gene promoter (AtmybS), the ribulose biphosphate carboxylase (RBCS) promoters-(e.g., the tomato RBCS1, RBCS2 and RBCS3A genes, which are expressed in leaves and light-grown seedlings, while RBCS 1 and RBCS2 are expressed in developing tomato fruits, and/or a ribulose bisphosphate carboxylase promoter which is expressed almost exclusively in mesophyll cells in leaf blades and leaf sheaths at high levels, etc.), and the like. See, e.g., Matsuoka et a., (1993) Tissue-specific light-regulated expression directed by the promoter of a C4 gene, maize pyruvate, orthophosphate dikinase, in a C3 plant, rice, PNAS USA 90(20):9586-9Q; (2000) Plant Cell Phvsiol. 4l(n:42-48: (200 H Plant Mol. Biol. 45(1): 1-15: Shiina,T. etal., (1997) Identification of Promoter Elements involved in the cytosolic Ca+2 mediated photoregulation of maize cab-mi expression, Plant Phvsiol. 115:477-483; Casal (1998) Plant Phvsiol. 116:1533-1538: Li (1996) FEES Lett. 379:117-121; Busk (1997) Plant J. 11:1285-1295; and, Meier (1997) FEES Lett. 415:91-95; and, Matsuoka (1994) Plant J. 6:311-319. Other leaf-specific promoters include, for example, Yamamoto et al. (1997) Plant J. 12(2):255-265: Kwon et al. (1994) Plant Phvsiol. 105:357-67; Yamamoto et al.<br>
(1994)	Plant Cell'physiol. 35(5):773-778; Gotor et al. (1993) Plant J. 3:509-18; Orozco et<br>
al. (1993) Plant Mol.Biol. 23(6): 1129-1138; and Matsuoka et al (1993) Proc. Natl. Acad.<br>
. USA 90(20):9586-9590.<br>
[0131]	In certain embodiments, senescence specific promoters can be used (e.g., a<br>
tomato promoter active during fruit ripening, senescence and abscission of leaves, a maize promoter of gene encoding a cysteine protease, and the like). See, e.g., Blume (1997) Plant 112:731-746; Griffiths et al., (1997) Sequencing, expression pattern andRFLP mapping of a senescence-enhanced cDNAfrom Zea Mays with high homology to oryzain gamma and aleurain, Plant Mol. Biol. 34(5):815-21; Zea mays partial seel gene for cysteine protease, promoter region and 5' coding region, Genbank AJ494982; Kleber-Janke, T. and Krupinska, K. (1997) Isolation ofcDNA clones for genes showing enhanced expression in barley leaves during dark-induced senescence as well as during senescence under field conditions, Planta 203(3): 332-40; and, Lee, RHet al., (2001) Leaf senescence in rice plants: cloning and characterization of senescence up-regulated genes, J. Exp. Bot. 52(358):1117-21.<br>
[0132]	In other embodiments, anther-specific promoters can be used. Such<br>
promoters are known in the art or can be discovered by known techniques; see, e.g., Bhalla and Singh (1999) Molecular control of male fertility in Brassica Proc. 10th Annual Rapeseed Congress. Canberra, Australia; van Tunen et al. (1990) Pollen- and anther-specific chi promoters from petunia: tandem promoter regulation of the chiA gene. Plant Cell 2:393-40; Jeon et al. (1999) Isolation and characterization of an anther-specific gene, RA8,from rice (Oryza sativa L). Plant Molecular Biology 39:35-44; and Twell et al (1993) Activation and developmental regulation of an Arabidopsis anther-specific promoter in microspores and pollen ofNicotiana tabacum. Sex. Plant Reprod. 6:217-224.<br>
[0133]	Root-preferred promoters are known and can be selected from the many<br>
available from the literature or isolated de novo from various compatible species. See, for example, Hire et al. (1992) Plant Mol. Biol. 20(2):207-218 (soybean root-specific glutamine synthetase gene); Keller and Baumgartner (1991) Plant Cell 3(10):1051-1061 (root-specific control element in the GRP 1.8 gene of French bean); Sanger et al. (1990) Plant Mol. Biol. 14(3):433-443 (root-specific promoter of the mannopine synthase (MAS) gene of Agrobacterium tumefaciens}; and Miao et al. (1991) Plant Cell 3(1): 11-22 (full-length cDNA clone encoding cytosolic glutamine synthetase (GS), which is expressed in roots and root nodules of soybean). See also Bogusz et al. (1990) Plant Cell 2(7):633-641, where two<br>
root-specific promoters isolated from hemoglobin genes from the nitrogen-fixing nonlegume Parasponia andersonii and the related non-nitrogen-fixing nonlegume Trema tomentosa are described. The promoters of these genes were linked to a {3-glucuronidase reporter gene and introduced into both the nonlegume Nicotiana tabacum and the legume Lotus corniculatus, and in both instances root-specific promoter activity was preserved. Leach and Aoyagi (1991) describe their analysis of the promoters of the highly expressed rolC and rolD root-inducing genes of Agrobacterium rhizogenes (see Plant Science (Limerick) 79(l):69-76). They concluded that enhancer and tissue-preferred DNA determinants are dissociated in those promoters. Teeri et al. (1989) used gene fusion to lacZ to show that the Agrobacterium T-DNA gene encoding octopine synthase is especially active in the epidermis of the root tip and that the TR21 gene is root specific in the intact<br>
«<br>
plant and stimulated by wounding in leaf tissue (see, e.g., EMBO J. 8(2):343-350). The TR1' gene, fused to nptll (neomycin phosphotransferase II) showed similar characteristics. Additional root-preferred promoters include the VfENOD-GRP3 gene promoter (Kuster et al. (1995) Plant Mol. Biol. 29(4):759-772); and rolB promoter (Capana et al. (1994) Plant Mol. Biol. 25(4):681-691. See also, U.S. Patent Nos. 5,837,876; 5,750,386; 5,633,363; 5,459,252; 5,401,836; 5,110,732; and 5,023,179.<br>
[0134]	"Seed-preferred" promoters include both "seed-specific" promoters (those<br>
promoters active during seed development such as promoters of seed storage proteins) as well as "seed-germinating" promoters (those promoters active during seed germination). See, e.g., Thompson et al. (1989) BioEssays 10:108, herein incorporated by reference. Such seed-preferred promoters include, but are not limited to, Ciml (cytokinin-induced message); cZ19Bl (maize 19 kDa zein); milps (myo-inositol-1 -phosphate synthase); mZE40-2, also known as Zm-40 (U.S. Patent 6,403,862); nuclc (U.S. Patent 6,407,315); and celA (cellulose synthase) (see WO 00/11177). Gama-zein is an endosperm-specific promoter. Glob-1 is an embryo-specific promoter. For dicots, seed-specific promoters include, but are not limited to, bean p-phaseolin, napin, p-conglycinin, soybean lectin, cruciferin, and the like. For monocots, seed-specific promoters include, but are not limited to, a maize 15 kDa zein promoter, a 22 kDa zein promoter, a 27 kDa zein promoter, a g-zein promoter, a 27 kD y-zein promoter (such as gzw64A promoter, see Genbank Accession #S78780), a waxy promoter, a shrunken 1 promoter, a shrunken 2 promoter, a globulin 1 promoter (see Genbank Accession # L22344), an Itp2 promoter (Kalla, et al., Plant Journal 6:849-860<br>
(1994); U.S. Patent 5,525,716), ciml promoter (see U.S. Patent 6,225,529) maize endl and end2 promoters (See U.S. patent 6,528,704 and application 10/310,191, filed December 4, 2002); nucl promoter (U.S. patent 6,407,315); Zm40 promoter (U.S. patent 6,403,862); eepl and eep2; led (U.S. patent application 09/718,754); thioredoxinH promoter (U.S. provisional patent application 60/514,123); mliplS promoter (U.S. patent 6,479,734); PCNA2 promoter; and the shrunken-2 promoter. (Shaw et al., Plant Phys 98:1214-1216, 1992; Zhong Chen et al., PNAS USA 100:3525-3530,2003) However, other promoters useful in the practice of the invention are known to those of skill in the art such as nucellain promoter (See C. Linnestad, et al., Nucellain, A Barley Homolog of the Dicot Vacuolar-Processing Proteasem Is Localized in Nucellar Cell Walls, Plant Physiol. 118:1169-80 (1998), knl promoter (See S. Hake and N. Ori,- The Role ofhwttedl in Meristem Functions, B8: INTERACTIONS AND INTERSECTIONS IN PLANT PATHWAYS, COEUR D'ALENE, IDAHO, KEYSTONE SYMPOSIA, February 8-14, 1999, at 27.), and F3.7 promoter (Baszczynski et al., Maydica 42:189-201 (1997)), etc. In certain embodiments, spatially acting promoters such as glbl, an embryo-preferred promoter; or gamma zein, an endosperm-preferred promoter; or a promoter active in the embryo-surrounding region (see U.S. Patent Application 10/786,679, filed February 25, 2004), or BETL1 (See G. Hueros, et al, Plant Physiology 121:1143-1152 (1999) and Plant Cell 7:747-57 (June 1995)), are useful, including promoters preferentially active in female reproductive tissues, and those active in meristematic tissues, particularly in meristematic female reproductive tissues. See also, WO 00/12733, where seed-preferred promoters from endl and end.2 genes are disclosed.<br>
[0135]	A tissue-specific promoter can drive expression of operably linked sequences<br>
i.p tissues other than the target tissue. Thus, as used herein, a tiss'.:?-specific promoter is one that drives expression preferentially in the target tissue, but can also lead to some expression in other tissues as well.<br>
[0136]	The use of temporally-acting promoters is also contemplated by this<br>
invention. For example, promoters that act from 0-25 days after pollination (DAP), 4-21, 4-12, or 8-12 DAP can be selected, e.g., promoters such as ciml and Hp2.   Promoters that act from -14 to 0 days after pollination can also be used, such as SAG12 (See WO 96/29858, Richard M. Amasino, published 3 Oct. 1996) and ZAG1 or ZAG2 (See R.J. Schmidt, et al., Identification and Molecular Characterization ofZAGl, the Maize Homolog of the Arabidopsis Floral Homeotic GeneAGAMOUS, Plant-Cell 5(7): 729-37 (July 1993)).<br>
useful promoters include maize zag2.1, Zap (also known as ZmMADS; US patent application 10/387,937; WO 03/078590); and the maize tbl promoter (see also Hubbarda et al., Genetics 162:1927-1935, 2002).<br>
[0137]	Where overexpression of an ACC synthase RNA configuration nucleic acid<br>
is detrimental to the plant, one of skill will recognize that weak constitutive promoters can be used for low-levels of expression (or, in certain embodiments, inducible or tissue-specific promoters can be used). In those cases where high levels of expression are not harmful to the plant, a strong promoter, e.g., a t-RNA, or other pol m promoter, or a strong pol n promoter (e.g., the cauliflower mosaic virus promoter, CaMV, 35S promoter), can be used.<br>
[0138]	Where low level expression is desired, weak promoters will be used.<br>
Generally, by "weak promoter" is intended a promoter that drives expression of a coding sequence at a low level. By low level is intended at levels of about 1/1000 transcripts to about 1/100,000 transcripts to about 1/500,000 transcripts. Alternatively, it is recognized that weak promoters also encompass promoters that drive expression in only a few cells and not in others to give a total low level of expression. Where a promoter drives expression at unacceptably high levels, portions of the promoter sequence can be deleted or modified to decrease expression levels. Such weak constitutive promoters include, for example, the core promoter of the Rsyn7 promoter (WO 99/43838 and U.S. Patent No. 6,072,050), the core 35S CaMV promoter, and the like.<br>
[0139]	In certain embodiments of the invention, an inducible promoter can be used.<br>
For example, a plant promoter can be under environmental control. Such promoters are referred to as "inducible" promoters. Examples of environmental conditions that can alter transcription by inducible promoters include pathogen attack, anaerobic conditions, elevated temperature, and the presence of light. For example, the invention incorporates the drought-inducible promoter of maize (Busk (1997) Plant J. 11:1285-1295); the cold, drought, high salt inducible promoter from potato (Kirch (1997) Plant Mol. Biol. 33:897-909). and the like.<br>
[0140]	Pathogen-inducible promoters include those from pathogenesis-related<br>
proteins (PR proteins), which are induced following infection by a pathogen; e.g., PR proteins, SAR proteins, beta-l,3-glucanase, chitinase, etc. See, for example, Redolfi et al.<br>
(1983) Neth.J. Plant Pathol. 89:245-254; Uknes et al (1992) Plant Cell 4:645-656; and Van Loon (1985) Plant Mol. Virol. 4:111-116. See also the application entitled "Inducible Maize Promoters", U.S. Patent Application Serial No. 09/257,583, filed February 25,1999.<br>
[0141]	Of interest are promoters that are expressed locally at or near the site of<br>
pathogen infection. See, for example, Marineau et al. (1987) Plant Mol. Biol. 9:335-342; Matton et al. (1989) Molecular Plant-Microbe Interactions 2:325-331; Somsisch et al. (1986) Proc. Natl. Acad. Sci. 83:2427-2430; Somsisch et al (1988) Mol. Gen. Genet. 2:93-98; and Yang (1996) Proc. Natl. Acad. Sci. 93:14972-14977. See also, Chen et al (1996) Plant J. 10:955-966; Zhang et al (1994) Proc. Natl. Acad. Sci. 91:2507-2511; Warner et al. (1993) Plant J. 3:191-201; Siebertz et al (1989) Plant Cell 1:961-968; U.S. Patent No. 5,750,386 (nematode-inducible); and the references cited therein. Of particular interest is the inducible promoter for the maize PRms gene, whose expression is induced by the pathogen Fusarium moniliforine (see, for example, Cordero et al (1992) Physiol. Mol. Plant Path. 41; 189-200).<br>
[0142]	Additionally, as pathogens find entry into plants through wounds or insect<br>
damage, a wound-inducible promoter may be used in the constructions of the invention. Such wound-inducible promoters include potato proteinase inhibitor (pin II) gene (Ryan (1990) Ann. Rev. Phvtopath. 28:425-449; Duan et al (1996) Nature Biotechnology 14:494-498); wunl and wun2, US Patent No. 5,428,148; winl and win2 (Stanford eted. (1989) Mol. Gen. Genet. 215:200-208); systemin (McGurl et al (1992) Science 225:1570-1573); WIP1 (Rohmeier et al (1993) Plant Mol. Biol. 22:783-792; Eckelkamp et al. (1993) FEES Letters 325:73-76); MPI gene (Corderok et al (1994) Plant J. 6(2): 141-150); and the like.<br>
[0143]	Alternatively, plant promoters which are inducible upon exposure to plant<br>
hormones, such as auxins, are used to express the polynucleotides of the invention. For example, the invention can use the auxin-response elements El promoter subsequence (AuxREs) from the soybean (Glycine max L.) (Liu (1997) Plant Physiol. 115:397-407); the auxin-responsive Arabidopsis GST6 promoter (also responsive to salicylic acid and hydrogen peroxide) (Chen (1996) Plant J. 10: 955-966); the auxin-inducible parC promoter from tobacco; a plant biotin response element (Streit (1997) Mol. Plant Microbe Interact. 10:933-937); and the promoter responsive to the stress hormone abscisic acid (Sheen (1996) Science 274:1900-1902).<br>
0144]	Plant promoters which are inducible upon exposure to chemical reagents<br>
which can be applied to the plant, such as herbicides or antibiotics, are also used to express the polynucleotides of the invention. Depending upon the objective, the promoter can be a chemical-inducible promoter, where application of the chemical induces gene expression, or a chemical-repressible promoter, where application of the chemical represses gene expression. For example, the maize In2-2 promoter, activated by benzenesulfonamide herbicide safeners, can be used (De Veylder (1997) Plant Cell Phvsiol. 38:568-577); application of different herbicide safeners induces distinct gene expression patterns, including expression in the root, hydathodes, and the shoot apical meristem. An ACC synthase coding sequence or RNA configuration can also be under the control of, e.g., tetracycline-inducible and tetracycline-repressible promoters (see, e.g., Gatz et al. (1991) Mol. Gen. Genet. 227:229-237; U.S. Patent Nos. 5,814,618 and 5,789,156; and, Masgrau (1997) Plant J. 11:465-473 (describing transgenic tobacco plants containing the Avena saliva L. (oat) arginine decarboxylase gene with a tetracycline-inducible promoter); or, a salicylic acid-responsive element (Stange (1997) Plant J. 11:1315-1324. Other chemical-inducible promoters are known in the art and include, but are not limited to, the maize GST promoter, which is activated by hydrophobic electrophilic compounds that are used as pre-emergent herbicides, and the tobacco PR-la promoter, which is activated by salicylic acid. Other chemical-regulated promoters of interest include steroid-responsive promoters (see, for example, the glucocorticoid-inducible promoter in Schena et al. (1991) Proc. Natl. Acad. Sci. USA 88:10421-10425 and McNellis et al. (1998) Plant J. 14(2):247-257).<br>
[0145]	Endogenous promoters of genes related to herbicide tolerance and related<br>
phenotypes are also useful for driving expression of ACC synthase RNA configuration nucleic acids, e.g., P450 monooxygenases, glutathione-S-transferases, homoglutathione-S-transferases, glyphosate oxidases and 5-enolpyruvylshikimate-2-phosphate synthases. For example, a plant promoter attached to a polynucleotide of the invention can be useful when one wants to turn on expression in the presence of a particular condition, e.g., drought conditions, short growing conditions, density, etc.<br>
[0146]	Tissue specific promoters can also be used to direct expression of<br>
polynucleotides of the invention, including the ACC synthase RNA configuration nucleic acids, such that a polynucleotide of the invention is expressed only in certain tissues or stages of development, e.g., leaves, anthers, roots, shoots, etc. Tissue specific expression<br>
can be advantageous, for example, when expression of the polynucleotide in certain tissues is desirable while expression in other tissues is undesirable. Tissue specific promoters are transcriptional control elements that are only active in particular cells or tissues at specific times during plant development, such as in vegetative tissues or reproductive tissues. Examples of tissue-specific promoters under developmental control include promoters that initiate transcription only (or primarily only) in certain tissues, such as vegetative tissues, e.g., roots or leaves, or reproductive tissues, such as fruit, ovules, seeds, pollen, pistols, flowers, or any embryonic tissue. Reproductive tissue-specific promoters may be, e.g., anther-specific, ovule-specific, embryo-specific, endosperm-specific, integument-specific, seed and seed coat-specific, pollen-specific, petal-specific, sepal-specific, or some combination thereof.<br>
[0147]	It will be understood that numerous promoters not mentioned are suitable for<br>
use in this aspect of the invention, are well known and readily may be employed by those of skill in the manner illustrated by the discussion and the examples herein. For example, this invention contemplates using, when appropriate, the native ACC synthase promoters to drive the expression of the enzyme (or of ACC synthase polynucleotide sequences or subsequences) in a recombinant environment.<br>
[0148]	In preparing expression vectors of the invention, sequences other than those<br>
associated with the endogenous ACC synthase gene, mRNA or polypeptide sequence, or subsequence thereof, can optionally be used.   For example, other regulatory elements such as introns, leader sequences, polyadenylation regions, signal/localization peptides, etc. can also be included.<br>
•   f--<br>
[0149]	The vector comprising a polynucleotide of the invention can also include a<br>
marker gene which confers a selectable phenotype on plant cells. For example, the marker can encode biocide tolerance, particularly antibiotic tolerance, such as tolerance to kanamycin, G41S, bleomycin, hygromycin, or herbicide tolerance, such as tolerance to chlorosulfuron, or phophinothricin. Reporter genes which are used to monitor gene expression and protein localization via visualizable reaction products (e.g., beta-glucuronidase, beta-galactosidase, and chloramphenicol acetyltransferase) or by direct visualization of the gene product itself (e.g., green fluorescent protein, GFP; Sheen et al. (1995) The Plant Journal 8:777) can be used for, e.g., monitoiing transient gene expression in plant cells.<br>
[0150]	Vectors for propagation and expression generally will include selectable<br>
* iflarkers. Such markers also may be suitable for amplification or the vectors may contain<br>
additional markers for this purpose. In this regard, the expression vectors preferably contain one or more selectable marker genes to provide a phenotypic trait for selection of transformed host cells. Preferred markers include dihydrofolate reductase or neomycin Tesistance for eukaryotie cell culture, and tetracycline or ampicillin resistance genes for culturing E. coli and other prokaryotes. Kanamycin and herbicide resistance genes (PAT and BAR) are generally useful in plant systems.<br>
[0151]	Selectable marker genes, in physical proximity to the introduced DNA<br>
segment, are used to allow transformed cells to be recovered by either positive genetic selection or screening. The selectable marker genes also allow for maintaining selection pressure on a transgenic plant population, to ensure that the introduced DNA segment, and its controlling promoters and enhancers, are retained by the transgenic plant.<br>
[0152]	Many of the commonly used positive selectable marker genes for plant<br>
transformation have been isolated from bacteria and code for enzymes that metabolically detoxify a selective chemical agent which may be an antibiotic or a herbicide. Other positive selection marker genes encode an altered target which is insensitive to the inhibitor.<br>
[0153]	An example of a selection marker gene for'plant transformation is the BAR<br>
or PAT gene, which is used with the selecting agent bialaphos (Spencer et ai, J. Theor. Appl'd Genetics 79:625-631 (1990)). Another useful selection marker gene is the neomycin phosphotransferase II (nptll) gene, isolated from Tn5, which confers resistance to kanamycin when placed under the control of plant regulatory signals (Fraley et al., Prog, Nat'l Acad. Sci. (USA) 80:4803 (1983)). The hygromycin phosphotransferase gene, which confers resistance to the antibiotic hygromycin, is a further example of a useful selectable marker (Vanden Elzen et al., Plant Mol.Biol. 5:299 (1985)). Additional positive selectable marker genes of bacterial origin that confer resistance to antibiotics include gentamicin acetyl transferase, streptomycin phosphotransferase, aminoglycoside-3'-adenyl transferase and the bleomycin resistance determinant (Hayford et al., Plant Physiol. 86:1216 (1988); Jones etal, Mol. Gen. Genet. 210:86 (1987); Svab et al., Plant Mol. Biol. 14:197 (1990); Hille etal., Plant Mol. Biol. 7:171 (1986)).<br>
[0154]	Other positive selectable marker genes for plant transformation are not of<br>
acterial origin. These genes include mouse dihydrofolate reductase, plant 5-enolpyruvylshikimate-3-phosphate synthase and plant acetolactate synthase (Eichholtz et al., Somatic Cell Mol. Genet. 13:67 (1987); Shah et al, Science 233:478 (1986); Charest et al, Plant Cell Rep. 8:643 (1990)). Other examples of suitable selectable marker genes include, but are not limited to: genes encoding resistance to chloramphenicol, Herrera Estrella et al. (1983) EMBO J. 2:987-992; methotrexate, Herrera Estrella et al. (1983) Nature 303:209-213; Meijer et al. (1991) Plant Mol. Biol. 16:807-820; hygromycin, Waldron et al. (1985) Plant Mol Biol. 5:103-108; Zhijian et al. (1995) Plant Science 108:219-227; streptomycin, Jones et al. (1987) Mol. Gen. Genet. 210:86-91; spectinomycin, Bretagne-Sagnard et al. (1996) Transgenic Res: 5:131-131; bleomycin, Hille et al. (1990) Plant Mol. Biol. 7:171-176; sulfonamide, Guerineau et al. (1990) Plant Mol. Biol. 15:127-136; bromoxynil, Stalker et al. (1988) Science 242:419-423; glyphosate, Shaw et al. (1986) Science 233:478-481; phosphinothricin, DeBlock et al. (1987) EMBO J. 6:2513-2518.<br>
[0155]	Another class of useful marker genes for plant transformation with the DNA<br>
sequence requires screening of presumptively transformed plant cells rather than direct genetic selection of transformed cells for resistance to a toxic substance such as an antibiotic. These genes are particularly useful to quantitate or visualize the spatial pattern of expression of the DNA sequence in specific tissues and are frequently referred to as reporter genes because they can be fused to a gene or gene regulatory sequence for the investigation of gene expression. Commonly used genes for screening presumptively transformed cells include p-glucuronidase (GUS), p-galactosidase, luciferase, and chloramphenicol c:ctyltransfcrase (Jefferson, Plant Mol. Biol. Rep. 5:387 (1987); Tceri et al, EMBO J. 8:343 (1989); Koncz et al., Proc. Nat'l Acad. Set. (USA) 84:131 (1987); De Block et al., EMBQ J. 3:1681 (1984)). Examples of other suitable reporter genes known in the art can be found in, for example: Jefferson et al. (1991) in Plant Molecular Biology Manual, ed. Gelvin et al. (Kluwer Academic Publishers), pp. 1-33; DeWet et al. (1987) Mol. Cell. Biol. 7:725-737; Goff et al. (1990) EMBO J. 9:2517-2522; Kain et al. (1995) BioTeclmiques 19:650-655; and Chiu et al (1996) Current Biology 6:325-330. Another approach to the identification of relatively rare transformation events has been use of a gene that encodes a dominant constitutive regulator of the Zea mays anthocyanin pigmentation pathway (Ludwig et al, Science 247:449 (1990)).<br>
40156]	The appropriate DNA sequence may be inserted into the vector by any of a<br>
variety of well-known and routine techniques. In general, a DNA sequence for expression is joined to an expression vector by cleaving the DNA sequence and the expression vector with one or more restriction endonucleases and then joining the restriction fragments together using T4 DNA ligase. The sequence may be inserted in a forward or reverse orientation. Procedures for restriction and ligation that can be used to this end are well known and routine to those of skill. Suitable procedures in this regard, and for constructing expression vectors using alternative techniques, which also are well known and routine to those of skill, are set forth in great detail in Sambrook et al, MOLECULAR CLONING, A LABORATORY MANUAL, 2nd Ed.; Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York (1989).<br>
[0157]	A polynucleotide of the invention, optionally encoding the heterologous<br>
structural sequence of a polypeptide of the invention, generally will be inserted into the vector using standard techniques so that it is operably linked to the promoter for expression. (Operably linked, as used herein, includes reference to a functional linkage between a promoter and a second sequence, wherein the promoter sequence initiates and mediates transcription of the DNA corresponding to the second sequence. Generally, operably linked means that the nucleic acid sequences being linked are contiguous and, where necessary to join two protein coding regions, contiguous and in the same reading frame.) When the polynucleotide is intended for expression of a polypeptide, the polynucleotide will be positioned so that the transcription start site is located appropriately 5' to a ribosome binding site. The ribosome-binding site will be 5' to the AUG that initiates translation of the polypeptide to be expressed. Generally, there will be no other open reading frames that begin with an initiation codon, usually AUG, and lie between the ribosome binding site and the initiation codon. Also, generally, there will be a translation stop codon at the end of the polypeptide and there will be a polyadenylation signal in constructs for use in eukaryotic hosts. Transcription termination signals appropriately disposed at the 3' end of the transcribed region may also be included in the polynucleotide construct.<br>
[0158]	For nucleic acid constructs designed to express a polypeptide, the expression<br>
cassettes can additionally contain 5' leader sequences. Such leader sequences can act to enhance translation. Translation leaders are known in the art and include: picornavirus leaders, for example: EMCV leader (Encephalomyocarditis 5' noncoding region), Elroy-<br>
Stein et al. (19S9J Proc. Nat. Acad. Sti. USA 86:6126-6130; potyvirus leaders, for example, 'atev leader (Tobacco Etch Virus), Allison et al. (1986); MDMV leader (Maize Dwarf Mosaic Vims), Virology 154:9-20; human immunoglobulin heavy-chain binding protein (BiP), Macejak et al. (1991) Nature 353:90-94; untranslated leader from the coat protein mRNA of alfalfa mosaic virus (AMV RNA 4), Jobling et al. (1987) Nature 325:622-625); tobacco mosaic virus-leader (TMV), Gallic et al. (1989) Molecular Biology of RNA, pages 237-256; and maize chlorotic mottle virus leader (MCMV) Lommel et al. (1991) Virology 81:382-385. See also Della-Cioppa et al. (1987) Plant Physiology 84:965-968. The cassette can also contain sequences that enhance translation and/or mRNA stability such as introns. The expression cassette also typically includes, at the 3' terminus of the isolated nucleotide sequence of interest, a translational"termination region, e.g., one functional in plants.<br>
[0159]	In those instances where it is desirable to have the expressed product of the<br>
isolated nucleotide sequence directed to a particular organelle, particularly the plastid, amyloplast, or to the endoplasmic reticulum, or secreted at the cell's surface or extracellularly, the expression cassette can further comprise a coding sequence for a transit peptide. Such transit peptides are well known in the art and include, but are not limited to: the transit peptide. for the acyl carrier protein, the small subunit of RTJBISCO, plant EPSP synthase, and the like.<br>
[0160]	In preparing the expression cassette, the various DNA fragments can be<br>
manipulated, so as to provide for the DNA sequences in the proper orientation and, as appropriate, in the proper reading frame. Toward this end, adapters or linkers can be employed to join the DNA fragments or other manipulations can be involved to provide for convenient restriction sites, removal of superfluous DNA, removal of restriction sites, or the like. For this purpose, in vitro mutagenesis, primer repair, restriction digests, annealing, and resubstitutions such as transitions and transversions, can be involved.<br>
[0161]	As noted herein, the present invention provides vectors capable of expressing<br>
genes of interest under the control of the regulatory elements. In general, the vectors should be functional in plant cells. At times, it may be preferable to have vectors that are functional in other host cells, e.g., in E. coli (e.g., for production of protein for raising antibodies, DNA sequence analysis, construction of inserts, or obtaining quantities of<br>
nucleic acids). Vectors and procedures for cloning and expression in E. coli are discussed in Sambrook et al. (supra).<br>
[0162]	The transformation vector, comprising the promoter of the present invention<br>
operably linked to an isolated nucleotide sequence in an expression cassette, can also contain at least one additional nucleotide sequence for a gene to be co-transformed into the organism. Alternatively, the additional sequence(s) can be provided on another transformation vector.<br>
[0163]	The vector containing the appropriate DNA sequence as described elsewhere<br>
herein, as well as an appropriate promoter, and other appropriate control sequences, may be introduced into an appropriate host using a variety of well-known techniques suitable to expression therein of a desired RNA and/or polypeptide. The present invention also relates to host cells containing the above-described constructs. The host cell can be a higher eukaryotic cell, such as a plant cell, or a lower eukaryotic cell, such as a yeast cell, or the host cell can be a prokaryotic cell, such as a bacterial cell.<br>
[0164]	Introduction of the construct into the host cell can be effected by calcium<br>
phosphate transfection, DEAE-dextran mediated transfection, microinjection, cationic lipid-mediated transfection, electroporation, transduction, scrape loading, ballistic introduction, infection or other methods. Such methods are described in many standard laboratory manuals, such as Davis etal, BASIC METHODS IN MOLECULAR BIOLOGY, (1986) and Sambrook et al, MOLECULAR CLONING: A LABORATORY MANUAL, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989).<br>
[OltoS]	Representative examples of appropriate hosts include bacterial cells, such as<br>
streptococci, staphylococci, E. coli, streptomyces and Salmonella typhimurium cells; fungal cells, such as yeast cells and Aspergillus cells; insect cells such as Drosophila S2 and Spodoptera Sf9 cells; animal cells such as CHO, COS and Bowes melanoma cells; and plant cells. The plant cells may be derived from a broad range of plant types, particularly monocots such as the species of the Family Graminiae including Sorghum bicolor and 2&amp;a mays, as well as dicots such as soybean (Glycine max) and canola (Brassica napus, Brassica rapa ssp.). Preferably, plants include maize, soybean, sunflower, safflower, canola, wheat, barley, rye, alfalfa/rice, oat, lawn grass, and sorghum; however, the isolated nucleic acid and proteins of the present invention can be used in species from the genera: Ananas,<br>
Antirrhinum, Arabidopsis, Arachis, Asparagus, Atropa, Avena, Brassica, Bromus, Browaalia, Camellia, Capsicum, Ciahorium, Citrus, Cocos, Cofea, Cucumis, Cucurbita, Datura, Daucus, Digitalis, Ficus, Fragaria, Geranium,Glydne, Gossypium, Helianthus, Heterocallis, Hordeum, Hyoscyamus, Ipomoea, Juglans, Lactuca, Linum, Lolium, Lotus, Lycopersicon, Majorana, Mangifera, Manihot, Medicago. Musa, Nemesis, Nicotiana, Olea, Onobrychis, Oryza, Panieum, Pelargonium, Pennisetum, Persea, Petunia, Phaseolus, Pisum, Psidium, Ranunculus, Raphanus, Rosa, Salpiglossis, Secale, Senecio, Solatium, Sinapis, Sorghum, Theobroma, Triticum, Trifolium, Trigonella, Vigna, Vitis, and Zea,among many other examples (e.g., other genera noted herein).<br>
[0166]	The promoter regions of the invention may be isolated from any plant,<br>
including, but not limited to, maize (corn; Zea mays), canola (Brassica napus, Brassica rapa ssp.), alfalfa (Medicago saliva), rice (Oryza saliva), rye (Secale cereale), sorghum (Sorghum bicolor, Sorghum vulgare), sunflower (Helianthus animus), wheat (Triticum aestivum), soybean (Glycine max), tobacco (Nicotiana labacwn), potato (Solanum tuberosiim), peanuts (Arachis hypogaea), cotton (Gossypium hirsutwn), sweet potato (Ipomoea batatus), cassava (Manihot esculenta), coffee (Cofea spp.), coconut (Cocos nucifera), pineapple (Ananas comosus), citrus trees (Citrus spp.), cocoa (Theobroma cacao), tea (Camellia sinensis), banana (Musa spp), avocado (Persea americana), fig (Ficus casica), guava (Psidium guajava), mango (Mangifera indica), olive (Olea europaea), oats, barley, vegetables, ornamentals, and conifers. Preferably, plants include maize, soybean, sunflower, safflower, canola, wheat, bailey, rye, alfalfa, rice, oat, lawn grass, and sorghum.<br>
[0167]	Hosts for a great variety of expression constructs are well known, and those<br>
of skill will be enabled by the present disclosure readily to select a host for expressing a polypeptide in accordance with this aspect of the present invention.<br>
[0168]	The engineered host cells can be cultured in conventional nutrient media,<br>
which may be modified as appropriate for, inter alia, activating promoters, selecting transformants or amplifying genes. Culture conditions, such as temperature, pH and the like, previously used with the host cell selected for expression generally will be suitable for expression of nucleic acids and/or polypeptides of the present invention, as will be apparent to those of skill in the art.<br>
[0169]	Mature proteins can be expressed in mammalian cells, yeast, bacteria, or<br>
other cells under the control of appropriate promoters. Cell-free translation systems can also be employed to produce such proteins using RNAs derived from the DNA constructs of the present invention.<br>
[0170]	Following transformation of a suitable host strain and growth of the host<br>
strain to an appropriate cell density, where the selected promoter is inducible it is induced by appropriate means (e.g., temperature shift or exposure to chemical inducer) and cells are cultured for an additional period.<br>
[0171]	Cells typically then are harvested by centrifugation, disrupted by physical or<br>
chemical means, and the resulting crude extract retained for further purification. Microbial cells employed in expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical disruption, or use of cell lysing agents; such methods are well known to those skilled in the art.<br>
METHODS OF INHIBITING ETHYLENE PRODUCTION<br>
[0172]	The invention also provides methods for inhibiting ethylene production in a<br>
plant (and plants produced by such methods). For example, a method of inhibiting ethylene production comprises inactivating one or more ACC synthase genes in the plant, wherein the one or more ACC synthase genes encodes one or more ACC synthases. Typically, at least one of the one or more ACC synthases comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more identity to SEQ ID NO:7 (pACS2), SEQ ID NO:8 (pACS6), SEQ ID NO:9 (pAC7) or SEQ ID NO: 11 (pCCRA178R).<br>
Antisense, Sense. RNA Silencing or Interference Configurations<br>
[0173]	The one or more ACC synthase gene can be inactivated by introducing and<br>
expressing transgenic sequences, e.g., antisense or sense configurations, or RNA silencing or interference configurations, etc, encoding one or more ACC synthases, or a subsequence thereof, and a promoter, thereby inactivating the one or more ACC synthase genes compared to a corresponding control plant (e.g., its non-transgenic parent or a non-transgenic plant of the same species). See also the section entitled "Polynucleotides of the Invention." The at least one polynucleotide sequence can be introduced by techniques including, but not limited to, e.g., electroporation, micro-projectile bombardment,<br>
Agrobacterium-mediated transfer, or other available methods. See also, the section entitled "Plant Transformation," herein. In certain aspects of the invention, the polynucleotide is linked to the promoter in a sense orientation or in an antisense orientation or is configured for RNA silencing or interference.<br>
[0174]	In certain situations it may be preferable to silence or down-regulate certain<br>
genes, such as ACC synthase genes. Relevant literature describing the application of homology-dependent gene silencing includes: Jorgensen, Trends Biotechnol. 8 (12):340-344 (1990); Flavell, Proc. Nat'l. Acad. Sci. (USA) 91:3490-3496 (1994); Finnegan et al., Bio/Technology 12: SS3-888 (1994); Neuhuber et al., Mol. Gen. Genet. 244:230-241 (1994); Flavell et al. (1994) Proc. Natl. Acad. Sci. USA 91:3490-3496; Jorgensen et al. (1996) Plant Mol, Biol. 31:957-973; Johansen and Camngton (2001) Plant Physiol. 126:930-938; Broin et al. (2002) Plant Cell 14:1417-1432; Stoutjesdijk et al (2002) Plant Physiol 129:1723-1731; Yu et al. (2003) Phytochemistry 63:753-763: and U.S. Patent Nos. 5,034,323, 5,283,184, and 5,942,657. Alternatively, another approach to gene silencing can be with the use of antisense technology (Rothstein et al in Plant Mol. Cell. Biol. 6:221-246 (1989); Liu et al (2002) Plant Physiol 129:1732-1743 and U.S. Patent Nos. 5,759,829 and 5,942,657.<br>
[0175]	Use of antisense nucleic acids is well known in the art. .An. antisense nucleic<br>
acid has a region of complementarity to a target nucleic'acid, e.g., an ACC synthase gene, mRNA, or cDNA. The antisense nucleic acid can be RNA, DNA, a PNA or any other appropriate molecule.  A duplex can form between the antisense sequence and its complementary sense sequence, resulting in inactivation of the gene. The antisense nucleic acid can inhibit gene expression by forming a duplex with an RNA transcribed from the gene, by forming a triplex with duplex DNA, etc. An antisense nucleic acid can be produced, e.g., for an ACC synthase gene by a number of well-established techniques (e.g., chemical synthesis of an antisense RNA or oligonucleotide (optionally including modified nucleotides and/or linkages that increase resistance to degradation or improve cellular uptake) or in vitro transcription). Antisense nucleic acids and their use are described, e.g., in USP 6,242,258 to Haselton and Alexander (June 5,2001) entitled "Methods for the selective regulation of DNA and RNA transcription and translation by photoactivation"; USP 6,500,615; USP 6,498,035; USP 6,395,544; USP 5,563,050; E. Schuch et al (1991) Svtnn Soc. Exo Biol 45:117-127: de Lange et al., (1995) Gun-Top Microbiol Immunol 197:57-75; Hamilton et al. (1995) Curr Top Microbiol Immunol 197:77-89; Finnegan et al.,<br>
(1996) Proc Natl Acad Sci USA 93:8449-8454; Uhlmann and A. Pepan (1990), Chem. Rev. 90:543; P. D. Cook (1991), Anti-Cancer Drug Design 6:585; J. Goodchild, Bioconjugate Chem. 1 (1990) 165; and, S. L. Beaucage and R. P. Iyer (1993), Tetrahedron 49:6123; and F. Eckstein, Ed. (1991), Oligonucleotides and Analogues-A Practical Approach. IRL Press.<br>
[0176]	Catalytic RNA molecules or ribozymes can also be used to inhibit expression<br>
of ACC synthase genes. It is possible to design ribozymes that specifically pair with virtually any target RNA and cleave the phosphodiester backbone at a specific location, thereby functionally inactivating the target RNA. In carrying out this cleavage, the ribozyme is not itself altered, and is thus capable of recycling and cleaving other molecules. The inclusion of ribozyme sequences within antisense RNAs confers RNA-cleaving activity upon them, thereby increasing the activity of the constructs.<br>
[0177]	A number of classes of ribozymes have been identified. For example, one<br>
class of ribozymes is derived from a number of small circular RNAs that are capable of self-cleavage and replication in plants. The RNAs can replicate either alone (viroid RNAs) or with a helper virus (satellite RNAs). Examples of RNAs include RNAs from avocado sunblotch viroid and the satellite RNAs from tobacco ringspot virus, lucerne transient streak virus, velvet tobacco mottle virus, solanum nodiflorum mottle virus and subterranean clover mottle virus. The design and use of target RNA-specific ribozymes has been described. See, e.g., Haseloff et al. (1988) Nature. 334:585-591.<br>
[0178]	Another method to inactivate an ACC synthase gene by inhibiting expression<br>
is by sense suppression. Introduction of expression cassettes in which a nucleic acid is configured in the sense orientation with respect to the promoter has been shown to be an effective means by which to block the transcription of a desired target gene. See, e.g., Napoli et al. (1990), The Plant Cell 2:279-289, and U.S. Pat. Nos. 5,034,323, 5,231,020, and 5,283,184.<br>
[0179]	Isolated or recombinant plants which include one or more inactivated ACC<br>
synthase gene can also be produced by using RNA silencing or interference (RNAi), which can also be termed post-transcriptional gene silencing (PTGS) or cosuppression. In the context of this invention, "RNA silencing" (also called RNAi or RNA-mediated interference) refers to any mechanism through which the presence of a single-stranded or, typically, a double-stranded RNA in a cell results in inhibition of expression of a target gene<br>
comprising a sequence identical or nearly identical to that of the RNA, including, but not limited to, RNA interference, repression of translation of a target mRNA transcribed from the target gene without alteration of the mRNA's stability, and transcriptional silencing (e.g., histone acetylation and heterochromatin formation leading to inhibition of transcription of the target mRNA). In "RNA interference," the presence of the single-stranded or double-stranded RNA in the cell leads to endonucleolytic cleavage and then degradation of the target mRNA.<br>
[0180]	In one embodiment, a transgene (e.g., a sequence and/or subsequence of an<br>
ACC synthase gene or coding sequence) is introduced into a plant cell to inactivate one or more ACC synthase genes by RNA silencing or interference (RNAi). For example, a sequence or subsequence includes a small subsequence, e.g., about 21-25 bases in length (with, e.g., at least 80%, at least 90%, or about 100% identity to one or more of the ACC synthase gene subsequences), a larger subsequence, e.g., about 25-100 or about 100-2000 (or about 200-1500, about 250-1000, etc.) bases in length (with at least one region of about 21-25 bases of at least 80%, at least 90%, or 100% identity to one or more ACC synthase gene subsequences), and/or the entire coding sequence or gene. In one embodiment, a transgene includes a region in the sequence or subsequence that is about 21-25 bases in length with at least 80%, at least 90%, or about 100% identity to the ACC synthase gene or coding sequence.<br>
[0181]	Use of RNAi for inhibiting gene expression in a number of cell types<br>
(including, e.g., plant cells) and organisms, e.g., by expression of a hairpin (stem-loop) RNA or of the two strands of an interfering RNA, for example, is well described in the literature, as are methods for determining appropriate interfering RNA(s) to target a desired gene, e.g., an ACC synthase gene, and for generating such interfering RNAs. For example, RNA interference is described e.g., in US patent application publications 20020173478, 20020162126, and 20020182223 and in Cogoni and Macino (2000) "Post-transcriptionnal gene silencing across kingdoms" Genes Dev., 10:638-643; Guru T. (2000), "A silence that speaks volumes" Nature 404: 804-808; Hammond et al., (2001), "Post-transcriptional Gene Silencing by Double-stranded RNA" Nature Rev. Gen. 2: 110-119; Napoli et al., (1990) "Introduction of a chalcone synthase gene into Petunia results in reversible co-suppression of homologous genes in trans" Plant Cell 2:279-289; Jorgensen et al., (1996), "Chalcone synthase cosuppression phenotypes in petunia flowers: comparison of sense vs. antisense<br>
constructs and single-copy vs. complex T-DNA sequences" Plant Mol.Biol.. 31:957-973; l&amp;mon GJ. (2002) "RNA interference" Nature.. Jul 11; 418(6894):244-51; UedaR. (2001) "RNAi: a new technology in the post-genomte sequencing era" J Neurogenet.;15(3-4):193-204; Ullu et al (2002) "RNA interference: advances and questions" Philos Trans R Soc Lond B Biol Sci. Jan 29;357(1417):65-70; Waterhouse et al., (1998) Proc Natl Acad Sci " USA 95:133959-13964;Schmid et al (2002) "Combinatorial RNAi: a method for evaluating the functions of gene families in Drosophila" Trends NeuroscL Feb;25(2V.71-4; Zeng et al. (2003) "MicroRNAs and small interfering RNAs can inhibit mRNA expression by similar mechanisms" Proc. Natl. Acad. Sci. USA 100:9779-9784; Doench et al. (2003) "siRNAs can function as miRNAs" Genes &amp; Dev. 17:438-442; Bartel and Bartel (2003) "MicroRNAs: At the root of plant development?" Plant Physiology 132:709-717; Schwarz and Zamore<br>
(2002)	"Wlvy do miRNAs live in the miRNP?" Genes &amp; Dev. 16:1025-1031; Tang et al.<br>
(2003)	"A biochemical framework for RNA silencing in plants" Genes &amp; Dev. 17:49-63;<br>
Meister et al. (2004) "Sequence-specific inhibition ofmicroRNA- and siRNA-induced RNA<br>
silencing' RNA 10:544-550; Nelson et al. (2003) "Vie microRNA world: Small is mighty"<br>
Trends Biochem. Sci. 28:534-540; Dykxhoorn et al. (2003) "Killing the messenger: Short<br>
RNAs that silence gene expression" Nature Reviews Molec. and Cell Biol. 4:457-467;<br>
McManus and Sharp (2002) "Gene silencing in mammals by small interfering RNAs"<br>
Nature Reviews Genetics 3:737-747; Hutvagner and Zamore (2002) "RNAi: Nature abhors<br>
a double strand Curr Opin Genet &amp; Dev 200:225-232; and Agami (2002) "RNAi and<br>
related mechanisms and their potential use for tJierapy" Curr Opin Chem Biol 6:829-834.<br>
[0182]	The ACC synthase polynucleotide sequence(s) or subsequence(s) expressed<br>
to induce RNAi can be expressed, e.g., under control of a constitutive promoter, an inducible promoter, or a tissue specific promoter. Expression from a tissue-specific promoter can be advantageous in certain embodiments. For example, expression from a<br>
}<br>
leaf-specific promoter can inactivate one or more ACC synthase genes in the leaf, producing a staygreen phenotype, without inactivating ACC synthase genes in the root (which can decrease flood tolerance). Similarly, expression from an anther-specific promoter can inactivate one or more ACC synthase genes in the anther, producing a male sterility phenotype, without inactivating ACC synthase genes in the remainder of the plant. Such approaches are optionally combined, e.g., to inactivate one or more ACC synthase genes in both leaves and anthers.<br>
Transposons<br>
J0183]	The one or more ACC synthase genes can also be inactivated by, e.g.,<br>
transposon based gene inactivation. In one embodiment, the inactivating step comprises producing one or more mutations in an ACC synthase gene sequence, where the one or more mutations in the ACC synthase gene sequence comprise one or more transposon insertions, thereby inactivating the one or more ACC synthase gene compared to a corresponding control plant. For example, the one or more mutations comprise a homozygous disruption in the one or more ACC synthase gene or the one or more mutations comprise a heterozygous disruption in the one or more ACC synthase gene or a combination of both homozygous disruptions and heterozygous disruptions if more than one ACC synthase gene is disrupted.<br>
[0184]	Transposons were first identified in maize by Barbara McClintock in the late<br>
1940s. The Mutator family of transposable elements, e.g., Robertson's Mutator (Mu) transposable elements, are typically used in plant, e.g., maize, gene mutagenesis, because they are present in high copy number (10-100) and insert preferentially within and around<br>
genes.<br>
[0185]	Transposable elements can be categorized into two broad classes based on<br>
their mode of transposition. These are designated Class I and Class II; both have applications as mutagens and as delivery vectors. Class I transposable elements transpose by an RNA intermediate and use reverse transcriptases, i.e., they are retroelements. There are at least three types of Class I transposable elements, e.g., retrotransposons, retroposons, SINE-like elements.<br>
[0186]	Retrotransposons typically contain LTRs, and genes encoding viral coat<br>
proteins (gag) and reverse transcriptase, RnaseH, integrase and polymerase (pol) genes. Numerous retrotransposons have been described in plant species. Such retrotransposons mobilize and translocate via a RNA intermediate in a reaction catalyzed by reverse transcriptase and RNase H encoded by the transposon. Examples fall into the Tyl-cop/a and Ty3-gypsy groups as well as into the SINE-like and LENE-like classifications. A more<br>
i<br>
detailed discussion can be found in Kumar and Bennetzen (1999) Plant Retrotransposons in Annual Review of Genetics 33:479. In addition, DNA transposable elements such as Ac, Taml and En/Spm are also found in a wide variety of plant species, and can be utilized in the invention.<br>
FOIST]	Transposons (and IS elements) are common tools for introducing mutations<br>
n plant cells. These mobile genetic elements are delivered to cells, e.g., through a sexual cross, transposition is selected for and the resulting insertion mutants are screened, e.g., for a phenotype of interest. Plants comprising disrupted ACC synthase genes can be introduced into other plants by crossing the isolated or recombinant plants with a non-disrupted plant, e.g., by a sexual cross. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed.  The location of a TN within a genome of an isolated or recombinant plant can be determined by known methods, e.g., sequencing of flanking regions as described herein. For example, a PCR reaction from the plant can be used to amplify the sequence, which can then be diagnostically sequenced to confirm its origin. Optionally, the insertion mutants are screened for a desired phenotype, such as the inhibition of expression or activity of ACC synthase, inhibition or reduced production of ethylene, staygreen potential, etc. compared to a control plant.<br>
TILLING<br>
[0188]	TILLING can also be used to inactivate one or more ACC synthase gene.<br>
TILLING is Targeting Induced Local Lesions IN Genomics. See, e.g., McCallum et al., (2000), "Targeting Induced Local Lesions IN Genomics (TILLING) for Plant Functional Genomics" Plant Physiology 123:439-442; McCallum et al., (2000) "Targeted screening for induced mutations" Nature Biotechnology 18:455-457; and, Colbert et al., (2001) "High-Throughput Screening for Induced Point Mutations" Plant Physiology 126:480-484.<br>
[0189]	TILLING combines high density point mutations with rapid sensitive<br>
detection of the mutations. Typically, ethylmethanesulfonate (EMS) is used to mutagenize plant seed. EMS alkylates guanine, which typically leads to mispairing. For example, seeds are soaked in an about 10-20 mM solution of EMS for about 10 to 20 hours; the seeds are washed and then sown. The plants of this generation are known as Ml. Ml plants are then self-fertilized. Mutations that are present in cells that form the reproductive tissues are inherited by the next generation (M2). Typically, M2 plants are screened for mutation in the desired gene and/or for specific phenotypes.<br>
[0190]	For example, DNA from M2 plants is pooled and mutations in an ACC<br>
synthase gene are detected by detection of heteroduplex formation. Typically, DNA is prepared from each M2 plant and pooled. The desired ACC synthase gene is amplified by PCR. The pooled sample is then denatured and annealed to allow formation of<br>
heteroduplexes. If a mutation is present in one of the plants; the PCR products will be of &gt;two types: wild-type and mutant. Pools that include the heteroduplexes are identified by separating the PCR reaction, e.g., by Denaturing High Performance Liquid Chromatography (DPHPLC). DPHPLC detects mismatches in heteroduplexes created by melting and annealing of heteroallelic DNA. Chromatography is performed while heating the DNA. Heteroduplexes have lower thermal stability and form melting bubbles resulting in faster movement in the Chromatography column. When heteroduplexes are present in addition to the expected homoduplexes, a double peak is seen. As a result, the pools that carry the mutation in an ACC synthase gene are identified. Individual DNA from plants that make up the selected pooled population can then be identified and sequenced. Optionally, the plant possessing a desired mutation in an ACC syntfiase can be crossed with other plants to remove background mutations.<br>
[0191]	Other mutagenic methods can also be employed to introduce mutations in an<br>
ACC synthase gene. Methods for introducing genetic mutations into plant genes and selecting plants with desired traits are well known. For instance, seeds or other plant material can be treated with a mutagenic chemical substance, according to standard techniques. Such chemical substances include, but are not limited to, the following: diethyl sulfate, ethylene imine, and N-nitroso-N-ethylurea. Alternatively, ionizing radiation from sources such as X-rays or gamma rays can be used.<br>
[0192]	Other detection methods for detecting mutations in an ACC synthase gene<br>
can be employed, e.g., capillary electrophoresis (e.g., constant denaturant capillary electrophoresis and single-stranded conformational polymorphism). In another example, heteroduplexes can be detected by using mismatch repair enzymology (e.g., CEL I endonuclease from celery). CEL I recognizes a mismatch and cleaves exactly at the 3' side of the mismatch. The precise base position of the mismatch can be determined by cutting with the mismatch repair enzyme followed by, e.g., denaturing gel electrophoresis.  See, e.g., Oleykowski et al., (1998) "Mutation detection using a novel plant endonuclease" Nucleic Acid Res. 26:4597-4602; and, Colbert et al., (2001) "High-Throughput Screening for Induced Point Mutations" Plant Physiology 126:480-484.<br>
[0193]	The plant containing the mutated ACC synthase gene can be crossed with<br>
other plants to introduce the mutation into another plant. This can be done using standard breeding techniques.<br>
Homologous Recombination<br>
K01 94]	Homologous recombination can also be used to inactivate one or more ACC<br>
synthase genes.  Homologous recombination has been demonstrated in plants. See, e.g., Puchta et al. (1994), Experientia 50: 277-284; Swoboda et al. (1994), EMBO J. 13: 484-489; Offringa et al. (1993), Proc. Natl. Acad. Sci. USA 90: 7346-7350; Kempin et al. (1997) - Nature 389:802-803; and, Terada et al., (2002) "Efficient gene targeting by homologous recombination in rice" Nature Biotechnology. 20(10): 1030-1034.<br>
[0195]	Homologous recombination can be used to induce targeted gene<br>
modifications by specifically targeting an ACC synthase gene in vivo. Mutations in selected portions of an ACC synthase gene sequence (including 5' upstream, 3' downstream, and intragenic regions) such as those provided herein are made in vitro and introduced into the desired plant using standard techniques. The mutated gene will interact with the target ACC synthase wild-type gene in such a way that homologous recombination and targeted replacement of the wild-type gene will occur in transgenic plants, resulting in suppression of ACC synthase activity.<br>
METHODS FOR MODULATING STAYGREEN POTENTIAL IN A PLANT<br>
[0196]	Methods for modulating staygreen potential in plants are also features of the<br>
invention. The ability to introduce different degrees of staygreen,potential into plants offers a flexible and simple approach to introduce this trait in a purpose-specific manner: for example, introduction of a strong staygreen trait for improved grain-filling or for silage in areas with longer or dryer growing seasons versus the introduction of a moderate staygreen trait for silage in ar^ns with shorter growing seasons. In addition, the staygreen potential of a plant of the invention can include, e.g., (a) a reduction in the production of at least one ACC synthase specific mRNA; (b) a reduction in the production of an ACC synthase; (c) a reduction in the production of ethylene; (d) a delay in leaf senescence; (e) an increase of drought resistance; (f) an increased time in maintaining photosynthetic activity; (g) an increased transpiration; (h) an increased stomatal conductance; (i) an increased CO2 assimilation; (j) an increased maintenance of CO2 assimilation; or (k) any combination of (a)-(j); compared to a corresponding control plant, and the like.<br>
[0197]	For example, a method of the invention can include: a) selecting at least one<br>
ACC synthase gene to mutate, thereby providing at least one desired ACC synthase gene; b)<br>
introducing a mutant form of the at least one desired ACC synthase gene into the plant; and, £) expressing the mutant form, thereby modulating staygreen potential in the plant.  Plants produced by such methods are also a feature of the invention.<br>
[0198]	The degree of staygreen potential introduced into a plant can be determined<br>
by a number of factors, e.g., which ACC synthase gene is selected, whether the mutant gene member is present in a heterozygous or homozygous state, or by the number of members of this family which are inactivated, or by a combination of two or more such factors. In one embodiment, selecting the at least one ACC synthase gene comprises determining a degree (e.g., weak (e.g., ACS2), moderate or strong (e.g., ACS6)) of staygreen potential desired. For example, ACS2 lines show a weak staygreen phenotype, with a delay in senescence of about 1 week. The ACS6 lines show a strong staygreen phenotype (e.g., leaf senescence is delayed about 2-3 weeks or more). The ACS7 lines can also show a strong staygreen phenotype (e.g., leaf senescence is delayed about 2-3 weeks or more). For example, the ACC synthase gene is selected for encoding a specific ACC synthase, such as, SEQ ID NO:7 (pACS2), SEQ ID NO:S (pACS6), SEQ ID NO:9 (pAC7), or SEQ ID NO:11 (pCCRA178R). In one embodiment, two or more ACC synthase genes are disrupted (e.g., ACS2 and ACS6), e.g., to produce a strong staygreen phenotype. In other embodiments, three or more ACC synthase genes are disrupted.<br>
[0199]	Once the desired ACC synthase gene is selected, a mutant form of the ACC<br>
synthase gene is introduced into a plant. In certain embodiments, the mutant form is introduced by Agrobacterium-mediated transfer, electroporation, micro-projectile bombardment, homologous recombination or a sexual cross. In certain embodiments, the mutant form includes, e.g., a heterozygous mutation in the at least one ACC synthase gene, a homozygous mutation in the at least one ACC synthase gene or a combination of homozygous mutation and heterozygous mutation if more than one ACC synthase gene is selected. In another embodiment, the mutant form includes a subsequence of the at least one desired ACC synthase gene in an antisense, sense or RNA silencing or interference configuration.<br>
[0200]	Expression of the mutant form of the ACC synthase gene or result of<br>
expression of the mutant form can be determined in a number of ways. For example, detection of expression products is performed either qualitatively (presence or absence of one or more product of interest) or quantitatively (by monitoring the level of expression of<br>
one or more product of interest). In one embodiment, the expression product is an RNA impression product. The invention optionally includes monitoring an expression level of a nucleic acid or polypeptide as noted herein for detection of ACC synthase in a plant or in a population of plants. Monitoiing levels of ethylene or ACC can also be used for detection of inhibition of expression or activity of a mutant form of the ACC synthase gene.<br>
[0201]	In addition to increasing tolerance to drought stress in plants of the invention<br>
compared to a control plant, another important aspect of the invention is that higher density planting of plants of the invention can be possible, leading to increased yield per acre of corn. Most of the increase yield per acre of com over the last century has come from increasing tolerance to crowding, which is a stress in, e.g., maize. Methods for modulating stress, e.g., increasing tolerance for crowding, in a plant are also a feature of the invention. For example, a method of the invention can include: a) selecting at least one ACC synthase gene to mutate, thereby providing at least one desired ACC synthase gene; b) introducing a mutant form of the at least one desired ACC synthase gene into the plant; and, c) expressing the mutant form, thereby modulating stress in the plant.  Plants produced by such methods are also a feature of the invention. When the ethylene production is reduced in a plant by a mutant form of a desired ACC synthase gene, the plant does not perceive crowding. Thus, plants of the invention can be planted at higher density than currently practiced by farmers.<br>
[0202]	In another aspect, inactivation of one or more ACC synthase genes as<br>
described herein can influence response to disease or pathogen attack.<br>
METHODS FOR MODULATING STERILITY IN A PLANT<br>
[0203]	Methods for modulating sterility, e.g., female or male sterility, in plants are<br>
also features of the invention. The ability to introduce female or male sterility into plants permits rapid production of female or male sterile lines, e.g., for use in commercial breeding programs, e.g., for production of hybrid seed, where cross-pollination is desired.<br>
[0204]	ACC synthase knockout plants, particularly ACS6 knockouts and<br>
ACS2/ACS6 double knockouts, have been observed to shed less pollen than wild-type plants, suggesting disruption of ethylene production as a novel means of modulating plant sterility.<br>
[0205]	For example, a method of the invention can include: a) selecting at least one<br>
ACC synthase gene to mutate., thereby providing at least one desired ACC synthase gene; b)<br>
introducing a mutant form of the at least one desired ACC synthase gene into the plant; and, ) expressing the mutant form, thereby modulating sterility in the plant.  Plants produced by such methods are also a feature of the invention.<br>
[0206]	Essentially all of the features noted above apply to this embodiment as well,<br>
as relevant, for example, with respect to the number of ACC synthase genes disrupted, techniques for introducing the mutant form of the ACC synthase gene into the plant, polynucleotide constructs, and the like.<br>
[0207]	In one class of embodiments, the at least one ACC synthase gene is disrupted<br>
by insertion of a transposon, by a point mutation, or by constitutive expression of a transgene comprising an ACC synthase polynucleotide in an antisense, sense, or RNA silencing or interference configuration. Such lines can be propagated by exogenously providing ethylene, for example, by spraying the plants at an appropriate developmental stage with 2-chloroethylphosphonic acid (CEPA), which breaks down in water to produce ethylene.<br>
[0208]	In another class of embodiments, the at least one ACC synthase gene is<br>
disrupted by expression of a transgene comprising an ACC synthase polynucleotide in an antisense, sense, or RNA silencing or interference configuration under4he control of an inducible promoter, such that sterility can be induced and/or repressed as desired. In yet another class of embodiments, the at least one ACC synthase gene is disrupted by expression of a transgene comprising an ACC synthase polynucleotide in an antisense, sense, or RNA silencing or interference configuration under the control of a tissue-specific promoter, e.g., an anther-specific promoter to produce male sterile plants. Again, if necessary, such lines can be propagated by exogenously providing ethylene (e.g., by spraying with CEPA.<br>
SCREENING/CHARACTERIZATION OF PLANTS OR PLANT CELLS OF THE INVENTION<br>
[0209]	The plants of this invention can be screened and/or characterized either<br>
genotypically, biochemically, phenotypically or a combination of two or more of the these to determine the presence, absence, and/or expression (e.g., amount, modulation, such as a decrease or increase compared to a control cell, and the like) of a polynucleotide of the invention, the presence, absence, expression, and/or enzymatic activity of a polypeptide of<br>
the invention, modulation of stay green potential, modulation of crowding, and/or Modulation of ethylene production. See, e.g., Figure 19.<br>
[0210]	Genotypic analysis can be performed by any of a number of well-known<br>
techniques, including PCR amplification of genomic nucleic acid sequences and hybridization of genomic nucleic acid sequences or expressed nucleic acid sequences with specific labeled probes (e.g., Southern blotting, northern blotting, dot or slot blots, etc.).<br>
[0211]	For example, the Trait Utility System for Corn (TUSC), developed by<br>
Pioneer Hybrid Int., is a powerful PCR-based screening strategy to identify Mu transposon insertions in specific genes without the need for an observable phenotype. The system utilizes, e.g., TER-PCR in which one PCR primer is derived from the target gene and the other (Mu-TIR) from the terminal-inverted-repeat (TIR) region of Mu. Using these primers in PCR reactions of DNA pooled from a large population of Mu containing plants, successful amplification is identified by Southern hybridization using the target gene as the probe. Screening the individuals within a positive pool is then performed to identify the candidate line containing insertion of a Mu element in the target gene. In order to determine whether an insertion event is limited to somatic cells or is present in the germ line (and therefore represents a heritable change), progeny from a candidate are optionally subjected to the same PCR/Southem hybridization analysis used in-the original screen.<br>
[0212]	Biochemical analysis can also be performed for detecting, e.g., the presence,<br>
the absence or modulation (e.g., decrease or increase) of protein production (e.g., by ELISAs, western blots, etc.), the presence and/or amount of ethylene produced, and the like. For example, expressed polypeptides can be recovered and purified from isolated or recombinant cell cultures by any of a number of methods well known in the ait, including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography (e.g., using any of the tagging systems noted herein), hydroxylapatite chromatography, and lectin chromatography. Protein refolding steps can be used, as desired, in completing configuration of the mature protein. Finally, high performance liquid chromatography (HPLC) can be employed in the final purification steps. In addition to the references noted above, a variety of purification methods are well known in the art, including, e.g., those set forth in Sandana (1997) Bioseparation of Proteins, Academic Press, Inc.; and Bollag et al. (1996) Protein Methods. 2nd Edition Wiley-<br>
Liss, NY; Walker (1996) The Protein Protocols Handbook Humana Press, NJ, Harris and Bgal (1990) Protein Purification Applications: A Practical Approach IRL Press at Oxford,<br>
4^^<br>
Oxford, England; Harris and Angal Protein Purification Methods: A Practical Approach IRL Press at Oxford, Oxford, England; Scopes (1993) Protein Purification: Principles and Practice 3rd Edition Springer Verlag, NY; Janson and Ryden (1998) Protein Purification: ' Principles, High Resolution Methods and Applications, Second Edition Wiley-VCH, NY; and Walker (1998) Protein Protocols on CD-ROM Humana Press, NJ.<br>
[0213]	Chemicals, e.g., ethylene, ACC, etc., can be recovered and assayed from the<br>
cell extracts. For example, internal concentrations of ACC can be assayed by gas chromatography-mass spectroscopy, in acidic plant extracts as ethylene after decomposition in alkaline hypochlorite solution, etc. The concentration of ethylene can be determined by, e.g., gas chromatography-mass spectroscopy, etc. See, e.g., Nagahama, K., Ogawa, T., Fujii, T., Tazaki, M., Tanase, S., Morino, Y. and Fukuda, H. (1991) "Purification and properties of an ethyleneforming enzyme from Pseudomonas syringae" J. Gen. Microbiol. 137: 2281-2286. For example, ethylene can be measured with a gas chromatograph equipped with, e.g., an alumina based column (such as an HP-PLOT A1203 capillary column) and a flame ionization detector.<br>
[0214]	Phenotypic analysis includes, e.g., analyzing changes in chemical<br>
composition (e.g., as described under biochemical analysis), morphology, or physiological properties of the plant. For example, morphological changes can include, but are not limited to, increased staygreen potential, a delay in leaf senescence, an increase in drought resistance, an increase in crowding resistance, etc. Physiological properties can include, e.g., increased sustained photosynthesis, increased transpiration, increased stomatal conductance, increased CO2 assimilation, longer maintenance of COz assimilation, etc.<br>
[0215]	A variety of assays can be used for monitoring staygreen potential. For<br>
example, assays include, but are not limited to, visual inspection, monitoring photosynthesis measurements, and measuring levels of chlorophyll, DNA, RNA and/or protein content of, e.g., the leaves.<br>
PLANTS OF THE INVENTION<br>
[0216]	Plant cells of the invention include, but are not limited to, meristem cells,<br>
Type I, Type II, and Type III callus, immature embryos, and gamettc cells such as<br>
microspores, pollen, sperm and egg. In certain embodiments, the plant cell of the invention isAom a dicot or monocot. A plant regenerated from the plant cell(s) of the invention is<br><br>
also a feature of the invention.<br>
[0217]	In one embodiment, the plant cell is in a plant, e.g., a hybrid plant,<br>
comprising a staygreen potential phenotype. In another embodiment, the plant cell is in a plant comprising a sterility phenotype, e.g., a male sterility phenotype. Through a series of breeding manipulations, the disrupted ACC synthase gene can be moved from one plant line to another plant line. For example, the hybrid plant can be produced by sexual cross of a plant comprising a disruption in one or more ACC synthase genes and a control plant.<br>
[0218]	Knockout plant cells are also a feature of the invention. In a first aspect, the<br>
invention provides for an isolated or recombinant knockout plant cell comprising at least one disruption in at least one endogenous ACC synthase gene (e.g., a nucleic acid sequence, or complement thereof, comprising, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ ID NO:3 (gACS7)). The disruption inhibits expression or activity of at least one ACC synthase protein compared to a corresponding control plant cell lacking the disruption. In one embodiment, the at least one endogenous ACC synthase gene comprises two or more endogenous ACC synthase genes. In another embodiment, the at least one endogenous ACC synthase gene comprises three or more endogenous ACC synthase genes.   In certain embodiments, the at least one disruption results in reduced ethylene production by the knockout plant cell as compared to the control plant cell.<br>
[0219]	In one aspect of the invention, the disruption of an ACC synthase gene in a<br>
plant cell comprises one or more transposons, wherein the one or more transposons are in the at least one endogenous ACC synthase gene. In another aspect, the disruption includes one or more point mutations in at least one endogenous ACC synthase gene. Optionally, the disruption is a homozygous disruption in the at least one ACC synthase gene. Alternatively, the disruption is a heterozygous disruption in the at least one ACC synthase gene. In certain embodiments, more than one ACC synthase gene is involved and there is more than one disruption, which can include homozygous disruptions, heterozygous disruptions or a combination of homozygous disruptions and heterozygous disruptions. See also sections herein entitled "Transposons and TILLING."<br>
[0220]	In another embodiment, the disruption of an ACC synthase gene is produced<br>
inhibiting expression of the ACC synthase gene. For example, a knockout plant cell is produced by introducing at least one polynucleotide sequence comprising an ACC synthase nucleic acid sequence, or subsequence thereof, into a plant cell, such that the at least one polynucleotide sequence is linked to a promoter in a sense or antisense orientation. The polynucleotide sequence comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7), or SEQ ID NO.: 10 (CCRA178R), or a subsequence thereof, or a complement thereof. For example, the knockout plant cell Can be produced by introducing at least one polynucleotide sequence comprising one or more subsequences of an ACC synthase nucleic acid sequence configured for RNA silencing or interference. The polynucleotide optionally comprises a vector, expression cassette, or the like. In another aspect, the knockout plant cell is produced by homologous recombination. See also sections herein entitled "Antisense, Sense, RNA Silencing or Interference Configurations" and "Homologous Recombination."<br>
[0221]	Knockout plants that comprise a staygreen potential phenotype are a feature<br>
of the invention. Typically, the staygreen potential phenotype in the knockout plant results from a disruption in at least one endogenous ACC synthase gene. In one embodiment, the disruption comprises one or more transposons, and the disruption inhibits expression or activity of at least one ACC synthase protein compared to a corresponding control plant. In anot^or embodiment, the disruption comprises one or more point mutations and inhibits expression or activity of the at least one ACC synthase protein compared to a corresponding control. In certain embodiments, the at least one endogenous ACC synthase gene comprises a nucleic acid sequence, or complement thereof, comprising, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ED NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ ID NO:3 (gACS7), or a complement thereof. In certain embodiments, the knockout plant is a hybrid plant.<br>
[0222]	The invention also features knockout plants that comprise a transgenic plant<br>
with a staygreen potential phenotype. For example, a transgenic plant of the invention<br>
includes a staygreen potential phenotype resulting from at least one introduced transgene 4vhich inhibits ethylene synthesis, wherein said at least one introduced transgene comprises a nucleic acid sequence encoding at least one ACC synthase or subsequence thereof, which nucleic acid sequence comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7) or SEQ ID NO: 10 (CCRA178R), or a subsequence thereof, or a complement thereof, and modifies a level of expression or activity of the at least one ACC synthase. Typically, the configuration is a sense, antisense, or RNA silencing or interference configuration. A transgenic plant of the invention can also include a staygreen potential phenotype resulting from at least one introduced transgene which inhibits ethylene synthesis, wherein said at least one introduced transgene comprises a nucleic acid sequence encoding subsequences of at least one ACC synthase, which at least one ACC synthase comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ID N0:7 (pACS2), SEQ ID NO:8 (pACS6), SEQ ID NO.:9 (pACS7), or SEQ ID NO:1 l(pCCRA178R), or a conservative variation thereof, and is in an RNA silencing or interference configuration, and modifies a level of expression or activity of the at least one ACC synthase. In one aspect, the transgene optionally comprises a tissue-specific promoter or an inducible promoter (e.g., a leaf-specific promoter, a drought-inducible promoter, or the like).<br>
[0223]	The invention also features knockout plants that have a sterility phenotype,<br>
e.g., a male or female sterility phenotype. Thus, one class of embodiments provides a knockout plant comprising a male sterility phenotype which results from at least one disruption in at least one endogenous ACC synthase gene. The disruption inhibits expression or activity of at least one ACC synthase protein compared to a corresponding control plant. For example, ACS2, ACS6, and ACS7 can be disrupted, singly or in any combination (e.g., ACS6, or ACS2 and ACS6). Typically, the at least one disruption results in reduced ethylene production by the knockout plant as compared to the control plant.<br>
[0224]	In one embodiment, the at least one disruption comprises one or more<br>
transposons in the at least one endogenous ACC synthase gene. In another embodiment, the<br>
at least one disruption comprises one or more point mutations in the at least one endogenous ACC synthase gene. In other embodiments, the at least one disruption is introduced into the Knockout plant by introducing at least one polynucleotide sequence comprising one or more subsequences of an ACC synthase nucleic acid sequence configured for RNA silencing or interference (or, alternatively, in a sense or antisense configuration). As noted, the - polynucleotide sequence is optionally under the control of an inducible or tissue-specific (e.g., anther-specific) promoter.<br>
[0225]	In one embodiment, the male sterility phenotype comprises reduced pollen<br>
shedding by the knockout plant as compared to the control plant. For example, the knockout plant can shed at most 50%, 25%, 10%, 5%, or 1% as much pollen as the control plant, or it can shed no detectable pollen.<br>
[0226]	The invention also features knockout plants that comprise a transgenic plant<br>
with a male sterility phenotype. For example, a transgenic plant of the invention includes a male sterility phenotype resulting from at least one introduced transgene which inhibits ethylene synthesis, wherein said at least one introduced transgene comprises a nucleic acid sequence encoding at least one ACC synthase or subsequence thereof, which nucleic acid sequence comprises, e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%, about 99.5% or more, sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ID NO:6 (cACS7) or SEQ ID NO: 10 (CCRA17SR), or a subsequence thereof, or a complement thereof, and modifies a level of expression or activity of the at least one ACC synthase. Typically, the configuration is a sense, antisense, or RNA silencing or interference configuration. As noted, the transgene optionally comprises a tissue-specific promoter (e.g., an anther-specific promoter) or an inducible promoter.<br>
[0227]	Essentially any plant can be used in the methods and compositions of the<br>
invention. Such species include, but are not restricted to members of the families: Poaceae (formerly Graminae, including Zea mays (corn), rye, triticale, barley, millet, rice, wheat, oats, etc.); Leguminosae (including pea, beans, lentil, peanut, yam bean, cowpeas, velvet beans, soybean, clover, alfalfa, lupine, vetch, lotus, sweet clover, wisteria, sweetpea, etc.); Compositae (the largest family of vascular plants, including at least .1,000 genera, including important commercial crops such as sunflower) and Rosaciae (including raspberry, apricot,<br>
almond, peach, rose, etc.), as well as nut plants (including, walnut, pecan, hazelnut, etc.),<br><br>
%rest trees (including Pinus, Quercus, Pseutotsuga, Sequoia, Populus, etc.), and other<br>
common crop plants (e.g., cotton, sorghum, lawn grasses, tomato, potato, pepper, broccoli, cabbage, etc.)<br>
[0228]	Additional plants, as well as those specified above, include plants from the<br>
genera: Acamptoclados, Achnatherum, Achnella, Acroceras, Aegilops, Aegopgon, Agroelymus, Agrohordeum, Agropogon, Agropyron, Agrositanion, Agrostis, Aira, Allolepis, Alloteropsis, Alopecums, Amblyopyrwn, Ammophila, Ampelodesmos, Amphibromus, Amphicarpum, Amphilophis, Anastrophus, Anatherum, Andropogron, Anemathele, Aneurolepidium, Anisantha, Anthaenantia, Anthephora, Anthochloa, Anthoxanthum, Apera, Apluda, Archtagrostis, Arctophila, Argillochlod, Aristida, Arrhenatherum, Arthraxon, Anhrostylidium, Arundinaria, Anmdinella, Arundo, Aspris, Atheropogon, Avena (e.g., oats), Avenella, Avenochloa, Avenula, Axonopus, Bambusa, Beckmannia, Blepharidachne, Blepharoneuron, Bothriochloa, Bouteloua, Brachiaria, Brachyelytrum, Brachypodium, Briza, Brizopyrum, Bromelica, Bromopsis, Bromus, Buchloe, Bulbilis, Calamagrostis, Calamovilfa, Campulosus, Capriola, Catabrosa, Catapodium, Cathestecum, Cenchropsis, Cenchrus, Centotheca, Ceratochloa, Chaetochloa, Chasmanthium, Chimouobambusa, ChionocMoa, Chloris, Chondrosum, Chrysopon, Chusquea, Cinna, Cladoraphis, Coelorachis, Coix, Coleanthus, Colpodium, Coridochloa, Comucopiae, Cortaderia, Corynephorus, Cottea, Critesion, Crypsis, Ctenium, Cutandia, Cylindropyrwn, Cymbopogon, Cynodon, Cynosurus, Cytrococcum, Dactylis, Dactyloctenium, Danthonia, Dasyochloa, Dasyprwn, Davyella, Dendrocalamus, Deschampsia, Desmazeria, Deyeuxia, Diarina, Diarrhena, Dichantheimm, Dichauthium, Dfchelachne, Diectomus, Digital la, Diiiieria, Dimorpostachys, Dinebra, Diplachne, Dissanthelium, Dissochondrus, Distichlis, Drepanostachyum, Dupoa, Dupontia, Echinochloa, Ectosperma, Ehrharta, Eleusine, Elyhordeum, Elyleymus, Elymordeum, Elymus, Elyonurus, Elysitanion, Elytesion, Elytrigia, Enneapogon, Enteropogon, Epicampes, Eragrostis, Eremochloa, Eremopoa, Eremopynun, Erianthus, Ericoma, Erichloa, Eriochrysis, Erioneuron, Euchlaena, Euclasta, Eulalia, Eulaliopsis, Eustachys, Fargesia, Festuca, FestuloUwn, Fingerhuthia, Fluminia, Garnotia, Gastridium, Gaudinia, Gigantochloa, Glyceria, Graphephorum, Gymnopogon, Gynerium, Hackelochloa, Hainardia, Hakonechloa, Haynaldia, Heleochloa, Helictotrichon, Hemarthria, Hesperochloa, Hesperostipa, Heteropogon, Hibanobambusa, Hierochloe,<br>
Hilaria, Holcus, Homalocenchrus, Hordewn (e.g., barley), Hydrochloa, Hymenachne, tfyparrhenia, Hypogynium, Hystrix, Ichnanthus, Imperata, Indocalamus, Isachne, Ischaemum, Ixophorus, Koeleria, Korycarpus, Lagunis, Lamarckia, Lasiacis, Leersia, Leptochloa, Leptochloopsis, Leptocoiyphiwn, Leptoloma, Leptogon, Leptums, Lerchenfeldia, Leucopoa, Leymostachys, Leymus, Limnodea, Lithachne, Loliuin, Lophochkiena, Lophochloa, Lophopynun, Ludolfia, Luziola, Lycunis, Lygeum, Maltea, Manisuris, Megastachya, Melica, Melinis, Mibora, Microchloa, Microlaena, Microstegium, Milium, Miscanthus, Mnesithea, Molinia, Monanthochloe, Monenna, Monroa, Muhlenbergia, Nardus, Nassella, Nazia, Neeragrostis, Neoschischkinia, Neostapfia, Neyraudia, Nothoholcus, Olyra, Opizia, Oplismenus, Orcuttia, Oryza (e.g., rice), Oiyzopsis, Otatea, Oxytenanthera, Panicularla, Panicum, fappophorum, Parapholis, Pascopyrwn, Paspalidium, Paspalum, Pennisetum (e.g., millet), Phalaris, Phalaroides, Phanopymm, Pharus, Phippsia, Phleum, Pholiunis, Phragmites, Phyllostachys, Piptatherum, Piptochaetium, Pleioblastus, Pleopogon, Pleuraphis, Pleuropogon, Poa, Podagrostis, Polypogon, Polytrias, Psathyrostachys, Pseudelymus, Pseudoroegneria, Pseitdosasa, Ptilagrostis, Puccindlia, Pucciphippsia, Redfieldia, Reimaria, Reimarochloa, Rhaphis, Rhombolytrum, Rhynchelytrum, Roegneria, Rostraria, Rottboellia, Rytilix, Sacchannn, Sacciolepis, Sasa, Sasaella, Sasamorpha, Savastana, Schedonnardur, Schismus, Schizachne, Schizcichyrium, Schizostachywn, Sclerocliloa, Scleropoa, Scleropogon, Scolochloa, Scribneria, Secale (e.g., rye), Semiamndinaria, Sesleria, Setaria, Shibataea, Sieglingia, Sinanmdinaria, Sinobambusa, Sinocalamus, Sitanion, Sorghastrmn, Sorghum, Spartina, Sphenopholis, Spodiopogon, Sporobolus, Stapfia, Steinchisma, Stenotaphrum, Stipa, Stipagrostis, Snporyzopsis, Swallenia, Synlherisma, Taeniatherum, Terrellia, Teirelymus, Thamnocalarnus, Themeda, Thiuopyrum, Thuarea, Thysanolaena, Torresia, Torreyochloa, Trachytiia, Tmchypogon, Tragus, Trichachne, Trichloris, Tricholaena, Trichoneura, Tridens, Triodia, Triplasis, Tripogon, Tripsacinn, Trisetobromus, Trisetum, Tnticosecale, Triticwn (e.g., wheat), Tuctona, Uniola, Urachne, Uralepis, Urochloa, Vahlodea, Valota, Vaseyochloa, Ventenata, Vetiveria, Vilfa, Vulpia, Willkommia, Yushania, Zea (e.g., corn), Zizania, Zizaniopsis, and Zoysia.<br>
PLANT TRANSFORMATION<br>
[0229]	Nucleic acid sequence constructs of the invention (e.g., isolated nucleic<br>
acids, recombinant expression cassettes, etc.) can be introduced into plant cells, either in<br>
"Uure or in the organs of plants, by a variety of conventional techniques. For example, techniques include, but are not limited to, infection, transduction, transfection, transvection and transformation. The nucleic acid sequence constructs can be introduced alone or with other polynucleotides. Such other polynucleotides can be introduced independently, co-introduced, or introduced joined to polynucleotides of the invention.<br>
[0230]	Techniques for transforming a wide variety of higher plant species are well<br>
known and described in the technical and scientific literature. See, e.g., Payne et al. (1992) Plant Cell and Tissue Culture in Liquid Systems John Wiley &amp; Sons, Inc. New York, NY (Payne); Gamborg and Phillips (eds) (1995) Plant Cell. Tissue and Organ Culture; Fundamental Methods Springer Lab Manual, Springer-Verlag (Berlin Heidelberg New York) (Gamborg); Croy, (ed.) (1993) Plant Molecular Biology Bios Scientific Publishers, Oxford, U.K; Jones (ed) (1995) Plant Gene Transfer and Expression Protocols— Methods in Molecular Biology. Volume 49 Humana Press Towata NJ, and as well as others, etc., as well as, e.g., Weising et al. (1988) Ann. Rev. Genet. 22:421. See, also, WO 95/06123 entitled "Fertile, Transgenic Maize Plants and Methods for Their Production" published on 2 March 1995. Numerous protocols for establishment of transformable protoplasts from a variety of plant types and subsequent transformation of the cultured protoplasts are available in the art and are incorporated herein by reference. For example, see, Hashimoto et al. (1990) Plant Phvsiol. 93:857; Fowke and Constabel (eds) (1994) Plant Protoplasts; Saunders et al. (1993) Applications of Plant In Vitro Technology Symposium. UPM 16-18; and Lyznik et al. (1991) BioTechniques 10:295, each of which is incorporated herein by reference. Numerous methods are available in the art to accomplish chloropiast transformation and expression (e.g., Daniell et al. (1998) Nature Biotechnology 16:346: O'Neill et al. (1993) The Plant Journal 3:729; Maliga (1993) TIBTECH 11:1).<br>
[0231]	For example, nucleic acid sequences can be introduced directly into the<br>
genomic DNA of a plant cell using techniques such as electroporation, PEG poration, particle bombardment, silicon fiber delivery, or microinjection of plant cell protoplasts or embryogenic callus, or the nucleic acid sequence constructs can be introduced directly to plant tissue using ballistic methods, such as particle bombardment. Exemplary particles include, but are not limited to, tungsten, gold, platinum, and the like. Alternatively, the nucleic acid sequence constructs can be introduced by infection of cells with viral vectors, or by combining the nucleic acid sequence constructs with suitable T-DNA flanking regions<br>
and introduced into a conventional Agrobacterium tionefaciens host vector. The virulence functions of the Agrobacterium host will direct the insertion of the construct and adjacent marker into the plant cell DNA when the plant cell is infected by the bacteria. See, U.S. Patent No. 5,591,616.<br>
[0232]	Microinjection techniques are known in the art and well described in the<br>
scientific and patent literature (see, e.g., Jones (ed) (1995) Plant Gene Transfer and Expression Protocols- Methods in Molecular Biology, Volume 49 Humana Press Towata NJ, and as well as others). The introduction of nucleic acid sequence constructs using polyethylene glycol precipitation is described in Paszkowski et al (1984) EMBQ J 3:2717. Electroporation techniques are described in Fromm et al. (1985) Proc Nat'l Acad Sci USA 82:5824. Ballistic transformation techniques are described in Klein et al. (1987) Nature 327:70; and Weeks et al. Plant Physiol 102:1077 and by Tomes, D. et a/., IN: Plant Cell, Tissue and Organ Culture: Fundamental Methods, Eds. O.L. Gamborg and G.C. Phillips, Chapter 8, pgs. 197-213 (1995). (See also Tomes et al, U.S. Patents 5,886,244; 6,258,999; 6,570,067; 5,879,918).<br>
[0233]	Viral vectors which are plant viruses can also be used to introduce<br>
polynucleotides of the invention into plants. Viruses are typically useful as vectors for expressing exogenous DNA sequences in a transient manner in plant hosts. In-contrast to agrobacterium mediated transformation which results in the stable integration of DNA sequences in the plant genome, viral vectors are generally replicated and expressed without the need for chromosomal integration. Plant virus vectors offer a number of advantages, specifically: a) DNA copies of viral genomes can be readily manipulated in E.coli, and transcribed in vitro, where necessary, to produce infectious RNA copies; b) naked DNA, RNA, or vims particles can be easily introduced into mechanically wounded leaves of intact plants; c) high copy numbers of viral genomes per cell results in high expression levels of introduced genes; d) common laboratory plant species as well as monocot and dicot crop species are readily infected by various virus strains; e) infection of whole plants permits repeated tissue sampling of single library clones; f) recovery and purification of recombinant virus particles is simple and rapid; and g) because replication occurs without chromosomal insertion, expression is not subject to position effects. See, e.g., Scholthof, Scholthof and Jackson, (1996) "Plant virus gene vectors for transient expression of foreign proteins in plants," Annu. Rev, of Phytopathol. 34:299-323.<br>
[0234]	Plant viruses cause a range of diseases, most commonly mottled damage to<br>
leaves, so-called mosaics. Other symptoms include necrosis, deformation, outgrowths, and generalized yellowing or reddening of leaves. Plant viruses are known which infect every major food-crop, as well as most species of horticultural interest. The host range varies between viruses, with some viruses infecting a broad host range (e.g., alfalfa mosaic virus infects more than 400 species in 50 plant families) while others have a narrow host range, sometimes limited to a single species (e.g. barley yellow mosaic virus). Appropriate vectors can be selected based on the host used in the methods and compositions of the invention.<br>
[0235]	In certain embodiments of the invention, a vector includes a plant virus, e.g.,<br>
either RNA (single or double stranded) or DNA (single-stranded or doubled-stranded) virus. Examples of such viruses include, but are not limited to, e.g., an alfamovirus, a bromovirus, a capillovirus, a carlavirus, a carmovirus, a caulimovirus, a closterovirus, a comovirus, a cryptovirus, a cucumovirus, a dianthovirus, a fabavirus, a fijivirus, a furovirus, a geminivirus, a hordeivirus, a ilarvirus, a luteovirus, a machlovirus, a maize chlorotic dwarf virus, a marafivirus, a necrovirus, a nepovirus, a parsnip yellow fleck virus, a pea enation mosaic virus, a potexvirus, a potyvirus, a reovirus, a rhabdovirus, a sobemovirus, a tenuivirus, a tobamovirus, a tobravirus, a tomato spotted wilt virus, a tombusvirus, a tymovirus, or the like.<br>
[0236]	Typically, plant viruses encode multiple proteins required for initial<br>
infection, replication and systemic spread, e.g. coat proteins, helper factors, replicases, and movement proteins. The nucleotide sequences encoding many of these proteins are matters of public knowledge, and accessible through any of a number of databases, e.g. (Genbank: available on the World Wide Web at ncbi.nlm.nih.gov/genbank/ or EMBL: available on the World Wide Web at ebi.ac.uk.embl/).<br>
[0237]	Methods for the transformation of plants and plant cells using sequences<br>
derived from plant viruses include the direct transformation techniques described above relating to DNA molecules, see e.g., Jones, ed. (1995) Plant Gene Transfer and Expression Protocols. Humana Press, Totowa, NJ. In addition, viral sequences can be cloned adjacent T-DNA border sequences and introduced via Agrobacterium mediated transformation or Agroinfection.<br>
[0238]	Viral particles comprising the plant virus vectors including polynucleotides<br>
of the invention can also be introduced by mechanical inoculation using techniques well known in the art; see, e.g., Cunningham and Porter, eds. (1997) Methods in Biotechnology, Vol.3. Recombinant Proteins from Plants: Production and Isolation of Clinically Useful Compounds, for detailed protocols. Briefly, for experimental purposes, young plant leaves are dusted with silicon carbide (carborundum), then inoculated with a solution of viral transcript or encapsidated virus and gently rubbed. Large scale adaptations for infecting crop plants are also well known in the art, and typically involve mechanical maceration of leaves using a mower or other mechanical implement, followed by localized spraying of viral suspensions, or spraying leaves with a buffered virus/carborundum suspension at high pressure. Any of these above mentioned techniques can be adapted to the vectors of the invention, and are useful for alternative applications depending on the choice of plant vims and host species, as well as the scale of the specific transformation application.<br>
[0239]	In some embodiments, Agrobacterium mediated transformation techniques<br>
are used to transfer the ACC synthase sequences or subsequences of the invention to transgenic plants. Agrobacteriurn-mediated transformation is widely used for the transformation of dicots; however, certain monocots can also be transformed by •Agrobacterium. For example, Agrobacterium transformation of rice is described by Hiei et al. (1994) Plant J. 6:271; US Patent No. 5,187,073; US Patent No. 5,591,616; Li et al. (1991) Science, in China 34:54; and Raineri et al. (1990) Bio/Technology 8:33. Transformed maize, barley, triticale and asparagus by Agrobacterium mediated transformation have also been described (Xu et al. (1990) Chinese J Bot 2:81).<br>
[0240]	Agrobacterium mediated transformation techniques take advantage of the<br>
ability of the tumor-inducing (Ti) plasmid of A. tumefaciens to integrate into a plant cell genome to co-transfer a nucleic acid of interest into a plant cell. Typically, an expression vector is produced wherein the nucleic acid of interest, such as an ACC synthase RNA configuration nucleic acid of the invention, is ligated into an autonomously replicating plasmid which also contains T-DNA sequences. T-DNA sequences typically flank the expression cassette nucleic acid of interest and comprise the integration sequences of the plasmid. In addition to the expression cassette, T-DNA also typically includes a marker sequence, e.g., antibiotic resistance genes. The plasmid with the T-DNA and the expression cassette are then transfected into Agrobacterium cells. Typically, for effective<br>
transformation of plant cells, the A. tumefaciens bacterium also possesses the necessary vir regions on a plasmid, or integrated into its chromosome. For a discussion of Agrobacterium mediated transformation, see, Firoozabady and Kuehnle, (1995) Plant Cell Tissue and Organ Culture Fundamental Methods. Gamborg and Phillips (eds.).<br>
[0241]	Agrobacterium tumefaciens-medhated transformation techniques are well<br>
described in the scientific literature. See, for example Horsch et al, Science 233: 496-498 (1984), and Fraley et al, Proc. Natl. Acad. Sci(USA) 80: 4803 (1983). Although Agrobacterium is useful primarily in dicots, certain monocots can be transformed by Agrobacterium. For instance, Agrobacterium transformation of maize is described in U.S. Patent No. 5,550,318.<br>
[0242]	Other methods of transfection or transformation include: (1) Agrobacterium<br>
rhizogenes-mediated transformation (see, e.g., Lichtenstein and Fuller In: Genetic Engineering, vol. 6, PWJ Rigby, Ed., London, Academic Press, 1987; and Lichtenstein, C. P., and Draper, J,. In: DNA Cloning, Vol. II, D. M. Glover, Ed., Oxford, IRI Press, 1985); Application PCT/US87/02512 (WO 88/02405 published Apr. 7,1988) describes the use of A.rhizogenes strain A4 and its Ri plasmid along with A. tumefaciens vectors pARCS or pARC16, (2) liposome-mediated DNA uptake (see, e.g., Freeman et al., Plant Cell Physiol. 25: 1353, 1984), and (3) the vortexing method (see, e.g.,&gt; Kindle, Proc. Nat'l. Acad. Sci.QJSA) 87: 1228, (1990).<br>
[0243]	DNA can also be introduced into plants by direct DNA transfer into pollen as<br>
described by Zhou et al., Methods in Enzvmologv. 101:433 (1983); D. Hess, Intern. Rev. Cytol., 107:367 (1987); Luo et al.. Plant Mol. Biol. Reporter, 6:165(1988). Expression of polypeptide coding genes can be obtained by injection of the DNA into reproductive organs of a plant as described by Pena et al., Nature 325:274 (1987). DNA can also be injected directly into the cells of immature embryos and the rehydration of desiccated embryos as described by Neuhaus et al., Theor. Appl. Genet.. 75:30 (1987); and Benbrook et al., in Proceedings Bio Expo. 1986, Butterworth, Stoneham, Mass., pp. 27-54 (1986). A variety of plant viruses that can be employed as vectors are known in the art and include cauliflower mosaic virus (CaMV), geminivirus, brome mosaic virus, and tobacco mosaic virus.<br>
[0244]	Other references describing suitable methods of transforming plant cells<br>
include microinjection, Crossway et al. (1986) Biotechniques 4:320-334; electroporation,<br>
Riggs et al. (1986) Proc. Natl. Acad. Sci. USA 83:5602-5606;<br>
*	transformation, see for example, Townsend et al. U.S. Patent 5,563,055; direct gene<br>
transfer, Paszkowski et al. (1984) EMBO J. 3:2717-2722; and ballistic particle acceleration,<br>
see for example, Sanford et al U.S. Patent 4,945,050; Tomes et al. (1995) in Plant Cell,<br>
Tissue, and Organ Culture: Fundamental Methods, ed. Gamborg and Phillips (Springer-<br>
-	Verlag, Berlin); and McCabe et al. (1988) Biotechnology 6:923-926. Also see Weissinger<br>
et al (1988) Annual Rev. Genet. 22:421-477; Sanford et al (1987) Paniculate Science and<br>
Technology 5:27-37 (onion); Christou etal. (1988) Plant Physiol 87:671-674 (soybean);<br>
McCabe et al (1988) Bio/Technology 6:923-926 (soybean); Datta et al (1990)<br>
Biotechnology 8:736-740 (rice); Klein et al (1988) Proc. Natl. Acad. Sci. USA 85:4305-<br>
4309 (maize); Klein et al (1988) Biotechnolog\»6:559-563 (maize); Klein et al (1988)<br>
Plant Physiol 91:440-444 (maize); Fromm et al (1990) Biotechnology 8:833-839;<br>
Hooydaas-Van Slogteren et al (1984) Nature (London) 311:763-764; Bytebier et al. (1987)<br>
Proc. Natl Acad. Sci. USA 84:5345-5349 (Liliaceae); De Wet et al (1985) in The<br>
Experimental Manipulation of Ovule Tissues, ed. G. P. Chapman et al (Longman, New<br>
York), pp. 197-209 (pollen); Kaeppler et al (1990) Plant Cell Reports 9:415-418; and<br>
Kaeppler et al (1992) Tlieor. Appl. Genet. 84:560-566 (whisker-mediated transformation);<br>
D.Halluin et al (1992) Plant Cell 4:1495-1505 (electroporation); Li et al. (1993.) Plant Cell<br>
Repons 12:250-255 and Christou et al (1995) Annals of Botany 75:407-413 (rice); Osjoda<br>
et al. (1996) Nature Biotechnology 14:745-750 (maize via Agrobacterium tumefaciens); all<br>
of which are herein incorporated by reference.<br>
Regeneration of Isolated, Recombinant orTrangenic Plants<br>
[0245]	Transformed plant cells which are derived by plant transformation<br>
techniques and isolated or recombinant plant cells, including those discussed above, can be cultured to regenerate a whole plant which possesses the desired genotype (i.e., a knockout ACC synthase nucleic acid), and/or thus the desired phenotype, e.g., staygreen phenotype, sterility phenotype, crowding resistant phenotype, etc. The desired cells, which can be identified, e.g., by selection or screening, are cultured in medium that supports regeneration. The cells can then be allowed to mature into plants. For example, such regeneration techniques can rely on manipulation of certain phytohormones in a tissue culture growth medium, typically relying on a biocide and/or herbicide marker which has been introduced together with the desired nucleotide sequences. Alternatively, screening can be performed<br>
:o screen for inhibition of expression and/or activity of ACC synthase, reduction in ethylene production conferred by the knockout ACC synthase nucleic acid sequence, etc. Plant regeneration from cultured protoplasts is described in Evans et al. (1983) Protoplasts Isolation and Culture. Handbook of Plant Cell Culture, pp 124-176, Macmillan Publishing Company, New York; Davey, (1983) Protoplasts, pp. 12-29, Birkhauser, Basal 1983; Dale, Protoplasts (1983) pp. 31-41, Birkhauser, Basel; and, Binding (1985) Regeneration of Plants. Plant Protoplasts pp 21-73, CRC Press, Boca Raton. Regeneration can also be obtained from plant callus, explants, organs, or parts thereof. Such regeneration techniques are described generally in Klee et al. (1987) Ann Rev of Plant Phys 38:467. See also, e.g., Payne and Gamborg. For transformation and regeneration of maize see, for example, U.S. Patent 5,736,369.<br>
[0246]	Plants cells transformed with a plant expression vector can be regenerated,<br>
e.g., from single cells, callus tissue or leaf discs according to standard plant tissue culture techniques. It is well known in the art that various cells, tissues, and organs from almost any plant can be successfully cultured to regenerate an entire plant. Plant regeneration from cultured protoplasts is described in Evans et al, Protoplasts Isolation and Culture, Handbook of Plant Cell Culture, Macmillilan Publishing Company, New York, pp. 124-176 (1983); and Binding, Regeneration of Plants, Plant Protoplasts, CRC Press, Boca Raton, pp. 21-73 (1985).<br>
[0247]	The regeneration of plants containing the foreign gene introduced by<br>
Agrobacterium from leaf explants can be achieved as described by Horsch et al., Science. 227:1229-1231 (1985). After transformation with Agrobacterium, the explants typically are transferred to selection medium. One of skill will realize that the selection medium depends on the selectable marker that is co-transfected into the explants. In this procedure, transformants are grown in the presence of a selection agent and in a medium that induces the regeneration of shoots in the plant species being transformed as described by Fraley et al, Proc. Nat'l. Acad. Sci. (U.S.A).. 80:4803 (1983). This procedure typically produces shoots, e.g., within two to four weeks, and these transformant shoots (which are typically, e.g., about 1-2 cm in length) are then transferred to an appropriate root-inducing medium containing the selective agent and an antibiotic to prevent bacterial growth. Selective pressure is typically maintained in the root and shoot medium.<br>
[0248]	Typically, the transformants will develop roots in about 1-2 weeks and form<br>
plantlets. After the plantlets are about 3-5 cm in height, they are placed in sterile soil in fiber pots. Those of skill in the art will realize that different acclimation procedures are used to obtain transformed plants of different species. For example, after developing a root and shoot, cuttings, as well as somatic embryos of transformed plants, are transferred to medium for establishment of plantlets. For a description of selection and regeneration of transformed plants, see, e.g., Dodds and Roberts (1995) Experiments in Plant Tissue Culture. 3rd Ed., Cambridge University Press. Transgenic plants of the present invention may be fertile or sterile.<br>
[0249]	Regeneration can also be obtained from plant callus, explants, organs, or<br>
parts thereof. Such regeneration techniques are described generally in Klee et al,, Ann. Rev, of Plant Phvs. 38: 467-486 (1987). The regeneration of plants from either single plant protoplasts or various explants is well known in the art. See, for example, Methods for Plant Molecular Biology, A. Weissbach and H. Weissbach, eds., Academic Press, Inc., San Diego, Calif. (1988). This regeneration and growth process includes the steps of selection of transformant cells and shoots, rooting the transformant shoots and growth of the plantlets in soil. For maize cell culture and regeneration see generally, The Maize Handbook, Freeling and Walbot, Eds., Springer, New York (1994); Com and Com Improvement., 3rd edition, Sprague and Dudley Eds., American Society of Agronomy, Madison, Wisconsin (198S).<br>
[0250]	One of skill will recognize that after the recombinant expression cassette is<br>
stably incorporated in transgenic plants and confirmed to be operable, it can be introduced into other plants by sexual crossing. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed.<br>
[0251]	In vegetatively propagated crops, mature transgenic plants can be propagated<br>
by the taking of cuttings or by tissue culture techniques to produce multiple identical plants. Selection of desirable transgenics is made and new varieties are obtained and propagated vegetatively for commercial use. In seed-propagated crops, mature transgenic plants can be self-crossed to produce a homozygous inbred plant. The inbred plant produces seed containing the newly introduced heterologous nucleic acid. These seeds can be grown to produce plants that would produce the selected phenotype. Mature transgenic plants can also be crossed with other appropriate plants, generally another inbred or hybrid, including, for example, an isogenic untransformed inbred.<br>
r0252]	Parts obtained from the regenerated plant, such as flowers, seeds, leaves,<br>
branches, fruit, and the like are included in the invention, provided that these parts comprise cells comprising the isolated nucleic acid of the present invention. Progeny and variants, and mutants of the regenerated plants are also included within the scope of the invention, provided that these plants comprise the introduced nucleic acid sequences.<br>
[0253]	Transgenic plants expressing the selectable marker can be screened for<br>
transmission of the nucleic acid of the present invention by, for example, standard immunoblot and DNA detection techniques. Transgenic lines are also typically evaluated on levels of expression of the heterologous nucleic acid. Expression at the RNA level can be determined initially to identify and quantitate expression-positive plants. Standard techniques for RNA analysis can be employed and include PCR amplification assays using oligonucleotide primers designed to amplify only the heterologous RNA templates and solution hybridization assays using heterologous nucleic acid-specific probes. The RNA-positive plants can then be analyzed for protein expression by Western immunoblot analysis using the specifically reactive antibodies of the present invention. In addition, in situ hybridization and immunocytochemistry according to standard protocols can be done using heterologous nucleic acid specific polynucleotide probes and antibodies, respectively, to localize sites of expression within transgenic tissue. Generally, a number of transgenic lines are usually screened for the incorporated nucleic acid to identify and select plants with the most appropriate expression profiles.<br>
[0254]	Some embodiments comprise a transgenic plant that is homozygous for the<br>
added heterologous nucleic acid; i.e., a transgenic plant that contains two added nucleic acid sequences, one gene at the same locus on each chromosome of a chromosome pair. A homozygous transgenic plant can be obtained by sexually mating (selfing) a heterozygous (aka hemizygous) transgenic plant that contains a single added heterologous nucleic acid, germinating some of the seed produced and analyzing the resulting plants produced for altered expression of a polynucleotide of the present invention relative to a control plant (i.e., native, non-transgenic). Back-crossing to a parental plant and out-crossing with a non-transgenic plant, or with a plant transgenic for the same or another trait or traits, are also contemplated.<br>
[0255]	It is also expected that the transformed plants will be used in traditional<br>
'Oreeding programs, including TOPCROSS pollination systems as disclosed in US 5,706,603 and US 5,704,160, the disclosure of each of which is incorporated herein by reference.<br>
[0256]	In addition to Berger, Ausubel and Sambrook, useful general references for<br>
plant cell cloning, culture and regeneration include Jones (ed) (1995) Plant Gene Transfer and Expression Protocols— Methods in Molecular Biology, Volume 49 Humana Press Towata NJ; Payne et al. (1992) Plant Cell and Tissue Culture in Liquid Systems John Wiley &amp; Sons, Inc. New York, NY (Payne); and Gamborg and Phillips (eds) (1995) Plant Cell, Tissue and Organ Culture; Fundamental Methods Springer Lab Manual, Springer-Verlag (Berlin Heidelberg New York) (Gamborg). A variety of cell culture media are described in Atlas and Parks (eds) The Handbook of Microbiological MediaJ1993) CRC Press, Boca Raton, FL (Atlas). Additional information for plant cell culture is found in available commercial literature such as the Life Science Research Cell Culture Catalogue (1998) from Sigma- Aldrich, Inc (St Louis, MO) (Sigma-LSRCCC) and, e.g., the Plant Culture Catalogue and supplement (1997) also from Sigma-Aldrich, Inc (St Louis, MO) (Sigma-PCCS). Additional details regarding plant cell culture are found in Croy, (ed.) (1993) Plant Molecular Biology Bios Scientific Publishers, Oxford, U.K.<br>
"STACKING" OF CONSTRUCTS AND TRAITS<br>
[0257]	In certain embodiments, the nucleic acid sequences of the present invention<br>
can be used in combination ("stacked") with other polynucleotide sequences of interest in order to create plants with a desired phenotype. The polynucleotides of the present invention may be stacked with any gene or combination of genes, and the combinations generated can include multiple copies of any one or more of the polynucleotides of interest. The desired combination may affect one or more traits; that is, certain combinations may be created for modulation of gene expression affecting ACC synthase activity and/or ethylene production. Other combinations may be designed to produce plants with a variety of desired traits, including but not limited to traits desirable for animal feed such as high oil genes (e.g., U.S. Patent No. 6,232,529); balanced amino acids (e.g. hordothionins (U.S. Patent Nos. 5,990,389; 5,885,801; 5,885,802; and 5,703,409); barley high lysine (Williamson et al. (19S7) Eur. J. Biochem. 165:99-106; and WO 98/20122); and high methionine proteins (Pedersen et al. (1986) J. Biol. Chem. 261:6279; Kirihara et al. (1988)<br>
&amp;ene 71:359; and Musumura et al. (1989) Plant Mol. Biol 12:123)); increased digestibility (e.g., modified storage proteins (U.S. Application Serial No. 10/053,410, filed November 7, 2001); and thioredoxins (U.S. Application Serial No. 10/005,429, filed December 3, 2001)), the disclosures of which are herein incorporated by reference. The polynucleotides of the present invention can also be stacked with traits desirable for insect, disease or herbicide resistance (e.g., Bacillus thuringiensis toxic proteins (U.S. Patent Nos. 5,366,892; 5,747,450; 5,737,514; 5723,756; 5,593,881; Geiseret al (1986) Gene 48:109); lectins (Van Damme et al. (1994) Plant Mol. Biol. 24:825); fumonisin detoxification genes (U.S. Patent No. 5,792,931); avirulence and disease resistance genes (Jones et al. (1994) Science 266:789; Martin et al. (1993) Science 262:1432; Mindrinos et al. (1994) Cell 78:1089); acetolactate synthase (ALS) mutants that lead to herbicide resistance such as the S4 and/or Hra mutations; inhibitors of glutamine synthase such as phosphinothricin or basta (e.g., bar gene); and glyphosate resistance (EPSPS gene)); and traits desirable for processing or process products such as high oil (e.g., U.S. Patent No. 6,232,529); modified oils (e.g., fatty acid desaturase genes (U.S. Patent No. 5,952,544; WO 94/11516)); modified starches (e.g., ADPG pyrophosphorylases (AGPase), starch synthases (SS), starch branching enzymes (SEE) and starch debranching enzymes (SDBE)); and polymers or bioplastics (e.g., U.S. patent No. 5.602,321; beta-ketothiolase, polyhydroxybutyrate synthase, and acetoacetyl-CoA reductase (Schubert et al. (1988) J. Bacterial. 170:5837-5847) facilitate expression of polyhydroxyalkanoates (PHAs)), the disclosures of which are herein incorporated by reference. One could also combine the polynucleotides of the present invention with polynucleotides affecting agronomic traits such as male sterility (e.g., see U.S. Patent No. 5.583,210), stalk strength, flowering thiic, or transformation technology traits such as cell cycle regulation or gene targeting (e.g. WO 99/61619; WO 00/17364; WO 99/25821), the disclosures of which are herein incorporated by reference.<br>
[0258]	These stacked combinations can be created by any method, including but not<br>
limited to cross breeding plants by any conventional or TopCross methodology, or genetic transformation. If the traits are stacked by genetically transforming the plants, the polynucleotide sequences of interest can be combined at any time and in any order. For example, a transgenic plant comprising one or more desired traits can be used as the target to introduce further traits by subsequent transformation. The traits can be introduced simultaneously in a co-transformation protocol with the polynucleotides of interest provided<br>
by any combination of transformation cassettes. For example, if two sequences will be introduced, the two sequences can be contained in separate transformation cassettes (trans) or contained on the same transformation cassette (cis). Expression of the sequences of interest can be driven by the same promoter or by different promoters. In certain cases, it may be desirable to introduce a transformation cassette that will suppress the expression of a polynucleotide of interest. This may be accompanied by any combination of other suppression cassettes or over-expression cassettes to generate the desired combination of traits in the plant.<br>
USE IN BREEDING METHODS<br>
[0259]	The transformed plants of the invention may be used in a plant breeding<br>
program. The goal of plant breeding is to combine, in a single variety or hybrid, various desirable traits. For field crops, these traits may include, for example, resistance to diseases and insects, tolerance to heat and drought, reduced time to crop maturity, greater yield, and better agronomic quality. With mechanical harvesting of many crops, uniformity of plant characteristics such as germination and stand establishment, growth rate, maturity, and plant and ear height is desirable. Traditional plant breeding is an important tool in developing new and improved commercial crops. This invention encornpasses-methods for producing a maize plant by crossing a first parent maize plant with a second parent maize plant wherein one or both of the parent maize plants is a transformed plant displaying a staygreen phenotype, a sterility phenotype, a crowding resistance phenotype, or the like, as described herein.<br>
[0260]	Plant breeding techniques known in the art and used in a maize plant<br>
breeding program include, but are not limited to, recurrent selection, bulk selection, mass selection, backcrossing, pedigree breeding, open pollination breeding, restriction fragment length polymorphism enhanced selection, genetic marker enhanced selection, doubled haploids, and transformation. Often combinations of these techniques are used.<br>
[0261]	The development of maize hybrids in a maize plant breeding program<br>
requires, in general, the development of homozygous inbred lines, the crossing of these lines, and the evaluation of the crosses. There are many analytical methods available to evaluate the result of a cross. The oldest and most traditional method of analysis is the observation of phenotypic traits. Alternatively, the genotype of a plant can be examined.<br>
~J)262]	A genetic trait which has been engineered into a particular maize plant using<br>
transformation techniques can be moved into another line using traditional breeding techniques that are well known in the plant breeding arts. For example, a backcrossing approach is commonly used to move a transgene from a transformed maize plant to an elite inbred line, and the resulting progeny would then comprise the transgene(s). Also, if an inbred line was used for the transformation, then the transgenic plants could be crossed to a different inbred in order to produce a transgenic hybrid maize plant. As used herein, "crossing" can refer to a simple X by Y cross, or the process of backcrossing, depending on the context.<br>
[0263]	The development of a maize hybrid in a maize plant breeding program<br>
involves three steps: (1) the selection of plants from various germplasm pools for initial breeding crosses; (2) the selfing of the selected plants from the breeding crosses for several generations to produce a series of inbred lines, which, while different from each other, breed true and are highly uniform; and (3) crossing the selected inbred lines with different inbred lines to produce the hybrids. During the inbreeding process in maize, the vigor of the lines decreases. Vigor is restored when two different inbred lines are crossed to produce the hybrid. An important consequence of the homozygosity and homogeneity of the inbred lines is that the hybrid created by crossing a defined pair of inbreds will always be the same. Once the inbreds that give a superior hybrid have been identified, the hybrid seed can be reproduced indefinitely as long as the homogeneity of the inbred parents is maintained.<br>
[0264]	Transgenic plants of the present invention may be used to produce, e.g., a<br>
single cross hybrid, a three-way hybrid or a double cross hybrid. A single cross hybrid is produced when two inbred lines are crossed to produce the Fl progeny. A double cross hybrid is produced from four inbred lines crossed in pairs (A x B and C x D) and then the two Fl hybrids are crossed again (A x B) x (C x D). A three-way cross hybrid is produced from three inbred lines where two of the inbred lines are crossed (A x B) and then the resulting Fl hybrid is crossed with the third inbred (A x B) x C.   Much of the hybrid vigor and uniformity exhibited by Fl hybrids is lost in the next generation (F2). Consequently, seed produced by hybrids is consumed rather than planted.<br>
ANTIBODIES<br>
L0265]	The polypeptides of the invention can be used to produce antibodies specific<br>
for the polypeptides of SEQ ID NO:7-SEQ ID NO:9 and SEQ ID NO.: 11, and conservative variants thereof. Antibodies specific for, e.g., SEQ ID NOs: 7-9 and 11, and related variant polypeptides are useful, e.g., for screening and identification purposes, e.g., related to the activity, distribution, and expression of ACC synthase.<br>
[0266]	Antibodies specific for the polypeptides of the invention can be generated by<br>
methods well known in the art. Such antibodies can include, but are not limited to, polyclonal, monoclonal, chimeric, humanized, single chain, Fab fragments and fragments produced by an Fab expression library.<br>
[0267]	Polypeptides do not require biological activity for antibody production. The<br>
full length polypeptide, subsequences, fragments or oligopeptides can be antigenic. Peptides used to induce specific antibodies typically have an amino acid sequence of at least about 10 amino acids, and often at least 15 or 20 amino acids. Short stretches of a polypeptide, e.g., selected from among SEQ ID NO:7-SEQ ID NO:9 and SEQ ED NO: 11, can be fused with another protein, such as keyhole limpet hemocyanin, and antibody produced against the chimeric molecule.<br>
[0268]	Numerous methods for producing polyclonal and monoclonal antibodies are<br>
known to those of skill in the art and can be adapted to produce antibodies specific for the polypeptides of the invention, e.g., corresponding to SEQ ID NO:7-SEQ ID NO:9 and SEQ ID NO: 11. See, e.g., Coligan (1991) Current Protocols in Immunology Wiley/Greene, NY; and Harlow and Lane (1989) Antibodies; A Laboratory Manual Okl &gt;Spnng Harbor Press, NY; Stites et al. (eds.) Basic and Clinical Immunology (4th ed.) Lange Medical Publications, Los Altos, CA, and references cited therein; Coding (1986) Monoclonal Antibodies: Principles and Practice (2d ed.) Academic Press, New York, NY; Fundamental Immunology, e.g., 4th Edition (or later),W.E. Paul (ed.), Raven Press, N.Y. (199S); and Kohler and Milstein (1975) Nature 256: 495-497. Other suitable techniques for antibody preparation include selection of libraries of recombinant antibodies in phage or similar vectors. See, Huse et al. (1989) Science 246: 1275-1281; and Ward, et al. (1989) Nature 341: 544-546. Specific monoclonal and polyclonal antibodies and antisera will usually bind<br>
vith a KD of at least about 0.1 µM, preferably at least about 0.01µM or better, and most typically and preferably, 0.001 uM or better.<br>
KITS FOR MODULATING STAYGREEN POTENTIAL OR STERILITY<br>
[0269]	Certain embodiments of the invention can optionally be provided to a user as<br>
- a kit. For example, a kit of the invention can contain one or more nucleic acid, polypeptide, antibody, diagnostic nucleic acid or polypeptide, e.g., antibody, probe set, e.g., as a cDNA microarray, one or more vector and/or cell line described herein. Most often, the kit is packaged in a suitable container. The kit typically further comprises one or more additional reagents, e.g., substrates, labels, primers, or the like for labeling expression products, tubes and/or other accessories, reagents for collecting-samples, buffers, hybridization chambers, cover slips, etc. The kit optionally further comprises an instruction set or user manual detailing preferred methods of using the kit components for discovery or application of gene sets. When used according to the instructions, the kit can be used, e.g., for evaluating expression or polymorphisms in a plant sample, e.g., for evaluating ACC synthase, ethylene production, staygreen potential, crowding resistance potential, sterility, etc. Alternatively, the kit can be used according to instructions for using at least one ACC synthase polynucleotide sequence to control staygreen potential in a plant.<br>
[0270]	As another example, a kit for modulating sterility, e.g., male sterility, in a<br>
plant includes a container containing at least one polynucleotide sequence comprising a nucleic acid sequence, wherein the nucleic acid sequence is. e.g., at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 99%; about 99.5% or more, identical to SEQ ID NO:i (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ED NO:5 (cACS6), SEQ ID NO:6 (cAC7) or SEQ ID NO: 10 (CCRA178R), or a subsequence thereof, or a complement thereof. The kit optionally also includes instructional materials for the use of the at least one polynucleotide sequence to control sterility, e.g., male sterility, in a plant.<br>
OTHER NUCLEIC ACID AND PROTEIN ASSAYS<br>
[0271]	In the context of the invention, nucleic acids and/or proteins are manipulated<br>
according to well known molecular biology methods. Detailed protocols for numerous such procedures are described in, e.g., in Ausubel et al. Current Protocols in Molecular Biology<br>
(supplemented through 2004) John Wiley &amp; Sons, New York ("Ausubel"); Sambrook et al. Molecular Cloning - A Laboratory Manual (2nd Ed.), Vol. 1-3, Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989 ("Sambrook"), and Berger and Kimmel Guide to Molecular Cloning Techniques, Methods in Enzymology volume 152 Academic Press, Inc., San Diego, CA ("Berger").<br>
[0272J	In addition to the above references, protocols for in vitro amplification<br>
techniques, such as the polymerase chain reaction (PCR), the ligase chain reaction (LCR), Q(3-replicase amplification, and other RNA polymerase mediated techniques (e.g., NASBA), useful e.g., for amplifying polynucleotides of the invention, are found in Mullis et al. (1987) U.S. Patent No. 4,683,202; PCR Protocols A Guide to Methods and Applications (Innis et al. eds) Academic Press Inc. San Diego, CA (1990) ("Innis"); Arnheim and Levinson (1990) C&amp;EN 36; The Journal Of NIH Research (1991) 3:81; Kwoh et al. (1989) Proc Natl Acad Sci USA 86.1173; Guatelli et al. (1990) Proc Natl Acad Sci USA 87:1874; Lomell et al. (1989) J Clin Chem 35:1826; Landegren et al. (1988) Science 241:1077; Van Brunt (1990) Biotechnology 8:291; Wu and Wallace (1989) Gene 4: 560; Barringer et al. (1990) Gene 89:117, and Sooknanan and Malek (1995) Biotechnology 13:563. Additional methods, useful for cloning nucleic acids in the context of the invention, include Wallace et al. U.S. Pat. No. 5,426,039. Improved methods of amplifying large nucleic acids by PCR are summarized in Cheng et al. (1994) Nature 369:684 and the references therein.<br>
[0273]	Certain polynucleotides of the invention can be synthesized utilizing various<br>
solid-phase strategies involving mononucleotide- and/or trinucleotide-based phosphoramidite coupling chemistry. For example, nucleic acid sequences can be<br>
synthesized by the sequential addition of activated monomers and/or trimers to an	<br>
elongating polynucleotide chain. See e.g., Caruthers, M.H. et al. (1992) Meth Enzymol 211:3. In lieu of synthesizing the desired sequences, essentially any nucleic acid can be custom ordered from any of a variety of commercial sources, such as The Midland Certified Reagent Company (<a href="../cdn-cgi/l/email-protection.html" class="__cf_email__" data-cfemail="6508061706250a090c020a164b060a08">[email&#160;protected]</a>) (Midland, Texas), The Great American Gene Company (available on the World Wide Web at genco.com) (Rarnona, CA), ExpressGen, Inc. (available on the World Wide Web at expressgen.com) (Chicago EL), Operon Technologies, Inc. (available on the World Wide Web at operon.com) (Alameda CA), and many others.<br>
EXAMPLES<br>
The following examples are offered to illustrate, but not to limit the claimed invention.<br>
EXAMPLE 1; ISOLATION OF MAIZE ACC SYNTHASE KNOCKOUTS<br>
[0274]	Because ethylene has been associated with promoting leaf senescence in<br>
some species, to introduce staygreen potential into, e.g., maize, we undertook to reduce ethylene biosynthesis in maize leaves through the inactivation of ACC synthase genes. The maize ACC synthase gene family is composed of three members: ACS2, ACS6, and ACS7. In order to isolate ethylene mutants, we screened for disruptions of each member of the ACC synthase gene family using the Trait Utility System for Com (TUSC). To date, we have determined the exact Mu insertion site for 8 mutant lines (three ACS6 and five ACS2) by sequencing across the Mu/ACC synthase junction. Five insertions were stably inherited; their positions are indicated in Figure 2, which also schematically depicts ACS7.<br>
[0275]	A pronounced staygreen phenotype was observed in leaves of those plants in<br>
which a single gene member of the ACS family was present in a heterozygous mutant state (see Figure 3, Panels A, B, C, and D). When present in a homozygous mutant state, an even more pronounced staygreen phenotype was observed (see Figure 4). In Figure 4, a leaf from the \vild-type (left), heterozygous ACC synthase knockout (middle), and homozygous ACC synthase knockout (right) was sheathed for seven (7) days, in the dark. Leaves from homozygous ACC synthase knockout plants exhibited a greater staygreen trait than leaves of the heterozygous ACC synthase knockout and exhibited a substantially greater staygreen trait than leaves of wild type plants.<br>
[0276]	The degree of staygreen potential introduced was gene member specific.<br>
Consequently, a strong staygreen trait was introduced with the mutation of one member (e.g., ACS6), while a less pronounced staygreen phenotype was introduced with the mutation of another member (e.g., ACS2). Therefore, the degree of staygreen potential introduced into a line can be controlled by which mutant gene member is introduced, whether the mutant gene member is present in a heterozygous or homozygous state, and by the number of members of this family which are inactivated (e.g., ACS2/ACS6 double mutants have a strong staygreen phenotype). Traits associated with improved hybrid standability include resistance to stalk rot and leaf blights, genetic stalk strength, short plant height and ear placement, and high staygreen potential.<br>
[0277]	Typically, leaves follow a typical progression from initiation through<br>
expansion ultimately ending in senescence. The carbon fixation capacity also increases during expansion and ultimately declines to low levels throughout senescence. See, e.g., Gay AP, and Thomas H (1995) Leaf development in Lolium temulentum: photosynthesis in relation to growth and senescence. New Phytologist 130: 159-168. This is of particular relevance to cereal species where yield potential is largely dependent upon the ability of the plant to fix carbon and store this carbon in the seed, mainly in the form of starch. Both the timing at which senescence is initiated and the rate at which it progresses can have a significant impact on the overall carbon a particular leaf can ultimately contribute to a plant. See, e.g., Thomas H, and Howarth CJ (2000) Five ways to stay green. Journal of Experimental Botany 51: 329-337. This is of particular relevance to those crops where yield potential is reduced by adverse environmental conditions that induce premature leaf senescence. Stay-green is a general term used to describe a phenotype whereby leaf senescence (most easily distinguished by yellowing of the leaf associated with chlorophyll ' degradation) is delayed compared to a standard reference. See, e.g., Thomas and Howarth, supra. In sorghum, several stay-green genotypes have been identified which exhibit a delay in leaf senescence during grain filling and maturation. See, e.g., Duncan RR, et al. (1981) Descriptive comparison of senescent and non-senescent sorghum genotypes. Agronomy Journal 73: 849-853. Moreover, under conditions of limited water availability, which normally hastens leaf senescence (e.g., Rosenow DT, and Clark LE (1981) Drought tolerance in sorghum. In: Loden HD, Wilkinson D, eds. Proceedings of the 36th annual corn and sorghum industry research conference, 18-31). these genotypes retain more green leaf area and continue to fill grain normally (e.g., McBee GG, et al. (1983) Effect of senescence and non-senescence on carbohydrates in sorghum during late kernel maturity states. Crop Science 23: 372-377; Rosenow DT, et al. (1983) Drought-tolerant sorghum and cotton gennplasm. Agricultural Water Management 7: 207-222; and, Borrell AK, Douglas ACL (1996) Maintaining green leaf area in grain sorghum increases yield in a water-limited environment. In: Foale MA, Henzell RG, Kneipp JF, eds. Proceedings of the third Australian sorghum conference. Melbourne: Australian Institute of Agricultural Science. Occasional Publication No. 93). The stay-green phenotype has also been used as a selection criterion for the development of improved varieties of corn, particularly with regard to the development of drought-tolerance. See, e.g., Russell WA (1991) Genetic improvement of maize yields. Advances in Agronomy 46: 245-298; and, Bruce et al.,<br>
2), Molecular and physiological approaches to maize improvement far drought tolerance. Journal of Experimental Botany. 53 (366): 13-25.<br>
[0278]	Five fundamentally distinct types of stay-green have been described. See,<br>
e.g., Thomas H, and Smart CM (1993) Crops that stay green. Annals of Applied Biology 123: 193-219; and, Thomas and Howarth, supra. In Type A stay-green, initiation of the senescence program is delayed, but then proceeds at a normal rate. In Type B stay-green, while initiation of the senescence program is unchanged, the progression is comparatively slower. In Type C stay-green, chlorophyll is retained even though senescence (as determined through measurements of physiological function such as photosynthetic capacity) proceeds at a normal rate. Type D stay-green is more artificial in that killing of the leaf (i.e. by freezing, boiling or drying) prevents initiation of the senescence program thereby stopping the degradation of chlorophyll. In Type E stay-green, initial levels of chlorophyll are higher while initiation and progression of leaf senescence are unchanged, thereby giving the illusion of a relatively slower progression rate. Type A and B are functional stay-greens as photosynthetic capacity is maintained along with chlorophyll content and are the types associated with increased yield and drought tolerance in sorghum. Despite the potential importance of this trait, in particular the benefits associated with increasing yield and drought tolerance, very little progress has been made in understanding the biochemical, physiological or molecular basis for genetically-determined stay-green. See, e.g., Thomas and Howarth, supra.<br>
[0279]	A number of environmental and physiological conditions have been shown<br>
to significantly alter the timing and progression of leaf senescence and can provide some insight into the basis for this trait. Among environmental factors, light is probably the most significant and it has long been established that leaf senescence can be induced in many plant species by placing detached leaves in darkness. See, e.g., Weaver LM, Amasino RM (2001) Senescence is induced in individually darkened Arabidopsis leaves, but inhibited in whole darkened plants. Plant Physiology 127: 876-886. Limited nutrient and water availability have also been shown to induce leaf senescence prematurely (e.g., Rosenow DT, Quisenberry JE, Wendt CW, Clark LE (1983) Drought-tolerant sorghum and cotton gennplasin. Agricultural Water Management 7: 207-222). Among physiological determinants, growth regulators play a key role in directing the leaf senescence program. Of particular relevance is the observation that modification of cytokinin levels can<br>
significantly delay leaf senescence. For example, plants transformed with isopentenyl<br>
transferase (ipt), an Agrobactenum gene encoding a rate-limiting step in cytokinin<br>
biosynthesis, when placed under the control of a senescence inducible promoter, resulted in<br>
autoregulated cytokinin production and a strong stay-green phenotype. See, e.g., Gan S,<br>
Amasino RM (1995) Inhibition of leaf senescence by autoregulated production of cytokinin.<br>
Science 270: 1986-1988. Ethylene has also been implicated in controlling leaf senescence<br>
(e.g., Davis KM, and Grierson D (1989) Identification ofcDNA clones for tomato<br>
(Lycopersicon esculentum Mill.) mRNAs that accumulate during fruit ripening and leaf<br>
senescence in response to ethylene. Planta 179: 73-80) and plants impaired in ethylene<br>
production or perception also show a delay in leaf senescence (e.g., Picton S, et al., (1993)<br>
Altered fruit ripening and leaf senescence in tomatoes expressing an antisense ethylene-<br>
forming enzyme transgene. The Plant Journal 3: 469-481; Grbic V, and Bleeker AB (1995)<br>
Ethylene regulates the timing of leaf senescence inArabidopsis. The Plant Journal S: 95-<br>
102; and, John I, et al., (1995) Delayed leaf senescence in ethylene-deficient ACC-oxidase<br>
antisense tomato plants: molecular and physiological analysis. The Plant Journal 7:483-<br>
490), which can be phenocopied by exogenous application of inhibitors of ethylene<br>
biosynthesis and action (e.g., Abeles FE, et al., (1992) Ethylene in Plant Biology.<br>
Academic Press, San Diego, CA).	.<br>
[0280]	The identification and analysis of mutants in Arabidopsis and tomato that are<br>
deficient in ethylene biosynthesis and perception are valuable in establishing the important role that ethylene plays in plant growth and development. Mutant analysis has also been instrumental in identifying and characterizing the ethylene signal transduction pathway. While many ethylene mutants have been identified in :iicot plants (e.g., Arabidcp-is and tomato), no such mutants have been identified in monocots (e.g., rice, wheat, and corn). Here the identification of maize mutants deficient in ACC synthase, the first enzyme in the ethylene biosynthetic pathway, are described. These mutants are critical in elucidating the regulatory roles that ethylene plays throughout cereal development as well as its role in regulating responses to environmental stress. Knowledge obtained from such mutant analysis will increase the understanding of the role of ethylene in maize development and will be pertinent to other cereal crop species.<br>
[0281]	Mutants were deficient in ethylene production and exhibited a staygreen<br>
phenotype. Staygreen was observed under normal growth conditions and following<br>
prolonged conditions of drought that induced premature onset of leaf senescence in wild-type plants. In addition to the maintenance of chlorophyll during water stress, ACC synthase-deficient leaves maintained photosynthetic function and continued to assimilate CO2. Surprisingly, reducing ethylene production improved leaf function in all leaves under normal growth conditions and maintained a high level of function in drought-stressed plants even for those leaves in which senescence had not been induced in similar age leaves of wild-type plants. These findings indicate that ethylene may serve to regulate leaf function under normal growth conditions as well as in response to conditions of drought.<br>
MATERIALS AND METHODS<br>
Cloning of ACC svnthase genes from Zea mays<br>
[0282]	To facilitate cloning of ACC synthase gene(s) from maize, primers were<br>
designed to regions highly conserved between multiple monocot and dicot species using sequence information currently available in GenBank. Initial PCR reactions were carried out on maize genomic DNA using primers ACCF1 (ccagatgggcctcgccgagaac; SEQ ID NO: 12) and ACC1 (gttggcgtagcagacgcggaacca; SEQ ID NO: 13) and revealed the presence of three fragments of different sizes. All three fragments were sequenced and confirmed to be highly similar in sequence to other known ACC synthase genes.<br>
[0283]	To obtain the entire genomic sequences for each of these genes, all three<br>
fragments were radiolabeled with dCTP using the Prime-a-Gene labeling system (Promega) and used to screen an EMBL3 maize (B73) genomic library (Stratagene) according to methods described in Sambrook, supra. Hybridization was carried out overnight at 30°C in buffer containing 5X SSPE, 5XDenhardt s, 50% Fcxinamide and 1% SDS. Blots were washed sequentially at 45°C in IX SSPE and 0.1X SSPE containing 0.1% SDS and exposed to film at -80°C with an intensifier screen. A total of 36 confluent plates (150 mm diameter) were screened. Putative positives plaques were subsequently screened directly by PCR using the above primers to identify which clones contained fragments corresponding to the three fragments initially identified. PCR screening was accomplished using HotStarTaq (Qiagen). Reactions contained IX buffer, 200(iM of each dNTP, 3uM MgCl2,0.25uJVI forward and reverse primer, 1.25U HotStarTaq and lµ1 primary phage dilution (1/600 total in SM buffer) as a template in a total reaction volume of 25ui. Reaction conditions were as follows: 95°C/15 min. (1 cycle); 95°C/1 min, 62°C/1 min, 72°C/2 min (35 cycles); 72°C/5<br>
nin (1 cycle). Samples were separated on a 1% agarose gel and the products were visualized following staining with ethidium bromide. All fragments amplified were also<br>
subjected to restriction analysis to identify other potential sequence specific differences<br>
independent of subsequence size.<br>
[0284]	To facilitate sequencing the remaining portions of these genes, primers<br>
ACCF1 and ACC1 were used in conjunction with primers specific to either the left (gacaaactgcgcaactcgtgaaaggt; SEQ ID NO: 14) or right (ctcgtccgagaataacgagtggatct; SEQ ED NO: 15) arm of the EMBL3 vector to amplify each half of the gene. Takara LA Taq (Panvera) was used to amplify the fragments due to the large size. Reactions contained lul phage dilution (1/600 total in SM buffer) and 2 ^M each primer (final concentration), IX buffer (final concentration), 400µM dNTP mix (final concentration) and 1.25 U LA Taq in 25 ul total volume. Reactions were carried out under the following conditions: 98°C/1 min. (1 cycle); 98°C/30 sec, 69°C/15 min. (35 cycles); 72°C/10 min (1 cycle). Amplified products were purified using the StrataPrep PCR purification'kit (Stratagene) and sent to the sequencing facility at the University of Florida, Gainesville for direct sequencing.<br>
Identification of ACC synthase knock out mutants<br>
[0285]	Maize has proven to be a rich source of mutants, in .part due to the presence<br>
of active or previously active transposable element systems within-its genome. Depending precisely on the location of the insertion site in a gene, a transposon can partially or completely inactivate expression of a gene. Gene inactivation may or may not have an observable phenotype depending upon the amount of redundancy (i.e. presence of multiple family members and the tissue specificity of the family members). Trait Utility System for . Corn (TUSC), developed by Pioneer Hybrid Int., is a powerful PCR-based screening strategy to identify MM transposon insertions in specific genes without the need for an observable phenotype. This screening approach is best suited to target genes that have been previously isolated from maize. The system utilizes TIR-PCR in which one PCR primer is derived from the target gene and the other (Mu-TIR) from the terminal-inverted-repeat (TIR) region of MM. Using these primers in PCR reactions of DNA pooled from a large population of MM containing plants, successful amplification is identified by Southern hybridization using the target gene as the probe. Screening the individuals within a positive pool is then performed to identify the candidate line containing insertion of a MM element in the target gene. In order to determine whether an insertion event is limited to somatic cells<br>
*®r is present in the germ line (and therefore represents a heritable change), progeny from a candidate are subjected to the same PCR/Southern hybridization analysis used in the original screen.<br>
[0286]	A research effort was established to identify knockout mutants in ethylene<br>
biosynthesis. To accomplish this, four primers (ACCF1, ccagatgggcctcgccgagaac, SEQ ID NO: 12; ACC-1, gttggcgtagcagacgcggaacca, SEQ ID NO:13; ACC-C, cagttatgtgagggcacaccctacagcca, SEQ ID NO: 16; ACC-D, catcgaatgccacagctcgaacaacttc, SEQ ID NO: 17) specific to the maize ACC synthase genes discussed above were used to screen for Mu insertions in combination with the Mu-TIR primer<br>
(aagccaacgcca(a/t)cgcctc(c/t)atttcgt; SEQ ID NO:IS). The initial screening resulted in the putative identification of 19 separate lines carrying Mu insertions in the maize ACC synthase multigene family. Seed from each of these lines was planted and DNA was extracted from the leaf of each individual. For DNA isolation, 1cm2 of seedling leaf was isolated from each plant and placed into a 1.5ml centrifuge tube containing some sand. Samples were quick-frozen in liquid nitrogen and ground to a fine powder using a disposable pestle (Fisher Scientific). 600µl of extraction buffer (lOOmM Tris (pH 8.0), 50mM EDTA, 20QmM NaCI, 1% SDS, lOµJ/ml p-mercaptoethanol) was added immediately and mixed thoroughly. 700f.il Phenol/Chloroform (1:1) was added and samples were centrifuged 10 min at 12,000 µm. 5OOµl supernatant was removed to a new tube and the nucleic acid precipitated at-20°C following addition of 1/10 vol 3M sodium acetate and 1 vol isopropanol. Total nucleic acid was pelleted by centrifugation at 12,000 µm, washed 3X with 75% ethanol and resuspended in 600µ,l H2O. PCR screening was accomplished using HotStarTaq (Qiagen). Reactions contained IX buffer, 200nM of each dNTP, 3mM MgCl2,0.25uM ACC synthase specific primer (ACCF1, ACC-1, ACC-C or ACC-D), 0.25uJVI Mu specific primer (MuTIR), 0.25µl HotStarTaq and 1.5µ1 total nucleic acid as a template in a total reaction volume of 25µ1. Reaction conditions were as follows: 95°C/15 mm. (1 cycle); 95°C/1 min, 62°C/1 min, 72°C/2 min (35 cycles); 72°C/5 min (1 cycle). PCR products were separated on a 1% agarose gel, visualized following staining with ethidium bromide and transferred to nylon membranes according to methods described in Sambrook et al. (1989). Southern blot analysis was performed as described above for library screening except the hybridization temperature was increased to 45°C. BCl (backcross 1) seed was planted from each of the 13 putative mutant lines and screened by<br>
PCR/Southem analysis (as just described). Of these lines, only 5 were found to be stably inherited. These five lines were backcrossed an additional 4 times to minimize the effects of unrelated Mu insertions.<br>
[0287]	BC5 seed was self-pollinated to generate homozygous null individuals.<br>
Homozygous null individual lines were identified by PCR using Takara LA Taq and the ACCF1 and ACC-1 primers. Reactions contained l|ul leaf DNA, 2 uM each primer (final concentration), IX buffer (final concentration), 400juM dNTPmix (final concentration) and 1.25 U LA Taq in 25 u,l total volume. Reactions were carried out under the following conditions: 98°C/1 min. (1 cycle); 98°C/30 sec, 69°C/15 rnin. (35 cycles); 72°C/10 min (1 cycle). PCR of wild-type B73 DNA using these primers and conditions results in the amplification of three different sized fragments corresponding to the three genes identified. Individuals which are either wild-type or heterozygous for one of the null insertion-alleles display this characteristic pattern while those which are homozygous for one of the null insertion-aUeles are missing the subsequence corresponding to the gene in which the insertion is located.<br>
[0288]	To determine the exact location of the Mu insertion site, PCR products from<br>
each of the lines were amplified using either the ACCF1 or ACC-1 primer in combination with the MuTIR primer. These fragments were then sequenced across the My/target gene junction using the Mu-TIR primer. The location of these Mu elements within each of the ACC synthase genes is shown in Figure 2.<br>
Protein Extraction<br>
[0289]	For total protein isolation, leaves of B73 or mutant plants were collected at<br>
the indicated times, quick-frozen in liquid nitrogen and ground to a fine powder. One nil of extraction buffer (20mM HEPES (pH 7.6), lOOmM KC1,10% Glycerol) was added to approximately 0.1 g frozen powder and mixed thoroughly. Samples were centrifuged 10 min at 10,000 rpm, the supernatant removed to a new tube and the concentration determined spectrophotometrically according to the methods of Bradford (1976). See, Bradford MM (1976) A rapid and sensitive method for the quantitation ofmicrogram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochern. 72: 248-254. (See Figure 18).<br>
Chlnrnphvll extraction<br>
[0290]	Leaves were frozen in liquid nitrogen and ground to a fine powder. Samples<br>
of approximately 0.1 g were removed to a 1.5 ml tube and weighed. Chlorophyll was extracted 5X with 1 ml (or 0.8 ml) of 80% acetone. Individual extractions were combined and the final volume adjusted to 10 ml (or 15 ml) with additional 80% acetone. Chlorophyll content (a+b) was determined spectrophotometrically according to the methods of Wellburn (1994). See, Wellbum, A.R. (1994) The spectral determination of chlorophylls a and b, as well as total caretenoids, using various solvents with spectrophotometers of different resolution. J. Plant Phvsiol. 144:307-313. (See Figure 17).<br>
Measurement of photosynthesis<br>
[0291]	Plants were grown in the field under normal and drought-stress conditions.<br>
Normal plants were watered for eight hours twice a week. For drought-stressed plants, water was limited to approximately four hours per week for a period starting approximately one week before pollination and continuing through three weeks after pollination. During the period of limited water availability, drought-stressed plants showed visible signs of wilting and leaf rolling. Transpiration, stomatal conductance and COz assimilation were determined with a portable TPS-1 Photosynthesis System (PP Systems). Each leaf on a plant was measured at forty days after pollination. Values represent a mean of six determinations. See Figures 5 and 6.<br>
DNA and RNA purification<br>
[0292]	For total nucleic acid isolation, leaves of B73 are collected at desired times,<br>
quick-frozen in liquid nitrogen and ground to a fine powder. Ten ml of extraction buffer (lOOmM Tris (pH 8.6), SOmMEDTA, 200rriivl NaCl, 1% SDS, lOjAl/nil p-meicaptoethanol) is added and mixed thoroughly until thawed. Ten ml of Phenol/Chloroform (1:1, vol:vol) is added and mixed thoroughly. Samples are centrifuged 10 min at 8,000 rpm, the supernatant is removed to a new tube and the nucleic acid is precipitated at -20°C following addition of 1/10 vol 3M sodium acetate and 1 vol isopropanol. Total nucleic acid is pelleted by centrifugation at 8,000 rpm and resuspended in 1 ml TE. One half of the prep is used for DNA purification and the remaining half is used for RNA purification. (Alternatively, DNA or total nucleic acids can be extracted from 1 cm2 of seedling leaf, quick-frozen in liquid nitrogen, and ground to a fine powder. 600 (.U of extraction buffer [100 mM Tris (pH 8.0), 50 mM EDTA, 200 mM NaCl, 1% SDS, 10 u,l/ml p-mercaptoethanol] is added and the<br>
sample mixed. The sample is extracted with 700 µ1phenol/chloroform (1:1) and centrifuged for 10 min at 12,000 rpm. DNA is precipitated and resuspended in 600 µ1 H2O.)<br>
[0293]	For DNA purification, 500 µ,g DNase-free RNase is added to the tube and<br>
incubated at 37°C for 1 hr. Following RNase digestion, an equal volume of Phenol/Chloroform (1:1, vohvol) is added and mixed thoroughly. Samples are centrifuged 10 min at 10,000 rpm, the supernatant is removed to a new tube and the DNA precipitated at -20°C following addition of 1/10 vol 3M sodium acetate and 1 vol isopropanol. DNA is resuspended in sterile water and the concentration is determined spectrophotometrically. To determine DNA integrity, 20 mg of DNA is separated on a 1.8% agarose gel and visualized following staining with ethidium bromide. RNA is purified by 2 rounds of LiCl2 precipitation according to methods described by Sambrook et al, supra.<br>
Real-time RT-PCR analysis<br>
[0294]	Fifty ng total RNA is treated with RQ1 DNase (Promega) to ensure that no<br>
contaminating DNA is present. Two jig total RNA is used directly for cDNA synthesis using the Omniscript RT kit (Qiagen) with oligo-dT(20) as the primer.<br>
[0295]	Analysis of transcript abundance is accomplished using the QuantiTect<br>
SYBR Green PCR kit (Qiagen). Reactions contain IX buffer, 0.5 µ1 of the reverse transcription reaction (equivalent to 50 ng total RNA) and Q.25µM (final concentration) forward and reverse primers (see table 2 below) in a total reaction volume of 25µl.<br>
TABLE 2<br><br>
(Table Removed) <br>
[0296]	Reactions are carried out using an ABI PRISM 7700 sequence detection<br>
system under the following conditions: 95°C/15 min. (1 cycle); 95°C/30 sec, 62°C/30 sec, 72°C/2 min (50 cycles); 72°C/5 min (1 cycle). Each gene is analyzed a minimum of four times.<br>
[0297]	All the primer combinations are initially run and visualized on an agarose gel<br>
to confirm the presence single product of the correct size. All amplification products are subcloned into the pGEM-T Easy vector system (Promega) to use for generation of standard curves to facilitate conversion of expression data to a copy/µg RNA basis.<br>
Ethylene determination<br>
[0298]	Ethylene was measured from the second fully-expanded leaf of seedlings<br>
leaves at the 4-leaf stage or from the terminal 15 cm of leaves of plants 20, 30, or 40 days after pollination (DAP). Leaves were harvested at the indicated times and allowed to recover for 2 hr prior to collecting ethylene, between moist paper towels. Leaves were placed into glass vials and capped with a rubber septum. Following a 3-4 hour incubation, 0.9 mL of headspace was sampled from each vial and the ethylene content measured using a 6850 series gas chromatography system (Hewlett-Packard, Palo Alto, CA) equipped with a HP Plot alumina-based capillary column (Agilent Technologies, Palo Alto, CA). Tissue fresh weight was measured for each sample. Three replicates were measured and the average and standard deviation reported.<br>
Western blot analysis<br>
[0299]	B73 leaves were collected at the indicated times and ground in liquid<br>
nitrogen to a fine powder. One ml of extraction buffer [20 mM HEPES (pH 7.6), 100 mM KCI, 10% glycerol, 1 mM PMSF] was added to approximately 0.1 g frozen powder and mixed thoroughly. Cell debris was pelleted by centrifugation at 10,000 rpm for 10 min and the protein concentration determined as described (Bradford, 1976). Antiserum raised against the large subunit of rice Rubisco was obtained from Dr. Tadahiko Mae (Tohoku University, Sendai, Japan). Protein extracts were resolved using standard SDS-PAGE and the protein transferred to 0.22 /xm nitrocellulose membrane by electroblotting. Following transfer, the membranes were blocked in 5% milk, 0.01% thimerosal in TPBS (0.1% TWEEN 20, 13.7 mM NaCl, 0.27 mM KCI, 1 mM Na2HPO4, 0.14 mM KH2PO4) followed by incubation with primary antibodies diluted typically 1:1000 to 1:2000 in TPBS with 1% milk for 1.5 hrs. The blots were then washed twice with TPBS and incubated with goat anti-rabbit horseradish peroxidase-conjugated antibodies (Southern Biotechnology Associates, Inc.) diluted to 1:5000 to 1:10,000 for 1 hr. The blots were washed twice with TPBS and the signal detected typically between 1 to 15 min using chemiluminescence (Amersham Corp).<br>
RESULTS<br>
Identification of ACC Synthase Knockout Mutants<br>
[0300]	Three genes encoding ACC synthase were isolated from the inbred B73 and<br>
sequenced (see, e.g., SEQ ID NOs: 1-11). Two members of the family (i.e., ACS2 and ACS7) are closely related (97% amino acid identity) whereas the third gene (i.e., ACS
[0301]	For a description of ACC synthase expression patterns during endosperm and<br>
embryo development, see Gallic and Young (2004) The ethylene biosynthetic and perception machinery is differentially expressed during endosperm and embryo development Mol Gen Genomics 271:267-2S 1.<br>
ACS6 or ACS2 Gene Disruption Reduces Ethvlene Synthesis<br>
[0302]	The level of ethylene evolution in maize leaves increased as a function of<br>
leaf age (Figure 19 Panel C), At 20 DAP, the highest level of ethylene was observed in leaf 1 (the oldest surviving leaf) which by 30 DAP had progressed to leaf 3 and by 40 DAP (i.e., kernel maturity) to leaves 4-5. To determine whether Mu disruptions of ACS6 or ACS2 described above reduced ethylene evolution, ethylene was measured from leaf 4 of wild-type and mutant plants. Ethylene evolution from acs2 plants was approximately 55% of wild-type plants, a level that was similar for all acs2 mutant alleles (Figure 19 Panels A-B). Ethylene evolution from acs6 plants was only 10% of that from wild-type plants (Figure 19 Panel B). Ethylene evolution from acs2lacs6 double mutant plants was similar to that from acs6 plants. These data suggest that loss of ACS6 expression results in a greater reduction in the ability of maize leaves to produce ethylene than does the loss of ACS2 expression.<br>
Disruption ofACS6 Confers a Staygreen Phenotype<br>
[0303]	A substantial increase in ethylene evolution correlated with the appearance<br>
of visible signs of senescence in wild-type leaves, suggesting that ethylene may promote the entry of leaves into the senescence program. If so, a delay in the senescence of acs6 leaves, which produce significantly less ethylene, would be expected. To test this possibility, homozygous (i.e., acs6/acsd), heterozygous (i.e., ACS6/acs6), and wild-type (i.e., ACS6/ACS6) plants were field-grown until 50 days after pollination. At this.stage, the oldest wild-type leaves had senesced, whereas the corresponding A CS6/acs6 leaves were just beginning to senesce and acs6/acs6 leaves remained fully green. These observations suggest that the level of ethylene evolution may determine the timing of leaf senescence.<br>
[0304]	Senescence can also be induced following prolonged exposure to darkness.<br>
To determine whether a reduction in ethylene evolution can delay dark-induced senescence, leaves from adult plants were covered with sheaths to exclude light for two weeks. The leaves from younger plants (i.e., 20 DAP) were used to ensure that age-related senescence would not occur during the course of the experiment and they remained attached to the plant. Greenhouse grown maize was also employed to avoid any heating that might occur in the field as a consequence of the sheathing. Following the two-week dark-treatment, senescence was observed for virtually the entire region of wild-type leaves that was covered (the region covered by the sheath is indicated by the distinct transition from yellow to green, Figure 4 on left). The tip of ACS6/acs6 leaves had undergone dark-induced senescence but<br>
tne rest of the covered region showed significantly less senescence (Figure 4 in center). In contrast, acs6/acs6 leaves remained fully green (Figure 4 on right). The degree of senescence correlated with the amount of ethylene produced by each, in which ACS6/acs6 leaves produced just 70% of wild-type ethylene and acs6/acs6 leaves produced only 14.6% of wild-type ethylene. These results suggest that ethylene mediates the onset of dark-induced senescence as it does natural senescence. They also indicate that the ACS6/acs6 heterozygous mutant with a loss of one copy of ACS6 produces less ethylene and exhibits a weak staygreen phenotype similar to that observed for the acs2 mutant which also exhibited a moderate (i.e., 40%) reduction in ethylene evolution.<br>
[0305]	To examine if exogenous ACC could complement the acs6 mutant and<br>
reverse its staygreen phenotype, the third oldest, sixth, and ninth leaf from ACS6/ACS6, acs2/acs2, and acs6/acs6 plants were subject to dark-induced senescence at 20 DAP by covering them with sheaths for 7 days. All leaves were fully green at the onset of the experiment and remained attached to the plant. acs6/acs6 plants were watered daily with water or 100 uM ACC for 7 days. Following 7 days, dark-induced senescence had initiated in wild-type (i.e., ASC6/ASC6) leaves although it had not progressed to the extent observed following a 2 week dark treatment. The extent of dark-induced senescence increased as a function of leaf age such that leaf 3 exhibited more senescence than did leaf 6 or leaf 9 (which were younger), suggesting that competency for senescence increases with leafage. Dark-induced senescence was also observed in leaf 3 of the acs2 homozygous mutant although it was less pronounced than that observed in the corresponding wild-type leaves. Although no acs6 homozygous leaves exhibited dark-induced senescence consistent with the observations made in Fig. 4, dark-induced senescence similar to that of wild-type leaves was observed when acs6 leaves were watered with 100 µM ACC for 7 days. The ACC treatment did not affect acs6 leaves that were not sheathed, demonstrating that the senescence observed for sheathed acs6 leaves was specific to the dark-treatment.<br>
[0306]	Determination of the level of chlorophyll a + b from leaf 3 confirmed the<br>
visual results in that acs6 leaves retained substantially more chlorophyll after the 7 day dark treatment than did wild-type leaves but did not do so when watered with 100 \iM ACC (Figure 20 Panel A). The treatment with ACC in itself did not induce premature loss of chlorophyll as chlorophyll was not lost from unsheathed leaves of acs6 mutants watered with ACC. acs2 leaves retained only moderately greater amount of chlorophyll did wildtype<br>
. Similar results were observed for leaf 6 and leaf 9 although the level of chlorophyll in these younger leaves was higher than in the older leaf 3 samples as was expected (Figure 20 Panel A). Similar trends were observed for total soluble leaf protein: acs6 leaves retained substantially more protein following the dark treatment than did wild-type leaves but did not do so when watered with 100 µM ACC (Figure 20 Panel B).<br>
[0307]	Western analysis for ribulose biscarboxylase (Rubisco) demonstrated a<br>
substantial loss of Rubisco from dark-treated B73 leaves that was greater with the oldest leaves (leaf 3) than the youngest (leaf 9) (Figure 20 Panel C). Dark-treated acs6 leaves retained substantially more Rubisco than did dark-treated wild-type leaves and acs2 leaves retained a moderate level of Rubisco (Figure 20 Panel C). Dark-treated acs6 leaves watered with 100 u,M ACC lost an amount of Rubisco similar to that of dark-treated B73 leaves suggesting that ACC complemented the loss of ACC synthase expression. No loss of Rubisco was observed in ACC-treated acs6 leaves when they remained in the light demonstrating that treatment with ACC alone did not reduce the level of Rubisco. These data demonstrate that the staygreen phenotype, which involves retention of chlorophyll and leaf protein such as Rubisco, can be complemented by exogenous ACC, suggesting that the delay in senescence in these plants is a consequence of the reduction in loss of ACC synthase expression in the acs6 mutant.<br>
Reducing Ethylene Delays Natural Leaf Senescence and Reduces Loss of<br>
Chlorophyll and Protein<br>
[0308]	Drought is known to induce premature onset of leaf senescence. To<br>
investigate whether the drought response is mediated by ethylene and to determine whether reducing ethylene evolution may increase drought tolerance in maize, hemo?;ygous acs6 and acs2 mutant plants and wild-type plants were field-grown under well-watered (eight hours twice a week) and water-stressed conditions (four hours per week for a one month period that initiated approximately one week before pollination and continued for 3 weeks after pollination). During the period of limited water availability, drought-stressed plants exhibited leaf wilting and rolling, visible confirmation of drought stress. Following the water stress treatment, the extent of leaf senescence and function was measured.<br>
[0309]	Senescence of the oldest leaves was evident in wild-type plants under well-<br>
watered conditions and even more significantly during drought conditions. Similar results were observed for acs2 plants. In contrast, no visible sign of senescence was observed in<br>
acs6 leaves under well-watered or drought conditions. Interestingly, anthocyanin production was also reduced in acs6 leaves.<br>
[0310]	To confirm that the staygreen phenotype con-elates with enhanced levels of<br>
chlorophyll, the level of chlorophyll a and b was measured. Chlorophyll decreased with leaf<br>
age as well as with the age of the plants (Figure 17 Panels A and D). As expected, the<br>
greatest decrease in chlorophyll correlated with the visible onset of senescence. Under well-<br>
watered conditions, the level of chlorophyll in acs6 (ACS6 0/0) leaves was up to 8-fold<br>
higher than in the corresponding leaves of wild-type plants that had initiated senescence.<br>
Surprisingly, the level of chlorophyll in all acs6 leaves, including the youngest, was<br>
substantially higher than in wild-type plants (Figure 17 Panel A). The level of chlorophyll<br>
in acs2 (ACS2 0/0) leaves was moderately higher than in wild-type plants. These results<br>
indicate that the increase in chlorophyll content inversely correlates with the level of<br>
ethylene production: the moderate reduction in ethylene in acs2 plants correlated with a<br>
moderate increase in chlorophyll content whereas the large reduction in ethylene in acs6<br>
plants correlated with a substantial increase in chlorophyll content. These results also<br>
demonstrate that reducing ethylene increases the level of chlorophyll even in young leaves<br>
that are exhibiting maximum leaf function (see below).	. \.<br>
[0311]	Under drought conditions, the level of chlorophyll-was reduced in mutant<br>
and wild-type plants but decreased to an even greater extent in wild-type plants (Figure 17 Panels B-C). For example, the level of chlorophyll in leaf 5 of water-stressed wild-type plants decreased 2.5-fold relative to non-drought plants whereas it decreased by only 20% in leaf 5 of water-stressed acs6 plants (Figure 17 Panel C). Consequently, reducing eiiiylene evolution resulted in a level of chlorophyll in the oldest leaves of acs6 plants that was up to 20-fold higher than in the corresponding leaves of wild-type plants. As observed for non-drought plants, the level of chlorophyll was higher in all acs6 leaves, including the youngest. Chlorophyll content in acs2 leaves also remained moderately higher under drought conditions than in wild-type plants. Thus, loss of ACS6 expression reduced responsiveness to water stress in that chlorophyll content was substantially maintained under those stress conditions that had elicited a significant loss of chlorophyll in wild-type plants.<br>
[0312]	Leaf protein also declined with leaf age and with plant age (Figure 18 Panel<br>
D). As observed for chlorophyll, the most substantial decrease in protein correlated with the<br>
visible onset of senescence (Figure 18 Panel D). Under non-drought conditions, the level of protein in acs6 leaves was up to 2-fold higher than in the corresponding leaves of wild-type plants that had initiated senescence (Figure 18 Panel A). As observed for chlorophyll, the level of protein in all acs6 leaves, including the youngest, was substantially higher than in wild-type plants and the level of protein in acs2 leaves was moderately higher than in wild-type plants (Figure 18 Panel A). Exposure to conditions of drought resulted in a greater decrease of protein in the oldest wildtype leaves than was observed in acs6 leaves (Figure 18 Panels B-C). As observed for non-drought plants, the level of protein was higher in all acs6 leaves, including the youngest. These results parallel those for chlorophyll and indicate that protein content inversely con-elates with the level of ethylene evolution. They also demonstrate that loss of ACC synthase expression in the acs6 mutant reduced responsiveness to water stress in that protein levels were substantially maintained under those stress conditions that had elicited a significant reduction of protein in wild-type plants.<br>
Reducing Ethylene Maintains Leaf Function During Well-watered and<br>
Drought Conditions<br>
[0313]	The maintenance of chlorophyll and protein in acs6 leaves suggests that leaf<br>
function, e.g., the ability to transpire and assimilate COa, may also be maintained. To investigate this, the rate of transpiration, stomatal conductance, and rate of COa assimilation were measured in every leaf of well watered acs6 and wild-type plants at 40 DAP when the lower leaves of wild-type plants had begun to senesce. The youngest leaves of acs6 plants exhibited a higher rate of transpiration (Figure 5 Panel A) and stomatal conductance (Figure 5 Panel B) than control plants whereas no significant difference was observed in older leaves. In contrast, the rate of CO2 assimilation was substantially higher in.all leaves of acs6 plants than in control plants (Figure 5 Panel C). Specifically, older leaves of acs6 plants exhibited more than a 2-fold higher rate of CO2 assimilation than wild-type plants and the rate of COa assimilation in younger leaves increased from 50 to 100% (Figure 5 Panel C).<br>
[0314]	The effect of reducing ethylene on the maintenance of leaf function under<br>
drought conditions was also investigated. The rate of transpiration (Figure 5 Panel D) and stomatal conductance (Figure 5 Panel E) were significantly reduced in wild-type leaves when subjected to conditions of drought (i.e., four hours per week for a one month period starting approximately one week before pollination and continuing through three weeks<br>
after pollination) whereas they remained largely unaffected in acs6 leaves, resulting in a "substantially higher rate of transpiration (Figure 5 Panel D) and increased stomatal conductance (Figure 5 Panel E) for the mutant. In addition, drought treatment resulted in a significant decrease in the rate of CC-2 assimilation in wild-type leaves but not in acs6 leaves, resulting in up to a 2.5-fold increase in CO2 assimilation in younger acs6 leaves and up to a 6-fold increase in older acs6 leaves than in the control (Figure 5 Panel F). These results indicate that ethylene controls leaf function during conditions of drought and a reduction in its production results in a delay of leaf senescence in older leaves while maintaining leaf function in all leaves thus providing greater tolerance to drought. Similar, though less pronounced, results were obtained for ACS2 (Figure 6 Panels A-C).<br>
DISCUSSION<br>
[0315]	In summary, ACC synthase mutants affecting the first step in ethylene<br>
biosynthesis were isolated in maize. These mutants exhibited a delay in natural, dark-induced, and drought-induced leaf senescence and a staygreen phenotype. The delay in senescence was reversible following exposure to ethylene. ACC synthase mutant leaves exhibited a substantially higher rate of COa assimilation during growth under normal or drought conditions. Surprisingly, improved leaf function was observed in all ACC synthase mutant leaves, including the youngest which had not entered either natural or drought-induced senescence programs. These observations suggest that ethylene mediates the response of maize to water stress and that decreasing ethylene production serves as a means to maintain leaf performance during water stress and thereby increase its tolerance to drought conditions. As noted, ACC synthase mutants can have other advantageous phenotypes, e.g., male sterility phenotypes, crowding resistance phenotypes, altered pathogen resistance, and the like.<br>
[0316]	The above examples show that ethylene plays a significant role in regulating<br>
the onset of leaf senescence in maize whether during growth under well-watered conditions or during conditions of drought which normally induces premature leaf senescence. The reduction in ethylene evolution resulting from loss of ACS6 expression is largely responsible for directing natural and drought-induced leaf senescence. While not intending to be limited by any particular theory, loss of ACS6 expression may directly delay entry into the senescence program or may affect total ACC synthase expression from all gene<br>
lumbers. Knockout of ACS2 alone reduced ethylene production by approximately 40% and<br>
IT<br>
did result in a small increase in chlorophyll and protein. In contrast, ethylene production in acs6 leaves was reduced up to 90% and acs6 leaves contained substantially higher levels of chlorophyll and protein. These observations suggests that entry into the senescence program may be controlled by more than one gene family member.<br>
[0317]	The level of chlorophyll and protein in wild-type leaves was reduced<br>
substantially following water-stress but remained unaffected in acs6 leaves. These results indicate two roles for ethylene in maize leaves: under normal growth conditions, ethylene may help to maintain the correct level of chlorophyll and protein in a leaf, whereas during water stress, ethylene may serve to reduce the level of both. The observation that a 40% reduction in ethylene resulted in a moderate increase in chlorophyll and protein whereas a 90% reduction resulted in a substantially larger increase in chlorophyll and protein suggests that these leaf components may be quantitatively controlled by the level of ethylene produced in leaves. Greater increases in leaf chlorophyll and protein might be expected if ethylene production were reduced even further.<br>
[0318]	Loss of chlorophyll and protein in wild-type maize subjected to drought<br>
conditions was accompanied by decreased rates of transpiration, stomatal conductance, and COi assimilation. In contrast, maintenance of chlorophyll and protein levels in leaves of acs6 plants subjected to drought conditions was accompanied by the maintenance of transpiration, stomatal conductance, and CC&gt;2 assimilation. These results suggest that reducing ethylene not only confers a staygreen phenotype but actually maintains leaf function under stress conditions. The observation that ethylene controls the onset of leaf senescence is consistent with the role of this hormone in other species such as Arabidopsis and tomato (Davis and Grierson (1989) Identification ofcDNA clones for tomato (Lycopersicon esculentum Mill.) mRNAs that accumulate during fruit ripening and leaf senescence in response to ethylene. Planta 179:73-80; Abeles et al. (1992). Ethylene in Plant Biology. (San Diego: Academic Press); Picton et al. (1993). Altered fruit ripening and leaf senescence in tomatoes expressing an antisense ethylene-fonning enzyme transgene Plant J. 3:469-481; Grbic and Bleecker (1995) Ethylene regulates the timing of leaf senescence in Arabidopsis Plant J. 8:95-102; John et al. (1995) Delayed leaf senescence in ethylene-deficient ACC-oxidase antisense tomato plants: molecular and physiological analysis Plant J. 7:483-490). The observation that a reduction in ethylene evolution would<br>
increase the level of chlorophyll and protein and increase the rate of CO2 assimilation in all feaves, including the youngest, was unexpected. This suggests that ethylene plays an active role in controlling aspects of leaf function well before a leaf enters a senescence program. Equally unexpected was the observation that a reduction in ethylene would affect the water-stress response of all leaves. These findings suggest that increased tolerance to conditions of drought can be easily introduced into maize, and optionally other grain species, through a reduction in the level of ethylene produced in leaves.<br>
EXAMPLE 2; SEQUENCE ALIGNMENTS AND PHYLOGENETIC ANALYSIS<br>
[0319]	A phylogenetic analysis of ACC synthase sequences described herein, e.g.,<br>
(A47 (also known as ACS2 or ACC2 herein), A50 (also known as ACS? or ACC7 herein), A65 (also known as ACS6 or ACC6 herein)) from maize, with ACC synthase sequences from other species, is shown in Figure 7 (ACSgrowtree2), where Arabidopsis sequences are indicated by (At ACS...), tomato sequences are indicated by (LeACS...), rice sequences are indicated by (indica (OsiACS...), and japonica (OsjACS...)), wheat sequences are indicated by (TaACS...), and banana sequences are indicated by (MaACS...). In the analysis, the indicated ACC synthases fall into two subfamilies. One of the subfamilies is further subdivided into monocot (Zm (maize), Osi, Osj, Ta, Ma) ACS genes and dicot (At, Le) ACS genes.<br>
[0320]	Various peptide consensus sequences alignments of ACC synthase sequences<br>
described herein, e.g., (A47 (also known as ACS2 or ACC2 herein), A50 (also known as ACS7 or ACC7 herein), A65 (also known as ACS6 or ACC6 herein)) from maize (Zm), with ACC synthase sequences from other species are shown in Figures 8-16. A Pretty program is used (e.g., available on the SeqWeb (GCG) web page) to determine the consensus sequence with different stringencies (e.g., most stringent (identical), stringent (similar amino acids), or least stringent (somewhat similar amino acids). The stringency is indicated in each figure after "consensus sequence." The Gap Weight is 8 and the GapLengthWeight is 2.<br>
EXAMPLE 3; ACC SYNTHASE KNOCKOUTS BY HAIRPIN RNA EXPRESSION<br>
[0321]	As noted previously, knockout plant cells and plants can be produced, for<br>
example, by introduction of an ACC synthase polynucleotide sequence configured for RNA silencing or interference. This example describes hairpin RNA expression cassettes for<br>
juodifying ethylene production and staygreen phenotype, e.g., in maize. As noted previously, knockout of ACC synthase(s), e.g., by hpRNA expression, can result in plants or plant cells having reduced expression (up to and including no detectable expression) of one or more ACC synthases.<br>
[0322]	Expression of hairpin RNA (hpRNA) molecules specific for ACC synthase<br>
genes (e.g., promoters, other untranslated regions, or coding regions) that encode ACC synthases in plants can alter ethylene production and staygreen potential, sterility, crowding resistance, etc. of the plants, e.g., through RNA interference and/or silencing.<br>
[0323]	hpRNA constructs of ACS2 (PHP20600) and ACS6 (PHP20323) were<br>
generated by linking a ubiquitin promoter to an inverted repeat of a portion of the coding sequence of either the ACS2 or ACS6 gene (see Figures 21 and 22, Panels A-C). Each construct was transformed into maize using Agrobacterium-mediated transformation techniques. Nucleic acid molecules and methods for preparing the constructs and transforming maize were as previously described and known in the art; see, e.g., the sections herein entitled "Vectors, Promoters, and Expression Systems," "Plant Transformation," "Other Nucleic Acid and Protein Assays," and the following example "Transformation of Maize".<br>
[0324]	Expression of hpRNA specific for either ACS2 or ACS6 coding sequences<br>
resulted in maize plants that displayed no abnormalities in vegetative and reproductive growth. A total of 36 and 40 individual maize transgenic events were generated for ACS2-and ACS6-hairpin constructs, respectively (Figure 23, Panels A and B).<br>
t0325]	Approximately 10 low copy number events per hpPv'NA construct were<br>
selected for additional backcrossing and transgene evaluation. Staygreen potential phenotype is evaluated for the backcrossed lines comprising the hpRNA transgene(s), e.g., as described herein (for example, by visual inspection, measurements of photosynthetic activity, determination of chlorophyll or protein content, or the like, under normal and drought or other stress conditions).<br>
EXAMPLE 4; TRANSFORMATION OF MAIZE<br>
BIOLISTICS<br>
[0326]	The inventive polynucleotides contained within a vector are transformed into<br>
embryogenic maize callus by particle bombardment, generally as described by Tomes, D. et al, IN: Plant Cell, Tissue and Organ Culture: Fundamental Methods, Eds. O.L. Gamborg and G.C. Phillips, Chapter 8, pgs. 197-213 (1995) and as briefly outlined below. Transgenic maize plants are produced by bombardment of embryogenically responsive immature embryos with tungsten particles associated with DNA plasmids. The plasmids typically comprise or consist of a selectable marker and an unselected structural gene, or a selectable marker and an ACC synthase polynucleotide sequence or subsequence, or the like.<br>
Preparation of Particles:<br>
[0327]	Fifteen mg of tungsten particles (General Electric), 0.5 to 1.8 u,, preferably 1<br>
to l.S /Lt, and most preferably 1 \i, are added to 2 ml of concentrated nitric acid. This suspension is sonicated at 0°C for 20 minutes (Branson Sonifier Model 450,40% output, constant duty cycle). Tungsten particles are pelleted by centrifugation at 10000 rpm (Biofuge) for one minute, and the supernatant is removed. Two milliliters of sterile distilled water are added to the pellet, and brief sonication is used to resuspend the particles. The suspension is pelleted, one milliliter of absolute ethanol is added to the pellet, and brief sonication is used to resuspend the particles. Rinsing, pelleting, and resuspending of the particles is performed two more times with sterile distilled water, and finally the particles are resuspended in two milliliters of sterile distilled water. The particles are subdivided into 250-^1 aliquots and stored frozen.<br>
Preparation of Particle-Plasmid DNA Association:<br>
[0328]	The stock of tungsten particles are sonicated briefly in a water bath sonicator<br>
(Branson Sonifier Model 450, 20% output, constant duty cycle) and 50 pil is transferred to a microfuge tube. The vectors are typically cis: that is, the selectable marker and the gene (or other polynucleotide sequence) of interest are on the same plasmid.<br>
[0329]	Plasmid DNA is added to the particles for a final DNA amount of 0.1 to 10<br>
pig in 10 u,L total volume, and briefly sonicated. Preferably, 10 u,g (1 u.g/u,L in TE buffer) total DNA is used to mix DNA and particles for bombardment. Fifty microliters (50 uJL) of<br>
sterile aqueous 2.5 M CaCl2 are added, and the mixture is briefly sonicated and vortexed. Twenty microliters (20 µL) of sterile aqueous 0.1 M spermidine are added and the mixture is briefly sonicated and vortexed. The mixture is incubated at room temperature for 20 minutes with intermittent brief sonication. The particle suspension is centrifuged, and<br>
_the supernatant is removed. Two hundred fifty microliters (250 µL) of absolute ethanol are added to the pellet, followed by brief sonication. The suspension is pelleted, the supernatant is removed, and 60 µl of absolute ethanol are added. The suspension is sonicated briefly before loading the particle-DNA agglomeration cnto macrocarriers.<br>
Preparation of Tissue<br>
[0330]	Immature embryos of maize variety High Type II are the target for particle<br>
bombardment-mediated transformation. This genotype is the F1 of two purebred genetic lines, parents A and B, derived from the cross of two known maize inbreds, A1SS and B73. Both parents are selected for high competence of somatic embryogenesis, according to Armstrong et al., Maize Genetics Coop. News 65:92 (1991).<br>
[0331]	Ears from FI plants are selfed or sibbed, and embryos are aseptically<br>
dissected from developing caryopses when the scutellum first becomes opaque. This stage occurs about 9-13 days post-pollination, and most generally about 10. days post-pollination, depending on growth conditions. The embryos are about 0.75 to 1.5 millimeters long. Ears are surface sterilized with 20-50% Clorox for 30 minutes, followed by three rinses with sterile distilled water.<br>
[0332]	Immature embryos are cultured with the scutellum oriented upward, on<br>
embryogenic induction medium comprised of N6 basal salts, Eriksson vitamins, 0.5 mg/l thiamine HC1, 30 gm/1 sucrose, 2.88 gm/1 L-proline, 1 mg/12,4-dichlorophenoxyacetic acid, 2 gm/1 Gelrite, and 8.5 mg/1 AgNO3. Chu et al., Sci. Sin. 18:659 (1975); Eriksson, Phvsiol. Plant 18:976 (1965). The medium is sterilized by autoclaving at 121°C for 15 minutes and dispensed into 100 X 25 mm Petri dishes. AgNO3 is filter-sterilized and added to the medium after autoclaving. The tissues are cultured in complete darkness at 28°C. After about 3 to 7 days, most usually about 4 days, the scutellum of the embryo swells to about double its original size and the protuberances at the coleorhizal surface of the scutellum indicate the inception of embryogenic tissue. Up to 100% of the embryos display this response, but most commonly, the embryogenic response frequency is about 80%.<br>
[0333]	When the embryogenic response is observed, the embryos are transferred to<br>
a medium comprised of induction medium modified to contain 120 gm/1 sucrose. The embryos are oriented with the coleorhizal pole, the embryogenically responsive tissue, upwards from the culture medium. Ten embryos per Petri dish are located in the center of a Petri dish in an area about 2 cm in diameter. The embryos are maintained on this medium for 3-16 hour, preferably 4 hours, in complete darkness at 28°C just prior to bombardment with particles associated with plasmid DNAs containing the selectable and unselectable marker genes.<br>
[0334]	To effect particle bombardment of embryos, the particle-DNA agglomerates<br>
are accelerated using a DuPont PDS-1000 particle acceleration device. The particle-DNA agglomeration is briefly sonicated and 10 uJ are deposited on macrocarriers and the ethanol is allowed to evaporate. The macrocarrier is accelerated onto a stainless-steel stopping screen by the rupture of a polymer diaphragm (rupture disk). Rupture is effected by pressurized helium. The velocity of particle-DNA acceleration is determined based on the rupture disk breaking pressure. Rupture disk pressures of 200 to 1SOO psi are used, with 650 to 1100 psi being preferred, and about 900 psi being most highly preferred. Multiple disks are used to effect a range of rupture pressures.<br>
[0335]	The shelf containing the plate with embryos is placed 5.1 cm below the<br>
bottom of the macrocarrier platform (shelf #3). To effect particle bombardment of cultured immature embryos, a rupture disk and a macrocarrier with dried particle-DNA agglomerates are installed in the device. The He pressure delivered to the device is adjusted to 200 psi above the rupture disk breaking pressure. A Petri dish with the target embryos is placed into the vacuum chamber and located in the projected path of accelerated particles. A vacuum is created in the chamber, preferably about 28 in Hg. After operation of the device, the vacuum is released and the Petri dish is removed.<br>
[0336]	Bombarded embryos remain on the osmotically-adjusted medium during<br>
bombardment, and 1 to 4 days subsequently. The embryos are transferred to selection medium comprised of N6 basal salts, Eriksson vitamins, 0.5 mg/1 thiamine HC1, 30 gm/1 sucrose, 1 mg/1 2,4-dichlorophenoxyacetic acid, 2 gm/1 Gelrite, 0.85 mg/1 Ag NO3 and 3 mg/1 bialaphos (Herbiace, Meiji). Bialaphos is added filter-sterilized. The embryos are subcultured to fresh selection medium at 10 to 14 day intervals. After about 7 weeks, embryogenic tissue, putatively transformed for both selectable and unselected marker genes,<br>
i61iferates from about 7% of the bombarded embryos. Putative transgenic tissue is rescued, and that tissue derived from individual embryos is considered to be an event and is propagated independently on selection medium. Two cycles of clonal propagation are achieved by visual selection for the smallest contiguous fragments of organized embryogenic tissue.<br>
[0337]	A sample of tissue from each event is processed to recover DNA. The DNA<br>
is restricted with a restriction endonuclease and probed with primer sequences designed to amplify DNA sequences overlapping the ACC synthase and non-ACC synthase portion of the plasmid. Embryogenic tissue with amplifiable sequence is advanced to plant regeneration.<br>
[0338]	For regeneration of transgenic plants, embryogenic tissue is subcultured to a<br>
medium comprising MS salts and vitamins (Murashige &amp; Skoog, Physiol. Plant 15: 473 (1962)), 100 mg/l myo-inositol, 60 gm/1 sucrose, 3 gm/1 Gelrite, 0.5 mg/1 zeatin, 1 mg/1 indole-3-acetic acid, 26.4 ng/1 cis-trans-abscissic acid, and 3 rng/1 bialaphos in 100 X 25 mm Petri dishes, and is incubated in darkness at 28°C until the development of well-formed, matured somatic embryos can be seen. This requires about 14 days. Well-formed somatic embryos are opaque and cream-colored, and are comprised of an identifiable scutellum and coleoptile. The embryos are individually subcultured to -a germination medium comprising MS salts and vitamins, 100 mg/1 myo-inositol, 40 gm/1 sucrose and 1.5 gm/1 Gelrite in 100 X 25 mm Petri dishes and incubated under a 16 hour light:8 hour dark photoperiod and 40 meinsteinsm^sec"1 from cool-white fluorescent tubes. After about 7 days, the somatic embryos have germinated and produced a well-defined shoot and root. The individual plants are subcultured to germination medium in 125 X 25 mm glass tubes to allow further plant development. The plants are maintained under a 16 hour light: 8 hour dark photoperiod and 40 meinsteinsm"2sec"' from cool-white fluorescent tubes. After about 7 days, the plants are well-established and are transplanted to horticultural soil, hardened off, and potted into commercial greenhouse soil mixture and grown to sexual maturity in a greenhouse. An elite inbred line is used as a male to pollinate regenerated transgenic plants.<br>
AGROBACTERIUM-MEDIATED<br>
[0339]	When Agrobacterium-mediated transformation is used, the method of Zhao<br>
is employed as in PCT patent publication WO98/32326, the contents of which are hereby<br>
Incorporated by reference. Briefly, immature embryos are isolated from maize and the embryos contacted with a suspension of Agrobacterium (step 1: the infection step). In this step the immature embryos are preferably immersed in an Agrobacterium suspension for the initiation of inoculation. The embryos are co-cultured for a time with the Agrobacterium (step 2: the co-cultivation step). Preferably the immature embryos are cultured on solid medium following the- infection step. Following this co-cultivation period an optional "resting" step is contemplated. In this resting step, the embryos are incubated in the presence of at least one antibiotic known to inhibit the growth of Agrobacterium without the addition of a selective agent for plant transformants (step 3: resting step). Preferably the immature embryos are cultured on solid medium with antibiotic, but without a selecting agent, for elimination of Agrobacterium and for a resting phase for the infected cells. Next, inoculated embryos are cultured on medium containing a selective agent and growing transformed callus is recovered (step 4: the selection step). Preferably, the immature embryos are cultured on solid medium with a selective agent resulting in the selective growth of transformed cells. The callus is then regenerated into plants (step 5: the regeneration step) and preferably calli grown on selective medium are cultured on solid medium to regenerate the plants.<br>
EXAMPLE 5. EXPRESSION OF TRANSGENES IN MONOCOTS<br>
[0340]	A plasmid vector is constructed comprising a preferred promoter operably<br>
linked to an isolated polynucleotide comprising an ACC synthase polynucleotide sequence or subsequence (e.g., selected from SEQ ID NOs:l-6 and 10). This construct can then be introduced into maize cells by the following procedure.<br>
[0341]	Immature maize embryos are dissected from developing caryopses derived<br>
from crosses of maize lines. The embryos are isolated 10 to 11 days after pollination when they are 1.0 to 1.5 mm long. The embryos are then placed with the axis-side facing down and in contact with agarose-solidified N6 medium (Chu et al. (1975) Sci. Sin. Peking 18:659-668). The embryos are kept in the dark at 27°C. Friable embryogenic callus, consisting of undifferentiated masses of cells with somatic proembryoids and embryoids borne on suspensor structures, proliferates from the scutellum of these immature embryos. The embryogenic callus isolated from the primary explant can be cultured on N6 medium and sub-cultured on this medium every 2 to 3 weeks.<br>
The plasmid p35S/Ac (Hoechst Ag, Frankfurt, Germany) or equivalent may be used in transformation experiments in order to provide for a selectable marker. This plasmid contains the Pat gene (see European Patent Publication 0 242 236) which encodes phosphinothricin acetyl transferase (PAT). The enzyme PAT confers resistance to herbicidal glutamine synthetase inhibitors such as phosphinothricin. The pat gene in p35S/Ac is under the control of the 35S promoter from Cauliflower Mosaic Virus (Odell et al. (1985) Nature 313:810-812) and comprises the 3' region of the nopaline synthase gene from the T-DNA of the Ti plasmid of Agrobacterium tumefaciens.<br>
[0343]	The particle bombardment method (Klein et al. (1987) Nature 327:70-73)<br>
may be used to transfer genes to the callus culture cells. According to this method, gold particles (1 µm in diameter) are coated with DNA using the following technique. Ten jig of plasmid DMAs are added to 50 µL of a suspension of gold particles (60 mg per mL). Calcium chloride (50 µL of a 2.5 M solution) and spermidine free base (20 µL of a 1.0 M solution) are added to the particles. The suspension is vortexed during the addition of these solutions. After 10 minutes, the tubes are briefly centrifuged (5 sec at 15,000 rpm) and the supernatant removed. The particles are resuspended in 200 µL of absolute ethanol, centrifuged again and the supernatant removed, The ethanol rinse is performed again and the particles resuspended in a final volume of 30 pL of ethanol. An aliquot (5 µL) of the DNA-coated gold particles can be placed in the center of a Kaptoh flying disc (Bio-Rad Labs). The particles are then accelerated into the com tissue with a Biolistic PDS-1000/He (Bio-Rad Instruments, Hercules CA), using a helium pressure of 1000 psi, a gap distance of 0.5 cm and a flying distance of 1.0 cm.<br>
[0344]	For bombardment, the embryogenic tissue is placed on filter paper over<br>
agarose-solidified N6 medium. The tissue is arranged as a thin lawn and covers a circular area of about 5 cm in diameter. The petri dish containing the tissue can be placed in the chamber of the PDS-1000/He approximately 8 cm from the stopping screen. The air in the chamber is then evacuated to a vacuum of 28 inches of Hg. The macrocarrier is accelerated with a helium shock wave using a rupture membrane that bursts when the He pressure in the shock tube reaches 1000 psi.<br>
[0345]	Seven days after bombardment the tissue can be transferred to N6 medium<br>
that contains gluphosinate (2 mg per liter) and lacks casein or proline. The tissue continues to grow slowly on this medium. After an additional 2 weeks the tissue can be transferred to<br>
fresh N6 medium containing gluphosinate. After 6 weeks, areas of about 1 cm in diameter of actively growing callus can be identified on some of the plates containing the glufosinate-supplemented medium. These calli may continue to grow when sub-cultured on the selective medium.<br>
[0346]	Plants can be regenerated from the transgenic callus by first transferring<br>
clusters of tissue to N6 medium supplemented with 0.2 mg per liter of 2,4-D. After two weeks the tissue can be transferred to regeneration medium (Fromm et al, (1990) Bio/Technology 5:833-839).<br>
EXAMPLE 6. EXPRESSION OF TRANSGENES IN DICOTS<br>
[0347]	Soybean embryos are bombarded with a plasmid comprising a preferred<br>
promoter operably linked to a heterologous nucleotide sequence comprising an ACC synthase polynucleotide sequence or subsequence (e.g., selected from SEQ ID NOs:l-6 and 10), as follows. To induce somatic embryos, cotyledons of 3-5 mm in length are dissected from surface-sterilized, immature seeds of the soybean cultivar A2872, then cultured in the light or dark at 26°C on an appropriate agar medium for six to ten weeks. Somatic embryos producing secondary embryos are then excised and placed into a suitable liquid medium. After repeated selection for clusters of somatic embryos that multiply as early, globular-staged embryos, the suspensions are maintained as described below.<br>
[0348]	Soybean embryogenic suspension cultures can be maintained in 35 ml liquid<br>
media on a rotary shaker, 150 rpm, at 26°C with fluorescent lights on a 16:8 hour day/night schedule. Cultures are sub-cultured every two weeks by inoculating approximately 35 mg of tissue into 35 ml of liquid medium.<br>
[0349]	Soybean embryogenic suspension cultures may then be transformed by the<br>
method of particle gun bombardment (Klein et al. (1987) Nature (London) 327:70-73, U.S. Patent No. 4,945,050). A DuPont Biolistic PDS1000/HE instrument (helium retrofit) can be used for these transformations.<br>
[0350]	A selectable marker gene that can be used to facilitate soybean<br>
transformation is a transgene composed of the 35S promoter from Cauliflower Mosaic Virus (Odell et al. (19S5) Nature 373:810-812), the hygromycin phosphotransferase gene from plasmid pJR225 (from E. coli; Gritz et al. (1983) Gene 25:179-188), and the 3' region of the nopaline synthase gene from the T-DNA of the Ti plasmid of Agrobacterium<br>
jMiefaciens. The expression cassette of interest, comprising the preferred promoter and a heterologous ACC synthase polynucleotide, can be isolated as a restriction fragment. This fragment can then be inserted into a unique restriction site of the vector carrying the marker gene.<br>
[0351]	To 50µl of a 60 mg/ml 1 µm gold particle suspension is added (in order):<br>
5µDNA (1 µg/µ1), 20 µl spermidine (0.1 M), and 50 Hi CaCl2 (2.5 M). The particle preparation is then agitated for three minutes, spun in a microfuge for 10 seconds and the supernatant removed. The DNA-coated particles are then washed once in 400 µl 70% ethanol and resuspended in 40 µl of anhydrous ethanol. The DNA/particle suspension can be sonicated three times for one second each. Five microliters of the DNA-coated gold particles are then loaded on each macro carrier disk.<br>
[0352]	Approximately 300-400 mg of a two-week-old suspension culture is placed<br>
in an empty 60x15 mm petri dish and the residual liquid removed from the tissue with a pipette. For each transformation experiment, approximately 5-10 plates of tissue are normally bombarded. Membrane rupture pressure is set at 1100 psi, and the chamber is evacuated to a vacuum of 28 inches mercury. The tissue is placed approximately 3.5 inches away from the retaining screen and bombarded three times. Following'bornbardment, the tissue can be divided in half and placed back into liquid and cultured as described above.<br>
[0353]	Five to seven days post bombardment, the liquid media may be exchanged<br>
with fresh media, and eleven to twelve days post-bombardment with fresh media containing 50 mg/ml hygromycin. This selective media can be refreshed weekly. Seven to eight weeks post-bombardment, green, transformed tissue may be observed growing from untransformed, necrotic embryogenic clusters. Isolated green tissue is removed and inoculated into individual flasks to generate new, clonally propagated, transformed embryogenic suspension cultures. Each new line may be treated as an independent transformation event. These suspensions can then be subcultured and maintained as clusters of immature embryos or regenerated into whole plants by maturation and germination of individual somatic embryos.<br>
[0354]	It is understood that the examples and embodiments described herein are for<br>
illustrative purposes only and that various modifications or changes in light thereof will be<br>
suggested to persons skilled in the art and are to be included within the spirit and purview of this application and scope of the appended claims.<br>
[0355]	While the foregoing invention has been described in some detail for purposes<br>
of clarity and understanding, it will be clear to one skilled in the art from a reading of this disclosure that various changes in form and detail can be made without departing from the true scope of the invention. For example, all the techniques and apparatus described above can be used in various combinations. All publications, patents, patent applications, and/or other documents cited in this application are incorporated by reference in their entirety for all purposes to the same extent as if each individual publication, patent, patent application, and/or other document were individually indicated to be incorporated by reference for all purposes.<br><br><br><br>
CLAIMS<br>
WHAT IS CLAIMED IS:<br>
1.	An isolated or recombinant knockout plant cell comprising at least one disruption in<br>
at least one endogenous ACC synthase gene, wherein the disruption inhibits expression or<br>
activity of at least one ACC synthase protein compared to a corresponding control plant cell<br>
lacking the disruption.<br>
2.	The plant cell of claim 1, wherein the at least one endogenous ACC synthase gene<br>
comprises a nucleic acid sequence, or complement thereof, comprising at least about 85%<br>
sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ ID NO:3<br>
(gACS7).<br>
3.	The plant cell of claim 1, wherein the at least one endogenous ACC synthase gene<br>
comprises two or more endogenous ACC synthase genes.<br>
4.	The plant cell of claim 1, wherein the at least one endogenous ACC synthase gene<br>
comprises three or more endogenous ACC synthase genes.<br>
5.	The plant cell of claim 1, wherein the at least one disruption results in reduced<br>
ethylene production by the knockout plant cell as compared to the control plant cell.<br>
6.	The plant cell of claim 1, wherein the at least one disruption comprises one or more<br>
transposons, wherein the one or more transposons are in the at least one endogenous ACC<br>
synthase gene.<br>
7.	The, plant cell of claim 6, wherein the at least one disruption is a homozygous<br>
disruption in the at least one ACC synthase gene.<br>
8.	The plant cell of claim 6, wherein the at least one disruption is a heterozygous<br>
disruption in the at least one ACC synthase gene.    <br>
9.	The plant cell of claim 1, wherein the at least one disruption comprises one or more<br>
point mutations, wherein the one or more point mutations are in the at least one endogenous<br>
ACC synthase gene.<br>
10.	The plant cell of claim 1, wherein the at least one disruption is introduced into the<br>
knockout plant cell by introducing at least one polynucleotide sequence comprising an ACC<br>
synthase nucleic acid sequence, or subsequence thereof, into the knockout plant cell, such<br>
that the at least one polynucleotide sequence is linked to a promoter in a sense or antisense<br>
- V orientation, and wherein the at least one polynucleotide sequence comprises at least about<br>
85% sequence identity to SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ED NO:6 (cACS7), or SEQ ID NO: 10 (CCRA178R), or a subsequence thereof, or a complement thereof.<br>
11.	The plant cell of claim 10, wherein the at least one polynucleotide further comprises<br>
a vector.<br>
12.	The plant cell of claim 11, wherein the vector comprises a viral vector.<br>
13.	The plant cell of claim I, wherein the at least one disruption is introduced into the<br>
knockout plant cell by introducing at least one polynucleotide sequence comprising one or<br>
more subsequences of an ACC synthase nucleic acid sequence configured for RNA<br>
silencing or interference.<br>
14.	The plant cell of claim 13, wherein the at least one polynucleotide further comprises<br>
a vector.<br>
15.	The plant cell of claim 14, wherein the vector comprises a viral vector.<br>
16.	The plant cell of claim 1, wherein the plant cell is from a plant of the Poaceae<br>
{Gramineae) family.<br>
17.	The plant cell of claim 16, wherein the plant is a genus selected from the group<br>
consisting of: Acamptoclados, Achnathennn, Achnella, Acroceras, Aegihps, Aegopgon,<br>
Agroelyinus, Agrohordeum, Agropogon, Agropyron, Agrositanion, Agrostis, Aira, Allolepis,<br>
Alloteropais, Alopecums, Amblyopymm, 'Amtnophlla, Ampelodesmos, Amphibromus,<br>
Amphicarpum, Amphilophis, Anastroplms, Anathennn, Andropogron, Anemathele,<br>
Aneuwlepidium, Anisantha, Anthaenantia, Anthephora, Anthochloa, Anthoxanthum, Apera,<br>
Apluda, Archtagrostis, Arctophila, Argilhchloa, Aristida, Arrhenathemm, Aithraxon,<br>
Arthrostylidkan, Anmdinaria, Arundinella, Arundo, Aspris, Atheropogon, Avena, Avenella,<br>
Avenochloa, Avenula, Axonopus, Bambusa, Beckmannia, Blepharidachne, Blepharoneuron,<br>
Bothriochloa, Bouteloua, Brachiaria, Brachyelytrum, Brachypodium, Briza, Brizopynan,<br>
Bromelica, Bromopsis, Bromus, Biichloe, Bulhilis, Calamagrostis, Calamovilfa,<br>
Campulosus, Capriola, Catabrosa, Catapodium, Cathesteciun, Cenchropsis, Cenchnts,<br>
Centotheca, Ceratochloa, Chaetochloa, ChasinanThium, Chimonobambusa, Chionochloa,<br>
Chloris, Chondrosum, Chrysopon, Chusquea, Cinna, Cladoraphis, Coelorachis, Coix,<br>
Coleanthus, Colpodium, Coridochloa, Comucopiae, Cortaderia, Coryneplwrus, Cottea, Critesion, Crypsis, Ctenium, Cutandia, Cylindropyrum, Cymbopogon, Cynodon, Cynosurus, Cytrococcum, Dactylis, Dactylocteniwn, Danthonia, Dasyochloa, Dasyprum, Davyella, Dendrocalamus, Deschampsia, Desmazeria, Deyeitxia, Diaiina, Diarrhena, Dichanthelium, Dichanthium, Dichelachne, Diectomus, Digitaria, Dimeria, Dimorpostachys, Dinebra, Diplachne, Dissantheliwn, Dissochondms, Distichlis, Drepanostachyum, Dupoa, Dupontia, Echinochloa, Ectospenna, Ehrharta, Eleusine, Elyhordeum, Elyleymus, Elymordeum, Elymus, Elyonums, Elysitanion, Elytesion, Elytrigia, Enneapogon, Enteropogon, Epicampes, Eragrostis, Eremochloa, Eremopoa, Eremopyrum, Erianthus, Ericoma, Erichloa, Eriochrysis, Erioneuron, Euchlaena, Euclasta, Eulalia, Eulaliopsis, Eustachys, Fargesia, Festuca, Festuloliinn, Fingerhiithia, Fluminia, Gamotia, Gastridium, Gaudinia, Gigantochloa, Glyceria, Graphephonim, Gymnopogon, Gyneriwn, Hackelochloa, Hainardia, Hakonechloa, Haynaldia, Heleochloa, Helictotrichon, Hemarthria, Hesperochloa, Hesperostipa, Heteropogon, Hibanobambusa, Hierochloe, Hilaria, Holcus. Homalocenchrus, Hordewn, Hydrochloa, Hymenachne, Hyparrhenia, Hypogynium, Hystrix, Ichnanthus, Imperata, Indocalamm, Isachne, Ischaemiun, Ixophorus, Koeleria, Korycarpus, Lagiirus, Lamarckia, Lasiacis, Leersia, Leptochloa, Leptochloopsis, Leptocoiyphium, Leptoloma, Leptogon, Lepturus, Lerchejifeldia, Leucopoa, Leymostachys, Leymus, Limnodea, Lithachne, Loliinn, Lophochlaena, Lophochloa, Lophopyrum, Ludolfia, Luziola, Lycurus, Lygeum, Maltea, Manisuris, Megastachya, Melica, Melinis, Mibora, Microchloa, Microlaena, Microstegium, Milium, Miscanthus, Mnesithea, Molinia, Monanthochloe, Monerma, Monroa, Muhlenbergia, Nardus, Nassella, Nazia, Neeragrostis, Neoschischkinia, Neostapfia, Nc-raudia, Noihoholcus, Olyra, Opkla, CpUamcnus, Orcuttia, Oryza, Oryzopsis, Otatea, Oxytenanthera, Panicularia, Panicum, Pappophorum, Parapholis, Pascopynun, Paspalidiwn, Paspalum, Pennisentm, Phalaris, Phalaroides, Phanopyrum, Pharus, Phippsia, Phleuin, Pholiums, Phragmites, Phyllostachys, Piptatherum, Piptochaetium, Pleioblastus, Pleopogon, Pleuraphis, Pleuropogon, Poa, Podagrostis, Polypogon, Polytrias, Psathyrostachys, Pseiidelymus, Pseudoroegneria, Pseudosasa, Ptilagrostis, Pucdnellia, Piicciphippsia, Redfieldia, Reimaria, Reimarochloa, Rhaphis, Rhombolytrum, Rhynchelytrum, Roegneria, Rostraria, Rottboellia, Rytilix, Sacchantm, Sacciolepis, Sasa, Sasaella, Sasamorpha, Savastana, Schedonnardus, Schismus, SchizacJine, Schizachyrium, Schizostachywn, Sclerochloa, Scleropoa, Scleropogon, Scolochloa, Scribneria, Secale, Semianmdinaria. Sesleria, Setaria, Shibataea,<br>
Sieglingia, Sinarundinaria, Sinobambusa, Sinocalamus, Sitanion, Sorghastrum, Sorghum, Spartina, Sphenopholis, Spodiopogon, Sporobolus, Stapfia, Steinchisma, Stenotaphrum, Stipa, Stipagrostis, Stiporyzopsis, Swallenia, Syntherisma, Taeniathenan, Terrellia, Terrelymus, Thamnocalamus, Themeda, Thinopyrwn, Thuarea, Thysanolaena, Torresia, Torreyochloa, Trachynia, Trachypogon, Tragus, Trichachne, Trichloris, Tricholaena, Tnchoneura, Tridens, Triodia, Triplasis, Tripogon, Tripsacum, Trisetobromus, Trisetum, Tnticosecale, Triticum, Tuctona, Uniola, Urachne, Uralepis, Urochloa, Vahlodea, Valota, Vaseyochloa, Ventenata, Vetiveria, Vilfa, Vulpia, Willkommia, Yushania, Zea, Zizania, Zizaniopsis, and Zoysia.<br>
18.	The plant cell of claim 17, wherein the plant is Zea mays, wheat, rice, sorghum,<br>
barley, oat, lawn grass, or rye.<br>
19.	The plant cell of claim 1, wherein the plant cell is from a dicot or monocot.<br>
20.	The plant cell of claim 1, wherein the plant cell is in a hybrid plant comprising a<br>
staygreen potential phenotype.<br>
21.	The plant cell of claim 1, wherein the plant cell is in a plant comprising a male<br>
sterility phenotype.<br>
22.	A plant regenerated from the plant cell of claim 1-.<br>
23.	A knockout plant comprising a staygreen potential phenotype, the staygreen<br>
potential phenotype resulting from a disruption in at least one endogenous ACC synthase<br>
gene, wherein the disruption comprises one or more transposons or one or more point<br>
mutations, and wherein the disruption inhibits expression or activity of at least one ACC<br>
synthase protein compared to a corresponding control plant.<br>
24.	The knockout plant of claim 23, wherein the at least one endogenous ACC synthase<br>
gene comprises a nucleic acid sequence, or complement thereof, comprising at least about<br>
85% sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), or SEQ ID<br>
NO:3 (gACS7), or a complement thereof.<br>
25.	The knockout plant of claim 23, wherein the staygreen potential phenotype of the<br>
transgenic plant comprises increased drought resistance, increased time for maintaining a<br>
photosynthetically active plant, or delayed leaf senescence, compared to the corresponding<br>
control plant.<br>
26.	The knockout plant of claim 23, wherein the knockout plant is a hybrid plant.<br>
27.	The knockout plant of claim 23, wherein the knockout plant is from the Poaceae<br>
(Gramineae) family.<br>
28.	The knockout plant of claim 27, wherein the knockout plant is Zea mays, wheat,<br>
rice, sorghum, barley, oat, lawn grass, or rye.<br>
29.	A transgenic knockout plant comprising a staygreen potential phenotype, said<br>
staygreen potential phenotype resulting from at least one introduced transgene which<br>
inhibits ethylene synthesis, wherein said at least one introduced transgene comprises a<br>
nucleic acid sequence encoding at least one ACC synthase, which nucleic acid sequence<br>
comprises at least about 85% sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2<br>
(gACS6), SEQ ED NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ<br>
ID NO:6 (cACS7) or SEQ ID NO: 10 (CCRA178R), or a subsequence thereof, or a<br>
complement thereof, and is in a configuration that modifies a level of expression or activity<br>
of the at least one ACC synthase.<br>
30.	The transgenic knockout plant of claim 29, wherein the configuration comprises an<br>
antisense, sense or RNA silencing or interference configuration.<br>
31.	The transgenic knockout plant of claim 29, wherein the transgene comprises a<br>
tissue-specific promoter or an inducible promoter.<br>
32.	The transgenic knockout plant of claim 29, wherein the staygreen potential<br>
phenotype of the transgenic plant comprises increased drought resistance, increased time for<br>
maintaining a photosynthetically active plant, or delayed leaf senescence, compared to a<br>
corresponding control plant.<br>
33.	The transgenic knockout plant of claim 29, wherein the plant is from the Poaceae<br>
(Gramineae) family.<br>
34.	The transgenic knockout plant of claim 33, wherein the plant is Zea mays, wheat,<br>
rice, sorghum, barley, oat, lawn grass, or rye.	35.	A transgenic plant comprising a staygreen potential phenotype, said staygreen<br>
potential phenotype resulting from at least one introduced transgene which inhibits ethylene<br>
synthesis, wherein said at least one introduced transgene comprises a nucleic acid sequence<br>
encoding a subsequence of at least one ACC synthase, which at least one ACC synthase<br>
comprises at least about 85% sequence identity to SEQ ID NO:7 (pACS2), SEQ ID NO:8 (pACS6), SEQ ID NO.:9 (pACS7), or SEQ ID NO: 11 (pCCRA17SR), is in an RNA silencing or interference configuration, and modifies a level of expression or activity of the at least one ACC synthase.<br>
36.	The transgenic plant of claim 35, wherein the staygreen potential phenotype of the<br>
transgenic plant comprises increased drought resistance, increased time for maintaining a<br>
photosynthetically active plant, or delayed leaf senescence, compared to a corresponding<br>
control plant.<br>
37.	The transgenic plant of claim 35, wherein the plant is from the Poaceae<br>
(Grannneae) family.<br>
38.	The transgenic plant of claim 37, wherein the plant is Zea mays, wheat, rice,<br>
sorghum, barley, oat, lawn grass or rye.<br>
39.	An isolated or recombinant polynucleotide comprising a member selected from the<br>
group consisting of:<br><br>
(a)	a polynucleotide, or a complement thereof, comprising at least about 85%<br>
sequence identity to SEQ ID NO: 1 (gACS2), SEQ ID NO:2 (gACS6), SEQ<br>
ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ<br>
ID NO:6 (cACS7), or SEQ ID NO: 10 (CCRA178R), or a subsequence<br>
thereof;<br>
(b)	a polynucleotide, or a complement thereof, encoding a polypeptide sequence<br>
of SEQ ED NO:7 (pACS2), SEQ ED NO:8 (pACS6), SEQ ID NO.:9<br>
(pACS7), or SEQ ID NO: 11 (pCCRA178R), or a subsequence thereof, or a<br>
conservative variation thereof; and,<br>
(c)	a polynucleotide, or a complement thereof, that hybridizes under stringent<br>
conditions over substantially the entire length of a polynucleotide<br>
subsequence comprising at least 100 contiguous nucleotides of SEQ ID<br>
NO: 1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID<br>
NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ DD NO:6 (cACS7), or SEQ ID<br>
NO:10(CCRA178R).<br>
40.	The isolated or recombinant polynucleotide of claim 39, wherein the polynucleotide<br>
inhibits ethylene production when expressed in a plant.<br>
41.	The isolated or recombinant polynucleotide of claim 39, further comprising an<br>
expression cassette comprising a promoter operably linked to the polynucleotide.<br>
42.	The isolated or recombinant polynucleotide of claim 41, wherein the promoter<br>
comprises a constitutive promoter.<br>
43.	The isolated or recombinant polynucleotide of claim 41, wherein the promoter<br>
comprises an inducible promoter.<br>
44.	The isolated or recombinant polynucleotide of claim 41, wherein the promoter<br>
comprises a tissue-specific promoter.<br>
45.	The isolated or recombinant polynucleotide of claim 44, wherein the tissue-specific<br>
promoter comprises a leaf-specific promoter or an anther-specific promoter.<br>
46.	The isolated or recombinant polynucleotide of claim 41, wherein the polynucleotide<br>
is linked to the promoter in an antisense orientation.<br>
47.	The isolated or recombinant polynucleotide of claim 41, wherein the polynucleotide<br>
is linked to the promoter in a sense orientation.<br>
48.      The isolated or recombinant polynucleotide of claim 41, wherein the polynucleotide is in an RNA silencing or interference configuration.<br>
49.	The isolated or recombinant polynucleotide of claim 39, further comprising a vector<br>
comprising a promoter operably linked to the polynucleotide.<br>
50.	The isolated or recombinant polynucleotide of claim 49, wherein the vector is a viral<br>
vector.<br>
51.	A knockout plant comprising a male sterility phenotype, the male sterility phenotype<br>
resulting from at least one disruption in at least one endogenous ACC synthase gene, the<br>
disruption inhibiting expression or activity of at least one ACC synthase protein compared<br>
to a corresponding control plant.<br>
52.	The knockout plant of claim 51, wherein the at least one disruption results in<br>
reduced ethylene production by the knockout plant as compared to the control plant.<br>
53.	The knockout plant of claim 51, wherein the at least one disruption comprises one or<br>
more transposons, wherein the one or more transposons are in the at least one endogenous<br>
ACC synthase gene.<br>
54.	The knockout plant of claim 51, wherein the at least one disruption comprises one or<br>
more point mutations, wherein the one or more point mutations are in the at least one<br>
endogenous ACC synthase gene.<br>
55.	The knockout plant of claim 51, wherein the at least one disruption is introduced<br>
into the knockout plant by introducing at least one polynucleotide sequence comprising one<br>
or more subsequences of an ACC synthase nucleic acid sequence configured for RNA<br>
silencing or interference.<br>
56.	The knockout plant of claim 51, wherein the male sterility phenotype comprises<br>
reduced pollen shedding by the knockout plant as compared to the control plant.<br>
57.	A transgenic knockout plant comprising a male sterility phenotype, said male<br>
sterility phenotype resulting from at least one introduced transgene which inhibits ethylene<br>
synthesis, wherein said at least one introduced transgene comprises a nucleic acid sequence<br>
encoding at least one ACC synthase, which nucleic acid sequence comprises at least about<br>
85% sequence identity to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6), SEQ ID NO:3<br>
(gACSV), SEQ ID NO:4 (cACS2), SEQ ID NO:5 (cACS6), SEQ ED NO:6 (cACS7) or SEQ<br>
ED NO: 10 (CCRA178R), or a subsequence thereof, or a complement thereof, and is in a<br>
configuration that modifies a level of expression or activity of the at least one ACC<br>
synthase.<br>
58.	The transgenic knockout plant of claim 57, wherein the configuration comprises an<br>
antisense, sense or RNA silencing or interference configuration.<br>
59.	The transgenic knockout plant of claim 57, wherein the transgene comprises a<br>
tissue-specific promoter or an inducible promoter.<br>
60.	A method of inhibiting ethylene production in a plant, the method comprising:<br>
inactivating one or more ACC synthase genes in the plant, wherein the one or more ACC synthase genes encode one or more ACC synthases, wherein at least one of the one or more ACC synthases comprises at least about 85% identity to SEQ ED NO:7 (pACS2), SEQ ED NO:8 (pACS6), SEQ ED NO:9 (pAC7) or SEQ ED NO: 11 (pCCRAHSR).<br>
61.	The method of claim 60, wherein at least one of the one or more ACC synthase<br>
genes is at least about 85% identical to SEQ ID NO:1 (gACS2), SEQ ID NO:2 (gACS6) or<br>
SEQ ID N0:3 (gAC7), or a complement thereof.<br>
62.	The method of claim 60, wherein the inactivating step comprises introducing one or<br>
more mutations into an ACC synthase gene sequence, wherein the one or more mutations in<br>
the ACC synthase gene sequence comprise one or more transposons, thereby inactivating<br>
the one or more ACC synthase genes compared to a corresponding control plant.<br>
63.	The method of claim 62, wherein the one or more mutations comprise a<br>
homozygous disruption in the one or more ACC synthase genes.<br>
64.	The method of claim 62, wherein the one or more mutations comprise a<br>
heterozygous disruption in the one or more ACC synthase genes.<br>
65.	The method of claim 62, wherein the one or more mutations are introduced by a<br>
sexual cross.<br>
66.	The method of claim 60, wherein the inactivating step comprises introducing one or<br>
more mutations into an ACC synthase gene sequence, wherein the one or more mutations in<br>
the ACC synthase gene sequence comprise one or more point mutations, thereby<br>
inactivating the one or more ACC synthase genes compared to a corresponding control<br>
plant.<br>
67.	The method of claim 60, wherein the inactivating step comprises:<br><br>
(a)	introducing into the plant at least one polynucleotide sequence, wherein the at<br>
least one polynucleotide sequence comprises a nucleic acid encoding one or<br>
more ACC synthases, or a subsequence thereof, and a promoter, which promoter<br>
functions in plants to produce an RNA sequence; and,<br>
(b)	expressing the at least one polynucleotide sequence, thereby inactivating the one<br>
or more ACC synthase genes compared to a corresponding control plant.<br>
68.	The method of claim 67, wherein the at least one polynucleotide sequence is<br>
introduced by electroporation, micro-projectile bombardment or Agrobacterium-mediated<br>
transfer.<br>
69.	The method of claim 67, wherein the polynucleotide is linked to the promoter in a<br>
sense orientation.<br>
70.      The method of claim 67, wherein the polynucleotide is linked to the promoter in an antisense orientation.<br>
71.	The method of claim 67, wherein the polynucleotide is configured for RNA<br>
silencing or interference.<br>
72.	The method of claim 67, wherein the promoter comprises a tissue-specific promoter<br>
or an inducible promoter.<br>
73.	The method of claim 60, wherein the plant is from the Poaceae (Gramineae) family.<br>
74.	The plant of claim 73, wherein the plant is Zi?a mays, wheat, rice, sorghum, barley,<br>
oat, lawn grass, or rye.<br>
75.	A plant produced by the method of claim 60.<br>
76.	A method for modulating staygreen potential in a plant, the method comprising:<br><br>
a)	selecting at least one ACC synthase gene to mutate, thereby providing at least one<br>
desired ACC synthase gene;<br>
b)	introducing a mutant form of the at least one desired ACC synthase gene into the<br>
plant; and,<br>
c)	expressing the mutant form, thereby modulating staygreen potential in the plant.<br>
77.	The method of claim 76, wherein the staygreen potential of the plant comprises:<br>
(a)	a reduction in the production of at least one ACC synthase specific mRNA;<br>
(b)	a reduction .in the production of an ACC synthase;<br>
(c)	a reduction in the production of ethylene;<br>
(d)	a delay in leaf senescence;<br>
(e)	an increase in drought resistance;<br>
(f)	an increased time in maintaining photosynthetic activity;<br>
(g)	an increased transpiration;<br>
(h) an increased stomatal conductance;<br>
(i) an increased CO2 assimilation;<br>
(j) an increased time in maintaining CO2 assimilation; or,<br>
(k) any combination of (a)-(j); compared to a corresponding control plant.<br>
78.	The method of claim 76, wherein the selecting the at least one ACC synthase gene<br>
comprises determining a degree of staygreen potential desired,<br>
79.	The method of claim 78, wherein the degree of staygreen potential desired is weak,<br>
moderate or strong.<br>
80.	The method of claim 76, wherein the ACC synthase gene encodes an ACC synthase<br>
selected from the group consisting of: SEQ ED NO:7 (pACS2), SEQ ID NO:8 (pACS6),<br>
SEQ ID NO:9 (pAC7) and SEQ ID NO: 11 (pCCRA178R).<br>
81.	The method of claim 76, wherein the mutant form comprises a heterozygous<br>
mutation in the at least one ACC synthase gene.<br>
82.	The method of claim 76, wherein the mutant form comprises a homozygous<br>
mutation in the at least one ACC synthase gene.<br>
83.	The method of claim 76, wherein the mutant form comprises a subsequence of the at<br>
least one desired ACC synthase gene in an antisense, sense or RNA silencing or interference<br>
configuration.<br>
84.	The method of claim 76, wherein the mutant form is introduced by Agrobacterium-<br>
mediated transfer, electroporation, micro-projectile bombardment, or a sexual cross.<br>
85.	The method of claim 76, wherein the plant is from the Poaceae (Gramineae) family.<br>
86.	The plant of claim 85, wherein the plant is Zea mays, wheat, rice, sorghum, bra-ley,<br>
oat, lawn grass, or rye.<br>
87.	A plant produced by the method of claim 76.<br>
88.	A kit for modulating staygreen potential in a plant, the kit comprising: a container<br>
containing at least one polynucleotide sequence comprising a nucleic acid sequence,<br>
wherein the nucleic acid sequence is at least about 85% identical to SEQ ID NO:1 (gACS2),<br>
SEQ ID N0:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5<br>
(cACS6), SEQ ED NO:6 (cAC7) or SEQ ED NO: 10 (CCRA178R), or a subsequence thereof,<br>
or a complement thereof.<br>
89.	The kit of claim 88, wherein said kit further comprises instructional materials for the<br>
use of the at least one polynucleotide sequence to control staygreen potential in a plant.<br>
90.	A kit for modulating male sterility in a plant, the kit comprising: a container<br>
containing at least one polynucleotide sequence comprising a nucleic acid sequence,<br>
wherein the nucleic acid sequence is at least about 85% identical to SEQ ID NO:1 (gACS2),<br>
SEQ ID NO:2 (gACS6), SEQ ID NO:3 (gACS7), SEQ ID NO:4 (cACS2), SEQ ID NO:5<br>
(cACS6), SEQ ID NO:6 (cAC7) or SEQ ID NO: 10 (CCRA17SR), or a subsequence thereof,<br>
or a complement thereof.<br>
91.	The kit of claim 90, wherein said kit further comprises instructional materials for the<br>
use of the at least one polynucleotide sequence to control male sterility in a plant.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtQWJzdHJhY3QtKDEwLTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Abstract-(10-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">407-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtQ2xhaW1zLSgxMC0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Claims-(10-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgwNi0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Correspondence Others-(06-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMC0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Correspondence Others-(10-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNC0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Correspondence Others-(14-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyMS0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Correspondence Others-(21-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LURFTE5QLTIwMDYtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">407-DELNP-2006-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZGVzY3JpcGl0b24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-descripiton (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtRHJhd2luZ3MtKDEwLTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Drawings-(10-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">407-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LURFTE5QLTIwMDYtRm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">407-DELNP-2006-Form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtRm9ybS0yLSgxMC0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Form-2-(10-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtRm9ybS0zLSgwNi0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Form-3-(06-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtRm9ybS0zLSgxNC0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Form-3-(14-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtR1BBLSgyMS0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-GPA-(21-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">407-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtcGN0LTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtcGN0LTMyNi5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-pct-326.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtcGN0LTM3My5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA3LWRlbG5wLTIwMDYtUGV0aXRpb24tMTM3LSgwNi0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">407-delnp-2006-Petition-137-(06-08-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="257902-a-composition-for-polythiourethane-optical-materials-and-a-process-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="257904-her2-antibody-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>257903</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>407/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>22-Nov-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Nov-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jan-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PIONEER HI-BRED INTERNATIONAL INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 CAPITAL SQUARE, 400 LOCUST STREET, DES MOINES, IO 50309 U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GALLIE DANIEL R.</td>
											<td>3121 NEWELL, DRIVE, RIVERSIDE. CA 92507 U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YOUNG TODD</td>
											<td>1522 S. SAGEBRUSH ROAD, PALM SPRINGS, CA 92264 U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MEELEY ROBERT</td>
											<td>3214 EDWARDS AVENUE, DES MOINES, IA 50312 U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>B01J</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/020180</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-06-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/480,861</td>
									<td>2003-06-23</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script data-cfasync="false" src="../cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/257903-engineering-single-gene-controlled-staygreen-potential-into-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:08:32 GMT -->
</html>
