<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258119-a-method-for-improving-plant-growth-characteristics by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:32:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258119:A METHOD FOR IMPROVING PLANT GROWTH CHARACTERISTICS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR IMPROVING PLANT GROWTH CHARACTERISTICS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Abstract The present invention concerns a method for improving plant growth characteristics by increasing activity of DP protein in shoot tissue or by increasing expression of a DP-encoding gene in shoot tissue of a plant. The invention also concerns transgenic plants having improved growth characteristics, which plants are made by the methods of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Plants having improved growth characteristics and a method for<br>
making the same<br>
The present invention concerns a method for improving plant growth characteristics. More specifically, the present invention concerns a method for improving plant growth characteristics by increasing expression in plant shoot tissue of an E2F Dimerisation Partner (DP)-encoding gene and/or by increasing activity, in plant shoot tissue, of a DP protein. The present invention also concerns plants transformed with a DΒ-encoding gene, under the control of a shoot-preferred control element, which plants have improved growth characteristics relative to corresponding wild-type plants.<br>
Given the ever-increasing world population, it remains a major goal of research to improve the efficiency of agriculture. Conventional means for crop and horticultural improvements utilise selective breeding techniques to identify plants having desirable characteristics. However, such selective breeding techniques have several drawbacks, namely that these techniques are typically labour intensive and result in plants that often contain heterogenous genetic components that may not always result in the desirable trait being passed on from parent plants. In contrast, advances in molecular biology have allowed mankind to more precisely manipulate the germplasm of plants. Genetic engineering of plants entails the isolation and manipulation of genetic material (typically in the form of DNA or RNA) and the subsequent introduction of that genetic material into a plant. Such technology has led to the development of plants having various improved economic, agronomic or horticultural traits. Traits of particular economic interest include high yield and biomass.<br>
The ability to improve one or more plant growth characteristics would have many applications in areas such as crop enhancement, plant breeding, production of ornamental plants, arboriculture, horticulture, forestry and in the production of algae or plants (for use as bioreactors for example, for the production of pharmaceuticals, such as antibodies or vaccines, or for the bioconversion of organic waste, or for use as fuel, in the case of high-yielding algae and plants).<br>
It has now been found that increasing expression of a DΒ-encoding gene and/or increasing activity of a DP protein in shoot-tissue of a plant, gives plants having improved growth characteristics relative to corresponding wild-type plants.<br><br>
DP proteins are widely conserved proteins and are involved in control of the cell cycle (Gutierrez et a/. (2002) Current Opinion in Plant Biology 5: 480-486). DP factors act together with E2F factors to form a heterodimer, capable of initiating transcription of S-phase specific genes. The identification of E2F factors, DP factors and E2F-DΒ-like (DEL) factors was reported by Magyar et a/., 2000 (FEBS letters, 486: 79-97). Based on sequence comparison, Arabidopsis genes encoding these proteins were grouped into distinct categories as described in Vandepoele et aL, 2002 (Plant Cell 14(4): 903-16), which is incorporated herein by reference as if fully set forth. Furthermore, the structural characteristics of typical DP proteins were reported by Magyar et a/., which is also incorporated herein by reference as if fully set forth. For example, Figures 3A and B of Magyar et aL show the location of a characteristic DNΑ-binding domain and a dimerisation domain in Arabidopsis DP proteins. Figure 5 of Vandepoele et al. nicely illustrates that DP proteins are distinct from related proteins such as E2F factors and DELs on account of the presence of one DNΑ-binding domain and one dimerisation domain.<br>
Published International patent application WO00/47614 in the name of Pioneer Hi-Bred suggests that controlling DP expression using tissue-specific or cell-specific promoters provides a differential growth characteristic. In particular, it suggests that (i) using a seed-specific promoter will stimulate cell division rate and result in increased seed biomass; (ii) using a strongly-expressed early tassel-specific promoter will enhance development of this entire reproductive structure; and (Hi) that using a root-specific promoter will result in larger roots and faster growth (i.e., more biomass accumulation). However, no plants having such differential growth characteristics were generated or exemplified in ffte application.<br>
The later filed International patent application WO01/21644, in the name of Consejo Superior De Investigaciones Cientificas, merely suggests, without exemplification, that plant growth may be controlled by expression of a recombinant DP. Despite the statement “particularly useful are nucleic acids of which the expression is controlled using a tissue-specific promoter or a chemically-inducible promoter”, there was no mention or exemplification in the application of plants transformed with such nucleic acids and having improved growth characteristics.<br>
Despite the above suggestions, no improved transgenic plants have been generated using the teachings of the prior art, indicating that using DP as suggested in the prior art is ineffective in improving plant growth characteristics.<br><br>
4 Unexpectedly, it has now been found that plant growth characteristics may be improved by increasing activity of a DP protein in plant shoot-tissue and/or by increasing expression in plant shoot tissue of a DΒ-encoding gene.<br>
The present invention provides a method for improving plant growth characteristics relative to corresponding wild-type plants, comprising increasing activity of a DP protein or a homologue thereof specifically in plant shoot tissue and/or by increasing expression of a DΒ-encoding gene or a functional variant thereof in plant shoot tissue.<br>
Advantageously, performance of the method according to the present invention gives plants having a variety of improved growth characteristics relative to corresponding wild-type plants, especially increased biomass. The improved growth characteristics may be stable and inheritable to further generations.<br>
The term “growth characteristic” encompasses increased biomass, among other features related to growth as detailed hereinafter.<br>
The term “biomass” refers to the amount of biological material produced. An increase in biomass may be in one or more parts of a plant relative to the biomass of corresponding reference plants, for example relative to the biomass of corresponding wild-type plants. The plants according to the invention are characterised by increased aboveground biomass, which is particularly important for crop plants grown for their vegetative tissues. For sflage com, for example, typical parameters of’economic value are aboveground biomass and the energy content of leaves; for trees and sugarcane, a typical parameter of economical value is aboveground biomass of the stems.<br>
The term “increased biomass” as used herein may also encompass increased yield, particularly increased seed yield.<br>
The term “increased yield” as defined herein is taken to mean an increase in any one or more of the following, each relative to corresponding wild type plants: (i) increased biomass (weight) of one or more parts of a plant, particularly aboveground (harvestable) parts, increased root biomass or increased biomass of any other harvestable part; (ii) increased seed yield, which may result from an increase in seed biomass (seed weight) and which may be an increase in the seed weight per plant or on an individual seed basis, and which increase in seed weight may be due to altered seed dimensions, such as seed length and/or seed width and/or seed area; (iii) increased number of (filled) seeds; (iv) increased seed size, which may also influence<br><br>
‘the composition of seeds; (v) increased seed volume, which may also influence the composition of seeds; (vi) increased harvest index, which is expressed as a ratio of the yield of harvestable parts, such as seeds, over the total biomass; and (vii) increased thousand kernel weight (TKW), which is extrapolated from the number of filled seeds counted and their total weight An increased TKW may result from an increased seed size and/or seed weight.<br>
Taking com as an example, a yield increase may be manifested as one or more of the following: increase in the number of plants per hectare or acre, an increase in the number of ears per plant, an increase in the number of rows, number of kernels per row, kernel weight, thousand kernel weight, ear length/diameter, among others. Taking rice as an example, a yield increase may be manifested by an increase in one or more of the following: number of plants per hectare or acre, number of panicles per plant, number of spikelets per panicle, number of flowers per panicle, increase in the seed filling rate, increase in thousand kernel weight, among others. An increase in yield may also result in modified architecture, or may occur as a result of modified architecture.<br>
Since the transgenic plants according to the present invention have increased biomass relative to corresponding wild type plants, it is likely that these plants exhibit an increased growth rate (during at least part of their life cycle), relative to the growth rate of corresponding wild type plants at a corresponding stage in their life cycle. The increased growth rate may be specific to one or more parts of a plant (including seeds), or may be throughout substantially the whole plant. A plant having an increased growth rate may even exhibit early flowering. The increase in growth rate may take place at one or more stages in the1 life cycle of a plant or during substantially the whole plant life cycle. Increased growth rate during the early stages in the life cycle of a plant may reflect enhanced vigour. The increase in growth rate may alter the harvest cycle of a plant allowing plants to be sown later and/or harvested sooner than would otherwise be possible. If the growth rate is sufficiently increased, it may allow for the sowing of further seeds of the same plant species (for example sowing and harvesting of rice plants followed by sowing and harvesting of further rice plants all within one conventional growing period). Similarly, if the growth rate is sufficiently increased, it may allow for the sowing of further seeds of different plants species (for example the sowing and harvesting of rice plants followed by, for example, the sowing and optional harvesting of soy bean, potatoes or any other suitable plant). Harvesting additional times from the same rootstock in the case of some plants may also be possible. Altering the harvest cycle of a plant may lead to an increase in annual biomass production per acre (due to an increase in the number of times (say in a year) that any particular plant may be grown and harvested). An increase in growth rate may also allow for the cultivation of transgenic plants in a wider geographical area than their wild-type<br><br>
counterparts, since the territorial limitations for growing a crop are often determined by adverse environmental conditions either at the time of planting (early season) or at the time of harvesting (late season). Such adverse conditions may be avoided if the harvest cycle is shortened. The growth rate may be determined by deriving various parameters from growth curves plotting growth experiments, such parameters may be: T-Mid (the time taken for plants to reach 50% of their maximal size) and T-90 (time taken for plants to reach 90% of their maximal size), amongst others.<br>
The term “growth characteristic” as used herein, also encompasses plant architecture. The plants according to the invention exhibit altered architecture, which is manifested in altered shape and size of aboveground parts due to the increased biomass. This characteristic may be advantageous for many ornamental plants. The term “architecture” as used herein encompasses the appearance or morphology of a plant, including any one or more structural features or combination of structural features, such as shape, size, number, position, texture, arrangement and pattern of cells, tissues, organs or groups of cells, tissues or organs of a plant Particularly preferred are plants having any one or more of: increased number, size, shape of tillers (or corresponding plant part); increased number of branches and/or leaves.<br>
An improvement in any of the aforementioned growth characteristics occurs whether the plant is under non-stress conditions or whether the plant is exposed to various stresses compared to control plants. Plants typically respond to exposure to stress by growing more slowly. In conditions of severe stress, the plant may even stop growing altogether. Mild stress on the other hand is defined herein as being any stress to which a plant is exposed which does not result in the plant ceasing to grow altogether. Due to advances in agricultural practices (irrigation, fertilization, pesticide treatments) severe stresses are not often encountered in cultivated crop plants. As a consequence, the compromised growth induced by mild stress is often an undesirable feature for agriculture. Mild stresses are the typical stresses to which a plant may be exposed. These stresses may be the everyday biotic and/or abiotic (environmental) stresses to which a plant is exposed. Typical abiotic or environmental stresses include temperature stresses caused by atypical hot or cold/freezing temperatures; salt stress; water stress (drought or excess water). Abiotic stresses may also be caused by chemicals. Biotic stresses are typically those stresses caused by pathogens, such as bacteria, viruses, fungi and insects.<br>
The abovementioned growth characteristics may advantageously be modified in a variety of plant species.<br><br>
‘ The term “plant” as used herein encompasses whole plants, ancestors and progeny of the<br>
plants and plant parts, including seeds, shoots, stems, roots (including tubers), and plant cells,<br>
tissues and organs. The term “plant” also therefore encompasses suspension cultures,<br>
embryos, meristematic regions, callus tissue, leaves, seeds, roots, shoots, gametophytes,<br>
sporophytes, pollen, and microspores. Plants that are particularly useful in the methods of the<br>
invention include all plants which belong to the superfamily Viridiplantae, in particular<br>
monocotyledonous and dicotyledonous plants including a fodder or forage legume, ornamental<br>
plant, food crop, tree, or shrub selected from the list comprising Acacia spp., Acer spp.,<br>
Actinidia spp.,Aesculus spp.f Agathis australis, Aibizia amara, Alsophila tricolor, Andropogon<br>
spp.t Arachis spp, Areca catechu, Astelia fragrans, Astragalus cicer, Baikiaea plurijuga, Betula<br>
spp., Brassica spp., Bruguiera gymnorrhiza, Burkea africana, Butea frondosa, Cadaba<br>
farinosa, Calliandra spp, Camellia sinensis, Canna indica, Capsicum spp, Cassia spp.,<br>
Centroema pubescens, Chaenomeles spp.,Cinnamomum cassia, Coffea arabica,<br>
Colophospermum mopane, Coronillia varia, Cotoneaster serotina, Crataegus spp., Cucumis<br>
spp., Cupressus spp., Cyathea dealbata, Cydonia oblonga, Cryptomeria japonica,<br>
Cymbopogon spp., Cynthea dealbata, Cydonia oblonga, Dalbergia monetaria, Davallia<br>
divaricata, Desmodium spp., Dicksonia squarosa, Diheteropogon amplectens, Dioclea spp,<br>
Dolichos spp., Dorycnium rectum, Echinochloa pyramidalis, Ehrartia spp., Eleusine coracana,<br>
Eragrestis spp., Erythrina spp., Eucalyptus spp., Euclea schimperi, Eulalia villosa, Fagopyrum<br>
spp., Feijoa sellowiana, Fragaria spp.r Flemingia spp, Freycinetia banksii, Geranium<br>
thunbergii, Ginkgo biloba, Glycine javanica, Gliricidia spp, Gossypium hirsutum, Grevillea spp.,<br>
Guibourtia coleosperma, Hedysarum spp., Hemarthia altissima, Heteropogon contortus,<br>
Hordeum vulgare, Hyparrhenia rufa, Hypericum erectum, Hyperthelia dissoluta, Indigo<br>
incarnate, Iris spp., Leptarrhena pyroiifoUa, Lespediza spp., Lettuca spp., Leucaena<br>
leucocephala, Loudeiia simplex, Lotonus bainesii, Lotus spp., Macrotyloma axSlare, Malus<br>
spp., Manihot esculenta, Medicago sativa, Metasequoia glyptostroboides, Musa sapientum,<br>
Nicotianum spp., Onobrychis spp.t Omithopus spp., Oryza spp., Peltophorum africanum,<br>
Pennisetum spp., Persea gratissima, Petunia spp., Phaseolus spp., Phoenix canariensis,<br>
Phormium cookianum, Photinia spp., Picea glauca, Pinus spp., Pisum sativum, Podocarpus<br>
totara, Pogonarthria fleckii, Pogonarthria squamosa, Populus spp., Prosopis cineraria,<br>
Pseudotsuga menziesii, Pterolobium stellatum, Pyrus communis, Quercus spp., Rhaphiolepsis<br>
umbellate, Rhopalostylis sapida, Rhus natalensis, Ribes grossularia, Ribes spp., Robinia<br>
pseudoacacia, Rosa spp., Rubus spp., Salix spp., Schyzachyrium sanguineum, Sciadopitys<br>
verticillata, Sequoia sempervirens, Sequoiadendron giganteum, Sorghum bicolor, Spinacia<br>
spp., Sporobolus Umbriatus, Stiburus alopecuroides, Stylosanthos humilis, Tadehagi spp,<br>
Taxodium distichum, Themeda triandra, Trifblium spp., TriUcum spp., Tsuga heterophylla,<br>
Vaccinium spp., Vicia spp.Vitis vinifera, Watsonia pyramidata, Zantedeschia aethiopica, Zea<br><br>
mays, amaranth, artichoke, asparagus, broccoli, Brussels sprouts, cabbage, canola, carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, rice, soybean, straw, sugar beet, sugar cane, sunflower, tomato, squash tea, trees, grasses (including forage grass) and algae, amongst others.<br>
According to a preferred feature of the present invention, the plant is a crop plant, such as phytoplankton, soybean, sunflower, canola, rapeseed, cotton, alfalfa, tomato, potato, lettuce, tobacco, papaya, squash, poplar, eucalyptus, pine, leguminosa, flax, lupinus and sorghum. According to a further preferred embodiment of the present invention, the plant is a monocotyledonous plant, such as sugarcane, onion or bamboo. Further preferably the plant is a cereal, such as rice, maize (including forage com), wheat, barley, millet, oats and rye.<br>
The term “DP” means E2F Dimerisation Partner. As the name suggests, a DP protein is capable of dimerizing with an E2F/DEL transcription factor. This may be tested by, for example, a Two-Hybrid assay as described in Magyar et a/., 2000 (FEBS letters, 486: 79-97) or by co-immunoprecipitation. The structural characteristics of typical DP proteins were reported by Magyar et a/., which reference is incorporated herein as if fully set forth. For example, Figures 3A and B of Magyar et a/, show the location of a characteristic DNΑ-binding domain and a dimerisation domain in Arabidopsis DP proteins. Figure 5 of Vandepoele et a/., 2002 (Plant Cell 14(4): 903-16), nicely illustrates that DP proteins are distinct from related proteins such as E2F factors and DELs on account of the presence of one DNΑ-binding domain and one dimerisation domain. Figure 3 shows the location of these domains iri the Arabidopsis thaliana DPb sequence. Based on the knowledge available, a person skilled in the art would readily be able to identify a DP protein.<br>
According to a preferred feature of the invention, the DP protein is as represented by SEQ ID NO 2 or homologues of SEQ ID NO 2. Specific examples of such homologues include Arabidopsis thaliana DP proteins as described Magyar et a/., 2000 (FEBS, 486(1): 79-87), Triticurn aestivum DP proteins as described in Ramirez-Parra &amp; Gutierrrez, 2000 (FEBS, 86(1): 73-8) and Impatiens, soybean and corn DP proteins as described in published International patent application WO99/53075 in the name of Du Pont.<br>
Preferably, the DP protein or homologue thereof as defined herein refers to a polypeptide having in increasing order of preference at least 40%, 41%, 42%, 43%, 44%, 45%, 46%, 47%, 48%, 49%, 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%,<br><br>
‘80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% sequence identity to a DP protein, for example, to any one of SEQ ID NO 2, 4, 13, 15, 17, 19, 21 and 23.<br>
DP proteins of Arabidopsis thaliana may be subdivided into two different classes, as shown in Vandepoele et a/., 2002 (Plant Cell.,14(4): 903-16), DPa and DPb. The members of both classes are also encompassed by the term “homologue” as used herein. Advantageously, these different classes of DP proteins, or their encoding nucleic acids, may be used in the methods of the present invention.<br>
The DP nucleic acid or DP protein useful in the methods of the invention is preferably obtained from a plant, preferably from a dicotyledoneous plant, further preferably from the family Brassicaceae, more preferably from Arabidopsis thaliana. According to a further embodiment, the DP polypeptide is a DPb polypeptide. A “DPb* refers to a protein that is more closely related to AtDPb than to AtDPa. Whether a query sequence can be classified as a DPa or as a DPb may be determined by calculating the percentage sequence identity or by establishing the presence of conserved motifs as described hereinafter. Another method for identifying a query sequence as a DPa or as a DPb is to introduce the query sequence into a phylogenetic tree, such as the one shown in Figure 5. A DPb protein should cluster closer to AtDPb than to AtDPa.<br>
Preferred DP polypeptides or homologues, useful in the methods of the present invention, are those having, in increasing order of preference, at least 40%, 41%, 42%, 43%, 44%, 45%, 46%, 47%, 48%, 49%, 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% sequence identity to a DP protein, for example, a DP protein as represented by any one of SEQ ID NO 2, 4, 13, 15, 17, 19, 21 and 23. The percentage identity may be calculated over a conserved region typically present in all DP proteins. This region starts from about residues CEKVES (e.g., from position 111 of SEQ ID NO 2) to about FVLKTM (e.g., to position 290 of SEQ ID NO 2); see Figure 3.<br>
Three motifs are particularly conserved in a subclass of DP proteins, which subclass comprises DPb of Arabidopsis thaliana. The consensus sequences for these “DPb” motifs are represented herein by SEQ ID NO 9 (motif 1, LDIXXDDA), SEQ ID NO 10 (motif 2, KKKK/RR) and SEQ ID NO 11 (motif 3, AXGXDK) (see Figure 3).<br><br>
Preferably, these motifs are present in the DP polypeptide or homologues used in the methods of the present invention. Figure 3 shows an alignment of DP proteins with the location of the “DPb” motifs. As can be seen from the alignment, refining the consensus sequences is possible. For example, at position 4 in motif 1 there is a high probability for a Q or an H residue and at position 5 there is a high probability for a G or an A residue. Also in motif 3, at position 2 there is a high probability for a V, T or A residue and at position 4 there is a high probability for a P or an A residue. A person skilled in the art will recognize that a DPb motif may deviate by for example 1 or 2 mismatches from the consensus DPb motifs as represented by SEQ ID NO 9,10 or 11 without losing any functionality.<br>
The aforementioned newly identified “DPb” motifs may also be used to search databases and to identify homologous DPb polypeptides and encoding sequences.<br>
The identification of protein domains, motifs and boxes, would be well within the realm of a person skilled in the art. Protein domain information may be available through PRODOM (http://www.biochem.ud.ac.uk/bsm/dbbrowser/jj/prodomsrchjj.html),	PIR<br>
(http://pir.georgetown.edu/), PROSITE (http://au.expasy.org/PROSITE/) or pFAM (http://pFAM.wustl.edu/) databases. Software programs designed for such domain searching include, but are not limited to, MotifScan, MEME, SIGNALSCAN, and GENESCAN. MotifScan is a preferred software program and is available at (http://hits.isb-sib.ch/cgi-bin/PFSCAN, which program uses the protein domain information of PROSITE and pFAM. A MEME algorithm (Version 3.0) may be found in the GCG package or at http://www.sdsc.edd/MEME/meme. SIGNALSCAN version 4.0 is available at http://biosci.cbs.umn.edu/software/sigscan.html. GENESCAN may be found at http://gnomic.stanford.edu/GENESCANW.html.<br>
A DP polypeptide or homoiogue may be found in (public) sequence databases. Methods for the alignment and identification of DP protein homologues in sequence databases are well known in the art. Such methods, involve screening sequence databases with the sequences provided by the present invention, for example, SEQ ID NO 2, 4,13, 15, 17, 19, 21 and 23 (or SEQ ID NO 1). Different search algorithms and software for the alignment and comparison of sequences are well known in the art and include for example GAP, BESTFIT, BLAST, FASTA and TFASTA. Preferably the BLAST software is used, which calculates percent sequence identity and performs a statistical analysis of the similarity between the sequences. The suite of programs referred to as BLAST programs has 5 different implementations: three designed for nucleotide sequence queries (BLASTN, BLASTX, and TBLASTX) and two designed for protein sequence queries (BLASTP and TBLASTN) (Coulson, Trends in Biotechnology: 76-80,<br><br>
1994; Birren et a/., GenomeAnalysis, 1: 543, 1997). The software for performing BLAST analysis is publicly available through the National Centre for Biotechnology Information. Useful sequence databases include, but are not limited, to Genbank (http://www.ncbi.nlm.nih.gov/web/Genbank), the European Molecular Biology Laboratory Nucleic acid Database (EMBL) (http.7w.ebi.ac.uk/ebkJocs/embl»db.html) or versions thereof or the MIPS database (http://mips.gsf.de/).<br>
Preferred DP polypeptides useful in the methods of the present invention have at least 40% sequence identity with any one of SEQ ID NO 2,4,13,15,17,19, 21 and 23. The percentage of sequence identity, may be calculated using a pairwise global alignment program implementing the algorithm of Needleman-Wunsch (J. MoL Biol. 48: 443-453, 1970), which maximizes the number of matches and minimises the number of gaps. For the calculation of the abovementioned percentages, the program needle (EMBOSS package) may be used with a gap opening penalty of 10 and gap extension penalty of 0.1. For proteins, the blosum62 matrix with a word length of 3 is preferably used. For nucleic acids, the program needle uses the matrix “DNΑ-fuir, with a word-length of 11, as provided by the EMBOSS package. The Needleman-Wunsch algorithm is best suited for analysing related protein sequences over their full length. Alternatively, analysing related proteins and determining the percentage of sequence identity as mentioned above, may be calculated in the conserved region, domains or motifs as mentioned above.<br>
Examples of polypeptides falling under the definition of “a DP polypeptide or homologue thereof are Arabidopsis thaliana DPb (SEQ ID NO 2 and corresponding encoding sequence SEQ ID NO 1). Other examples of DP proteins are given in Figure 3, together with their Genbank accession number, their coding sequence, and their protein sequence, represented by SEQ ID NO 12 to 23. The genome sequences of Arabidopsis thaliana and Oryza sativa are now available in public databases such as Genbank and other genomes are currently being sequenced. Therefore, it is expected that further homologues will readily be identifiable by sequence alignment with any one of SEQ ID NO 1 to 4 or 12 to 23 using the programs BLASTX or BLASTP or other programs.<br>
Despite what may appear to be a relatively low sequence homology of “at least 40% identity”, DP proteins are highly conserved with all of them having a DNΑ-binding domain and a dimerisation domain. It is to be understood that the term DP polypeptide or homologue thereof is not to be limited to the sequences represented by SEQ ID NO 2, 4, 13, 15, 17, 19, 21 and 23, but that any polypeptide having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain)<br><br>
t<br>
and (ii) a DNΑ-binding domain for binding DNA, may be useful in the methods of the invention. A DP polypeptide having such features retains similar functional and/or biological activity or at least part of the functional and/or biological activity of a DP protein. Biological activity is the activity of the protein when it is in its natural environment. The dimerisation activity (i.e., the capability of DP to dimerize with an E2F transcription factor) may be tested by, for example, a Two-Hybrid assay such as described in Magyar et a/., 2000 (FEBS letters, 486: 79-97) or using co-immunoprecipitation.<br>
A DP polypeptide/protein or homologue thereof is encoded by a “DΒ-encoding nucleic acid” or by a “DΒ-encoding gene”; the terms are used interchangeably herein and mean a nucleic acid encoding a DP polypeptide or homologue thereof as described hereinabove. Examples of DΒ-encoding nucleic acids include those represented by any one of SEQ ID NO 1, 3, 12, 14, 16, 18,20 or 22, Use of DΒ-encoding nucleic acids and functional variants thereof may be suitable in practicing the methods of the present invention. Functional variants of DΒ-encoding nucleic acids include portions of such nucleic acids and/or nucleic acids capable of hybridising with a DΒ-encoding nucleic acid. Functional variants (portions or hybridising sequences) useful in the methods of the invention are those encoding polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA, may be useful in the methods of the invention.<br>
The term portion as used herein refers to a piece of DNA comprising at least 80 nucleotides and which portion encodes a polypeptide having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA. Such portions may be useful in the methods of the invention. The portion may be prepared, for example, by making one or more deletions to a DΒ-encoding nucleic acid. The portion is preferably a portion of a nucleic acid as represented by any one of SEQ ID NO 1, 3, 12, 14, 16, 18, 20 or 22, and which portion meets the aforementioned requirements.<br>
Another functional variant of a DΒ-encoding nucleic acid is a nucleic acid capable of hybridizing, preferably under stringent conditions, with a DΒ-encoding nucleic acid. Such a hybridizing sequence as defined herein is at least 80 nucleotides in length and encodes a polypeptide having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA. Such hybridizing sequences may be useful in the methods of the invention. The hybridizing sequence is preferably capable of hybridizing to a nucleic acid<br><br>
as represented by any one of SEQ ID NO 1, 3, 12, 14, 16, 18, 20 and 22, and which hybridising sequenoe meets the aforementioned requirements.<br>
The term “hybridising” as used herein means annealing to a substantially homologous complementary nucleotide sequences in a hybridization process. The hybridisation process may occur entirely in solution, i.e. both complementary nucleic acids are in solution. The hybridisation process may also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. The hybridisation process may furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro-cellulose or nylon membrane or immobilised by e.g. photolithography to e.g. a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, sodium/salt concentration and hybridisation buffer composition.<br>
Hybridization occurs under reduced stringency conditions, preferably under stringent conditions. The stringency of hybridisation is influenced by conditions such as temperature, salt concentration, ionic strength and hybridisation buffer composition. Hybridisation occurs under reduced stringency conditions, preferably under stringent conditions. Examples of stringency conditions are shown in Table 1 below: stringent conditions are those that are at least as stringent as, for example, conditions Α-L; and reduced stringency conditions are at least as stringent as, for example, conditions M-R.<br>
Table 1: Examples of hybridisation and wash conditions<br><br><br><br>
+ The “hybrid length” is the anticipated length for the hybridising nucleic acid. When nucleic acids of known sequence are hybridised, the hybrid length may be determined by aligning the sequences and identifying the conserved regions described herein.<br>
t SSPE (1 *SSPE is 0.15 M NaCI, 10 mM NaH2P04, and 1.25 mM EDTA, pH 7.4) may be substituted for SSC (1 *SSC is 0.15 M NaCI and 15 mM sodium citrate) in the hybridisation<br><br>
‘ and wash buffers; washes are performed for 15 minutes after hybridisation is complete. The hybridisations and washes may additionally include 5 x Denhardfs reagent, .5-1.0% SDS, 100 μg/ml denatured, fragmented salmon sperm DNA, 0.5% sodium pyrophosphate, and up to 50% formamide. *Tb-Tr The hybridization temperature for hybrids anticipated to be less than 50 base pairs in length should be 5-10 °C less than the melting temperature Tm of the hybrids there Tm is determined according to the following equations. For hybrids less than 18 base pairs in length, Tm (°C.) = 2 (# of A + T bases) + 4 (# of G + C bases). For hybrids between 18 # and 49 base pairs in length, Tm (°C.) = 81.5 + 16.6 (log.sub.10[Na+]) + 0.41 (% G + C) -(600/N), where N is the number of bases in the hybrid, and [Na+] is the concentration of sodium ions in the hybridization buffer ([NA+] for 1 *SSC = .165 M). ± The present invention encompasses the substitution of any one, or more DNA or RNA hybrid partners with either a PNA, or a modified nucleic acid.<br>
Other functional variants of DΒ-encoding nucleic acids useful in the methods of the invention include allelic variants of a DΒ-encoding nucleic acid, splice variants, variants due to the degeneracy of the genetic code, family members of a DΒ-encoding nucleic acid and variants interrupted by one or more intervening sequences, such as introns, spacer sequences or transposons. Each of the aforementioned functional variants is one capable of encoding a polypeptide having the features characteristic of DPs, namely (i) dimensation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA; such functional variants may be useful in the methods of the invention. DΒ-encoding nucleic acids or functional variants thereof may be in the form of DNA or a complement thereof, RNA, cDNA, genomic DNA, synthetic DNA as a whole or a part, double-stranded or single-stranded nucleic acid.<br>
The sequence given in SEQ ID NO: 3 is an example of a splice variant of SEQ ID NO 1. Other examples of splice variants are found in Oryza sativa where two DPb proteins each have two different splice forms: AAO72709.1 and AY224589, which are splice variants of the same genomic DNA, and AA072671.1 and AY224551 which are splice variants of the same genomic DNA and encoding the other DPb protein. The term “splice variant” as used herein encompasses variants of a nucleic acid in which selected introns and/or exons have been excised, replaced or added. Splice variants suitable for use in the methods of the invention are those encoding polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA. Such splice variants may be found in nature or may be prepared by selectively retaining these functional segments of the protein. Methods for making splice variants are well known in the art.<br><br>
Another functional variant DΒ-encoding nucleic acid is an allelic variant of a DΒ-encoding gene. Allelic variants exist in nature and encompassed within the methods of the present invention is the use of these natural alleles. Allelic variants also encompass Single Nucleotide Polymorphisms (SNPs) as well as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDELs form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms. Allelic variants suitable for use in the methods of the invention are those encoding polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA.<br>
The DΒ-encoding nucleic acid or functional variant thereof may be derived from any natural or artificial source. The source may be a microbial source, such as bacteria, yeast or fungi, or plant, algal or animal (including human) source. The nucleic acid may be modified from its native form in composition and/or genomic environment through deliberate human manipulation. The nucleic acid is preferably of plant origin, whether from the same plant species (for example to the one in which it is to be introduced) or whether from a different plant species. The nucleic acid may be isolated from a dicotyledonous species, preferably from the family Brassicaceae, further preferably from Arabidopsis thaliana. More preferably, the DΒ-encoding gene isolated from Arabidopsis thaliana and is as represented by SEQ ID NO 1 or 3, and the encoded DP amino acid sequence is as represented by SEQ ID NO 2 or 4. Other preferred sequences are as represented by SEQ ID NO 12, 14, 16, 18, 20 and 22 and the corresponding amino acid sequence as represented by SEQ ID NO 13, 15,17, 19, 21 or 23.<br>
The DΒ-encoding nucleic acid sequences useful in the methods of the present invention may have in increasing order of preference, at least 30%, 31%, 32%, 33%, 34%, 35%, 36%, 37%, 38%, 39%, 40%, 41%, 42%, 43%, 44%, 45%, 46%, 47%, 48%, 49%, 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% sequence identity to a DΒ-encoding nucleic acid, for example, to any one of SEQ ID NO 1,3, 12, 14, 16, 18, 20 or 22.<br>
“Homologues” of a protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes having an amino acid substitution, deletion and/or insertion relative to the unmodified protein in question and having similar biological and functional activity as the unmodified protein from which they are derived. To produce such homologues, amino acids of the protein<br><br>
may be replaced by other amino- acids having similar properties (such as similar hydrophobicity, hydrophilic’rty, antigenicity, propensity to form or break α-helical structures or β-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company). Homologues useful in the methods of the invention are those polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA. A DP polypeptide having such features retains similar functional and/or biological activity or at least part of the functional and/or biological activity of a DP protein.<br>
Two special forms of homologues, namely orthologues and paraiogues, are evolutionary concepts used to describe ancestral relationships of genes. The term “orthologues” relates to genes in different organisms that are homologous due to an ancestral relationship. The term “paraiogues” relates to gene-duplications within the genome of a species leading to paralogous genes. The term “homologues” as used herein also encompasses paraiogues and orthologues of a DP protein, which are also useful in practising the methods of the invention.<br>
Orthologues of a DP protein in other plant species may easily be found by performing a reciprocal Blast search, which involves searching one or more sequence databases with a query gene or protein (for example, any one of SEQ ID NO 1 to 4 or 12 to 23), using for example, a BLAST program. The highest-ranking subject genes that result from this search are then used as a query sequence in a similar BLAST search. Those genes that have as a highest match against the original query sequence are considered to be orthologous genes. For example, to find a rice orthologue of an Arabidopsis thaliana gene, one may perform a BLASTN or TBLASTX analysis on a rice database such as the Oryza satfva Nipponbare database available at the NCBI website (http://www.ncbi.nlm.nih.gov). In a next step, the highest ranking rice sequences are used in a reverse BLAST search on an Arabidopsis thaliana sequence database. The method may be used to identify orthologues from many different species, for example, from corn.<br>
Paraiogues of a DP protein may easily be found by performing a Blast search on sequences of the same species from which the “query” DP protein is derived. From the sequences that are selected by the Blast search, true paraiogues may be identified as those having the highest sequence identity.<br>
Orthologues and paraiogues useful in the methods of the invention are those polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be<br><br>
attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA. Preferably, a DP paralogue further comprises the DPb motifs as described hereinafter.<br>
Functional variants of DΒ-encoding nucleic acids and homologues of DP proteins as defined hereinabove may occur in nature and may be isolated from nature. Once the sequence of a homologue is known, and its corresponding encoding sequence, the person skilled in the art will be able to isolate the corresponding DP nucleic acid from biological material such as genomic libraries, for example, by the technique of PCR. One example of such an experiment is outlined in Example 1. Alternatively, when the sequence is not known, new DP proteins may be isolated from biological material via hybridization techniques based on probes from known DP proteins. Alternatively and/or additionally, functional variants of DΒ-encoding nucleic acids or homologues of DP proteins may be manmade via techniques involving, for example, mutation (substitution, insertion or deletion) or derivation. These functional variants are herein referred to as “derivatives”, which derivatives are also useful in the methods of the present invention so long as the derivative is a polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA, or in the case of a nucleic acid, the derivative encodes such a polypeptide.<br>
Derivatives of a protein may readily be made using peptide synthesis techniques well known in the art, such as solid phase peptide synthesis and the like, or by protein engineering via recombinant DNA manipulations. The manipulation of DNA sequences to produce substitution, insertion or deletion variants of a protein are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DNA are well known to those skilled in the art and include M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols.<br>
A homologue may also be in the format of a substitutional variant. The term “substitutional variants1’ of a DP protein refers to those variants in which at least one residue in an amino acid sequence has been removed and a different amino acid inserted in its place. Amino acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the polypeptide; insertions usually are of the order of about 1-10 amino acids, and deletions can range from about 1-20 amino acids. Preferably, amino acid substitutions comprise conservative amino acid substitutions.<br><br>
Homologues may also be in the form of an “inserBonal variant” of a protein in which one or more amino acids are introduced into a predetermined site in the DP protein. Insertions may comprise amino-terminal and/or carboxy-terminal fusion as well as intrα-sequence insertion of single or multiple amino acids. Generally, insertions within the amino acid sequence are of the order of about 1 to 10 amino acids. Examples of amino- or carboxy-terminal fusions include fusion of the binding domain or activation domain of a transcriptional activator as used in the yeast two-hybrid system, phage coat proteins, (histidine)6-tag, glutathione S-transferase-tag, protein A, maltose-binding protein, dihydrofolate reductase, Tag-100 epitope, c-myc epitope, FLAG-epitope, lacZ, CMP (calmoduiin-binding peptide), HA epitope, protein C epitope and VSV epitope.<br>
Homologues in the form of “deletion variants” are characterised by the removal of one or more amino acids from the protein.<br>
The DP polypeptide or homologue thereof may be a derivative in the form of a peptide, oligopeptide, polypeptide, protein or enzyme, characterised by substitutions, and/or deletions and/or additions of naturally and non-naturally occurring amino acids compared to the amino acids of a naturally-occurring DP protein. A derivative may also comprise one or more non-amino acid substituents compared to the amino acid sequence from which it is derived. Such non-amino acid substituents include, for example, non-naturally occurring amino acids, a reporter molecule or other ligand, covalently or non-covalently bound to the amino acid sequence. Such a reporter molecule may be bound to facilitate the detection of the DP protein.<br>
Another type of DP polypeptide useful in the methods of the present invention is an active fragment of a DP protein. “Active fragments” of a DP protein comprise at least 80 contiguous amino acid residues of a DP protein, which residues retain similar biological and/or functional activity to a naturally occurring protein or a part thereof. Active fragments suitable for use in the methods of the invention include polypeptides having the features characteristic of DPs, namely (i) dimerisation activity (which may be attributable to a partial, or preferably to a whole, dimerisation domain) and (ii) a DNΑ-binding domain for binding DNA. Furthermore, active fragments include fragments of a DP protein starting at the second or third or further internal methionine residues; these fragments originate from protein translation, starting at internal ATG codons.<br>
The activity of a DP polypeptide or a homologue thereof or the expression of a DΒ-encoding gene or functional variant thereof may be increased by introducing a genetic modification (preferably in the locus of a gene encoding DP or a homologue thereof). The locus of a gene<br><br>
as defined herein is taken to mean a genomic region, which includes the gene of interest and 10KB uβ- or downstream of the coding region.<br>
The genetic modification may be introduced, for example, by any one (or more) of the following methods: TDNA activation, TILLING, site-directed mutagenesis, directed evolution, homologous recombination or by introducing into a plant a DΒ-encoding nucleic acid or functional variant thereof encoding a DP polypeptide or a homologue thereof. Following introduction of the genetic modification, there follows an optional step of selecting for increased activity of a DP polypeptide, which increased activity gives plants having improved growth characteristics.<br>
T-DNA activation tagging (Hayashi et al. Science (1992) 1350-1353) involves insertion of T-DNA usually containing a promoter (may also be a translation enhancer or an intron), in the genomic region of the gene of interest or 10KB uβ- or downstream of the coding region of a gene in a configuration such that such promoter directs expression of the targeted gene. Typically, regulation of expression of the targeted gene by its natural promoter is disrupted and the gene fells under the control of the newly introduced promoter. The promoter is typically embedded in a T-DNA. This T-DNA is randomly inserted into the plant genome, for example, through Agrobacteriurn infection and leads to overexpression of genes near to the inserted T-DNA. The resulting transgenic plants show dominant phenotypes due to overexpression of genes close to the introduced promoter. The promoter to be introduced may be any promoter capable of directing expression of a gene in the desired organism, in this case a plant. For example, constitutive, tissue-preferred, cell type-preferred and “inducible promoters are all suitable for use in T-DNA activation.<br>
A genetic modification may also be introduced in the locus of a DΒ-encoding gene using the technique of TILLING (Targeted Induced L.ocal Lesions ]N Genomes). This is a mutagenesis technology useful in generating and/or identifying, and isolating mutagenised variants of a DΒ-encoding nucleic acid encoding a DP protein capable of exhibiting DP activity. TILLING also allows selection of plants carrying such mutant variants. These mutant variants may even exhibit higher DP activity than that exhibited by the gene in its natural form. TILLNG combines high-density mutagenesis with high-throughput screening methods. The steps typically followed in TILLING are: (a) EMS mutagenesis (Redei and Koncz, 1992; Feldmann et a/., 1994; Lightner and Caspar, 1998); (b) DNA preparation and pooling of individuals; (c) PCR amplification of a region of interest; (d) denaturation and annealing to allow formation of heteroduplexes; (e) DHPLC, where the presence of a heteroduplex in a pool is detected as an extra peak in the chromatogram; (f) identification of the mutant individual; and (g) sequencing<br><br>
of the mutant PCR product. Methods for TILLING are well known in the art (McCallum Nat Biotechnol. 2000 Apr; 18(4)1455-7, reviewed by Stemple 2004 (TILLING a high-throughput harvest for functional genomics. Nat Rev Genet. 2004 5(2): 145-50)).<br>
Site-directed mutagenesis may be used to generate variants of DΒ-encoding nucleic acids or functional variants thereof encode active proteins. Several methods are available to achieve site-directed mutagenesis; the most common being PCR based methods (Current Protocols in Molecular Biology. Wiley Eds. http^/www.4ulr.com/products/cun-entprotocols/index.html).<br>
Directed evolution may also be used to generate variants of DΒ-encoding nucleic acids. This consists of iterations of DNA shuffling followed by appropriate screening and/or selection to generate variants of DΒ-encoding nucleic acids or portions ttiereof encoding DP polypeptides or portions thereof having a modified biological activity (Castle et ah, (2004) Science 304(5674): 1151-4; US patents 5,811,238 and 6,395,547).<br>
TDNA activation, TILLING and site-directed mutagenesis are examples of technologies that enable the generation of novel alleles and DP variants that retain DP function and which are therefore useful in the methods of the invention.<br>
Homologous recombination allows introduction in a genome of a selected nucleic acid at a defined position. Homologous recombination is a standard technology used routinely in biological sciences for lower organism such as yeast or the moss physcomitrella. Methods for performing homologous recombination in plants have been described not only for model plants (Offringa etal. Extrachromosomal homologous recombination and gene targeting in plant cells after Agrobacterium-mediated transformation. 1990 EMBO J. 1990 Oct; 9(10): 3077-84) but also for crop plants, for example rice (Terada R et a/. Nat. Biotechnol. 2002 Efficient gene targeting by homologous recombination in rice; Lida and Terada Curr Opin Biotechnol. 2004 15(2): 132-8: A tale of two integrations, transgene and T-DNA: gene targeting by homologous recombination in rice). The nucleic acid to be targeted (which may be a DP nucleic acid or variant thereof as hereinbefore defined) need not be targeted to the locus of a DP gene, but may be introduced in, for example, regions of high expression. The nucleic acid to be targeted may be an improved allele used to replace the endogenous gene or may be introduced in addition or the endogenous gene.<br>
According to a preferred embodiment of the invention, plant growth characteristics may be improved by introducing and expressing in a plant a nudeic acid or functional variant thereof<br><br>
encoding a DP polypeptide or a homologue thereof, wherein the nucleic acid is specifically expressed in plant shoot tissue.<br>
According to a preferred aspect of the present invention, enhanced or increased expression of the a DΒ-encoding nucleic acid or functional variant thereof is envisaged. Methods for obtaining enhanced or increased expression of genes or gene products are well documented in the art and include, for example, overexpression driven by appropriate promoters, the use of transcription enhancers or translation enhancers. Isolated nucleic acids which serve as promoter or enhancer elements may be introduced in an appropriate position (typically upstream) of a non-heterologous form of a polynucleotide so as to upregulate expression of a DΒ-encoding nucleic acid or functional variant thereof. For example, endogenous promoters may be altered in vivo by mutation, deletion, and/or substitution (see, Kmiec, U.S. Pat No. 5,565,350; Zarling et a/., PCT/US93/03868), or isolated promoters may be introduced into a plant cell in the proper orientation and distance from a gene of the present invention so as to control the expression of the gene.<br>
According to the methods of the present invention, the activity of a DP polypeptide is increased specifically in shoot tissue, and preferably this is mediated by increased expression of a DΒ-encoding nucleic acid specifically in shoot tissue. The term “shoot” as used herein encompasses all aerial parts of the plants. Typical shoot-tissues include, but are not limited to, tissues of stems, branches, leaves, buds, flowers, reproductive organs, seeds, and shoot-derived structures such as stolons, corms, bulbs or tubers. Preferably in the methods of the present invention, the DΒ-encoding gene is preferentially expressed in young shoot tissue. ‘<br>
Preferably, shoot-specific expression of a DΒ-encoding gene is achieved by a shoot-specific promoter which is operably linked to a DΒ-encoding gene. Therefore, according to a preferred embodiment of the invention there is provided a method for improving plant growth characteristics relative to corresponding wild-type plants, comprising the introduction into a plant of a nucleic acid encoding a DP protein, which nucleic acid is specifically expressed in shoot-tissue.<br>
The term “shoot-specific promoter” as defined herein refers to a promoter that is at least 5 times stronger in the shoots than in other plant organs, such as the roots and is preferentially, but not exclusively, expressed in aerial parts of a plant. The term “tissue-specific” promoter is used interchangeably herein with a “tissue-preferred” promoter. Alternatively, shoot-specific expression of a DΒ-encoding gene may be mediated by selective transformation techniques, such as use of ballistics for the transformation of aerial tissues. Alternatively, shoot-specific<br><br>
expression may be achieved by T-DNA tagging, a technique well known to those skilled in the art and involving introducing a promoter randomly in a plant and selecting for those plants in which DP expression is increased specifically in shoot tissue.<br>
If polypeptide expression is desired, it is generally desirable to include a polyadenylation region at the 3’-end of a polynucleotide coding region. The polyadenylation region can be derived from the natural gene, from a variety of other plant genes, or from T-DNA. The 31 end sequence to be added may be derived from, for example, the nopaline synthase or octopine synthase genes, or alternatively from another plant gene, or from any other eukaryotic gene.<br>
An intron sequence may also be added to the 5’ untranslated region or the coding sequence of the partial coding sequence to increase the amount of the mature message that accumulates in the cytosol. Inclusion of a spliceable intron in the transcription unit in both plant and animal expression constructs has been shown to increase gene expression at both the mRNA and protein levels up to 1000-fold, Buchman and Berg, Mol. Cell biol. 8:4395-4405 (1988); Callis et a/., Genes Dev. 1:1183-1200 (1987). Such intron enhancement of gene expression is typically greatest when placed near the 51 end of the transcription unit Use of the maize introns Adh1-S intron 1, 2, and 6, the Bronze-1 intron are known in the art; see The Maize Handbook, Chapter 116, Freeling and Walbot, Eds., Springer, N.Y. (1994).<br>
The invention also provides genetic constructs and vectors to facilitate introduction and/or expression of the nucleotide sequences useful in the methods according to the invention.<br>
Therefore, according to a further embodiment of the present invention, there is provided a genetic construct comprising:<br>
(a)	a DΒ-encoding nucleic acid or functional variant thereof;<br>
(b)	one or more transcription control sequences capable of driving expression of the nucleic acid of (a) in shoot tissue; and optionally<br>
(c)	a transcription termination sequence.<br>
Constructs useful in the methods according to the present invention may be constructed using recombinant DNA technology well known to persons skilled in the art. The gene constructs may be inserted into vectors, which may be commercially available, suitable for transforming into plants and suitable for maintenance and expression of the gene of interest in the transformed cells. Preferably, the genetic construct according to the present invention is a plant expression vector, suitable for introduction and/or maintenance and/or expression of a nucleic acid in a plant cell, tissue, organ or whole plant.<br><br>
An example of a genetic construct according to the present invention is represented by SEQ ID NO 8 and depicts a DP gene under the control of a rice betα-expansin promoter, followed by a double transcription termination sequence (see Figure 2).<br>
The present invention provides genetic constructs as described above wherein the control sequence of (b) is a shoot-tissue preferred promoter, such as a betα-expansin promoter or a promoter having a comparable expression profile to the betα-expansin promoter.<br>
The nucleic acid acoording to (a) is advantageously any DΒ-encoding nucleic acid, such as any of the nucleic acids described hereinbefore. A preferred nucleic acid is a nucleic acid represented by SEQ ID NO 1, 2, 12, 14, 16, 18, 20 or 22 or a functional variant thereof as described hereinabove, or is a nucleic acid encoding a protein as represented by SEQ ID NO 2, 4, 13, 15, 17,19, 21 or 23 or a homologue thereof as described hereinabove.<br>
Also provided is use of the aforementioned constructs in the methods of the invention.<br>
Plants are transformed with a vector comprising the sequence of interest (i.e., a DΒ-encoding nucleic acid or functional variant thereof). The sequence of interest is operably linked to one or more control sequences, preferably to a promoter. The term “promoter* refers to a transcription control sequence. The promoter of (b) is operable in a plant and is preferably plant-derived. The terms transcription control sequence” and “promoter” are used interchangeably herein and refer to regulatory nucleic acids “capable of effecting expression of the sequences to which they are operably linked. Encompassed by the aforementioned terms are transcriptional regulatory sequences derived from a classical eukaryotic genomic gene (including the TATA box which is required for accurate transcription initiation, with or without a CCAAT box sequence) and additional regulatory elements (i.e., upstream activating sequences, enhancers and silencers) which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific manner. Also included within the term is a transcriptional regulatory sequence of a classical prokaryotic gene, in which case it may include a -35 box sequence and/or-10 box transcriptional regulatory sequences. The term “regulatory element” also encompasses a synthetic fusion molecule or derivative, which confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ.<br>
The term “operably linked” as used herein refers to a functional linkage between the promoter sequence and the gene of interest, such that the promoter sequence is able to initiate<br><br>
transcription of the gene of interest Preferably, the gene of interest is operably linked in the sense orientation to the promoter.<br>
Advantageously, any promoter may be used for the methods of the invention, provided that it has a shoot-tissue-specific expression pattern. These promoters have, when compared to a strong constitutive promoter (such as the strong constitutive/ubiquitous CaMV35S promoter), a lower expression level in roots.<br>
One example of such a promoter is the rice betα-expansin promoter EXPB9, represented herein by SEQ ID NO 7. This promoter may be isolated from the Oryza sativa (japonica cultivar-group) chromosome 10, BAG OSJNBa0082M15, where it is located upstream of EXPB9 gene encoding the mRNA as represented by the Genbank accession number AF261277. The term “shoot-specific promoter” as used herein therefore also means a promoter that has the same or similar activity, as the rice betα-expansin promoter EXPB9 in Oryza sativa. Similar activity in this context means an activity that is at most 2(Mbld higher or lower than the betα-expansin promoter EXPB9, preferably at most 10-fold higher or lower or 5-fold higher or lower or 3-fold higher or lower.<br>
One method to measure the promoter strength is through the use of promoter-betα-glucuronidase fusions. The promoter may be fused to the Escherichia coli UidA gene encoding betα-glucuronidase and transformed into a plant. Proteins are extracted from the plant material and GUS activity is measured (Jefferson et a/., 1987, EMBO J. 20;6(13):3901-7). Promoter activity is then calculated as the optical density in units per mg of extracted protein.<br>
Preferably, the shoot-preferred promoter is expressed during vegetative growth of a plant or in young shoot-tissue. Therefore, GUS activity is preferably measured from tissues after germination. These measurements are preferably performed during vegetative growth of a plant, for example after 2, preferably after 4 weeks post germination.<br>
Another example of a shoot-tissue-preferred promoter is a protochlorophyl reductase promoter.<br>
Optionally, the genetic construct may also comprise one or more terminator sequences. The term “transcription termination sequence” encompasses a control sequence at the end of a transcriptional unit, which signals 3’ processing and polyadenylation of a primary transcript and termination of transcription. Additional regulatory elements, such as transcriptional or<br><br>
translations! enhancers, may be incorporated in the genetic construct Those skilled in the art will be aware of suitable terminator and enhancer sequences.<br>
The genetic constructs of the invention may further include an origin of replication, which is required for maintenance and/or replication in a specific cell type. One example is when a genetic construct is required to be maintained in a bacterial cell as an episomal genetic element (e.g. plasmid or cosmid). Preferred origins of replication include, but are not limited to, thefl-ori and colEl<br>
The genetic construct may optionally comprise a selectable marker gene. As used herein, the term “selectable marker gene” includes any gene that confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells that are transfected or transformed with a nucleic acid construct of the invention. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance, that introduce a new metabolic trait or that allow visual selection. Examples of selectable marker genes include genes conferring resistance to antibiotics (such as nptll that phosphorylates neomycin and kanamycin, or hpt, phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta; aroA or gox providing resistance against glyphosate), or genes that provide a metabolic trait (such as manA that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example β-glucuronidase, GUS), luminescence (such as lucrferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives thereof).<br>
The present invention also encompasses plants obtainable by the methods according to the present invention. The present invention therefore provides plants obtainable by the method according to the present invention, which plants have introduced therein an DΒ-encoding nucleic acid or functional variant thereof as defined hereinbefore.<br>
The invention also provides a method for the production of transgenic plants having improved growth characteristics, comprising introduction and expression in a plant of a DΒ-encoding nucleic acid or functional variant thereof.<br>
There is therefore provided a method for the production of a transgenic plant comprising:<br>
(a)	introducing into a plant cell a DΒ-encoding nucleic acid or functional variant thereof, preferably introducing a genetic construct as described above;<br>
(b)	cultivating said plant cell under conditions promoting plant growth and development.<br><br>
The DΒ-encoding nucleic acid or functional variant thereof or genetic construct is as defined above. The transgenic plants produced by the abovementioned method have improved plant growth characteristics relative to corresponding wild-type plants, such growth characteristics being any of the aforementioned.<br>
“lntroducing” the DΒ-encoding nucleic acid or the genetic construct into a plant cell is preferably achieved by transformation. The term “transformation” as used herein encompasses the transfer of an exogenous polynucleotide into a host cell, irrespective of the method used for transfer. Plant tissue capable of subsequent clonal propagation, whether by organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention. The choice of tissue depends on the particular plant species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem). The polynucleotide may be transiently or stably introduced into a host cell and may be maintained non-integrated, for example, as a plasmid. Alternatively, it may be integrated into the host genome. Preferably, the DΒ-encoding nucleic acid is stably integrated in the genome of the plant cell, which may be achieved, for example, using a plant transformation vector or a plant expression vector having T-DNA borders, which flank the nucleic acid to be introduced into the genome.<br>
Transformation of a plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce the gene of interest into a suitable ancestor cell. Transformation methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment, transformation using viruses or pollen and microprojection. Methods may be selected from the calcium/polyethylene glycol method for protoplasts (Krens, FA. et a/., 1882, Nature 296, 72-74; Negrutiu I. ef a/., June 1987, Plant Mol. Biol. 8, 363-373); electroporation of protoplasts (Shillito R.D. et a/., 1985 Bio/Technol 3, 1099-1102); microinjection into plant material (Crossway A. ef a/., 1986, Mol. Gen Genet 202, 179-185); DNA or RNΑ-coated particle bombardment (Klein T.M. et a/., 1987, Nature 327, 70) infection with (non-integrative) viruses and the like. A preferred method for the production of transgenic plants according to the invention is an Agrobactierium-mediated transformation method.<br>
Transgenic rice plants are preferably produced via Agrobacacterium-mediated transformation using any of the well-known methods for rice transformation, such as the ones described in any of the following: published European patent application EP1198985, Aldemita and Hodges<br><br>
(Planta, 1996, 199: 612-617,); Chan et al (Plant Mol. Biol., 1993, 22 (3): 491-506,); Hiei et al. (Plant J., 1994, 6 (2): 271-282,); which disclosures are incorporated by reference herein as if fully set forth. In the case of corn transformation, the preferred method is as described in either Ishida et a/. (Nat. Biotechnol., 1996, 14(6): 745-50) or Frame et al. (Plant Physiol., 2002, 129(1): 13-22), which disclosures are incorporated by reference herein as if fully set forth.<br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more markers, which are co-transformed with the DΒ-encoding gene.<br>
The resulting transformed plant cell, cell group, or plant tissue, may then be used to regenerate a whole transformed plant via regeneration techniques well known to persons skilled in the art. Therefore, cultivating the plant cell under conditions promoting plant growth may encompass the steps of selecting and/or regenerating and/or growing to maturity.<br>
Following DNA transfer and regeneration, putatively transformed plants may be evaluated using Southern analysis for instance to monitor the presence of the gene of interest, determine copy number and/or genomic organisation. Alternatively or additionally, expression levels of the newly introduced DNA may be monitored using Northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the art.<br>
The generated transformed plants may be propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfedto give homozygous second generation (or T2) transformants, and the T2 plants further propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; clonal transformants (e.g., all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rootstock grafted to an untransformed scion).<br>
The present invention also provides host cells containing an isolated nucleic acid molecule encoding a DP or a functional variant thereof or a genetic construct as mentioned hereinbefore. Preferred host cells are plant cells. Also provided are plant cells, tissues, organs and whole plants and plant parts and propagules thereof that have been transformed with and which comprise a DΒ-encoding gene or a functional variant thereof or genetic construct of the invention. The present invention clearly extends to plants obtainable by any of the methods described hereinbefore. The present invention extends to plants which have increased<br><br>
expression levels of a DΒ-encoding nucleic acid and/or increased levels and/or activity of a DP protein specifically in shoot-tissue. The present invention also encompasses progeny of a primary transformed cell, tissue, organ or whole plant or plant part, the only requirement being that progeny exhibit the same genotypic characteristics) as the parent plants from which they are derived. The plants according to the present invention have improved growth characteristics relative to corresponding wild type plants. The invention also extends to any part of the plant according to the invention, preferably a harvestable part, such as, but not limited to, seed, leaf, fruit, flower, stem culture, stem, rhizome, root, tuber, bulb and cotton fiber.<br>
The present invention also concerns use of a nucleic acid encoding a DP protein or homologue thereof under the control of a shoot-preferred promoter for improving any one or more of the aforementioned growth characteristics.<br>
DΒ-encoding nucleic acids or functional variants thereof, or DP polypeptides or homologues thereof, may find use in breeding programmes in which a DNA marker is identified which may be genetically linked to a DΒ-encoding gene or functional variant thereof. The DΒ-encoding genes or functional variants thereof, or DP protein or homologues thereof may be used to define a molecular marker. This DNA or protein marker may then be used in breeding programs to select plants having altered growth characteristics. The DΒ-encoding gene or functional variant thereof is preferably a nucleic acid as represented by any one of SEQ ID NO 1,3,12, 14,16, 18,20 or 22.<br>
Allelic variants of a DΒ-encoding gene or functional variant thereof may also find use in marker-assisted breeding programmes. Such breeding programmes sometimes require introduction of allelic variation by mutagenic treatment of the plants, using for example EMS mutagenesis; alternatively, the programme may start with a collection of allelic variants of natural origin. Identification of allelic variants then takes place by, for example, PCR. This is followed by a selection step for selection of superior allelic variants of the sequence in question and which give rise improved growth characteristics in a plant relative to corresponding wild type plants. Selection is typically carried out by monitoring growth performance of plants containing different allelic variants of the sequence in question, for example, different allelic variants of any one of SEQ ID NO 1, 3, 12, 14, 16, 18, 20 or 22. Growth performance may be monitored in a greenhouse or in the field. Further optional steps include crossing plants in which the superior allelic variant was identified with another plant. This could be used, for example, to make a combination of interesting phenotypic features.<br><br>
A DΒ-encoding nucleic acid or functional variant thereof may also be used as a probe for genetically and physically mapping the genes of which they are a part, and as markers for traits linked to those genes. Such information may be useful in plant breeding in order to develop lines with desired phenotypes. Such use of DΒ-encoding nucleic acids or functional variants thereof only requires a nucleic acid sequence of at least 15 nucleotides in length. The DΒ-encoding nucleic acids or functional variants thereof may be used as restriction fragment length polymorphism (RFLP) markers. Southern blots (Maniatis) of restriction-digested plant genomic DNA may be probed with the DΒ-encoding nucleic acids or functional variants thereof. The resulting banding patterns may then be subjected to genetic analyses using computer programs such as MapMaker (Lander et a/. (1987) Genomics 1:174-181) in order to construct a genetic map. In addition, the nucleic acids may be used to probe Southern blots containing restriction endonuclease-treated genomic DNAs of a set of individuals representing parent and progeny of a defined genetic cross. Segregation of the DNA polymorphisms is noted and used to calculate the position of the DΒ-encoding nucleic acids or functional variants thereof in the genetic map previously obtained using this population (Botstein et a/. (1980) Am. J. Hum. Genet. 32:314-331).<br>
The production and use of plant gene-derived probes for use in genetic mapping is described in Bematzky and Tanksley (1986) Plant Mol. Biol. Reporter 4:37-41. Numerous publications describe genetic mapping of specific cDNA clones using the methodology outlined above or variations thereof. For example, F2 intercross populations, backcross populations, randomly mated populations, near isogenic lines, and other sets of individuals may be used for mapping. Such methodologies are well known to those skilled in the art<br>
The nucleic acid probes may also be used for physical mapping (i.e., placement of sequences on physical maps; see Hoheisel et a/, in Non mammalian Genomic Analysis: A Practical Guide, Academic press 1996, pp. 319-346, and references cited therein).<br>
In another embodiment, the nucleic acid probes may be used in direct fluorescence in situ hybridization (FISH) mapping (Trask (1991) Trends Genet. 7:149-154). Although current methods of FISH mapping favor use of large clones (several to several hundred KB; see Laan et ai (1995) Genome Res. 5:13-20), improvements in sensitivity may allow performance of FISH mapping using shorter probes.<br>
A variety of nucleic acid amplification-based methods of genetic and physical mapping may be carried out using the aforementioned DΒ-encoding nucleic acids. Examples include allele-specific amplification (Kazazian (1989) J. Lab. Clin. Med 11:95-96), polymorphism of PCR-<br><br>
amplified fragments (CAPS; Sheffield et a/. (1993) Genomics 16:325-332), allele-spectfic ligation (Landegren et al. (1988) Science 241:1077-1080), nucleotide extension reactions (Sokolov (1990) Nucleic Acid Res. 18:3671), Radiation Hybrid Mapping (Walter et a/. (1997) Nat. Genet 7:22-28) and Happy Mapping (Dear and Cook (1989) Nucleic Acid Res. 17:6795-6807). For these methods, the sequence of a nucleic acid is used to design and produce primer pairs for use in the amplification reaction or in primer extension reactions. The design of such primers is well known to those skilled in the art. In methods employing PCR-based genetic mapping, it may be necessary to identify DNA sequence differences between the parents of the mapping cross in the region corresponding to the instant nucleic acid sequence. This, however, is generally not necessary for mapping methods.<br>
DΒ-encoding nucleic acids or functional variants thereof or DP polypeptides or homologues thereof may also find use as growth regulators. Since these molecules have been shown to be useful in improving the growth characteristics of plants, they would also be useful growth regulators, such as herbicides or growth stimulators. The present invention therefore provides a composition comprising a DΒ-encoding nucleic acid or functional variant thereof or a DP polypeptide or homologue thereof, together with a suitable carrier, diluent or excipient, for use as a growth regulator.<br>
The methods according to the present invention result in plants having improved growth characteristics, as described hereinbefore. These advantageous growth characteristics may also be combined with other economically advantageous traits, such as further yield-enhancing traits, tolerance to various stresses, traits modifying various architectural features and/or biochemical and/or physiological features.<br>
The present invention will now be described with reference to the following figures in which:<br>
Fig. 1 is a map of the binary vector pEXP::AtDPb for expression in Oryza sativa of the Arabidopsis thaliana DPb gene (intemai reference CDS006) under the control of a rice betα-expansin promoter (betα-EXPB9 promoter with intemai reference PRO0061). The AfDPb expression cassette further comprises a T-zein and T-rbcS-deltaGA double transcription termination sequence. This expression cassette is located within the left border (LB repeat, LB Ti C58) and the right border (RB repeat, RB Ti C58) of the nopaline Ti plasmid. Within the T-DNA there is further provided a selectable and a screenable marker, both under control of a constitutive promoter and followed by polyA or a T-NOS transcription terminator sequence. This vector farther comprises an origin of replication (pBR322 ori + bom) for bacterial replication and a bacterial selectable marker (Spe/SmeR) for bacterial selection.<br><br>
Fig. 2 gives some examples of the sequences described in the present application. SEQ ID NO 2 also shows underlined a region typically conserved in DP proteins.<br>
Fig. 3 shows an alignment of DP proteins with the location of conserved consensus DPb motifs represented as SEQ ID NO 9 (motif 1), 10 (motif 2) and 11 (motif 3). DNΑ-binding domain and dimerisation domain of AtDPb are indicated. The location of a highly conserved region common to all DP proteins is indicated with dashed brackets. A multiple sequence alignment across the entire sequence was done using CLUSTAL W (Higgins et a/., (1994) Nucleic Acids Res. 22:4673-4680), with the BLOSSUM 62 matrix and with the parameters GAPOPEN 10, GAPEXT 0.05 and GAPDIST 8. Genbank accession numbers for the sequences are also provided.<br>
Fig 4 shows a cladogram corresponding to the multiple alignment of Figure 3. The cladogram was generated using ClustalW. Genbank accession numbers for the sequences are provided.<br>
Fig. 5 shows a phylogram view of DP proteins. The phylogram gives the length of the branches and the distance between the nodes in proportion to the evolutionary distance between the sequences. The cladogram view was generated by ClustalW. The two groups of DP proteins (Dpa and DPb) may be distinguished from one another by the presence or absence of a KKKK/RR motif which is only present in DPb.<br>
Examples<br>
The present invention will now be described with reference to the following examples, which are by way of illustration alone.<br>
DNA Manipulation<br>
Unless otherwise stated, recombinant DNA techniques are performed according to standard protocols described in Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York or in Volumes 1 and 2 of Ausubel et aL (1998), Current Protocols in Molecular Biology. Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfase (1993) by R.D.D. Croy, published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications (UK).<br>
Example 1: Cloning of Arabidopsis thaliana DPb<br>
The Arabidopsis DPb gene (internal reference CDS006) was amplified by PCR using as template an Arabidopsis thaliana seedling cDNA library (Invitrogen, Paisley, UK). After reverse<br><br>
transcription of RNA extracted from seedlings, the cDNA fragments were cloned into pCMV Sport 6.0. Average insert size of the cDNA library was 1.5 kb, and original number of clones was about 1.59x107 cfii. The original titer of 9.6x105 cfu/ml was brought to 6x1011 cfu/ml after amplification of the library. After plasmid extraction of the clones, 200 ng of plasmid template was used in a 50 μl PCR mix. The primers used for PCR amplification, prm0319 with the sequence 5’ GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACAATGACAACTACTGGG TCTAATTCT 3’ (SEQ ID NO 5) and prm0320 with the sequence 5’ GGGGACCACTTTGTAC AAGAAAGCTGGG7TCAATTCTCCGGCTTCAT 3’ (SEQ ID NO 6), comprise an AttB site for Gateway recombination cloning (italics). PCR was performed using Hifi Taq DNA polymerase in standard conditions. A PCR fragment of the expected length was amplified and purified also using standard methods. The first step of the Gateway procedure, the BP reaction, was then performed, during which the PCR fragment recombines in vivo with the pDONR201 plasmid to produce the “entry clone”, p0424. Plasmid pDONR201 was purchased from Invltrogen, as part of the Gateway® technology.<br>
Example 2: Vector construction (pEXP::AtDPb)<br>
The entry done p0424 was subsequently used in an LR reaction with p3169, a destination<br>
vector used for Oryza sativa transformation. This vector contains as functional elements within the T-DNA borders, a plant selectable marker, a screenable marker and a Gateway cassette intended for LR in vivo recombination with the sequence of interest already cloned in the entry clone. Upstream of this Gateway cassette lies the rice betα-expansin promoter (internal reference PRO061) for shoot4issue-preferred expression of the gene of interest. After the LR recombination step, the resulting expression vector pEXP::AtDPb (Fig. 1) was transformed into Agrobacterium strain LBA4044 and subsequently into Oryza sativa van Nipponbare plants. Transformed rice plants were grown and examined for various growth characteristics as described in Example 3.<br>
Example 3: Evaluation of TO, T1 and T2 rice plants transformed with pEXP::AtDPb<br>
Approximately 15 to 20 independent TO transformants were generated. Primary transformants were transferred from tissue culture chambers to a greenhouse for growing and harvest of T1 seed. Six events of which the T1 progeny segregated 3/1 for presence/absence of the transgene were retained. “Null plants” or “Null segregants” or “Nuilizygotes” were plants treated in the same way as transgenic plants, but from which the transgene had segregated. Null plants may also be described as homozygous negative transformants. For each of these events, approximately 10 T1 seedlings containing the transgene (hetero- and homo-zygotes),<br><br>
and approximately 10 T1 seedlings lacking the transgene (nullizygotes), were selected by PCR.<br>
Based on the results of the T1 evaluation, three events which showed improved growth characteristics at the T1 level were chosen for further characterisation in the T2 and further generations. Seed batches from the positive T1 plants (both hetero- and homozygotes) were screened by monitoring marker expression. For each chosen event, the heterozygote seed batches were then selected for T2 evaluation. An equal number of positives and negatives within each seed batch were transplanted for evaluation in the greenhouse (i.e., for each event, 40 plants were grown of which 20 were positive for the transgene and 20 negative for the transgene). Therefore, for the three events, a total of 120 plants were evaluated in the T2 generation.<br>
T1 and T2 plants were transferred to a greenhouse and were evaluated for vegetative growth parameters as described hereunder.<br>
(I) Statistical analysis of numeric data<br>
A two factor ANOVA (analyses of variance) corrected for an unbalanced design was used as statistical evaluation model for the numeric values of the observed plant phenotypic characteristics. The numerical values were submitted to a t-test and an F test. The β-value was obtained by comparing the t value to the t distribution or alternatively by comparing the F value to the F distribution. The β-value represents the probability that the null hypothesis (null hypothesis being “there is no effect of the transgene”) incorrect<br>
A t-test was performed on all the values of all plants of one event. Such a t-test was repeated for each event and for each growth characteristic. The t-test was carried out to check for an effect of the gene within one transformation event, also named herein a “line-specific effect”. In the t-test, the threshold for a significant line-specific effect is set at 10% probability level. Therefore, data with a β-value of the t test under 10% means that the phenotype observed in the transgenic plants of that line is caused by the presence of the gene. Within one population of transformation events, some events may be under or below this threshold. This difference may be due to the difference in position of the transgene in the genome. It is not uncommon that a gene might only have an effect in certain positions of the genome. Therefore, the above-mentioned “line-specific effecf is also referred to as “position-dependent effect”.<br>
An F-test was carried out on all the values measured for all plants of all events. An F-test was repeated for each growth characteristic. The F-test was carried out to check for an effect of the<br><br>
gene over all the transformation events and to verify an overall effect of the gene, also named herein “gene effect”. In the F-test, the threshold for a significant global gene effect is set at 5% probability level. Therefore, data with a β-value of the F test under 5% mean that the observed phenotype is caused by more than just the presence of the gene and or the position of the transgene in the genome. A “gene effect” is an indication for the wide applicability of the gene in transgenic plants.<br>
(II) Vegetative growth measurements<br>
The selected plants were grown in a greenhouse. Each plant received a unique barcode label to link unambiguously the phenotyping data to the corresponding plant The selected plants were grown on soil in 10 cm diameter pots under the following environmental settings: photoperiod= 11.5 h, daylight intensity= 30,000 lux or more, daytime temperature= 28°C or higher, night time temperature= 22°C, relative humidity= 60-70%. Transgenic plants and the corresponding nullizygotes were grown side-by-side at random positions. From the stage of sowing until the stage of maturity (which is the stage were there is no more increase in biomass) the plants were passed weekly through a digital imaging cabinet. At each time point digital images (2048x1536 pixels, 16 million colours) were taken of each plant from at least 6 different angles. The parameters described below were derived in an automated way from the digital images using image analysis software.<br>
Aboveground area<br>
Plant aboveground area was determined by counting the total number of pixels from aboveground plant parts discriminated from the background. This value was averaged for the pictures taken on the same time point from the different angles and was converted to a physical surface value expressed in square mm by calibration. Experiments show that the aboveground plant area, which corresponds to the total maximum area, measured this way correlates with the biomass of plant parts aboveground.<br>
On average, pEXP::DPb transgenic plants in T1 generation showed an increase in aboveground area of 8% with a β-value in the F-test of 0.08. One out of the three best positive T1 lines showed an increase in aboveground area of 30% with a β-value from the t-test of 0.01. In the T2 generation, this line showed an 18% increase in aboveground area with a β-value from the t-test of 0.03.<br>
Example 4: GUS expression driven by beta expansin promoter<br>
The betα-expansin promoter was cloned into the pDONR201 entry plasmid of the GatewayTM system (Life Technologies) using the “BP recombination reaction”. The identity and base pair<br><br>
composition of the cloned insert was confirmed by sequencing and the resulting plasmid was also tested via restriction digests.<br>
In order to clone the promoter in front of a reporter gene, each entry clone was subsequently used in an “LR recombination reaction” (Gateway TM) with a destination vector. This destination vector was designed to operably link the promoter to the Escherichia coli betα-glucuronidase (GUS) gene via the substitution of the Gateway recombination cassette in front of the GUS gene. The resulting reporter vectors, comprising the promoter operably linked to GUS were subsequently transformed into Agrobacterium strain LBA4044 and subsequently into rice plants using standard transformation techniques.<br>
Transgenic rice plants were generated from transformed cells. Plant growth was performed under normal conditions.<br>
The plants or plant parts to be tested were covered with 90% ice-cold acetone and incubated for 30 min at 4 °C. After 3 washes of 5 min with Tris buffer [15,76 g Trizma HCI (Sigma T3253) + 2,922 g NaCI in 1 litre bi-distilled water, adjusted to pH 7,0 with NaOH], the material was covered by a Tris/ferricyanate/X-Gluc solution [9,8 ml Tris buffer + 0,2 ml ferricyanate stock (0,33 g Potassium ferricyanate (Sigma P3667) in 10 ml Tris buffer)+ 0,2 ml X-Gluc stock (26,1 mg X-Gluc (Europa Bioproducts ML 113A) in 500 pi DMSO)]. Vacuum infiltration was applied for 15 to 30 minutes. The plants or plant parts were incubated for up to 16 hours at 37 °C until development of blue colour was visible. The samples were washed 3 times for 5 minutes with Tris buffer.^Chlorophyll was extracted in ethanol series of 50%, 70% and 90% (each for 30 minutes).<br><br>
Claims<br>
1.	Method for improving plant growth characteristics relative to corresponding wild-type plants, comprising increasing activity of a DP polypeptide or homologue thereof specifically in plant shoot tissue and/or by increasing expression of a DP-encoding gene or a functional variant thereof in plant shoot tissue, and optionally selecting for plants having improved growth characteristics.<br>
2.	Method according to claim 1, wherein said increased activity and/or increased expression is effected by introducing a genetic modification preferably in the locus of a gene encoding a DP polypeptide or a homologue thereof.<br>
3.	Method according to claim 2, wherein said genetic modification is effected by any one or more of site-directed mutagenesis, directed evolution, homologous recombination, TILLING, T-DNA activation and by introducing into a plant a DP-encoding nucleic acid or functional variant thereof encoding a DP polypeptide or a homologue thereof.<br>
4.	Method for improving plant growth characteristics relative to corresponding wild-type plants, comprising introducing into a plant, and expressing specifically in shoot tissue, a DP-encoding nucleic acid or a functional variant thereof.<br>
5.	Method according to claim 4, wherein said functional variant is a portion of a DP-encoding nucleic acid or is a sequence capable of hybridising to a DP-encoding nucleic acid, wherein said functional variant encodes a polypeptide having: (i) dimerisation activity attributable to a partial, or preferably to a whole, dimerisation domain; and (ii) a DNA-binding domain for binding DNA.<br>
6.	Method according to any one of claims 1 to 5, wherein said DP-encoding nucleic acid or functional variant thereof or DP polypeptide or homologue thereof is of plant origin, preferably from a dicotyledoneous plant, further preferably from the family Bmssicaceae, more preferably from Arabidopsis thaliana.<br>
7.	Method according to any one of claims 1 to 6, wherein said DP-encoding nucleic acid is:<br>
(i) a nucleic acid comprising SEQ ID NO 1, 3, 12, 14, 16, 18, 20 or 22 or a functional variant of any, wherein said nucleic acid or functional variant encodes a polypeptide comprising (i) dimerisation activity attributable to a<br><br>
partial, or preferably to a whole, dimerisation domain; and (ii) a DNA-binding domain for binding DNA; or (ii) a nucleic acid encoding a protein as represented by SEQ ID NO 2, 4, 13, 15, 17, 19, 21 or 23 or encoding a homologue of any, wherein said protein or homologue comprises (i) dimerisation activity attributable to a partial, or preferably to a whole, dimerisation domain; and (ii) a DNA-binding domain for binding DNA.<br>
8.	Method according to any one of claims 4 to 7, wherein said nucleic acid or functional variant thereof is under the control of a promoter capable of specifically expressing said DP-encoding nucleic acid in shoot-tissue.<br>
9.	Method according to claim 8, wherein said promoter has a comparable expression profile to a beta-expansin promoter.<br>
10.	Method according to any one of claims 1 to 9, wherein said improved plant growth characteristics is increased biomass relative to corresponding wild-type plants.<br>
11.	Method according to any one of claims 1 to 10, wherein said improved plant growth characteristics is increased yield, particularly increased seed yield relative to corresponding wild-type plants.<br>
12.	Method according to any one of claims 1 to 11, wherein said improved plant growth characteristic is increased growth rate relative to corresponding wild-type plants.<br>
13.	Method according to any one of claims 1 to 12, wherein said improved plant growth characteristic is altered architecture, particularly one or more of increased number, size, shape of tillers, or corresponding plant part; increased number of branches and/or leaves.<br>
14.	Method according to any one of claims 1 to 13, wherein said plant is a monocotyledonous plant.<br>
15.	Plants obtainable by a method according to any one of claims 1 to 14.<br>
16.	Genetic construct comprising:<br>
(a) a DP-encoding nucleic acid or a functional variant thereof;<br><br>
(b)	one or more transcription control sequences capable of driving expression of the nucleic acid of (a) in shoot-tissue; and optionally<br>
(c)	a transcription termination sequence.<br><br>
17.	Genetic construct according to claim 13, wherein said control sequence is capable of specifically expressing said DP-encoding nucleic acid in young shoot-tissue.<br>
18.	Genetic construct according to claim 16 or 17, wherein said control sequence is a promoter having a comparable expression profile to a beta-expansin promoter.<br>
19.	Transgenic host cell comprising a genetic construct as defined in any one of claims 16 to 18.<br>
20.	Transgenic plant or plant part comprising a genetic construct as defined in any one of claims 16 to 18.<br>
21.	Method for the production of a transgenic plant having improved growth characteristics relative to corresponding wild type plants, which method comprises:<br><br>
a)	introducing into a plant cell a genetic construct according to any one of claims 16to18;<br>
b)	cultivating said plant cell under conditions promoting plant growth and development.<br><br>
22.	Transgenic plant having improved growth characteristics relative to corresponding wild type plants resulting from a DP-encoding nucleic acid or functional variant thereof under the control of a shoot-specific promoter introduced into said plant.<br>
23.	Transgenic plant according to claim 15, 19, 20 or 22, wherein said plant is a monocotyledonous plant, such as sugar cane or a cereal, such as rice, wheat, barley, maize, millet, rye, oats or sorghum.<br>
24.	Plant part, preferably a harvestable part, such as a seed, or a propagule of a plant as defined in any one of claims 15,19, 20, 22 and 23.<br>
25.	Progeny of a plant as defined in any one of claims 15, 19, 20 22, 23 and 24, wherein said progeny exhibit the same genotypic characteristic(s) as the parent plants from which they are derived.<br><br>
26.	Products directly derived from a plant or progeny as defined in any one of claims 15, 19,20 22,2310 25.<br>
27.	Use of a DP-encoding nucleic acid or functional variant thereof under the control of a shoot-specific promoter for improving plant growth characteristics selected from one or more of: increasing plant biomass, increasing yield, increased seed yield, altering plant architecture and increasing growth rate.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgICAyMy0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   AMENDED CLAIMS   23-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgIDI1LTExLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   AMENDED CLAIMS  25-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMuICAyNC0wNS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   AMENDED CLAIMS.  24-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAgMjMtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   AMENDED PAGES OF SPECIFICATION   23-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OLiAgMjQtMDUtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   AMENDED PAGES OF SPECIFICATION.  24-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgMjMtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   CORRESPONDENCE OTHERS   23-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAwNy0wMy0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   CORRESPONDENCE OTHERS  07-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAxMi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   CORRESPONDENCE OTHERS  12-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyMS0wOS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   CORRESPONDENCE OTHERS  21-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyNS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   CORRESPONDENCE OTHERS  25-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgRk9STS0zICAgMjMtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   FORM-3   23-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgRk9STS0zICAyNS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   FORM-3  25-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgRk9STS4zICAyNC0wNS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   FORM.3  24-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAgMjMtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   OTHER PATENT DOCUMENT   23-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyNS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   OTHER PATENT DOCUMENT  25-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICAgUE9XRVIgT0YgQVRUT1JORVkgICAyMy0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006   POWER OF ATTORNEY   23-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDI4LTA1LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006  CORRESPONDENCE OTHERS  28-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQuICAyNC0wNS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006  EXAMINATION REPORT REPLY RECEIVED.  24-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQuICAyNC0wNS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006  OTHER PATENT DOCUMENT.  24-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1DSEVOUC0yMDA2ICBQT1dFUiBPRiBBVFRPUk5FWSAgMjgtMDUtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4368-CHENP-2006  POWER OF ATTORNEY  28-05-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM2OC1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">4368-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258118-an-automatic-transmission-for-motor-vehicle.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258120-in-vitro-assay-for-the-detection-of-activity-of-sodium-calcium-exchange-protein.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258119</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4368/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>49/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Dec-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Dec-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Nov-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3,B-9052 ZWIJNAARDE,BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FRANKARD,VALERIE</td>
											<td>BIERENBERG 47,B-1640 SINT-GENESIUS-RODE, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>REUZEAU,CHRISTOPHE</td>
											<td>LA CHAPALLE GONAGUET, F-24350 TOCANE,FRANCE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/415</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/52465</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-05-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/576,250</td>
									<td>2004-06-02</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>04102392.0</td>
									<td>2004-05-28</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258119-a-method-for-improving-plant-growth-characteristics by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:32:42 GMT -->
</html>
