<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258235-a-kit-for-the-detection-of-at-least-one-mutation-relating-to-a-hiv-1-protein-of-reverse-transcriptase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:13:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258235:&quot;A KIT FOR THE DETECTION OF AT LEAST ONE MUTATION RELATING TO A HIV-1 PROTEIN OF REVERSE TRANSCRIPTASE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A KIT FOR THE DETECTION OF AT LEAST ONE MUTATION RELATING TO A HIV-1 PROTEIN OF REVERSE TRANSCRIPTASE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed are compositions including primers and probes, which are capable of interacting with the disclosed nucleic acids, such as the nucleic acids encoding the reverse transcriptase or protease of HIV as disclosed herein. Thus, provided is an oligonucleotide comprising any one of the nucleotide sequences set for in SEQ ID NOS:l-89, and 96-104. Also provided are the oligonucleotides consisting of the nucleotides as set forth in SEQ ID NOS:l-89, and 96-104. Each of the disclosed oligonucleotides is a probe or a primer. Also provided are mixtures of primers and probes and for use in RT-PCR and primary PCR reactions disclosed herein. Provided are methods for the specific detection of several mutations in HIV. Mutations in both the reverse transcriptase and the protease of HIV can be detected using the methods described herein.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>REAL-TIME PCR POINT MUTATION ASSAYS FOR DETECTING HIV-1 RESISTANCE TO ANTIVIRAL DRUGS<br>
CROSS REFERENCE TO RELATED PATENT APPLICATION<br>
This application claims priority to U.S. Provisional Application No. 60/577,696 filed June 7, 2004, and which is herein incorporated by reference in its entirety.<br>
BACKGROUND<br>
The HIV pandemic now exceeds 40 million persons and its expansion is being met with an increased use of anti-HIV drugs to care for the lives of those affected. Emergence of drug resistance is expected to increase as the use of these drugs for the clinical management of HIV-1 infected persons increases worldwide. Highly active antiretroviral therapy (HAART) containing a combination of three antiretroviral drugs is currently recommended and has been effective in reducing mortality and morbidity. Four classes of drugs are available that inhibit either virion entry (e.g., T-20), nucleotide extension by viral reverse transcriptase (e.g., 3TC, d4T), reverse transcriptase enzymatic activity (e.g., nevirapine, efavirenz), or the viral protease (e.g., nelfinavir, lopinavir). Drug resistance that is conferred by mutations is frequently selected in viruses from patients failing antiretroviral therapy and is considered a major cause of treatment failure.<br>
Current treatment guidelines recommend baseline drug resistance testing for the selection of optimal drug regimens for patients initiating antiretroviral therapy. Accurate identification of any resistant viruses the person carries will help guide the selection of treatment regimens with fully active drugs. Drug resistance testing is performed through the use of phenotypic or genotypic assays. Phenotypic assays measure drug susceptibilities of patient-derived viruses and provide direct evidence of drug resistance. However, phenotypic assays are culture-based, complex, laborious, and costly. Genotypic assays are frequently used to detect mutations associated with drug resistance by sequence analysis of the viral RNA from plasma. These assays are also complex and are insensitive to the detection of low levels of mutants, such as what might be present early in the emergence of resistance or which might persist at low set points in the absence of treatment. Commonly-used sequencing methods do not reliably detect mutants present at levels below 20-30% of the total viral population within a sample. Described in this application are PCR-based drug resistance detection assays that are able to detect drug-resistant viruses present at frequencies as low as 0.5%-0.04% within the plasma of infected persons. These sensitivities are 40-500-times greater than what has been achieved by conventional sequence testing.<br>
Although drug resistance is frequently seen in patients failing antiretroviral therapy, a substantial prevalence (~8-25%) of transmitted drug-resistant HIV-1 is found among drug-naive populations, supporting the need for baseline drug resistant testing. Because drug-resistant mutants are generally less fit than wild type viruses in the absence of drug, many drug-resistance mutations revert back to wildtype over time and become gradually undetectable in plasma. However, the drug-resistant viruses that become undetectable in plasma remain archived in the patients and are re-selected when drugs are used. Therefore, it is important to have sensitive assays that can accurately detect the presence of low frequency drug-resistant mutants. Data from the use of the sensitive real-time PCR assays described in this patent application demonstrate clearly that conventional sequencing of drug-naive persons underestimates the prevalence of transmitted drug resistance (Johnson et al., 13th HDR Workshop, Tenerife, Spain, 2004). Testing transmitted drug resistant viruses for additional mutations by the sensitive assays identified new mutants that increased the prevalence of resistance within the population by another 2 to 8%. The increases imply that drug resistance mutations are transmitted at frequencies 20-80% higher than previously reported. Therefore, these data demonstrate the poorer sensitivity of sequencing methods for baseline drug resistance testing.<br>
Drug resistance testing is also indicated for patients receiving HAART to manage treatment failures and to help guide the selection of new HAART regimens with active drugs. Recent data have pointed to the importance of sensitive drug resistance assays for this testing and associate low-frequency drug-resistant<br><br>
viruses that are not detectable by conventional sequencing with poor treatment outcomes (Mellors et al., llth CROI, 2004; Jourdain et al., JID 2004) (1). These studies reported that persons exposed to a non-nuclfclside reverse transcriptase inhibitor (NNRTI) who generated resistance mutations detectable only by sensitive assays, and not by conventional sequencing, respond more poorly to subsequent NNRTI-containing regimens. Data from the subtype C HIV-1 assays reported herein show that more than one-third of the drug-resistant viruses that emerge from intrapartum single-dose nevirapine intervention arc not identified by conventional population sequencing (Johnson et al., 12th CROI 2005). The detection of the substantial numbers of low-frequency drug-resistant viruses will be important for selecting a regimen with fully active drugs.<br>
In clinical monitoring of treated persons, the greater sensitivity of the present real-time PCR resistance assays over conventional sequencing may allow earlier detection of resistance mutations that emerge during treatment and provide advance notice of possible declines in response to therapy. Early detection will help guide clinicians in modifying drug regimens in an effort to prevent treatment failure and the emergence of high-level drug resistance. Methods with greater sensitivity in detecting low levels of resistant virus, below what is capable by conventional sequence analysis, are important for improving clinical management of patients under HAART. The substantially higher sensitivity, the simplicity, the high throughput capability, and the low cost of the present real-time PCR drug resistance assays are all advantages over conventional sequence analysis.<br>
SUMMARY OF THE INVENTION<br>
Disclosed are compositions including primers and probes, which are capable of interacting with the disclosed nucleic acids, such as the nucleic acids encoding the reverse transcriptase or protease of HIV as disclosed herein. Thus, provided is an oligonucleotide comprising any one of the nucleotide sequences set forth in SEQ ID NOS:l-89, and 96-104. Also provided is an oligonucleotide consisting of any of the nucleotide sequences set forth in SEQ ID NOS:l-89, and 96-104. Each of the disclosed oligonucleotides is a probe or a primer. Also provided are mixtures of primers and mixtures of primers and probes and for use in RT-PCR and primary PCR reactions disclosed herein. Kits comprising the primers or probes are provided. Provided are methods for the specific detection of several mutations in HIV. Mutations in both the reverse transcriptase and the protease of HIV can be detected using the methods described herein.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figures 1A and 1B are schematic illustrations of the principle of the present assay. Figure 2 A shows the sensitivity of the assay for 184V. Figure 2B shows the lower limit for 184V detection in clinical specimens. Figure 3 shows the performance of the 184V assay on clinical specimens. Figure 4 shows the ACT frequency distribution for clinical samples.<br>
Figure 5 shows the ACT values for real-time PCR analysis of the 103N mutation in pre-NVP and post-NVP plasma samples.<br>
DETAILED DESCRIPTION<br>
In an effort to improve the detection of mutations associated with HIV-1 drug resistance, provided are PCR-based point mutation assays. The present methodology allows testing for different point mutations in patient samples at an achievable sensitivity of 1-2 log greater than conventional sequencing. The principle of the present assay is to compare the differential amplifications of a mutation-specific PCR and a total<br><br>
copy (common) PCR, which detects all sequences present. The assay can use template generated from RT-PGR of viral RNA or from PCR of proviral DNA from infected cells (Fig.l).<br>
Two important HIV-1 reverse transcriptase mutations that significantly compromise the success of treatment with reverse transcriptase inhibitors are 103N and 184V. The 103N mutation is frequently selected in patients failing treatment with non-nucleoside RT inhibitors (e.g., ncvirapine, cfavirenz). Likewise, the frequent appearance of the 184V mutation following exposure to nucleoside inhibitors lamivudine (3TC) and abacavir, and it's seemingly rapid disappearance after discontinuation of therapy, makes accurate measure of these mutations important for surveillance and clinical management.<br>
The simplicity, greater sensitivity, and high-throughput capabilities of the present real-time PCR methodology make it useful for screening large numbers of samples, which allows the implementation of universal resistance testing and protracted surveillance of resistance mutations<br>
The methods disclosed herein have multiple applications including (1) resistance testing for clinical management of HIV-infected persons receiving anti-HIV drugs (for detecting emergence of resistant viruses in treated persons, and as a pre-treatment evaluation of patient baseline HIV in order to tailor the most appropriate drug combination), (2) use in blood bank screening as a nucleic acid test (NAT), due to the high sensitivity and high throughput capability of the assays, (3) the ability to measure plasma viral loads, since the assays are inherently quantitative, (4) use as a screening tool for monitoring the spread of resistant HIV, (5) use as a research tool to study the emergence and biology of drug resistance mutations, (6) detection of resistance mutations in both subtype B and non-B subtypes of HIV-1, (7) detection of resistance mutations in HIV-2, and (8) identification of specific panels of mutations that arc designed to address each of the described uses. The reagents and specific usages developed here are unique.<br>
As used in the specification and the appended claims, the singular forms "a," "an" and "the" include plural referents unless the context clearly dictates otherwise. Thus, for example, reference to "a primer" includes mixtures of two or more such primers, and the like.<br>
Compositions<br>
Disclosed are compositions including primers and probes, which are capable of interacting with the disclosed nucleic acids, such as the nucleic acids encoding the reverse transcriptase or protease of HIV as disclosed herein and in the literature.<br>
Thus, provided is an oligonucleotide comprising a nucleotide sequence as set forth in any of SEQ ID NOS:l-89, and 96-104. Also provided is an oligonucleotide consisting of any one of the nucleotide sequences set forth in SEQ ID NOS: 1-89, and 96-104. Thus, provided is an oligonucleotide comprising the sequence selected from the group consisting of the nucleotides as set forth in the sequence listing as SEQ ID NOS: 1-89, and 96-104. Each of the disclosed oligonucleotides is a probe or a primer. Each can be used independently of the others in an amplification method or in a hybridization/probing method. One or more of the probes or primers can be used together in the compositions and methods for detecting mutations. Specific examples of such compositions and methods are described herein.<br>
A nucleotide is a molecule that contains a base moiety, a sugar moiety and a phosphate moiety. Nucleotides can be linked together through their phosphate moieties and sugar moieties creating an internucleoside linkage. The base moiety of a nucleotide can be adenin-9-yl (A), cytosin-1-yl (C), guanin-9-yl (G), uracil-1-yl (U), and thymin-1-yl (T). The sugar moiety of a nucleotide is a ribose or a deoxyribose. The phosphate moiety of a nucleotide is pentavalent phosphate. A non-limiting example of a nucleotide would be 3'-AMP (3'-adenosine monophosphate) or 5'-GMP (5'-guanosine monophosphate). The term "nucleotide" includes nucleotides and nucleotide analogs, preferably groups of nucleotides comprising oligonucleotides, and refers to any compound containing a heterocyclic compound bound to a phosphorylated sugar by an N-glycosyl link or any monomer capable of complementary base pairing or any polymer capable of hybridizing to an oligonucleotide. <br>
The term "nucleotide analog" refers to molecules that can be used in place of naturally occurring bases in nucleic acid synthesis and processing, preferably enzymatic as well as chemical synthesis and processing, particularly modified nucleotides capable of base pairing. A nucleotide analog is a nucleotide which contains some type of modification to one of the base, sugar, or phosphate moieties. Modifications to nucleotides are well known in the art and would include for example, 5-methylcytosinc (5-me-C), 5-hydroxymethyl cytosine, xanthine, hypoxanthine, and 2-aminoadenine as well as modifications at the sugar or phosphate moieties. This term includes, but is not limited to, modified purines and pyrimidines, minor bases, convertible nucleosides, structural analogs of purines and pyrimidines, labeled, derivatized and modified nucleosides and nucleotides, conjugated nucleosides and nucleotides, sequence modifiers, terminus modifiers, spacer modifiers, and nucleotides with backbone modifications, including, but not limited to, ribose-modified nucleotides, phosphoramidates, phosphorothioates, phosphonamidites, methyl phosphonates, methyl phosphoramidites, methyl phosphonamidites, 5'-p-cyanoethyl phosphoramiditcs, methylenephosphonates, phosphorodithioates, peptide nucleic acids, achiral and neutral intcrnucleotidic linkages and nonnucleotide bridges such as polyethylene glycol, aromatic polyamidcs and lipids. Optionally, nucleotide analog is a synthetic base that does not comprise adenine, guaninc, cytosine, thymidine, uracil or minor bases. These and other nucleotide and nucleoside derivatives, analogs and backbone modifications are known in the art (e.g., Piccirilli J. A. et al. (1990) Nature 343:33-37; Sanghvi et al (1993) In: Nucleosides and Nucleotides as Antitumor and Antiviral Agents, (Eds. C. K. Chu and D. C. Baker) Plenum, New York, pp. 311-323; Goodchild J. (1990) Bioconjugate Chemistry 1:165-187; Beaucage et al. (1993) Tetrahedron 49:1925-1963).<br>
Nucleotide substitutes include molecules having similar functional properties to nucleotides, but which do not contain a phosphate moiety, such as peptide nucleic acid (PNA). Nucleotide substitutes are molecules that will recognize nucleic acids in a Watson-Crick or Hoogsteen manner, but which are linked together through a moiety other than a phosphate moiety. Nucleotide substitutes are able to conform to a double helix type structure when interacting with the appropriate target nucleic acid.<br>
There are a variety of molecules disclosed herein that are nucleic acid based. The disclosed nucleic acids are made up of for example, nucleotides, nucleotide analogs, or nucleotide substitutes. Non-limiting examples of these and other molecules are discussed herein.<br>
The term "oligonucleotide" means a naturally occurring or synthetic polymer of nucleotides, preferably a polymer comprising at least three nucleotides and more preferably a polymer capable of hybridization. Oligonucleotides may be single-stranded, double-stranded, partially single-stranded or partially double-stranded ribonucleic or deoxyribonucleic acids, including selected nucleic acid sequences, heteroduplexes, chimeric and hybridized nucleotides and oligonucleotides conjugated to one or more nonoligonucleotide molecules. In general, the nucleotides comprising a oligonucleotide are naturally occurring deoxyribonucleotides, such as adenine, cytosine, guanine or thymine linked to 2'-deoxyribose, or ribonucleotides such as adenine, cytosine, guanine or uracil linked to ribose. However, an oligonucleotide also can contain nucleotide analogs, including non-naturally occurring synthetic nucleotides or modified naturally occurring nucleotides. Such nucleotide analogs are well known in the art and commercially available, as are polynucleotides containing such nucleotide analogs (Lin et al., Nucl. Acids Res. 22:5220-5234 (1994); Jellinek et al., Biochemistry 34:11363-11372 (1995); Pagratis et al., Nature Biotechnol. 15:68-73 (1997), each of which is incorporated herein by reference).<br>
The term "polynucleotide" is used broadly herein to mean a sequence of two or more deoxyribonucleotides or ribonucleotides that are linked together by a phosphodiester bond. As such, the term "polynucleotide" includes RNA and DNA, which can be a gene or a portion thereof, a cDNA, a synthetic polydeoxyribonucleic acid sequence, or the like, and can be single stranded or double stranded, as well as a DNA/RNA hybrid. Furthermore, the term "polynucleotide" as used herein includes naturally occurring nucleic acid molecules, which can be isolated from a cell, as well as synthetic molecules, which can be prepared, for example, by methods of chemical synthesis or by enzymatic methods such as by the polymerase chain reaction (PCR). In various embodiments, a polynucleotide of the invention can contain nucleoside or nucleotide analogs, or a backbone bond other than a phosphodiester bond. In general, the nucleotides comprising a polynucleotide are naturally occurring deoxyribonucleotides, such as adenine,<br>
cytosine, guanine or thymine linked to 2'-deoxyribose, or ribonucleotides such as adcnine, cytosine, guanine or uracil linked to ribose. However, a polynucleotide also can contain nucleotide analogs, incluning non-naturally occurring synthetic nucleotides or modified naturally occurring nuclcotides. Such nucleotide analogs are well known in the art and commercially available, as are polynucleotides containing such nucleotide analogs (Lin et al, Nucl. Acids Res. 22:5220-5234 (1994); Jellinek et al., Biochemistry 34:11363-11372 (1995); Pagratis et al., Nature Biotechnol. 15:68-73 (1997), each of which is incorporated herein by reference).<br>
The covalent bond linking the nucleotides of a polynucleotide generally is a phosphodiestcr bond. However, the covalent bond also can be any of numerous other bonds, including a thiodiester bond, a phosphorothioate bond, a peptide-like bond or any other bond known to those in the art as useful for linking nucleotides to produce synthetic polynucleotides (see, for example, Tam et al., Nucl. Acids Res. 22:977-986 (1994); Ecker and Crooke, BioTechnology 13:351360 (1995), each of which is incorporated herein by reference). The incorporation of non-naturally occurring nucleotide analogs or bonds linking the nucleotides or analogs can be particularly useful where the polynucleotide is to be exposed to an environment that can contain a nucleolytic activity, including, for example, a tissue culture medium or upon administration to a living subject, since the modified polynucleotides can be less susceptible to degradation. Functional analogs of naturally occurring polynucleotides can bind to RNA or DNA, and include peptide nucleic acid (PNA) molecules.<br>
"Probes" are molecules capable of interacting with a target nucleic acid, typically in a sequence specific manner, for example through hybridization. The hybridization of nucleic acids is well understood in the art and discussed herein. Typically a probe can be made from any combination of nucleotides or nucleotide derivatives or analogs available in the art.<br>
"Primers" are a subset of probes which are capable of supporting some type of enzymatic manipulation and which can hybridize with a target nucleic acid such that the enzymatic manipulation can occur. A primer can be made from any combination of nucleotides or nucleotide derivatives or analogs available in the art which do not interfere with the enzymatic manipulation.<br>
The oligonucleotides of SEQ ID NOS:l-89, and 96-104 can be modified in insubstantial ways and yet retain substantially the same hybridization strength and specificity as described herein. These parameters are easily measured in assays such as those taught herein. Thus, one of skill in the art will be able to envision a number of nucleotide substitutions to the disclosed sequences, so long as they retain 80% sequence similarity with the specifically disclosed sequence. Primers and probes of the invention can include sequences having at least 85%, 90%, 95%, 96%, 97%, 98% or 99% similarity to one of SEQ ID NOS:l-89, and 96-104 are envisioned. More specifically, primers and probes with substitutions based on known sequences of the HIV-1 protease or reverse transcriptase are envisioned because these alternative sequences are envisioned by the person of skill in this art.<br>
In certain embodiments the primers are used to support DNA amplification reactions. Typically the primers are capable of being extended in a sequence specific manner. Extension of a primer in a sequence specific manner includes any methods wherein the sequence and/or composition of the nucleic acid molecule to which the primer is hybridized or otherwise associated directs or influences the composition or sequence of the product produced by the extension of the primer. Extension of the primer in a sequence specific manner therefore includes, but is not limited to, PCR, DNA sequencing, DNA extension, DNA polymerization, RNA transcription, or reverse transcription. Techniques and conditions that amplify the primer in a sequence specific manner are preferred. In certain embodiments the primers are used for the DNA amplification reactions, such as PCR or direct sequencing. It is understood that in certain embodiments the primers can also be extended using non-enzymatic techniques, where for example, the nucleotides or oligonucleotides used to extend the primer are modified such that they will chemically react to extend the primer in a sequence specific manner. Typically the disclosed primers hybridize with the disclosed nucleic acids or region of the nucleic acids or they hybridize with the complement of the nucleic acids or complement of a region of the nucleic acids.<br>
The oligonucleotides described herein include primers and probes effective for cross subtype reactive PCR, as such, they are capable of detecting mutations in a variety of HIV subtypes. The following printers and probes can also include additions known to those skilled in the art. Examples of such additions include, but are not limited to, molecules for linking the primer to a substrate, and the like. Furthermore, if desired, a nucleic acid molecule of the invention can incorporate a detectable moiety. As used herein, the term "detectable moiety" is intended to mean any suitable label, including, but not limited to, enzymes, fluorophores, biotin, chromophores, radioisotopes, colored particles, electrochemical, chemical-modifying or chemiluminescent moieties. Examples include (i) enzymes which can catalyze color or light emitting (luminescence) reactions and (ii) fluorophores. The detection of the detectable moiety can be direct provided that the detectable moiety is itself detectable, such as, for example, in the case of fluorophores. Alternatively, the detection of the detectable moiety can be indirect. In the latter case, a second moiety reactable with the detectable moiety, itself being directly detectable is preferably employed. The detectable moiety may be inherent to a molecular probe. Common fluorescent moieties include: fluorescein, cyanine dyes, coumarins, phycoerythrin, phycobiliproteins, dansyl chloride, Texas Red, and lanthanide complexes.<br>
Provided are the following: An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 1; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:2. An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 3; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 4; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 5; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 6; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 7; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 8; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 9; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 10. An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:11; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 12; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 13; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 14; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 15; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 16; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 17; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 18; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 19; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:20; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:21; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:22; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:23; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 24; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 25; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 26; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 27; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 28; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 29; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 30; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:31; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:32; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:33; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:34; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:35; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 36; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:37; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:38; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:39; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:40; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:41; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:42; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:43; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:44; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:45; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:46. An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:47; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:48; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:49; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:50; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:51; An oligonucleotide comprising the nucleotides as<br>
set forth in SEQ ID NO:52; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:53; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:54; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:55; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:56; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:57; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:58; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:59; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:60; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:61; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:62; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 63; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 64; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:65; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:66; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:67; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:68; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:69; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:70; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:71; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:72; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:73; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:74; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:75; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:76; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:77; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:78; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:79; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:80; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:81; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:82; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:83;. An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:84; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 85; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:86; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:87; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:88; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:89; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:96; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 97; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:98; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO:99; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 100; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 101; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 102; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 103; An oligonucleotide comprising the nucleotides as set forth in SEQ ID NO: 104. Thus, provided is an oligonucleotide comprising the sequence selected from group consisting of the nucleotides as set forth in the sequence listing as SEQ ID NO: 1-89, and 96-104.<br>
Also provided are mixtures of primers for use in RT-PCR and primary PCR reactions disclosed herein. Thus, a mixture of primers comprising SEQ ID NO:1 and 3 is provided. This mixture can be used for the reverse transcription-PCR (RT-PCR) reaction and the primary PCR reaction for HIV. It reverse transcribes and amplifies the HIV protease region comprising positions 30 and 90 in addition to the region of the reverse transcriptase gene comprising the mutations described herein.<br>
Provided is a mixture of primers comprising SEQ ID NOS:2 and 3. This mixture does not reverse transcribe or amplify the protease regions of interest, but is useful for the analysis of the reverse transcriptase.<br>
A mixture of primers comprising SEQ ID NOS:4 and 6 is provided. This mixture is for the RT-PCR and primary PCR reactions for HIV. It also reverse transcribes and amplifies the HIV protease region comprising positions 30 and 90 in addition to the region of the reverse transcriptase gene comprising the mutations described herein.<br>
Also provided is a mixture of primers comprising SEQ ID NOS:5 and 6. This mixture does not reverse transcribe or amplify the protease regions of interest, but is useful for the analysis of the reverse transcriptase.<br>
Provided are oligonucleotide mixtures for use in the mutation-specific PCR reactions disclosed herein. Detection can be achieved so long as any of the disclosed forward primers are paired with any of the reverse primers for a given mutation.<br>
Thus, provided is a mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:22,23,24 and 25. This is a forward primer mixture for the 103N mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer consisting of SEQ ID NO:26.<br>
Also provided is a mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS: 59,60 and 61. This is a forward primer mixture for the 103N mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer consisting of SEQ ID NO:26.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:33,34 and 35 is provided. This is a forward primer mixture for the 184V mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:36.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:88, 89, 102, 103, and 104 is provided. This is a forward primer mixture for the 184V mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:85.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:62,63,64,65,96 and 97 is provided. This is a forward primer mixture for the 41L mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:66.<br>
A mixture of primers comprising SEQ ID NOS: 10 and 98 and a reverse primer is provided. This mixture includes a forward primer for the 65R mutation-specific PCR reaction. The reverse primer can, for example, be a primer comprising or consisting of SEQ ID NO: 11.<br>
A mixture of primers comprising SEQ ID NOS:69 and 70 is provided. This is a forward primer mixture for the 67N mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:12,13, and 71 is provided. This is a forward primer mixture for the 69T mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:8 and 14.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:2,16,17,18,19, and 100 is provided. This is a forward primer mixture for the 70R mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:20,72, or 73.<br>
A mixture of primers comprising SEQ ID NOS:28 and 29 is provided. This is a forward primer mixture for the 181C mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:30. <br>
A mixture of primers comprising SEQ ID NOS: 83 and 84 is provided. This is a forward primer mixture for the protease 181C mutation-specific PCR reaction. The mixture can further include a reverse primer. For exarr^le, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 85.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:38,39,74,75, and 101 is provided. This is a forward primer mixture for the 215T mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:45.<br>
A mixture of primers comprising SEQ ID NO:40 and a reverse primer is provided. This is a primer mixture for the 215Y mutation-specific PCR reaction. The reverse primer can be, for example, a primer comprising or consisting of SEQ ID NO:45.<br>
A mixture of primers comprising SEQ ID NO:41 and a reverse primer is provided. This is a primer mixture for the 215F mutation-specific PCR reaction. The reverse primer can be, for example, a primer comprising or consisting of SEQ ID NO:45.<br>
A mixture of primers comprising SEQ ID NO:42 and a reverse primer is provided. This is a primer mixture for the 215S mutation-specific PCR reaction. The reverse primer can, for example, be a primer comprising or consisting of SEQ ID NO:45.<br>
A mixture of primers comprising SEQ ID NO:43 and a reverse primer is provided. This is a primer mixture for the 215C mutation-specific PCR reaction. The reverse primer can, for example, be a primer comprising or consisting of SEQ ID NO:45.<br>
A mixture of primers comprising SEQ ID NO:44 and a reverse primer is provided. This is a primer mixture for the 215D mutation-specific PCR reaction. The reverse primer can, for example, be a primer comprising or consisting of SEQ ID NO:45.<br>
A mixture of primers comprising SEQ ID NOS:48 and 49 is provided. This is a forward primer mixture for the protease 30N mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:50.<br>
A mixture of primers comprising one or more primers selected from the group consisting of SEQ ID NOS:53,54,55,78,79 and 80 is provided. This is a forward primer mixture for the protease 90M mutation-specific PCR reaction. The mixture can further include a reverse primer. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:56 and 81.<br>
Also provided are mixtures of primers for mutation-specific PCR reaction for reverse transcriptase and protease. These mixtures can comprise a forward and reverse primer for a reverse transcriptase mutation and a forward and reverse primer for a protease mutation. The forward primers in the mixture can include any forward primer for the specific RT mutation to be detected and any forward primer for the protease mutation to be detected. These mixtures can be used to simultaneously detect both an RT mutation and a protease mutation. An example of such a mixture of primers comprises or consists of SEQ ID NOS: 33,34, 35, 78,79, and 80. This is a forward primer mixture for the reverse transcriptase 184V and the 90M protease mutations. The mixture can further include reverse primers. For example, the reverse primers can comprise or consist of SEQ ID NOS: 36 and 81.<br>
The mixtures (and methods) disclosed herein can utilize reverse primers other than those exemplified. The exemplified reverse primers were found to work well. However, the requirements of the reverse primer in the present method are typical of reverse primers designed and used routinely, and other reverse primers can be routinely made and used. It is expected that the reverse primer will be within about 40 to 250 bases from the forward primer. It is also expected that the reverse primer will be positioned in a stable location lacking a degree of variability that would impede binding. The reverse primer is most likely to be located<br><br>
in the RT gene or the protease gene, but the exact location is routinely variable based on the usual criteria for reverse primer positioning.<br>
Amplification mixtures are provided that include a probe for use in a real time PCR reaction. The mixtures can thus include a forward primer, a reverse primer and a probe. For example, an amplification mixture is provided comprising a forward primer or a mixture of forward primers that amplifies the 103N, 65R, 69T and 70R mutations, wherein the mixture further comprises an oligonucleotide having the nucleotidcs as set forth in SEQ ID NO:9. This is an example of a probe that can be used in any of these mutation-specific PCR reactions. This probe can also be used in the total copy PCR reaction.<br>
An amplification mixture is provided comprising a forward primer or a mixture of forward primers that amplifies the 41L mutations, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NO:67. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
An amplification mixture is provided comprising a forward primer or a mixture of forward primers that amplifies the 65R, 67N, and 69T mutations, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NO:68. This is an example of a probe that can be used in any of these mutation-specific PCR reactions.<br>
An amplification mixture is provided comprising a forward primer or a mixture of forward primers that amplifies the 70R mutation, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:9 or 67. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the 181C and 184V mutations, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NO:32. This is an example of a probe that can be used in either of these mutation-specific PCR reactions.<br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the 215 mutations, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:47, 76, or 77. These are examples of probes that can be used in any of these mutation-specific PCR reactions.<br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the protease 30N mutation, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NO:52. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the protease 90M mutation, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:58 or 82. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the 103N mutation, wherein the mixture further comprises an oligonucleotide having the nucleotides as set forth in SEQ ID NO: 9. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the 181C mutation, wherein the mixture further comprises an oligonucleotide having the \ nucleotides as set forth in SEQ ID NOS:86 or 87. These are examples of probes that can be used in mutation-specific PCR reactions for this mutation.<br><br>
An amplification mixture is provided comprising a forward primer or mixture of forward primers that amplifies the 184V mutation, wherein the mixture further comprises an oligonucleotide having the nuciotides as set forth in SEQ ID NOS:86, or 87. These are examples of probes that can be used in mutation-specific PCR reactions for this mutation.<br>
The probe can incorporate a detectable moiety. As used herein, the term "detectable moiety" is intended to mean any suitable label, including, but not limited to, enzymes, fluorophores, biotin, chromophores, radioisotopes, colored particles, electrochemical, chemical-modifying or chemiluminescent moieties. Examples include (i) enzymes which can catalyze color or light emitting (luminescence) reactions and (ii) fluorophores. The detection of the detectable moiety can be direct provided that the detectable moiety is itself detectable, such as, for example, in the case of fluorophores. Alternatively, the detection of the detectable moiety can be indirect. In the latter case, a second moiety reactable with the detectable moiety, itself being directly detectable is preferably employed. The detectable moiety may be inherent to a molecular probe. Common fluorescent moieties include: fluorescein, cyaninc dyes, coumarins, phycoerythrin, phycobiliproteins, dansyl chloride, Texas Red, and lanthanide complexes.<br>
The size of the primers or probes for interaction with the nucleic acids can be any size that supports the desired enzymatic manipulation of the primer, such as DNA amplification or the simple hybridization of the probe or primer. A typical primer or probe would be at least, less than or equal to 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, . 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 125, 150, 175, 200, 225, 250, 275, 300, 325, 350, 375, 400, 425, 450, 475, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3500, or 4000 nucleotides long. Primers or probes of any length between the specified numbers are specifically contemplated.<br>
The primers for the reverse transcriptase gene or protease gene typically will be used to produce an ^ amplified DNA product that contains a region of the reverse transcriptase gene or protease gene containing the relevant site(s) of the mutation(s) of interest. In general, typically the size of the product will be such that the size can be accurately determined to within 3, or 2 or 1 nucleotides. This product can be at least, less than or equal to 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 125, 150, 175, 200, 225, 250, 275, 300, 325, 350, 375, 400, 425, 450, 475, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3500, or 4000 nucleotides long.<br>
In the mixtures and methods described herein, the specific probes described are merely examples. Applying routine skill to the teaching herein, the person in this field can envision and make additional probes that will function in the PCR compositions and methods described.<br>
A polynucleotide comprising naturally occurring nucleotides and phosphodiester bonds can be chemically synthesized or can be produced using recombinant DNA methods, using an appropriate polynucleotide as a v template. In comparison, a polynucleotide comprising nucleotide analogs or covalent bonds other than phosphodiester bonds generally will be chemically synthesized, although an enzyme such as T7 polymerase can incorporate certain types of nucleotide analogs into a polynucleotide and, therefore, can be used to produce such a polynucleotide recombinantly from an appropriate template (Jellinek et al, supra, 1995, incorporated herein by reference).<br>
For example, the nucleic acids, such as, the oligonucleotides to be used as primers can be made using standard chemical synthesis methods or can be produced using enzymatic methods or any other known ' method.   Such methods can range from standard enzymatic digestion followed by nucleotide fragment isolation (see for example, Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Edition (Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989) Chapters 5, 6, incorporated herein by reference) to purely synthetic methods, for example, by the cyanoethyl phosphoramidite method using a Mimgen or Beckman System 1Plus DNA synthesizer (for example, Model 8700 automated synthesizer of Milligen-Biosearch, Burlington, MA or ABI Model 380B). Synthetic methods useful for making oligonucleotides are also described by Ikuta et al., Ann. Rev. Biochem. 53:323-356 (1984), (phosphotriester and phosphite-triester methods, incorporated herein by reference), and Narang ct al., Methods Enzymol., 65:610-620 (1980) , incorporated herein by reference, (phosphotriester method). Protein nucleic acid molecules can be made using known methods such as those described by Nielsen et al., Bioconjug. Chem. 5:3-7 (1994), incorporated herein by reference.<br>
Also disclosed herein are kits that are drawn to reagents that can be used in practicing the methods disclosed herein. The kits can include any reagent or combination of reagents discussed herein or that would be understood to be required or beneficial in the practice of the disclosed methods. For example, the kits could include primers to perform the amplification reactions discussed in certain embodiments of the methods, as well as the buffers and enzymes required to use the primers as intended. Specific guidance as to the components of the kits is provided herein, including buffers, primers and probes. For example, disclosed is a kit for detecting a mutation in the reverse transcriptase gene or protease gene of HIV, comprising one or more of the oligonucleotides set forth in SEQ ID Nos:l-92.<br>
For further general information, an example of coding sequences of an HIV-1 protease and an HIV-1 reverse transcriptase are provided below. Also provided are accession numbers for these and other HIV-1 protease and an HIV-1 reverse transcriptase coding sequences. Accession numbers for amino acid sequences of the HIV-1 reverse transcriptase and the HIV-1 protease are also provided. This information, along with sequence information on many more examples of HIV-1 protease and reverse transcriptase proteins and coding sequences, are in the art. As such, they constitute a part of the disclosure of the present application.<br>
HIV-1 Subtype B Genome<br>
Accession Number: NC_001802, K03455<br>
HIV-1 Protease   <br>
Genome Location: 1799..2095<br>
Additional Similar Nucleotide Examples:   Accession Numbers: U31398, AJ279618, AJ279682, AJ279683, AJ279684<br>
Protein: Accession Number: NP_705926 HIV-1 Reverse Transcriptase<br>
Genome Location: 2096..3775<br>
Additional  Similar Nucleotide  Examples:     Accession Numbers:  U28646,  U28647,  U28648, U28649,U53870,U53871<br>
Protein: Accession Number: NP_705927<br>
HIV-1 Subtype C Genome<br>
Accession Number: AY162225, AY158533, DQ011180, DQ011173, AY049710<br>
HIV-1 Protease<br>
Genome Location: 2215.. .2511<br>
Additional   Similar  Nucleotide   Examples:      Accession  Numbers:      AYS 10039,   AYS 10043, AY589869<br>
Protein: Accession Number: AAR92431 HIV-1 Reverse Transcriptase<br>
Genome Location: 2512. ..3477<br>
Additional   Similar   Nucleotide   Examples:      Accession   Numbers:   AYS 10056,   AYS 10047, AY589935, AF468458<br>
Protein: Accession Number: AAR92448<br>
HIV-1 Subtype D Genome<br>
Accession Number: AY322189, AY773341, AJ320484<br>
HIV-1 Protease<br>
Exemplary Sequence<br>
Genome Location: 1719. ..2015<br>
Additional Similar Nucleotide Examples: Accession Numbers: AJ296664 Protein: Accession Number: CAC03695 HIV-1 Reverse Transcriptase<br>
Genome Location: 2016.. .2978<br>
Additional Similar Nucleotide Examples: Accession Numbers: AF388101<br>
Protein: Accession Number: AAL84043<br>
Methods<br>
Provided are methods for the specific detection of several mutations in HIV. Mutations in both the reverse transcriptase and the protease of HIV can be detected using the methods described herein. The methods are highly sensitive and specific. Specific examples of such methods are described. However, it is recognized that modifications of the exemplified methods using the alternative methods disclosed can be routinely accomplished. Any viral RNA can be used in the present invention. Such RNA is not limited to that obtained from plasma or serum, but can also be intracellular RNA that has not been packaged. Detection can be achieved so long as any of the disclosed forward primers are paired with any of the reverse primers for a given mutation. The following methods describe specific sets of primers that achieve especially sensitive levels of detection.<br>
A method for detecting the 103N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer selected from the group consisting of SEQ ID NO:3 and SEQ ID NO:6 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer set selected from the group consisting of SEQ ID NOS: 1,2,4 and 5 to produce a DNA product; and (c) contacting the DNA product of step (b) with a reverse primer and a primer set selected from the group consisting of SEQ ID NOS:22,23,24 and 25 and SEQ ID NOS:59,60 and 61 to amplify HIV-1 DNA containing the 103N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:26. In the methods disclosed, the presence of an amplification signal within a certain number of cycles after signal detection in the total copy PCR reaction indicates the presence of the respective mutation. This method, for use with an RNA template, detects the 103N mutation in either or both of Subtype B and Subtype C. SEQ ID NOS: 4 and 5 are forward RT-PCR (for RNA) and primary PCR (for DNA) primers for Subtype C. SEQ ID NO:4 includes protease sequences while SEQ ID NO:5 is for reverse transcriptase only. SEQ ID NOS: 1 and 2 are forward RT-PCR (for RNA) and primary PCR (for DNA) primers for Subtype B. SEQ ID NO:1 includes protease sequences while SEQ ID NO:2 is for reverse transcriptase only.<br>
Details of the RT-PCR (steps (a) and (b)) and secondary PCR (step (c)) for the detection methods starting with RNA are described in the Examples. In step (c) of these methods, a set of primers is used, including at least a primer pair comprising a reverse primer and one of the disclosed forward primers for the respective mutation. In step (b) of the methods starting with RNA, the choice of amplifying both the reverse transcriptase and the protease are provided by an exemplary primary PCR forward primer that includes protease and an exemplary primary forward primer for reverse transcriptase only.<br>
Each forward primer disclosed for the RT-PCR reaction or the primary PCR reaction in the methods disclosed works independently. If a protease analysis is to be done, then the Fl primers must be used for the RT-PCR or primary PCR steps. Reverse transcriptase analyses can be performed from the F2+reverse primer products alone (the F2 primers are slightly more sensitive than the Fl primers, thus can provide the user with a more sensitive test). In step (b) of the methods starting with RNA, there is reverse primer remaining in the reaction product from step (a).<br>
The RT step of the present methods can utilize RT primers other than those described. The only requirement is that the primers generate a template in the relevant region of the reverse transcriptase gene or in the protease gene or both.A further method for detecting the 103N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS: 1,2,4 and 5 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a reverse primer and a primer set selected from the group consisting of SEQ ID NOS:22,23,24 and 25 and SEQ ID NOS:59,60 and 61 to amplify HIV-1 DNA containing the 103N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:26. This method, for use with a DNA template, detects the 103N mutation in either or both of Subtype B and Subtype C.<br>
Details of the primary PCR and secondary PCR steps for the detection methods starting with DNA are described in the Examples. In step (b) of these methods, a set of primers is used, including at least a primer pair comprising a reverse primer and one of the disclosed forward primers for the respective mutation. In step (a) of the methods starting with DNA, the choice of amplifying both the reverse transcriptase and the protease are provided by an exemplary primary PCR forward primer that includes protease and an exemplary forward primer for reverse transcriptase only. Each forward primer disclosed for the primary PCR reaction in the method beginning with DNA works independently. Thus, the RT-only primer and the protease-included primer can be used independently with a reverse primer. If a protease analysis is to be done, then the Fl primers must be used for the RT-PCR or primary PCR steps. Reverse transcriptase analyses can be performed from the F2+reverse primer products alone (the F2 primers are slightly more sensitive than the Fl primers, thus can provide the user with a more sensitive test).<br>
Amplification methods are provided that include a probe for use in a real time PCR reaction. The methods can thus include the use of a forward primer, a reverse primer and a probe. For example, an amplification method is provided comprising a forward primer or a mixture of forward primers that amplifies the protease 90M, and the reverse transcriptase 103N, 65R, and 70R mutations, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:9. This is an example of a probe that can be used in any of these mutation-specific PCR reactions. This probe can also be used in the total copy PCR reaction.<br>
A method for detecting a Subtype B 184V mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:33,34 and 35 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 184V mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:36.<br>
A method for detecting a Subtype B 184V mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:land 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:33,34 and 35 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 184V mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:36.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 181C and Subtype B 184V mutations, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:32. This is an example of a probe that x can be used in either of these mutation-specific PCR reactions.<br>
A method for detecting a Subtype B 41L mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to<br>
produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c)contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:62,63,64 and 65 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 41L mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:66.<br>
A method for detecting a Subtype B 41L mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:63,96,97,64, and 65 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 41L mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:66.<br>
A method for detecting a Subtype B 41L mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS: land 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS: 62,63,64 and 65 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 41L mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:66.<br>
A method for detecting a Subtype B 41L mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:land 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS: 63,96,97,64, and 65 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 41L mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:66.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 41L mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:67. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype B 65R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer comprising SEQ ID NO: 10 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 65R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:11.<br>
A method for detecting a Subtype B 65R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and x (c) contacting the DNA product of step (b) with a primer comprising SEQ ID NO:98 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 65R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 11.<br>
A method for detecting a Subtype B 65R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer comprising SEQ ID NO: 10 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 65R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 11.<br>
A method for detecting a Subtype B 65R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer comprising SEQ ID NO:98 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 65R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 11.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 65R mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:9, 68, or 99. These are examples of probes that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype B 67N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:69 and 70 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 67N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8.<br>
A method for detecting a Subtype B 67N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:69 and 70 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 67N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 67N mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:68. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype B 69T mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:12 and 13 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 69T mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 14.<br>
A method for detecting a Subtype B 69T mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a)<br>
with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:12 and 71 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 69T mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8.<br>
A method for detecting a Subtype B 69T mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:12 and 13 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 69T mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 14.<br>
A method for detecting a Subtype B 69T mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:12 and 71 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 69T mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 69T mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:9 or 68. These are examples probes that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype B 70R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS: 16,17,18 and 19 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 70R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:20.<br>
A method for detecting a Subtype B 70R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NO:2 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 70R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:72 and 73.<br>
A method for detecting a Subtype B 70R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NO: 100 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 70R mutation. The reverse primer is routinely N selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:72 and 73.A method for detecting a Subtype B 70R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:16,17,18 and 19 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 70R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:20.<br>
A method for detecting a Subtype B 70R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NO:2 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 70R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:72 and 73.<br>
A method for detecting a Subtype B 70R mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NO: 100 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 70R mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:72 and 73.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 70R mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:9 or 67. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype B 103N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:22,23,24 and 25 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 103N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:26.<br>
A method for detecting a Subtype B 103N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:22,23,24 and 25 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 103N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:26.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 103N mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:9. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype B 181C mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a)<br>
with a primer selected from the group consisting of SEQ ID NOS:l and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:28 and 29 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 181C mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:30.<br>
A method for detecting a Subtype B 181C mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS: 28 and 29 and a reverse primer to amplify HIV-1 DNA containing a Subtype B 181C mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:30.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype B 181C mutation, wherein the method further comprises using an oligonuclcotide having the nucleotides as set forth in SEQ ID NO:32. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.A method for detecting a Subtype B 21ST mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 or SEQ ID NOS:74 and 75 or SEQ ID NOS: 101 and 75 to produce a DNA product; (c) contacting the DNA product of step (b) with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:38 and 39 to amplify HIV-1 DNA containing a Subtype B 215 mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:45.<br>
A method for detecting a Subtype B 215 mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; (c) contacting the DNA product of step (b) with a reverse primer and a primer selected from the group consisting of SEQ ID NOS :40,41,42,43 and 44 to amplify HIV-1 DNA containing a Subtype B 215 mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:45.<br>
A method for detecting a Subtype B 215 mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:40,41,42,43 and 44 to amplify HIV-1 DNA containing a Subtype B 215 mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:45.<br>
In the present methods of detecting a mutation at position Subtype B 215, any or all of the Y, F, S, C or D mutations can be detected. Thus, to detect any mutation at this position, the forward primers can be used together in the reaction mixture. To detect a specific mutation, the forward primer for that mutation would be used alone. Specific combinations of mutations at 215 can be identified by using the desired subset of the disclosed forward primers.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies Subtype B 215 mutations, wherein the method further comprises using an oligonucleotide having<br>
the nucleotides as set forth in SEQ ID NOS:47, 76, or 77. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting the 30N mutation in the protease of HIV-1 Subtype B is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:48 and 49 and a reverse primer to amplify HIV-1 DNA containing the 3ON mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:50.<br>
A method for detecting the 3ON mutation in the protease of HIV-1 Subtype B is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:land 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:48 and 49 and a reverse primer to amplify HIV-1 DNA containing the 3ON mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:50.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies the protease 3ON mutation of HIV-1 Subtype B, wherein the method further comprises the use of an oligonucleotide having the nucleotides as set forth in SEQ ID NO:52. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting the 90M mutation in the protease of HIV-1 Subtype B is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:53,54, and 55 and a reverse primer to amplify HIV-1 DNA containing the 90M mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:56.<br>
A method for detecting the 90M mutation in the protease of HIV-1 Subtype B is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS: 55,78,79, and 80 and a reverse primer to amplify HIV-1 DNA containing the 90M mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:81.<br>
A method for detecting the 90M mutation in the protease of HIV-1 Subtype B is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:53,54, and 55 and a reverse primer to amplify HIV-1 DNA containing the 90M mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:56.<br>
A method for detecting the 90M mutation in the protease of HIV-1 Subtype B is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:55,78,79, and 80 and a reverse primer to amplify HIV-1 DNA containing the 90M mutation.The reverse primer is routinely selected based on the well-known criteria for such selections, which arc described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:81.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies the protease 90M mutation, wherein the method further comprises the use of an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:58 or 82. These are examples probes that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype C 103N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:6 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:3 and 4 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:59,60, and 61 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 103N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:26.<br>
A method for detecting a Subtype C 103N mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS: 59,60, and 61 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 103N mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:26.An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype C 103N mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NO:9. This is an example of a probe that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype C 181C mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:6 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:3 and 4 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:83 and 84 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 181C mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:85.<br>
A method for detecting a Subtype C 181C mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:3 and 4 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:83 and 84 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 181C mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:85.<br>
An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype C 103N mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NOS: 86 or 87. These are examples of probes that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for detecting a Subtype C 184V mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:6 to<br>
produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:3 and 4 to produce a DNA product; and (c)contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:88 and 89 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 184V mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO: 85.<br>
A method for detecting a Subtype C 184V mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:6 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:3 and 4 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:102,103 and 104 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 184V mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:85.<br>
A method for detecting a Subtype C 184V mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS: 88 and 89 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 184V mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:85.<br>
A method for detecting a Subtype C 184V mutation in the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS: 102,103 and 104 and a reverse primer to amplify HIV-1 DNA containing a Subtype C 184V mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:85.An amplification method is provided comprising a forward primer or mixture of forward primers that amplifies a Subtype C 184V mutation, wherein the method further comprises using an oligonucleotide having the nucleotides as set forth in SEQ ID NOS:86 or 87. These are examples of probes that can be used in mutation-specific PCR reactions for this mutation.<br>
A method for amplifying the reverse transcriptase of HIV-1 is provided, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer comprising SEQ ID NO:7 and a reverse primer to amplify a region encoding the reverse transcriptase of HIV-1. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8. This can be a common amplification method of the invention. The total copy reaction can be used to provide the baseline for the mutation-specific real time PCR reactions disclosed herein. Alternatively, matched wildtype primers can be used as a control, as is known to one skilled in the art.<br>
A method for amplifying the reverse transcriptase of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:1 and 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer comprising SEQ ID NO:7 and a reverse primer to amplify a region encoding the reverse transcriptase of HIV-1. The reverse primer is routinely selected based on the well-known criteria for such selections, which arc described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NO:8.<br>
The amplification methods disclosed herein can utilize reverse primers other than those exemplified. The exemplified reverse primers were found to work well. However, the requirements of the reverse primer in thepresent method are typical of reverse primers designed and used routinely, and other reverse primers can be routinely made and used. It is expected that the reverse primer will be within about 40 to 250 bases from the forward primer. It is also expected that the reverse primer will be positioned in a stable location lacking variability to a degree that would impede binding. The reverse primer is most likely to be located in the RT gene or the protease gene, but the exact location is routinely variable based on the usual criteria for reverse primer positioning.<br>
Methods disclosed herein can further include detection, in the same mixture, of a specified RT mutation and a specified protease mutation. For example, provided is a method for detecting a 184V mutation in the reverse transcriptase of HIV-1 and a 90M mutation in the protease of HIV-1, comprising (a) reverse transcribing RNA extracted from HIV-1 with a primer comprising SEQ ID NO:3 to produce a reverse transcription reaction product; (b) contacting the reverse transcription product of step (a) with a primer selected from the group consisting of SEQ ID NOS:1 and 2 to produce a DNA product; and (c) contacting the DNA product of step (b) with a primer set comprising SEQ ID NOS:33,34, 35, 55, 78,79, and 80 and a reverse primer to amplify HIV-1 DNA containing a 184V and a 90M mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:36 and 81.<br>
A method for detecting a 184V mutation in the reverse transcriptase of HIV-1 and a 90M mutation in the protease of HIV-1 is provided, comprising (a) contacting DNA with a reverse primer and a primer selected from the group consisting of SEQ ID NOS:land 2 to amplify the DNA; and (b) contacting the amplified DNA of step (a) with a primer set comprising SEQ ID NOS:33,34,35,55,78,79, and 80 and a reverse primer to amplify HIV-1 DNA containing a 184V and a 90M mutation. The reverse primer is routinely selected based on the well-known criteria for such selections, which are described herein and elsewhere. For example, the reverse primer can be a primer comprising or consisting of SEQ ID NOS:36 and 81.<br>
A variety of technologies related to real-time (or kinetic) PCR have been adapted to perform point mutation and SNP detection. Mutation detection using real-time amplification relies on the ability to detect amplified segments of nucleic acid as they are during the amplification reaction. Three basic real-time detection methodologies exist: (i) increased fluorescence of double strand DNA specific dye binding, (ii) decreased quenching of fluorescence during amplification, and (iii) increased fluorescence energy transfer during amplification (Wittwer, C. et al. Biotechniques 22:130-138, 1997). All of these techniques are non-gel based and each strategy is disclosed.<br>
A variety of dyes are known to exhibit increased fluorescence in response to binding double stranded DNA. Production of wild type or mutation containing PCR products are continuously monitored by the increased fluorescence of dyes such as ethidium bromide or Syber Green as they bind to the accumulating PCR product. Note that dye binding is not selective for the sequence of the PCR product, and high non-specific background can give rise to false signals with this technique.<br>
A second detection technology for real-time PCR, known generally as exonuclease primers (e.g., TaqMan® probes), utilizes the 5' exonuclease activity of thermostable polymerases such as Taq to cleave dual-labeled probes present in the amplification reaction (Wittwer, C. et al. Biotechniques 22:130-138, 1997; Holland, P et al PNAS 88:7276-7280, 1991, incorporated herein by reference). While complementary to the PCR product, the probes used in this assay are distinct from the PCR primer and are dually-labeled with both a molecule capable of fluorescence and a molecule capable of quenching fluorescence. When the probes are intact, intramolecular quenching of the fluorescent signal within the DNA probe leads to little signal. When the fluorescent molecule is liberated by the exonuclease activity of the polymerase during amplification, the quenching is greatly reduced leading to increased fluorescent signal.<br>
An additional form of real-time PCR also capitalizes on the intramolecular quenching of a fluorescent molecule by use of a tethered quenching moiety. The molecular beacon technology utilizes hairpin-shaped moreeules with an internally-quenched fluorophore whose fluorescence is restored by binding to a DNA target of interest (Kramer, R. et al. Nat. Biotechnol. 14:303-308, 1996, incorporated herein by reference). Increased binding of the molecular beacon probe to the accumulating PCR product can be used to specifically detect SNPs present in genomic DNA.<br>
A final, general fluorescent detection strategy used for detection of point mutations and SNP in real time utilizes synthetic DNA segments known as hybridization probes in conjunction with a process known as fluorescence resonance energy transfer (FRET) (Wittwer, C. et al. Biotechniques 22:130-138, 1997; Bernard, P. et al. Am. J. Pathol. 153:1055-1061, 1998, incorporated herein by reference). This technique relies on the independent binding of labeled DNA probes on the target sequence. The close approximation of the two probes on the target sequence increases resonance energy transfer from one probe to the other, leading to a unique fluorescence signal. Mismatches caused by SNPs that disrupt the binding of cither of the probes can be used to detect mutant sequences present in a DNA sample.<br>
Parameters for selective hybridization between two nucleic acid molecules are well known to those of skill in the art. For example, in some embodiments selective hybridization conditions can be defined as stringent hybridization conditions. For example, stringency of hybridization is controlled by both temperature and salt concentration of either or both of the hybridization and washing steps. For example, the conditions of hybridization to achieve selective hybridization may involve hybridization in high ionic strength solution (6X SSC or 6X SSPE) at a temperature that is about 12-25°C below the Tm (the melting temperature at which half of the molecules dissociate from their hybridization partners) followed by washing at a combination of temperature and salt concentration chosen so that the washing temperature is about 5°C to 20°C below the Tm. The temperature and salt conditions are readily determined empirically in preliminary experiments in which samples of reference DNA immobilized on filters are hybridized to a labeled nucleic acid of interest and then washed under conditions of different stringencies. Hybridization temperatures are typically higher for DNA-RNA and RNA-RNA hybridizations. The conditions can be used as described above to achieve stringency, or as is known in the art. (Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989; Kunkel et al. Methods Enzymol. 1987:154:367, 1987 which is herein incorporated by reference for material at least related to hybridization of nucleic acids, which are incorporated herein in their entireties). A preferable stringent hybridization condition for a DNA:DNA hybridization can be at about 68°C (in aqueous solution) in 6X SSC or 6X SSPE followed by washing at 68°C. Stringency of hybridization and washing, if desired, can be reduced accordingly as the degree of complementarity desired is decreased, and further, depending upon the G-C or A-T richness of any area wherein variability is searched for. Likewise, stringency of hybridization and washing, if desired, can be increased accordingly as homology desired is increased, and further, depending upon the G-C or A-T richness of any area wherein high homology is desired, all as known in the art.<br>
The present invention is more particularly described in the following examples which are intended as illustrative only since numerous modifications and variations therein will be apparent to those skilled in the art.<br>
EXAMPLES<br>
The following examples are put forth so as to provide those of ordinary skill in the art with a complete disclosure and description of how the compounds, compositions, articles, devices and/or methods claimed herein are made and evaluated, and are intended to be purely exemplary of the invention and are not intended to limit the scope of what the inventors regard as their invention. Efforts have been made to ensure accuracy with respect to numbers (e.g., amounts, temperature, etc.), but some errors and deviations should be accounted for. Unless indicated otherwise, parts are parts by weight, temperature is in °C or is at ambient temperature, and pressure is at or near atmospheric.<br>
This example describes the development and application of real-time PCR-based point mutation assays for the 103N and 184V mutations in the reverse transcriptase (RT) of HIV-1. The assay measures the<br>
differential amplifications of total copy and mutation-specific reactions that target codons of interest. In evaluating mutation-containing plasmids diluted in backgrounds of wild type plasmid, the assays were able to achieve a mutation detection limit of 0.04% and 0.2% 103N and 184V, respectively. To evaluate the performance of these assays with clinical specimens, 77 known wild-type samples were first analyzed. None of the wild-type samples was positive for the 184V mutation, while one sample (1.3%) scored positive for 103N. Conversely, in plasma samples known to have viruses carrying the 103N mutation and/or the 184V mutation, 103N was detected in 54 of 55 positive specimens (98%), and 184V in 65 of 67 (97%). To determine whether any mutation-containing samples were undetected by conventional sequence analysis, the present assays were applied to a test population of HIV-1-positive treatment-naive persons documented to have RT mutations other than 103N and 184V. The 103N assay detected 4 positive samples (2.4%) in 165 plasma samples previously found absent of 103N (clones are currently being screened for the presence of low-level mutants). Likewise, in 173 samples previously determined to be negative for the 184V mutation, three samples scored positive (1.1%) for 184V by this assay. Two were later verified to have the mutation (at frequencies of 1.4% and 5.5%) by sequence analysis of clones. The data demonstrate that currently used sequence analysis is failing to detect resistant HIV-1 present as minority species in clinical specimens. The data also demonstrate that these real-time PCR assays for the detection of the 103N and 184V mutations are sensitive and specific. Given the low cost, high-throughput capability, and greater sensitivity than conventional testing, these assays will be useful for detecting drug resistance-associated mutations and could aid in the clinical management of HIV-1 infection.<br>
Clinical Samples<br>
Wild type HIV-1 subtype B samples were obtained from the plasma of 23 treatment-naive persons (2) with no detectable resistance mutations, and from 54 sera collected in the early 1980's, prior to the development of antiretroviral drugs. 67 specimens confirmed by sequence analysis to have virus carrying the mutation comprised mutation-positive samples. The test population encompassed a second group of 173 treatment-naive patients (partially referenced in 2), all with RT mutations other than 103N or 184V. Approximately 17% of the treatment-naive specimens were from persons documented to be recently infected. Results obtained from evaluation of the wild type and mutation-confirmed samples were used to define the assay cutoffs.<br>
Reverse Transcriptase-PCR<br>
HIV-1 genomic RNA was extracted (Qiagen UltraSens RNA kit) from patient plasma or serum. Primary amplifications of HIV-1 template were generated by reverse transcriptase-PCR using primers that demarcated the first half of the RT sequence, or when desired, the forward primer was shifted upstream to also include the entire protease region. The minimum copy numbers from which these reactions could successfully amplify were 5 and 10 RNA copies, respectively.<br>
Real-time PCR<br>
Baseline measurements for viral copies in test samples were determined using HIV-1 RT total copy primers with a total copy probe (Fig. 1A). Preliminary analysis for detecting the presence of the mutation was performed using primer mixtures to compensate for polymorphic variability in the primer binding sites. The 103N-specific mixture incorporated four different primers, and the 184V-specific reaction used three primers. The primers can be mixed at optimal ratios to equilibrate the differences in primer affinities. Examples of such ratios are provided below. It is recognized that the optimization of primer ratios is routine given the teaching of the primers and primer mixtures themselves. One of skill can envision alternative ratios.<br>
The mutation-specific primers were designed to maximize specificity for annealing to the mutated nucleotide(s), thus having a reduced affinity for wild type sequences (3,4). The probes for each reaction were 5'labeled with FAM and quenched with QSY-7. The choice of fluorophore and quencher can be routinely varied. Common fluorophores include HEX, ROX, Texas Red, TAMRA, JOE, Cy3, Cy5, SYBR and VIC. There are others that often overlap the above spectra and can be used. The Bio-Rad fluorophore table contains a more complete listing of fluorophores that can be used for this method.<br>
Degradation of the fluorescent probes during chain elongation removes the fluorophore from the proximity of the quencher and generates the fluorescent signals, reported as relative fluorescent units (RFU), that inerease with each amplification cycle (Fig. IB). The cycle number where the fluorescence emission exceeds the software-derived threshold is called the threshold cycle (CT) and is the unit of measure when comparing the differences in amplification levels (ACT) of the total copy and mutation-specific reactions.<br>
All reactions were performed using an iCycler real-time PCR system (Bio-Rad) and AmpliTaq Gold polymerase (Applied Biosystems). Any hot-start polymerase will work in this method. The differences between these are in their ability to extend from mismatched primers. Assay cutoffs (limits) arc established for each polymerase. Other usable polymerases include, but are not limited to, AmpliTaq Platinum (Applied Biosystems) and iTaq (Bio-Rad) . PCR annealing was at 50°C for 15 seconds and extension at 60°C for 30 seconds (See detailed PCT protocol below). Samples that were just above the cutoff (
Assay Sensitivity<br>
Assay detection limits were tested against dilutions of mutation-containing plasmid clones and from PCR products from both lab-adapted HIV-1 and patient-derived mutant virus spiked into a background of wild type virus template. The amounts of mutant input comprised 100%-0.001% of the total virus population.Protocol for HIV Real-Time PCR Point Mutation Assays<br>
I.	Sample preparation<br>
Extract viral RNA from  1000 : 1 plasma or proviral DNA from -7.5 x  105 cells (Qiagen<br>
Ultrasensitive ViralAmp or Promega Wizard Genomic DNA kits,<br>
respectively)<br>
II.	For RNA Template<br>
Primary (general) RT-PCR -<br>
Use 5 :1 extracted RNA per RT-PCR as follows:<br>
(RT step) -11  4 8 6 4 10 1<br>
Per reaction, add 5 :1 RNA to a total of 40 :1 of reagents prepared as follows:<br>
(final cone. 5 mM) (final cone. 1.5 mM each) (final cone. 400 nM) (20 U final) (50 U final) + 5:1 sample RNA 40 :1 total<br>
DEPC water 1 Ox buffer II MgCl2 dNTPs<br>
Reverse primer* RNase inhibitor* MuLV RT*<br>
*Heat the RNA in aliquotted mastermix for 2 minutes at 94°C then immediately place on ice prior to adding the RNase inhibitor and RT.<br>
T reaction:<br>
39°C for 1 hour, 94°C for 5 minutes, 4°C hold(PCR step) -<br>
Add the entire RT reaction to 60 :1 PCR mix prepared per reaction as follows:<br>
sterile water 1 Ox buffer Forward primer0 AmpliTaq LD<br>
48:1<br>
8:1<br>
3 :1 (final cone. 120nM)<br>
1:1 (5 U final)<br>
60 :1 mix<br>
+ 40 :1 RT reaction 100:1 total PCR<br>
PCR controls (in duplicate):   1) water = blank,<br>
2)	uninfected human DNA = (-),<br>
3)plasmid @ 1000 copies/rxn spiked in human DNA - (+), or a 105<br>
 → 102/10 :1   plasmid copy number series for quantitation (also see<br>
III.      For DNA Template Primary (general) PCR -<br>
Per reaction, add 10 :1 DNA (a higher concentration of template increases chances of encountering resistant pro viruses) to a total of 100 :1 of reagents prepared as follows:<br>
Sterile water 1 Ox buffer dNTPs<br>
Forward primer" Reverse primer" AmpliTaq (LD)<br>
67:1<br>
10:1<br>
6 :1 (final cone. 600 :M each)<br>
3 :1 (final cone. 120nM)<br>
3 :1 (final cone. 120nM)<br>
1 :1 (5 U final)<br>
+ 10 :1 sample DNA 100:1 total PCR  The 100 :1 may be diluted real-time PCR secondary overloaded with provide sufficient studies.<br>
1° PCR conditions:<br>
95 °C for 4 minutes<br>
95 °C for 45 seconds, 50°C for 30 seconds, 72°C for 2 minutes<br>
72°C for 5 minutes 4°C hold    <br>
x 40 cycles    <br>
primary PCR reaction (1:10-1:20) prior to the reaction to ensure the reaction is not template and to template for future<br>
IV.<br>
Mutation-specific (secondary) real-time PCR<br>
Principle - A sequence-specific probe, labeled with a fluorophore and a quencher, anneals to a region flanked by locus-specific primers. PCR extension from the primers degrades the intervening probe releasing the quencher from the proximity of the fluorophore, thus increasing the level of detectable fluorescence.   The amplification cycle at which the level of product (i.e., amount of degraded probe) is measurable above background, is the threshold cycle (TC).  This value directly correlates with the amount of starting template and is the unit of measure when making comparisons between amplification levels. Procedure-Use 2 :1 of each primary reaction in duplicate reactions of both a total copy and mutation-specific hot-start real-time PCR. Prepare for each of the secondary reactions:<br>
sterile water 1 Ox buffer Forward primer(s) Reverse primer dNTPs fluoro-probe AmpliTaq Gold (Perkin-Elmer/ABI)<br>
30.5 :1 5:1    <br>
4 :1 (final cone. 320 nM) 4 :1 (final cone. 320 nM) 2 :1 (final cone. 400 :M) 2:1 (final cone. 160nM) 0.5 :1 (2.5 U final) + 2:1 primary reaction 50 :1 total PCR reaction    <br>
2° Real-time PCR conditions:<br>
95 °C for 11 minutes (includes normalization time)<br>
95 °C for 30 seconds, 50°Cfor 15 seconds, 59 °C for 30 seconds<br>
x 50 cycles<br>
4 °C hold<br>
The mutation-specific tests(c) can be qualitative, by comparing to the common (total copy) primers (b) using only the 1000 copy positive control, or quantitative, by using the wild type and mutation-inclusive plasmid copy number dilution series. The quantitation can be performed without or in conjunction with a separate mutation-independent (mi) test (d), for quantitation by comparing the CT of the mutation reaction to the CT of the primer complementary to the shared overlapping sequence (i.e., same locus) (for examples of mi primers, see the primer list below). f The plasmid standards can be prepared and aliquotted as 40-cycle reactants of which 2 :1 are used for the copy standards in each secondary reaction plate.<br>
All mutation-specific PCRs are evaluated relative to the concomitant total copy (or wild type) reaction, the difference being ACT. Mutation-specific reactions with a ACT below the experimentally derived cutoff are scored positive.<br>
An advantage of the present invention is in detection sensitivity of the various subtypes of HIV from various countries of the world. For example, the sets disclosed in the primer set below are particularly sensitive to detection of HIV subtypes across the spectrum of HIV. ReTi-HIV Assay Oligonucleotide list:<br>
Primers below are for the listed mutations. All forward primers for each mutation can be mixed for general surveillance testing or the primers can be used individually or mixed and matched for detecting/monitoring distinct polymorphisms associated with that mutation.    The primer proportions exemplified for these mixtures are routinely adjustable using the optimization methods routinely practiced in this field. IUPAC codes: M= A or C; R= A or G; W= A or T; S= C or G; Y = C or T; K= G or T Notes: FAM=6-carboxyfluorescein, however, any fluorophore may be used; the ""marks arc where the quencher is placed<br>
67 and 69 REV are the same as the comREV primer<br>
§ Test performed in reverse orientation where the reverse primers detect the mutation &amp; Test for the wildtype codon (absence of mutation) + Same as the 41L probe £ Same as the RT-PCR primer <br>
RTPF2               <br>
      Simian Immunodeficiency virus (SIV) has strong clinical, pathological, virological and immunological analogies with HIV infection of humans. Infection of macaques with SIV provides a valuable model for exploring crucial issues related to both the pathogenesis and prevention of HIV infection. The model offers a unique setting for mutation detection testing, preclinical evaluation of drugs, vaccines and gene-therapies against HIV, and can identify many virus and host determinants of lentiviral disease. As such, the present invention can be utilized in conjunction with SIV nucleotide sequences. Provided below are exemplary SIV sequences for use with the present invention. The SIVmac 65R mutation-specific reaction can be compared against the total copy (common) reaction in the same way as described previously for HIV oligonucleo tides. Macaque SIV Reverse Transcriptase<br>
Accession number: AY588945, M33262, AY599201, AY597209, M19499<br>
Genome Location:  1954.. .4907<br>
Additional Similar Nucleotide Examples: Accession Numbers: U65787 Protein: Accession Number: AAV65312 SIVmac    <br>
Oligonucleotide Mixture Proportions<br>
The following is a list of primers disclosed above with an example of the ratios/proportions of these primers that can be used to specifically and sensitively detect the respective mutations.    <br>
103N and 184V Assay Sensitivity with Virus Mixtures<br>
When tested against plasmid clones the mixed-primer assay for 103N was able to distinguish as little as 0.04% 103N in within a wild type background.  The assay for 184V yielded discernable CTs for 184V plasmids when comprising as little as 0.2% of the population (Fig. 2 A). 103N and 181C Assay Performance on Clinical Samples<br>
To determine the overall assay performance on clinical specimens and establish the assay cutoff values, the data for the known patient-derived wild types and the 103N and 184V mutants were collated. An example of the performance of the 184V assay on a clinical specimen that carried the mutation and on a sample that had only wild type virus is shown in Fig. 3. The resulting distribution of collated ACTs revealed the best placement for the 103N cutoff to be a ACT of 12 and the 184V cutoff to be at a ACT of 8.5. That is, a ACT below these values is scored positive for the respective mutation, while a ACT above is scored as having only wild type (Fig. 4). As a group, the ACTs of the specimens documented to have mutations were significantly different from the ACTs of the wild type samples and samples possessing other mutations (P
Using the 12.0 ACT cutoff for the 103N assay, none of 69 specimens documented to have mutations other than 103N scored positive. With the 8.5 ACT cutoff for 184V, one specimen previously determined to be negative for 184V scored positive (ACT of 7.1), giving the assay an overall specificity of 98.6%. This discordant sample was from a chronically infected, treatment-naive person infected with virus carrying 41L and 215D RT mutations.<br>
Table I. ACT Measures for Each Group of Clinical Samples<br>
(Table Removed)<br>
Performance of the 70R, 90M and 67N Assays on transmitted drug-resistant viruses<br>
the Subtype B 70R assay cutoff = 9.0 cycles, 90M assay cutoff = 10.0 cycles, and 67N assay cutoff = 9.0 cycles.<br>
To reduce both the chance of false-positive results and the detection of naturally-occurring resistance-associated polymorphisms, assay cutoffs of 0.2-0.5% mutant virus were used for screening purposes. The sensitivities and specificities of the assays on genotyped clinical samples carrying the mutations of interest were found to range between 95-99%. Real-time PCR screening of the 147 transmitted HIV-1 carrying resistance-related mutations detected additional mutations that expanded the spectrum of drugs to which the viruses were resistant. The added mutants increased the prevalence of 90M from 8% to 10% (+25%), of 184V from 10% to 12% (+20%), of 70R from 9% to 14% (+56%), and of 67N from 7% to 12% (+71%).<br>
HIV-1 Subtype C 103N and 181C findings from a study examining the emergence of resistance in women receiving intrapartum single-dose nevirapine<br>
The subtype C HIV-1 103N assay cutoff = 1 1.0 cycles, and the 181C assay cutoff = 9.0 cycles.<br>
The 103N real-time assay confirmed the absence of detectable 103N in all 50 pre-NVP baseline samples (ACT range of 12.0-23.0 cycles, mean ACT= 15.9 cycles) (figure 5). The assay successfully detected 103N in all 10 post-NVP positive control specimens (ACT range of 2.8-9.8 cycles, mean ACT = 6.6 cycles). Of the 40 post-NVP specimens that had no detectable NVP -related mutations by sequencing, the real-time PCR assay found 16 (40%) were positive for 103N (ACT range of 6.9-10.6 cycles, mean ACT = 8.9 cycles) (figure 3, table 1). The ACT values for the new-found 103N-positive specimens were significantly lower than the pre-NVP specimens (AACT) (paired Figure 5 shows the real-time PCR analysis of the 103N mutation in pre-NVP and post-NVP plasma samples. Seq +/-, sequence analysis positive/negative for 103N; PCR+/-, real-time PCR positive/negative for 103N. A ACT value at or above the cutoff indicates 103N is not detected, a value below indicates the presence of 103N. T-tcst, p
The present real-time PCR primer-mix point mutation assay for the HIV-1 103N and 184V RT mutation were able to detect as little as 0.04% and 0.2% mutant virus, respectively, in HIV-1 plasmid dilutions. The primer designs were robust and worked well with the very high sequence variability in the clinical specimens examined. The ACTs of the mutation-positive specimens formed a distinct cluster from the wild type samples and samples with other mutations. These assays have shown acceptable performance on 282 samples of plasma-derived HIV-1, providing a sensitivity of 97.2-100% and a specificity of 98.6%.<br>
The benefits of real-time PCR-based testing include the following: 1) The real-time reaction requires a one-step setup, decreasing the potential for user error; 2) High throughput: reactions performed in 96-well plate allowing up to 40 patient samples per plate with results in 
References<br>
Throughout this application, various publications are referenced. The disclosures of these publications in their entireties are hereby incorporated by reference into this application in order to more fully describe the state of the art to which this invention pertains. More specifically, they are incorporated for the teaching recited in the context in which they are disclosed herein.<br>
1.	Low-Frequency NNRTI-resistant variants contribute to failure of efavirenz-containing regimens.   J.<br>
Mellors, S. Palmer, D. Nissley, M. Kearney, E. Halvas, C. Bixby, L. Demeter, S. Eshelman, K. Bennett, S.<br>
Hart, F. Vaida, M. Wantman, J. Coffin, and S. Hammer. Abstract from the 11th CROI, San Francisco, CA<br>
(Feb9-ll,2004).<br>
2.	The epidemiology of antiretroviral drug resistance among drug-naive HIV-1 infected persons in 10 U.S.<br>
cities. H.S. Weinstock, I. Zaidi, W. Heneine, D. Bennett, J.G. Garcia-Lerma, J.M. Douglas, Jr., M. LaLota,<br>
G. Dickinson, S. Schwarcz, L. Torian, D. Wendell, S. Paul, G.A. Goza, J. Ruiz, B. Boyett, and J.E. Kaplan.<br>
(JID in press)<br><br>
3.	Changes in human immunodeficiency virus type 1 populations after treatment interruption in patients<br>
failing antiretroviral therapy.   Hance AJ, Lemiale V, Izopet J, Lecossier D, Joly V, Massip P, Mammano<br>
F, Descamps D, Brun-Vezinet F, Clavel F. J Virol 2001 Jul;75(14):6410-7.<br>
4.	New real-time PCR Assay quantifies K103N NNRTI-resistant variant at a frequency as low as 0.01%. S<br>
Palmer, V. Boltz, F. Maldarelli, E Halvas, J Mellors and J Coffin. Abstract from the Third HIV DRP<br>
Symposium: Antiviral Drug Resistance, Chantilly, VA (Dec 8-11, 2002)<br>
It will be apparent to those skilled in the art that various modifications and variations can be made in the present invention without departing from the scope or spirit of the invention. Other embodiments of the invention will be apparent to those skilled in the art from consideration of the specification and practice of the invention disclosed herein. It is intended that the specification and examples be considered as exemplary only, with a true scope and spirit of the invention being indicated by the following claims.<br><br><br><br><br><br>
We claim:—<br>
1.	A method for detecting a mutation in a nucleic acid sequence relating to a HIV-1 protein of<br>
reverse transcriptase or protease in a subject, comprising:<br>
contacting a DNA molecule encoding the HIV-1 protein with a first primer and second primer to generate a first polymerase-chain-reaction (PCR) population of a common amplicon, wherein the DNA molecule is derived from a HIV-1 virus;<br>
isolating a portion of the first PCR population of the common amplicon; and<br>
contacting the portion of the first PCR population of the common amplicon with a forward primer and a reverse primer to generate a second PCR population of a mutation-specific amplicon.<br>
2.	The method of claim 1, wherein a detection limit for the mutation is less than 20% with the<br>
20% representing a percentage of HIV-1 viruses carrying the mutation in a total population of HIV-1<br>
viruses in the subject.<br>
3.	The method of claim 1  further comprising: obtaining the DNA molecule by reverse<br>
transcribing a RNA molecule encoding the HIV-1 protein in the subject.<br>
4.	The method of claim 1 further comprising: sequencing the second PCR population of the<br>
mutation specific amplicon.<br>
5.	The method of claim 1, wherein the mutation relating to the HIV-1 protein of reverse<br>
transcriptase is selected from the group consisting of 103N, 184V, 41L, 65R, 67N, 69T, 70R, 181C, 215T,<br>
215Y, 215F, 215S, 215C, and 215D.<br>
6.	The method of claim 1, wherein the mutation relating to the HIV-1 protein of protease is<br>
selected from the group consisting of 3ON and 90M.<br>
7.	The method of claim 1, wherein the forward primer for detecting the mutation relating to<br>
the HIV-1 protein of reverse transcriptase is selected from the group consisting of SEQ ID NOs: 22, 23,<br>
24, 25, 59, 60, 61, 33, 34, 35, 88, 89, 102, 103, 104, 62, 63, 64, 65, 96, 97, 10, 98, 69, 70, 7, 12, 13, 71, 2,<br>
16, 17, 18, 19, 100, 28, 29, 83, 84, 38, 39, 74, 75, 101, 40, 41, 42, 43, and 44.<br>
8.	The method of claim 1, wherein the forward primer for detecting the mutation relating to<br>
the HIV-1 protein of protease is selected from the group consisting of SEQ ID NOs: 48, 49, 53, 54, 55,<br>
78, 79, and 80.<br>
9.	The method of claim 1, wherein the reverse primer for detecting the mutation relating to<br>
the HIV-1 protein of reverse transcriptase is selected from the group consisting of SEQ ID NOs: 26, 36,<br>
85, 66, 11, 8, 14, 20, 72, 73, 30, 85, and 45.<br>
10.	The method of claim 1, wherein the reverse primer for detecting the mutation relating to<br>
the HIV-1 protein of protease is selected from the group consisting of SEQ ID NOs: 50, 56, and 81.<br>
11.	The method of claim 1, wherein the portion of the first PCR population of the common<br>
amplicon are further contacted by a probe selected from a group consisting of SEQ ID NOs: 9, 86, 87, 32,<br>
67, 68, 32, 47, 76, 77, 52, 58, and 82.<br>
12.	The method of claim 1, wherein the first primer is selected from the group consisting of<br>
SEQ ID NOs: 3 and 6.<br>
13.	The method of claim 1, wherein the second primer is selected from the group consisting of<br>
SEQ ID NOs:l,2,4, and 5.<br>
14.	An oligonucleotide primer composition for the detection of at least one mutation relating to<br>
a HIV-1 protein of reverse transcriptase by hybridizing with a nucleic acid encoding the HIV-1 protein of<br>
reverse transcriptase, wherein the at least one mutation is selected from the group consisting of 103N,<br>
184V, 41L, 65R, 67N, 69T, 70R, 181C, 215T, 215Y, 215F, 215S, 215C, and 215D, comprising:<br>
at least one of:<br>
a primer selective for the 103N mutation, the primer selected from the group of: SEQ ID NOs: 22, 23,24,25, 59,60, 61, and 26;<br>
a primer selective for the 184V mutation, the primer selected from the group of: SEQ ID NOs: 33, 34, 35, 88, 89, 102, 103, 104, 36, and 85;<br>
a primer selective for the 41L mutation, the primer selected from the group of: SEQ ID NOs: 62, 63, 64, 65, 96, 97, and 66;<br>
a primer selective for the 65R mutation, the primer selected from the group of: SEQ ID NOs: 10, 98, and 11;<br>
a primer selective for the 67N mutation, the primer selected from the group of: SEQ ID NOs: 69, 70, and 8;<br>
a primer selective for the 69T mutation, the primer selected from the group of: SEQ ID NOs: 12, 13, 71,8, and 14;<br>
a primer selective for the 70R mutations, the primer selected from the group of: SEQ ID NOs: 2, 16, 17, 18, 19, 100, 20, 72, and 73;<br>
a primer selective for the 181 C mutation, the primer selected from the group of: SEQ ID NOs: 28, 29, 83, 84, and 30;<br>
a primer selective for the 215T mutation, the primer selected from the group of: SEQ ID NOs: 38, 39,74,75,101, and 45;<br>
a primer selective for the 215Y mutation, the primer selected from the group of SEQ ID NOs: 40 and 45;<br>
a primer selective for the 215F mutation, the primer selected from the group of SEQ ID NOs: 41 and 45;<br>
a primer selective for the 215S mutation, the primer selected from the group of SEQ ID NOs: 42 and 45;<br>
a primer selective for the 215C mutation, the primer selected from the group of SEQ ID NOs: 43 and 45; and<br>
a primer selective for the 215D mutation, the primer selected from the group SEQ ID NOs: 44 and 45.<br>
15.	An oligonucleotide primer composition for the detection of at least one mutation relating to<br>
a HIV-1 protein of protease by hybridizing with a nucleic acid encoding the HIV-1 protein of protease,<br>
wherein the at least one mutation is selected from the group consisting of 3ON and 90M, comprising:<br><br>
at least one of:<br>
a primer selective for the 30N mutation, the primer selected from the group of: SEQ ID NOs: 48, 49, 50 for mutation 30N; and<br>
a primer selective for the 90M mutation, the primer selected from the group of: SEQ ID NOs: 53, 54, 55,78, 79, 80, 56, and 81.<br>
16.	A kit for detecting a mutation relating to a HIV-1 protein of reverse transcriptase or<br>
protease in a subject, comprising:<br>
a mutation-specific polymerase-chain-reaction (PCR) based detection assay for a nucleic acid encoding the HIV-1 protein of protease or reverse transcriptase; and<br>
instructions for performing the mutation-specific PCR-based detection assay to generate a first PCR population of a common amplicon for the HIV-1 protein, generate a second PCR population of a mutation-specific amplicon using a portion of the first PCR population of the common amplicon as template, and evaluate the ratio between the first PCR population and the second PCR population so as to detect the mutation in the nucleic acid encoding the HIV-1 protein.<br>
17.	A kit for detecting a mutation relating to a HIV-1 protein of reverse transcriptase or<br>
protease in a subject, comprising:<br>
a mutation-specific polymerase-chain-reaction (PCR) based detection assay for a nucleic acid encoding the HIV-1 protein of protease or reverse transcriptase, wherein a detection limit for the mutation is less than 20%; and<br>
instructions for performing the mutation-specific PCR-based detection assay to generate a first PCR population of a common amplicon for the HIV-1 protein, generate a second PCR population of a mutation-specific amplicon using a portion of the first PCR population of the common amplicon as template, and evaluate the ratio between the first PCR population and the second PCR population so as to detect the mutation in the nucleic acid encoding the HIV-1 protein.<br>
18.	A method for detecting a mutation in a nucleic acid substantially as herein described with<br>
reference to the foregoing description, examples, tables and the accompanying drawings.<br>
19.	An oligonucleotide primer composition substantially as herein described with reference to<br>
the foregoing description, examples, tables and the accompanying drawings.<br>
20.	A kit for detecting a mutation relating to a HIV-1 substantially as herein described with<br>
reference to the foregoing description, examples, tables and the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1BYnN0cmFjdC0oMjEtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Abstract-(21-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1BYnN0cmFjdC0oMzEtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Abstract-(31-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">19-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Bc3NpZ25tZW50LSgzMS0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Assignment-(31-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1DbGFpbXMtKDEyLTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Claims-(12-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1DbGFpbXMtKDMxLTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Claims-(31-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktREVMTlAtMjAwNy1DbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">19-DELNP-2007-Claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDAxLTExLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence Others-(01-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDA3LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence Others-(07-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDEyLTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence Others-(12-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE3LTEwLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence Others-(17-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDIwLTExLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence Others-(20-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDMxLTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence Others-(31-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDA2LTA2LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence-Others-(06-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDIxLTExLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Correspondence-Others-(21-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktREVMTlAtMjAwNy1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDI0LTAxLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-DELNP-2007-Correspondence-Others-(24-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktREVMTlAtMjAwNy1Db3JyZXNwb25kZW5jZS1PdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">19-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktREVMTlAtMjAwNy1EZXNjcmlwdGlvbiAoQ29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-DELNP-2007-Description (Complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1EcmF3aW5ncy0oMTctMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Drawings-(17-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">19-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Gb3JtLTEtKDMxLTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Form-1-(31-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">19-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Gb3JtLTItKDIxLTExLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Form-2-(21-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Gb3JtLTItKDMxLTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Form-2-(31-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">19-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1Gb3JtLTMtKDA3LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Form-3-(07-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktREVMTlAtMjAwNy1Gb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">19-DELNP-2007-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">19-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1HUEEtKDAxLTExLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-GPA-(01-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktREVMTlAtMjAwNy1HUEEtKDI0LTAxLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-DELNP-2007-GPA-(24-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1wY3QtMjM3LnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1wY3Qtbm90aWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-pct-notification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1wY3Qtc2VhcmNoIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">19-delnp-2007-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1QZXRpdGlvbi0xMzctKDA2LTA2LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Petition-137-(06-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTktZGVsbnAtMjAwNy1QZXRpdGlvbi0xMzctKDMxLTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">19-delnp-2007-Petition-137-(31-12-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258234-process-for-producing-coated-paper-and-coated-paper.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258236-an-auxotrophic-pseudomonas-cell-for-use-in-a-bacterial-expression-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258235</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>19/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Dec-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Dec-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Jan-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GOVERNMENT OF THE UNITED STATES OF AMERICA as represented by THE SECRETARY, DEPARTMENT OF HEALTH AND HUMAN SERVICES, CENTERS FOR DISEASE CONTROL AND PREVENTION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGY TRANSFER OFFICE, 4770 BUFORD HIGHWAY (K-79), ATLANTA, GA 30341, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HENEINE, WAILD</td>
											<td>2830 WOODLAND PARK DRIVE, ATLANTA, GA 30345, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JOHNSON JEFFREY, A</td>
											<td>4372 COLEMAN DRIVE, STONE MOUNTAIN GA 30083, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/70</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/19907</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-06-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/577 696</td>
									<td>2004-12-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258235-a-kit-for-the-detection-of-at-least-one-mutation-relating-to-a-hiv-1-protein-of-reverse-transcriptase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:13:29 GMT -->
</html>
