<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258203-method-for-detection-of-camp-and-cgmp by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:18:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258203:METHOD FOR DETECTION OF cAMP AND cGMP</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR DETECTION OF cAMP AND cGMP</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an in vitro method for detecting cAMP or cGMP comprising a) contacting a mixture with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher, and b) measuring the change in fluorescence. Furthermore, the present invention relates to the use of said method for determining the cAMP or cGMP concentration in the mixture, for determining the activity of a receptor wherein the signal transduction of the receptor comprises cAMP or cGMP and for screening of a ligand for such a receptor.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp; <br>
THE PATENS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
METHOD FOR DETECTION OF cAMP AND cGMP;<br>
F. HOFFMANN-LA ROCHE AG., A CORPORATION ORGANIZED AND EXISTING UNDER THE LAWS OF SWITZERLAND, WHOSE ADDRESS IS GRENZACHERSTRASSE 124, CH-4070 BASEL, SWITZERLAND.<br>
THE FOLLOWING SPECIFICATION PARTICULARLY DESCRIBES THE INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED.<br><br><br>
Nucleotides play an important role in the signal transduction of a cell. For example, the nucleotide cAMP is involved in the signal transduction of G-protein-coupled receptors (GPCRs).<br>
GPCRs form the largest class of pharmaceutical drug targets. In order to test the activity of potential new medicines on GPCRs in drug discovery, assays are needed which monitor the function of the receptors in their cellular environment. For high throughput screening, very early in the drug discovery process, these functional assays need to be simple with a low number of steps, sensitive to detect minute effects of early compounds and they need to comprise a robust readout to be applied in an automated fashion.<br>
Functional assays for GPCRs can be set-up by detection of second messenger molecules reflecting the activation state of the receptor. Such a second messenger molecule is cyclic adenosine monophosphate (cAMP) formed upon modulation of the adenylyl cyclase activity. An overview on commercially available, functional assay kits that determine the level of cellular cAMP based on fluorescence or chemiluminescence readout is given by W. Thomsen et al. (Current Opinion in Biotechnology 2005, 16, 655-665).<br>
These kits are time-consuming and complex. Therefore, the present invention provides a simple assay for detecting cAMP and its use for indirectly detecting receptor activity and for screening of ligands.<br>
This invention is based on the surprising fact that conjugates with cAMP derivates designed by modification of the nucleobase may quench the fluorescence emission of a fluorophore while cAMP does not quench the fluorophore.<br>
Therefore, the present invention provides an in vitro method for detecting cAMP in a mixture comprising a) contacting a mixture comprising cAMP with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher, and b) measuring the change in fluorescence.<br>
Surprisingly, the assay may also be used to detect cGMP in a mixture. Unlike cAMP has cGMP quenching capabilities, but only in a concentration of about ImM and above. However, cellular cGMP concentrations are usually in the range of several orders of magnitude below ImM.<br>
Therefore, the present invention provides an in vitro method for detecting cAMP or cGMP in a mixture comprising a) contacting a mixture comprising cAMP or cGMP with a<br><br><br>
complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher, and b) measuring the change in fluorescence.<br>
Preferably, the measured change of the fluorescence is compared with a control. The control may be a standard curve with predetermined amount of nucleotide.<br>
The term "mixture" refers to two or more substances mixed together in such a way that each remains unchanged. The mixture includes but is not restricted to cell lysates, cell culture supematants, biological fluids such as serum, plasma, urine, bronchial lavage fluid, sputum, biopsies like cerebrospinal fluid.<br>
The term "nucleotide" refers to a molecule comprising a nucleobase, a sugar, and one (MP), two (DP) or three (TP) phosphate groups. There are five nucleobases: Adenine (A), Uracil (U), Thymine (T), Guanine (G) and Cytosine (C). The sugar is either a ribose or a desoxyribose (d).<br>
The term "cyclic nucleotide" refers to a nucleotide in which the phosphate group is bonded to two of the sugar's hydroxyl groups, forming a cyclical or ring structure. These include cyclic AMP (cAMP), cyclic CMP (cCMP), cyclic TMP (cTMP), cyclic UMP (cUMP) and cyclic GMP (cGMP).<br>
The term "cAMP quencher" as used herein refers to a derivate of cAMP which is able to quench the fluorophore used in the assay. The quencher is covalently linked to said fluorophore. The cAMP quencher is a derivate of cAMP whereby the nucleobase is substituted with an electron donor. The position of the nucleobase for a substitution is position 2 or 8 as shown in Figure 1 IB. Preferred derivates of cAMP are 2-(6-Aminohexyl)amino-cAMP (2-AHA-cAMP), 8-(6-Aminohexyl)amino-cAMP (8-AHA-cAMP), 8- (8- Amino- 3, 6- dioxaoctylamino)-cAMP (8-ADOA-cAMP), 8-Hydroxy-cAMP (8-OH-cAMP), 8-(4-Mercaptobutylthio)-cAMP (8-MBT-cAMP). The most preferred derivative is 2-AHA-cAMP.<br>
The term "fluorophore" refers to a molecule emitting fluorescence when excited with a specific wavelength and being statically quenchable. A preferred fluorophore is an oxazine derivate as described in EP 747 447 such as for example MR121, Evoblue30 or JA314. Further preferred fluorophores which may be used are ATTO 590, ATTO 655, ATTO 680, Atto 700 (Atto-Tec GmbH, Am Eichenhang 50, 57076 Siegen, Germany). More preferably, the fluorophore is MR121 or Atto 700.<br><br><br>
The term "dequencher" as used herein refers to a molecule which reverses the quenching effect of the cAMP quencher by binding to the cAMP quencher. It is essential that the dequencher is able to bind the cAMP quencher as well as the cyclic nucleotide which shall be detected i.e. cAMP or cGMP.<br>
The dequencher may be a binding protein. Preferably, the dequencher is a cAMP-specific antibody. The antibody may be a monoclonal or a polyclonal antibody. If the dequencher is a protein, the pH value has to be adapted to a range in which the protein is capable of binding. Normally, this range is pH 6.8 to 7.8.<br>
Methods for producing antibody which are specific for a nucleotide are well known to the skilled in the art. Kohler and Milstein (Nature 1975, 256: 495-497), for example, describe methods for producing monoclonal antibodies.<br>
The detection complex comprises a tracer and a dequencher. The tracer consists of a fluorophore covalently linked to a cAMP quencher. The detection complex shows fluorescence if excited. When the detection complex is contacted with a nucleotide, the dequencher binds a certain percentage of the free nucleotide and leaves the detection complex. The tracer is now not anymore dequenched and shows only low or no fluorescence when excited. This decrease of the fluorescence is indicative for the amount of nucleotide present in the system.<br>
The change of fluorescence may be compared with a control such as a standard curve. A standard curve may be established by measuring the change in fluorescence for predetermined amounts of the nucleotide which shall be detected.<br>
The present invention further provides the use of the methods as described above for determining the cAMP concentration in a mixture. Furthermore, the methods as described above may be used for determining the activity of receptors wherein the signal transduction of these receptors comprises cAMP or cGMP. Preferably, the receptor is a G-protein coupled receptor (GPCR).<br>
Therefore, the present invention also provides a method for determining the activity of a receptor, wherein the signal transduction of this receptor comprises cAMP or cGMP, comprising<br>
a) contacting cells expressing the said receptor with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher,<br><br><br>
b)	lysing the cells and<br>
c)	measuring the change in fluorescence.<br>
Alternatively, step b) may be prior of step a).<br>
A preferred embodiment is a method for determining the activity of a GPCR comprising<br>
a)	contacting cells expressing the GPCR with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher,<br>
b)	lysing the cells and<br>
c)	measuring the change in fluorescence.<br>
Alternatively, step b) may be prior of step a).<br>
Preferably, the measured change of the fluorescence is compared with a control. The control may be a standard curve with predetermined amount of cAMP or cGMP.<br>
Furthermore, the present invention provides the use of the methods as described above for screening ligands of a receptor wherein the signal transduction of this receptor comprises cAMP or cGMP. Preferably, said receptor is a GPCR.<br>
Therefore, the present invention also provides a method for screening a ligand for a receptor, wherein the signal transduction of this receptor comprises cAMP or cGMP, comprising:<br>
a)	contacting a candidate compound with cells expressing said receptor,<br>
b)	adding a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher,<br>
c)	lysing cells and<br>
d)	measuring the change in fluorescence.<br>
Alternatively, step c) may be prior of step b).<br>
A preferred embodiment is a method for screening a ligand for a GPCR comprising<br>
a)	contacting a candidate compound with cells expressing said GPCR<br>
b)	adding to the cells a complex of a tracer and a dequencher, wherein the tracer is a<br><br><br>
fluorophore covalently linked to a cAMP quencher,<br>
c)	lysing cells<br>
d)	measuring the change in fluorescence.<br>
Alternatively, step c) may be prior of step b).<br>
Preferably, the measured change of the fluorescence is compared with a control. The control may be a standard curve with predetermined amount of nucleotide.<br>
A cell expressing a receptor such as for example GPCR may be a cell which expresses said receptor endogenously or a cell transgenic for said receptor.<br>
Such transgenic cell may be created by methods known in the art. A preferred method comprises the following steps: cloning a DNA which encodes the receptor of interest, inserting said DNA into a vector and introducing said vector into the cell. Preferably, said vector comprises stretches which are homologous to stretches in the genome of the cell suitable for homologous recombination and thereby allowing the targeted insertion of the DNA encoding the receptor into the genome of the cell.<br>
The term "ligand" as used herein refers to a molecule that binds to a receptor. A ligand may be an agonist, an antagonist, a modulator, partial agonist or a partial antagonist.<br>
The term "agonist" refers to a molecule that binds to a receptor and triggers a response in the cell. The term "partial agonist" refers to a molecule that partially activates a receptor. The term "antagonist" refers to a molecule that binds to the receptor but fails to activate the receptor and actually blocks it from activation by an agonist.<br>
The present invention also relates to a kit comprising a dequencher and a fluorophore covalently linked with cAMP quencher as described above. The kit may comprise the dequencher and the modified fluorophore separately or in a complex.<br>
The kit may also comprise any other components deemed appropriate in the context of measuring the level(s) of the respective fluorophores, such as suitable buffers, filters, etc.<br>
Optionally, the kit may additionally comprise a user's manual for interpreting the results of any measurement(s) with respect to determining the GPCR activity. Particularly,<br><br><br>
such manual may include information for the interpretation of the measured change in fluorescence, preferably a standard curve.<br>
Having now generally described this invention, the same will become better understood by reference to the specific examples, which are included herein for purpose of illustration only and are not intended to be limiting unless otherwise specified, in connection with the following figures.<br><br><br>
Figures<br>
Figure 1 shows the molecular structure of the fluorophore MR121 as NHS ester as it was used for labeling of the cAMP derivative in the examples.<br>
Figure 2 shows the molecular structure of MR121-2-AHA-cAMP.<br>
Figure 3 shows a schematic representation of the assay principle with the example of determining cAMP concentration in a cell. The cells are lysed and a detection mix is added to the cell lysates. The detection mix comprises a complex of a fluorophore (e.g. MR121) linked to a cAMP quencher (cAMP derivate, e.g. 2-AHA-cAMP) and cAMP antibody (e.g. mouse monoclonal anti-cAMP antibody). The quenching effect of the quencher on the fluorophore is revoked by antibody (= dequencher). When contacted with cAMP, the antibody binds it instead of the cAMP derivate. Without the dequenching effect of the antibody, the cAMP quencher quenches the fluorophore and causes a decrease of fluorescent signal. The higher the cAMP concentration, the lower is the fluorescence.<br>
Figure 4 shows a graphical representation of Stern-Volmer-Plots of MR121 and Tryptophane (a), 2-AHA-cAMP (b) and cAMP (c). Io/I: change in fluorescence intensity ( lo fluorescence intensity in the absence and I in the presence of a quencher), to/t: change of lifetime of the fluorescence (to fluorescence lifetime in the absence and t in the presence of a quencher).<br>
Figure 5 shows a graphical representation of the decrease of fluorescence intensity (I0/I) for different cAMP derivatives (8-AHA-cAMP, 2-AHA-cAMP, N-6-Aminohexyl-cAMP, 8-OH-cAMP, 8-ADOA-cAMP, 2'-AEC-cAMP, 8-MBT-cAMP). The large deviation from linearity for 8-AHA-cAMP above ~5 mM is due to solubility problems. Io/I: change in fluorescence intensity<br>
Figure 6: shows a graphical representation of the quenching and dequenching of the fluorescence of the tracer MR121-2-AHA-cAMP using two different specific cAMP antibodies A and B. Ab: Antibody<br>
Figure 7 shows a graphical representation of the titration of the anti-cAMP antibody at a fixed tracer concentration (10 nM MR121-2-AHA-cAMP). A fit of the experimental data resulted in a Kd value of 28 nM.<br>
Figure 8 shows a graphical representation of two cAMP standard curves, one without cells (filled diamonds) and one in the presence of cells (open diamonds), fitting of the curves gives IC50=109 nM and 119 nM without and with cells, respectively.<br><br><br>
Figure 9 shows a graphical representation of a Forskolin dose response curve and a cAMP standard curve with l0,000 cells/well, fitting of the curves gives lC5o=102 nM for cAMP and 3.7 uM for forskolin.<br>
Figure 10 shows a graphical representation of Stern-Vollmer Plots of the following molecules: MR121-8-AHA-cAMP, ATTO590-8-AHA-cAMP and ATT0655-8-AHA-cAMP.<br>
Figure 11 shows the chemical structure of cyclic adenosine monophosphate (cAMP) (A) and the purine ring of cAMP (B).<br>
Figure 12 shows the chemical structures of cAMP derivates A): 2-(6-Aminohexyl)amino-cAMP (2-AHA-cAMP), B): 8-Hydroxy-cAMP (8-OH-cAMP) C): 8-(4-Mercaptobutylthio)-cAMP (8-MBT-cAMP) D): 8- (8- Amino- 3, 6- dioxaoctylamino) (8-ADOA-cAMP) E): 8-(6-Aminohexyl)amino-cAMP (8-AHA-cAMP) F): N-6-Aminohexyl-cAMP G): 2'-AEC-cAMP (AEC = 3-Amino-9-ethyl carbazole)<br>
Figure 13 shows a graphical representation of the quenching of MR121 by cAMP, cGMP, cUMP and cTMP up to 13mM (a), (b) shows a graphical representation of the quenching of MR121 by cAMP and cGMP in the concentrations up to 10 (J.M in a Stern-Vollmer Plot. In low concentrations, neither cAMP nor cGMP quenches MR121.<br>
Figure 14 shows a graphical representation of the decrease of fluorescence in relation to the concentration of cAMP and cGMP (standard curves).<br>
Figure 15 shows a graphical representation of the decrease of fluorescence in relation to the concentration of cAMP (cAMP standard curve) with Atto700-2-AHA-cAMP as tracer and anti-cAMP antibody as dequencher. Fitting of the curve gives IC50=48 nM.<br><br><br>
Examples<br>
Commercially available reagents referred to in the examples were used according to manufacturer's instructions unless otherwise indicated.<br>
Reagents and Instrumentation<br>
cAMP, cGMP, 2-AHA-cAMP, 8-AHA-cAMP, 8-ADOA-cAMP, 2'-AEC-cAMP, 8-MBT-cAMP, N-6-Aminohexyl-cAMP and 8-OH-cAMP were purchased from Biolog Life Science Institute (28071 Bremen, Germany), MR121 NHS ester and N-6-Aminohexyl-cAMP from Roche diagnostics (Penzberg, Germany) and Atto 700 NHS ester from Atto-Tec GmbH (Atto-Tec GmbH, Am Eichenhang 50, 57076 Siegen, Germany). Monoclonal mouse anti-cAMP antibodies were produced in-house as described by Kohler and Milstein (Nature 1975, 256: 495-497)<br>
All experiments were performed in PBS (pH 7.4) containing 0.1% BSA (Albumin, bovine serum, &gt;96%, essentially fatty acid free, A6003, Sigma-Aldrich Chemie GmbH, Industriestrasse 25, CH-9471 Buchs, Switzerland). To lyse the cells a 3x PBS lysis buffer (pH 7.4) was used containing 0.1% BSA (Albumine from bovine serum, fraction V, &gt;96%, 05480, Sigma-Aldrich Chemie GmbH), 0.45% Triton® X-100 (9342, Sigma-Aldrich Chemie GmbH), 0.075% NP40 (Nonidet P40 Substitute, 19628, USB Corporation, Cleveland, OH USA) and 0.3%o NaN3 (purum p.a., &gt;99%, 71290, Sigma-Aldrich Chemie GmbH). Growth medium for the cells was F-12K (Gibco 21127-002) with 10% FCS and 1% Penicillin-Streptomycin (Gibco 15140-122).<br>
All fluorescence intensity measurements were carried out by means of a plate::vision fluorescence reader (Evotec Technologies GmbH, Schnackenburgallee 114, D-22525 Hamburg, Germany) equipped with a high pressure Xe arc lamp using for the measurements with MR121 an excitation filter at 630 nm (bandwidth 50 nm) and an emission filter at 695 nm (bandwidth 55 nm) and for the measurements with Atto 700 an excitation filter at 655 nm (bandwidth 50 nm) and an emission filter at 710 nm (bandwidth 40 nm). The fluorescent intensity was adjusted to about 60% of the maximal signal of the employed iCCD camera by using attenuation filters and varying exposure times. The lifetime measurements were performed by means of a plate::vision TRF reader (Evotec Technologies GmbH, Schnackenburgallee 114, D-22525 Hamburg, Germany) with an OPO system as light source (GWU Lasertechnik Vertriebsgesellschaft m.b.H., 50374 Erftstadt, Germany) tuned to 630 nm for excitation and the emission filter at 695 nm.<br><br><br>
All experiments were done in 384 well microtiter plates (Costar 384, black with clear, flat bottom, tissue culture treated, Prod. No. 3712), the total assay volume varying from 30 to 40 ul<br>
Example 1: Quenching of MR121<br>
1.1	Quenching capabilities of cyclic nucleotides<br>
20 \i\ MR121 (in PBS, end concentration 20 nM) and 20 ul quencher in PBS (DMSO end concentration in the assay: 1.25%) were pipetted into a microtiter plate. Fluorescence and lifetime were measured after an incubation time of 30 minutes at RT. The tested concentrations of the cyclic nucleotides were 0, 0.1, 0.2, 0.4, 0.8, 1.6, 3.2, 6.4 and 12.8 mM.<br>
cGMP quenches MR121 at higher concentrations (Figure 13a). However, this quenching does not affect the assay window because such concentrations are not reached in a cGMP assay. In low concentrations up to l0mM (corresponds to range of cellular concentrations), neither cAMP nor cGMP quench the fluorophore (Figure 13b). In the displacement assay with the MR121-2-AHA-cAMP tracer the assay window for displacement with cAMP or cGMP goes from 1 (MR121-2-AHA-cAMP dequenched upon binding to the antibody) to 1.7 I/I0 (MR121 quenched by 2-AHA-cAMP).<br>
1.2	Quenching with cAMP derivates<br>
In order to analyze the quenching mechanism, measurements of fluorescence intensity and fluorescence lifetime of free MR121 as a function of different molecules (Tip, cAMP, 2-AHA-cAMP, 8-AHA-cAMP, 8-ADOA-cAMP, 2'-AEC-cAMP, N-6-Aminohexyl-cAMP, 8-MBT-cAMP and 8-OH-cAMP, see Figure 4, 5) were carried out.<br>
20 ul MR121 (in PBS, end concentration 20 nM) and 20 ul quencher in PBS (DMSO end concentration in the assay: 1.25%) were pipetted into a microtiter plate. Fluorescence and lifetime were measured after an incubation time of 30 minutes at RT. A decrease in fluorescence intensity without a change in fluorescence lifetime is observed with Trp. Surprisingly the decrease of fluorescence intensity is even more pronounced for 2-AHA-cAMP, 8-AHA-cAMP, 8-ADOA-cAMP, 8-MBT-cAMP and 8-OH-cAMP (with constant fluorescence lifetime) while cAMP, 2'-AEC-cAMP and N-6-Aminohexyl-cAMP show no quenching at all. In analogy to references 2-5 we conclude that MR121 forms a non-fluorescent ground state complex with 2-AHA-cAMP, 8-AHA-cAMP, 8-ADOA-cAMP, 8-<br><br><br>
MBT-cAMP and 8-OH-cAMP but not with cAMP, 2'-AEC-cAMP and N-6-Aminohexyl-cAMP.<br>
In 2-AHA-, 8-AHA-, and 8-ADOA-cAMP the linker is coupled via an amine bond, in 8-MBT-cAMP via a sulfur and in 8-OH-cAMP via an oxygen bond directly to the purine ring system of adenosine. However in N-6-Aminohexyl-cAMP and 2'-AEC-cAMP the linker is coupled either to the amine group or the ribose moiety of adenosine. Apparently the modification of the adenosine leads to a change in the electronic states of the ring system which then favors the complex formation with the oxazine dye.<br>
In Figure 5 the change in fluorescence intensity (Io/I) is plotted for all cAMP derivatives investigated. The association constant, Ks, was calculated from the linear region (up to 3.2 mM) of the plot giving 514 M"1, 475 M"\ 512 M"', 758 M"1 and 236 M"1 for 8-AHA-cAMP, 8-ADOA-cAMP, 2-AHA-cAMP, 8-MBT-cAMP and 8-OH-cAMP, respectively. This quenching is very efficient, the Ks for most of the cAMP derivatives is more than twice compared to tryptophane reported in the literature (-220 M"1).<br>
When MR121-2-AHA-cAMP is bound to a specific cAMP antibody this non-fluorescent complex cannot be formed and an increase in MR121 fluorescence intensity is observed (Figure 6). The fluorescence intensity of 20 nM free MR121, 20 nM MR121-2-AHA-cAMP (maximum quenching) and a mixture of 20 nM MR121-2-AHA-cAMP and 1000 nM antibody (maximum dequenching) was measured in PBS containing 0.1% BSA.<br>
Compared to free MR121 (100%) the tracer shows a fluorescence intensity of 25%. Antibody A dequenches the fluorescence to 61%, antibody B to 87%. Apparently the degree of dequenching also depends on the binding affinity of the anti-cAMP antibody and on the amino acid sequence forming the binding domain of the antibody. A tryptophane in proximity to the binding domain of the antibody may quench the MR121 fluorescence to a certain degree. This can explain that different antibodies show different degrees of dequenching.<br>
For the assay it is crucial that the antibody recognizes the tracer which is a labeled and modified cAMP and "pure" cAMP produced in the cells. For the further development of the assay we used antibody A although the dequenching is less compared to antibody B. However, the displacement of the tracer with cAMP is better with antibody A and the window full quenching - full dequenching is still good enough to develop a robust assay.<br>
Example 2: Kd determination of anti-cAMP antibody<br><br><br>
The binding affinity of the anti-cAMP antibody (antibody A) to MR121-2-AHA-cAMP (Kd) was determined by titrating the antibody against several tracer concentrations (2, 6, 10, 15 and 20 nM) in lx lysis buffer. 20 uJ anti-cAMP dilutions were pipetted in 4 replicates into a 384 well microtiter plate. Then 20 ul MR121-2-AHA-cAMP were added to each well and the fluorescence intensity was read after 30 minutes incubation at RT on a 384 well shaker. The data were normalized by setting the values without antibody to 0%. Figure 7 shows the corresponding titration curves. Fitting to a four parameter model yields Kd values from 25 to 29 nM for the different tracer concentrations.<br>
Example 3: dose response curves<br>
cAMP<br>
Based on Kd=28 nM 40 nM anti-cAMP antibody and 20 nM MR121-2-AHA-cAMP were chosen for a sensitive detection of cAMP with an IC50 around 100 nM. Figure 8 displays two cAMP standard curves, one without cells and one with cells. For the standard curve without cells 20 ml cAMP dilutions in PBS containing 0.1% BSA were titrated into a 384 well microtiter plate and then 10 ul of the detection mix (120 nM anti-cAMP and 60 nM MR121-2-AHA-cAMP in 3x lysis buffer) were added. For the standard curve with cells l0'000 cells/well (CHO-K1) were plated in 25 ul medium and kept 20h at 37°C. The medium was then removed and 20 ul cAMP dilutions in PBS + 0.1% BSA followed by 10 ul detection mix were added. The plates were then incubated at RT on a 384 well shaker before reading the fluorescent signal. The determined IC50's are similar for both standard curves (109 and 119 nM without and with cells, respectively, see Figure 8).<br>
The stability of the cAMP standard curve with respect to time and to tolerability against DMSO and BSA were tested without cells using the detection mix in lysis buffer. With up to 5% DMSO and 0.8% BSA no change in IC50 and no significant drop of signal were observed. Signal and IC50 are stable for at least 6h.<br>
Figure 15 shows a cAMP dose response curve done with 5 nM Atto700-2-AHA-cAMP tracer and 5 nM antibody giving an IC50 of 48 nM. cAMP was diluted in PBS containing 0.1% BSA. 20 ul of the cAMP dilutions were titrated into a 384 well microtiter plate and 10 ul detection mix were added. The plate was incubated at RT on a 384 well shaker before reading the fluorescent signal.<br>
cGMP<br><br><br>
Figure 14 shows a comparison of the standard curves for cAMP and cGMP done with 20 nM MR121-2-AHA-cAMP tracer und 40 nM antibody giving an IC50 of 134 nM for cAMP and 387 nM for cGMP. These IC50 values as well as the sensitivity for cAMP and cGMP depend on the selection of the antibody. A different antibody may be more sensitive for cGMP and less sensitive for cAMP.<br>
cAMP and cGMP were diluted in PBS containing 0.1% BSA. For the standard curve without cells 20 ml of cAMP dilutions and 20ul cGMP dilutions were titrated into a 384 well microtiter plate and then 10 ul of the detection mix (40 nM anti-cAMP and 20 nM MR121-2-AHA-cAMP in 3x lysis buffer) were added. The plates were then incubated at room temperature on a 384 well shaker before reading the fluorescent signal.<br>
Example 4: cAMP cellular assay<br>
A forskolin dose response curve was performed with CHO-K1 cells. Cell numbers from 2'500 to 20'000 cell/well were plated in 25 ul medium and incubated for 20h at 37°C. The medium was removed and after adding 10 ul of PBS with 0.1% BSA and 1 mM IBMX the cells were incubated for 60 minutes at 37°C. 10 ul of forskolin dilutions in PBS + 0.1% BSA + 1 mM IBMX from 0.01 to 300 uM (final concentration in 20 ul assay volume) were added and incubated for another 30 minutes at 37°C. Then the cells were lysed by adding 10 ul detection mix (20 nM MR121-2-AHA-cAMP + 40 nM anti-cAMP, final concentration in 30 ul) in 3x lysis buffer and the fluorescence signal was read after 30 minutes incubation at RT on a 384 well shaker. With l0’000 cells/well the maximal cAMP level and minimal IC50 for Forskolin (3.7 uM) was reached. Figure 9 shows a Forskolin dose response curve and a cAMP standard curve with l0’000 cells/well.<br>
Example 5: Quenching of ATTO590 and ATT0655<br>
20 ul ATTO590 and ATT0655 respectively (in PBS, end concentration 20 nM) and 20 ul quencher (8-AHA-cAMP) in PBS (DMSO end concentration in the assay: 1.25%) were pipetted into a microtiter plate. Fluorescence and lifetime were measured after an incubation time of 30 minutes at RT. The results were incorporated in a Stern-Vollmer-Plot. The association constant, Ks, was calculated from the linear region (up to 3.2 mM) of the plot.<br><br><br>
Atto590 and Atto655 are quenched by 8-AHA-cAMP with Ks=282 M"1 and Ks=347 M"1, respectively (Figure 10).<br><br><br>
WE CLAIM<br>
1.	In vitro method for detecting cAMP or cGMP comprising<br>
a)	contacting a mixture comprising cAMP with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher, and<br>
b)	measuring the change in fluorescence.<br><br>
2.	The method according to claim 1, wherein the mixture is a cell lysate.<br>
3.	An in vitro method for determining the activity of a receptor wherein the signal transduction of the receptor comprises cAMP or cGMP<br><br>
a)	contacting cells expressing said receptor with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher,<br>
b)	lysing the cells and<br>
b) measuring the change in fluorescence.<br>
4.	An in vitro method for screening of a ligand for a receptor wherein the signal<br>
transduction of the receptor comprises cAMP or cGMP comprising<br>
a)	contacting a candidate compound with a cell expressing said receptor<br>
b)	adding to the cells a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher,<br>
c)	lysing cells, and<br>
d)	measuring the change in fluorescence.<br><br>
5.	The method according to claim 3 or 4 wherein the receptor is a GPCR.<br>
6.	The method according to any one of claims 1 to 5 wherein the cAMP quencher is 2-AHA-cAMP, 8-AHA-cAMP, 8-ADOA-cAMP, 8-OH-cAMP or 8-MBT-cAMP.<br>
7.	The method according to any one of claims 1 to 5, wherein the cAMP quencher is 2-AHA-cAMP.<br>
8.	The method according to any one of the claims 1 to 7, wherein the fluorophore is MR121, Evoblue30, JA314 or Atto700.<br>
9.	The method according to any one of the claims 1 to 8, wherein the measured change of the fluorescence is compared with a control.<br><br><br>
10.	Use of the method according to any one of the claims 1, 2, 6 to 9 for determining cAMP or cGMP concentration in a mixture.<br>
11.	A kit comprising a dequencher and a fluorophore covalently linked with a cAMP quencher.<br>
12.	The kit according of claim 11 wherein the cAMP derivative is 2-AHA-cAMP, 8-AHA-cAMP, 8-ADOA-cAMP, 8-OH-cAMP or 8-MBT-cAMP.<br>
13.	Methods and uses substantially as herein before described especially with reference to the foregoing examples.<br>
Dated this 18th day of December, 2007<br><br><br><br>
Abstract<br>
The present invention relates to an in vitro method for detecting cAMP or cGMP comprising a) contacting a mixture with a complex of a tracer and a dequencher, wherein the tracer is a fluorophore covalently linked to a cAMP quencher, and b) measuring the change in fluorescence. Furthermore, the present invention relates to the use of said method for determining the cAMP or cGMP concentration in the mixture, for determining the activity of a receptor wherein the signal transduction of the receptor comprises cAMP or cGMP and for screening of a ligand for such a receptor.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1hYnN0cmFjdC5kb2M=" target="_blank" style="word-wrap:break-word;">2492-mum-2007-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2492-mum-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1hc3NpZ25tZW50KDExLTEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2492-mum-2007-assignment(11-1-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1DTEFJTVMoQU1FTkRFRCktKDE1LTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-CLAIMS(AMENDED)-(15-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1DTEFJTVMoQU1FTkRFRCktKDUtMy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-CLAIMS(AMENDED)-(5-3-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1DTEFJTVMoTUFSS0VEIENPUFkpLSgxNS0xLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-CLAIMS(MARKED COPY)-(15-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1DTEFJTVMoTUFSS0VEIENPUFkpLSg1LTMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-CLAIMS(MARKED COPY)-(5-3-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1jbGFpbXMuZG9j" target="_blank" style="word-wrap:break-word;">2492-mum-2007-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">2492-mum-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1jb3JyZXNwb25kZW5jZSgxMC00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-mum-2007-correspondence(10-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1DT1JSRVNQT05ERU5DRSgxMS02LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-CORRESPONDENCE(11-6-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1DT1JSRVNQT05ERU5DRSgyMS0xMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-CORRESPONDENCE(21-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1jb3JyZXNwb25kZW5jZS1yZWNlaXZlZC5wZGY=" target="_blank" style="word-wrap:break-word;">2492-mum-2007-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-mum-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">2492-mum-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1FUCBET0NVTUVOVCgyMC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-EP DOCUMENT(20-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1GT1JNIDE4KDIxLTEyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-FORM 18(21-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtIDIodGl0bGUgcGFnZSktKDE4LTEyLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form 2(title page)-(18-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtIDMoMTAtNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form 3(10-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1GT1JNIDMoMTEtNi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-FORM 3(11-6-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1GT1JNIDMoMjAtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-FORM 3(20-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtLTIuZG9j" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1tdW0tMjAwNy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">2492-mum-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1HRU5FUkFMIFBPV0VSIE9GIEFUVE9STkVZKDE1LTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-GENERAL POWER OF ATTORNEY(15-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1HRU5FUkFMIFBPV0VSIE9GIEFUVE9STkVZKDUtMy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-GENERAL POWER OF ATTORNEY(5-3-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1KQVBBTkVTRSBET0NVTUVOVCgyMC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-JAPANESE DOCUMENT(20-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1QRVRJVElPTiBVTkRFUiBSVUxFIDEzNygyMC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-PETITION UNDER RULE 137(20-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1SRVBMWSBUTyBFWEFNSU5BVElPTiBSRVBPUlQoMTUtMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-REPLY TO EXAMINATION REPORT(15-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1SRVBMWSBUTyBFWEFNSU5BVElPTiBSRVBPUlQoMjAtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-REPLY TO EXAMINATION REPORT(20-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1SRVBMWSBUTyBFWEFNSU5BVElPTiBSRVBPUlQoNS0zLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-REPLY TO EXAMINATION REPORT(5-3-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1VUyBET0NVTUVOVCgxNS0xLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-US DOCUMENT(15-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1VUyBET0NVTUVOVCgyMC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-US DOCUMENT(20-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5Mi1NVU0tMjAwNy1VUyBET0NVTUVOVCg1LTMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2492-MUM-2007-US DOCUMENT(5-3-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="258202-a-method-for-the-preparation-of-hollow-cellulose-vessels.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258204-a-floating-combi-bar-and-method-of-producing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258203</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2492/MUM/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Dec-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Dec-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Dec-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F. HOFFMANN-LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>GRENZACHERSTRASSE 124,CH-4070 BASEL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THILO ENDERLE</td>
											<td>AM ROTEN WEG 9, D-79618 RHEINFELDEN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HUGUES MATILE</td>
											<td>FALKNERSTRASSE 1, CH-4001 BASEL</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DORIS ROTH</td>
											<td>SPALENVORSTADT 45, CH-4051 BASEL</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N21/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>06126927.0</td>
									<td>2006-12-21</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258203-method-for-detection-of-camp-and-cgmp by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:18:49 GMT -->
</html>
