<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258398-an-isolated-and-purified-enzyme-cocktail by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:43:08 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258398:&quot;AN ISOLATED AND PURIFIED ENZYME COCKTAIL&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED AND PURIFIED ENZYME COCKTAIL&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed herein are fungal nucleic acid sequences that encode fungal nucleic acid sequences that encode novel polypeptides. Also disclosed are polypeptides encoded by these nucleic acid sequences, as well as derivatives, variants, mutants, or fragments of the aforementioned polypeptide, polynucleotide, or antibody. The novel leucine aminopeptidase (LAP) and other amino and carboxypeptidase polypeptides, referred to herein as EXOX nucleic acids as proteins of the invention are useful in a variety of medical, research, and commercial applications.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL FUNGAL PROTEINS AND NUCLEIC ACIDS ENCODING SAME<br>
FIELD OF THE INVENTION<br>
The present invention relates to novel polypeptides, and the nucleic acids encoding them, having unique catalytic properties. More particularly, the invention relates to nucleic acids encoding novel leucine aminopeptidase (LAP) and other amino- and carboxy-peptidases polypeptides, which will be herein collectively referred to as EXOX, as well as vectors, host cells, antibodies, and recombinant methods for producing these nucleic acids and polypeptides. These genes have been identified in two different fungal species, Trichophyton rubrum and Aspergillusfianigatus.<br>
BACKGROUND OF THE INVENTION<br>
Bacteria, yeast and filamentous fungi, as well as specialized cells of plants, invertebrates and vertebrates express membrane proteins useful for the uptake of amino acids, dipeptides and tripeptides. Lubkowitz et al., Microbiology 143:387-396 (1997); Hauser et al.,Mo\. Membr. Biol. 18(1):105-112 (2001); Stacey 
Aminopeptidase activity, which may also play a role in the development of fungus during infection, has been detected in the mycelium and culture supernatant of a species of fungi (De Bersaques &amp; Dockx, Arch. Belg. Dermatol. Syphiligr. 29:135-140 (1973); Danew &amp; Friedrich, Mykosen 23:502-511 (1980)), however, no aminopeptidase or carboxypeptidase has been isolated and characterized from dermatophytes to date.<br>
SUMMARY OF THE INVENTION<br>
The invention is based in part upon the discovery of isolated polypeptides containing the mature form of an amino acid sequence selected from SEQ ID NOs: 3,6,9,12,15,18, 21,24,27,30, 33, and 35. The invention also provides isolated polypeptides containing an amino acid sequence selected from SEQ ID NOs. 3,6, 9,12,15,18, 21, 24,27, 30, 33, and 35, as well as isolated polypeptides that are at least 90% identical to polypeptides having these sequences, wherein the polypeptide optionally has aminopeptidase or carboxypeptidase activity. For example, the polypeptide may be a leucine aminopeptidase such as ruLAP2.<br>
Also provided are isolated polypeptides having one or more conservative amino acid substitutions. Such polypeptides may possess aminopeptidase activity.<br>
The invention also encompasses polypeptides that are naturally occurring allelic variants of the sequence selected from the group consisting of SEQ ID NOs: 3,6,9,12,15, 18,21,24,27, 30,33, and 35. These allelic variants include amino acid sequences that are the translations of nucleic acid sequences differing by one or more nucleotides from nucleic acid sequences selected from the group consisting of SEQ ID NOs: 3,6, 9,12,15,18,21,24,27, 30,33, and 35. The variant polypeptide where any amino acid changed in the chosen sequence is changed to provide a conservative substitution.<br>
The invention also involves a method of removing particular amino acids from peptides, for instance tags from recombinant proteins, wherein the active polypeptide removing amino acid is a polypeptide having an amino acid sequence at least 90% identical to a polypeptide having the amino acid sequence selected from the group consisting of SEQ ID NOs: 3, 6,9,12,15,18, 21,24,27, 30, 33, and 35, or a biologically active fragment thereof.<br>
Any of the polypeptides of the invention may be naturally occurring. Further, any of these polypeptides can be in a composition including a carrier, and the composition can be in a kit including one or more containers.<br>
Also provided are dermatophytes containing the polypeptides of the invention. For example, suitable dermatophytes include Epidermophytonfloccosum, Microsporum audouinii, Microsporum ferrugineum, Trichophyton concentricum, Trichophyton kanei, Trichophyton megninii, Trichophyton mentagrophytes, Trichophyton raubitschekii, Trichophyton rubrum, Trichophyton schoenleinii, Trichophyton soudanense, Trichophyton tonsurans, Trichophyton violaceum, Trichophyton yaoundei, Microsporum canis,<br>
Microsporum equinum, Microsporum nanum, Microsporum persicolor, Trichophyton equinum, Trichophyton simii, Trichophyton verrucosum, Microsporum gypseum, Trichophyton ajelloi, and Trichophyton terrestre.<br>
The invention also provides microbial culture supernatants containing the polypeptides of the invention.<br>
The invention also relates to the use of therapeutics in the manufacture of a medicament for treating a syndrome associated with a human disease, where the therapeutic includes the polypeptides of the invention and the disease is selected from a pathology associated with these polypeptides.<br>
The invention also relates to methods of degrading a polypeptide substrate. Such methods include contacting the polypeptide substrate with one or more of the polypeptides, which have been isolated. For example, the polypeptide substrate can be a full-length protein. Further, the one or more isolated polypeptides can be used to sequentially digest the polypeptide substrate. The polypeptide substrate can be selected from denatured casein, gliadin, gluten, bovine serum albumin or fragments thereof. For example, the isolated polypeptide can be an aminopeptidase, which can be a leucine aminopeptidase such as mLAP2.<br>
The invention further relates to methods for identifying a potential therapeutic agent for use in treatment of fungal infections, wherein the fungal infection is related to aberrant expression or aberrant physiological interactions of the polypeptides of the invention. Such methods include providing a cell expressing the polypeptide and having a property or function ascribable to the polypeptide, contacting the cell with a composition comprising a candidate substance, and determining whether the substance alters the property or function ascribable to the polypeptide. If no alteration is observed in the presence of the substance when the cell is contacted with a composition in the absence of the substance, the substance is identified as a potential therapeutic agent. For example, the property or function ascribable to the polypeptide can be aminopeptidase or carboxypeptidase activity.<br>
The invention further relates to methods of treating a pathological state in a mammal by administering a polypeptide to the mammal in an amount that is sufficient to alleviate the pathological state. Typically, the polypeptide has an amino acid sequence at least 90% identical to a polypeptide containing the amino acid sequence selected from SEQ ID NOs: 3, 5,9,12,15,18,21,24,27,30,33, and 35, or a biologically active fragment thereof. The jathological state to be treated include a fungal infection, celiac disease, digestive tract<br>
malabsorption, sprae, an allergic reaction and an enzyme deficiency. For example, the allergic reaction can be a reaction to gluten.<br>
The invention additionally relates to methods of treating a pathological state in a mammal by administering a protease inhibitor to the mammal in an amount that is sufficient to alleviate the pathological state. The protease inhibitor includes an amino acid sequence at least 90% identical to a polypeptide having the amino acid sequence selected from SEQ ID NOs:3,6,9,12,15,18,21,24,27,30, 33, and 35, or a biologically active fragment thereof. For example, the pathological state can be a fungal infection.<br>
The invention further relates to isolated polypeptides having an amino acid sequence selected from SEQ ID NOs: 3,6,9,12,15,18,21,24,27,30,33, and 35. These polypeptides can be produced by culturing a cell under conditions that lead to expression of the polypeptide. In some embodiments, the cell includes a vector containing an isolated nucleic acid molecule having a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 2, 5,8,11,14,17,20,23,26,29,32, and 34. Optionally, the cell may be a fungal cell, a bacterial cell, an insect cell (with or without a baculovirus), a plant cell and a mammalian cell.<br>
The invention also provides isolated nucleic acid molecules containing a nucleic acid sequence selected from SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26,29,32, and 34. For example, such nucleic acid molecules can be naturally occurring.<br>
The invention also relates to nucleic acid molecules that differ by a single nucleotide from a nucleic acid sequence selected from SEQ ID NOs: 2,5,8,11,14,17, 20,23,26,29, 32, and 34 as well as to isolated nucleic acid molecules encoding the mature form of a polypeptide having an amino acid sequence selected from the group consisting of SEQ ID NOs: 3,6,9,12, 15,18,21,24,27,30, 33, and 35. Further, the nucleic acid molecules can be ones that hybridizes under stringent conditions to the nucleotide sequence selected from the group consisting of SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26, 29,32, and 34 or a complement of that nucleotide sequence. In some embodiments, the nucleic acid molecules can be included in a vector, that further includes a promoter operably linked to said nucleic acid molecule. Also provided are cells that include the vector.<br>
The invention also provides methods of producing polypeptides of the invention. The methods include culturing a cell under conditions that lead to expression of the polypeptide and the cell includes a vector having an isolated nucleic acid molecule containing a nucleic acid sequence selected from SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26,29, 32, and 34. In<br>
some instances, the cell is selected from a fungal cell, a bacterial cell, an insect cell, a plant cell or mammalian cell.<br>
The invention also relates to methods for producing a protein by culturing a dermatophyte containing the polypeptide under conditions sufficient for the production of the protein and isolating the protein from the dermatophyte culture. For example, the protein can be a secreted protein. Likewise, the protein can also be an aminopeptidase or a carboxypeptidase. Specifically, the aminopeptidase can be a Icucine aminopeptidase, such as ruLAP2. Additionally, the dermatophyte can be selected from Epidermophyton floccosum, Microsporum audouinii, Microsporum ferrugineum, Trichophyton concentricum, Trichophyton kanei, Trichophyton megninii, Trichophyton mentagrophytes, Trichophyton raubitschekii, Trichophyton rubrum, Trichophyton schoenleinii, Trichophyton soudanense, Trichophyton tonsurans, Trichophyton violaceum, Trichophyton yaoundei, Microsporum canis, Microsporum equinum, Microsporum nanum, Microsporum persicolor, Trichophyton equinum, Trichophyton mentagrophytes, Trichophyton simii, Trichophyton verrucosum, Microsporum gypseum, Trichophyton ajelloi, and Trichophyton terrestre.<br>
The produced proteins can be applied to polypeptide substrates. In some instances, the produced protein can degrade the polypeptide or can sequentially digests a full-length polypeptide substance. Optionally, the polypeptide substrate length can be from 2 to 200 amino acids.<br>
In some instances, the produced protein adds one or more amino acids to the polypeptide substrate. In other instances, the produced protein removes one or more amino acids from the polypeptide substrate to form a modified polypeptide substrate, and the produced protein subsequently adds one or more amino acids to the modified polypeptide substrate, thereby forming a polypeptide product comprising a different amino acid sequence than the polypeptide substrate.<br>
The invention also provides methods for treating mycoses in a patient suffering therefrom. Such methods include administering an effective amount of an inhibitor with the activity of an EXOX protein selected from SEQ ID NOS:3,6,9,12,15,18,21,24,27,30, 33, and 35. For example, the EXOX protein can include SEQ ID NO: 2.<br>
The invention further provides methods of degrading a polypeptide substrate. These methods include contacting the polypeptide substrate with one or more of the isolated polypeptides of the invention. Optionally, the polypeptide substrate is a full-length protein, and the one or more isolated polypeptides can be polypeptides that sequentially digest the<br>
polypeptide substrate. The polypeptide substrate can be selected from denatured casein, gliadin, gluten, bovine serum albumin or fragments thereof. Further, in some instances, the isolated polypeptide is an aminopeptidase. The aminopeptidase can be a leucine aminopeptidase, such as ruLAP2.<br>
Additionally, the method optionally contacting the polypeptide substrate with one or more proteases. Li some instances, the proteases are selected from trypsin, pronase, chymotrypsin, and proteinaseK.<br>
The invention further provides methods of removing amino acids from the amino terminus of a protein. The methods include contacting the protein with one or more of the isolated polypeptides of the invention, hi some instances, the amino terminus of a protein includes a His tag. In other instances the amino terminus of a protein includes an Xaa-Pro tag. Optionally, Xaa is an amino acid including at least two vicinal nucleophilic groups, with examples including serine, thieonine ot cysteine.<br>
The invention further provides isolated polypeptides of the invention that can have reverse proteolytic activity.<br>
The invention further provides methods of adding one or more amino acids to a polypeptide substrate. The method includes contacting the polypeptide substrate with one or more of the isolated polypeptides of the invention.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used hi the practice of the present invention, suitable methods and materials are described below. All publications, patent applications, patents, and other references mentioned herein are incorporated by reference in then* entirety. In the case of conflict, the present specification, including definitions, will control. In addition, the materials, methods, and examples are illustrative only and are not intended to be limiting.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 is a photograph of a Western blot of T. rubrum supernatant preparation probed with anti-A. oryzae Alp (Panel A, left) and Mep antisera (Panel C, right). Panel B shows a 10% SDS-PAGE gel stained with Coomassie blue. In lane 1, the proteins of 0.25 ml of T. rubrum culture supernatant were precipitated with TCA before loading on the SDS-PAGE gel. 0.2 g of purified recombinant A. oryzae ALP and MEP were loaded on lane 2 and lane 3, respectively. The molecular mass of protein standards are shown in the left margin.<br>
FIG. 2 is a photograph of a SDS-PAGE gel illustrating a protein profile of recombinant ruLAP2 (1,2), fuLAP2 (3,4), ruLAPl (5,6) and fuLAPl (7, 8) produced in P. pastoris. 1 g of each purified recombinant LAP was loaded on a 10% SDS-PAGE gel. Lanes 2,4,6 and 8 show the proteins deglycosylated by N-glycosidase F treatment. The gel was stained with Coomassie brilliant blue R-250.<br>
FIG. 3 is a photograph of a Western blot of T. rubrum culture supernatant and recombinant LAPs used as controls probed with anti-ruLAP2 (lanes 1-4) and anti-ruLAPl antisera (lanes 5-8). In lane 1,2,5 and 6 the proteins of 0.25 ml of T. rubrum culture supernatant was precipitated with TCA before loading on the SDS-PAGE gel. 0.1 g of purified recombinant ruLAP2 (lanes 3,4) and ruLAPl (lanes 7, 8) was loaded as a control. N-glycosidase F was used for deglycosylation of proteins. The molecular mass of protein standards are shown in the left margin.<br>
FIG. 4 is a graph of the enzymatic activity of T. rubrum AMPP (aminopeptidase P) at various pH values. It appears that AMPP has activity over a broad range of pH values, from pH 6 to 11.<br>
FIG. 5 is a graph of the enzymatic activity of T. rubrum AMPP at various temperatures. The enzyme exhibits activity at temperatures ranging from 25 to 60 C with an optimal temperature of 50C.<br>
FIG. 6 is a graph showing the digestion of gliadin 14mer with ruLAPl over 4h at 37°C with an E/S ratio (w:w) of 1/50.<br>
FIG. 7 is a graph showing the digestion of gliadin 14mer with DPPIV alone and with a DPPIV/ruLAP2 cocktail.<br>
FIG. 8 is a graph showing the digestion of gliadin 33mer with ruDPPIV over 4h at 37°C with an E/S ratio (w:w) of 1/50.<br>
FIG. 9 is a graph showing the digestion of gliadin 33mer with a DPPIV/ruLAP2 cocktail.<br>
FIGS. 10A and 10B are mass spectrum of Gly-Ser-proNPY (A) before and (B) after digestion with ruLAP2.<br>
FIGS. 11A and 1 IB are mass spectra of Ala-proNPY (A) before and (B) after digestion with ruLAP2.<br>
FIGS. 12A and 12B are mass spectra of TG47 (A) before and (B) after digestion with ruLAP2.<br>
FIGS. 13A and 13B are mass spectra of desMet-G-CSF (A) before and (B) after digestion with DPPIV.<br>
FIG. 14 is an alignment of deduced amino acid sequences of aminopeptidases of the M28E subfamily.<br>
FIG. 15 is an alignment of deduced amino acid sequences of aminopeptidases of the M28A subfamily.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As used herein, the term protease is synonymous with peptidase, proteolytic enzyme and peptide hydrolase. The proteases include all enzymes that catalyse the cleavage of the peptide bonds (CO-NH) of proteins, digesting these proteins into peptides or free amino<br>
acids. Exopeptidases act near the ends of polypeptide chains at the amino (N) or carboxy (C)<br>
terminus. Those acting at a free N terminus liberate a single amino acid residue and are termed aminopeptidases. A large variety of highly specific proteases are involved in a number of different biological and physiological processes. Thus, these represent targets of choice for new drug applications as well as for controlled peptidic and/or proteic degradations.<br>
Dermatophytes are human and animal pathogenic fungi, which cause cutaneous infections. Vanbreuseghem et al., GUIDE PRATIQUE DE MYCOLOOIE MEDICALE ET VETERINAIRE. (1978); Kwong-Chong &amp; Bennet, MEDICAL MYCOLOGY (1992); Weitzman &amp; Summerbell, Clin. Microbiol. Rev. 8:240-259 (1995). Examples of dermatophytes include, but are note limited to, T. ajelloi, A. uncinatum, K. ajelloi, T. aster aides, T. mentagrophytes, T. concentricum, T. cruris, E.floccosum, T. dankalienese, G. dankaliensis, T. equinum, T. equinum var. autotrophicum, T. equinum var. equinum, T. erinacei, T.fischeri, T.flavescens, T.floccosum, E.floccosum, T. gloriae, T. gourvilii, T. granulare, T. granulosum, T. gypseum, T. inguinale, T. interdigitale, T. intertriginis, T. kanei, T. krajdenii, T. longfusum, T. megninii, A. quinckanum, A. benhamiae, A. vanbreuseghemii, T. pedis, T. proliferans, T. quickaneum, T. radiolatum, T. mentrophytes var. erinacei, T. mentagrophytes var. interdigitale, T. mentagrophytes var. mentagrophytes, T. mentagrophytes var. nodulare, T. mentagrophytes var. quinnckeanum, T. niveum, T. nodulare, T. persicolor, M. persicolor, T. phaseolforme, T. proliferans, T. purpureum, T. quinckeanum, T. radiolatum, T. raubitschekii, T. rubrum, S. ruber, T. schoenleinii, T. simii, A. simii, T. soudanense, T. sulphureum, T. tonsurans, A. insingulare, A. lenticularum, A. quadrifldum, T. tonsurans, T. sulphureum, T. terrestre, T. tonsurans var. sulphureum, T. tonsurans var tonsurans subvar. perforans, T. vanbreuseghemii, T. verrucosum, T. violaceum, T. yaoundei, E.floccosum, M. audouinii, M. ferrugineum, T. kanei, T. megninii, T. mentragrophytes, T. raubitschekii, T. schoenleinii, T. Soudanese, T. violaceum, M. canis, M. equinum, M. nanum, M. persicolor, T. verrucosum, and M. gypseum. Among the pathogenic species isolated in hospitals and private practices in Europe, Trichophyton rubrum, T. mentagrophytes and Microsporum canis are most commonly observed. Monod etal, Dermatology, 205:201-203 (2002). In fact, dermatophytes can grow exclusively in the stratum comeum, nails or hair, and digest components of the cornified cell envelope. To date, all investigated dermatophytes produce proteolytic activity in vitro and many investigators report the isolation and characterization of one or two secreted endoproteases from an individual species. For a review, see Monod et al, Int. J. Med. Microbiol. 292:405-419 (2002). In particular, M. canis was shown to possess<br>
two gene families encoding endoproteases of the 88 (subtilisins) and M36 (fungalysins) family as classified in the MEROPS proteolytic enzyme database (at http://merops.sanger.ac.uk/). Brouta et al., Infect. Immun. 70:5676-5683 (2002); Descamps et al., J Invest. Dermatol. 70:830-835 (2002). One member of each isolated M. canis gene family encoded one of the two previously characterized endoproteases from culture supernatants. Mignon et al, Med. Mycol. 36:395-404 (1998); Brouta et al., Med. Mycol. 39:269-275 (2001). Both enzymes were shown to be keratinolytic and produced during infection in cats. Mignon et al, Med. Mycol. 36:395-404 (1998); Brouta et al, Med. Mycol. 39:269-275 (2001). This proteolytic activity enables derrnatophytes to grow exclusively in the stratum corneum, nails or hair, and to use digested components of the cornified cell envelope, i.e., single amino acids or short peptides, as nutrients for in vivo growing.<br>
Two new leucine aminopeptidases (LAP) from the dermatophyte T. rubrum, ruLAPl and ruLAP2 are described herein. T. rubrum is a species of the genus Trichophyton, which includes, e.g., T. ajelloi, T. asteroides, T. mentagrophytes, T. concentricum, T. cruris, T. dankalienese, T. equinum, T. equinum var. autotrophicum, T. equinum var. equinum, T. erinacei, T.fischeri, T. flavescens, T.floccosum, T. gloriae, T. gourvilii, T. granulare, T. granulosum, T. gypseum, T. inguinale, T. interdigitale, T. intertriginis, T. kanei, T. krajdenii, T. longfusum, T. megninii, T. pedis, T. proliferans, T. quickaneum, T. radiolatum, T. mentrophytes var. erinacei, T. mentagrophytes var. interdigitale, T. mentagrophytes var. mentagrophytes, T. mentagrophytes var. nodulare, T. mentagrophytes var. quinnckeanum, T. niveum, T. nodulare, T. persicolor, T. phaseolforme, T. proliferans, T. purpureum, T. quinckeanum, T. radiolatum, T. raubitschekii, T. schoenleinii, T. simii, T. soudanense, T. sulphureum, T. tonsurans, T. sulphureum, T. terrestre, T. tonsurans var. sulphur eum, T. tonsurans var tonsurans subvar. perforans, T. vanbreuseghemii, T. verrucosum, T. violaceum, T. yaoundei, T. kanei, T. raubitschekii, T. Soudanese. The properties of both LAPs were compared to those of the secreted enzymes encoded by the orthologue genes of the opportunistic fungus Aspergillus fumigatus, fuLAPl and fuLAP2, and the commercially available microsomal LAP from porcine kidney (pkLAP) (MEROPS&gt;M1 family). AH of these enzymes exhibit a leucine aminopeptidase activity. Also, the A. fumigatus aminopepeptidases fuLAPl and fuLAP2 display about 70% amino acid identity with the A. oryzae orthologues reported in United States Patent NOs. 6,127,161 and 5,994,1 13, which are incorporated herein by reference. Furthermore, ruLAP2 appears to be unique because (i) ruLAPl and ruLAP2 display about 50% amino acid identity with the A. fumigatus<br>
orthologues fuLAPl and fuLAP2 and with the A. oryzae orthologues reported in United State patents # 6,127,161 and 5,994,113; (ii) a cocktail of ruLAP2 and a trypsin-like endoprotease originating from the P. pastoris expression system sequentially digests a full length polypeptide chain such as denatured casein; (iii) a cocktail of ruLAP2 and ruDPPIV (another exoprotease of T. rubrum) degrades a fragment of gliadin known to be resistant to protease action, thereby providing evidence that ruLAP2 alone or in combination with ruDPPIV could be used for the treatment of celiac disease or any disease of the digestive tract such as malabsorption; (iv) ruLAP2 in combination with other proteases (cocktails) is useful in the food industry, such as degrading substrates for bitterness, theves degradation, treatment of meat, soap industry, degrading prions, degrading viruses, and degrading toxic or contaminant proteins; (v) and, since ruLAP2 and/or other proteases secreted by the the fungi is necessary for dermatophytes to grow on the cornified substrate of the nail, inhibitors of ruLAP2 and/or other proteases secreted by the fungi would be a new method of treatment for mycoses.<br>
This invention provides novel fungal nucleic acids and proteins, which have leucine aminopeptidase activity. LAPs play a role in diverse functions including, but not limited to blood clotting, controlled cell death, tissue differentiation, tumor invasion, and in the infection cycle of a number of pathogenic microorganisms and viruses making these enzymes a valuable target and a powerful tool for new Pharmaceuticals. Besides having a function in physiology, aminopepetidases also have commercial applications, mainly in the detergent and food industries. Microorganisms, such as fungi, are an excellent source of these enzymes due to their broad biochemical diversity and their susceptibility to genetic manipulation. Microorganisms degrade proteins and utilize the degradation products as nutrients for their growth.Thus, the novel LAPs identified herein are useful in a multitude of industrial applications including but not limited to hydrolysis of proteins in the food industry, degradation of by-products (e.g., feathers); degradation of prions; degradation of proteins for proteomics; hydrolysis of polypeptides for amino acid analysis; wound cleaning (e.g., attacking the dead tissue); prothesis cleaning and/or preparation; fabric softeners; soaps; cleaning or disinfection of sceptic tanks or any container (such as vats of retention, bottles, etc.) containing proteins that should be removed or sterilized; and cleaning of surgical instruments.<br>
This invention provides novel enzymes and enzyme cocktails, i.e. a mixture of more than one enzyme that digest insoluble protein structures, such as the cornified cell envelope into short peptides and free amino acids. In fact, in addition to endoproteases of the S8 and<br>
M36 family, T. rubrum secretes two LAPs each with different substrate activity. RuLAPl and ruLAP2 each belong to the same family of LAPs (MEROPS&gt;M28). The properties of both LAPs were compared to those of the secreted enzymes encoded by the orthologue genes of the opportunistic fungus A. fumigatus, MAPI and fuLAP2, and the commercially available microsomal LAP from porcine kidney (pkLAP) (MEROPS&gt;M1 family). All of these enzymes exhibit leucine aminopeptidase activity. Furthermore, ruLAP2 has an original primary structure and is unique in that it is able, in the presence of ruDPPIV, to sequentially digest a polypeptide chain, such as a fragment of gliadin known to be resistant to other proteases. Partially purified ruLAP2 is also able, in the presence of a trypsin-like endoprotease originating from the P. pastoris expression system, to sequentially digest a full-length polypeptide chain, such as denatured casein.<br>
The invention is based, in part, upon the isolation of novel nucleic acid sequences that encode novel polypeptides. The novel nucleic acids and their encoded polypeptides are referred to individually as ruLAPl, ruLAP2, fuLAPl and fuLAP2. The nucleic acids, and their encoded polypeptides, are collectively designated herein as "EXOX".<br>
The novel EXOX nucleic acids of the invention include the nucleic acids whose sequences are provided in Tables 1 A, IB, 2A, 2B, 3A, 3B, 4A, 4B, 5A, 5B, 6A, 6B, 7A, 7B, 8A, 8B, 9A, 9B, 10A, 10B, 11A, 1 IB, and 12A, or a fragment, derivative, analog or homolog thereof. The novel EXOX proteins of the invention include the protein fragments whose sequences are provided in Tables 1C, 2C, 3C, 4C, 5C, 6C, 7C, 8C, 9C, IOC, 11C, and 12B. The individual EXOX nucleic acids and proteins are described below.<br>
Also, within the scope of this invention is a method of using protease inhibitors in the treatment or prevention of a fungal infection and/or opportunistic infection due to fungi, yeast cells and/or bacteria.<br>
Using a reverse genetic approach, two aminopeptidases secreted by T. rubrum have been characterized in comparison with orthologues from A. fumigatus and the microsomal aminopeptidase pkLAP from porcine kidney. The four fungal enzymes identified herein (ruLAPl, fuLAPl, ruLAP2 and fuLAP2) as well as pkLAP share a common preference for Leu-AMC as a substrate, and function as leucine aminopeptidases. In addition, the aminopeptidase pkLAP, which acts also with an extremely high efficiency towards Ala-AMC, is also called alanine aminopeptidase (MEROPS&gt;M 1.001).<br>
The EXOX nucleic acids of the invention, encoding EXOX proteins, include the nucleic acids whose sequences are provided herein or fragments thereof. The invention also<br>
includes mutant or variant nucleic acids any of whose bases may be changed from the corresponding base shown herein, while still encoding a protein that maintains its EXOX-like activities and physiological functions, or a fragment of such a nucleic acid. The invention further includes nucleic acids whose sequences are complementary to those described herein, including nucleic acid fragments that are complementary to any of the nucleic acids just described. The invention additionally includes nucleic acids or nucleic acid fragments, or complements thereto, whose structures include chemical modifications. Such modifications include, by way of nonlimiting example, modified bases and nucleic acids whose sugar phosphate backbones are modified or derivatized. These modifications are carried out at least in part to enhance the chemical stability of the modified nucleic acid, such that they may be used, for example, as antisense binding nucleic acids in therapeutic applications in a subject<br>
The EXOX proteins of the invention include the EXO proteins whose sequences are provided herein. The invention also includes mutant or variant proteins any of whose residues may be changed from the corresponding residue shown herein, while still encoding a protein that maintains its EXO-like activities and physiological functions, or a functional fragment thereof. The invention further encompasses antibodies and antibody fragments, such as Fab or (Fab)2, that bind immunospecifically to any of the proteins of the invention.<br>
EXOX nucleic acids and proteins are useful in potential therapeutic applications such as the treatment of fungal infections. The EXOX nucleic acids, proteins and inhibitors also have other functions that include but are not limited to: (i) biotechnology reagent for improved protein production, e.g., tag removal, production of rare amino acids; (ii) drug development for certain disease indications, e.g., celiac disease (gluten intolerance); (iii) drug development for dermatological conditions, e.g., anti-mycosis agents, wart treatment, wound healing; (iv) cosmetology, e.g., with peeling tools, depilation, dermabrasion and dermaplaning; (v) food industry, e.g., production of nutrition supplements, sweetners, generating hypoallergenic foods by predigestion; (vi) disinfecting agent, e.g., decontaminating protein-based contaminants such as prions or viruses (by digesting coat protein), cleaning surgery instruments or preparing items for surgery such as prosthesis or medical devices; (vii) sanitizing or recycling certain wastes, e.g., feathers, bones, hair and fur; (viii) cleaning agent, e.g., shampoo or liquid detergent.<br>
Inhibitors of the EXOs, specifically of ruLAP2, may also be used as fungal anti-mycotic agents to treat mycoses. The LAPs themselves may also be used to treat diseases of the digestive tract, such as malabsorption or celiac disease, which is caused by wheat gluten.<br>
Gluten is the characteristic term for the protein mixture of glutelins and gliadins (prolamines) found in cereals. Due to its inherent physicochemical properties such as acting as a binding and extending agent, gluten is commonly used as an additive in food. Detection of gluten is important in the quality control and selection of food for individuals with diseases related to or caused by gluten intolerance including, gluten intolerance enteropathy, celiac disease, sprue and related allergic reactions, where a diet free from the gluten contained in wheat, rye barley, and in some cases oat is necessary.<br>
Exoprotease Nucleic Acids and Polypeptides<br>
T. rubrum aminopeptidase activity demonstrated here and previous studies on subtilisins and metalloproteases secreted by M. canis show that dermatophytes secrete a battery of proteases similar to those of the Aspergillus species in a medium containing protein as sole carbon and nitrogen source. Moreover, two genes, ruDPPIV and ruDPPV: EMBL AF082514 for ruDPPV, coding for dipeptidyl-aminopeptidases highly similar to DPPIV and DPPV from bothA.foimigatus and A. oryzae (Beauvais et al., J. Biol. Chem. 272:6238-6244 (1997); Beauvais et al, Infec. Immun. 65:3042-3047 (1997); Doumas et al., Appl. Environ. Microbiol. 64:4809-4815 (1998); Doumas et al., J. Food Mycol. 2:271-279 (1999)) were isolated from genomic and cDNA libraries of T. rubrum. The intron-exon structures of the T. rubrum genes encoding these proteases are similar to the homologous genes isolated from A. fumigatus and A. oryzae. These results are not surprising since the teleomorphs ofAspergillus species and the teleomorphs of dermatophyte species are closely related, as they belong to the same taxonomic group of Ascomycetes producing prototunicate asci in cleistothecia (class Eurotiomycetes). In contrast to the genes encoding subtilisins and fungalysins, ruLAPl and ruLAP2 are not members of large gene families in the T. rubrum genome.<br>
RuLAPl displays about 50% amino acid identity with fuLAPl and/or LAP1 (See Tables 19A and 20. These three enzymes structurally belong to the same subfamily M28E as Aeromonas and Vibrio leucyl aminopeptidases (MEROPS&gt;M28.002). In addition, ruLAP2 displays about 50% amino acid identity with fuLAP2 and/or LAP2 (See Tables 19B and 21). These three enzymes structurally belong to the same subfamily M28A as the vacuolar protease Y of S. cerevisiae (MEROPS&gt;M28.001) and the Streptomyces griseus secreted aminopeptidase (MEROPS&gt;M28.00X). In addition, the members of the M28A and M28E subfamilies share low similarities. However, the amino acids of the two Zn1"1" binding sites in these aminopeptidases are conserved and were identified in the fungal LAPs characterized<br>
herein (See Tables 20 and 21). In S. griseus and Aeromonas proteolytica secreted aminopeptidases, the two amino acid residues His and Asp bind a first Zn** ion and two additional residues His and Glu bind a second Zn** ion, while a second Asp residue bridges the two Zn"1"1" ions. Greenblatt et al, J. Mol. Biol. 265:620-636 (1997); Hasselgren et al.J. Biol. Inorg. Chem. 6:120-127 (2001). Substitution of Zn** by different divalent ions in 5. griseus secreted aminopeptidase is affected by Ca** and has variable effects. Ben-Meir et al., Eur. J. Biochem 212:107-112 (1993); Lin et al., J. Biol. Inorg. Chem. 2:744-749 (1997); Hasselgren et al.J. Biol. Inorg. Chem. 6:120-127 (2001). The aminopeptidases of this invention were found to be sensitive to different ions. Like the S. griseus aminopeptidase, ruLAP2 and fuLAP2 are highly activated by Co1"1".<br>
RuLAP2 and fuLAP2 possess substantially different proteolytic activities despite a high percentage of sequence identity. In particular, ruLAP2 is able to efficiently hydrolyze Asp- and Glu-7-amine-4-methylcoumarin (AMC), and ruLAP2 is the sole LAP identified so far that is able, first in the presence of ruDPPFV, to digest a peptide of gliadin known to be resistant to digestion by gastric and pancreatic proteases, or second, in the form of a partially purified extract that contains a trypsin-like endoprotease originating from the P. pastoris expression system, to digest a full length polypeptide chain such as denatured casein. The ability of a LAP to degrade a long polypeptide is not predictable solely on the basis of its capacity to cleave aminoacyl-AMC residues. Particular properties of dermatophyte enzymes have been observed with endoproteases secreted by M. canis. The 31.5 kDa M canis subtilisin and the 43.5 kDa M canis metalloprotease are both able to digest keratine azure in contrast to homologous secreted proteases from A. Jumigatus and A. oryzae. As dermatophytes evolved from their natural habitat in soil, they have developed a strategy of infection using particular proteases to degrade the keratinized tissues. The unique properties of ruLAP2 could reflect highly specialized organisms parasiting the stratum corneum and the nails.<br>
In addition to the LAPs disclosed herein, a series of novel proteases have also been isolated from the pathogenic fungi T. rubrum and are disclosed below. Like the LAPs these proteases are all characterised as exoproteases. They include: two carboxypeptidases, a prolylaminopeptidase, an amino peptidase P, a prolidase, and a dipeptidylpeptidase IV. Two additional novel proteases have been also characterized: a leucine aminopeptidase (caLAPl) from Microsporum canis and meLAPl, a Trichophyton mentagrophytes leucine aminopeptidase.ruLAP2<br>
ruLAP2 is a T. rubrum leucine aminopeptidase. A ruLAP2 nucleic acid of 1757 nucleotides (SEQ ID NO:1) is shown in Table 1A(Table Removed)<br>
A disclosed ruLAP2 open reading frame ("ORF") of 1488 nucleotides begins with an ATG start codon at position 1 (underlined in Table IB) (Table Removed)<br>
A disclosed ruLAP2 nucleic acid (SEQ ID NO: 2) encodes a protein having 495 ammo acid residues (SEQ ID NO: 3), which is presented in Table 1C using the one-letter amino acid code(Table Removed)<br>
The disclosed ruLAP2 has homology to the amino acid sequences shown in the BLAST data listed in Table ID, IE, and IF.<br>
The following program options were used:<br>
tblastn - compares the protein "Sequence 1" against the nucleotide "Sequence 2" which has been translated in all six reading frames<br>
blastx - compares the nucleotide "Sequence 1" against the protein "Sequence 2" blastp - for protein - protein comparisons<br>
In all BLAST alignments herein, the "E-value" or "Expect" value is a numeric indication of the probability that the aligned sequences could have achieved their similarity to the BLAST query sequence by chance alone, within the database that was searched. The Expect value (E) is a parameter that describes the number of hits one can "expect" to see just by chance when searching a database of a particular size. It decreases exponentially with the Score (S) that is assigned to a match between two sequences. Essentially, the E value describes the random background noise that exists for matches between sequences(Table Removed)<br>
A disclosed ruLAPl open reading frame ("ORF") of 1122 nucleotides begins with an ATG codon (underlined in Table 2B) at position 1(Table Removed)<br>
A disclosed ruLAPl nucleic acid (SEQ ED NO: 5) encodes a protein having 377 amino acid residues (SEQ ID NO: 6), which is presented in Table 2C using the one-letter amino acid code.<br>
(Table Removed)The disclosed ruLAPl has homology to the amino acid sequences shown in the blast data listed in Table 2D, 2E, and 2F. This data was analyzed by the program pairwise blast. (Table Removed)fuLAP2<br>
fuLAP2 is an A. fumigatus leucine aminopeptidase. A fuLAP2 nucleic acid of 1557 nucleotides is shown in Table 3A (SEQ ID NO: 7) (Table Removed)<br>
A disclosed fuLAP2 open reading frame ("ORF") of 1497 nucleotides begins with an ATG codon (underlined in Table 3B) at position 1(Table Removed)<br>
A disclosed fuLAP2 nucleic acid (SEQ ID NO: 8) encodes a protein having 498 amino acid residues (SEQ ID NO: 9), which is presented in Table 3C using the one-letter amino acid code(Table Removed) 			<br>
The disclosed fuLAP2 has homology to the amino acid sequences shown in the BLAST data listed in Table 3D, 3E, and 3F. This data was analyzed by the program PAIRWISE BLAST(Table Removed)APl<br>
fuLAPl is an A.fumigatus leucine aminopeptidase. A fuLAPl nucleic acid of 1298 nucleotides is shown in Table 4A (SEQ ID NO: 10) (Table Removed)<br>
A disclosed fuLAPl open reading frame ("ORF") of 1167 nucleotides begins with an ATG codon at position 1 (underlined in Table 4B).<br>
(Table Removed)A disclosed fuLAPl nucleic acid (SEQ ID NO: 11) encodes a protein having 388 amino acid residues (SEQ ID NO: 12), which is presented in Table 4C using the one-letter amino acid code. (Table Removed)<br>
The disclosed fuLAPl has homology to the amino acid sequences shown in the BLAST data listed in Table 4D, 4E, and 4F. This data was analyzed by the program PAIRWISE BLAST(Table Removed)<br>
ruCBPSl<br>
ruCBPSl is a T. rubrum carboxypeptidase. Genomic DNA sequence of a ruCBPSl nucleic acid of 2106 nucleotides (SEQ ID NO: 13) is shown in Table 5 A.<br>
(Table Removed)<br>
A ruCBPSl nucleic acid of 1989 (SEQ ID NO: 14) is shown in Table 5B. A disclosed ruCBPSl open reading frame ("ORF") begins with an ATG start codon at position 1 (underlined in Table 5B).<br>
(Table Removed)A disclosed ruCBPSl nucleic acid (SEQ ID NO: 14) encodes a protein having 662 amino acid residues (SEQ ID NO: IS), which is presented in Table SC using the one-letter amino acid code.<br>
(Table Removed)The disclosed ruCBPS 1 has homology to the ammo acid sequences shown in the BLAST data listed in Table 5D, 5E and 5F. This data was analyzed by the program PAIRWISE BLAST(Table Removed)riiCBPSl*<br>
ruCBPSr is a T. rubrum carboxypeptidase. Genomic DNA sequence of a ruCBPSl' nucleic acid of 2030 nucleotides (SEQ ID NO: 16) is shown in Table 6A. Table 6A. ruCBPSP genomic nucleotide sequence (SEQ ID NO: 16). (Table Removed)<br>
A disclosed ruCBPSl' nucleic acid (SEQ ID NO: 17) encodes a protein having 652 amino acid residues (SEQ ID NO: 18), which is presented in Table 6C using the one-letter amino acid code.<br>
Table 6C. Encoded ruCBPSl' protein sequence (SEQ ID NO: 18) (Table Removed)<br>
The disclosed ruCBPSl' has homology to the amino acid sequences shown in the BLAST data listed in Table 6D, 6E and 6F. This data was analyzed by the program PAIRWISE BLAS(Table Removed)<br>
One aspect of the invention pertains to isolated nucleic acid molecules that encode EXOX polypeptides or biologically active portions thereof. Also included in the invention are nucleic acid fragments sufficient for use as hybridization probes to identify EXOX-encoding nucleic acids (e.g., EXOX mRNAs) and fragments for use as PCR primers for the amplification and/or mutation of EXOX nucleic acid molecules. As used herein, the term "nucleic acid molecule" is intended to include DNA molecules (e.g., cDNA or genomic DNA), RNA molecules (e.g., mRNA), analogs of the DNA or RNA generated using<br>
nucleotide analogs, and derivatives, fragments and homologs thereof. The nucleic acid molecule may be single-stranded or double-stranded.<br>
An EXOX nucleic acid can encode a mature EXOX polypeptide. As used herein, a "mature" form of a polypeptide or protein disclosed in the present invention is the product of a naturally occurring polypeptide or precursor form or proprotein. The naturally occurring polypeptide, precursor or proprotein includes, by way of nonlimiting example, the full-length gene product, encoded by the corresponding gene. Alternatively, it may be defined as the polypeptide, precursor or proprotein encoded by an ORF described herein. The product "mature" form arises, again by way of nonlimiting example, as a result of one or more naturally occurring processing steps as they may take place within the cell, or host cell, in which the gene product arises. Examples of such processing steps leading to a "mature" form of a polypeptide or protein include the cleavage of the N-terminal methionine residue encoded by the initiation codon of an ORF, or the proteolytic cleavage of a signal peptide or leader sequence. Thus a mature form arising from a precursor polypeptide or protein that has residues 1 to N, where residue 1 is the N-terminal methionine, would have residues 2 through N remaining after removal of the N-terminal methionine. Alternatively, a mature form arising from a precursor polypeptide or protein having residues 1 to N, in which an N-terminal signal sequence from residue 1 to residue M is cleaved, would have the residues from residue M+l to residue N remaining. Further as used herein, a "mature" form of a polypeptide or protein may arise from a step of post-translational modification other than a proteolytic cleavage event. Such additional processes include, by way of non-limiting example, glycosylation (N-, O- and W types), myristoylation, phosphorylation, sulfation, N-terminus cyclisation, or C-terminus amidation. In general, a mature polypeptide or protein may result from the operation of only one of these processes, or a combination of any of them.<br>
The term "probes", as utilized herein, refers to nucleic acid sequences of variable length, preferably between at least about 10 nucleotides (nt), 100 nt, or as many as approximately, e.g., 6,000 nt, depending upon the specific use. Probes are used in the detection of identical, similar, or complementary nucleic acid sequences. Longer length probes are generally obtained from a natural or recombinant source, are highly specific, and much slower to hybridize than shorter-length oligomer probes. Probes may be single- or double-stranded and designed to have specificity in PCR, membrane-based hybridization technologies, or ELISA-like technologies.<br>
The term "isolated" nucleic acid molecule, as utilized herein, is one, which is separated from other nucleic acid molecules, which are present in the natural source of the nucleic acid. Preferably, an "isolated" nucleic acid is free of sequences, which naturally flank the nucleic acid (e.g., sequences located at the 5'- and 3'-termini of the nucleic acid) in the genomic DNA of the organism from which the nucleic acid is derived. For example, in various embodiments, the isolated EXOX nucleic acid molecules can contain less than about 5 kb, 4 kb, 3 kb, 2 kb, 1 kb, 0.5 kb or 0.1 kb of nucleotide sequences which naturally flank the nucleic acid molecule in genomic DNA of the cell/tissue/species from which the nucleic acid is derived. Moreover, an "isolated" nucleic acid molecule, such as a cDNA molecule, can be substantially free of other cellular material or culture medium when produced by recombinant techniques, or of chemical precursors or other chemicals when chemically synthesized. Particularly, it means that the nucleic acid or protein is at least about 50% pure, more preferably at least about 85% pure, and most preferably at least about 99% pure.<br>
As used herein, the term "recombinant" when used with reference to a cell indicates that the cell replicates a heterologous nucleic acid, or expresses a peptide or protein encoded by a heterologous nucleic acid. Recombinant cells can contain genes that are not found within the native (non-recombinant) form of the cell. Recombinant cells can also contain genes found in the native form of the cell wherein the genes are modified and re-introduced into the cell by artificial means. The term also encompasses cells that contain a nucleic acid endogenous to the cell that has been modified without removing the nucleic acid from the cell; such modifications include those obtained by gene replacement, site-specific mutation, and related techniques. One skilled in the art will recognize that these cells can be used for unicellular or multicellular transgenic organisms, for example transgenic fungi producing EXOX.<br>
A nucleic acid molecule of the invention, e.g., a nucleic acid molecule having the nucleotide sequence of SEQ ID NOs: 2,5, 8, 11,14,17, 20,23,26, 29, 32, or 34 or a complement of this aforementioned nucleotide sequence, can be isolated using standard molecular biology techniques and the sequence information provided herein. Using all or a portion of the nucleic acid sequence of SEQ ID NOs: 2,5,8,11,14,17,20,23,26,29,32, or 34 as a hybridization probe, EXOX molecules can be isolated using standard hybridization and cloning techniques (e.g., as described in Sambrook et al., (eds.), MOLECULAR CLONING: A LABORATORY MANUAL 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor,<br>
NY, 1989; and Ausubel et al., (eds.), CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, New York, NY, 1993.)<br>
A nucleic acid of the invention can be amplified using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. The nucleic acid so amplified can be cloned into an appropriate vector and characterized by DNA sequence analysis. Furthermore, oligonucleotides corresponding to EXOX nucleotide sequences can be prepared by standard synthetic techniques, e.g., using an automated DNA synthesizer.<br>
As used herein, the term "oligonucleotide" refers to a series of linked nucleotide residues, which oligonucleotide has a sufficient number of nucleotide bases to be used in a PCR reaction. A short oligonucleotide sequence may be based on, or designed from, a genomic or cDNA sequence and is used to amplify, confirm, or reveal the presence of an identical, similar or complementary DNA or RNA in a particular cell or tissue. Oligonucleotides comprise portions of a nucleic acid sequence having about 10 nt, 50 nt, or 100 nt in length, preferably about 15 nt to 30 nt in length. In one embodiment of the invention, an oligonucleotide comprising a nucleic acid molecule less than 100 nt in length would further comprise at least 6 contiguous nucleotides of SEQ ID NOs: 2, 5, 8, 11, 14, 17, 20, 23, 26, 29, 32, or 34, or a complement thereof. Oligonucleotides may be chemically synthesized and may also be used as probes.<br>
In another embodiment, an isolated nucleic acid molecule of the invention comprises a nucleic acid molecule that is a complement of the nucleotide sequence shown in SEQ ID NOs: 2, 5, 8, 1 1 , 14, 1 7, 20, 23, 26, 29, 32, or 34, or a portion of this nucleotide sequence (e.g., a fragment that can be used as a probe or primer or a fragment encoding a biologically-active portion of a EXOX polypeptide). A nucleic acid molecule that is complementary to the nucleotide sequence shown in SEQ ID NOs: 2, 5, 8, 1 1, 14, 17, 20, 23, 26, 29, 32, or 34 is one that is sufficiently complementary to the nucleotide sequence shown in SEQ ID NOs: 2, 5, 8, 1 1, 14, 17, 20, 23, 26, 29, 32, or 34 that it can hydrogen bond with little or no mismatches to the nucleotide sequence shown in SEQ ID NOs: 2, 5, 8, 1 1, 14, 17, 20, 23, 26, 29, 32, or 34, thereby forming a stable duplex.<br>
As used herein, the term "complementary" refers to Watson-Crick or Hoogsteen base pairing between nucleotide units of a nucleic acid molecule. The term "binding" means the physical or chemical interaction between two polypeptides or compounds or associated polypeptides or compounds or combinations thereof. Binding includes ionic, non-ionic, van<br>
der Waals, hydrophobic interactions, and the like. A physical interaction can be either direct or indirect. Indirect interactions may be through or due to the effects of another polypeptide or compound. Direct binding refers to interactions that do not take place through, or due to, the effect of another polypeptide or compound, but instead are without other substantial chemical intermediates.<br>
Fragments provided herein are defined as sequences of at least 6 (contiguous) nucleic acids or at least 4 (contiguous) amino acids, a length sufficient to allow for specific hybridization in the case of nucleic acids or for specific recognition of an epitope in the case of amino acids, respectively, and are at most some portion less than a full length sequence. Fragments may be derived from any contiguous portion of a nucleic acid or amino acid sequence of choice. Derivatives are nucleic acid sequences or amino acid sequences formed from the native compounds either directly or by modification or partial substitution. Analogs are nucleic acid sequences or amino acid sequences that have a structure similar to, but not identical to, the native compound but differ from it with respect to certain components or side chains. Analogs may be synthetic or from a different evolutionary origin and may have a similar or opposite metabolic activity compared to wild type. Homologs or orthologs are nucleic acid sequences or amino acid sequences of a particular gene that are derived from different species.<br>
Derivatives and analogs may be full length or other than full length, if the derivative or analog contains a modified nucleic acid or amino acid, as described below. Derivatives or analogs of the nucleic acids or proteins of the invention include, but are not limited to, molecules comprising regions that are substantially homologous to the nucleic acids or proteins of the invention, in various embodiments, by at least about 70%, 80%, or 95% identity (with a preferred identity of 80-95%) over a nucleic acid or amino acid sequence of identical size or when compared to an aligned sequence in which the alignment is done by a computer homology program known in the art, or whose encoding nucleic acid is capable of hybridizing to the complement of a sequence encoding the aforementioned proteins under stringent, moderately stringent, or low stringent conditions. See, e.g., Ausubel et a/., CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, New York, NY, 1993, and below.<br>
A "homologous nucleic acid sequence" or "homologous amino acid sequence," or variations thereof, refer to sequences characterized by a homology at the nucleotide level or amino acid level as discussed above. Homologous nucleotide sequences encode those<br>
sequences coding for isoforms of EXOX polypeptides. Isoforms can be expressed in the same organism as a result of, for example, alternative splicing of RNA. Alternatively, isoforms can be encoded by different genes. In the invention, homologous nucleotide sequences can include nucleotide sequences encoding an EXOX polypeptide of species other than fungi. Homologous nucleotide sequences also include, but are not limited to, naturally occurring allelic variations and mutations of the nucleotide sequences set forth herein. Homologous nucleic acid sequences include those nucleic acid sequences that encode conservative amino acid substitutions (see below) in SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26,29, 32, or 34, as well as a polypeptide possessing EXOX biological activity. Various biological activities of the EXOX proteins are described below.<br>
A EXOX polypeptide is encoded by the open reading frame ("ORF") of an EXOX nucleic acid. A stretch of nucleic acids comprising an ORF is uninterrupted by a stop codon. An ORF that represents the coding sequence for a full protein begins with an ATG "start" codon and terminates with one of the three "stop" codons, namely, TAA, TAG, or TGA. For the purposes of this invention, an ORF may be any part of a coding sequence, with or without a start codon, a stop codon, or both. For an ORF to be considered as a good candidate for coding for a bonafide cellular protein, a minimum size requirement is often set, e.g., a stretch of DNA that would encode a protein of SO ammo acids or more.<br>
The nucleotide sequences determined from the cloning of the fungal EXOX genes allows for the generation of probes and primers designed for use in identifying and/or cloning EXOX homologues in other species, as well as EXOX homologues from other fungi. The probe/primer typically comprises a substantially purified oligonucleotide. The oligonucleotide typically comprises a region of nucleotide sequence that hybridizes under stringent conditions to at least about 12,25, 50,100,150,200,250,300,350 or 400 consecutive sense strand nucleotide sequence of SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26, 29,32, or 34; or an anti-sense strand nucleotide sequence of SEQ ED NOs: 2, 5,8, 11,14,17, 20,23,26,29,32, or 34; or of a naturally occurring mutant of SEQ ID NOs: 2,5,8,11,14, 17,20,23,26,29, 32, or 34.<br>
"A polypeptide having a biologically-active portion of an EXOX polypeptide" refers to polypeptides exhibiting activity similar, but not necessarily identical to, an activity of a polypeptide of the invention, including mature forms, as measured in a particular biological assay, with or without dose dependency. A nucleic acid fragment encoding a "biologically-active portion of EXOX" can be prepared by isolating a portion SEQ ID NOs: 2,5,8,11,14,<br>
17, 20,23,26, 29, 32, or 34 that encodes a polypeptide having a EXOX biological activity (the biological activities of the EXOX proteins are described below), expressing the encoded portion of EXOX protein (e.g., by recombinant expression in vitro) and assessing the activity of the encoded portion of EXOX.<br>
EXOX Nucleic Acid and Polypeptide Variants<br>
The invention further encompasses nucleic acid molecules that differ from the nucleotide sequences shown in SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26,29,32, or 34 due to degeneracy of the genetic code and thus encode the same EXOX proteins that are encoded by the nucleotide sequences shown in SEQ ID NOs: 2, 5, 8, 11,14, 17, 20, 23, 26, 29, 32, or 34. In another embodiment, an isolated nucleic acid molecule of the invention has a nucleotide sequence encoding a protein having an amino acid sequence shown in SEQ ID NOs: 3,6,9,12,15,18,21,24,27,30,33, or 35. In addition to the fungal EXOX nucleotide sequences shown in SEQ ID NOs: 2,5, 8,11,14,17,20,23,26,29,32, and 34, it will be appreciated by those skilled in the art that DNA sequence polymorphisms that lead to changes in the amino acid sequences of the EXOX polypeptides may exist within a population of various species. Such genetic polymorphisms in the EXOX genes may exist among individual fungal species within a population due to natural allelic variation. As used herein, the terms "gene" and "recombinant gene" refer to nucleic acid molecules comprising an open reading frame (ORF) encoding an EXOX protein, preferably a fungal EXOX protein. Such natural allelic variations can typically result in 1-5% variance in the nucleotide sequence of the EXOX genes. Any and all such nucleotide variations and resulting amino acid polymorphisms in the EXOX polypeptides, which are the result of natural allelic variation and that do not alter the functional activity of the EXOX polypeptides, are intended to be within the scope of the invention.<br>
Moreover, nucleic acid molecules encoding EXOX proteins from other species, and, thus, that have a nucleotide sequence that differs from the fungal sequence SEQ ID NOs: 2, 5,8,11,14,17,20,23,26,29,32, or 34 are intended to be within the scope of the invention. Nucleic acid molecules corresponding to natural allelic variants and homologues of the EXOX cDNAs of the invention can be isolated based on their homology to the fungal EXOX nucleic acids disclosed herein using the fungal cDNAs, or a portion thereof, as a hybridization probe according to standard hybridization techniques under stringent hybridization conditions.<br>
Accordingly, in another embodiment, an isolated nucleic acid molecule of the invention is at least 6 nucleotides in length and hybridizes under stringent conditions to the nucleic acid molecule comprising the nucleotide sequence of SEQ ID NOs: 2, 5, 8,11,14, 17, 20,23,26, 29, 32, or 34.<br>
In another embodiment, the nucleic acid is at least 10,25, 50, 100,250, 500, 750, 1000,1500, or 2000 or more nucleotides in length. In yet another embodiment, an isolated nucleic acid molecule of the invention hybridizes to the coding region. As used herein, the term "hybridizes under stringent conditions" is intended to describe conditions for hybridization and washing under which nucleotide sequences at least 60% homologous to each other typically remain hybridized to each other.<br>
Homologs or other related sequences (e.g., orthologs, paralogs) can be obtained by low, moderate or high stringency hybridization with all or a portion of the particular fungal sequence as a probe using methods well known in the art for nucleic acid hybridization and cloning.<br>
As used herein, the phrase "stringent hybridization conditions" refers to conditions under which a probe, primer or oligonucleotide will hybridize to its target sequence, but to no other sequences. Stringent conditions are sequence-dependent and will be different in different circumstances. Longer sequences hybridize specifically at higher temperatures than shorter sequences. Generally, stringent conditions are selected to be about 5°C lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength and pH. The Tm is the temperature (under defined ionic strength, pH and nucleic acid concentration) at which 50% of the probes complementary to the target sequence hybridize to the target sequence at equilibrium. Since the target sequences are generally present at excess, at Tm, 50% of the probes are occupied at equilibrium. Typically, stringent conditions will be those in which the salt concentration is less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion (or other salts) at pH 7.0 to 8.3 and the temperature is at least about 30°C for short probes, primers or oligonucleotides (e.g., 10 nt to 50 nt) and at least about 60°C for longer probes, primers and oligonucleotides. Stringent conditions may also be achieved with the addition of destabilizing agents, such as formamide.<br>
Stringent conditions are known to those skilled in the art and can be found in Ausubel el al., (eds.), CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6. Preferably, the conditions are such that sequences at least about 65%, 70%, 75%, 85%, 90%, 95%, 98%, or 99% homologous to each other typically remain<br>
hybridized to each other. A non-limiting example of stringent hybridization conditions are hybridization in a high salt buffer comprising 6X SSC, 50 mM Tris-HCl (pH 7.5), 1 mM EDTA, 0.02% PVP, 0.02% Ficoll, 0.02% BSA, and 500 mg/ml denatured salmon sperm DNA at 65°C, followed by one or more washes in 0.2X SSC, 0.01% BSA at 50°C. An isolated nucleic acid molecule of the invention that hybridizes under stringent conditions to the sequences of SEQ ID NOs: 2,5,8,11,14,17,20,23,26,29,32, or 34 corresponds to a naturally-occurring nucleic acid molecule. As used herein, a "naturally-occurring" nucleic acid molecule refers to an RNA or DNA molecule having a nucleotide sequence that occurs in nature (e.g., encodes a natural protein).<br>
In a second embodiment, a nucleic acid sequence that is hybridizable to the nucleic acid molecule comprising the nucleotide sequence of SEQ ID NOs: 2,5, 8,11,14,17,20,23, 26,29,32, or 34 or fragments, analogs or derivatives thereof, under conditions of moderate stringency is provided. A non-limiting example of moderate stringency hybridization conditions are hybridization in 6X SSC, 5X Denhardt's solution, 0.5% SDS and 100 mg/ml denatured salmon sperm DNA at 55°C, followed by one or more washes in IX SSC, 0.1% SDS at 37°C. Other conditions of moderate stringency that may be used are well-known within the art. See, e.g., Ausubel et al. (eds.), 1993, CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, NY, and Kriegler, 1990; GENE TRANSFER AND EXPRESSION, A LABORATORY MANUAL, Stockton Press, NY.<br>
In a third embodiment, a nucleic acid that is hybridizable to the nucleic acid molecule comprising the nucleotide sequences of SEQ ID NOs: 2, 5,8,11,14,17,20,23,26,29,32, or 34 or fragments, analogs or derivatives thereof, under conditions of low stringency, is provided. A non-limiting example of low stringency hybridization conditions are hybridization in 35% formamide, 5X SSC, 50 mM Tris-HCl (pH 7.5), 5 mM EDTA, 0.02% PVP, 0.02% Ficoll, 0.2% BSA, 100 mg/ml denatured salmon sperm DNA, 10% (w/v) dextran sulfate at 40°C, followed by one or more washes in 2X SSC, 25 mM Tris-HCl (pH 7.4), 5 mM EDTA, and 0.1% SDS at 50°C. Other conditions of low stringency that may be used are well known in the art (e.g., as employed for cross-species hybridizations). See, e.g., Ausubel et al. (eds.), 1993, CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, NY, and Kriegler, 1990, GENE TRANSFER AND EXPRESSION, A LABORATORY MANUAL, Stockton ress, NY; Shilo &amp; Weinberg, Proc Nail Acad Sci USA 78:6789-6792 (1981).<br>
Conservative Mutations<br>
In addition to naturally-occurring allelic variants of EXOX sequences that may exist in the population, the skilled artisan will further appreciate that changes can be introduced by mutation into the nucleotide sequences of SEQ ED NOs: 2,5, 8,11,14,17,20,23,26,29,32, or 34 thereby leading to changes in the amino acid sequences of the encoded EXOX proteins, without altering the functional ability of said EXOX proteins. For example, nucleotide substitutions leading to amino acid substitutions at "non-essential" amino acid residues can be made in the sequence of SEQ ID NOs: 3,6, 9,12,15,18,21,24,27,30, 33, or 35. A "non-essential" amino acid residue is a residue that can be altered from the wild-type sequences of the EXOX proteins without altering their biological activity, whereas an "essential" amino acid residue is required for such biological activity.<br>
As used herein, the term "biological activity" or "functional activity" refers to the natural or normal function of the EXO proteins, for example the ability to degrade other proteins. Amino acid residues that are conserved among the EXOX proteins of the invention are predicted to be particularly non-amenable to alteration. Amino acids for which conservative substitutions can be made are well known within the art. One of skill in the art will recognize that each codon in a nucleic acid (except AUG, which is ordinarily the only codon for methionine) can be modified to yield a functionally identical molecule by standard techniques. Furthermore, individual substitutions, deletions or additions which alter, add or delete a single amino acid or a small percentage of amino acids (typically less than 5%, more typically less than 1%) in an encoded sequence are "conservative mutations" where the alterations result in the substitution of an amino acid with a chemically similar amino acid.<br>
Another aspect of the invention pertains to nucleic acid molecules encoding EXOX proteins that contain changes in amino acid residues that are not essential for activity. Such EXOX proteins differ in amino acid sequence from SEQ ID NOs: 3,6, 9,12,15,18,21,24, 27,30,33, or 35 yet retain biological activity. In one embodiment, the isolated nucleic acid molecule comprises a nucleotide sequence encoding a protein, wherein the protein comprises an amino acid sequence at least about 45% homologous to the amino acid sequences of SEQ ID NOs: 3,6,9,12,15,18,21,24,27,30,33, or 35. Preferably, the protein encoded by the nucleic acid molecule is at least about 60% homologous to SEQ ID NOs: SEQ ID NOS: 3,6, 9,12,15,18,21,24,27,30,33, or 35; more preferably at least about 70% homologous to SEQ ID NOs: 3,6,9,12,15,18,21,24, 27, 30, 33, or 35; still more preferably at least about<br>
80% homologous to SEQ ID NOS: SEQ ID NOs: 3,6,9,12,15,18,21,24, 27, 30,33, or 35;<br>
even more preferably at least about 90% homologous to SEQ DD NOs: 3, 6,9,12,15,18,21, 24,27,30, 33, or 35; and most preferably at least about 95% homologous to SEQ ID NOs: 3, 6, 9,12,15,18, 21,24,27,30,33, or 35.<br>
An isolated nucleic acid molecule encoding an EXOX protein homologous to the protein of SEQ ID NOs: 3,6, 9,12,15,18,21,24,27,30,33, or 35 can be created by introducing one or more nucleotide substitutions, additions or deletions into the nucleotide sequence of SEQ ID NOs: 2, 5, 8,11,14,17,20,23,26,29,32, or 34 such that one or more amino acid substitutions, additions or deletions are introduced into the encoded protein.<br>
Mutations can be introduced into SEQ ID NOs: 3,6,9,12,15,18,21,24,27,30,33, or 35 by standard techniques, such as site-directed mutagenesis, PCR-mediated mutagenesis and DNA shuffling. Preferably, conservative amino acid substitutions are made at one or more predicted, non-essential amino acid residues. Single base substitutions are among the most common changes to human DNA. These base changes can occur in the coding or the non-coding regions of the DNA. If they occur in the coding region, they can be conservative or non-conservative substitutions. A "conservative amino acid substitution" is a new ammo acid that has similar properties and is one in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Non-conservative substitutions refer to a new amino acid, which has different properties. Families of amino acid residues having similar side chains have been defined within the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (eg., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), nonpolar side chains (e.g., alanine, valine, leucine, isoleucine, proline, hydroxyproline, phenylalanine, methionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). Thus, for a conservative substitution, a predicted non-essential amino acid residue in the EXOX protein is replaced with another amino acid residue from the same side chain family. Alternatively, in another embodiment, mutations can be introduced randomly along all or part of an EXOX coding sequence, such as by saturation mutagenesis, and the resultant mutants can be screened for EXOX biological activity to identify mutants that retain activity. Following mutagenesis of SEQ ID NOs: 2,5, 8,11,14,17,20,23,26,29, 32, or 34, the encoded protein can be expressed by any recombinant technology known in the art and the activity of the protein can be determined.<br>
The relatedness of amino acid families may also be determined based on side chain interactions. Substituted amino acids may be fully conserved "strong" residues or fully conserved "weak" residues. The "strong" group of conserved amino acid residues may be any one of the following groups: STA, NEQK, NHQK, NDEQ, QHRK, MILV, MILF, HY, FYW, wherein the single letter amino acid codes are grouped by those amino acids that may be substituted for each other. Likewise, the "weak" group of conserved residues may be any one of the following: CSA, ATV, SAG, STNK, STPA, SGND, SNDEQK, NDEQHK, NEQHRK, HFY, wherein the letters within each group represent the single letter amino acid code.<br>
In one embodiment, a mutant EXOX protein can be assayed for (/) the ability to form protein:protein interactions with other EXOX proteins, other cell-surface proteins, or biologically-active portions thereof, (if) complex formation between a mutant EXOX protein and a EXOX ligand; or (Hi) the ability of a mutant EXOX protein to bind to an intracellular target protein or biologically-active portion thereof; (e.g. avidin proteins).<br>
In yet another embodiment, a mutant EXOX protein can be assayed for the ability to regulate a specific biological function (e.g., proteolytic activity).<br>
EXOX Polypeptides<br>
A polypeptide according to the invention includes a polypeptide including the amino acid sequence of EXOX polypeptides whose sequences are provided in SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, and 35. The invention also includes a mutant or variant protein any of whose residues may be changed from the corresponding residues shown in SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35 while still encoding a protein that maintains its EXOX activities and physiological functions, or a functional fragment thereof.<br>
In general, an EXOX variant that preserves EXOX-like function includes any variant in which residues at a particular position in the sequence have been substituted by other amino acids, and further include the possibility of inserting an additional residue or residues between two residues of the parent protein as well as the possibility of deleting one or more residues from the parent sequence. Any amino acid substitution, insertion, or deletion is encompassed by the invention. In favorable circumstances, the substitution is a conservative substitution as defined above.<br>
One aspect of the invention pertains to isolated EXOX proteins, and biologically active portions thereof, or derivatives, fragments, analogs or homologs thereof. Biologically<br>
active portions refer to regions of the EXOX proteins, which are necessary for normal function, for example, aminopeptidase activity. Also provided are polypeptide fragments suitable for use as immunogens to raise anti-EXOX antibodies. In one embodiment, native EXOX proteins can be isolated from cells, tissue sources or culture supematants by an appropriate purification scheme using appropriate protein purification techniques. In another embodiment, EXOX proteins are produced by recombinant DNA techniques. Alternative to recombinant expression, an EXOX protein or polypeptide can be synthesized chemically using standard peptide synthesis techniques.<br>
An "isolated" or "purified" polypeptide or protein or biologically-active portion thereof is substantially free of cellular material or other contaminating proteins from the cell or tissue source from which the EXOX protein is derived, or substantially free from chemical precursors or other chemicals when chemically synthesized. The language "substantially free of cellular material" includes preparations of EXOX proteins hi which the protein is separated from cellular components of the cells from which it is isolated or recombinantly-produced. Li one embodiment, the language "substantially free of cellular material" includes preparations of EXOX proteins having less than about 30% (by dry weight) of non-EXOX proteins (also referred to herein as a "contaminating protein"), more preferably less than about 20% of non-EXOX proteins, still more preferably less than about 10% of non-EXOX proteins, and most preferably less than about 5% of non-EXOX proteins. When the EXOX protein or biologically-active portion thereof is recombinantly-produced, it is also preferably substantially free of any constituent of the culture medium, e.g., culture medium components may represent less than about 20%, more preferably less than about 10%, and most preferably less than about 5% of the EXOX protein preparation.<br>
The language "substantially free of chemical precursors or other chemicals" includes preparations of EXOX proteins in which the protein is separated from chemical precursors or other chemicals that are involved in the synthesis of the protein. In one embodiment, the language "substantially free of chemical precursors or other chemicals" includes preparations of EXOX proteins having less than about 30% (by dry weight) of chemical precursors or non-EXOX chemicals, more preferably less than about 20% chemical precursors or non-EXOX chemicals, still more preferably less than about 10% chemical precursors or non-EXOX chemicals, and most preferably less than about 5% chemical precursors or non-EXOX chemicals. Furthermore, "substantially free of chemical precursors or other<br>
chemicals" would include oxidation byproducts. One of skill in the art would know how to prevent oxidation, for example, by keeping chemicals in an oxygen free environment.<br>
Biologically-active portions of EXOX proteins include peptides comprising amino acid sequences sufficiently homologous to or derived from the amino acid sequences of the EXOX proteins (e.g., the amino acid sequence shown in SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35) that include fewer amino acids than the full-length EXOX proteins, and exhibit at least one activity of an EXOX protein. Typically, biologically active portions comprise a domain or motif with at least one activity of the EXOX protein. A biologically active portion of an EXOX protein can be a polypeptide that is, for example, 10, 25, 50, 100 or more amino acid residues in length.<br>
Moreover, other biologically active portions, in which other regions of the protein are deleted, can be prepared by recombinant techniques and evaluated for one or more of the functional activities of a native EXOX protein.<br>
Li an embodiment, the EXOX protein has an amino acid sequence shown in SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21 , 24, 27, 30, 33, or 35. In other embodiments, the EXOX protein is substantially homologous to SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35, and retains the functional activity of the protein of SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35, yet differs in amino acid sequence due to natural allelic variation or mutagenesis, as described in detail, below. Accordingly, in another embodiment, the EXOX protein is a protein that comprises an amino acid sequence at least about 90% homologous to the amino acid sequence SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35, and retains the functional activity of the EXOX proteins of SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35. As used herein, the term "biological activity" or "functional activity" •efers to the natural or normal function of the EXO proteins, for example the ability to legrade other proteins.<br>
)etermining Homology Between Two or More Sequences<br>
To determine the percent of similarity or homology of two amino acid sequences or of wo nucleic acid sequences, the sequences are aligned for optimal comparison purposes (e.g., ,aps can be introduced in the sequence of a first amino acid or nucleic acid sequence for ptimal alignment with a second amino acid or nucleic acid sequence). The amino acid ssidues or nucleotides at corresponding amino acid positions or nucleotide positions are then ompared. When a position in the first sequence is occupied by the same amino acid residue<br>
or nucleotide as the corresponding position in the second sequence, then the molecules are homologous at that position (i.e., as used herein amino acid or nucleic acid "homology" is equivalent to amino acid or nucleic acid "identity").<br>
The nucleic acid sequence homology may be determined as the degree of identity between two sequences. The homology may be determined using computer programs known in the art, such as GAP software provided in the GCG program package. See Needleman &amp; Wunsch, J. Mol. Biol 48:443-453 1970. Using GCG GAP software with the following settings for nucleic acid sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the analogous nucleic acid sequences referred to above exhibits a degree of identity preferably of at least 70%, 75%, 80%, 85%, 90%, 95%, 98%, or 99%, with the CDS (encoding) part of the DNA sequence shown in SEQ ID NOs: 2, 5, 8, 11, 14, 17, 20, 23, 26, 29, 32, and 34.<br>
The term "sequence identity" refers to the degree to which two polynucleotide or polypeptide sequences are identical on a residue-by-residue basis over a particular region of comparison. The term "percentage of sequence identity" is calculated by comparing two optimally aligned sequences over that region of comparison, determining the number of positions at which the identical nucleic acid base (e.g., A, T, C, G, U, or I, in the case of nucleic acids) occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the region of comparison (e.g., the window size), and multiplying the result by 100 to yield the percentage of sequence identity. The term "substantial identity" as used herein denotes a characteristic of a polynucleotide sequence, wherein the polynucleotide comprises a sequence that has at least 80 percent sequence identity, preferably at least 85 percent identity and often 90 to 95 percent sequence identity, more usually at least 99 percent sequence identity as compared to a reference sequence over a comparison region.<br>
Chimeric and Fusion Proteins<br>
The invention also provides EXOX chimeric or fusion proteins. As used herein, a EXOX "chimeric protein" or "fusion protein" comprises a EXOX polypeptide operatively-linked to a non-EXOX polypeptide. An "EXOX polypeptide" refers to a polypeptide having an amino acid sequence corresponding to an EXOX protein (SEQ ID NOs: 3, 6, 9, 12, 15, 18, 21, 24, 27, 30, 33, or 35), whereas a "non-EXOX polypeptide" refers to a polypeptide having an amino acid sequence corresponding to a protein that is not substantially homologous to the<br>
EXOX protein, e.g., a protein that is different from the EXOX protein and that is derived from the same or a different organism. Within an EXOX fusion protein the EXOX polypeptide can correspond to all or a portion of an EXOX protein. In one embodiment, a EXOX fusion protein comprises at least one biologically active portion of a EXOX protein. In another embodiment, an EXOX fusion protein comprises at least two biologically active portions of an EXOX protein. In yet another embodiment, an EXOX fusion protein comprises at least three biologically active portions of an EXOX protein. Within the fusion protein, the term "operatively-linked" is intended to indicate that the EXOX polypeptide and the non-EXOX polypeptide are fused in-frame with one another. The non-EXOX polypeptide can be fused to the N-terminus and/or C-terminus of the EXOX polypeptide.<br>
In one embodiment, the fusion protein is a GST-EXOX fusion protein in which the EXOX sequences are fused to the C-terminus of the GST (glutathione S-transferase) sequences. Such fusion proteins can facilitate the purification of recombinant EXOX polypeptides.<br>
In another embodiment, the fusion protein is an EXOX protein containing a heterologous signal sequence at its N-terminus. In certain host cells (e.g., mammalian host cells), expression and/or secretion of EXOX can be increased through use of a heterologous signal sequence.<br>
In yet another embodiment, the fusion protein is an EXOX-immunoglobulin fusion protein in which the EXOX sequences are fused to sequences derived from a member of the immunoglobulin protein family. The EXOX-immunoglobulin fusion proteins of the invention can be incorporated into pharmaceutical compositions and administered to a subject to inhibit an interaction between a EXOX ligand and a EXOX protein on the surface of a cell, to thereby suppress EXOX-mediated signal transduction in vivo. The EXOX-immunoglobulin fusion proteins can be used to affect the bioavailability of an EXOX cognate ligand. Inhibition of the EXOX ligand/EXOX interaction may be useful therapeutically for both the treatment of proliferative and differentiative disorders, as well as modulating (e.g. promoting or inhibiting) cell survival. Moreover, he EXOX-immunoglobulin fusion proteins of the invention can be used as immunogens to produce anti-EXOX antibodies in a subject, to purify EXOX ligands, and in screening assays to identify molecules that inhibit the interaction of EXOX with an EXOX ligand.<br>
A EXOX chimeric or fusion protein of the invention can be produced by standard recombinant DNA techniques. For example, DNA fragments coding for the different<br>
polypeptide sequences are ligated together in-frame in accordance with conventional techniques, e.g., by employing blunt-ended or stagger-ended termini for ligation, restriction enzyme digestion to provide for appropriate termini, filling-in of cohesive ends as appropriate, alkaline phosphatase treatment to avoid undesirable joining, and enzymatic ligation. In another embodiment, the fusion gene can be synthesized by conventional techniques including automated DNA synthesizers. Alternatively, PCR amplification of gene fragments can be carried out using anchor primers that give rise to complementary overhangs between two consecutive gene fragments that can subsequently be annealed and reamplified to generate a chimeric gene sequence (See, e.g., Ausubel et al. (eds.) CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, 1992). Moreover, many expression vectors are commercially available that already encode a fusion moiety (e.g., a GST polypeptide). A EXOX-encoding nucleic acid can be cloned into such an expression vector such that the fusion moiety is linked in-frame to the EXOX protein.<br>
EXOX Agonists and Antagonists<br>
The invention also pertains to variants of the EXOX proteins that function as either EXOX agonists (e.g., mimetics) or as EXOX antagonists. Variants of the EXOX protein can be generated by mutagenesis (e.g., discrete point mutation or truncation of the EXOX protein). An agonist of the EXOX protein can retain substantially the same, or a subset of, the biological activities of the naturally occurring form of the EXOX protein. An antagonist of the EXOX protein can inhibit one or more of the activities of the naturally occurring form of the EXOX protein by, for example, competitively binding to a downstream or upstream member of a cellular signaling cascade, which includes the EXOX protein. Thus, specific biological effects can be elicited by treatment with a variant of limited function. In one embodiment, treatment of a subject with a variant having a subset of the biological activities of the naturally occurring form of the protein has fewer side effects in a subject relative to treatment with the naturally occurring form of the EXOX proteins.<br>
Variants of the EXOX proteins that function as either EXOX agonists (e.g., mimetics) or as EXOX antagonists can be identified by screening combinatorial libraries of mutants (e.g., truncation mutants) of the EXOX proteins for EXOX protein agonist or antagonist activity, In one embodiment, a variegated library of EXOX variants is generated by combinatorial mutagenesis at the nucleic acid level and is encoded by a variegated gene library. A variegated library of EXOX variants can be produced by, for example,<br>
enzymatically ligating a mixture of synthetic oligonucleotides into gene sequences such that a degenerate set of potential EXOX sequences is expressible as individual polypeptides, or alternatively, as a set of larger fusion proteins (e.g., for phage display) containing the set of EXOX sequences therein. There are a variety of methods, which can be used to produce libraries of potential EXOX variants from a degenerate oligonucleotide sequence. Chemical synthesis of a degenerate gene sequence can be performed in an automatic DNA synthesizer, and the synthetic gene then ligated into an appropriate expression vector. Use of a degenerate set of genes allows for the provision, in one mixture, of all of the sequences encoding the desired set of potential EXOX sequences. Methods for synthesizing degenerate oligonucleotides are well-known within the art. See, e.g., Narang, Tetrahedron 39:3 (1983); Itakura et al., Annu. Rev. Biochem. 53:323 (1984); Itakura et al., Science 198:1056 (1984); Dee et al., Nucl. Acids Res. 11:477 (1983).<br>
Polypeptide Libraries<br>
In addition, libraries of fragments of the EXOX protein coding sequences can be used to generate a variegated population of EXOX fragments for screening and subsequent selection of variants of an EXOX protein. In one embodiment, a library of coding sequence fragments can be generated by treating a double stranded PCR fragment of an EXOX coding sequence with a nuclease under conditions wherein nicking occurs only about once per molecule, denaturing the double stranded DNA, renaturing the DNA to form double-stranded DNA that can include sense/antisense pairs from different nicked products, removing single stranded portions from reformed duplexes by treatment with Si nuclease, and ligating the resulting fragment library into an expression vector. By this method, expression libraries can be derived which encode N-terminal and internal fragments of various sizes of the EXOX proteins.<br>
Various techniques are known in the art for screening gene products of combinatorial libraries made by point mutations or truncation, and for screening cDNA libraries for gene products having a selected property. Such techniques are adaptable for rapid screening of the gene libraries generated by the combinatorial mutagenesis of EXOX proteins. The most widely used techniques, which are amenable to high throughput analysis, for screening large gene libraries typically include cloning the gene library into replicable expression vectors, transforming appropriate cells with the resulting library of vectors, and expressing the combinatorial genes under conditions in which detection of a desired activity facilitates<br>
isoiauon 01 me vcgior encoding uie gene whose product was detected. Recursive ensemble mutagenesis (REM), a new technique that enhances the frequency of functional mutants in the libraries, can be used in combination with the screening assays to identify EXOX variants. See, e.g., Arkin &amp; Yourvan, Proc. Natl. Acad. Sci. USA 89:7811-7815 (1992); Delgrave et al., Protein Engineering 6:327-331 (1993).<br>
Libraries can also be generated by DNA shuffling. DNA shuffling uses related genes from different species or genes that are related in their function, fragments them and reassembles them through recombination. It can then be determined if the recombined genes comprise usable or potentially interesting products. Any recombined gene found to be useful are again fragmented and reassembled to form new recombinant genes. As the various fragments of different species and genes are annealed and extended, diversity is created in the library. The process can be performed until a protein of interest is found. The important factors in creating recombined genes with DNA shuffling include the temperature at which annealing occurs, the similarity of the genes and the size of the DNA fragments.<br>
Stemmer et al., Nature 370:389-391 (1994); Stemmer, Proc. Natl. Acad. USA 91:10747-10751 (1994); U.S. Pat. No. 5,603,793; U.S. Pat. No. 5,830,721; and U.S. Pat. No. 5,811,238, which are incorporated herein by reference, describe e.g., in vitro protein shuffling methods, e.g., by repeated cycles of mutagenesis, shuffling and selection as well as a variety of methods of generating libraries of displayed peptides and antibodies as well as a variety of DNA reassembly techniques following DNA fragmentation, and their application to mutagenesis in vitro and in vivo. Moreover, various applications of DNA shuffling technology are also known in the art. In addition to the publications noted above, see U.S. Pat. No. 5,837,458, which provides for the evolution of new metabolic pathways and the enhancement of bio-processing through recursive shuffling techniques, and Crameri et a/., Nature Medicine 2(1): 100-103 (1996), which describes antibody shuffling for antibody phage libraries. See also, WO95/22625, WO97/20078, WO96/33207, WO97/33957, WO98/27230, WO97/35966, WO98/31837, WO98/13487, WO98/13485 and WO989/42832.<br>
Expression Vectors<br>
Another aspect of the invention pertains to vectors, preferably expression vectors, containing a nucleic acid encoding an EXOX protein, or derivatives, fragments, analogs or homologs thereof. As used herein, the term "vector" refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a<br>
"plasmid", which refers to a circular double stranded DNA loop into which additional DNA segments can be ligated. Another type of vector is a viral vector, wherein additional DNA segments can be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication and episomal mammalian vectors). Other vectors (e.g., non-episomal mammalian vectors) are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. Moreover, certain vectors are capable of directing the expression of genes to which they are operatively linked. Such vectors are referred to herein as "expression vectors". In general, expression vectors of used in recombinant DNA techniques are often in the form of plasmids. In the present specification, "plasmid" and "vector" can be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is intended to include such other forms of expression vectors, such as viral vectors (e.g., replication defective retroviruses, adenoviruses and adeno-associated viruses), which serve equivalent functions.<br>
The production of a functional protein is intimately related to the cellular machinery of the organism producing the protein. E. coli has typically been the "factory" of choice for the expression of many proteins because its genome has been fully mapped and the organism is easy to handle; grows rapidly; requires an inexpensive, easy-to-prepare medium for growth; and secretes protein into the medium which facilitates recovery of the protein. However, E. coli is a prokaryote and lacks intracellular organelles, such as the endoplasmic reticulum and the golgi apparatus that are present in eukaryotes, which contain enzymes which modify the proteins being produced. Many eukaryotic proteins can be produced in E. coli but these maybe produced in a nonfunctional, unfinished form, since glycosylation or post-translational modifications do not occur.<br>
Therefore, researchers have recently turned to eukaryotic yeast, mammalian and plant expression systems for protein production. For example, the methanoltrophic yeast P. pastoris has become a powerful host for the heterologous expression of proteins during the last few years and has been established as an alternative eukaryotic host for the expression of human proteins with high-throughput technologies.<br>
As another example, plants are being utilized as expression hosts for large-scale heterologous expression of proteins and offer potential advantages of cost-effectiveness, scalability and safety over traditional expression systems. There are currently a variety of<br>
plant heterologous expression systems including transient expression, plant cell-suspension cultures, recombinant plant viruses and chloroplast transgenic systems. While proteins expressed in plants have some variations from mammalian proteins (e.g., glycosylation), there is currently no evidence that these differences result in adverse reactions in human patients. See, e.g., Julian et al., Nat. Rev. Gen. 4:794-805 (2003).<br>
Another suitable heterologous expression system uses insect cells, often in combination with baculovirus expression vectors. Baculovirus vectors available for expressing proteins in cultured insect cells, e.g., SF9 cells include the pAc series (Smith et al, Mol. Cell. Biol. 3: 2156-2165 (1983)) and the pVL series (Lucklow &amp; Summers, Virology 170: 31-39 (1989)).<br>
Host cells of the invention can also be used to produce non-human transgenic animals in which exogenous sequences have been introduced into their genome. The transgenic animal is a non-human animal, preferably a mammal, more preferably a rodent such as a rat or mouse, in which one or more of the cells of the animal includes a transgene. Other examples of transgenic animals include, e.g., non-human primates, sheep, dogs, cows, goats, chickens, amphibians. Methods for generating transgenic animals via embryo manipulation and micro-injection, particularly animals such as mice, have become conventional in the art and are described, for example, in U.S. Patent Nos. 4,736,866; 4,870,009; and 4,873,191; and Hogan, 1986. In: MANIPULATING THE MOUSE EMBRYO, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. Similar methods are used for production of other transgenic animals.<br>
Pichia pastoris Expression System<br>
One such eukaryotic yeast is the methanoltrophic Pichia pastoris. P. pastoris has been developed to be an outstanding host for the production of foreign proteins since its alcohol oxidase promoter was isolated and cloned: The P. pastoris transformation was first reported in 1985. The P. pastoris heterologous protein expression system was developed by Phillips Petroleum, see, e.g., U.S. Patent NOs. 4,855,231,4,857,467,4,879,231 and 4,929,555, each of which is incorporated herein by reference. This system is currently marketed by Invitrogen. Compared to other eukaryotic expression systems, Pichia offers many advantages, because it does not have the endotoxin problem associated with bacteria nor the viral contamination problem of proteins produced in animal cell cultures. Furthermore, P. pastoris can utilize methanol as a carbon source in the absence of glucose. The P. pastoris<br>
expression system uses the methanol-induced alcohol oxidase (AOX1) promoter, which controls the gene that codes for the expression of alcohol oxidase, the enzyme that catalyzes the first step in the metabolism of methanol. This promoter has been characterized and incorporated into a series of P. pastoris expression vectors. Since the proteins produced in P. pastoris are typically folded correctly and secreted into the medium, the fermentation of genetically engineered P. pastoris provides an excellent alternative to E. coli expression systems. Furthermore, P. pastoris has the ability to spontaneously glycosylate expressed proteins, which also is an advantage over E. coli. A number of proteins have been produced using this system, including tetanus toxin fragment, Bordatella pertussis pertactin, human serum albumin and lysozyme.<br>
Tag Removal with EXOX Proteins<br>
Several systems have been developed to allow for rapid and efficient purification of recombinant proteins expressed in bacteria. Most of these rely on the expression of the protein as a fusion protein with a glutathione-S-transferase (GST) domain, a calmodulin binding peptide (CBP) or a His-tag. For example, the expression of polypeptides in frame with glutathione S-transferase (GST) allows for purification of the fusion proteins from crude bacterial extracts under nondenaturing conditions by affinity chromatography on glutathione agarose.<br>
Furthermore, this vector expression system generally incorporates a specific protease cleavage site to facilitate proteolysis of the bacterial fusion proteins, which is, depending on the vector used, a thrombin, enterokinase or Factor Xa protease cleavage site. Thrombin specifically cleaves target proteins containing the recognition sequence Leu-Val-Pro-ArgJOly-Ser (SEQ ID NO: 44). The enterokinase cleavage site is Asp-Asp-Asp-Asp-Lys| (SEQ ID NO: 45). Like enterokinase, Factor Xa cleaves at the C-terminal side of its recognition sequence De-Glu-Gly-Argi (SEQ ID NO: 46), and can therefore be used for removing all vector-encoded sequences from appropriately designed constructs. All of these enzymes are now commercially available in a high purity to avoid secondary cleavage arising from contaminating proteases. These enzymes are provided either in a kit often including all the tools for the enzyme capture, or biotinylated to facilitate removal of the enzyme from cleavage reaction medium. More recently Qiagen also developed the TAGZyme system for an efficient removal of N-terminal His tags from proteins which involves exopeptidases that cleave dipeptides sequentially from the N-terminus up to a "stop point" amino acid motif,<br>
which is either JLys-Xaa-, JArg-Xaa-, JXaa-Xaa-Pro-Xaa-, |Xaa-Pro-Xaa-Xaa- or JGln-Xaa-.<br>
Although it is not always necessary to remove the short His affinity tag (whatever the number of His residues) from a recombinant protein after purification, there are some applications, such as structural analysis by X-ray crystallography or NMR, where removal of the tag is desirable. The same thing is also true for the residual residues Gly-Ser of the thrombin cleavage site or any supplementary residual N-terminal amino acid that could be still present and which could be related to the expression system used.<br>
A more recent approach to affinity purification involves utilizing a condensation reaction between a carbonyl group and a molecule with two vicinal nucleophilic groups. Examples of amino acids with two vicinal nucleophilic groups includes, e.g., serine, threonine and cysteine. Purifying a protein or peptide involves forming a reversible covalent bond formed by between, e.g., an N-terminal cysteine, threonine or serine residue, and an appropriate resin. See Villain et a/., Chem. &amp; Biol. 8:673-679 (2001). Addition of a pair of residues, e.g., Thr-Pro, Cys-Pro or Ser-Pro, to the N-terminus of a recombinant protein, or of a protein (peptide) obtained by chemical synthesis, permits two-step purification: (1) purification by covalent capture; and (2) removal of the di-peptide tag. This method permits efficient recovery of recombinant protein in its mature form, without the di-peptide flag sequence.<br>
Reverse Proteolytic Activity of EXOX Proteins<br>
Another aspect of the invention pertains to methods of adding one or more amino acids to amino acids, peptides, oligopeptides, polypeptides or any composition with an accessible secondary amine, by using the reverse proteolytic activity of one or more EXOX proteins. As used herein, the term "reverse proteolytic activity" refers to enzymatic activity that catalyzes the addition of one or more amino acids to an amino acid, a peptide, an oligopeptide, a polypeptide or any composition with an accessible secondary amine. One of ordinary skill in the art will recognize that, under suitable thermodynamic conditions, proteolytic enzymes can have reverse proteolytic activity.<br>
An example of a proteolytic enzyme with reverse proteolytic activity is trypsin, which is a pancreatic serine protease with substrate specificity based upon positively charged lysine and arginine side chains. Trypsin is widely used in the manufacture of human insulin from porcine insulin, which is similar to the human form except the last amino acid residue in the<br>
B-chain is alanine rather than threonine. Reacting porcine insulin with a threonine ester in the presence of trypsin yields a human insulin threonine ester by removing the terminal alanine and adding the threonine ester. Subsequent treatment of the human insulin threonine ester with trifluoroacetic acid hydrolyzes the ester to yield human insulin.<br>
In some embodiments, the EXOX proteins are used to catalyze reverse proteolytic reactions. In some instances, the EXOX proteins are incubated with a polypeptide and one or more amino acids under conditions permitting the addition of the one or more amino acids to the polypeptide.<br>
There are multiple utilities for using the EXOX proteins of the present invention as reverse proteolytic enzymes. For example, the reverse proteolytic activity of the EXOX proteins can be used in the synthesis of a polypeptide chain. The EXOX proteins can also be used as a coupling agent to add one or more amino acids to another amino acid, a polypeptide, or any composition with an accessible secondary amine.<br>
Pharmaceutical Compositions<br>
The EXOX nucleic acid molecules, EXOX proteins, and anti-EXOX antibodies (also referred to herein as "active compounds") of the invention, and derivatives, fragments, analogs and homologs thereof, can be incorporated into pharmaceutical compositions suitable for administration. Such compositions typically comprise the nucleic acid molecule, protein, or antibody and a pharmaceutically acceptable carrier. As used herein, "pharmaceutically acceptable carrier" is intended to include any and all solvents, dispersion media, coatings, antibacterial and antifungal agents, isotonic and absorption delaying agents, and the like, compatible with pharmaceutical administration. Suitable carriers are described in the most recent edition of Remington's Pharmaceutical Sciences, a standard reference text in the field, which is incorporated herein by reference. Preferred examples of such carriers or diluents include, but are not limited to, water, saline, Ringer's solutions, dextrose solution, and 5% human serum albumin. Liposomes and non-aqueous vehicles such as fixed oils may also be used. The use of such media and agents for pharmaceutically active substances is well known in the art. Except insofar as any conventional media or agent is incompatible with the active compound, use thereof in the compositions is contemplated. Supplementary active compounds can also be incorporated into the compositions.<br>
Encapsulation technologies are also widely applied in many industries. Examples include Pharmaceuticals for controlled release of drugs; pigments in foods and beverages;<br>
antioxidants in foods; and controlled release of insect pheromones in agriculture. Capsules, microcapsules and microspheres are small spherical particles, which contain an active ingredient within the particle matrix or attached to the particle surface. For example, encapsulation in biodegradable alginate microparticles has been shown. Bioencapsulation technologies are intended to encapsulate cells, enzymes, and biologically active materials.<br>
A pharmaceutical composition of the invention is formulated to be compatible with its intended route of administration. Examples of routes of administration include parenteral, e.g., intravenous, intradermal, subcutaneous, oral (e.g., inhalation), transdermal (e.g., topical), transmucosal, and rectal administration. Solutions or suspensions used for parenteral, intradermal, or subcutaneous application can include the following components: a sterile diluent such as water for injection, saline solution, fixed oils, polyethylene glycols, glycerine, propylene glycol or other synthetic solvents; antibacterial agents such as benzyl alcohol or methyl parabens; antioxidants such as ascorbic acid or sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid (EDTA); buffers such as acetates, citrates or phosphates, and agents for the adjustment of tonicity such as sodium chloride or dextrose. The pH can be adjusted with acids or bases, such as hydrochloric acid or sodium hydroxide. The parenteral preparation can be enclosed in ampoules, disposable syringes or multiple dose vials made of glass or plastic.<br>
Pharmaceutical compositions suitable for injectable use include sterile aqueous solutions (where water soluble) or dispersions and sterile powders for the extemporaneous preparation of sterile injectable solutions or dispersion. For intravenous administration, suitable carriers include physiological saline, bacteriostatic water, Cremophor EL™ (BASF, Parsippany, N. J.) or phosphate buffered saline (PBS). In all cases, the composition must be sterile and should be fluid to the extent that easy syringeability exists. It must be stable under the conditions of manufacture and storage and must be preserved against contamination by microorganisms, such as bacteria, fungi or viruses. The carrier can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (for example, glycerol, propylene glycol, and liquid polyethylene glycol, and the like), and suitable mixtures thereof. The proper fluidity can be maintained, for example, by the use of a coating such as lecithin, by the maintenance of the required particle size in the case of dispersion and by the use of surfactants. Prevention of the action of microorganisms can be achieved by various antibacterial and antifungal agents, for example, parabens, chlorobutanol, phenol, ascorbic acid, thimerosal, and the like. In many cases, it will be preferable to include isotonic agents,<br>
for example, sugars, polyalcohols such as mannitol, sorbitol, or sodium chloride in the composition. Prolonged absorption of the injectable compositions can be brought about by including in the composition an agent that delays absorption, for example, aluminum monostearate and gelatin.<br>
Sterile injectable solutions can be prepared by incorporating the active compound (e.g., an EXOX protein or anti-EXOX antibody) in the required amount in an appropriate solvent with one or a combination of ingredients enumerated above, as required, followed by filtered sterilization. Generally, dispersions are prepared by incorporating the active compound into a sterile vehicle that contains a basic dispersion medium and the required other ingredients from those enumerated above. In the case of sterile powders for the preparation of sterile injectable solutions, methods of preparation are vacuum drying and freeze-drying that yields a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof.<br>
A crude preparation of cell culture medium from T. rubrum or transgenic fungi producing EXOX, or EXOX purified from T. rubrum or transgenic fungi producing EXOX can be administered orally since the proteases are secreted. Oral compositions generally include an inert diluent or an edible carrier. They can be enclosed in gelatin capsules or compressed into tablets. For the purpose of oral therapeutic administration, the active compound can be incorporated with excipients and used in the form of tablets, troches, or capsules. Oral compositions can also be prepared using a fluid carrier for use as a mouthwash, wherein the compound in the fluid carrier is applied orally and swished and expectorated or swallowed. Fharmaceutically compatible binding agents, and/or adjuvant materials can be included as part of the composition. The tablets, pills, capsules, troches and the like can contain any of the following ingredients, or compounds of a similar nature: a binder such as microcrystalline cellulose, gum tragacanth or gelatin; an excipient such as starch or lactose, a disintegrating agent such as alginic acid, Primogel, or corn starch; a lubricant such as magnesium stearate or Sterotes; a glidant such as colloidal silicon dioxide; a sweetening agent such as sucrose or saccharin; or a flavoring agent such as peppermint, methyl salicylate, or orange flavoring.<br>
For administration by inhalation, the compounds are delivered in the form of an aerosol spray from pressured container or dispenser, which contains a suitable propellant, e.g., a gas such as carbon dioxide, or a nebulizer.<br>
Systemic administration can also be by transmucosal or transdermal means. For transmucosal or transdermal administration, penetrants appropriate to the barrier to be permeated are used in the formulation. Such penetrants are generally known in the art, and include, for example, for transmucosal administration, detergents, bile salts, and fusidic acid derivatives. Transmucosal administration can be accomplished through the use of nasal sprays or suppositories. For transdermal administration, the active compounds are formulated into ointments, salves, gels, or creams as generally known in the art.<br>
The compounds can also be prepared in the form of suppositories (e.g., with conventional suppository bases such as cocoa butter and other glycerides) or retention enemas for rectal delivery.<br>
In one embodiment, the active compounds are prepared with carriers that will protect the compound against rapid elimination from the body, such as a controlled release formulation, including implants and microencapsulated delivery systems. Biodegradable, biocompatible polymers can be used, such as ethylene vinyl acetate, polyanhydrides, polyglycolic acid, collagen, polyorthoesters, and polylactic acid. Methods for preparation of such formulations will be apparent to those skilled in the art. The materials can also be obtained commercially from, for example, Alza Corporation and Nova Pharmaceuticals, Inc. Liposomal suspensions (including liposomes targeted to infected cells with monoclonal antibodies to viral antigens) can also be used as pharmaceutically acceptable carriers. These can be prepared according to methods known to those skilled in the art, for example, as described in U.S. Patent No. 4,522,811.<br>
It is especially advantageous to formulate oral or parenteral compositions in dosage unit form for ease of administration and uniformity of dosage. Dosage unit form as used herein refers to physically discrete units suited as unitary dosages for the subject to be treated; each unit contains a predetermined quantity of active compound calculated to produce the desired therapeutic effect in association with the required pharmaceutical carrier. The specification for the dosage unit forms of the invention are dictated by and directly dependent on the unique characteristics of the active compound and the particular therapeutic effect to be achieved, and the limitations inherent in the art of compounding such an active :ompound for the treatment of individuals.<br>
The nucleic acid molecules of the invention can be inserted into vectors and used as &gt;ene therapy vectors. Gene therapy vectors can be delivered to a subject by, for example, ntravenous injection, local administration (see, e.g., U.S. Patent No. 5,328,470) or by<br>
stereotactic injection. See. e.g., Chen, et al, Proc. Natl. Acad. Sci. USA 91:3054-3057 (1994). The pharmaceutical preparation of the gene therapy vector can include the gene therapy vector in an acceptable diluent, or can comprise a slow release matrix in which the gene delivery vehicle is imbedded. Alternatively, where the complete gene delivery vector can be produced intact from recombinant cells, e.g., retroviral vectors, the pharmaceutical preparation can include one or more cells that produce the gene delivery system.<br>
The pharmaceutical compositions can be included in a container, pack, or dispenser together with instructions for administration.<br>
Examples<br>
Example 1; Methods and Materials Strains and plasmids<br>
A clinical isolate, T. rubrum CHUV 862-00, was used in this study. E. coli LE392 was used for the propagation of the bacteriophage XEMBL3 (Promega, Wallisellen, Switzerland). All plasmid-subcloning experiments were performed in E. coli DH5a using plasmid pMTL2I. Chambers et al., Gene 68:139-149 (1988). P. pastoris GSI15 and the expression vector pKJ113 (Borg-von Zepelin et al., Mol. Microbiol. 28:543-554 (1998)) were used to express recombinant peptidases. It is known in the art that P. pastoris can be utilized to express a multitude of recombinant proteins.<br>
7. rubrum growth media<br>
T. rubrum was grown on Sabouraud agar and liquid medium (Bio~Rad, Munchen, Germany) or, to promote production of proteolytic activity, in liquid medium containing 0.2% soy protein (Supro 1711, Protein Technologies International, St.Louis, MO) as a sole nitrogen and carbon source. No salt was added in this medium. Those skilled in the art will recognize it is also possible to utilize growth media in which salt is added to the medium. A volume of 100 ml of liquid medium was inoculated with a plug of freshly growing mycelium in 800 ml.-tissue culture flasks. The cultures were incubated 10 days at 30°C without shaking.<br>
Genomic and cDNA libraries<br>
A T. rubrum genomic DNA library was prepared using DNA isolated from freshly<br>
growing mycelium. (Yelton et al, Proc. Natl. Acad. Sci. USA. 81:1470-1474 (1984). The DNA was partially digested with SauSA and DNA fragments ranging from 12 to 20 kb were isolated from low-melting-point agarose (Roche Diagnostics, Rotkreuz, Switzerland) with agarase (Roche Diagnostics). These DNA fragments were inserted into bacteriophage XEMBL3 using an appropriate cloning system (Promega).<br>
A T. rubrum cDNA library was prepared in a pSPORT6 plasmid (Invitrogen Life Technologies; Rockville, Maryland, USA) using the microquantity mRNA system and 500 jig of total RNA. The RNA was prepared from 10-day-old cultures in soy protein liquid medium (10x100 ml). The mycelium was ground under liquid nitrogen to a fine powder using a mortar and pestle, and the total RNA was isolated using an RNeasy total RNA purification kit for plant and fungi (Qiagen, Basel, Switzerland).<br>
An A. fumigatus cDNA library was previously constructed with the CHUVI92-88 strain grown 40 h at 30°C in liquid medium containing 0.2% collagen as a sole nitrogen and carbon source (Monod et al, 1991). Total RNA was extracted as described (Applegate and Monod) and the mRNA was purified using oligo(dT) cellulose (Sigma, Buchs, Switzerland) according to standard protocols (Sambrook et al., 1989). A library was prepared with this mRNA using lambda phage gtll (Promega) and the protocols of the manufacturer. Table 13 shows T. rubrum and A. fumigatus genes encoding aminopeptidases(Table Removed)<br>
LAP gene cloning<br>
Recombinant plaques (104) of the genomic library were immobilized on GeneScreen nylon membranes (MEN Life science products, Boston, MA). The filters were hybridized<br>
with 32P-labelled probe using low-stringency conditions. Monod et al, Mol. Microbiol. 13:357-368 (1994). All positive plaques were purified and the associated bacteriophage DNAs were isolated as described by Grossberger. Grossberger, Nucleic Acid Res. 15:6737 (1987). Hybridizing fragments from EMBL3 bacteriophages were subcloned into pMTL2I following standard procedures. Nucleotide sequencing was performed by Microsynth (Balgach, Switzerland).<br>
Isolation of cDNA by standard PCR<br>
T. rubrum and A.fumigatus cDNAs were obtained by PCR using DNA prepared from 106 clones of the cDNA libraries. PCR was performed according to standard conditions using homologous primers derived from DNA sequences of the different peptidase genes (Table 13). Two hundred ng of DNA, 10 ul of each sense and antisense oligonucleotides at a concentration of 42 mM and 8 ul of deoxynucleotide mix (containing 10 rnM of each dNTP) were dissolved in 100 ul PCR buffer (10 mM Tris-HCI pH 8.3, 50 mM KCI and 1.5 mM MgCb). To each reaction 2.5 units of AmpliTAQ DNA polymerase (Perkin Ehner, Zurich, Switzerland) were added. The reaction mixtures was incubated 5 mm at 94°C, subjected to 25 cycles of 0.5 mm at 94°C, 0.5 mm at 55°C and 0.5 mm at 72°C and finally incubated 10 mm at 72°C.<br>
Production of recombinant LAPs<br>
Expression plasmids were constructed by cloning cDNA PCR products in the multiple cloning site of the E. coli - P. pastoris shuttle vector pKJ113. The PCR products were purified using a PCR purification kit (Roche Diagnostics) and digested by restriction enzymes for which a site was previously designed at the 5' extremity of the primers (Table 14). P. pastoris GSI15 (Invitrogen) was transformed by electroporation with 10 pg of plasmid DNA linearized by EcoRl or Smal. Transformants selected on histidine-deficient medium (1 M sorbitol, 1% (w/v) dextrose, 1.34% (w/v) yeast nitrogen base (YNB) without amino acids, 4 x 10'5% (vr/v) biotin, 5 x 10"3% amino acids (e.g. 5 x 10"3% (w/v) of each Lglutamic acid, L-methionine, L-lysine, L-leucine, L-isoleucine), 2% (w/v) agarose) were screened for insertion of the construct at the AOX1 site on minimal methanol plates (1.34% (w/v) YNB without amino acids, 4 x 10"5 % (wlv) biotin, 0.5% (v/v) methanol, 2% (w/v) agarose). The transformants unable to grow on media containing only methanol as a carbon source were assumed to contain the construct at the correct yeast genomic location by integration events<br>
in the AOXI locus displacing the AOX1 coding region. These transformants were grown to near saturation (OD 20 at 600 nra) at 30°C in 10 ml of glycerol-based yeast media (0.1 M potassium phosphate buffer at pH 6.0, containing 1% (w/v) yeast extract, 2% (w/v) peptone, 1.34% (w/v) YNB without amino acids, 1% (vN) glycerol and 4 x 1 % fw/v) bion'n). Cells were harvested and resuspended in 2 ml of the same medium with 0,5% (v/v) methanol instead of glycerol and incubated for 2 days. After 2 days of incubation, the supernatant was harvested and tested for protein production on SDS-PAGE gels. Recombinant peptidase enzymes were produced in large quantities from 400 ml cell culture supernatant<br>
Table 14 describes materials used for the expression of the different LAPs in P. pastoris(Table Removed)<br>
Purification of recombinant LAPs<br>
The secreted proteins from 400 ml of P. pastoris culture supernatant were concentrated by ultrafiltration using an Amicon cell and an Ultracel Amicon YM3O membrane (30 kDa cut-off) (Millipore, Volketswil, Switzerland). The concentrate was washed with 50 mM Tris-HCl, pH 7.5 and applied to a Mono Q-Sepharose (Amersham Pharmacia, Diibendorf, Switzerland) column equilibrated with the same buffer. After washing the column with 50 mM Tris-HCl, pH 7.5, elution was performed with a linear gradient of 0-0.5 M NaCl at a flow-rate of 1 ml/min. The different fractions eluted from the Mono Q-Sepharose column were screened for enzymatic activity using Leucine-7-amino-4-methylcoumarin (Leu-AMC) as a substrate and LAP-containing fractions were pooled. After concentration in an Amicon ultrafiltration cell with an Ultracel Amicon YM30 . membrane and washing with 20mM Tris-HCl, pH 6.0, the LAP extract was loaded on a size exclusion Superose 6 FPLC column (Amersham Pharmacia) and elution was performed at a flow-rate of 0.2 ml/min using 20mM Tris-HCl, pH 6.0 as eluant. The eluted active fractions were pooled The LAP enzyme was concentrated to a final volume of 0.4-1.0 ml in a Centricon concentrator witf a 30 kDa cut-off (Millipore) at 4°C prior to further functional characterization.<br>
In an alternative purification scheme, each step of purification was performed at 4°C. The secreted proteins from 400 ml of P. pastoris culture supernatant were concentrated by ultrafiltratior using an Amicon cell and an Ultracel Amicon YM3O membrane (30 kDa cut-off) (Millipore, Volketswil, Switzerland). The concentrate was washed with 100 ml of 20 mM sodium acetate, pH 6.0 and applied to a Mono Q-Sepharose (Amersham Pharmacia, Dubendorf, Switzerland) column equilibrated with the same buffer. After washing the column with 20 mM Tris-HCl pH 6.0 buffer, the enzyme was eluted with a linear gradient of 0-0.2 M NaCl at a flow-rate of 1 ml/min over 142 min. The different fractions eluted from the Mono Q-Sepharose column were screened for enzymatic activity using Leucine-7-amino-4-methylcoumarin (Leu-AMC) as a substrate (see below) and LAP-containing fractions were pooled. After concentration in an Amicon ultrafiltration cell with an Ultracel Amicon YM30 membrane and washing with PBS, the LAP extract was loaded on a size exclusion Superdex 200 FPLC column (Amersham Pharmacia) using 20 mM sodium acetate pH 6.0 buffer and elution was performed at a flow-rate of 0.2 ml/min. The eluted active fractions were pooled. The LAP enzyme was subjected to further characterization after concentration to a final volume of 0.4-1.0 ml in a Centricon concentrator with a 30 kDa cut-off (Millipore) at 4°C.<br>
A fraction containing ruLAP2 activity elutes from MonoQ at 30-40 min (approx. 50 mM NaCl) and at 65-70 min with superdex 200 = Peak 3. However, a large amount of LAP2 activity<br>
was not retained and eluted in the flow-through at 1 M NaCl. Therefore, after desalting this fractioi with 20 mM sodium acetate, the sample was applied on the same MonoQ column with a wider gradient between 0 and 1 M NaCl over 142 min at 0.5 ml/min. A first peak of activity eluates at 7-15 min corresponding to 70-140 mM NaCl and a second peak elutes at 150-250 mM NaCl (with more activity content). The fraction at 70-140 mM NaCl elutes at 78-80 min on Superdex and was therefore pooled with peak 3 obtained above. The fraction at 150-250 mM NaCl gives two active fractions eluting respectively at 44-49 min (Peak 1) and 50-63 min (Peak 2) on Superdex .<br>
Protein extract analysis<br>
Protein extracts were analyzed by SDS-PAGE with a separation gel of 12% polyacrylamide Gels were stained with Coomassie brilliant blue R-250 (Bio-Rad). N-glycosidase F digestion was performed as previously described. Doumas et al., Appl. Environ. Microbiol. 64:4809-4815 (1998]<br>
Western blots<br>
The membranes were first stained with Red-Ponceau and the major protein bands were marked with a needle. Immunoblots were performed using rabbit antisera and alkaline phosphatase conjugated goat anti-rabbit IgG (Bio-Rad) or peroxidase-conjugated goat anti-rabbit IgG (Amersham Pharmacia) as secondary labeled antibodies. Rabbit antisera to ruLAPl, ruLAP2, A. oryzae secreted alkaline protease (ALP) and A. oryzae secreted neutral protease (NPI) of the fungalysin family (Doumas et al, J. Food Mycol. 2:271-279 (1999)) were made by Eurogentec (Liege, Belgium) using purified recombinant enzyme.<br>
Aminopeptidase activity assay<br>
Aminopeptidase activity was determined using different fluorogenic aminoacyl-4-methylcoumaryl-7-amide derivatives of peptides and the internally quenched fluorogenic substrate Lys(Abz)-Pro-Pro-pNA for specific determination of aminopeptidase P activity. Stockel et al, Ad\ Exp. Med. Biol. 421:31-35 (1997). All substrates were from Bachem (Bubendorf, Switzerland). Substrate stock solutions were prepared at 0.1 M according to the recommendations of the manufacturer and stored at -20°C. The reaction mixture contained a concentration of 5 mM substrate and enzyme preparation (between 56 and 2,662 ng per assay depending on the cleavage activity of each enzyme for the substrates) in 25 ^1 of 50 mM Tris-HCl buffer adjusted at the optimal pH for each LAP (between 7 and 8). After incubation at 37°C for 60 min, the reaction was terminated by adding 5 \i\ of glacial acetic acid and the reaction mixture was diluted with 3.5 ml of<br>
water. The released 7-amino-4-methylcoumarin (AMC) was measured using a spectrofluorophotometer (Perkin Elmer LS-5 fluorometer, Zurich, Switzerland) at an excitation wavelength of 370 nm and an emission wavelength of 460 nm. A standard curve made with synthetic AMC was used to assess the released AMC. The released diprolyl-p-nitroanilide was measured at an excitation wavelength of 310 nm and an emission wavelength of 410 nm. The LA activities were expressed in nmoles of released AMC or pNA/min/|Ag protein.<br>
Table IS details the hydrolytic activity of different LAPs toward various aminoacyl-MCA comparison (%) to Leu-MCA used as a standard.<br>
(Table Removed)Effect of various chemical reagents on LAPs<br>
Inhibitors and metallic cations were pre-incubated with the enzymes for 15 min at 37°C. Then, Leu-AMC at a 5 mM final concentration was added. After further incubation for 60 min, enzyme activity was measured as described above. The inhibitors and their concentrations tested 
MgCl2, MnCfe, Coda, ZnCfe., NiCl2, CuClz were tested at concentrations of 0.5 mM and 1 mM.<br>
Table 16 details the hydrolytic activity of different EXOXs in the presence of various protease inhibitors using Leu-MCA as a substrate for LAP. The activity is given as a percentage of the activity of control enzymatic reaction without inhibitor.<br>
(Table Removed)Table 17 details the hydrolytic activity of different EXOs in the presence of various cations using Leu-MCA as a substrate for LAP. The activity is given as the percentage of the activity of control enzymatic reaction without any cation.<br>
(Table Removed)Optimal pH of activity of EXOXs<br>
The optimal pH for enzymatic activities was determined using the Ellis and Morrison buffo system. Ellis &amp; Morrison, Methods Enzymol. 87:405-426 (1982). The buffer contained three components with different pKa values while the ionic strength of buffer remained constant throughout the entire pH range examined. The pH of the buffer was adjusted from 6 to 11 in half-pH unit increments with 1M HC1 or 1M NaOH. The assay conditions for activity on Leu-AMC substrates was the same as above except that the Tris/HCl buffer was replaced by the Ellis and Morrison buffer (composition) at the pH values indicated.<br>
Table 18 details characteristics of native and recombinant T. rubrum and A. fumigatus secreted aminopeptidases. (Table Removed)§NI: means not determined<br>
"The value in brackets corresponds to full-length polypeptide without prosequence.94-<br>
Temperature optima of activity of EXOXs<br>
The optimal temperature conditions were determined by measuring the enzymatic activity £ their pH optima after incubating each of the LAPs with Leu-AMC (5 mM) at 20,30, 40, 50,60,70 and 80°C for 10,30 and 60 min.<br>
Proteolytic assays<br>
The proteolytic activity was measured using resorufin-labeled casein in phosphate buffer (20 mM; pH 7.4). The reaction mixture contained 0.02% substrate in a total volume of 0.5 ml. Afte incubation at 37°C, the undigested substrate was precipitated by trichloroacetic add (4% final concentration) and separated from the supernatant by centrifugation. The absorbance at 574 nm of the supernatant was measured after alkalinization by adding 500 uJ Tris buffer (500 mM; pH 9.4). For practical purposes, one unit (U) of proteolytic activity was defined as that producing an absorbance of 0.001 per min.<br>
Example 2; T. rubrum secreted proteolytic activity<br>
T. rubrum was grown at 30°C in a medium containing 0.2% soy protein as a sole carbon an nitrogen source. After 14 days of growth, a concomitant clarification of the culture medium was noted and a substantial proteolytic activity (400 U ml*1) detected using resorufin-labeled casein as substrate. This proteolytic activity was 15% and 85% inhibited by PMSF and ortho-phenanthroline. respectively, attesting that serine and metalloproteases were secreted by T. rubrum. Western blot analysis of culture supernatant revealed that T. rubrum, like M. canis, secreted endoproteases of the subtilisin family (MEROPS&gt;S8) and of the fungalysin family (MEROPS&gt;M36) similar to the alkaline protease ALP and the neutral metalloprotease NPI secreted by A. oryzae (See Figure 1). In addition, a high activity on substrates such as Leu-AMC and Leu-pNA was detected in the T. rubrum culture supernatant.<br>
Example 3; T. rubrum secreted aminopeptidase activity<br>
The nucleotide sequences of Microsporum canis endoprotease genes showed 50-70% similarity to homologous genes encoding the subtilisins and the fungalysins secreted by A. oryzae and A.fumigatus. In addition, the M. canis and Aspergittus genes showed colinear intron-exon structures. Therefore, DNA sequences available for A. oryzae and Sacharomyces cerevisiae genes coding for aminopeptidases were used to design probes for screening a T. rubrum genomic DNA<br>
library. Characterization of the T. rubrum secreted aminopeptidases in comparison to those secret* by the opportunist A. fumigatus was performed using recombinant proteins.<br>
Example 4: Cloning of genes encoding T. rubrum and A. fumieatus aminopeptidases<br>
Tables 19A and 19B detail a pairwise comparison of various LAPs(Table Removed)<br>
Figure 14 is an alignment of deduced ammo acid sequences of aminopeptidases of the M28E subfamily. Putative signal sequence processing sites are underlined. A putative KR processing site in ruLAPl is indicated by a solid triangle. The amino acids of the two Zn~* binding sites in 5. griseus aminopeptidase and conserved in the other LAPs are indicated by an open arrow The alignment was performed with the Pileup algorithm implemented in the GCG package of the University of Wisconsin and reformatted with Boxshade 3.2. AbispLAPl is for LAP ofAgaricus bisporus.<br>
Figure 15 is an alignment of deduced amino acid sequences of aminopeptidases of the M28A subfamily. Putative signal sequence processing sites are underlined. Two amino acid residues, His and Asp, conserved in the fungal LAPs and binding a first Zn** ion in S. griseus<br>
aminopeptidase are indicated by open triangles. Two additional residues His and Glu binding a second Zn*"1" ion are indicated by solid diamonds, while the Asp residue bridging die two Zn"1"*" ions is indicated by an open arrow. The * represent methionine residues found only in ruLAP2. The alignment was performed with the Pileup algorithm implemented in the GCG package of the University of Wisconsin and reformatted with Boxshade 3.2.<br>
The amino acid sequences GPGINDDGSG (SEQ ID NO: 36) and DM(Q/M)ASPN (SEQ IE NO: 37) were found in a A. oryzae secreted 52 kDa aminopeptidase (U.S. patent 6,127,161) and the S. cerevisiae aminopeptidase. Nishizawa et al., J. Biol. Chem. 269:13651-13655 (1994). From these data, two consensus oligonucleotides (GGXATXAAYGAYGAYGGXTCXGG (SEQ ID NO: 38) and TTXGGXGAXGCXATCATRTC (SEQ ID NO: 39) were used as sense and antisense, respectively, to amplify DNA from T. rubrum. A 220bp PCR product was obtained and sequenced. The deduced amino acid sequence showed high similarity to the amino acid sequence of the A. oryzae and the S. cerevisiae aminopeptidases. This 220bp PCR fragment was used as a probe for screening a X phage EMBL3 T. rubrum genomic DNA library and a nucleotide sequence coding foi a putative aminopeptidase (ruLAP2) was found. A nucleotide sequence coding for a similar secreted aminopeptidase (fuLAP2) was found in the A.fumigatus genome sequence (at website address www.TIGR.com).<br>
A 1200 bp fragment containing the nucleotide sequence of the gene encoding an A. oryzae 31 kDa aminopeptidase (U.S. patent 5,994,113) was obtained by PCR of A. oryzae genomic DNA using the oligonucleotides GCATTCCTGUGATGCCCGGGCCG (sense) (SEQ ID NO: 40) and TTACTTAGCAAGCTCAGTGACGAAGCCGAC (antisense) (SEQ ID NO: 41). This fragment was used as a probe for a second screening of the 7! rubrum genomic DNA library. A nucleotide sequence (EMBL) similar to those coding for the A. oryzae 30 kDa aminopeptidase and to another putative secreted aminopeptidase from the A.fumigatus genome sequence (at website address www.TlGR.com) was found in \ phage EMBL3 DNA of the T. rubrum genomic library. These T. rubrum and A.fumigatus putative aminopeptidases were called ruLAPl and fuLAPl, respectively.<br>
The identified nucleotide sequencesof ruLAPl, ruLAP2, fuLAPl and fuLAP2 each contain a 17-20 amino acid signal sequence. The intron-exon structure of the T. rubrum and A.fumigatus genes was determined by sequencing a PCR product using 5'-sense and 3'-antisense primers based on isolated genomic DNA (See Table 14) and total DNA from a pool of 108 clones of the T. rubrum or A.fumigatus cDNA libraries as a target. The first of the three introns in ruLAP2 was in position similar to that of the unique intron offuLAP2 (See Table 13). The genes ruLAPl andJuLAPl have similar colinear structures with two introns and three exons.<br>
Example 5: Production of recombinant T. rubrum and A. fumieatus aminopeptidases<br>
The T. rubrum and A.fianigatus cDNAs obtained by RT-PCR were cloned in pKJ113 (Borg-von Zepelin et al., 1998) and expressed in P. pastoris. Depending on the peptidase produced, about 10-80 yg/ml of active enzyme on Leu-AMC was obtained (See Table 18). Under identical culture conditions wild type P. pastoris did not secrete any leucine aminopeptidase activity into the culture medium. SDS-PAGE analysis of recombinant ruLAP2, fuLAPl and fuLAP2 secreted by P. pastoris transformants showed a smearing band (Figure 2). Upon treatment with N-glycosidase F, only a major band with a faster migration appeared on the gels attesting that, in contrast to ruLAPl, these three LAPs were glycoproteins (Figure 2). The apparent molecular mass of each deglycosylated recombinant LAP was close to that of the calculated molecular mass of the polypeptide chain deduced from the nucleotide sequence of the genes encoding the protease. The deduced primary structures (amino acid sequences) of each recombinant enzyme are provided in Table 18.<br>
Example 6; Detection of ruLAPl and ruLAPl in T. rubrum culture supernatant<br>
Using anti-ruLAPl antiserum, an accumulation of a LAP1 product with an electrophoretic mobility higher than that of recombinant ruLAPl was detected in the T. rubrum culture supernatant (See Figure 3).<br>
Using anti-ruLAP2 antiserum, Western blot analysis of a T. rubrum culture supernatant revealed that T. rubrum secreted glycosylated LAP2 with the same electrophoretic mobility as that of the recombinant enzyme from P. pastoris (See Figure 3).<br>
Example 7; Properties of recombinant LAPs<br>
The aminopeptidases ruLAPl, ruLAP2, fuLAPl, fuLAP2, as well as the microsomal jorcine kidney aminopeptidase (pkLAP) each efficiently hydrolyzed Leu-AMC. This substrate was ised to determine the optimum temperature and pH of activity, and to further characterize the inzymes by measuring the effect of (i) various known peptidase inhibitors (See Table 16) and (ii) lifTerent divalent ions (See Table 17). Each LAP was capable of cleaving Leu-AMC at 20°C and lad a temperature optimum ranging from 40 to 50°C. The optimum pH was between 7.0 and 8.5 See Table 18). A 10 min pre-treatment at 80°C totally and irreversibly inactivated the enzymes.<br>
The aminopeptidases tested were strongly or totally inhibited by amastatin (See Table 16) at concentration of 500 ^iM. RuLAPl, fuLAPl and pkLAP were also inhibited by bestatin, but this<br>
inhibitor had only partial inhibitory effect on both ruLAP2 and fuLAP2. Of the chelating agents tested, ortho-phenantroline totally inhibited the five enzymes at concentrations of 1 and 5 mM. FuLAPl, ruLAP2 and ruLAPl were more sensitive to EDTA than the other LAPs. E64 and p-chloromercuribenzoate (cysteine protease inhibitors) blunted the activity of ruLAP2 indicating the presence of critical thiol residues for activity on the amino acid sequence of this enzyme. Leupeptin (serine/cysteine protease inhibitor), PMSF (serine protease inhibitor), benzamidine, TLCK and TPCK had no clear inhibitory effects on all the LAPs tested. Surprisingly, fuLAPl and ruLAPl exhibited some sensitivity to 0.1 mM pepstatin (aspartic acid protease inhibitor).<br>
With the exception of fuLAPl, which exhibits a general sensitivity to divalent ions, Co** ions increased the activity of the LAPs from 200% to 900% at a concentration up to ImM. The four fungal LAPs showed variable sensitivities to divalent cations. For instance, fuLAP2 was activated by Mn"1"*" and Ca**, while fuLAPl was inhibited by the same ions. The microsomal pkLAP, highly activated by Zn, Ni and Cu** differs from the four fungal LAPs of the M28 family.<br>
The hydrolytic activity of the enzymes toward different aminoacyl-AMC was compared to Leu-AMC used as a reference (See Table IS). Following the aminopeptidase tested, various preferences for the different aminoacyl residue were detected. For example, the aminopeptidase pkLAP differs from the four fungal LAPs by an extremely high efficiency towards Ala-AMC and Arg-AMC. ruLAPl was clearly the most selective for Leu-AMC. However, some other preferential cleavage activities were observed with ruLAP2, fuLAPl and fiiLAP2. For instance Ser- and Pro-AMC were more efficiently cleaved by ruLAP2, whereas fuLAPl appreciated Arg-, Val-, and Phe-AMC. Only ruLAP2 efficiently cleaved Asp- and Glu-AMC. None of these enzymes exhibited an aminopeptidase P activity since they were not able to cleave Lys(Abz)-Pro-Pro-pNA.<br>
Example 8; Application of ruLAP2 together with ruDPPIV in the digestion of eltodin peptides<br>
Celiac disease (CD) is a digestive disease that damages the small intestine and interferes with absorption of nutrients from food. People who have celiac disease cannot tolerate a protein called gluten, which is found in wheat, rye and barley. When people with celiac disease eat foods containing gluten, their immune system responds by damaging the small intestine. The disease has a prevalence of » 1:200 in most of the world's population groups and the only treatment for celiac disease is to maintain a life-long, strictly gluten-free diet. For most people, following this diet will stop symptoms, heal existing intestinal damage, and prevent further damage.<br>
The principal toxic components of wheat gluten are a family of Pro-and Gin-rich proteins called gliadins, which are resistant to degradation in the gastrointestinal tract and contain several T-<br>
cellstimulatory epitopes. There is some controversy about the epitopes that effectively induce an immunological activation of HLA-DQ2 positive gut-derived and peripheral T cells (Vader etal., Gastroenterology 122:1729-1737 (2002)) because different in vitro systems have been used for these studies. The capacity of gliadin peptides to induce toxicity in an organ culture model of CD does not correspond to that of stimulating T-cells and vice versa. McAdam &amp; Sollid, Gut 47: 743-745 (2000). Moreover, the binding of many gluten epitopes to HLA-DQ2 and HLA-DQ8 but not all is enhanced by deamidation of certain glutamine residues into glutamic acids through the action of the small intestinal enzyme tissue transglutaminase, which potentiates their ability to stimulate T-cells. Molberg et al., Nat. Med. 4:713-717 (1998). However, deamidation is not an absolute requirement for T-cell activation. Arentz-Hansen et al., Gastroenterology 123:803-809 (2002).<br>
Other strategies for treating or preventing CD, with the ultimate hope being an alternative for the "gluten free" diet, have been suggested over the last years, including inhibition of T-cellactivation by compounds that block peptide binding to HLA-DQ2, inhibitors of tissue transglutaminase that prevent gluten deamidation (Sollid, Nat. Rev. Immunol. 2:647-655 (2002)) and peroral peptidase supplementation. This latter approach is considered to aid complete digestion of immunostimulatory peptides by involvement of bacterial prolyl endopeptidases which have broad tolerance for proline-containing peptides. Shan et al., Science 297:2275-2279 (2002); Hausch et al, Am. J. Physiol. Gastrointest Liver Physiol. 283:G996-G1003 (2002). A relatively large fragment of gliadin that is resistant to digestive enzymes degradation was identified. Furthermore, this peptide was shown to be a potent stimulator of different HLA-DQ2-restricted T cell clones derived from intestinal biopsies of CD patients stimulated with gluten, each of these clones recognizing a different epitope of the 33mer. The prolyl endopeptidase, which has a preference for Pro-Xaa-Pro motif, is able to cleave the 33mer gliadin peptide and the synergistic effect of brush border aminopeptidase rapidly decreases the T-cell stimulatory potential of the peptide.<br>
Though there are stable homologs to this 33mer in barley and rye, these gluten peptide motifs that are described as resistant to gastrointestinal degradation were used in our case as model substrates for different LAPs, either alone or in combination with ruDPPIV: PQPQLPYPQPQLPY [SEQ ID NO: 42)(14mer) corresponding to fragment 82-95 of ct/fJ gliadin AIV (P04724) or<br>
:QLQPFPQPQLPYPQPQLPYPQPQLPYPQPQPF (SEQ ID NO: 43) (33mer) corresponding to<br>
ragment 57-89 of gliadin MM1 (P 18573).<br>
A N-terminal acetylated form of the 33mer (Ac-33rher) was also synthesized as control for the digestion experiments with exopeptidases to preclude any endoproteolytic cleavage by a contaminant enzyme.<br>
The enzymes that have been evaluated include: ruLAPl (aminopeptidase I ofTrichophyton rubrwri), ruLAP2 (aminopeptidase II of Trichophyton rubrutn), orLAP2 (aminopeptidase II of Aspergillus orizae), fuLAP2 (aminopeptidase II of Aspergittus jumigatus), MicpKLAP (microsomal leucine aminopeptidase from porcine kidney, Sigma), CytpKLAP (cytosolic leucine aminopeptidase from porcine kidney, Sigma), and ruDPPIV.<br>
Synthesis of the pep tides:<br>
Solid-phase synthesis was performed on a custom-modified 430A peptide synthesizer from Applied Biosystems, using in situ neutralization/2-(IH-benzotriazol-l-yl)-l,1,1,3,3-tetramethyluronium hexa fluoro-phosphate (HBTU) activation protocols for stepwise Boc chemistry chain elongation on a standard -O-CH2-phenylacetamidomethyl resin. SchnOlzer et aL, Int. J. Peptide Protein Res. 40:180-193 (1992).<br>
At the end of the synthesis, the peptides were deprotected and cleaved from the resin by treatment with anhydrous HF for Ihr at 0°C with 5% p-cresol as a scavenger. After cleavage, the peptides were precipitated with ice-cold diethylether, dissolved in aqueous acetonitrile and lyophilized. The peptides were purified by RP-HPLC with a Qg column from Waters by using linear gradients of buffer B (90% acetonitile/10% H2O/0.1% trifluoroacetic acid) in buffer A (HfeO/0.1% trifluoroacetic acid) and UV detection at 214nm. Samples were analyzed by electrospray mass spectrometry with a Platform II instrument (Micromass, Manchester, England).<br>
Conditions of degradation reaction:<br>
Incubation was carried out at 37°C in 50mM Tris-HCl, pH7.2  supplemented withlmM CoCl2with a substrate concentration of Img/mL and an E/S ratio of 1:20. The reaction was stopped by acidification with CH3COOH and the medium analysed by RP-HPLC on a Cg column using a 2%/min CH3CN gradient in 0.1% TFA. All peaks were characterized by ESI-MS.<br>
Digestion of the 14mer:<br>
As shown in Figure 6, the 14mer is not digested with ruLAP2 within 4h. There is no change in the HPLC profile when compared with the control. In fact, digestion results only in the cleavage of the N-terminal Proline. On the other hand, supplementation with ruDPPIV results in a complete<br>
breakdown in amino acids and dipeptides, while ruDPPFV alone is not able to hydrolyse the peptide (Figure 7).<br>
Digestion of the 33mer:<br>
Digestion of the 33mer with ruLAP2 alone results in partial degradation (less than 50%) of the peptide within 4h (data not shown). This peptide is not a substrate for ruDPPIV (Figure 8). However, when both enzymes are mixed, the 33mer is totally digested (Figure 9) into amino acids and dipeptides some of which could be identified by ESI-MS (Y, L, F, P, PY, and PF).<br>
The same HPLC pattern is obtained when ruDPPIV is mixed with ruLAP2 or fuLAP2. However, with ruLAPl some higher molecular weight compounds are still present, but represent less than 10% of the initial substrate.<br>
On the other hand, incubation with microsomal porcine kidney aminopeptidase results only in a partial deletion of N-terminal Leu and C-terminal Phe (due to a carboxypeptidasic contaminant) and addition of DPPIV does not modify the profile. Cytosolic porcine kidney aminopeptidase is totally inactive towards the 33mer.<br>
The stability of the Ac-gliadin 33mer in the digestion experiments with either LAP or DPPIV alone, or mixed together, confirms that a free amino group is required for the complete breakdown of the gliadin 33mer by these exopeptidases.<br>
Digestion with other enzymes:<br>
Digestion with Pronase (E/S=l/25) over 20h is only partial (less than 40%) and the addition of ruLAP2 (both enzymes at an E/S rartio (w:w) of 1:50) does not improve the hydrolysis. On the other hand, addition of DPPIV under the same conditions results in a complete breakdown of the peptide due to the complementary action of an aminopeptidase and dipeptidylpeptidase. Chymotrypsin alone or supplemented with ruLAP or DPPIV is not able to breakdown the peptide.<br>
Example 9; Application of ruLAPZ in the processing of expressed recombinant proteins fused with another protein or with a N-terminal Tag<br>
LAP2 was evaluated in the cleavage of the Gly-Ser from the N-terminus of proNPY and of a supplementary Ala from the N-terminus of the same peptide. In order to widen the applicability of LAP2 either alone or in conjunction with another exopeptidase in the processing of larger recombinant proteins, a G-CSF recombinant protein (Cys17-*Ser, Lys16123"34'40 —»Arg) with an N-terminal sequence Met-Thr-Pro-, was successively incubated with ruLAP2 and ruDPPJV to remove sequentially Met and Thr-Pro dipeptide from the 175 residue protein.<br>
Digestion of Gly-Ser-proNPY with ruLAP2:<br>
The peptide was incubated overnight at 37°C and Img/ml in a 50mM Tris.HCl, ImM CoCl2 buffer with ruLAP2 at an E/S ratio of 1:20 and 1:100 (w:w). The digested material was isolated by RP-HPLC and characterized by ESI-MS. As shown in Figure 10, incubation with ruLAP2 results in the cleavage of the two N-terminal residues Gly-Ser with a theoretical loss of 144.1 amu (found 144.2). The same result is obtained at an 1:100 E/S ratio. Digestion halts when the enzyme reaches a Xaa-Pro-motif, which in case of proNPY is Tyr-Pro.<br>
Digestion of Ala-proNPY with ruLAP2:<br>
Conditions of incubation were the same as for Gly-Ser-proNPY. Figure 1 IB shows that the N-terminal alanine was almost totally removed (molecular mass loss of 71 amu) from proNPY.<br>
Successive cleavage of Met and Thr-Pro from the N-terminus of G-CSF:<br>
The mutant analogue of G-CSF known as TG47 used in these experiments is methionyl-[C17S, K16,23,34,40R] G-CSF with a theoretical mass of 18,894.90 for the refolded protein.<br>
Digestion with ruLAP2:<br>
Stock solution of G-CSF (1.9mg/ml in PBS containing 0.1% Sarcosyl) was diluted 4 times in 50mM Tris-HCl at pH7.2 supplemented with ImM CoCl2, and incubated with ruLAP2 (E/S=1/20 and 1:100, w:w) for 15h at 37°C . The solution was diluted with 30% (v:v) acetonitrile, acidified with acetic acid and the protein isolated by RP-HPLC for MS characterization. As shown in Figure 12A and B, the overnight incubation results in the complete cleavage of the N-terminal methionine with a theoretical mass loss of 131.2 amu. With an E/S ratio (w: w) of 1:100, traces of uncleaved material are still present after an overnight incubation.<br>
This experiment was repeated at a 2mg scale in order to isolate the truncated material on a semi-preparative RP-HPLC column, by carrying out the digestion with a E/S ratio of 1:25 (w:w) at 37°C over 15h. The isolated material (O.Smg) was characterized by ESI-MS (Figure 12B, desMet-G-CSF, calculated molecular mass at 18,763.7 amu; measured molecular mass at 18,762.5).<br>
Digestion of desMet-G-CSF with DPPIV:<br>
The freeze-dried material was suspended at a Img/ml concentration in 50mM Tris-HCl, pH 7.5 containing 0.1% Sarcosyl and incubated overnight at 37°C with DPPIV at an E/S ratio of 1/20<br>
(w:w). The protein was isolated by RP-HPLC as before and characterized by ESI-MS (Figures 13A and B). DPPIV digestion (Figure 13B) results in the cleavage of the N-terminal dipeptide Thr-Pro (calculated molecular mass of 18,564.8 uma; measured molecular mass at 18,563). Traces of undigested material are still present in the reaction medium.<br>
Thus, a sequential application of LAP2 and DPPIV results in the efficient removal of an N-terminal sequence from a recombinant protein. Digestion with ruLAP2 is halted when the enzyme reaches a "stop point" amino acid motif, such as Xaa-Pro-Xaa, or the Xaa-Pro motif, which may be specifically introduced as a LAP2 "stop point", is subsequently cleaved with DPPIV.<br>
However, initial cleavage of the N-terminal residues is highly dependent on the sequence since the Met(His)6 tag was not removed from Met(His)e -proNPY by incubating with LAP and DPPIV.<br>
OTHER EMBODIMENTS<br>
Although particular embodiments have been disclosed herein in detail, this has been done by way of example for purposes of illustration only, and is not intended to be limiting with respect to the scope of the appended claims, which follow. In particular, it is contemplated by the inventors that various substitutions, alterations, and modifications may be made to the invention without departing from the spirit and scope of the invention as defined by the claims. The choice of nucleic acid starting material, clone of interest, or library type is believed to be a matter of routine for a person of ordinary skill in the art with knowledge of the embodiments described herein. Other aspects, advantages, and modifications are considered to be within the scope of the following claims.<br><br><br><br><br>
What is claimed is:<br>
1.	An enzyme cocktail containing a carrier and an isolated aminopeptidase<br>
originating from dermatophytes, comprising the mature form of an amino acid<br>
sequence selected from the group consisting of the combination of:<br>
a) a leucine aminopeptidase (LAP) of SEQ ID NOs: 3 and/or 6, a biologically active fragment thereof, mixtures thereof and/or naturally occurring allelic variants thereof and/or sequences having at least 90% of identity, b) with a dipeptidyl peptidase IV (DPP IV) of SEQ ID NO: 35, a biologically active fragment thereof, and/or naturally occurring allelic variants thereof and/or sequences having at least 90% of identity.<br>
2.	The enzyme cocktail of claim 1, characterized in that the dermatophyte is<br>
selected from the group consisting of Epidermophyton floccosum, Microsporum<br>
audouinii, Microsporum ferrugineum, Trichophyton concentricum, Trichophyton kanei,<br>
Tiichophyton megninii, Trichophyton mentagrophytes, Trichophyton raubitschekii,<br>
Trichophyton rubrum, Trichophyton schoenleinii, Trichophyton soudanense,<br>
Trichophyton tonsurans, Trichophyton violaceum, Trichophyton yaoundei,<br>
Microsporum cam's, Microsporum equinum, Microsporum nanum, Microsporum<br>
persicolor, Trichophyton equinum, Trichophyton simii, Trichophyton verrucosum,<br>
Microsporum gypseum, Trichophyton ajelloi, and Trichophyton terrestre.<br>
3.	The enzyme cocktail of any of claims 1 or 2, further comprising one or more<br>
proteases.<br>
4.	The enzyme cocktail of claim 3, wherein said one or more proteases are<br>
selected from the group consisting of trypsin, pronase, chymotrypsin, and<br>
proteinaseK.<br>
5.	The enzyme cocktail of any of claims 1-4, characterized in that the leucine<br>
aminopeptidase (LAP) is ruLAP2.<br>
6.	The enzyme cocktail of any of claims 1-5, for use in the degradation, hydrolysis<br>
or maturation of polypeptide substrates or proteins.<br>
7.	A kit comprising, in one or more containers, the enzyme cocktail of any of<br>
claims 1-5.<br>
8.	Use of the enzyme cocktail of any of claims 1-5, in the hydrolysis of proteins, for<br>
the degradation of by-products, toxic or contaminant proteins; for the degradation of<br>
prions or viruses; for the degradation of proteins for proteomics; for the degradation of<br>
cornified substrate; for the hydrolysis of polypeptides for amino acid analysis; for<br>
wound cleaning; for wound healing; for cosmetology such as peeling tools, depilation,<br>
dermabrasion and dermaplaning; for prothesis cleaning and/or preparation; for fabric<br>
softeners; for soaps; for cleaning or disinfection of septic tanks or any container<br>
containing proteins that should be removed or sterilized; and for cleaning of surgical<br>
instruments.<br>
9.	Use of the enzyme cocktail of any of claims 1 -5, in the manufacture of a<br>
medicament for treating a syndrome associated with a human disease, said disease<br>
being selected from the group consisting of celiac disease, digestive tract<br>
malabsorption, an allergic reaction, an enzyme deficiency, a fungal infection, mycoses<br>
and sprue.<br>
10.	The use of the enzyme cocktail according to claim 9, characterized in that the<br>
allergic reaction is a reaction to gluten or fragments thereof.<br>
11.	A method of degrading a polypeptide substrate, the method comprising bringing<br>
the polypeptide substrate into contact with the enzyme cocktail of any of claims 1-5.<br>
12.	The method of claim 11, characterized in that said enzyme cocktail sequentially<br>
digests a full-length polypeptide substrate or a full-length protein.<br>
13.	The method of any of claims 11 or 12, characterized in that the polypeptide<br>
substrate is selected from the group consisting of casein, gliadin, gluten, bovine serum<br>
albumin or fragments thereof.<br>
14.	The method of any of claims 11 to 13, characterized in that the polypeptide<br>
substrate length is from 2 to 200 amino acids.<br>
15.	A method of removing amino acids from the amino terminus of a protein,<br>
'"omprising bringing said protein into contact with the enzyme cocktail of any of claims<br>
5.<br>
16.	The method of claim 15, characterized in that the amino terminus of said protein<br>
comprises a His tag.<br>
17.	A composition adapted for the removal of amino acids from the amino terminus<br>
of a protein, comprising bringing said protein into contact with a dipeptidyl peptidase<br>
IV (DPP IV) of SEQ ID NO: 35, a biologically active fragment thereof, and/or naturally<br>
occurring1 allelic variants thereof and/or sequences having at least 90% of identity.<br>
18.	The composition of claim 17, characterized in that the amino terminus of said<br>
protein comprises a Xaa-Pro tag.<br>
19.	The composition of claim 18, characterized in that Xaa is an amino acid<br>
comprising at least two vicinal nucleophilic groups.<br>
20.	The composition of claim 19, characterized in that said amino acid is serine,<br>
threonine or cysteine.<br><br>
21.	A method for recovering a protein in its mature form without the di-peptide tag,<br>
comprising the steps of:<br>
(1)	purification by covalent capture and,<br>
(2)	removal of the di-peptide tag with the composition of any of claims 17-20.<br>
22. An enzyme cocktail, use of enzyme cocktail, a method of degrading a polypeptide substrate, a composition adapted for the removal of the amino acids from the amino terminus of a protein, a method of recovering a protein in its mature form without the di-peptide tag such as herein described with reference to the accompanying drawings and as illustrated in the foregoing examples.<br>
22.	An  enzyme  cocktail,   use of enzyme  cocktail,  a  method  of degrading  a<br>
polypeptide substrate, a composition adapted for the removal of the amino acids<br>
from the amino terminus of a protein, a method of recovering a protein in its<br>
mature form without the di-peptide tag such as herein described with reference to<br>
the accompanying drawings and as illustrated in the foregoing examples<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwNS1kZWwtMjAxMi1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE1LTExLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1505-del-2012-Correspondence Others-(15-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwNS1kZWwtMjAxMi1Gb3JtLTMtKDE1LTExLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1505-del-2012-Form-3-(15-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LS1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE1LTExLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1618-delnp-2006--Correspondence Others-(15-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LUFic3RyYWN0LSgxMi0wNi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Abstract-(12-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LUNsYWltcy0oMTItMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Claims-(12-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Correspondence Others-(28-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-DELNP-2006-Correspondence-Others-(11-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Correspondence-Others-(12-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTctMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-DELNP-2006-Correspondence-Others-(17-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LURyYXdpbmdzLSgxMi0wNi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Drawings-(12-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LUZvcm0tMS0oMjgtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Form-1-(28-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LUZvcm0tMi0oMTItMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-Form-2-(12-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWZyb20tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-from-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWZyb20tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-from-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWZyb20tMjYucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-from-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWZyb20tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-from-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LWZyb20tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-from-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1ERUxOUC0yMDA2LUdQQS0oMTEtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-DELNP-2006-GPA-(11-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1ERUxOUC0yMDA2LUdQQS0oMTctMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1618-DELNP-2006-GPA-(17-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxOC1kZWxucC0yMDA2LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1618-delnp-2006-pct-416.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258397-method-of-discriminating-at-least-two-cell-populations-of-and-application.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258399-process-of-drying-and-purifing-a-gas-flowing-to-a-catalyst-regeneration-vessel.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258398</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1618/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>FUNZYME BIOTECHNOLOGIES SA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHEMIN D&#x27;ALCIRE 1, CH-1228 PLAN-LES-QUATES, SWITZERLAND,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STOCKLIN, RETO</td>
											<td>ATHERIS LABORATORIES, DR. RETO STOCKLIN ET SYLVIE STOCKLIN ASSOCIES, 1 CH. D&#x27;ALCIRE, ARARE, CH-1228 PLAN-LES-QUATES GENEVA, SWITZERLAND,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MONOD, MICHEL</td>
											<td>18 AVENUE VIRGILE ROSSEL, CH-1012 LAUSANNE, SWITZERLAND,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>GROUZMANN, ERIC</td>
											<td>57 CHEMIN DU CREUX DE CORSY, CH-1093 LA CONVERSION, SWITZERLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IL2004/002963</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/498,318</td>
									<td>2003-08-25</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258398-an-isolated-and-purified-enzyme-cocktail by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:43:09 GMT -->
</html>
