<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258384-a-heterologous-exo-endo-cellulase-fusion-construct-and-fusion-protein-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:45:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258384:&quot;A HETEROLOGOUS EXO-ENDO CELLULASE FUSION CONSTRUCT AND FUSION PROTEIN THEREOF&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A HETEROLOGOUS EXO-ENDO CELLULASE FUSION CONSTRUCT AND FUSION PROTEIN THEREOF&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a heterologous exo-endo cellulase fusion construct, which encodes a fusion protein aving cellulolytic activity comprising a catalytic domain derived from a fungal exo-cellobiohydrolase and a catalytic domain derived om an endoglucanase. The invention also relates to vectors and fungal host cells comprising the heterologous exo-endo cellulase ision construct as well as methods for producing a cellulase fusion protein and enzymatic cellulase compositions.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>EXO-ENDO CELLULASE FUSION PROTEIN<br>
SPONSORED RESEARCH AND DEVELOPMENT<br>
Portions of this work were funded by Subcontract No. ZCO-0-30017-01 with the National Renewable Energy Laboratory under Prime Contract No. DE-AC36-99G010337 with the United States Department of Energy. Accordingly, the United States Government may have certain rights in the invention.<br>
RELATED APPLICATIONS<br>
The present application claims priority to U.S. Provisional Patent Application Serial No. 60/556,598, entitled "Exo-Endo Cellulase Fusion Protein" and filed on March 25, 2004.<br>
FIELD OF THE INVENTION<br>
The present invention relates to a heterologous exo-endo cellulase fusion construct, which encodes a fusion protein having cellulolytic activity comprising a catalytic domain derived from a fungal exo-cellobiohydrolase and a catalytic domain derived from an endoglucanase. The invention also relates to vectors and fungal host cells comprising the heterologous exo-endo cellulase fusion construct as well as methods for producing a cellulase fusion protein and enzymatic cellulase compositions.<br>
BACKGROUND OF THE INVENTION<br>
Cellulose and hemicellulose are the most abundant plant materials produced by photosynthesis. They can be degraded and used as an energy source by numerous microorganisms, including bacteria, yeast and fungi, which produce extracellular enzymes capable of hydrolysis of the polymeric substrates to monomeric sugars (Aro et al., 2001). As the limits of non-renewable resources approach, the potential of cellulose to become a major renewable energy resource is enormous (Krishna et al., 2001). The effective utilization of cellulose through biological processes is one approach to overcoming the shortage of foods, feeds, and fuels (Ohmiya et al., 1997).<br>
Cellulases are enzymes that hydrolyze cellulose (beta-1,4-glucan or beta D-glucosidic linkages) resulting in the formation of glucose, cellobiose, cellooligosaccharides, and the like. Cellulases have been traditionally divided into three major classes: endoglucanases (EC 3.2.1.4) ("EG"), exoglucanases or cellobiohydrolases (EC 3.2.1.91) ("CBH") and beta-glucosidases ([beta] -D-glucoside glucohydrolase; EC 3.2.1.21) ("BG") (Knowles et al., 1987 and Schulein, 1988). Endoglucanases act mainly on the amorphous parts of the cellulose fiber, whereas cellobiohydrolases are also able to degrade crystalline cellulose.<br>
Cellulases are known to be produced by a large number of bacteria, yeast and fungi. Certain fungi produce a complete cellulase system capable of degrading crystalline forms of cellulose, such that the cellulases are readily produced in large quantities via fermentation.<br>
In order to efficiently convert crystalline cellulose to glucose the complete cellulase system comprising components from each of the CBH, EG and BG classifications is required, with isolated components less effective in hydrolyzing crystalline cellulose (Filho et al., 1996). In particular, the combination of EG-type cellulases and CBH- type cellulases interact to more efficiently degrade cellulose than either enzyme used alone (Wood, 1985; Baker et al., 1994; and Nieves et al., 1995).<br>
Additionally, cellulases are known in the art to be useful in the treatment of textiles for the purposes of enhancing the cleaning ability of detergent compositions, for use as a softening agent, for improving the feel and appearance of cotton fabrics, and the like (Kumar et al., 1997). Cellulase-containing detergent compositions with improved cleaning performance (US Pat. No. 4,435,307; GB App. Nos. 2,095,275 and 2,094,826) and for use in the treatment of fabric to improve the feel and appearance of the textile (US Pat. Nos. 5,648,263, 5,691,178, and 5,776,757, and GB App. No. 1,358,599), have been described in the literature.<br>
Hence, cellulases produced in fungi and bacteria have received significant attention. In particular, fermentation of Trichoderma spp. (e.g., Trichoderma longibrachiatum or Trichoderma reesei) has been shown to produce a complete cellulase system capable of degrading crystalline forms of cellulose. Over the years, Trichoderma cellulase production has been improved by classical mutagenesis, screening, selection and development of highly refined, large scale inexpensive fermentation conditions. While the multi-component cellulase system of Trichoderma spp. is able to hydrolyze cellulose to glucose, there are cellulases from other microorganisms, particularly bacterial strains, with different properties<br>
for efficient cellulose hydrolysis, and it would be advantageous to express these proteins in a filamentous fungus for industrial scale cellulase production. However, the results of many studies demonstrate that the yield of bacterial enzymes from filamentous fungi is low (Jeeves ef al., 1991).<br>
In this invention, a heterologous exo-endo cellulase fusion construct, which includes the coding region of a fungal exo-cellobiohydrolase (CBH) catalytic domain and a coding region of an endoglucanase (EG) catalytic domain, has been introduced and expressed in a filamentous fungi host cell to increase the yield and effectiveness of cellulase enzymes.<br>
SUMMARY OF THE INVENTION<br>
In a first aspect, the invention includes a heterologous exo-endo cellulase fusion construct comprising in operable linkage from the 5' end of said construct, (a) a DNA molecule encoding a signal sequence, (b) a DNA molecule encoding a catalytic domain of an exo-cellobiohydrolase, and (c) a DNA molecule encoding an endoglucanase catalytic domain.<br>
In a first embodiment of this aspect, the heterologous exo-endo cellulase fusion construct further comprises a linker sequence located 3' of the catalytic domain of the exo-cellobiohydrolase and 5' of the catalytic domain of the endoglucanase. In a second embodiment, the heterologous exo-endo cellulase fusion construct lacks the cellulose binding domain (CBD) of the exo-cellobiohydrolase. In a third embodiment, the heterologous exo-endo cellulase fusion construct further comprises a kexin site located after the linker sequence and before the coding region of the endoglucanase catalytic domain. In a fourth embodiment, the heterologous exo-endo fusion construct will comprise a promoter of a filamentous fungus secretable protein, said promoter located in operable linkage 5' of the coding region of the exo-cellobiohydrolase catalytic domain. In a fifth embodiment, the promoter is a cbh promoter and preferably a cbhl promoter derived from 7. reesei. In a sixth embodiment, the exo-cellobiohydrolase is a CBH1 and particularly a CBH1 having an amino acid sequence of at least 90% sequence identity with the sequence set forth in SEQ ID NO.: 6. In a seventh embodiment, the endoglucanase catalytic domain is derived from a bacterial endoglucanase. In an eighth embodiment, the bacterial endoglucanase catalytic domain is selected from the group consisting of an Acidothermus cellulolyticus GH5A endoglucanase I (E1) catalytic domain; an Acidothermus cellulolyticus GH74 endoglucanase (GH74-EG) catalytic domain: and a Thermobifida fusca E5 endoglucanase (Tf-E5) catalytic domain. In a ninth embodiment, the heterologous exo-<br>
endo cellulase fusion construct lacks the cellulose binding domain of the endoglucanase. In a tenth embodiment, the endoglucanase is an Acidothermus cellulolyticus GH5A E1 and particularly the Acidothermus cellulolyticus GH5A E1 having an amino acid sequence of at least 90% sequence identity with the sequence set forth in SEQ ID NO. 8. In an eleventh embodiment, the heterologous exo-endo cellulase fusion construct comprises a terminator sequence located 3' to the endoglucanase catalytic domain. In a twelfth embodiment, the heterologous fusion construct comprises a selectable marker.<br>
In a second aspect, the invention includes a vector comprising in operable linkage a promoter of a filamentous fungus secretable protein, a DNA molecule encoding a signai sequence, a DNA molecule encoding a catalytic domain of a fungal exo-cellobiohydrolase, a DNA molecule encoding a catalytic domain of an endoglucanase, and a terminator. In one embodiment, the vector will further include a selectable marker. In a second embodiment, the vector will comprise a linker located 3' of the exo-cellobiohydrolase (CBH) catalytic domain and 5' of the EG catalytic domain. In a third embodiment, the vector will lack the CBH cellulose binding domain. In a fourth embodiment, the vector will comprise a kexin site. In a fifth embodiment the catalytic domain of the endoglucanase is derived from a bacterial endoglucanase. In a sixth embodiment, the vector lacks the cellulose binding domain of the endoglucanase.<br>
In a third aspect, the invention includes a fungal host cell transformed with a heterologous exo-endo cellulase fusion construct or a fungal host cell transformed with a vector comprising a heterologous exo-endo cellulase fusion construct.<br>
In a fourth aspect, the invention includes a recombinant fungal cell comprising the heterologous exo-end cellulase fusion construct or a vector comprising the same.<br>
In a particularly preferred embodiment of the third and fourth aspects, the fungal host cell is a Trichoderma host cell and more particularly a strain of T. reesei. In another embodiment of these aspects, native cellulase genes, such as cbhl, cbh2, eg/7 and eg/2 have been deleted from the fungal cells. In a third embodiment, the native cellulose binding domain has been deleted from the fungal cells.<br>
In a fifth aspect, the invention includes an isolated cellulase fusion protein having cellulolytic activity which comprises an exo-cellobiohydrolase catalytic domain and an endoglucanase catalytic domain, wherein the exo-cellobiohydrolase lacks a cellulose binding domain. In one embodiment of this aspect, the exo-cellobiohydrolase is a CBHL In a second embodiment, the catalytic domain of the endoglucanase is derived from a bacterial cell. In a third embodiment, the bacterial cell is a strain of Acidothermus<br>
cellulolyticus. In a fourth embodiment, the invention concerns a cellulolytic composition comprising the isolated cellulase fusion protein.<br>
In a sixth aspect, the invention includes a method of producing an enzyme having cellulolytic activity comprising, a) stably transforming a filamentous fungal host cell with a heterologous exo-endo cellulase fusion construct or vector as defined above in the first aspect and second aspect; b) cultivating the transformed fungal host cell under conditions suitable for said fungal host cell to produce an enzyme having cellulolytic activity; and c) recovering said enzyme.<br>
In one embodiment of this aspect, the filamentous fungal host cell is a Trichoderma cell, and particularly a T. reesei host cell. In a second embodiment, the exo-cellobiohydrolase is a CBH1 and the endoglucanase is an Acidothermus cellulolyticus endoglucanase or a Thermobifida fusca endoglucanase. In a third embodiment, the recovered enzyme is a cellulase fusion protein, components of the cellulase fusion protein, or a combination of the cellulase fusion protein and the components thereof. In a fourth embodiment, the recovered enzyme(s) is purified.<br>
In an seventh aspect, the invention includes a Trichoderma hosi cell which expresses a cellulase fusion protein, wherein said fusion protein comprises a catalytic domain of an exo-cellobiohydrolase and a catalytic domain of an endoglucanase, wherein the exo-cellobiohydrolase lacks a cellulose binding domain. In one embodiment, the Trichoderma host cell is a T. reesei cell. In a second embodiment, the exo-cellobiohydrolase is a CBH1 and the endoglucanase is an Acidothermus cellulolyticus endoglucanase and particularly an Acidothermus cellulolyticus E1 or GH74 endoglucanase. In a third embodiment, the endoglucanase lacks a cellulose binding domain. In a fourth embodiment, the T. reesei host cell includes deleted native cellulase genes.<br>
In an eighth aspect, the invention includes a fungal cellulase composition comprising a cellulase fusion protein or components thereof, wherein the fusion protein or components thereof is the product of a recombinant Trichoderma spp. In one embodiment, the cellulase fusion protein is a CBW\-Acidothermus cellulolyticus E1 fusion protein and the components are the cleaved products, CBH1 and Acidothermus cellulolyticus E1, wherein each component has cellulolytic activity.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 is a representation of a heterologous exo-endo cellulase fusion construct encompassed by the invention, which includes a Trichoderma reesei cbhl promoter, a cbhl<br>
core (cbhl signal sequence and cbhl catalytic domain), a cbhl linker sequence, a kexin site, an E1 core (an Acidothermus cellulolyticus E1 endoglucanase catalytic domain), a cbM terminator and an A. nidulans amdS selectable marker.<br>
Figure 2 is a DNA sequence (SEQ ID NO: 1) of the T. reeseicbhl signal sequence (SEQ ID NO: 2); the T. reesei cbhl catalytic domain (SEQ ID NO: 3), and the T. reesei cbhl linker (SEQ ID NO: 4). The signal sequence is underlined, the catalytic domain is in bold, and the linker sequence is in italics.<br>
Figure 3 shows the predicted amino acid sequence (SEQ ID NO: 5) based on the nucleotide sequence provided in Figure 2, wherein the signal peptide is underlined, the catalytic domain, represented by (SEQ ID NO: 6), is in bold, and the linker is in italics.<br>
Figure 4 is an illustration of a nucleotide sequence (SEQ ID NO: 7) encoding an Acidothermus cellulolyticus GH5A endoglucanase I (E1) catalytic domain.    ,<br>
Figure 5 is the predicted amino acid sequence (SEQ ID NO: 8) of the Acidothermus cellulolyticus GH5A E1 catalytic domain based on the nucleotide sequence provided in Figure 4.<br>
Figures 6A asd 6B are an illustration of a nucleotide sequence (SEQ ID NO: 9) encoding an Acidothermus cellulolyticus GH74- EG catalytic domain.<br>
Figure 7 is the predicted amino acid sequence (SEQ ID NO: 10) of the Acidothermus cellulolyticus GH74-EG based on the nucleotide sequence provided in Figures 6A and 6B.<br>
Figure 8 is an illustration of a nucleotide sequence (SEQ ID NO: 11) encoding the CBD, linker and catalytic domain of endoglucanase 5 (E5) of Thermobifida fusca.<br>
Figure 9 is the predicted amino acid sequence (SEQ ID NO: 12) of the CBD, linker and E5 based on the nucleotide sequence provided in Figure 8. Figure 10 is the nucleotide sequence(2656 bases) (SEQ ID NO: 13) of a heterologous cellulase fusion construct described in example 1 comprising, the T. reese/CBH1 signal sequence; the catalytic domain of the T. reesei CBH1; the T. reesei CBH1 linker sequence; a kexin cleavage site which includes codons for the amino acids SKR and the sequence coding for the Acidothermus cellulolyticus GH5A-E1 catalytic domain.<br>
Figure 11 is the predicted amino acid sequence (SEQ ID NO: 14) of the cellulase fusion protein based on the nucleic acid sequence in Figure 10.<br>
Figure 12 provides a schematic diagram of the pTrex4 plasmid, which was used for expression of a heterologous exo-endo cellulase fusion construct (CBH1-endogiucanase) as described in the examples and includes the Trichoderma reesei cbhl promoter, the T.<br>
reese/CBH1 signal sequence, catalytic domain, and linker sequences, a kexin cleavage site and an endoglucanase gene of interest inserted between a Spel and AscI site, a CBH1 Trichoderma reesei terminator and the amdS Aspergillus nidulans acetamidase marker gene.<br>
Figures 13A - E provide the nucleotide sequence (SEQ ID NO:15) (10239 bp) of the pTrex4 plasmid of Figure 12 without the catalytic domain of the EG gene of interest.<br>
Figure 14 illustrates a SDS-PAGE gel of supernate samples of shake flask growth of clones of a T. reesei strain deleted for the cellulases, cbhl, cbh2, egH and eg/2 and transformed with the CBH1-E1 fusion construct. Lanes 1 and 10 represent MARK 12 Protein Standard (Invitrogen, Carlsbad, CA). Lanes 2-8 represent various transformants and lane 9 represents the untransformed T. reesei strain. The upper arrow indicates the cellulase fusion protein and the lower arrow indicates the cleaved E1 catalytic domain.<br>
Figure 15 illustrates a SDS-PAGE gel of supernate samples of shake flask growth of clones of a T. reesei strain deleted for the cellulases, cbhl, cbh2, egH and eg/2 and transformed with the CBH1-GH74 fusion construct. Lane 1 represents the untransformed control. Lane 3 represents MARK 12 Protein Standard (Invitrogen, Carlsbad, CA). Lanes 2 and 4-12 represent various transformants. The upper arrow indicates the CBH1-GH74 fusion protein and the lower arrow indicates the cleaved GH74 catalytic domain.<br>
Figure 16 illustrates a SDS-PAGE gel of supernate samples of shake flask growth of clones of a 7". reesei strain deleted for the cellulases, cbhl, cbh2, egH and eg/2 and transformed with the CBH1-TfE5 fusion construct. Lanel represents MARK 12 Protein Standard (Invitrogen, Carlsbad, CA). Lane 2 represents the untransformed strain and lanes 3-12 represent various transformants. Arrows indicate new bands observed in the CBH1-TfE5 expressing transformants.<br>
Figure 17 illustrates the % cellulose conversion to soluble sugars over time for a T. reesei parent strain comprising native cellulase genes with a corresponding T. reesei strain which expresses the CBH1-E1 fusion protein and reference is made to example 3.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention will now be described in detail by way of reference only using the following definitions and examples. All patents and publications, including all sequences disclosed within such patents and publications, referred to herein are expressly incorporated by reference.<br>
Unless defined otherwise herein, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Singleton, etal., DICTIONARY OF MICROBIOLOGY AND MOLECULAR BIOLOGY, 2o ED., John Wiley and Sons, New York (1994), and Hale &amp; Marham, THE HARPERCOLLINS DICTIONARY OF BIOLOGY, Harper Perennial, NY (1991) provide one of skill with a general dictionary of many of the terms used in this invention. Practitioners are particularly directed to Sambrook et al., MOLECULAR CLONING: A LABORATORY MANUAL (Second and Third Editions (1989 and 2001), Cold Spring Harbor Press, Plainview, N.Y., and Ausubel FM et al., Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York, NY, 1993, for definitions and terms of the art.<br>
It is to be understood that this invention is not limited to the particular methodology, protocols, and reagents described, as these may vary. Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, the preferred methods and materials are described. Numeric ranges are inclusive of the numbers defining the range. Unless otherwise indicated, nucleic acids are written left to right in 5' to 3' orientation; amino acid sequences are written left to right in amino to carboxy orientation, respectively.<br>
Other objects, features and advantages of the present invention will become apparent from the following detailed description. It should be understood, however, that the detailed description and specific examples, while indicating preferred embodiments of the invention, are given by way of illustration only, since various changes and modifications within the scope and spirit of the invention will become apparent to one skilled in the art from this detailed description.<br>
1. DEFINITIONS<br>
The term "heterologous exo-endo cellulase fusion construct" refers to a nucleic acid construct that is composed of parts of different genes in operable linkage. The components include, from the 5' end, a DNA molecule encoding an exo-cellobiohydrolase catalytic domain and a DNA molecule encoding an endoglucanase catalytic domain.<br>
The term "cellulase fusion protein" or "fusion protein having cellulolytic activity" refers to an enzyme, which has an exo-cellobiohydrolase catalytic domain and an endoglucanase catalytic domain and exhibits cellulolytic activity.<br>
The term "components of a cellulase fusion protein" refers to individual (cleaved) fragments of the cellulase fusion protein, wherein each fragment has cellulolytic activity and includes one of the catalytic domains of the fusion protein.<br>
The term "cellulase" refers to a category of enzymes capable of hydrolyzing cellulose (beta-1,4-glucan or beta D-glucosidic linkages) polymers to shorter cello-oligosaccharide oligomers, cellobiose and/or glucose.<br>
The term "exo-cellobiohydrolase" (CBH) refers to a group of cellulase enzymes classified as EC 3.2.1.91. These enzymes are also known as exoglucanases or cellobiohydrolases. CBH enzymes hydrolyze cellobiose from the reducing or non-reducing end of cellulose. In general, a CBH1 type enzyme preferentially hydrolyzes cellobiose from the reducing end of cellulose and a CBH2 type enzyme preferentially hydrolyzes the non-reducing end of cellulose.<br>
The term "endoglucanase" (EG) refers to a group of cellulase enzymes classified as EC 3.2.1.4. An EG enzyme hydrolyzes internal beta-1,4 glucosidic bonds of the cellulose.<br>
The term "beta-glucosidases" refers to a group of cellulase enzymes classified as EC 3.2.1.21.<br>
"Cellulolytic activity" encompasses exoglucanase activity, endoglucanase activity or both types of enzymatic activity.<br>
The term "catalytic domain" refers to a structural portion or region of the amino acid sequence of a cellulase which possess the catalytic activity of the cellulase. The catalytic domain is a structural element of the cellulase tertiary structure that is distinct from the cellulose binding domain or site, which is a structural element which binds the cellulase to a substrate, such as cellulose.<br>
The term "cellulose binding domain (CBD)" as used herein refers to a portion of the amino acid sequence of a cellulase or a region of the enzyme that is involved in the cellulose binding activity of a cellulase. Cellulose binding domains generally function by non-covalently binding the cellulase to cellulose, a cellulose derivative or other polysaccharide equivalent thereof. CBDs typically function independent of the catalytic domain.<br>
A nucleic acid is "operably linked" when it is placed into a functional relationship with another nucleic acid sequence. For example, DNA encoding a signal peptide is operably linked to DNA encoding a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide; a promoter is operably linked to a coding sequence if it affects the transcription of the sequence. Generally, "operably linked" means<br>
that the DNA sequences being linked are contiguous, and, in the case of the heterologous exo-endo cellulase fusion construct contiguous and in reading frame.<br>
As used herein, the term "gene" means the segment of DNA involved in producing a polypeptide chain, that may or may not include regions preceding and following the coding region, e.g. 5' untranslated (51 UTR) or "leader" sequences and 3' UTR or "trailer" sequences, as well as intervening sequences (introns) between individual coding segments (exons).<br>
The term "polypeptide" as used herein refers to a compound made up of a single chain of amino acid residues linked by peptide bonds. The term "protein" as used herein may be synonymous with the term "polypeptide" or may refer, in addition, to a complex of two or more polypeptides.<br>
The term "nucleic acid molecule", "nucleic acid" or "polynucleotide" includes RNA, DNA and cDNA molecules. It will be understood that, as a result of the degeneracy of the genetic code, a multitude of nucleotide sequences encoding a given protein such as a cellulase fusion protein of the invention may be produced.<br>
A "heterologous" nucleic acid sequence has a portion of the sequence, which is not native to the cell in which it is expressed. For example, heterologous, with respect to a control sequence refers to a control sequence (i.e. promoter or enhancer) that does not function in nature to regulate the same gene the expression of which it is currently regulating. Generally, heterologous nucleic acid sequences are not endogenous to the cell or part of the genome in which they are present, and have been added to the cell, by infection, transfection, transformation, microinjection, electroporation, or the like. A "heterologous" nucleic acid sequence may contain a control sequence/DNA coding sequence combination that is the same as, or different from a control sequence/DNA coding sequence combination found in the native cell. The term heterologous nucleic acid sequence encompasses a heterologous exo-endo cellulase fusion construct according to the invention.<br>
As used herein, the term "vector" refers to a nucleic acid sequence or construct designed for transfer between different host cells. An "expression vector" refers to a vector that has the ability to incorporate and express heterologous DNA sequences in a foreign cell. An expression vector may be generated recombinantly or synthetically, with a series of specified nucleic acid elements that permit transcription of a particular nucleic acid in a target cell. The recombinant expression cassette can be incorporated into a plasmid, chromosome, mitochondrial DNA, virus, or nucleic acid fragment.<br>
As used herein, the term "plasmid" refers to a circular double-stranded (ds) DNA construct used as a cloning vector, and which forms an extrachromosomal self-replicating genetic element in many bacteria and some eukaryotes.<br>
As used herein, the term "selectable marker" refers to a nucleotide sequence which is capable of expression in cells and where expression of the selectable marker confers to cells containing the expressed gene the ability to grow in the presence of a corresponding selective agent, or under corresponding selective growth conditions.<br>
As used herein, the term "promoter" refers to a nucleic acid sequence that functions to direct transcription of a downstream gene. The promoter will generally be appropriate to the host cell in which the target gene is being expressed. The promoter together with other transcriptional and translational regulatory nucleic acid sequences (also termed "control sequences") are necessary to express a given gene. In general,, the transcriptional and translational regulatory sequences include, but are not limited to, promoter sequences, ribosomal binding sites, transcriptional start and stop sequences, translational start and stop sequences, and enhancer or activator sequences.<br>
The term "signal sequence" or "signal peptide" refers to a sequence of amino acids at the N-terminal portion of a protein, which facilitates the secretion of the mature form of the protein outside the cell. The mature form of the extracellular protein lacks the signal sequence which is cleaved off during the secretion process.<br>
By the term "host cell" is meant a cell that contains a heterologous exo-endo cellulase fusion construct encompassed by the invention or a vector including the same and supports the replication, and/or transcription or transcription and translation (expression) of the heterologous exo-endo cellulase construct. Host cells for use in the present invention can be prokaryotic cells, such as E. coli, or eukaryotic cells such as yeast, plant, insect, amphibian, or mammalian cells. In general, host cells are filamentous fungi.<br>
The term "filamentous fungi" includes all filamentous fungi recognized by those of skill in the art. A preferred fungus is selected from the subdivision Eumycota and Oomycota and particularly from the group consisting of Aspergillus, Trichoderma, Fusarium, Chrysosporium, Penicillium, Humicola, Neurospora, or alternative sexual forms thereof such as Emericella and Hypocrea (See, Kuhls et al., 1996).<br>
The filamentous fungi are characterized by vegetative mycelium having a cell wall composed of chitin, glucan, chitosan, mannan, and other complex polysaccharides, with vegetative growth by hyphal elongation and carbon catabolism that is obligately aerobic.<br>
The term "derived" encompasses the terms originated from, obtained or obtainable from and isolated from.<br>
An "equivalent" amino acid sequence is an amino acid sequence that is not identical to an original reference amino acid sequence but includes some amino acid changes, which may be substitutions, deletions, additions or the like, wherein the protein exhibits essentially the'same qualitative biological activity of the reference protein. An equivalent amino acid sequence will have between 80% - 99% amino acid identity to the original reference sequence. Preferably the equivalent amino acid sequence will have at least 85%, 90%, 93%, 95%, 96%, 98% and 99% identity to the reference sequence.<br>
A "substitution" results from the replacement of one or more nucleotides or amino acid by different nucleotides or amino acids, respectively. Substitutions are usually made in accordance with known conservative substitutions, wherein one class of amino acid is substituted with an amino acid in the same class. A "non-conservative substitution" refers to the substitution of an amino acid in one class with an amino acid from another class.<br>
A "deletion" is a change in a nucleotide or amino acid sequence in which one or more nucleotides or amino acids are absent.<br>
An "addition" is a change in a nucleotide or amino acid sequence that has resulted from the insertion of one or more nucleotides or amino acid as compared to an original reference sequence.<br>
As used herein, "recombinant" includes reference to a cell or vector, that has been modified by the introduction of heterologous nucleic acid sequences or that the cell is derived from a cell so modified. Thus, for example, recombinant cells express genes that are not found in identical form within the native (non-recombinant) form of the cell or express native genes that are otherwise abnormally expressed, under expressed or not expressed at all as a result of deliberate human intervention.<br>
As used herein, the terms "transformed", "stably transformed" or "transgenic" with reference to a cell means the cell has a heterologous nucleic acid sequence according to the invention integrated into its genome or as an episomal plasmid that is maintained through multiple generations.<br>
The term "introduced" in the context of inserting a heterologous exo-endo cellulase fusion construct or heterologous nucleic acid sequence into a cell, means "transfection", "transformation" or "transduction" and includes reference to the incorporation of a heterologous nucleic acid sequence or heterologous exo-endo cellulase fusion construct into a eukaryotic or prokaryotic cell where the heterologous nucleic acid sequence or<br>
heterologous exo-endo cellulase nucleic acid construct may be incorporated into the genome of the cell (for example, chromosome, plasmid, plastid, or mitochondria! DNA), converted into an autonomous replicon, or transiently expressed (for example, transfected mRNA).<br>
As used herein, the term "expression" refers to the process by which a polypeptide is produced based on the nucleic acid sequence of a gene. The process includes both transcription and translation.<br>
It follows that the term "cellulase fusion protein expression" or "fusion expression" refers to transcription and translation of a "heterologous exo-endo cellulase fusion construct" comprising the catalytic domain of an exo-cellobiohydrolase and the catalytic domain of an endoglucanase, the products of which include precursor RNA, mRNA, polypeptide, post-translationally processed polypeptides, and derivatives thereof.<br>
As used herein, the term "purifying" generally refers to subjecting recombinant nucleic acid or protein containing cells to biochemical purification and/or column chromatography.<br>
As used herein, the terms "active" and "biologically active" refer to a biological activity associated with a particular protein, such as the enzymatic activity associated with a cellulase. It follows that the biological activity of a given protein refers to any biological activity typically attributed to that protein by those of skill in the art. As used herein, the term "enriched" means that the concentration of a cellulase enzyme found in a fungal cellulase composition is greater relative to the concentration found in a wild type or naturally occurring fungal cellulase composition. The terms enriched, elevated and enhanced may be used interchangeably herein.<br>
A "wild type fungal cellulase composition" is one produced by a naturally occurring fungal source and which comprises one or more BG, CBH and EG components wherein each of these components is found at the ratio produced by the fungal source.<br>
Thus, to illustrate, a naturally occurring cellulase system may be purified into substantially pure components by recognized separation techniques well published in the literature, including ion exchange chromatography at a suitable pH, affinity chromatography, size exclusion and the like. A purified cellulase fusion protein or components thereof may then be added to the enzymatic solution resulting in an enriched cellulase solution. It is also possible to elevate the amount of EG or CBH produced by a microbe by expressing a cellulase fusion protein encompassed by the invention.<br>
"A", "an" and "the" include plural references unless the context clearly dictates otherwise.<br>
As used herein the term "comprising" and its cognates are used in their inclusive sense: that is equivalent to the term "including" and its corresponding cognates. "ATCC" refers to American Type Culture Collection located in Manassas VA 20108 (ATCC www/atcc.org).<br>
"NRRL" refers to the Agricultural Research Service Culture Collection, National Center for Agricultural utilization Research (and previously known as USDA Northern<br>
Regional Research Laboratory), Peoria, ILL.<br>
2. PREFERRED EMBODIMENTS<br>
A. Components and Construction of Heterologous Exo-Endo Cellulase Fusion Constructs and Expression Vectors.<br>
A heterologous exo-endo cellulase fusion construct or a vector comprising a heterologous exo-endo cellulase fusion construct may be introduced into and replicated in a filamentous fungal host cell for protein expression and secretion.<br>
In some embodiments, the heterologous exo-endo cellulase fusion construct comprises in operable linkage from the 5' end of said construct, optionally a signal peptide, a DNA molecule encoding a catalytic domain of an exo-cellobiohydrolase, and a DNA molecule encoding a catalytic domain of an endoglucanase. In other embodiments, the components of the heterologous exo-endo cellulase fusion construct comprise in operable linkage from the 5' end of said construct, optionally a signal peptide, a DNA molecule encoding a catalytic domain of an exo-cellobiohydrolase, optionally a DNA molecule encoding the CBD of an endoglucanase, and a DNA molecule encoding a catalytic domain of the endoglucanase.<br>
In other embodiments the construct will comprise in operable linkage from the 5' end of said construct optionally a signal peptide, a DNA molecule encoding a catalytic domain of an exo-cellobiohydrolase, optionally a DNA molecule encoding the CBD of the exo-cellobiohydrolase, a linker, optionally a DNA molecule encoding the CBD of an endoglucanase, and a DNA molecule encoding a catalytic domain of the endoglucanase.<br>
In a further embodiment the heterologous exo-endo cellulase fusion construct or vector comprising a heterologous exo-endo cellulase fusion construct includes in operable linkage from the 5' end, a promoter of a filamentous fungus secretable protein; a DNA molecule encoding a signal sequence; a DNA molecule encoding a catalytic domain of an<br>
exo-cellobiohydrolase, optionally a DNA molecule encoding the exo-cellobiohydrolase CBD; a DNA molecule encoding a catalytic domain of an endoglucanase; and a terminator. Further the vector may include a DNA molecule encoding the CBD of the endoglucanase said CBD located 5' to the DNA molecule encoding the endoglucanase catalytic domain.<br>
In one embodiment a preferred heterologous exo-endo cellulase fusion construct or expression vector will not include the exo-cellobiohydrolase CBD. In another embodiment, a preferred expression vector will include a promoter of a filamentous fungus secretable protein, a DNA molecule encoding an exo-cellobiohydrolase signal sequence, a DNA molecule encoding a catalytic domain of an exo-cellobiohydrolase, a linker, a DNA molecule encoding a catalytic domain of an endoglucanase, and a terminator, wherein the vector lacks the CBD of the exo-cellobiohydrolase and optionally lacks the CBD of the endoglucanase. In a preferred embodiment, the coding sequence for the endoglucanase catalytic domain (either including the endoglucanase CBD or lacking the endoglucanase CBD) will not include an endoglucanase signal sequence. Reference is made to Figures 1, 10 and 12 as examples of embodiments including an expression vector and heterologous exo-endo cellulase fusion construct of the invention.<br>
Exemplary promoters include both constitutive promoters and inducible promoters. Examples include the promoters from the Aspergillus niger, A. awamoriorA. oryzae glucoamylase, alpha-arnylase, or alpha-glucosidase encoding genes; the A. nidulans gpdA or trpC genes; the Neurospora crassa cbhl or trpl genes; the A. niger or Rhizomucor miehei aspartic proteinase encoding genes; the T. reesei cbhl, cbh2, eg!1, eg/2, or other cellulase encoding genes; a CMV promoter, an SV40 early promoter, an RSV promoter, an EF-1a promoter, a promoter containing the tet responsive element (TRE) in the tet-on or tet-off system as described (ClonTech and BASF), the beta actin promoter. In some embodiments the promoter is one that is native to the fungal host cell to be transformed. In one preferred embodiment, the promoter is an exo-cellobiohydrolase cbhl or cbh2 promoter and particularly a cbhl promoter, such as a T. reesei cbhl promoter. The T. reesei cbhl promoter is an inducible promoter, and reference is made to GenBank Accession No. D86235.<br>
The DNA sequence encoding an exo-cellobiohydrolase catalytic domain is operably linked to a DNA sequence encoding a signal sequence. The signal sequence is preferably that which is naturally associated with the exo-cellobiohydrolase to be expressed. Preferably the signal sequence is encoded by a Trichoderma or Aspergillus gene which encodes a CBH. More preferably the signal sequence is encoded by a Trichoderma gene<br>
which encodes a CBH1. In further embodiments, the promoter and signal sequence of the heterologous exo-endo cellulase fusion construct are derived from the same source. In some embodiments, the signal sequence is a Trichoderma cb/71 signal.sequence that is operably linked to a Trichoderma cbM promoter. In further embodiments the signal sequence has the amino acid sequence of SEQ ID NO: 2 or an equivalent sequence or a sequence having at least 95% identity thereto.<br>
Most exo-cellobiohydrolases (CBHs) and endoglucanases (EGs) have a multidomain structure consisting of a catalytic domain separated from a cellulose binding domain (CBD) by a linker peptide (Suurnakki et al., 2000). The catalytic domain contains the active site whereas the CBD interacts with cellulose by binding the enzyme to it (van Tilbeurgh et al., 1986 and Tomme et al., 1988).<br>
Numerous cellulases have been described in the scientific literature, examples of which include: from Trichoderma reesei: Shoemaker, S. et al., Bio/Technology, 1:691-696, 1983, which discloses CBH1; Teeri, T. et al., Gene, 51:43-52, 1987, which discloses CBH2; Penttila, M. et al., Gene, 45:253-263,1986, which discloses EG1; Saloheimo, M. et al., Gene, 63:11-22, 1988, which discloses EG2; Okada, M. et al., Appl. Environ. Microbiol., 64:555-563, 1988, which discloses EGS; Saloheimo, M. eta!., Eur. J. Biochem., 249:584-591, 1997, which discloses EG4; and Saloheimo, A. et al., Molecular Microbiology, 13:219-228, 1994, which discloses EGS. Exo-cellobiohydrolases and endoglucanases from species other than Trichoderma have also been described e.g., Ooi et al., 1990, which discloses the cDNA sequence coding for endoglucanase F1-CMC produced by Aspergillus aculeatus; Kawaguchi T ef al., 1996, which discloses the cloning and sequencing of the cDNA encoding beta-glucosidase 1 from Aspergillus aculeatus; Sakamoto et al., 1995, which discloses the cDNA sequence encoding the endoglucanase CMCase-1 from Aspergillus kawachii IFO 4308; and Saarilahti et al., 1990 which discloses an endoglucanase from Erwinia carotovara. The sequences encoding these enzymes may be used in the heterologous exo-endo cellulase fusion construct or vector of the invention.<br>
In some embodiments, the catalytic domain is derived from a CBH1 type exo-cellobiohydrolase and in other embodiments the catalytic domain is derived from a CBH2 type exo-cellobiohydrolase. In some embodiments, the CBH1 or CBH2 catalytic domain is derived from a Trichoderma spp.<br>
In one embodiment, the catalytic domain of an exo-cellobiohydrolase is encoded by a nucleic acid sequence of a Trichoderma reesei cbh"\. In some embodiments the nucleic acid is the sequence of SEQ ID NO:3 and nucleotide sequences homologous thereto.<br>
In other embodiments, the catalytic domain will have the amino acid sequence of SEQ ID NO: 6 and equivalent amino acid sequences thereto. Further DNA sequences encoding any equivalents of said amino acid sequences of SEQ ID NO: 6, wherein said equivalents have a similar qualitative biological activity to SEQ ID NO: 6 may be incorporated into the heterologous exo-endo cellulase fusion construct.<br>
In some embodiments, heterologous exo-endo cellulase fusion constructs encompassed by the invention will include a linker located 3' to the sequence encoding the exo-cellobiohydrolase catalytic domain and 5' to the sequence encoding the endoglucanase catalytic domain. In some preferred embodiments, the linker is derived from the same source as the catalytic domain of the exo-cellobiohydrolase. Preferably the linker will be derived from a Trichoderma cbhl gene. One preferred linker sequence is illustrated in Figure 3. In other embodiments, the heterologous exo-endo cellulase fusion construct will include two or more linkers. For example a linker may be located not only between the coding sequence of the CBH catalytic domain and the coding sequence of the EG catalytic domain but also between the coding region of the CBH CBD and the coding region of the EG CBD. Further linkers may be located between the CBD of the endoglucanase and the catalytic domain of the endoglucanase. In general, a linker may be between about 5 to 60 amino acid residues, between about 15 to 50 amino acid residues, and between about 25 to 45 amino acid residues. Reference is made to Srisodsuk M. et al., 1993 for a discussion of the linker peptide of T. reesei CBH1.<br>
In addition to the linker sequence, a heterologous exo-endo cellulase fusion construct or expression vector of the invention may include a cleavage site, such as a protease cleavage site. In one preferred embodiment, the cleavage site is a kexin site which encodes the dipeptide Lys-Arg.<br>
In a preferred embodiment, the heterologous exo-endo cellulase fusion construct and an expression vector including the same will lack the CBD of the CBH. In other embodiments the CBD will be included in the construct or vector.<br>
The heterologous exo-endo cellulase fusion constructs include a coding sequence for the catalytic domain of an endoglucanase. Endoglucanases are found in more than 13 of the Glycosyl Hydrolase families using the classification of Coutinho, P. M. et al. (1999) Carbohydrate-Active Enzymes (CAZy) server at (afmb.cnrs-mrs.fr/~cazy/CAZY/index). Preferably the catalytic domain is derived from a bacterial endoglucanase. As described above numerous bacterial endoglucanases are known.<br>
Particularly preferred DNA sequences encoding a catalytic domain of a bacterial endoglucanase include:<br>
a)	the DNA of SEQ ID NO: 7 encoding an Acidothermus cellulolyticus GH5A<br>
endoglucanase I (E1) catalytic domain having amino acid sequence SEQ ID NO: 8;<br>
b)	the DNA of SEQ ID NO: 9 encoding an Acidothermus cellulolyticus GH74<br>
endoglucanase catalytic domain having amino acid sequence SEQ ID NO: 10;<br>
c)	the DNA of SEQ ID NO: 11 encoding a Thermobifida fusca E5 endoglucanase<br>
having amino acid sequence SEQ ID NO: 12 and<br>
d)	DNA sequences or homologous DNA sequences encoding any equivalents of<br>
said amino acid sequences of SEQ ID NOs: 8, 10 and 12 wherein said equivalents have a<br>
similar qualitative biological activity to said sequences.<br>
In some preferred embodiments, the endoglucanase is an Acidothermus cellulolyticus E1 and reference is made to the an Acidothermus cellulolyticus endoglucanases disclosed in WO 9105039; WO 9315186; USP 5,275,944; WO 9602551; USP 5,536,655 and WO 0070031. Also reference is made to GenBank U33212. In some embodiments, the Acidothermus cellulolyticus E1 has an amino acid sequence of a least 90%, 93%, 95% and 98% sequence identity with the sequence set forth in SEQ ID NO: 6.<br>
As stated above homologous nucleic acid sequences to the nucleic acid sequences illustrated in SEQ ID NOs: 1, 3, 7, 9 and 11 may be used in a heterologous cellulase fusion construct or vector according to the invention. Homologous sequences include sequences found in other species, naturally occurring alleljc variants and biologically active functional derivatives. A homologous sequence will have at least 80%, 85%, 88%, 90%, 93%, 95%, 97%, 98% and 99% identity to one of the sequences of SEQ ID NOs: 1, 3, 7, 9 and 11 when aligned using a sequence alignment program. For example, a homologue of a given sequence has greater than 80% sequence identity over a length of the given sequence e.g., the coding sequence for the Tf-E5 catalytic domain as described herein.<br>
For a given heterologous exo-endo cellulase fusion construct or components of the construct it is appreciated that as a result of the degeneracy of the genetic code, a number of coding sequences can be produced that encode a protein having the same amino acid sequence. For example, the triplet CGT encodes the amino acid arginine. Arginine is alternatively encoded by CGA, CGC, CGG, AGA, and AGG. Therefore it is appreciated that such substitutions in the coding region fall within the nucleic acid sequences covered by the present invention. Any and all of these sequences can be utilized in the same way as described herein for a CBH catalytic domain or a bacterial EG catalytic domain.<br>
Exemplary computer programs which can be used to determine identity between two sequences include, but are not limited to, the suite of BLAST programs, e.g., BLASTN, BLASTX, and TBLASTX, BLASTP and TBLASTN, publicly available on the Internet at www.ncbi.nlm.nih.gov/BI.AST/. See also, Altschul, etal., 1990 and Altschul, etal., 1997.<br>
Sequence searches are typically carried out using the BLASTN program when evaluating a given nucleic acid sequence relative to nucleic acid sequences in the GenBank DNA Sequences and other public databases. The BLASTX program is preferred for searching nucleic acid sequences that have been translated in all reading frames against amino acid sequences in the GenBank Protein Sequences and other public databases. Both BLASTN and BLASTX are run using default parameters of an open gap penalty of 11.0, and an extended gap penalty of 1.0, and utilize the BLOSUM-62 matrix. (See, e.g., Altschul, eral., 1997.)<br>
A preferred alignment of selected sequences in order to determine "% identity" between two or more sequences, is performed using for example, the CLUSTAL-W program in MacVector version 6.5, operated with default parameters, including an open gap penalty of 10.0, an extended gap penalty of 0.1, and a BLOSUM 30 similarity matrix.<br>
In one exemplary approach, sequence extension of a nucleic acid encoding a CBH or EG catalytic domain may be carried out using conventional primer extension procedures as described in Sambrook eral., supra, to detect CBH or bacterial EG precursors and processing intermediates of mRNA that may not have been reverse-transcribed into cDNA and/or to identify ORFs that encode the catalytic domain or full length protein. In yet another aspect, the entire or partial nucleotide sequence of the nucleic acid sequence of the T. reesei chb\ or GH5a-E1 may be used as a probe. Such a probe may be used to identify and clone out homologous nucleic acid sequences from related organisms.<br>
Screening of a cDNA or genomic library with the selected probe may be conducted using standard procedures, such as described in Sambrook et al., (1989). Hybridization conditions, including moderate stringency and high stringency, are provided in Sambrook et al., supra.<br>
In addition, alignment of amino acid sequences to determine homology or identity between sequences is also preferably determined by using a "sequence comparison algorithm." Optimal alignment of sequences for comparison can be conducted, e.g., by the local homology algorithm of Smith &amp; Waterman, Adv. Appl. Math. 2:482 (1981), by the homology alignment algorithm of Needleman &amp; Wunsch, J. Mol. Biol. 48:443 (1970), by the<br>
search for similarity method of Pearson &amp; Lipman, Proc. Nat'lAcad. Sci. USA 85:2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, WI), by visual inspection or MOE by Chemical Computing Group, Montreal Canada.<br>
An example of an algorithm that is suitable for determining sequence similarity is the BLAST algorithm, which is described in Altschul, et al., J. Mol. Biol. 215:403-410 (1990) and reference is also made to Henikoff &amp; Henikoff, Proc. Natl. Acad. Sci. USA 89:10915 (1989)). Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information (<www.ncbi.nlm.nih.gov>).<br>
The heterologous exo-endo cellulase fusion construct according to the invention may also include a terminator sequence. In some embodiments the terminator and the promoter are derived from the same source, for example a Trichoderma exo-cellobiohydrolase gene. In other embodiments the terminator and promoter are derived from different sources. In preferred embodiments the terminator is derived from a filamentous fungal source and particular a Trichoderma. Particularly suitable terminators include cbhl derived from a strain of Trichoderma specifically T. reesei and the glucoamylase terminator derived from Aspergillus niger or A. awamori (Nunberg et al., 1984 and Boeletal., 1984).<br>
The heterologous exo-endo cellulase fusion construct or a vector comprising a fusion construct may also include a selectable marker. The choice of the proper selectable marker will depend on the host cell, and appropriate markers for different hosts are well known in the art. Typical selectable marker genes include argB from A. nidulans or 7. reesei, amdS from A. nidulans, pyr4 from Neurospora crassa or T. reesei, pyrG from Aspergillus niger or A. nidulans. Markers useful in vector systems for transformation of Trichoderma are described in Finkelstein, Chap. 6, in BIOTECHNOLOGY OF FILAMENTOUS FUNGI, Finkelstein et al eds. Butterworth-Heinemann, Boston, MA 1992. The amdS gene from Aspergillus nidulans encodes the enzyme acetamidase that allows transformant cells to grow on acetamide as a nitrogen source (Kelley et al., EMBO J. 4:475-479 (1985) and Penttila et al., Gene 61:155-164 (1987)). The selectable marker (e.g. pyrG) may restore the ability of an auxotrophic mutant strain to grow on a selective minimal medium and the selectable marker (e.g. olic37) may confer to transformants the ability to grow in the presence of an inhibitory drug or antibiotic<br>
A typical heterologous exo-endo cellulase fusion construct is depicted in Figures 1 and 10. Methods used to ligate a heterologous exo-endo cellulase fusion construct encompassed by the invention and other heterologous nucleic acid sequences and to insert them into suitable vectors are well known in the art. Linking is generally accomplished by ligation at convenient restriction sites, and if such sites do not exist, synthetic oligonucleotide linkers are used in accordance with conventional practice. Additionally vectors can be constructed using known recombination techniques.<br>
Any vector may be used as long as it is replicable and viable in the cells into which it is introduced. Large numbers of suitable cloning and expression vectors are described in Sambrook et al., 1989, Ausubel FM et al., 1993, and Strathern et al., 1981, each of which is expressly incorporated by reference herein. Further appropriate expression vectors for fungi are described in van den Hondel, C.A.M.J.J. et al. (1991) In: Bennett, J.W. and Lasure, L.L. (eds.) More Gene Manipulations in Fungi. Academic Press, pp. 396-428. The appropriate DNA sequence may be inserted into a vector by a variety of procedures. In general, the DNA sequence is inserted into an appropriate restriction endonuclease site(s) by standard procedures. Such procedures and related sub-cloning procedures are deemed to be within the scope of knowledge of those skilled in the art. Exemplary useful plasmids include pUC18, pBR322, pUC100, pSL1180 (Pharmacia Inc., Piscataway, NJ) and pFB6. Other general purpose vectors such as in Aspergillus, pRAX and in Trichoderma, pTEX maybe also be used (Figures 12 and 13).<br>
In one embodiment, a preferred vector is the vector disclosed in Figures 12 and 13, wherein the vector includes the nucleic acid sequence encoding the CBD, linker and catalytic domain of the Thermobifida fusca endoglucanase 5 (SEQ ID NO: 12). In another embodiment, a preferred vector is the vector disclosed in Figures 12 and 13, wherein the vector includes the nucleotide sequence encoding an Acidothermus cellulolyticus GH5A endoglucanase catalytic domain (SEQ ID NO: 8).<br>
B. Target Host Cells.<br>
In one embodiment of the present invention, the filamentous fungal parent or host cell may be a cell of a species of, but not limited to, Trichoderma sp., Penicillium sp., Humicola sp., Chrysosporium sp., Gliocladium sp., Aspergillus sp., Fusarium sp., Neurospora sp., Hypocrea sp., and Emericella sp. As used herein, the term "Trichoderma" or "Trichoderma sp." refers to any fungal strains which have previously been classified as Trichoderma or are currently classified as Trichoderma. Some preferred species for<br>
Trichoderma fungal parent cells include Trichoderma longibrachiatum (reesei), Trichoderma viride, Trichoderma koningii, and Trichoderma harzianum cells. Particularly preferred host cells include cells from strains of 7. reesei, such as RL-P37 (Sheir-Neiss, et al., Appl. Microbiol. Biotechnol. 20:46-53 (1984) and functionally equivalent and derivative strains, such as Trichoderma reesei strain RUT-C30 (ATCC No. 56765) and strain QM9414 (ATCC No. 26921). Also reference is made to ATCC No. 13631, ATCC No. 26921, ATCC No. 56764, ATCC No. 56767 and NRRL 1509.<br>
Some preferred species for Aspergillus fungal parent cells include Aspergillus niger, Aspergillus awamori, Aspergillus aculeatus, and Aspergillus nidulans cells. In one embodiment, the strain comprises Aspergillus niger, for example A. niger var. awamo/v'dgr246 (Goedegebuur et al, (2002) Curr. Genet 41: 89-98) and GCDAP3, GCDAP4 and GAP3-4 (Ward, M, et al., (1993), Appl. Microbiol. Biotechnol. 39:738-743). In some instances it is desired to obtain a filamentous host cell strain such as a Trichoderma host cell strain which has had one or more cellulase genes deleted prior to introduction of a heterologous exo-endo cellulase fusion construct encompassed by the invention. Such strains may be prepared by the method disclosed in U.S. Patent No. 5,246,853, U.S. Patent 5,861,271 and WO 92/06209, which disclosures are hereby incorporated by reference. By expressing a cellulase fusion protein or components thereof having cellulolytic activity in a host microorganism that is missing one or more cellulase genes, the identification and subsequent purification procedures are simplified. Any gene from Trichoderma sp. which has been cloned can be deleted, for example, the cbhl, cbh2, eg!1, and eg/2 genes as well as those encoding EG3 and/or EG5 protein (see e.g., U.S. Patent No. 5,475,101 and WO 94/28117, respectively). Gene deletion may be accomplished by inserting a form of the desired gene to be deleted or disrupted into a plasmid by methods known in the art.<br>
Parental fungal cell lines are generally cultured under standard conditions with media containing physiological salts and nutrients, such as described by Pourquie, J. et al., BIOCHEMISTRY AND GENETICS OF CELLULOSE DEGRADATION, eds. Aubert J.P. et al., Academic Press pp. 71-86 (1988) and llmen, M. et al., Appl. Environ. Microbiol. 63:1298 -1306 (1997). Also reference is made to common commercially prepared media such as yeast Malt Extract (YM) broth, Luria Bertani (LB) broth and Sabouraud Dextrose (SD) broth.<br>
C. Introduction of a Heterologous Exo-Endo Cellulase Fusion Construct or Vector into Fungal Host Cells and Culture Conditions.<br>
A host fungal cell may be genetically modified (i.e., transduced, transformed or transfected) with a heterologous exo-endo cellulase fusion construct according to the invention, a cloning vector or an expression vector comprising a heterologous exo-endo cellulase fusion construct. The methods of transformation of the present invention may result in the stable integration of all or part of the construct or vector into the genome of the filamentous fungus. However, transformation resulting in the maintenance of a self-replicating extra-chromosomal transformation vector is also contemplated.<br>
Many standard transformation methods can be used to produce a filamentous fungal cell line such as a Trichoderma or Aspergillus cell line that express large quantities of a heterologous protein. Some of the published methods for the introduction of DNA constructs into cellulase-producing strains of Trichoderma include Lorito, Hayes, DiPietro and Harman (1993) Cuir. Genet. 24: 349-356; GolDNAn, VanMontagu and Herrera-Estrella (1990) Curr. Genet. 17:169-174; Penttila, Nevalainen, Ratto, Salminen and Knowles (1987) Gene 61: 155-164, EP-A-0 244 234 and also Hazeil B. et al., 2000; for Aspergillus include Yelton, Hamer and Timberlake (1984) Proc. Natl. Acad. Sci. USA 81: 1470-1474; for Fusarium include Bajar, Podila and Kolattukudy, (1991) Proc. Natl. Acad. Sci. USA 88: 8202-8212; for Streptomyces include Hopwood et al., (1985) Genetic Manipulationof Streptomyces: A Laboratory Manual, The John Innes Foundation, Norwich, UK; and for Bacillus include Brigidi, DeRossi, Bertarini, Riccardi and Matteuzzi, (1990), FEMS Microbiol. Lett. 55: 135-138.<br>
Other methods for introducing a heterologous exo-endo cellulase fusion construct or vector into filamentous fungi (e.g., H. jecorina) include, but are not limited to the use of a particle or gene gun (biolistics), permeabilization of filamentous fungi cells walls prior to the transformation process (e.g., by use of high concentrations of alkali, e.g., 0.05 M to 0.4 M CaC12 or lithium acetate), protoplast fusion, electroporation, or agrobacterium mediated transformation (US Patent 6,255,115).<br>
An exemplary method for transformation of filamentous fungi by treatment of protoplasts or spheroplasts with polyethylene glycol and CaCI2 is described in Campbell, et al., (1989) Curr. Genet. 16:53-56, 1989 and Penttila, M. et al., (1988) Gene, 63:11-22 and Penttila, M. et al., (1987) Gene 61:155-164.<br>
Any of the well-known procedures for introducing foreign nucleotide sequences into host cells may be used. It is only necessary that the particular genetic engineering<br>
procedure used be capable of successfully introducing at least one gene into the host cell capable of expressing the heterologous gene.<br>
The invention includes the transformants of filamentous fungi especially Trichoderma cells comprising the coding sequences for the cellulase fusion protein. The invention further includes the filamentous fungi transformants for use in producing fungal cellulase compositions, which include the cellulase fusion protein or components thereof.<br>
Following introduction of a heterologous exo-endo cellulase fusion construct comprising the exoglucanase catalytic domain coding sequence and the endoglucanase catalytic domain coding sequence, the genetically modified cells can be cultured in conventional nutrient media as described above for growth of target host cells and modified as appropriate for activating promoters and selecting transformants.   The culture conditions, such as temperature, pH and the like, are those previously used for the host cell selected for expression, and will be apparent to those skilled in the art. Also preferred culture conditions for a given filamentous fungus may be found in the scientific literature and/or from the source of the fungi such as the American Type Culture Collection (ATCC; www.atcc.org/).<br>
Stable transformants of filamentous fungi can generally be distinguished from unstable transformants by their faster growth rate and the formation of circular colonies with a smooth rather than ragged outline on solid culture medium. Additionally, in some cases, a further test of stability can be made by growing the transformants on solid non-selective medium, harvesting the spores from this culture medium and determining the percentage of these spores which will subsequently germinate and grow on selective medium.<br>
The progeny of cells into which such heterologous exo-endo cellulase fusion constructs, or vectors including the same, have been introduced are generally considered to comprise the fusion protein encoded by the nucleic acid sequence found in the heterologous cellulase fusion construct.<br>
In one exemplary application of the invention encompassed herein a recombinant strain of filamentous fungi, e.g., Trichoderma reesei, comprising a heterologous exo-endo cellulase fusion construct will produce not only a cellulase fusion protein but also will produce components of the cellulase fusion protein. In some embodiments the recombinant cells including the cellulase fusion construct will produce an increased amount of cellulolytic activity compared to a corresponding recombinant filamentous fungi strain grown under essentially the same conditions but genetically modified to include separate heterologous<br>
nucleic acid constructs encoding an exo-cellobiohydrolase catalytic domain and/or an endoglucanase catalytic domain.<br>
D. Analysis of Protein Expression<br>
In order to evaluate the expression of a cellulase fusion protein of the invention by a cell line that has been transformed with a heterologous exo-endo cellulase fusion construct, assays can be carried out at the protein level, the RNA level or by use of functional bioassays particular to exo-cellobiohydrolase activity or endoglucanase activity and/or production.<br>
In general, the following assays can be used to determine integration of cellulase fusion protein expression constructs and vector sequences, Northern blotting, dot blotting (DNA or RNA analysis), RT-PCR (reverse transcriptase polymerase chain reaction), in situ hybridization, using an appropriately labeled probe (based on the nucleic acid coding sequence), conventional Southern blotting and autoradiography.<br>
In addition, the production and/or expression of a cellulase enzyme may be measured in a sample directly, for example, by assays for cellobibhydrolase or endoglucanase activity, expression and/or production. Such assays are described, for example, in Becker et al., Biochem J. (2001) 356:19-30; Mitsuishi et al., FEBS (1990) 275:135-138. Shoemaker et al. 1978; and Schulein 1988) each of which is expressly incorporated by reference herein. The ability of CBH1 to hydrolyze isolated soluble and insoluble substrates can be measured using assays described in Srisodsuk et al., J. Biotech. (1997) 57:49-57 and Nidetzky and Claeyssens Biotech. Bioeng. (1994) 44:961-966. Substrates useful for assaying exo-cellobiohydrolase, endoglucanase or B-glucosidase activities include crystalline cellulose, filter paper, phosphoric acid swollen cellulose, cellooligosaccharides, methylumbelliferyl lactoside, methylumbelliferyl cellobioside, orthonitrophenyl lactoside, paranitrophenyl lactoside, orthonitrophenyl cellobioside, paranitrophenyl cellobioside.<br>
In addition, protein expression, may be evaluated by immunological methods, such as immunohistochemical staining of cells, tissue sections or immunoassay of tissue culture medium, e.g., by Western blot or ELISA. Such immunoassays can be used to qualitatively and quantitatively evaluate expression of a cellulase, for example CBH. The details of such methods are known to those of skill in the art and many reagents for practicing such methods are commercially available.<br>
In an embodiment of the invention, the cellulase fusion protein which is expressed by the recombinant host cell will be about 0.1 to 80% of the total expressed cellulase. In other embodiments, the amount of expressed fusion protein will be in the range of about 0.1mg to 100g; about 0.1mg to 50 g and 0.1 mg to 10g protein per liter of culture media.<br>
E. Recovery And Purification Of Cellulase Fusion Proteins and Components Thereof.<br>
In general, a cellulase fusion protein or components of the cellulase fusion protein produced in cell culture are secreted into the medium and may be recovered and optionally purified, e.g., by removing unwanted components from the cell culture medium. However, in some cases, a cellulase fusion protein or components thereof may be produced in a cellular form necessitating recovery from a cell lysate. In such cases the protein is purified from the cells in which it was produced using techniques routinely employed by those of skill in the art. Examples include, but are not limited to, affinity chromatography (van Tilbeurgh et al., FEBS Lett. 16:215, 1984), ion-exchange chromatographic methods (Goyal etat., Bioresource Technol. 36:37-50,1991; Fliess etal., Eur. J. Appl. Microbiol. Biotechnol. 17:314-318, 1983; Bhikhabhai etal., J. Appl. Biochem. 6:336-345, 1984; Ellouz etal., J. Chromatography 396:307-317, 1987), including ion-exchange using materials with high resolution power (Medve etal., J. Chromatography A 808:153-165, 1998), hydrophobic interaction chromatography (Tomaz and Queiroz, J. Chromatography A 865:123-128, 1999), and two-phase partitioning (Brumbauer, etal., Bioseparation 7:287-295, 1999).<br>
Once expression of a given cellulase fusion protein is achieved, the proteins thereby produced may be purified from the cells or cell culture by methods known in the art and reference is made to Deutscher, Methods in Enzymology, vol. 182, no. 57, pp. 779, 1990; and Scopes, Methods Enzymol. 90: 479-91, 1982. Exemplary procedures suitable for such purification include the following: antibody-affinity column chromatography, ion exchange chromatography; ethanol precipitation; reverse phase HPLC; chromatography on silica or on a cation-exchange resin such as DEAE; chromatofocusing; SDS-PAGE; ammonium sulfate precipitation; and gel filtration using, e.g., Sephadex G-75.<br>
A purified form of a cellulase fusion protein or components thereof may be used to produce either monoclonal or polyclonal antibodies specific to the expressed protein for use in various immunoassays. (See, e.g., Hu etal., Mol Cell Biol. vol.11, no. 11, pp. 5792-5799, 1991). Exemplary assays include ELISA, competitive immunoassays, radioimmunoassays, Western blot, indirect immunofluorescent assays and the like.<br>
F. Utility of enzymatic compositions comprising the cellulase fusion proteins or components thereof.<br>
The cellulase fusion protein and components comprising the catalytic domains of the cellulase fusion protein find utility in a wide variety applications, including use in detergent compositions, stonewashing compositions, in compositions for degrading wood pulp into sugars (e.g., for bio-ethanol production), and/or in feed compositions. In some embodiments, the cellulase fusion protein or components thereof may be used as cell free extracts. In other embodiments, the fungal cells expressing a heterologous exo-endo cellulase fusion construct are grown under batch or continuous fermentation conditions.<br>
A classical batch fermentation is a closed system, wherein the composition of the medium is set at the beginning of the fermentation and is not subject to artificial alterations during the fermentation. Thus, at the beginning of the fermentation the medium is inoculated with the desired organism(s). In this method, fermentation is permitted to occur without the addition of any components to the system. Typically, a batch fermentation qualifies as a "batch" with respect to the addition of the carbon source and attempts are often made at controlling factors such as pH and oxygen concentration. The metabolite and biomass compositions of the batch system change constantly up to the time the fermentation is stopped. Within batch cultures, cells progress through a static lag phase to a high growth log phase and finally to a stationary .phase where growth rate is diminished or halted. If untreated, cells in the stationary phase eventually die. In general, cells in log phase are responsible for the bulk of production of end product.<br>
A variation on the standard batch system is the "fed-batch fermentation" system, which also finds use with the present invention. In this variation of a typical batch system, the substrate is added in increments as the fermentation progresses. Fed-batch systems are useful when catabolite repression is apt to inhibit the production of products and where it is desirable to have limited amounts of substrate in the medium. Measurement of the actual substrate concentration in fed-batch systems is difficult and is therefore estimated on the basis of the changes of measurable factors such as pH, dissolved oxygen and the partial pressure of waste gases such as CO2. Batch and fed-batch fermentations are common and well known in the art.<br>
Continuous fermentation is an open system where a defined fermentation medium is added continuously to a bioreactor and an equal amount of conditioned medium is<br>
removed simultaneously for processing. Continuous fermentation generally maintains the cultures at a constant high density where cells are primarily in log phase growth.<br>
Continuous fermentation allows for the modulation of one factor or any number of factors that affect cell growth and/or end product concentration. For example, in one embodiment, a limiting nutrient such as the carbon source or nitrogen source is maintained at a fixed rate an all other parameters are allowed to moderate. In other systems, a number of factors affecting growth can be altered continuously while the cell concentration, measured by media turbidity, is kept constant. Continuous systems strive to maintain steady state growth conditions. Thus, cell loss due to medium being drawn off must be balanced against the cell growth rate in the fermentation. Methods of modulating nutrients and growth factors for continuous fermentation processes as well as techniques for maximizing the rate of product formation are well known in the art of industrial microbiology.<br>
In some applications, the cellulase fusion protein and components thereof find utility in detergent compositions, stonewashing compositions or in the treatment of fabrics to improve their feel and appearance. A detergent composition refers to a mixture which is intended for use in a wash medium for the laundering of soiled cellulose containing fabrics. A stonewashing composition refers to a formulation for use in stonewashing cellulose containing fabrics. Stonewashing compositions are used to modify cellulose containing fabrics prior to sale, i.e., during the manufacturing process. In contrast, detergent compositions are intended for the cleaning of soiled garments and are not used during the manufacturing process.<br>
In the context of the present invention, such compositions may also include, in addition to cellulases, surfactants, additional hydrolytic enzymes, builders, bleaching agents, bleach activators, bluing agents and fluorescent dyes, caking inhibitors, masking agents, cellulase activators, antioxidants, and solubilizers.<br>
Surfactants may comprise anionic, cationic and nonionic surfactants such as those commonly found in detergents. Anionic surfactants include linear or branched alkylbenzenesulfonates; alkyl or alkenyl ether sulfates having linear or branched alkyl groups or alkenyl groups; alkyl or alkenyl sulfates; olefinsulfonates; and alkanesulfonates. Ampholytic surfactants include quaternary ammonium salt sulfonates, and betaine-type ampholytic surfactants. Such ampholytic surfactants have both the positive and negative charged groups in the same molecule. Nonionic surfactants may comprise polyoxyalkylene ethers, as well as higher fatty acid alkanolamides or alkylene oxide adduct thereof, fatty acid glycerine monoesters, and the like.<br><br>
Cellulose containing fabric may be any sewn or unsewn fabrics, yarns or fibers made of cotton or non-cotton containing cellulose or cotton or non-cotton containing cellulose blends including natural cellulosics and manmade cellulosics (such as jute, flax, ramie, rayon, and lyocell). Cotton-containing fabrics are sewn or unsewn fabrics, yarns or fibers made of pure cotton or cotton blends including cotton woven fabrics, cotton knits, cotton denims, cotton yarns, raw cotton and the like.<br>
Preferably the cellulase compositions comprising the cellulase fusion protein or components thereof are employed from about 0.00005 weight percent to about 5 weight percent relative to the total detergent composition. More preferably, the cellulase compositions are employed from about 0.0002 weight percent to about 2 weight percent relative to the total detergent composition.<br>
Since the rate of hydrolysis.of cellulosic products may be increased by using a transformant having a heterologous cellulase fusion cpnstruct inserted into the genome, products that contain cellulose or heteroglycans can be degraded at a faster rate and to a greater extent. Products made from cellulose such as paper, cotton, cellulosic diapers and the like can be degraded more efficiently in a landfill. Thus, the fermentation product obtainable from the transformants or the transformants alone may be used in compositions to help degrade by liquefaction a variety of cellulose products that add to the overcrowded landfills.<br>
Cellulose-based feedstocks are comprised of agricultural wastes, grasses and woods and other low-value biomass such as municipal waste (e.g., recycled paper, yard clippings, etc.). Ethanol may be produced from the fermentation of any of these cellulosic feedstocks. However, the cellulose must first be converted to sugars before there can be conversion to ethanol. A composition containing an enhanced amount of cellulolytic activity due to the inclusion of a cellulase fusion protein or components thereof may find utility in ethanol production<br>
Ethanol can be produced via saccharification and fermentation processes from cellulosic biomass such as trees, herbaceous plants, municipal solid waste and agricultural and forestry residues. However, the ratio of individual cellulase enzymes within a naturally occurring cellulase mixture produced by a microbe may not be the most efficient for rapid conversion of cellulose in biomass to glucose. It is known that endoglucanases act to produce new cellulose chain ends which themselves are substrates for the action of cellobiohydrolases and thereby improve the efficiency of hydrolysis of the entire cellulase system. Therefore, the use of increased or optimized endoglucanase activity from a<br>
cellulase fusion protein or components thereof may greatly enhance the production of ethanol and sugar which can be converted by fermentation to other chemicals.<br>
Thus, the inventive cellulase fusion protein and components thereof find use in the hydrolysis of cellulose to its sugar components. In one embodiment, the cellulase fusion protein or components thereof are added to the biomass prior to the addition of a fermentative organism. In another embodiment, the cellulase fusion protein or components thereof are added to the biomass at the same time as a fermentative organism. Optionally, there may be other cellulase components present in either embodiment.<br>
EXPERIMENTAL<br>
The present invention is described in further detail in the following examples which are not in any way intended to limit the scope of the invention.<br>
In the disclosure and experimental section, which follows, the following abbreviations apply:<br>
CBH1-E1 (T. reesei CBH1 catalytic domain and linker fused to an Acidothermus cellulolyticus GH5A endoglucanase I catalytic domain);<br>
CBH1-74E (T. reesei CBH1 catalytic domain and linker fused to an Acidothermus cellulolyticus GH74 endoglucanase catalytic domain);<br>
CBH1-TfE5 (T. reesei CBH1 catalytic domain and linker fused to a Thermobifida fusca E5 endoglucanase cellulose binding domain, linker and Thermobifida fusca E5 endoglucanase catalytic domain;<br>
wt% (weight percent); °C (degrees Centigrade); rpm (revolutions per minute); H2O (water); dH2O (deionized water); aa (amino acid); bp (base pair); kb (kilobase pair); kD (kilodaltons); g (grams); pg (micrograms); mg (milligrams); uL (microliters); ml and ml (milliliters); mm (millimeters); urn (micrometer); M (molar); mM (millimolar); uM (micromolar); U (units); MW (molecular weight); sec (seconds); min(s) (minute/minutes); hr(s) (hour/hours); PAGE (polyacrylamide gel electrophoresis); phthalate buffer, (sodium phthalate in water, 20mN, pH 5.0); PBS (phosphate buffered saline [150 mM NaCI, 10 mM sodium phosphate buffer, pH 7.2]);SDS (sodium dodecyl sulfate); Tris (tris(hydroxymethyl)aminomethane); w/v (weight to volume); w/w (weight to weight); v/v (volume to volume); and Genencor (Genencor International, Inc., Palo Alto, CA).<br>
EXAMPLE 1 Construction of a CBH1-E1 Fusion Vector<br>
The CBH1-E1 fusion construct included the 7. reeseicbhl promoter; the 7. reesei cbhl gene sequence from the start codon to the end of the cbh\ linker and an additional 12 bases of DNA 5' to the start of the endoglucanase coding sequence, a stop codon and the 7. reesei cbh\ terminator (see Figures 10 and 11). The additional 12 DNA bases (ACTAGTAAGCGG)) (SEQ ID NO. 16) code for the restriction endonuclease Spel and the amino acids Ser, Lys, and Arg.<br>
The plasmid E1-pUC19 which contained the open reading frame for the E1 gene locus was used as the DNA template in a PCR reaction. (Equivalent plasmids are described in US Patent 5,536,655, which describes cloning the E1 gene from the actinomycete Acidothermus cellulolyticus ATCC 43068, Mohagheghi A. et al., 1986). Standard procedures for working with plasmid DNA and amplification of DNA using the polymerase chain reaction (PCR) are found in Sambrook, et al., 2001.<br>
The following two primers were used to amplify the coding region of the catalytic domain of the E1 endoglucanase. Forward Primer 1 = EL-316 (containing a Spel site):<br>
GCTTATACTAGTAAGCGCGCGGGCGGCGGCTATTGGCACAC (SEQ ID NO: 17) Reverse Primer 2 = EL-317 (containing an AscI site and stop codon-reverse compliment):<br>
GCTTATGGCGCGCCTTAGACAGGATCGAAAATCGACGAC (SEQ ID NO: 18). The reaction conditions were as follows using materials from the PLATINUM Pfx DNA Polymerase kit (Invitrogen, Carlsbad, CA): 1 pi dNTP Master Mix (final concentration 0.2mM); 1 ul primer 1 (final cone 0.5 uM); 1 ul primer 2 (final cone 0.5 uM); 2ul DNA template (final cone 50 - 200 ng); 1ul 50mM MgSO4 (final cone 1mM); 5ul 10 x Pfx Amplification Buffer; 5uMO x PCRx Enhancer Solution; 1µl Platinum Pfx DNA Polymerase (2.5U total); 33ul water for 50 ul total reaction volume.<br>
Amplification parameters were: step 1 - 94°C for 2 min (1st cycle only to denature antibody bound polymerase); step 2 - 94°C for 45 sec; step 3 - 60°C for 30 sec; step 4 -68°C for 2 min; step 5 - repeated step 2 for 24 cycles and step 6 - 68°C for 4 min.<br>
The appropriately sized PCR product was cloned into the Zero Blunt TOPO vector and transformed into chemically competent TopIO E. coli cells (Invitrogen Corp., Carlsbad, CA) - plated onto to appropriate selection media (LA with 50 ppm with kanamycin and grown overnight at 37°C. Several colonies were picked from the plate media and grown<br>
overnight in 5 ml cultures at 37°C in selection media (LB with 50 ppm kanamycin) from which plasmid mini-preps were made. Plasmid DNA from several clones was restriction digested to confirm the correct size insert. The correct sequence was confirmed by DNA sequencing. Following sequence verification, the E1 catalytic domain was excised from the TOPO vector by digesting with the restriction enzymes Spel and Ascl. This fragment was ligated into the pTrex4 vector which had been digested with the restriction enzymes Spel and Ascl (see, Figures 12 and 13).<br>
The ligation mixture was transformed into MM294 competent E. co//cells, plated onto appropriate selection media (LA with 50 ppm carbenicillin) and grown overnight at 37°C. Several colonies were picked from the plate media and grown overnight in 5 ml cultures at 37°C in selection media (LA with 50 ppm carbenicillin) from which plasmid mini-preps were made. Correctly ligated CBH1-E1 fusion protein vectors were confirmed by restriction digestion.<br>
EXAMPLE 2<br>
Transformation and Expression the CBH1-E1 Fusion Construct into<br>
a T. reesei host strain.<br>
Various T. reesei strains were transformed with the CBH1-E1 fusion construct. The host strains included a derivative of T. reesei RL-P37 and a derivative of T. reesei wherein the native cellulase genes (cbhl, cbh2, eg!1 and eg/2) were deleted.<br>
Approximately one-half swab (or 1 -2 cm2) of a plate of a sporulated T. reesei derivative of strain RL-P37 (Sheir-Neiss, et al., 1984) mycelia (grown on a PDA plate for 7 days at 28°C) was inoculated into 50ml of YEG (5g/L yeast extract plus 20g/L glucose) broth in a 250 ml, 4-baffled shake flask and incubated at 30°C for 16-20 hours at 200 rpm. The mycelia was recovered by transferring the liquid volume into 50ml conical tubes and spinning at 2500 rpm for 10 minutes. The supernatant was aspirated off. The mycelial pellet was transferred into a 250 ml, CA Corning bottle containing 40ml of B glucanase solution and incubated at 30°C, 200 rpm for 2 hrs to generate protoplasts for transformation. Protoplasts were harvested by filtration through sterile miracloth into a 50ml conical tube. They were pelleted by spinning at 2000 rpm for 5 minutes, the supernate was aspirated off. The protoplast pellet was washed once with 50ml of 1.2 M sorbitol, spun down, aspirated, and washed with 25ml of sorbitol CaCI2. Protoplasts were counted and then pelleted again at 2000 rpm for 5 min, the supernate was aspirated off, and the protoplast pellet was resuspended in a sufficient volume of sorbitol/CaCI2 to generate a<br>
protoplast concentration of 1.25 x 108 protoplasts per ml. This constitutes the protoplast solution.<br>
Aliquots of up to 20 pg of expression vector DNA (in a volume no greater than 20 pi) were placed into 15ml conical tubes and the tubes were put on ice. Then 200ul of the protoplast solution was added, followed by 50ul PEG solution to each transformation aliquot. The tubes were mixed gently and incubated on ice for 20 min. Next, an additional 2 ml of PEG solution was added to the transformation aliquot tubes, followed by gentle inversion and incubation at room temperature for 5 minutes. Next 4 ml of Sorbitol/CaCI2 solution was added to the tubes (generating a total volume of 6.2 ml). This transformation mixture was divided into 3 aliquots each containing about 2ml. An overlay mixture was created by adding each of these three aliquots to three tubes containing 10 ml of melted acetamide/sorbitol top agar (kept molten by holding at 50°C) and this overlay mixture was poured onto a selection plate of acetamide/sorbitol agar. The transformation plates were then incubated at 30°C for four to seven days.<br>
The transformation was performed with amdS selection. Acetamide/sorbitol plates and overlays were used for the transformation. For the selection plates, the same plates were used, but without sorbitol. Transformants were purified by transfer of isolated colonies to fresh selective media containing acetamide.<br>
With reference to the examples the following solutions were made as follows.<br>
1)	40 ml p-D-glucanase solution was made up in 1.2M sorbitol and included<br>
600mg |3-D-glucanase and 400mg MgSO4 -7H20 (Catalog No. 0439-1, InterSpex Products<br>
Inc., San Mateo, CA).<br>
2)	200 ml PEG solution contained 50g polyethylene glycol 4000 (BDH Laboratory<br>
Supplies Poole, England) and 1.47g CaCI2 '2H20 made up in dH2O.<br>
3)	Sorbitol/ CaCI2 contained 1.2M sorbitol and 50mM CaCI2.<br>
4)	Acetamide/sorbitol agar:<br>
Part 1 - 0.6g acetamide (Aldrich, 99% sublime.), 1.68g CsCI, 20g glucose, 20g KH2PO4, 0.6g MgSO4 -7H2O, 0.6g CaCI2 -2H2O, 1 ml 1000 x salts (see below), adjusted to pH 5.5, brought to volume (300 mis) with dH2O, filtered and sterilized. Part 2 - 20g Noble agar and 218g sorbitol brought to volume (700mls) with dH2O and autoclaved.   Part 2 was added to part 1 for a final volume of 1L<br>
5)	1000 x Salts - 5g FeSO4 -7H20,1.6g MnS04 -H2O, 1.4g ZnSO4 -7^0,1g CoCI2<br>
•6H2O were combined and the volume was brought to 1L with dH2O. The solution was<br>
filtered and sterilized.<br>
6)  Acetamide/sorbitol top agar is prepared as is acetamide/sorbitol agar except that top agar is substituted for noble agar.<br>
The transformation procedure used was similar to that decribed in Penttila et al., Gene 61: 155-164,1987.<br>
Individual fungal transformants were grown up in shake flask culture to determine the level of fusion protein expression. The experiments were conducted essentially as described in example 1 of U.S. Patent 5,874,276 with the following modification: 16 g/L of alpha-lactose was substituted for cellulose in TSF medium. The highest level of cleaved E1 protein expression from a transformant in shake flasks was estimated to be greater than 3 g/L.<br>
In general, the fermentation protocol as described in Foreman et al. (Foreman et al. (2003) J. Biol. Chem 278:31988-31997) was followed. Vogels minimal medium (Davis et al., (1970) Methods in Enzymology 17A, pg 79 -143 and Davis, Rowland, NEUROSPORA, CONTRIBUTIONS OF A MODEL ORGANISM, Oxford University Press, (2000)) containing 5% glucose was inoculated with 1.5 ml frozen spore suspension. After 48 hours, each culture was transferred to 6.2L of the same medium in a 14L Biolafitte fermenter. The fermenter was run at 25°C, 750 RPM and 8 standard liters per minute airflow. One hour after the initial glucose was exhausted, a 25% (w/w) lactose feed was started and fed in a carbon limiting fashion to prevent lactose accumulation. The concentrations of glucose and lactose were monitored using a glucose oxidase assay kit or a glucose hexokinase assay kit with beta-galactosidase added to cleave lactose, respectively (Instrumentation Laboratory Co., Lexington, MA). Samples were obtained at regular intervals to monitor the progress of the fermentation. Collected samples were spun in a 50ml centrifuge tube at 3/4 speed in an International Equipment Company (Needham Heights, MA) clinical centrifuge.<br>
Shake flask grown supernatant samples were run on BIS-TRIS SDS -PAGE gels (Invitrogen), under reducing conditions with MOPS (morpholinepropanesulfonic acid) SDS running buffer and LDS sample buffer. The results are provided in Figure 14.<br>
EXAMPLE 3 Assay of Cellulolytic Activity from Transformed Trichoderma reesei Clones<br>
The following assays and substrates were used to determine the cellulolytic activity of the CBH1-E1 fusion protein.<br>
Pretreated corn stover (PCS) - Corn stover was pretreated with 2% w/w H2S04 as described in Schell, D. et al., J. Appl. Biochem. Biotechnol. 105:69 - 86 (2003) and followed by multiple washes with deionized water to obtain a pH of 4.5. Sodium acetate was added to make a final concentration of 50mM and this was titrated to pH 5.0.<br>
Measurement of Total Protein - Protein concentration was measured using the bicinchoninic acid method with bovine serum albumin as a standard. (Smith P.K. et al., Biochem 150:76 - 85, 1985).<br>
Cellulose conversion (Soluble sugar determinations) was evaluated by HPLC according to the methods described in Baker et al., Appl. Biochem. Biotechnol. 70-72:395 -403(1998).<br>
A standard cellulosic conversion assay was used in the experiments. In this assay enzyme and buffered substrate were placed in containers and incubated at a temperature over time. The reaction was quenched with enough 100 mM Glycine, pH 11.0 to bring the pH of the reaction mixture to at least pH10. Once the reaction was quenched, an aliquot of the reaction mixture was filtered through a 0.2 micron membrane to remove solids. The filtered solution was then assayed for soluble sugars by HPLC as described above. The cellulose concentration in the reaction mixture was approximately 7%. The enzyme or enzyme mixtures were dosed anywhere from 1 to 60 mg of total protein per gram of cellulose.<br>
In one set of experiments the percent conversion of 13.8% PCS (7.06% cellulose) at 55°C for 1 day was evaluated using 10 mg enzyme/g cellulose in 50 mM acetate buffer at 55°C. Samples were agitated at 700 rpm. Comparisons were made between supernatants from growth of 1) a T. reesei parent strain which included the native cellulase genes and 2) a corresponding T. reesei CBH1-E1 fusion strain transformed according to the examples herein. The amount of E1 protein expressed by this strain was 10% w/w (estimated by PAGE as a percent of total protein). Samples were quenched at various times up to 24 hours.<br>
The results are presented in Figure 17, and it is observed that the CBH1-E1 fusion protein outperforms the parent. It took about 6 hours for the CBH1-E1 fusion protein to yield 20% cellulose conversion, while it requires 10 hours for the parent cellulase to reach 20% hydrolysis.<br>
EXAMPLE 4<br>
Transformation and Expression the CBH1-74E Fusion Construct<br>
into T. reesei<br>
The CBH1-74E fusion construct was designed according to the procedures described above in example 1 with the following differences. The forward primer was designed to maintain the reading frame translation and included a Lys-Arg kexin cleavage site (underlined). The reverse primer encodes a stop codon (the reverse compliment) at the end of the catalytic domain.<br>
Primers were ordered with 5 prime phosphates to enable subsequent blunt cloning. The GH74 catalytic domain was amplified with the following forward and reverse primers: GH74 forward primer bluntF4 -<br>
CTAAGAGAGCGACGACTCAGCCGTACACCTGGAGCAACGTGGC (SEQ ID NO: 19) and GH74 reverse primer bluntR4 -3TACGATCCGGACGGCGCACCACCAATGTCCCCGTATA (SEQ ID NO: 20).   a<br>
Amplification was performed using Stratagene's Herculase High Fidelity Polymerase (Stratagene, La Jolla, CA). The amplification conditions for the GH74 catalytic domain were:<br>
An isolated fragment of DNA encompassing the GH74 catalytic domain was used as the template for PCR (approximately 0.2 ug of DNA). U.S. Pat. Appln. No. 20030108988 describes the cloning of GH74. (GH74 is referred to as Avilll in the published patent application).<br>
(Table Removed)<br>
All PCR products were gel purified and treated with Mung Bean Nuclease to produce blunt ends prior to ligation. The amplified, blunted fragment was ligated into pTrex4 vector that had been digested with the restriction enzymes Spel and AscI followed by nuclease digestion to remove the 3' overhangs thereby creating blunt ends. The newly created vector was then transformed into Ecoli. Plasmid DNA was isolated from colonies of transformed E.coli. Since the amplified GH74 fragment could insert into pTrex4 in two different orientations, restriction digests were performed to discern clones with correctly oriented insert. Putative clones were confirmed by DNA sequencing. Transformation of the fusion vector into T.reesei was performed using biolistic transformation according to the teaching of Hazell, B. W. et a!., Lett. Appl. Microbiol. 30:282-286 (2000).<br>
Expression of the CBH1-74E fusion protein was determined as described above for expression of the CBH1-E1 fusion protein in Example 2 . The highest level of cleaved GH74 protein expression from a transformant in shake flasks was estimated to be greater then 3 g/L.<br>
Shake flask grown supernatant samples were run on BIS-TRIS SDS -PAGE gels (Invitrogen), under reducing conditions with MOPS (morpholinepropanesulfonic acid) SDS running buffer and LDS sample buffer. The results are provided in Figure 15.<br>
EXAMPLE 5<br>
Transformation and Expression the CBHI-TfE5 Fusion Construct into T. reesei<br>
The CBH1-TfE5 fusion construct was designed according to the procedures described above in example 1 with the following differences. A plasmid equivalent to that described in Collmer &amp; Wilson, Bio/technol. 1: 594 -601(1983) carrying theTfE5 gene was<br>
used as the DNA template to amplify the TfE5. The following primers were used to amplify the TfE5 endoglucanase<br>
EL-308 (which contains a Spel site) - forward primer -<br>
GCTTATACIAGIAAGCGCGCCGGTCTCACCGCCACAGTCACC (SEQ ID NO: 21) and EL-309 (which contains a AscI site) reverse primer-GCTTATGGCGCGCCTCAGGACTGGAGCTTGCTCCGC (SEQ ID NO: 22).<br>
Transformation was as described in example 2 above. The highest level of cleaved TfE5 protein expression from a transformant in shake flasks was estimated to be greater than 2 g/L.<br>
Shake flask grown supernatant samples were run on BIS-TRIS SDS -PAGE gels (Invitrogen), under reducing conditions with MOPS (morpholinepropanesulfonic acid) SDS running buffer and LDS sample buffer. The results are provided in Figure 16.<br>
REFERENCES<br>
Altschul, S. F., etal., J. Mol. Biol. 215:403-410, 1990. Altschul, S. F., etal., Nucleic Acids Res. 25:3389-3402, 1997.<br>
Aro N, Saloheimo A, llmen M, Penttila M. ACEII, a novel transcriptional activator involved in regulation of cellulase and xylanase genes of Trichoderma reesei. J Biol Chem. 2001 Jun 29;276(26):24309-14. (Epub 2001 Apr 13.)<br>
Aubert J.P. et al, p11 et seq., Biochemistry and Genetics of Cellulose Degradation, eds. Aubert, J. P., Beguin, P., Millet, J., Federation of European Microbiological Societies, Academic Press, 1988<br>
Ausubel G. M., et al. CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, New York, N.Y., 1993.<br>
Baker et al., Appl. Biochem. and Biotechnol. 45/46:245 - 256, 1994. Bhikhabhai, R. etal., J. Appl. Biochem. 6:336, 1984. Boel et al. EMBO J 3:1581-1585 1984. Brumbauer, A. etal., Bioseparation 7:287-295, 1999. Collmer, A. and D.B. Wilson Bio/Technol/1:594 - 601, 1983. Deutscher, M.P., Methods Enzymol. 182:779-80, 1990. Ellouz, S. et al., J. Chromatography 396:307, 1987. Filho, etal. Can. J. Microbiol. 42:1-5, 1996. Fliess, A., etal., Eur. J. Appl. Microbiol. Biotechnol. 17:314, 1983. Goedegebuur et al., Curr. Genet. 41 :89 - 98, 2002.<br>
Goyal, A. et al. Bioresource Technol. 36:37, 1991. Hazell, B. W. et al., Lett. Appl. Microbiol. 30:282-286, 2000. Herref al., Appl. Microbiol. Biotechnol. 5:29-36, 1978. Huet al., Mol. Cell. Biol. 11:5792-9, 1991. Jeeves et al., Biotechnol. Genet. Eng. Rev. 9:327 -369, 1991. Kawaguchi, T et al., Gene 173(2):287-8, 1996. Kelley et al. EMBO J. 4 :475-479, 1985. Knowles, J. et al., TIBTECH 5, 255-261, 1987. Krishna, S. et al., Bioresource Tech. 77:193-196, 2001. Kuhls K. et al., Proc. Natl. Acad. Sci. USA 93(15): 7755 - 7760, 1996. Kumar, A., et al., Textile Chemist and Colorist 29:37-42, 1997. Medve, J. et al., J. Chromatography A808:153, 1998. Mohagheghi, A. et al., Int. J. Syst. Bacteriol. 36:435 - 443, 1986. 'Nieves et al., Appl. Biochem. and Biotechnol. 51/52 211 - 223, 1995. Nunberg et al. Mol. Cell Biol. 4:2306-2315 1984. Ohmiya et al., Biotechnol. Gen. Engineer. Rev. 14:365-414, 1997. Okada, M. et al., Appl. Environ. Microbiol., 64:555-563, 1988. Ooi et al., Nucleic Acid Res. 18:5884, 1990 Penttila et al., Gene45:253-263,1986. Penttila et al., Gene 61: 155 - 164, 1987. Penttila et al., Gene 63: 103-112,1988.<br>
Pere, J., et al., In Proc. Tappi Pulping Conf., Nashville, TN, 27-31, pp. 693-696, 1996. Saarilahti et al., Gene 90:9-14,1990. Sakamoto et al., Curr. Genet. 27:435-439, 1995. Saloheimo M, et al., Gene63:11-22, 1988. Saloheimo, A. et al., Molecular Microbiology, 13:219-228, 1994. Saloheimo, M. et al., Eur. J. Biochem., 249:584-591, 1997.<br>
Sambrook et al., MOLECULAR CLONING: A LABORATORY MANUAL (Second Edition), Cold Spring Harbor Press, Plainview, N.Y., 1989. Schulein, Methods Enzymol., 160, 25, pages 234 et seq, 1988. Scopes, Methods Enzymol. 90 Pt E:479-90, 1982. Shoemaker et al., Biochem. Biophys. Acat. 523 :133-146 1978. Shoemaker, S. et al., Bio/Technology, 1:691-696, 1983 Srisodsuk, M. et al. J. Biol. Chem. 268(28): 20756 - 20761, 1993.<br>
Strathern et al., eds. (1981) The Molecular Biology of the Yeast Saccharomyces, Cold Spring<br>
Harbor Press, Plainview. N.Y..<br>
Suurnakki, A. et al., Cellulose 7:189-209, 2000.<br>
Teeri, T. et al., Gene, 51:43-52, 1987<br>
Van Tilbeurgh, H. et al., FEBS Lett. 16:215,1984.<br>
Tomaz, C. and Queiroz, J., J. Chromatography A 865:123-128, 1999.<br>
Tomme, P. et al., Eur. J. Biochem. 170:575-581, 1988.<br>
Van Tilbeurgh, H. et al., FEBS Lett. 204:223-227, 1986.<br>
Ward, M. et al., Appl. Microbiol. Biotechnol. 39:738 - 743, 1993.<br>
Wood, Biochem. Soc. Trans., 13, pp. 407-410, 1985.<br>
Wood et al., METHODS IN ENZYMOLOGY, 160, 25, p. 87 et seq., Academic Press, New York,<br>
1988.<br><br><br><br><br><br>
CLAIMS<br>
1.	A heterologous exo-endo cellulase fusion construct comprising in operable<br>
linkage from the 5' end of said molecule<br>
a.	a DNA molecule encoding a signal sequence;<br>
b.	a DNA molecule encoding a catalytic domain of an exo-<br>
cellobiohydrolase; and<br>
c.	a DNA molecule encoding an endoglucanase catalytic domain.<br>
2.	The heterologous exo-endo cellulase fusion construct according to Claim 1<br>
further comprising a linker sequence located 3' of the catalytic domain of the exo-<br>
cellobiohydrolase and 5' of the catalytic domain of the endoglucanase.<br>
3.	The heterologous exo-endo cellulase fusion construct according to Claim 1<br>
further comprising a kexin site located after the linker sequence and before the coding<br>
region of the endoglucanase catalytic domain.<br>
4.	The heterologous exo-endo fusion construct according to Claim 1 further<br>
comprising a promoter of a filamentous fungus secretable protein, said promoter located<br>
in operable linkage 5' of the coding region of the exo-cellobiohydrolase catalytic domain.<br>
5.	The heterologous exo-endo fusion construct according to Claim 4 wherein the<br>
promoter is a cbh promoter.<br>
6.	The heterologous exo-endo fusion construct according to Claim 5 wherein the<br>
promoter is a cbhl promoter derived from 7. reesei.<br>
7.	The heterologous exo-endo fusion construct according to Claim 1 wherein the<br>
exo-cellobiohydrolase is a CBH1.<br>
8.	The heterologous exo-endo fusion construct according to Claim 7 wherein said<br>
CBH1 comprises an amino acid sequence of at least 90% sequence identity with the<br>
sequence set forth in SEQ ID NO.: 6.<br>
9.	The heterologous exo-endo fusion construct according to Claim 1 wherein the<br>
endoglucanase catalytic domain is derived from a bacterial endoglucanase.<br>
10.	The heterologous exo-endo fusion construct according to Claim 9 wherein the<br>
bacterial endoglucanase catalytic domain is selected from the group consisting of an<br>
Acidothermus cellulolyticus GH5A endoglucanase I (E1) catalytic domain; an<br>
Acidothermus cellulolyticus GH74 endoglucanase (GH74-EG) catalytic domain: and a<br>
Thermobifida fusca E5 endoglucanase (Tf-E5) catalytic domain.<br>
11.	The heterologous exo-endo fusion construct according to Claim 10 wherein<br>
the endoglucanase is an Acidothermus cellulolyticus GH5A E1 catalytic domain.<br>
12.	The heterologous exo-endo fusion construct according to Claim 10 wherein<br>
the Acidothermus cellulolyticus GH5A E1 catalytic domain having an amino acid<br>
sequence of at least 90% sequence identity with the sequence set forth in SEQ ID NO. 8.<br>
13.	The heterologous exo-endo fusion construct according to Claim 1 further<br>
comprising a terminator sequence located 3' to the endoglucanase catalytic domain.<br>
14.	The heterologous exo-endo fusion construct according to Claim 1 further<br>
comprising a selectable marker.<br>
15.	A vector comprising in operable linkage a promoter of a filamentous fungus<br>
secretable protein, a DNA molecule encoding a signal sequence, a DNA molecule<br>
encoding a catalytic domain of a fungal exo-cellobiohydrolase, a DNA molecule encoding<br>
a catalytic domain of an endoglucanase, and a terminator.<br>
16.	The vector according to Claim 15 further comprising a selectable marker.<br>
17.	The vector according to Claim 15 further comprising a linker located 3' of the<br>
exo-cellobiohydrolase (CBH) catalytic domain and 5' of the EG catalytic domain.<br>
18.	The vector according to Claim 15 further comprising a kexin site.<br>
19.	The vector according to Claim 15 wherein the catalytic domain of the<br>
endoglucanase is derived from a bacterial endoglucanase.<br>
20.	A fungal host cell transformed with a heterologous exo-endo cellulase fusion<br>
construct according to Claim 1.<br>
21.	A fungal host cell transformed with a vector according to Claim 15.<br>
22.	A recombinant fungal cell comprising the heterologous exo-endo cellulase<br>
fusion construct according to Claim 1.<br>
23.	A recombinant fungal cell comprising a vector according to Claim 15.<br>
24.	The recombinant fungal cell according to Claim 22 wherein the fungal host<br>
cell is a Trichoderma host cell.<br>
25.	The recombinant fungal cell according to Claim 22 wherein the fungal host<br>
cell is a strain of T. reesei.<br>
26.	The recombinant fungal cell according to Claim 22 wherein at least one gene<br>
selected from the group consisting of the cbh1, cbh2, eg/1 and eg/2 has been deleted<br>
from the fungal cells.<br>
27.	An isolated cellulase fusion protein having cellulolytic activity which<br>
comprises an exo-cellobiohydrolase catalytic domain and an endoglucanase catalytic<br>
domain.<br>
28.	The isolated cellulase fusion protein according to Claim 27 wherein the exo-<br>
cellobiohydrolase is a CBH1.<br>
29.	The isolated cellulase fusion protein according to Claim 27 wherein the<br>
catalytic domain of the endoglucanase is derived from a bacterial cell.<br>
30.	The isolated cellulase fusion protein according to Claim 29 wherein bacterial<br>
cell is a strain of Acidothermus cellulolyticus.<br>
31.	A cellulolytic composition comprising the isolated cellulase fusion protein<br>
according to Claim 29.<br>
32.	A method of producing an enzyme having cellulolytic activity comprising, a)<br>
stably transforming a filamentous fungal host cell with a heterologous exo-endo cellulase<br>
fusion construct according to Claim 1; b) cultivating the transformed fungal host cell under<br>
conditions suitable for said fungal host cell to produce an enzyme having cellulolytic<br>
activity; and c) recovering said enzyme.<br>
33.	The method according to Claim 32 wherein the filamentous fungal host cell is<br>
a Trichoderma cell.<br>
34.	The method according to Claim 32 wherein the filamentous fungal host cell is<br>
a T. reesei host cell.<br>
35.	The method according to Claim 32 wherein the exo-cellobiohydrolase is a<br>
CBH1 and the endoglucanase is selected from the group consisting of an Acidothermus<br>
cellulolyticus endoglucanase and a Thermobifida fusca endoglucanase.<br>
36.	The method according to Claim 32 wherein the recovered enzyme is selected<br>
from the group consisting of a cellulase fusion protein, components of the cellulase fusion<br>
protein, and a combination of the cellulase fusion protein and the components thereof.<br>
37.	The method according to Claim 32 wherein the recovered enzyme(s) is<br>
purified.<br>
38.	A Trichoderma host cell which expresses a cellulase fusion protein, wherein<br>
said fusion protein comprises a catalytic domain of an exo-cellobiohydrolase and a<br>
catalytic domain of an endoglucanase.<br>
39.	The Trichoderma host cell according to Claim 38 wherein the host cell is a T.<br>
reesei cell.<br>
40.	The Trichoderma host cell according to Claim 38 wherein the exo-<br>
cellobiohydrolase is a CBH1 and the endoglucanase is an Acidothermus cellulolyticus<br>
endoglucanase.<br>
41.	The Trichoderma host cell according to Claim 38 wherein the endoglucanase<br>
is either an Acidothermus cellulolyticus E1 or GH74 endoglucanase.<br>
42.	The Trichoderma host cell according to Claim 38 wherein at least one gene<br>
selected from the group consisting of the cbh1, cbh2, eg/7 and eg/2 has been deleted<br>
from the host cell.<br>
43.	A fungal cellulase composition comprising a cellulase fusion protein or<br>
components thereof, wherein the fusion protein or components thereof is the product of a<br>
recombinant Trichoderma spp. according to Claim 38.<br>
44.	A fungal cellulase composition according to Claim 43 wherein the cellulase<br>
fusion protein is a CBH1-Acidothermus cellulolyticus E1 fusion protein and the<br>
components are the cleaved products, CBH1 and Acidothermus E1.<br></www.ncbi.nlm.nih.gov></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUFic3RyYWN0LSgwNS0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Abstract-(05-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUFic3RyYWN0LSgxOC0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Abstract-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNsYWltcy0oMDUtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Claims-(05-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNsYWltcy0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Claims-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Correspondence Others-(05-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDgtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Correspondence Others-(08-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4824-DELNP-2006-Correspondence Others-(17-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMjUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Correspondence-others (25-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Correspondence-Others-(12-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Correspondence-Others-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDA1LTA5LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Description (Complete)-(05-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LURyYXdpbmdzLSgwNS0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Drawings-(05-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUZvcm0tMS0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Form-1-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUZvcm0tMTggKDI1LTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Form-18 (25-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUZvcm0tMi0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Form-2-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1ERUxOUC0yMDA2LUZvcm0tMy0oMDgtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4824-DELNP-2006-Form-3-(08-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUZvcm0tMy0oMTItMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Form-3-(12-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1ERUxOUC0yMDA2LUZvcm0tMy0oMTctMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4824-DELNP-2006-Form-3-(17-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LUdQQS0oMTItMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-GPA-(12-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMDUtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Petition-137-(05-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgyNC1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMTItMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4824-delnp-2006-Petition-137-(12-07-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258383-polymerization-catalyst-comprising-solid-catalyst-particles-and-method-of-preparing-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258385-cap-for-drug-delivery-devices.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258384</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4824/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Aug-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENENCOR INTERNATIONAL INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BOWER BENJAMIN S.</td>
											<td>8479 MAYHEWS LANDING ROAD, NEWARK, CA 94560, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LARENAS EDMUND A.</td>
											<td>301 NEVADA, MOSS BEACH, 5 94038, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MITCHINSON COLIN</td>
											<td>381 MYRTLE STREET, HALF MOON BAY , 5 94019, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/56</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/010158</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-03-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/556,598</td>
									<td>2004-03-25</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258384-a-heterologous-exo-endo-cellulase-fusion-construct-and-fusion-protein-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:45:57 GMT -->
</html>
