<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258396-methods-and-compositions-for-analyzing-ahasl-genes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:43:32 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258396:METHODS AND COMPOSITIONS FOR ANALYZING AHASL GENES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS AND COMPOSITIONS FOR ANALYZING AHASL GENES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to methods and compositions for analyzing plant acetohydroxy acid synthase large subunit (AHASL) genes. In particular, the invention relates to methods for the detection of wild-type AHASL alleles and mutant AHASL alleles that encode imidazolinone-tolerant AHASL proteins. The methods involve the use of PCR amplification and novel compositions comprising allele-specific and gene-specific primers to detect the presence of mutant and/or wild-type alleles present at the individual AHASL genes of a plant. Specifically, the methods and compositions are useful for analyzing the three AHASL genes of Triticum aestivum and the two AHASL genes of Triticum turgidum ssp. durum.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHODS AND COMPOSITIONS FOR ANALYZING AHASL GENES<br>
FIELD OF THE INVENTION<br>
This invention relates to the field of gene analysis, particularly to novel methods for the identification of wild-type and herbicide-tolerant alleles of plant AHASL genes.<br>
BACKGROUND OF THE INVENTION<br>
Acetohydroxyacid synthase (AHAS; EC 4.1.3.18, also known as acetolactate synthase or ALS), is the first enzyme that catalyzes the biochemical synthesis of the branched-chain amino acids valine, leucine and isoleucine (Singh (1999) "Biosynthesis of valine, leucine and isoleucine," in Plant Amino Acids, Singh, B.K., ed., Marcel Dekker Inc. New York, New York, pp. 227-247). AHAS is the site of action of four structurally diverse herbicide families including the suifonylureas (LaRossa and Falco (1984) Trends Bio-technol. 2:158-161), the imidazolinones (Shaner et al. (1984) Plant Physiol. 76:545-546), the triazolopyrimidines (Subramanian and Gerwick (1989) "Inhibition of acetolactate synthase by triazolopyrimidines," in Biocatalysis in Agricultural Biotechnology, tyVhitaker, J.R. and Sonnet, P.E. eds., ACS Symposium Series, American Chemical Society, Washington, D.C., pp. 277-288), and the pyrimidyloxybenzoates (Subramanian et al. (1990) Plant Physiol. 94: 239-244.). Imidazolinoffe and sulfonylurea herbicides are widely used in modem agriculture due to their effectiveness at very low application rates and relative non-toxicfty in animals. By inhibiting AHAS activity, these families of herbicides prevent further growth and development of susceptible plants including many weed species. Several examples of commercially available imidazolinone herbicides are PURSUIT® (imazethapyr), SCEPTER® (imazaquin) and ARSENAL® (imazapyr). Examples of sulfonylurea herbicides are chlorsulfuron, rnetsulfuron methyl, sulfometuron methyl, chlorimuron ethyl, thifensulfuron methyl, tribenuron methyl, ben-sulfuron methyl, nicosulfuron, ethametsutfuron methyl, rimsulfuron, triflusuifuron methyl, triasulfuron, primisulfuron methyl, cinosulfuron, amidosulfiuon, fluzasulfuron, imazosulfuron, pyrazosulfuron ethyl and halosulfuron.<br>
Due to their high effectiveness and low-toxicity, imidazolinone herbicides are favored for application by spraying over the top of a wide area of vegetation. The ability to spray an herbicide over the top of a wide range of vegetation decreases the costs associated with plantation establishment and maintenance, and decreases the need for site preparation prior to use of such chemicals. Spraying over the top of a desired tolerant species also results in the ability to achieve maximum yield potential of the desired species due to the absence of competitive species. However, the ability to use such spray-over techniques is dependent upon the presence of imidazoiinone-resistant species of the desired vegetation in the spray over area.<br><br>
Among the major agricultural crops, some leguminous species such as soybean are naturally resistant to imidazolinone herbicides due to their ability to rapidly metabolize the herbicide compounds (Shanerand Robinson (1985) Weed Sol. 33:469-471). Other crops such as corn (Newhouse etaL (1992) Plant PhysioL 100:882886) and rice (Barrett et a/. (1989) Crop Safeners for Herbicides, Academic Press, New York, pp. 195-220) are somewhat susceptible to imidazolinone herbicides. The differential sensitivity to the imidazoiinone herbicides is dependent on the chemical nature of the particular herbicide and differential metabolism of the compound from a toxic to a non-toxic form in each plant (Shaner et al (1984) Plant Physiol. 76:545-546; Brown et a/., (1987) Pestle. Biochem. PhysioL 27:24-29). Other plant physiological differences such as absorption and translocation also play an important role in sensitivity (Shaner and Robinson (1985) WeedSci. 33:46^471).<br>
Plants resistant to jmidazolinones, sulfonylureas and triazolopyrimidines have been successfully produced using seed, microspore, pollen, and callus mutagenesis in Zea mays, Arabidopsis thaliana, Brassica napus, Glycine max, and Nicotiana tabacum (Sebastian et al (1989) Crop ScL 29:1403-1408; Swanson et a/., 1989 Theor. Appl. Genet 78:525-530; Newhouse etaL (1991) Theor. AppL Genet 83:65-70; Sathasivan et al. (1991) Plant PhysioL 97:1044-1050; Mourand etaL (1993) J. Heredity 84:91-96). In all cases, a single, partially dominant nuclear gene conferred resistance. Four imidazolinone resistant wheat plants were also previously isolated following seed mutagenesis of Triticum aestivum L cv. Fide! (Newhouse etaL (1992) Plant PhysioL 100:882-886). Inheritance studies confirmed that a single, partially dominant gene conferred resistance. Based on allelic studies, the authors concluded that the mutations in the four identified lines were located at the same locus. One of the Fidel cultivar resistance genes was designated FS-4 (Newhouse et al. (1992) Plant PhysioL 100:882-886). Computer-based modeling of the three dimensional conformation of the AHAS-inhibitor complex predicts several amino acids in the proposed inhibitor binding pocket as sites where induced mutations would likely confer selective resistance to imidazolinones (Ott et aL (1996) J. Mot. Biol. 263:359-368). Wheat plants produced with some of these rationally designed mutations in the proposed binding sites of the AHAS enzyme have in fact exhibited specific resistance to a single class of herbicides (Ott et aL (1996) MoL Biol. 263:359-368).<br>
Plant resistance to imidazolinone herbicides has also been reported in a number of patents. U.S. Patent Nos. 4,761,373, 5,331,107, 5,304,732,6,211,438, 6,211,439 and 6,222,100 generally describe the use of an altered AHAS gene to elicit herbicide resistance in plants, and specifically discloses certain imidazolinone resistant com lines. U.S. Patent No. 5,013,659 discloses plants exhibiting herbicide resistance due to mutations in at least one amino acid in one or more conserved regions. The mutations described therein encode either cross-resistance for imidazolinones and sulfonylureas or sulfonyiurea-specific resistance, but imidazolinone-specific resistance is not described. Additionally, U.S. Patent No. 5,731,180 and U.S. Patent No. 5,767,361 discuss an iso-<br><br>
lated gene having a single amino acid substitution in a wild-type monocot AHAS amino acid sequence that results in imidazolinone-specific resistance. In plants, as in all other organisms examined, the AHAS enzyme is comprised of two subunits: a large subunlt (catalytic role) and a small subunit (regulatory role) (Dugg)eby and Pang (2000) J, Biochem. Mol. B/o/. 33:1-36). The large subunit (termed AHASL) may be encoded by a single gene as in the case of Arabidopsis and rice or by multiple gene family members as in maize, canola, and cotton. Specific, single-nucleotide substitutions in the large subunit confer upon the enzyme a degree of insensitivity to one or more classes of herbicides (Chang and Duggleby (1998) Biochem J. 333:765-777).<br>
Mutations in genes encoding the AHAS large subunit, which are referred to herein as AHASL genes, are the molecular basis of herbicide tolerance in CLEARFIELD® crops which have increased tolerance to imidazolinone herbicides. Because each of these mutations results in a semi-dominant phenotype one mutation in a heterozygous state may be sufficient to produce a level Qf herbicide tolerance that is sufficient for many crop productions systems. However, for particular herbicide applications, and in cases with crop plants having multiple AHASL genes such as wheat, combinations of mutations are desired to achieve an increased level of resistance to herbicides. For example, bread wheat, Triticum aestivum L, contains three homoeologous acetohydroxyacid synthase large subunit genes. Each of the genes exhibit significant expression based on herbicide response and biochemical data from mutants in each of the three genes (Ascenzi etaL (2003) International Society of Plant Molecular Biologists Congress, Barcelona, Spain, Ref. No, S10-17). The coding sequences of all three genes share extensive homology at the nucleotide level (WO 03/014357). Through sequencing the AHASL genes from several varieties of Triticum aestivum, the molecular basis of herbicide tolerance in most imidazolinone (IMI)-tolerant lines was found to be the mutation S653(40N, indicating a serine to asparagine substitution at a position equivalent to the serine at amino acid 653 in Arabidopsis thaliana (WO 03/01436; WO 03/014357). The S653(&gt;W)N mutation in each gene is shaded in Figures 1A, 1B, and 1C. This mutation is due to a single nucleotide polymorphism (SNP) in the DNA sequence encoding the AHASL protein.<br>
One goal of plant breeders is to introduce imidazolinone tolerance into existing wheat lines by inducing the S653(40N mutation in the existing lines or by crossing non-IMI-tolerant lines with IMI-tolerant lines following by backcrossing and selection for imidazoiinone tolerance. Another goal of plant breeders is to produce wheat plants with increased levels of imidazolinone tolerance, beyond the levels of tolerance seen in wheat plants possessing a single S653(AQN mutation in a single wheat AHASL gene. Thus, it is desirable to breed wheat plants that possess combinations of S65304£)N mutations at two or more of the AHASL genes. In addition, it is also desirable to breed wheat plants that are homozygous for the mutant S653(i4f)N allele at one or more of the AHASL genes. However, to develop the desired wheat plants, rapid methods for identifying the desired plants are needed. Existing methods of detecting wheat plants<br><br>
with imidazolinone tolerance are not well suited for use in the development of plants that possess more than a single S653(At)N allele at a single AHASL gene. Existing methods of identifying plants with enhanced imidazolinone tolerance include field or greenhouse herbicide spray tests and biochemical assays for AHAS activity. Such methods are time consuming, however, and generally not suited for distinguishing, among large numbers of individual plants, subtle increases in imidazolinone tolerance that may occur when a second S653(Af)N allele is introduced into a wheat plant<br>
Alternative methods for identifying desired plants include DNA-based methods. For example, the AHASL genes, or portions thereof, can be amplified from genomic DNA by polymerase chain reaction (PCR) methods and the resulting amplified AHASL gene or portion thereof can be sequenced to identify the mutant S653(Af)N allele and the particular AHASL gene that it is present in. However, such a DNA-sequencing-based method is not practical for large numbers of samples. Another approach involves that use of radiolabejled or non-isotopicaliy tagged, allele-speciflc oligonucleotides (ASOs) as probes for dot blots of genomic DNA or polymerase chain reaction (PCR) amplified DNA (Connor et a/. (1983) Proa Natl. Acad. SoL USA 80:276-282; Orkin et a/. (1983) J. Clin. Invest 71:775-779; Brun et a/. (1988) NucL Acids Res, 16:352; and Bugawan et al. (1988) Biotechnology 6:943-947. While such an approach is useful for distinguishing between two alleles at a single locus, this approach is not useful for the wheat AHASL genes, because three AHASL genes are nearly identical (Figure 1) in region surround the SNP that gives rise to the mutant S653(/4f)N AHASL protein. Thus, a set of six oligonucieotide probes could not be developed that would be able to distinguish between the mutant and wild-type alleles at each of the three wheat AHASL genes.<br>
One method that can be adapted for rapidly screening large numbers of individuals for the analysis of an SNP is the amplification refractory mutation system (ARMS) (Newton et al. (1989) NucL Acids Res. 17:2503-2516). This PCR-based method can be used to distinguish two alleles of a gene that differ by a single nudeotide and can also be used to distinguish heterozygotes from homozygotes for either ailele by inspection of the PCR products after agarose gel electrophoresis and ethidium-bromide staining. The ARMS method is based on the premise that oligonucleotides with a mismatched 31-residue will not function as primers in PCR under the appropriate conditions (Newton et al. (1989) NucL Acids Res. 17:2503-2516). While this method has proven useful for the analysis of an SNP at a single gene, whether this method, or a other PCR-based methods, can be used be used for the analysis of the SNP that gives rise to the S653(/tf)N mutation in each of the three wheat AHASL genes has not been reported.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides methods for analyzing plant AHASL genes. The methods are directed to detecting in samples comprising plant genomic DNA the presence<br><br>
of the wild-type allele and/or a mutant allele at each of the AHASL genes in a plant's genome. The mutant AHASL alleles of the invention encode imidazolinone-tolerant AHASL proteins comprising the S653(At)N substitution. At the DNA-Ievel, the mutant allele results from a G-to-A transition at the position that corresponds to nucleotide 1958 oi the Arabidopsis AHASL nucleotide sequence set forth in EMBL Accession No. X51514. The methods are particularly directed to analyzing the AHASL genes of plants that comprise two or more AHASL genes, including, but not limited to, Triticum aestivum and Triticum turgidum $$p, durum.<br>
In a first aspect, the invention provides a method for detecting a mutant allele of an AHASL gene that confers on a plant tolerance to imidazolinone herbicides. The method comprises the steps of: (a) obtaining genomic DNA from a plant, particularly a wheat plant; (b) using the DNA as a template for a PCR amplification comprising the DNA, polymerase, deoxyribonucleotide triphosphates, a forward AH4SL-gene-specific primer, a reverse AH4SL-gene-specific primer, and a mutant-allele-specrfic primer, and (c) detecting the products of the PCR amplification. The mutant-allele-specrfic primer comprises a nucleotide sequence with a 51 end and a 31 end, wherein the nucleotide sequence corresponds to the coding strand of an AHASL gene, the 3" end nucleotide corresponds to the site of the G-to-A point mutation, and the 31 end nucleotide is cytidi-ne. In addition the mutant-allele-specific primer is capable of annealing to a region of ' an AHASL gene that is nested between the annealing sites of the forward and reverse AH4SL-gene-specific primers. In an embodiment of the invention, the mutant-allele-specific primer comprises a nucleotide sequence with a 5' end and a 31 end, wherein the nucleotide sequence is capable of annealing to the complement of nucleotides 3 to 23 of SEQ ID NO; 12 and has a cytidine at the 3' end.<br>
In a second aspect, the invention provides a method for analysis of a plant AHASL gene, particularly an AHASL gene selected from the group consisting of AHASLID, AHASL1B, and AHASL1A of Triticum aestivum and the AHASL1B and AHASL1A of Triticum turgidum ssp. durum. The method comprises the steps of: (a) obtaining genomic DNA from a plant, particularly a wheat plant; (b) using the DNA as a template for a first PCR amplification comprising said DNA, polymerase, deoxyribonucleotide triphosphates, a forward AHASL-gerne-specificif primer, a reverse AHASL-gene-specific primer, and a mutant-allele-specific primer as described above; (c) using the DNA as a template for a second PCR amplification comprising said DNA, polymerase, deoxyribonucleotide triphosphates, the forward AHtASL-gene-specific primer, the reverse AHASL-gene-specific primer, and a wild-type-aliele-specific primer; and (d) detecting the products of said first and said second PCR amplifications. The wild-type-allele-specific primer comprises a nucleotide sequence with a 51 end and a 3' end, wherein the nucleotide sequence corresponds to the coding strand of an AHASL gene, the 31 end nucleotide corresponds to the site of the G-to-A point mutation, and the 31 end nucleotide is guanosine. The wild-type-allele-specific primers of the invention are capable of annealing to a region of an AHASL gene that Is nested between the annealing sites of the forward and reverse AHASL-gene-specific primers. In another embodiment of<br><br>
the invention, the wild-type-allele-specific primer comprises a first nucleotide sequence with a 5! end and a 31 end, wherein the first nucleotide sequence is capable of annealing to the complement of nucleotides 4 to 23 of SEQ ID NO: 10 and has a guanosine at the 31 end.<br>
In a third aspect, the invention provides a method for analysis of an AHASL gene, involving an initial PCR amplification of at least a fragment of each of the AHASL genes in a plant, so as to enrich a sample of genomic DNA for the AHASL genes or fragments thereof. The steps of the method comprise: (a) obtaining genomic DNA from a plant, particularly a wheat plant; (b) using the DNA as a template in a pre-amplification comprising the DNA, deoxyribonucleotide triphosphates, polymerase, a forward AHASL primer, and a reverse AHASL primer, so as to produce pre-amplified DNA; (c) using the pre-amplified DNA as a template for a first PCR amplification comprising the pre-amplified DNA, polymerase, deoxyribonucleotide triphosphates, a forward AHASL-gene-specific primer, a reverse AHASL-gene-specific primer, arid a mutant-allele-specific primer as described above; (d) using the pre-amplified DNA as a template for a second PCR amplification comprising the pre-amplified DNA, polymerase, deoxyribonucleotide triphosphates, said forward 4H4SL-gene-specific primer, the reverse AH4SL-gene-specific primer, and a wild-type-allele-specific primer as described above; and (e) detecting the products of said first and said second PCR amplifications. The mutant-allele-specific primer and the wild-type-allele-specific primer are capable of annealing to a region of an AHASL gene that is nested between the annealing sites of the forward and reverse y4H4SL-gene-specific primers. In addition, the forward and reverse &gt;4rtASL-gene-specific primers are nested between the annealing sites of the forward and reverse AHASL primers.<br>
The present invention provides the oligonucleotide primers that are described above. These primers include, but are not limited to, a mutant-allele-specific primer comprising the nucleotide sequence set forth in SEQ ID NO: 3, the wild-type-allele-specific primer comprising the nucleotide sequence set forth in SEQ ID NO: 4, the forward AHASL-gene-specific primers comprising the nucleotide sequences set forth in SEQ ID NOS: 5 and 6, the reverse AHASL-gene-$peo\f\c primers comprising the nucleotide sequences set forth in SEQ ID NOS: 7, 8, and 9, the forward AHASL primer comprising the nucleotide sequence set forth in SEQ ID NO: 1, and the reverse AHASL primer comprising the nucleotide sequence set forth in SEQ ID NO: 2.<br>
The present invention also provides kits for performing the methods of the invention as described above. Such kits comprise at least one forward iAHASL-gene-specific primer, at least one reverse AHASL-gene-specific primer, a mutant-allele-specific primer, at least one polymerase enzyme capable of catalyzing the PCR amplification of a first fragment of a wheat AHASL gene and a second fragment of a wheat AHASL gene. The kits of the invention can further comprise any additional components that are required for the performing methods of the present invention as described herein.<br><br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1A depicts the nucleotide (SEQ ID NO: 10) and amino acid (SEQ ID NO: 11) sequences of the wild-type (WT) AHASL1D and nucleotide (SEQ ID NO: 12) and amino acid (SEQ ID NO: 13) sequences of the mutant (MUT) AHASL1D. The shaded region indicates the site of the mutation in the mutant sequences. Figure 1B depicts the nucieotide (SEQ ID NO: 14) and amino acid (SEQ ID NO: 15) sequences of the wild-type (WT) AHASL1B and nucleotide (SEQ ID NO: 16) and amino acid (SEQ ID NO: 17) sequences of the mutant (MUT) AHASL1B. The shaded region indicates the site of the mutation in the mutant sequences. Figure 1C depicts the nucleotide (SEQ ID NO: 18) and amino acid (SEQ ID NO: 19) sequences of the wild-type (WT) AHASL1A and nucleotide (SEQ ID NO: 20) and amino acid (SEQ ID NO: 21) sequences of the mutant (MUT) AHASL1A. The shaded region indicates the site of the mutation in the mutant sequences. Figure 2 is a schematic representation of one embodiment of the invention. Figure 3 is a partial alignment of the three wheat AHASL cDNA sequences with the forward and reverse AHASL primers (shaded).<br>
Figure 4 is a partial alignment of the three wheat AHASL cDNA sequences with the wild-type-allele specific, mutant-ailele-specific, forward ^H4SL-gene-specific and reverse AH/ASL-gene-specific primers. The forward &gt;4/i4SL-gene-specific primers are shown in light shading. The reverse 4HASL-gene-specific primers are shown in dark shading. The wild-type-allele specific and mutant-allele-specific primers are in italics and boldface type, respectively.<br>
Figure 5 is a negative photographic image of a single UV-transilluminated, ethidium-bromide-treated agarose gel. Four wheat genotypes were analyzed by all six assays as described in Example 3. The wheat genotypes tested are: Kirschauff (indicated as "wild-type" in Figure 5), CV9804, G-208, and K-42. Of the four genotypes, CV9804, G-208, and K-42 possess the IMI-tolerance trait. Above the? bands the reaction is denoted by a capital letter (above the panels) and by a lower-case letter within the panels. For example the left most pair of bands within each panel is the assay for AHASL1D (D), wild-type allele (w)( while the right-most lanes are AHASL1A (A) mutant-allele (m) reactions. The center two lanes in each panel are represent the result of the wild-type allele (w) and the mutant-allele (m) assays for AHASL1B (B). In each lane, the control band is the upper band (longer fragment) and the diagnostic band is the lower (shorter fragment).<br>
Figure 6 is a table of percent nucleotide sequence identities from pairwise comparisons of the wheat AHASL gene coding sequences. Hexaploid L1D, Hexaploid L1B, and Hexaploid L1A denote the AHASL1D, AHASL1B, and AHASL1A genes, respectively, from Tritioum aestivum. Tetraplold L1B, and Tetraploid L1A denote the AHASL1B and AHASL1A genes, respectively, from 7. Turgidum ssp. durum.<br><br>
SEQUENCE LISTING<br>
The nucleic acid and amino acid sequences listed in the accompanying sequence listing are shown using standard letter abbreviations for nucleotide bases, and three-letter code for amino acids. The nucleic acid sequences follow the standard convention of beginning at the 51 end of the sequence and proceeding forward (i.e.,, from left to right in each line) to the 3' end. Only one strand of each nucleic acid sequence is shown, but the complementary strand is understood to be included by any reference to the displayed strand. The amino acid sequences follow the standard convention of beginning at the amino terminus of the sequence and proceeding forward (/.a, from left to right in each line) to the carboxy terminus.<br>
SEQ ID NO: 1 sets forth the nucleotide sequence of the forward AHASL primer, which<br>
is also referred to herein as CM-F.<br>
SEQ ID NO: 2 sets forth the nucieotide sequence of the reverse AHASL primer, which<br>
is also referred to herein as CM-R.<br>
SEQ ID NO: 3 sets forth the nucleotide sequence of the mutant-allele-specific primer,<br>
which is also referred to herein as MU-F.	:<br>
SEQ ID NO: 4 sets forth the nucleotide sequence of the wild-type-allele-specific primer,<br>
which is also referred to herein as WT-F.	.<br>
SEQ ID NO: 5 sets forth the nucleotide sequence of a forward. y4K4SZ.-gene-sperific<br>
primer, which is also referred to herein as 1A.D-F.<br>
SEQ ID NO: 6 sets forth the nucleotide sequence of a forward ArtASL^gene-specific<br>
primer, which is also referred to herein as 1B-F.<br>
SEQ ID NO: 7 sets forth the nucleotide sequence of a reverse ^AHASL-gene-specific<br>
primer, which is also referred to herein as 1D-R.<br>
SEQ ID NO: 8 sets forth the nucleotide sequence of a reverse ^HASL-gene-specific<br>
primer, which is also referred to herein as 1B-R.<br>
SEQ ID NO: 9 sets forth the nucleotide sequence of a reverse y4H4SL-gene-specific<br>
primer, which is also referred to herein as 1A-R.<br>
SEQ ID NO: 10 sets forth the nucleotide sequence of a portion of the wild-type allele of<br>
AHASL1D that is depicted in Figure 1A.<br>
SEQ ID NO: 11 sets forth the amino acid sequence encoded by the portion of the<br>
nucleotide sequence of the wild-type allele olAHASUD that is depicted in Figure 1A<br>
SEQ ID NO: 12 sets forth the nucleotide sequence of a portion of the mutant allele of<br>
AHASL1D that is depicted in Figure 1A.<br>
SEQ ID NO: 13 sets forth the amino acid sequence encoded by the portion of the<br>
nucleotide sequence of the wild-type allele of AHASL1D that is depicted in Figure 1A.<br>
SEQ ID NO: 14 sets forth the nucleotide sequence of a portion of the wild-type allele of<br>
AHASL1B that is depicted in Figure 1B.<br>
SEQ ID NO: 15 sets forth the amino acid sequence encoded by the portion of the<br>
nucleotide sequence of the wild-type allele olAHASLIB that is depicted in Figure 1B.<br><br>
SEQ ID NO: 16 sets forth the nucleoticle sequence of a portion of the mutant allele of<br>
AHASL1B that is depicted in Figure 1B.<br>
SEQ ID NO: 17 sets forth the amino acid sequence encoded by the portion of the<br>
nucleotide sequence of the wild-type allele oiAHASUB that is depicted in Figure 1B.<br>
SEQ ID NO: 18 sets forth the nucleotide sequence of a portion of the wild-type allele of<br>
AHASL1A that is depicted in Figure 1C.<br>
SEQ ID NO: 19 sets forth the amino acid sequence encoded by the portion of the<br>
nucleotide sequence of the wild-type allele oiAHASUA that is depicted in Figure 1C.<br>
SEQ ID NO: 20 sets forth the nucleotide sequence of a portion of the mutant allele of<br>
AHASL1A that is depicted in Figure 1C.<br>
SEQ ID NO: 21 sets forth the amino acid sequence encoded by the portion of the<br>
nucleotide sequence of the wild-type allele of AHASL1A that is depicted in Figure 1C,<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention is directed to rapid methods for analyzing the genomes of plants, particularly for analyzing the AHASL genes therein. The methods of the invention find particular use in analyzing the AHASL genes of plants with multiple AHASL genes such as, for example, bread wheat, Triticum aestivum Lr and durum wheat, T. Turgidum ssp. durum. 7. aestivum comprises in its hexaploid genome three highly similar, but distinct, AHASL genes, designated as AHASL1D, AHASL1B, and AHASL1A. In contrast, T. turgidum ssp. durum comprises in its tetraploid genome two highly similar, but distinct, AHASL genes, designated as AHASL1B, and AHASL1A gene. While not identical, AHASL1B and AHASL1A of T. aestivum are closely related to AHASL1B and AHASL1A of T. turgidum ssp. durum as is apparent from sequence alignments and calculations of percentage sequence identity. Figure 6 provides nucleotide sequence identities from pairwise comparisons of the AHASL gene coding sequences of T. aestivum and T. turgidum ssp, durum.<br>
The methods of the invention involve determining whether a mutant allele is present at one or more of the AHASL genes in the genome of a plant. The mutant alleles comprise nucleotide sequences that encode imidazolinone-tolerant AHASL proteins comprising the S653(,4f)N substitution. The methods further involve determining whether there is present at one or more of the AHASL genes a wild-type AHASL allele. In fact, the methods of the invention allow for the determination of the zygosity of each of the AHASL genes in a plant For example, the methods of the invention can be used to rapidly determine—for each of the three AHASL genes in a wheat plant (or two AHASL genes in the case of durum wheat)—whether a wheat plant is homozygous for the mutant AHASL allele, heterozygous, or homozygous for the wild-type allele. Thus, the methods of the invention find use in breeding programs for the production of imidazolinone-tolerant wheat plants having one, two, three, four, five, or six mutant AHASL alleles in their genomes. The following terms used herein are defined below.<br><br>
A "primer11 is a single-stranded oligonucleotide, having a 31 end and a 5' end, that is capable of annealing to an annealing site on a target DNA strand, and the primer serves as an initiation point for DNA synthesis by a DNA polymerase, particularly in a PCR amplification, Such a primer may or may not be fully complementary to its annealing site on the target DNA<br>
An "annealing" site on a strand of a target DNA is the site to which a primer is capable of annealing in the methods of the present invention.<br>
Generally for the amplification of a fragment of a gene by PCR, a pair of primers that anneal to opposite strands of a double-stranded DNA molecule are employed. By standard convention, the "forward primer" anneals to the non-coding strand of the gene and the "reverse primer" primer anneals to the coding strand. A "nested primer" is a primer that has an annealing site that lies between a region of DNA that is amplified by a particular pair of forward and reverse primers. A "nested PCR" is amplification of a smaller fragment that is a portion of a larger fragment that was or is amplified by PCR. Generally, a first PCR is conducted with a particular pair of forward and reverse primers to produce a first fragment Then a second PCR is conducted with a nested forward and a nested reverse primer, using the first fragment as a template such that a second fragment is produced. However, in certain embodiments of the invention a forward primer, a reverse primer, and a nested primer are used in the same PCR amplification. The forward, primer and the nested primer anneal to the same strand of DNA and amplification of two fragments can occur.during PCR: a larger fragment resulting from amplification of the region of the DNA template between the annealing sites of the forward primer and the reverse primer and a smaller fragment resulting from amplification of the region of the DNA template between the annealing sites of the nested primer and the reverse primer.<br>
Throughout the specification, the terms "mutant allele," "mutant AHASL allele," or "mutant AHASL gene." Unless indicated otherwise herein, these terms refer to a polynu-cleotide that encodes an imidazolinone-tolerant AHASL protein comprising the S653(Af)N substitution. This amino acid substitution is a result of a point mutation from G to A in the position that corresponds to nucleotide 1958 of the Arabidopsis AHASL nucleotide sequence set forth in EMBL Accession No. X51514, which is herein incorporated by reference. See, Sathasivan et a/. (1990) Nucl. Acids Res, 18:2188; herein incorporated by reference.<br>
In contrast, unless indicated otherwise, the terms "wild-type allele," "wild-type AHASL allele," or "wild-type AHASL gene" allele refer to a polynucleotide that encodes an AHASL protein that lacks the S653(4f)N substitution. Such a "wild-type alleie/1 "wild-type AHASL allele," or "wild-type AHASL gene" may, or may not, comprise mutations, other than the mutation that causes the S653(Af)N substitution. The invention involves the use of a number of primers for PCR amplification. These primers are described in detail below.<br><br>
A "mutant-allele-spetific primer" is a primer that can be used in the methods of the invention for the PCR amplification of a fragment of a mutant AHASL allele but allows relatively little or no amplification of a wild-tyge AHASL allele or portion thereof. A "wiid-type-allele-specific primer11 is a primer that can be used in the methods of the invention for the PCR amplification of a fragment of a wild-type AHASL allele but allows relatively little or no amplification of a mutant AHASL allele or portion thereof. A "forward AHASL-gene-specific primer" of the invention and a "reverse AHASL-gene-specific primer" of the invention are used as a primer pair in the amplification of a fragment of a particular AHASL gene, such as, for example, AHASL1D, AHASL1B, and AHASL1A of T. aestivum and AHASL1B, and AHASL1A of T. turgidum ssp. durum. The forward AHASL-gene-specific primer of the invention and the reverse AHASL-gene-specific primer of the invention include any pair of primers that can be used to produce a gene-specific amplification product, or fragment, that includes the site of the SNP that gives rise to the mutant AHASL allele. For example, the primer pair of SEQ ID NO: 6 (forward AHASL1B specific primer) and SEQ ID NO: 8 (reverse AHASL1B)t when used in the methods of the present invention, is capable of specifically amplifying a fragment of wheat AHASL1Bt but not a fragment of either wheat AHASL1D or AHASL1A. Thus, the invention does not depend on a particular forward AHASL-gene-specific primer or a reverse AHA SL-gene-speciflc primer." Rather, a primer pair consisting of a particular forward AHASL-gene-specific primer and a particular reverse<br>
-   AHASL-gene-specific primer is capable of amplifying a fragment of only one specific AHASL gene from a plant. Those of ordinary skill in the art will recognize that numerous forward and reverse AHASL-gene-specific primers can be designed for use in the methods of the present invention. The methods of the present invention encompass the use of all such forward and reverse AHASL-gene-specific primers. In addition, a reverse AHASL-gene-specific primer of the invention, when used in a PCR amplification together with either the wild-type-allele-specific primer of the invention or the mutant-allele-specific primer of invention, is capable of amplifying a fragment of only one specific AHASL gene from a plant. For example, the primer pair of SEQ ID NO: 3 (mutant-ailele-specific primer) and SEQ ID NO: 8 (reverse AHASL1B), when used in the methods of the present invention, is capable of specifically amplifying a fragment of the wheat AHASL1B gene, but not a fragment of the wheat AHASL1D or AHASL1A genes. Similarly, the primer pair of SEQ ID NO: 4 (wild-type-allele-specific primer) and SEQ ID NO: 8 (reverse AHASL1B), when used in the methods of the present invention, is capable of specifically amplifying a fragment of the wheat AHASL1B gene, but not a fragment of the wheat AHASL1D or AHASL1A genes. For the amplification of a fragment of the AHASL1D gene of 7". aestivum, a preferred forward AHASL-gene-specific primer has the nucleotide sequence set forth in SEQ ID NO: 5 and a reverse AHASL-gene-specific primer has the nucleotide sequence set<br>
'     forth in SEQ ID NO: 7.<br>
For the amplification of a fragment of the AHASL1B genes of both T. aestivum and T. turgidum ssp. durum, a preferred forward AHASL-gene-specific primer has the nucteo-<br><br>
tide sequence set forth in SEQ ID NO: 6 and a reverse &gt;4H4S/.-gene-specific primer has the nucleoticle sequence set forth in SEQ ID NO: 8.<br>
For the amplification of a fragment of the AHASL1A genes of both T. aestivum and 7. turgidum ssp. durum, a preferred forward AHASL-gene-specific primer has the nucleo-tide sequence set forth in SEQ ID NO: 5 and a reverse AHASL-gene-specific primer has the nucleotide sequence set forth in SEQ ID NO: 9. For plants with more than one AHASL gene in their genomes, a "forward AHASL primer11 of the invention and "reverse AHASL primer" of the invention" are used as a primer pair in the amplification of a fragment of each of the AHASL genes of a plant That is, the pair of the forward AHASL primer and the reverse AHASL primer are designed to not discriminate between the different AHASL genes in a in a plant and thus, are generic AHASL primers for a plant species of interest The amplification product or fragment that results from a PCR amplification of genomic DNA with the forward AHASL primer and the reverse AHASL primer includes the annealing sites of the forward and reverse ABASL-gene-specific primers described above. The particular PCR amplification of the invention that involves the use of the forward AHASL primer and the reverse AHASL primer is referred to herein as the pre-amplification step. In certain embodiments of the invention, the preramplification step is employed to enrich the genomic DNA with fragments comprising AHASL genes for use in subsequent PCR amplifications of the invention. The product of the pre-amplification step is referred to herein as pre-amplified DNA. Unless otherwise indicated herein, "polymerase" refers to a DNA polymerase, particularly a DNA polymerase that is suitable for use in one or more of the PCR amplifications of the present invention.<br>
The present invention involves three aspects. The first aspect of the invention comprises a first PCR amplification of genomic DNA or pre-amplified DNA, to detect a mutant AHASL allele from a plant The second aspect includes the first PCR amplification of the first aspect and adds a second PCR amplification of a genomic DNA7 or the pre-amplified DNA, to detect the wild-type AHASL allele from a plant The third aspect includes the first PCR amplification of the first aspect, the second PCR amplification of the second aspect, and adds a third PCR amplification to amplify the AHASL genes or fragments thereof using genomic DNA as the template source. This third PCR amplification, also referred to herein as pre-amplification, is performed prior to the first and second PCR amplifications. The product of the third PCR amplification, also referred to herein as pre-amplified DNA, is used as the DNA template source for the first and second PCR amplifications.<br>
The first aspect of the invention Is a method for detecting a mutant allele of an AHASL gene that confers tolerance to Imidazolinone herbicides on a plant plant The mutant allele of the invention encodes an imidazolinone-tolerant AHASL protein comprising the S653(&gt;4f)N substitution. The method is directed to determining whether there is a mutant AHASL allele present at a specific AHASL gene of a plant In an embodiment of the invention, the plant is 7, aestivum or 7 turgidum ssp, durum and the AHASL gene<br><br>
is selected from the group consisting of AHASL1D, AHASL1B, and AHASL1A of T. aestivum and the AHASL1B and AHASL1A of 7. turgidum ssp. durum. An important feature of the first aspect of the invention is the discovery that the mutant-allele-specific primer having the nucleotide sequence set forth in SEQ ID NO: 3, together with a reverse &gt;AH4SL-gene specific primer of the invention, can be used to selectively amplify by PCR a portion of the mutant AHASL allele, white allowing relatively little or no amplification of the corresponding region of the wild-type AHASL allele. Typically, primers that are developed for PCR methods for distinguishing between two alleles that vary by a single nucleotide substitution are designed such that the primers reflect the same polymorphism at their 3' end. The basis for this primer design is the premise that oligonucleotides with a mismatched 3' end-residue will not function as primers in PCR under the appropriate conditions (Newton et al (1989) NuoL Acids Res. 17:2503-2516; and Wu et at. (1989) Pmc. NatL Acad. Sci. USA 86: 2757-2760). In contrast, the nucleotide at the 3' end of the mutant allele-specific primer mismatches both the mutant and wild-type alleles at the site of the SNP. Primers that were designed to have a 3! end nucleotide that mismatches the wild-type wheat AHASL allele but matches the mutant allele at the site of SNP were unexpectedly found to allow the amplification of the both the mutant and wild-type alleles. Therefore, other primer designs were tested. The surprising result was that a primer such as the rnutant-allele-specific primer having the nucleotide sequences set forth SEQ ID NO: 3, which has a 31 end* residue that mismatches both the wild-type and mutant alleles at the site of the SNP, was capable of allowing the amplification of the mutant wheat AHASL allele while allowing little or no amplification of the wild-type wheat AHASL allele. The mutant-allele-specific primers of the invention are designed to anneal to the non-coding strand of at least one AHASL gene of interest in the region of the G-to-A point mutation that gives to the S653(At)N substitution in an AHASL protein. In particular, the mutant-allele-specific primers of the invention comprise a nucleotide sequence with a 5f end and a 3' end, wherein the nucleotide sequence corresponds, to the coding strand of an AHASL gene, the 3f end nucleotide corresponds to the site of the G-to-A point mutation, and the 31 end nucleotide is cytidine. Thus, such a primer comprises at least one nucleotide that is not identical to the corresponding region of the coding strand of the AHASL gene of interest<br>
For analyzing the AHASL genes of wheat, the invention provides a mutant allele-specific primer that is capable annealing to the mutant AHASL alleles for each of the wheat AHASL genes. The region of DNA in the vicinity of the SNP that results in the mutant AHASL allele of the invention is nearly identical across the three 7. aestivum AHASL genes and the two 7*. turgidum ssp. durum AHASL genes. Thus, the mutant-allele-specific primers of the invention are capable annealing to the mutant AHASL alleles for each of the three 7. aestivum AHASL genes and the two 7. turgidum ssp. durum AHASL genes. Such mutant-allele-specific primers of the invention comprise a nucleotide sequence with a 5! end and a 31 end, wherein the nucleotide sequence is capable of annealing to the complement of nucleotides 3 to 23 of SEQ ID NO: 12, has<br><br>
a cytidlne at the 3' end, and is capable of annealing to a region of an AHASL gene that is nested between the annealing sites of the forward and reverse AHASL-gene-specific primers of the invention. A preferred mutant-allele-specific primer of the invention has the nucleotide sequence set forth in SEQ ID NO: 3.<br>
The first PCR amplification comprises three primers, the mutant-allele-specific primer and a pair of primers designed to selectively amplify a specific AHASL gene in a plant. The primer pair comprises a forward AHASL-gene-specific primer and a reverse AHASL-gene-specific primer that are designed for the selective PCR amplification of a fragment of only one specific AHASL gene from a plant.<br>
Using the methods of the invention, there are two results from the first PCR amplification of genomic DNA from a plant The results can be detected by, for example, aga-rose gel electorphoresls of the PCR products followed by ethidium-bromide staining of the DNA in the gel and visualization in the presence of UV light. If the specific AHASL gene comprises at least one mutant allele, the first PCR amplification produces two DNA fragments, a larger fragment resulting from amplification of the region of the AHASL gene bounded by the annealing sites of the forward and reverse AHASL-gene-specific primers and a smaller fragment resulting from amplification of the region of the AHASL gene bounded by the annealing sites of the mutant-allele-specific primer and the reverse AHASL-gene-specific primer. If, however, the specific AHASL gene does not comprise at least one mutant allele, then the first PCR amplification produces only the larger fragment<br>
The second aspect of the invention is method for analysis of an AHASL gene. The invention is directed to determining whether there is a wild-type allele, a mutant allele, or both present at a specific AHASL gene of a plant. In an embodiment of the invention, the plant is T. aestivum or T. turgidum ssp. durum and the AHASL gene is selected from the group consisting o1AHASL1Dt AHASL1B, and AHASL1A of T. aestivum and the AHASL1B and AHASL1A of T, turgidum ssp. durum. The second aspect of the invention adds to the first PCR amplification of the first aspect a second PCR amplification involving the amplification of a portion of a wild-type AHASL allele. The second PCR involves the wild-type-allele-specific primer. The second aspect of the invention involves the use of a wild-type-allele-specific primer of the invention. A wild-type-allele-specific primer of the invention, together with a reverse AHASL-gene-specific primer of the invention, can be used to selectively amplify by PCR a portion of the wild-type AHASL allele, while allowing relatively little or no amplification of the corresponding region of a mutant AHASL allele. The wild-type-allele-specific primers of the invention are designed to anneal to the non-coding strand of at least one AHASL gene of interest in the region of the G-to-A point mutation that gives to the S653(At)N substitution in an AHASL protein. The wild-type-allele-specific primers comprise a nucleotide sequence with a 51 end and a 31 end, wherein the nucleotide sequence corresponds to the coding strand of an AHASL gene, the 31 end nucleotide corresponds to the site of the G-to-A point mutation, and the 31 end nucleotide is guanosine.<br><br>
For analyzing the AHASL genes of wheat, the invention provides a wild-type allele-specific primer that is capable annealing to the wild-type AHASL alleles for each of the wheat AHASL genes. Similar to the mutant-allele-specific primers of the invention, the wild-type-aliele-specific primers of the invention are not designed to distinguish among the individual wheat AHASL genes, and thus are capable annealing to the wild-type AHASL alleles for each of the three T. aestlvum AHASL genes and the two T. turgidum ssp. durum AHASL genes. Such wild-type-allele-specific primers of the invention comprise a nudeotide sequence with a 51 end and a 31 end, wherein the nucleotide sequence is capable of annealing to the complement of nucleotides 4 to 23 of SEQ ID NO: 10, has a guanosine at the 3' end, and is capable of annealing to a region of an AHASL gene that is nested between the annealing sites of the forward and reverse ^HASL-gene-specific primers of the invention. A preferred wild-type-aliele-specific primer of the invention has the nucleotide sequence set forth in SEQ ID NO; 4. Typically, in the method of the second aspect, the genomic DNA used in the second PCR amplification is from the same plant as the genomic DNA used in the first PCR amplification. Preferably, the first and second PCR amplifications each involve the use of a portion of the same isolated genomic DNA sample and that such a genomic DNA sample can be subjected to a pre-amplification step as described herein to enrich the genomic DNA sample for AHASL gene sequences or fragments thereof. It is recognized, however, that the methods of the second aspect can involve the use of genomic DNA isolated from a first plant for the first PCR amplification and the use of genomic DNA isolated from a second wheat plant for the second PCR amplification, if the first and second plants are known to be genetically identical.<br>
The second PCR amplification comprises three primers, the wild-type-allele-specific primer and a pair of primers designed to selectively amplify a specific AHASL gene in a plant. The primer pair comprises a forward yAHASL-gene-specific primer and a reverse AH/ASL-gene-specific primer that are designed for the selective PCR amplification of a fragment of only one specific AHASL gene from a plant. The specific AHASL gene is the same AHASL gene as in the first PCR amplification. Preferably, the forward AHASL-gene-specific primer and the reverse AH/ASL-gene-specific primer for the second PCR amplification are identical to those used in the first PCR amplification. The methods of the invention, however, do not depend on the use of identical forward and reverse iA/iASL-gene-specific primer pairs in both the first and second PCR amplifications.<br>
There are two results from the second PCR amplification of genomic DNA from a plant. The products of the second PCR amplification can be detected as described above for the detection of the products of the first PCR amplification or by any other method for detecting PCR products known in the art. If the specific AHASL gene comprises at least one wild-type AHASL allele, the second PCR amplification produces two DNA fragments, a larger fragment resulting from amplification of the region of the AHASL gene bounded by the annealing sites of the forward and reverse AHASL-gene-specific primers and a smaller fragment resulting from amplification of the region of the AHASL<br><br>
gene bounded by the annealing sites of the wiid-type-a)lele-specific primer and the reverse 4H4SL~gene-specific primer. If, however, the specific AHASL gene does not comprise at least one wild-type AHASL allele, then the first PCR amplification produces only the larger fragment.<br>
In both the first and second PCR amplifications, the larger fragment is produced whether or not, the smaller fragment is produced. Thus, the larger fragment serves as an internal control to indicate that there was a successful first or second PCR amplification. If a first and/or second PCR amplification fails to produce the larger fragment, then that PCR was unsuccessful and should be repeated. The methods of the invention do not depend on performing the first and second PCR amplifications simultaneously and in the same thermocycler. Typically, however, the first and second PCR amplifications of the genomic DNA of a particular plant will be performed simultaneously in the same thermocycler. Also, if the analysis of two, three, or more AHASL genes of a plant is desired, the first and second PCR amplifications for each of the AHASL genes can be conducted simultaneously in the same thermocycler. Furthermore, if the analysis of the one or more AHASL genes of two or more plants is desired, the first and second PCR amplifications for each of the wheat plants can be performed simultaneously in the same thermocycler, depending on the capacity of the thermocycler. In an embodiment of the invention that is suited for the high-throughput analysis of multiple wheat plants, the first and second PCR amplifications are preformed in a thermocycler that accommodates a standard 384-well PCR plates. The use of such a thermocycler and PCR plate allows for the simultaneous performance of the first and second PCR amplifications for each of the three AHASL genes for up to 64 wheat plants.<br>
The third aspect of the invention comprises the method of the second aspect of the invention and further comprises a pre-ampiification step. The pre-amplification step involves a third PCR amplification that is performed prior to the first and second PCR amplifications. The pre-amplification allows the amplification of at least a fragment of each of the AHASL genes so as to enrich the genomic DNA sample for the AHASL genes, or fragments thereof. For example, in the case of wheat, the pre-ampiification allows the amplification of at least a fragment of each of the three AHASL genes (or two in the case of durum wheat) so as to enrich the genomic DNA sample for the three AHASL genes, or fragments thereof. By enriching the genomic DNA sample for the AHASL genes, or fragments thereof, the quantity and quality of the products produced by the first and second PCR amplifications can be improved. While the methods of the invention do not depend on any particular mechanism, it is presumed that the improvement in the first and second PCR amplifications from the pre-amplification step is the result of an increased amount of AHASL templates in the pre-amplified genomic DNA.<br>
The pre-ampiification step involves the use of a forward AHASL primer and a reverse AHASL primer for PCR amplification. For a plant that has two or more AHASL genes in its genome, the forward AHASL primer and the reverse AHASL primer are generic<br><br>
AHASL primers that do not discriminate between the two or more AHASL genes in the plant Any pair of forward and reverse AHASL primers can be employed in a method of the invention, wherein such a pair of primers is capable of amplifying by PCR a fragment from the same region of all of the AHASL genes in a plant and such a fragment includes the annealing sites of the forward and reverse AHASL-gene-specific primers described above.<br>
If desired, the products of the pre-amplificatlon step can be subjected to exonuclease digestion by any method known in the art to reduce or eliminate single-stranded DNA, particularly any forward and/or reverse AHASL primers that remain after the pre-amplification is completed. Such an exonuclease digestion of the pre-amplified ge-nomic DNA, prior to the first and second PCR amplifications, can further improve the first and second PCR amplifications. For example, Exonuclease T (RNase T), S1 nu-clease from Aspergillus oryzae, mung bean nuclease, or Exonuclease I Escherichia coli may be used to remove the single-stranded DNA from the pre-amplification products. Alternatively, the products of the pre-amplrfication step can be purified to remove the forward and reverse AHASL primers. Any method may be used for this purification step, including, but not limited to, commercially available PCR purification methods such as the Wizard MagneSil PCR Cleanup System (Promega Corp., Madison, Wl, USA).<br>
In one embodiment, the methods of the invention find use in determining the zygosity of an individual wheat plant at one* two, or three of the specific AHASL genes therein, namely AHASL1D, AHASL1B, and AHASL1A. In particular, the methods of second and third aspect can be used to determine zygosity at one or more of AHASL1D, AHASL1B, and AHASL1A. This is accomplished by conducting the methods of the second and third aspects one, two, or three times, each time using a different pair of forward and reverse AHASL-gene-specific primers for the first and second PCR amplifications, wherein each primer pair is designed to selectively amplify a fragment of one of the three wheat AHASL genes.<br>
In a preferred embodiment of the invention, the zygosity of single wheat plant is determined for AHASL1D, AHASL1B, and AHASL1A. Genomic DNA from a wheat plant is pre-amplified using a forward AHASL primer having the nudeotides sequence set forth in SEQ ID NO: 1 and a reverse AHASL primer having the nudeotides sequence set forth in SEQ ID NO: 2. The pre-amplified genomic DNA is digested with Exonuclease I. Three pairs of first and second PCR amplifications are then conducted. For the first pair of PCR amplifications, the forward AHASL-gene-specific primer and reverse Atfv4S-gene-specific primers are designed to selectively amplify AHASLID and have the nucleotide sequences set forth in SEQ ID NO: 5 and 7, respectively. For the second pair of PCR amplifications, the forward AH4SL-gene-specific primer and reverse &gt;4H&gt;4S-gene-specific primers are designed to selectively amplify AHASLIB and have the nucleotide sequences set forth in SEQ ID NO: 6 and 8, respectively. For the third pair of PCR amplifications, the forward AHASL -gene-specific primer and reverse AHAS-gene-spectfic primers are designed to selectively amplify AHASLIA and have the nu-<br><br>
cleotide sequences set forth in SEQ ID NO: 5 and 9, respectively. In each instance, the mutant-allele-specific primer is SEQ ID NO: 3, and the wild-type-allele-specific primer is SEQ ID NO: 4. The products of the six PCR amplifications are then detected by agarose gel electrophoresis and ethidiunvbromide staining.<br>
The present invention also provides kits for performing the methods of the invention as described herein. Such kits comprise at least one forward AHASL-gene-specific primer, at least one reverse AHASL-gene-specific primer, a mutant-allele-specific primer, at least one polymerase enzyme capable of catalyzing the PCR amplification of a first fragment of a wheat AHASL gene and a second fragment of a wheat AHASL gene, wherein the first fragment is between the annealing site of-the forward AHASL-gene-specific primer and the annealing site of the reverse AHASL-gene-specific primer and the second fragment is between.the annealing site of the mutant-allele-specific primer and the annealing site of the reverse AHASL-gene-specific primer. The kits of the invention can further comprise at least one additional component selected from the group consisting of wild-type-allele-specific primer, a forward AHASL primer, a reverse AHASL primer, an additional polymerase enzyme, a concentrated buffer solution, a solution of MgCI2, deoxyribonucleotide triphosphates, and Exonuclease I. The methods of the invention involve the use of PCR for amplifying DNA. Oligonucleo-tide primers can be designed for use in PCR reactions to amplify corresponding DNA sequences from genomic DNA or cDNA extracted from any organism of interest. Methods for designing PCR primers are generally known in the art and are disclosed in Sambrook et a/. (1989) Molecular Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor Laboratory Press, Plainview, New York); herein incorporated by reference. See also, Innis et a/., eds. (1990) PCR Protocols: A Guide to Methods and Applications (Academic Press, New York); Innis and Gelfand, eds, (1995) PCR Strategies (Academic Press, New York); Innis and Gelfand, eds. (1999) PCR Methods Manual (Academic Press, New York); Dtetmaier.et a/,, eds. (2002) Rapid Cycle Real Time PCR -Methods and Applications, (Springer Verlag, New York); Theophilus and Raphley, eds. (2002) PCR Mutation Detection Protocols (Humana Press, New York); and Bartlett and Stirling, eds. (2003) PCR Protocols (Humana Press, New York); all of which are herein incorporated by reference. Other known methods of PCR that can be used in the methods of the invention include, but are not limited to, methods using paired primers, nested primers, single specific primers, degenerate primers, gene-specific primers, mixed DNA/RNA primers, vector-specific primers, partially-mismatched primers, and the like.<br>
While the invention does not depend on PCR primers of any particularly number of nucleotides, it is recognized that the portion of a PCR primer that anneals to its complementary target on the template DNA will generally be between about 10 and 50 contiguous nucleotides, preferably 10,11,12,13,14,15,16,17,18,19, 20, 21, 22, 23, 24, 25,26, 27, 28, 29, 30, or more contiguous nucleotides. However, a PCR primer of the invention can further comprise on its 5' end additional nucleotides that are not intended<br><br>
to anneal to the target such as, for example, a DNA sequence comprising one or more restriction enzyme recognition sites.<br>
The methods of the invention involve the use of DNA polymerases for PCR amplification of DNA. Any DNA polymerase known in the art that is capable of amplifying a target DNA by PCR may be used in the methods of the invention. The methods of the invention do not depend on a particular DNA polymerase for PCR amplification of DNA, only that such polymerases are capable of amplifying one or more of the plant AHASL genes or fragments thereof. Preferably, the DNA polymerases of the invention are thermostable DNA polymerases, including but not limited to: Taq polymerases; Pfu polymerases; themostable DNA polymerases from Thermococcus gorgonarious which are also known as Tgo DNA polymerases; thermostable DNA polymerases from Ther-mocoocus litoralis such asf for example, those that are known as Vent® DNA polymerases (Perler, F. et a/. (1992) Proc. Nat!. Acad. ScL USA 89, 5577), thermostable DNA polymerases from Pyrococcus species GB-D such as, for example, those that are known as Deep Vent® DNA polymerases (Xu, M. et a/. (1993) Cell 75,1371-1377); and modified versions and mixtures thereof.<br>
In one embodiment of the invention, the PCR amplifications of the first and second aspects of the Invention described above comprise "hot-start" PCR. Such "hot start" PCR involves methods that are known in art and is generally employed to reduce nonspecific amplification products. Typically, "hot start" PCR methods involve combining the template DNA, primers, and other reaction components with the exception of the DNA polymerase, then holding the reaction mixture, for a period of time prior to introduction of the DNA polymerase, at a temperature above the threshold temperature for non-specific binding of the primers to the template, then adding DNA polymerase to the reaction mixture while the reaction mixture is maintained at a temperature above the threshold temperature.<br>
Although the methods of the present invention do not depend on "hot start" PCR or a particular thermostable DNA polymerase, the preferred DNA polymerase for the PCR amplifications of the first and second aspects of the invention is HotStarTaq DNA Polymerase (Qiagen Inc., Valencia, CA, USA). HotStar DNA polymerase is a modified form of Taq DNA polymerase that is supplied by the manufacturer in an inactive state that has no DNA polymerase activity at ambient temperatures. The inactivity at ambient temperatures prevents extension of nonspecificaily annealed primers and primer-dimers formed at low temperatures during the setup of PCR reaction mixtures and while the tubes are heated to reach the initial denaturation temperature. HotStarTaq DNA Polymerase is activated by a 15-minute incubation at 95°C which can be incorporated into any existing thermal-cycler program. Thus, HotStar DNA polymerase allows for hot start PCR without the need to physically separate the DNA polymerase from the other components of the reaction mixture during the time that reaction mixture is heated to a temperature above the threshold temperature.<br>
In another embodiment of the invention, the PCR amplification of the third aspect of the invention, also referred to herein as the pre-amp!ification, comprises at least one proof-<br><br>
reading DNA polymerase, which comprises a 3'to-5' exonucleases activity. Such proofreading DNA polymerases include, but are not limited to Tgo DNA polymerases, Vent® DNA polymerases, and Deep Vent® DNA polymerases as described above. In a preferred embodiment of the invention, the pre-amplificaiton comprises the Expand High Fidelity PCR System (Roche Applied Science, Indianapolis, IN, USA). The Expand High Fidelity PCR System comprises Taq DNA polymerase and a proofreading DNA polymerase, Tgo DNA polymerase, that minimizes mutations that may result from the inadvertent misincorporation of a non-complementary nucleotide. Such PCR-induced mutations could interfere with the PCR amplification of the first and second aspects of the invention as disclosed herein. Therefore, a high-fidelity amplification, such as that which can be obtained with a PCR amplification comprising a proofreading DNA polymerase, is desired for the pre-amplification.<br>
The methods of the invention involve the amplification of a target DNA sequence by PCR.   In certain embodiments of the invention, the target DNA sequence will amplified directly from a sample comprising genomic DNA isolated from at least one plant or part, organ, tissue, or ceil thereof. Those of ordinary skill in the art will recognize that the amount or concentration of genomic DNA will depend on any number of factors including, but not limited to, the PCR conditions (e.g. annealing temperature, denatura-tion temperature, the number of cycles, primer concentrations, dNTP concentrations, and the like), the thermostable DNA polymerase, the sequence of the primers, and the sequence of the target. Typically, in the embodiments of the invention described herein, the concentration of genomic DNA is at least about 5 ng/yL to about 100 ng/pL In addition to PCR amplification, the methods of the invention can involve various techniques of molecular biology including, for example, DNA isolation, particularly genomic DNA isolation, digestion of DNA by restriction enzymes and nucleases, DNA ligation, DNA sequencing, agarose gei electrophoresis, detection of DNA by ethidium-bromide staining, and the like. Such techniques are generally known in the art and are disclosed, for example, in Sambrook ef al. (1989) Molecular Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor Laboratory Press, Plainview, New York). The methods of the invention involve the use of genomic DNA isolated from a plant. The methods of the invention do not depend on genomic DNA isolated by any particular method. Any method known in the art for isolating, or purifying, from a plant, genomic DNA, which can be used a source of template DNA for the PCR amplifications described above, can be employed in the methods of the invention. See, for example, Stein etal. ((2001) Plant Breeding, 12:354-356); Clark, ed. ((1997) Plant Molecular Biology — A Laboratory Manual, Springer-Verlag, New York, pp. 3-15); Miller ef a/.,((1988) Nucleic Acids Research, 16:1215); ail of which are herein incorporated by reference.   Preferably, such methods for isolating plant genomic DNA are suited, or can be adapted by one of ordinary skill in the art, for the isolation of genomic DNA from relatively large numbers of tissue samples of plants. In an embodiment of the invention, genomic DNA is isolated from wheat plants using a DNeasy® kit according to the manufacturer's instructions (Qiagen Inc., Valencia, CA, USA). In another embodiment,<br><br>
genomic DNA is isolated from wheat plants using a MagneSil® kit according to the manufacturer's instructions (Promega Corp., Madison, Wl, USA). For the methods of the present invention, genomic DNA can be isolated from whole plants or any part, organ, tissue, or cell thereof. For example, genomic DNA can be isolated from seedlings, leaves, stems, roots, inflorescences, seeds, embryos, tillers, •coleotiles, anthers, stigmas, cultured cells, and the like. Furthermore, the invention does not depend on the isolation of genomic DNA from plants or parts, organs, tissues, or cells thereof that are of any particular developmental stage. The methods can employ genomic DNA that is isolated from, for example, seedlings or mature plants, or any part, organ, tissue or cell thereof. Furthermore, the invention does not depend on plants that are grown under any particular conditions. The plants can be grown, for example, under field conditions, in a greenhouse, or a growth chamber, in culture, or even hydroponically in a greenhouse or growth chamber. Typically, the plants will be grown in conditions of light, temperature, nutrients, and moisture that favor the growth and development of the plants.<br>
The methods of invention involve detecting the products of the PCR amplifications, particularly the first and second PCR amplifications. Typically, the PCR products are detected by first separating the products in a substrate on the basis of molecular weight and then detecting each of the separated PCR products in the substrate. In a preferred embodiment of the invention, the PCR products are detected by agarose gel electro-phoresis of the PCR products followed by ethidiumrbromide staining of the DNA in the gel and visualization in the gel by florescence in the presence of UV light. However, any detection method suitable for separating polynucleotides can be used to detect the PCR products of the invention including, but not limited to, gel electrophoresis, high performance liquid chromatography, capillary eiectrophoresis, and the like. Substrates for such methods include, for example, agarose, polyacrylamide, diethylaminoetyl cellulose, hydroxyalkyl cellulose, sepharose, polyoxyethylene, and the like. The PCR amplifications of the invention can involve the use of one or more primers that are labeled, for example, radioactively, or with a fluorescent dye, a luminescent label, a paramagnetic label, or any other label suitable for the detection of nucleic acids. When the PCR amplifications involve one or more of such a labeled primers, the detection step can include the detection of the radioactive, fluorescent, luminescent, paramagnetic, or other label by any methods known in the art for detecting such a label. The methods of the invention as disclosed herein can be used to analyze the AHASL genes of any plant species of interest. Examples of plant species of interest include, but are not limited to, com or maize {Zea mays), Brassica sp. (e.g., S. napus, B. rapa, B. juncea), particularly those Brassioa species useful as sources of seed oil, alfalfa (Medicago sativa), rice (Oryza sativa), rye (Secale cereale), sorghum {Sorghum blcolor, Sorghum vulgare), millet (e.g., pearl millet (Pennisetum glaucum), proso millet (Panicum miliaceum), foxtail millet (Setaria italica), finger millet (Eleusine coracana)), sunflower (Helianthus annuus), safflower {Carthamus tinctorius), wheat (Triticum aestivum, T. Turgldum $$p. durum), soybean (Glyclne max), tobacco (Nicotiana tabacum), potato<br><br>
{Solanum tuberosum), peanuts (Arachis hypogaea), cotton {Gossypium barbadense, Gossypium hirsutum), sweet potato (Ipomoea batatus), cassava (Manihot esculenta), coffee {Coffea spp.), coconut (Cocos nucifera), pineapple (Ananas comosus), citrus trees {Citrus spp.), cocoa (Theobroma cacao), tea (Camellia sinensis), banana (Musa spp.), avocado (Persea americana), fig (Ficus casica), guava (Psidium guajava), mango •(Mangifera indica), olive (Olea europaea), papaya (Carica papaya), cashew (Anacardium occldentale), macadamia (Macadamia integrifolia), almond (Prunus amygdalus), sugar beets (Beta vulgaris), sugarcane (Sacchanim spp.), oats, barley, vegetables, ornamentals, and conifers. Preferably, plants of the present invention are crop plants (for example, wheat, com, rice, barley, oats, sugar beet, alfalfa, sunflower, Brassica, soybean, cotton, safflower, peanut, sorghum, millet, tobacco, etc.), more preferably grain plants (for example, wheat, corn, rice, barley, sorghum, rye, triticale, etc.), yet more preferably wheat plants. The following examples are offered by way of illustration and not by way of limitation.<br>
EXAMPLE 1: The AHASL Genes of Wheat<br>
Three acetohydroxyacid synthase large subunit (AHASL) sequence variants were<br>
identified from sequencing of wheat cDNAs. The genes corresponding to these<br>
variants were mapped to their respective genome and chromosome arm (6L) and were<br>
named on the basis of the genome in which they reside (e.g., AHASL on genome A =<br>
AHASL1A). Nucleotide sequences for multiple varieties of Triticum aestivum<br>
AHASL1A, AHASL1B and AHASL1D transcripts were obtained. These sequences<br>
comprise the full coding sequences for the mature polypeptides. See, U.S. Application<br>
entitled "Polynucleotides Encoding Mature AHASL Proteins For Creating<br>
Imidazolinone-Tolerant Plants", filed concurrently herewith, herein incorporated in its<br>
entirety by reference.	.<br>
. A comparison of imidazolinone (IMI)-tolerant mutant varieties with wild-type progenitors . revealed that the most common type of mutation was a G-to-A transition, which produces a serine (S) to asparagine (N) substitution in a position corresponding to S653 in the model taxon, Arabldopsis thaliana (Sathasivan et al. (1991) Plant Physiol. 97:1044-1050; Hattori et a/. (1992) Mol. Gen. Genet. 232:167-173). The semi-dominant mutation, now termed S653(^f)N, is also found in other IMkolerant crops. See, for example, U.S. Patent No. 5,731,180; and U.S. Application Serial No. 10/695,089, filed October 28, 2003.<br>
EXAMPLE 2: A Rapid Method for Distinguishing the Wheat AHASL Genes and Their Respective Alleles<br>
To facilitate the rapid identification of S653(AQN mutations in wheat, methods for allele-specific PCR assays for mutations in each of the three AHASL genes were developed. The IMI-toIerant varieties used for the assay development are set forth in Table 1.<br><br><br>
From the nucleotide sequences of the mutant and wild-type wheat AHASL genes, a method involving a set of six allele-specific PCR assays for high-throughput genotyping of bread and durum wheat lines containing the S653(At)N mutation was developed. The assays were designed as ailele-specific PCR methods using a pair of primers, in which one primer is specific to the allele (mutant or wild type) and the second specific to gene {AHASL1D vs. AHASL1B vs. AHASL1A). Adding a second gene-specific primer upstream of the ailele-specific primer added a further refinement in that each reaction now contained an internal control for DNA quantity/quality. Each reaction could now produce two products. The "diagnostic" band is the product of the aliele-specific and gene-specific primer pair, while a higher molecular weight "control" band represents the product of both gene-specific primers. The presence of the control band in the absence of the diagnostic band indicates that the reaction was successful and the allele in question is not present in the sample. Finally, a preliminary DNA amplification step was added to increase the template quantity and purity in the ailele-specific PCR step, thus ensuring consistent amplification from DNA samples of varied quantity and quality. See, Figure 2 for a schematic overview of this embodiment of the invention.<br>
An important aspect of the method is the development of the mutant ailele-specific primer MU-F (SEQ ID NO: 3). Typically, primers that are developed fdr PCR methods for distinguishing between two alieles that vary by a single nucleotide substitution are designed such that the primers reflect the same polymorphism at their 31 end. The basis for this primer design is the premise that oligonucieotides with a mismatched 31 end-residue will not function as primers in PCR under the appropriate conditions (Newton et a/. (1989) NucL Acids Res, 17:2503-2516; and Wu et at, (1989) Proc. NatL Acad. ScL USA 86: 2757-2760). In contrast, the nucleotide at the 3f end of the MU-F mismatches both the mutant and wild-type alieles at the site of the SNP. Primers that were designed to have a 31 end nucleotide that mismatches the wild-type allele but matches the mutant allele at the site of SNP were unexpectedly found to allow the amplification of the wild-type allele. Therefore, other primer designs were tested. The unexpected result was that a primer such as MU-F, which has a 31 end residue that mismatches both the wild-type and mutant alieles at the site of the SNP, could allow<br><br>
the amplification of the mutant allele but not the wild-type allele in the methods of the invention.<br>
Although the MU-F primer is capable of annealing to a DNA sequence that is exactly complementary to MU-F, a mutant allele of a wheat AHASL that comprises such a DNA sequence is expected to be very rare and has not been detected in nature. Such a mutant allele would have a cytidine at the same nucleotide that is the site of the SNP that results in the S653(ATN mutation. Commonly, wheat plants having the S653(AT)N mutation are produced by ethane methylsulfpnate (EMS) mutagenesis. EMS generates primarily G-C-to-AT transitions. The S653(4f)N mutation results from such a G-C-to-A-T transition. See, Figures 1A, 1B, and 1C.<br>
Figure 2 is a schematic representation of one embodiment of the invention in which two PCR reactions are used to detect the S653(Atf)N mutation and determine the zygosity of AHASL1B. Any number of reactions (one to six) may be used depending on applica-. tion. For example, if one is interested only in the presence of the S653(At)N mutation in only AHASL1A and AHASL1B, but not whether the mutations are homozygous, then only two reactions are required.<br>
This gel-based PCR test consists of nested PCR with the first round consisting of a single reaction with common primers (CM-F, CM-R). A second round consisting of a single or multiple individual reactions, up to three {AHASL1A, AHASL1B, AHASL1D) for the mutant (IMI-tolerant) allele and one for the wild-type (susceptible) allele. Each of these, second-round reactions contains three primers, the nested forward and reverse gene-specific primers (e.g. 1B-F, 1B-R) plus an allele-specific primer (ie., WT-F or MU-F) with the mutated/wild-type base on its 31 primer end for amplification of the resistant/susceptible locus respectively. The larger PCR product (resulting from amplification between the forward and reverse gene-specific primers) is.expected to be produced in all instances, and thust this PCR product serves as a positive control for the PCR reaction [control band). The smaller PCR product (resulting from amplification between the allele-specific primer and reverse gene-specific primer) is an allele-specific PCR product {diagnostic band). For each PCR reaction, the smaller PCR product is expected to be produced only when the particular AHASL gene of interest comprises at least one AHASL allele that corresponds to the AHASL allele-specific primer (i.e., WT-F or MU-F) that was included in the PCR reaction mixture. The reaction products are separated on a 3.5 % Agarose-1000 gel. Agarose-1000 is cross-linked agarose that can be obtained from Invitrogen Corp., Carlsbad, CA, USA.<br>
EXAMPLE 3: Analysis of the AHASL Genes of Four Wheat Genotypes<br>
A sample containing wheat genomic DNA is obtained by any method known in the art for purifying genomic DNA from plant tissues, particularly wheat. However, when comparing two or more wheat plants, an equivalent amount of tissue from each of the plants should be used for the purification of the genomic DNA so as to ensure that samples from each of the plants will contain similar concentrations of genomic DNA,<br><br>
Typically, the DNA concentration of the sample is about 100 ng DNA per µL If the DNA concentration is greater than this range one should dilute the sample to about 50 ng DNA per/µL While the method of the invention does not depend on a particular DNA concentration, the DNA concentration of the sample is preferably between about 5 and about 100 ng DNA per µL<br>
The first-round PCR amplification was performed using the Expand High Fidelity PCR System (Roche Applied Science, Indianapolis, IN, USA). A first round PCR reaction comprising the components set forth in Table 2 was prepared. The Expand High Fidelity PCR System was used because this system comprises, in addition to Taq poly-merase, a proofreading DNA polymerase (Tgo DNA poiymerase) that minimizes mutations that may result from the inadvertent incorporation a non-complementary nucleo-tides. Such PCR-induced mutations could interfere with the second-round PCR amplification described below. Thus, a high-fidelity amplification was desired for the first-round PCT. Therefore, the use of a PCR system that comprises a proofreading DNA polymerase—such as, for example, the Expand High Fidelity PCR System—was employed for the first-round amplification.<br>
TABLE 2. Components of First-Round PCR Reaction<br><br>
Expand High Fidelity buffer, 10X concentration without MgCI2 Expand High Fidelity Enzyme mix contains Tag DNA polymerase and Tgo DNA polymerase, a thermostable DNA polymerase with proofreading activity.<br>
The first round PCR amplification (also referred to herein above as the "pre-amplification") was performed under in a TGradient Thermocycler (Biornetra GmbH, Goettingen, Germany). The following thermocycling conditions were employed sequentially; (1) 94°C for 3 minutes; (2) 25 cycles of 94°C for 30 seconds, 56°C for 1 minute; and 72°C for 45 seconds; and (3) 72°C for 7 minutes. After completion of the first round PCR amplification, an aliquot of the PCR product solution is is subjected to digestion with Exonuclease I (Amersham Biosciences Corp., Piscataway, NJ, USA) as follows. To a 2µL aliquot of the PCR product solution, 0.05<br><br>
µL EXOI (10 units/µL) and 4.95 f/µ H20 were added. The resulting mixture was incubated at 37°C for 1 hour followed by incubation at 72°C for 15 minutes. Following the second incubation, the ExoI reaction mixture was diluted sequentially. First, 100//L of H2O was added to the ExoI reaction mixture and then mixed. Next, a 10 JµL aliquot was removed from the diluted ExoI reaction and further diluting by adding to 190//L of H2O. A 1 /µL aliquot of resulting diluted solution was used as the DNA containing sample for each second round PCR.<br>
The second round PCR amplification can be used to the detect the presence of the mutant and wild-type alleles for each of the three AHASL genes, AHASL1D, AHASL1B, and AHASL1A, if desired. Alternatively, the presence of either a wild-type or mutant allele at a single AHASL gene or two AHASL genes can be determined. Depending on the desired outcome, the second round PCR amplifications may include 1,2,3,4, 5, or 6 separate amplifications employing the six combinations of primers that are set forth in Table 3,<br>
TABLE 3. Second-Round PCR primer for the Detection of Wild-Type and Mutant Alleles of AHASL Genes<br><br>
Second round PCR amplifications were performed using HotStar Taq DNA Polymerase (Qiagen, Inc., Valencia, CA, USA). A second round PCR reaction comprising the components set forth in Table 4 was prepared for each group of three primers. A "hot start-type" DNA polymerase was selected for the second-round PCR to enhance the probability of amplifying only the desired target sequences.<br><br>
TABLE 4. Comoonents of Second-Round PCR Reaction<br>
The second-round PCR amplification was performed under in a TGradient Thermocy-cler (Biometra GmbH, Goettingen, Germany). The following thermocycling conditions were employed sequentially: (1) 95°C for 15 minutes; (2) eight touchdown cycles with the annealing temperature decreased 1°C with each successive cycle with the a first cycle of 94°C for 1 minute, 68°C (annealing temperature) for 1 minute, and 72°C for 1 minute, and in cycles 2-8, the annealing temperature was 67°C, 66°C, 65°C, 64°C, 63°C, 62°C, and 61°C, respectively (annealing time of 1 minute for each cycle); (3) 26 cycles of 94°C for 1 minute, 60°C for 1 minute, and 72°C, for 1 minute; and (4) a final extension at 72°C for 10 minutes.<br>
Following the second-round PCR amplification, the second-round PCR products were subjected to agarose gel electrophoresis in a horizontal apparatus by methods known to those of ordinary skill in the art. Following electrophoresis, the gel was subjected to ethidium-bromide staining, and the PCR products detected by florescence in the presence of UV light.<br>
Figure 5 provides an example of the results that were obtained using the methods of the invention with four wheat genotypes, "wild type", CV9804, G-208, and K-42, For each genotype, six second-round PCR amplifications were performed to allow the determination of the zygosity at each of the three AHASL alleles. For example the leftmost pair of bands within each panel is the assay for AHASL1D, wild-type allele, while the right-most pair of bands within each panel is the assay for AHASL1A mutant allele. In each lane, the control band is the upper band (about 750 bp for AHASL1D and AHASL1A, and about 580 bp for AHASL1B) and the diagnostic band is the lower band (about 270 bp for AHASL1D and AHASL1A, and about 150 bp for AHASL1B). The control band result was produced by amplification between the forward primer 1 and the reverse primer (see Table 3). The diagnostic band was produced by amplification between the forward primer 2 (WT-F or MU-F). From Figure 5, one can conclude that CV9804 is homozygous for the mutant allele of AHASL1D, G-208, homozygous for the mutant allele of AHASL1B, and K-42, homozygous for the mutant allele of AHASL1A.<br><br>
These results are in agreement with genotypes determined from standard genetic analyses (data not shown).<br>
The methods described herein above have been tested with wheat plants that are known to be homozygous for the mutant allele, heterozygous, and homozygous for the wild-type allele at each of the wheat AHASL genes. The results of these tests confirm (data not shown) that the methods disclosed herein can be used to rapidly analyze the genome of wheat plants. Thus, the methods of the present invention can be used to rapidly determine whether a particular wheat plant is homozygous for the mutant AHASL allele, heterozygous, or homozygous for the wild-type allele for each of that plant's AHASL genes. Accordingly, the present invention provides methods that can be employed by plant breeders in the production of imidazolinone-tolerant wheat plants having one, two, three, four, five, or six mutant AHASL alleles in their genomes. All publications and patent applications mentioned in the specification are indicative of the level of those skilled in the art to which this invention pertains. All publications and patent applications are herein incorporated by reference to the same extent as if each individual publication or patent application was specifically and individually indicated to be incorporated by reference.<br>
Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be obvious that certain changes and modifications may be practiced within the scope of the appended claims.<br><br><br>
THAT WHICH IS CLAIMED:<br>
1.	A method for detecting a mutant allele of a wheat acetohydroxyacid synthase<br>
large subunit (AHASL) gene that confers tolerance to imidazolinohe herbicides<br>
on a wheat plant, said method comprising the steps of:<br>
(a)	obtaining genomic DNA from a wheat plant;<br>
(b)	using said DNA as a template for a PCR amplification comprising said<br>
DNA, polymerase, deoxyribonucleotide triphosphates, a forward AHASL-<br>
gene-specific primer, a reverse 4H4SL-gene-specific primer, and a mutant-<br>
allele-specific primer comprising a nucleotide sequence with a 51 end and a<br>
31 end, wherein said nucleotide sequence is capable of annealing to the<br>
complement of nucleotides 3 to 23 of SEQ ID NO; 12 and said nucleotide<br>
sequence has a cytidine at the 3r end; and<br>
(c)	detecting the products of said PCR amplification;<br>
wherein said mutant-allele-specific primer is capable of annealing to a region of an AHASL gene that is nested between the annealing sites of said forward and reverse AHASL-gene-specific primers.<br>
2.	The method of claim 1, wherein said DNA has been subjected to a pre-<br>
amplification before step (b) said pre-ampiification comprising said DNA, poly<br>
merase, deoxyribonucleotide triphosphates, a forward AHASL primer, and a re<br>
verse AHASL primer, wherein said forward and reverse AHASL-gene-specific<br>
primers are capable of annealing to regions of an AHASL gene that are nested<br>
between the annealing sites of said forward and reverse AHASL primers.<br>
3.	The method of claim 2, wherein said DNA is digested with exonuclease following<br>
said pre-amplification and before step (b).<br>
4.	The method of claim 2, wherein said forward AHASL primer and said reverse<br>
AHASL primer are designed to anneal to AHASL1A, AHASL1B, and AHASL1D.<br>
5.	The method of claim 2, wherein said forward AHASL primer comprises the nu<br>
cleotide sequence set forth in SEQ ID NO: 1.<br>
6.	The method of claim 2, wherein said reverse AHASL primer comprises the nu<br>
cleotide sequence set forth in SEQ ID NO: 2.<br>
7.	The method of claim 1, wherein said mutant-allele-specific primer comprises the<br>
nucleotide sequence set forth in SEQ ID NO: 3.<br>
8.	The method of claim 1, wherein said AHASL gene is AHASL1D.<br><br>
9.	The method of claim 8, wherein said forward AHASL-gene-specific primer has<br>
the sequence set forth in SEQ ID NO: 5 and said reverse AHASL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 7.<br>
10.	The method of claim 1, wherein said AHASL gene is AHASL1B.<br>
11.	The method of claim 10, wherein said forward AHASL-gene-specific primer has<br>
the sequence set forth in SEQ ID NO: 6 and said reverse AHASL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 8.<br>
12.	The method of claim 1, wherein said AHASL gene is AHASL1A.<br>
13.	The method of claim 12, wherein said forward AH4SL-gene-speciflc primer has<br>
the sequence set forth in SEQ ID NO: 5 and said reverse AHASL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 9.<br>
14.	The method of claim 1, wherein said detecting comprises gel eiectrophoresis and<br>
ethidium-bromide staining.<br>
15.	A method for analysis of a wheat AHASL gene, said method comprising the steps<br>
of:<br><br>
(a)	obtaining genomic DNA from a wheat plant;<br>
(b)	using said DNA. as a template for a first PCR amplification comprising said<br>
DNA, polymerase, deoxyribonudeotide triphosphates, a forward AHASL-<br>
gene-specific primer, a reverse AHASL-gene-specific primer, and a mutant-<br>
allele-specific primer comprising a first nucleotide sequence with a 5f end<br>
and a 3fend, wherein said first nucleotide sequence is capable of annealing<br>
to the complement of nucleotides 3 to 23 of SEQ ID NO: 12 and said first<br>
nucleotide sequence has a cytidine at the 3'. end;<br>
(c)	using said DNA as a template for a second PCR amplification comprising<br>
said DNA, polymerase, deoxyribonudeotide triphosphates, said forward<br>
AH4SL-gene-specific primer, said reverse AWASL-gene-specific primer,<br>
and a wild-type-allele-specific primer comprising a second nucleotide se-<br>
quence with a 51 end and a 3'end, wherein said second nucleotide se<br>
quence is capable of annealing to the complement of nucieotides 4 to 23 of<br>
SEQ ID NO: 10 and said second nucleotide sequence has a guanosine at<br>
the 31 end; and<br>
(d)	detecting the products of said first and said second PCR amplifications;<br>
wherein said wiid-type-alleie-specific primer and said mutant-allele-specific<br>
primer are capable of annealing to a region of an AHASL gene that is<br>
nested between the annealing sites of said forward and reverse AHASL-<br>
gene-specific primers.<br><br>
16.	The method of claim 15, wherein said DNA has been subjected to a pre-<br>
amplification before step (b), said pre-amplification comprising said DNA, poly-<br>
merase, deoxyribonucleotide triphosphates, a forward AHASL primer, and a re-<br>
verse AHASL primer, wherein said forward and reverse 4H4SL-gene-specific<br>
primers are capable of annealing to regions of an AHASL gene that are nested<br>
between the annealing sites of said forward and reverse AHASL primers.<br>
17.	The method of claim 16, wherein said DNA is digested with exonuclease follow-<br>
ing said pre-amplification and before step (b).<br>
18.	The method of claim 16, wherein said forward AHASL primer and said reverse<br>
AHASL primer are designed to anneal to AHASL1A, AHASL1B, and AHASL1D.<br>
19.	The method of claim 16, wherein said forward AHASL primer comprises the nu-<br>
cleotide sequence set forth in SEQ ID NO: 1.<br>
20.	The method of claim 16, wherein said reverse AHASL primer comprises the nu-<br>
cleotide sequence set forth in SEQ ID NO: 2,<br>
21.	The method of claim 15, wherein said mutant-allele-specific primer comprises the<br>
nucleotide sequence set forth in SEQ ID NO: 3.<br>
22.	The method of claim 15, wherein said wild-type-allele-specific primer comprises<br>
the nucleotide sequence set forth in SEQ ID NO: 4.<br>
23.	The method of claim 15, wherein said AHASL gene is AHASL1D.<br>
24.	The method of claim 23, wherein said forward AH4SL-gene~specific primer has<br>
the sequence set forth in SEQ ID NO: 5 and said reverse AHdSLrgene-specific<br>
primer has the sequence set forth in SEQ ID NO: 7.<br>
25.	The method of claim 15, wherein said AHASL gene is AHASL1B.<br>
26.	The method of claim 25, wherein said forward AHASL-gene-specrfic primer has<br>
the sequence set forth in SEQ ID NO: 6 and said reverse AH/ASaL--gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 8.<br>
27.	The method of claim 15, wherein said AHASL gene is AHASL1A.<br><br>
28.	The method of claim 27, wherein said forward 4H4SL~gene-specific primer has<br>
the sequence set forth in SEQ ID NO: 5 and said reverse 4H4SL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO; 9.<br>
29.	The method of claim 15, wherein said detecting comprises gel electrophoresis<br>
and ethidiurn-bromide staining.<br>
30.	A method for analysis of a wheat AHASL gene, said method comprising the steps<br>
of:<br><br>
(a)	obtaining genomic DNA from a wheat plant;<br>
(b)	using said DNA as a template in a pre-amplification comprising said DNA,<br>
deoxyribonucleotide triphosphates, polymerase, a forward AHASL primer,<br>
and a reverse AHASL primer, so as to produce pre-amplified DNA;<br>
(c)	using said pre-amplified DNA as a template for a first PCR amplification<br>
comprising said pre-amplified DNA, polymerase, deoxyribonucleotide<br>
triphosphates, a forward AHASL-gene-specific primer, a reverse AHASL-<br>
gene-specific primer, and a mutant-allele-specific primer comprising a first<br>
nucleotide sequence with a 51 end and a 3(end, wherein said first nucleotide<br>
sequence is capable of annealing to the complement of nucleotides 3 to 23<br>
of SEQ ID NO: 12 and said first nucleotide sequence has a cytidine at the<br>
3fend;	;	.<br>
(d)	using said pre-amplified DNA as a template for a second PCR amplification<br>
comprising said pre-amplified DNA, polymerase, deoxyribonucleotide<br>
triphosphates, said forward AHASL-generspecific primer, said reverse<br>
i4H4SL-gene-specific primer, and a wild-type-allele-spedfic primer compris<br>
ing a second nucleotide sequence with a 5f end and a 3! end, wherein said<br>
second nucleotide sequence, is capable of annealing to the complement of<br>
nucleotides 4 to 23 of SEQ ID NO: 10 and said second nucleotide se<br>
quence has a guanosine at the 3' end; and<br>
(e)	detecting the products of said first and said second PCR amplifications;<br>
wherein said wild-type-allele-specific primer and said mutant-allele-specific<br>
primer are capable of annealing to a region of an AHASL gene that is<br>
nested between the annealing sites of said forward and reverse AHASL-<br>
gene-specific primers.<br><br>
31.	The method of claim 30, wherein said pre-amplified DNA is digested with exonu-<br>
clease before step (c).<br>
32.	An oligonucleotide primer for PCR amplification of a fragment of an allele of an<br>
AHASL gene, said primer having the nucieotide sequence set forth in SEQ ID<br>
NO: 3 or SEQ ID NO: 4.<br><br>
33.	A kit for determining whether a wheat plant comprises in its genome a mutant<br>
allele of a wheat AHASL gene that confers tolerance to imiclazolinone herbicides<br>
on a wheat plant, said kit comprising:<br>
(a)	a forward AHASL-gene-specific primer, a reverse AHASL-gene-specific<br>
primer, and a mutant-allele-specific primer comprising a nucieotide se-<br>
quence with a 51 end and a 3' end, wherein said nucieotide sequence is ca-<br>
pable of annealing to the complement of nudeotides 3 to 23 of SEQ ID NO:<br>
12, said nucieotide sequence has a cytidine at the 3f end, and said mutant-<br>
allele-specific primer is capable of annealing to a region of an AHASL gene<br>
that is nested between the annealing sites of said forward and reverse<br>
AHASL-gene-specific primers;<br>
(b)	a polymerase enzyme capable of catalyzing the PCR amplification of a first<br>
fragment of a wheat AHASL gene and a second fragment of a wheat<br>
AHASL gene, wherein the first fragment is between said annealing site of<br>
said forward AHASL-gene-specific primer and said annealing site of said<br>
reverse AHASL-gene-specific primer and tfie second fragment is between<br>
the annealing site of said mutant-allele-specffic primer and said annealing<br>
site of said reverse AHASL-gene-specific primer.<br>
34.	The kit of claim 33, wherein said mutant-allele-specific primer comprises the nu<br>
cieotide sequence set forth in SEQ ID NO: 3.<br>
35.	The kit of claim 33, wherein said AHASL gene is AHASL1D.<br>
36.	The kit of claim 35, wherein said forward AHASL-gene-specrfic primer has the<br>
sequence set forth in SEQ ID NO: 5 and said reverse AHASL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 7.<br>
37.	The kit of claim 33, wherein said AHASL.gene is AHASL1B.<br>
38.	The kit of claim 37, wherein said forward AHASL-gene-specific primer has the<br>
sequence set forth in SEQ ID NO: 6 and said reverse AHASL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 8.<br>
39.	The kit of claim 33, wherein said AHASL gene is AHASL1A.<br>
40.	The kit of claim 39, wherein said forward AHASL-gene-specific primer has the<br>
sequence set forth in SEQ ID NO: 5 and said reverse AHASL-gene-specific<br>
primer has the sequence set forth in SEQ ID NO: 9.<br>
41.	The kit of claim 33T further comprising a forv<br>
AHASL primer.<br>
Dated this 20 day of October 2006 <br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgIDI2LTEyLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   AMENDED  PAGES OF SPECIFICATION   26-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgICAyNi0xMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   AMENDED CLAIMS   26-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgIDI2LTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   AMENDED CLAIMS  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAgMjYtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   AMENDED PAGES OF SPECIFICATION   26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDE3LTEyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   CORRESPONDENCE OTHERS 17-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAgMjYtMTItMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   EXAMINATION REPORT REPLY RECEIVED   26-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   EXAMINATION REPORT REPLY RECEIVED  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgRk9STS0xICAgMjYtMTItMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   FORM-1   26-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgRk9STS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   FORM-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgRk9STS0zICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   FORM-3  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgRk9STS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   FORM-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   OTHER PATENT DOCUMENT  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UIDEgIDI2LTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   OTHER PATENT DOCUMENT 1  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UIDIgIDI2LTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   OTHER PATENT DOCUMENT 2  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgT1RIRVJTICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   OTHERS  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICAgUE9XRVIgT0YgQVRUT1JORVkgIDI2LTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006   POWER OF ATTORNEY  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1DSEVOUC0yMDA2ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDIxLTA5LTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">3876-CHENP-2006  CORRESPONDENCE OTHERS  21-09-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2IGNvcnJlc3BvbmRhbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006 correspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2IGRlc2NyaXB0dWlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3876-chenp-2006 descriptuion(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">3876-chenp-2006 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3876-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258395-a-method-of-operation-for-the-slurry-polymerisation-of-olefins.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258397-method-of-discriminating-at-least-two-cell-populations-of-and-application.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258396</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3876/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Oct-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF SE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-67056, LUDWIGSHAFEN, GERMANY</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SINGH, BIJAY</td>
											<td>301 CONNEMARA DRIVE, CARY, NC 27519, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ZHAO, CHENGYAN</td>
											<td>105 FRINGE TREE COURT, CARY, NC 27519, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ASCENZI, ROBERT</td>
											<td>741 LANDING LANE, CARY, NC 27511, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/02844</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-03-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/805,973</td>
									<td>2004-03-22</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258396-methods-and-compositions-for-analyzing-ahasl-genes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:43:33 GMT -->
</html>
