<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258414-microbial-host-cells-for-the-fermentive-production-of-1-butanol by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:39:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258414:&quot;MICROBIAL HOST CELLS FOR THE FERMENTIVE PRODUCTION OF 1- BUTANOL&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;MICROBIAL HOST CELLS FOR THE FERMENTIVE PRODUCTION OF 1- BUTANOL&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods for the fermentive production of four carbon alcohols is provided. Specifically, butanol, preferably 1-bu¬tanol is produced by the fermentive growth of a recombinant bacterium expressing a 1- butanol biosynthetic pathway.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TITLE<br>
FERMENTIVE PRODUCTION OF FOUR CARBON ALCOHOLS CROSS-REFERENCE TO RELATED APPLICATION<br>
This application claims priority under 35 U.S.C. §119 from U.S. Provisional Application Serial No. 60/721677, filed September 29, 2005, and from U.S. Provisional Application Serial No. 60/814470, filed June 16, 2006.<br>
FIELD OF THE INVENTION<br>
The invention relates to the field of industrial microbiology and the production of alcohols. More specifically, 1-butanoi is produced via industrial fermentation of a recombinant microorganism. BACKGROUND OF THE INVENTION<br>
Butanoi is an important industrial chemical, useful as a fuel additive, as a feedstock chemical in the plastics industry, and as a foodgrade extractant In the food and flavor industry. Each year 10 to12 billion pounds of butanoi are produced by petrochemical means and the need for this commodity chemical will likely increase.<br>
Methods for the chemical synthesis of 1-butanoi are known, such as the Oxo Process, the Reppe Process, and the hydrogenation of crotonaldehyde (Ullmann's Encyclopedia of Industrial Chemistry, 6th edition, 2003, Wiley-VCHVerlag GmbH and Co., Weinheim, Germany, Vol. 5, pp. 716-719). These processes use starting materials derived from petrochemicals and are generally expensive and are not environmentally friendly. The production of 1-butanol from plant-derived raw materials would minimize green house gas emissions and would represent an advance in the art.<br>
Methods for producing 1-buanol by biotransformation of other organic chemicals are aiso known. For example, Muramoto et al. (JP63017695) describe a method for the production of alcohols, including butanoi, from aldehydes using strains of Pseudomonas. Additionally, Kuehnle et al. (EP 1149918) describe a process for preparing 1-butanol and 2-butanoI by the oxidation of hydrocarbons by various strains of Rhodococcus ruber.<br>
Methods of producing butanoi by fermentation are also known,<br>
 process produces a mixture of acetone, 1-butanol and ethanol and is referred to as the ABE processes (Blaschek et al., U.S. Patent No. 6,358,717).  Acetone-butanol-ethanol (ABE) fermentation by Clostridium acetobutylicum is one of the oldest known industrial fermentations, and the pathways and genes responsible for the production of these solvents have been reported (Girbal etal., Trends in Biotechnology 16:11-16 (1998)). The actual fermentation, however, has been quite complicated and difficult to control. ABE fermentation has declined continuously since the 1950s, and almost all butanol is now produced via petrochemical routes, as described above. In a typical ABE fermentation, butyric, propionic, lactic and acetic acids are first produced by C. acetobutylicum, the culture pH drops and undergoes a metabolic "butterfly" shift, and 1-butanol, acetone, isopropanol and ethanql are then formed.   In conventional ABE fermentations, the 1-butanol yield from glucose is low, typically around 15 percent and rarely exceeding 25 percent. Consequently, the 1-butanol concentration in conventional ABE fermentations is usually lower than 1.3 percent.<br>
Attempts to maximize 1-butanol production from the ABE process by the elimination of all the other solvent by-products have not been totally successful, and thus, the process produces significant amounts of acetone which is not useful as a gasoline additive. A process for the fermentive production of butanol where 1-butanol is the sole product would represent an advance in the art.<br>
There is a need, therefore, for an environmentally responsible, cost-effective process for the production of 1-butanol as a single product The present invention addresses this need through the discovery of a recombinant microbial production host expressing a 1-butanol biosynthetic pathway.<br>
SUMMARY OF THE INVENTION<br>
The invention provides a recombinant microorganism having an engineered 1-butanol biosynthetic pathway. The engineered microorganism may be used for the commercial production of 1-butanol. Accordingly the invention provides a recombinant microbial host cell comprising at least one DNA molecule encoding a polypeptide that<br>
 to product conversion selected from the group consisting of:<br>
a)	acetyl-CoA to acetoacetyl-CoA<br>
b)	acetoacetyl-CoA to 3-hydroxybutyryl-CoA<br>
c)	3-hydroxybutyryl-CoA to crotonyl-CoA<br>
d)	crotonyl-CoA to butyryl-CoA<br>
e)	butyryl-CoA to butyraldehyde and<br>
f)	butyraldehyde to1-butanol;<br>
wherein the at least one DNA molecule is heterologous to said microbial host cell and wherein said microbial host cell, produces 1-butanol.<br>
In another embodiment the invention provides a method for the production of 1-butanol comprising:<br>
i) providing a recombinant microbial host cell comprising at least one DNA molecule encoding a polypeptide that catalyzes a substrate to product conversion selected from the group consisting of:<br>
a)	acetyl-CoA to acetoacetyl-CoA<br>
b)	acetoacetyl-CoA to 3-hydroxybutyryl-CoA<br>
cj 3-hydroxybutyryl-CoA to crotonyl-CoA<br><br>
d)	crotonyl-CoA to butyryl-CoA<br>
e)	butyryl-CoA to butyraldehyde and<br>
f)	butyraldehyde to1-butanol;<br>
wherein the at least one DNA molecule is heterologous to said microbial host cell; and<br>
ii) contacting the host cell of (i) with a fermentable carbon substrate<br>
under conditions whereby 1-butanol is produced.<br>
BRIEF DESCRIPTION OF THE FIGURES AND SEQUENCE DESCRIPTIONS<br>
The invention can be more fully understood from the following detailed description, figure, and the accompanying sequence descriptions, which form a part of this application.<br>
Figure 1 shows the 1-butanol biosynthetic pathway. The steps labeled "a", "b", "c", "d", "e", and "f represent the substrate to product conversions described below.<br>
Sequences conform with 37 C.F.R. 1.821-1.825 ("Requirements for Patent Applications Containing Nucleotide Sequences and/or Amino Acid Sequence Disclosures - the Sequence Rules") and are consistent with World Intellectual Property Organization (WIPO) Standard ST.25 (1998) and the sequence.listing requirements of the EPO and PCT (Rules 5.2 and 49.5(a-bis), and Section 208 and Annex C of the Administrative Instructions). The symbols and format used for nucleotide and amino acid sequence data comply with the rules set forth in 37 C.F.R. §1.822.<br>
Table 1 Summary of Gene and Protein SEQ ID Numbers<br>
(Table Removed)<br>
SEQ ID N0s:17-44 are the nucleotide sequences of oligonucleotide primers used to amplify the genes of the 1-butanol biosynthetic pathway.<br>
SEQ ID NOs:45-72 are the nucleotide sequences of oligonucleotide primers used for sequencing.<br>
SEQ ID NOs:73-75 are the nucleotide sequences of oligonucleotide primers used to construct the transformation vectors described in Example 9.<br>
SEQ ID N0:76 is the nucleotide sequence of the codon-optimized CAC0462 gene referred to herein as CaTER.<br>
SEQ ID N0:77 is the nucleotide sequence of the codon-optimized EgTER gene, referred to herein as EgTER(opt).<br>
SEQ ID N0:78 is the nucleotide sequence of the codon-optimized a/of gene, referred to herein as aid (opt).<br>
SEQ ID N0:79 is the nucleotide sequence of the plasmid pFP988.<br>
SEQ ID N0:'s 80-127,160-185, and 190-207 are the nucleic acid sequences of cloning, sequencing, or PCR screening primers used for the cloning, sequencing, or screening of the genes of the 1-butanol biosynthetic pathway described herein, and are more fully described in<br>
Tables 4 and 5.<br>
SEQ ID NO:156 is the nucleotide sequence of the cscBKA gene<br>
cluster.<br>
SEQ ID NO: 157 is the amino acid sequence of sucrose hydrolase<br>
(CscA).<br>
SEQ ID N0:158 is the amino acid sequence of D-fructokinase<br>
(CscK).<br>
SEQ ID N0:159 is the amino acid sequence of sucrose permease<br>
(CscB).<br><br>
is the nucleotide sequence of the codon optimized tery gene described in Example 17.<br>
SEQ ID NO: 187 is the amino acid sequence of the butyl-CoA dehydrogenase (ter) encoded by the codon optimized tery gene (SEQ ID N0:186).<br>
SEQ ID NO: 188 is the nucleotide sequence of the codon optimized aldy gene described in Example 17.<br>
SEQ ID N0:189 is the amino acid sequence of the butyraldehyde dehydrogenase (aid) encoded by the codon optimized aldy gene (SEQ ID N0:188).<br>
SEQ ID NO:208 is the nucleotide sequence of the template DNA used in Example 14.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention relates to methods for the production of 1-butanol using recombinant microorganisms. The present invention meets a number of commercial and industrial needs. Butanol is an important industrial commodity chemical with a variety of applications, where its potential as a fuel or fuel additive is particularly significant. Although only a four-carbon alcohol, butanol has an energy content similar to that of gasoline and can be blended with any fossil fuel. Butanol is favored as a fuel or fuel additive as it yields only CO2 and little or no SOx or NOx when burned in the standard internal combustion engine. Additionally butanol is less corrosive than ethanol, the most preferred fuel additive to date.<br>
In addition to its utility as a biofuel or fuel additive, butanol has the potential of impacting hydrogen distribution problems in the emerging fuel cell industry. Fuel cells today are plagued by safety concerns associated with hydrogen transport and distribution. Butanol can be easily reformed for its hydrogen content and can be distributed through existing gas stations in the purity required for either fuel cells or vehicles.<br>
Finally the present invention produces butanol from plant derived carbon sources, avoiding the negative environmental impact associated with standard petrochemical processes for butanol production.<br>
The following definitions and abbreviations are to be used for the interpretation of the claims and the specification.<br>
invention" or "present invention" as used herein is a non-limiting term and is not intended to refer to any single embodiment of the particular invention but encompasses all possible embodiments as described in the specification and the claims.<br>
"ABE" is the abbreviation for the Acetone-Butanol-Ethanol fermentation process.<br>
The term "1-butanol biosynthetic pathway" means the enzyme pathway to produce 1-butanol from acetyl-coenzyme A (acetyl-CoA).<br>
The term "acetyl-CoA acetyltransferase" refers to an enzyme that catalyzes the conversion of two molecules of acetyl-CoA to acetoacetyl-CoA and coenzyme A (CoA). Preferred acetyl-CoA acetyltransferases are acetyl-CoA acetyltransferases with substrate preferences (reaction in the forward direction) for a short chain acyl-CoA and acetyl-CoA and are classified as E.G. 2.3.1.9 [Enzyme Nomenclature 1992, Academic Press, San Diego]; although, enzyme's with a broader substrate range (E.G. 2.3.1.16) will be functional as well. Acetyl-CoA acetyltransferases are available from a number of sources, for example, Escherichia coli (GenBank Nos: NP_416728 (SEQ ID NO:129), NC_000913 (SEQ ID N0:128); NCBI (National Center for Biotechnology Information) amino acid sequence, NCBI nucleotide sequence), Clostridium acetobutylicum (GenBank Nos: NP_349476.1 (SEQ ID N0:2), NC_003030 (SEQ ID N0:1); NPJ49242 (SEQ ID N0:4), NC_001988 (SEQ ID N0:3), Bacillus subtilis (GenBank Nos: NP_390297 (SEQ ID NO:131), NC_000964 (SEQ ID NO:130)), and. Saccharomyces cerevisiae (GenBank Nos: NP_015297 (SEQ ID N0:133), NC_001148 (SEQ ID NO:132)).<br>
The term "3-hydroxybutyryl-CoA dehydrogenase" refers to an enzyme that catalyzes the conversion of acetoacetyl-CoA to 3-hydroxybutyryl-CoA. 3-Hydroxybutyryl-CoA dehydrogenases may be reduced nicotinamide adenine dinucleotide (NADH)-dependent, with a substrate preference for (S)-S-hydroxybutyryl-CoA or (R)-3-hydroxybutyryl-CoA and are classified as E.G. 1.1.1.35 and E.G. 1.1.1.30, respectively. Additionally, 3-hydroxybutyryl-CoA dehydrogenases may be reduced nicotinamide adenine dinucleotide phosphate (NADPH)-dependent, with a<br>
for (S)-3-hydroxybutyryl-CoA or (R)-3-hydroxybutyryl-CoA and are classified as E.G. 1.1.1.157 and E.G. 1.1.1.36, respectively. 3-Hydroxybutyryl-CoA dehydrogenases are available from a number of sources, for example, C. acetobutylicum (GenBank NOs: NP_349314 (SEQ ID N0:6), NC_003030 (SEQ ID N0:5)), a subtilis (GenBank NOs: AAB09614 (SEQ ID N0:135), U29084 (SEQ ID N0:134)), Ralstonia eutropha (GenBank NOs: YP_294481 (SEQ ID N0:137), IMC_007347 (SEQ ID N0:136)), and Alcaligenes eutrophus (GenBank NOs: AAA21973 (SEQ ID N0:139), J04987 (SEQ ID NO:138)).<br>
The term "crotonase" refers to an enzyme that catalyzes the conversion of 3-hydroxybutyryl-CoA to crotonyl-CoA and HzO. Crotonases may have a substrate preference for (S)-3-hydroxybutyryl-CoA or (R)-3-hydroxybutyryl-CoA and are classified as E.G. 4.2.1.17 and E.G. 4.2.1.55, respectively. Crotonases are available from a number of sources, for example, £ cb//(GenBank NOs: NP_415911 (SEQ ID NO:141), NC_000913 (SEQ ID N0:140)), C. acetobutylicum (GenBank NOs: NP_349318 (SEQ ID N0:8), NC_003030 (SEQ ID N0:6)), B. subtilis (GenBank NOs.: CAB13705 (SEQ ID N0:143), Z99113 (SEQ ID NO:142)), and Aeromonas caviae (GenBank NOs: BAA21816 (SEQ ID NO:145), D88825 (SEQ ID NO: 144)).<br>
The term "butyryl-CoA dehydrogenase" refers to an enzyme that catalyzes the conversion of crotonyl-CoA to butyryl-CoA. Butyryl-CoA dehydrogenases may be either NADH-dependent or NADPH-dependent and are classified as E.G. 1.3.1.44 and E.G. 1.3.1.38, respectively. Butyryl-CoA dehydrogenases are available from a number of sources, for example, C. acetobutylicum (GenBank NOs: NP_347102 (SEQ ID NO:10), NC_003030 (SEQ ID N0:9))), Euglena gracilis (GenBank NOs: Q5EU90 SEQ ID N0:147), AY741582 SEQ ID N0:146)), Streptomyces collinus (GenBank NOs: AAA92890 (SEQ ID NO:149), U37135 (SEQ ID N0:148)), and Streptomyces coelicolor (GenBank NOs: CAA22721 (SEQ ID N0:151), AL939127 (SEQ ID NO:150)).<br>
The term "butyraldehyde dehydrogenase" refers to an enzyme that catalyzes the conversion of butyryl-CoA to butyraldehyde, using NADH or NADPH as cofactor. Butyraldehyde dehydrogenases with a preference for<br>
E.C. 1.2.1.57 and are available from, for example, Clostridium beijerinckli (GenBank NOs: AAD31841 (SEQ ID NO:12), AF157306 (SEQ ID NO:11)) and C. acetobutylicum (GenBank NOs: NPJ49325 (SEQ ID NO:153), NC_001988 (SEQ ID NO:152)).<br>
The term "butanol dehydrogenase" refers to an enzyme that catalyzes the conversion of butyraldehyde to 1-butanol, using either NADH or NADPH as cofactor. Butanol dehydrogenases are available from, for example, C. acetobutylicum (GenBank NOs: NP_149325 (SEQ ID NO:153), NC_001988 SEQ ID NO:152; note: this enzyme possesses both aldehyde and alcohol dehydrogenase activity); NP_349891 (SEQ ID NO:14), NC_003030 (SEQ ID NO:13); and NP_349892 (SEQ ID NO:16), NC_003030 (SEQ ID N0:15)) and £ co//(GenBank NOs: NP_417484 (SEQ ID N0:155), NC_000913 (SEQ ID N0:154)).<br>
The term "a facultative anaerobe" refers to a microorganism that can grow in both aerobic and anaerobic environments.<br>
The term "carbon substrate" or "fermentable carbon substrate" refers to a carbon source capable of being metabolized by host organisms of the present invention and particularly carbon sources selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and one-carbon substrates or mixtures thereof.<br>
The term "gene" refers to a nucleic acid fragment that is capable of being expressed as a specific protein, optionally including regulatory sequences preceding (5' non-coding sequences) and following (3' non-coding sequences) the coding sequence. "Native gene" refers to a gene as found in nature with its own regulatory sequences. "Chimeric gene" refers to any gene that is not a native gene, comprising regulatory and coding sequences that are not found together in nature. Accordingly, a chimeric gene may comprise regulatory sequences and coding sequences that are derived from different sources, or regulatory sequences and coding sequences derived from the same source, but arranged in a manner different than that found in nature. "Endogenous gene" refers to a native gene in its natural location in the genome of an organism. A "foreign" or "heterologous gene" refers to a gene not normally found in the host organism, but that is introduced into the host organism by gene genes can comprise native genes inserted into a non-native organism, or chimeric genes. A "transgene" is a gene that has been introduced into the genome by a transformation procedure.<br>
As used herein the term "coding sequence" refers to a DNA sequence that codes for a specific amino acid sequence. "Suitable regulatory sequences" refer to nucleotide sequences located upstream (5' non-coding sequences), within, or downstream (3' non-coding sequences) of a coding sequence, and which influence the transcription, RNA processing or stability, or translation of the associated coding sequence. Regulatory sequences may include promoters, translation leader sequences, introns, polyadenylation recognition sequences, RNA processing site, effector binding site and stem-loop structure.<br>
The term "promoter" refers to a DNA sequence capable of controlling the expression of a coding sequence or functional RNA. In general, a coding sequence is located 3' to a promoter sequence. Promoters may be derived in their entirety from a native gene, or be composed of different elements derived from different promoters found in nature, or even comprise synthetic DNA segments. It is understood by those skilled in the art that different promoters may direct the expression of a gene in different tissues or cell types, or at different stages of development, or in response to different environmental or physiological conditions. Promoters which cause a gene to be expressed in most cell types at most times are commonly referred to as "constitutive promoters". It is further recognized that since in most cases the exact boundaries of regulatory sequences have not been completely defined, DNA fragments of different lengths may have identical promoter activity.<br>
The term "operably linked" refers to the association of nucleic acid sequences on a single nucleic acid fragment so that the function of one is affected by the other. For example, a promoter is operably linked with a coding sequence when it is capable of affecting the expression of that coding sequence (i.e., that the coding sequence is under the transcriptional control of the promoter). Coding sequences can be operably linked to regulatory sequences in sense or antisense orientation.<br>
Expression", as used herein, refers to the transcription and stable accumulation of sense (mRNA) or antisense RNA derived from the nucleic acid fragment of the invention. Expression may also refer to translation of mRNA into a polypeptide.<br>
As used herein the term 'transformation" refers to the transfer of a nucleic acid fragment into a host organism, resulting in genetically stable inheritance. Host organisms containing the transformed nucleic acid fragments are referred to as "transgenic" or "recombinant" or "transformed" organisms.<br>
The terms "plasmid", "vector" and "cassette" refer to an extra chromosomal element often carrying genes which are not part of the central metabolism of the cell, and usually in the form of circular double-stranded DNA fragments. Such elements may be autonomously replicating sequences, genome integrating sequences, phage or nucleotide sequences, linear or circular, of a single- or double-stranded DNA or RNA, derived from any source, in which a number of nucleotide sequences have been joined or recombined into a unique construction<br>
which is capable of introducing a promoter fragment and DNA sequence for a selected gene product along with appropriate 3' untranslated sequence into a cell. Transformation cassette" refers to a specific vector containing a foreign gene and having elements in addition to the foreign gene that facilitates transformation of a particular host cell. "Expression cassette" refers to a specific vector containing a foreign gene and having elements in addition to the foreign gene that allow for enhanced expression of that gene in a foreign host.<br>
As used herein the term "codon degeneracy" refers to the nature in the genetic code permitting variation of the nucleotide sequence without effecting the amino acid sequence of an encoded polypeptide. The skilled artisan is well aware of the "codon-bias" exhibited by a specific host cell in usage of nucleotide codons to specify a given amino acid. Therefore, when synthesizing a gene for improved expression in a host cell, it is desirable to design the gene such that its frequency of codon usage approaches the frequency of preferred codon usage of the host cell.<br>
" as it refers to genes or coding regions of nucleic acid molecules for transformation of various hosts, refers to the alteration of codons in the gene or coding regions of the nucleic acid molecules to reflect the typical codon usage of the host organism without altering the polypeptide encoded by the DNA.<br>
Standard recombinant DNA and molecular cloning techniques used here are well known in the art and are described by Sambrook, J., Fritsch, E. F. and Maniatis, T., Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989) (hereinafter "Maniatis"); and by Silhavy, T. J., Bennan, M. L and Enquist, L. W., Experiments with Gene Fusions, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1984); and by Ausubel, F. M. et al., Current Protocols in Molecular Biology, published by Greene Publishing Assoc. and Wiley-lnterscience (1987). The 1-Butanol Biosvnthetic Pathway<br>
Carbohydrate utilizing microorganisms employ the Embden-Meyerhof-Parnas (EMP) pathway, the Entner-Doudoroff pathway and the pentose phosphate cycle as the central, metabolic routes to provide energy and cellular precursors for growth and maintenance. These pathways have in common the intermediate glyceraldehyde-3-phosphate and, ultimately, pyruvate is formed directly or in combination with the EMP pathway. Subsequently, pyruvate is transformed to acetyl-coenzyme A (acetyl-CoA) via a variety of means, including reaction with the pyruvate dehydrogenase complex, pyruvate-formate lyase, and pyruvate-ferredoxin oxidoreductase. Acetyl-CoA serves as a key intermediate, for example, in generating fatty acids, amino acids and secondary metabolites. The combined reactions of sugar conversion to acetyl-CoA produce energy (e.g. adenosine-5'-triphosphate, ATP) and reducing equivalents (e.g. reduced nicotinamide adenine dinucleotide, NADH, and reduced nicotinamide adenine dinucleotide phosphate, NADPH). NADH and NADPH must be recycled to their oxidized forms (NAD* and NADP+, respectively). In the presence of inorganic electron acceptors (e.g. O2, NO3" and SO42"), the reducing equivalents may be used to augment the energy pool; alternatively, a reduced carbon by product may be formed.<br>
ethanol and 1-butanol resulting from the fermentation of carbohydrate are examples of the latter.<br>
This invention enables the production of 1-butanol from carbohydrate sources with recombinant microorganisms by providing a complete 1-butanol biosynthetic pathway from acetyl-CoA to1-butanol, as shown in Figure 1. This biosynthetic pathway, generally lacking in the rnicrobial community due to the absence of genes or the lack of appropriate gene regulation, comprises the following substrate to product conversions:<br>
a)	acetyl-CoA to acetoacetyl-CoA, as catalyzed for example by<br>
acetyl-CoA acetyltransferase;<br>
b)	acetoacetyl-CoA to 3-hydroxybutyryl-CoA, as catalyzed for<br>
example by 3-hydroxybutyryl-CoA dehydrogenase;<br>
c)	3-hydroxybutyryl-CoA to crotonyl-CoA, as catalyzed for example<br>
by crotonase;<br>
d)	crotonyl-CoA to butyryl-CoA, as catalyzed for example by<br>
butyryl-CoA dehydrogenase;<br>
e)	butyryl-CoA to butyraldehyde, as catalyzed for example by<br>
butyraldehyde dehydrogenase; and<br>
f)	butyraldehyde to1-butanol, as catalyzed for example by butanol<br>
dehydrogenase.<br>
The pathway requires no ATP and generates NAD* and/or NADP+, thus, balances with the central metabolic routes that generate acetyl-CoA. The ability of natural organisms to produce -1-butanol by fermentation is  . rare and exemplified most prominently by Clostridium beijerinckii and Clostridium acetobutylicum. The gene organization and gene regulation for Clostridium acetobutylicum has been described (L. Girbal and P. Soucaille, Trends in Biotechnology 216:11 -16 (.1998)). However, many of these enzyme activities are associated also with alternate pathways, for example, hydrocarbon utilization, fatty acid oxidation, and poly-hydroxyalkanoate metabolism. Thus, in providing a recombinant pathway from acetyl-CoA to 1-butanol, there exist a number of choices to fulfill the individual reaction steps, and the person of skill in the art will be able to utilize publicly available sequences to construct the relevant pathways. A entative number of genes known in the art and useful in the construction of the 1-butanol biosynthetic pathway are listed below in Table 2.<br>
Table 2 Sources of 1-Buatnol  <br>
(Table Removed)<br>
Microbial Hosts for 1-Butanol Production<br>
Microbial hosts for 1-butanol production may be selected from bacteria, cyanobacteria, filamentous fungi and yeasts. The microbial host' used for 1-butanol production is preferably tolerant to 1-butanol so that the yield is not limited by butanol toxicity. Microbes that are metabolically active at high titer leyels of 1 -butanol are not well known in the art. Although butanol-tolerant mutants have been isolated from solventogenic Clostridia, little information is available concerning the butanol tolerance of other potentially useful bacterial strains. Most of the studies on the comparison of alcohol tolerance in bacteria suggest that butanol is more toxic than ethanol (de Cavalho et al., Morose. Res. Tech. 64:215-22 (2004) and Kabelitz et al., FEMS Microbiol. Lett. 220:223-227 (2003)). Tomas et al. (J. Bacteriol. 186:2006-2018 (2004)) report that the yield of butanol during fermentation in Clostridium acetobutylicum may be limited by butanol toxicity. The primary effect of butanol on Clostridium acetobutylicum is disruption of membrane functions (Hermann et al., Appl. Environ. Microbiol. 50:1238-1243 (1985)).<br>
The microbial hosts selected for the production of 1-butanol are preferably tolerant to 1-butanol and are able to convert carbohydrates to 1-butanol.  The criteria for selection of suitable microbial hosts include the following: intrinsic tolerance to 1-butanol, high rate of glucose utilization, availability of genetic tools for gene manipulation, and the ability to generate stable chromosomal alterations.<br>
Suitable host strains with a tolerance for 1-butanol may be identified by screening based on the intrinsic tolerance of the strain. The intrinsic tolerance of microbes to 1-butanol may be measured by determining the concentration of 1-butanol that is responsible for 50% inhibition of the growth rate (IC50) when grown in a minimal medium. The IC50 values may be determined using methods known in the art. For example, the microbes of interest may be grown in the presence of various amounts of  measuring tne optical density at 600 nanometers. The doubling time may be calculated from the logarithmic part of the growth curve and used as a measure of the growth rate. The concentration of 1-butanol that produces 50% inhibition of growth may be determined from a graph of the percent inhibition of growth versus the 1-butanol concentration. Preferably, the host strain should have an IC50 for 1-butanol of greater than about 0.5% weight/volume.<br>
The microbial host for 1-butanol production should also utilize glucose at a high rate. Most microbes are capable of utilizing carbohydrates. However, certain .environmental microbes cannot utilize carbohydrates to high efficiency, and therefore would not be suitable hosts.<br>
The ability to genetically modify the host is essential for the production of any recombinant microorganism. The mode of gene transfer technology may be by electroporation, conjugation, transduction or natural transformation. A broad range of host conjugative plasmids and drug resistant markers are available. The cloning vectors are tailored to the host organisms based on the nature of antibiotic resistance markers that can function in that host.<br>
The microbial host also has to be manipulated in order to inactivate competing pathways for carbon flow by deleting various genes. This requires the availability of either transposons to direct inactivation or chromosomal integration vectors.  Additionally, the production host should be amenable to chemical mutagenesis so that mutations to improve intrinsic 1-butanol tolerance may be obtained.<br>
Based on the criteria described above, suitable microbial hosts for the production of 1-butanol include, but are not limited to, members of the genera Clostridium, Zymomonas, Escherichia, Salmonella, Rhodococcus, Pseudomonas, Bacillus, Lactobacillus, Enterococcus, Alcaligenes, Klebsiella, Paenibacillus, Arthrobacter, Corynebacterium, Brevibacterium, Pichia, Candida, Hansenula and Saccharomyces. Preferred hosts include: Eschehchia coli, Alcaligenes eutrophus, Bacillus licheniformis, Paenibacillus macerans, Rhodococcus erythropolis, Pseudomonas putida,tarum, Enterococcus faecium, Enterococcus gallinarium, Enterococcus faecalis, Bacillus subtilis and Saccharomyces cerevisiae. Construction of Production Host<br>
Recombinant organisms containing the necessary genes that will encode the enzymatic pathway for the conversion of a fermentable carbon substrate to 1-butanol may be constructed using techniques well known in the art. In the present invention, genes encoding the enzymes of the 1-butanol biosynthetic pathway, I.e., acetyl-CoA acetyltransferase, 3-hydroxybutyryl-CoA dehydrogenase, crotonase, butyryl-CoA dehydrogenase, butyraldehyde dehydrogenase, and butanol dehydrogenase, may be isolated from various sources, as described above.<br>
Methods of obtaining desired genes from a bacterial genome are common and well known in the art of molecular biology. For example, if the sequence of the gene is known, suitable genomic libraries may be created by restriction endonuclease digestion and may be screened with probes complementary to the desired gene sequence. Once the sequence is isolated, the DNA may be amplified using standard primer-directed amplification methods such as polymerase chain reaction (Mullis, U.S. Patent No. 4,683,202) to obtain amounts of DNA suitable for transformation using appropriate vectors. Tools for codon optimization for expression in a heterologous host are readily available. Some tools for codon optimization are available based on the GC content of the host organism. The GC content of some exemplary microbial hosts is given Table 3.<br>
Table 3 GC Content of Microbial Hosts<br><br>
(Table Removed)<br>
Once the relevant pathway genes are identified and isolated they may be transformed into suitable expression hosts by means well known in the art. Vectors or cassettes useful for the transformation of a variety of host cells are common and commercially available from companies such<br>
as EPICENTRE® (Madison, Wl), Invitrogen Corp. (Carlsbad, CA), Stratagene (La Jolla, CA), and New England Biolabs, Inc. (Beverly, MA). Typically, the vector or cassette contains sequences directing transcription and translation of the relevant gene, a selectable marker, and sequences allowing autonomous replication or chromosomal integration. Suitable vectors comprise a region 5' of the gene which harbors transcriptional initiation controls and a region 3' of the DNA fragment which controls transcriptional termination. Both control regions may be derived from genes homologous to the transformed host cell, although it is to be understood that such control regions may also be derived from genes that are not native to the specific species chosen as a production host.<br>
Initiation control regions or promoters, which are useful to drive expression of the relevant pathway coding regions in the desired host cell are numerous and familiar to those skilled in the art. Virtually any promoter capable of driving these genetic elements is suitable for the present invention including, but not limited to, CYC1, HIS3, GAL1, GAL10, ADH1, PGK, PH05, GAPDH, ADC1, TRP1, URA3, LEU2, ENO, TPI, CUP1, FBA, GPD, and GPM (useful for expression in Saccharomyces); AOX1 (useful for expression in Pichia); and lac, ara, tet, trp, IPL, PR, 77, tac, and trc (useful for expression in Escherichia coli, Alcaligenes, and Pseudomonas); the amy, apr, npr promoters and various phage promoters expression in Bacillus subtilis, Bacillus licheniformis, and Paenibacillus macerans; nisA (useful for expression Gram-positive bacteria, Eichenbaum et al. Appl. Environ. Microbiol. 64(8):2763-2769 (1998)); and the synthetic P11 promoter (useful for expression in Lactobacillus plantarum, Rud et al., Microbiology 152:1011-1019 (2006)).<br>
Termination control regions may also be derived from various genes native to the preferred hosts. Optionally, a termination site may be unnecessary, however, it is most preferred if included.<br>
Certain vectors are capable of replicating in a broad range of host bacteria and can be transferred by conjugation. The complete and annotated sequence of pRK404 and three related vectors-pRK437, pRK442, and pRK442(H) are available . These derivatives have proven to be valuable tools for genetic manipulation in Gram-negative bacteria (Scott et al., Plasmid 50(1 ):74-79 (2003)).   Several plasmid derivatives of broad-host-range Inc P4 plasmid RSF1010 are also available with promoters that can function in a range of Gram-negative bacteria. Plasmid pAYC36 and pAYC37, have active promoters along with multiple cloning sites to allow for the heterologous gene expression in Gram-negative bacteria.<br>
Chrompsomal gene replacement tools are also widely available. For example, a thermosensitive variant of the broad-host-range replicon pWV101 has been modified to construct a plasmid pVE6002 which can be used to create gene replacement in a range of Gram-positive bacteria (Maguin et al., J. Bacteriol. 174(17):5633-5638 (1992)). Additionally, in vitro transposomes are available to create random mutations in a variety of genomes from commercial sources such as EPICENTRE®.<br>
The expression of the 1-butanol biosynthetic pathway in various preferred microbial hosts is described in more detail below.<br>
Expression of the 1-butanol biosvnthetic pathway in E. coll Vectors or cassettes useful for the transformation of £ coll are common and commercially available from the companies listed above. For example, the genes of the 1-butanol biosynthetic pathway may be isolated from various strains of Clostridium, cloned into a modified pUC19 into E.Coli/NM522, as described in txample 11. The expression of the 1-butanol biosynthetic pathway in several other strains of E. coli is described in Example 13.<br>
Expression of the 1-butanol biosvnthetic pathway in Rhodococcus<br>
ervthropolis<br>
A series of E. coli-Rhodococcus shuttle vectors are available for expression in R. erythropolis, including, but not limited to pRhBR17 and pDA71 (Kostichka et al., Appl. Microbiol. Biotechnol 62:61-68(2003)). Additionally, a series of promoters are available for heterologous gene expression in R. erythropolis (see for example Nakashima et al., Appl. Envir. Microbiol 70:5557-5568 (2004), and Tao et al., Appl. Microbiol. Biotechnol. 2005, DO! 10.1007/s00253-005-0064).  Targeted gene disruption of chromosomal genes in R. erythropolis may be created using the method described by Tao et al., supra, and Brans et al. (Appl. Envir. Microbiol. 66: 2029-2036 (2000)).<br>
The heterologous genes required for the production of 1-butanol, as described above, may be cloned initially in pDA71 or pRhBR71 and transformed into E. coli. The vectors may then be transformed into R. erythropolis by electroporation, as described by Kostichka et al., supra. The recombinants may be grown in synthetic medium containing glucose and the production of 1-butanol can be followed using methods known in the art.<br>
Expression of the 1-butanol biosynthetic pathway in Bacillus Subtilis<br>
Methods for gene expression and creation of mutations in B. Subtilis are also well known in the art. For example, the genes of the 1-butanol biosynthetic pathway may be isolated from various strains of Clostridium, cloned into a modified pUC19 vector and transformed into Bacillus subtilis BE1010, as described in Example 12. Additionally, the six genes of the 1-biosynthetic pathway can be split into two operons for expression, as described in Example 14. The first three genes of the pathway (thl, hbd, and erf) were integrated into the chromosome of Bacillus subtilis BE1010 (Payne and Jackson, J. Bacteriol. 173:2278-2282 (1991)). The last three genes (EgTER, aid, and bdhB) were cloned into Bs and transformed into the Bacillus strain carrying the integrated 1-butanol genes<br>
Expression of the 1-butanol biosvnthetic pathway in Bacillus licheniformis<br>
Most of the plas'mids and shuttle vectors that replicate in B. subtilis may be used to transform 6, licheniformis by either protoplast transformation or electroporation. For example, the genes required for the production of 1-butanol may be cloned in plasmids pBE20 or pBE60 derivatives (Nagarajan et al., Gene 114:121-126 (1992)). Methods to transform B. licheniformis are known in the art (for example see Fleming et al. Appl. Environ. Microbiol., 61 (11 ):3775-3780 (1995)).   The plasmids constructed for expression in B. subtilis may also be transformed into B. licheniformis to produce a recombinant microbial host that produces 1-butanol.<br>
Expression of the 1-butanol biosvnthetic pathway in Paenibacillus macerans<br>
Plasmids may be constructed as described above for expression in B. subtilis and used to transform Paenibacillus macerans by protoplast transformation to produce a recombinant microbial host that produces 1-butanol.<br>
Expression .of the 1-butanol biosvnthetic pathway in Alcaligenes (Ralstonia) eutrophus<br>
Methods for gene expression and creation of mutations in Ralstonia eutrophus are known in the art (see for example Taghavi et al., Appl. Environ. Microbiol., 60(10):3585-3591 (1994)). The genes for the 1-butanol biosynthetic pathway may be cloned in any of the broad host range vectors described above, and electroporated to generate recombinants that produce 1-butanol. The polyhydroxy butyrate pathway in Ralstonia has been described in detail and a variety of genetic techniques to modify the Ralstonia eutrophus genome is known, and those tools can be applied for engineering the 1-butanol biosynthetic pathway.<br><br>
4-butanol biosvnthetic pathway in Pseudomonas outida<br>
Methods for gene expression in Pseudomonas putida are known in the art (see for example Ben-Bassaf et al., U.S. Patent No. 6,586,229, which is incorporated herein by reference). For example, the butanol pathway genes may be inserted into pPCU18 and this ligated DNA may be electroporated into electrocompetent Pseudomonas putida DOT-T1 C5aAR1 cells to generate recombinants that produce 1-butanol.<br>
Expression of the 1-butanol biosvnthetic pathway in Saccharomvces cerevisiae<br>
Methods for gene expression in Saccharomyces cerevisiae are known in the art (see for example Methods in Enzymology, Volume 194, Guide to Yeast Genetics and Molecular and Cell Biology (Part A, 2004, Christine Guthrie and Gerald R. Fink (Eds.), Elsevier Academic Press, San Diego, CA). Expression of genes in yeast typically requires a promoter, followed by the gene of interest, and a transcriptional terminator. A number of yeast promoters can be used in constructing expression cassettes for genes encoding the 1-butanol biosynthetic pathway, including, but not limited to constitutive promoters FBA, GPD, and GPM, and the inducible promoters GAL1, GAL10, and CUP1. Suitable transcriptional terminators include, but are not limited to FBAt, GPDt, GPMt, ERGIOt, and GAL1t. Suitable promoters, transcriptional terminators, and the genes of the 1-butanol biosynthetic pathway may be cloned into yeast 2 micron (2µ) plasmids, as described in Example 17.<br>
Expression of the 1-butanol biosvnthetic pathway in Lactobacillus plantarum<br>
The Lactobacillus genus belongs to the Lactobacillales family and many plasmids and vectors used in the transformation of Bacillus subtilis and Streptococcus may be used for lactobacillus. Non-limiting examples of suitable vectors include pAM/91 and derivatives thereof (Renault et al., Gene 183:175-182 (1996); and O'Suliivan etal., Gene 137:227-231 (1993)); pMBB1 and pHWSOO, a derivative of pMBB1 (Wyckoff et al. AppL Environ. Microbiol. 62:1481-1486 (1996)); pMG1, a conjugative plasmid (Tanimoto et al., J. Bacteriol. 184:5800-5804 (2002)); pNZ9520<br>
1., Appl. Environ. Microbiol. 63:4581-4584 (1997)); pAM401 (Fujimoto et al., Appl. Environ. Microbiol. 67:1262-1267 (2001)); and pAT392 (Arthur et al., Antimicrob. Agents Chemother. 38:1899-1903 (1994)). Several ptaismids from Lactobacillus plantarum have also been reported (e.g., van Kranenburg R, Golic N, Bongers R, Leer RJ, de Vos WM, Siezen RJ, Kleerebezem M. Appl. Environ. Microbiol. 2005 Mar; 71(3): 1223-1230). For example, expression of the 1-butanol biosynthetic pathway in Lactobacillus plantarum is described in Example 18.<br>
Expression of the 1-butanol biosvnthetic pathway in Enterococcus faecium, Enterococcus aallinarium. and Enterococcus faecalis<br>
The Enterococcus genus belongs to the Lactobacillales family and many plasmids and vectors used in the transformation of Lactobacillus, Bacillus subtilis, and Streptococcus may be used for Enterococcus. Non-limiting examples of suitable vectors include pAM/?1 and derivatives thereof (Renault eta!., Gene 183:175-182 (1996); and O'Sullivan etal., Gene 137:227-231 (1993)); pMBB1 and pHW800, a derivative of pMBB1 (Wyckoff et al. Appl. Environ. Microbiol. 62:1481-1486 (1996)); pMG1, a conjugative plasmid (Tanimoto et al., J. Bacteriol. 184:5800-5804 (2002)); pNZ9520 (Kleerebezem et al., Appl. Environ. Microbiol. 63:4581-4584 (1997)); pAM401 (Fujimoto eial,Appl. Environ. Microbiol. 67:1262-1267 (2001)); and pAT392 (Arthur et al., Antimicrob. Agents Chemother. 38:1899-1903 (1994)). Expression vectors for E. faecalis using the nisA gene from Lactococcus may also be used (Eichenbaum et al., Appl. Environ. Microbiol. 64:2763-2769 (1998). Additionally, vectors for gene replacement in the £. faecium chromosome may be used (Nallaapareddy etal., Appl. Environ. Microbiol. 72:334-345 (2006)). For example, expression of the 1-butanol biosynthetic pathway in Enterococcus faecalis is described in Example 19. Fermentation Media<br>
Fermentation media in the present invention must contain suitable carbon substrates. Suitable substrates may include but are not limited to monosaccharides such as glucose and fructose, oligosaccharides such as lactose or sucrose, polysaccharides such as starch or cellulose or<br>
unpurified mixtures from renewable feedstocks such as cheese whey permeate, cornsteep liquor, sugar beet molasses, and barley malt. Additionally the carbon substrate may also be one-carbon substrates such as carbon dioxide, or methanol for which metabolic conversion into key biochemical intermediates has been demonstrated. In addition to one and two carbon substrates methylotrophic organisms are also known to utilize a number of other carbon containing compounds such as methylamine, glucosamine and a variety of amino acids for metabolic activity. For example, methylotrophic yeast are known to utilize the carbon from methylamine to form trehalose or glycerol (Beilion et al., Microb. Growth C1 Compd., [Int. Symp.], 7th (1993), 415-32. Editor(s): Murrell, J. Collin; Kelly, Don P. Publisher: Intercept, Andover, UK). Similarly, various species of Candida will metabolize alanine or oleic acid (Suiter et al., Arch. Microbiol. 153:485-489 (1990)). Hence it is contemplated that the source of carbon utilized in the present invention may encompass a wide variety of carbon containing substrates and will only be limited by the choice of organism.<br>
Although it is contemplated that all of the above mentioned carbon substrates and mixtures thereof are suitable in the present invention, preferred carbon substrates are glucose, fructose, and sucrose.<br>
In addition to an appropriate carbon source, fermentation media must contain suitable minerals, salts, cofactors, buffers and other components, known to those skilled in the art, suitable for the growth of the cultures and promotion of the enzymatic pathway necessary for 1-butanol production. Culture Conditions<br>
Typically cells are grown at a temperature in the range of about 25 °C to about 40 °C in an appropriate medium. Suitable growth media in the present invention are common commercially prepared media such as Luria Bertani (LB) broth, Sabouraud Dextrose (SD) broth or Yeast medium (YM) broth. Other defined or synthetic growth media may also be used and the appropriate medium for growth of the particular microorganism will be known by one skilled in the art of microbiology or fermentation science. The use of agents known to modulate catabolite repression directly or<br>
 adenosine 2':3'-monophosphate, may also be incorporated into the fermentation medium.<br>
Suitable pH ranges for the fermentation are between pH 5.0 to pH 9.0, where pH 6.0 to pH 8.0 is preferred as the initial condition.<br>
Fermentations may be performed under aerobic or anaerobic conditions, where anaerobic or microaerobic conditions are preferred.<br>
The amount of 1-butanol produced in the fermentation medium can be determined using a number of methods known in the art, for example, high performance liquid chromatography (HPLC) or gas chromatography (GC). Industrial Batch and Continuous Fermentations<br>
The present process employs a batch method of fermentation. A classical batch fermentation is a closed system where the composition of the medium is set at the beginning of the fermentation and not subject to artificial alterations during the fermentation. Thus, at the beginning of the fermentation the medium is inoculated with the desired organism or organisms, and fermentation is permitted to occur without adding anything to the system. Typically, however, a "batch" fermentation is batch with respect to the addition of carbon source and attempts are often made at controlling factors such as pH and oxygen concentration. In batch systems the metabolite and biomass compositions of the system change constantly up to the time the fermentation is stopped. Within batch cultures cells moderate through a static lag phase to a high growth log phase and finally to a stationary phase where growth rate is diminished or halted. If untreated, cells in the stationary phase will eventually die. Cells in log phase generally are responsible for the bulk of production of end product or intermediate.<br>
A variation on the standard batch system is the Fed-Batch system. Fed-Batch fermentation processes are also suitable in the present invention and comprise a typical batch system with the exception that the substrate is added in increments as the fermentation progresses. Fed-Batch systems are useful when catabolite repression is apt to inhibit the metabolism of the cells and where it is desirable to have limited amounts of substrate in the media. Measurement of the actual substrate<br>
Batch systems is difficult and is therefore estimated on the basis of the changes of measurable factors such as pH, dissolved oxygen and the partial pressure of waste gases such as CO2. Batch and Fed-Batch fermentations are common and well known in the art and examples may be found in Thomas D. Brock in Biotechnology: A Textbook of Industrial Microbiology, Second Edition (1989) Sinauer Associates, Inc., Sunderland, MA.r or Deshpande, Mukund V., Appl. Biochem. Biotechnol., 36:227, (1992), herein incorporated by reference.<br>
Although the present invention is performed in batch mode it is contemplated that the method would be adaptable to continuous fermentation methods. Continuous fermentation is an open system where a defined fermentation medium is added continuously to a bioreactor and an equal amount of conditioned media is removed simultaneously for processing. Continuous fermentation generally maintains the cultures at a constant high density where cells are primarily in log phase growth.<br>
Continuous fermentation allows for the modulation of one factor or any number of factors that affect cell growth or end product concentration. For example, one method will maintain a limiting nutrient such as the carbon source or nitrogen level at a fixed rate and allow all other parameters to moderate. In other systems a number of factors affecting growth can be altered continuously while the cell concentration, measured by media turbidity, is kept constant. Continuous systems strive to maintain steady state growth conditions and thus the cell loss due to the medium being drawn off must be balanced against the cell growth rate in the fermentation. Methods of modulating nutrients and growth factors for continuous fermentation processes as well as techniques for maximizing the rate of product formation are well known in the art of industrial microbiology and a variety of methods are detailed by Brock, supra.<br>
It is contemplated that the present invention may be practiced using either batch, fed-batch or continuous processes and that any known mode of fermentation would be suitable. Additionally, it is contemplated that cells may be immobilized on a substrate as whole cell catalysts and subjected to fermentation conditions for 1-butanol production.<br><br>
Isolation from the Fermentation Medium<br>
The bioproduced 1-butanol may be isolated from the fermentation medium using methods known in the art. For example, solids may be removed from the fermentation medium by centrifugation, filtration, decantation, .or the like. Then, the 1-butanol may be isolated from the fermentation medium, which has been treated to remove solids as described above, using methods such as distillation, liquid-liquid extraction, or membrane-based separation. Because 1-butanol forms a low boiling point, azeotropic mixture with water, distillation can only be used to separate the mixture up to its azeotropic composition. Distillation may be used in combination with another.separation method to obtain separation around the azeotrope. Methods that may be used in combination with distillation to isolate and purify 1-butanol include, but are not limited to, decantation, liquid-liquid extraction, adsorption, and membrane-based techniques. Additionally, 1-butanol may be isolated using azeotropic distillation using an entrainer (see for example Doherty and Malone, Conceptual Design of Distillation Systems, McGraw Hill, New York, 2001).<br>
The 1-butanol-water mixture forms a heterogeneous azeotrope so that distillation may be used in combination with decantation to isolate and purify the 1-butanol. In this method, the 1-butanol containing fermentation broth is distilled to near the azeotropic composition. Then, the azeotropic mixture is condensed, and the 1-butanol is separated from the fermentation medium by decantation. The decanted aqueous phase may be returned to the first distillation column as reflux. The 1-butanol-rich decanted organic phase may be further purified by distillation in a second distillation column.<br>
The 1-butanol may also be isolated from the fermentation medium using liquid-liquid extraction in combination with distillation. In this method, the 1-butanol is extracted from the fermentation broth using liquid-liquid extraction with a suitable solvent. The 1-butanol-containing organic phase is then distilled to separate the 1-butanol from the solvent.<br>
Distillation in combination with adsorption may also be used to<br>
the fermentation medium. In this method, the fermentation' broth containing the 1-butanol is distilled to near the azeotropic composition and then the remaining water is removed by use of an adsorbent, such as molecular sieves (Aden et al. Lignocellulosic Biomass to Ethanol Process Design and Economics Utilizing Co-Current Dilute Acid Prehydrolysis and Enzymatic Hydrolysis .for Corn Stover, Report NREL/TP-510-32438, National Renewable Energy Laboratory, June 2002).<br>
Additionally, distillation in combination with pervaporation may be . used to isolate and purify the 1-butanol from the fermentation medium. In this method, the fermentation broth containing the 1-butanol is distilled to near the azeotropic composition, and then the remaining water is removed by pervaporation through a hydrophilic membrane (Quo et al., J. Membr. Sci. 245,199-210(2004)).<br>
EXAMPLES<br>
The present invention is further defined in the following Examples. It should be understood that these Examples, while indicating preferred embodiments of the invention, are given by way of illustration only. From the above discussion and these Examples, one skilled in the art can ascertain the essential characteristics of this invention, and without departing from the spirit and scope thereof, can make various changes and modifications of the invention to adapt it to various uses and conditions. GENERAL METHODS<br>
Standard recombinant DNA and molecular cloning techniques used in the Examples are well known in the art and are described by Sambrook, J., Fritsch, E. F. and Maniatis, T. Molecular Cloning: A Laboratory Manual; Cold Spring Harbor Laboratory Press: Cold Spring Harbor, (1989) (Maniatis) and by T. J. Silhavy, M. L. Bennan, and L W. Enquist, Experiments with Gene Fusions, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1984) and byAusubel, F. M. et al., Current Protocols in Molecular Biology, pub. by Greene Publishing Assoc. and Wiley-lnterscience (1987).<br>
Methods suitable for the maintenance and growth of bacterial cultures are well known in the art. Techniques suitable for use in the following examples may be found as set out in Manual of Methods for General Bacteriology (Phiilipp Gerhardt, R. G. E. Murray, Ralph N. Costilow, Eugene W. Nester, Willis A. Wood, Noel R. Krieg and G. Briggs Phillips, eds), American Society for Microbiology, Washington, DC. (1994)) or by Thomas D. Brock in Biotechnology: A Textbook of Industrial Microbiology, Second Edition, Sinauer Associates, Inc., Sunderland, MA (1989). All reagents, restriction enzymes and materials used for the growth and maintenance of bacterial cells were obtained from Aldrich Chemicals (Milwaukee, Wl), BD Diagnostic Systems (Sparks, MD), Life Technologies (Rockville, MD), or Sigma Chemical Company (St. Louis, MO) unless otherwise specified.<br>
The oligonucleotide primers used for cloning in the following Examples are given in Table 4. The primers used to sequence or screen the cloned genes are given in Table 5. All the oligonucleotide primers were synthesized by Sigma-Genosys (Woodlands, TX).<br>
Table 4<br>
Oligonucleotide Cloning Primers<br>
(Table Removed)<br>
Table 5 Sequencing and PCR Screening Primers<br>
(Table Removed)<br>
Methods for Determining 1-Butanol Concentration in Culture Media<br>
The concentration of 1-butanol in the culture media can be determined by a number of methods known in the art. For example, a specific high performance liquid chromatography (HPLC) method utilized a Shodex SH-1011 column with a Shodex SH-G guard column, both purchased from Waters Corporation (Milford, MA), with refractive index (Rl) detection. Chromatographic separation was achieved using 0.01 M H2S04 as the mobile phase with a flow rate of 0.5 mL/min and a column temperature of 50 °C. 1-Butanol had a retention time of 52.8 min under the conditions used. Alternatively, gas chromatography (GC) methods are available. For example, a specific GC method utilized an HP-INNOWax column (30 m x 0.53 mm id,1 ;/m film thickness, Agilent Technologies, Wilmington, DE), with a flame ionization detector (FID). The carrier gas was helium at a flow rate of 4.5 mL/min, measured at 150 °C with constant head pressure; injector split was 1:25 at 200 °C; oven temperature was 45 °C for 1 min, 45 to 220 "C at 10 "C/min, and 220 °C for 5 min; and FID detection was employed at 240 °C with 26 mL/min helium makeup gas. The retention time of 1-butanol was 5.4 min. A similar GC method using a<br>
@rlin-'t2l?lIWMlSB(FFAP) CB column (25 m x 0.25 mm id X 0.2 pm film thickness, Varian, Inc., Palo Alto, CA) was also used.<br>
The meaning of abbreviations is as follows: "s" means second(s), "min" means minute(s), "h" means hour(s), "psi" means pounds per square inch, "nm" means nanometers, "d" means day(s), "uL" means microliter(s), "mL" means milliliter(s), V means iiter(s), "mm" means millimeter(s), "nm" means nanometers, "mM" means millimolar, "M" means molar, "mmol" means millimole(s), "umole" means micromole(s)", "g" means gram(s), "ug" means microgram(s) and "ng" means nanogram(s), "PCR" means polymerase chain reaction, "OD" means optical density, "ODeoo" means<br>
the optical density measured at a wavelength of 600 nm, ODsso" means<br>
the optical density measured at a wavelength of 550 nm, "kDa" means kilodaltons, "g" means the gravitation constant, "rprn" means revolutions per minute, "bp" means base pair(s), "kbp" means kilobase pair(s),"% w/v" means weight/volume percent, % v/v" means volume/volume percent, "HPLC" means high performance Hquid chromatography, and "GC" means gas chromatography.<br>
EXAMPLE 1 Cloning and Expression of Acetvl-CoA Acetvltransferase<br>
The purpose of this Example was to express the enzyme acetyi-CoA acetyltransferase, also referred to herein as acetoacetyl-CoA thiolase, in E. coll The acetoacetyl-CoA thiolase gene thIA was cloned from C. acetobutylicum (ATCC 824) and expressed in E. coll The thIA gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA using PCR, resulting in a 1.2 kbp product.<br>
The genomic DNA from Clostridium acetobutylicum (ATCC 824) was either purchased from the American Type Culture Collection (ATCC, Manassas, VA) or was isolated from Clostridium acetobutylicum (ATCC 824) cultures, as described below.<br>
Genomic DNA from Clostridium acetobutylicum (ATCC 824) was prepared from anaerobically grown cultures. The Clostridium strain was grown in 10 ml of Clostridial growth medium (Lopez-Contreras et al.,<br>
69(2), 869-877 (2003)) in stoppered and crimped 100 mL Bellco serum bottles (Bellco Glass Inc., Vineland, NJ) in an anaerobic chamber at 30 °C. The inoculum was a single colony from a 2X YTG plate (Kishii, et al., Antimicrobial Agents &amp; Chemotherapy, 47(1), 77-81 (2003)) grown in a 2.5 L MGC AnaeroPak™ (Mitsubishi Gas Chemical America Inc. New York, NY) at 37 °C.<br>
Genomic DNA was prepared using the Centra Puregene® kit (Centra Systems, Inc., Minneapolis, MN; catalog no. D-6000A) with modifications to the manufacturer's instruction (Wong et al., Current Microbiology, 32, 349-356 (1996)). The thIA gene was amplified from Clostridium acetobutylicum (ATCC 824) genomic DNA by PCR using primers N7 and N8 (see Table 4), given as SEQ ID NOs:21 and 22, respectively. Other PCR amplification reagents were supplied in manufacturers' kits for example, Kod HiFi DNA Polymerase (Novagen Inc., Madison, Wl; catalog no. 71805-3) and used according to the manufacturer's protocol. Amplification was carried out in a DNA Thermocycler GeneAmp 9700 (PE Applied Biosystems, Foster city, CA).<br>
For expression studies the Gateway cloning technology (Invitrogen Corp., Carlsbad, CA) was used. The entry vector pENTR/SD/D-TOPO allowed directional cloning and provided a Shine-Dalgamo sequence for the gene of interest. The destination vector pDEST14 used a T7 promoter for expression of the gene with no tag. The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD:TOPOthlA. The pENTR construct was transformed into E. co//Top10 (Invitrogen) cells and plated according to manufacturer's recommendations. Transformants were grown overnight and plasmid DNA was prepared using the QIAprep Spin Miniprep kit (Qiagen, Valencia, CA; catalog no. 27106) according to manufacturer's recommendations. Clones were submitted for sequencing with M13 Forward and Reverse primers (see Table 5), given as SEQ ID NOs:45 and 46, respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. Additional sequencing primers,<br>
(see Table 5), given as SEQ ID N0s:47 and 48, respectively, were needed to completely sequence the PCR product. The nucleotide sequence of the open reading frame (ORF) for this gene and the predicted amino acid sequence of the enzyme are given as SEQ ID N0:1 and SEQ ID NO:2, respectively.<br>
To create an expression clone, the thIA gene was transferred to the pDEST 14 vector by recombination to generate pDEST14th!A. The pDEST14thlA vector was transformed into BL21-A1 cells. Transformants were inoculated into LB medium supplemented with 50A/g/mL of ampiciliin and grown overnight.  An aliquot of the overnight culture was used to inoculate 50 ml of LB supplemented with 50 //g/mL of ampiciliin. The culture was Incubated at 37 °C with shaking until the OD6oo reached 0.6-0.8. The culture was split into two 25-mL cultures and arabinose was added to one of the flasks to a final concentration of 0.2% by weight. The negative control flask was not Induced with arabinose. The flasks were incubated for 4 h at 37 °C with shaking. Cells were harvested by centrifugation and the cell pellets were resuspended in 50 mM MOPS, pH 7.0 buffer.  The cells were disrupted either by sonication or by passage through a French Pressure Cell. The whole cell lysate was centrifuged yielding the supernatant or cell free extract and the pellet or the insoluble fraction. An aliquot of each fraction (whole cell lysate, cell free extract and insoluble fraction) was resuspended in SDS (MES) loading buffer (Invitrogen), heated to 85 °C for 10 min and subjected to SDS-PAGE analysis (NuPAGE 4-12% Bis-Tris Gel, catalog no. NP0322Box, Invitrogen). A protein of the expected molecular weight of about 41 kDa, as deduced from the nucleic acid sequence, was present in the induced culture but not in the uninduced control.<br>
Acetoacetyl-CoA thiolase activity in the cell free extracts was measured as degradation of a Mg2+-acetoacetyl-CoA complex by monitoring the decrease in absorbance at 303 nm. Standard assay conditions were 100 mM Tris-HCI pH 8.0,1 mM DTT (dithiothreitol) and 10<br>
mM MgCl2- The cocktail was equilibrated for 5 min at 37 °C; then the cell-free extract was added. The reaction was initiated with the addition of<br>
4-CoA plus 0.2 mM CoA. Protein concentration was measured by either the Bradford method or by the Bicinchoninic Kit (Sigma, catalog no. BCA-1). Bovine serum albumin (Bio-Rad, Hercules, CA) was used as the standard in both cases. In one typical assay, the specific activity of the ThIA protein in the induced culture was determined to be 16.0//mol mgm1 min'1 compared to 0.27 //mol mg"1 mirf1 in the uninduced culture.<br>
EXAMPLE 2 Cloning and Expression of Acetvl-CoA Acetvltransferase<br>
The purpose of this Example was .to express the enzyme acetyl-CoA acetyltransferase, also referred to herein as acetoacetyl-CoA thiolase, in £ co//. The acetoacetyl-CoA thiolase gene thIB was cloned from C. acetobutylicum (ATCC 824) and expressed in E. co//. The thIB gene was amplified from C, acetobutylicum (ATCC 824) genomic DNA using PCR.<br>
The thIB gene was cloned and expressed in the same manner as the thIA gene described in Example 1. The C. acetobutylicum (ATCC 824) genomic DNA was amplified by PCR using primers N15 and N16 (see Table 4), given as SEQ ID N0s:27 and 28, respectively, creating a 1.2 kbp product. The forward primer incorporated four bases (CCAC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasrnid pENTRSDD-TOPOthlB. Clones were submitted for sequencing with M13 Forward and Reverse primers, given as SEQ ID NOs:45 and 46 respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. Additional sequencing primers, N15SeqF1 and N16SeqR1 (see Table 5), given as SEQ ID NOs:49 and 50 respectively, were needed to completely sequence the PCR product. The nucleotide sequence of the open reading frame (ORF) for this gene and the predicted amino acid sequence of the enzyme, are given as SEQ ID N0:3 and SEQ ID NO:4, respectively.<br>
To create an expression clone, the thIB gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate pDEST14thlB. The pDEST14thlB vector was transformed into BL21-AI cells and<br>
T7 promoter was induced by addition of arabinose, A protein of the expected molecular weight of about 42 kDa, as deduced from the nucleic acid sequence, was present in the induced culture, but not in the uninduced control. Enzyme assays were performed as described in Example 1. In one typical assay, the specific activity of the ThIB protein in the induced culture was determined to be 14.9/ymol mg'1 min"1 compared to 0.28 j/mol mg"1 mirf1 in the uninduced culture.<br>
EXAMPLE 3 Cloning and Expression of 3-Hvdroxvbutvrvl-CoA Dehvdroaenase<br>
The purpose of this Example was to clone the hbd gene from C. acetobutylicum (ATCC 824} and express it in E. coli. The hbd gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA using PCR.<br>
The hbd gene was cloned and expressed using the method described in Example 1, The ribof gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primers N5 and N6 (see Table 4) given as SEQ ID NOs:19 and 20 respectively, creating a 881 bp product. The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD-TOPOhbd. Clones were submitted for sequencing with Ml 3 Forward and Reverse primers, given as SEQ ID NOs:45 and 46 respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. Additional sequencing primers, N5SeqF2 and N6SeqR2 (see Table 5), given as SEQ ID NOs:51 and 52 respectively, were needed to completely sequence the PCR product. The nucleotlde sequence of the open reading frame (ORF) for this gene and the predicted amino acid sequence of the enzyme are given as SEQ ID NO:5 and SEQ ID NO:6, respectively.<br>
To create an expression clone, the hbd gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate pDEST14hbd. The pDEST14hbd vector was transformed into BL21-AI cells and expression from the T7 promoter was induced by addition of arabinose, as described in Example 1. A protein of the expected molecular weight of<br>
3DdB^&amp;fE&gt;aQ£Heduced from the nucleic acid sequence, was present in the induced culture, but was absent in the uninduced control.<br>
Hydroxybutyryl-CoA dehydrogenase activity was determined by measuring the rate of oxidation of NADH as measured by the decrease in absorbance at 340 nm. A standard assay mixture contained 50 mM MOPS, pH 7.0,1 mM DTT and 0.2 mM NADH. The cocktail was equilibrated for 5 min at 37 °C and then the cell free extract was added. Reactions were initiated by addition of the substrate, 0.1 mM acetoacetyl-CoA. In one typical assay, the specific activity of the BHBD protein in the induced culture was determined to be 57.4jumol mg"1 min"1 compared to 0.885//mol mg"1 min"1 in the uninduced culture.<br>
EXAMPLE 4 Cloning and Expression of Crotonase<br>
The purpose of this Example was 'to clone the crt gene from C. acetobutylicum (ATCC 824) and express it in E. coll. The crt gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA using PCR.<br>
The erf gene was cloned and expressed using the method described in Example 1. The crt gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primers N3 and N4 (see Table 4), given as SEQ ID N0s:17and 18| respectively, creating a 794 bp product. The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD-TOPOcrt. Clones were submitted for sequencing with M13 Forward and Reverse primers, given as SEQ ID N0s:45 and 46 respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. The nucleotide sequence of the open reading frame (ORF) for this gene and its predicted amino acid sequence are given as SEQ ID NO:7 and SEQ ID NO:8, respectively.<br>
To create an expression clone, the crt gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate pDEST14crt. The pDESTUcrt vector was transformed into BL21-A) cells and expression from the T7 promoter was induced by addition of arabinose, as described in Example 1. A protein of the expected molecular weight of<br>
SpOtSBBEiaaeduced from the nucleic acid sequence, was present in much greater amounts in the induced culture than in the uninduced control.<br>
Crotonase activity was assayed as described by Stern (Methods Enzymol. 1, 559-566, (1954)). In one typical assay, the specific activity of the crotonase protein, in the induced culture was determined to be 444 A/mol mg'1 min'1 compared to 47yt/mol mg'1 min'1 in the uninduced culture.<br>
EXAMPLE 5 Cloning and Expression of Butvrvl-CoA Dehvdroaenase<br>
The purpose of this Example was to express the enzyme butyryl-CoA dehydrogenase, also referred to herein as trans-2-Enoyl-CoA reductase, in.,E. coli.   The CAC0462 gene, a putative trans-2-enoyl-CoA reductase homolog, was cloned from C. acetobutylicum (ATCC 824) and expressed in E. coli. The CAC0462 gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA using PCR.<br>
The CAC0462 gene was cloned and expressed using the method described in Example 1. The CAC0462 gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primers N17 and N21 (see Table 4), given as SEQ ID N0s:29 and 30, respectively, creating a 1.3 kbp product The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD-TOPOCAC0462. Clones were submitted for sequencing with M13 Forward and Reverse primers, given as SEQ ID NO:45 and 46 respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. Additional sequencing primers, N22SeqF1 (SEQ ID N0:53), N22SeqF2 (SEQ ID N0:54), N22SeqF3 (SEQ ID NO:55), N23SeqR1(SEQ ID N0:56), N23SeqR2 (SEQ ID NO:57), and N23SeqR3 (SEQ ID NO:58) (see Table 5) were needed to completely sequence the PCR product. The nucleotide sequence of the open reading frame (ORF) for this gene and the predicted amino acid sequence of the enzyme are given as SEQ ID N0:9 and SEQ ID N0:10, respectively.<br>
To create an expression clone, the CAC0462 gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate<br>
2. The pDEST14CA0462 vector was transformed into BL21-AI cells and expression from the T7 promoter was induced by addition of arabinose, as described in Example 1. Analysis by SDS-PAGE showed no overexpressed protein of the expected molecular weight in the negative control or in the induced culture. The C. acetobutylicum CAC0462 gene used many rare £ co//codons. To circumvent problems with codon usage the pRARE plasmid (Novagen) was transformed into BL21-AI ceils harboring the pDEST14CAC0462 vector. Expression studies with arabinose induction were repeated with cultures carrying the pRARE vector. A protein of the expected molecular weight of about 46 kDa was present in the induced culture but not in the uninduced control.<br>
Trans-2-enoyi-CoA reductase activity was assayed as described by Hoffmeister et al. (J. Biol. Chem. 280,4329-4338 (2005)). In one typical assay, the specific activity of the TER CAC0462 protein in the induced culture was determined to be 0.694 jumol mg"1 min"1 compared to 0.0128 fjmol mg"1 min"1 in the uninduced culture.<br>
EXAMPLE 6 Cloning and Expression of Butvraldehvde Dehvdrogenase (Acetvlating)<br>
The purpose of this Example was to clone the a/of gene from C. beijerinckii (ATCC 35702) and express it in E. coll. The aid gene was amplified from C. beijerinckii (ATCC 35702) genomic DNA using PCR.<br>
The aid gene was cloned and expressed using the method described in Example 1. The aid gene was amplified from C. beijerinckii (ATCC 35702) genomic DNA (prepared from anaerobicaliy grown cultures, as described in Example 1) by PCR using primers N27 F1 and N28 R1 (see Table 4), given as SEQ ID N0s:31 and 32 respectively, creating a 1.6 kbp product. The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD-TOPOald. Clones were submitted for sequencing with M13 Forward and Reverse primers, given as SEQ ID NOs:45 and 46 respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. Additional sequencing primers, N31SeqF2 (SEQ ID N0:59), N31SeqF3 (SEQ ID NO:6Q), N31SeqF4 (SEQ ID<br>
iu NU:/^), N31SeqR2 (SEQ ID N0:62), N31SeqR3 (SEQ ID NO:63), N31SeqR4 (SEQ ID NO:64), and N31SeqR5 (SEQ ID N0:65) (see Table 5) were needed to completely sequence the PCR product. The nucleotide sequence of the open reading frame (ORF) for.tjiis gene and the predicted amino acid sequence of the enzyme are given as SEQ ID NO:11 and SEQ ID N0:12, respectively.<br>
To create an expression clone, the aid gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate pDEST14ald. The pDEST14ald vector was transformed into BL21-AI cells and expression from the T7 promoter was induced by addition of arabinose, as described in Example 1. A protein of the expected molecular weight of about 51 kDa, as deduced from the nucleic acid sequence, was present in the induced culture, but not in the uninduced control.<br>
Acylating aldehyde dehydrogenase activity was determined by monitoring the formation of NADH, as measured by the increase in absorbance at 340 nm, as described by Husemann et al. (Appl. Microbiol. Biotechnol. 31:435-444 (1989)). In one typical assay, the specific activity of the Aid protein in the induced culture was determined to be 0.106//mol mg"1 min"1 compared to 0.01 //mol mg"1 min"1 in the uninduced culture<br>
EXAMPLE 7 Cloning and Expression of Butanol Dehvdrogenase<br>
The purpose of this Example was to clone the bdhB gene from C. acetobutylicum (ATCC 824) and express it in E. coli. The bdhB gene was amplified from C. acetobutylicum (ATCC 824) genomic DMA using PCR.<br>
The bdhB gene was cloned and expressed using the method described in Example 1. The bdhB gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primers N11 and N12 (see Table 4), given as SEQ ID NOs:25 and 26, respectively, creating a 1.2 kbp product. The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD-TOPObdhB. The translational start codon was also changed from "GTG" to "ATG" by the primer sequence. Clones were submitted for sequencing with M13 Forward and Reverse primers, given  and 46 respectively, to confirm that the genes inserted in the correct orientation and to confirm the sequence. Additional sequencing primers, N11SeqF1 (SEQ ID N0:66), N11SeqF2 (SEQ ID NO:67), N12SeqR1 (SEQ ID NO:68), and N12SeqR2 (SEQ ID NO:69), (see Table 5) were needed to completely sequence the PCR product. The nucleotide sequence of the open reading frame (ORF) for this gene and the predicted amino acid sequence of the enzyme are given as SEQ ID N0:13 and SEQ ID N0:14, respectively.<br>
To create an expression clone, the bdhB gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate pDEST14bdhB. The pDEST14bdhB vector was transformed into BL21-AI cells and expression from the T7 promoter was induced by addition of arabinose, as described in Example 1. A protein of the expected molecular weight of about 43 kDa, as deduced from the nucleic acid sequence, was' present in the induced culture, but not in the uninduced control.<br>
Butanol dehydrogenase activity was determined from the rate of oxidation of NADH as measured by the decrease in absorbance at 340 nm as described by Husemann and Papoutsakis, supra. In one typical assay, the specific activity of the BdhB protein in the induced culture .was determined to be 0.169//mol mg"1 min"1 compared to 0.022/;mol mg"1 min"1 in the uninduced culture.<br>
EXAMPLE 8 Cloning and Expression of Butanol Dehydroqenase<br>
The purpose of this Example was to clone the bdhA gene from C. acetobutylicum 824 and express it in E. coli. The bdhA gene was amplified from C. acetobutylicum 824 genomic DNA using PCR.<br>
The bdhA gene was cloned and expressed using the method described in Example 1. The bdhA gene was amplified from C. acetobutylicum 824 genomic DNA by PCR using primers N9 and N10 (see Table 4), given as SEQ ID N0s:23 and 24, respectively, creating a 1.2 kbp product. The forward primer incorporated four bases (CACC) immediately adjacent to the translational start codon to allow directional cloning into pENTR/SD/D-TOPO (Invitrogen) to generate the plasmid pENTRSDD-TOPObdhA. Clones, given as SEQ ID NOs:45 and 46 respectively, to<br>
these inserted in the correct orientation and to confirm the sequence. Additional sequencing primers, N9SeqF1 (SEQ ID N0:70) and N10SeqR1 (SEQ ID N0.71), (see Table 5) were needed to completely sequence the PCR product. The nucleotide sequence of the open reading .frame (ORF) for this gene and the predicted amino acid sequence of the enzyme are given as SEQ ID NO:15 and SEQ ID NO:16, respectively.<br>
To create an expression clone, the bdhA gene was transferred to the pDEST 14 (Invitrogen) vector by recombination to generate pDEST14bdriA. The pDEST14bdhA vector was transformed into BL21-AI cells and expression from the T7 promoter-was induced by addition of arabinose, as described in Example 1. A protein of the expected molecular weight of about 43 kDa, as deduced from the nucleic acid sequence, was present in the induced culture, but not in the uninduced control.<br>
Butanol dehydrogenase activity was determined from the rate of oxidation of NADH as measured by the decrease in absorbance at 340 nm, as described by Husemann and Papoutsakis, supra. In one typical assay, the specific activity of the BdhA protein in the induced culture was determined to be 0.102 //mol mg~1 min"1 compared to 0.028 /jmo\ mg"1 min" 1 in the uninduced culture<br>
EXAMPLE 9<br>
Construction of a Transformation Vector for the Genes in the 1 -Butanol Biosvnthetic Pathway - Lower Pathway<br>
To construct a transformation vector comprising the genes encoding the six steps in the 1-butanoI biosynthetic pathway, the genes encoding the 6 steps in the pathway were divided into two operons. The upper pathway comprises the first four steps catalyzed by acetyl-CoA acetyltransferase, 3-hydroxybutyryl-CoA dehydrogenase, crotonase, and butyryl-CoA dehydrogenase. The lower pathway comprises the last two steps, catalyzed by butyraldehyde dehydrogenase and butanol dehydrogenase.<br>
The purpose of this Example was to construct the lower pathway operon. Construction of the upper pathway operon is described in Example 10.<br>
genes were amplified by PCR with pnmers that incorporated restriction sites for later cloning and the forward primers contained an optimized E. co//ribosome binding site (AAAGGAGG). PCR products were TOPO cloned into the pCR 4Blunt-TOPO vector and transformed into £ co//Top10 cells (Invitrogen). Plasmid DNA was prepared from the TOPO clones and the sequence of the genes was verified. Restriction enzymes and T4 DNA ligase (New England Biolabs, Beverly, MA) were used according to manufacturer's recommendations. For cloning experiments, restriction fragments were purified by gel electrophoresis using QIAquick-Gel Extraction kit (Qiagen).<br>
After confirmation of the sequence, the genes were subcloned into a modified pUC19 vector as a cloning platform. The pUC19 vector was modified by a Hindlll/Sapl digest, creating pUC19dHS. The digest removed the lac promoter adjacent to the MCS (multiple cloning site), preventing transcription of the operons in the vector.<br>
The aid gene was amplified from C. beijerinckii ATCC 35702 genomic DNA by PCR using primers N58 and N59 (see Table 4), given as SEQ ID N0s:41 and 42, respectively, creating a 1.5 kbp product. The forward primer incorporated the restriction sites Aval and BstEII and a RBS (ribosome binding site). The reverse primer incorporated the Hpal restriction site. The PCR product was cloned into pCRBIunt II-TOPO creating pCRBIuntll-ald. Plasmid DNA was prepared from the TOPO clones and the sequence of the genes verified with primers M13 Forward (SEQ ID N0:45), M13 Reverse (SEQ ID N0:46), N31SeqF2 (SEQ ID NO:59), N31SeqF3 (SEQ ID NO:60), N31SeqF4 (SEQ ID NO:61), N32SeqR1(SEQ ID NO:72), N31SeqR2 (SEQ ID NO:62), N31SeqR3 SEQ ID N0:63), N31SeqR4 (SEQ ID N0:64), and N31SeqR5 (SEQ ID N0:65) (see Table 5).<br>
The bdhB gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primers N64 and N65 (see Table 4), given as SEQ ID N0s:43 and 44, respectively, creating a 1.2 kbp product. The forward primer incorporated an Hpal restriction site and a RBS. The reverse primer incorporated a Pmel and a Sphl restriction site. The PCR product was cloned into pCRBIunt II-TOPO creating pCRBIuntll-bdhB.<br>
prepared from the TOPO clones and the sequence of the genes verified with primers M13 Forward (SEQ ID NO:45), M13 Reverse (SEQ ID N0:46), N11SeqF1 (SEQ ID NO:66), N11SeqF2 (SEQ ID N0:67), Nl2SeqR1 (SEQ ID NO:68), and N12SeqR2 (SEQ ID N0:69) (see Table 5).<br>
To construct the lower pathway operon, a 1.2 kbp Sphl and Hpal fragment from pCRBIuntll-bdhB, a 1.4 kbp Hpal and Sphl fragment from pCRBIuntll-ald, and the large fragment from a Aval and Sphl digest of pUC19dHS were ligated together. The three-way ligation created pUC19dHS-ald-bdhB.<br>
The pUC19dHS-ald-bdhB vector was digested with BstEII and Pmel releasing a 2.6 kbp fragment that was cloned into pBenBP, an E. coil-Bacillus subtilis shuttle vector. Plasmid pBenBP was created by modification of the pBE93 vector, which is described by Nagarajan, WO 93/24631 (Example 4). The Bacillus amyloliquefaciens neutral protease promoter (NPR), signal sequence and the phoA gene were removed from pBE93 with a Ncol/Hindlll digest. The NPR promoter was PCR amplified from pBE93 by primers BenF and BenBPR, given by SEQ ID NOs:73 and 75, respectively. Primer BenBPR incorporated BstEII, Pmel and Hindlll sites downstream of the promoter. The PCR product was digested with Ncol and Hindlll and the fragment was cloned into the corresponding sites in the vector pBE93 to create pBenBP. The lower operon fragment was subcloned into the BstEII and Pmel sites in pBenBP creating pBen-ald-bdhB.<br>
Assays for butyraldehyde dehydrogenase and butanol dehydrogenase activity were conducted on crude extracts using the methods described above. Both enzyme activities were demonstrated at levels above the control strain that contained an empty vector.<br>
EXAMPLE 10 (Prophetic)<br>
Construction of a Transformation Vector for the Genes in the 1-Butanol Biosvnthetic Pathway - Upper Pathway<br>
The purpose of this prophetic Example is to describe how to assemble the upper pathway operon. The general approach is the same as described in Example 9.<br>
is amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primer pair N44 and N45 (see Table 4), given as SEQ ID NOs:33 and 34, respectively, creating a 1.2 kbp product. The forward primer incorporates a Sphl restriction site and a ribosome binding site (RBS). The reverse primer .incorporates AscI and Pstl restriction sites. The PCR product is cloned into pCR4 Blunt-TOPO creating pCR4 Blunt-TOPO-thlA. Plasmid DNA is prepared from the TOPO clones and the sequence of the genes is verified with primers M13 Forward (SEQ ID N0:45), M13 Reverse (SEQ ID NO:46), N7SeqF1 (SEQ ID N0:47), and N7SeqR1 (SEQ ID N0:48) (see Table 5).<br>
The hbd gene is amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primer pair N42 and N43 (see Table 4) given as SEQ ID NOs:35 and 36, respectively, creating a 0.9 kbp product. The forward primer incorporates a Sail restriction site and a RBS. The reverse primer incorporates a Sphl restriction site. The PCR product is cloned into pCR4 Blunt-TOPO creating pCR4 Blunt-TOPO-hbd. Plasmid DNA is prepared from the TOPO clones and the sequence of the genes verified with primers M13 Forward (SEQ ID NO:45), M13 Reverse (SEQ ID N0:46), N5SeqF2 (SEQ ID N0:51), and N6SeqR2 (SEQ ID N0:52) (see Table 5).<br>
The CAC0462 gene is codon optimized for expression in E. coll as primary host and B. subtilis as a secondary host. The new gene called CaTER, given as SEQ ID N0:76, is synthesized by Genscript Corp (Piscataway, NJ). The gene CaTER is cloned in the pUC57 vector as a BamHI-Sall fragment and includes a RBS, producing plasmid pUC57-CaTER.<br>
The erf gene is amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primer pair N38 and N39 (see Table 4), given as SEQ ID NOs:39 and 40, respectively, creating a 834 bp product. The forward primer incorporates EcoRI and Mlul' restriction sites and a RBS. The reverse primer incorporates a BamHI restriction site. The PCR product is cloned into pCR4 Blunt-TOPO creating pCR4 Blunt-TOPO-crt. Plasmid DNA is prepared from the TOPO clones and the sequence of the<br>
primers M13 Forward (SEQ ID N0:45) and M13 Reverse(SEQ ID N0:46) (see Table 5).<br>
After confirmation of the sequence, the genes are subcloned into a modified pUC19 vector as a cloning platform. The pUC19 vector was modified by a Sphl/Sapl digest, creating pUC19dSS. The digest removed the lac promoter adjacent to the MCS, preventing transcription of the operons in the vector.<br>
To construct the upper pathway operon pCR4 Blunt-TOPO-crt is digested with EcoRI and BamHI releasing a 0.8 kbp crt fragment. The pUC19dSS vector is also digested with EcoRI .and BamHI releasing a 2.0 kbp vector fragment. The crt fragment and the vector fragment are ligated together using T4 DMA ligase (New England Biolabs) to form pUC19dSS-crt. The CaTER gene is inserted into pCU19dSS-crt by digesting pUC57-CaTERwith BamHI and Sail,1 releasing a 1.2 kbp CaTER fragment. The pUC19dSS-crt is digested with BamHI and Sail and the large vector fragment is ligated with the CaTER fragment, creating pUC19dSS-crt-CaTER. To complete the operon a 884 bp Sail and Sphl fragment from pCR4 Blunt-TOPO-hbd, a 1.2 kb Sphl and Pstl thIA fragment from pCR4 Blunt-TOPO-thIA and the large fragment from a Sail and Pstl digest of pUC19dSS-crt-CaTER are ligated. The product of the 3-way ligation is pUC19dSS-crt-CaTER-hbd-thlA.<br>
The pUC19dSS-crt-CaTER-hbd-thlA vector is digested with Mlul and AscI releasing a 4.1 kbp fragment that is cloned into a derivative of pBE93 (Caimi, W02004/018645, pp. 39-40) an E. coli-B. subtilis shuttle vector, referred to as pBenMA. Plasmid pBenMA was created by modification of the pBE93 vector. The Bacillus amyloliquefaciens neutral protease promoter (NPR), signal sequence and the phoA gene are removed from pBE93 with a Ncol/Hindlll digest. The NPR promoter is PCR amplified from pBE93 by primers BenF and BenMAR, given as SEQ ID NOS:73 and 74, respectively. Primer BenMAR incorporates Mlul, AscI, and Hindlll sites downstream of the promoter. The PCR product was digested with Ncol and Hindlll and the fragment is cloned into the corresponding sites in the vector pBE93, creating pBenMA. The upper into the Mlul and AscI sites in pBenMA creating pBen-crt-hbd-CaTER-thlA.<br>
EXAMPLE 11 (Prophetic)<br>
Expression of the 1-Butanol Biosynthetic Pathway in E. coli The purpose of this prophetic Example is to describe how to express the 1-butanol biosynthetic pathway in E. coli.<br>
The plasmids pBen-crt-hbd-CaTER-thlA and pBen-ald-bdhB, constructed as described in Examples 10 and 9, respectively, are transformed into £ coli NM522 (ATCC 47000) and expression of the genes in each operon is monitored by SDS-PAGE analysis, enzyme assay and Western analysis. For Westerns, antibodies are raised to synthetic peptides by Sigma-Genosys (The Woodlands, TX). After confirmation of expression of all the genes, pBen-ald-bdhB is digested with EcoRI and Pmel to release the NPR promoter-ald-bdhB fragment. The EcoRI digest of the fragment is blunt ended using the Klenow fragment of DMA polymerase (New England Biolabs, catalog no. M0210SJ. The plasmid pBen-crt-hbd-CaTER-thlA is digested with Pvull to create a linearized blunt ended vector fragment. The vector and NPR-ald-bdhB fragment are ligated, creating p1B1 0.1 and p1B1 O.2, containing the complete 1-butanol biosynthetic pathway with the NPR promoter-ald-bdhB fragment in opposite orientations.   The plasmids p1B1 0.1 and p1B1 O.2 are transformed into E. coli NM522 and expression of the genes are monitored as previously described.<br>
E. coli strain NM522/p1B1 O.1 orNM522/p1B1.O.1 is inoculated into a 250 ml shake flask containing 50 mL of medium and shaken at 250 rpm and 35 °C. The medium is composed of: dextrose, 5 g/L; MOPS, 0.05 M; ammonium sulfate, 0.01 M; potassium phosphate, monobasic, 0.005 M; S10 metal mix, 1% (v/v); yeast extract, 0.1% (w/v); casamino acids, 0.1% (w/v); thiamine, 0.1 mg/L; praline, 0.05 mg/L; and biotin 0.002 mg/L, and is titrated to pH 7.0 with KOH. S10 metal mix contains: MgCI2,200 mM; CaCI2, 70 mM; MnCI2, 5 mM; FeCI3l 0.1 mM; ZnCI2, 0.1 mM; thiamine hydrochloride, 0.2 mM; CuSO4,172 uM; CoCI2, 253 uM; and Na2Mo04, 242 uM. After 18 to 24 h, 1-butanol is detected by HPLC or G.C analysis, as described in the General Methods section.<br>
EXAMPLE 12 fProphetic) Expression of the 1-Butanol Biosvnthetic Pathway in Bacillus subtijfs<br>
The purpose of this prophetic Example is to describe how to express the 1-butano! biosynthetic pathway in Bacillus subtilis. The same approach as described in Example 11 is used.<br>
The upper and lower operons constructed as described in Examples 10 and 9, respectively, are used. The plasmids p1B1 0.1 and p1B1 0.2 are transformed into Bacillus subtilis BE 1010 (J. Bacterial. 173:2278-2282 (1991)) and expression of the genes in each operon is monitored as described in Example 11.<br>
B. subtilis strain BE1010/p1B1 O.1 or BE1010/ p1B1 O.2 is inoculated into a 250 ml_ shake flask containing 50 ml of medium and shaken at 250 rpm and 35 °C for 18 h. The medium is composed of. dextrose, 5 g/L; MOPS," 0.05 M; giutamic acid, 0.02 M; ammonium sulfate, 0.01 M; potassium phosphate, monobasic buffer, 0.005 M; S10 metal mix (as described in Example 11), 1% (v/v); yeast extract, 0.1% (w/v); casamino acids, 0.1% (w/v); tryptophan, 50 mg/L; methionine, 50 mg/L; and iysine, 50 mg/L, and is titrated to pH 7.0 with KOH. After 18 to 24 h, 1-butanol is detected by HPLC or GC analysis, as described in the General Methods section.   <br>
EXAMPLE 13 Production of 1-Butanol from Glucose using Recombinant £ coli.<br>
This Example describes the production of 1-butanol in E. coli. Expression of the genes encoding the 6 steps of the 1-butanol biosynthetic pathway was divided into three operons. The upper pathway comprised the first four steps encoded by thIA, hbd, crt and EgTER in one operon. The next step, encoded by aid, was provided by a second operon. The last step in the pathway, encoded by yqhD, was provided in a third operon. 1-Butanol production was demonstrated in E. coli strains comprising the three operons.<br>
Unless otherwise indicated in the text, cloning primers described in this Example are referenced by their SEQ ID NO: in Table 4, and sequencing and PCR screening primers are referenced by their SEQ ID NO: in Table 5.<br>
The thIA gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primer pair N44 and N45 (see Table 4), given as SEQ ID NOs:33 and 34, respectively, creating a 1.2 kbp product. The forward primer incorporated a SphI. restriction site and a ribosome binding site (RBS). The reverse primer incorporated AscI and Pstl restriction sites. The PCR product was cloned into pCR4Blunt-TOPO (Invitrogen Corp., Carlsbad, CA) creating pCR4Blunt-TOPO-thlA. Plasmid DNA was prepared from the TOPO clones and the sequence of the genes was verified with primers M13 Forward (SEQ ID NO:45), M13 Reverse .(SEQ ID NO:46), N7SeqF1 (SEQ ID NO.-47), and N7SeqR1 (SEQ ID NO:48) (see Table 5).<br>
3-Hvdroxvbutvryl-CoAdehvdrogenase. The hbd gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primer pair N42 and N43 (see Table 4) giveri as SEQ ID NOs:35 and 36, respectively, creating a 0.9 kbp product. The forward primer incorporated a Sail restriction site and a RBS. The reverse primer incorporated a Sphl restriction site. The PCR product was cloned into pCR4Blunt-TOPO creating pCR4Blunt-TOPO-hbd. Plasmid DNA was prepared from the TOPO clones and the sequence of the genes verified with primers M13 Forward (SEQ ID NO:45), M13 Reverse (SEQ ID N0:46), N5SeqF2 (SEQ ID N0:51), and N6SeqR2 (SEQ ID N0:52) (see Table 5).<br>
Crotonase. The crt gene was amplified from C. acetobutylicum (ATCC 824) genomic DNA by PCR using primer pair N38 and N39 (see Table 4), given as SEQ ID NOs:39 and 40, respectively, creating a 834 bp product. The forward primer incorporated EcoRI and Mlul restriction sites and a RBS. The reverse primer incorporated a BamHI restriction site. The PCR product was cloned into pCR4Blunt-TOPO creating pCR4Blunt-TOPO-crt. Plasmid DNA was prepared from the TOPO clones and the sequence of the genes was verified with primers M13 Forward (SEQ ID N0:45) and M13 Reverse (SEQ ID N0:46) (see Table 5).<br>
Butvrvl-CoA Dehvdrogenase (trans-2-enovl-CoA reductase). The CAC0462 gene was synthesized for enhanced codon usage in E. coll as primary host and B. subtilis as a secondary host. The new gene (CaTER, SEQ ID NO:76) was synthesized and cloned by Genscript Corporation<br>
p,UC57 vector as a BamHI-Sall fragment and includes a RBS.<br>
An alternative gene for butyryl-CoA dehydrogenase from Euglena gracilis (TER, GenBank No. Q5EU90) was synthesized for enhanced codon usage in E. coli and Bacillus subtilis. The gene was synthesized and cloned by GenScript Corporation into pUC57 creating pUC57::EgTER. Primers N85 and N86, (SED ID NO: 80 and 81 respectively) together with pUC57::EgTER as template DNA, provided a PCR fragment comprising 1224 bp from pUC57::EgTER DNA. The sequence of the 1224 bp is given as SEQ ID N0:77, where bp 1-1218 is the coding sequence (cds) of EgTER(opt). EgTER(opt) is a codon optimized TER gene, lacking the normal mitochondrial presequence so as to be functional in E. coli (Hoffmeisteret al., J. Bid. Chem. 280:4329 (2005)).<br>
EgTER(opt) was cloned into pCR4Blunt-TOPO and its sequence was confirmed with primers M13 Forward (SEQ ID N0:45) and M13 Reverse (SEQ ID N0:46). Additional sequencing primers N62SeqF2 (SEQ ID N0:114), N62SeqF3 (SEQ ID NO:115), N62SeqF4 (SEQ ID N0:116), N63SeqR1(SEQ ID N0:117), N63SeqR2 (SEQ ID NO:118) N63SeqR3 (SEQ ID NO:119) and N63SeqR4 (SEQ ID N0:120) were needed to completely sequence the PCR product. The 1.2 kbp EgTER(opt) sequence was then excised with Hinc\\ and Pmel and cloned into pET23+ (Novagen) linearized with H/ncll. Orientation of the EgTER(opt) gene to the promoter was confirmed by colony PCR screening with primers T7Primer and N63SeqR2 (SEQ ID N0s:82 and 118 respectively). The resulting plasmid, pET23+::EgTER(opt), was transformed into BL21 (DE3) (Novagen) for expression studies.<br>
Trans-2-enoyl-CoA reductase activity was assayed as described by Hoffmeister et al., J. Biol. Chem. 280:4329 (2005). In a typical assay, the specific activity of the EgTER(opt) protein in the induced BL21 (DE3) /pET23+::EgTER(opt) culture was determined to be 1.9//mol mg"1 min"1 compared to 0.547//mol mg"1 min"1 in the uninduced culture.<br>
The EgTER(opt) gene was then cloned into the pTrc99a vector under the control of the trc promoter. The EgTER(opt) gene was isolated as a 1287-bp BamHI/Sa/l fragment from pET23+::EgTER(opt). The 4.2<br>
was linearized witn samm/sa/l. The vector and fragment were ligated creating the 5.4 kbp pTrc99a-EgTER(opt). Positive clones were confirmed by colony PCR with primers Trc99aF and N63SeqR3 (SEQ ID NOs:83 and 119 respectively) producing a 0.5 kb product.<br>
Construction of plasmid pTrc99a-E-C-H-T comprising genes encoding acetvl-CoA acetvltransferase (thIA), 3-hvdroxvbutvrvl-CoA dehvdroaenase (hbd), crotonase (erf), and butvrvl-CoA dehvdrogenase (trans-2-enovl-CoA reductase. EqTER(opt)). To initiate the construction of a four gene operon comprising the upper pathway (EgTER(opt), erf, hbd and thIA), pCR4Blunt-TOPO-crt was digested with EcoRi and BamHI releasing a 0,8 kbp erf fragment. The pUC19dSS vector (described in Example 10) was also digested with EcoRI and BamHI releasing a 2.0 kbp vector fragment. The erf fragment and the vector fragment were ligated together using T4 DNA ligase (New England Biolabs) to form pUC19dSS-crt. The CaTER gene was inserted into pUC19dSS-crt by digesting pUC57-CaTERwith BamHI and Sail, releasing a 1.2 kbp CaTER fragment. The pUC19dSS-crt was digested with BamHI and Sail and the large vector fragment was ligated with the CaTER fragment, creating pUC19dSS-crt- CaTER. To complete the operon a 884 bp Sail and Sphl fragment from pCR4Blunt-TOPO-hbd, a 1.2 kb Sphl and Pstl thIA fragment from pCR4Blunt-TOPO-thlA and the large fragment from a Sail and Pstl digest of pUC19dSS-crt-CaTER were ligated. The product of the  3-way ligation was named pUC19dSS-crt-CaTER-hbd-thlA or  . pUC19dss::Operon1.<br>
Higher butyryl-CoA dehydrogenase activity was obtained from<br>
pTrc99a-EgTER(opt) than from CaTER constructs, thus, an operon derived from pTrc99a-EgTER(opt) was constructed. The CaTER gene was removed from pUC19dss::Operon1 by digesting with BamHI/Sa! I and gel purifying the 5327-bp vector fragment. The vector was treated with Kienowand religated creating pUC19dss::Operon 1 ACaTer. The 2934-bp crt-hbd-thlA (C-H-T) fragment was then isolated as a EcoRI/Pstl fragment from pUC19dss:Operon 1 ACaTer. The C-H-T fragment was treated with ends. The vector pTrc99a-EgTER(opt) was digested with Sail and the ends treated with Klenow. The blunt-ended vector and the blunt-ended C-H-T fragment were ligated to create pTrc99a-E-C-H-T. Colony PCR reactions were performed with primers IM62SeqF4 and N5SeqF4 (SEQ ID NOs: 116 and 84 respectively) to confirm the orientation of the insert.<br>
Construction of plasmids pBHR T7-ald and pBHR-Ptrc-ald(opt) comprising genes encoding butvraldehvde dehvdroqenase (aid and a/offopt)). The PT7-ald operon was sub-cloned from pDEST14-ald (Example 6) into the broad host range plasmid pBHR1 .(MoBitec, Goettingen, Germany) to create pBHR1 PT7-ald, The pBHR1 plasmid is compatible with pUC19 or pBR322 plasmids so pBHR1 PT7-ald can be used in combination with pUC19 or pBR322 derivatives carrying the upper pathway operon for 1-butano! production in E. coll. The pDEST14-ald plasmid was digested with Bgl II and treated with the Klenow fragment of DNA polymerase to make blunt ends. The plasmid was then digested with EcoRI and the 2,245 bp PT7-ald fragment was gel-purified.   Plasmid pBHR1 was digested with Seal and EcoRI and the 4,883 bp fragment was gel-purified. The PT7-ald fragment was ligated with the pBHR1 vector, creating pBHR T7-ald.  Colony PCR amplification of transformants with primers T-ald(BamHI) and B-ald(EgTER) (SEQ ID NOs:85 and 86 respectively) confirmed the expected 1.4 kb PCR product. Restriction mapping .of pBHR T7-ald clones with EcoRI and Drdl confirmed the expected 4,757 and 2,405 bp fragments.<br>
For butyraldehyde dehydrogenase activity assays, the plasmid pBHR T7-ald was transformed into BL21Star™(DE3) cells (Invitrogen) and expression from the T7 promoter was induced by addition of L-arabinose as described in Example 1. Acylating aldehyde dehydrogenase activity was determined by monitoring the formation of NADH, as measured by the increase in absorbance at 340 nm, as described in Example 6.<br>
An alternative DNA sequence for the a/of gene from Clostridium beijerinckii ATCC 35702 was synthesized (optimizing for codon usage in E. co//and Bacillus subtilis) and cloned into pUC57 by GenScript<br>
creating the plasmid pl)C57-ald(opt). pUC57-ald(opt) was digested with Sacl and Sail to release a 1498 bp fragment comprising the condon optimized gene, a/d(opt) and a RBS already for E coll The sequence of the 1498 bp fragment is given as SEQ ID NO:78.<br>
pTrc99a was digested with Sacl and Sail giving a 4153 bp vector fragment, which was ligated with the 1498 bp a/c/(opt) fragment to create pTrc-ald(opt). Expression of the synthetic gene, a/d(opt), is under the control of the IPTG-inducible Ptrc promoter.<br>
The Ptrc-ald(opt) operon was subcloned into the broad host range plasmid pBHR1 (MoBitec) in order to be compatible with the upper pathway plasmid described above. The Pirc-ald(opt) fragment was PCR-amplified from pTrc99A::ald(opt) with T-Ptrc(BspEI) and B-aldopt(Scal), (SEQ ID N0s,:87 and 88 respectively) incorporating BspEI and Seal restriction sites within the corresponding primers. The PCR product was digested with .BspEI and Seal. The plasmid pBHR1 was digested with Seal and BspEI and the 4,883 bp fragment was gel-purified. The Ptic-ald(opt) fragment was ligated with the pBHR1 vector, creating pBHR-PcatPtrc-ald(opt). Restriction mapping of the pBHR-PcatPtrc-ald(opt) clones with Seal and BspEI confirmed the expected 4,883 and 1,704 bp fragments. To remove the plasmid-born cat promoter (Peat) region, plasmid pBHR-PcatPtrc-ald(opt) was digested with BspEI and Aatll and the 6,172 bp fragment was gel-purified. T-BspEIAatll and B-BspEIAatll (SEQ ID NOs:89 and 90 respectively) were mixed in a solution containing 50 mM NaCI, 10 mM Tris-HCI, and 10 mM MgCI2 (pH7.9) to a final concentration of 100 p,M and hybridized by incubating at 75 °C for 5 min and slowly cooling to room temperature. The hybridized oligonucleotides were ligated with the 6,172 bp fragment, creating pBHR-Ptrc-ald(opt).<br>
Construction of E. co//'strains expressing butanol dehvdrogenase (vdhD). E. coll contains a native gene (yqhD) that was identified as a 1,3-propanediol dehydrogenase (U.S. Patent No. 6,514,733). The yqhD gene has 40% identity to the gene adhB in Clostridium, a probable NADH-dependent butanol dehydrogenase. The yqhD gene was placed under the<br>
Session of a variant of the glucose isomerase promoter 1.6GI (SEQ ID N0:91) in £. co//strain MG1655 1.6yqhD::Cm (WO 2004/033646) using X Red technology (Datsenko and Wanner, Proc. Natl. 4cad. Sci. U.S.A. 97:6640 (2000)). Similarly, the native promoter was replaced by the 1.5GI promoter (WO 2003/089621) (SEQ ID N0:92), creating strain MG1655 1.5GI-y'qhD::Cm, thus, replacing the 1.6GI promoter of MG1655 1.6yqhD::Cm with the 1.5GI promoter.<br>
A P1 lysate was prepared from MG1655 1.5GI yqhD::Cm and the cassette moved to expression strains, MG1655 (DE3), prepared from E. constrain MG1655 and a lambda DE3 lysogenization kit (Invitrogen), and BL21 (DE3) (Invitrogen).creating MG1655 (DE3) 1.5GI-yqhD::Cm and BL21 (DE3) 1.5GI:yqhD::Cm, respectively.<br>
'    Demonstration of 1 -butanol production from recombinant E. coll. E. co//strain MG1655 (DE3) 1.5GI-yqhD::Cm was transformed with plasmids pTrc99a-E-C-H-T and pBHR T7-ald to produce the strain, MG1655 (DE3) 1.5GI-yqhD::Cm/ pTrc99a-E-C-H-T/ pBHR T7-ald. Two independent isolates were initially grown in LB medium containing 50 ng/mL kanamycin and 100 p.g/mL carbenicillin. The cells were used to inoculate shake flasks (approximately 175 ml_ total volume) containing 15,50 and 150 mL of TM3a/glucose medium (with appropriate antibiotics) to represent high, medium and low oxygen conditions, respectively. TM3a/glucose medium contained (per liter): 10 g glucose, 13.6 g KH2PO4, 2.0 g citric acid monohydrate, 3.0 g (NH4)2S04, 2.0 g MgSO4-7H20, 0.2 g CaCI2-2H20, 0.33 g ferric ammonium citrate, 1.0 mg thiamine-HCI, 0.50 g yeast extract, and 10 ml_ trace elements solution, adjusted to pH 6.8 with NH4OH. The solution of trace elements contained: citric acid-H20 (4.0 g/L), MnS04«<br>
H2O (3.0 g/L), NaCI (1.0 g/L), FeS04-7H2O (0.10 g/L), CoCl2-6H2O (0.10 g/L), ZnS04-7H20 (0.10 g/L), CuSO4-5H2O (0.010 g/L), H3BOs (0.010 g/L), and Na2Mo04-2H2O (0.010 g/L). The'flasks were inoculated at a<br>
starting OD6oo of £ 0.01 units and incubated at 34 °C with shaking at 300 rpm. The flasks containing 15 and 50 mL of medium were capped with vented caps; the flasks containing 150 mL, were.capped with non-vented .<br>
IPTG was added to a final concentration of 0.04 mM; the OD60o of the flasks at the time of addition was £ 0.4 units.<br>
Approximately 15 h after induction, an aliquot of the broth was analyzed by HPLC (Shodex Sugar SH1011 column) with refractive index (Rl) detection and GC (Varian CP-WAX 58(FFAP) CB column, 25 m x 0.25 mm id X 0.2//m film thickness) with flame ionization detection (FID) for 1-butanol content, as described in the General Methods section. The results of the 1-butanol determinations are given in Table 6.<br>
Table 6<br>
Production of 1-butanol bv E. co//strain MG1655 (DE3) 1.5GI-vahD::Cm/ pTrc99a-E-C-H-T/ pBHR T7-ald.<br>
(Table Removed)<br>
The two independent isolates of MG1655 (DE3) 1.5GI-yqhD::Cm/ pTrc99a-E-C-H-T/ pBHR T7-ald were tested for 1-butanol production in an identical manner except that the medium contained 5 g/L yeast extract. The results are shown in Table 7.<br>
Table 7<br>
Production of 1-butanol bvE. co//strain MG1655 (DE3) 1.5GI-vqhD::Cm/ pTrc99a-E-C-H-T/ pBHR T7-ald.<br><br>
(Table Removed)<br>
E. constrain BL21 (DE3) 1.5GI-yqhD::Cm was transformed with plasmids pTrc99a-E-C-H-T and pBHR T7-ald to produce the strain, BL21 (DE3) 1.5GI-yqhD::Cm / pTrc99a-E-C-H-T/ pBHR T7-ald. Two independent isolates were tested for 1-butanol production exactly as described above. The results are given in Tables 8 and 9.<br>
Table 8<br>
Production of 1-butanol bv E. coll strain BL21 (DE3) 1.5GI-vahD::Cm / pTrc99a-E-C-H-T/ pBHR T7-ald.<br>
 (Table Removed)<br><br>
Table 9<br>
Production of 1-butanol bv E. coll strain BL21 (DE3^ 1.5GI-vahD::Cm / DTrc99a-E-C-H-T/ pBHR T7-ald.<br>
(Table Removed)<br>
E. co//strain MG1655 1.5GI-yqhD::Cm was transformed with plasmids pTrc99a-E-C-H-T and pBHR-Ptrc-ald(opt) to produce the strain, MG1655 1.5GI-yqhD::Cm/pTrc99a-E-C-H-T/pBHR-Ptrc-ald(opt). Two isolates were initially grown in LB medium containing 50 ug/mL kanamycin and 100 |j.g/mL carbenicillin. The cells were used to inoculate shake flasks (approximately 175 ml total volume) containing 50 and 150 ml of TM3a/glucose medium (with appropriate antibiotics). The flasks were inoculated at a starting OD550 of £ 0.04 units and incubated as described above, with and without induction. IPTG was added to a final concentration of 0.4 mM; the OD550 of the flasks at the time of addition was between 0.6 and 1.2 units. In this case, induction was not necessary for 1-butanol pathway gene expression because of the leakiness of the IPTG inducible promoters and the constitutive nature of the 1.5GI promoter; however, induction provided a wider range of expression. Approximately 15 h after induction, an aliquot of the broth was analyzed by GC with flame ionization detection for 1-butano! content, as described above. The results are given in Table 10. Fortherecombinant E. constrains, 1-butanol was produced in all cases; in separate experiments, wild type E. coll strains were shown to produce no detectable 1 -butanol (data not shown).<br>
Table 10<br>
Production of 1-butanol bvE co//strain MG1655 1.5GI-vahD::Cm/ <br>
(Table Removed)<br>
EXAMPLE 14 Production of 1-Butanol from Glucose using Recombinant B. subtilis.<br>
This Example describes the production of 1-butanol in Bacillus subtilis. The six genes of the 1-biosynthetic pathway, encoding six enzyme activities, were split into two operons for expression. The first three genes of the pathway (thl, hbd, and crt) were integrated into the chromosome of Bacillus subtilis BE1010 (Payne and Jackson, J. Bacteriol. 173:2278-2282 (1991)). The last three genes (EgTER, aid, and bdhB) were cloned into an expression plasmid and transformed into the Bacillus strain carrying the integrated 1-butanol genes.<br>
Unless otherwise indicated in the text, cloning primers described in this Example are referenced by their SEQ ID NO: in Table 4, and sequencing and PCR screening primers are referenced by their SEQ ID NO: In Table 5.<br>
Integration Plasmid. Plasmid pFP988 is a Bacillus integration vector that contains an £ .coll replicon from pBR322, an ampicillin antibiotic marker for selection in E. coll and two sections of homology to the sacB gene in the Bacillus chromosome that directs integration of the vector and intervening sequence by homologous recombination. Between the sacB homology regions is the Pamy promoter and signal sequence that can direct the synthesis and secretion of a cloned gene, a His-Tag and erythromycin as a selectable marker for Bacillus. The Pamy promoter and signal sequence is from Bacillus amyloliquefaciens alpha-amylase. The promoter region also contains the lacO. sequence for regulation of expression by a lad represser protein. The sequence of pFP988 (6509 bp) is given as SEQ ID NO:79.<br>
Since the 1-butanol pathway genes were to be expressed in the cytoplasm, the amylase signal sequence was deleted. Plasmid pFP988<br>
imers Pamy/lacO F and Pamy/lacO R creating a 317 bp (0.3 kbp) product that contained the Pamy/lacO promoter. The 5' end of the Pamy/lacO F primer incorporated a BsrGI restriction site followed by an EcoRI site. The 5' end of the Pamy/lacO R primer incorporated a BsrGI restriction site followed by a Pmel restriction site. The PCR product was TOPO cloned into pCR4Blunt-TOPO creating pCR4Blunt-TOPO-Pamy/lacO. Plasmid DNA was prepared from overnight cultures and submitted for sequencing with M13 Forward and M13 Reverse primers (SEQ ID N0:45 and SEQ ID N0:46, respectively) to ensure no mutation had been introduced into the promoter. .A clone of pCR4Blunt-TOPO-Pamy/lacO was digested with BsrGI and the 0.3 kbp fragment was gel-purified. The vector pFP988 was digested with BsrGI resulting in deletion of 11 bp from the 5' sacB homology region and the removal of the Pamy/lacO promoter arid signal sequence and His-tag. The 6 kbp BsrGI digested vector was gel-purified and ligated with Pamy/lacO BsrGI insert. The resulting plasmids were screened with primers Pamy SeqF2 and Pamy SeqR to determine orientation of the promoter. The correct clone restored the Pamy/lacO promoter to its original orientation and was named pFP988Dss.<br>
The cassette with genes thl-crtwas constructed by SOE (splicing by overlap extension). The genes were amplified using as template pUC19dss::Operpn1. The thl primers were Top TF and Bot TR amplifying a 0.9 kbp product. The crt primers were Top CF and Bot OR amplifying a 1.3 kbp product. The two genes were joined by SOE with PCR amplification using primers Top TF and Bot CR generating a 2.1 kbp product that was TOPO cloned into pCR4Blunt-TOPO creating pCR4Blunt-TOPO-T-C. Clones were submitted for sequencing to confirm the sequence. The plasmid pCR4Blunt-TOPO-T-C was digested with BstEll and Pmel releasing a 2.1 kbp fragment that was gel-purified. The insert was treated with Klenow polymerase to blunt the BstEll site. Vector pFP988Dss was digested with Pmel and treated with calf intestinal alkaline phosphatase (New England BioLabs) to prevent self-ligation. The 2.1 kbp thl-crt fragment and the digested pFP988Dss were ligated and transformed into E. co//Top10 cells. Transformants were screened by<br>
SCMiacripl!«ia:5&amp;ith Pamy SeqF2 and N7SeqR2 for a 0.7 kbp product, the correct product was called pFP988Dss-T-C.<br>
Construction of the thl-crt cassette created unique Sail and Spel sites between the two genes. To add the hbd gene to the cassette, the hbd gene was subcloned from pCR4Blunt-TOPO-hbd as a 0.9 kbp Sall/Spel fragment. Vector pFP988Dss-T-C was digested with Sail and Spel and the 8 kbp vector fragment was gel-purified. The vector and hbd insert were ligated and transformed into £ co//Top10 cells. Transformants were screened by PCR amplification with primers Pamy SeqF and N3SeqF3 for a 3.0 kbp fragment. The resulting plasmid was named pFP988Dss-T-H-C.<br>
The Pamy promoter subsequently was replaced with the Pspac promoter from plasmid pMUTIN4 (Vagneref a/., Microbiol. 144:3097-3104 (1998)). The Pspac promoter was amplified from pMUTIN4 with primers Spac F and Spac R as a 0.4 kbp product and TOPO cloned into pCR4Blunt-TOPO. Transformants were screened by PCR amplification with M13 Forward and M13 Reverse primers for the presence of a 0.5 kbp insert. Positive clones were submitted for sequencing with the same primers. Plasmid pCR4Blunt-TOPO-Pspac was digested with Smal and Xhol and the 0.3 kbp fragment was gel-purified. Vector pFP988Dss-T-H-C was digested with Smal and Xhol and the 9 kbp vector was isolated by gel purification. The digested vector and Pspac insert were ligated and transformed into E. co//Top10 cells. Transformants were screened by PCR amplification with primers SpacF Seq and N7SeqR2. Positive clones gave a 0.7 kbp product. Plasmid DNA was prepared from positive clones and further screened by PCR amplification with primers SpacF Seq and N3SeqF2. Positive clones gave a 3 kbp PCR product and were named pFP988DssPspac-T-H-C.<br>
Integration into 5. subtilis BE1010 to form B. subtilis AsacB::T-H-C::erm #28 comprising exogenous thl. hbd. and crt genes. Competent cells of 6. subtilis BE1010 were prepared as described in Doyle et a/., J. Bacteriol. 144:957-966 (1980). Competent cells were harvested by centrifugation and the cell pellets were resuspended in a small volume of it. To 1 volume of competent cells, 2 volumes of SPII-EGTA medium (Methods for General and Molecular Bacteriology, P. Gerhardt et al., Eds, American Society for Microbiology, Washington, DC (1994)) was added. Aliquots of 0.3 ml of cells were dispensed into test tubes and the plasmid pFP988DssPspac-T-H-C was added to the tubes. Cells were incubated for 30 minutes at 37 °C with shaking, after which 0.1 ml of 10% yeast extract was added to each tube and the cells were further incubated for 60 min. Transformants were plated for selection on LB erythromycin plates using the double agar overlay method (Methods for General and Molecular Bacteriology, supra). Transformants were initially screened by PCR amplification with primers Pamy SeqF and N5SeqF3. Positive clones that amplified the expected 2 kbp PCR product were further screened by PCR amplification. If insertion of the cassette into the chromosome had occurred via a double crossover event then primer set sacB Up and N7SeqR2 and primer set sacB Dn and N4SeqR3 would amplify a 1.7 kbp and a 2.7 kbp product respectively. A positive clone was identified and named B. subtilis AsacB::T-H-C::erm #28.<br>
Plasmid Expression of EgTER. aid, and bdhB genes. The three remaining 1-butanol genes were expressed from plasmid pHT01 (MoBitec). Plasmid pHT01 is a Bacillus-E. coli shuttle vector that replicates via a theta mechanism. Cloned proteins are expressed from the GroEL promoter fused to a /acO sequence. Downstream of the lacO is the efficient RBS from the gsiB gene followed by a MCS. The aid gene was .amplified by PCR with primers AF BamHI and AR Aat2 using pUC19dHS-ald-bdhB (described in Example 9) as template, creating a 1.4 kbp product. The product was TOPO cloned into pCR4-TOPO and transformed into £. co//Top10 cells. Transformants were screened with M13 Forward and M13 Reverse primers. Positive clones amplified a 1.6 kbp product Clones were submitted for sequencing with primers M13 forward and M13 reverse, N31SeqF2, N31SeqF3, N32SeqR2, N32SeqR3 and N32SeqR4. The plasmid was named pCR4TOPO-B/A-ald.<br>
Vector pHT01 and plasmid pCR4TOPO-B/A-ald were both digested with BamHI and Aatll. The 7.9 kbp vector fragment and the 1.4 kbp aid  .<br>
I together to create pHT01-ald. The ligation was transformed into E. coli TopIO cells and transformants were screened by PCR amplification with primers N31SeqF1 and HT R for a 1.3 kbp product.<br>
To add the last two steps of the pathway to the pHT01 vector, two cloning schemes were designed. For both schemes, EgTER and bdhB were amplified together by SOE. Subsequently, the EgTER-ibcf/? fragment was either cloned into pHT01-ald creating pHT01-ald-EB or cloned into pCR4-TOPO-B/A-ald creating pCR4-TOPO-ald-EB. The ald-EgTer-bdhB fragment from the TOPO vector was then cloned into pHT01 creating pHT01-AEB.<br>
An EgTER-bdhB fragment was PCR amplified using primers Forward 1 (E) and Reverse 2 (B), using template DNA given as SEQ ID N0:208. The resulting 2.5 kbp PCR product was TOPO cloned into pCR4Blunt-TOPO, creating pCR4Blunt-TOPO-E-B. The TOPO reaction was transformed into £ co//Top10 cells. Colonies were screened with M13 Forward and M13 Reverse primers by PCR amplification. Positive clones generated a 2.6 kbp product. Clones of pCR4Blunt-TOPO-E-B were submitted for sequencing with primers M13 Forward and Reverse, N62SeqF2, N62SeqF3, N62SeqF4, N63SeqR1, N63SeqR2, N63SeqR3, N11Seq F1 and N11Seq F2, N12SeqR1 and N12SeqR2.<br>
Plasmid pCR4Blunt-TOPO-E-B was digested with Hpal and Aatll to release a 2.4 kbp fragment. The E-B fragment was treated with Klenow polymerase to blunt the end and then was gel-purified. Plasmid pHT01-ald was digested with Aatll arid treated with Klenow polymerase to blunt the ends. The vector was then treated with calf intestinal alkaline phosphatase and was gel-purified. The E-B fragment was ligated to the linearized vector pHT01-ald, transformed into E coli TopIO cells, and selected on LB plates containing 100 p,g/ml_ ampicillin. Transformants were screened by PCR amplification with primers N3SeqF1 and N63SeqR1 to give a 2.4 kbp product. The resulting plasmid, pHT01-ald-EB, was transformed into JM103 cells, a recA* E. coll strain. Plasmids prepared from recA* strains form more multimers than recA~ strains. Bacillus subtilis transforms more efficiently with plasmid multimers rather wds for General and Molecular Bacteriology, supra}. Plasmid DNA was prepared from JM103 and transformed into competent 6. subtilis AsacB::T-H-C::erm #28 forming strain B. subtilis AsacB::T-H-C::erm #28/pHT01-ald-EB. Competent cells were prepared and transformed as previously described. Transformants were selected on LB plates containing 5 ng/mL chloramphenicol and screened by colony PCR with the primers N31SeqF1 and N63SeqR4 for a 1.3 kbp product.<br>
In the alternate cloning strategy, pCR4Blunt-TOPO-E-B was digested with.Hpal and Aatll releasing a 2.4 kbp fragment that was gel-purified. Plasmid pCR4-TOPO-B/A-ald was digested with Hpal and Aatll and the 5.4 kbp vector fragment was gel-purified. The vector fragment from pCR4-TOPO-B/A-ald was ligated with the Hpal-Aatll E-B fragment creating pCR4-TOPO-ald-EB. The ligation was transformed into E. coli TopIO cells and the resulting transformants were screened by PCR amplification with primers N11 SeqF2 and. N63SeqR4 for a 2.1 kbp product. Plasmid pCR4-TOPO-ald-EB was digested with BamHI and Aatll and Sphl. The BamHl/Aatll digest releases a 3.9 kbp ald-EB fragment that was gel-purified. The purpose of the Sphl digest was to cut the remaining vector into smaller fragments so that it would not co-migrate on a gel with the ald-EB insert. Vector pHT01 was digested with BamHI and Aatll and the 7.9 kbp vector fragment was gel-purified. The vector and ald-EB insert fragments were ligated to form plasmid pHT01-AEB and transformed into E. coli TopIO cells. Colonies were screened by PCR amplification with primers N62SeqF4 and HT R for a 1.5 kbp product. Plasmid was prepared and transformed into JM103. Plasmid DNA was prepared from JM103 and transformed into competent 6. subtilis AsacB::T-H-C::erm #28 forming strain 6. subtilis AsacB::T-H-C::erm #287 pHT01-AEB. Competent BE1010 cells were prepared and transformed as previously described. Bacillus transformants were screened by PCR amplification with primers N31SeqF1 and NB3SeqR4 for a 1.3 kbp product.<br>
production from recombinant B. subtms.<br>
Three independent isolates of each strain of B. subtilis AsacB::T-H-C::erm #28/pHT01-ald-EB and B. subtilis AsacB::T~H-C::erm #28/ pHT01-AEB were inoculated into shake flasks (approximately 175 ml total volume) containing 15 ml of medium. A 6. subtilis BE1Q10 strain lacking the exogenous 1-butanol, six gene pathway was also included as a negative control. The medium contained (per liter): 10 mL of 1 M (NH4)2S04; 5 ml of 1 M potassium phosphate buffer, pH 7.0; 100 mL of 1 M MOPS/KOH buffer, pH 7.0; 20 ml of 1 M L-glutamic acid, potassium salt; 10 g glucose; 10 ml of 5 g/L each of L-methionine, L-tryptophan, and L-lysine; 0.1 g each of yeast extract and casamino acids; 20 ml of metal mix; and appropriate antibiotics (5 mg chloramphenicol and erythromycin for the recombinant strains). The metal mix contained 200 mM MgCI2, 70 mM CaCI2, 5 mM MnCI2,'0.1 mM FeCI3, 0.1 mM ZnCI2, 0.2 mM thiamine hydrochloride, 172 uM CuSO4, 253 uM CoCI2, and 242 uM Na2Mo04. The flasks were inoculated at a starting OD6oo of £ 0.1 units, sealed with non-vented caps, and incubated at 37 °C with shaking at approximately 200 rpm.<br>
Approximately 24 h after inoculation, an aliquot of the broth was analyzed by HPLC (Shodex Sugar SH 1011 column) with refractive index (Rl) detection and GC (Varian CP-WAX 58(FFAP) CB column, 0.25 mm X 0.2 fjm X 25 m) with flame ionization detection (FID) for 1-butanol content, as described in the General Methods section. The results of the 1-butanol determinations are given in Table 1 1 .<br>
Table 11<br>
Production of 1-butanol by strains 6. subtilis AsacB::T-H-C::erm #28/pHT01-ald-EB and B. subtilis AsacB::T-H-C::erm #287 pHT01-AEB<br>
(Table Removed)<br><br>
EXAMPLE 15 Production of 1-Butanol from Glucose or Sucrose bv Recombinant E. coli.<br>
To endow £ coli MG1655 with the ability to use sucrose as the carbon and energy source for 1-butanol production, a sucrose utilization gene cluster (cscBKA) from plasmid pScrl (described below) was subcloned into pBHR-Ptrc-ald(opt) (described in Example 13) in this organism. The sucrose utilization genes (cscA, csck, and cscB) encode a sucrose hydrolase (CscA), given as SEQ ID NO:157, D-fructokinase (CscK), given as SEQ ID NO: 158, and sucrose permease (CscB), given as SEQ ID N0:159. To allow constitutive expression of the three genes from their natural promoter, the sucrose-specific represser gene, cscR, that regulates the gene cluster is not present in the construct.<br>
Cloning and expression of the sucrose utilization gene cluster cscBKA in plasmid pBHR-Ptrc-aldfopt). The sucrose utilization gene cluster cscBKA, given as SEQ ID N0:156, was isolated from genomic DMA of a sucrose-utilizing £ coli strain derived from £ coli strain ATCC 13281. The genomic DNA was digested to completion with BamHI and EcoRI. Fragments having an average size of about 4 kbp were isolated from an agarpse gel, ligated to plasmid pLitmus28 (New England Biolabs, Beverly, MA), which was then digested with BamHI and EcoRI. The resulting DNA was transformed into ultracompetent £ co//TOP10F' (Invitrogen, Carlsbad, CA). The transformants were plated on MacConkey agar plates containing 1 % sucrose and 100 ^g/mL ampicillin and screened for purple colonies. Plasmid DNA was isolated from the purple transformants and sequenced using primers M13 Forward (SEQ ID N0:45), M13 Reverse (SEQ ID NO:46), scrt (SEQ ID N0:1.60), scr2 (SEQ ID NO:161), scr3 (SEQ ID NO:162), and scr4 (SEQ ID N0:163). The plasmid containing cscB, cscK, and cscA (cscBKA) genes was designated pScrl was digested with Xhol and treated with the Klenow fragment of DNA polymerase to make blunt ends. The plasmid was then digested with Agel, and the 4,179 bp cscBKA gene cluster fragment was gel-purified. Plasmid pBHR-Ptrc-ald(opt) was prepared as described in Example 13 and was digested with Agel and Nael. The resulting 6,003 bp pBHR-Ptrc-ald(opt) fragment was gel-purified. The cscBKA fragment was ligated with the pBHR-Ptrc-ald(opt), yielding pBHR-Ptrc-ald(opt)-cscAKB. Plasmid pBHR-Ptrc-ald(opt)-cscAKB was transformed into E. coli NovaXG electrocompetent cells (Novagen, Madison, Wl) and sucrose utilization was confirmed by plating the transformants.on McConkey agar plates containing 2% sucrose and 25 |o,g/mL kanamycin. In the pBHR-Ptrc-ald(opt)-cscAKB construct, the sucrose utilization genes were cloned downstream of Ptrc-ald(opt) as a separate fragment in the order csc&gt;4, cscK, and cscB.<br>
Alternatively, the sucrose utilization genes were cloned in the opposite direction in pBHR-Ptrc-ald(opt). Plasmid pBHR-Ptrc-aid(opt) was digested with Seal and Agel, and the 5,971 bp pBHR-Ptrc-ald(opt) fragment was gel-purified. The 4,179 bp cscBKA fragment, prepared as described above, was ligated with the pBHR-Ptrc-ald(opt) fragment, yielding pBHR-Ptrc-ald(opt)-cscBKA. Plasmid pBHR-Ptrc-a!d(opt)-cscBKA was transformed into E. coli NovaXG electrocompetent cells (Novagen, Madison, Wl) and sucrose utilization was confirmed by plating the transformants on McConkey agar plates containing 2% sucrose and 25 p,g/mL kanamycin. In the pBHR-Ptrc-ald(opt&gt;-cscBKA construct, the sucrose utilization genes were cloned as a separate fragment downstream of Ptrc-ald(opt) in the order cscB, cscK, and cscA<br>
Demonstration of 1-butanol production from glucose or sucrose using recombinant E. coli. E. constrain MG1655 1.5GI-yqhD::Cm (described in Example 13) was transformed with plasmids pTrc99a-E-C-H-T (prepared as described in'Example 13) and pBHR-Ptrc-ald(opt)-cscAKB or pBHR-Ptrc-ald(opt)-cscBKA to produce two strains, MG16551.5GI-yqhD::Cm/pTrc99a-E-C-H-T/pBHR-Ptrc-ald(opt)-cscAKB #9 and MG1655 1.5GI-yqhD::Cm/pTrc99a-E-C-H-T/pBHR-Ptrc-ald(opt)-  cultures of the two strains were prepared by growing the cells in LB medium containing 25 |o,g/mL of kanamycin and 100 u.g/mL of carbenicillin. These cells were then used to inoculate shake flasks (approximately 175 ml total volume) containing 50, 70 and 150 ml of TM3a/glucose medium (with appropriate antibiotics) to represent high, medium and low oxygen conditions, respectively, as described in Example 13. A third strain, £ co//MG1655/pScr1, grown in TM3a/glucose medium containing 100 ng/mL carbenicillin, was used as a negative control. For each of the strains, an identical set of flasks was prepared with TM3a/sucrose medium (with appropriate antibiotics). TM3a/sucrose medium is identical to TM3a/glucose medium except that sucrose (10 g/L) replaces glucose. The flasks were inoculated at a starting OD550 of ^ 0.03 units and incubated as described in Example 13. With the exception of the negative control flasks, IPTG was added to the flasks (final concentration of 0.04 mM) when the cultures reached an OD550 between 0.2 and 1.8 units. The cells were harvested when the ODsso of the cultures increased at least 3-fold.<br>
Approximately 24 h after inoculation, an aliquot of the broth was analyzed by HPLC (Shodex Sugar SH1011 column) with refractive index (Rl) detection and GC (HP-INNOWax column, 30 m x 0.53 mm id, 1 jam film thickness) with flame ionization detection (FID) for 1-butanol content, as described in the General Methods section.<br>
The concentrations of 1-butanol in cultures following growth in the glucose and sucrose-containing media are given in Table 12 and Table 13, respectively. Both recombinant E. constrains containing the 1-butanol biosynthetic pathway produced 1-butanol from glucose and sucrose under all oxygen conditions, while the negative control strain produced no detectable 1-butanol.<br>
Table 12<br>
Production of 1-butanol from glucose by recombinant E. constrains MG1655 1.5GI-vahD::Cm/pTrc99a-E-C-H-T/pBHR-Ptrc-aldfoptVcscAKB #9 and MG1655 1.5GI-vahD::Crn/pTrc99a-E-C-H-T/DBHR-Ptrc-aldfaDtV<br>
(Table Removed)<br>
Table 13 Production of 1-butanol from sucrose by recombinant E. constrains.<br>
(Table Removed)<br>
 EXAMPLE 16<br>
Production of 1-Butanol from Sucrose using Recombinant B. subtilis. This example describes the production of 1-butanol from sucrose using recombinant Bacillus subtilis. Two independent isolates of B. subtilis strain AsacB::T-H-C::erm #28/pHT01-ald-EB (Example 14) were examined for 1-butanol production essentially as described in Example 14.<br>
Toculated into shake flasks (approximately 175 ml total volume) containing either 20 mL or i 00 ml of medium to simulate high and low oxygen conditions, respectively. Medium A was exactly as described in.Example 14, except that glucose was replaced with 5 g/L of sucrose. Medium B was identical to the TM3a/glucose medium described in Example 13, except that glucose was replaced with 10 g/L sucrose and the medium was supplemented with (per L) 10 mL of a 5 g/L solution of each of L-methionine, L-tryptophan, and L-lysine. The flasks were inoculated at a starting OD550 of ^ 0.1 units, capped with vented caps, and incubated at 34 °C with shaking at 300 rpm.<br>
Approximately 24 h after inoculation, an aliquot of the broth was analyzed by GC (HP-iNNOWax column, 30 m x 0.53 mm id, 1.0 jim film thickness) with FID detection for 1-butanol content, as described in the General Methods section. The results of the 1-butanol determinations are given in Table 14. The recombinant Bacillus strain containing the 1-butanol biosynthetic pathway produced detectable levels of 1-butanol under high and low oxygen conditions in both media.<br>
Table 14<br>
Production of 1-butanol from sucrose bv B. subtilis strain AsacB::T-H-C::erm #28/pHT01-ald-EB<br>
(Table Removed)<br><br>
EXAMPLE 17<br>
Production of 1-Butanol from Glucose and Sucrose Using Recombinant Saccharomvces cerevisiae<br>
This Example describes the production of 1-butanol in the yeast Saccharomyces cerevisiae. Of the six genes encoding enzymes catalyzing the steps in the 1-butanol biosynthetic pathway, five were cloned into three compatible yeast 2 micron (2p.) plasmids and co-expressed in Saccharomyces cerevisiae. The "upper pathway" is defined as the first three enzymatic steps, catalyzed by acetyl-CoA acetyltransferase (thIA, thiolase), 3-hydroxybutyryl-CoA dehydrogenase (hbd), and crotonase (erf). The lower pathway is defined as the fourth (butyl-CoA dehydrogenase, ter) and the fifth (butylaldehyde dehydrogenase, aid) enzymatic steps of the pathway. The last enzymatic step of the 1-butanol pathway is catalyzed by alcohol dehydrogenase, which may be encoded by endogenous yeast genes, e.g., adhl and adhll.<br>
Expression of genes in yeast typically requires a promoter, followed by the gene of interest, and a transcriptional terminator. A number of constitutive yeast promoters were used in constructing expression cassettes for genes encoding the 1-butanol biosynthetic pathway, including FBA, GPD, and GPM promoters. Some inducible promoters, e.g. GAL1, GAL10, CUP1 were also used in intermediate plasmid construction, but not in the final demonstration strain. Several transcriptional terminators were used, including FBAt, GPDt, GPMt, ERGIOt, and GAL1t. Genes encoding the 1-butanol biosynthetic pathway were first subcloned into a yeast plasmid flanked by a promoter and a terminator, which yielded expression cassettes for each gene. Expression cassettes were optionally combined in a single vector by gap repair cloning, as described below. For example, the three gene cassettes encoding the upper pathway were subcloned into a yeast 2\i plasmid. The ter and aid genes were each expressed individually in the 2ja plasmids.<br>
all three plasmids in a single yeast strain resulted in a functional 1-butanol biosynthetic pathway. Alternatively, several DNA fragments encoding promoters, genes, and terminators were directly combined in a single vector by gap repair cloning.<br>
Methods for constructing plasmids and strains in veast Saccharomvces cerevisiae. Basic yeast molecular biology protocols including transformation, cell growth, gene expression, gap repair recombination, etc. are described in Methods in Enzymology, Volume 194, Guide to Yeast Genetics and Molecular and Cell Biology (Part A, 2004, Christine Guthrie and Gerald R. Fink (Eds.), Elsevier Academic Press, San Diego, CA).<br>
The plasmids used in this Example were £. co//-S. cerevisiae shuttle vectors, pRS423, pRS424, pRS425, and pRS426 (American Type Culture Collection, Rockville, MD), which contain an £. coli replication origin (e.g., pMB1), a yeast 2ja origin of replication, and a marker for nutritional selection. The selection markers for these four vectors are His3 (vector pRS423), Trp1 (vector pRS424), Leu2 (vector pRS425) and Ura3 (vector pRS426). These vectors allow strain propagation in both E. colt and yeast strains. A yeast haploid strain BY4741 (MATa his3A1 leu2AO met15AO ura3AO) (Research Genetics, Huntsville, AL, also available from ATCC 201388) and a diploid strain BY4743 (MATa/alpha his3Al/his3A1 leu2AO/leu2AO lys2AO/LYS2 MET15/met15AO ura3AO/ura3AO) (Research Genetics, Huntsville, AL, also available from ATCC 201390) were used as hosts for gene cloning and expression. Construction of expression vectors for genes encoding 1-butanol biosynthetic pathway enzymes were performed by either standard molecular cloning techniques in £ coli or by the gap repair recombination method in yeast.<br>
The gap repair cloning approach takes advantage of the highly efficient homologous recombination in yeast. Typically, a yeast vector DNA is digested (e.g., in its multiple cloning site) to create a "gap" in its sequence. A number of insert DMAs of interest are generated that contain a &gt; 21 bp sequence at both the 5' and the 3' ends that sequentially overlap with each other, and with the 5' and 3' terminus of the vector DNA. For<br>
a yeast expression vector for "Gene X', a yeast promoter and a yeast terminator are selected for the expression cassette. The promoter and terminator are amplified from the yeast genomic DNA, and Gene X is either PCR amplified from its source organism or obtained from a cloning vector comprising Gene X sequence. There is at least a 21 bp overlapping sequence between the 5' end of the linearized vector and the promoter sequence, between the promoter and Gene X, between Gene X and the terminator sequence, and between the terminator and the 3' end of the linearized vector. The "gapped" vector and the insert DNAs are then co-transformed into a yeast strain and plated on the SD minimal dropout medium, and colonies are selected for growth of cultures and mini preps for plasmid DNAs. The presence of correct insert combinations can be confirmed by PCR mapping. The plasmid DNA isolated from yeast (usually low in concentration) can then be transformed into an E. coli strain, e.g. TOP10, followed by mini preps and restriction mapping to further verify the plasmid construct. Finally the construct can be verified by sequence analysis. Yeast transformants of positive plasmids are grown in SD medium for performing enzyme assays to characterize the activities of the enzymes expressed by the genes of interest.<br>
Yeast cultures were grown in YPD complex medium or Synthetic Minimal dropout medium containing glucose (SD medium) and the appropriate compound mixtures that allow complementation of the nutritional selection markers on the plasmids (Methods in Enzymology, Volume 194, Guide to Yeast Genetics and Molecular and Cell Biology, 2004, Part A, pp.13-15). The sugar component in the SD drop out medium was 2% glucose. For 1-Butanol production, yeast cultures were also grown in Synthetic Minimal dropout medium with 2% sucrose (SS medium).<br>
For enzyme activity analysis, a single colony of each strain was streaked onto a fresh plate containing SD minimal drop out medium and incubated at 30 °C for 2 days. The cells on this plate were used to inoculate 20 ml of SD drop out medium and in a 125 ml shake flask and grown overnight at 30 °C, with shaking at 250 rpm. The optical density (ODBoo) of the overnight culture was measured, and the culture was diluted<br>
81ir0DDilpO^CIliii"250 ml of the same medium in a 1.0 L shaKe TiasK, and grown at 30 °C with shaking at 250 rpm to an OD6oo of between 0.8 to 1.0. The cells were then harvested by centrifugation at 2000 x g for 10 mln, and resuspended in 20 ml of 50 mM Tris-HCI buffer, pH 8.5. Enzyme assays were performed as described above.<br>
Construction of plasmid pNY102 for thIA and hbd co-expression. A number of dual expression vectors were constructed for the co-expression of thIA and hbd genes. The Saccharomyces cerevisiae ERG 10 gene is a functional ortholog of the thIA gene. Initially, a dual vector of ERG10 and hbd was constructed using the yeast GAL1-GAL10 divergent dual promoter, the GAL1 terminator (GAL1t) and the ERG 10 terminator (ERGIOt). The ERG10 gene-ERG10t DMA fragment was PCR amplified from genomic DNA of Saccharomyces cerevisiae strain BY4743, using primers OT731 (SEQ ID NO:164) and OT732 (SEQ ID NO:165 ). The yeast GAL1-GAL10 divergent promoter was also amplified by PCR from . BY4743 genomic DNA using primers OT733 (SEQ ID NO:166) and OT734 (SEQ ID NO:167). The hbd gene was amplified from £ coll plasmid pTrc99a-E-C-H-T (described in Example 13) using PCR primers OT735 (SEQ ID NO:168) and OT736 (SEQ ID N0:169). GAL1t was amplified from BY4743 genomic DNA using primers OT737 (SEQ ID N0:170) and OT738 (SEQ ID N0:171). Four PCR fragments, erg?0-ERG10t, GAL1-GAL10 promoters, hbd, and GAL1t, were thus obtained with approximately 25 bp overlapping sequences between each adjacent PCR fragment. GAL1t and ERG10-ERG10t fragments each contain approximately 25 bp overlapping sequences with the yeast vector pRS425. To assemble these sequences by gap repair recombination, the DNA fragments were co-<br>
i<br>
transformed into the yeast strain BY4741 together with vector pRS425 which was digested with BamHI and Hindi 11 enzymes. Colonies were selected from SD-Leu minimal plates, and clones with inserts were identified by PCR amplification. The new plasmid was named pNY6 (pRS425.ERG10t-erg10-GAL10-GAL1-hbd-GAL1t). Further confirmation was performed by restriction mapping.<br>
The yeast strain BY4741 (pNY6), prepared by transforming plasmid pNY6 into S. cerevisiae BY4741, showed good Hbd activity but no thiolase<br>
SiWfy/Ri&amp;Sffle'lack of thiolase activity, the ERG10 gene was replaced with the thIA gene by gap repair recombination. The thIA gene was amplified from £ co//vector pTrc99a-E-C-H-T by PCR using primers OT797 (SEQ ID N0:172) which adds a Sphl restriction site, and OT798 (SEQ ID N0:173) which adds an Asci restriction site. Plasmid pNY6 was digested with Sphl and Pstl restriction enzymes, gel-purified, and co-transformed into yeast BY4741 along with the PCR product of thIA. Due to the 30 bp overlapping sequences between the PCR product of thIA and the digested pNY6, the thIA gene was recombined into pNY6 between the GAL10 promoter and the ERG1 Ot terminator. This yielded plasmid pNY7 (pRS425.ERG10t-thlA- GAL10-GAL1-hbd-GAL1t), which was verified by PCR and restriction mapping.<br>
In a subsequent cloning step based on gap repair recombination, the GAL10 promoter in pNY7 was replaced with the CUP1 promoter, and the GAL1 promoter was replaced with the strong GPD promoter. This plasmid, pNY10 (pRS425. ERG10t-thlA- CUP1- GPD -hbd-GAL1t) allows for the expression of the thIA gene under CUP1, a copper inducible promoter, and the expression of the hbd gene under the GPD promoter.<br>
!<br>
The CUP1 promoter sequence was PCR amplified from yeast BY4743 genomic DNA using primers OT806 (SEQ ID NO:174), and OT807 (SEQ . ID NO:175). The GPD promoter was amplified from BY4743 genomic DNA using primers OT808 (SEQ ID N0:176) and OT809 (SEQ ID N0:177). PCR products of the CUP1 and the GPD promoters were combined with pNY7 plasmid digested with Ncol and Sphl restriction enzymes. From this gap repair cloning step, plasmid pNY10 was constructed, which was verified by PCR and restriction mapping. Yeast BY4741 strain containing pNY10 had Hbd activity, but no ThIA activity. The Hbd activity under GPD promoter was significantly improved compared to the GAL1 promoter controlled Hbd activity (1.8 U/mg vs. 0.40 U/mg). Sequencing analysis revealed that the thIA gene in pNY10 had a one base deletion near the 3' end, which resulted in a truncated protein. This explains the lack of thiolase activity in the strain.<br>
Plasmid pNY12 was constructed with the correct thIA gene sequence. The thIA gene was cut from the vector pTrc99a-E-C-H-T by<br>
Ascl. The FBAi promoter was PCR amplified from BY4743 genomic DNA using primers OT799 (SEQ ID NO:178 ) and OT761 (SEQ ID N0:179), and digested with Sail and Sphl restriction enzymes. The thIA gene fragment and FBA1 promoter fragment were ligated into plasmid pNY10 at Ascl and Sail sites, generating plasmid pNY12 (pRS425.ERG10t-th!A-FBA1), which was confirmed by restriction mapping. pNY12 was transformed into yeast strain BY4741 and the resulting transformant showed a ThIA activity of 1.66 U/mg.<br>
The FBA1 promoter-#7//4 gene fragment from pNY12 was re-subcloned into pNY10. The pNY10-vector was cut with the Ascl restriction enzyme and ligated with the Ascl digested FBA1 promoter-^W gene fragment isolated from plasmid pNY12. This created a new plasmid with two possible insert orientations. The clones with FBA1 and GPD   . promoters located adjacent to each other in opposite orientation were chosen and this plasmid was named pNY.102. pNY102 (pRS425. ERG10t-thlA- FBAI- GPD -hbd-GAL1t) was verified by restriction mapping. Strain DPD5206was made by transforming pNY102 into yeast strain BY4741. The ThIA activity of DPD5206 was 1.24 U/mg and the Hbd activity was 0.76 U/mg.<br>
Construction of plasmid pNY11 for erf expression. The erf gene expression cassette was constructed by combining the GPM1 promoter, the erf gene, and the GPM1t terminator into vector pRS426 using gap repair recombination in yeast. The GPM1 promoter was PCR amplified from yeast BY4743 genomic DNA using primers OT803 (SEQ ID N0:180) and OT804 (SEQ ID NO:181). The erf gene was amplified using PCR primers OT785 (SEQ ID NO:182) and OT786 (SEQ ID N0:183)from E. coli plasmid pTrc99a-E-C-H-T. The GPM1t terminator was PCR amplified from yeast BY4743 genomic DNA using OT787 (SEQ ID NO:184) and OT805 (SEQ ID NO: 185). Yeast vector pRS426 was digested with BamHI and Hindlll and was gel-purified. This DNA was co-transformed with the PCR products of the GPM1 promoter, the erf gene and the GPM1 terminator into yeast BY4741 competent cells. Clones with the correct inserts were verified by PCR and restriction mapping and the resulting<br>
 (pNY11: pRS426-GPM1-crt-GPM1t) had a Crt activity of 85 U/mg.<br>
Construction of olasmid pNY103 for thJA, hbd and crt co-expression. For the co-expression of the upper 1 -butanol pathway enzymes, the crt gene cassette from pNY11 was subcloned into plasmid pNY102 to create an hbd, thIA, and crt expression vector. A 2,347 bp DNA fragment containing the GPM1 promoter, the crt gene, and the GPM1 terminator was cut from plasmid pNYHwith Sac! and Notl restriction enzymes and cloned into vector pNY102, which was digested with Notl and partially digested with Sacl, producing the expression vector pNY103 (pRS425. ERG10t-thlA- FBA1- GPD -hbd-GAL1t-GPM1t-crt-GPM1). Following confirmation of the presence of all three cassettes in pNY103 by digestion with Hindlll, the plasmid was transformed into yeast BY4743 cells and the transformed yeast strain was named DPD5200. When grown under standard conditions, DPD5200 showed ThIA, Hbd, and Crt enzyme activities of 0.49 U/mg, 0.21 U/mg and 23.0 U/mg, respectively.<br>
Construction of plasmid pNY8 for aid expression. Acodon optimized gene named tery (SEQ ID N0:186), encoding the Ter protein (SEQ ID N0:1,87), and a codon optimized gene named aldy (SEQ ID N0:188), encoding the Aid protein (SEQ ID NO:189) were synthesized using preferred codons of Saccharomyces cerevisiae. Plasmid pTERy containing the codon optimized fergene and pALDy containing the codon optimized aid gene were made by DNA2.0 (Palo Alto, CA).<br>
To assemble pNY8 (pRS426.GPD-ald-GPDt), three insert fragments including a PCR product of the GPD promoter (synthesized from primers OT800 (SEQ ID N0:190) and OT758, (SEQ ID NO:191), and BY4743 genomic DNA), an aldy gene fragment excised from pALDy by digestion with Ncol and Sfil (SEQ ID N0:188), and a PCR product of the GPD terminator (synthesized from primers OT754 (SEQ ID NO:192) and OT755 (SEQ ID N0:193), and BY4743 genomic DNA) were recombined with the BamHl', Hindlll digested pRS426 vector via gap repair recombination cloning. Yeast BY4741 transformation clones were analyzed by PCR mapping. The new plasmid thus constructed, pNY8, was further confirmed by restriction mapping. The yeast BY4741pNY8 were analyzed Tor Aid activity and the specific activity towards butyryl-CoA was approximately 0.07 U/mg.<br>
Construction of plasmids pNY9 and PNY13 for ter expression. The codon optimized tery gene was cloned into vector pRS426 under control of the FBA1 promoter by gap repair cloning. The FBA1 promoter was PCR amplified from yeast BY4743 genomic DNA using primers OT760 (SEQ ID N0:194) and OT792 (SEQ ID NO:195). The tery gene was obtained by digestion of plasmld pTERy by Sphl and Notl restriction enzymes that resulted in the fragment given as SEQ ID NO: 186. The PCR fragment of FBA1 terminator was generated by PCR from yeast BY4743 genomic DNA using primers OT791 (SEQ ID NO:196) and OT765 (SEQ ID N0:197). Three DNA fragments, the FBA1 promoter, the ter gene and the FBA1 terminator, were combined with the BamHI, Hindlll digested pRS426 vector and transformed into yeast BY4741 by gap repair recombination. The resulting plasmid, pNY9 (pRS426-FBA1-tery-FBA1t) was confirmed by PCR mapping, as well as restriction digestion. The yeast BY4741 transformant of pNY9 produced a Ter activity of 0.26 U/mg.<br>
To make the final 1-butanol biosynthetic pathway strain, it was necessary to construct a yeast expression strain that contained several plasmids, each with a unique nutritional selection marker. Since the parent vector pRS426 contained a Ura selection marker, the ter expression cassette was subcloned into vector pRS423, which contained a His3 marker. A 3.2 kb fragment containing the FBA1-tery-FBA1t cassette was isolated from plasmid pNY9 by digestion with Sacl and Xhol restriction enzymes, and ligated into vector pRS423 that was cut with these same two enzymes. The new plasmid, pNY13 (pRS423~FBA1-tery-FBA1t) was mapped by restriction digestion. pNY13 was transformed into BY4741 strain and the transformant was cultured in SD-His medium, yielding a strain with a Ter activity of 0.19 U/mg.<br>
Construction of a yeast strain containing 1-butanol biosvnthetic pathway genes for demonstration of 1-butanol production. As described above, yeast strain DPD5200 was constructed by transformation of plasmid pNY103 into S. cerevisiae strain BY4743, which allows co-expression of thIA, hbd and crt genes. Yeast competent cells of DPD5200<br>
above, and plasmids pNY8 ana pNYia were co-transformed into DPD5200, generating strain DPD5213. DPD5213 allows for the simultaneous constitutive expression of five genes in the 1-butanol biosynthetic pathway, thIA, hbd, crt, ferand aid. Strain DPD5212 (S. cerevisiae strain BY4743 transformed with empty plasmids, pRS425 and pRS426) was used as a negative control. Four independent isolates of strain DPD5213 were grown on SD-Ura,-Leu,-His dropout minimal medium in the presence of either 2% glucose or 2% sucrose to allow the growth complementation of all three plasmids. A single isolate of DPD5212 was similarly grown in appropriate medium.<br>
To demonstrate 1-butanol production by aerobic cultures, a single colony of each strain was streaked onto a fresh agar plate containing SD minimal drop out growth medium (containing 2% glucose) or SS minimal drop out growth medium (containing 2% sucrose) and incubated at 30 °C for 2 days. Cells from these plates were used to inoculate 20 ml of the minimal drop out medium (either SD or SS) in 125 ml plastic shake flasks and were grown overnight at 30 °C with shaking at 250 rpm. The optical density (ODeoo) of the overnight culture was measured, the culture was diluted to ODeooof 0.1 in 25 mL of the same medium in a 125 ml shake flask, and grown at 30 °C with shaking at 250 rpm.<br>
Aliqubts of the culture were removed at 24 h and 48 h.for GC analysis of 1-butanol production (HP-INNOWax column, 30 m x 0.53, mm id, 1 urn film thickness) with FID detection, as described in the General Methods section. The results of the GC analysis are given in Table 15.<br>
Table 15<br>
Production of 1-butanol from glucose and sucrose by S. cerevisiae strain DPD5213<br>
(Table Removed)<br>
EXAMPLE 18 (Prophetic) Expression of the 1-Butanol Biosynthetic Pathway in Lactobacillus<br>
plantarum<br>
The purpose of this prophetic Example is to describe how to express the 1-butanol biosynthetic pathway in Lactobacillus plantarum. The six genes of the 1-butanol pathway, encoding six enzyme activities, are divided into two operons for expression. The first three genes of the pathway (thl, hbd, and erf, encoding the enzymes acetyl-CoA acetyltransferase, 3-hydroxybutyryl-CoA dehydrogenase, and crotonase, respectively) are integrated into the chromosome of Lactobacillus plantarum by homologous recombination using the method described by Hols et al. (Appl. Environ. Microbiol. 60:1401-1413 (1994)). The last three genes (EgTER, aid, and bdhB, encoding the enzymes butyryl-CoA dehydrogenase, butyraldehyde dehydrogenase and butanol dehydrogenase, respectively) are cloned into an expression plasmid and transformed into the Lactobacillus strain carrying the integrated upper pathway 1-butanol genes. -Lactobacillus is grown in MRS medium (Difco Laboratories, Detroit, Ml) at 37 °C. Chromosomal DNA is isolated from Lactobacillus plantarum as described by Moreira et al. (BMC Microbiol. 5:15(2005)).<br>
J'SOtalrrfe'iiiSiilitBiO'The thl-hbd-crt cassette under the control of the synthetic P11 promoter (Rud et al., Microbiology 152:1011-1019 (2006)) is integrated into the chromosome of Lactobacillus plantarum ATCC BAA-793 (NCIMB 8826) at the ldhL.1 locus by homologous recombination. To build the IdhL integration targeting vector, a DNA fragment from Lactobacillus plantarum (Genbank NC_004567) with homology to IdhL is PCR amplified with primers LDH EcoRV F (SEQ ID N0:198) and LDH AatllR (SEQ ID N0:199). The 1986 bp PCR fragment is cloned into pCR4Blunt-TOPO and sequenced. The pCR4Blunt-TOPO-ldhl_1 clone is digested with EcoRV and Aatll releasing a 1982 bp ldhi_1 fragment that is gel-purified. The integration vector pFP988, described in Example 14, is digested with Hindlll and treated with Klenow DNA polymerase to blunt the ends. The linearized plasmid is then digested with Aatll and-the 2931 bp vector fragment is gel-purified. The EcoRV/Aatll IdhU fragment is ligated with the pFP988 vector fragment and transformed into E. co//Top10 cells. Transformants are selected on LB agar plates containing ampiciliin (100 [iQ/ml) and are screened by colony PCR to confirm construction of pFP988-ldhL<br>
To add a selectable marker to the integrating DNA, the Cm gene with its promoter is PCR amplified from pC194 (Genbank NC_002013) with primers Cm F (SEQ ID N0:200) and Cm R (SEQ ID NO:201), amplifying a 836 bp PCR product. The amplicon is cloned into pCR4Blunt-TOPO and transformed into £. co//Top10 cells, creating pCR4BIunt-TOPO-Cm. After sequencing to confirm that no errors are introduced by PCR, the Cm cassette is digested from pCR4Blunt-TOPO-Cm as an 828 bp Mlul/Swal fragment and is gel-purified.  The IdhL-homology containing integration vector pFP988-ldhL is digested with Mlul and Swal and the 4740 bp vector fragment is gel-purified. The Cm cassette fragment is ligated with the pFP988-ldhL vector creating pFP988-DIdhl_::Cm.<br>
Finally the thl-hbd-crt cassette from pFP988Dss-T-H-C, described in Example 14, is modified to replace the amylase promoter with the synthetic P11 promoter. Then, the whole operon is moved into pFP988-DldhL::Cm. The P11 promoter is built by oligonucleotide annealing with<br>
N0:202) and P11 R (SEQ ID N0:203). The annealed oligonucleotide is gel-purified on a 6% Ultra PAGE gel (Embi Tec, San Diego, CA). The plasmid pFP988Dss-T-H-C is digested with Xhol and Smal and the 9 kbp vector fragment is gel-purified. The isolated P11 fragment is ligated with the digested pFP988Dss-T-H-C to create pFP988-P11-T-H-C. Plasmid pFP988-P11-T-H-C is digested with Xhol and BamHI and the 3034 bp P11-T-H-C fragment is gel-purified. pFP988-DldhL::Cm is digested with Xhol and BamHI and the 5558 bp vector fragment isolated. The upper pathway operon is ligated with the integration vector to create pFP988-DldhL-P11-THC::Cm.<br>
Integration of pFP988-Dldhl-Pf 1-THC::Cm into L. olantarum BAA-793 to form L. olantarum AldhH ::T-H-C::Cm comprising exogenous thl, hbd, and crt genes. Electrocompetent cells of L. plantarum are prepared as described by Aukrust, T.W., et al. (In: Electroporation Protocols for Microorganisms; Nickoloff, J.A., Ed.; Methods in Molecular Biology, Vol. 47; Humana Press, Inc., Totowa, NJ, 1995, pp 201-208). After electroporation, cells are outgrown in MRSSM medium (MRS medium<br>
supplemented with 0.5 M sucrose and 0.1 M MgC^) as described by<br>
Aukrust et al. supra for 2 h at 37 °C without shaking. Electroporated cells are plated for selection on MRS plates containing chloramphenicol (10 fjglml) and incubated at 37 °C. Transformants are initially screened by colony PCR amplification to confirm integration, and initial positive clones are then more rigorously screened by PCR amplification with a battery of primers.<br>
Plasmid Expression of EgTER. aid, and bdhB genes. The three remaining 1-butanol genes are expressed from plasmid pTRKHS (O'Sullivan DJ and Klaenhammer TR, Gene 137:227-231 (1993)) under the control of the L. plantarum IdhL promoter (Ferain et al., J. Bacteriol. 176:596-601 (1994)). The IdhL promoter is PCR amplified from the genome of L. plantarum ATCC BAA-793 with primers PldhL F (SEQ ID N0:204) and PldhL R (SEQ ID NO:205). The 369 bp PCR product is cloned into pCR4Blunt-TOPO and sequenced. The resulting plasmid,<br>
is digested with Sacl and BamHI releasing the 359 bp PldhL fragment.<br>
pHT01-ald-EB, described in Example 14, is digested with Sacl and BamHI and the 10503 bp vector fragment is recovered by gel purification. The PldhL fragment and vector are ligated creating pHT01-Pldhl-ald-EB.<br>
To subclone the IdhL promoter-ald-EgTER-bdh cassette, pHT01-   . Pldhl-ald-EB is digested with Mlul and the ends are treated with Klenow DNA polymerase. The linearized vector is digested with Sail and the 4270 bp fragment containing the PldhL-AEB fragment is gel-purified. Plasmid pTRKHS is digested with Sail and EcoRV and the gel-purified vector fragment is ligated with the PldhL-AEB fragment. The ligation mixture is transformed into £. co//Top 10 cells and transformants are plated on Brain Heart Infusion (BHI, Difco Laboratories, Detroit, Ml) plates containing erythromycin (150 mg/L). Transformants are screened by PCR to 9onfirm construction of pTRKH3-ald-E-B. The expression plasmid, pTRKH3-ald-E-B is transformed into L. plantarum AldhLI ::T-H-C::Cm by electroporation, as described above.<br>
L. plantarum AldhL1::T-H-C::Cm containing pTRKH3-ald-E-B is inoculated into a 250 mL shake flask containing 50 mL of MRS medium plus erythromycin (10 //g/mL) and grown at 37 °C for 18 to .24 h without shaking. After 18 h to 24,1-butanol is detected by HPLC or GC analysis, as described in the General Methods section.<br>
EXAMPLE 19 (Prophetic)<br>
Expression of the 1-Butanol Biosvnthetic Pathway in Enterococcus<br>
faecalis<br>
The purpose of this prophetic Example is to describe how to express the 1-butanol biosynthetic pathway in Enterococcus faecalis. The complete genome sequence of Enterococcus faecalis strain V583, which is used as the host strain for the expression of the 1-butanol biosynthetic pathway in this Example, has been published (Paulsen et al., Science 299:2071-2074 (2003)). Plasmid pTRKHS (O'Suilivan DJ and Klaenhammer TR, Gene 137:227-231 (1993)), an E. co///Gram-positive shuttle vector, is used for expression of the six genes (thIA, hbd, crt,<br>
e 1-butanol pathway in one operon. pTRKHS contains an £ coll plasmid p15A replication origin and the pAM|31 replicon, and two antibiotic resistance selection markers, tetracycline resistance and erythromycin resistance. Tetracycline resistance is only<br>
expressed in E. coli, and erythromycin resistance is expressed in both E. co//and Gram-positive bacteria. Plasmid pAMpI derivatives can replicate in E. faecalis (Poyart et al., FEMS Microbiol. Lett. 156:193-198 (1997)). The inducible nisA promoter (PnisA), which has been used for efficient control of gene expression by nisin in a variety of Gram-positive bacteria including Enterococcus faecalis (Eichenbaum et al., Appl. Environ. Microbiol. 64:2763-2769 (1998)), is used to control expression of the six desired genes encoding the enzymes of the 1-butanol biosynthetic pathway.<br>
The linear DNA fragment (215 bp) containing the nisA promoter (Chandrapati et al., Mol. Microbiol. 46(2):467-477 (2002)) is PCR-amplified from Lactococcus lactis genomic DNA with primers F-PnisA(EcoRV) (SEQ ID N0:206) and R-PnisA(Pmel BamHI) (SEQ ID NO:207). The 215 bp PCR fragment is digested with EcoRV and BamHI, and the resulting PnisA fragment is gel-purified. Plasmid pTRKHS is digested with EcoRV and BamHI and the vector fragment is gel-purified. The linearised pTRKHS is ligated with the PnisA fragment. The ligation mixture is transformed into E. co//Top10 cells by electroporation and transformants are selected following overnight growth at 37 °C on LB agar plates containing erythromycin (25 |ig/mL). The transformants are then screened by colony PCR with primers F-PnisA(EcoRV) and R-PnisA(BamHI) to confirm the correct clone of pTRKHS-PnisA.<br>
Plasmid pTRKHS-PnisA is digested with Pmel and BamHI, and the vector is gel-purified. Plasmid pHT01-a/d-EgTEf?-jbc(hB is constructed as described in Example 14 and is digested with Smal and BamHI, and the 2,973 bp ald-EgTER-bdhB fragment is gel-purified. The 2,973 bp a/cf-EgTER-bdhB fragment is ligated into the pTRKHS-PnisA vector at the Pmel and BamHI sites. The ligation mixture is transformed into E. coli TopIO cells by electroporation and transformants are selected following<br>
overnight on LB agar plates containing erythromycin (25 µg/mL). The transformants are then screened by colony PCR with primers a/of forward primer N27F1 (SEQ ID NO: 31) and bdhB reverse primer N65 (SEQ ID NO: 44). The resulting plasmid is named pTRKHS-PnisA-ald-EgTER-bdhB (= pTRKH3-A-E-B).<br>
Plasmid pTRKH3-A-E-B is purified from the trahsformant and used for further cloning of the remaining genes (thIA, hbd, erf) into the BamHl site located downstream of the bdhB gene. Plasmid pTRKH3-A-E-B is digested with BamHl and treated with the Klenow fragment of DNA polymerase to make blunt ends. Plasmid pFP988Dss-th!A-hbd.-crt (=pFP988Dss-T-H-C) is constructed as described in Example 14 and is digested with Smal and BamHl. The resulting 2,973 bp thIA-hbd-crt fragment is treated with the Klenow fragment of DNA polymerase to make blunt ends and is gel-purified. The 2,973 bp thIA-hbd-crt fragment is ligated with the linearised pTRKH3-A-E-B. The ligation mixture is transformed into E. co//Top10 cells by electroporation and transformants are selected following overnight growth at 37 °C on LB agar plates containing erythromycin (25 µg/mL). The transformants are then screened by colony PCR with primers thIA forward primer N7 (SEQ ID NO: 21) and crt reverse primer N4 (SEQ ID NO: 18). The resulting plasmid is named pTRKH3-PnisA-ald-EgTER-bdhB-thlA-hbd-crt (= pTRKH3-A-E-B-T-H-C). Plasmid pTRKH3-A-E-B-T-H-C is prepared from the E coll transformants and transformed into electro-competent E. faecalis V583 cells by electroporation using methods known in the art (Aukrust, T.W., et al. In: Electroporation Protocols for Microorganisms; Nickoloff, J.A., Ed.; Methods in Molecular Biology, Vol. 47; Humana Press, Inc., Totowa, NJ., 1995, pp 217-226), resulting in E. faecalis V583/pTRKH3-A-E-B-T-H-C. The second plasmid containing nisA regulatory genes, nisR and nisK, the acfd9 spectinomycin resistance gene, and the pSH71 origin of replication is transformed into £. faecalis V583/pTRKH3-A-E-B-T-H-C by electroporation. The plasmid containing pSH71 origin of replication is compatible with pAMß1 derivatives in E. faecalis (Eichenbaum et al., supra). Double drug resistant transformants are selected on LB agar in (25 µg/mL) and spectinomycin (100<br>
The resulting E faecalis strain V583S harboring two plasmids, i.e., an expression plasmid (pTRKH3-A-E-B-T-H-C) and a regulatory plasmid (pSH71-nisRK), is inoculated into a 250 ml shake flask containing 50 mL of Todd-Hewitt broth supplemented with yeast" extract (0.2%) (Fischetti et al., J. Exp. Med. 161:1384-1401 (1985)), nisin (20 µg/mL) (Eichenbaum et al., supra), erythromycin (25 µg/mL), and spectinomycin (100 µg/mL). The flask is incubated without shaking at 37 °C for 1 8 to 24 h, after which time, 1-butanol production is measured by HPLC or GC analysis, as described in the General Methods section.<br><br><br><br><br><br>
We claim:<br>
1.	A recombinant microbial host cell comprising at least one DNA<br>
molecule encoding a polypeptide that catalyzes a substrate to product<br>
conversion selected from the group consisting of:<br>
a)	acetyl-CoA to acetoacetyl-CoA<br>
b)	acetoacetyl-CoA to 3-hydroxybutyryl-CoA<br>
c)	3-hydroxybutyryl-CoA to crotonyl-CoA<br>
d)	crotonyl-CoA to butyryl-CoA<br>
e)	butyryl-CoA to butyraldehyde and<br>
f)	butyraldehyde tol-butanol;<br>
wherein the at least one DNA molecule is heterologous to said microbial host cell and wherein said microbial host cell produces 1-butanol.<br>
2.	A host cell according to claim 1 wherein:<br>
a)	the polypeptide that catalyzes a substrate to product conversion<br>
of acetyl-CoA to acetoacetyl-CoA is acetyl-CoA acetyltransferase;<br>
b)	the polypeptide that catalyzes a substrate to product conversion<br>
of acetoacetyl-CoA to 3-hydroxybutyryl-CoA is 3-hydroxybutyryl-<br>
CoA dehydrogenase;<br>
c)	the polypeptide that catalyzes a substrate to product conversion<br>
of 3-hydroxybutyryl-CoA to crotonyl-CoA is crotonase;<br>
d)	the polypeptide that catalyzes a substrate to product conversion<br>
of crotonyl-CoA to butyryl-CoA is butyryl-CoA dehydrogenase;<br>
e)	the polypeptide that catalyzes a substrate to product conversion<br>
of butyryl-CoA to butyraldehyde is butyraldehyde dehydrogenase;<br>
and<br>
f)	the polypeptide that catalyzes a substrate to product conversion<br>
of butyraldehyde to 1-butanol is butanol dehydrogenase.<br>
3.	A host cell according to claim 1 wherein the cell is selected from<br>
the group consisting of: a bacterium, a cyanobacterium, a filamentous<br>
fungus and a yeast.<br>
4.	A host cell according to claim 3 wherein the cell is a member of a<br>
genus selected from the group consisting of Clostridium, Zymomonas, Escherichia, Salmonella, Rhodococcus, Pseudomonas, Bacillus, Lactobacillus, Bnterococcus, Alcaligenes, Klebsiella, Paenibacillus, Arthrobacter, Corynebacterium, Brevibacterium, Pichia, Candida, Hansenula and Saccharomyces.<br>
5.       A host cell according to claim 1 wherein:<br>
a)	the acetyl-CoA acetyltransferase has an amino acid sequence<br>
selected from the group consisting of SEQ ID NO:2, SEQ ID NO:4,<br>
SEQ ID NO: 129, SEQ ID NO: 131, and SEQ ID NO: 133;<br>
b)	the 3-hydroxybutyryl-CoA dehydrogenase has an amino acid<br>
sequence selected from the group consisting of SEQ ID NO:6, SEQ<br>
ID NO: 135, SEQ ID NO: 137, and SEQ ID NO: 139;<br>
c)	the crotonase has an amino acid sequence selected from the<br>
group consisting of SEQ ID NO:8, SEQ ID NO: 141, SEQ ID<br>
NO: 143, and SEQ ID NO: 145;<br>
d)	the butyryl-CoA dehydrogenase has an amino acid sequence<br>
selected from the group consisting of SEQ ID NO: 10, SEQ ID<br>
NO: 147, SEQ ID NO: 149, SEQ ID NO: 151, and SEQ ID NO: 187;<br>
e)	the butyraldehyde dehydrogenase has an amino acid sequence<br>
selected from the group consisting of SEQ ID NO: 12, SEQ ID<br>
NO: 153, and SEQ ID NO: 189; and<br>
f)	the butanol dehydrogenase has an amino acid sequence selected<br>
from the group consisting of SEQ ID NO: 14,  SEQ ID NO: 16, SEQ<br>
ID NO: 153, SEQ ID NO: 155, and SEQ ID NO: 157.<br>
6.	A host cell according to claim 1 wherein the host cell is a<br>
facultative anaerobe.<br>
7.	A method for the production of 1-butanol comprising:<br>
i) providing a recombinant microbial host cell comprising at least one DNA molecule encoding a polypeptide that catalyzes a substrate to product conversion selected from the group consisting of:<br>
a)	acetyl-CoA to acetoacetyl-CoA<br>
b)	acetoacetyl-CoA to 3-hydroxybutyryl-CoA<br>
c)	3-hydroxybutyryl-CoA to crotonyl-CoA<br>
d)	crotonyl-CoA to butyryl-CoA<br>
e)	butyryl-CoA to butyraldehyde and<br>
f)	butyraldehyde to 1-butanol;<br>
wherein the at least one DNA molecule is heterologous to  said microbial<br>
host cell ; and<br>
ii) contacting the host cell of (i) with a fermentable carbon substrate under conditions whereby 1-butanol is produced.<br><br>
8.	A method according to claim wherein the fermentable carbon<br>
substrate is selected from the group consisting of monosaccharides,<br>
oligosaccharides, and polysaccharides.<br>
9.       A method according to claim  wherein the conditions whereby 1<br>
butanol is produced are anaerobic.<br>
10.     A method according to claim 27 wherein the conditions whereby 1 butanol is produced are microaerobic.<br>
11.	A method according to claim 7 wherein the host cell is contacted<br>
with the carbon substrate in minimal media.<br>
12.	A method according to claim 7 wherein<br><br>
a)	the polypeptide that catalyzes a substrate to product conversion<br>
of acetyl-CoA to acetoacetyl-CoA is acetyl-CoA acetyltransferase;<br>
b)	the polypeptide that catalyzes a substrate to product conversion<br>
of acetoacetyl-CoA to 3-hydroxybutyryl-CoA is 3-hydroxybutyryl-<br>
CoA dehydrogenase;<br>
c)	the polypeptide that catalyzes a substrate to product conversion<br>
of 3-hydroxybutyryl-CoA to crotonyl-CoA is crotonase;<br>
d)	the polypeptide that catalyzes a substrate to product conversion<br>
of crotonyl-CoA to butyryl-CoA is butyryl-CoA dehydrogenase;<br>
e)	the polypeptide that catalyzes a substrate to product conversion<br>
of butyryl-CoA to butyraldehyde is butyraldehyde dehydrogenase;<br>
f)	the polypeptide that catalyzes a substrate to product conversion<br>
of butyraldehyde to 1-butanol is butanol dehydrogenase.<br><br>
13.	A method according to claim7 wherein the host cell is selected<br>
from the group consisting of: a bacterium, a cyanobacterium, a<br>
filamentous fungus and a yeast.<br>
14.	A method according to claim 13 wherein the host cell is a member<br>
of a genus selected from the group consisting of Clostridium, Zymomonas,<br>
Escherichia, Salmonella, Rhodococcus, Pseudomonas, Bacillus,<br>
Lactobacillus, Enterococcus, Alcaligenes, Klebsiella, Paenibacillus,<br>
Arthrobacter, Corynebacterium, Brevibacterium, Pichia, Candida,<br>
Hansenula and Saccharomyces.<br>
15.	A method according to claim 12 wherein<br>
a)	the acetyl-CoA acetyltransferase has an amino acid sequence<br>
selected from the group consisting of SEQ ID NO:2, SEQ ID NO:4,<br>
SEQ ID NO: 129, SEQ ID NO: 131, and SEQ ID NO: 133<br>
b)	the 3-hydroxybutyryl-CoA dehydrogenase has an amino acid<br>
sequence selected from the group consisting of SEQ ID NO:6, SEQ<br>
ID NO: 135, SEQ ID NO: 137, and SEQ ID NO: 139;<br>
c)	the crotonase has an amino acid sequence selected from the<br>
group consisting of SEQ ID NO:8, SEQ ID NO: 141, SEQ ID<br>
NO: 143, and SEQ ID NO: 145;<br>
d)	the butyryl-CoA dehydrogenase has an amino acid sequence<br>
selected from the group consisting of SEQ ID NO: 10, SEQ ID<br>
NO: 147, SEQ ID NO: 149, SEQ ID NO: 151, and SEQ ID NO: 187;<br>
e)	the butyraldehyde dehydrogenase has an amino acid sequence<br>
selected from the group consisting of SEQ ID NO: 12, SEQ ID<br>
NO: 153, and SEQ ID NO: 189;  and<br>
f)	the butanol dehydrogenase has an amino acid sequence selected<br>
from the group consisting of SEQ ID NO: 14,  SEQ ID NO: 16, SEQ<br>
ID NO: 153, SEQ ID NO: 155, and SEQ ID NO: 157.<br>
16.     A method according to Claim 27 wherein the host cell is a facultative anaerobe.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDAwOC0xLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDYtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1812-DELNP-20008-1-Correspondence Others-(06-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LTItQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNS0wNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-2-Correspondence Others-(15-04-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LTItRm9ybS0xLSgxNS0wNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-2-Form-1-(15-04-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUFic3RyYWN0LSgwNC0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Abstract-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LUFzc2lnbm1lbnQtKDIwLTA0LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-Assignment-(20-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUNsYWltcy0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Claims-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Correspondence Others-(09-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-Correspondence Others-(24-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Correspondence-Others-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjAtMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-Correspondence-Others-(20-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LUZvcm0tMTgtKDI0LTA2LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-Form-18-(24-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUZvcm0tMi0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Form-2-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUZvcm0tMy0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Form-3-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LUdQQS0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-GPA-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LUdQQS0oMjAtMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-GPA-(20-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LVBDVC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-PCT-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1ERUxOUC0yMDA4LVBDVC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1812-DELNP-2008-PCT-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LVBldGl0aW9uLTEzNy0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Petition-137-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxMi1kZWxucC0yMDA4LVBldGl0aW9uLTEzOC0oMDQtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1812-delnp-2008-Petition-138-(04-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="258413-systems-and-methods-for-discovering-and-interacting-with-services.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258415-delivery-system-for-low-calorie-bulking-agents.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258414</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1812/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Feb-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BUTAMAX ADVANCED BIOFUELS LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>200 POWDER MILL ROAD, WILMINGTON, DELAWARE 19803, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GAIL K. DONALDSON,</td>
											<td>3 GREAT CIRCLE ROAD, NEWARK, DE 19711, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LIXUAN LISA HUANG,</td>
											<td>1423 OLD WILMINGTON ROAD, HOCKESSIN, DE 19707, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LORI ANN MAGGIO HALL,</td>
											<td>3413 NORTH ROCKFIELD DRIVE, WILMINGTON, DE 19801, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VASANTHA NAGARAJAN,</td>
											<td>702 HOPETON ROAD, WILMINGTON, DE 19807, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>CHARLES E. NAKAMURA,</td>
											<td>2 MOUNT VERNON DRIVE, CLAYMONT, DE 19703, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>WONCHUL SUH,</td>
											<td>8 SALINA COURT, PIERSONS RIDGE, HOCKESSIN, DE 19707, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 7/16</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/038007</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-09-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/814,470</td>
									<td>2006-06-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/721,677</td>
									<td>2005-09-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258414-microbial-host-cells-for-the-fermentive-production-of-1-butanol by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:39:57 GMT -->
</html>
