<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258470-methods-for-controlling-pests-using-rnai by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:28:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258470:METHODS FOR CONTROLLING PESTS USING RNAI</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS FOR CONTROLLING PESTS USING RNAI</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for controlling pest infestation using double stranded RNA molecules. The invention provides methods for producing transgenic cells expressing the double stranded RNA molecules, as well as compositions and commodity products containing or treated with such molecules.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>. <br><br>
FIELD OF THE INVENTION<br>
The present invention relates generally to genetic control of pest infestations. More specifically, the present invention relates to recombinant technologies for repressing or inhibiting expression of target coding sequences in a pest.<br>
INTRODUCTION<br>
Insect and other pests can cause injury and even death by their bites or stings. Additionally, many pests transmit bacteria and other pathogens that cause diseases. I'or example, mosquitoes transmit pathogens that cause malaria, yellow fever, encephalitis, and other diseases. The bubonic plague, or black death, is caused by bacteria that infect rats and other rodents. Compositions for controlling microscopic pest infestations have been provided in the form of antibiotic, antiviral, and antifungal compositions. Methods for controlling infestations by pests, such as nematodes and insects, have typically been in the form of chemical compositions that are applied to surfaces on which pests reside, or administered to infested animals in the form of pellets, powders, tablets, pastes, or capsules.<br>
Commercial crops are often the targets of insect attack. Substantial progress has been made in the last a few decades towards developing more efficient methods and compositions for controlling insect infestations in plants. Chemical pesticides have been very effective in eradicating pest infestations. However, there are several disadvantages to using chemical pesticides. Not only are they potentially detrimental to the environment, but chemical pesticides are not selective and can pose harm to non-target flora and fauna. Chemical pesticides persist in the environment and generally are slow to be metabolized, if at all. they accumulate in the food chain, and particularly in the higher predator species. Accumulation of chemical pesticides results in the development of resistance to the agents and in species higher up the evolutionary ladder, they can act as mutagens and/or carcinogens and cause irreversible and deleterious genetic modifications.<br>
Because of the dangers associated with chemical pesticides, biological approaches have been developed for controlling pest infestations, l-or example, biological control using protein Cry3A from Bacillus thuringiensls  have effectively controlled Colorado potato beetle larvae either as formulations sprayed onto the foliage or expressed in the leaves of potatoes. An alternative biological agent is double stranded RNA (dsRNA). Over the last few years, downregulation of genes (also referred to as "gene silencing') in multicellular organisms by means of RNA interference has become a well-established technique.<br><br>
RNA Interference (RNAi) provides a potentially powerful tool for controlling gene expression because of its specificity of target selection and remarkably high efficiency in target mRNA suppression, RNAi refers to the process of sequence-specific post-transcriptional gene silencing mediated by short interfering RNAs (siRNAs) (/amorc. P. et al.. Cell 101:25-33 (2000); Fire, A. et al. Nature 391:806 (1998); Hamilton et a!.. Science 286, 950-951 (1999); Lin et al., Nature 402:128-129 (1999)). While the mechanics underlying RNAi arc not fully characterized, it is thought that the presence of dsRNA in cells triggers RNAi by activating the ribonuclcasc III en/yme Dicer ( Zamore, P. et al., (2000); Hammond et al., Nature 404, 293 (2000)). Dicer processes the dsRNA into short pieces called short interfering RNAs (siRNAs), which are about 21 to about 23 nucleotides long and comprise about 19 base pair duplexes (Zamore et al., (2000); Hlbashir et al.. Genes Dev., 15, 188 (2001)). Following delivery into cells, the siRNA molecules associate with an cndonucleasc complex, commonly referred to as an RNA-induced silencing complex (RISC), which brings together the antisense strand of the siRNA and the cellular mRNA gene target. RISC cleaves the mRNA, which is then released and degraded. Importantly, RISC is then capable of degrading additional copies of the target mRNA.<br>
Accordingly, the present invention provides methods and compositions for controlling pest infestation by repressing, delaying, or otherwise reducing gene expression within a particular pest.<br>
SUMMARY OF THE INVENTION<br>
In one aspect, the invention provides an isolated polynucleotide sequence comprising a nucleic acid sequence set forth in SFQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19,21,23,49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 157K 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120  2338, 2339,<br><br>
2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 - 2460, 2461, 2466, 2471, 2476 and 2481. In one embodiment, a double stranded ribonucleotide sequence is produced from the expression of a polynucleotide sequence, wherein contact of said ribonucleotide sequence by a pest inhibits the growth of said pest. In a further embodiment, contact of the sequence inhibits expression of a nucleotide sequence substantially complementary to said sequence. In another embodiment, a cell is transformed with the polynucleotide. In a further embodiment, the cell is a bacterial, yeast, or algal cell. In a still further embodiment, a food product, such as stored grains, pet food, or powdered chocolate, comprises the cell transformed with the polynucleotide, in yet another embodiment, a composition , such as a spray, powder, pellet, gel. capsule, food product, garment bag, and book, comprising the polynucleotide. In yet another embodiment, the invention provides a pesticide comprising the polynucleotide. In another embodiment, the invention provides a method for protecting an object, such as wood, tree, book binding, cloth, and a food storage container, from pest infestation, comprising treating said surface with a composition comprising the polynucleotide.<br>
In another aspect, the invention provides a polynucleotide sequence having at least 70% sequence identity to a nucleic acid sequence set forth in SEQ ID NOs: I, 3, 5, 7, 9, II, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 2! 5, 220. 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, nil, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617. 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730-2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481. In one embodiment, a double stranded ribonucleotide sequence is produced from the expression of a polynucleotide sequence, wherein contact of said ribonucleotide sequence by a pest inhibits the growth of said pest. In a further embodiment, contact of the sequence inhibits expression of a nucleotide sequence substantially complementary to said sequence. In another embodiment, a cell is transformed with the polynucleotide.  In a further embodiment, the cell is a bacterial, yeast, or algal cell.<br><br>
In a still further embodiment, a food product, such as stored grains, pet food, or powdered chocolate, comprises the cell transformed with the polynucleotide. In yet another embodiment, a composition , such as a spray, powder, pellet, gel, capsule, food product, garment bag, and book, comprising the polynucleotide. In yet another embodiment, the invention provides a pesticide comprising the polynucleotide. In another embodiment, the invention provides a method for protecting an object, such as wood, tree, book binding, cloth, and a food storage container, from pest infestation, comprising treating said surface with a composition comprising the polynucleotide.<br>
In another aspect, the invention provides a method for controlling pest infestation, comprising exposing a pest to a composition comprising a polynucleotide sequence that inhibits a pest biological activity. In one embodiment, the polynucleotide sequence is set forth in any of SEQ ID NOs: 1,3,5,7,9, 11, 13, 15, 17, 19,21,23,49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 - 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090. 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384    2460, 2461, 2466, 2471, 2476 and 2481.<br>
In other embodiments, the invention provides for the use of the isolated nucleotide sequence, the double stranded ribonucleotide sequence, the cell, the composition, or the pesticide for preventing or treating an infestation, such as insect, nematode, or fungal infestation.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1-LD; Survival of L. decemlineata on artificial diet treated with dsRNA, Insects of the second larval stage were fed diet treated with 50 µl of topically-applied solution of dsRNA (targets or gfp control). Diet was replaced with fresh diet containing topically-<br><br>
applied dsRNA after 7 days. The number of surviving insects were assessed at days 2, 5, 7, 8, 9, &amp; 13. the percentage of surviving larvae was calculated relative to day 0 (start of assay). Target LD006: (SliQ ID NO: 178); Target LD007 (SEQ ID NO: 183); Target LDOIO (SHQ ID NO: 188); Target LD011 (S1:Q ID NO: 193); Target 1.D014 (SHQ ID NO: 198); gfp dsRNA(SEQ lDNO:235).<br>
Figure 2-LI): Survival of L. decemlineata on artificial diet treated with dsRNA. Insects of the second larval stage were fed diet treated with 50 \i\ of topically-applied solution of dsRNA (targets or gfp control). Diet was replaced with fresh diet only after 7 days. The number of surviving insects was assessed at days 2, 5, 6, 7, 8, 9, 12, &amp; 14. The percentage of surviving larvae was calculated relative to day 0 (start of assay). Target LDOOl (SHQ ID NO: 163); Target HD002 (SHQ ID NO: 168); Target HD003 (SHQ ID NO: 173); Target HD015 (S1:Q ID NO: 215); Target HD016 (SHQ ID NO: 220); gfp dsRNA (SHQ ID NO: 235).<br>
Figure 3-LD: Average weight of A. decemlineata larvae on potato leaf discs treated with dsRNA. Insects of the second larval stage were fed leaf discs treated with 20 \x\ of a topically-applied solution (10 ng/|il) of dsRNA (target HD002 or gfp). After two days the insects were transferred on to untreated leaves every day.<br>
Figure 4-Ll): Survival of L. decemlineata on artificial diet treated with shorter versions of target HD014 dsRNA and concatemcr dsRNA. Insects of the second larval stage were fed diet treated with 50 \x\ of topically-applied solution of dsRNA (gfp or targets). Ihe number ol'surviving insects were assessed at days 3, 4, 5, 6, &amp; 7. The percentage of surviving larvae were calculated relative to day 0 (start of assay).<br>
Figure 5-Ll); Survival of L, decemlineata larvae on artificial diet treated with different concentrations of dsRNA of target LD002 (a), target HD007 (b), target HDOIO (c), target HDOll (d), target LD014 (e), target LD015 (t), LD016 (g) and target HD027 (h). Insects of the second larval stage were fed diet treated with 50 |iil of topically-applied solution of dsRNA. Diet was replaced with fresh diet containing topically-applied dsRNA after 7 days. Ihe number of surviving insects were assessed at regular intervals. The percentage of surviving larvae were calculated relative to day 0 (start of assay).<br>
Figure 6-LO, ITTects of E . coli strains expressing dsFWA target HDOIO on survival of larvae of the (Colorado potato beetle, Leptinotarsa decemlineata, over time. The two bacterial strains were tested in separate artificial diet-based bioassays: (a) AB309-I05; data points  for pGBNJ003  and pGN29 represent average mortality values  from  5  different<br><br>
bacterial clones, (b) BI.2](DH3); data points for pGBNJ003 and pGN29 represent average mortality values from 5 different and one single bacterial clones, respectively. FEror bars represent standard deviations.<br>
Figure 7-LD, Effects of different clones of E; coli strains (a) AB309-I05 and (b) BL21(Df;3) expressing dsRNA target LD010 on survival of larvae of the Colorado potato beetle, Leptinotarsa decemlineata. 12 days post infestation. Data points are average mortality values for each clone for pGN29 and pGBNJ003. Clone 1 of AB309-I05 harbouring plasmid pCiBNJ()03 showed 100% mortality towards CPB at this timepoint. Krror bars represent standard deviations.<br>
Figure 8-LD. Effects of different clones of E. coli strains (a) AB309-I05 and (b) BI2l(DE3) expressing dsRNA target I.DOIO on growth and development of larval survivors of the Colorado potato beetle, Lepiinotarsa decemlimaia, 1 days post infestation, Data points are % average larval weight values for each clone (one clone for pGN29 and five eloncs for pGBNJ003) based on the data of fable 10. Diet only treatment represents 100% normal larval weight.<br>
Figure 9-LI). Survival of larvae of the Colorado potato beetle, Leptinotarsa decemlineata, on potato plants sprayed by double-stranded RNA-producing bacteria 7 days post infestation. Number of larval survivors were counted and expressed in terms of % mortality. I he bacterial host strain used was the RNaselll-deficicnt strain AB309-105. Insect gene target was LD010.<br>
Figure 10-LD. Growth/developmental delay of larval survivors of the Colorado potato beetle, Leptinotarsa decemlineata, fed on potato plants sprayed with dsRNA-producing bacteria 11 days post infestation. The bacterial host strain used was the RNaselll-dellcient strain Ati309-105. Data figures represented as percentage of normal larval weight; 100 % of normal larval weight given for diet only treatment. Insect gene target was LDOIO. l{rror bars represent standard deviations.<br>
Figure 11-IJ). Resistance to potato damage caused by larvae of the Colorado potato beetle, Leptinotarsa dceemlineata, by double-stranded RNA-producing bacteria 7 days post infestation. Left, plant sprayed with 7 units of bacteria AB309-I05 containing the pGN29 plasmid; right plant sprayed with 7 units of bacteria Ab309-105 containing the pGBNJ003 plasmid. One unit is defined as the equivalent of I ml of a bacterial suspension at OD value of I at 600 nm. Insect gene target was LDOIO.<br><br>
Figure 12-Ll). Survival of L decemlineata adults on potato leaf discs treated with dsRNA. Young adult insects were fed double-stranded-RNA-trcated leaf discs for the first two days and were then placed on untreated potato foliage. The number of surviving insects were assessed regularly; mobile insects were recorded as insects which were alive and appeared to move normally; moribund insects were recorded as insects which were alive but appeared sick and slow moving these insects were not able to right themselves once placed on their backs, larget LD002 (SEQ ID NO: 168); Target I.D010 (SEQ ID NO: 188); Target LD014 (SHQ ID NO: 198); larget LD016 (SHQ ID NO: 220); gfp dsRNA (SEQ ID NO: 235).<br>
Figure 13-LD. Hffccts of bacterial produced target double-stranded RNA against larvae of A. decemlineata. Fifty µL of an OD 1 suspension of heat-treated bacteria expressing dsRNA (SliQ ID NO: 188) was applied topically onto the solid artificial diet in each well of a 48-well plate. CPB larvae at L2 stage were placed in each well. At day 7, a picture was taken of the CPB larvae in a plate containing (a) diet with bacteria expressing target 10 double-stranded RNA, (b) diet with bacteria harbouring the empty vector pGN29, and, (c) diet only.<br>
Figure I4-IJ) Hffects on CPB larval survival and growth of different amounts of inactivated E. coli AB309-105 strain harbouring plasmid pGBNJ003 topically applied to potato foliage prior to insect infestation, fen I.l larvae were fed treated potato for 7 days. Amount of bacterial suspension sprayed on plants: 0.25 U, 0.08 U, 0.025 U, 0.008 U of target 10 and 0.25 IJ of pGN29 (negative control; also included is Milii-Q water). One unit (U) is delined as the equivalent bacterial amount present in I ml of culture with an optical density value of I at 600nm. A total volume of 1.6 ml was sprayed on to each plant. Insect gene target was LD010.<br>
Figure 15-LD Resistance to potato damage caused by CPB larvae by inactivated E. coli AB309-I05 strain harbouring plasmid pGBNJ003 seven days post infestation, (a) water, (b) 0.25 U E. coli AB309-105 harbouring pGN29, (c) 0.025 U E. coli AB309-105 harbouring pGBNJ003, (d) 0.008 U E. coli AB309-105 harbouring pGBNJ003. One unit (U) is defined as the equivalent bacterial amount present in 1 ml of culture with an optical density value of 1 at 600nm. A total volume of 1.6 ml was sprayed on to each plant. Insect gene target was LD010.<br>
Figure 1-PC: Effects of ingested target dsRNAs on survival and growth of P. cochteariae larvae. Neonate larvae were fed oilseed rape leaf discs treated with 25 µlof<br><br>
topically-applied solution of 0.1 µ/il dsRNA (targets or gfp control). Afer 2 days, the insects were transferred onto fresh dsRNA-treated leaf discs. At day 4, larvae from one replicate for every treatment were collected and placed in a Petri dish containing fresh untreated oilseed rape foliage, The insects were assessed at days 2, 4, 7, 9 &amp; 11. (a) Survival of K. varivestis larvae on oilseed rape leaf discs treated with dsRNA. The percentage of surviving larvae was calculated relative to day 0 (start of assay), (b) Average weights of P. cochleariae larvae on oilseed rape leaf discs treated with dsRNA. Insects from each replicate were weighed together and the average weight per larva determined. Error bars represent standard deviations. Target 1: SHQ ID NO: 473; target 3: SEQ ID NO: 478; target 5: SHQ ID NO: 483 -; target 10: SHQ ID NO: 488; target 14: SEQ ID NO: 493; target 16: SEQ ID NO: 498; target 27: SEQ ID NO: 503; gfp dsRNA: SEQ ID NO: 235.<br>
Figure 2-PC: Survival of I\ cochleariae on oilseed rape leaf discs treated with different concentrations of dsRNA of (a) target PC010 and (b) target PC027. Neonate larvae were placed on leaf discs treated with 25 µL of topically-applied solution of dsRNA. Insects were transferred to fresh treated leaf discs at day 2. At day 4 for target PC010 and day 5 for target PC027, the insects were transferred to untreated leaves. The number of surviving insects were assessed at days 2, 4, 7, 8, 9 &amp; 11 for PC010 and 2, 5, 8, 9 &amp; 12 for PC027. The percentage of surviving larvae was calculated relative to day 0 (start of assay).<br>
Figure 3'PC: Effects E. coli strain AB309-105 expressing dsRNA target PC010 on survival of larvae of the mustard leaf beetle, p  cochleariae, over time. Data points for each treatment represent average mortality values from 3 different replicates. Error bars represent standard deviations. Target 10: SEQ ID NO: 488<br>
Figure 1-FV: Survival of E, varivestis larvae on bean leaf discs treated with dsRNA. Neonate larvae were fed bean leaf discs treated with 25µL of topically-applied solution of 1 µg/µl dsRNA (targets or gfp control), Afer 2 days, the insects were transferred onto fresh dsRNA-treated leaf discs. At day 4, larvae from one treatment were collected and placed in a plastic box containing fresh untreated bean foliage. The insects were assessed for mortality at days 2, 4, 6, 8 &amp; 10. the percentage of surviving larvae was calculated relative to day 0 (start of assay). Target 5: SEQ ID NO: 576; target 10: SEQ ID NO: 586; target 15: SEQ ID NO: 591; target 16: SEQ ID NO: 596; gfp dsRNA: SEQ ID NO: 235.<br><br>
Figure 2-KV: Effects of ingested target dsRNAs on surival of E. varivestis adults and resistance to snap bean foliar insect damage, (a) Surivival of A', varivestis adults on bean leaf treated with dsRNA. Adults were fed bean leaf discs treated with 75µl of topically-applied solution of 0.1 |ig/µl dsRNA (targets or gfp control). After 24 hours, the insects were transferred onto fresh dsRNA-treated leaf discs. After a further 24 hours, adults from one treatment were collected and placed in a plastic box containing potted fresh untreated whole bean plants. The insects were assessed for mortality at days 4, 5, 6, 7, 8, &amp; 11. The percentage of surviving adults was calculated relative to day 0 (start of assay), target 10: S1;Q ID NO: 586; target 15: SEQ ID NO: 591; target 16: SEQ ID NO: 596; gfp dsRNA: SEQ ID NO: 235. (b) Resistance to bean foliar damage caused by adults of the E. varivestis by dsRNA. Whole plants containing insects from one treatment (see (a)) were checked visually for foliar damage on day 9. (i) target 10; (ii) target 15; (iii) target 16; (iv) gfp dsRNA; (v) untreated.<br>
Figure l-TC: Survival of T. castaneum larvae on artificial diet treated with dsRNA of target 14. Neonate larvae were fed diet based on a tlour/milk mix with 1 mg dsRNA target 14. Control was water (without dsRNA) in diet. Four replicates of 10 first instar larvae per replicate were performed for each treatment. The insects were assessed for survival as average percentage means at days 6, 17, 31, 45 and 60. The percentage of surviving larvae was calculated relative to day 0 (start of assay). Error bars represent standard deviations. rargelTC014:SEQ lD NO: 878.<br>
Figure 1-MP: Effect of ingested target 27 dsRNA on the survival of Myzus persicae nymphs, first instars were placed in feeding chambers containing 50µl  of liquid diet with 2 µg/µl dsRNA (target 27 or gfp dsRNA control). Per treatment, 5 feeding chambers were set up with 10 instars in each feeding chamber. Number of survivors were assessed at 8 days post start of bioassay. Error bars represent standard deviations. target MP027: SliQ ID NO: 1061; gfpdsRNA: SliQ ID NO: 235.<br>
Figure 1-NL: Survival of Nilaparvata lugens on liquid artificial diet treated with dsRNA. Nymphs of the first to second larval stage were fed diet supplemented with 2 mg/ml solution of dsRNA targets in separate bioassays: (a) NE002, NE003, NE005, NE010; (b) NE009, NE016; (c) NE014, NE018; (d) NE013, NE015, NL021. Insect survival on targets were compared to diet only and diet with gfp dsRNA control at same concentration. Diet was<br><br>
replaced with fresh diet containing dsRNA every two days. I he number of surviving insects were assessed every day<br>
Figure 2-NL: Survival o{ Nilaparvata lugens on liquid artificial diet treated with differenl concentrations of target dsRNA NL002. Nymphs of the first to second larval stage were fed diet supplemented with 1, 0.2, 0.08, and 0.04 mg/ml (final concentration) of NL002. Diet was replaced with fresh diet containing dsRNA every two days, fhc numbers of surviving insects were assessed every day.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention provides a means for controlling pest infestations by exposing a pest to a target coding sequence that post-transcriptionally represses or inhibits a requisite biological function in the pest. Following exposure to a target sequence, the target forms a the dsRNA corresponding to part or whole of an essential pest gene and causes down regulation of the pest target via RNA interference (RNAi). As a result of the down regulation of mRNA, the dsRNA prevents expression of the target pest protein and hence causes death, growth arrest, or sterility of the pest.<br>
The present invention finds application in any area where it is desirable to inhibit viability, growth, development or reproduction of a pest, or to decrease pathogenicity or infectivity of a pest. Practical applications include, but are not limited to, (I) protecting plants against pest infestation; (2) pharmaceutical or veterinary use in humans and animals (for example to control, treat, or prevent insect infections in humans and animals); (3) protecting materials against damage caused by pests; and (4) protecting perishable materials (such as Ibodstuffs, seed, etc.) against damage caused by pests.<br>
Administering or exposing a double stranded ribonucleic acid molecule to a pest results in one or more of the following attributes: reduction in feeding by the pest, reduction in viability of the pest, death of the pest, inhibition of differentiation and development of the pest, absence of or reduced capacity for sexual reproduction by the pest, muscle formation, juvenile hormone formation, juvenile hormone regulation, ion regulation and transport, maintenance of cell membrane potential, amino acid biosynthesis, amino acid degradation, sperm formation, pheromone synthesis, pheromone sensing, antennae formation, wing formation, leg formation, development and differentiation, egg formation, larval maturation,<br><br><br><br>
Biological activity refers to the biological behavior and effects of a protein or peptide and its manifestations on a pest. For example, an inventive RNAi may prevent translation of a particular mRNA, thereby inhibiting the biological activity of the protein encoded by the mRNA or other biological activity of the pest.<br>
In the present description, an RNAi molecule may inhibit a biological activity in a pcsl, resulting in one or more of the following attributes: reduction in feeding by the pest, reduction in viability of the pest, death of the pest, inhibition of differentiation and development of the pest, absence of or reduced capacity for sexual reproduction by the pest, muscle formation, juvenile hormone formation, juvenile hormone regulation, ion regulation and transport, maintenance of cell membrane potential, amino acid biosynthesis, amino acid degradation, sperm formation, pheromone synthesis, pheromone sensing, antennae formation, wing formation, leg formation, development and differentiation, egg formation, larval maturation, digestive cn/yme formation, haemolymph synthesis, haemolymph maintenance, neurotransmission, cell division, energy metabolism, respiration, apoptosis, and any component of a eukaryotic cells' cytoskeletal structure, such as, for example, actins and tubulins.<br>
Complementary DNA (cDNA) refers to single-stranded DNA synthesized from a mature mRNA template. Though there arc several methods, cDNA is most often synthesized from mature (fully spliced) mRNA using the enzyme reverse transcriptase.   This enzyme operates on a single strand of mRNA, generating its complementary DNA based on the pairing of RNA base pairs (A, U, G, C) to their DNA complements (T, A, C, G).   Two nucleic acid strands arc substantially complementary when at least 85% of their bases pair.<br>
Desired Polynucleotide; a desired polynucleotide of the present invention is a genetic element, such as a promoter, enhancer, or terminator, or gene or polynucleotide that is to be transcribed and/or translated in a transformed cell that comprises the desired polynucleotide in its genome. If the desired polynucleotide comprises a sequence encoding a protein product, the coding region may be operably linked to regulatory elements, such as to a promoter and a terminator, that bring about expression of an associated messenger RNA transcript and/or a protein product encoded by the desired polynucleotide. Thus, a "desired polynucleotide" may comprise a gene that is operably linked in the 5'- to 3'- orientation, a promoter, a gene that encodes a protein, and a terminator. Alternatively, the desired polynucleotide may comprise a gene or fragment thereof, in a 'sense' or "antisense'<br><br>
orientation, the transcription of which produces nucleic acids that may affect expression of an endogenous gene in the host cell. A desired polynucleotide may also yield upon transcription a doublc-strandcd RNA product upon that initiates RNA interference of a gene to which the desired polynucleotide is associated. A desired polynucleotide of the present invention may be positioned within a vector, such that the left and right border sequences flank or are on either side of the desired polynucleotide. The present invention envisions the stable integration of one or more desired polynucleotides into the genome of at least one host cell. A desired polynucleotide may be mutated or a variant of its wild-type sequence. It is understood that all or part of the desired polynucleotide can be integrated into the genome of a host. It also is understood that the term 'desired polynucleotide' encompasses one or more of such polynucleotides.   Thus, a vector of the present invention may comprise one, two, three, four, live, six, seven, eight, nine, ten, or more desired polynucleotides.<br>
"Exposing' encompasses any method by which a pest may come into contact with a dsRNA, wherein the dsRNA comprises annealed complementary strands, one of which has a nucleotide sequence which is complementary to at least part of the nucleotide sequence of a pest target gene to be down-regulated. A pest may be exposed to the dsRNA by direct uptake (e.g. by feeding), which does not require expression of dsRNA within the pest. Alternatively, a pest may come into direct contact with a composition comprising the dsRNA. For example, a pest may come into contact with a surface or material treated with a composition comprising a dsRNA. A dsRNA may be expressed by a prokaryotic (for instance, but not limited to, a bacterial) or eukaryotic (for instance, but not limited to, a yeast) host cell or host organism.<br>
Foreign; 'foreign,' with respect to a nucleic acid, means that that nucleic acid is derived from non-host organisms. According to the present invention, foreign DNA or RNA represents nucleic acids that are naturally occurring in the genetic makeup of viruses, mammals, fish or birds, but arc not naturally occurring in the host that is to be transformed. Thus, a foreign nucleic acid is one that encodes, for instance, a polypeptide that is not naturally produced by the transformed host. A foreign nucleic acid does not have to encode a protein product.<br>
Gene: refers to a polynucleotide sequence that comprises control and coding sequences necessary for the production of a polypeptide or precursor.   J he polypeptide can be encoded by a full length coding sequence or by any portion of the coding sequence. A gene may constitute an uninterrupted coding sequence or it may include one or more introns, bound by the appropriate splice junctions. Moreover, a gene may contain one or more<br><br>
modifications in cither the coding or the untranslated regions that could affect the biological activity or the chemical structure ol'the expression product, the rate of expression, or the manner of expression control. Such modifications include, but arc not limited to, mutations, insertions, deletions, and substitutions of one or more nucleotides. In this regard, such modified genes may be referred to as 'variants" of the 'native" gene.<br>
Genetic element: a "genetic clement" is any discreet nucleotide sequence such as, but not limited to, a promoter, gene, terminator, intron, enhancer, spacer, 5'-untranslated region, 3'-untranslated region, or recombinase recognition site.<br>
Genetic modification: stable introduction of a nucleic acid into the genome of certain organisms by applying methods in molecular and cell biology.<br>
"Gene suppression" or "down-regulation of gene expression" or "inhibition of gene expression" are used interchangeably and refer to a measurable or observable reduction in gene expression or a complete abolition of detectable gene expression, at the level of protein product and/or mRNA product from the target gene. Down-regulation or inhibition of gene expression is "specific" when down-regulation or inhibition of the target gene occurs without manifest effects on other genes of the pest.<br>
Depending on the nature of the target gene, down-regulation or inhibition of gene expression in cells of a pest can be confirmed by phenotypic analysis of the cell or the whole pest or by measurement of mRNA or protein expression using molecular techniques such as RNA solution hybridization, nuclease protection. Northern hybridization, reverse transcription, gene expression monitoring with a microarray, antibody binding, enzyme-linked immunosorbent assay (ELISA), Western blotting, radioimmunoassay (RIA), other immunoassays, or fluorescence-activated cell analysis (FACS).<br>
Gymnosperm, as used herein, refers to a seed plant that bears seed without ovaries. Examples of gymnospcrms include conifers, cycads, ginkgos, and ephedras.<br>
Homology, as used herein relates to sequences; Protein, or nucleotide sequences are likely to be homologous if they show a "significant" level of sequence similarity or more preferably sequence identity.   1 ruely homologous sequences are related by divergence from a common ancestor gene. Sequence homologs can be of two types:(i) where homologs exist in different species they arc known as orthologs. e.g. the α -globin genes in mouse and human are orthologs; (ii) paralogucs are homologous genes in within a single species, e.g. the a- and β- globin genes in mouse are paralogs.<br>
Host cell: refers to a microorganism, a prokaryotic cell, a cukaryotic cell, or cell line cultured as a unicellular entity that may be, or has been, used as a recipient for a recombinant<br><br>
vector or other transfer of polynucleotides, and includes the progeny of the original cell that has been transfected. The progeny of a single cell may not necessarily be completely identical in morphology or in genomic or total DNA complement as the original parent due to natural accidental, or deliberate mutation.<br>
Introduction: as used herein, refers to the insertion of a nucleic acid sequence into a cell, by methods including infection, transfcction, transformation, or transduction.<br>
Insect pests as used herein pests are include but are not limited to: from the order 1 epidoptcra, for example, Acleris spp,, Adoxophyes spp., Aegeria spp., Agrotis spp., Alabama argillaceae, Amylois spp,, Anticarsia gemmatalis, Archips spp, Argyrotaenia spp,, Autographa spp., Busseola fusca. Cadra cautella, Carposina nipponensis, Chilo spp,, Choristoneura spp., Clysia amhiguella, Cnaphalocrocis spp., Cnephasia spp., Cochylis spp., Coleophora spp., Crocidolomia binotalis, Cryptophlebia leucotreta, Cydia spp., Diatraea spp.. Diparopsis castanea, Earias spp., Ephestia spp., Eucosma spp., Eupoecilia ambiguella, Euproclis spp., Euxoa spp., Grapholita spp., Iledya nubiferana, lleliothis spp., Ilellula undalis, llyphantria cunea, Keiferia lycopersicella, Leucoptera scitella, Lithocollethis spp., Lohesia botrana, Lymantria spp,, Lyonetia spp., Malacosoma spp,, Mamestra brassicae, Manduca sexta, Operophtera spp,, Ostrinia Nubilalis, Pammene spp„ Pandemis spp,, Panolis flammea, Pectinophora gossypiella, Phthorimaea operculella, Pieris rapae, Pieris spp., Plutella xylostelkh Prays spp., Scirpophaga spp., Sesamia spp., Sparganothis spp,, Spodoptera spp.. Synan/hedon spp., Thaumetopoea spp., Tortrix spp., Prichoplusia ni and Yponomeuta spp.from the order Coleoptera, for example, Agriotes spp,, Anthonomus spp,, Atomaria linearis, Chaetocnema tibialis. Cosmopolites spp,, Curculio spp., Dermestes spp„ Epilachna spp., Eremnus spp., Leptinotarsa decemlineata, Lissorhoptrus spp., Melolontha spp,, Orycaephilus spp., Otiorhynchus spp., Phlyctinus spp., Popillia spp., Psylliodes spp„ Rhizopertha spp., Scarabeidae, Sitophilus spp,, Sitotroga spp,, Tenebrio spp., Tribolium spp, and Trogoderma spp,;<br>
from the order Orthoptera, for example, Blatta spp,, Blattella spp., Gryllotalpa spp,, Leucophaea maderae, Locusta spp,, Periplaneta ssp., and Schistocerca spp.;<br>
from the order Isoptera, for exampl;e ,Reticulitemes ssp;<br>
from the order Psocoptera, for example ,Liposcelis spp.;<br>
from the order Anoplura, for example,Ilaematopinus spp., Einognathus spp., Pediculus spp.. Pemphigus spp. and Phylloxera spp.;<br>
from the order Mallophaga, for example,Damalinea spp. and Trichodectes spp.;<br><br><br>
Monocotylcdonous plant (monocot) is a flowering plant having embryos with one cotyledon or seed leaf, parallel leaf veins, and flower parts in multiples of three. Examples of monocols include, but are not limited to turfgrass, maize, rice, oat, wheat, barley, sorghum, orchid, iris, lily, onion, and palm.<br><br>
Pest or target pest refers to insects, arachnids, crustaceans, fungi, bacteria, viruses, nematodes, flalworms, roundworms, pinworms, hookworms, tapeworms, trypanosomes, schistosomes, botflies, fleas, licks, mites, and hcc and the Mke that are pervasive in the human environment. A pest may ingest or contact one or more cells, tissues, or products produced by an organism transformed with a double stranded gene suppression agent, as well as a material or surface treated with a double stranded gene suppression agent.<br>
Nematodes, or roundworms, arc one of the most common phyla of animals, with over 20,000 different described species (over 15,000 are parasitic). They are ubiquitous in freshwater, marine, and terrestrial environments, where they often outnumber other animals in both individual and species counts, and arc found in locations as diverse as Antarctica and oceanic trenches, further, there are a great many parasitic forms, including pathogens in most plants and animals.<br><br>
Normal cell refers to a cell of an untransformed phenotype or exhibiting a morphology of a non-transformed cell of the tissue type being examined.<br>
Operably linked: combining two or more molecules in such a fashion that in combination they function properly in a cell, For instance, a promoter is operably linked to a structural gene when the promoter controls transcription of the structural gene.<br>
Orthologs are genes that are related by vertical descent from a common ancestor and encode proteins with the same function in different species. Due to their separation following a speciation event, orthologs may diverge, but usually have similarity at the scqence and structure levels,   two genes that are derived from a common ancestor and encode proteins with similar function arc rcfered to as orthologous. Identification of orthologs is critical for reliable predictions of gene function in newly sequenced genomes.<br>
'Pest control agent", or 'gene suppression agent" refers to a particular RNA molecule comprising  a  first  RNA  segment and a second  RNA  segment,  wherein the<br><br>
complementarity between the first and the second RNA segments results in the ability of the two segments lo hybridi/e in vivo and in vitro to form a double stranded molecule, ll may generally be preferable lo include a third RNA segment linking and stabilizing the first and second sequences such that a stem can be formed linked together at one end of each oi'the first and second segments by the third segment to forms a loop, so that the entire structure forms into a stem and loop structure, or even more tightly hybridizing structures may form into a stem-loop knotted structure. Alternatively, a symmetrical hairpin could be formed without a third segment in which there is no designed loop, but for steric reasons a hairpin would create its own loop when the stem is long enough to stabilize itself, fhe first and the second RNA segments will generally lie within the length of the RNA molecule and be substantially inverted repeats of each other and linked together by the third RNA segment. The first and the second segments correspond invariably and not respectively to a sense and an antisense sequence with respect to the target RNA transcribed from the target gene in the target insect pest that is suppressed by the ingestion of the dsRNA molecule.<br>
The pest control agent can also be a substantially purified (or isolated) nucleic acid molecule and more specifically nucleic acid molecules or nucleic acid fragment molecules thereof from a genomic DNA (gDNA) or cDNA library. Alternatively, the fragments may comprise smaller oligonucleotides having from about 15 to about 250 nucleotide residues, and more preferably, about 15 to about 30 nucleotide residues.<br>
Pesticide refers to any substance or mixture of substances intended for preventing, destroying, repelling, or mitigating any pest. A pesticide may be a chemical substance or biological agent used against pests including insects, pathogens, weeds, nematodes, and microbes that compete with humans for food, destroy property, spread disease, or are a nuisance.<br>
Phenotype is a distinguishing feature or characteristic of an organism, which may be altered according to the present invention by integrating one or more 'desired polynucleotides' and/or screenable/selectabic markers into the genome of at least one cell of a transformed organism. The 'desired polynucleotide(S)' and/or markers may confer a change in the phenotype of a transformed organism, by modifying any one of a number of genetic, molecular, biochemical, physiological, or morphological characteristics or properties of the transformed cell or organism as a whole.<br>
Plant and plant tissue: a 'plant" is any of various photosynthetic, eukaryotic, multicellular organisms of the kingdom Plantae characteristically producing embryos, containing chloroplasts, and having cellulose cell walls. A part of a plant, i.e., a 'plant<br><br>
tissue" may be treated according to the methods of the present invention to prevent pest infestation on the plant or on the part of the plant. Many suitable plant tissues can be treated according to the present invention and include, but are not limited to, somatic embryos, pollen, leaves, stems, calli, stolons, microtubers, and shoots.   Ihus, the present invention envisions the treatment of angiosperm and gymnospcrm plants such as acacia, alfalfa, apple, apricot, artichoke, ash tree, asparagus, avocado, banana, barley, beans, beet, birch, beech, blackberry, blueberry, broccoli, brussels sprouts, cabbage, canola, cantaloupe, carrot, cassava, cauliflower, cedar, a cereal, celery, chestnut, cherry, Chinese cabbage, citrus, clcmintinc, clover, coffee, corn, cotton, cowpea, cucumber, cypress, eggplant, elm, endive, eucalyptus, fennel, figes, fir, geranium, grape, grapefruit, groundnuts, ground cherry, gum hemlock, hickory, kale, kiwifruit, kohlrabi, larch, lettuce, leek, lemon, lime, locust, pine, maidenhair, mai/c, mango, maple, melon, millet, mushroom, mustard, nuts, oak, oats, okra, onion, orange, an ornamental plant or flower or tree, papaya, palm, parsley, parsnip, pea, peach, peanut, pear, peat, pepper, persimmon, pigeon pea, pine, pineapple, plantain, plum, pomegranate, potato, pumpkin, radicchio, radish, rapcsccd, raspberry, rice, rye, sorghum,, sallow, soybean, spinach, spruce, squash, strawberry, sugarbeet, sugarcane, sunflower, sweet potato, sweet corn, tangerine, tea, tobacco, tomato, trees, triticale, turf grasses, turnips, a vine, walnut, watercress, watermelon, wheat, yams, yew, and zucchini.<br>
According to the present invention 'plant tissue' also encompasses plant cells. FManl cells include suspension cultures, callus, embryos, meristematic regions, callus tissue, leaves, roots, shoots, gametophytes, sporophytes, pollen, seeds and microspores. Plant tissues may be at various stages of maturity and may be grown in liquid or solid culture, or in soil or suitable media in pots, greenhouses or fields. A plant tissue also refers to any clone of such a plant, seed, progeny, propagule whether generated sexually or asexually, and descendcnts of any of these, such as cuttings or seed.<br>
Promoter is intended to mean a nucleic acid, preferably I3NA that binds RNA polymerase and/or other transcription regulatory elements. As with any promoter, the promoters of the current invention will facilitate or control the transcription of DNA or RNA to generate an mRNA molecule from a nucleic acid molecule that is operably linked to the promoter. As stated earlier, the RNA generated may code for a protein or polypeptide or may code for an RNA interfering, or antisense molecule.<br>
Polynucleotide is a nucleotide sequence, comprising a gene coding sequence or a fragment thereof, a promoter, an intron, an enhancer region, a polyadenylation site, a translation initiation site, 5' or 3' untranslated regions, a reporter gene, a selectable marker or<br><br>
the like.   The polynucleotide may comprise single stranded or double stranded DNA or RNA. the polynucleotide may comprise modified bases or a modified backbone.   T he polynucleotide may be genomic, an RNA transcript (such as an mRNA) or a processed nucleotide sequence (such as a cDNA).   The polynucleotide may comprise a sequence in cither sense or antisense orientations.<br>
An isolated polynucleotide is a polynucleotide sequence that is not in its native state, e.g., the polynucleotide is comprised of a nucleotide sequence not found in nature or the polynucleotide is separated from nucleotide sequences with which it typically is in proximity or is next to nucleotide sequences with which it typically is not in proximity.<br>
Recombinant nucleotide sequence refers to a nucleic acid molecule that contains a genetically engineered modification through manipulation via mutagenesis, restriction enzymes, and the like.<br>
RNA interference (RNAi) refers to sequence-specific or gene-specific suppression of gene expression (protein synthesis) that is mediated by short interfering RNA (siRNA).<br>
Sequence identity: as used herein, 'sequence identity' or 'identity" in the context of two nucleic acid sequences includes reference to the residues in the two sequences which are the same when aligned for maximum correspondence over a specified region.<br>
As used herein, percentage of sequence identity means the value determined by comparing two optimally aligned sequences over a comparison window, wherein the portion of the polynucleotide sequence in the comparison window may comprise additions or deletions (i.e., gaps) as compared to the reference sequence (which does not comprise additions or deletions) for optimal alignment of the two sequences, The percentage is calculated by determining the number of positions at which the identical nucleic acid base occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison and multiplying the result by 100 to yield the percentage of sequence identity.<br><br><br><br>
Transformation: A process by which a nucleic acid is stably inserted into the genome of an organism.   I ransformation may occur under natural or artificial conditions using various methods well known in the art. Transformation may rely on any known method for the insertion of nucleic acid sequences into a prokaryotic or eukaryotic host cell, including microorganism-mediated transformation, viral infection, whiskers, electroporation.<br><br>
microinjection, polyethylene glycol-treatment, heat shock, lipofection, and particle bombardment.<br>
Transgenic organism comprises at least one cell in which an exogenous nucleic acid has been stably integrated, A transgenic organism according to the invention is for instance a bacterial, or eukaryotic, such as a yeast, host cell or host organism. The bacterium can be chosen from the group comprising Gram-negative and Gram-positive bacteria, such as, but not limited to, Escherichia spp. (e.g. E, coli). Bacillus spp. (e.g. B, thurin^iensis), Rhizohium spp., iMctohacillius spp., Lactococcus spp., etc.. The yeast can be chosen from the group comprising Saccharomyces spp., etc.<br>
Variant: a 'variant," as used herein, is understood to mean a nucleotide sequence that deviates from the standard, or given, nucleotide or amino acid sequence of a particular gene or protein.   The terms, 'isoform," 'Isotype," and 'analog" also refer to "'variant" forms of a nucleotide sequence. "Variant" may also refer to a "shuffled gene" such as those described in Maxygen-assigned patents.<br>
It is understood that the present invention is not limited to the particular methodology, protocols, vectors, and reagents, etc., described herein, as these may vary. It is also to be understood that the terminology used herein is used for the purpose of describing particular embodiments only, and is not intended to limit the scope of the present invention. It must be noted that as used herein and in the appended claims, the singular forms "a," "an," and "the" include plural reference unless the context clearly dictates otherwise. Thus, for example, a reference to "a gene" is a reference to one or more genes and includes equivalents thereof known to those skilled in the art and so forth.<br>
1. Target Pests<br>
The present invention provides methodology and constructs for controlling pest infestations by administering, or otherwise exposing, to a pest a target coding sequence that post-transcriptionally represses or inhibits a requisite biological function in the pest. As used herein, the term "pest" refers to insects, arachnids, crustaceans, fungi, bacteria, viruses, nematodes, flatworms, roundworms, pinworms, hookworms, tapeworms, trypanosomes, schistosomes, botllies, fleas, ticks, mites, and lice and the like that are pervasive in the human environment. A pest may ingest or contact one or more cells, tissues, or products produced by an organism transformed with a double stranded gene suppression agent, as well as a surface or material treated with a double stranded gene suppression agent.<br><br>
A 'pest resistance'" trail is a characteristic of a transgenic host that causes the host to be resistant to attack from a pest that typically inflicts damage to the host. Such pest resistance can arise from a natural mutation or more typically from incorporation of recombinant DNA that confers pest resistance, lo impart pest resistance to a transgenic host, a recombinant DNA can, for example, be transcribed into a RNA molecule that forms a dsRNA molecule within the tissues or fluids of the recombinant host. The dsRNA molecule is comprised in part of a segment of RNA that is identical to a corresponding RNA segment encoded from a PNA sequence within a pest that prefers to feed on the recombinant host, lixpression of the gene within the target pest is suppressed by the dsRNA, and the suppression of expression of the gene in the target pest results in the host being pest resistant.<br>
Suitable pests include any organism that causes damage to another organism. I he invention contemplates insect, nematode, and fungal pests in particular.<br>
Insect as used herein can be any insect, meaning any organism belonging to the Kingdom Animals, more specific to the Phylum Arthropoda, and to the Class Insecta or the Class Arachnida.   fhe methods of the invention are applicable to all insects and that are susceptible to gene silencing by RNA interference and that are capable of internalising double-stranded RNA from their immediate environment.<br>
In one embodiment of the invention, the insect may belong to the following orders: Acari, Araneae, Anoplura, Coleoptcra, Collcmbola, Dermaptera, Dictyoptera, Diplura, Diptera, l-mbioplera, liphemcroptcra, Grylloblatodea, llemiptera, Homoptera, Ilymenoptera, Isopiera, Lepidoptcra, Mallophaga, Mecoptera, Neuroptera, Odonata, Orthoptera, Phasmida, Plecoptcra, Protura, Psocoptcra, Siphonaptera, Siphunculata, fhysanura, Strepsiptera, Thysanoptera, Trichoptera, and /oraptera.<br>
in preferred, but non-limiting, embodiments and methods of the invention the insect is chosen from the group consisting of:<br>
(1) an insect which is a plant pest, such as but not limited to Nilaparvata spp. (e.g. N. lugens (brownplanthopper)); Laodelphax spp. (e.g. L. striatellus (small brown planthopper)); Nephotcttix spp, (e.g. N. virescens or N. cincticeps (green leafhopper), or N.nigropictus (rice leafhopper)); Sogatella spp. (e.g. S. furcifera (while-backed planthopper)); Blissus spp. (e.g. B. leucoptenis leucopterus (chinch bug)); Scotinophora spp. (e.g. S. vermidulate (rice blackbug)); Acrosternum spp. (e.g. A. hilare (green stink bug)); Parnara spp. (e.g. P. guttata (rice skipper)); Chilo spp. (e.g. C. suppressalis (rice striped stem borer), C. auricilius (gold-fringed stem borer), or C. polychrysus (dark-headed stem borer)); Chilotraea spp. (e.g. C. polychrysa (rice stalk borer)); Sesamia spp. (e.g. .S'. inferens   (pink rice borer)); Tryporyza<br><br><br>
(e.g. A. gossypii (cotton aphid), A. mellifera): Lygus spp. (e.g, A. lineolaris (tarnished plant bug) or .. hesperus (western tarnished plant bug)); Euschistus spp. (e.g. E. conspersus (conspersc stink bug)); Chlorochroa spp. (c,g. C. sayi (Say stinkbug)); Nezara spp. (e.g. N. vitidida (southern green stinkbug)); Thrips spp. (e.g. T. tabaci (onion thrips)); Franklmiella<br><br><br>
spp.Culicoidess spp., Diaphorina spp., Diaprepes spp., Euclidia spp., Glossina spp., Gryllus spp., Hydropsyche spp., Julodis spp., Lonornia spp,, Lutzomyia spp., Lysiphebus spp, Meladema spp, Mycetophagus spp., Nasonia spp., Oncometopia spp., Papilio spp., Pediculus<br><br><br>
I he methods of the invention arc applicable to all nematodes and that are susceptible to gene silencing by RNA interference and that arc capable of internalising double-stranded RNA from their immediate environment.<br>
in one embodiment of the invention, the nematode may belong to the family of the Heterodcridae. encompassing the genera ileterodera and Globodera.<br><br>
In preferred, but non-limiting, embodiments and methods of the invention the insect is chosen from the group eoprising but not limited to:<br>
(1)	a nematode which is a plant pathogenic nematode, such as but not limited to: Mcloidogyne spp. (e.g. M. incognita, M. javanica, M. graminicola, M. arcnaria, M. chitwoodi, M. hapla or M, paranaensis); Ileterodera spp, (e.g. II, oryzae, II. glycines, II. zeae or II. schachiii): Glohodera spp, (e.g, G. pallida or G, rostochiensis); Rotylenchulus spp. (e.g. R. rcniformis); Pralylenchus spp, (e.g. P. coffeae, P. Zeae or P, goodeyi); Radopholus spp. (e.g. R. similis)\ Ilirschmaniella spp. (e.g. //. oryzae); Ancylostoma spp. (e.g. A. caninum, A. ceylanicnm, A. duodenale or A. tiihaeforme); Anisakid; Aphelenchoides spp. (e.g. A. Besseyi); Ascarids; Ascaris spp., (e.g. A. suum ox A. lumhridoides)\ Belonolaimus spp.; Brugia spp. (e.g. B. malayi or B. pahangi)\ Bursaphelenchus spp.; Caenorhahditis spp. (e.g. C elegans, C. hriggsae or C. remanei); Clostridium spp. (e.g. C acetohutylicum); Cooperia spp. (e.g. C. oncophora)\ Criconemoides spp.; Cyathostomum spp. (e.g. C. catinatum, C. coronatum or C. paferafum): Cylicocyclus spp. (e.g. C. insigne, C nassatus or C. radiatus); Cylicostephanus spp. (e.g. C goldi or ('. longihursatus); Diphyllobothrium: Dirofilaria spp. (e.g. D. immitis); Ditylenchus spp. (e.g. D. dipsaci, D. destructor or D. Angustus); Enterohius spp. (e.g. E. vermicularis); Ilaemonchus spp. (e.g. //. contortus); Ilelicotylenchus spp.; Hoplolaimus spp.; IJtomosoides spp. (e.g. L. sigmodontis); Longidorus spp. (e.g, /.. macrosoma)\ Necator spp. (e.g. A^. americanus); Nippostrongytus spp. (e.g. N. hrasiliensis) ; Onchocerca spp. (e.g. (). volvulus): Ostertagia spp. (e.g. O. ostertagi); Parastrongyloides spp. (e.g. P. trichosuri); Paratrichodorus spp. (e.g. P, minor or P. teres); Parelaphostrongylus spp. (e.g. P, ienuis)\ Radophulus spp.; Scutellonerna. spp.; Strongyloides spp. (e.g. S. Ratti or .S". stercoralis); Teladorsagia spp. (e.g. 7'. circumcincta); Toxascaris spp. (e.g. T. leonina); Toxocara spp. (e.g. T. canis or 7! cati)\ Trichinella spp. (e.g. 7! hritovi, T. spiralis or T, spirae); Trichodorus spp, (e.g. T. similis)\Trichuris spp. (e.g. T, muris, T, vulpis or T, trichiura); Tylenchulus spp.; Tylenchorhynchus spp.; Uncinaria spp. (e.g. U. stenocephala); Wuchereria spp. (e.g. W. hancrofii): Xiphinema spp. (e.g. X. Index orX americanum).<br>
(2)	a nematode capable of infesting humans such as, but not limited to: Enterohius vermicularis, the pinworm that causes enterobiasis; Ascaris lumbridoides, the large intestinal roundworm that causes ascariasis; Necator and Ancylostoma, two types of hookworms that cause ancylostomiasis; Trichuris trichiura, the whipworm that causes trichuriasis; Strongyloides stercoralis that causes strongyloidiasis; and Trichonella spirae that causes trichinosis; Brugia malayi and Wuchereria hancrofti, the filarial nematodes associated with the worm infections known as lymphatic filariasis and its gross manifestation, elephantiasis, and<br><br>
Onchocerca volvulus that causes river blindness. Transfer of nematodes to humans may also occur through blood-feeding mosquitoes which have fed upon infected animals or humans;<br>
(3)	a nematode capable of infesting animals such as, but not limited to: dogs (Hookworms e.g. Ancylostoma caninum or Uncinaria stenocephala, Ascarids e.g. Toxocara canis or Toxascaris leonina, or Whipworms e.g. Trichuris vulpis), cats (Hookworms e.g. Ancylostoma lubaeforme, Ascarids e.g. Toxocara cati), tlsh (herring worms or cod worms e.g. AnisakicL or tapeworm e.g. Diphyllohothrium), sheep (Wire worms e. g. Ilaemonchus coniorlus) and cattle (Cjastro-intestinal worms e.g. Ostertagia ostertagi, Cooperia oncophora)(4)	a nematode that causes unwanted damage to substrates or materials, such as nematodes that attack foodstuffs, seeds, wood, paint, plastic, clothing etc. Examples of such nematodes include but arc not limited to Meloidogyne spp. (e.g. M. incognita, M. javanica, M. arenaria, M. graminicola, M. chitwoodi or M hapla); Heterodera spp. (e.g. //. oryzae, II. glycines, II. zeae or //. schachtii); Globodera spp. (e.g. G, pallida or G. rostochiensis): Ditylenchus spp. (e.g. D. dipsaci, D, destructor or D. angustus); Belonolaimus spp.; Rotylenchulus spp. (e.g. R. reniformis); Pratylenchus spp. (e.g. P. coffeae, P. goodeyi or P. zeae): Radopholus spp. (e.g. R. Similis); Hirschmaniella spp. (e.g. //. oryzae); Aphelenchoides spp. (e.g. A. hesseyi): Criconemoides spp.; Longidorus spp.; I-Ielicotylenchus spp.; lloplolaimus spp.; Xiphinema spp.; Paratrichodorus spp. (e.g. P, minor); Tylenchorhynchus spp;<br>
(5)	virus transmitting nematodes (e.g. Longidorus macrosoma: transmits prunus necrotic ring spot virus, Xiphinema americanum: transmits tobacco ring spot virus, Paratrichadorus teres: transmits pea early browning virus, or Trichodorus similis: transmits tobacco rattle virus).<br>
fungal pests of particular interest include but arc not limited to the following. In one embodiment of the invention, the fungus may be a mold, or more particularly a filamentous fungus. In other embodiments of the invention, the fungus may be a yeast.<br>
In one embodiment the fungus may be an ascomycetes fungus, i.e. a fungus belonging to the Phylum Ascomycota,<br>
In preferred, but non-limiting, embodiments of the invention the fungal cell is chosen from the group consisting of:<br>
(1) a fungal cell of or a cell derived from a plant pathogenic fungus, such as but not limited to Acremoniella spp., Allernaria spp. (e.g. Alternaria hrassicola or Alternaria solani), Ascochyta spp. (e.g. Ascochyta pisi), Botrytis spp. f^e.g. Botrytis cinerea or Botryotinia fuckeliana), Cladosporium spp., Cercospora spp. ("e.g. Cercospora kikuchii or<br><br><br>
(2) a fungal cell of, or a cell derived from a fungus capable of infesting humans such as, but not limited to, Candida spp., particularly Candida albicam; Dermatophytes including Epidemiophylon spp.. Trichophyton spp, and Microsporum spp.; Aspergillus spp. (particularly Aspergillus flavus, Aspergillus fumigatus, Aspergillus nidulans, Aspergillus niger or  Aspergillus  terreus);  Blastomyces dermatittdis;  Paracoccidioides  hrasiliensis;<br><br><br>
II. Identification of Target Sequences<br>
The present invention provides a method for identifying and obtaining a nuelcic acid comprising a nucleotide sequence for producing a dsRNA or siRNA. For example, such a method comprises: (a) probing a cDNA or genomic DNA library with a hybridization probe comprising all or a portion of a nucleotide sequence or a homolog thereof from a targeted pest; (b) identifying a DNA clone that hybridi/es with the hybridization probe; (c) isolating the DNA clone identified in step (b); and (d) sequencing the cDNA or genomic DNA fragment that comprises the clone isolated in step (c) wherein the sequenced nucleic acid molecule transcribes all or a substantial portion of the RNA nucleotide acid sequence or a homolog thereof.<br>
Additionally, the present invention contemplates a method for obtaining a nucleic acid fragment comprising a nucleotide sequence for producing a substantial portion of a dsRNA or siRNA comprising: (a) synthesizing first and a second oligonucleotide primers corresponding to a portion of one of the nucleotide sequences from a targeted pest; and (b) amplifying a cDNA or genomic DNA template in a cloning vector using the first and second oligonucleotide primers of step (a) wherein the amplified nucleic acid molecule transcribes a substantial portion of a dsRNA or siRNA of the present invention.<br>
In practicing the present invention, a target gene may be derived from any pest that causes damage to another organism. Several criteria may be employed in the selection of preferred target genes,   fhe gene is one whose protein product has a rapid turnover rate, so<br><br>
that dsRNA inhibition will result in a rapid decrease in protein levels. In certain embodiments it is advantageous to select a gene for which a small drop in expression level results in deleterious effects for the recipient pest. If it is desired to target a broad range of insect species, for example, a gene is selected that is highly conserved across these species. Conversely, for the purpose of conferring specificity, in certain embodiments of the invention, a gene is selected that contains regions that are poorly conserved between individual insect species, or between insects and other organisms. In certain embodiments it may be desirable to select a gene that has no known homologs in other organisms.<br>
As used herein, the term "derived from" refers to a specified nucleotide sequence that may be obtained from a particular specified source or species, albeit not necessarily directly from that specified source or species.<br>
In one embodiment, a gene is selected that is expressed in the insect gut. Targeting genes expressed in the gut avoids the requirement for the dsRNA to spread within the insect. Target genes for use in the present invention may include, for example, those that share substantial homologies to the nucleotide sequences of known gut-expressed genes that encode protein components of the plasma membrane proton V-ATPase (Dow et at., 1997,; Dow, 1999), This protein complex is the sole energizer of epithelial ion transport and is responsible for alkalinization of the midgut lumen. The V-A'fPase is also expressed in the Malpighian tubule, an outgrowth of the insect hindgut that functions in fiuid balance and detoxification of foreign compounds in a manner analogous to a kidney organ of a mammal.<br>
In another embodiment, a gene is selected that is essentially involved in the growth, development, and reproduction of an insect. Exxemplary genes include but arc not limited to the structural subunits of ribosomal proteins and a beta-coatamer gene, CHI)3 gene. Ribosomal proteins such as S4 (RpS4) and S9(RpS9) are structural constituents of the ribosome involved in protein biosynthesis and which are components of the cylosolic small ribosomal subunit, the ribosomal proteins such as L9 and I&gt;19 are structural constituent of ribosome involved in protein biosynthesis which is localised to the ribosome. The beta-coatamer gene in C. elegans encodes a protein which is a subunit of a multimcric complex that forms a membrane vesicle coat Similar sequences have been found in diverse organisms such as Amhidopsis thaliana. Drosophila melanogaster^ and Saccharomyces cerevisiae^ Related sequences are found in diverse organisms such as Leptinotarsa decemlinealcu Phaedon cochleariae, Epilachna varivetis, Anthonomus grandis, Triholium castaneum, Myzus persicae, Nilaparvala lugens. Chilo suppressalis, Plutella xylostella and Acheta domesticus. Other target genes for use in the present invention may include, for example.<br><br>
those that play important roles in viability, growth, development, reproduction, and infectiviiy. These target genes include, for example, house keeping genes, transcription factors, and insect specific genes or lethal knockout mutations in Caenorhabditis or Drosophila. fhe target genes for use in the present invention may also be those that are from other organisms, e.g., from a nematode (e.g., Meloidogyne spp. or Heterodera spp.), other insects or araehnidae (e.g. Leptinotarsa spp., Phaedon spp., Epilachna spp., Anthonomus spp., Triholium spp., Myzus spp,, Nilaparvata spp., Chilo spp., Plutella spp., ox Acheta spp.,. Additionally, the nucleotide sequences for use as a target sequence in the present invention may also be derived from viral, bacterial, fungal, insect or fungal genes whose functions have been established from literature and the nucleotide sequences of which share substantial similarity with the target genes in the genome of an insect.<br>
for many of the insects that are potential targets for control by the present invention, there may be limited information regarding the sequences of most genes or the phenotype resulting from mutation of particular genes. Therefore, genes may be selected based on available information available concerning corresponding genes in a model organism, such as Caenorhahditis or Drosophila, or in some other insect species. Genes may also be selected based on available sequence information for other species, such as nematode or fungal species, in which the genes have been characterized. In some cases it will be possible to obtain the sequence of a corresponding gene from a target insect by searching databases, such as GenBank, using either the name of the gene or the gene sequence. Once the sequence is obtained, PCR may be used to amplify an appropriately selected segment of the gene in the insect for use in the present invention.<br>
In order to obtain a DNA segment from the corresponding gene in an insect species, for example, PCR primers may be designed based on the sequence as found in C elegans or Drosophila. or an insect from which the gene has already been cloned. The primers arc designed to amplify a DNA segment of sufficient length for use in the present invention. Amplification conditions are selected so that amplification will occur even if the primers do not exactly match the target sequence. Alternately, the gene, or a portion thereof, may be cloned from a genomic DNA or cDNA library prepared from the insect pest species, using a known insect gene as a probe. Techniques for performing PCR and cloning from libraries are known. Further details of the process by which DNA segments from target insect pest species may be isolated based on the sequence of genes previously cloned from an insect species are provided in the Examples.   One of ordinary skill in the art will recogni/c that a<br><br>
variety of techniques may be used to isolate gene segments from insect pest species that correspond to genes previously isolated from other species.<br>
III. Methods for inhibiting or suppressing a target gene<br>
The present invention provides methods for inhibiting gene expression of one or multiple target genes in a target pest using stabilized dsRNA methods, The invention is particularly useful for modulating eukaryotic gene expression, in particular modulating the expression of genes present in pests that exhibit a digestive system pH level that is from about 4.5 to about 9.5, more preferably from about 5.0 to about 8,0, and even more preferably from about 6.5 to about 7.5. I-or pests with a digestive system that exhibits pH levels outside of these ranges, delivery methods may be desired for use that do not require ingestion of dsRNA molecules.<br>
I he methods of the invention encompass the simultaneous or sequential provision of two or more different double-stranded RNAs or RNA constructs to the same insect, so as to achieve down-regulation or inhibition of multiple target genes or to achieve a more potent inhibition of a single target gene.<br>
Alternatively, multiple targets are hit by the provision of one double-stranded RNA that hits multiple target sequences, and a single target is more efficiently inhibited by the presence of more than one copy of the double stranded RNA fragment corresponding to the target gene. Thus, in one embodiment of the invention, the double-stranded RNA construct comprises multiple dsRNA regions, at least one strand of each dsRNA region comprising a nucleotide sequence that is complementary to at least part of a target nucleotide sequence of an insect target gene. According to the invention, the dsRNA regions in the RNA construct may be complementary to the same or to different target genes and/or the dsRNA regions may be complementary to targets from the same or from different insect species. Use of such dsRNA constructs in a plant host cell, thus establishes a more potent resistance to a single or to multiple insect species in the plant. In one embodiment, the double stranded RNA region comprises multiple copies of the nucleotide sequence that is complementary to the target gene. Alternatively, the dsRNA hits more than one target sequence of the same target gene. the invention thus encompasses isolated double stranded RNA constructs comprising at least two copies of said nucleotide sequence complementary to at least part of a nucleotide sequence of an insect target. DsRNA that hits more than one of the above-mentioned targets, or a combination of different dsRNA against different of the above mentioned targets are developed and used in the methods of the present invention. Suitable dsRNA nucleotides and<br><br>
dsRNA constructs arc described in WO2006/046148 by applicant, which is incorporated herein in its entirity.<br>
Ihe terms "hit", "hits" "hits" are alternative wordings to indicate that at least one of the strands of the dsRNA is complementary to, and as such may bind to, the target gene or nucleotide sequence.<br>
The modulatory effect of dsRNA is applicable to a variety of genes expressed in the pests including, for example, endogenous genes responsible for cellular metabolism or cellular transformation, including house keeping genes, transcription factors, and other genes which encode polypeptides involved in cellular metabolism.<br>
As used herein, the phrase 'Inhibition of gene expression" or 'Inhibiting expression of a target gene in the cell of an pest' refers to the absence (or observable decrease) in the level of protein and/or mRNA product from the target gene. Specificity refers to the ability to inhibit the target gene without manifest effects on other genes of the cell and without any effects on any gene within the cell that is producing the dsRNA molecule. Ihe inhibition of gene expression of the target gene in the pest may result in novel phenotypic traits in the pest.<br>
'Gene suppression" refers to any of the well-known methods for reducing the levels of gene transcription to mRNA and/or subsequent translation of the mRNA. Gene suppression is also intended to mean the reduction of protein expression from a gene or a coding sequence including posttranscriptional gene suppression and transcriptional suppression, Posttranscriptional gene suppression is mediated by the homology between of all or a part of a mRNA transcribed from a gene or coding sequence targeted for suppression and the corresponding double stranded RNA used for suppression, and refers to the substantial and measurable reduction of the amount of available mRNA available in the cell for binding by ribosomes. The transcribed RNA can be in the sense orientation to effect what is called co-suppression, in the anti-sense orientation to effect what is called anti-sense suppression, or in both orientations producing a dsRNA to effect what is called RNA interference (RNAi).<br>
I'ranscriptional suppression is mediated by the presence in the cell of a dsRNA gene suppression agent exhibiting substantial sequence identity to a promoter DNA sequence or the complement thereof to effect what is referred to as promoter trans suppression. Gene suppression may be effective against a native host gene associated with a trait, e.g., to provide hosts with reduced levels of a protein encoded by the native gene or with enhanced or reduced levels of an affected metabolite. Gene suppression can also be effective against target genes in pests that may ingest or contact material containing gene suppression agents.<br><br>
specifically designed to inhibit or suppress the expression of one or more homologous or complementary sequences in the cells of the pest.<br>
A beneficial method of post transcriptional gene suppression in hosts employs both sense-oriented and anti-sensc-oriented, transcribed RNA which is stabilized, e,g,, as a hairpin and stem and loop structure. A preferred DNA construct for effecting post transcriptional gene suppression is one in which a first segment encodes an RNA exhibiting an anti-sense orientation exhibiting substantial identity to a segment of a gene targeted for suppression, which is linked to a second segment in sense orientation encoding an RNA exhibiting substantial complementarity to the first segment. Such a construct forms a stem and loop structure by hybridization of the first segment with the second segment and a loop structure from the nucleotide sequences linking the two segments (see WO94/0I550, WO98/05770, US 2002/0048814, and US 2003/0018993).<br>
According to one embodiment of the present invention, there is provided a nucleotide sequence, for which in vitro expression results in transcription of a stabilized RNA sequence that is substantially homologous to an RNA molecule of a targeted gene in a pest that comprises an RNA sequence encoded by a nucleotide sequence within the genome of the pest. I hus, after the pest uptakes the stabilized RNA sequence, or is otherwise exposed to the dsRNA, a down-regulation of the nucleotide sequence corresponding to the target gene in the cells of a target pest is affected.<br>
Inhibition of a target gene using the stabilized dsRNA technology of the present invention is sequence-specific in that nucleotide sequences corresponding to the duplex region of the RNA are targeted for genetic inhibition. RNA containing a nucleotide sequences identical to a portion of the target gene is preferred for inhibition. RNA sequences with insertions, deletions, and single point mutations relative to the target sequence have also been found to be effective for inhibition. In performance of the present invention, it is preferred that the inhibitory dsRNA and the portion of the target gene share at least from about 80% sequence identity, or from about 85% sequence identity, or from about 90% sequence identity, or from about 95% sequence identity, or from about 99% sequence identity, or even about 100% sequence identity. Alternatively, the duplex region of the RNA may be defined functionally as a nucleotide sequence that is capable of hybridizing with a portion of the target gene transcript. A less than full length sequence exhibiting a greater homology compensates for a longer less homologous sequence. The length of the identical nucleotide sequences may be at least about 25, 50, 100, 200, 300, 400, 500 or at least about 1000 bases. Normally, a sequence of greater than 20-100 nucleotides should be used, though<br><br>
a sequence of greater than about 200-300 nucleotides would be preferred, and a sequence of greater than about 500-1000 nucleotides would be especially preferred depending on the si/e of the target gene. 1 he invention has the advantage of being able to tolerate sequence variations that might be expected due to genetic mutation, strain polymorphism, or evolutionary divergence, The introduced nucleic acid molecule may not need to be absolute homology, may not need to be full length, relative to cither the primary transcription product or fully processed mRNA of the target gene. Therefore, those skilled in the art need to realize that, as disclosed herein, 100% sequence identity between the RNA and the target gene is not required to practice the present invention.<br>
IV.	Methods for preparing dsRNA<br>
dsRNA molecules may be synthesized either in vivo or in vitro, fhe dsRNA may be formed by a single self-complementary RNA strand or from two complementary RNA strands. Endogenous RNA polymerase of the cell may mediate transcription in vivo, or cloned RNA polymerase can be used for transcription in vivo or in vitro. Inhibition may be targeted by specific transcription in an organ, tissue, or cell type; stimulation of an environmental condition (e.g., infection, stress, temperature, chemical inducers); and/or engineering transcription at a developmental stage or age. The RNA strands may or may not be polyadenylated; the RNA strands may or may not be capable of being translated into a polypeptide by a cell's translational apparatus.<br>
A RNA, dsRNA, siRNA, or miRNA of the present invention may be produced chemically or enzymatically by one skilled in the art through manual or automated reactions or in vivo in another organism. RNA may also be produced by partial or total organic synthesis; any modified ribonucleotide can be introduced by in vitro enzymatic or organic synthesis, fhe RNA may be synthesized by a cellular RNA polymerase or a bacteriophage RNA polymerase (e.g., T3, T7, SP6). 'fhe use and production of an expression construct are known in the art (see, for example, WO 97/32016; U.S. Pat. No's. 5,593, 874, 5,698,425, 5,712,135, 5,789,214, and 5,804,693). If synthesized chemically or by in vitro enzymatic synthesis, the RNA may be purified prior to introduction into the cell. For example, RNA can be purified from a mixture by extraction with a solvent or resin, precipitation, electrophoresis, chromatography, or a combination thereof Alternatively, the RNA may be used with no or a minimum of purification to avoid losses due to sample processing, fhe RNA may be dried for storage or dissolved in an aqueous solution. The solution may contain buffers or salts to promote annealing, and/or stabilization of the duplex strands.<br>
V.	Polynucleotide Sequences<br><br>
Provided according to the invention are nucleotide sequences, the expression of which results in an RNA sequence which is substantially homologous to an RNA molecule of a targeted gene in a pest that comprises an RNA sequence encoded by a nucleotide sequence within the genome of the pest. Thus, after ingestion of the dsRNA sequence down-regulation of the nucleotide sequence of the target gene in the cells of the pest may be obtained resulting in a deleterious effect on the maintenance, viability, proliferation, reproduction, and infestation of the pest.<br>
Mach 'nucleotide sequence" set forth herein is presented as a sequence of deoxyribonucleotides (abbreviated A, G, C and 1). However, by 'nucleotide sequence' of a nucleic acid molecule or polynucleotide is intended, for a DNA molecule or polynucleotide, a sequence of deoxyribonucleotides, and for an RNA molecule or polynucleotide, the corresponding sequence of ribonucleotides (A, G, C and U) where each thymidine deoxynucleotide (1) in the specified deoxynucleotide sequence in is replaced by the ribonucleotide uridine (U).<br>
As used herein, "nucleic acid" refers to a single or double-stranded polymer of deoxyribonucleotidc or ribonucleotide bases read from the 5' to the 3' end. A nucleic acid may also optionally contain non-naturally occurring or altered nucleotide bases that permit correct read through by a polymerase and do not reduce expression of a polypeptide encoded by that nucleic acid. ^'Nucleotide sequence' or 'nucleic acid sequence" refers to both the sense and antiscnsc strands of a nucleic acid as either individual single strands or in the duplex.<br>
The term "ribonucleic acid" (RNA) is inclusive of RNAi (inhibitory RNA), dsRNA (double stranded RNA), siRNA (small interfering RNA), mRNA (messenger RNA), miRNA (micro-RNA), IRNA (transfer RNA, whether charged or discharged with a corresponding acylated amino acid), and cRNA (complementary RNA) and the term 'deoxyribonucleic acid" (DNA) is inclusive of cDNA and genomic DNA and DNA-RNA hybrids.<br>
The words 'nucleic acid segment", 'nucleotide sequence segment", or more generally "segment" will be understood by those in the art as a functional term that includes both genomic sequences, ribosomal RNA sequences, transfer RNA sequences, messenger RNA sequences, operon sequences and smaller engineered nucleotide sequences that express or may be adapted to express, proteins, polypeptides or peptides.<br>
Accordingly, the present invention relates to an isolated nucleic molecule comprising a polynucleotide having a sequence selected  from the group consisting of any of the<br><br>
polynucleotide sequences of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 5 17, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1 161 -1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349. 2354. 2359, 2364, 2366, 2368, 2370, 2372, 2384 - 2460, 2461, 2466, 2471, 2476 and 2481. The invention also provides functional fragments of the polynucleotide sequences of SEQ ID NOs: 1,3,5,7,9, 11, 13, 15, 17, 19,21,23,49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097. 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481. The invention further provides complementary nucleic acids, or fragments thereof, to any of the polynucleotide sequences ofSl-0 ID NOs: 1,3, 5, 7,9, 11, 13, 15, 17, 19,21,23,49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257. 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795. 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647,<br><br>
1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039. 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085. 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366. 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481, as well as a nucleic acid, comprising at least 15 contiguous bases, which hybridizes to any of the polynucleotide sequences of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168. 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257. 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 5 15, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773. 778, 783, 788. 793, 795, 797, 799, 801, 813 - 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1 161 -1571. 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481.<br>
Ihe present invention also provides orthologous sequences, and complements and fragments thereof, of the polynucleotide sequences of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091. 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, nil, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055. 2060, 2065, 2070, 2075. 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120<br>
2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384	2460, 2461,<br>
2466, 2471. 2476 and 2481 of the invention. Accordingly, the invention encompasses target genes which are insect orthologs of a gene comprising a nucleotide sequence as represented in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19,21,23,49  158, 159, 160, 163, 168,<br><br>
173. 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581. 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093. 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481. By way of example, insect orlhologues may comprise a nucleotide sequence as represented in any of Si-;Q ID NOs: 49-123, 275-434, 533-562, 621-738, 813-852, 908-1010, 1161-1437, 1730-1987, 2120-2290, 2384-2438, or a fragment thereof of at least 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26 or 27 nucleotides . A non-limiting list of insect or arachnida orthologs genes or sequences comprising at least a fragment of 15, preferably at least 17 bp of one of the sequences of the invention is given in Tables 4.<br>
The invention also encompasses target genes which arc nematode orthologs of a gene comprising a nucleotide sequence as represented in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 111l, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686,<br>
1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730	 2039, 2040, 2045, 2050,<br>
2055, 2060. 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476, and 2481 of the invention. By way of example, nematode orthologs may comprise a nucleotide sequence as represented in any of SBQ ID NOs: 124-135, 435-446, 563, 564, 739-751, 853, 854, 1011-1025, 1438-1473, 1988-2001, 2291-2298, 2439-2440 of the invention, or a fragment of at least 15, 16, 17, 18, 19, 20 or 21 nucleotides thereof. According<br><br>
lo another aspect, the invention thus encompasses any of the methods described herein for controlling nematode growth in an organism, or for preventing nematode infestation of an organism susceptible to nemataodc infection, comprising contacting nematode cells with a double-stranded RNA, wherein the double-stranded RNA comprises annealed complementary strands, one of which has a nucleotide sequence which is complementary to at least part of the nucleotide sequence of a target gene comprising a fragment of at least 17, 18, 19, 20 or 21 nucleotides of any of the sequences as represented in SEQ IDNOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19,21,23,49- 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198,203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 111 I, 1113, 1161 - 1571, 1572, 1577, 15S2, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481, whereby the double-stranded RNA is taken up by the fungus and thereby controls growth or prevents infestation. The invention also relates to nematode-resistant transgenic plants comprising a fragment of at least 17, 18, 19, 20 or 21 nucleotides of any of the sequences as represented in SHQ ID NOs: I, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220. 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890. 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079. 1081. 1083. 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109. nil. 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055. 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481. A non-limiting list of nematode orthologs genes or sequences<br><br>
comprising at least a rragmcnl of 15, preferably at least 17 bp of one of the sequences of the invention is given in fables 5.<br>
According to another embodiment, the invention encompasses target genes which arc fungal orthologs of a gene comprising a nucleotide sequence as represented in any of SEQ II) N():s I, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, nil, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104. 2106. 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481 of the invention. By way of example, fungal orthologs may comprise a nucleotide sequence as represented in any of SHQ ID NOs:136-158, 447-472, 565-575, 752-767, 855-862, 1026-1040, 1474-1571, 2002-2039, 2299-2338, 2441-2460, or a fragment of at least 17, 18, 19, 20, 21, 22, 23, 24, 25, 26 or 27 nucleotides thereof.. According to another aspect, the invention thus encompasses any of the methods described herein for controlling fungal growth on a cell or an organism, or Ibr preventing fungal infestation of a cell or an organism susceptible to fungal infection, comprising contacting fungal cells with a double-stranded RNA, wherein the double-stranded RNA comprises annealed complementary strands, one of which has a nucleotide sequence which is complementary to at least part of the nucleotide sequence of a target gene comprising a fragment of at least 17, 18, 19, 20 or 21 nucleotides of any of the sequences as represented in SliQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 - 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1 161 - 1571, 1572, 1577. 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647,<br><br>
1652, 1657. 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085. 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364. 2366. 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481, whereby the double-stranded RNA is taken up by the fungus and thereby controls growth or prevents infestation, the invention also relates to fungal-resistant transgenic plants comprising a fragment of at least 17, 18, 19, 20 or 21 of any of the sequences as represented in S1;Q IDNOS: 1, 3, 5,7,9, 11, 13, 15, 17, 19, 21, 23, 49 - 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581. 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046. 1051. 1056, 1061, 1071. 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481. A non-limiting list of fungal orthologs genes or sequences comprising at least a fragment of 15, preferably at least 17 bp of one of the sequences of the invention is given in Tables 6.<br>
In a further embodiment, a dsRNA molecule of the invention comprises any of SHQ IDNOs: 1,3,5,7,9, 11, 13, 15, 17, 19,21,23,49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 -862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102. 2104. 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370. 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481, though the sequences set forth in<br><br>
SEQ ID NOs: 1,3,5,7, 9, II, 13, 15, 17, 19,21,23,49 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, nil, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481 are not limiting. A dsRNA molecule of the invention can comprise any contiguous target gene from a pest species (e.g., about 15 to about 25 or more, or about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more contiguous nucleotides).<br>
By 'isolated' nucleic acid molecule(s) is intended a nucleic acid molecule, DNA or RNA, which has been removed from its native environment. For example, recombinant DNA molecules contained in a DNA construct are considered isolated for the purposes of the present invention, further examples of isolated DNA molecules include recombinant DNA molecules maintained in heterologous host cells or purified (partially or substantially) DNA molecules in solution. Isolated RNA molecules include in vitro RNA transcripts of the DNA molecules of the present invention. Isolated nucleic acid molecules, according to the present invention, further include such molecules produced synthetically.<br>
Nucleic acid molecules of the present invention may be in the form of RNA, such as mRNA, or in the form of DNA, including, for instance, cDNA and genomic DNA obtained by cloning or produced synthetically. The DNA or RNA may be double-stranded or single-stranded. Single-stranded DNA may be the coding strand, also known as the sense strand, or it may be the non-coding strand, also referred to as the anti-sense strand.<br>
VI.       Sequence Analysis<br>
Unless otherwise indicated, all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer (such as the Model 373 from Applied Biosystems, Inc.). Therefore, as is known in the art for any DNA sequence determined by this automated approach, any nucleotide sequence determined herein<br><br>
may contain some errors. Nucleotide sequences determined by automation are typically at least about 95% identical, more typically at least about 96% to at least about 99.9% identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence can be more precisely determined by other approaches including manual DNA sequencing methods well known in the art. As is also known in the art, a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence may be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule, beginning at the point of such an insertion or deletion.<br>
In another aspect, the invention provides an isolated nucleic acid molecule comprising a polynucleotide which hybridizes under stringent hybridization conditions to a portion of the polynucleotide in a nucleic acid molecule of the invention described above. By a polynucleotide which hybridizes lo a 'portion' of a polynucleotide is intended a polynucleotide (either DNA or RNA) hybridizing to at least about 15 nucleotides, and more preferably at least about 20 nucleotides, and still more preferably at least about 30 nucleotides, and even more preferably more than 30 nucleotides of the reference polynucleotide, fhcse fragments that hybridize to the reference fragments are useful as diagnostic probes and primers. For the purpose of the invention, two sequences hybridize when they form a double-stranded complex in a hybridization solution of 6X SSC, 0.5% SDS, 5X Dcnhardt's solution and 100µg of non-specific carrier DNA. See Ausubel et ai, section 2.9, supplement 27 (1994). Sequences may hybridize at "moderate stringency,' which is defined as a temperature of 60 °C in a hybridization solution of 6X SSC, 0.5% SDS, 5X Denhardt's solution and 100µg of non-specific carrier DNA. For "high stringency" hybridization, the temperature is increased to 68 °C. Following the moderate stringency hybridization reaction, the nucleotides are washed in a solution of 2X SSC plus 0.05% SDS for five times at room temperature, with subsequent washes with O.IX SSC plus 0.1% SDS at 60 °C for Ih. For high stringency, the wash temperature is increased to 68 °C. For the purpose of the invention, hybridized nucleotides are those that are detected using 1 ng of a radiolabeled probe having a specific radioactivity of 10,000 cpm/ng, where the hybridized nucleotides are clearly visible following exposure to X-ray film at -70 °C for no more than 72 hours.<br>
1 he present application is directed lo such nucleic acid molecules which are at least 60 %, 65 %, 70 %, 75 %&gt;, 80 %, 85 %, 90%, 95%, 96%, 97%, 98%, 99% or 100% identical to<br><br>
a nucleic acid sequence described in any of SEQ ID NOs; 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 158, 159. 160. 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083. 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632. 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 - 2460, 2461, 2466, 2471, 2476 and 2481, Preferred, however, are nucleic acid molecules which are at least 95%, 96%, 97%, 98%, 99% or 100% identical to the nucleic acid sequence shown in of SBQ ID NOs: 1,3,5,7,9, 11. 13, 15, 17, 19,21,23,49 158. 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203. 208, 215. 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478. 483, 488. 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 767, 768, 773, 778, 783. 788. 793, 795, 797, 799, 801, 813 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073. 1075, 1077, 1079, 1081, 1083. 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, nil, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612. 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039. 2040. 2045. 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102. 2104, 2106, 2)08. 2120 2338, 2339, 2344, 2349. 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481. Differences between two nucleic acid sequences may occur at the 5' or 3' terminal positions of the reference nucleotide sequence or anywhere between those terminal positions, interspersed either individually among nucleotides in the reference sequence or in one or more contiguous groups within the reference sequence.<br>
As a practical matter, whether any particular nucleic acid molecule is at least 95%, 96%, 97%. 98% or 99% identical to a reference nucleotide sequence refers to a comparison made between two molecules using standard algorithms well known in the art and can be<br><br>
determined conventionally using publicly available computer programs such as the BLASTN algorithm. See Altschul et al.. Nucleic Acids Res, 25:3389-3402 (1997).<br>
In one embodiment of the invention, a nucleic acid comprises an antiscnsc strand having about 15 to about 30 (e.g., about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30) nucleotides, wherein the antiscnsc strand is complementary to a RNA sequence or a portion thereof encoding a protein that controls cell cycle or homologous recombination, and wherein said siNA further comprises a sense strand having about 15 to about 30 (e.g., about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30) nucleotides, and wherein said sense strand and said antisense strand are distinct nucleotide sequences where at least about 15 nucleotides in each strand are complementary to the other strand.<br>
In one embodiment, the present invention provides double-stranded nucleic acid molecules of that mediate RNA interference gene silencing. In another embodiment, the siNA molecules of the invention consist of duplex nucleic acid molecules containing about 15 to about 30 base pairs between oligonucleotides comprising about 15 to about 30 (e.g., about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30) nucleotides. In yet another embodiment, siNA molecules of the invention comprise duplex nucleic acid molecules with overhanging ends of about 1 to about 32 (e.g., about 1, 2, or 3) nucleotides, for example, about 21-nucleotide duplexes with about 19 base pairs and 3'-terminal mononucleotide, dinucleotide, or trinucleotide overhangs. In yet another embodiment, siNA molecules of the invention comprise duplex nucleic acid molecules with blunt ends, where both ends are blunt, or alternatively, where one of the ends is blunt.<br>
An siNA molecule of the present invention may comprise modified nucleotides while maintaining the ability to mediate RNAi. 'fhe modified nucleotides can be used to improve in vitro or in vivo characteristics such as stability, activity, and/or bioavailability. For example, a siNA molecule of the invention can comprise modified nucleotides as a percentage of the total number of nucleotides present in the siNA molecule. As such, a siNA molecule of the invention can generally comprise about 5% to about 100% modified nucleotides (e.g., about 5%, 10%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95% or 100% modified nucleotides). The actual percentage of modified nucleotides present in a given siNA  ecule will depend on the total number of nucleotides present in the siNA. If the siNA molecule is single stranded, the percent modification can be based upon the total number of nucleotides present in the single stranded siNA molecules. Likewise, if the siNA molecule is double stranded, the percent modification can be based<br><br>
upon the total number of nucleotides present in the sense strand, antisensc strand, or both the sense and antisensc strands.<br>
VII     Nucleic Acid Constructs<br>
A recombinant nucleic acid vector may, for example, be a linear or a closed circular plasmid. The vector system may be a single vector or plasmid or two or more vectors or plasmids that together contain the total nucleic acid to be introduced into the genome of the bacterial host. In addition, a bacterial vector may be an expression vector. Nucleic acid molecules as set forth in SHQ ID NOs: 1,3,5,7,9, 11, 13, 15, 17, 19,21,23,49 158, 159, 160, 16L 162, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 240 246, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 508-512, 513,515, 517, 519, 521, 533 -575, 576, 581, 586, 591, 596,601, 603,605,607,609,621 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 - 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1066-1070, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, nil, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617. 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 2460, 2461, 2466, 2471, 2476 and 2481, or fragments thereof can, for example, be suitably inserted into a vector under the control of a suitable promoter that functions in one or more microbial hosts to drive expression of a linked coding sequence or other DNA sequence. Many vectors arc available for this purpose, and selection of the appropriate vector will depend mainly on the size of the nucleic acid to be inserted into the vector and the particular host cell to be transformed with the vector. Each vector contains various components depending on its function (amplification of DNA or expression of DNA) and the particular host cell with which it is compatible. The vector components for bacterial transformation generally include, but are not limited to, one or more of the following: a signal sequence, an origin of replication, one or more selectable marker genes, and an inducible promoter allowing the expression of exogenous DNA.<br>
Promoters<br>
"Operably linked", as used in reference to a regulatory sequence and a structural nucleotide sequence, means that the regulatory sequence causes regulated expression of the<br><br>
linked structural nucleotide sequence. 'Regulatory sequences" or 'control elements" refer to nucleotide sequences located upstream (5' noncoding sequences), within, or downstream (3' non-translated sequences) of a structural nucleotide sequence, and which influence the timing and level or amount of transcription, RNA processing or stability, or translation of the associated structural nucleotide sequence. Regulatory sequences may include promoters, translation leader sequences, introns, enhancers, stem-loop structures, repressor binding sequences, and polyadenylation recognition sequences and the like.<br>
An expression vector for producing a mRNA can also contain an inducible promoter that is recognized by the host bacterial organism and is operably linked to the nucleic acid encoding, for example, the nucleic acid molecule coding the D. v. virgifera mRNA or fragment thereof of interest. Inducible promoters suitable for use with bacterial hosts include p-lactamasc promoter, E. coli X phage PL and PR promoters, and E. coli galactose promoter, arabinose promoter, alkaline phosphatase promoter, tryptophan (trp) promoter, and the lactose operon promoter and variations thereof and hybrid promoters such as the tac promoter. I lowever, other known bacterial inducible promoters are suitable.<br>
In certain embodiments, the genes can be derived from different insects in order to broaden the range of insects against which the agent is effective. When multiple genes are targeted lor suppression or a combination of expression and suppression, a polycistronic DNA clement can be fabricated as illustrated and disclosed in Fillatti, Application Publication No. US 2004-0029283.<br>
Selectable marker genes<br>
A recombinant DNA vector or construct of the present invention will typically comprise a selectable marker that confers a selectable phcnotype on transformed cells. Selectable markers may also be used to select for cells that contain the exogenous nucleic acids encoding polypeptides or proteins of the present invention. The marker may encode biocide resistance, such as antibiotic resistance (e.g., kanamycin, G418 bleomycin, hygromyein, etc.). Examples of selectable markers include, but are not limited to, a neo gene which codes for kanamycin resistance and can be selected for using kanamycin, G418, eic, a bar gene which codes for bialaphos resistance; a nitrilase gene which confers resistance to bromoxynil, and a methotrexate resistant DHFR gene. Hxamples of such selectable markers are illustrated in U.S. Patents 5,550,318; 5,633,435; 5,780,708 and 6,118,047.<br>
A recombinant vector or construct of the present invention may also include a screenablc marker, Screenable markers may be used to monitor expression. [Exemplary screenable markers include a p-glucuronidasc or uidA gene (GUS) which encodes an cn/yme<br><br>
lor which various chromogcnic substrates arc known (Jefferson, 1987; Jefferson el al. 1987); a p-lactamasc gene (Sutcliffe et ai, 1978), a gene which encodes an en/yme for which various chromogcnic substrates arc known (e.g., PADAC, a chromogcnic cephalosporin); a luciferasc gene (Ow et al,, 1986) a xylH gene (Zukowsky et ai, 1983) which encodes a catechol dioxygcnasc that can convert chromogcnic catechols; an a-amylase gene (Ikatu el ai. 1990); a tyrosinase gene (Katz el ai, 1983) which encodes an enzyme capable of oxidizing tyrosine to DOPA and dopaquinone which in turn condenses to melanin; an a -galactosidase, which catalyzes a chromogcnic a-galactose substrate.<br>
A transformation vector can be readily prepared using methods available in the art. The transformation vector comprises one or more nucleotide sequences that is/are capable of being transcribed to an RNA molecule and that is/are substantially homologous and/or complementary to one or more nucleotide sequences encoded by the genome of the insect, such that upon uptake of the RNA there is down-regulation of expression of at least one of the respective nucleotide sequences of the genome of the pest.<br>
VIIl.   Methods for Genetic Engineering<br>
The present invention contemplates introduction of a nucleotide sequence into a organism to achieve pest inhibitory levels of expression of one or more dsRNA molecules. The inventive polynucleotides and polypeptides may be introduced into a host cell, such as bacterial or yeast cell, by standard procedures known in the art for introducing recombinant sequences into a target host cell. Such procedures include, but are not limited to, transaction, infection, transformation, natural uptake, calcium phosphate, electroporation, microinjection biolistics and microorganism-mediated transformation protocols. The methods chosen vary with the host organism.<br>
A transgenic organism of the present invention is one that comprises at least one cell it its genome in which an exogenous nucleic acid has been stably integrated. Thus, a transgenic organism may contain only genetically modified cells in certain parts of its structure.<br>
Accordingly, the present invention also encompasses a transgenic cell or organism comprising any of the nucleotide sequences or recombinant DNA constructs described herein. The invention further encompasses prokaryotic cells (such as, but not limited to, gram-positive and gram-negative bacterial cells) and cukaryotic cells (such as, but not limited to, yeast cells or plant cells).<br>
For example, the present invention contemplates introducing a target gene into a bacterium, such as Lactobacillus.    The nucleic acid constructs can be integrated into a<br><br>
bacterial genome with an integrating vector. Integrating vectors typically contain at least one sequence homologous to the bacterial chromosome that allows the vector to integrate. Integrations appear to result from recombinations between homologous DNA in the vector and the bacterial chromosome. For example, integrating vectors constructed with DNA from various Bacillus strains integrate into the Bacillus chromosome (HP 0 127,328). Integrating vectors may also be comprised of bacteriophage or transposon sequences. Suicide vectors are also known in the art.<br>
Construction of suitable vectors containing one or more of the above-listed components employs standard recombinant DNA techniques. Isolated plasmids or DNA Iragments are cleaved, tailored, and re-ligated in the form desired to generate the plasmids required. Examples of available bacterial expression vectors include, but are not limited to, the multifunctional E. colt cloning and expression vectors such as Bluescripf*'^ (Stratagene, La Jolla, CA), in which, for example, a D. v. virgifera protein or fragment thereof, may be ligated into the vector in frame with sequences for the amino-terminal Met and the subsequent 7 residues of p-galactosidase so that a hybrid protein is produced; plN vectors (Van lleekc and Schuster, 1989); and the like.<br>
ITe invention also contemplates introducing a target gene into a yeast cell. A yeast recombinant construct can typically include one or more of the following; a promoter sequence, fusion partner sequence, leader sequence, transcription termination sequence, a selectable marker, fhese elements can be combined into an expression cassette, which may be maintained in a replicon, such as an extrachromosomal clement (e.g., plasmids) capable of stable maintenance in a host, such as yeast or bacteria. I he replicon may have two replication systems, thus allowing it to be maintained, for example, in yeast for expression and in a prokaryolic host for cloning and amplification. Hxamples of such yeast-bacteria shuttle vectors include Yf:p24 (Botstein et ai, 1979), pCl/1 (Brake et al., 1984), and YRpl7 (Stinchcomb el al, 1982). In addition, a replicon may be either a high or low copy number plasmid. A high copy number plasmid will generally have a copy number ranging from about 5 to about 200, and typically about 10 to about 150. A host containing a high copy number plasmid will preferably have at least about 10, and more preferably at least about 20.<br>
Useful yeast promoter sequences can be derived from genes encoding cn/ymes in the metabolic pathway. Examples of such genes include alcohol dehydrogenase (ADH) (HP 0 284044), enolase, glucokinase, glucosc-6-phosphate isomerase, glyceraldehyde-3-phosphate-dehydrogenase (GAP or GAPDH), hexokinase, phosphofruetokinase, 3-phosphoglycerate mutase, and pyruvate kinase (PyK) (HP 0 3215447).   The yeast PHO5 gene, encoding acid<br><br>
phosphatase, also provides useful promoter sequences (Myanohara ei al., 1983), In addition, synthetic promoters that do not occur in nature also function as yeast promoters, Hxamples of such hybrid promoters include the ADH regulatory sequence linked to the GAP transcription activation region (U.S. Patent No. 4,876,197 and 4,880,734). Examples of transcription terminator sequences and other ycast-recogni/ed termination sequences, such as those coding for glycolytic enzymes, are known to those of skill in the art.<br>
Alternatively, the expression constructs can be integrated into the yeast genome with an integrating vector. Integrating vectors typically contain at least one sequence homologous to a yeast chromosome that allows the vector to integrate, and preferably contain two homologous sequences flanking the expression construct. Integrations appear to result from recombinations between homologous DNA in the vector and the yeast chromosome (Orr-Weaver et ai, 1983). An integrating vector may be directed to a specific locus in yeast by selecting the appropriate homologous sequence for inclusion in the vector. See Orr-Weaver el al, supra. One or more expression constructs may integrate, possibly affecting levels of recombinant protein produced (Rinc et ai, 1983).<br>
IX,       Quantifying inhibition of target gene expression<br>
Inhibition of target gene expression may be quantified by measuring either the endogenous target RNA or the protein produced by translation of the target RNA and the consequences of inhibition can be confirmed by examination of the outward properties of the cell or organism, fechniques for quantifying RNA and proteins are well known to one of ordinary skill in the art. Multiple selectable markers are available that confer resistance to ampicillin, bleomycin, chloramphenicol, gentamycin, hygromycin, kanamycin, lincomycin, methotrexate, phosphinothricin, puromycin, spectinomycin, rifampicin, and tetracyclin, and the like.<br>
In certain embodiments gene expression is inhibited by at least 10%, preferably by at least 33%, more preferably by at least 50%, and yet more preferably by at least 80%. In particularly preferred embodiments of the invention gene expression is inhibited by at least 80%, more preferably by at least 90%, more preferably by at least 95%, or by at least 99% within cells in the pest so a significant inhibition takes place. Significant inhibition is intended to refer to sufficient inhibition that results in a detectable phenotype (e.g., cessation of larval growth, paralysis or mortality, etc.) or a detectable decrease in RNA and/or protein corresponding to the target gene being inhibited. Although in certain embodiments of the invention   inhibition   occurs   in   substantially   all   cells  of the   pest,   in   other  preferred<br><br>
embodiments inhibition occurs in only a subset of cells expressing the gene. For example, if the target gene plays an essential role in cells in an insect alimentary tract, inhibition of the gene within these cells is sufficient to exert a deleterious effect on the insect.<br>
X. Exposing pest to dsRNA<br>
A pest can be exposed to a dsRNA in any suitable manner that permits administering the dsRNA to the pest. For example, the pest can be contacted with the dsRNA in pure or substantially pure form, for example an aqueous solution containing the dsRNA. In one embodiment, the insect may be simply "soaked" or "sprayed' with an aqueous solution comprising the dsRNA. Alternatively, the pest may be 'sprayed" with a solution comprising a dsRNA.<br>
Alternatively, the dsRNA may be linked to a food component of the pest, such as a food component for a mammalian pathogenic pest, in order to increase uptake of the dsRNA by the insect. Ingestion by a pest permits delivery of the pest control agents to the pest and results in down-regulation of a target gene in the host. Methods for oral introduction may include, for example, directly mixing dsRNA with a, pest's food, as well as engineered approaches in which a species that is used as food is engineered to express the dsRNA or siRNA, then fed to the pest to be affected. For example, a bacteria, such as Lactobacillus, may be transformed with a target sequence and then fed to a pest. In one embodiment, for example, the dsRNA or siRNA molecules may be incorporated into, or overlaid on the top of the insect's diet.<br>
In other embodiments the pest may be contacted with a composition containing the inventive dsRNA. The composition may, in addition to the dsRNA, contain further excipients, diluents, or carriers.<br>
Ihe dsRNA may also be incorporated in the medium in which the pest grows or infests. I'or example, a dsRNA may be incorporated into a food container or protective wrapping as a means for inhibiting pest infestation. Wood, for example, may be treated with a solution comprising a dsRNA to prevent pest infestation.<br>
In other embodiments, the dsRNA is expressed in a bacterial or fungal cell and the bacterial or fungal cell is taken up or eaten by the insect species.<br>
As illustrated in the examples, bacteria can be engineered to produce any of the dsRNA or dsRNA constructs of the invention. These bacteria can be eaten by the insect species. When taken up, the dsRNA can initiate an RNAi response, leading to the degradation of the target mRNA and weakening or killing of the feeding insect. Alternatively, dsRNA producing bacteria or yeast cells can be sprayed directly onto the crops.<br><br>
Some bacteria have a very close interaction with the host plant, such as, but not limited to, symbiotic Rhizohium with the Legminosea (for example Soy), Such recombinant bacteria could be mixed with the seeds (for instance as a coating) and used as soil improvers.<br>
A virus such as a baculovirus which specifically infects insects may be also be used. This ensures safety for mammals, especially humans, since the virus will not infect the mammal, so no unwanted RNAi effect will occur..<br>
Possible applications include intensive greenhouse cultures, for instance crops that arc less interesting from a GMO point of view, as well as broader field crops such as soy.<br>
rhis approach has several advantages, eg: since the problem of possible dicing by a plant host is not present, it allows the delivery of large dsRNA fragments into the gut lumen of the feeding pest; the use of bacteria as insecticides does not involve the generation of transgenic crops, especially for certain crops where transgenic variants are difficult to obtain; there is a broad and flexible application in that different crops can be simultaneously treated on the same field and/or different pests can be simultaneously targeted, for instance by combining different bacteria producing distinct dsRNAs.<br>
XL Products<br>
fhe present invention provides numerous products that can encompass a dsRNA for use in controlling pests. For example, the invention provides pharmaceutical or veterinary compositions for treating or preventing a pest disease or infection of humans or animals, respectively. Such compositions comprise at least one dsRNA or FWA construct, or nucleotide sequence or recombinant DNA construct encoding the dsRNA or RNA construct, wherein the RNA comprises annealed complementary strands, one of which has a nucleotide sequence which corresponds to a target nucleotide sequence of an pest target gene that causes the disease or infection, and at least one carrier, excipient, or diluent suitable for pharmaceutical use.<br>
Alternatively, a pharmaceutical or veterinary composition may be used as a composition suitable for topical use, such as application on the skin of an animal or human, for example, a dsRNA may be used in a liquid composition to be applied to the skin as drops, gel, aerosol, cream, ointment, etc. Additionally, a dsRNA may be integrated into a transdermal patch or other medical device for treating or preventing a disease or condition. Other conventional pharmaceutical dosage forms may also be produced, including tablets, capsules, pessaries, transdermal patches, suppositories, etc. Ihe chosen form will depend upon the nature of the target pest and hence the nature of the disease it is desired to treat.<br><br>
Oral vaccines, for example, can be produced using the inventive constructs and methods. For example, a vaccine can be constructed by producing a dsRNA in bacteria (e.g. lactohacillus) which can be included in food and functions as an oral vaccine against insect infection. Accordingly, the invention provides constructs and methods for treating and/or preventing a pest disease or condition, comprising administering to a subject in need of such treatment and/or prevention, any of the compositions as herein described, said composition comprising at least one double-stranded RNA or double stranded RNA construct comprising annealed complementary strands, one of which has a nucleotide sequence which is complementary to at least part of a nucleotide sequence of a pest target gene that causes the disease or condition.<br>
While the inventive compositions may be used for treating a disease or condition in a subject patient, the compositions and methods may also be used as a means for protecting a substrate or material from pest infestation. The nature of the excipients included in the composition and the physical form of the composition may vary depending upon the nature of the substrate that it is desired to treat.<br>
!'\)r example, such a composition may be a coaling or a powder that can be applied to a substrate as a means for protecting the substrate from infestation by an insect and thereby preventing pest-induced damage to the substrate or material, fhus, in one embodiment, the composition is in the form of a coating on a suitable surface which adheres to, and is eventually ingested by an insect which comes into contact with the coating. Such a composition can be used to protect any substrate or material that is susceptible to infestation by or damage caused by a pest, for example foodstuffs and other perishable materials, and substrates such as wood.<br>
For example, the composition may be a liquid that is brushed or sprayed onto or imprinted into the material or substrate to be treated. Thus, a human user can spray the insect or the substrate directly with the composition<br><br>
for example, houses and other wood products can be destroyed by termites, powder post beetles, and carpenter ants. By treating wood or house siding with a composition comprising a dsRNA, it may be possible to reduce pest infestation. likewise, a tree trunk may be treated with a composition comprising a dsRNA.<br>
Flour beetles, grain weevils, meal moths, and other pests feed on stored grain, cereals, pel food. powdered chocolate, and almost everything else in the kitchen pantry that is not protected. Accordingly, the present invention provides a means for treating cereal boxes and other food storage containers and wrapping with a composition comprising a target dsRNA.<br><br>
Larvae of clothes moths eat clothes made from animal products, such as fur, silk and wool. 1 hus, it may be desirable to treat hangers, closet organizers, and garment bags with the inventive dsRNA, Book lice and silverfish are pests of libraries because they eat the starchy glue in the bindings of books. Accordingly, the present invention provides compositions for treating books from pest infestation and destruction.<br>
In one embodiment, the composition is in the form of a bait. The bait is designed to lure the insect to come into contact with the composition. Upon coming into contact therewith, the composition is then internalized by the insect, by ingestion for example and mediates RNAi to thus kill the insect. The bait may depend on the species being targeted. An allraclant may also be used, the attractanl may be a pheromone, such as a male or female pheromonc for example. The attractant acts to lure the insect to the bait, and may be targeted for a particular insect or may attract a whole range of insects. The bait may be in any suitable form, such as a solid, paste, pellet or powdered form.<br>
The bait may also be carried away by the insect back to the colony. The bait may then act as a food source for other members of the colony, thus providing an effective control of a large number of insects and potentially an entire insect pest colony, This is an advantage associated with use of the double stranded RNA or bacteria expressing the dsRNA of the invention, because the delayed action of the RNAi mediated effects on the pests allows the bait to be carried back to the colony, thus delivering maximal impact in terms of exposure to the insects,<br>
fhe baits may be provided in a suitable "housing" or "trap". Such housings and traps arc commercially available and existing traps may be adapted to include the compositions of the invention, the housing or trap may be box-shaped for example, and may be provided in pre-formed condition or may be formed of foldable cardboard for example. Suitable materials for a housing or trap include plastics and cardboard, particularly corrugated cardboard, the inside surfaces of the traps may be lined with a sticky substance in order to restrict movement of the insect once inside the trap. The housing or trap may contain a suitable trough inside which can hold the bait in place. A trap is distinguished from a housing because the insect can not readily leave a trap following entry, whereas a housing acts as a "feeding station" which provides the insect arachnid with a preferred environment in which they can feed and feel safe from predators.<br>
It is clear that numerous products and substrates can be treated with the inventive compositions for reducing pest infestation. Of course, the nature of the excipients and the physical form of the composition may vary depending upon the nature of the substrate that is<br><br>
desired to treat, l-'or example, the composilion may be a liquid that is brushed or sprayed onto or imprinted into the material or substrate to be treated, or a coaling that is applied lo the material or substrate to be treated.<br>
***************************<br>
Specific examples are presented below of methods for identifying target sequences and introducing the sequences into various cells and compositions. They are meant to be exemplary and not as limitations on the present invention.<br>
Example 1: Silencing Celesans target genes in C. elesans in High Throughput<br>
Screening<br>
A C.  elegans  genome wide library was prepared in the pGN9A vector (WO 01/88121) between two identical TT-promoters and terminators, driving its expression in the sense and antisensc direction upon expression of the 17 polymerase, which was induced by IPTG. This library was transformed into the bacterial strain AB301-105 (DliS) in 96 well plate format, for the genome wide screening, these bacterial cells were fed lo the nuclease deficient C. elesans nuc-l(el392) strain.<br>
feeding the dsRNA produced in the bacterial strain AB3O1-105 (DH3), to C. elegans nuc-1 (cl392) worms, was performed in a 96 well plate format as follows: nuc-1 eggs were transferred to a separate plate and allowed lo hatch simultaneously at 20 °C for synchronization of the LI generation, 96 well plates were filled with 100 µL liquid growlh medium comprising IPTG and with 10 µl bacterial cell culture of ODeool AB301-105 (DK3) of the C. elegans dsRNA library carrying each a vector with a C. elegans genomic fragment for expression of the dsRNA. To each well, 4 of the synchronized LI worms were added and were incubated at 25 °C for at least 4 to 5 days. These experiments were performed in quadruplicate. In the screen 6 controls weree used:<br>
pGN29    negative control, wild type<br>
pGZl     unc-22 =-  twitcher phenotypc<br>
pG/18    chitin synthase    embryonic lethal<br>
pG/25    pos~l    embryonic lethal<br>
-	pGZ59 - hli-4D    acute lethal<br>
-	ACC    acetyl co-enzym A carboxylase    acute lethal<br>
Alter 5 days, the phenotypc of the C. elegans nuc-1 (el392) worms fed with the bacteria producing dsRNA were compared to the phenotypc of worms fed with the empty<br><br>
vector (pGN29) and the other controls. The worms that were fed with the dsRNA were screened for lethality (acute or larval) lethality for the parent (Po) generation, (embryonic) lethality for the first filial (Fl) generation, or for growth retardation of Po as follows: (i) Acute lethality of Po: LI's have not developed and arc dead, this phenotype never gives progeny and the well looks quite empty; (ii) (Larval) lethajjty of Po: Po died in a later stage than LL this phenotype also never gives progeny. Dead larvae or dead adult worms are found in the wells; (iii) Lethality_for Fl; LLs have developed until adult stage and are still alive. 1 his phenotype has no progeny. I his can be due to sterility, embryonic lethality (dead eggs on the bottom of well), embryonic arrest or larval arrest (eventually ends up being lethal): (iv) Arrested in growth and growth retardation/delay: Compared to a well with normal development and normal #  of progeny.<br>
for the target sequences presented in Table 1 A, it was concluded that dsRNA mediated silencing of the C. elegans target gene in nematodes, such as C elegans, had a fatal effect on the growth and viability of the worm.<br>
Subsequent to the above dsRNA silencing experiment, a more detailed phenotyping experiment was conducted in C. elegans in a high throughput format on 24 well plates,   fhe dsRNA library produced in bacterial strain AB301-105 (DL3), as described above, was fed to C. elegans nuc-l (el392) worms on 24 well plates as follows: nuc-l eggs were transferred to a separate plate and allowed to hatch simultaneously at 20 C for synchronization of the LI generation. Subsequently 100 of the synchronized LI worms were soaked in a mixture of 500 µl S-complete fed medium, comprising 5 |ag/mL cholesterol, 4 |aL/mL PHG and ImM IPfG, and 500 µL of bacterial cell culture of 0D60 )1 AB301-1O5 (DH3) of the C elegans dsRNA library carrying each a vector with a C. elegans genomic fragment for expression of the dsRNA. Ihe soaked LI worms were rolled for 2 hours at 25 C.<br>
After centrifugation and removal of 950 µl of the supernatant, 5 µl of the remaining and resuspcnded pellet (comprising about 10 to 15 worms) was transferred in the middle of each well of a 24 well plate, filled with a layer of agar LB broth. The inoculated plate was incubated at 25°C for 2 days. At the adult stage, 1 adult worm was singled and incubated at 25°C for 2 days for inspection of its progeny. The other adult worms are inspected in situ on the original 24 well plate. These experiments were performed in quadruplicate.<br>
This detailed phenotypic screen was repeated with a second batch of worms, the only difference being that the worms of the second batch were incubated at 20 C for 3 days.<br>
The phenotype of the worms fed with C elegans dsRNA was compared to the phenotype of C elegans nuc-l (el392) worms fed with the empty vector.<br><br>
Based on this experiment, it was concluded that silencing the C. elegans target genes as represented in fable 1A had a fatal effect on the growth and viability of the worm and that the target gene is essential to the viability of nematodes. Therefore these genes are good target genes to control (kill or prevent from growing) nematodes via dsRNA mediated gene silencing. Accordingly, the present invention encompasses the use of nematode orthologs of the above C. elegans target gene to control nematode infestation in a variety of organisms and materials.<br>
Example 2: Identification of E , melanosaster orthologous <br>
As described above in Hxample 1, numerous C. elegans lethal sequenes were identified and can be used for identifying orthologs in other species and genera. I-'or example, the C\ elegans lethal sequences can be used to identify orthologous D. melanogaslers sequences. That is, each C. elegans sequence can be querried against a public database, such as GenBank, for orthologous sequences in D. melanogaster. Potential D. nielanogaster orthologs were selected that share a high degree of sequence homology (H value preferably less than or equal to 1H-30) and the sequences arc blast reciprocal best hits, the latter means that the sequences from different organisms (e.g. C elegans and D. nielanogaster) are each other's top blast hits. For example, sequence C from C elegans is compared against sequences in D. nielanogaster using BLAS1\ If sequence C has the D. nielanogaster sequence D as best hit and when D is compared to all the sequences of C elegans, also turns out to be sequence C, then D and C are reciprocal best hits. This criterium is often used to define orthology, meaning similar sequences of different species, having similar function. 1 he D, nielanogaster sequence identifiers are represented in Table 1 A.<br>
Example 3: Leptinotarsa decemlineata (Colorado potato beetle)<br>
A,        Cloning partial gene sequences from Leptinotarsa decemlineata<br>
High quality, intact RNA was isolated from 4 different larval stages of Leptinotarsa ckcemlineata (Colorado potato beetle; source: Jerocn van Schaik, IZntocarc CV Biologische CJewasbeschcrming, Postbus 162, 6700 AD Wageningcn, the Netherlands) using IRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manufacturer's instructions (Cat. Nr. 1700, Promcga).    cDNA was generated using a commercially available kit (Superscript "^ 111<br><br>
Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Roekville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the LDOOl, LD002, LD003, LD006, I.D007, LD010, LDOl I, LD014, LD015, LD0I6 and LD018 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat. Nr. N8080240, Applied Biosystems) following the manufacturer's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Tabic 2-LD, which displays I.eptintarsa decemlineata target genes including primer sequences and cDNA sequences obtained, These primers were used in respective PCR reactions with the following conditions: 10 minutes at 95'°Cfollowed by 40 cycles of 30 seconds at 95°C,, 1 minute at 55°C and 1 minute at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel lixtraction kit. Cat. Nr. 28706, Qiagen), cloned into the pCR8/GW/topo vector (Cat. Nr. K2500 20, Invitrogcn), and sequenced. The sequences of the resulting PCR products are represented by the respectiveSHQ ID NOs as given in Table 2-LD and are referred to as the partial sequences. The corresponding partial amino acid sequence arc represented by the respective SHQ ID NOs as given in Table 3-LD, where the start of the reading frame is indicated in brackets.<br>
B.        dsRNA production of the Leptinotarsa decemlineata genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit 17 RibomaxTM  Express RNAi System (Cat. Nr. PI700, Promcga). First two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 17 promoter.<br>
Tor each of the target genes, the sense 17 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-Ll). The conditions in the PCR reactions were as follows: 4 minutes at 95°C, followed by 35 cycles of 30 seconds at 95°C, 30 seconds at 55°C and 1 minute at 72°C, followed by 10 minutes at 72^C. The anti-sense 17 template was generated using specific forward and specific 17 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-Ll).    The resulting PCR products were analyzed on agarose gel and purified by PCR<br><br>
purification kit (Qiaquick PCR Purification  Kit, Cat. Nr.  28106, Qiagen) and NaCI04<br>
precipitation.   The generated 17 forward and reverse templates were mixed to be transcribed<br>
and the resulting RNA strands were annealed, DNase and RNase treated, and purified by<br>
sodium acetate, following the manufacturer's instructions.  The sense strand of the resulting<br>
dsRNA for each of the target genes is given in Table 8-LD.  Table 8-LD displays sequences<br>
for  preparing  ds   RNA   fragments  of Leptinotarsa  decemlineata target  sequences  and<br>
concatemer sequences, including primer sequences.<br>
C.        Screening dsRNA targets using artificial diet for activity against<br>
Leptinotarsa decemlineata<br>
Artificial diet for the Colorado potato beetle was prepared as follows (adapted from Gclman et al., 2001, J. Ins. Sc, vol. 1, no. 7, 1-10): water and agar were autoclaved, and the remaining ingredients (shown in Table 2 below) were added when the temperature dropped to 55°C At this temperature, the ingredients were mixed well before the diet was aliquoted into 24-well plates (Nunc) with a quantity of 1ml of diet per well. The artificial diet was allowed to solidify by cooling at room temperature. Diet was stored at 4 °C for up to three weeks.<br><br><br><br>
Fifty µL of a solution of dsRNA at a concentration of 1 mg/ml was applied topically onto the solid artificial diet in the wells of the multiwell plate. The diet was dried in a iaminair How cabin. Per treatment, twenty-four Colorado potato beetle larvae (2nd  stage), with two insects per well, were tested. The plates were stored in the insect rearing chamber at 25 1 2°C 60 % relative humidity, with a 16:8 hours light:dark photoperiod. The beetles were assessed as live or dead every 1, 2 or 3 days. After seven days, for targets LD006, I.D007, LD010, IJ^Oll, and LD014, the diet was replaced with fresh diet with topically applied dsRNA at the same concentration (1 mg/ml); for targets LD001, LD002, LD003, LD015, and LD016, the diet was replaced with fresh diet only. The dsRNA targets were compared to diet only or diet with topically applied dsRNA corresponding to a fragment of the GFP (green fluorescent protein) coding sequence (SEQ ID NO: 235).<br>
Feeding artificial diet containing intact naked dsRNAs to L. decemlineata larvae resulted in significant increases in larval mortalities as indicated in two separate bioassays (Figures 1LD-2LD).<br>
All dsRNAs tested resulted ultimately in 100 % mortality after 7 to 14 days. Diet with or without GFP dsRNA sustained the insects throughout the bioassays with very little or no mortality.<br><br>
Typically, in all assays observed, CPB second-stage larvae fed normally on diet with or without dsRNA for 2 days and molted to the third larval stage. At this new larval stage the CPB were observed to reduce significantly or stop altogether their feeding, with an increase in mortality as a result.<br>
D,        Bioassay of dsRNA targets using potato leaf discs for activity against the Leptinotarsa decemlineata<br>
An alternative bioassay method was employed using potato leaf material rather than artificial diet as food source for CPB. Discs of approximately 1.1 cm in diameter (or 0.95 cm") were cut out off leaves of 2 to 3-wcck old potato plants using a suitably-si/cd cork borer. Ircatcd leaf discs were prepared by applying 20µl of a 10 ng/µil solution of target LD002 dsRNA or control gfp dsRNA on the adaxial leaf surface. The leaf discs were allowed to dry and placed individually in 24 wells of a 24-well multiplate (Nunc). A single second-larval stage CPB was placed into each well, which was then covered with tissue paper and a multiwcll plastic lid. The plate containing the insects and leaf discs were kept in an insect chamber at 28"C with a photoperiod of 16h light/8h dark, yhe insects were allowed to feed on the leaf discs for 2 days after which the insects were transferred to a new plate containing fresh treated leaf discs, thereafter, the insects were transferred to a plate containing untreated leaf discs every day until day 7. Insect mortality and weight scores were recorded.<br>
f'eeding potato leaf discs with surface-applied intact naked dsRNA of target LD002 to<br>
A. decemlincaia larvae resulted in a significant increase in larval mortalities (i.e. at day 7 all<br>
insects were dead; 100 % mortality) whereas control gfp dsRNA had no effect on CPB<br>
survival, 'farget LD002 dsRNA severely affected the growth of the larvae after 2 to 3 days<br>
whereas the larvae fed with gfp dsRNA at the same concentration developed as normal<br>
(Figure 3-U)).<br>
1).        Screening shorter versions of dsRNAs using artificial diet for activity against Leptinotarsa decemlineata<br>
This example exemplifies the finding that shorter (60 or lOObp) dsRNA fragments on their own or as concatcmcr constructs arc sufficient in causing toxicity towards the Colorado potato beetle.<br>
LDF014, a target known to induce lethality in Colorado potato beetle, was selected for this example. 1 his gene encodes a V-A'fPase subunit B (SEQ ID NO:; 15).<br><br>
A 100 base pair fragment, LD014 Fl, at position 195-294 on SBQ ID NO:: 15 (SliQ ID NO: 159) and a 60 base pair fragment, LD014 F2, at position 235-294 on SJ:Q ID NO:: 15 (SF;Q ID NO: 160) were further selected. See also Table 7-LD,<br>
Iwo concatemers of 300 base pairs, LD014 CI and I.D014_C2, were designed (SHQ ID NO:: 161 and SHQ ID NO:: 162). LD014 CI contained 3 repeats of the 100 base pair fragment described above (SEQ ID NO:: 159) and LD0I4 C2 contained 5 repeats of the 60 base pair fragment described above (SHQ ID NO:: 160). See also Table 7-LD.<br>
Ihe fragments LD014 Fl and I.D014 F2 were synthesized as sense and antisensc primers. These primers were annealed to create the double strands DNA molecules prior to cloning. Xha] and Xmal restrictions sites were included at the 5' and 3' ends of the primers, respectively, to facilitate the cloning.<br>
The concatemers were made as 300 base pairs synthetic genes. Xbal and Xmal restrictions sites were included at the 5' and 3' ends of the synthetic DNA fragments, respectively, to facilite the cloning.<br>
fhc 4 DNA molecules, i.e. the 2 single units (LD014 Fl &amp; FD014 F2) and the 2 concatemers (LD014 CI &amp; LD0I4 C2), were digested whh Xha] 'and Xmal and subcloncd in pBluescriptll SKf linearised by Xba\ and Xmal digests, resulting in recombinant plasmids pi, p2, p3, &amp; p4, respectively.<br>
Double-stranded RNA production: dsRNA was synthesized using the commercially available kit 17 RibomaxTM  Express RNAi System (Cat. Nr. P1700, Promega). First two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative lo the 17 promoter. For LD0I4 Fl, the sense T7 template was generated using the specific T7 Ibrward primer oGBM159 and the specific reverse primer oGBM164 (represented herein as Si:0 ID NO:: 204 and SHQ ID NO: 205, respectively) in a PCR reaction with the following conditions: 4 minutes at 95°C, followed by 35 cycles of 30 seconds at 95X, 30 seconds at 55°C and I minute at 72°C, followed by 10 minutes at 72°C. The anti-sense T7 template was generated using the specific forward primer oGBM163 and the specific 17 reverse primer 0GBMI6O (represented herein as SEQ ID NO: 206 and SEQ ID NO:: 207, respectively) in a PCR reaction with the same conditions as described above. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaCI04 precipitation. The generated T7 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, Dnase and Rnase treated, and purified by sodium acetate, following the<br><br>
manufacturer's instructions. The sense strand of the resulting dsRNA is herein represented by SHQ ID NO: 203.<br>
For LD014 F2, the sense 17 template was generated using the specific T7 forward primer OCGM161 and the specific reverse primer 0GBMI66 (represented herein as SBQ ID NO:: 209 and SliQ ID NO:: 210, respectively) in a PCR reaction with the following conditions: 4 minutes at 95°C, followed by 35 cycles of 30 seconds at 95°C, 30 seconds at 55"C and 1 minute at 72°C, followed by 10 minutes at 72°C. The anti-sense T7 template was generated using the specific forward primer oGBM165 and the specific T7 reverse primer 0GBMI62 (represented herein as SHQ ID NO:: 211 and SHQ ID NO:: 212, respectively) in a PCR reaction with the same conditions as described above. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaC104 precipitation. Ihe generated T7 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, Dnase and Rnasc treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA is herein represented by SHQ ID NO: 208.<br>
Also for the concatemers, separate single 5' \7 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 11 promoter. The recombinant plasmids p3 and p4 containing HD014 CI &amp; LD014 C2 were linearised with Xbal or Xwal, the two linear fragments for each construct purified and used as template for the in vitro transcription assay, using the 17 promoters flanking the cloning sites. Double-stranded RNA was prepared by in vitro transcription using the T7 RiboMAX'^ Hxpress RNAi System (Promega). The sense strands of the resulting dsRNA for HD014 CI and LD014 C2 are herein represented by SHQ ID NO:: 213 and2114, respectively.<br>
Shorter sequences of target LD014 and concatemers were able to induce lethality in Leptinotarsa decemlincata, as shown in Figure 4-LD,<br>
G.        Screening dsRNAs at different concentrations using artificial diet for activity against Leptinotarsa decemlineata<br>
fifty µl of a solution of dsRNA at serial ten-fold concentrations from 1 lag/^il (for target HD027 from 0.1 |ig/^il)down to 0.01 ng/^l was applied topically onto the solid artificial diet in the wells of a 24-well plate (Nunc). The diet was dried in a laminair flow cabin. Per treatment, twenty-four Colorado potato beetle larvae (2" stage), with two insects per well, were tested. The plates were stored in the insect rearing chamber at 25 + 2 °C, 60 % relative<br><br>
humidity, with a 16:8 hours light:dark photoperiod. The beetles were assessed as live or dead at regular intervals up lo day 14. After seven days, the diet was replaced with fresh diet with topically applied dsRNA at the same concentrations. The dsRNA targets were compared to diet only.<br>
Feeding artificial diet containing intact naked dsRNAs of different targets to L decernlineata larvae resulted in high larval mortalities at concentrations as low as between O.I and 10 ng dsRNA/fj! as shown in Figure 5-LD.<br>
H.        Cloning of a CPB gene fragment in a vector suitable for bacterial production of insect-active double-stranded RNA<br>
While any efficient bacterial promoter may be used, a DNA fragment corresponding to an MLB gene target was cloned in a vector for the expression of double-stranded RNA in a bacterial host (See WO 00/01846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98X\ 30 seconds at 55X and 2 minutes at 72°C, followed by 10 minutes at 72°C. Yhe resulting PCR fragment is analyzed on agarose gcf purified (QlAquick Gel Hxtraction kit, Cat. Nr. 28706, Qiagcn), blunt-end cloned into Srf 1-lincari/ed pGNA49A vector (reference to WO00188121 Al), and sequenced. The sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8. The recombinant vector harboring this sequence is named pGBNJ003.<br>
The sequences of the specific primers used for the amplification of target gene fragment LD010 are provided in fable 8 (forward primer SHQ ID NO:: 191 and reverse primer SEQ ID NO:: 190). The template used was the pCR8/GW/topo vector containing the fD0q0 sequence (SliQ 'D NO:: 11). fhe primers were used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds al 55X and 2 minutes at 72°C, followed by 10 minutes at 72T. The resulting PCR fragment was analyzed on agarose gel, purified (QlAquick Gel Extraction kit, Cat Nr. 28706, Qiagcn), blunt-end cloned into Srf 1-linearizcd pGNA49A vector (reference to WO 00/188121 Al), and sequenced. The sequence of the resulting PCR product corresponds to SEQ ID NO:; 188 as given in Table 8. The recombinant vector harboring this sequence was named pGBNJ003.<br><br>
L	Expression and production of a double-stranded RNA target in<br>
two strains of Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below were followed in order to express suitable levels of insect-active double-stranded RNA of target Ll010 in bacteria. An RNascIil-deficicnt strain, AB309'105, was used in comparison to wild-type RNaseHl-containing bacteria, BL21(DB3).<br>
Transformation of AB309-l()5 and BL21(DE3)<br>
Three hundred ng of the plasmid was added to and gently mixed in a 50 µl aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3). l^he cells were incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C for 5 minutes, after which the cells were placed back on ice for a further 5 minutes. Four hundred and fifty µL of room temperature SOC medium was added to the cells and the suspension incubated on a shaker (250 rpm) at 37 °C for 1 hour. One hundred µL of the bacterial cell suspension was transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 µg/ml carbenicillin antibiotic. The culture was incubated on an Innova 4430 shaker (250 rpm) at 37oC overnight (16 to 18 hours).<br>
Chemical induction of douhle-stranded RNA expression in AB3()9-1()5 and BL21(DE3)<br>
Expression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(Dli3) was made possible since all the genetic components for controlled expression arc present. In the presence of the chemical inducer isopropylthiogalactosidc, or IPTG, the r7 polymerase will drive the transcription of the target sequence in both anliscnse and sense directions since these arc flanked by oppositely oriented 17 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture was measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth, Fiftly ml of this culture was transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 °C for 10 minutes. The supernatant was removed and the bacterial pellet resuspendcd in 50 ml of fresh S complete medium (SNC medium plus 5 |ag/ml cholesterol) supplemented with 100 ng/ml carbenicillin and 1 mM IPTG. The bacteria were induced for 2 to 4 hours at room temperature.<br>
HEat treatment of bacteria<br>
Bacteria were killed by heat treatment in order to minimize the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria expressing double-<br><br>
stranded RNA is not a prerequisite for inducing toxicity towards the insects due to RNA interference, fhc induced bacterial culture was ecntrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 °C for 20 minutes in a water bath. After heat treatment, the bacterial pellet was resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes were prepared and used in the bioassays for each refreshment. The tubes were stored at -20 °C until further use-<br>
J.	Laboratory trials to test Escherichia coli expressing dsRNA target<br>
LD010 against Lcptinotursa dccemlineata<br>
fwo bioassay methods were employed to test double-stranded RNA produced in Escherichia coli against larvae of the Colorado potato beetle: (!) artificial diet-based bioassay, and. (2) plant-based bioassay.<br>
Artificial diet-based bioassays<br>
Artificial diet for the Colorado potato beetle was prepared as described previously in lixample 4. A half milliliter of diet was dispensed into each of the wells of a 48-well multiwell test plate (Nunc). For every treatment, fifty \x\ of an OD 1 suspension of heat-treated bacteria (which is equivalent to approximately 5x10 bacterial cells) expressing dsRNA was applied topically onto the solid diet in the wells and the plates were allowed to dry in a laminair flow cabin. Her treatment, forty-eight 2nd stage Colorado potato beetle larvae, one in each well containing diet and bacteria, were tested. Each row of a plate (i.e. 8 wells) was considered as one replicate, fhe plates were kept in the insect rearing chamber at 25 1 2 "C, 60 i 5 % relative humidity, with a 16:8 hours light:dark photopcriod. After every 4 days, the beetles were transferred to fresh diet containing topically-applied bacteria. The beetles were assessed as alive or dead every one or three days post infestation. For the survivors, growth and development in terms of larval weight was recorded on day 7 post infestation.<br>
For IWascIII-dcficient E. coli strain ABSOQ-IOS, bacteria containing plasmid pGBNJ003 and those containing the empty vector pGN29 (reference to WO 00/188121A I) were tested in bioassays for CPB toxicity. Bacteria harboring the pGBNJ003 plasmid showed a clear increase in insect mortality with time, whereas little or no mortality was observed for pGN29 and diet only control (Figures 6a-LD &amp; 7a-LD). ^fhe growth and development of Colorado potato beetle larval survivors, 7 days after feeding on artificial diet containing bacteria expressing dsRNA target LD010, was severely impeded (Table 10-LD, Figure 8a-LI)).<br><br>
1-or E. coli strain BL21(I)H3), bacteria containing plasmid pGBNJ003 and those containing the empty vector pCjN29 were tested against the Colorado potato beetle larvae. Similar detrimental effects were observed on larvae fed diet supplemented with BL21(D13) bacteria as for the RNAselll-deficient strain, AB309-105 (Figures 6b-Ll) &amp; 7b-Ll)). However, the number of survivors for the five clones were higher for BL2I(DI:3) than for AB309-105; at day 12, average mortality values were approximately 25 % lower for this strain compared to the RNase III deficient strain. Also, the average weights of survivors fed on diet containing BL21(Dl-3) expressing dsRNA corresponding to target LD010 was severely reduced (Table 10-LD, Figure 8b-LD).<br>
The delay in growth and development of the CPB larvae fed on diet containing either of the two bacterial strains harboring plasmid pGBNJ003 was directly correlated to feeding inhibition since no frass was visible in the wells of refreshed plates from day 4 onwards when compared to bacteria harboring the empty vector pGN29 or the diet only plate. This observation was similar to that where CPB was fed on in vitro transcribed double-stranded RNA topically applied to artificial diet (see Example 3D); here, cessation of feeding occurred from day 2 onwards on treated diet.<br>
Plant-based hioassays<br>
Whole potato plants were sprayed with suspensions of chemically induced bacteria expressing dsRNA prior to feeding the plants to CPB larvae. The potato plants of variety 'line 5' were grown from tubers to the 8-12 unfolded leaf stage in a plant growth room chamber with the following conditions: 25 i 2T, 60 % relative humidity, 16:8 hour lightidark photoperiod. The plants were caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent larval escape, f'itlteen Colorado potato beetle larvae at the LI stage were placed on each treated plant in the cage. Plants were treated with a suspension of E. coli AB309-105 harboring the pGBNJ003 plasmids (clone 1; Figure 7a-LD) or pG"N29 plasmid (clone 1; see Figure 7a-LD). Different quantities of bacteria were applied to the plants: 66, 22, and 7 units, where one unit is defined as 109 bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of 1.6 ml was sprayed on the plant with the aid of a vaporizer. One plant was used per treatment in this trial. The number of survivors were counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of A. coli bacterial strain AB3O9-105 expressing target dsRNA from pGBNJ003 led to a dramatic increase in insect mortality when compared<br><br>
to pGN29 control.  The mortality count was maintained when the amount of bacteria cell<br>
suspension was diluted 9-fold (Figure 9-LD). The average weights of the larval survivors at<br>
day 11 on plants sprayed with bacteria harboring the pGBNJ003 vector were approximately<br>
10-fold less than that of pGN29 (Figure 10-LD). Feeding damage by CPli larvae of the<br>
potato plant sprayed with bacteria containing the pGBNJ003 plasmid was much reduced<br>
when compared to the damage incurred on a potato plant sprayed with bacteria containing the<br>
empty vector pGN29 (Figure 11-LD).<br>
rhesc experiments showed that double-stranded RNA corresponding to an insect gene<br>
target sequence produced  in either wild-type or RNascIIl-deficicnt bacterial expression<br>
systems is toxic towards the insect in terms of substantial increases in insect mortality and<br>
growth/development delay for larval survivors. It is also clear from these experiments that an<br>
exemplification was provided for the effective protection of plants/crops from insect damage<br>
by the use of a spray of a formulation consisting of bacteria expressing double-stranded RNA<br>
corresponding to an insect gene target.<br>
K,        Testing various culture suspension densities of Escherichia coli expressing dsRNA target LD010 against Leptinoiarsa decemlineata<br>
Preparation and treatment of bacterial cultures are described in Example 3J. Three¬fold serial dilutions of cultures (starting from 0,25 unit equivalents) of Escherichia coli RNAselll-deficient strain AB309-105 expressing double-stranded RNA of target LD010 were applied to foliages of the potato plant of variety 'Bintje' at the 8-12 unfolded leaf stage, fen LI larvae of the L. decemlineata were placed on the treated plants with one plant per treatment. Scoring for insect mortality and growth impediment was done on day 7 (i.e., 7 days post infestation).<br>
As shown in Figure 14-LD, high CPB larval mortality (90 to 100 %) was recorded after 1 week when insects were fed potato plants treated with a topical application by fine spray of heat-inactivated cultures of E.coli harboring plasmid pGBNJ003 (for target 10 dsRNA expression) at densities 0.25, 0.08 and 0.025 bacterial units. At 0.008 units, about a third of the insects were dead, however, the surviving insects were significantly smaller than those in the control groups {E. coli harbouring the empty vector pGN29 and water only), feeding damage by CPB larvae of the potato plant sprayed with bacteria containing the pGBNJ003 plasmid at concentrations 0.025 or 0.008 units was much reduced when compared to the damage incurred on a potato plant sprayed with bacteria containing the empty vector pGN29 (Figure 15-LD).<br><br>
L. Adults are extremely susceptible to orally ingested dsRNA corresponding to target genes.<br>
The example provided below highlights the finding that adult insects (and not only insects of the larval stage) are extremely susceptible to orally ingested dsRNA corresponding to target genes.<br>
l"0ur targets were chosen for this experiment: targets 2, 10, 14 and 16 (Sl:Q ID NO: 168. 188, 198 and 220, respectively). Gf P fragment dsRN A (SHQ 11^ NO: 235) was used as a control. Young adults (2 to 3 days old) were picked at random from our laboratory-reared culture with no bias towards insect gender, len adults were chosen per treatment. The adults were prestarvcd for at least 6 hours before the onset of the treatment. On the first day of treatment, each adult was fed four potato leaf discs (diameter 1.5 cm ) which were pretreated with a topical application of 25µl of 0.1µ/µl target dsRNA (synthesized as described in Example 3A; topical application as described in Example 3E) per disc. Each adult was confmed to a small petridish (diameter 3 cm) in order to make sure that all insects have ingested equal amounts of food and thus received equal doses of dsRNA. the following day, each adult was again fed four treated leaf discs as described above. On the third day, all ten adults per treatment were collected and placed together in a cage consisting of a plastic box (dimensions 30 cm x 20 cm x 15 cm) with a fine nylon mesh built into the lid to provide good aeration. Inside the box, some moistened filter paper was placed in the base. Some (untreated) potato foliage was placed on top of the paper to maintain the adults during the experiment. From day 5, regular assessments were carried out to count the number of dead, alive (mobile) and moribund insects. For insect moribundity, adults were laid on their backs to check whether they could right themselves within several minutes; an insect was considered moribund only if it was not able to turn onto its front.<br>
Clear specific toxic effects of double-stranded RNA correpsonding to different targets towards adults of the Colorado potato beetle, Leptinotarsa decemlineata, were demonstrated in this experiment (Figure 12-LD). Double-stranded RNA corresponding to a gfp fragment showed no toxicity towards CPB adults on the day of the final assessment (day 19). This experiment clearly showed that the survival of CPB adults was severely reduced only after a few days of exposure to dsRNA when delivered orally. For example, for target 10, on day 5, 5 out of 10 adults were moribund (sick and slow moving); on day 6, 4 out of 10 adults were dead with three of the survivors moribund; on day 9 all adults were observed dead.<br>
As a consequence of this experiment, the application of target double-stranded RNAs against insect pests may be broadened to include the two life stages of an insect pest (i.e.<br><br>
larvae and adults) which could cause extensive crop damage, as is the case with the Colorado potato beetle.<br>
Example 4: Phaedon cochleariae (mustard leaf beetle)<br>
A.        Cloning of a partial sequence of the Phaedon cochleariae (mustard leaf beetle) PCOOl, PC003, PC005, PC010, PC014, PC016 and PC027 genes via family PCR<br>
High quality, intact RNA was isolated from the third larval stage of Phaedon cochleariae (mustard leaf beetle; source: Dr. Caroline Muller, Julius-von-Sachs-Institutc for Biosciences, Chemical Kcology Group, University of Wuerzburg, Julius-von-Sachs-Plat/ 3, D-97082 Wucr/burg, Germany) using 1 Rl/ol Reagent (Cat. Nr. 15596-026/] 5596-018, Invitrogen. Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase (Cat. Nr. 1700, Promega) treatment following the manufacturer's instructions. cDNA was generated using a commercially available kit (Superscript "^ 111 Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the PCOOl, PC003, PC005, PC010, PC014, PC016 and PC027 genes, a scries of PCR reactions with degenerate primers were performed using Amplilaq Gold (Cat. Nr. N8080240, Applied Biosystems) following the manafacturcr's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-PC, Table 2-PC displays Phaedon cochleariae target genes including primer sequences and cDNA sequences obtained. These primers were used in respective PCR reactions with the following conditions: 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C1 minute at 55°Cand 1 minute at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr, 28706, Qiagcn), cloned into the pCR4/'J'OP() vector (Cat. Nr. K4530-20, Invitrogen) and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-PC and are referred to as the partial sequences.<br>
The corresponding partial amino acid sequence are represented by the respective SliQ ID NO:s as given in Table 3-PC. Table 3-PC provides amino acid sequences of cDNA clones, and the start of the reading frame is indicated in brackets.<br>
U.        dsRNA production of the Phaedon cochleariae genes<br><br>
dsRNA was synthesized in milligram amounts using the commercially available kit T7 RibomaxTM  Hxprcss RNAi System (Cat. Nr. P1700, Promcga). first two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 17 promoter.<br>
For each of the target genes, the sense T7 template was generated using specific 17 forward and specific reverse primers. 1 he sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-PC. Table 8-PC provides details for preparing ds RNA fragments of Fhaedon cochleariae target sequences, including primer sequences.<br>
The conditions in the PCR reactions were as follows: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95°C, 30 seconds at 60°C and 1 minute at 72°C, followed by 15 cycles of 30 seconds at 95°C, 30 seconds at 50°C and 1 minute at 72"C followed by 10 minutes at 72°C. The anti-sense 17 template was generated using specific forward and specific 17 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-PC. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaCl(&gt;i precipitation. The generated 17 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, DNasc and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-PC.<br>
C.        Laboratory trials to test dsRNA targets, using oilseed rape leaf discs for activity<br>
against Phaedon cochleariae larvae<br>
The example provided below is an exemplification of the finding that the mustard leaf beetle (MLB) larvae are susceptible to orally ingested dsRNA corresponding to own target genes.<br>
To test the different double-stranded RNA samples against MLB larvae, a leaf disc assay was employed using oilseed rape {Brassica napus variety SW Oban; source: Nick Balaam. Sw Seed Ltd., 49 North Road, Abington, Cambridge, CBl 6AS, UK) leaf material as food source. The insect cultures were maintained on the same variety of oilseed rape in the insect chamber at 25 f 2 'C and 60 i 5 % relative humidity with a photoperiod of I6h light/8h dark. Discs of approximately LI cm in diameter (or 0.95 cm^) were cut out off<br><br>
leaves of 4- to 6-wcck old rape plants using a suitably-sized cork borer. Double-stranded RNA samples were diluted to 0,1µg/µl in Milli-Q water containing 0.05% Triton X-100, Ireated leaf discs were prepared by applying 25 µl of the diluted solution of target PC001, PC003, PC005, PC010, PC014, PC016, PC027 dsRNA and control gfp dsRNA or 0.05 % Triton X-100 on the adaxial leaf surface. The leaf discs were left to dry and placed individually in each of the 24 wells of a 24-well multiplatc containing 1 ml of gellified 2% agar which helps to prevent the leaf disc from drying out. Two neonate MLB larvae were placed into each well of the plate, which was then covered with a multiwell plastic lid. The plate (one treatment containing 48 insects) was divided into 4 replicates of 12 insects per replicate (each row). The plate containing the insects and leaf discs were kept in an insect chamber at 25 2oC and 60  5 % relative humidity with a photoperiod of 16h light/8h dark. The insects were fed leaf discs for 2 days after which they were transferred to a new plate containing freshly treated leaf discs. 1 hereafter, 4 days after the start of the bioassay, the insects from each replicate were collected and transferred to a Petri dish containing untreated fresh oilseed rape leaves. Larval mortality and average weight were recorded at days 2, 4 7, 9 and 11.<br>
P. cochleariae larvae fed on intact naked target dsRNA-treated oilseed rape leaves resulted in significant increases in larval mortalities for all targets tested, as indicated in I'igure 1(a). festcd double-stranded RNA for target PC010 led to 100 % larval mortality at day 9 and for target PC027 at day 11. For all other targets, signficantly high mortality values were reached at day 11 when compared to control gfp dsRNA, 0.05% Trition X-IOO alone or untreated leaf only: (average value in percentage - confidence interval with alpha 0.05) PCOOl (94.4 i 8.2); PC003 (86.1 i 4.1); PC005 (83.3 ±. 7.8); PC0I4 (63.9 ± 20.6); PC0I6 (75.0 \ 16.8); gfp dsRNA (11.1 i: 8.2); 0.05% Triton X-100 (19.4 ± 10.5); leaf only (8.3 ± 10.5).<br>
Larval survivors were assessed based on their average weight. For all targets tested, the mustard leaf beetle larvae had significantly reduced average weights after day 4 of the bioassay; insects fed control gfp dsRNA or 0.05% Iriton X-100 alone developed normally, as for the larvae on leaf only (Figure l(b)-PC).<br>
D,	Laboratory trials to screen dsRNAs at different concentrations using oilseed<br>
rape leaf discs for activity against Phaedon cochleariae larvae<br><br>
Twenty-fiveµL of a solution of dsRNA from target PC010 or PC027 at serial ten-fold concentrations from 0.1µg/fµl down to 0.1 ng/)il was applied topically onto the oilseed rape leaf disc, as described in I-xample 4D above. As a negative control, 0.05% Triton X-100 only was administered to the leaf disc. Per treatment, twenty-four mustard leaf beetle neonate larvae, with two insects per well, were tested. The plates were stored in the insect rearing chamber at 25 i 2 T, 60 i 5 % relative humidity, with a 16:8 hours lighf.dark photoperlod. At day 2, the larvae were transferred on to a new plate containing fresh dsRNA-treated leaf discs. At day 4 for target PC010 and day 5 for target PC027, insects from each replicate were transferred to a Petri dish containing abundant untreated leaf material, fhe beetles were assessed as live or dead on days 2, 4, 7, 8, 9, and 11 for target PC010, and 2, 5, 8, 9 and 12 for target PC027.<br>
feeding oilseed rape leaf discs containing intact naked dsRNAs of the two different targets, PC010 and PC027, to P. cochleahae larvae resulted in high mortalities at concentrations down to as low as 1 ng dsRNA/|µl solution, as shown in Figures 2 (a) and (b). Average mortality values in percentage i- confidence interval with alpha 0.05 for different concentrations of dsRNA for target PC010 at day 11, 0 nvg/µil: 8.3 ± 9.4; 0.1 µg/µl: 100; 0.01µg/µl: 79,2 i. 20.6; 0.001 µg/M!:58.3 1 9.4; 0.0001 µg/µl: 12.5 i 15,6; and for target PC027 at day 12, 0 µg/µl: 8.3 i 9.4; 0.1 µg/µil: 95.8 i-8.2; 0.01 µg/µl: 95.8 i 8.2; 0.001 µ/µl: 83.3 L 13.3; 0.0001 µg/µil: 12.5 i 8.2.<br>
E,        Cloning of a MLB gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an example of cloning a DNA fragment corresponding to an MLB gene target in a vector for the expression of double-stranded RNA in a bacterial host, although any vector comprising a 17 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes arc provided in fable 8. I he template used is the pCR8/GW/topo vector containing any of target sequences, fhc primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72°C. The rcsuking PCR fragment is analyzed on agarose gel, purified (QIAquick Gel Extraction kit. Cat. Nr. 28706, Qiagen), blunt-end cloned into Srf 1-lineari/ed p(iNA49A vector (reference to WO00188121A1), and sequenced,   fhe sequence<br><br>
of the resulting PCR product corresponds to the respective sequence as given in Table 8. The recombinant vector harbouring this sequence is named pGBNJ00(to be completed).<br>
The sequences of the specific primers used for the amplification of target gene fragment PC010 arc provided in Table 8-PC.  fhe template used was the pCR8/GW/topo vector containing the PC010 sequence (SHQ (D NO: 253). The primers were used in a touch¬down PCR reaction with the following conditions: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95"C, 30 seconds at 60°C with temperature decrease of-0.5 T per cycle and I minute at 72°C, followed by 15 cycles of 30 seconds at 95°C, 30 seconds at 50°C and 1 minute at 72°C , followed by 10 minutes at 72°C. The resulting PCR fragment was analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into .S&gt;/l-linearized pGNA49A vector (reference to WO00188I2IA1), and sequenced. The sequence of the resulting PCR product corresponds to SRQ ID NO: 488 as given in Tabic 8-PC. 'fhe recombinant vector harbouring this sequence was named pGCDJOOl.<br>
F.	Expression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli AB309-105<br>
'Ihe procedures described below are followed in order to express suitable levels of insect-active double-stranded RNA of insect target in bacteria.    In this experiment, an RNaselll-deficient strain, AB309^105 is used, y ransformation of A B309-105<br>
rhrcc hundred ng of the plasmid were added to and gently mixed in a 50 \x\ aliquot of ice-chilled chemically competent E. coli strain AB309-105. l^he cells were incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 T for 5 minutes, after which the cells were placed back on ice for a further 5 minutes. Four hundred and fiftyµl of room temperature SOC medium was added to the cells and the suspension incubated on a shaker (250 rpm) at 37 °C for 1 hour. One hundredµl of the bacterial cell suspension was transferred to a 500 ml conical fiask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 ng/ml carbenicillin antibiotic. The culture was incubated on an Innova 4430 shaker (250 rpm) at 37 °C overnight (16 to 18 hours).<br>
Chemical induction of double-stranded liNA expression inAB309-105<br>
lixpression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 was made possible since all the genetic components for controlled expression arc present. In the presence of the chemical inducer isopropylthiogalactosidc, or<br><br>
IPTG, the T7 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are flanked by oppositely oriented T7 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture was measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture was transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 T for 10 minutes. The supernatant was removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5µg/ml cholesterol) supplemented with 100 µg/ml carbenicillin and 1 mM IPTG. The bacteria were induced for 2 to 4 hours at room temperature.<br>
Ikat treatment of bacteria<br>
Bacteria were killed by heat treatment in order to minimize the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria expressing double-stranded RNA is not a prerequisite for inducing toxicity towards the insects due to RNA interference. The induced bacterial cuhure was centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 ^C for 20 minutes in a water bath. After heat treatment, the bacterial pellet was resuspended in a total volume of 50 ml of 0.05% Triton X-100 solution. The tube was stored at 4 "C until further use<br>
G.        Laboratory trials to test Escherichia coli expressing dslWA targets against<br>
Phaedon cochleiiriae<br>
Leaf disc bioassays<br>
rhe leaf-disc bioassay method was employed to test double-stranded RNA from target PC010 produced in Escherichia coli (from plasmid pGCDJOOl) against larvae of the mustard leaf beetle. Leaf discs were prepared from oilseed rape foliage, as described in Example 4. Twenty µl of a bacterial suspension, with an optical density measurement of 1 at 600 nm wavelength, was pipetted onto each disc. The leaf disc was placed in a well of a 24-multiwell plate containing 1 ml gellitled agar. On each leaf disc were added two neonate larvae. For each treatment, 3 replicates of 16 neonate larvae per replicate were prepared. The plates were kept in the insect rearing chamber at 25 ± 2 ""C and 60 ± 5 % relative humidity, with a 16:8 hours light:dark photoperiod. At day 3 (i.e. 3 days post start of bioassay), larvae were transferred to a new plate containing fresh treated (same dosage) leaf discs. The leaf material was refreshed every other day from day 5 onwards. The bioassay was scored on<br><br>
mortality and average weight. Negative controls were leaf discs treated with bacteria harbouring plasmid pGN29 (empty vector) and leaf only.<br>
A clear increase in mortality of P. cochleariae larvae with time was shown after the insects were fed on oilseed rape leaves treated with a suspension of RNaselll-deficient E. coli strain AB309-105 containing plasmid pGCDJOOl, whereas very little or no insect mortality was observed in the case of bacteria with plasmid pGN29 or leaf only control (Figure 3-PC),<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria expressing dsRNA prior to feeding the plants to MLB. The are grown from in a plant growth room chamber The plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. MLB are placed on each treated plant in the cage. Plants are treated with a suspension of £. coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 10 bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of £. coli bacterial strain AB309-105 expressing target dsRNA from pGBNJ003 leed to a dramatic increase in insect mortality when compared to pGN29 control. These experiments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deflcient bacterial expression systems is toxic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these experiments that an exemplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria expressing double-stranded RNA corresponding to an insect gene target.<br>
Example 5: Epilachna varivetis (Mexican bean beetle)<br>
A,        Cloning Epilaclma varivetis partial gene sequences<br>
High quality, intact RNA was isolated from 4 different larval stages of Epilachna varivetis (Mexican bean beetle; source: Thomas Dorsey, Supervising Entomologist, New<br><br>
Jersey Department of Agriculture, Division of Plant Industry, Bureau of Biological Pest Control, Phillip Aiampi Beneficial Insect Laboratory, PO Box 330, Trenton, New Jersey 08625-0330, USA) using TRIzol Reagent (Cat, Mr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manafacturer's instructions (Cat. Nr, 1700, Promega). cDNA was generated using a commercially available kit (Superscript ™ III Reverse Transcriptase, Cat. Nr. 18080044, Invitrogcn, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the EV005, EV009, EVOlO, EV015 and EV016 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat. Nr. N8080240, Applied Biosystems) following the manufacturer's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-EV, which displays Epilachna varivetis target genes including primer sequences and cDNA sequences obtained. These primers were used in respective PCR reactions with the following conditions: for EV005 and EV009, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 50°C  and 1 minute 30 seconds at 72°C, followed by 7 minutes at 72°C; for EV014, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 53°C and 1 minute at 72X, followed by 7 minutes at 72°C; for EVOlO and EV016, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95'^C, 1 minute at 54°C and 1 minute 40 seconds at 72°C, followed by 7 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR4/rOPO vector (Cat. Nr. K4530^20, Invitrogcn), and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-EV and arc referred to as the partial sequences. The corresponding partial amino acid sequences are represented by the respective SEQ ID NO;s as given in Table 3-EV, where the start of the reading frame is indicated in brackets.<br>
B.        dsRNA production of the Epilachna varivetis genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit 17 Ribomax™ Express RNAi System (Cat. Nr. PI700, Promega). First two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 17 promoter.<br><br>
For each of the target genes, the sense T7 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-EV.<br>
The conditions in the PCR reactions were as follows: 1 minute at95°C followed by 20 cycles of 30 seconds at 9°C 30 seconds at 6µ and 1 minute at 72°C followed by 15 cycles of 30 seconds at 95°C30 seconds at 50°C and 1 minute at 72°C followed by 10 minutes at 72°C. The anti-sense T7 template was generated using specific forward and specific T7 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-EV. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagcn) and NaCI04 precipitation. The generated 17 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-EV.<br>
C.        Laboratory trials to test dsRNA targets using bean leaf discs for activity against<br>
Epilachna varivetis larvae<br>
The example provided below is an exemplification of the finding that the Mexican bean beetle (MBB) larvae are susceptible to orally ingested dsRNA corresponding to own target genes.<br>
To test the different double-stranded RNA samples against MBB larvae, a leaf disc assay was employed using snap bean (Phaseolus vulgaris variety Montano; source: Aveve NV, Belgium) leaf material as food source. The same variety of beans was used to maintain insect cultures in the insect chamber at 25 ± 2 T and 60 ± 5 % relative humidity with a photoperiod of I6h light/8h dark. Discs of approximately 1.1 cm in diameter (or 0.95 cm3) were cut out off leaves of I- to 2-week old bean plants using a suitably-sized cork borer. Double-stranded RNA samples were diluted to I |ig/µl in Milli-Q water containing 0.05% Triton X-100. Treated leaf discs were prepared by applying 25 ^\ of the diluted solution of target Ev005, EvOlO, Ev015, EvOI6 dsRNA and control gfp dsRNA or 0.05 % Triton X-100 on the adaxial leaf surface. The leaf discs were left to dry and placed individually in each of the 24 wells of a 24-well multiplate containing I ml of gellified 2 % agar which helps to<br><br>
prevent the leaf disc from drying out. A single neonate MBB larva was placed into each well of a plate, which was then covered with a multiwell plastic lid. T he plate was divided into 3 replicates of 8 insects per replicate (row). The plate containing the insects and leaf discs were kept in an insect chamber at 25 ± 2 °CC and 60 ± 5 % relative humidity with a photoperiod of 16h light/8h dark. The insects were fed on the leaf discs for 2 days after which the insects were transferred to a new plate containing freshly treated leaf discs. Thereafter, 4 days after the start of the bioassay, the insects were transferred to a petriplate containing untreated fresh bean leaves every day until day 10. Insect mortality was recorded at day 2 and every other day thereafter.<br>
Feeding snap bean leaves containing surface-applied intact naked target dsRNAs to £. varivestls larvae resulted in significant increases in larval mortalities, as indicated in Figure 1. Tested double-stranded RNAs of targets EvOlO, Ev015, &amp; Ev016 led to 100 % mortality after 8 days, whereas dsRNA of target Ev005 took 10 days to kill all larvae. The majority of the insects fed on treated leaf discs containing control gfp dsRNA or only the surfactant Triton X-100 were sustained throughout the bioassay (Figure 1-EV),<br>
D,        Laboratory trials to test dsRNA targets using bean leaf discs for activity against<br>
Epilachna varivestis adults<br>
The example provided below is an exemplification of the finding that the Mexican bean beetle adults are susceptible to orally ingested dsRNA corresponding to own target genes.<br>
In a similar bioassay set-up as for Mexican bean beetle larvae, aduh MBBs were tested against double-stranded RNAs topically-applied to bean leaf discs. Test dsRNA from each target EvOlO, Ev015 and Ev016 was diluted in 0.05 % Triton X-100 to a final concentration of 0.1 \iglµL. Bean leaf discs were treated by topical application of 30 µL of the test solution onto each disc. The discs were allowed to dry completely before placing each on a slice of gellified 2 % agar in each well of a 24-well multiwell plate. Three-day-old adults were collected from the culture cages and fed nothing for 7-8 hours prior to placing one adult to each well of the bioassay plate (thus 24 adults per treatment). The plates were kept in the insect rearing chamber (under the same conditions as for MBB larvae for 24 hours) after which the adults were transferred to a new plate containing fresh dsRNA-treated leaf discs. After a further 24 hours, the adults from each treatment were collected and placed in a plastic<br><br>
box with dimensions 30 cm x 15 cm x 10 cm containing two potted and untreated 3-week-old bean plants. Insect mortality was assessed from day 4 until day ! 1.<br>
All three target dsRNAs (Ev010, Ev015 and Ev016) ingested by adults of Epilachna varivestis resulted in significant increases in mortality from day 4 (4 days post bioassay start), as shown in Figure 2(a)-EV. From day 5, dramatic changes in feeding patterns were observed between insects fed initially with target-dsRNA-treated bean leaf discs and those that were fed discs containing control gfp dsRNA or surfactant Triton X-IOO. Reductions in foliar damage by MBB adults of untreated bean plants were clearly visible for all three targets when compared to gfp dsRNA and surfactant only controls, albeit at varying levels; insects fed target 15 caused the least damage to bean foliage (Figure 2(b)-EV).<br>
E,	Cloning of a MBB gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an example of cloning a DNA fragment corresponding to an MLB gene target in a vector for the expression of double-stranded RNA in a bacterial host, although any vector comprising a 'f7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8-EV. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72^C, followed by 10 minutes at 72^C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into Sr/I-linearized pGNA49A vector (reference to WO00188121AI), and sequenced. The sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8-EV. The recombinant vector harbouring this sequence is named pGBNJOOXX.<br>
F,	Expression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to express suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselM-deficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation ofAB309'I05 andBL2l(DE3)<br><br>
Three hundred ng of the plasmid are added to and gently mixed in a 50µl aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL2I(DE3). The cells are incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty µL of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 T for 1 hour. One hundred ^l of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 iµg/ml carbenicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) at 37 °C  overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA expression inAB309~105 andBL21(DE3)<br>
Expression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309~105 or BL21(DE3) is made possible since all the genetic components for controlled expression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the 17 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are flanked by oppositely oriented f 7 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the cuUure then centrifuged at 3000 g at 15 T for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 [ag/ml cholesterol) supplemented with 100 |ig/ml carbenicillin and 1 mM IFTG. The bacteria are induced for 2 to 4 hours at room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimize the risk of contamination of the artificial diet in the test plates. However, heal treatment of bacteria expressing double-stranded RNA is not a prerequisite for inducing toxicity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 "C for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment, fhe tubes are stored at -20 °C until further use.<br><br>
G.        Laboratory trials to test Escherichia coli expressing dsRNA targets against<br>
Epilachna varivetis<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria expressing dsRNA prior to feeding the plants to MBB. fhe are grown from in a plant growth room chamber. The plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. MMB are placed on each treated plant in the cage. Plants are treated with a suspension of £. coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 10^ bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of £. coli bacterial strain AB3G9-105 expressing target dsRMA from pGBNJ003 lead to a dramatic increase in insect mortality when compared to pGN29 control. These experiments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial expression systems is toxic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these experiments that an exemplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria expressing double-stranded RNA corresponding to an insect gene target.<br>
Example 6: Anthonomus grandis (cotton boll weevil)<br>
A*        Cloning Anthonomus grandis partial sequences<br>
High quality, intact RNA was isolated from the 3 instars of Anthonomus grandis (cotton boll weevil; source: Dr. Gary Benzon, Benzon Research Inc., 7 Kuhn Drive, Carlisle, Pennsylvania 17013, USA) using TRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manafacturer's   instructions  (Cat.   Nr.   1700,   Promega).   cDNA   was   generated   using  a<br><br>
commercially available kit (Superscript TM  111 Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
fo isolate cDNA sequences comprising a portion of the AGOG], AG005, AGO 10, AGO 14 and AGO 16 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat, Nr, N8080240, Applied Biosystems) following the manafacturer's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-AG. These primers were used in respective PCR reactions with the following conditions: for AGOOl, AG005 and AG0i6, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95^C, 1 minute at 50"C and 1 minute and 30 seconds at 72°C, followed by 7 minutes at 72X; for AG010, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 54°C and 2 minutes and 30 seconds at 72°C, followed by 7 minutes at 72°C; for AGO 14, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 55°C and 1 minute at 72°C, followed by 7 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QlAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR8/GW/T0P0 vector (Cat. Nr. K2500-20, Invitrogen) and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO;s as given in Table 2-AG and are referred to as the partial sequences. The corresponding partial amino acid sequence are represented by the respective SEQ ID NO:s as given in Table 3-AG.<br>
B,        dsRNA production of the Anthonomus gmndis (cotton boll weevil) genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit T7 Ribomax^^ Express RNAi System (Cat. Nr, P1700, Promega), First two separate single 5' T7 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 17 promoter.<br>
For each of the target genes, the sense 17 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-AG. A touehdown PCR was performed as follows: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95°C, 30 seconds at 60°C with a decrease in temperature of 0.5°C per cycle and 1 minute at 72°C , followed by 15 cycles of 30 seconds at 95°C, 30 seconds at 50°C and 1 minute at 72°C, followed by 10 minutes at 72°C, The anti-sense T7 template was generated using specific forward and specific T7 reverse primers in a PCR reaction with the same conditions as<br><br>
described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-AC The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaC104 precipitation. The generated T7 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-AG.<br>
C,        Laboratory trials to test dsRNA targets, using artificial diet for activity against<br>
the larvae of the house cricket  Acheta domesticus<br>
House crickets, Acheta domesticus, were maintained at Insect Investigations Ltd. (origin: Blades Biological Ltd., Kent, UK). The insects were reared on bran pellets and cabbage leaves. Mixed sex nymphs of equal size and no more than 5 days old were selected for use in the trial. Double-stranded RNA was mixed with a wheat-based pelleted rodent diet (rat and mouse standard diet, B &amp; K Universal Ltd., Grimston, Aldbrough, Hull, UK). The diet, BKOOIP, contains the following ingredients in descending order by weight: wheat, soya, wheatfeed, barley, pellet binder, rodent 5 vit min, fat blend, dicalcium phosphate, mould carb. The pelleted rodent diet was finely ground and heat-treated in a microwave oven prior to mixing, in order to inactivate any enzyme components. All rodent diet was taken from the same batch in order to ensure consistency. The ground diet and dsRNA were mixed thoroughly and formed into small pellets of equal weight, which were allowed to dry overnight at room temperature.<br>
Double-stranded RNA samples from targets and gfp control at concentrations 10 µ g/µ l are applied in the ratio 1 g ground diet plus 1 ml dsRNA solution, thereby resulting in an application rate of 10 mg dsRNA per g pellet. Pellets are replaced weekly. The insects are provided with treated pellets for the first three weeks of the trial. Thereafter untreated pellets are provided. Insects are maintained within lidded plastic containers (9 cm diameter, 4.5 cm deep), ten per container. Each arena contains one treated bait pellet and one water source (damp cotton wool ball), each placed in a separate small weigh boat. The water is replenished adlib throughout the experiment.<br><br>
Assessments are made at twice weekly intervals, with no more than four days between assessments, until all the control insects had either died or moulted to the adult stage (84 days). At each assessment the insects are assessed as live or dead, and examined for abnormalities. From day 46 onwards, once moulting to adult commences, all insects (live and dead) are assessed as nyumph or adult. Surviving insects are weighed on day 55 of the trial. Four replicates are performed for each of the treatments. During the trial the test conditions are 25 to 33 °C and 20 to 25 % relative humidity, with a 12:12 hour light:dark photoperiod.<br>
D.        Cloning of a MLB gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an example of cloning a DNA fragment corresponding to an MLB gene target in a vector for the expression of double-stranded RNA in a bacterial host, although any vector comprising a T7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98^C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72^C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into Srf 1-linearized pGNA49A vector (reference to WO00188121A1), and sequenced. The sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8. The recombinant vector harbouring this sequence is named pGBNJOOXX.<br>
E.        Expression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to express suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselll-deficicnt strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation ofAB309-105 andBL2l(DE3)<br>
Three hundred ng of the plasmid are added to and gently mixed in a 50 µl aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 mi utes before subjecting them to a heat shock treatment of 37 "C for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred<br><br>
and fifty µL of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 °C for 1 hour. One hundred \x\ of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 µg/ml carbcnicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) at 37 T overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA expression inAB309~105 and BL21(DE3)<br>
Expression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled expression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the r7 polymerase will drive the transcription of the target sequence in both antiscnse and sense directions since these are flanked by oppositely oriented T7 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15oC for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 |ag/ml cholesterol) supplemented with 100µ g/ml carbcnicillin and 1 mM IFFG. The bacteria arc induced for 2 to 4 hours at room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria expressing double-stranded RNA is not a prerequisite for inducing toxicity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 °C  for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes arc stored at -20 X until further use.<br>
F,	Laboratory trials to test Escherichia coli expressing dsRNA targets against<br>
Antlionomus grandis<br>
Plant-based bioassays<br><br>
Whole plants are sprayed with suspensions of chemically induced bacteria expressing dsRNA prior to feeding the plants to CBW, The are grown from in a plant growth room chamber. The plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. CBW are placed on each treated plant in the cage. Plants are treated with a suspension of E. coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 10 bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength, in each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension off", coli bacterial strain AB309-I05 expressing target dsRNA from pGBNJ003 lead to a dramatic increase in insect mortality when compared to pGN29 control. These experiments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial expression systems is toxic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these experiments that an exemplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria expressing double-stranded RNA corresponding to an insect gene target.<br>
Example 7: Tribolium castaneum (red flour beetle)<br>
A.        Cloning Triholium castaneum partial sequences<br>
High quality, intact RNA was isolated from all the different insect stages o^ Tribolium castaneum (red fiour beetle; source; Dr. Lara Senior, Insect Investigations Ltd., Capital Business Park, Wentloog, Cardiff, CF3 2PX, Wales, UK) using TRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manafacturer's instructions (Cat. Nr. 1700, Promega). cDNA was generated using a commercially available kit (Superscript "^ 111 Reverse Transcriptase, Cat.<br><br>
Nr.   18080044,   Invitrogen,   Rockville,   Maryland,   USA)   following   the   manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the TCOOl, TC002, TCOlO, TC014 and TC015 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat. Nr. N8080240, Applied Biosystems) following the manafacturer's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-TC. These primers were used in respective PCR reactions with the following conditions: 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at SO'°C  and 1 minute and 30 seconds at 72°C, followed by 7 minutes at 72°C (TCOOl, VCQ\4, TC015); 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C , 1 minute at 54°C and 2 minutes and 30 seconds at 72°C, followed by 7 minutes at 72'°C  (TCOlO); 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 53°C and 1 minute at 72°C, followed by 7 minutes at 72°C  (TC002) . The resulting PCR fragments were analyzed on agarose gel, purified (QlAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR8/GW/T0P0 vector (Cat. Nr. K2500-20, Invitrogen), and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-TC and are referred to as the partial sequences. The corresponding partial amino acid sequences are represented by the respective SEQ ID NO:s as given in Table 3-TC.<br>
B,        dsRNA production of the TrihoUum castaneum genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit 17 RibomaxTM  Express RNAi System (Cat. Nr. PI700, Promcga). First two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 17 promoter.<br>
For each of the target genes, the sense T7 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-TC. The conditions in the PCR reactions were as follows: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95°C, 30 seconds at 60°C (-0.5°C/cycle) and 1 minute at 72°C, followed by 15 cycles of 30 seconds at 95°C , 30 seconds at 50°C and 1 minute at 72°C, followed by 10 minutes at 72°C. The anti-sense T7 template was generated using specific forward and specific 17 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the<br><br>
respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-TC. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaC104 precipitation. The generated T7 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, UNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-TC,<br>
C.        Laboratory trials to test dsRNA targets, using artificial diet for activity<br>
against Tribolium castaneum larvae<br>
The example provided below is an exemplification of the finding that the red fiour beetle (RFB) larvae are susceptible to orally ingested dsRNA corresponding to own target genes.<br>
Red fiour beetles, Tribolium castaneum, were maintained at Insect Invesfigafions Ltd. (origin: Imperial College of Science, Technology and Medicine, Silwood Park, Berkshire, UK). Insects were cultured according to company SOP/251/01. Briefly, the beetles were housed in plastic jars or tanks. These have an open top to allow ventilation. A piece of netting was fitted over the top and secured with an elastic band to prevent escape. The larval rearing medium (fiour) was placed in the container where the beetles can breed. The stored product beetle colonies were maintained in a controlled temperature room at 25 ± 3 °C with a 16:8 hour light:dark cycle.<br>
Double-stranded RNA from target TC014 (with sequence corresponding to SEQ ID NO: -799) was incorporated into a mixture of fiour and milk powder (wholemeal flour: powdered milk in the ratio 4:1) and left to dry overnight. Each replicate was prepared separately: 100 µl of a 10 \i%l\x\ dsRNA solution (1 mg dsRNA) was added to 0.1 g fiour/milk mixture. The dried mixture was ground to a fine powder. Insects were maintained within Petri dishes (55 mm diameter), lined with a double layer of filter paper. The treated diet was placed between the two filter paper layers, fen first instar, mixed sex larvae were placed in each dish (replicate). Four replicates were performed for each treatment. Control was Milli-Q water. Assessments (number of survivors) were made on a regular basis. During the trial, the test conditions were 25-33 T and 20-25 % relative humidity, with a 12:12 hour light:dark photoperiod.<br><br>
Survival of larvae of T  castaneum over time on artificial diet treated with target TC014 dsRNA was significantly reduced when compared to diet only control, as shown In Figure 1.<br>
D.        Cloning of a RFB gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an example of cloning a DNA fragment corresponding to an RFB gene target in a vector for the expression of double-stranded RNA in a bacterial host, although any vector comprising a T7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8-TC. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into Srf -linearized pGNA49A vector (reference to WO00188121A1), and sequenced, The sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8-TC. The recombinant vector harbouring this sequence is named pGBNJOO XX.<br>
E.        Expression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to express suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselll-deficient strain, AB309-I05, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation ofAB309-105 and BL21(DE3)<br>
Three hundred ng of the plasmid are added to and gently mixed in a 50 \\\ aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C  for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty µL of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 ^C for 1 hour. One hundred µ l of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani<br><br>
(LB) broth supplemented with 100 |ag/ml carbenicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) al 37 °C  overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA expression inAB309-105 and BL21(DE3)<br>
Expression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-I05 or BL21(DE3) is made possible since all the genetic components for controlled expression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the T7 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are flanked by oppositely oriented T7 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 °C for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 µ glµ l cholesterol) supplemented with 100 (ig/ml carbenicillin and 1 mM IPTG. The bacteria are induced for 2 to 4 hours al room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria expressing double-stranded RNA is not a prerequisite for inducing toxicity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 °C for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes are stored at -20 °C until further use.<br>
F.	Laboratory trials to test Escherichia coli expressing dsRNA targets against<br>
Tribolium castaneum<br>
Plant-based bioassavs<br>
Whole plants are sprayed with suspensions of chemically induced bacteria expressing dsRNA prior to feeding the plants to RFB. Ihe are grown from in a plant growth room chamber. Ihe plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered<br><br>
with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. RFB are placed on each treated plant in the cage. Plants are treated with a suspension of A; coli AB309™105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 10 bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength, in each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of E . coli bacterial strain AB309-105 expressing target dsRNA from pGBNJ003 leed to a dramatic increase in insect mortality when compared to pGN29 control. These experiments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial expression systems is toxic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these experiments that an exemplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria expressing double-stranded RNA corresponding to an insect gene target.<br>
Example 10: MvzMS persicae (arccn peach aphid)<br>
A.        Cloning Myzus persicae partial sequences<br>
High quality, intact RNA was isolated from nymphs of Myzus persicae (green peach aphid; source: Dr. Rachel Down, Insect &amp; Pathogen Interactions, Central Science Laboratory, Sand Hutton, York, Y041 ILZ, UK) using TRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manafacturer's instructions (Cat. Nr. 1700, Promega). cDNA was generated using a commercially available kit (Superscript ™ 111 Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the MPOOl, MP002, MP010, MP016 and MP027 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat. Nr. N8080240, Applied Biosyslems) following the manafacturer's instructions.<br><br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-MP, These primers were used in respective PCR reactions with the following conditions: for MPOOl, MP002 and IV1P016, 10 minutes at 95°C , followed by 40 cycles of 30 seconds at 95°C, 1 minute at 50°C  and 1 minute 30 seconds at 72°C, followed by 7 minutes at 72°C; for MP027, a touchdown program was used: 10 minutes at 95°C, followed by 10 cycles of 30 seconds at 95°C, 40 seconds at 60°C with a decrease in temperature of TC per cycle and 1 minute !0 seconds at 72°C, followed by 30 cycles of 30 seconds at 95°C, 40 seconds at 50°C and I minute 10 seconds at 72°C, followed by 7 minutes at 72°C; for MPOlO, 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 54°C and 3 minutes at 72°C, followed by 7 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Mr, 28706, Qiagen), cloned into the pCR8/GW/T0P0 vector (Cat. Nr, K2500-20, Invitrogen), and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-MP and are referred to as the partial sequences. The corresponding partial amino acid sequences are represented by the respective SEQ ID NO:s as given in Table 3-MP.<br>
B.        dsRNA production of Myzus persicae genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit T7 RibomaxTM  Express RNAi System (Cat. Nr. P1700, Promega). First two separate single 5' T7 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the T7 promoter.<br>
For each of the target genes, the sense T7 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-MP. A touchdown PCR was performed as follows: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95°C , 30 seconds at SS^µ  (for MPOOl, MP002, MP016, MP027 and gfp) or 30 seconds at 50°C (for MPOlO) with a decrease in temperature of 0,5°C per cycle and 1 minute at 72°C, followed by 15 cycles of 30 seconds at 95°C, 30 seconds at 45°C  and 1 minute at 72°C  followed by 10 minutes at 72°C. The anti-sense T7 template was generated using specific forward and specific 17 reverse primers in a PCR reaction with the same conditions as described above. 1 he sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-MP. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106,<br><br>
Qiagen) and NaCI04 precipitation. The generated T7 forward and reverse templates were mixed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-MP.<br>
C.        Laboratory trials to test dsRNA targets using liquid artiflcial diet for<br>
activity sigmnst Myzus persicae<br>
Liquid artificial diet for the green peach aphid, Myzus persicae, was prepared based on the diet suitable for pea aphids {Acyrthosiphon pisum), as described by Febvay et al. (1988) [Influence of the amino acid balance on the improvement of an artificial diet for a biotype of Acyrthosiphonpisum (Homoptera: Aphididae). Can. J. Zool. 66: 2449-2453], but with some modifications. The amino acids component of the diet was prepared as follows: in mg/100ml, alanine 178.71, beta-alanine 6.22, arginine 244,9, asparagine 298,55, aspartic acid 88.25, cysteine 29.59, glutamic acid 149.36, glutamine 445.61, glycine 166.56, histidine 136.02, isoleucine 164.75, leucine 231.56, lysine hydrochloride 351.09, methionine 72.35, ornithine (liCl) 9.41, phenylalanine 293, proline 129.33, serine 124.28, threonine 127.16, tryptophane 42.75, tyrosine 38,63, L-valine 190.85. The amino acids were dissolved in 30 ml Milli-0 H2O except for tyrosine which was first dissolved in a few drops of 1 M HCl before adding to the amino acid mix. The vitamin mix component of the diet was prepared as a 5 x concentrate stock as follows: in mg/L, amino benzoic acid 100, ascorbic acid 1000, biotin 1, calcium panthothenate 50, choline chloride 500, folic acid 10, myoinositol 420, nicotinic acid 100, pyridoxine hydrochloride 25, riboflavin 5, thiamine hydrochloride 25. The riboflavin was dissolved in 1 ml H20 at 50 "C and then added to the vitamin mix stock. Ihe vitamin mix was aliquoted in 20 ml per aliquot and stored at -20 °C . One aliquot of vitamin mix was added to the amino acid solution. Sucrose and MgSO4.7H2O was added with the following amounts to the mix: 20 g and 242 mg, respectively. Trace metal stock solution was prepared as follows: in mg/100mI, CUSO4.5H2O 4.7, FeCb.eHsO 44.5, MnCl2.4H20 6,5, NaCl 25.4, ZnCb 8.3.  fen ml of the trace metal solution and 250 mg KH2PO4 was added to the diet and Milli-Q water was added to a final liquid diet volume of 100 ml. The pH of the diet was adjusted to 7 with 1 M KOH solution. The liquid diet was filter-sterilised through an 0.22 ^m filter disc (Millipore).<br>
Green peach aphids {Myzmpersicae\ source: Dr. Rachel Down, Insect &amp; Pathogen Interactions, Central Science Laboratory, Sand Hutton, York, Y041 ILZ, UK) were reared<br><br>
on 4- to 6-week-old oilseed rape (Brassica napus variety SW Oban; source: Nick Balaam, Sw Seed Ltd., 49 North Road, Abington, Cambridge, CBI 6AS, UK) in aluminium-framed cages containing 70µm mesh in a controlled environment chamber with the following conditions: 23 ±2 °C and 60 ±5 % relative humidity, with a 16:8 hours light:dark photopcriod.<br>
One day prior to the start of the bioassay, adults were collected from the rearing cages and placed on fresh detached oilseed rape leaves in a Petri dish and left overnight in the insect chamber. The following day, first-instar nymphs were picked and transferred to feeding chambers. A feeding chamber comprised of 10 first instar nymphs placed in a small Petri dish (with diameter 3 cm) covered with a single layer of thinly stretched parafilm M onto which 50 µl of diet was added. The chamber was scaled with a second layer of parafilm and incubated under the same conditions as the adult cultures. Diet with dsRNA was refreshed every other day and the insects' survival assessed on day 8 i.e. 8^ day post bioassay start. Per treatment, 5 bioassay feeding chambers (replicates) were set up simultaneously. Test and control (gfp) dsRNA solutions were incorporated into the diet to a final concentration of 2 µg/µl. The feeding chambers were kept at 23 ±2 °C and 60 ±5 % relative humidity, with a 16:8 hours light:dark photopcriod. A Mann-Whitney test was determined by GraphPad Prism version 4 to establish whether the medians do differ significantly between target 27 (MP027) and gfp dsRNA.<br>
In the bioassay, feeding liquid artificial diet supplemented with intact naked dsRNA from target 27 (SBQ ID NO: 1061) to nymphs of Myzuspersicae using a feeding chamber, resulted in a significant increase in mortality, as shown in Figure 1. Average percentage survivors for target 27, gfp dsRNA and diet only treatment were 2, 34 and 82, respectively. Comparison of target 027 with gfp dsRNA groups using the Mann-Whitney test resulted in an one-tailed P-value of 0.004 which indicates that the median of target 027 is significantly different (P 
D.        Cloning of a GPA gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an example of cloning a DNA fragment corresponding to a GPA gene target in a vector for the expression of double-stranded RNA in a bacterial host, although any<br><br>
vector comprising a T7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8-^MP. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel Extraction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into .S&gt;/l-linearized pGNA49A vector (reference to WO0018812IA1), and sequenced, l^he sequence of the resulting PCR product corresponds to the respective sequence as given in Tabic 8-MP. The recombinant vector harbouring this sequence is named pGBNJOOXX.<br>
E.        Expression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to express suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselll-deficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation ofAB309-105 andBL2l(DE3)<br>
Three hundred ng of the plasmid are added to and gently mixed in a 50 µL aliquot of ice-chilled chemically competent E, coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty µL of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 °C for 1 hour. One hundredµl of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 (ig/ml carbenicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) at 37 °C overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA expression inAB309-105 andBL21(DE3)<br>
Expression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled expression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the T7 polymerase will drive the transcription of the target<br><br>
sequence in both antisense and sense directions since these are flanked by oppositely oriented 17 promoters.<br>
rhe optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 °C for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 µg/ml cholesterol) supplemented with 100 |ig/ml carbenicillin and I mM IPTG, The bacteria are induced for 2 to 4 hours at room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the lest plates. However, heat treatment of bacteria expressing double-stranded RNA is not a prerequisite for inducing toxicity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 *°C  for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1,5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment, the tubes are stored at -20 T until further use.<br>
F.        Laboratory trials to test Escherichia coli expressing dslWA targets against<br>
Myzus persicae<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria expressing dsRNA prior to feeding the plants to GPA. The are grown from in a plant growth room chamber. The plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. GPA are placed on each treated plant in the cage. Plants are treated with a suspension of E, coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 109 bacterial cells in I ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between 1 and 10 ml s sprayed on the<br><br>
plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of E. coll bacterial strain AB309-105 expressing target dsRNA from pGBNJ003 lead to a dramatic increase in insect mortality when compared to pGN29 control. These experiments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial expression systems is toxic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these experiments that an exemplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria expressing double-stranded RNA corresponding to an insect gene target.<br>
Example U: Nilaparvata lusens (Brown Plant Hopper) A.        Cloning Nilaparvata lugens partial sequences<br>
From high quality total RNA of Nilaparvata lugens (source: Dr. J. A. Gatehouse, Dept. Biological Sciences, Durham University, UK) cDNA was generated using a commercially available kit (SuperScriptTM HI Reverse Transcriptase, Cat N°, 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's protocol.<br>
To isolate cDNA sequences comprising a portion of the Nilaparvata lugens NLOOI, NL002, NL003, NL004, NL005, NL006, NL007, NL008, NL009, NL010, NLOll, NL012, NL0I3, NL014, NL015, NL016, NL018, NL019, NL021, NL022, and NL027 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat N°. N8080240; Applied Biosystcms) following the manufacturer's protocol.<br>
rhe sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-NL. These primers were used in respective PCR reactions with the following conditions: for NLOOl: 5 minutes at QS'°C , followed by 40 cycles of 30 seconds at 95°C, 1 minute at 55°C and 1 minute at 72°C, followed by 10 minutes at 72°C: for NL002: 3 minutes at 95°C followed by 40 cycles of 30 seconds at 95°C, 1 minute at SS'°C  and 1 minute at 72°C, followed by 10 minutes at 72°C; for NL003: 3 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 61 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL004: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C; 1 minute at 51 °C and 1 minute at 72 °C; for NL005: 10 minutes at 95 °C , followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 1 minute at 72°C, followed by 10 minutes at 72°C; for NL006: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95<br><br>
O/"^<br>
C, I minute at 55 °C and 3 minute 30 seconds at 72 °C, followed by 10 minutes at 72°C ; for NL007: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 1 minute 15 seconds at 72 °C, followed by 10 minutes at 72°C; for NL008: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 53 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL009: 10 minutes at 95 b°C followed by 40 cycles of 30 seconds at 95 °C, I minute at 55 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL010: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 2 minute 30 seconds at 72 "C, followed by 10 minutes at 1TC\ for NLOl I: 10 minutes at 95 °C  followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 55 °C and 1 minute at 72 °C ; for NL012: 10 minutes at 95 °C  followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 55 °C and 1 minute at 72 °C ; for NL013: 10 minutes al 95 °C  followed by 40 cycles of 30 seconds at 95 ""C, 1 minute at 54 °C and 1 minute 10 seconds at 72 °C, followed by 10 minutes at 72°C ; for NL0I4: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 53 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL015: 10 minutes at 95 °C  followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 1 minute 40 seconds at 72 °C, followed by 10 minutes at 72°C; for NL016: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 1 minute 40 seconds at 72 °C, followed by 10 minutes at 72°C; forNLOlS; 10 minutes at 95 °C  followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 1 minute 35 seconds at 72 °C, followed by 10 minutes at 72°C ; for NL019: 10 minutes at 95 °C  followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 55 °C  and 1 minute at 72 °C  followed by 10 minutes at 72°C; for NL021: 10 minutes at 95 °C  followed by 40 cycles of 30 seconds at 95 °C, I minute at 54 °C  and 1 minute 45 seconds al 72 °C, followed by 10 minutes at 72°C : for NL022; 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, I minute at 54 °C and 1 minute 45 seconds at 72 °C, followed by 10 minutes at 72°C; and for NL027: 10 minutes at 95 °C, followed by 40 cycles of 30 seconds at 95 °C, 1 minute at 54 °C and 1 minute 45 seconds at 72 °C, followed by 10 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QlAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR8/GW/topo vector (Cat. Nr. K2500 20, Invitrogen), and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-NL and are referred to as the partial sequences. The corresponding partial amino acid sequences are represented by the respective SEQ ID NO:s as given in Table 3-NL.<br><br>
B.        Cloning of a partial sequence of the Nilaparvata lugens NL023 gene via EST<br>
sequence<br>
From high quality total RNA of Nilaparvata lugens (source: Dr. J. A. Gatehouse, Dept. Biological Sciences, Durham University, UK) cDNA was generated using a commercially available kit (SuperscriptTM  III Reverse Transcriptase, Cat N°. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's protocol.<br>
A partial cDNA sequence, NL023, was amplified from Nilaparvata lugens cDNA which corresponded to a Nilaparvata lugens EST sequence in the public database Genbank with accession number CAH65679.2. To isolate cDNA sequences comprising a portion of the NL023 gene, a series of PCR reactions with EST based specific primers were performed using PerfectShot™ E°C Taq (Cat N°, RR005A, Takara Bio Inc.) following the manafacturer's protocol.<br>
For NL023, the specific primers oGBKW002 and oGBKWOOS (represented herein as SEQ ID NO: 1157 and SEQ ID NO: 1158, respectively) were used in two independent PCR reactions with the following conditions: 3 minutes at 95 °C, followed by 30 cycles of 30 seconds at 95 °C, 30 seconds at 56 °C  and 2 minutes at 72 °C, followed by 10 minutes at 72°C. The resulting PCR products were analyzed on agarose gel, purified (QlAquick® Gel E°C traction Kit; Cat. N". 28706, Qiagen), cloned into the pCR4-1^0PO vector (Cat N°. K4575-40, Invitrogen) and sequenced. The consensus sequence resulting from the sequencing of both PCR products is herein represented by SEQ ID NO:  1111 and is referred to as the partial sequence of the NL023 gene. The corresponding partial amino acid sequence is herein reperesented as SEQ ID NO: 1112.<br>
C.        dsRNA production of Nilaparvata lugens genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit T7 Riboma°C '^ E°C press RNAi System (Cat. Nr. PI700, Promega). First two separate single 5' T7 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the M promoter.<br>
For each of the target genes, the sense T7 template was generated using specific r7 forward and specific reverse primers, fhe sequences of the respective primers for amplifying the sense template for each of the target genes are given in fable 4. The conditions in the PCR reactions were as follows: for NLOOl: 4 minutes at 94 °C, followed by 35 cycles of 30 seconds at 94 °C, 30 seconds at 60 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL002: 4 minutes at 94 °C, followed by 35 cycles of 30 seconds at 94 °C, 30<br><br>
seconds at 60 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL003: 4 minutes at 94 °C, followed by 35 cycles of 30 seconds at 94 °C, 30 seconds at 66 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C ; for NL004: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 54 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL005: 4 minutes at 95 "C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 57 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL006: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 54 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL007: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 51 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NLOOS; 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 54 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL009: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 54 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL010: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 54 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NLOll: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 53 °C and I minute at 72 °C, followed by 10 minutes at 72°C; for NL012: 4 minutes at 95 °C, followed by 35 cycles of 30secondes at 95 °C, 30 seconds at 53 °C and 1 minute at 72 °C , followed by 10 minutes at 72°C; for NL0I3: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 55 "C and 1 minute at 72 °C, followed by 10 minutes at ll°C ; for NL014: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 51 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL015: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 55 °C and I minute at 72 °C, followed by 10 minutes at 72°C; for NL016: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 57 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL0I8: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 55 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL019: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 54 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL021: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 55 °C and I minute at 72 °C, followed by 10 minutes at 72°C; for NL022: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 53 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C; for NL023: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 52 °C and I minute at 72 °C, followed by 10 minutes at 72°C; and for NL027: 4 minutes at 95 °C, followed by 35 cycles of<br><br>
30secondes at 95 °C, 30 seconds at 52 "C and 1 minute at 72 °C, followed by 10 minutes at 72°C. The anti-sense T7 template was generated using specific forward and specific T7 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 4-NL. The resulting PCR products were analyzed on agarose gel and purified by PCR puritlcation kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen). The generated T7 forward and reverse templates were mi°C ed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions, but with the following modification: RNA pcppet is washed twice in 70% ethanol. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-NL.<br>
The template DNA used for the PCR reactions with T7 primers on the green fiuorescent protein (gfp) control was the plasmid pPD96,12 (the Fire Lab, http://genome-www.stanford.edu/group/fire/), which contains the wild-type gfp coding sequence interspersed by 3 synthetic introns. Double-stranded RNA was synthesized using the commercially available kit 17 RiboMA°C ™ E°C press RNAi System (Cat,N^ P1700, Promega). First two separate single 5' 'V7 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the T7 promoter. For gfp, the sense 77 template was generated using the specific T7 FW primer oGAUI83 and the specific RV primer oGAU182 (represented herein as SEQ ID NO: 236 and SEQ ID NO: 237 , respectively) in a PCR reaction with the following conditions: 4 minutes at 95 °C, followed by 35 cycles of 30 seconds at 95 °C, 30 seconds at 55 °C and 1 minute at 72 °C, followed by 10 minutes at 72°C. 'fhe anti-sense 17 template was generated using the specific FW primer 0GAUI8I and the specific T7 RV primer oGAU184 (represented herein as SEQ ID NO: 238 and SEQ ID NO; 239 , respectively) in a PCR reaction with the same conditions as described above. The resulting PCR products were analyzed on agarose gel and purified (QIAquick® PCR Purification Kit; Cat. N°. 28106, Qiagen). The generated T7 FW and RV templates were mi°C ed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by precipitation with sodium acetate and isopropanol, following the manufacturer's protocol, but with the following modification: RNA peppet is washed twice in 70% ethanol. 1 he sense strands of the resulting dsRNA is herein represented by SEQ ID NO: 235.<br><br>
D.        Laboratory trials to screen dsRNA targets using liquid artificial diet for activity<br>
against Nilaparvata lugens<br>
Liquid artificial diet (MMD-1) for the rice brown pianthopper, Nilaparvata lugens, was prepared as described by Koyama (1988) [Artificial rearing and nutritional physiology of the planthoppcrs and leafhoppers (Homoptcra: Delphacidae and Deltocephalidae) on a holidic diet. JARQ 22: 20-27], but with a modification in final concentration of diet component sucrose; 14.4 % (weight over volume) was used. Diet components were prepared as separate concentrates: 10 °C  mineral stock (stored at 4 °C ), 2 °C  amino acid stock (stored at -20 "C) and 10 °C  vitamin stock (stored at -20 °C). The stock components were mi°C ed immediately prior to the start of a bioassay to 4/3 °C  concentration to allow dilution with the test dsRNA solution (4 °C  concentration), pH adjusted to 6.5, and filter-sterilised into appro°C imately 500 |al aliquots.<br>
Rice brown pianthopper {Nilaparvata lugens) was reared on two-to-three month old rice {Oryza saliva cv Taichung Native 1) plants in a controlled environment chamber: 27 ± 2 ""C, 80 % relafive humidity, with a 16:8 hours light:dark photoperiod. A feeding chamber comprised 10 first or second instar nymphs placed in a small petri dish (with diameter 3 cm) covered with a single layer of thinly stretched parafilm M onto which 50 µl of diet was added. The chamber was sealed with a second layer of parafilm and incubated under the same conditions as the adult cultures but with no direct light e°C posure. Diet with dsRNA was refreshed every other day and the insects' survival assessed daily. Per treatment, 5 bioassay feeding chambers (replicates) were set up simultaneously. Test and control (gfp) dsRNA solutions were incorporated into the diet to a final concentration of 2 mg/ml. The feeding chambers were kept at 27 ± 2 T, 80 % relative humidity, with a 16:8 hours light:dark photoperiod. Insect survival data were analysed using the Kaplan-Meier survival curve model and the survival between groups were compared using the logrank test (Prism version 4.0).<br>
Feeding liquid artificial diet supplemented with intact naked dsRNAs to Nilaparvata lugens in vitro using a feeding chamber resulted in significant increases in nymphal mortalities as shown in four separate bioassays (Figures l(a)-(d)-NL; Tables la-d-NL). These results demonstrate that dsRNAs corresponding to different essential BPH genes showed significant to°C icity towards the rice brown pianthopper.<br>
Effect of gfp dsRNA on BPII survival in these bioassays is not significantly different<br>
to survival on diet only<br>
Tables lOa-d-NL show a summary of the survival of Nilaparvata lugens on artificial diet supplemented with 2 mg/ml (final concentration) of the following targets; in Table 10(a)-NL: NL002, NL003, NL005, NL010; in Table 10(b)-NL NL009, NL016; in Table<br><br>
10(c)-NL NL0I4, NL0I8; and in Table 10(d)-NL NL0!3, NL0I5, NL02I. In the survival analysis column, the effect of RNAi is indicated as follows: + = significantly decreased survival compared to gfp dsRNA control (alpha 
E.        Laboratory trials to screen dsRNAs at different concentrations using artificial<br>
diet for activity against Nilaparvata lugens<br>
Fiftyµl of liquid artificial diet supplemented with different concentrations of target NL002 dsRNA, namely 1, 0.2, 0.08, and 0.04 mg/ml (final concentration), was applied to the brown planthoppcr feeding chambers. Diet with dsRNA was refreshed every other day and the insects' survival assessed daily. Per treatment, 5 bioassay feeding chambers (replicates) were set up simultaneously. The feeding chambers were kept at 27 i 2 T, 80 % relative humidity, with a 16:8 hours light:dark photoperiod. Insect survival data were analysed using the Kaplan-Meier survival curve model and the survival between groups were compared using the logrank test (Prism version 4.0).<br>
Feeding liquid artificial diet supplemented with intact naked dsRNAs of target NL002 at different concentrations resulted in significantly higher BPH mortalities at final concentrations of as low as 0.04 mg dsRNA per ml diet when compared with survival on diet only, as shown in Figure 2-NL and Table 9-NL. Table 9-NL summarizes the survival of Nilaparvata higens artificial diet feeding trial supplemented with 1, 0.2, 0.08, &amp; 0.04 mg/ml (final concentration) of target NL002. in the survival analysis column the effect of RNAi is indicated as follows: + = significantly decreases survival compared to diet only control (alpha 
F.	Cloning of a BPH gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an e°C ample of cloning a DNA fragment corresponding to a BPH gene target in a vector for the e°C pression of double-stranded RNA in a bacterial host, although any vector comprising a T7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br><br>
The sequences of the specific primers used for the amplification of target genes are provided in fable 8. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into Srf I-linearized pGNA49A vector (reference to WO00188121A1), and sequenced. The sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8-NL. The recombinant vector harbouring this sequence is named pGBNJOO.<br>
G, E°C pression and production of a double-stranded RNA target in two strains of Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to e°C press suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselll-deficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation of AB309-105 and BL21(DE3)<br>
Three hundred ng of the plasmid are added to and gently mi°C ed in a 50 \°C \ aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C  for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty |al of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37°CC for 1 hour. One hundred [i\ of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 |ig/ml carbenicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) at 37 °C overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA e°C pression inAB309-105 and BL21(DE3)<br>
E°C pression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled e°C pression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the T7 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are fianked by oppositely oriented 'f7 promoters.<br><br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of I by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 °C  for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5µg/ml cholesterol) supplemented with 100 |ig/ml carbenicillin and 1 mM IPTG. The bacteria are induced for 2 to 4 hours at room temperature.<br>
Heal treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the test plates. Flowever, heat treatment of bacteria e°C pressing double-stranded RNA is not a prerequisite for inducing to°C icity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 °C for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes arc stored at -20 °C until further use.<br>
H. Laboratory trials to test Escherichia coli e°C pressing dsRNA targets against<br>
Nilaparvata lugens<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria e°C pressing dsRNA prior to feeding the plants to BPH. 7hc are grown from in a plant growth room chamber. The plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. BPl 1 are placed on each treated plant in the cage. Plants are treated with a suspension of K coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 10 bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br><br>
Spraying plants with a suspension of E. coli bacterial strain AB309-105 e°C pressing target dsRNA from pGBNJ003 leed to a dramatic increase in insect mortality when compared to pGN29 control. These e°C periments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial e°C pression systems is to°C ic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these e°C periments that an e°C emplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria e°C pressing double-stranded RNA corresponding to an insect gene target.<br>
E°C ample 10: CItilo suppressalis (rice striped stem borer)<br>
A,        Cloning of partial sequence of the Chilo suppressalis genes via family PCR<br>
High quality, intact RNA was isolated from the 4 different larval stages of Chilo suppressalis (rice striped stem borer) using TRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manafacturer's instructions (Cat, Nr. 1700, Promega). cDNA was generated using a commercially available kit (Superscript ^^ 111 Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the CSOOl, CS002, CS003, CS006, CS007, CS009, CSOl 1, CS013, CS014, CS015, CS016 and CS018 genes, a series of PCR reactions with degenerate primers were performed using Amplilaq Gold (Cat. Nr. N8080240, Applied Biosystems) following the manafacturer's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-CS. These primers were used in respective PCR reactions with the following conditions: 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, 1 minute at 55°C and 1 minute at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR4/T0P0 vector (Cat, Nr. K2500-20, Invitrogen), and sequenced. The sequences of the resulting PCR products are represented by the respective SHQ ID NO:s as given in Table 2-CS and are referred to as the partial sequences. The corresponding partial amino acid sequences arc represented by the respective SEQ ID NO;s as given in Table 3-CS.<br><br>
B,        dsRNA production of the Chilo suppressalis genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit \1 RibomaTM  E°C press RNAi System (Cat. Nr. PI700, Promega), First two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the T7 promoter.<br>
For each of the target genes, the sense T7 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-CS. The conditions in the PCR reactions were as follows: 4 minutes at 95°C, followed by 35 cycles of 30 seconds at 95°C, 30 seconds at 55°C and 1 minute at 72°C, followed by 10 minutes at 72°C. The anti-sense T7 template was generated using specific forward and specific T7 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Table 8-CS. The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaClO4 precipitation. The generated T7 forward and reverse templates were mi°C ed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Tabic 8-CS.<br>
C.	Laboratory trials to test dsRNA targets, using artificial diet for activity against Chilo<br>
suppressalis larvae<br>
Rice striped stem borers, Chilo suppressalis, (origin: Syngenta, Stein, Switzerland) were maintained on a modified artificial diet based on that described by Kamano and Sato, 1985 (in: Handbook of Insect Rearing. Volumes I &amp; II. P Singh and RF Moore, eds., Elsevier Science Publishers, Amsterdam and New York, 1985, pp 448). Briefly, a litre diet was made up as follows: 20 g of agar added to 980 ml of Milli-Q water and autoclaved; the agar solution was cooled down to appro°C imately 55 °C  and the remaining ingredients were added and mi°C ed thoroughly: 40 g corn flour (Polenta), 20 g cellulose, 30 g sucrose, 30 g casein, 20 g wheat germ (toasted), 8 g Wesson salt mixture, 12 g Vanderzant vitamin mi°C  1.8 g sorbic acid, 1.6 g nipagin (methylparaben), 0.3 g aureomycin, 0.4 g cholesterol and 0.6 g L-cysteine. fhe diet was cooled down to appro°C . 45o"C and poured into rearing trays or cups. The diet was left to set in   a horizontal laminair flow cabin. Rice leaf sections with oviposited eggs<br><br>
were removed from a cage housing adult moths and pinned to the solid diet in the rearing cup or tray. Eggs were left to hatch and neonate larvae were available for bioassays and the maintenance of the insect cultures. During the trials and rearings, the conditions were 28 ± 2 "C and 80 ± 5 % relative humidity, with a 16:8 hour light:dark photoperiod.<br>
The same artificial diet is used for the bioassays but in this case the diet is poured equally in 24 multiwell plates, with each well containing 1 ml diet. Once the diet is set, the test formulations are applied to the diet's surface (2 cm ), at the rate of 50 µL of 1µg/µl dsRNA of target. The dsRNA solutions are left to dry and two first instar moth larvae are placed in each well. After 7 days, the larvae are transferred to fresh treated diet in multiwell plates. At day 14 (i.e. 14 days post bioassay start) the number of live and dead insects is recorded and e°C amined for abnormalities. Twenty-four larvae in total are tested per treatment.<br>
An alternative bioassay is performed in which treated rice leaves are fed to neonate larvae of the rice striped stem borer. Small leaf sections of Indica rice variety Taichung native 1 are dipped in 0.05 % lYiton °C -100 solution containing I |ig/µl of target dsRNA, left to dry and each section placed in a well of a 24 multiwell plate containing gellified 2 % agar. 1 wo neonates are transferred from the rearing tray to each dsRNA treated leaf section (24 larvae per treatment). After 4 and 8 days, the larvae are transferred to fresh treated rice leaf sections. The number of live and dead larvae are assessed on days 4, 8 and 12; any abnormalities are also recorded.<br>
D.        Cloning of a SSB gene fragment in a vector suitable for bacterial production of<br>
insect-active doublc-strandcd RNA<br>
What follows is an e°C ample of cloning a DNA fragment corresponding to an SSB gene target in a vector for the e°C pression of double-stranded RNA in a bacterial host, although any vector comprising a T7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO000I846),<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C , 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel E°C traction kit. Cat. Nr. 28706, Qiagen), blunt-end cloned into Srf I-lineari/ed pGNA49A vector (reference to WOOOl 88121 Al), and sequenced. The sequence<br><br>
of the resulting PCR product corresponds to the respective sequence as given in Table 8-CS. The recombinant vector harbouring this sequence is named pGBNJOO°C °C .<br>
E,        E°C pression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to e°C press suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselll-deficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation of AB309-105 and BL21(DH3)<br>
Three hundred ng of the plasmid are added to and gently mi°C ed in a 50 µl aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 minutes before subjecting them lo a heat shock treatment of 37 °C  for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty µl of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 "C for 1 hour. One hundred \°C \ of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 \°C g/ml carbenicillin antibiotic. The cuhure is incubated on an Innova 4430 shaker (250 rpm) at 37 °C  overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA e°C pression inAB309-J05 and BL21(DE3)<br>
E°C pression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled e°C pression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the T7 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are flanked by oppositely oriented 17 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 °C for 10 minutes. The supernatant is removed and the bacterial pellet rcsuspended in 50 ml of fresh S complete medium (SNC medium plus 5 |ig/ml cholesterol) supplemented with 100 |ig/ml carbenicillin and 1 mM IFTG. Ihe bacteria are induced for 2 lo 4 hours at room temperature.<br>
Heat treatment of bacteria<br><br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria e°C pressing double-stranded RNA is not a prerequisite for inducing to°C icity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80pC for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes are stored at -20 °C until further use.<br>
F.	Laboratory trials to test Escherichia coli e°C pressing dsRNA targets against Chilo<br>
suppressalis<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria e°C pressing dsRNA prior to feeding the plants to SSB. The are grown from in a plant growth room chamber. The plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. SSB are placed on each treated plant in the cage. Plants are treated with a suspension of £. coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 10^ bacterial cells in I ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between I and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of E.. coli bacterial strain AB309-105 e°C pressing target dsRNA from pGBNJ003 leed to a dramatic increase in insect mortality when compared to pGN29 control. These e°C periments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial e°C pression systems is to°C ic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these e°C periments that an e°C emplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria e°C pressing double-stranded RNA corresponding to an insect gene target.<br><br>
E°C ample 9: Plutella °C xlostella (Diamondback moth)<br>
A.        Cloning of a partial sequence of the Plutella °C ylostella<br>
High quality, intact RNA was isolated from all the different larval stages of Plutella °C ylostella (Diamondback moth; source: Dr. Lara Senior, Insect Investigations Ltd., Capital Business Park, Wentloog, Cardiff, CF3 2P°C  Wales, UK) using TRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manufacturer's instructions (Cat. Nr. 1700, Promega). cDNA was generated using a commercially available kit (Superscript TM  III Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the P°C OOl, P°C 009, P°C 010, P°C 015, P°C 016 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat. Nr. N8080240, Applied Biosystcms) following the manufacturer's instructions.<br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-P°C . These primers were used in respective PCR reactions with the following conditions: 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C, I minute at 50°C and 1 minute and 30 seconds at 72°C, followed by 7 minutes at 72°C (for P°C OOl, P°C 009, P°C 015, P°C 016); 10 minutes at 95°C, followed by 40 cycles of 30 seconds at 95°C  1 minute at 54°C and 2 minute and 30 seconds at 72°C , followed by 7 minutes at 72°C (for P°C 010). The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR8/GW/TOPO vector (Cat. Nr. K2500-20, Invitrogen) and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-P°C  and are referred to as the partial sequences, 'fhe corresponding partial amino acid sequence are represented by the respective SEQ ID NO:s as given in Table 3-P°C .<br>
B.        dsRNA production of the Plutella °C ylostella genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit '\1 Riboma°C "^ E°C press RNAi System (Cat. Nr. PI700, Promega). First two separate single 5' 17 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the 17 promoter.<br><br>
For each of the target genes, the sense T7 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Tabic 8-P°C . The conditions in the PCR reactions were as follows: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95°C, 30 seconds at 60°C (-0.5°C/cycle) and I minute at 72°C, followed by 15 cycles of 30 seconds at 95T, 30 seconds at 50°C and 1 minute at 72T, followed by 10 minutes at 72°C. Ihe anti-sense T7 template was generated using specific forward and specific T7 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Tabic 8-P°C . The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaC104 precipitation. The generated 17 forward and reverse templates were mi°C ed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-P°C .<br>
C.	Laboratory trials to test dsRNA targets, using artificial diet for activity against<br>
Plutella °C ylostella larvae<br>
Diamond-back moths, Plutella °C ylostella, were maintained at Insect Investigations Ltd. (origin: Newcastle University, Newcastle-upon-Tyne, UK). The insects were reared on cabbage leaves. First instar, mi°C ed se°C  larvae (appro°C imately 1 day old) were selected for use in the trial. Insects were maintained in Eppendorf tubes (1.5 ml capacity). Commercially available Diamond-back moth diet (Bio-Serv, NJ, USA), prepared following the manafacturer's instructions, was placed in the lid of each tube (0.25 ml capacity, 8 mm diameter). While still liquid, the diet was smoother over to remove e°C cess and produce an even surface.<br>
Once the diet has set the test formulations are applied to the diet's surface, at the rate of 25 µl undiluted formulation (1µg/|µl dsRNA of targets) per replicate. The test formulations are allowed to dry and one first instar moth larva is placed in each tube. The larva is placed on the surface of the diet in the lid and the tube carefully closed. The tubes are stored upside down, on their lids such that each larva remains on the surface of the diet. Twice weekly the larvae are transferred to new Eppendorf tubes with fresh diet. The insects are provided with treated diet for the first two weeks of the trial and thereafter with untreated diet.<br><br>
Assessments are made twice weekly for a total of 38 days at which point all larvae are dead. At each assessment the insects are assessed as live or dead and e°C amined for abnormalities. Forty single larva replicates are performed for each of the treatments. During the trial the test conditions are 23 to 26 "C and 50 to 65 % relative humidity, with a 16:8 hour lightidark photoperiod.<br>
D.        Cloning of a DBM gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an e°C ample of cloning a DNA fragment corresponding to a DBM gene target in a vector for the e°C pression of double-stranded RNA in a bacterial host, although any vector comprising a T7 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8-P°C . The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at 98°C, followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72°C, followed by 10 minutes at 72°C. The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into .Vz/I-linearized pGNA49A vector (reference to WO00188121A1), and sequenced. The sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8-P°C . The recombinant vector harbouring this sequence is named pGBNJOO°C °C . E.        E°C pression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br>
The procedures described below are followed in order to e°C press suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselll-deficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). rransformation ofAB309-105 and BL21(DE3)<br>
fhree hundred ng of the plasmid are added to and gently mi°C ed in a 50 µl aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty µL of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 °C for 1 hour. One hundred µl of the bacterial cell suspension is transferred to a 500 ml conical fiask containing 150 ml of liquid Luria-Bertani<br><br>
(LB) broth supplemented with 100µg/ml carbenicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) at 37 ""C overnight (16 to 18 hours).<br>
Chemical induction of double-stranded liNA e°C pression inAB309-l()5 and BL21(DE3)<br>
E°C pression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled e°C pression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the T7 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are flanked by oppositely oriented T7 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of I by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 o"C for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 |ig/ml cholesterol) supplemented with 100 \iglm\ carbenicillin and 1 mM IPTG. The bacteria are induced for 2 to 4 hours at room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria e°C pressing double-stranded RNA is not a prerequisite for inducing to°C icity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80oC  for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes are stored at -20 T until further use.<br>
F.	Laboratory trials to test Escherichia coli e°C pressing dsliNA targets against<br>
Plutella °C ylostella<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria e°C pressing dsRNA prior to feeding the plants to DBM. The are grown from in a plant growth room chamber. Ihe plants are caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered<br><br>
with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. DBM are placed on each treated plant in the cage. Plants are treated with a suspension of £. coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 109 bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. Ihe number of survivors are counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of £". coli bacterial strain AB309-105 e°C pressing target dsRNA from pGBNJ003 leed to a dramatic increase in insect mortality when compared to pGN29 control. These e°C periments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial e°C pression systems is to°C ic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these e°C periments that an e°C emplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria e°C pressing double-stranded RNA corresponding to an insect gene target.<br>
E°C ample 12: Aclieta ilomesticus (house cricket)<br>
A,        Cloning °C 7fe/a domesticus partial sequences<br>
High quality, intact RNA was isolated from all the different insect stages of Acheta domesticus (house cricket; source: Dr. Lara Senior, Insect Investigations Ltd., Capital Business Park, Wentloog, Cardiff, CF3 2P°C  Wales, UK) using fRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation was removed by DNase treatment following the manafacturer's instructions (Cat. Nr. 1700, Promega). cDNA was generated using a commercially available kit (Superscript ^^ III Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of the ADOOl, AD002, AD009, ADO 15 and ADO 16 genes, a series of PCR reactions with degenerate primers were performed using Amplitaq Gold (Cat. Nr. N8080240, Applied Biosystems) following the manafacturer's instructions.<br><br>
The sequences of the degenerate primers used for amplification of each of the genes are given in Table 2-AD. These primers were used in respective PCR reactions with the following conditions: 10 minutes at 95'°C , followed by 40 cycles of 30 seconds at 95°C, 1 minute at 50°C and 1 minute and 30 seconds at 72°C, followed by 7 minutes at 72°C. The resulting PCR fragments were analyzed on agarose gel, purified (QIAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), cloned into the pCR8/GW/topo vector (Cat. Nr. K2500 20, Invitrogen) and sequenced. The sequences of the resulting PCR products are represented by the respective SEQ ID NO:s as given in Table 2-AD and are referred to as the partial sequences. The corresponding partial amino acid sequence are represented by the respective SEQ ID NO:s as given in Table 3-AD.<br>
B.        dsRNA production of the Acheta domesticus genes<br>
dsRNA was synthesized in milligram amounts using the commercially available kit T7 Ribomax TM  E°C press RNAi System (Cat. Nr. PI 700, Promega). First two separate single 5' T7 RNA polymerase promoter templates were generated in two separate PCR reactions, each reaction containing the target sequence in a different orientation relative to the T7 promoter.<br>
For each of the target genes, the sense T7 template was generated using specific T7 forward and specific reverse primers. The sequences of the respective primers for amplifying the sense template for each of the target genes are given in Table 8-AD. The conditions in the PCR reactions were as follows: 1 minute at 95°C, followed by 20 cycles of 30 seconds at 95°C, 30 seconds at 60°C (^0.5°C/cycle) and 1 minute at 72°C, followed by 15 cycles of 30 seconds at 95°C, 30 seconds at 50°C and 1 minute at 72°C, followed by 10 minutes at 72°C. The anti-sense T7 template was generated using specific forward and specific T7 reverse primers in a PCR reaction with the same conditions as described above. The sequences of the respective primers for amplifying the anti-sense template for each of the target genes are given in Tabic 8-AD, The resulting PCR products were analyzed on agarose gel and purified by PCR purification kit (Qiaquick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaC104 precipitation. 1 he generated T7 forward and reverse templates were mi°C ed to be transcribed and the resulting RNA strands were annealed, DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. The sense strand of the resulting dsRNA for each of the target genes is given in Table 8-AD. C.        Laboratory trials to test dsRNA targets, using artificial diet for activity against<br>
Acheta domesticus larvae<br><br>
House crickets, Acheta domesticiis, were maintained at Insect Investigations Ltd. (origin: Blades Biological Ltd., Kent, UK). The insects were reared on bran pellets and cabbage leaves. Mi°C ed se°C  nymphs of equal size and no more than 5 days old were selected for use in the trial. Double-stranded RNA is mi°C ed with a wheat-based pelleted rodent diet (rat and mouse standard diet, B &amp; K Universal Ltd., Grimston, Aldbrough, Hull, UK). The diet, BKOOIP, contains the following ingredients in descending order by weight: wheat, soya, wheatfeed, barley, pellet binder, rodent 5 vit min, fat blend, dicalcium phosphate, mould carb. The pelleted rodent diet is finely ground and heat-treated in a microwave oven prior to mi°C ing, in order to inactivate any enzyme components. All rodent diet is taken from the same batch in order to ensure consistency. The ground diet and dsRNA are mi°C ed thoroughly and formed into small pellets of equal weight, which are allowed to dry overnight at room temperature.<br>
Double-stranded RNA samples from targets and gfp control at concentrations 10 |ig/jil were applied in the ratio 1 g ground diet plus 1 ml dsRNA solution, thereby resulting in an application rate of 10 mg dsRNA per g pellet. Pellets are replaced weekly. The insects are provided with treated pellets for the first three weeks of the trial. Thereafter untreated pellets are provided. Insects are maintained within lidded plastic containers (9 cm diameter, 4.5 cm deep), ten per container. Each arena contains one treated bait pellet and one water source (damp cotton wool ball), each placed in a separate small weigh boat. The water is replenished ad lib throughout the e°C periment.<br>
Assessments are made at twice weekly intervals, with no more than four days between assessments, until all the control insects had either died or moulted to the adult stage (84 days). At each assessment the insects are assessed as live or dead, and e°C amined for abnormalities. From day 46 onwards, once moulting to adult has commenced, all insects (live and dead) are assessed as nymph or adult. Surviving insects are weighed on day 55 of the trial. Four replicates are performed for each of the treatments. During the trial the test conditions are 25 to 33 °C  and 20 to 25 % relative humidity, with a 12:12 hour light:dark photoperiod.<br>
D.        Cloning of a HC gene fragment in a vector suitable for bacterial production of<br>
insect-active double-stranded RNA<br>
What follows is an e°C ample of cloning a DMA fragment corresponding to a HC gene target in a vector for the e°C pression of double-stranded RNA in a bacterial host, although any<br><br>
vector comprising a 17 promoter or any other promoter for efficient transcription in bacteria, may be used (reference to WO0001846).<br>
The sequences of the specific primers used for the amplification of target genes are provided in Table 8. The template used is the pCR8/GW/topo vector containing any of target sequences. The primers are used in a PCR reaction with the following conditions: 5 minutes at \98°C , followed by 30 cycles of 10 seconds at 98°C, 30 seconds at 55°C and 2 minutes at 72'°C , followed by 10 minutes at 72opC The resulting PCR fragment is analyzed on agarose gel, purified (QIAquick Gel E°C traction kit, Cat. Nr. 28706, Qiagen), blunt-end cloned into Srf 1-linearized pGNA49A vector (reference to WO00I88121A1), and sequenced, fhe sequence of the resulting PCR product corresponds to the respective sequence as given in Table 8-AD. The recombinant vector harbouring this sequence is named pGBNJOO°C °C . E,        E°C pression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309^105, and, (2) BL21(DE3)<br>
Ihe procedures described below are followed in order to e°C press suitable levels of insect-active double-stranded RNA of insect target in bacteria. An RNaselU-dcficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL21(DE3). Transformation of AB309-105 and BL21(DE3)<br>
Three hundred ng of the plasmid are added to and gently mi°C ed in a 50 µl aliquot of ice-chilled chemically competent E. coli strain AB309-105 or BL21(DE3), The cells arc incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 °C for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fif^y µl of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 '°C  for 1 hour. One hundred µL of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Euria-Bertani (LB) broth supplemented with 100 µg/ml carbenicillin antibiotic. I'he culture is incubated on an Innova 4430 shaker (250 rpm) at 37 °C overnight (16 to 18 hours).<br>
Chemical induction of double-stranded RNA e°C pression inAB309-105 and BL21(DE3)<br>
E°C pression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled e°C pression are present. In the presence of the chemical inducer isopropylthiogalacloside, or IPTG, the T7 polymerase will drive the transcription of the target sequence in both antiscnse and sense directions since these are flanked by oppositely oriented T7 promoters.<br><br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth, fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 T for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 µg/ \µl  cholesterol) supplemented with 100 µg/ml carbenicillin and 1 mM IPTG. The bacteria are induced for 2 lo 4 hours at room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order to minimise the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria e°C pressing double-stranded RNA is not a prerequisite for inducing to°C icity towards the insects due to RNA interference. 1 he induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 '°C  for 20 minutes in a water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes are stored at -20 °C until further use.<br>
F,	Laboratory trials to test Escherichia cvli e°C pressing dsltNA targets against<br>
Acheta Uomesticus<br>
Plant-based bioassays<br>
Whole plants are sprayed with suspensions of chemically induced bacteria e°C pressing dsRNA prior to feeding the plants to MC. The are grown from in a plant growth room chamber. The plants arc caged by placing a 500 ml plastic bottle upside down over the plant with the neck of the bottle firmly placed in the soil in a pot and the base cut open and covered with a fine nylon mesh to permit aeration, reduce condensation inside and prevent insect escape. HC are placed on each treated plant in the cage. Plants are treated with a suspension of A; coli AB309-105 harbouring the pGBNJOOl plasmids or pGN29 plasmid. Different quantities of bacteria are applied to the plants: for instance 66, 22, and 7 units, where one unit is defined as 109  bacterial cells in 1 ml of a bacterial suspension at optical density value of 1 at 600 nm wavelength. In each case, a total volume of between 1 and 10 ml s sprayed on the plant with the aid of a vaporizer. One plant is used per treatment in this trial. The number of survivors arc counted and the weight of each survivor recorded.<br>
Spraying plants with a suspension of E. coli bacterial strain AB309-105 e°C pressing target dsRNA from pGBNJ003 leed to a dramatic increase in insect mortality when compared<br><br>
to pGN29 control. These e°C periments show that double-stranded RNA corresponding to an insect gene target sequence produced in either wild-type or RNaselll-deficient bacterial e°C pression systems is to°C ic towards the insect in terms of substantial increases in insect mortality and growth/development delay for larval survivors. It is also clear from these e°C periments that an e°C emplification is provided for the effective protection of plants/crops from insect damage by the use of a spray of a formulation consisting of bacteria e°C pressing double-stranded RNA corresponding to an insect gene target.<br>
E°C ample 13: Pyricularia urisea (rice blast)<br>
A.        Cloning P, grisea partial sequences<br>
High quality, intact RNA is isolated from different growth stages of P. grisea using TRIzol Reagent (Cat. Nr. 15596-026/15596-018, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions. Genomic DNA present in the RNA preparation is removed by DNase treatment following the manafacturer's instructions (Cat. Nr. 1700, Promega). cDNA is generated using a commercially available kit (Superscript TM  III Reverse Transcriptase, Cat. Nr. 18080044, Invitrogen, Rockville, Maryland, USA) following the manufacturer's instructions.<br>
To isolate cDNA sequences comprising a portion of a target gene, PCR is performed with degenerate primers using Amplitaq Gold (Cat, Nr. N8080240, Applied Biosystems) following the manafacturer's instructions. The resultant PCR products are fractionated and sequenced.<br>
B.        dsRNA production o°C  P, gr/se^ genes<br>
dsRNA is synthesized in milligram amounts using a commercially available kit, such as T7 Riboma°C '^ E°C press RNAi System (Cat. Nr. PI700, Promega), following the manufacturer's instructions. The resulting PCR products are analyzed on an agarose gel and purified by a PCR purification kit (e.g. Qi^quick PCR Purification Kit, Cat. Nr. 28106, Qiagen) and NaC104 precipitation. The producr 17 forward and reverse templates are mi°C ed and the resuhing RNA strands are annealed, then DNase and RNase treated, and purified by sodium acetate, following the manufacturer's instructions. C,        E°C pression and production of a double-stranded RNA target in two strains of<br>
Escherichia coli: (1) AB309-105, and, (2) BL21(DE3)<br><br>
The procedures described below are followed in order to e°C press suitable levels of fungal double-stranded FWA of fungal target in bacteria. An RNaseIll-deficient strain, AB309-105, is used in comparison to wild-type RNaselll-containing bacteria, BL2I(DE3). Transformation of AB309-105 and BL21(DE3)<br>
Three hundred ng of the plasmid arc added to and gently mi°C ed in a 50 µl aliquot of ice-chilled chemically competent E, coli strain AB309-105 or BL21(DE3). The cells are incubated on ice for 20 minutes before subjecting them to a heat shock treatment of 37 oC for 5 minutes, after which the cells are placed back on ice for a further 5 minutes. Four hundred and fifty µL of room temperature SOC medium is added to the cells and the suspension incubated on a shaker (250 rpm) at 37 °C for 1 hour. One hundred µL of the bacterial cell suspension is transferred to a 500 ml conical flask containing 150 ml of liquid Luria-Bertani (LB) broth supplemented with 100 µg/ml carbenicillin antibiotic. The culture is incubated on an Innova 4430 shaker (250 rpm) at 37 °C overnight (16 to 18 hours).<br>
Chemical induction of double-stranded UNA e°C pression inAB309-105 and BL21(DE3)<br>
E°C pression of double-stranded RNA from the recombinant vector, pGBNJ003, in the bacterial strain AB309-105 or BL21(DE3) is made possible since all the genetic components for controlled e°C pression are present. In the presence of the chemical inducer isopropylthiogalactoside, or IPTG, the 17 polymerase will drive the transcription of the target sequence in both antisense and sense directions since these are fianked by oppositely oriented T7 promoters.<br>
The optical density at 600 nm of the overnight bacterial culture is measured using an appropriate spectrophotometer and adjusted to a value of 1 by the addition of fresh LB broth. Fifty ml of this culture is transferred to a 50 ml Falcon tube and the culture then centrifuged at 3000 g at 15 °C for 10 minutes. The supernatant is removed and the bacterial pellet resuspended in 50 ml of fresh S complete medium (SNC medium plus 5 |ig/ml cholesterol) supplemented with 100 µglml carbenicillin and 1 mM IPTG. The bacteria are induced for 2 to 4 hours at room temperature.<br>
Heat treatment of bacteria<br>
Bacteria are killed by heat treatment in order lo minimise the risk of contamination of the artificial diet in the test plates. However, heat treatment of bacteria e°C pressing double-slranded RNA is not a prerequisite for inducing to°C icity towards the insects due to RNA interference. The induced bacterial culture is centrifuged at 3000 g at room temperature for 10 minutes, the supernatant discarded and the pellet subjected to 80 °C for 20 minutes in a<br><br>
water bath. After heat treatment, the bacterial pellet is resuspended in 1.5 ml MilliQ water and the suspension transferred to a microfuge tube. Several tubes are prepared and used in the bioassays for each refreshment. The tubes are stored at -20 °C  until further use.<br><br><br><br><br><br><br><br><br><br><br><br>
What is claimed is:<br>
1.	An isolated polynucleotide sequence comprising a nucleic acid sequence set forth in SEQ ID NOs: I, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 - 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 - 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 - 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 - 2460, 2461, 2466, 2471, 2476 and 2481.<br>
2.	A polynucleotide sequence having at least 70% sequence identity to a nucleic acid sequence set forth in SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49 - 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 - 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 - 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 -2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 - 2460, 2461, 2466,2471,2476 and 2481.<br>
3.	A double stranded ribonucleotide sequence produced from the expression of a polynucleotide sequence of any of claims 1 or 2, wherein contact of said ribonucleotide sequence by a pest inhibits the growth of said pest.<br><br>
4.	The  ribonucleolidc sequence of claim 3, wherein contact of said sequence inhibits expression of a nucleotide sequence substantially complementary to said sequence,<br>
5.	A cell transformed with the polynucleotide of any of claims 1-3.<br>
6.	The cell of claim 5, wherein said cell is a bacterial, yeast, or algal cell<br>
7.	A food product comprising the cell of claim 6.<br>
8.	The food product of claim 7, wherein said food product is selected from the group consisting of stored grains, pet food, and powdered chocolate.<br>
9.	A composition comprising the polynucleotide of any of claims 1-3.<br>
10.	The composition of claim 9, wherein said composition is selected from the group consisting of a spray, powder, pellet, gel, capsule, food product, garment bag, and book.<br>
11.	A method for controlling pest infestation, comprising exposing a pest to a composition comprising a polynucleotide sequence that inhibits a pest biological activity.<br>
12.	The method of claim 11, wherein said polynucleotide sequence is set forth in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 49- 158, 159, 160, 163, 168, 173, 178, 183, 188, 193, 198, 203, 208, 215, 220, 225, 230, 247, 249, 251, 253, 255, 257, 259, 275-472, 473, 478, 483, 488, 493, 498, 503, 513, 515, 517, 519, 521, 533 - 575, 576, 581, 586, 591, 596, 601, 603, 605, 607, 609, 621 - 767, 768, 773, 778, 783, 788, 793, 795, 797, 799, 801, 813 - 862, 863, 868, 873, 878, 883, 888, 890, 892, 894, 896, 908- 1040, 1041, 1046, 1051, 1056, 1061, 1071, 1073, 1075, 1077, 1079, 1081, 1083, 1085, 1087, 1089, 1091, 1093, 1095, 1097, 1099, 1101, 1103, 1105, 1107, 1109, 1111, 1113, 1161 - 1571, 1572, 1577, 1582, 1587, 1592, 1597, 1602, 1607, 1612, 1617, 1622, 1627, 1632, 1637, 1642, 1647, 1652, 1657, 1662, 1667, 1672, 1677, 1682, 1684, 1686, 1688, 1690, 1692, 1694, 1696, 1698, 1700, 1702, 1704, 1730 - 2039, 2040, 2045, 2050, 2055, 2060, 2065, 2070, 2075, 2080, 2085, 2090, 2095, 2100, 2102, 2104, 2106, 2108, 2120 - 2338, 2339, 2344, 2349, 2354, 2359, 2364, 2366, 2368, 2370, 2372, 2384 ™ 2460, 2461, 2466, 2471, 2476 and 2481.<br>
13.	A pesticide comprising a polynucleotide of any of claims 1-3.<br>
14.	A method for protecting an object from pest infestation, comprising treating said surface with a composition comprising the polynucleotide of any of claims 1-3.<br><br>
15.	The method of claim 14, wherein said object is selected from the group consisting of wood, tree, book binding, cloth, and a food storage container.<br>
16.	Use of an isolated nucleotide sequence according to any of claims 1-3, a double stranded ribonucleotide sequence according to any of claims 3-4, a cell according to any of claims 5-6, a composition according to any of claims 9-10, or a pesticide according to claim 13, for preventing or treating an insect infestation.<br>
17.	Use of an isolated nucleotide sequence according to any of claims N3, a double<br>
stranded ribonucleotide sequence according to any of claims 3-4, a cell according to any of<br>
claims 5-6, a composition according to any of claims 9-10, or a pesticide according to claim<br>
13, for preventing or treating a nematode infestation.<br>
18.	Use of an isolated nucleotide sequence according to any of claims 1-3, a double<br>
stranded ribonucleotide sequence according to any of claims 3-4, a cell according to any of<br>
claims 5-6, a composition according to any of claims 9-10, or a pesticide according to claim<br>
13, for preventing or treating a fungal infection.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgIENPUlJFU1BPTkRFTkNFICAgT1RIRVJTICAgMjAtMDUtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008    CORRESPONDENCE   OTHERS   20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgIEZPUk0tMSAgIDIwLTA1LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008    FORM-1   20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgIEZPUk0tMTMgICAyMC0wNS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008    FORM-13   20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgIEZPUk0tNSAgICAyMC0wNS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008    FORM-5    20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgIE9USEVSICAgRE9DVU1FTlQgICAgMjAtMDUtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008    OTHER   DOCUMENT    20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMTctMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   AMENDED  PAGES OF SPECIFICATION  17-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCBDTEFJTVMgIDE3LTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   AMENDED CLAIMS  17-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgICBPVEhFUlMgICAyMC0wNS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE   OTHERS   20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAwNy0wOC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE OTHERS  07-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAxNy0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE OTHERS  17-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE OTHERS  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyOC0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE OTHERS  28-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAzMC0wOC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE OTHERS  30-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLiAgMzAtMDgtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   CORRESPONDENCE OTHERS.  30-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAyMS0wOS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED  21-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgRk9STS0xICAyMC0wNS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   FORM-1  20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgRk9STS0xMyAgIDIwLTA1LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   FORM-13   20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgRk9STS0zICAyMS0wOS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   FORM-3  21-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgRk9STS0zICAzMC0wOC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   FORM-3  30-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgRk9STS01ICAgMjAtMDUtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   FORM-5   20-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   OTHER PATENT DOCUMENT  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAzMC0wOC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   OTHER PATENT DOCUMENT  30-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgT1RIRVJTICAzMC0wOC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   OTHERS  30-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgUE9XRVIgT0YgQVRUT1JORVkgIDI4LTExLTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   POWER OF ATTORNEY  28-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyAgMTctMTEtMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyAxICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING 1  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyAyICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING 2  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyAzICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING 3  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyA0ICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING 4  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyA1ICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING 5  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICAgU0VRVUVOQ0UgTElTVElORyA2ICAxNy0xMS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008   SEQUENCE LISTING 6  17-11-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBBTUVOREVEIENMQUlNUyAxOS0wMy0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  AMENDED CLAIMS 19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gMTktMDMtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  AMENDED PAGES OF SPECIFICATION 19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSAuT1RIRVJTIDE5LTAzLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  CORRESPONDENCE .OTHERS 19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMTktMDMtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  CORRESPONDENCE OTHERS 19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBGT1JNLTEgIDE5LTAzLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  FORM-1  19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBGT1JNLTEzICAxOS0wMy0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  FORM-13  19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBGT1JNLTUgIDE5LTAzLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  FORM-5  19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDA4ICBGT1JNLTUgMTktMDMtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2008  FORM-5 19-03-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1768-chenp-2008 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1jaGVucC0yMDA4LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1768-chenp-2008-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2OC1DSEVOUC0yMDEzIEZPUk0tMTMgMjAtMDUtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1768-CHENP-2013 FORM-13 20-05-2013.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258469-polymerisation-process.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258471-a-process-for-preparing-a-crystalline-1-2-ethanedisulfonic-acid-salt.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258470</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1768/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>03/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Apr-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DEVGEN NV</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 30 B-9052 ZWIJNAARDE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VAN BLEU, ELS,</td>
											<td>WAKKEBROEKKSTRAAT 7 B-9290 BERLARE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN BLEU, ELS,</td>
											<td>WAHREBAOEKSTRAAT 7 B-9290 BAELANE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PECQUEUR, FREDERIC</td>
											<td>ALLEE DES PRES JARDINS 2 59320 HALLENNS-LEZ-HAUBOURDIN</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KUBLER, LAURENT</td>
											<td>78 ROUTE DE GENEVE F-01700 BEYNOST</td>
										</tr>
										<tr>
											<td>5</td>
											<td>DEGRAVE, LIES</td>
											<td>MAGDALENA HUYSSTRAAT 21 8700 TIELT</td>
										</tr>
										<tr>
											<td>6</td>
											<td>FELDMANN, PASCALE</td>
											<td>GOUDBROKAATSTRAAT 2 B-9030 GENT</td>
										</tr>
										<tr>
											<td>7</td>
											<td>KUBLER, LAURENT</td>
											<td>9 RUE CENTRALE F-01700 BEYNOST</td>
										</tr>
										<tr>
											<td>8</td>
											<td>PLAETINCK, GEERT</td>
											<td>PONTSTRAAT 16 B-9820 BOTTELARE</td>
										</tr>
										<tr>
											<td>9</td>
											<td>NOOREN, IRENE</td>
											<td>ELI HEIMANSHOF 64 NL 2341 PH OEGSTGEEST</td>
										</tr>
										<tr>
											<td>10</td>
											<td>DAMME, NICOLE</td>
											<td>COLIJINSTRAAT 6 B-9770 KRUISHOUTEM</td>
										</tr>
										<tr>
											<td>11</td>
											<td>REMORY, ISABEL</td>
											<td>RESSEGEMSTRAAT 170-APP 2 9551 RESSEGEM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82, A01H 5/00,C12N 15/11</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2006/004008</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-09-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/837,910</td>
									<td>2006-08-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/758,191</td>
									<td>2006-01-12</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/771,160</td>
									<td>2006-02-07</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>PCT/IB2006/003446</td>
									<td>2006-09-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/718,034</td>
									<td>2005-09-16</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258470-methods-for-controlling-pests-using-rnai by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:28:48 GMT -->
</html>
