<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258468-improved-human-interferon-molecules by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:29:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258468:IMPROVED HUMAN INTERFERON MOLECULES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">IMPROVED HUMAN INTERFERON MOLECULES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Modified human interferon polypeptides and uses thereof are provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Improved Human Interferon Molecules and Their Uses<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
[01]	This application claims priority to U.S. provisional patent application 60/687,173<br>
filed June 3, 2005 entitled "Improved Human Interferon Molecules and Their Uses" and U.S. provisional patent application 60/753,375 filed December 21, 2005 entitled "Improved Human Interferon Molecules and Their Uses", the specifications of which are incorporated herein in their entirety.<br>
FIELD OF THE INVENTION<br>
[02]	This invention relates to interferon polypeptides modified with at least one non-<br>
naturally encoded amino acid.<br>
BACKGROUND OF THE INVENTION<br>
[03]	The growth hormone (GH) supergene family (Bazan, F. Immunology Today 11:<br>
350-354 (1990); Mott, H. R. and Campbell, L D. Current Opinion in Structural Biology 5:114-121 (1995); Silvennoinen, O. and Ihle, J. N. (1996) SIGNALING BY THE HEMATOPOIETIC CYTOKINE RECEPTORS) represents a set of proteins with similar structural characteristics. Each member of this family of proteins comprises a four helical bundle. While there are still more members of the family yet to be identified, some members of the family include the following: growth hormone, prolactin, placental lactogen, erythropoietin (EPO), thrombopoietin (TPO), interleukin-2 (IL-2), IL-3, IL-4, IL-5, IL-6, IL-7, IL-9, IL-10, IL-Il, IL-12 (p35 subunit), IL-13, IL-15, oncostatin M, ciliary neurotrophic factor, leukemia inhibitory factor, alpha interferon, beta interferon, gamma interferon, omega interferon, tau interferon, epsilon interferon, granulocyte-colony stimulating factor (G-CSF), granulocyte-macrophage colony stimulating factor (GM-CSF), macrophage colony stimulating factor (M-CSF) and cardiotrophin-l (CT-1) ("the GH supergene family"). Members of the GH supergene family have similar secondary and tertiary structures, despite the fact that they generally have limited amino acid or DNA sequence identity. The shared structural features allow new members of the gene family to be readily identified. The general structure of IFNα-2 is shown in Figure 1.<br><br>
[04]	Interterons are relatively small, single-chain giycoprotems released by cells<br>
invaded by viruses or exposed to certain other substances. Interferons are presently grouped into three major classes, designated: 1) leukocyte mterferon (interferon-αlpha, α-interferon, IFN-α), 2) fibroblast interferon (interferon-beta, β-interferon, IFN-β), and 3) immune interferon (interferon-gamma, γ-interferon, IFN-γ). In response to viral infection, lymphocytes synthesize primarily α-interferon (with omega interferon, IFN-ω), while infection of fibroblasts usually induces production of β-interferon. IFNα and IFNβ share about 20-30 percent amino acid sequence homology. The gene for human DFN-β lacks introns, and encodes a protein possessing 29% amino acid sequence identity with human IFN-α, suggesting that IFN-α and IFN-β genes have evolved from a common ancestor (Taniguchi et al.. Nature 285 547-549 (1980)). By contrast, EFN-γ is synthesized by lymphocytes in response to mitogens. IFNα, IFN β and IFNω are laiov/n to induce MHC Class I antigen expression and are referred to as type I interferons, while IFNγ induces MHC Class II antigen expression, and is referred to as type II interferon. Pestka et al. in Armu. Rev. Immunol. (2004) 22:929-79, which is incorporated by reference herein in its entirety, describes class 2 α-helical cytokines including interferons (IFN-α, -β, -ε, -K, -ω, -δ, -τ , and -γ) as well as interferon-like molecules such as limitin, IL-28A, IL-28B, and IL-29 as well as the ligands, receptors, and signal transduction pathways employed by these molecules. The mterferons have different species and many allelic variants. In additional, interferons with novel activities and mutant sequences have been isolated from cells from patients with various diseases.<br>
[05]	A large number of distinct genes encoding different species of IFNα have been<br>
identified. Alpha interferons fall into two major classes, I and II, each containing a plurality of discrete proteins (Baron et al., Critical Reviews in Biotechnology 10, 179-190 (1990); Nagata et al., Nature 287, 401-408 (1980); Nagata et al., Nature 284, 316-320 (1980); Streuli et al., Science 209, 1343-1347 (1980); Goeddel et al., Nature 290, 20-26 (1981); Lawn et al.. Science 212, 1159-1162 (1981); Ullrich et al., J. Mol. Biol 156, 467-486 (1982); Weissmann et al., Phil. Trans. R. Soc. Lond. B299, 7-28 (1982); Lund et al., Proc. Natl, Acad. Sci. 81, 2435-2439 (1984); Capon et al, Mol. Cell. Biol. 5, 768 (1985)). The various IFN-α species include IFN-αA (IFN-α2), IFN-αB, IFN-αC, IFN-αCl, IFN-αD (EFN-αl), IFN-oE, IFN-αF, IFN-αG, IFN-αH, IFN-αl, IFN-oJl, IFN"CxJ2, IFN-αK, IFN-αL, IFN-α4B, IFN-α5, IFN-α6, IFN-α74, IFN-α76 IFN"a4a), IFN-α88, and alleles thereof Trotta et al. in "Approval Standards for Alfa Interferon Subtypes" Drug Information Journal 34:1231-1246 (2000), which is incorporated by reference herein in its entirety, describe the members of the human IFN a gene family and proteins and the biological activities of this family including the immunomodulatory, antiproliferative, anti-<br><br><br>
assurance, biological activity, and clinical safety and efficacy issues that relate to recombinant versions of proteins in this family. Release tests and physicochemical characterization tests are also discussed. IFN α21, IFN α4, IFN α10. and IFN α3 are other interferon proteins that have been previously described.<br>
[06]	Interferons were originally derived from naturally occurring sources, such as<br>
buffy coat leulcocytes and fibroblast cells, optionally using inducing agents to increase interferon<br>
production. Interferons have also been produced by recombinant DNA technology.<br>
[07]	The cloning and expression of recombinant IFNαA (IFNαA, also known as<br>
 IFNα2) was described by Goeddel et al., Nature 287, 411 (1980). The amino acid sequences of  IFNαA, B, C, D, F, G, H, K and L, along with the encoding nucleotide sequences, are described by Pestka in Archiv. Biochem. Biophys. 221, 1 (1983). The cloning and expression of mature IFNβ is described by Goeddel et al,. Nucleic Acids Res. 8, 4057 (1980). The cloning and expression of mature IFNγ are described by Gray et al., Nature 295, 503 (1982). IFNω has been described by Capon et al., Mol. Cell. Biol. 5, 768 (1985). IFNT has been identified and disclosed by Whaley et al., J. Biol. Chem. 269,10864-8 (1994).<br>
[08]	Interferons   have   a   variety   of   biological   activities,   including   anti-viral,<br>
immunoregulatory and anti-proliferative properties, and have been utilized as therapeutic agents for treatment of diseases such as cancer, and various viral diseases. As a CLASS, interferon-a's have been shown to inhibit various types of cellular proliferation, and are especially useful for the treatment of a variety of cellular proliferation disorders frequently associated with cancer, particularly hematologic malignancies such as leukemias. These proteins have shown antiproliferative activity against multiple myeloma, chronic lymphocytic leukemia, low-grade lymphoma, Kaposi's sarcoma, chronic myelogenous leukemia, renal-cell carcinoma, urinary bladder tumors and ovarian cancers (Bonnem, E. M. et al. (1984) J. Biol. Response Modifiers 3:580; Oldham, R. K. (1985) Hospital Practice 20:71).<br>
[09]	In addition, interferon-α may have important neuroregulatory functions in the<br>
CNS.  Structural and functional similarities have been shown between  IFNα and endorphins. It<br><br>
has been reported that the IFNα molecule contams distinct domains that mediate immune and opioid-like effects and that the µ opioid receptor may be involved in the analgesic effect of IFNα. Analgesic domains of the tertiary structure of interferon-α have been described which locate around the 122nd Tyr residue of the molecule and includes the Phe residues 36, 38, and 123 (Wang et al. J. Neuroimmunol. (2000) 108:64-67 and Wang et al. NeuroReport (2001) 12(4):857-859, which are incorporated by reference herein). Specifically, Wang et al. found that an interferon-α mutant at residue 36 (F36S) resulted in a complete loss of analgesic activity and a reduction of anti-viral activity. Another IFN-α mutant CF38S) resulted in a complete loss of analgesic activity and almost a complete loss of anti-viral activity. Other mutants of IFNα that have been studied include F38L and Y129S. Wang et al. describe these two mutants in studies investigating fever induced by human IFNα, and found that this side effect of IFNα therapy is mediated by IFNα's interaction with opioid receptor and a subsequent induction of prostaglandin E2 (J. of Neuroimmunology (2004) 156:107-112). Modulating the interaction between IFN and opioid receptors may be critical in the development of novel IFN therapeutics to prevent side effects involving this family of receptors. Prostaglandins modulate CNS functions including but not limited to, the generation of fever, the sleep/wake cycle, and the perception of pain. They are produced by the enzymatic activity of cyclooxygenases COX-1 and COX-2.<br>
[10]	The administration of IFN-α may also result in a number of neuiopsychiatric side<br>
effects including depression (Wichers and Maes, Rev. Psychiat Neurosci. (2004) 29(1):11-17).<br>
Wichers and Maes indicate serotonin (5-HT) brain neurotransmission and the induction of the<br>
enzyme IDO (indolamine 2,3-dioxygenase) are involved. Other hypotheses involve nitric oxide<br>
and soluble ICAM-1 induction by IFN. Modulating the mechanisms by which IFN causes such<br>
side effects may be critical in the development of novel IFN therapeutics.<br>
[11]	Specific examples of commercially available IFN products include IFNγ-lb<br>
(Actimmune®), IFNβ-l® (Avonex®, and Rebif®), IFN(3-lb (Betaseron®), IFN alfacon-1 (Infergen®), IFNα-2 (Intron A®), IFNα-2a (Roferon-α®), Peginterferon alfα-2a (PEGASYS®, and Peginterferon alfα-2b (PEG-Intron®). Some of the problems associated with the production of PEGylated versions of IFN proteins are described in Wang et al. (2002) Adv. Drug Deliv. Rev. 54:547-570; and Pedder, S.C. Semin Liver Dis. 2003;23 Suppl 1:19-22. Wang et al. characterized positional isomers of PEG-Intron®, and Pedder at al. compared PEGAS YS® with PEG-Intron® describing the lability of the PEGylation chemistries used and effects upon formulation.   PEGASYS® is comprised of nine identifiable isoforms, which specific isoforms<br><br>
differing in anti-viral activity (Foser et al., Pharmacogenomics J 2003; 3:312), Despite the number of IFN products currently available on the market, there is still an unmet need for interferon therapeutics. In particular, interferon therapeutics that modulate one or more side effects found with current IFN therapeutics are of interest.<br>
[12]	Covaient   attachment   of  the   hydrophilic   polymer   poly(ethyJene   glycol),<br>
abbreviated PEG, is a method of increasing water solubility, bioavailability, increasing serum half-life, increasing therapeutic half-life, modulating immunogenicity, modulating biological activity, or extending the circulation time of many biologically active molecules, including proteins, peptides, and particularly hydrophobic molecules. PEG has been used extensively in pharmaceuticais, on artificial implants, and in other applications where biocompatibility, lack of toxicity, and lack of immunogenicity are of importance. In order to maximize the desired properties of PEG, the total molecular weight and hydration state of the PEG polymer or polymers attached to the biologically active molecule must be sufficiently high to impart the advantageous characteristics typically associated with PEG polymer attachment, such as increased water solubility and circulating half life, while not adversely impacting the bioactivity of the parent molecule.<br>
[13]	PEG derivatives are frequently linked to biologically active molecules through<br>
reactive chemical functionalities, such as lysine, cysteine and histidine residues, the N-terminus and carbohydrate moieties. Proteins and other molecules often have a limited number of reactive sites available for polymer attachment Often, the sites most suitable for modification via polymer attachment play a significant role in receptor binding, and are necessary for retention of the biological activity of the molecule. As a result, mdiscriminate attachment of polymer chains to such reactive sites on a biologically active molecule often leads to a significant reduction or even total loss of biological activity of the polymer-modified molecule. R. Clark et al., (1996), J. Biol. Chem., 271:21969-21977. To form conjugates having sufficient polymer molecular weight for imparting the desired advantages to a target molecule, prior art approaches have typically involved random attachment of numerous polymer arms to the molecule, thereby increasing the risk of a reduction or even total loss in bioactivity of the parent molecule.<br>
[14)	Reactive sites that form the loci for attachment of PEG derivatives to proteins are<br>
dictated by the protein's structure. Proteins, including enzymes, are composed of various sequences of alphα-αmino acids, which have the general structure H2N—CHR—COOH. The alpha amino moiety (H2N-) of one amino acid joins to the carboxyl moiety (—COOH) of an adjacent amino acid to form amide linkages, which can be represented as —(NH—CHR—CO)n —,<br><br>
where the subscript "n" can equal hundreds or thousanas.   The tragment represented by R can<br>
contain reactive sites for protein biological activity and for attachment of PEG derivatives.<br>
[15]	For example, in the case of the amino acid lysine, there exists an —NH2 moiety in<br>
the epsilon position as well as in the alpha position. The epsilon "NH2 is free for reaction under<br>
conditions of basic pH. Much of the art in the field of protein derivatization with PEG has been<br>
directed to developing PEG derivatives for attachment to the epsilon —NH2 moiety of lysine<br>
residues present in proteins. "Polyethylene Glycol and Derivatives for Advanced PEGylation",<br>
Nektar Molecular Engineering Catalog, 2003, pp. 1-17. These PEG derivatives all have the<br>
common limitation, however, that they cannot be installed selectively among the often numerous<br>
lysine residues present on the surfaces of proteins. This can be a significant limitation in<br>
instances where a lysine residue is important to protein activity, existing in an enzyme active site<br>
for example, or in cases where a lysine residue plays a role in mediating the interaction of the<br>
protein with other biological molecules, as in the case of receptor binding sites.<br>
[16]	A second and equally important complication of existing methods for protein<br>
PEGylation is that the PEG derivatives can undergo undesired side reactions with residues other than those desired. Histidine contains a reactive imino moiety, represented structurally as — N(H)—, but many chemically reactive species that react with epsilon —NH3 can also react with — N(H)—. Similarly, the side chain of the amino acid cysteine bears a free sulfhydryl group, represented structurally as -SH. In some instances, the PEG derivatives directed at the epsilon --NH2 group of lysine also react with cysteine, histidine or other residues. This can create complex, heterogeneous mixtures of PEG-derivatized bioactive molecules and risks destroying the activity of the bioactive molecule being targeted. It would be desirable to develop PEG derivatives that permit a chemical functional group to be introduced at a single site within the protein that would then enable the selective coupling of one or more PEG polymers to the bioactive molecule at specific sites on the protein surface that are both well-defined and predictable.<br>
[17]	In addition to lysine residues, considerable effort in the art has been directed<br>
toward the development of activated PEG reagents that target other amino acid side chains, including cysteine, histidine and the N-terminus. See, e.g., U.S. Pat. No. 6,610,281 which is incorporated by reference herein, and "Polyethylene Glycol and Derivatives for Advanced PEGylation", Neklar Molecular Engineering Catalog, 2003, pp. 1-17. A cysteine residue can be introduced site-selectively into the structure of proteins using site-directed mutagenesis and other techniques known in the art, and the resulting free sulfhydryl moiety can be reacted with PEG derivatives that bear thiol-reactive functional groups.    This approach is comphcated.<br><br>
however, in max me introdcution of a free sulfhydryl<br>
 group can complicate the expression, folding and stability of the resulting protein. Thus, it would be desirable to have a means to introduce a chemical functional group into bioactive molecules that enables the selective coupling of one or more PEG polymers to the protein while simultaneously being compatible with (i.e., not engaging in undesired side reactions with) sulfhydryls and other chemical functional groups typically found in proteins.<br>
[18]	As can be seen from a sampling of the art, many of these derivatives that have<br>
been developed for attachment to the side chains of proteins, in particular, the - NH2 moiety on<br>
the lysine amino acid side chain and the -SH moiety on the cysteine side chain, have proven<br>
problematic in their synthesis and use. Some form unstable linlcages with the protein that are<br>
subject to hydrolysis and therefore decompose, degrade, or are otherwise unstable in aqueous<br>
environments, such as in the bloodstream. See Pedder, S.C, Semin Liver Dis. 2003;23 Suppl<br>
1:19-22 for a discussion of the stability of linkages in PEG-Intron®. Some form more stable<br>
linkages, but are subject to hydrolysis before the linkage is formed, which means that the<br>
reactive group on the PEG derivative may be inactivated before the protein can be attached.<br>
Some are somewhat toxic and are therefore less suitable for use in vivo. Some are too slow to<br>
react to be practically useful. Some result in a loss of protein activity by attaching to sites<br>
responsible for the protein's activity. Some are not specific in the sites to which they will attach,<br>
which can also result in a loss of desirable activity and in a lack of reproducibility of results. In<br>
order to overcome the challenges associated with modifying proteins with poly(ethylene glycol)<br>
moieties, PEG derivatives have been developed that are more stable (e.g., U.S. Patent 6,602,498,<br>
which is incorporated by reference herein) or that react selectively with thiol moieties on<br>
molecules and surfaces (e.g., U.S. Patent 6,610,281, which is incorporated by reference herein).<br>
There is clearly a need in the art for PEG derivatives that are chemically inert in physiological<br>
environments until called upon to react selectively to form stable chemical bonds.<br>
[19]	Recently, an entirely new technology in the protein sciences has been reported,<br>
which promises to overcome many of the limitations associated with site-specific modifications of proteins. Specifically, new components have been added to the protein biosynthetic machinery of the prokaryote Escherichia coli (E. coli) (e.g., L, Wang, et al., (2001), Science 292:498-500) and the eukaryote Sacchromyces cerevisiae (S. cerevisiae) (e.g., J. Chin et al., Science 301:964-7 (2003)), which has enabled the incorporation of non-genetically encoded amino acids to proteins in vivo. A number of new amino acids with novel chemical, physical or biological properties, including photoaffinity labels and photoisomerizable amino acids, photocrosslinking amino acids (see, e.g., Chin, J. W., et al. (2002) Proc. Natl. Acad. Sci. U. S.<br><br>
A, 99.1.11020; and, Chin, J. W., et al,, (2002) J, Am. Chem. soc. 124:9026-9027), keto<br>
amino acids, heavy atom containing amino acids, and glycosylated amino acids have been<br>
incorporated efficiently and with high fidelity into proteins in E. coli and in yeast in response to<br>
the amber codon, TAG, using this methodology. See, e.g., J. W. Chin et al., (2002), Journal of<br>
the American Chemical Society 124:9026-9027; J. W. Chin, &amp; P. G. Schultz, (2002),<br>
ChemBioChem 3(11):1135-1137; J. W. Chin, et al., (2002), PNAS United States of America<br>
99:11020-11024; and, L. Wang, &amp; P. G. Schultz, (2002), Chem. Comrn., 1:1-11. All references<br>
are incorporated by reference in their entirety. These studies have demonstrated that it is<br>
possible to selectively and routinely introduce chemical functional groups, such as ketone<br>
groups, alkyne groups and azide moieties, that are not found in proteins, that are chemically<br>
inert to all of the functional groups found in the 20 common, geneticallγ-encoded amino acids<br>
and that may be used to react efficiently and selectively to form stable covalent linlcages.<br>
[20]	The ability to incorporate non-genetically encoded amino acids into proteins<br>
permits the introduction of chemical functional groups that could provide valuable alternatives to the naturallγ-occurring functional groups, such as the epsilon -NH2 of lysine, the sulfhydryl -SH of cysteine, the imino group of histidine, etc. Certain chemical functional groups are known to be inert to the functional groups found in the 20 common, genetically-encoded amino acids but react cleanly and efficiently to form stable linkages. Azide and acetylene groups, for example, are known in the art to undergo a Huisgen [3+2] cycloaddition reaction in aqueous conditions in the presence of a catalytic amount of copper. See, e.g., Tomoe, et al., (2002) J. Org. Chem. 67:3057-3064; and, Rostovtsev, et al., (2002) Angew. Chem. Int. Ed. 41:2596-2599. By introducing an azide moiety into a protein structure, for example, one is able to incorporate a functional group that is chemically inert to amines, sulfhydryls, carboxylic acids, hydroxyl groups found in proteins, but that also reacts smoothly and efficiently with an acetylene moiety to form a cycloaddition product. Importantly, in the absence of the acetylene moiety, the azide remains chemically inert and unreactive in the presence of other protein side chains and under physiological conditions.<br>
[21]	The present invention addresses, among other things, problems associated with<br>
the activity and production of interferon polypeptides, and also addresses the production of an interferon polypeptide with improved biological or pharmacological properties, such as improved therapeutic half-life and/or modulation of one or more biological activities or side effects found with current IFN therapeutics.<br><br>
BRIEF SUMMARY OF THE INVENTION<br>
[22]	This invention provides hlFN polypeptides comprising one or more non-naturally<br>
encoded amino acids.<br>
[23]	In some embodiments, the hlFN polypeptide comprises one or more post-<br>
translational modifications.  In some embodiments, the hlFN polypeptide is linked to a linker,<br>
polymer, or biologically active molecule. In some embodiments, the hlFN polypeptide is linked<br>
to a bifunctional polymer, bifunctional linker, or at least one additional hlFN polypeptide.<br>
[24]	In some embodiments, the non-naturally encoded amino acid is linked to a water<br>
soluble polymer. In some embodiments, the water soluble polymer comprises a poly(ethylene glycol) moiety. In some embodiments, the non-naturally encoded amino acid is linlced to the water soluble polymer with a linker or is bonded to the water soluble polymer. In some embodiments, the poly(ethylene glycol) molecule is a bifunctional polymer. In some embodiments, the bifunctional polymer is linked to a second polypeptide. In some embodiments, the second polypeptide is a hlFN polypeptide. In some embodiments, the hlFN polypeptide comprises one or more naturallγ-encoded amino acid substitutions, additions, or deletions. In some embodiments, the hlFN polypeptide comprises one or more naturally encoded amino acid substitutions.<br>
[25]	In some embodiments, the hlFN polypeptide comprises at least two amino acids<br>
linked to a water soluble polymer comprising a poly(ethylene glycol) moiety.    In some<br>
embodiments, at least one amino acid is a non-naturally encoded amino acid.<br>
[26]	In some embodiments, one or more non-naturally encoded amino acids are<br>
incorporated at any position in one or more of the following regions corresponding to secondary structures in IFN as follows: 1-9 (N-terminus), 10-21 (A helix), 22-39 (region between A helix and B helix), 40-75 (B helix), 76-77 (region between B helix and C helix), 78-100 (C helix), 101-110 (region between C helix and D helbc), 111-132 (D helix), 133-136 (region between D and E helix), 137-155 (E helix), 156-165 (C-terminus) (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3, other interferons, or interferon-like cytokines such as limitin). In some embodiments, one or more non-naturally encoded amino acid are substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide): before position 1 (i.e., at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,25, 26, 27, 28,29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71,<br><br>
72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92,93,94, 95, 96, 97, 98,99,100,101,102,103,104,105,106,107,108,109,110, 111, 112,113,114, 115,116,117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156,157,158,159, 160,161, 162,163, 164,165, or 166 (i.e. at the carboxyl terminus). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in IFN: before position 1 (i.e. at the N terminus), 1,2, 3, 4. 5, 6, 7, 8, 9, 12, 13, 16, 19, 20, 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 40, 41, 42, 45, 46, 48, 49, 50,51, 58, 61,64,65, 68,69, 70, 71,73,74,77,78,79, 80, 81, 82, 83, 85, 86,89,90, 93,94, 96, 97, 100, 101, 103, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161,162,163,164, 165,166 (i.e. at the carboxyl terminus of the protein) (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in IFN: 6,9,12,13,16,41,45,46,48,49,61,64,65,96,100,101, 103, 106, 107, 108,110, 111,113, 114, 117, 120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 100, 106, 107, 108, 111, 113, 114 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 41,45,46,48,49 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 61, 64, 65, 101,103,110,117, 120,121,149 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 6, 9, 12, 13, 16, 96, 156, 159, 160, 161, 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 2, 3, 4, 5, 7. 8,16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 34, 78, 107 (SEQ ID NO: 2, or<br><br>
the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these or other positions is linked to a water soluble polymer, including but not limited to positions: before position 1 (i.e., at the N-termmus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,25,26, 27,28,29, 30, 31,32, 33, 34, 35, 36, 37, 38, 39,40,41, 42, 43, 44,45,46,47,48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161,162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus) (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these positions is linked to a water soluble polymer, including but not limited to positions: before position 1 (i.e. at the N terminus), 1, 2, 3,4, 5, 6, 7, 8, 9, 12, 13, 16,19, 20, 22, 23, 24,25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 40, 41,42,45,46, 48,49, 50, 51, 58,61,64, 65, 68, 69, 70, 71, 73, 74, 77, 78, 79, 80, 81, 82, 83, 85, 86, 89, 90, 93, 94, 96, 97, 100, 101, 103, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164, 165, 166 (i.e. at the carboxyl terminus) (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 6, 9, 12, 13, 16, 41, 45,46,48, 49, 61, 64, 65, 96, 100, 101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 100, 106, 107, 108, 111, 113, 114 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linlced to a water soluble polymer at one or more of the following positions: 41, 45, 46, 48,49 (SEQ ID NO: 2, or the coiresponding ammo acids in SEQ ED NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 61, 64, 65, 101, 103, 110, 117, 120, 121, 149 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following<br><br>
positions: 6, 9, 12, 13. 16, 96, 156, 159, 160, 161, 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[27]	In some embodiments, the one or more non-naturally encoded amino acids at one<br>
or more of the following positions is linked to one or more water-soluble polymer: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153,<br>
158,	163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding ammo acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linked to one or more water-soluble polymer: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the water soluble polymer is coupled to the IFN polypeptide to a non-naturally encoded amino acid at one or more of the following amino acid positions: 6, 9, 12, 13, 16, 41, 45, 46,48, 49, 61, 64, 65, 96, 100, 101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments the water soluble polymer is coupled to the IFN polypeptide at one or more of the following amino acid positions: 6, 9, 12, 13,16,41, 45,46, 48, 49, 61, 64, 65, 96, 100, 101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121, 149, 156,<br>
159,	160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these positions is linked to one or more water soluble polymers, positions: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions providing an antagonist: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97,105,109, 112, 118,148, 149,152, 153, 158, 163,164,165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide); a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on the intended site selected and desired activity. Human IFN antagonists include, but are not limited to, hlFN pol5T)eptides with one or more non-naturally encoded amino acid substitutions at 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94,124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, or any combinations thereof (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide).<br><br>
[28]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide): 31, 134,34,38, 129,36, 122,37, 121,41, 125, 124, 149, 117,39, 118, 120, 107, 108, 106, 100, 111,113,114,41,45,46,48,49,61,64,65,101,103,102, 110,117,120,121,149,96,6,9,12, 13, 16, 68, 70,109, 159, 161, 156, 160, 162, 24, 27, 78, 83, 85, 87, 89, 164. In one embodiment, a non-naturally encoded amino acid is substituted at position 38 of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these or other positions is linked to a water soluble polymer, including but not limited to positions: 31, 134, 34, 38, 129, 36,122,37,121,41, 125,124,149,117,39,118,120,107,108,106,100,111,113,114,41,45, 46,48,49,61,64,65, 101, 103,102,110, 117, 120, 121, 149,96,6,9, 12, 13,16,68,70, 109,<br>
159,	161, 156, 160, 162, 24, 27, 78, 83, 85, 87, 89, 164 (as in SEQ ID NO: 2, or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide).<br>
[29]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3): before position 1 (i.e., at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43,44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159,<br>
160,	161, 162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus) and one or more natural<br>
amino acid substitutions. In some embodiments, the one or more non-naturally encoded amino<br>
acids are coupled to a water soluble polymer. In some embodiments, the one or more non-<br>
naturally encoded amino acids are coupled to PEG. In one embodiment, the natural amino acid<br>
substitution is R149Y. In some embodiments, the natural amino acid substitution is R149E. In<br>
some embodiments, the natural amino acid substitution is R149S. In one embodiment, the non-<br>
natural ammo acid substitution is at position 107 and the natural amino acid substitution is<br>
R149Y. In one embodiment, the non-natural amino acid substitution is at position 106 and the<br>
natural amino acid substitution is R149Y. In some embodiments, the one or more naturally<br>
encoded amino acid substitution is at one or more of the following positions of hlFN (SEQ ID<br><br>
No: 3 or the corresponding amino acids in SEQ ID NO: 1 or 3), including but not limited to: 10, 16, 13, 79, 83, 85, 86, 87, 90, 91, 93, 94, 96, 120, 121, 124, 125, 128, 149. In some embodiments, the one or more naturally encoded amino acid substitution is one or more of the following substitutions (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3), including but not limited to: GlOE, M16R, RI3E, T79R, K83Q, K83S, Y85L, T86S, E87S, Q90R, Q91E, N93Q, D94V, E96K, R120K, K121T, Q124R, R125G, L128R, R149Y, R149E, R149S. In some embodiments, the natural amino acid substitution is at position 1 (the N-terminus). In some embodiments, one or more non-naturally encoded amino acids are substituted at one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's): 107, 78, 34, In some embodiments, the non-naturally encoded amino acid at one or more of these positions is coupled to a water soluble polymer: 107,78,34.<br>
[30]	One or more amino acids found in a limitin sequence may be substituted into a<br>
hlFN polypeptide (hybrid limitin/hlFN polypeptides). Examples include but are not limited to the natural amino acid substitutions described in the previous paragraph. Alternatively, a set of amino acids found in an interferon polypeptide may be replaced by a set of amino acids found in a limitin sequence. A set of amino acids may comprise contiguous amino acids or amino acids present in different portions of the molecule but are involved in a structural characteristic or biological activity of the polypeptide. The mouse limitin molecule has an improved CFU-GM toxicity profile compared to other IFNα proteins. Alignment of human IFNα-2a with the limitin protein sequence showed 30% amino acid identity. 50% sequence conservation was also observed. In particular, a prominent deletion in the limitin sequence between the C and D helices (in the loop between C and D hehces) was observed. The "HV" mutant was generated with the following substitutions in hIFNα-2a (SEQ ID NO: 2): D77-D94 is replaced with the mouse limitin sequence HERALDQLLSSLWRELQV. The "CD" mutant was generated with the following substitutions in hIFNα-2a (SEQ ID NO: 2): V105-D114 with GQSAPLP. This hybrid molecule with the loop region from limitin substituted into the human IFNα-2a protein ("CD" mutant) was found to have equivalent anti-viral activity as the WHO EFN standard. In addition to the one or more limitin amino acids, the hlFN polypeptide may comprise one or more non-naturally encoded amino acids at any one or more positions of the hlFN polypeptide. In some embodiments, the one or more non-naturally encoded amino acids may be linked to a water soluble polymer such as PEG or bonded directly to a water soluble polymer such as PEG. In addition to the natural amino acid substitutions in the HV or the CD mutant, one or more additional natural amino acid substitutions may be found in the hlFN  polypeptide.<br><br>
[31]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 34, 39, 45, 46,<br>
64, 65, 68, 78, 85, 87, 101, 107, 108, 111, 114,118, 124,125,145,146, 153, 156, 96, 149 (SEQ<br>
ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO:<br>
1, 3, or any other IFN polypeptide). In some embodiments, the hlFN polypeptide comprises one<br>
or more non-naturally encoded amino acids at one or more of the following positions linlced to<br>
one or more water-soluble polymer: 6, 16, 34, 39, 45, 46, 64, 65, 68, 78, 85, 87, 101, 107, 108,<br>
111, 114, 118, 124, 125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding<br>
amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN<br>
polypeptide). In Some embodiments, the hlFN polypeptide comprises one or more non-naturally<br>
encoded amino acids at one or more of the following positions linlced to one or more water-<br>
soluble polymer: 6, 16, 34, 39,45, 46, 64, 65, 68, 78, 85, 87, 101, 107, 108, 111, 114, 118, 124,<br>
125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the<br>
corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide) and comprises one<br>
or more naturally encoded amino acid substitution. In some embodiments, the hlFN pol3^eptide<br>
comprises one or more non-naturally encoded amino acids at one or more of the following<br>
positions linlced to one or more water-soluble polymer: 6, 16, 34, 39, 45, 46, 64, 65, 68, 78, 85,<br>
87, 101, 107, 108, 111, 114, 118, 124, 125, 145, 146, 153, 156,96, 149 (SEQ ID NO: 2, or the<br>
corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any otlier<br>
IFN polypeptide) and comprises one or more of the following naturally encoded amino acid<br>
substitutions GlOE, M16R, R13E, T79R, K83Q, K83S, Y85L, T86S, E87S, Q90R, Q91E,<br>
N93Q, D94V, E96K, R120K, K121T, Q124R. R125G, L128R, R149Y, R149E, R149S.<br>
[32]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[33]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions:  6, 16, 37, 45, 46, 78,<br><br>
87, 89, 107, 108 and one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45,46, 78, 87, 89, 107,108 that is linked to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 that is bonded to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, B87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[34]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3), In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the correspondmg amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[35]	In some embodunents, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45,46, 78, 87, 107, and 108 that is linked to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 that is bonded to a water soluble polymer and comprises one or more of the following naturally encoded amino<br><br>
acid substitutions: 179R. K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[36]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 37, 45,46, 89, and 107<br>
(SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some<br>
embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids<br>
at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In<br>
some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino<br>
acids at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[37]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 37,45,46, 89, and 107<br>
and comprises one or more of the following naturally encoded amino acid substitutions: T79R,<br>
L80A, Y85L, Y85S, E87S (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1<br>
or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally<br>
encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 that is<br>
linked to a water soluble polymer and comprises one or more of the following naturally encoded<br>
amino acid substitutions: T79R, L80A, Y85L, Y85S, E87S (SEQ ID NO: 2 or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide<br>
comprises one or more non-naturally encoded amino acids at one or more of the following<br>
positions; 37, 45, 46, 89, and 107 that is bonded to a water soluble polymer and comprises one<br>
or more of the following naturally encoded amino acid substitutions: T79R, L80A, Y85L,<br>
Y85S, E87S (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[38]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions; 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 128, 129, 131, 132, 133, 134, 135, 136, 137, 158, 159, 160, 161, 162, 163, 164, 165 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 23, 24, 25, 26,27, 28, 30, 31, 32, 33,128, 129, 131,132,133,134,135,136,137, 158, 159,160, 161, 162, 163, 164, 165 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the followine positions:  23. 24. 27. 31, 128, 131, 134, 158 (SEO<br><br>
ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linlced or bonded to a water soluble polymer: 23, 24, 27, 31, 128, 131, 134, 158 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO; 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 24, 27, 31, 128, 131, 134 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 24, 27, 31, 128, 131, 134 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[39]	In some embodiments, the IFN polypeptides of the invention comprise one or<br>
more non-naturally encoded amino acids at one or more of the following positions providing an antagonist: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153,158, 163, 164, 165, or any combination thereof (SEQ ID NO: 2, or the corresponding amino acids in SEQ ED NO: 1 or 3); a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on the site selected and desired activity. Human IFN antagonists include, but are not limited to, those with one or more substitutions at 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94,124,125, 127, 128, 129,131, 132,133,134,135,136, 137, or any combination thereof (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[40]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's): 31, 134, 34, 38, 129, 36, 122, 37, 121, 41, 125, 124, 149, 117,39, 118, 120, 107, 108, 106, 100, 111, 113, 114,41,45,46,48,49,61, 64, 65, 101, 103, 102, 110, 117, 120, 121, 149, 96, 6, 9, 12, 13, 16, 68, 70, 109, 159, 161, 156, 160, 162, 24, 27, 78, 83, 85, 87, 89, 164. In one embodiment, a non-naturally encoded amino acid is substituted at position 38 of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, the non-naturally encoded amino acid at these or other positions is linked to a water soluble polymer, including but not limited to positions: 31, 134, 34, 38, 129, 36, 122, 37, 121, 41, 125, 124, 149, 117, 39, 118, 120, 107, 108, 106, 100, HI, 113, 114, 41, 45, 46, 48, 49, 61, 64, 65, 101, 103, 102, 110, 117, 120, 121, 149, 96, 6, 9, 12, 13, 16, 68, 70,109, 159,161, 156, 160,162, 24, 27, 78, 83, 85, 87, 89, 164.<br><br>
[41J	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acid and a naturally encoded amino acid substitution. In some embodiments, the non-naturally encoded amino acid present in the hlFN polypeptide is linlced to a water soluble polymer and the hlFN polypeptide comprises one or more naturally encoded amino acid substitution.<br>
[42]	In some embodiments, the hlFN polypeptide comprises a substitution, addition<br>
or deletion that modulates affinity of the hlFN polypeptide for a hlFN polypeptide receptor. In<br>
some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that<br>
increases the stability of the hlFN polypeptide when compared with the stability of the<br>
corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the<br>
hlFN polypeptide comprises a substitution, addition, or deletion that modulates the<br>
immunogenicity of the hlFN polypeptide when compared with the immunogenicity of the<br>
corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the<br>
hlFN polypeptide comprises a substitution, addition, or deletion that modulates serum half-life<br>
or circulation time of the hlFN polypeptide when compared with the serum half-life or<br>
circulation time of the corresponding hlFN without the substitution, addition, or deletion.<br>
[43]	In some embodiments, the hlFN polypeptide comprises a substitution, addition,<br>
or deletion that modulates hlFN polypeptide receptor conformation when compared with hlFN polypeptide receptor conformation with the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that modulates one or more downstream signaling events of the hlFN polypeptide receptor when compared to the downstream signaling events of the hlFN polypeptide receptor with the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that moddates hlFN polypeptide receptor binding kinetics when compared to hlFN polypeptide receptor binding kinetics with the corresponding hlFN without the substitution, addition, or deletion.<br>
[44]	In some embodiments, the hlFN polypeptide comprises a substitution, addition,<br>
or deletion that increases the aqueous solubility of the hlFN polypeptide when compared to aqueous solubility of the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hEFN polypeptide comprises a substitution, addition, or deletion that increases the solubility of the hlFN polypeptide produced in a host cell when compared to the solubility of the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that increases<br><br>
the expression of the hIFN popypeptide in a host cell or increases increases synthesis in vitro when compared to the expression or synthesis of the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that increases protease resistance of the hlFN polypeptide when compared to the protease resistance of the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that modulates interaction with one or more members of the opioid family of receptors. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that modulates 5-HT brain neurotransmission. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that modulates induction of indoleamine 2,3-dioxygenase (IDO). In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that modulates one or more of the biological activities of IFN, including but not limited to, side effects found with current IFN therapeutics. In some embodiments, the hlFN polypeptide comprises a substitution, addition, or deletion that has modulated toxicity when compared to the toxicity of the corresponding hlFN without the substitution, addition, or deletion. In some embodiments, the hlFN polypeptide comprises a non-naturally encoded amino acid linked to a water soluble polymer that modulates one or more side effects found with IFN. In some embodiments, the hlFN polypeptide comprises a non-naturally encoded amino acid hnked to a water soluble polymer and has modulated toxicity. In some embodiments, the hlFN polypeptide comprises a substitution, addition, deletion, or non-naturally encoded amino acid that has modulated anti-viral activity when compared to the anti-viral activity of the corresponding hlFN without the substitution, addition, deletion, or non-naturally encoded amino acid. In some embodiments, the hlFN polypeptide comprises a substitution, addition, deletion, or non-naturally encoded amino acid that has modulated immunogenicity when compared to the immunogenicity of the corresponding hlFN without the substitution, addition, deletion, or non-naturally encoded amino acid. In some embodiments, the hlFN polypeptide comprises a substitution, addition, deletion, or non-naturally encoded amino acid that has modulated antitumor activity when compared to the anti-tumor activity of the corresponding hlFN without the substitution, addition, deletion, or non-naturally encoded amino acid. In some embodiments, the hlFN polypeptide comprises a substitution, addition, deletion, or non-naturally encoded amino acid that has modulated anti-infective activity when compared to the anti-infective activity of the corresponding hlFN without the substitution, addition, deletion, or non-naturally encoded amino acid. In some embodiments, the hlFN polypeptide comprises a substitution, addition, deletion, or non-naturally encoded amino acid that has modulated prophylaxis activity for infectious<br><br>
agents when compared to the prophylaxis  activity for infectious agents of the corresponding hlFN without the substitution, addition, deletion, or non-naturally encoded amino acid. In some embodiments, the hlFN polypeptide comprises a substitution, addition, deletion, or non-naturally encoded amino acid that has modulated tumor prophylaxis activity when compared to the tumor prophylaxis activity of the corresponding hlFN without the substitution, addition, deletion, or non-naturally encoded amino acid.<br>
[45]	In some embodiments the amino acid substitutions in the hlFN polypeptide may<br>
be with naturally occurring or non-naturally occurring amino acids, provided that at least one<br>
substitution is with a non-naturally encoded amino acid.<br>
[46]	In some embodiments, the non-naturally encoded amino acid comprises a<br>
carbonyl group, an acetyl group, an aminooxy group, a hydrazine group, a hydrazide group, a<br>
semicarbazide group, an azide group, or an alkyne group.<br>
[47]	In some embodiments, the non-naturally encoded amino acid comprises a<br>
carbonyl group. In some embodiments, the non-naturally encoded amino acid has the structure:<br><br>
amino terminus modification group, and R4 is H, an amino acid, a polypeptide, or a carboxy<br>
terminus modification group.<br>
[48]	In some embodiments, the non-naturally encoded amino acid comprises an<br>
aminooxy group.   In some embodiments, the non-naturally encoded amino acid comprises a<br>
hydrazide group. In some embodiments, the non-naturally encoded amino acid comprises a<br>
hydrazine group.    In some embodiments, the non-naturally encoded amino acid residue<br>
comprises a semicarbazide group.<br>
[49]	In some embodiments, the non-naturally encoded amino acid residue comprises<br>
an azide group. In some embodiments, the non-naturally encoded amino acid has the structure:<br><br><br>
[50]	In some embodiments, the non-naturally encoded amino acid comprises an<br>
alkyne group. In some embodiments, the non-naturally encoded amino acid has the structure:<br><br>
present; m is 0-10, R2 is H, an amino acid, a polypeptide, or an amino terminus modification<br>
group, and R3 is H, an amino acid, a polypeptide, or a carboxy terminus modification group.<br>
[51]	In some embodiments, the pol3T3eptide is a hlFN polypeptide agonist, partial<br>
agonist, antagonist, partial antagonist, or inverse agonist. In some embodiments, the hlFN polypeptide agonist, partial agonist, antagonist, partial antagonist, or inverse agonist comprises a non-naturally encoded amino acid linked to a water soluble polymer. In some embodiments, the water soluble polymer comprises a poly(ethylene glycol) moiety. In some embodiments, the hlFN polypeptide agonist, partial agonist, antagonist, partial antagonist, or inverse agonist comprises a non-naturally encoded amino acid and one or more post-translational modification, linker, polymer, or biologically active molecule. In some embodiments, the non-naturally encoded amino acid linked to a water soluble polymer is present within the Site II region (the region of the protein encompassing the AC helical-bundle face, amino terminal region of helix A and a portion of helix C) of the hIFN polypeptide. In some embodiments, the hlFN polypeptide comprising a non-naturally encoded amino acid linked to a water soluble polymer prevents dimerization of the hlFN polypeptide receptor by preventing the hlFN polypeptide antagonist from binding to a second hlFN polypeptide receptor molecule.<br>
[52]	The  present  invention  also  provides  isolated  nucleic  acids  comprising  a<br>
polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 21 or 22 wherein the polynucleotide comprises at least one selector codon. In some embodiments, the selector codon is selected from the group consisting of an amber codon, ochre codon, opal codon, a unique codon, a rare codon, a five-base codon, and a four-base codon.<br>
[53]	The present invention also provides methods of making a h IFN pol ypeptide<br>
linked to a water soluble polymer. In some embodiments, the method comprises contacting an isolated hlFN polypeptide comprising a non-naturally encoded amino acid with a water soluble polymer comprising a moiety that reacts with the non-naturally encoded amino acid. In some embodiments, the non-naturally encoded amino acid incorporated into the hlFN polypeptide is reactive toward a water soluble polymer that is otherwise unreactive toward any of the 20 common amino  acids.     In  some  embodiments,  the  non-naturally  encoded  amino  acid<br><br>
incorporated into the hlFN polypeptide is reactive toward a linker, polymer, or biologically<br>
active molecule that is otherwise unreactive toward any of the 20 common amino acids.<br>
[54]	In some embodiments, the hlFN polypeptide linked to the water soluble polymer<br>
is made by reacting a hlFN polypeptide comprising a carbonyl-containing amino acid with a<br>
poly(ethylene glycol) molecule comprising an aminooxy, hydrazine, hydrazide or semicarbazide<br>
group.   In some embodiments, the aminooxy, hydrazine, hydrazide or semicarbazide group is<br>
linked to the poly(ethylene glycol) molecule through an amide linlcage.<br>
[55]	In some embodiments, the hlFN polypeptide linked to the water soluble polymer<br>
is made by reacting a poly(ethylene glycol) molecule comprising a carbonyl group with a<br>
polypeptide comprising a non-naturally encoded   amino acid that comprises an aminooxy,<br>
hydrazine, hydrazide or semicarbazide group.<br>
[56]	In some embodiments, the hlFN polypeptide linked to the water soluble polymer<br>
is made by reacting a hlFN polypeptide comprising an alkyne-containing amino acid with a<br>
poly(ethylene glycol) molecule comprising an azide moiety. In some embodiments, the azide or<br>
alkyne group is linked to the poly(ethylene glycol) molecule through an amide linlcage.<br>
[57]	In some embodiments, the hlFN polypeptide linked to the water soluble polymer<br>
is made by reacting a hlFN polypeptide comprising an azide-containing amino acid with a<br>
poly(ethylene glycol) molecule comprising an alkyne moiety. In some embodiments,.the azide<br>
or alkyne group is linked to the poly(ethylene glycol) molecule through an amide linkage.<br>
[58]	In some embodiments, the poly(ethylene glycol) molecule has a molecular weight<br>
of between about 0.1 and about 100 kDa.   In some embodiments, the poly(ethylene glycol)<br>
molecule has a molecular weight of between 0.1 kDa and 50 kDa.<br>
[59]	In some embodiments, the poly(ethylene glycol) molecule is a branched polymer.<br>
In some embodiments, each branch of the poly(ethylene glycol) branched polymer has a<br>
molecular weight of between 1 kDa and 100 kDa, or between 1 kDa and 50 kDa.<br>
[60]	In some embodiments, the water soluble polymer linked to the hlFN polypeptide<br>
comprises a polyalkylene glycol moiety.   In some embodiments, the non-naturally encoded<br>
amino acid residue incorporated into the hlFN polypeptide comprises a carbonyl group, an<br>
aminooxy group, a hydrazide group, a hydrazine, a semicarbazide group, an azide group, or an<br>
alkyne  group.     In  some  embodiments,  the  non-naturally  encoded  amino  acid residue<br>
incorporated into the hEFN polypeptide comprises a carbonyl moiety and the water soluble<br>
polymer comprises an aminooxy, hydrazide, hydrazine, or semicarbazide moiety.   In some<br>
embodiments, the non-naturally encoded amino acid residue incorporated into the hlFN<br>
polypeptide comprises an alkyne moiety and the water soluble polymer comprises an azide<br><br>
moiety, m some embodiments, the non-naturally encoded ammo acid residue incorporated into the hlFN polypeptide comprises an azide moiety and the water soluble polymer comprises an alkyne moiety.<br>
[61]	The present invention also provides compositions comprising a hlFN polypeptide<br>
comprising a non-naturally encoded amino acid and a pharmaceutically acceptable carrier.   In<br>
some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer.<br>
[62]	The present invention also provides cells comprising a polynucleotide encoding<br>
the hlFN polypeptide comprising a selector codon. In some embodiments, the cells comprise an orthogonal KNA synthetase and/or an orthogonal tRNA for substituting a non-naturally encoded amino acid into the hlFN polypeptide.<br>
[63]	The present invention also provides methods of making a h IFN pol ypeptide<br>
comprising a non-naturally encoded amino acid. In some embodiments, the methods comprise<br>
culturing cells comprising a polynucleotide or polynucleotides encoding a hlFN polypeptide, an<br>
orthogonal RNA synthetase and/or an orthogonal tRNA under conditions to permit expression of<br>
the hlFN polypeptide; and purifying the hlFN polypeptide from the cells and/or culture medium.<br>
[64]	The present invention also provides methods of increasing therapeutic half-life,<br>
serum half-life or circulation time of hlFN polypeptides. The present invention also provides<br>
methods of modulating immunogenicity of hlFN polypeptides. The present invention also<br>
provides methods of modulating toxicity of hlFN polypeptides. The present invention also<br>
provides methods of modulating side effects of current EFN therapeutics. In some embodiments,<br>
the methods comprise substituting a non-naturally encoded amino acid for any one or more<br>
amino acids in naturally occurring hlFN polypeptides and/or linking the hlFN polypeptide to a<br>
linker, a polymer, a water soluble polymer, or a biologically active molecule.<br>
[65]	The present invention also provides methods of treating a patient in need of such<br>
treatment with an effective amount of a hEFN molecule of the present invention. In some<br>
embodiments, the methods comprise administering to the patient a therapeuticallγ-effective<br>
amount of a pharmaceutical composition comprising a hlFN polypeptide comprising a non-<br>
naturally-encoded amino acid and a pharmaceutically acceptable carrier. In some embodiments,<br>
the non-naturally encoded amino acid is linlced to a water soluble polymer.<br>
[66]	The present invention also provides hlFN polypeptides comprising a sequence<br>
shown in SEQ ID NO: 1, 2, 3 or any other hlFN polypeptide sequence or interferon-like cytokine such as limitin, except that at least one amino acid is substituted by a non-naturally encoded amino acid. In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer.     In some embodiments, the water soluble polymer comprises a<br><br>
poly(ethylene glycol) moiety in some embodiments, tne non-naturally encoded amino acid comprises a carbonyl group, an aminooxy group, a hydrazide group, a hydrazine group, a semicarbazide group, an azide group, or an alkyne group.<br>
[67]	The present invention also provides pharmaceutical compositions comprising a<br>
pharmaceutically acceptable carrier and a hlFN polypeptide comprising the sequence shown in SEQ ID NO: 1, 2, 3 or any other IFN polypeptide sequence or interferon-like cytokine such as limitin, wherein at least one amino acid is substituted by a non-naturally encoded amino acid. In some embodiments, the non-naturally encoded amino acid comprises a saccharide moiety. In some embodiments, the water soluble polymer is linked to the polypeptide via a saccharide moiety. In some embodiments, a linker, polymer, or biologically active molecule is linked to the hlFN polypeptide via a saccharide moiety.<br>
[68]	The present invention also provides a hlFN polypeptide comprising a water<br>
soluble polymer linked by a covalent bond to the hlFN polypeptide at a single amino acid. In some embodiments, the water soluble polymer comprises a poly(ethylene glycol) moiety. In some embodiments, the amino acid covalently linked to the water soluble polymer is a non-naturally encoded amino acid present in the polypeptide.<br>
[69]	The present invention provides a hlFN polypeptide comprising at least one linker,<br>
polymer, or biologically active molecule, wherein said linker, polymer, or biologically active molecule is attached to the polypeptide through a functional group of a non-naturally encoded amino acid ribosomally incorporated into the polypeptide. In some embodiments, the polypeptide is monoPEGylated. The present invention also provides a hlFN polypeptide comprising a linker, polymer, or biologically active molecule that is attached to one or more non-naturally encoded amino acid wherein said non-naturally encoded amino acid is ribosomally incorporated into the polypeptide at pre-selected sites.<br>
[70]	In another embodiment, conjugation of the hlFN polypeptide comprising one or<br>
more non-naturally occurring amino acids to another molecule, including but not limited to PEG, provides substantially purified hlFN due to the unique chemical reaction utilized for conjugation to the non-natural amino acid. Conjugation of hlFN comprising one or more non-naturally encoded amino acids to another molecule, such as PEG, may be performed with other purification techniques performed prior to or following the conjugation step to provide substantially pure hlFN. BRIEF DESCRIPTION OF THE DRAWINGS<br>
[71]	Figure 1 - A diagram of the general structure for the four helical bundle protein<br>
Interferon alphα-2 (IFNα-2) is shown.<br><br>
[72]	Figure 2 - A diagram of results obtained from the pSTATl assay is shown.<br>
[73]	Figure 3 - A diagram showing the specificity of the pSTATI assay is shown.<br>
[74]	Figure 4 - A diagram of the effect of endotoxin on the pSTATI assay is shown.<br>
[75]	Figure 5 - A diagram of results obtained from an anti-βroliferation assay is<br>
shown.<br>
[76]	Figure 6 - A diagram of the effect of endotoxin on the anti-βroliferation assay is<br>
shown.<br>
[77]	Figure 7 - A diagram of results obtained from an assay measuring induction of<br>
HLA expression is shown.<br>
[78]	Figure 8 - A diagram of Tyk2 phosphorylation in U266 cells is shown.<br>
[79]	Figure 9 - A diagram of CPE results obtained with nine PEGylated hlFN<br>
polypeptides and PEGASYS® is shown.<br>
[80]	Figure 10 - A graph of CFU-GM colony count vs. International Units is shown<br>
for PEGASYS® and four hlFN polypeptides.<br>
[81]	Figure 11 - An analysis of anti-viral IC50 and KD for IFNR2 is shown.<br>
[82]	Figure 12 - Results from a hematopoietic toxicity assay are shown.<br>
[83]	Figure 13 - Results from a hematopoietic toxicity assay are shown.<br>
[84]	Figure 14 -Results from a hematopoietic toxicity assay are shown.<br>
[85]	Figure 15 - Results from a hematopoietic toxicity assay are shown.<br>
[86]	Figure 16 - Results from a hematopoietic toxicity assay are shown.<br>
[87]	Figure 17 - Results from a hematopoietic toxicity assay are shown.<br>
[88]	Figure 18 - A diagram of CPE results obtained with five PEGylated hlFN<br>
polypeptides and PEGASYS®.<br>
[89]	Figure 19 - A diagram is shown of the structure of linear, 30 kDa monomethoxγ-<br>
poly(ethylene glycol)-2-αminooxy ethylamine carbamate hydrochloride.<br>
[90]	Figure   20   -  A   diagram   is   shown   of  the   synthesis   of  mPEG   (40K)<br>
aminoethoxyamine hydrochloride from mPEG (40K) p-nitrophenolcarbonate.<br>
DEFINITIONS<br>
[91]	It is to be understood that this invention is not limited to the particular<br>
methodology, protocols, cell lines, constructs, and reagents described herein and as such may<br>
vary.     It is also to be understood that the terminology used herein is for the purpose of<br>
describing particular embodiments only, and is not intended to limit the scope of the present<br>
invention, which will be limited only by the appended claims.<br><br>
[92]	As used herein and in the appended claims, the singular forms "a," "an," and<br>
"the" include plural reference unless the context clearly indicates otherwise. Thus, for example, reference to a "hlFN" is a reference to one or more such proteins and includes equivalents thereof Icnown to those of ordinary skill in the art, and so forth.<br>
[93]	Unless defined otherwise, all technical and scientific terms used herein have the<br>
same meaning as conraionly understood to one of ordinary skill in the art to which this invention belongs. Although any methods, devices, and materials similar or equivalent to those described herein can be used in the practice or testing of the invention, the preferred metliods, devices and materials are now described.<br>
[94]	All publications and patents mentioned herein are  incorporated herein by<br>
reference for the purpose of describing and disclosing, for example, the constructs and methodologies that are described in the publications, which might be used in connection with the presently described invention. The publications discussed herein are provided solely for their disclosure prior to the filing date of the present application. Nothing herein is to be construed as an admission that the inventors are not entitled to antedate such disclosure by virtue of prior invention or for any other reason.<br>
[95]	The term "substantially purified" refers to a hlFN polypeptide that may be<br>
substantially or essentially free of components that normally accompany or interact with the protein as found in its naturally occurring environment, i.e. a native cell, or host cell in the case of recombinantly produced hlFN polypeptides. hlFN polypeptide that may be substantially free of cellular material includes preparations of protein having less than about 30%, less than about 25%, less than about 20%, less than about 15%, less than about 10%, less than about 5%, less than about 4%, less than about 3%, less than about 2%, or less than about 1% (by dry weight) of contaminating protein. When the hlFN polypeptide or variant thereof is recombinantly produced by the host cells, the protein may be present at about 30%, about 25%, about 20%, about 15%, about 10%, about 5%, about 4%, about 3%, about 2%, or about 1% or less of the dry weight of the cells. When the hlFN polypeptide or variant thereof is recombinantly produced by the host cells, the protein may be present in the culture medium at about 5g/L, about 4g/L, about 3g/L, about 2g/L, about Ig/L, about 750mg/L, about 500mg/L, about 250mg/L, about 100mg/L, about 50mg/L, about 10mg/L, or about Img/L or less of the dry weight of the cells. Thus, "substantially purified" hlFN polypeptide as produced by the methods of the present invention may have a purity level of at least about 30%, at least about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 55%, at least about 60%, at least about 65%, at least about 70%, specifically, a purity level of at least about 75%, 80%, 85%, and more<br><br>
specifically, a purity level of at least about 90%, a purity level of at least about 95%, a purity level of at least about 99% or greater as determined by appropriate methods such as SDS/PAGE analysis, RP-HPLC, SEC, and capillary electrophoresis.<br>
[96]	A 'Recombinant host cell" or "host cell" refers to a cell that includes an<br>
exogenous polynucleotide, regardless of the method used for insertion, for example, direct uptake, transduction, f-mating, or other methods known in the art to create recombinant host cells. The exogenous polynucleotide may be maintained as a nonintegrated vector, for example, a plasmid, or alternatively, may be integrated into the host genome.<br>
[97]	As used herein, the term "medium" or "media" includes any culture medium,<br>
solution, solid, semi-solid, or rigid support that may support or contain any host cell, including<br>
bacterial host cells, yeast host cells, insect host cells, plant host cells, eukaryotic host cells,<br>
mammalian host cells, CHO cells, prokaryotic host cells, E. coli, or Pseudomonas host cells, and<br>
cell contents. Thus, the term may encompass medium in which the host cell has been grown,<br>
e.g., medium into which the hlFN polypeptide has been secreted, including medium either<br>
before or after a proliferation step. The term also may encompass buffers or reagents that<br>
contain host cell lysates, such as in the case where the hlFN polypeptide is produced<br>
intracellularly and the host cells are lysed or disrupted to release the hlFN polypeptide.<br>
[98]	"Reducing agent," as used herein with respect to protein refolding, is defined as<br>
any compound or material which maintains sulfhydryl groups in the reduced state and reduces intrα- or intermolecular disulfide bonds. Suitable reducing agents include, but are not limited to, dithiothreitol (DTT), 2-mercaptoethanol, dithioerythritol, cysteine, cysteamine (2-aminoethanethiol), and reduced glutathione. It is readily apparent to those of ordinary skill in the art that a wide variety of reducing agents are suitable for use in the methods and compositions of the present invention.<br>
[99]	"Oxidizing agent," as used hereinwith respect to protein refolding, is defined as<br>
any compound or material which is capable of removing an electron from a compound being oxidized. Suitable oxidizing agents include, but are not limited to, oxidized glutathione, cystine, cystamine, oxidized dithiothreitol, oxidized erythreitol, and oxygen. It is readily apparent to those of ordinary sldll in the art that a wide variety of oxidizing agents are suitable for use in the methods of the present invention.<br>
[100]	"Denaturing agent" or "denaturant," as used herein, is defined as any compound<br>
or material which will cause a reversible unfolding of a protein. The strength of a denaturing agent or denaturant will be determined both by the properties and the concentration of the particular denaturing agent or denaturant.   Suitable denaturing agents or denaturants may be<br><br>
chaotropes, detergents, organic solvents, water miscible solvents, phospholipids, or a<br>
combination of two or more such agents. Suitable chaotropes include, but are not limited to,<br>
urea, guanidine, and sodium thiocyanate. Useful detergents may include, but are not limited to,<br>
strong detergents such as sodium dodecyl sulfate, or polyoxyethylene ethers (e.g. Tween or<br>
Triton detergents), Sarkosyl, mild non-ionic detergents (e.g., digitonin), mild cationic detergents<br>
such as N-&gt;2,3-(Dioleyoxy)-propyl-N,N,N-trimethylanimonium, mild ionic detergents (e.g.<br>
sodium cholate or sodium deoxycholate) or zwitterionic detergents including, but not limited to,<br>
sulfobetaines (Zwittergent), 3-(3-chlolamidopropyl)dimethylanimonio-1-βropane sulfate<br>
(CHAPS), and 3-(3-chlolamidopropyl)dimethylammonio-2-hydroxγ-l-βropane sulfonate<br>
(CHAPSO). Organic, water miscible solvents such as acetonitrile, lower alkanols (especially C2<br>
- C4 alkanols such as ethanol or isopropanol), or lower alkandiols (especially C2 - C4 alkandiols<br>
such as ethylene-glycol) may be used as denaturants. Phospholipids useful in the present<br>
invention may be naturally occurring phospholipids such as phosphatidylethanolamine,<br>
phosphatidylcholine, phosphatidylserine, and phosphatidylinositol or synthetic phospholipid<br>
derivatives	or        variants	such	as	dihexanoylphosphatidylcholine	or<br>
diheptanoylphosphatidylcholine.<br>
[101]	"Refolding," as used herein describes any process, reaction or method which<br>
transforms disulfide bond containing polypeptides from an improperly folded or unfolded state<br>
to a native or properly folded conformation with respect to disulfide bonds.<br>
[102]	"Cofolding," as used herein, refers specifically to refolding processes, reactions,<br>
or methods which employ at least two polypeptides which interact with each other and result in the transformation of unfolded or improperly folded polypeptides to native, properly folded polypeptides.<br>
[103]	As used herein, "interferon" or "IFN" shall include those polypeptides and<br>
proteins that have at least one biological activity of an interferon, including but not limited to IFNα, IFN[3, IFNy, IFNco, IFNe, or IFNt or interferon-like cytokines such as limitin (such as those  described  in  U.S.  Patent  4,414,150;  4,456,748;  4,727,138;  4,762,791,  4,929,554 5,096,705;  4,695,623; 4,614,651; 4,678,751;  4,925,793;  5,460,811;  5,120,832;  4,780,530 4,908,432;  4,970,161; 4,973,479; 4,975,276;  5,098,703;  5,278,286;  5,661,009;  6,372,206 6,433,144;  6,472,512;  6,572,853;  6,703,225;  6,200,780;  6,299,869;  6,300,475;  6,323,006 6,350,589;  5,705,363; 5,738,845; 5,789,551;  6,117,423;  6,174,996;  5,540,923;  5,541,293 5,541,312; 5,554,513; 5,593,667 which are incorporated by reference herein), as well as IFN analogs, IFN isoforms, IFN mimetics, IFN fragments, hybrid IFN proteins, fusion proteins, oligomers and multimers, homologues, glycosylation pattern variants, variants, splice variants.<br><br>
and mutents, thereof regardless of the biological acitivity of same, and of the method of synthesis or manufacture thereof including, but not limited to, recombinant (whether produced from cDNA, genomic DNA, synthetic DNA or other form of nucleic acid), in vitro, in vivo, by microinjection of nucleic acid molecules, synthetic, transgenic, and gene activated methods. Specific examples of IFN include, but are not limited to, IFNγ-lb (Actimmune®), IFNβ-la (Avonex®, and Rebif®), IFNβ-lb (Betaseron®), consensus IFN, IFN alfacon-1 (Infergen®), IFNα-2 (Intron A®), IFNα-2a (Roferon-A®), Peginterferon alfα-2a (PEGASYS®), Peginterferon alfα-Zb (PEG-Intron®), IFN analog, IFN mutants, altered glycosylated human IFN, and PEG conjugated IFN analogs. Specific examples of cells modified for expression of endogenous human IFN are described in Devlin et al., J. Leukoc. Biol. 41:306 (1987); U.S. Patent Nos. 6,610,830; 6,482,613; 6,489,144; 6,159,712; 5,814,485; 5,710,027; 5,595,888; 4,966,843; which are incorporated by reference herein. See also, U.S. Patent Nos. 6,716,606; 6,379,661; 6,004,548; 5,830,705; 5,582,823; 4,810,643; and 6,242,218, which are incorporated by reference herein, for expression of GH family members.<br>
[104]	The term "human IFN (hlFN)" or "hlFN polypeptide" refers to mterferon or IFN<br>
as described above, as well as a polypeptide that retains at least one biological activity of a naturallγ-occurring hlFN. The term "hlFN polypeptides" or "hlFN" also includes the pharmaceutically acceptable salts and prodrugs, and prodrugs of the salts, polymorphs, hydrates, solvates, biologicallγ-αctive fragments, biologicallγ-αctive variants and stereoisomers of the naturallγ-occurring human IFN as well as agonist, mimetic, and antagonist variants of the naturallγ-occurring human IFN and polypeptide fusions thereof. Examples of hlFN polypeptides include, but are not limited to, those described in U.S. Patent No. 4,604,284; 5,582,824; 6,531,122; 6,204,022; 6,120,762; 6,046,034; 6,036,956; 5,939,286; 5,908,626; 5,780,027; 5,770,191; 5,723,125; 5,594,107; 5,378,823; 4,898,931; 4,892,743, which are incorporated by reference herein. Fusions comprising additional amino acids at the amino terminus, carboxyl terminus, or both, are encompassed by the term "hlFN polypeptide." Exemplary fusions include, but are not limited to, e.g., methionyl IFN in which a methionine is linlced to the N-terminus of hlFN resulting from the recombinant expression of the mature form of hlFN laclcing the secretion signal peptide or portion thereof, fusions for the purpose of purification (including but not limited to, to poly-histidine or affinity epitopes), fusions with serum albumin binding peptides and fusions with serum proteins such as serum albumin. Also the term '"hIFN polypeptide" also includes hybrid molecules that have one or more amino acid substitutions from limitin or other interferon-like cytokines. U.S. Patent No. 5,750,373, which is incorporated by reference herein, describes a method for selecting novel proteins such as growth<br><br>
hormone and antibody fragment variants having altered binding properties for their respective<br>
receptor molecules. The method comprises fusing a gene encoding a protein of interest to the<br>
carboxy terminal domain of the gene III coat protein of the filamentous phage Ml 3. The<br>
naturallγ-occurring hlFN nucleic acid and amino acid sequences for full-length and mature<br>
forms are Icnovm, as are variants such as single amino acid variants or splice variants.<br>
[105]	Consensus interferon is a recombmant type 1 interferon containing 166 amino<br>
acids. Consensus IFN was derived by scanning the sequences of several natural alpha interferons and assigning the most frequently observed amino acid in each corresponding position. Consensus IFN, when compared on an equal mass basis with IFNα-2a and α-2b in in vitro assays, typically displays 5-10 times higher biological activity (Blatt et al. J. Interferon Cytokine Res. 1996;l6;489-99).<br>
[106]	Modified hlFN polypeptides may exhibit one or more properties or biological<br>
activities found with a different interferon molecule. For example, a hlFN polypeptide that was<br>
generated from an IFNα-2a amino acid sequence and that comprises one or more non-naturally<br>
encoded amino acid that is imPEGylated or PEGylated may exhibit one or more biological<br>
activity that is found with IFNβ. One such activity may be anti-βroliferative activity.<br>
[107]	For the complete full-length naturallγ-occurring IFNα-2a amino acid sequence as<br>
well as the mature naturallγ-occurring IFNα-2a amino acid sequence, see SEQ ID NO: 1, and SEQ ID NO: 2, respectively, herein. In some embodiments, hlFN polypeptides of the invention are substantially identical to SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, or any other sequence of an interferon polypeptide or interferon-like cytokine such as limitin. Nucleic acid molecules encoding hlFN mutants and mutant hlFN polypeptides are well known and include, but are not limited to, those disclosed in U.S. Patent No.: 6,331,525; 6,069,133; 5,955,307; 5,869,293; 5,831,062; 5,081,022; 5,004,689; 4,738,931; 4,686,191; which are incorporated by reference herein. Examples of hlFN mutants include those disclosed in U.S. Patent Nos. 6,514,729 and 5,582,824, which are incorporated by reference herein.<br>
[108]	Interferons   have   a   variety   of  biological   activities,   including   anti-viral,<br>
immunoregulatory and anti-βroliferative properties, and have been utilized as therapeutic agents for treatment of diseases such as cancer, and various viral diseases. Interferon-α's have been shown to inhibit various types of cellular proliferation, and are especially useful for the treatment of a variety of cellular proliferation disorders frequently associated with cancer, particularly hematologic malignancies such as leukemias. These proteins have shown antiproliferative activity against multiple myeloma, chronic lymphocytic leukemia, low-grade lymphoma, Kaposi's sarcoma, chronic myelogenous leukemia, renal-cell carcinoma, urinary<br><br>
bladder tumors and ovarian cancers (Bonnem, E. M. et al (1984) J. Biol. Response Modifiers 3:580; Oldham, R. K. (1985) Hospital Practice 20:71).<br>
[109]	 IFNα's are useful against various types of viral infections (Finter, N. B. et aL<br>
(1991) Drugs 42(5):749). Interferon-α's have shown activity against human papillomavirus infection, Hepatitis B, and Hepatitis C infections (Finter, N. B. et al., 1991, supra; Kashima, H. et al, (1988) Laryngoscope 98:334; Dusheiko, G. M. et al. (1986) J. Hematology 3 (Supple. 2):S199; Davis, G L et al. (1989) N, England J. Med. 321:1501). The role of interferons and interferon receptors in the pathogenesis of certain autoimmune and inflammatory diseases has also been investigated (Benoit, P. et al. (1993) J. Immunol. 150(3):707). In addition, mterferon-α has been approved for use for the treatment of diseases such as hairy cell leukemia, renal cell carcinoma, basal cell carcinoma, malignant melanoma, AIDS-related Kaposi's sarcoma, multiple myeloma, chronic myelogenous leukemia, non-Hodgkin's lymphoma, laryngeal papillomatosis, mycosis fungoides, condyloma acuminata, chronic hepatitis B, hepatitis C, chronic hepatitis D, and chronic non-α, non-B/C hepatitis.<br>
[110]	Interferons have been implicated in the pathogenesis of various autoimmune<br>
diseases, such as systemic lupus erythematoses, Behcet's disease, and insulin-dependent diabetes mellitus (IDDM, also referred to as type I diabetes). It has been demonstrated in a transgenic mouse model that β cell expression of IFN-α can cause insulitis and IDDM, and IFN-α antagonists (including antibodies) have been proposed for the treatment of IDDM (WO 93/04699, published Mar. 18, 1993). Impaired IFN-γ and IFN-α production has been observed in multiple sclerosis (MS) patients. IFN-α has been detected in the serum of many AIDS patients, and it has been reported that the production of IFN-7 is greatly suppressed in suspensions of mitogen-stimulated mononuclear cells derived from AIDS patients. For a review see, for example. Chapter 16, "The Presence and Possible Pathogenic Role of Interferons in Disease", In: Interferons and other Regulatory Cytolcines, Edward de Maeyer (1988, John Wiley and Sons publishers). Alpha and beta interferons have been used in the treatment of the acute viral disease herpes zoster (T. C, Merigan et al, N. Engl. J. Med. 298, 981-987 (1978); E. Heidemann et al, Onkologie 7,210-212 (1984)), chronic viral infections, e.g. hepatitis C and hepatitis B infections (R. L. Knobler et al., Neurology 34(10): 1273-9 (1984); M. A. Faerkkilae et al.. Act Neurol. Sci, 69, 184-185 (1985)). rIFNα-2a (Roferon®, Roche) is an injection formulation indicated in use for the treatment of hairy cell leukemia and AIDS-related Kaposi's sarcoma. Recombinant IFNα-2b (Intron A®, Schering) has been approved for the treatment of hairy cell leukemia, selected cases of condylomata acuminata, AIDS-related Kaposi's sarcoma, chronic hepatitis C, and chronic hepatitis B infections in certain patients. Compositions of multiple subtypes of IFNα<br><br>
are also used to treat a variety of diseases (Multiferon , Viragen, Inc.). IFNylb (Actimmune®, Intermune Pharmaceuticals, Inc.) is commercially available for the treatment of chronic granulomatous disease and malignant osteopetrosis.<br>
[Ill]	The biologic activities of type I IFNs have been disclosed and are known in the<br>
art, and can be found, for example, in Pfeffer, Semin. Oncol. 24 (suppl 9), S9-63-S9-69 (1997) and U.S. Patent No.: 6,436,391; 6,372,218; 6,270,756; 6,207,145; 6,086,869; 6,036,949 6,013,253; 6,007,805; 5,980,884; 5,958,402; 5,863,530; 5,849,282; 5,846,526; 5,830,456 5,824,300; 5,817,307; 5,780,021; 5,624,895; 5,480,640; 5,268,169; 5,208,019; 5,196,191 5,190,751; 5,104,653; 5,019,382; 4,959,210; which are incorporated by reference herein. A related application is U.S. Patent Application entitled "Modified Human Interferon Polypeptides and Their Uses" published as US 2005/0220762 on October 6, 2005, which is mcorporated by reference herein.<br>
[112]	 IFNα's are members of the diverse helical-bundle superfamily of cytokine genes<br>
(Sprang, S. R. et al. (1993) Curr. Opin. Struct. Biol. 3:815-827). The human interferon α's are encoded by a family of over 20 tandemly duplicated nonallelic genes that share 85-98% sequence identity at the amino acid level (Hence, K. et al. (1985) J. Mol. Biol. 185:227-260). Human IFNβ is a regulatory polypeptide with a molecular weight of about 22 kDa consisting of 166 amino acid residues. It can be produced by most cells in the body, in particular fibroblasts, in response to viral infection or exposure to other agents. It binds to a multimeric cell surface receptor, and productive receptor binding results m a cascade of intracellular events leading to the expression of IFNβ inducible genes which in tum produces effects which can be classified as anti-viral, anti-βroliferative and immunomodulatory.<br>
[113]	The amino acid sequence of human IFNβ is known and was reported for example<br>
by Taniguchi, Gene 10:11-15, 1980, and in EP 83069, EP 41313 and U.S. Pat. No. 4,686,191 which are incorporated by reference herein. Crystal structures have been reported for human and murine IFNβ, respectively (Proc. Natl. Acad. Sci. USA 94:11813-11818, 1997; J. Mol. Biol. 253:187-207, 1995; U.S. Patent No.: 5,602,232; 5,460,956; 5,441,734; 4,672,108; which are incorporated by reference herein). They have been reviewed in Cell Mol. Life Sci. 54:1203-1206, 1998. Variants of IFNβ have been reported (WO 95/25170, U.S. Patent No. 6,572,853, U.S. Pat No, 5,545,723, U.S. Pat No. 4,914,033, EP 260350, U.S. Pat No. 4,588,585, U.S. Pat No. 4,769,233, Stewart et al, DNA Vol. 6 no. 2 1987 pp. 119-128, Runkel et al, 1998, J. Biol. Chem. 273, No. 14, pp. 8003-8008, which are incorporated by reference herein). Expression of IFNβ in CHO cells has been reported (U.S. Pat No. 4,966,843, U.S. Pat No. 5,376,567 and U.S. Pat. No. 5,795,779, which are incorporated by reference herein).    IFNβ molecules with a<br><br>
particular glycosylation pattern and methods for their preparation have been reported (EP 287075 and EP 529300).<br>
[114]	Commercial preparations of IFNβ are sold under the names Betaseron® (also<br>
termed interferon β1b, which is non-glycosylated, produced using recombinant bacterial cells, has a deletion of the N-terminal methionine residue and the C17S mutation), and Avonex® and Rebif® (also termed interferon βqa, which is glycosylated, produced using recombinant mammalian cells) for treatment of patients with multiple sclerosis, have shown to be effective in reducing the exacerbation rate, and more patients remain exacerbation-free for prolonged periods of time as compared with placebo-treated patients. Furthermore, the accumulation rate of disability is reduced (Neurol. 51:682-689, 1998).<br>
[115]	Comparison of IFNβ la and pi b with respect to structure and function has been<br>
presented in Pharmaceut. Res. 15:641-649, 1998. IFNβ has been shown to delay the progression of multiple sclerosis, a relapsing then progressive inflammatory degenerative diseeise of the central nervous system. IFNβ may have inhibitory effects on the proliferation of leukocytes and antigen presentation. IFNβ may modulate the profile of cytokine production towards an antiinflammatory phenotype. IFNβ can reduce T-cell migration by inhibiting the activity of T-cell matrix metalloproteases. These activities are likely to act in concert to account for the mechanism of IFNβ in MS (Neurol. 51:682-689, 1998).<br>
[116]	IFNβ may be used for the treatment of osteosarcoma, basal cell carcinoma,<br>
cervical dysplasia, glioma, acute myeloid leukemia, multiple myeloma, Hodgkin's disease, breast carcinoma, melanoma, and viral infections such as papilloma virus, viral hepatitis, herpes genitalis, herpes zoster, herpetic keratitis, herpes simplex, viral encephalitis, cytomegalovirus pneumonia, and rhinovirus. Various side effects are associated with the use of current preparations of IFNβ, including injection site reactions, fever, chills, myalgias, arthralgias, and other flu-like symptoms (Clin. Therapeutics, 19:883-893,1997).<br>
[117]	Given the multitude of side effects with current IFNβ products, their association<br>
with frequent injection, the risk of developing neutralizing antibodies impeding the desired therapeutic effect of IFNβ, and the potential for obtaining more optimal therapeutic IFNβ levels with concomitant enhanced therapeutic effect, there is clearly a need for improved IFNβ-like molecules.<br>
[118]	Additional side effects found in patients receiving interferon therapy include flu-<br>
like symptoms such as fatigue, headache, and fever, anorexia, myelosuppression, and neutropenia.   Side effects such as these can require cessation of treatment or a reduction of<br><br>
dosage. See Jonasch, E. Oncologist 2001; 6:34-55, which is incorporated by reference. IFNα2a therapy has other associated toxicities including thrombocytopenia. Among the most common side effects of IFNα current therapy are pyrexia, headache, rigors, and myalgia. Side effects that lead to dose reductions are most commonly hematological consequences of bone marrow suppression, including anemia and neutropenia. Adverse effects of IFNs are problematic even with IFN/ribavirin combination therapy; the combined use of a PEGylated interferona CPEGASYS® or PEG-Intron®) and Ribavirin (Copegus is the current standard of care in HCV therapy. Moreover, patients with HCV genotype 1 (la or lb) have been shown to be much less lilcely to respond to combination therapy than HCV genotype 2 or 3. Also, these combination regimens are found to be remarkably less effective in patients with a high viral load. Thus, there is clearly a need for improved IFN-like therapeutics.<br>
[119]	Other interferons (IFN-є, IFN-K, IFN-Δ, IFN-Τ) and four interferon-like cytokines<br>
(limitin, IL-28A, IL-28B, IL-29) have been described. The interaction of interferons and<br>
interferon-like molecules with receptor molecules as well as their biological activities and<br>
clinical applications are described by Pestka, S. et al. in "Interferons, interferon-like cytokines,<br>
and their receptors," Immunol Rev. 2004 Dec;202:8-32, which is incorporated by reference<br>
herein. Type I receptor chains IFNαRI and IFNαR2 are utilized by a number of the interferons<br>
including IFN-α, IFN-β, and limitin. The gene encoding sequence of IFNαRI was originally<br>
cloned by Uze et al. Cell 1990; 60:225, which is incorporated by reference herein. It is a 110<br>
kDa protein, whereas IFNαR2 occurs in two different forms. The two forms of IFNαR2 are<br>
from the same gene and are generated as differentially spliced products (Lutfalla et al. EMBO J<br>
1995; 14:5100; Domanski et al. J. Biol. Chem. 1995; 270:21606; and Novick et al. Cell 1994;<br>
77:391, which are incorporated by reference herein). The short form ( IFNαR2b) has a<br>
molecular mass of 51 kDa and the long form ( IFNαR2c) is 90-100 kDa. Thus, two forms of the<br>
receptor complex thus exist, with IFNαRI associating with IFNαR2b or IFNαR2c. Colamonici<br>
et al. J. Biol. Chem. 1994; 269:5660, which is incorporated by reference herein, have shown that<br>
both types of receptor transduce signals and mediate the biological effects of interferons. IFNs<br>
utilize the Jak-Stat signed transduction pathway in addition to other pathways.<br>
[120]	Limitin (Interferon-zeta) has been found only in mice and was isolated bfised on<br>
its ability to inhibit the proliferation of a myelomonocytic leukemia cell line (Oritani, K. et al. Nature Medicine 2000 6(6):659-666, which is incorporated herein by reference). Limitin (SEQ ID NO: 23) has been shown to kill or arrest the proliferation of certain lympho-hematopoietic cell lines (B lymphopoiesis), but showed little influence on erythropoiesis or myelopoiesis.<br><br>
Sequence analysis of the 182 amino acid protein has shown homology to IFN-α and IFN-β (31.9% and 25.9% identical in 166 overlapping amino acids) and a hydrophobic set of residues at the amino terminus. Kawamoto et al. J Virol. 2003 Sep;77(17):9622-31, which is incorporated by reference, describe studies of the antiviral activity of limitin with encephalomyocarditis virus (EMCV) and herpes simplex virus (HSV) infected cells, and plaque formation in mouse hepatitis virus (MHV) infected cells.<br>
[121]	Differences between limitin and interferons such as IFNα have been discussed,<br>
including, but not limited to, differences in signal transduction pathways for antiviral effects, in<br>
the antiviral effects themselves, and in myelosuppressive effects. See Kawamoto et al. J Virol.<br>
2003 Sep;77(17):9622-31. Kawamoto et al. Experimental Hematology 2004; 32:797-805, which<br>
is incorporated by reference herein, describes differences found between limitin and IFN-α in<br>
assays measuring antiviral, immunomodulatory, antitumor, and myelosuppressive activity and in<br>
an in vivo study. Limitin was found to separate anti-viral activity from bone marrow toxicity.<br>
[122]	Thirteen different alpha interferons transmit distinct signals through a single<br>
paired α/β two chain receptor complex. Modulating of components involved in one or more signaling pathways mediated through the same receptor complex may provide optimization of the antiviral effects of hlFN polypeptides, and modulation of toxic side effects. A number of molecules are involved with downstream signaling pathways. Antisense oligonucleotides to CrkL and Crkll that inhibited protein expression of these molecules were used by Platanias et al. and were found to reverse the inhibition of IFNα or IFNy on proliferation of bone marrow cells (CFU-GM and BFU-E). IFNα is known to activate the STATl pathway, and knockout mouse studies point to the importance of the STATl signaling cascade in fighting viral infections. Durbin et al. (Cell. 1996 Feb 9;84(3):443-50) have shown that mice homozygous for a STATl knockout develop spontaneous viral infections unless they axe raised m a sterile environment. In addition, signaling through STAT5 and CrkL ultimately lead to activation of the RAPl protein which may be necessary for producing growth-inhibitory signals. The phosphorylation of STAT3 may also be measured,<br>
[123]	Various references disclose modification of polypeptides by polymer conjugation<br>
or glycosylation. The term "hlFN polypeptide" includes polypeptides conjugated to a polymer such as PEG and may be comprised of one or more additional derivitizations of cysteine, lysine, or other residues. In addition, the hlFN polypeptide may comprise a linker or polymer, wherein the amino acid to which the linker or polymer is conjugated may be a non-natural amino acid according to the present mvention, or may be conjugated to a naturally encoded amino acid utilizing techniques known in the art such as coupling to lysine or cysteine.<br><br>
[l24] Polymer  modification of native IFNβ or a C17S variant thereof has been<br>
reported (EP 229108, U.S. Pat, No. 5,382,657; EP 593868; U.S. Pat No. 4,917,888 and WO<br>
99/55377, which are incorporated by reference herein). U.S. Pat. No. 4,904,584 discloses<br>
PEGylated lysine depleted polypeptides, wherein at least one lysine residue has been deleted or<br>
replaced with any other amino acid residue. WO 99/67291 discloses a process for conjugating a<br>
protein with PEG, wherein at least one amino acid residue on the protein is deleted and the<br>
protein is contacted with PEG under conditions sufficient to achieve conjugation to the protein.<br>
WO 99/03887 discloses PEGylated variants of polypeptides belonging to the growth hormone<br>
superfamily, wherein a cysteine residue has been substituted with a non-essential amino acid<br>
residue located in a specified region of the polypeptide. Examples of PEGylated IFN molecules<br>
include those disclosed in U.S. Patent No.: 6,524,570; 6,250,469; 6,180,096; 6,177,074;<br>
6,042,822; 5,981,709; 5,951,974; 5,908,621; 5,738,846; 5,711,944; 5,382,657, which are<br>
incorporated by reference herein. IFNβ is mentioned as one example of a polypeptide belonging<br>
to the growth hormone superfamily. WO 00/23114 discloses glycosylated and pegylated IFNβ.<br>
WO 00/23472 discloses IFNβ fusion proteins. WO 00/26354 discloses a method of producing a<br>
glycosylated polypeptide variant with reduced allergenicity, which as compared to a<br>
corresponding parent polypeptide comprises at least one additional glycosylation site. IFNβ is<br>
disclosed as one example among many polypeptides that can be modified according to the<br>
technology described in U.S. Pat. No. 5,218,092, which is incorporated by reference herein.<br>
U.S. Pat. No. 5,218,092, which is incorporated by reference herein, discloses modification of<br>
granulocyte colony stimulating factor (G-CSF) and other polypeptides so as to introduce at least<br>
one additional carbohydrate chain as compared to the native polypeptide.<br>
[125]	The term "hlFN polypeptide" also includes glycosylated hlFN, such as but not<br>
limited to, polypeptides glycosylated at any amino acid position, N-linked or O-linked glycosylated forms of the polypeptide. These forms included, but are not limited to, a polypeptide with an O-lined glycosylation site at position 129 of SEQ ID NO: L or the equivalent position of SEQ ID NO: 2 or 3, or any other IFN polypeptide (Adolf et al., Biochem. J. 276:511 (1991)).<br>
[126]	Variants containing single nucleotide changes are also considered as biologically<br>
active variants of hlFN polypeptide. In addition, splice variants are also included. The term * "hlFN polypeptide" also includes hlFN polypeptide heterodimers, homodimers, heteromultimers, or homomultimers of any one or more hlFN polyeptides or any other polypeptide, protein, carbohydrate, polymer, small molecule, linker, ligand, or other biologically active molecule of any type, linked by chemical means or expressed as a fusion protein, as well<br><br>
as polypeptide analogous containinf for example, specinc deletions or otner modifications yet maintain biological activity.<br>
[127]	All references to amino acid positions in hlFN described herein are based on the<br>
position in SEQ ID NO: 2, unless otherwise specified (i.e., when it is stated that the comparison is based on SEQ ID NO: 1, 3, or other hlFN sequence or interferon-Iike cytokine such as limitin). Those of skill in the art will appreciate that amino acid positions corresponding to positions in SEQ ID NO: I, 2, 3, or any other EPN sequence or interferon-like cytokine such as limitin can be readily identified in any other hlFN molecule or interferon-like cytokine such as limitin such as hlFN fusions, variants, fragments, etc. For example, sequence alignment programs such as BLAST can be used to align and identify a particular position in a protein that corresponds with a position in SEQ ID NO: 1, 2, 3, or other IFN sequence or interferon-like cytokine such as limitin. Substitutions, deletions or additions of amino acids described herein in reference to SEQ ID NO: 1, 2, 3, or other IFN sequence are intended to also refer to substitutions, deletions or additions in corresponding positions in hlFN fusions, variants, fragments, interferon-like cytokines such as limitin, etc. described herein or known in the art and are expressly encompassed by the present invention.<br>
[128]	The  term  "hlFN  polypeptide"  or "hlFN"  encompasses  hlFN  polypeptides<br>
comprising one or more amino acid substitutions, additions or deletions. hlFN polypeptides of the present invention may be comprised of modifications with one or more natural amino acids in conjunction with one or more non-natural amino acid modification. For example, a hlFN polypeptide may comprise a non-naturally encoded amino acid substitution as well as a natural amino acid substitution for the first amino acid at the N-terminus. Exemplary substitutions in a wide variety of amino acid positions in naturallγ-occurring hlFN polypeptides have been described, including but not limited to substitutions that modulate one or more of the biological activities of the hlFN polypeptide, such as but not limited to, increase agonist activity, increase solubility of the polypeptide, decrease protease susceptibility, convert the polypeptide into an antagonist, etc. and are encompassed by the term " hlFN polypeptide.<br>
[129]	Human IFN antagonists include, but are not limited to, those with substitutions<br>
at: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165, or any combination thereof (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN sequence); a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on the site selected and the desired activity. Human IFN antagonists include, but are not limited to, those with substitutions at 22,23, 24,25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77,<br><br>
78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94, 124,125. 127, 128, 129,131, 132, 133, 134, 135, 136, 137, or any combination thereof (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, hlFN antagonists comprise at least one substitution in the regions 1-9 (N-terminus), 10-21 (A helix), 22-39 (region between A helix and B helix), 40-75 (B helix), 76-77 (region between B helix and C helix), 78-100 (C helix), 101-110 (region between C helix and D helix), 111-132 (D helix), 133-136 (region between D and E helix), 137-155 (E helix), 156-165 (C-terminus) that cause IFN to act as an antagonist. In other embodiments, the exemplary sites of incorporation of a non-naturally encoded amino acid include residues within the amino terminal region of helix A and a portion of helix C. In other embodiments, the above-listed substitutions are combined with additional substitutions that cause the hlFN polypeptide to be a hlFN antagonist. In some embodiments, the hlFN antagonist comprises a non-naturally encoded amino acid linked to a water soluble polymer that is present in a receptor binding region of the hlFN molecule.<br>
[130]	In some embodiments, the hlFN polypeptides further comprise an addition,<br>
substitution or deletion that modulates biological activity of the hlFN polypeptide. For example, the additions, substitutions or deletions may modulate one or more properties or activities of hlFN. For example, the additions, substitutions or deletions may modulate affinity for the hlFN polypeptide receptor, modulate (including but not limited to, increases or decreases) receptor dimerization, modulate downstream signaling events post hlFN receptor binding, stabilize receptor dimers, modulate circulating half-life, modulate therapeutic half-life, modulate stability of the polypeptide, modulate cleavage by proteases, modulate dose, modulate release or bioavailability, modulate one or more side effects found with current IFN therapeutics, modulate toxicity, facilitate purification, or improve or alter a particular route of administration. Similarly, hlFN polypeptides may comprise protease cleavage sequences, reactive groups, antibodγ-binding domains (including but not limited to, FLAG or polγ-His) or other affinity based sequences (including but not limited to, FLAG, polγ-His, GST, etc.) or linked molecules (including but not limited to, biotin) that improve detection (including but not limited to, GFP), purification or other traits of the polypeptide.<br>
[131]	The term "hlFN polypeptide" also encompasses homodimers, heterodimers,<br>
homomultimers, and heteromultimers that are linked, including but not limited to those linked directly via non-naturally encoded amino acid side chains, either to the same or different non-naturally encoded amino acid side chains, to naturallγ-encoded amino acid side chains, or indirectly via a linker.    Exemplary linkers including but are not limited to, small organic<br><br>
compounds, water soluble polymers of a variety of lengths such as poly(ethyIene glycol) or polydextran, or polypeptides of various lengths.<br>
[132]	A "non-naturally encoded amino acid" refers to an amino acid that is not one of<br>
the 20 common amino acids or pyrrolysine or selenocysteine. Other terms that may be used synonymously vrith the term "non-naturally encoded amino acid " are "non-natural amino acid/' "unnatural amino acid," "non-naturallγ-occurring amino acid," and variously hyphenated and non-hyphenated versions thereof The term "non-naturally encoded amino acid" also includes, but is not limited to, amino acids that occur by modification (e.g. post-translational modifications) of a naturally encoded amino acid (including but not limited to, the 20 common amino acids or pyrrolysine and selenocysteine) but are not themselves naturally incorporated into a growing polypeptide chain by the translation complex. Examples of such non-naturallγ-occurring amino acids include, but are not limited to, A^-αcetylglucosaminyl-L-serine, A'^-acetylglucosaminyl-L-threonine, and 0-βhosphotyrosine.<br>
[133]	An "amino terminus modification group" refers to any molecule that can be<br>
attached to the amino terminus of a polypeptide. Similarly, a "carboxy terminus modification group" refers to any molecule that can be attached to the carboxy terminus of a polypeptide. Terminus modification groups include, but are not limited to, various water soluble polymers, peptides or proteins such as serum albumin, or other moieties that increase serum half-life of peptides.<br>
[134]	The terms "functional group", "active moiety", "activating group", "leaving<br>
group", "reactive site", "chemically reactive group" and "chemically reactive moiety" are used<br>
in the art and herein to refer to distinct, definable portions or units of a molecule. The terms are<br>
somewhat synonymous in the chemical arts and are used herein to indicate the portions of<br>
molecules that perform some function or activity and are reactive with other molecules.<br>
[135]	The term "linlcage" or "linker" is used herein to refer to groups or bonds that<br>
normally are formed as the result of a chemical reaction and typically are covalent linkages. Hydrolytically stable linkages means that the linkages are substantially stable in water and do not react with water at useful pH values, including but not limited to, under physiological conditions for an extended period of time, perhaps even indefinitely. Hydrolytically unstable or degradable linlcages mean that the linkages are degradable in water or in aqueous solutions, including for example, blood. Enzymatically unstable or degradable linkages mean that the linkage can be degraded by one or more enzymes. As understood in the art, PEG and related polymers may include degradable linlcages in the polymer backbone or in the linker group between the polymer backbone and one or more of the terminal functional groups of the polymer<br><br>
molecule. For example, ester linkages formed by the reaction of PEG carboxylic acids or<br>
activated PEG carboxylic acids with alcohol groups on a biologically active agent generally<br>
hydrolyze under physiological conditions to release the agent. Other hydrolytically degradable<br>
linlcages include, but are not limited to, carbonate linkages; imine linkages resulted from<br>
reaction of an amine and an aldehyde; phosphate ester linlcages formed by reacting an alcohol<br>
with a phosphate group; hydrazone linkages which are reaction product of a hydrazide and an<br>
aldehyde; acetal linlcages that are the reaction product of an aldehyde and an alcohol; orthoester<br>
linkages that are the reaction product of a formate and an alcohol; peptide linkages formed by an<br>
amine group, including but not limited to, at an end of a polymer such as PEG, and a carboxyl<br>
group of a peptide; and oligonucleotide linkages formed by a phosphoramidite group, including<br>
but not limited to, at the end of a polymer, and a 5' hydroxy! group of an oligonucleotide.<br>
[136]	The  term  "biologically  active  molecule",  "biologically  active  moiety"   or<br>
"biologically active agent" when used herein means any substance which can affect any physical<br>
or biochemical properties of a biological system, pathway, molecule, or interaction relating to an<br>
organism, including but not limited to, viruses, bacteria, bacteriophage, transposon, prion,<br>
insects, fungi, plants, animals, and humans. In particular, as used herein, biologically active<br>
molecules include, but are not limited to, any substance intended for diagnosis, cure, mitigation,<br>
treatment, or prevention of disease in humans or other animals, or to otherwise enhance physical<br>
or mental well-benig of humans or animals. Examples of biologically active molecules include,<br>
but are not limited to, peptides, proteins, enzymes, small molecxile drugs, hard drugs, soft drugs,<br>
carbohydrates, inorganic atoms or molecules, dyes, lipids, nucleosides, radionuclides,<br>
oligonucleotides, toxins, cells, viruses, liposomes, microparticles and micelles. Classes of<br>
biologically active agents that are suitable for use with the invention include, but are not limited<br>
to, drugs, prodrugs, radionuclides, imaging agents, polymers, antibiotics, fungicides, anti-viral<br>
agents, anti-inflammatory agents, anti-tumor agents, cardiovascular agents, anti-αnxiety agents,<br>
hormones, growth factors, steroidal agents, microbially derived toxins, and the like.<br>
[137]	A "bifunctional polymer" refers to a polymer comprising two discrete functional<br>
groups that are capable of reacting specifically with other moieties (including but not limited to, amino acid side groups) to form covalent or non-covalent linkages, A bifunctional linker having one functional group reactive with a group on a particular biologically active component, and another group reactive with a group on a second biological component, may be used to form a conjugate that includes the first biologically active component, the bifunctional linker and the second biologically active component. Many procedures and linker molecules for attachment of various compounds to peptides are known. See, e.g., European Patent Application No. 188,256;<br><br>
U.S. Patent Nos. 4,671,958, 4,659,839, 4,414,148, 4,699,784; 4,680,338; and 4,569,789 which<br>
are incorporated by reference herein. A "multi-functional polymer" refers to a polymer<br>
comprising two or more discrete functional groups that are capable of reacting specifically with<br>
other moieties (including but not limited to, amino acid side groups) to form covalent or non-<br>
covalent linlcages. A bi-functional polymer or multi-functional polymer may be any desired<br>
length or molecular weight, and may be selected to provide a particular desired spacing or<br>
conformation between one or more molecules linked to hlFN and its receptor or hlFN.<br>
[138]	Where substituent groups are specified by their conventional chemical formulas,<br>
written from left to right, they equally encompass the chemically identical substituents that would result from writing the structure from right to left, for example, the structure -CH2O- is equivalent to the structure -OCH2-.<br>
[139]	The   term   "substituents"   includes   but   is   not   limited   to   "non-interfering<br>
substituents". 'Non-interfering substituents" are those groups that yield stable compounds. Suitable non-interfering substituents or radicals include, but are not limited to, halo, C1-C10 alkyl, C2-C10 alkenyl, C2-C10 allcynyl, C1-C10 alkoxy, C1-C12 aralkyl, C1-C12 alkaryl, C3-C12 cycloalkyl, C3-C12 cycloalkenyl, phenyl, substituted phenyl, toluoyl, xylenyl, biphenyl, C2-C12 alkoxyalkyl, C2-C12 alkoxyaryl, C7-C12 aryloxyalkyl, C7-C12 oxyaryl, C1-C6 alkylsulfmyl, C1-C10 alkylsulfonyl, —(CH2)m —O-(C1-C10 alkyl) wherein m is from 1 to 8, aryi, substituted aryl, substituted alkoxy, fluoroalkyl, heterocyclic radical, substituted heterocyclic radical, nitroalkyl, --NO2, -CN, -NRC(O)-(C1-C10 alkyl), -C(O)-(C1-C10 alkyl), C2-C10 alkyl thioalkyl, -C(O)O--( C1-C10 alkyl), -OH, -SO2, =S, -COOH, -NR2, carbonyl, -.C(O)-(C1-C10 alkyl)-CF3, -C(O)—CF3, -C(O)NR2, -(C1-C10 aryI)-S-(C6-C10 aryl), ,C(O)-(CrC,o aryl), -(CH2)m -O-(-(CH2)m-O-(C1-C10 alkyl) wherein each m is from 1 to 8, -C(O)NR2, "C(S)NR2, - SO2NR2. -NRC(O) NR2, -NRC(S) NR2, salts thereof, and the like. Each R as used herein is H, allcyl or substituted allcyl, aryl or substituted aryl, aralkyl, or alkaryl.<br>
[140]	The term "halogen" includes fluorine, chlorine, iodine, and bromine.<br>
[141]	The term "alkyl," by itself or as part of another substituent, means, unless<br>
otherwise stated, a straight or branched chain, or cyclic hydrocarbon radical, or combination thereof, which may be fully saturated, mono- or polyunsaturated and can include di- and multivalent radicals, having the number of carbon atoms designated (i,e. C1-C10 means one to ten carbons). Examples of saturated hydrocarbon radicals include, but are not limited to, groups such as methyl, ethyl, n-βropyl, isopropyl, n-butyl, t-butyl, isobutyl, sec-butyl, cyclohexyl, (cyclohexyl)methyl, cyclopropylmethyl, homologs and isomers of, for example, n-βentyl, n-hexyl, n-heptyl, n-octyl, and the like.  An unsaturated alkyl group is one havmg one or more<br><br>
double bonds of triple. Examples of unsaturated alkyl groups include, but are not limited to, vinyl, 2-βropenyl, crotyl, 2-isopentenyl, 2-(butadienyl), 2,4-pentadienyl, 3-(l,4-pentadienyl), ethynyl, 1- and 3-propynyl, 3-butynyl, and the higher homologs and isomers. The term "alkyl," unless otherwise noted, is also meant to include those derivatives of alkyl defined in more detail below, such as "heteroalkyl." Alkyl groups which are limited to hydrocarbon groups are termed "homoalkyl".<br>
[142]	The term "allcylene" by itself or as part of another substituent means a divalent<br>
radical derived from an alkane, as exemplified, but not limited, by the structures -CH2CH2- and -CH2CH2CH2CH2-, and further includes those groups described below as "heteroalkylene." Typically, an alkyl (or alkylene) group will have from 1 to 24 carbon atoms, with those groups having 10 or fewer carbon atoms being a particular embodiment of the methods and compositions described herein. A "lower ailcyl" or "lower alkylene" is a shorter chain alkyl or alkylene group, generally having eight or fewer carbon atoms.<br>
[143]	The terms "alkoxy," "alkylamino" and "alkylthio" (or thioalkoxy) are used in<br>
their conventional sense, and refer to those alkyl groups attached to the remainder of the<br>
molecule via an oxygen atom, an amino group, or a sulfur atom, respectively.<br>
[144]	The term "heteroalkyl," by itself or in combination with another term, means,<br>
unless otherwise stated, a stable straight or branched chain, or cyclic hydrocarbon radical, or combinations thereof, consisting of the stated number of carbon atoms and at least one heteroatom selected from the group consisting of O, N, Si and S, and wherein the nitrogen and sulfur atoms may optionally be oxidized and the nitrogen heteroatom may optionally be quatemized. The heteroatom(s) O, N and S and Si may be placed at any interior position of the heteroalkyl group or at the position at which the alkyl group is attached to the remainder of the molecule. Examples include, but are not limited to, -CH2-CH2-O-CH3, -CH2-CH2-NH-CH3, -CH2-CH2-N(CH3)-CH3, -CH2-S-CH2-CH3, -CH2-CH2,-S(O)-CH3, -CH2-CH2-S(O)2-CH3, -CH=CH-O-CH3, "Si(CH3)3, -CH2-CH=N-OCH3, and -CH=CH-N(CH3)-CH3. Up to two heteroatoms may be consecutive, such as, for example, -CH2-NH-OCH3 and -CH2-O-Si(CH3)3. Similarly, the term "heteroalkylene" by itself or as part of another substituent means a divalent radical derived firom heteroalkyl, as exemplified, but not limited by, -CH2-CH2-S-CH2-CH2- and -CH2-S-CH2-CH2-NH-CH2-. For heteroalkylene groups, the same or different heteroatoms can also occupy either or both of the chain termini (including but not limited to, alkyleneoxy, alkylenedioxy, alkyleneamino, alkylenediamino, aminooxyalkylene, and the like). Still further, for alkylene and heteroalkylene linking groups, no orientation of the linking group is implied by<br><br>
the direction in which the formula of the linking group is written. For example, the formula -C(O)2R'" represents both-C(O)2R'- and -R'C(O)2".<br>
[145]	The terms "cycloallcyl" and "heterocycloalkyl", by themselves or in combination<br>
with other terms, represent, unless otherwise stated, cyclic versions of "alkyl" and "heteroalkyl", respectively. Thus, a cycloalkyl or heterocycloalkyl include saturated, partially unsaturated and fully unsaturated ring linkages. Additionally, for heterocycloalkyl, a heteroatom can occupy the position at which the heterocycle is attached to the remainder of the molecule. Examples of cycloalkyl include, but are not limited to, cyclopentyl, cyclohexyl, 1-cyclohcxenyl, 3-cyclohexenyl, cycloheptyl, and the like. Examples of heterocycloalkyl include, but are not limited to, l-(l,2,5,6-tetrahydropyridyI), 1-piperidinyl, 2-piperidinyl, 3-piperidinyl, 4-morpholinyl, 3-morpholinyI, tetrahydrofuran-2-γl, tetrahydrofuran-3-γl, letrahydrothien-2-γl, tetrahydrothien-3-γl, 1-piperazinyl, 2-piperazinyl, and the like. Additionally, the term encompasses bicyclic and tricyclic ring structures. Similarly, the term "heterocycloalkylene" by itself or as part of another substituent means a divalent radical derived from heterocycloalkyl, and the term "cycloallcylene" by itself or as part of another substituent means a divalent radical derived from cycloalkyl.<br>
[146]	As used herein, the term "water soluble polymer" refers to any polymer that is<br>
soluble in aqueous solvents. Linkage of water soluble polymers to hlFN polypeptides can result in changes including, but not limited to, increased or modulated serum half-life, or increased or modulated therapeutic half-life relative to the unmodified form, modulated immunogenicity, modulated physical association characteristics such as aggregation and multimer formation, altered receptor binding, altered receptor dimerization or multimerization, modulated toxicity, and modulation of one or more the biological activities of IFN including side effects found with current IFN therapeutics. The water soluble polymer may or may not have its own biological activity, and may be utilized as a linker for attaching hlFN to other substances, including but not limited to one or more hlFN polypeptides, or one or more biologically active molecules. Suitable polymers include, but are not limited to, polyethylene glycol, polyethylene glycol propionaldehyde, mono C1-C10 alkoxy or aryloxy derivatives thereof (described in U.S. Patent No. 5,252,714 which is incorporated by reference herein), monomethoxγ-βolyethylene glycol, polyvinyl pyrrolidone, polyvinyl alcohol, polyamino acids, divinylether maleic anhydride, N-(2-Hydroxypropyl)-methacrylamide, dextran, dextran derivatives including dextran sulfate, polypropylene glycol, polypropylene oxide/ethylene oxide copolymer, polyoxyethylated polyol, heparin, heparin fragments, polysaccharides, oligosaccharides, glycans, cellulose and cellulose derivatives, including but not limited to methylcellulose and carboxymethyl cellulose, starch and<br><br>
starch derivatives, polypeptides, polyalkylene glycol and derivatives thereof, copolymers of<br>
polyalkylene glycols and derivatives thereof, polyvinyl ethyl ethers, and alphα-betα-βoly[(2-<br>
hydroxyethyl)-DL-αspartamide, and the like, or mixtures thereof.   Examples of such water<br>
soluble polymers include, but are not limited to, polyethylene glycol and serum albumin.<br>
[147]	As used herein, the term "polyalkylene glycol" or "poly(alkene glycol)" refers to<br>
polyethylene glycol (poly(ethylene glycol)), polypropylene glycol, polybutylene glycol, and derivatives thereof The term "polyalkylene glycol" encompasses both Imear and branched polymers and average molecular weights of between 0.1 kDa and 100 kDa. Other exemplary embodiments are listed, for example, in commercial supplier catalogs, such as Shearwater Corporation's catalog "Polyethylene Glycol and Derivatives for Biomedical Applications" (2001).<br>
[148]	The term "aryl" means, unless otherwise stated, a polyunsaturated, aromatic,<br>
hydrocarbon substituent which can be a single ring or multiple rings (including but not limited to<br>
from 1 to 3 rings) which are fused together or linked covalently. The term "heteroaryl" refers to<br>
aryl groups (or rings) that contain from one to four heteroatoms selected from N, O, and S,<br>
wherein the nitrogen and sulfur atoms are optionally oxidized, and the nitrogen atom(s) are<br>
optionally quatemized. A heteroaryl group can be attached to the remainder of the molecule<br>
through a heteroatom. Non-limiting examples of aryl and heteroaryl groups include phenyl, 1-<br>
naphthyl, 2-naphthyl, 4-biphenyl, 1-pyrrolyl, 2-pyrrolyl, 3-pyrrolyl, 3-pyrazolyl, 2-imidazolyl,<br>
4-imidazolyl, pyrazinyl, 2-oxazolyl, 4-oxazolyl, 2-βhenyl-4-oxazolyl, 5-oxazolyl, 3-isoxazolyl,<br>
4-isoxazolyl, 5-isoxazolyl, 2-thiazolyl, 4-thiazolyl, 5-thiazolyl, 2-furyl, 3-furyl, 2-thienyl, 3-<br>
thienyl, 2-pyridyl, 3-pyridyl, 4-pyridyl, 2-pyrimiidyl, 4-pyrimidyl, 5-benzothiazolyl, purinyl, 2-<br>
benzimidazolyl, 5-indolyl, 1-isoquinolyl, S-isoquinolyl, 2-quinoxalinyl, 5-quinoxalinyl, 3-<br>
quinolyl, and 6-quinolyl. Substituents for each of the above noted aryl and heteroaryl ring<br>
systems are selected from the group of acceptable substituents described below.<br>
[149]	For brevity, the term "aryl" when used in combination with other terms<br>
(including but not limited to, aryloxy, arylthioxy, arylalkyl) includes both aryl and heteroaryl rings as defined above. Thus, the term "arylalkyl" is meant to include those radicals in which an aryl group is attached to an alkyl group (including but not limited to, benzyl, phenethyl, pyridylmethyl and the like) including those alkyl groups in which a carbon atom (including but not limited to, a methylene group) has been replaced by, for example, an oxygen atom (including but not limited to, phenoxymethyl, 2-pyridyloxymethyl, 3-(l-naphthyloxy)propyl, and the like).<br><br>
[150]	Each of the above terms (including but not limited to, "alkyl" "heteroalkyl,"<br>
"aryl" and "heteroaryl") are meant to include both substituted and unsubstituted forms of the<br>
indicated radical. Exemplary substituents for each type of radical are provided below.<br>
[151]	Substituents for the alkyl and heteroalkyl radicals (including those groups often<br>
referred to as alkylene, alkenyl, heteroalkylene, heteroalkenyl, alkynyl, cycloalkyl, heterocycloalkyl, cycloalkenyl, and heterocycloalkenyl) can be one or more of a variety of groups selected from, but not limited to: -OR', =0, =NR', =N-OR\ -NR'R", -SR', -halogen, -SiR'R"R»', -OC(O)R', -C(O)R', -CO2R', -CONR'R", -OC(O)NR'R' -NR"C(O)R" -NR'-C(O)NR"R"', -NR"C(O)2R' -NR-C(NR'R"R"')=NR"", -NR-CCNR'R")=NR"" -S(O)R', -S(O)2R', -S(O)2NR'R", -NRSO2R', -CN and -NO2 in a number ranging from zero to (2m'+l), where m' is the total number of carbon atoms in such a radical. R', R", R'" and R"" each independently refer to hydrogen, substituted or unsubstituted heteroalkyl, substituted or unsubstituted aryl, including but not limited to, aryl substituted with 1-3 halogens, substituted or unsubstituted allcyl, alkoxy or thioalkoxy groups, or arylalkyl groups. When a compound of the invention includes more than one R group, for example, each of the R groups is independently selected as are each R', R", R'" and R"" groups when more than one of these groups is present. When R' and R" are attached to the same nitrogen atom, they can be combined with the nitrogen atom to form a 5-, 6-, or 7-membered ring. For example, -NR'R" is meant to include, but not be limited to, l-pyrrolidinyl and 4-morpholinyl. From the above discussion of substituents, one of skill in the art will understand that the term "alkyl" is meant to include groups including carbon atoms bound to groups other than hydrogen groups, such as haloalkyl (including but not limited to, -CF3 and -CH2CF3) and acyl (including but not limited to, -C(O)CH3, -C(O)CF3, -C(O)CH2OCH3, and the like).<br>
[152]	Similar to the substituents described for the alkyl radical, substituents for the aryl<br>
and heteroaryl groups are varied and are selected from, but are not limited to: halogen, -OR', =O, =NR', =N-OR', -NR'R", -SR', -halogen, -SiR'R"R"', -OC(O)R', -C(O)R', -CO2R', -CONR'R", -OC(O)NR'R", -NR"C(O)R', -NR'-C(O)NR"R"', -NR"C(O)2R', -NR-C(NR'R"R'")=NR"", -NR-C(NR'R")=NR'", -S(O)R', -S(O)2R', -S(O)2NR'R", -NRSO2R', -CN and -NO2, -R', -N3, -CH(Ph)2, fluoro(C1-C4)alkoxy, and fluoro(1i-C4)alkyl, in a number ranging from zero to the total number of open valences on the aromatic ring system; and where R', R", R"' and R"" are independently selected from hydrogen, alkyl, heteroalkyl, aryl and heteroaryl. When a compound of the invention includes more than one R group, for example, each of the R groups is independently selected as are each R', R", R'" and R"" groups when more than one of these groups is present.<br><br>
[153]	AS used herein, the term "modulated serum half-life" means the positive or<br>
negative change in circulating half-life of a modified hlFN relative to its non-modified form.<br>
Serum half-life is measured by taking blood samples at various time points after administration<br>
of hlFN, and determining the concentration of that molecule in each sample. Correlation of the<br>
serum concentration with time allows calculation of the serum half-life, Increased serum half-<br>
life desirably has at least about two-fold, but a smaller increase may be useful, for example<br>
where it enables a satisfactory dosing regimen or avoids a toxic effect. In some embodiments,<br>
the increase is at least about three-fold, at least about five-fold, or at least about ten-fold.<br>
[154]	The term "modulated therapeutic half-life" as used herein means the positive or<br>
negative change in the half-life of the therapeutically effective amount of hlFN, relative to its non-modified form. Therapeutic half-life is measured by measuring pharmacokinetic and/or pharmacodynamic properties of the molecule at various time points after administration. Increased therapeutic half-life desirably enables a particular beneficial dosing regimen, a particular beneficial total dose, or avoids an undesired effect. In some embodiments, the increased therapeutic half-life results from increased potency, increased or decreased binding of the modified molecule to its target, increased or decreased breakdown on the molecule by enzynies such as proteases, or an increase or decrease in another parameter or mechanism of action of the non-modified molecule.<br>
[155]	The term "isolated," when applied to a nucleic acid or protein, denotes that the<br>
nucleic acid or protein is free of at least some of the cellular components with which it is associated in the natural state, or that the nucleic acid or protein has been concentrated to a level greater than the concentration of its in vivo or in vitro production. It can be in a homogeneous state. Isolated substances can be in either a dry or semi-dry state, or in solution, including but not limited to, an aqueous solution. It can be a component of a pharmaceutical composition that comprises additional pharmaceutically acceptable carriers and/or excipients. Pxirity and homogeneity are typically determined using analytical chemistry techniques such as polyacrylamide gel electrophoresis or high performance liquid chromatography. A protein which is the predominant species present in a preparation is substantially purified. In particular, an isolated gene is separated from open reading frames which flank the gene and encode a protein other than the gene of interest. The term "purified" denotes that a nucleic acid or protein gives rise to substantially one band in an electrophoretic gel. Particularly, it may mean that the nucleic acid or protein is at least 85% pure, at least 90% pure, at least 95% pure, at least 99% or greater pure.<br><br>
[156]	The term "nucleic acid" refers to deoxyribonucleotides, deoxyribonucleosides,<br>
ribonucleosides, or ribonucleotides and polymers thereof in either single- or double-stranded form. Unless specifically limited, the term encompasses nucleic acids containing known analogues of natural nucleotides which have similar binding properties as the reference nucleic acid and are metabolized in a manner similar to naturally occurring nucleotides. Unless specifically limited otherwise, the term also refers to oligonucleotide analogs including PNA (peptidonucleic acid), analogs of DNA used in antisense technology (phosphorothioates,<br>
phosphoroamidates, and the like). Unless otherwise indicated, a particular nucleic acid sequence<br>
ft<br>
also implicitly encompasses conservatively modified variants thereof (including but not limited to, degenerate codon substitutions) and complementary sequences as well as the sequence explicitly indicated. Specifically, degenerate codon substitutions may be achieved by generating sequences in which the third position of one or more selected (or all) codons is substituted with mixed-base and/or deoxyinosine residues (Batzer et aL, Nucleic Acid Res. 19:5081 (1991); Ohtsuka et aL, 1 Biol Chem, 260:2605-2608 (1985); Rossolini ei al, Mol Cell Probes 8:91-98 (1994)).<br>
[157]	The terms "polypeptide," "peptide" and "protein" are used interchangeably herein<br>
to refer to a polymer of amino acid residues. That is, a description directed to a polypeptide applies equally to a description of a peptide and a description of a protein, and vice versa. The terms apply to naturally occurring amino acid polymers as well as amino acid polymers in which one or more amino acid residues is a non-naturally encoded amino acid. As used herein, the terms encompass amino acid chains of any length, including full length proteins, wherein the amino acid residues are linked by covalent peptide bonds.<br>
[158]	The term "amino acid" refers to naturally occurring and non-naturally occurring<br>
amino acids, as well as amino acid analogs and amino acid mimetics that function in a manner similar to the naturally occurring amino acids. Naturally encoded amino acids are the 20 common amino acids (alanine, arginine, asparagine, aspartic acid, cysteine, glutamine, glutamic acid, glycine, histidine, isoleucine, leucine, lysine, methionine, phenylalanine, proline, serine, threonine, tryptophan, tyrosine, and valine) and pyrrolysine and selenocysteine. Amino acid analogs refers to compounds that have the same basic chemical structure as a naturally occurring amino acid, i.e., an a carbon that is bound to a hydrogen, a carboxyl group, an amino group, and an R group, such as, homoserine, norleucine, methionine sulfoxide, methionine methyl sulfonium. Such analogs have modified R groups (such as, norleucine) or modified peptide backbones, but retain the same basic chemical structure as a naturally occurring amino acid.<br><br>
[159]	Amino acids may be referred to herein by either their commonly known three<br>
letter symbols or by the one-letter symbols recommended by the lUPAC-IUB Biochemical Nomenclature Commission. Nucleotides, likewise, may be referred to by their commonly accepted single-letter codes.<br>
[160]	"Conservatively modified variants" applies to both amino acid and nucleic acid<br>
sequences. With respect to particular nucleic acid sequences, "conservatively modified variants" refers to those nucleic acids which encode identical or essentially identical amino acid sequences, or where the nucleic acid does not encode an amino acid sequence, to essentially identical sequences. Because of the degeneracy of the genetic code, a large number of functionally identical nucleic acids encode any given protein. For instance, the codons GCA, GCC, GCG and GCU all encode the amino acid alanine. Thus, at every position where an alanine is specified by a codon, the codon can be altered to any of the corresponding codons described without altering the encoded polypeptide. Such nucleic acid variations are "silent variations," which are one species of conservatively modified variations. Every nucleic acid sequence herein which encodes a polypeptide also describes every possible silent variation of the nucleic acid. One of ordinary skill in the art will recognize that each codon in a nucleic acid (except AUG, which is ordinarily the only codon for methionine, and TGG, which is ordinarily the only codon for tryptophan) can be modified to yield a functionally identical molecule. Accordingly, each silent variation of a nucleic acid which encodes a polypeptide is implicit in each described sequence.<br>
[161]	As to amino acid sequences, one of ordinary skill in the art will recognize that<br>
individual substitutions, deletions or additions to a nucleic acid, peptide, polypeptide, or protein sequence which alters, adds or deletes a single amino acid or a small percentage of amino acids in the encoded sequence is a "conservatively modified variant" where the alteration results in the deletion of an amino acid, addition of an amino acid, or substitution of an amino acid with a chemically similar amino acid. Conservative substitution tables providing functionally similar amino acids are known to those of ordinarj' skill in the art. Such conser\'atively modified variants are in addition to and do not exclude polymorphic variants, interspecies homologs, and alleles of the invention.<br>
[162]	Conservative substitution tables providing ftmctionally similar amino acids are<br>
known to those of ordinary skill in the art. The following eight groups each contain amino acids that are conservative substitutions for one another:<br>
1)	Alanine (A), Glycine (G);<br>
2)	Aspartic acid (D), Glutamic acid (E);<br><br>
3)	Asparagme (N), Glutamme (Q);<br>
4)	Arginine (R), Lysine (K);<br>
5)	Isoleucine (I), Leucine (L), Methionine (M), Valme (V);<br>
6)	Phenylalanine (F), Tyrosine (Y), Tryptophan (W);<br>
7)	Serine (S), Threonine (T); and<br>
8)	Cysteine (C), Methionine (M)<br>
(see, e.g., Creighton, Proteins: Structures and Molecular Properties (W H Freeman &amp; Co.; 2nd edition (December 1993)<br>
[163]	The terms "identical" or percent "identity," in the context of two or more nucleic<br>
acids or polypeptide sequences, refer to two or more sequences or subsequences that are the same. Sequences are "substantially identical" if they have a percentage of amino acid residues or nucleotides that are the same {i.e., about 60% identity, about 65%, about 70%, about 75%, about 80%, about 85%, about 90%, or about 95% identity over a specified region), when compared and aligned for maximum correspondence over a comparison window, or designated region as measured using one of the following sequence comparison algorithms (or other algorithms available to persons of ordinary skill in the art) or by manual alignment and visual inspection. This definition also refers to the complement of a test sequence. The identity can exist over a region that is at least about 50 amino acids or nucleotides in length, or over a region that is 75-100 amino acids or nucleotides in length, or, where not specified, across the entire sequence of a polynucleotide or polypeptide.<br>
[164]	For sequence comparison, typically one sequence acts as a reference sequence, to<br>
which test sequences are compared. When using a sequence comparison algorithm, test and reference sequences are entered into a computer, subsequence coordinates are designated, if necessary, and sequence algorithm program parameters are designated. Default program parameters can be used, or alternative parameters can be designated. The sequence comparison algorithm then calculates the percent sequence identities for the test sequences relative to the reference sequence, based on the program parameters.<br>
[165]	A "comparison window", as used herein, includes reference to a segment of any<br>
one of the number of contiguous positions selected from the group consisting of from 20 to 600, usually about 50 to about 200, more usually about 100 to about 150 in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned. Methods of alignment of sequences for comparison are known to those of ordinary skill in the art. Optimal alignment of sequences for comparison can be conducted, including but not limited to, by the local homology algorithm of Smith and<br><br>
 of Needleman and Wunsch (1970) J. Mol Biol 48:443, by the search for similarity method of Pearson and Lipman (1988) Proc. Natl Acad Sci, USA 85:2444, by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, WI), or by manual alignment and visual inspection (see^ e.g., Ausubel et aL, Current Protocols in Molecular BioIog)&gt; (1995 supplement)).<br>
[166]	One example of an algorithm that is suitable for determining percent sequence<br>
identity and sequence similarity are the BLAST and BLAST 2.0 algorithms, which are described in Altschul et al (1997) Nuc. Acids Res. 25:3389-3402, and Altschul ef al (1990) J. Mol Biol 215:403-410, respectively. Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information available at the World Wide Web at ncbi.nlm.nih.gov. The BLAST algorithm parameters W, T, and X determine the sensitivity and speed of the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a wordlength (W) of 11, an expectation (E) or 10, M=5, N=-4 and a comparison of both strands. For amino acid sequences, the BLASTP program uses as defaults a wordlength of 3, and expectation (E) of 10, and the BLOSUM62 scoring matrix {see Henikoff and Henikoff (1992) Proc. Natl Acad Sci USA 89:10915) alignments (B) of 50, expectation (E) of 10, M=5, N=-4, and a comparison of both strands. The BLAST algorithm is typically performed with the "low-complexity" filter turned off.<br>
[167]	The BLAST algorithm also performs a statistical analysis of the similarity<br>
between two sequences (see, e.g., Karlin and Altschul (1993) Proc. Natl Acad. ScL USA 90:5873-5787). One measure of similarity provided by the BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar to a reference sequence if the smallest sum probability in a comparison of the test nucleic acid to the reference nucleic acid is less than about 0.2, or less than about 0.01, or less than about 0.001.<br>
[168]	The phrase "selectively (or specifically) hybridizes to" refers to the binding,<br>
duplexing, or hybridizing of a molecule only to a particular nucleotide sequence imder stringent hybridization conditions when that sequence is present in a complex mixture (including but not limited to, total cellular or library DNA or RNA).<br>
[169]	The  phrase  "stringent  hybridization   conditions"  refers  to  hybridization   of<br>
sequences of DNA, RNA, PNA, or other nucleic acid mimics, or combinations thereof under<br><br>
conditions of low ionic strength and high temperature as is known in the art. Typically, under stringent conditions a probe will hybridize to its target subsequence in a complex mixture of nucleic acid (includmg but not limited to, total cellular or library DNA or RNA) but does not hybridize to other sequences in the complex mixture. Stringent conditions are sequence-dependent and will be different in different circumstances. Longer sequences hybridize specifically at higher temperatures. An extensive guide to the hybridization of nucleic acids is found in Tijssen, Laboratory Techniques in Biochemistry and Molecular Biology—Hybridization with Nucleic Probes, "Overview of principles of hybridization and the strategy of nucleic acid assays" (1993). Generally, stringent conditions are selected to be about 5-10° C lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength pH. The T^ is the temperature (under defined ionic strength, pH, and nucleic concentration) at which 50% of the probes complementary to the target hybridize to the target sequence at equilibrium (as the target sequences are present in excess, at Tm, 50% of the probes are occupied at equilibrium). Stringent conditions may be those in which the salt concentration is less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion concentration (or other salts) at pH 7.0 to 8.3 and the temperature is at least about 30°C for short probes (including but not limited to, 10 to 50 nucleotides) and at least about 60° C for long probes (including but not limited to, greater than 50 nucleotides). Stringent conditions may also be achieved with the addition of destabilizing agents such as formamide. For selective or specific hybridization, a positive signal may be at least two times background, optionally 10 times background hybridization. Exemplary stringent hybridization conditions can be as following: 50% formamide, 5X SSC, and 1% SDS, incubating at 42°C, or 5X SSC, 1% SDS, incubating at 65°C, with wash in 0.2X SSC, and 0.1% SDS at 65°C. Such washes can be performed for 5, 15, 30, 60, 120, or more minutes.<br>
[170]	As used herein, the term "eukaryote" refers to organisms belonging to the<br>
phylogenetic domain Eucarya such as animals (including but not limited to, mammals, insects, reptiles, birds, etc.), ciliates, plants (including but not limited to, monocots, dicots, algae, etc.), fungi, yeasts, flagellates, microsporidia, protists, etc.<br>
[171]	As used herein, the term "non-eukaryote" refers to non-eukaryotic organisms.<br>
For example, a non-eukaryotic organism can belong to the Eubacteria (including but not limited to, Escherichia coli, Thermus thermophilus, Bacillus stearothermophilus, Pseudomonas fluorescens, Pseudomonas aeruginosa, Pseudomonas putida, etc.) phylogenetic domain, or the Archaea (including but not limited to, Methanococcus jannaschii, Methanobacterium thermoautotrophicum, Halobacterium such as Haloferax volcanii and Halobacterium species<br><br>
NRC'l, Archaeoglobus fulgidus, Pyrococcus furiosus, Pyrococcus horikoshii, Aeuropyrum pernix, etc.) phylogenetic domain.<br>
[172]	The term "subject" as used herein, refers to an animal, in some embodiments a<br>
mammal, and in other embodiments a human, who is the object of treatment, observation or<br>
experiment.<br>
[173]	The term "effective amount" as used herein refers to that amount of the modified<br>
non-natural amino acid polypeptide being administered which will relieve to some extent one or<br>
more of the symptoms of the disease, condition or disorder being treated.    Compositions<br>
containing  the   modified  non-natural   amino   acid  polypeptide   described   herein   can   be<br>
administered for prophylactic, enhancing, and/or therapeutic treatments.<br>
[174]	The terms "enhance" or "enhancing" means to increase or prolong either in<br>
potency or duration a desired effect.   Thus, in regard to enhancing the effect of therapeutic<br>
agents, the term "enhancing" refers to the ability to increase or prolong, either in potency or<br>
duration, the effect of other therapeutic agents on a system. An "enhancing-effective amount,"<br>
as used herein, refers to an amount adequate to enhance the effect of another therapeutic agent in<br>
a desired system.   When used in a patient, amounts effective for this use will depend on the<br>
severity and course of the disease, disorder or condition, previous therapy, the patient's health<br>
status and response to the drugs, and the judgment of the treating physician.<br>
[175]	The term "modified," as used herein refers to any changes made to a given<br>
polypeptide, such as changes to the length of the polypeptide, the amino acid sequence, chemical<br>
structure, co-translational modification, or post-translational modification of a polypeptide. The<br>
form "(modified)" term means that the polypeptides being discussed are optionally modified,<br>
that is, the polypeptides under discussion can be modified or unmodified.<br>
[176]	The term "post-translationally modified" refers to any modification of a natural<br>
or non-natural amino acid that occurs to such an amino acid after it has been incorporated into a<br>
polypeptide chain. The term encompasses, by way of example only, co-translational in vivo<br>
modifications, co-translational in vitro modifications (such as in a cell-free translation system),<br>
post-translational in vivo modifications, and post-translational in vitro modifications.<br>
[177]	In prophylactic applications, compositions containing the modified non-natural<br>
amino acid polypeptide are administered to a patient susceptible to or othervn.se at risk of a<br>
particular disease, disorder or condition.  Such an amount is defined to be a "prophylactically<br>
effective amoimt."  In this use, the precise amounts also depend on the patient's state of health,<br>
weight, and the like.  It is considered well within the skill of the art for one to determine such<br><br>
 effective amounts of routine experimentation (e.g., a aose escalation clinical trial).<br>
[178]	The term "protected" refers to the presence of a "protecting group" or moiety that<br>
prevents reaction of the chemically reactive functional group under certain reaction conditions.<br>
The protecting group will vary depending on the type of chemically reactive group being<br>
protected. For example, if the chemically reactive group is an amine or a hydrazide, the<br>
protecting group can be selected from the group of tert-butyloxycarbonyl (t-Boc) and 9-<br>
fluorenylmethoxycarbonyl (Fmoc). If the chemically reactive group is a thiol, the protecting<br>
group can be orthopyridyldisulfide. If the chemically reactive group is a carboxylic acid, such as<br>
butanoic or propionic acid, or a hydroxyl group, the protecting group can be benzyl or an alkyl<br>
group such as methyl, ethyl, or tert-butyl. Other protecting groups known in the art may also be<br>
used in or with the methods and compositions described herein, including photolabile groups<br>
such as Nvoc and MeNvoc. Other protecting groups known in the art may also be used in or<br>
with the methods and compositions described herein.<br>
[179]	By way of example only, blocldng/protecting groups may be selected from:<br><br>
[180]	Other protecting groups are described in Greene and Wuts, Protective Groups in<br>
Organic Synthesis, 3rd Ed., John Wiley &amp; Sons, New York, NY, 1999, which is incorporated herem by reference in its entirety.<br>
[181]	In therapeutic applications, compositions containing the modified non-natural<br>
amino acid polypeptide are administered to a patient already suffering from a disease, condition or disorder, in an amount sufficient to cure or at least partigilly arrest the symptoms of the disease, disorder or condition.   Such an amount is defined to be a "therapeutically effective<br><br>
amount," and will depend on the severity and course of the disease, disorder or condition, previous therapy, the patient's health status and response to the drugs, and the judgment of the treating physician. It is considered well within the skill of the art for one to determine such therapeutically effective amounts by routine experimentation (e.g., a dose escalation clinical trial).<br>
[182]	The term "treating" is used to refer to either prophylactic and/or therapeutic<br>
treatments.<br>
[183]	Non-naturally encoded amino acid polypeptides presented herein may include<br>
isotopicallγ-labelled compounds with one or more atoms replaced by an atom having an atomic<br>
mass or mass number different from the atomic mass or mass number usually found in nature.<br>
Examples of isotopes that can be incorporated into the present compounds include isotopes of<br>
hydrogen, carbon, nitrogen, oxygen, fluorine and chlorine, such as 2H, 3H, 13C, '14C, 15N, 18O,<br>
17O,35S, 18F, 36Cl, respectively. Certain isotopicallγ-labelled compounds described herein, for<br>
example those into which radioactive isotopes such as 3H and '14C are incorporated, may be<br>
useful in drug and/or substrate tissue distribution assays. Further, substitution with isotopes<br>
such as deuterium, i.e., H, can afford certain therapeutic advantages resulting from greater<br>
metabolic stability, for example increased in vivo half-life or reduced dosage requirements.<br>
[184]	All isomers including but not limited to diastereomers. enantiomers, and mixtures<br>
thereof are considered as part of the compositions described herein. In additional or furtlier embodiments, the non-naturally encoded amino acid polypeptides are metabolized upon administration to an organism in need to produce a metabolite that is then used to produce a desired effect, including a desired therapeutic effect. In further or additional embodiments are active metabolites of non-naturally encoded amino acid polypeptides,<br>
[185]	In some situations, non-naturally encoded amino acid polypeptides may exist as<br>
tautomers. In addition, the non-naturally encoded amino acid polypeptides described herein can exist in unsolvated as well as solvated forms with pharmaceutically acceptable solvents such as water, ethanol, and the like. The solvated forms are also considered to be disclosed herein. Those of ordinary skill in the art will recognize that some of the compounds herein can exist in several tautomeric forms. All such tautomeric forms are considered as part of the compositions described herein.<br>
[186]	Unless otherwise indicated, conventional methods of mass spectroscopy, NMR,<br>
HPLC, protein chemistry, biochemistry, recombinant DNA techniques and pharmacology, v^thin the skill of the art are employed.<br><br>
DETAILED DESCRIPTION<br>
I.	Introduction<br>
[187]	Interferon molecules comprising at least one unnatural amino acid are provided in<br>
the invention. In certain embodiments of the invention, the hlFN polypeptide with at least one unnatural amino acid includes at least one post-translational modification. In one embodiment, the at least one post-translational modification comprises attachment of a molecule including but not limited to, a labels a dye, a polymer, a water-soluble polymer, a derivative of polyethylene glycol, a photocrosslinker, a radionuclide, a cytotoxic compound, a drug, an affinity label, a photoaffinity label, a reactive compound, a resin, a second protein or polypeptide or polypeptide analog, an antibody or antibody firagment, a metal chelator, a cofactor, a fatty acid, a carbohydrate, a polynucleotide, a DNA, a RNA, an antisense polynucleotide, a saccharide, a water-soluble dendrimer, a cyclodextrin, an inhibitory ribonucleic acid, a biomaterial, a nanoparticle, a spin label, a fluorophore, a metal-containing moiety, a radioactive moiety, a novel functional group, a group that covalently or noncovalently interacts with other molecules, a photocaged moiety, an actinic radiation excitable moiety, a photoisomerizable moiety, biotin, a derivative of biotin, a biotin analogue, a moiety incorporating a heavy atom, a chemically cleavable group, a photocleavable group, an elongated side chain, a carbon-linked sugar, a redox-αctive agent, an amino thioacid, a toxic moiety, an isotopically labeled moiety, a biophysical probe, a phosphorescent group, a chemiluminescent group, an electron dense group, a magnetic group, an intercalating group, a chromophore, an energy transfer agent, a biologically active agent, a detectable label, a small molecule, a quantum dot, a nanotransmitter, a radionucleotide, a radiotransmitter, a neutron-capture agent, or any combination of the above or any other desirable compound or substance, comprising a second reactive group to at least one unnatural amino acid comprising a first reactive group utilizing chemistry methodology that is known to one of ordinary skill in the art to be suitable for the particular reactive groups. For example, the first reactive group is an alkynyl moiety (including but not limited to, in the unnatural amino acid p-βropargyloxyphenylalanine, where the propargyl group is also sometimes referred to as an acetylene moiety) and the second reactive group is an azido moiety, and [3+2] cycloaddition chemistry methodologies are utilized. In another example, the first reactive group is the azido moiety (including but not limited to, in the unnatural amino acid p-azido-L-βhenylalanine) and the second reactive group is the alkynyl moiety. In certain embodiments of the modified hlFN polypeptide of the present invention, at least one unnatural<br><br>
amino acid (including but not limited to, unnatural amino acid containing a keto functional group) comprising at least one post-translational modification, is used where the at least one post-translational modification comprises a saccharide moiety. In certain embodiments, the post-translational modification is made in vivo in a eulcaryotic cell or in a non-eukaryotic cell. A linker, polymer, water soluble polymer, or other molecule may attach the molecule to the polypeptide. The molecule may be linked directly to the polypeptide.<br>
[188]	In  certain embodiments, the protein includes  at least  one  post-translational<br>
modification that is made in vivo by one host cell, where the post-translational modification is not normally made by another host cell type. In certain embodiments, the protein includes at least one post-translational modification that is made in vivo by a eukaryotic ceil, where the post-translational modification is not normally made by a non-eukaryotic cell. Examples of post-translational modifications include, but are not limited to, glycosylation, acetylation, acylation, lipid-modification, palmitoylation, palmitate addition, phosphorylation, glycolipid-linkage modification, and the like. In one embodiment, the post-translational modification comprises attachment of an oligosaccharide to an asparagine by a GIcNAc-αsparagine linkage (including but not limited to, where the oligosaccharide comprises (GlcNAc-Man)2-Man-GlcNAc-GlcNAc, and the like). In another embodiment, the post-translational modification comprises attachment of an oligosaccharide (including but not limited to, Gal-GalNAc, Gal-GlcNAc, etc.) to a serine or threonine by a GalNAc-serine, a GalNAc-threonine, a GlcNAc-serine, or a GlcNAc-threonine linkage. In certain embodiments, a protein or polypeptide of the invention can comprise a secretion or localization sequence, an epitope tag, a FLAG tag, a polyhistidine tag, a GST fusion, and/or the like. Examples of secretion signal sequences include, but are not limited to, a prokaryotic secretion signal sequence, a eukaryotic secretion signal sequence, a eukaryotic secretion signal sequence 5'-optimized for bacterial expression, a novel secretion signal sequence, pectate lyase, secretion signal sequence, Omp A secretion signal sequence, and a phage secretion signal sequence. Examples of secretion signal sequences, include, but are not limited to, STII (prokaryotic), Fd GIII and Ml3 (phage), Bgl2 (yeast), and the signal sequence bla derived from a transposon.<br>
[189]	The protein or polypeptide of interest can contain at least one, at least two, at least<br>
three, at least four, at least five, at least six, at least seven, at least eight, at least nine, or ten or more unnatural amino acids. The unnatural amino acids can be the same or different, for example, there can be 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or more different sites in the protein that comprise 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or more different unnatural amino acids.    In certain<br><br>
embodiments, at least one, but fewer than all, of a particular amino acid present m a naturally occurring version of the protein is substituted with an unnatural amino acid.<br>
[190]	The present invention provides methods and compositions based on members of<br>
the GH supergene family, hi particular hlFN, comprising at least one non-naturally encoded amino acid. Introduction of at least one non-naturally encoded amino acid into hlFN can allow for the application of conjugation chemistries that involve specific chemical reactions, including, but not limited to, with one or more non-naturally encoded amino acids while not reacting with the commonly occurring 20 amino acids. In some embodiments, hlFN comprising the non-naturally encoded amino acid is linked to a water soluble polymer, such as polyethylene glycol (PEG), via the side chain of the non-naturally encoded amino acid. This invention provides a highly efficient method for the selective modification of proteins with PEG derivatives, which involves the selective incorporation of non-genetically encoded amino acids, including but not limited to, those amino acids containing functional groups or substituents not found in the 20 naturally incorporated amino acids, including but not limited to a ketone, an azide or acetylene moiety, into proteins in response to a selector codon and the subsequent modification of those amino acids with a suitably reactive PEG derivative. Once incorporated, the amino acid side chains can then be modified by utilizing chemistry methodologies known to those of ordinary skill in the art to be suitable for the particular functional groups or substituents present in the non-naturally encoded amino acid. Known chemistry methodologies of a wide variety are suitable for use in the present invention to incorporate a water soluble polymer hito the protein. Such methodologies include but are not limited to a Huisgen [3+2] cycloaddition reaction (see, e,g,, Padwa, A. in Comprehensive Organic Synthesis. Vol. 4. (1991) Ed. Trost, B. M., Pergamon, Oxford, p. 1069-1109; and, Huisgen, R. in 1,3-Dipolar Cvcloaddition Chemistry, (1984) Ed. Padwa, A., Wiley, New York, p. 1-176) with, including but not limited to, acetylene or azide derivatives, respectively.<br>
[191]	Because the Huisgen [3+2] cycloaddition method involves a cycloaddition rather<br>
than a nucleophilic substitution reaction, proteins can be modified with extremely high selectivity. The reaction can be carried out at room temperature in aqueous conditions with excellent regioselectivity (1,4 &gt; 1.5) by the addition of catalytic amounts of Cu(I) salts to the reaction mixture. See, e.g., Tomoe, et al., (2002) J. Org. Chem. 67:3057-3064; and, Rostovtsev, et al., (2002) Angew. Chem. hit Ed, 41:2596-2599; and WO 03/101972, A molecule that can be added to a protein of the invention through a [3+2] cycloaddition includes virtually any molecule with a suitable functional group or substituent including but not limited to an azido or<br><br>
acerylene aenvative. These molecules can be added to an unnatural amino acid with an acetylene group, including but not limited to, p-propargyloxyphenylalanine, or azido group, including but not limited to p-azido-phenylalanine, respectively.<br>
[192]	The five-membered ring that results from the Huisgen [3+2] cycloaddition is not<br>
generally reversible in reducing environments and is stable against hydrolysis for extended periods in aqueous environments. Consequently, the physical and chemical characteristics of a wide variety of substances can be modified under demanding aqueous conditions with the active PEG derivatives of the present invention. Even more importantly, because the azide and acetylene moieties are specific for one another (and do not, for example, react with any of the 20 common, geneticallγ-encoded amino acids), proteins can be modified in one or more specific sites with extremely high selectivity.<br>
[193]	The invention also provides water soluble and hydroltiically stable derivatives of<br>
PEG derivatives and related hydrophilic polymers having one or more acetylene or azide moieties. The PEG polymer derivatives that contain acetylene moieties are highly selective for coupling with azide moieties that have been introduced selectively into proteins in response to a selector codon. Similarly, PEG polymer derivatives that contain azide moieties are highly selective for coupling with acetylene moieties that have been introduced selectively into proteins in response to a selector codon.<br>
[194]	More specifically, the azide moieties comprise, but are not limited to, alkyl<br>
azides, aryl azides and derivatives of these azides. The derivatives of the alkyl and aryl azides can include other substituents so long as the acetylene-specific reactivity is maintained. The acetylene moieties comprise alkyl and aryl acetylenes and derivatives of each. The derivatives of the alkyl and aryl acetylenes can include other substituents so long as the azide-specific reactivity is maintained.<br>
[195]	The present invention provides conjugates of substances having a wide variety of<br>
functional groups, substituents or moieties, with other substances including but not limited to a label; a dye; a polymer; a water-soluble polymer; a derivative of polyethylene glycol; a photocrosslinlcer; a radionuclide; a cytotoxic compound; a drug; an affinity label; a photoaffinity label; a reactive compound; a resin; a second protein or polypeptide or polypeptide analog; an antibody or antibody fragment; a metal chelator; a cofactor; a fatty acid; a carbohydrate; a polynucleotide; a DNA; a RNA; an antisense polynucleotide; a saccharide; a water-soluble dendrimer; a cyclodextrin; an inhibitory ribonucleic acid; a biomaterial; a nanoparticle; a spin label; a fluorophore, a metal-containing moiety; a radioactive moiety; a novel functional group;<br><br>
a group that covalently or noncovalently interacts with other molecules; a photocaged moiety; an actinic radiation excitable moiety; a photoisomerizable moiety; biotin; a derivative of biotin; a biotin analogue; a moiety incorporating a heavy atom; a chemically cleavable group; a photocleavable group; an elongated side chain; a carbon-linked sugar; a redox-αctive agent; an amino thioacid; a toxic moiety; an isotopically labeled moiety; a biophysical probe; a phosphorescent group; a chemiluminescent group; an electron dense group; a magnetic group; an intercalating group; a chromophore; an energy transfer agent; a biologically active agent; a detectable label; a small molecule; a quantum dot; a nanotransmitter; a radionucleotide; a radiotransmitter; a neutron-capture agent; or any combination of the above, or any other desirable compound or substance. The present invention also includes conjugates of substances having azide or acetylene moieties with PEG polymer derivatives having the corresponding acetylene or azide moieties. For example, a PEG polymer containing an azide moiety can be coupled to a biologically active molecule at a position in the protein that contains a non-genetically encoded amino acid bearing an acetylene functionality. The linkage by which the PEG and the biologically active molecule are coupled includes but is not limited to the Huisgen [3+2] cycloaddition product.<br>
[196]	It is well established in the art that PEG can be used to modify the surfaces of<br>
biomaterials (see, e.g., U.S. Patent 6,610,281; Mehvar, R., J. Pharm Pharm Sci., 3(1):125-136<br>
(2000) which are incorporated by reference herein). The invention also includes biomaterials<br>
comprising a surface having one or more reactive azide or acetylene sites and one or more of the<br>
azide- or acetylene-containing polymers of the mvention coupled to the surface via the Huisgen<br>
[3+2] cycloaddition linkage. Biomaterials and other substances can also be coupled to the azide-<br>
or acetylene-αctivated polymer derivatives through a linkage other than the azide or acetylene<br>
linlcage, such as through a linkage comprising a carboxylic acid, amine, alcohol or thiol moiety,<br>
to leave the azide or acetylene moiety available for subsequent reactions.<br>
[197]	The invention includes a method of synthesizing the azide- and acetylene-<br>
containing polymers of the invention. In the case of the azide-containing PEG derivative, the azide can be bonded directiy to a carbon atom of the polymer. Alternatively, the azide-containing PEG derivative can be prepared by attaching a linking agent that has the azide moiety at one terminus to a conventional activated polymer so that the resulting polymer has the azide moiety at its terminus. In the case of the acetylene-containing PEG derivative, the acetylene can be bonded directly to a carbon atom of the polymer. Alternatively, the acetylene-containing PEG derivative can be prepared by attaching a linking agent that has the acetylene moiety at one<br><br>
lermmus to a conventional activated polymer so that the resulting polymer has the acetylene moiety at its terminus.<br>
[198]	More specifically, in the case of the azide-containing PEG derivative, a water<br>
soluble polymer having at least one active hydroxyl moiety undergoes a reaction to produce a substituted polymer having a more reactive moiety, such as a mesylate, tresylate, tosylate or halogen leaving group, thereon. The preparation and use of PEG derivatives containing sulfonyl acid halides, halogen atoms and other leaving groups are known to those of ordinary skill in the art. The resulting substituted polymer then undergoes a reaction to substitute for the more reactive moiety an azide moiety at the terminus of the polymer. Alternatively, a water soluble polymer having at least one active nucleophilic or electrophilic moiety undergoes a reaction with a linking agent that has an azide at one terminus so that a covalent bond is formed between the PEG polymer and the linking agent and the azide moiety is positioned at the terminus of the polymer. Nucleophilic and electrophilic moieties, including amines, thiols, hydrazides, hydrazines, alcohols, carboxylates, aldehydes, ketones, thioesters and the like, are known to those of ordinary skill in the art.<br>
[199]	More specifically, in the case of the acetylene-containing PEG derivative, a water<br>
soluble polymer having at least one active hydroxyl moiety undergoes a reaction to displace a<br>
halogen or other activated leaving group fi'om a precursor that contains an acetylene moiety.<br>
Alternatively, a water soluble polymer having at least one active nucleophilic or electrophilic<br>
moiety undergoes a reaction with a linking agent that has an acetylene at one terminus so that a<br>
covalent bond is formed between the PEG polymer and the linking agent and the acetylene<br>
moiety is positioned at the terminus of the polymer. The use of halogen moieties, activated<br>
leaving group, nucleophilic and electrophilic moieties in the context of organic synthesis and the<br>
preparation and use of PEG derivatives is well established to practitioners in the art.<br>
[200]	The invention also provides a method for the selective modification of proteins to<br>
add other substances to the modified protein, including but not limited to water soluble polymers such as PEG and PEG derivatives containing an azide or acetylene moiety. The azide- and acetylene-containing PEG derivatives can be used to modify the properties of surfaces and molecules where biocompatibility, stability, solubility and lack of immunogenicity are important, while at the same time providing a more selective means of attaching the PEG derivatives to proteins than was.previously known in the art.<br>
II	Growth Hormone Supergene Family<br><br>
[201]	The following proteins include those encoded by genes of the growtli hormone<br>
(GH) supergene family (Bazan, F., Immunology Today 11: 350-354 (1990); Bazan, J. F. Science<br>
257: 410-413 (1992); Mott, H. R. and Campbell, I. D., Current Opinion in Structural Biology 5:<br>
114-121 (1995); Silvennoinen, 0. and Ihle, J. N., SIGNALLING BY THE HEMATOPOIETIC<br>
CYTOK.INE RECEPTORS (1996)): growth hormone, prolactin, placental lactogen, erythropoietin<br>
(EPO), thrombopoietin (TPO), interleuldn-2 (IL-2), IL-3,11-4, IL-5, IL-6, IL-7, IL-9, IL-10, IL-<br>
11, IL-12 (p35 subunit), IL-13, IL-15, oncostatin M, ciliary neurotrophic factor (CNTF),<br>
leukemia inhibitory factor (LIF), alpha interferon, beta interferon, epsilon interferon, gamma<br>
interferon, omega interferon, tau interferon, granulocyte-colony stimulating factor (G-CSF),<br>
granulocyte-macrophage colony stimulating factor (GM-CSF), macrophage colony stimulating<br>
factor (M-CSF) and cardiotrophin-1 (CT-1) ("the GH supergene family"). It is anticipated that<br>
additional members of this gene family will be identified in the future through gene cloning and<br>
sequencing. Members of the GH supergene family have similar secondary and tertiary<br>
structures, despite the fact that they generally have limited amino acid or DNA sequence<br>
identity. The shared structural features allow new members of the gene family to be readily<br>
identified and the non-natural amino acid methods and compositions described herein similarly<br>
applied. Given the extent of structural homology among the members of the GH supergene<br>
family, non-naturally encoded amino acids may be incorporated into any members of the GH<br>
supergene family using the present invention. Each member of this family of proteins comprises<br>
a four helical bundle. The general structure of family member IFNα-2 is shown in Figure 1.<br>
[202]	Structures of a number of cytokines, including G-CSF (Zink et al,, FEBS Lett.<br>
314:435 (1992); Zinlc et al., Biochemistry 33:8453 (1994); Hill et al., Proc. Natl. Acad. Sci.USA 90:5167 (1993)), GM-CSF (Diederichs, K., et al Science 154: 1779-1782 (1991); Walter et al, 1 Mol Biol 224:1075-1085 (1992)), IL-2 (Bazan, J. F. and McKay, D. B. Science 257: 410-413 (1992)), IL-4 (Redfield et al., Biochemistry 30: 11029-11035 (1991); Powers et al. Science 256:1673-1677 (1992)), and IL-5 (Milbum et al., Nature 363: 172-176 (1993)) have been determined by X-ray diffraction and NMR studies and show striking conservation with the GH structure, despite a lack of significant primary sequence homology. IFN is considered to be a member of this family based upon modeling and other studies (Lee et al., J. Interferon Cytokine Res. 15:341 (1995); Murgolo et al., Proteins 17:62 (1993); Radhakrishnan et al. Structure 4:1453 (1996); Klaus et al., J. Mol. Biol. 274:661 (1997)). EPO is considered to be a member of this family based upon modeling and mutagenesis studies (Boissel et aL., J. Biol Chem. 268: 15983-15993 (1993); Wen et al., J. Biol. Chem. 269: 22839-22846 (1994)). All of the above cytokines and growth factors are now considered to comprise one large gene family.<br><br>
[203]	In addition to sharing similar secondary and tertiary structures, members ot this<br>
family share the property that they must oligomerize cell surface receptors to activate intracellular signaling pathways. Some GH family members, including but not limited to; GH and EPO, bind a single type of receptor and cause it to form homodimers. Other family members, including but not limited to, IL-2, IL-4, and IL-6, bind more than one type of receptor and cause the receptors to form heterodimers or higher order aggregates (Davis et al., (1993), Science 260: 1805-1808; Paonessa et al., (1995), EMBO J. 14: 1942-1951; Mott and Campbell, Current Opinion in Structural Biology 5: 114-121 (1995)). Mutagenesis studies have shown that, like GH, these other cytokines and growth factors contain multiple receptor binding sites, typically two, and bind their cognate receptors sequentially (Mott and Campbell Current Opinion in Structural Biology 5: 114-121 (1995); Matthews et al,, (1996) Proc. Natl Acad. Sci. USA 93: 9471-9476). Like GH, the primary receptor binding sites for these other family members occur primarily in the four alpha helices and the α-B loop. The specific amino acids in the hehcal bundles that participate in receptor binding differ amongst the family members. Most of the cell surface receptors that interact with members of the GH supergene family are structurally related and comprise a second large multi-gene family. See, e,g, U.S. Patent No. 6,608,183, which is incorporated by reference herein.<br>
[204]	A general conclusion reached from mutational studies of various members of the<br>
GH supergene family is that the loops joining the alpha helices generally tend to not be involved in receptor binding. In particular the short B-C loop appears to be non-essential for receptor binding in most, if not all, family members. For this reason, the B-C loop may be substituted with non-naturally encoded amino acids as described herein in members of the GH supergene family. The α-B loop, the C-D loop (and D-E loop of interferon/ IL-10-lilce members of the GH superfamily) may also be substituted with a non-naturally-occurring amino acid. Amino acids proximal to helix A and distal to the final helix also tend not to be involved in receptor binding and also may be sites for introducing non-naturallγ-occurring amino acids. In some embodiments, a non-naturally encoded amino acid is substituted at any position within a loop structure, including but not limited to, the first 1, 2, 3,4, 5, 6, 7, or more amino acids of the α-B, B-C, C-D or D-E loop. In some embodiments, one or more non-naturally encoded amino acids are substituted within the last 1, 2, 3, 4, 5, 6, 7, or more amino acids of the α-B, B-C, C-D or D-E loop.<br>
[205]	Certain members of the GH family, including but not limited to, EPO, IL-2, IL-3,<br>
IL-4, IL-6, G-CSF, GM-CSF, TPO, IL-10, IL-12 p35, IL-13, IL-15 and beta interferon contain N-linked and/or  0-linked sugars.     The glycosylation sites in the proteins occur almost<br><br>
exclusively in the loop regions ana not in the alpna helical bundles.. Because the loop regions generally are not involved in receptor binding and because they are sites for the covalent attachment of sugar groups, they may be useful sites for introducing non-naturallγ-occurring amino acid substitutions into the proteins. Amino acids that comprise the N- and 0-linked glycosylation sites in the proteins may be sites for non-naturallγ-occurring amino acid substitutions because these amino acids are surface-exposed. Therefore, the natural protein can tolerate bulky sugar groups attached to the proteins at these sites and the glycosylation sites tend to be located away from the receptor binding sites.<br>
[206]	Additional members of the GH supergene family are likely to be discovered in the<br>
future. New members of the GH supergene family can be identified through computer-αided<br>
secondary and tertiary structure analyses of the predicted protein sequences, and by selection<br>
tecliniques designed to identify molecules that bind to a particular target. Members of the GH<br>
supergene family typically possess four or five amphipathic helices joined by non-helical amino<br>
acids (the loop regions). The proteins may contain a hydrophobic signal sequence at their N-<br>
terminus to promote secretion from the cell. Such later discovered members of the GH<br>
supergene family also are included within this mvention. A related application is International<br>
Patent Application entitled "Modified Four Helical Bundle Polypeptides and Their Uses"<br>
published as WO 05/074650 on August 18, 2005, which is incorporated by reference herein.<br>
[207]	Thus, the description of the growth hormone supergene family is provided for<br>
illustrative purposes and by way of example only and not as a limit on the scope of the methods, compositions, strategies and techniques described herein. Further, reference to GH and IFN polypeptides in this application is intended to use the generic term as an example of any member of the GH supergene family. Thus, it is understood that the modifications and chemistries described herein with reference to hlFN polypeptides or protein can be equally applied to any member of the GH supergene family, including those specifically listed herein.<br>
///.	General Recombinant Nucleic Acid Methods For Use With The Invention<br>
[208]	In numerous embodiments of the present invention, nucleic acids encoding a<br>
hlFN polypeptide of interest will be isolated, cloned and often altered using recombinant methods. Such embodiments are used, including but not limited to, for protein expression or during the generation of variants, derivatives, expression cassettes, or other sequences derived from a hlFN polypeptide. In some embodiments, the sequences encoding the polypeptides of the invention are operably linked to a heterologous promoter. Isolation of hlFN and production of IFN in host cells are described in, e.g., U.S. Patent Nos. 6,489,144; 6,410,697; 6,159,712;<br><br>
5,955,307;  5,814,485;  5,710,027;  5,595,888;  5,391,713;  5,244,655;  5,196,323;  5,066,786;<br>
4,966,843; 4,894,330; 4,364,863, which are incorporated by reference herein.<br>
[209]	A nucleotide sequence encoding a hlFN polypeptide comprising a non-naturally<br>
encoded amino acid may be synthesized on the basis of the amino acid sequence of the parent polypeptide, including but not limited to, having the amino acid sequence shown in SEQ ID NO: 2 (hlFN), and then changing the nucleotide sequence so as to effect introduction (i.e., incorporation or substitution) or removal (i.e., deletion or substitution) of the relevant amino acid residue(s). The nucleotide sequence may be conveniently modified by site-directed mutagenesis in accordance with conventional methods. Alternatively, the nucleotide sequence may be prepared by chemical synthesis, including but not limited to, by using an oligonucleotide synthesizer, wherein oligonucleotides are designed based on the amino acid sequence of the desired polypeptide, and preferably selecting those codons that are favored in the host cell in which the recombinant polypeptide will be produced. For example, several small oligonucleotides coding for portions of the desired polypeptide may be synthesized and assembled by PCR, ligation or ligation chain reaction. See, e,g., Barany, el al„ Proc, Natl. Acad ScL 88: 189-193 (1991); U.S. Patent 6,521,427 which are incorporated by reference herein.<br>
[210]	This invention utilizes routine techniques in the field of recombinant genetics.<br>
Basic texts disclosing the general methods of use in this invention include Sambrook et al.. Molecular Cloning, A Laboratory Manual (3rd ed. 2001); Kriegler, Gene Transfer and Expression: A Laboratory Manual (1990); and Current Protocols in Molecular Biology (Ausubel et al, eds., 1994)).<br>
[211]	Genera! texts which describe molecular biological techniques include Berger and<br>
Kimmel, Guide to Molecular Cloning Techniques. Methods in Enzvmology volume 152 Academic Press, Inc., San Diego, CA (Berger); Sambrook et al, Molecular Cloning - A Laboratory Manual r2nd Ed.V Vol. 1-3. Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989 ("Sambrook") and Current Protocols in Molecular Biology, F.M. Ausubel et al., eds., Current Protocols, a joint venture between Greene Publishing Associates, Inc. and John Wiley &amp; Sons, Inc., (supplemented through 1999) ("Ausubel")). These texts describe mutagenesis, the use of vectors, promoters and many other relevant topics related to, including but not limited to, the generation of genes or polynucleotides that include selector codons for production of proteins that include unnatural amino acids, orthogonal tRNAs, orthogonal synthetases, and pairs thereof.<br><br>
[212]	Various types of mutagenesis are used in the invention for a variety of purposes,<br>
including but not limited to, to produce novel synthetases or tRNAs, to mutate tRNA molecules, to mutate polynucleotides encoding synthetases, to produce libraries of tRNAs, to produce libraries of syntlietases, to produce selector codons, to insert selector codons that encode unnatural amino acids in a protein or polypeptide of interest. They include but are not limited to site-directed, random point mutagenesis, homologous recombination, DNA shuffling or other recursive mutagenesis methods, chimeric construction, mutagenesis using uracil containing templates, oligonucleotide-directed mutagenesis, phosphorothioate-modified DNA mutagenesis, mutagenesis using gapped duplex DNA or the like, or any combination thereof Additional suitable methods include point mismatch repair, mutagenesis iising repair-deficient host strains, restriction-selection and restriction-βurification, deletion mutagenesis, mutagenesis by total gene synthesis, double-strand brealc repair, and the lilce. Mutagenesis, including but not limited to, involving chimeric constructs, are also included in the present invention. In one embodiment, mutagenesis can be guided by known information of the naturally occurring molecule or altered or mutated naturally occrring molecule, including but not limited to, sequence, sequence comparisons, physical properties, tertiary, or quaternary structure, crystal structure or the like.<br>
[213]	The texts and examples found herein describe these procedures.    Additional<br>
information is found in the following publications and references cited within: Ling et al.. Approaches to DNA mutagenesis: an overview. Anal Biochem. 254(2): 157-178 (1997); Dale et al, Oligonucleotide-directed random mutagenesis using the phosphorothioate method, Methods Mol. Biol. 57:369-374 (1996); Smith, In vitro mutagenesis, Ann. Rev. Genet. 19:423-462 (1985); Botstein &amp; Shortle, Strategies and applications of in vitro mutagenesis. Science 229:1193-1201 (1985); Carter, Site-directed mutagenesis, Biochem. J. 237:1-7 (1986); Kunkel The efficiency of oligonucleotide directed mutagenesis, in Nucleic Acids &amp; Molecular Biology (Eckstein, F. and Lilley, D.M.J, eds., Springer Verlag, Berlin) (1987); Kunkel, Rapid and efficient site-specific mutagenesis Mnthout phenotypic selection, Proc. Natl. Acad. Sci. USA 82:488-492 (1985); Kunlcel et al.. Rapid and efficient site-specific mutagenesis Mnthout phenotypic selection. Methods in Enzvmol. 154, 367-382 (1987); Bass et al., Mutant Trp repressors with new DNα-binding specificities, Science 242:240-245 (1988); Zoller &amp; Smith, Oligojiucleotide-directed mutagenesis using M13-derived vectors: an efficient and general procedure for the production of point mutations in any DNA fragment. Nucleic Acids Res. 10:6487-6500 (1982); Zoller &amp; Smith, Oligonucleotide-directed mutagenesis of DNA fragments cloned into Ml3 vectors, Methods in Enzvmol. 100:468-500 (1983); Zoller &amp; Smith, Oligonucleotide-directed mutagenesis: a simple method using two oligonucleotide primers and<br><br>
a single-stranded UNA template, Methods in Enzymol. 154:329-350 (1987); Taylor et al,. The use of phosphorothioate-modified DNA in restriction enzyme reactions to prepare nicked DMA, Nucl. Acids Res. 13: 8749-8764 (1985); Taylor et al.. The rapid generation of oligonucleotide-directed mutations at high frequency using phosphorothioate-modified DNA, Nucl. Acids Res. 13: 8765-8785(1985); Nakamaye &amp; Eclcstein, Inhibition of restriction endonuclease Ncj I cleavage by phosphorothioate groups and its application to oligonucleotide-directed mutagenesis, Nucl. Acids Res. 14: 9679-9698 (1986); Sayers et al., 5'-3' Exonucleases in phosphorothioate-based oligonucleotide-directed mutagenesis, NucL Acids Res. 16:791-802 (1988); Sayers et al.. Strand specific cleavage of phosphorothioate-containing DNA by reaction with restriction endonucleases in the presence of ethidium bromide, (1988) Nucl. Acids Res. 16: 803-814; Kramer et al., The gapped duplex DNA approach to oligonucleotide-directed mutation construction, Nucl. Acids Res. 12: 9441-9456 (1984); Kramer &amp; Fritz Oligonucleotide-directed construction of mutations via gapped duplex DNA, Methods in Enzvmol. 154:350-367 (1987); Kramer et al.. Improved enzymatic in vitro reactions in the gapped duplex DNA approach to oligonucleotide-directed construction of mutations, Nucl. Acids Res. 16: 7207 (1988); Fritz et al., Oligonucleotide-directed construction of mutations: a gapped duplex DNA procedure without enzymatic reactions in vitro, Nucl. Acids Res. 16: 6987-6999 (1988); Kramer et al., Different base/base mismatches are corrected with different efficiencies by the methyl-directed DNA mismatch-repair system of E. coli, Cell 38:879-887 (1984); Carter et al.. Improved oligonucleotide site-directed mutagenesis using Ml 3 vectors, Nucl. Acids Res. 13: 4431-4443 (1985); Carter, Improved oligonucleotide-directed mutagenesis using MIS vectors. Methods in Enzvmol. 154: 382-403 (1987); Eghtedarzadeh &amp; Henikoff, Use of oligonucleotides to generate large deletions, Nucl. Acids Res, 14: 5115 (1986); Wells et al.. Importance of hydrogen-bond formation in stabilizing the transition state of subtilisin, Phil. Trans. R. Soc. Lond. A 317: 415-423 (1986); Nambiar et al., Total synthesis and cloning of a gene coding for the ribonuclease S protein. Science 223: 1299-1301 (1984); Sakmar and Khorana, Total synthesis and expression of a gene for the alphα-subunit of bovine rod outer segment guanine nucleotide-binding protein (transducin), Nucl. Acids Res. 14: 6361-6372 (1988); Wells et al., Cassette mutagenesis: an efficient method for generation of multiple mutations at defined sites. Gene 34:315-323 (1985); Grundstr5m et al., Oligonucleotide-directed mutagenesis by microscale 'shot-gun' gene synthesis, NucL Acids Res. 13: 3305-3316 (1985); Mandecki, Oligonucleotide-directed double-strand break repair in plasmids of Escherichia coli: a method for site-specific mutagenesis, Proc. Natl. Acad. Sci. USA. 83:7177-7181 (1986); Arnold, Protein engineering for unusual environments. Current Opinion in Biotechnologv 4:450-455 (1993); Sieber, et al., Nature<br><br>
Biotecnnoiogy, 19,456-460-(2000); W.P.C Stemmer, Nature 370, 389-91(1994) and, 1. A. Lorimer, I. Pastan, Nucleic Acids Res. 23, 3067-8 (1995). Additional details on many of the above methods can be found in Methods in Enzvmology Volume 154, which also describes useful controls for trouble-shooting problems with various mutagenesis methods.<br>
[214]	Oligonucleotides, e.g., for use in mutagenesis of the present invention, e.g.,<br>
mutating libraries of synthetases, or altering tRNAs, are typically sjmthesized chemically according to the solid phase phosphoramidite triester method described by Beaucage and Caruthers, Tetrahedron Letts. 22(20):1859-1862, (1981) e.g., using an automated synthesizer, as described in Needham-VanDevanter et al., Nucleic Acids Res., 12:6159-6168 (1984).<br>
[215]	The invention also relates to eulcaryotic host cells, non-eukaryotic host cells, and<br>
organisms for the in vivo incorporation of an unnatural amino acid via orthogonal tRNA/RS pairs. Host cells are genetically engineered (including but not limited to, transformed, transduced or transfected) with the polynucleotides of the invention or constructs which include a polynucleotide of the invention, including but not limited to, a vector of the invention, which can be, for example, a cloning vector or an expression vector. For example, the coding regions for the orthogonal tRNA, the orthogonal tRNA synthetase, and the protein to be derivatized are operably linked to gene expression control elements that are functional in the desired host cell. The vector can be, for example, in the form of a plasmid. a cosmid, a phage, a bacterium, a virus, a naked polynucleotide, or a conjugated polynucleotide. The vectors are introduced into cells and/or microorganisms by standard methods including electroporation (Fromm et al, Proc. Natl. Acad. Sci. USA 82, 5824 (1985)), infection by viral vectors, high velocity ballistic penetration by small particles with the nucleic acid either within the matrix of small beads or particles, or on the surface (Klein et al,. Nature 327, 70-73 (1987)), and/or the like.<br>
[216]	The engineered host cells can be cultured in conventional  nutrient media<br>
modified as appropriate for such activities as, for example, screening steps, activating promoters or selecting transformants. These cells can optionally be cultured into transgenic organisms. Other useful references, including but not limited to for cell isolation and culture (e.g., for subsequent nucleic acid isolation) include Freshney (1994) Culture of Animal Cells, a Manual of Basic Technique, third edition, Wileγ- Liss, New York and the references cited therein; Payne et al (1992) Plant Cell and Tissue Culture in Liquid Svstems John Wiley &amp; Sons, Inc. New York, NY: Gamborg and Phillips (eds.) (1995) Plant Cell, Tissue and Organ Culture: Fundamental Methods Springer Lab Manual, Springer-Verlag (Berlin Heidelberg New York) and Atlas and Parks (eds.) The Handbook of Microbiological Media (1993) CRC Press, Boca Raton, FL.<br><br>
[217]	Several well-known methods of introducing target nucleic acids into cells are<br>
available, any of which can be used in the invention. These include: fusion of the recipient cells with bacterial protoplasts contaming the DNA, electroporation, projectile bombardment, and infection with viral vectors (discussed further, below), etc. Bacterial cells can be used to amplify the number of plasmids containing DNA constructs of this invention. The bacteria are grown to log phase and the plasmids within the bacteria can be isolated by a variety of methods known in the art (.see, for instance, Sambrook). In addition, kits are commercially available for the purification of plasmids from bacteria, (see, e.g., EasyPrepTM, FlexiPrepTM, both from Pharmacia Biotech; StrataCleanTM from Stratagene; and, QIAprep'TMfrom Qiagen). The isolated and purified plasmids are then further manipulated to produce other plasmids, used to transfect cells or incorporated into related vectors to infect organisms. Typical vectors contain transcription and translation terminators, transcription and translation initiation sequences, and promoters useful for regulation of the expression of the particular target nucleic acid. The vectors optionally comprise generic expression cassettes containing at least one independent terminator sequence, sequences permitting replication of the cassette in eukaryotes, or prokaryotes, or both, (including but not limited to, shuttle vectors) and selection markers for both prokaryotic and eukaryotic systems. Vectors are suitable for replication and integration in prokaryotes, eulcaryotes, or both. See, Gillam &amp; Smith, Gene 8:81 (1979); Roberts, et al, Nature. 328:731 (1987); Schneider, E., et ai. Protein Expr. Purif 6(1):10-14 (1995); Ausubel, Sambrook, Berger (all supra), A catalogue of bacteria and bacteriophages useful for cloning is provided, e.g., by the ATCC, e.g.. The ATCC Catalogue of Bacteria and Bacteriophage (1992) Gherna et al (eds) published by the ATCC. Additional basic procedures for sequencing, cloning and other aspects of molecular biology and underlying theoretical considerations are also found in Watson et al (1992) Recombinant DNA Second Edition Scientific American Books, NY. In addition, essentially any nucleic acid (and virtually any labeled nucleic acid, whether standard or non-standard) can be custom or standard ordered from any of a variety of commercial sources, such as the Midland Certified Reagent Company (Midland, TX available on the World Wide Web at mcrc.com), The Great American Gene Company (Ramona, CA available on the World Wide Web at genco.com), ExpressGen Inc. (Chicago, IL available on the World Wide Web at expressgen.com), Operon Technologies Inc. (Alameda, CA) and many others.<br>
SELECTOR CODONS<br>
[218]	Selector codons of the invention expand the genetic codon framework of protein<br>
biosynthetic machinery. For example, a selector codon includes, but is not limited to, a unique<br><br>
three base codon, a nonsense codon, such as a stop codon, including but not limited to, an amber codon (UAG), an ochre codon, or an opal codon (UGA), an unnatural codon, a four or more base codon, a rare codon, or the like. It is readily apparent to those of ordinary skill in the art that there is a wide range in the number of selector codons that can be introduced into a desired gene or polynucleotide, including but not limited to, one or more, two or more, three or more, 4, 5, 6, 7, 8, 9, 10 or more in a single polynucleotide encoding at least a portion of the hlFN polypeptide.<br>
[219]	In one embodiment, the methods involve the use of a selector codon that is a stop<br>
codon for the incorporation of one or more unnatural amino acids in vivo. For example, an 0-tRNA is produced that recognizes the stop codon, including but not limited to, UAG, and is aminoacylated by an O-RS with a desired unnatural amino acid. This O-tRNA is not recognized by the naturally occurring host's aminoacyl-tRNA synthetases. Conventional site-directed mutagenesis can be used to introduce the stop codon, including but not limited to, TAG, at the site of interest in a polypeptide of interest See, e.g., Sayers, J.R., et al. (1988), 5'-3' Exonucleases in phosphorothioate-based oligonucleotide directed mutagenesis. Nucleic Acids Res. 16:791-802. When the 0-RS, 0-tRNA and the nucleic acid that encodes the polypeptide of interest are combined in vivo, the unnatural amino acid is incorporated in response to the UAG codon to give a polypeptide containing the unnatural amino acid at the specified position.<br>
[220]	The incorporation of unnatural amino acids in vivo can be done without<br>
significant perturbation of the eukaryotic host cell. For example, because the suppression efficiency for the UAG codon depends upon the competition between the 0-tRNA, including but not limited to, the amber suppressor tRNA, and a eukaryotic release factor (including but not limited to, eRF) (which binds to a stop codon and initiates release of the growing peptide from the ribosome), the suppression efficiency can be modulated by, mcluding but not limited to, increasing the expression level of 0-tRNA, and/or the suppressor tRNA.<br>
[221]	Unnatural amino acids can also be encoded with rare codons. For example, when<br>
the arginine concentration in an in vitro protein synthesis reaction is reduced, the rare arginine codon, AGG, has proven to be efficient for insertion of Ala by a synthetic tRNA acylated with alanine. See, e.g., Ma et al., Biochemistry. 32:7939 (1993). In this case, the synthetic tRNA competes with the naturally occurring tRNAArg, which exists as a minor species in Escherichia coli. Some organisms do not use all triplet codons. An unassigned codon AGA in Micrococcus luteus has been utilized for insertion of amino acids in an in vitro transcription/translation<br><br>
extract. See, e.g., Kowal and Oliver, Nucl. Acid. Res.. 25: 4685(1994), components ot me present invention can be generated to use these rare codons in vivo.<br>
[222]	Selector codons also comprise extended codons, including but not limited to, four<br>
or more base codons, such as, four, five, six or more base codons. Examples of four base codons include, but are not limited to, AGGA, CUAG, UAGA, CCCU and the like. Examples of five base codons include, but are not limited to, AGGAC, CCCCU, CCCUC, CUAGA, CUACU, UAGGC and the like. A feature of the invention includes using extended codons based on frameshift suppression. Four or more base codons can insert, including but not limited to, one or multiple unnatural amino acids into the same protein. For example, in the presence of mutated 0-tRNAs, including but not limited to, a special frameshift suppressor tRNAs, with anticodon loops, for example, with at least 8-10 nt anticodon loops, the four or more base codon is read as single amino acid. In other embodiments, the anticodon loops can decode, including but not limited to, at least a four-base codon, at least a five-base codon, or at least a six-base codon or more. Since there are 256 possible four-base codons, multiple unnatural amino acids can be encoded in the same cell using a four or more base codon. See, Anderson et al., (2002) Exploring the Limits of Codon and Anticodon Size, Chemistry and Biology. 9:237-244; Maghery, (2001) Expanding the Genetic Code: Selection of Efficient Suppressors of Four-base Codons and Identification of "Shifty" Four-base Codons with a Library Approach in Escherichia coli, J. Mol. Biol. 307: 755-769.<br>
[223]	For example, four-base codons have been used to incorporate unnatural amino<br>
acids into proteins using in vitro biosynthetic methods. See, e.g., Ma et al., (1993) Biochemistry, 32:7939; and Hohsaka et al., (1999) J. Am. Chem. Soc, 121:34. CGGG and AGGU were used to simultaneously incorporate 2-naphthylalanine and an NBD derivative of lysine into streptavidin in vitro with two chemically acylated fi-αmeshift suppressor tRNAs. See, e.g.. Hohsaka et aL, (1999) J, Am. Chem. Soc. 121:12194. In an in vivo study, Moore et al. examined the ability of tRNALeu derivatives with NCUA anticodons to suppress UAGN codons (N can be U, A, G, or C), and found that the quadruplet UAGA can be decoded by a tRNALeu With, a UCUA anticodon with an efficiency of 13 to 26% with little decoding in the 0 or -1 frame. See. Moore et al.. (2000) J. Mol. Biol.. 298:195. In one embodiment, extended codons based on rare codons or nonsense codons can be used in the present invention, which can reduce missense readthrough and frameshift suppression at other unwanted sites.<br>
[224]	For a given system, a selector codon can also include one of the natural three<br>
base codons, where the endogenous system does not use (or rarely uses) the natural base codon.<br><br>
For example, this includes a system that is lacking a tRNA that recognizes the natural three base codon, and/or a system where the three base codon is a rare codon.<br>
[225]	Selector codons optionally include unnatural base pairs.   These unnatural base<br>
pairs further expand the existing genetic alphabet. One extra base pair increases the number of triplet codons from 64 to 125. Properties of third base pairs include stable and selective base pairing, efficient enzymatic incorporation into DNA with high fidelity by a polymerase, and the efficient continued primer extension after synthesis of the nascent unnatural base pair. Descriptions of unnatural base pairs which can be adapted for methods and compositions include, e.g., Hirao, et al., (2002) An unnatural base pair for incorporating amino acid analogues into protein. Nature Biotechnology, 20:177-182. See, also, Wu, Y., et al., (2002) J, Am. Chem. Soc. 124:14626-14630. Other relevant publications are listed below.<br>
[226]	For in vivo usage, the unnatural nucleoside is membrane permeable and is<br>
phosphorylated to form the corresponding triphosphate. In addition, the increased genetic information is stable and not destroyed by cellular enzymes. Previous efforts by Benner and others took advantage of hydrogen bonding patterns that are different from those in canonical Watson-Crick pairs, the most noteworthy example of which is the iso-C:isD-G pair. See, e.g., Switzer et al., (1989) J. Am. Chem. Soc. 111:8322; and Piccirilli et al., (1990) Nature. 343:33; Kool, (2000) Curr. Opin. Chem. Biol.. 4:602. These bases in general mispair to some degree with natural bases and cannot be enzymatically replicated. Kool and co-workers demonstrated that hydrophobic packing interactions between bases can replace hydrogen bonding to drive the formation of base pair. See, Kool, (2000) Curr. Opin. Chem. Biol.. 4:602; and Guckian and Kool, (1998) Angew. Chem. Int. Ed. Enpl,. 36, 2825. In an effort to develop an unnatural base pair satisfying all the above requirements, Schultz, Romesberg and co-workers have systematically synthesized and studied a series of unnatural hydrophobic bases. A PICS:PICS self-βair is found to be more stable than natural base pairs, and can be efficiently incorporated into DNA by Klenow fragment of Escherichia coli DNA polymerase I (KF). See, e.g., McMinn et al., (1999) J. Am. Chem. Soc, 121:11585-6; and Ogawa et al., (2000) J. Am. Chem. Soc, 122:3274. A 3MN:3MN self-βair can be synthesized by KF with efficiency and selectivity sufficient for biological function. See, e.g., Ogawa et al., (2000) J. Am. Chem. Soc, 122:8803. However, both bases act as a chain terminator for further rephcation. A mutant DNA polymerase has been recently evolved that can be used to replicate the PICS self pair. In addition, a 7AI self pair can be replicated. See, e.g., Tae et al, (2001) J. Am. Chem. Soc, 123:7439. A novel metallobase pair, DipicPy, has also been developed, which forms a stable<br><br>
pair-upon binding Cu(II). See, Meggers et a].. (2000) J. Am. Chem. Soc 122:10714. Because extended codons and unnatural codons are intrinsically orthogonal to natural codons, the methods of the invention can take advantage of this property to generate orthogonal tRNAs for them.<br>
[227]	A translational ,bypassing system can also be used to incorporate an unnatural<br>
amino acid in a desired polypeptide. In a translational bypassing system, a large sequence is incorporated into a gene but is not translated into protein. The sequence contains a structure that serves as a cue to induce the ribosome to hop over the sequence and resume translation downstream of the insertion.<br>
[228]	In certain embodiments, the protein or polypeptide of interest (or portion thereof)<br>
in the methods and/or compositions of the invention is encoded by a nucleic acid. Typically, the nucleic acid comprises at least one selector codon, at least two selector codons, at least three selector codons, at least four selector codons, at least five selector codons, at least six selector codons, at least seven selector codons, at least eight selector codons, at least nine selector codons, ten or more selector codons.<br>
[229]	Genes coding for proteins or polypeptides of interest can be mutagenized using<br>
methods well-known to one of ordinary skill in the art and described herein to include, for example, one or more selector codon for the incorporation of an unnatural amino acid. For example, a nucleic acid for a protein of interest is mutagenized to include one or more selector codon, providing for the incorporation of one or more unnatural amino acids. The invention includes any such variant, including but not limited to, mutant, versions of any protein, for example, including at least one unnatural amino acid. Similarly, the invention also includes corresponding nucleic acids, i.e., any nucleic acid with one or more selector codon that encodes one or more uimatural amino acid.<br>
[230]	Nucleic acid molecules encoding a protein of interest such as a hlFN polypeptide<br>
may be readily mutated to introduce a cysteine at any desired position of the polypeptide. Cysteine is widely used to introduce reactive molecules, water soluble polymers, proteins, or a wide variety of other molecules, onto a protein of interest. Methods suitable for the incorporation of cysteine into a desired position of a polypeptide are known to those of ordinary skill in the art, such as those described in U.S. Patent No. 6,608,183, which is incorporated by reference herein, and standard mutagenesis techniques.<br><br>
IV.	Non-Naturally Encoded Amino Acids<br>
[231]	A very wide variety of non-naturally encoded amino acids are suitable for use in<br>
the present invention. Any number of non-naturally encoded amino acids can be introduced into<br>
a hlFN polypeptide. In general, the introduced non-naturally encoded amino acids are<br>
substantially chemically inert toward the 20 common, genetically-encoded amino acids (i.e.,<br>
alanine, arginine, asparagine, aspartic acid, cysteine, glutamine, glutamic acid, glycine, histidine,<br>
isoleucine, leucine, lysine, methionine, phenylalanine, proline, serine, threonine, tryptophan,<br>
tyrosine, and valine). In some embodiments, the non-naturally encoded amino acids include<br>
side chain functional groups that react efficiently and selectively with functional groups not<br>
found in the 20 common amino acids (including but not limited to, azido, ketone, aldehyde and<br>
aminooxy groups) to form stable conjugates. For example, a hlFN polypeptide that includes a<br>
non-naturally encoded amino acid containing an azido functional group can be reacted with a<br>
polymer (including but not limited to, poly(ethylene glycol) or, alternatively, a second<br>
polypeptide containing an alkyne moiety to form a stable conjugate resulting for the selective<br>
reaction of the azide and the alkyne functional groups to form a Huisgen [3+2] cycloaddition<br>
product.<br>
[232]	The generic structure of an alpha-amino acid is illustrated as follows (Formula I):<br><br>
[233]	A non-naturally encoded amino acid is typically any structure having the above-<br>
listed formula wherein the R group is any substituent other than one used in the twenty natural amino acids, and may be suitable for use in the present invention. Because the non-naturally encoded amino acids of the invention typically differ from the natural amino acids only in the structure of the side chain, the non-naturally encoded amino acids form amide bonds with other amino acids, including but not limited to, natural or non-naturally encoded, in the same manner in which they are formed in naturally occurring polypeptides. However, the non-naturally encoded amino acids have side chain groups that distinguish them from the natural amino acids. For example, R optionally comprises an alkyl-, aryl-, acyl-, keto-, azido-, hydroxyl-, hydrazine, cyano-, halo-, hydrazide, alkenyl, alkynl, ether, thiol, seleno-, sulfonyl-, borate, boronate, phospho, phosphono, phosphine, heterocyclic, enone, imine, aldehyde, ester, thioacid, hydroxylamine, amino group, or the like or any combination thereof. Other non-naturally occurring amino acids of interest that may be suitable for use in the present invention include.<br><br>
out are not limited to, amino acids comprising a photoactivatable cross-linker, spin-labeled<br>
amino acids, fluorescent amino acids, metal binding amino acids, metal-containing amino acids,<br>
radioactive amino acids, amino acids with novel functional groups, amino acids that covalently<br>
or noncovalently interact with other molecules, photocaged and/or photoisomerizable amino<br>
acids, amino acids comprising biotin or a biotin analogue, glycosylated amino acids such as a<br>
sugar substituted serine, other carbohydrate modified amino acids, keto-containing amino acids,<br>
amino acids comprising polyethylene glycol or polyether, heavy atom substituted amino acids,<br>
chemically cleavable and/or photocleavable amino acids, amino acids with an elongated side<br>
chains as compared to natural amino acids, including but not limited to, polyethers or long chain<br>
hydrocarbons, including but not limited to, greater than about 5 or greater than about 10 carbons,<br>
carbon-linked sugar-containing amino acids, redox-αctive amino acids, amino thioacid<br>
containing amino acids, and amino acids comprising one or more toxic moiety.<br>
[234]	Exemplary non-naturally encoded amino acids that may be suitable for use in the<br>
present invention and that are useful for reactions with water soluble polymers include, but are<br>
not limited to, those with carbonyl, aminooxy, hydrazine, hydrazide, semicarbazide, azide and<br>
alkyne reactive groups. In some embodiments, non-naturally encoded amino acids comprise a<br>
saccharide moiety. Examples of such amino acids include N-acetyl-L-glucosaminyl-L-serine, N-<br>
acetyl-L-galactosaminyl-L-serine,	N-acetyl-L-glucosaminyl-L-threonine,	N-acetyl-L-<br>
glucosaminyl-L-asparagine and O-mannosaminyl-L-serine. Examples of such amino acids also include examples where the naturallγ-occuring N- or O-linkage between the amino acid and the saccharide is replaced by a covalent linkage not commonly found in nature - including but not limited to, an alkene, an oxime, a thioether, an amide and the like. Examples of such amino acids also include saccharides that are not commonly found in naturallγ-occuring proteins such as 2-deoxγ-glucose, 2-deoxygalactose and the like.<br>
[235]	Many   of  the   non-naturally   encoded   amino   acids   provided   herein   are<br>
commercially available, e.g,, from Sigmα-aldrich (St. Louis, MO, USA), Novabiochem (a division of EMD Biosciences, Darmstadt, Germany), or Peptech (Burlington, MA, USA). Those that are not commercially available are optionally synthesized as provided herein or using standard methods known to those of ordinary slcill in the art. For organic synthesis techniques, see, e.g., Organic Chemistry by Fessendon and Fessendon, (1982, Second Edition, Willard Grant Press, Boston Mass.); Advanced Organic Chemistry by March (Third Edition, 1985, Wiley and Sons, New York); and Advanced Organic Chemistry by Carey and Sundberg (Third Edition, Parts A and B, 1990, Plenum Press, New York). See, also, U.S. Patent No. 7,045,337 and U.S. Patent Application Publication 2003/0108885, which are incorporated by reference herein. In<br><br>
addition to unnatural amino acids that contain novel side chains, unnatural amino acids that may be suitable for use in the present invention also optionally comprise modified backbone structures, including but not limited to, as illustrated by the structures of Formula II and III:<br><br>
wherein Z typically comprises OH, NH2, SH, NH-R' or S-R'; X and Y, which can be the same or different, typically comprise S or O, and R and R', which are optionally the same or different, are typically selected from the same list of constituents for the R group described above for the unnatural amino acids having Formula I as well as hydrogen. For example, unnatural amino acids of the invention optionally comprise substitutions in the amino or carboxyl group as illustrated by Formulas II and III. Unnatural amino acids of this type include, but are not limited to, α-hydroxy acids, α-thioacids, α-aminothiocarboxylates, including but not limited to, with side chains corresponding to the common twenty natural amino acids or imnatural side chains. In addition, substitutions at the α-carbon optionally include, but are not limited to, L, D, or α-a-disubstituted amino acids such as D-glutamate, D-alanine, D-methyl-O-tyrosine, aminobutyric acid, and the like. Other structural alternatives include cyclic amino acids, such as proline analogues as well as 3, 4 ,6, 7, 8, and 9 membered ring proline analogues, β and y amino acids such as substituted β-alanine and γ-amino butyric acid.<br>
[236]	Many unnatural amino acids are based on natural amino acids, such as tyrosine,<br>
glutamine, phenylalanine, and the like, and are suitable for use in the present invention. Tyrosine analogs include, but are not limited to, parα-substituted tyrosines, ortho-substituted tyrosines, and meta substituted tyrosines, where the substituted tyrosine comprises, including but not limited to, a keto group (including but not limited to, an acetyl group), a benzoyl group, an amino group, a hydrazine, an hydroxyamine, a thiol group, a carboxy group, an isopropyl group, a methyl group, a C6 - C20 straight chain or branched hydrocarbon, a saturated or unsaturated hydrocarbon, an 0-methyl group, a polyether group, a nitro group, an alkynyl group or the like.<br><br>
In addition, multiply substituted aryl rings are also contemplated, Glutamme analogs tnat may be suitable for use in the present invention include, but are not limited to, α-hydroxy derivatives, γ-substituted derivatives, cyclic derivatives, and amide substituted glutamine derivatives. Example phenylalanine analogs that may be suitable for use in the present invention include, but are not limited to, parα-substituted phenylalanines, ortho-substituted phenyalanines, and metα-substituted phenylalanines, where the substituent comprises, including but not limited to, a hydroxy group, a methoxy group, a methyl group, an allyl group, an aldehyde, an azido, an iodo, a bromo, a keto group (including but not limited to, an acetyl group), a benzoyl, an alkynyl group, or the like. Specific examples of unnatural amino acids that may be suitable for use in the present invention include, but are not limited to, a,p-acetyI-L- phenylalanine, an O-methyl-L-tyrosine, an L-3-(2-naphthyi)alanine, a 3-methyl-phenylalanine, an O-4-allyl-L-tyrosine, a 4-propyl-L-tyrosine, a tri-O-acetyl-GlcNAcβ-serine, an L-Dopa, a fluorinated phenylalanine, an isopropyl-L-phenylalanine, a β-azido-L-phenylalanine, a p-acyl-L-phenylalanme, a p-benzoyl-L-phenyialanine, an L-phosphoserine, a phosphonoserine, a phosphonotyrosine, a p-iodo-phenylalanine, a p-bromophenylalanine, a β-amino-L-phenylaianine, an isopropyl-L-phenylalanine, and a p-propargyloxγ-phenylalanine, and the like. Examples of structures of a variety of unnatural amino acids that may be suitable for use in the present invention are provided in, for example, WO 2002/085923 entitled "In vivo incorporation of unnatural amino acids." See also Kiick et al., (2002) Incorporation of azides into recombinant proteins for chemoselective modification by the Staudinger ligation, PNAS 99:19-24, which is incorporated by reference herein, for additional methionine analogs.<br>
[237]	In one embodiment,  compositions of a hlFN polypeptide that include an<br>
unnatural amino acid (such as p-(propargyloxy)"phenyalanine) are provided. Various compositions comprising p-(propargyloxy)-phenyalanine and, including but not limited to, proteins and/or cells, are also provided. In one aspect, a composition that includes the p-(propargyioxy)-phenyalanine unnatural amino acid, further includes an orthogonal tRNA. The unnatural amino acid can be bonded (including but not limited to, covalently) to the orthogonal tRNA, including but not limited to, covalently bonded to the orthogonal tRNA though an amino-acyl bond, covalently bonded to a 3'OH or a 2'OH of a terminal ribose sugar of the orthogonal tRNA, etc.<br>
[238]	The chemical moieties via unnatural amino acids that can be incorporated into<br>
proteins offer a variety of advantages and manipulations of the protein. For example, the unique reactivity of a keto functional group allows selective modification of proteins with any of a<br><br>
number of hydrazine- or hydroxylamine-containing reagents in vitro and in vivo. A heavy atom unnatural amino acid, for example, can be useful for phasing X-ray structure data. The site-specific introduction of heavy atoms using unnatural amino acids also provides selectivity and flexibility in choosing positions for heavy atoms. Photoreactive uimatural amino acids (including but not limited to, amino acids with benzophenone and arylazides (including but not limited to, phenylazide) side chains), for example, allow for efficient in vivo and in vitro photocrosslinldng of protein. Examples of photoreactive unnatural amino acids include, but are not limited to, p-azido-phenylalanine and p-benzoyl-phenylalanine. The protein with the photoreactive uimatural amino acids can then be crosslinked at will by excitation of the photoreactive group-providing temporal control. In one example, the methyl group of an unnatural amino can be substituted with an isotopically labeled, including but not limited to, methyl group, as a probe of local structure and dynamics, including but not limited to, with the use of nuclear magnetic resonance and vibrational spectroscopy. Alkynyl or azido functional groups, for example, allow the selective modification of proteins with molecules through a [3+2] cycloaddition reaction.<br>
[239]	A non-natural amino acid incorporated into a polypeptide at the amino terminus<br>
can be composed of an R group that is any substituent other than one used in the twenty natural amino acids and a 2nd reactive group different from the NH2 group normally present in α-amino acids (see Formula I). A similar non-natural amino acid can be incorporated at the carboxyl terminus with a 2nd reactive group different from the COOH group normally present in α-amino acids (see Formula I).<br>
[240]	The unnatural amino acids of the invention may be selected or designed to<br>
provide additional characteristics unavailable in the twenty natural amino acids. For example, unnatural amino acid may be optionally designed or selected to modify the biological properties of a protein, e.g., into which they are incorporated. For example, the following properties may be optionally modified by inclusion of an unnatural amino acid into a protein: toxicity, biodistribution, solubility, stability, e.g., thermal, hydrolytic, oxidative, resistance to enzymatic degradation, and the like, facility of purification and processing, structural properties, spectroscopic properties, chemical and/or photochemical properties, catalytic activity, redox potential, half-life, ability to react with other molecules, e.g., covalently or noncovalently, and the like.<br><br>
CHEMICAL SYNTHESIS OF UNNATURAL AMINO ACIDS<br>
[241]	Many of the unnatural amino acids suitable for use in the present invention are<br>
commercially available, e.g., from Sigma (USA) or Aldrich (Milwaukee, WI, USA). Those that are not commercially available are optionally synthesized as provided herein or as provided in various publications or using standard methods known to those of ordinary skill in the art. For organic synthesis techniques, see, e.g., Organic Chemistry by Fessendon and Fessendon, (1982, Second Edition, Willard Grant Press, Boston Mass.); Advanced Organic Chemistry by March (Third Edition, 1985, Wiley and Sons, New York); and Advanced Organic Chemistry by Carey and Simdberg (Third Edition, Parts A and B, 1990, Plenum Press, New York).   Additional publications describing the synthesis of unnatural amino acids include, e.g., WO 2002/085923 entitled "In vivo incorporation of Unnatural Amino Acids;" Matsoukas et al., (1995) J. Med. Chem., 38, 466O-4669; King, F.E. &amp; Kidd, D.A.A. (1949) A New Synthesis ofGlutamine and of γ-Dipeptides of Glutamic Acid from Phthylated Intermediates. J. Chem. Soc. 3315-3319; Friedman, O.M. &amp; Chatterrji, R. (1959) Synthesis of Derivatives of Glutamine as Model Substrates for Anti-Tumor Agents, J. Am. Chem. Soc. 81, 375O-3752; Craig, J.C. et al. (1988) Absolute    Configuration    of   the    Enantiomers    of    7'Chloro-4    [[4-(diethylamino)-1 methylbutyl]amino]quinoline  (Chloroquine).  J.  Org.  Chem   53,   1167-1170; Azoulay,  M., Vilmont, M. &amp; Frappier, F. (1991) Glutamine analogues as Potential Antimalarials, Eur. J. Med. Chem. 26, 201-5; Koskinen, A.M.P. &amp; Rapoport, H. (1989) Synthesis of 4-Substituted Prolines as Conformationally Constrained Amino Acid Analogues. J. Org. Chem. 54, 1859-1866; Christie, B.D. &amp; Rapoport, H. (1985) Synthesis of Optically Pure Pipecolates from L-Asparagine.    Application to the Total Synthesis of (+)-apovincamine through Amino Acid Decarbonylation and Iminium Ion Cyclization. J. Org. Chem. 50:1239-1246; Barton et ah, (1987) Synthesis of Novel alpho-amino-acids and Derivatives  Using Radical Chemistry: Synthesis of L- and D-alphα-amino-adipic Acids, L-alphα-aminopimelic Acid and Appropriate Unsaturated Derivatives, Tetrahedron 43:4297-4308: and, Subasinghe et al., (1992) Quisqualic acid analogues: synthesis of betα-heterocyclic l-aminopropanoic acid derivatives and their activity at a novel quisqualate-sensitized site, J. Med. Chem. 35:4602-7. See also, U.S. Patent Publication No. US 2004/0198637 entitled "Protein Arrays," which is incorporated by reference herein.<br>
A.	Carbonyl reactive groups<br><br>
[242]	Amino acids with a carbonyl reactive group allow for a variety of reactions to<br>
link molecules (including but not limited to, PEG or other water soluble molecules) via nucleophilic addition or aldol condensation reactions among others.<br><br>
terminus modification group, and R4 is H, an amino acid, a polypeptide, or a carboxy terminus modification group. In some embodiments, n is 1, R1 is phenyl and R2 is a simple alkyl (i.e., methyl, ethyl, or propyl) and the ketone moiety is positioned in the para position relative to the alkyl side chain. In some embodiments, n is 1, R1 is phenyl and R2 is a simple alkyl (i.e., methyl, ethyl, or propyl) and the ketone moiety is positioned in the meta position relative to the alkyl side chain.<br>
[244]	The synthesis of p-acetyl-(+/-)-phenylalanine and /77-acetyl-(+/-)-phenylalanine is<br>
described in Zhang, Z., et al.. Biochemistry 42: 6735-6746 (2003), which is incorporated by reference herein. Other carbonyl-containing amino acids can be similarly prepared by one of ordinary skill in the art.<br>
[245]	In some embodiments, a polypeptide comprising a non-naturally encoded amino<br>
acid is chemically modified to generate a reactive carbonyl functional group. For instance; an aldehyde functionality useful<br>
 for conjugation reactions can be generated from a functionality<br>
having adjacent amino and hydroxyl groups. Where the biologically active molecule is a<br>
polypeptide, for example, an TV-terminal serine or threonine (which may be normally present or<br>
may be exposed via chemical or en2ymatic digestion) can be used to generate an aldehyde<br>
functionality under mild oxidative cleavage conditions using periodate. See, e.g., Gaertner, et<br>
ai, Bioconjug. Chem. 3: 262-268 (1992); Geoghegan, K. &amp; Stroh, J., Bioconjug. Chem, 3:138-<br>
146 (1992); Gaertner et aL, J, Biol Chem. 269:7224-7230 (1994). However, methods known in<br>
the art are restricted to the amino acid at the N-terminus of the peptide or protein.<br>
[246]	In the present invention, a non-naturally encoded amino acid bearing adjacent<br>
hydroxyl and amino groups can be incorporated into the polypeptide as a "masked" aldehyde functionality. For example, 5-hydroxylysine bears a hydroxyl group adjacent to the epsilon amine. Reaction conditions for generating the aldehyde typically involve addition of molar excess of sodium metaperiodate under mild conditions to avoid oxidation at other sites within the polypeptide.   The pH of the oxidation reaction is typically about 7.0.   A typical reaction<br><br>
involves the addition of about 1,5 molar excess of sodium meta periodate to a buffered solution of the polypeptide, followed by incubation for about 10 minutes in the dark. See, e.g, U.S. Patent No. 6,423,685, which is incorporated by reference herein.<br>
[247]	The  carbonyl functionality can be reacted selectively with a hydrazine-,<br>
hydrazide-5 hydroxylamine-, or semicarbazide-containing reagent under mild conditions in<br>
aqueous solution to form the corresponding hydrazone, oxime, or semicarbazone linlcages,<br>
respectively, that are stable under physiological conditions. See, e.g,, Jencks, W. P., J. Am,<br>
Chem. Soc, 81, 475-481 (1959); Shao, J. and Tarn, J. P., J. Am. Cham. Soc, 117:3893-3899<br>
(1995). Moreover, the unique reactivity of the carbonyl group allows for selective modification<br>
in the presence of the other amino acid side chains. See, e.g., Cornish, V. W., et al^ J, Am.<br>
Chem. Soc. 118:8150-8151 (1996); Geoghegan, K. F. &amp; Stroh, J. G., Bioconjug, Chem. 3:138-<br>
146 (1992); Mahal, L. K., et al. Science 276:1125-1128 (1997).<br>
B.	Hydrazine, hydrazide or semicarbazide reactive groups<br>
[248]	Non-naturally encoded amino acids containing a nucleophilic group, such as a<br>
hydrazine, hydrazide or semicarbazide, allow for reaction with a variety of electrophilic groups<br>
to form conjugates (including but not limited to, with PEG or other water soluble polymers).<br>
[249]	Exemplary hydrazine, hydrazide or semicarbazide -containing amino acids can be<br><br>
wherein n is 0-10; R1 is an alkyl, aryl, substituted alkyl, or substituted aryl or not present; X, is O, N, or S or not present; R2 is H, an amino acid, a polypeptide, or an amino terminus modification group, and R3 is HL, an amino acid, a polypeptide, or a carboxy terminus modification group.<br>
[250]	In some embodiments, n is 4, R1 is not present, and X is N.    In some<br>
embodiments, n is 2, R1 is not present, and X is not present. In some embodiments, n is 1, R1 is<br>
phenyl, X is O, and the oxygen atom is positioned para to the alphatic group on the aryl ring.<br>
[251]	Hydrazide-, hydrazine-, and semicarbazide-containing amino acids are available<br>
from commercial sources. For instance, L-glutamate-y-hydrazide is available from Sigma Chemical (St. Louis, MO). Other amino acids not available commercially can be prepared by one of ordinary skill in the art. See, e.g., U.S. Pat. No. 6,281,21 L which is incorporated by reference herein.<br><br>
[252]	Polypeptides containing non-naturally encoded amino acids that bear hydrazide,<br>
hydrazine or semicarbazide functionalities can be reacted efficiently and selectively with a<br>
variety of molecules that contain aldehydes or other functional groups with similar chemical<br>
reactivity. See, e.g., Shao, J. and Tarn, J., J, Am. Cham. Soc. 117:3893-3899 (1995). The unique<br>
reactivity of hydrazide, hydrazine and semicarbazide functional groups makes them significantly<br>
more reactive toward aldehydes, ketones and other electrophilic groups as compared to the<br>
nucleophilic groups present on the 20 common amino acids (including but not limited to, the<br>
hydroxyl group of serine or threonine or the amino groups of lysine and the N-terminus).<br>
C.	Aminooxy-containing amino acids<br>
[253]	Non-naturally encoded amino acids containing an aminooxy (also called a<br>
hydroxylamine) group allow for reaction with a variety of electrophilic groups to form<br>
conjugates (including but not limited to, with PEG or other water soluble polymers). Like<br>
hydrazines, hydrazides and semicarbazides, the enhanced nucleophilicity of the aminooxy group<br>
permits it to react efficiently and selectively with a variety of molecules that contain aldehydes<br>
or other functional groups with similar chemical reactivity. See, e.g., Shao, J. and Tam, J., J.<br>
Am. Chem. Soc. 117:3893-3899 (1995); H. Hang and C. Bertozzi, acc. Chem. Res. 34: 727-736<br>
(2001). Whereas the result of reaction with a hydrazine group is the corresponding hydrazone,<br>
however, an oxime results generally from the reaction of an aminooxy group with a carbonyl-<br>
containing group such as a ketone.<br>
[254]	Exemplary amino acids containing aminooxy groups can be represented as<br><br>
wherein n is 0-10; Ri is an alkyl, aryl, substituted alkyl, or substituted aryl or not present; X is 0, N, S or not present; m is 0-10; Y = C(O) or not present; R2 is H, an amino acid, a polypeptide, or an amino terminus modification group, and R3 is H, an amino acid, a polypeptide, or a carboxy terminus modification group. In some embodiments, n is 1, R1 is phenyl, X is 0, m is 1, and-Y is present. In some embodiments, n is 2, R1 and X are not present, m is 0, and Y is not present.<br>
[255]	Aminooxy-containing amino acids can be prepared from readily available amino<br>
acid precursors (homoserine, serine and threonine). See, e.g., M. Carrasco and R. Brown, J. Org. Chem. 68: 8853-8858 (2003). Certain aminooxy-containing amino acids, such as L-2-amino-4-(aminooxy)butyric acid), have been isolated fi'om natural sources (Rosenthal, G., Life<br><br>
Sci. 60: 1635-1641 (1997). Other aminooxy-containing amino acids can be prepared by one ot<br>
ordinary skill in the art.<br>
D.	Azide and alkyne reactive groups<br>
[256]	The unique reactivity of azide and alkyne functional  groups makes them<br>
extremely useful for the selective modification of polypeptides and other biological molecules. Organic azides, particularly alphatic azides, and alkynes are generally stable toward common reactive chemical conditions. In particular, both the azide and the alkyne functional groups are inert toward the side chains (i.e., R groups) of the 20 common amino acids found in naturallγ-occuring polypeptides. When brought into close proximity, however, the "spring-loaded" nature of the azide and alkyne groups is revealed and they react selectively and efficiently via Huisgen [3+2] cycloaddition reaction to generate the corresponding triazole. See, e.g.. Chin J., et al. Science 301:964-7 (2003); Wang, Q., et al, J. Am, Chem. Soc. 125, 3192-3193 (2003); Chin, J. W., et al,J Am, Chem. Soc. 124:9026-9027(2002).<br>
[257]	Because the Huisgen cycloaddition reaction involves a selective cycloaddition<br>
reaction (see, e.g.. Padwa, A., in CoMPREHENsrvE ORGANIC SYNTHESIS, Vol. 4, (ed. Trost, B. M., 1991), p. 1069-1109; Huisgen, R. in 1,3-DIPOIARCYCLOADDTTION CHEMISTRY, (ed. Padwa, A., 1984) , p. 1-176 ) rather than a nucleophilic substitution, the incorporation of non-naturally encoded amino acids bearing azide and allcyne-containing side chains permits the resultant polypeptides to be modified selectively at the position of the non-naturally encoded amino acid. Cycloaddition reaction involving azide or alkyne-containing hlFN polypeptide can be carried out at room temperature under aqueous conditions by the addition of Cu(II) (including but not limited to, in the form of a catalytic amount of CUSO4) in the presence of a reducing agent for reducing Cu(II) to Cu(I), in situ, in catalytic amount. See, e.g,, Wang, Q,, et aL, J, Am. Chem. Soc. 125, 3192-3193 (2003); Tomoe, C. W., et al, J. Org. Chem. 67:3057-3064 (2002); Rostovtsev, et aL, AngeM\ Chem. Int. Ed. 41:2596-2599 (2002). Exemplary reducing agents include, including but not limited to, ascorbate, metallic copper, quinine, hydroquinone, vitamin K, glutatliione, c3'-steine, Fe  , Co  , and an applied electric potential.<br>
[258]	In some cases, where a Huisgen [3+2] cycloaddition reaction between an azide<br>
and an alkyne is desired, the hlFN polypeptide comprises a non-naturally encoded amino acid comprising an alkyne moiety and the water soluble polymer to be attached to the amino acid comprises an azide moiety. Alternatively, the converse reaction (i.e., with the azide moiet&gt;' on the amino acid and the allcyne moietj' present on the water soluble polymer) can also be performed.<br><br>
[259]	The azide functional group can also be reacted selectively with a water soluble<br>
polymer containing an aryl ester and appropriately functionalized with an aryl phosphine moiety to generate an amide linkage. The aryl phosphine group reduces the azide in situ and the resulting amine then reacts efficiently with a proximal ester linlcage to generate the corresponding amide. See, e.g., E. Saxon and C. Bertozzi, Science 287, 2007-2010 (2000). The azide-containing amino acid can be either an alkyl azide (including but not limited to, 2-amino-6-azido-l-hexanoic acid) or an aryl azide (p-azido-phenylalanine).<br>
[260]	Exemplary water soluble polymers containing an aryl ester and a phosphine<br>
moiety can be represented as follows:<br><br>
hydrogen, substituted or unsubstituted heteroallcyl, substituted or unsubstituted aryl, including but not limited to, aryl substituted with 1-3 halogens, substituted or unsubstituted alkyl, alkoxy or thioalkoxy groups, or arylalkyl groups. When a compound of the invention includes more than one R group, for example, each of the R groups is independently selected as are each R\ R", R"' and R"" groups when more than one of these groups is present. When R' and R" are attached to the same nitrogen atom, they can be combined with the nitrogen atom to form a 5-, 6-, or 7-membered ring. For example, -NR'R" is meant to include, but not be limited to, 1-pyrrolidinyl and 4-morpholinyl. From the above discussion of substituents, one of skill in the art will understand that the term "alkyl" is meant to include groups including carbon atoms bound to groups other than hydrogen groups, such as haloalkyl (including but not limited to, -CF3 and -CH2CF3) and acyl (including but not limited to, -C(O)CH3, -C(O)CF3, -C(O)CH2OCH3, and the like).<br>
[261]	The azide functional group can also be reacted selectively with a water soluble<br>
polymer containing a thioester and appropriately functionalized with an aryl phosphine moiety to generate an amide linkage. The aryl phosphine group reduces the azide in situ and the resulting amine then reacts efficiently with the thioester linkage to generate the corresponding amide. Exemplary water soluble polymers containing a thioester and a phosphine moiety can be represented as follows:<br><br><br>
modification group, and R3 is H, an amino acid, a polypeptide, or a carboxy terminus modification group. In some embodiments, n is 1, R1 is phenyl, X is not present, m is 0 and the acetylene moiety is positioned in the para position relative to the alkyl side chain. In some embodiments, n is 1, R1 is phenyl, X is O, m is 1 and the propargyloxy group is positioned in the para position relative to the alkyl side chain (i.e., O-propargyl-tyrosine). In some embodiments, n is 1, R1 and X are not present and m is 0 (i.e., proparylglycine).<br>
[263]	Alkyne-containing amino  acids  are commercially available.     For example,<br>
propargylglycine is commercially available from Peptech (Burlington, MA). Alternatively, alkyne-containing amino acids can be prepared according to standard methods. For instance, p-propargyloxyphenylalanine can be synthesized, for example, as described in Deiters, A., et al, J. Am. Chem, Soc. 125: 11782-11783 (2003), and 4-alkynyl-L-phenylalanine can be synthesized as described in Kayser, B., et al, Tetrahedron 53(7): 2475-2484 (1997). Other alkyne-containing amino acids can be prepared by one of ordinary skill in the art.<br><br>
modification group, and R3 is H, an amino acid, a polypeptide, or a carboxy terminus modification group. In some embodiments, n is 1, R1 is phenyl, X is not present, m is 0 and the azide moiety is positioned para to the alkyl side chain. In some embodiments, n is 0-4 and R1 and X are not present, and m=0. In some embodiments, n is 1, R1 is phenyl, X is O, m is 2 and the β-azidoethoxy moiety is positioned in the para position relative to the alkyl side chain.<br><br>
[265]	Azide-containing amino acids are available from commercial sources.    For<br>
instance, 4-azidophenylalanine can be obtained from Chem-lmpex International, Inc. (Wood Dale, IL). For those azide-containing amino acids that are not commercially available, the azide group can be prepared relatively readily using standard methods known to those of ordinary skill in the art, including but not limited to, via displacement of a suitable leaving group (including but not limited to, halide, mesylate, tosylate) or via opening of a suitably protected lactone. See, e,g., Advanced Organic Chemistry by March (Third Edition, 1985, Wiley and Sons, New York).<br>
£.	Aminothiol reactive groups<br>
[266]	The unique reactivity of betα-substituted aminothiol functional groups makes<br>
them extremely useful for the selective modification of polypeptides and other biological molecules that contain aldehyde groups via formation of the thiazolidine. See, e.g., J. Shao and J. Tarn, J, Am. Chem. Soc. 1995, 117 (14) 3893-3899. hi some embodiments, betα-substituted aminothiol amino acids can be incorporated into hlFN polypeptides and then reacted with water soluble polymers comprising an aldehyde functionality. In some embodiments, a water soluble polymer, drug conjugate or other payload can be coupled to a hlFN polypeptide comprising a betα-substituted aminothiol amino acid via formation of the thiazolidine. CELLULAR UPTAKE OF UNNATURAL AMINO ACIDS<br>
[267]	Unnatural amino acid uptake by a cell is one issue that is typically considered<br>
when designing and selecting unnatural amino acids, including but not limited to, for incorporation into a protein. For example, the high charge density of α-amino acids suggests that these compounds are unlikely to be cell permeable. Natural amino acids are taken up into the eukaryotic cell via a collection of protein-based transport systems. A rapid screen can be done which assesses which unnatural amino acids, if any, are talcen up by cells. See, e.g., the toxicity assays in, e.g., U.S. Patent Publication No. US 2004/0198637 entitled "Protein Arrays'" which is incorporated by reference herein; and Liu, D.R. &amp; Schultz, P.O. (1999) Progress toward the evolution of an organism with an expanded genetic code. PNAS United States 96:478O-4785. Although uptake is easily analyzed with various assays, an alternative to designing unnatural amino acids that are amenable to cellular uptake pathways is to provide biosynthetic pathways to create amino acids in vivo, BIOSYNTHESIS OF UNNATURAL AMINO ACIDS<br>
[268]	Many biosynthetic pathways already exist in cells for the production of amino<br>
acids and other compounds. While a biosynthetic method for a particular vmnatural amino acid may not exist in nature, including but not limited to, in a cell, the invention provides such methods.    For example, biosynthetic pathways for unnatural amino acids are optionally<br><br>
generated in host cell by adding new enzymes or modifying existing host cell pathways. Additional new enzymes are optionally naturally occurring enzymes or artificially evolved enzymes. For example, the biosynthesis of p-aminophenylalanine (as presented in an example in WO 2002/085923 entitled "In vivo incorporation of unnatural amino acids") relies on the addition of a combination of known enzymes from other organisms. The genes for these enzymes can be introduced into a eukaryotic cell by transforming the cell with a plasmid comprising the genes. The genes, when expressed in the cell, provide an enzymatic pathway to synthesize the desired compound. Examples of the types of enzymes that are optionally added are provided in the examples below. Additional enzymes sequences are found, for example, in Genbank. Artificially evolved enzymes are also optionally added into a cell in the same manner. In this manner, the cellular machinery and resources of a cell are manipulated to produce unnatural amino acids.<br>
[269]	A variety of methods are available for producing novel enzymes for use in<br>
biosynthetic pathways or for evolution of existing pathways. For example, recursive recombination, including but not limited to, as developed by Maxygen, Inc. (available on the World Wide Web at maxygen.com), is optionally used to develop novel enzymes and pathways. See, e.g., Stemmer (1994), Rapid evolution of a protein in vitro by DMA shuffling. Nature 370(4):389-391; and, Stemmer, (1994), DMA shuffling by random fragmentation and reassembly: In vitro recombination for molecular evolution, Proc. Natl. Acad. Sci. USA., 91:10747-10751. Similarly DesignPath™, developed by Genencor (available on the Worid Wide Web at genencorxom) is optionally used for metabolic pathway engineering, including but not limited to, to engineer a pathway to create O-methyl-L-tyrosine in a cell. This technology reconstructs existing pathways in host organisms using a combination of new genes, including but not limited to, those identified through functional genomics, and molecular evolution and design. Diversa Corporation (available on the World Wide Web at diversa.com) also provides technology for rapidly screening libraries of genes and gene pathways, including but not hmited to, to create new pathways.<br>
[270]	Typically, the unnatural amino acid produced with an engineered biosynthetic<br>
pathway of the invention is produced in a concentration sufficient for efficient protein biosynthesis, including but not limited to, a natural cellular amoimt, but not to such a degree as to affect the concentration of the other amino acids or exhaust cellular resources. Typicall concentrations produced in vivo in this manner are about 10 mM to about 0.05 mM. Once a cell is transformed with a plasmid comprising the genes used to produce enzymes desired for a<br><br>
specific pathway and an unnatural amino acid is generated, in vivo selections are optionally used to further optimize the production of the unnatural amino acid for both ribosomal protein synthesis and cell growth.<br>
POLYPEPTIDES WITH UNNATURAL AMINO ACIDS<br>
[271]	The incorporation of an unnatural amino acid can be done for a variety of<br>
purposes, including but not limited to, tailoring changes in protein structure and/or function, changing size, acidity, nucleophilicity, hydrogen bonding, hydrophobicity, accessibility of protease target sites, targeting to a moiety (including but not limited to, for a protein array), adding a biologically active molecule, attaching a polymer, attaching a radionuclide, modulating serum half-life, modulating tissue penetration (e.g. tumors), modulating active transport, modulating tissue, cell or organ specificity or distribution (e.g. liver), modulating immunogenicity, modulating protease resistance, etc. Alterations in signal transduction may be achieved through site specific PEGylation of hlFN. Proteins that include an unnatural amino acid can have enhanced or even entirely new catalytic or biophysical properties. For example, the following properties are optionally modified by inclusion of an unmatural amino acid into a protein: toxicity, biodistribution, structural properties, spectroscopic properties, chemical and/or photochemical properties, catalytic ability, half-life (including but not limited to, serum half-life), ability to react with other molecules, including but not limited to, covalently or noncovalently, and the like. The compositions including proteins that include al least one unnatural amino acid are useful for, including but not limited to, novel therapeutics, diagnostics, catalytic enzymes, industrial enzymes, binding proteins (including but not limited to, antibodies), and including but not limited to, the study of protein structure and function. See, e.g., Dougherty, (2000) Unnatural Amino Acids as Probes of Protein Structure and Function, Current Opinion in Chemical Biology. 4:645-652.<br>
[272]	In one aspect of the invention, a composition includes at least one protein with at<br>
least one, including but not limited to, at least two, at least three, at least four, at least five, at least six, at least seven, at least eight, at least nine, or at least ten or more unnatural amino acids. The xmnatural amino acids can be the same or different, including but not limited to, there can be 1, 2, 3, 4, 5, 6, 7, 8, 9, or 10 or more different sites in the protein that comprise 1, 2, 3,4, 5, 6, 7, 8, 9, or 10 or more different unnatural amino acids. In another aspect, a composition includes a protein with at least one, but fewer than all, of a particular amino acid present in the protein is substituted with the unnatural amino acid. For a given protein with more than one uimatural amino acids, the unnatural amino acids can be identical or different (including but not limited to.<br><br>
the protein can include two or more different types of unnatural amino acids, or can include two<br>
of the same unnatural amino acid).  For a given protein with more than two unnatural amino<br>
acids, the unnatural amino acids can be the same, different or a combination of a multiple<br>
unnatural amino acid of the same kind with at least one different unnatural amino acid.<br>
[273]	Proteins or polypeptides of interest with at least one unnatural amino acid are a<br>
feature of the invention. The invention also includes polypeptides or proteins with at least one unnatural amino acid produced using the compositions and methods of the invention. An excipient (including but not limited to, a pharmaceutically acceptable excipient) can also be present with the protein.<br>
[274]	By producing proteins or polypeptides of interest with at least one unnatural<br>
amino acid in eukaryotic cells, proteins or polypeptides will typically include eukaryotic post-translational modifications. In certain embodiments, a protein includes at least one imnatural amino acid and at least one post-translational modification that is made in vivo by a eukaryotic cell, where the post-translational modification is not made by a prokaryotic cell. For example, the post-translation modification includes, including but not limited to, acetylation, acylation, lipid-modification, palmitoylation, palmitate addition, phosphorylation, glycolipid-linlcage modification, glycosylation, and the like. In one aspect, the post-translational modification includes attachment of an oligosaccharide (including but not limited to, (GlcNAc-Man)2"Man-GlcNAc-GlcNAc)) to an asparagine by a GlcNAc-asparagine linlcage. See Table 1 which lists some examples of N-linked oligosaccharides of eukaryotic proteins (additional residues can also be present, which are not shown). In another aspect, the post-translational modification includes attachment of an oligosaccharide (including but not lunited to, Gal-GalNAc, Gal-GlcNAc, etc.) to a serine or threonine by a GalNAc-serine or GalNAc-threonine linkage, or a GlcNAc-serine or a GlcNAc-threonine linkage.<br><br><br><br>
[275]	In yet another aspect, the post-translation modification includes proteolytic<br>
processing of precursors (including but not limited to, calcitonin precursor, calcitonin gene-related peptide precursor, preproparathyroid hormone, preproinsulin, proinsulin, prepro-opiomelanocortin, pro-opiomelanocortin and the like), assembly into a multisubunit protein or macromolecular assembly, translation to another site in the cell (including but not limited to, to organelles, such as the endoplasmic reticulum, the Golgi apparatus, the nucleus, lysosomes, peroxisomes, mitochondria, chloroplasts, vacuoles, etc., or through the secretory pathway). In certain embodiments, the protein comprises a secretion or localization sequence, an epitope tag, a FLAG tag, a polyhistidine tag, a GST fusion, or the iilce. U.S. Patent Nos, 4,963,495 and 6,436,674, which are incorporated herein by reference, detail constructs designed to improve secretion of hGH polypeptides.<br>
[2761	One advantage of an unnatural amino acid is that it presents additional chemical<br>
moieties that can be used to add additional molecules. These modifications can be made in vivo in a eukaryotic or non-eulcaryotic cell, or in vitro. Thus, in certain embodiments, the post-translational modification is through the unnatural amino acid. For example, the post-translational modification can be through a nucleophilic-electrophilic reaction. Most reactions currently used for the selective modification of proteins involve covalent bond formation between nucleophilic and electrophilic reaction partners, including but not limited to the reaction of α-haloketones with histidine or cysteine side chains. Selectivity in these cases is determined by the number and accessibility of the nucleophilic residues in the protein.   In proteins of the<br><br>
invention, other more selective reactions can be used such as the reaction of an unnatural keto-amino acid with hydrazides or aminooxy compounds, in vitro and in vivo. See, e.g,, Cornish, et al., (1996) J. Am. Chem. Soc, 118:8150.8151; Mahal, et al., (1997) Science, 276:1125-1128; Wang, et al., (2001) Science 292:498-500; Chin, et al, (2002) J. Am. Chem. Soc. 124:9026-9027; Chin, et al., (2002) Proc. Natl. Acad, Sci.. 99:1102O-11024; Wang, et al., (2003) Proc. Natl. Acad, Sci., 100:56-61; Zhang, et al., (2003) Biochemistry, 42:6735-6746; and. Chin, et al., (2003) Science, 301:964-7, all of which are incorporated by reference herem. This allows the selective labeling of virtually any protein with a host of reagents including fluorophores, crosslinking agents, saccharide derivatives and cytotoxic molecules. See also, U.S. Patent No. 6,927,042 entitled "Glycoprotein synthesis," which is incorporated by reference herein. Post-translational modifications, mcluding but not limited to, through an azido amino acid, can also made through the Staudinger ligation (mcluding but not limited to, with triarylphosphine reagents). See, e.g., Kiick et al., (2002) Incorporation of azides into recombinant proteins for chemoselective modification by the Staudinger ligation, PNAS 99:19-24.<br>
[277]	This invention provides  another highly  efficient method  for the  selective<br>
modification of proteins, which involves the genetic incorporation of unnatural amino acids, including but not limited to, containing an azide or alkynyl moiety into proteins in response to a selector codon. These amino acid side chains can then be modified by, including but not limited to, a Huisgen [3+2] cycloaddition reaction {see, e.g., Padwa, A. in Comprehensive Organic Synthesis. Vol. 4, (1991) Ed. Trost, B. M., Pergamon, Oxford, p. 1069-1109; and, Huisgen, R. in L3-Dipolar Cycloaddition Chemistry. (1984) Ed. Padwa, A., Wiley, New Yorlc, p. 1-176) with, including but not limited to, allcynyl or azide derivatives, respectively. Because this method involves a cycloaddition rather than a nucleophilic substitution, proteins can be modified with extremely high selectivity. This reaction can be carried out at room temperature in aqueous conditions with excellent regioselectivity (1,4 &gt; 1,5) by the addition of catalytic amounts of Cu(I) salts to the reaction mixture. See, e.g., Tomoe, et al, (2002) J. Org. Chem. 67:3057-3064; and, Rostovtsev, et al., (2002) Aneew. Chem. Int. Ed. 41:2596-2599. Another method that can be used is the ligand exchange on a bisarsenic compound with a tetracysteine motif, see, e.g., Griffin, et al, (1998) Science 281:269-272.<br>
[2781	A molecule that can be added to a protein of the invention through a [3+2]<br>
cycloaddition includes virtually any molecule with an azide or alkynyl derivative. Molecules include, but are not limited to, dyes, fluorophores, crosslinking agents, saccharide derivatives, polymers (including but not limited to, derivatives of polyethylene glycol), photocrosslinkers.<br><br>
cytotoxic compounds, affinity labels, derivatives of biotin, resins, beads, a second protein or polypeptide (or more), polynucleotide(s) (including but not limited to, DNA, RNA, etc.), metal chelators, cofactors, fatty acids, carbohydrates, and the like. These molecules can be added to an unnatural amino acid with an alkynyl group, including but not limited to, p-propargyloxyphenylalanine, or arido group, including but not limited to, p-azido-phenylalanine, respectively.<br>
V,	In vivo generation of klFN polypeptides comprising non-geneticallγ-encoded<br>
amino acids<br>
[279]	The hlFN polypeptides of the invention can be generated in vivo using modified<br>
tRNA and tRNA synthetases to add to or substitute amino acids that are not encoded in<br>
naturallγ-occurring systems.<br>
[280]	Methods for generating tRNAs and tRNA synthetases which use amino acids that<br>
are not encoded in naturallγ-occurring systems are described in, e.g., U.S. Patent No. 7,045,337<br>
(Serial No. 10/126,927) and U.S. Patent Application Publication 2003/0108885 (Serial No.<br>
10/126,931) which are incorporated by reference herein.   These methods involve generating a<br>
translational machinery that functions independently of the synthetases and tRNAs endogenous<br>
to the translation system (and are therefore sometimes referred to as "orthogonal").  Typically,<br>
the translation system comprises an orthogonal tRNA (O-tRNA) and an orthogonal aminoacyl<br>
tRNA synthetase (O-RS). Typically, the O-RS preferentially aminoacylates the O-tRNA with at<br>
least one non-naturally occurring amino acid in the translation system and the O-tRNA<br>
recognizes at least one selector codon that is not recognized by other tRNAs in the system. The<br>
translation system thus inserts the non-naturallγ-encoded amino acid into a protein produced in<br>
the system, in response to an encoded selector codon, thereby "substituting" an amino acid into a<br>
position in the encoded polypeptide.<br>
[281]	A wide variety of orthogonal tRNAs and aminoacyl tRNA synthetases have been<br>
described in the art for inserting particular synthetic amino acids into polypeptides, and are<br>
generally   suitable  for  use   in   the  present   invention.      For   example,   keto-specific   O-<br>
tRNA/aminoacyl-tRNA synthetases are described in Wang, L., ef al.Proc. Natl Acad. Sci USA<br>
100:56-61 (2003) and Zhang, Z. et al., Biochem. 42(22):6735-6746 (2003). Exemplary O-RS, or<br>
portions thereof, are encoded by polynucleotide sequences and include amino acid sequences<br>
disclosed in U.S. Patent No. 7,045,337 and U.S. Patent Application Publication 2003/0108885,<br>
each incorporated herein by reference.  Corresponding O-tRNA molecules for use with the O-<br>
RSs are also described in U.S. Patent No. 7,045,337 (Serial No. 10/126,927) and U.S. Patent<br><br>
Application Publication 2003/0108885 (Serial No. 10/126,931) which are incorporated by reference herein.<br>
[282]	An example of an azide-specific O-tRNA/aminoacyl-tRNA synthetase system is<br>
described in Chin, J. W„ et aL, J. Am. Chem. Soc. 124:9026-9027 (2002). Exemplary O-RS sequences forp-azido-L-phe include, but are not limited to, nucleotide sequences SEQ ID NOs: 14-16 and 29-32 and ammo acid sequences SEQ ID NOs: 46-48 and 61-64 as disclosed in U.S. Patent Application Publication 2003/0108885 (Serial No. 10/126,931) which is incorporated by reference herein. Exemplary O-tRNA sequences suitable for use in the present invention include, but are not limited to, nucleotide sequences SEQ ID NOs: 1-3 as disclosed in U.S. Patent Application Publication 2003/0108885 (Serial No. 10/126,931) which is incorporated by reference herein. Other examples of O-tRNA/aminoacyl-tRNA synthetase pairs specific to particular non-naturally encoded amino acids are described in U.S. Patent No. 7,045,337 (Serial No. 10/126,927) which is incorporated by reference herein. O-RS and O-tRNA that incorporate both keto- and azide-containing amino acids in S. cerevisiae are described in Chin, J. W., et al. Science 301:964-967 (2003).<br>
[283]	Several other orthogonal pairs have been reported. Glutaminyl {see, e.g,, Liu, D.<br>
R., and Schultz, P. G. (1999) Proc. Natl. Acad. Sci. U. S. A. 96:478O-4785), aspartyl {see, e.g.,<br>
Pastmak, M., et aL, (2000) Helv. Chim. Acta 83:2277-2286), and tyrosyl {see, e.g,, Ohno, S., et<br>
al., (1998) J. Biochem. (Tokyo, Jpn.) 124:1065-1068; and, Kowal, A. K., et al., (2001) Proc.<br>
Natl. Acad. Sci, U. S. A. 98:2268-2273) systems derived from S. cerevisiae tRNA's and<br>
synthetases have been described for the potential incorporation of uimatural amino acids in E.<br>
coli. Systems derived from the E, coli glutaminyl {see, e.g., Kowal, A. K., et ah, (2001) Proc.<br>
Natl. Acad. Sci. U. S. A. 98:2268-2273) and tyrosyl {see, e.g., Edwards, H., and Schimmel, P.<br>
(1990) Mol. Cell. Biol. 10:1633-1641) synthetases have been described for use in S. cerevisiae.<br>
The E. coli tyrosyl system has been used for the incorporation of 3-iodo-L-tyrosine in vivo^ in<br>
mammalian cells. See, Salcamoto, K., et aL, (2002) Nucleic Acids Res. 30:4692-4699.<br>
[284]	Use of O-tRNA/aminoacyl-tRNA synthetases involves selection of a specific<br>
codon which encodes the non-naturally encoded amino acid. While any codon can be used, it is generally desirable to select a codon that is rarely or never used in the cell in which theO-tRNA/aminoacyl-tRNA synthetase is expressed. For example, exemplary codons include nonsense codon such as stop codons (amber, ochre, and opal), four or more base codons and other natural three-base codons that are rarely or unxised.<br>
[285]	Specific selector codon(s) can be introduced into appropriate positions in the<br>
hlFN polynucleotide coding sequence using mutagenesis methods known in the art (including<br><br>
but not limited to, site-specific mutagenesis, cassette mutagenesis, restnction selection mutagenesis, etc.),<br>
[286]	Methods for generating components of the protein biosynthetic machinery, such<br>
as O-RSs, O-tRNAs, and orthogonal O-tRNA/O-RS pairs that can be used to incorporate a non-naturally encoded amino acid are described in Wang, L., et al, Science 292: 498-500 (2001); Chin, J. W., et al, 1 Am. Chem, Soc, 124:9026-9027 (2002); Zhang, Z. e! al. Biochemistry Al\ 6735-6746 (2003). Methods and compositions for the in vivo incorporation of non-naturally encoded amino acids are described m U.S. Patent No. 7,045,337 (Serial No. 10/126,927) which is incorporated by reference herein. Methods for selecting an orthogonal tRNα-tRNA synthetase pair for use in in vivo translation system of an organism are also described in U.S. Patent No. 7,045,337 (Serial No. 10/126,927) and U.S. Patent Application Publication 2003/0108885 (Serial No. 10/126,931) which are incorporated by reference herein. PCT Pubhcation No. WO 04/035743 entitled "Site Specific Incorporation of Keto Amino Acids into Proteins," which is incorporated by reference herein in its entirety, describes orthogonal RS and tRNA pairs for the incorporation of keto amino acids. PCT Publication No. WO 04/094593 entitled "Expanding the Eulcaryotic Genetic Code," which is incorporated by reference herein in its entirety, describes orthogonal RS and tRNA pairs for the incorporation of non-naturally encoded amino acids in eulcaryotic host cells.<br>
[287]	Methods for producing at least one recombinant orthogonal aminoacyl-tRNA<br>
synthetase (O-RS) comprise: (a) generating a library of (optionally mutant) RSs derived from at<br>
least one aminoacyl-tRNA synthetase (RS) from a first organism, including but not limited to, a<br>
prokaryotic organism, such as Methanococcus jannaschii, Methanobacterium<br>
thermoautotrophicum, Halobacterium, Escherichia coli A. fulgidus, P. furiosus, P. horikoshii,<br>
A. pernix, T. thermophilus, or the like, or a eukaryotic organism; (b) selecting (and/or screening)<br>
the library of RSs (optionally mutant RSs) for members that aminoacylate an orthogonal tRNA<br>
(O-tRNA) in the presence of a non-naturally encoded amino acid and a natural amino acid,<br>
thereby providing a pool of active (optionally mutant) RSs; and/or, (c) selecting (optionally<br>
through negative selection) the pool for active RSs (including but not limited to, mutant RSs)<br>
that preferentially aminoacylate the O-tRNA in the absence of the non-naturally encoded amino<br>
acid, thereby providing the at least one recombinant O-RS; wherein the at least one recombinant<br>
O-RS preferentially aminoacylates the O-tRNA with the non-naturally encoded amino acid.<br>
[288]	In one embodiment, the RS is an inactive RS. The inactive RS can be generated<br>
by mutating an active RS. For example, the inactive RS can be generated by mutating at least about 1, at least about 2, at least about 3, at least about 4, at least about 5, at least about 6, or at<br><br>
least about 10 or more amino acids to different amino acids, including but not limited to, alanine.<br>
[289]	Libraries of mutant RSs can be generated using various techniques known in the<br>
art, including but not limited to rational design based on protein three dimensional RS structure, or mutagenesis of RS nucleotides in a random or rational design technique. For example, the mutant RSs can be generated by site-specific mutations, random mutations, diversity generating recombination mutations, chimeric constructs, rational design and by other methods described herein or Icnown in the art.<br>
[290]	In one embodiment, selecting (and/or screening) the library of RSs (optionally<br>
mutant RSs) for members that are active, including but not limited to, that aminoacylate an orthogonal tRNA (O-tRNA) in the presence of a non-naturally encoded amino acid and a natural amino acid, includes: introducing a positive selection or screening marker, including but not limited to, an antibiotic resistance gene, or the like, and the library of (optionally mutant) RSs into a plurality of cells, wherein the positive selection and/or screening marker comprises at least one selector codon, including but not limited to, an amber, ochre, or opal codon; growing the plurality of cells in the presence of a selection agent; identifying cells that survive (or show a specific response) in the presence of the selection and/or screening agent by suppressing the at least one selector codon in the positive selection or screening marker, thereby providing a subset of positively selected cells that contains the pool of active (optionally mutant) RSs. Optionally, the selection and/or screening agent concentration can be varied.<br>
[291]	In    one   aspect,    the    positive   selection    marker    is    a    chloramphenicol<br>
acetyltransferase (CAT) gene and the selector codon is an amber stop codon in the CAT gene. Optionally, the positive selection marker is a P-lactamase gene and the selector codon is an amber stop codon in the β-lactamase gene. In another aspect the positive screening marker comprises a fluorescent or luminescent screening marker or an affinity based screening marker (including but not limited to, a cell surface marker).<br>
[292]	In one embodiment, negatively selecting or screening the pool for active RSs<br>
(optionally mutants) that preferentially aminoacylate the O-tRNA in the absence of the non-naturally encoded amino acid includes: introducing a negative selection or screening marker with the pool of active (optionally mutant) RSs from the positive selection or screenmg into a plurality of cells of a second organism, wherein the negative selection or screening marker comprises at least one selector codon (including but not limited to, an antibiotic resistance gene, including but not limited to, a chloramphenicol acetyltransferase (CAT) gene); and, identifying cells that survive or show a specific screening response in a first medium supplemented with the<br><br>
non-naturally encoded amino acid and a screening or selection agent, but fail to survive or to show the specific response in a second medium not supplemented with the non-naturally encoded amino acid and the selection or screening agent, thereby providing surviving cells or screened cells with the at least one recombinant O-RS. For example, a CAT identification protocol optionally acts as a positive selection and/or a negative screening in determination of appropriate O-RS recombinants. For instance, a pool of clones is optionally replicated on growth plates containing CAT (which comprises at least one selector codon) either with or without one or more non-naturally encoded amino acid. Colonies growing exclusively on the plates containing non-naturally encoded amino acids are thus regarded as containing recombinant O-RS. In one aspect, the concentration of the selection (and/or screening) agent is varied. In some aspects the first and second organisms are different. Tlius, the first and/or second organism optionally comprises: a prokaryote. a eukaryote, a mammal, an Escherichia coli, a fungi, a yeast, an archaebacterium, a eubacterium, a plant, an insect, a protist, etc. In other embodiments, the screening marker comprises a fluorescent or luminescent screening marker or an affinity based screening marker.<br>
[293]	In another embodiment, screening or selecting (including but not limited to,<br>
negatively selecting) the pool for active (optionally mutant) RSs includes: isolating the pool of active mutant RSs from the positive selection step (b); introducing a negative selection or screening marker, wherein the negative selection or screening marker comprises at least one selector codon (including but not limited to, a toxic marker gene, including but not limited to, a ribonuclease bamase gene, comprising at least one selector codon), and the pool of active (optionally mutant) RSs into a plurality of cells of a second organism; and identifying cells that survive or show a specific screening response in a first medium not supplemented with the non-naturally encoded amino acid, but fail to survive or show a specific screening response in a second medium supplemented with the non-naturally encoded amino acid, thereby providing surviving or screened cells with the at least one recombinant O-RS, wherein the at least one recombinant O-RS is specific for the non-naturally encoded amino acid. In one aspect, the at least one selector codon comprises about two or more selector codons. Such embodiments optionally can include wherein the at least one selector codon comprises two or more selector codons, and wherein the first and second organism are different (including but not limited to, each organism is optionally, including but not limited to, a prokaryote, a eukaryote, a mammal, an Escherichia coli, a fungi, a yeast, an archaebacteria, a eubacteria, a plant, an insect, a protist, etc.). Also, some aspects include wherein the negative selection marker comprises a ribonuclease bamase gene (which comprises at least one selector codon). Other aspects include<br><br>
wherein the screening marker optionally comprises a fluorescent or luminescent screening<br>
marker or an affinity based screening marker. In the embodiments herein, the screenings and/or<br>
selections optionally include variation of the screening and/or selection stringency.<br>
[294]	In one embodiment, the methods for producing at least one recombinant<br>
orthogonal aminoacyl-tRNA synthetase (O-RS) can further comprise; (d) isolating the at least one recombinant O-RS; (e) generating a second set of O-RS (optionally mutated) derived from the at least one recombmant O-RS; and, (f) repeating steps (b) and (c) until a mutated O-RS is obtained that comprises an ability to preferentially aminoacylate the O-tRNA. Optionally, steps (d)-(f) are repeated, including but not limited to, at least about two times. In one aspect, the second set of mutated O-RS derived from at least one recombinant O-RS can be generated by mutagenesis, including but not limited to, random mutagenesis, site-specific mutagenesis, recombination or a combination thereof.<br>
[295]	The stringency of the selection/screening steps, including but not limited to, the<br>
positive selection/screening step (b), the negative selection/screening step (c) or both the positive and negative selection/screening steps (b) and (c), in the above-described methods, optionally includes varying the selection/screening stringency. In another embodiment, the positive selection/screening step (b), the negative selection/screening step (c) or both the positive and negative selection/screening steps (b) and (c) comprise using a reporter, wherein the reporter is detected by fluorescence-activated cell sorting (FACS) or wherein the reporter is detected by luminescence. Optionally, the reporter is displayed on a cell surface^ on a phage display or the lilce and selected based upon afBnity or catalytic activity involving the non-naturally encoded amino acid or an analogue. In one embodiment, the mutated synthetase is displayed on a cell surface, on a phage display or the like.<br>
[296]	Methods for producing a recombinant orthogonal tRNA (O-tRNA) include: (a)<br>
generating a library of mutant tRNAs derived from at least one tRNA, including but not limited to, a suppressor tRNA, from a first organism; (b) selecting (including but not limited to, negatively selecting) or screening the library for (optionally mutant) tRNAs that are aminoacylated by an aminoacyl-tRNA synthetase (RS) from a second organism in the absence of a RS from the first organism, thereby providing a pool of tRNAs (optionally mutant); and, (c) selecting or screening the pool of tRNAs (optionally mutant) for members that are aminoacylated by an introduced orthogonal RS (O-RS), thereby providing at least one recombinant O-tRNA; wherein the at least one recombinant O-tRNA recognizes a selector codon and is not efficiency recognized by the RS from the second organism and is preferentially aminoacylated by the O-RS. In some embodiments the at least one tRNA is a suppressor tRNA<br><br>
and/or comprises a unique three base codon of natural and/or unnatural bases, or is a nonsense codon, a rare codon, an unnatural codon, a codon comprising at least 4 bases, an amber codon, an ochre codon, or an opal stop codon. In one embodiment, the recombinant O-tRNA possesses an improvement of orthogonality. It will be appreciated that in some embodiments, O-tRNA is optionally imported into a first organism from a second organism without the need for modification. In various embodiments, the first and second organisms are either the same or different and are optionally chosen from, including but not limited to, prokaryotes (including but not limited to, Methanococcus jannaschii, Methanobacterium thermoautotrophicum, Escherichia coli, Halobacterium, etc.), eukaryotes, mammals, fungi, yeasts, archaebacteria, eubacteria, plants, insects, protists, etc. Additionally, the recombinant tRNA is optionally aminoacylated by a non-naturally encoded amino acid, wherein the non-naturally encoded amino acid is biosynthesized in vivo either naturally or through genetic manipulation. The non-naturally encoded amino acid is optionally added to a growth medium for at least the first or second organism.<br>
[297]	In one aspect, selecting (including but not limited to, negatively selecting) or<br>
screening the library for (optionally mutant) tRNAs that are aminoacylated by an aminoacyl-tRNA synthetase (step (b)) includes: introducing a toxic marker gene, wherein the toxic marker gene comprises at least one of the selector codons (or a gene that leads to the production of a toxic or static agent or a gene essential to the organism wherein such marker gene comprises at least one selector codon) and the library of (optionally mutant) tRNAs into a plurality of cells from the second organism; and, selecting surviving cells, wherein the surviving cells contain the pool of (optionally mutant) tRNAs comprising at least one orthogonal tRNA or nonfunctional tRNA. For example, surviving cells can be selected by using a comparison ratio cell density assay.<br>
[298]	In another aspect, the toxic marker gene can include two or more selector codons.<br>
In another embodiment of the methods, the toxic marker gene is a ribonuclease bamase gene, where the ribonuclease bamase gene comprises at least one amber codon. Optionally, the ribonuclease bamase gene can include two or more amber codons.<br>
[299]	In one embodiment, selecting or screening the pool of (optionally mutant) tRNAs<br>
for members that are aminoacylated by an introduced orthogonal RS (O-RS) can include: introducing a positive selection or screening marker gene, wherein the positive marker gene comprises a drug resistance gene (including but not limited to, β-lactamase gene, comprising at least one of the selector codons, such as at least one amber stop codon) or a gene essential to the organism, or a gene that leads to detoxification of a toxic agent, along with the O-RS, and the<br><br>
pool of (optionally mutant) tRNAs into a plurality of cells from the second organism; and, identifying surviving or screened cells grown in the presence of a selection or screening agent, including but not limited to, an antibiotic, thereby providing a pool of cells possessing the at least one recombinant tRNA, where the at least one recombinant tRNA is aminoacylated by the O-RS and inserts an amino acid into a translation product encoded by the positive marker gene, in response to the at least one selector codons. In another embodiment, the concentration of the selection and/or screening agent is varied.<br>
[300]	Methods for generating specific O-tRNA/O-RS pairs are provided.    Methods<br>
include: (a) generating a library of mutant tRNAs derived from at least one tRNA from a first<br>
organism; (b) negatively selecting or screening the library for (optionally mutant) tRNAs that<br>
are aminoacylated by an aminoacyl-tRNA synthetase (RS) from a second organism in the<br>
absence of a RS from the first organism, thereby providing a pool of (optionally mutant) tRNAs;<br>
(c) selecting or screening the pool of (optionally mutant) tRNAs for members that are<br>
aminoacylated by an introduced orthogonal RS (O-RS), thereby providing at least one<br>
recombinant O-tRNA. The at least one recombinant O-tRNA recognizes a selector codon and is<br>
not efficiency recognized by the RS from the second organism and is preferentially<br>
aminoacylated by the O-RS. The method also includes (d) generating a library of (optionally<br>
mutant) RSs derived from at least one aminoacyl-tRNA synthetase (RS) from a third organism;<br>
(e) selecting or screening the library of mutant RSs for members that preferentially aminoacylate<br>
the at least one recombinant O-tRNA in the presence of a non-naturally encoded amino acid and<br>
a natural amino acid, thereby providing a pool of active (optionally mutant) RSs; and, (f)<br>
negatively selecting or screening the pool for active (optionally mutant) RSs that preferentially<br>
aminoacylate the at least one recombinant O-tRNA in the absence of the non-naturally encoded<br>
amino acid, thereby providing the at least one specific O-tRNA/O-RS pair, wherein the at least<br>
one specific O-tRNA/O-RS pair comprises at least one recombinant O-RS that is specific for the<br>
non-naturally encoded amino acid and the at least one recombinant O-tRNA. Specific O-<br>
tRNA/O-RS pairs produced by the methods are included. For example, the specific O-tRNA/O-<br>
RS pair can include, including but not limited to, a mutRNATyr-mutTyrRS pair, such as a<br>
mutRNATyr-SS12TyrRS pair, a mutRNALeu-mutLeuRS pair, a mutRNAThr-mutThrRS pan, a<br>
mutRNAGlu-mutGluRS pair, or the like. Additionally, such methods include wherein the first<br>
and third organism are the same (including but not limited to, Methanococcus jannaschii).<br>
[301]	Methods for selecting an orthogonal tRNα-tRNA synthetase pair for use in an in<br>
vivo translation system of a second organism are also included in the present invention.   The methods include: introducing a marker gene, a tRNA and an aminoacyl-tRNA synthetase (RS)<br><br>
isolated or derived from a first organism into a first set of cells from the second organism;<br>
introducing the marker gene and the tRNA into a duplicate cell set from a second organism; and,<br>
selecting for surviving cells in the first set that fail to survive in the duplicate cell set or<br>
screening for cells showing a specific screening response that fail to give such response in the<br>
duplicate cell set, wherein the first set and the duplicate cell set are grown in the presence of a<br>
selection or screening agent, wherein the surviving or screened cells comprise the orthogonal<br>
tRNα-tRNA synthetase pair for use in the in the in vivo translation system of the second<br>
organism. In one embodiment, comparing and selecting or screening includes an in vivo<br>
complementation assay. The concentration of the selection or screening agent can be varied,<br>
[302]	The organisms of the present invention comprise a variety of organism and a<br>
variety of combinations. For example, the first and the second organisms of the methods of the present invention can be the same or different. In one embodiment, the organisms are optionally a prokaryotic organism, including but not limited to, Methanococcus jannaschii Methanobacterium thermoautotrophicum, Halobacterium, Escherichia coli A. fulgidus, P. furiosus, P. horikoshii, A. pernix, T. thermophilus, or the like. Alternatively, the organisms optionally comprise a eulcaryotic organism, including but not limited to, plants (including but not limited to, complex plants such as monocots, or dicots), algae, protists. fungi (including but not limited to, yeast, etc), animals (including but not limited to, mammals, insects, arthropods, etc.), or the like. In another embodiment, the second organism is a prokaryotic organism, including but not limited to, Methanococcus Jannaschii, Methanobacterium thermoautotrophicum, Halobacterium, Escherichia coli, A. fulgidus, Halobacterium, P, furiosus, P. horikoshii, A. pernix, T. thermophilus, or the like. Alternatively, the second organism can be a eulcaryotic organism, including but not limited to, a yeast, a animal cell, a plant cell, a fungus, a mammalian cell, or the like. In various embodiments the first and second organisms are different.<br>
VI.	Location of non-naturally-occurring amino acids in hlFNpolypeptides<br>
[303]	The present invention contemplates incorporation of one or more non-naturallγ-<br>
occurring amino acids into hlFN polypeptides. One or more non-naturallγ-occurring amino acids may be incorporated at a particular position which does not disrupt activity of the polypeptide. This can be achieved by maldng "conservative" substitutions, including but not limited to, substituting hydrophobic ammo acids with hydrophobic amino acids, bulky amino acids for bulky amino acids, hydrophilic amino acids for hydrophilic amino acids and/or inserting the non-naturally-occurring amino acid in a location that is not required for activity.<br><br>
[304]	Regions of hlFN can be illustrated as follows, wherein the amino acid positions<br>
in hIFN are according to SEQ ID N0:2:<br>
1-9 (N-terminus), 1O-21 (A helix), 22-39 (region between A helix and B helix), 4O-75 (B helix), 76-77 (region between B helix and C helix), 78-100 (C helix), 101-110 (region between C helix and D helix), 111-132 (D helix), 133-136 (region between D and E helix) 137-155 (E helix) 156-165 (C-terminus).<br>
[305]	A variety of biochemical and structural approaches can be employed to select the<br>
desired sites for substitution with a non-naturally encoded amino acid within the hlFN polypeptide. It is readily apparent to those of ordinary skill in the art that any position of the polypeptide chain is suitable for selection to incorporate a non-naturally encoded amino acid, and selection may be based on rational design or by random selection for any or no particular desired purpose. Selection of desired sites may be for producing a hlFN molecule having any desired property or activity, including but not limited to, agonists, super-agonists, inverse agonists, antagonists, receptor binding modulators, receptor activity modulators, dimer or multimer formation, no change to activity or property compared to the native molecule, or manipulating any physical or chemical property of the polypeptide such as solubility, aggregation, or stability. For example, locations in the polypeptide required for biological activity of hlFN polypeptides can be identified using point mutation analysis, alanine scanning or homolog scanning methods known in the art. See, e.g., Di Marco et al., Biochem Biophys Res Com 202:1445 (1994); Walter et al.. Cancer Biotherapy &amp; Radiophaim. 13:143 (1998); Runkel et al., J.B.C. 273:8003 (1998) for IFN. U.S. Patent No. 5,580,723; 5,834,250; 6,013,478; 6,428,954; and 6,451,561, which are incorporated by reference herein, describe methods for the systematic analysis of the structure and function of polypeptides such as hGH by identifying active domains which influence the activity of the polypeptide with a target substance. Residues other than those identified as critical to biological activity by alanine or homolog scanning mutagenesis may be good candidates for substitution with a non-naturally encoded amino acid depending on the desired activity sought for the polypeptide. Alternatively, the sites identified as critical to biological activity may also be good candidates for substitution with a non-naturally encoded amino acid, again depending on the desired activity sought for the polypeptide. Another alternative would be to simply make serial substitutions in each position on the polypeptide chain with a non-naturally encoded amino acid and observe the effect on the activities of the polypeptide. Alternatively, residues that modulate one or more of the biological activities of IFN including side effects found with current IFN therapeutics may be good candidates for substitution with a non-naturally encoded amino acid.  Such side effects include<br><br>
but are not limited to, neutropenia. Alternatively, residues that modulate toxicity may be good candidates for substitution with a non-naturally encoded amino acid. Such residues may include, but are not limited to, residues that interact with one or more members of the opioid family of receptors and/or residues to modulate the induction of indoleamine 2,3-dioxygenase. Residues that improve antiviral activity may be good candidates for substitution with a non-naturally encoded amino acid. Chimeric hlFN polypeptides may be designed incorporating regions or sites from limitin into IFNα to provide an improved hlFN polypeptide with retained or improved antiviral activity and/or modulated side effects and/or modulated toxicity. One or more residues in hlFN may be substituted with one or more residues found in limitin. One or more residues in the C-D loop may be good candidates for substitution with a non-naturally encoded amino acid or other modifications, including but not limited to, substitution of residues in IFN with residues found in limitin. A majority of the residues in the C-D loop of hlFN may be substituted with residues found in limitin. Suitable residues from limitin for substitution into hlFN may be determined by performing a comparison between the sequences, three-dimensional structure, secondary structure, one or more biological activities, or receptor binding of limitin and hlFN. It is readily apparent to those of ordinary skill in the art that any means, technique, or method for selecting a position for substitution with a non-natural amino acid into any polypeptide is suitable for use in the present invention.<br>
[306]	The structure and activity of naturallγ-occurring mutants of hlFN polypeptides<br>
that contain deletions can also be examined to determine regions of the protein that are likely to be tolerant of substitution with a non-naturally encoded amino acid. In a similar manner, protease digestion and monoclonal antibodies can be used to identify regions of hlFN that are responsible for binding the hlFN receptor. Once residues that are likely to be intolerant to substitution with non-naturally encoded amino acids have been eliminated, the impact of proposed substitutions at each of the remaining positions can be examined from the three-dimensional crj'stal structure of the hlFN and its binding proteins. X-ray crystallographic and NMR structures of hlFN are also available in the Protem Data Bank (including 1RH2 and IITF) (PDB, available on the World Wide Web at rcsb.org), a centralized database containing three-dimensional structural data of large molecules of proteins and nucleic acids, as well as U.S. Patent No. 5,602,232; 5,460,956; 5,441,734; 4,672,108, which are incorporated by reference herein. Models may be made investigating the secondary and tertiary structure of polypeptides, if three-dimensional structural data is not available. Thus, those of ordinarj' skill in the art can readily identify amino acid positions that can be substituted with non-naturally encoded amino acids.<br><br>
[307]	In some embodiments, the hlFN polypeptides of the invention comprise one or<br>
more non-naturally occurring amino acids positioned in a region of the protein that does not disrupt the helices or beta sheet secondary structure of the polypeptide.<br>
[308]	Exemplary residues of incorporation of a non-naturally encoded amino acid may<br>
be those that are excluded from potential receptor binding regions (including but not limited to.<br>
Site I and Site 11), may be fully or partially solvent exposed, have minimal or no hydrogen-<br>
bonding interactions with nearby residues, may be minimally exposed to nearby reactive<br>
residues, and may be in regions that are highly flexible (including but not limited to, C-D loop)<br>
or structurally rigid (including but not limited to, B helix) as predicted by the three-dimensional<br>
crystal structure, secondary, tertiary, or quatemarj' structure of the hlFN polypeptide, bound or<br>
unbound to its receptor. Residues of incorporation of a non-naturally encoded amino acid may<br>
be those that modulate one or more of the biological activities of IFN including side effects<br>
found with current IFN therapeutics. Residues of incorporation of a non-naturally encoded<br>
amino acid may be those that modulate toxicity. Such residues include, but are not limited to,<br>
residues that are involved with binding to one or more members of the opioid receptor family<br>
and/or residues that modulate induction of indoleamine 2,3-dioxygenase.<br>
[309]	In some embodiments, one or more non-naturally encoded amino acid are<br>
incorporated or substituted in one or more of the following regions corresponding to secondary structures in IFN wherein the amino acid positions in hlFN are according to SEQ ID NO: 2:<br>
1-9 (N-terminus), 1O-21 (A helix), 22-39 (region between A helix and B helix), 4O-75 (B helix), 76-77 (region between B helix and C helix), 78-100 (C helix), 101-110 (region between C helix and D helbc), 111-132 (D helix), 133-136 (region between D and E helix) 137-155 (E helix) 156-165 (C-terminus).<br>
[310]	In some embodiments, one or more non-naturally encoded amino acid are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide): before position 1 (i.e., at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18,19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39,40, 41,42, 43, 44,45,46, 47,48,49, 50,51, 52, 53, 54, 55, 56, 57, 58, 59, 60,61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus).  In some  embodiments,  one  or more  non-naturally  encoded  amino  acids  are<br><br>
incorporated in one or more of the following positions in IFN: before position I (i.e, at the N terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 12, 13, 16, 19,20, 22, 23, 24, 25,26, 27,28, 30, 31, 32,33, 34, 35, 40, 41, 42,45, 46, 48, 49, 50, 51, 58, 61, 64, 65, 68, 69, 70, 71, 73, 74, 77, 78, 79, 80, 81, 82, 83, 85, 86, 89, 90, 93, 94, 96, 97, 100, 101, 103, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164, 165, 166 (i.e. at the carboxyl terminus of the protein) (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, one or more non-naturally encoded amino acids are incorporated in one or more of the following positions in IFN: 6, 9, 12, 13, 16, 41, 45, 46, 48, 49, 61, 64, 65, 96, 100, 101, 103, 106,107, 108, 110,111,113,114, 117, 120,121, 149, 156, 159.160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 100, 106, 107, 108, 111, 113, 114 (SEQ ID NO: 2, or the corresponding amino acids in other IFN's), In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 41, 45, 46, 48, 49 (SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 61, 64, 65, 101, 103, 110, 117, 120, 121, 149 (SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 6, 9, 12, 13, 16, 96, 156, 159, 160, 161, 162 (SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally occurring amino acids at one or more of the following positions: 2, 3,4, 5, 7, 8,16, 19, 20,40,42,50,51, 58,68,69,70,71, 73,97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these or other positions is linked to a water soluble polymer, including but not limited to positions: before position 1 (i.e., at the N-terminus), 1,2,3,4,5,6,7,8,9, 10,11, 12, 13, 14, 15, 16, 17, 18, 19,20,21,22,23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40,41, 42,43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75,<br><br>
76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 9«, 99, lUU, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149. 150, 151, 152, 153, 154, 155, 156, 157,<br>
158,	159, 160, 161, 162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus) (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-natural]y encoded amino acid at these or other positions is linlced to a water soluble polymer, including but not limited to positions: before position 1 (i.e. the N terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9,12, 13, 16,19, 20, 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 40, 41, 42, 45, 46, 48, 49, 50, 51, 58, 61, 64, 65, 68, 69, 70, 71, 73, 74, 77, 78, 79, 80, 81, 82, 83, 85, 86, 89, 90, 93, 94, 96, 97, 100, 101, 103, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164, 165, 166 (i.e. the carboxyl terminus) (SEQ ID NO: 2, or the corresponding amino acids in other IFN's). In some embodiments, the water soluble polymer is coupled at one or more amino acid positions: 6, 9, 12, 13, 16, 41, 45, 46, 48, 49, 61, 64, 65, 96, 100, 101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 12L 149, 156,<br>
159,	160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 100, 106, 107, 108, 111, 113, 114 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 41, 45, 46, 48, 49 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 61, 64, 65, 101, 103, 110, 117, 120, 121, 149 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 6, 9, 12, 13, 16, 96, 156, 159, 160, 161, 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[311]	In some embodiments, the one or more non-naturally encoded amino acids at one<br>
or more of the following positions is linlced to one or more water-soluble polymer: 2, 3, 4, 5, 7, 8, 16, 19,20,40,42,50,51,58,68,69,70,71,73,97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one<br><br>
or more non-naturally encoded amino acids at one or more ot the following positions is linked to one or more water-soluble polymer: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the water soluble polymer is coupled to the IFN polypeptide to a non-naturally encoded amino acid at one or more of the following amino acid positions: 6, 9, 12, 13,16,41,45,46, 48,49, 61, 64, 65, 96, 100, 101 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments the water soluble polymer is coupled to the IFN polypeptide at one or more of the following amino acid positions: 6, 9, 12, 13,16,41,45,46,48, 49,61,64,65,96, 100, 101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121. 149, 156, 159, 160, 161 and 162 (SEQ ED NO: 2, or the corresponding amino acid in SEQ ID NO: 1,3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these positions is linked to one or more water soluble polymers, positions: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, tlie IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions providing an antagonist: 2, 3, 4, 5,1, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70,71,73,97,105, 109, 112,118,148,149,152,153,158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide); a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on the intended site selected and desired activity. Human IFN antagonists include, but are not limited to, hlFN polypeptides with one or more non-naturally encoded amino acid substitutions at 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, or any combinations thereof (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide).<br>
[312]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding ammo acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide): 31, 134,34,38,129,36,122,37, 121,41,125, 124, 149, 117,39, 118, 120, 107, 108, 106,100, 111,113,114,41,45,46,48,49,61,64,65,101,103,102,110,117,120,121,149,96,6,9,12, 13,16, 68, 70, 109,159, 161,156,160, 162, 24, 27, 78, 83, 85, 87, 89,164. In one embodiment, a non-naturally encoded amino acid is substituted at position 38 of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide). In some<br><br>
embodiments, the non-naturally encoded amino acid at one or more of these or other positions is linked to a water soluble polymer, including but not limited to positions: 31 134, 34, 38, 129, 36,122,37,121,41, 125,124,149, 117,39, 118, 120, 107, 108, 106, 100, 111, 113,114,41,45, 46, 48, 49, 61, 64, 65, 101, 103, 102, 110, 117, 120, 121, 149, 96, 6, 9, 12, 13, 16, 68, 70, 109,<br>
159,	161, 156, 160, 162, 24, 27, 78, 83, 85, 87, 89, 164 (as in SEQ ID NO: 2, or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide).<br>
[313]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3): before position 1 (i.e., at the N-terminus), 1,2,3,4,5,6,7,8,9, 10, 11,12, 13,14, 15, 16, 17, 18, 19,20,21,22,23,24,25,26, 27,28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39,40,41, 42,43,44,45, 46, 47,48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119. 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159,<br>
160,	161, 162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus) and one or more natural<br>
amino acid substitutions. In some embodiments, the one or more non-naturally encoded amino<br>
acids are coupled to a water soluble polymer. In some embodiments, the one or more non-<br>
naturally encoded amino acids are coupled to PEG. In one embodiment, the natural amino acid<br>
substitution is R149Y. In some embodiments, the natural amino acid substitution is R149E. In<br>
some embodiments, the natural amino acid substitution is R149S. In one embodiment, the non-<br>
natural amino acid substitution is at position 107 and the natural amino acid substitution is<br>
R149Y. In one embodiment, the non-natural amino acid substitution is at position 106 and the<br>
natural amino acid substitution is R149Y. In some embodiments, the one or more naturally<br>
encoded amino acid substitution is at one or more of the following positions of hlFN (SEQ ID<br>
NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3), including but not limited to:<br>
10, 16, 13, 79, 83, 85, 86, 87, 90, 91, 93, 94, 96, 120, 121, 124, 125, 128, 149. In some<br>
embodiments, the one or more naturally encoded amino acid substitution is one or more of the<br>
following substitutions (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or<br>
3), including but not limited to: GlOE, M16R, R13E, T79R, K83Q, K83S, Y85L, T86S, E87S,<br>
Q90R, Q91E, N93Q, D94V, E96K, R120K, K121T, Q124R, R125G, L128R, R149Y, R149E,<br>
R149S. In some embodiments, the natural amino acid substitution is at position 1 (the N-<br>
terminus).     In some embodiments, one or more non-naturally encoded amino acids are<br><br>
substituted at one or more of the following positions of hlFN (as in SEQ ID NU: 2, or the corresponding amino acids in other IFN's): 107, 78, 34. In some embodiments, the non-naturally encoded amino acid at one or more of these positions is coupled to a water soluble polymer: 107,78,34.<br>
[314]	One or more amino acids found in a limitin sequence may be substituted into a<br>
hlFN polypeptide (hybrid limitin/hlFN polypeptides). Examples include but are not limited to<br>
the natural amino acid substitutions described in the previous paragraph. Alternatively, a set of<br>
amino acids found in an interferon polypeptide may be replaced by a set of amino acids found in<br>
a limitin sequence. A set of amino acids may comprise contiguous amino acids or amino acids<br>
present in different portions of the molecule but are involved in a structural characteristic or<br>
biological activity of the polypeptide. The mouse limitin molecule has an improved CFU-GM<br>
toxicity profile compared to other IFNα proteins. Alignment of human IFNα-2a with the<br>
limitin protein sequence showed 30% amino acid identity. 50% sequence conservation was also<br>
observed. In particular, a prominent deletion in the limitin sequence between the C and D<br>
helices (in the loop between C and D helices) was observed. The "HV" mutant was generated<br>
with the following substitutions in hIFNα-2a (SEQ ID NO: 2): D77-D94 is replaced with the<br>
mouse limitin sequence HERALDQLLSSLWRELQV. The 'CD" mutant was generated with<br>
the following substitutions in hIFNα-2a (SEQ ID NO: 2): V105-D114 with GQSAPLP. This<br>
hybrid molecule with the loop region from limitin substituted into the human rFNα-2a protein<br>
("CD" mutant) was found to have equivalent anti-viral activity as the WHO IFN standard. In<br>
addition to the one or more limitin amino acids, the hlFN polypeptide may comprise one or<br>
more non-naturally encoded amino acids at any one or more positions of the hlFN polypeptide.<br>
In some embodiments, the one or more non-naturally encoded amino acids may be linked to a<br>
water soluble polymer such as PEG or bonded directly to a water soluble polymer such as PEG.<br>
In addition to the natural amino acid substitutions in the HV or the CD mutant, one or more<br>
additional natural amino acid substitutions may be found in the hlFN polypeptide.<br>
[315]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 34, 39, 45, 46, 64, 65, 68, 78, 85, 87, 101, 107, 108, 111, 114, 118, 124,125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions linked to one or more water-soluble polymer: 6, 16, 34, 39, 45, 46, 64, 65, 68, 78, 85, 87, 101, 107, 108, 111, 114, 118, 124, 125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding<br><br>
amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN<br>
polypeptide). In some embodiments, the hlFN polypeptide comprises one or more non-naturally<br>
encoded amino acids at one or more of the following positions linked to one or more water-<br>
soluble polymer: 6, 16,34,39,45,46,64,65,68,78,85,87, 101, 107, 108, 111, 114, 118,124,<br>
125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the<br>
corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide) and comprises one<br>
or more naturally encoded amino acid substitution. In some embodiments, the hlFN polypeptide<br>
comprises one or more non-naturally encoded amino acids at one or more of the following<br>
positions linked to one or more water-soluble polymer: 6, 16, 34, 39, 45, 46, 64^ 65, 68, 78, 85,<br>
87, 101, 107, 108, 111, 114, 118, 124, 125, 145, 146, 153, 156,96, 149 (SEQ ID NO: 2, or the<br>
corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any otlier<br>
IFN polypeptide).and comprises one or more of the following naturally encoded amino acid<br>
substitutions GlOE, M16R, R13E, T79R, K83Q, K83S, Y85L, T86S, E87S, Q90R, Q91E,<br>
N93Q, D94V, E96K, R120K, K121T, Q124R, R125G, L128R, R149Y, R149E, R149S.<br>
[316]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the followmg positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[317]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 and one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the coiresponding amino acids in SEQ ED NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 that is linked to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino<br><br>
acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 that is bonded to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[318]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO; 2 or the correspondmg amino acids in SEQ ID NO: 1 or 3), In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[319]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6,16, 37, 45,46, 78, 87,107, and 108 that is linked to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 that is bonded to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[320]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 37,45,46, 89, and 107 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer.   In<br><br>
some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded ammo<br>
acids at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[321]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 37,45,46, 89, and 107<br>
and comprises one or more of the following naturally encoded amino acid substitutions: T79R,<br>
L80A, Y85L, Y85S, E87S (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1<br>
or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally<br>
encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 that is<br>
linked to a water soluble polymer and comprises one or more of the following naturally encoded<br>
amino acid substitutions: T79R, L80A, YSSL, Y85S, E87S (SEQ ID NO: 2 or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide<br>
comprises one or more non-naturally encoded amino acids at one or more of the following<br>
positions: 37, 45, 46, 89, and 107 that is bonded to a water soluble polymer and comprises one<br>
or more of the following naturally encoded amino acid substitutions: T79R, L80A, Y85L,<br>
Y85S, E87S (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[322]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturaliy encoded amino acids at one or more of the following positions: 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 128, 129, 131, 132, 133, 134, 135, 136, 137, 158, 159, 160, 161, 162, 163, 164, 165 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 128, 129,131, 132, 133, 134, 135, 136, 137, 158, 159,160, 161, 162, 163, 164, 165 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 23, 24, 27, 31, 128, 131, 134, 158 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 23, 24, 27, 31, 128, 131, 134, 158 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 24, 27, 31, 128, 131, 134 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following<br><br>
positions that is linked or bonded to a water soluble polymer: 24,27, 31, 128,131,134 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[323]	In some embodiments, the IFN polypeptides of the invention comprise one or<br>
more non-naturally encoded amino acids at one or more of the following positions providing an antagonist: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165, or any combination thereof (SEQ ID NO: 2, or the corresponding amino acids in other IFN's); a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on the site selected and the desired activity. Human IFN antagonists include, but are not limited to, those with one or more substitutions at 22, 23, 24,25,26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77,78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, or any combination thereof (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[324]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's): 31, 134, 34, 38, 129, 36, 122, 37, 121, 41, 125, 124, 149, 117,39, 118, 120, 107, 108, 106, 100, 111, 113, 114,41,45,46,48,49,61, 64, 65, 101, 103,102, 110, 117, 120, 121,149, 96, 6, 9,16, 68, 70,109, 159, 161, 156, 160, 162, 12, 13, 24, 27, 78, 83, 85, 87, 89, 164, In one embodiment, a non-naturally encoded amino acid is substituted at position 38 of hlFN (as in SEQ ID NO; 2, or the corresponding amino acids in other IFN's). In some embodiments, the non-naturally occurring amino acid at these or other positions is linked to a water soluble polymer, including but not limited to positions: 31,134, 34, 38, 129, 36, 122, 37, 121, 41, 125, 124, 149, 117, 39, 118, 120, 107, 108, 106, 100, 111, 113, 114,41,45,46,48,49,61,64,65, 101, 103,102,110, 117, 120, 121, 149,96,6,9, 12, 13, 16, 68, 70,109,159, 161,156,160, 162, 24, 27, 78, 83, 85, 87, 89, 164.<br>
[325]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acid and a naturally encoded amino acid substitution. In some embodiments, the non-naturally encoded amino acid present in the hlFN polypeptide is linked to a water soluble polymer and the hlFN polypeptide comprises one or more naturally encoded amino acid substitution.<br>
[326]	A wide variety of non-naturally encoded amino acids can be substituted for, or<br>
incorporated into, a given position in a hlFN polypeptide. In general, a particular non-naturally encoded amino acid is selected for incorporation based on an examination of the three dimensional crystal structure of a hlFN polypeptide with its receptor, a preference for<br><br>
conservative substitutions (i.e., aryl-based non-naturally encoded amino acids, such as p-<br>
acetylphenylalanine or O-propargyltyrosine substituting for Phe, Tyr or Tip), and the specific<br>
conjugation chemistry that one desires to introduce into the hlFN polypeptide (e.g., the<br>
introduction of 4-azidophenylalanine if one wants to effect a Huisgen [3+2] cycloaddition with a<br>
water soluble polymer bearing an alkyne moiety or a amide bond formation with a water soluble<br>
polymer that bears an aryl ester that, in turn, incorporates a phosphine moiety).<br>
[327]	In one embodiment, the method further includes incorporating into the protein the<br>
unnatural amino acid, where the unnatural amino acid comprises a first reactive group; and contacting the protein with a molecule (including but not limited to, a label, a dye, a polymer, a water-soluble polymer, a derivative of polyethylene glycol, a photocrosslinker, a radionuclide, a cytotoxic compound, a drug, an affinity label, a photoaffinity label, a reactive compound, a resin, a second protein or polypeptide or polypeptide analog, an antibody or antibody fragment, a metal chelator, a cofactor, a fatty acid, a carbohydrate, a polynucleotide, a DNA, a RNA, an antisense polynucleotide, a saccharide, a water-soluble dendrimer, a cyclodextrin, an inhibitory ribonucleic acid, a biomaterial, a nanoparticle, a spin label, a fluorophore, a metal-containing moiety, a radioactive moiety, a novel functional group, a group that covalently or noncovalently interacts with other molecules, a photocaged moiety, an actinic radiation excitable moiety, a photoisomerizable moiety, biotin, a derivative of biotin, a biotin analogue^ a moiet)' incorporating a heavy atom, a chemically cleavable group, a photocleavable group, an elongated side chain, a carbon-linked sugar, a redox-active agent, an amino thioacid, a toxic moiety, an isotopically labeled moiety, a biophysical probe, a phosphorescent group, a chemiluminescent group, an electron dense group, a magnetic group, an intercalating group, a chromophore, an energy transfer agent, a biologically active agent, a detectable label, a small molecule, a quantum dot, a nanotransmitter, a radionucleotide, a radiotransmitter, a neutron-capture agent, or any combination of the above, or any other desirable compound or substance) that comprises a second reactive group. The first reactive group reacts with the second reactive group to attach the molecule to the unnatural amino acid through a [3+2] cycloaddition. In one embodiment, the first reactive group is an alkunyl or azido moiety and the second reactive group is an azido or alkynyl moiety. For example, the first reactive group is the alkynyl moiety (including but not limited to, in unnatural amino acid p-propargyloxyphenylalanine) and the second reactive group is the azido moiety. In another example, the first reactive group is the azido moiety (including but not limited to, in the unnatural amino acid p-azido-L-phenylalanine) and the second reactive group is the alkynyl moiety.<br><br>
[328]	In some cases, the non-naturally encoded amino acid substitution(s) will be<br>
combined with other additions, substitutions or deletions within the hlFN polypeptide to affect other biological traits of the hlFN polypeptide. In some cases, the other additions, substitutions or deletions may increase the stability (including but not limited to, resistance to proteolytic degradation) of the hlFN polypeptide or increase affinity of the hlFN polypeptide for its receptor. In some cases, the other additions, substitutions or deletions may increase the solubility (including but not limited to, when expressed in E. coli or other host cells) of the hlFN polypeptide. In some embodiments additions, substitutions or deletions may increase the polypeptide solubility following expression in E, coli or other recombinant host cells. In some embodiments sites are selected for substitution with a naturally encoded or non-natural amino acid in addition to another site for incorporation of a non-natural amino acid that results in increasing the polypeptide solubility following expression in E. coli recombinant host cells. In some embodiments, the hlFN polypeptides comprise another addition, substitution or deletion that modulates affinity for the hlFN polypeptide receptor, modulates (including but not limited to, increases or decreases) receptor dimerization, modulated downstream signaling events after hlFN receptor binding, stabilizes receptor dimers, modulates circulating half-life, modulates release or bio-availability, facilitates purification, or improves or alters a particular route of administration. hlFN polypeptides of the invention may modulate one or more of the biological activities of IFN including side effects found with current IFN therapeutics. hlFN polypeptides of the invention may modulate toxicity found with current IFN therapeutics. Similarly, hlFN polypeptides can comprise chemical or enzyme cleavage sequences, protease cleavage sequences, reactive groups, antibodγ-binding domains (including but not limited to, FLAG or polγ-His) or other affinity based sequences (including, but not limited to, FLAG, polγ-His, GST, etc.) or linlced molecules (including, but not limited to, biotin) that improve detection (including, but not limited to, GFP). purification, transport through tissues or membranes, prodrug release or activation, hlFN size reduction, or other traits of the polypeptide.<br>
[329]	In some embodiments, the substitution of a non-naturally encoded amino acid<br>
generates a hlFN antagonist. A subset of exemplary sites for incorporation of one or more non-naturally encoded amino acid hiclude: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71,73,97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (as in SEQ ID NO: 2, or the corresponding amino acids in other IFNs). Another subset of exemplary sites for incorporation of a non-naturally encoded amino acid include: 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ<br><br>
ID NO: 1 or 3). In some embodiments, hlFN antagonists comprise at least one substitution in the regions 1-9 (N-terminus), 1O-21 (A helix), 22-39 (region between A helix and B helix), 4O-75 (B helix), 76-77 (region between B helix and C helix), 78-100 (C helix), 101-110 (region between C helix and D helix), 11M32 (D helix), 133-136 (region between D and E helix), 137-155 (E helix), 156-165 (C-terminus) that cause IFN to act as an antagonist. In other embodiments, the exemplary sites of incorporation of a non-naturally encoded amino acid include residues within the amino terminal region of helix A and a portion of helix C. In other embodiments, the above-listed substitutions are combined with additional substitutions that cause the hlFN polypeptide to be a hlFN antagonist. In some embodiments, the hlFN antagonist comprises a non-naturally encoded amino acid linked to a water soluble polymer that is present in a receptor binding region of the hlFN molecule.<br>
[330]	In some cases, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, or more amino acids are substituted<br>
with one or more non-naturally-encoded amino acids. In some cases, the hlFN polypeptide<br>
further includes 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, or more substitutions of one or more non-naturally<br>
encoded amino acids for naturally-occurring amino acids. In some embodiments, one or more<br>
residues in the following regions of hlFN are substituted with one or more non-naturally<br>
encoded amino acids: 1-9 (N-terminus), 10-21 (A helix), 22-39 (region between A helix and B<br>
helix), 4O-75 (B helix), 76-77 (region between B helix and C helix), 78-100 (C helix), 101-110<br>
(region between C helix and D helix), 111-132 (D helix), 133-136 (region between D and E<br>
helix), 137-155 (E helix), 156-165 (C-terminus). In some cases, one or more non-naturally<br>
encoded residues are linked to one or more lower molecular weight linear or branched PEGs<br>
(approximately -^ 5-20 kDa in mass or less), thereby enhancing binding affinity and comparable<br>
serum half-life relative to the species attached to a single, higher molecular weight PEG.<br>
[331]	Sites for incorporation in hlFN of one or more non-naturally encoded amino<br>
acids include combinations of the following residues (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's or interferon-like cytoldnes): before position 1 (i.e. at the N-terminus), 1. 2, 3, 4, 5, 6, 7, 8, 9, 12,13,16, 19, 20, 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35,40,41, 42, 45, 46, 48, 49, 50, 51, 58, 61, 64, 65, 68, 69, 70, 71, 73, 74, 77, 78, 79, 80, 81, 82, 83, 85, 86, 89, 90, 93, 94, 96, 97, 100, 101, 103, 105, 106, 107, 108, 109, 110, 111, 112. 113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164, 165, 166 (i.e. at the carboxyl terminus of the protein) or any combination thereof. Sites for incorporation in hlFN of one or more non-naturally encoded amino acids include combinations of the following residues (as in SEQ ID NO: 2, or the corresponding amino acids in other IFN's or interferon-like cytokines): 31, 134,<br><br>
34, 38, 129, 36, 122, 37, 121, 41,125,124, 149,117, 39,118, 120,107,108, 106, 100, 111.113, 114,41,45,46,48,49,61,64,65,101, 103, 102,110, 117, 120, 121, 149,96,6,9, 16,68,70, 109, 159, 161,156,160,162,12, 13,24,27, 78, 83, 85, 87, 89, 164.<br>
VIL	Expression in Non-eukaryotes and Eukaryotes<br>
[332]	To obtain high level expression of a cloned hlFN polynucleotide, one typically<br>
subclones polynucleotides encoding a hlFN polypeptide of the invention into an expression vector that contains a strong promoter to direct transcription, a transcription/translation terminator, and if for a nucleic acid encoding a protein, a ribosome binding site for translational initiation. Suitable bacterial promoters are known to those of ordinary skill in the art and described, e.g., in Sambrook et ah and Ausubel et al<br>
[333]	Bacterial expression systems for expressing hlFN polypeptides of the invention<br>
are available in, including but not limited to, E. coH, Bacillus sp,, Pseudomonas fluorescens, Pseudomonas aeruginosa, Pseudomonas putida, and Salmonella (Palva et aL, Gene 22:229-235 (1983); Mosbach et aL, Nature 302:543-545 (1983)). Kits for such expression systems are commercially available. Eukaryotic expression systems for mammalian cells, yeast, and insect cells are known to those of ordinary skill in the art and are also commercially available. In cases where orthogonal tRNAs and aminoacyl tRNA synthetases (described above) are used to express the hlFN polypeptides of the invention, host cells for expression are selected based on their ability to use the orthogonal components. Exemplary host cells include Gram-positive bacteria (including but not limited to B, brevis, B. subtilis, or Streptomyces) and Gram-negative bacteria (£E. coli, Pseudomonas fluorescens, Pseudomonas aeruginosa, Pseudomonas putida), as well as yeast and other eukaryotic cells. Cells comprising O-tRNA/O-RS pairs can be used as described herein.<br>
[334]	A eulcaryotic host cell or non-eukaryotic host cell of the present invention<br>
provides the ability to synthesize proteins that comprise unnatural amino acids in large useful quantities. In one aspect, the composition optionally includes, including but not limited to, at least 10 micrograms, at least 50 micrograms, at least 75 micrograms, at least 100 micrograms, at least 200 micrograms, at least 250 micrograms, at least 500 micrograms, at least 1 milligram, at least 10 milligrams, at least 100 milligrams, at least one gram, or more of the protein that comprises an unnatural amino acid, or an amount that can be achieved with in vivo protein production methods (details on recombinant protein production and purification are provided herein). In another aspect, the protein is optionally present in the composition at a concentration of, including but not limited to, at least 10 micrograms of protein per liter, at least 50<br><br>
micrograms of protein per liter, at least 75 micrograms of protein per liter, at least lOU micrograms of protein per liter, at least 200 micrograms of protein per liter, at least 250 micrograms of protein per liter, at least 500 micrograms of protein per liter, at least 1 milligram of protein per liter, or at least 10 milligrams of protein per liter or more, in, including but not limited to, a cell lysate, a buffer, a pharmaceutical buffer, or other liquid suspension (including but not limited to, in a volume of, including but not limited to, anywhere from about 1 nl to about 100 L or more). The production of large quantities (including but not limited to, greater that that typically possible with other methods, including but not limited to, in vitro translation) of a protein in a eulcaryotic cell including at least one unnatural amino acid is a feature of the invention.<br>
[335]	A eukaryotic host cell or non-eulcaryotic host cell of the present invention<br>
provides the ability to biosynthesize proteins that comprise unnatural amino acids in large useful quantities. For example, proteins comprising an unnatural amino acid can be produced at a concentration of, including but not limited to, at least 10 µg/Iiter, at least 50µg/liter, at least 75 µg/liter, at least 100µg/liter, at least 200 µg.'liter, at least 250 µg/liter, or at least 500µg/liter, at least Img/liter, at least 2mg/liter, at least 3 mg/liter, at least 4 mg/liter, at least 5 mg/liter, at least 6 mg/liter, at least 7 mg/liter, at least 8 mg/liter, at least 9 mg/liter, at least 10 mg/liter, at least 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, 600, 700, 800, 900 mg/liter, 1 g/liter, 5 g/liter, 10 g/liter or more of protein m a cell extract, cell lysate, culture medium, a buffer, and/or the like.<br>
I.	Expression Systems. Culture, and Isolation<br>
[336]	hlFN polypeptides may be expressed in any number of suitable expression<br>
systems including, for example, yeast, insect cells, mammalian cells, and bacteria. A description of exemplary expression systems is provided below.<br>
[337]	Yeast As used herein, the term "yeast" includes any of the various yeasts capable<br>
of expressing a gene encoding a hlFN polypeptide. Such yeasts include, but are not limited to, ascosporogenous yeasts (Endomycetales), basidiosporogenous yeasts and yeasts belonging to the Fungi imperfecti (Blastomycetes) group. The ascosporogenous yeasts are divided into two families, Spermophthoraceae and Saccharomycetaceae. The latter is comprised of four subfamilies, Schizosaccharomycoideae (e.g., genus Schizosaccharomyces), Nadsonioideae, Lipomycoideae and Saccharomycoideae (e.g., genera Pichia, Klvyveromyces and Saccharomyces). The basidiosporogenous yeasts include the genera Leucosporidium, Rhodosporidium, Sporidiobolus, Filobasidium, and Filobasidiella,    Yeasts belonging to the<br><br>
Fungi Imperfecti (Blastomycetes) group are divided into two families, Sporobolomyceiaceae<br>
(e.g., genera Sporobolomyces and Bullera) and Oyptococcaceae (e.g.j genus Candida),<br>
[338]	Of particular interest for use with the present invention are species within the<br>
genera Pichia, Klytyveromyces, Saccharomyces, Schizosaccharomyces, Haiisermla, Torulopsis, and Candida, including, but not limited to, F. pastoris, P. guillerimondii S. cerevisiae, S. carlsbergensis, S. diastaticus, S. douglasii, S. kluyveri, S, norbensis, S. oviformis, K. lactis, K. fragilis, C. albicans, C. maltosa, and H. polymorpha.<br>
[339]	The selection of suitable yeast for expression of hlFN polypeptides is within the<br>
skill of one of ordinary slcill in the art. In selecting yeast hosts for expression, suitable hosts<br>
may include those shown to have, for example, good secretion capacity, low proteolytic activity,<br>
good secretion capacity, good soluble protein production, and overall robustness. Yeast are<br>
generally available from a variety of sources including, but not limited to, the Yeast Genetic<br>
Stock Center, Department of Biophysics and Medical Physics, University of California<br>
(Berkeley, CA), and the American Type Culture Collection ("ATCC") (Manassas, VA).<br>
[340]	The term "yeast host" or "yeast host cell" includes yeast that can be, or has been,<br>
used as a recipient for recombinant vectors or other transfer DNA. The term includes the progeny of the original yeast host cell that has received the recombinant vectors or other transfer DNA. It is understood that the progeny of a single parental cell may not necessarily be completely identical in morphology or in genomic or total DNA complement to the original parent, due to accidental or deliberate mutation. Progeny of the parental cell that are sufficiently similar to the parent to be characterized by the relevant property, such as the presence of a nucleotide sequence encoding a hlFN polypeptide, are included m the progeny intended by this definition.<br>
[341]	Expression and transformation vectors, including extrachromosomal replicons or<br>
integrating vectors, have been developed for transformation into many yeast hosts. For example, expression vectors have been developed for S. cerevisiae (Sikorski et al.. GENETICS (1989) 122:19; Ito et al., J. BACTERIOL. (1983) 153:163; Hinnen et al., PROC. NATL. ACAD. Sci. USA (1978) 75:1929); C albicans (Kurtz et al., MOL. CELL. BlOL. (1986) 6:142); C maltosa (Kunze et aL, J. BASJC MICROBIOL. (1985) 25:141); H. polymorpha (Gleeson et al, J. GEN. MICROBIOL. (1986) 132:3459; Roggenkamp et al., MOL. GENETICS AND GENOMICS (1986) 202:302); K. fragilis (Das et al., J. BACTERIOL. (1984) 158:1165); K lactis (De Louvencourt et al., J. BACTERIOL, (1983) 154:737; Van den Berg et al., BIOTECHNOLOGY (NY) (1990) 8:135); P. guillehmondii (Kunze et al., J. BASIC MICROBIOL. (1985) 25:141); P. pastoris (U.S. Patent Nos. 5,324,639;  4,929,555;  and 4,837,148;  Cregg  et  al.,  MOL.  CELL.  BIOL.  (1985) 5:3376);<br><br>
Schizosaccharomyces pombe (Beach et al, NATURE (1982) 300:706); and Y. lipolydca; A, nidulans (Ballance et al., B]OCHEM. BiOPHYS. RES. COMMUN. (1983) 112:284-89; Tilbum et al.. GENE (1983) 26:205-221; and Yelton et ai,, PROC. NATL. ACAD. SO. USA (1984) 81:1470^74); A. niger (Kelly and Hynes, EMBO J. (1985) 4:475-479); T reesia (EP 0 244 234); and filamentous fungi such as, e.g., Nevrospora, Penicillium, Tolypocladium (WO 91/00357), each incorporated by reference herein.<br>
[342]	Control sequences for yeast vectors are well known to those of ordinary skill in<br>
the art and include, but are not limited to, promoter regions from genes such as alcohol dehydrogenase (ADH) (EP 0 284 044); enolase; glucolcinase; glucose-6-phosphate isomerase; giyceraldehyde-3-phosphate-dehydrogenase (GAP or GAPDH); hexokinase; phosphofructokinase; 3-phosphoglycerate mutase; and pyruvate kinase (PyK) (EP 0 329 203). The yeast PH05 gene, encoding acid phosphatase, also may provide useful promoter sequences (Miyanohara et al., PROC. NATL. ACAD. SCI. USA (1983) 80:1), Other suitable promoter sequences for use with yeast hosts may include the promoters for 3-phosphoglycerate kinase (Hitzeman et al., J. BiOL. CHEM. (1980) 255:12073); and other glycolytic enzymes, such as pyruvate decarboxylase, triosephosphate isomerase, and phosphoglucose isomerase (Holland et al., BIOCHEMISTRY (1978) 17:4900; Hess et al., J. ADV. ENZYME REG. (1969) 7:149). Inducible yeast promoters having the additional advantage of transcription controlled by growth conditions may include the promoter regions for alcohol dehydrogenase 2; isocytochrome C; acid phosphatase; metallothionein; glyceraldehyde-3-phosphate dehydrogenase; degradative enzymes associated with nitrogen metabolism; and enzymes responsible for maltose and galactose utilization. Suitable vectors and promoters for use in yeast expression are further described in EP 0 073 657.<br>
[343]	Yeast enhancers also may be used with yeast promoters.    In addition, synthetic<br>
promoters may also function as yeast promoters. For example, the upstream activating sequences (UAS) of a yeast promoter may be joined with the transcription activation region of another yeast promoter, creating a synthetic hybrid promoter. Examples of such hybrid promoters include the ADH regulatory sequence linlced to the GAP transcription activation region. See U.S. Patent Nos. 4,880,734 and 4,876,197, which are incorporated by reference herein. Other examples of hybrid promoters include promoters that consist of the regulatory sequences of the ADH2, GAL4, GALIO, or PH05 genes, combined with the transcriptional activation region of a glycolytic enzyme gene such as GAP or PyK. See EP 0 164 556. Furthermore, a yezist promoter may include naturally occurring promoters of non-γeast origin that have the ability to bind yeast RNA polymerase and initiate transcription.<br><br>
[344]	Other control elements that may comprise part of the yeast expression vectors<br>
include terminators, for example, from GAPDH or the enolase genes (Holland et a!., J. BIOL, CHEM. (1981) 256:1385). In addition, the origin of replication from the 2µ„ plasmid origin is suitable for yeast. A suitable selection gene for use in yeast is the trpl gene present in the yeast plasmid. See Tschumper et al., GENE(1980) 10:157; Kingsmanet al., GENE(1979) 7:141. The trpl gene provides a selection marker for a mutant strain of yeast lacking the ability to grow in tryptophan. Similarly, Leu2-deficient yeast strains (ATCC 20,622 or 38,626) are complemented by known plasmids bearing the Leu2 gene.<br>
[345]	Methods of introducing exogenous DNA into yeast hosts are known to those of<br>
ordmary skill in the art, and typically include, but are not limited to, either the transformation of spheroplasts or of intact yeast host cells treated with alkali cations. For example, transformation of yeast can be carried out according to the method described in Hsiao et al., PROC. NATL. ACAD, Sci. USA (1979) 76:3829 and Van Solingen et al., J. BACT. (1977) 130:946. However, other methods for introducing DNA into cells such as by nuclear injection, electroporation, or protoplast fusion may also be used as described generally in SAMBROOK ET AL.. MOLECULAR CLONING: A LAB. MANUAL (2001). Yeast host cells may then be cultured using standard techniques known to those of ordinary skill in the art.<br>
[346]	Other methods for expressing heterologous proteins in yeast host cells are known<br>
to those of ordinary skill in the art. See generally U.S. Patent Publication No. 20020055169,<br>
U.S. Patent Nos. 6,361,969; 6,312,923; 6,183,985; 6,083,723; 6,017,731; 5,674,706; 5,629,203;<br>
5,602,034; and 5,089,398; U.S. Reexamined Patent Nos. RE37343 and RE35,749; PCT<br>
Published Patent Applications WO 99/07862; WO 98/37208; and WO 98/26080; European<br>
Patent Applications EP 0 946 736; EP 0 732 403; EP 0 480 480; WO 90/10277; EP 0 340 986;<br>
EP 0 329 203; EP 0 324 274; and EP 0 164 556. See also Gellissen et al., ANTONIE VAN<br>
LEEUWENHOEK (1992) 62(K2):79-93; Romanes et al.. YEAST (1992) 8(6):423-488; Goeddel,<br>
METHODS IN ENZYMOLOGY (1990) 185:3-7, each incorporated by reference herein.<br>
[347]	The yeast host strains may be grown in fermentors during the amplification stage<br>
using standard feed batch fermentation methods known to those of ordinary skill in the art. The fermentation methods may be adapted to account for differences in a particular yeast host's carbon utilization pathway or mode of expression control. For example, fermentation of a Saccharomyces yeast host may require a single glucose feed, complex nitrogen source (e.g., casein hydrolysates), and multiple vitamin supplementation. In contrast, the methylotrophic yeast P, pastoris may require glycerol, methanol, and trace mineral feeds, but only simple ammonium (nitrogen) salts for optimal growth and expression.   See, e.g., U.S. Patent No.<br><br>
5,324,639; Elliott et al, J. PROTEIN CHEM. (1990) 9:95; and Fieschko et al., BIOTECH. BIOENG. (1987) 29:1113, incorporated by reference herein.<br>
[348]	Such fermentation methods, however, may have  certain  common  features<br>
independent of the yeast host strain employed. For example, a growth limiting nutrient, typically carbon, may be added to the fermentor durmg the amplification phase to allow maximal growth. In addition, fermentation methods generally employ a fermentation medium designed to contain adequate amounts of carbon, nitrogen, basal salts, phosphorus, and other minor nutrients (vitamins, trace minerals and salts, etc.). Examples of fermentation media suitable for use with Pichia are described m U.S. Patent Nos. 5,324,639 and 5,231,178, which are incorporated by reference herein.<br>
[349]	Baculovirus-Infected Insect Cells   The term "insect host" or "insect host cell"<br>
refers to a insect that can be, or has been, used as a recipient for recombinant vectors or other transfer DNA. The term includes the progeny of the original insect host cell that has been transfected. It is understood that the progeny of a single parental cell may not necessarily be completely identical in morphology or in genomic or total DNA complement to the original parent, due to accidental or deliberate mutation. Progeny of the parental cell that are sufficiently similar to the parent to be characterized by the relevant property, such as the presence of a nucleotide sequence encoding a hlFN polypeptide, are included in the progeny intended by this definition.<br>
[350]	The selection of suitable insect cells for expression of hlFN polypeptides is<br>
known to those of ordmary skill in the art. Several insect species are well described in the art and are commercially available including Aedes aegypti, Bombyx mori, Drosophila melanogaster, Spodoptera frugiperda, and Trichoplusia rtl In selecting insect hosts for expression, suitable hosts may include those shown to have, inter alia, good secretion capacity, low proteolytic activity, and overall robustness. Insect are generally available from a variety of sources including, but not limited to, the Insect Genetic Stock Center, Department of Biophysics and Medical Physics, University of California (Berkeley, CA); and the American Type Culture Collection ("ATCC") (Manassas, VA).<br>
[351]	Generally, the components of a baculovirus-infected insect expression system<br>
include a transfer vector, usually a bacterial plasmid, which contains both a fi-agment of the baculovirus genome, and a convenient restriction site for insertion of the heterologous gene to be expressed; a wild type baculovirus with sequences homologous to the baculovirus-specific fragment in the transfer vector (this allows for the homologous recombination of the heterologous gene in to the baculovirus genome); and appropriate insect host cells and growth<br><br>
media. The materials, methods and techniques used in constructing vectors, transtectmg cens, picking plaques, growing cells in culture, and the like are known in the art and manuals are available describing these techniques.<br>
[352]	After inserting the heterologous gene into the transfer vector, the vector and the<br>
wild type viral genome are transfected into an insect host cell where the vector and viral genome recombine. The packaged recombinant virus is expressed and recombinant plaques are identified and purified. Materials and methods for baculovirus/insect cell expression systems are commercially available in kit form from, for example, Invitrogen Corp. (Carlsbad, CA). These techniques are generally known to those of ordinary skill in the art and fully described in SUMMERS AND SMITH, TEXAS AGRICULTURAL EXPERIMENT STATION BULLETIN NO. 1555 (1987), herein incorporated by reference. See also, RICHARDSON, 39 METHODS IN MOLECULAR BIOLOGY: BACULOVIRUS EXPRESSION PROTOCOLS (1995); AUSUBEL ET AL., CURRENT PROTOCOLS IN MOLECULAR BIOLOGY 16.9-16.11 (1994); KING AND POSSEE, THE BACULOVIRUS SYSTEM: A LABORATORY GUIDE (1992); and O'REILLY ET AL., BACULOVIRUS EXPRESSION VECTORS: A LABORATORY MANUAL (1992),<br>
[353]	Indeed, the production of various heterologous proteins using baculovirus/insect<br>
cell expression systems is known to those of ordinary skill in the art. See, e,g,, U.S. Patent Nos. 6,368,825; 6,342,216; 6,338,846; 6,261,805; 6,245,528, 6,225,060; 6,183,987; 6,168,932 6,126,944; 6,096,304; 6,013,433; 5,965,393; 5,939,285; 5,891,676; 5,871,986; 5,861,279 5,858,368; 5,843,733; 5,762,939; 5,753,220; 5,605,827; 5,583,023; 5,571,709; 5,516,657 5,290,686; WO 02/06305; WO 01/90390; WO 01/27301; WO 01/05956; WO 00/55345 WO 00/20032; WO 99/51721; WO 99/45130; WO 99/31257; WO 99/10515; WO 99/09193 WO 97/26332; WO 96/29400; WO 96/25496; WO 96/06161; WO 95/20672; WO 93/03173 WO 92/16619; WO 92/02628; WO 92/01801; WO 90/14428; WO 90/10078; WO 90/02566 WO 90/02186; WO 90/01556; WO 89/01038; WO 89/01037; WO 88/07082, which are incorporated by reference herein.<br>
[354]	Vectors that are useful in baculovirus/insect cell expression systems are known in<br>
the art and include, for example, insect expression and transfer vectors derived from the baculovirus Autographacalifornica nuclear polyhedrosis virus (AcNPV), which is a helper-independent, viral expression vector. Viral expression vectors derived from this system usually use the strong viral polyhedrin gene promoter to drive expression of heterologous genes. See generally, O'Reilly ET AL., BACULOVIRUS EXPRESSIO-N VECTORS: A LABORATORY MANUAL (1992).<br><br>
[355]	Prior to inserting the foreign gene into the baculovirus genome, tne above-<br>
described components, comprising a promoter, leader (if desired), coding sequence of interest, and transcription termination sequence, are typically assembled into an intermediate transplacement construct (transfer vector). Intermediate transplacement constructs are often maintained in a replicon, such as an extra chromosomal element (e.g., plasmids) capable of stable maintenance in a host, such as bacteria. The replicon will have a replication system, thus allowing it to be maintained in a suitable host for cloning and amplification. More specifically, the plasmid may contain the polyhedrin polyadenylation signal (Miller, ANN. REV. MICROBIOL. (1988) 42:177) and a prokaryotic ampicillin-resistance {amp) gene and origin of replication for selection and propagation in E. coli.<br>
[356]	One commonly used transfer vector for introducing foreign genes into AcNPV is<br>
pAc373. Many other vectors, known to those of skill in the art, have also been designed including, for example, pVL985, which alters the polyhedrin start codon from ATG to ATT, and which introduces a BamHI cloning site 32 base pairs downstream from the ATT. See Luckow and Summers, VIROLOGY 170:31 (1989). Other commercially available vectors include, for example, PBlueBac4.5/V5-His; pBlueBacHis2; pMelBac; pBlueBac4.5 (Invitrogen Corp., Carlsbad, CA).<br>
[357]	After insertion of the heterologous gene, the transfer vector and wild type<br>
baculoviral genome are co-transfected into an insect cell host. Methods for introducing<br>
heterologous DNA into the desired site in the baculovirus virus are known in the art. See<br>
SUMMERS AND SMITH, TEXAS AGRICULTURAL EXPERIMENT STATION BULLETIN NO. 1555 (1987);<br>
Smith et al., MoL CELL. BlOL. (1983) 3:2156; Luckow and Summers, VIROLOGY (1989) 170:31.<br>
For example, the insertion can be into a gene such as the polyhedrin gene, by homologous<br>
double crossover recombination; insertion can also be into a restriction enzyme site engineered<br>
into the desired baculovirus gene. See Miller et al., BIOESSAYS (1989) 11(4):91.<br>
[358]	Transfection may be accomplished by electroporation. See TROTTER AND WOOD,<br>
39 METHODS IN MOLECULAR BIOLOGY (1995); Mann and King, J. GEN. VIROL. (1989) 70:3501. Alternatively, liposomes may be used to transfect the insect cells with the recombinant expression vector and the baculovirus. See, e.g., Liebman et al., BIOTECHNIQUES (1999) 26(1):36; Graves et al., BIOCHEMISTRY (1998) 37:6050; Nomura et al„ J. BIOL. CHEM. (1998) 273(22):13570; Schmidt et al., PROTEIN EXPRESSION AND PURIFICATION (1998) 12:323; Siffert et al., NATURE GENETICS (1998) 18:45; TILKINS ET AL.. CELL BIOLOGY: A LABORATORY HANDBOOK 145-154 (1998); Cai et al.. PROTEIN EXPRESSION AND PURIFICATION (1997) 10:263; Dolphin et al., NATURE GENETICS (1997) 17:491; Kost et al. GENE (1997) 190:139; Jakobsson et<br><br>
al., J. BIOL. CHEM. (1996) 271:22203; Rowles et aL, J. BIOL. CHEM. (1996); Reverey et al, X BlOL. CHEM. (1996) 271(39):23607-10; Stanley et al., J. BiOL. CHEM. (1995) 270:4121; Sisk et al, X VIROL, (1994) 68(2):766; and Peng et al., BloTECHNlQUES (1993) 14(2):274. Commercially available liposomes include, for example, Cellfectin® and Lipofectin® (Invitrogen, Corp., Carlsbad, CA). In addition, calcium phosphate transfection may be used. See TROTTER AND WOOD, 39 METHODS IN MOLECULAR BIOLOGY (1995); Kitts, NAR (1990) 18(19):5667; and Mann and King, X GEN. VIROL. (1989) 70:3501.<br>
[359]	Baculovirus expression vectors usually contain a baculovirus promoter.    A<br>
baculovirus promoter is any DNA sequence capable of binding a baculovirus RNA polymerase<br>
and initiating the downstream (3') transcription of a coding sequence (e.g., structural gene) into<br>
mRNA. A promoter will have a transcription initiation region which is usually placed proximal<br>
to the 5' end of the coding sequence. This transcription initiation region typically includes an<br>
RNA polymerase binding site and a transcription initiation site. A baculovirus promoter may<br>
also have a second domain called an enhancer, which, if present, is usually distal to the<br>
structural gene. Moreover, expression may be either regulated or constitutive.<br>
[360]	Structural genes, abundantly transcribed at late times in the infection cycle,<br>
provide particularly useful promoter sequences.  Examples include sequences derived from the<br>
gene encoding the viral polyhedron protein (FRIESEN ET AL., The Regulation of Baculovirus<br>
Gene Expression in THE MOLECULAR BIOLOGY OF BACULOVIRUSES (1986); EP 0 127 839 and 0<br>
155 476) and the gene encoding the p10 protein (Vlak et al., J. GEN. VIROL. (1988) 69:765).<br>
[361]	The newly formed baculovirus expression vector is packaged into an infectious<br>
recombinant baculovirus and subsequently grown plaques may be purified by techniques known<br>
to those of ordinary skill in the art. See Miller et aL, BiOESSAYS (1989) 11(4):91; SUMMERS AND<br>
SMITH, TEXAS AGRICULTURAL EXPERIMENT STATION BULLETIN NO. 1555 (1987).<br>
[362]	Recombinant baculovirus expression vectors have been developed for infection<br>
into several insect cells. For example, recombinant baculoviruses have been developed for, inter alia, Aedes aegypti (ATCC No. CCL-125), Bombyx mori (ATCC No. CRL-8910), Drosophila melanogaster (ATCC No. 1963), Spodoptera frugiperda, and Trichoplusia ni. See Wright, NATURE (1986) 321:718; Carbonell et al., J. VIROL. (1985) 56:153; Smith et al., MOL. CELL. BIOL. (1983) 3:2156. See generally, Fraser et al., IN VITRO CELL. DEV. BIOL. (1989) 25:225. More specifically, the cell lines used for baculovirus expression vector systems commonly include, but are not limited to, Sf9 {Spodoptera frugiperda) (ATCC No. CRL-1711), Sf21 {Spodoptera frugiperda) (Invitrogen Corp., Cat No. 11497-013 (Carlsbad, CA)), Tri-368 (Trichopidsia ni), and High-FiveTM BTI-TN-5B1-4 (Trichopulsia ni).<br><br>
[363]	Cells and culture media are commercially available for both direct and tusion<br>
expression of heterologous polypeptides in a baculovirus/expression, and cell culture technology is generally known to those of ordinary skill in the art,<br>
[364]	E. Colt Pseudomonas species, and other Prokaryotes     Bacterial expression<br>
techniques are known to those of ordinary skill in the art. A wide variety of vectors are available for use in bacterial hosts. The vectors may be single copy or low or high multicopy vectors. Vectors may serve for cloning and/or expression. In view of the ample literature concerning vectors, commercial availability of many vectors, and even manuals describing vectors and their restriction maps and characteristics, no extensive discussion is required here. As is well-known, the vectors normally involve markers allowing for selection, which markers may provide for cytotoxic agent resistance, prototrophy or immunity. Frequently, a plurality of markers is present, which provide for different characteristics.<br>
[365]	A bacterial promoter is any DNA sequence capable of binding bacterial RNA<br>
polymerase and initiating the downstream (3') transcription of a coding sequence (e.g. structural gene) into mRNA. A promoter will have a transcription initiation region which is usually placed proximal to the 5' end of the coding sequence. This transcription initiation region typically includes an RNA polymerase binding site and a transcription initiation site. A bacterial promoter may also have a second domain called an operator, that may overlap an adjacent RNA polymerase binding site at which RNA synthesis begins. The operator permits negative regulated (inducible) transcription, as a gene repressor protein may bind the operator and thereby inhibit transcription of a specific gene. Constitutive expression may occur in the absence of negative regulatory elements, such as the operator. In addition, positive regulation may be achieved by a gene activator protein binding sequence, which, if present is usually proximal (5') to the RNA polymerase binding sequence. An example of a gene activator protein is the catabolite activator protein (CAP), which helps initiate transcription of the lac operon in Escherichia coli (E. coli) [Raibaud et ah, ANNU. REV. GENET. (1984) 18:173}. Regulated expression may therefore be either positive or negative, thereby either enhancing or reducing transcription.<br>
[366]	Sequences encoding metabolic pathway enzymes provide particularly useful<br>
promoter sequences. Examples include promoter sequences derived from sugar metabolizing enzymes, such as galactose, lactose (lac) [Chang et al.. NATURE (1977) 198:1056], and maltose. Additional examples include promoter sequences derived from biosynthetic enzymes such as tryptophan (trp) [Goeddel et al., Nuc. ACIDS RES. (1980) 8:4057; Yelverton et al., NUCL. ACIDS RES. (1981) 9:731; U.S. Pat. No. 4,738,921; EP Pub. Nos. 036 776 and 121 775, which are<br><br>
incorporated by reference herein]. The β-galactosidase (bia) promoter system [Weissmann<br>
(1981) "The cloning of interferon and other mistakes." In Interferon 3 (Ed. I. Gresser)],<br>
bacteriophage lambda PL [Shimatake et al., NATURE (1981) 292:128] and T5 [U.S. Pat. No.<br>
4,689,406, which are incorporated by reference herein] promoter systems also provide useiul<br>
promoter sequences. Preferred methods of the present invention utilize strong promoters, such<br>
as the T7 promoter to induce hlFN polypeptides at high levels. Examples of such vectors are<br>
kmown to those of ordmary skill in the art and include the pET29 series from Novagen, and the<br>
pPOP vectors described in WO99/05297, which is incorporated by reference herein. Such<br>
expression systems produce high levels, of hlFN polypeptides in the host without compromising<br>
host cell viability or growth parameters. pET19 (Novagen) is another vector known in the art.<br>
[367]	In addition, synthetic promoters which do not occur in nature also function as<br>
bacterial promoters. For example, transcription activation sequences of one bacterial or bacteriophage promoter may be joined with the operon sequences of another bacterial or bacteriophage promoter, creating a synthetic hybrid promoter [U.S. Pat. No. 4,551,433, which is incorporated by reference herein]. For example, the tac promoter is a hybrid trp-lac promoter comprised of both trp promoter and lac operon sequences that is regulated by the lac repressor [Amann et al., GEME (1983) 25:167; de Boer et al., PROC. NATL. ACAD. Sci. (1983) 80:21]. Furthermore, a bacterial promoter can include naturally occurring promoters of non-bacterial origin that have the ability to bind bacterial RNA polymerase and initiate transcription. A naturally occurring promoter of non-bacterial origin can also be coupled with a compatible RNA polymerase to produce high levels of expression of some genes in prokaryotes. The bacteriophage T7 RNA polymerase/promoter system is an example of a coupled promoter system [Studier et al., J. MOL. BIOL. (1986) 189:113; Tabor et al., Proc Natl. Acad, Sci. (1985) 82:1074]. In addition, a hybrid promoter can also be comprised of a bacteriophage promoter and an E. coli operator region (EP Pub. No. 267 851).<br>
[368]	In addition to a functioning promoter sequence, an efficient ribosome binding site<br>
is also useful for the expression of foreign genes in prokaryotes. In E. coli, the ribosome binding site is called the Shine-Dalgamo (SD) sequence and includes an initiation codon (ATG) and a sequence 3-9 nucleotides in length located 3-11 nucleotides upstream of the initiation codon [Shine et al., NATURE (1975) 254:34], The SD sequence is thought to promote binding of mRNA to the ribosome by the pairing of bases between the SD sequence and the 3' and of E. coh 16S rRNA [Steitz et al. "Genetic signals and nucleotide sequences in messenger RNA", In Biological Regulation and Development: Gene Expression (Ed. R. F. Goldberger, 1979)]. To express eukaryotic genes and prokaryotic genes with weak ribosome-binding site [Sambrook et<br><br>
al. "Expression of cloned genes in Escherichia coli", Molecular Cloning: A Laboratory Manual, 1989].<br>
[369]	The term "bacterial host" or "bacterial host cell" refers to a bacterial that can be,<br>
or has been, used as a recipient for recombinant vectors or other transfer DNA. The term<br>
includes the progeny of the original bacterial host eel! that has been transfected. It is<br>
imderstood that the progeny of a single parental cell may not necessarily be completely identical<br>
in morphology or in genomic or total DNA complement to the original parent, due to accidental<br>
or deliberate mutation. Progeny of the parental cell that are sufficiently similar to the parent to<br>
be characterized by the relevant property, such as the presence of a nucleotide sequence<br>
encoding a hlFN polypeptide, are included in the progeny intended by this definition.<br>
[370]	The selection of suitable host bacteria for expression of hlFN polypeptides is<br>
known to those of ordinary skill m the art. In selecting bacterial hosts for expression, suitable hosts may include those shown to have, inter alia, good inclusion body formation capacity, low proteolytic activity, and overall robustness. Bacterial hosts are generally available from a variety of sources including, but not limited to, the Bacterial Genetic Stock Center, Department of Biophysics and Medical Physics, University of California (Berkeley, CA); and the American Type Culture Collection ("ATCC") (Manassas, VA). Industrial/pharmaceutical fermentation generally use bacterial derived from K strains (e.g. W3110) or from bacteria derived from B strains (e.g. BL21). These strains are particularly useftil because their growth parameters are extremely well known and robust. In addition, these strains are non-pathogenic, which is commercially important for safety and environmental reasons. Other examples of suitable E, coli hosts include, but are not limited to, strains of BL21, DHIOB, or derivatives thereof In another embodiment of the methods of the present invention, the E. coli host is a protease minus strain including, but not limited to, OMP- and LON-. The host cell strain may be a species of Pseudomonas, including but not limited to, Pseudomonas fluorescens, Pseudomonas aeruginosa, and Pseudomonas putida, Pseudomonas fluorescens biovar 1, designated strain MBlOl, is known to be useful for recombinant production and is available for therapeutic protein production processes. Examples of a Pseudomonas expression system include the system available from The Dow Chemical Company as a host strain (Midland, MI available on the World Wide Web at dow.com). U.S. Patent Nos. 4,755,465 and 4,859,600, which are incorporated by reference herein, describe the use of Pseudomonas strains as a host cell for hGH production.<br>
[3711	Once a recombinant host cell strain has been established (i.e., the expression<br>
construct has been introduced into the host cell and host cells with the proper expression<br><br>
construct are isolated), the recombinant host cell strain is cultured under conditions appropnate for production of hlFN polypeptides. As will be apparent to one of skill in the art, the method of culture of the recombinant host cell strain will be dependent on the nature of the expression construct utilized and the identity of the host celL Recombinant host strams are normally cultured using methods that are known to those of ordinary skill in the art. Recombinant host cells are typically cultured in liquid medium containing assimilatable sources of carbon, nitrogen, and inorganic salts and, optionally, containing vitamins, amino acids, growth factors, and other proteinaceous culture supplements known to those of ordinary skill in the art. Liquid media for culture of host cells may optionally contain antibiotics or anti-fungals to prevent the growth of undesirable microorganisms and/or compounds including, but not limited to, antibiotics to select for host cells containing the expression vector.<br>
[372]	Recombinant host cells may be cultured in batch or continuous formats, with<br>
either cell harvesting (in the case where the hlFN polypeptide accumulates intracellularly) or harvesting of culture supernatant in either batch or continuous formats. For production in prokaryotic host cells, batch culture and cell harvest are preferred.<br>
[373]	The hlFN polypeptides of the present invention are normally purified after<br>
expression in recombinant systems. Voss et al. Biochem J. (1994) 298:719-725; Rosendahl et al. Bioconjugate Chem (2005) 16:20O-207; Swaminathan et al. Protein Expression and Purification (1999) 15:236-242; and Neves et al. Protein Expression and Purification (2004) 35:353-359, which are incorporated by reference herein, describe methods of expression, purification, isolation and characterization of recombinant interferons. The hlFN polypeptide may be purified from host cells or culture medium by a variety of methods known to the art. hlFN polypeptides produced in bacterial host cells may be poorly soluble or insoluble (in the form of incliision bodies). In one embodiment of the present invention, amino acid substitutions may readily be made in the hlFN polypeptide that are selected for the purpose of increasing the solubility of the recombinantly produced protein utilizing the methods disclosed herein as well as those known in the art. In the case of insoluble protein, the protein may be collected from host cell lysates by centrifugation and may further be followed by homogenization of the cells. In the case of poorly soluble protein, compounds including, but not limited to, polyethylene imine (PEI) may be added to induce the precipitation of partially soluble protein. The precipitated protein may then be conveniently collected by centrifugation. Recombinant host cells may be disrupted or homogenized to release the inclusion bodies from within the cells using a variety of methods known to those of ordinary skill in the art. Host cell disruption or homogenization may be performed using well known techniques including, but not<br><br>
limited to, enzymatic cell disruption, sonication, dounce homogenization, or high pressure release disruption. In one embodiment of the method of the present invention, the high pressure release technique is used to disrupt the E. colt host cells to release the inclusion bodies of the hlFN polypeptides. When handling inclusion bodies of hlFN polypeptide, it may be advantageous to minimize the homogenization time on repetitions in order to maximize the yield of inclusion bodies without loss due to factors such as solubilization, mechanical shearing or proteolysis.<br>
[374]	Insoluble or precipitated hlFN polypeptide may then be solubilized using any of a<br>
number of suitable solubilization agents known to the art. The hlFN polyeptide may be solubilized with urea or guanidine hydrochloride. The volume of the solubilized hlFN polypeptide-BP should be minimized so that large batches may be produced using conveniently manageable batch sizes. This factor may be significant in a large-scale commercial setting where the recombinant host may be grown in batches that are thousands of liters in volume. In addition, when manufacturing hlFN polypeptide in a large-scale commercial setting, in particular for human pharmaceutical uses, the avoidance of harsh chemicals that can damage the machinery and container, or the protein product itself, should be avoided, if possible. It has been shown in the method of the present invention that the milder denaturing agent urea can be used to solubilize the hlFN polypeptide inclusion bodies in place of the harsher denaturing agent guanidine hydrochloride. The use of urea significantly reduces the risk of damage to stainless steel equipment utilized in the manufacturing and purification process of hlFN polypeptide while efficiently solubilizing the hlFN polypeptide inclusion bodies.<br>
[375]	In the case of soluble hlFN protein, the hlFN may be secreted into the<br>
periplasmic space or into the culture medium. In addition, soluble hlFN may be present in the<br>
cytoplasm of the host cells. It may be desired to concentrate soluble hlFN prior to performing<br>
purification steps. Standard techniques known to those of ordinary skill in the art may be used<br>
to concentrate soluble hlFN from, for example, cell lysates or culture medium. In addition,<br>
standard techniques known to those of ordinary skill in the art may be used to disrupt host cells<br>
and release soluble hlFN from the cytoplasm or periplasmic space of the host cells.<br>
[376]	When hlFN polypeptide is produced as a fusion protein, the fusion sequence may<br>
be removed. Removal of a fusion sequence may be accomplished by enzymatic or chemical cleavage. Enzymatic removal of fusion sequences may be accomplished using methods known to those of ordinary slcill in the art. The choice of enzyme for removal of the fusion sequence will be determined by the identity of the fusion, and the reaction conditions will be specified by the choice of enzyme as will be apparent to one of ordinary skill in the art.   Chemical cleavage<br><br>
may be accomplished using reagents known to those of ordinary skill in the art, mcludmg DUI not limited to, cyanogen bromide, TEV protease, and other reagents. The cleaved hlFN polypeptide may be purified from the cleaved fusion sequence by methods known to those of ordinary skill in the art. Such methods mil be determined by the identity and properties of the fusion sequence and the hlFN polypeptide, as will be apparent to one of ordinary skill in the art. Methods for purification may include, but are not limited to, size-exclusion chromatography, hydrophobic interaction chromatography, ion-exchange chromatography or dialysis or any combination thereof.<br>
[377]	The hlFN polypeptide may also be purified to remove DNA from the protein<br>
solution. DNA may be removed by any suitable method known to the art, such as precipitation<br>
or ion exchange chromatography, but may be removed by precipitation with a nucleic acid<br>
precipitating agent, such as, but not limited to, protamine sulfate. The hlFN polypeptide may be<br>
separated from the precipitated DNA using standard well known methods including, but not<br>
limited to, centrifugation or filtration. Removal of host nucleic acid molecules is an important<br>
factor in a setting where the hlFN polypeptide is to be used to treat humans and the methods of<br>
the present invention reduce host cell DNA to pharmaceutically acceptable levels.<br>
[378]	Methods for small-scale or large-scale fermentation can also be used in protein<br>
expression, including but not limited to, fermentors, shake flasks, fluidized bed bioreactors,<br>
hollow fiber bioreactors, roller bottle culture systems, and stirred tank bioreactor systems. Each<br>
of these methods can be performed in a batch, fed-batch, or continuous mode process.<br>
[379]	Human hlFN pol ypeptides of the invention can generally be recovered using<br>
methods standard in the art. For example, culture medium or cell lysate can be centrifuged or filtered to remove cellular debris. The supernatant may be concentrated or diluted to a desired volume or diafiltered into a suitable buffer to condition the preparation for further purification. Further purification of the hlFN polypeptide of the present invention includes separating deamidated, clipped, acetylated, and oxidized forms of the hlFN polypeptide variant from the intact form.<br>
[380]	Any of the following exemplary procedures can be employed for purification of<br>
hlFN polypeptides of the invention: affinity chromatography; anion- or cation-exchange chromatography (using, including but not limited to, DEAE SEPHAROSE); chromatography on silica; High Performance Liquid Chromatography (HPLC); reverse phase HPLC; gel filtration (using, including but not limited to, SEPHADEX G-75); hydrophobic interaction chromatography; size-exclusion chromatography, metal-chelate chromatography; ultrafiltration/diafiltration;     ethanol     precipitation;      ammonium     sulfate     precipitation;<br><br>
chromatofocusing; displacement chromatography; electrophoretic procedures (including but not limited to preparative isoelectric focusing), differential solubility (including but not limited to ammonium sulfate precipitation), SDS-pAGE, or extraction. "Protein Purification" (1998) Janson et Ryden eds. (A. John Wiley and Sons, Inc. Publishers) describe various methods of chromatography and electrophoresis as well as modifications that can be performed within each method for optimization of purification schemes.<br>
[381]	Proteins  of the present  invention,  including  but  not  limited  to,   proteins<br>
comprising unnatural amino acids, antibodies to proteins comprising unnatural amino acids, binding partners for proteins comprising unnatural amino acids, etc., can be purified, either partially or substantially to homogeneity, according to standard procedures known to and used by those of skill in the art. Accordingly, polypeptides of the invention can be recovered and purified by any of a number of methods knownto those of ordinary skill in the art, including but not limited to, ammonium sulfate or ethanol precipitation, acid or base extmction, column chromatography, affinity column chromatography, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, hydroxylapatite chromatography, lectin chromatography, gel electrophoresis and the like. Protein refolding steps can be used, as desired, in making correctly folded mature proteins. High performance liquid chromatography (HPLC), affinity chromatography or other suitable methods can be employed in final purification steps where high purity is desired. In one embodiment, antibodies made against unnatural amino acids (or proteins comprising unnatural amino acids) are used as purification reagents, including but not limited to, for affinitγ-based purification of proteins comprising one or more unnatural amino acid(s). Once purified, partially or to homogeneity, as desired, the polypeptides are optionally used for a wide variety of utilities, including but not limited to, as assay components, therapeutics, prophylaxis, diagnostics, research reagents, and/or as immunogens for antibody production.<br>
[382]	In addition to other references noted herein, a variety of purification/protein<br>
folding methods are known to those of ordinary skill in the art, including, but not limited to, those set forth in R. Scopes, Protein Purification, Springer-Verlag, N.Y. (1982); Deutscher, Methods in Bnzvmology Vol. 182: Guide to Protein Purification, Academic Press, Inc. N.Y. (1990); Sandana, (1997) Bioseparation of Proteins. Academic Press, Inc.; Bollag et al. (1996) Protein Methods, 2nd Edition Wiley-Liss, NY; Walker, (1996) The Protein Protocols Handbook Humana Press, NJ, Harris and Angal, (1990) Protein Purification Applications: A Practical Approach IRL Press at Oxford, Oxford, England; Harris and Angal, Protein Purification Methods: A Practical Approach IRL Press at Oxford, Oxford, England; Scopes, (1993) Protein<br><br>
Purification: Principles and Practice 3rd Edition Springer Verlag, NY; Janson and Kyden, (iyy»j Protein Purification: Principles, High Resolution Methods and Applications. Second Edition Wiley-VCH, NY; and Walker (1998), Protem Protocols on CD-ROM Humana Press, NJ; and<br>
the references cited therein.<br>
[383]	One advantage of producing a protein or polypeptide of interest with an unnatural<br>
amino acid in a eukaryotic host cell or non-eukaryotic host cell is that typically the proteins or polypeptides will be folded in their native conformations. However, in certain embodiments of the invention, those of skill in the art will recognize that, after synthesis, expression and/or purification, proteins can possess a conformation different from the desired conformations of the relevant polypeptides. In one aspect of the invention, the expressed protein or polypeptide is optionally denatured and then renatured. This is accomplished utilizing methods Icnown in the art, including but not limited to, by adding a chaperonin to the protein or polypeptide of interest, by solubilizing the proteins in a chaotropic agent such as guanidine HCl, utilizing protein disulfide isomerase, etc.<br>
[384]	In general, it is  occasionally desirable to  denature and reduce  expressed<br>
polypeptides and then to cause the polypeptides to re-fold into the preferred conformation. For example, guanidine, urea, DTT, DTE, and/or a chaperonin can be added to a translation product of interest. Methods of reducing, denaturing and renaturing proteins are Icnown to those of ordinary skill in the art (see, the references above, and Debinski, et al. (1993) J. Biol. Chem.. 268: 14065-14070; Kreitman and Pastan (1993) Bioconiug. Chem., 4: 581-585; and Buchner, et al., (1992) Anal. Biochem., 205: 263-270). Debinski, et al., for example, describe the denaturation and reduction of inclusion body proteins in guanidine-DTE. The proteins can be refolded in a redox buffer containing, including but not limited to, oxidized glutathione and L-arginine. Refolding reagents can be flowed or otherwise moved into contact with the one or more polypeptide or other expression product, or vice-versa.<br>
[385]	In the case of prokaryotic production of hlFN polypeptide, the hlFN polypeptide<br>
thus produced may be misfolded and thus lacks or has reduced biological activity. The bioactivity of the protein may be restored by "refolding". In general, misfolded hlFN polypeptide is refolded by solubilizing (where the hlFN polypeptide is also insoluble), unfolding and reducing the polypeptide chain using, for example, one or more chaotropic agents (e.g. urea and/or guanidine) and a reducing agent capable of reducing disulfide bonds (e.g. dithiothreitol, DTT or 2-mercaptoethanol, 2-ME). At a moderate concentration of chaotrope, an oxidizing agent is then added (e.g., oxygen, cystine or cystamine), which allows the reformation of<br><br>
disulfide bonds. hlFN polypeptide may be refolded using standard methods known in the art, such as those described in U.S. Pat. Nos. 4,511,502, 4,511,503, and 4,512,922, which are incorporated by reference herein. The hlFN polypeptide may also be cofolded with other proteins to form heterodimers or heteromultimers.<br>
[386]	After refolding, the hlFN may be further purified.   Purification of hlFN may be<br>
accomplished using a variety of techniques known to those of ordinary skill in the art, including hydrophobic interaction chromatography, size exclusion chromatography, ion exchange chromatography, reverse-phase high performance liquid chromatography, affinity chromatography, and the like or any combination thereof. Additional purification may also include a step of drying or precipitation of the purified protein.<br>
[387]	After  purification,  hlFN  may  be  exchanged  into  different  buffers  and/or<br>
concentrated by any of a variety of methods known to the art, including, but not limited to, diafiltration and dialysis. hlFN that is provided as a single purified protein may be subject to aggregation and precipitation.<br>
[388]	The purified hlFN may be at least 90% pure (as measured by reverse phase high<br>
performance liquid chromatography, RP-HPLC, or sodium dodecyl sulfate-polyacrylamide gel electrophoresis, SDS-pAGE) or at least 95% pure, or at least 98% pure, or at least 99% or greater pure. Regardless of the exact numerical value of the purity of the hlFN, the hlFN is sufficiently pure for use as a pharmaceutical product or for further processing, such as conjugation with a water soluble polymer such as PEG.<br>
[389]	Certain hlFN molecules may be used as therapeutic agents in the absence of other<br>
active ingredients or proteins (other than excipients, carriers, and stabilizers, serum albumin and the like), or they may be complexed with another protein or a polymer.<br>
[390]	Genera] Purification Methods   Any one of a variety of isolation steps may be<br>
performed on the cell lysate, extract, culture medium, inclusion bodies, periplasmic space of the host cells, cytoplasm of the host cells, or other material, comprising hlFN polypeptide or on any hlFN polypeptide mixtures resulting from any isolation steps including, but not limited to, affinity chromatography, ion exchange chromatography, hydrophobic interaction chromatography, gel filtration chromatography, high performance liquid chromatography ("HPLC"), reversed phase-HPLC ("RP-HPLC"), expanded bed adsorption, or any combination and/or repetition thereof and in any appropriate order.<br>
[391]	Equipment and other necessary materials used in performing the techniques<br>
described herein are commercially available. Pumps, fraction collectors, monitors, recorders, and entire systems are available from, for example. Applied Biosystems (Foster City, CA), Bio-<br><br>
Rad Laboratories, Inc. (Hercules, CA), and Amersham Biosciences, Inc. (Piscataway, NJ);. Chromatographic materials including, but not limited to, exchange matrix materials, media, and buffers are also available from such companies.<br>
[392]	Equilibration, and other steps in the column chromatography processes described<br>
herein such as washing and elution, may be more rapidly accomplished using specialized equipment such as a pump. Commercially available pumps include, but are not limited to, HILOAD® Pump P-50, Peristaltic Pump P-1, Pump P-901, and Pump P-903 (Amersham Biosciences, Piscataway, NJ).<br>
[393]	Examples of fraction collectors include Redifrac Fraction Collector, FRAC-100<br>
and FRAC-200 Fraction Collectors, and SUPERFRAC® Fraction Collector (Amersham Biosciences, Piscataway, NJ). Mixers are also available to form pH and linear concentration gradients. Commercially available mixers include Gradient Mixer GM-1 and In-Line Mixers (Amersham Biosciences, Piscataway, NJ).<br>
[394]	The  chromatographic process  may be monitored using  any  commercially<br>
available monitor. Such monitors may be used to gather information like UV, pH, and<br>
conductivity. Examples of detectors include Monitor UV-1, UVICORD® S 11, Monitor UV-M<br>
II, Monitor UV-900. Monitor UPC-900, Monitor pH/C-900, and Conductivity Monitor<br>
(Amersham Biosciences, Piscataway, NJ). Indeed, entire systems are commercially available<br>
including the various AKTA® systems from Amersham Biosciences (Piscataway, NJ).<br>
[395]	In one embodiment of the present invention, for example, the hlFN polypeptide<br>
may be reduced and denatured by first denaturing the resultant purified hlFN polypeptide in urea, followed by dilution into TRIS buffer containing a reducing agent (such as DTT) at a suitable pH. In another embodiment, the hlFN polypeptide is denatured in urea in a concentration range of between about 2 M to about 9 M, followed by dilution in TRIS buffer at a pH in the range of about 5.0 to about 8.0. The refolding mixture of this embodiment may then be incubated. In one embodiment, ±e refolding mixture is incubated at room temperature for four to twentγ-four hours. The reduced and denatured hlFN polypeptide mixture may then be further isolated or purified.<br>
[396]	As stated herein, the pH of the first hlFN polypeptide mixture may be adjusted<br>
prior to performing any subsequent isolation steps. In addition, the first hlFN polypeptide mixture or any subsequent mixture thereof may be concentrated using techniques known in the art. Moreover, the elution buffer comprising the first hlFN polypeptide mixture or any subsequent mixture thereof may be exchanged for a buffer suitable for the next isolation step using techniques known to those of ordinary skill in the art.<br><br>
[397]	Ion Exchange Chromatography    In one embodiment,  and as an optional,<br>
additional step, ion exchange chromatography may be performed on the first hlFN polypeptide mixture. See generally ION EXCHANGE CHROMATOGRAPHY: PRINCIPLES AND METHODS (Cat. No. 18-1114-21, Amersham Biosciences (Piscataway, NJ)). Commercially available ion exchange columns include HTTRAP® HIPREP®, and HILOAD® Columns (Amersham Biosciences, Piscataway, NJ). Such columns utilize strong anion exchangers such as Q SEPHAROSE® Fast Flow, Q SEPHAROSE® High Performance, and Q SEPHAROSE® XL; strong cation exchangers such as SP SEPHAROSE® High Performance, SP SEPHAROSE® Fast Flow, and SP SEPHAROSE® XL; weak anion exchangers such as DEAE SEPHAROSE® Fast Flow; and weak cation exchangers such as CM SEPHAROSE® Fast Flow (Amersham Biosciences, Piscataway, NJ). Anion or cation exchange column chromatography may be performed on the hlFN polypeptide at any stage of the purification process to isolate substantially purified hlFN polypeptide. The cation exchange chromatography step may be performed using any suitable cation exchange matrix. Useful cation exchange matrices mclude, but are not limited to, fibrous, porous, non-poroiis, microgranular, beaded, or cross-linked cation exchange matrix materials. Such cation exchange matrix materials include, but are not limited to, cellulose, agarose, dextran, polyacrylate, polyvinyl, polystyrene, silica, polyether, or composites of any of the foregoing.<br>
[398]	Following adsorption of the hlFN polypeptide to the cation exchanger matrix,<br>
substantially purified hlFN polypeptide may be eluted by contacting the matrix with a buffer having a sufficiently high pH or ionic strength to displace the hlFN polypeptide from the matrix. Suitable buffers for use in high pH elution of substantially purified hlFN polypeptide include, but are not limited to, citrate, phosphate, formate, acetate, HEPES, and MES buffers ranging in concentration from at least about 5 mM to at least about 100 mM.<br>
[399]	Reverse-phase Chromatography RP-HPLC may be performed to purify proteins<br>
following suitable protocols that are known to those of ordinary skill in the art. See, e.g., Pearson et al., ANAL BIOCHEM. (1982) 124:217-230 (1982); Rivier et al., J. CHROM. (1983) 268:112-119; Kunitani et al., J. CHROM. (1986) 359:391-402. RP-HPLC may be performed on the hlFN polypeptide to isolate substantially purified hlFN polypeptide. In this regard, silica derivatized resins with alkyl fimctionalities with a wide variety of lengths, including, but not limited to, at least about C3 to at least about C30, at least about C3 to at least about C20, or at least about C3 to at least about C185 resins may be used. Alternatively, a polymeric resin may be used. For example, TosoHaas Amberchrome CG1000sd resin may be used, which is a styrene polymer resin.  Cyano or polymeric resins with a wide variety of alkyl chain lengths may also be used.<br><br>
Furthermore, the RP-HPLC colunm may be washed with a solvent such as ethanol. J he Source RP column is another example of a RP-HPLC column.<br>
[400]	A suitable elution buffer containing an ion pairing agent and an organic modifier<br>
such as methanol, isopropanol, tetrahydrofuran, acetonitrile or ethanol, may be used to elute the hlFN polypeptide from the RP-HPLC column. The most commonly used ion pairing agents include, but are not limited to, acetic acid, formic acid, perchloric acid, phosphoric acid, trifluoroacetic acid, heptafluorobutyric acid, triethylamine, tetramethylammonium, tetrabutylammonium, triethylammonium acetate. Elution may be performed using one or more gradients or isocratic conditions, with gradient conditions preferred to reduce the separation time and to decrease pealc width. Another method involves the use of two gradients with different solvent concentration ranges. Examples of suitable elution buffers for use herein may include, but are not limited to, ammonium acetate and acetonitrile solutions.<br>
[401]	Hydrophobic Interaction Chromatography Purification Techniques Hydrophobic<br>
interaction chromatography (HIC) may be performed on the hlFN polypeptide. See generally HYDROPHOBIC INTERACTION CHROMATOGRAPHY HANDBOOK: PRINCIPLES AND METHODS (Cat. No. 18-102O-90, Amersham Biosciences (Piscataway, NJ) which is incorporated by reference herein. Suitable HIC matrices may include, but are not limited to, alkyl- or aryl-substituted matrices, such as butyl-, hexyl-, octyl- or phenyl-substituted matrices including agarose, cross-linked agarose, sepharose, cellulose, silica, dextran, polystyrene, poly(methacrylate) matrices, and mixed mode resins, includmg but not limited to, a polyethyleneamine resin or a butyl- or phenyl-substituted poly(methacrylate) matrix. Commercially available sources for hydrophobic interaction column chromatography include, but are not limited to, HITRAP®, HIPREP®, and HILOAD® columns (Amersham Biosciences, Piscataway, NJ).<br>
[402]	Briefly, prior to loading, the HIC column may be equilibrated using standard<br>
buffers known to those of ordinary skill in the art, such as an acetic acid/sodium chloride solution or HEPES containing ammonium sulfate. Ammonium sulfate may be used as the buffer for loading the HIC column. After loading the hlFN polypeptide, the column may then washed using standard buffers and conditions to remove unwanted materials but retaining the hlFN polypeptide on the HIC column. The hlFN polypeptide may be eluted with about 3 to about 10 column volumes of a standard buffer, such as a HEPES buffer containing EDTA and lower ammonium sulfate concentration than the equilibrating buffer, or an acetic acid/sodium chloride buffer, among others. A decreasing linear salt gradient using, for example, a gradient of potassiimi phosphate, may also be used to elute the hlFN molecules.   The eluant may then be<br><br>
concentrated, for example, by filtration such as diafiltration or ultrafiltration. Diafiltration may be utilized to remove the salt used to elute the hlFN polypeptide.<br>
[403]	Other Purification Techniques Yet another isolation step using, for example, gel<br>
filtration (GEL FILTRATION: PRINCIPLES AND METHODS (Cat. No. 18-1022-18, Amersham Biosciences, Piscataway, NJ) which is incorporated by reference herein, hydroxyapatite chromatography (suitable matrices include, but are not limited to, Hα-Ultrogel, High Resolution (Calbiochem), CHT Ceramic Hydroxyapatite (BioRad), Bio - Gel HTP Hydroxyapatite (BioRad)), HPLC, expanded bed adsorption, ultrafiltration, diafiltration, lyophilization, and the like, may be performed on the first hlFN polypeptide mixture or any subsequent mixture thereof, to remove any excess salts and to replace the buffer with a suitable buffer for the next isolation step or even formulation of the final drug product.<br>
[404]	The yield of hlFN polypeptide, including substantially purified hlFN polypeptide,<br>
may be monitored at each step described herein using techniques Icnown to those of ordinary skill in the art Such techniques may also be used to assess the yield of substantially purified hlFN polypeptide following the last isolation step. For example, the yield of hlFN polypeptide may be monitored using any of several reverse phase high pressure liquid chromatography columns, having a variety of alkyl chain lengths such as cyano RP-HPLC, C18RP-HPLC; as well as cation exchange HPLC and gel filtration HPLC.<br>
[4051	III specific embodiments of the present invention, the yield of hlFN after each<br>
purification step may be at least about 30%, at least about 35%, at least about 40%, at least about<br>
45%, at least about 50%, at least about 55%, at least about 60%, at least about 65%, at least<br>
about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at<br>
least about 91%, at least about 92%, at least about 93%, at least about 94%, at least about 95%,<br>
at least about 96%, at least about 97%, at least about 98%, at least about 99%, at least about<br>
99.9%, or at least about 99.99%, of the hlFN in the starting material for each purification step.<br>
[406]	Purity may be determined using standard techniques, such as SDS-pAGE, or by<br>
measuring hlFN polypeptide using Western blot and ELISA assays. For example, polyclonal antibodies may be generated against proteins isolated from negative control yeast fermentation and the cation exchange recovery. The antibodies may also be used to probe for the presence of contaminating host cell proteins.<br>
[407]	RP-HPLC material Vydac C4 (Vydac) consists of silica gel particles, the surfaces<br>
of vfhich carry C4-alkyl chains. The separation of hlFN polypeptide fi-om the proteinaceous impurities is based on differences in the strength of hydrophobic interactions. Elution is performed with an acetonitrile gradient in diluted trifluoroacetic acid.   Preparative HPLC is<br><br>
performed using a stainless steel column (filled with 2.8 to 3.2 liter of Vydac C4 silicagel). The Hydroxyapatite Ultrogel eluate is acidified by adding trifluoroacetic acid and loaded onto the Vydac C4 column. For washing and elution an acetonitrile gradient in diluted trifluoroacetic acid is used. Fractions are collected and immediately neutralized with phosphate buffer. The hlFN pol}T)eptide fractions which are within the IPC limits are pooled.<br>
[408]	DEAE Sepharose (Pharmacia) material consists of diethylaminoethyl (DEAE)-<br>
groups which are covalently bound to the surface of Sepharose beads. The binding of hlFN polypeptide to the DEAE groups is mediated by ionic interactions. Acetonitrile and trifluoroacetic acid pass through the column without being retained. After these substances have been washed off, trace impurities are removed by washing the column with acetate buffer at a low pH. Then the column is washed with neutral phosphate buffer and hlFN polypeptide is eluted with a buffer with increased ionic strength. The column is packed with DEAE Sepharose fast flow. The column volume is adjusted to assure a hlFN polypeptide load in the range of 3-10 mg hlFN polypeptide/ml gel. The column is washed with water and equilibration buffer (sodium/potassium phosphate). The pooled fi-actions of the HPLC eluate are loaded and the column is washed with equilibration buffer. Then the column is washed with washing buffer (sodium acetate buffer) followed by washing with equilibration buffer. Subsequently, hlFN polypeptide is eluted from the column with elution buffer (sodium chloride, sodium/potassium phosphate) and collected in a single fraction in accordance with the master elution profile. The eluate of the DEAE Sepharose column is adjusted to the specified conductivity. The resulting drug substance is sterile filtered into Teflon bottles and stored at -70°C.<br>
[409]	Additional methods that may be employed include, but are not limited to, steps to<br>
remove endotoxins. Endotoxins are lipopolγ-saccharides (LPSs) which are located on the outer membrane of Gram-negative host cells, such as, for example, Escherichia coli. Methods for reducing endotoxin levels are known to one of ordinary skill in the art and include, but are not limited to, purification techniques using silica supports, glass powder or hydroxyapatite, reverse-phase, affinity, size-exclusion, anion-exchange chromatography, hydrophobic interacttion chromatography, a combination of these methods, and the like. Modifications or additional methods may be required to remove contaminants such as co-migrating proteins from the polypeptide of interest. Methods for measuring endotoxin levels are known to one of ordinary skill in the art and include, but are not limited to, Limulus Amebocyte Lysate (LAL) assays. The EndosafeTM-pTS assay is a colorimetric, single tube system that utilizes cartridges preloaded with LAL reagent, chromogenic substrate, and control standard endotoxin along with a handheld<br><br>
spectrophotometer. Alternate methods include, but are not limited to, a Kinetic LAL method that is turbidimetric and uses a 96 well format.<br>
[410]	A wide variety of methods and procedures can be used to assess the yield and<br>
purity of a hlFN protein comprising one or more non-naturally encoded amino acids, including but not limited to, the Bradford assay, SDS-pAGE, silver stained SDS-pAGE, coomassie stained SDS-pAGE, mass spectrometry (including but not limited to, MALDI-TOF) and otlier methods for characterizing proteins known to one of ordinary skill in the art.<br>
[411]	Additional methods include, but are not limited to: SDS-pAGE coupled with<br>
protein staining methods, immunoblotting, matrix assisted laser desorption/ionization-mass<br>
spectrometry (MALDI-MS), liquid chromatography/mass spectrometry, isoelectric focusing,<br>
analytical anion exchange, chromatofocusing, and circular dichroism.<br>
VIIL	Expression in Alternate Systems<br>
[412]	Several strategies have been employed to introduce unnatural amino acids into<br>
proteins in non-recombinant host cells, mutagenized host cells, or in cell-free systems. These systems are also suitable for use in making the hlFN polypeptides of the present invention. Derivatization of amino acids with reactive side-chains such as Lys, Cys and Tyr resulted in the conversion of lysine to N -acetyl-lysine. Chemical synthesis also provides a straightforward method to incorporate unnatural amino acids. With the recent development of enzymatic ligation and native chemical ligation of peptide fragments, it is possible to make larger proteins. See, e.g., P. E. Dawson and S. B. H. Kent, Annu. Rev. Biochem. 69:923 (2000). Chemical peptide ligation and native chemical ligation are described in U.S. Patent No. 6,184,344, U.S. Patent Publication No. 2004/0138412, U.S. Patent Publication No. 2003/0208046, WO 02/098902, and WO 03/042235, which are incorporated by reference herein. A general in vitro biosynthetic method in which a suppressor tRNA chemically acylated with the desired unnatural amino acid is added to an in vitro extract capable of supporting protein biosynthesis, has been used to site-specifically incorporate over 100 unnatural amino acids into a variety of proteins of virtually any size. See, e,g,, V. W. Cornish, D. Mendel and P. G. Schultz, Angew. Chem. Int. Ed. Engl.. 1995, 34:621 (1995); C.J. Noren, S.J. Anthonγ-Cahill, M.C. Griffith, P.G. Schultz, A general method for site-specific incorporation of unnatural amino acids into proteins, Science 244:182-188 (1989); and, J.D. Bain, C.G. Glabe, T.A. Dix, A.R. Chamberlin, E.S. Diala, Biosynthetic site-specific incorporation of a non-natural amino acid into a polypeptide, J. Am. Chem. Soc. 111:8013-8014(19891 Abroad range of functional groups has been introduced into proteins for studies of protein stability, protein folding, enzyme mechanism, and signal transduction.<br><br>
[413]	An in vivo method, termed selective pressure incorporation, was developed to<br>
exploit the promiscuity of wild-type synthetases. See, e.g., N, Budisa, C. Mmks, S. Alefelder, W. Wenger, F. M. Dong, L. Moroder and R. Huber, FASEB J., 13:41 (1999). An auxotrophic strain, in which the relevant metabolic pathway supplying the cell with a particular natural amino acid is switched off, is grown in minimal media containing limited concentrations of the natural amino acid, while transcription of the target gene is repressed. At the onset of a stationary growth phase, the natural amino acid is depleted and replaced with the unntural amino acid analog. Induction of expression of the recombinant protein results in the accumulation of a protein containing the unnatural analog. For example, using this strategy, o, m and p-fluorophenylalanines have been incorporated into proteins, and exhibit two characteristic shoulders in the UV spectrum which can be easily identified, see, e.g., C. Minks, R. Huber, L. Moroder and N. Budisa, Anal. Biochem., 284:29 (2000); trifluoromethionine has been used to replace methionine in bacteriophage T4 lysozyme to study its interaction with chitooligosaccharide ligands by 19F NMR, see, e.g., H. Duewel, E. Daub, V. Robinson and J. F. Honek, Biochemistry. 36:3404 (1997); and trifluoroleucine has been incorporated in place of leucine, resulting in increased thermal and chemical stability of a leucine-zipper protein. See, e.g., Y. Tang, G. Ghirianda, W. A. Petka, T. Nakajima, W. F. DeOrado and D. A. Tirrell, Angew. Chem. Int. Ed. Engl.. 40:1494 (2001). Moreover, selenomethionine and telluromethionine are incorporated into various recombinant proteins to facilitate the solution of phases in X-ray crystallography. See, e.g., W, A. Hendrickson, J. R. Horton and D. M. Lemaster, EMBO J., 9:1665 (1990); J. O. Boles, K. Lewmski, M. Kunkle, J. D. Odom, B. Dunlap, L. Lebioda and M. Hatada, Nat. Struct. Biol.. 1:283 (1994); N. Budisa, B. Steipe, P. Demange, C. Eckerskom, J. Kellermann and R. Huber, Eur. J. Biochem,. 230:788 (1995); and, N. Budisa, W. Kambrock, S. Steinbacher, A. Humm, L. Prade, T. Neuefeind, L. Moroder and R. Huber, J. Mol. Biol.. 270:616 (1997). Methionine analogs with alkene or alkyne functionalities have also been incorporated efficiently, allowing for additional modification of proteins by chemical means. See, e.g., J. C. van Hest and D. A. Tirrell, FEES Lett,, 428:68 (1998); J. C. van Hest, K. L. Kiick and D. A. Tirrell, J. Am. Chem. Soc. 122:1282 (2000); and, K, L. Kiick and D. A. Tirrell, Tetrahedron, 56:9487 (2000); U.S. Patent No. 6,586,207; U.S. Patent Publication 2002/0042097, which are incorporated by reference herein.<br>
[414]	The success of this method depends on the recognition of the unnatural amino<br>
acid analogs by aminoacyl-tRNA synthetases, which, in general, require high selectivity to insure the fidelity of protein translation. One way to expand the scope of this method is to relax the substrate specificity of aminoacyl-tRNA synthetases, which has been achieved in a limited<br><br>
number of cases. For example, replacement of Ala by Gly in Escherichia coll phenylalanyl-tRNA synthetase (PheRS) increases the size of substrate binding pocket, and results in the acyiation of tRNAPhe by p-Cl-phenylalanine (p-Cl-phe). See, M. Ibba, P. Kast and H. Hennecke, Biochemistry, 33:7107 (1994). An Escherichia coli strain harboring this mutant PheRS allows the incorporation of p-Cl-phenylaianine or p-Br-phenylalanine in place of phenylalanine. See, e.g., M. Ibba and H. Hennecke, FEBS Lett., 364:272 (1995); and, N. Shaima, R. Furter, P. Kast and D. A. Tirrell, FEBS Lett., 467:37 (2000). Similarly, a point mutation Phel30Ser near the amino acid binding site of Escherichia coli tyrosyl-tRNA synthetase was shown to allow azatyrosine to be incorporated more efficiently than tyrosine. See, F. Hamano-Talcaku, T. Iwama, S. Saito-γano, K. Talcaku, Y. Monden, M. Kitabatake, D. Soli and S. Nishimura, J. Biol. Chem., 275:40324 (2000).<br>
[415]	Another strategy to incorporate unnatural amino acids into proteins in vivo is to<br>
modify synthetases that have proofreading mechanisms. These synthetases cannot discriminate and therefore activate amino acids that are structurally similar to the cognate natural amino acids. This error is corrected at a separate site, which deacylates the mischarged amino acid from the tRNA to maintain the fidelity of protein translation. If the proofreading activity of the synthetase is disabled, structural analogs thai are misactivated may escape the editing function and be incorporated. This approach has been demonstrated recently with the valyl-tRNA synthetase (ValRS). See, V. Doring, H. D. Mootz, L. A. Nangle, T. L. Hendriclcson, V. de Crecγ-Lagard, P. Schimmel and P. Marliere, Science, 292:501 (2001). ValRS can misaminoacylate tRNAVal with Cys, Thr, or aminobutyrate (Abu); these noncognate amino acids are subsequently hydrolyzed by the editing domain. After random mutagenesis of the Escherichia coli chromosome, a mutant Escherichia coli strain was selected that has a mutation in the editing site of ValRS. This edit-defective ValRS incorrectly charges tRNAVal with Cys. Because Abu stencally resembles Cys (-SH group of Cys is replaced with -CHS in Abu), the mutant ValRS also incorporates Abu into proteins when this mutant Escherichia coli strain is grown in the presence of Abu. Mass spectrometric analysis shows that about 24% of valines are replaced by Abu at each valine position in the native protein.<br>
[416]	Solid-phase synthesis and semisynthetic methods have also allowed for the<br>
synthesis of a number of proteins containing novel amino acids. For example, see the following publications and references cited within, which are as follows: Crick, F.H.C., Barrett, L. Brenner, S. Watts-Tobin, R General nature of the genetic code for proteins. Nature. 192:1227-1232 (1961); Hofmann, K., Bohn, H. Studies on polypeptides. XXXVL The effect of pyrazole-imidazole replacements on the S-protein activating potency of an S-peptide fragment, J. Am<br><br>
Chem, 88(24):5914-5919 (1966); Kaiser, E,T. Synthetic approaches to biologically active peptides and proteins including envzmes. Ace Chem Res. 22:47-54 (1989); Nakatsuka, T., Sasaki, T., Kaiser, E.T. Peptide segment coupling catalyzed by the semisynthetic enzyme thiosubtilisin, J Am Chem See. 109:3808-3810 (1987); Schnolzer, M., Kent, S B H. Constructing proteins by dovetailing unprotected synthetic peptides: backbone-engineered HIV protease. Science. 256(5054):221-225 (1992); Chailcen, I.M. Semis}mthetic peptides and proteins, CRC Crit Rev Biochem. 11(3):255-301 (1981); OfFord, R.E. Protein engineering by chemical means? Protein Eng.. 1(3):151-157 (1987); and, Jackson, D.Y., Bumier, J., Quan, C, Stanley, M., Tom, J., Wells, J.A. A Designed Peptide Ligasefor Total Synthesis of Ribonuclease A with Unnatural Catalytic Residues, Science, 266(5183):243 (1994).<br>
[417]	Chemical modification has been used to introduce a variety of unnatural side<br>
chains, including cofactors, spin labels and oligonucleotides into proteins in vitro. See, e.g., Corey, D.R., Schultz, P.O. Generation of a hybrid sequence-specific single-stranded deoxyribonuclease, Science, 238(4832):1401-1403 (1987); Kaiser, E.T., Lawrence D.S., Rokita, S.E. The chemical modification of enzymatic specificity, Annu Rev Biochem. 54:565-595 (1985); Kaiser, E.T., Lawrence, D.S. Chemical mutation of enyzme active sites, Science. 226(4674):505-511 (1984); Neet, K.E., Nanci A, Koshland, D.E. Properties of thiol-subtilisin. J Biol. Chem, 243(24):6392-6401 (1968); Polgar, L.B. et M.L. Bender, A new enzyme containing a synthetically formed active site. Thiol-subtilisin, J. Am Chem Soc. 88:3153-3154 (1966); and. Pollack, SJ., Nakayama, G. Schultz, P.G. Introduction ofnucleophiles and spectroscopic probes into antibody combinins sites. Science. 242(4881):1038-1040 (1988).<br>
[418]	Alternatively, biosynthetic methods that employ chemically modified aminoacyl-<br>
tRNAs have been used to incorporate several biophysical probes into proteins synthesized in vitro. See the following publications and references cited within: Bruimer, J. NCM' Photolabeling and crosslinking methods, Annu. Rev Biochem, 62:483-514 (1993); and, Krieg, U.C, Walter, P., Hohnson, A.E. Photocrosslinking of the signal sequence of nascent preprolactin of the 54-kilodalton polypeptide of the signal recognition parti.cle, Proc. Natl Acad. Sci. 83(22):8604-8608(1986).<br>
[419]	Previously, it has been shown that unnatural amino acids can be site-specifically<br>
incorporated into proteins in vitro by the addition of chemically aminoacylated suppressor tRNAs to protein synthesis reactions programmed with a gene containing a desired amber nonsense mutation. Using these approaches, one can substitute a number of the common twenty amino acids with close structural homologues, e.g., fluorophenylalanine for phenylalanine, using strains auxotropic for a particular amino acid.  See, e.g., Noren, C.J., Anthonγ-Cahill, Griffith,<br><br>
M.C., Schultz, P.G. A general method for site-specific incorporation of unnatural amino acids into protein. Science. 244; 182-188 (1989); M.W. Nowak, et al., Science 268:439-42 (1995); Bain, J.D., Glabe, C.G., Dix, T.A., Chamberlin, A.R., Diala, E.S. Biosynthetic site-specific Incorporation of a non-natural amino acid into a polypeptide, J. Am Chem Soc, 111 :8013-8014 (1989); N. Budisa et al., FASEB J. 13:41-51 (1999); Ellman, J.A., Mendel, D., Anthonγ-Cahill, S., Noren, CJ., Schultz, P.G. Biosynthetic method for introducing unnatural amino acids site-specifically into proteins. Methods in Enz,. vol. 202, 301-336 (1992); and, Mendel, D., Cornish, V.W. &amp; Schultz, P.G. Site-Directed Mutagenesis with an Expanded Genetic Code, Annu Rev Biophvs. Biomol Struct. 24, 435-62 (1995).<br>
[420]	For example, a suppressor tRNA was prepared that recognized the stop codon<br>
UAG and was chemically aminoacylated with an unnatural amino acid. Conventional site-directed mutagenesis was used to introduce the stop codon TAG, at the site of interest in the protein gene. See, e.g., Sayers, J.R., Schmidt, W. Eckstein, F. 5'-3' Exonucleases in phosphorothioate-based olignoucleotide-directed mutagensis. Nucleic Acids Res. 16(3):791-802 (1988). When the acylated suppressor tRNA and the mutant gene were combined in an in vitro transcription/translation system, the unnatural amino acid was incorporated in response to the UAG codon which gave a protein containing that amino acid at the specified position. Experiments using [ H]-phe and experiments with α-hydroxy acids demonstrated that only the desired amino acid is incorporated at the position specified by the UAG codon and that this amino acid is not incorporated at any other site in the protein. See, e,g„ Noren, et al, supra; Kobayashi et al., (2003) Nature Structural Biology 10(6):425-432; and, Ellman, J.A., Mendel, D., Schultz, P.G. Site-specific incorporation of novel backbone structures into proteins, Science, 255(5041):197-200(1992).<br>
[421J	A tRNA may be aminoacylated with a desired amino acid by any method or<br>
technique, including but not limited to, chemical or enzymatic aminoacylation.<br>
[422]	Aminoacylation may be accomplished by aminoacyl tRNA synthetases or by<br>
other enzymatic molecules, including but not limited to, ribozymes. The term "ribozyme" is interchangeable with "catalytic RNA." Cech and coworkers (Cech, 1987, Science, 236:1532-1539; McCorkle et al., 1987, Concepts Biochem. 64:221-226) demonstrated the presence of naturally occurring RNAs that can act as catalysts (ribozymes). However, although these natural RNA catalysts have only been shown to act on ribonucleic acid substrates for cleavage and sphcing, the recent development of artificial evolution of ribozymes has expanded the repertoire of catalysis to various chemical reactions. Studies have identified RNA molecules that can catalyze aminoacyl-RNA bonds on their own (2')3-termini (Illangakekare et al., 1995 Science<br><br>
267:643-647), and an RNA molecule which can transfer an amino acid from one RNA molecule to another (Lohse et al., 1996, Nature 381:442-444).<br>
[423]	U.S. Patent Application Publication 2003/0228593, which is incorporated by<br>
reference herein, describes methods to construct ribozymes and their use in aminoacylation of tRNAs with naturally encoded and non-naturally encoded amino acids. Substrate-immobilized forms of enzymatic molecules that can aminoacylate tRNAs, including but not limited to, ribozymes, may enable efficient affinity purification of the aminoacylated products. Examples of suitable substrates include agarose, sepharose, and magnetic beads. The production and use of a substrate-immobilized form of ribozyme for aminoacylation is described in Chemistry and Biology 2003, 10:1077-1084 and U.S. Patent Application Publication 2003/0228593, which are incorporated by reference herein.<br>
[424]	Chemical aminoacylation methods  include,  but are not limited to, those<br>
introduced by Hecht and coworkers (Hecht, S. M. Ace. Chem. Res. 1992, 25, 545; Heclcler, T. G.; Roesser, J. R.; Xu, C; Chang, P.; Hecht, S. M. Biochemistry 1988, 27, 7254; Hecht, S. M.; Alford, B. L.; Kuroda, Y,; Kitano, S. J. Biol. Chem. 1978, 253, 4517) and by Schultz, Chamberlin, Dougherty and others (Cornish, V. W.; Mendel, D.; Schultz, P. G. Angew. Chem. Int. Ed. Engl. 1995, 34, 621; Robertson, S. A.; EUman, J. A.; Schultz, P. G. J. Am. Chem. Soc, 1991, 113, 2722; Noren, C. J.; Anthonγ-Cahill, S. J.; Griffith, M. C; Schultz, P. G. Science 1989, 244, 182; Bain, J. D.; Glabe, C. G.; Dix, T. A.; Chamberlin, A. R. J. Am. Chem. Soc. 1989, 111, 8013; Bain, J. D. et al. Nature 1992, 356, 537; Gallivan, J. P.; Lester, H. A.; Dougherty, D. A. Chem. Biol. 1997, 4, 740; Turcatti, et al. J. Biol. Chem. 1996, 271, 19991; Nowak, M. W. et al. Science, 1995, 268, 439; Saks, M. E. et al. J. Biol. Chem. 1996, 271, 23169; Hohsaka, T. et al. J. Am. Chem. Soc. 1999, 121, 34), which are incorporated by reference herein, to avoid the use of synthetases in aminoacylation. Such methods or other chemical aminoacylation methods may be used to aminoacylate tRNA molecules.<br>
[425]	Methods for generating catalytic RNA may involve generating separate pools of<br>
randomized ribozyme sequences, performing directed evolution on the pools, screening the pools for desirable aminoacylation activity, and selecting sequences of those ribozymes exhibiting desired aminoacylation activity.<br>
[426]	Ribozymes can comprise motifs and/or regions that facilitate acylation activity,<br>
such as a GGU motif and a U-rich region. For example, it has been reported that U-rich regions can facilitate recognition of an amino acid substrate, and a GGU-motif can form base pairs with<br><br>
the 3' termini of a tRNA. In combination, the GGU and motif and U-rich region facilitate simultaneous recognition of both the amino acid and tRNA simultaneously, and thereby facilitate aminoacylation of the 3' terminus of the tRNA.<br>
[427]	Ribozymes can be generated by in vitro selection using a partially randomized<br>
r24mini conjugated with tRNAAsn cccG. followed by systematic engineering of a consensus sequence found in the active clones. An exemplary ribozyme obtained by this method is termed "Fx3 ribozyme" and is described in U.S. Pub. App. No. 2003/0228593, the contents of which is incorporated by reference herein, acts as a versatile catalyst for the synthesis of various aminoacyl-tRNAs charged with cognate non-natural amino acids.<br>
[428]	Immobilization on a substrate may be used to enable efficient affinity purification<br>
of the aminoacylated tRNAs. Examples of suitable substrates include, but are not limited to, agarose, sepharose, and magnetic beads. Ribozymes can be immobilized on resins by taking advantage of the chemical structure of RNA, such as the 3'-cis-diol on the ribose of RNA can be oxidized with periodate to yield the corresponding dialdehyde to facilitate immobilization of the RNA on the resin. Various types of resins can be used including inexpensive hydrazide resins wherein reductive amination malces the interaction between the resin and the ribozyme an irreversible linlcage. Synthesis of aminoacyl-tRNAs can be significantly facilitated by this on-column aminoacylation technique. Kouroulclis et al. Methods 2005; 36:239-4 describe a column-based aminoacylation system.<br>
[429]	Isolation of the aminoacylated tRNAs can be accomplished in a variety of ways.<br>
One suitable method is to elute the aminoacylated tRNAs from a column with a buffer such as a sodium acetate solution with 10 mM EDTA, a buffer containing 50 mM N-(2-hydroxyethyl)piperazine-N'-(3-propanesulfonic acid), 12,5 mM KCl, pH 7.0, 10 mM EDTA, or simply an EDTA buffered water (pH 7.0).<br>
[430]	The aminoacylated tRNAs can be added to translation reactions in order to<br>
incorporate the amino acid with which the tRNA was aminoacylated in a position of choice in a polypeptide made by the translation reaction. Examples of translation systems in which the aminoacylated tRNAs of the present invention may be used include, but are not limited to cell lysates. Cell lysates provide reaction components necessary for in vitro translation of a polypeptide from an input mRNA. Examples of such reaction components include but are not limited to ribosomal proteins, rRNA, amino acids, tRNAs, GTP, ATP, translation initiation and elongation factors and additional factors associated with translation. Additionally, translation systems may be batch translations or compartmentalized translation. Batch translation systems<br><br>
combine reaction components in a single compartment while compartmentalized translation systems separate the translation reaction components from reaction products that can inhibit the translation efficiency. Such translation systems are available commercially.<br>
[431]	Further,  a coupled transcription/translation  system  may  be  used.   Coupled<br>
transcription/translation systems allow for both transcription of an input DNA into a corresponding mRNA, which is in turn translated by the reaction components. An example of a commercially available coupled transcription/translation is the Rapid Translation System (RTS, Roche Inc.). The system includes a mixture containing E. coli lysate for providing translational components such as ribosomes and translation factors. Additionally, an RNA polymerase is included for the transcription of the input DNA into an mRNA template for use in translation. RTS can use compartmentalization of the reaction components by way of a membrane interposed between reaction compartments, including a supply/waste compartment and a transcription/translation compartment.<br>
[432]	Aminoacylation of tRNA may be performed by other agents, including but not<br>
limited to, transferases, polymerases, catalytic antibodies, multi-functional proteins, and the like.<br>
[433]	Lu et al. in Mol Cell. 2001 Oct;8(4):759-69 describe a method in which a protein<br>
is chemically ligated to a synthetic peptide containing unnatural amino acids (expressed protein ligation).<br>
[434]	Microinjection techniques have also been use incorporate unnatural amino acids<br>
into proteins. See, e.g., M. W. Nowak, P. C. Kearney, J. R. Sampson, M. E. Saks, C. G. Labarca, S. K. Silverman, W. G. Zhong, J. Thorson, J. N. Abelson, N. Davidson, P. G. Schultz, D. A. Dougherty and H. A. Lester, Science. 268:439 (1995); and, D. A. Dougherty, Cuir. Qpin. Chem. Biol.. 4:645 (2000). A Xenopus oocyte was coinjected with two RNA species made in vitro: an-mRNA encoding the target protein with a UAG stop codon at the amino acid position of interest and an amber suppressor tRNA aminoacylated with the desired unnatural amino acid. The translational machinery of the oocyte then inserts the unnatural amino acid at the position specified by UAG. This method has allowed in vivo structure-function studies of integral membrane proteins, which are generally not amenable to in vitro expression systems. Examples include the incorporation of a fluorescent amino acid into tachykinin neuroldnin-2 receptor to measure distances by fluorescence resonance energy transfer, see, e.g., G. Turcatti, K. Nemeth, M. D. Edgerton, U. Meseth, F. Talabot, M. Peitsch, J. Knowles, H. Vogel and A. Chollet, I Biol. Chem.. 271:19991 (1996); the incorporation of biotinylated amino acids to identify surface-exposed residues in ion channels, see, e.g., J. P. Gallivan, H. A. Lester and D. A. Dougherty,  Chem.  Biol., 4:739  (1997);  the  use  of caged tyrosine  analogs  to  monitor<br><br>
conformational changes in an ion channel in real time, see, e.g., J. C. Miller, S. K. Silverman, P,<br>
M. England, D, A. Dougherty and H. A. Lester, Neuron, 20:619 (1998); and, the use of alpha<br>
hydroxy amino acids to change ion channel backbones for probing their gating mechanisms. See,<br>
e.g., P. M. England, Y. Zhang, D. A. Dougherty and H. A. Lester, Cell, 96:89 (1999); and, T.<br>
Lu, A. Y. Tmg, J. Mainland, L. Y. Jan, P. G. Schultz and J, Yang, Nat. Neurosci.. 4:239 (2001).<br>
[435]	The ability to incorporate unnatural amino acids directly into proteins in vivo<br>
offers a wide variety of advantages including but not limited to, high yields of mutant proteins, technical ease, the potential to study the mutant proteins in cells or possibly in living organisms and the use of these mutant proteins in therapeutic treatments and diagnostic uses. The ability to mclude unnatural amino acids with various sizes, acidities, nucleophilicities, hydrophobicities, and other properties into proteins can greatly expand our ability to rationally and systematically manipulate the structures of proteins, both to probe protein function and create new proteins or organisms with novel properties,<br>
[436]	In  one  attempt to  site-specifically incorporate  parα-F-phe,  a  yeast  amber<br>
suppressor tRNAPheCUA /phenylalanyl-tRNA synthetase pair was used in a p-F-phe resistant,<br>
Phe auxotrophic Escherichia coli strain. See, e.g., R. Furter, Protein Sci.. 7:419 (1998).<br>
[437]	It may also be possible to obtain expression of a hlFN polynucleotide of the<br>
present invention xising a cell-free (in-vitro) translational system. Translation systems may be cellular or cell-free, and may be prokaryotic or eukaryotic. Cellular translation systems include, but are not limited to, whole cell preparations such as penneabilized cells or cell cultures wherein a desired nucleic acid sequence can be transcribed to mRNA and the mRNA translated. Cell-free translation systems are commercially available and many different types and systems are well-known. Examples of ceil-free systems include, but are not limited to, prokaryotic lysates such as Escherichia coli lysates, and eukaryotic lysates such as wheat germ extracts, insect cell lysates, rabbit reticulocyte lysates, rabbit oocyte lysates and human cell lysates. Eulcaryotic extracts or lysates may be preferred when the resulting protein is glycosylated, phosphorylated or otherwise modified because many such modifications are only possible in eukaryotic systems. Some of these extracts and lysates are available commercially (Promega; Madison, Wis.; Stratagene; La JoUa, Calif.; Amersham; Arlington Heights, 111.; GIBCO/BRL; Grand Island, N.Y.). Membranous extracts, such as the canine pancreatic extracts containing microsomal membranes, are also available which are useful for translating secretory proteins. In these systems, which can include either mRNA as a template (in-vitro translation) or DNA as a template (combined in-vitro transcription and translation), the in vitro synthesis is directed by the ribosomes.   Considerable effort has been applied to the development of cell-free protein<br><br>
expression systems. See, e.g., Kim, D.M. and J.R. Swartz, Biotechnology and Bioengineering, 74 :309-316 (2001); Kim, D.M. and J.R. Swartz, Biotechnology, Letters, 22, 1537-1542, (2000); Kim, D.M., and J.R. Swartz, Biotechnology Progress, 16, 385-390, (2000); Kim, D.M., and J.R. Swartz, Biotechnology and Bioengineering, 66, 18O-188, (1999); and Patnaik, R. and J.R, Swartz, Biotechniques 24, 862-868, (1998); U.S. Patent No. 6,337,191; U.S. Patent Publication No. 2002/0081660; WO 00/55353; WO 90/05785, which are incorporated by reference herein. Another approach that may be applied to the expression of hlFN polypeptides comprising a non-naturally encoded amino acid includes the mRNα-peptide fusion technique. See, e.g., R. Roberts and J. Szostak, Proc. Natl Acad, Set (USA) 94:12297-12302 (1997); A. Frankel, et ai. Chemistry &amp; Biology 10:1043-1050 (2003). In this approach, an mRNA template linlced to puromycin is translated into peptide on the ribosome. If one or more tRNA molecules has been modified, non-natural amino acids can be incorporated into the peptide as well. After the last mRNA codon has been read, puromycin captures the C-terminus of the peptide. If the resulting mRNα-peptide conjugate is found to have interesting properties in an in vitro assay, its identity can be easily revealed from the mRNA sequence. In this way, one may screen libraries of hlFN polypeptides comprising one or more non-naturally encoded amino acids to identify polypeptides having desired properties. More recently, in vitro ribosome translations with purified components have been reported that permit the synthesis of peptides substituted with non-naturally encoded amino acids. See, e.g., A. Forster et al.. Proa, Natl Acad. Sci (USA) 100:6353 (2003).<br>
[438]	Reconstituted translation systems may also be used.    Mixtures of purified<br>
translation factors have also been used successfully to translate mRNA into protein as well as combinations of lysates or lysates supplemented with purified translation factors such as initiation factor-1 (IF-1), IF-2, IF-3 (a or (3), elongation factor T (EF-Tu), or termination factors. Cell-free systems may also be coupled transcription/translation systems wherein DNA is introduced to the system, transcribed into mRNA and the mRNA translated as described in Current Protocols in Molecular Biology (F. M. Ausubel et al. editors, Wiley Interscience, 1993), which is hereby specifically incorporated by reference. RNA transcribed in eukaryotic transcription system may be in the form of heteronuclear RNA (hnRNA) or 5'-end caps (7-meihyl guanosine) and 3'-end poly A tailed mature mRNA, which can be an advantage in certain translation systems. For example, capped mRNAs are translated with high efficiency in the reticulocyte lysate system.<br><br>
IX	Macromolecular Polytners Coupled to hlFN Polypeptides<br>
[439]	Various modifications to the non-natural amino acid polypeptides described<br>
herein can be effected using the compositions, methods, techniques and strategies described herein. These modifications include the incorporation of further functionality onto the non-natural amino acid component of the polypeptide, including but not limited to, a label; a dye; a polymer; a water-soluble polymer; a derivative of polyethylene glycol; a photocrosslinker; a radionuclide; a cytotoxic compound; a drug; an affinity label; a photoaffmity label; a reactive compound; a resin; a second protein or polypeptide or polypeptide analog; an antibody or antibody firagment; a metal chelator; a cofactor; a fatty acid; a carbohydrate; a polynucleotide; a DNA; a RNA; an antisense polynucleotide; a saccharide; a water-soluble dendrimer; a cyclodextrin; an inhibitory ribonucleic acid; a biomaterial; a nanoparticle; a spin label; a fluorophore, a metal-containing moiety; a radioactive moiety; a novel functional group; a group that covalently or noncovalently interacts with other molecules; a photocaged moiety; an actinic radiation excitable moiety; a photoisomerizable moiety; biotin; a derivative of biotin; a biotin analogue; a moiety incorporating a heavy atom; a chemically cleavable group; a photocleavable group; an elongated side chain; a carbon-linked sugar; a redox-active agent; an amino thioacid; a toxic moiety; an isotopically labeled moiety; a biophysical probe; a phosphorescent group; a chemiluminescent group; an electron dense group; a magnetic group; an intercalating group; a chromophore; an energy transfer agent; a biologically active agent; a detectable label; a small molecule; a quantum dot; a nanotransmitter; a radionucleotide; a radiotransmitter; a neutron-capture agent; or any combination of the above, or any other desirable compound or substance. As an illustrative, non-limiting example of the compositions, methods, techniques and strategies described herein, the following description will focus on adding macromolecular polymers to the non-natural amino acid polypeptide with the understanding that the compositions, methods, techniques and strategies described thereto are also applicable (with appropriate modifications, if necessary and for which one of skill in the art could make with the disclosures herein) to adding other functionalities, including but not limited to those listed above.<br>
[440]	A wide variety of macromolecular polymers and other molecules can be linked to<br>
hlFN polypeptides of the present invention to modulate biological properties of the hlFN polypeptide, and/or provide new biological properties to the hlFN molecule. These macromolecular polymers can be linked to the hlFN polypeptide via a naturally encoded amino acid, via a non-naturally encoded amino acid, or any functional substituent of a natural or non-natural amino acid, or any substituent or functional group added to a natural or non-natural amino acid.   The molecular weight of the polymer may be of a wide range, including but not<br><br>
limited to, between about 100 Da and about 100,000 Da or more. The molecular weight of the polymer may be between about 100 Da and about 100,000 Da, mcluding but not limited to, 100,000 Da, 95,000 Da, 90,000 Da, 85,000 Da, 80,000 Da, 75,000 Da, 70,000 Da, 65,000 Da, 60,000 Da, 55,000 Da, 50,000 Da, 45,000 Da, 40,000 Da, 35,000 Da, 30,000 Da, 25,000 Da, 20,000 Da, 15,000 Da, 10,000 Da, 9,000 Da, 8,000 Da, 7,000 Da, 6,000 Da, 5,000 Da, 4,000 Da, 3,000 Da, 2,000 Da, 1,000 Da, 900 Da, 800 Da, 700 Da, 600 Da, 500 Da, 400 Da, 300 Da, 200 Da, and 100 Da In some embodiments, the molecular weight of the polymer is between about 100 Da and 50,000 Da. In some embodiments, the molecular weight of the polymer is between about 100 Da and 40,000 Da. In some embodiments, the molecular weight of the polymer is between about 1,000 Da and 40,000 Da. In some embodiments, the molecular weight of the polymer is between about 5,000 Da and 40,000 Da. In some embodiments, the molecular weight of the polymer is between about 10,000 Da and 40,000 Da. In some embodiments, the molecular weight of the polymer is 30,000 Da, In some embodiments, the molecular weight of the polymer is 40,000 Da.<br>
[441]	The present  invention  provides  substantially  homogenous  preparations  of<br>
polymerrprotein conjugates. "Substantially homogenous" as used herein means that polymer:protein conjugate molecules are observed to be greater than half of the total protein. The polymerrprotein conjugate has biological activity and the present "substantially homogenous" PEGylated hlFN polypeptide preparations provided herein are those which are homogenous enough to display the advantages of a homogenous preparation, e.g., ease in clinical application in predictability of lot to lot pharmacokinetics.<br>
[442]	One may  also choose to prepare a mixture of polymeriprotein conjugate<br>
molecules, and the advantage provided herein is that one may select the proportion of mono-polymerrprotein conjugate to include in the mixture. Thus, if desired, one may prepare a mixture of various proteins with various numbers of polymer moieties attached (i.e., di-, tri-, tetra-, etc.) and combine said conjugates with the mono-polymer:protein conjugate prepared using the methods of the present invention, and have a mixture with a predetermined proportion of mono-polymer:protein conjugates.<br>
[443]	The poljoner selected may be water soluble so that the protein to which it is<br>
attached does not precipitate in an aqueous environment, such as a physiological environment. The polymer may be branched or unbranched. For therapeutic use of the end-product preparation, the polymer will be pharmaceutically acceptable.<br>
[444]	Examples of polymers include but are not limited to polyalkyl ethers and alkoxγ-<br>
capped analogs thereof (e.g., polyoxyethylene glycol, polyoxyethylene/propylene glycol and<br><br>
methoxy or ethoxγ-capped analogs thereof, especially polyoxyethylene glycol, the latter is also<br>
known as polyethyleneglycol or PEG); polyvinylpyrrolidones; polyvinylalkyl ethers;<br>
polyoxazolines, polyalkyl oxazolines and polyhydroxyalkyl oxazolines; polyacrylamides,<br>
polyalkyl	acrylamides,	and	polyhydroxyalkyl	acrylamides	(e.g.,<br>
polyhydroxypropylmethacrylamide and derivatives thereof); polyhydroxyallcyl acrylates; polysialic acids and analogs thereof; hydrophilic peptide sequences; polysaccharides and their derivatives, including dextran and dextran derivatives, e.g., carboxymethyldextran, dextran sulfates, aminodextran; cellulose and its derivatives, e.g., carboxymethyl cellulose, hydroxyalkyl celluloses; chitin and its derivatives, e.g., chitosan, succinyl chitosan, carboxymethylchitin, carboxymethylchitosan; hyaluronic acid and its derivatives; starches; alginates; chondroitin sulfate; albumin; pullulan and carboxymethyl pullulan; polyaminoacids and derivatives thereof, e.g., polyglutamic acids, polylysines, polyaspartic acids, polyaspartamides; maleic anhydride copolymers such as: styrene maleic anhydride copolymer, divinylethyl ether maleic anhydride copolymer; polyvinyl alcohols; copolymers thereof; terpolymers thereof; mixtures thereof; and derivatives of the foregoing.<br>
[445]	The proportion of polyethylene glycol molecules to protein molecules will vary,<br>
as will their concentrations in the reaction mixture. In general, the optimum ratio (in terms of efficiency of reaction in that there is minimal excess unreacted protein or polymer) may be determined by the molecular weight of the polyethylene glycol selected and on the number of available reactive groups available. As relates to molecular weight, typically the higher the molecular weight of the polymer, the fewer number of polymer molecules which may be attached to the protein. Similarly, branching of the polymer should be taken into account when optimizing these parameters. Generally, the higher the molecular weight (or the more branches) the higher the polymer:protein ratio.<br>
[446]	As used herein, and when contemplating PEG: hlFN polypeptide conjugates, the<br>
term "therapeutically effective amount" refers to an amount which gives a decrease in viral levels that provides benefit to a patient. The amount will vary from one individual to another and will depend upon a number of factors, including the overall physical condition of the patient and the underlying cause of the condition. The amount of hlFN polypeptide used for therapy gives an acceptable decrease m viral level. A therapeutically effective amount of the present compositions may be readily ascertained by one of ordinary skill in the art using publicly available materials and procedures.<br>
[447]	The water soluble polymer may be any structural form including but not limited to<br>
linear, forked or branched. Typically, the water soluble polymer is a poly(alkylene glycol), such<br><br>
as poly(ethylene glycol) (PEG), but other water soluble polymers can also be employed.  By<br>
way of example, PEG is used to describe certain embodiments of this invention,<br>
[448]	PEG is a well-known, water soluble polymer that is commercially available or can<br>
be prepared by ring-opening polymerization of ethylene glycol according to methods known to those of ordinary slcil] in the art (Sandler and Karo, Polymer Synthesis, Academic Press, New York, Vol. 3, pages 138-161). The term "PEG" is used broadly to encompass any polyethylene glycol molecule, without regard to size or to modification at an end of the PEG, and can be represented as linked to the hlFN polypeptide by the formula: XO-(CH2CH2O)n-CH2CH2-Y<br>
where n is 2 to 10,000 and X is H or a terminal modification, including but not limited to, a C1-4 allcyl, a protecting group, or a terminal functional group.<br>
[449]	In some cases, a PEG used in the invention terminates on one end with hydrox}' or<br>
methoxy, i.e., X is H or CH3 ("methoxy PEG"). Alternatively, the PEG can terminate with a reactive group, thereby forming a bifunctional polymer. Typical reactive groups can include those reactive groups that are commonly used to react with the functional groups found in the 20 common amino acids (including but not limited to, maleimide groups, activated carbonates (including but not limited to, p-nitrophenyl ester), activated esters (including but not limited to, N-hydroxysuccinimide, p-nitrophenyl ester) and aldehydes) as well as functional groups that are inert to the 20 common amino acids but that react specifically with complementary functional groups present in non-naturally encoded amino acids (including but not limited to, azide groups, alkyne groups). It is noted that the other end of the PEG, which is shown in the above formula by Y, will attach either directly or indirectly to a hlFN polypeptide via a naturallγ-occurring or non-naturally encoded amino acid. For instance, Y may be an amide, carbamate or urea linkage to an amine group (including but not limited to, the epsilon amine of lysine or the N-terminus) of the polypeptide. Alternatively, Y may be a maleimide linkage to a thiol group (including but not limited to, the thiol group of cysteine). Alternatively, Y may be a linkage to a residue not commonly accessible via the 20 common amino acids. For example, an azide group on the PEG can be reacted with an alkylene group on the hlFN polypeptide to form a Huisgcn [3+2] cycloaddition product. Alternatively, an alkyne group on the PEG can be reacted with an azide group present in a non-naturally encoded amino acid to form a similar product In some embodiments, a strong nucleophile (including but not limited to, hydrazine, hydrazide, hydroxylamine, semicarbazide) can be reacted with an aldehyde or ketone group present in a non-naturally encoded amino acid to form a hydrazone, oxime or semicarbazone, as applicable, which in some cases can be further reduced by treatment with an appropriate reducing egent.<br><br>
Alternatively, the strong nucleophile can be incorporated into the hlFN polypeptide via a non-naturally encoded amino acid and used to react preferentially with a ketone or aldehyde group present in the water soluble polymer.<br>
[450]	Any molecular mass for a PEG can be used as practically desired, including but<br>
not limited to, from about 100 Daltons (Da) to 100,000 Da or more as desired (including but not limited to, sometimes 0.1-50 kDa or 1O-40 IcDa). The molecular weight of PEG may be of a wide range, including but not limited to, between about 100 Da and about 100,000 Da or more, PEG may be between about 100 Da and about 100,000 Da, including but not limited to, 100,000 Da, 95,000 Da, 90,000 Da, 85,000 Da, 80,000 Da, 75,000 Da, 70,000 Da, 65,000 Da, 60,000 Da, 55,000 Da, 50,000 Da, 45,000 Da, 40,000 Da, 35,000 Da, 30,000 Da, 25,000 Da, 20,000 Da, 15,000 Da, 10,000 Da, 9,000 Da, 8,000 Da, 7,000 Da, 6,000 Da, 5,000 Da, 4,000 Da, 3,000 Da, 2,000 Da, 1,000 Da, 900 Da, 800 Da, 700 Da, 600 Da, 500 Da, 400 Da, 300 Da, 200 Da, and 100 Da. In some embodiments, PEG is between about 100 Da and 50,000 Da. In some embodiments, PEG is between about 100 Da and 40,000 Da. In some embodiments, PEG is between about 1,000 Da and 40,000 Da. In some embodiments, PEG is between about 5,000 Da and 40,000 Da. In some embodiments, PEG is between about 10,000 Da and 40,000 Da. In some embodiments, the molecular weight of the PEG is 30,000 Da. In some embodiments, the molecular weight of the PEG is 40,000 Da. Branched chain PEGs, including but not limited to, PEG molecules with each chain having a MW ranging from 1-100 kDa (including but not limited to, 1-50 kDa or 5-20 kDa) can also be used. The molecular weight of each chain of the branched chain PEG may be, including but not limited to, between about 1,000 Da and about 100,000 Da or more. The molecular weight of each chain of the branched chain PEG may be between about 1,000 Da and about 100,000 Da, including but not limited to, 100,000 Da, 95,000 Da, 90,000 Da, 85,000 Da, 80,000 Da, 75,000 Da, 70,000 Da, 65,000 Da, 60,000 Da, 55,000 Da, 50,000 Da, 45,000 Da, 40,000 Da, 35,000 Da, 30,000 Da, 25,000 Da, 20,000 Da, 15,000 Da, 10,000 Da, 9,000 Da, 8,000 Da, 7,000 Da, 6,000 Da, 5,000 Da, 4,000 Da, 3,000 Da, 2,000 Da, and 1,000 Da. In some embodiments, the molecular weight of each chain of the branched chain PEG is between about 1,000 Da and 50,000 Da. In some embodiments, the molecular weight of each chain of the branched chain PEG is between about 1,000 Da and 40,000 Da. In some embodiments, the molecular weight of each chain of the branched chain PEG is between about 5,000 Da and 40,000 Da. In some embodiments, the molecular weight of each chain of the branched chain PEG is between about 5,000 Da and 20,000 Da. A wide range of PEG molecules are described in, including but not limited to, the Shearwater Polymers, Inc. catalog, Nektar Therapeutics catalog, incorporated herein by reference.<br><br>
[451]	Generally, at least one terminus of the PEG molecule is available for reaction with<br>
the non-naturallγ-encoded amino acid. For example, PEG derivatives bearing alkyne and azide moieties for reaction with amino acid side chains can be used to attach PEG to non-naturally encoded amino acids as described herein. If the non-naturally encoded amino acid comprises an azide, then the PEG will typically contain either an alkyne moiety to effect formation of the [3+2] cycloaddition product or an activated PEG species (i.e., ester, carbonate) containing a phosphine group to effect formation of the amide linkage. Alternatively, if the non-naturally encoded amino acid comprises an alkyne, then the PEG will typically contain an azide moiety to effect formation of the [3+2] Huisgen cycloaddition product. If the non-naturally encoded amino acid comprises a carbonyl group, the PEG will typically comprise a potent nucleophile (including but not limited to, a hydrazide, hydrazine, hydroxylamine, or semicarbazide functionality) in order to effect formation of corresponding hydrazone, oxime, and semicarbazone linkages, respectively. In other alternatives, a reverse of the orientation of the reactive groups described above can be used, i.e., an azide moiety in the non-naturally encoded amino acid can be reacted with a PEG derivative containing an allcyne.<br>
[452]	In some embodiments, the hlFN polypeptide variant with a PEG derivative<br>
contains a chemical fiinctionality that is reactive with tiie chemical functionality present on the side chain of the non-naturally encoded amino acid.<br>
[453]	The invention provides in some embodiments azide- and acetylene-containing<br>
polymer derivatives comprising a water soluble polymer backbone having an average molecular weight from about 800 Da to about 100,000 Da. The polymer backbone of the water-soluble polymer can be poly(ethylene glycol). However, it should be understood that a wide variety of water soluble polymers including but not limited to poly(ethylene)glycol and other related polymers, including poly(dextran) and poly(propylene glycol), are also suitable for use in the practice of this invention and that the use of the term PEG or poly(ethylene glycol) is intended to encompass and include all such molecules. The term PEG includes, but is not limited to, poly(ethylene glycol) in any of its forms, including bifunctional PEG, multiarmed PEG, derivatized PEG, forked PEG, branched PEG, pendent PEG (i.e. PEG or related polymers having one or more functional groups pendent to the polymer backbone), or PEG with degradable linkages therein.<br>
[454]	PEG is typically clear, colorless, odorless, soluble in water, stable to heat, inert to<br>
many chemical agents, does not hydrolyze or deteriorate, and is generally non-toxic. Poly(ethylene glycol) is considered to be biocompatible, which is to say that PEG is capable of<br><br>
coexistence with living tissues or organisms without causing harm. More specifically, PEG is substantially non-immunogenic, which is to say that PEG does not tend to produce an immune response in the body. When attached to a molecule having some desirable function in the body, such as a biologically active age, the PEntG tends to mask the agent and can reduce or eliminate any immune response so that an organism can tolerate the presence of the agent. PEG conjugates tend not to produce a substantial immune response or cause clotting or other undesirable effects. PEG having the formula - CH2CH2O-(CH2CH2O)„ - CH2CH2", where n is from about 3 to about 4000, typically from about 20 to about 2000, is suitable for use in the present invention. PEG having a molecular weight of from about 800 Da to about 100,000 Da are in some embodiments of the present invention particularly useful as the polymer backbone. The molecular weight of PEG may be of a wide range, including but not limited to, between about 100 Da and about 100,000 Da or more. The molecular weight of PEG may be between about 100 Da and about 100,000 Da, including but not limited to, 100,000 Da, 95,000 Da, 90,000 Da, 85,000 Da, 80,000 Da, 75,000 Da, 70,000 Da, 65,000 Da, 60,000 Da, 55,000 Da, 50,000 Da, 45,000 Da, 40,000 Da, 35,000 Da, 30,000 Da, 25,000 Da, 20,000 Da, 15,000 Da, 10,000 Da, 9,000 Da, 8,000 Da, 7,000 Da, 6,000 Da, 5,000 Da, 4,000 Da, 3,000 Da, 2,000 Da, 1,000 Da, 900 Da, 800 Da, 700 Da, 600 Da, 500 Da, 400 Da, 300 Da, 200 Da, and 100 Da. In some embodiments, the molecular weight of PEG is between about 100 Da and 50,000 Da. In some embodiments, the molecular weight of PEG is between about 100 Da and 40,000 Da. In some embodiments, the molecular weight of PEG is between about 1,000 Da and 40,000 Da. In some embodiments, the molecular weight of PEG is between about 5,000 Da and 40.000 Da. In some embodiments, the molecular weight of PEG is between about 10,000 Da and 40,000 Da. In some embodiments, the molecular weight of PEG is 30,000 Da. In some embodiments, the molecular weight of PEG is 40,000 Da.<br>
[455]	The polymer backbone can be linear or branched. Branched polymer backbones<br>
are generally known in the art. Typically, a branched polymer has a central branch core moiety and a plurality of linear polymer chains linked to the central branch core, PEG is commonly used in branched forms that can be prepared by addition of ethylene oxide to various polyols, such as glycerol, glycerol oligomers, pentaerythritol and sorbitol. The central branch moiety can also be derived from'several amino acids, such as lysine. The branched poly(ethylene glycol) can be represented in general form as R(-pEG-OH)ni in which R is derived from a core moiety, such as glycerol, glycerol oligomers, or pentaerythritol, and m represents the number of arms.<br><br>
Multi-armed PEG molecules, such as those described m U.S. Pat. Nos. 5,932,462; 5,643,575;<br>
5,229,490; 4,289,872; U.S. Pat Appl. 2003/0143596; WO 96/21469; and WO 93/21259, each of<br>
which is incorporated by reference herein in its entirety, can also be used as the polymer<br>
backbone.<br>
[456]	Branched PEG can also be in the form of a forked PEG represented by PEG("-<br>
YCHZ2)n, where Y is a linking group and Z is an activated terminal group linked to CH by a<br>
chain of atoms of defined length.<br>
[457]	Yet another branched form, the pendant PEG, has reactive groups, such as<br>
carboxyl, along the PEG backbone rather than at the end of PEG chains.<br>
[458]	In addition to these forms of PEG, the polymer can also be prepared with weak or<br>
degradable linkages in the backbone. For example, PEG can be prepared with ester linkages in<br>
the polymer backbone that are subject to hydrolysis. As shown below, this hydrolysis results in<br>
cleavage of the polymer into fragments of lower molecular weight:<br>
-pEG-CO2-pEG-+H2O -&gt; PEG-CO2H+HO-pEG"<br>
It is understood by those of ordinary skill in the art that the term poly(ethylene glycol) or PEG<br>
represents or includes all the forms known in the art including but not limited to those disclosed<br>
herein.<br>
[459]	Many other polymers are also suitable for use in the present invention. In some<br>
embodiments, polymer backbones that are water-soluble, with from 2 to about 300 termini, are<br>
particularly useful in the invention. Examples of suitable polymers include, but are not limited<br>
to, other poly(alkyIene glycols), such as poly(propylene glycol) ("PPG"), copolymers thereof<br>
(including but not limited to copolymers of ethylene glycol and propylene glycol), terpolymers<br>
thereof, mixtures thereof, and the like. Although the molecular weight of each chain of the<br>
polymer backbone can vary, it is typically in the range of from about 800 Da to about 100,000<br>
Da, often from about 6,000 Da to about 80,000 Da. The molecular weight of each chain of the<br>
polymer backbone may be between about 100 Da and about 100,000 Da, including but not<br>
limited to, 100,000 Da, 95,000 Da, 90,000 Da, 85,000 Da, 80,000 Da, 75,000 Da, 70,000 Da,<br>
65,000 Da, 60,000 Da, 55,000 Da, 50,000 Da, 45,000 Da, 40,000 Da, 35,000 Da, 30,000 Da,<br>
25,000 Da, 20,000 Da, 15,000 Da, 10,000 Da, 9,000 Da, 8,000 Da, 7,000 Da, 6,000 Da, 5,000<br>
Da, 4,000 Da, 3,000 Da, 2,000 Da, 1,000 Da, 900 Da, 800 Da, 700 Da, 600 Da, 500 Da, 400 Da,<br>
300 Da, 200 Da, and 100 Da. In some embodiments, the molecular weight of each chain of the<br>
polymer backbone is between about 100 Da and 50,000 Da.    In some embodiments, the<br><br>
molecular weight of each chain of the polymer backbone is between about 100 Da and 40,000 Da. In some embodiments, the molecular weight of each chain of the polymer backbone is between about 1,000 Da and 40,000 Da. In some embodiments, the molecular weight of each chain of the polymer backbone is between about 5,000 Da and 40,000 Da. In some embodiments, the molecular weight of each chain of the polymer backbone is between about 10,000 Da and 40,000 Da.<br>
[460]	Those of ordinary skill in the art will recognize that the foregoing list for<br>
substantially water soluble backbones is by no means exhaustive and is merely illustrative, and that all polymeric materials having the qualities described above are contemplated as being suitable for use in the present invention.<br>
[461]	In some embodiments of the present invention the polymer derivatives are<br>
"multi-functional", meaning that the polymer backbone has at least two termini, and possibly as<br>
many as about 300 termini, functionalized or activated with a functional group. Multifunctional<br>
polymer derivatives include, but are not limited to, linear polymers having two termini, each<br>
terminus being bonded to a functional group which may be the same or different<br>
[462]	In one embodiment, the polymer derivative has the structure:<br>
X—A—POLY— B—N-N=N wherein:<br>
N=N=N is an azide moiety;<br>
B is a linking moiety, which may be present or absent;<br>
POLY is a water-soluble non-antigenic polymer;   .<br>
A is a linlcing moiety, which may be present or absent and which may be the same as B or<br>
different; and<br>
X is a second functional group.<br>
Examples of a linking moiety for A and B include, but are not limited to, a multiplγ-functionalized alkyl group containing up to 18, and may contain between 1-10 carbon atoms. A heteroatom such as nitrogen, oxygen or sulfur may be included with the alkyl chain. The alkyl chain may also be branched at a heteroatom. Other examples of a linking moiety for A and B include, but are not limited to, a multiply functionalized aryl group, containing up to 10 and may contain 5-6 carbon atoms. The aryl group may be substituted with one more carbon atoms, nitrogen, oxygen or sulfur atoms.    Other examples of suitable linking groups include those<br><br>
linking groups described in U.S. Pat. Nos. 5,932,462; 5,643,575; and U.S. Pat. Appl. Publication<br>
2003/0143596, each of which is incorporated by reference herein. Those of ordinary skill in the<br>
art will recognize that the foregoing list for linking moieties is by no means exhaustive and is<br>
merely illustrative, and that all linking moieties having the qualities described above are<br>
contemplated to be suitable for use in the present invention.	*<br>
[463]	Examples of suitable functional groups for use as X include, but are not limited<br>
to, hydroxyl, protected hydroxy!, alkoxyl, active ester, such as N-hydroxysuccinimidyl esters<br>
and l-benzotriazolyl esters, active carbonate, such as N-hydroxysuccinimidyl carbonates and 1-<br>
benzotriazolyl carbonates, acetal, aldehyde, aldehyde hydrates, alkenyl, acrylate, methacrylate,<br>
acrylamide, active sulfone, amine, aminooxy, protected amine, hydrazide, protected hydrazide,<br>
protected thiol, carboxylic acid, protected carboxylic acid, isocyanate, isothiocyanate,<br>
maleimide, vinylsulfone, dithiopyridine, vinylp3aidine, iodoacetamide, epoxide, glyoxals,<br>
diones, mesylates, tosylates, tresylate, alkene, ketone, and azide. As is understood by those of<br>
ordinary skill in the art, the selected X moiety should be compatible with the azide group so that<br>
reaction with the azide group does not occur. The azide-containing polymer derivatives may be<br>
homobifunctional, meaning that the second functional group (i.e., X) is also an azide moiety, or<br>
heterobifunctional, meaning that the second functional group is a different functional group.<br>
[464]	The term "protected" refers to the presence of a protecting group or moiety that<br>
prevents reaction of the chemically reactive functional group under certain reaction conditions. The protecting group will vary depending on the type of chemically reactive group being protected. For example, if the chemically reactive group is an amine or a hydrazide, the protecting group can be selected from the group of tert-butyloxycarbonyl (t-Boc) and 9-fluorenylmethoxycarbonyl (Fmoc). If the chemically reactive group is a thiol, the protecting group can be orthopyridyldisulfide. If the chemically reactive group is a carboxylic acid, such as butanoic or propionic acid, or a hydroxyl group, the protecting group can be benzyl or an alkyl group such as methyl, ethyl, or tert-butyl. Other protecting groups known in the art may also be used in the present invention.<br>
[465]	Specific examples of terminal functional groups in the literature include, but are<br>
not limited to, N-succinimidyl carbonate (see e.g., U.S. Pat. Nos. 5,281,698, 5,468,478), amine (see, e.g., Buckmann et al. Malcromol. Chem. 182:1379 (1981), Zalipsky et al. Eur. Polym. J. 19:1177 (1983)), hydrazide (See, e.g., Andresz et al. Makromol. Chem. 179:301 (1978)), succinimidyl propionate and succinimidyl butanoate (see, e.g., Olson et al. in Poly(ethylene<br><br>
glycol) Chemistry &amp; Biological Applications, pp 17O-181, Harris &amp; Zalipsky Eds., ACS,<br>
Washington, D.C., 1997; see also U.S. Pat No. 5,672,662), succinimidyl succinate (See, e.g.,<br>
Abuchowski et al. Cancer Biochem. Biophys. 7:175 (1984) and Joppich et al. Makromol. Chem.<br>
180:1381 (1979), succinimidyl ester (see, e.g., U.S. Pat. No. 4,670,417), benzotriazx)le carbonate<br>
(see, e.g., U.S. Pat. No. 5,650,234), glycidyl ether (see, e.g., Pitha et al. Eur. J Biochem. 94:1,1<br>
(1979), Elling et al., Biotech. Appl. Biochem. 13:354 (1991), oxycarbonylimidazole (see, e.g.,<br>
Beauchamp, et al., Anal Biochem. 131:25 (1983), Tondelli et al. J. Controlled Release 1:251<br>
(1985)), p-nitrophenyl carbonate (see, e.g., Veronese, et al., Appl. Biochem. Biotech., 11: 141<br>
(1985); and Sartore et al., Appl. Biochem. Biotech., 27:45 (1991)), aldehyde (see, e.g., Harris et<br>
al J. Polym. Sci. Chem. Ed. 22:341 (1984), U.S. Pat. No. 5,824,784, U.S. Pat. No. 5,252,714),<br>
maleimide (see, e.g., Goodson et al. Biotechnology (NY) 8:343 (1990), Romani et al. in<br>
Chemistry of Peptides and Proteins 2:29 (1984)), and Kogan, Synthetic Comm. 22:2417 (1992)),<br>
orthopyridyl-disulfide (see, e.g., Woghiren, et al. Bioconj. Chem. 4:314(1993)), acrylol (see,<br>
e.g., Sawhney et al., Macromolecules, 26:581 (1993)), vinylsulfbne (see, e.g., U.S. Pat. No.<br>
5,900,461). All of the above references and patents are incorporated herein by reference.<br>
[466]	In certain embodiments of the present invention, the polymer derivatives of the<br>
invention comprise a polymer backbone having the structure:<br>
X—CH2CH2O--(CH2CH2O)n --CH2CH2 -N=N=N wherein:<br>
X is a functional group as described above; and<br>
n is about 20 to about 4000.<br>
In another embodiment, the polymer derivatives of the invention comprise a polymer backbone<br>
having the structure:<br>
X—CH2CH2O--(CH2CH20)n -CH2CH2 - O-(CH2)m-W-N=N=N wherein:<br>
W is an aliphatic or aromatic linker moiety comprising between 1-10 carbon atoms;<br>
n is about 20 to about 4000; and<br>
X is a functional group as described above, m is between 1 and 10.<br>
[467]	The azide-containing PEG derivatives of the invention can be prepared by a<br>
variety of methods Icnown in the art and/or disclosed herein. In one method, shown below, a<br>
water soluble polymer backbone having an average molecular weight from about 800 Da to<br>
about 100,000 Da, the polymer backbone having a first terminus bonded to a first functional<br>
group and a second terminus bonded to a suitable leaving group, is reacted with an azide anion<br><br>
(which may be paired with any of a number of suitable counter-ions, including sodium, potassium, tert-butylammonium and so forth). The leaving group undergoes a nucleophilic displacement and is replaced by the azide moiety, affording the desired azide-containing PEG polymer.<br>
X-pEG-L + N3" -&gt; X-pEG- N3<br>
[468]	As shown, a suitable polymer backbone for use in the present invention has the<br>
formula X-pEG-L, wherein PEG is poly(ethylene glycol) and X is a functional group which<br>
does not react with azide groups and L is a suitable leaving group. Examples of suitable ,<br>
functional groups include, but are not limited to, hydroxyl, protected hydroxyl, acetal, alkenyl,<br>
amine, ammooxy, protected amine, protected hydrazide, protected thiol, carboxylic acid,<br>
protected carboxylic acid, maleimide, dithiopyridine, and vinylpyridine, and ketone.  Examples<br>
of suitable leaving groups include, but are not limited to, chloride, bromide, iodide, mesylate,<br>
tresylate, and tosylate.<br>
[469]	In another method for preparation of the azide-containing polymer derivatives of<br>
the present invention, a linking agent bearing an azide functionality is contacted with a water<br>
soluble polmyer backbone having an average molecular weight from about 800 Da to about<br>
100,000 Da, wherein the linking agent bears a chemical functionality that will react selectively<br>
with a chemical functionality on the PEG polymer, to form an azide-containing polymer<br>
derivative product wherein the azide is separated from the polymer backbone by a linking group.<br>
[470]	An exemplary reaction scheme is shown below:<br>
X-pEG-M + N-linker-N=N=N -&gt; PG-X-pEG-linlcer-N=N=N<br>
wherein:<br>
PEG is poly(ethylene glycol) and X is a capping group such as alkoxy or a functional group as<br>
described above; and<br>
M is a functional group that is not reactive with the azide functionality but that will react<br>
efficiently and selectively with the N functional group.<br>
[471]	Examples of suitable functional groups include, but are not limited to, M being a<br>
carboxylic acid, carbonate or active ester if N is an amine; M being a ketone if N is a hydrazide<br>
or aminooxy moiety; M being a leaving group if N is a nucleophile.<br>
[472]	Purification of the crude product may be accomplished by knovm methods<br>
including, but are not limited to, precipitation of the product followed by chromatography, if<br>
necessary.<br><br>
[473]	A more specific example is shown below in the case of PEG diamine, in which<br>
one of the amines is protected by a protecting group moiety such as tert-butyl-Boc and the<br>
resulting mono-protected PEG diamine is reacted with a linking moiety that bears the azide<br>
functionality:<br>
B0CHN-pEG-NH2 + HO2C-(CH2)3-N=N=N<br>
[474]	In this instance, the amine group can be coupled to the carboxylic acid group<br>
using a variety of activating agents such as thionyl chloride or carbodiimide reagents and N-<br>
hydroxysuccinimide or N-hydroxybenzotriazole to create an amide bond between the<br>
monoamine PEG derivative and the azide-bearing linker moiety. After successful formation of<br>
the amide bond, the resulting N-tert-butyl-Boc-protected azide-containing derivative can be used<br>
directly to modify bioactive molecules or it can be further elaborated to install other useful<br>
functional groups. For instance, the N-t-Boc group can be hydrolyzed by treatment with strong<br>
acid to generate an omegα-amino-pEG-azide. The resulting amine can be used as a synthetic<br>
handle to install other useful functionality such as maleimide groups, activated disulfides,<br>
activated esters and so forth for the creation of valuable heterobifunctional reagents.<br>
[475]	Heterobifunctional derivatives are particularly useful when it is desired to attach<br>
different molecules to each terminus of the polymer. For example, the omegα-N-amino-N-azido<br>
PEG would allow the attachment of a molecule having an activated electrophilic group, such as<br>
an aldehyde, ketone, activated ester, activated carbonate and so forth, to one terminus of the<br>
PEG and a molecule having an acetylene group to the other terminus of the PEG.<br>
[476]	In another embodiment of the invention, the polymer derivative has the structure:<br>
X—A—POLY— B—C=C-R wherein:<br>
R can be either H or an alkyl, alkene, alkyoxy, or aryl or substituted aryl group;<br>
B is a linlcing moiety, which may be present or absent;<br>
POLY is a water-soluble non-antigenic polymer;<br>
A is a linking moiety, which may be present or absent and which may be the same as B or<br>
different; and<br>
X is a second functional group.<br>
[477]	Examples of a linlcing moiety for A and B include, but are not limited to, a<br>
multiplγ-functionalized alkyl group containing up to 18, and may contain between 1-10 carbon<br>
atoms,  A heteroatom such as nitrogen, oxygen or sulfur may be included with the alkyl chain.<br><br>
The alkyl chain may also be branched at a heteroatom. Other examples of a linking moiety for<br>
A and B include, but are not limited to, a multiply functionalized aryl group, containing up to 10<br>
and may contain 5-6 carbon atoms. The aryl group may be substituted with one more carbon<br>
atoms, nitrogen, oxygen, or sulfur atoms. Other examples of suitable linking groups include<br>
those linking groups described in U.S. Pat. Nos. 5,932,462 and 5,643,575 and U.S. Pat. Appl.<br>
Publication 2003/0143596, each of which is incorporated by reference herein. Those of ordinary<br>
skill in the art will recognize that the foregoing list for linking moieties is by no means<br>
exhaustive and is intended to be merely illustrative, and that a wide variety of linlcing moieties<br>
having the qualities described above are contemplated to be useful in the present invention.<br>
[478]	Examples of suitable functional groups for use as X include hydroxyl, protected<br>
hydroxyl, alkoxyl, active ester, such as N-hydroxysuccinimidyl esters and 1-benzotriazolyl<br>
esters, active carbonate, such as N-hydroxysuccinimidyl carbonates and 1-benzotriazolyl<br>
carbonates, acetal, aldehyde, aldehyde hydrates, alkenyl, acrylate, methacrylate, acrylamide,<br>
active sulfone, amine, aminooxy, protected amine, hydrazide, protected hydrazide, protected<br>
thiol, carboxylic acid, protected carboxylic acid, isocyanate, isothiocyanate, maleimide,<br>
vinylsulfone, dithiopyridine, vinylpyridine, iodoacetamide, epoxide, glyoxals, diones, mesylates,<br>
tosylates, and tresylate, alkene, ketone, and acetylene. As would be understood, the selected X<br>
moiety should be compatible with the acetylene group so that reaction with the acetylene group<br>
does not occur. The acetylene-containing polymer derivatives may be homobifunctional,<br>
meaning that the second functional group (i.e., X) is also an acetylene moiety, or<br>
heterobifunctional, meaning that the second functional group is a different functional group.<br>
[479]	In another embodiment of the present invention,  the polymer derivatives<br>
comprise a polymer backbone having the structure:<br>
X—CH2CH2O-(CH2CH20)n -CH2CH2 - O-(CH2)ni-C≡CH<br>
wherein:<br>
X is a functional group as described above;<br>
n is about 20 to about 4000; and<br>
m is between 1 and 10.<br>
Specific examples of each of the heterobifunctional PEG polymers are shown below.<br>
[480]	The acetylene-containing PEG derivatives of the invention can be prepared using<br>
methods Icnown to those of ordinary skill in the art and/or disclosed herein. In one method, a<br>
water soluble polymer backbone having an average molecular weight from about 800 Da to<br><br>
about 100,000 Da, the polymer backbone having a first terminus bonded to a first functional group and a second tenmnus bonded to a suitable nucleophilic group, is reacted with a compound that bears both an acetylene functionality and a leaving group that is suitable for reaction with the nucleophilic group on the PEG. When the PEG polymer bearing the nucleophilic moiety and the molecule bearing the leaving group are combined, the leaving group undergoes a nucleophilic displacement and is replaced by the nucleophilic moiety, affording the desired acetylene-containing polymer.<br>
X-pEG-Nu + L-a-C -&gt; X-pEG-Nu-a-C≡CR'<br>
[481]	As shown, a preferred polymer backbone for use in the reaction has the formula<br>
X-pEG-Nu, wherein PEG is poly(ethylene glycol), Nu is a nucleophilic moiety and X is a<br>
functional group that does not react with Nu, L or the acetylene functionality.<br>
[482]	Examples of Nu include, but are not limited to, amine,  alkoxy, aryloxy,<br>
sulfhydryl, imino, carboxylate, hydrazide, aminoxy groups that would react primarily via a SN2-type mechanism. Additional examples of Nu groups include those functional groups that would react primarily via an nucleophilic addition reaction. Examples of L groups include chloride, bromide, iodide, mesylate, tresylate, and tosylate and other groups expected to undergo nucleophilic displacement as well as ketones, aldehydes, thioesters, olefins, alphα-beta unsaturated carbonyl groups, carbonates and other electrophilic groups expected to undergo addition by nucleophiles.<br>
[483]	In another embodiment of the present invention, A is an aliphatic linker of<br>
between 1-10 carbon atoms or a substituted aryl ring of between 6-14 carbon atoms. X is a<br>
functional group which does not react with azide groups and L is a suitable leaving group<br>
[484]	In   another   method   for   preparation   of  the   acetylene-containing   polymer<br>
derivatives of the invention, a PEG polymer having an average molecular weight from about 800<br>
Da to about 100,000 Da, bearing either a protected functional group or a capping agent at one<br>
terminus and a suitable leaving group at the other terminus is contacted by an acetylene anion.<br>
[485]	An exemplary reaction scheme is shown below:<br>
X-pEG-L + -C=CR' -&gt; X-pEG-C≡CR'<br>
wherein:<br>
PEG is poly(ethylene glycol) and X is a capping group such as alkoxy or a functional group as<br>
described above; and<br><br>
R' is either H, an alkyl, alkoxy, aryl or aryloxy group or a substituted alkyl, alkoxyl, aryl or aryloxy group.<br>
[486]	In the example above, the leaving group L should be sufficiently reactive to<br>
undergo SN2-type displacement when contacted with a sufficient concentration of the acetylene anion. The reaction conditions required to accomplish SN2 displacement of leaving groups by acetylene anions are known to those of ordinary skill in the art.<br>
[487]	Purification of the crude product can usually be accomplished by methods Icnown<br>
in the art including, but are not limited to, precipitation of the product followed by chromatography, if necessary.<br>
[488]	Water soluble polymers can be linked to the hlFN polypeptides of the invention.<br>
The water soluble polymers may be linked via a non-naturally encoded amino acid incorporated in the hlFN polypeptide or any functional group or substituent of a non-naturally encoded or naturally encoded amino acid, or any fiinctional group or substituent added to a non-naturally encoded or naturally encoded amino acid. Alternatively, the water soluble polymers are linked to a hlFN polypeptide incorporating a non-naturally encoded amino acid via a naturallγ-occurring amino acid (including but not limited to, cysteine, lysine or the amine group of the N-terminal residue). In some cases, the hlFN polypeptides of the invention comprise 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 non-natural amino acids, wherein one or more non-naturallγ-encoded amino acid(s) are linked to water soluble polymer(s) (including but not limited to, PEG and/or oligosaccharides). In some cases, the hlFN polypeptides of the invention further comprise 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, or more naturallγ-encoded amino acid(s) linked to water soluble polymers. In some cases, the hlFN polypeptides of the invention comprise one or more non-naturally encoded amino acid(s) linlced to water soluble polymers and one or more naturallγ-occurring amino acids linked to water soluble polymers. In some embodiments, the water soluble polymers used in the present invention enhance the serum half-life of the hlFN polypeptide relative to the unconjugated form.<br>
[489]	The number of water soluble polymers linked to a hlFN polypeptide (i.e., the<br>
extent of PEGylation or glycosylation) of the present invention can be adjusted to provide an altered (including but not limited to, increased or decreased) pharmacologic, pharmacokinetic or pharmacodynamic characteristic such as in vivo half-life. In some embodiments, the half-life of hlFN is increased at least about 10, 20, 30, 40, 50, 60, 70, 80, 90 percent, 2- fold, 3-fold, 4-fold, 5-fold, 6-fold, 7-fold, 8-fold, 9-fold, 1O-fold, 11-fold, 12-fold, 13-fold, 14-fold, 15-fold, 16-fold,<br><br>
17-fold, 18-fold, 19-fold, 2O-fold, 25-fold, 3O-fold, 35-fold, 4O-fold, 45-fold, 5O-fold, or at least about 10O-fold over an unmodified polypeptide.<br>
PEG derivatives containing a  strong nucleophilic group (i.e.,  hvdrazidc. hydrazine, hydroxvlamine or semicarbazide)<br>
[490]	In one embodiment of the present invention, a hlFN polypeptide comprising a<br>
carbonyi-containing non-naturally encoded amino acid is modified with a PEG derivative that<br>
contains a terminal hydrazine, hydroxylamine,.hydrazide or semicarbazide moiety that is linked<br>
directly to the PEG backbone.<br>
[491]	In some embodiments, the hydroxylamine-terminal PEG derivative will have the<br>
structure:<br>
RO-(CH2CH2O)n-O-(CH2)m-O-NH2<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10 and n is 10O-1,000 (i.e., average<br>
molecular weight is between 5-40 kDa).<br>
[492]	In some embodiments, the hydrazine- or hydrazide-containing PEG derivative will<br>
have the structure:<br>
RO-(CH2CH20)n-O-(CH2)m-X-NH-NH2<br>
where R is a simple alkyl (methyl, ethyl propyl, etc.), m is 2-10 and n is 10O-1,000 and X is<br>
optionally a carbonyl group (C=O) that can be present or absent.<br>
[493]	In some embodiments, the semicarbazide-containing PEG derivative will have the<br>
structure:<br>
RO-(CH2CH20)n -O-(CH2)m-NH-C(O)-NH-NH2<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10 and n is 100-1,000.<br>
[494]	In another embodiment of the invention, a hlFN polypeptide comprising a<br>
carbonyi-containing amino acid is modified with a PEG derivative that contains a terminal<br>
hydroxylamine, hydrazide, hydrazine, or semicarbazide moiety that is linlced to the PEG<br>
backbone by means of an amide linkage.<br>
[495]	In some embodiments, the hydroxylamine-terminal PEG derivatives have the<br>
structure:<br>
RO-(CH2CH2O)n-O-(CH2)2-NH-C(O)(CH2)m-O-NH2<br>
where R is a simple alkyl (meth)n, ethyl, propyl, etc.), m is 2-10 and n is 100-1,000 (i.e., average<br>
molecular weight is between 5-40 kDa).<br>
[496]	In some embodiments, the hydrazine- or hydrazide-containing PEG derivatives<br>
have the structure:<br><br>
RO-(CH2CH2O)„-O-(CH2)2-NH-C(O)(CH2)m-X-NH-NH2<br>
where R is a simple allcyl (methyl, ethyl, propyl, etc.), ra is 2-10, n is 10O-1,000 and X is<br>
optionally a carbonyl group (C=O) that can be present or absent.<br>
[497]	In some embodiments, the semicarbazide-containing PEG derivatives have the<br>
structure:<br>
RO-(CH2CH20)n-O-(CH2)2-NH-C(O)(CH2)ni-NH-C(O)-NH-NH2<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10 and n is 100-1,000.<br>
[498]	In anotlier embodiment of the invention, a hlFN polypeptide comprising a<br>
carbonyl-containing amino acid is modified with a branched PEG derivative that contains a<br>
terminal hydrazine, hydroxylamine, hydrazide or semicarbazide moiety, with each chain of the<br>
branched PEG having a MW ranging from 10-40 kDa and, may be from 5-20 kDa.<br>
[499]	In another embodiment of the invention, a hlFN polypeptide comprising a non-<br>
naturally encoded amino acid is modified with a PEG derivative having a branched structure.<br>
For instance, in some embodiments, the hydrazine- or hydrazide-terminal PEG derivative will<br>
have the following structure:<br>
[RO-(CH2CH20)n-O-(CH2)2-NH-C(O)]2CH(CH2)„.-X-NH-NH2<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10 and n is 10O-1,000, and X is<br>
optionally a carbonyl group (C=0) that can be present or absent.<br>
[500]	In some embodiments, the PEG derivatives containing a semicarbazide group will<br>
have the structure:<br>
[RO-(CH2CH20)n-O-(CH2)2-C(O)-NH-CH2-CH2]2CH-X-(CH2)m-NH-C(O)-NH-NH2<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), X is optionally NH, O, S, C(O) or not<br>
present, m is 2-10 and n is 10O-1,000.<br>
[501]	In some embodiments, the PEG derivatives containing a hydroxylamine group will<br>
have the structure;<br>
[RO-(CH2CH20)n-O-(CH2)2-C(O).NH-CH2-CH2]2CH-X-(CH2)m-O-NH2<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), X is optionally NH, O, S, 0(O) or not<br>
present, m is 2-10 and n is 10O-1,000.<br>
[502]	The degree and sites at which the water soluble polymer(s) are linked to the hlFN<br>
polypeptide can modulate the binding of the hlFN polypeptide to the hlFN polypeptide receptor<br>
at Site 1. In some embodiments, the linkages are arranged such that the hlFN polypeptide binds<br>
the hlFN polypeptide receptor at Site 1 with a Kd of about 400 nM or lower, with a Kd of 150<br>
nM or lower, and in some cases with a Kd of 100 nM or lower, as measured by an equilibrium<br><br>
binding assay, such as that described in Spencer et al, J. Biol Chem., 263:7862-7867 (1988) for hGH.<br>
[503]	Methods and chemistry for activation of polymers as well as for conjugation of<br>
peptides are described in the literature and are known in the art. Commonly used methods for<br>
activation of polymers include, but are not limited to, activation of functional gi'oups with<br>
cyanogen bromide, periodate, glutaraldehyde, biepoxides, epichlorohydrin, divinylsulfone,<br>
carbodiimide, sulfonyl halides, trichlorotriazine, etc. (see, R. F. Taylor, (1991), PROTEFN<br>
IMMOBILISATION. FUNDAMENTAL AND APPLICATIONS, Marcel Dekker, N.Y.; S. S. Wong, (1992),<br>
CHEMISTRY OF PROTEIN CONJUGATION AND CROSSLINKING, CRC Press, Boca Raton; G. T.<br>
Hermanson et al, (1993), IMMOBILIZED AFFINITY LIGAND TECHNIQUES, Academic Press, N.Y.;<br>
Dunn, R.L., et al, Eds. POLYMERIC DRUGS AND DRUG DELIVERY SYSTEMS, ACS<br>
Symposium Series Vol. 469, American Chemical Society, Washington, D,C, 1991).<br>
[504]	Several reviews and monographs on the functionalization and conjugation of PEG<br>
are available. See, for example, Harris, Macromol. Chem, Phys, C25: 325-373 (1985); Scouten, Methods in Enzymology 135: 3O-65 (1987); Wong et al. Enzyme Microh, Technol 14: 866-874 (1992); Delgado et al. Critical Reviews in Therapeutic Drug Carrier Systems 9: 249-304 (1992); Zaiipslcy, Bioconjugate Chem. 6: 15O-165 (1995).<br>
[505]	Methods for activation of polymers can also be found in WO 94/17039, U.S. Pat.<br>
No. 5,324,844, WO 94/18247, WO 94/04193, U.S. Pat. No. 5,219,564, U.S. Pat. No. 5,122,614,<br>
WO 90/13540, U.S. Pat No. 5,281,698, and WO 93/15189, and for conjugation between<br>
activated polymers and enzymes including but not limited to Coagulation Factor VIII (WO<br>
94/15625), hemoglobin (WO 94/09027), oxygen carrying molecule (U.S. Pat. No. 4,412,989),<br>
ribonuclease and superoxide dismutase (Veronese at al, App. Biochem. Biotech, 11: 141-52<br>
(1985)). All references and patents cited are incorporated by reference herein.<br>
[506]	PEGylation (i.e., addition of any water soluble polymer) of hlFN polypeptides<br>
containing a non-naturally encoded amino acid, such as ;?-azido-L-phenylaIanine, is carried out by any convenient method. For example, hlFN polypeptide is PEGylated with an alkyne-terminated mPEG derivative. Briefly, an excess of solid mPEG(5000)-O-CH2-CsCH is added, with stirring, to an aqueous solution of p-azido-L-phe-containing hlFN polypeptide at room temperature. Typically, the aqueous solution is buffered with a buffer having a pKa near the pH at which the reaction is to be carried out (generally about pH 4-10). Examples of suitable buffers for PEGylation at pH 7.5, for instance, include, but are not limited to, HEPES, phosphate, borate, TRIS-HCl, EPPS, and TES. The pH is continuously monitored and adjusted if necessary. The reaction is typically allowed to continue for between about 1-48 hours.<br><br>
[507]	The reaction products are subsequently subjected to hydrophobic interaction<br>
chromatography to separate the PEGylated hlFN polypeptide variants from free mPEG(5000)-O-CH2≡CsCH and any high-molecular weight complexes of the pegylated hlFN polypeptide which may form when unblocked PEG is activated at both ends of the molecule, thereby crosslinking hlFN polypeptide variant molecules. The conditions during hydrophobic interaction chromatography are such that free mPEG(5000)-O-CH2-CsCH flows through the column, while any crosslinked PEGylated hlFN polypeptide variant complexes elute after the desired forms, which contain one hlFN polypeptide variant molecule conjugated to one or more PEG groups. Suitable conditions vary depending on the relative sizes of the cross-linked complexes versus the desired conjugates and are readily determined by those of ordinary skill in the art. The eluent containing the desired conjugates is concentrated by ultrafiltration and desalted by diafiltration.<br>
[508]	If necessary, the PEGylated hlFN polypeptide obtained from the hydrophobic<br>
chromatography can be purified further by one or more procedures known to those of ordinary sldll in the art including, but are not limited to, affinity chromatography; anion- or cation-exchange chromatography (using, including but not limited to, DEAE SEPHAROSE); chromatography on silica; reverse phase HPLC; gel filtration (using, including but not limited to, SEPHADEX G-75); hydrophobic interaction chromatography; size-exclusion chromatography, metal-chelate chromatography; ultrafiltration/diafiltration; ethanol precipitation; ammonium sulfate precipitation; chromatofocusing; displacement chromatography; electrophoretic procedures (including but not limited to preparative isoelectric focusing), differential solubility (including but not limited to ammonium sulfate precipitation), or extraction. Apparent molecular weight may be estimated by GPC by comparison to globular protein standards (Preneta, AZ in PROTEIN PURIFICATION METHODS, A PRACTICAL APPROACH (Harris &amp; Angal, Eds.) IRL Press 1989, 293-306). The purity of the hGH-pEG conjugate can be assessed by proteolytic degradation (including but not limited to, trypsin cleavage) followed by mass spectrometry analysis. Pepinsky RB., et aL, J, Pharmcol &amp; Exp. Ther. 297(3): 1059-66 (2001).<br>
[509]	A water soluble polymer linlced to an amino acid of a hlFN polypeptide of the<br>
invention can be further derivatized or substituted without limitation. Azide-containing PEG derivatives<br>
[510]	In another embodiment of the invention, a hlFN polypeptide is modified with a<br>
PEG derivative that contains an azide moiety that will react with an alkyne moiety present on the side chain of the non-naturally encoded amino acid.   In general, the PEG derivatives will<br><br>
have an average molecular weight ranging from 1-100 kDa and, in some embodiments, from 1O-<br>
40kDa.<br>
[511]	In some embodiments, the azide-terminal PEG derivative will have the structure:<br>
RO-(CH2CH20)„-O-(CH2)n,-N3<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10 and n is 10O-1,000 (i.e., average<br>
molecular weight is between 5-40 kDa).<br>
[512]	In another embodiment, the azide-terminal PEG derivative will have the structure:<br>
RO-(CH2CH2O)n-O-(CH2)m-NH-C(O)-(CH2)p-N3<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10, p is 2-10 and n is  10O-1,000<br>
(i.e., average molecular weight is between 5-40 kDa).<br>
[513]	In another embodiment of the invention, a hlFN polypeptide comprising a alkyne-<br>
containing amino acid is modified with a branched PEG derivative that contains a terminal azide<br>
moiety, with each chain of the branched PEG having a MW ranging fi:om 1O-40 kDa and may be<br>
firom 5-20 kDa.   For instance, in some embodiments, the azide-terminal PEG derivative will<br>
have the following structure:<br>
[RO-(CH2CH2O)n-O-(CH2)2-NH-C(O)]2CH(CH2)Tn-X-(CH2)pN3<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10, p is 2-10, and n is 10O-1,000,<br>
and X is optionally an O, N, S or carbonyl group (C=0), in each case that can be present or<br>
absent.<br>
Alkyne-containing PEG derivatives<br>
[514]	In another embodiment of the invention, a hlFN polypeptide is modified with a<br>
PEG derivative that contains an alkyne moiety that will react with an azide moiety present on<br>
the side chain of the non-naturally encoded amino acid.<br>
[515]	In some embodiments, the alkyne-tennmal PEG derivative will have the following<br>
structure:<br>
RO-(CH2CH2O)n-O-(CH2)m-CsCH<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10 and n is 10O-1,000 (i.e., average<br>
molecular weight is between 5-40 kDa).<br>
[516]	In another embodiment of the invention, a hlFN polypeptide comprising an<br>
alkyne-containing non-naturally encoded amino acid is modified with a PEG derivative that<br>
contains a terminal azide or terminal alkyne moiety that is linked to the PEG backbone by means<br>
of an amide linkage.<br>
[517]	In some embodiments, the alkyne-terminal PEG derivative will have the following<br>
structure:<br><br>
RO-(CH2CH2O)n-O-(CH2)nrNH-C(O)-(CH2)p-C=CH<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10, p is 2-10 and n is 100-1,000.<br>
[518]	In another embodiment of the invention, a hlFN polypeptide comprising an azide-<br>
containing amino acid is modified with a branched PEG derivative that contains a terminal<br>
alkyne moiety, with each chain of the branched PEG having a MW ranging from 10-40 kDa and<br>
may be from 5-20 kDa. For instance, in some embodiments, the alkyne-terminal PEG derivative<br>
will have the following structure:<br>
[RO-(CH2CH20)n-0-(CH2)2-NH-C(0)]2CH(CH2)m-X-(CH2)pC=CH<br>
where R is a simple alkyl (methyl, ethyl, propyl, etc.), m is 2-10, p is 2-10, and n is 100-1,000,<br>
and X is optionally an 0, N, S or carbonyl group (C=0), or not present.<br>
Phosphine-containing PEG derivatives<br>
[519]	In another embodiment of the invention, a hlFN polypeptide is modified with a<br>
PEG derivative that contains an activated fimctional group (including but not limited to, ester,<br>
carbonate) fiirther comprising an aryl phosphine group that will react with an azide moiety<br>
present on the side chain of the non-naturally encoded amino acid.    In general, the PEG<br>
derivatives will have an average molecular weight ranging from 1-100 kDa and, in some<br>
embodiments, from 10-40 kDa.<br>
[520]	In some embodiments, the PEG derivative will have the structure:<br><br>
S(0)2R\ "S(0)2NR'R", -CN and -NO2. R', R", R"' and R"" each independently refer to hydrogen, substituted or unsubstituted heteroalkyl, substituted or unsubstituted aryl, including but not limited to, aryl substituted with 1-3 halogens, substituted or unsubstituted alkyl, alkoxy or thioallcoxy groups, or arylalkyl groups. When a compound of the invention includes more than one R group, for example, each of the R groups is independently selected as are each R\ R", R"' and R'"' groups when more than one of these groups is present.  When R' and R" are<br><br>
attached to the same nitrogen atom, they can be combined with the nitrogen atom to form a 5-, 6-, or 7-membered ring. For example, -NR'R" is meant to include, but not be limited to, 1 -pyrrolidinyl and 4-morpholinyl. From the above discussion of substituents, one of skill in the art will understand that the term "alkyl" is meant to include groups including carbon atoms bound to groups other than hydrogen groups, such as haloallcyl (including but not limited to, -CF3 and -CH2CF3) and acyl (including but not limited to, -C(O)CH3, -C(O)CF3, -C(O)CH2OCH3, and the like).<br>
Other PEG derivatives and General PEGvlation technioues<br>
[522]	Other exemplary PEG molecules that may be linked to hlFN polypeptides, as well<br>
as  PEGylation  methods  include  those  described  in,   e.g.,  U.S.   Patent  Publication  No. 2004/0001838; 2002/0052009; 2003/0162949; 2004/0013637; 2003/0228274; 2003/0220447 2003/0158333; 2003/0143596; 2003/0114647; 2003/0105275; 2003/0105224; 2003/0023023 2002/0156047; 2002/0099133; 2002/0086939; 2002/0082345; 2002/0072573; 2002/0052430 2002/0040076; 2002/0037949; 2002/0002250; 2001/0056171; 2001/0044526; 2001/0021763 U.S. Patent No. 6,646,110; 5,824,778; 5,476,653; 5,219,564; 5,629,384; 5,736,625; 4,902,502 5,281,698;  5,122,614;  5,473,034;  5,516,673;  5,382,657;  6,552,167;  6,610,281;  6,515,100 6,461,603;  6,436,386;  6,214,966;  5,990,237; 5,900,461;  5,739,208;  5,672,662;  5,446,090 5,808,096;  5,612,460;  5,324,844;  5,252,714;  6,420,339;  6,201,072;  6,451,346;  6,306,821 5,559,213; 5,747,646; 5,834,594; 5,849,860; 5,980,948; 6,004,573; 6,129,912; WO 97/32607, EP 229,108, EP 402,378, WO 92/16555, WO 94/04193, WO 94/14758, WO 94/17039, WO 94/18247, WO 94/28024, WO 95/00162, WO 95/11924, WO95/13090, WO 95/33490, WO 96/00080, WO 97/18832, WO 98/41562, WO 98/48837, WO 99/32134, WO 99/32139, WO 99/32140, WO 96/40791, WO 98/32466, WO 95/06058, EP 439 508, WO 97/03106, WO 96/21469, WO 95/13312, EP 921  131, WO 98/05363, EP 809 996, WO 96/41813, WO 96/07670, EP 605 963, EP 510 356, EP 400 472, EP 183 503 and EP 154 316, which are mcorporated by reference herein.  Any of the PEG molecules described herein may be used in any form, including but not limited to, single chain, branched chain, multiarm chain, single functional, bi-functional, multi-functional, or any combination thereof<br>
Enhancing affinity for serum albumin<br>
[523]	Various molecules can also be fused to the hlFN polypeptides of the invention to<br>
modxilate the half-life of hlFN polypeptides in serum. In some embodiments, molecules are linked or fused to hlFN polypeptides of the invention to enhance affinity for endogenous serum albumin in an animal.<br><br>
[524]	For example, m some cases, a recombinant fusion of a hlFN polypeptide and an<br>
albumin binding sequence is made. Exemplary albumin binding sequences include, but are not limited to, the albumin binding domain from streptococcal protein G {see, e.g., Makrides et al, J. Pharmacol Exp, Ther 277:534-542 (1996) and Sjolander et al, J, Immunol Methods 201:115-123 (1997)), or albumin-binding peptides such as those described in, e.g., Dennis, et al,J, Biol Chem, 277:35035-35043 (2002).<br>
[525]	In other embodiments, the hlFN polypeptides of the present invention are acylaled<br>
with fatty acids. In some cases, the fatty acids promote binding to serum albumin. See, e.g., Kurtzhals, etal, Biochem. J. 312:725-731 (1995).<br>
[526]	In other embodiments, the hlFN polypeptides of the invention are fused directly<br>
with serum albumin (including but not limited to, human serum albumin). Those of skill in the<br>
art will recognize that a wide variety of other molecules can also be linked to hlFN in the present<br>
invention to modulate binding to serum albumin or other serum components.<br>
X.	Glycosylation of hlFN Polypeptides<br>
[527]	The invention includes hlFN polypeptides incorporating one or more non-<br>
naturally encoded amino acids bearing saccharide residues. The saccharide residues may be either natural (including but not limited to, N-acetylglucosamine) or non-natural (including but not limited to, 3-fluorogalactose). The saccharides may be linked to the non-naturally encoded amino acids either by an N- or O-linked glycosidic linkage (including but not limited to, N-acetylgalactose-L-serine) or a non-natural linkage (including but not limited to, an oxime or the corresponding C- or S-iinked glycoside).<br>
[528]	The saccharide (including but not limited to, glycosyl) moieties can be added to<br>
hlFN polypeptides either in vivo or in vitro. In some embodiments of the invention, a hlFN<br>
polypeptide comprising a carbonyl-containing non-naturally encoded amino acid is modified<br>
with a saccharide derivatized with an aminooxy group to generate the corresponding<br>
glycosylated polypeptide linked via an oxime linlcage. Once attached to the non-naturally<br>
encoded amino acid, the saccharide may be further elaborated by treatment with<br>
glycosyltransferases and other enzymes to generate an oligosaccharide bound to the hlFN<br>
polypeptide. See, e.g., H. Liu, et al. J. Am. Chem. Soc. 125: 1702-1703 (2003).<br>
[529]	In  some  embodiments  of the  invention,  a hlFN polypeptide comprising  a<br>
carbonyl-containing non-naturally encoded amino acid is modified directly with a glycan with defined structure prepared as an aminooxy derivative. One of ordinary skill in the art will recognize that other functionalities, including azide, alkyne, hydrazide, hydrazine, and semicarbazide, can be used to link the saccharide to the non-naturally encoded amino acid.<br><br>
[530]	In some embodiments of the invention, a hlFN polypeptide comprising an azide or<br>
alkynyl-containing non-naturally encoded amino acid can then be modified by, including but not limited to, a Huisgen [3+2] cycloaddition reaction with, including but not limited to, alkynyl or azide derivatives, respectively. This method allows for proteins to be modified with extremely high selectivity.<br>
XI,	GH Supergene Family Member Dimers and Multimers<br>
[531]	The   present   invention   also   provides   for   GH   supergene   family   member<br>
combinations (including but not limited to hlFN and hlFN analogs) such as homodimers, heterodimers, homomultimers, or heteromultimers (i.e., trimers, tetramers, etc) where a GH supergene family member polypeptide such as hlFN containing one or more non-naturally encoded amino acids is bound to another GH supergene family member or variant thereof or any other polypeptide that is a non-GH supergene family member or variant thereof, either directly to the polypeptide backbone or via a linker. Due to its increased molecular weight compared to monomers, the GH supergene family member, such as hlFN, dimer or multimer conjugates may exhibit new or desirable properties, including but not limited to different pharmacological, pharmacokinetic, pharmacodynamic, modulated therapeutic half-life, or modulated plasma half-life relative to the monomeric GH supergene family member. In some embodiments, the GH supergene family member, such as hlFN, dimers of the invention will modulate the dimerization of the GH supergene family member receptor. In other embodiments, the GH supergene family member dimers or multimers of the present invention will act as a GH supergene family member receptor antagonist, agonist, or modulator.<br>
[532]	In some embodiments, one or more of the hlFN molecules present in a hlFN<br>
containing dimer or multimer comprises a non-naturally encoded amino acid linked to a water soluble polymer that is present within the Site II binding region. As such, each of the hlFN molecules of the dimer or multimer are accessible for binding to the hlFN polypeptide receptor via the Site I interface but are unavailable for binding to a second hlFN polypeptide receptor via the Site 11 interface. Thus, the hlFN polypeptide dimer or multimer can engage the Site I binding sites of each of two distinct hlFN polypeptide receptors but, as the hlFN molecules have a water soluble polymer attached to a non-genetically encoded amino acid present in the Site II region, the hlFN polypeptide receptors cannot engage the Site II region of the hlFN polypeptide ligand and the dimer or multimer acts as a hlFN polypeptide antagonist. In some embodiments, one or more of the hlFN molecules present in a hlFN polypeptide containing dimer or multimer comprises a non-naturally encoded amino acid linked to a water soluble polymer that is present within the Site I binding region, allowing binding to the Site II region.  Alternatively, in some<br><br>
embodiments one or more of the hlFN molecules present in a hlFN polypeptide containing dimer or multimer comprises a non-naturally encoded amino acid linlced to a water soluble polymer that is present at a site that is not within the Site I or Site II binding region, such that both are available for binding. In some embodiments a combination of hlFN molecules is used having Site I, Site II, or both available for bmding. A combination of hlFN molecules wherein at least one has Site I available for binding, and at least one has Site n available for binding may provide molecules having a desired activity or property. In addition, a combination of hlFN molecules having both Site I and Site II available for binding may produce a super-agonist hlFN molecule.<br>
[533]	In some embodiments, the GH supergene family member polypeptides are linlced<br>
directly, including but not limited to, via an Asn-Lys amide linkage or Cys-Cys disulfide linkage. In some embodiments, the linked GH supergene family member polypeptides, and/or the linked non-GH supergene family member, will comprise different non-naturally encoded amino acids to facilitate dimerization, including but not limited to, an alkyne in one non-naturally encoded amino acid of a first hlFN polypeptide and an azide in a second non-naturally encoded amino acid of a second GH supergene family member polypeptide will be conjugated via a Huisgen [3+2] cycloaddition. Alternatively, a first GH supergene family member, and/or the linked non-GH supergene family member, polypeptide comprising a ketone-containing non-naturally encoded amino acid can be conjugated to a second GH supergene family member polypeptide comprising a hydroxylamine-containing non-naturally encoded amino acid and the polypeptides are reacted via formation of the corresponding oxime.<br>
[534]	Alternatively, the two GH supergene family member polypeptides, and/or the<br>
linked non-GH supergene family member, are linked via a linker. Any hetero- or homo-bifunctional linker can be used to link the two GH supergene family members, and/or the linked non-GH supergene family member, polypeptides, which can have the same or different primary sequence. In some cases, the linker used to tether the GH supergene family member, and/or the linked non-GH supergene family member, polypeptides together can be a bifunctional PEG reagent. The linker may have a wide range of molecular weight or molecular length. Larger or smaller molecular weight linkers may be used to provide a desired spatial relationship or conformation between the GH superfamily member and the linked entity. Linkers having longer or shorter molecular length may also be used to provide a desired space or flexibility between the GH superfamily member and the linked entity. Similarly, a linker having a particular shape or conformation may be utilized to impart a particular shape or conformation to the GH superfamily member or the linked entity, either before or after the GH superfamily member<br><br>
reaches its target This optimization of the spatial relationship between the GH superfamily member and the linked entity may provide new, modulated, or desired properties to the molecule.<br>
[535]	In some embodiments, the invention provides water-soluble bifunctional linkers<br>
that have a dumbbell structure that includes: a) an azide, an alkyne, a hydrazine, a hydrazide, a<br>
hydroxylamine, or a carbonyl-containing moiety on at least a first end of a polymer backbone;<br>
and b) at least a second functional group on a second end of the polymer backbone. The second<br>
functional group can be the same or different as the first functional group. The second functional<br>
group, in some embodiments, is not reactive with the first functional group. The invention<br>
provides, in some embodiments, water-soluble compounds that comprise at least one arm of a<br>
branched molecular structure. For example, the branched molecular structure can be dendritic.<br>
[536]	In some embodiments, the invention provides multimers comprising one or more<br>
GH supergene family member, such as hlFN, formed by reactions with water soluble activated polymers that have the structure: R-(CH2CH2O)n-O-(CH2)m-X<br>
wherein n is from about 5 to 3,000, m is 2-10, X can be an azide, an alkyne, a hydrazine, a hydrazide, an aminooxy group, a hydroxylamine, an acetyl, or carbonyl-containing moiety, and R is a capping group, a functional group, or a leaving group that can be the same or different as X. R can be, for example, a functional group selected from the group consisting of hydroxyl, protected hydroxyl, alkoxyl, N-hydroxysuccinimidyl ester, l-benzotriazolyl ester, N-hydroxysuccinimidyl carbonate, l-benzotriazolyl carbonate, acetal, eddehyde, aldehyde hydrates, alkenyl, acrylate, methacrylate, acrylamide, active sulfone, amine, aminooxy, protected amine, hydrazide, protected hydrazide, protected thiol, carboxylic acid, protected carboxylic acid, isocyanate, isothiocyanate, maleimide, vinylsulfone, dithiopyridine, vinylpyridine, iodoacetamide, epoxide, glyoxals, diones, mesylates, tosylates, and tresylate, alkene, and ketone.<br>
XI.	Measurement of hlFN Polypeptide Activity and Affinity of hlFN Polypeptide<br>
for the hIFN Polypeptide Receptor<br>
[537]	The hlFN receptor can be prepared as described in U.S. Patent No. 6,566,132;<br>
5,889,151; 5,861,258; 5,731,169; 5,578,707, which is incorporated by reference herein.   hlFN<br>
polypeptide activity can be determined using standard or known in vitro or in vivo assays.  For<br>
example, cells or cell lines that modulate growth or MHC Class I or II antigen production in<br>
response to hlFN or bind hlFN (mcluding but not limited to, cells containing active IFN<br>
receptors such as human lymphoblastoid Daudi cells, or recombinant IFN receptor producing<br>
cells) can be used to monitor hlFN receptor binding. For a non-pEGylated or PEGylated hlFN<br><br>
polypeptide comprising a non-natural amino acid, the affinity of the hormone for its receptor can be measured by using techniques known in the art such as a BIAcoreTM biosensor (Pharmacia). In vivo animal models as well as human clinical trials for testing hlFN activity include those described in, e.g., Kontsek et al,. Acta Virol. 43:63 (1999); Youngster et al.. Current Pharma Design 8:2139 (2002); Kozlowski et al., BioDrugs 15:419 (2001); U.S. Patent No. 6,180,096; 6,177,074; 6,042,822; 5,981,709; 5,951,974; 5,908,621; 5,711,944; 5,738,846, which are incorporated by reference herein.<br>
[538]	Regardless of which methods are used to create the present hlFN analogs, the<br>
analogs are subject to assays for biological activity. Tritiated thymidine assays may be conducted to ascertain the degree of cell division. Other biological assays, however, may be used to ascertain the desired activity. Biological assays such as assaying for the ability to inhibit viral replication, also provides indication of IFN activity. See Bailon et al., Bioconj. Chem. 12:195 (2001); Forti et al., Meth. Enzymol. 119:533 (1986); Walter et al.. Cancer Biother. &amp; Radiophami. 13:143 (1998); DiMarco et al., BioChem. Biophys. Res. Com. 202:1445 (1994); Foser et al. Pharmacogenomics Journal 3:312-319 (2003); and U.S. Patent No.: 4,675,282; 4,241,174; 4,514,507; 4,622,292; 5,766,864, which are incorporated by reference herein. Assays such as those described by Oritani et al. Nature Medicine 2000 6(6):659-666; Kawamoto et al. Experimental Hematology 2004 32:797-805; and Kawamoto et al. J Virol. 2003 Sep;77(17):9622-31 may be also used to assess the biological activity and potential side effects of hlFN polypeptides of the invention.<br>
[539]	Platanias et al. in Experimental Hematology 1999; 27:1583-1592, which is<br>
incorporated by reference herein, discuss signaling pathways activated by interferons including the Jak-Stat pathway, the insulin receptor substrate (IRS)/PI-3'-kinase pathway, and the activation of CBL and the Crk-signaling pathway. The importance of the Vav proto-oncogene and its role in IFN-dependent growth inhibition and the role of tyrosine phosphatases in type I IFN signaling, as well as molecules critical to signaling cascades with the type II interferon receptor were discussed. Assays evaluating signaling and pathways downstream from interferon receptor binding may be used to evaluate hlFN polypeptides of the invention, including studies described in Platanias et al. in Experimental Hematology 1999 27:1315-1321, which is incorporated by reference herein, that investigate two members of the Crk-familly of proteins, CrkL and Crkll.<br>
[540]	To assess the effect of hlFN polypeptides on normal myeloid (CFU-GM) and<br>
erythroid progenitors (BFU-E), colony formation assays such as those described by Kawamoto et al. Experimental Hematology 2004 32:797-805 or Giron-Michel, Leukemia 2002 16:1135-<br><br>
1142, incorporated by reference herein, may be be used. Clonal proliferation of megakaryocytes<br>
may also be performed. There exists a correlation between colony formation assays and bone<br>
marrow toxicity. In vivo effects on normal hematopoiesis may be measured in, for example,<br>
peripheral blood or bone marrow of mice after injection with hlFN polypeptides or effects on<br>
body temperature in mice may be measured as described by Kawamoto et al, Oligoadenylate<br>
synthetase mRNA may also be used as a biomarker for antiviral activity (branched DNA assay).<br>
[541]	Viral replication assays or in vivo studies may be performed with hlFN<br>
polypeptides of the invention to screen for anti-viral activity. Viral replication assays are known to those skilled in the art and may involve HCV (Hepatitis C Virus), VSV (Vesicular Stomatitis Virus), or EMCV (Encephalomyocarditis Virus). HCV replicon assays involving human WISH or Huh-7 cells may be performed with RNA expression measured by various methods including, but not limited to, RT-pCR, Real Time PCR, or branched DNA methods. Different HCV genotypes may be used, including but not limited to, HCV genotypes la and lb. The reduction of cytopathic effect (CPE) of cells such as baby hamster kidney BHK21 cells infected with VSV may also be measured with hlFN polypeptides. Various cell lines and calculation algorithms may be used to determine potency in CPE assays. Comparisons are made with known reference standards and international units may be calculated. Other assays include, but are not limited to, assays that involve HCV replication in specific ceil lines and human foreskin fibroblast cells infected with EMCV (Trotta et al., Drug Information Journal (2000); 34:1231-1246), which is incorporated by reference herein.<br>
[542]	Other in vitro assays may be used to ascertain biological activity. In general, the<br>
test for biological activity should provide analysis for the desired result, such as increase or<br>
decrease in biological activity (as compared to non-altered IFN), different biological activity (as<br>
compared to non-altered IFN), receptor affinity analysis, or serum half-life analysis.<br>
[543]	It was previously reported that Daudi cells will bind " I-labeled murine IFN and<br>
that this binding can be competed for by addition of unlabeled EFN (See e.g. U.S. Patent No. 5,516,515; 5,632,988). The ability of natural IFN and hlFN to compete for binding of ^^^ I-IFN to human and murine leukemic cells is tested. Highly purified natural IFN (&gt;95% pure; 1 ^.g) is iodinated [Tejedor, et al., Anal.Biochem., 127, 143 (1982)], and is separated fix)m reactants by gel filtration and ion exchange chromatography. The specific activity of the natural ^^^ I-IFN may be appro&gt;dmately 100 |iCi/p,g protein,<br>
[544]	Trotta et al. Drug Information Journal 2000; 34:1231-1246, which is incorporated<br>
by reference herein, discuss the numerous biological activities of IFNα including immunomodulatory, antiproliferative, antiviral, and antimicrobial activities.   Different human<br><br>
IFN a species exhibit varied relative levels of biological activities despite the high degree of amino acid sequence homology. Assays known those skilled in the art to measure immunomodulatory, antiproliferative, antiviral, and antimicrobial activities of one or more IFN a species may be used to evaluate hlFN polypeptides of the invention.<br>
[545]	In vitro assays that measure secretion of prostaglandins, e.g., PGE2, may be<br>
performed to evaluate hlFN polypeptides of the invention. Prostaglandins modulate a number of CNS functions such as the generation of fever and the perception of pain, and one of the side effects of current IFN therapies is fever.<br>
[546]	Transcriptional activity of monoPEGylated interferon-a-2a isomers is described<br>
in Foser et al. Pharmacogenomics Journal 2003; 3:312-319, which is incorporated by reference<br>
herein in its entirety, using oligonucleotide array transcript analysis. Similar assays may be<br>
performed with hlFN polypeptides of the invention. Cell lines that may be used in gene<br>
expression studies include but are not limited to, melanoma cell lines such as ME15. ME 15 or<br>
similar cell lines may also be used to investigate functional properties of hlFN polypeptides of<br>
the invention. Alternate assays to DNA microarrays may be performed with hlFN polypeptides<br>
of the invention to provide mRNA profiling data, differential gene expression information, or<br>
altered gene expression data (change in the level of a transcription or translation products).<br>
Cellular arrays may also be performed. DNA microarray or DNA chip studies involve<br>
assembling PCR products of a group of genes or all genes within a genome on a solid surface in<br>
a high density format or array. General methods for array construction and use are available (see<br>
Schena M, Shalon D, Davis R W, Brown P O., Quantitative monitoring of gene expression<br>
patterns with a complementary DNA microarray. Science. Oct 20, 1995; 270(5235); 467-70). A<br>
DNA microarray allows the analysis of gene expression patterns or profile of many genes to be<br>
performed simultaneously by hybridizing the DNA microarray comprising these genes or PCR<br>
products of these genes with cDNA probes prepared from the sample to be analyzed. DNA<br>
microarray or "chip" technology permits examination of gene expression on a genomic scale,<br>
allowing transcription levels of many genes to be measured simultaneously. Methods and<br>
materials for various arrays and similar analyses are known to those of ordinary skill in the art,<br>
[547]	The above compilation of references for assay methodologies is not exhaustive,<br>
and those skilled in the art will recognize other assays useful for testing for the desired end result. Alterations to such assays are known to those of ordinary skill in the art.<br><br>
XIIL	Measurement  of Potency,  Functional In   Vivo Half-Life,   Toxicity,  and<br>
Pharmacokinetic Parameters<br>
[548]	An important aspect of the invention is the prolonged biological half-life that is<br>
obtained by construction of the hlFN polypeptide with or without conjugation of the polypeptide to a water soluble polymer moiety. The rapid decrease of hlFN polypeptide serum concentrations has made it important to evaluate biological responses to treatment with conjugated and non-conjugated hlFN polypeptide and variants thereof. The conjugated and non-conjugated hlFN polypeptide and variants thereof of the present invention may have prolonged serum half-lives also after subcutaneous or i.v. administration, making it possible to measure by, e.g. ELISA method or by a primary screening assay. ELISA or RIA kits from either BioSource International (Camarillo, CA) or Diagnostic Systems Laboratories (Webster, TX) may be used. Another example of an assay for the measurement of in vivo half-life of IFN or variants thereof is described in Kozlov/ski et al., BioDrugs 15:419 (2001); Ballon et al., Bioconj. Chem. 12:195 (2001); Youngster et al., Current Pharm. Design 8:2139 (2002); U.S. Pat. No. 6,524,570; 6,250,469; 6,180,096; 6,177,074; 6,042,822; 5,981,709; 5,591,974; 5,908,621; 5,738,846, which are incorporated by reference herein. Measurement of in vivo biological half-life is carried out as described herein.<br>
[549]	The potency and functional in vivo half-life of a hlFN polypeptide comprising a<br>
non-naturally encoded amino acid can be determined according to the protocol described in U.S.<br>
Patent No. 5,711,944; 5,382,657, which are incorporated by reference herein.<br>
[550]	Pharmacokinetic parameters for a hlFN polypeptide comprising a non-naturally<br>
encoded amino acid can be evaluated in normal Sprague-Dawley male rats (N=5 animals per<br>
treatment group). Animals will receive either a single dose of 25 ug/rat iv or 50 ug/rat sc, and<br>
approximately 5-7 blood samples will be taken according to a pre-defined time course, generally<br>
covering about 6 hours for a hlFN polypeptide comprising a non-naturally encoded amino acid<br>
not conjugated to a water soluble polymer and about 4 days for a hlFN polypeptide comprising a<br>
non-naturally encoded amino acid and conjugated to a water soluble polymer. Pharmacokinetic<br>
data for hlFN polypeptides is well-studied in several species and can be compared directly to the<br>
data obtained for hlFN polypeptides comprising a non-naturally encoded amino acid. See<br>
Mordenti J., et al, Pharm. Res. 8(11):1351-59 (1991) for studies related to hGR<br>
[551]	Pharmacokinetic parameters can also be evaluated in a primate, e.g., cynomolgus<br>
monkeys. Typically, a single injection is administered either subcutaneously or intravenously, and serum hlFN levels are monitored over time.<br><br>
[552]	Uno et al., which is incorporated by reference herein, in J Interferon Cytokine<br>
Res. 1998 Dec;18(12):1011-8 describe the induction of 2',5' oligoadenylate synthetase in THP-1 cells in response to IFN. In particular, the assay described is sensitive to IFNα and may be useful as a bioassay for serum interferon.<br>
[553]	Animal models used to study aniviral activity of hlFN polypeptides include, but<br>
are not limited to, the chimpanzee HCV (Purcell RH, FEMS Microbiol Rev. 1994 Jul; 14(3): 181-91), tlie HCV-Trimera mouse (Ilan E et al. J Infect Dis. 2002 Jan 15;185(2):153-61), and the Alb-uPA transgenic mouse models (Mercer DF et al. Nat Med. 2001 Aug;7(8):927-33; Kneteman, N et al (2003) 10th HCV Meeting Kyoto Japan, P-187), all of which are incorporated by reference herein. Other animal models may be used to evaluate side effects such as fever, depression, and pain threshold.<br>
[554]	hlFN polypeptides of the invention may be evaluated using a human bone<br>
marrow NOD/SCID reconstitution model. Variations to this protocol are known to one of ordinary skill in the art. With this model, the transplantation protocol is as follows: NOD/SCID mice are housed under sterile conditions in microisolator cages. Just before and for 2 months after total body irradiation mice receive acidified H2O (pH=3). The NOD/SCID mice are sublethally irradiated 325 - 350 cGy using a 137Cs γ-irradiator (approximately 1 cGy/min). Cells are transplanted with human MNC (mononuclear cells from bone marrow or cord blood) by i.v. injection on the day of irradiation. For the reconstitution analysis, at 6-8 weeks post-injection mice are bled through the tail vein, or retro-orbitally, collected into heparinized vacutubes. Peripheral blood cells are analyzed by FACS using anti-human CD45 and anti-mouse CD45 to assess reconstitution. Mice with &gt;0.1% human cells in the peripheral blood are considered positive.<br>
[5551	Ths i^ce are treated v^th PEGASYS® or hlFN polypeptide with a dosing<br>
regimen. Various dosing regimens may be used. Mice are s.c. injected with drug vs. buffer, weeldy dosing. On week 4, the animals are sacrificed and peripheral blood, spleen, liver, and bone marrow are harvested for analysis. The samples axe processed in the foUov/ing manner: Peripheral blood is collected into heparinized vacutubes. The spleen and hver are removed and a single cell suspension generated. Tibia and femur bone marrow is harvested. Levels of PEGASYS® OT hlFN polypeptide are measured in blood and tissue by methods including but not limited to, ELISA. Peled et al. in Science (1999) 283:845-848 and Kim et al. in Stem Cells (1999) 17:286-294 discuss the repopulating cells of SCID mice.<br><br>
[556]	Cynomolgus monkeys are used to evaluate in vivo activity and bone marrow<br>
toxicity of hlFN polypeptides. Induction of 2',5'-O1igoadenylate synthetase (2'5'-OAS) is measured in monkeys and reflects activity of hlFN polypeptides. Bone marrow toxicity is evaluated by measuring circulating blood cells, including but not limited to neutrophils, RBCs, and platelets as well as potentially collecting and evaluating bone marrow.<br>
[557]	Additional animal models for evaluating hlFN polypeptides of the invention<br>
include animal models studying depression. Capuron, L et al. in Am. J. Psychiatry 2003; 160:1342-1344 present clinical data suggesting a correlation between IFN-a, the HP A axis, and depression. During the first twelve weeks of IFN-a therapy, the plasma concentrations of adrenocorticotropic hormone (ACTH), Cortisol, and interleukin-6 (IL-6) were measured in patients both immediately before and one, two, and three hours after administration. The patients were also evaluated for symptoms of major depression during the course of the study. It was found that patients who subsequently met symptom criteria for major depression exhibited higher ACTH and Cortisol responses (but not IL-6 responses) to the initial administration of EFN- a than patients who did not meet the symptom criteria.<br>
[558]	Yamano, M. et al. JPET 2000; 292:181-187, which is incorporated by reference<br>
herein, discusses Sumiferon, a lymphoblastoid preparation of human IFN- a (Sumitomo Pharmaceuticals), and YM643 (IFN-alphacon-1; Amgen) and their effects on mice. Upon dosing i.v. both compounds induced immobility, a depression-like behavior, in mice, as measured by the Tail Suspension Test (TST) in a dose-dependent manner. Similar results were achieved by s.c. and c.v. injection, although dose levels and general dosing regimens differed. Imipramine (a tricyclic anti-depressant and down-regulator of the HPA axis) was found to significantly reduce IFN-induced immobility, whereas indomethacin (a cyclooxygenase inhibitor) and naloxone (an opioid receptor antagonist) did not reduce the observed IFN-induced immobility. Yamano et al. also showed that the CRF antagonist CP-154,526 blocks IFN-induced immobility and does so in a dose-dependent manner, implicating the involvement of the HPA axis in IFN-a induced depression.<br>
[559]	The Tail Suspension Test is an alternative to the "behavioral despair" or swim<br>
test in which an animal is forced to swim in a particular area without an escape. With the Tail Suspension Test, the animal is suspended from its tail for defined length of time, e.g., six minutes, and cannot escape. During the test, various measurements are taken including but not limited to, the number of times each animal enters into an escape behavior (called an event) e.g. struggling episodes, the duration of the event, and the average strength of each event   Tail<br><br>
Suspension Test equipment is available, for example, from Lafayette Instrument (Lafayette,<br>
Indiana).<br>
[560]	Other studies involve animal models of other side effects found with current IFN<br>
therapies, such as neutropenia. Animal studies performed may be performed in combination<br>
with Ribavirin or another compound.<br>
[561]	The specific activity of hlFN polypeptides in accordance with this invention can<br>
be determined by various assays known in the art.    The biological activity of the hlFN<br>
polypeptide muteins, or fragments thereof, obtained and purified in accordance with this<br>
invention can be tested by methods described or referenced herein or known to those of ordinary<br>
skill in the art.<br>
XIV.	Administration and Pharmaceutical Compositions<br>
[562]	The polypeptides or proteins of the invention (including but not limited to, hlFN,<br>
synthetases, proteins comprising one or more unnatural amino acid, etc.) are optionally<br>
employed for therapeutic uses, including but not limited to, in combination with a suitable<br>
pharmaceutical carrier.   Such compositions, for example, comprise a therapeutically effective<br>
amount of the compound, and a pharmaceutically acceptable carrier or excipient. Such a carrier<br>
or excipient includes, but is not limited to, saline, buffered saline, dextrose, water, glycerol,<br>
ethanol,   and/or  combinations thereof     The  formulation  is  made  to   suit  the  mode   of<br>
administration.   In general, methods of administering proteins are known to those of ordinary<br>
skill in the art and can be applied to administration of the polypeptides of the invention.<br>
[563]	Therapeutic compositions comprising one or more polypeptide of the invention<br>
are optionally tested in one or more appropriate in vitro and/or in vivo animal models of disease, to confirm efficacy, tissue metabolism, and to estimate dosages, according to methods Icnown to those of ordinary skill in the art. In particular, dosages can be initially determined by activity, stability or other suitable measures of unnatural herein to natural amino acid homologues (including but not limited to, comparison of a hlFN polypeptide modified to include one or more unnatural amino acids to a natural amino acid hlFN polypeptide), i.e., in a relevant assay.<br>
[564]	Administration is by any of the routes normally used for introducing a molecule<br>
into ultimate contact with blood or tissue cells. The unnatural amino acid polypeptides of the invention are administered in any suitable manner, optionally with one or more pharmaceutically acceptable carriers. Suitable methods of administering such polypeptides in the context of the present invention to a patient are available, and, although more than one route can be used to<br><br>
administer a particular composition, a particular route can often provide a more immediate and more effective action or reaction than another route.<br>
[565]	Pharmaceutically acceptable carriers are determined in part by the particular<br>
composition being administered, as well as by the particular method used to administer the composition. Accordingly, there is a wide variety of suitable formulations of pharmaceutical compositions of the present invention.<br>
[566]	hlFN polypeptides of the invention may be administered by any conventional<br>
route suitable for proteins or peptides, including, but not limited to parenterally, e.g. injections including, but not limited to, subcutaneously or intravenously or any other form of injections or infusions. Polypeptide compositions can be administered by a number of routes including, but not limited to oral, intravenous, intraperitoneal, intramuscular, transdermal, subcutaneous, topical, sublingual, or rectal means. Compositions comprising non-natural amino acid polypeptides, modified or unmodified, can also be administered via liposomes. Such administration routes and appropriate formulations are generally known to those of skill in the art. The hlFN polypeptide comprising a non-naturally encoded amino acid, may be used alone or in combination with other suitable components such as a pharmaceutical carrier.<br>
[567]	The  hlFN polypeptide  comprising  a non-natural  amino  acid,  alone  or  in<br>
combination with other suitable components, can also be made into aerosol formulations (i.e., they can be "nebulized") to be administered via inhalation. Aerosol formulations can be placed into pressurized acceptable propellants, such as dichlorodifluoromethane, propane, nitrogen, and the like.<br>
[568]	Formulations suitable for parenteral administration, such as, for example, by<br>
intraarticular (in the joints), intravenous, intramuscular, intradermal, intraperitoneal, and subcutaneous routes, include aqueous and non-aqueous, isotonic sterile injection solutions, which can contain antioxidants, buffers, bacteriostats, and solutes that render the formulation isotonic wth the blood of the intended recipient, and aqueous and non-aqueous sterile suspensions that can include suspending agents, solubilizers, thickening agents, stabilizers, and preservatives. The formulations of hlFN can be presented in unit-dose or multi-dose sealed containers, such as ampules and vials.<br>
[569]	Parenteral administration and intravenous administration are preferred methods<br>
of administration. In particular, the routes of administration already in use for natural amino acid homologue therapeutics (includmg but not limited to, those typically used for EPO, GH, G-<br><br>
CSF, GM-CSF, IFNs, interleukins, antibodies, and/or any other pharmaceutically delivered protein), along with formulations in current use, provide preferred routes of administration and formulation for the polypeptides of the invention.<br>
[570]	The dose administered to a patient, in the context of the present invention, is<br>
sufficient to have a beneficial therapeutic response in the patient over time, or, including but not limited to, to inhibit infection by a pathogen, or other appropriate activity, depending on the application. The dose is determined by the efficacy of the particular vector, or formulation, and the activity, stability or serum half-life of the unnatural amino acid polypeptide employed and the condition of the patient, as well as the body weight or surface area of the patient to be treated. The size of the dose is also determined by the existence, nature, and extent of any adverse side-effects that accompany the administration of a particular vector, formulation, or the like in a particular patient.<br>
[571]	In determining the  effective  amount of the  vector  or  formulation  to  be<br>
administered in the treatment or prophylaxis of disease (including but not limited to, cancers, inherited diseases, diabetes, AIDS, or the like), the physician evaluates circulating plasma levels, formulation toxicities, progression of the disease, and/or where relevant, the production of anti-unnatural amino acid polypeptide antibodies.<br>
[572]	The dose administered, for example, to a 70 kilogram patient, is typically in the<br>
range equivalent to dosages of currentlγ-used therapeutic proteins, adjusted for the altered activity or serum half-life of the relevant composition. The vectors or pharmaceutical formulations of this invention can supplement treatment conditions by any known conventional therapy, including antibody administration, vaccine administration, administration of cytotoxic agents, natural amino acid polypeptides, nucleic acids, nucleotide analogues, biologic response modifiers, and the like.<br>
[573]	For admuiistration, formulations of the present invention are administered at a<br>
rate determined by the LD-50 or ED-SO of the relevant formulation, and/or observation of any side-effects of the unnatural amino acid polypeptides at various concentrations, including but not limited to, as applied to the mass and overall health of the patient. Administration can be accomplished via single or divided doses.<br>
[574]	If a patient undergoing infusion of a formulation develops fevers, chills, or<br>
muscle aches, he/she receives the appropriate dose of aspirin, ibuprofen, acetaminophen or other pain/fever controlling drug.   Patients who experience reactions to the infusion such as fever,<br><br>
muscle aches, and chills are premedicated 30 minutes prior to the future infusions with either aspirin, acetaminophen, or, including but not limited to, diphenhydramine. Meperidine is used for more severe chills and muscle aches that do not quiclcly respond to antipyretics and antihistamines.   Cell infusion is slowed or discontinued depending upon the severity of the<br>
reaction.<br>
[575]	Human hlFN polypeptides of the mvention can be administered directly to a<br>
mammalian subject. Administration is by any of the routes normally used for introducing hlFN polypeptide to a subject The hlFN polypeptide compositions according to embodiments of the present invention include those suitable for oral, rectal, topical, inhalation (including but not limited to, via an aerosol), buccal (including but not limited to, sub-lingual), vaginal, parenteral (including but not limited to, subcutaneous, intramuscular, intradermal, intraarticu] ar, intrapleural, intraperitoneal, inracerebral, intraarterial, or intravenous), topical (i.e., both skin and mucosal surfaces, including airway surfaces) and transdermal administration, although the most suitable route in any given case will depend on the nature and severity of the condition being treated. Administration can be either local or systemic. The formulations of compounds can be presented in unit-dose or multi-dose sealed containers, such as ampoules and vials. hlFN polypeptides of the invention can be prepared in a mixture in a unit dosage injectable form (including but not limited to, solution, suspension, or emulsion) with a pharmaceutically acceptable carrier. hlFN polypeptides of the invention can also be administered by continuous infusion (using, including but not limited to, minipumps such as osmotic pumps), single bolus or slow-release depot formulations.<br>
[576]	Formulations  suitable  for  administration  include  aqueous  and  non-aqueous<br>
solutions, isotonic sterile solutions, which can contain antioxidants, buffers, bacteriostats, and solutes that render the formulation isotonic, and aqueous and non-aqueous sterile suspensions that can include suspending agents, solubilizers, thickening agents, stabilizers, and preservatives. Solutions and suspensions can be prepared from sterile powders, granules, and tablets of the kind previously described.<br>
[577]	Freeze-drying is a commonly employed technique for presenting proteins which<br>
serves to remove water from the protein preparation of interest. Freeze-drying, or lyophilization, is a process by which the material to be dried is first frozen and then the ice or frozen solvent is removed by sublimation in a vacuum environment An excipient may be included in pre-lyophilized formulations to enhance stability during the freeze-drymg process and/or to improve stability of the lyophilized product upon storage. Pikal, M. Biopharm. 3(9)26-30 (1990) and Arakawa et al. Pharm. Res. 8(3):285-291 (1991).<br><br>
[578]	The spray drying of pharmaceuticals is also known to those of ordinary skill in the<br>
art. For example, see Broadhead, J. et al., "The Spray Drying of Pharmaceuticals," in Drug Dev.<br>
Ind. Pharm, 18 (11 &amp; 12), 1169-1206 (1992). In addition to small molecule pharmaceuticals, a<br>
variety of biological materials have been spray dried and these include: enzymes, sera, plasma,<br>
micro-organisms and yeasts. Spray drying is a useful technique because it can convert a liquid<br>
pharmaceutical preparation into a fine, dustless or agglomerated powder in a one-step process.<br>
The basic technique comprises the following four steps; a) atomization of the feed solution into<br>
a spray; b) spraγ-air contact; c) drying of the spray; and d) separation of the dried product from<br>
the drying air. U.S. Patent Nos. 6,235,710 and 6,001,800, which are incorporated by reference<br>
herein, describe the preparation of recombinant erythropoietin by spray drying.<br>
[579]	The    pharmaceutical    compositions    of   the    invention    may    comprise    a<br>
pharmaceutically acceptable carrier, excipient, or stabilizer. Pharmaceutically acceptable carriers are determined in part by the particular composition being administered, as well as by the particular method used to administer the composition. Accordingly, there is a wide variety of suitable formulations of pharmaceutical compositions (including optional pharmaceutically acceptable carriers, excipients, or stabilizers) of the present invention (see, e.g.. Remington's Pharmaceutical Sciences, 17th ed. 1985)).<br>
[580]	Suitable carriers include, but are not limited to, buffers containing succinate,<br>
phosphate, borate, HEPES, citrate, histidine, imidazole, acetate, bicarbonate, and other organic acids; antioxidants including but not limited to, ascorbic acid; low molecular weight polypeptides including but not limited to those less than about 10 residues; proteins, including but not limited to, serum albumin, gelatin, or immunoglobulins; hydrophilic polymers including but not limited to, polyvinylpyrrolidone; amino acids including but not limited to, glycine, glutamine, asparagine, arginine, histidine or histidine derivatives, methionine, glutamate, or lysine; monosaccharides, disaccharides, and other carbohydrates, including but not limited to, trehalose, sucrose, glucose, mannose, or dextrins; chelating agents including but not limited to, EDTA and edentate disodium; divalent metal ions including but not limited to, zinc, cobalt, or copper; sugar alcohols including but not limited to, mannitol or sorbitol; salt-forming counter ions including but not limited to, sodium and sodium chloride; and/or nonionic surfactants including but not limited to TweenTM (including but not limited to, Tween 80 (polysorbate 80) and Tween 20 (polysorbate 20), PluronicsTM and other pluronic acids, including but not limited to, and other pluronic acids, including but not limited to, pluronic acid F68 (poloxamer 188), or PEG. Suitable surfactants include for example but are not limited to polyethers based upon poly(ethylene oxide)-poly(propylene oxide)-poly(ethylene oxide), i.e., (PEO-pPO-pEO), or<br><br>
poly(propylene oxide)-poly(ethyIene oxide)-poly(propylene oxide), i.e., (PPO-pEO-pPO), or a combination thereof. PEO-pPO-pEO and PPO-pEO-pPO are commercially available under the trade names Pluronics™, R-pluronics™, TelronicsTM and R-Tetronics™ (BASF Wyandotte Corp., Wyandotte, Mich.) and are further described in U.S. Pat. No. 4,820,352 incorporated herein in its entirety by reference. Other ethylene/polypropylene block polymers may be suitable surfactants. A surfactant or a combination of surfactants may be used to stabilize PEGylated hlFN against one or more stresses including but not limited to stress that results from agitation. Some of the above may be referred to as "bulking agents." Some may also be referred to as "tonicity modifiers." Antimicrobial preservatives may also be applied for product stability and antimicrobial effectiveness; suitable preservatives include but are not limited to, benzyl alcohol, bezalkonium chloride, metacresol, methyl/propyl parabene, cresol, and phenol, or a combination thereof.<br>
[581]	Formulations of interferon molecules, including but not limited to, interferon<br>
alpha and PEGylated forms of interferon alpha are described in U.S. Patent Nos. 5,762,923; 5,766,582; and 5,935,566, which are incorporated by reference herein in their entirety. Methods for manufacture and to test stability of interferon molecules are also described. Allen et al. International Journal of Pharmaeutics (1999) 187:259-272 also discusses the formulation of a hybrid mterferon-a molecule and degradation products found under various conditions such as high and lowpH.<br>
{582]	hlFN polypeptides of the invention, including those linked to water soluble<br>
polymers such as PEG can also be administered by or as part of sustained-release systems. Sustained-release compositions include, including but not limited to, semi-permeable polymer matrices in the form of shaped articles, including but not limited to, films, or microcapsules. Sustained-release matrices include from biocompatible materials such as poly(2-hydroxyethyl methacrylate) (Langer et al, J, Biomed. Mater, Res., 15: 267-277 (1981); Langer, Chem. Tech,, 12: 98-105 (1982), ethylene vinyl acetate (Langer et al, supra) or polγ-D-(-)-3-hydroxybutyric acid (EP 133,988), polylactides (polylactic acid) (U.S. Patent No. 3,773,919; EP 58,481), polyglycolide (polymer of glycolic acid), polylactide co-glycolide (copolymers of lactic acid and glycolic acid) polyanhydrides, copolymers of L-glutamic acid and gammα-ethyl-L-glutamate (Sidman et ah, Biopolymers, 22, 547-556 (1983), poly(ortho)esters, polypeptides, hyaluronic acid, collagen, chondroitin sulfate, carboxylic acids, fatty acids, phospholipids, polysaccharides, nucleic acids, polyamino acids, amino acids such as phenylalanine, tyrosine, isoleucine, polynucleotides, polyvinyl propylene, polyvinylpyrrolidone and silicone. Sustained-release compositions also include a liposomally entrapped compound. Liposomes containing the<br><br>
compound are prepared by methods known per se; DE 3,218,121; Eppstein et al, Proc. Natl.<br>
Acad Set U.S.A., 82: 3688-3692 (1985); Hwang et al., Proc, Natl Acad Set U.S.A., 77: 403O-<br>
4034 (1980); EP 52,322; EP 36,676; U.S. Patent No. 4,619,794; EP 143,949; U.S. Patent No.<br>
5,021,234; Japanese Pat Appln. 83-118008; U.S. Pat. Nos. 4,485,045 and 4,544,545; and EP<br>
102,324. All references and patents cited are incorporated by reference herein.<br>
[583]	Liposomally entrapped hlFN polypeptides can be prepared by methods described<br>
in, e.g., DE 3,218,121; Eppstein et al, Proc. Natl Acad Set U.S.A., 82: 3688-3692 (1985); Hwang et al, Proc. Natl Acad. ScL US.A,, 77: 403O-4034 (1980); EP 52,322; EP 36,676; U.S. Patent No, 4,619,794; EP 143,949; U.S. Patent No. 5,021,234; Japanese Pat. Appln. 83-118008; U.S. Patent Nos. 4,485,045 and 4,544,545; and EP 102,324. Composition and size of liposomes are well known or able to be readily determined empirically by one of ordinary skill in the art. Some examples of liposomes as described in, e.g., Park JW, et al, Proc. Natl Acad. ScL USA 92:1327-1331 (1995); Lasic D and Papahadjopoulos D (eds): MEDICAL APPLICATIONS OF LIPOSOMES (1998); Drummond DC, et al. Liposomal drug delivery systems for cancer therapy, in Teicher B (ed): CANCER DRUG DISCOVERY AND DEVELOPMENT (2002); Park JW, et al, Clin Cancer Res. 8:1172-1181 (2002); Nielsen UB, et al, Biochim. Biophys. Acta 1591(1-3):109-118 (2002); Mamot C, et al. Cancer Res. 63: 3154-3161 (2003). All references and patents cited are incorporated by reference herein.<br>
[584]	The dose administered to a patient in the context of the present invention should<br>
be sufficient to cause a beneficial response in the subject over time. Generally, the total pharmaceutically effective amount of the hlFN polypeptide of the present invention administered parenterally per dose is in the range of about 0.01 ^g/kg/day to about 100 ^g/kg, or about 0.05 mg/kg to about 1 mg/kg, of patient body weight, although this is subject to therapeutic discretion. The frequency of dosing is also subject to therapeutic discretion, and may be more frequent or less frequent than the commercially available hlFN polypeptide products approved for use in humans. Generally, a PEGylated hlFN polypeptide of the invention can be administered by any of the routes of administration described above.<br>
XV.	Therapeutic Uses of hJFN Polypeptides of the Invention<br>
[585]	The hlFN polypeptides of the invention are useful for treating a wide range of<br>
disorders.<br>
[586]	An agonist hlFN variant may act to stimulate the immune system of a mammal by<br>
increasing its immune function, whether the increase is due to antibody mediation or cell<br>
mediation, and whether the immune system is endogenous to the host treated with the hlFN<br><br>
polypeptide or is transplanted from a donor to the host recipient given the hlFN polypeptide (as in bone marrow transplants). "Immune disorders" include any condition in which the immune system of an individual has a reduced antibody or cellular response to antigens than normal, including those individuals with small spleens with reduced immunity due to drug (e.g., chemotherapeutic) treatments. Examples individuals with immune disorders include, e.g., elderly patients, individuals undergoing chemotherapy or radiation therapy, individuals recovering from a major illness, or about to undergo surgery, individuals with AIDS, Patients with congenital and acquired B-cell deficiencies such as hypogammaglobulmemia, common varied agammaglobulinemia, and selective immunoglobulin deficiencies (e.g., IgA deficiency, patients infected with a virus such as rabies with an incubation time shorter than the immune response of the patient; and individuals with hereditary disorders such as diGeorge syndrome. IFNα's exhibit many immunomodulatory activities, see Zoon et al., (1986) In, The Biology of the Interferon System. Cantell and Schellenkens, Eds., Martinus Nyhofif Publishers, Amsterdam).<br>
[587]	Administration of the hlFN products of the present invention results in any of the<br>
activities demonstrated by commercially available IFN preparations in humans. The pharmaceutical compositions containing hlFN may be formulated at a strength effective for administration by various means to a human patient experiencing disorders that may be affected by IFN agonists or antagonists, such as but not limited to, anti-proliferatives, anti-inflammatory, or antivirals are used, either alone or as part of a condition or disease.<br>
[588]	The hlFN of the present invention may thus be used to interrupt or modulate a<br>
viral replication cycle, modulate inflarmnation, or as anti-proliferative agents. Among the conditions treatable by the present invention include HCV, HBV, and other viral infections, tumor cell growth or viability, and multiple sclerosis. The invention also provides for administration of a therapeutically effective amount of another active agent such as an anticancer chemotherapeutic agent. The amount to be given may be readily determined by one skilled in the art based upon therapy with hlFN.<br>
[589]	Although IFNs were first discovered by virologists, their first clinical use (in<br>
1979) was as therapeutic agents for myeloma (Joshua et al., (1997) Blood Rev. 11(4): 191-200). IFNα's have since been shown to be efficacious against a myriad of diseases of viral, malignant, angiogenic, allergic, inflammatory, and fibrotic origin (Tilg, (1997) Gastroenterolog)'. 112(3): 1017-1021). It has also proven efficacious in the treatment of metastatic renal carcinoma and chronic myeloid leukemia (Williams and Linch, (1997) Br. J. Hosp. Med. 57(9):436-439).<br><br>
Clinical uses of IFNs are reviewed in Gresser (1997) J. Leukoc. Biol 61(5):567-574 and Pfeffer<br>
(1997) Semin. Oncol. 24(3 Suppl. 9):S9-S63S969, which are incorporated by reference herein.<br>
[590]	Average quantities of the hlFN may vary and in particular should be based upon<br>
the recommendations and prescription of a qualified physician. The exact amount of hlFN is a matter of preference subject to such factors as the exact type of condition being treated, the condition of the patient being treated, as well as the other ingredients in the composition. The invention also provides for administration of a therapeutically effective amount of another active agent. The amount to be given may be readily determined by one of ordinary skill in the art based upon therapy with hlFN.<br>
[591]	Pharmaceutical  compositions  of the  invention  may be  manufactured  in  a<br>
conventional manner.<br>
EXAMPLES<br>
{592]	The following examples are offered to illustrate, but do not to limit the claimed<br>
invention.<br>
Example 1<br>
[593]	This example describes one of the many potential sets of criteria for the selection<br>
of preferred sites of incorporation of non-naturally encoded amino acids into bIFN.<br>
[594]	This example demonstrates how preferred sites within the hlFN polypeptide were<br>
selected for introduction of a non-naturally encoded amino acid. The crystal structure with PDB ID 1RH2 and the NMR structure IITF (twentγ-four different NMR structures) were used to determine preferred positions into which one or more non-naturally encoded amino acids could be introduced. The coordinates for these structures are available from the Protein Data Bank (PDB) or via The Research Collaboratory for Structural Bioinformatics PDB available on the World Wide Web at rcsb.org.<br>
[595]	Sequence numbering used in this example is according to the amino acid<br>
sequence of mature hlFN shown in SEQ ID NO: 2. The following criteria were used to evaluate each position of hlFN for the introduction of a non-naturally encoded amino acid: the residue (a) should not interfere with binding of hlFN based on structural analysis of crystallographic structures of hlFN conjugated with hlFN receptor, b) should not be affected by alanine scanning mutagenesis, (c) should be surface exposed and exhibit minimal van der Waals or hydrogen bonding interactions with surrounding residues, (d) should be either deleted or variable in hlFN<br><br>
variants, (e) would result in conservative changes upon substitution with a non-naturally encoded amino acid and (f) could be found in either highly flexible regions (including but not limited to CD loop) or structurally rigid regions (including but not limited to Helix B). Publications used in site evaluation mclude: Bioconj. Chemistry 2001 (12) 195-202; Current Pharmaceutical Design 2002 (8) 2139-2157; Neuroreport 2001 (12), 857-859; BBRC 1994 (202) 1445-1451; Cancer Biotherapy + Radiopharmaceuticals 1998 (voll3) 143-153; Structure 1996 (14) 1453-1463; JMB 1997 (274) 661-675. In addition, further calculations were performed on the hlFN molecule, utilizing the Cx program (Pintar et al. Bioinformatics, 18, pp 980) to evaluate the extent of protrusion for each protein atom. In some embodiments, one or more non-naturally encoded amino acid are substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide): before position 1 (i.e., at the N-terminus), 1, 2, 3, 4, 5, 6, 7^ 8, 9, 10,<br>
II,	12, 13,14, 15, 16, 17,18, 19,20,21,22,23,24,25,26,27,28,29,30, 31,32, 33, 34,35,36,<br>
37, 38, 39, 40,41,42,43,44,45, 46,47,48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62,<br>
63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88,<br>
89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110,<br>
III,	112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129,<br>
130", 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148,<br>
149, 150, 151, 152, 153, 154, 155,156, 157, 158, 159, 160, 161, 162, 163, 164, 165, or 166 (i.e.<br>
at the carboxyl terminus). In some embodiments, one or more non-naturally encoded amino<br>
acid are substituted at, but not limited to, one or more of the following positions of hlFN (as in<br>
SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN<br>
polypeptide): before position 1 (i.e., at the N-terminus), 1, 2, 3, 4, 5,6, 7, 8, 9,12, 13,16,19, 20,<br>
22,23,24,25,26,27,28, 30, 31, 32, 33,34, 35,40,41,42,45,46,48,49, 50, 51, 58, 61,64, 65,<br>
68, 69, 70, 71, 73, 74, 77, 78, 79, 80, 81, 82, 83, 85, 86, 89, 90, 93, 94, 96, 97, 100, 101, 103,<br>
105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129,<br>
131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164,<br>
165, or 166 (i.e. at the carboxyl terminus). In some embodiments, one or more non-naturally<br>
encoded amino acids are incorporated in one or more of the following positions in IFN: 6, 9,12,<br>
13,16,41,45,46,48,49,61,64,65,96, 100,101,103, 106, 107, 108, 110, 111, 113, 114, 117,<br>
120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acids in<br>
SEQ ID NO: 1 or 3). In some embodiments, the IFN polypeptides of the invention comprise one<br>
or more non-naturally encoded amino acids at one or more of the following positions: 100, 106,<br>
107,   108,  111,   113,  114 (SEQ ID NO: 2,  or the corresponding amino acid in or the<br><br>
corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions; 41,45,46,48,49 (SEQ ID NO; 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 61, 64, 65, 101, 103, 110, 117, 120, 121, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding' amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 6, 9, 12, 13, 16, 96, 156, 159, 160, 161, 162 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 2, 3,4, 5, 7, 8,16,19,20,40,42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: I5 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these or other positions is linked to a water soluble polymer, including but not limited to positions: before position 1 (i.e., at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11,12, 13,14, 15,16, 17, 18, 19, 20,21, 22, 23, 24, 25. 26, 27, 28, 29, 30, 31,32, 33, 34,35,36, 37, 38,39,40,41,42,43.44,45,46.47,48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58. 59, 60. 61, 62, 63, 64, 65, 66, 67. 68. 69, 70. 71, 72, 73, 74. 75. 76, 77, 78, 79, 80, 81, 82. 83. 84. 85, 86. 87, 88. 89, 90, 91, 92, 93, 94, 95,96, 97,98,99, 100,101, 102,103, 104, 105,106, 107. 108, 109, 110. 111. 112, 113, 114, 115. 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134. 135, 136, 137, 138, 139, 140, 141. 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160. 161, 162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus) (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these positions is linked to one or more water soluble polymers, including but not limited to positions: before position 1 (i.e. the N terminus), 1, 2, 3, 4, 5, 6, 7. 8, 9, 12, 13, 16, 19, 20, 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35. 40, 41, 42, 45. 46. 48, 49.<br><br>
50, 51,58, 61, 64, 65, 68,69, 70,71, 73, 74, 77,78, 79,80, 81, 82, 83, 85, 86, 89, 90, 93, 94,96, 97, 100, 101, 103, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 117, 118, 120. 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164, 165, 166 (i.e. at the carboxyl terminus) (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid is linked to a water soluble polymer at one or more of the following positions: 6, 9,12, 13,16, 41,45, 46,48, 49, 61, 64, 65, 96, 100, 101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linked to one or more water-soluble polymer: 100, 106, 107, 108, 111, 113, 114 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linked to one or more water-soluble polymer: 41,45, 46,48, 49 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linked to one or more water-soluble polymer: 61, 64, 65, 101, 103, 110, 117, 120, 121, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linked to one or more water-soluble polymer: 6, 9, 12, 13, 16, 96, 156, 159, 160, 161, 162 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linlced to one or more water-soluble polymer: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the one or more non-naturally encoded amino acids at one or more of the following positions is linked to one or more water-soluble polymer: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the water soluble polymer is coupled to the IFN polypeptide to a non-naturally encoded amino acid at one or more of the following amino acid positions: 6, 9, 12, 13, 16, 41, 45,46,48,49,61,64,65,96, 100,101,103, 106,107,108,110, 111, 113,114,117, 120, 121,<br><br>
149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments the water soluble polymer is coupled to the IFN polypeptide at one or more of the following amino acid positions: 6, 9, 12, 13,16,41,45,46,48,49,61, 64,65,96, 100,101, 103, 106, 107, 108, 110, 111, 113, 114, 117, 120, 121, 149, 156, 159, 160, 161 and 162 (SEQ ID NO: 2, or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these positions is linked to one or more water soluble polymers, positions: 34, 78, 107 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the IFN polypeptides of the invention comprise one or more non-naturally encoded amino acids at one or more of the following positions providing an antagonist: 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide); a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on the intended site selected and desired activity. Human IFN antagonists include, but are not limited to, hlFN polypeptides with one or more non-naturally encoded amino acid substitutions at 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94, 124, 125,127, 128, 129, 131, 132,133, 134, 135, 136,137, or any combinations thereof (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide).<br>
[596]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide): 31, 134,34,38, 129,36, 122,37, 121,41, 125,124,149,117,39,118,120, 107, 108, 106, 100, 111,113,114,41,45,46,48,49,61,64,65,101,103,102,110,117,120,121,149,96,6,9,12, 13,16, 68, 70,109, 159, 161,156, 160,162, 24,27, 78. 83, 85, 87, 89, 164. In one embodiment, a non-naturally encoded amino acid is substituted at position 38 of hlFN (as in SEQ ID NO: 2, or the corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide). In some embodiments, the non-naturally encoded amino acid at one or more of these or other positions is linked to a water soluble polymer, including but not limited to positions: 31, 134, 34, 38, 129, 36,122,37,121,41,125,124,149,117,39, 118, 120,107,108,106,100,111,113,114,41,45, 46,48,49,61,64,65, 101, 103, 102, 110, 117,120, 121, 149,96,6,9, 12, 13, 16,68,70, 109,<br><br>
159,	161, 156, 160, 162, 24, 27, 78, 83, 85, 87, 89, 164 (as in SEQ ID NO: 2, or the<br>
corresponding amino acids in SEQ ID NO: 1 or 3, or any other IFN polypeptide).<br>
[597]	In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at, but not limited to, one or more of the following positions of hlFN (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3): before position 1 (i.e., at the N-tenninus), 1, 2, 3,4,5, 6, 7, 8,9,10.11,12,13,14,15,16, 17, 18,19, 20,21,22,23,24,25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159,<br>
160,	161, 162, 163j 164, 165, or 166 (i.e. at the carboxyl terminus) and one or more natural<br>
amino acid substitutions. In some embodiments, the one or more non-naturally encoded amino<br>
acids are coupled to a water soluble polymer. In some embodiments, the one or more non-<br>
naturally encoded amino acids are coupled to PEG. In one embodiment, the natural amino acid<br>
substitution is R149Y. In some embodiments, the natural amino acid substitution is R149E. In<br>
some embodiments, the natural amino acid substitution is R149S. In one embodiment, the non-<br>
natural amino acid substitution is at position 107 and the natural amino acid substitution is<br>
R149Y. In one embodiment, the non-natural amino acid substitution is at position 106 and the<br>
natural amino acid substitution is R149Y. In some embodiments, the one or more naturally<br>
encoded amino acid substitution is at one or more of the following positions of hDFN (SEQ ID<br>
NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3), including but not limited to:<br>
10, 16, 13, 79, 83, 85, 86, 87, 90, 91, 93, 94, 96, 120, 121, 124, 125, 128, 149. In some<br>
embodiments, the one or more naturally encoded amino acid substitution is one or more of the<br>
following substitutions (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or<br>
3), including but not limited to: GlOE, M16R, R13E, T79R, K83Q, K83S, Y85L, T86S, E87S,<br>
Q90R, Q91E, N93Q, D94V, E96K, R120K, K121T, Q124R, R125G, L128R, R149Y, R149E,<br>
R149S. In some embodiments, the natural amino acid substitution is at position 1 (the N-<br>
terminus). In some embodiments, one or more non-naturally encoded amino acids are<br>
substituted at one or more of the following positions of hlFN (as in SEQ ID NO: 2, or the<br>
corresponding amino acids in other IFN's): 107, 78, 34. In some embodiments, the non-<br>
naturally encoded amino acid at one or more of these positions is coupled to a water soluble<br>
polymer: 107,78,34.<br><br>
[598]	One or more amino acids found m a limitin sequence may be substituted into a<br>
hlFN polypeptide (hybrid limitin/hlFN polypeptides). Examples include but are not limited to<br>
the natural amino acid substitutions described in the previous paragraph. Alternatively, a set of<br>
amino acids found in an interferon polypeptide may be replaced by a set of amino acids found in<br>
a limitin sequence, A set of amino acids may comprise contiguous amino acids or amino acids<br>
present m different portions of the molecule but are involved in a structural characteristic or<br>
biological activity of the polypeptide. The mouse limitin molecule has an improved CFU-GM<br>
toxicity profile compared to other IFNα proteins. Alignment of human IFNα-2a with the<br>
limitin protein sequence showed 30% amino acid identity. 50% sequence conservation was also<br>
observed. In particular, a prominent deletion in the limitin sequence between the C and D<br>
helices (in the loop between C and D helices) was observed. The *'HV" mutant was generated<br>
with the following substitutions in hIFNα-2a (SEQ ID NO: 2): D77-D94 is replaced with the<br>
mouse limitin sequence HERALDQLLSSLWRELQV. The "CD' mutant was generated witii<br>
the following substitutions in hIFNα-2a (SEQ ID NO: 2): V105-D114 with GQSAPLP. This<br>
hybrid molecule with the loop region from limitin substituted into the human IFNα"2a protein<br>
("CD" mutant) was found to have equivalent anti-viral activity as the WHO IFN standard. In<br>
addition to the one or more limitin amino acids, the hlFN polypeptide may comprise one or<br>
more non-naturally encoded amino acids at any one or more positions of the hlFN polypeptide.<br>
In some embodiments, the one or more non-naturally encoded amino acids may be linked to a<br>
water soluble polymer such as PEG or bonded directly to a water soluble polymer such as PEG.<br>
In addition to the natural amino acid substitutions in the HV or the CD mutant, one or more<br>
additional natural amino acid substitutions may be found in the hlFN polypeptide.<br>
[599J	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 34, 39, 45, 46, 64,65,68,78,85,87, 101, 107, 108,111,114, 118,124, 125, 145, 146, 153, 156,96, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the hlFN polypeptide comprises one or nlore non-naturally encoded amino acids at one or more of the following positions linlced to one or more water-soluble polymer; 6, 16, 34, 39, 45, 46, 64, 65, 68, 78, 85, 87, 101, 107, 108, 111, 114, 118, 124, 125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other IFN polypeptide). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions linked to one or more water-soluble polymer: 6, 16, 34, 39, 45, 46, 64, 65, 68, 78, 85, 87, 101, 107, 108, 111, 114, 118, 124,<br><br>
125, 145, 146, 153, 156, 96, 149 (SEQ ID NO: 2, or the corresponding amino acid in or the<br>
corresponding amino acid in SEQ ID NO: 1,3, or any other IFN polypeptide) and comprises one<br>
or more naturally encoded amino acid substitution. In some embodiments, the hlFN polypeptide<br>
comprises one or more non-naturally encoded amino acids at one or more of the following<br>
positions linked to one or more water-soluble polymer: 6,16, 34, 39, 45, 46, 64, 65, 68, 78, 85,<br>
87, 101, 107, 108, 111, 114, 118, 124, 125,145,146, 153, 156,96, 149 (SEQ ID NO: 2, or the<br>
corresponding amino acid in or the corresponding amino acid in SEQ ID NO: 1, 3, or any other<br>
IFN polypeptide) and comprises one or more of the following naturally encoded amino acid<br>
substitutions GlOE, M16R, R13E, T79R, K83Q, K83S, Y85L, T86S, E87S, Q90R, Q91E,<br>
N93Q, D94V, E96K, R120K, K121T, Q124R, R125G, L128R, R149Y, R149E, R149S.<br>
[600]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[601]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 and one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3), In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 that is linked to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 89, 107, 108 that is bonded to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, L80A, K83S, Y85L, Y85S, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br><br>
[602]	In some embodiments, the hlFlM polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45,46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br>
[603]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3), In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6, 16, 37, 45, 46, 78, 87, 107, and 108 that is linked to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO; 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 6,16, 37, 45, 46, 78, 87, 107, and 108 that is bonded to a water soluble polymer and comprises one or more of the following naturally encoded amino acid substitutions: T79R, K83S, Y85L, T86S, E87S, Q91E (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[604]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is linked to a water soluble polymer. In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) that is bonded to a water soluble polymer.<br><br>
[605]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 37,45,46, 89, and 107<br>
and comprises one or more of the following naturally encoded amino acid substitutions: T79R,<br>
L80A, Y85L, Y85S, E87S (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1<br>
or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally<br>
encoded amino acids at one or more of the following positions: 37, 45, 46, 89, and 107 that is<br>
linked to a water soluble polymer and comprises one or more of the following naturally encoded<br>
amino acid substitutions: T79R, L80A, Y85L, Y85S, E87S (SEQ ID NO: 2 or the<br>
corresponding amino acids in SEQ ID NO: I or 3). In some embodiments, the hlFN polypeptide<br>
comprises one or more non-naturally encoded amino acids at one or more of the following<br>
positions: 37, 45, 46, 89, and 107 that is bonded to a water soluble polymer and comprises one<br>
or more of the following naturally encoded amino acid substitutions: T79R, L80A, Y85L,<br>
Y85S, E87S (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3).<br>
[606]	In some embodiments, the hlFN polypeptide comprises one or more non-<br>
naturally encoded amino acids at one or more of the following positions: 23,24, 25, 26, 27, 28, 30, 31, 32, 33, 128, 129, 131, 132, 133, 134, 135, 136, 137, 158, 159, 160, 161, 162, 163, 164, 165 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 128, 129, 131, 132,133, 134, 135, 136, 137, 158, 159, 160, 161, 162, 163, 164, 165 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 23, 24, 27, 31, 128, 131, 134, 158 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 23, 24, 27, 31, 128, 131, 134, 158 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions: 24, 27, 31, 128, 131, 134 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). In some embodiments, the hlFN polypeptide comprises one or more non-naturally encoded amino acids at one or more of the following positions that is linked or bonded to a water soluble polymer: 24, 27, 31, 128, 131,134 (SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3). Example 2<br><br><br><br><br><br><br><br><br>
[607]	This example details cloning and expression of a modified hlFN polypeptide in<br>
E. coli<br>
[608]	This example demonstrates how a hlFN polypeptide including a non-naturally<br>
encoded amino acid can be expressed m E, coli See Nagata et. al.. Nature, vol. 284, 316-320<br>
(1980) and U.S. Patent No. 4,364,863, which is incorporated by reference. cDNA encoding the<br>
full length hlFN and the mature form of hlFN lacking the N-terminal signal sequence are shown<br>
m SEQ ID NO: 21 and SEQ ED NO: 22, respectively. The full length and mature hlFN encoding<br>
cDNA is inserted into the pBAD HISc, pET20b, and pET19b expression vectors following<br>
optimization of the sequence for cloning and expression without altering amino acid sequence.<br>
[609]	An introduced translation system that comprises an orthogonal tRNA (0-tRNA)<br>
and an orthogonal aminoacyl tRNA synthetase (0-RS) is used to express hlFN containing a non-naturally encoded amino acid. The 0-RS preferentially aminoacylates the 0-tRNA with a non-naturally encoded amino acid.  In turn the translation system inserts the non-naturally encoded amino acid into hlFN, in response to an encoded selector codon. Table 2: 0-RS and 0-tRNA sequences.<br>
[610]	The transformation of E. coli with plasmids containing the modified hlFN gene<br>
and the orthogonal aminoacyl tRNA synthetase/tRNA pair (specific for the desired non-naturally<br><br>
encoded amino acid) allows the site-specific incorporation of non-naturally encoded amino acid into the hlFN polypeptide. The transformed E, coli, grown at 37°C in media containing between 0.01 - 100 mM of the particular non-naturally encoded amino acid, expresses modified hlFN with high fidelity and efficiency. The His-tagged hlFN containing a non-naturally encoded amino acid is produced by the E. coli host cells as inclusion bodies or aggregates. The aggregates are solubilized and affinity purified under denaturing conditions in 6M guanidine HCl. Refolding is performed by dialysis at 4oC overnight in 50mM TRIS-HCl, pH 8.0, 40µM CUSO4, and 2% (w/v) Sarkosyl The material is then dialyzed against 20mM TRIS-HCl, pH 8.0, 100mM NaCl, 2mM CaCl2, followed by removal of the His-tag. See Boissel et al., (1993) 268:15983-93. Methods for purification of hlFN are well known in the art and are confirmed by SDS-PAGE, Western Blot analyses, or electrospray-ionization ion trap mass spectrometry and the like.<br>
[611]	To generate hlFN polypeptides with a 6 His tag at the N terminus, hlFN<br>
nucleotide sequences were cloned downstream of the tag. The transformation of E. coli (BL21 (DE3)) with constructs containing the hlFN polynucleotide sequence and the orthogonal aminoacyl tRNA synthetase/tRNA pair (specific for the desired non-naturally encoded amino acid) allowed site-specific incorporation of non-naturally encoded amino acid (p-acetyl-phenylalanine) into the hlFN polypeptide.<br>
Example 3<br>
[612]	This example details introduction of a carbonyl-containing amino acid and<br>
subsequent reaction with an aminooxy-containing PEG.<br>
[613]	This Example demonstrates a method for the generation of a hlFN polypeptide<br>
that incorporates a ketone-containing non-naturally encoded amino acid that is subsequently<br>
reacted with an aminooxy-containing PEG of approximately 5,000 MW.  Each of the residues<br>
identified according to the criteria of Example 1, including but not limited to, 100, 106, 107,<br>
108, 111, 113, 114 (hlFN) is separately substituted with a non-naturally encoded amino acid<br>
having the following structure:<br><br><br>
[614]	The sequences utilized for site-specific incorporation of p-acetyl-phenylalanine<br>
into hlFN are SEQ ID NO: 2 (hlFN), and SEQ ID NO: 4 (muttRNA), and 16, 17 or 18 (TyrRS<br>
LWl, 5, or 6) described in Example 2 above.	(<br>
[615]	Once modified, the hlFN polypeptide variant comprising the carbonyl-containing<br>
amino acid is reacted with an aminooxy-containing PEG derivative of the form: R-PEG(N)-0-(CH2)n-0-NH2<br>
where R is methyl, n is 3 and N is approximately 5,000 MW. The purified hlFN containing p--acetylphenylalanine dissolved at 10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, MO) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, MO) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, MO) pH 4.5, is reacted with a 10 to 100-fold excess of aminooxy-containing PEG, and then stirred for 10 - 16 hours at room temperature (Jenclcs, W, J. Am, Chem. Soc. 1959, 81, pp 475). The PEG-hlFN is then diluted into appropriate buffer for immediate purification and analysis.<br>
Example 4<br>
[616]	Conjugation with a PEG consisting of a hydroxylamine group linked to the PEG<br>
via an amide linkage.<br>
[617]	A PEG reagent having the following structure is coupled to a ketone-containing<br>
non-naturally encoded amino acid using the procedure described in Example 3:<br>
R-PEG(N)-O-(CH2)2-NH-C(O)(CH2)n-O-NH2<br>
where R = methyl, n=4 and N is approximately 20,000 MW.   The reaction, purification, and<br>
analysis conditions are as described in Example 3.<br>
Example 5<br>
[618]	This example details the introduction of two distinct non-naturally encoded<br>
amino acids into hlFN polypeptides.<br>
[619]	This example demonstrates a method for the generation of a hlFN polypeptide<br>
that incorporates non-naturally encoded amino acid comprising a ketone functionality at two positions among the residues identified according to Example 1, wherein X* represents a non-naturally encoded amino acid.   The hlFN polypeptide is prepared as described in Examples 1 and 2, except that the selector codon is introduced at two distinct sites within the nucleic acid. Example 6<br>
[620]	This example details conjugation of hlFN polypeptide to a hydrazide-containing<br>
PEG and subsequent in situ reduction.<br><br>
[621]	A hIFN polypeptide incorporating a carbonyl-containing amino acid is prepared<br>
according to the procedure described in Examples 2 and  3.    Once modified, a hydrazide-containing PEG having the followmg structure is conjugated to the hlFN polypeptide;      ' R-PEG(N)-O-(CH2)2-NH-C(O)(CH2)n-X-NH-NH2<br>
where R = methyl, n=2 and N = 10,000 MW and X is a carbonyl (C=0) group. The purified hlFN containing ;?-acetylphenylalanine is dissolved at between 0.1-10 mg/mL in 25 mM MES (Sigma Chemical, St. Louis, MO) pH 6.0, 25 mM Hepes (Sigma Chemical, St. Louis, MO) pH 7.0, or in 10 mM Sodium Acetate (Sigma Chemical, St. Louis, MO) pH 4.5, is reacted with a 1 to 100-fold excess of hydrazide-containing PEG, and the corresponding hydrazone is reduced in sitv by addition of stock IM NaCNBHs (Sigma Chemical, St, Louis, MO), dissolved in H2O, to a final concentration of 10-50 mM. Reactions are carried out in the dark at 4 °C to RT for 18-24 hours. Reactions are stopped by addition of 1 M Tris (Sigma Chemical, St. Louis, MO) at about pH 7.6 to a final Tris concentration of 50 mM or diluted into appropriate buffer for immediate purification.<br>
Example 7<br>
[622]	This example details introduction of an alkyne-containing amino acid into a hlFN<br>
polypeptide and derivatization with mPEG-azide.<br>
[623]	Any of the residues of hlFN identified according to Example 1, including but not<br>
limited to, 100, 106, 107, 108, 111, 113, 114 are substituted with this non-naturally encoded<br>
amino acid:<br><br>
[624]	The sequences utilized for site-specific incorporation of p-propargyl-tyrosine into<br>
hlFN are SEQ E) NO: 2 (hlFN), SEQ ID NO: 4 (muttRNA, M. jannaschii mtRNA), and 9,<br>
10 or 11 described in Example 2 above.    The hlFN pol ypeptide containing the propargyl<br>
tyrosine is expressed in E. coli and purified using the conditions described in Example 3.<br>
[625]	The purified hlFN containing propargyl-tyrosine dissolved at between 0.1-10<br>
mg/mL in PB buffer (100 mM sodium phosphate, 0.15 M NaCl, pH = 8) and a 10 to 1000-fold excess of an azide-containing PEG is added to the reaction mixture. A catalytic amount of CUSO4 and Cu wire are then added to the reaction mixture.   After the mixture is incubated<br><br>
(including but not limited to, about 4 hours at room temperature or 37° C, or overnight at 4°C),<br>
H2O is added and the mixture is Filtered through a dialysis membrane. The sample can be<br>
analyzed for the addition.<br>
[626]	In this Example, the PEG has the following structure:<br>
R-PEG(N)-0-(CH2)2-NH-C(0)(CH2)n-N3<br>
where R is methyl, n is 4 and N is 10,000 MW.<br>
Example 8<br>
[627]	This example details substitution of a large, hydrophobic amino acid in a hlFN<br>
polypeptide with propargyl tyrosine.<br>
[628]	A Phe, Trp or Tyr residue present within one the following regions of hlFN: 1-9<br>
(N-terminus), 10-21 (A helix), 22-39 (region between A helix and B helix), 40-75 (B helix), 76-<br>
77 (region between B helix and C helix), 78-100 (C helix), 101-110 (region between C helix and<br>
D helix), 111-132 (D helix),  133-136 (region between D and E helix), 137-155 (E helix), 156-<br>
165 (C-terminus), (as in SEQ ID NO: 2 or the corresponding amino acids of other IFN<br>
polypeptides), is substituted with the following non-naturally encoded amino acid as described<br><br>
[629]	Once modified, a PEG is attached to the hlFN polypeptide variant comprising the<br>
alkyne-containing amino acid. The PEG will have the following structure:<br>
Me-PEG(N)-0-(CH2)2-N3<br>
and coupling procedures would follow those in Example 7. This generates a hlFN polypeptide variant comprising a non-naturally encoded amino acid that is approximately isosteric with one of the naturally-occurring, large hydrophobic amino acids and which is modified with a PEG derivative at a distinct site within the polypeptide.<br>
Example 9<br>
[630]	This example details generation of a hlFN polypeptide homodimer, heterodimer,<br>
homomultimer, or heteromultimer separated by one or more PEG linkers.<br>
[631]	The alkyne-containing hlFN polypeptide variant produced in Example 7 is<br>
reacted with a bifunctional PEG derivative of the form:<br>
N3-(CH2)n-C(O)-NH-(CH2)2-O-PEG(N)-O-(CH2)2-NH-C(O)-(CH2)n-N3<br><br>
where n is 4 and the PEG has an average MW of approximately 5,000, to generate the corresponding hlFN polypeptide homodimer where the two hlFN molecules are physically separated by PEG. In an analogous manner a hlFN polypeptide may be coupled to one or more other polypeptides to form heterodimers, homomultimers, or heteromultimers. Coupling, purification, and analyses will be performed as in Examples 7 and 3.<br>
Example 10<br>
[632]	This example details coupling of a saccharide moiety to a hlFN polypeptide.<br>
[633]	One residue of the following is substituted with the non-natural encoded amino<br>
acid below: 1, 2, 3, 4, 5, 6, 7, 8, 9, 12, 13, 16, 19, 20, 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 40, 41, 42, 45, 46, 48, 49, 50, 51, 58, 61, 64, 65, 68, 69, 70, 71, 73, 74, 77, 78, 79, 80, 81, 82, 83, 85, 86, 89, 90, 93, 94, 96, 97,100, 101, 103, 105, 106, 107, 108, 109,110, 111,112,113, 114, 117, 118, 120, 121, 124, 125, 127, 128, 129, 131, 132, 133, 134, 135, 136, 137, 148, 149, 152, 153, 156, 158, 159, 160, 161, 162, 163, 164, 165 (as in SEQ ID NO: 2, or the corresponding amino acids of other IFN polypeptides).<br><br>
[634]	Once modified, the hlFN polypeptide variant comprising the carbonyl-containing<br>
amino acid is reacted with a β-linked aminooxy analogue of N-acetylglucosamine (GlcNAc). The hlFN polypeptide variant (10 mg/mL) and the aminooxy saccharide (21 mM) are mixed in aqueous 100 mM sodium acetate buffer (pH 5.5) and incubated at 37°C for 7 to 26 hours. A second saccharide is coupled to the first enzymatically by incubating the saccharide-conjugated hlFN polypeptide (5 mg/mL) with UDP-galactose (16 mM) and (i-l,4-galac3l:osyItransferase (0.4 units/mL) in 150 mM HEPES buffer (pH 7.4) for 48 hours at ambient temperature (Schanbacher et al. 1 Biol Chem. 1970, 245, 5057-5061). Example 11<br>
[635]	This example details generation of a PEGylated hlFN polypeptide antagonist.<br>
[636]	One of the following residues, 2, 3, 4, 5, 7, 8, 16, 19, 20, 40, 42,, 50, 51, 58, 68,<br>
69, 70, 71, 73, 97, 105, 109, 112, 118, 148, 149, 152, 153, 158, 163, 164, 165, (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) is substituted with the following non-naturally encoded amino acid as described in Example 3; a hlFN polypeptide comprising one of these substitutions may potentially act as a weak antagonist or weak agonist depending on<br><br>
the site selected and the desired activity. One of the following residues, 22, 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 34, 35, 74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94, 124, 125, 127, 128, 129,131, 132,133, 134,135,136,137, (hlFN; SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3) is substituted with the folio whig non-naturally encoded amino acid as described in Example 3.<br><br>
[637]	Once modified, the hlFN polypeptide variant comprising the carbonyl-containing<br>
amino acid will be reacted with an aminooxy-containing PEG derivative of the form: R-PEG(N)-O-(CH2)n-O-NH2<br>
where R is methyl, n is 4 and N is 20,000 MW to generate a hlFN polypeptide antagonist comprising a non-naturally encoded amino acid that is modified with a PEG derivative at a single site within the polypeptide. Coupling, purification, and analyses are performed as in Example 3.<br>
Example 12<br>
Generation of a hlFN polypeptide homodimer, heterodimer, homomultimer, or heteromultimer<br>
in which the hlFN Molecules are Linked Directly<br>
[638]	A hlFN polypeptide variant comprising the alkyne-containing amino acid can be<br>
directly coupled to another hlFN polypeptide variant comprising the azido-containing amino<br>
acid, each of which comprise non-naturally encoded amino acid substitutions at the sites<br>
described in, but not limited to, Example 10.   This will generate the corresponding hlFN<br>
polypeptide homodimer where the two hlFN polypeptide variants are physically joined.  In an<br>
analogous manner a hlFN polj'peptide polypeptide may be coupled to one or more other<br>
polypeptides to form heterodimers, homomultimers, or heteromultimers. Coupling, purification,<br>
and analyses are performed as in Examples 3, 6, and 7.<br>
Example 13<br>
PEG-OH + Br-(CH2)n-C=CR'-&gt; PEG-O-(CH2)n-CsCR'<br>
A	B<br>
[639]	The polyalkylene glycol (P-OH) is reacted with the alkyl halide (A) to form the<br>
ether (B).   In these compounds, n is an integer from one to nine and R' can be a straight- or<br>
branched-chain, saturated or unsaturated C1, to C20 alkyl or heteroalkyi group. R' can also be a<br><br>
C3 to C7 saturated or unsaturated cyclic allcyl or cyclic heteroalkyl, a substituted or unsubstituted aryl or heteroaryl group, or a substituted or unsubstituted alkaryl (the alkyl is a C1 to C20 saturated or unsaturated alkyl) or heteroalkaryl group. Typically, PEG-OH is polyethylene glycol (PEG) or monomethoxy polyethylene glycol (mPEG) having a molecular weight of 800 to 40,000 Daltons (Da).<br><br>
[640]	mPEG-OH with a molecular weight of 20,000 Da (mPEG-OH 20 kDa; 2,0 g, 0.1<br>
mmol, Sunbio) was treated with NaH (12 mg, 0.5 mmol) in THF (35 mL). A solution of propargyl bromide, dissolved as an 80% weight solution in xylene (0.56 mL, 5 mmol, 50 equiv., Aldrich), and a catalytic amount of KI were then added to the solution and the resulting mixture was heated to reflux for 2 hours. Water (1 mL) was then added and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. This CH2Cl2 solution was added to diethyl ether (150 mL) drop-wise. The resulting precipitate was collected, washed with several portions of cold diethyl ether, and dried to afford propargyl-O-PEG.<br><br>
[641]	The mPEG-OH with a molecular weight of 20,000 Da (mPEG-OH 20 kDa; 2.0 g,<br>
0.1 nrniol, Sunbio) was treated with NaH (12 mg, 0,5 mmol) in THF (35 mL). Fifty equivalents of 5-bromo-l-pentyne (0.53 mL, 5 mmol, Aldrich) and a catalytic amount of KI were then added to the mixture. The resulting mixture was heated to reflux for 16 hours. Water (1 mL) was then added and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. This CH2Cl2 solution was added to diethyl ether (150 mL) drop-wise. The resulting precipitate was collected, washed with several portions of cold diethyl ether, and dried to afford the corresponding allcyne. 5-chloro-l-pentyne may be used in a similar reaction.<br><br><br><br>
[642]	To a solution of 3-hydroxybenzyIalcohol (2.4 g, 20 mmol) in THF (50 mL) and<br>
water (2.5 mL) was first added powdered sodium hydroxide (1.5 g, 37.5 mmol) and then a<br>
solution of propargyl bromide, dissolved as an 80% weight solution in xylene (3.36 mL, 30<br>
mmol). The reaction mixture was heated at reflux for 6 hours. To the mixture was added 10%<br>
citric acid (2.5 mL) and the solvent was removed under vacuum. The residue was extracted with<br>
ethyl acetate (3x15 mL) and the combined organic layers were washed with saturated NaCl<br>
solution (10 mL), dried over MgSO4 and concentrated to give the 3-propargyloxybenzyl alcohol.<br>
[643]	Methanesulfonyl chloride (2.5 g, 15.7 mmol) and triethylamine (2,8 mL, 20<br>
mmol) were added to a solution of compound 3 (2.0 g, 11.0 mmol) m CH2Cl2 at 0°C and the reaction was placed in the refrigerator for 16 hours. A usual work-up afforded the mesylate as a pale yellow oil. This oil (2.4 g, 9.2 mmol) was dissolved in THF (20 mL) and LiBr (2.0 g, 23.0 mmol) was added. The reaction mixture was heated to reflux for 1 hour and was then cooled to room temperature. To the mixture was added water (2.5 mL) and the solvent was removed under vacuum. The residue was extracted with ethyl acetate (3 x 15 mL) and the combined organic layers were washed with saturated NaCI solution (10 mL), dried over anhydrous Na2SO4, and concentrated to give the desired bromide.<br>
[644]	mPEG-OH 20 kDa (1.0 g, 0.05 mmol, Sunbio) was dissolved in THF (20 mL)<br>
and the solution was cooled in an ice bath. NaH (6 mg, 0.25 mmol) was added with vigorous stirring over a period of several minutes followed by addition of the bromide obtained from above (2.55 g, 11.4 mmol) and a catalytic amount of KI. The cooling bath was removed and the resulting mixture was heated to reflux for 12 hours. Water (1.0 mL) was added to the mixture and the solvent was removed under vacuum. To the residue was added CH2CI2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. Dropwise addition to an ether solution (150 mL) resulted in a white precipitate, which was collected to yield the PEG derivative.<br><br><br>
[645]	The terminal alkyne-containing poly(ethylene glycol) polymers can also be<br>
obtained by coupling a poly(ethylene glycol) polymer containing a terminal functional group to a reactive molecule containing the alkyne functionality as shown above, n is between 1 and 10. R' can be H or a small alkyl group from C1 to C4.<br>
Example 18<br>
(1)	H02C-(CH2)2-CsCH + NHS +DCC-&gt; NHSO-C(O)-(CH2)2-CsCH<br>
(2)	mPEG-NH2 + NHSO-C(O)-(CH2) 2-C=CH -&gt; mPEG-NH-C(O)-(CH2)2-CsCH<br>
[646]	4-pentynoic acid (2.943 g, 3.0 mmol) was dissolved in CH2Cl2 (25 mL).   N-<br>
hydroxysuccinimide (3.80 g, 3.3 mmol) and DCC (4.66 g, 3,0 mmol) were added and the solution was stirred overnight at room temperature. The resulting crude NHS ester 7 was used in the following reaction without further purification.<br>
[647]	mPEG-NH2 with a molecular weight of 5,000 Da (mPEQ-NH2, 1 g, Sunbio) was<br>
dissolved in THF (50 mL) and the mixture was cooled to 4oC. NHS ester 7 (400 mg, 0.4 mmol) was added portion-wise with vigorous stirring. The mixture was allowed to stir for 3 hours while warming to room temperature. Water (2 mL) was then added and the solvent was removed under vacuum. To the residue was added CH2Cl2 (50 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and the volume was reduced to approximately 2 mL. This CH2Cl2 solution was added to ether (150 mL) drop-wise. The resulting precipitate was collected and dried in vacuo.<br>
Example 19<br>
[648]	This Example represents the preparation of the methane sulfonyl ester of<br>
poly(ethylene glycol), which can also be referred to as the methanesulfonate or mesylate of poly(ethylene glycol). The corresponding tosylate and the halides can be prepared by similar procedures.<br>
mPEG-OH + CH3SO2Cl + N(Et) 3 -&gt; mPEG-O-SO2CH3 -&gt; mPEG-N3<br>
[649]	The mPEG-OH (MW = 3,400, 25 g, 10 mmol) in 150 mL of toluene was<br>
azeotropically distilled for 2 hours under nitrogen and the solution was cooled to room temperature. 40 mL of dry CH2Cl2 and 2.1 mL of dry triethylamine (15 mmol) were added to the solution.   The solution was cooled in an ice bath and 1.2 mL of distilled methanesulfonyl<br><br>
chloride (15 mmol) was added dropwise. The solution was stirred at room temperature under nitrogen overnight, and the reaction was quenched by adding 2 mL of absolute ethanol. The mixture was evaporated under vacuum to remove solvents, primarily those other than toluene, filtered, concentrated again under vacuum, and then precipitated into 100 mL of diethyl ether. The filtrate was washed with several portions of cold diethyl ether and dried in vacuo to afford the mesylate.<br>
[650]	The mesylate (20 g, 8 mmol) was dissolved in 75 ml of THF and the solution was<br>
cooled to 4 °C. To the cooled solution was added sodium azide (1.56 g, 24 mmol). The reaction was heated to reflux under nitrogen for 2 hours. The solvents were then evaporated and the residue diluted with CH2Cl2 (50 mL). The organic fraction was washed with NaCl solution and dried over anhydrous MgSO4. The volume was reduced to 20 ml and the product was precipitated by addition to 150 ml of cold dry ether.<br>
Example 20<br>
(1)	N3-C6H4-CO2H -&gt; N3-C6H4CH2OH<br>
(2)	N3-C6H4CH2OH -&gt; Br-CH2-C6H4-N3<br>
(3)	mPEG-OH + BT-CH2-C6H4-N3 -&gt; mPEG"0-CH2-C6H4-N3<br>
[651]	4-azidobenzyl alcohol can be produced using the method described in U.S. Patent<br>
5,998,595, which is incorporated by reference herein. Methanesulfonyl chloride (2,5 g, 15.7<br>
mmol) and triethylamine (2.8 mL, 20 mmol) were added to a solution of 4-azidobenzyl alcohol<br>
(1.75 g, 11.0 mmol) in CH2Cl2 at 0 °C and the reaction was placed in the refrigerator for 16<br>
hours. A usual work-up afforded the mesylate as a pale yellow oil. This oil (9.2 mmol) was<br>
dissolved in THF (20 mL) and LiBr (2.0 g, 23.0 mmol) was added. The reaction mixture was<br>
heated to reflux for 1 hour and was then cooled to room temperature. To the mixture was added<br>
water (2.5 mL) and the solvent was removed under vacuum. The residue was extracted with<br>
ethyl acetate (3x15 mL) and the combined organic layers were washed with saturated NaCl<br>
solution (10 mL), dried over anhydrous Na2SO4, and concentrated to give the desired bromide.<br>
[652]	mPEG-OH 20 kDa (2.0 g, 0.1 mmol, Sunbio) was treated with NaH (12 mg, 0.5<br>
mmol) in THF (35 mL) and the bromide (3,32 g, 15 mmol) was added to the mixture along with a catalytic amount of KI. The resulting mixture was heated to reflux for 12 hours. Water (1.0 mL) was added to the mixture and the solvent was removed under vacuum. To the residue was added CH2Cl2 (25 mL) and the organic layer was separated, dried over anhydrous Na2SO4, and<br><br>
the volume was reduced to approximately 2 mL. Dropwise addition to an ether solution (150 mL) resulted in a precipitate, which was collected to yield mPEG-O-CH2-C6H4--N3.<br>
Example 21<br>
NH2-PEG-O-CH2CH2C02H + N3-CH2CH2CO2-NHS -&gt; N3-CH2CH2-C(0)NH-PEG-O-CH2CH2CO2H<br>
[653]	NH2-PEG-O-CH2CH2CO2H (MW 3,400 Da, 2.0 g) was dissolved in a saturated<br>
aqueous solution of NaHCOs (10 mL) and the solution was cooled to 0°C. S-azido-l-N-hydroxysuccinimido propionate (5 equiv.) was added with vigorous stirring. After 3 hours, 20 mL of H2O was added and the mixture was stirred for an additional 45 minutes at room temperature. The pH was adjusted to 3 with 0.5 N H2SO4 and NaCl was added to a concentration of approximately 15 wt%. The reaction mixture was extracted with CH2C12 (100 mL X 3), dried over Na2SO4 and concentrated. After precipitation with cold diethyl ether, the product was collected by filtration and dried under vacuum to yield the omega-carboxy-azide PEG derivative.<br>
Example 22<br>
mPEG-OMs = HCsCLi -&gt; mPEG-O-CH2-CH2-OC-H<br>
[654]	To a solution of lithium acetylide (4 equiv.), prepared as known in the art and<br>
cooled to -78°C in THF, is added dropwise a solution of mPEG-OMs dissolved in THF with vigorous stirring. After 3 hours, the reaction is permitted to warm to room temperature and quenched with the addition of 1 mL of butanol. 20 mL of H2O is then added and the mixture was stirred for an additional 45 minutes at room temperature. The pH was adjusted to 3 with 0.5 N H2SO4 and NaCl was added to a concentration of approximately 15 wt%. The reaction mixture was extracted with CH2Cl2 (100 mL x 3), dried over Na2SO4 and concentrated. After precipitation with cold diethyl ether, the product was collected by filtration and dried under vacuum to yield the l-(but-3-ynyloxy)-methoxypolyethylene glycol (mPEG). Example 23<br>
[655]	The  azide-  and  acetylene-containing  amino  acids  were  incorporated  site-<br>
selectively into proteins using the methods described in L. Wang, et al., (2001), Science 292:498-500, J.W. Chin et al.. Science 301:964-7 (2003)), J. W. Chin et al,, (2002), Journal of the American Chemical Society 124:9026-9027; J. W. Chin, &amp; P. G. Schultz, (2002), Chem Bio<br><br>
Chem 11:1135-1137; J, W. Chin, et al., (2002), PNAS United States of America 99:11020-11024: and, L. Wang, &amp; P. G. Schultz, (2002), Chem. Comm.. 1-10. Once the amino acids were incorporated, the cycloaddition reaction was carried out with 0,01 mM protein in phosphate buffer (PB), pH 8, in the presence of 2 mM PEG derivative, 1 mM CUSO4, and -1 mg Cu-wire for 4 hours at 37 oC.<br>
[656]	The synthesis of p-Acetyl-D,L-phenylalanine (pAF) and m-PEG-hydroxylamine<br>
derivatives is performed as described below. The racemic pAF is synthesized using the<br>
previously described procedure in Zhang, Z., Smith, B. A. C, Wang, L., Brock, A., Cho, C. &amp;<br>
Schultz, P. G,, Biochemistry, (2003) 42, 6735-6746. To synthesize the m-PEG-hydroxylamine<br>
derivative, the following procedures are completed. To a solution of (N-t-Boc-aminooxy)acetic<br>
acid (0.382 g, 2.0 mmol) and 1,3-Diisopropyicarbodiimide (0.16 mL, 1.0 mmol) in<br>
dichloromethane (DCM, 70mL), which is stirred at room temperature (RT) for 1 hour, methoxy-<br>
polyethylene glycol amine (m-PEG-NH2, 7.5 g, 0.25 mmol, Mt. 30 K, from BioVectra) and<br>
Diisopropylethylamine (0.1 mL, 0.5 mmol) are added. The reaction is stirred at RT for 48 hours,<br>
and then is concentrated to about 100 mL. The mixture is added dropwise to cold ether (800<br>
mL). The t-Boc-protected product precipitated out and is collected by filtering, washed by ether<br>
3xl00mL. It is further purified by re-dissolving in DCM (100 mL) and precipitating in ether<br>
(800 mL) twice. The product is dried in vacuum, confirmed by NMR and Nihydrin test.<br>
[657]	The deBoc of the protected product (7.0 g) obtained above is carried out in 50%<br>
TFA/DCM (40 mL) at 0 °C for 1 hour and then at RT for 1.5 hour. After removing most of TFA in vacuimi, the TFA salt of the hydroxylamine derivative is converted to the HCl salt by adding 4N HCl in dioxane (ImL) to the residue. The precipitate is dissolved in DCM (50 mL) and re-precipitated m ether (800 mL). The final product is collected by filtering, washed with ether 3x lOOmL, dried in vacuum, stored under nitrogen. Other PEG (5K, 20K) hydroxylamine derivatives can be synthesized using the same procedure. Example 24<br>
[658]	hlFN polypeptides were isolated according to the following protocol. Variations<br>
to the refolding method have also been performed to isolate hlFN polypeptides of the invention, modifying the composition of the buffer that receives the solubilized IFN. Additional modifications such as the addition of Cu++ or other mild oxiding agents during or after the procedure, altering denaturing agents or reducing agents, altering the pH to assess refolding efficiency at various pHs may be performed. Colunm-based refolding methods using HIC, dye-<br><br>
based, size-exclusion, or ion-exchange resins may be used for refolding. Other modifications to such methods are known to those of ordinary skill in the art.<br>
[659]	To generate hlFN polypeptides with a 6 His tag at the N terminus, hlFN<br>
nucleotide sequences were cloned downstream of the tag. The transformation of E. coli (BL21 (DE3)) with constructs containing the hlFN polynucleotide sequence and the orthogonal aminoacyl tRNA synthetase/tRNA pair (specific for the desired non-naturally encoded amino acid) allowed site-specific incorporation of non-naturally encoded amino acid (p-acetyl-phenylalanine) into the hlFN polypeptide.<br>
Inclusion Body Preparation and Refolding<br>
[660]	Fresh or frozen E. coli host cell pellets were resuspended in approximately<br>
lOmL/g of 20mM Tris pH 7.5, 200 mM NaCl, 1 mM EDTA.  The pellets were sonicated six<br>
times for 30 seconds with 1 minute incubations on ice between each sonication.   1 mg/mL<br>
lysozyme and DNase I were added to each sonicated sample, and the samples were incubated for<br>
30 minutes at room temperature.<br>
[661]	The samples were centrifuged at 12,000 rpm for 10 minutes at 4°C, and the<br>
supematants for each sample were removed for analysis. The pellets were washed three times<br>
with 40 ml of chilled IB Wash Buffer 1 (20 mM Tris, pH 7.5, 100 mM NaCl, 1 mM EDTA, 1%<br>
Triton) by detaching them from the sides of the tubes and resuspending the pellets by sonication<br>
for 30 seconds,   In between each wash, the samples were centrifuged at 12,000 rpm for 5<br>
minutes at 4°C. The pellets were then resuspended/washed twice with 40 ml of chilled IB Wash<br>
Buffer 2 (20 mM Tris, pH 7.5, 100 mM NaCl, 1 mM EDTA).   In between each wash, the<br>
samples were centnfuged at 12,000 rpm for 5 minutes at 4°C.<br>
[662]	After the two sets of washes, the pellets were solubilized by douncing in 5-20 ml<br>
of Solubilization Buffer (50mM Tris, pH 7.5, 8M Gdn-HCl).  The minimum amount of buffer<br>
necessary to completely solubilize each pellet was used.   Samples were then centrifiiged at<br>
12,000 for 15 minutes at 4°C to remove any insoluble particles, and the supematants were<br>
transferred into fresh tubes.<br>
[663]	An aliquot was taken from each sample, and the aliquots were diluted 20X in<br>
Solubilization Buffer, and the OD280 for each sample was measured to determine protein<br>
concentration. The Extinction coefficient at 280 nm is 22,500 M-1 or 1.17 mg/ml -1.<br>
[664]	The inclusion bodies were aliquoted into 3-5 ml aliquots and stored at -80°C.<br>
0.15 mg/ml IFN final (5 mg/mL solution in 6-8M GndHCl, pH 8) were injected in three aliquots<br><br>
into a fast-stirring buffer (20-50mM Tris HCI, pH 8.2, 0.5 M L-arginine, 10% Glycerol or Sucrose) (plus 200-250 mM Gnd from solubilized IFN) at 4°C. The injections were performed under the surface of the solution to avoid foaming. After injection, slow stirring was used for approximately 16-24 hours at 4°C. After the completion of the refolding reaction, the samples were processed further.<br>
[665]	The samples were concentrated with Amicon Stirring cells (YM-10 membrane)<br>
to approximately 1-1.5 mg/ml (15-30 ml). The samples were then dialyzed against 30 mM Tris, pH 7.8, 20 mM NaCI, 5% glycerol overnight at 4°C. The samples were diluted to 50 ml with Q HP Buffer A (10 mM Tris, pH 7.5), and purification was performed on a 5 ml Q HP AICTA column using a gradient of 0-30% B (10 mM Tris, pH 7.5, 1 M NaCl) over fifteen column volumes. Tlie column was washed with 2,5 M NaCl, followed by IM NaCl / IM NaOH. SDS-PAGE analysis was performed on the column fi-actions, and the fiuctions containing monomeric refolded hlFN polypeptides were pooled. The pH of each pool was adjusted by adding 1/20 volume of IM NaAc, pH 4.5, and each pool was dialyzed overnight against 20 mM NaAc, pH 4, 20 mM NaCl, 5% glycerol. The hlFN polypeptides were concentrated to &gt;1 mg/ml for PEGylation. PEGylation<br>
[666]	Oxyamino-derivatized 30K PEG is added to hlFN polypeptide at a 1:12 molar<br>
ratio (30mg / mg IFN/ml), and the mixture is incubated at 28°C for 16-48 hours. Figure 19 shows the 30K PEG used in the conjugation. Isolation of PEGylated hlFN polypeptides<br>
[667]	PEGylated hlFN polypeptides were isolated using the Source Q 30 column. The<br>
buffers used with this column were Buffer A (lOmM Tris, pH 8.0); Buffer B (10mM Tris, pH 8.0; IM NaCl); and Sanitization Buffer (IM NaOH: IM NaCI). The fractions of PEGylated hlFN were pooled, dialyzed against the following storage buffer: 20mM NaAcetate, pH 6, 0.005% Tween, 125 mM NaCl. The samples were then concentrated to &gt; 8 microM and stored at 4^C.<br>
[668]	Buffer lines were primed and washed with the appropriate buffers, and the<br>
column and sample line were sanitized with 30-40 mL Sanitization Buffer. The sample line was rinsed with Buffer A to remove any traces of Sanitization Buffer, and the line was primed for sample addition. The column was equilibrated with 5-10 column volumes of Buffer A, until parameters such as UV and Conductivity were stable.<br><br>
[669]	A sample of PEGylated hlFN polypeptide was injected onto the column, and the<br>
column was washed with 4 column volumes of Buffer A. The gradient used was the following: 0-20% Buffer B for 20 column volumes; 100% Buffer B for 4 column volumes. l-L5mL fractions were collected from the column and were analyzed by SDS-PAGE to determine fractions for pooling. PEG monomer fractions were pooled. The column was prepared for the next sample by the sanitizing column and sample line with 30-40 mL Sanitization Buffer. Example 25<br>
[670]	This example details the measurement of hlFN activity and affinity of hlFN<br>
polypeptides for the hlFN receptor. Biacore studies f Receptor Bindin£ Affinity)<br>
[671]	The sequence for the IFNAR2 extmcellular domain (consisting of 206 amino<br>
acids ending with sequence LLPPGQ) was amplified from clone MHS1011-61064 (OpenBiosystems, Huntsville, AL). This insert was cloned into the pET20 expression vector (Novagen) downstream of the T7 promoter. Protein expression was induced with 0.4 mM IPTG in BL21(DE3) cells (Novagen).<br>
[672]	Since the expressed protein was insoluble, the inclusion bodies were purified<br>
from lysed cells and solubilized in 6M GndCl. A 5 ml aliquot (50 mg amotmt) was reduced with 10 mM DTT for 45 minutes at 37°C. Then the mixture was injected into 200 ml of refolding buffer which consisted of 50 mM Tris pH 8, 20 mM NaCl, 0.5 M Arginine, 10% glycerol at 4°C and incubated overnight with gentle stirring,<br>
[673]	The refolding reaction was then concentrated to 25 ml using an Amicon stirring<br>
cell, and dialyzed overnight against 20mM Tris, pH 8,20 mM NaCl, 10% glycerol. Monomeric refolded IFNAR BCD was purified on HP Q Sepharose using the AKTA FPLC system (Amersham). Purified IFNAR2 ECD was immobilized on CM5 Biacore chip using a lysine-specific coupling procedure recommended by the manufacturer. About 200 RUs of functional protein were immobilized. Various concentrations of IFN variants in HBS-EP buffer (Biacore) were injected at a flow rate of 50 mcl/minute over the flowcell containing immobilized IFNAR2, and a control flowcell containing immobilized bovine serum albumin. Sensograms generated were fit to the 1:1 interaction model to calculate kon, koff and Kd values using BiaEvaluation software (Biacore). N-terminal 6-His hlFN polypeptides comprising the non-naturally encoded amino acid p-acetyl-phenylalanine (pAF) were tested, as well as PEGylated hlFN polypeptides comprising pAF after isolation via the methods described in Example 25.  Controls utilized in<br><br>
these studies included IFNαA obtained from Sigma, N-6 His tagged wild-type IFNα and PEGASYS®. Data collected for hlFN polypeptides is shown in Table 3. Non-PEGylated hlFN polypeptide with a para-acetylphenylalanine substitution at position 149 of SEQ ID NO: 2 showed altered binding Icinetics in this assay (faster Icoff).<br><br><br><br><br><br>
[674]	To assess the biological activity of modified hIFNa2a polypeptides, an assay<br>
measuring phosphorylation of STATl, a signal transducer and activator of transcription family<br>
member, was performed using the human monocytic leukemia THP-l cells.   Activation of<br>
STATl has been shown to be essential for the antiviral activity of IFNs (Durbin JE et al. Cell<br>
1996 84:443-450).   The human monocyte line THP-1 was purchased from ATCC (Manassas^<br>
VA) and was routinely passaged in RPMl 1640, sodium pyruvate, penicillin, streptomycin, 10%<br>
heat-inactivated fetal bovine serum and 50uM 2-mercaptoethanol. The cells were maintained at<br>
37°C in a humidified atmosphere of 5% CO2.<br>
[675]	The THP-1 cells were starved overnight in assay media containing 1% heat<br>
inactivated charcoal/dextran treated fetal bovine serum before stimulation with increasing<br>
concentrations of hEFN polypeptides for 30 minutes at 37°C. All stimulated cells were fixed for<br>
permeabilization   and   stained  with  the   appropriate   phospho-antibody   as   suggested  by<br>
manufacturer (Cell Signaling Technology, Beverly, MA). Sample acquisition was performed on<br>
the FACS Array with acquired data analyzed on the Flowjo software (Tree Star Inc., Ashland,<br>
OR).   EC50 values were derived from dose response curves plotted with mean fluorescent<br>
intensity agamst protein concentration utilizing SigmaPlot. (Figure 2)<br>
[676]	Additional   controls   were   performed   to   confirm   that   the   IFN   induced<br>
phosphorylation of STATl was selective for the IFNa receptor 2 (IFNAR2).   Serum starved<br>
FHP-l were pre-incubated with increasing concentrations of IFNAR2 extracellular domain<br>
;ECDR2) for 15 minutes at 37°C before the addition of IFN at an EC80 dose. The extracellular<br>
iomain of the IFNa receptor 2 was found to compete effectively against IFNaA stimulated<br>
STATl phosphorylation indicating that IFNaA induced phosphorylation of STATl is selective<br>
1<br>
br the IFNAR2. Figure 3 shows that ECDR2 competes with IFNaA induced pSTAT activity.<br><br>
[677]	A potential contaminant after purification of hlFN polypeptides is endotoxin. To<br>
determine if endotoxin affects the phosphorylation of STATU a range of endotoxin levels were tested in this cellular assay. Figure 4 shows that endotoxin did not have a substantial effect on the assay at the levels tested. The endotoxin concentration range tested (as shown) provided comparable MFI (PE) to a control sample containing no endotoxin.<br>
[678]	Alternatively, phosphorylation of STATl may be measured with RayBio® Cell-<br>
Based Statl (Tyr701) ELISA kit (Raybiotech, Norcross, GA). With this kit, the assay is performed in 96 well plates and has a colorimetric readout measurable by a spectrophotometric plate reader.<br>
Measurement of Anti-proliferative Activity<br>
[679]	hlFN polypeptides were also assayed for anti-proliferative activity. A prominent<br>
effect of IFNa's is their ability to inhibit cell growth, which is of major importance in determining anti-tumor action. The human lymphoblastoid Daudi cell line has proven to be extremely sensitive to IFNa's, and it has been used to measure antiproliferative activity in many IFNa's and derived hybrid polypeptides (Meister et al, J Gen Virol. (1986) Aug; 67 (Pt 8): 1633-43). Use of this cell line has been facilitated by its ability to be grown in suspension cultures (Evinger and Pestka, (1981) Methods EnzymoL 79:362-368).<br>
[680]	The human Daudi B cell line was purchased from ATCC (Manassas, VA) and<br>
grown in RPMI 1640 supplemented with sodium pyruvate, penicillin, streptomycin (Invitrogen,<br>
Carlsbad, San Diego) and 10% heat inactivated fetal bovine serum (Hyclone, Logan, UT). The<br>
cell culture was maintained at 37αC in a humidified atmosphere of 5% CO2.<br>
[681]	The Daudi cells were plated at a density of 1 X 104 cells/well in a flat-bottom, 96-<br>
well plate. The cells were activated with increasing concentration of IFNa2A in triplicates per dose concentration. Following a 4-day incubation period at 37°C with 5% CO2, 40 ul of CellTiter 96 Aqueous One solution Reagent (Promega Corporation, Madison, WI) was added to each well, and the culture was allowed to incubate for an additional 3 hours. Absorbance was read at 490 nm using a Spectromax. EC50S were obtained from dose response curves plotted with OD490nm (average of triphcates) against protein concentration with SigmaPlot. See Figure 5.<br>
[682]	Since   a  potential   contaminant  after  purification  of hlFN  polypeptides  is<br>
endotoxin, a range of endotoxin levels were tested in the anti-proliferation assay. Figure 6 shows that endotoxin did not have a substantial affect on the assay at the levels tested.<br><br>
Measurement of Phosvhorylated Tvk2<br>
[683]	Kawamoto et al. in Experimental Hematology 2004 32:797-805 and Ishida et al.<br>
Experimental Hematology 2005 33:495-503, which are incorporated by reference herein, discuss signaling differences between limitin and IFNα involving Tyk2, CrIcL, Crldl, and Daxx and side effects of IFN. Assays measuring phosphorylation of CrkL, Crkll and Tyk2 may be used to evaluate hlFN polypeptides of the invention. Evaluating the phosphorylation of Crkll in U-266 may be performed as described by Platanias et al. in Experimental Hematology 1999; 27:1315-1321, which is incorporated by reference herein. For measuring the phosphorylation of Tyk2, the human U266B1 cells (ATCC, Manassas, VA) were maintained in RPMI 1640 supplemented with sodium pyruvate, penicillin, streptomycin (Invitrogen, Carlsbad, San Diego) and 15% heat inactivated fetal bovine serum. The cell culture was maintained at 37°C in a humidified atmosphere of 5% CO2. Serum starved U266 cells were activated at 37°C with IFNa2A for 5 minutes when measuring Tyk2. All stimulated cells were fixed, permeabilized and stained with the appropriate phospho-antibody as suggested by the manufacturer (Cell Signaling Technology, Beverly, MA). Sample acquisition was performed on the FACS Array with acquired data analyzed on the Flowjo software (Tree Star Inc., Ashland, OR). EC50 values were derived from dose response curves plotted with mean fluorescent intensity against protein concentration utilizing SigmaPlot. Competition assays with the extracellular domain of IFNAR2 are useful for determining specificity. See Figure 8 for Tyk2.<br>
[684]	Table 4 and 5 summarize data obtained with hlFN polypeptides that are IFNa2a<br>
variants. pSTATl data, anti-proliferation data, Kd values obtained from affinity studies with the extracellular domain of IFNAR2 are shown. hlFN polypeptides comprising the non-naturally encoded amino acid p-acetyl-phenylalanine (pAF) were tested, as well as PEGylated hlFN polypeptides comprising pAF. Controls utilized in these studies included WHO IFNa2A, IFNαdA obtained from Sigma, and PEGASYS®.<br><br><br><br><br><br><br><br><br><br>
Colony Formation Assays<br>
[685]	Colony formation  assays  such as those described by  Giron-Michel,  J.  in<br>
Leukemia 2002 16:1135-1142 may be used to evaluate proliferation of progenitor cells by hlFN polypeptides of the invention. Cord blood may be used in colony formation assays.<br>
Evaluation of the  Toxicity of hlFN Polypeptides on Human Myeloid and Erythroid Progenitors<br>
[686]	Using a methylcelluose-based in vitro colony forming assay, the hematopoietic<br>
toxicity often compoxmds {nine hlFN polypeptides and PEGASYS®) was tested,<br>
[687]	Cells:  Normal human bone marrow light density cells (Poietics Inc., Maryland)<br>
were stored at -152°C until required for the assay. On the day of the experiment, the cells were thawed rapidly at 37°C, the contents of the vial were diluted in 10 mis of Iscove's medium containing 2% fetal bovine serum and washed by centrifugation. The supernatant was discarded, and the cell pellet resuspended in a known volume of Iscove's medium containing 2% FBS. A cell count (3% glacial acetic acid) and viability assessment by Trypan Blue exclusion was performed.<br>
[688]	Test samples: Compounds were 40X stocks in a buffer of 20 mM NaAc, 50 mM<br>
NaCl, 5% glycerol, pH 6.0. Dilutions of each compound were prepared with the same buffer to generate stock concentrations of 144, 72, 36, 18, and 9 ug/ml. When added to methylcellulose, the buffer was at a final concentration of 2,5% and compound concentrations ranged from 3.6-1225 ug/ml. A control containing 50 EU/ml endotoxin in a buffer of 20 mM NaAc, 50 mM NaCl, 5% glycerol, pH 6.0 was diluted with the same buffer 1:40 into methylcellulose to examine the effects of similar levels of endotoxin,<br>
689]	Method:     Clonogenic  progenitors  of the  erythroid  (CFU-E  and  BFU-E),<br>
jranulocyte-monocyte (CFU-GM) and multipotential (CFU-GEMM) lineages were assessed in<br><br>
methylcellulose-based medium (MethoCult 4434) containing saturating concentrations of Ihe<br>
cytolcines SCF (Stem Cell Factor; 50 ng/mL), GM-CSF (Granulocyte Macrophage Colony<br>
Stimulating Factor; 10 ng/mL), IL-3 (Interleukin 3; 10 ng/mL), and EPO (Erythropoietin; 3<br>
U/mL). CFU-E is a small erythroid colony derived from the most mature erythroid colony<br>
forming cells. It contains one to two clusters with a total number of 8-200 erythroblasts. BFU-E<br>
is a larger erythroid colony derived from a more primitive cell It contains greater than 200<br>
erythroblasts. CFU-GM is a colony that is derived from a colony forming cell capable of<br>
producing colonies with forty or more granulocyt-monocyte and/or macrophage cells. CFU-<br>
GEMM is a colony that contains cells from more than one lineage. It is derived from the most<br>
primitive colony forming cell and contains erythroid cells as well as twenty or more<br>
granulocytes, macrophages, and megakaryocytes. As controls, cultures were also set-up<br>
containing vehicle (20 mM NaAc, 50 mM NaCl, 5% glycerol, pH 6.0), vehicle containing<br>
endotoxin, and various concentrations of 5-Fluorouracil (a known myelosuppressive compound).<br>
Standard cultures containing no buffer or compound were also established.<br>
[690]	Clonogenic progenitors of the erythroid (CFU-E and BFU-E), granulocyte-<br>
monocyte/myeloid (CFU-GM) and multipotential (CFU-GEMM) lineages were set up in the<br>
methylceliulose-based media described. The compounds were added to the MethoCult to give<br>
final concentrations of 3.6, 1.8, 0.9, 0.45, and 0.225 ug/ml. Vehicle control cultures containing<br>
no compound but equivalent concentrations of vehicle buffer, endotoxin control cultures<br>
containing no compound but equivalent concentrations of endotoxin, as well as standard controls<br>
containing no compounds or vehicle buffer were also initiated. In addition, cultures were<br>
initiated with 5-Fluorouracil (5-FU) at 5, 1, 0.5, and 0.1 ug/ml to serve as positive controls for<br>
toxicity. All cultures were set up in triplicate at 1X104 cells per culture. Following 14 days in<br>
culture, the colonies were assessed and scored. The colonies were divided into the following<br>
categories, based on size and morphology: CFU-E, BFU-E, CFU-GM, and CFU-GEMM.<br>
[691]	Results and analysis:   Triplicate cultures for CFU-E, BFU-E, CFU-GM, and<br>
CFU-GEMM were enumerated. In addition, the distribution of colony types as well as general colony and cellular morphology were analyzed. The variance in colony number detected in replicate cultures is representative of the coefficient of variation for colony enumeration. The results are shown in Table 6. For statistical analysis, all compounds were compared to the vehicle control. Standard t-tests were performed to assess if there was a difference in the number<br><br>
of colonies generated between control and treated cultures. Due to the potential subjectivity of<br><br><br><br>
[692]	The number and distribution of colonies detected in the vehicle and endotoxin<br>
controls were no different from the standard control (containing no compound and no vehicle buffer). A dose-dependent toxic effect (inhibitory effect on erythroid and myeloid progenitor proliferation) was seen in cultures incubated with 5-fluorouracil. Complete inhibition of growth was seen at the highest concentration of 5 ug/ml. A significant decrease from control numbers was also seen at 1 ug/ml for erythoid and myeloid growth and at 0.5 ug/ml for myeloid growth only, indicating that 5-FU is more toxic to the myeloid cell lineage. Colony numbers recovered to near control levels at 0.1 ug/ml.<br>
[693]	In the presence of each hlFN polypeptide or PEGASYS®, the morphology of the<br>
resulting erythroid and myeloid colonies was not perturbed.  However, where colony numbers<br><br>
were affected so was the size of both colony types. All hlFN polypeptides and PEGASYS® were found to be toxic to the total number of colony forming cells (myeloid and erythroid progenitors) at all concentrations tested, showing a slight dose dependent effect. hlFN polypeptides and PEGASYS® did cause a slight gradual dose dependent effect on progenitor compounds. PEGASYS® was found not to be significantly toxic at 0.225 ug/ml, and 6HisM16pAF-30K PEG and 6HisN65pAF-30K PEG were found not to be significantly toxic to erythoid progenitors at 0.45 (6HisM16pAF-30K PEG only) and 0.225 ug/ml. However, for all compounds the total number of CFC was significantly inhibited at all concentrations tested. Colony size was also found to be smaller in the presence of all compounds and size reflected the toxicity seen. For example, colonies in the presence of 3.6 ug/ml PEGASYS® were smaller than those observed at 0.225 ug/ml.<br>
[694]	Further analysis of the CFU-GM colony count (Y-axis) is shown as Figure 10,<br>
with International Units of anti-viral activity on the X-axis. The specific activity of PEGASYS® in anti-viral assays is 1 ng/unit. Mass concentrations of the PEGylated hlFN polypeptides were converted to International Units using the anti-viral IC50 values measured in the VSV replication assay. The CFU-GM colony count for the control standard is plotted on the far right of the graph. CFU-GM colony counts for the five concentrations of PEGASYS® tested in the assay are plotted. There is an increase in colony count as PEGASYS® concentration is decreased from approximately 3600 units/ml to 225 units/ml. The colony counts of the PEGylated hlFN polypeptides are also shown.<br>
[695]	All hlFN polypeptides showed a concentration dependent effect on the CFU-GM<br>
count. On a per unit basis, hlFN polypeptides 6HisI24pAF-30K PEG and 6HisN65pAF-30K<br>
PEG appeared to be more toxic than PEGASYS® in this assay. 6HisN65pAF-30K PEG was at<br>
least 5-fold less potent than PEGASYS® in anti-viral assays, but was found to bind tightly to the<br>
IFNR2 receptor. In contrast, 6HisE78pAF-30K PEG appeared to be approximately two fold less<br>
toxic having equal colony counts as PEGASYS® at 2 fold higher unit concentrations. For<br>
example, the 6HisE78pAF-30K PEG colony count of 22 at a concentration of 1600 units/mi is<br>
compared to the PEGASYS® colony count of 21 at 900 units/ml. Further studies of toxicity<br>
include but are not limited to CFU-GM toxicity studies with 12 point concentration curves run<br>
from 15000 to 1 unit/ml for each hlFN polypeptide to determine the IC50 for CFU-GM toxicity.<br>
1696]	Table  7 shows hematopoietic toxicity data obtained with PEGASYS® and<br>
Ribavirin.    PEGASYS® was in a buffer of 20 mM Na Acetate, 137 mM NaCl, 0.005%<br><br>
polysorbate 80, pH6.0 and had a starting concentration of 180 ug/ml. Ribavirin (Calbiochem cat # 555580) in PBS was at a starting concentration of 7.63 mg/ml. Dilutions of each were tested, as shown in Table 7.<br>
 .<br>
Anti-viral assays<br>
[697]	Antiviral activity of hlFN polypeptides may be measured by a variety of assays.<br>
One such assay involves studying the reduction of cytopathic effect (CPE) of cells infected with Vesicular Stomatitis Virus (VSV). VSV (ATCC) is produced by infecting the baby hamster kidney cell line BHK21 (ATCC). Briefly, cells are infected with various amounts of virus, and 24 hours post infection the supernatant containing virus is collected. Cell debris is eliminated by centifugation of the supernatant for 5 minutes at 1,200 rpm. The stock of virus is stored in 1 ml aliquots at -80° C. The titer of virus is calculated by using plaque assays. Titration is expressed<br><br>
by PFU/ml (plaque-fonning unit). Additional assays for testing antiviral activity of hlFN polypeptides of the invention include, but are not limited to, an assay based on the HCV replicon (Dr. Brent Korba at Department of Microbiology and Immunology, Georgetown University Medical Center). A variety of hlFN polypeptides are assayed in the stably HCV RNA-replicating cell line, AVA5; this cell line is derivative of the human hepatoblastoma Huh7. Intracellular HCV RNA levels are measured 24 hours after the last dose of hlFN polypeptide has been added. The RNA levels are measured using standard blot hybridization techniques. Two strains of HCV replicon are tested, HCV strain la and lb. Another HCV assay is based upon measurment of HCV replication in specific human cells that support HCV replication (Dr. Brent E. Korba or Dr. Thomas I. Michalak, M.D., Ph.D. (Faculty of Medicine Health Science Centre Memorial University St. John's, Canada)). Another assay to assess the biological activity of hlFN polypeptides utilizes woodchuck hepatocytes. Human IFN-α up-regulates class I MHC antigen expression and 2'-5-OAS mRNA m this type of cells (Dr. Thomas I. Michalak, M.D., Ph.D). The effects of hlFN polypeptides of the invention on Class I MHC expression and/or 2'-5'-0AS levels are measured in the hepatocytes.<br>
[698]	To evaluate hlFN polypeptide activity, 3x104 human WISH cells (ATCC) were<br>
seeded in a 96 well/plate and were subsequently infected with 10,000 PFU of VSV per well. At<br>
the time of infection, different amounts of hlFN polypeptide were added. 48 hours post-infection<br>
the CPE was evaluated; 42-48 hours was the minimum time required to obtain 100% CPE. CPE<br>
was identified by staining the cells with 0.1 ml of 1 mg/ml 3-(4,5-dimethylthiazol-2-yl) 2,5-<br>
diphenyltetrazolium bromide (MTT) followed by spectrophotometric reading at 570 nm with<br>
690 nm as the reference wavelength. MTT measures metabolic reduction in mitochondria.<br>
[699]	Table 8 summarizes the IC50 values obtained with PEGASYS® and thirty-three<br>
hlFN polypeptides that have a substitution at the site indicated and were PBGylated. UnPEGyiated hlFN polypeptides with a para-acetylphenylalanine substitution were also tested with this assay.   Figure 9 shows dose response curves obtained with nine PEGylated hlFN<br><br><br><br>
[700]	Site specific PEGylation was found to alter the activity of IFN and may separate<br>
various biochemical properties of the protein. This effect of site specific PEGylation was found as a result of an analysis of the anti-viral IC50 data with the KD of each hlFN polypeptide for the high affinity IFN receptor IFNR2, as shown in Figure 11. The size of the bubbles represents the measured KD of each hlFN polypeptide for IFNR2. Two hlFN polypeptides, 6HisI24pAF-30K PEG and 6HisE107pAF-30K PEG, were approximately five fold more potent in the cell based anti-viral assay than PEGASYS®. Also, there was a lack of correlation between IFNR2 receptor KD and anti-viral activity. Thus, there is evidence that site specific PEGylation can differentially alter the biochemical activity of IFN. Thus, hlFN polypeptides have been produced that have improved anti-viral potency that also differ in various other measured biochemical properties.<br>
Example 26 Other assays<br>
Measurement of HLA-A.B.C Expression<br>
[701]	An assay measuring the increase in cell surface MHC class I expression in THP-1<br>
cells may also be used to test activity of hlFN pol}'peptides. The human monocyte line THP-1 was cultured as indicated above. THP-1 cells, at a cell concentration of 5 X 10^ cell/ml, were first differentiated in the presence of 5 nM phorbol 12-myristate 13-acetate (Sigma, St Louis, MO). Differentiated THP-1 cells were then stimulated with increasing concentrations of IFNa2A for 24 hours at 37°C. IFNa2A activated cells were then harvested; washed and stained for surface HLA expression. Immunofluorescent staining with APC-conjugated anti-HLA-A,B,C antibody in the presence of propidium iodide (BD Biosciences, San Diego, CA) enabled sample analysis on the FAGS Array with acquired data analysis on the Flowjo software. EC50 values were estimated from dose response curves plotted with mean fluorescent intensity against protein concentration utilizing SigmaPlot. See Figure 7.<br>
PGE2 and cAMP assays<br><br>
[702]	Examples of in vitro assays that are used to evaluate hlFN polypeptides include,<br>
but are not limited to, assays that detect an increase in prostaglandin E2 (PGE2) or decrease in cAMP as described for example, in Wang et al, J. of Neuroimmunology 2004 156:107-112 and Jiang et al, Neurochemistry 2000 36:193-196, which are incorporated herein by reference, Additional citations describing PGE2 secretion assays include but are not limited to, Mazzucco et al. J. Leulcoc Biol 1996; 60(2):271-7; Hoozemans et al. Exp. Gerontol. 2001 Mar; 36(3):559-70; Fiebich et al. J Neurochem. 2000 Nov; 75(5):2020-8; Schwende et al. J Leukoc Biol 1996; 59(4):555-61; and Hoozemans et al. Acta Neuropathol (Berl.) 2001 Jan;101(l):2-8, which are incorporated herem by reference. Modifications to these methods are known to one of ordinary skill in the art,<br>
[703]	Cell based assays are performed, for example, comparing hlFN polypeptides of<br>
the invention with controls that modulate PGE2 or cAMP levels or current IFN therapeutics. Such assays measuring PGE2 secretion involve cell lines such as human THP-1 leukemia cells and human neuroblastoma cells (SK-N-SH). Differentiated THP-1 cells challenged with lipopolysaccharide (LPS) showed an increase in secretion of PGE2. In SK-N-SH cells, IL-lp increased PGE2 secretion. Such assays may be used to investigate the correlation of fever induction and/or the analgesic effect found with current IFN therapies with modulated secretion of prostaglandin E2 (PGE2) or cAMP levels.<br>
[704]	cAMP levels are detected as follows.   5 X 104 SK-N-SH neuroblastoma cells<br>
(ATCC) are allowed to adhere overnight onto a 96-well assay plate (Applied Biosystems, Foster City, CA). Cells are then stimulated at 37°C for 15 minutes with (a) 10 uM forskolin alone; (b) 10 uM forskolin plus hlFN polypeptides (60 pM for non-PEG version and 600 pM for PEG form) and (c) 10 uM forskolin plus hlFN polypeptides plus 20 uM naloxone (an opioid receptor antagonist). Cells are then lysed, and the lysates are incubated with a cAMP-alkaline phosphatase conjugate and an anti-cAMP antibody for 1 hour at room temperature. A chemiluminescent substrate is then added and the level of light emission measured on a Topcount luminescent reader. The amount of cAMP present in the lysates is quantitated from a cAMP standard curve.<br>
Gene Expression Profiling<br>
[705]	Transcriptional activity of monoPEGylated interferon-α-2a isomers is described<br>
in Foser et al. Pharmacogenomics Journal 2003; 3:312-319 using oligonucleotide array transcript analysis. Cell lines that may be used in expression studies include but are not limited<br><br>
to melanoma cell lines such as ME15. Alternate assays to DNA arrays may be performed with hlFN polypeptides of the invention to provide mRNA profiling or differential gene expression information.<br>
Additional Assess<br>
[706]	Assays to support preliminary formulation studies include assays for protein de-<br>
amidation, aggregation, oxidation, acetylation, and other stability indicating assays.    Other assays include, but are not limited to, assays that investigate other potential degradation products, including but not limited to, disulfide rearrangements and proteolytic degradation products. Example 27<br>
[707]	This   example  describes  an  alternate  system  for  cloning,  expression   and<br>
purification of hlFN polypeptides of the invention. Possible advantages of this alternate system include, but are not limited to, the deletion of one or more steps that involve refolding of hlFN polypeptides. NusA fusion proteins and TEV are discussed in Davis et al. Biotechnol Bioeng (1999); 65:382-388; Shih et al Protein Science 2005; 14:936-941; and U.S. Patent Nos. 5,162,601, 5,532,142; 5,766,885; and 5,989,868, all of which are incorporated by reference herein.<br>
Cloning and constructs for Nus-hlFN polypeptide fusions<br>
[708]	To  generate NusA-hlFN  fusions,  hlFN  nucleotide  sequences were  cloned<br>
downstream of NusA between Spel and Kpnl restriction sites, separated by the sequence GSGENLYFQ (which includes a GSG linker and the recognition sequence for TEV). All initial clomng steps were completed using Novagen's pET44 vector, and then the whole Nus-IFN fusion cassette was transfered into another vector that allows for expression of hlFN controlled by the T71ac promoter and induction of protein expression with 1 mM IPTG. The transformation of E. coli (Origami 2) with constructs containing the modified hlFN polynucleotide sequence and the orthogonal aminoacyl tRNA synthetase/tRNA pair (specific for the desired non-naturally encoded amino acid) allows the site-specific incorporation of non-naturally encoded amino acid into the hlFN polypeptide.<br>
Purification and cleavage of Nus-hlFN fusions<br>
[709]	A 5 ml LB culture was grown ft-om a single colony of E. coli transformed with a<br>
construct encoding the NusA-hlFN fusion. 0.5 liters of LB with ampicillin was inoculated, and the non-naturally encoded amino acid p-acetyl-phenylalanine was added to the culture.   The<br><br>
culture was grown overnight at 28°C-32°C. When the OD of the culture was greater than 0,8-1, the culture was transfered to 30°C, and induction was performed with ImM EPTG. The culture continued to grow for 4 hours at 30°C, and a sample was collected for SDS-PAGE analysis. Tlie cells were collected by centrifugation and frozen at -80°C.<br>
[710]	The cells were lysed in 25-30 ml of 50 mM Tris, pH 8, 200 mM NaCl, 5%<br>
glycerol. The samples were sonicated in a glass beaker for 25 sec 5-6 times, while keeping the samples on ice. The cells were incubated with 10 uL DNAse for 30 minutes at room temperature. The samples were then spun down and the supernatant removed, 5 mM imidazole was added to the supernatant.<br>
[711]	The supernatant was then loaded onto a 5 ml HisTrap column at 3 ml/min. Prior<br>
to loading the supernatant, the HisTrap column is equilibrated with 20 mM Tris pH 8, 300 mM NaCI, 0.005% Tween80, 10 mM imidazole (Buffer A). After washing the column with Buffer A, elution was performed with the following step gradient: 4 column volumes of 8% Buffer B (20 mM Tris pH 8, 300 mM NaCl, 0.005% Tween80, 0.5M imidazole); 5 column volumes of 30% Buffer B; and 100% Buffer B wash. The NusA-hlFN fusion eluted during the 30% Buffer B step. The column was washed with IM NaOH/ IM NaCl and was stripped and recharged (or replaced) after five runs.<br>
[712]	Samples from the step gradient were analyzed by SDS-PAGE, and the fractions<br>
with the fusion (ca 75 kDa) were pooled and dialyzed against 20 mM Tris, pH 8, 20 mM NaCl,<br>
5% glycerol overnight at 4°C. The sample was diluted to 30 uM protein with dialysis buffer.<br>
The following was added to the sample: cysteamine to 0.2 mM, EDTA to ImM, and TEV to 0.8<br>
uM. The samples were then incubated for 24-48 hours at 4°C, or overnight at room temperature.<br>
[713]	After the incubation, the samples were diluted 1:3 with ddH2O and loaded onto a<br>
5 ml Q HP column. Elution was performed with a 0-40% gradient of 10 mM Tris, pH 8 (Buffer<br>
A') into Buffer B' (10 mM Tris, pH 8, IM NaCl) over eighteen column volumes.   hlFN<br>
polypeptides eluted at approximately 100 mM NaCl; TEV at 250 mM NaCl, and NusA at 400<br>
mM NaCl. The column was washed with 2.5 mM NaCl, followed by IM NaCl/lM NaOH.<br>
[714]	Eluted fractions were analyzed by SDS-PAGE. hlFN polypeptide fractions were<br>
pooled, and the pH of the pool was adjusted by adding 1/20 volume of IM NaAc, pH 4.5, and the hlFN pool was then dialyzed against 20 mM NaAc, pH 4, 20 mM NaCl, 5% glycerol overnight. The sample was then concentrated to greater than 1 mg/ml, keeping the volume greater than 300 mcl. The extinction coefficient at 280 nm is 22,500 M-1 or 1.17 mg/ml ~ <br>
PEGylation<br>
[715]	Oxyamino-derivatized 3 OK PEG was added to hlFN polypeptide at a 1:12 molar<br>
ratio (30mg / mg IFN/ml), and the mixture was incubated at 28°C for 16-48 hours. Figure 19 shows the 30K PEG used in the conjugation. The sample was diluted ten times with SP buffer A (50 mM Na acetate, pH 5). To purify the PEGylated hlFN polypeptides, a 5 ml SP HP AKTA column was used with a gradient of 0-50% SP buffer B (50 mM Na acetate, pH 5, 0.5 M NaCL 10% ethylene glycol) over ten column volumes. PEGylated IFN eluted around 100 mM salt, followed by non-PEGylated monomer (baseline separation). The fractions of PEGylated liIFN were pooled, dialyzed against the following storage buffer: 20mM NaAcetate, pH 6, 0.005% Tween, 125 mM NaCl. The samples were then concentrated to &gt; 8 microM and stored at 4°C.<br>
Purification of TEV<br>
[716]	Purified TEV was generated with the following protocol. 0.5-2 liter cultures of<br>
BL21(DE3)RIL cells (Stratagene) were transformed with pRK793-TEV(S219V) plasmid (from<br>
NCI) at 30°C. Induction was performed when the OD600 was 0.8-1.2, and growth was<br>
continued for 4-5 hours at 30°C. The cells were resuspended in 20-30 ml of NTA buffer (50<br>
mM Tris, pH 7, 300 mM NaCl). The samples were then sonicated five times for thirty seconds<br>
on ice, incubated at room temperature for thirty minutes, and centrifuged for twenty minutes at<br>
12,000 rpm. The supernatant was removed, and imidazole was added to 15 mM.<br>
[717]	The supernatant was loaded onto a 5 mL HisTrap column equilibrated in NTA<br>
buffer supplemented with 15 mM imidazole. Elution was performed with a gradient of 0-100% B (NTA buffer with 0.5M imidazole) over eighteen colimm volumes. TEV eluted as a broad peak starting 200 mM imidazole. The eluted fractions were analyzed by SDS-PAGE, and the fractions containing clean TEV were pooled. The pooled fractions were dialyzed overnight against the following buffer (30 mM Tris, pH 7.2, 200 mM NaCl, 0.5 mM EDTA, 1 mM DTT). The dialyzed sample was centrifuged to remove any precipitate.<br>
[718]	The spun material was then dialyzed four-fold with ddH2O and loaded onto 5 ml<br>
SP FF AKTA column equilibrated with Buffer A (10 mM Tris pH 7, 50 mM NaCl). The<br>
column was eluted with a linear gradient 0-100%) B (lOmM Tris, pH 7, 0.5M NaCl) over<br>
eighteen column volumes. The eluted TEV fractions were pooled, and the following was added<br>
to the pool: 1 mM DTT, 1 mM EDTA, 20%) glycerol. The sample was concentrated to &gt; 20uM<br>
(ext, coeff. 32,500, pi is 9.6), aliquoted at 500 mcl/tube, and stored at -20°C.<br>
[719]	Anti-viral aclivity<br><br>
[720]	To  evaluate the activity of hlFN polypeptides generated via the methods<br>
described in this Example, 3x104 human WISH cells (ATCC) were seeded in a 96 well/plate and were subsequently infected with 10,000 PFU of VSV per well. At the time of infection, different amounts of hlFN polypeptide were added. 48 hours post-infection the CPE was evaluated; 42-48 hours was the minimum time required to obtain 100% CPE. CPE was identified by staining the cells with 0.1 ml of 1 mg/ml 3-(4j5-dimethylthiazol"2-yl) 2,5-diphenyltetrazolium bromide (MTT) followed by spectrophotometric reading at 570 nm with 690 nm as the reference wavelength. MTT measures metabolic reduction in mitochondria. The antiviral activity of five PEGylated hlFN polypeptides and PEGASYS® is shown in Figure 18. Methionyl hlFN polypeptides comprising a non-naturally encoded amino acid are designated with an "M-" hlFN polypeptides without a methionine as the first amino acid are also generated using methods involving NusA.<br>
Example 28<br>
[721]	This example describes methods to measure in vitro and in vivo activity of hlFN<br>
polypeptides of the invention.<br>
Cell Binding Assays.<br>
[722]	Cells (3x106) are incubated in duplicate in PBS/1% BSA (100µl) in the absence<br>
or presence of various concentrations (volume: 10 µl) of unlabeled IFN, hlFN or GM-CSF and<br>
in the presence of 125  1-IFN (approx. 100,000 cpm or 1 ng) at 0°C for 90 minutes (total volume:<br>
120 nl). Cells are then resuspended and layered over 200 [d ice cold FCS in a 350µl plastic<br>
centrifuge tube and centrifuged (1000 g; 1 minute). The pellet is collected by cutting off the end<br>
of the tube and pellet and supernatant counted separately in a gamma counter (Packard),<br>
[723]	Specific binding (cpm) is determined as total binding in the absence of a<br>
competitor (mean of duplicates) minus binding (cpm) in the presence of 100-fold excess of unlabeled IFN (non-specific binding). The non-specific binding is measured for each of the cell types used. Experiments are run on separate days using the same preparation of 125I-IFN and should display internal consistency. 125I-IFN demonstrates binding to the Daudi cells. The binding is inhibited in a dose dependent manner by unlabeled natural IFN or hlFN, but not by GM-CSF or other negative control. The ability of hlFN to compete for the binding of natural '^^ I-IFN, similar to natural IFN, suggests that the receptors recognize both forms equally well.<br>
In Vivo Studies fi-om PEGylated IFN<br>
[724]	PEG-hlFN, unmodified hlFN and buffer solution are administered to mice or rats.<br>
The results will show superior activity and prolonged half life of the PEGylated hlFN of the<br><br>
present invention compared to unmodified hlFN which is indicated by significantly increased iiihibition of viral replication using the same dose per mouse or rat.<br>
Measurement of the in vivo Half-life of Conjugated and Non-coniugated hlFN and Variants Thereof.<br>
[725]	Male Sprague Dawley rats (about 7 weeks old) are used.    On the day of<br>
administration, the weight of each animal is measured, 100 ng per kg body weight of the non-conjugated and conjugated hlFN samples are each injected intravenously into the tail vein of three rats. At I minute, 30 minutes, 1,2,4, 6, and 24 hours after the injection, 500 µl of blood is withdravm from each rat while under CO2 -anesthesia. The serum is isolated from the blood samples by centrifugation (4° C, 18000xg for 5 minutes). The serum samples are stored at -80° C until the day of analysis. The amount of active IFN in the serum samples is quantified by the IFN in vitro activity assay after thaving the samples on ice.<br>
Antiviral activity<br>
[726]	There are many assays known to those skilled in the art that measure the degree<br>
of resistance of cells to viruses (McNeill TA, J Immunol Methods. (1981) 46(2):121-7). These assays generally can be categorized into three types: inhibition of cytopathic effect; virus plaque formation; and reduction of virus yield. Viral cytopathic effect assays measure the degree of protection induced in cell cultures pretreated with IFN and subsequently infected with viruses. Vesicular stomatitis virus, for instance, is an appropriate virus for use in such an assay. This type of assay is convenient for screening numerous different IFNs, as it can be performed in 96-well plates. An example of a cytopathic effect reduction assay is detailed in Wang et al. NeuroReport (2001) 12(4);857-859. Plaque-reduction assays measure the resistance of IFN-treated cell cultures to a plaque-forming virus (for instance, measles virus). One benefit to this assay is that it allows precise measurement of a 50% reduction in plaque formation. Finally, virus yield assays measure the amount of virus released from cells during, for instance, a single growth cycle. Such assays are useful for testing the antiviral activity of IFNs against viruses that do not cause cytopathic effects, or that do not build plaques in target-cell cultures. The multiplicity of infection (moi) is an important factor to consider when using either plaque-reduction or virus-yield assays.<br>
[727]	Other clinically important interferon characteristics are also easily assayed in the<br>
laboratory setting. One such characteristic is the ability of an interferon polypeptide to bind to specific cell-surface receptors. For instance, some IFNα-2as exhibit different cell-surface properties compared to IFNα-2b, the IFN most widely used in clinical trials. While IFNα-2b is an effective antiviral agent, it causes significant adverse side effects. Interferons that exhibit<br><br>
distinct binding properties from IFNα-2b may not cause the same adverse effects. Therefore,<br>
interferons that compete poorly with IFNα-2b for binding sites on cells are of clinical interest.<br>
Competitive interferon binding assays are well known in the art (Hu et al., J Biol Chem. (1993)<br>
Jun 15;268(17):12591-5; Di Marco et al., (1994) Biochem. Biophys. Res. Comm. 202:1445-<br>
1451, which are incorporated by reference herein). In general, such assays involve incubation of<br>
cell culture cells with a mixture of 125 I-Iabeled IFNα-2b and an unlabeled interferon of interest.<br>
Unbound interferon is then removed, and the amount of bound label (and by extension, bound125  I-labeled IFNα-2b) is measured. By comparing the amount of label that binds to cells in the<br>
presence or absence of competing interferons, relative binding affinities can be calculated,<br>
[728]	Another prominent effect of IFNα's is their ability to inhibit cell growth, which is<br>
of major importance in determining anti-tumor action. Growth inhibition assays are well<br>
established, and usually depend on cell counts or uptake of tritiated thymidine ([3H] thymidine)<br>
or another radiolabel. The human lymphoblastoid Daudi cell line has proven to be extremely<br>
sensitive to IFNα's, and it has been used to measure antiproliferative activity in many IFNα's<br>
and derived hybrid polypeptides (Meister et al., J Gen Virol. (1986) Aug; 67 (Pt 8):1633-43).<br>
Use of this cell line has been facilitated by its ability to be grown in suspension cultures<br>
(Evinger and Pestka, (1981) Methods Enzymol. 79:362-368). IFNα's also exhibit many<br>
immunomodulatory activities (Zoon et al., (1986) In, The Biology of the Interferon System.<br>
Cantell and Schellenkens, Eds., Martinus Nyhoff Publishers, Amsterdam).<br>
[729]	Although IFNs were first discovered by virologists, their first clinical use (in<br>
1979) was as therapeutic agents for myeloma (Joshua et al., (1997) Blood Rev. 11(4): 191-200). IFNα's have since been shown to be efficacious against a myriad of diseases of viral, malignant, angiogenic, allergic, inflammatory, and fibrotic origin (Tilg, (1997) Gastroenterology. 112(3):1017-1021). It has also proven efBcacious in the treatment of metastatic renal carcinoma and chronic myeloid leukemia (Williams and Linch, (1997) Br. J. Hosp. Med. 57(9):436-439). Clinical uses of IFNs are reviewed in Gresser (1997) J. Leukoc. Biol. 61(5):567-574 and Pfeffer (1997) Semin. Oncol. 24(3 Suppl. 9):S9-S63S969.<br>
Analgesia Assays<br>
[730]	Pain threshold studies may be performed with hlFN polypeptides to investigate<br>
the analgesic effect of these compounds.  One such analgesia assay is described in Wang et al. Neuro Report (2001) 12(4):857-859, which is incorporated by reference herein. Example 29<br>
[731]	This example describes methods to measure in vivo activity and toxicity of hlFN<br>
polypeptides of the invention.<br><br>
[732]	Animal models used to study antiviral activity of hlFN polypeptides include, but<br>
are not limited to, the chimpanzee HCV (Purcell RH, FEMS Microbiol Rev. 1994 Jul;14(3):181-91), the HCV-Trimera mouse (Ilan E et aL J Infect Dis. 2002 Jan 15;185(2):153-61), and the Alb-uPA transgenic mouse models (Mercer DP et al. Nat Med. 2001 Aug;7(8):927-33; Kneteman, N et al (2003) 10th HCV Meeting Kyoto Japan, P-187), all of which are incorporated by reference herein. Woodchuck hepatitis virus (WHV) infection in woodchucks is a useful model to study the pathogenesis, prevention, and treatment of HBV infection (Berraondo, P. et al, J. Med. Virology 2002; 68, 424-432). The condition of woodchuclcs exposed prenatally to WHV is similar to HBV infection. Compounds, either PEGylated human interferon polypeptides, PEGylated woodchuck interferon compound analogs, or PEGylated human interferon polypeptides modified in some way so as to increase their species cross-reactivity towards woodchuck sufficiently for antiviral efficacy evaluation, are evaluated in the woodchuck hepatitis model.<br>
[733]	Cynomolgus monkeys are also used to study in vivo activity and bone marrow<br>
toxicity. Activity is assayed by measuring the induction of downstream markers including but not limited to 2',5'-'OAS by hlFN polypeptides of the invention compared to current IFN therapeutics such as PEGASYS®. Circulating blood cells, including but not limited to neutrophils, RBCs, and platelets are evaluated in bone marrow toxicity studies after administration of hlFN polypeptides compared to current IFN therapeutics such as PEGASYS®. 2',5'-OAS is an enzyme induced by interferon and is directly related to the anti-viral activity of interferon. This enzyme produces a mixture of oligonucleotides (2-5A) which are thought to activate the inactive RNAse present in cells and hinder the protein synthesis of cells or viruses through the breakdown of mRNA. Detection of 2',5'-OAS activity may be performed by a variety of assays known to those of ordinary skill in the art, including but not limited to, radioimmunoassay (RLA.; EIKEN) which measures levels of 2-5A. Cynomolgus serum, for example, may be used to measure 2',5'-GAS activity levels. Neopterin is a non-species specific immune marker that is produced by macrophages upon exposure to interferon. Detection of neopterin may be performed by a variety of assays known to those of ordinary skill in the art, including but not limited to, EIA (IBL). Levels of otiier molecules may be measured in such studies, including but not limited to Cortisol levels. Cortisol, an element of the HPA axis, has been implicated in depression, stress, and weight gain. Detection of Cortisol may be performed by a variety of assays known to those of ordinary skill in the art, including but not limited to, EIA (RnD). Cytokine panels may be performed throughout the study after administration of hlFN polypeptides of the invention and levels of various cytokines compared to basal levels to<br><br>
evaluate toxicity of the hlFN polypeptides. A variety of assays known to those of ordinary skill in the art may be used, including but not limited to, ELISA, bead arrays (BD Biosciences) and other assay formats (Luminex) and may involve different types of readouts, including but not limited to, fluorescent readouts. Measurement of compound levels in serum or plasma may be detected using a variety of assays Icnown to tliose of ordinary skill in the art, including but not limited to, ELISA (PEL). Antibodies used for detection by ELISA or other methods may be anti-IFN alpha antibodies or anti-PEG antibodies for hlFN polypeptides that are PEGylated. Various assay formats known to those of ordinary skill in the art may be used for detection including those involving fluorescence.<br>
[734]	Evaluation of hlFN polypeptides of the invention is performed with the following<br>
pre-ciinical plan. In Study 1, a dose-range finding pharmacokinetics (PK), pharmacodynamics<br>
(PD), and safety study is initiated in female cynomolgus monkeys and involves placebo and four<br>
doses of PEGASYS® (0, 1.5, 15, 50, and 150 ug/kg). The monkeys are acclimated for two<br>
weeks. 15, 50 and 150 ug/kg of PEGASYS® is admmistered on Day 1 and Day 5, 1.5ug/kg of<br>
PEGASYS® is administered on Day 1 only. There are three female monlceys in each of the five<br>
groups. The duration of in-life is 14 days with an option to proceed for at least an additional<br>
week in the event that data so warrant. The endpoints are compound levels in the plasma<br>
measured by integrating the AUG through Day 8 and Day 15; and 2',5'-OAS and neopterin<br>
levels through Day 15. In this study, hematology studies are performed investigating the effect<br>
of the administered hlFN polypeptides on neutrophils and platelets on Day 3, Day 8 and Day 15.<br>
A prebleed is also done on the animals for comparison. Cytokine panels are also performed<br>
through Day 15, measuring levels of various cytokines including but not limited to, IL-2, IL-4,<br>
IL-5, IL-6, TNF, and IFN-y. Cortisol is measured as well through Day 15. Rectal body<br>
temperature of the monkeys is also be measured, since fever is also a known side effect of<br>
current IFN therapeutics,	.   ,<br>
[735]	After one or two doses have been selected fi-om the first study. Study 2 involves<br>
comparing hlFN polypeptides of the invention with PEGASYS® at the one or two doses. If a single dose is suitable for a comparison of PK, efficacy, and toxicity, then the particular dose of PEGASYS® is tested in cynomolgus monkeys and compared to one or more hlFN polypeptides at the same dose, and placebo. Three female monkeys are in each group; the acclimation period is two weeks. As in Study 1, 2',5'-OAS levels, hematology, body temperature, Cortisol, and cytokine panels are studied. If two doses are required i.e. a low dose is required for PK and efficacy and another dose is needed for toxicity comparison, then PEGASYS® is compared to one or more hlFN polypeptides at both doses and placebo.    As in Study 1, 2',5'-OAS levels.<br><br>
hematology, body temperature, Cortisol, and cytokine panels are studied. The end of study in-life is Day 15, with an option to proceed for at least an additional week in the event that data so warrant.<br>
[736]	Bone marrow is collected and evaluated from animals exposed to hlFN<br>
polypeptides of the invention. For example, bone marrow aspirates and/or core biopsies from treated animals (e.g. from primates) are examined histologically, and the ratio of myeloid to nucleated erythroid cells (M:E ratio) is estimated. A low M:B ratio, in conjunction with neutropenia, may be indicative of granulocytic hypoplasia with decreased production of neutrophils. Isolated bone marrow cells from treated animals are assayed for hematopoietic progenitors. That is, cell isolates are subjected to CFU-GM or BFU-E colony assay. The results (i.e. number of hematopoietic progenitor cells) would show the effect, if any, of treatment on myeloid and/or erythroid progenitor cells.<br>
[737]	Such bone marrow toxicity studies may involve hlFN polypeptides administered<br>
via different routes, including but not limited to, subcutaneously and intravenously. Intravenous<br>
administration may allow for longer dosing regimens and better evaluation of bone marrow<br>
toxicity than subcutaneous administration. Such improvements may result from a modulation of<br>
immunogenicity or an antibody response to the compound administered. Toxicity studies<br>
include evaluation of neutrophil and platelet number/mL of plasma or serum.<br>
[738]	Other animal models include but are not limited to, depression models.   One<br>
mouse model involves the Tail-Suspension Test.  Assays measuring the levels of Cortisol and ACTH in mice after administration of hlFN polypeptide are also performed.  Both Cortisol and ACTH are components of the HP A axis and have been implicated in playing a role in interferon-treatment induced depression in patients. Example 30<br>
[739]	This example details the isolation of hlFN polypeptides and the measurement of<br>
hlFN activity and affinity of these hlFN polypeptides for the hlFN receptor. Purification by Copper Column 500 ml LB Shake Flask Expression:<br>
[740]	For each hlFN polypeptide, E. coli transformed with a construct encoding an<br>
orthogonal tRNA (J17 described in U.S. Patent Publication No. 20030108885) and an orthogonal aminoacyl tRNA synthetase (E9 described in U.S. Patent Application No. 11/292,903 entitled "Compositions of Aminoacyl-tRNA Synthetase and Uses Thereof) with a mutated hlFN polynucleotide were streaked onto a plate to obtain colonies. 2 mL LB containing<br><br>
antibiotic was inoculated with a single colony from the freshly transformed plate. The culture was incubated under constant agitation (250 rpm) at 37°C until the culture reached an OD600 of approximately 0.05-0.3 (approximately 5-6 hours). 500 mL LB (containing antibiotic and 4 mM pAF) was inoculated with all or part of the 2 mL culture, normalizing the inoculation to synchronize cultures. The culture was incubated under constant agitation (250 rpm) at 37°C until culture reached an OD600 of approximately LO (approximately 4-5 hours). The culture was then induced with 1 mM IPTG, and the culture was incubated overnight at 37°C. The cells were harvested by centrifugation at 8000 rpm/12000g; 15 minutes; 4°C. The cell paste was scraped out and stored in 40 mL Oak Ridge tubes. The cell pellets were then frozen at -80°C until the inclusion body preparation was initiated. Inclusion Body Preparation from 500mL Shake Flask:<br>
[741]	Buffer 1 was 50 mM NaAc pH 6.0; 100 mM NaCl; 1 mM EDTA; 1.0 % Triton<br>
X-100. Buffer 2 was 50 mM NaAc pH 6.0; 100 mM NaCl; 1 mM EDTA. Table 9 shows the procedure used for inclusion body preparation.<br><br>
[742]	For the Lysis step, cells were resuspended in 23 mL chilled Buffer 1 by shaking<br>
for 1 hour at 4°C. The Avestin C3 was cleaned and rinsed with 100 mL water and then with 30 mL Buffer 1. The cells were then homogenized with 2 passes in Avestin at 15,000-20,000 psi with cooling at 4°C in the following manner: 1) The sample was added. 2) The sample was processed once until Avestin reservoir was nearly empty; the sample was collected in the original tube (also done for subsequent steps). 3) The collected sample was added back to the Avestin reservoir and re-homogenized, 4) The sample was washed out by homogenizing 12 mL Buffer 1; this was added to the sample. 5) The sample was transferred to 40 mL Oak Ridge Centrifuge tube.   6) The sample was stored and 100 mL water was run through the Avestin,<br><br>
adding it slowly while running to flush the system,  7) 30 mL Buffer 1 was then added to the<br>
reservoir and homogenized to prepare for next sample.   8) The steps were then repeated to<br>
process the next sample.<br>
[743]	For the Wash and Rinse steps, a spatula was used to loosen the pellet off the tube<br>
after pouring off supernatant from the last wash and pouring in fresh buffer.   Sonication was<br>
performed at 75% power with a nonnal tip, not a microtip.  The tip was rinsed with water and<br>
wiped with a Kimwipe between samples.<br>
[744]	During the first 4 steps, the samples were spun in 40 mL Oak Ridge tubes used to<br>
store cell pellets. Rinse 2 was spun by splitting each sample to 2 15 mL conical tubes.<br>
Solubilization and Refolding:<br>
[745]	Solubilization was performed with 8M GndHCl pH between 5.5-8.5 using 50<br>
mM NaAc or Tris. The inclusion bodies were solubilized to a final concentration between 5-10<br>
mg/mL. The material was refolded or stored at -80°C for long term storage.<br>
[746]	Refolding was performed by diluting solubilized material to a final total protein<br>
concentration of 0.5 mg/mL in 4°C Refold Buffer (100mM NaAc, pH 6.0 or 50mM Tris, pH 8.0;<br>
0.1% Tween 20). The refolding mixture was stored at 4°C for 24 to 72 hours.<br>
Cu Purification:<br>
[747]	The refolded sample was allowed to warm to room temperature.    A final<br>
concentration of 50µM CuCI was added to the refolded sample, and the sample was incubated at<br>
room temperature for 10-30 minutes.   The sample was filtered with a 0.22µm PES filter and<br>
loaded onto a Chelating Sepharose HP column (GE Healthcare) charged with Cu and<br>
equilibrated with Cu Buffer A (50mM NaAc, pH 5.0; 150 µM NaCl; 0.1% Tween 20). A linear<br>
gradient was run to 100% Cu Buffer B (100 mM NaAc, pH 3.5; 150 mM NaCl; 0.1% Tween 20)<br>
over 18 column volumes, and samples were collected in tubes containing ImM EDTA.<br>
SP HP Purification of Cu Pool:<br>
[748]	The hlFN polypeptide peak was collected from the Cu column and loaded<br>
straight onto a SP HP column (QE Healthcare) equilibrated in SP Buffer A (50mM NaAc, pH<br>
5.0; ImM EDTA). A linear gradient was run to 50% SP Buffer B (50mM NaAc, pH 5.0; 0.5M<br>
NaCl; 10% ethylene glycol; ImM EDTA) over 15 column volumes.<br>
PEGylation and Purification:<br>
[749]	The hlFN polypeptide samples were pooled from the SP HP column and<br>
concentrated to 1.0-2.0 mg/mL. 10% Acetic Acid was added to the sample to drop the pH down<br><br>
to 4.0. Oxyamino-derivatized 30K PEG was added to hlFN polypeptide at a 1:12 molar ratio (30mg / mg IFN/ml), and the mixture was incubated at 28oC for 48-72 hours. Figure 19 shows the 3 OK PEG used in the conjugation. One hEFN polypeptide comprising the non-nalural amino acid para-acetylphenylalanine at position 107 was conjugated to a branched 40K PEG. This PEG is shown in Figure 20 as compound IV (mPEG (40K) aminoethoxyamine hydrochloride), and the figure shows the synthesis scheme from mPEG (40K) p-nit'ophenolcarbonate. The molar ratio and conjugation conditions as well as the purification method used for the 40K PEGylated product was the same as the 30K PEGylated products. The sample was diluted tenfold with SP Buffer A (50 mM Na acetate, pH 5; ImM EDTA), To purify the PEGylated hlFN, a SP HP column (GE Healthcare) was used with a gradient of 0-50% SP buffer B (50 mM Na acetate, pH 5, 0.5 M NaCl, 10% ethylene glycol; ImM EDTA) over 15 column volumes. PEGylated IFN eluted around 100 mM salt, followed by non-PEGylated monomer (baseline separation). The firactions of PEGylated hlFN were pooled, dialyzed against the following storage buffer: 20mM NaAcetate, pH 6, 0.005% Tween, 125 mM NaCl. Samples were concentrated to 1.0-2.0mg/mL and stored at -80°C.<br>
[750]	The method used to functionalize the glycerol-based branched 40K PEG shown<br>
was as follows (Figure 20). To a mixture of mPEG(40 K)/7-nitrophenolcarbonate I (20 g, 0.5 mmol) and A^-aminoethoxyphthalimide hydrochloride 11 (0,6 g, 2.5 mmol) in N,N-dimethylformamide-methylene chloride (1:2, 150 mL) were added diisopropylethylamine (DIEA, 0.27 mL, 1.5 mmol) and 4-dimethylaminopyridine (DMAP, cat.). The resultant mixture was stirred at room temperature for 20 h. Ethyl acetate-hexanes (1:1, 1 L) was added. The precipitate was filtered, washed with ethylacetate-hexanes (1 L) and dried in vacuo to afford product III as a white powder,<br>
[751]	mPEG(40K) N-aminoethoxyphthalimide III fi-om the previous step was dissolved<br>
in a 0.5 M solution of hydrazine in methylene chloride-methanol (1:1, 100 mL). The resultant mixture was stirred at room temperature for 1.0 h and then washed with aqueous HCl solution (0.5 N, 300 mL). The organic layer was dried over anhydrous Na2SO4, filtered and concentrated. The residue was dissolved in CH2Cl2 (100 mL). Ethyl acetate-hexanes (1:1, 1 L) was added to precipitate the mPEG(40K) aminoethoxyamine hydrochloride product IV (17.6 g, 88% from I) as a white powder. Biacore studies (Receptor Bindins Affinity)<br><br>
[752]	The sequence for the IFNAR2 extracellular domain (consisting of 206 amino<br>
acids ending with sequence LLPPGQ) was amplified from clone MHS1011-61064 (OpenBiosystems, Huntsville, AL). This insert was cloned into the pET2D expression vector (Novagen) downstream of the T7 promoter. Protein expression was induced with 0.4 mM IPTG in BL21(DE3) cells (Novagen).<br>
[753]	Smce the expressed protein was insoluble, the inclusion bodies were purified<br>
from lysed cells and solubilized in 6M GndCI. A 5 ml aliquot (50 mg amount) was reduced with 10 mM DTT for 45 minutes at 37°C. Then the mixture was injected into 200 ml of refolding buffer which consisted of 50 mM Tris pH 8, 20 mM NaCl, 0.5 M Arginine, 10% glycerol at 4°C and incubated overnight with gentle stirring.<br>
[754]	The refolding reaction was then concentrated to 25 ml using an Amicon stirring<br>
cell, and dialyzed overnight against 20mM Tris, pH 8, 20 mM NaCl, 10% glycerol. Monomeric refolded IFNαR BCD was purified on HP Q Sepharose using the AKTA FPLC system (Amersham). Purified IFNαR2 BCD was immobilized on CMS Biacore chip using a lysine-specific coupling procedure recommended by the manufacturer. About 200 RUs of functional protein were immobilized. Various concentrations of IFN variants in HBS-EP buffer (Biacore) were injected at a flow rate of 50 mcl/minute over the flowcell containing immobilized IFNαR2, and a control flowcell containing inraiobilized bovine serum albumin, Sensograms generated were fit to the 1:1 interaction model to calculate kon, kofr and K<j values using biaevaluation software the receptor binding affinity of hlfn polypeptides pegylated at non-naturally encoded amino acid paf were observed. additional mutants tested that and had a naturally substitution position methionyl wild-type interferon was included as control sample. table shows kon ko ka obtained. shown do not have tag refolded described in this example. parameters for ifn : interaction determined by spr></j><br><br><br>
[755]	To evaluate hlFN polypeptide anti-viral activity, 3x10' human WISH cells<br>
(ATCC) were seeded in a 96 well/plate and were subsequently infected with 10,000 PFU of VSV per well. At the time of infection, different amounts of hlFN polypeptide were added. 48 hours post-infection the CPE was evaluated; 42-48 hours was the minimum time required to obtain 100% CPE. CPE was identified by staining the cells with 0.1 ml of 1 mg/ml 3-(4,5-dimethylthiazol-2-yl) 2,5-diphenyltetrazolium bromide (MTT) followed by spectrophotometric reading at 570 nm with 690 nm as the reference wavelength. MTT measures metabolic reduction in mitochondria.<br>
[756]	Table 11 and 12 shows the % relative anti-viral activity based on the IC50 values<br>
obtained with with hlFN polypeptides that have a non-natural amino substitution (para-icetylphenylalanine, pAF) at the site indicated and were PEGylated at that site. A few of the iIFN polypeptides had a natural amino acid substitution at position 149 as well. A couple of nutants were made using the NusA system described in Example 27 and contain a methionine at he N-terminus (designated with "M-")- PEGASYS® was used as a control.<br><br><br><br>
Example 31<br>
[757]	This example details the measurement of hlFN activity and affinity of hlFN<br>
polypeptides for the hlFN receptor. The hlFN polypeptides described in this Example were generated using the NusA system in Example 27. Natural amino acid substitutions were made to the IFNα2a polypeptide sequence (SEQ ID NO: 2) based on a sequence comparison between limitin and IFNα2a. The "M-" indie ates that methionine was the first amino acid in the polypeptide. The M-IFN "HV" or "HV" mutant was generated with the following substitutions in hIFNα-2a (SEQ ID NO: 2): D77-D94 is replaced with the mouse limitin sequence HERALDQLLSSLWRELQV. The M-IFN "CD" or "CD" mutant was generated with the following substitutions in hIFNα-2a (SEQ ID NO: 2): V105-D114 with GQSAPLP. Biacore studies {Receptor Bindirts Affinity)<br>
[758]	The sequence for the IFNαR2 extracellular domam (consisting of 206 amino<br>
acids   ending   with   sequence   LLPPGQ)   was   amplified   from   clone   MHS1011-61064<br><br>
(OpenBiosystems, Huntsville, AL). This insert was cloned into the pET20 expression vector (Novagen) downstream of the T7 promoter. Protein expression was induced with 0.4 mM IPTG in BL21(DE3) cells (Novagen).<br>
[759]	Since the expressed protein was insoluble, the inclusion bodies were purified<br>
from lysed cells and solubilized in 6M GndCl. A 5 ml aliquot (50 mg amount) was reduced with 10 mM DTT for 45 minutes at 37°C. Then the nature was injected into 200 ml of refolding buffer which consisted of 50 mM Tris pH 8, 20 mM NaCl, 0.5 M Arginine, 10% glycerol at 4°C and mcubated overnight with gentle stirring.<br>
[760]	The refolding reaction was then concentrated to 25 ml using an Amicon stirring<br>
cell, and dialyzed overnight against 20mM Tris, pH 8, 20 mM NaCl, 10% glycerol. Monomeric<br>
refolded IFNαR BCD was purified on HP Q Sepharose using the AKTA FPLC system<br>
(Amersham). Purified IFNαR2 BCD was immobilized on CM5 Biacore chip using a lysine-<br>
specific coupling procedure recommended by the manufacturer. About 200 RUs of functional<br>
protein were immobilized. Various concentrations of IFN variants in HBS-EP buffer (Biacore)<br>
were injected at a flow rate of 50 mcl/minute over the flowcell containing immobilized IFNαR2,<br>
and a control flowcell containing immobilized bovine serum albumin. Sensograms generated<br>
were fit to the 1:1 interaction model to calculate kon, koff and Kd values using BiaEvaluation<br>
software (Biacore). Methionyl wild-type interferon (M-WT) was included as a control sample.<br>
[761]	Table 13 shows the kon, koa and Kd obtained with hlFN polypeptides comprising<br>
one or more natural amino acid substitution. These mutants were made using the NusA system described in Example 27 and contain a methionine at the N-terminus (designated with an "M-"). The sequences for the mutants were generated based on amino acid differences found between IFNα2a and limitin.<br>
[762]	Table 14 shows the ICon, koff, and Kd obtained with hlFN polypeptides comprising<br>
a non-natural amino acid substitution (para-acetylphenylalanine) that was PEGylated with the linear 30K PEG shown in Figure 19. These mutants were made using the NusA system described above and contain a methionine at the N-terminus (designated with an "M-").<br><br><br><br>
Anti-viral assays<br>
[763]	To evaluate hlFN polypeptide anti-viral activity, 3x10"* hiunan WISH cells<br>
(ATCC) were seeded in a 96 well/plate and were subsequently infected with 10,000 PFU of<br>
VSV per well. At the time of infection, different amounts of hlFN polypeptide were added. 48<br>
hours post-infection the CPE was evaluated; 42-48 hours was the minimum time required to<br>
obtain 100% CPE.   CPE was identified by staining the cells with 0.1 ml of 1 mg/ml 3-(4,5-<br>
dimethylthiazol-2-yl) 2,5-diphenyltetrazolium bromide (MTT) followed by spectrophotometric<br>
reading at 570 nm with 690 nm as the reference wavelength.    MTT measures metabolic<br>
reduction in mitochondria.<br>
[764]	Table 15 shows the % relative anti-viral activity based on the IC50 values<br>
obtained with hlFN polypeptides that comprise one or more natural amino substitutions.<br><br><br>
Example 32<br>
[765]	This example details the measurement of hlFN activity and affinity of hlFN<br>
polypeptides for the hlFN receptor.    The hlFN polypeptides described in this Example were<br>
generated using the following method.<br>
Purification by HIC column<br>
500 ml LB Shake Flask Expression:<br>
[766]	For each hlFN polypeptide, E. coli transformed with a construct encoding an<br>
orthogonal tRNA (J17 described in U.S. Patent Publication No.  20030108885) and an<br>
orthogonal  aminoacyl  tRNA  synthetase  (E9  described  in  U.S.  Patent  Application  No.<br>
11/292,903 entitled "Compositions of Aminoacyl-tRNA Synthetase and Uses Thereof) with a<br>
mutated hlFN polynucleotide were streaked onto a plate to obtain colonies. 2 mL LB containing<br>
antibiotic was inoculated with a single colony from the freshly transformed plate.  The culture<br>
was incubated under constant agitation (250 rpm) at 37°C until the culture reached an OD600 of<br>
approximately 0.05-0,3 (approximately 5-6 hours). 500 mL LB (containing antibiotic and 4 mM<br><br>
pAF) was inoculated with all or part of the 2 mL culture, normalizing the inoculation to synchronize cultures. The culture was incubated under constant agitation (250 rpm) at 37°C until culture reached an OD600 of approximately 1.0 (approximately 4-5 hours). The culture was then induced with 1 mM IPTG, and the culture was incubated overnight at 37°C. The cells were harvested by centrifugation at 8000 rpni/12000g; 15 minutes; 4°C. The cell paste was scraped out and stored in 40 mL Oalc Ridge tubes. The cell pellets were then frozen at -80°C until the inclusion body preparation was initiated. Inclusion Body Preparation from 500mL Shake Flask:<br>
[767]	Buffer 1 was 50 mM NaAc pH 6.0; 100 mM NaCl; 1 mM EDTA; 1.0 % Triton<br>
X-100. Buffer 2 was 50 mM NaAc pH 6.0; 100 mM NaCI; 1 mM EDTA. Table 16 shows the procedure used for inclusion body preparation.<br><br>
[768]	For the Lysis step, cells were resuspended in 23 mL chilled Buffer 1 by shaking<br>
for 1 hour at 4'°C. The Avestin C3 was cleaned and rinsed with 100 mL water and then with 30 mL Buffer 1. The cells were then homogenized with 2 passes in Avestin at 15,000-20,000 psi with cooling at 4°C in the following manner: 1) The sample was added. 2) The sample was processed once until Avestin reservoir was nearly empty; the sample was collected in the original tube (also done for subsequent steps). 3) The collected sample was added back to the Avestin reservoir and re-homogenized. 4) The sample was washed out by homogenizing 12 mL Buffer 1; this was added to the sample. 5) The sample was transferred to 40 mL Oak Ridge Centrifuge tube. 6) The sample was stored and 100 mL water was run through the Avestin, adding it slowly while running to flush the system. 7) 30 mL Buffer 1 was then added to the reservoir and homogenized to prepare for next sample. 8) The steps were then repeated to process the next sample.<br><br>
[769]	For the Wash and Rinse steps,' a spatula was used to loosen the pellet off the tube<br>
after pouring off supernatant from the last wash and pouring in fresh buffer.   Sonication was<br>
performed at 75% power with a normal tip, not a microtip. The tip was rinsed with water and<br>
wiped with a Kimwipe between samples.<br>
[770]	During the first 4 steps, the samples were spun in 40 mL Oak Ridge tubes used to<br>
store cell pellets. Rinse 2 was spun by splitting each sample to 2 15 mL conical tubes.<br>
Solubilization and Refolding:<br>
[771]	Solubilization was done with 8M GndHCl pH between 5.5-8.5 using 50 mM<br>
NaAc or Tris.   The inclusion bodies were solublized to a final concentration between 5-10<br>
mg/mL.   A final concentration of 10mM 6-Mercaptoethanol was added, and the sample was<br>
incubated at room temperature for 30-60 minutes. The material was refolded or stored at -80°C<br>
for long term storage.<br>
[772]	The refolding was performed by diluting the solubilized material to a final total<br>
protein concentration of 0.5mg/mL in 4**C Refold Buffer (50mM Tris, pH 8.3; 0.5M Arginine).<br>
The refolding mixture was stored at 4°C for 2-4 days.<br>
HIC Purification:<br>
[773]	hlFN polypeptide refold samples were allowed to warm to room temperature,  A<br>
final concentration of 1.5M NaCl was added to the sample, and the sample was incubated at<br>
room temperature for 30-60 minutes. Samples were filtered with a .22µM PES filter and loaded<br>
onto a Butyl 650M column (Tosoh) equilibrated in HIC Buffer A (20µM sodium phosphate, pH<br>
7.0; 1.5M NaCl; 2M Urea). A linear gradient to 100% HIC Buffer B (20mM sodium phosphate,<br>
pH 7.0; 2M Urea) was run over 20 column volumes.<br>
SP HP Purification of HIC Pool:<br>
[774]	The IFN polypeptide peak was collected from the Butyl coluirm and diluted to<br>
less than 30m/S with water. The pH of samples were adjusted to 3.0 with HCl and were loaded<br>
onto a SP HP column (GE Healthcare) equilibrated in SP Buffer Al (50mM NaAc, pH 3.0;<br>
ImM EDTA). After the sample was loaded, the column was washed with 2 column volumes of<br>
SP Buffer A followed by 4 column volumes of SP Buffer A2 (50mM NaAc, pH 5.0; ImM<br>
EDTA). A linear gradient was run from 100% SP Buffer A2 to 50% SP B Buffer (50mM NaAc,<br>
pH 5.0; 0.5M NaCl; 10% ethylene glycol; ImM EDTA) over 15 column volumes.<br>
PEGylation and Pmfication:<br><br>
[775]	The hlFN polypeptide samples were pooled fi-om the SP HP column and<br>
concentrated to 1.0-2.0 mg/mL, 10% Acetic Acid was added to the sample to drop the pH down to 4.0. Oxyamino-derivatized 30K PEG was added to hlFN polypeptide at a 1:12 molar ratio (30mg / mg IFN/ml), and the mixture was incubated at 28°C for 48-72 hours. Figure 19 shows the 30K PEG used in the conjugation. The sample was diluted ten fold with SP Buffer A (50 mM Na acetate, pH 5; ImM EDTA), To purify the PEGylated hlFN, a SP HP column (GE Healthcare) was used with a gradient of 0-50% SP buffer B (50 mM Na acetate, pH 5, 0.5 M NaCl, 10% ethylene glycol; ImM EDTA) over 15 column volumes. PEGylated IFN eluted around 100 mM salt, followed by non-PEGylated monomer (baseline separation). The fractions of PEGylated hDFN were pooled, dialyzed against the following storage buffer: 20mM NaAcetate, pH 6, 0.005% Tween, 125 mM NaCl. Samples were concentrated to 1.0-2.0 mg/mL and stored at -80°C.<br>
Biacore studies (Receptor Bindins Affinity)<br>
[776]	Tlie sequence for the IFNαR2 extracellular domain (consisting of 206 amino<br>
acids ending with sequence LLPPGQ) was amplified from clone MHSl011-61064 (OpenBiosystems, Huntsville, AL). This insert was cloned into the pET20 expression vector (Novagen) downstream of the T7 promoter. Protein expression was induced with 0.4 mM IPTG in BL21(DE3) cells (Novagen),<br>
[777]	Since the expressed protein was insoluble, the inclusion bodies were purified<br>
from lysed cells and solubilized in 6M GndCl. A 5 ml aliquot (50 mg amount) was reduced with 10 mM DTT for 45 minutes at 37oC, Then the mixture was injected into 200 ml of refolding buffer which consisted of 50 mM Tris pH 8, 20 mM NaCl, 0.5 M Arginine, 10% glycerol at 4oC and incubated overnight with gentle stirring.<br>
[778]	The refolding reaction was then concentrated to 25 ml using an Amicon stirring<br>
cell, and dialyzed overnight against 20mM Tris, pH 8, 20 mM NaCl, 10% glycerol. Monomeric refolded IFNαR BCD was purified on HP Q Sepharose using the AKTA FPLC system (Amersham). Purified IFNαR2 ECD was immobilized on CM5 Biacore chip using a lysine-specific coupling procedure recommended by the manufacturer. About 200 RUs of functional protein were immobilized. Various concentrations of IFN variants in HBS-EP buffer (Biacore) were injected at a flow rate of 50 mcl/minute over the flowcell containing immobilized IFNαR2, and a control flowcell containing immobilized bovine serum albumin. Sensograms generated were fit to the 1:1 interaction model to calculate kon, koff and Kd values using BiaEvaluation<br><br>
software (Biacore). Methionyl wild-type interferon(M-WT) and PEGASYS® were included as control samples.<br>
[779]	Table 17 shows the kon, koff, and Kd obtained with hlFN polypeptides comprising<br>
the non-naturally encoded amino acid pAF and were PEGylated at the non-naturally encoded amino acid with the linear 30K PEG as shown in Figure 19. In addition, some of the hlFN polypeptides tested have a natural amino acid substitution. For example, T79R/N45 pAcF -30K PEG was generated by substituting the non-natural amino acid pAF at position 45 of SEQ ID NO: 2 and the threonine amino acid at position 79 was substituted with the naturally encoded amino acid arginine.<br>
  R<br>
Anti-viral assays<br>
[780]	To evaluate hlFN polypeptide anti-viral activity, 3x104 human WISH cells<br>
(ATCC) were seeded in a 96 well/plate and were subsequently infected with 10,000 PFU of VSV per well. At the time of infection, different amounts of hlFN polypeptide were added. 48 hours post-infection the CPE was evaluated; 42-48 hours was the minimum time required to obtain 100% CPE. CPE was identified by staming the cells with 0.1 ml of 1 mg/ml 3-(4,5-dimethyIthiazol-2-yI) 2,5-diphenyltetrazolium bromide (MTT) followed by spectrophotometric<br><br>
reading at 570 nm with 690 nm as the reference wavelength, MTT measures metabolic reduction in mitochondria.<br>
[781]	Table 18 shows the % relative anti-viral activity based on the IC50 values<br>
obtained with hlFN polypeptides that have a non-natural amino substitution at the site indicated and were PEGylated at that site. hlFN polypeptides with a lower IC50 value show higher % relative activity. Some of the PEGylated hlFN polypeptides in addition had a naturally encoded amino acid substitution (T79R; Y85L; E87S; Y85S).<br><br>
Example 33<br>
Measurement of Anti-proliferative Activity<br>
[782]	hlFN polypeptides described in the previous examples (Examples 30-32) were<br>
also assayed for anti-proliferative activity.   A prominent effect of IFNα's is their ability to<br>
inhibit cell growth, which is of major importance in determining anti-tumor action. The human<br>
lymphoblastoid Daudi cell line has proven to be extremely sensitive to IFNα's, and it has been<br>
used to measure antiproliferative activity in many IFNα's and derived hybrid polypeptides<br>
(Meister et al., J Gen Virol. (1986) Aug; 67 (Pt 8):1633-43). Use of this cell line has been<br><br>
facilitated by its ability to be grown in suspension cultures (Evinger and Pestka, (1981) Methods Enzymol. 79;362-368).<br>
[783]	The human Daudi B cell line was purchased from ATCC (Manassas, VA) and<br>
grown in RPMI 1640 supplemented with 10% heat inactivated fetal bovine serum (Hyclone,<br>
Logan, UT). The cell culture was maintained at 37oC in a humidified atmosphere of 5% CO2.<br>
[784]	The Daudi cells were plated at a density of 1 X 104 cells/well in a flat-bottom, 96-<br>
well plate. The cells were treated with increasing concentration of IFNα2A in triplicates per dose concentration in a final volume of 200ul. Following a 4-day incubation period at 37°C with 5% CO2, 20 ul of WST-1 solution Reagent (Roche cat# 1644807) was added to each well, and the culture was allowed to incubate for an additional 6 hours. Absorbance was read at 440 mn using a Spectromax. IC50S were obtained from dose response curves plotted with OD 440nm (average of triplicates) against protein concentration with SigmaPlot.<br>
[785]	Table 19 shows the results obtained with different hlFN molecules that are<br>
nonPEGylated and PEGylated. "Cu" indicates polypeptides that were made using the copper colunm as described in Example 30. "HIC" indicates polypeptides that were made using the HIC method as described in Example 32.<br><br><br><br>
Colony Formation Assays<br>
[786]	Colony formation assays  such as those described by  Giron-Michel, J.  in<br>
Leukemia 2002 16:1135-1142 may be used to evaluate proliferation of progenitor cells by hlFN polypeptides of the invention. Cord blood may be used in colony formation assays.<br>
Evaluation of the  Toxicity of hlFN Polypeptides on Human Myeloid and Erythroid Progenitors<br>
[787]	Using a methylcelluose-based in vitro colony forming assay, the hematopoietic<br>
toxicity of twenty one nonPEGylated compounds (twenty hlFN polypeptides with Roferon*^ as control) and eighteen PEGylated compounds (seventeen polypeptides PEGrylated with a 3 OK PEG with PEGASYS® as a control) were tested. The nonPEGylated hlFN polypeptides were made by the NusA method shown in Example 27 and were characterized as shown in Example 31. PEGylated hlFN polypeptides were generated and characterized as shown in Examples 30 and 31.<br><br>
[788]	Cells:  Normal human bone marrow light density cells (Poietics Inc., Maryland)<br>
were stored at -152°C until required for the assay. On the day of the experiment, the cells were thawed rapidly at 37°C, and the contents of the vial were diluted in 10 mis of Iscove's medium containing 2% fetal bovine serum and washed by centrifugation. The supernatant was discarded, and the cell pellet resuspended in a known volume of Iscove's medium containing 2% FBS. A cell count (3% glacial acetic acid) and viability assessment by Trypan Blue exclusion was performed,<br>
[789]	Test samples:   Compounds were in a buffer of 20 mM NaAc, 125 mM NaCI,<br>
0.005% Tween 80, pH 6.0. Each compoimd was tested in ten test concentrations in a series of five fold dilutions. Triplicates were performed at each dose concentration. Dilutions of each compound were prepared with the same buffer to generate the required dilutions. "^Tien added to methylcellulose, the buffer was at a final concentration of 6,7% and compound concentrations ranged from 5 - 0.00000256 ug/ml for nonPEGylated hlFN test polypeptides and 26 -0.O0OO133 ug/ml for 30K PEGylated hlFN test polypeptides. In instances where the initial 30K PEGylated hlFN test polypeptides concentrations are low, a concentration range of 14 -0.000007168 ug/ml was adopted instead.<br>
[790]	Method:      Clonogenic   progenitors   of the   erythroid   (CFU-E  and   BFU-E),<br>
granulocyte-monocyte (CFU-GM) and multipotential (CFU-GEMM) lineages were assessed in methylcellulose-based medium (MethoCult 4434) containing saturating concentrations of the cytokines SCF (Stem Cell Factor; 50 ng/mL), GM-CSF (Granulocyte Macrophage Colony Stimulating Factor; 10 ng/mL), IL-3 (Interleukin 3; 10 ng/mL), and EPO (Erythropoietin; 3 U/mL). CFU-E is a small erythroid colony derived from the most mature erythroid colony forming cells. It contains one to two clusters with a total number of 8-200 erythroblasts. BFU-E is a larger erythroid colony derived from a more primitive cell. It contains greater than 200 erythroblasts. CFU-GM is a colony that is derived from a colony forming cell capable of producing colonies with forty or more granulocyte-monocyte and/or macrophage cells. CFU-GEMM is a colony that contains cells from more than one lineage. It is derived from the most primitive colony forming cell and contains erythroid cells as well as twenty or more granulocytes, macrophages, and megakaryocytes. At each assay set up, a standard control containing MethoCult'^'^ 4434, cells and media and a buffer control containing MethoCull^"^ 4434, cells and equivalent amount of buffer but no compoimd were included to establish a condition in which no toxicity should be observed.<br><br>
[791]	Clonogenic progenitors of the erythroid (CFU-E and BFU-E), granulocyte-<br>
monocyte/myeloid (CFU-GM) and multipotential (CFU-GEMM) lineages were set up in the methylcellulose-based media described. The compounds were added to the MethoCultTM to give final concentrations as described above. Vehicle control cultures containing no compound but equivalent concentrations of vehicle buffer as well as standard controls containing no compounds or vehicle buffer but media alone were also initiated. All cultures were set up in triplicate at 1x10' cells per culture. Following 14 days in culture, the colonies were assessed and scored. The colonies were divided into the following categories, based on size and morphology: CFU-E, BFU-E, CFU-GM, and CFU-GEMM.<br>
[792]	Results and analysis:   Triplicate cultures for CFU-E, BFU-E, CFU-GM, and<br>
CFU-GEMM were enumerated. In addition, the distribution of colony types as well as general colony and cellular morphology were analyzed. In all test sets, no difference between the numbers of colonies generated in standard cultures compared to the buffer controls were observed. However, in the presence of each nonPEGylated or PEGylated hlFN polypeptides, colony numbers as well as colony size of both the myeloid and erythroid lineage were perturbed when compared to standard and buffer assay controls. The morphology of the resulting erythroid and myeloid colonies was however not affected. A dose dependent toxic effect on both myeloid and erythroid progenitors were seen for all nonPEGylated and PEGylated hlFN tested including compound controls Roferon® and PEGASYS®.<br>
[793]	Further analysis of test compounds were performed to normalize for differences<br>
in anti-viral activity observed between compounds. Mass concentrations of hIEN polypeptides used m the colony formation assay were divided by the anti-viral IC50 values measured in the VSV replication assay. The myeloid and erythroid colony counts were then plotted on Y-axis against the normalized protein concentration on a log x-axis. Relative toxicity of each test polypeptides compared to the appropriate compound control was determined by the factor rho where rho represents the shift in x-axis between two curves. When rho&gt;l, test compound is more potent and hence more toxic than control. When rho&gt;l, the fold improvement was represented as a negative number that has the value of rho (- rho) (See Tables 22 and 23). When rho<l test compound is less toxic than control. when rho the fold improvement was represented as a positive number that has value of tables and for nonpegylated hlfn polypeptides roferon used pegylated pegasys></l><br><br><br><br>
Comparison of hlFN Polypeptides<br>
[794]	For the VSV assay, more active molecules have a lower IC50 value. If the VSV<br>
IC50 of the test compound was less than the VSV IC50 of M-WT (wild-type interferon with a methionine at the N-terminus), then the fold improvement was expressed as VSV IC50 M-WT/VSV IC50 test compound. If the VSV IC50 of the test compound was greater than the VSV IC50 of M-WT, then the fold improvement was expressed as -(VSV IC50 test compoundA^SV IC50 M-WT). Table 22 shows the fold improvement data for a set of bIFN polypeptides described also in Example 33.<br><br>
[795]	For the VSV assay, more active molecules have a lower IC50 value.  If the VSV<br>
IC50 of the test compound was less than the VSV IC50 of PEGASYS®, then the fold improvement was expressed as VSV IC50 PEGASYS®A/SV IC50 test compound. If the VSV IC50 of the test compound was greater than the VSV IC50 of PEGASYS®, then the fold<br><br><br><br><br>
Example 35<br>
Human Clinical Trial of the Safety and/or Efficacy of PEGylated hlFN Comprising a Non-naturally Encoded Amino Acid.<br>
[797]	A Phase 0 study is performed to investigate microdosing of hERN polypeptides of<br>
the invention. This study involves a small number of subjects. These studies involve measuring changes in molecular markers with administration of a fraction of the therapeutic dose. These same markers correlate with a therapeutic response when the drug is administered to an affected individual. The drug-response information obtained is useful for Phase I and Phase II trial design.<br>
[798]	Objective      To compare the safety and pharmacokinetics of subcutaneously<br>
administered PEGylated recombinant human hlFN comprising a non-naturally encoded amino acid with the commercially available hlFN products Roferon A® or Intron A®.<br>
[799]	Patients    Eighteen healthy volunteers ranging between 20-40 years of age and<br>
weighing between 60-90 kg are enrolled in the study. The subjects will have no clinically significant abnormal laboratory values for hematology or serum chemistry, and a negative urine toxicology screen, HIV screen, and hepatitis B surface antigen. They should not have any evidence of the following: hypertension; a history of any primary hematologic disease; history of significant hepatic, renal, cardiovascular, gastrointestinal, genitourinary, metabolic, neurologic disease; a history of anemia or seizure disorder; a known sensitivity to bacterial or mammalian-derived products, PEG, or human serum albumin; habitual and heavy consumer to beverages containing caffeine; participation in any other clinical trial or had blood transfused or donated within 30 days pf study entry; had exposure to hlFN within three months of study entry; had an illness within seven days of study entry; and have significant abnormalities on the pre-study physical examination or the clinical laboratory evaluations within 14 days of study entry. All subjects are evaluable for safety and all blood collections for pharmacokinetic analysis are collected as scheduled. All studies are performed with institutional ethics committee approval and patient consent.<br><br>
[800]	Study Design  This will be a Phase I,  single-cancer,open-label,  randomized, two-<br>
period crossover study in healthy male volunteers. Eighteen subjects are randomly assigned to one of two treatment sequence groups (nine subjects/group). IFN is administered over two separate dosing periods as a bolus s.c. injection in the upper thigh using equivalent doses of the PEGyiated hlFN comprising a non-naturally encoded amino acid and the commercially available product chosen. The dose and frequency of administration of the commercially available product is as instructed in the package label. Additional dosing, dosing frequency, or other parameter as desired, usmg the commercially available products may be added to the study by including additional groups of subjects. Each dosing period is separated by a 14-day washout period. Subjects are confined to the study center at least 12 hours prior to and 72 hours following dosing for each of the two dosing periods, but not between dosing periods. Additional groups of subjects may be added if there are to be additional dosing, frequency, or other parameter, to be tested for the PEGyiated hlFN as well. Multiple formulations of IFN that are approved for human use may be used in this study. Roferon A® and/or Intron A® are commercially available IFN products approved for human use. The experunental formulation of hlFN is the PEGyiated hlFN comprising a non-naturally encoded amino acid.<br>
[801]	Blood Sampling  Serial blood is drawn by direct vein puncture before and after<br>
administration of hlFN. Venous blood samples (5 mL) for determination of serum IFN concentrations are obtained at about 30, 20, and 10 minutes prior to dosing (3 baseline samples) and at approximately the following times after dosing: 30 minutes and at 1,2, 5, 8, 12, 15, 18, 24, 30, 36, 48, 60 and 72 hours. Each serum sample is divided into two aliquots. All serum samples are stored at -20°C. Serum samples are shipped on dry ice. Fasting clinical laboratory tests (hematology, serum chemistry, and urinalysis) are performed immediately prior to the initial dose on day I, the morning of day 4, immediately prior to dosing on day 16, and the morning of day 19.<br>
[802]	Bioanalvtical Methods    An ELISA kit procedure (BioSource International<br>
(Camarillo, CA)), is used for the detennination of serum IFN concentrations.<br>
[803]	Safety Determinations Vital signs are recorded immediately prior to each dosing<br>
(Days 1 and 16), and at 6, 24, 48, and 72 hours after each dosing. Safety determinations are based on the incidence and type of adverse events and the changes in clinical laboratory tests from baseline. In addition, changes from pre-study in vital sign measurements, including blood pressure, and physical examination results are evaluated.<br><br>
[804]	Data Analysis Post-dose serum concentration values are corrected for pre-dose<br>
baseline IFN concentrations by subtracting from each of the post-dose values the mean baseline IFN concentration determined from averaging the IFN levels from the three samples collected at 30, 20, and 10 minutes before dosing. Pre-dose serum IFN concentrations are not included in the calculation of the mean value if they are below the quantification level of the assay. Pharmacokinetic parameters are determined from serum concentration data corrected for baseline IFN concentrations. Pharmacokinetic parameters axe calculated by model independent methods on a Digital Equipment Corporation VAX 8600 computer system using the latest version of the BIOAVL software. The following pharmacoldnetics parameters are determined: peak serum concentration (Cmax); time to pealc serum concentration (tmax); area under the concentration-time curve (AUC) from time zero to the last blood sampling time (AUCo-72) calculated with the use of the linear trapezoidal rule; and terminal elimination half-life (ti/2), computed from the elimination rate constant. The elimination rate constant is estimated by linear regression of consecutive data points in the terminal linear region of the log-linear concentration-time plot. The mean, standard deviation (SD), and coefficient of variation (CV) of the pharmacokinetic parameters are calculated for each treatment. The ratio of the parameter means (preserved formulation/non-preserved formulation) is calculated.<br>
[805]	Safety Results The incidence of adverse events is equally distributed across the<br>
treatment groups. There are no clinically significant changes from baseline or pre-study clinical laboratory tests or blood pressures, and no notable changes from pre-study in physical examination results and vital sign measurements. The safety profiles for the two treatment groups should appear similar.<br>
[806]	Pharmacoldnetic   Results      Mean   serum   IFN   concentration-time   profiles<br>
(uncorrected for baseline IFN levels) in all 18 subjects after receiving a single dose of commercially available hlFN (e.g. Roferon A® or Intron A®) are compared to the PEGylated hlFN comprising a non-naturally encoded amino acid at each time point measured. All subjects should have pre-dose baseline IFN concentrations within the normal physiologic range. Pharmacokinetic parameters are determined from serum data corrected for pre-dose mean baseline IFN concentrations and the Cmax and tmax are determined. The mean tmax for hlFN (e.g. Roferon®) is significantly shorter than the tmax for the PEGylated hlFN comprising the non-naturally encoded amino acid. Terminal half-life values are significantly shorter for hlFN (e.g. Intron A®) compared with the terminal half-life for the PEGylated hlFN comprising a non-naturally encoded amino acid.<br><br>
[807]	Although the present study is conducted in healthy male subjects, similar<br>
absorption characteristics and safety profiles would be anticipated in other patient populations; such as male or female patients with cancer or chronic renal failure, pediatric renal failure patients, patients in autologous predeposit programs, or patients scheduled for elective surgery. Alternatively, the Phase I study is performed in HCV subjects to evaluate PEGylated hlFN polypeptide.<br>
[808]	In conclusion, subcutaneously administered single doses of PEGylated hlFN<br>
comprising non-naturally encoded amino acid will be safe and well tolerated by healthy male subjects or HCV subjects. Based on a comparative incidence of adverse events, clinical laboratory values, vital signs, and physical examination results, the safety profiles of hlFN (e.g. Roferon A®) and PEGylated hEFN comprising non-naturally encoded amino acid will be equivalent. The PEGylated hlFN comprising non-naturally encoded amino acid potentially provides large clinical utilitj' to patients and health care providers.<br>
Example 36<br>
[809]	This example details methods used to generate hlFN polypeptides comprising a<br>
non-naturally encoded amino acid. Purified refolded hlFN polypeptides were conjugated to poly(ethylene glycol) according to the methods described below. Modifications to the methods described below and alternate methods to refold polypeptides such as hlFN polypeptides are known to those of ordinary skill in the art. Modifications to the methods described include, but are not limited to, the addition, substitution, or subtraction of one or more steps mentioned below. Potential methods that may be used or one or more modifications to the methods described include, but are not limited to, washing inclusion bodies with reagents that reduce the amount of contaminant proteins that potentially interact with the hlFN polypeptide such reagents include but are not limited to urea and deoxycholic acid; adding a reagent to the refolding reaction such reagents include but are not limited to a denaturant, guanidine HCl, urea, cysteine, and catalytic amounts of a reducing agent. Columns other than those described may be used in to isolate or purify hlFN polypeptides comprising a non-naturally encoded amino acid. Such columns include but are not limited to a hydrophobic interaction chromatography (HIC) column such as a butyl, octyl, or phenyl HIC column. Alternate conditions and solutions may be used for isolation or purification of hlFN polypeptides, including but not limited to the use of alternate salts such as ammonium sulfate, sodium citrate, and other salts; the adjustment of buffer components including but not limited to, the adjustment of salt concentrations to facilitate<br><br>
binding to a column used m purification/isolation; alterations in pH; the replacement of one or<br>
more reagents; and the inclusion of one or more reagents, including but not limited to, arginine.<br>
[810]	Alternate methods include but are not limited to, solubilization of cell pellets with<br>
guamdine and other methods known to those of ordinary skill in the art.   Different strains of bacteria may also be used in the expression of hlFN polypeptides. Purification oflFN inclusion bodies<br>
[811]	For each hlFN polypeptide, E, coli strains BL21(A1). BL2](DE3) or W3110(B2)<br>
transformed with a construct encoding an orthogonal tRNA (J17 described in U.S. Patent Publication No. 20030108885) and an orthogonal aminoacyl tRNA synthetase (E9 described in U.S. Patent Application No. 11/292,903 entitled "Compositions of Aminoacyl-tRNA Synthetase and Uses Thereof) with a mutated hlFN polynucleotide. The bacteria were plated onto LB Agar with ampicillin plates and incubated overnight at 37°C. Colonies were selected and grown in culture to make glycerol stocks.<br>
[812]	5 ul aliquots of glycerol stock were used to make a 5 ml starter culture in the<br>
morning. The starter culture was added to LB media containing para-acetylphenylalanine (pAF) in the morning (IX LB; 4 mM pAF; IX AMP), and the cells were induced in the evening or when the culture had an OD of greater than 1.0. The culture was grown/induced at 37^C degrees with shaking at 250 rpm overnight.<br>
[813]	The next morning the OD was measured of the cultures, and the cells were<br>
pelleted. The OD measurements of the cultures were between about 3 and about 8. The cell pellet was either frozen at -80°C or the inclusion bodies were prepared from the pellet immediately. An aliquot was analyzed by SDS-PAGE with a Coomassie stain to check expression and the length of product produced. The bacterial pellet was resuspended in 35 ml of IBl buffer (50 mM NaAc, pH 6.0; 100 mM NaCl; 1 mM EDTA; 0.1% Triton X). Then the sample was sonicated six times to resuspend and lyse the cells. 30 second bursts of sonication were followed by 1 minute incubations on ice. The bacterial lysate was centrifuged in a 50 ml Oaloidge tube at 13,000 rpm for 20 minutes at 4°C, and the supernatant was discarded. The pellet was washed four times with IBl buffer using sonication to resuspend. The inclusion bodies were spun down, and the supernatant was discarded. The pellet was washed two times with IB2 buffer (50 mM NaAc, pH 6.0; 100 mM NaCl; 1 mM EDTA), and tlie supernatant was discarded. The pellet was resuspended in 5-15 ml of 8M Guanidine HCl (8 M GnHCl; 50mM NaAc pH 6.0) and were resuspended using a douncer.   The concentration of the sample was<br><br>
determined using a 100x dilution, and the OD was measured at 280 mn. The IFN extinction coefficient is 22800 or 1.17. Sample concentrations were normalized to 5.0 mg/ml protein. The purity of the inclusion bodies was determined by SDS PAGE gel with a Coomasie stain. Twenty-seven hlFN polypeptides with a para-acetylphenylalanine substitution were suppressed, and inclusion bodies for twenty-seven mutants were isolated. Each of the twenty-seven mutants generated had one non-naturally encoded amino acid substitution in which para-acetylphenylalanine replaced a naturally encoded amino acid in hlFN (SBQ ID NO: 2). The 27 hlFN polypeptides generated had a substitution at one of the following positions: 23, 24, 25, 26, 27, 28, 30, 31, 32, 33, 128, 129,131,132,133, 134,135, 136, 137,158, 159, 160, 161, 162,163, 164, and 165.<br>
Refolding hlFN using purified inclusion bodies<br>
[814]	100 mg of 5.0 mg/ml hlFN was added to 400 ml Refolding Buffer (50 mM NaAc<br>
pH 6.0; 0.1% Tween 20) while spinning slowly at 4°C. Refolding was performed for 24 to 48 hours. The refolding mixture was then brought to room temperature. Then 50uM CuCl was added, and the sample was mixed. The sample was incubated for 15 minutes at room temperature.<br>
Protein purification and PEGylation<br>
[815]	Any precipitate that formed during refolding was spun down or filtered, and the<br>
correctly folded hlFN polypeptide was purified using a copper chelate column. Copper Column Buffer A was 50 mM NaAc, pH 5.0; 0.15 M NaCl; 0.1% Tween 20. Copper Column Buffer B was 100 mM NaAc. pH 3.5; 0.15 M NaCl; 0.1% Tween 20.<br>
[816]	An aliquot was checked at this point by SDS-PAGE with a Commassie stain.<br>
Pooled fractions from the copper column were loaded onto a 5 ml SP HP cation exchange column. The 5 ml SP HP Buffer A was 50 mM NaAc pH 6.0; 1 mM EDTA. The 5 ml SP HP buffer B was 50 mM NaAc pH 6.0; 1 mM EDTA; 500 mM NaCl; 10% Ethylene glycol. The amount of total protein was measured at this point using by measuring the OD at 280 nm, and SDS-PAGE with a Commassie stain was also performed.<br>
[817]	Fractions fi"om the copper chelate column were pooled and were concentrated to<br>
greater than 1.0 mg/ml and 10% acetic acid was added to each sample to bring the pH to 4.0. The linear 3OK PEG shown in Figure 19 was added at a 12:1 molar ratio. The sample was incubated at 28°C for 24 to 48 hours. The sample was then diluted 10-fold with Milli-Q water, and the PEGylated hlFN polypeptide was purified over a 5 ml SP HP column. SDS-PAGE was<br><br>
performed with a Commassie stain to analyze the sample and determine the level of higher molecular weight species and unPEGylated contaminants. Fractions from the SP column were collected, and then the pool was dialyzed against Storage Buffer (25 mM NaAc, pH 6.0; 120 mM NaCl; 0.005% Tween 80). An aliquot of the final product was then analyzed by SDS-PAGE. The final product was concentrated to 180 ug/ml, and aliquots were stored at -80 °C. Example 37 VSV Anti-Viral Assay<br>
[818]	Antiviral activity of hlFN polypeptides may be measured by a variety of assays.<br>
The antiviral activity of PEGylated hlFN polypeptides was determined using the Vesicular Stomatitis Virus (VSV). The culture media for this assay was DMEM. 10% FBS, 1% Penicillin/Streptomycin, 5 m! HEPES. Additional reagents included RPMI without FBS and Phenol Red. The concentration of MTT stock used was 5 mg/mL in PBS. This stock solution was stored for only two weeks at 4°C.<br>
[819]	Human WISH cells were seeded at 30,000 cells/well in 50µL of culture media.<br>
The following day, 2x serial dilutions were performed of the hlFN polypeptides in culture media. PEGASYS® was used as a control in these experiments. In triplicate, 100 µL of the diluted hlFN polypeptides was added to the WISH cells for a final well volume of 150µL. The cells and hlFN polypeptides were incubated for 6 hours at 37° C in a CO2 incubator. After this six hour incubation, 10,000 PFU of VSV was added per well, in a volume of 50µL of media; 20 µL of VSV was diluted in 5 ml of media per 96 well/plate. The media was removed by aspiration forty-five hours post-infection. The plates were gently tapped on paper towels to remove residual media.<br>
[820]	50µL of 1 mg/mL MTT (3-(4,5-dimethylthia2ol-2-yl) 2,5-diphenyltetrazolium<br>
bromide) prepared in RPMI without Phenol Red and FBS was added from a 5 mg/mL MTT stock solution. I mg/mL MTT was usually prepared fresh for every assay. Next, the plates were incubated for three hours at 37°C in the CO2 incubator. The MTT was carefully removed by aspiration. 50 µL of isopropanol was added per well. The plates were placed on a plate-shaker for 30 to 40 seconds to complete the formation of MTT. The plates were read at 560 nm with 690 nm as a reference wavelength. The data was plotted and the IC50 calculated using the Sigma-Plot program.<br>
[821]	Table 25 summarizes the anti-viral activity of 20 hlFN polypeptides that were<br>
conjugated to 30K PEG via a para-acetylphenylalanine  substitution  in the polypeptide.<br><br>
The second column lists the averaged IC50 values for each compound across multiple anti-viral<br>
activity experiments. The third column is the % Efficacy relative to the averaged IC50 values<br>
found in the second column.<br>
[822]	The fourth column is the Average % Efficacies.   For this data, the % Efficacy<br>
was established by comparing IC50S to the IC50 of PEGASYS® in individual experiments. Then<br>
the % Efficacies from all experiments were averaged.<br>
Table 25:<br><br><br><br>
Example 38<br>
[823]	This example details the measurement of hlFN activity and affinity of hlFN<br>
polypeptides for the hlFN receptor.<br>
Biacore studies {Receptor Bindine: Affinity)<br>
[824]	The sequence for the IFNαR2 extracellular domain (consisting of 206 amino<br>
acids   ending   with   sequence   LLPPGQ)   was   amplified   &amp;om   clone   MHS1011-61064<br>
(OpenBiosystems, Huntsville, AL). This insert was cloned into the pET20 expression vector<br>
(Novagen) downstream of the T7 promoter.   Protein expression was induced with 0.4 mM IPTG<br>
in BL21(DE3) cells (Novagen).<br>
[825]	Since the expressed protein was insoluble, the inclusion bodies were purified<br>
from lysed cells and solubilized in 6M GndCl. A 5 ml aliquot (50 mg amount) was reduced with<br>
10 mM DTT for 45 minutes at 37°C.  Then the mixture was injected into 200 ml of refolding<br>
buffer which consisted of 50 mM Tris pH 8,20 mM NaCl, 0.5 M Arginine, 10% glycerol at 4^*0<br>
and incubated overnight with gentle stirring.<br>
[826]	The refolding reaction was then concentrated to 25 ml using an Amicon stirring<br>
cell, and dialyzed overnight against 20mM Tris, pH 8, 20 mM NaCl, 10% glycerol. Monomeric<br>
refolded IFNαR BCD was purified on HP Q Sepharose using the AKTA FPLC system<br>
(Amersham).   Purified IFNαR2 BCD was immobilized on CM5 Biacore chip using a lysine-<br>
specific coupling procedure recommended by the manufacturer. About 200 RUs of functional<br>
protein were immobilized. Various concentrations of IFN variants in HBS-EP buffer (Biacore)<br>
were injected at a flow rate of 50 mcl/minute over the flowcell containing immobilized IFNαR2,<br>
and a control flowcell containing immobilized bovine serum albumin. Sensograms generated<br>
were fit to the 1:1 interaction model to calculate kon, koff and Kd values using BiaEvaluation<br>
software (Biacore).   PEGASYS   was included as a control sample. Table 26 shows the average<br>
Kons koff, and Kd obtained with the hlFN polypeptides characterized in Example 37. The standard<br>
deviations (SD) calculated for each value are also shown.<br>
Table 26:<br><br><br><br>
Example 39<br>
[827]	hlFN polypeptides of the invention may be evaluated with a number of different<br>
assays. Potential assays include, but are not limited to, mRNA gene expression profiling, CFU<br>
assays including but not limited to CFU-GM and CFU-MK assays, assays measuring<br>
CrkL/STAT phosphorylation, MHC Class I expression assays, and n^RNA profiling, as well as<br>
animal studies such as pharmacokinetics studies in rats, studies using a NOD/SCID<br>
reconstitution model, and pharmacokinetics/pharmacodynamics in monkeys. Colony formation<br>
assays such as those described by Giron-Michel, J. in Leukemia 2002 16:1135-1142 or<br>
described herein may be used to evaluate proliferation of progenitor cells by hlFN polypeptides<br>
of the invention. Bone marrow or cord blood may be used in colony formation assays.<br>
[828]	To assess the biological activity of modified hlFN polypeptides of the invention,<br>
an assay measuring phosphorylation of STAT3, a signal transducer and activator of transcription<br>
family member, is performed using the human adenocarcinoma HeLa cell line. The human cell<br>
line HeLa is purchased from ATCC (Manassas, VA) and is routinely passaged in DMEM plus<br>
10% heat-inactivated fetal bovine serum. The cells are maintained at 37°C in a humidified<br>
atmosphere of 5% CO2. The HeLa cells are starved overnight in assay media without fetal<br>
bovine serum before stimulation with increasing concentrations of hlFN polypeptides for 10<br>
minutes at 37°C. Phosphorylation of STAT3 may be measured with PathScan Phospho-STAT3<br>
(Tyr705) ELISA Kit (Cell Signaling Technology). With this kit, the assay is performed m 96<br>
well plates and has a colorimetric readout measurable by a spectrophotometric plate reader.<br>
[829]	The hlFN polypeptides of the invention may be evaluated using a NOD/SCID<br>
reconstitution model. PK/PD studies in monkeys may be performed as described previously. Cynomolgus monkeys are also used to study in vivo activity and bone marrow toxicity. Activity is assayed by measuring the induction of downstream markers including but not limited to 2',5'-OAS by hlFN polypeptides of the invention. Circulating blood cells, including but not limited to neutrophils, RBCs, and platelets are evaluated in bone marrow toxicity studies after administration of hlFN polypeptides. Bone marrow is collected and evaluated from animals exposed to hlFN polypeptides of the invention. For example, bone marrow aspirates and/or core biopsies from treated animals (e.g. from primates) are examined histologically, and the ratio of myeloid to nucleated erythroid cells (M;E ratio) is estimated. A low M;E ratio, in conjunction with neutropenia, may be indicative of granulocytic hypoplasia with decreased production of<br><br>
neutrophils. Isolated bone marrow cells from treated animals are assayed for hematopoietic progenitors. That is, cell isolates are subjected to CFU-GM or BFU-E colony assay. The results (i.e. number of hematopoietic progenitor cells) would show the effect, if any, of treatment on myeloid and/or erythroid progenitor cells. Such bone marrow toxicity studies may involve hlFN polypeptides administered via different routes, including but not limited to, subcutaneously and intravenously. Intravenous administration may allow for longer dosing regimens and better evaluation of bone marrow toxicity than subcutaneous administration. Such improvements may result from a modulation of immunogenicity or an antibody response to the compound administered. Toxicity studies include evaluation of neutrophil and platelet number/mL of plasma or serum.<br>
[830]	Assays to support preliminary formulation studies may be performed which<br>
include but are not limited to, assays for protein de-amidation, aggregation, oxidation, acetylation, and other stability indicating assays. Other assays may be used that investigate other potential degradation products including, but not limited to, disulfide rearrangements and proteolytic degradation products.<br>
[831]	It is understood that the examples and embodiments described herein are for<br>
illustrative purposes only and that various modifications or changes in light thereof will be suggested to persons skilled in the art and are to be included within the spirit and purview of this application and scope of the appended claims. All publications, patents, and patent applications cited herein are hereby incorporated by reference herein in their entirety for all purposes.<br><br>
TABLE 27: hlFN Sequences Cited.<br><br><br>
WHAT IS CLAIMED IS:<br>
1.	A hlFN polypeptide comprising one or more non-naturally encoded amino acids<br>
having at least one modulated biological activity.<br>
2.	The hlFN polypeptide of claim 1, v^herein the hlFN polypeptide comprises one or<br>
:     more post-translational modifications.<br>
3.	The hlFN polypeptide of claim 1, wherein the polypeptide is linked to a linlcer, polymer, or biologically active molecule.<br>
4.	The hlFN polypeptide of claim 3, wherein the polypeptide is linlced to a water soluble polymer.<br>
5.	The hlFN polypeptide of claim 1, wherein the polypeptide is linked to a bifunctional polymer, bifunctional linker, or at least one additional hlFN polypeptide.<br>
6.	The hlFN polypeptide of claim 5, wherein the bifunctional linker or polymer is linked to a second polypeptide.<br>
7.	The hlFN polypeptide of claim 6, wherein the second polypeptide is a hlFN polypeptide.<br>
8.	The hlFN polypeptide of claim 4, wherein the water soluble polymer comprises a poly(ethylene glycol) moiety,<br>
9.	The hlFN polypeptide of claim 4, wherein said water soluble polymer is linked to a non-naturally encoded amino acid present in said hlFN polypeptide.<br>
10.	The hlFN polypeptide of claim 1, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 1-9, 10-21, 22-39, 40-<br>
75,	76-77, 78-100, 101-110, 111-132, 133-136, 137-155, 156-165 from SEQ IDNO: 2.<br>
11.	The hlFN polypeptide of claim 1, wherein the non-naturally encoded amino acid<br>
is substituted at a position selected from the group consisting 6f residues before position 1 (i.e.,<br>
at the N-terminus), 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15,16,17,18,19, 20,21, 22, 23,<br>
24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43,44, 45,46, 47, 48, 49,<br>
50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75,<br>
76,	77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95,96, 97, 98, 99,100,<br><br>
101, 102,103,104, 105,106,107,108,109, 110, 111, 112,113,114,115,116,117,118, 119, 120, 121,122, 123, 124,125, 126, 127,128, 129, 130, 131, 132, 133, 134, 135,136,137, 138, 139, 140, 141, 142, 143,144, 145, 146, 147, 148,149,150,151,152,153,154,155,156,157, 158, 159,160, 161, 162,163,164,165, or 166 (i.e. at the carboxyl terminus), and any combination thereof from SEQ ID NO: 2.<br>
12.	The hlFN polypeptide of claim 11, wherein the non-natuTEiIly encoded amino acid is substituted at a position selected from the group consisting of residues 100,106, 107, 108, 111,113,114, and any combination thereof from SEQ ID NO: 2.<br>
13.	The hlFN polypeptide of claim 11, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 41, 45,46,48, 49, and any combination thereof from SEQ ID NO: 2.<br>
14.	The hlFN polypeptide of claim 11, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 61, 64, 65,101, 103, 110, 117, 120, 121, 149, and any combination thereoffrom SEQ ID NO: 2.<br>
15.	The hlFN polypeptide of claim 11, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 6, 9,12,13, 16, 96, 156, 159, 160, 161, 162, and any combination thereof, from SEQ ID NO: 2.<br>
16.	The hlFN polypeptide of claim 11, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 2,3,4,5,7, 8, 16, 19, 20,40, 42, 50, 51, 58, 68, 69, 70, 71, 73, 97,105,109,112, 118,148,149, 152,153,158, 163, 164, 165, and any combination thereof from SEQ ID NO: 2.<br>
17.	The hlFN polypeptide of claim 4, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues before position 1 (i.e., at the N-terminus), 1,2, 3,4, 5,6, 7, 8,9,10,11,12,13,14, 15,16,17, 18, 19,20,21, 22,23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40,41, 42,43, 44, 45,46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99,100. 101, 102,103, 104, 105, 106,107,108, 109,110,111, 112, 113,114,115, 116,117,118, 119, 120, 121, 122,123, 124, 125,126, 127,128,129, 130,131,132, 133, 134, 135,136,137, 138, 139,140,141,142,143, 144, 145, 146, 147,148,149, 150, 151, 152, 153, 154, 155, 156, 157,<br><br>
158, 159,160,161, 162, 163, 164, 165, or 166 (i.e. at the carboxyl terminus), and any combination thereof from SEQ ID NO: 2.<br>
18.	The hlFN polypeptide of claim 17, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 6, 9,12, 13,16,41,45, 46,48,49,61,64,65,96,100,101,103,106,107,108,110,111,113,114, 117, 120, 121, 149, 156, 159, 160, 161 and 162, and any combination thereof, from SEQ ID NO: 2.<br>
19.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition or deletion that modulates affinity of the hlFN polypeptide for a hlFN receptor.<br>
20.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition or deletion that increases the stability or solubility of the<br>
hlFN polypeptide.<br>
21.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition or deletion that increases the expression of the hlFN polypeptide in a recombinant host cell.<br>
22.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition or deletion that increases protease resistance of the hlFN polypeptide.<br>
23.	The hlFN polypeptide of claim 1, wherein the non-naturally encoded amino acid is reactive toward a linker, polymer, or biologically active molecule that is otherwise unreactive toward any of the 20 common amino acids in the polypeptide.<br>
24.	The hlFN polypeptide of claim 1, wherein the non-naturally encoded amino acid comprises a carbonyl group, an aminooxy group, a hydrazine group, a hydrazide group, a semicarbazide group, an azide group, or an alkyne group.<br>
25.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid comprises a carbonyl group.<br>
26.	The hlFN polypeptide of claim 25, wherein the non-naturally encoded amino acid has the structure:<br><br><br>
27.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid comprises an aminooxy group.<br>
28.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid comprises a hydra2dde group.<br>
29.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid<br>
comprises a hydrazine group.<br>
30.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid<br>
residue comprises a semicarbazide group.<br>
31.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid<br>
residue comprises an azide group.<br><br>
modification group, and R3 is H, an amino acid, a polypeptide, or a carboxy terminus modification group.<br>
33.	The hlFN polypeptide of claim 24, wherein the non-naturally encoded amino acid comprises an alkyne group.<br>
34.	The hlFN polypeptide of claim 33, wherein the non-naturally encoded amino acid has the structure:<br><br><br>
group, and R3 is H, an amino acid, a polypeptide, or a carboxy tenninus modification group.<br>
35.	The hlFN polypeptide of claim 4, wherein the water soluble polymer has a molecular weight of between about 0.1 IcDa and about 100 kDa.<br>
36.	The hlFN polypeptide of claim 35, wherein the water soluble polymer has a molecular weight of between about 0.1 IcDa and about 50 kDa.<br>
37.	The hlFN polypeptide of claim 4, which is made by reacting a hlFN polypeptide<br>
comprising a carbonyl-containing amino acid with a water soluble polymer comprising an<br>
aminooxy, hydrazine, hydrazide or semicarbazide group.<br>
38.	The hlFN polypeptide of claim 37, wherein the aminooxy, hydrazine, hydrazide<br>
or semicarbazide group is linked to the water soluble polymer through an amide linkage.<br>
39.	The hlFN polypeptide of claim 4, which is made by reacting a water soluble<br>
polymer comprising a carbonyl group with a polypeptide comprising a non-naturally encoded<br>
amino acid that comprises an aminooxy, a hydrazine, a hydrazide or a semicarbazide group.<br>
40.	The hlFN polypeptide of claim 4, which is made by reacting a hlFN polypeptide<br>
comprising an alkyne-containing amino acid with a water soluble polymer comprising an azide<br>
moiety.<br>
41.	The hlFN polypeptide of claim 4, which is made by reacting a hlFN polypeptide<br>
comprising an azide-containing amino acid with a water soluble polymer comprising an alkyne<br>
moiety.<br>
42.	The hlFN polypeptide of claim 24, wherein the aade or alkyne group is linked to<br>
a water soluble polymer through an amide linkage.<br>
43.	The hlFN polypeptide of claim 4, wherein the water soluble polymer is a<br>
branched or multiarmed polymer.<br><br>
44.	The hlFN polypeptide of claim 43, wherein each branch of the branched polymer has a molecular weight of between about 1 kDa and about 100 kDa.<br>
45.	The hlFN polypeptide of claim 1, wherein the polypeptide is a hlFN antagonist.<br>
46.	The hlFN polypeptide of claim 45, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 2, 3, 4, 5, 7, 8, 16, 19, 20,40,42, 50, 51, 58, 68, 69,70,71, 73,97,105,109,112,118,148,149,152,153, 158, 163, 164, 165, and any combination thereof from SEQ ID NO: 2.<br>
47.	The hlFN polypeptide of claim 45, wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues 22, 23, 24, 25, 26, 27, 28,30, 31, 32, 33, 34, 35,74, 77, 78, 79, 80, 82, 83, 85, 86, 89, 90, 93, 94,124, 125, 127, 128, 129, 131, 132, 133, 134,135, 136,137, and any combination thereof from SEQ ID NO: 2.<br>
48.	The hlFN polypeptide of claim 45, wherein the polypeptide comprises one or more post-translalional modification, linlcer, polymer, or biologically active molecule.<br>
49.	The hlFN polypeptide of claim 48, wherein the polymer comprises a moiety selected from a group consisting of a water soluble polymer and poly(ethylene glycol).<br>
50.	The hlFN polypeptide according to claim 45, wherein the non-naturally encoded amino acid is present within the the receptor-binding region of the hlFN polypeptide.<br>
51.	The hlFN polypeptide according to claim 45, wherein the polypeptide prevents dimerization of the hlFN receptor.<br>
52.	The hlFN polypeptide of claim 1, wherein the non-naturally encoded amino acid comprises a saccharide moiety.<br>
53.	The hlFN polypeptide of claim 3, wherein the linker, polymer, or biologically active molecule is linked to the polypeptide via a saccharide moiety.<br>
54.	An isolated nucleic acid comprising a polynucleotide that hybridizes under stringent conditions to SEQ ID NO:21 or SEQ ID NO:22, wherein the polynucleotide comprises at least one selector codon.<br><br>
55.	The isolated nucleic acid of claim 54, wherein the selector codon is selected from the group consisting of an amber codon, ochre codon, opal codon, a unique codon, a rare codon, and a four-base codon.<br>
56.	A method of making the hlFN polypeptide of claim 3, the method comprising contacting an isolated hlFN polypeptide comprising a non-naturally encoded amino acid with a linker, polymer, or biologically active molecule comprising a moiety that reacts with the non-naturally encoded amino acid.<br>
57.	The method of claim 56, wherein the polymer comprises a moiety selected from a group consisting of a water soluble polymer and a poly(ethylene glycol).<br>
58.	The method of claim 56, wherein the non-naturally encoded amino acid comprises a carbonyl group, an aminooxy group, a hydrazide group, a hydrazine group, a semicarbazide group, an azide group, or an alkyne group.<br>
59.	The method of claim 56, wherein the non-naturally encoded amino acid comprises a carbonyl moiety and the linlcer, polymer, or biologically active molecule comprises an aminooxy, a hydrazine, a hydrazide or a semicarbazide moiety.<br>
60.	The method of claim 59, wherein the aminooxy, hydrazine, hydrazide or semicarbazide moiety is linked to the linker, polymer, or biologically active molecule through an amide linkage.<br>
61.	The method of claim 56, wherein the non-naturally encoded amino acid comprises an alkyne moiety and the linlcer, polymer, or biologically active molecule comprises an azide moiety.<br>
62.	The method of claim 56, wherein the non-naturally encoded amino acid comprises an azide moiety and the linlcer, polymer, or biologically active molecule comprises an allcyne moiety.<br>
63.	The method of claim 58, wherein the azide or alkyne moiety is linked to a linker, polymer, or biologically active molecule through an amide linkage,<br>
64.	The method of claim 57, wherein the poly(ethylene glycol) moiety has an average molecular weight of between about 0.1 kDa and about 100 kDa.<br><br>
65.	The method of claim 57, wherein the poly(ethylene glycol) moiety is a branched or multiarmed polymer.<br>
66.	A composition comprising the hIFN polypeptide of claim 1 and a pharmaceutically acceptable carrier.<br>
67.	The composition of claim 66^ wherein the non-naturally encoded amino acid is linked to a water soluble polymer.<br>
68.	A method of treating a patient having a disorder modulated by hlFN comprising administering to the patient a therapeutically-effective amount of the composition of claim 66.<br>
69.	A cell comprising the nucleic acid of claim 54.<br>
70.	The cell of claim 69, wherein the cell comprises an orthogonal tRNA synthetase or an orthogonal tRNA.<br>
71.	A method of making a hlFN polypeptide comprising a non-naturally encoded amino acid, the method comprising, culturing cells comprising a polynucleotide or polynucleotides encoding a hlFN polypeptide and comprising a selector codon, an orthogonal RNA synthetase and an orthogonal tRNA under conditions to permit expression of the hlFN polypeptide comprising a non-naturally encoded amino acid; and purifying the hlFN polypeptide.<br>
72.	A method of increasing serum half-life or circulation time of a hlFN polypeptide, the method comprising substituting one or more non-naturally encoded amino acids for any one or more naturally occurring amino acids in the hlFN polypeptide.<br>
73.	The hlFN polypeptide of claim 1 wherein said polypeptide is chemically synthesized.<br>
74.	The hlFN polypeptide of claim 1, wherein the polypeptide is ribosomally synthesized.<br>
75.	A hlFN polypeptide comprising a non-naturally encoded amino acid , wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of 23, 24, 25, 26, 27, 28, 30, 31, 32, 33,128,129, 131, 132, 133, 134,135, 136, 137,<br><br>
158,159,160,161,162,163,164,165 of SEQ ID NO: 2 or the corresponding amino acids in SEQ ID NO: 1 or 3.<br>
76.	The hlFN polypeptide of claim 75, wherein the non-naturally encoded amino acid comprises a carbonyl group, an aminooxy group, a hydrazide group, a hydrazine group, a semicarbazide group, an azide group, or an alkyne group.<br>
77.	The hEFN polypeptide of claim 75, wherein the non-naturally encoded amino acid is coupled to a water soluble polymer.<br>
78.	The hlFN polj'peptide of claim 75, wherein the water soluble polymer is a poly(ethylene glycol) moiety having a molecular weight of between about OA kDa and about 100 kDa.<br>
79.	The hlFN polypeptide of claim 78, wherein the poly(ethylene glycol) moiety is a branched or multiarmed polymer.<br>
80.	The hlFN polypeptide of claim 79, wherein the poly(ethylene glycol) moiety has a molecular weight of between about 1 kDa and about 100 kDa.<br>
81;	A composition comprising the hlFN polypeptide of claim 75 and a<br>
pharmaceutically acceptable carrier.<br>
82.	A hlFN polypeptide comprising one or more amino acid substitution, addition or deletion that increases the expression of the hlFN polypeptide in a recombinant host cell.<br>
83.	A hlFN polypeptide comprising a water soluble polymer linked by a covalent bond to the hlFN polypeptide at a single amino acid.<br>
84.	The hlFN polypeptide of claim 83, wherein the water soluble polymer comprises a poly(ethylene glycol) moiety.<br>
85.	The hlFN polypeptide of claim 83, wherein the amino acid covalently linlced to the water soluble polymer is a non-naturally encoded amino acid.<br>
86.	The hlFN polypeptide of claim 1 wherein said non-naturally encoded amino acid is linked to a polyethylene elvcol) molecule.<br><br>
87.	The hlFN polypeptide of clahn 1 comprising at least one linker,  polymer, or biologically active molecule, wherein said linker, polymer, or biologically active molecule is attached to the polypeptide through a functional group of a non-naturally encoded amino acid ribosomally incorporated into the polypeptide.<br>
88.	The hlFN polypeptide of claim 87, wherein said hlFN polypeptide is monoPEGylated.<br>
89.	The hlFN polypeptide of claim 1 comprising a linker, polymer or biologically active molecule that is attached to one or more non-naturally encoded amino acid wherein said non-naturally encoded amino acid is ribosomally incorporated into the polypeptide at preselected sites.<br>
90.	The hlFN polypeptide of claim 89, wherein the hlFN polypeptide comprises one said linker, polymer, or biologically active molecule.<br>
91.	The hlFN polypeptide of claim 1, wherein said hlFN polypeptide comprises one or more post-translational modification, linker, polymer, biologically active molecule, water soluble polymer, bifimctional polymer, bifunctional linker, additional hlFN polypeptide, second polypeptide, or poly(ethylene glycol) moiety, and wherein said hlFN polypeptide is selected from a group consisting of consensus IFN, IFNα, IFNp, IFNε, IFNγ, IFNω, IFNT, IFNα-la, IFNα-lb, IFNα-2a, IFNα-2b, IFNp-la, IFNβ-lb, and IFNy-la.<br>
92.	The hlFN polypeptide of claim 1, wherein said hlFN polypeptide comprises one or more amino acid substitution, addition, or deletion that increases affinity of the hlFN polypeptide for a hEFN receptor, increases the stability or solubility of the hlFN polypeptide, increases the expression of the hlFN polypeptide in a recombinant host cell or synthesized in vitro, or increases protease resistance of the hlFN polypeptide, and wherein said hlFN polypeptide is selected from a group consisting of consensus IFN, IFNα, IFNp, EFNe, IFN7, IFNco, IFNT, IFNα-la, IFNα-lb, IFNα-2a, IFNα-2b, IFNp-la, IFNP-lb, and IFNy-la.<br>
93.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition, or deletion that modulates immunogenicity of the hlFN polypeptide.<br><br>
94.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition, or deletion that modulates serum half-life or circulation time of the hlFN polypeptide.<br>
95.	A method of modulating immunogenicity of a hlFN polypeptide, the method comprising substituting one or more non-naturally encoded amino acids for any one or more naturally occurring amino acids in the hlFN polypeptide.<br>
96.	The hlFN polypeptide of claim 1 wherein the non-naturally encoded amino acid is substituted at a position selected from the group consisting of residues: 31, 134, 34, 38, 129, 36, 122,37, 121,41, 125, 124, 149, 117,39, 118, 120, 107,108,106,100, 111,113, 114,41,45, 46, 48, 49, 61, 64, 65, 101, 103, 102, 110, 117, 120, 121, 149, 96, 6, 9, 12, 13, 16, 68, 70, 109, 159, 161, 156, 160, 162, 24, 27, 78, 83, 85, 87, 89, 164, or any combination thereof (SEQ ID NO: 2).<br>
97'.	The hlFN polypeptide of claim 1 wherein the non-natunilly encoded amino acid<br>
is substituted at position 107 (SEQ ID NO: 2).<br>
98.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition, or deletion that modulates interaction with opioid receptors.<br>
99.	The hlFN polypeptide of claim 98, wherein said opioid receptor is the µ. opioid receptor.<br>
100.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition, or deletion that modulates 5-HT brain neurotransmission.<br>
101.	The hlFN polypeptide of claim 1, wherein the hlFN polypeptide comprises one or more amino acid substitution, addition, or deletion that modulates induction of indoleamine 2,3-dioxygenase.<br>
102.	The hlFN polypeptide of claim 1 or 96, wherein said hlFN has antiviral activity and reduced non-antiviral activity and side effects compared with hlFN.<br><br>
103.	The hlFN polypeptide of claim 1 further comprising one or more amino acid<br>
substitutions selected from the group consisting of GlOE, M16R, R13E, T79R, K83Q, K83S, YSSL, T86S, E87S, Q90R, Q9IE, N93Q, D94V, E96K, R120K, K121T, Q124R, R125G, L128R, R149Y, R149E, R149S, D77-D94 OF SEQ ID NO: 2 is replaced with HERALDQLLSSLWRELQV, and V105-D114 of SEQ ID NO: 2 is replaced within GQSAPLP.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">024-chenp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtYXNzaWduZW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">024-chenp-2008-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">024-chenp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">024-chenp-2008-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0LWNoZW5wLTIwMDgtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">024-chenp-2008-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIEFNRU5ERUQgQ0xBSU1TICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   AMENDED CLAIMS  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIEFNRU5ERUQgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMjYtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   AMENDED PAGES OF SPECIFICATION  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIEZPUk0tMSAgMjYtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   FORM-1  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIEZPUk0tMyAgMjYtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   FORM-3  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIE9USEVSIFBBVEVOVCBET0NVTUVOVCAgMjYtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   OTHER PATENT DOCUMENT  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIE9USEVSIFBBVEVOVCBET0NVTUVOVCAxICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   OTHER PATENT DOCUMENT 1  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgIE9USEVSUyAgMjYtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">24-CHENP-2008   OTHERS  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgMjYtMDMtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">24-CHENP-2008  CORRESPONDENCE OTHERS   26-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtQ0hFTlAtMjAwOCAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAyNi0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">24-CHENP-2008  EXAMINATION REPORT REPLY RECEIVED  26-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtY2hlbnAtMjAwOCBjb3JyZXNwb25kYW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">24-chenp-2008 correspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQtY2hlbnAtMjAwOCBmb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">24-chenp-2008 form-18.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258467-method-of-modeling-the-time-gradient-of-the-state-of-a-steel-volume-by-means-of-a-computer-and-corresponding-objects.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258469-polymerisation-process.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258468</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>24/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>03/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Jan-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AMBRX, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>10975 NORTH TORREY PINES ROAD, SUITE 100, LA JOLLA, CA 92037, USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HAYS, PUTNAM, ANNA-MARIA, A</td>
											<td>11522 CESPED DRIVE, SAN DIEGO, CA 92124, USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KIMMEL, BRUCE, E</td>
											<td>13236 JACARTE COURT, SAN DIEGO, CA 92130, USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHO, HO, SUNG</td>
											<td>5225 FIORE TERRACE, APT, 107, SAN DIEGO, CA 92122, USA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SIM-BEE-CHENG</td>
											<td>7564 CHARMANT DRIVE, 1827 , SAN DIEGO, CA 92064, USA.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LITZINGER, DAVID, C</td>
											<td>13976 ARBOLITOS DRIVE, POWAY, CA 92064, USA.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MARIANI, ROBERTO</td>
											<td>350 W. ASH STREET, 1208, SAN DIEGO, CA 92101, USA.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>KRAYNOV, VADIM</td>
											<td>5457 WHITE OAK LANE, SAN DIEGO, CA 92130, USA.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>KNUDSEN, NICK</td>
											<td>10709 MATINAL CIRCLE, SAN DIEGO, CA 92127, USA.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>DANIEL, THOMAS, O</td>
											<td>2201 VIA PRAVIA, LA JOLLA, CALIFORNIA 92037, USA.</td>
										</tr>
										<tr>
											<td>10</td>
											<td>KODER, ALAN</td>
											<td>9845 PASEO MONTALBAN, SAN DIEGO, CA 92129, USA.</td>
										</tr>
										<tr>
											<td>11</td>
											<td>BUSSELL, STUART</td>
											<td>2385 AVENIDA HELECHO, CARLSBAD, CA 92009, USA.</td>
										</tr>
										<tr>
											<td>12</td>
											<td>MIAO, ZHENWEI</td>
											<td>15887 SUNNYFIELD PLACE, SAN DIEGO, CA 92127, USA.</td>
										</tr>
										<tr>
											<td>13</td>
											<td>MORROW, THERESA</td>
											<td>2760 CAMINTO SAN MARINO, DEL MAR, CA 92014, USA.</td>
										</tr>
										<tr>
											<td>14</td>
											<td>LIU, JUNJIE</td>
											<td>5004 FIELD STREET, SAN DIEGO, CA 92110, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 38/21</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/021738</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-06-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/687,173</td>
									<td>2005-06-03</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/753,375</td>
									<td>2005-12-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258468-improved-human-interferon-molecules by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:29:12 GMT -->
</html>
