<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258566-a-polypeptide-selected-from-the-group-consisting by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:09:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258566:&quot;A POLYPEPTIDE SELECTED FROM THE GROUP CONSISTING&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A POLYPEPTIDE SELECTED FROM THE GROUP CONSISTING&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to enzymes and processes. In particular there is described a host cell transformed or transfected with a nucleic acid encoding a bacterial phytase enzyme and variants thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ENZYMES<br>
The present invention relates to phytases, nucleotide sequences for same, methods of production of phytases and their use.<br>
FIELD OF THE INVENTION<br>
The present invention relates to the field of enzymes for additives to feedstuffs. More specifically, the present invention relates to phytases which can be used for enhancing phosphate digestion in foods and animal feeds.<br>
TECHNICAL BACKGROUND AND PRIOR ART<br>
Phytate is the major storage form of phosphorus in cereals and legumes. However, monogastric animals such as pig, poultry and fish are not able to metabolise or absorb phytate (or phytic acid) and therefore it is excreted, leading to phosphorous pollution in areas of intense livestock production. Moreover, phytic acid also acts as an antinutritional agent in monogastric animals by chelating metal agents such as calcium, copper and zinc.<br>
In order to provide sufficient phosphates for growth and health of these animals, inorganic phosphate is added to their diets. Such addition can be costly and further increases pollution problems.<br>
Through the action of phytase, phytate is generally hydrolysed by to give lower inositol-phosphates and inorganic phosphate. Phytases are useful as additives to animal feeds where they improve the availability of organic phosphorus to the animal and decrease phosphate pollution of the environment (Wodzinski RJ, Ullah AH. Adv Appl Microbiol. 42,263-302 (1996)).<br>
A number of phytases of fungal (Wyss M. et al. Appl. Environ. Microbiol. 65 (2), 367-373 (1999); Berka R.M. et al. Appl. Environ. Microbiol. 64 (11), 4423-4427 (1998); Lassen S. et al Appl. Environ. Microbiol. 67 (10), 4701-4707 (2001)) and<br><br>
bacterial (Greiner R. et al Arch. Biochem. Biophys. 303 (1), 107-113 (1993); Kerovuo et al. Appl. Environ. Microbiol. 64 (6), 2079-2085 (1998); Kim H.W. et al Biotechnol. Lett. 25, 1231-1234 (2003); Greiner R. et al. Arch. Biochem. Biophys. 341 (2), 201-206 (1997); Yoon S.J. et al. Enzyme and microbial technol. 18, 449-454 (1996); Zinin N.V. et al FEMS Microbiol. Lett. 236, 283-290 (2004))) origin have been described in the literature.<br>
However, to date, none of these phytases display the properties required for effective use as an animal feed supplement. In particular, fungal phytases tend to be proteolytically unstable (Igbasan F.A. et al. Arch. Anim. Nutr. 53,353-373 (2000)) and therefore susceptible to degradation, while most bacterial phytases have a narrow substrate specificity for phytate alone and poorly degrade inositol phosphates of intermediate degrees of phosphorylation (Greiner R. et al., Arch. Biochem. Biophys. 303 (1), 107-113 (1993); Kerovuo J et al. Biochem. J. 352, 623-628 (2000)).<br>
Accordingly, there is a need for improved phytases. SUMMARY OF THE INVENTION<br>
In a broad aspect, the present invention relates to phytases derived from a bacterium and modified forms thereof. In particular the invention relates to phytases derived from the bacterium, Buttiauxella sp. and variant/modified forms thereof selected and/or engineered for improved characteristics compared to the wild-type (parent) enzyme.<br>
The present invention is advantageous as it provides for novel phytases that have properties making them particularly useful and efficient as feed enzymes. In particular the invention relates to isolated and/or purified novel phytase polypeptides as described herein, or a functional fragment, or variants or modified forms thereof, or modified form thereof. The invention also provides the nucleic acid sequence encoding said phytase.<br>
To be efficient as an enzyme additive to food or animal feed, the phytase has to combine a number of different properties. In order to be able to degrade phytic acid in<br><br>
the acidic environment of an animal's stomach it has to be active at low pH, preferably over a broad range of pH values. In addition, it has to have high specific activity and preferably high thermostability to enable the protein to withstand high temperatures commonly used in preparation of feedstuffs such as feed pellets.<br>
It is also important that the enzyme has broad substrate specificity allowing it to hydrolyse not only phytate but also intermediate products of phytate degradation such as inositol pentaphosphates, tetraphosphates and triphosphates. Studies on phytate degradation in pigs show that these inositol oligophosphates otherwise remain largely insoluble in the small and large intestine and thus inaccessible to alkaline phosphatases produced by the animal and gut microflora (Schlemmer U. et al. Arch. Anim. Nutr. 55, 255-280 (2001)). Variations in substrate specificity profiles of different enzymes have been identified. For example, inositol-triphosphates generated by the phytase from B. subtilis are essentially resistant to further hydrolysis by this enzyme [Kerovuo J. et al. Biochem J. 352,623-628 (2000)].<br>
In another aspect of the invention there is provided a plasmid or a vector system, or a transformed or a transgenic organism comprising a novel phytase as described herein or a modified form thereof.<br>
In another aspect the present invention relates to transgenic organisms modified to express a novel phytase as described herein or a modified form thereof, and therefore being capable of producing a phytase. The present invention further provides means and methods for the biotechnological production of phytases and their use as feed supplements.<br>
Aspects of the present invention are presented in the claims and in the following commentary.<br>
For ease of reference, these and further aspects of the present invention are now discussed under appropriate section headings. However, the teachings under each section are not necessarily limited to each particular section.<br><br>
As used with reference to the present invention, the terms "produce", "producing", "produced", "produceable", "production" are synonymous with the respective terms "prepare", "preparing", "prepared", "preparation", "generated", "generation" and "preparable".<br>
As used with reference to the present invention, the terms "expression", "expresses", "expressed" and "expressable" are synonymous with the respective terms "transcription", "transcribes", "transcribed" and "transcribable".<br>
As used with reference to the present invention, the terms "transformation" and "transfection" refer to a method of introducing nucleic acid sequences into hosts, host cells, tissues or organs.<br>
Other aspects concerning the nucleotide sequences which can be used in the present invention include: a construct comprising the sequences of the present invention; a vector comprising the sequences for use in the present invention; a plasmid comprising the sequences for use in the present invention; a transformed cell comprising the sequences for use in the present invention; a transformed tissue comprising the sequences for use in the present invention; a transformed organ comprising the sequences for use in the present invention; a transformed host comprising the sequences for use in the present invention; a transformed organism comprising the sequences for use in the present invention. The present invention also encompasses methods of expressing the nucleotide sequence for use in the present invention using the same, such as expression in a host cell; including methods for transferring same. The present invention further encompasses methods of isolating the nucleotide sequence, such as isolating from a host cell.<br>
Other aspects concerning the amino acid sequences for use in the present invention include: a construct encoding the amino acid sequences for use in the present invention; a vector encoding the amino acid sequences for use in the present invention; a plasmid encoding the amino acid sequences for use in the present invention; a transformed cell expressing the amino acid sequences for use in the present invention; a transformed tissue expressing the amino acid sequences for use in the present invention; a transformed organ expressing the amino acid sequences for use in the present invention;<br><br>
a transformed host expressing the amino acid sequences for use in the present invention; a transformed organism expressing the amino acid sequences for use in the present invention. The present invention also encompasses methods of purifying the amino acid sequence for use in the present invention using the same, such as expression in a host cell; including methods of transferring same, and then purifying said sequence.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows SDS PAGE analysis of the recombinant phytase from Buttiauxella PI-29 purified by DEAE-Sepharose chromatography. The figure presents the scanning trace of a digital photographic image of the lane containing a sample of the Buttiauxella phytase.<br>
Figure 2 shows the pH profile of the phytase from Buttiauxella PI-29<br>
Figure 3 shows substrate specificity of the purified recombinant phytase from Buttiauxella PI-29 with inositol phosphate fractions of different degree of phosphorylation and model substrates. Abbreviations: IP6 - phytic acid, IPS, IP4 and IPS - mixtures of isomeric inositol penta-, tetra- and triphosphates respectively. Fru P2 - fructose 1,6-diphosphate, Fru PI - fructose 6-phosphate.<br>
SEQ ID NO: 1 lists the sequence obtained for identification of the bacterial strain.<br>
SEQ ID NO: 2 lists the polynucleotide sequence comprising the phytase gene from Buttiauxella?l-29.<br>
SEQ ID NO: 3 lists the amino acid sequence of the phytase gene from Buttiauxella Pl-29.<br>
DETAILED DISCLOSURE OF INVENTION<br>
The present invention features an enzyme comprising the amino acid sequence corresponding to Buttiauxella sp. phytase or a modified form, a homologue, a variant, a functional equivalent or an effective fragment thereof.<br><br>
The term "phytase" means a protein or polypeptide which is capable of catalysing the hydrolysis of esters of phosphoric acid, including phytate and releasing inorganic phosphate. Some phytases in addition to phytate, are capable of hydrolysing at least some of the inositol-phosphates of intermediate degrees of phosphorylation.<br>
The term "corresponding to Buttiauxella sp. phytase" means that the enzyme need not have been obtained from a source of Buttiauxella sp. Instead, the enzyme preferably has to have the same functional characteristics or sequence as that of Buttiauxella sp. phytase. For example, the Buttiauxella sp. Phytase may be a variant derived from a Buttiauxella sp., but which is not naturally present in Buttiauxella species.<br>
The Buttiauxella spp. include, Buttiauxella agrestis, Buttiauxella brennerae, Buttiauxella ferragutiae, Buttiauxella gaviniae, Buttiauxella izardii, Buttiauxella noackiae, Buttiauxella warmboldiae. Stains of the Buttiauxella species are available from DSMZ, the German National Resource Centre for Biological Material. Phytases are preferably identified from Buttiauxella spp by the methods described herein, for example hybridisation to SEQ ID No 2. Preferred Buttiauxella spp. strains for isolating polypeptides and or polynucleotides of the invention are listed in the examples.<br>
The terms "wild type phytase" or "wild type" in accordance with the invention describe a phytase enzyme with an amino acid sequence found in nature.<br>
The terms "phytase enzyme variant" "phytase variant" or "variant" in accordance with the invention describe a phytase enzyme with an amino acid sequence derived from the amino acid sequence of a parent phytase but differing by one or more amino acid substitutions, insertions, and/or deletions, which together are referred to as "mutations". It is envisaged that a phytase enzyme variant may also be a parent phytase enzyme for further rounds of methods of preparing phytase variants such as molecular evolution.<br>
The term "homologous polypeptide(s)", according to the present invention, described also as "homologues" herein, describe polypeptides, preferably phytases enzymes (i.e.<br><br>
"homologous phytases" or "homologous enzymes") with a sequence identity of more than 75% compared to a first polypeptides/phytases/enzymes amino acid sequence, preferably has at least 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% sequence homology.<br>
The term "functional equivalent thereof means that the enzyme has to have about the same functional characteristics as that of Buttiauxella sp. phytase. The term "modified form" or "variant" means that the enzyme has been modified from its original form but retains the same enzymatic functional characteristics as that of Buttiauxella sp. phytase. In particular, the terms "variant" or "modified form" encompass phytase enzymes with an amino acid sequence derived from the amino acid sequence of the parent/wild-type phytase and having one or more amino acid substitutions, insertions, and/or deletions, which together are referred to as mutations. Modified forms or variants may display altered enzyme characteristics compared to the parent enzyme. Preferably, modified forms or variants have one or more of the following enhanced phenotypes: increased thermostability and/or; an increased proteolytic (for example pepsin) stability and/or; an increased specific activity and/or; broader substrate specificity and/or; an activity over a broader pH range. The term "functional" or "effective" fragment means a fragment or portion of the Buttiauxella sp. phytase that retains about the same enzymatic function or effect.<br>
Preferably the phytase enzyme of this aspect of the present invention has the same sequence or a sequence that is at least 75% identical (homologous) to that of Buttiauxella sp. phytase.<br>
Suitably, the enzyme comprises the amino acid sequence as shown in SEQ ID NO: 3 or a sequence having at least 75% identity (homology) thereto or a functional fragment thereof. In a preferred embodiment, the invention provides an isolated and/or purified polypeptide having the amino acid sequence as set out in SEQ ID NO: 3 or a sequence having at least 75% identity (homology) thereto or an effective fragment thereof. When referring to SEQ ID No 3 and polypeptides comprising SEQ ID No 3, it is envisaged that this also refers to polypeptides that are co- or post-translationally processed during expression, for example by signal peptide cleavage. Post-translational cleavage may also occur at the C-terminal. Therefore in a preferred<br><br>
embodiment the effective fragment thereof (also referred to as functional fragment thereof) is the mature polypeptide produced by the native host or a suitable appropriate expression host.<br>
In another embodiment, the phytase is characterised in that it is derived from Buttiauxella sp. strain PI-29 deposited under accession number NCIMB 41248.<br>
In a preferred embodiment, the invention relates to a phytase in accordance with any embodiment of the first aspect of the invention that comprises one or more mutations at the following positions (numbering according to the numbering in SEQ ID No. 3):<br>
59, 70, 122, 125, 167, 193, 197, 204, 209, 211, 221, 223, 225, 240, 242, 244, 268, 281,289,294,303,336,351.<br>
These positions are characterized in that mutagenesis of the enzyme at these positions lead to an improvement in the desired enzyme characteristics.<br>
The following substitutions may be preferred variants:<br>
K 59 A, C, D, E, F, G, H, I, L, M, N, P, Q, R, S, T, V, W,or Y T 70 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W,or Y A 122 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W,or Y D 125 A, C, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y T 167 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W, or Y H 193 A, C, D, E, F, G, I, K, L, M, N, P, Q, R, S, T, V, W, or Y . F 197 A, C, D, E, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y T 204 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W, or Y T 209 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W, or Y A 211 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y G 221 A, C, D, E, F, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y 1223 A, C, D, E, F, G, H, K, L, M, N, P, Q, R, S, T, V, W, or Y S 225 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, T, V, W, or Y K 240 A, C, D, E, F, G, H, I, L, M, N, P, Q, R, S, T, V, W, or Y A 242 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y<br><br>
D244A,C3E3F,G3H3I,K3L3M3N,P,Q,R,S3T,V3W3orY A 268 C, D, E, F, G, H, I, K, L, M3 N, P, Q, R, S3 T, V, W3 or Y L 281 A3 C, D, E, F, G, H, I, K, M3 N, P, Q, R, S, T, V, W, or Y Q 289 A, C, D3 E, F3 G, H, I, K, L, M, N, P., R, S, T, V, W3 or Y A 294 C, D5 E3 F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y N 303 A, C, D3 E, F, G, H, I, K, L, M, P, Q, R, S3 T, V, W3 or Y 1336 A, C, D, E, F, G, H, K, L, M, N, P, Q, R, S, T, V, W, or Y N 351. A, C, D, E, F, G, H, I, K, L3 M, P, Q, R, S, T, V, W, or Y<br>
By "conservative mutations" is meant mutations to amino acid residues that are conservative in terms of the amino acid characteristics compared to the amino acid residue indicated. Amino acid characteristics include the size of the residue, hydrophobicity, polarity, charge, pK-value, and other amino acid characteristics known in the art. Preferred conservative mutations are listed below as conservative substitutions.<br>
In a particularly preferred embodiment, the mutations are at one or more of the following positions: K59; T167; K240; T167; K240; D244; Q289; T209 and F197<br>
Preferred mutations at these specific positions are listed above, more preferred mutations include: K59E; T167V; K240T; T167I; K240E; D244C; Q289Y; T209K andF197S.<br>
In a further preferred embodiment, there is provided a phytase comprising a combination of mutations selected from the group consisting of:<br>
D125E/H193R;<br>
A294E/N303K;<br>
T167I/K240T;<br>
D223E/K240E/N351D;<br>
Tl 67I/K24017A294E/N303K;<br>
Tl 67I/K240E/A242S/A294E/N303K;<br>
A122T/T167I/F197S/T209K/A211P/K240E/A294E/N303K;<br>
Al 22T/T167I/F197S/T209K/A211P/K240E/A242S/S281L/Q289 Y/A294E/N303K;<br><br>
A122T/D125E/H193R/F197S/T209K/A211P/S221N/G225A/K240E/A294E/N303K;<br>
D125E/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268 V/Q289H/A294E/<br>
N303K;<br>
Al 22T/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268V/Q289H/A294E/<br>
N303K/I336F and<br>
N70 Y/D125E/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268 V/Q289H/<br>
A294E/N303K.<br>
Accordingly, a preferred phytase in accordance with the present invention is a variant comprising the amino acid sequence listed as SEQ ID NO: 3 or an effective fragment thereof, (or homologue thereof, preferably the phytase is characterised in that it is derived from Buttiauxella sp. strain PI-29 deposited under accession number NCIMB 41248, or homologue thereof), except having one or more of the amino acid mutations listed above or one of the combinations of mutations listed above.<br>
In these embodiments, the nomenclature indicates a phytase comprising the amino acid sequence set out in SEQ ID NO: 3 with the mutations Indicated by reference to the positions of the amino acids in SEQ ID NO: 3. The nomenclature is described in more detail below.<br>
Suitably these variants show improved characteristics with respect to any one of the following: temperature stability, pH range, pepsin stability, specific activity, substrate specificity, and broader substrate specificity. Suitable methods for determining these characteristics are disclosed herein.<br>
In particular, the improvements in phytase characteristics are directed to the enzyme stability under food and feed processing conditions, to the enzyme stability during stomach transit, and to the enzyme activity and stability in human or animal stomach and/or intestinal tract making the improved variants particularly suitable for use as feed supplements. Thus, such improvements comprise among other parameters the increase in stability at elevated temperatures, preferably at temperatures above 65 °C, the increase in stability against proteolytic digestion, preferably protease of the digestive tract such as pepsin, the increase in catalytic activity at low pH, preferably<br><br>
catalytic activity below pH 5.5, and the general efficiency of releasing phosphate groups from phytate, and preferably in addition inositol phosphates.<br>
Nomenclature<br>
In the present description and claims the conventional one-letter and three-letter codes for amino acid residues are used. For ease of reference, mutations in enzyme variants are described by use of the following nomenclature: amino acid residue in the parent enzyme; position; substituted amino acid residue(s). According to this nomenclature, the substitution of, for instance, an alanine residue for a glycine residue at position 20 is indicated as Ala20Gly or A20G. The deletion of alanine in the same position is shown as Ala20* or A20*. The insertion of an additional amino acid residue (e.g. a glycine) is indicated as Ala20AlaGly or A20AG. The deletion of a consecutive stretch of amino acid residues (e.g. between alanine at position 20 and glycine at position 21) is indicated as A(Ala20-Gly21) or A(A20-G21). When a parent enzyme sequence contains a deletion in comparison to the enzyme sequence used for numbering an insertion in such a position (e.g. an alanine in the deleted position 20) is indicated as *20Ala or *20A. Multiple mutations are separated by a plus sign or a slash. For example, two mutations in positions 20 and 21 substituting alanine and glutamic acid for glycine and serine, respectively, are indicated as A20G+E21S or A20G/E21S. When an amino acid residue at a given position is substituted with two or more alternative amino acid residues these residues are separated by a comma or a slash. For example, substitution of alanine at position 30 with either glycine or glutamic acid is indicated as A20G,E or A20G/E, or A20G, A20E. When a position suitable for modification is identified herein without any specific modification being suggested, it is to be understood that any amino acid residue may be substituted for the amino acid residue present in the position. Thus, for instance, when a modification of an alanine in position 20 is mentioned but not specified, it is to be understood that the alanine may be deleted or substituted for any other amino acid residue (i.e. any one of R,N,D,C,Q,E,G,H,I,L,K,MSF5P,S,T,W,Y,V).<br>
Suitably, the phytase or functional equivalent of the present invention is characterised in that said phytase has a specific activity of at least 100 U/mg or 200U/mg, preferably at least, 300 U/mg wherein said specific activity is determined by<br><br>
incubating said phytase in a solution containing 2mM phytate, O.SmM CaCk in 200mM sodium acetate buffer at pH 3.5 as detailed in Example 1. The phytase of the present invention or functional equivalent thereof may also suitably be characterised in that said phytase has an activity maxima around pH 4 - 4.5 wherein said activity is determined by incubating said phytase in a solution containing 2mM phytate, O.SmM CaCla in 200mM sodium acetate buffer. The phytase of the present invention may also be suitably characterised in that said phytase has 40% or more of a maximum activity observed at pH 2.5 and 5.5 wherein glycine hydrochloride buffer is used to determine activity at pH 2.5.<br>
Suitably, in one embodiment, the phytase or functional equivalent of the present invention is characterised hi that said phytase has a specific activity of 330 U/mg or higher wherein said specific activity is determined by incubating said phytase in a solution containing 2mM phytate, O.SmM CaCla in 200mM sodium acetate buffer at pH 3.5. In another embodiment, the phytase of the present invention or functional equivalent thereof may also suitably be characterised in that said phytase has two activity maxima around pH 3 and pH 4 - 4.5 wherein said activity is determined by incubating said phytase in a solution containing 2mM phytate, O.SmM CaCk in 200mM sodium acetate buffer.<br>
In another aspect, the invention provides an isolated and/or purified nucleic acid molecule or nucleotide sequence coding for the enzyme comprising the amino acid sequence corresponding to Buttiaiaella sp. phytase, or a homologue thereof. Suitably said isolated and/or purified nucleic acid molecule encodes a polypeptide comprising the amino acid sequence as shown in SEQ ID NO: 3 or a sequence having at least 75% identity (homology) thereto or an effective fragment thereof. In one embodiment, the nucleic acid molecule encodes a polypeptide comprising SEQ ID NO:3 and including mutations at the preferred positions listed herein or any of the specific mutations or combinations of mutations listed herein. In another embodiment, the invention provides an isolated and/or purified nucleic acid molecule comprising a nucleotide sequence that is the same as, or is complementary to, or contains any suitable codon substitutions for any of those of SEQ ID NO: 2 or comprises a sequence which has at least 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% sequence homology with SEQ ID NO: 2.<br><br>
In a yet further aspect, the invention relates to a nucleotide sequence and to the use of a nucleotide sequence shown as:<br>
(a)	the nucleotide sequence presented as SEQ ID No.2,<br>
(b)	a nucleotide sequence that is a variant, homologue, derivative or fragment of the<br>
nucleotide sequence presented as SEQ ID No. 2;<br>
(c)	a nucleotide sequence that is the complement of the nucleotide sequence set out<br>
in SEQ ID No. 2;<br>
(d)	a nucleotide sequence that is the complement of a variant, homologue, derivative<br>
or fragment of the nucleotide sequence presented as SEQ ID No 2;<br>
(e)	a nucleotide sequence that is capable of hybridising to the nucleotide sequence<br>
set out in SEQ ID No. 2;<br>
(f)	a nucleotide sequence that is capable of hybridising to a variant, homologue,<br>
derivative or fragment of the nucleotide sequence presented as SEQ ID No. 2;<br>
(g)	a nucleotide sequence that is the complement of a nucleotide sequence that is<br>
capable of hybridising to the nucleotide sequence set out in SEQ ID No. 2;<br>
(h)      a nucleotide sequence that is the complement of a nucleotide sequence that is<br>
capable of hybridising to a variant, homologue, derivative or fragment of the nucleotide<br>
sequence presented as SEQ ID No. 2;<br>
(i)       a nucleotide sequence that is capable of hybridising to the complement of the<br>
nucleotide sequence set out in SEQ ID No.2;<br>
(j)       a nucleotide sequence that is capable of hybridising to the complement of a<br>
variant, homologue, derivative or fragment of the nucleotide sequence presented as SEQ<br>
ID No. 2.<br>
A nucleotide is considered to hybridise to one of the above nucleotides (e), (f), (g), (h), (i) or (j) if it is capable of hybridising under conditions of medium stringency, more preferably high stringency, even more preferably under very high stringency conditions.<br>
To prepare a hybridisation blot standard molecular biology protocols for blotting may be used (e.g. Southern blotting for DNA hybridisations. The amount of target DNA depends on the relative abundance of the target sequence. If a pure target sequence is<br><br>
to be used, between 1 and Spicograms of DNA per kilobase of polynucleotides are preferred. Typically, the detection limit is about 0.5 pg DNA for a radioactive probe with specific activity of 109 dpm/mg. which is equivalent to a single-copy gene 500 bp in length in 3.3 mg genomic DNA of a complex genome (e.g. human). In practice one will use approx. 10 mg of genomic DNA - for example to screen for organisms, such as micro-organisms, which contain a phytase encoding polynucleoti.de of the invention. If the target DNA is bacterial or, for example, a plasmid one will have to dilute the DNA accordingly to avoid over exposure.  The target DNA is blotted, e.g. by dot blotting, or via blotting from an electrophoresis gel. Preferred conditions are described in 'Membrane Transfer and Detection Methods, Amersham International pic, UK .- PI/162/85/1).     Hybond N+ positively charged nylon membrane is preferably used (Amersham Life Science).     The probe is preferably prepared according to Pharmacia's 'Ready to Go DNA™ labelling kit to prepare a probe of &gt; 1X109 dpm/microgram. The probe is used in hybridisation buffer at a concentration of 1x106 dpm per millilitre hybridisation buffer. Blots are preferably pre hybridised in hybridisation buffer (6 x SSC, 5 x denhardt's solution, and 0.5% SDS, and denatured salmon sperm DNA to 100 mg/ml. buffer) for an hour at 65'C, followed by hybridisation in hybridisation buffer containing the denatured labelled probe with shaking for 12 hours at 65'C.   The blot(s) are then washed with a suitable volume wash buffer (typically 50ml) hi 2xSSC, 0.1% SDS for 30 minutes at 65'C, followed by a second wash in a suitable volume wash buffer (typically 50ml) in either the same wash buffer (2xSSC, 0.1% SDS) for medium stringency washing, or 0.1% x SSC, 0.1% SDS for 10 minutes at 65'C (high stringency stringency), the second wash can be repeated at 70'C for very high stringency washing.<br>
The nucleotide sequence of the present invention may comprise sequences that encode for SEQ ID No. 3 or ah effective fragment thereof or a variant, modified form, homologue or derivative thereof.<br>
In particular, the invention provides a plasmid or vector system comprising a phytase as described herein or a homologue or derivative thereof. Preferably, the plasmid or vector system comprises a nucleic acid sequence as set out in SEQ ID No: 2 or a sequence that is at least 75% homologous thereto or an effective fragment thereof. Suitably the plasmid or vector system is an expression vector for the expression of<br><br>
any of the enzymes encoded by a nucleic acid sequence as set out in any of SEQ ID No: 2 or a sequence that is at least 75% homologous (identical) thereto in a microorganism. Suitable expression vectors are described herein. In addition, the invention provides a plasmid or vector system for expression of any of the modified enzymes or variants or functional fragments described herein. Suitable expression vectors are described herein.<br>
Phytase variants<br>
The present invention is drawn to the improvement of the characteristics of a parent phytase by modification of one or more amino acid residues of the amino acid sequence of the parent phytase.<br>
Phytase enzymes used as parent enzymes according to the present invention include wild-type phytases from bacteria, in particular, preferably phytases obtainable from or derived from Buttiauxella sp. having the amino acid sequence as given in Seq ID No. 3 or an effective fragment thereof, or an amino acid sequence with an identity to Seq ID No 3 of more than 75%, preferably of more than 80%, more preferably of more than 90%, even more preferably of more than 95%, 96%, 97%, 98% preferably of more than 99% (i.e. homologous polypeptide), or an effective fragment thereof.<br>
Improved phytase enzyme variants of the invention preferably have an identity of an identity to Seq ID No 3 or an effective fragment thereof of more than 75%, preferably of more than 80%, more preferably of more than 90%, more preferably of more than 95%, 96%, 97%, 98% preferably of more than 99%. However, it is also envisaged that variants may be heterologous, (i.e. not homologous) to SEQ ID No3. For example variants produced by recombination techniques such as exo-mediated recombinantion, or family shuffling, may result in variants, although prepared using the parent phytase according to the present invention, may have less than 75% homology.<br>
Sequence alignments as well as the determination of sequence identities may suitably be done by means of computer programs known in the art such as GAP provided in the GCG program package (Needleman, S. B. and Wunsch, C. D.,  <br>
Molecular Biology, 48, p. 443-453). GAP may be used with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1. Sequence alignments are used, for example, to determine corresponding positions in homologous polypeptides.<br>
Phytase enzymes were characterized after heterologous expression in one or more of the following expression hosts: Escherichia coll K12; Bacillus subtilis; Saccharomyces cerevisiae. Other expression hosts may be used.<br>
Improvements in phytase characteristics according to the present invention are directed to the use in food and feed processing as well as for the use as an additive to food and feed products. In particular, improvements are directed to the stability under food and feed processing conditions, to the stability during stomach transit, and to the activity and stability in human or animal stomach and/or intestinal tract. Such improvements comprise among other parameters the increase in stability at elevated temperatures, preferably at temperatures above 65 °C, the increase in stability against proteolytic digestion, preferably protease of the digestive tract, the increase in catalytic activity at low pH, preferably catalytic activity belowpH 5.5, and the general efficiency of releasing phosphate groups from phytate.<br>
The increase in stability at elevated temperatures is quantified by the inactivation temperature of the enzyme. The inactivation temperature is defined as the temperature at which the residual activity of a phytase enzyme after incubation for a certain duration and subsequent cooling to room temperature is 50% of the residual activity of the same phytase enzyme incubated for the same duration under the same conditions at room temperature. Thermostability differences are the differences in °C between the inactivation temperatures of two enzymes.<br>
The positions and/or regions to be mutated to obtain improved characteristics were found by analysis of the sequence and the structure of the wild type phytases as well as by mutagenesis of parent enzymes, in particular by introduction of mutations into the wild-type amino acid sequence as given in Seq ID No 3, and screening for enzyme variants with improved characteristics. Thereby, certain regions as well as positions<br><br>
within the parent enzymes have been identified to be significant for improving the characteristics of the phytase enzymes.<br>
Therefore, the invention relates to phytase variants with improved characteristics that, when compared to the parent phytases comprise mutations at one or more of the following positions (numbering according to the numbering in Seq ID No 3):<br>
59, 70, 122, 125, 167, 193, 197, 204, 209, 211, 221, 223, 225, 240, 242, 244, 268, 281,289,294,303,336,351.<br>
and/or at corresponding positions in a phytase homologous to the phytase as shown in the amino acid sequence of Seq ID No 3. These positions are characterized in that mutagenesis of these positions lead to an improvement in the enzymes characteristics.<br>
K59E; T167V; K240T; T167I; K240E; D244C; Q289Y; T209K and F197S. or conservative mutations at each position. Specific preferred combinations of mutations include:<br>
D125E/H193R;<br>
A294E/N303K;<br>
T167I/K240T;<br>
D223E/K240E/N351D;<br>
Tl 67I/K240T/A294E/N303K;<br>
Tl 67I/K240E/A242S/A294E/N303K;<br>
A122T/T167I/F197S/T209K/A211P/K240E/A294E/N303K;<br>
A122T/T167I/F197S/T209KyA211P/K240E/A242S/S281L/Q289Y/A294E/N303K;<br>
A122T/D125E/H193R/F197S/T209K/A211P/S221N/G225A/K240E/A294E/ N303K;<br>
Dl 25E/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268V/Q289H/A294E/<br>
N303K;<br>
A122T/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268V/Q289H/A294E/<br>
N303K/I336F and<br><br>
N70 Y/D125E/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268 V/Q289H/ A294E/N303K.<br>
Methods of preparing phytase variants<br>
In a broad embodiment the invention provides methods of preparing phytase enzyme variant(s).<br>
In a preferred embodiment the method of preparing a phytase enzyme variant comprises the following sequential steps:<br>
a)	Selecting at least one parent phytase enzyme, wherein the at least one parent<br>
phytase enzyme is selected from i) a polypeptide comprising the amino acid<br>
sequence corresponding to a Buttiauxella sp. phytase, or a modified form, a<br>
homologous polypeptide, a variant, a functional equivalent or an effective<br>
fragment, thereof, as herein described or ii) at least one phytase enzyme<br>
variant as herein described;<br>
b)	Generating at least one phytase variant by introducing at least one alteration of<br>
said parent phytase enzyme which is an insertion, a deletion or a substitution<br>
or combination thereof, of an amino acid residue in said parent phytase<br>
enzyme to obtain at least one phytase enzyme variant;<br>
c)	Screening for said at least one phytase enzyme variant to identify an improved<br>
phytase enzyme variant, which compared to the parent phytase enzyme has<br>
improved property/properties selected from:<br>
i.     higher thermal stability and/or ii.     specific activity and/or iii.     proteolytic stability<br>
d) Preparing said improved phytase enzyme variant, preferably to produce an isolated and/or purified phytase enzyme variant.<br>
In a preferred embodiment, during step b) a population of phytase enzyme variants is generated and in step c) at least a proportion of said population of phytase enzyme variants is screened.<br><br>
In a preferred embodiment step a) comprises subjecting a nucleotide sequence according to claims 11 to 13 encoding a parent phytase enzyme to mutagenesis, and step b) comprises expressing the mutated nucleotide sequence obtained in step (a) in a host cell, and step c) comprises screening for host cells, or extract(s) thereof, for an improved phytase enzyme variant with said improved property/properties.<br>
In a further embodiment after step c), and optionally d), further comprise at least one subsequent round of repeating steps a) to c), and optionally d) wherein, preferably, in said subsequent round(s), the at least one parent phytase enzyme of step a) is selected from said at least one phytase enzyme variant and/or an improved phytase variant prepared according to the method. ,<br>
In a further preferably embodiment step c) comprises screening for host cells expressing an improved phytase enzyme variant which compared to either i) said parent phytase enzyme and/or ii) a polypeptide comprising SEQ ID No 3 of functional fragment thereof, has a thermal stability difference of at least 2.5.<br>
In a further embodiment step c) comprises screening for host cells expressing an improved phytase enzyme variant which compared to either i) said parent phytase enzyme and/or ii) a polypeptide comprising SEQ ID No 3 of functional fragment thereof has pepsin stability of at least 30.<br>
In a further embodiment step c) comprises screening for host cells expressing an improved phytase enzyme variant which compared to either i) said parent phytase enzyme and/or ii) a polypeptide comprising SEQ ID No 3 has a specific activity ratio when compared to the phytase encoded by SEQ ID No 3 of at least 110.<br>
The invention also provides a method of preparing a phytase enzyme variant, which method comprises:<br>
a) Selecting a parent phytase enzyme, wherein the parent phytase enzyme is selected from<br>
i.         a parent phytase enzyme with at least 75% homology to SEQ ID No 3 of functional fragment thereof<br><br>
ii.        a parent phytase enzyme derived from Buttiauxella spp. iii.       at least one phytase enzyme variant<br>
b)	Making at least one alteration which is an insertion, a deletion or a substitution<br>
of an ammo acid residue in the parent phytase enzyme to obtain a phytase enzyme<br>
variant,<br>
c)	Screening for a phytase enzyme variant which compared to the parent phytase<br>
enzyme has improved characteristics, as herein described, preferably selected from<br>
one or more of:<br>
i.         higher thermal stability and/or ii.        specific activity and/or iii.       proteolytic stability and/or<br>
d)	Preparing the phytase enzyme variant<br>
Optionally, at least steps a) to c) may be repeated in one or more subsequent (iterative) rounds. Therefore, it is envisaged that the parent phytase enzyme is a phytase enzyme variant prepared by previous rounds of the above method a) to c).<br>
In a further embodiment the invention provides a method of preparing a phytase variant, comprising the following steps:<br>
(a)	providing a parent phytase enzyme, selected from<br>
(i) a phytase enzyme with at least 75% homology to SEQ ID No. 3 or functional fragment thereof<br>
(ii) a phytase enzyme derived from Buttiaxella spp., (iii) at least one phytase enzyme variant<br>
(b)	generating a population of phytase variants by alteration of the parent phytase,<br>
Preferably, said alteration(s) is/are obtained through an insertion,  deletion or<br>
substitution of at least one amino acid residue in the parent phytase, or any<br>
combination thereof.<br>
(c)	screening of the population for a phytase variant which compared to the parent<br>
phytase enzyme has improved characteristics, as herein described, preferably<br>
selected from one or more of::<br>
(i) higher thermal stability and/or (ii) higher specific activity and/or (iii) higher proteolytic stability,<br><br>
 (d)	selecting one or more phytase variants from the population of phytases.<br>
(e)	optionally repeating steps (a) to (c)   cyclically, and preferably wherein phytase<br>
variants selected in one cycle are used a first phytases in the following cycle.<br>
In a further embodiment the invention provides a method of preparing a phytase enzyme variant, which method comprises:<br>
a)	Subjecting nucleotide  sequence  encoding  a parent phytase  enzyme  to<br>
mutagenesis, wherein the parent phytase enzyme is selected from<br>
i.         a parent phytase enzyme with at least 75% homology to SEQ ID No 3 or<br>
functional fragment thereof<br>
ii.        a parent phytase enzyme derived from Buttiauxella spp. iv.       at least one phytase enzyme variant<br>
b)	Expressing the mutated nucleotide sequence obtained in step(a) in a host cell,<br>
and<br>
c)	Screening for host cells expressing a for a phytase enzyme variant which<br>
compared to the parent phytase enzyme has improved characteristics, as herein<br>
described, preferably selected from one or more of::<br>
i) higher thermal stability and/or ii) higher specific activity and/or iii) higher proteolytic stability and/or<br>
d)	Preparing the phytase enzyme variant expressed by the host cell<br>
Optionally, steps a) to c), optionally including d) may be repeated in one or more subsequent (iterative) rounds.<br>
In a further embodiment the invention provides a method of preparing a phytase variant, comprising the following steps:<br>
(a)	subjecting a nucleotide sequence encoding a parent phytase enzyme to<br>
mutagenesis to generate a population of altered nucleotide variants, wherein preferably, said alterations) is/are obtained through an insertion, deletion or substitution of at least one amino acid residue in the parent phytase, or any combination thereof., and wherein the parent phytase enzyme is selected from<br>
(i) a phytase enzyme with at least 75% homology to SEQ ID No. 3 or functional fragment thereof<br><br>
 (ii) a phytase enzyme derived from Buttiaxella spp., (iii) at least one phytase enzyme variant<br>
(b)	expressing the population of nucleotide variants obtained is step (a) in a<br>
population of corresponding host cells, and<br>
(c)	screening of the population for a phytase variant which compared to the parent<br>
phytase enzyme has unproved characteristics, as herein described, preferably<br>
selected from one or more of::<br>
(i) higher thermal stability and/or (ii) higher specific activity and/or (iii) higher proteolytic stability,<br>
(d)	selecting one or more phytase variants from the population of phytases.<br>
(e)	optionally repeating steps (a) to (c)   cyclically, and preferably wherein phytase<br>
variants selected in one cycle are used a first phytases in the following cycle.<br>
Where appropriate, in the above methods of preparing a phytase enzyme variant, the said nucleotide sequence is preferably a DNA sequence.<br>
The nucleotide sequence is preferably an isolated and/or purified nucleic acid molecule or nucleotide sequence coding for the enzyme comprising the amino acid sequence corresponding to Buttiauxella sp. phytase, or a homologue thereof as herein described.<br>
The phytase parent is preferably selected from SEQ ID No 3 or functional fragment thereof or a homologue of the Buttiauxella sp phytase as disclosed in SEQ ID No 3.as herein described.<br>
In the above embodiments of the invention, which relate to methods of preparing phytase enzyme variant the parent phytase enzyme / nucleotide encoding parent phytase enzyme/nucleotide is preferably a wildtype phytase.<br>
However, in another the parent may be a variant prepared by previous rounds of mutagenesis. i.e. In one embodiment, the methods of preparing phytase enzyme variants are iterative, wherein the steps a) to c) (optionally including step d)) are repeated at least more that once. In such embodiments the mutagenesis methods used<br><br>
in the first round of mutagenesis is preferably error prone PCR, more preferably error threshold PCR. Subsequent rounds may also be error prone PCR, more preferably error threshold PCR, but may alternatively be recombination based mutagenesis, wherein groups of at least two independent improved variants are identified in a first rounds of mutagenesis are, during a second or subsequent round of mutagenesis recombined to give at least one recombinant variant (e.g. using family shuffling or recombinase chain reaction methods).<br>
It will be apparent to the skilled person that alternative mutagenesis methods can also be used, including rational design, site scanning mutagenesis, or chemically/radiation induced mutagenesis.<br>
In the above embodiments of the invention, which relate to methods of preparing phytase enzyme variant the phytase enzyme variant is preferably screened for higher thermal stability.<br>
In the above embodiments of the invention, which relate to methods of preparing phytase enzyme variant the phytase enzyme variant is preferably screened for at least a single parameter, preferably selected from higher thermal stability, higher proteolytic stability or higher specific activity, most preferably higher thermal stability.<br>
Preferably, the screening for said first parameter, is performed in at least a first round of mutagenesis comprising of at least steps a) to c), wherein c) comprises at least the screening for said first parameter. This first round of mutagenesis can then be followed by further (iterative) rounds of mutagenesis and selection comprising steps a) to c) (optionally including d)) wherein the selection in said further rounds can be selected from the same selection parameter as used in step c) of the said first round, or alternatively a different parameter.<br>
During iterative rounds of the above methods of preparing a phytase enzyme variant, preferably said first parameter is selected from higher thermal stability, higher proteolytic stability or higher specific activity, most preferably higher thermal stability. Preferably, when a first round of mutagenesis has been performed to select<br><br>
variants with a higher thermal stability, during a subsequent (iterative) round(s) of mutagenesis comprising steps a) to c), said parameter is selected from higher thermal stability, higher proteolytic stability or higher specific activity, most preferably higher proteolytic stability or higher specific activity.<br>
In the above embodiments of the invention, which relate to methods of preparing phytase en2yme variant the phytase enzyme variant is preferably screened for higher thermal stability and higher proteolytic stability and higher specific activity in at least one round of mutagenesis (steps a to c), preferably more than one round, i.e. iterative rounds of selection.<br>
The parent phytase enzyme is preferably derived from Buttiauxella PI-29 .<br>
In methods of preparing a phytase enzyme variant, which method comprises subjecting DNA sequence encoding a parent phytase enzyme to mutagenesis, the DNA sequence encoding a parent phytase enzyme is preferably subjected to random mutagenesis, more preferably error prone PCR, even more preferably error threshold PCR.<br>
The preferred methods of mutagensis of DNA sequence encoding a parent phytase enzyme is error prone PCR, more preferably error threshold PCR, other methods of mutagensis may be used either in place of error prone/threshold PCR or in conjunction with error prone/threshold PCR. See Example 12 which provides references for suitable error prone PCR and error threshold PCR methods. Another suitable method for mutagenic PCR is disclosed by Cadwell and Joyce (PCR Methods Appl. 3(1994), 136-140)"<br>
The term 'expression in a host cell' when used in the context of the embodiments which refer to 'a method of preparing a phytase enzyme variant' is preferably defined as production of the phytase enzyme variant in a living organism, organ or cell as herein defined. Preferable hosts are Escherichia coll K12; Bacillus subtilis; Saccharomyces cerevisiae, Phytase enzymes detailed herein were characterized after heterologous expression in one or more of the following expression hosts: Escherichia coli K12; Bacillus subtilis; Saccharomyces cerevisiae.<br><br>
However, it is considered that for the purpose of selection the phytase enzyme variants as herein described, such methods may employ in vitro methods of expression of phytase enzyme variants, preferably for use in step (c) of said methods, which utilise the transcription and translation machinery isolated from of one or more cells isolated from one or more living organism or viruses. Such in vitro production of variant phytases on the invention can also be used for selecting preferred variant phytases. In vitro expression can be suitably performed using standard techniques. For reference please see {In vitro Expression Guide1 available from Promega Inc (Part# BR053).<br>
Definitions of Variant Phenotypes.<br>
Variants with higher thermal stability (thermal stability difference) are preferably determined using the methods disclosed in Example 12.<br>
The variant phytase enzyme prepared by the method of preparing phytase enzyme variants preferably has a thermal stability difference (T.D) of at least 1.5, more preferably 2, 2.5, 3, 3.5, 4, 5, 6, 1, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 most preferably at least 20.<br>
Variants with higher proteolytic stability are preferably determined by the methods disclosed in Example 12.<br>
Preferably the phytase enzyme variant of the invention has a proteolytic stability (residual activity) of at least 45%, preferably 50%, 55%, more preferably at least 60% or 65%, most preferably at least 70%.<br>
Preferably the phytase variant of the invention has a specific activity of greater than 100% of wt activity at pH 4.0, preferably greater than 105%, 110%, more preferably greater than 114%.<br><br>
Further Variant Embodiments<br>
In a further embodiment the invention provides methods for the preparation of an animal feed comprising a phytase enzyme variant, said method comprising the sequential steps of i) performing one or more of the above methods of preparing a phytase enzyme variant, and ii) Adding the prepared phytase enzyme variant to an animal feed.<br>
In a specific embodiment the invention provides a method for the preparation of an animal feed comprising a phytase enzyme variant, said method comprising<br>
a)	Selecting a parent phytase enzyme, wherein the parent phytase enzyme is selected<br>
from<br>
i.         a parent phytase enzyme with at least 75% homology to SEQ ID No 3 or<br>
functional fragment thereof, ii.        a parent phytase enzyme derived from Buttiauxella spp, preferably<br>
Buttiauxella PI-29. and/or iii.       at least one phytase enzyme variant;<br>
b)	Making at least one alteration which is an insertion, a deletion or a substitution of<br>
an ammo acid residue in the parent phytase enzyme to obtain a phytase enzyme<br>
variant.<br>
c)	Screening for a phytase enzyme variant which compared to the parent phytase<br>
enzyme has improved characteristics, as herein described, preferably selected from<br>
one or more of::<br>
i.	higher thermal stability and/or<br>
ii.	specific activity and/or<br>
iii.	proteolytic stability and/or<br>
d)	Preparing the phytase enzyme variant<br>
e)	Adding the prepared phytase enzyme variant to an animal feed.<br>
In a further embodiment the invention provides methods for the preparation of an animal feed comprising a phytase enzyme variant.<br><br>
a)	Subjecting nucleotide (preferably DNA) sequence encoding a parent phytase<br>
enzyme to mutagenesis, wherein said nucleotide is selected from a nucleotide which<br>
encodes<br>
i.         a parent phytase enzyme with at least 75% homology to SEQ ID No 3 of<br>
functional fragment thereof, and/or<br>
ii.        a  parent   phytase   enzyme   derived   from   Buttiawcella   spp,   preferably<br>
ButtiauxellaPl-29.<br>
b)	Expressing the mutated nucleotide (preferably DNA) sequence obtained in step(a)<br>
in a host cell, and<br>
c)	Screening for host cells expressing a for a phytase enzyme variant which compared<br>
to the parent phytase enzyme has improved characteristics, as herein described,<br>
preferably selected from one or more of::<br>
i. higher thermal stability and/or ii. higher specific activity and/or iii. higher proteolytic stability and/or<br>
d)	Preparing the phytase enzyme variant expressed by the host cell<br>
f) Adding the prepared phytase enzyme variant to an animal feed.<br>
The described embodiments and preferred aspects of the method of preparing a phytase enzyme variant also apply to the above methods of preparing an animal feed comprising a phytase enzyme variant.<br>
In another aspect of the invention there is provided a host cell transformed or transfected with a nucleic acid encoding a phytase as described herein.<br>
Suitably, the host cell in accordance with this aspect of the invention comprises a phytase which comprises an amino acid sequence, or functional fragment thereof, as set out in SEQ ID NO: 3 or a sequence that is at least 75% homologous thereto.<br>
In a preferred embodiment, said host cell produces a phytase.<br>
In a further aspect of the invention there is provided a host cell transformed or transfected with a nucleic acid encoding a phytase in accordance with the invention. Preferably, the phytase is a Buttiauxella sp. phytase as described herein or a<br><br>
homologue or derivative thereof. Suitably, said phytase enzyme comprises an amino acid sequence, or functional fragment thereof, as set out in any of SEQ ID No: 3 or a sequence that is at least 75% homologous (identical) thereto. Preferably, said host cell produces a phytase.<br>
In one embodiment, the nucleotide sequence which can be used in the present invention is obtainable from (though it does not have to be actually obtained from) Buttiauxella sp., although it will be recognised that enzymes isolated and/or purified from equivalent strains may equally be used.<br>
Suitably the host cell is derived from a microorganism including bacteria and fungi, including yeast. In a particularly preferred embodiment the host cell is a prokaryotic bacterial cell. Suitable bacterial host cells include bacteria from different prokaryotic taxonomic groups including proteobacteria, including members of the alpha, beta, gamma, delta and epsilon subdivision, gram-positive bacteria such as Actinomycetes, Firmicutes, Clostridium and relatives, flavobacteria, cyanobacteria, green sulfur bacteria, green non-sulfur bacteria, and archaea. Particularly preferred are the Enterobacteriaceae such as Escherichia coll proteobacteria belonging to the gamma subdivision and low GC-Gram positive bacteria such as Bacillus.<br>
Suitable fungal host cells include yeast selected from the group consisting of Ascomycota including Saccharomycetes such as Pichia, Hansenula, and Saccharomyces, Schizosaccharmycetes such as Schizosaccharomyces pombe and anamorphic Ascomycota including Aspergillus.<br>
Other suitable eukaryotic host cells include insect cells such as SF9, SF21, Trychplusiani and M121 cells. For example, the polypeptides according to the invention can advantageously be expressed in insect cell systems. As well as expression in insect cells in culture, phytase genes can be expressed in whole insect organisms. Virus vectors such as baculovirus allow infection of entire insects. Large insects, such as silk moths, provide a high yield of heterologous protein. The protein can be extracted from the insects according to conventional extraction techniques. Expression vectors suitable for use in the invention include all vectors which are capable of expressing foreign proteins in insect cell lines.<br><br>
Other host cells include plant cells selected from the group consisting of protoplasts, cells, calli, tissues, organs, seeds, embryos, ovules, zygotes, etc. The invention also provides whole plants that have been transformed and comprise the recombinant DNA of the invention.<br>
The term "plant" generally includes eukaryotic alga, embryophytes including Bryophyta, Pteridophyta and Spermatophyta such as Gymnospermae and Angiospermae.<br>
Preferably, said host cell is a microorganism. Preferred microorganisms include prokaryotic bacterial cells preferably, E. coli, B. subtilis and other species of the genus Bacillus, yeast, preferably, Hansenula polymorpha and Schizosaccharomyces pombe.<br>
In another aspect of the invention there is provided a bacterial cell strain Buttiauxella sp Pl-29 deposited by Danisco Global Innovation, Sokeritehtaantie 20, FIN-02460 Kantvik, Finland on 22 September 2004 under accession number NCIMB 41248. Such a cell can be incorporated directly into feed.<br>
In another aspect, there is provided a method for the production of phytases comprising transfecting a host cell with an expression vector or plasmid in accordance with the invention, culturing said host cell under conditions for the expression of the phytase and extracting said phytase from the host cell culture media.<br>
Suitably said method is for the production of a phytase comprising expressing an ammo acid sequence as set out in SEQ ID NO: 3 or a sequence having at least 75% homology thereto or an effective fragment thereof in a host cell and extracting the secreted protein from the host cell culture medium.<br>
Another aspect of the invention provides a feed composition comprising a phytase in accordance with the invention. Preferably, the feed composition comprises a phytase at a concentration of 10-10000 U/kg feed, preferably, 200-2000U/kg feed, more preferably, 500-1000 U/kg feed.<br><br>
In one embodiment, the feed composition comprises a host cell in accordance with the invention.<br>
In a further aspect there is provided the use of a phytase in accordance with the invention in food or animal feed.<br>
PREFERABLE ASPECTS<br>
Preferable aspects are presented in the accompanying claims and in the following description and Examples section.<br>
ADDITIONAL ADVANTAGES<br>
The present invention is advantageous as it provides phytases that have a number of properties that make them particularly useful as additives to animal feeds.<br>
In particular, the phytases of the present invention are active at low pH and, preferably in the range pH 2 to 5.5 with activity maximum around pH 4 - 4.5. Suitably the phytases of the present invention are active at low pHs (retaining about 40% of the maximum activity at pH 2.5) of the stomach environment.<br>
Furthermore, the phytases of the present invention are efficiently secreted both in the native host and during heterologous expression thus leading to more efficient production and isolation for addition to feed.<br>
Moreover, the phytases of the present invention, preferably have a broad substrate specificity including penta- terra, tri and di- phosphate substrates thereby increasing the total phosphate available for absorption by the animal (available phosphate). The phytases of the present invention also, preferably, have a high specific activity in the region of 300 U/mg +/- approximately 10%, preferably at least 300 U/mg.<br><br>
The products of the present invention may be used as additives/supplements to foods and feed. The products may also be useful in the commercial production of various inositol-phosphates.<br>
PHYTATE/PHYTIC ACID/PHYTASES<br>
Phytic acid (wyo-inositol hexakisphosphate) is an important constituent in cereals, legumes and oilseed crops. The salt form, phytate, is the major storage form of phosphorous in these plants.<br>
Phytases catalyse phosphate monoester hydrolysis of phytic acid which results in the step-wise formation of myo-inositol pentakis-, tetrakis-, tris-, bis- and monophosphates, as well as the liberation of inorganic phosphate.<br>
The terms "wild type phytase" or "wild type" as used herein refer to a phytase enzyme with an amino acid sequence found in nature.<br>
The terms "phytase variant" or "variant" or "modified form" refer to a phytase enzyme with an amino acid sequence derived from the amino acid sequence of a parent phytase having one or more amino acid substitutions, insertions, and/or deletions, which together are referred to as "mutations".<br>
The terms "parent phytase" or "parent enzyme" refer to a phytase enzyme from which a phytase variant is derived. A parent phytase can be a wild type phytase or another phytase variant. In particular, in the present invention, a "parent phytase" may be derived from a Buttiauxella sp.. Suitably, the "parent phytase" is derived from Buttiauxella strain PI-29 as described herein which, preferably has the ammo acid sequence set out in SEQ ID N0:3.<br>
ISOLATED<br>
In one aspect, preferably the nucleotide or amino acid sequence is hi an isolated form. The term "isolated" means that the sequence is at least substantially free from at least<br><br>
one other component with which the sequence is naturally associated in nature and as found in nature.<br>
PURIFIED<br>
In one aspect, preferably the nucleotide or amino acid sequence is in a purified form. The term "purified" means that the sequence is in a relatively pure state at least 1%, 5% pure or 10% pure, more preferably at least 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90% pure. In a preferred embodiment, when referring to a polypeptide, the purity as defined above is determined in terms of being purified from other polypeptides by SDS-PAGE electrophoresis. In a preferred embodiment, when referring to a polynucleotide, the purity as defined above is determined in terms of being purified from other polynucleotides.<br>
NUCLEOTIDE SEQUENCE<br>
The scope of the present invention encompasses nucleotide sequences encoding enzymes having the specific properties as defined herein.<br>
The term "nucleotide sequence" as used herein refers to an oligonucleotide sequence, nucleic acid or polynucleotide sequence, and variant, homologues, fragments and derivatives thereof (such as portions thereof). The nucleotide sequence may be of genomic or synthetic or recombinant origin, which may be double-stranded or single-stranded whether representing the sense or anti-sense strand.<br>
..».<br>
The term "nucleotide sequence" or "nucleic acid molecule" in relation to the present invention includes genomic DNA, cDNA, synthetic DNA, and RNA. Preferably it means DNA, more preferably cDNA sequence coding for the present invention.<br>
In a preferred embodiment, the nucleotide sequence when relating to and when encompassed by the per se scope of the present invention does not include the native nucleotide sequence according to the present invention when in its natural environment and when it is linked to its naturally associated sequence(s) that is/are also in its/their natural environment. For ease of reference, we shall call this preferred embodiment the<br><br>
"non-native nucleotide sequence". In this regard, the term "native nucleotide sequence" means an entire nucleotide sequence that is in its native environment and when operatively linked to an entire promoter with which it is naturally associated, which promoter is also in its native environment However, the amino acid sequence encompassed by scope the present invention can be isolated and/or purified post expression of a nucleotide sequence in its native organism. Preferably, however, the amino acid sequence encompassed by scope of the present invention may be expressed by a nucleotide sequence in its native organism but wherein the nucleotide sequence is not under the control of the promoter with which it is naturally associated within that organism.<br>
PREPARATION OF A NUCLEOTIDE SEQUENCE<br>
Typically, the nucleotide sequence encompassed by scope of the present invention or the nucleotide sequences for use in the present invention are prepared using recombinant DNA techniques (i.e. recombinant DNA). However, in an alternative embodiment of the invention, the nucleotide sequence could be synthesised, in whole or in part, using chemical methods well known in the art (see Caruthers MH et al, (1980) Nuc Acids Res Symp Ser 215-23 and Horn T et al, (1980) Nuc Acids Res Symp Ser 225-232).<br>
A nucleotide sequence encoding either an enzyme which has the specific properties as defined herein or an enzyme which is suitable for modification may be identified and/or isolated and/or purified from any cell or organism producing said enzyme. Various methods are well known within the art for the identification and/or isolation and/or purification of nucleotide sequences. By way of example, PCR amplification techniques to prepare more of a sequence may be used once a suitable sequence has been identified and/or isolated and/or purified.<br>
By way of further example, a genomic DNA and/or cDNA library may be constructed using chromosomal DNA or messenger RNA from the organism producing the enzyme. If the amino acid sequence of the enzyme or a part of the amino acid sequence of the enzyme is known, labelled oligonucleotide probes may be synthesised and used to identify enzyme-encoding clones from the genomic library prepared from<br><br>
the organism. Alternatively, a labelled oligonucleotide probe containing sequences homologous to another known enzyme gene could be used to identify enzyme-encoding clones. In the latter case, hybridisation and washing conditions of lower stringency are used.<br>
Alternatively, enzyme-encoding clones could be identified by inserting fragments of genomic DNA into an expression vector, such as a plasmid, transforming enzyme-negative bacteria with the resulting genomic DNA library, and then plating the transformed bacteria onto agar plates containing a substrate for the enzyme (e.g. maltose for a glucosidase (maltase) producing enzyme), thereby allowing clones expressing the enzyme to be identified.<br>
In a yet further alternative, the nucleotide sequence encoding the enzyme may be prepared synthetically by established standard methods, e.g. the phosphoroamidite method described by Beucage S.L. et al, (1981) Tetrahedron Letters 22, p 1859-1869, or the method described by Matihes et al., (1984) EMBO J. 3, p 801-805. In the phosphoroamidite method, oligonucleotides are synthesised, e.g. in an automatic DNA synthesiser, purified, annealed, ligated and cloned in appropriate vectors.<br>
The nucleotide sequence may be of mixed genomic and synthetic origin, mixed synthetic and cDNA origin, or mixed genomic and cDNA origin, prepared by ligating fragments of synthetic, genomic or cDNA origin (as appropriate) in accordance with standard techniques. Each ligated fragment corresponds to various parts of the entire nucleotide sequence. The DNA sequence may also be prepared by polymerase chain reaction (PCR) using specific primers, for instance as described in US 4,683,202 or in Saiki HKetal., (Science (1988) 239,pp487-491).<br>
Due to degeneracy in the genetic code, nucleotide sequences may be readily produced in which the triplet codon usage, for some or all of the ammo acids encoded by the original nucleotide sequence, has been changed thereby producing a nucleotide sequence with low homology to the original nucleotide sequence but which encodes the same, or a variant, amino acid sequence as encoded by the original nucleotide sequence. For example, for most amino acids the degeneracy of the genetic code is at the third position in the triplet codon (wobble position) (for reference see Stryer,<br><br>
Lubert, Biochemistry, Third Edition, Freeman Press, ISBN 0-7167-1920-7) therefore, a nucleotide sequence in which all triplet codons have been "wobbled" in the third position would be about 66% identical to the original nucleotide sequence however, the amended nucleotide sequence would encode for the same, or a variant, primary amino acid sequence as the original nucleotide sequence.<br>
Therefore, the present invention further relates to any nucleotide sequence that has alternative triplet codon usage for at least one amino acid encoding triplet codon, but which encodes the same, or a variant, polypeptide sequence as the polypeptide sequence encoded by the original nucleotide sequence.<br>
Furthermore, specific organisms typically have a bias as to which triplet codons are used to encode amino acids. Preferred codon usage tables are widely available, and can be used to prepare codon optimised genes. Such codon optimisation techniques are routinely used to optimise expression of transgenes in a heterologous host.<br>
MOLECULAR EVOLUTION<br>
Once an enzyme-encoding nucleotide sequence has been isolated and/or purified, or a putative enzyme-encoding nucleotide sequence has been identified, it may be desirable to modify the selected nucleotide sequence, for example it may be desirable to mutate the sequence in order to prepare an enzyme in accordance with the present invention.<br>
Mutations may be introduced using synthetic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites.<br>
A suitable method is disclosed in Morinaga et al (Biotechnology (1984) 2, p646-649). Another method of introducing mutations into enzyme-encoding nucleotide sequences is described in Nelson and Long (Analytical Biochemistry (1989), 180, p 147-151).<br>
Instead of site directed mutagenesis, such as described above, one can introduce mutations randomly for instance using a commercial kit such as the GeneMorph PCR<br><br>
mutagenesis kit from Stratagene, or the Diversify PCR random mutagenesis kit from Clontech.<br>
A third method to obtain novel sequences is to fragment non-identical nucleotide sequences, either by using any number of restriction enzymes or an enzyme such as Dnase I, and reassembling full nucleotide sequences coding for functional proteins. Alternatively one can use one or multiple non-identical nucleotide sequences and introduce mutations during the reassembly of the full nucleotide sequence.<br>
Thus, it is possible to produce numerous site directed or random mutations into a nucleotide sequence, either in vivo or in vitro, and to subsequently screen for improved functionality of the encoded polypeptide by various means.<br>
As a non-limiting example, mutations or natural variants of a polynucleotide sequence can be recombined with either the wildtype or other mutations or natural variants to produce new variants. Such new variants can also be screened for improved functionality of the encoded polypeptide. The production of new preferred variants may be achieved by various methods well established in the art, for example the Error Threshold Mutagenesis (WO 92/18645), oligonucleotide mediated random mutagenesis (US 5,723,323), DNA shuffling (US 5,605,793), exo-mediated gene assembly (WO 0058517). Other suitable methods are described, for example in WO 0134835, WO 02/097130, WO 03/012100, WO03/057247, WO 2004/018674, US 6,303,344 and US 6,132,970.<br>
The application of the above-mentioned and similar molecular evolution methods allows the identification and selection of variants of the enzymes of the present invention which have preferred characteristics without any prior knowledge of protein structure or function, and allows the production of non-predictable but beneficial mutations or variants. There are numerous examples of the application of molecular evolution in the art for the optimisation or alteration of enzyme activity, such examples include, but are not limited to one or more of the following: optimised expression and/or activity in a host cell or in vitro, increased enzymatic activity, altered substrate and/or product specificity, increased or decreased enzymatic or<br><br>
structural stability, altered enzymatic activity/specificity in preferred environmental conditions, e.g. temperature, pH, substrate<br>
The above molecular evolution methods may be used in the methods of preparing a phytase enzyme variant as herein described.<br>
AMINO ACID SEQUENCES<br>
The scope of the present invention also encompasses amino acid sequences of enzymes having the specific properties as defined herein.<br>
As used herein, the term "amino acid sequence" is synonymous with the term "polypeptide" and/or the term "protein". In some instances, the term "amino acid sequence" is synonymous with the term "peptide". In some instances, the term "amino acid sequence" is synonymous with the term "enzyme".<br>
The amino acid sequence may be prepared/isolated from a suitable source, or it may be made synthetically or it may be prepared by use of recombinant DNA techniques.<br>
The enzyme encompassed in the present invention may be used in conjunction with other enzymes. Thus the present invention also covers a combination of enzymes wherein the combination comprises the enzyme of the present invention and another enzyme, which may be another enzyme according to the present invention. This aspect is discussed hi a later section.<br>
Preferably the amino acid sequence when relating to and when encompassed by the per se scope of the present invention is not a native enzyme. In this regard, the term "native enzyme" means an entire enzyme that is in its native environment and when it has been expressed by its native nucleotide sequence.<br><br>
VARIANTS/HOMOLOGUES/DERIVATIVES<br>
The present invention also encompasses the use of variants, homologues and derivatives of any amino acid sequence of an enzyme or of any nucleotide sequence encoding such an enzyme.<br>
Here, the term "homologue" means an entity having a certain homology with the amino acid sequences and the nucleotide sequences. Here, the term "homology" can be equated with "identity". Suitably, "homologous" in this context refers to the percentage of sequence identity between two enzymes after aligning their sequences using alignment algorithms as described in more detail below.<br>
In the present context, a homologous amino acid sequence is taken to include an amino acid sequence which may be at least 75, 80, 81, 85 or 90% identical, preferably at least 95, 96, 97, 98 or 99% identical to the sequence. Typically, the homologues will comprise the same active sites etc. - e.g as the subject amino acid sequence. Although homology can also be considered in terms of similarity (i.e. amino acid residues having similar chemical properties/functions), in the context of the present invention it is preferred to express homology in terms of sequence identity.<br>
By "functional fragment" is meant a fragment of the polypeptide that retains that characteristic properties of that polypeptide. In the context of the present invention, a functional fragment of a phytase enzyme is a fragment that retains the carotenoid cleavage capability of the whole protein.<br>
In the present context, a homologous nucleotide sequence is taken to include a nucleotide sequence which may be at least 75, 80, 81, 85 or 90% identical, preferably at least 95, 96, 97, 98 or 99% identical to a nucleotide sequence encoding an enzyme of the present invention (the subject sequence). Typically, the homologues will comprise the same sequences that code for the active sites etc. as the subject sequence. Although homology can also be considered in terms of similarity (i.e. amino acid residues having similar chemical properties/functions), in the context of the present invention it is preferred to express homology in terms of sequence identity.<br>
For the amino acid sequences and the nucleotide sequences, homology comparisons can be conducted by eye, or more usually, with the aid of readily available sequence comparison programs. These commercially available computer programs can calculate % homology between two or more sequences.<br>
% homology may be calculated over contiguous sequences, i.e. one sequence is aligned with the other sequence and each amino acid in one sequence is directly compared with the corresponding amino acid in the other sequence, one residue at a time. This is called an "ungapped" alignment. Typically, such ungapped alignments are performed only over a relatively short number of residues.<br>
Although this is a very simple and consistent method, it fails to take into consideration that, for example, in an otherwise identical pah" of sequences, one insertion or deletion will cause the following amino acid residues to be put out of alignment, thus potentially resulting in a large reduction in % homology when a global alignment is performed. Consequently, most sequence comparison methods are designed to produce optimal alignments that take into consideration possible insertions and deletions without penalising unduly the overall homology score. This is achieved by inserting "gaps" in the sequence alignment to try to maximise local homology.<br>
However, these more complex methods assign "gap penalties" to each gap that occurs in the alignment so that, for the same number of identical amino acids, a sequence alignment with as few gaps as possible - reflecting higher relatedness between the two compared sequences - will achieve a higher score than one with many gaps. "Affme gap costs" are typically used that charge a relatively high cost for the existence of a gap and a smaller penalty for each subsequent residue in the gap. This is the most commonly used gap scoring system. High gap penalties will of course produce optimised alignments with fewer gaps. Most alignment programs allow the gap penalties to be modified. However, it is preferred to use the default values when using such software for sequence comparisons. For example when using the GCG Wisconsin Bestfit package the default gap penalty for amino acid sequences is -12 for a gap and -4 for each extension.<br><br>
Calculation of maximum % homology therefore firstly requires the production of an optimal alignment, taking into consideration gap penalties. A suitable computer program for carrying out such an alignment is the GCG Wisconsin Bestfit package (Devereux et al 1984 Nuc. Acids Research 12 p387). Examples of other software than can perform sequence comparisons include, but are not limited to, the BLAST package (see Ausubel et al, 1999 Short Protocols in Molecular Biology, 4th Ed -Chapter 18), FASTA (Altschul et al, 1990 J. Mol Biol 403-410) and the GENEWORKS suite of comparison tools. Both BLAST and FASTA are available for offline and online searching (see Ausubel et al, 1999, Short Protocols in Molecular Biology, pages 7-58 to 7-60).<br>
However, for some applications, it is preferred to use the GCG Bestfit program. A new tool, called BLAST 2 Sequences is also available for comparing protein and nucleotide sequence (see FEMS Microbiol Lett 1999 174(2): 247-50; FEMS Microbiol Lett 1999 177(1): 187-8 <a href="../cdn-cgi/l/email-protection.html" class="__cf_email__" data-cfemail="4928272d3d283d2028272809272a2b206727252467272021672e263f">[email&#160;protected]</a>).<br>
Although the final % homology can be measured in terms of identity, the alignment process itself is typically not based on an all-or-nothing pair comparison. Instead, a scaled similarity score matrix is generally used that assigns scores to each pairwise comparison based on chemical similarity or evolutionary distance. An example of such a matrix commonly used is the BLOSUM62 matrix - the default matrix for the BLAST suite of programs. GCG Wisconsin programs generally use either the public default values or a custom symbol comparison table if supplied (see user manual for further details). For some applications, it is preferred to use the public default values for the GCG package, or in the case of other software, the default matrix, such as BLOSUM62.<br>
Alternatively, percentage homologies may be calculated using the multiple alignment feature in DNASIS™ (Hitachi Software), based on an algorithm, analogous to CLUSTAL (Higgins DG &amp; Sharp PM (1988), Gene 73(1), 237-244).<br><br>
Once the software has produced an optimal alignment, it is possible to calculate % homology, preferably % sequence identity. The software typically does this as part of the sequence comparison and generates a numerical result.<br>
In a preferable aspect of the present invention the following software and settings for calculating percentage sequence homology/identity are used. For amino acid sequences percentage of identities (homology) or "positives" are calculated by the AlignX VectorNTI (Vector NTI Advance 9.1 from Invitrogen Corporation, Carlsbad, California, USA.) for each possible pair of amino acid sequences. Settings are default parameters (Gap opening penalty -10, Gap extension penalty 0.1).<br>
For nucleic acid sequences percentage of identities (homology) or "positives" are calculated by the AlignX VectorNTI programme from Informax Inc. (USA) for each possible pair of nucleic acid sequences. Settings are default settings for DNA are : Gap opening penalty: 15 and Gap extension penalty: 6.66. (same settings for multiple alignments).<br>
Preferably the amino acid identity (homology) is calculated across the full-length amino acid sequence or for nucleic acid to a corresponding polynucleotide which encodes the respective the full-length ammo acid sequence.<br>
The sequences may also have deletions, insertions or substitutions of ammo acid residues which produce a silent change and result in a functionally equivalent substance. Deliberate amino acid substitutions may be made on the basis of similarity in amino acid properties (such as polarity, charge, solubility, hydrophobicity, hydrophilicity, and/or the amphipathic nature of the residues) and it is therefore useful to group amino acids together in functional groups. Amino acids can be grouped together based on the t&gt;rot&gt;erties of their side chain alone. However it is more useful to include mutation data as well. The sets of amino acids thus derived are likely to be conserved for structural reasons. These sets can be described in the form of a Venn diagram (Livingstone C.D. and Barton G.J. (1993) "Protein sequence alignments: a strategy for the hierarchical analysis of residue conservation" ComputAppl Biosci. 9: 745-756)(Taylor W.R. (1986) "The classification of amino acid conservation" J.Theor.Biol. 119; 205-218). Conservative substitutions may be made, for example<br><br>
according to the table below which describes a generally accepted Venn diagram grouping of amino acids.<br><br>
(Table Removed)   The present invention also encompasses homologous substitution (substitution and replacement are both used herein to mean the interchange of an existing amino acid residue, with an alternative residue) that may occur i.e. like-for-like substitution such as basic for basic, acidic for acidic, polar for polar etc. Non-homologous substitution may also occur i.e. from one class of residue to another or alternatively involving the inclusion of unnatural amino acids such as ornithine (hereinafter referred to as Z), diaminobutyric acid ornithine (hereinafter referred to as B), norleucine ornithine (hereinafter referred to as O), pyriylalanine, thienylalanine, naphthylalanine and phenylglycine.<br>
Replacements may also be made by unnatural amino acids.<br>
Variant amino acid sequences may include suitable spacer groups that may be inserted between any two ammo acid residues of the sequence including alkyl groups such as methyl, ethyl or propyl groups in addition to amino acid spacers such as glycine or P-alanine residues. A further form of variation, involves the presence of one or more amino acid residues hi peptoid form, will be well understood by those skilled hi the art. For the avoidance of doubt, "the peptoid form" is used to refer to variant amino acid residues wherein the a-carbon substituent group is on the residue's nitrogen atom<br>
rather than the a-carbon. Processes for preparing peptides in the peptoid form are known in the art, for example Simon RJ et al., PNAS (1992) 89(20), 9367-9371 and Horwell DC, Trends Biotechnol (1995) 13(4), 132-134.<br>
The nucleotide sequences for use in the present invention may include within them synthetic or modified nucleotides. A number of different types of modification to oligonucleotides are known in the art. These include methylphosphonate and phosphorothioate backbones and/or the addition of acridine or polylysine chains at the 3' and/or 5' ends of the molecule. For the purposes of the present invention, it is to be understood that the nucleotide sequences described herein may be modified by any method available in the art. Such modifications may be carried out in order to enhance the in vivo activity or life span of nucleotide sequences of the present invention.<br>
The present invention also encompasses the use of nucleotide sequences that are complementary to the sequences presented herein, or any derivative, fragment or derivative thereof. If the sequence is complementary to a fragment thereof then that sequence can be used as a probe to identify similar coding sequences in other organisms etc.<br>
Polynucleotides which are not 100% homologous to the sequences of the present invention but fall within the scope of the invention can be obtained in a number of ways. Other variants of the sequences described herein may be obtained for example by probing DNA libraries made from a range of individuals, for example individuals from different populations. In addition, other homologues may be obtained and such homologues and fragments thereof in general will be capable of selectively hybridising to the sequences shown in the sequence listing herein. Such sequences may be obtained by probing cDNA libraries^made from or_gOTprrdcJDl^iiteuT^^m ofi^.species, and^ probing such libraries with probes comprising all or part of any one of the sequences in the attached sequence listings under conditions of medium to high stringency. Similar considerations apply to obtaining species homologues and allelic variants of the polypeptide or nucleotide sequences of the invention.<br>
Variants and strain/species homologues may also be obtained using degenerate PCR which will use primers designed to target sequences within the variants and homologues encoding conserved amino acid sequences within the sequences of the present invention. Conserved sequences can be predicted, for example, by aligning the amino acid sequences from several variants/homologues. Sequence alignments can be performed using computer software known in the art. For example the GCG Wisconsin PileUp program is widely used.<br>
The primers used in degenerate PCR will contain one or more degenerate positions and will be used at stringency conditions lower than those used for cloning sequences with single sequence primers against known sequences.<br>
Alternatively, such polynucleotides may be obtained by site directed mutagenesis of characterised sequences. This may be useful where for example silent codon sequence changes are required to optimise codon preferences for a particular host cell in which the polynucleotide sequences are being expressed. Other sequence changes may be desired in order to introduce restriction enzyme recognition sites, or to alter the property or function of the polypeptides encoded by the polynucleotides.<br>
Polynucleotides (nucleotide sequences) of the invention may be used to produce a primer, e.g. a PCR primer, a primer for an alternative amplification reaction, a probe e.g. labelled with a revealing label by conventional means using radioactive or non-radioactive labels, or the polynucleotides may be cloned into vectors. Such primers, probes and other fragments will be at least 15, preferably at least 20, for example at least 25, 30 or 40 nucleotides in length, and are also encompassed by the term polynucleotides of the invention as used herein.<br>
Polynucleotides such as DNA polynucleotidesand prjpbes^^pidjng ^<br>
may be produced recombinantly, synthetically, or by any means available to those of<br>
skill in the art. They may also be cloned by standard techniques.<br>
hi general, primers will be produced by synthetic means, involving a stepwise manufacture of the desired nucleic acid sequence one nucleotide at a time. Techniques for accomplishing this using automated techniques are readily available in the art.<br>
Longer polynucleotides will generally be produced using recombinant means, for example using a PCR (polymerase chain reaction) cloning techniques. The primers may be designed to contain suitable restriction enzyme recognition sites so that the amplified DNA can be cloned into a suitable cloning vector.<br>
BIOLOGICALLY ACTIVE<br>
Preferably, the variant sequences etc. are at least as biologically active as the sequences presented herein.<br>
As used herein "biologically active" refers to a sequence having a similar structural function (but not necessarily to the same degree), and/or similar regulatory function (but not necessarily to the same degree), and/or similar biochemical function (but not necessarily to the same degree) of the naturally occurring sequence.<br>
In particular, variant sequences or modified forms thereof have a similar en2ymatic profile to the profile of the phytase identified herein. This profile includes characteristics such as being a secreted protein, having a pH optimum in the range of pH 2 to 5.5, preferably 4.0-4.5, retaining at least 50% of the maximum activity over the pH range 2.0-5.5 and/or having a specific activity over 300 U/mg.<br>
HYBRIDISATION<br>
The present invention also encompasses sequences that are complementary to the nucleic acid sequences of the present invention or sequences that are capable of hybridising either to the sequences of the present invention or to sequences that are complementary thereto.<br>
The term "hybridisation" as used herein shall include "the process by which a strand of nucleic acid joins with a complementary strand through base pairing" as well as the process of amplification as carried out in polymerase chain reaction (PCR) technologies.<br>
The present invention also encompasses the use of nucleotide sequences that are capable of hybridising to the sequences that are complementary to the sequences presented herein, or any derivative, fragment or derivative thereof.<br>
The term "variant" also encompasses sequences that are complementary to sequences that are capable of hybridising to the nucleotide sequences presented herein.<br>
Preferably, the term "variant" encompasses sequences that are complementary to sequences that are capable of hybridising under stringent conditions (e.g. 50°C and 0.2xSSC {IxSSC = 0.15 M NaCl, 0.015 M Na3citrate pH 7.0}) to the nucleotide sequences presented herein.<br>
More preferably, the term "variant" encompasses sequences that are complementary to sequences that are capable of hybridising under high stringent conditions (e.g. 65°C and O.lxSSC {IxSSC = 0.15 M NaCl, 0.015 M Na3citrate pH 7.0}) to the nucleotide sequences presented herein.<br>
The present invention also relates to nucleotide sequences that can hybridise to the nucleotide sequences of the present invention (including complementary sequences of those presented herein).<br>
The present invention also relates to nucleotide sequences that are complementary to sequences that can hybridise to the nucleotide sequences of the present invention (including complementary sequences of those presented herein).<br>
Also included within the scope of the present invention are polynucleotide sequences that are capable of hybridising to the nucleotide sequences presented herein under conditions of intermediate to maximal stringency.<br>
In a preferred aspect, the present invention covers nucleotide sequences that can hybridise to the nucleotide sequence of the present invention, or the complement thereof, under stringent conditions (e.g. 50°C and 0.2xSSC).<br>
In a more preferred aspect, the present invention covers nucleotide sequences that can hybridise to the nucleotide sequence of the present invention, or the complement thereof, under high stringent conditions (e.g. 65°C and O.lxSSC).<br>
SITE-DIRECTED MUTAGENESIS<br>
Once an enzyme-encoding nucleotide sequence has been isolated and/or purified, or a putative enzyme-encoding nucleotide sequence has been identified, it may be desirable to mutate the sequence in order to prepare an enzyme of the present invention.<br>
Mutations may be introduced using synthetic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites.<br>
A suitable method is disclosed in Morinaga et al, (Biotechnology (1984) 2, p646-649). Another method of introducing mutations into enzyme-encoding nucleotide sequences is described in Nelson and Long {Analytical Biochemistry (1989), 180, p 147-151). A further method is described in Sarkar and Sommer (Biotechniques (1990), 8, p404-407 - "The megaprimer method of site directed mutagenesis").<br>
RECOMBINANT<br>
In one aspect the sequence for use in the present invention is a recombinant sequence -i.e. a sequence that has been prepared using recombinant DNA techniques.<br>
These recombinant DNA techniques are within the capabilities of a person of ordinary skill in the art. Such techniques are explained in the literature, for example, J. _Sambrool^_E._F..Fri1s^_and T\ Maniatis, 1989, Molecular Oomng:^ALaboratory Manual, Second Edition, Books 1-3, Cold Spring Harbor Laboratory Press.<br>
SYNTHETIC<br>
In one aspect the sequence for use in the present invention is a synthetic sequence -i.e. a sequence that has been prepared by in vitro chemical or enzymatic synthesis. It<br>
h optimal codon usage for host organisms - such as the methylotrophic yeasts Pichia and Hansenula.<br>
EXPRESSION OF ENZYMES<br>
The nucleotide sequence for use in the present invention may be incorporated into a recombinant replicable vector. The vector may be used to replicate and express the nucleotide sequence, in enzyme form, in and/or from a compatible host cell.<br>
Expression may be controlled using control sequences eg. regulatory sequences.<br>
The enzyme produced by a host recombinant cell by expression of the nucleotide sequence may be secreted or may be contained intracellularly depending on the sequence and/or the vector used. The coding sequences may be designed with signal sequences which enhance direct secretion of the substance coding sequences through a particular prokaryotic or eukaryotic cell membrane.<br>
Advantageously, the enzymes of the present invention are secreted. EXPRESSION VECTOR<br>
The terms "plasmid", "vector system" or "expression vector" means a construct capable of in vivo or in vitro expression. In the context of the present invention, these constructs may be used to introduce genes encoding enzymes into host cells. Suitably, the genes whose expression is introduced may be referred to as "expressible transgenes".<br>
Preferably, the expression vector is incorporated into the genome of a suitable host organism. The term "incorporated" preferably covers stable incorporation into the genome.<br>
The nucleotide sequences described herein including the nucleotide sequence of the present invention may be present in a vector in which the nucleotide sequence is operably linked to regulatory sequences capable of providing for the expression of the nucleotide sequence by a suitable host organism.<br>
The vectors for use in the present invention may be transformed into a suitable host cell as described below to provide for expression of a polypeptide of the present invention.<br>
The choice of vector eg. a plasmid, cosmid, or phage vector will often depend on the host cell into which it is to be introduced.<br>
The vectors for use in the present invention may contain one or more selectable marker genes- such as a gene, which confers antibiotic resistance eg. ampicillin, kanamycin, chloramphenicol or tetracyclin resistance. Alternatively, the selection may be accomplished by co-transformation (as described in WO91/17243).<br>
Vectors may be used in vitro, for example for the production of RNA or used to transfect, transform, transduce or infect a host cell.<br>
Thus, in a further embodiment, the invention provides a method of making nucleotide sequences of the present invention by introducing a nucleotide sequence of the present invention into a replicable vector, introducing the vector into a compatible host cell, and growing the host cell under conditions which bring about replication of the vector.<br>
The vector may further comprise a nucleotide sequence enabling the vector to replicate in the host cell in question. Examples of such sequences are the origins of replication of plasmids pUC19, pACYC177, pUBHO, pE194, pAMBl, pUlOl, pTZ12andpETll.<br>
RTTftTTTATORY SEQUENCES<br>
In some applications, the nucleotide sequence for use in the present invention is operably linked to a regulatory sequence which is capable of providing for the expression of the nucleotide sequence, such as by the chosen host cell. By way of example, the present invention covers a vector comprising the nucleotide sequence of<br>
the present invention operably linked to such a regulatory sequence, i.e. the vector is an expression vector.<br>
The term "operably linked" refers to a juxtaposition wherein the components described are in a relationship permitting them to function in their intended manner. A regulatory sequence "operably linked" to a coding sequence is ligated in such a way that expression of the coding sequence is achieved under condition compatible with the control sequences.<br>
The term "regulatory sequences" includes promoters and enhancers and other expression regulation signals.<br>
The term "promoter" is used in the normal sense of the art, e.g. an RNA polymerase binding site.<br>
Enhanced expression of the nucleotide sequence encoding the enzyme of the present invention may also be achieved by the selection of heterologous regulatory regions, e.g. promoter, secretion leader and terminator regions.<br>
Preferably, the nucleotide sequence according to the present invention is operably linked to at least a promoter.<br>
Examples of suitable promoters for directing the transcription of the nucleotide sequence in a bacterial, fungal or yeast host are well known in the art.<br>
CONSTRUCTS<br>
The term "construct" - which is synonymous with terms such as "conjugate", "cassette"<br>
and "hybrid" - includes a nucleotide sequence for use according to the present invention directly or indirectly attached to a promoter.<br>
An example of an indirect attachment is the provision of a suitable spacer group such as an intron sequence, such as the Shl-intron or the ADH intron, intermediate the promoter and the nucleotide sequence of the present invention. The same is true for the term<br>
"fused" in relation to the present invention which includes direct or indirect attachment. In some cases, the terms do not cover the natural combination of the nucleotide sequence coding for the protein ordinarily associated with the wild type gene promoter and when they are both in their natural environment.<br>
The construct may even contain or express a marker, which allows for the selection of the genetic construct.<br>
For some applications, preferably the construct of the present invention comprises at least the nucleotide sequence of the present invention operably linked to a promoter.<br>
HOST CELLS<br>
The term "host cell" - in relation to the present invention includes any cell that comprises either the nucleotide sequence or an expression vector as described above and which is used in the recombinant production of an enzyme having the specific properties as defined herein or in the methods of the present invention.<br>
Thus, a further embodiment of the present invention provides host cells transformed or transfected with a nucleotide sequence that expresses the enzymes described in the present invention. The cells will be chosen to be compatible with the said vector and may for example be prokaryotic (for example bacterial), fungal, yeast or plant cells. Preferably, the host cells are not human cells.<br>
Examples of suitable bacterial host organisms are gram positive or gram negative bacterial species.<br>
In apreferred^embodiment, the host cell is Esgherichia Coli asi.it_hasi been pbseryed that Buttiauxella PI-29 phytase is efficiently secreted in E.coli. Phytase enzymes, including variants were characterized after heterologous expression in one or more of the following expression hosts: Escherichia coli K12; Bacillus subtilis; Saccharomyces cerevisiae. These hosts are therefore also preferre.<br>
Depending on the nature of the nucleotide sequence encoding the enzyme of the present invention, and/or the desirability for further processing of the expressed protein, eukaryotic hosts such as yeasts or other fungi may be preferred. In general, yeast cells are preferred over fungal cells because they are easier to manipulate. However, some proteins are either poorly secreted from the yeast cell, or in some cases are not processed properly (e.g. hyperglycosylation in yeast). In these instances, a different fungal host organism should be selected.<br>
The use of suitable host cells - such as yeast, fungal and plant host cells - may provide for post-translational modifications (e.g. myristoylation, glycosylation, truncation, lapidation and tyrosine, serine or threonine phosphorylation) as may be needed to confer optimal biological activity on recombinant expression products of the present invention.<br>
The host cell may be a protease deficient or protease minus strain. The genotype of the host cell may be modified to improve expression.<br>
Examples of host cell modifications include protease deficiency, supplementation of rare tRNA's, and modification of the reductive potential in the cytoplasm to enhance disulphide bond formation.<br>
For example, the host cell E. coli may overexpress rare tRNA's to improve expression of heterologous proteins as exemplified/described hi Kane (Curr Opin Biotechnol (1995), 6, 494-500 "Effects of rare codon clusters on high-level expression of heterologous proteins in E.coli"}. The host cell may be deficient in a number of reducing enzymes thus favouring formation of stable disulphide bonds as exemplified/described in Bessette (Proc NatlAcadSci USA (1999), 96,13703-13708 " Efficient folding of proteins with multiple disulphide bonds hi the Escherichia coli cytoplasm").<br>
In one embodiment, host cells in the context of the present invention include those cells that can be added directly into animal feed.<br>
ORGANISM<br>
The term "organism" in relation to the present invention includes any organism that could comprise the nucleotide sequence coding for the enzymes as described in the present invention and/or_ products obtained therefrom, and/or wherein a rjromoter can allow expression of the nucleotide sequence according to the present invention when present in the organism.<br>
Suitable organisms may include a prokaryote, fungus, yeast or a plant.<br>
The term "transgenic organism" in relation to the present invention includes any organism that comprises the nucleotide sequence coding for the enzymes as described in the present invention and/or the products obtained therefrom, and/or wherein a promoter can allow expression of the nucleotide sequence according to the present invention within the organism. Preferably the nucleotide sequence is incorporated in the genome of the organism.<br>
The term "transgenic organism" does not cover native nucleotide coding sequences in their natural environment when they are under the control of their native promoter which is also in its natural environment.<br>
Therefore, the transgenic organism of the present invention includes an organism comprising any one of, or combinations of, the nucleotide sequence coding for the enzymes as described in the present invention, constructs according to the present invention, vectors according to the present invention, plasmids according to the present invention, cells according to the present invention, tissues according to the present invention, or the products thereof.<br>
For example the transgenic organism may also comprise the nucleotide sequence coding for the enzyme of the present invention under the control of a heterologous promoter.<br>
TRANSFORMATION OF HOST CELLS/ORGANISM<br>
As indicated earlier, the host organism can be a prokaryotic or a eukaryotic organism. Examples of suitable prokaryotic hosts include E. coll and Bacillus subtilis.<br>
Teachings on the transformation of prokaryotic hosts is well documented in the art, for example see Sambrook et al (Molecular Cloning: A Laboratory Manual, 2nd edition, 1989, Cold Spring Harbor Laboratory Press). Other suitable methods are set out in the Examples herein. If a prokaryotic host is used then the nucleotide sequence may need to be suitably modified before transformation - such as by removal of introns.<br>
Filamentous fungi cells may be transformed using various methods known in the art-such as a process involving protoplast formation and transformation of the protoplasts followed by regeneration of the cell wall in a manner known. The use ofAspergillus as a host microorganism is described in EP 0 238 023.<br>
Another host organism can be a plant. A review of the general techniques used for transforming plants may be found in articles by Potrykus (Annu Rev Plant Physiol Plant Mol Biol [1991] 42:205-225) and Christou (Agro-Food-Industry Hi-Tech March/April 1994 17-27). Further teachings on plant transformation may be found in EP-A-0449375.<br>
General teachings on the transformation of fungi, yeasts and plants are presented in following sections.<br>
TRANSFORMED FUNGUS<br>
A host organism may be a fungus - such as a filamentous fungus. Examples of suitable such hosts include any member belonging to the genera Thermomyces, Acremonium, Aspergillus, Penicillium, Mucor, Neurospora, Trichoderma and the like.<br>
Teachings on transforming filamentous fungi are reviewed in US-A-5741665 which states that standard techniques for transformation of filamentous fungi and culruring<br>
the fungi are well known in the art. An extensive review of techniques as applied to N. crassa is found, for example in Davis and de Serres, Methods Enzymol (1971) 17A: 79-143.<br>
Further teachings_on transforming filamentous fungi are reviewed in US-A-5674707.<br>
In one aspect, the host organism can be of the genus Aspergillus, such as Aspergillus niger.<br>
A transgenic Aspergillus according to the present invention can also be prepared by following, for example, the teachings of Turner G. 1994 (Vectors for genetic manipulation. In: Martinelli S.D., Kinghorn J.R.( Editors) Aspergillus: 50 years on. Progress in industrial microbiology vol 29. Elsevier Amsterdam 1994. pp. 641-666).<br>
Gene expression in filamentous fungi has been reviewed in Punt et al. (2002) Trends Biotechnol 2002 May;20(5):200-6, Archer &amp; Peberdy Grit Rev Biotechnol (1997) 17(4):273-306.<br>
TRANSFORMED YEAST<br>
In another embodiment, the transgenic organism can be a yeast.<br>
A review of the principles of heterologous gene expression in yeast are provided in, for example, Methods Mol Biol (1995), 49:341-54, and Curr Opin Biotechnol (1997) Oct;8(5):554-60<br>
In this regard, yeast - such as the species Saccharomyces cerevisi or Pichia pastoris (see<br>
EMS MicrobioLRev4200jQ Jgene expression.<br>
A review of the principles of heterologous gene expression in Saccharomyces cerevisiae and secretion of gene products is given by E Hinchcliffe E Kenny (1993, "Yeast as a vehicle for the expression of heterologous genes", Yeasts, Vol 5, Anthony H Rose and J Stuart Harrison, eds, 2nd edition, Academic Press Ltd.).<br>
For the transformation of yeast, several transformation protocols have been developed. For example, a transgenic Saccharomyces according to the present invention can be prepared by following the teachings of Hinnen et al, (1978, Proceedings of the National Academy of Sciences pfthe^ USA 75, 1929); Beggs, J D (1978, Nature, JLondon, 275. 104); and Ito, H eta! (1983, J Bacteriology 153,163-168).<br>
The transformed yeast cells may be selected using various selective markers - such as auxotrophic markers dominant antibiotic resistance markers.<br>
TRANSFORMED PLANTS/PLANT CELLS<br>
A host organism suitable fpr the present invention may be a plant. A review of the general techniques may be found in articles by Potrykus (Annu Rev Plant Physiol Plant Mol Biol [1991] 42:205-225) and Christou (Agro-Food-Industry Hi-Tech March/April 1994 17-27).<br>
CULTURING AND PRODUCTION<br>
Host cells transformed with the nucleotide sequence of the present invention may be cultured under conditions conducive to the production of the encoded enzyme and which facilitate recovery of the enzyme from the cells and/or culture medium.<br>
The medium used to cultivate the cells may be any conventional medium suitable for growing the host cell in questions and obtaining expression of the enzyme.<br>
The protein produced by a recombinant cell may be displayed on the surface of the ,ce!L,<br>
The enzyme may be secreted from the host cells and may conveniently be recovered from the culture medium using well-known procedures.<br>
SECRETION<br>
It may be desirable for the enzyme to be secreted from the expression host into the culture medium from where the enzyme may be more easily recovered. According to the present invention, the secretion leader sequence may be selected on the basis of the desired expression host. Hybrid signal sequences may also be used with the context of the present invention.<br>
Typical examples of heterologous secretion leader sequences are those originating from the fungal amyloglucosidase (AG) gene (glaA - both 18 and 24 amino acid versions e.g. from Aspergillus), the a-factor gene (yeasts e.g. Saccharomyces, Kluyveromyces and Hansenuld) or the a-amylase gene (Bacillus).<br>
By way of example, the secretion of heterologous proteins in E. coli is reviewed in Methods Enzymol (1990) 182:132-43.<br>
DETECTION<br>
A variety of protocols for detecting and measuring the expression of the amino acid sequence are known in the art. Examples include enzyme-linked immunosorbent assay (ELISA), radioimmunoassay (RIA) and fluorescent activated cell sorting (FACS).<br>
A wide variety of labels and conjugation techniques are known by those skilled in the art and can be used in various nucleic and amino acid assays.<br>
A number of companies such as Pharmacia Biotech (Piscataway, NJ), Promega (Madison, WI), and US Biochemical Corp (Cleveland, OH) supply commercial kits and protocols for these procedures.<br>
Suitable reporter molecules or labels include those radionuclides, enzymes, fluorescent, chemiluniinescent, or chromogenic agents as well as substrates, cofactors, inhibitors, magnetic particles and the like. Patents teaching the use of such labels<br><br>
include US-A-3,817,837; US-A-3,850,752; US-A-3,939,350; US-A-3,9965345; US-A-4,277,437; US-A-4,2753149 and US-A-4,3 66,241.<br>
Also, recombinant immunoglobulins may be produced as shown in US-A-4,816,567.<br>
Other suitable assays for detecting phytase activity are known in the art and exemplified herein.<br>
FUSION PROTEINS<br>
The amino acid sequence for use according to the present invention may be produced as a fusion protein, for example to aid in extraction and purification. Examples of fusion protein partners include glutathione-S-transferase (GST), 6xHis, GAL4 (DNA binding and/or transcriptional activation domains) and (p-galactosidase). It may also be convenient to include a proteolytic cleavage site between the fusion protein partner and the protein sequence of interest to allow removal of fusion protein sequences.<br>
Preferably, the fusion protein will not hinder the activity of the protein sequence.<br>
Gene fusion expression systems in E. coli have been reviewed hi Curr Opin Biotechnol (1995) 6(5):501-6.<br>
In another embodiment of the invention, the amino acid sequence may be ligated to a heterologous sequence to encode a fusion protein. For example, for screening of peptide libraries for agents capable of affecting the substance activity, it may be useful to encode a chimeric substance expressing a heterologous epitope that is recognised by a commercially available antibody.<br><br>
ADDITIONAL SEQUENCES<br>
The sequences for use according to the present invention may also be used in .-conjunctionwith, pne^prmprejidditional nroteins of interest (POIs) or nucleotide sequences of interest (NOIs).<br>
Non-limiting examples of POIs include: Xylanase, Upases, acid phosphatases and/or others. These include enzymes that, for example, modulate the viscosity of the feed. The NOI may even be an antisense sequence for any of those sequences.<br>
The POI may even be a fusion protein, for example to aid in extraction and purification or to enhance in vivo phytate metabolism.<br>
The POI may even be fused to a secretion sequence.<br>
Other sequences can also facilitate secretion or increase the yield of secreted POL Such sequences could code for chaperone proteins as for example the product of Aspergillus niger cyp B gene described in UK patent application 9821198.0.<br>
The NOI coding for POI may be engineered in order to alter their activity for a number of reasons, including but not limited to, alterations, which modify the processing and/or expression of the expression product thereof. By way of further example, the NOI may also be modified to optimise expression in a particular host cell. Other sequence changes may be desired in order to introduce restriction enzyme recognition sites.<br>
jrhe-NOLcjodingJortiiePOIjiiay include wjnj.tjyntheticor modified nucleotides-such as methylphosphonate and phosphorothioate backbones<br>
The NOI coding for the POI may be modified to increase intracellular stability and half-life. Possible modifications include, but are not limited to, the addition of flanking sequences of the 5' and/or 3' ends of the molecule or the use of<br><br>
phosphorothioate or 2' 0-methyl rather than phosphodiesterase linkages within the backbone of the molecule.<br>
ANTIBODIES<br>
One aspect of the present invention relates to amino acids that are immunologically reactive with the amino acid of SEQ ID No. 3.<br>
Antibodies may be produced by standard techniques, such as by immunisation with the substance of the invention or by using a phage display library.<br>
For the purposes of this invention, the term "antibody", unless specified to the contrary, includes but is not limited to, polyclonal, monoclonal, chimeric, single chain, Fab fragments, fragments produced by a Fab expression library, as well as mimetics thereof. Such fragments include fragments of whole antibodies which retain their binding activity for a target substance, Fv, F(ab') and F(ab')2 fragments, as well as single chain antibodies (scFv), fusion proteins and other synthetic proteins which comprise the antigen-binding site of the antibody. Furthermore, the antibodies and fragments thereof may be humanised antibodies. Neutralising antibodies, i.e., those which inhibit biological activity of the substance polypeptides, are especially preferred for diagnostics and therapeutics.<br>
If polyclonal antibodies are desired, a selected mammal (e.g., mouse, rabbit, goat, horse, etc.) is immunised with the sequence of the present invention (or a sequence comprising an immunological epitope thereof). Depending on the host species, various adjuvants may be used to increase immunological response.<br>
^Serum jEroni JheL_nnmunijsed_anm.aLis__cpllected_and treated according to known procedures. If serum containing polyclonal antibodies to the sequence of the present invention (or a sequence comprising an immunological epitope thereof) contains antibodies to other antigens, the polyclonal antibodies can be purified by immunoaffinity chromatography. Techniques for producing and processing polyclonal antisera are known in the art. In order that such antibodies may be made,<br><br>
the invention also provides polypeptides of the invention or fragments thereof haptenised to another polypeptide for use as immunogens in animals or humans.<br>
Monoclonal antibodies directed against the sequence of the present invention (or a sequence comprisingjin immunological epitope thereof) can also be readily produced by one skilled in the art and include, but are not limited to, the hybridoma technique Koehler and Milstein (1975 Nature 256:495-497), the human B-cell hybridoma technique (Kosbor et al, (1983) Immimol Today 4:72; Cote et al, (1983) Proc Natl Acad Sci 80:2026-2030) and the EBV-hybridoma technique (Cole et al, (1985) Monoclonal Antibodies and Cancer Therapy, Alan Rickman Liss Inc, pp 77-96).<br>
In addition, techniques developed for the production of "chimeric antibodies", the splicing of mouse antibody genes to human antibody genes to obtain a molecule with appropriate antigen specificity and biological activity may be used (Morrison et al., (1984) Proc Natl Acad Sci 81:6851-6855; Neuberger et al., (1984) Nature 312:604-608; Takeda et al, (1985) Nature 314:452-454).<br>
Alternatively, techniques described for the production of single chain antibodies (US Patent No. 4,946,779) can be adapted to produce the substance specific single chain antibodies.<br>
Antibody fragments which contain specific binding sites for the substance may also be generated. For example, such fragments include, but are not limited to, the F(ab')2 fragments which can be produced by pepsin digestion of the antibody molecule and the Fab fragments which can be generated by reducing the disulfide bridges of the F(ab')2 fragments. Alternatively, Fab expression libraries may be constructed to allow rapid and easy identification of monoclonal Fab fragments with the desired specificity (Huse WD et al, (1989) Science 256:1275-128 1).<br>
LARGE SCALE APPLICATION<br>
In one preferred embodiment of the present invention, the amino acid sequence encoding a C. freundii-deiivQd phytase or the methods of the present invention are used for large scale applications. In particular, the methods of the present invention<br><br>
may be used for the large scale production of phytases for industrial use as additives/supplements to food or feed compositions.<br>
Preferably the amino acid sequence is produced in a quantity of from 5 g per litre to<br>
_ about JO g .per litre^Oheto^	of the host<br>
organism.<br>
Preferably the amino acid sequence is produced in a quantity of from lOOmg per litre to about 900mg per litre of the total cell culture volume after cultivation of the host organism.<br>
Preferably the amino acid sequence is produced in a quantity of from 250mg per litre to about SOOmg per litre of the total cell culture volume after cultivation of the host organism.<br>
USE OF PHYTASES<br>
As stated above, the present invention also relates to the production of phytases as described herein.<br>
In particular, the present invention also relates to the use of the amino acid sequences as disclosed herein in the production of organic and inorganic phosphate compounds.<br>
Thus, the present invention further relates to the use of the nucleotide sequences encoding phytases in generating expression vectors or systems for the expression of the phytases.<br>
In addition, the present invention relates to the use j)f such expression vectors_Qr_ systems in the generation of host cells which express phytases.<br>
The invention further relates to the use of modified host cells in the generation of precursors of organic and inorganic phosphate compounds or in the generation of specific organic phosphate compounds.<br><br>
Suitable organic and inorganic phosphate compounds include myo-inositol pentakis-, tetrakis-, tris-, bis- and monophosphates.<br>
Suitably, the invention therefore provides a method of producing an organic ^phosphate,, compoundj:ompnsirig Jreating^a^phytate with a phytase derived from Buttiauxella sp.. Preferably, the method is characterised in that the enzyme comprises the ammo acid sequences shown as SEQ ID NOs: 3 or a sequence having at least 75% identity (homology) thereto or an effective fragment, or modified form thereof. Suitably, the organic phosphate is phytate or all possible stereoisomers of myo-inositol di-, tri-, tetra, and pentaphosphates. Other suitable organic phosphates include inositol-tetraphosphates and inositol-oligophosphates. In a preferred embodiment, the method is an in vivo biotechnological process.<br>
Such methods for producing an organic phosphate compound may suitably comprise the steps of:<br>
a)	providing   a   host   cell   that   comprises   expressible   transgenes   comprising<br>
Buttiauxella sp. phytase;<br>
b)	culturing the transgenic organism under conditions suitable for expression of the<br>
transgene; and<br>
c)	recovering the organic phosphate compound from the culture.<br>
The compounds can be used for a number of applications including in assays for the characterisation of phytases. Some inositol phosphates are involved as signal molecules in intracellular regulation and can be used research chemicals.<br>
In another aspect there is provided a method for production of food or animal feed.<br>
Animal feed is typically produced in feed mills in which raw materials are first<br>
_grpjjnd tp^a ^suitable pa^^^	appropriate additives.,. The feed<br>
may then be produced as a mash or pellets; the later typically involves a method by which the temperature is raised to a target level and then the feed is passed through a die to produce pellets of a particular size. Subsequently liquid additives such as fat and enzyme may be added. The pellets are allowed to cool prior to transportation. Production of animal feed may also involve an additional step that includes extrusion or expansion prior to pelleting.<br><br>
Accordingly, the invention further provides the use of an amino acid sequence<br>
encoding a phytase or a host cell expressing a phytase to produce a phytase for use in<br>
1 the manufacture of a food or feed product. In one aspect, there is provided a use of an<br>
^ amuio_acid^sequence_as^described herein in the manufacture of a food or feed product.<br>
In another aspect, there is provided a use of a host cell in accordance with the<br>
invention in the manufacture of a food or feed product.   In another aspect, there is<br>
provided a use of an expression vector or system in accordance with the invention in<br>
the manufacture of a food or feed product.<br>
The present invention also covers using the enzymes as a component of feed combinations with other components to deliver to animals.<br>
COMBINATION WITH OTHER COMPONENTS<br>
The enzymes of the present invention may be used in combination with other components or carriers.<br>
Suitable carriers for feed enzymes include wheat (coarsely ground). In addition there are a number of encapsulation techniques including those based on fat/wax coverage, adding plant gums etc.<br>
Examples of other components include one or more of: thickeners, gelling agents, emulsifiers, binders, crystal modifiers, sweetners (including artificial sweeteners), rheology modifiers, stabilisers, anti-oxidants, dyes, enzymes, carriers, vehicles, excipients, diluents, lubricating agents, flavouring agents, colouring matter, suspending agents, disintegrants, granulation binders etc. These other components _rnay/be_natural. These other comnonents may be prepared by use of chemical and/or enzymatic techniques.<br>
As used herein the term "thickener or gelling agent" as used herein refers to a product that prevents separation by slowing or preventing the movement of particles, either droplets of immiscible liquids, air or insoluble solids.<br><br>
The term "stabiliser" as used here is defined as an ingredient or combination of ingredients that keeps a product (e.g. a food product) from changing over time.<br>
The term "emulsifier" as used herein refers to an ingredient (e.g. a food product -ingredient)-thatprevents-the,separatianjof emulsions.<br>
As used herein the term "binder" refers to an ingredient (e.g. a food ingredient) that binds the product together through a physical or chemical reaction.<br>
The term "crystal modifier" as used herein refers to an ingredient (e.g. a food ingredient) that affects the crystallisation of either fat or water.<br>
"Carriers" or "vehicles" mean materials suitable for compound administration and include any such material known in the art such as, for example, any liquid, gel, solvent, liquid diluent, solubiliser, or the like, which is non-toxic and which does not interact with any components of the composition in a deleterious manner.<br>
Examples of nutritionally acceptable carriers include, for example, grain, water, salt solutions, alcohol, silicone, waxes, petroleum jelly, vegetable oils, and the like.<br>
Examples of excipients include one or more of: microcrystalline cellulose and other celluloses, lactose, sodium citrate, calcium carbonate, dibasic calcium phosphate, glycine, starch, milk sugar and high molecular weight polyethylene glycols.<br>
Examples of disintegrants include one or more of: starch (preferably corn, potato or tapioca starch), sodium starch glycollate, croscarmellose sodium and certain complex silicates.<br>
Examples of granulation binders include one or more of: polyvinylpyrronaone, hydroxypropylmethylcellulose (HPMC), hydroxypropylcellulose (HPC), sucrose, maltose, gelatin and acacia.<br>
Examples of lubricating agents include one or more of: magnesium stearate, stearic acid, glyceryl behenate and talc.<br><br>
Examples of diluents include one or more of: water, ethanol, propylene glycol and glycerin, and combinations thereof.<br>
The other components may be used simultaneously (e.g when they are in admixture together or even when they are delivered by different routes) or sequentially (e.g they may be delivered by different routes).<br>
As used herein the term "component suitable for animal or human consumption" means a compound which is or can be added to the composition of the present invention as a supplement which may be of nutritional benefit, a fibre substitute or have a generally beneficial effect to the consumer.<br>
By way of example, the components may be prebiotics such as alginate, xanthan, pectin, locust bean gum (LEG), inulin, guar gum, galacto-oligosaccharide (GOS), fructo-oligosaccharide (FOS), lactosucrose, soybean oligosaccharides, palatinose, isomalto-oligosaccharides, gluco-oligosaccharides and xylo-oligosaccharides.<br>
FOOD OR FEED SUBSTANCE<br>
The compounds may be used as - or in the preparation of - a food or feed substance. Here, the term "food" is used in a broad sense - and covers food and food products for humans as well as food for animals (i.e. a feed). The term "feed" is used with reference to products that are fed to animals in the rearing of livestock. In a preferred aspect, the food or feed is for consumption by monogastric animals such as pig, poultry and fish.<br>
The food or feed mav be in the form of a solution or as a solid - depending on the use and/or the mode of application and/or the mode of administration.<br>
FOOD AND FEED INGREDIENTS AND SUPPLEMENTS<br>
The compounds may be used as a food or feed ingredient.<br><br>
As used herein the term "food or feed ingredient" includes a formulation, which is or can be added to foods or foodstuffs and includes formulations which can be used at low levels in a wide variety of products.<br>
,The food ingredient may be ir^tiie form of a solution or as a solid - depending on the<br>
*<br>
use and/or the mode of application and/or the mode of administration. The compounds may be - or may be added to - food supplements. FOODS AND FEED COMPOSITIONS<br>
Feed compositions for monogastric animals typically include compositions comprising plant products which contain phytate. Such compositions include cornmeal, soybean meal, rapeseed meal, cottonseed meal, maize, wheat, barley and sorghum-based feeds.<br>
The phytases described herein may be - or may be added to - foods or feed substances and compositions.<br>
The present Invention also provides a method of preparing a food or a feed ingredient or supplement, the method comprising admixing phytases produced by the process of the present invention or the composition according to the present invention with another food ingredient. The method for preparing or a food ingredient is also another aspect of the present invention. Methods for preparing animal feed are set out above. The enzyme can be added also in the form of a solid formulation, or as a feed additive, such as a pre-mix. A solid form is typically added before or during the mixing step; and a liquid form is typically added after the pelleting step.<br>
PHARMACEUTICAL<br>
The phytases of the present invention may also be used in pharmaceutical preparations or for combination into food stuffs in order to provide some pharmaceutical effect. For example, EP 1,389,915 describes the use of a phytase in a<br><br>
food or drink for increasing the availability of Calcium, Iron and/or Zinc of the food or drink for humans.<br>
In addition,  EP   1,392,353  describes a medicament  or nutritional  supplement _ containing phytase,jwMchisjJseMfQrjncreasm calcium and iron, and for combating deficiency diseases.<br>
Here, the term "pharmaceutical" is used in a broad sense - and covers Pharmaceuticals and/or nutraceuticals for humans as well as pharmaceuticals and/or nutraceuticals for animals (i.e. veterinary applications). In a preferred aspect, the pharmaceutical is for human use and/or for animal husbandry.<br>
The pharmaceutical can be for therapeutic purposes — which may be curative or palliative or preventative in nature. The pharmaceutical may even be for diagnostic purposes.<br>
When used as - or in the preparation of - a pharmaceutical, the product and/or the compounds of the present invention may be used in conjunction with one or more of: a pharmaceutically acceptable carrier, a pharmaceutically acceptable diluent, a pharmaceutically acceptable excipient, a pharmaceutically acceptable adjuvant, a pharmaceutically active ingredient.<br>
The pharmaceutical may be in the form of a solution or as a solid - depending on the use and/or the mode of application and/or the mode of administration.<br>
PHARMACEUTICAL INGREDIENT<br>
The product and/or the compounds of the present invention may be used as pharmaceutical ingredients. Here, the product and/or the composition of the present invention may be the sole active component or it may be at least one of a number (i.e. 2 or more) active components.<br><br>
The pharmaceutical ingredient may be in the form of a solution or as a solid -depending on the use and/or the mode of application and/or the mode of administration.<br>
.The, pharmaceutical Jngredienynay bein the form of an effervescent product to improve the dissolving properties of the pharmaceutical.<br>
FORMS<br>
The product and/or the compounds of the present invention may be used in any suitable form - whether when alone or when present in a composition. Likewise, phytases produced in accordance with the present invention (i.e. ingredients - such as food ingredients, functional food ingredients or pharmaceutical ingredients) may be used in any suitable form.<br>
Suitable examples of forms include one or more of: tablets, pills, capsules, ovules, solutions or suspensions, which may contain flavouring or colouring agents, for immediate-, delayed-, modified-, sustained-, pulsed- or controlled-release applications.<br>
By way of example, if the product and/or the composition are used hi a tablet form -such as for use as a functional ingredient - the tablets may also contain one or more of; excipients, disintegrants, granulation binders, or lubricating agents.<br>
Examples of nutritionally acceptable carriers for use in preparing the forms include, for example, water, salt solutions, alcohol, silicone, waxes, petroleum jelly and the like.<br>
Preferred excipients for the forms include lactose, starch, a cellulose, milk sugar or high molecular weight polyethylene glycols.<br>
For aqueous suspensions and/or elixirs, carotenoid cleavage compounds may be combined with various sweetening or flavouring agents, colouring matter or dyes,<br><br>
with emulsifying and/or suspending agents and with diluents such as water, ethanol, propylene glycol and glycerin, and combinations thereof.<br>
The forms may also include gelatin capsules; fibre capsules, fibre tablets etc. GENERAL RECOMBINANT DNA METHODOLOGY TECHNIQUES<br>
The present invention employs, unless otherwise indicated, conventional techniques of chemistry, molecular biology, microbiology, recombinant DNA and immunology, which are within the capabilities of a person of ordinary skill in the art. Such techniques are explained hi the literature. See, for example, J. Sambrook, E. F. Fritsch, and T. Maniatis, 1989, Molecular Cloning: A Laboratory Manual, Second Edition, Books 1-3, Cold Spring Harbor Laboratory Press; Ausubel, F. M et al. (1995 and periodic supplements; Current Protocols in Molecular Biology, ch. 9, 13, and 16, John Wiley &amp; Sons, New York, N.Y.); B. Roe, J. Crabtree, and A. Kahn, 1996, DNA Isolation and Sequencing: Essential Techniques, John Wiley &amp; Sons; M. J. Gait (Editor), 1984, Oligonucleotide Synthesis: A Practical Approach, Irl Press; and, D. M. J. Lilley and J. E. Dahlberg, 1992, Methods of Enzymology: DNA Structure Part A: Synthesis and Physical Analysis of DNA Methods in Enzymology, Academic Press. Each of these general texts is herein incorporated by reference.<br>
EXAMPLES<br>
The invention is now further illustrated in the following non-limiting examples. Example 1. Phytase activity assay,<br>
Micro-1 = microliter<br>
Phytase assays were carried out in microtitre plates. The reaction mixture (100 micro-1) contained: 2 mM phytate and 0.8 mM CaCl2 in 200 mM sodium acetate buffer, pH 3.5 -Tie reaction was allowed to proceed for 1 h at 37°C after which tune the released phosphate was measured by a refinement of a known procedure (Heinonen<br><br>
J.K., Lahti RJ. Anal Biochem. 113 (2), 313-317 (1981)). Briefly, 200 micro-1 of a freshly prepared AMM solution (7.5 N HaSO^ 15 mM ammonium molybdate and acetone - 1:1:2) was added to the 100 micro-1 reaction mixture in each microtitre plate well. The absorbance at 390 nm was measured not earlier than 10 min and not later thanjtijmin. after addition j^<br>
determined by building a calibration curve with phosphate solutions of known concentrations. For assaying phytase activity at different pH values the following (all 200 mM) buffers were used: glycine/HCl between pH 2.0 and 3.0, sodium acetate/acetic acid between pH 3.5 and 5.5, Tris/maleic acid between pH 6.0 and 7.5.<br>
Example 2.  Phytase-producing strain Pl-29.<br>
Bacterial strain Pl-29 was originally isolated from a mass of decaying plant material collected from a bottom of a puddle in the forest of Southern Finland. The strain can be aerobically cultivated at 30°C on many simple culture media e.g. LB (1% peptone, 0.5% yeast extract, 1% NaCl, pH 7.4) or low phosphate medium PP1 (1% peptone, 1%, beef extract, 0.5%, yeast extract, CaC^ - 0.2M. The medium is prepared by adjusting the pH to about 11 with NaOH and boiling for 10 min. The precipitate is removed by filtration, pH re-adjusted to 5.5 and the medium sterilised by autoclaving forl5minatl21°C).<br>
After growth in liquid PP1 medium the strain was found to exhibit phytase activity both at pH 3.5 and 5.5 (assayed as described in Example 1). The ratio of activities at 3.5 and 5.5 was about 1.3. The activity was also measured separately in the cells and culture supernatant of P3-42. According to these measurements, most of all phytase activity was cell-bound with 10-20% of activity found in culture medium supernatant.<br>
=TJhej3tram is depositedjwitLNCT Example 3. Isolation of chromosomal DNA from the strain Pl-29.<br>
Chromosomal DNA was prepared essentially by the standard procedure (Ausubel et al, Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York, 1996). A 250 ml culture grown overnight at 30°C in LB medium was centrifuged at 10, 000<br><br>
rpm for 30 min, washed in 20 ml of 50 mM tris-HCl, 5 mM EDTA pH 8 and re-suspended in 10 ml of cold TES ( 50 mM tris-HCl, 5 mM EDTA, 15 % glucose pH 8). Lysozyme was added to 10 mg/ml, and the cell suspension was incubated at 37°C for 30 - 60 min until lysis occurred, ascertained for by dilution of 100 micro-1 of the Teaction mixture into 1ml of4 %^SDS-and checking for-a^slimy"-Consistency^At this time, SDS and Proteinase K (Sigma) were added to a final concentration of 1% and 0.5 mg/ml respectively. The reaction mixture was incubated for 30 min at 56 °C followed by addition of 2 ml of 5 M NaCl and 1.4 ml 10% cetyltrimethylammonium bromide (Sigma). The incubation was continued for 15 min 65 °C. The solution was extracted once with chloroform/isoamyl alcohol (24:1) and once with phenol/chloroform. After the extractions, the water phase was mixed with 0.6 vol of isopropanol, the DNA precipitate collected by centrifugation (10 000 rpm, 15 min), washed with 70 % ethanol, vacuum dried and re-suspended in 2 ml of 10 mM tris-HCl, 1 mM EDTA pH 8, 5 micro-g/ml. RNAse.<br>
Example 4. Taxomomic identification of the bacterial strain Pl-29.<br>
A fragment of the 16S rRNA gene of the strain Pl-29 was amplified by the<br>
polymerase chain reaction (PCR) with Tag DNA polymerase (Roche) using the<br>
primers;	536f	(CAGCMGCCGCGGTAATWC)	and	1392r<br>
(ACGGGCGGTGTGTRC), (Lane, D. J. In Nucleic acid techniques in bacterial systematics, Stackbrandt, E. and Goodfellow, M. eds, John Wiley &amp; Sons, New York: pp 115-117 (1991)). The following program was used: 1) initial DNA denaturation step of 5 min at 95 °C; 2) 30 cycles of 1 min at 94°C, 1 min at 55°C, 1 min at 72°C; 3) a final extension step of 70 °C for 10 min. The PCR products, approximately 900 base pairs in size, were purified by electrophoresis in a 0.8 % agarose gel and extracted from the gel using a Gel Purification Kit (Qiagen) according to the manufaetoer-s4nstractions.<br>
(Norway) as a commercial service. The sequenced area is listed as SEQ ID No 1. This sequence was compared to DNA sequences in the GenBank database (http://www.ncbi.nlni.nih.gov/blastyr). The highest matches (688-689 out of 691 nucleotides, 99,6- 99.7%) were found with the sequences of 16S RNA gene from several Buttiauxella stains such as B. izardii DSM 9397, B. gaviniae DSM 9393 and<br><br>
B. noackiae ATCC 51607T . Therefore, strain PI-29 can be taxonomically classified as Buttiauxella sp.<br>
Example 5. Cloning of the phytase gene from Buttiauxella sp. Pl-29.<br>
Chromosomal DNA from the Buttiauxella sp. Pl-29 has been partially digested with restriction endonuclease Sau3A. and the digest fractionated on 1% agarose gel. The DNA fragments of 3 to 5 kb were isolated from the gel using a gel Purification Kit (Qiagen) and ligated with 5flwHI-digested dephosphorylated lambda-ZAP arms (Stratagene).  Subsequent steps for library construction followed the instructions of Stratagene's   ZAP Express Predigested Vector/Gigapack Cloning Kit. The phage form of the library was converted into a plasmid form by the "mass excision" procedure as described by the manufacturer (Stratagene). Screening of plasmid library was done by similarly to the earlier published methods for the detection of phytase activity on Petri plates (Howson and Davis. Enzyme Microb. Technol. 5, 377-382 (1983); Chen J.C. Biotechnology techniques 12 (10) 751-761 (1998); Riccio MX. et al, J. Appl. Microbiol. 82, 177-185 (1997)).   Several phytase-positive clones were isolated and purified by sub-cloning. These isolates were grown in liquid culture (LB medium at 30°C and 200 rpm for about 24 h) and phytase activity was measured (Example 1) in the resulting cell suspensions. One clone that had the highest phytase activity (about 1.2 U/ml at pH 3.5) was selected for subsequent characterisation. Plasmid DNA was isolated from this clone, named pBK(Pl-29), and characterised by partial DNA sequencing  of the insert DNA (sequencing service was obtained from Medprobe (Norway)).  The sequence comprising the phytase gene is listed as SEQ ID No: 2. The deduced sequence of the Buttiauxella sp. Pl-29 phytase is listed as SEQ ID No: 3. Comparison of the SEQ ID No: 3 with the sequences in GenBank using the BLAST service provided by NCBI (litrp://www.ncbi.:tu'm.nih.gov/blastA identified the 4)hy^aseJromj9M£wra&amp;^<br>
closest known homologue of the Buttiauxella sp. Pl-29 phytase. However, the level of homology is rather low - only about 74% of amino acid residues are identical in both proteins.<br><br>
Example 6. Amplification and expression of phytase gene from Buttiauxella sp. Pl-29<br>
=Phytase gene was ampjified bv PCR. Chromosomal DNA of the strain Buttiauxella<br>
sp. Pl-29 was used as template and oligonucleotides o29-5<br>
(GGAATTCATATGACGATCTCTGCGTTTAAC)	and	o29-3<br>
(GGAATTCGGATCCTTA-     CTGTAGCTGGCAGCCTG)     as     primers.     The amplification was carried out using the Expand High Fidelity PCR System kit (Roche). The following program was used: 1) initial DNA denaturation for 3 min at 94 °C; 2) 35 cycles of 45 sec at 94 °C, 45 sec at 55 °C, 1 min at 68 °C3 1 min at 72 °C, 1 min at 74 °C; 3) a final extension step of 10 min at 72°C. The resulting PCR product was purified by electrophoresis in a 0.8 % agarose gel followed by DNA extraction from the gel using a Gel Purification Kit (Qiagen). The purified PCR product was digested with the restriction enzymes Ndel and BarriHl and isolated from the reaction mixture by the PCR Purification Kit (Qiagen). The vector plasmid pETl la (Novagen) was digested with the restriction endonucleases Ndel and BamHL, de-phosphorylated using shrimp alkaline phosphatase (Roche) and purified by electrophoresis in a 0.8 % agarose gel. The linearised plasmid DNA band was excised from the gel and purified using a Gel Purification Kit (Qiagen). The two purified DNA fragments were ligated using T4 DNA ligase   (Roche). The ligation reaction was precipitated with 70% ethanol, washed with ethanol and re-suspended directly into 50 micro-1 of electrocompetent E. coli XLl-Blue MRF' cells. The suspension was transferred to a 0.1 cm electroporation cuvette (BioRad) and electroporated using a Gene Pulser Xcell (BioRad) set at 1800 V, 25 micro-F and 200 Ohms. Immediately after electroporation 1 ml of LB medium was added, the cell suspension was transferred to a 15 ml plastic tube (Falcon) and incubated at 37 °C with shaking (200 _jcpm) fpr^ te-The toansfprm<br>
g/ml and incubated overnight at 37 °C. 24 transformants were grown in liquid culture and the cultures used for assaying phytase activity and isolation of plasmid DNA. One clone producing highest phytase activity and generating the expected restriction pattern of the plasmid DNA was selected. The plasmid contained by this clone named pETll(Pl-29) was used to transform the expression host strain BL21(DE3)pLysS (Novagen). The transformed cell suspension, was shaken for 1 h at 37°C in LB<br><br>
containing 2 % glucose and inoculated into 50 ml of LB containing ampicillin (100 micro-g/ml) and glucose (2%) and grown overnight at 30 °C with shaking (200 rpm). The OD of the resulting culture was measured at 600 nm and the culture was used to inoculate 1 1 of LB + ampicillin (100 micro-g/ml) to an ODsoo of 0.04. Growth was continued overnight at 30 °C. The phytase activity in such cultures was typically 8-12 U/ml. About 40% of phytase activity was secreted to the culture medium and the rest remained associated with the cells. These observations show that Buttiauxella PI-29 phytase is secreted in E.coli somewhat more efficiently than in its native host. The activity in the culture of a control strain BL21(DE3)pLysS transformed with pETll grown under the same conditions was below 0.0.5 U/ml.<br>
Example 7. Purification of the recombinant phytase from Buttiauxella Pl-29.<br>
The culture of BL21(DE3) pLysS transformed with pETll(Pl-29) was centrifuged to remove the bacterial cells, concentrated using a rotary evaporator to about 1/10 of the original volume and dialysed against water until the conductivity of the solution decreased below 250 micro-S/cm. The pH of the solution was adjusted to 8.0 with tris base and it was applied to a column (3x20 cm) of DEAE Sepharose Fast Flow (Roche) equilibrated with 25 mM tris-HCl, pH 8.0. The column was washed with the equilibration buffer at a flow rate of 3 ml/rnin for 30 min followed by elution with three successive gradients of NaCl in 25 mM tris-HCl, pH 8.0: 0-50 mM, 50-150 mM and 150-500 mM. Each of the three gradients was programmed for Ih with a constant flow rate of 3 ml/min. 9 ml fractions were collected and assayed for phytase activity. One strong peak of activity was detected. The protein in the peak fraction was concentrated using Centriplus concentrators (Amicon) and analysed SDS PAGE using a 12% gel and the standard Laemmli buffer system. The results of this analysis indicated that the preparation of recombinant Buttiauxella Pl-29 phytase obtained by DEAE Sepharose contains a single prpjm'nentjprotein component. Semi-quantitative analyses based on scanning of the digital image of the gel (Fig 1) indicate the purity of about 65%.<br><br>
Example 8. pH profile of the recombinant phytase from Buttianxella ¥1-29<br>
Dependence of the activity of the Buttiauxella PI-29 phytase from (purified according to the Example 7) on pH was studied in buffers and under conditions described in _ Example 1.   The enzyme was active in a broad  pH_area_(2-5.5)jvith a maximum around pH 4.5 and a strong "shoulder" of the curve around pH 3 (Fig 2).<br>
Example 9. Substrate specificity of the recombinant phytase from Buttiauxella Pl-29.<br>
The fractions of inositol phosphates containing three, four or five phosphates per inositol residue were isolated by ion-exchange chromatography from a partial hydrolysate of phytic acid treated with fungal phytase (Natuphos).   Production and purification of these preparations was a commercial service of BioChemis Ltd (St.Petersburg, Russia).    Contamination of each fraction with inositol-phosphates having a different degree of phosphorylation was less that 5% as judged by HPLC (Sandberg A.S., Ahderinne R. J. Food Sci, 51 (3), 547-550).    Commercial fructose 1,6-diphosphate and fructose 6-phosphate (Sigma) were used as model substrates used to estimate the specificity of the Buttiauxella Pl-29 phytase towards di- and monophosphate substrates. The activity of the Buttiauxella phytase purified according to the Example 7 with different substrates was measured by the standard assay (Example 1) at pH 3.5 using 2 mM concentrations of substrates in the final reaction mixture.   The results (Fig 3) indicate that that the enzyme has extremely broad substrate specificity. Its activity with penta-, tetra- and triphosphates was essentially equal or somewhat higher than the activity with phytic acid as substrate. Even fructose 1,6-diphosphate - a poor substrate for most phytases, was rather efficiently hydrolysed by Buttiauxella sp. phytase (in particular the Pl-29 derived phytase). _HvdrolyMs_QffractQse 6-phosphate was also detectable although much less efficient than the hydrolysis of other tested substrates.<br><br>
Example 10. Specific activity of the recombinant phytase from Buttiauxella Pl-<br>
Specific activity of the Buttiauxella PI-29 phytase was estimated using the purified preparation accordingJtojheExample J^TJiephytese activity was measured at pH 3.5 according to the Example 1. Phytase concentration was calculated by measuring total protein concentration with BCA Protein Assay Kit (Pierce) and correcting it by phytase content estimated by SDS PAGE (Example 7). According to these measurements, the specific activity of the recombinant phytase from Buttiauxella Pl-29 is about 300 U/mg at 37'C.<br>
Example 11. Generation and characterisation of phytase variants.<br>
Phytase variants were constructed by mutagenesis of the nucleotide sequence SEQ ID No. 2 using mutagenesis methods as listed above such as the methods disclosed in Morinaga et al (Biotechnology (1984) 2, p 646-649), or in Nelson and Long (Analytical Biochemistry (1989), 180, p 147-151), or the Error Threshold Mutagenesis protocol described in WO 92/18645. Another suitable method for mutagenic PCR is disclosed by Cadwell and Joyce (PCR Methods Appl. 3(1994), 136-140).<br>
Phytase enzyme variants were characterized after heterologous expression in one or more of the following expression hosts: Escherichia coll K12; Bacillus subtilis; Saccharomyces cerevisiae.<br>
Phytase variants were derived which differed in one or more arnino acid positions<br>
from SEQ ID No 3, including two positions, three positions, four positions, five<br>
positions,^k=positions,,seympositiQns.,,e<br>
eleven   positions,   twelve   positions.      Where   appropriate   iterative   rounds   of<br>
mutagenesis, as herein described were performed.<br><br>
Characterization of phytase variants<br>
1.   Thermostability<br>
The thermostability of such variants was characterized by the inactivation temperature of the enzyme. The inactivation temperature was determined by measuring the residual activity of the phytase enzyme after incubation for 10 min at different temperatures and subsequent cooling to room temperature. The inactivation temperature is the temperature at which the residual activity is 50% compared to the residual activity after incubation for the same duration under the same conditions at room temperature. Where appropriate interpolations and extrapolations from the measured activity data are done in order to determine the temperature corresponding to 50% residual activity. Thermostability differences in °C were calculated by subtracting the inactivation temperatures of two enzymes from each other.<br>
(Table Removed) ther characteristics were also Jmproyed.=    i<br>
Thermostability, specific activity, and pepsin stability of selected variants were compared using assays as described above. The pepsin stability of such variants was characterized by residual activities measured at pH 3.5, 37 °C after pepsin incubation compared to control conditions (residual activity = activity after pepsin incubation / activity after incubation under control conditions). The pepsin incubation was performed for 2 hours at pH 2.0, 0.25 mg/ml pepsin, 1 mM CaC12 and 5 mg/ml BSA at 37°C. Control conditions were 2 hours at pH 5.0,1 mM CaC12 and 5 mg/ml BSA at 37°C.<br>
Table 2 shows properties of selected variants (derived from and compared to wt phytase according to Seq ID No. 3).<br>
(Table Removed) pecific activity for a variant derived from the parent phytase shown in Seq ID No 3: <br>
All publications mentioned in the above specification, and references cited in said publications, are herein incorporated by reference. Various modifications and variations of the described methods and system of the present invention will be apparent to those skilled in the art without departing from the scope and spirit of the present invention. Although^the Jnvention Jias been described[ in connection \vjth_ specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in molecular biology or related fields are intended to be within the scope of the following claims.<br><br><br><br>
1.	A polypeptide comprising the amino acid sequence corresponding to a Buttiauxella<br>
equivalent or an effective fragment, thereof.<br>
2.	A polypeptide as claimed in claim 1 comprising the amino acid sequence as shown<br>
in SEQ ID NO: 3 or a sequence having at least 75% identity (homology), preferably<br>
has at least 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity (homology), thereto<br>
or a functional fragment, thereof.<br>
3.	A polypeptide according to claims 1 or 2 wherein said polypeptide is isolated<br>
and/or purified.<br>
4.	A polypeptide according to claims 1 to 3, wherein said polypeptide exhibits phytase<br>
activity.<br>
5.	A polypeptide according to claims 1 to 4, characterised in that it is obtainable from,<br>
preferably derived from, Buttiauxella sp. strain PI-29 deposited under accession<br>
number NCIMB 41248.<br>
6.	A polypeptide according to claims 4 or 5 or functional equivalent thereof<br>
characterised in that said phytase has a specific activity of at least 100, more<br>
preferably at least 200, most preferably at least 300 U/mg, wherein said specific<br>
activity is determined by incubating said phytase in a solution containing 2mM<br>
phytate, O.SmM CaC^ in 200mM sodium acetate buffer at pH 4.5, at a temperature of<br>
7.	A polypeptide according to claims 4 to 6 or functional equivalent thereof<br>
characterised in that said phytase has an activity maximum at around pH 3 to 6,<br>
preferably about pH 4 to 5, most preferably about pH 4.5, wherein said activity is<br>
determined by incubating said phytase in a solution containing 2mM phytate, O.SmM<br>
in 200mM sodium acetate buffer at a temperature of 37°C.<br><br>
8.	A polypeptide as claimed in any of claims 1 to 7 that comprises mutations in at<br>
least one of the following positions (numbering according to the numbering in SEQ<br>
ID No. 3): 59, 70, 122, 125, 167, 193, 197, 204, 209, 211, 221, 223, 225, 240, 242,<br>
,244,268,281,289,294, JJD33.6_or.<br>
9.	A polypeptide as claimed in claim 8 wherein said phytase comprises one or more of<br>
the following mutations:<br>
K 59 A, C, D, E, F, G, H, I, L, M, N, P, Q, R, S, T, V, W,or Y;<br>
or<br>
T 70 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W,or Y;<br>
or<br>
A 122 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W,or Y;<br>
or<br>
D 125 A, C, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
T 167 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W, or Y;<br>
or<br>
H 193 A, C, D, E, F, G, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
F 197 A, C, D, E, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
T 204 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W, or Y;<br>
or<br>
T 209 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, V, W, or Y;<br>
or<br>
A,2UJC,,D, E, F, AH,I,-K, LM,JO,a RXMLmX<br>
or<br>
G 221 A, C, D, E, F, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
1223 A, C, D, E, F, G, H, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
S 225 A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, T, V, W, or Y;<br><br>
or<br>
K 240 A, C, D, E, F, G, H, I, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
A 242 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
D 244 A, C, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
A 268 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
L 281 A, C, D, E, F, G, H, I, K, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
Q 289 A, C, D, E, F, G, H, I, K, L, M, N, P, R, S, T, V, W, or Y;<br>
or<br>
A 294 C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
N 303 A, C, D, E, F, G, H, I, K, L, M, P, Q, R, S, T, V, W, or Y;<br>
or<br>
1 336 A, C, D, E, F, G, H, K, L, M, N, P, Q, R, S, T, V, W, or Y;<br>
or<br>
N 351. A, C, D, E, F, G, H, I, K, L, M, P, Q, R, S, T, V, W, or Y;<br>
10.	A polypeptide / phytase enzyme as claimed in claims 8 or 9 comprising at least<br>
one mutation selected from the group consisting of:  K59E; T167V; K240T; T167I;<br>
K240E; D244C; Q289Y; T209KorF197S.<br>
11.	A polypeptide / phytase enzyme as claimed in claim 8 to 10 comprising a<br>
combination of mutations selected from the group consisting of :<br>
A294E/N303K; or<br>
T167I/K240T; or<br>
D223E/K240E/N351D;or<br>
T167I/K240T/A294E/N303K; or<br>
T167I/K240E/A242S/A294E/N303K; or<br>
A122T/T1 67I/F197S/T209K/A21 1P/K240E/A294E/N303K; or<br><br>
A122T/T167I/F197S/T209K7A211P/K240E/A242S/S281L/Q289Y/A294E/N303K;<br>
or<br>
A122T/D125E/H193R/F197S/T209K/A211P/S221N/G225A/K240E/A294E/ N303K;<br>
or<br>
N303K; or<br>
A122T/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268V/Q289H/A294E/<br>
N303K/I336F or<br>
N70 Y/D125E/T167I/H193R/F197S/T204I/T209K/A211P/K240E/A268V/Q289H/<br>
A294E/N303K.<br>
12.	A polypeptide / phytase enzyme as claimed in claim 8 to 11 wherein said isolated<br>
polypeptide or phytase enzyme is an improved phytase enzyme variant.which<br>
compared to the parent phytase enzyme has unproved property/properties selected<br>
from:<br>
i.     higher thermal stability and/or ii.     specific activity and/or iii.     proteolytic stability<br>
13.	An isolated nucleic acid molecule selected from<br>
i)        An isolated nucleic acid molecule coding for the phytase as claimed in any<br>
of claims 1 to 12. ii)       a nucleic acid which hybridises to a nucleic  acid coding for the<br>
polypeptide  SEQ  ID No.  3  or homologue thereof, under medium<br>
stringency conditions, preferably high or very high stringency conditions, iii)       An isolated nucleic acid molecule comprising the sequence as set out in<br>
SEQ ID NO: 2 or homologue thereof. -<br>
under medium stringency conditions, preferably high or very high<br>
stringency conditions.<br>
14.	An isolated nucleic acid molecule according to claim 13 encoding a polypeptide<br>
comprising the arnino acid sequence as shown in SEQ ID NO: 3 or a sequence having<br><br>
at least 75% 80%, 85%, 90%, 95% or 99% sequence identity (homology) thereto or an effective fragment, thereof.<br>
15.	An isolated nucleic acid molecule according to claims 13 or 14 comprising a<br>
 ..nucleotidesequences that Jsthesames, qrjs complementary to, or contains any<br>
suitable codon substitutions for any of those of SEQ ID NO. 2, or comprises a sequence which has at least 75%, 80%, 85%, 90%, 95% or 99% sequence homology with SEQ ID NO: 2.<br>
16.	An isolated nucleic acid molecule according to claims 13 to 15, where in said<br>
nucleic acid molecule encodes the polypeptide of claims 8 to 12.<br>
17.	A plasmid or vector system comprising a nucleotide according to claims 13 to 16.<br>
18.	A plasmid or vector system according to claim 17 wherein said plasmid or vector<br>
system is an expression vector for the expression of any of the enzymes according to<br>
claims 1 to 12, and/ or comprise polynucleotides according to claims 13 to 16, in a<br>
microorganism.<br>
19.	A host cell transformed or transfected with a plasmid or vector system as claimed<br>
in any of claims 17 or 18.<br>
20.	A host cell as claimed in claim 19 which comprises aphytase which comprises an<br>
amino acid sequence, or functional fragment thereof, as set out in SEQ ID NO: 3 or a<br>
homologous polypeptide sequence that is at least 75% homologous thereto.<br>
21.	A host cell as claimed in claim 19 or 20 wherein said host cell is derived from a<br>
yeast such as H. polymorpha, S. pombe, S. cerevisiae, and filamentous fungi, such as Trichoderma spp. andAspergittusspp. such as A. oryzae.<br>
22.	A host cell as claimed in claim 21 wherein said microorganism is a prokaryotic<br>
bacterial cell, preferably E. colL<br><br>
23.	A bacterial cell strain Buttiauxella sp. PI-29 deposited under accession number<br>
NCIMB 41248.<br>
24.	A method of producing a polypeptide comprising expressing an amino acid<br>
sequence according to.claims_1., and/or expressing the pj)laccording<br>
to claims 13 to 16, in a host cell and separating said phytase from the host cell culture medium.<br>
25.	A method for production of food or animal feed comprising a step of spraying a<br>
polypeptide as claimed hi any of claims 1 to 12 in liquid form onto said food or<br>
animal feed.<br>
26.	A method for production of food or animal feed comprising a step of mixing the<br>
polypeptide as claimed hi any of claims 1 to 12 as a dry product with said food or<br>
animal feed.<br>
27.	Use of a phytase as claimed in any of claims 1 to 12 in food or animal feed.<br>
28.	A food or animal feed composition comprising either i) a phytase as claimed in<br>
any of claims 1 to 12 and/or ii) an animal feed produced by the method according to<br>
claims 25 or 26.<br>
29.	A method of preparing a phytase enzyme variant, said method comprises the<br>
following sequential steps:<br>
a)	Selecting at least one parent phytase enzyme, wherein the at least one parent<br>
phytase enzyme is selected from i) the polypeptides according to claims 1-12<br>
and/or ii) at least one phytase enzyme variant;<br>
b)	Jjenerating allesLonphyjaejyariarit by introducing at least one alteration of<br>
said parent phytase enzyme which is an insertion, a deletion or a substitution or combination thereof, of an ammo acid residue hi said parent phytase enzyme to obtain at least one phytase enzyme variant;<br>
c)	Screening for said at least one phytase enzyme variant to identify an unproved<br>
phytase enzyme variant, which compared to the parent phytase enzyme has<br>
unproved property/properties selected from:<br><br>
i.  higher thermal stability and/or ii.   specific activity and/or iii.  proteolytic stability<br>
d) Preparing said improved phytase enzyme variant, preferably to produce an isolated .and/or pirrified phytase,enzyme variant.<br>
30.	A method according to claim 29, wherein during step b) a population of phytase<br>
enzyme variants is generated and in step c) at least a proportion of said population of<br>
phytase enzyme variants is screened.<br>
31.	A method according to claims 29 or 30, wherein step a) comprises subjecting a<br>
nucleotide sequence according to claims 11 to 13 encoding a parent phytase enzyme<br>
to mutagenesis, and step b) comprises expressing the mutated nucleotide sequence<br>
obtained in step (a) in a host cell, and step c) comprises screening for host cells, or<br>
extract(s) thereof, for an improved phytase enzyme variant with said improved<br>
property/properties.<br>
32.	A method according to claims 29 to 31 which, after step c), and optionally d),<br>
further comprise at least one subsequent round of repeating steps a) to c), and<br>
optionally d) wherein, preferably, in said subsequent round(s), the at least one parent<br>
phytase enzyme of step a) is selected from said at least one phytase enzyme variant<br>
and/or an improved phytase variant prepared according to the method of claims 29 to<br><br>
33.	A method according to claims 29 to 32 wherein step c) comprises screening for<br>
host cells expressing an improved phytase enzyme variant which compared to either i)<br>
said parent phytase enzyme and/or ii) a polypeptide comprising SEQ ID No 3, has a<br>
_ihftrmaLstfihi1ity,difFerencenfaUeast2.5.	<br>
34. A method according to claims 29 to 33 wherein step c) comprises screening for host cells expressing an improved phytase enzyme variant which compared to either i) said parent phytase enzyme and/or ii) a polypeptide comprising SEQ ID No 3 has pepsin stability of at least 30.<br><br>
35.	A method according to claims 29 to 34 wherein step c) comprises screening for<br>
host cells expressing an improved phytase enzyme variant which compared to either i)<br>
said parent phytase enzyme and/or ii) a polypeptide comprising SEQ ID No 3 has a<br>
specific activity ratio when compared to the phytase encoded by SEQ ID No 3 of at<br>
= .least 110.<br>
36.	An improved phytase variant prepared according to the method of claims 29 to<br>
35.<br>
37.	A polynucleic  acid construct,  preferably  a DNA  construct comprising a<br>
polynucleic acid sequence encoding an improved phytase enzyme variant according to<br>
any of claims 16 or claim 36.<br><br>
38.	A recombinant expression vector which comprises a polynucleic acid construct<br>
according to claim 37.<br>
39.	A host cell which is transformed with a DNA polynucleic acid according to claim<br>
37 and/or a vector according to claim 38.<br>
40.	A method for production of an improved phytase enzyme variant, comprising<br>
culturing the host cell according to claim 39 under conditions permitting the<br>
expression of said improved phytase enzyme variant.<br>
41.	A feed or food composition, comprising at least one improved phytase enzyme<br>
variant of any of claims 8 to 12 or 36, or host cell of claim 38, wherein said food or<br>
feed composition is, preferably prepared according to the methods of claims 25 or 26.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2178-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUFic3RyYWN0LSgxOS0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Abstract-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUNsYWltcy0oMDctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Claims-(07-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUNsYWltcy0oMTktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Claims-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Correspondence Others-(07-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Correspondence Others-(25-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Correspondence-Others-(02-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Correspondence-Others-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUZvcm0tMTMtKDE5LTA3LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Form-13-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUZvcm0tMTMtKDI1LTEwLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Form-13-(25-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUZvcm0tMi0oMTktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Form-2-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUZvcm0tMy0oMDctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Form-3-(07-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUZvcm0tMy0oMTktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Form-3-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LUdQQS0oMTktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-GPA-(19-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3OC1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTktMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2178-delnp-2007-Petition-137-(19-07-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258565-oil-based-suspension-concentrates.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258567-a-method-for-predicting-the-state-of-the-gastric-mucosa.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258566</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2178/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Mar-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DANISCO A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LANGEBROGADE 1, P.O.BOX 17, DK-1001 COPENHAGEN K, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANDREI MIASNIKOV</td>
											<td>TVARANTIE 92, FIN-10160 DEGERBY, FINLAND</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VIJAY KUMAR</td>
											<td>HAMEENTIE 35 A 40, FIN-00500 HELSINKI, FINLAND</td>
										</tr>
										<tr>
											<td>3</td>
											<td>OLIVER KENSCH</td>
											<td>NATTERMANNALLEE 1, 50829 COLOGNE, GERMANY</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KLAUS PELLENGAHR</td>
											<td>NATTERMANNALLEE 1, 50829 COLOGNE, GERMANY</td>
										</tr>
										<tr>
											<td>5</td>
											<td>BIRGITTA LEUTHNER</td>
											<td>NATTERMANNALLEE 1, 50829 COLOGNE, GERMANY</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ULRICH KETTLING</td>
											<td>NATTERMANNALLEE 1, 50829 COLOGNE, GERMANY</td>
										</tr>
										<tr>
											<td>7</td>
											<td>ANDRE KOLTERMANN</td>
											<td>NATTERMANNALLEE 1, 50829 COLOGNE, GERMANY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P19/04; C07H21/04; C08B1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2005/003376</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>GB0423139.5</td>
									<td>2004-10-18</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script data-cfasync="false" src="../cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258566-a-polypeptide-selected-from-the-group-consisting by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:09:49 GMT -->
</html>
