<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258586-oil-and-seeds-with-reduced-saturate-levels-of-fatty-acids by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:06:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258586:&quot;OIL AND SEEDS WITH REDUCED SATURATE LEVELS OF FATTY ACIDS&quot;.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;OIL AND SEEDS WITH REDUCED SATURATE LEVELS OF FATTY ACIDS&quot;.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The subject invention provides &quot;no sat&quot; canola oil. The subject invention also provides Seeds that can be used to produce such oils. Plants that produce these seeds are also included within the subject invention. All of this was surprisingly achieved by using a delta-9 desaturase gene in canola. This technology can be applied to other plants as disclosed herein. Oils of the subject invention have particularly advantageous characteristics and fatty acid profiles, which were not heretofore attained. The subject invention still further provides a plant-optimized delta-9 desaturase gene. The subject invention still further provides a plant-optimized delta-9 desaturase gene. In some preferred embodiments, a preferred plant comprises at least two copies of a delta-9 desaturase gene of the subject invention. Seeds produced by such plants surprisingly do not exhibit effects of gene silencing but rather have further surprising reductions in levels of total saturates.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>CERTAIN PLANTS WITH "NO SATURATE" OR REDUCED SATURATE LEVELS OF FATTY ACIDS IN SEEDS. AND OIL DERIVED FROM THE SEEDS<br>
Cross-Reference to Related Application<br>
[oooi]	The subject application claims priority to U.S. provisional application serial number<br>
60/617,532 filed on October 8, 2004.<br>
Background of the Invention<br>
[0002]	Vegetable-derived oils have gradually replaced animal-derived o ils and fats as the major<br>
source of dietary fat intake. However, saturated fat intake in most industrialized nations has remained at about 15% to 20% of total caloric consumption. In efforts to promote healthier lifestyles, the United States Department of Agriculture (USDA) has recently recommended that saturated fats make up less than 10% of daily caloric intake. To facilitate consumer awareness, current labeling guidelines issued by the USDA now require total saturated fatty acid levels be less than 1.0 g per 14 g serving to receive the "low-sat" label and less than 0.5 g per 14 g serving to receive the "no-sat" label. This means that the saturated fatty acid content of plant oils needs to be less than 7% and 3.5% to receive the "low sat" and "no sat" label, respectively. Since issuance of these guidelines, there has been a surge in consumer demand for "low-sat" oils. To date, this has been met principally with canola oil, and to a much lesser degree with sunflower and safflower oils.<br>
The characteristics of oils, whether of plant or animal origin, are determined predominately by the number of carbon and hydrogen atoms, as well as the number and position of double bonds comprising the fatty acid chain. Most oils derived from plants are composed of varying amounts of palmitic (16:0), stearic (18:0), oleic (18:1), linoleic (18:2) and linolenic (18:3) fatty acids. Conventionally, palmitic and stearic acids are designated as "saturated" because their carbon chains are saturated with hydrogen atoms and hence have no double bonds; they contain the maximal number of hydrogen atoms possible. However, oleic, linoleic, and linolenic are 18-carbon fatty acid chains having one, two, and three double bonds, respectively, therein. Oleic acid is typically considered a mono-unsaturated fatty acid, whereas linoleic and linolenic are considered to be poly-unsaturated fatty acids. The U.S. Department of Agriculture defines "no saturates" or "no sat" products as a product having less than 3.5% by weight combined saturated fatty acids (as compared to the total amount of fatty acids).<br><br>
[0004]	While unsaturated fats (monounsaturated and polyunsaturated) are beneficial (especially<br>
when consumed in moderation), saturated and trans fats are not. Saturated fat and trans fat raise LDL cholesterol levels in the blood. Dietary cholesterol also raises LDL cholesterol and may contribute to heart disease even without raising LDL. Therefore, it is advisable to choose foods low in saturated fat, trans fat, and cholesterol as part of a healthful diet.<br>
[oms]	The health value of high levels of monounsaturates, particularly oleic acid, as the major<br>
dietary fat constituent has been established by recent studies. Such diets are thought to reduce the incidence of arteriosclerosis that results from diets high in saturated fatty acids. There is accordingly a need for an edible vegetable oil having a high content of monounsaturates. Seed mutagenesis has been used to produce a rapeseed oil with no more than 4% saturated fatty acid content (PCT International Patent Application Publication Number WO 91/15578).<br>
[0006]	Over 13% of the world's supply of edible oil in 1985 was produced from the oilseed crop<br>
species Brassica, commonly known as rapeseed or mustard. Brassica is the third most important source of edible oil, ranking behind only soybean and palm. Because Brassica is able to germinate and grow at relatively low temperatures, it is also one of the few commercially important edible oilseed crops that can be cultivated in cooler agricultural regions, as well as serving as a winter crop in more temperate zones. Moreover, vegetable oils in general, and rapeseed oil in particular, are gaming increasing consideration for use in industrial applications because they have the potential to provide performance comparable to that of synthetic or mineral/naphthenic-based oils with the very desirable advantage of also being biodegradable.<br>
[0007]	Canola oil has the lowest level of saturated fatty acids of all vegetable oils. "Canola"<br>
refers to rapeseed (Brassica) which has an erucic acid (C22:l) content of at most 2 percent by weight based on the total fatty acid content of a seed (preferably at most 0.5 percent by weight and most preferably essentially 0 percent by weight) and which produces, after crushing, an air-dried meal containing less than 30 micromoles per gram of defatted (oil-free) meal. These types of rapeseed are distinguished by their edibility in comparison to more traditional varieties of the species.<br>
[ooo8]	Modification of vegetable oils may be effected chemically. This approach has been used<br>
to obtain a salad/cooking oil which contains saturated fatty acids of less than about 3% (U.S. Pat. No. 4,948,811); the oil may be formed by chemical reaction, or by physical separation of the saturated lipids. A general reference is made to using "genetic engineering" to achieve an oil of the desired characteristics (see column 3, line 58 et seq.). However, there is no detailed<br><br>
disclosure of how any particular oilseed plant could be so modified to provide a vegetable oil of the characteristics desired.<br>
[ooo9]	Typically, the fatty acid composition of vegetable oils has instead been modified through<br>
traditional breeding techniques. These techniques utilize existing germplasm as a source of naturally occurring mutations that affect fatty acid composition. Such mutations are uncovered and selected for by the use of appropriate screening, in conjunction with subsequent breeding. For example, such an approach has been used to decrease the amount of the long chain fatty acid erucate in rapeseed oil (Stefansson, B. R. (1983) in High and Low Erucic Acid Rapeseed Oils, Kramer J. K. G. et al., eds; Academic Press, New York; pp. 144-161), and to increase the amount of the monounsaturated fatty acid oleate in corn oil (U.S. patent application, Ser. No. 07/554,526).<br>
[ooio]	Recently, attempts have been made to increase the pool of available mutations from<br>
which to select desired characteristics through the use of mutagens. However, mutagens generally act by inactivation or modification of genes already present, resulting in the loss or decrease of a particular function. The introduction of a new characteristic through mutagenesis thus often depends on the loss of some trait already present. In addition, the achievement of desired goals with mutagens is generally uncertain. Only a few types of modified fatty acid compositions in vegetable oils have been achieved using this approach. One example of such a "created" mutation which affects fatty acid composition is the decrease of polyunsaturated fatty acids, in particular of linoleate and linolenate, in rapeseed oil, with a concomitant increase in the monounsaturated fatty acid oleate (Auld, M., et al., (1992) Crop Sci. in press). Another is the decrease of saturated fatty acids in rapeseed oil (PCT International Patent Application Publication Number WO 91/15578). However, the biochemistry of seed oil synthesis is complex, and not well understood; there may be several mechanisms which contribute to the changes in the fatty acid compositions observed in rapeseed oil (PCT International Patent Application Publication Number WO 91/15578). The use of mutagenesis to affect such changes is essentially random, and non-specific.<br>
'imi i]	The possibility of modifying fatty acid composition through the use of genetic<br>
engineering would, in theory, allow the precise, controlled introduction of specific desirable genes, as well as the inactivation of specific undesirable genes or gene products. Thus, novel traits completely independent of genes already present could be introduced into plants, or preselected genes could be inactivated or modified. However, one predicate to making effective use<br><br>
of genetic engineering to modify fatty acid compositions is a reasonably accurate model of the mechanisms at work in the plant cell regulating fatty acid synthesis and processing.<br>
loon]	U.S. Patent No. 6,495,738 (see also WO 99/50430) shows that the levels of saturated<br>
fatty acids in corn oil and tobacco seeds can be altered by expressing a fungal palmitate-CoA delta-9 desaturase within a plant cell. These proteins most likely enzymatically desaturate palmitate-CoA molecules, preferentially, by removing two hydrogen atoms and adding a double bond between the 9th and 10th carbon atoms from the CoA portion of the molecule, thus producing palmitoleic-CoA (16:1 delta-9). The palmitoleic-CoA is ultimately incorporated into seed oil thus lowering the total saturate levels of said oil. The total saturated fatty acid level of corn oil, averaging about 13.9%, does not meet the current labeling guidelines discussed above. Furthermore, corn is typically not considered to be an oil crop as compared to soybean, canola, sunflower, and the like. In fact, the oil produced and extracted from corn is considered to be a byproduct of the wet milling process used in starch extraction. Because of this, there has been little interest in modifying the saturate levels of corn oil.<br>
[ooii]	It is postulated that, in oilseeds, fatty acid synthesis occurs primarily in the plastid, and<br>
that the newly synthesized fatty acids are exported from the plastid to the cytoplasm. In the cytoplasm they are utilized in the assembly of triglycerides, which occurs in the ehdoreticular membranes.<br>
[ooi4]	The major product of fatty acid synthesis is palmitate (16:0), which appears to be<br>
efficiently elongated to stearate (18:0). While still in the plastid, the saturated fatty acids may then be desaturated, by an enzyme known as delta-9 desaturase, to introduce one or more carbon-carbon double bonds. Specifically, stearate may be rapidly desaturated by a plastidial delta-9 desaturase enzyme to yield oleate (18:1). In fact, palmitate may also be desaturated to palmitoleate (16:1) by the plastidial delta-9 desaturase, but this fatty acid appears in only trace quantities (0-0.2%) in most vegetable oils.<br>
[ooisj	Thus, the major products of fatty acid synthesis in the plastid are palmitate, stearate, and<br>
oleate. In most oils, oleate is the major fatty acid synthesized, as the saturated fatty acids are present in much lower proportions.<br>
[0016]	Subsequent desaturation of plant fatty acids outside the plastid in the cytoplasm appears<br>
to be limited to oleate, which may be desaturated to linoleate (18:2) and linolenate (18:3). In addition, depending on the plant, oleate may be further modified by elongation (to 20:1, 22:1,<br><br>
and/or 24:1), or by the addition of functional groups. These fatty acids, along with the saturated fatty acids palmitate and stearate, may then be assembled into triglycerides.<br>
loon]	The plant delta-9 desaturase enzyme is soluble. It is located in the plastid stroma, and<br>
uses newly synthesized fatty acids esterified to ACP, predominantly stearyl-ACP, as substrates. This is in contrast to the yeast delta-9 desaturase enzyme, which is located in the endoplasmic reticular membrane (ER, or microsomal), uses fatty acids esterified to Co-A as substrates, and desaturates both the saturated fatty acids palmitate and stearate. U.S. Patent Nos. 5,723,595 and 6,706,950 relate to a plant desaturase.<br>
[oois]	The yeast delta-9 desaturase gene has been isolated from Saccharomyces cerevisiae,<br>
cloned, and sequenced(Stukey, J. E. etal.,J. Biol. Chem. 264:16537-16544 (1989); Stukey, J. E. et al, J. Biol. Chem. 265:20144-20149 (1990)). This gene has also been used to transform the same yeast strain under conditions in which it is apparently overexpressed, resulting in increased storage lipid accumulation in the transformed yeast cells as determined by fluorescence microscopy using Nile Red as a stain for triglycerides (U.S. Pat. No. 5,057,419). The fatty acid composition was not characterized. This reference contains a general discussion of using information from the isolated yeast delta-9 desaturase gene to first isolate other desaturase genes from yeast, or from other organisms, and then to re-introduce these genes into a yeast or plant under conditions. It is speculated that this could lead to high expression in order to modify the oil produced and its fatty acid composition.<br>
fooi9]	Subsequently, it was reported that the yeast delta-9 desaturase gene had been introduced<br>
into tobacco leaf tissue (Polashcok, J. et al., FASEB J. 5:A1157 (1991) and was apparently expressed in this tissue. Further, this gene was expressed in tomato. See Wang et al., J. Agric Food Chem. 44:3399-3402 (1996); and C. Wang et al, Phytochemistry 58:227-232 (2001). While some increases in certain unsaturates and some decreases in some saturates were reported for both tobacco and tomato, tobacco and tomato are clearly not oil crops. This yeast gene was also introduced into Brassica napus (see U.S. Patent No. 5,777,201). Although a reduction in palmitate and stearate (saturates) and an increase in palmitoleate and oleate (unsaturates) was reported (see Tables la and Ib in Example 7 of that patent), this reference is discussed in more detail towards the beginning of the Detailed Description section, below. WO 00/11012 and U.S. Patent No. 6,825,335 relate to a synthetic yeast desaturase gene for expression in a plant, wherein the gene comprises a desaturase domain and a cyt bs domain. The Background section of these references discuss fatty acid synthesis in detail.<br><br>
0020]	The performance characteristics, whether dietary or industrial, of a vegetable oil are<br>
substantially determined by its fatty acid profile, that is, by the species of fatty acids present in the oil and the relative and absolute amounts of each species. While several relationships between fatty acid profile and performance characteristics are known, many remain uncertain. Notwithstanding, the type and amount of unsaturation present in a vegetable oil have implications for both dietary and industrial applications.<br>
w2ij	Standard canola oil contains about 8-12% linolenic acid, which places it in a similar<br>
category as soybean oil with respect to oxidative, and hence flavor, stability. The oxidative stability of canola oil can be improved in a number of ways, such as by hydrogenating to reduce the amount of unsaturation, adding antioxidants, and blending the oil with an oil or oils having better oxidative stability. For example, blending canola oil with low linolenic acid oils, such as sunflower, reduces the level of 18:3 and thus improves the stability of the oil. However, these treatments necessarily increase the expense of the oil, and can have other complications; for example, hydrogenation tends to increase both the level of saturated fatty acids and the amount of trans unsaturation, both of which are undesirable in dietary applications.<br>
roo22]	High oleic oils are available, but, in addition to the possible added expense of such<br>
premium oils, vegetable oils from crops bred for very high levels of oleic acid can prove unsatisfactory for industrial uses because they retain fairly high levels of polyunsaturated fatty acids, principally linoleic and/or linolenic. Such oils may still be quite usable for dietary applications, including use as cooking oils, but have inadequate oxidative stability under the more rigorous conditions found in industrial applications. Even the addition of antioxidants may not suffice to bring these oils up to the levels of oxidative stability needed for industrial applications; this is probably due to the levels of linolenic acid, with its extremely high susceptibility to oxidation, found in these oils.<br>
[0023]	Oxidative stability is important for industrial applications to extend the life of the<br>
lubricant under conditions of heat and pressure and in the presence of chemical by-products. In such applications linolenic acid, and to a lesser extent linoleic acid, are again most responsible for poor oxidative stability.<br>
[o24]	Therefore, it would be desirable to obtain a variety of Brassica napus which is<br>
agronomically viable and produces seed oil having a level of oxidative stability sufficient to qualify it for use in dietary applications, and which would additionally be either sufficiently<br><br>
stable alone, or, depending on the precise application, sufficiently responsive to antioxidants, to find use in industrial applications.<br>
[0025]	European Patent Application EP 323753, U.S. Patent No. 5,840,946, and U.S. Patent No.<br>
5,638,637 are directed to rapeseed oil having an oleic content of 80-90% (by weight, of total fatty acid content) and not more than 2% erucic acid. Mutagenesis was used to improve the oleic acid content. The claims of the '946 patent further specify that the oil also has an erucic acid content of no more than 2%, and alpha-linolenic acid content of less than 3.5%, and a saturated fatty acid content in the form of stearic and palmitic of no more than 7%. These patents relate to mutagenesis followed by selection.<br>
[0026]	U.S. Patent Nos. 5,387,758; 5,434,283; and 5,545,821 are directed to rapeseed having 2-<br>
4% combined stearic and palmitic acids (by weight), and an erucic acid content of no more than about 2% by weight. Mutagenesis was used to lower the stearic and palmitic acid content.<br>
[0027]	International Application WO 92/03919, and U.S. Patent Nos. 5,668,299; 5,861,187; and<br>
6,084,157 are directed to rapeseed seeds, plants, and oils having altered fatty acid profiles. Several such profiles are mentioned, all of which contemplate a maximum erucic acid content of about 2%, combined with palmitic acid content of from about 7% to about 12%, linoleic content of about 14% to about 20%, stearic acid content of from about 0.8% to about 1.1%, and alpha-linolenic acid content of about 7% to about 9%, as well as certain ranges of FDA saturates. These patents define saturated fatty acids and "FDA saturates" as the sum of lauric (C12:0), myristic (C14:0), palmitic (C16:0), and stearic (C18:0) acids.<br>
[oo28]	International Application WO 93/06714, and U.S. Patent Nos. 6,270,828; 6,562,397;<br>
6,680,396; and 6,689,409 are directed to canola oil and seeds with reduced glucosinolates (and thus reduced sulfur), as well as an alpha-linolenic acid content of about 2% to about 7%.<br>
[0029]	U.S. Patent No. 6,169,190 relates to oil from canola seed having an oleic fatty acid<br>
content of approximately 71-77% and a linolenic acid content of less than about 3%. Oleic:linolenic ratios between 34-55 are also claimed.<br>
[0030]	U.S. Patent Nos. 6,063,947 and 5,850,026 claim oil obtained from canola seeds, related<br>
canola plants, and methods of producing the oil, wherein the oil has an oleic acid content greater than about 80% (about 86-89%), a linoleic acid content of about 2% to about 6%, an alpha-linolenic acid content of less than 2.5% (about 1-2%), and an erucic acid content of less than about 2% (after hydrolysis). These patents relate to seed-specific inhibition of microsomal oleate desaturase (a delta-12 desaturase which converts oleic acid to linoleic acid) and microsomal<br><br>
linoleate desaturase (a delta-15 desaturase which converts linoleic acid to alpha-linolenic acid)<br>
gene expression.<br>
[oo3i]	U.S. Patent No. 5,952,544 claims fragments of a plant plastid or microsomal delta-15<br>
fatty acid desaturase enzyme, which catalyzes a reaction between carbons 15 and 16.<br>
[0032]	U.S. Patent Nos. 4,627,192 and 4,743,402 relate to sunflower seeds and sunflower oil<br>
having an oleic acid content of approximately 80-94% (relative to the total fatty acid content<br>
thereof) and a ratio of linoleic to oleic of less than about 0.09. These sunflower plants were<br>
obtained by traditional breeding techniques.<br>
[0033]	WO 2003002751 relates to the use of kinase genes and the like to alter the oil phenotype<br>
of plants.<br>
[0034]	The ability of delta-9 desaturase genes to significantly (and desirably) affect the fatty acid<br>
profile of already-beneficial oil seed crops, particularly to decrease the levels of saturated fats<br>
without adversely affecting other aspects of the plant and oil, is unpredictable.<br>
Brief Summary of the Invention<br>
[oo3sj	The subject invention provides "no sat" canola oil. The invention also relates in part to<br>
methods for reducing saturated fatty acids in certain plant seeds. These results were surprisingly achieved by the use of a delta-9 desaturase gene in canola (Brassica). This technology can be applied to other plants as disclosed herein. Included in the subject invention are plants, preferably canola, capable of producing such oils and seeds. The subject invention also provides seeds and oils from said plants wherein the oils have particularly advantageous characteristics and fatty acid profiles, which were not heretofore attained. The subject invention still further provides a plant-optimized delta-9 desaturase gene. In some preferred embodiments, a preferred plant comprises at least two copies of a delta-9 desaturase gene of the subject invention. Seeds produced by such plants surprisingly do not exhibit effects of gene silencing but rather have further surprising reductions in levels of total saturates.<br>
Brief Description of the Figures<br>
[0036]	Figure 1 shows that a greater than 60% reduction of saturated fatty acids was achieved in<br>
Arabidopsis. This graph summarizes T2 and T3 seed data for a single Arabidopsis event.<br><br>
[0037]	Figure 2 shows a reduction in "sats" of up to 60-70% in T2 Arabidopsis seeds from 18<br>
additional transformants. Data illustrated in this graph was a combination of the numerical data shown in Table 8 and earlier numerical data.<br>
[0038]	Figure 3 shows that saturated fats were reduced by over 43% in Westar canola (and a<br>
50% reduction was achieved when 24:0 was included).<br>
[0039]	Figure 4A shows a bar graph comparing total saturates of seeds from various canola<br>
plants comprising Event 36-11.19 compared to a control. Figures 4B and 4C present numerical data illustrated by the bar graph.<br>
[0040]	Figure 5A shows a bar graph comparing total saturates of seeds from various canola<br>
plants comprising Event 218-11.30 compared to a control. Figures 5B and 5C present numerical data illustrated by the bar graph.<br>
[(mi]	Figures 6A-F show half-seed data from the T3 field trials. Figures 6A and 6B clearly<br>
show the reductions in C 16:0 and increases in C 16:1 in the transgenic events as compared to the nulls (events in which the transgene segregated out of the plant) and wild-type controls (non-transformed lines). Figures 6C and 6D clearly show the reductions in C18:0 and increases in Cl 8:1 in the transgenic events as compared to the nulls and wild-type controls. Figures 6E and 6F clearly show the reductions in C20:0 and C22:0, respectively, in the transgenic events as compared to the nulls and wild-type controls.<br>
[0042]	Figures 6G and 6H clearly show shifts and reductions in C16:0, and shifts and increases<br>
in C16:1 in the transgenic events, as compared to the nulls and wild-type controls. Figures 61 and 6J clearly show shifts and reductions in C18:0, and shifts and increases in C18:l in the transgenic events, as compared to the nulls and wild-type controls. Figures 6K and 6L show similar bar graphs for Cl8:2 and C18:3. Figure 6M further illustrates reductions in total saturates, as compared to already very good Nex 710 lines. Figure 6N shows distributions for 1000 seeds.<br>
[0043]	Figures 7A and 7B illustrate data obtained using the protocol of Example 16.<br>
[0044]	Figures 8 and 9 are pictures of two gels run with DNA from F3 plants, as discussed in<br>
Example 19.<br>
Brief Description of the Sequences<br>
[0045]	SEQ ID NO: 1 shows the nucleic acid sequence of the open reading frame for the plant-<br>
optimized, delta-9 desaturase gene used herein.<br><br>
10046]	SEQ ID NO:2 shows the sequence of the ORF of SEQ ID NO: 1 preceded by a Kozak<br>
sequence and a BamHI cloning site (residues 1-10), plus a translational terminator at the end of<br>
the ORF (residues 1379-1381).<br>
[0047]	SEQ ID NO:3 shows the nucleic acid sequence of the delta-9 forward B primer used to<br>
amplify the delta-9 gene.<br>
[0048]	SEQ ID NO:4 shows the nucleic acid sequence of the delta-9 reverse B primer used to<br>
amplify the delta-9 gene.<br>
[0049]	SEQ ID NO:5 shows the amino acid sequence encoded by SEQ ID NO:1.<br>
Detailed Description of the Invention<br>
[ooso]	The subject invention provides "no sat" canola oil. The invention also relates in part to<br>
methods for reducing saturated fatty acids in certain plant seeds. These results were surprisingly achieved through the use of a delta-9 desaturase gene to surprisingly produce "no sat" levels of fatty acids in plants, preferably oil plants, and still more preferably canola (Brassica). The subject invention includes such plants and also provides seeds and oils from said plants wherein the oils have particularly advantageous characteristics and fatty acid profiles, which were not heretofore attained.<br>
[oosi]	The Aspergillns nidulans microsomal delta-9-CoA desaturase gene is exemplified herein.<br>
This delta-9 desaturase is a membrane-bound enzyme and catalyzes the reaction of 16:0-CoA and 18:0-CoA to 16:l-CoA and 18:l-CoA (adding a double bond at the delta-9 location). The subject invention was further surprising in that the levels of other saturates, such as C20:0, C22:0, and C24:0, were also very surprisingly and advantageously reduced, while C16:l and C18:l unsaturates were increased (with little or no increases in C18:2 and C18:3, or even reductions of these relatively less stable polyunstaturates in some cases). Heretofore, it was unclear whether this would be a good enzyme (including whether the gene could be sufficiently expressed) in Brassica and other "good" oil seed plants, which already have a desirable (yet not optimal) fatty acid profile. (For example, it yielded only a 10% decrease in saturates in corn.)<br>
[0052]	As mentioned in the Background section, given the complex fatty acid profiles and<br>
metabolic pathways of different organisms and plants, and the different physical cell machinery thereof, even if this gene and enzyme could have an effect in Brassica, the effects could not be expected to be beneficial. As discussed in the Background section, increases in one or more types of desirable fatty acids often resulted in decreases of other desirable fatty acids, increases<br><br>
in undesirable fatty acids, and agronomic penalties (i.e., other outright adverse effects on the modified plants). It was also surprising that the subject invention can be practiced without corresponding adverse effects to other valuable agronomic characteristics such as pod size, seed yield, seed size, oil yield, and the like. There were no adverse effects in plants homozygous for a simple transgenic insert. Some double homozygous stacks (made by crossing two transgenic events) exhibited decrease in pod number and seed set; the cause is yet unknown. However, Table 27 contains stacks (that is, apparently increased copy number events) having seed yields similar to non-transgenic controls and also 'no sat' composition.<br>
[0053]	Yet another reason for unpredictably arises because of differences between desaturases,<br>
and even between yeast, fungal, plant, and animal delta-9 desaturases. Differences in the desaturases can be attributed in part to differences in cell structures of the source organisms for the various desaturases. A yeast desaturase from U.S. Patent No. 5,777,201 is discussed above in the Background section. It is longer than the Apergillus desaturase exemplified herein (510 amino acids vs. 455 amino acids). In addition, it has only about 52% identity over about 400 amino acids (as determined by both BLAST and BestFit, a Smith-Waterman program; both done in EMBOSS). Tables la and Ib of Example 7 of that patent show that the reductions in saturates achieved using the yeast desaturase were much weaker than those achieved according to the subject invention with the exemplified Aspergillus desaturase in canola. There are various factors that can be possible explanations for the relatively weaker performance of the yeast desaturase. For example, that protein might be inherently instable in plants (while the subject desaturase is quite apparently very stable in canola). These can also be different in other enzyme properties, such as catalytic efficiencies, substrate affinities, cofactor affinities, and the like.<br>
[oos4]	Compared to the safflower desaturase of U.S. Patent Nos. 5,723,595 and 6,706,950, the<br>
safflower desaturase is shorter (396 amino acids) than the presently exemplified Aspergillus desaturase (455 amino acids). The safflower desaturase is also found in the plastid, while the subject Aspergillus desaturase is found in the ER/microsomes/cytoplasmic compartment. Furthermore, the safflower desaturase uses acyl-ACP substrates found in the plastid, while the Aspergillus desaturase uses acyl-CoA substrates found in the cytoplasmic compartment. Thus, for the subject invention, it was not known if a substantial portion of the pool of acyl-CoA substrates would be available to the Aspergillus desaturase.<br>
[ooss]	Thus, it was with great surprise that the subject delta-9 desaturase was found to be able to<br>
yield canola plants, seeds, and oil therefrom having excellent properties, particularly for<br><br>
improving food qualities of the oil. Very surprisingly, a greater than 60% reduction of saturated fatty acids was achieved in Arabidopsis, and a greater than 43% reduction of saturated fatty acids was achieved in canola. Again, it is important to note that this was achieved in a plant that already yielded one of the best fatty acid profiles of any suitable plant. This invention was also used to achieve surprising and advantageous fatty acid profiles and ratios, as shown and discussed in more detail below. Although stearic acid is considered to be a saturated fatty acid, it has been found to have cholesterol-lowering effects. Thus, relatively higher levels of stearic acid can be beneficial. Similarly, relatively higher levels of arachidonic acid can be desirable. As shown in data herein, oil from seeds of the subject invention have advantageous profiles of these two fatty acids, together with desirable levels of vaccenic acid, for example. Also shown herein is that advantageous levels of these fatty acids and/or total saturates are present in combination with desirable plant height, yield, and other beneficial characteristics in the commercial-quality plants of the subject invention (as opposed to dwarf plants, for example). Again, exemplary data for such plants of the subject invention are presented herein.<br>
[0056]	It should be noted that the subject invention is not limited to the exemplified desaturase.<br>
Various desaturases and delta-9 desaturases are available in GENBANK, and sequence alignments can be performed, using standard procedures, to observe and compare differences in the sequences of the enzymes. Enzymes similar to that exemplified herein can be used according to the subject invention.<br>
[0057]	For example, the subject Aspergillus desaturase has two domains. The first domain<br>
(approximately the amino-terminal two-thirds of the molecule) is the desaturase domain, and the second domain (roughly the C-terminal third of the molecule) is a cytochrome b5 domain. Residues 62-279, for example, of SEQ ID NO:5 can be aligned with residues 4-233 of fatty acid desaturase gnl|CDD|25523 pfam00487, for example. Residues 332-407 of SEQ ID NO:5 can be aligned with residues 1-74 of gnl|CDD|22935 pfam00173 (cytochrome b5 domain). Residues 17-305 of SEQ ID NO: 5 can be aligned with residues 3-288 of the lipid metabolism domain of fatty acid desaturase gnl|CDD|l 1113 COG1398 (OLE1). Residues 301-449 of SEQ IDNO:5 can be aligned with residues 11-163 of CYB5 (cytochrome b involved in lipid metabolism) of gnl|CDD| 14396 COG5274. The desaturase domain of SEQ ID NO:5 lacking the cytochrome b5 could be functional, as this is the general structure of plant plastidial desaturases. There is also a published presumed microsomal pine desaturase (LOCUS AF438199) which uses acyl-CoA substrates found in the cytoplasmic compartment, and it lacks the cytbS domain. It might<br><br>
also be possible to swap the Aspergillus cytochrome b5 domain with that of another organism, even one from a plant cytoplasmic desaturase. These domains, or segments encoding either or both of these domains, can be used as probes to define molecules of the subject invention, as discussed in more detail below.<br>
[DOSS]	Thus, the genes and proteins useful according to the subject invention include not only<br>
the specifically exemplified full-length sequences, but also portions, segments and/or fragments (including internal and/or terminal deletions compared to the full-length molecules) of these sequences, variants, mutants, chimerics, and fusions thereof. Proteins used in the subject invention can have substituted amino acids so long as they retain the characteristic enzymatic activity of the proteins specifically exemplified herein. "Variant" genes have nucleotide sequences that encode the same proteins or equivalent proteins having functionality equivalent to an exemplified protein. The terms "variant proteins" and "equivalent proteins" refer to proteins having the same or essentially the same biological/functional activity as the exemplified proteins. As used herein, reference to an "equivalent" sequence refers to sequences having amino acid substitutions, deletions, additions, or insertions that improve or do not adversely affect functionality. Fragments retaining functionality are also included in this definition. Fragments and other equivalents that retain the same or similar function, as a corresponding fragment of an exemplified protein are within the scope of the subject invention. Changes, such as amino acid substitutions or additions, can be made for a variety of purposes, such as increasing (or decreasing) protease stability of the protein (without materially/substantially decreasing the functionality of the protein).<br>
[0059]	Variations of genes may be readily constructed using standard techniques for making<br>
point mutations, for example. In addition, U.S. Patent No. 5,605,793, for example, describes methods for generating additional molecular diversity by using DNA reassembly after random fragmentation. Variant genes can be used to produce variant proteins; recombinant hosts can be used to produce the variant proteins. Using these "gene shuffling" techniques, equivalent genes and proteins can be constructed that comprise any 5,10,15,20,25,30,35,40,45,50,55,60 (for example) contiguous residues (amino acid or nucleotide) of any sequence exemplified herein.<br>
[0060]	Fragments of full-length genes can be made using commercially available exonucleases<br>
or endonucleases according to standard procedures. For example, enzymes such as Bal31 or site-directed mutagenesis can be used to systematically cut off nucleotides from the ends of these<br><br>
genes. Also, genes that encode active fragments may be obtained using a variety of restriction enzymes. Proteases may be used to directly obtain active fragments of these proteins.<br>
[oo6i]	It is within the scope of the invention as disclosed herein that the subject proteins may be<br>
truncated and still retain functional activity. By "truncated protein" it is meant that a portion of a protein may be cleaved and yet still exhibit enzymatic activity after cleavage. Furthermore, effectively cleaved proteins can be produced using molecular biology techniques wherein the DNA bases encoding said protein are removed either through digestion with restriction endonucleases or other techniques available to the skilled artisan. After truncation, said proteins can be expressed in heterologous systems such as Escherichia coli, baculoviruses, plant-based viral systems, yeast and the like and then placed in insect assays as disclosed herein to determine activity. It is well-known in the art that truncated proteins can be successfully produced so that they retain functional activity while having less than the entire, full-length sequence. It is well known in the art that B. t. toxins can be used in a truncated (core toxin) form. See, e.g., Adang et al, Gene 36:289-300 (1985), "Characterized full-length and truncated plasmid clones of the crystal protein of Bacillus thuringiensis subsp kurstaki HD-73 and their toxicity to Manduca sexta," There are other examples of truncated proteins that retain insecticidal activity, including the insect juvenile hormone esterase (U.S. Pat. No. 5,674,485 to the Regents of the University of California). As used herein, the term "toxin" is also meant to include functionally active truncations.<br>
[0062]	Proteins and genes for use according to the subject invention can be defined, identified,<br>
and/or obtained by using oligonucleotide probes, for example. These probes are detectable nucleotide sequences which may be detectable by virtue of an appropriate label or may be made inherently fluorescent as described in International Application No. WO 93/16094. The probes (and the polynucleotides of the subject invention) may be DNA, RNA, or PNA. In addition to adenine (A), cytosine (C), guanine (G), thymine (T), and uracil (U; for RNA molecules), synthetic probes (and polynucleotides) of the subject invention can also have inosine (a neutral base capable of pairing with all four bases; sometimes used in place of a mixture of all four bases in synthetic probes). Thus, where a synthetic, degenerate oligonucleotide is referred to herein, and "N" or "n" is used generically, "N" or "n" can be G, A, T, C, or inosine. Ambiguity codes as used herein are in accordance with standard IUPAC naming conventions as of the filing of the subject application (for example, R means A or G, Y means C or T, etc.).<br><br>
 [DOGS]	As is well known in the art, if a probe molecule hybridizes with a nucleic acid sample, it<br>
can be reasonably assumed that the probe and sample have substantial homology /similarity /identity. Preferably, hybridization of the polynucleotide is first conducted followed by washes under conditions of low, moderate, or high stringency by techniques well-known in the art, as described in, for example, Keller, G.H., M.M. Manak (1987) DNA Probes, Stockton Press, New York, NY, pp. 169-170. For example, as stated therein, low stringency conditions can be achieved by first washing with 2x SSC (Standard Saline Citrate)/0.1% SDS (Sodium Dodecyl Sulfate) for 15 minutes at room temperature. Two washes are typically performed. Higher stringency can then be achieved by lowering the salt concentration and/or by raising the temperature. For example, the wash described above can be followed by two washings with O.lx SSC/0.1% SDS for 15 minutes each at room temperature followed by subsequent washes with O.lx SSC/0.1% SDS for 30 minutes each at 55° C. These temperatures can be used with other hybridization and wash protocols set forth herein and as would be known to one skilled in the art (SSPE can be used as the salt instead of SSC, for example). The 2x SSC/0.1 % SDS can be prepared by adding 50 ml of 20x SSC and 5 ml of 10% SDS to 445 ml of water. 20x SSC can be prepared by combining NaCl (175.3 g/0.150 M), sodium citrate (88.2 g/0.015 M), and water, adjusting pH to 7.0 with 10 N NaOH, then adjusting the volume to 1 liter 10% SDS can be prepared by dissolving 10 g of SDS in 50 ml of autoclaved water, then diluting to 100ml.<br>
[0064]	Detection of the probe provides a means for determining in a known manner whether<br>
hybridization has been maintained. Such a probe analysis provides a rapid method for identifying toxin-encoding genes of the subject invention. The nucleotide segments which are used as probes according to the invention can be synthesized using a DNA synthesizer and standard procedures. These nucleotide sequences can also be used as PCR primers to amplify genes of the subject invention.<br>
urns]	Hybridization with a given polynucleotide is a technique that can be used to identify,<br>
find, and/or define proteins and genes of the subject invention. As used herein, "stringent" conditions for hybridization refers to conditions which achieve the same, or about the same, degree of specificity of hybridization as the conditions described herein. Hybridization of immobilized DNA on Southern blots with 32P-labeled gene-specific probes can be performed by standard methods (see, e.g., Maniatis, T., E.F. Fritsch, J. Sambrook [1982] Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY). In general,<br><br>
hybridization and subsequent washes are carried out under conditions that allowed for detection of target sequences. For double-stranded DNA gene probes, hybridization can be carried out overnight at 20-25° C below the melting temperature (Tm) of the DNA hybrid in 6x SSPE, 5x Denhardt's solution, 0.1% SDS, 0.1 mg/ml denatured DNA. The melting temperature is described by the following formula (Beltz, G.A., K.A. Jacobs, T.H. Eickbush, P.T. Cherbas, and F.C. Kafatos [1983] Methods of Enzymology, R. Wu, L. Grossman and K. Moldave [eds.] Academic Press, New York 100:266-285):<br>
Tm = 81.5° C + 16.6 Log[Na+] + 0.41(%G+C) - 0.61(%formamide) - 600/length of<br>
duplex in base pairs. Washes are typically carried out as follows:<br>
1)	Twice at room temperature for 15 minutes in Ix SSPE, 0.1% SDS (low<br>
stringency wash).<br>
2)	Once at Tm-20° C for 15 minutes in 0.2x SSPE, 0.1 % SDS (moderate stringency<br>
wash).<br>
[0066]	For oligonucleotide probes, hybridization can be carried out overnight at 10-20° C below<br>
the melting temperature (Tm) of the hybrid in 6x SSPE, 5x Denhardt's solution, 0.1 % SDS, 0.1 mg/ml denatured DNA. Tm for oligonucleotide probes was determined by the following formula: Tm (° C) = 2(number T/A base pairs) + 4(number G/C base pairs) (Suggs, S.V., T. Miyake, E.H. Kawashime, MJ. Johnson, K. Itakura, and R.B. Wallace [1981] ICN-UCLA Symp. Dev. Biol. Using Purified Genes, D.D. Brown [ed.], Academic Press, New York, 23:683-693).<br>
[oo67j	Washes can be carried out as follows:<br>
1)	Twice at room temperature for 15 minutes Ix SSPE, 0.1% SDS (low stringency<br>
wash).<br>
2)	Once at the hybridization temperature for 15 minutes in Ix SSPE, 0.1% SDS<br>
(moderate stringency wash).<br>
[0068]	In general, salt and/or temperature can be altered to change stringency. With a labeled<br>
DNA fragment &gt;70 or so bases in length, the following conditions can be used:<br>
Low:	1 or 2x SSPE, room temperature<br>
Low:	1 or 2x SSPE, 42° C<br>
Moderate:       0.2x or Ix SSPE, 65° C<br>
High:	O.lxSSPE, 65° C.<br><br>
 [0069]	Duplex formation and stability depend on substantial complementarity between the two<br>
strands of a hybrid, and, as noted above, a certain degree of mismatch can be tolerated. Therefore, the probe sequences of the subject invention include mutations (both single and multiple), deletions, insertions of the described sequences, and combinations thereof, wherein said mutations, insertions and deletions permit formation of stable hybrids with the target polynucleotide of interest. Mutations, insertions, and deletions can be produced in a given polynucleotide sequence in many ways, and these methods are known to an ordinarily skilled artisan. Other methods may become known in the future.<br>
[0070]	Because of the degeneracy/redundancy of the genetic code, a variety of different DNA<br>
sequences can encode the amino acid sequences disclosed herein. It is well within the skill of a person trained in the art to create alternative DNA sequences that encode the same, or essentially the same, enzymes. These variant DNA sequences are within the scope of the subject invention.<br>
[oo7i]	The subject invention include, for example:<br>
1)	proteins obtained from wild type organisms;<br>
2)	variants arising from mutations;<br>
3)	variants designed by making conservative amino acid substitutions; and<br>
4)	variants produced by random fragmentation and reassembly of a plurality of different<br>
sequences that encode the subject TC proteins (DNA shuffling).  See e.g. U.S. Patent No.<br>
5,605,793.<br>
[0072]	The DNA sequences encoding the subject proteins can be wild type sequences, mutant<br>
sequences, or synthetic sequences designed to express a predetermined protein. DNA sequences designed to be highly expressed in plants by, for example, avoiding polyadenylation signals, and using plant preferred codons, are particularly useful.<br>
[0073]	Certain proteins and genes have been specifically exemplified herein. As these proteins<br>
and genes are merely exemplary, it should be readily apparent that the subject invention comprises use of variant or equivalent proteins (and nucleotide sequences coding for equivalents thereof) having the same or similar functionality as the exemplified proteins. Equivalent proteins will have amino acid similarity (and/or homology) with an exemplified enzyme (or active fragment thereof). Preferred polynucleotides and proteins of the subject invention can be defined in terms of narrower identity and/or similarity ranges. For example, the identity and/or similarity of the enzymatic protein can be 40,41,42, 43, 44,45, 46,47,48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60,61,62,63,64, 65,66,67,68,69,70, 71, 72, 73,74,75,76,77, 78,79, 80, 81, 82,<br><br>
83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, or 99% as compared to a sequence exemplified or suggested herein. Any number listed above can be used to define the upper and lower limits. For example, a protein of the subject invention can be defined as having 50-90% identity, for example, with an exemplified protein.<br>
[0074]	Unless otherwise specified, as used herein, percent sequence identity and/or similarity of<br>
two nucleic acids is determined using the algorithm of Karlin and Altschul (1990), Proc. Natl. Acad. Sci. USA 87:2264-2268, modified as in Karlin and Altschul (1993), Proc. Natl. Acad. Sci. USA 90:5873-5877. Such an algorithm is incorporated into the NBLAST and XBLAST programs of Altschul et al. (1990), J. Mol. Biol. 215:402-410. BLAST nucleotide searches are performed with the NBLAST program, score = 100, wordlength =12. Gapped BLAST can be used as described in Altschul et al. (1997), Nucl. Acids Res. 25:3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (NBLAST and XBLAST) are used. See NCBI/NIH website.<br>
[oo75]	To obtain gapped alignments for comparison purposes, the AlignX function of Vector<br>
NTI Suite 8 (InforMax, Inc., North Bethesda, MD, U.S.A.), can be used employing the default parameters. Typically these would be a Gap opening penalty of 15, a Gap extension penalty of 6.66, and a Gap separation penalty range of 8. Two or more sequences can be aligned and compared in this manner or using other techniques that are well-known in the art. By analyzing such alignments, relatively conserved and non-conserved areas of the subject polypeptides can be identified. This can be useful for, for example, assessing whether changing a polypeptide sequence by modifying or substituting one or more amino acid residues can be expected to be tolerated.<br>
[0076]	The amino acid homology/similarity/identity will typically (but not necessarily) be<br>
highest in regions of the protein that account for its activity or that are involved in the determination of three-dimensional configurations that are ultimately responsible for the activity. In this regard, certain amino acid substitutions are acceptable and can be expected to be tolerated. For example, these substitutions can be in regions of the protein that are not critical to activity. Analyzing the crystal structure of a protein, and software-based protein structure modeling, can be used to identify regions of a protein that can be modified (using site-directed mutagenesis, shuffling, etc.) to actually change the properties and/or increase the functionality of the protein.<br><br>
 [0077]	Various properties and three-dimensional features of the protein can also be changed<br>
without adversely affecting the activity/functionality of the protein. Conservative amino acid substitutions can be expected to be tolerated/to not adversely affect the three-dimensional configuration of the molecule. Amino acids can be placed in the following classes: non-polar, uncharged polar, basic, and acidic. Conservative substitutions whereby an amino acid of one class is replaced with another amino acid of the same type fall within the scope of the subject invention so long as the substitution is not adverse to the biological activity of the compound. The following list provides examples of amino acids belonging to each class.<br>
	Class of Amino Acid	Examples of Amino Acids	<br>
Nonpolar	Ala, Val, Leu, He, Pro, Met, Phe, Trp<br>
Uncharged Polar	Gly, Ser, Thr, Cys, Tyr, Asn, Gin<br>
Acidic	Asp, Glu<br>
Basic	Lys, Arg, His<br>
[W78]	In some instances, non-conservative substitutions can also be made. The critical factor is<br>
that these substitutions must not significantly detract from the functional/biological/enzymatic activity of the protein.<br>
[0079]	To obtain high expression of heterologous genes in plants, for example, it may be<br>
preferred to reengineer said genes so that they are more efficiently expressed in plant cells. Sequences can be designed for optimized expression in plants, generally, or they can be desigened for optimized expression in a specific type of plant. Canola is one such plant where it may be preferred to re-design the heterologous gene(s) prior to transformation to increase the expression level thereof in said plant. Therefore, an additional step in the design of genes encoding a fungal protein, for example, is reengineering of a heterologous gene for optimal expression in a different type of organism. Guidance regarding the production of synthetic genes that are optimized for plant expression can be found in, for example, U.S. Patent No. 5,380,831. A sequence optimized for expression in plants is exemplified herein as SEQ ID NO:1 (which encodes the exemplified protein, as shown in SEQ ID NO:5).<br>
[ooso]	As used herein, reference to "isolated" polynucleotides and/or proteins, and "purified"<br>
proteins refers to these molecules when they are not associated with the other molecules with which they would be found in nature. Thus, reference to "isolated" and/or "purified" signifies the involvement of the "hand of man" as described herein. For example, a fungal polynucleotide<br><br>
 (or "gene") of the subject invention put into a plant for expression is an "isolated polynucleotide." Likewise, a protein of the subject invention when produced by a plant is an "isolated protein."<br>
A "recombinant" molecule refers to a molecule that has been recombined. When made in reference to a nucleic acid molecule, the term refers to a molecule that is comprised of nucleic acid sequences that are joined together by means of molecular biological techniques. The term "recombinant" when made in reference to a protein or a polypeptide refers to a protein molecule that is produced using one or more recombinant nucleic acid molecules.<br>
[0082]	The term "heterologous" when made in reference to a nucleic acid sequence refers to a<br>
nucleotide sequence that is ligated to, or is manipulated to become ligated to, a nucleic acid sequence to which it is not joined in nature, or to which it is joined at a different location in nature. The term "heterologous" therefore indicates that the nucleic acid molecule has been manipulated using genetic engineering, i.e. by human intervention. Thus, a gene of the subject invention can be operably linked to a heterologous promoter (or a "transcriptional regulatory region" which means a nucleotide sequence capable of mediating or modulating transcription of a nucleotide sequence of interest, when the transcriptional regulatory region is operably linked to the sequence of interest). Preferred heterologous promoters can be plant promoters. A promoter and/or a transcriptional regulatory region and a sequence of interest are "operably linked" when the sequences are functionally connected so as to permit transcription of the sequence of interest to be mediated or modulated by the transcriptional regulatory region. In some embodiments, to be operably linked, a transcriptional regulatory region may be located on the same strand as the sequence of interest. The transcriptional regulatory region may in some embodiments be located 5' of the sequence of interest. In such embodiments, the transcriptional regulatory region may be directly 5' of the sequence of interest or there may be intervening sequences between these regions. The operable linkage of the transcriptional regulatory region and the sequence of interest may require appropriate molecules (such as transgenic activator proteins) to be bound to the transcriptional regulatory region, the invention therefore encompasses embodiments in which such molecules are provided, either in vitro or in vivo.<br>
[0083]	There are a number of methods for obtaining the proteins for use according to the subject<br>
invention. For example, antibodies to the proteins disclosed herein can be used to identify and isolate other proteins from a mixture. Specifically, antibodies may be raised to the portions of the proteins that are most constant and most distinct from other proteins. These antibodies can<br><br>
then be used to specifically identify equivalent proteins with the characteristic activity by immunoprecipitation, enzyme linked immunosorbent assay (ELISA), or immuno-blotting. Antibodies to the proteins disclosed herein, or to equivalent proteins, or to fragments of these proteins, can be readily prepared using standard procedures. Such antibodies are an aspect of the subject invention.<br>
[0084]	A protein "from" or "obtainable from" any of the subject isolates referred to or suggested<br>
herein means that the protein (or a similar protein) can be obtained from the exemplified isolate or some other source, such as another fungal or bacterial strain, or a plant (for example, a plant engineered to produce the protein). "Derived from" also has this connotation, and includes polynucleotides (and proteins) obtainable from a given type of fungus or bacterium, for example, wherein the polynucleotide is modified for expression in a plant, for example. One skilled in the art will readily recognize that, given the disclosure of a fungal gene and protein, a plant can be engineered to produce the protein. Antibody preparations, nucleic acid probes (DNA and RNA, for example), and the like may be prepared using the polynucleotide and/or amino acid sequences disclosed herein and used to screen and recover other protein genes from other (natural) sources.<br>
[DOSS]	Oils of the subject invention retain a high degree of oxidative stability but contain lower<br>
levels of saturated fatty acids and higher levels of unsaturated fatty acids. Preferred oils of the subject invention have less than 3.5% total saturated fatty acid content, oleic content of at least 75% (and preferably and surprisingly less than 80%), and a poly unsaturated fatty acid content of less than 20% (and more preferably less than 15%, still more preferably less than 10%, and even more preferably less than 9%). The subject invention can also be used to achieve canola seed having total saturated fatty acid content (C:14, C:16, C:18, C:20, C:22, and C:24) of not more than (and preferably less than) 2.5% of the total fatty acid content, preferably with the oleic acid ranges as mentioned above). 18:2 and 18:3 levels, which contribute to oil instability, are not increased or are preferably reduced (for food applications). End points for ranges for any of these particular fatty acids, any combinations thereof, and particularly for either one or both of the Cl 8 polyunsaturates, can be obtained from any of the Figures and Tables provided herein.<br>
[0086]	The subject invention can be used to provide agronomically elite canola seed that results<br>
in a refined/deodorized oil with less than 3.5% total saturates. Oil derived from these plants can be used to formulate various end products, or they can be used as stand-alone frying oil for "no sat" (or "low sat") products.<br><br>
 [0087]	Unless indicated otherwise, the saturated fatty acid content of a given collection of canola<br>
seeds can be determined by standard procedures wherein the oil is removed from the seeds by crushing the seeds and is extracted as fatty acid methyl esters following reaction with methanol and sodium hydroxide. The resulting ester is then analyzed for fatty acid content by gas liquid chromatography using a capillary column which allows separation on the basis of the degree of unsaturation and chain length. This analysis procedure is described in, for example, J. K. Daun et al., J. Amer. Oil Chem. Soc. 60: 1751-1754 (1983).<br>
[DOSS]	The fatty acid composition of canola seed was determined as described below for either<br>
"half-seed" analysis, "single/whole seed" analysis, or "bulk seed" analyses. For "half-seed" analyses, a portion of cotyledonary tissue from the embryo was removed and analyzed; the remaining seed was then saved, and could be germinated if desired. Although the half-seed technique can be somewhat unreliable in selecting stable genetically controlled fatty acid mutations (and subsequent breeding and crosses), the subject invention demonstrates that preferred genes can be introduced and used to create stable lines. Unlike uncharacterized mutations, it is well known in the art that a gene can be introduced and stably maintained in plants. Thus, the analysis set forth herein demonstrates the utility of the subject genes and that canola oil having the indicated characteristics can be attained.<br>
[0089]	"No saturates" (i.e., No Sat) levels of fatty acids were reached in seeds from transgenic<br>
lines derived from commercial Nexera 710 (canola) germplasm. The No Sat level is defined as less than 3.5% combined saturates. In addition, reduced saturates were seen in both the Westar canola line, and another Crucifer (Arabidopsis) with the same transformation construct. Notably, saturate levels in single seeds were down to 2.6 to 2.7% for some seeds. The subject invention can also be used to produce seeds with 2.5% or less total saturates. This is important because oil processing can add ~0.5-1% to the total saturate "score," meaning that the processed oil product can still measurably reach the FDA-defined No Sat level using standard testing procedures. Having this level of tolerance not only permits for some levels of contamination (by higher saturate seeds) of testing equipment (especially if the plant operator does a poor job of keeping seed batches distinct), but also permits for some level of variation in field growth conditions (such as high temperatures, which tend to create more saturates) and cross-pollination by pollen drifting from unimproved canola in adjacent fields (which dilutes desirable genes).<br>
The U.S. Food and Drug Administration defines "saturated fat" as "A statement of the number  of grams  of saturated  fat  in  a  serving  defined  as  the  sum  of all  fatty<br><br>
acids containing no double bonds." 21 CFR 101.9(c)(2)(i). Unless otherwise specified, this is the definition used herein for "total saturates" and "total saturated fat." A serving of a food product is considered to have "no saturated fat" if the product "contain[s] less than 0.5 gram of total fat in a serving." 21 CFR 101.9(c)(2)(i). "Total fat" is defined as "A statement of the number of grams of total fat in a serving defined as total lipid fatty acids and expressed as triglycerides." 21 CFR101.9(c)(2). "Serving sizes" for various types of foods are defined in 21 CFR 101.12(b), which defines a serving of oil as 1 tablespoon or 15 ml. As used herein, this is understood to mean 14 grams. Thus, "no sat" canola oil (or canola oil comprising no saturated fat) is defined herein as canola oil having less than 0.5 grams of total saturated fat in a serving (14 grams of canola oil comprising 14 grams of fat). Stated another way, "no sat" canola oil comprises less than 3.57% total saturates (0.5 grams of total saturates divided by 14 grams of total fat). Unless specified otherwise, all percent fatty acids herein are percent by weight of the oil of which the fatty acid is a component.<br>
[oi)9jj	As shown herein, the subject invention can surprisingly be used to obtain oil from canola<br>
seeds wherein said oil comprises less than 3.57% total saturates. Oil can be obtained from the subject seeds using procedures that are well-known in the art, as mentioned in the preceding paragraphs, and the oil can be assayed for content using well-known techniques, including the techniques exemplified herein. Unless otherwise specified, analysis that was used to generate half-seed oils data and field oils data used a base-catalyzed transesterification reaction (AOCS Ce 2-66, alternative method). The protocol is similar to the saponification/acid esterification protocol described herein, except the saponification/acid esterification protocol measure total lipids, of which the majority are the same fatty acids from triacylglycerides detected by the base-catalyzed transesterification reaction.<br>
[0092]	In the commercial Nexera 710 germplasm, levels of 18:3 fatty acids, those that contribute<br>
to oxidative instability, were relatively unchanged. Thus, the subject invention not only provides plants, seeds, and oils with lower saturated fat, but also plants, seeds, and oils that very surprisingly maintain other beneficial characteristics. That is, the plants and genes of the subject invention can surprisingly be used without adversely affecting other advantageous characteristics of the plants.<br>
roo93j	In preferred embodiments, the subject invention provides plants comprising more than<br>
one expressed copy of a delta-9 desaturase gene of the subject invention. Results presented herein show that expressing multiple copies of this gene surprisingly improved the fatty acid<br><br>
profile of canola plants (saturated fat levels were greatly reduced). This is surprising in part because the art was heretofore unpredictable regarding the expression of multiple copies of the same gene. "Gene silencing" is one known phenomenon that teaches against using multiple copies (inserted at different locations in the genome, for example) of a heterologous gene. It is also not ideal to attempt to obtain multiple transformation events. Thus, there was no motivation to produce plants comprising more than one (two, three, four, and the like) delta-9 desaturase event. There was also no expectation that such plants would actually have improved characteristics.<br>
[0094]	Two examples of Cruciferous plants are specifically exemplified herein: Brassica napus<br>
(canola) and Arabidopsis. However, as is known in the art, other Brassica species and other Crucifers can be used for, for example, breeding and developing desired traits in canola and the like. Other such plants that can thus be used according to the subject invention include Brassica rapa, Brassicajuncea, Brassica carinata, Brassica nigra, Brassica oleracea, Raphanus sativus, and Sinapis alba. Soybeans, soybean plants, and soybean oil can also be tested for improvement according to the subject invention.<br>
[009S]	In preferred embodiments, delta-9 desaturase genes of the subject invention are optimized<br>
for plant expression. Thus, the subject invention also provides a plant-optimized delta-9 desaturase gene. Optimization exemplified herein included introducing preferred codons and a Kozak translational initiator region, and removing unwanted sequences. The gene was driven by the beta-phaseolin promoter (a strong dicot seed storage protein promoter).<br>
[0096]	Promoters for which expression coincides with oil synthesis (e.g. ACP, elongase) can be<br>
used to further reduce saturates, as expression occurs earlier than for storage proteins. (Prior tobacco constructs used the nos 3' UTR, and prior corn constructs used the constitutive maize Ubiquitin-1 promoter and nos 3' UTR.) Other dicot seed promoters can be used according to the subject invention, including vicilin, lectin, cruciferin, glycinin, and conglycinin promoters, plant seed promoters disclosed in US20030005485 Al, elongase promoters in US20030159173 Al, and the ACP promoter in U.S. Patent No. 5,767,363. See also, for example, US6100450A (seed specific, expesses in embryo, column 8 line 8); US20030159173A1 (section 0044 seed specific promoter; examples are USP, hordein, ACP, napin, FatB3, and FatB4); WO9218634A1 (introduction discusses seed-specific promoters from other patents pages 1 through 7; WO0116340A1 (page 7 line 13 provides a definition of a "seed specific" promoter, which typically expresses at less than 5% in other tissues; page 10 lines 19-29 discusses seed storage<br><br>
proteins like albumins, globulins, vicilin and legumin-like proteins, non-storage oleosins, promoters associated with fatty acid metabolism like ACP, saturases, desaturases, elongases); WO2003014347A2 (promoter definition p23-25: preferably 2x greater for seed-specific); US20030233677A1 (section 0033 provides "seed promoter" examples [napin, ACCase, 2S albumin, phaseolin, oleosin, zein, glutelin, starch synthase, starch branching enzyme]); WO2003092361A2 (page 15 provides a definition for "promoter"; the top of page 17 provides promoter examples and patent references (storage proteins only) including zeins, 7S storage proteins, Brazil nut protein, phe-free protein, albumin, beta-conglycinin, US, alpha-hordothionin, arcelin, lectins, glutenin); US20030148300 Al (see Claim 8, including the napin promoter, the phaseolin promoter, the soybean trypsin inhibitor promoter, the ACP promoter, stearoyl-ACP desaturase promoter, the soy 7S promoter, the oleosin promoter, the conglycinin promoter, oleosin promoters, embryogenesis-abundant protein promoters, embryo globulin promoters, arcelin 5, the napin promoter, and the acid chitinase promoter); U.S. Patent No. 5,777,201 (column 6, lines 30-50, constitutive promoters, seed- and/or developmentally regulated promoters e.g. plant fatty acid lipid biosynthesis genes [ACPs, acyltransferases, desaturases, lipid transfer proteins] or seed promoters [napin, cruciferin, conglycinin, lectins] or inducible promoters [light, heat, wound inducers]).<br>
[0097]	The plastids of higher plants are an attractive target for genetic engineering. Chloroplast<br>
(a type of plastid) transformation has been achieved and is advantageous. See e.g. U.S. Patent Nos. 5,932,479; 6,004,782; and 6,642,053. See also U.S. Patent Nos. 5,693,507 and 6,680,426. Advantages of transformation of the chloroplast genome include: potential environmental safety because transformed chloroplasts are only maternally inherited and thus are not transmitted by pollen out crossing to other plants; the possibility of achieving high copy number of foreign genes; and reduction in plant energy costs because importation of proteins into chloroplasts, which is highly energy dependent, is reduced.<br>
[oo98]	Plant plastids (chloroplasts, amyloplasts, elaioplasts, etioplasts, chromoplasts, etc.) are<br>
the major biosynthetic centers that, in addition to photosynthesis, are responsible for producing many industrially important compounds such as amino acids, complex carbohydrates, fatty acids, and pigments. Plastids are derived from a common precursor known as a proplastid; thus, the plastids in a given plant species all have the same genetic content.<br>
[0099]	Plastids of most plants are maternally inherited. Consequently, unlike heterologous genes<br>
expressed in the nucleus, heterologous genes expressed in plastids are not disseminated in pollen.<br><br>
Therefore, a trait introduced into a plant plastid will not be transmitted to wild-type relatives. This offers an advantage for genetic engineering of plants for tolerance or resistance to natural or chemical conditions, such as herbicide tolerance, as these traits will not be transmitted to wild-type relatives.<br>
[ooiooj	The plastid genome (plastome) of higher plants is a circular double-stranded DNA<br>
molecule of 120-160 kb which may be present in 1,900-50,000 copies per leaf cell (Palmer, 1991). In general, plant cells contain 500-10,000 copies of a small 120-160 kilobase circular genome, each molecule of which has a large (approximately 25 kb) inverted repeat. Thus, it is possible to engineer plant cells to contain up to 20,000 copies of a particular gene of interest; this can potentially result in very high levels of foreign gene expression.<br>
[ooioi]	Oils of the subject invention are applicable for, and can be specially tailored for,<br>
industrial as well as various food uses. Aside from cooking oil, itself, the subject invention also includes "no sat" products such as potato chips and the like (see U.S. Patent No. 6,689,409, which claims a fried food composition comprising potatoes and a canola oil; the subject invention, however, can be used to improve the compositions described in the '409 patent).<br>
[ooio2]	Plants of the subject invention can be crossed with other plants to achieve various<br>
desirable combinations of characteristics and traits. Even further improvements can be made by crossing the subject plants, using known breeding technique and other advantageous sources of germplasm such as other canola lines having additional or other beneficial traits and characteristics. Another example would be crosses with a line having a plastidial delta-9 desaturase.<br>
room]	Thus, the subject invention can be used to achieve less than 3.5% total saturated fatty<br>
acids in commercial oil under variable environmental conditions (and less than 3% total saturated fatty acids in seed oil in breeder seed). This can be accomplished with no reduction in the quality and quantity of storage proteins, with no increase in indigestible fiber in canola meal, and no negative impact on seed yield (or other desirable agronomic traits) per acre.<br>
[ooio4]	Following is a list of the common names of fatty acids, as used herein, together with their<br>
number of carbon atoms and double bonds. Saturated fats have zero double bonds.<br><br>
(Table Removed)   All patents, patent applications, provisional applications, and publications referred to or cited herein are incorporated by reference in their entirety to the extent they are not inconsistent with the explicit teachings of this specification.<br>
loom]	Unless specifically indicated or implied, the terms "a", "an", and "the" signify "at least<br>
one" as used herein.<br>
[00107]	Following are examples that illustrate procedures for practicing the invention. These<br>
examples should not be construed as limiting. All percentages are by weight and all solvent mixture proportions are by volume unless otherwise noted.<br>
Example 1 - Delta-9 desaturase gene rebuilding<br>
A delta-9 desaturase gene of the subject invention was redesigned for plant expression through a combination of changing Aspergillus nidulans sequence to plant-preferred translational codons, introducing unique restriction enzyme sites, and removing unwanted sequences and some secondary structure. The redesigned gene was synthesized by Operon, Inc. The sequence of the open reading frame for this polynucleotide is provided here as SEQ ID NO:1. The<br>
sequence of the ORF preceded by a Kozak sequence and a BamHI cloning site (caps), plus a translational terminator at the end of the ORF (caps), is provided in SEQ ID NO:2.<br>
Example 2 - Delta-9 desaturase plant transformation vector construction<br>
[00109]	The BamHI-BstEll gene fragment was cloned into a vector between the Pv beta-phaseolin<br>
promoter and Pv beta-phaseolin 3' UTR (pPhas-UTR). This construct was named pOIL. The promoter-gene-UTR fragment was excised from pOIL by digestion with NotI, blunted, and cloned into the blunt Pmel site of vector pOEAl. The final vector was named pPD9-OEAl.<br>
Example 3 — Plant transformation with pPD9-OEAl<br>
foouoj	Plasmid vector pPD9-OEAl was transformed into Agrobacterium tumefaciens [strain<br>
C58GV3101 (C58ClRifR) pMP90 (GmR). Koncz and Schell, Mol. Gen. Genet (1986)]. The delta 9-desaturase plants were then obtained by Agrobacterium tumefaciens mediated plant transformation<br>
[ooiii]	Arabidopsis was transformed with the "dip method," a procedure well known in the art.<br>
Plants were selfed, and dried seed was collected for FAME (fatty acid methyl ester) analysis.<br>
[ooii2]	The protocol used for canola transformation was as described by Katavic [Katavic,<br>
Campbell, L., Friesen, L., Palmer, D., Keller, W., and Taylor, D.C. (1996), "Agrobacterium-mediated genetic transformation of selected high erucic acid B. napus cultivars," /* Canadian Plant Tissue Culture and Genetic Engineering Conference, Saskatoon, SK, June 1-4,1996], with modifications for DAS's Nexera line. Hypocotyl sections were isolated from 6-day-old seedlings of B. napus, cv Westar or Nexera 710 and were cultured on callus initiation medium prior to transformation. On the day of transformation, the hypocotyls were coincubated with an Agrobacterium culture containing the plasmid pPD9-OEAl with the trait gene such that a fragment of plasmid DNA including the delta 9-desaturase gene was incorporated into the cell chromosome. After a co-cultivation period, the hypocotyls were transferred to callus initiation medium containing glufosinate ammonium as the selection agent. Healthy, resistant callus tissue was obtained and repeatedly transferred to fresh selection medium for approximately 12 to 16 weeks. Plants were regenerated and transferred to Convirons growth chambers. Plants were selfed to obtain seed. If transgenic plants were sterile, they were crossed with pollen from unmodified Nexera 710 lines. Dry seed was harvested for FAME analysis.<br>
Example 4 - Canola Event Sorting Process, and Summary of Canola Results<br>
Unless otherwise specified, the following procedures were used to obtain the Nex 710/Delta-9 canola seeds, the data for which is presented in subsequent Examples.<br>
[ooi 14]	In general, there were four main steps for developing, selecting, or sorting events: sorting<br>
initial transgenic events, sorting Tl and T2 seed, sorting Tl or T2 plants, and sorting events by field performance. Transformed callus was first regenerated to TO plants.<br>
[mns]	For the initial sorting, 107 putative transgenics were screened by agronomics and by<br>
southern blotting (simple or complex, i.e., more than 3 copies). Multiple seeds per event were screened. Tl seed saturates were determined, C16:1/C16:0 ratios were determined (to infer catalytic efficiency), and segregation of biochemical phenotypes were determined. Based on these data, and seed availability (timing, amount), a limited subset of seed was advanced.<br>
[ooii6]	For sorting Tl and T2 seed (a few events were advanced by one generation), half-seed<br>
analysis was conducted, and possible homozygotes were identified. Based on this data, half-seeds from segregating populations were selected for greenhouse growth.<br>
loom]	The next main step was sorting Tl or T2 plants (a few events were advanced one<br>
generation). Southerns were conducted to determine transgene integration complexity. Zygosity was also determined by INVADER assays (Third Wave Technologies, Inc.). These data were used to select seed for field trials.<br>
[ooiis]	For Tl greenhouse studies, 30 seeds per event were subjected to half-seed analysis. Nex<br>
710 canola was used as a commercial check. All individuals within the event were zygosity sampled to determine allele copy number for Delta-9 and PAT. This was followed by PCR and indoleacetamide hydrolase (IAAH; a negative scoreable or screenable marker) analysis. All individuals within the event were leaf painted to determine the PAT segregation ratio. Southern analysis was then conducted on those individuals approaching a "no sat" profile, positive IAAH, and homozygous for PAT and Delta-9.<br>
[ooii9]	These were then used for T2 analysis (100 seeds from each of the above plants were half-<br>
seed analyzed). INVADER was used to verify copy number and to see if the event was segregating for D9 and PAT. Leaf painting was used to see if lines are segregating for PAT. 10 seed bulk fatty acid data was collected from plants based on half-seed data, INVADER results, LP, and lAAH-positive.<br>
[00120]	The fourth main step was sorting events by field performance (plant T2 or T3 seed,<br>
analyze T2 or T3 plants, then T3 or T4 seed). There was a wide sampling of transgenic events.<br>
Agronomics, Southerns, and zygosity were analyzed.   Batch seed oils analysis was also conducted. Based on these data, events were selected for crossing to increase gene dosage.<br>
The following selection criteria was used to advance lines to field studies. 224 lines (including nulls) from 23 events (6 reps/entry) were evaluated in replicated nurseries at 3 locations. 6 T3 and 17 T2 events were planted. Selection of lines/event going to field was based on insert number and half-seed analysis followed by a 10-seed bulk fatty acid analysis of seed from each plant. The percent total sat range of selected lines was in the approximate range of 3.3-4.5%. The following T3 events were selected for further development:<br>
 (plus sib-nulls) for further copy number verification and to determine if lines were segregating for PAT. The D9 lines were chosen based on having less than 3.5% total saturates.<br>
[00124]	10 Southern tissue samples were taken from the 3 most promising T2 lines, which were<br>
chosen based on having less than 3.5% total saturates. All tissue-sampled plants were self-pollinated. Fatty acid analysis was determined based on 10 seed bulk from selfed plants (455 samples), and 1 gram of bulk seed sample from OP rows (1445).<br>
Generally, no major agronomic penalty associated with Delta-9 was observed, and some lines exhibited an approximately 10% increase in seed weight. Agronomic results are discussed in more detail below in Example 18. General observations regarding distributions of individual fatty acid components are discussed below in Examples 11-13. Summaries of mean TSAT data, % changes in TSATs, and % changes for certain fatty acid components for events 218-11.30,36-11.19, and 69-11.19 are presented in Tables 4-7. Generally, -30% to -40% reductions in TSATs were observed, relative to the sib-null and wild type. However, even further improvements are discussed in more detail below and can be made with further crosses, for example.<br>
(TableRemoved)    <br>
Example 5 - Molecular characterization of plants<br>
loons]	Leaf samples were taken for DNA analysis to verify the presence of the transgenes by<br>
PCR and Southern analysis, and occasionally to confirm expression of PAT protein by ELISA.<br>
[00129]	5 A. Protocol for PCR analysis for delta-9 desaturase. The following two primers were<br>
used:<br>
Delta-9 forward B:<br>
5' TGA GTT CAT CTC GAG TTC ATG 3' (SEQIDNO:3)<br>
Delta-9 reverse B:<br>
5' GAT CCA ACA ATG TCT GCT CC 3' (SEQIDNO:4) This primer pair yields a -1380 bp fragment after amplifying the delta-9 gene.<br>
The following cycling protocol was used in this screen with an MJ Tetrad thermal cycler:<br>
1.	94° C, 2 minutes<br>
2.	94° C, 1 minutes<br>
3.	50° C, 2 minutes<br>
4.	72° C, 3 minutes, + 5 seconds / cycle extension<br>
5.	repeat Steps 2-4 25 times<br>
6.	4° C until ready for analysis, or at least 2 minutes<br>
SB. Protocol for the extraction of plant genomic DNA for Southern analysis. The DNeasy Plant Maxi Kit from Qiagen was used. The protocol in the booklet was used with the following changes to the elution part. Buffer AE was diluted 1:10 with DNA grade water (Fisher No. BP561-1). Two elutions were performed using 0.75ml of the diluted AE buffer pre-warmed to 65° C. DNA was precipitated with isopropanol and washed with 70% ethanol. The DNA pellet was resuspended in 100 ul of IX TE buffer. DNA concentration was quantitated. 6 ug of DNA was aliquoted and adjusted to a final volume of 40 ul. Samples were stored at -20° C.<br>
[00132]	5C. FAME analysis (Direct FAME Synthesis from Seeds with Methanolic HbSQdl<br>
[ooi33j	The protocol for FAME analysis was as follows.<br>
[ooi34]	GC Specs<br>
[ooi35]	Gas Chromatograph: Hewlett-Packard 6890 with dual injection ports and dual flame<br>
ionization detectors.<br>
[ooi36]	Data System: HP Chemstation, Leap Technologies, Carrboro, NC 27510, PAL System.<br>
[00137]	Column: J&amp;W capillary column, DB-23, 60M x 0.25mm i.d. with 0.15 microfilm<br>
thickness, maximum operating temperature 250° C. Catalog number: 122-2361.<br><br>
 [OOI38J	Temperature profile: Equilibration time: 1 minute. Initial temperature: 50° C. Initial<br>
time: 3 minutes. Increase rate: 40° C/minute. Final temperature: 240° C. Final time: 7.25<br>
minute.<br>
[00139]	FAME procedures for Arabidopsis. Add 100 ul (50 ug) 15:0 Standard into a clean<br>
16x125 mm glass tube (Internal Standard stock solution: 500 ug/ml of C15:0 TAG in 2:1<br>
chloroform: isopropanol). Dry Standard under nitrogen in evaporation water bath at 55° C.<br>
[ooi40]	When dry add 2 ml 1N methanolic H2SO4 with 2% BMP (for 100 ml: 95,22 ml methanol,<br>
2.772 ml H2SO4, 2 ml DMP = 2,2-dimethoxypropane). Heat tube to 85° C.<br>
[ooi4i]	Weigh ~5 mg Arabidopsis seeds into clean 16x125 mm glass tube and record exact<br>
weight.<br>
[00142]	To destroy lipases, add hot meth. H2SO4 with Standard to tube with seeds, incubate for 15<br>
minutes at 85° C.<br>
[00143]	Cool vial down to -50° C, then crush seeds with glass pestle in mini grinder.<br>
[00144]	Transfer sample back into tube and incubate for at least one hour under nitrogen at 85° C.<br>
Cool vial on ice. Add first 0.5 ml 0.9% NaCl, then 250 ul 17:0 Standard in hexane (0.1335<br>
mg/ml methyl ester stock solution). Vortex, centrifuge at lOOOg for 5 minutes.<br>
[00145]	Transfer 100-200 ul with Pasteur pipette into 1.5 ml vial with conical insert (0.5 ml).<br>
[00146]	Inject 5 ul into GC.<br>
[00147]	FAME procedures for Canola. Add 100 ul (50 ug) 15:0 Standard into a clean 16x125<br>
mm glass tube (Internal Standard stock solution:    500 ng/ml of C15:0 TAG in 2:1<br>
chlorofornrisopropanol). Dry Standard under nitrogen in evaporation water bath at 55° C.<br>
loom]	When dry add 2 ml IN methanolic H2SO4 with 2% DMP (for 100 ml: 95,22 ml methanol,<br>
2.772 ml H2SO4, 2 ml DMP = 2,2-dimethoxypropane). Heat tube to 85° C.<br>
[ooi49]	Weigh one canola seed in clean tube and record exact weight.<br>
[miso]	To destroy lipases, add seed sample to tube containing Standard with hot methanol<br>
H2SO4&gt; incubate for 15 minutes at 85° C.<br>
[ooisi]	Cool vials down to -50° C, then crush seeds with glass pestle.<br>
[ooi52]	Incubate for at least 1 hour under N2 at 85° C.<br>
[00153]	Cool vial on ice. Add first 0.5 ml 0.9% NaCl, then 250 \i\ 17:0 Standard in hexane<br>
(0.1335 mg/ml methyl ester stock solution). Vortex, centrifuge at lOOOg for 5 minutes.<br>
[00154]	Transfer 100-200 ul with Pasteur pipette into 1.5 ml vial with conical insert (0.5 ml).<br>
fooissj	Inject 5 ul into GC.<br><br>
Example 6 - Arabidopsis Results<br>
[M156]	Initial results are illustrated in Figure 1, showing that a greater than 60% reduction of<br>
saturated fatty acids was achieved. Also, more 16:1 (5.9% for example) than 16:0 (4.4%) was achieved.<br>
[ooi57]	No Sat Oil via A9-CoA-Desaturase Approach<br>
[ooiss]	FAME analysis<br>
[00159]	T2 seeds from approximately 18 additional transformants were analyzed. This data (see<br>
Table 8 and Figure 2) show a reduction in "sats" of up to 60-70%. This is an even stronger reduction in saturated fatty acids than the initial data (see Figure 1) indicated. It is important to note that the T2 generation is still segregating; thus, even better performing lines in following generations are expected. This point is true for all Tl, T2, T3, and other initial generations (including canola lines) as reported elsewhere herein, until the trait is fixed and the line is homozygous for the transgene. (Stable lines and plants where the traits are fixed were produced and are described in subsequent Examples.)<br>
Values are from a single sample prep and GC run (not averages)<br>
TF-21 behaves as a wild-type (non-transformed) plant; possible explanations include gene silencing or non-transgenic escape (inadequate selection with glufosinate herbicide)<br>
"?" indicates that identity of the peak on the GC chromatogram is questionable, or unknown<br>
Example 7 - Westar data<br>
Protocols similar to those described in Example 5C were applied to canola lines derived from well-known "Westar" canola. As illustrated in Figure 3, the indicated saturated fats were reduced by over 43%, and a 50% reduction was achieved when 24:0 was included.<br>
[00161]<br>
[00162]Example 8 - Exemplary Nexera 710 data<br>
Protocols similar to those described elsewhere herein were applied to canola lines derived from well-known "Nexera 710" commercially elite canola. Total saturates were calculated used methodology discussed herein and as specified below.<br>
Total saturates are derived from the sum of 16:0 + 18:0 + 20:0 + 22:0 + 24:0 fatty acids. Some notable saturate levels in single seeds are presented in Table 9. Oil profiles are presented as mol% values. The mol% value incorporates the formula weight of each specific fatty acid into the calculation. It uses the mass of a given fatty acid species (peak area, or the same value used to directly calculate % fatty acid), divided by the formula weight for that fatty acid species.<br><br>
[00163]<br><br>
Profiles from the seeds with the lowest total saturates were analyzed (seeds 113 a 11.19 #4,113a 11.19 #8,5 11.19 #6 and 5 11.19 #8). Unmodified Nexera 710 germplasm values came from the same FAME analysis run. Plants were grown in the Convirons growth chamber, so actual mol% values may differ from field grown seed. In general, transgenic plants show a reduction in 16:0, 18:0 and 20:0, and increases in 16:1. 18:0 levels generally fell from an average of 1.4% (upper 2.08%, lower 0.81%) in Nexera 710 to 0.1% average (upper 0.6%, lower 0%) in select transgenic material. Also, 16:0 levels fell from an average of 4.6% (upper 5.12% to lower 4%) to 3.0% (upper 3.41% to lower 2.63%). Likewise, the 20:0 levels dropped from 0.5% average in Nexera 710 to 0% in the selected transgenics. The 16:1 levels were undetectable in Nexera 710, increasing to an average of 2.3% (upper 2.71% to lower 1.72%) in<br>
transgenics. The average 18:3 levels were slightly increased in the small transgenic population, but the range of values overlapped with the unmodified Nexera 710 samples. These results can be summarized as follows:<br><br>
[m)i64]	Example 9 - Further Canola Data<br>
[00165]	Protocols similar to those described elsewhere herein were applied to additional canola<br>
lines derived from well-known "Nexera 710" commercially elite canola. Total saturates and the weight percent of the individual types of fatty acids, as indicated below, were calculated using methodology discussed herein.<br>
[00166]	Figures 4A-C and 5A-C show representative results, from Events 36-11.19 and 218-<br>
11.30 respectively, that demonstrate reduced saturated fatty levels that are obtainable by practicing the subject invention. By making further manipulations according to the subject invention, the saturated fat levels exemplified here can be even further reduced. All of this data were obtained from selfed transgenic canola plants as indicated.<br>
[00167]	In summary, for Event 36-11.19, T2 half seed analysis from greenhouse-grown plants had<br>
total saturates as low as 2.57%. Total saturates for T3 whole seeds, from greenhouse-grown plants, were as low as 3.66%. Results are shown graphically in Figure 4A (numerical data are in Figures 4B and 4C). For Event 218-11.30 greenhouse-grown plants, T2 half seed analysis revealed total saturates to be as low as 2.71%. T3 whole seeds had total saturates as low as 3.37%. Results are shown graphically in Figure 5A (numerical data are in Figures SB and 5C). For reference, NATREON has 6.5% total saturates, on average, under field conditions.<br>
[00168]	By making further improvements according to the subject invention (such as additional<br>
rounds of selfing, crosses with other superior lines, increasing desaturase gene copy number [either by additional transformation, by further breeding crosses, and the like], changing timing of desaturase expression, and mutagenesis), even greater levels of reduction of total saturated fat levels can be achieved.<br><br>
(Table Removed)   gain, Figures 4A-C and 5A-C show some representative results that show fatty acid profiles of various plants having events 218-11.30 and 36-11.19. Generally, these results demonstrate that not only are the 16:0 and 18:0 levels greatly reduced (with a resulting increase in corresponding unsaturated levels), but the 20:0,22:0, and 24:0 levels are also advantageously, and surprisingly and unexpectedly, reduced. In some cases, 18:2 and 18:3 levels can also be reduced, which enhances the oxidative stability of the improved oil. Furthermore, any of the ratios suggested above (such as 16:0-16:1,18:0-18:1, and, for example, 18:0-[20:0+22:0+24:0]) can be used to define advantageous results of practicing the subject invention. Combined percent reductions in total C20:0 + C22:0 + C24:0 are also surprisingly achieved according to the subject invention. Thus, the subject invention provides plants have advantageous and improved fatty acid profiles, as exemplified herein. By making further improvements according to the subject invention, even better reductions in saturates, increases in "no sats," and better ratios can be achieved.<br>
loom]	For example, various calculations, using the following data from Figure 5C and Figure<br>
4B, can be used to illustrate accomplishments of the subject invention. Sample data from Figure 5C and Figure 4B are presented in the following Table. The amount of each indicated fatty acid is indicated for each event and in parentheses for the relevant control plant(s).<br>
loom]	Looking at the 218-11.30 event, the total contribution to saturates by the C20:0, C22:0,<br>
and C24:0 components is 1.08% in the control, but these components are advantageously decreased to 0.26% in a canola line of the subject invention. This represents an over 4-fold decrease in these saturates. Likewise, each component can be considered individually. Again looking at the 218-11.30 event, the C20:0 component is 0.62% in the control / wild-type, while it is reduced about 5.6 times in the plant line ofthe subject invention (down to 0.11%). The C22:0 component is reduced about 2 2/3 times: 0.12% in the d-9 desaturase plant line, which is down from 0.32% in the control line that lacks the desaturase gene. C24:0 is reduced about 4 2/3 times, from 0.14% down to 0.03 %.<br><br>
 [mm]	For Event 36 (or 36-11.19), two lines have C20:0, C22:0, and C24:0 content of 0.32,<br>
0.15, and 0.04, and 0.30, 0.14, and 0.07, respectively. Compared to the control having 0.64, 0.42, and 0.2 1 respectively, these lines have about half the C20:0, and at least about a three-fold reduction in C22:0 and C24:0. The first line mentioned above actually exhibits an over 5X reduction in C24:0.<br>
[ooi 75]	It will quickly become apparent that a great number of similar calculations can be made<br>
for any of the other lines of the subject invention, for any of these preferred fatty acid components. These illustrations should not be construed as limiting, and any such novel reductions and ratios can be used to define the subject invention.<br>
Example 12 - Further Half-Seed Data of Subsequent Generations<br>
Further half-seed FAME analysis is set forth in Table 16. This Figure shows total saturates as low as 2.64% in a T3 generation and 2.66% in a T4 generation. Table 17 shows the copy number of D-9 desaturase genes present in the respective lines (see Sample ID in Table 16 and ID column in Table 17). Effects of copy number are discussed in more detail below in Examples Hand 19.<br>
Example 13 - Further Analysis of Half-Seed Data from Example 12. and Further Data Showing Fatty Acid Shifts, Increases in Unsaturates. and Decreases in Saturates<br>
[ooi 77}	Half-seed data from the T3 field trials was plotted to illustrate various comparisons of the<br>
fatty acid contents to the "total sat" data. Figures 6A and 6B clearly show the reductions in C16:0 and increases in C16:l in the transgenic events as compared to the nulls (events with a non-functional insert) and wild-type controls (non-transformed lines). Figures 6C and 6D clearly show the reductions in Cl 8:0 and increases in Cl 8:1 in the transgenic events as compared to the nulls and wild-type controls. Figures 6E and 6F clearly show the reductions in C20:0 and C22:0, respectively, in the transgenic events as compared to the nulls and wild-type controls.<br>
loons]	Similar results, using data obtained as discussed in Example 4, are also illustrated with<br>
bar graphs. Figures 6G and 6H clearly show shifts and reductions in C16:0, and shifts and increases in C16:l in the transgenic events, as compared to the nulls and wild-type controls. Figures 61 and 6J clearly show shifts and reductions in Cl 8:0, and shifts and increases in Cl 8:1 in the transgenic events, as compared to the nulls and wild-type controls. Figures 6K and 6L show similar bar graphs for C18:2 and C18:3.<br>
imi79]	The plots discussed above and Figure 6M also clearly illustrate the very surprising<br>
reduction in total saturates, as compared to already very good Nex 710 lines. Figure 6N shows distributions for 1000 seeds.<br>
Example 14 - Decreasing saturated fat levels with multiple Delta-9 desaturase genes<br>
[00180]	Results from greenhouse increases and field trials suggest that there is a relationship<br>
between the reduction in total saturates and Aspergillus delta-9 desaturase copy number.<br>
[ooisi]	14A: FAME Analysis Protocol for Event Sorting and Effect of Transgene Copy Number<br>
[00182]	Sample preparation<br>
[ooisi]	1.        Obtain weight of individual seeds and place into plastic mother plate.<br>
[ooi84]	2.        Add 2, 1/8" balls to each well<br>
[miss]	3.        Take mother plate to liquid handler Hamilton:<br>
add 400 uL of heptane with IS (C11:0) and surrogate (C15:0 FAEE) then add 100 uL of sodium methoxide (.5N)<br>
[ooi86]	4.        Cap inserts with strips caps and Geno-grind for 5 minutes (Ix @ 500)<br>
[00187]	5.        Replace strip cap<br>
foo/ssj	6.        Add plate lid with a rubber mat (extra sealing). Tape the lid with black electrical<br>
tape and remove bottom of mother plate to expose vials<br>
[ooi89]	7.        Place the plate onto vortex/heater for 15 min. at 37°C / 60 rpm (the vortex wells<br>
are filled with sand)<br>
[mm]	8.        Centrifuge plate @ 3500 rpm for 2 min.<br>
[00191]	9.        Place the bottom lid back and remove lid/strip cap. Transfer 350 ul of top layer<br>
into the daughter plate using Hamilton.<br>
loom]	10.      Then add 400 uL of heptane with IS and surrogate to the extraction plate.<br>
[00193]	11.      Repeat steps 5 to 10 two more times (for a total of 3 transfers)<br>
[ooi94]	12.      Keep the extract plate on the Hamilton after last transfer. Transfer 50 ul of extract<br>
into glass insert mounted in aluminum block containing 450 ul heptane with Cl 1<br>
[OH195]	13.       Inject on GC<br>
[00196]	GC/FID analysis<br>
[00197]	GC parameters:<br>
[ooi98]	Injection 1 ul splitless per sample<br>
[ooi99]	Column-<br>
[00200]	DB23, 15 meters, 0.25 mm I.D. and 0.1 Sum film thickness<br>
[00201]	GC parameters-<br>
[00202]	Oven temperature program-<br>
[00203]	70°C ho Id for 2.15min (splitless)<br>
IW204]	70°C - 150° @ 25°C min.,<br>
loom]	150°C - 180° @ 5°C min.,<br>
loom]	180°C - 220° @ 25°C min.,<br>
[00207]	220°C hold for 2 min.<br>
[00208]	Injector temp. - 230°C<br>
[00209]	Detector temp. - 240°C<br>
[002W]	Make-up gas - Nitrogen @ 25mL/ min.<br>
[002H]	FID fuel - air @ 400mL/min.,<br>
[00212]	hydrogen @ 40mL/ min<br>
[00213]	front injector: purge time 1 min.<br>
[00214]	purge flow 35 ml/min<br>
[00215]	back injector: purge time 2.15 min.<br>
 [W)2i6]	purge flow 35 ml/min.<br>
[00217]	front inlet pressure: 1.0 mL/min - constant flow<br>
[00218]	back inlet pressure: 1.0 mL/min - constant flow<br>
[00219]	Run time - 14.95 minutes,<br>
[00220]	Flow rates - helium constant flow @ 1 mL/ min<br>
[00221]	Acquisition sequence<br>
[00222]	96 samples are splited between front and back column to minimize the build on the liner.<br>
The sample list is built with 5 injection methods corresponding to the type of sample being<br>
injected. The first 5 samples injected are always:<br>
[00223]	1.        Matrix<br>
[00224]	2.        Matrix<br>
[00225]	3.         Standard 1<br>
[00226]	4.        Canola Positive Control<br>
[00227]	5.        Reagent Blank<br>
[00228]	6-27.   Canola samples<br>
[00229]	33.      Standard2<br>
[00230]	34-54. Canola samples<br>
[00231]	55.       Standards<br>
[00232]	Each list comprises 3 events of 16 samples (48 samples).<br>
[00233]	Standard contains 25 ppm FAMES total distributed as follow:<br><br><br>
[00234]<br><br>
14B: Preliminary Southern Blot Analysis to Approximate Copy Number<br><br>
roo235]	The DNA preparation protocol used for these purposes was as follows. Approximately 6<br>
micrograms of DNA was digested with Hindlll, and digested DNA was run on a 0.75% agarose gel. Blotting onto positively charged nylon membrane and hybridization followed typical protocols (Maniatis, Roche Applied Science, Inc.). The probe consisted of a DIG-labeled (kit from Roche Applied Science, Inc.) PCR product derived from the Aspergillus delta-9 desaturase gene. Washes were done twice for 5 minutes in room-temperature 2XSSC/0.1% SDS, then twice in 65+C 0.1XSSC/0.1% SDS. Hybridized bands were visualized with the DIG-Luminescent Detection Kit according to manufacturer's guidelines (Roche Applied Science, Inc.). Hybridizing bands were counted, and transgenic samples were initially described as 'Simple' if they displayed 1 to 3 bands, or "Complex" if more than 3 bands.<br>
[00236]	14C: Comparison of Transgene Copy Number and Reduction in Saturated Fatty Acids in<br>
Transgenic Events<br>
[00237]	The following definitions apply to the <br><br><br>
roo23<jj></jj><br>
The "Ratio of Saturate Reduction" was used to rank events because it generally used seed from the same transgenic event. This direct comparison helps reduce variability between plants caused by tissue culture and growing plants at different times.<br>
The above data shows an apparent gene dosage effect; more copies of the transgene tend to cause a more effective reduction in saturated fatty acids. For example, there are 57 "non-control" plants represented above. These 57 plants can be divided into three groups of 1 9 plants. The top set of plants (exhibiting the best reductions in saturates and the lowest levels of saturates) have 1 1 of 19 "complex" events (more than 3 copies of the desaturase gene). This set contained 8 of 1 9 events characterized as 'Simple' or 'probably Simple.' The middle set of 19 plants had only 6 of 1 9 "complex" events ( 1 3 of 1 9 "simple" or "probably simple" events). Still further, the third set of 1 9 plants (showing, relatively, the least reductions in saturates) contained only 3 complex events, with 16 of the 19 events being "simple" or "probably simple." Thus,<br><br>
plants with cells having more than 3 copies of the desaturase appear to show a better reduction in saturates than plants with cells having only 1 copy of the gene.<br>
Example 15 - Segregation of Oleic and Vaccenic Acids<br>
[00240]	Vaccenic acid is a Cl 8:1 with the double bond in the delta-11 position. Vaccenic acid is<br>
formed by elongating delta-9 C16:l outside of the plastid. The following is important because other analytical methods discussed herein combined oleic and vaccenic acid peaks together into a single percent composition that was labeled as "oleic." That is, there was no separation of the two unless otherwise indicated. By subtracting out the vaccenic acid contribution, it is presently demonstrated that the percent contribution of oleic acid is, preferably and advantageously (and surprisingly), maintained at less than 80% while still achieving a reduction (to "no sat" or "low sat" levels) of overall) total saturates. Two types of analyses were used to demonstrate this, as set forth in the following two Examples.<br>
Example 16 - Canola Delta-9 Seed Extraction and Analysis for Vaccenic Acid SOP<br>
[00241]	Figures 7A and 7B illustrate data obtained using the following protocol.<br>
[00242]	Sample preparation (same as before)<br>
[00243]	1.        Obtain weight of individual seeds and place into plastic mother plate.<br>
[00244]	2.        Add 2, 1/8" balls to each well<br>
[00245]	3.        Take mother plate to liquid handler Hamilton<br>
add 400 uL of heptane with IS (Cl 1:0) and surrogate (C15:0 FAEE) then add 100 uL of sodium methoxide (,5N)<br>
[00246]	4.        Cap inserts with strips caps and Geno-grind for 5 minutes (Ix @ 500)<br>
[00247]	5.        Replace strip cap<br>
[00248]	6.        Add plate lid with a rubber mat (extra sealing). Tape the lid with black electrical<br>
tape and remove bottom of mother plate to expose vials<br>
[00249]	1.        Place the plate onto vortex/heater for 15 min. at 37°C /60 rpm (the vortex wells<br>
are filled with glass beads)<br>
[00250]	8.        Centrifuge plate @ 3500 rpm for 2 min.<br>
[00251]	9.        Place the bottom lid back and remove lid/strip cap. Transfer 350 pi'of top layer<br>
into the daughter plate using Hamilton.<br>
[00252]	10.       Then add 400 uL of heptane with IS and surrogate to the extraction plate.<br><br>
 [00253]	11.       Repeat steps 5 to 10 two more times (for a total of 3 transfers)<br>
[00254]	12.      Keep the extract plate on the Hamilton after last transfer. Transfer 50 ul of extract<br>
into glass insert mounted in aluminum block containing 450 ul heptane with IS Cl 1<br>
[00255]	13.       Inject on GC<br>
[00256]	GC/FID analysis (different from FAMEs profile)<br>
[00257]	GC parameters:<br>
[00258]	Injection 1 ul splitless per sample<br>
[00259]	Column-<br>
[M260]	BPX 70 from SGE, 15 meters, 0.25 mm I.D. and 0.25 urn film thickness<br>
[00261]	GC parameters-<br>
[00262]	Oven temperature program-<br>
[00263]	70°C hold for 2.15min (splitless)<br>
[00264]	70°C - 140° @ 25°C~ min.,<br>
[00265]	140°C hold for 14 min<br>
[00266]	140°C - 180° @ 10°C~ min.,<br>
[00267]	180°C hold for 3 min<br>
[00268]	180°C - 220° @ 25°C min.,<br>
[00269]	220°C hold for 3 min.<br>
[00270]	Injector temp. - 230°C<br>
[00271]	Detector temp. - 240°C<br>
[<m272 make-up gas nitrogen min.></m272>
[00273]	FID fuel - air @ 400mL/ min.,<br>
[00274]	hydrogen @ 40mL/ min<br>
[00275]	front injector: purge time 1 rninA<br>
[00276]	purge flow 35 ml/min<br>
[00277]	back injector: purge time 2.15 min.<br>
[00278]	purge flow 35 ml/min.<br>
[00279]	front inlet pressure: 1.0 mL/min - constant flow<br>
[00280]	back inlet pressure: 1.0 mL/min - constant flow<br>
[oo28ij	Run time - 30.55 minutes,<br>
[00282]	Flow rates - helium constant flow @ 1 mL/ min<br>
[00283]	Acquisition sequence<br><br><br>
[00284]<br>
[00285] [00286] [00287] [00288] [00289] [00290] [00291] [00292] [00293] [00294]<br><br>
96 samples are splited between front and back column to minimize the build on the liner. The sample list is built with 5 injection methods corresponding to the type of sample being injected. The first 5 samples injected are always:<br>
1.	Matrix<br>
2.	Matrix<br>
3.	Standardl<br>
4.	Reagent Blank<br>
5-32.   Canola samples<br>
33.       Standard2<br>
34-54. Canola samples<br>
55.      Standards<br>
Each list comprises 6 events of 8 samples (1 seed = 1 sample) (48 samples). Standard contains 200 ppm FAMES total distributed as follow:<br><br>
Example 18 - Achieving "No Sats" While Maintaining Superior Agronomic Traits<br>
[00339]	Agronomic measurements were made at the various field sites, comparing the transgenics<br>
and controls. The conclusion was that overall, the transgenic plants behaved like and exhibited<br>
similar traits (aside from much-improved saturate levels) the Nex 710 controls. Thus, the subject<br>
transgene(s) has no consistent negative effect on plant health. A summary of the traits and rating<br>
system is provided in Table 23. Various criteria were used at various field site locations.<br>
[003-to]	Tables 24-27 numerically illustrate some representative results.    The following<br>
abbreviations are used in those figures:<br>
DTP = Days to Flower	EOF = Days to End of Flower<br>
HT = Height	SC = Sterile Counts<br>
DTM = Days to Maturity       LSV = Late Season Vigor<br>
LOD = Lodging	SD WT = Seed Weight<br>
[oo34i]	Table 24 shows agronomic data for lines from Event #218-11.30.  Table 25 shows<br>
agronomic data for lines from Event #36-11.19. Table 26 shows agronomic data for lines from Event 69-11.19.The foregoing demonstrates that the subject invention, i.e. achieving "no sat" canola, can be practiced, surprisingly, without adversely affecting other important agronomic traits.<br>
xample 19 - Dose Effect: Further Lowering of Saturates by Insertion of Multiple Copies of 5-9 Desaturase Genes<br>
[00342]	This Example further shows that "stacking" multiple copies of 8-9 desaturase genes has a<br>
surprising dose effect, which can be used to even further reduce saturates in oil seed plants such as canola. The following is greenhouse data measured using FAME procedures unless otherwise indicated.<br>
[00343]	Table 27 reports F3 10 seed bulk fatty acid data from native Nex 710, the simple events<br>
(218,36,69), and from each of the F3 seed packages tracing back to F2 plants that were selected on the basis of InVader assays. (The last two columns of Table 27 show approximate average total saturates for the respective plants, and approximate average reduction in total saturates, as compared to the Nex 710 control.) At that generation, Southern data was not used. Thus, based on saturate expression and InVader assays, a selection of suspected stacks and suspected parental siblings as well as nulls was made to be reconfirmed by growing out F3 plants and re-testing using Southerns.<br>
[00344]	For example, while not statistically analyzed, samples from the 41 "stack" lines have an<br>
average total saturates of 3.5%. The 21 "single" lines have an average total saturates of 3.84%.<br>
[00345]	Table 28 contains a summary of the suspected F3 stacks, suspected parental siblings, and<br>
nulls that were replanted for confirmation of copy number and zygosity. Lines named TDN0400141, TDN0400142, TDN0400145, TDN0400155, TDN0400158, and TDN0400160 were suspected to be homozygous stacks. Lines TDN0400189, TDN0400143, TDN0400197, TDN0400167, and TDN0400184 were suspected to be parental siblings out of the stacks. Lines TDN0400198, TDN0400199 were advanced as nulls selfed out of the stacks. TDN0400202, TDN0400204, and TDN0400208 are the simple events. This material is also currently in the field or recently harvested. Thus, field data is not yet available.<br>
[00346]	Figures 8 and 9 are pictures of two gels run with DNA from F3 plants. Similar issues<br>
with isolating DNA for Southern analysis were encountered at this step, so not all 9 of the plants submitted appeared in gels. Based on the gels, lines TDN0400141, TDN0400142, and possibly TDN0400158 (only 2 plants) are homozygous stacks. TDN0400145 is still segregating for event 36, and TDN0400155 is still segregating for event 69. Line TDN0400160 appears to have an odd segregation of bands which may indicate that it is segregating for all 3 simple events. Follow-up will be conducted TDN0400160. Plant #8 of from this cross has a low sat level of 2.6% which is consistent with being a triple stack of high zygosity. Additional molecular<br>
analysis could confirm this. Not all of the lines suspected to be parental siblings appear in the gels (Lines TDN0400184, TDN0400189, and TDN0400197 highlighted in the Event column of Table 29, discussed below). Based on the single plant fatty acid data, it appears that the 9 plants from TDN0400184 (suspected of being a sibbed-out simple event 218) have as low sat levels as the suspected stacks. That is, the trend in the data is that stacks consistently show a reduction in saturates compared to "sibbed-out" events (single transgenic events recovered from crosses of two transgenic events).<br>
[00347]	Table 29 contains 10 seed bulk fatty acid data from Nex 710, the simple events, and each<br>
single F4 plant. Nine plants of each suspected stack, null, simple event, and the like were regrown, tissue sampled for molecular analysis, and kept through to seed set for fatty acid analysis.<br><br><br><br><br>
Claims<br>
Claim 1.<br>
A canola plant that produces seed having an oil fraction comprising less than 3.5% total saturates and less than 80% oleic acid.<br>
Claim 2.<br>
The plant of claim 1 wherein said oil fraction comprises 70% to 78% oleic acid.<br>
Claim 3.<br>
The plant of claim 1 wherein said oil fraction comprises no more than 3% linolenic acid.<br>
Claim 4.<br>
The plant of claim 1 wherein said oil fraction comprises 70% to 78% oleic acid and no more than 3.5% linolenic acid.<br>
Claim 5.<br>
Seed produced by the canola plant of claim 1.<br>
Claim 6.<br>
Canola oil comprising less than 3.5 % total saturates and less than 80% oleic acid.<br>
Claim 7.<br>
A fried food composition comprising potato material and canola oil according to claim 6.<br>
Claim 8.<br>
The canola plant of claim 1 wherein said oil fraction has no more than 2.7% total saturates.<br><br>
Claim 9.<br>
The canola seed of claim 5 having an oil fraction comprising no more than 2.7% total saturates.<br>
Claim 10.<br>
The canola oil of claim 6 having an oil fraction comprising no more than 2.7% total saturates.<br>
Claim 11.<br>
A method of producing a fried food composition wherein said method comprises frying potato material in canola oil according to claim 6.<br>
Claim 12.<br>
A canola plant comprising at least one polynucleotide, stably incorporated in a genome of said plant, that encodes a delta-9 desaturase protein wherein the full complement of a nucleic acid molecule that encodes a protein of SEQ ID NO:5 maintains hybridization, after wash, with said polynucleotide, wherein said wash conditions are 2x SSC (Standard Saline Citrate) and 0.1% SDS (Sodium Dodecyl Sulfate) for 15 minutes at room temperature.<br>
Claim 13.<br>
The plant of claim 12 wherein said nucleic acid molecule comprises SEQ ID NO:1.<br>
Claim 14.<br>
The plant of claim 12 wherein said wash conditions are O.lx SSC and 0.1% SDS for 15 minutes at room temperature.<br>
Claim 15.<br>
The plant of claim 12 wherein said wash conditions are O.lx SSC and 0.1% SDS for 30 minutes at 55° C.<br>
Claim 16.<br>
The plant of claim 12 wherein said genome comprises two of said polynucleotides.<br><br>
Claim 17.<br>
The plant of claim 12 wherein said genome comprises three of said polynucleotides.<br>
Claim 18.<br>
The plant of claim 12 wherein said polynucleotide is operably linked to a seed-specific promoter.<br>
Claim 19.<br>
A plant produced by the seed of claim 5.<br>
Claim 20.<br>
A method of reducing saturated fat in the oil fraction of at least one seed of atransgenic canola plant, as compared to a wild-type oil fraction of seeds of a corresponding wild-type canola plant, wherein said method comprises producing a canola plant that expresses a polynucleotide that encodes a delta-9 desaturase protein wherein a nucleotide molecule that encodes said protein hybridizes with the molecule of SEQ ID NO:1.<br>
Claim 21.<br>
The method of claim 20 wherein the oil fraction comprises less than 80% oleic acid.<br>
Claim 22.<br>
The method of claim 20 wherein said protein causes a reduction in palmitic acid (16:0), a reduction in behenic acid (22:0), and an increase in palmitoleic acid (16:1) in the oil fraction of said seed of said transgenic canola plant relative to said corresponding wild-type canola plant.<br>
Claim 23.<br>
The method of claim 20 wherein said saturated fat is reduced by at least 60%.<br>
Claim 24.<br>
A polynucleotide comprising a sequence of nucleotides shown in SEQ ID NO:1.<br><br>
25.	The plant of claim I wherein said plant is at least 100cm in height<br>
with an average seed weight above 3 mg.<br>
26.	A canola plant substantially as herein described with reference to<br>
the     foregoing     description,     examples,     tables     and     the<br>
accompanying drawings.<br>
27.	Seed   produced   by the  canola  plant  substantially  as   herein<br>
described with reference to the foregoing description, examples,<br>
tables and the accompanying drawings.<br>
28.	Canola oil substantially as herein described with reference to the<br>
foregoing description, examples, tables and the accompanying<br>
drawings.<br>
29.	A fried food substantially as herein described with reference to<br>
the     foregoing     description,     examples,     tables     and     the<br>
accompanying drawings.<br>
30.	The canola seed substantially as herein described with reference<br>
to   the   foregoing   description,    examples,    tables   and   the<br>
accompanying drawings.<br>
31.	A method substantially as herein described with reference to the<br>
foregoing description, examples, tables and the accompanying<br>
drawings.A polynucleotide substantially as herein described with reference<br>
to   the   foregoing   description,   examples, tables   and   the<br>
accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LTEtQWJzdHJhY3QtKDA1LTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-1-Abstract-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LTEtQXNzaWdubWVudC0oMDUtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-1-Assignment-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LTEtQ2xhaW1zLSgwNS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-1-Claims-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LTEtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgwNS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-1-Correspondence Others-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LTItQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgwNS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-2-Correspondence Others-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LTItUGV0aXRpb24tMTM3LSgwNS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-2-Petition-137-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Correspondence Others-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Correspondence Others-(07-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Correspondence Others-(13-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDYtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Correspondence-Others-(06-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTctMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2573-DELNP-2007-Correspondence-Others-(17-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Correspondence-Others-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LURyYXdpbmdzLSgwNS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Drawings-(05-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUZvcm0tMy0oMDYtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-Form-3-(06-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LUdQQS0oMTMtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-GPA-(13-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1ERUxOUC0yMDA3LUdQQS0oMTctMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2573-DELNP-2007-GPA-(17-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU3My1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2573-delnp-2007-pct-210.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258585-phosphoramidate-alkylator-prodrugs.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258587-method-to-increase-the-efficiency-of-polymer-drag-reduction-for-marine-and-industrial-applications.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258586</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2573/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Apr-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOW AGROSCIENCES LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>9330 ZIONSVILLE ROAD , INDIANAPOLIS , IN 46268-1054, UNITED STATES OF AMERICA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THOMPSON , MARK</td>
											<td>623 SILVER WRAITH COURT ZIONSVILLE, IN 46077 , UNITED STATES OF AMERICA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>REDDY, SAM</td>
											<td>3704 SUMTER WAY CARMEL , IN 46032, UNITED STATES OF AMERICA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/036052</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/617,532</td>
									<td>2004-10-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258586-oil-and-seeds-with-reduced-saturate-levels-of-fatty-acids by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:06:01 GMT -->
</html>
