<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258552-homogentisate-prenyl-transferase-hpt-nucleic-acids-and-polypeptides-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:12:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258552:&quot;HOMOGENTISATE PRENYL TRANSFERASE (&quot;HPT&quot;) NUCLEIC ACIDS AND POLYPEPTIDES AND USES THEREOF&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;HOMOGENTISATE PRENYL TRANSFERASE (&quot;HPT&quot;) NUCLEIC ACIDS AND POLYPEPTIDES AND USES THEREOF&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is in the field of plant genetics and biochemistry. More specifically, the present invention relates to genes and polypeptides associated with the tocopherol biosynthesis pathway, namely those encoding homogentisate prenyl transferase activity, and uses thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>HOMOGENTISATE PRENYL TRANSFERASE ("HPT") NUCLEIC ACIDS AND POLYPEPTIDES, AND USES THEREOF<br>
This application claims priority to U.S. No. 60/365,202 filed March 19,2002, the disclosure of which is incorporated herein by reference in its entirety.<br>
The present invention is in the field of plant genetics and biochemistry. More specifically, the present invention relates to genes and polypeptides associated with the tocopherol biosynthesis pathway, namely those encoding homogentisate prenyl transferase activity, and uses thereof.<br>
Isoprenoids are ubiquitous compounds found in all living organisms. Plants synthesize a diverse array of greater than 22,000 isoprenoids (Connolly and Hill, Dictionary of Terpenoids, Chapman and Hall, New York, NY (1992)). In plants, isoprenoids play essential roles in particular cell functions such as production of sterols, contributing to eukaryotic membrane architecture, acyclic polyprenoids found in the side chain of ubiquinone and plastoquinone, growth regulators like abscisic acid, gibberellins, brassinosteroids or the photosynthetic pigments chlorophylls and carotenoids. Although the physiological role of other plant isoprenoids is less evident, like that of the vast array of secondary metabolites, some are known to play key roles mediating the adaptative responses to different environmental challenges. In spite of the remarkable diversity of structure and function, all isoprenoids originate from a single metabolic precursor, isopentenyl diphosphate (IPP) (Wright, (1961) Annu. Rev. Biachem., 20:525-548; and Spurgeon and Porter, In: Biosynthesis of Isoprenoid Compounds, Porter and Spurgeon (eds.) John Wiley, NY, Vol. 1, pp. 1-46 (1981)).<br>
A number of unique and interconnected biochemical pathways derived from the isoprenoid pathway leading to secondary metabolites, including tocopherols, exist in chloroplasts of higher plants. Tocopherols not only perform vital functions in plants, but are also important from mammalian nutritional perspectives. In plastids, tocopherols account for up to 40% of the total quinone pool. Tocopherols are an important component of mammalian diets. Epidemiological evidence indicates that tocopherol supplementation can result in decreased risk for cardiovascular disease and cancer, can aid in immune function, and is associated with prevention or retardation of a number of degenerative disease processes in humans (Traber and Sies, Annu. Rev. Nutr., 16:321-347 (1996)). Tocopherol functions, in part, by stabilizing the lipid bilayer of biological membranes (Skrypin and Kagan, Biochim. Biopliys. Acta, 815:209 (1995); Kagan, N.Y. Acad Sci., p. 121 (1989); Gomez-Fernandez et al., Ann. N.Y. Acad. Sci., p. 109 (1989)), reducing polyunsaturated fatty acid (PUFA) free<br>
radicals generated by lipid oxidation (Fukuzawa et al., Lipids, 17:511-513 (1982)), and scavenging oxygen free radicals, lipid peroxy radicals and singlet oxygen species (Diplock et al, Ann. N Y Acad. Sci., 570:72 (1989); Fryer, Plant Cell Environ., 15(4):381-392 (1992)).<br>
The compound α-tocopherol, which is often referred to as vitamin E, belongs to a class of lipid-soluble antioxidants that includes α, ß , and 5-tocopherols and α, ß, , and 8-tocotrienols. Although α,ß , and δ-tocopherols and α, ß,, and δ-tocotrienols are sometimes referred to collectively as "vitamin E", vitamin E is more appropriately defined chemically as α-tocopherol. Vitamin E, or cc-tocopherol, is significant for human health, in part because it is readily absorbed and retained by the body, and therefore has a higher degree of bioactivity than other tocopherol species (Traber and Sies, Annu. Rev. Nutr., 16:321-347 (1996)). Other tocopherols, however, such as ß, , and δ-tocopherols also have significant health and nutritional benefits.<br>
Tocopherols are primarily synthesized only by plants and certain other photosynthetic organisms, including cyanobacteria. As a result, mammalian dietary tocopherols are obtained almost exclusively from these sources. Plant tissues vary considerably in total tocopherol content and tocopherol composition, with a-tocopherol the predominant tocopherol species found in green, photosynthetic plant tissues. Leaf tissue can contain from 10-50 µg of total tocopherols per gram fresh weight, but most of the world's major staple crops (e.g., rice, corn, wheat, potato) produce low to extremely low levels of total tocopherols, of which only a small percentage is a-tocopherol (Hess, Vitamin E, a-tocopherol, In: Antioxidants in Higher Plants, R. Alscher and J. Hess, (eds.), CRC Press, Boca Raton., pp. 111-134 (1993)). Oil seed crops generally contain much higher levels of total tocopherols, but a-tocopherol is present only as a minor component in most oilseeds (Taylor and Barnes, Chemy Ind, Oct:722-726 (1981)).<br>
The recommended daily dietary intake of 15-30 mg of vitamin E is quite difficult to achieve from the average American diet For example, it would take over 750 grams of spinach leaves, in which a-tocopherol comprises 60% of total tocopherols, or 200-400 grams of soybean oil to satisfy this recommended daily vitamin E intake. While it is possible to augment the diet with supplements, most of these supplements contain primarily synthetic vitamin E, having eight stereoisomers, whereas natural vitamin E is predominantly composed of only a single isomer. Furthermore, supplements tend to be relatively expensive, and the general population is disinclined to take vitamin supplements on a regular basis. Therefore,<br>
there is a need in the art for compositions and methods that either increase the total tocopherol production or increase the relative percentage of α-tocopherol produced by plants.<br>
In addition to the health benefits of tocopherols, increased α-tocopherol levels in crops have been associated with enhanced stability and extended shelf life of plant products (Peterson, Cereal-Chem., 72(l):21-24 (1995); Ball, Fat-soluble vitamin assays in food analysis. A comprehensive review, London, Elsevier Science Publishers Ltd. (1988)). Further, tocopherol supplementation of swine, beef, and poultry feeds has been shown to significantly increase meat quality and extend the shelf life of post-processed meat products by retarding post-processing lipid oxidation, which contributes to the undesirable flavor components (Sante and Lacourt, J. Set. Food Agric., 65(4):503-507 (1994); Buckley et al, J. of Animal Science, 73:3122-3130 (1995)).<br>
TOCOPHEROL BIOSYNTHESIS<br>
The plastids of higher plants exhibit interconnected biochemical pathways leading to secondary metabolites including tocopherols. The tocopherol biosynthetic pathway in higher plants involves condensation of homogentisic acid and phytylpyrophosphate to form 2-methylphytylplastoquinol (Fiedler et al., Planta, 155:511-515 (1982); Soll et al., Arch. Biochem. Biophys., 204:544-550 (1980); Marshall et al. Phytochem., 24:1705-1711 (1985)). This plant tocopherol pathway can be divided into four parts: 1) synthesis of homogentisic acid (HGA), which contributes to the aromatic ring of tocopherol; 2) synthesis of phytylpyrophosphate, which contributes to the side chain of tocopherol; 3) joining of HGA and phytylpyrophosphate via a homogentisate prenyl transferase followed by a subsequent cyclization; and 4) S-adenosyl methionine dependent methylation of an aromatic ring, which affects the relative abundance of each of the tocopherol species. See Figure 1.<br>
Various genes and their encoded proteins that are involved in tocopherol biosynthesis include those listed in the table below:<br>
(Table Removed)<br>
The "Gene IDs" given in the table above identify the gene associated with the listed enzyme. Any of the Gene IDs listed in the table appearing herein in the present disclosure refer to the gene encoding the enzyme with which the Gene ID is associated in the table.<br>
As used herein, HPT, HPT2, PPT, slr1736, and ATPTC each refer to proteins or genes encoding proteins that have the same activity.<br>
SYNTHESIS OF HOMOGENTISIC ACID<br>
Homogentisic acid is the common precursor to both tocopherols and plastoquinones. In at least some bacteria the synthesis of homogentisic acid is reported to occur via the conversion of chorismate to prephenate and then to p-hydroxyphenylpyruvate via a bifunctional prephenate dehydrogenase. Examples of bifunctional bacterial prephenate dehydrogenase enzymes include the proteins encoded by the tyrA genes of Erwinia herbicola and Escherichia coll The tyrA gene product catalyzes the production of prephenate from chorismate, as well as the subsequent dehydrogenation of prephenate to form p-hydroxyphenylpymvate (p-HPP), the immediate precursor to homogentisic acid. p-HPP is then converted to homogentisic acid by hydroxyphenylpyruvate dioxygenase (HPPD). In contrast, plants are believed to lack prephenate dehydrogenase activity, and it is generally believed that the synthesis of homogentisic acid from chorismate occurs via the synthesis and conversion of the intermediate arogenate. Since pathways involved in homogentisic acid<br>
.ynthesis are also responsible for tyrosine formation, any alterations in these pathways can also result in the alteration in tyrosine synthesis and the synthesis of other aromatic amino acids.<br>
SYNTHESIS OF PHYTYLPYROPHOSPHATE<br>
Tocopherols are a member of the class of compounds referred to as the isoprenoids. Other isoprenoids include carotenoids, gibberellins, terpenes, chlorophyll and abscisic acid. A central intermediate in the production of isoprenoids is isopentenyl diphosphate (IPP). Cytoplasmic and plastid-based pathways to generate IPP have been reported. The cytoplasmic based pathway involves the enzymes acetoacetyl CoA thiolase, HMGCoA synthase, HMGCoA reductase, mevalonate kinase, phosphomevalonate kinase, and mevalonate pyrophosphate decarboxylase.<br>
Recently, evidence fox the existence of an alternative, plastid based, isoprenoid biosynthetic pathway emerged from studies in the research groups of Rohmer and Arigoni (Eisenreich et al., Chem. Bio., 5:R221-R233 (1998); Rohmer, Prog. Drug. Res., 50:135-154 (1998); Rohmer, Comprehensive Natural Products Chemistry, Vol. 2, pp. 45-68, Barton and Nakanishi (eds.), Pergamon Press, Oxford, England (1999)), who found that the isotope labeling patterns observed in studies on certain eubacterial and plant terpenoids could not be explained in terms of the mevalonate pathway. Arigoni and coworkers subsequently showed that 1-deoxyxylulose, or a derivative thereof, serves as an intermediate of the novel pathway, now referred to as the MEP pathway (Rohmer et al., Biochem, J., 295:517-524 (1993); Schwarz, PhD. thesis, Eidgenossiche Technische Hochschule, Zurich, Switzerland (1994)). Recent studies showed the formation of 1-deoxyxylulose 5-phosphate (Broers, Ph.D. thesis, EidgenSssiche Technische Hochschule, Zurich, Switzerland (1994)) from one molecule each of glyceraldehyde 3-phosphate (Rohmer, Comprehensive Natural Products Chemistry, Vol. 2, pp. 45-68, Barton and Nakanishi (eds.), Pergamon Press, Oxford, England (1999)) and pyruvate (Eisenreich et al., Chem. Biol, 5:R223-R233 (1998); Schwarz supra; Rohmer et al., J. Am. Chem. Soc., 118:2564-2566 (1996); and Sprenger et al., Proc. Natl. Acad. Sci. (U.SA.), 94:12857-12862 (1997)) by an enzyme encoded by the dxs gene (Lois et al, Proc. Natl. Acad Sci (U.SA.), 95:2105-2110 (1997); and Lange et al, Proc. Natl. Acad Sci (USA.), 95:2100-2104 (1998)). 1-Deoxyxylulose 5-phosphate can be further converted into 2-C-methylerythritol 4-phosphate (Arigoni et al., Proc. Natl. Acad ScL (U.SA), 94:10600-10605 (1997)) by a reductoisomerase encoded by the dxr gene (Bouvier et al, Plant<br>
Physiol, 117:1421-1431 (1998); and Rohdich et al., Proc. Natl. Acad. Sci. (U.S.A.), 96:11758-11763(1999)).<br>
Reported genes in the MEP pathway also include ygbP, which catalyzes the conversion of 2-C-methylerythritol 4-phosphate into its respective cytidyl pyrophosphate derivative and ygbB, which catalyzes the conversion of 4-phosphocytidyl-2-C-methyl-D-erythritol into 2-C-methyl-D-erythritol, 3,4-cyclophosphate. These genes are tightly linked on the E. coli genome (Herz et al., Proc. Natl Acad. Sci. (USA.), 97(6):2485-2490 (2000)).<br>
Once 1PP is formed by the MEP pathway, it is converted to GGDP by GGPDP synthase, and then to phytylpyrophosphate, which is the central constituent of the tocopherol side chain.<br>
COMBINATION AND CYCUZATION<br>
Homogentisic acid is combined with either phytyl-pyrophosphate or solanyl-pyrophosphate by homogentisate prenyl transferase (HPT) forming 2-methylphytyl plastoquinol or 2-memylsolanyl plastoquinol, respectively. 2-methylsolanyl plastoquinol is a precursor to the biosynthesis of plastoquinones, while 2-methylphytyl plastoquinol is ultimately converted to tocopherol.<br>
METHYLATION OF THE AROMATIC RING<br>
The major structural difference between each of the tocopherol subtypes is the position of the methyl groups around the phenyl ring. Both 2-methylphytyl plastoquinol and 2-methylsolanyl plastoquinol serve as substrates for the plant enzyme 2-methylphytylplastoquinol/2-methylsolanylplastoquinol memyltransferase (Tocopherol Methyl Transferase 2; Methyl Transferase 2; MT2; tMT2), which is capable of methylating a tocopherol precursor. Subsequent methylation at the 5 position of --tocopherol by -tocopherol methyl-transferase (GMT) generates the biologically active α-tocopherol.<br>
Some plants e.g. soy produce substantial amounts of delta and subsequently beta-tocopherol in their seed. The formation of δ-tocopherol or ß-tocopherol can be prevented by the overexpression of tMT2, resulting in the methylation of the δ-tocopherol precursor, 2-methyi phytyl plastoquinone to form 2,3-dimethyl-5-phytyl plastoquinone followed by cyclization with tocopherol cyclase to form -tocopherol and a subsequent methylation by GMT to form α-tocopherol. In a possible alternative pathway,ß-tocopherol is directly converted to α-tocopherol by tMT2 via the methylation of the 3 position (see, for example, Biochemical Society Transactions, 11:504-510 (1983); Introduction to Plant Biochemistry, 2nd<br>
edition, Chapter 11 (1983); Vitamin Hormone, 29:153-200 (1971); Biochemical Journal, 109:577 (1968); and, Biochemical and Biophysical Research Communication, 28(3):295 (1967)). Since all potential mechanisms for the generation of α-tocopherol involve catalysis by tMT2, plants that are deficient in this activity accumulate δ-tocopherol and ß-tocopherol. Plants which have increased tMT2 activity tend to accumulate -tocopherol and a-tocopherol. Since there is limited GMT activity in the seeds of many plants, these plants tend to accumulate y-tocopherol.<br>
There is a need in the art for nucleic acid molecules encoding enzymes involved in tocopherol biosysnthesis, as well as related enzymes and antibodies for the enhancement or alteration of tocopherol production in plants. There is a further need for transgenic organisms expressing those nucleic acid molecules involved in tocopherol biosynthesis, which are capable of nutritionally enhancing food and feed sources.<br>
SUMMARY OF THE INVENTION<br>
The present invention includes and provides a substantially purified nucleic acid molecule encoding an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11, 57-58, and 90.<br>
The present invention includes and provides a substantially purified polypeptide molecule comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,57-58, and 90.<br>
The present invention, includes and provides an antibody capable of specifically binding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,57-58, and 90.<br>
The present invention includes and provides a substantially purified nucleic acid molecule encoding a polypeptide having homogentisate prenyl transferase activity comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 43 and 44.<br>
The present invention includes and provides a substantially purified polypeptide ' having homogentisate prenyl transferase activity comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 43 and 44.<br>
The present invention includes and provides a transformed plant comprising an introduced nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof.<br>
The present invention includes and provides a transformed plant comprising an introduced first nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, and an introduced second nucleic acid molecule encoding an enzyme selected from the group consisting of tyrA, prephenate dehyrogenase, tocopherol cyclase, dxs, dxr, GMT, MT1, tMT2, GCPE, GGPPS, HPPD, AANT1, IDI, GGH, and complements thereof.<br>
The present invention includes and provides a transformed plant comprising a nucleic acid molecule comprising an introduced promoter region which functions in plant cells to cause the production of an mRNA molecule, wherein said introduced promoter region is linked to a transcribed nucleic acid molecule having a transcribed strand and a non-transcribed strand, wherein said transcribed strand is complementary to a nucleic acid molecule encoding a polypeptide selected from the group consisting of SEQ ID NOs: 5, 9-11,43-44,57-58, and 90, and wherein said transcribed nucleic acid molecule is linked to a 3' non-translated sequence that functions in the plant cells to cause termination of transcription and addition of polyadenylated ribonucleotides to a 3' end of the mRNA sequence.<br>
The present invention includes and provides a method of producing a plant having a seed with an increased total tocopherol level comprising: (A) transforming said plant with an introduced nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,57-58, and 90; and (B) growing said transformed plant<br>
The present invention includes and provides a method of producing a plant having a seed with an increased total tocopherol level comprising: (A) transforming said plant with an introduced first nucleic acid molecule, wherein said first nucleic acid molecule encodes a polypeptide having an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and an introduced second nucleic acid molecule encoding an enzyme selected from the group consisting of tyrA, prephenate dehydrogenase, tocopherol cyclase, dxs, dxr, GMT, MT1, tMT2, GGPPS, GCPE, HPPD, AANT1, IDI, GGH, and complements thereof; and (B) growing said transformed plant<br>
The present invention includes and provides a seed derived from a transformed plant comprising an introduced nucleic acid molecule encoding a polypeptide comprising an amino<br>
acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44, 57-58, and 90.<br>
The present invention includes and provides a seed derived from a transformed plant comprising an introduced first nucleic acid molecule encoding an introduced polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5, 9-11,43,44,57,58, and 90 and an introduced second nucleic acid encoding an enzyme selected from the group consisting of tyrA, prephenate dehydrogenase, tocopherol cyclase, dxs, dxr, GMT, MT1, GCPE, tMT2, GGPPS, HPPD, AANT1, IDI, GGH, and complements thereof.<br>
The present invention includes and provides a substantially purified polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 wherein said amino acid sequence is not derived from a nucleic acid molecule that is derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato. The present invention includes and provides said substantially purified polypeptide wherein more than one more amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
The present invention includes and provides a substantially purified nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 wherein said nucleic acid molecule is not derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato. The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more man one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
The present invention includes and provides a substantially purified nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 wherein said nucleic acid molecule is not derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Sulfolobus, Aeropyum, Trichodesmium erythraeum, Chloroflexus aurantiacus, sorghum, wheat, tomato, or leek. The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than<br>
one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49 and 92-95.<br>
The present invention includes and provides a plant transformed with a nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 wherein said nucleic acid molecule is not derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Sulfolobus, Aeropyum, Trichodesmium erythraeum, Chloroflexus aurantiacus, sorghum, wheat, tomato, or leek. The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
The present invention includes and provides a substantially purified polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 wherein said polypeptide does not comprise any of the amino acid sequences set form in sequence listings in WO 00/68393 (which sequences are incorporated herein by reference); WO 00/63391 (which sequences are incorporated herein by reference); WO 01/62781 (which sequences are incorporated herein by reference); or WO 02/33060 (which sequences are incorporated herein by reference); and does not comprise SEQ ID NOs: 1-11,43-45,57-58,61-62, or 90 from the present application.<br>
The present invention includes and provides a substantially purified polypeptide comprising more than onean amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
The present invention includes and provides a substantially purified nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49,,and 92-95 wherein said nucleic acid molecule does not comprise any of the nucleic acid sequences set form in sequence listings in WO 00/68393; WO 00/63391; WO 01/62781; or WO 02/33060; and does not comprise SEQ ID NOs: 27-36, 59-60,88-89, and 91 from the present application, or the gene with Genebank Accession Nos. AI897027 or AW 563431 The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
The present invention includes and provides a plant transformed with a nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 wherein said nucleic acid molecule does not comprise any of the nucleic acid sequences set forth in sequence listings in WO 00/68393; WO 00/63391; WO 01/62781; or WO 02/33060; and does not comprise SEQ ID NOs: 27-36; 59-60,88-89, and 91 from the present application, or the gene with Genebank Accession Nos. AI897027 or AW 563431.   The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
The present invention includes and provides a substantially purified nucleic acid molecule comprising a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 31,34-36,59-60, and 91.<br>
The present invention includes and provides for homogentisate prenyl transferases discovered using one or more of the alignments of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-35b and 36.<br>
DESCRIPTION OF THE NUCLEIC AND AMINO ACID SEQUENCES<br>
SEQ ID NO: 1 sets forth a.Nostoc punctiforme homogentisate prenyl transferase polypeptide.<br>
SEQ ID NO: 2 sets forth an Anabaena homogentisate prenyl transferase polypeptide.<br>
SEQ ID NO: 3 sets forth a Synechocystis homogentisate prenyl transferase polypeptide.<br>
SEQ ID NO: 4 sets forth a Zea mays homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 5 sets forth a Glycine max homogentisate prenyl transferase polypeptide (HPT1-2).<br>
SEQ ID NO: 6 sets forth a Glycine max homogentisate prenyl transferase polypeptide (HPT1-1).<br>
SEQ ID NO: 7 sets forth an Arabidopsis thaliana homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 8 sets forth a partial Cuphea pulcherrima homogentisate prenyl transferase polypeptide.<br>
SEQ ID NO: 9 sets forth a leek homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 10 sets forth a wheat homogentisate prenyl transferase polypeptide<br>
(HPT1).<br>
SEQ ED NO: 11 sets forth a Cuphea pulcherrima homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NOs: 12-15 represent domains from SEQ ID NOs: 1-8.<br>
SEQ ID NOs: 16-26 set forth primer sequences.<br>
SEQ ID NO: 27 sets forth a nucleic acid molecule encoding a Nostoc punctiforme homogentisate prenyl transferase polypeptide.<br>
SEQ ID NO: 28 sets forth a nucleic acid molecule encoding an Anabaena homogentisate prenyl transferase polypeptide.<br>
SEQ ID NO: 29 sets forth a nucleic acid molecule encoding a Synechocystis homogentisate prenyl transferase polypeptide,<br>
SEQ ID NO: 30 sets forth a nucleic acid molecule encoding a Zea mays homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 31 sets forth a nucleic acid molecule encoding a Glycine max homogentisate prenyl transferase polypeptide (HPT1-2).<br>
SEQ ID NO: 32 sets forth a nucleic acid molecule encoding a Glycine max homogentisate prenyl transferase polypeptide (HPT1-1).<br>
SEQ ID NO: 33 sets forth a nucleic acid molecule encoding an Arabidopsis thaliana homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 34 sets forth a nucleic acid molecule encoding a Cuphea pulcherrima homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 35 sets forth a nucleic acid molecule encoding a leek homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NO: 36 sets forth a nucleic acid molecule encoding a wheat homogentisate prenyl transferase polypeptide (HPT1).<br>
SEQ ID NOs: 37-38 set forth primer sequences.<br>
SEQ ID NOs: 39-42 set forth domains from SEQ ID NOs: 1-7 and 9-11.<br>
SEQ ID NO: 43 sets forth a homogentisate prenyl transferase polypeptide from Trichodesmium erythraeum.<br>
SEQ ID NO: 44 sets forth a homogentisate prenyl transferase polypeptide from Chloroflexus aurantiacus.<br>
SEQ ID NO: 45 sets forth a putative sequence for an Arabidopsis thaliana homogentisate prenyl transferase polypeptide (HPT2).<br>
SEQ ID NOs: 46-49 represent domains from SEQ ID NOs: 1-4,6-7,9-11,57-58 and 91.<br>
SEQ ID NOs: 50-56 set forth primer sequences.<br>
SEQ ID NO: 57 sets forth an Arabidopsis thaliana homogentisate prenyl transferase polypeptide (HPT2).<br>
SEQ ID NO: 58 sets forth an Oryza sativa homogentisate prenyl transferase polypeptide (HPT2).<br>
SEQ ID NO: 59 sets forth a nucleic acid molecule encoding an Arabidopsis ihaliana homogentisate prenyl transferase polypeptide (HPT2).<br>
SEQ ID NO: 60 sets forth a nucleic acid molecule encoding an Oryza sativa homogentisate prenyl transferase polypeptide (HPT2).<br>
SEQ ID NO: 61 sets forth a putative homogentisate prenyl transferase polypeptide from Arabidopsis tlialiana (HPT2).<br>
SEQ ID NO: 62 sets forth a putative homogentisate prenyl transferase polypeptide from Arabidopsis tlialiana (HPT2).<br>
SEQ ID NO: 63 sets forth an EST from Arabidopsis thaliana.<br>
SEQ ID NO: 64 sets forth an EST from Medicago truncatula.<br>
SEQ ID NO: 65 sets forth an EST from Medicago truncatula developing stem.<br>
SEQ ID NO: 66 sets forth an EST from Medicago truncatula developing stem.<br>
SEQ ID NO: 67 sets forth an EST from Medicago truncatula developing stem.<br>
SEQ ID NO: 68 sets forth an EST from mixed potato tissues.<br>
SEQ ID NO: 69 sets forth an EST from Arabidopsis tlwliana, Columbia ecotype flower buds.<br>
SEQ ID NO: 70 sets forth an EST from Arabidopsis thaliana.<br>
SEQ ID NO: 71 sets forth an EST from Medicago truncatula.<br>
SEQ ID NO: 72 sets forth an EST from Glycine max.<br>
SEQ ID NOs: 73-83 and 84-87 set forth primer sequences.<br>
SEQ ID NO: 88 sets forth a nucleic acid molecule encoding a homogentisate prenyl transferase polypeptide from cyanobacteria Trichodesmium erythraeum.<br>
SEQ ID NO: 89 sets forth a nucleic acid molecule encoding a homogentisate prenyl transferase polypeptide from photobacteria Chloroflexus aurantiacus.<br>
SEQ ID NO: 90 sets forth a Glycine max homogentisate prenyl transferase polypeptide (HPT2).<br>
SEQ ID NO: 91 sets forth a nucleic acid molecule encoding a homogentisate prenyl transferase polypeptide from Glycine max (HPT2).<br>
SEQ ID NOs: 92-95 represent domains from SEQ ID NOs: 1-4,6-7,9-11,43-44, 57-58, and 90.<br>
Note: cyanobacteria and photobbacteria have one HPT. Plants have both HPT1 and HPT2. In soy, there are two variations of HPT1, HPT1-1 and HPT1-2, as well as HPT2.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 is a schematic diagram of the tocopherol biosynthetic pathway.<br>
Figures 2a-2c depicts a sequence alignment for several homogentisate prenyl transferase polypeptides SEQ ID NOs: 1-8).<br>
Figures 3a-3c depicts a sequence alignment for several homogentisate prenyl transferase polypeptides (SEQ ID NOs: 1-7, and 9-11).<br>
Figure 4 provides a schematic of the expression construct pCGN10800.<br>
Figure 5 provides a schematic of the expression construct pCGN10801.<br>
Figure 6 provides a schematic of the expression construct pCGN10803.<br>
Figure 7 provides a schematic of the expression construct pCGN10822.<br>
Figure 8 provides bar graphs of HPLC data obtained from seed extracts of transgenic Arabidopsis containing pCGN10822, which provides of the expression of the ATPT2 sequence (SEQ ID NO: 33), in the sense orientation, from the napin promoter. Provided are graphs for α, , and δ-tocopherols, as well as total tocopherol for 22 transformed lines, as well as a nontransformed (wild-type) control.<br>
Figure 9 provides a bar graph of HPLC analysis of seed extracts from Arabidopsis plants transformed with a pCGN10803 (lines 1387 through 1624, enhanced 35S-ATPT2, in the antisense orientation), a nontransformed (wt) control, and an empty vector transformed control.<br>
Figure 10 provides a schematic of the expression construct pMON36581.<br>
Figure 11 provides a schematic of the expression construct pMON69933.<br>
Figure 12 provides a schematic of the expression construct pMON69924.<br>
Figure 13 provides a schematic of the expression construct pMON69943.<br>
Figure 14 provides a bar graph of total tocopherol levels in recombinant soy lines.<br>
Figure 15 depicts pMON 69960.<br>
Figure 16 depicts pMON 36525.<br>
Figure 17 depicts pMON 69963.<br>
Figure 18 depicts pMON 69965.<br>
Figure 19 depicts pMON 10098.<br>
Figure 20 depicts pMON 69964.<br>
Figure 21 depicts pMON 69966.<br>
Figure 22 depicts results of seed total tocopherol analysis.<br>
Figure 23 depicts results of seed total tocopherol analysis.<br>
Figure 24 depicts the alignments of SEQ ID NOs: 1-4,6-7,9-11,57, and 90.<br>
Figure 25 depicts motifs V through VIII, SEQ ID NOs: 46-49.<br>
Figure 26 depicts a sequence tree derived from a multiple alignment shown from SEQ ID NOs: 1-7,9-11,43,44,57-58, and 90.<br>
Figure 27 depicts pMON81028.<br>
Figure 28 depicts pMON81023.<br>
Figure 29 depicts pMON36596.<br>
Figure 30 depicts pET30a(+) vector.<br>
Figure 31 depicts pMON69993.<br>
Figure 32 depicts pMON69992.<br>
Figures 33a-33c depicts a sequence alignment for several homogentisate prenyl transferase polypeptide SEQ ID NOs: 1-4,6-7,9-11,43-44,57-58, and 90.<br>
Figure 34 depicts motifs IX through XII, SEQ ID NOs: 92-95.<br>
Figure 35 depicts motifs I-IV, SEQ ID NOs: 39-42.<br>
Figure 36 depicts motifs A-D.<br>
DETAILED DESCRIPTION<br>
The present invention provides a number of agents, for example, nucleic acid molecules and polypeptides associated with the synthesis of tocopherol, and provides uses of such agents.<br>
AGENTS<br>
The agents of the present invention will preferably be "biologically active" with respect to either a structural attribute, such as the capacity of a nucleic acid to hybridize to another nucleic acid molecule, or the ability of a protein to be bound by an antibody (or to compete with another molecule for such binding). Alternatively, such an attribute may be catalytic and thus involve the capacity of the agent to mediate a chemical reaction or response. The agents will preferably be "substantially purified". The term "substantially purified", as used herein, refers to a molecule separated from substantially all other molecules normally associated with it in its native environmental conditions. More preferably a substantially purified molecule is the predominant species present in a preparation. A substantially purified molecule may be greater than about 60% free, preferably about 75% free, more preferably about 90% free, and most preferably about 95% free from the other molecules (exclusive of solvent) present in the natural mixture. The term "substantially purified' is not intended to encompass molecules present in their native environmental conditions.<br>
The agents of the present invention may also be recombinant As used herein, the term recombinant means any agent (e.g., PNA, peptide etc.), that is, or results, however indirectly, from human manipulation of a nucleic acid molecule.<br>
It is understood that the agents of the present invention may be labeled with reagents that facilitate detection of the agent (e.g., fluorescent labels, Prober et al., Science, 238:336-340 (1987); Albarella et al., EP144 914; chemical labels, Sheldon et al., U.S. Patent 4,582,789; Albarella et al., U.S. Patent 4,563,417; modified bases, Miyoshi et al., EP 119448).<br>
NUCLEIC ACBD MOLECULES<br>
Agents of the present invention include nucleic acid molecules. In a preferred aspect of the present invention the nucleic acid molecule comprises a nucleic acid sequence that encodes a homogentisate prenyl transferase. As used herein, a homogentisate prenyl transferase is any plant protein that is capable of specifically catalyzing the formation of 2-methyl-6-phytylbenzoquinol (2-methyl-6-gearanylgeranylbenzquinol) from phytyl-DP (GGDP) and homogentisate.<br>
An example of a more preferred homogentisate prenyl transferase is a polypeptide with the amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,<br>
43-44,55,58, and 90. In a more preferred embodiment, the homogentisate prenyl transferase is encoded by any nucleic acid molecule encoding an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,55,58, and 90.<br>
In another preferred aspect of the present invention the nucleic acid molecule of the present invention comprises a nucleic acid sequence encoding a polypeptide selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,55,58, and 90, and complements thereof and fragments of either.<br>
In another preferred aspect of the present invention the nucleic acid molecule of the present invention comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 31,34-36,59-60, and 91.<br>
In another embodiment, the present invention includes nucleic acid molecules encoding polypeptides having a region of conserved amino acid sequence shown in any of Figures Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-b and 36, and complements of those nucleic acid molecules. In a preferred embodiment, the present invention includes nucleic acid molecules encoding polypeptides comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95, and complements of those nucleic acid molecules. The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes nucleic acid molecules encoding polypeptides comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95, and complements of those nucleic acid molecules. In another embodiment, the present invention includes nucleic acid molecules encoding polypeptides having homogentisate prenyl transferase activity and a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-35b and 36, and complements of those nucleic acid molecules, hi a preferred embodiment, the present invention includes nucleic acid molecules encoding polypeptides having homogentisate prenyl transferase activity and comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 and complements of those nucleic acid molecules.   The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more man one<br>
amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes nucleic acid molecules encoding polypeptides having homogentisate prenyl transferase activity and comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95, and complements of those nucleic acid molecules. In another embodiment, the present invention includes nucleic acid molecules, excluding nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Tea mays, Glycine max, Arabidopsis tltaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato, encoding polypeptides having a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-35b and 36, and complements of those nucleic acid molecules. In a preferred embodiment, the present invention includes nucleic acid molecules, excluding nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Tea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chioroflexus aurantiacus, wheat, leek, canola, cotton, or tomato, encoding polypeptides comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.and complements of those nucleic acid molecules. The present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes nucleic acid molecules, excluding nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chioroflexus aurantiacus, wheat, leek, canola, cotton, or tomato, encoding polypeptides comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In another embodiment, the present invention includes nucleic acid molecules, excluding nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Tea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chioroflexus aurantiacus, wheat, leek, canola, cotton, Sulfolobus, Aeropyum, sorghum, or tomato, encoding polypeptides having homogentisate prenyl transferase activity and a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b,<br>
33a-33c, 34a-34b, 35a-35b and 36 and complements of those nucleic acid molecules. In a preferred embodiment, the present invention includes nucleic acid molecules, excluding nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thalian, Oryza sativa, Trichodesmiun erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato, encoding polypeptides having homogentisate prenyl transferase activity and comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95. Hie present invention includes and provides said nucleic acid molecule wherein the polypeptide further comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes nucleic acid molecules, excluding nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato, encoding polypeptides having homogentisate prenyl transferase activity and comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95.<br>
In one embodiment of a method of the present invention, any of the nucleic acid sequences or polypeptide sequences, or fragments of either, of the present invention can be used to search for related sequences. In a preferred embodiment, a member selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,57-58, and 90 is used to search for related sequences. In a preferred embodiment, a member selected from the group consisting of SEQ ID NOs: 31,34-36,59-60,88-89, and 91 is used to search for related sequences. In another embodiment, any of the motifs or regions of conserved sequence shown in Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-35b and 36 are used to search for related amino acid sequences, In a preferred embodiment, a member selected from the group consisting of SEQ ID NOs: 39-42 and 46-49 is used to search for related sequences. In one embodiment, one or more of SEQ ID NOs: 39-42,46-49, and 92-95 is used to search for related sequences. As used herein, "search for related sequences" means any method of determining relatedness between two sequences, including, but not limited to, searches that compare sequence homology: for example, a PBLAST search of a database for relatedness to a single amino acid sequence. Other searches may be conducted using profile based methods,<br>
such as the HMM (Hidden Markov model) META-MEME (http://metameme.sdsc.edu/mhmn-Unks.html), PSI-BLAST<br>
(http://www.ncbi.nlm.nih.gov/BLAST/). The present invention includes and provides for homogentisate prenyl transferases discovered using one or more of the alignments of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-35b and 36.<br>
As used herein, a nucleic acid molecule is said to be "derived from" a particular organism, species, ecotype, etc., when the sequence of the nucleic acid molecule originated from that organism, species, ecotype, etc. "Derived from" therefore includes copies of nucleic acid molecules derived through, for example, PCR, as well as synthetically generated nucleic acid molecules having the same nucleic acid sequence as the original organism, species, ecotype, etc. Likewise, a polypeptide is said to be "derived from" a nucleic acid molecule when that nucleic acid molecule is used to code for the polypeptide, whether the polypeptide is enzymatically generated from the nucleic acid molecule or synthesized based on the sequence information inherent in the nucleic acid molecule.<br>
The present invention includes the use of the above-described conserved sequences and fragments thereof in transgenic plants, other organisms, and for other uses, including, without limitation, as described below.<br>
In another preferred aspect of the present invention a nucleic acid molecule comprises nucleotide sequences encoding a plastid transit peptide operably fused to a nucleic acid molecule that encodes a protein or fragment of the present invention.<br>
In another preferred embodiment of the present invention, the nucleic acid molecules of the present invention encode mutant tocopherol homogentisate prenyl transferase enzymes. As used herein, a mutant enzyme is any enzyme that contains an amino acid that is different from the amino acid in the same position of a wild type enzyme of the same type.<br>
It is understood that in a further aspect of nucleic acid sequences of the present invention, the nucleic acids can encode a protein that differs from any of the proteins in that one or more amino acids have been deleted, substituted or added without altering the function. For example, it is understood that codons capable of coding for such conservative amino acid substitutions are known in the art<br>
In one aspect of the present invention the nucleic acids of the present invention are said to be introduced nucleic acid molecules. A nucleic acid molecule is said to be "introduced".if it is inserted into a cell or organism as a result of human manipulation, no<br>
matter how indirect. Examples of introduced nucleic acid molecules include, without limitation, nucleic acids that have been introduced into cells via transformation, transfection, injection, and projection, and those that have been introduced into an organism via conjugation, endocytosis, phagocytosis, etc.<br>
One subset of the nucleic acid molecules of the present invention is fragment nucleic acids molecules. Fragment nucleic acid molecules may consist of significant portion(s) of, or indeed most of, the nucleic acid molecules of the present invention, such as those specifically disclosed. Alternatively, the fragments may comprise smaller oligonucleotides (having from about 15 to about 400 nucleotide residues and more preferably, about 15 to about 30 nucleotide residues, or about 50 to about 100 nucleotide residues, or about 100 to about 200 nucleotide residues, or about 200 to about 400 nucleotide residues, or about 275 to about 350 nucleotide residues).<br>
A fragment of one or more of the nucleic acid molecules of the present invention may be a probe and specifically a PCR probe. A PCR probe is a nucleic acid molecule capable of initiating a polymerase Activity while in a double-stranded structure with another nucleic acid. Various methods for determining the structure of PCR probes and PCR techniques exist in the art. Computer generated searches using programs such as Primer3 (www-genome.wi.mitedu/cgi-bin/primer/primer3.cgi), STSPipeline (www-genome.wi.mit.edu/cgi-bm/www-STSJPipeiine), or GeneUp (Pesole et al, BioTechniques, 25:112-123 (1998)), for example, can be used to identify potential PCR primers.<br>
Nucleic acid molecules or fragments thereof of the present invention are capable of specifically hybridizing to other nucleic acid molecules under certain circumstances. Nucleic acid molecules of the present invention include those that specifically hybridize to those nucleic acid molecules disclosed herein, such as those encoding any of SEQ ID NOs: 5,9-11, 43-44,57-58, and 90, and complements thereof. Nucleic acid molecules of the present invention include those that specifically hybridize to a nucleic acid molecules comprising a member selected from the group consisting of SEQ ID NOs: 31,34-36,59-60, and 91, and complements thereof.<br>
As used herein, two nucleic acid molecules are said to be capable of specifically hybridizing to one another if the two molecules are capable of forming an anti-parallel, double-stranded nucleic acid structure.<br><br>
A nucleic acid molecule is said to be the "complement" of another nucleic acid molecule if they exhibit complete complementarity. As used herein, molecules are said to exhibit "complete complementarity" when every nucleotide of one of the molecules is complementary to a nucleotide of the other. Two molecules are said to be "minimally complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under at least conventional "low-stringency" conditions. Similarly, the molecules are said to be "complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under conventional "high-stringency" conditions. Conventional stringency conditions are described by Sambrook et al., Molecular Cloning, A Laboratory Manual, 2nd Ed, Cold Spring Harbor Press, Cold Spring Harbor, NY (1989), and by Haymes et al., Nucleic Acid Hybridization, A Practical Approach IRL Press, Washington, DC (1985). Departures from complete complementarity are therefore permissible, as long as such departures do not completely preclude the capacity of the molecules to form a double-stranded structure. Thus, in order for a nucleic acid molecule to serve as a primer or probe it need only be sufficiently complementary in sequence to be able to form a stable double-stranded structure under the particular solvent and salt concentrations employed.<br>
Appropriate stringency conditions which promote DNA hybridization are, for example, 6.0 X sodium chloride/sodium citrate (SSC) at about 45°C, followed by a wash of 2.0 X SSC at 20-25°C, are known to those skilled in the art or can be found in Current Protocols in Molecular Biology, John Wiley &amp; Sons, NY (1989), 6.3.1-6.3.6. For example, the salt concentration in the wash step can be selected from a low stringency of about 2.0 X SSC at 50°C to a high stringency of about 0.2 X SSC at 65°C. In addition, the temperature in the wash step can be increased from low stringency conditions at room temperature, about 22°C, to high stringency conditions at about 65°C. Both temperature and salt may be varied, or either the temperature or the salt concentration may be held constant while the other variable is changed.<br>
In a preferred embodiment, a nucleic acid of the present invention will specifically hybridize to one or more of the nucleic acid molecules described herein and complements thereof, such as those encoding any of SEQ ID NOs: 5,9-11,43-44,57-58, and 90, under moderately stringent conditions, for example at about 2.0 X SSC and about 65°C.<br>
In a particularly preferred embodiment, a nucleic acid of the present invention will include those nucleic acid molecules that specifically hybridize to one or more nucleic acid molecules encoding any of SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, under high stringency conditions such as 0.2 X SSC and about 65°C.<br>
In one aspect of the present invention, the nucleic acid molecules of the present invention have one or more nucleic acid sequences encoding SEQ ID NOs: 5,9-11,43-44, 57-58, and 90, or complements thereof. In another aspect of the present invention, one or more of the nucleic acid molecules of the present invention share between about 100% and about 90% sequence identity with one or more of the nucleic acid sequences encoding SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, and fragments of either. In a further aspect of the present invention, one or more of the nucleic acid molecules of the present invention share between about 100% and about 95% sequence identity with one or more of the nucleic acid sequences encoding SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, and fragments of either. In a more preferred aspect of the present invention, one or more of the nucleic acid molecules of the present invention share between about 100% and about 98% sequence identity with one or more of the nucleic acid sequences encoding SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, and fragments of either. In an even more preferred aspect of the present invention, one or more of the nucleic acid molecules of the present invention share between about 100% and about 99% sequence identity with one or more of the sequences encoding SEQ ID NOs: 5,9-11,43-44, 57-58, and 90, and complements thereof, and fragments of either.<br>
In a preferred embodiment the percent identity calculations are performed using BLASTSN or BLASTP (default, parameters, version 2.0.8, Altschul et al, Nucleic Acids Res., 25:3389-3402(1997)).<br>
A nucleic acid molecule of the present invention can also encode a homolog polypeptide. As used herein, a homolog polypeptide molecule or fragment thereof is a counterpart protein molecule or fragment thereof in a second species (e.g., corn rubisco small subunit is a homolog of Arabidopsis rubisco small subunit). A homolog can also be generated by molecular evolution or DNA shuffling techniques, so that the molecule retains at least one functional or structure characteristic of the original polypeptide (see, for example, U.S. Patent 5,811,238).<br><br>
In another embodiment, the homolog is selected from the group consisting of alfalfa, Arabidopsis, barley, Brassica campestris, Brassica napus, oilseed rape, broccoli, cabbage, canola, citrus, cotton, garlic, oat, Allium, flax, an ornamental plant, peanut, pepper, potato, rapeseed, rice, rye, sorghum, strawberry, sugarcane, sugarbeet, tomato, wheat, poplar, pine, fir, eucalyptus, apple, lettuce, lentils, grape, banana, tea, turf grasses, sunflower, soybean, corn, Phaseolus, crambe, mustard, castor bean, sesame, cottonseed, linseed, safflower, and oil palm. More particularly, preferred homologs are selected from canola, corn, Brassica campestris, Brassica napus, oilseed rape, soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, rapeseed, safflower, oil palm, flax, and sunflower. In an even more preferred embodiment, the homolog is selected from the group consisting of canola, rapeseed, corn, Brassica campestris, Brassica napus, oilseed rape, soybean, sunflower, safflower,oil palms, and peanut. In a particularly preferred embodiment, the homolog is soybean. In a particularly preferred embodiment, the homolog is canola. In a particularly preferred embodiment, the homolog is oilseed rape.<br>
In a preferred embodiment, nucleic acid molecules encoding SEQ ID NOs: 5,9-11, 43-44,57-58, and 90, and complements thereof, and fragments of either; or more preferably encoding SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, can be utilized to obtain such homologs.<br>
In another further aspect of the present invention, nucleic acid molecules of the present invention can comprise sequences that differ from those encoding a polypeptide or fragment thereof due to the fact that a polypeptide can have one or more conservative amino acid changes, and nucleic acid sequences coding for the polypeptide can therefore have sequence differences. It is understood that codons capable of coding for such conservative amino acid substitutions are known in the art<br>
It is well known in the art that one or more amino acids in a native sequence can be substituted with other amino acid(s), the charge and polarity of which are similar to that of the native amino acid, i.e„ a conservative amino acid substitution. Conservative substitutes for an amino acid within the native polypeptide sequence can be selected from other members of the class to which the amino acid belongs. Amino acids can be divided into the following four groups: (1) acidic amino acids; (2) basic amino acids; (3) neutral polar amino acids; and (4) neutral nonpolar amino acids. Representative amino acids within these various groups include, but are not limited to, (1) acidic (negatively charged) amino acids such as aspartic<br><br>
acid and glutamic acid; (2) basic (positively charged) amino acids such as arginine, histidine, and lysine; (3) neutral polar amino acids such as glycine, serine, threonine, cysteine, cystine, tyrosine, asparagine, and glutamine; and (4) neutral nonpolar (hydrophobic) amino acids such as alanine, leucine, isoleucine, valine, proline, phenylalanine, tryptophan, and methionine.<br>
Conservative amino acid substitution within the native polypeptide sequence can be made by replacing one amino acid from within one of these groups with another amino acid from within the same group. In a preferred aspect, biologically functional equivalents of the proteins or fragments thereof of the present invention can have ten or fewer conservative amino acid changes, more preferably seven or fewer conservative amino acid changes, and most preferably five or fewer conservative amino acid changes. The encoding nucleotide sequence will thus have corresponding base substitutions, permitting it to encode biologically functional equivalent forms of the polypeptides of the present invention.<br>
It is understood that certain amino acids may be substituted for other amino acids in a protein structure without appreciable loss of interactive binding capacity with structures such as, for example, antigen-binding regions of antibodies or binding sites on substrate molecules. Because it is the interactive capacity and nature of a protein that defines that protein's biological functional activity, certain amino acid sequence substitutions can be made in a protein sequence and, of course, its underlying DNA coding sequence and, nevertheless, a protein with like properties can still be obtained. It is thus contemplated by the inventors that various changes may be made in the peptide sequences of the proteins or fragments of the present invention, or corresponding DNA sequences that encode said peptides, without appreciable loss of their biological utility or activity. It is understood that codons capable of coding for such amino acid changes are known in the art<br>
In making such changes, the hydropathic index of amino acids may be considered. The importance of the hydropathic,mino acid index in conferring interactive biological function on a protein is generally understood in the art (Kyte and Doolittle, J. Mot Biol., 157:105-132 (1982)). It is accepted that the relative hydropathic character of the amino acid contributes to the secondary structure of the resultant polypeptide, which in turn defines the interaction of the protein with other molecules, for example, enzymes, substrates, receptors, DNA, antibodies, antigens, and the like.<br>
Each amino acid has been assigned a hydropathic index on the basis of its bydrophobicity and charge characteristics (Kyte and Doolittle, J. Mol. Biol, 157:105-132<br>
(1982)); these are isoleucine (+4.5), valine (+4.2), leucine (+3.8), phenylalanine (+2.8), cysteine/cystine (+2.5), methionine (+1.9), alanine (+1.8), glycine (-0.4), threonine (-0.7), serine (-0.8), tryptophan (-0.9), tyrosine (-1.3), proline (-1.6), histidine (-3.2), glutamate (-3.5), glutamine (-3.5), aspartate (-3.5), asparagine (-3.5), lysine (-3.9), and arginine (-4.5).<br>
In making such changes, the substitution of amino acids whose hydropathic indices are within ±2 is preferred, those that are within ±1 are particularly preferred, and those within ±0.5 are even more particularly preferred.<br>
It is also understood in the art that the substitution of like amino acids can be made effectively on the basis of hydrophilicity. U.S. Patent 4,554,101 states that the greatest local average hydrophilicity of a protein, as governed by the hydrophilicity of its adjacent amino acids, correlates with a biological property of the protein.<br>
As detailed in U.S. Patent 4,554,101, the following hydrophilicity values have been assigned to amino acid residues: arginine (+3.0), lysine (+3.0), aspartate (+3.0±1), glutamate (+3.0±1), serine (+0.3), asparagine (+0.2), glutamine (+0.2), glycine (0), threonine (-0.4), proline (-0.5±1), alanine (-0.5), histidine (-0.5), cysteine (-1.0), methionine (-1.3), valine (-1.5), leucine (-1.8), isoleucine (-1.8), tyrosine (-2.3), phenylalanine (-2.5), and tryptophan (-3.4).<br>
In making such changes, the substitution of amino acids whose hydrophilicity values are within ±2 is preferred, those that are within ±1 are particularly preferred, and those within ±0.5 are even more particularly preferred.<br>
In a further aspect of the present invention, one or more of the nucleic acid molecules of the present invention differ in nucleic acid sequence from those for which a specific sequence is provided herein because one or more codons has been replaced with a codon that encodes a conservative substitution of the amino acid originally encoded.<br>
Agents of the present invention include nucleic acid molecules that encode at least about a contiguous 10 amino acid region of a polypeptide of the present invention, more preferably at least about a contiguous 25,40,50,100, or 125 amino acid region of a polypeptide of the present invention.<br>
In a preferred embodiment, any of the nucleic acid molecules of the present invention can be operably linked to a promoter region that functions in a plant cell to cause the production of an mRNA molecule, where the nucleic acid molecule that is linked to the<br>
promoter is heterologous with respect to that promoter. As used herein, "heterologous" means not naturally occurring together.<br>
The nature of the coding sequences of non-plant genes can distinguish them from plant genes as well as many other heterologous genes expressed in plants. For example, the average A+T content of bacteria can be higher than that for plants. The A+T content of the genomes (and thus the genes) of any organism are features of that organism and reflect its evolutionary history. While within any one organism genes have similar A+T content, the A+T content can vary tremendously from organism to organism. For example, some Bacillus species have among the most A+T rich genomes while some Steptomyces species are among the least A+T rich genomes (about 30 to 35% A+T).<br>
Due to the degeneracy of the genetic code and the limited number of codon choices for any amino acid, most of the "excess" A+T of the structural coding sequences of some Bacillus species, for example, ate found in the third position of the codons. That is, genes of some Bacillus species have A or T as the third nucleotide in many codons. Thus A+T content in part can determine codon usage bias. In addition, it is clear that genes evolve for maximum function in the organism in which they evolve. This means that particular nucleotide sequences found in a gene from one organism, where they may play no role except to code for a particular stretch of amino acids, have the potential to be recognized as gene control elements in another organism (such as transcriptional promoters or terminators, polyA addition sites, intron splice sites, or specific mRNA degradation signals). It is perhaps surprising that such misread signals are not a more common feature of heterologous gene expression, but this can be explained in part by the relatively homogeneous A+T content (about 50%) of many organisms. This A+T content plus the nature of the genetic code put clear constraints on the likelihood of occurrence of any particular oligonucleotide sequence. Thus, a gene from E. coli with a 50% A+T content is much less likely to contain any particular A+T rich segment than a gene from B. thuringiensis. The same can be true between genes in a bacterium and genes in a plant, for example.<br>
Any of the nucleic acid molecules of the present invention can be altered via any methods known in the art in order to make the codons within the nucleic acid molecule more appropriate for me organism in which the nucleic acid molecule is located. That is, the present invention includes the modification of any of the nucleic acid molecules disclosed herein to improve codon usage in a host organism.<br>
It is preferred that regions comprising many consecutive A+T bases or G+C bases are disrupted since these regions are predicted to have a higher likelihood to form hairpin structure due to self-complementarity. Therefore, insertion of heterogeneous base pairs would reduce the likelihood of self-complementary secondary structure formation which are known to inhibit transcription and/or translation in some organisms. In most cases, the adverse effects may be minimized by using sequences which do not contain more than five consecutive A+T or G+C.<br>
PROTEIN AND PEPTIDE MOLECUUES<br>
A class of agents includes one or more of the polypeptide molecules encoded by a nucleic acid agent of the present invention, A particular preferred class of proteins is that having an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11, 43-44, 57-58, and 90, and fragments thereof.<br>
In another embodiment, the present invention includes polypeptides having a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, 35a-35b and 36. In an embodiment, the present invention includes polypeptides comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95. The present invention includes and provides said substantially purified polypeptide wherein more than one amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95. In a further preferred embodiment the present invention includes polypeptides comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,4649, and 92-95.<br>
In another embodiment, the present invention includes polypeptides having homogentisate prenyl transferase activity and a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 25a-25c, 33a-33c, 34a-34b, 35a-35b and 36. In an embodiment, the present invention includes polypeptides having homogentisate prenyl transferase activity and comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95. The present invention includes and provides said substantially purified polypeptide wherein more than one amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes polypeptides having homogentisate prenyl transferase activity and comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In another embodiment, the present invention includes polypeptides having a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 25a-25c, 33a-33c, 34a-34b, 35a-35b or 36, excluding polypeptides derived from nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, Sulfolobus, Aeropyum, sorghum, or tomato. In a preferred embodiment, the present invention includes polypeptides comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95 excluding polypeptides derived from nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato. The present invention includes and provides said substantially purified polypeptide wherein more than onethe amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes polypeptides comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95, excluding polypeptides derived from nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato.<br>
In another embodiment, the present invention includes polypeptides having homogentisate prenyl transferase activity and a region of conserved amino acid sequence shown in any of Figures 2a-2c, 3a-3c, 25a-25c, 33a-33c, 34a-34b, 35a-35b or 36, excluding polypeptides derived from nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato. In a preferred embodiment, the present invention includes polypeptides having homogentisate prenyl transferase activity and comprising a sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95, excluding polypeptides derived from nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato. The present invention includes and<br>
provides said substantially purified polypeptide wherein more than one amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
In a further preferred embodiment the present invention includes polypeptides having homogentisate prenyl transferase activity and comprising two or more, three or more, or four sequences selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95, excluding polypeptides derived from nucleic acid molecules derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato.<br>
Polypeptide agents may have C-terminal or N-terminal amino acid sequence extensions. One class of N-terminal extensions employed in a preferred embodiment are plastid transit peptides. When employed, plastid transit peptides can be operatively linked to the N-terminal sequence, thereby permitting the localization of the agent polypeptides to plastids. In an embodiment of the present invention, any suitable plastid targeting sequence can be used. Where suitable, a plastid targeting sequence can be substituted for a native plastid targetting sequence, for example, for the CTP occurring natively in the tocopherol homogentisate prenyl transferase protein. In a further embodiment, a plastid targeting sequence that is heterologous to any homogentisate prenyl transferase protein or fragment described herein can be used. In a further embodiment, any suitable, modified plastid targetting sequence can be used. In another embodiment, the plastid targeting sequence is a CTP1 sequence (see WO 00/61771).<br>
In a preferred aspect a protein of the present invention is targeted to a plastid using either a native transit peptide sequence or a heterologous transit peptide sequence. In the case of nucleic acid sequences corresponding to nucleic acid sequences of non-higher plant organisms such as cynobacteria, such nucleic acid sequences can be modified to attach the coding sequence of the protein to a nucleic acid sequence of a plastid targeting peptide.<br>
As used herein, the terms "protein", "peptide molecule", or "polypeptide" include any molecule that comprises five or more amino acids. It is well known in the art that protein, peptide, or polypeptide molecules may undergo modification, including post-translational modifications, such as, but not limited to, disulfide bond formation, glycosylation, phosphorylation, or oligomerization. Thus, as used herein, the terms "protein", "peptide molecule", or "polypeptide" include any protein that is modified by any biological or non-<br>
biological process. The terms "amino acid" and "amino acids" refer to all naturally occurring L-amino acids. This definition is meant to include norleucine, norvaline, ornithine, homocysteine, and homoserine.<br>
One or more of the protein or fragments thereof, peptide molecules, or polypeptide molecules may be produced via chemical synthesis, or more preferably, by expression in a suitable bacterial or eukaryotic host. Suitable methods for expression are described by Sambrook et al, In: Molecular Cloning, A Laboratory Manual, 2nd Edition, Cold Spring Harbor Press, Cold Spring Harbor, NY (1989) or similar texts.<br>
A "protein fragment" is a peptide or polypeptide molecule whose amino acid sequence comprises a subset of the amino acid sequence of that protein. A protein or fragment thereof that comprises one or more additional peptide regions not derived from that protein is a "fusion" protein. Such molecules may be derivatized to contain carbohydrate or other moieties (such as keyhole limpet hemocyanin). Fusion protein or peptide molecules of the present invention are preferably produced via recombinant means.<br>
Another class of agents comprises protein, peptide molecules, or polypeptide<br>
molecules, or fragments or fusions thereof comprising SEQ ID NOs: 5,9-11,43-44,57-58,<br>
and 90, and fragments thereof in which conservative, non-essential, or non-relevant amino<br>
acid residues have been added, replaced, or deleted. Computerized means for designing<br>
modifications in protein structure are known in the art (Dahiyat and Mayo, Science, 278:82-87<br>
(1997)).	<br>
A protein, peptide, or polypeptide of the present invention can also be a homolog protein, peptide, or polypeptide. As used herein, a homolog protein, peptide, or polypeptide or fragment thereof is a counterpart protein, peptide, or polypeptide or fragment thereof in a second species. A homolog can also be generated by molecular evolution or DNA shuffling techniques, so that the molecule retains at least one functional or structure characteristic of the original {see, for example, U.S. Patent 5,811,238).<br>
In another embodiment, the homolog is selected from the group consisting of alfalfa, Arabidopsis, barley, broccoli, cabbage, canola, citrus, cotton, garlic, oat, Allium, flax, an ornamental plant, peanut, pepper, potato, rapeseed, rice, rye, sorghum, strawberry, sugarcane, sugarbeet, tomato, wheat, poplar, pine, fir, eucalyptus, apple, lettuce, lentils, grape, banana, tea, turf grasses, sunflower, soybean, corn, and Phaseolus. More particularly, preferred homologs are selected from canola, rapeseed, corn, Brassica campestris, Brassica napus,oilseed rape, soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, safflower, oil palm, flax, and sunflower. In an even more preferred embodiment, the homolog is selected from the group consisting of canola, rapeseed, corn, Brassica campestris, Brassica napus, oilseed rape, soybean, sunflower, safflower, oil palms, and peanut. In a preferred embodiment, the homolog is soybean. In a preferred embodiment, the homolog is canola. In a preferred embodiment, the homolog is oilseed rape.<br>
In a preferred embodiment, the nucleic acid molecules of the present invention or complements and fragments of either can be utilized to obtain such homologs.<br>
Agents of the present invention include proteins and fragments thereof comprising at least about a contiguous 10 amino acid region preferably comprising at least about a contiguous 20 amino acid region, even more preferably comprising at least about a contiguous 25,35,50,75, or 100 amino acid region of a protein of the present invention. In another preferred embodiment, the proteins of the present invention include between about 10 and about 25 contiguous amino acid region, more preferably between about 20 and about 50 contiguous amino acid region, and even more preferably between about 40 and about 80 contiguous amino acid region.<br>
PLANT CONSTRUCTS AND PLANT TRANSFORMANTS<br>
One or more of the nucleic acid molecules of the present invention may be used in plant transformation or transfection. Exogenous genetic material may be transferred into a plant cell and the plant cell regenerated into a whole, fertile, or sterile plant Exogenous genetic material is any genetic material, whether naturally occurring or otherwise, from any source that is capable of being inserted into any organism.<br>
In a preferred aspect of the present invention the exogenous genetic material comprises a nucleic acid sequence of the present invention, more preferably one that encodes homogentisate prenyl transferase. In another preferred aspect of the present invention the exogenous genetic material of the present invention comprises a nucleic acid sequence encoding an amino acid sequence selected from me group consisting of SEQ ID NOs: 5,9-11, 43-44,57-58, and 90, and complements thereof and fragments of either. In a further aspect of the present invention the exogenous genetic material comprises a nucleic acid sequence encoding an amino acid sequence selected from the group consisting of SEQ ID NOs: 5,9-11, 43-44,57-58, and 90, and fragments of SEQ ID NOs: 5,9-11,43-44,57-58, and 90.<br>
In an embodiment of the present invention, exogenous genetic material encoding a homogentisate prenyl transferase enzyme or fragment thereof is introduced into a plant with one or more additional genes. In one embodiment, preferred combinations of genes include a nucleic acid molecule of the present invention and one or more of the following genes: tyrA {e.g., WO 02/089561 and Xia et al., J. Gen. Microbiol, 138:1309-1316 (1992)), tocopherol cyclase {e.g., WO 01/79472), prephenate dehydrogenase, dxs {e.g. Lois et al., Proc. Natl. Acad. Sci. (U.SA.), 95(5):2105-2110 (1998)), dxr {e.g., U.S. Pub. 2002/0108814A and Takahashi et al, Proc. Natl. Acad. Sci (U.SA.), 95 (17), 9879-9884 (1998)), GGPPS {e.g., Bartley and Scolnik, Plant Physiol, 104:1469-1470 (1994)), HPPD {e.g., Norris et al., Plant Physiol, 117:1317-1323 (1998)), GMT(e.g„ U.S. Application 10/219,810, filed August 16, 2002), tMT2 {e.g., U.S. Application 10/279,029, filed October 24,2002), AANT1 (e.g;, WO 02/090506), IDI(E.C.:5.3.3.2; Blanc etal., In: Plant Gene Register, PRG96436; and Sato et al, DNA Res., 4:215-230 (1997)), GGH (Grapes et al., Planta. 213-620 (2001)), or a plant ortholog and an antisense construct for homogentisic acid dioxygenase (Kridl et al, Seed Sci. Res., 1:209:219 (1991); Keegstra, Cell, 56(2):247-53 (1989); Nawrath, et al, Proc. Natl Acad. Sci. (U.SA.), 91:12760-12764 (1994); Cyanobase, www.kazusa.or.jp/cyanobase; Smith et al., Plant J., 11:83-92 (1997); WO 00/32757; ExPASy Molecular Biology Server, http://us.expasy.org/enzyme; MT1 WO 00/10380; gcpE, WO 02/12478; Saint Guily et al., Plant Physiol, 100(2): 1069-1071 (1992); Sato et al, J. DNA Res., 7(l):31-63 (2000)). In such combinations, in some crop plants, e.g., canola, a preferred promoter is a napin promoter and a preferred plastid targeting sequence is a CTP1 sequence. It is preferred mat gene products are targeted to the plastid.<br>
In a preferred combination a nucleic acid molecule encoding a homogentisate prenyl transferase polypeptide and a nucleic acid molecule encoding any of the following enzymes: tyrA, prephenate dehydrogenase, tocopherol cyclase, dxs, dxr, GGPPS, HPPD, tMT2, MT1, GCPE, AANT1, IDI, GGH, GMT, or a plant ortholog and an antisense construct for homogentisic acid dioxygenase are introduced into a plant<br>
For any of the above combinations, a nucleic acid molecule encoding a homogentisate prenyl transferase polypeptide encodes a polypeptide comprising a sequence selected from the group consisting of SEQ ID NOs: 5,9-11,43-44,57-58, and 90. In another preferred embodiment, a nucleic acid molecule encoding a homogentisate prenyl transferase polypeptide encodes a polypeptide comprising one or more of SEQ ID NOs: 39-42,46-49,<br>
and 92-95. In a preferred embodiment, the homogentisate prenyl transferase polypeptide does not have an amino acid sequence that is derived from a nucleic acid derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, wheat, leek, canola, cotton, or tomato.<br>
Such genetic material may be transferred into either monocotyledons or dicotyledons including, but not limited to canola, corn, soybean, Arabidopsis phaseolus, peanut, alfalfa, wheat, rice, oat, sorghum, rapeseed, rye, tritordeum, millet, fescue, perennial ryegrass, sugarcane, cranberry, papaya, banana, safflower, oil palms, flax, muskmelon, apple, cucumber, dendrobiurn, gladiolus, cnrysanthemum, liliacea, cotton, eucalyptus, sunflower, Brassica campestris, Brassica napus, oilseed rape, turfgrass, sugarbeet, coffee and dioscorea (Christou, In: Particle Bombardment for Genetic Engineering of Plants, Biotechnology Intelligence Unit Academic Press, San Diego, CA (1996)), with canola, corn, Brassica campestris, Brassica napus, oilseed rape, rapeseed, soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, safflower, oil palm, flax, and sunflower preferred, and canola, rapeseed, corn, Brassica campestris, Brassica napus, oilseed rape, soybean, sunflower, safflower, oil palms, and peanut preferred, In a more preferred embodiment, the genetic material is transferred into canola. In another more preferred embodiment, the genetic material is transferred into oilseed rape. In another particularly preferred embodiment, the genetic material is transferred into soybean.<br>
Transfer of a nucleic acid molecule that encodes a protein can result in expression or overexpression of that polypeptide in a transformed cell or transgenic plant. One or more of the proteins or fragments thereof encoded by nucleic acid molecules of the present invention may be overexpressed in a transformed cell or transformed plant Such expression or overexpression may be the result of transient or stable transfer of the exogenous genetic material.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of tocopherols.<br>
In a preferred embodiment expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of α-tocopherols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of Y-tocopherols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of δ-tocopherols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of ß-tocopherols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of tocotrienols.<br>
m a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of a-tocotrienols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of Y-tocotrienols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of δ-tocotrienols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of ß-tocotrienols.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an increased level of plastoquinols.<br>
In any of the embodiments described herein, an increase in -tocopherol, α-tocopherol, or both can lead to a decrease in the relative proportion of ß-tocopherol, δ-tocopherol, or both. Similarly, an increase in -tocotienol, α-tocotrienol, or both can lead to a decrease in the relative proportion of ß tocotrienol, δ-tocotrienol, or both.<br>
In another embodiment, expression overexpression of a polypeptide of the present invention in a plant provides in that plant, or a tissue of that plant, relative to an untransformed plant or plant tissue, with a similar genetic background, an increased level of a homogentisate prenyl transferase, protein or fragment thereof.<br>
In some embodiments, the levels of one or more products of the tocopherol biosynthesis pathway, including any one or more of tocopherols, α-tocopherols, -tocopherols, δ-tocopherols, ß-tocopherols, tocotrienols, α-tocotrienols, -tocotrienols, δ-tocotrienols ß-tocotrienols are increased by greater than about 10%, or more preferably greater than about 25%, 35%, 50%, 75%, 80%, 90%, 100%, 150%, 200%, 1,000%, 2,000%, or 2,500%. The levels of products may be increased throughout an organism such as a plant or localized in one or more specific organs or tissues of the organism. For example, the levels of products may be increased in one or more of the tissues and organs of a plant including without limitation: roots, tubers, stems, leaves, stalks, fruit, berries, nuts, bark, pods, seeds and flowers. A preferred organ is a seed.<br>
In some embodiments, the levels of one or more products of the tocopherol biosynthesis pathway, including any one or more of tocopherols, a-tocopherols, -tocopherols, δ-tocopherols, ß tocopherols, tocotrienols, α-tocotrienols, -tocotrienols, δ-tocotrienols, ß-tocotrienols are increased so that they constitute greater than about 10%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% of the total tocopherol content of the organism or tissue. The levels of products may be increased throughout an organism such as a plant or localized in one or more specific organs or tissues of the organism. For example, the levels of products may be increased in one or more of the tissues and organs of a plant including without limitation: roots, tubers, stems, leaves, stalks, fruit, berries, nuts, bark, pods, seeds and flowers. A preferred organ is a seed.<br>
In a preferred embodiment, expression of enzymes involved is tocopherol, tocotrienol or plastoquinol synthesis in me seed will result in an increase in -tocopherol levels due to the absence of significant levels of GMT activity in those tissues. In another preferred embodiment, expression of enzymes involved in tocopherol, tocotrienol, or plastoquinol synthesis in photosyhthetic tissues will result in an increase in α-tocopherol due to the higher levels of GMT activity in those tissues relative to the same activity in seed tissue.<br>
In another preferred embodiment, the expression of enzymes involved in tocopherol, tocotrienol, or plastoquinol synthesis in the seed will result in an increase in the total tocopherol, tocotrienol, or plastoquinol level in the plant.<br>
In some embodiments, the levels of tocopherols or a species such as α-tocopherol may be altered. In some embodiments, the levels of tocotrienols may be altered. Such alteration can be compared to a plant with a similar background<br>
In another embodiment, either the α-tocopherol level, α-tocotrienol level, or both of plants that natively produce high levels of either α-tocopherol, α-tocotrienol or bom (e.g., sunflowers), can be increased by the introduction of a gene coding for a homogentisate prenyl transferase enzyme.<br>
In a preferred aspect, a similar genetic background is a background where the organisms being compared share about 50% or greater of their nuclear genetic material. In a more preferred aspect a similar genetic background is a background where the organisms being compared share about 75% or greater, even more preferably about 90% or greater of their nuclear genetic material. In another even more preferable aspect, a similar genetic background is a background where the organisms being compared are plants, and the plants are isogenic except for any genetic material originally introduced using plant transformation techniques.<br>
In another preferred embodiment, expression or overexpression of a polypeptide of the present invention in a transformed plant may provide tolerance to a variety of stress, e.g. oxidative stress tolerance such as to oxygen or ozone, UV tolerance, cold tolerance, or fungal/microbial pathogen tolerance.<br>
As used herein in a preferred aspect, a tolerance or resistance to stress is determined by the ability of a plant, when challenged by a stress such as cold to produce a plant having a higher yield than one without such tolerance or resistance to stress. In a particularly preferred aspect of the present invention, the tolerance or resistance to stress is measured relative to a plant with a similar genetic background to the tolerant or resistance plant except that the plant reduces the expression, expresses, or over expresses a protein or fragment thereof of the present invention.<br>
Exogenous genetic material may be transferred into a host cell by theuse of a DNA vector or construct designed for such a purpose. Design of such a vector is generally within<br>
the skill of the art (see, Plant Molecular Biology: A Laboratory Manual, Clark (ed.), Springer, NY (1997)).<br>
A construct or vector may include a plant promoter to express the polypeptide of choice. In a preferred embodiment, any nucleic acid molecules described herein can be operably linked to a promoter region which functions in a plant cell to cause the production of an mRNA molecule. For example, any promoter that functions in a plant cell to cause the production of an mRNA molecule, such as those promoters described herein, without limitation, can be used. Li a preferred embodiment, the promoter is a plant promoter. A number of promoters that are active in plant cells have been described in the literature. These include the nopaline synthase (NOS) promoter (Ebert et al., Proc. Natl. Acad. Sci. (U.SA), 84:5745-5749 (1987)), the octopine synthase (OCS) promoter (which is carried on tumor-inducing plasmids of Agrobacterium tumefaciens), the caulimovirus promoters such as the cauliflower mosaic virus (CaMV) 19S promoter (Lawton et al., Plant Mol Biol, 9:315-324 (1987)) and the CaMV 35S promoter (Odell et al., Nature, 313:810-812 (1985)), the figwort mosaic virus 35S-promoter, the light-inducible promoter from the small subunit of ribulose-l,5-bis-phosphate carboxylase (ssRUBISCO), the Adh promoter (Walker et al., Proc. Natl. Acad. Sci. (U.S.A.), 84:6624-6628 (1987)), the sucrose synthase promoter (Yang et al.,Proc. Natl. Acad. Sci. (U.SA.), 87:4144-4148 (1990)), the R gene complex promoter (Chandler et al., The Plant Cell, 1:1175-1183 (1989)) and the chlorophyll a/b binding protein gene promoter, etc. These  promoters have been used to create DNA constructs that have been expressed in plants; see, e.g., WO 84/02913. The CaMV 35S promoters are preferred for use in plants. Promoters known or found to cause transcription of DNA in plant cells can be used in the present invention.<br>
For the purpose of expression in source tissues of the plant, such as the leaf, seed, root or stem, it is preferred that the promoters utilized have relatively high expression in these specific tissues. Tissue-specific expression of a protein of the present invention is a particularly preferred embodiment For this purpose, one may choose from a number of promoters for genes with tissue- or cell-specific or enhanced expression. Examples of such promoters reported in the literature include the chloroplast glutamine synthetase GS2 promoter from pea (Edwards et al., Proc. Natl. Acad. Sci. (U.SA.), 87:3459-3463 (1990)), the chloroplast fructose-l,6-biphosphatase (FBPase) promoter from wheat (Uoyd et al., Mol. Gen. Genet., 225:209-216 (1991)), the nuclear photosynthetic ST-LS1 promoter from potato<br>
(Stockhaus et al, EMBO J., 8:2445-2451 (1989)), the serine/threonine kinase (PAL) promoter and the glucoamylase (CHS) promoter from Arabidopsis thaliana. Also, reported to be active in photosynthetically active tissues are the ribulose-l,5-bisphosphate carboxylase (RbcS) promoter from eastern larch (Larix laricina), the promoter for the cab gene, cab6, from pine (Yamamoto et al, Plant Cell Physiol, 35:773-778 (1994)), the promoter for the Cab-1 gene from wheat (Fejes et al., Plant Mol Biol, 15:921-932 (1990)), the promoter for the CAB-1 gene from spinach (Lubberstedt et al, Plant Physiol, 104:997-1006 (1994)), the promoter for the cablR gene from rice (Luan et al, Plant Cell, 4:971-981 (1992)), the pyruvate, orthophosphate dikinase (PPDK) promoter from corn (Matsuoka et al., Proc. Natl Acad, Sci. (U.SA.), 90:9586-9590 (1993)), the promoter for the tobacco Lhcbl*2 gene (Cerdan et al., Plant Mol Biol, 33:245-255 (1997)), the Arabidopsis ilialiana SUC2 sucrose-H+ symporter promoter (Truerm't et al., Planta., 196:564-570 (1995)) and the promoter for the thylakoid membrane proteins from spinach (psdD, psdF, psaE, PC, FNR, atpC, atpD, cab, rbcS). Other promoters for the chlorophyll a/b-binding proteins may also be utilized in the present invention, such as the promoters for LhcB gene and PsbP gene from white mustard {Sinapis alba; Kretsch et al., Plant Mol Biol, 28:219-229 (1995)).<br>
For the purpose of expression in sink tissues of the plant, such as the tuber of the potato plant, the fruit of tomato, or the seed of corn, wheat, rice and barley, it is preferred that the promoters utilized in the present invention have relatively high expression in these specific tissues. A number of promoters for genes with tuber-specific or tuber-enhanced expression are known, including the class I patatin promoter (Bevan et al., EMBO J., 8:1899-1906 (1986); Jefferson et al, Plant Mol Biol, 14:995-1006 (1990)), the promoter for the potato tuber ADPGPP genes, both the large and small subunits, the sucrose synthase promoter (Salanoubat and Belliard, Gene, 60:47-56 (1987), Salanoubat andBelliard, Gene, 84:181-185 (1989)), the promoter for the major tuber proteins including the 22 kd protein complexes and protease inhibitors (Hannapel, Plant Physiol, 101:703-704 (1993)), the promoter for the granule-bound starch synthase gene (GBSS) (Visser et al., Plant Mol. Biol, 17:691-699 (1991)) and other class I and II patatins promoters (Koster-Topfer et al, Mol Gen. Genet., 219:390-396 (1989); Mignery et al., Gene., 62:27-44 (1988)).<br>
Other promoters can also be used to express a polypeptide in specific tissues, such as seeds or fruits. Indeed, in a preferred embodiment, the promoter used is a seed specific promoter. Examples of such promoters include the 5' regulatory regions from such genes as<br>
napin (Kridl et al, SeedSci. Res., 1:209:219 (1991)), phaseolin (Bustos et al, Plant Cell, l(9):839-853 (1989)), soybean trypsin inhibitor (Riggs et al, Plant Cell, 1(6):609-621 (1989)), ACP (Baerson et al., Plant Mol Biol, 22(2):255-267 (1993)), stearoyl-ACP desaturase (Slocombe et al., Plant Physiol, 104(4): 167-176 (1994)), soybean a' subunit of ß-conglycinin (soy 7s, (Chen et al, Proc. Natl Acad. Sci, 83:8560-8564 (1986))), and oleosin (see, for example, Hong et al., Plant Mol Biol, 34(3):549-555 (1997)). Further examples include the promoter for ß-conglycinin (Chen et al., Dev. Genet., 10:112-122 (1989)). Also included are the zeins, which are a group of storage proteins found in com endosperm. Genomic clones for zein genes have been isolated (Pedersen et al., Cell, 29:1015-1026 (1982), and Russell et al, Transgenic Res., 6(2):157-168) and the promoters from these clones, including the 15 kD, 16 kD, 19 kD, 22 kD, 27 kD and genes, could also be used. Other promoters known to function, for example, in com include the promoters for the following genes: waxy, Brittle, Shrunken 2, Branching enzymes I and IL starch synthases, debranching enzymes, oleosins, glutelins and sucrose synthases. A particularly preferred promoter for corn endosperm expression is the promoter for the glutelin gene from rice, more particularly the Osgt-1 promoter (Zheng et al., Mol Cell Biol, 13:5829-5842 (1993)). Examples of promoters suitable for expression in wheat include those promoters for the ADPglucose pyrosynthase (ADPGPP) subunits, the granule bound and other starch synthase, the branching and debranching enzymes, the embryogenesis-abundant proteins, the gliadins and the glutenins. Examples of such promoters in rice include those promoters for the ADPGPP subunits, the granule bound and other starch synthase, the branching enzymes, the debranching enzymes, sucrose synthases and the glutelins. A particularly preferred promoter is the promoter for rice glutelin, Osgt-1. Examples of such promoters for barley include those for the ADPGPP subunits, the granule bound and other starch synthase, the branching . enzymes, the debranching enzymes, sucrose synthases, the hordeins, the embryo globulins and the aleurone specific proteins. A preferred promoter for expression in the seed is a napin promoter. Another preferred promoter for expression is an Arcelin 5 promoter.<br>
Root specific promoters may also be used. An example of such a promoter is the promoter for the acid chitinase gene (Samac et al., Plant Mol Biol, 25:587-596 (1994)). Expression in root tissue could also be accomplished by utilizing the root specific subdomains of the CaMV35S promoter that have been identified (Lam et al., Proc. Natl Acad. Sci<br>
(U.S.A.), 86:7890-7894 (1989)). Other root cell specific promoters include those reported by Conkling et al., Plant Physiol, 93:1203-1211 (1990).<br>
Other preferred promoters include 7Sα' (Beachy et al., EMBO J., 4:3047 (1985); Schuler et al, Nucleic Acid Res., 10(24):8225-8244 (1982)); USP 88 and enhanced USP 88 (U.S. Patent Application 60/377,236, filed May 3,2002, incorporated herein by reference); and 7Sα, (U.S. Patent Application 10/235,618).<br>
Additional promoters that may be utilized are described, for example, in U.S. Patents 5,378,619; 5,391,725; 5,428,147; 5,447,858; 5,608,144; 5,608,144; 5,614,399; 5,633,441; 5,633,435; and 4,633,436. In addition, a tissue specific enhancer may be used (Fromm et al., The Plant Cell, 1:977-984 (1989)).<br>
Constructs or vectors may also include, with the coding region of interest, a nucleic acid sequence that acts, in whole or in part, to terminate transcription of that region. A number of such sequences have been isolated, including the Tr7 3' sequence and the NOS 3' sequence (Ingelbrecht et al., The Plant Cell, 1:671-680 (1989); Bevan et al., Nucleic Acids Res., 11:369-385 (1983)). Regulatory transcript terrnination regions can be provided in plant expression constructs of this present invention as well. Transcript termination regions can be provided by the DNA sequence encoding the gene of interest or a convenient transcription termination region derived from a different gene source, for example, the transcript termination region that is naturally associated with the transcript initiation region. The skilled artisan will recognize that any convenient transcript termination region that is capable of terminating transcription in a plant cell can be employed in the constructs of the present invention.<br>
A vector or construct may also include regulatory elements. Examples of such include the Adh intron 1 (Calks et al., Genes and Develop., 1:1183-1200 (1987)), the sucrose synthase intron (Vasil et al., Plant Physiol, 91:1575-1579 (1989)) and the TMV omega element (Gallie et al., The Plant Cell, 1:301-311 (1989)). These and other regulatory elements may be included when appropriate.<br>
A vector or construct may also include a selectable marker. Selectable markers may also be used to select for plants or plant cells that contain the exogenous genetic material. Examples of such include, but are not limited to: a neo gene (Potrykus et al., Mol Gen. Genet., 199:183-188 (1985)), which codes for kanamycin resistance and can be selected for using kanamycin, RptII, G418, hpt etc.; a bar gene which codes for bialaphos resistance; a<br>
mutant EPSP synthase gene (Hinchee et al, Bio/Tedinology, 6:915-922 (1988); Reynaerts et al., Selectable and Screenable Markers. In: Gelvin and Schilperoort, Plant Molecular Biology Manual, Kluwer, Dordrecht (1988); Reynaerts et al., Selectable and Screenable Markers. In: Gelvin and Schilperoort, Plant Molecular Biology Manual, Kluwer, Dordrecht (1988)), aadA (Jones et al., Mol. Gen. Genet. (1987)), which encodes glyphosate resistance; a nitrilase gene which confers resistance to bromoxynil (Stalker et al., J. Biol. Client., 263:6310-6314 (1988)); a mutant acetolactate synthase gene (ALS) which confers imidazolinone or sulphonylurea resistance (EP 0154 204 (Sept 11,1985)), ALS (D'Halluin et al., Bio/Technology, 10:309-314 (1992)), and a methotrexate resistant DHFR gene (Thillet et al., J. Biol. Chem., 263:12500-12508 (1988)).<br>
A vector or construct may also include a transit peptide. Incorporation of a suitable chloroplast transit peptide may also be employed (EP 0 218 571). Translational enhancers may also be incorporated as part of the vector DNA. DNA constructs could contain one or more 5' non-translated leader sequences, which may serve to enhance expression of the gene products from the resulting mRNA transcripts. Such sequences may be derived from the promoter selected to express the gene or can be specifically modified to increase translation of the mRNA. Such regions may also be obtained from viral RNAs, from suitable eukaryotic genes, or from a synthetic gene sequence. For a review of optimizing expression of transgenes, see Koziel et al., Plant Mol. Biol, 32:393-405 (1996). A preferred transit peptide is CTP1.<br>
A vector or construct may also include a screenable marker. Screenable markers may be used to monitor expression. Exemplary screenable markers include: a ß-glucuronidase or uidA gene (GUS) which encodes an enzyme for which various chromogenic substrates are known (Jefferson, Plant Mol Biol, Rep., 5:387-405 (1987); Jefferson et al., EMBO J., 6:3901-3907 (1987)); an R-locus gene, which encodes a product that regulates the production of anthocyanin pigments (red color) in plant tissues (Dellaporta et al., Stadler Symposium, 11:263-282 (1988)); a ß-lactamase gene (Sutcliffe et al., Proc. Natl Acad. Set (USA.), 75:3737-3741 (1978)), a gene which encodes an enzyme for which various chromogenic substrates are known (e.g., PADAC, a chromogenic cephalosporin); a luciferase gene (Ow et al., Science, 234:856-859 (1986)); a xylE gene (Zukowsky et al., Proc. Natl Acad. Sci (V.SA.), 80:1101-1105 (1983)) which encodes a catechol dioxygenase that can convert chromogenic catechols; an α-amylase gene (Ikatu et al., Bio/Technol, 8:241-242 (1990)); a<br>
tyrosinase gene (Katz et al., J. Gen. Microbiol., 129:2703-2714 (1983)) which encodes an enzyme capable of oxidizing tyrosine to DOPA and dopaquinone which in turn condenses to melanin; an α-galactosidase, which will turn a chromogenic α-galactose substrate.<br>
included within the terms "selectable or screenable marker genes" are also genes that encode a secretable marker whose secretion can be detected as a means of identifying or selecting for transformed cells. Examples include markers that encode a secretable antigen that can be identified by antibody interaction, or even secretable enzymes that can be detected catalytically. Secretable proteins fall into a number of classes, including small, diffusible proteins that are detectable, (e.g., by ELLSA), small active enzymes that are detectable in extracellular solution (e.g., α-amyiase, ß-lactamase, phosphinothricin transferase), or proteins that are inserted or trapped in the cell wall (such as proteins that include a leader sequence such as that found in the expression unit of extension or tobacco PR-S). Other possible selectable and/or screenable marker genes will be apparent to those of skill in the art.<br>
There are many methods for introducing transfonrming nucleic acid molecules into plant cells. Suitable methods are believed to include virtually any method by which nucleic acid molecules may be introduced into a cell, such as by Agrobacterium infection or direct delivery of nucleic acid molecules such as, for example, by PEG-mediated transformation, by electroporation or by acceleration of DNA coated particles, and the like. (Potrykus, Ann. Rev. Plant Physiol. Plant Mol. Biol, 42:205-225 (1991); Vasil, Plant Mol. Biol, 25:925-937 (1994)). For example, electroporation has been used to transform corn protoplasts (Fromm et al., Nature, 312:791-793 (1986)).<br>
Other vector systems suitable for introducing transforming DNA into a host plant cell include but are not limited to binary artificial chromosome (BIBAC) vectors (Hamilton et al, Gene, 200:107-116 (1997)); and transfection with RNA viral vectors (Della-Cioppa et al., Ann. N.Y. Acad. Sci. (1996), 792 (Engineering Plants for Commercial Products and Applications, 57-61). Additional vector systems also include plant selectable YAC vectors such as those described in Mullen et al, Molecular Breeding, 4:449-457 (1988).<br>
Technology for introduction of DNA into cells is well known to those of skill in the art Four general methods for delivering a gene into cells have been described: (1) chemical methods (Graham and van der Eb, Virology, 54:536-539 (1973)); (2) physical methods such as microinjection (Capecchi, Cell, 22:479-488 (1980)), electroporation (Wong and Neumann, Biochem. Biophys. Res. Commun., 107:584-587 (1982); Fromm et al, Proc. Natl. Acad. Sci<br>
(U.SA.), 82:5824-5828 (1985); U.S. Patent 5,384,253); the gene gun (Johnston and Tang, Methods Cell Biol, 43:353-365 (1994)); and vacuum infiltration (Bechtold et al., C.R. Acad. Sci. Paris, Life Sci., 316:1194-1199 (1993)); (3) viral vectors (Clapp, Clin. Perinatol., 20:155-168 (1993); Lu et al, J. Exp. Med., 178:2089-2096 (1993); Eglitis and Anderson, Biotechniques, 6:608-614 (1988)); and (4) receptor-mediated mechanisms (Curiel et al, Hum. Gen. Ther., 3:147-154 (1992), Wagner et al, Proc. Natl Acad. Sci. (U.S.A.), 89:6099-6103 (1992)).<br>
Acceleration methods that may be used include, for example, microprojectile bombardment and the like. One example of a method for delivering transforming nucleic acid molecules into plant cells is microprojectile bombardment This method has been reviewed by Yang and Christou (eds.), Particle Bombardment Technology for Gene Transfer, Oxford Press, Oxford, England (1994). Non-biological particles (microprojectiles) may be coated with nucleic acids and delivered into cells by a propelling force. Exemplary particles include those comprised of tungsten, gold, platinum and the like.<br>
A particular advantage of microprojectile bombardment, in addition to it being an effective means of reproducibly transforming monocots, is that neither the isolation of protoplasts (Cristou et al., Plant Physiol, 87:671-674 (1988)) nor the susceptibility to Agrobacterium infection is required. An illustrative embodiment of a method for delivering DNA into com cells by acceleration is a biolistics α-particle delivery system, which can be used to propel particles coated with DNA through a screen, such as a stainless steel or Nytex screen, onto a filter surface covered with corn cells cultured in suspension. Gordon-Kamm et al., describes the basic procedure for coating tungsten particles with DNA (Gordon-Kamm et al., Plant Cell, 2:603-618 (1990)). The screen disperses the tungsten nucleic acid particles so that they are not delivered to the recipient cells in large aggregates. A particle delivery system suitable for use with the present invention is the helium acceleration PDS-1000/He gun, which is available from Bio-Rad Laboratories (Bio-Rad, Hercules, CA) (Sanford et al., Technique, 3:3-16 (1991)).<br>
For the bombardment, cells in suspension may be concentrated on filters. Filters containing the cells to be bombarded are positioned at an appropriate distance below the microprojectile stopping plate. If desired, one or more screens are also positioned between the gun and the cells to be bombarded.<br><br>
Alternatively, immature embryos or other target cells may be arranged on solid culture medium. The cells to be bombarded are positioned at an appropriate distance below the microprojectile stopping plate. If desired, one or more screens are also positioned between the acceleration device and the cells to be bombarded. Through the use of techniques set forth herein one may obtain 1000 or more loci of cells transiently expressing a marker gene. The number of cells in a focus that express the exogenous gene product 48 hours post-bombardment often ranges from one to ten, and average one to three.<br>
In bombardment transformation, one may optimize the pre-bombardment culturing conditions and the bombardment parameters to yield the maximum numbers of stable transformants. Both the physical and biological parameters for bombardment are important in this technology. Physical factors are those that involve manipulating the DNA/microprojectile precipitate or those that affect the flight and velocity of either the macro- or microprojectiles. Biological factors include all steps involved in manipulation of cells before and immediately after bombardment, the osmotic adjustment of target cells to help alleviate the trauma associated with bombardment and also the nature of the transforming DNA, such as linearized DNA or intact supercoiled plasmids. It is believed that pre-bombardment manipulations are especially important for successful transformation of immature embryos.<br>
In another alternative embodiment, plastids can be stably transformed. Methods disclosed for plastid transformation in higher plants include the particle gun delivery of DNA containing a selectable marker and targeting of the DNA to the plastid genome through homologous recombination (Svab et al., Proc. Natl Acad. Sci. (U.SA.), 87:8526-8530 (1990); Svab and Maliga, Proc. Natl. Acad. Sci. (U.SA.), 90:913-917 (1993); Staub and Maliga, EMBO J., 12:601-606 (1993); U.S. Patents 5,451,513 and 5,545,818).<br>
Accordingly, it is contemplated that one may wish to adjust various aspects of the bombardment parameters in small scale studies to fully optimize the conditions. One may particularly wish to adjust physical parameters such as gap distance, flight distance, tissue distance and helium pressure. One may also minimize the trauma reduction factors by modifying conditions that influence the physiological state of the recipient cells and which may therefore influence transformation and integration efficiencies. For example, the osmotic state, tissue hydration and the subculture stage or cell cycle of the recipient cells may be<br><br>
adjusted for optimum transformation. The execution of other routine adjustments will be known to those of skill in the art in light of the present disclosure.<br>
Agrobacterium-mediated transfer is a widely applicable system for introducing genes into plant cells because the DNA can be introduced into whole plant tissues, thereby bypassing the need for regeneration of an intact plant from a protoplast The use of Agrobacterium-mediated plant integrating vectors to introduce DNA into plant cells is well known in the art. See, for example, the methods described by Fraley et al., Bio/Technology, 3:629-635 (1985) and Rogers et al., Methods Enzymol, 153:253-277 (1987). Further, the integration of the Ti-DNA is a relatively precise process resulting in few rearrangements. The region of DNA to be transferred is defined by the border sequences and intervening DNA is usually inserted into the plant genome as described (Spielmanri et al., Mol Gen. Genet, 205:34 (1986)).<br>
Modern Agrobacterium transformation vectors are capable of replication in E. coli as well as Agrobacterium, allowing for convenient manipulations as described (Klee et al., In: Plant DNA Infectious Agents, Hohn and Schell (eds.), Springer-Verlag, NY, pp. 179-203 (1985)). Moreover, technological advances in vectors for Agrofcacferium-mediated gene transfer have improved the arrangement of genes and restriction sites in the vectors to facilitate construction of vectors capable of expressing various polypeptide coding genes. The vectors described have convenient multi-linker regions flanked by a promoter and a polyadenylation site for direct expression of inserted polypeptide coding genes and are suitable for present purposes (Rogers et al., Methods Enzymol., 153:253-277 (1987)). In addition, Agrobacterium containing both armed and disarmed Ti genes can be used for the transformations. In those plant strains where Agrobacterium-mediated transformation is efficient, it is the method of choice because of the facile and defined nature of the gene transfer.<br>
A transgenic plant formed using Agrobacterium transformation methods typically contains a single gene on one chromosome. Such transgenic plants can be referred to as being heterozygous for the added gene. More preferred is a transgenic plant that is homozygous for the added structural gene; i.e., a transgenic plant that contains two added genes, one gene at the same locus on each chromosome of a chromosome pair. A homozygous transgenic plant can be obtained by sexually mating (selfing) an independent segregant, transgenic plant that<br>
contains a single added gene, germinating some of the seed produced and analyzing the resulting plants produced for the gene of interest.<br>
It is also to be understood that two different transgenic plants can also be mated to produce offspring that contain two independendy segregating, exogenous genes. Selfing of appropriate progeny can produce plants that are homozygous for both added, exogenous genes that encode a polypeptide of interest Back-crossing to a parental plant and out-crossing with a non-transgenic plant are also contemplated, as is vegetative propagation.<br>
Transformation of plant protoplasts can be achieved using methods based on calcium phosphate precipitation, polyethylene glycol treatment, electroporation and combinations of these treatments (see, for example, Potrykus et al., Mol Gen. Genet., 205:193-200 (1986); Lorz etal., Mol. Gen. Genet., 199:178 (1985); Fromm et al., Nature, 319:791 (1986); Uchimiya et al., Mol. Gen. Genet, 204:204 (1986); Marcotte et al., Nature, 335:454-457 (1988)).<br>
Application of these systems to different plant strains depends upon the ability to regenerate that particular plant strain from protoplasts. Illustrative methods for the regeneration of cereals from protoplasts are described (Fujimura et al., Plant Tissue Culture Letters, 2:74 (1985); Toriyama et al., Theor. Appl. Genet., 205:34 (1986); Yamada et al., Plant Cell Rep., 4:85 (1986); Abdullah et al., Biotechnology, 4:1087 (1986)).<br>
To transform plant strains that cannot be successfully regenerated from protoplasts, other ways to introduce DNA into intact cells or tissues can be utilized. For example, regeneration of cereals from immature embryos or explants can be effected as described (Vasil, Biotechnology, 6:397 (1988)). In addition, "particle gun" or high-velocity microprojectile technology can be utilized (Vasil et al., Bio/Technology, 10:667 (1992)).<br>
Using the latter technology, DNA is carried through the cell wall and into the cytoplasm on the surface of small metal particles as described (Klein et al., Nature, 328:70 (1987); Klein et al, Proc Natl Acad. Set (U.SA.), 85:8502-8505 (1988); McCabe et al, Bio/Technology, 6:923 (1988)). The metal particles penetrate through several layers of cells and thus allow the transformation of cells within tissue explants.<br>
Other methods of cell transformation can also be used and include but are not limited to introduction of DNA into plants by direct DNA transfer into pollen (Hess et al., Intern Rev. Cytol, 107:367 (1987); Luo et al, Plant Mol Biol Reporter, 6:165 (1988)), by direct injection of DNA into reproductive organs of a plant (Pena et al, Nature, 325:274 (1987)), or by direct<br>
injection of DNA into the cells of immature embryos followed by the rehydration of desiccated embryos (Neuhaus et al, Theor. Appl. Genet, 75:30 (1987)).<br>
The regeneration, development and cultivation of plants from single plant protoplast transformants or from various transformed explants is well known in the art (Weissbach and Weissbach, In: Methods for Plant Molecular Biology, Academic Press, San Diego, CA, (1988)). This regeneration and growth process typically includes the steps of selection of transformed cells, culturing those individualized cells through the usual stages of embryonic development through the rooted plantlet stage. Transgenic embryos and seeds are similarly regenerated. The resulting transgenic rooted shoots are thereafter planted in an appropriate plant growth medium such as soil.<br>
The development or regeneration of plants containing the foreign, exogenous gene that encodes a protein of interest is well known in the art. Preferably, the regenerated plants are self-pollinated to provide homozygous transgenic plants. Otherwise, pollen obtained from the regenerated plants is crossed to seed-grown plants of agronomically important lines. Conversely, pollen from plants of these important lines is used to pollinate regenerated plants. A transgenic plant of the present invention containing a desired polypeptide is cultivated using methods well known to one skilled in the art<br>
There are a variety of methods for the regeneration of plants from plant tissue. The particular method of regeneration will depend on the starting plant tissue and the particular plant species to be regenerated.<br>
Methods for transforming dicots, primarily by use of Agrobacterium tumefaciens and obtaining transgenic plants have been published for cotton (U.S. Patents 5,004,863; 5,159,135; and 5,518,908); soybean (U.S. Patents 5,569,834 and 5,416,011; McCabe et al., Biotechnology, 6:923 (1988); Christou et al., Plant Pltysiol. 87:671-674 (1988)); Brassica (U.S. Patent 5,463,174); peanut (Cheng et al., Plant Cell Rep., 15:653-657 (1996), McKently et al. Plant Cell Rep., 14:699-703 (1995)); papaya; pea (Grant et al., Plant CeU Rep., 15:254-258 (1995)); and Arabidopsis thaliana (Bechtold et al., C.R. Acad. Set Paris, Life Sci, 316:1194-1199 (1993)). The latter method for transforming Arabidopsis thaliana is commonly called "dipping" or vacuum infiltration or germplasm transformation.<br>
Transformation of monocotyledons using electroporation, particle bombardment and Agrobacterium have also been reported. Transformation and plant regeneration have been achieved in asparagus (Bytebier et al., Proc. Natl. Acad. Sci (U.SA.), 84:5354 (1987)); barley<br>
(Wan and Lemaux, Plcuit Physiol, 104:37 (1994)); corn (Rhodes et al. Science, 240:204<br>
(1988); Gordon-Kamm et al, Plant Cell, 2:603-618 (1990); Fromm et al, Bio/Technology,<br>
8:833 (1990); Koziel et al, Bio/Technology, 11:194 (1993); Armstrong et al, Crop Science,<br>
35:550-557 (1995)); oat (Somers et al, Bio/Teclmology, 10:1589 (1992)); orchard grass (Horn<br>
et al., Plant Cell Rep., 7:469 (1988)); rice (Toriyama et al., Theor Appl Genet., 205:34<br>
(1986); Part et al., Plant Mol. Biol, 32:1135-1148 (1996); Abedinia et al., Aust. J. Plant<br>
Physiol, 24:133-141 (1997); Zhang and Wu, Tlieor. Appl. Genet., 76:835 (1988); Zhang et<br>
al., Plant Cell Rep., 7:379 (1988); Battraw and Hall, Plant Set, 86:191-202 (1992); Christou<br>
et al., Bio/Technology, 9:957 (1991)); rye (De la Pena et al., Nature, 325:274 (1987));<br>
sugarcane (Bower and Birch, Plant J., 2:409 (1992)); tall fescue (Wang et al.,<br>
Bio/Teclmology, 10:691 (1992)); and wheat (Vasil et al., Bio/Technology, 10:667 (1992); U.S.<br>
Patent 5,631,152),	'<br>
Assays for gene expression based on the transient expression of cloned nucleic acid constructs have been developed by introducing the nucleic acid molecules into plant ceils by polyethylene glycol treatment, electroporation, or particle bombardment (Marcotte et al., Nature, 335:454-457 (1988); Marcotte et al., Plant Cell, 1:523-532 (1989); McCarty et al., Cell, 66:895-905 (1991); Hattori et al., Genes Dev., 6:609-618 (1992); Goff et al., EMBO J., 9:2517-2522 (1990)). Transient expression systems may be used to functionally dissect gene constructs {see generally, Mailga et al., Methods in Plant Molecular Biology, Cold Spring Harbor Press, NY (1995)).<br>
Any of the nucleic acid molecules of the present invention may be introduced into a plant cell in a permanent or transient manner in combination with other genetic elements such as vectors, promoters, enhancers, etc. Further, any of the nucleic acid molecules of the present invention may be introduced into a plant cell in a manner that allows for expression or overexpression of the protein or fragment thereof encoded by the nucleic acid molecule.<br>
Cosuppression is the reduction in expressionlevels, usually at the level of RNA, of a particular endogenous gene or gene family by the expression of a homologous sense construct that is capable of transcribing mRNA of the same strandedness as the transcript of the endogenous gene (Napoli et al, Plant Cell, 2:279-289 (1990); van der Krol et al, Plant Cell, 2:291-299 (1990)). Cosuppression may result from stable transformation with a single copy nucleic acid molecule that is homologous to a nucleic acid sequence found with the cell (Prolls and Meyer, Plant J., 2:465-475 (1992)) or with multiple copies of a nucleic acid<br>
molecule that is homologous to a nucleic acid sequence found with the cell (Mittlesten et al., Mol. Gen. Genet., 244:325-330 (1994)). Genes, even though different, linked to homologous promoters may result in the cosuppression of the linked genes (Vaucheret, C.R. Acad. Sci. III, 316:1471-1483 (1993); Flavell, Proc. Natl. Acad. Sci. (U.S.A.), 91:3490-3496 (1994)); van Blokland et al., Plant J., 6:861-877 (1994); Jorgensen, Trends Biotechnol, 8:340-344 (1990); Meins and Kunz, In: Gene Inactivation and Homologous Recombination in Plants, Paszkowski (ed.), pp. 335-348, Kluwer Academic, Netherlands (1994)).<br>
It is understood that one or more of the nucleic acids of the present invention may be introduced into a plant cell and transcribed using an appropriate promoter with such transcription resulting in the cosuppression of an endogenous protein.<br>
Antisense approaches are a way of preventing or reducing gene function by targeting the genetic material (Mol et al., FEBS Lett., 268:427-430 (1990)). The objective of the antisense approach is to use a sequence complementary to the target gene to block its expression and create a mutant cell line or organism in which the level of a single chosen protein is selectively reduced or abolished. Antisense techniques have several advantages over other "reverse genetic" approaches. The site of inactivation and its developmental effect can be manipulated by the choice of promoter for antisense genes or by the timing of external application or microinjection. Antisense can manipulate its specificity by selecting either unique regions of the target gene or regions where it shares homology to other related genes (Hiatte et al., In: Genetic Engineering, Setlow (ed.), Vol. 11, New York: Plenum 49-63 (1989)).<br>
Antisense RNA techniques involve introduction of RNA that is complementary to the target mRNA into cells, which results in specific RNA:RNA duplexes being formed by base pairing between the antisense substrate and the target mRNA (Green et al, Annu. Rev. Biochem., 55:569-597 (1986)). Under one embodiment, the process involves the introduction and expression of an antisense gene sequence. Such a sequence is one in which part or all of the normal gene sequences are placed under a promoter in inverted orientation so that the "wrong" or complementary strand is transcribed into a noncoding antisense RNA that hybridizes with the target mRNA and interferes with its expression (Takayama and Inouye, Crit.Rev. Biochem. Mol Biol, 25:155-184 (1990)). An antisense vectoris constructed by standard procedures and introduced into cells by transformation, transfection, electroporation, microinjection, infection, etc. The type of transformation and choice of vector will determine<br>
whether expression is transient or stable. The promoter used for the antisense gene may influence the level, timing, tissue, specificity, or inducibility of the antisense inhibition.<br>
It is understood that the activity of a protein in a plant cell may be reduced or depressed by growing a transformed plant cell containing a nucleic acid molecule whose non-transcribed strand encodes a protein or fragment thereof. A preferred protein whose activity can be reduced or depressed, by any method, is a homogentisate prenyl transferase.<br>
Postrranscriptional gene silencing (PTGS) can result in virus immunity or gene silencing in plants. PTGS is induced by dsRNA and is mediated by an RNA-dependent RNA polymerase, present in the cytoplasm, which requires a dsRNA template. The dsRNA is formed by hybridization of complementary transgene mRNAs or complementary regions of the same transcript Duplex formation can be accomplished by using transcripts from one sense gene and one antisense gene colocated in the plant genome, a single transcript that has self-complementarity, or sense and antisense transcripts from genes brought together by crossing. The dsRNA-dependent RNA polymerase makes a complementary strand from the transgene mRNA and RNAse molecules attach to this complementary strand (cRNA). These cRNA-RNase molecules hybridize to the endogene mRNA and cleave the single-stranded RNA adjacent to the hybrid. The cleaved single-stranded RNAs are further degraded by other host RNases because one will lack a capped 51 end and the other will lack a poly (A) tail (Waterhouse et al., PNAS, 95:13959-13964 (1998)).<br>
It is understood that one or more of the nucleic acids of the present invention may be introduced into a plant cell and transcribed using an appropriate promoter with such transcription resulting in the postrranscriptional gene silencing of an endogenous transcript<br>
Antibodies have been expressed in plants (Hiatt et al., Nature, 342:76-78 (1989); Conrad and Fielder, Plant Mol. Biol, 26:1023-1030 (1994)). Cytoplasmic expression of a scFv (single-chain Fv antibody) has been reported to delay infection by artichoke mottled crinkle virus. Transgenic plants that express antibodies directed against endogenous proteins may exhibit a physiological effect (Philips et al., EMBOJ., 16:4489-4496 (1997); Marion-Poll, Trends in Plant Science, 2:447-448 (1997)). For example, expressed anti-abscisic antibodies have been reported to result in a general perturbation of seed development (Philips et al., EMBO J., 16:4489-4496 (1997)).<br>
Antibodies that are catalytic may also be expressed in plants (abzymes). The principle behind abzymes is that since antibodies may be raised against many molecules, this<br>
recognition ability can be directed toward generating antibodies that bind transition states to force a chemical reaction forward (Persidas, Nature Biotechnology, 15:1313-1315 (1997); Baca et al., Ann. Rev. Biophys. Biomol. Struct., 26:461-493 (1997)). The catalytic abilities of abzymes may be enhanced by site directed mutagenesis. Examples of abzymes are, for example, set forth in U.S. Patents: 5,658,753; 5,632,990; 5,631,137; 5,602,015; 5,559,538; 5,576,174; 5,500,358; 5,318,897; 5,298,409; 5,258,289; and 5,194,585.<br>
It is understood that any of the antibodies of the present invention may be expressed in plants and mat such expression can result in a physiological effect It is also understood that any of the expressed antibodies may be catalytic.<br>
The present invention also provides for parts of the plants, particularly reproductive or storage parts, of the present invention. Plant parts, without limitation, include seed, endosperm, ovule and pollen. In a particularly preferred embodiment of the present invention, the plant part is a seed. In one embodiment the seed is a constituent of animal feed.<br>
In another embodiment, the plant part is a fruit, more preferably a fruit with enhanced shelf life. In another preferred embodiment, the fruit has increased levels of a tocopherol. In another preferred embodiment, the fruit has increased levels of a tocotrienol.<br>
The present invention also provides a container of over about 10,000, more preferably<br>
about 20,000, and even more preferably about 40,000 seeds where over about 10%, more<br>
preferably 25%, more preferably 50%, and even more preferably 75% or 90% of the seeds are<br>
seeds derived from a plant of the present invention.	<br>
The present invention also provides a container of over about 10 kg, more preferably 25 kg, and even more preferably 50 kg seeds where over about 10%, more preferably 25%, more preferably 50%, and even more preferably 75% or 90% of the seeds are seeds derived from a plant of the present invention.<br>
Any of the plants or parts thereof of the present invention may be processed to produce a feed, meal, protein, or oil preparation, including oil preparations high in total tocopherol content and oil preparations high in any one or more of each tocopherol component listed herein. A particularly preferred plant part for this purpose is a seed. In a preferred embodiment the feed, meal, protein or oil preparation is designed for livestock animals or humans, or both. Methods to produce feed, meal, protein and oil preparations are known in the art See, for example, U.S. Patents 4,957,748; 5,100,679; 5,219,596; 5,936,069; 6,005,076; 6,146,669; and 6,156,227. In a preferred embodiment, the protein preparation is a<br>
high protein preparation. Such a high protein preparation preferably has a protein content of greater than about 5% w/v, more preferably 10% w/v, and even more preferably 15% w/v. In a preferred oil preparation, the oil preparation is a high oil preparation with an oil content derived from a plant or part thereof of the present invention of greater than about 5% w/v, more preferably 10% w/v, and even more preferably 15% w/v. In a preferred embodiment the oil preparation is a liquid and of a volume greater than about 1,5,10, or 50 liters. The present invention provides for oil produced from plants of the present invention or generated by a method of the present invention. Such an oil may exhibit enhanced oxidative stability. Also, such oil may be a minor or major component of any resultant product. Moreover, such oil may be blended with other oils. In a preferred embodiment, the oil produced from plants of the present invention or generated by a method of the present invention constitutes greater than about 0.5%, 1%, 5%, 10%, 25%, 50%, 75%, or 90% by volume or weight of the oil component of any product In another embodiment, the oil preparation may be blended and can constitute greater than about 10%, 25%, 35%, 50%, or 75% of the blend by volume. Oil produced from a plant of the present invention can be admixed with one or more organic solvents or petroleum distillates.<br>
Plants of the present invention can be part of or generated from a breeding program. The choice of breeding method depends on the mode of plant reproduction, the heritability of the trait(s) being improved, and the type of cultivar used commercially (e.g., FI hybrid cultivar, pureline cultivar, etc.). Selected, non-limiting approaches, for breeding the plants of the present invention are set forth below. A breeding program can be enhanced using marker assisted selection of the progeny of any cross. It is further understood that any commercial and non-commercial cultivars can be utilized in a breeding program. Factors such as, for example, emergence vigor, vegetative vigor, stress tolerance, disease resistance, branching, flowering, seed set, seed size, seed density, standability, and threshability etc. will generally dictate the choice.<br>
For highly heritable traits, a choice of superior individual plants evaluated at a single location will be effective, whereas for traits with low heritability, selection should be based on mean values obtained from replicated evaluations of families of related plants. Popular selection methods commonly include pedigree selection, modified pedigree selection, mass selection, and recurrent selection. In a preferred embodiment a backcross or recurrent breeding program is undertaken.<br>
The complexity of inheritance influences choice of the breeding method. Backcross breeding can be used to transfer one or a few favorable genes for a highly heritable trait into a desirable cultivar. This approach has been used extensively for breeding disease-resistant cultivars. Various recurrent selection techniques are used to improve quantitatively inherited traits controlled by numerous genes. The use of recurrent selection in self-pollinating crops depends on the ease of pollination, the frequency of successful hybrids from each pollination, and the number of hybrid offspring from each successful cross.<br>
Breeding lines can be tested and compared to appropriate standards in environments representative of the commercial target area(s) for two or more generations. The best lines are candidates for new commercial cultivars; those still deficient in traits may be used as parents to produce new populations for further selection.<br>
One method of identifying a superior plant is to observe its performance relative to other experimental plants and to a widely grown standard cultivar. If a single observation is inconclusive, replicated observations can provide a better estimate of its genetic worth. A breeder can select and cross two or more parental lines, followed by repeated selfing and selection, producing many new genetic combinations.<br>
The development of new cultivars requires the development and selection of varieties, the crossing of these varieties and the selection of superior hybrid crosses. The hybrid seed can be produced by manual crosses between selected male-fertile parents or by using male sterility systems. Hybrids are selected for certain single gene traits such as pod color, flower color, seed yield, pubescence color, or herbicide resistance, which indicate that the seed is truly a hybrid. Additional data on parental lines, as well as the phenotype of the hybrid, influence the breeder's decision whether to continue with the specific hybrid cross.<br>
Pedigree breeding and recurrent selection breeding methods can be used to develop cultivars from breeding populations. Breeding programs combine desirable traits from two or more cultivars or various broad-based sources into breeding pools from which cultivars are developed by selfing and selection of desired phenotypes. New cultivars can be evaluated to determine which have commercial potential.<br>
Pedigree breeding is used commonly for the improvement of self-pollinating crops. Two parents who possess favorable, complementary traits are crossed to produce an F1. A F2 population is produced by selfing one or several F1's. Selection of me best individuals from the best families is carried out Replicated testing of families can begin in the F4 generation to<br>
improve the effectiveness of selection for traits with low heritability. At an advanced stage of inbreeding (i.e., F6 and F7), the best lines or mixtures of phenotypically similar lines are tested for potential release as new cultivars.<br>
Backcross breeding has been used to transfer genes for a simply inherited, highly heritable trait into a desirable homozygous cultivar or inbred line, which is the recurrent parent. The source of the trait to be transferred is called the donor parent. The resulting plant is expected to have the attributes of the recurrent parent {e.g., cultivar) and the desirable trait transferred from the donor parent. After the initial cross, individuals possessing the phenotype of the donor parent are selected and repeatedly crossed (backcrossed) to the recurrent parent. The resulting parent is expected to have the attributes of the recurrent parent {e.g., cultivar) and the desirable trait transferred from the donor parent.<br>
The single-seed descent procedure in the strict sense refers to planting a segregating population, harvesting a sample of one seed per plant, and using the one-seed sample to plant the next generation. When the population has been advanced from the F2 to the desired level of inbreeding, die plants from which lines are derived will each trace to different F2  individuals. The number of plants in a population declines each generation due to failure of some seeds to germinate or some plants to produce at least one seed. As a result, not all of the F2 plants originally sampled in the population will be represented by a progeny when generation advance is completed.<br>
In a multiple-seed procedure, breeders commonly harvest one or more pods from each plant in a population and thresh them together to form a bulk. Part of the bulk is used to plant the next generation and part is put in reserve. The procedure has been referred to as modified single-seed descent or the pod-bulk technique.<br>
The multiple-seed procedure has been used to save labor at harvest It is considerably faster to thresh pods with a machine than to remove one seed from each by hand for the single-seed procedure. The multiplerseed procedure also makes it possible to plant the same number of seeds of a population each generation of inbreeding.<br>
Descriptions of other breeding methods that are commonly used for different traits and crops can be found in one of several reference books (e.g., Fehr, Principles of Cultivar Development, Vol. 1, pp. 2-3 (1987)).<br>
A transgenic plant of the present invention may also be reproduced using apomixis. Apomixis is a genetically controlled method of reproduction in plants where the embryo is<br>
formed without union of an egg and a sperm. There are three basic types of apomictic reproduction: 1) apospory where the embryo develops from a chromosomally unreduced egg in an embryo sac derived from the nucleus; 2) diplospory where the embryo develops from an unreduced egg in an embryo sac derived from the megaspore mother cell; and 3) adventitious embryony where the embryo develops directly from a somatic cell. In most forms of apomixis, pseudogamy, or fertilization of the polar nuclei to produce endosperm is necessary for seed viability. In apospory, a nurse cultivar can be used as a pollen source for endosperm formation in seeds. The nurse cultivar does not affect the genetics of the aposporous apomictic cultivar since the unreduced egg of the cultivar develops parthenogeneticalry, but makes possible endosperm production. Apomixis is economically important, especially in transgenic plants, because it causes any genotype, no matter how heterozygous, to breed true. Thus, with apomictic reproduction, heterozygous transgenic plants can maintain their generic fidelity throughout repeated life cycles. Methods for the production of apomictic plants are known in the art. See, U.S. Patent 5,811,636.<br>
OTHER ORGANISMS<br>
A nucleic acid of the present invention may be introduced into any cell or organism such as a mammalian cell, mammal, fish cell, fish, bird cell, bird, algae cell, algae, fungal cell, fungi, or bacterial cell. A protein of the present invention may be produced in an appropriate cell or organism. Preferred host and transformants include: fungal cells such as Aspergillus, yeasts, mammals, particularly bovine and porcine, insects, bacteria, and algae. Particularly preferred bacteria are Agrobacteruim tumefaciens and E. coli.<br>
 Methods to transform such cells or organisms are known in the art (EP 0 238 023; Yelton et al., Proc. Natl Acad. Set (U.S.A), 81:1470-1474 (1984); Malardier et al., Gene, 78:147-156 (1989); Becker and Guarente, In: Abelson and Simon (eds.), Guide to Yeast Genetics and Molecular Biology, Method Enzymol, Vol. 194, pp. 182-187, Academic Press, Inc., NY; Ito et al., J. Bacteriology, 153:163 (1983); Hinnen et al., Proc. Natl. Acad Sci. (USA.), 75:1920 (1978); Bennett and LaSure (eds.), More Gene Manipualtionins in fungi, Academic Press, CA (1991)). Methods to produce proteins of the present invention are also known (Kudla et al., EMBO, 9:1355-1364 (1990); Jarai and Buxton, Current Genetics, 26:2238-2244 (1994); Verdier, Yeast, 6:271-297 (1990); MacKenzie et al., Journal of Gen. Microbiol, 139:2295-2307 (1993); Hartl et al., TIBS, 19:20-25 (1994); Bergenron et al, TIBS, 19:124-128 (1994); Demolder et al., J. Biotechnology, 32:179-189 (1994); Craig,<br>
Science, 260:1902-1903 (1993); Gething and Sambrook, Nature, 355:33-45 (1992); Puig and Gilbert, J., Biol. Chem., 269:7764-7771 (1994); Wang and Tsou, FASEB Journal, 7:1515-1517 (1993); Robinson et al, Bio/Techiology, 1:381-384 (1994); Enderlin and Ogrydziak, Yeast, 10:67-79 (1994); Fuller et al, Proc. Natl. Acad. Sci. (U.S.A.), 86:1434-1438 (1989); Julius et al., Cell, 37:1075-1089 (1984); Julius etal, Cell, 32:839-852 (1983)).<br>
In a preferred embodiment, overexpression of a protein or fragment thereof of the present invention in a cell or organism provides in that cell or organism, relative to an untransformed cell or organism with a similar genetic background, an increased level of tocopherols.<br>
In a preferred embodiment, overexpression of a protein or fragment thereof of the present invention in a cell or organism provides in mat cell or organism, relative to an untransformed cell or organism with a similar genetic background, an increased level of α-tocopherols.<br>
In a preferred embodiment, overexpression of a protein or fragment thereof of the present invention in a cell or organism provides in that cell or organism, relative to an untransformed cell or organism with a similar genetic background, an increased level of -tocopherols.<br>
In another preferred embodiment, overexpression of a protein or fragment thereof of the present invention in a cell or organism provides in that cell or organism, relative to an untransformed cell or organism with a similar genetic background, an increased level of α-tocotrienols.<br>
In another preferred embodiment, overexpression of a protein or fragment thereof of the present invention in a cell or organism provides in that cell or organism, relative to an untransformed cell or organism with a similar genetic background, an increased level of -tocotrienols.<br>
ANTIBODIES<br>
One aspect of the present invention concerns antibodies, single-chain antigen binding molecules, or other proteins that specifically bind to one or more of the protein or peptide molecules of the present invention and their homologs, fusions or fragments. In a particularly preferred embodiment, the antibody specifically binds to a protein having me amino acid sequence set forth in SEQ ID NOs: 5,9-11,43-44,57-58, and 90, or fragments thereof.<br>
Antibodies of the present invention may be used to quantitatively or qualitatively detect the protein or peptide molecules of the present invention, or to detect post translational modifications of the proteins. As used herein, an antibody or peptide is said to "specifically bind" to a protein or peptide molecule of the present invention if such binding is not competitively inhibited by the presence of non-related molecules.<br>
Nucleic acid molecules that encode all or part of the protein of the present invention can be expressed, via recombinant means, to yield protein or peptides that can in turn be used to elicit antibodies that are capable of binding the expressed protein or peptide. Such antibodies may be used in immunoassays for that protein. Such protein-encoding molecules, or their fragments may be a "fusion" molecule (i.e., a part of a larger nucleic acid molecule) such that, upon expression, a fusion protein is produced. It is understood that any of the nucleic acid molecules of the present invention may be expressed, via recombinant means, to yield proteins or peptides encoded by these nucleic acid molecules.<br>
The antibodies that specifically bind proteins and protein fragments of the present invention may be polyclonal or monoclonal and may comprise intact immunoglobulins, or antigen binding portions of immunoglobulins fragments (such as (F(ab'), F(ab')2)), or single-chain immunoglobulins producible, for example, via recombinant means. It is understood that practitioners are familiar with the standard resource materials that describe specific conditions and procedures for the construction, manipulation and isolation of antibodies (see, for example, Harlow and Lane, In: Antibodies: A Laboratory Manual, Cold Spring Harbor Press, Cold Spring Harbor, NY (1988)).<br>
As discussed below, such antibody molecules or their fragments may be used for diagnostic purposes. Where the antibodies are intended for diagnostic purposes, it may be desirable to derivatize them, for example with a ligand group (such as biotin) or a detectable marker group (such as a fluorescent group, a radioisotope or an enzyme).<br>
The ability to produce antibodies mat bind the protein or peptide molecules of the present invention permits the identification of mimetic compounds derived from those molecules. These mimetic compounds may contain a fragment of the protein or peptide or merely a structurally similar region and nonetheless exhibits an ability to specifically bind to antibodies directed against that compound.<br>
EXEMPLARY USES<br>
Nucleic acid molecules and fragments thereof of the present invention may be employed to obtain other nucleic acid molecules from the same species (nucleic acid molecules from com may be utilized to obtain other nucleic acid molecules from corn). Such nucleic acid molecules include the nucleic acid molecules that encode the complete coding sequence of a protein and promoters and flanking sequences of such molecules. In addition, such nucleic acid molecules include nucleic acid molecules that encode for other isozymes or gene family members. Such molecules can be readily obtained by using the above-described nucleic acid molecules or fragments thereof to screen cDNA or genomic libraries. Methods for forming such libraries are well known in the art<br>
Nucleic acid molecules and fragments thereof of the present invention may also be employed to obtain nucleic acid homologs. Such homologs include the nucleic acid molecules of plants and other organisms, including bacteria and fungi, including the nucleic acid molecules that encode, in whole or in part, protein homologues of other plant species or other organisms, sequences of genetic elements, such as promoters and transcriptional regulatory elements. Such molecules can be readily obtained by using the above-described nucleic acid molecules or fragments thereof to screen cDNA or genomic libraries obtained from such plant species. Methods for forming such libraries are well known in the art. Such homolog molecules may differ in their nucleotide sequences from those coding for one or more of SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof because complete complementarity is not needed for stable hybridization. The nucleic acid molecules of the present invention therefore also include molecules that, although capable of specifically hybridizing with the nucleic acid molecules may lack "complete complementarity".<br>
Any of a variety of methods may be used to obtain one or more of the above-described nucleic acid molecules (Zamechik et al., Proc. Natl Acad. Sci (USA.), 83:4143-4146 (1986); Goodchild et al., Proc Natl Acad. Sci. (U.SA), 85:5507-5511 (1988); Wickstrom et al, Proc. Natl. Acad. Sci. (U.SA.), 85:1028-1032 (1988); Holt et al, Molec. Cell Biol, 8:963-973 (1988); Gerwirtz et al., Science, 242:1303-1306 (1988); Anfossi et al., Proc. Natl Acad Sci. (U.SA), 86:3379-3383 (1989); Becker et al., EMBO J., 8:3685-3691 (1989)). Automated nucleic acid synthesizers may be employed for this purpose. In lieu of such synthesis, the disclosed nucleic acid molecules may be used to define a pair of primers that<br>
can be used with the polymerase chain reaction (Mullis et al, Cold Spring Harbor Symp. Quant. Biol, 51:263-273 (1986); Erlich et al., EP 50 424; EP 84 796; EP 258 017; EP 237 362; Mullis, EP 201 184; Mullis et al, U.S. Patent 4,683,202; Erlich, U.S. Patent 4,582,788; and Saiki et al, U.S. Patent 4,683,194) to amplify and obtain any desired nucleic acid molecule or fragment.<br>
Promoter sequences and other genetic elements, including but not limited to transcriptional regulatory flanking sequences, associated with one or more of the disclosed nucleic acid sequences can also be obtained using the disclosed nucleic acid sequence provided herein. In one embodiment, such sequences are obtained by incubating nucleic acid molecules of the present invention with members of genomic libraries and recovering clones that hybridize to such nucleic acid molecules thereof. In a second embodiment, methods of "chromosome walking", or inverse PCR may be used to obtain such sequences (Frohman et al, Proc. Natl Acad Scl (U.SA.), 85:8998-9002 (1988); Ohara et al., Proc. Natl Acad. Sci (U.SA.), 86:5673-5677 (1989); Pang et al., Biotechniques, 22:1046-1048 (1977); Huang et al., Methods Mol Biol, 69:89-96 (1997); Huang et al., Method Mol Biol, 67:287-294 (1997); Benkel et al., Genet. Anal., 13:123-127 (1996); Haiti et al., Methods Mol. Biol, 58:293-301 (1996)). The term "chromosome walking" means a process of extending a genetic map by successive hybridization steps.<br>
The nucleic acid molecules of the present invention may be used to isolate promoters of cell enhanced, cell specific, tissue enhanced, tissue specific, developmentally or environmentally regulated expression profiles. Isolation and functional analysis of the 5' flanking promoter sequences of these genes from genomic libraries, for example, using genomic screening methods and PCR techniques would result in the isolation of useful promoters and transcriptional regulatory elements. These methods are known to those of skill in the art and have been described (see, for example, Birren et al., Genome Analysis: Analyzing DNA, 1, (1997), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY). Promoters obtained utilizing the nucleic acid molecules of the present invention could also be modified to affect their control characteristics. Examples of such modifications would include but are not limited to enhancer sequences. Such genetic elements could be used to enhance gene expression of new and existing traits for crop improvement,<br>
Another subset of the nucleic acid molecules of the present invention includes nucleic acid molecules that are markers. The markers can be used in a number of conventional ways<br>
in the field of molecular genetics. Such markers include nucleic acid molecules encoding SEQ ID NOs: 5,9-11,43-44,57-58, and 90, and complements thereof, and fragments of either that can act as markers and other nucleic acid molecules of the present invention that can act as markers.<br>
Genetic markers of the present invention include "dominant" or "codominant" markers. "Codominant markers" reveal the presence of two or more alleles (two per diploid individual) at a locus. "Dominant markers" reveal the presence of only a single allele per locus. The presence of the dominant marker phenotype (e.g., a band of DNA) is an indication that one allele is in either the homozygous or heterozygous condition. The absence of the dominant marker phenotype (e.g., absence of a DNA band) is merely evidence that "some other" undefined allele is present. In the case of populations where individuals are predominantly homozygous and loci are predominately dimorphic, dominant and codominant markers can be equally valuable. As populations become more heterozygous and multi-allelic, codominant markers often become more informative of the genotype than dominant markers. Marker molecules can be, for example, capable of detecting polymorphisms such as single nucleotide polymorphisms (SNPs).<br>
The genomes of animals and plants naturally undergo spontaneous mutation in the course of their continuing evolution (Gusella, Ann Rev. Biochem., 55:831-854 (1986)). A "polymorphism" is a variation or difference in the sequence of the gene or its flanking regions that arises in some of the members of a species. The variant sequence and the "original" sequence co-exist in the species' population. In some instances, such co-existence is in stable or quasi-stable equilibrium.<br>
A polymorphism is thus said to be "allelic", in that, due to the existence of the  . polymorphism, some members of a population may have the original sequence (i.e., the original "allele") whereas other members may have the variant sequence (i.e., the variant "allele"). In the simplest case, only one variant sequence may exist and the polymorphism is thus said to be di-allelic. In other cases, the species' population may contain multiple alleles and the polymorphism is termed tri-allelic, etc. A single gene may have multiple different unrelated polymorphisms. For example, it may have a di-allelic polymorphism at one site and a multi-allelic polymorphism at another site.<br>
The variation that defines the polymorphism may range from a single nucleotide variation to the insertion or deletion of extended regions within a gene. In some cases, the<br>
DNA sequence variations are in regions of the genome that are characterized by short tandem repeats (STRs) that include tandem di- or tri-nucleotide repeated motifs of nucleotides. Polymorphisms characterized by such tandem repeats are referred to as "variable number tandem repeat" ("VNTR") polymorphisms. VNTRs have been used in identity analysis (Weber, U.S. Patent 5,075,217; Armour et al., FEBS Lett., 307:113-115 (1992); Jones et al., Eur. J. Haematol, 39:144-147 (1987); Horn et al., PCT Application WO 91/14003; Jeffreys, EP 370 719; Jeffreys, U.S. Patent 5,175,082; Jeffreys et al.,Amer. J. Hunt. Genet., 39:11-24 (1986); Jeffreys et al.. Nature, 316:76-79 (1985); Gray et al, Proc. R. Acad. Soc Land, 243:241-253 (1991); Moore et al., Genomics, 10:654-660 (1991); Jeffreys et al.,Anim. Genet., 18:1-15 (1987); Hillel et al., Anim. Genet., 20:145-155 (1989); Hillel et al., Genet, 124:783-789 (1990)).<br>
The detection of polymorphic sites in a sample of DNA may be facilitated through the use of nucleic acid amplification methods. Such methods specifically increase the concentration of polynucleotides that span the polymorphic site, or include that site and sequences located either distal or proximal to it. Such amplified molecules can be readily detected by gel electrophoresis or other means.<br>
In an alternative embodiment, such polymorphisms can be detected through the use of a marker nucleic acid molecule that is physically linked to such polymorphism(s). For this purpose, marker nucleic acid molecules comprising a nucleotide sequence of a polynucleotide located within 1 mb of the polymorphism(s) and more preferably within 100kb of the polymorphism(s) and most preferably within 10kb of the polymorphism(s) can be employed.<br>
The identification of a polymorphism can be determined in a variety of ways. By correlating the presence or absence of it in a plant with the presence or absence of a phenotype, it is possible to predict the phenotype of that plant Ji a polymorphism creates or destroys a restriction endonuclease cleavage site, or if it results in the loss or insertion of DNA (e.g., a VNTR polymorphism), it will alter the size or profile of the DNA fragments that are generated by digestion with that restriction endonuclease. As such, organisms that possess a variant sequence can be distinguished from those having the original sequence by restriction fragment analysis. Polymorphisms that can be identified in this manner are termed "restriction fragment length polymorphisms" (RFLPs) (Glassberg, UK Patent Application 2135774; Skolnick et al., Cytogen. Cell Genet., 32:58-67 (1982); Botstein et al, Ann. J. Hum.<br>
Genet., 32:314-331 (1980); Fischer et al., PCT Application WO 90/13668; Uhlen, PCT Application WO 90/11369).<br>
Polymorphisms can also be identified by Single Strand Conformation Polymorphism (SSCP) analysis (Elles, Methods in Molecular Medicine: Molecular Diagnosis of Genetic Diseases, Humana Press (1996)); Orita et al., Genomics, 5:874-879 (1989)). A number of protocols have been described for SSCP including, but not limited to, Lee et al., Anal. Biochem., 205:289-293 (1992); Suzuki et al, Anal. Biochem., 192:82-84 (1991); Lo et al.. Nucleic Acids Research, 20:1005-1009 (1992); Sarkar et al., Genomics, 13:441-443 (1992). It is understood that one or more of the nucleic acids of the present invention, may be utilized as markers or probes to detect polymorphisms by SSCP analysis.<br>
Polymorphisms may also be found using a DNA fingerprinting technique called amplified fragment length polymorphism (AFLP), which is based on the selective PCR amplification of restriction fragments from a total digest of genomic DNA to profile that DNA (Vos et al, Nucleic Acids Res., 23:4407-4414 (1995)). This method allows for the specific co-amplification of high numbers of restriction fragments, which can be visualized by PCR without knowledge of the nucleic acid sequence. It is understood that one or more of the nucleic acids of the present invention may be utilized as markers or probes to detect polymorphisms by AFLP analysis or for fingerprinting RNA.<br>
Polymorphisms may also be found using random amplified polymorphic DNA (RAPD) (Williams et al, Nucl. Acids Res., 18:6531-6535 (1990)) and cleaveable amplified polymorphic sequences (CAPS) (Lyamichev et al., Science, 260:778-783 (1993)). It is understood mat one or more of the nucleic acid molecules of the present invention, may be utilized as markers or probes to detect polymorphisms by RAPD or CAPS analysis.<br>
Single Nucleotide Polymorphisms (SNPs) generally occur at greater frequency than other polymorphic markers and are spaced with a greater uniformity throughout a genome than other reported forms of polymorphism. The greater frequency and uniformity of SNPs means mat there is greater probability that such a polymorphism will be found near or in a genetic locus of interest than would be the case for other polymorphisms. SNPs are located in protein-coding regions and noncoding regions of a genome. Some of these SNPs may result in defective or variant protein expression {e.g.., as a result of mutations or defective splicing). Analysis (genotyping) of characterized SNPs can require only a plus/minus assay rather than a lengthy measurement, permitting easier automation.<br>
SNPs can be characterized using any of a variety of methods. Such methods include the direct or indirect sequencing of the site, the use of restriction enzymes (Botstein et al, Am. J. Hum. Genet., 32:314-331 (1980); Konieczny and Ausubel, Plant J., 4:403-410 (1993)), enzymatic and chemical mismatch assays (Myers et al., Nature, 313:495-498 (1985)), allele-specific PCR (Newton et al, Nucl. Acids Res., 17:2503-2516 (1989); Wu et al, Proc. Natl Acad. Sci. (U.S.A), 86:2757-2760 (1989)), ligase chain reaction (Barany, Proc. Natl Acad. Sci. (U.SA.), 88:189-193 (1991)), single-strand conformation polymorphism analysis (Labmnc et al., Am. J. Hum. Genet., 48:1115-1120 (1991)), single base primer extension (Kuppuswamy et al., Proc. Natl Acad Sci. (U.SA.), 88:1143-1147 (1991), Goelet, U.S. Patent 6,004,744; Goelet, U.S. Patent 5,888,819), solid-phase EUSA-based oligonucleotide ligation assays (Nikiforov et al., Nucl. Acids Res., 22:4167-4175 (1994)), dideoxy fingerprinting (Sarkar et al., Genomics, 13:441-443 (1992)), oligonucleotide fluorescence-quenching assays (Livak et al, PCR Meiliods Appl, 4:357-362 (1995a)), 5'-nuclease allele-specific hybridization TaqMan™ assay (livak et al, Nature Genet., 9:341-342 (1995)), template-directed dye-terminator incorporation (TDI) assay (Chen and Kwok, Nucl Acids Res., 25:347-353 (1997)), allele-specific molecular beacon assay (Tyagi et al., Nature Biotech., 16:49-53 (1998)), Pinpoint assay (Haff and Smirnov, Genome Res., 7:378-388 (1997)), dCAPS analysis (Neff et al, Plant J., 14:387-392 (1998)), pyrosequencing (Ronaghi et al., Analytical Biochemistry, 267:65-71 (1999); Ronaghi et al., WO 98/13523; Nyren etal., WO 98/28440; www.pyrosequencing.com), using mass spectrometry, e.g. the Masscode™ system (Howbert et al, WO 99/05319; Howbert et al., WO 97/27331; www.apigene.com; Becker et al., WO 98/26095; Becker et al., WO 98/12355; Becker et al, WO 97/33000; Monforte et al, U.S. Patent 5,965,363), invasive cleavage of oligonucleotide probes (Lyamichev et al., Nature Biotechnology, 17:292-2%; www.tw.tcom), and using high density oligonucleotide rrays (Hacia et al., Nature Genetics, 22:164-167; www.affymetrix.com).<br>
Polymorphisms may also be detected using allele-specific oligonucleotides (ASO), which, can be for example, used in combination with hybridization based technology including Southern, Northern, and dot blot hybridizations, reverse dot blot hybridizations and hybridizations performed on microarray and related technology.<br>
The stringency of hybridization for polymorphism detection is highly dependent upon a variety of factors, including length of the allele-specific oligonucleotide, sequence composition, degree of complementarity (i.e., presence or absence of base mismatches),<br>
concentration of salts and other factors such as formamide and temperature. These factors are important both during the hybridization itself and during subsequent washes performed to remove target polynucleotide that is not specifically hybridized. Li practice, the conditions of the final, most stringent wash are most critical. In addition, the amount of target polynucleotide that is able to hybridize to the allele-specific oligonucleotide is also governed by such factors as the concentration of both the ASO and the target polynucleotide, the presence and concentration of factors that act to "tie up" water molecules, so as to effectively concentrate the reagents (e.g., PEG, dextran, dextran sulfate, etc.), whether the nucleic acids are immobilized or in solution, and the duration of hybridization and washing steps.<br>
Hybridizations are preferably performed below the melting temperature (Tm) of the ASO. The closer the hybridization and/or washing step is to the Tm, the higher the stringency. Tm for an oligonucleotide may be approximated, for example, according to the following formula: Tm = 81.5 + 16.6 x (log10[Na+]) + 0.41 x (%G+C) - 675/n; where [Na+] is the molar salt concentration of Na+ or any other suitable cation and n = number of bases in the oligonucleotide. Other formulas for approximating Tm are available and are known to those of ordinary skill in the art.<br>
Stringency is preferably adjusted so as to allow a given ASO to differentially hybridize to a target polynucleotide of uie correct allele and a target polynucleotide of the incorrect allele. Preferably, there will be at least a two-fold differential between the signal produced by the ASO hybridizing to a target polynucleotide of the correct allele and the level of the signal produced by the ASO cross-hybridizing to a target polynucleotide of the incorrect allele (e.g., an ASO specific for a mutant allele cross-hybridizing to a wild-type allele). In more preferred embodiments of the present invention, there is at least a five-fold signal differential. In highly preferred embodiments of the present invention, there is at least an order of magnitude signal differential between the ASO hybridizing to a target polynucleotide of the correct allele and the level of the signal produced by the ASO cross-hybridizing to a target polynucleotide of the incorrect allele.<br>
While certain methods for detecting polymorphisms are described herein, other detection methodologies may be utilized. For example, additional methodologies are known and set forth, in Birren et al., Genome Analysis, 4:135-186; A Laboratory Manual. Mapping Genomes, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1999); Maliga et al., Methods in Plant Molecular Biology. A Laboratory Course Manual, Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, NY (1995); Paterson, Biotechnology Intelligence Unit: Genome Mapping in Plants, R.G. Landes Co., Georgetown, TX, and Academic Press, San Diego, CA (1996); The Com Handbook, Freeling and Walbot, (eds.), Springer-Verlag, New York, NY (1994); Methods in Molecular Medicine: Molecular Diagnosis of Genetic Diseases, Elles, (ed.), Humana Press, Totowa, NJ (1996); Clark, (ed.), Plant Molecular Biology: A Laboratory Manual, Springer-Verlag, Berlin, Germany (1997).<br>
Factors for marker-assisted selection in a plant breeding program are: (1) the markers) should co-segregate or be closely linked with the desired trait; (2) an efficient means of screening large populations for the molecular marker(s) should be available; and (3) the screening technique should have high reproducibility across laboratories and preferably be economical to use and be user-friendly.<br>
The genetic linkage of marker molecules can be established by a gene mapping model such as, without limitation, the flanking marker model reported by Lander and Botstein, Genetics, 121:185-199 (1989) and the interval mapping, based on maximum likelihood methods described by Lander and Botstein, Genetics, 121:185-199 (1989) and implemented in the software package MAPMAKER/QTL (Lincoln and Lander, Mapping Genes Controlling Quantitative Traits Using MAPMAKER/QTL, Whitehead Institute for Biomedical Research, MA (1990). Additional software includes Qgene, Version 2.23 (1996), Department of Plant Breeding and Biometry, 266 Emerson Hall, Cornell University, Ithaca, NY. Use of Qgene software is a particularly preferred approach.<br>
A maximum likelihood estimate (MLB) for the presence of a marker is calculated, together with an MLE assuming no QTL effect, to avoid false positives. A log10 of an odds ratio (LOD) is then calculated as: LOD = log10 (MLE for the presence of a QTL/MLE given no linked QTL).<br>
The LOD score essentially indicates how much more likely the data are to have arisen assuming the presence of a QTL than in its absence. The LOD threshold value for avoiding a false positive with a given confidence, say 95%, depends on the number of markers and the length of the genome. Graphs indicating LOD thresholds are set forth in Lander and Botstein, Genetics, 121:185-199 (1989) and further described by Arus and Moreno-Gonzalez, Plant Breeding, Hayward et al, (eds.) Chapman &amp; Hall, London, pp. 314-331 (1993).<br>
In a preferred embodiment of the present invention the nucleic acid marker exhibits a LOD score of greater than 2.0, more preferably 2.5, even more preferably greater than 3.0 or<br>
4.0 with the trait or phenotype of interest. In a preferred embodiment, the trait of interest is altered tocopherol levels or compositions or altered tocotrienol levels or compositions.<br>
Additional models can be used. Many modifications and alternative approaches to interval mapping have been reported, including the use of non-parametric methods (Kruglyak and Lander, Genetics, 139:1421-1428 (1995)). Multiple regression methods or models can also be used, in which the trait is regressed on a large number of markers (Jansen, Biometrics in Plant Breeding, van Oijen and Jansen (eds.), Proceedings of the Ninth Meeting of the Eucarpia Section Biometrics in Plant Breeding, The Netherlands, pp. 116-124 (1994); Weber and Wricke, Advcaices in Plant Breeding, Blackwell, Berlin, 16 (1994)). Procedures combining interval mapping with regression analysis, whereby the phenotype is regressed onto a single putative QTL at a given marker interval and at the same time onto a number of markers that serve as "cofactors", have been reported by Jansen and Stam, Genetics, 136:1447-1455 (1994); and Zeng, Genetics, 136:1457-1468 (1994). Generally, the use of cofactors reduces the bias and sampling error of the estimated QTL positions (Utz and Melchinger, Biometrics in Plant Breeding, van Oijen and Jansen (eds.), Proceedings of the Ninth Meeting of the Eucarpia Section Biometrics in Plant Breeding, The Netherlands, pp. 195-204 (1994), thereby improving the precision and efficiency of QTL mapping (Zeng, Genetics, 136:1457-1468 (1994)). These models can be extended to multi-environment experiments to analyze genotype-environment interactions (Jansen et at, Theo. Appl. Genet., 91:33-37 (1995)).<br>
It is understood that one or more of the nucleic acid molecules of the present invention may be used as molecular markers. It is also understood that one or more of the protein molecules of the present invention may be used as molecular markers.<br>
In a preferred embodiment, the polymorphism is present and screened for in a mapping population, e.g. a collection of plants capable of being used with markers such as polymorphic markers to map genetic position of traits. The choice of appropriate mapping population often depends on the type of marker systems employed (Tanksley et al., J.P. Gustafson and R. Appels (eds.). Plenum Press, NY, pp. 157-173 (1988)). Consideration must be given to the source of parents (adapted vs. exotic) used in the mapping population. Chromosome pairing and recombination rates can be severely disturbed (suppressed) in wide crosses (adapted x exotic) and generally yield greatly reduced linkage distances. Wide<br>
crosses will usually provide segregating populations with a relatively large number of polymorphisms when compared to progeny in a narrow cross (adapted x adapted).<br>
An F2 population is the first generation of selfing (self-pollinating) after the hybrid seed is produced. Usually a single F1 plant is selfed to generate a population segregating for all the genes in Mendelian (1:2:1) pattern. Maximum genetic information is obtained from a completely classified F2 population using a codominant marker system (Mather, Measurement of linkage in Heredity: Methuen and Co., (1938)). In the case of dominant markers, progeny tests (e.g., F3, BCF2) are required to identify the heterozygotes, in order to classify the population. However, this procedure is often prohibitive because of the cost and time involved in progeny testing. Progeny testing of F2 individuals is often used in map construction where phenotypes do not consistently reflect genotype (e.g„ disease resistance) or where trait expression is controlled by a QTL. Segregation data from progeny test populations (e.g., F3 or BCF2) can be used in map construction. Marker-assisted selection can then be applied to cross progeny based on marker-trait map associations (F2, F3), where linkage groups have not been completely disassociated by recombination events (i.e., maximum disequilibrium).<br>
Recombinant inbred lines (RIL) (genetically related lines; usually &gt;F5, developed from continuously selfing F2 lines towards homozygosity) can be used as a mapping population. Information obtained from dominant markers can be maximized by using RIL because all loci are homozygous or nearly so. Under conditions of tight linkage (i.e., about 
Backcross populations e.g., generated from a cross between a successful variety (recurrent parent) and another variety (donor parent) carrying a trait not present in the former) can be utilized as a mapping population. A series of backcrosses to the recurrent parent can be made to recover most of its desirable traits. Thus a population is created consisting of individuals nearly like the recurrent parent but each individual carries varying amounts or mosaic of genomic regions from the donor parent Backcross populations can be useful for mapping dominant markers if all loci in the recurrent parent are homozygous and the donor<br>
and recurrent parent have contrasting polymorphic marker alleles (Reiter et al, Proc. Natl. Acad. Sci. (U.S.A.), 89:1477-1481 (1992)). Information obtained from backcross populations using either codominant or dominant markers is less than that obtained from F2 populations because one, rather than two, recombinant gamete is sampled per plant. Backcross populations, however, are more informative (at low marker saturation) when compared to RDLs as the distance between linked loci increases in RIL populations (i.e., about 0.15% recombination). Increased recombination can be beneficial for resolution of tight linkages, but may be undesirable in the construction of maps with low marker saturation.<br>
Near-isogenic lines (NIL) (created by many backcrosses to produce a collection of individuals that is nearly identical in genetic composition except for the trait or genomic region under interrogation) can be used as a mapping population. In mapping with NILs, only a portion of the polymorphic loci is expected to map to a selected region.<br>
Bulk segregant analysis (BS A) is a method developed for the rapid identification of linkage between markers and traits of interest (Michelmore et al., Proc. Natl. Acad. Sci. (U.SA.), 88:9828-9832 (1991)). In BSA, two bulked DNA samples are drawn from a segregating population originating from a single cross. These bulks contain individuals that are identical for a particular trait (resistant or susceptible to particular disease) or genomic region but arbitrary at unlinked regions (i.e., heterozygous). Regions unlinked to the target region will not differ between the bulked samples of many individuals in BSA.<br>
In an aspect of the present invention, one or more of the nucleic molecules of the present invention are used to determine the level (i.e., the concentration of mRNA in a sample, etc.) in a plant (preferably canola, com, Brassica campestris, oilseed rape, rapeseed, soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, safflower, oil palm, flax or sunflower) or pattern (i.e., the kinetics of expression, rate of decomposition, stability profile, etc.) of the expression of a protein encoded in part or whole by one or more of the nucleic acid molecule of the present invention (collectively, the "Expression Response" of a cell or tissue).<br>
As used herein, the Expression Response manifested by a cell or tissue is said to be "altered" if it differs from the Expression Response of cells or tissues of plants not exhibiting the phenotype. To determine whether a Expression Response is altered, the Expression Response manifested by the cell or tissue of the plant exhibiting the phenotype is compared with that of a similar cell or tissue sample of a plant not exhibiting the phenotype. As will be<br>
appreciated, it is not necessary to re-determine the Expression Response of the cell or tissue sample of plants not exhibiting the phenotype each time such a comparison is made; rather, the Expression Response of a particular plant may be compared with previously obtained values of normal plants. As used herein, the phenotype of the organism is any of one or more characteristics of an organism (e.g., disease resistance, pest tolerance, environmental tolerance such as tolerance to abiotic stress, male sterility, quality improvement or yield etc.). A change in genotype or phenotype may be transient or permanent Also as used herein, a tissue sample is any sample that comprises more than one cell. In a preferred aspect, a tissue sample comprises cells that share a common characteristic (e.g., Derived from root, seed, flower, leaf, stem or pollen etc.).<br>
In one aspect of the present invention, an evaluation can be conducted to determine whether a particular mRNA molecule is present. One or more of the nucleic acid molecules of the present invention are utilized to detect the presence or quantity of the mRNA species. Such molecules are then incubated with cell or tissue extracts of a plant under conditions sufficient to permit nucleic acid hybridization. The detection of double-stranded probe-mRNA hybrid molecules is indicative of the presence of the mRNA; the amount of such hybrid formed is proportional to the amount of mRNA. Thus, such probes may be used to ascertain the level and extent of the mRNA production in a plant's cells or tissues. Such nucleic acid hybridization may be conducted under quantitative conditions (thereby providing a numerical value of the amount of the mRNA present). Alternatively, the assay may be   conducted as a qualitative assay that indicates either that the mRNA is present, or that its level exceeds a user set, predefined value.<br>
A number of methods can be used to compare the expression response between two or more samples of cells or tissue. These methods include hybridization assays, such as northerns, RNAse protection assays, and in situ hybridization. Alternatively, the methods include PCR-type assays. In a preferred method, the expression response is compared by hybridizing nucleic acids from the two or more samples to an array of nucleic acids. The array contains a plurality of suspected sequences known or suspected of being present in the cells or tissue of the samples.<br>
An advantage of in situ hybridization over more conventional techniques for the detection of nucleic acids is that it allows an investigator to determine the precise spatial population (Angerer et al., Dev. Biol, 101:477-484 (1984); Angerer et al., Dev. Biol,<br>
112:157-166 (1985); Dixon et al., EMBO J., 10:1317-1324 (1991)). In situ hybridization may be used to measure the steady-state level of RNA accumulation (Hardin et al, J. Mol. Biol, 202:417-431 (1989)). A number of protocols have been devised for in situ hybridization, each with tissue preparation, hybridization and washing conditions (Meyerowitz, Plant Mol. Biol. Rep., 5:242-250 (1987); Cox and Goldberg, Li: Plant Molecular Biology: A Practical Approach, Shaw (ed.), pp. 1-35, IRL Press, Oxford (1988); Raikhel et al., In situ RNA hybridization in plant tissues, In: Plant Molecular Biology Manual, Vol. B9:l-32, Kluwer Academic Publisher, Dordrecht, Belgium (1989)).<br>
In situ hybridization also allows for the localization of proteins within a tissue or cell (Wilkinson, In Situ Hybridization, Oxford University Press, Oxford (1992); Langdale, In Situ Hybridization In: The Corn Handbook, Reeling and Walbot (eds.), pp. 165-179, Springer-Verlag, NY (1994)). It is understood that one or more of the molecules of the present invention, preferably one or more of the nucleic acid molecules or fragments thereof of the present invention or one or more of the antibodies of the present invention may be utilized to detect the level or pattern of a protein or mRNA thereof by in situ hybridization.<br>
Fluorescent in situ hybridization allows the localization of a particular DNA sequence along a chromosome, which is useful, among other uses, for gene mapping, following chromosomes in hybrid lines, or detecting chromosomes with translocations, transversions or deletions. In situ hybridization has been used to identify chromosomes in several plant species (Griffor et al., Plant Mol. Biol, 17:101-109 (1991); Gustafson et al., Proc. Natl. Acad Sci. (USA.), 87:1899-1902 (1990); Mukai and Gill, Genome, 34:448-452 (1991); Schwarzacher and Heslop-Harrison, Genome, 34:317-323 (1991); Wang et al., Jpn. J. Genet., 66:313-316 (1991); Parra and Windle, Nature Genetics, 5:17-21 (1993)). It is understood that the nucleic acid molecules of the present invention may be used as probes or markers to localize sequences along a chromosome.<br>
Another method to localize the expression of a molecule is tissue printing. Tissue printing provides a way to screen, at the same time on the same membrane many tissue sections from different plants or different developmental stages (Yomo and Taylor, Planta, 112:35-43 (1973); Harris and Chrispeels, Plant Physiol, 56:292-299 (1975); Cassab and Varner, J. Cell Biol, 105:2581-2588 (1987); Spruce et al., Phytochemistry, 26:2901-2903 (1987); Barres.et al., Neuron, 5:527-544 (1990); Reid and Pont-Lezica, Tissue Printing: Tools for the Study of Anatomy, Histochemistry and Gene Expression, Academic Press, New York,<br>
NY (1992); Reid et al, Plant Physiol, 93:160-165 (1990); Ye et al., Plant J., 1:175-183 (1991)).<br>
One skilled in the art can refer to general reference texts for detailed descriptions of known techniques discussed herein or equivalent techniques. These texts include Current Protocols in Molecular Biology, Ausubel et al., (eds.), John Wiley &amp; Sons, NY (1989), and supplements through September (1998), Molecular Cloning, A Laboratory Manual, Sambrook et al., 2nd Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY (1989), Genome Analysis: A Laboratory Manual 1: Analyzing DNA, Birren et al, Cold Spring Harbor Press, Cold Spring Harbor, NY (1997); Genome Analysis: A Laboratory Manual 2: Detecting Genes, Birren et al., Cold Spring Harbor Press, Cold Spring Harbor, NY (1998); Genome Analysis: A Laboratory Manual 3: Cloning Systems, Birren et al., Cold Spring Harbor Press, Cold Spring Harbor, NY (1999); Genome Analysis: A Laboratory Manual 4: Mapping Genomes, Birren et al., Cold Spring Harbor Press, Cold Spring Harbor, NY (1999); Plant Molecular Biology: A Laboratory Manual, Clark, Springer-Verlag, Berlin, (1997); Methods in Plant Molecular Biology, Maliga et al., Cold Spring Harbor Press, Cold Spring Harbor, NY (1995). These texts can, of course, also be referred to in making or using an aspect of the present invention. It is understood that any of the agents of the present invention can be substantially purified and/or be biologically active and/or recombinant.<br>
Having now generally described the present invention, the same will be more readily understood through reference to the following examples that are provided by way of illustration, and are not intended to be limiting of the present invention, unless specified. EXAMPLE 1: Identification of homogentisate prenyl transferase Sequences This example sets forth methods used to analyze homogentisate prenyl transferase sequences from various sources in order to identify motifs common to homogentisate prenyl transferase that are contained therein.<br>
Homogentisate prenyl transferase sequences from Soy, Arabidopsis, Corn and Cuphea (partial) are cloned and sequenced from EST sequences found in an EST database. Synechocystis, Nostoc, and Anabaena are obtained from Genbank. These sequences (representing SEQ ID NOs: 1-8) are then aligned with respect to each other using the multiple alignment software ClustalX, which is described by Thompson et al, Nucleic Acids Research, 24:4876-4882 (1997). The multiple alignment of the protein sequences is visualized and edited using Genedoc, which is described by Nicholas et al, EMBNEW.NEWS, 4:14 (1997).<br>
Using the aforementioned multiple alignment tool, four motifs (A-D) are identified, as shown in Figures 2a-2c, wherein motifs A-D are set forth. These motifs are represented by SEQ ID NOs: 12-15. The Cuphea sequence is removed from motif D because the sequence had multiple errors towards the 3' end that generated apparent frame shift errors.<br>
The specificity of these motifs is demonstrated using a Hidden Markov Model (HMM) that is built using an HMMER(version 2.2g) software package (Eddy, Bioinformatics, 14:755-763 (1998)). A HMM search is performed on a cDNA sequence database containing full insert sequence from different plant species. This search identifies two new homogentisate prenyl transferase sequences (SEQ ID NOs: 9-10) in addition to several partial homogentisate prenyl transferase sequences. The two new homogentisate prenyl transferase sequences identified are from leek and wheat. This search also identifies a complete Cuphea sequence (SEQ ID NO: 11) with no errors. A second alignment is generated using the aforementioned multiple alignment tool, as shown in Figures 3a-3c. This alignment has the leek, wheat, and full Cuphea sequences incorporated. Motifs I-IV (SEQ ID NOs: 39-42) are shown.<br>
Specificity is also tested by using each motif sequence to search the non-redundant amino acid database downloaded from Genbank available through NCBL All four motifs identify three homogentisate prenyl transferase found in the aforementioned non-redundant amino acid database, as follows: Nostoc, Synechocystis, Arabidopsis. Motifs II and IV also identified some genomic variants of an uncharacterized Arabidopsis protein. Motifs I and III only identified known homogentisate prenyl transferase at an E value of 0.001 or lower. EXAMPLE 2: Preparation of Expression Constructs<br>
A plasmid containing the napin cassette derived from pCGN3223 (described in U.S. Patent 5,639,790, the entirety of which is incorporated herein by reference) is modified to make it more useful for cloning large DNA fragments containing multiple restriction sites, and to allow the cloning of multiple napin fusion genes into plant binary transformation vectors. An adapter comprised of the self annealed oligonucleotide of sequence<br>
CGCGATTTAAATGGCGCGCGCCCTCACTGCAGGCCTGGGCGCGCCATTTAAA<br>
T (SEQ ID NO: 16) is ligated into the cloning vector pBC SK+ (Stratagene) after digestion with the restriction endonuclease BssHII to construct vector pCGN7765. Plasmids pCGN3223 and pCGN7765 are digested with Not1 and ligated together. The resultant vector,<br>
pCGN7770, contains the pCGN7765 backbone with the napin seed specific expression<br>
cassette from pCGN3223.<br>
The cloning cassette pCGN7787 comprises essentially the same regulatory elements as<br>
pCGN7770, with the exception that the napin regulatory regions of pCGN7770 have been<br>
replaced with the double CAMV 35S promoter and the polyadenylation and transcriptional<br>
termination region.<br>
A binary vector for plant transformation, pCGN5139, is constructed from pCGN1558<br>
(McBride and Summerfelt, Plant Molecular Biology, 14:269-276 (1990)). The polylinker of<br>
pCGN1558 is replaced as a Hindin/Asp718 fragment with a polylinker containing unique<br>
restriction endonuclease sites, AscI, PacI, Xbal, Swal, BamHI, and Notl. The Asp718 and<br>
HindHT restriction endonuclease sites are retained in pCGN5139.<br>
A series of binary vectors are constructed to allow for the rapid cloning of DNA<br>
sequences into binary vectors containing transcriptional initiation regions (promoters) and<br>
transcriptional termination regions.<br>
The plasmid pCGN8618 is constructed by ligating oligonucleotides 5-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGG-3' (SEQ ID NO: 17) and 5'-<br>
TCGACCTGCAGGAAGCTrGCGGCCGCGGATCC-3' (SEQ ID NO: 18) into SalII/XhoI-digested pCGN7770. A fragment containing the napin promoter, polylinker and napin 3' region is excised from pCGN8618 by digestion with Asp718I; the fragment is blunt-ended by filling in the 5' overhangs with Klenow fragment then ligated into pCGN5139 that is digested with Asp7181 and HinIII and blunt-ended by filling in the 5' overhangs with Klenow fragment A plasmid containing the insert oriented so that the napin promoter is closest to the blunted Asp718I site of pCGN5139 and the napin 3' is closest to the blunted Hindm site is subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid is designated pCGN8622.<br>
 The plasmid pCGN8619 is constructed by ligating oligonucleotides 51-TCGACCTGCAGGAAGCTGCGGCCGCGGATCC-3' (SEQ ID NO: 19) and 5'-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGG-3' (SEQ ID NO: 20) into SalI/XhoI-digested pCGN7770. A fragment containing the napin promoter, polylinker and napin 3' region is removed from pCGN8619 by digestion with Asp718I; the fragment is blunt-ended by filling in the 5' overhangs with Klenow fragment then ligated into pCGN5139 that is digested with Asp7181 and Hindin and blunt-ended by filling in the 5' overhangs with<br><br>
Klenow fragment A plasmid containing the insert oriented so that the napin promoter is closest to the blunted Asp7181 site of pCGN5139 and the napin 3' is closest to the blunted HindHI site is subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid is designated pCGN8623.<br>
The plasmid pCGNS620 is constructed by ligating oligonucleotides 5'-TCGAGGATCCGCGGCCX3CAAGCTTCCTGCAGGAGCT-3, (SEQ ID NO: 21) and 5'-CCTGCAGGAAGCTTGGCGGCCGCGGATCC-3' (SEQ ID NO: 22) into Sall/SacI-digested pCGN7787. A fragment containing the d35S promoter, polylinker and trial 3' region is removed from pCGN8620 by complete digestion with Asp718I and partial digestion with NotI The fragment is blunt-ended by filling in the 5' overhangs with Klenow fragment then ligated into pCGN5139 that is digested with Asp718I and Hindin and blunt-ended by filling in the 5' overhangs with Klenow fragment A plasmid containing the insert oriented so mat the d35S promoter is closest to the blunted Asp718I site of pCGN5139 and the tml 3' is closest to the blunted HindHI site is subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid is designated pCGN8624.<br>
The plasmid pCGN8621 is constructed by ligating oligonucleotides 5'-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCCAGCT-3' (SEQ ID NO: 23) and 5'-GGATCCGCGGCCGCAAGCTTCCTGCAGG-3' (SEQ ID NO: 24) into Sall/SacI-digested pCGN7787. A fragment containing the d35S promoter, polylinker and tml 3' region is removed from pCGN8621 by complete digestion with Asp7181 and partial digestion with NotI.<br>
The fragment is blunt-ended by filling in the 5' overhangs with Klenow fragment then ligated into pCGN5139 mat had been digested with Asp718I and HindIII and blunt-ended by filling in the 5' overhangs with Klenow fragment A plasmid containing the insert oriented so that the d35S-promoter is closest to the blunted Asp718I site of pCGN5139 and the tml 3' is closest to the blunted Hindin site is subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid is designated pCGN8625.<br>
The plasmid construct pCGN8640 is a modification of pCGN8624 described above. A 938bp PstI fragment isolated from transposon Tn7 which encodes bacterial spectinomycin and streptomycin resistance (Fling et al., Nucleic Acids Research, 13(19):7095-7106 (1985)), a<br>
determinant for E. coli and Agrobacterium selection, is blunt-ended with Pfu polymerase. The blunt-ended fragment is ligated into pCGN8624 that had been digested with Spel and blunt-ended with Pfu polymerase. The region containing the PstI fragment is sequenced to confirm both the insert orientation and the integrity of cloning junctions.<br>
The spectinomycin resistance marker is introduced into pCGN8622 and pCGN8623 as follows. A 7-7 Kbp AvrII-SnaBI fragment from pCGN8640 is ligated to a 10.9 Kbp AvrII-SnaBI fragment from pCGN8623 or pCGN8622, described above. The resulting plasmids are pCGN8641 and pCGN8643, respectively.<br>
The plasmid pCGN8644 is constructed by ligating oligonucleotides 5'-GATCACCTGCAGGAAGCTTGCGGCCGCGGATCCAATGCA-3, (SEQ ID NO: 25) and 5' TTGGATCCGCGGCCGCAAGCTTCCTGCAGGT-3, (SEQ ID NO: 26) into BamHI-PstI digested pCGN8640.<br>
Synthetic oligonucleotides are designed for use in Polymerase Chain Reactions (PCR) to amplify the coding sequences of each of the nucleic acids that encode the polypeptides of SEQ ID NOs: 1-7,9-11,43-44,57-58, and 90 for the preparation of expression constructs.<br>
The coding sequences of each of the nucleic acids that encode the polypeptides of SEQ ID NOs: 1-7,9-11,43-44,57-58, and 90 are all amplified and cloned into the TopoTA vector (Invitrogen). Constructs containing the respective homogentisate prenyl transferase sequences are digested with NotI and Sse8387I and cloned into the turbobinary vectors described above.<br>
Synthetic oligonucleotides were designed for use in Polymerase Chain Reactions (PCR) to amplify SEQ ID NO: 33 for the preparation of expression constructs and are provided in the table below:<br>
(Table Removed)<br>
SEQ ID NO: 33 was amplified using the respective PCR primers shown in the table above and cloned into the TopoTA vector (Invitrogen). Constructs containing the respective homogentisate prenyl transferase sequences were digested with NotI and Sse8387I and cloned into the turbobinary vectors described above.<br>
SEQ ID NO: 33 was cloned in the sense orientation into pCGN8640 to produce the plant transformation construct pCGN10800 (Figure 4). SEQ ID NO: 33 is under control of the enhanced 35S promoter.<br>
SEQ ID NO: 33 was also cloned in the antisense orientation into the construct pCGN8641 to create pCGN10801 (Figure 5). This construct provides for the antisense expression of SEQ ID NO: 33 from the napin promoter.<br>
SEQ ID NO: 33 was also cloned in the sense orientation into the vector pCGN8643 to create the plant transformation construct pCGN10822 (Figure 7). This construct provides for the sense expression of SEQ ID NO: 33 from the napin promoter.<br>
SEQ ID NO: 33 was also cloned in the antisense orientation into the vector pCGN8644 to create the plant transformation construct pCGN10803 (Figure 6). This construct provides for the antisense expression of SEQ ID NO: 33 from the enhanced 35S promoter.<br>
EXAMPLE 3: Plant Transformation<br>
Transgenic Brassica plants are obtained by Agrobacteriurn-mediated transformation as described by Radke et al, Theor. Appl. Genet., 75:685-694 (1988); Plant Cell Reports, 11:499-505 (1992). Transgenic Arabidopsis thaliana plants may be obtained by Agrobacterium-mediated transformation as described by Valverkens et al., Proc Nat. Acad. Set, 85:5536-5540 (1988), or as described by Bent et al., Science, 265:1856-1860 (1994), or Bechtold et al., CR.Acad.ScL life Sciences, 316:1194-1199 (1993). Other plant species may be similarly transformed using related techniques.<br>
Alternatively, microprojectile bombardment methods, such as described by Klein et al., Bio/Technology, 10:286-291 may also be used to obtain nuclear transformed plants.<br>
EXAMPLE 4: Identification of additional homogentisate prenyl transferase<br>
In order to identify additional homogentisate prenyl transferase, motifs identified through sequence homology are used to search a database of cDNA sequences containing full insert sequences. The cDNA database is first translated in all six frames and then a HMM search is done using a HMM model built for the motifs. All HMM hits are annotated by performing a blast search against a non-redundant amino acid database. All motifs are sensitive and identify homogentisate prenyl transferase sequences present in the database. Novel homogentisate prenyl transferase sequences are thereby discovered.<br>
EXAMPLE 5: Transgenic Plant Analysis<br>
Arabidopsis plants transformed with constructs for the sense or antisense expression of the homogentisate prenyl transferase proteins are analyzed by High Performance Liquid Chromatography (HPLC) for altered levels of total tocopherols and tocotrienols, as well as altered levels of specific tocopherols and tocotrienols (e.g. α, ß , and S-tocopherol/tocotrienol).<br>
Extracts of leaves and seeds are prepared for HPLC as follows. For seed extracts, 10 mg of seed is added to 1 g of microbeads (Biospec) in a sterile microfuge tube to which 500 ul 1% pyrogallol (Sigma Chem)/ethanol is added. The mixture is shaken for 3 minutes in a mini Beadbeater (Biospec) on "fast" speed. The extract is filtered through a 0.2 um filter into an autosampler tube. The filtered extracts are men used in HPLC analysis described below.<br>
Leaf extracts are prepared by mixing 30-50 mg of leaf tissue with 1 g microbeads and freezing in Uquid nitrogen until extraction. For extraction, 500 ul 1% pyrogallol in ethanol is added to the leaf/bead mixture and shaken for 1 minute on a Beadbeater (Biospec) on "fast" speed. The resulting mixture is centrifuged for 4 minutes at 14,000 rpm and filtered as described above prior to HPLC analysis.<br>
HPLC is performed on a Zorbax silica HPLC column (4.6 mm X 250 mm), using a fluorescent detection monitor, with excitation and emission spectra set at 290 nm and 336 nm, respectively. Solvent A is hexane and solvent B is methyl-t-butyl ether. The injection volume is 20 ul, the flow rate is 1.5 ml/min, the run time is 12 min (40°C) using the table below:<br>
Time	Solvent A	Solvent B<br>
0min.	90%	10%<br>
10	min.	90%                                  10%<br>
11	min.	25%	              10%.<br>
12	min	90%	10%<br>
Tocopherol standards in 1% pyrogallol/ethanol are also run for comparison (alpha tocopherol, gamma tocopherol, beta tocopherol, delta tocopherol, and tocopherol (tocol) (all from Matreya, State College, PA, or Calbiochem, La Jolla, CA.)).<br>
Standard curves for alpha, beta, delta, and gamma tocopherol are calculated using Chemstation software. The absolute amount of component x is: Absolute amount of x= Response, x RFX x dilution factor where Responsex is the area of peak x, RFX, is the response<br>
factor for component x (Amountx/Responsex) and the dilution factor is 500 ul. The ng/mg tissue is found by: total ng component/mg plant tissue.<br>
Results of the HPLC analysis of seed extracts of transgenic Arabidopsis lines containing pMON 10822 for the expression of SEQ ID NO: 33 from the napin promoter are provided in Figure 8.<br>
HPLC analysis results of Arabidopsis seed tissue expressing the SEQ ID NO: 33 sequence from the napin promoter (pMON10822) demonstrates an increased level of tocopherols in the seed. Total tocopherol levels are increased as much as 50 to 60% over the total tocopherol levels of non-transformed (wild-type) Arabidopsis plants (Figure 8).<br>
Results of the HPLC analysis of seed extracts of transgenic Arabidopsis lines 1387-1624 containing pMON10803 for the antisense expression of SEQ ID NO: 33 from the enhanced 35S promoter are provided in Figure 9. Two lines, 1393 and 1401, show a substantial reduction in overall tocopherol levels, supporting the position that HPT is a homogentisate prenyl transferase involved in the synthesis of tocopherol.<br>
Results of the HPLC analysis of seed extracts of transgenic Arabidopsis lines containing constructs for the expression of SEQ ID NOs: 5,9-11,43-44,57-58, and 90 are obtained.<br>
Results of the HPLC analysis of seed extracts of transgenic Arabidopsis lines containing constructsfor the expression of SEQ ID NOs: 5,9-11,43-44,57-58, and 90 from the enhanced 35S promoter are obtained.<br>
EXAMPLE 6: Expression of a homogentisate prenyl transferase as single gene, and in combination with HPPD and tyrA in soy<br>
The Arabidopsis homogentisate prenyl transferase (ATPT2) (SEQ ID NO: 33) was cloned in a soy binary vector harboring an Arcelin 5 expression cassette. This expression cassette consisted of an Arcelin 5-promoter, a multi cloning site, and the Arcelin 5 3'-untranslated sequence in the order as described. Vector construction for this construct and the following constructs was performed using standard cloning techniques well established in the art and described in lab manuals such as (Sambrook et al. 2001). The resulting binary vector for soy seed-specific expression of ATPT2 was designated pMON36581 (Figure 10). Similarly the Syneclwcystis homogentisate prenyl transferase (slr1736) (SEQ ID NO: 29) was fused to a chloroplast target peptide (CTPl), and cloned into the Arcelin 5 soy seed-specific expression cassette. The resulting binary plasmid was designated pMON69933 (Figure 11).An additional binary plasmid for seed-specific co-expression of the Arabidopsis p-hydroxyphenylpyruvate dioxygenase (HPPDAO and the bifunctional prephenate dehydrogenase from Erwinia herbicola (tyrAEh) (see WO 02/089561) was constructed by fusing the HPPDArgene and the ryrAEh-gene to the chloroplast target peptides, CTP2, and CTP1, respectively. These fusion genes were subsequently cloned into the multi cloning site of soy seed-specific expression cassettes consisting of the p7Sa'-promoter, a multi cloning site, and the E9 3'-untranslated region. The HPPDAI expression cassette was cloned into a binary vector downstream of the tyrAEh expression cassette resulting in the formation of pMON69924 (Figure 12).<br>
A fourth plasmid was constructed by cloning the Arcelin 5-expression cassette for slrl736 (SEQ ID NO: 29), downstream of the HPPDAt, and the tyrAEh expression cassettes, resulting in the formation of pMON69943 (Figure 13).<br>
Each of these binary constructs was transformed into soybean. R1 seed pools from plants harboring these constructs were analyzed for tocopherol content and composition. For constructs pMON36581 and pMON69933, the seed for analysis were chosen at random. Seed from plants transformed with pMON69924 and pMON69943 showed a segregating dark phenotype. This phenotype has been associated with the presence of increased levels of homogentisic acid as a result of the expression of trans genes HPPD and tyrA. Seed without dark coloration did have wild-type tocopherol levels and were not transgenic. For this reason colored seed were chosen for analysis of plants transformed with pMON69924 or pMON69943. For the impact of the HPT expression on total tocopherol accumulation in a single gene vector, or in a multi gene vector, seed from non-transformed soy, or seed transformed with pMON69924 served as controls, respectively. Figure 14 summarizes the tocopherol data obtained from these experiments. While expression of ATPT2 or slrl736 increased total tocopherol and tocotrienol levels in soy moderately, the impact of HPT expression in the context of a multi gene vector was much more pronounced. Figure 14 demonstrates a significantly increased level of tocopherol and tocotrienol accumulation for pMON69943 compared to pMON69924 lines. These data suggest that combination of an HPT with tyrA, and HPPD can substantially enhance tocopherol biosynthesis in soy.<br>
Western analysis is carried out to detect the transgene expression in tissues harboring the gene of interest (GOT) expression cassette using the GOI protein specific antibody.<br>
Northern analysis is done for detecting the mRNA level of the transgene using the GOI sequence specific radiolabelled probe.<br>
EXAMPLE 7: Identification of additional homogentisate prenyl transferase sequences In an analysis of the non-redundant amino acid database, Motifs II and IV (SEQ ID NOs: 40 and 42 identified in addition to HPT sequences, two genomic variants of Arabidopsis thaliana sequence related to HPTs (SEQ ID NOs: 61-62). These sequences are based on insillico prediction from genomic sequence by gene prediction algorithms. Further bioinformatic analysis showed that these sequences encoded an additional homogentisate prenyl transferase related to HPT. Both sequences (SEQ ID NOs: 61-62) were used to search the non-redundant amino acid database. The BLAST search results indicated that these sequences are related most to HPT sequences from cyanobactefia (SEQ ID NOs: 1-3) and Arabidopsis (SEQ ID NO: 7).<br>
Alignment of gil5229898 (970 aa)(SEQ ID NO: 61) and gil0998133 (441 aa) (SEQ ID NO: 62) showed that<br>
a)	C terminal half of gil5229898 (SEQ ID NO: 61) overlaps with gil0998133 (SEQ ID NO: 62);<br>
b)	the last 40-50 aa in the C terminal portions of these two proteins do not align; and<br>
c)	the N terminal of gi 15229898 does not align also with HPTs (SEQ ID NOs: 1-7, and 9-11). These findings indicate discrepancy in the coding sequence prediction reported in Genbank.<br>
In order to verify the predicted sequence, the B AC sequence of the Arabidopsis genome corresponding to the region was downloaded from Genbank (gill2408742jgbjAC016795.6|ATAC016795,100835bp). Coding sequences were predicted from this BAC clone using the FGENESH (Solovyev V.V. (2001) Statistical approaches in Eukaryotic gene prediction: in Handbook of Statistical genetics (eds. Balding D. et al.\ John Wiley &amp; Sons, Ltd, p. 83-127) gene prediction program. FGENESH predicted 28 proteins from mis BAC clone. To identify new homogentisate prenyl transferase proteins among 28 FGENESH predicted proteins, all 28 predicted proteins were blasted against the non-redundant amino acid database. FGENESH predicted protein No. 25 (402aa) (SEQ ID NO: 45) was most similar to gil0998133 (441 aa) (SEQ ID NO: 62), C terminal half of gil5229898 (970 aa) (SEQ ID NO: 61) and HPTs (SEQ ID NOs: 1-7, and 9-11.)<br>
To provide functional and transcriptional evidence and to confirm the coding sequence for this gene, plant EST sequences database comprising proprietary and public sequence was searched. We found several ESTs (SEQ ID NOs: 63-72) which match the N terminal and C terminal portions of this gene. The new gene was named HPT2 (SEQ ID NO: 59) from Arabidopsis. The HPT2 (SEQ ID NO: 57) sequence is quite distinct from HPT1 (SEQ ID NO: 7).<br>
HPT2 (SEQ ID NO: 57) from Arabidopsis is also known as Tocopherol Synthase (TS). Present data suggests that the overexpression of TS leads to a similar increase in the amount of overall tocopherol, over the wild type, as with HPT1 (SEQ ID NO: 33). However, the enzymes may have different biochemical characteristics because the overexpression of TS results in less production of the delta tocopherol than the overexpression of HPT1 (SEQ ID NO: 33).<br>
The presence of chloroplast transit peptide in the HPT2 Arabidopsis sequence (SEQ ID NOs: 45 and 57) was verified using ChloroP program (Olof Emanuelssonl, Henrik Nielsenl, 2, and Gunnar von HeijnelChloroP, a neural network-based method for predicting chloroplast transit peptides and their cleavage sites. Protein Science: 8: 978-984,1999).<br>
In addition to SEQ ID NOs: 1,7, and 9-11(HPT), SEQ ID NOs: 57-58, and 90 (HPT2) were added to the alignment, see Figures 24-25 and the resulting motifs analyzed. Motif V (SEQ ID NO: 46), VII (SEQ ID NO: 48), and VIE (SEQ ID NO: 49) are specific to HPT and HPT2 sequences. A HMM search of the non-redundant amino acid database using these motifs identified only cyanobacteria (SEQ ID NOs: 1-3, and 43), photobacteria (SEQ ID NO: 44), and plant HPTs (SEQ ID NOs: 7, and 61-62). Motif VII (SEQ ID NO: 48) identified distantly related ubiA prenyl transerase from bacteria in addtion to homogentisate prenyl transferase. However, the sensitivity of Motif VII to homogentisate prenyl transferase was higher. Homogentisate prenyl transferases had lower e-values by several orders and hihger alignment score (higher than 30). HPT2 sequences are distinct from HPT and cyanobacterial HPTs as demonstrated by the sequence dendogram in Figure 26.<br>
SEQ ID NOs: 43-44 were added to an alignment with SEQ ID NOs: 14,6-7,9-11, 57-58, and 91, see Figures 33-34, and the resulting motifs (SEQ ID NOs: 92-95, Motifs IXXII) were analyzed. Specificity of these motifs to homogentisate prenyl transferases was confirmed by HMM search. A non redundant database containing more than 134M sequence was searched using HMM models built from the alignments shown in Figure 34 for Motifs<br>
' IX- XII. E value limits for the search was set at 1.0. All four motifs identified only homogentisate prenyl transferase from cyanobacteria, photobacteria and Arabidopsis. Upper E values limits for Motif IX, X, XI, and XII were 0.9,11E10-11,0.03,8E108 respectively. The small size of motifs resulted in higher E values for Motif IX and XI.<br>
EXAMPLE 8: Transformation and expression of a wild type Arabidopsis HPT2 gene<br>
in sense and antisense orientations with respect to seed-specific and<br>
constitutive promoters in Arabidopsis thialiana. The HPT2 full-length cDNA (SEQ ED NO: 59) is excised from an EST clone, CPR230005 (pMON69960-Figure 15), with Sail and NotI enzymes, blunt-ended and cloned in between the napin promoter and napin 3' end at blunt-ended SalI site in sense and antisense orientations with respect to the napin promoter in pMON36525 (Figure 16) to generate recombinant binary vectors pMON69963 (Figure 17) and pMON69965 (Figure 18), respectively.  The sequence of the HPT2 cDNA is confirmed by sequencing with napin 5'-sense (5,-GTGGCTCGGClTCACTTTTTAC-3,) (SEQ ID NO: 50) and napin 3'-antisense (5'-CCACACTCATATCACCGTGG-3') (SEQ ID NO: 51) primers using standard sequencing methodology. The HPT2 cDNA used to generate the pMON69963 and pMON69965 is also cloned in between the enhanced 35S promoter and E9-3' end at blunt-ended BglH and BamHI sites of pMON10098 (Figure 19) to generate the pMON69964 (Figure 20) and pMON69966 (Figure 21) in sense and antisense orientations with respect to the enhanced 35S promoter, respectively. Additional HPT2 internal primers synthesized to completely sequence the whole HPT2 cDNA are listed in the table below:<br>
A list of primers used for confirming the HPT2 cDNA sequence.<br>
(Table Removed)<br>
The plant binary vectors pMON69963, and pMON69965 are used in Arabidopsis thaliana plant transformation to direct the sense and antisense expression of the HPT2, in the<br>
embryo. The binary vectors pMON69964, and pMON69966 are used in Arabidopsis thalicna plant transformation for sense and antisense expression of the HPT2 in whole plant. The binary vectors are transformed into ABI strain Agrobacterium cells by electroporation (Bio-Rad Electroprotocol Manual, Dower et al, Nucleic Acids Res., 16:6127-6145 (1988)). Transgenic Arabidopsis thaliana plants are obtained by Agrobacterium-mediated transformation as described by Valverkens et al., Proc. Nat. Acad. Sci., 85:5536-5540 (1988), Bent et al, Science, 265:1856-1860 (1994), and Bechtold et at, C.R. Acad. Sci., Life Sciences, 316:1194-1199 (1993). Transgenic plants are selected by sprinkling the transformed T1 seeds onto the selection plates containing MS basal salts (4.3 g/L), Gamborg'a B-5,500X (2.0 g/L), sucrose (10 g/L), MES (0.5 g/L), phytagar (8 g/L), carbenicillhi (250 mg/L), cefotaxime (100 mg/L), plant preservation medium (2 ml/L), and kanamycin (60 mg/L) and then vernalizing them at 4°C in the absence of light for 2-4 days. The seeds are transferred to 23°C, and 16/8 hours light/dark cycle for 5-10 days until seedlings emerge. Once one set of true leaves are formed on (he kanamycin resistant seedlings, they are transferred to soil and grown to maturity. The transgenic lines generated through kanamycin selection are grown under two different light conditions. One set of the transgenic lines are grown under 16 hrs light and 8 hrs dark and another set of the transgenic lines are grown under 24 hrs light to study the effect of light on seed tocopherol levels. The T2 seed harvested from the transformants is analyzed for tocopherol content. The results from the seed total tocopherol analysis from lines grown under both normal and high light conditions are presented in Figures 22 and 23. Seed-specific overexpression of HPT2 under normal and high light conditions produced a significant 1.6- and 1.5-fold increase in total tocopherol levels (alpha=0.05; Tukey-KramerHSD) (SAS institute, 2002, JPM version 5.0).<br>
Expression of HPT2 using the constitutive promoter, e35S, produced about 20% increase in seed total tocopherol levels as compared to control under both light conditions. Maximum tocopherol level reduction in lines harboring the enhanced 35S::HPT2 antisense construct was 20%. Overall, the significant increase in seed total tocopherol level in the Arabidopsis tJialiana lines harboring the HPT2 driven by the napin promoter suggests that HPT2 plays a key role in tocopherol biosynthesis.<br>
Western analysis is carried out to detect the transgene expression in tissues harboring the gene of interest (GOI) expression cassette using the GOI protein specific antibody.<br>
Northern analysis is done for detecting the mRNA level of the transgene using the GOI sequence specific radiolabelled probe.<br>
EXAMPLE 9: Preparation of plant binary vector for expression of HPT2 from Arabidopsis in combination with tocopherol pathway genes<br>
To investigate the combinatorial effect of HPT2 with other key enzymes in the pathway, a plant binary vector containing seed-specifically expressed hydroxyphenylpyruvate dioxygenase (HPPD), bifunctional prephenate dehydrogenase tyrA, and HPT2 (pMON81028 - Figure 27) is prepared. The pMON81028 is made by exercising the pNapin::HPT2::Napin 3' expression cassette from pMON81023 (Figure 28) with Bspl20I and NotI enzymes and ligating it to pMON36596 (Figure 29) at NotI site. The pMON36596 contains the pNapin::CTP2::HPPD::Napin 3' and pNapm::CTPl::TyrA::Napin 3' expression cassettes. The pMON81028 is transformed into Arabidopsis thaliana plant using the method described in Example 8.<br>
EXAMPLE 10: Preparation of construct for bacterial expression of HPT2 from<br>
	Arabidopsis<br>
The EST clone CPR23005 attaining the HPT2 full length cDNA is used as a template for PCR to amplify the HPT2 cDNA fragment codes for the mature form of the HPT2 protein. Two sets of PCR products are generated to clone at the pET30a(+) vector (Novagen, Inc.) (Figure 30) to produce HPT2 protein with and without his tag. The primer set BXK174 (5'- CACATATGGCATGTTCTCAGGTTGGTGCTGC-3,) (SEQ ID NO: 84) and BXK176 (5'-GCGTCGACCTAGAGGAAGGGGAATAACAG-3') (SEQ ID NO: 85) is used for cloning HPT2 at the Ndel and SalI sites of pET30a(+), behind the T7 promoter to generate mature HPT2 protein without the his tag. The resulting recobmbinant vector is named pMON69993 (Figure 31). The primer set BXK175 (5'-<br>
CAACCATGGCATGT^CrcAGGT^GGTGCTGC-3,) (SEQ ID NO: 86) and BXK176 (5'-Ga3Ta3ACCTAGAGGAAGGGGAATAACAG-3') (SEQ ID NO: 87) is used to generate HPT2 PCR product to clone at the Ncol and SalI sites of pET30a(+) to produce mature HPT2 with his tag. The recombinant vector is named as pMON69992 (Figure 32). The pMON69993 and pMON69992 is used for producing bacterial expressed HPT2 to carry out enzyme assays to confirm its homogentisate prenyl transferase activity and specificity towards geranylgeranyl pyrophosphosphate, phytyl pyrophophaste and solanyl pyrophosphate substrates.<br><br><br><br><br><br><br><br>
We claim:<br>
1.	A substantially purified nucleic acid molecule encoding an amino acid sequence selected from the group consisting of SEQ J-D NOs: 5, 9-11, 57-58 and 90.<br>
2.	A substantially purified polypeptide molecule comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5, 9-11, 57-58 and 90.<br>
3.	An antibody capable of specifically binding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5, 9-11, 57-58, and 90.<br>
4.	A nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5, 9-11, 43-44, 57-58 and 90 introduced in a transformed plant.<br>
5.	The nucleic acid molecule as claimed in claim 4, wherein said transformed plant is selected from the group consisting of alfalfa, Arabidopsis thaliana, barley, Brassica napus, Brassica campestris, oilseed rape, broccoli, cabbage, citrus, canola, cotton, garlic, oat, Allium, flax, an ornamental plant, peanut, pepper, potato, rapeseed, rice, rye, sorghum, strawberry, sugarcane, sugar beet, tomato, wheat, poplar, pine, fir, eucalyptus, apple, lettuce, lentils, grape, banana, tea, turf grasses, sunflower, soybean, chick peas, com, Phaseolus, crambe, mustard, castor bean, sesame, cottonseed, linseed, safflower, and oil palm.<br>
6.	The nucleic acid molecule as claimed in claim 4, wherein said transformed plant is selected from the group consisting of oilseed rape, soybean and canola.<br>
7.	The nucleic acid molecule as claimed in claim 4, wherein said transformed plant comprises tissue with at least one of an increased total tocopherol level and an increased tocotrienol level relative to a plant with a similar genetic background but lacking said introduced nucleic acid molecule.<br>
8.	The nucleic acid molecule as claimed in claim 4, wherein said transformed plant produces a seed with at least one of an increased total tocopherol level and an increased total tocotrienol level relative to a plant with a similar genetic background but lacking said introduced nucleic acid molecule.<br>
9.	The nucleic acid molecule as claimed in claim 4, wherein said nucleic acid molecule is<br>
operably linked to a promoter.<br>
10.	The nucleic acid molecule as claimed in claim 9, wherein said promoter is a seed-specific promoter.<br>
11.	The nucleic acid molecule as claimed in claim 10, wherein said promoter is selected from the promoters consisting of: napin, 7S alpha, 7S alpha', USP 88, enhanced USP 88, Arcelin 5, and Oleosin.<br>
12.	The nucleic acid molecule as claimed in claim 4, further comprising an introduced second nucleic acid molecule encoding an enzyme selected from the group consisting of tyrA, prephenate dehydrogenase, tocopherol cyclase, dxs, dxr, GGPPS, HPPD, AANT1, IDI, GGH, GMT, tMT2, MT1, GCPE, and complements thereof.<br>
13.	The nucleic acid molecule as claimed in claim 12, wherein said transformed plant is selected from the group consisting of alfalfa, Arabidopsis thaliana, barley, Brassica napus, Brassica campestris, oilseed rape, broccoli, cabbage, citrus, canola, cotton, garlic, oat, Allium, flax, an ornamental plant, peanut, pepper, potato, rapeseed, rice, rye, sorghum, strawberry, sugarcane, sugar beet, tomato, wheat, poplar, pine, fir, eucalyptus, apple, lettuce, lentils, grape, banana, tea, turf grasses, sunflower, soybean, chick peas, com, Phaseolus, crambe, mustard, castor bean, sesame, cottonseed, linseed, safflower, and oil palm.<br>
14.	The nucleic acid molecule as claimed in claim 12, wherein said transformed plant is selected from the group consisting of canola, oilseed rape, and soybean.<br>
15.	The nucleic acid molecule as claimed in claim 12, wherein said transformed plant comprises tissue with increased α-tocopherol levels relative to a plant with a similar genetic background but lacking said introduced nucleic acid molecules.<br>
16.	The nucleic acid molecule as claimed in claim 12, wherein said transformed plant produces a seed with at least one of an increased total tocopherol level and an increased tocotrienol level relative to a plant with a similar genetic background but lacking said introduced nucleic acid molecules.<br>
17.	The nucleic acid molecule as claimed in claim 12, wherein at least one of said first and second nucleic acids molecule is operably linked to a promoter.<br>
18.	The nucleic acid molecule as claimed in claim 17, wherein said promoter is a seed specific promoter.<br>
19.	The nucleic acid molecule as claimed in claim 18, wherein said promoter is selected from the promoters consisting of: napin, 7S alpha, 7S alpha', USP 88, enhanced USP 88, Arcelin 5, and Oleosin.<br>
20.	A nucleic acid molecule comprising an introduced promoter region which functions in plant cells to cause the production of an mRNA molecule, wherein said introduced promoter region is linked to a transcribed nucleic acid molecule having a transcribed strand and a non-transcribed strand, wherein said transcribed strand is complementary to a nucleic acid molecule encoding a polypeptide selected from the group consisting of SEQ ID NOs: 5, 9-11, 43-44, 57-58, and 90, and wherein said transcribed nucleic acid molecule is linked to a 3' non-translated sequence that functions in the plant cells to cause termination of transcription and addition of polyadenylated ribonucleotides to a 3' end of the mRNA sequence.<br>
21.	The nucleic acid molecula as claimed in claim 20, wherein the expression of homogentisate prenyl transferase is reduced relative to a plant with a similar genetic background but lacking said introduced nucleic acid molecule.<br>
22.	A method of producing a plant having a seed with an increased total tocopherol level comprising: (A) transforming said plant with an introduced nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5, 9-11, 43-44, 57-58 and 90, and (B) growing said transformed plant.<br>
23.	The method of producing a plant as claimed in claim 22, wherein said plant is selected from the group consisting of alfalfa, Arabidopsis thaliana, barley, Brassica napus, Brassica campestris, oilseed rape, broccoli, cabbage, canola, citras, cotton, garlic, oat, Allium, flax, an ornamental plant, peanut, pepper, potato, rapeseed, rice, rye, sorghum, strawberry, sugarcane, sugar beet, tomato, wheat, poplar, pine, fir, eucalyptus, apple, lettuce, lentils, grape, banana, tea, turf grasses, sunflower, soybean, com, Phaseolus, crambe, mustard, castor bean, sesame, cottonseed, linseed, safflower, and oil palm.<br>
24.	The method as claimed in claim 22, wherein said plant is selected from the group consisting of canola, oilseed rape, and soybean.<br>
25.	The method as claimed in claim 22, wherein the plant is transformed with an introduced second nucleic acid molecule encoding an enzyme selected from the group consisting of tyrA,<br>
prephenate dehydrogenase, tocopherol cyclase, dxs, dxr, GGPPS, HPPD, AANT1, IDI, GGH, tMT2, GMT, Mil, GCPE, and complements thereof.<br>
26.	The method as claimed in claim 22 wherein said transformed plant is that as claimed in claim<br>
4.<br>
27.	The method as claimed in claim 22 or 26 wherein said seed has at least one of an increased total tocopherol level and an increased tocotrienol level relative to a seed from a plant having a similar genetic background but lacking said introduced nucleic acid molecule.<br>
28.	The method as claimed in claim 22 wherein said seed is derived from a transformed plant as \ claimed in claim 12 comprising an introduced first nucleic acid molecule encoding an introduced polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 5, 9-11, 43-44, 57-58, and 90 and an introduced second nucleic acid encoding an enzyme selected from the group consisting of tyrA, prephenate dehydrogenase, tocopherol cyclase, dxs, dxr, GGPPS, HPPD, AANT1, IDI, GGH, tMT2, MT1, GCPE, GMT, and complements thereof.<br>
29.	The method as claimed in claim 22 wherein said seed has at least one of an increased total tocopherol level and an increased tocotrienol level relative to a seed from a plant having a similar genetic background but lacking said introduced nucleic acid molecule.<br>
30.	A substantially purified polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95, wherein said amino acid sequence is not derived from a nucleic acid molecule that is derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato.<br>
31.	The polypeptide as claimed in claim 30, wherein more than one amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
32.	The polypeptide as claimed in claim 31, wherein said amino acid is not derived from a nucleic acid that is derived from Sulfolobus, Aeropyrum, or sorghum.<br>
33.	A substantially purified nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95 wherein said nucleic acid molecule is not derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato.<br>
34.	The nucleic acid molecule as claimed in claim 33, wherein the polypeptide comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
35.	The polypeptide as claimed in claim 34, wherein said nucleic acid molecule is not derived from a nucleic acid that is derived from Sulfolobus, Aeropyrum, or sorghum.<br>
36.	A nucleic acid molecule introduced in a transformed plant, said nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95, wherein said nucleic acid molecule is not derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, Trichodesmium erythraeum, Chloroflexus aurantiacus, wheat, leek, canola, cotton, or tomato.<br>
37.	The nucleic acid molecule as claimed in claim 36, wherein said nucleic acid molecule is not derived from Sulfolobus, Aeropyrum, or sorghum.<br>
38.	A substantially purified nucleic acid molecule comprising a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 31, 34-36, 59-60, and 91.<br>
39.	A substantially purified polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95, wherein said amino acid sequence does not comprise any of the amino acid sequences set forth in sequence listings in WOOO/68393, WO 00/63391, WO 01/62781, or WO 02/33060, and does not comprise SEQ ID NOs 1-11,43-45, 57-58, 61-62, and 90 from the present application.<br>
40.	The polypeptide as claimed in claim 39, wherein more than one amino acid sequence is selected from the group consisting of SEQ ID NOs: 39-42,46-49, and 92-95.<br>
41.	A substantially purified nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95, wherein said nucleic acid molecule does not comprise any of nucleic acid sequences set forth in sequence listings in WOOO/68393, WO 00/63391, WO 01/62781, or WO 02/33060, does not comprise the nucleic acid sequence with Genebank Accession Nos. Al 897027 or AW 563431, and does not comprise SEQ ID NOs: 27-36, 59-60, 88-89, and 91 of the present application.<br>
42.	The nucleic acid molecule as claimed in claim 44, wherein the polypeptide further comprises<br>
more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95.<br>
43.	A nucleic acid molecule introduced in a transformed plant, said nucleic acid molecule encoding a polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95, wherein said nucleic acid molecule does not comprise any of nucleic acid sequences set forth in sequence listings in WOOO/68393, WO 00/63391, WO 01/62781, or WO 02/33060, does not comprise does not comprise the nucleic acids sequence with Genebank Accession Nos. Al 897027 or AW 563431 and does not comprise SEQ ID NOs: 27-36, 59-60, 88-89, and 91 of the present application.<br>
44.	The nucleic acid molecule as claimed in claim 43, wherein the polypeptide comprises more than one amino acid sequence selected from the group consisting of SEQ ID NOs: 39-42, 46-49, and 92-95.<br>
45.	A substantially purified polypeptide with homogentisate prenyl transferase activity<br>
comprising an amino acid sequence selected from the group consisting of SEQ ID NOs: 43-44.<br>
46.	A substantially purified nucleic acid molecule encoding a polypeptide with homogentisate prenyl transferase activity comprising an amino acid sequence selected from the group consisting of SEQ ID NOs. 43-44.<br>
47.	The method as claimed in claim 28, wherein the introduced second nucleic acid is GMT.<br>
48.	The method as claimed in claim 28 and 47, wherein said seed has at least one of an increased total tocopherol level and an increased total tocotrienol level relative to a seed having a similar genetic background but lacking said introduced nucleic acid molecules, and at least about 90% of the total tocopherol is alpha tocopherol.<br>
49.	The substantially purified nucleic acid molecule as claimed in claim 48, wherein the nucleic acid molecule comprises a sequence selected from the group consisting of SEQ ID NOs. 88-89.<br><br>
50.	Homogentisate prenyl transferase polypeptide sequences identified using any of the alignments set forth in the group consisting of Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, and 35a-35b in a profile based model, excluding the amino acid sequences set forth in sequence listings in WOOO/68393, WO 00/63391, WO 01/62781, or WO 02/33060 and do not comprise SEQ ID NOs 1-11,43-45, 57-58, 61-62, and 91 from the present application.<br>
51.	Homogentisate prenyl transferase polypeptide sequences identified using any of the alignments set forth in Figures 2a-2c, 3a-3c, 24a-24b, 25a-25b, 33a-33c, 34a-34b, and 35a-35b in<br>
a profile based model, wherein said amino acid sequence is not derived from a nucleic acid molecule that is derived from Nostoc punctiforme, Anabaena, Synechocystis, Zea mays, Glycine max, Arabidopsis thaliana, Oryza sativa, wheat, leek, canola, cotton, or tomato.<br>
52.	The homogentisate prenyl transferase as claimed in claim 50 or 51, wherein the profile based<br>
model is an HMM model.<br>
53.	The homogentisate prenyl transferase as claimed in claim 52, wherein the profile based sequence search method is a HMM model generated using HMMER package version 2.2g with default parameters.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUFic3RyYWN0LSgzMC0wMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Abstract-(30-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUNsYWltcy0oMDItMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Claims-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUNsYWltcy0oMjMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Claims-(23-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUNsYWltcy0oMzAtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Claims-(30-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Correspondence Others-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDYtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Correspondence Others-(06-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzAtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Correspondence Others-(30-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDItMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Correspondence-Others-(02-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTAtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Correspondence-Others-(10-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Correspondence-Others-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Correspondence-Others-(23-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDMwLTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Description (Complete)-(30-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LURyYXdpbmdzLSgxMC0wMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Drawings-(10-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUZvcm0tMTMtKDAyLTA4LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Form-13-(02-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUZvcm0tMi0oMzAtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Form-2-(30-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUZvcm0tMy0oMDItMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Form-3-(02-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUZvcm0tMy0oMDYtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Form-3-(06-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUZvcm0tMy0oMTAtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-Form-3-(10-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LUZvcm0tMy0oMTgtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-Form-3-(18-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1ERUxOUC0yMDA0LUdQQS0oMzAtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2862-DELNP-2004-GPA-(30-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LXVyLUNsYWltcy0oMjMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-ur-Claims-(23-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LXVyLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDYtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-ur-Correspondence Others-(06-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LXVyLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-ur-Correspondence-Others-(23-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg2Mi1kZWxucC0yMDA0LXVyLUZvcm0tMy0oMDYtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2862-delnp-2004-ur-Form-3-(06-09-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258551-process-for-the-preparation-of-2-azabicyclo-3-3-0-octane-3-carboxylic-acid-derivatives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258553-a-signaling-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258552</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2862/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jan-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Jan-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Sep-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>MAILZONE E2NA, 800 N.LINDBERGH BOULEVARD, ST. LOUIS, MISSOURI 63167, U.S.A</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VALENTIN HENRY E</td>
											<td>873 M FOXSPRINGS DRIVE, CHESTERFIELD, MISSOURI 63017, U.S.A</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VENKATESH TYAMAGONDLU V</td>
											<td>12374 SUMMERHOUSE DRIVE, APT. #12, ST. LOUIS, MISSOURI 63146, U.S.A</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KARUNANANDAA BALASULOJINI</td>
											<td>724 BELLERIVE MANOR, CREVE COEUR, MISSOURI 63141, U.S.A</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/08468</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-03-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/365,202</td>
									<td>2002-03-19</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>10/391,363</td>
									<td>2003-03-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258552-homogentisate-prenyl-transferase-hpt-nucleic-acids-and-polypeptides-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:12:24 GMT -->
</html>
