<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258881-mutant-aox-1-promoters by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:07:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258881:&quot;MUTANT AOX 1 PROMOTERS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;MUTANT AOX 1 PROMOTERS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A mutant Pichia pastoris alcohol oxidase 1 (AOX1) promoter of the wild type Pichia pastoris AOX1 promoter (SEQ ID No. 1) comprising at least one mutation selected from the group consisting of: a) a transcription factor binding site (TFBS), b) nucleotides 170 to 235 (-784 to -719), nucleotides 170 to 191 (-784 to -763), nucleotides 192 to 213 (-762 to -741) , nucleotides 192 to 210 (-762 to -744), nucleotides 207 to 209 (-747 to -745), nucleotides 214 to 235 (-740 to -719), nucleotides 304 to 350 (-650 to -604), nucleotides 364 to 393 (-590 to -561), nucleotides 434 to 508 (-520 to -446), nucleotides 509 to 551 (445 to -403), nucleotides 552 to 560 (-402 to -394), nucleotides 585 to 617 (-369 to -337), nucleotides 621 to 660 (-333 to -294), nucleotides 625 to 683 (-329 to -271), nucleotides 736 to 741 (-218 to -213), nucleotides 737 to 738 (-217 to -216), nucleotides 726 to 755 (-228 to -199), nucleotides 784 to 800 (-170 to -154) or nucleotides 823 to 861 (-131 to -93) of Seq ID No. 1, and combinations thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Mutant AOX1 Promoters<br>
The present invention relates to mutant Pichia pastoris AOX1 promoters.<br>
S. cerevisiae has dominated (and still dominates) the scientific and biotechnological use as eukaryotic model organism and production system. In the last century another yeast gained great attraction: the fission yeast Schizosaccharomyces pombe. For its attribute to reproduce only by means of fission S. pombe gained outstanding attention as model organism and by today it is the most intensely studied yeast species in terms of molecular genetics and cell biology, along with S. cerevisiae. Among over 700 different yeast species known to date, the two yeasts mentioned above can provide only a limited set of interesting attributes for technological and scientific applications. Since the 1970's or 80's more and more yeast species with outstanding characteristics were investigated for biotechnology and research. These so called non-conventional yeasts (NCY) or non-Saccharomyces yeasts (in this case the term Saccharomyces includes the yeast Schizosaccharomyces pombe) are developed for several reasons: they possess either medical importance like Candida albicans or technological relevance like Yarrowia lipolytica and Kluyveromyces lactis which have the ability to grow,on particular substrates (e.g. n-alkanes, lactose) .E.g.<br>
the most common human fungal pathogen C. albicans is studied extensively to reveal the nature of the virulence factors involved in pathogenesis therefore becoming the model organism for pathogenic yeasts. Another well established group of NCY are the methylotrophic yeasts Pichia pastoris and Hansenula polymorpha (Pichia angusta) which are superior to S. cerevisiae in terms of recombinant protein production and studies of peroxisome biogenesis. These are only the most prominent members of non-conventional yeasts still having either technological or academic attraction. To date several other species are also of particular interest and this group will grow rapidly the next years.<br>
Sugars, the most abundant class of molecules in nature, are utilised by all known yeasts. Although there are great differences in substrate acceptance from species to species (see Table 1), the conversion of glucose 6-phospate or fructose 6-phosphate to pyruvate is a common theme in their metabolism. Anyhow, the<br><br>
enzymatic equipment for the glycolytic pathway varies significantly among different yeasts. While in S. cerevisiae most of the enzymes are known and characterised, at least partially, only a few enzymes were described in NCYs. Some of the functions needed for glycolysis are mediated by several genes/enzymes in some yeasts, especially those playing an additional role in control or regulation of the metabolism and/or being on a branching point like glucokinase/hexokinase, phosphofructokinase and gly-ceraldehyde 3-phosphate dehydrogenase. Normally, the isoenzymes are regulated differentially indicating diverse functions under changing environmental prerequisites. Some of the genes encoding for glycolytic enzymes are constitutive and highly expressed, e.g. the S. cerevisiae PGK1 (phosphoglycerate kinase) or the P. pastoris GAP gene (glyceraldehyde 3-phosphate dehydrogenase) while other enzymes are strictly regulated like the ENO1 (eno-lase) gene of S. cerevisiae.<br>
Table 1: Selected yeasts of biotechnological interest with relevant commercial substrates other than glucose and fructose  <br>
(Table Removed)<br>
The fate of pyruvate in metabolism varies significantly between yeast species and culture conditions. In S. cerevisiae and other so called Crabtree positive yeasts, respiration is inhibited by glucose and related sugars. This leads to the transformation of pyruvate via the pyruvate decarboxylase to ethanol and CO2, even under high amounts of oxygen, which is also known as fermentation. In Crabtree negative yeasts, where the majority of NCY is belonging to, transformation of pyruvate to ethanol occurs only under anaerobic conditions. Under aerobic conditions pyruvate is oxidised to CO2 via the pyruvate dehydrogenase and the tricarboxylic acid (TCA) cycle. The TCA cycle is of outstanding interest for the cell metabolism due to the fact that it is the only way for the oxidation of sugars to C02. Oxidation to C02 results in production of NADH, which is used for energy production. Furthermore TCA cycle intermediates are the major sources of metabolites for biosynthetic purposes. Due to the removal of intermediates the TCA cycle has to be refilled to keep it running. The main anaplerotic reactions in yeasts are the pyruvate carboxylase and the glyoxylate cycle. The first one is the major pathway when growing on ammonium as sole nitrogen source while the latter one is needed when growing on carbon sources with less than 3 carbon atoms. In contrast to this eminent interest almost nothing is known about genes or enzymes involved in the TCA cycle in NCYs. NADH generated by catabolic reactions, either in the cytosol or in mitochondria, has to be re-oxidised to NAD* to keep the reactions running. In Crabtree negative yeasts (e.g. Pichia pastoris) under aerobic conditions NADH is reoxidised mainly through the respiratory chain. The situation is significantly different in Crabtree positive yeasts like S. cerevisiae where respiration and fermentation coexists. When grown on glucose under aerobic conditions, respiration is repressed by glucose and fermentation occurs. Under these conditions NAD* is regenerated by the formation of ethanol (NADH pro-<br>
duced by glycolysis) or glycerol. Respiration in yeasts differs from the animal paradigm of this pathway as described in every biochemistry textbook. First, some yeasts, like S. cerevisiae and Kluyveromyces lactis, are lacking complex I of the respiratory chain. In these yeasts NAD* regeneration is done without pumping protons through the inner mitochondrial membrane by external and internal NADH dehydrogenases. The second major difference, found in Crabtree negative yeasts, fungi and plants, is an alternative respiration pathway in parallel to complex III and IV of the cytochrome chain. This alternative respiration is mediated by a so called alternative oxidase which transfers electrons directly from the ubiquinone pool to oxygen without pumping protons through the inner mitochondrial membrane.<br>
NADPH for biosynthetic purposes is produced in the oxidative part of the pentose phosphate pathway (PPP). Other very important metabolites provided by this pathway are ribose 5- phosphate and erythrose 4-phosphate, needed for synthesis of nucleic acids and nucleotide cofactors and for the synthesis of aromatic amino acids, respectively. There are still many gaps in the information about genes and their corresponding enzymes involved in the PPP in non-conventional yeasts. A few enzymes were isolated from Candida utilis, S. pombe and K. lactis. Compositional and kinetic characterisation revealed several differences between these enzymes. Due to the lack of information the influence on the PPP in these yeasts cannot be estimated but it has been shown that e.g. phosphoglucose isomerase mutants of K. lactis, which are deficient in glycolysis, are able to grow in glucose media, in contrast to S. cerevisiae. This observation indicates that the capacity of the pentose phosphate pathway in K. lactis is sufficient for growth on glucose as carbon source. In methylotrophic yeasts, an additional transketolase (dihydroxyacetone synthase) could be found. This enzyme is localised in peroxisomes and confers the assimilation of formaldehyde into the cell metabolism by condensation with xylulose 5-phosphate with formation of dihydroxyacetone and glyceraldehyde 3-phosphate.<br>
Yeasts as unicellular eukaryotic organisms provide attractive expression systems for recombinant protein production. They combine the pros of bacteria, like well-developed genetic manipulation techniques, simple, safe and therefore cheap (large-scale) cultivation techniques, with the main benefit of euka-<br>
ryotic expression systems, namely eukaryotic protein processing. Due to the above-mentioned reasons S. cerevisiae has dominated this field for many years resulting in a large number of proteins (e.g. insulin, HBsAg, HSA) produced in this organism. S. cerevisiae shows some limitations due to hyperglycosylation, retention of secreted proteins in the periplasmic space, plasmid instability and low product yields. To overcome the limitations of this single organism a small set of non-conventional yeasts has been developed as hosts for heterologous gene expression. Among others, K. lactis, Y. lipolytica and the methylotrophic yeasts Candida boidinii, H. polymorpha and P. pastoris were used, but only the latter 2 species gained outstanding commercial interest. Schizosaccharomyces pombe exhibits some characteristics with close proximity to higher eukaryotes which makes this yeast a very attractive host for heterologous protein production: (1) the transcription initiation mechanism is more similar to that of higher eukaryotes, (2) some mammalian promoters are functional in S. pombe, (3) the capability of RNA-splicing, highlighting in a similarity of components of the splicosome to that of mammalians, (4) the mammalian endoplasmatic reticulum retention signal KDEL can be recognised, (5) the existence of galactose residues in glycoproteins and (6) some other posttranslational modifications like acetylation and isoprenyla-tion of proteins are performed in a more similar way to mammalian than yeast cells. Several of the above-mentioned features might increase the importance of S. pombe in recombinant protein production in the near future in respect to production of authentic heterologous proteins and high-throughput applications thereof, like structural and functional genomics.<br>
All microorganisms possess mechanisms to adapt their metabolism for optimal utilisation of nutrients available in the environment. Fast and accurate adaptation to these environmental constraints is the major factor controlling growth and other physiological parameters of all organisms. For yeast, as for most microorganisms glucose is the preferred carbon and energy source. Therefore it is not surprising that glucose, the most abundant monosaccharide in nature, is a major messenger for cells affecting growth and development of these organisms by regulation of gene expression, mainly, but not exclusively, on the transcriptional control level. Genomic transcription analys-<br>
is revealed that a considerable amount of genes is regulated by the environmental determined glucose level. Genes with known metabolic function in glucose utilisation like low-affinity glucose transporters and glycolytic enzymes as well as genes encoding ribosomal proteins are induced by glucose. On the other hand glucose represses a large set of genes, including genes involved in utilisation of alternative carbon sources, gluconeo-genesis, the glyoxylate cycle, peroxisomal functions and respiration. Repression of respiration (Crabtree effect) occurs only in a few yeast species (fermentative yeasts, Crabtree positive) like Saccharomyces cerevisiae while in the majority of yeast species glucose does not repress respiration (Crabtree negative). Although a broad knowledge on the glucose repression machinery was achieved over the last 20 years, mainly based on the yeast Sacaharcanyces cerevisiae, its actual mechanism, especially the upstream parts of glucose sensing and signalling, is not fully understood. Nevertheless, to get a better understanding of the present work, a few main players of carbon catabolite repression as described for S. cerevisiae are described briefly below.<br>
The SNF1 gene encodes for a Ser/Thr protein kinase which can be found in high molecular mass complexes in yeast cells. It is regulated by conformational changes within the complex caused by phosphorylation in the regulatory subunit of Snflp. To date 3 upstream kinases (Paklp, Elmlp and TosSp) are identified to phosphorylate and therefore activate Snflp. Its activity is absolutely required for the derepression of a wide variety of genes repressed by glucose. Hence it is not surprising that Snflp or homologues are widely conserved in eukaryotes.<br>
The zink finger protein Miglp is able to bind to promoter regions of a wide variety of genes repressed by glucose. It is acting most probably by recruiting the general represser complex Ssn6(Cyc8)-Tuplp. The function of Miglp is controlled by the protein kinase Snfl, yet there is no clear evidence for a direct phosphorylation. Miglp is localised in the nucleus in its non-phosphorylated form. Glucose depletion causes phosphorylation of Miglp followed by translocation to the cytoplasm. When glucose is added to the cells Miglp quickly moves back to the nucleus and represses transcription.<br>
Adrlp also belongs to the family of zink finger proteins and<br>
was found to be a positive effector of peroxisomal proteins and the ADH2 gene, encoding for the glucose repressed alcohol dehyd-rogenase II. ADR1 expression is downregulated by glucose through the cyclic AMP (cAMP)-dependent protein kinase at high cAMP levels. The main regulatory effect appears at the mRNA translation level, but regulatory effects on transcription as well as mRNA stability were also observed, depending on the S. cerevisi-ae strain analysed.<br>
For a large number of genes including many of the genes involved in respiratory metabolism transcription is activated on non-fermentable carbon sources by the Hap2/3/4/5 complex. For a few genes involved in respiration like CYC1 (encoding for iso-1-cytochrome c) and COX6 (cytochrome c oxidase subunit VI) it has been established that Snfl is required for derepression after growth- on glucose. Transcription of HAP4 is repressed when glucose is present, nonetheless a direct involvement of either Hap4p or Snflp in derepression could not be shown.<br>
Gcrlp is a major transcription activator protein of glyco-lytic genes (e.g. enolase, glyceraldehyde 3-phosphate dehydro-genase). Gcrlp, together with the general transcription factor Raplp is the principal item of glycolytic gene expression in respect to coordination of transcription and it is absolutely necessary for high level expression. Genomic expression pattern of wild-type and S. cerevisiae gcrl mutant growing on various carbon sources revealed 53 open reading frames (ORFs), including genes of the glycolysis, as Gcrlp dependent.<br>
This description of some transcription factors and of the Snflp and Miglp pathway should give a short overview of some players in the glucose repression network. It should be noticed that there are more regulatory cycles than the Snflp-pathway for glucose repression. Although a broad knowledge on glucose sensing and signalling has been achieved the last 20 years, major questions still remain unanswered: what is the nature of the glucose signal and how are the known signalling pathways regulated and integrated.<br>
A limited number of yeast species is able to grow on methan-ol as sole carbon and energy source. They are belonging to one of the four genera Pichia, Hansenula, Candida and Torulopsis and share a general methanol utilisation pathway which is expressed after derepression or induction with methanol (see 1.3.1). Since<br>
initial reactions of this pathway are compartmentalised within peroxisomes, these organelles are also induced. Due to the strong induction of peroxisomes, the yeasts Candida boidinii, Pichia methanolica, Pichia paatoris and Hansenula polymorpha were frequently used in cell biology to study peroxisome biogenesis and function.<br>
As mentioned above methylotrophic yeasts share a common methanol utilisation pathway. The first step is the oxidation of methanol to formaldehyde and hydrogen peroxide, catalysed by alcohol oxidases (AOX, EC 1.1.3.13). The toxic H202 is disarmed to oxygen and water by the action of catalase. Both enzymes are sequestered in peroxisomes. Formaldehyde is either oxidised by two subsequent dehydrogenase reactions or assimilated in the cell metabolism by the condensation with xylulose 5-phosphate (Xu5P). Formaldehyde is oxidised to formate and further on to carbon dioxide by a glutathione (GSH)-dependent formaldehyde dehydrogenase and a formate dehydrogenase, both localised in the cytosol. NADH, generated in both reactions, is used to produce energy for growth on methanol. The condensation reaction takes place within the peroxisomes and is catalysed by the above mentioned transketolase dihydroxyacetone synthase. The resulting C3-compounds dihydroxyacetone (DBA) and glyceraldehyde 3-phos-phate (GAP) are further metabolised in the cytosol. After a phosphorylation of DHA, fructose 1,6-bisphosphate (FBP) is formed by an aldolase reaction of dihydroxyacetone phosphate (DHAP) and GAP. FBP is converted to fructose 6-phosphate by a phosphatase and xylulose 5-phosphate (Xu5P) is regenerated in the pentose phosphate pathway. One third of the GAP generated enters the gluconeogenesis pathway for cell constituent synthesis.<br>
The key enzymes of the methanol utilisation pathway, alcohol oxidase and formate dehydrogenase, are produced at very high levels after induction with methanol. Alcohol oxidase can account for more than 30% of the total soluble protein, dihydroxyacetone synthase and formate dehydrogenase up to 20%. The peroxisomes, which are also induced, can account for about 80% of the cell volume. Promoter sequences of several methanol utilisation genes were developed for recombinant protein production. Among others, these strong and inducible promoters are a main reason for the wide use of Pichia pastoris and Hansenula polymorpha as<br>
protein production hosts.<br>
In H. polymorpha and C. boidinii one gene encodes for an alcohol oxidase: MOX (methanol oxidase, H. polymorpha) and AODl (alcohol oxidase, C. boidinii). 2 genes were found in the two Pichia species P. pastoris (AOX1 and AQX2) and P. methanolica (AUG1 and AUG2r alcohol utilising gene, or MODI and MOD2), with Aoxlp and Auglp being the main alcohol oxidase. Comparison of coding regions revealed 73-85% similarity on the amino acid level between the methylotrophic yeasts [1]. The homology between the P. pastoris AOX1 and AOX2 ORFs (open reading frames) is 92% and 97% on the nucleotide and the amino acid sequence levels, respectively [2, 3]. Alcohol oxidase is an octameric flavoprotein containing one non-covalently bound FAD or a modified analogue (mFAD) per subunit. AOX translation occurs on free ribosomes followed by a posttranslational import into peroxi-somes. Translocation into peroxisomes is targeted by a PTS1 (type 1 peroxisome targeting signal) sequence at its extreme C-terminus. Aox oligomers are formed only after the import into the peroxisomal matrix.<br>
In C. boidinii and P. pastoris no Aox oligomers could be found in the cytosol in contrast to the dihydroxyacetone syn-thase, which forms a dimer in the cytosol prior to its translo-cation into the peroxisomal matrix. Not only the alcohol oxidase 1 promoter sequence of Pichia pastoris, but also the enzyme is of biotechnological interest due to a broad substrate range (un-saturated and saturated primary alcohols with short to moderate chain length) and a high stability under various reaction conditions. Regulation of all alcohol oxidase genes occurs on the transcription level and most probably at the transcription initiation stage. Although AOX1 and AOX2 are regulated similarly (mRNA not detectable on glycerol or glucose, detectable at carbon starvation phase, high amounts on methanol), their 5 -flanking regions share no significant homology [2, 4].<br>
Each AOX locus exhibits a putative RNA polymerase binding site (TATAAA; Goldberg-Hogness or TATA box) at position -43 relative to the primary transcription initiation site. Both P. pastoris AOX mRNA leader sequences are extremely rich in A residues and unusually long for yeasts (115 nucleotides (nt) for AOX1 and 160 nt for AOX2). The translation initiation regions around the ATG start codon (Kozak sequence; AOX1: CGAAACG ATG GCT, AOX2:<br>
GAGAAAA ATG GCC) are consistent with previously described consensus sequences for S. cerevisiae and higher eukaryotes. The physiological role of the second alcohol oxidase gene in P. pas-toris and P. methanolica is still obscure. Disruption of AOX1 or AUG1 causes severe growth defects in these strains (the so-called methanol utilisation slow (Mut8) phenotype) while aox2 and aug2 strains show comparable growth rates to the wild-type strain. 9 multiple forms of alcohol oxidase were observed in P. methanolica representing a random oligomerisation of the 2 gene products Auglp and Aug2p. AUG1 and AUG2 are regulated differentially: at carbon starvation and low methanol concentration only Auglp could be detected, and with increasing methanol concentration the Aug2p to Auglp ratio increases. The shift to octamers with elevated Aug2p content is due to an increase in AUG2 expression, regulated on the transcription level. Km values for methanol of the two homooctamers of Auglp and Aug2p are about 0.56 an 5.6 mM, respectively. Together with the finding, that disruption of AUG1 causes a growth defect at low methanol concentrations [5), these results implicate that AUG2 is an advantage for P. methanolica when growing at higher methanol concentrations. In Pichia pastoris neither the role of the AQX2. gene was analysed in further detail nor were favourable conditions for possessing a second alcohol oxidase gene found. Since laboratory conditions represent only a very small fraction of conditions free-living microorganisms are confronted, there should be situations in nature where the AOX2 gene is of selective importance to P. pastoris.<br>
C. boidinii AOD1 and H. polymorpha MOX expression is strictly repressed during growth on glucose or ethanol as sole carbon source, derepressed on glycerol and strongly induced on methanol. Expression of these two enzymes is also repressed when glucose and methanol are present in the medium. If glycerol is present methanol is able to induce gene expression. Transcription of AOD1 and MOX is also derepressed at carbon starvation and repressed when ethanol is present [6-9]. Two distinct regulatory mechanisms are responsible for repression of the methanol utilisation metabolism by ethanol or glucose [10, 11]. In Pichia pastoris the situation is significantly different: AOX1 is repressed when glucose, ethanol or glycerol is present in the media (in non-growth limiting concentrations). Derepression at<br>
carbon starvation and induction by methanol are similar to AOD1 and MOX. Carbon sources with which AOX1 expression is derepressed are e.g. sorbitol, mannitol, trehalose and alanine [12].<br>
Upon shift from methanol to a repressing carbon source like glucose or ethanol, peroxisomes are degraded within hours during adaptation to the new carbon source. Proteolytic degradation in the yeast vacuole again follows two distinct mechanisms when adapted to glucose or ethanol, called micro- and macroautophagy, respectively.<br>
As mentioned above, the methylotrophic yeasts Pichia pastor is and Hansenula polymorpha are widely used for recombinant protein production. Op to now, more than 500 proteins have been produced in P. pastoris. Their development was driven by a few characteristics, which brings them advantages among recombinant expression hosts: 1) they share the general advantages of yeasts in terms of genetic manipulation and cultivation technology (laboratory- and large-scale); 2) the ability to grow to extremely high cell densities; and 3) the high-level production of recombinant protein (secreted or intracellular). The strong in-ducible promoters of genes encoding for reactions of the methanol utilisation pathway were developed for recombinant protein production. The most widely used ones are the promoter regions of the alcohol oxidase genes AOX1 and MOX of P. pastoris and H. polymorpha, respectively. But also other promoter regions of the methanol utilisation pathway genes were used to drive recombinant protein production: FMD (formate dehydrogenase) and DAS1 (dihydroxyacetone synthase) promoters in H. polymorpha and C. boidinii and the FLD1 (formaldehyde dehydrogenase) promoter in P. pastoris. The latter one can also be induced with methylamine as sole nitrogen source with glucose as a carbon source. Promoters for constitutive expression of foreign genes are also available: the GAP (glyceraldehyde 3-phosphate dehydrogenase) promoter element in P. pastoris and the PMA1 (encoding for the plasma membrane H+ -ATPase) promoter in H. polymorpha. Several auxotrophic host strain/marker gene-combinations were developed for P. pastoris (e.g. HIS4) and H. polyraorpha (e.g. LEU2 and URA3) . Dominant selection markers are also available (e.g. Zeocin™, G418 resistance). Gene integration into methylotrophic yeasts is done mainly (if not exclusively) by homologous integ-<br>
ration. Vectors bearing an ARS (autonomously replicating sequence) region are also available but they are usually quite unstable if selection pressure is released which results in their limited technological application. In P. pastoris the foreign gene is integrated site-specifically either in the AOX1 or the HIS4 locus. Other possible integration sites are e.g. the GAP locus (for GAP expression) or any other selection marker loci (e.g. ADEl, URA3, ARG4 and LEU2). In H. polymorpha expression cassettes are randomly integrated in a head-to-tail arrangement leading to mitotically stable integrants with a high copy-number (up to 100). However, a high copy-number often does not result in a high-level expression. Additional factors of great influence are: structure of the integration cassette, nature and structure of the protein to be expressed and the integration site. Especially the integration cassette structure is of great influence on the effect of gene dosage. A further discussion how to optimise the expression cassette and the gene dosage is given in [13, 14] . Methylotrophic yeasts are belonging to the group of Crabtree negative yeasts thus ethanol production occurs at very low level when grown under aerobic conditions. Due to this fact these yeasts can be grown to very high cell densities in fer-menter cultures resulting in high product yields. AOX1 driven protein production can be further increased 3-5 times when the methanol concentration in the bioreactor is in growth-limiting spheres. The fact that P. pastoris secretes under standard conditions only low amounts of endogenous proteins makes every secreted recombinant protein the most abundant in the medium. Secretion can serve as a substantial first step in the downstream purification process. For protein secretion, the S. cerevisiae MFαl (mating factor a) prepro leader sequence and sequences derived from the acid phosphatase (PH01) are widely used in P. pastoris and H. polymorpha. In some cases sufficient secretion was obtained with plant, fungal and mammalian proteins bearing their natural secretion signals. As mentioned above, yeasts are capable of performing posttranslational modifications like disulfide bond formation, processing of signal sequences (e.g. prepro leader sequence of MFal), lipid addition and N- and 0-linked glycosylation. While in mammalian cells highly complex N- and 0-linked oligosaccharide-structures composed of a variety of sugars (e.g. N-acetylglucosamine, galactose, and sialic acid)<br>
are produced, most yeasts generate high mannose type structures lacking some sugar entities like galactose or sialic acid. These non-mammalian structures can result in severe problems for therapeutic application mainly due to their high potential ira-munogenicity. In H. polymorpha and P. pastor is, in contrast to S. cerevisiae, hypermannosylation is less abundant and no hyper-immunogenic terminal a-1,3- linked mannoses are incorporated in N-linked oligosaccharides. To overcome the problems of immuno-genicity (and some others like low stability in the blood flow) efforts are on the way to humanise yeast-derived oligosaccharide structures, and, as recent literature reveals, especially in P. pastor is. To date, the vast majority of research issues and commercial processes rely on the wellknown yeast S. cerevisiae. Due to increasing knowledge about non-conventional yeasts, together with the apparent advantages in terms of large-scale fermentation and glycosylation issues, H. polymorpha and P. pastoris are rapidly becoming the yeast of choice. This is emphasised by the fact that several production processes were implemented in industry.<br>
In the WO 02/081650 the identification of AOX1 promoter regions is disclosed, which may be used for the construction of mutant AOX1 promoters. Since the deleted sequence regions of the AOX1 promoter disclosed therein are very long, the accumulated effect and not the single effects of the distinct regulatory sequences of the promoter can be observed. However, such an approach will not allow the development of strongly enhanced promoters. Especially when constructing new promoters having enhanced features by deleting or duplicating parts of the original promoter the knowledge of the exact regulatory sequence range is required.<br>
It is an object of the present invention to provide an improved AOX1 promoter with enhanced properties in order to facilitate downstream processing in protein production, to increase time-space-yields and to help to upgrade product quality.<br>
Another object is to provide a strong AOX1 promoter in a vector or a host strain which anticipates partly or entirely glucose repression. It is of advantage to have a promoter which drives strong expression in presence of high glucose concentrations .<br>
A further object of the present invention is to provide an<br>
AOX1 promoter which allows the production of a protein employing a reduced amount of methanol or without methanol. Such promoters would have a significant impact on industrial production processes. Due to safety issues special equipment is needed for production plants employing methanol as an inductor. This contradicts Pichia pastoris applications in many less specialised production plants. In addition protein stability in presence of methanol can hamper methanol based induction of protein expression. This is less critical for the production of robust industrial proteins, but becomes a major issue for e.g. secreted therapeutical proteins.<br>
The construction of such promoters requires the knowledge of specific portions (e.g. regulatory elements, transcription factor binding sites) of the wild-type Pichia pastoris AOX1 promoter which - when mutated somehow - show an effect on the expression behavior. Therefore it is an object of the present invention to identify these portions and to provide therefore the means to create AOX1 promoters with enhanced features.<br>
Therefore the present invention relates to a mutant Pichia pastoris alcohol oxidase 1 (AOX1) promoter of the wild type Pichia pastoris AOX1 promoter (SEQ ID No. 1) comprising at least one mutation selected from the group consisting of:<br>
a)	a transcription factor binding site (TFBS),<br>
b)	nucleotides 170 to 235 (-784 to -719), nucleotides 170 to<br>
191 (-784 to -763), nucleotides 192 to 213 (-762 to -741), nuc<br>
leotides 192 to 210 (-762 to -744), nucleotides 207 to 209 (-747<br>
to -745), nucleotides 214 to 235 (-740 to -719), nucleotides 304<br>
to 350 (-650 to -604), nucleotides 364 to 393 (-590 to -561),<br>
nucleotides 434 to 508 (-520 to -446), nucleotides 509 to 551<br>
(-445 to -403), nucleotides 552 to 560 (-402 to -394), nucle<br>
otides 585 to 617 (-369 to -337), nucleotides 621 to 660 (-333<br>
to -294), nucleotides 625 to 683 (-329 to -271), nucleotides 736 to 741 (-218 to -213), nucleotides 737 to 738 (-217 to -216), nucleotides 726 to 755 (-228 to -199), nucleotides 784 to 800 (-170 to -154) or nucleotides 823 to 861 (-131 to -93) of Seq ID No. 1, and combinations thereof. The (negative) numbers in parenthesis throughout the description reflect the corresponding positions of the promoter in relation to the translation start codon (e.g. ATG). For instance, "A" of "ATG" in a nucleic acid sequence comprising NxGACTATGNy corresponds to the position +1,<br>
whereas "T" before "A" of "ATG" corresponds to position -1.<br>
According to the present invention the mutant AOX1 promoter comprises at least one mutation within a transcription factor binding site and/or one of the nucleic acid sequence ranges outlined above. It turned out that especially these regions of the AOX1 promoter are suited to modify said promoter in order to alter its features. Of course also a combination of the mutations outlined above may be introduced to enhance the characteristic features of an AOX1 promoter (e.g. two TFBS mutations selected from a), one TFBS mutation selected from a) and one mutation selected from b), one mutation selected from a) and two mutations selected from b)). For instance, a mutation of a TFBS may be combined with a mutation within nucleotides 737 to 738 (-217 to -216) and/or nucleotides 207 to 209 (-747 to -745) of Seq ID No. 1. The expression of a protein under the control of an AOXl promoter in Pichia pastoris is induced generally by the addition of methanol and inhibited by the presence of glucose in the medium. In order to enhance or to reduce the effect of said medium additives on the protein expression, the promoter is preferably mutated in the promoter regions as outlined above. The efficacy of the mutated AOXl promoters to produce a protein of interest varies depending on the amount (i.e. copies) of vector integrated into the chromosome of the host. Especially multicopy strains turned out to show enhanced promoter effects. Since the antibiotic resistance of Pichia strains depends on the number of antibiotic resistance cassettes (vectors introduced into a host comprise preferably an antibiotic resistance cassette allowing the host to grow on/in a medium comprising an antibiotic as selective marker) integrated into the chromosome of said host, multicopy strains may be produced by applying increasing concentrations of antibiotic (within the range of lOμg/ml to lOmg/ml, preferably 50μg/ml to lOOOμg/ml; depending on the antibiotic used; for instance, geneticin: 0.1 to lOmg/ml, preferably 0.2 to 5mg/ml, particularly 0.25 to 4mg/ml, zeocin: 10 to 5000μg/ml, preferably 50 to 3000μg/ml, particularly 100 to 2000μg/ml) onto the selective agar plates to increase the selection pressure (e.g. [14]; Scorer, C.A. et al. (1994) Bio/Technology 12:181-184) . However, it was found that the growth of cells harbouring a multiplicity of antibiotic resistance cassettes is not only dependent on the concentration of the antibiotic but also time<br>
dependent. Therefore, multicopy strains are able to grow to a detectable colony on a medium containing the same concentration of antibiotic in a shorter period of time than singlecopy strains. This behaviour enables the person skilled in the art to detect and to isolate multicopy strains before singlecopy strains begin to grow. For instance, a strain harbouring one single copy of an antibiotic resistance cassette grows on an agar plate to a detectable colony size in 72h, whereas the same strain harbouring more than one copy of said cassette grows in 24 to 48h to the same size.<br>
Especially multicopy strains harbouring an AOX1 promoter with mutations within nucleotides 694 to 723 (-260 to -231) and within nucleotides 737 to 738 (-217 to -216) showed surprisingly enhanced expression rates.<br>
In order to increase the protein expression efficiency of a host in the presence of methanol, nucleotides 170 to 235 (-784 to -719) of the AOX1 promoter (SEQ ID No. 1) are preferably mutated. A mutation in this region increases the protein expression to 120 to 130 % compared to the wild type AOXl promoter, provided that the plasmid carrying the mutant AOX1 promoter is only once integrated into the chromosome/genome of the host (single copy mutant). However, mutations within all other above mentioned regions reduce or does not affect the efficacy of methanol to induce protein expression. In contrast thereto, mutations in the promoter regions of the wild type AOXl promoter (as outlined above) lead - depending on the mutation - to increased and decreased protein expression under derepression conditions (e.g. see Table 13, example 1) .<br>
However, recombinant strains harbouring more than one copy of mutated AOXl promoters result in strains having an enhanced activity under derepression and methanol induced conditions (multicopy strains, e.g. see Fig. 7, example 2). In detail, multicopy strains harbouring mutations within nucleotides 694 and 723 of SEQ ID No. 1 (d6), within nucleotides 694 and 723 (-260 and -231) of SEQ ID No. 1 (d6), within nucleotides 694 and 723 (-260 and -231) and within nucleotides 304 and 350 (-650 and -604) of SEQ ID No. 1 (d2d6), within TFBS, especially within Rapl, Gcrl, QA-1F, Hsf_l, Adrl, Hsf_2, MatlMC, abaA and Hap2345, show an increased expression under derepression conditions and/or under methanol induction compared to the expression of<br>
proteins under the control of the wild type AOX1 promoter. Under derepression conditions some of these multicopy strains show protein expressions which are about 10 fold increased compared to expressions under the control of the wild type promoter. In presence of methanol as inductor the expression efficiency is more than 5 times enhanced when a promoter according to the present invention is employed. Therefore, these mutations, especially when present in the host in a multicopy form, are preferably employed for the expression of proteins. The combination of two or more of the above mentioned mutations may further enhance the promoter strength (see e.g. Fig. 7, example 2).<br>
The transcription factor binding sites may be identified experimentally (e.g. by mobility shift or footprint analysis) or by sequence comparison with known transcription factor binding sites (e.g. by computer analysis, [15]).<br>
The knowledge of promoter regions which influence the strength and the characteristics of said promoter may be used to design promoters with distinct properties (high protein expression under derepression conditions and/or high protein expression in presence of methanol). Furthermore these properties may be enhanced or altered if these mutant promoters are integrated one or more times into the genome of the host (e.g. see examples 1 to 3).<br>
However, in some cases the promoter activity should be decreased instead of increased. Especially the co-expression of regulatory proteins like kinases, phosphorylases and helper proteins, such as e.g. chaperones, protein disulfide isomerase, cis-trans isomerases, foldases, protein disulfide isomerases and proteases, is in many cases required to be low in comparison to the main product, which may be produced by the cell under the wild-type promoter or under an enhanced promoter according to the present invention. Especially the combined expression of two different products (e.g. a helper protein and the main product) under the control of an AOX1 promoter with increased activity and an AOX1 promoter with decreased activity (in comparison to the wild-type activity), respectively, turned out to be advantageous, because the expression rate of the main and the secondary product differs even more instead of using wild-type AOX1 promoters. Reduced expression may be preferably obtained by deleting activator binding sites like HSF or HAP or by inserting<br>
represser binding sites into the wild-type AOX1 promoter. Hence, the use of AOX1 promoter with reduced activity prevents the overloading of the protein expression machinery of the cell, which would have the consequence, that the yield of the main product would be reduced. For instance, Bessette PH et al. (PNAS USA (1999) 96:13703-13708) could show that the expression of an active polypeptide could be increased significantly by the co-expression of a thioredoxin.<br>
According to a preferred embodiment the promoter comprises further a mutation within nucleotides 694 to 723 (-260 to -231) and/or nucleotides 729 to 763 (-225 to -191) of Seq ID No. 1.<br>
A mutation affecting these nucleotide ranges in combination with a mutation as outlined above results in even more enhanced promoter activity. For instance, a double mutation affecting nucleotides 694 to 723 (-260 to -231) and nucleotides 737 to 738 (-217 to -216) of Seq ID No. 1 lead to a promoter showing higher expression levels under derepression as well as under induced conditions compared to the expression levels under the same conditions of the wild type promoter. The effect of this double mutation can be enhanced when the nucleic acid comprising the promoter is introduced in the cell in more than one copy (resulting in a multi copy clone).<br>
The mutation is preferably a deletion, a substitution, an insertion, an inversion and/or a multiplication.<br>
In order to modify the characteristics of the wild type AOX1 promoter of Pichia pastoris several mutation types are possible. The promoter stretches comprising the above mentioned regions (transcription factor binding sites (TFBS), nucleotides 170 to 235 (-784 to -719), 170 to 191 (-784 to -763), 192 to 213 (-762 to -741), 192 to 210 (-762 to -744), 207 to 209 (-747 to -745), 214 to 235 (-740 to -719), 304 to 350 (-650 to -604), 364 to 393 (-590 to -561), 434 to 508 (-520 to -446), 509 to 551 (-445 to -403), 552 to 560 (-402 to -394), 585 to 617 (-369 to -337), 621 to 660 (-333 to -294), 625 to 683 (-329 to -271), 694 to 723 (-260 to -231), 729 to 763 (-225 to -191), 736 to 741 (-218 to -213), 737 to 738 (-217 to -216), 726 to 755 (-228 to -199), 784 to 800 (-170 to -154) or 823 to 861 (-131 to -93) of Seq ID No. 1) may be partially or completely deleted, partially or completely substituted with other nucleotides or nucleic acid sequences, disrupted by insertion of single nucleotides or nuc-<br>
leic acid sequences, inverted partially or completely or multiplied. All these mutations lead to a change in promoter activity, because structural features and/or recognition/binding sites for e.g. transcription factors are affected by said mutations. However, these changes may lead to an increased or decreased activity of the promoter compared to the wild type promoter .<br>
It is well known in the prior art that the<br>
multiplication/duplication of specific nucleic acid stretches may increase the promoter activity. The regulation of gene expression of many eukaryotic promoters, especially yeast promoters, involves multiple interactions between transcription factors bound within a promoter. Multiple sites may be required for the functioning of even the smallest cis-acting elements. In yeast cells, upstream activator sequences (UAS) are necessary for transcription. They work in either orientation and at variable distance with respect to the TATA box and transcription start site, but in contrast to enhancers in higher eukaryotes, they must be upstream from these basal elements. UAS are targets of several transcriptional activators.<br>
Most repression phenomena in yeast cells result from the in-activation or absence of transcription factors. However, some negative regulatory sites (upstream repression sequences (ORS)) could also be identified.<br>
Based upon deletion analysis of the P. pastoris AOX2 promoter three regulatory regions were found, two negative acting regions (URS1 and URS2) and a positive acting domain (UAS) [3] . For the H. polymorpha MOX promoter two upstream activating sequences (UAS1 and UAS2) and one represser binding site (URS1) were also described [8], Corresponding sequences could also be identified on AOX1 promoters (nucleotides 585 to 614 (-369 to -340) and 725 to 756 (-229 to -198), showing similarities to AOX2 UAS [3], as well as nucleotides 622 to 656 (-332 to -298) [8]). The multiplication (2, 3, 4, 5, 6 or 7 times UAS) of these nucleic acid stretches may result in a promoter with an enhanced strength leading to even more powerful protein expression. Therefore the construction of promoters comprising multiple UAS, preferably involving the above mentioned sequence regions similar to the AOX2 and MOX UAS, or other multiple sequence stretches (e.g. the nucleic acid sequences ranges outlined above) falls<br>
also within the scope of the present invention and is considered to be a preferred embodiment. An activating sequence is usually within a few hundred basepairs of a promoter. For example, most activating -sequences are within about 200 to 400 basepairs of the promoter that is enhanced. Further upstream the promoter usually contains further enhancers and transcription factor binding sites.<br>
At least one mutation of the AOX1 promoter may be introduced by standard methods known to a person skilled in the art (e.g. Molecular Cloning: A Laboratory Manual (Third Edition), J. Sam-brook and D. Russell, 2001, Cold Spring Harbor Laboratory Press).<br>
According to a preferred embodiment of the present invention the transcription factor binding site (TFBS) is selected from the group consisting of Hapl, Hsf, Hap234, abaA, Stre, Rapl, Adrl, MatlMC, Gcrl and QA-1F.<br>
The mutation of at least one of these TFBS results in mutant promoters with varying characteristics (see example 2) .<br>
Preferably, the transcription factor binding site (TFBS) Hapl comprises nucleotides 54 (-900) to 58 (-896) of Seq ID No. 1, Hsf nucleotides 142 (-812) to 149 (-805) and 517 (-437) to 524 (-430) of Seq ID No. 1, Hap234 nucleotides 196 (-758) to 200 (-754), 206 (-748) to 210 (-744) and 668 (-286) to 672 (-282) of Seq ID No. 1, abaA nucleotides 219 (-735) to 224 (-730) of Seq ID No. 1, Stre nucleotides 281 (-673) to 285 (-669) of Seq ID No. 1, Rapl nucleotides 335 (-619) to 339 (-615) of Seq ID No. 1, Adrl nucleotides 371 (-583) to 377 (-577) of Seq ID No. 1, MatlMC nucleotides 683 (-271) to 687 (-267) of Seq ID No. 1, Gcrl nucleotides 702 (-252) to 706 (-248) of Seq ID No. 1 and QA-1F nucleotides 747 (-207) to 761 (-193) of Seq ID No. 1.<br>
These TFBS may be identified experimentally or by comparison with known TFBS of other promoters (e.g. promoters of euka-ryotes) with the assistance of computer programmes (e.g. see example 1) .<br>
A summary of the influence of mutant AOX1 promoters on the expression of proteins, peptides or functional nucleic acids is provided in table 2 (in comparison to the wild-type activity) .<br>
Table 2: Influence of mutants of the wild-type AOX1 promoter on the expression of proteins, peptides or functional nucleic<br>
Acids <br>
(Table Removed)<br>
1 Expression rate in comparison to the decreased, + increased<br>
Another aspect of the present invention relates to a nucleic acid molecule comprising a mutant Pichia pastoris alcohol oxi-dase 1 (AOX1) promoter according to the present invention and a nucleic acid encoding a protein, peptide or functional nucleic acid, wherein the promoter and said nucleic acid are operably linked together.<br>
The mutant AOX1 promoter can be linked to a gene encoding for a protein (e.g. enzyme), a peptide (e.g. hormone) or functional nucleic acid (e.g. siRNA). The resulting nucleic acid fragment may be used to express e.g. a protein when introduced into an organism, preferably a yeast, especially a Pichia pas-toris strain. The construction of said nucleic acid molecule is well known to the person skilled in the art and can be performed with standard molecular biological methods (e.g. Molecular Cloning: A Laboratory Manual (Third Edition), J. Sambrook and D.<br>
Russell, 2001, Cold Spring Harbor Laboratory Press; manual "Pi-chia Expression Kit", Invitrogen Corp.).<br>
"Operably linked" refers to a first sequence(s) being positioned sufficiently proximal to a second sequence(s) so that the first sequence(s) can exert influence over the second sequence(s) or a region under control of that second sequence. For instance, a promoter can be operably linked to a gene so that the gene will be expressed under the control of the promoter, which would typically be 5' to the gene. Usually, a core promoter would be within a few hundred base pairs from the start site of translation. About 30 bp downstream there is usually a downstream promoter element.<br>
Another aspect of the present invention relates to a vector comprising a mutant Pichia pastor is alcohol oxidase 1 (AOX1) promoter according to the present invention or a nucleic acid molecule as outlined above.<br>
In order to introduce the mutant promoter, optionally operably linked to a nucleic acid encoding for a protein, peptide or functional nucleic acid, into a host, preferably into a methylo-trophic yeast strain (e.g. a Pichia pastoris strain), said promoter-has to be provided in a vector, which may be used for the transformation of said host. For instance, said vectors may be yeast episomal plasmids (YEp), yeast integrative plasmids (Yip) or yeast artificial chromosomes. Such vectors comprise usually an origin of replication (if amplification in microbial hosts is needed) and a selection marker for the propagation of the vectors in E. coli, promoters and terminators for the recombinant protein expression in yeast and selection markers for yeast. Non-integrative vectors further comprise an autonomous replicating sequence (ARS), which ensures the stability of the vector in the cell (e.g. Myers, A. M., et al. (1986) Gene 45: 299-310). Integrative vectors, which do not harbour AR sequences, comprise sequence regions which are homologous to regions of the genome. Alternatively linear DNA, e.g originating from PCR can be used for transformation.<br>
Another aspect of the present invention relates to a cell comprising at least one mutant Pichia pastoris alcohol oxidase 1 (AOX1) promoter, at least one nucleic acid fragment or at least one vector as disclosed herein. The introduction of a nucleic acid molecule harbouring a mutant AOX1 promoter (e.g. vector,<br>
wherein the promoter is operably linked to a nucleic acid encoding for a protein) into a host may be done e.g. by electropora-tion. Said nucleic acid molecule is integrated into the chromosome after its introduction into said host in a single copy or in multiple copies or present in the cell as a single copy or multicopy autonomous replicating plasmid. If several mutant promoters are used, they can all be linked with one single gene (coding for a protein or functional nucleic acid (e.g. Ribozyme, antisense RNA etc.), an identical protein or different proteins (e.g. 1 promoter variant is linked to a selection marker and another mutant promoter is linked to another protein which should be expressed). Therefore within the scope of the present invention singlecopy strains comprising one copy of the AOX1 promoter operably linked to a nucleic acid encoding for a protein, a pep-tide or a functional nucleic acid as well as multicopy strains comprising more than one, preferably at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15 or 20 copies of the AOX1 promoter operably linked to a nucleic acid encoding for a protein, a peptide or a functional nucleic acid are preferably produced.<br>
According to a preferred embodiment of the present invention said cell is a eukaryotic cell, in particular a yeast cell, preferably a methylotrophic yeast cell.<br>
Preferably the methylotrophic yeast cell is selected from the group consisting of Candida, Hansenula, Pichia and Toruplos-is, especially a Pichia pastoris cell.<br>
AOX1 promoters as well as mutated variants therefrom may be functionally introduced in a very large number of different yeast cells, including methylotrophic (e.g. Pichia pastoris) and non-methylotrophic (e.g. Saccharomyces cerevisiae) cells. The transferability of promoters to other organisms, especially of AOX1 and MOX promoters, is known to the person skilled in the art. Although the substrate specificity and some regulatory features are different in different yeasts (e.g. Pichia pastoris, Hansenula polymorpha and Saccharomyces cerevisiae) a recognition of foreign promoters was demonstrated (e.g. Raschke, W.C., et al., Gene, 1996. 177:163-7; Pereira, G.G. and C.P. Hollenberg, Eur J Biochem, 1996. 238:181-91). For instance, the H. polymorpha MOX promoter is recognised in S. cerevisiae, repressed in presence of glucose and de-repressed under carbon source limitation. Similarly the AOX1 promoter can be employed in H. poly-<br>
morpha and is regulated in the same way as the MOX promoter. The ZZA1 promoter, which is closely related to the AOX1 promoter could also be successfully employed in S. cerevisiae.<br>
Another aspect of the present invention relates to a kit for the expression of a selected protein or transcription to a functional RNA, comprising<br>
i) a vector as defined above, and<br>
ii) a cell capable to express said protein or functional RNA under the control of a promoter according to the present invention.<br>
The vector according to the present invention can be used in a kit for the expression of a selected protein or transcription of a functional RNA (e.g. ribozyme, antisense RNA, RNAi).<br>
According to a preferred embodiment of the present invention said cell is a yeast cell, preferably a methylotrophic yeast cell.<br>
Preferably the methylotrophic yeast cell is selected from the group consisting of Candida, Hansenula, Pichia and Toruplos-is, especially a Pichia pastoris cell.<br>
Another aspect of the present invention relates to a method for the expression of a recombinant protein, peptide or functional nucleic acid in a cell comprising the following steps:<br>
-	providing a vector or a nucleic acid molecule comprising an<br>
AOX1 promoter according to the present invention and a nucleic<br>
acid encoding for a protein, peptide or functional nucleic acid,<br>
said promoter being operably linked to said nucleic acid,<br>
-	transforming said cell with said vector or said nucleic acid<br>
molecule,<br>
culturing the transformed cell in a suitable culture medium, optinally inducing expression of said protein, peptide or functional nucleic acid and<br>
-	isolating said expressed protein, peptide or functional nuc<br>
leic acid.<br>
According to a preferred embodiment of the present invention said cell is a yeast cell, preferably a methylotrophic yeast cell.<br>
Preferably the methylotrophic yeast cell is selected from the group consisting of Candida, Hansenula, Pichia and Toruplos-is, especially a Pichia pastoris cell. Another aspect of the present invention relates to the use of a nucleic acid molecule,<br>
a vector or a cell according to the present invention for the expression of a protein, peptide or functional nucleic acid.<br>
When a nucleic acid molecule, a vector or a cell according to the present invention is used for the expression of a protein, peptide or functional nucleic acid it is advantageous to choose an appropriate AOX1 promoter which fulfils the requirements posed for the expression (e.g. high or constitutive expression under derepression conditions (= without the addition of glucose to the medium) or under methanol induced conditions). Suitable mutant AOX1 promoters can be selected with the help of table 2.<br>
Another aspect of the present invention relates to a method for the isolation of super expression clones comprising the steps:<br>
a)	introducing a nucleic acid molecule or vector comprising<br>
a mutated methanol inducible promoter, preferably an AOX1 pro<br>
moter, operably linked to a nucleic acid encoding for a protein<br>
or to a functional nucleic acid and a marker resistance gene<br>
into a cell,<br>
b)	transferring the cell of step a) to a medium comprising<br>
an appropriate selective marker, a non-repressing carbon source<br>
and methanol for the selective growth of super expression clones<br>
under inducing conditions or to a medium comprising an appropri<br>
ate selective marker and a non-repressing carbon source without<br>
methanol for the selective growth of super expression clones un<br>
der derepressing conditions,<br>
c)	incubating the cell from step b) on said medium,<br>
d)	isolating a colony of the cell obtained from step c) and<br>
e)	detecting super expressing clones by determining the ex<br>
pression rate of said cell.<br>
The construction of super or high expression clones harbouring a vector or a nucleic acid comprising a mutated methanol inducible promoter requires methods enabling the person skilled in the art to isolate these clones. Such a method is provided herein. The first step of said method is the introduction of the promoter comprising nucleic acid (e.g. vector) into a suitable cell, which is able to regulate said promoter. The promoter itself may be mutated by genetic engineering or by chemical (e.g. bisulfite, nitrite, fumaric acid, hydrazine) or physical (e.g. radiation, especially UV radiation) mutagenesis. In a further<br>
step the cells harbouring said mutated promoter are transferred to a medium, preferably to a solid medium directly or via a liquid medium, which comprises an antibiotic (e.g. Zeocin) and sorbitol (or another non-repressing carbon source as described, e.g. in [12], in particular alanine, mannitol or trehalose) for the growth of high expression clones under derepression conditions and which comprises further methanol, if high expression clones under induced conditions should be discovered. By including glucose to the media together with methanol glucose non-repressed and methanol induced transformants might be isolated (to prevent methanol volatilisation the medium may be stored during incubation in a methanol saturated or methanol comprising atmosphere) . After the cultivation of the cells in or on a suitable medium, said cells are isolated from said medium and may be used for further analysis (e.g. determination of the exact expression rate, isolation of the promoter in order to analyse the changes in the nucleic acid sequence of the promoter compared to the wild type promoter). The non-repressing carbon sources used in the method according to the present invention and disclosed, e.g. in [12], are preferably employed in an amount of 0.1 to 10%, preferably in an amount of 0.2 to 5%, more preferably in an amount of 0.3 to 3%, in particular in an amount of 0.5 to 1%. A preferred non-repressing carbon source is selected from the group consisting of alanine, mannitol, sorbitol, trehalose, lactose and combinations thereof.<br>
The selection of suitable marker resistance gene depends on the marker used to select transformants. For instance, if Zeocin is used as marker the marker resistance gene to be introduced into the vector under the control of the mutant AOX1 promoter is the Sh ble gene. If the nucleic acid encodes for a protein or a peptide the resulting/expressed protein may be a fusion protein. It is especially advantageous to provide the marker resistance gene under the control of the mutant AOX1 promoter, because in such a case the expression rate of the marker resistance gene product depends also on the promoter strength and behaviour of the mutated promoter. For instance, a strong promoter responsible for high expression of the nucleic acid product will also increase the expression rate of the marker resistance gene product. Such clones have a selective advantage over clones with a promoter exhibtiting a reduced promoter strength. This allows<br>
the selection of super expression clones directly after the regeneration from the transformation of the cells.<br>
The expression rate is preferably determined by methods like gel electrophoresis (e.g. SDS-PAGE), antibody binding (e.g. ELISA), quantitative (reverse transcriptase) PCR (e.g. real time RT-PCR), enzymatic activity (e.g. if the expressed protein is an enzyme) or fluorometrically (protein with a characteristic emission spectrum like green fluorescent protein).<br>
Promoters (transformants) showing increased expression in absence of otherwise repressing C-sources (in the case of AOX1 promoter glucose) are selected by selective growth of transformed cells in/on media containing a non-repressing carbon source. Promoters (transformants) showing increased expression in absence of otherwise repressing C-sources (in the case of AOX1 promoter glucose) in presence of an inductor (e.g methanol) are selected by selective growth of transformed cells in/on media containing a non-repressing carbon source and the inductor (e.g. methanol) . The inductor can also be a non-repressing carbon source. Superexpressing clones are selected by combining a multicopy leading to higher resistance against antibiotics (e.g Zeocin) or higher productivity of an essential media component (e.g. Leu, His, Arg, Ura) with the regulatory selection described above.<br>
The media compositions to be used in a method according to the present invention may be obtained directly from manufacturers or distributors from kits, cells and vectors relating to Pi-chia Pastoris (e.g Invitrogen). The methanol concentration in the medium may be preferably 0.05 to 15%, more preferably 0.1 to 10%, particularly 0.3 to 5%. In the scientific literature different methanol concentrations for different cultivation conditions are described. For instance, shaking flasks may contain 1% methanol or less (Guarna MM, et al. (1997) Biotech. Bioeng. 56:279-286), fermentation processes may contain 0.5% methanol (Damascene LM, et al. (2004) Protein Expr Purif 37:18-26; Hell-wig S., et al. (2001) Biotechnol Bioeng 74:344-352; Hellwig S., et al. (1999) Biotechnol Appl Biochem 30:267-275).<br>
The enhanced expression of multicopy clones may depend not only on the presence of more than one copy of mutated promoter in a cell but also due to the fact that there is a lack of several transcription factors, because these factors may be bound<br>
to the high number of transcription factor binding sites in said cell. This could be shown by comparison of the expression rate under methanol inducing conditions with the expression rate under derepression conditions wherein it could be found that the enhanced expression rate is not only an effect of the copy num" ber of the mutated AOX1 promoter in the cell (no linear effect). For instance, strain d6*F10 shows such characteristics.<br>
The medium used to isolate super expression clones may comprise further media components like leucine, uracil, arginine, histidine and/or adenine and sorbitol may be exchanged by glucose in order to identify promoter variants which show a reduced repression in the presence of glucose compared to wild-type promoter variants.<br>
When auxotrophic strains are used, the cell may be transferred to a medium comprising sorbitol (or other no-repressing carbon sources) and containing individual media components (e.g. leucine, uracil, arginine, histidine and adenine) for the selective growth of super expression clones under derepressing conditions employing auxotrophy markers (step b)).<br>
The commonly used P(TEF)-Zeo resistance marker in AOX1 promoter comprising vectors leads to constitutive expression of the zeocin resistance protein and therefore allows the isolation of multicopy clones by resistance against higher concentrations of the antibiotic. The described new method allows to combine this effect with regulatory features to detect promoters and multicopy clones which lead to higher expression under certain controllable regulatory circumstances (e.g. derepressed expression, induced expression etc.). This makes it possible to dedect new promoters with altered regulatory properties and also clones where multicopy clones lead to enhanced expression under such special regulatory conditions.<br>
"Super expression clones" are expression clones which express more of a protein or of a functional nucleic acid under the control of the mutated promoter than under the control of the wild type promoter or more of a protein or functional nucleic acid than by applying vectors with usually used promoter-selection marker combinations such as P(TEF)-Zeo. The expression rate of the "super expression clones" according to the the present invention may be at least 20%, preferably at least 50%, more preferably at least 100%, particularly at least 500% in-<br>
creased compared to the expression rate of the same protein or peptide or functional nucleic acid under the control of the wild-type promoter (mean value plus two to three times standard deviation). "Super expression clones" may preferably comprise more than one copy of the mutated promoter or nucleic acid molecule according to the present invention. Alternatively, "super expression clones" may also be denominated "high expression clones".<br>
According to the present invention "methanol inducible promoters" are promoters whose activity is regulated by the presence of methanol in the culture medium. Such promoters are preferably AOX1 (from Pichia pastoris) or MOX (from Hansenula polymorpha) promoters or any other methanol inducible and glucose repressed promoter derived from methylotrophic yeasts such as e.g. FMD, PLDf DAS (e.g. see table 6, example 1).<br>
According to a preferred embodiment of the present invention the selective marker is an antibiotic, preferably zeocin.<br>
The selective marker to be used in the medium depends on the fact which molecular characteristic of the cell can be used to distinguish a cell harbouring the nucleic acid or vector comprising a mutated or wild-type methanol inducible promoter from a cell which does not harbour said nucleic acid or vector. Selective markers may therefore be antibiotics (the genes for antibiotic resistance can be found in the vector or nucleic acid introduced in said cell). To compensate auxotrophy of certain strains the selective marker in the medium may be a substance like leucine, uracil, arginine, histidine and adenine, depending on the type of auxotrophy.<br>
Preferably, the nucleic acid molecule, the vector and the cell are a nucleic acid, a vector and a cell according to the present invention.<br>
According to a preferred embodiment of the present invention the nucleic acid molecule or vector is introduced into the cell by transformation by standard methods known to a person skilled in the art, preferably electroporation, chemical transformation, protoplast fusion or by particle bombardment (see e.g. Current Protocols in Molecular Biology, John Wiley &amp; Sons, Edited by: Fred M. Ausubel et al.; Molecular Cloning: A Laboratory Manual (Third Edition), J. Sambrook and D. Russell, 2001, Cold Spring Harbor Laboratory Press).<br>
The present invention is further illustrated by the following figures and examples without being restricted thereto.<br>
Fig. 1 shows SDS-PAGE of GFP-Zeo expressing P. pastoris strains in microscale before induction with methanol (A) and 24 (B) and 72 (C) hours after induction. Samples were prepared as described in example 1 h). Lane 1 is X-33 (negative control), Lane 2-4 are X-33 GFP-Zeo strains Mut8 A9, D2 and E2, Lane 5 is X-33 d6*F10. A strong band at 42 kDa is present in all GFP-Zeo clones.<br>
Fig. 2 shows an overview of sequence deletions within the AOXi promoter region and some transcription factor binding sites. Regions deltal-9 were deleted by overlap extension PCR.<br>
Fig. 3 shows a bar chart of the fluorescence intensity of AOXI promoter variants in microscale after derepression (carbon starvation). Cells were grown on 1% glucose in microscale. The data represents the mean ± SD of 4 independent measurements. RFU: relative fluorescence units; WT: P. pastoris strain GFP-Zeo D2 with GFP-Zeo under the control of the wild type AOXI promoter; D1-D9: P. pastoris strains with deletion constructs AOX1A 1-A9 in front of the GFP-Zeo gene; EX. excitation wavelength; EM: emission wavelength.<br>
Fig. 4 shows a bar chart of the fluorescence intensity of AOXI promoter variants in microscale after methanol induction. Cells were grown on 1% glucose in microscale. The data represents the mean ± SD of 4 independent measurements. RFU: relative fluorescence units; WT: P. pastoris strain GFP-Zeo D2 with GFP-Zeo under the control of the wild type AOXI promoter; D1-D9: P. pastoris strains with deletion constructs AOX1A 1-A9 in front of the GFP-Zeo gene; EX. excitation wavelength; EM: emission wavelength.<br>
Fig. 5 shows a bar chart of the fluorescence intensity of selected AOXI promoter variants in microscale. Expression levels under derepressing as well as inducing conditions of single copy strains and multicopy strains with wild type and A6 promoter variants are shown. The data represents the mean ± SD of 4 independent measurements. WT: single copy GFP-Zeo strain with wild type AOXI promoter (GFP-Zeo D2), D6: single copy AOX1A6* clone; WT_E2: multicopy GFP-Zeo clone with wild type AOXI promoter; D6* F10: multicopy AOX1A6* clone (X-33 d6FlO).<br>
Fig. 6 shows the result of a drop test of P. pastoris<br>
strains on MD and MDM agar plates with distinct Zeocin™ concentrations. Cells were grown on BMD(1%) medium to a OD595 of 1.5, diluted in steps of 10 to a final dilution rate of 105 and transferred to the agar plates using a 48 pin replicator. Numbers on top of the picture denote the dilution factor which is the same for all plates. MD medium was prepared as described above. Meth-anol in MDM-^Zeo plates was added to a final concentration of about 0.5%. Zeocin™ was added to final concentrations of 100, 200 and 500 μg/ml, respectively. X-33: P. pastoris X-33, A9: P. pastoris GFP-Zeo Muts A9, D2: P. pastoris GFP-Zeo D2, E2: P. pastoris GFP-Zeo E2.<br>
Fig. 7 shows the expression level of several multicopy strains in comparison to reference strains; a) activity under derepressing conditions; b) activity after methanol induction.<br>
Fig. 8 shows the expression level of A6* multicopy strains under derepressing and induced conditions compared to reference strains.<br>
EXAMPLES:<br>
Example 1:<br>
Material and methods:<br>
a) DNA Preparation/Purification Kits:<br>
Several commercially available DNA preparation and purification kits have been used according to the supplied manuals (see Table 3).<br>
Table 3; DNA Preparation and Purification Kits  <br>
(Table Removed)<br>
b)	TOPO® Cloning:<br>
TOPO® cloning was performed according to the supplied manuals (for cloning into pCR®4 Blunt-TOPO® and for cloning into pCR®-Blunt 11-TOPO®). Always 4 μl of PCR product were used for cloning. 2 and 4 μl of every cloning reaction were transformed into One Shot® chemically competent E. coll TOP10F' cells (In-vitrogen Corp., Carlsbad, CA, USA) according to the above'mentioned protocols.<br>
c)	E. coli transformation:<br>
Transformation of ligation reactions and plasmids into E. coli was performed according to the SEM (simple and efficient method)-Protocol [16]. Chemically competent E. coli TOP10F' cells were used for transformation.<br>
d)	Pichia pastoris transformation:<br>
Preparation of competent Pichia pastoris cells: A single colony of the desired Pichia pastoris host strain was used to inoculate 50 ml YPD (2% glucose) in a 300 ml baffled wide-necked Erlenmeyer flask. After an overnight incubation at 30°C, 60% humidity and 130 rpm (Pilot Shake® RC-2 TE) a certain volume of this pre-culture was used to inoculate 200 ml of YPD (2% glucose) in a 2 1 baffled wide-necked Erlenmeyer flask to an optical density of about 0.1 at 595 run (OD595) . The culture was grown under the same conditions as the pre-culture to an optical density of 1.0 to 1.5. Cells were pelleted at 4°C and 4000 rpm for 10 minutes and resuspended in 200 ml ice-cold sterile water. This procedure was repeated 3 time with re-suspension of the cells in 100 ml water, 10 ml 1 M sorbitol and 0.5 ml 1 M sorbit-ol, respectively.<br>
10 μg of the desired plasmid were linearised with Bglll and NotI (each 50 u) over night in a final volume of 300 μl. After restriction digestion the DNA was precipitated in EtOH and 0.3 M sodium acetate according to a standard protocol [16]. DNA was<br>
dissolved in 11 μl sterile ddH2O and desalted using a MF-Milli-pore™ Membrane Filter (see Table 12) for about 1-2 h. If PCR-product was used for transformation about 4-5 pg DNA was processed as described above starting at EtOH precipitation.<br>
For each transformation 80 μl of the prepared cells were mixed with 10 μg DNA as described above and incubated for 5 minutes on ice. The mixture was transferred to ice-cold electro-transformation cuvettes (Bio-Rad) and pulsed at 200 Ω, 25 μF and 2.5 kV. 1 ml of icecold 1 M sorbitol was added immediately. The suspension was transferred to a sterile 12 ml PP-tube (Greiner, Frickenhausen, Germany, #184261) and incubated for 2 hours at 30°C without shaking. After this regeneration phase aliquots were plated dn selection plates. For selection of transformants with high expression under inducing condition, the cells were plated on MSM-Zeo plates containing minimal media with sorbitol (or any other non-repressing carbon source) methanol and zeocin. For the selection of clones showing high expression under dere-pressing conditions, the cells can be plated on minimal sorbitol zeo plates lacking methanol. The inclusion of glucose to methanol containing selection plates enables the detection of glucose non-repressed expression clones and their promoters.<br>
e) Colony PCR:<br>
A single colony of the desired Pichia strain was resuspended in 100 μl ddH20 in a 100 μl microtube and heated for 5 to 10 minutes at 95°C. After centrifugation at 13,200 rpm for 1 minute 10 μl of supernatant were used as template for PCR reaction. 5 μl of this first PCR round were used as template for a second one. 5 μl of the second PCR round were used for a control gel. PCR reactions contained 10 pmol of each primer (AOXl__col and GF-Prev), 200 μM of each dNTP and 2.5 units of Hot Star Taq® DNA polymerase (QIAGEN) or Taq DNA polymerase (Promega) under buffer conditions according to the supplied manuals in a final volume of 50 μl. For sequencing the second PCR product was purified using the QIAquick® PCR Purification Kit.<br>
Table 4: Temperature programme for colony PCR <br>
(Table Removed)<br>
f)	Pichia pastoris genomic DNA isolation:<br>
The desired F. pastoris strain was grown overnight in 5 ml YPD in a sterile 12 ml PP-tube on a rotation barrel at 30°C to a final OD595 of 5-10. 1.5 ml of the culture were used for DNA isolation using the Easy-DNA™ Kit according to the supplied protocol.<br>
g)	Protein assay:<br>
Measurement of protein concentration in solution has long been used in biochemistry. One of their major applications is to normalise a wide variety of biochemical methods to the total protein amount as is done in the present case for the oxygen consumption rates. The most commonly used ways to determine protein concentrations are the Bradford, Lowry and BCA™ methods. These methods have definite limitations in respect of sensitivity, dynamic range and compatibility to specific reagents. Between these 3 assays, Bradford and Lowry are more reliable and reproducible than the BCA™. On the other hand Lowry and Bradford possess severe limitations when detergents and/or reducing agents are present which results in high blank values. Thus the BCA™ assay is the method of choice after a chemical lysis. Protein concentrations were determined using the BCA™-assay after chemical cell lysis with Y-Per® and BSA as standard according to the instruction manuals (Pierce Biotechnology Inc.) therefore only the main steps will be described briefly below. 200 μl of the cultures were centrifuged at 4000 rpm and 4°C for 5 minutes. After discarding the supernatant the pellet was resuspended in 100 ul Y-Per® by pipetting up and down. The suspension was incubated in 1.5 ml microtubes in a Thermomixer at room temperature and 600 rpm for 20 minutes. After the cell debris was pelleted at 13,200 rpm and room temperature for 10 minutes the supernatant was transferred into a new microtube and stored at 4°C for the BCA™ assay or SDS-PAGE. 25 ul sample were mixed in a mi-croplate well with 200 μl BCA™ working reagent (reagent A: re-<br><br>
agent B = 50:1), agitated thoroughly for 30 seconds and covered tightly with plate sealers (Promega). After incubation for 30 minutes at 37 DC and cooling to room temperature the absorption was determined at 562 nm using a Spectramax Plus 384 plate reader. If necessary, samples were diluted with ddH20 prior to the BCA assay.<br>
h) SDS-PAGE:<br>
Samples for SDS-PAGE were prepared by chemical cell lysis using Y-Per® as reagent as described in the section above. 10 μl of lysate were mixed with 10 ul 2x SSB (sigma sample buffer) and incubated at 95 °C for 5-10 min and 15 μl of this mixture were loaded on the protein gel. Electrophoresis was performed with 180 V for about 1 h and protein bands were detected using Coomassie™ blue staining.<br>
Table 5; Gel preparation for SDS-PAGE <br>
(Table Removed)<br>
i) Glucose assay:<br>
Glucose concentrations were determined using the Glucose- UV Hexokinase method without deproteinisation (DIPRO med Handels GmbH, Weigelsdorf, Austria, Prod. no. D590522). 50 μl of Pichia cultures were transferred in a PCR microplate and centrifuged at 4000 rpm for 5 minutes. 10 μl of supernatant were added to 190 μl hexokinase reagent in an UV-Star microplate and incubated at room temperature for 15 minutes. After incubation absorption at 340 nm was determined using a Spectramax Plus 384 plate reader.<br>
j) Drop tests:<br>
P. pastoris strains were grown in BMD{1%) to a final OD595 of 1.5 and diluted in steps of 10 to a final dilution rate of 105. Transfer on agar plates was done with a 48 pin replicator. The plates were incubated at 30°C until colonies appeared (usually 2 days on MD plates).<br>
k) Sequence alignments:<br>
All sequence alignments were done using MultAlin at the INRA homepage (Institut National de la Recherche Agronomique, Paris, Prance) {prodes.toulouse.inra.fr/multalin/multalin.html) [17] or with ClustalW at the European Bioinformatics Institute (EBI, www.ebi.ac.ch/clustalw) [18]. For sequence comparison with MultAlin always the DNA sequence similarity matrix was used for comparison.<br>
Genes of the methanol utilisation pathway and most peroxi-somal genes are regulated in a similar way in respect to glucose repression, derepression at carbon starvation and induction through methanol. A similar transcriptional regulation with a defined set of transcription factors (repressers as well as in-ducers) should be responsible for this regulation pattern. Transcription factor binding sites within these promoter regions should show some conserved regions. Multiple sequence alignment between promoter regions of coregulated genes should reveal the conserved binding sites of the transcription factors involved in regulation of the accordant genes. Several genes of the methylo-trophic yeasts P. pastoris, H. polymorpha and C. boidinii were reported to be coregulated and their promoter sequences were isolated (Table 6).<br>
Table 6: Coregulated genes of the methanol utilisation pathway or peroxisomal genes from the methylotrophic yeasts P. pastor is, ff. poJymorpha and C. boidinii. <br>
(Table Removed)<br>
1) Transcription factor analysis:<br>
Transcription factor analysis was done with Matlnspector Re lease professional 6.1 Jan. 2003 within the GenomatixSuite 1.6.1 at Genomatix Software GmbH Servers [15] . PAOX1 sequence from pPICZ B was used to search for transcription factor binding sites using the Matrix Family Library Version 3.1.1 April 2003 group ALL fungi.lib (www.genomatix.de).<br>
m) Primers:<br>
Table 7: List of primers used for the described examples (syn-thesised by MWG Biotech AG, Ebersberg, Germany) <br>
(Table Removed)<br>
* Tm calculated using Equation 2 (QuikChange® multi site-directed mutagenesis kit)<br>
Example 1.1: Cloning of the reporter construct GFP-Zeo was used as a reporter for gene expression driven by AOXl promoter variants. Sequences surrounding the ATG start codon were constructed to fulfil minimal requirements of Kozak consensus sequences for highly expressed genes in yeast. To change the promoter regions in front of the GFP-Zeo gene an EcoRI restriction site was inserted (Table 8) by overlap extension PCR.<br>
Table 8: Comparison of translation initiation site and surrounding sequences between the AOXl sequence used in this example (derived from pPICZ) and the AOXl sequence of P. pastoris strain NRRL Y-11430 (Genbank AN: U96967, [2]). .EcoRI restriction site is underlined and minimal Kozak requirements at positions -3 and +4 are labelled in bold letters.<br>
-3 +1 +4<br>
P(AOX1)-GFP AAAACAACTA ATTATTgaAa gaattcAACc ATGGCTAgCa<br>
AOXl (U96967) AAAACAACTA ATTATTcgA	AACg ATGGCTAtCc<br>
PCR-based production of reporter system components P(AOX1) was amplified using 10 ng of the vector pPICZ-B ARS1 as template. The reaction also contained 10 pmol of each primer (P(AOXl)forw and P(AOXl)rev, respectively), 200 μM of each dNTP and 2.5 U Synergy™ polymerase in appropriate buffer conditions in a final volume of 50 μl.<br>
AOXl TT was amplified similarly to the AOXl promoter. AOXlTTforw and AOXlTTrev were used as primer in this reaction. Both PCR reactions were performed in a thermocycler for 30 cycles (95°C, 1 min; 55°C, 30 s; 68°C, 2 min 30 s) with an ini-<br>
tial denaturation step of 5 min at 95°C and a final extension step of 10 min at 68 °C. 2 μl of first PCR round were used for amplification in a second round under the same conditions above. The only difference was an increase in extension temperature to 72°C.<br>
GFP-Zeo [19] was amplified using 25 ng of the vector pTracerra-CMV2 as template. The reaction also contained 10 pmol of each primer (GFP-Zeo forw and GFP-Zeo rev, respectively), 200 μM of each dNTP and 2.5 U Synergy™ polymerase in appropriate buffer conditions in a final volume of 50 μl. PCR was performed in a thermocycler (see Table 8) for 30 cycles (95°C, 1 min; 55°C, 30 s; 72°C, 2 min 30 s) with an initial denaturation step of 5 min at 95°C and a final extension step of 10 min at 72°C.<br>
All PCR products were purified by agarose gel electrophores-is prior to overlap extension PCR. The reaction contained 10 ng P(AOX1), 5 ng AOX1 TT and 50 ng GFP-Zeo prepared as described above as templates, 200 uM of each dNTP and 2.5 U Synergy™ polymerase in appropriate buffer conditions in a final volume of 50 ul. PCR was performed in a thermocycler (see Table 8) for 30 cycles (95°C, 1 min; 53°C, 50 s; 68°C, 3 min 30 s) with an initial denaturation step of 5 min at 95°C and a final extension step of 10 min at 68 °C. After 10 cycles 10 ill of a mixture containing 10 pmol of the outer primers P(AOXl)forw and AOXlTTrev, again 200 μM of each dNTP and 2.5 U Synergy™ polymerase in appropriate buffer conditions were added. The PCR was continued as programmed after this addition. The obtained PCR product with the desired size of about 2.4 kb was purified on an agarose gel. The purified product was cloned into pCR®4Blunt-TOPO® vector and sequenced. Sequencing revealed 4 mutations and 1 deletion within the reporter construct.<br>
The base pair deletion site was found at position -15 of the original promoter sequence. Since this position was within the multiple cloning site of all pPICZ vectors (A, B and C; inside the Sful restriction site) the deletion should not influence the promoter activity and therefore was not corrected. The first mutation (T—&gt;C) was found in the promoter region at position -828. The other 3 mutations were found within the GFP-Zeo coding sequence at positions +122, +507 and +1075, respectively.<br>
The G—&gt;A conversion at position +122 changes the GGA codon of Gly to a GAA 'codon which results in a G41A amino acid change.<br><br>
T—&gt;C conversion at + 507 is a silent mutation changing only a codon of R169. The last mutation (T—&gt;C) at position +1075 changes the TGA stop codon to the Arginine codon CGA. The mutations -828, +122 and +1075 were repaired with the QuikChange® multi site-directed mutagenesis kit after constructing the pAOX vector. The silent mutation at position +507 and the mutation in the polylinker were not changed since it did not introduce a rare codon.<br>
pAOX was constructed by excising the Paoxi-GFP-Zeo-AOXlTT fragment from pCR®4Blunt- TOPO® vector with Kpnl and NotI and inserting it into the pBlueScript® SK- vector between the Kpnl and NotI site.<br>
The mutations found in the AOX1 promoter and the GFP-Zeo sequence were corrected using the QuikChange® multi site-directed mutagenesis kit (Stratagene, Amsterdam, The Netherlands). The PCR reaction was performed according to the supplied manual containing 100 ng pAOX, 100 ng of mutagenic primers (423forwf 1372forw and 2325forw, respectively) and 1 ul QuikChange® multi enzyme blend in appropriate buffer conditions in a final volume of 25 ul in a thermocycler for 30 cycles (95°C, 1 min; 55°C, 1 min; 65°C/ 10 min 30 s) with an initial denaturation step of 1 min at 95°C. Dpnl digestion and chemical transformation into E. coll XL10-GOLD® (Invitrogen Corp.) cells was done according to the supplied manual. Correction of all 3 mutations was verified by sequencing.<br>
Example 1.2: Construction of AOX1 promoter deletions<br>
Left arms of the AOX1 promoter were synthesised using P(AOXl)forw as forward primer and AOX n rev (n=1...9) as reverse primers. Right arms were synthesised with 10 pmol of AOX n forw (n=1...9) as forward primers and P(AOXl)rev as reverse primer. All arms were synthesised using 12 ng of the vector pAOX as template and 10 pg of each primer. The reaction also contained 10 pmol of each primer, 200 μM of each dNTP and 0.6 U Pwo DNA poly-merase in appropriate buffer conditions in a final volume of 50 μl. PCR was performed in a thermocycler for 30 cycles (95°C, 1 min; 55°C, 1 min; 68°C, 1 min 30 s) with an initial denaturation step of 5 min at 95°C and a final extension step of 10 min at 68 °C. All arms were agarose gel purified prior to the use as template for overlap extension PCR.<br><br>
Table deletions  <br>
9: Overlap primer pairs and arm length for promoter <br>
(Table Removed)<br>
The reaction contained 10 ng of each arm prepared as described above as templates, 200 μM of each dNTP and 0.6 U Pwo DNA polymerase in appropriate buffer conditions in a final volume of 50 μl. PCR was performed in a thermocycler for 30 cycles (95°C, 45 s; 60°C, 45 s; 68°C, 2 min) with an initial de-naturation step of 5 min at 95°C and a final extension step of 10 min at 68 °C. After 10 cycles 20 ul of a mixture containing 10 pmol of the outer primers P(AOXl)forw and P(AOXl)rev, again 200 μM of each dNTP and 0.6 U Pwo DNA polymerase in appropriate buffer conditions were added. The PCR was continued as programmed after addition of the mixture.<br>
The obtained PCR products with the desired size of about 898-947 bp were purified on an agarose gel and cloned into pCR®4Blunt-TOPO® (Δ2, Δ4, Δ5, Δ7 and Δ8) or into pCR®-Blunt II-TOPO® vector (Δl, Δ3, Δ6 and Δ9) and sequenced.<br>
pAOXA vectors were constructed by excising the PAOXIA fragments from TOPO® vectors with BglII and EcdRI and inserting them into the pAOX vector between Bglll and EcoRI site instead of the wild type AOX1 promoter. The resulting vectors were verified by sequencing.<br>
Example 1.3: Pichia pastoris transformation and analysis of<br>
transformants<br>
Pichia pastoris transformation was done as described earlier. Selection for Integration of PAoxi (or PAOiΔ) -GFP-Zeo-AOXl TT was done by spreading the transformed and regenerated Pichia cells in aliquots on MSM-Zeo agar plates.<br>
Pichia pastoris strains were grown in deep well plates containing 300 μl BMD(1%) per well at 28°C, 320 rpm and 80% humidity for 60 hours at room temperature. After this time, 50 ul were taken for determination of the GFP-fluorescence. Induction was performed by adding 250 ul BMM2/well followed by a further incubation of 72 h. Methanol was refilled by adding 50 μl BMM10 after 10, 24 and 48 hours. Once more GFP fluorescence was measured after 72 h of methanol induction.<br>
Analysis of reporter enzyme expression Expression of GFP-Zeo in Pichia pastoris was analysed by fluorescence detection of GFP in the Spectramax Gemini XS plate reader with excitation at 395 nm and emission at 507 nm. 50 ul of P. pastoris cultures cultivated in deep well plates as described above were diluted 1+3 with ddH20 in FIA microtiter plates. Due to the limited sample amount only single measurements were performed. All means ± standard deviations given are calculated from at least 3 different cultures (wells).<br>
If the integration cassette is integrated in the AOX1 locus without replacing the AOX1 gene, the recombinant Pichia strain is able to grow on methanol with a wild type rate, while replacement of the AOX1 gene by double crossover results in a much slower growth rate on methanol. These two growth phenotypes are called methanol utilisation plus (Mut*) and methanol utilisation slow (Muts), respectively. For analysis of the methanol utilisation phenotype, Pichia pastoris microscale cultures were transferred on MM and MD agar plates using a 96-pin replicator and incubated at 30°C for 2 days. After 2 days colonies appear on both plates if the Pichia strain possesses Mut* phenotype while with Muts phenotypic strains only on MD plates colonies arise.<br>
All Pichia strains which are derived from transformations of pAOX or one of the pAOXA plasmids were analysed by colony PCR and deletion constructs also by sequencing to assure the promoter sequence in front of the reporter gene (GFP-Zeo).<br>
Example 1.4: Directed evolution of the AOX1 promoter<br>
While PCR mutagenesis on coding regions of genes is well de-<br>
veloped and established nothing is known about mutagenesis on promoter regions. Due to the lack of knowledge several mutagenesis conditions were performed: To minimise bias in mutational spectrum, two different polymerases were used, a Taq DNA poly-merase and the Mutazyme® DNA polymerase (Stratagene Inc.)- Du® to the fact that knowledge on mutation frequency for evolution of promoter sequences is completely lacking, several mutation frequencies (theoretically 1 to ~14/kb) were tested.<br>
Mutagenesis using Hot Star Taq® DNA polymerase: Mutagenic PCR was performed on the promoter sequence in a 100 μl reaction volume according to [20] . The reaction contained 12 ng pAOX, 40 pmol of each primer, (P(AOXl)forw and P(AOXl)rev), dNTPs (200 μM dGTP, 200 uM dATP, 1 mM dTTP, ImM dCTP) and 5 0 Hot Star Taq® DNA polymerase in appropriate buffer conditions. MgCl2 concentration was increased to 7 mM (usually 3 mM) to alter the error rate of the polymerase. PCR was performed in a thermocycler for 30 cycles (95°C, 45 s; 55°C, 45 s; 72°C, 1 min 30 s) with an initial denaturation step of 15 min at 95°C and a final extension step of 10 min at 72°C.<br>
The GeneMorph® random mutagenesis kit was performed on the promoter sequence in a final volume of 50 μl according to the supplied manual. Different amounts of the vector pAOX as template were used (see Table 10). 12.5 pmol of each primer, P(AOXl)forw and P(AOXl)rev were used. PCR reaction was performed in a thermocycler for 30 cycles (95°C, 30 s; 55°C, 30 s; 68°C, 1 min 30 s) with an initial denaturation step of 1 min at 95°C and a final extension step of 10 min at 68°C.<br>
Table 10: Amount of template used in the GeneMorph® PCR reaction <br>
(Table Removed)<br>
A first round of mutagenesis with conditions described above (Taq, 3x GeneMorph®) was performed. To get higher mutation fre-<br>
quency the GeneMorph® reaction #3 was used as template for a second PCR round. Taq and GeneMorph® #2 and 13 conditions were used.<br>
Prior to the transformation into Pichia pastoria X-33 GFP-Zeo Mut8 A9 cells, all PCR reactions were precipitated and desalted as described earlier. The standard transformation and regeneration procedure was used. Selection for promoters induced in glucose medium was done by spreading 150 μl aliquots of transformed cell suspension on MD agar plates containing 100-500 μg/ml Zeocin™ and incubation on 30°C for 2 days.<br>
Example 1.5: Results and discusssion<br>
I) Characterisation of the reporter system<br>
To date, a large variety of GFP variants are in use in molecular biology. Although differing only in a few point mutations, their characteristics differ enormously. Apart from improved folding properties their fluorescence spectra as well as their quantum yields and therefore intensities differ a lot. Green fluorescent proteins can be divided into two main groups, depending on their excitation maximum: wild type GFP variants with an excitation maximum at 395 nm and a minor maximum at 470 nm, and red-shifted GFP variants with an excitation maximum at 480-490 nm. According to its amino acid sequence, cycle-3-GFP belongs to the group of wild type GFP variant with an excitation maximum at 395 nm.<br>
To control the spectral properties when expressed in Pichia pastoris fluorescence spectra were determined. The overall excitation maximum of the cycle-3-GFP in GFP-Zeo is 395 nm, while the second maximum at 478 nm is evanescent. The emission spectrum reveals an emission maximum of 510 nm. Of the two excitation wavelengths suggested by the manual 395 nm is preferred and was used for all further measurements.<br>
Self-absorption is a very frequent phenomenon in fluorescence spectroscopy. At high concentrations of the fluorophor, photons emitted in a region overlapping the absorption (excitation) spectrum can be absorbed (radiative energy transfer). Lower fluorescence intensity will be observed if self-absorption (emission inner filter effect) will occur. This leads to an underestimation of promoter activities. With no inner filter effect fluorescence intensity increases in a linear way as the fluorophor increases. Thus increasing volumes of GFP-Zeo ex-<br>
pressing Pichia pastoris cells were tested for their fluorescence activity.<br>
Up to 3000 RFU no emission inner filter effect was detectable on the cell level. Self-absorption within the cells, caused by the accumulation of GFP, could not be evaluated. A linear increase in fluorescence over the whole 72 hours of induction phase was detected. For that reason an inner filter effect within the cells seems to be not likely. Thus the accumulation of GFP-Zeo within the nucleus is no problem for its quantitation. No inner filter effect occurs within the range of single copy promoter activities determined in this study. Due to the lack of self-absorption underestimation of promoter activities is not likely to occur. The inner filter effect observed by others is most probably caused by the usage of a different GFP variant with a much smaller Stokes shift and therefore overlapping excitation and emission spectra. One has to be careful when comparing results of GFP expression experiments. The usage of several GFP variants with distinct spectral properties, but also with optimised codon usages and therefore quite different expression levels in different expression hosts complicates the comparability of results of different labs.<br>
II) AOX1 promoter activity in microscale<br>
Small scale cultivation of microbial cells (e.g. yeast, bacteria) is usually done in shake flask cultures. Inoculation and cultivation of large microbial libraries in shake flasks are labour and time intensive resulting in high costs. In recent years microscale cultivation systems using deep-well microtiter plates were developed as an alternative. Due to the parallel handling of e.g. 96 or 384 strains/cultures and the less material needed, microtiter systems are superior to shake flasks in terms of labour, time and therefore cost intensities. Due to several reasons, the major drawbacks of microtiter systems, small sample volume and low aeration efficiency, are less relevant: (1) technical advances in analytical systems lead to lower detection limits of a large number of compounds resulting in very low sample volumes needed; (2) methods and devices for growth in deep-well microtiter plates were also improved. It has been shown in a few studies that aeration rates and therefore growth conditions in microtiter plates are similar to shake flasks. It has also been demonstrated that real-time studies on the GAL1<br>
promoter in S. cerevisiae using cycle-3-GFP as reporter protein are consistent with shake flask studies.<br>
AOX1 promoter driven GFP-Zeo expression was studied in deep-well microtiter plates as described above. After cell growth on glucose an induction phase with methanol as carbon and energy source follows. Induction of the AOX1 promoter with methanol in Pichia pastoris cells possessing the PAOXl-GFP-Zeo-AOXl TT expression cassette led to a fast increase in GFP fluorescence. Until 72 h GFP fluorescence increased in a linear way. Expression of GFP-Zeo would continue if methanol is added. If not, methanol depletes through evaporation 'and consumption within 24 h and GFP-Zeo expression decreases to a derepressed level.<br>
The increase in GFP-Zeo fluorescence was also in accordance with GFP-Zeo protein as was shown by SDS-PAGE. Upon methanol induction a protein band of about 42 kDa appeared which became more intensive as fluorescence increased. The strong band at 42 kDa was found in all GFP-Zeo clones while in the negative control (X-33 wild type) no band appeared. Also in the sample of X-33 d6*F10 after 72 hours of methanol induction a strong band was found (Fig. 1C, Lane 5). Although not normalised a clear correlation between the intensities of the 42 kDa bands and the appropriate fluorescence levels is assessable.<br>
III) Transcription factor binding sites<br>
As described earlier, consensus sequences for binding sites of several transcription factors are known. Sequence analysis of the AOX1 promoter sequence revealed several putative transcription factor binding sites, with a few hits of special interest. Among heat shock factor and stress response element motif, binding sites of a few transcription factors generally known to be involved in glucose regulation were found. The most interesting binding sites were summarised in Table 11 and Fig. 2.<br>
Table 11: Transcription factor (TF) binding sites found within the AOX1 promoter sequence. Base pairs in capital letters denote the core sequence (the 4 highest conserved, consecutive residues of the matrix), underlined base pairs show a high information content (Ci&gt;60 of a maximum of 100) . <br>
(Table Removed)<br>
* The given position is marked in respect to the translation start point (ATG) of the GFP-Zeo gene; core sequences of putative transcription factor binding sites are shown in capital letters c denotes homology to the complementary strand<br>
IV)	Regulatory sequences in methanol regulated genes<br>
Several sequences are described in literature to be involved<br>
in regulation of methanol inducible genes. Based on deletion analysis of the P. pastoris AOX2 promoter three regulatory regions were described, two negative acting regions (URS1 and URS2, upstream repression sequences) and a positive acting domain (UAS, upstream activation sequence) [3]. For the H. poly-morpha MOX promoter two upstream activating sequences (UAS1 and UAS2) and one represser binding site (ORS1) were also described<br>
[8].<br>
V)	Deletion constructs of AOX1 promoter<br>
Based on the transcription factor analysis and the multiple sequence alignment 9 promoter regions were chosen for deletion by overlap extension PCR as described earlier. The AOX1 promoter deletion constructs were cloned into the pAOX vector to replace the "wild type AOX1" promoter 5 to the reporter gene GFP-Zeo. The plasmids were linearised and integrated into the Pichia pastoris genome.<br>
Table 12: Sequences deleted in the AOXl promoter constructs <br>
(Table Removed)<br>
* The given positions are marked in respect to Seq ID No. 1<br>
Integrants were analysed for GFP-Zeo expression and for integration of the correct promoter sequence in front of the GFP-Zeo gene as described above. Single copy integrants were analysed in further detail for their GFP-Zeo expression levels in different carbon sources in microscale. In all constructs (deletion and wild type) no GFP fluorescence could be detected as long as glucose or glycerol was present in the medium (with and without methanol) . Upon carbon starvation, representing dere-pressing conditions, a slight increase in GFP fluorescence was detected. Compared to wild type some promoter variants showed remarkable differences (Fig. 3). A significant lower promoter activity was found in 6 constructs (Δ3, Δ4, Δ5, Δ7, Δ8 and Δ9,<br>
see Pig. 3) under derepressing conditions. Δl possessed wild type activity while the constructs Δ2 and Δ6* resulted in significantly higher GFP-Zeo expression. Expression level of the latter one was remarkably higher than the wild type level.<br>
Upon methanol induction all variants showed significant decreased promoter activity with only one exception: Δl which resulted in around 20% higher activity compared to wild type. The decrease in activity of all other variants is quite significant as can be seen in Fig. 4.<br>
Promoter activity of all variants and wild type constructs normalised on methanol-induced wild type activity is summarised in Table 13.<br>
Table 13: Fluorescence intensity of AOX1 promoter variants in ndcroscale. The data represents the mean SD of 4 independent measurements. Fluorescence intensity after 72 h methanol induction of WT promoter (100%) is 987 ±81. No fluorescence was de-tectable as long as glucose is present in the medium. <br>
(Table Removed)<br>
Deletion of the TATA box in construct A8 resulted in a massive destruction of the promoter with a severe decrease of activity at derepressing and inducing conditions of about 90% and 80%, respectively. By elimination of the experimentally determined (Ellis, S.B., et al., Mol. Cell. Biol. (1985) 5:1111-<br><br>
1121) transcription initiation start (A9) no such strong effect on the expression level was observed. It is one of the best deletion constructs after methanol induction. As expected, the TATA box has a severe impact on the transcription level. In contrast the transcription initiation start seems to be not as important as the TATA box. Another region in the defined distance to the TATA box may act as a transcription start after deletion of the original one. One can speculate on the effect of this deletion on several stages of the expression process (e.g. transcription initiation, mRNA stability, translation initiation) since the 5' end of the mRNA was changed by the deletion.<br>
Only two constructs, A2 and Δ6*, show a significant higher expression level after derepression. Putative transcription factor binding sites of Raplp and Gcrlp are included in the deleted sequences. In addition, the putative transcription factor binding site of QA-1F is very close to the deleted sequences of Δ6*. Noteworthy, Raplp and Gcrlp binding sites are known to act in a synergistic manner when present in promoter sequences [21] . The general transcription factor Raplp has diverse cellular functions (e.g. telomere structure, mating, translation, glyco-lysis) dependent on the sequence context of its binding site and the appropriate transcription factors [22-24] . As mentioned before, Gcrlp is the major item of regulation and coordination of glycolytic genes and is absolutely necessary for high level ex-<br>
%<br>
pression in S. cerevisiae. Binding sites of Raplp and Gcrlp are found in close proximity in the core region of upstream activating sequence (UAS) of glycolytic genes and Gcrlp binding is alleviated by bending the DNA by Raplp. On the other hand an adjacent Raplp binding site is not an absolute requirement for Gcrlp dependent activation of genes. It seems that Gcrlp can facilitate the binding to its binding site when higher numbers of CT-boxes are present. Although a clear interaction of Raplp with Gcrlp as well as Gcrlp with Gcrlp was described, some other factors are suggested to interact with Gcrlp and/or Raplp modulating the activity of the complex. A broad knowledge on the induction mechanism was achieved during the last 3 decades.<br>
The described essential close proximity of Gcrlp and Raplp binding sites in functional UAS described above could not be found in the AOX1 promoter sequence. In contrast, the two binding sites are 367 bp apart. Among the putative Gcrlp binding<br><br>
site, its core sequence CTTCC is present 2 times in the AOX1 promoter sequence, but none of them immediately adjacent to the Raplp binding site or another CTTCC motif. Therefore a syner-gistic action of these two binding sites as found in many glyco-lytic genes seems not to be likely. Due to the fact that the putative roles of Raplp and Gcrlp are represser proteins for AOX1 under derepression conditions, a new mode of (inter-) action of the two proteins for this putative novel cellular function is possible.<br>
An involvement of the Δ6* deletion (including the putative Gcrlp binding site) in repression upon carbon starvation is emphasised by the observation of multicopy strains with very high GFP-Zeo expression without methanol induction. GFP-Zeo expression of the best clone of the Δl - Δ9 series, called P. pastoris X-33 d6*F10, is shown in Fig. 5. GFP-Zeo expression is about 10% higher after derepression (60h in microscale) in this A6* multicopy strain than in a single copy wild type promoter strain (X-33 GFP-Zeo D2) after methanol induction. Expression level of P. pastoris X-33 d6*F10 after methanol induction is also much higher than a multi copy strain with wild type promoter (X-33 GFP-Zeo E2) .<br>
P. pastoris AOX1 and DAS1 and H. polymorpha MOX promoter regions promote expression of the reporter enzyme beta-galactosi-dase (lacZ of E. coli) in S. cerevisiae [9]. Regulation pattern of these genes in S. cerevisiae is similar to their natural hosts: glucose represses gene expression. At carbon starvation conditions expression is slightly derepressed and glycerol as carbon source induces expression. Beta-galactosidase levels expressed under the control of AOX1 and DAS1 regulatory regions in S. cerevisiae are comparable to those obtained with the strong S. cerevisiae CYC1 (constitutive) and GAL2 (inducible) promoters [9] . It was demonstrated that expression driven by the MOX promoter is also induced by ethanol, methanol and oleic acid in S. cerevisiae. Another very important finding is the involvement of Adrlp in derepression/induction of the promoter. Adrlp, a positive effector of ADH2 (alcohol dehydrogenase 2) and some peroxi-somal proteins in S. cerevisiae [25], is also a positive effector of the MOX promoter when glucose is lacking in the medium.<br>
As mentioned before regulation pattern of the AOX1 and the MOX gene are significantly different in their natural hosts due<br><br>
to the derepression of MOX when glycerol is present. Using the AOX1 promoter region in H. polymorpha revealed that the AOX1 promoter is not repressed by glycerol in the heterologous host [26]. Thus, the heterologous AOX1 promoter seems to be regulated like the homologous MOX promoter. This results in the suggestion that the significant differences in regulation pattern between P. pastoris and H. polymorpha are due to the overall transcrip-tional response to different carbon sources in these two yeasts. Meaning, while the glycerol and glucose repression machinery are (partially) identical in P. pastoris, in H. polymorpha (like in S. cerevisiae) the situation is different and glycerol does not use the glucose repression machinery.<br>
Two of the three putative HAP2/3/4/5 binding sites found in the AOX1 promoter sequence are within the Δl deletion construct and the third in Δ5. Sequence deletion of Δl results in an increase in promoter activity upon methanol induction while no effect on the derepression promoter level was observed. In contrast, deletion of Δ5 results in a severe decrease in promoter activity under derepression as well as induction conditions. In the Δl deletion a putative Aspergillus nidulans abaA binding site was found. The abaA gene product is a transcriptional activator which is involved in conidiophore (asexual reproductive apparatus) development in A. nidulans [27]. Since all putative binding sites are possible activator sequences [27], their deletion should have a negative effect on the expression level as found in the A5 construct. Due to the fact that both deletions are very long another binding site might be responsible for the observed effect. The fact that deletion of Al has the opposite effect on the expression level indicates that one of the putative binding sites is a represser motif, or another binding site is present which exceeds the effects of deletion of the putative HAP and abaA binding sites thereby increasing the expression level.<br>
Nonetheless, the HAP complex is known to be responsible for upregulation of genes involved in respiratory and energy metabolism in S. cerevisiae. Regulation of respiration is controlled by oxygen level as well as the carbon source present in the medium, both mediated by the Hap complex. In the fermentative yeast S. cerevisiae, several genes and therefore functions of the respiratory chain as well as the citric acid cycle are<br><br>
repressed by glucose. Glucose repression of respiration is partially mediated by the Hap complex, namely by the absence of Hap4p as long as glucose is present. In contrast, oxygen-dependent regulation seems to be regulated by Haplp [28]. Homologues of the Hap complex genes were isolated in the respiratory yeast K. lactis. Genes involved in respiration are constitutively expressed in respiratory yeasts, even in presence of glucose. To date, almost every respiratory chain gene has been shown to be regulated independently from the Hap complex [29]. The role of the Hap complex seems to be in coordinating carbon and nitrogen assimilation, as has also been found in S. cerevisiae [30] and Aspergillus nidulans [29].<br>
The first step in the methanol utilisation pathway, mainly catalysed by the AOX1 gene product in P. pastoris, is oxygen-consuming. Most of the genes involved in energy metabolism and almost every gene encoding for oxygen-consuming enzymes is regulated by oxygen, mainly by Haplp and/or Hap2/3/4/5p [28]. When grown on methanol as sole energy and carbon source, the methanol utilisation pathway results in carbon assimilation and energy production. An involvement of the Hap complex recognition motif TTCCAA. in the regulation of the AOX1 promoter makes intuitive sense.<br>
The A4 construct, which includes a second putative HSF binding site, resulted in a 30% decrease of promoter activity after derepression and induction. Therefore HSF is a general enhancer of AOX1 gene expression under derepressing as well as induction conditions. In 5. cerevisiae several stress conditions like heat shock, oxidative stress and glucose starvation led to the activation of HSF. It has also been demonstrated that the protein kinase Snflp, one of the "metabolic master switches", is involved in phosphorylation and therefore activation of HSF upon carbon starvation [31]. Thus an involvement of HSF in full activation of AOX1 upon glucose starvation (with or without induction) occurs.<br>
Expression studies on the AOX1 promoter using truncated versions as well as variants with deleted sequences are disclosed in the prior art [32, 33].<br>
Table 14: Results of the promoter studies by Inan et al. [32, 33]; Induction was performed with 0.5% methanol as carbon<br>
source, repression with 0.5% methanol and 0.5% ethanol; Start positions denote the 5' end of the sequence in the AOX1 promoter in respect to the translation start point (ATG)  <br>
(Table Removed)<br>
The construct Inan_BCDEF, which starts at 153 (-801) (Table 14) revealed a binding site of at least one activator protein upstream of 153 (-801). Candidates for this activator binding site are the binding sites of Haplp (52 to 66, -902 to -888) and HSF (135 and 155, -819 to -799) on the complementary strand found with Matlnspector. Truncation at the SacI restriction site (210-215 (-744 to -739)) resulted in a promoter reaching nearly wild type promoter activity (Geoff Lin Cereghino, Poster, Sixth Meeting on "Current Topics in Gene expression and Proteomics", San Diego, October 19-22, 2003) . To reach the wild type promoter level with the SacI truncated promoter construct (pHWGO, Geoff Lin Cereghino, poster), a second binding site for a represser protein may be present upstream of 210 (-744) whose deletion has the same impact, but in the opposite direction, on the promoter<br>
activity. The location of the represser protein is between 169 (-784) and 210 (-744) because the Δl construct (Δ 169 (-784) to 234 (-719)) contains a represser binding site. Deletion of Δl results in a 20% increase of promoter activity (Table 14) which is in the range of the decrease by deletion of the activator protein binding site.<br>
By comparison with Δ4 (Δ 508 (-445) to 550 (-403)) the location of the represser binding site can be further refined to a sequence between 433 (-520) and 508 (-445) because the Δ4 deletion includes a positively acting transcription factor, HSF at 516 to 536 (-438 to - 418) . If the positively acting HSF (if it is HSF) is located within the proposed region, a stronger effect of the represser binding site between 433 and 508 (-520 and -445) can be suggested. If the binding site for HSF is located in the region between 508 and 536 (-445 and -418) another activator binding site is located between 536 and 560 (-418 and -393) . If not, it is likely to be the same binding site. As the In-anABCD_F (Δ 560 to 709 (-393 to -245)) variant with only 16% wild type activity also the Δ5 construct (624 to 682 (-329 to -271)) results in a decrease of about 70% of the wild type level. As expected, deletion of the Inan B fragment from the full length promoter (results in InanA_CDEF) as well as from<br>
Inan_BCDEF (results in Inan	CDEF) results in a decrease to 63<br>
and 64% of the longer fragment, respectively. In contrast,<br>
while deletion of the C fragment from the full length promoter<br>
results in an increase of about 10% in promoter activity, dele<br>
tion from the truncated Inan	CDEF fragment leads to a decrease<br>
from 49 to 14% (Table 14). The explanation is a synergistic binding of transcription factors dependent on the context of their binding sites. Between 713 and 760 (-241 to -194) a last activator protein binding site is located (Geoff Lin Cereghino, Poster San Diego). Again, by the Δ7 construct (Δ 729 to 763, -225 to -191) the location of the activator could be refined downstream to 729 (-225).<br>
To conclude, several regions were found which had a strong impact on the expression level of the AOX1 promoter. Combining all known regulatory sites from the example provided herein and from other authors, excluding the regions containing the TATA box and the transcription initiation site, at least 10 regulatory sites exist on the PAOXI promoter sequence.<br>
The data provided revealed the orchestral regulation of the AOX1 promoter: several factors are necessary to bind to the DNA for maximum expression level. Under inducing conditions several positive acting transcription factors (activators) bind to the DNA while most represser proteins did not bind resulting in high level expression. While derepressed, the promoter activity reached only a small percentage (~3%) of the induced level. This is most likely due to less activator and more represser proteins binding to the promoter region. Under repressing conditions one can assume that no activators and several repressers bind to the DNA with a further increase of the represser/activator ratio under repressing conditions.<br>
It has been demonstrated for the glucose repressed ADH2 (alcohol dehydrogenase 2) promoter of S. cerevisiae that binding of activator proteins (e.g. Adrlp) immediately adjacent to nucle-osomes lead to destabilisation and therefore rearrangement of the chromatin upon derepression. The rearrangement takes place in the region of the TATA box and the transcription initiation site therefore increasing their accessibility. Due to the higher accessibility formation of a stable pre-initiation complex takes place therefore increasing the promoter activity to a basal level. Among the binding of several transcription factors to enhance the PAOXI driven expression, a similar mechanism, at least for derepression is assumable. Taken all the data and assumptions together, regulation of the AOX1 promoter is highly complex and the putative binding sites of several (positively and negatively acting) transcription factors reveals highly coordinated machinery which is able to integrate a wide variety of signals for the regulation of the AOX1 promoter.<br>
VI) PCR mutagenesis of AOX1 promoter<br>
Here it has been demonstrated that specific mutations within core sequences of transcription factor binding sites result in significant alterations of their effector force. Assumably a few activator and represser proteins act on the AOX1 promoter to result in its very strong regulation (almost no activity under glucose, very high activity in methanol) . Therefore random mutagenesis of the AOX1 promoter should result in several promoter variants with destroyed or reduced represser binding site activities. A set of PCR reactions with different mutation rates was performed. The resulting promoter variants were transformed into<br>
P. pastoris GFP-Zeo Mut8 A9 strain where the AOX1 gene was replaced by the GFP-Zeo strain. Replacement of the wild type AOX1 promoter by mutagenised promoter variants should occur to a particular rate. Screening for promoter variants with higher expression rate when glucose is present in the medium was done on MD-Zeo agar plates.<br>
Spreading on MD agar plates containing 100 μg/ml Zeocin™ resulted in plates blotched with Pichia pastoris cells and no single colonies are apparent. It seems that selection pressure was not enough to repress growth of the wild type strain. Although no fluorescence could be detected in the P. pastoris GFP-Zeo Mut8 A9 strain when glucose is present, a few GFP-Zeo proteins might be expressed in the cell conferring Zeocin™ resistance. To test higher Zeocin™ concentrations for growth inhibition of the GFP-Zeo Mut8 A9 strain drop tests as described earlier were performed.<br>
As one can clearly see in Fig. 6 increase to 200 ug/ml did not decrease cell viability (compared to 100 μg/ml) of P. pastoris strains bearing a GFP-Zeo gene under the control of the AOX1 promoter, but increase to 500 μg/ml did. It was expected that mutagenesis of the promoter should result in only slightly increased expression levels therefore a selection pressure of 500 μg/ml Zeocin™ seems to be too high. Finally 350 μg/ml were chosen for all further screenings of mutagenesis promoter variants.<br>
Due to the very complex transcriptional regulation with many promoter regions involved a random mutagenesis approach using a high mutagenesis rate is advantageous.<br>
Example 2: Generation of promoter deletions<br>
Based on the results of example 1 a second generation of deletion variants was generated. In contrast to the first series in these new deletion constructs only small and specific sequence stretches of the putative transcription factor binding sites (5-15 bp) were deleted (Table 15).<br>
Table 15: Effects of deletion of specific transcription factor binding sites on the expression level upon derepression (glucose starvation) and methanol induction. Mutations A1-A9 as well as combinations of single mutations are also quoted. All<br>
numbers are relative promoter activities compared to the wild type promoter activity under the same conditions.<br>
(Table Removed)<br>
Materials and methods;<br>
a) Mutagenesis:<br>
All deletions were introduced using the two-stage site-directed mutagenesis-protocol according to Wang et al.[34]. In a first step two separate reactions (one for a forward and one for a reverse primer) were assessed (100 ng pAOX template, 15 pmol primer, 200 μM of each dATP, dTTP, dCTP and dGTP, 2.5 U PfuOl-tra™ polymerase in a total volume of 50 μl in appropriate buffer conditions) . 25 μl of these 2 PCR reactions were combined and a second PCR reaction step was performed.<br>
1 μl of Dpnl restriction enzyme (10 u/μl) was added to 30 μl of the second PCR reaction step and incubated for 1 h at 37 °C. 1-5 μl of Dpnl digested PCR reaction were transformed into elec-trocompetent E. coli cells [16] and plated on LB-Amp plates after a 1 h regeneration time in SOC medium.<br>
Table 16: Primers for site-directed mutagenesis of tran-scription factor binding site deletions  <br>
(Table Removed)<br>
b) Pichia pastoris transformation and characterisation of clones:<br>
Plasmids constructed as described above were prepared and transformed into Pichia pastoris as described in example 1.<br>
Results and Discussion;<br>
A strong effect on the expression level is observed with the short mutations of example 2 as already described for the larger deletions of example 1 where all mutations have a significant either positive or negative effect on the promoter activity. Short deletions of specific transciption factor binding sites have strong effects on the promoter activity and give a more precise information about the regulatory properties of individual regulatory sites (e. g transcription factor binding sites). Gcrl is of special interest since its binding site is included in the A6 deletion. Sequencing of the promoter region of a pAOXA 6 deletion mutant and a colony PCR products of genomic DNA of Pichia pastoris clones revealed an additional deletion in the promoter region (Deletion of the nucleotides 737 to 738 (-217 to -216) of SEQ ID No. 1). Due to the fact that this promoter variant leads to an increased promoter activity resulting consequently in a higher expression rate under derepressing conditions the additional mutation can be introduced into a promoter according to the present invention to increase protein expression under these conditions.<br>
Promoter activity of QA-1F clones with an additional deletion (Deletion of nucleotides 736 to 741 (-218 to -213) of SEQ ID No. 1) is significantly different compared to the AQA-1F promoter without this additional deletion: The activity changes from ~30% (derepression) and -100% (Induction) of wild type activity (AOX1ΔQA-1F, see table 15) to -140% and ~70%, respectively (AOXlΔQA-lFzus, see table 15). The additional deletion of these 6 nucleotids seems to have a dramatic influence on the promoter activity. Thus a new promoter variant bearing this mutation (Δ736-741) was introduced by the site-directed mutagen-esis protocol as described above. Both mutations which came up<br>
two times accidentially and independently in this region resulted in an increase of the promoter activity under derepressing conditions. It is notable that there is an increase in promoter activity although there is a second and most probably negatively influencing mutation in both constructs.<br>
A combination of A2 and A6 (A2A6) was generated similar to single deletions by overlap extension PCR. It is clearly shown in table 17 that a deletion of both fragments results in a very strong decrease of promoter activity under derepression as well as induction conditions. Since there is no additional TA deletion in this construct compared to the A6* construct as aforementioned also this result supports the speculation that the ac-cidantially arised additional mutation (A737-38) is responsible for the increase in promoter activity upon carbon starvation.<br>
Several deletions result in a dramatic decrease of promoter activity (e.g. Hsf, but also Hapl and Hap2345_l). These putative binding sites are brilliant targets for a sequence duplication which should result in an increase of promoter activity.<br>
Interestingly, in 2 out of 4 clones of the A736-741 variant generated by site-directed mutagenesis a new deletion of 9 nuc-leotides (TTGGTATTG) at position 552 to 560 (-402 to -394) was found. The effect that deletions were found in a distinct region was also found in ΔHsf_2 constructs. Such an effect is expected to be due to local sequence homology. Thus such additionally deleted regions (Δ552-560, Δ737-38 and Δ736-41) and the sequences in close proximity (5 bp up- and downstream) are also putative transcription factor binding sites and therefore highly interesting targets for deletions and duplications. The deletion variant Δ736-41 results in an enhanced reduction of the expression level under methanol inducing conditions.<br>
Multicopy-strains:<br>
In most cases generation of multicopy strains results in GFP-Zeo super expressing strains. In many cases these strains have higher expression levels than the d6*FlO strains, mainly under methanol inducing conditions. The generation of multicopy strains was achievable with several constructs, especially with the A6* construct, the double deletion d2d6, constructs including Δl, Δ2 and Δ6* deletions as well as e.g. Gcrl, Rapl, abaA, Hap2345_l, but also e.g. QA-1F, Adrl, Hsf_2_MatlMC and Hsf_2_Hap2345_l (see figure 7). In these strains a higher ex-<br>
pression rate under inducing conditions is found compared to the d6*F10 strain. In contrast the d6*F10 strain was able to produce more GFP-Zeo than any other strain generated up to know under derepressing conditions. Repeated transformation of Δ6* construct into Pichia pastor is results in a high number of multicopy strains with comparable activity to the d6*F10 strain, especially under derepressing conditions (figure 8).<br>
Using wild-type promoter constructs, a much lower frequency of multicopy-strains (e.g. E2 strain) was observed than using promoter variants. Although 2-4 times more transformants were analysed, the expression level of the best transformant E2 is only twice as high as single copy transformants. In conclusion transformation of promoter variants result in a higher frequency of multicopy strains and these strains are multiple more productive than mulitcopy wild-type promoter strains.<br>
Example 3: Alternative reporter proteins<br>
To test the practicability of all the GFP-Zeo results for other, basically well expressed and industrially relevant proteins (e.g. enzymes) some promoter variants were cloned in front of such reporter enzymes (e.g. PaHNLSa and HRP).<br>
Cloning:<br>
Promoter variants were cloned into vectors pPICZαB-HRP-WT [35]und pGAPZ A-PaHNL5α. For the promoter exchange in pPICZαB-HRP-WT a Ndel restriction site was inserted at the 5' end of the promoter by site-directed mutagenesis (100 ng vector as template, primer NdelPICZfor and NdelPICZrev - see table 18) . The resulting vector was called pPICZαB-NdeI-HRP-WT.<br>
Table 18: Primer for site-directed mutagenesis for introduction of a Ndel restriction site in pPICZαB-HRP-WT and for promoter exchange  <br>
(Table Removed)<br>
For the pGAPZ A-PaHNL5α expression clone the PaHNL5α gene<br><br>
was first cloned from a pHIL-D2 vector (Glieder, A., et al. An-gew. Chemie Int. Ed. 2003) into a pGAPZ A vector, resulting in plasmid pGAPZA-PaHNL5α. Cloning of promoter variants into pGAPZ A-PaHNL5a could be done directly after EcoRI/BglII digestion of pGAPZ A-PaHNL5α and pAOXA plasmids. For an exchange in pPICZOB-Ndel-HRP-WT the promoter variants were amplified by PCR using primers AOX1NDE1 and AOXlrev (see table 18, 10 ng pAOXA, 10 pmol primer AOX1NDE1 and AOXlrev, 200 μM each dNTP, 0.6 u Phusion™ Polymerase in appropriate buffer conditions and a total volume of 50 μl). The PCR products and the pPICZOCB-NdeI-HRP-WT plasmid were cloned employing Ndel/HindIII restriction sites.<br>
Transformation, growth and enzyme assays:<br>
For Pichia pastorls transformation all HRP vectors were linearised by Wdel and all PaHNL5α plasmids by BglII. Transformation was performed as described in example 1. Growth of P. pastoris strains was also done as described in example 1 with only a few exceptions. The amount of initial BMD(1%) was increased to 350 μl and after 60 hours 100 μl of culture were taken for centrifu-gation (4000 rpm, 4°C, 10 min) . Methanol induction was done exactly as described in example 1.<br>
50 μl (derepression or 10 μl (induction) of supernatant from centrifugation were taken HNL assay and 15 ul at both conditions for HRP assay.<br>
HRP assay (according to [35]):<br>
15 μl supernatant were added to 150 μl ImM ABTS/2,9mM H202 in 50mM NaOAc buffer pH 4.5 in PS microtiter plates. The absorption was followed for 5 minutes at 405nm in a Spectramax Plus384 platereader (Molecular Devices, Sunnyvale, CA, USA).<br>
HNL assay (according to [36]):<br>
50 μl or 10 μl of supernatant were added to 100 or 140 μl 0.05M phosphate-citrate buffer pH 5.0 in an UV-Star microtiter plate. The reaction was started by adding 50 μl 0.06 M mandelonitrile solution (in 0.003 M phosphate-citrate buffer pH 3.5) and followed for 5 minutes at 280 run in a Spectramax Plus384 platereader (Molecular Devices, Sunnyvale, CA, USA) .<br>
Results and discussion:<br>
The results using the alternative reporter proteins PaHNLSa and HRP clearly show the transferability of promoter activity detected using GFP-Zeo (Table 17) .<br>
Due to the lower sensitivity of the HRP assay the expression<br>
level at derepressing conditions was below the detection limit. Thus HRP expression could not be determined under derepressing conditions.<br>
Table 17: Promoter activity of several AOX1 promoter variants with alternative reporter enzymes (in brackets the relative activity compared to the wild type promoter under the same con-ditions is quoted (derepression and induction, respectively))  <br>
(Table Removed)<br>
n.d. not detectable<br>
To transfer the multicopy selection to the alternative reporter systems, AOX1 promoter variants were cloned in the appropriate HRP and PaHNL5α plasmids in front of the Zeocin resist-ence gene thus replacing the TEF1 promoter.<br>
Example 4: Alternative reporter protein GFP<br>
To test the promoter variants with GFP, promoter variants described in examples 1 and 2 were cloned in front of a cycle-3 GFP gene.<br>
Cloning:<br>
Internal BamHI and Xhol restriction sites in the cycle-3 GFP in vector pAOX were deleted by site-directed mutagenesis employing primers Bam-del-f and Xho-del-f (Table 19) and 100 ng vector as template. The GFP Fragment was amplified by PCR from the resulting plasmid (10 ng) employing primers GFP-Zeo forw (Seq. ID No. 4, Table 7, 10 pmol) and wtGFP-XhoI-r (Table 19, 10 pmol) and Phusion™ polymerase under appropriate conditions. The resulting PCR product could be cloned into vector pPICZ B employing<br>
EcoRI/XhoI restriction cut and ligation using T4 DNA Ligase. The resulting plasmid was named pPICZ-GFP.<br>
Cloning of all promoter variants into pPICZ-GFP could be done directly after BgrlII/EcoRI digestion of pPICZ-GFP and pAOXD plasmids.<br>
Table 19: Primer for site-directed mutagenesis of the cycle-3-GFP in vector pAOX and amplification of the GFP Fragment<br>
thereof. (Table Removed)<br><br>
Transformation, growth and GFP detection:<br>
For Pichia pastoris transformation all plasmids were linearized by BgllZ. Transformation was performed as described in example 1. After transformation and a 2 h regeneration phase cells were plated on YPD-Zeo agar plates containing 100 ug/ml Zeocin.<br>
Growth of Pichia pastoris, methanol induction and measurement of GFP fluorescence was done exactly as described in example 1.<br>
Results and discussion:<br>
Again, the results using GFP as reporter system show the transferability of promoter activity detected using GFP-Zeo (Table 20).<br>
Multicopy-Strains:<br>
As described in example 1 and 2, the occurrence of multicopy-strains using Zeocin as selection marker is very common. The frequency of multicopy-strains could be increased enormously by increasing the concentration of Zeocin on the selection plates to 500 and 1000 ug/ml, respectively.<br>
Table 20: Relative promoter activity of several AOX1 promoter variants with GPP and GFP-Zeo as reporter gene compared to the wild type promoter under the same conditions (derepression and induction, respectively)  <br>
(Table Removed)<br>
Example 5: Sufficiency series using GFP<br>
To test small parts of the AOX1 promoter in a system free of almost all the transcription factor binding sites, the AOX1 promoter was cut a few base pairs in front of the TATA box at positions -176 and -194 which results in basal promoter elements<br>
AOX176 and AOX194 (Table 21). To allow subsequent cloning of promoter elements in front of the basal promoter fragments as well as cloning of the basal promoter a BspTI and an JScoRI restriction site were inserted at the 5' and the 3' end, respectively.<br>
Table 21: Sequence of basal AOX1 promoter elements AOX176 and AOX194 and promoter fragments 737 and 201-214 which will be added in front of basal promoter variants. Restriction sites BspTI<br>
and EcoRI are underlined. <br>
(Table Removed)<br><br>
Cloning:<br>
Basal AOXlelements were amplified from pAOX (10 ng) using primers AOXlbasalrv (Table 21, 10 pmol) and AOXbasalfwn (10 pmol, AOX194) and AOXl76fw (10 pmol, AOX176), respectively. PCR was performed using Phusion™ polymerase (0.6 u) at appropriate conditions in a total volume of 50 μl.<br>
Promoter variant AOX176-737 was amplified by PCR using primers AOXlbasalrv and 737-38AOX176 as described above. Promoter variant AOX176-201-214 was amplified by PCR using primers AOXlbasalrv and 201-214AOX176 as described above.<br>
The resulting PCR products could be cloned into vector pPICZ-GFP employing Bglll/EcoRI restriction cut and ligation using T4 DNA Ligase thereby replacing the wild type AOX1 promoter.<br>
The 4 oligonucleotides Leu2basallf, Leu2basal2f, Leu2basallr and Leu2basal2r (25pmol each) were mixed in a total volume of 20 μl, heated to 95°C for 2 minutes and cooled down to room temperature slowly. 3 μl of the mixture were ligated with 159 ng of a pPICZ-GFP BglII/EcoRI fragment for 6 h at 16°C. After transformation into E. coli the resulting vector was called pLeu2basal-GFP.<br>
Promoter variant Leu2-737 was amplified by PCR using primers LEU2basalrv and 737-38Leu2 and pLeu2basal-GFP as template as described above. The resulting PCR product could be cloned into vector pPICZ-GFP employing BglII/EcoRI restriction cut and liga-tion using T4 DNA Ligase thereby replacing the wild type AOX1 promoter. The resulting plasmid was called pLeu2-GFP-737.<br>
Table 22: Primer for generation of basal promoter elements<br>
and sufficiency constructs. <br>
(Table Removed)<br><br>
Transformation, growth and GFP detection:<br>
For Pichia pastoris transformation all plasmids were linearized by BamHI. Transformation was performed as described in example 1. After transformation and a 2 h regeneration phase cells were plated on YPD-Zeo agar plates containing 100 pg/ml Zeocin.<br>
Growth of Pichia pastoris, methanol induction and measurement of GFP fluorescence was done exactly as described in example 1.<br>
Results and discussion;<br>
This experiment shows that addition of small elements identified in examples 1 and 2 could be used to increase the promoter strength of basal promoter elements derived from the AOX1 promoter or from the Saccharomyces cerevisiae LEU2 promoter.<br>
Multicopy-Strains:<br>
The occurrence and frequency of multicopy strains found after transformation is exactly the same as described in Example 4. The different site of linearization within the plasmid didn't have any influence on the generation of multicopy-strains.<br>
Table 23: Promoter activity of basal promoter elements without and after addition of small AOX1 promoter fragments supposed to act as regulator binding sites. GFP has been used as reporter protein. Singlecopy strains as well as multicopy strains are shown. (Table Removed)<br>
References:<br>
[I]	Nakagawa, T., et al. (1999) Yeast 15(12), 1223-30.<br>
[2] Koutz, P., et al. (1989) Yeast 5(3), 167-77.<br>
[3] Ohi, H., et al. (1994) Mol. Gen. Genet. 243(5), 489-99.<br>
[4] Gregg, J. M., et al.(1989) Mol. Cell. Biol. 9(3), 1316-23.<br>
[5] Nakagawa, T., et al. (2002) Yeast 19(12), 1067-73.<br>
[6] Sakai, Y., et al. (1998) J. Bacteriol. 180(22), 5885-5890.<br>
[7] Genu, V., et al. (2003) Eur. J. Biochem. 270(11), 2467-2475.<br>
[8] Goedecke, S., et al. (1994) Gene 139(1), 35-42.<br>
[9] Tschopp, J. F., et al. (1987) Nucleic Acids Res. 15(9),<br>
3859-76.<br>
[10]   Parpinello,  G.,  et al.   (1998)   J.  Bacteriol.  180(11),   2958-<br>
67.<br>
[11]	Alamae, T. et al. (1994) Yeast 10(11), 1459-66.<br>
[12] Inan, M. et al. (2001) J. Biosci. Bioeng. 92(6), 585-589. [13] Sreekrishna, K., et al. (1997) Gene 190(1), 55-62. [14] Romanes, M., et al. .(1998) Methods Mol. Biol. 103(55-72. [15] Quandt, K., et al. (1995) Nucleic Acids Res. 23(23), 4878-84.<br>
[16] Ausubel, F. M., et al., Current Protocols in Molecular Biology, John Wiley and Sons, New York, 2003. [17] Corpet, F. (1988) Nucleic Acids Res. 16(22), 10881-90. [18] Chenna, R., et al. (2003) Nucleic Acids Res. 31(13), 3497-500.<br>
[19] Bennett, R. P., et al. (1998) BioTechniques 24(3), 478-82. [20] Farinas, E. T., et al. (2001) Adv. Synth. Catal. 343(6), 601-606.<br>
[21] Huie, M. A. et al. (1996) Yeast 12(4), 307-17. [22] Lopez, M. C., et al. (1998) PNAS U. S. A. 95(24), 14112-7. [23] Zeng, X., et al. (1997) Genetics 147(2), 493-505. [24] Del Vescovo, V., et al. (2004) J. Mol. Biol. 338(5), 877-93.<br>
[25] Simon, M., et al. (1992) Yeast 8(4), 303-9. [26] Raschke, W. C., et al. (1996) Gene 177(1-2), 163-7. [27] Andrianopoulos, A. et al. (1994) Mol. Cell. Biol. 14(4), 2503-15.<br>
[28] Kwast, K. E., et al. (1998) J. Exp. Biol. 201 ( Pt 8)(1177-95. [29] Bourgarel, D., et al. (1999) Mol. Microbiol. 31(4), 1205-<br>
 15.<br>
[30]	Dang, V. D., et al. (1996) J. Bacteriol. 178(7), 1842-9.<br>
[31]	Hahn, J. S. et al. (2004) J. Biol. Chem. 279(7), 5169-76.<br>
[32]	WO 02/081650<br>
[33]	US 6,699,691<br>
[34]	Wang, W. et al. (1999) Biotechniques 26(4), 680-2.<br>
[35]	Morawski, B., et al. (2000) Protein Eng 13(5), 377-84.<br>
[36]	Weis, R., et al. (2004) FEMS Yeast Res.  5 , 179-189.<br><br><br><br><br>
Claims:<br>
1.	A mutant Pichia pastoris alcohol oxidase 1 (AOX1) promoter of<br>
the wild type Pichia pastoris AOX1 promoter (SEQ ID No. 1) com<br>
prising at least one mutation selected from the group consisting<br>
of:<br>
a)	a transcription factor binding site (TFBS),<br>
b)	nucleotides 17-0 to 235 (-784 to -719), nucleotides 170 to<br>
191 (-784 to -763), nucleotides 192 to 213 (-762 to -741), nuc<br>
leotides 192 to 210 (-762 to -744), nucleotides 207 to 209 (-747<br>
to -745), nucleotides 214 to 235 (-740 to -719), nucleotides 304<br>
to 350 (-650 to -604), nucleotides 364 to 393 (-590 to -561),<br>
nucleotides 434 to 508 (-520 to -446), nucleotides 509 to 551<br>
(-445 to -403), nucleotides 552 to 560 (-402 to -394), nucleotides 585 to 617 (-369 to -337), nucleotides 621 to 660 (-333 to -294), nucleotides 625 to 683 (-329 to -271), nucleotides 736 to 741 (-218 to -213), nucleotides 737 to 738 (-217 to -216), nucleotides 726 to 755 (-228 to -199), nucleotides 784 to 800<br>
(-170 to -154) or nucleotides 823 to 861 (-131 to -93) of Seq ID No. 1, and combinations thereof.<br>
2.	Promoter according to claim 1, characterised in that the pro<br>
moter comprises further a mutation of nucleotides 694 to 723<br>
(-260 to -231) and/or nucleotides 729 to 763 (-225 to -191) of Seq ID No. 1.<br>
3.	Promoter according to claim 1 or 2, characterised in that the<br>
mutation is a deletion, a substitution, an insertion, an inver<br>
sion and/or a multiplication.<br>
4.	Promoter according to any one of claims 1 to 3, characterised<br>
in that the transcription factor binding site (TFBS) is selected<br>
from the group consisting of Hapl, Hsf, Hap234, abaA, Stre,<br>
Rapl, Adrl, MatlMC, Gcrl and QA-1F.<br>
5.	Promoter according to claim 4, characterised in that the<br>
transcription factor binding site (TFBS) Hapl comprises nucle<br>
otides 54 to 58 of Seq ID No. 1, Hsf nucleotides 142 to 149 and<br>
517 to 524 of Seq ID No. 1, Hap234 nucleotides 196 to 200, 206<br>
to 210 and 668 to 672 of Seq ID No. 1, abaA nucleotides 219 to<br>
224 of Seq ID No. 1, Stre nucleotides 281 to 285 of Seq ID No. 1, Rapl nucleotides 335 to 339 of Seq ID No. 1, Adrl nucleotides 371 to 377 of Seq ID No. 1, MatlMC nucleotides 683 to 687 of Seq ID No. 1, Gcrl nucleotides 702 to 706 of Seq ID No. 1 and QA-1F nucleotides 747 to 761 of Seq ID No. 1.<br>
6.	Nucleic acid molecule comprising at least one mutant Pichia<br>
pastoris alcohol oxidase 1 (AOX1) promoter according to any one<br>
of claims 1 to 5 and at least one nucleic acid encoding a pro<br>
tein (peptide) or a functional nucleic acid , characterised in<br>
that said promoter and said nucleic acid are operably linked to<br>
gether forming a single- or multi-copy expression cassette.<br>
7.	Vector comprising a mutant Pichia pastoris alcohol oxidase 1<br>
(AOX1) promoter according to any one of claims 1 to 5 or a nuc<br>
leic acid molecule according to claim 6.<br>
8.	Cell comprising at least one mutant Pichia pastoris alcohol<br>
oxidase 1 (AOX1) promoter according to any one of claims 1 to 5,<br>
at least one nucleic acid fragment according to claim 6 or at<br>
least one vector according to claim 7.<br>
9.	Cell according to claim 8, characterised in that said cell is<br>
a eukaryotic cell, in particular a yeast cell, preferably a<br>
methylotrophic yeast cell.<br><br>
10.	Cell according to claim 9, characterised in that said<br>
methylotrophic yeast cell is selected from the group consisting<br>
of Candida, Hansenula, Pichia and Toruplosis.<br>
11.	Cell according to any one of claims 8 to 10, characterised<br>
in that the cell is a Pichia pastoris cell.<br>
12.	Kit for the expression of a selected protein comprising<br>
i) a vector according to claim 7, and<br>
ii) a cell capable to express said protein under the control of a promoter according to any one of claims 1 to 5.<br>
13.	Kit according to claim 12, characterised in that said cell<br>
is a yeast cell, preferably a methylotrophic yeast cell.<br>
14.	Kit according to claim 13, characterised in that said<br>
methylotrophic yeast cell is selected from the group consisting<br>
of Candida, Hansenula, Pichia and Toruplosis.<br>
15.	Kit according to any one of claims 12 to 14, characterised<br>
in that the cell is a Pichia pastoris cell.<br>
16.	Method for the expression of a recombinant protein, peptide<br>
or functional nucleic acid in a cell comprising the following<br>
steps:<br>
providing a nucleic acid molecule according to claim 6 or a vector according to claim 7 comprising an AOX1 promoter according to any one of claims 1 to 5 and a nucleic acid encoding for a protein, peptide or functional nucleic acid, said promoter being operably linked to said nucleic acid,<br>
transforming said cell with said vector or said nucleic acid molecule,<br>
culturing the transformed cell in a suitable culture medium,<br>
optinally inducing expression of said protein, peptide or functional nucleic acid and<br>
isolating said expressed protein, peptide or functional nucleic acid.<br>
17.	Method according to claim 16, characterised in that said<br>
cell is a yeast cell, preferably a methylotrophic yeast cell.<br>
18.	Method according to claim 17, characterised in that said<br>
methylotrophic yeast cell is selected from the group consisting<br>
of Candida, Hansenula, Pichia and Toruplosis.<br>
19.	Method according to any one of claims 16 to 18, character<br>
ised in that the cell is a Pichia pastoris cell.<br>
20.	Use of a nucleic acid molecule according to claim 6, a vec<br>
tor according to claim 7 or a cell according to any one of<br>
claims 8 to 11 for the expression of a protein, peptide or func<br>
tional nucleic acid.<br>
21.	Method for the isolation of super expression clones compris-<br>
ing the steps:<br>
a)	introducing a nucleic acid molecule or vector comprising<br>
a mutated methanol inducible promoter, preferably an AOX1 pro<br>
moter, operably linked to a nucleic acid encoding for a protein<br>
or to a functional nucleic acid and to a marker resistance gene<br>
into a cell,<br>
b)	transferring the cell of step a) to a medium comprising<br>
an appropriate selective marker, a non-repressing carbon source<br>
and methanol for the selective growth of super expression clones<br>
under inducing conditions or to a medium comprising an appropri<br>
ate selective marker and a non-repressing carbon source without<br>
methanol for the selective growth of super expression clones un<br>
der derepressing conditions,<br>
c)	incubating the cell from step b) on said medium,<br>
d)	isolating a colony of the cell obtained from step c) and<br>
e)	detecting super expressing clones by determining the ex<br>
pression rate of said cell.<br><br>
22.	Method according to claim 21, characterised in that the se<br>
lective marker is an antibiotic, preferably Zeocin or geneticin.<br>
23.	Method according to claim 21 or 22, characterised in that<br>
the selective marker is Zeocin and the marker resistance gene is<br>
sh ble gene.<br>
24.	Method according to any one of claims 21 to 23, character<br>
ised in that the nucleic acid molecule is a nucleic acid mo<br>
lecule according to claim 6.<br>
25.	Method according to any one of claims 21 to 23, character<br>
ised in that the vector is a vector according to claim 7.<br>
26.	Method according to any one of claims 21 to 25, character<br>
ised in that the cell is a yeast cell, preferably a methylo<br>
trophic yeast cell.<br>
27.	Method according to claim 26, characterised in that said<br>
methylotrophic yeast cell is selected from the group consisting<br>
of Candida, Hansenula, Pichia and Toruplosis.<br>
28.	Method according to any one of claims 21 to 27, character<br>
ised in that the cell is a Pichia pastoris cell.<br>
29.	Method according to any one of claims 21 to 28, character<br>
ised in that the non-repressing carbon source is selected from<br>
the group consisting of alanine, mannitol, sorbitol, threhalose,<br>
lactose and combinations thereof.<br>
30.	Method according to any one of claims 21 to 29, character<br>
ised in that the nucleic acid molecule or vector is introduced<br>
into the cell by transformation, preferably electroporation or<br>
chemical transformation, or by protoplast fusion, or by particle<br>
bombardment.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LS1HUEEtKDE5LTEwLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007--GPA-(19-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUFic3RyYWN0LSgwMy0wNS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Abstract-(03-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUFzc2lnbm1lbnQtKDIwLTEyLTIwMDcpLi5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Assignment-(20-12-2007)..pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUFzc2lnbm1lbnQtKDI3LTEyLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Assignment-(27-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUFzc2lnbm1lbnQtKDMwLTA0LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Assignment-(30-04-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Correspondence Others-(10-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Correspondence Others-(15-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6085-DELNP-2007-Correspondence Others-(25-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Correspondence-Others-(02-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDMtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Correspondence-Others-(03-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDktMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">6085-DELNP-2007-Correspondence-Others-(09-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Correspondence-Others-(19-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LURyYXdpbmdzLSgwMy0wNS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Drawings-(03-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMS0oMjAtMTItMjAwNykuLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-1-(20-12-2007)..pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMTMtKDI3LTEyLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-13-(27-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMTgtKDAyLTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-18-(02-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMi0oMDMtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-2-(03-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMi0oMjAtMTItMjAwNykuLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-2-(20-12-2007)..pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMy0oMDItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-3-(02-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1ERUxOUC0yMDA3LUZvcm0tMy0oMDktMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">6085-DELNP-2007-Form-3-(09-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMy0oMTAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-3-(10-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUZvcm0tMy0oMTktMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Form-3-(19-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1ERUxOUC0yMDA3LUZvcm0tMy0oMjUtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6085-DELNP-2007-Form-3-(25-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUdQQS0oMTAtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-GPA-(10-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LUdQQS0oMzAtMDQtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-GPA-(30-04-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LVBlaXRpdGlvbi0xMzctKDAzLTA1LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Peitition-137-(03-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA4NS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6085-delnp-2007-Petition-137-(10-09-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258880-a-protective-device-for-an-electrical-connection-of-a-transformer.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258882-light-emitting-unit-and-light-emitting-assembly-including-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258881</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6085/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Feb-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Feb-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Aug-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>TECHNISCHE UNIVERSITAT GRAZ</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RECHBAUERSTRASSE 12, A-8010 GRAZ AUSTRIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HARTNER FRANZ,</td>
											<td>JOKAMINISTRASSE 14/6, A-8010 GRAZ AUSTRIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GLIDER ANTON,</td>
											<td>WETZAWINKEL 20, A-8200 GLEISDORF AUSTRIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AT2006/000079</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-02-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>A 304/2005</td>
									<td>2005-02-23</td>
								    <td>Austria</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258881-mutant-aox-1-promoters by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:07:24 GMT -->
</html>
