<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258905-process-for-large-scale-production-of-plasmid-dna-by-e-coli-fermentation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:02:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258905:&quot;PROCESS FOR LARGE SCALE PRODUCTION OF PLASMID DNA BY E. COLI FERMENTATION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PROCESS FOR LARGE SCALE PRODUCTION OF PLASMID DNA BY E. COLI FERMENTATION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates generally to a method for increasing the yield of plasmid DNA production. The method includes the steps of selecting a highly productive clonal subtype of a strain of E, coli, including but not limited to the DH5 strain, harboring a DNA plasmid and cultivating said clonal subtype with fed-batch fermentation in a chemically-defined medium. The plasmid DNA production process described herein can generate record quantities of plasmid DNA when said highly productive clonal subtypes are cultivated on an industrial scale. The disclosed method can be used for the production of pharmaceutical grade DNA for use in polynucleotide vaccination and gene therapy treatment regimens.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to process for large scale production of plasmid dna by e. coli<br>
fermentation<br>
FIELD OF THE INVENTION<br>
The present invention relates to a process for the production of plasmid DNA comprising the steps of: (a) selecting a highly productive clonal subtype of a strain of E. coli harboring a DNA plasmid, wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain; and, (b) cultivating said highly productive clonal subtype with fed-batch fermentation in chemically-defined medium. In one embodiment of the present invention, the plasmid DNA production process described herein generates large quantities of plasmid DNA as a result of cultivating said highly productive clonal subtypes on an industrial scale. The present invention further relates to methods for selecting highly productive clonal subtypes of a strain of E. coli, including but not limited to highly productive clones of DH5 cells, for the production of plasmid DNA. The process and methods disclosed herein can be used to generate pharmaceutical grade plasmid DNA for polynucleotide vaccination and gene therapy treatment regimens.<br>
BACKGROUND OF THE INVENTION<br>
DNA vaccines are an innovative approach for inducing protective immunity against specific diseases encompassing the targeted delivery of plasmid DNA to cells (Montgomery, D.L. et al., 1993, Cell Biol. 169:244-247; Ulmer, J.B. et al., 1993, Science 259:1745-1749). DNA vaccines are capable of producing neutralizing antibodies, as well as inducing the more preferable cell-mediated immune ("CMI") responses. Typically, DNA vaccines are generated by first inserting into a plasmid a gene encoding an antigen of interest, said plasmid containing a promoter active in mammalian cells. The plasmid is then transformed into a recombinant microbial host such as Escherichia coli ("E. coli") where it is amplified and then purified. The plasmid DNA, normally suspended in saline, is administered to the body by either injection directly into muscle cells or by particle bombardment. The plasmid DNA internalized by the muscle cells is transcribed and translated, and the expressed protein is transported to the cell's surface for T-cell presentation. This mode of action results in subsequent humoral and CMI responses against the expressed antigen. Importantly, the administered plasmid DNA is non-infectious, does not replicate, and only encodes the protein of interest. Preclinical immunogenicity and efficacy of DNA vaccines in disease models have been demonstrated for a number of infectious diseases including cancer, allergy and autoimmune diseases (for review, see Gurunathan, S. et al., Ann. Rev. Immunol. 2000; 18:927-974). Clinical trials assessing the ability of DNA vaccines to generate protective immune responses against HIV, malaria,<br><br>
influenza, hepatitis B and cancer have been reported (for review, see Gurunathan, S. et al., 2000, Curr. Opin. Immunol. 12:442-447; Shroff, K. et al., 1999, PSTT 2:205-212; and Restifo, N. &amp; S. Rosenberg, 1999, Curr. Opin. Oncol. 11:50-57). Recently, mixed modality vaccines have demonstrated a promising strategy whereby DNA vaccines are combined with other gene-delivery systems. Preclinical data has shown that administering plasmid DNA as a prime, followed by another gene-based vector system encoding the same antigen as a boost, results in greater immune responses than if either vector is used for both the prime and boost.<br>
Plasmid DNA has additionally been approved for gene therapy treatment. Gene therapy encompasses the administration of a functional gene into the body, delivery of said gene to the target cell, and expression of the therapeutic product with the intent to selectively correct or modulate disease conditions. Gene therapy represents an alternative for the prevention, treatment or cure of genetic defects. Many plasmid DNA-based gene therapy clinical trials have been initiated (for review, see Mountain, A., 2000, TIB TECH 18:119-128; and Ferber, D., 2001, Science 294:1638-1642).<br>
For use in both polynucleotide vaccination and gene therapy regimes, genes in the form of DNA plasmids can be formulated like conventional pharmaceutical products and administered directly to patients. The potential number of human users for DNA vaccines or gene therapy to combat disease, either as part of a prophylactic or therapeutic regimen, is very high, creating a large demand for plasmid DNA. DNA vaccines for veterinary diseases will likely further increase this demand. Additionally, milligram quantities of plasmid DNA may be needed for effective treatment since it has been shown that only a small number of plasmid molecules presented to a cell reach the nucleus where the gene of interest is expressed (Leitner, W. et al., 2000, Vaccine 18:765-777). Thus, the manufacture and purification of large quantities of pharmaceutical-grade DNA is crucial.<br>
High yield plasmid DNA production processes are necessary to fully develop and exploit the advantages that both DNA vaccine and gene therapy treatment options have to offer. For these reasons, there is a continued need to increase the productivity of plasmid DNA manufacturing and purification methodologies. Many described methods for increasing plasmid DNA production for use in gene therapy or polynucleotide vaccination focus on the plasmid purification step, i.e. the downstream part of the production process; however, much less is known about how to optimize the initial fermentation step of the production process for the generation of plasmid DNA, especially for production at an industrial scale. Despite prior investigations into small scale plasmid DNA purification methodologies, it has been difficult to scale up the manufacture and purification of clinical-grade plasmid DNA (Prazeres, D.M.F. et al., 1999, TIBTECH 17:169-174).   Using non-optimized laboratory conditions<br>
for the production of plasmid DNA invariably leads to very low (5-40 mg/L) volumetric yields. Increasing the productivity of plasmid DNA manufacturing processes requires the concomitant optimization of plasmid copy number (i.e., specific yield) and bioma'fS concentration (i.e., volumetric yield). While some techniques identified for optimizing fermentation methods for recombinant protein production by E. coli on a commercial scale may be translatable to processes aimed at the overproduction of plasmid, the conditions facilitating optimal protein expression will likely differ to some degree from those necessary for achieving high plasmid copy number.<br>
PCT International Application PCT/US96/09746 (International publication number WO 96/40905) discloses a fed-batch fermentation method for generating production scale quantities of pharmaceutical grade plasmid DNA in a microorganism at high efficiencies whereby growth rate is limited to achieve optimum yield.<br>
PCT International Application PCT/EP98/01122 (International publication number WO 98/37179) discloses the use of chemically-defined medium for the fermentative production of valuable compounds on an industrial scale, in addition to the selection of a high growth strain on said chemically-defined medium after mutagenic treatment.<br>
U.S. Patent Nos. 5,981,735 and 6,503,738, issued to Thatcher et al. on Nov. 9,1999 and Jan. 7,2003, respectively, disclose a scalable method for the production of highly purified plasmid DNA in E. coli consisting of growing plasmid-containing cells to a high biomass in exponential growth and lysing the cells by raising the pH of the culture to a value in which chromosomal DNA is denatured but plasmid DNA is reversibly renatured.<br>
O'Kennedy, R. et al. (2000, J. Biotechnol. 76:175-183) show that culturing E. coli DHSoc cells harboring the plasmid pSV|3 in a semi-defined medium results in higher plasmid specific yields over the standard complex Luria Bertrani ("LB") medium formulation, demonstrating the existence of an optimum carbon/nitrogen ratio.<br>
The present invention discloses a highly productive, scalable and reproducible process for the production of plasmid DNA. The process combines the selection of highly productive clones of E. coli with the induction of plasmid amplification during fermentation as a result of utilizing a limited nutrient feeding regime in a chemically-defined medium. This process is useful for the production of plasmid DNA for gene therapy and genetic vaccination for a number of human and animal diseases, including HTV, hepatitis C and rabies.<br>
SUMMARY OF THE INVENTION<br>
The present invention discloses a process for the production of plasmid DNA, comprising the steps of: (a) selecting a highly productive clonal subtype of a strain of E. coli harboring a DNA plasmid; and, (b) cultivating said highly productive clonal subtype with fed-batch fermentation in<br>
chemically-defined medium. According to the present invention, highly productive clonal subtypes of a strain of E. coli, including but not limited to QH5 cells, harboring a DNA plasmid exhibit a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain that are similarly tested. In one embodiment of the present invention, the plasmid DNA production process described herein generates record quantities of plasmid DNA as a result of cultivating said highly productive clonal subtypes on an industrial scale. Thus, the DNA production process of the present invention can result in an increase in yield of plasmid DNA when compared to other large-scale, plasmid DNA production processes.<br>
The present invention further relates to a plasmid DNA production process as described above wherein said selection component comprises a two-step process: a first selection step wherein potential highly productive clonal subtypes of a strain of E. coli, including but not limited to the DBS strain, are isolated; followed by a second selection step wherein said potential highly productive clonal subtypes previously isolated are tested in a small-scale fermentation system to determine which clonal subtypes are highly productive. In one embodiment of the present invention, said potential highly productive clonal subtypes are selected on a chemically-defined medium.<br>
In one embodiment of the present invention, colonies of potential highly productive clonal subtypes of E. coli selected by methods disclosed hereto are phenotypically gray when plated on blood agar as an indicator. In another embodiment of the present invention, colonies of potential highly productive clonal subtypes of E. coli are phenotypically cream when plated on chemically-defined agar medium and incubated until a population of both cream-colored colonies and cream-colored colonies with brown, bulls-eyed centers has formed.<br>
In a further embodiment of the present invention, the productivity (i.e., plasmid copy number per cell) of said potential highly productive clonal subtypes of E. coli, including but not limited to DH5 cells, selected in step one of the selection process of the present invention is determined after cultivating said clonal subtypes in a small-scale fermentation system. In one embodiment of the present invention, this small-scale fermentation system consists of shake flask fermentation with a nutrient feeding regime. The shake flask fermentation system simulates the fermentation regime used in the ultimate production protocol to generate the desired plasmid DNA. In another embodiment of the present invention, the clonal subtypes evaluated using a small-scale fermentation system, including but not limited to a shake flask fermentation system, are cultivated in a chemically-defined medium. In a further embodiment, a carbon and/or nitrogen solution is continuously fed to said small-scale fermentation system when said clonal subtypes are in mid-logarithmic phase of growth.<br>
The present invention relates to a process for the production of plasmid DNA comprising cultivating highly productive E. coli clonal subtypes, including but not limited to highly productive clonal subtypes of DH5 cells, harboring a DNA plasmid using fed-batch technology. In one embodiment of the<br>
present invention, selection of potential highly productive E. coli clonal subtypes, as described herein,<br>
» occurs on chemically-defined medium. Both the subsequent evaluaJien-of said potential highly<br>
productive E. coli clonal subtypes to determine which clones exhibit a higher than normal specific productivity, and the final fermentation regime, occurs in chemically-defined medium. In one embodiment of the invention, the highly productive clonal subtypes identified as described herein are selected and/or cultivated in a chemically-defined medium selected from the group consisting of DME-P5, DME-B12, Medium C, Medium D, Medium E, Medium F and Medium G.<br>
The present invention further relates to a fermentation process, including but not limited to a large-scale fermentation process, for the production of plasmid DNA, as described herein, whereby the cultivation regime of the process comprises at least one production stage fermentation phase. In a further embodiment of the invention, a carbon and/or nitrogen solution is fed to a production stage fermentor when the selected, highly productive clonal subtype of B. coli harboring a DNA plasmid is in mid-logarithmic phase of growth. In another embodiment of the present invention, the feed solution comprises about 50% glycerol (v/v) and about 25% monosodium glutamate (w/v). In a further embodiment, the feed solution comprises about 60% glycerol (v/v).<br>
The present invention relates to methods for selecting a highly productive clonal subtype of E. coli, including but not limited to DH5 cells, for the production of plasmid DNA. In one embodiment of the present invention, said highly productive clonal subtypes of a strain of K coli are selected via a method comprising the steps of: (a) purifying colonies of a strain of E. coli harboring a DNA plasmid that are phenotypically gray when plated on blood agar as an indicator, wherein a gray-colored colony represents a potential highly productive clonal subtype; and, (b) testing productivity of said potential highly productive clonal subtypes, wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain tested under similar fermentation conditions. In another embodiment of the present invention, a highly productive clonal subtype of E. coli cells is selected via a method comprising the steps of: (a) incubating a strain of E. coli harboring a DNA plasmid plated on chemically-defined agar medium until a population of both cream-colored colonies and cream-colored colonies with brown, bulls-eye centers have formed; (b) purifying said cream-colored colonies from step (a), wherein a cream-colored colony represents a potential highly productive clonal subtype; and, (c) testing productivity of said potential highly productive clonal subtypes of step (b), wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain tested under similar fermentation conditions.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIGURE 1 shows the percentage of gray-colored colonies identified by the blood agar phenotype screening assay present in cultures of a DNA vaccine candidate, Influenza Ml. The percentage of gray colonies produced when plated on blood agar increased from 44% to 89% over the course of four enrichments, demonstrating the selective growth advantage of clonal isolates purified from gray phenotypic colonies over those purified from white-colored colonies.<br>
FIGURE 2 shows the results of a kinetic enrichment study using a HTV-Gag DNA plasmid. Varying ratios of white:gray phenotypic colonies were used to inoculate chemically-defined medium. This figure shows the percentage of gray-colored colonies identified by the blood agar phenotype screening assay present in the cultures over the course of five enrichment steps. After the third enrichment, the resulting cell populations consisted of greater than 95% gray phenotypic colonies in all the test flasks.<br>
FIGURE 3 summarizes key data gathered for a typical seed fermentor including airflow rate (panel A), agitation speed (panel A) and percent dissolved oxygen (panel B). This data was generated from a seed fermentor used to cultivate cells containing the VlJns-gag plasmid according to Plasmid Production Method 1 of Example 9.<br>
FIGURE 4 summarizes more key data gathered for a typical seed fermentor, as demonstrated by a seed fermentor used to cultivate cells containing the VlJns-gag plasmid according to Plasmid Production Method 1 of Example 9. The data includes oxygen uptake rate (panel A), carbon evolution rate (A) and optical density (panel B).<br>
FIGURE 5 summarizes the pH data for a typical seed fermentor, as demonstrated by a seed fermentor used to cultivate cells containing the VlJns-gag plasmid according to Plasmid Production Method 1 of Example 9.<br>
FIGURE 6 summarizes the key data collected for a typical production fermentor including airflow rate (panel A), agitation speed (panel A) and power (panel B). This data was generated from a production fermentor used to cultivate cells containing the VlJns-gag plasmid according to Plasmid Production Method 1 of Example 9.<br>
FIGURE 7 summarizes more key data gathered for a typical production fermentor, as demonstrated by a production fermentor used to cultivate cells containing the VlJns-gag plasmid according to Plasmid Production Method 1 of Example 9. The data includes pressure (panel A) and percent dissolved oxygen (panel B).<br>
FIGURE 8 summarizes key optical density (panel A), carbon dioxide evolution rate (panel B) and oxygen uptake rate (panel B) data gathered for a typical production fermentor, as demonstrated by a production fermentor used to cultivate cells containing the VlJns-gag plasmid according to Plasmid Production Method 1 of Example 9.<br>
FIGURE 9 summarizes key j&gt;H (panel A) and respiratory quotient (panel B) data gathered for a typical production fermentor, as demonstrated by a production fermentor used to cultivate cells containing the VUns-gag plasmid according to Plasmid Production Method 1 of Example 9.<br>
FIGURE 10 presents the carbon dioxide evolution rate ("CER") (panel A), growth (ODfioo) (panel A) and plasmid production profiles (panel B) from the production fermentor used to cultivate cells containing the VUns-gag plasmid according to Plasmid Production Method 2 of Example 9.<br>
FIGURE 11 presents the results obtained from the cultivation of cells containing the VUns-gag plasmid according to Plasmid Production Method 3 of Example 9. Panel A shows the carbon dioxide evolution rate, oxygen uptake rate ("OUR") and on-line optical density ("OD") measurements. The cells were cultivated in a production fermentor and fed with 50% glycerol (v/v) and 25% MSG (w/v) at a rate of 2,66 to 3.66 g/L/h. Panel B shows the volumetric (g plasmid/L) and specific yields (/Jg plasmid/mg dry cell weight), as well as thiamine, ammonium, and glycerol concentrations over the course of fermentation.<br>
FIGURE 12 presents the results obtained from the cultivation of cells containing the VUns-gag plasmid according to Plasmid Production Method 3 of Example 9 wherein the production fermentor was fed with 60% (v/v) glycerol at rates between 2.0 and 12 g/L/h. Panel A compares the specific yields (jig plasmid/mg dry cell weight) produced from each fermentation regime, while panel B compares the volumetric yields (g plasmid/L).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention describes a fermentation process that combines the benefits received from pre-selecting high producing bacterial clones with those related to cultivating bacteria under a limited nutrient feeding regime in a chemically-defined medium. Novel methods of selecting for highly productive clones of E. coli for the production of plasmid DNA, including but not limited to the production of plasmid DNA on a large scale, are disclosed. The process disclosed herein comprises the selection of highly productive clonal subtypes of E. coli cells capable of generating larger quantities of plasmid DNA per cell in comparison to non-selected, transformed cells; followed by the cultivation of said bacterial clones under a fermentation regime in a chemically-defined medium with limited nutrient feeding. Said fermentation regime ensures amplified plasmid productivity with a high cell density. The resulting process is highly productive, generating record high volumetric productivity in the order of 1-1.5 gram of plasmid/L, scalable and reproducible. Achieving high specific productivity (i.e., copy number per cell) and high yield of plasmid during the fermentation process is an important factor contributing to the efficient downstream purification of said plasmid DNA.<br>
The present invention relates to a process for the production of plasmid DNA comprising<br>
* •<br>
a first step of selecting a highly productive clonal subtype of a strajr^qf E. coli harboring a DNA plasmid, wherein said highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain. This selection process is exemplified herein, in both the specification and the Examples, using the DH5 strain of E. coli; however, this exemplification is not intended to limit the scope of the present invention to the use of E. coli DH5 cells solely for the fermentation process described herein. It will be known to one of skill in the art that alternate strains of E. coli can be furnished for use in the DNA production process of the present invention.<br>
The present invention is partly based upon the observation that E. coli DH5 cells transformed with a number of plasmid DNA vaccine candidates displayed culture heterogeneity, exhibiting two colony phenotypes with distinct morphologies when plated on differential and/or chemically-defined agar medium. Colony isolation and subsequent testing of each phenotype led to the discovery of specific phenotypic clonal isolates capable of increased plasmid amplification during fermentation, generating high quantities of clinical-grade plasmid DNA. Thus, the present invention describes a new fermentation process comprising a first step of selecting for highly productive clonal isolates of a strain of E. coli, including but not limited to the DH5 strain, for production of plasmid DNA.<br>
The highly productive clonal isolates selected by the methods described in the present invention are subjected to a fermentation process, including but not limited to a commercial fermentation process of a large scale, employing a limited nutrient feeding regime in a chemically-defined medium. E. coli is a non-fastidious microorganism that can grow both in rich complex organic media as well as salt-based chemically-defined media supplemented with an organic carbon source. Cultivation medium composition can both directly dictate biomass production and influence the microorganism's regulatory system, affecting plasmid volumetric yield (i.e., gram of plasmid per liter of medium) and specific yield (i.e., plasmid copy number per cell), respectively. Bacterial fermentation processes for the production of laboratory-scale DNA vaccine production generally employ a batch process in a complex and rich medium using either large shake flasks or small laboratory fermentors. Advantages of complex media are that the constituent raw materials are inexpensive, readily available and form a complete or nearly complete nutrient source for the microorganism. Batch fermentation also allows bacterial host cells to grow at a fast rate. However, complex fermentation media have several important disadvantages, especially for a large-scale commercial operation. Most importantly, complex raw materials have a chemically-undefined composition with a variable quality. A high oxygen supply is required when using complex medium for bacterial fermentation, along with high agitation speeds, aeration rates and pressures during large-scale fermentation. Foaming often occurs, due mainly to poor oxygen transfer, resulting in __  low plasmid productivity and inconsistent results. Thus, these fermentation regimes yield a low cell<br>
mass, ranging from 1 to 7 gram dry cell weight/liter, and modest plasmid yields only suitable for studies employing a limited number of small animals*(see, for example, Diogo, tyT. et al., 2000, Biotechnol. Bioeng. 68:576-583; Diogo, M. et al., 2001, /. Gene Med. 3:577-584; Drew, D. et al., 2000, Vaccine 18:2522-2532; Wang, Z. et al., 2001, Process Biochem. 36:1085-1093; and Reinikainen, P. et al., 1989, Biotechnol. Bioeng. 33:386-393).<br>
The present invention resolves the above problems by cultivating pre-selected, high-producing bacterial clones using a continuous feeding regime in a chemically-defined medium. In the past, product yields obtained using chemically-defined media on an industrial scale were typically considered to be substantially lower than those obtained using media containing complex raw materials. Thus, chemically-defined media have classically been applied to plasmid DNA production for either research purposes only or fermentation processes of a relatively small scale. The present invention describes a fermentation process for the generation of plasmid DNA using chemically-defined medium that yields record amounts of product, even with industrial-scale production processes. This process tolerates manageable operation conditions (e.g., agitation speed and aeration rate) since a reduced culture growth rate is achieved by controlled feeding of the key nutrient. While cell growth rate is greatly reduced during the nutrient feeding period, intracellular plasmid replication continues. Thus, the present invention describes a defined medium fermentation regime which maintains a high specific productivity. By controlling the microorganism's specific growth rate, the cell switches its internal cellular mechanisms from biomass generation to plasmid or protein production, resulting in amplified specific productivity (i.e., plasmid copy number per cell) (see Chen, W. et al., 1997, J. Ind. Microbiol. Biot. 18:4348; Riesenberg, D. et al, 1991, J. Biotechnol. 20:17-28). Chemically-defined media formulations also permit extensive analytical investigations, such as metabolic and quality control studies, and will be helpful in achieving a better position with respect to the regulatory environment by supporting safety and reproducibility claims, an important factor when designing a process for the production of plasmid DNA for genetic vaccination and gene therapy purposes.<br>
The present invention is drawn to a process for the production of plasmid DNA, including but not limited to the production of plasmid DNA on a large scale, which results in a record high yield of product. This process comprises selecting a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain, harboring a DNA plasmid, wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to similarly tested, non-selected, transformed E. coli clonal subtypes of the same strain. A highly productive clonal subtype selected as described herein is then cultivated with fed-batch fermentation in chemically-defined medium. The present invention further relates to a fermentation process as described above wherein said selection process to identify highly productive clonal subtypes of a strain of E. coli containing a DNA plasmid comprises a first selection step wherein potential highly productive clonal subtypes of E. coli are<br>
isolated; followed by a second selection step wherein said potential highly productive clonal subtypes previously isolated are evaluated in a small-scale fermentation system tcf Qetermine which clonal subtypes are highly productive. In the two-step selection process to identify a highly productive clonal subtype of E. coli described herein, the first selection step comprises the identification and subsequent purification of potential highly productive clonal subtypes of a strain of E. coli, including but not limited to the DH5 strain. In this first selection step, the pool of E. coli clonal subtypes, comprised of bacterial cells transformed with a DNA plasmid of interest, is reduced to include only those clonal variants that have the possibility of demonstrating an ability to generate a higher plasmid copy number per cell in comparison to the other transformed E. coli cells grown under similar fermentation conditions. Thus, the potential highly productive clonal subtypes of E. coli of the present invention have the potential of exhibiting a higher than normal specific productivity (i.e., plasmid copy number per cell).<br>
In one embodiment of the present invention, a method of selecting for potential highly productive clonal subtypes of a strain of E. coli, including but not limited to the DH5 strain, transformed with a DNA plasmid of interest comprises first observing a phenotypic heterogeneity in the colonies generated by said E. coli strain on agar medium; followed by the purification of those colonies that represent a minor component of the population of colonies generated by the transformed bacterial cells. One of skill hi the art can easily identify when phenotypic heterogeneity, including but not limited to heterogeneity hi morphological, physiological and/or biochemical characteristics, is present in a population of colonies of transformed bacterial cells. Said phenotypic heterogeneity may be due to a number of different factors, including but not limited to the presence of a clonal variant of the original bacterial strain possibly generated, for example, during the transformation process. If the phenotypic heterogeneity is due to the presence of a subtype of the original bacterial strain, one of skill hi the art will appreciate the potential of said phenotypic variant representing a clone of the original bacterial strain possessing altered growth characteristics, including but not limited to the characteristic of increased plasmid amplification.<br>
Bacterial clonal subtypes have been described in the scientific literature. Phenotype switching in Candida albicans occurs as a direct result of differential gene expression (Soil, D. et al., 1995, Can. J. Bot. 73:1049-1057). Two opaque-specific genes, PEP1 and OP4, and one white-specific gene, WH11, are responsible for the white to opaque phenotype switching in pathogenic Candida. While this is associated with virulence in Candida, a similar phenomenon may exist hi selecting bacterial clones with superior specific productivity. Colony variants have also been identified for pathogenic strains of Neisseria meningitidis. In this case, phenotype diversity is associated with intra-strain heterogeneity of lipopolysaccharides and class-5 outer membrane proteins (Poolman, J.T. et al., 1985, J. Med. Microbiol. 19:203-209). The effects of plasmid presence on the growth and enzymatic activity of E. coli DH5 has also been described by Mason, C.A. et al. (1989, Appl. Microbiol. Biotechnol. 32:54-60),<br>
demonstrating that plasmid copy number has a direct effect on the expression of host cell enzymes involved in carbon metabolism. Thus, the geieration of E. coli clonal subtypes with different growth characteristics may result from of a variety of different events, including but not limited to mutations induced by the DNA transformation process or stress imposed by cultivating the bacteria in a selectively enriched medium.<br>
Once a potential highly productive clonal isolate of E. coli containing a DNA plasmid of interest is identified and purified as described herein, the second step of the selection process is to evaluate each potential highly productive clonal subtype to determine which clonal subtypes are indeed highly productive. In one embodiment of the present invention, a small-scale fermentation system is used to test the productivity of said potential highly productive clonal subtypes isolated in step one of the selection process, ultimately identifying the clonal isolates isolated from round one of said selection process that generate a higher plasmid copy number per cell in comparison to non-selected E. coli cells transformed with the same DNA plasmid and grown under similar fermentation conditions. If a highly productive clonal subtype identified via this selection process is to be used in a large-scale fermentation regime for the production of commercial quantities of plasmid DNA, said small-scale fermentation system will simulate the fermentation conditions of the subsequent large-scale fermentation process. The specific productivity of non-selected E. coli cells (i.e., clonal isolates of E. coli that are not selected in the first round of the selection process described herein), including but not limited to DH5 cells, harboring a DNA plasmid can be readily determined by calculating the average productivity of a population of clonal isolates of said bacterial strain.<br>
One of skill in the art will recognize that bacterial clonal variants of the present invention may be identified in a number of different ways, including but not limited to observing a phenotypic (e.g., morphological) heterogeneity in a population of bacterial colonies plated on differential agar. In one embodiment of the present invention, potential highly productive clonal isolates of transformed E. coli DH5 cells form phenotypically gray-colored colonies when plated on blood agar. Said gray colonies appear irregularly shaped, flat and translucent. In comparison, the colonies formed by the major component of the population of transformed E. coli DH5 cells are white in color when plated on blood agar, circular in shape, and raised with a smooth texture. Said potential highly productive clonal isolates of E. coli DH5 that form gray-colored colonies on blood agar were initially identified when performing routine culture purity tests on final fermentor broth samples for Good Manufacturing Practice ("GMP") fermentations of influenza DNA vaccine candidates. A heterogeneous population of gray- and white-colored colonies was visible on blood agar plates. It was determined that the gray-colored colonies became the dominant phenotype in the fermentation broth samples. Clonal isolates of the gray- and white-colored colony-producing cells were purified and characterized for growth kinetics and plasmid productivity.  A correlation was identified between the clonal isolates that generate the gray phenotypic<br>
colonies and an increased specific productivity when compared to clonal isolates generating the white-<br>
colored colonies.	*<br>
The gray phenotypic colonies of transformed E. co/Q)H5 cells of the present invention which represent potential highly productive clonal isolates of said cells are distinguishable from the non-selected, white-colored DH5 colonies of the same strain when visualized on blood agar. Blood agar is a general purpose, non-selective, non-defined medium for the cultivation of fastidious and non-fastidious microorganisms. Blood agar plates typically contain 5% sheep's blood (by volume) and either a Columbia agar or a tryptic soy agar base. It is readily apparent to those skilled in the art that agar plates used to distinguish between two colony phenotypes should be incubated under conditions that facilitate bacterial colony formation. Said growth conditions can vary, for example, in incubation temperature, as well as in length of time of incubation. The incubation time and temperature may be adjusted according the conditions empirically determined to facilitate greatest visual distinction between the alternate colony phenotypes. In one embodiment of the present invention, potential highly productive clonal isolates of a strain of E. coli, including but not limited to E. coli DH5, are identified as gray-colored colonies when plated on 5% Columbia sheep's blood agar. Preferred incubation conditions to maximize the phenotypic differences between the gray- and white-colored DH5 colonies on blood agar is an incubation temperature of about 30°C for a time period of about 48 hours; however, one of skill in the art can appreciate that these numbers are not strict guidelines.<br>
In order to evaluate the growth characteristics of potential highly productive clonal isolates of the present invention, it is first necessary to purify said bacterial clones away from non-selected, transformed cells (i.e., those that produce white-colored colonies on blood agar). A clonal isolate represents a pure culture of a subtype of an original bacterial cell of interest. Thus, pure cultures are populations of cells arising from a single cell. When a mixture of cells is either spread or streaked onto an agar surface so that a single bacterial cell generates a completely separate colony, said colony represents a pure culture. Usually, said pure culture can be isolated, and subsequently propagated, by picking the single colony and re-spreading/re-streaking said colony onto another agar plate. Thus, one of skill in the art will appreciate that if multiple bacterial colonies come into contact with each other on an agar plate, repeated purification techniques using a spread plate or streak plate process will eventually result in the purification of a pure culture.<br>
In one embodiment of the present invention, potential highly productive clonal isolates of E. coli DH5 can be identified on blood agar because said clonal subtypes form gray-colored colonies in comparison to the white-colored colonies formed by the non-selected, transformed DH5 cells. However, said gray colonies, representing potential highly productive clonal isolates, are indistinguishable from colonies of non-selected, transformed DH5 cells when plated on chemically-defined agar medium. The potential highly productive clonal subtypes of the present invention that form gray-colored colonies on<br>
blood agar can be purified directly from said blood agar plates. Alternatively, it may be desirable to avoid all contact between the highly productive clonal isolates used in the final plasmid DNA production process of the present invention and any blood products. To purify potential highly productive clonal subtypes of a strain of E. coli that form phenotypically unique colonies on blood agar, for example, the gray-colored colonies as described herein, whereby the ultimate clonal subtype to be used in a final fermentation process has failed to contact any blood products, a duplicate plating technique can be utilized. A duplicate plating technique requires that the initial transformed E. coli cells, in additional to any subsequent purification intermediate, is duplicate plated on blood agar and a second type of agar medium, including but not limited to a chemically-defined agar medium. Once a heterogeneous population of phenotypically distinct colonies is visible on said blood agar plate, for example, a mixture of gray- and white-colored DH5 colonies, single colonies from the corresponding second agar plate are picked and re-spread/re-streaked onto both types of agar media. This selection and purification process is continued until the blood agar plate contains a uniform population of phenotypically distinct and unique colonies, for example, gray-colored colonies, whereby single colonies from the corresponding second agar medium represent potential highly productive clonal isolates that have never contacted blood products. In one embodiment of the present invention, the second agar medium described above is DME-P5 chemically-defined agar medium, described infra in Example 1. Once a clonal isolate of the present invention is purified, said cells may be periodically re-evaluated on blood agar, testing the purity of the culture.<br>
The first step of the process for plasmid DNA production described herein comprises selecting a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain, harboring a DNA plasmid, wherein said first step of the selection process is to identify and purify potential highly productive clonal subtypes of said strain that form phenotypically gray-colored colonies when plated on blood agar. The present invention further relates to a method for selecting a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain of E. coli, for plasmid DNA production comprising the steps of: (a) purifying colonies of a strata of E. coli harboring a DNA plasmid that are phenotypically gray on blood agar, wherein a gray-colored colony represents a potential highly productive clonal subtype; and, (b) testing productivity of said potential highly productive clonal subtypes, wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain tested under similar fermentation conditions, hi one embodiment of the present invention, the blood agar plates of said described method are incubated for about 48 hours at about 30°C.<br>
Another embodiment of the present invention comprises a method for selecting a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain of E. coli, for production of plasmid DNA comprising the steps of: (a) plating a strain of E. coli harboring a DNA<br><br>
plasmid on both blood agar and chemically-defined agar medium and incubating said plates until bacterial colonies form; (b) picking individual colonies, from the chemically-defined agar medium plate of which the corresponding blood agar plate contains a population of colonies^with a gray phenotype; (c) purifying individual colonies from step (b) on both blood agar and chemically-defined agar medium plates until the blood agar plate contains a uniform population of phenotypically gray colonies; (d) picking individual colonies purified in step (c) from the chemically-defined agar medium plate, said colonies representing potential highly productive clonal subtypes; and, (e) determining productivity of said potential highly productive clonal subtypes, wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli cells of the same strain under similar fermentation conditions. In one embodiment of the present invention, the chemically-defined agar medium described above is DME-P5 chemically-defined agar medium, described infra in Example 1.<br>
One of skill in the art will recognize that many different selection strategies are available to isolate potential highly productive bacterial clones of the present invention. One such strategy is described above, developed after observing that transformed E. coli DH5 cells may produce a heterogeneous population of phenotypically gray- and white-colored colonies on blood agar. While optimizing the fermentation regime for a number of DNA vaccine candidates, it was also observed that colonies formed from an initial transformed, recovered DH5 cell population displayed two distinct phenotypes on chemically-defined agar medium, DME-P5, described infra in Example 1, after extending the incubation period to about 5 days at about 37°C. Cream-colored colonies and cream-colored colonies containing brown centers were detected. It was later determined that clonal isolates generated from the cream-colored colonies have the potential of generating gray phenotypic colonies, described infra, when plated on blood agar. Thus, said cream-colored colonies represent a subset of E. coli DH5 cells transformed with a DNA plasmid that have the potential of being identified as highly productive clonal subtypes as per the present invention. It has been observed that clonal isolates exhibiting the cream-colored colony phenotype on DME-P5 selective agar medium can give rise to a mixed population of both the white and gray phenotypic colonies when plated on blood agar. However, no transformed DH5 clonal isolate that generates cream-colored colonies with brown centers have shown the ability to produce high plasmid DNA tilers or to give rise to gray phenotypic colonies on blood agar.<br>
The present invention relates to a fermentation process as described herein comprising a first step whereby a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain of E. coli, harboring a DNA plasmid is selected on chemically-defined medium, said selection process comprising a first selection step wherein potential highly productive clonal subtypes of E. coli are isolated. As described above, this first selection step reduces the pool of E. coli clonal subtypes of interest, comprised of bacterial cells of a particular strain transformed with a DNA plasmid, to those clones with the possibility of demonstrating an ability to generate a higher plasmid copy number per cell<br>
in comparison to non-selected, transformed E. coli cells of the samo^train grown under similar fermentation conditions. One embodiment of the present invention represents a method of selecting for said potential highly productive clonal subtypes of a strain of E. coli, including but not limited to the DH5 strain, transformed with a DNA plasmid of interest comprising first observing a phenotypic heterogeneity in the colonies generated by said bacterial cells when plated on differential and/or chemically-defined agar medium, followed by the purification of those colonies which represent a minor component of the population of colonies generated by said transformed cells, hi a further embodiment of the present invention, said potential highly productive clonal subtypes are phenotypically cream on chemically-defined agar medium that has been incubated until a population of both cream-colored colonies and cream-colored colonies with brown, bulls-eyed centers have formed. In a further embodiment of this aspect of the present invention, said cream-colored colonies are formed after incubating said chemically-defined agar medium at about 37°C for about 5 days.<br>
The present invention further relates to a method for selecting a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain of E. coli, for plasmid DNA production comprising the steps of: (a) incubating a strain of E. coli harboring a DNA plasmid plated on chemically-defined agar medium until a population of both cream-colored colonies and cream-colored colonies with brown, bulls-eye centers have formed; (b) purifying said cream-colored colonies from step (a), wherein a cream-colored colony represents a potential highly productive clonal subtype; and, (c) testing productivity of said potential highly productive clonal subtypes, wherein a highly productive clonal subtype exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli cells of the same strain tested under similar fermentation conditions. In a further embodiment of the present invention, the chemically-defined agar medium used in this method of selection, including but not limited to DME-P5 selective agar medium, described infra, is incubated at about 37°C for about 5 days.<br>
Another embodiment of the present invention comprises a method of selecting a highly productive clonal subtype of a strain of E. coli, including but not limited to the DH5 strain of E. coli, for plasmid DNA production comprising the steps of: (a) incubating a strain of E. coli harboring a DNA plasmid plated on chemically-defined agar medium until a population of both cream-colored colonies and cream-colored colonies with brown, bulls-eye centers have formed; (b) picking cream-colored colonies from step (a); (c) plating cells of said cream-colored colonies picked in step (b) on both blood agar and chemically-defined agar medium; (d) picking individual colonies from the chemically-defined agar medium of which the corresponding blood agar plate contains a population of colonies with a gray phenotype; (e) purifying individual colonies from step (d) on blood agar and chemically-defined agar nm ,mtii th»&gt; hlnnd near olate contains a uniform population of phenotypically gray colonies; (f)<br>
representing potential highly productive clonal subtypes; and, (g) testing productivity of said potential<br>
highly productive clonal subtypes, wherein a bighly productive clonal subtype exhibits a higher plasmid<br>
^jf * copy number per cell in comparison to similarly tested, non-selected, transformed E. coli cells of the same<br>
strain. In one embodiment of the present invention, the chemically-defined agar medium described in step (a) above, including but not limited to DME-P5 agar medium, described infra, is incubated at about 37°C for about 5 days; however, one of skill in the art will appreciate both that these incubation parameters are mere guidelines and how these guidelines can be altered to achieve a similar result.<br>
When initially observing the presence of a heterogeneous population of transformed E. coli cells, e.g., by visibly identifying bacterial colonies generated from said transformed cells on an agar medium displaying alternate morphological phenotypes, it is important to assess whether the apparent bacterial variants are indeed subtypes of said transformed E. coli cells and not mere contaminants. Possible contaminants may originate from foreign bacterial or non-bacterial sources. Additionally, when selecting for highly productive clones of a transformed bacterial strain for the production of plasmid DNA, it is essential to confirm that said plasmid is indeed contained within said bacterial cell. One of skill in the art will recognize that there are many ways to identify both foreign bacterial or non-bacterial contaminants, including but not limited plating clonal isolates on differential agar and performing fatty acid methyl ester ("FAME") analysis. Many techniques also exist to detect plasmid DNA within bacterial cells, including but not Limited to performing agarose gel electrophoresis of bacterial cell lysates or plating said bacteria on antibiotic containing agar medium corresponding to the antibiotic resistance gene within the plasmid. For example, in the present invention, purified, DH5 clonal isolates (i.e., those that formed either gray-colored or white-colored colonies on blood agar) were plated onto the following types of agar medium: DME-P5, see infra, containing neomycin since the plasmid in said clones contained a neomycin resistance marker; DME-P5 lacking neomycin; and LES Endo and Levine EMB, E. coli selective agar media used to distinguish and identify gram negative Enterobacteriaceae growth. Of the approximately 50 clonal isolates tested, all contained the plasmid conferring resistance. The clonal isolates tested also exhibited typical growth patterns for E. coli, particularly of the DH5 strain, showing an excellent E. coli profile match by FAME analysis.<br>
After selecting potential highly productive clonal subtypes of E. coli harboring a DNA plasmid of the present invention, the present invention includes evaluating said clonal subtypes to determine which clones identified from the first selection step possess a specific productivity greater than that of non-selected E. coli cells of the same strain, transformed with the same plasmid, and grown under similar fermentation conditions. In one embodiment of the present invention, the potential highly productive clonal isolates are evaluated using a small-scale fermentation system. The size of the small-scale fermentation system will depend upon the size of the ultimate fermentation process to be used for the selected, highly productive clonal subtypes described herein. If, for example, the highly productive<br>
clonal subtype of E. coli selected via the methods of the present invention is to be used in a fermentation process for large-scale production of plasmid DNA, a small-scale fermentation system in which to evaluate the potential highly productive clonal subtypes comprises a system that cultivates said bacterial isolates in flasks ranging from about 250 mL to about 2 L in size. Additionally, said potential highly productive clonal isolates are evaluated using a fermentation regime that simulates the final commercial, large-scale fermentation process. The small-scale fermentation system will allow for rapid screening of the potential highly productive clones and will generate productivity data that are consistent with the final fermentation process that will be used to generate the plasmid DNA. Likewise, if a selected, highly productive clonal subtype of E. coli is to be used in a smaller scale DNA production process, the small-scale fermentation regime of the present invention used to evaluate the productivity of the potential highly productive clonal subtypes isolated in step one of the selection process will comprise a system that cultivates the clonal subtypes in a smaller fermentation vessel than that which will be used in the final fermentation process. Again, the small-scale fermentation system will simulate the fermentation conditions of the ultimate fermentation regime for the production of plasmid DNA.<br>
In one embodiment of the present invention, the potential highly productive clonal isolates described herein are evaluated using a shake flask with feeding ("SEF") fermentation system whereby each flask is supplemented with continuous feeding. A SFF system represents a small-scale fermentation system wherein said clonal isolates are cultivated in a baffled shake flask no larger than about 1000 mL, and preferably, a 250 mL baffled shake flask. A highly productive clonal variant of the present invention that is identified after being evaluated in a SFF system, as described herein, can be used for the production of plasmid DNA in a large-scale, commercial fermentation process. In one embodiment of the present invention, the flasks of the SFF system used to evaluate the potential highly productive clonal isolates of E. coli, including but not limited to DH5 cells described herein, are continuously fed with a dilute glycerol/monosodium glutamate ("MSG") mixture, preferably a feed solution comprising about 4.6% glycerol (v/v) and about 2.9% MSG (w/v). Feeding is preferably initiated during the exponential growth phase of the bacteria (i.e., mid-logarithmic phase of growth) and mimics the fed-batch process used in the ultimate, large-scale fermentation process. Additionally, a slow feed strategy is used, preferably wherein said feed solution is delivered at approximately 6.4 /il/hour/mL broth, forcing the cells to grow in a slow, linear fashion. Again, this simulates the large-scale fermentation process. A preferable SFF system used to evaluate the clonal isolates selected via the methods described herein does not require pH control as it is impractical for a shake flask system. Preferably, a SFF system described herein will be properly synchronized in order to screen as many potential highly productive clonal isolates as possible.   Additionally, because of the inherent oxygen limitation in a shake flask, the clonal isolates tested in this manner are preferably cultivated in a medium that suDoorts a relatively low biomass without feeding. Therefore, when feeding is initiated, biomass can<br>
rise significantly without the culture becoming oxygen limited. In one embodiment of the present invention, the potential highly productive qjonal isolates described herein are cultivated in a chemically-defined medium related to DME-P5 medium, DME-B12 medium fsee infra Example 3 for specific composition).<br>
In one embodiment of the present invention, after evaluating the characteristics of the potential highly productive clonal subtypes of the present invention and determining which clonal subtypes identified and purified in round one of the selection process demonstrate a high specific productivity, as described supra, a highly productive clonal subtype is then cultivated with fed-batch fermentation in chemically-defined medium. Said highly productive clonal subtypes can be cultivated on an industrial scale, increasing the yield of large-scale production of plasmid DNA. Industrial- or large-scale microbial cell fermentation, as used herein, is considered to have a total fermentation volume greater than standard laboratory bioreactors which generally accommodate fermentation volumes of approximately 200L, 500L or 10QOL. Industrial- or large-scale microbial cell bioreactors can accommodate total fermentation volumes of greater than about 1000L, and can include fermentation vessels as large as 10,000 to 100,000 L.<br>
Two distinct fermentation technologies, batch and fed-batch, have been employed for plasmid over-production hi E. coli (see, for example, Riesenberg, D., 1991, Curr. Opin. Biotechnol. 2:380-384; Yee, L andH. Blanch, 1992, Biotechnol. 10:1550-1556; and Lee, S.Y., 1996, TIBTECH 14:98-105). Typically, batch fermentation is a cell culture process by which all the nutrients required for both cell growth and plasmid production are present in the fermentation vessel in. great excess at the tune of inoculation, obviating the need to make additions to the vessel. In batch fermentation, the growth rate is controlled through manipulation of environmental parameters (e.g., temperature, pH, oxygen supply) and the carbon source. Batch fermentation is severely limited with respect to achieving high biomass cultures, contributing to the generation of low plasmid volumetric yields. In a fed-batch process, either none or part of the compounds comprising one or more of the structural and/or catalytic elements of the fermentation medium is added to the fermentor in the initial phase of the fermentation process. Once the cells have attained a desired density, either all or the remaining part, respectively, of the compounds comprising one or more of the structural and/or catalytic elements is then fed to the fermentor. In fed-batch fermentation, the cell growth rate is controlled by the addition of these nutrients to the culture over an extended period of time. The compounds which are selected for feeding can be fed together or separate from each other. In a repeated fed-batch and a continuous fermentation process, the complete start medium is additionally fed during this fermentation stage. In a repeated fed-batch process, part of the fermentation broth comprising the biomass is removed at regular time intervals, whereas the removal of part of the fermentation broth occurs continuously in a continuous fed-batch fermentation process. The<br>
fermentation process is thereby replenished with a portion of fresh medium corresponding to the amount<br>
of withdrawn fermentation broth.	„<br>
By controlling nutrient availability to a level compatible with oxygen transfer capabilities of the fermentation vessel, accumulation of toxic by-products due to the creation of an oxygen-limited environment is avoided when using a fed-batch fermentation system. A desired growth rate is achieved by creating a constant environment through a tailored feeding regimen, using either a constant feed rate or following sophisticated feeding algorithms. Plasmid copy number can be positively influenced by maintaining a down-regulated growth rate, called plasmid amplification (see Reinikainen, P. et al., 1989, Biotechnol Bioeng. 33:386-393; Namdev, P.K., 1993, Biotechnol Bioeng. 41:660-670; Schmidt, T. et al, 2001, P/ace setter 5:4-6; and Chen, W.C., 1997, J. Indust. Microbiol. Biotechnol. 18:43-48). It is thought that this higher plasmid content is due to both greater plasmid stability and favored plasmid synthesis over other biochemical pathways. Thus, a two-phase strategy is often employed in productive plasmid DNA processes: (1) a biomass build-up phase where the cells grow exponentially; and (2) a slow growth phase achieved by fed-batch methodologies wherein plasmid amplification occurs. Importantly, each recombinant construct also presents its own limitations.<br>
A fermentation vessel contains the cultured cells submerged in a liquid nutrient medium. Cultivating medium is typically sterilized either before or after introduction into the fermentation vessel. In some cases, certain components of the medium cannot be sterilized together as chemical intermediates may form between certain components of the medium during heat sterilization, altering the composition of the medium and the concentration of certain nutrients in said medium. In these cases, the medium may be prepared as two or more separate media, wherein known incompatible components are kept apart during sterilization. These separate media can then be combined after sterilization to create the complete, sterile medium. Additionally, some substances, particularly certain proteins, are not amenable to heat sterilization as they may be heat-denatured. This can be avoided by filter sterilizing these substances before adding them to the cultivation medium.<br>
The fermentation vessel is usually equipped with a means to oxygenate the cells in solution. Typical means to oxygenate the tank include a stirring mechanism which is often part of the fermentation vessel itself or an inlet wherein air or oxygen is pumped into the vessel. Aside from a means to aerate the cultured cells, other useful components contained on the fermentation vessel and used in the present invention include, without limitation, probes for pH, dissolved oxygen and temperature measurements, a pressure sensor, as well as one or more ports for the addition of nutrient and/or other solutions.<br>
The present invention relates to a fermentation process for the production of plasmid DNA comprising the steps of first selecting a highly productive clonal subtype of a strain of E. coli harboring a DNA plasmid, as described supra, and then cultivating said highly productive clonal subtype<br>
with fed-batch fermentation. In one embodiment of the present invention, said highly productive clonal subtype is cultivated using a large-scale fermentation process. Bo* the selection process and die final fermentation regime of the present invention is executed using chemically-defined media. The chemically-defined media described herein have been designed to support the over-production of plasmid DNA. As used herein, the term "chemically-defined" media is understood to be media which are essentially composed of chemically-defined constituents. A fermentation medium which is essentially composed of chemically-defined constituents includes a medium which does not contain a complex carbon and/or nitrogen source. Therefore, a chemically-defined medium essentially does not contain undefined nitrogen (e.g., animal or plant protein, or protein hydrolysate compositions) or carbon sources (e.g., molasses or corn steep liquor). Instead, the nitrogen sources are well-defined inorganic or organic compounds, and the carbon source is a well-defined sugar. Additionally, a chemically-defined medium contains mineral components, such as salts, e.g., sulfates, phosphates and chlorides of alkaline and earth alkaline metals, and micronutrients.<br>
One embodiment of the present invention relates to a series of chemically-defined medium formulations which comprise a salt component of potassium phosphate monobasic (KHiPdj), potassium phosphate (KzHPCU) and ammonium sulfate ((NHSO). In a further embodiment, said salt component of the disclosed chemically-defined media optionally includes sodium chloride (Nad). A particular embodiment of the present invention relates to a chemically-defined medium which comprises about 7.0 g/L KH2PO4, about 7.0 g/L K2HPO4 and about 6.0 g/L (NHOaSO* and optionally comprises about 0.5 g/LNaCl.<br>
In a further embodiment of the present invention, said chemically-defined medium used for the selection and/or fermentation steps of the plasmid DNA production process disclosed herein comprises a salt component, as described above, in addition to a carbon source, including but not limited to glycerol, and/or a nitrogen source, including but not limited to monosodium glutamate ("MSG") and L-glutamic acid. MSG is the sodium salt of L-glutamic acid. A particular embodiment of the present Invention relates to a chemically-defined medium which comprises a salt component, as described herein, in addition to glycerol used as a carbon source, preferably at a concentration of about 10.0-15.0 g/L, and more preferably 15.0 g/L. Another particular embodiment of the present invention relates to a chemically-defined medium which comprises a salt component and a carbon source as described herein, in addition to a nitrogen source of MSG or L-glutamic acid at a concentration of about 5.0 g/L. In a still further embodiment of the present invention, said chemically-defined medium on which the highly productive clonal isolates of the present invention are selected and then later used for cultivation of said bacterial clones optionally comprises one or more of the following components: ucon, thiamine hydrochloride, MgSO4.7H2O, neomycin sulfate and trace elements.<br>
A chemically-defined medium of the present invention used to select and/or cultivate highly productive E. coli cells described herein includes, but is not limited to, a medium selected from the group consisting of DME-P5, DME-B12, Medium C, Medium D, Medium E, Medium F and Medium G (see infra Examples section for specific media compositions and preparation). It will be appreciated that these media are only examples. A person of skill in the art will be able to provide alternative chemically-defined media which permit selection and/or cultivation of highly productive E. coli cells, including but not limited to DH5 cells, harboring a DNA plasmid as described in the novel processes disclosed herein. As such, the present invention is not limited to use of the specific media compositions described and exemplified herein but is meant to include additional, non-exemplified chemically-defined media compositions which will be amenable to the selection and/or cultivation of highly productive clonal isolates of E. coli harboring a DNA plasmid. In one embodiment of the present, said chemically-defined medium used to cultivate a highly productive clonal subtype of E, coli identified as per the methods described herein is formulated to support fermentation of said bacteria on a large scale.<br>
The DNA plasmid vector cultivated by the methods described in the present invention can be any extra-chromosomal DNA molecule containing a gene(s) encoding a biological compound of interest, i.e. a transgene(s). The plasmid will contain elements required both for its maintenance and propagation in a microbial cell (e.g., E. coli), as well as for the subsequent expression of the transgene in the animal host. For bacterial propagation, an origin of replication is needed, in addition to any plasmid encoded function required for replication, such as a selectable marker for selection of successful transfonnants. For gene expression, the plasmid should be designed to maximize transient production of the transgene upon entry into the animal host. Components of the plasmid contributing to gene expression may include, but is not limited to, a eukaryotic promoter, a transcriptional termination and polyadenylation signal, and an enhancer element(s). A selected promoter for recombinant gene expression in animal cells may be homologous or heterologous, and may be constitutive or inducible, including but not limited to promoters from human cytomegalovirus/imrnediate-early (CMVIE), simian virus/early (SV40), human elongation factor-la (EF-la) and human ubiquitin C (UbC). One of ordinary skill hi the art will further recognize how to place these various components on a vector in a particular manner so as to render them functional. Plasmid DNA can be recombinandy engineered using techniques well known to those of ordinary skill in the art, see Sambrook, Fritsch, Maniatis, Molecular Cloning, A Laboratory Manual. Second Edition, Cold Spring Harbor Laboratory Press. 1989; and Current Protocols in Molecular Biology. Greene Publishing Assoc. &amp; Wiley, 1987; both of which are incorporated by reference herein. Use of low-copy-number vectors is undesirable for plasmid DNA production for polynucleotide vaccination or gene therapy because the product yields will be unfavorably low. The plasmid vector may be transfected or transformed into the host cell using a variety of well known methods such as calcium chloride transfection, electroporation, microinjection and the like.<br>
According to the present invention, the highly prodyctive clonal subtypes of a strain of E.<br>
Tf<br>
coli identified as per the methods disclosed herein can be culthsrteti at an industrial-scale for the production of large quantities of plasmid DNA. In such a case, said large-scale fermentation regime is generally initiated by direct inoculation of a seed fermentor, preferably a small flask (i.e., about 250 mL to about 2 liters), containing a starting batch medium with transformed cells, often called the "seed" stage of the fermentation process. If the final fermentation step in which to cultivate the highly productive clonal subtypes of E. coli identified as described herein encompasses a smaller scale fermentation regime, said "seed" stage can consist of directly inoculating a small container of starting batch medium (e.g., a 15 mL sterile tube). The starting batch culture medium will typically contain all nutrients essential for growth and multiplication of the cells. The starting batch culture medium of the present invention is preferably a chemically-defined medium, including but not limited to Medium D, Medium E, Medium F and Medium G (all of which are described infra). The inocula can consist of a thawed aliquot of working seed or large seed stock (see Example 6). The cells in the seed stage are grown to a desired density, and then the contents of the "seed" fermentor are transferred to a production fermentor under sterile conditions, initiating the "production" stage of the fermentation process. For large-scale fermentation regimes, it is the size of the production fermentation that dictates the designation of said fermentation regime as an "industrial-scale" or "large-scale." In one embodiment of the present invention, the timing for transfer between the seed and production fermentation phases is based on the transformed, bacterial cells attaining a mid-logarithmic phase of growth, as determined, for example, by on-line measurements of carbon dioxide evolution rate ("CER"). The cultivation medium of the present invention used in the production stage of fermentation is a chemically-defined medium, including but not limited Medium D, Medium E, Medium F and Medium G. The fermentation conditions of both the seed fermentor and the production fermentor can vary in terms of temperature, airflow rate, agitation speed, vessel pressure and pH. In one embodiment of the present invention, E. coli harboring a DNA plasmid of interest are grown at approximately 37°C. The airflow rate of the seed and production fermentors is preferably set with a range of about 0.25 to 1.00 vvm (volume of air/volume of broth/min). The agitation speed of the seed and production fermentors is preferably sent with a range of about 200 to 800 rpm; however, the agitation speed is dependent on the size of the fermentor, larger vessels requiring lower agitation rates. The pressure of the seed and production fermentors is maintained at a range of about 5 to about 20 PSI. As demand increases during fermentation, the dissolved oxygen level can be maintained by increasing the agitation speed. A dissolved oxygen level of greater or equal to about 30% is preferred, as well as a neutral pH. A neutral pH can be maintained by addition of 25% (v/v) phosphoric acid or 30% (v/v) sodium hydroxide. On-line measurements of dissolved oxygen levels, CER, oxygen uptake rate ("OUR"), pH and cell density of fermentation broth can be made. In an alternative process, the<br>
production stage can be directly inocukted from a frozen inoculum'squrce, skipping the seed<br>
fermentation step.<br>
The production stage of the fermentation process of the present invention comprises a fed-batch system. A preferred embodiment of the present invention comprises a method of increasing the yield of plasmid DNA production, including but not limited to large-scale production of plasmid DNA, comprising cultivating E. coli DH5 cells harboring a DNA plasmid of interest with a fed-batch fermentation regime, wherein the fed-batch regime is employed at the production stage of the. fermentation process. In one embodiment of the present invention, a carbon and/or a nitrogen source is fed to the fermentation vessel at the production stage. A carbon source may include, but is not limited to, glycerol, glucose, fructose, sucrose, maltose, lactose, sorbitol or other simple sugars. A nitrogen source may include, but is not limited to, protein hydrolysates of casein, lactalbumin, albumin, soy protein; meat protein; and, MSG or mixtures of individual amino acids. In one embodiment of the present invention, the carbon source is glycerol, and the nitrogen source is MSG. In another embodiment of the present invention, a feeding solution comprising about 50% glycerol (v/v) and about 25% MSG (w/v) is used. In a further embodiment of the present invention, a feeding solution comprising about 60% glycerol (v/v) is used. In the present invention, once the production stage culture attains logarithmic growth, the carbon and/or nitrogen feeding regimen (timing based on CER) is initiated and continues for about 24-30 hours. The objective of this feeding regimen is to reduce the growth rate, which is conducive to plasmid amplification. A considerably higher amount of carbon and nitrogen supply is tolerated in a fed-batch process as opposed to a batch process. Specifically, the amount of carbon and/or nitrogen source applied in a fed-batch process can be at least about two times higher than the highest amount applied in a batch process. This, in turn, leads to the production of a considerably greater amount of biomass in a fed-batch fermentation process in comparison to a batch fermentation process.<br>
In the present invention, the rate, timing, and volume of delivery of the feed solution into the production stage fermentor of the present invention is typically varied over the course of fermentation. For example, in one embodiment of the present invention, the feed rate is initiated at a lower set point and manually increased over a period of time to a highest value once respiratory activity (measured by CER) has peaked, varying approximately between 2.66 and 3.66 g/L/h. It is preferable that the feeding process results in the cells growing at a linear rate, as opposed to the exponential rate of growth that occurs prior to feeding. The reduced growth achieved during the feeding phase of the fermentation process allows for greater plasmid amplification during fermentation, leading to greater specific productivity and, ultimately, larger DNA plasmid yields. Alternatively, the production fermentor of the present invention can be fed at a constant rate of between about 2.0-12 g/L/h, more preferably at a constant rate not exceeding about 8.0 g/L/h, and most preferably at a constant rate of approximately 8.0 g/L/h .<br>
ci me cumvation or me c.. con nost ceils is complete, the plasmid DJNA may be obtained from the bacterial cell. E. coli cells containing the plasmid-of interest are first harvested from the fermentation medium to provide a cell paste or slurry. Any conventional means to harvest cells from a liquid medium is suitable, including but not limited to centrifugation or microfiltration. Subsequent purification typically involves a number of steps involving varying techniques such as filtration, precipitation, cesium chloride/ethidium bromide density gradients; and various forms of chromatography including ion/anion exchange, gel permeation, and reverse phase chromatography. Typically, several of these techniques will be employed in a series of steps to successively increase the purity of the plasmid DNA.<br>
All publications mentioned herein are incorporated by reference for the purpose of describing and disclosing methodologies and materials that might be used in connection with the present invention. Nothing herein is to be construed as an admission that the invention is not entitled to antedate such disclosure by virtue of prior invention.<br>
Having described preferred embodiments of the invention with reference to the accompanying figures, it is to be understood that the invention is not limited to those precise embodiments, and that various changes and modifications may be effected therein by one skilled in the art without departing from the scope or spirit of the invention as defined in the appended claims.<br>
The following examples are provided to illustrate the present invention without, however, limiting the same hereto.<br>
EXAMPLE 1 Identification and Evaluation of Two Distinct Phenotypes of E. coli DH5<br>
Chemicals - All chemicals are of reagent grade and purchased from either the Sigma Chemical Co. (St. Louis, MO) or Fisher Scientific Products (Springfield, NJ). API20E test strips were purchased from bioMerieux (Canada).<br>
Cultivation Media - DME-P5 chemically-defined medium contains the following ingredients: 7.0 g/L KH2PO4&gt; 7.0 g/L K2HPO4&gt; 6.0 g/L (NH^SO*, 5.0 g/L L-Glutamic Acid, 10 g/L glycerol, 0.5 g/L NaCl, and sodium hydroxide to bring the pH to 7.2. The medium was sterilized 30 min in the autoclave, liquid cycle. When cool, a 1:1000 dilution of a trace element mixture is added which consisted of the following trace elements dissolved in 10% HC1 and filter sterilized: 27 g/L Ferric Chloride (FeCl3.6H20), 2.0 g/L Zinc Chloride (ZnCl2.4H20), 2.0 g/L Cobalt Chloride (CoCl2.6H20), 2.0 g/L Sodium Molybdate (Na2MoO4.2H20), 1.0 g/L Calcium Chloride (CaCl2.2H20), 1.27 g/L Copper Chloride (CuCl2.2H20), and 0.5 g/L Boric Acid (H3BO3). This DME-P5/trace element mixture is stored at 4 °C. When the medium is used, a 1:120 dilution of the following Th/Mg/Neo solution is added: 24 g/L<br><br>
thiamine hydrochJoride, 240 g/L MgSO4.7HiO, and 9.6 g/L neomycin sulfate. Difco Bacto Agar, 15 g/L, was added to the above medium to prepafe agar plates. Columbjy^5-% sheep's blood agar (SBA) plates, Trypticase soy agar plates (TSA), and Levine EMB and LES Endo agar plates were purchased from Fisher Scientific.<br>
Culture Purity Assay - Fermentor broth samples were streaked onto TSA and blood agar plates and incubated at 25°C and 35°C. Plates were examined after 48 hours and 7 days for culture purity.<br>
Results -The initial observation of two distinct phenotypes of E. coli DH5 cells occurred during production of clinical material to support safety assessment studies for two potential Influenza DMA vaccines, Influenza NP and Ml. The final fermentor broth samples for the GMP fermentations exhibited two distinct colony morphologies on blood agar when tested for routine culture purity. The dominant phenotype was gray in color and appeared as an irregular shaped, flat and translucent colony. The minor phenotype was white in color and appeared as a smooth, raised and circular colony. The minor, white colonies were identical in morphology to the E. coli DH5 cloning host strain initially used. It was later determined that the two cellular phenotypes were present in the Pre-Master, Master and Working Cell Banks used for the fermentations (Table 1).<br>
Table 1. Population of white and gray pheuotypic colonies in Cell Bank and fermentation broth for Influenza NP and Ml constructs.<br>
(Table Removed)<br><br>
The presence of both phenotypes went undetected on the chemically-defined agar (DME-P5) used to maintain the cultures since both colony types appear identical when incubated for 48 hours at 37°C on this agar. Therefore, isolated colonies for each phenotype were obtained by duplicate streaking onto blood agar and DME-P5 agar plates. A pure gray colony was isolated from the chemically-defined<br>
colored colonies.	<br>
To further examine the gray and white clonal isolates, isolated colonies were streaked onto DME-P5 agar plates containing neomycin (the plasmid vector in these cultures contains the neomycin resistance marker), DME-P5 agar plates lacking neomycin, and E. coll differential agars (LES Endo and Levine EMB agars). Cells containing the plasmid can grow on both DME-P5 plates, with or without neomycin. LES Endo and Levine EMB agars are selective agars used to distinguish and identify gram negative Enterobacteriaceae and E. coli. Approximately 50 colonies isolated from the GMP fermentations and Working Cell Banks were evaluated on the above agars. All of the clonal isolates grew on DME-P5 plates supplemented with neomycin, suggesting that they contained the plasmid conferring resistance. The presence of the plasmid in both phenotypes was confirmed by agarose gel electrophoresis for the Influenza NP and Ml constructs. Growth of these clonal isolates on the differential agar plates was typical for E. coli. The excellent E. coli profile match by API20E analysis further confirmed that the gray and white clones were indeed E. coli. FAME (fatty acid methyl ester) analysis performed on representative colonies to determine the relatedness of the two types of colonies indicated that the clonal isolates were identical to each other and to the DH5 cloning host strain, regardless of their phenotype on blood agar. The above results were strong evidence that the gray and white colonies were not different organisms but phenotypic variants of E. coli DH5.<br>
A screening assay was developed to distinguish between the two colony morphologies using visual examination. A mixture of Influenza Ml white and gray clonal isolates were serially diluted, plated onto 5% Columbia Sheep's blood agar plates and incubated at 30°C, 37°C or 42°C for specified periods of time. The optimum condition for phenotype differentiation was an incubation period of 48 hours at 30°C. This blood agar phenotype screening assay was used for all future plating experiments and also served as an important tool for the screening of high producers for later constructs under development.<br>
EXAMPLE 2 E. coli DH5 Gray Phenotype Enrichment Study<br>
Influenza Ml Gray Phenotype Enrichment Study - Separate cultures of white and gray phenotypic colonies containing the Influenza Ml DNA plasmid vector were grown in DME-P5 chemically-defined medium to exponential phase. The cultures were then mixed in equal proportions and used to inoculate a fresh flask of DME-P5 medium. The starting OD60o for each enrichment flask was 0.001. The mixed culture was grown for 17 generations and then inoculated into a second enrichment flask. This ensured that the cells were in exponential growth at each transfer step. This process was<br>
repeated for a total of four enrichments. An aliquot of cells was frozen in 40% glycerol (v/v) from each enrichment step and analyzed by the blood*agar phenotype screeruj^g assay described in Example 1 to determine the percentage of gray phenotypic colonies after enrichment.<br>
HIV-Gag Gray Phenotype Enrichment Study - A kinetic enrichment experiment was set up using a construct that harbored the HIV-Gag gene. Pure cultures of HIV-Gag white colonies and HTV-Gag gray colonies were mixed at the following ratios: 100% gray, 5% gray, 20% gray, 50% gray, 70% gray and 0% gray. Each flask was grown for 17 generations in DME-P5 chemically-defined medium and similarly transferred five times as described in the Influenza Ml Gray Pheuotype Enrichment Study. After each enrichment, an aliquot of cells was removed and frozen in 40% glycerol (v/v). At the end of the experiment, these samples were serially diluted and analyzed by the blood agar phenotype screening assay described in Example 1 to enumerate the population of gray phenotypic colonies after enrichment.<br>
Results - Two separate experiments were performed to investigate the observation that the gray phenotypic E. coli DH5 colonies were enriched during growth in DME-P5 chemically-defined medium. Both Influenza Ml and HIV-Gag constructs were used to determine if the enrichment phenomenon was construct specific.<br>
The Influenza Ml gray phenotype enrichment study demonstrated that the gray phenotypic colonies have a selective growth advantage over the white colonies under the growth conditions tested. The percentage of Influenza Ml gray colonies increased from 44% to 89% over the course of four enrichments in DME-P5 medium (Figure 1).<br>
To further examine this observation, a kinetic enrichment experiment was performed using an HIV-Gag construct. Varying ratios of white:gray HIV-Gag phenotypic colonies were used to inoculate DME-P5 medium. Each test flask went through a total of five enrichments. After the third enrichment (51 generations), the resulting cell populations consisted of greater than 95% of gray phenotypic colonies in all of the test flasks (Figure 2). Furthermore, the flasks initially having 100% of gray colonies did not show any reversion to the white phenotype; while the flasks starting with 100% of white colonies did show a 2-10% increase of gray phenotypic colonies through the fifth enrichment.<br>
Based on these findings, kinetic growth studies were conducted on white and gray isolated cultures obtained from four separate E. coli constructs (Influenza Ml and NP, Herpes Simplex Virus gD, and HIV-Gag). Pure cultures of gray and white phenotypic colonies for each construct were grown separately in DME-P5 medium, and a growth curve for each culture was obtained during exponential growth to determine their specific growth rates (Table 2). In all instances, the gray phenotype exhibited a slight growth rate advantage (A/z = 0.01-0.06) over the white phenotype, supporting earlier findings of enrichment of gray phenotypic colonies in a heterogeneous gray/white population.<br>
Table 2. Specific growth rates (fi) for gray and white phenotypic colonies isolated from DNA<br>
vaccine cultures.	<br><br>
(Table Removed)<br>
EXAMPLES<br>
Correlation of Plasmid DNA Concentration to Gray Phenotypic Colonies of E. coli DH5<br>
Transformation - Two methods were used to transform E. coli DH5 cells to obtain clonal isolates to be screened for plasmid productivity. For the fkst method, 100 ng (2 jiL) of plasmid DNA was added to 100 jiL of E. coli DH5 competent cells. Competent cells were prepared using standard molecular biology practices. This mixture was stored on ice for 30 minutes and then subjected to heat treatment at 42°C for 90 seconds. These tubes were chilled on ice and then 800 fiL of DME-P5 medium was added to each tube. These tubes were then incubated at 37°C for 90 minutes for recovery of antibiotic resistance. The recovered culture was then spread plated onto DME-P5 agar plates and incubated for 36 hours to obtain transformants. The second method of transformation used electro-transformation and the Bio-Rad Pulser (Hercules, CA) system. For this procedure, 80 /iL of competent cells was mixed with 0.5-4.0 /tg of plasmid DNA. This mixture was transferred to a cold electroporation cuvette and pulsed once at a setting of 1.8 kV. To this cuvette, 1.0 mL of DME-P5 medium was added. This suspension was transferred to a sterile 15 mL centrifuge tube and incubated at 37°C for 3 hours to recover antibiotic resistance. After the recovery period, the suspension was plated onto, or diluted into, DME-P5 media and incubated at 37°C to obtain transformants.<br>
23 Liter Fermentation - A 23 L bioreactor containing 15 L of DME-P5 chemically-defined media was inoculated with 0.1% (v/v) of a thawed seed suspension. The bioreactor was operated with a 150 rpm agitation (minimum set point) and a 0.3 bar backpressure and sparged with air at a rate of 7.5 L/m. The dissolved oxygen tension was maintained at 30% by computer controlled ramping of the agitation. The pH was controlled at 7.2. When the ODeoo was between 8 and 10, the airflow and backpressure were increased to 12 L/m and 1 bar, respectively, and a solution consisting of 50% glycerol (v/v):25% L-glutamic acid (w/v) was fed at a rate of 3.2 mL/L/min. The fermentation was run for 50 hours. Specific productivity was determined using cell lysis and HPLC anion exchange methods.<br>
DME-B12 Cultivation Medium - DME-B12 medium is based on DME-P5 medium. DME-B12 medium consists of the following ingredients: 7.0 g/L KH2PO4, 14.0 g/L K2HPO4, 3.0 g/L<br>
(NH4)2SO4, 0.5 g/L NaCl and 2 ml/L Glycerol. The pH was adjusted to 7.2 with 50% NaOH, and then sterilized 30 min in the autoclave, liquid c^cle. When cool, a LlQOOjdilution of a trace element mixture was added which consisted of the following trace elements dissolved in 10% HCI and filter sterilized: 27 g/L Ferric Chloride (I<ecl3.6h20 g zinc chloride cobalt sodium molybdate calcium copper and boric acid this dme-b12 element mixture is stored at when the medium used a dilution of following th solution added: thiamine hydrochloride mgso4.7h20 neomycin sulfate.></ecl3.6h20>
Shake Flask Fermentation (SFF) - This method is a scaled down version of the 23-L fermentation process described above and uses a 250 mL test flask setup. The flasks were designed with holes in their caps for feed delivery. Test cultures were grown to exponential phase (ODeoo= 1.5-2.5) in DME-B12 medium. Once cultures reached this stage, a solution consisting of 4.6% glycerol (v/v) and 2.9% L-glutamic acid (w/v) was fed at a rate of 6.4 ^L/h/mL broth. A Watson Marlow 205U pump (Wilmington, MA) was used to deliver the feed solution. The flasks were incubated at 37°C for 40 hours with constant agitation at 220 rpm. Specific productivity was determined using cell lysis and HPLC anion exchange methods.<br>
Cell Lysis Procedure - 23 L fermentation or SFF OD60o samples were measured, and an OD10 pellet was made (10/OD of culture = p.1 of sample centrifuged at 14,000 rpm, 5 min) The supernatant was removed, and the pellet was lysed as follows using standard molecular biology reagents. The pellet was first resuspended in 0.5 mL of STET buffer (8% sucrose, 5% Triton X-100, 50 mM EDTA, 50 mM Trizma base), and 0.5 mL of a lysozyme solution (4 mg/mL) was then added. The tubes were vortexed to resuspend the cells. The tubes were then incubated for 45 min at 37°C and placed in a boiling water bath for 1 min. After boiling, the tubes were centrifuged for 15 min at 14,000 rpm. The supernatant was then poured into labeled HPLC vials to which 10 //L of RNASE was added. The supernatant was then analyzed by anion exchange chromatography to quantitate the amount of supercoiled plasmid DNA.<br>
Anion Exchange HPLC - The separation of supercoiled and relaxed plasmid DNA from lysed fermentation samples was achieved employing a Waters HPLC system (Milford, MA) comprised of three pumps, a UV detector, an autoinjector and a PC computer system. A GEN-PAK FAX anion exchange column (4.6 x 100 mm) (Waters Corporation) was used. Separation was obtained by using a gradient of 1 M NaCl in 25 mM Tris-HCl, 1 mM EDTA, pH 8 (Buffer B). Buffer A was identical to Buffer B minus NaCl. A 0.04 M phosphoric acid solution (Buffer C) was used to wash the column between injections. A constant flow rate of 0.75 mL/min was used. The initial gradient of 35/65 (v/v) A:B, 3 minutes, was increased to 25/75 (v/v) A:B over 30 minutes. The column was then washed for 6 min with Buffer C, followed by 10 min of Buffer B. The system was re-equilibrated back to 70/30 (v/v)<br>
A:B for 13 min before performing the next injection. Detection at 260 nm indicated that supercoiled<br>
• *<br>
plasmid DNA eluted after 10 min and related, open-circle plasmid^DNA eluted after 9.5 min. Specific productivity of supercoiled plasmid DNA was reported as ng plasmid DNA/mL OD2 pellet or fig plasmid DNA/mg DCW (dry cell weight).<br>
Results - The data obtained from the plating experiments for the Influenza NP and Ml Cell Bank cultures and GMP fermentation samples, summarized in Table 1 and Figure 1, suggested a correlation between high plasmid productivity and the gray phenotypic colonies identified herein. GMP fermentations of Influenza Ml and NP yielded supercoiled plasmid DNA liters of 36 /ig/mg DCW and 31 fig/rag DCW, respectively. In all cases, the broth consisted of 99% gray phenotypic colonies. The plasmid copy number was determined from cell lysates for Influenza Ml and NP white and gray isolates, confirming that microbial cells isolated from gray phenotypic colonies contain a higher plasmid copy number than cells isolated from white phenotypic colonies.<br>
It was apparent from the work done with the Influenza constructs that high levels of plasmid DNA can be produced when selecting for gray phenotypic colonies after transformation. Several other constructs showed a similar correlation between gray phenotypic colonies and high plasmid yield (Table 3). For one construct in particular, HSV-gD, productivity was increased from 15 /ig/mL OD2 pellet of supercoiled DNA.<br>
Table 3. Productivity data for white and gray clonal isolates<br>
(Table Removed)<br><br>
EXAMPLE 4 Selection Strategies to Enrich for Gray Phenotypic Colonies of E. coli DH5<br>
Two strategies were developed to select and enrich for gray phenotypic colonies of E. coli DH5 after transformation. The first of these strategies relied on the fact that gray phenotypic colonies demonstrate a higher specific growth rate than white phenotypic colonies. Thus, the gray colonies will out compete the white phenotype over successive enrichments. For this protocol, transformed and recovered cells were enriched in DME-P5 chemically-defined media, with subsequent plating onto blood agar and DME-P5 agar plates. Individual colonies were picked from the DME-P5 plates based on the percentage of the gray phenotypic colonies detected on their corresponding blood agar plates. This strategy was used to identify high plasmid DNA clonal isolates for three plasmids encoding viral proteins, HSV-gB, HTV-Gag and HTV-env. During these enrichments, the percentage of gray phenotypic cells always leveled off between 5-15% of the total cell population in the enrichment flasks.<br>
While developing a second strategy for isolating high producing, gray clonal isolates, it was observed that the initial transformed, recovered cell population displayed two distinct phenotypes when plated onto DME-P5 chemically-defined agar plates and incubated for 5 days at 37°C. Both cream-colored and cream-colored colonies with a brown bulls-eye center were detected. Cream-colored colonies gave rise to the desired high producing gray phenotypic colonies, while cream-colored colonies with brown centers did not. This was demonstrated with HSV-gD. Gray clonal isolates of HSV-gD, N-11A, N-19A and N5-1A (Table 3), all produced cream-colored colonies on DME-P5 chemically-defined agar and gray colonies on blood agar. These clonal isolates also demonstrated high liters of supercoiled plasmid DNA. No isolate forming cream-colored colonies with brown centers on DME-P5 chemically-defined agar were able to produce high plasmid DNA liters or give rise to the gray phenotype. However, a single colony exhibiting the cream-colored morphology on DME-P5 agar that is picked and expanded in liquid DME-P5 media can give rise to a mixed population consisting of both white and gray phenotypes. Therefore, cream-colored colonies obtained on DME-P5 chemically-defined agar plates after transformation are pre-disposed to produce the gray phenotype. This indicates that the gray phenotype evolves from a single colony during the post-transformation selective enrichment period. Although not every cream-colored isolate will yield a gray phenotype, these are the only colony types from which the gray phenotype has evolved. Therefore, it is important to select for gray phenotypic colonies by duplicate plating on blood agar and DME-P5 chemically-defined agar plates to ensure that a high-producing clonal isolate is selected.<br><br>
EXAMPLE 5<br>
»<br>
Improved Protocol for Master Cell Bankffoduction<br>
Using the improved selection and enrichment techniques described above, high producing isolates were obtained that were used as the seed cultures to produce the Pre-Master, Master ind Working Seed stock cultures for three DNA vaccine constructs, HSV-AgB, HSV-gD and HTV-Gag. iVhen comparing the blood agar plating results for the Pre-Master, Master and Working Cell Bank :ultures produced for these constructs with earlier results obtained for the Influenza Ml and NP onstructs (Table 1), the phenotypic diversity originally present in the Master Cell Bank cultures was :liminated and high plasmid producing strains were obtained (Table 4). This was extremely critical to the linical material produced from these constructs since a heterogeneous E. coli population could have an mpact on the consistency and yield of the fermentation process. The Working Cell Bank cultures for the flV-Gag construct were later used for GMP fermentation to produce the cell paste for clinical studies. •fo cultural heterogeneity was detected in this fermentation process. The gray phenotype also did not how any reversion to the white phenotype during enrichment studies (Figure 2), indicating the stability if this phenotype.<br>
T'able 4. Summary of Cell Bank plating results for the HSV-AgB, HSV-gD and HTV-Gag constructs.<br><br>
(Table Removed)<br>
EXAMPLE 6<br>
Preparation of Seed Stocks for the Production of Plasmid DNA by Cultivation of £. coli DH5<br>
An aliquot of frozen cells of a high plasmid producing strain of E. coli DH5 selected for 5 per the process described above and harboring the VUns HTV Flgag plasmid, contained in a source<br>
of medium A (Table 5). The flasks were incubated at 37 ± 0.5 °Qji«tfvshaking (@ 180-rpm on an orbital shaker with a 50 mm shaking diameter). Two of the flasks were used for monitoring purposes and the third flask was used to prepare the frozen cell suspension. When the optical density (OD60o) of the culture of the monitoring flasks reached a value in the range of 5 to 9, the optical density of the source flask was then checked to ensure that it had also reached a similar value. The source flask was then chilled on wet ice. An equal volume of chilled 40% glycerol solution in water (v/v) was added to the contents of the flask. The mixed suspension was dispensed (~ 1 mL) in cryovials that were immediately flash frozen on dry ice and stored at -65 °C. This first seed stock was labeled "pre-master." A frozen suspension from a pre-master via! was thawed and used to prepare the master seed stock according to the protocol described above. A frozen suspension from a master vial was thawed and used to prepare the working seed stock according to the same protocol.<br>
Table 5. Defined Culture Medium A.<br>
(Table Removed)<br>
 These components form the basal medium. They are dissolved in water and the pH is adjusted to 7.2<br>
with 50% NaOH. The basal medium is sterilized by filtration through a 0.22 fj.m membrane.<br>
1: A stock solution was prepared on the day of use by dissolving Thiamine-HCl (24g/L), MgSO47H2O<br>
(240 g/L) and Neomycin Sulfate (9.6 g/L) in deionized water and filter sterilizing it (0.22 ^tm membrane).<br>
An amount of 8.3mL/L of this stock solution is added per liter of medium to yield the desired final<br>
concentrations.<br>
2: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H2O (27 g/L),<br>
ZnCl2 (2 g/L), CoCk6H2O (2 g/L), Na2MoO4.2H2O (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L),<br>
and H3BO3 (0.5 g/L). The solution is filtered sterilized through a 0.22 /Am membrane.<br>
EXAMPLE 7<br>
«•	' *<br>
reparation of Large Seed Stocks for the Productfcn'of Plasmid DNA by Cultivation of E. coli DH5<br>
A laboratory supply of frozen cells of a high plasmid producing strain of E. coli DH5 selected for as per the process described herein and harboring the VUns HIV Flgag plasmid inoculum was prepared by filling two 250-mL baffled Erlenmeyer flasks with 30 mL of medium A (Table 5). Each flask was inoculated by pipetting 30 /J.L of thawed GMP HIV-FL-gag master seed from a 4-mL vial into each flask. The flasks were incubated at 37 ± 0.5 °C with shaking at 220-rpm on an orbital shaker (Adolf Kiihner AG; Birsfelden, Switzerland) with a 50 mm shaking diameter. One flask was available for monitoring, while the second served as the inoculum source for the bioreactor. When the optical density at 600 nm (OD«x)) of the culture in the monitoring flask achieved a value in the range of 5 to 9 (approximately 26 hours post-inoculation), the ODeoo of the source flask was measured to ensure it had attained a similar value.<br>
A 30-L bioreactor (B. Braun Biotech, Inc.; Allentown, PA), interfaced with a SCADA control system utilizing Gas Works software (Thermo ONIX Corp.), was sterilized in situ containing 15 L of medium B (Table 6). The prepared bioreactor was inoculated with 15 mL (0.1% v/v) of the fresh shake-flask culture prepared as described above by injecting the inoculum through a sterilized septum in a bioreactor port with a 20-mL syringe fitted with an 18-gauge needle. The cultivation conditions were: temperature, 37°C; back-pressure, 4.5 PSP, airflow, 7.5 slpm. The dissolved oxygen level (DO), carbon evolution rate (CER), oxygen uptake rate (OUR) (Prima V Mass Spectrometer Model 600, Thermo ONIX Corp., Houston, TX), cell density (OD probe, Moaitek, Bedford, MA) and pH were simultaneously measured and recorded on-line. The DO was maintained to a set-point (greater or equal to 30% of air saturation) by automatic-feedback cascade control of the agitation speed (between 250 and 750 rpm). The culture pH was maintained to 7.1±0.1 by automatic addition of a sterile 30% NaOH Solution. Once Mid-log phase growth was attained, indicated by an on-line measurement of the CER equal to 35 .-nmol/L/hr, a 5-L volume of the culture was aseptically transferred into a sterile 10-L plastic carboy containing 5 L of a 50% glycerol solution. The carboy was placed on a magnetic stirring plate in a oiologics safety cabinet and the contents were continuously mixed to provide even suspension of the cells throughout the filling procedure. Via a peristaltic pump and sterilized silicon tubing, 500-mL bottles (Nalgene) were filled with 300 mL of the working seed. The filled bottles were initially cooled by covering with dry-ice and then were transferred to a -65°C freezer for storage.<br>
Tabled. Defined Culture Medium B.<br>
(Table Removed)<br>
1: The ingredients were added to 15 L of water, and the fermentor was steam sterilized in place at 123°C<br>
for 25 min.<br>
2: A concentrated solution (thiamine: 24 g/L; MgSO4.7H2O: 240 g/L; neomycin sulfate 9.6 g/L) is<br>
prepared and filtered sterilized into the seed fermentor to yield the desired final concentration.<br>
3: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H20 (27 g/L),<br>
ZnCl2 (2 g/L), CoCl2.6H2O (2 g/L), Na2MoO4.2H2O (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L),<br>
and HsBOs (0.5 g/L). The trace element solution was added post sterilization.<br>
The pH of the medium was adjusted to 7.1 prior to inoculation.<br>
EXAMPLE 8 Analytical procedures for Plasmid Quantification<br>
Cell Lysis - Cells from each sample were prepared for lysis by calculating the culture volume that would be required to obtain an OD of 10 in 1 mL final volume, with subsequent centrifugation of the culture in an Eppendorf Centrifuge 5415 C (Westbury, NY) for five min at 14,000 rpm. The supernatant was discarded and the pellets were stored in a -70°C freezer until the time of lysis. Upon thawing, each pellet was resuspended in 500 /xL of STET Buffer (per L distilled H2O): Tris-EDTA Buffer (Sigma), 50 mL; 0.5 M EDTA pH 8 (Sigma), 190 mL; sucrose, 80 g; Triton X-100 (Sigma), 20 g; followed by 500 /J.L of Lysozyme Solution (per L STET buffer: lysozyme (Sigma), 0.4 g). The tubes were incubated at 37°C for 45 min in an Eppendorf Thermomixer R (Westbury, NY), shaking continuously at 500 rpm. After incubation, the tubes were inserted into a floating rack and set in boiling water for one minute. The cell debris was separated by centrifugation in an Eppendorf Centrifuge 5415 C for 15 min at 14,000 rpm. The supernatant of each tube was transferred into a 1.8 mL HPLC vial, along<br><br>
with 10 [iL of RNAce-It! Ribonuclease Cocktail (Stratagene, La Jolla, CA). The vials were each capped<br>
»<br>
and gently shaken to mix the contents.	or ' ' *'<br>
HPLC Assay - Plasmid DNA was quantified by use of a HPLC system (Gilson,<br>
Middleton, WI) equipped with a Waters Gen-Pak FAX column (4.6x100 mm) (Milford, MA). Separation of the supercoiled plasmid DNA was achieved with a gradient-based elution of a mobile phase consisting of Buffer A and Buffer B delivered at a rate of 0.75 mL/min. The concentration of Buffer A [per 973 mL HPLC-Grade H2O: 1 M Tris-HCl pH 8.0, 25 mL; 0.5 M EDTA pH 8.0, 2 mL] decreased from 70% to 35% over the first two minutes of the assay, while the concentration of Buffer B [per 773 mL HPLC-Grade H2O: 1 M Tris-HCl pH 8.0, 25 mL; 0.5 M EDTA pH 8.0, 2 mL; 5 M NaCl, 200 mL] increased from 30% to 65%. At this point, the sample was injected and the mobile phase continued at 35% Buffer A and 65% Buffer B for seven minutes while the sample components eluted. The mobile phase switched to 100% Buffer C [per L HPLC-Grade H2O: 85% H3PO4 HPLC-Grade, 4.61 mL] for 12 minutes for cleaning, at which point the mobile phase returned to 70% Buffer A and 30% Buffer B. Detection was performed at 260 nm at 25°C. Under these conditions, supercoiled DNA eluted after approximately 5 minutes. Supercoiled plasmid DNA concentrations were calculated against a standard curve generated using pure plasmid DNA. The specific and volumetric plasmid yields were calculated by the automatic integration of the supercoiled DNA peak, including the volume of cell culture needed to prepare the ODio pellet and the dry cell weight measurement (DCW).<br>
EXAMPLE 9 Plasmid Production Methods<br>
Plasmid Production Method 1 - A 20-L seed fermentor was prepared to contain<br>
approximately 12.7 kg of sterile, chemically defined medium C (Table 7). Two frozen working seed vials of E, coli DH5 strain harboring the VUns HTV Flgag plasmid, prepared as described in Example 6, were thawed at ambient temperature, and 6 mL of the cell suspension were added to 200 mL of saline phosphate buffer (per L of water; NaCl: 7.0 g, KH2PO4: 0.2 g, K2H2PO4: 0.674 g). The entire volume was pumped into the 20-L seed fermentor. Initial fermentation conditions were as follows: temperature, 37°C; airflow, 2 L/min; agitation, 100 rpm; and pressure, 0.5 bar. The initial value of the pH was in the 7.0-7.1 range. No pH control was used during the inoculum production. When the initial dissolved oxygen level decreased from 100% to 50%, at about 8-12 hours post inoculation, the airflow set point was manually increased to 6 L/min. As demand increased, dissolved oxygen level was maintained at a set point greater or equal to 30% throughout the remainder of the fermentation cycle by automatic ramping of the agitation speed within the range of 100-800 rpm. Dissolved oxygen level, CER, Oxygen Uptake Rate (OUR), pH, and cell density of the fermentation broth (using a Monitek OD probe and transmitter) were all measured<br><br>
on-line. After about 17-20 hours post inoculation the cells reached mid-log phase as indicated by on-line<br>
» measurements of both CER and culture density which reach abouj^5-5O rnMoIes/L/h and 0.80-1.0<br>
absorbance units (equivalent to an off-line OD @ 600 nmbetween 8-10), respectively. At that time, using a transfer bottle, 600 mL of the seed culture were aseptically transferred from the seed fermentor to the 1000-L production fermentor.<br>
Figures 3-5 summarize the key data gathered for a typical seed fermentor used to produce cells containing the VUns gag plasmid. Figure 3, panels A and B, clearly shows that aerobic cultivation was maintained at all time, as demonstrated by the dissolved oxygen value being above 30% saturation as a result of controlling both the air flow and agitation rate. The kinetics of the metabolic activity measurements, CER and OUR (Figure 4, panel A), and on-line optical density monitoring (Figure 4, panel B) clearly indicate that the cells were actively growing without limitation when they were transferred to the production fermentor. Finally, data presented in Figure 5 show that pH value declined during active growth to reach a final value of about 6.70-6.65.<br>
A 1000-L production fermentor was batched with approximately 600 L of sterile, chemically defined medium D (Table 8). Initial fermentation conditions were as follows: temperature, 37°C; airflow, 200 L/min; agitation, 100 rpm; and pressure, 7.5 PSI Dissolved oxygen level was maintained to a set point greater or equal to 30% of air saturation ambient pressure. The pH of the culture was maintained within the range of 7.0 to 7.2 by automatic addition of 25% (v/v) phosphoric acid or 30% (v/v) sodium hydroxide throughout the fermentation cycle. Dissolved oxygen level, CER, OUR, cell density (using on-line Monitek probe), and pH of the fermentation broth were all measured on-line. At about 15 hours after inoculation, when on-line measurements for CER read about 35-40 mMoles/L/h and for culture density read about 0.70-0.80 absorbance units (equivalent to an off-line OD @ 600 nm between 8-10), feeding of a solution containing 50% glycerol and 25% monosodium glutamate was initiated. To accommodate the high dissolved oxygen demand and in order to maintain the DO &gt; 30%, the vessel pressure and the airflow were both manually increased to 15 PSI and 600 L/min respectively (at the time of feeding initiation), while the agitation was automatically ramped via a computer feed-back loop. An amount, totaling approximately 75 L of feed solution was pumped over 35 hours (rates varying approximately between 2.66 to 3.66 g/L/h). The feeding was initiated at the lower set point and manually increased over a period of 2 hours to its high value once respiratory activity (measured by CER) has peaked.<br>
Figures 6-9 summarize the key data collected for these fermentations. During the feeding process the cells were growing at a linear rate with the CER and OUR remaining approximately constant with values between 35 to 60 mM/L/h and 40 to 60 mM/L/h, respectively (Figure 8, panels A and B). After about 50 hours of cultivation, the final culture density is about 1.5 absorbance on-line units,<br>
r to an off-line density (OD @ 600 nm) of about 65 units. Figure 6 (panels A and B) shows<br><br>
that, due to the combination of increasing back pressure, agitation rate (rpm) and air flow, the dissolved oxygen (%) remained at or greater than trie 30% set point. During pe'ak' demand (ca. 20 hours into the process), the oxygen demand peaked at about 80 mM/L/h. Growth and metabolism of the cells were constantly monitored through on-line optical density and mass-spectrometry readings. Figure 8 (panel A) shows that an exponential growth phase took place up to about 20 hours post inoculation (corresponding to peak OUR and CER (panel B)), followed by a more reduced growth achieved during the feeding phase of the process, where both respiratory activity and growth rate were directly dependent of the rate of nutrient feeding. The final specific and volumetric plasmid DNA productivities were 24.45 fig plasmid per nig of dry cell weight and 0.538 g/L, respectively.<br>
Table 7. Defined Culture Medium C<br>
(Table Removed)<br>
The seed fermentor is batched and sterilized with 8 kg of purified water.<br>
1: A 10X concentrated salts solution is prepared and the pH is adjusted to 7.2 with 50% NaOH. The<br>
concentrated basal salt solution is filtered sterilized into the seed fermentor to yield the desired final<br>
concentration.<br>
2: A concentrated solution (MSG: 250g/L; glycerol: 500g/L) is prepared and filtered sterilized into the<br>
seed fermentor to yield the desired final concentration.<br>
3: Ucon is prepared as a diluted solution (1.5 %) in water, heat sterilized and aseptically transferred to the<br>
seed fermentor to yield the desired final concentration.<br>
4: A concentrated solution (thiamine: 24 g/L; MgSO4.7H2O: 240 g/L; neomycin sulfate 9.6 g/L) is prepared and filtered sterilized into the seed fermentor to yield the desired final concentration. 5: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H2O (27 g/L), ZnCl2 (2 g/L), CoCl2.6H20 (2 g/L), Na2MoO(t.2H2O (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L), and H3BO3 (0.5 g/L). A diluted solution (12 mL in 200 mL of water) is prepared and filtered sterilized <br>
Table8. Defined Culture Medium D.<br>
(Table Removed)<br>
The production fermentor is batched and sterilized with 525 kg of purified water.<br>
1: A 10X concentrated salts solution is prepared and the pH is adjusted to 7.2 with 50% NaOH. The<br>
concentrated basal salt solution is filtered sterilized into sterile carboys. The sterile 10X solution is then<br>
pumped into the fermentor through a sterile filtration apparatus. A total volume of 60 L of the 10X<br>
solution is used to yield the desired final concentration.<br>
2: A concentrated solution (MSG: 250g/L; glycerol: 500g/L) is prepared and filtered sterilized into<br>
carboys. The concentrated solution is pumped into the fermentor at the time of batching (12 L) to yield<br>
the desired concentrations. Carboys containing the same formulation solution are used for feeding<br>
purpose.<br>
3: Ucon is prepared as a diluted solution (60%) in water, heat sterilized and aseptically transferred to the<br>
seed fermentor to yield the desired final concentration.<br>
4: A concentrated solution (thiamine: 24 g/L; MgSO4.7H2O: 240 g/L; neomycin sulfate: 9.6 g/L) is<br>
prepared and filtered sterilized into the seed fermentor to yield the desired final concentrations.<br>
5: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H2O (27 g/L),<br>
ZnCl2 (2 g/L), CoCl2.6H2O (2 g/L), Na2MoOtf.2H2O (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L),<br>
and HaBO3 (0.5 g/L). The solution (600 mL) is filtered sterilized into the production fermentor.<br>
After the additions, the weight of the fermentor is brought up to 600 kg by addition of purified water<br>
through a sterile filtration apparatus.<br>
pH of the fermentation is controlled by addition of sterile NaOH (30% in water) or phosphoric acid (25%<br>
in water). Both solutions are autoclave-sterilized.<br>
Plasmid Production Method 2 - A 30-L seed fermentor was batched with 15 L of medium E (Table 9). The seed reactor was inoculated with 300 mL of frozen inoculum of E. coli DH5 strain harboring the VUns HTV Flgag plasmid prepared as described in Example 7. The thawing procedure<br>
consisted of removing the frozen NalgeneJbottle from the freezer and placing it in a stationary 37°C water bath with periodic vigorous manual shaking. Approximately 20 minutes were required for complete thawing to occur. The bottle was subsequently removed from the water bath and the thawed inoculum was aseptically transferred to an inoculation assembly. The operating conditions were as follows: temperature, 37°C; airflow, 7.5 L/min; agitation, 250 rpm; and pressure 4.5 PSI.   The initial value of the pH was in the 7.0-7.1 range. No pH control was used during the inoculum production. As demand increased, dissolved oxygen level was maintained at a set point greater or equal to 30% throughout the remainder of the fermentation cycle by automatic ramping of the agitation speed within the range of 250-750 rpm.<br>
The cells reached a carbon evolution rate (CER) of 35 mM/L/h in about 7 hours. At that time, a volume of 75 mL of the inoculum fermentor was transferred to a 30-L production fennentor that contained 15 L of medium F (Table 10). The operating conditions were as follows: temperature, 37°C; airflow, 7.5 Umin; agitation, 250 rpm; and pressure, 4.5 PSI. Dissolved oxygen level was maintained to a set point, greater or equal to 30% of air saturation at ambient pressure, by automatic cascade control of the agitation speed, between 250-700 rpm, and by increase of the airflow and back pressure set points from 7.5 to 12 L/min and from 4.5 to 15 PSI, respectively. These increases were performed at the time of MSG-glycerol feeding. The pH of the culture was maintained within the range of 7.0 to 7.2 by automatic addition of 15% (v/v) phosphoric acid or 30% (v/v) sodium hydroxide throughout the fermentation cycle. Dissolved oxygen level, CER, OUR, and pH of the fermentation broth were all measured on-line.<br>
At about 9.8 hours after inoculation, when on-line measurements for CER read about 35mMoles/L/h feeding of a solution containing 50% glycerol and 25% monosodium glutamate was automatically initiated at a constant rate of 3.2 g/L/h. To accommodate the high dissolved oxygen demand associated with the higher cell concentration and in order to maintain the DO &gt; 30%, the vessel pressure and the airflow were both automatically increased to 15 PSI and 12 L/min respectively (at the time of feeding initiation). Additional control of the DO at greater than 30% was achieved using a computer control loop to the agitator (from 250 rpm to 750 rpm). An amount totaling approximately 2.1 L of feed solution was pumped over 45 hours.<br>
A maximum biomass of 19.9 g/L of dry cell weight was achieved and specific and volumetric plasmid DNA productivities were 29.6 /ig plasmid per mg of dry cell weight and 0.588 g/L, respectively. Figure 10 (panels A and B) presents the CER, growth (ODeoo), and plasmid production profiles.<br>
Table 9. Defined Culture Medium E<br>
(Table Removed)<br>
1 : The ingredients were added to 15 L of water, and the fermentor was steam sterilized in place at 123°C for 25 min.<br>
2: A concentrated solution (thiamine: 24 g/L; MgS04.7H2O: 240 g/L; neomycin sulfate: 9.6 g/L) is<br>
prepared and filtered sterilized into the seed fermentor to yield the desired final concentrations.<br>
3: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H2O (27 g/L),<br>
ZnCl2 (2 g/L), CoCl2.6H2O (2 g/L), Na2MoO4.2H20 (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L),<br>
and H3B03 (0.5 g/L). The solution is added post sterilization to the fermentor.<br>
The pH of the medium was adjusted to 7. 1 prior to inoculation.-<br><br>
Table 10. Uelmed Culture ivieaium.<br>
(Table Removed)<br>
1: Ingredients are added to 15 L of water, and the fermentor was steam sterilized in place at 123°C for 25 minutes.<br>
2: A concentrated solution (thiamine: 72 g/L; MgSO4.7H2O: 240 g/L) is prepared and filtered sterilized into the fermentor to yield the desired final concentrations.<br>
3: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H2O (27 g/L), ZnCl2 (2 g/L), CoCl2.6H2O (2 g/L), Na2MoO4.2H2O (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L), and HsBO3 (0.5 g/L). The solution is added post sterilization to the fermentor.<br>
The pH of the medium was adjusted to 7.1 prior to inoculation.<br>
Feed solution preparation: A concentrated solution (MSG: 250g/L; glycerol: 500g/L) is prepared and<br>
autoclaved at 121 C for 30 minutes.<br>
Plasmid Production Method 3- E. coli DH5 strain harboring the VUns HIV Flgag plasmid was cultivated in 30L fermentors containing 15 L of medium G (Table 11). Each production fermentor (30-L) was inoculated with a 300 mL volume of frozen culture prepared as described in Example 7 and thawed as described in Method 2 of this Example. Operating conditions of the fermentors were similar to those described in Method 2 of this Example with the exception that the agitation set point maximum was increased to 800 rpm. The fed-batch process included the initiation of a 60% glycerol feed solution at a constant feed rate when the carbon evolution rate (CER) reached 35 mmol/L/hour. Samples were collected at various time-points following the initiation of the feed solution to monitor plasmid production. The process was terminated 48 hours post inoculation.<br>
A total of 18 production batches were profiled at feed rates ranging from 2.0-12 g/L/h. The effect of each feed rate on plasmid yield was measured by profiling a minimum of two independent batches. Samples were collected throughout the fermentation, and each was assayed twice for plasmid yield in order to examine the standard deviation of the results. Online analyses included OUR/CER<br>
and on-line OD profiling for each batch. Figure 11 (panel A) shows the results obtained for<br>
one batch cultivated at a feed rate of 8.0 g/L/h. Metabolic analysis revealed that although the<br>
concentration of thiamine decreases over time, it was not entireljylepleted by the end of the fermentation<br>
(Figure 11, panel B). Glycerol was maintained at undetectable levels during the feeding portion of the process and was likely to be the limiting nutrient. The concentration of ammonium remained between 2.5 and 1.5 g/L throughout the cultivation (Figure 11, panel B). Analyses of the plasmid content show that there is no substantial increase in specific plasmid yield with constant feed rates greater than 8.0 g/L/h (Figure 12, panel A and B). Maximum plasmid titers of approximately 30-32 fig plasmid/mg DCW specific yield where achieved for feed rates ranging between 8.0-12.0 g/L/h. Dependent on the feeding regimen, volumetric yields ranging from 0.2 g/L to 1.3 g/L were achieved.<br>
Table 11. Defined Culture Medium G<br>
(Table Removed)<br>
1: The ingredients were added to 15 L of water and the fermentor was steam sterilized in place at 123°C<br>
for 25 min.<br>
2: A concentrated solution (thiamine: 72 g/L; MgSO4.7H2O: 240 g/L) is prepared and filtered sterilized<br>
into the seed fermentor to yield the desired final concentrations.<br>
3: Trace Elements are dissolved in 1.2N HC1 as a stock solution. Composition: FeCl3.6H2O (27 g/L),<br>
ZnCl2 (2 g/L), CoCk6H20 (2 g/L), Na2MoO4.2H2O (2 g/L), CaCl2.2H2O (1 g/L), CuCl22H2O (1.27 g/L),<br>
and H3BO3 (0.5 g/L). The solution is added post sterilization to the fermentor. Feed solution preparation.<br>
A concentrated solution of glycerol (600g/L) is prepared and autoclaved at 121°C for 60 min. The pH of<br>
the medium was adjusted prior inoculation to 7.1. The pH was controlled with a 50% ammonium<br>
hydroxide solution.<br><br><br><br><br><br>
We Claim:<br>
1.	A process for production of plasmid DNA comprising:<br>
(a)	selecting a highly productive clonal subtype of a strain of E. coli transformed<br>
with a DNA plasmid comprising:<br>
(i) observing a phenotypic heterogeneity in a population of colonies generated by the transformed E. coli, and selecting as potentially highly productive clonal subtypes those colonies that represent a minor component of said phenotypic heterogeneity in said population of colonies;<br>
(ii) purifying said potentially highly productive clonal subtypes and determining the productivity of said purified, potentially highly productive clonal subtypes by measuring the plasmid copy number per cell; and,<br>
(iii) selecting as a highly productive clonal subtype a potentially highly productive clonal subtype that exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain; and,<br>
(b)	cultivating said highly productive clonal subtype with fed-batch fermentation in<br>
chemically-defined medium in a fermentation volume of greater than  1000L,<br>
Characterized in that said phenotypic heterogeneity is observed after the transformed E. coli is<br>
grown on blood agar  30°C, and<br>
Characterized in that the potentially highly productive clonal subtypes that represent the minor component of said phenotypic heterogeneity are gray colored-colonies while the major component of said phenotypic heterogeneity are white-colored colonies.<br>
2.	The process of claim 1, wherein the potentially highly productive clonal subtypes<br>
are purified from the blood agar.<br>
3.	The process of claim 2, wherein the plasmid copy number per cell of the purified, potentially highly productive clonal subtypes is determined after cultivating said clonal subtypes in a shake flask with feeding fermentation system using chemically defined medium<br>
4.	The process of claim 3, wherein said strain of E. coli is DH5.<br>
5.	The process of claim 4, wherein said chemically-defined medium comprises a medium selected from the group consisting of Medium C, Medium D, Medium E, Medium F and Medium G.<br>
6.	The process of claim 1, wherein the process further comprises<br>
(a)	duplicate plating the transformed E. coli on blood agar and an agar that does not contain blood products;<br>
(b)	growing the E. coli at  30°C until distinct colonies are visible;<br>
(c)	observing the gray colonies as the minor component of said phenotypic heterogeneity on the blood agar;<br>
(d)	determining which colonies on the agar that does not contain blood products correspond to of the gray colonies on the blood agar; and<br>
(e)	purifying said colonies from the agar that does not contain blood products that correspond to of the gray colonies on the blood agar,<br>
wherein said purified colonies are the potentially highly productive clonal subtypes.<br>
7.	The process of claim 6, wherein the plasmid copy number per cell of the purified, potentially highly productive clonal subtypes is determined after cultivating said clonal subtypes in a shake flask with feeding fermentation system using chemically defined medium.<br>
8.	The process of claim 7, wherein said strain of E. coli is DH5.<br>
9.	The process of claim 1, wherein said chemically-defined medium comprises a medium selected from the group consisting of Medium C, Medium D, Medium E, Medium F and Medium G.<br>
10.	A process for production of plasmid DNA comprising:<br>
(a)	selecting a highly productive clonal subtype of a strain of E. coli transformed<br>
with a DNA plasmid comprising:<br>
(i) observing a phenotypic heterogeneity in a population of colonies generated by the transformed E. coli when incubated on blood agar at 30°C consisting of a minor component of gray-colored colonies and a major component of white-colored colonies, and selecting as potentially highly productive clonal subtypes the gray-colored colonies;<br>
(ii) purifying said potentially highly productive clonal subtypes, and determining the productivity of said purified, potentially highly productive clonal subtypes by measuring the plasmid copy number per cell; and,<br>
(iii) selecting as a highly productive clonal subtype a potentially highly productive clonal subtype that exhibits a higher plasmid copy number per cell in comparison to non-selected, transformed E. coli clonal subtypes of the same strain; and,<br>
(b)	cultivating said highly productive clonal subtype with fed-batch fermentation in<br>
chemically-defined medium.<br>
11.	The process of claim 6, wherein the agar that does not contain blood products is a chemically-defined agar medium.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUFic3RyYWN0LSgxMS0wNi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Abstract-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWFzc2lnbWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-assigment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUFzc2lnbm1lbnQtKDExLTA2LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Assignment-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUNsYWltcy0oMTEtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Claims-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LUNsYWltcy0oMjEtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-Claims-(21-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LUNvcnJlcG9uZGVuY2UgT3RoZXJzLSgyOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-Correpondence Others-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Correspondence Others-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-Correspondence Others-(14-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-Correspondence Others-(21-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzEtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-Correspondence Others-(31-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDUtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Correspondence-Others-(05-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTQtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Correspondence-Others-(14-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Correspondence-Others-(22-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWNvcnJvc3BvZW5kcy1vdGhlci5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-corrospoends-other.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDExLTA2LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Description (Complete)-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWRlc2N0cmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-desctription(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWRyYXdpbmcucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-drawing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LURyYXdpbmdzLSgxMS0wNi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Drawings-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUZvcm0tMS0oMTQtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Form-1-(14-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUZvcm0tMi0oMTEtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Form-2-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUZvcm0tMi0oMTQtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Form-2-(14-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUZvcm0tMy0oMTEtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Form-3-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUZvcm0tMy0oMTQtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Form-3-(14-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUZvcm0tNS0oMTQtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-Form-5-(14-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUdQQS0oMDUtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-GPA-(05-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUdQQS0oMTEtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-GPA-(11-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUdQQS0oMTQtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-GPA-(14-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1ERUxOUC0yMDA2LUdQQS0oMjItMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">4509-DELNP-2006-GPA-(22-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LXBjdC0zMjYucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-pct-326.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDUwOS1kZWxucC0yMDA2LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">4509-delnp-2006-pct-373.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258904-conversion-of-oxygenate-to-propylene-using-moving-bedtechnology-and-a-separate-heavy-olefin-interconversion-step.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258906-recombinant-protein-carrying-human-papillomavirus-epitopes-inserted-in-an-adenylate-cyclase-protein-or-fragment-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258905</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4509/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Feb-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>14-Feb-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Aug-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MERCK SHARP &amp;amp; DOHME CORP.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>126 EAST LINCOLN AVENUE, RAHWAY, NEW JERSEY 07065, UNITED STATES OF AMERICA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHARTRAIN MICHEL</td>
											<td>126 EAST LINCOLN AVENUE, RAHWAY, NJ 07065-0907, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BENTLEY LAURA KIZER</td>
											<td>126 EAST LINCOLN AVENUE, RAHWAY, NJ 07065-0907, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KRULEWICZ BARBARA ANN</td>
											<td>126 EAST LINCOLN AVENUE, RAHWAY, NJ 07065-0907, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LISTNER KRISTIN M</td>
											<td>126 EAST LINCOLN AVENUE, RAHWAY, NJ 07065-0907, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SUN WEN-JUN</td>
											<td>126 EAST LINCOLN AVENUE, RAHWAY, NJ 07065-0907, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>LEE CHANYONG BRIAN</td>
											<td>126 EAST LINCOLN AVENUE, RAHWAY, NJ 07065-0907, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 19/34</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/002911</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-01-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/541,894</td>
									<td>2004-02-04</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258905-process-for-large-scale-production-of-plasmid-dna-by-e-coli-fermentation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:02:37 GMT -->
</html>
