<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/258986-isopentenyl-transferase-sequences-and-methods-of-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:44:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 258986:&quot;ISOPENTENYL TRANSFERASE SEQUENCES AND METHODS OF USE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ISOPENTENYL TRANSFERASE SEQUENCES AND METHODS OF USE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods and compositions for modulating plant development are provided. Polynucleotide sequences and amino acid sequences encoding isopentenyl transferase (IPT) polypeptides are provided. The sequences can be used in a variety of methods including modulating root development, modulating floral development, modulating leaf and/or shoot development, modulating senescence, modulating seed size and/or weight, and modulating tolerance of plants to abiotic stress. Polynucleotides comprising an IPT promoter are also provided. The promoter can be used to regulate expression of a sequence of interest. Transformed plants, plant cell, tissues, and seed are also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ISOPENTENYL TRANSFERASE SEQUENCES AND METHODS OF USE<br>
FIELD OF THE INVENTION<br>
The invention relates to the field of genetic manipulation of plants,<br>
particularly the modulation of gene activity to affect plant development and growth.<br>
BACKGROUND OF THE INVENTION<br>
Cytokinins are a class of N6 substituted purine derivative plant hormones<br>
that regulate cell division and influence a large number of developmental events,<br>
such as shoot development, sink strength, root branching, control of apical<br>
dominance in the shoot, leaf development, chloroplast development, and leaf<br>
senescence (Mok et al. (1994) Cytokinins. Chemistry, Action and Function. CRC<br>
Press, Boca Raton, FLA, pp. 155-166; Morgan (1984) Advanced Plant Physiology<br>
ed. MB., Pitman, London, UK, pp53-75; and Letham (1994) Annual Review of<br>
Plant Physiol 34:163-197). In maize, cytokinins (CK) play an important role in<br>
establishing seed size, decreasing tip kernel abortion, and increasing seed set<br>
during unfavorable environmental conditions (Cheikh et al. (1994) Plant Physiol.<br>
106: 45-51;. Dietrich et al. (1995) Plant Physiol Biochem 33:327-36). Active<br>
cytokinin pools are regulated by rates of synthesis and degradation.<br>
Until recently, roots were believed to be the major site of cytokinin<br>
biosynthesis but evidence indicates that others tissues, such as shoot meristems<br>
and developing seeds, also have high cytokinin biosynthetic activity. It has been<br>
suggested that cytokinins are synthesized in restricted sites where cell<br>
proliferation is active. The presence of several AtlPT genes in Arabidopsis and<br>
their differential pattern of expression might serve this purpose.<br>
The catabolic enzyme isopentenyl transferase (IPT) directs the synthesis of<br>
cytokinins and plays a major role in controlling cytokinin levels in plant tissues. ,<br>
Multiple routes have been proposed for cytokinin biosynthesis. Transfer RNA<br>
degradation has been suggested to be a source of cytokinin, because some tRNA<br>
molecules contain an isopentenyladenosine (iPA) residue at the site adjacent to<br>
the anticodon (Swaminathan et al. (1977) -Biochemistry 16: 1355-1360). The<br>
modification is catalyzed by tRNA isopentenyl transferase (tRNA IPT; EC 2.5.1.8),<br>
'wh'lc'h '"has b'een identified in various organisms such as Escherichia coli,<br>
Saccharomyces cerevisiae, Lactobacillus acidophilus, Homo sapiens, and Zea<br>
mays (Bartz et al. (1972) Biochemie 54:31-39; Kline et al, (1969) Biochemistry<br>
8:4361-4371; Holtz etal. (1975) Hoppe-Seyler's Z. Physiol. Chem 356:1459-1464;<br>
Golovko et al. (2000) Gene 259:85-93; and, Holtz et al. (1979) Hoppe-Seyler's 2.<br>
Physiol. Chem 359:89-101). However, this pathway is not considered to be the<br>
main route for cytokinin synthesis (Chen et al. (1997) Physiol. Plant 101:665-673<br>
and McGraw et al. (1995) Plant Hormones, Physiology, Biochemistry and<br>
Molecular Biology. Ed. Davies, 98-117, Kluwer Academic Publishers, Dordrecht).<br>
Another possible route of cytokinin formation is de novo biosynthesis of<br>
iPMP by adenylate isopenteny! transferase (IPT; EC 2.5.1.27) with dimethylallyldiphosphate<br>
(DMAPP), AMP, ATP, and ADP as substrates. Our current<br>
knowledge of cytokinin biosynthesis in plants is largely deduced from studies on a<br>
possible analogous system in Agrobacterium tumefaciens. Cells of A.<br>
tumefaciens are able to infect certain plant species by inducing tumor formation in<br>
host plant tissues (Van Montagu et al. (1982) Curr Top Microbiol Immunol 96: 237-<br>
254; Hansen et al. (1999). Curr Top Microbiol Immunol 240:21-57). To do so, the<br>
A. tumefaciens cells synthesize and secrete cytokinins which mediate the<br>
transformation of normal host plant tissues into tumors or calli. This process is<br>
facilitated by the A. tumefaciens tumor-inducing plasmid which contains genes<br>
encoding the necessary enzyme and regulators for cytokinin biosynthesis.<br>
Biochemical and genetic studies revealed that Gene 4 of the tumor-inducing<br>
plasmid encodes an isopentenyl transferase (IPT), which converts AMP and<br>
DMAPP into isopentenyladenosine-5'-monophosphate (iPMP), the active form of<br>
cytokinins (Akiyoshi et al. (1984) Proc. Natl. Acad. Sci USA 87:5994-5998).<br>
Overexpression of the Agrobacterium ipt gene in a variety of transgenic plants has<br>
been shown to cause an increased level of cytokinins and elicit typical cytokinin<br>
responses in the host plant (Hansen et al. (1999) Curr Top Microbiol Immunol<br>
240:21-57). Therefore, it has been postulated that plant cells use machinery<br>
similar to that of A. tumefaciens cells for cytokinin biosynthesis. Arabidopsis IPT<br>
homologs have recently been identified in Arabidopsis and Petunia (Takei et al.<br>
(2001) J. Biol. Chem. 276: 26405-26410 and Kakimoto (2001) Plant Cell Physiol.<br>
42:677-685). Overexpression of the Arabidopsis IPT homologs in plants elevated<br>
cytokinin levels and elicited typical cytokinin responses in planta and under tissue<br>
culture conditions (Kakimoto (2001) Plant Cell Physiol. 42:677-685).<br>
Arabicldpsis ipT "genes are members of a small multigene family of nine<br>
different genes, two of which code for tRNA isopentenyl transferases, and seven<br>
of which encode a gene product with a cytokinin biosynthetic function.<br>
Biochemical analysis of the recombinant AtlPT4 protein showed that, in contrast to<br>
the bacterial enzyme, the Arabidopsis enzyme uses ATP as a substrate instead of<br>
AMP. Another plant IPT gene (Sho) was identified in Petunia hybrida using an<br>
activation tagging strategy (Zubko et a/. (2002) The Plant Journal 29:797-808).<br>
In view of the influence of cytokinins on a wide variety of plant<br>
developmental processes, including root architecture, shoot and leaf development,<br>
and seed set, the ability to manipulate cytokinin levels in higher plant cells, and<br>
thereby drastically effect plant growth and productivity, offers significant<br>
commercial value (Mok et a/. (1994) Cytokinins. Chemistry, Action and Function.<br>
CRC Press, Boca Raton, FLA, pp. 155-166),<br>
BRIEF SUMMARY OF THE INVENTION<br>
Compositions and methods of the invention comprise and employ<br>
isopentenyl transferase (IPT) polypeptides and polynucleotides that are involved in<br>
modulating plant development, morphology, and physiology.<br>
Compositions further include expression cassettes, plants, plant cells, and<br>
seeds having the IPT sequences of the invention. The plants, plant cells, and<br>
seeds of the invention may exhibit phenotypic changes, such as modulated<br>
(increased or decreased) cytokinin levels; modulated floral development;<br>
modulated root development; altered shoot to root ratio; increased seed size or an<br>
increased seed weight; increased plant yield or plant vigor; maintained or<br>
improved stress tolerance (e.g., increased or maintained size of the plant,<br>
minimized tip kernel abortion, increased or maintained seed set); decreased shoot<br>
growth; delayed senescence or an enhanced vegetative growth, all relative to a<br>
plant, plant cell, or seed not modified per the invention.<br>
Compositions of the invention also include IPT promoters, DNA constructs<br>
comprising the IPT promoter operably linked to a nucleotide sequence of interest,<br>
expression vectors, plants, plant cells, and seeds comprising these DNA<br>
constructs.<br>
Methods are provided for reducing or eliminating the activity of an IPT<br>
polypeptide in a plant, comprising introducing into the plant a selected<br>
'ploiyhuclebtide. in "specific methods, providing the polynucleotide decreases the<br>
level of cytokinin in the plant and/or modulates root development of the plant.<br>
Methods are also provided for increasing the level of an IPT polypeptide in<br>
a plant comprising introducing into the plant a selected polyn. In specific methods,<br>
expression of the IPT polynucleotide increases the level of a cytokinin in the plant;<br>
maintains or improves the stress tolerance of the plant; maintains or increases the<br>
size of the plant; minimizes seed abortion; increases or maintains seed set;<br>
increases shoot growth; increases seed size or seed weight; increases plant yield<br>
or plant vigor; modulates floral development; delays senescence; or increases leaf<br>
growth.<br>
Methods are also provided for regulating the expression of a nucleotide<br>
sequence of interest. The method comprises introducing into a plant a DNA<br>
construct comprising a heterologous nucleotide sequence of interest operably<br>
linked to an IPT promoter of the invention.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 provides an alignment of cytokinin biosynthetic enzymes from<br>
maize, petunia, and Arabidopsis. The amino acid sequences present in the<br>
alignment include ZmlPTI (SEQ ID N0:23), ZmlPT2 (SEQ ID NO:2), ZmlPT4<br>
(SEQ ID NO:6), ZmlPTS (SEQ ID N0:9), ZmlPT6 (SEQ ID N0:12), ZmlPTT (SEQ<br>
ID NO:15), ZmlPTS (SEQ ID NO:18), AtlPTI (SEQ ID NO:29), AtlPTS (SEQ ID<br>
N0:34), AtlPT4 (SEQ ID NO:30), AtlPTS (SEQ ID N0:35), AtlPT6 (SEQ ID<br>
N0:36), AtlPT7 (SEQ ID NO:37), AtlPTS (SEQ ID NO:38) and Sho (SEQ ID<br>
NO:31). Asterisks indicate amino acids conserved in many IPT proteins and the<br>
underlined amino acids represent a putative ATP/GTP binding site.<br>
Figure 2 provides a schematic of the structure of the ZmlPTI gene from<br>
Mo17 (SEQ ID NO: 21). Coding regions are indicated by the thick arrows and the<br>
CAATand a putative TATA box are shown.<br>
Figure 3 provides an amino acid sequence alignment of ZmlPTI (SEQ ID<br>
NO: 23, referred to as ZmlPT-Mo17) and a variant of ZmlPTI (SEQ ID NO:27,<br>
referred to as ZmlPT-B73). The sequences have 98% amino acid sequence<br>
identity. The consensus sequence for the ZmlPTI polypeptide is found in SEQ ID<br>
NO: 39.<br>
Figure 4 provides ppm values for- ZmlPTI in Lynx embryo libraries at<br>
various days after pollination (DAP).<br>
'Figure 5A snows tne detection of ZmlPTI in different maize organs using<br>
RT-PCR.<br>
Figure 5B shows the detection of ZmlPTI in developing kernels using RTPCR.<br>
Figure 6 shows a Southern blot with B73 or Mo17 genomic DNA digested<br>
by 3 different restriction enzymes. 40 y,g of genomic DNA was digested and run<br>
on a 0.8% agarose gel and transferred to a nylon membrane. The ZmlPT2-B73<br>
gene coding sequence was used as a probe.<br>
Figure 7 shows a Northern blot and relative expression of the ZmlPT2 gene<br>
in different vegetative organs and in whole kernels at different days after<br>
pollination (DAP). Transcript levels were measured in leaves (L), stalks (S), roots<br>
(R), and in whole kernels at 0, 5, 10, 15, 20 and 25 days after pollination, and<br>
quantified relative to abundance of cyclophilin transcripts.<br>
Figure 8 provides a Northern blots and relative expression of the ZmlPT2<br>
gene in kernels at different days after pollination. Transcript levels were measured<br>
in 0- to 5-DAP whole kernels and in 6- to 34-DAP kernels without pedicels, and<br>
quantified relative to abundance of cyclophilin transcripts. Zeatin riboside levels<br>
(the most abundant CK in corn kernels) were previously measured in the same<br>
samples and are indicated by the solid line (Brugiere et al. (2003) Plant Phsyiol<br>
132:1228-1240).<br>
Figure 9 provides ppm values in Lynx embryo libraries forZmlPT2.<br>
Figure 10 provides an alignment of the arnino acid sequences<br>
corresponding to Arabidopsis IPT proteins (AtlPT), the petunia IPT protein (Sho)<br>
and rice putative IPT proteins (OslPT). The sequences in the alignment are as<br>
follows: OslPT6 (SEQ ID NO: 57); OslPT8 (SEQ ID NO: 41); OslPTIO (SEQ ID<br>
NO: 59); OslPT11 (SEQ ID NO: 43); OslPT9 (SEQ ID NO: 61); OslPTS (SEQ ID<br>
NO: 63); OslPT2 (SEQ ID NO: 46); OslPTI (SEQ ID NO: 49); OslPT5 (SEQ ID<br>
NO: 52); OslPT4 (34394150) (SEQ ID NO: 66); OslPT7 (SEQ ID NO: 54); AtlPTI<br>
(AB062607) (SEQ ID NO: 29); AtlPTS (AB062610) (SEQ ID NO: 34); AtlPT4<br>
(AB062611) (SEQ ID NO: 30); AtlPTS (AB062608) (SEQ ID NO: 35); AtlPT6<br>
(AB062612) (SEQ ID NO: 36); AtlPT7 (AB062613) (SEQ ID NO: 37); AtlPTS<br>
(AB082614) (SEQ ID NO: 38); Sho (Petunia) (SEQ ID NO: 31); and, consensus<br>
(SEQ ID NO: 67).<br>
Figure 11 is a Northern blot that shows the relative expression of the<br>
Zm/PT2 gene at different days after pollination in different parts of the kernels.<br>
Transcript levels were measured in 0- to 25-DAP dissected kernels and quantified<br>
relative to abundance of 18S RNA transcripts.<br>
Figure 12 shows chromatograms related to the DMAPP::AMP isopentenyl<br>
transferase activity of Agrobacterium and maize purified recombinant protein.<br>
Figure 13 shows chromatograms related to further treatment of the reaction<br>
products of Figure 12.<br>
Figure 14 shows chromatograms related to the DMAPP::ATP isopentenyl<br>
transferase activity of the maize purified recombinant protein.<br>
Figure 15 is a Western blot of whole maize kernels at various days after<br>
pollinations.<br>
Figure 16 is a graphic representation of the TUSC results.<br>
Figure 17 is a phylogenetic tree of plant IPT sequences.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present inventions now will be described more fully hereinafter with<br>
reference to the accompanying drawings, in which some, but not all embodiments<br>
of the invention are shown. Indeed, these inventions may be embodied in many<br>
different forms and should not be construed as limited to the embodiments set<br>
forth herein; rather, these embodiments are provided so that this disclosure will<br>
satisfy applicable legal requirements. Like numbers refer to like elements<br>
throughout.<br>
Many modifications and other embodiments of the inventions set forth<br>
herein will come to mind to one skilled in the art to which this invention pertains<br>
having the benefit of the teachings presented in the foregoing descriptions and the<br>
associated drawings. Therefore, it is to be understood that the invention is not to<br>
be limited to the specific embodiments disclosed and that modifications and other<br>
embodiments are intended to be included within the scope of the appended<br>
claims. Although specific terms are employed herein, they are used in a generic<br>
and descriptive sense only and not for purposes of limitation,<br>
COMPOSITIONS<br>
Compositions of the invention include isopentenyl transferase (IPT)<br>
polypeptides and polynucleotides that are involved in modulating plant<br>
development, morphology, and physiology. Compositions of the invention further<br>
include IPT promoters that are capable of regulating transcription. In particular,<br>
tne present invention provides for isolated polynucleotides comprising nucleotide<br>
sequences encoding the amino acid sequences shown in SEQ ID NO: 2, 6, 9, 12,<br>
15, 18, 23, 27, 41, 43, 46, 49, 52, 54, 57, 59, 61, 63, 66, and 77. Further provided<br>
are isolated polypeptides having an amino acid sequence encoded by a<br>
polynucleotide described herein, for example those set forth in SEQ ID NO: 1, 3,<br>
4, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44, 47, 50, 53,<br>
55, 58, 60, 62, 64, 45, 48, 51, 56, 65, 69, 70, 71, 72, 73, 74, or 77, Additional<br>
compositions include the IPT promoter sequences set forth in SEQ ID NO: 25 or<br>
75, and promoter sequences as further isolated and characterized from the 5'<br>
regions provided herein for ZmlPT4 (SEQ ID NO: 5), ZmlPT5 (SEQ ID NO: 8),<br>
ZmlPT6 (SEQ ID NO: 11), ZmlPT7 (SEQ ID NO: 14), ZmlPT8 (SEQ ID NO: 17),<br>
ZmlPT9 (SEQ ID NO: 20), OslPTI (SEQ ID NO: 47), OslPT2 (SEQ ID NO: 44),<br>
OslPTS (SEQ ID NO: 62), OslPT4 (SEQ ID NO: 64), OslPTS (SEQ ID NO: 50),<br>
OslPT6 (SEQ ID NO: 55), OslPT7 (SEQ ID NO: 53), OslPTS (SEQ ID NO: 40),<br>
OslPT9 (SEQ ID NO: 60), OslPTI 0 (SEQ ID NO: 58), and OslPT11 (SEQ ID NO:<br>
42).<br>
The isopentenyl transferase polypeptides of the invention share sequence<br>
identity with members of the isopentenyl transferase family of proteins.<br>
Polypeptides in the IPT family have been identified in various bacteria and in<br>
Arabidopsis and Petunia. See, for example, (Kakimoto (2001) Plant Cell Physio.<br>
42:677-658); Takei et al. (2001) The Journal of Biological Chemistry 276:26405-<br>
26410; and Zubko et al. (2002) The Plant Journal 29:797-808). Members of the<br>
IPT family are characterized by having the consensus sequence<br>
GxTxxGK[ST]xxxxx[VLI]xxxxxxx[VLI][VLI]xxDxxQx{57,60}[VLI][VLI]xGG[ST] (SEQ<br>
ID NO:32) (where x denotes any amino acid residue, [ ] any one of the amino<br>
acids shown in [ ], and x{m,n} m to n amino acid residues in number). See,<br>
Kakimoto et al. (2001) Plant Cell Physiol. 42:677-85 and Kakirnoto et al. (2003) J.<br>
Plant Res. 116:233-9, both of which are herein incorporated by reference. IPT<br>
family members may also have ATP/GTP binding sites. An amino acid alignment<br>
of the maize IPT proteins along with Arabidopsis and petunia cytokinin<br>
biosynthetic enzymes is provided in Figure 1, and an amino acid alignment of the<br>
rice IPT proteins with Arabidopsis and petunia cytokinin biosynthetic enzymes is<br>
provided in Figure 10. Asterisks indicate a consensus sequences found in many<br>
cytokinin biosynthetic enzymes. The underlined amino acids indicate a putative<br>
ATP/GTP binding domains.<br>
'Isopentenyl transterase enzymes are involved in cytokinin biosynthesis,<br>
therefore the IPT polypeptides of the invention have "cytokinin synthesis activity."<br>
By "cytokinin synthesis activity" is intended enzymatic activity that generates<br>
cytokinins, derivatives thereof, or any intermediates in the cytokinin synthesis<br>
pathway. Cytokinin synthesis activity therefore includes, but is not limited to,<br>
DMAPP:AMP isopentenyltransferase activity (the conversion of AMP (adenosine-<br>
5'-monophosphate) and DMAPP into iPMP (isopentenyladenosine-5-<br>
monophosphate)), DMAPP:ADP isopentenyltransferase activity (the conversion of<br>
ADP (adenosine-5'-diphosphate) and DMAPP into iPDP (isopentenyladenosine-5-<br>
diphosphate)); DMAPP:ATP isopentenyltransferase activity (the conversion of<br>
ATP (adenosine-5'-triphosphate) and DMAPP into iPTP (isopentenyladenosine-51-<br>
triphosphate)), and DMAPP:tRNA isopentenyltransferase activity (the modification<br>
of cytoplasmic and/or mitrochondrial tRNAs to give isopentenyl). Cytokinin<br>
synthesis activity can further include a substrate comprising a second side chain<br>
precursor, other than DMAPP. Examples of side chain donors include compounds<br>
of terpenoid origin. For example, the substrate could be hydroxymethylbutenyl<br>
diphosphate (HMBPP) which would allow frans-zeatin riboside monophosphate<br>
(ZMP) synthesis. See, for example, Astot et a/. (2000) Proc Natl Acad Sci<br>
97:14778-14783 and Takei et a/. (2003) J Plant Res. 116(3):265-9.<br>
Cytokinin synthesis activity further includes the synthesis of intermediates<br>
involved in formation of ZMP. Methods to assay for the production of various<br>
cytokinins and their intermediates can be found, for example, in Takei et al. (2001)<br>
The Journal of Biological Chemistry 276:26405-26410, Zubo et a/. (2002) The<br>
Plant Journal 29:797-808; Kakimoto et al. (2001) Plant Cell Physio. 42:677-658,<br>
and Sun et al. (2003) Plant Physiology 737:167-176, each of which is herein<br>
incorporated by reference. "Cytokinin synthesis activity" also includes any<br>
alteration in a plant or plant cell phenotype that is characteristic of an increase in<br>
cytokinin concentration. Such cytokinin specific effects are discussed elsewhere<br>
herein and include, but are not limited to, enhanced shoot formation, reduced<br>
apical dominance, delayed senescence, delayed flowering, increased leaf growth,<br>
increased cytokinin levels in the plant, increased tolerance under stress,<br>
minimization of tip kernel abortion, increased or maintained seed set under stress<br>
conditions, and a decrease in root growth. Assays to measure or detect such<br>
phenotypes are known. See, for example, Miyawaki et al. (2004) The Plant<br>
Journal 37:128-138, Takei et al. (2001) The Journal of Biological Chemistry<br>
, 2'uB'b"ef a! (2002) The Plant Journal 29:797-808; Kakimoto ef<br>
a/. (2001) P/an? Ce// Pftys/o. 42:677-658, and Sun ef a/. (2003) Plant Physiology<br>
131 .-167-176, each of which is herein incorporated by reference. Additional<br>
phenotypes resulting from an increase in cytokinin synthesis activity in a plant are<br>
discussed herein.<br>
Compositions of the invention include IPT sequences that are involved in<br>
cytokinin biosynthesis. In particular, the present invention provides for isolated<br>
polynucleotides comprising nucleotide sequences encoding the amino acid<br>
sequences shown in SEQ ID NO: 2, 6, 9,12,15, 18, 23, 27, 41,43, 46,49, 52, 54,<br>
57, 59, 61, 63, 66, and 77. Further provided are polypeptides having an amino<br>
acid sequence encoded by a polynucleotide described herein, for example those<br>
set forth in SEQ ID NOS: 1, 3, 4, 5, 7, 8, 10, 11, 13,14, 16, 17,19, 20, 21, 22, 24,<br>
26, 28, 40, 42, 44, 47, 50, 53, 55, 58, 60, 62, 64, 45, 48, 51, 56, 65, 69, 70, 71, 72,<br>
73, or 74 and fragments and variants thereof. In addition, further provided are<br>
promoter sequences, for example, the sequence set forth in SEQ ID NO: 25 or 75,<br>
variants and fragments thereof.<br>
The invention encompasses isolated or substantially purified polynucleotide<br>
or protein compositions. An "isolated" or "purified" polynucleotide or protein, or<br>
biologically active portion thereof, is substantially or essentially free from<br>
components that normally accompany or interact with the polynucleotide or protein<br>
as found in its naturally occurring environment. Thus, an isolated or purified<br>
polynucleotide or protein is substantially free of other cellular material, or culture<br>
medium when produced by recombinant techniques, or substantially free of<br>
chemical precursors or other chemicals when chemically synthesized. Optimally,<br>
an "isolated" polynucleotide is free of sequences (optimally protein encoding<br>
sequences) that naturally flank the polynucleotide (i.e., sequences located at the<br>
5' and 3' ends of the polynucleotide) in the genomic DNA of the organism from<br>
which the polynucleotide is derived. For example, in various embodiments, the<br>
isolated polynucleotide can contain less than about 5 kb, 4 kb, 3 kb, 2 kb, 1 kb, 0.5<br>
kb, or 0.1 kb of nucleotide sequence that naturally flank the polynucleotide in<br>
genomic DNA of the cell from which the polynucleotide is derived. A protein that is<br>
substantially free of cellular material includes preparations of protein having less<br>
than about 30%, 20%, 10%, 5%, or 1% (by dry weight) of contaminating protein.<br>
When the protein of the invention or biologically active portion thereof is<br>
recombinantly produced, optimally culture medium represents less than about<br>
3b'%", 20%, 10%, 5%, or 1% (by dry weight) of chemical precursors or non-proteinof-<br>
interest chemicals.<br>
Fragments and variants of the disclosed polynucleotides and proteins<br>
encoded thereby are also encompassed by the present invention. By "fragment"<br>
is intended a portion of the polynucleotide or a portion of the amino acid sequence<br>
and hence protein encoded thereby. Fragments of a polynucleotide may encode<br>
protein fragments that retain the biological activity of the native protein and hence<br>
have cytokinin synthesis activity. Alternatively, fragments of a polynucleotide that<br>
are useful as hybridization probes generally do not encode fragment proteins<br>
retaining biological activity. Thus, fragments of a nucleotide sequence may range<br>
from at least about 20 nucleotides, about 50 nucleotides, about 100 nucleotides,<br>
and up to the full-length polynucleotide encoding the proteins of the invention.<br>
A fragment of an IPT polynucleotide that encodes a biologically active<br>
portion of an IPT protein of the invention will encode at least 15, 25, 30, 50, 100,<br>
150, 200, 225, 250, 275, 300, 310, 315, or 320 contiguous amino acids, or up to<br>
the total number of amino acids present in a full-length IPT protein of the invention<br>
(for example, 322, 364, 337, 338, 352, 388, 353, 352, 450, 590, 328, 325, 251,<br>
427, 417, 585, 455, 344, and 347 amino acids for SEQ ID NO: 2, 6, 9, 12, 15, 18,<br>
23, 27, 41, 43, 46, 49, 52, 54, 57, 59, 61, 63, and 66, respectively). Fragments of<br>
an IPT polynucleotide that are useful as hybridization probes or PCR primers<br>
generally need not encode a biologically active portion of an IPT protein.<br>
Thus, a fragment of an IPT polynucleotide may encode a biologically active<br>
portion of an IPT protein, or it may be a fragment that can be used as a<br>
hybridization probe or PCR primer using methods disclosed below. A biologically<br>
active portion of an IPT protein can be prepared by isolating a portion of one of the<br>
IPT polynucleotides of the invention, expressing the encoded portion of the IPT<br>
protein (e.g., by recombinant expression in vitro), and assessing the activity of the<br>
encoded portion of the IPT protein. Polynucleotides that are fragments of an IPT<br>
nucleotide sequence comprise at least 16, 20, 50, 75, 100, 150, 200, 250, 300,<br>
350, 400, 450, 500, 550, 600, 650, 700, 800, 900, 950, or 965 contiguous<br>
nucleotides, or up to the number of nucleotides present in a full-length IPT<br>
polynucleotide disclosed herein (for example, 1495, 969, 2901, 2654, 1095, 4595,<br>
1014, 1955, 1017, 1652, 1059, 3419, 1167, 1535, 3000, 1209, 1062, 1299, 1056,<br>
4682, 8463, 4470, 4114, 2599, 1284, 5030, 8306, 7608, 5075, 4777, 984, 975,<br>
753, 1254,1044, 1035,1284,1353, 1368,1758, and 1773 nucleotides for SEQ ID<br>
"NO: 'i; 3, 4, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44,<br>
47, 50, 53, 55, 58, 60, 62, 64, 45, 48, 51, 56, 65, 69, 70, 71, 72, 73, and 74,<br>
respectively).<br>
"Variants" is intended to mean substantially similar sequences. For<br>
polynucleotides, a variant comprises a deletion and/or addition of one or more<br>
nucleotides at one or more sites within the native polynucleotide and/or a<br>
substitution of one or more nucleotides at one or more sites in the native<br>
polynucleotide. As used herein, a "native" polynucleotide or polypeptide<br>
comprises a naturally occurring nucleotide sequence or amino acid sequence,<br>
respectively. For polynucleotides, conservative variants include those sequences<br>
that, because of the degeneracy of the genetic code, encode the amino acid<br>
sequence of one of the IPT polypeptides of the invention. Naturally occurring<br>
variants such as these can be identified with the use of well-known molecular<br>
biology techniques, as, for example, with polymerase chain reaction (PCR) and<br>
hybridization techniques as outlined below. Variant polynucleotides also include<br>
synthetically derived polynucleotide, such as those generated, for example, by<br>
using site-directed mutagenesis but which still encode an IPT protein of the<br>
invention. Generally, variants of a particular polynucleotide of the invention will<br>
have at least about 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%,<br>
90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or more sequence<br>
identity to that particular polynucleotide as determined by sequence alignment<br>
programs and parameters described elsewhere herein.<br>
Variants of a particular polynucleotide of the invention (i.e., the reference<br>
polynucleotide) can also be evaluated by comparison of the percent sequence<br>
identity between the polypeptide encoded by a variant polynucleotide and the<br>
polypeptide encoded by the reference polynucleotide. Thus, for example, isolated<br>
polynucleotides that encode a polypeptide with a given percent sequence identity<br>
to the polypeptide of SEQ ID NO: 2, 6, 9,12, 15,18, 23, 27,41,43, 46, 49, 52, 54,<br>
57, 59, 61, 63, 66, or 77 are disclosed, Percent sequence identity between any<br>
two polypeptides can be calculated using sequence alignment programs and<br>
parameters described elsewhere herein. Where any given pair of polynucleotides<br>
of the invention is evaluated by comparison of the percent sequence identity<br>
shared by the two polypeptides they encode, the percent sequence identity<br>
between the two encoded polypeptides is at least about 40%, 45%, 50%, 55%,<br>
"60'%'; 65%; 70%, 75%, 8Wo,"85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%,<br>
98%, 99% or more sequence identity,<br>
"Variant" protein is intended to mean a protein derived from the native<br>
protein by deletion or addition of one or more amino acids at one or more sites in<br>
the native protein and/or substitution of one or more amino acids at one or more<br>
sites in the native protein. Certain variant proteins encompassed by the present<br>
invention are biologically active, that is they continue to possess the desired<br>
biological activity of the native protein, that is, cytokinin synthesis activity, as<br>
described herein. Such variants may result from, for example, genetic<br>
polymorphism or from human manipulation. Biologically active variants of a native<br>
IPT protein of the invention will have at least about 40%, 45%, 50%, 55%, 60%,<br>
65%, 70%, 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%,<br>
99% or more sequence identity to the amino acid sequence for the native protein<br>
as determined by sequence alignment programs and parameters described<br>
elsewhere herein. A biologically active variant of a protein of the invention may<br>
differ from that protein by as few as 1-15 amino acid residues, as few as 1-10,<br>
such as 6-10, as few as 5, as few as 4, 3,2, or even 1 amino acid residue.<br>
The proteins of the invention may be altered in various ways including<br>
amino acid substitutions, deletions, truncations, and insertions. Methods for such<br>
manipulations are generally known in the art. For example, amino acid sequence<br>
variants and fragments of the IPT proteins can be prepared by mutations in the<br>
DNA. Methods for mutagenesis and polynucleotide alterations are well known in<br>
the art. See, for example, Kunkel (1985) Proc. Natl. Acad. Sci. USA 82:488-492;<br>
Kunkel et a/. (1987) Methods in Enzymol. 154:367-382; U.S. Patent No.<br>
4,873,192; Walker and Gaastra, eds. (1983) Techniques in Molecular Biology<br>
(MacMillan Publishing Company, New York) and the references cited therein.<br>
Guidance as to appropriate amino acid substitutions that do not affect biological<br>
activity of the protein of interest may be found in the model of Dayhoff et a/. (1978)<br>
Atlas of Protein Sequence and Structure (Natl. Biomed. Res. Found., Washington,<br>
D.C.), herein incorporated by reference. Conservative substitutions, such as<br>
exchanging one amino acid with another having similar properties, may be<br>
optimal.<br>
Thus, the genes and polynucleotides of the invention include both the<br>
naturally occurring sequences as well as mutant forms. Likewise, the proteins of<br>
the invention encompass naturally occurring proteins as well as variations and<br>
modified forms thereof." "Such variants will continue to possess the desired IPT<br>
activity. Obviously, the mutations that will be made in the DMA encoding the<br>
variant must not place the sequence out of reading frame and optimally will not<br>
create complementary regions that could produce secondary mRNA structure.<br>
The deletions, insertions, and substitutions of the protein sequences<br>
encompassed herein are not expected to produce radical changes in the<br>
characteristics of the protein. However, when it is difficult to predict the exact<br>
effect of the substitution, deletion, or insertion in advance of doing so, one skilled<br>
in the art will appreciate that the effect will be evaluated by routine screening<br>
assays. That is, the activity can be evaluated by assaying for cytokinin synthesis<br>
activity. See, for example, Takei et al. (2001) The Journal of Biological Chemistry<br>
276:26405-26410; Zubo et al. (2002) The Plant Journal 29:797-808; Kakimoto et<br>
al. (2001) Plant Cell Physio. 42:677-658; Sun et al. (2003) Plant Physiology<br>
737:167-176; and Miyawaki et al. (2004) The Plant Journal 37:128-138, all of<br>
which are herein incorporated by reference.<br>
Variant polynucleotides and proteins also encompass sequences and<br>
proteins derived from a mutagenic and recombinogenic procedure such as DNA<br>
shuffling. With such a procedure, one or more different IPT coding sequences can<br>
be manipulated to create a new IPT polypeptide possessing the desired<br>
properties. In this manner, libraries of recombinant polynucleotides are generated<br>
from a population of related sequence polynucleotides comprising sequence<br>
regions that have substantial sequence identity and can be homologously<br>
recombined in vitro or in vivo. For example, using this approach, sequence motifs<br>
encoding a domain of interest may be shuffled between the IPT gene of the<br>
invention and other known IPT genes to obtain a new gene coding for a protein<br>
with an improved property of interest, such as an Increased Km in the case of an<br>
enzyme. Strategies for such DNA shuffling are known in the art. See, for<br>
example, Stemmer (1994) Proc. Natl. Acad. Sci. USA 91:10747-10751; Stemmer<br>
(1994) Nature 370:389-391; Crameri et al. (1997) Nature Biotech. 15:436-438;<br>
Moore et al. (1997) J. Mol. Biol. 272:336-347; Zhang et al. (1997) Proc. Natl.<br>
Acad. Sci. USA 94:4504-4509; Crameri et al. (1998) Nature 391:288-291; and<br>
U.S. Patent Nos. 5,605,793 and 5,837,458.<br>
The compositions of the invention also include isolated polynucleotides<br>
comprising an IPT promoter nucleotide sequence as set forth in SEQ ID NO: 25 or<br>
75, and promoter sequences as further isolated and characterized from the<br>
regions 5' to the coding sequence provided as a part of SEQ ID NO: 5 (ZmlPT4),<br>
SEQ ID NO: 8 (ZmiPTS), SEQ ID NO: 11 (ZmlPT6), SEQ ID NO: 14 (ZmlPTT),<br>
SEQ ID NO: 17 (ZmiPTS), SEQ ID NO: 20 (ZmlPTS), SEQ ID NO: 47 (OslPTI),<br>
SEQ ID NO: 44 (OslPT2), SEQ ID NO: 62 (OslPTS), SEQ ID NO: 64 (OslPT4),<br>
SEQ ID NO: 50 (OslPTS), SEQ ID NO: 55 (OslPT6), SEQ ID NO: 53 (OslPT7),<br>
SEQ ID NO: 40 (OslPTS), SEQ ID NO: 60 (OslPTS), SEQ ID NO: 58 (OslPTIO),<br>
and SEQ ID NO: 42 (OslPT11).<br>
By "promoter" is intended a regulatory region of DNA usually comprising a<br>
TATA box capable of directing RNA polymerase II to initiate RNA synthesis at the<br>
appropriate transcription initiation site for a particular polynucleotide sequence. A<br>
promoter may additionally comprise other recognition sequences generally<br>
positioned upstream or 5' to the TATA box, referred to as upstream promoter<br>
elements, which influence the transcription initiation rate. The promoter<br>
sequences of the present invention regulate (i.e., repress or activate) transcription.<br>
It is recognized that additional domains can be added to the promoter<br>
sequences of the invention and thereby modulate the level of expression, the<br>
developmental timing of expression, or tissue type that expression occurs in. See<br>
particularly, Australian Patent No. AU-A-77751/94 and U.S. Patent Nos. 5,466,785<br>
and 5,635,618.<br>
Fragments and variants of the disclosed IPT promoter polynucleotides are<br>
also encompassed by the present invention. Fragments of a promoter<br>
polynucieotide may retain biological activity and hence retain transcriptional<br>
regulatory activity. Alternatively, fragments of a polynucleotide that are useful as<br>
hybridization probes generally do not retain biological activity. Thus, fragments of<br>
a promoter nucleotide sequence may range from at least about 20 nucleotides,<br>
about 50 nucleotides, about 100 nucleotides, and up to the full-length<br>
polynucleotide of the invention.<br>
Thus, a fragment of an IPT promoter polynucleotide may encode a<br>
biologically active portion of an IPT promoter, or it may be a fragment that can be<br>
used as a hybridization probe or PCR primer using methods disclosed below. A<br>
biologically active portion of the IPT promoter polynucleotides can be prepared by<br>
isolating a portion of one of the IPT promoter polynucleotide of the invention, and<br>
assessing the activity of the portion of the IPT promoter. Polynucleotides that are<br>
fragments of an IPT promoter comprise at least 16,20, 50, 75, 100,150, 200, 250,<br>
300, 350, 400, 450, 500, 550, 600, 650, 700, 800, 900, 1,000, 1,050, or 1,080<br>
contiguous nucleotides, or up to the number of nucleotides present in a full-length<br>
IPT promoter polynucleotide disclosed herein (for example, 1082 and 1920<br>
nucleotides for SEQ ID NOS: 25 and 75, respectfully).<br>
For a promoter polynucleotide, a variant comprises a deletion and/or<br>
addition of one or more nucleotides at one or more internal sites within the native<br>
polynucleotide and/or a substitution of one or more nucleotides at one or more<br>
sites in the native polynucleotide. Generally, variants of a particular promoter<br>
polynucleotide of the invention will have at least about 40%, 45%, 50%, 55%,<br>
60%, 65%, 70%, 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%,<br>
98%, 99% or more sequence identity to that particular polynucleotide as<br>
determined by sequence alignment programs and parameters described<br>
elsewhere herein.<br>
Variant polynucleotides also encompass sequences derived from a<br>
mutagenic and recombinogenic procedure such as DNA shuffling. With such a<br>
procedure, one or more different promoter sequences can be manipulated to<br>
create a new IPT promoter possessing the desired properties. Strategies for such<br>
DNA shuffling are described elsewhere herein.<br>
Methods are available in the art for determining if a promoter sequence<br>
retains the ability to regulate transcription. Such activity can be measured by<br>
Northern blot analysis. See, for example, Sambrook et a/. (1989) Molecular<br>
Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor Laboratory Press,<br>
Plainview, New York), herein incorporated by reference. Alternatively, biological<br>
activity of the promoter can be measured using assays specifically designed for<br>
measuring the activity and/or level of the polypeptide being expressed from the<br>
promoter. Such assays are known in the art. Also, known promoter elements can<br>
be identified within a putative promoter sequence. For example, the IPT1<br>
promoter (SEQ ID NO: 25) of the invention has a TATA-box at bp 688. A TATAbox<br>
like sequence can be found 48 bp upstream of the transcription start site<br>
(between bp 1035 and 1042). A potential CAAT box can be found between bp<br>
929 and 932.<br>
The polynucleotides of the invention (i.e., the IPT sequences and the IPT<br>
promoter sequences) can be used to isolate corresponding sequences from other<br>
organisms, particularly other plants, more particularly other monocots. In this<br>
manner, methods such as PCR, hybridization, and the like can be used to identify<br>
such sequences based on their sequence homology to the sequences set forth<br>
herein; Sequen"ces'"'isolaled"1l5ased on their sequence identity to the entire IPT<br>
sequences or the IPT promoter sequences set forth herein or to variants and<br>
fragments thereof are encompassed by the present invention. Such sequences<br>
include sequences that are orthologs of the disclosed sequences. "Orthologs" is<br>
intended to mean genes derived from a common ancestral gene and which are<br>
found in different species as a result of speciation. Genes found in different<br>
species are considered orthologs when their nucleotide sequences and/or their<br>
encoded protein sequences share at least 60%, 70%, 75%, 80%, 85%, 90%, 91%,<br>
92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or greater sequence identity.<br>
Functions of orthologs are often highly conserved among species. Thus, isolated<br>
polynucleotides that encode for an IPT protein or comprise an IPT promoter<br>
sequence and which hybridize under stringent conditions to the IPT sequences or<br>
the IPT promoter sequences disclosed herein, or to variants or fragments or<br>
complements thereof, are encompassed by the present invention.<br>
In a PCR approach, oligonucleotide primers can be designed for use in<br>
PCR reactions to amplify corresponding DNA sequences from cDNA or genomic<br>
DNA extracted from any plant of interest. Methods for designing PCR primers and<br>
PCR cloning are generally known in the art and are disclosed in Sambrook et a/.<br>
(1989) Molecular Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor<br>
Laboratory Press, Plainview, New York). See also Innis et a/., eds. (1990) PCR<br>
Protocols: A Guide to Methods and Applications (Academic Press, New York);<br>
Innis and Gelfand, eds. (1995) PCR Strategies (Academic Press, New York); and<br>
Innis and Gelfand, eds. (1999) PCR Methods Manual (Academic Press, New<br>
York). Known methods of PCR include, but are not limited to, methods using<br>
paired primers, nested primers, single specific primers, degenerate primers, genespecific<br>
primers, vector-specific primers, partially-mismatched primers, and the<br>
like.<br>
In hybridization techniques, all or part of a known polynucleotide is used as<br>
a probe that selectively hybridizes to other corresponding polynucleotides present<br>
in a population of cloned genomic DNA fragments or cDNA fragments (i.e.,<br>
genomic or cDNA libraries) from a chosen organism. The hybridization probes<br>
may be genomic DNA fragments, cDNA fragments, RNA fragments, or other<br>
oligonucleotides, and may be labeled with a detectable group such as 32P, or any<br>
other detectable marker. Thus, for example, probes for hybridization can be made<br>
by labeling synthetic oligonucleotides based on the IPT polynucleotides or the IPT<br>
sequences or me invention. Methods for preparation of probes for<br>
hybridization and for construction of cDNA and genomic libraries are generally<br>
known in the art and are disclosed in Sambrook et al. (1989) Molecular Cloning: A<br>
Laboratory Manual (2d ed., Cold Spring Harbor Laboratory Press, Plainview, New<br>
York).<br>
For example, the entire IPT polynucleotide or the IPT promoter sequence<br>
disclosed herein, or one or more portions thereof, may be used as a probe<br>
capable of specifically hybridizing to corresponding IPT polynucleotides,<br>
messenger RNAs, or promoter sequences. To achieve specific hybridization<br>
under a variety of conditions, such probes include sequences that are unique<br>
among IPT polynucleotide sequences or IPT promoter sequences and are<br>
optimally at least about 10 nucleotides in length, and most optimally at least about<br>
20 nucleotides in length. Such probes may be used to amplify corresponding IPT<br>
polynucleotides or IPT promoters from a chosen plant by PCR. This technique<br>
may be used to isolate additional coding sequences from a desired plant or as a<br>
diagnostic assay to determine the presence of coding sequences in a plant.<br>
Hybridization techniques include hybridization screening of plated DNA libraries<br>
(either plaques or colonies; see, for example, Sambrook et al. (1989) Molecular<br>
Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor Laboratory Press,<br>
Plainview, New York).<br>
Hybridization of such sequences may be carried out under stringent<br>
conditions. By "stringent conditions" or "stringent hybridization conditions" is<br>
intended conditions under which a probe will hybridize to its target sequence to a<br>
detectably greater degree than to other sequences (e.g., at least 2-fold over<br>
background). Stringent conditions are sequence-dependent and will be different in<br>
different circumstances. By controlling the stringency of the hybridization and/or<br>
washing conditions, target sequences that are 100% complementary to the probe<br>
can be identified (homologous probing). Alternatively, stringency conditions can<br>
be adjusted to allow some mismatching in sequences so that lower degrees of<br>
similarity are detected (heterologous probing). Generally, a probe is less than<br>
about 1000 nucleotides in length, optimally less than 500 nucleotides in length.<br>
Typically, stringent conditions will be those in which the salt concentration is<br>
less than about 1.5 M Na ion, typically about 0.01 to 1.0 M Na ion concentration<br>
(or other salts) at pH 7.0 to 8.3 -and the temperature is at least about 30°C for<br>
short probes (e.g., 10 to 50 nucleotides) and at least about 60°C for long probes<br>
 (e.g., greater man 5U nucieotides). Stringent conditions may also be achieved<br>
with the addition of destabilizing agents such as formamide. Exemplary low<br>
stringency conditions include hybridization with a buffer solution of 30 to 35%<br>
formamide, 1 M NaCI, 1% SDS (sodium dodecyl sulphate) at 37°C, and a wash in<br>
1X to 2X SSC (20X SSC * 3.0 M NaCI/0.3 M trisodium citrate) at 50 to 55°C.<br>
Exemplary moderate stringency conditions include hybridization in 40 to 45%<br>
formamide, 1.0 M NaCI, 1% SDS at 37°C, and a wash in 0.5X to 1X SSC at 55 to<br>
60°C. Exemplary high stringency conditions include hybridization in 50%<br>
formamide, 1 M NaCI, 1% SDS at 37°C, and a wash in 0.1X SSC at 60 to 65°C.<br>
Optionally, wash buffers may comprise about 0.1% to about 1% SDS. Duration of<br>
hybridization is generally less than about 24 hours, usually about 4 to about 12<br>
hours. The duration of the wash time will be at least a length of time sufficient, to<br>
reach equilibrium.<br>
Specificity is typically the function of post-hybridization washes, the critical<br>
factors being the ionic strength and temperature of the final wash solution. For<br>
DNA-DNA hybrids, the Tm can be approximated from the equation of Meinkoth and<br>
Wahl (1984) Anal, Biochem. 138:267-284: Tm = 81.5°C + 16.6 (log M) + 0.41<br>
(%GC) - 0.61 (% form) - 500/L; where M is the molarity of monovalent cations,<br>
%GC is the percentage of guanosine and cytosine nucieotides in the DNA, % form<br>
is the percentage of formamide in the hybridization solution, and L is the length of<br>
the hybrid in base pairs. The Tm is the temperature (under defined ionic strength<br>
and pH) at which 50% of a complementary target sequence hybridizes to a<br>
perfectly matched probe. Tm is reduced by about 1°C for each 1% of<br>
mismatching; thus, Tm, hybridization, and/or wash conditions can be adjusted to<br>
hybridize to sequences of the desired identity. For example, if sequences with<br>
&gt;90% identity are sought, the Tm can be decreased 10°C. Generally, stringent<br>
conditions are selected to be about 5°C lower than the thermal melting point (Tm)<br>
for the specific sequence and its complement at a defined ionic strength and pH.<br>
However, severely stringent conditions can utilize a hybridization and/or wash at 1,<br>
2, 3, or 4°C lower than the thermal melting point (Tm); moderately stringent<br>
conditions can utilize a hybridization and/or wash at 6, 7, 8, 9, or 10°C lower than<br>
the thermal melting point (Tm); low stringency conditions can utilize a hybridization<br>
and/or wash at 11, 12, 13, 14, 15, or 20°C lower than the thermal melting point<br>
(Tm). Using the equation, hybridization and wash compositions, and desired Tm,<br>
those of ordinary skill will understand that variations in the stringency of<br>
hybridization and/or wash solutions are inherently described. If the desired degree<br>
of mismatching results in a Tm of less than 45°C (aqueous solution) or 32°C<br>
(formarnide solution), it is optimal to increase the SSC concentration so that a<br>
higher temperature can be used. An extensive guide to the hybridization of<br>
nucleic acids is found in Tijssen (1993) Laboratory Techniques in Biochemistry<br>
and Molecular Biology—Hybridization with Nucleic Acid Probes, Part I, Chapter 2<br>
(Elsevier, New York); and Ausubel et al., eds. (1995) Current Protocols in<br>
Molecular Biology, Chapter 2 (Greene Publishing and Wiley-lnterscience, New<br>
York). See Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual (2d<br>
ed., Cold Spring Harbor Laboratory Press, Plainview, New York).<br>
The following terms are used to describe the sequence relationships<br>
between two or more polynucleotides or polypeptides: (a) "reference sequence",<br>
(b) "comparison window", (c) "sequence identity", and, (d) "percentage of<br>
sequence identity."<br>
(a) As used herein, "reference sequence" is a defined sequence used<br>
as a basis for sequence comparison. A reference sequence may be a subset or<br>
the entirety of a specified sequence; for example, as a segment of a full-length<br>
cDNA or gene sequence, or the complete cDNA or gene sequence.<br>
(b) As used herein, "comparison window" makes reference to a<br>
contiguous and specified segment of a polynucleotide sequence, wherein the<br>
polynucleotide sequence in the comparison window may comprise additions or<br>
deletions (i.e., gaps) compared to the reference sequence (which does not<br>
comprise additions or deletions) for optimal alignment of the two polynucleotides.<br>
Generally, the comparison window is at least 20 contiguous nucleotides in length,<br>
and optionally can be 30, 40, 50, 100, or longer. Those of skill in the art<br>
understand that to avoid a high similarity to a reference sequence due to inclusion<br>
of gaps in the polynucleotide sequence a gap penalty is typically introduced and is<br>
subtracted from the number of matches.<br>
Methods of alignment of sequences for comparison are well known in the<br>
art. Thus, the determination of percent sequence identity between any two<br>
sequences can be accomplished using a mathematical algorithm. Non-limiting<br>
examples of such mathematical algorithms are the algorithm of Myers and Miller<br>
(1988) CABIOS 4:11-17; the local alignment algorithm of Smith et al. (1981) Adv.<br>
Appl. Math. 2:482; the global alignment algorithm of Needleman and Wunsch<br>
(1970) J. Mo/. B/o/. 48:443-453; the search-for-local alignment method of Pearson<br>
and Lipman (1988) Proc. Natl. Acad. Sci. 85:2444-2448; the algorithm of Karlin<br>
and Altschul (1990) Proc. Natl. Acad. Scl. USA 872264, modified as in Karlin and<br>
Altschul (1993) Proc. Natl. Acad. Sci. USA 90:5873-5877.<br>
Computer implementations of these mathematical algorithms can be<br>
utilized for comparison of sequences to determine sequence identity. Such<br>
implementations include, but are not limited to: CLUSTAL in the PC/Gene<br>
program (available from Intelligenetics, Mountain View, California); the ALIGN<br>
program (Version 2.0) and GAP, BESTFIT, BLAST, FASTA, and TFASTA in the<br>
GCG Wisconsin Genetics Software Package, Version 10 (available from Accelrys<br>
Inc., 9685 Scranton Road, San Diego, California, USA). Alignments using these<br>
programs can be performed using the default parameters. The CLUSTAL<br>
program is well described by Higgins et al. (1988) Gene 73:237-244 (1988);<br>
Higgins et al. (1989) CABIOS 5:151-153; Corpet era/. (1988) Nucleic Acids Res.<br>
16:10881-90; Huang et al. (1992) CABIOS 8:155-65; and Pearson et al. (1994)<br>
Meth. Mol. Biol. 24:307-331. The ALIGN program is based on the algorithm of<br>
Myers and Miller (1988) supra. A PAM120 weight residue table, a gap length<br>
penalty of 12, and a gap penalty of 4 can be used with the ALIGN program when<br>
comparing amino acid sequences. The BLAST programs of Altschul et al (1990)<br>
J. Mol. Biol. 215:403 are based on the algorithm of Karlin and Altschul (1990)<br>
supra. BLAST nucleotide searches can be performed with the BLASTN program,<br>
score = 100, wordlength = 12, to obtain nucleotide sequences homologous to a<br>
nucleotide sequence encoding a protein of the invention. BLAST protein searches<br>
can be performed with the BLASTX program, score = 50, wordlength = 3, to obtain<br>
amino acid sequences homologous to a protein or polypeptide of the invention.<br>
To obtain gapped alignments for comparison purposes, Gapped BLAST (in<br>
BLAST 2.0) can be utilized as described in Altschul et al. (1997) Nucleic Acids<br>
Res. 25:3389. Alternatively, PSI-BLAST (in BLAST 2.0) can be used to perform<br>
an iterated search that detects distant relationships between molecules. See<br>
Altschul et al. (1997) supra. When utilizing BLAST, Gapped BLAST, PSI-BLAST,<br>
the default parameters of the respective programs (e.g., BLASTN for nucleotide<br>
sequences, BLASTX for proteins) can be used. See www.ncbi.nlm.nih.gov.<br>
Alignment may also be performed manually by inspection. ,<br>
Unless otherwise stated, sequence identity/similarity values provided herein<br>
refer to the value obtained using GAP Version 10 using the following parameters:<br>
% identity and % similarity for a nucleotide sequence using GAP Weight of 50 and<br>
Length Weight of 3, and the nwsgapdna.cmp scoring matrix; % identity and %<br>
similarity for an amino acid sequence using GAP Weight of 8 and Length Weight<br>
of 2, and the BLOSUM62 scoring matrix; or any equivalent program thereof. By<br>
"equivalent program" is intended any sequence comparison program that, for any<br>
two sequences in question, generates an alignment having identical nucleotide or<br>
amino acid residue matches and an identical percent sequence identity when<br>
compared to the corresponding alignment generated by GAP Version 10.<br>
GAP uses the algorithm of Needleman and Wunsch (1970) J. Mol. Bid.<br>
48:443-453, to find the alignment of two complete sequences that maximizes the<br>
number of matches and minimizes the number of gaps. GAP considers all<br>
possible alignments and gap positions and creates the alignment with the largest<br>
number of matched bases and the fewest gaps. It allows for the provision of a gap<br>
creation penalty and a gap extension penalty in units of matched bases. GAP<br>
must make a profit of gap creation penalty number of matches for each gap it<br>
inserts. If a gap extension penalty greater than zero is chosen, GAP must, in<br>
addition, make a profit for each gap inserted of the length of the gap times the gap<br>
extension penalty. Default gap creation penalty values and gap extension penalty<br>
values in Version 10 of the GCG Wisconsin Genetics Software Package for<br>
protein sequences are 8 and 2, respectively. For nucleotide sequences the<br>
default gap creation penalty is 50 while the default gap extension penalty is 3.<br>
The gap creation and gap extension penalties can be expressed as an integer<br>
selected from the group of integers consisting of from 0 to 200. Thus, for<br>
example, the gap creation and gap extension penalties can be 0,1, 2, 3,4, 5, 6, 7,<br>
8, 9,10, 15, 20, 25, 30, 35,40, 45, 50, 55, 60, 65 or greater.<br>
GAP presents one member of the family of best alignments. There may be<br>
many members of this family, but no other member has a better quality. GAP<br>
displays four figures of merit for alignments: Quality, Ratio, Identity, and Similarity.<br>
The Quality is the metric maximized in order to align the sequences. Ratio is the<br>
quality divided by the number of bases in the shorter segment. Percent Identity is<br>
the percent of the symbols that actually match. Percent Similarity is the percent of<br>
the symbols that are similar. Symbols that are across from gaps are ignored. A<br>
similarity is scored when the scoring matrix value for a pair of symbols is greater<br>
than or equal to 0.50, the similarity threshold. The scoring matrix used in Version<br>
10 of the GCG Wisconsin Genetics Software Package is BLOSUM62 (see<br>
Henikoff and Henikoff (1989) Proc. Natl. Acad. Sci. USA 89:10915).<br>
 (c) As used Herein,sequence identity" or "identity" in the context of two<br>
polynucleotides or polypeptide sequences makes reference to the residues in the<br>
two sequences that are the same when aligned for maximum correspondence<br>
over a specified comparison window. When percentage of sequence identity is<br>
used in reference to proteins it is recognized that residue positions which are not<br>
identical often differ by conservative amino acid substitutions, where amino acid<br>
residues are substituted for other amino acid residues with similar chemical<br>
properties (e.g., charge or hydrophobicity) and therefore do not change the<br>
functional properties of the molecule. When sequences differ in conservative<br>
substitutions, the percent sequence identity may be adjusted upwards to correct<br>
for the conservative nature of the substitution. Sequences that differ by such<br>
conservative substitutions are said to have "sequence similarity" or "similarity".<br>
Means for making this adjustment are well known to those of skill in the art.<br>
Typically this involves scoring a conservative substitution as a partial rather than a<br>
full mismatch, thereby increasing the percentage sequence identity. Thus, for<br>
example, where an identical amino acid is given a score of 1 and a nonconservative<br>
substitution is given a score of zero, a conservative substitution is<br>
given a score between zero and 1. The scoring of conservative substitutions is<br>
calculated, e.g., as implemented in the program PC/GENE (Intelligenetics,<br>
Mountain View, California).<br>
(d) As used herein, "percentage of sequence identity" means the value<br>
determined by comparing two optimally aligned sequences over a comparison<br>
window, wherein the portion of the polynucleotide sequence in the comparison<br>
window may comprise additions or deletions (i.e., gaps) as compared to the<br>
reference sequence (which does not comprise additions or deletions) for optimal<br>
alignment of the two sequences. The percentage is calculated by determining the<br>
number of positions at which the identical nucleic acid base or amino acid residue<br>
occurs in both sequences to yield the number of matched positions, dividing the<br>
number of matched positions by the total number of positions in the window of<br>
comparison, and multiplying the result by 100 to yield the percentage of sequence<br>
identity.<br>
The invention further provides plants having altered levels and/or activities<br>
of the IPT polypeptides of the invention. In some embodiments, the plants of the<br>
invention have stably incorporated into their genome the IPT sequences of the<br>
invention. In other embodiments, plants that are genetically modified at a genomic<br>
locus encoding an IPT polypeptide of the invention are provided. By "native<br>
genomic locus" is intended a naturally occurring genomic sequence. For some<br>
embodiments, the genomic locus is set forth in SEQ ID NO: 21, 40, 42, 44, 47, 50,<br>
53, 55, 58, 60, 62, or 64. The genomic locus may be modified to reduce or<br>
eliminate the activity of the IPT polypeptide. The term "genetically modified" as<br>
used herein refers to a plant or plant part that is modified in its genetic information<br>
by the introduction of one or more foreign polynucleotides, and the insertion of the<br>
foreign polynucleotide leads to a phenotypic change in the plant. By "phenotypic<br>
change" is intended a measurable change in one or more cell functions. For<br>
example, plants having a genetic modification at the genomic locus encoding the<br>
IPT polypeptide can show reduced or eliminated expression or activity of the IPT<br>
polypeptide. Various methods to generate such a genetically modified genomic<br>
locus are described elsewhere herein, as are the variety of phenotypes that can<br>
result from the modulation of the level/activity of the IPT sequences of the<br>
invention.<br>
The invention further provides plants having at least one DNA construct<br>
comprising a heterologous nucleotide sequence of interest operably linked to the<br>
IPT promoter of the invention. In further embodiments, the DNA construct is<br>
stably integrated into the genome of the plant.<br>
As used herein, the term plant includes reference to whole plants, plant<br>
parts or organs (e.g. leaves, stems, roots), plant cells, and seeds and progeny of<br>
same. Plant cell, as used herein, includes, without limitation, cells obtained from<br>
or found in seeds, suspension cultures, embryos, meristematic regions, callus<br>
tissue, leaves, roots, shoots, gametophytes, sporophytes, pollen and mlcrospores,<br>
as well as plant protoplasts and plant cell tissue cultures, plant calli, plant clumps,<br>
and plant cells that are intact in plants or parts of plants such as embryos, pollen,<br>
ovules, seeds, leaves, flowers, branches, fruit, kernels, ears, cobs, husks, stalks,<br>
roots, root tips, anthers, grain and the like. As used herein, "grain" refers to the<br>
mature seed produced by commercial growers for purposes other than growing or<br>
reproducing the species. Progeny, variants, and mutants of the regenerated<br>
plants are also included within the scope of the invention, provided that these<br>
parts comprise the introduced nucleic acid sequences.<br>
METHODS<br>
/. Providing Sequences<br>
The sequences of the present invention can be introduced/expressed in a<br>
host cell such as bacteria, yeast, insect, mammalian, or optimally plant cells. It is<br>
expected that those of skill in the art are knowledgeable in the numerous systems<br>
available for the introduction of a polypeptide or a nucleotide sequence of the<br>
present invention into a host cell. No attempt to describe in detail the various<br>
methods known for providing proteins in prokaryotes or eukaryotes will be made.<br>
By "host cell" is meant a cell which comprises a heterologous nucleic acid<br>
sequence of the invention. Host cells may be prokaryotic cells such as £ coli, or<br>
eukaryotic cells such as yeast, insect, amphibian, or mammalian cells. Host cells<br>
can also be monocotyledonous or dicotyledonous plant cells. In certain<br>
embodiments, the monocotyledonous host cell is a maize host cell.<br>
The use of the term "polynucleotide" is not intended to limit the present<br>
invention to polynucleotides comprising DNA. Those of ordinary skill in the art will<br>
recognize that polynucleotides can comprise ribonucleotides and combinations of<br>
ribonucleotides and deoxyribonucleotides. Such deoxyribonucieotides and<br>
ribonucleotides include both naturally occurring molecules and synthetic<br>
analogues. The polynucleotides of the invention also encompass all forms of<br>
sequences including, but not limited to, single-stranded forms, double-stranded<br>
forms, hairpins, stem-and-loop structures, and the like.<br>
The IPT polynucleotides or the IPT promoters of the invention can be<br>
provided in expression cassettes for expression in the plant of interest. The<br>
cassette will include 5' and 3' regulatory sequences operably linked to an IPT<br>
polynucleotide of the invention. "Operably linked" is intended to mean a functional<br>
linkage between two or more elements. For example, an operable linkage<br>
between a polynucleotide of interest and a regulatory sequence (i.e., a promoter)<br>
is a functional link that allows for expression of the polynucleotide of interest.<br>
Operably linked elements may be contiguous or non-contiguous. When used to<br>
refer to the joining of two protein coding regions, by operably linked is intended<br>
that the coding regions are in the same reading frame. The cassette may<br>
additionally contain at least one additional gene to be cotransformed into the<br>
organism. Alternatively, the additional gene(s) can be provided on multiple<br>
expression cassettes. An expression cassette may be provided with a plurality of<br>
restriction sites and/or recombination sites for insertion of the IPT polynucleotide<br>
"to be under the transcriptional regulation of the regulatory regions. The<br>
expression cassette may additionally contain selectable marker genes.<br>
In certain embodiments, the expression cassette will include in the 5'-3'<br>
direction of transcription, a transcriptional and translational initiation region (i.e., a<br>
promoter), an IPT polynucleotide of the invention, and a transcriptional and<br>
translational termination region (i.e., termination region) functional in plants. The<br>
regulatory regions (i.e., promoters, transcriptional regulatory regions, and<br>
translational termination regions) and/or the IPT polynucleotide of the invention<br>
may be native/analogous to the host cell or to each other. Alternatively, the<br>
regulatory regions and/or the IPT polynucleotide of the invention may be<br>
heterologous to the host cell or to each other. As used herein, "heterologous" in<br>
reference to a sequence is a sequence that originates from a foreign species, or, if<br>
from the same species, is substantially modified from its native form in<br>
composition and/or genomic locus by deliberate human intervention. For<br>
example, a promoter operably linked to a heterologous polynucleotide is from a<br>
species different from the species from which the polynucleotide was derived, or, if<br>
from the same/analogous species, one or both are substantially modified from<br>
their original form and/or genomic locus, or the promoter is not the native promoter<br>
for the operably-linked polynucleotide. As used herein, a chimeric gene comprises<br>
a coding sequence operably linked to a transcription initiation region that is<br>
heterologous to the coding sequence.<br>
While heterologous promoters can be used to express the IPT sequences,<br>
the native promoter sequences or other IPT promoters (e.g., SEQ ID NO: 25 or<br>
75) may also be used. Such constructs can change expression levels of IPT<br>
sequences in the plant or plant cell. Thus, the phenotype of the plant or plant cell<br>
can be altered.<br>
The termination region may be native with the transcriptional initiation<br>
region, may be native with the operably-linked IPT polynucleotide of interest, may<br>
be native with the plant host, or may be derived from another source (i.e., foreign<br>
or heterologous with reference to the promoter), the IPT polynucleotide of interest,<br>
the plant host, or any combination thereof. Convenient termination regions are<br>
available from the Ti-plasmid of A. tumefaciens, such as the octopine synthase<br>
and nopaline synthase termination regions. See also Guerineau etal. (1991) Mol.<br>
Gen. Genet. 262:141-144; Proudfoot (1991) Cell 64:671-674; Sanfacon et al.<br>
(1991) Genes Dev. 5:141-149; Mogen et al. (1990) Plant Cell 2:1261-1272;<br>
"Muhroe et al. (1990) Gene 91:151-158; Ballas et al. (1989) Nucleic Acids Res.<br>
17:7891-7903; and Joshi et al. (1987) Nucleic Acids Res. 15:9627-9639.<br>
Where appropriate, the polynucleotides may be optimized for increased<br>
expression in the transformed plant. That is, the polynucleotides can be<br>
synthesized using plant-preferred codons for improved expression. See, for<br>
example, Campbell and Gowri (1990) Plant Physiol. 92:1-11 for a discussion of<br>
host-preferred codon usage. Methods are available in the art for synthesizing<br>
plant-preferred genes. See, for example, U.S. Patent Nos. 5,380,831, and<br>
5,436,391, and Murray et al. (1989) Nucleic Acids Res. 17:477-498, herein<br>
incorporated by reference.<br>
Additional sequence modifications are known to enhance gene expression<br>
in a cellular host. These include elimination of sequences encoding spurious<br>
polyadenylation signals, exon-intron splice site signals, transposon-like repeats,<br>
and other such well-characterized sequences that may be deleterious to gene<br>
expression. The G-C content of the sequence may be adjusted to levels average<br>
for a given cellular host, as calculated by reference to known genes expressed in<br>
the host cell. When possible, the sequence is modified to avoid predicted hairpin<br>
secondary mRNA structures.<br>
The expression cassettes may additionally contain 5' leader sequences.<br>
Such leader sequences can act to enhance translation. Translation leaders are<br>
known in the art and include: picornavirus leaders, for example, EMCV leader<br>
(Encephalomyocarditis 5' noncoding region) (Elroy-Stein et al. (1989) Proc. Natl.<br>
Acad. Sci. USA 86:6126-6130); potyvirus leaders, for example, TEV leader<br>
(Tobacco Etch Virus) (Gallie et al. (1995) Gene 165(2):233-238), MDMV leader<br>
(Maize Dwarf Mosaic Virus) (Virology 154:9-20), and human immunoglobulin<br>
heavy-chain binding protein (BiP) (Macejak et al. (1991) Nature 353:90-94);<br>
untranslated leader from the coat protein mRNA of alfalfa mosaic virus (AMV RNA<br>
4) (Jobling et al. (1987) Nature 325:622-625); tobacco mosaic virus leader (TMV)<br>
(Gallie et al. (1989) in Molecular Biology of RNA, ed. Cech (Liss, New Yprk), pp.<br>
237-256); and maize chlorotic mottle virus leader (MCMV) (Lommel et al. (1991)<br>
Virology 81:382-385). See also, Della-Cioppa ef a/. (1987) Plant Physiol.<br>
84:965-968. Other methods known to enhance translation can also be utilized.<br>
In preparing the expression cassette, the various DNA fragments may be<br>
manipulated, so as to provide for the DNA sequences in the proper orientation<br>
and, as appropriate, in the proper reading frame. Toward this end, adapters or<br>
linkers" may "be" e'rn'plbye'd'Tololh the DNA fragments or other manipulations may be<br>
involved to provide for convenient restriction sites, removal of superfluous DNA,<br>
removal of restriction sites, or the like. For this purpose, in vitro mutagenesis,<br>
primer repair, restriction, annealing, resubstitutions, e.g., transitions and<br>
transversions, may be involved.<br>
The expression cassette can also comprise a selectable marker gene for the<br>
selection of transformed cells. Selectable marker genes are utilized for the selection<br>
of transformed cells or tissues. Marker genes include genes encoding antibiotic<br>
resistance, such as those encoding neomycin phosphotransferase II (NEO) and<br>
hygromycin phosphotransferase (HPT), as well as genes conferring resistance to<br>
herbicidal compounds, such as glufosinate ammonium, bromoxynil, imidazolinones,<br>
and 2,4-dichlorophenoxyacetate (2,4-D). Additional selectable markers include<br>
phenotypic markers such as B-galactosidase and fluorescent proteins such as<br>
green fluorescent protein (GFP) (Su et a/. (2004) Biotechnol Bioeng 85:610-9-and<br>
Fetter ef al. (2004) Plant Cell 76:215-28), cyan fluorescent protein (CYP) (Bolte et<br>
a/. (2004) J. Cell Science 117:943-54 and Kato ef al. (2002) Plant Physiol<br>
729:913-42), and yellow fluorescent protein (PhiYFP™ from Evrogen, see, Bolte et<br>
al. (2004) J, Cell Science 777:943-54). For additional selectable markers, see<br>
generally, Yarranton (1992) Curr. Opin. Biotech. 3:506-511; Christopherson.ef al.<br>
(1992) Proc. Natl. Acad. Sci. USA 89:6314-6318; Yao et al. (1992) Cell 71:63-72;<br>
Reznikoff (1992) Mot. Micrvbiol. 6:2419-2422; Barkley et al. (1980) in The Operon,<br>
pp. 177-220; Hu et al. (1987) Cell 48:555-566; Brown era/. (1987) Cell 49:603-612;<br>
Figge et al. (1988) Cell 52:713-722; Deuschle et al. (1989) Proc. Natl. Acad. Act.<br>
USA 86:5400-5404; Fuerst ef al. (1989) Proc. Natl. Acad. Sci. USA 86:2549-2553;<br>
Deuschle ef al. (1990) Science 248:480-483; Gossen (1993) Ph.D. Thesis,<br>
University of Heidelberg; Reines ef al. (1993) Proc. Natl. Acad. Sci. USA 90:1917-<br>
1921; Labow ef al. (1990) Mol. Cell. Biol. 10:3343-3356; Zambretti ef al. (1992) Proc.<br>
Natl. Acad. Sci. USA 89:3952-3956; Bairn ef al. (1991) Proc. Natl. Acad. Sci. USA<br>
88:5072-5076; Wyborski ef al. (1991) Nucleic Adds Res. 19:4647-4653; Hillenand-<br>
Wissrnan (1989) Topics Mol. Struc. Biol. 10:143-162; Degenkolb ef al. (1991)<br>
Antimicrob. Agents Chemother. 35:1591-1595; Kleinschnidt ef al. (1988)<br>
Biochemistry 27:1094-1104; Bonin (1993) Ph.D. Thesis, University of Heidelberg;<br>
Gossen ef al. (1992) Proc. Natl. Acad. Sci. USA 89:5547-5551; Oliva ef al. (1992)<br>
Antimicrob. Agents Chemother. 36:913-919; Hlavka et al. (1985) Handbook of<br>
Experimental Pharmacology, Vol. 78 (Springer-Verlag, Berlin); Gill ef al. (1988)<br>
"Nature 334:721-724! Such disclosures are herein incorporated by reference. The<br>
above list of selectable marker genes is not meant to be limiting. Any selectable<br>
marker gene can be used in the present invention.<br>
A number of promoters can be used in the practice of the invention,<br>
including the native promoter of the polynucleotide sequence of interest. The<br>
promoters can be selected based on the desired outcome. The nucleic acids can<br>
be combined with constitutive, inducible, tissue-preferred, or other promoters for<br>
expression in plants,<br>
Such constitutive promoters include, for example, the core promoter of the<br>
Rsyn7 promoter and other constitutive promoters disclosed in WO 99/43838 and<br>
U.S. Patent No. 6,072,050; the core CaMV 35S promoter (Odell ef a/. (1985)<br>
Nature 313:810-812); rice actin (McElroy et al. (1990) Plant Cell 2:163-171);<br>
ubiquitin (Christensen etal. (1989) Plant Mol. Biol. 12:619-632 and Christensen et<br>
al. (1992) Plant Mol. Biol. 18:675-689); pEMU (Last ef al. (1991) Theor. Appl.<br>
Genet. 81:581-588); MAS (Velten et al. (1984) EMBO J. 3:2723-2730); ALS<br>
promoter (U.S. Patent No. 5,659,026), and the like. Other constitutive promoters<br>
include, for example, U.S. Patent, Nos. 5,608,149; 5,608,144; 5,604,121;<br>
5,569,597; 5,466,785; 5,399,680; 5,268,463; 5,608,142; and 6,177,611.<br>
Tissue-preferred promoters can be utilized to target enhanced IPT<br>
expression within a particular plant tissue. Tissue-preferred promoters include<br>
Yamamoto etal. (1997) Plant J. 12(2):255-265; Kawamata etal. (1997) Plant Cell<br>
Physiol. 38(7):792-803; Hansen et al. (1997) Mol. Gen Genet. 254(3):337-343;<br>
Russell et al. (1997) Transgenic Res. 6(2): 157-168; Rinehart ef al. (1996) Plant<br>
Physiol. 112(3): 1331-1341; Van Camp ef al. (1996) Plant Physiol. 112(2):525-535;<br>
Canevascini et al. (1996) Plant Physiol. 112(2):513-524; Yamamoto ef al. (1994)<br>
Plant Cell Physiol. 35(5):773-778; Lam (1994) Results Probl. Cell Differ. 20:181-<br>
196; Orozco ef al. (1993) Plant Mol Biol. 23(6): 1129-1138; Matsuoka ef al. (1993)<br>
Proc Natl. Acad. Sci. USA 90(20):9586-9590; and Guevara-Garcia ef al. (1993)<br>
Plant J. 4(3):495-505. Such promoters can be modified, if necessary, for weak<br>
expression. See, also, U.S. Patent Application No. 2003/0074698, herein<br>
incorporated by reference.<br>
Leaf-preferred promoters are known in the art. See, for example,<br>
Yamamoto ef al. (1997) Plant J. 12(2):255-265; Kwon ef al. (1994) Plant Physiol.<br>
105:357-67; Yamamoto ef al. (1994) Plant Cell Physiol-. 35(5):773-778; Gotor ef al.<br>
(1993) Plant J. 3:509-18; Orozco ef al. (1993) Plant Mol. Biol. 23(6): 1129-1138;<br>
Ba'szczynskl'lira/. "Cl"988)*W&amp;c/. Acid Res. 16:4732; Mitra Qt al. (1994) P/anf<br>
Molecular Biology 26:35-93; Kayaya et al. (1995) Molecular and General Genetics<br>
248:668-674; and Matsuoka et al. (1993) Proc. Natl. Acad. Sci. USA 90(20):9586-<br>
9590. Senecence regulated promoters are also of use, such as, SAM22 (Crowell<br>
et al. (1992) Plant Mol. Biol. 78:459-466). See, also, U.S. Patent No. 5,589,052<br>
herein incorporated by reference.<br>
Root-preferred promoters are known and can be selected from the many<br>
available from the literature or isolated de novo from various compatible species.<br>
See, for example, Hire et al. (1992) Plant Mol. Biol. 20(2):207-218 (soybean rootspecific<br>
glutamine synthetase gene); Keller and Baumgartner (1991) Plant Cell<br>
3(10):1051-1061 (root-specific control element in the GRP 1.8 gene of French<br>
bean); Sanger et al. (1990) Plant Mol. Biol. 14(3):433-443 (root-specific promoter<br>
of the mannopine synthase (MAS) gene of Agrobacterium tumefaciens); and Miao<br>
et al. (1991) Plant Cell 3(1):11-22 (full-length cDNA clone encoding cytosolic<br>
glutamine synthetase (GS), which is expressed in roots and root nodules of<br>
soybean). See also Bogusz et al. (1990) Plant Cell 2(7):633-641, where two rootspecific<br>
promoters isolated from hemoglobin genes from the nitrogen-fixing<br>
nonlegume Parasponia andersonii and the related non-nitrogen-fixing nonlegume<br>
Trema tomentosa are described. The promoters of these genes were linked to a<br>
(3-glucuronidase reporter gene and introduced into both the nonlegume Nicotiana<br>
tabacum and the legume Lotus corniculatus, and in both instances root-specific<br>
promoter activity was preserved. Leach and Aoyagi (1991) describe their analysis<br>
of the promoters of the highly expressed rolC and rolD root-inducing genes of<br>
Agrobacterium rhizogenes (see Plant Science (Limerick) 79(1):69-76). They<br>
concluded that enhancer and tissue-preferred DNA determinants are dissociated<br>
in those promoters. Teeri et al. (1989) used gene fusion to lacZ to show that the<br>
Agrobacterium T-DNA gene encoding octopine synthase is especially active in the<br>
epidermis of the root tip and that the TR21 gene is root specific in the intact plant<br>
and stimulated by wounding in leaf tissue, an especially desirable combination of<br>
characteristics for use with an insecticidal or larvicidal gene (see EMBO J.<br>
8(2):343-350). The TR11 gene, fused to nptll (neomycin phosphotransferase II)<br>
showed similar characteristics. Additional root-preferred promoters include the<br>
VfENOD-GRP3 gene promoter (Kuster et al. (1995) Plant Mol. Biol. 29(4):759-<br>
772); rolB promoter (Gapana et al. (1994) Plant Mol. Biol. 25(4):681-691; and the<br>
CRWAQ81 root-preferred promoter with the ADH first intron (U.S. Patent<br>
PMllcalion lu"05/d'097633):' See also U.S. Patent Nos. 5,837,876; 5,750,386;<br>
5,633,363; 5,459,252; 5,401,836; 5,110,732; and 5,023,179.<br>
"Seed-preferred" promoters refers to those promoters active during seed<br>
development and may include expression in seed initials or related maternal<br>
tissue. Such seed-preferred promoters include, but are not limited to, Cim1<br>
(cytokinin-induced message); cZ19B1 (maize 19 kDa zein); milps (myo-inositol-1-<br>
phosphate synthase) (see WO 00/11177 and U.S. Patent No. 6,225,529; herein<br>
incorporated by reference). Gamma-zein is an endosperm-specific promoter.<br>
Globulin-1 (Glob-1) is a representative embryo-specific promoter. For dicots,<br>
seed-specific promoters include, but are not limited to, bean B-phaseolin, napin, (3-<br>
conglycinin, soybean lectin, cruciferin, and the like. For monocots, seed-specific<br>
promoters include, but are not limited to, maize 15 kDa zein, 22 kDa zein, 27 kDa<br>
zein, gamma-zein, waxy, shrunken 1, shrunken 2, globulin 1, etc. See also WO<br>
00/12733, where seed-preferred promoters from endl and end2 genes are<br>
disclosed; herein incorporated by reference. Additional embryo specific promoters<br>
are disclosed in Sato etal. (1996) Proc. Natl. Acad. Sci. 93:8117-8122; Nakase et<br>
al. (1997) Plant J 12:235-46; and Postma-Haarsma etal. (1999) Plant Mol. Biol.<br>
39:257-71. Additional endosperm specific promoters are disclosed in Albani et al.<br>
(1984) EMBO 3:1405-15; Albani et al. (1999) Theor. Appl. Gen. 98:1253-62;<br>
Albani et al. (1993) Plant J. 4:343-55; Mena et al. (1998) The Plant Journal<br>
116:53-62, and Wu et al. (1998) Plant Cell Physiology 39:885-889.<br>
Also of interest are promoters active in meristem regions, such as<br>
developing inflorescence tissues, and promoters which drive expression at or<br>
about the time of anthesis or early kernel development. This may include, for<br>
example, the maize Zag promoters, including Zag1 and Zag2 (see Schmidt et al.<br>
(1993) The Plant Cell 5:729-37; GenBank X80206; Theissen et al. (1995) Gene<br>
156:155-166; and U.S. patent application 10/817,483); maize Zap promoter (also<br>
known as ZmMADS; U.S. patent application 10/387,937; WO 03/078590); maize<br>
ckx1-2 promoter (U.S. patent publication 2002-0152500 A1; WO 02/0078438);<br>
maize eepl promoter (U.S. patent application 10/817,483); maize end2 promoter<br>
(U.S. Patent 6,528,704 and U.S. patent application 10/310,191); maize led<br>
promoter (U.S. patent application 09/718,754); maize F3.7 promoter (Baszczynski<br>
et al., Maydica 42:189-201 (1997)); maize tb1 promoter (Hubbarda et al., Genetics<br>
162: 1927-1935 (2002) and Wang et al. (1999) Nature 398:236-239); maize eep2<br>
promoter (U.S. patent application 10/817,483); maize thioredoxinH promoter (U.S.<br>
"provisional patent application 60/514,123); maize Zm40 promoter (U.S. Patent<br>
6,403,862 and WO 01/2178); maize ml_IP15 promoter (U.S. Patent 6,479,734);<br>
maize ESR promoter (U.S. patent application 10/786,679); rnaize PCNA2 promoter<br>
(U.S. patent application 10/388,359); maize cytokinin oxidase promoters (U.S.<br>
patent application 11/094,917); promoters disclosed in Weigal et al. (1992) Cell<br>
69:843-859; Accession No. AJ131822; Accession No. Z71981; Accession No.<br>
AF049870; and shoot-preferred promoters disclosed in McAvoy et al. (2003) Acta<br>
Hort. (ISHS) 625:379-385. Other dividing cell or meristematic tissue-preferred<br>
promoters that may be of interest have been disclosed in Ito et al. (1994) Plant<br>
Mol. Biol. 24:863-878; Regad etal. (1995) Mo. Gen. Genet. 248:703-711; Shaul et<br>
al. (1996) Pmc. Natl. Acad. Sci. 93:4868-4872; Ito et al. (1997) Plant J. 11:983-<br>
992; and Trehin et al. (1997) Plant Mol. Biol. 35:667-672, all of which are hereby<br>
incorporated by reference herein.<br>
Inflorescence-preferred promoters include the promoter of chalcone<br>
synthase (Van der Meer et al. (1990) Plant Mol. Biol. f 5:95-109), LAT52 (Twell et<br>
al. (1989) Mol. Gen. Genet. 217:240-245), pollen specific genes (Albani et al<br>
(1990) Plant Mol Biol. 15:605, Zm13 (Buerrero et al. (1993) Mol. Gen. Genet.<br>
224:161-168), maize pollen-specific gene (Hamilton etal. (1992) Plant Mol. Biol.<br>
18:211-218), sunflower pollen expressed gene (Baltz et al. (1992) The Plant<br>
Journal 2:713-721), and B. napus pollen specific genes (Arnoldo et al. (1992) J.<br>
Cell. Biochem, Abstract No. Y101204).<br>
Stress-inducible promoters include salt/water stress-inducible promoters<br>
such as P5CS (Zang et al. (1997) Plant Sciences 729:81-89); cold-inducible<br>
promoters, such as, cor15a (Haj'ela et al. (1990) Plant Physiol. 93:1246-1252),<br>
cor15b (Wlihelm et al. (1993) Plant Mol Biol 23:1073-1077), wsc120 (Ouellet et al.<br>
(1998) FEBS Lett. 423-324-328), ci7 (Kirch et al. (1997) Plant Mol Biol. 33:897-<br>
909), ci21A (Schneider et al. (1997) Plant Physiol. 773:335-45); drought-inducible<br>
promoters, such as, Trg-31 (Chaudhary et al (1996) Plant Mol. Biol. 30:1247-57);<br>
osmotic inducible promoters, such as, Rab17 (Vilardell et al. (1991) Plant Mol.<br>
Biol. 77:985-93) and osmotin (Raghothama et al. (1993) Plant Mol Biol 23:1117-<br>
28); and, heat inducible promoters, such as, heat shock proteins (Barros et al.<br>
(1992) Plant Mol. 79:665-75; Marrs et al. (1993) Dev. Genet. 74:27-41), and<br>
smHSP (Waters et al. (1996) J. Experimental Botany 47:325-338). Other stressinducible<br>
promoters include rip2 (U.S. Patent No. 5,332,808 and U.S. Publication<br>
Mb."'2003/0217393) "an'a "rd'2'9a (Yamaguchi-Shinozaki et al. (1993) Mol. Gen.<br>
Genetics 236:331-340).<br>
Stress-insensitive promoters can also be used in the methods of the<br>
invention. This class of promoters, as well as representative examples, are fruther<br>
described elsewhere herein.<br>
Nitrogen-responsive promoters can also be used in the methods of the<br>
invention. Such promoters include, but are not limited to, the 22 kDa Zein<br>
promoter (Spena et al. (1982) EMBO J 1: 1589-1594 and Muller et al. (1995) J.<br>
Plant Physiol 745:606-613); the 19 kDa zein promoter (Pedersen et al. (1982) Cell<br>
29:1019-1025); the 14 kDa zeln promoter (Pedersen et al. (1986) J. Biol. Chem.<br>
267:6279-6284), the b-32 promoter (Lohmer et al. (1991) EMBO J 70:617-624);<br>
and the nitrite reductase (NiR) promoter (Rastogi et al. (1997) Plant Mol Biol.<br>
34(3):465-76 and Sander et al. (1995) Plant Mol Biol. 27(1):165-77). For a review<br>
of consensus sequences found in nitrogen-induced promoters, see for example,<br>
Muller et al. (1997) The Plant Journal 72:281-291.<br>
Chemically-regulated promoters can be used to modulate the expression of<br>
a gene in a plant through the application of an exogenous chemical regulator.<br>
Depending upon the objective, the promoter may be a chemically-inducible<br>
promoter, where application of the chemical induces gene expression, or a<br>
chemical-repressible promoter, where application of the chemical represses gene<br>
expression. Chemical-inducible promoters are known in the art and include, but<br>
are not limited to, the maize ln2-2 promoter, which is activated by<br>
benzenesulfonamide herbicide safeners, the maize GST promoter, which is<br>
activated by hydrophobic electrophilic compounds that are used as pre-emergent<br>
herbicides, and the tobacco PR-1a promoter, which is activated by salicylic acid.<br>
Other chemical-regulated promoters of interest include steroid-responsive<br>
promoters (see, for example, the glucocorticoid-inducible promoter in Schena et<br>
al. (1991) Proc. Natl. Acad. Sci. USA 88:10421-10425 and McNellis et al. (1998)<br>
Plant J. 14(2):247-257) and tetracycline-inducible and tetracycline-repressible<br>
promoters (see, for example, Gatz et al. (1991) Mol. Gen. Genet. 227:229-237,<br>
and U.S. Patent Nos. 5,814,618 and 5,789,156), herein incorporated by reference.<br>
A promoter induced by cytokinin, such as the ZmCkx1-2 promoter (U.S.<br>
Patent 6,921,815, and pending U.S. patent application 11/074,144), may also be<br>
jsed in the methods and compositions of the invention. Such a construct would<br>
amplify biosynthesis of cytokinin occurring in developmental stages and/or tissues<br>
"oflnterest. Other cytokihin-inducible promoters are described in pending U.S.<br>
patent applications 11/094,917 and 60/627,394, all hereby incorporated by<br>
reference.<br>
Additional inducible promoters include heat shock promoters, such as<br>
Gmhsp17.5-E (soybean) (Czarnecka et at, (1989) Mol Cell Biol. 9(8): 3457-3463);<br>
APX1 gene promoter (Arabidopsis) (Storozhenko et al. (1998) Plant Physiol.<br>
118(3): 1005-1014): Ha hspl7.7 G4 (Helianthus annuus) (Almoguera et al. (2002)<br>
Plant Physiol. 129(1): 333-341; and Maize Hsp70 (Rochester et al. (1986) EMBO<br>
J. 5: 451-8.<br>
The methods of the invention involve introducing a polypeptide or<br>
polynucleotide into a plant. "Introducing" is intended to mean presenting to the<br>
plant the polynucleotide or polypeptide in such a manner that the sequence gains<br>
access to the interior of a cell of the plant. The methods of the invention do not<br>
depend on a particular method for introducing a sequence into a plant, only that<br>
the polynucleotide or polypeptides gains access to the interior of at least one cell<br>
of the plant. Methods for introducing polynucleotides or polypeptides into plants<br>
are known in the art and include, but are not limited to, stable transformation<br>
methods, transient transformation methods, and virus-mediated methods.<br>
"Stable transformation" is intended to mean that the nucieotide construct of<br>
interest introduced into a plant integrates into the genome of the plant and is<br>
capable of being inherited by the progeny thereof. "Transient transformation" is<br>
intended to mean that a sequence is introduced into the plant and is only<br>
temporarily expressed or present in the plant.<br>
Transformation protocols as well as protocols for introducing polypeptides<br>
or polynucleotide sequences into plants may vary depending on the type of plant<br>
or plant cell, i.e., monocot or dicot, targeted for transformation. Suitable methods<br>
of introducing polypeptides and polynucleotides into plant cells include<br>
microinjection (Crossway et al. (1986) Biotechniques 4:320-334), electroporation<br>
(Riggs et al. (1986) Proc. Nail. Acad. Sci. USA 83:5602-5606), Agrobacteriummediated<br>
transformation (U.S. Patent No. 5,563,055 and U.S. Patent No.<br>
5,981,840), direct gene transfer (Paszkowski et al. (1984) EMBO J. 3:2717-2722),<br>
and ballistic particle acceleration (see, for example, U.S. Patent Nos. 4,945,050;<br>
U.S. Patent No. 5,879,918; U.S. Patent No. 5,886,244; and, 5,932,782; Tomes et<br>
al. (1995) in Plant Cell, Tissue, and Organ Culture: Fundamental Methods, ed.<br>
Gamborg and Phillips (Springer-Verlag, Berlin); McCabe et al. (1988)<br>
Biotechnology 6:923-926); and Led transformation (WO 00/28058). Also see<br>
Weissinger et al. (1988) Ann. Rev. Genet. 22:421-477; Sanford et al. (1987)<br>
Particulate Science and Technology 5:27-37 (onion); Christou et al. (1988) Plant<br>
Physiol. 87:671-674 (soybean); McCabe et al. (1988) Bio/Technology 6:923-926<br>
(soybean); Finer and McMullen (1991) In Vitro Cell Dev. Biol. 27P:175-182<br>
(soybean); Singh etal. (1998) Theor. Appl. Genet. 96:319-324 (soybean); Datta et<br>
al. (1990) Biotechnology 8:736-740 (rice); Hoque et al. (2005) Plant Cell Tissue &amp;<br>
Organ Culture 82(1) :45-55 (rice); Sreekala et al. (2005) Plant Cell Reports<br>
24(2) :86-94 (rice); Klein et al. (1988) Proc. NatI. Acad. Sci. USA 85:4305-4309<br>
(maize); Klein et al, (1988) Biotechnology 6:559-563 (maize); U.S. Patent Nos.<br>
5,240,855; 5,322,783; and, 5,324,646; Klein et al. (1988) Plant Physiol.<br>
91:440-444 (maize); Fromm et al. (1990) Biotechnology 8:833-839 (maize);<br>
Hooykaas-Van Slogteren etal. (1984) Nature (London) 311:763-764; U.S. Patent<br>
No. 5,736,369 (cereals); Bytebier er al. (1987) Proc. NatI. Acad. Sci. USA<br>
84:5345-5349 (Liliaceae); De Wet ef al. (1985) in The Experimental Manipulation<br>
of Ovule Tissues, ed. Chapman etal. (Longman, New York), pp. 197-209 (pollen);<br>
Kaeppler et al. (1990) Plant Cell Reports 9:415-418 and Kaeppler et al. (1992)<br>
Theor. Appl. Genet. 84:560-666 (whisker-mediated transformation); D'Halluin et al.<br>
(1992) Plant Cell 4:1495-1505 (electroporation); Li et al. (1993) Plant Cell Reports<br>
12:250-255 and Christou and Ford (1995) Annals of Botany 75:407-413 (rice);<br>
Osjoda et al. (1996) Nature Biotechnology 14:745-750 (maize via Agrobacterium<br>
tumefaciens); all of which are herein incorporated by reference.<br>
In specific embodiments, the IPT sequences or the IPT promoter<br>
sequences of the invention can be provided to a plant using a variety of transient<br>
transformation methods. Such transient transformation methods include, but are<br>
not limited to, the introduction of the IPT protein or IPT promoter or variants and<br>
fragments thereof directly into the plant or the introduction of an IPT transcript into<br>
the plant. Such methods include, for example, microinjection or particle<br>
bombardment. See, for example, Crossway et al. (1986) Mol Gen. Genet.<br>
202:179-185; Nomura et al. (1986) Plant Sci. 44:53-58; Hepler et al. (1994) Proc.<br>
NatI. Acad. Sci. 91: 2176-2180 and Hush etal. (1994) The Journal of Cell Science<br>
707:775-784, all of which are herein incorporated by reference. Alternatively, the<br>
IPT polynucleotide or the IPT promoter can be transiently transformed into the<br>
plant using techniques known in the art. Such techniques include viral vector<br>
system and the precipitation of the polynucleotide in a manner that precludes<br>
§ub'sequent""refease of "the' DMA. Thus, the transcription from the particle-bound<br>
DMA can occur, but the frequency with which it is released to become integrated<br>
into the genome is greatly reduced. Such methods include the use of particles<br>
coated with polyethyenlimine (PEI; Sigma #P3143).<br>
In other embodiments, the polynucleotide of the invention may be<br>
introduced into plants by contacting plants with a virus or viral nucleic acids.<br>
Generally, such methods involve incorporating a nucleotide construct of the<br>
invention within a viral DNA or RNA molecule. It is recognized that an IPT<br>
polynucleotide of the invention may be initially synthesized as part of a viral<br>
polyprotein, which later may be processed by proteolysis in vivo or in vitro to<br>
produce the desired recombinant protein. Further, it is recognized that promoters<br>
useful for the invention also encompass promoters utilized for transcription by viral<br>
RNA polymerases. Methods for introducing polynucleotides into plants and<br>
expressing a protein encoded therein, involving viral DNA or RNA molecules, are<br>
known in the art. See, for example, U.S. Patent Nos. 5,889,191, 5,889,190,<br>
5,866,785, 5,589,367, 5,316,931, and Porta etal. (1996) Molecular Biotechnology<br>
5:209-221; herein incorporated by reference.<br>
Methods are known in the art for the targeted insertion of a polynucleotide<br>
at a specific location in the plant genome. In one embodiment, the insertion of the<br>
polynucleotide at a desired genomic location is achieved using a site-specific<br>
recombination system. See, for example, W099/25821, WO99/25854,<br>
WO99/25840, W099/25855, and WO99/25853, and US 6,187,994; 6,552,248;<br>
6,624,297; 6,331,661; 6,262,341; 6,541,231; 6,664,108; 6,300,545; 6,528,700;<br>
and 6,911,575, all of which are herein incorporated by reference. Briefly, the<br>
polynucleotide of the invention can be contained in a transfer cassette flanked by<br>
two non-recombinogenic recombination sites. The transfer cassette is introduced<br>
into a plant having stably incorporated into its genome a target site which is<br>
flanked by two non-recombinogenic recombination sites that correspond to the<br>
sites of the transfer cassette. An appropriate recombinase is provided and the<br>
transfer cassette is integrated at the target site. The polynucleotide of interest is<br>
thereby integrated at a specific chromosomal position in the plant genome.<br>
The cells that have been transformed may be grown into plants in<br>
accordance with conventional ways. See, for example, McCormick et at. (1986)<br>
Plant Cell Reports 5:81-84. These plants may then be grown, and pollinated with<br>
either the same transformed strain or different strains, and the resulting progeny<br>
having expression of the desired phenotypic characteristic identified. Two or more<br>
generations may be grown to ensure that expression of the desired phenotypic<br>
characteristic is stably maintained and inherited and then seeds harvested to<br>
ensure that expression of the desired phenotypic characteristic has been<br>
achieved, In this manner, the present invention provides transformed seed (also<br>
referred to as "transgenic seed") having a polynucleotide of the invention, for<br>
example, an expression cassette of the invention, stably incorporated into their<br>
genome.<br>
Pedigree breeding starts with the crossing of two genotypes, such as an<br>
elite line of interest and one other inbred line having one or more desirable<br>
characteristics (i.e., having stably incorporated a polynucleotide of the invention,<br>
having a modulated activity and/or level of the polypeptide of the invention, etc)<br>
which complements the elite line of interest. If the two original parents do not<br>
provide all the desired characteristics, other sources can be included in the<br>
breeding population. In the pedigree method, superior plants are selfed and<br>
selected in successive filial generations. In the succeeding filial generations the<br>
heterozygous condition gives way to homogeneous lines as a result of selfpollination<br>
and selection. Typically in the pedigree method of breeding, five or<br>
more successive filial generations of selfing and selection are practiced: F1 -» F2;<br>
F2-&gt; F3, F3 -&gt; F4; F4 ~&gt; F5, etc. After a sufficient amount of inbreeding,<br>
successive filial generations will serve to increase seed of the developed inbred.<br>
In specific embodiments, the inbred line comprises homozygous alleles at about<br>
95% or more of its loci.<br>
In addition to being used to create a backcross conversion, backcrossing<br>
can also be used in combination with pedigree breeding to modify an elite line of<br>
interest and a hybrid that is made using the modified elite line. Backcrossing can<br>
be used to transfer one or more specifically desirable traits from one line, the<br>
donor parent, to an inbred called the recurrent parent, which has overall good<br>
agronomic characteristics yet lacks that desirable trait or traits. However, the<br>
same procedure can be used to move the progeny toward the genotype of the<br>
recurrent parent but at the same time retain many components of the nonrecurrent<br>
parent by stopping the backcrossing at an early stage and proceeding<br>
with selfing and selection. For example, an F1, such as a commercial hybrid, is<br>
created. This commercial hybrid may be backcrossed to one of its parent lines to<br>
create a BC1 or BC2. Progeny are selfed and selected so that the newly<br>
"developed inbred has many or the attributes of the recurrent parent and yet<br>
several of the desired attributes of the non-recurrent parent This approach<br>
leverages the value and strengths of the recurrent parent for use in new hybrids<br>
and breeding.<br>
Therefore, an embodiment of this invention is a method of making a<br>
backcross conversion of a maize inbred line of interest, comprising the steps of<br>
crossing a plant of a maize inbred line of interest with a donor plant comprising a<br>
mutant gene or transgene conferring a desired trait (i.e., a modulation in the level<br>
of cytokinin (an increase or a decrease) or any plant phenotype resulting from the<br>
modulated cytokinin level (such plant phenotypes are discussed elsewhere<br>
herein)), selecting an F1 progeny plant comprising the mutant gene or transgene<br>
conferring the desired trait, and backcrossing the selected F1 progeny plant to a<br>
plant of the maize inbred line of interest. This method may further comprise the<br>
step of obtaining a molecular marker profile of the maize inbred line of interest and<br>
using the molecular marker profile to select for a progeny plant with the desired<br>
trait and the molecular marker profile of the inbred line of interest. In the same<br>
manner, this method may be used to produce F1 hybrid seed by adding a final<br>
step of crossing the desired trait conversion of the maize inbred line of interest<br>
with a different maize plant to make F1 hybrid maize seed comprising a mutant<br>
gene or transgene conferring the desired trait.<br>
Recurrent selection is a method used in a plant breeding program to<br>
improve a population of plants. The method entails individual plants cross<br>
pollinating with each other to form progeny. The progeny are grown and the<br>
superior progeny selected by any number of selection methods, which include<br>
individual plant, half-sib progeny, full-sib progeny, selfed progeny and topcrossing.<br>
The selected progeny are cross-pollinated with each other to form progeny for<br>
another population. This population is planted and again superior plants are<br>
selected to cross pollinate with each other. Recurrent selection is a cyclical<br>
process and therefore can be repeated as many times as desired. The objective<br>
of recurrent selection is to improve the traits of a population. The improved<br>
population can then be used as a source of breeding material to obtain inbred<br>
lines to be used in hybrids or used as parents for a synthetic cultivar. A synthetic<br>
cultivar is the resultant progeny formed by the intercrossing of several selected<br>
inbreds.<br>
'Mass selection is a useful technique when used in conjunction witn<br>
molecular marker enhanced selection. In mass selection seeds from individuals<br>
are selected based on phenotype and/or genotype. These selected seeds are<br>
then bulked and used to grow the next generation. Bulk selection requires<br>
growing a population of plants in a bulk plot, allowing the plants to self-pollinate,<br>
harvesting the seed in bulk and then using a sample of the seed harvested in bulk<br>
to plant the next generation. Instead of self pollination, directed pollination could<br>
be used as part of the breeding program.<br>
Mutation breeding is one of many methods that could be used to introduce<br>
new traits into an elite line. Mutations that occur spontaneously or are artificially<br>
induced can be useful sources of variability for a plant breeder. The goal of<br>
artificial mutagenesis is to increase the rate of mutation for a desired<br>
characteristic. Mutation rates can be increased by many different means including<br>
temperature, long-term seed storage, tissue culture conditions, radiation such as<br>
X-rays, Gamma rays (e.g., cobalt 60 or cesium 137), neutrons, (product of nuclear<br>
fission by uranium 235 in an atomic reactor), Beta radiation (emitted from<br>
radioisotopes such as phosphorus 32 or carbon 14), or ultraviolet radiation<br>
(preferably from 2500 to 2900nm), or chemical mutagens (such as base<br>
analogues (5-bromo-uracil), related compounds (8-ethoxy caffeine), antibiotics<br>
(streptonigrin), alkylating agents (sulfur mustards, nitrogen mustards, epoxides,<br>
ethylenamines, sulfates, sulfonates, sulfones, lactones), azide, hydroxylamine,<br>
nitrous acid, or acridines. Once a desired trait is observed through mutagenesis<br>
the trait may then be incorporated into existing germplasm by traditional breeding<br>
techniques, such as backcrossing. Details of mutation breeding can be found in<br>
"Principals of Cultivar Development," Fehr, 1993 Macmillan Publishing Company,<br>
the disclosure of which is incorporated herein by reference. In addition, mutations<br>
created in other lines may be used to produce a backcross conversion of elite<br>
lines that comprises such mutations.<br>
The present invention may be used for transformation of any plant species,<br>
including, but not limited to, rnonocots and dicots. Examples of plant species of<br>
interest include, but are not limited to, com (Zea mays), Brassica sp. (e.g., B. napus,<br>
B. rapa, B, juncea), particularly those Brassica species useful as sources of seed oil,<br>
alfalfa (Medicago sativa), rice (Oryza sativa), rye (Seca/e cereale), sorghum<br>
(Sorghum bicolor, Sorghum vulgare), millet (e.g., pearl millet (Pennisatum glaucum),<br>
proso millet (Pan/cum miliaceum), foxtail millet (Setaria italica), finger millet (Eleusine<br>
coracanaj), sunflower (Helianthus annuus), safflower (Carthamus tinctorius), wheat<br>
(Triticum aestivum], soybean (Glycine max), tobacco (Nicotiana tabacum), potato<br>
(Solatium tuberosum), peanuts (Arachis hypogaea), cotton (Gossypium<br>
barbadense, Gossypium hirsutum), sweet potato (Ipomoea batatus); cassava<br>
(Manihot esculenta), coffee (Coffea spp.), coconut (Cocos nucifera), pineapple<br>
(Ananas comosus), citrus trees (Citrus spp.), cocoa (Theobroma cacao), tea<br>
(Camellia s/nens/s), banana (Musa spp.), avocado (Persea americana), fig (Ficus<br>
casica), guava (Psidium guajava), mango (Mangifera indica), olive (O/ea europaea),<br>
papaya (Can'ca papaya), cashew (Anacardium occidentale), macadamia<br>
(Macadam/a integrifolia], almond (Prunus amygdalus), sugar beets (Beta vulgaris),<br>
sugarcane (Saccharum spp.), oats, barley, vegetables, ornamentals, and conifers.<br>
Vegetables include tomatoes (Lycopersicon esculentum), lettuce (e.g.,<br>
Lactuca sativa), green beans (Phaseolus vulgaris), lima beans (Phaseolus limensis),<br>
peas (Lathyrus spp.), and members of the genus Cucumis such as cucumber (C.<br>
sativus), cantaloupe (C. cantalupensis), and musk melon (C. me/o). Ornamentals<br>
include azalea (Rhododendron spp.), hydrangea (Macrophylla hydrangea), hibiscus<br>
(Hibiscus rosasanensis), roses (Rosa spp.), tulips (Tulipa spp.), daffodils (Narcissus<br>
spp.), petunias (Petunia hybrida), carnation (Dianthus caryophyllus), poinsettia<br>
(Euphorbia pulcherrima), and chrysanthemum.<br>
Conifers that may be employed in practicing the present invention include, for<br>
example, pines such as loblolly pine (Pinus taeda), slash pine (Pinus elliotii),<br>
ponderosa pine (Pinus ponderosa), lodgepole pine (Pinus contorts), and Monterey<br>
pine (Pinus radiata); Douglas-fir (Pseudotsuga menziesii); Western hemlock (Tsuga<br>
canadensis); Sitka spruce (Picea glauca); redwood (Sequoia sempervirens); true firs<br>
such as silver fir (Abies amabilis) and balsam fir (Abies balsamea)', and cedars such<br>
as Western red cedar (Thuja plicata) and Alaska yellow-cedar (Chamaecyparis<br>
nootkatensis). In specific embodiments, plants of the present invention are crop<br>
plants (for example, corn, alfalfa, sunflower, Brassica, soybean, cotton, safflower,<br>
peanut, sorghum, wheat, millet, tobacco, etc.). In other embodiments, com and<br>
soybean plants are optimal, and in yet other embodiments corn plants are optimal.<br>
Other plants of interest include grain plants that provide seeds of interest,<br>
oil-seed plants, and leguminous plants. Seeds of interest include grain seeds,<br>
such as maize, wheat, barley, rice, sorghum, rye, etc. Oil-seed plants include<br>
cotton, soybean, safflower, sunflower, Brassica, maize, alfalfa, palm, coconut, etc.<br>
Leguminous plants include beans and peas. Beans include guar, locust bean,<br>
tenugreeK, soybean, garden Beans, cowpea, mungbean, lima | bean, fava bean,<br>
lentils, chickpea, etc.<br>
Typically, an intermediate host cell will be used in the practice of this<br>
invention to increase the copy number of the cloning vector. With an increased<br>
copy number, the vector containing the nucleic acid of interest'can be isolated in<br>
significant quantities for introduction into the desired plant cells. In one<br>
embodiment, plant promoters that do not cause expression of the polypeptide in<br>
bacteria are employed.<br>
Prokaryotes most frequently are represented by various strains of £ coli;<br>
however, other microbial strains may also be used. Commonly used prokaryotic<br>
control sequences which are defined herein to include promoters for transcription<br>
initiation, optionally with an operator, along with ribosome binding sequences,<br>
include such commonly used promoters as the beta lactamase (penicillinase) and<br>
lactose (lac) promoter systems (Chang ef a/. (1977) Nature 198:1056), the<br>
tryptophan (trp) promoter system (Goeddel et a/. (1980) Nucleic Acids Res.<br>
8:4057) and the lambda derived P L promoter and N-gene ribosome binding site<br>
(Shimatake et al. (1981) Nature 292:128). The inclusion of selection markers in<br>
DNA vectors transfected in £ coli. is also useful. Examples of such markers<br>
include genes specifying resistance to ampicillin, tetracycline, or chloramphenicol.<br>
The vector is selected to allow introduction into the appropriate host cell.<br>
Bacterial vectors are typically of plasmid or phage origin. Appropriate bacterial<br>
cells are infected with phage vector particles or transfected with naked phage<br>
vector DNA. If a plasmid vector is used, the bacterial cells are transfected with the<br>
plasmid vector DNA. Expression systems for expressing a protein of the present<br>
invention are available using Bacillus sp. and Salmonella (Palva ef al. (1983)<br>
Gene 22:229-235); Mosbach et al. (1983) Nature 302:543-545).<br>
A variety of eukaryotic expression systems such as yeast, insect cell lines,<br>
plant and mammalian cells, are known to those of skill in the art. As explained<br>
briefly below, a polynucleotide of the present invention can be expressed in these<br>
eukaryotic systems. In some embodiments, transformed/transfected plant cells,<br>
as discussed infra, are employed as expression systems for production of the<br>
proteins of the instant invention.<br>
Synthesis of heterologous polynucleotides in yeast is well known (Sherman<br>
et al. (1982) Methods in Yeast Genetics, Cold Spring Harbor Laboratory). Two<br>
widely utilized yeasts for production of eukaryotic proteins are Saccharomyces<br>
cerews/ae ana Pichfa pastons. Vectors, strains, and protocols for expression in<br>
Saccharomyces and Pichia are known in the art and available from commercial<br>
suppliers (e.g., Invitrogen). Suitable vectors usually have expression control<br>
sequences, such as promoters, including 3-phosphoglycerate kinase or alcohol<br>
oxidase, and an origin of replication, termination sequences and the like as<br>
desired. A protein of the present invention, once expressed, can be isolated from<br>
yeast by lysing the cells and applying standard protein isolation techniques to the<br>
lists. The monitoring of the purification process can be accomplished by using<br>
Western blot techniques or radioimmunoassay or other standard immunoassay<br>
techniques.<br>
The sequences of the present invention can also be ligated to various<br>
expression vectors for use in transfecting cell cultures of, for instance,<br>
mammalian, insect, or plant origin. Illustrative cell cultures useful for the<br>
production of the peptides are mammalian cells. A number of suitable host cell<br>
lines capable of expressing intact proteins have been developed in the art, and<br>
include the HEK293, BHK21, and CHO cell lines. Expression vectors for these<br>
cells can include expression control sequences, such as an origin of replication, a<br>
promoter (e.g. the CMV promoter, a HSV tk promoter or pgk (phosphoglycerate<br>
kinase) promoter), an enhancer (Queen et a/. (1986) Immunol. Rev. 89:49), and<br>
necessary processing information sites, such as ribosome binding sites, RNA<br>
splice sites, polyadenylation sites (e.g., an SV40 large T Ag poly A addition site),<br>
and transcriptional terminator sequences. Other animal cells useful for production<br>
of proteins of the present invention are available, for instance, from the American<br>
Type Culture Collection,<br>
Appropriate vectors for expressing proteins of the present invention in<br>
insect cells are usually derived from the SF9 baculovirus. Suitable insect cell lines<br>
include mosquito larvae, silkworm, armyworm, moth and DrosophHa cell lines such<br>
as a Schneider cell line (See, Schneider (1987) J. Embryol. Exp. Morphol. 27:353-<br>
365).<br>
As with yeast, when higher animal or plant host cells are employed,<br>
polyadenylation or transcription terminator sequences are typically incorporated<br>
into the vector. An example of a terminator sequence is the polyadenylation<br>
sequence from the bovine growth hormone gene. Sequences for accurate splicing<br>
of the transcript may also be included. An example of a splicing sequence is the<br>
VP1 intron from SV40 (Sprague et a/. (1983) J. Virol. 45:773-781). Additionally,<br>
gene sequences to control replication in the host cell may be incorporated into the<br>
vector such as those found in bovine papilloma virus type-vectors (Saveria-Campo<br>
(1985) DNA Cloning Vol. II a.Practical Approach, D.M. Glover, Ed., IRL Press,<br>
Arlington, Virginia, pp. 213-238).<br>
Animal and lower eukaryotic (e.g., yeast) host cells are competent or<br>
rendered competent for transfection by various means. There are several wellknown<br>
methods of introducing DNA into animal cells. These include: calcium<br>
phosphate precipitation, fusion of the recipient cells with bacterial protoplasts<br>
containing the DNA, treatment of the recipient cells with liposomes containing the<br>
DNA, DEAE dextrin, electroporation, biolistics, and micro-injection of the DNA<br>
directly into the cells. The transfected cells are cultured by means well known in<br>
the art (Kuchler (1997) Biochemical Methods in Cell Culture and Virology,<br>
Dowden, Hutchinson and Ross, Inc.).<br>
//. Modulating the Concentration and/or Activity of an Isopentenvl<br>
Trans/erase Polvpeotide<br>
A method for modulating the concentration and/or activity of the polypeptide<br>
of the present invention in a plant is provided. In general, concentration and/or<br>
activity of the IPT polypeptide is increased or reduced by at least 1%, 5%, 10%,<br>
20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90% or more, relative to a native<br>
control plant, plant part, or cell which does not comprise the introduced sequence.<br>
Modulation of the concentration and/or activity may occur at one or more stages of<br>
development. In specific embodiments, the polypeptides of the present invention<br>
are modulated in monocots, such as maize.<br>
The expression level of the IPT polypeptide may be measured directly, for<br>
example, by assaying for the level of the IPT polypeptide in the plant, or indirectly,<br>
for example, by measuring the cytokinin synthesis activity in the plant. Methods<br>
for assaying for cytokinin synthesis activity are described elsewhere herein.<br>
In specific embodiments, the polypeptide or the polynucleotide of the<br>
invention is introduced into the plant cell. Subsequently, a plant cell having the<br>
introduced sequence of the invention is selected using methods known to those of<br>
skill in the art such as, but not limited to, Southern blot analysis, DNA sequencing,<br>
PCR analysis, or phenotypic analysis. A plant or plant part altered or modified by<br>
the foregoing embodiments is grown under plant forming- conditions for a time<br>
sufficient to modulate the concentration and/or activity of polypeptides of the<br>
" present invention in the plant".' Plant forming conditions are well known in the art<br>
and discussed briefly elsewhere herein.<br>
It is also recognized that the level and/or activity of the polypeptide may be<br>
modulated by employing a polynucleotide that is not capable of directing, in a<br>
transformed plant, the expression of a protein or an RNA. For example, the<br>
polynucleotides of the invention may be used to design polynucleotide constructs<br>
that can be employed in methods for altering or mutating a genomic nucleotide<br>
sequence in an organism. Such polynucleotide constructs include, but are not<br>
limited to, RNA:DNA vectors, RNA:DNA mutational vectors, RNA:DNA repair<br>
vectors, mixed-duplex oligonucleotides, self-complementary RNA:DNA<br>
oligonucleotides, and recombinogenic oligonucleobases. Such nucleotide<br>
constructs and methods of use are known in the art. See, U.S. Patent Nos.<br>
5,565,350; 5,731,181; 5,756,325; 5,760,012; 5,795,972; and 5,871,984; all of<br>
which are herein incorporated by reference. See also, WO 98/49350, WO<br>
99/07865, WO 99/25821, and Beetham et a/. (1999) Proc. Natl. Acad. Sci. USA<br>
96:8774-8778; herein incorporated by reference.<br>
It is therefore recognized that methods of the present invention do not<br>
depend on the incorporation of the entire polynucleotide into the genome, only that<br>
the plant or cell thereof is altered as a result of the introduction of the<br>
polynucleotide into a cell. In one embodiment of the invention, the genome may<br>
be altered following the introduction of a polynucleotide into a cell. For example,<br>
the polynucleotide, or any part thereof, may incorporate into the genome of the<br>
plant. Alterations to the genome include, but are not limited to, additions,<br>
deletions, and substitutions of nucleotides into the genome. While the methods of<br>
the present invention do not depend on additions, deletions, and substitutions of<br>
any particular number of nucleotides, it is recognized that such additions,<br>
deletions, or substitutions comprise at least one nucleotide.<br>
It is further recognized that modulating the level and/or activity of the IPT<br>
sequence can be performed to elicit the effects of the sequence only during<br>
certain developmental stages and to switch the effect off in other stages where<br>
expression is no longer desirable. Control of the IPT expression can be obtained<br>
via the use of induclble or tissue-preferred promoters. Alternatively, the gene<br>
could be inverted or deleted using site-specific recombinases, transposons or<br>
recombination systems, which would also- turn on or off expression of the IPT<br>
sequence.<br>
A "subject plant" or" plant cell" is one in which genetic alteration, such as<br>
transformation, has been effected as to a gene of interest, or is a plant or plant cell<br>
which is descended from a plant or cell so altered and which comprises the<br>
alteration. A "control" or "control plant" or "control plant cell" provides a reference<br>
point for measuring changes in phenotype of the subject plant or plant cell.<br>
A control plant or plant cell may comprise, for example: (a) a wild-type plant<br>
or cell, i.e., of the same genotype as the starting material for the genetic alteration<br>
which resulted in the subject plant or cell; (b) a plant or plant cell of the same<br>
genotype as the starting material but which has been transformed with a null<br>
construct (i.e. with a construct which has no known effect on the trait of interest,<br>
such as a construct comprising a marker gene); (c) a plant or plant cell which is a<br>
non-transformed segregant among progeny of a subject plant or plant cell; (d) a<br>
plant or plant cell genetically identical to the subject plant or plant cell but which is<br>
not exposed to conditions or stimuli that would induce expression of the gene of<br>
interest; or (e) the subject plant or plant cell itself, under conditions in which the<br>
gene of interest is not expressed.<br>
In the present case, for example, changes in cytokinin levels, including<br>
changes in absolute amounts of cytokinin, cytokinin ratios, cytokinin activity, or<br>
cytokinin distribution, or changes in plant or plant cell phenotype, such as<br>
flowering time, seed set, branching, senescence, stress tolerance, or root mass,<br>
could be measured by comparing a subject plant or plant cell to a control plant or<br>
plant cell.<br>
A. Increasing the Activity and/or Concentration of an Isopentenyl<br>
Transferase Polypeptide<br>
Methods are provided to increase the activity and/or concentration of the<br>
IPT polypeptide of the invention. An increase in the concentration and/or activity<br>
of the IPT polypeptide of the invention can be achieved by providing to the plant<br>
an IPT polypeptide. As discussed elsewhere herein, many methods are known in<br>
the art for providing a polypeptide to a plant including, but not limited to, direct<br>
introduction of the polypeptide into the plant, and introducing into the plant<br>
(transiently or stably) a polynucleotide construct encoding a polypeptide having<br>
cytokinin synthesis activity. It is also recognized that the methods of the invention<br>
may employ a polynucleotide that is not capable of directing, in the transformed<br>
plant, the expression of a protein or an RNA. Thus, the level and/or activity of an<br>
fPT polypeptide may be increased by altering the gene encoding the IPT<br>
polypeptide or its promoter. See, e.g., Kmiec, U.S. Patent 5,565,350; Zarling et<br>
Ql, PCT/US93/03868. Therefore mutagenized plants that carry mutations in IPT<br>
genes, where the mutations increase expression of the IPT gene or increase the<br>
cytokinin synthesis activity of the encoded IPT polypeptide are provided. As<br>
described elsewhere herein, methods to assay for an increase in protein<br>
concentration or an increase in cytokinin synthesis activity are known.<br>
B. Reducing the Activity and/or Concentration of an Isopentenyl<br>
Transferase Polypeptide<br>
Methods are provided to reduce or eliminate the activity and/or<br>
concentration of the IPT polypeptide by transforming a plant cell with an<br>
expression cassette that expresses a polynucleotide that inhibits the expression of<br>
the IPT polypeptide. The polynucleotide may inhibit the expression of an IPT<br>
polypeptide directly, by preventing translation of the IPT polypeptide messenger<br>
RNA, or indirectly, by encoding a molecule that inhibits the transcription or<br>
translation of an IPT polypeptide gene encoding an IPT polypeptide. Methods for<br>
inhibiting or eliminating the expression of a gene in a plant are well known in the<br>
art, and any such method may be used in the present invention to inhibit the<br>
expression of the IPT polypeptides.<br>
In accordance with the present invention, the expression of an IPT<br>
polypeptide is inhibited if the level of the IPT polypeptide is statistically lower than<br>
the level of the same IPT polypeptide in a plant that has not been genetically<br>
modified or mutagenized to inhibit the expression of that IPT polypeptide. In<br>
particular embodiments of the invention, the protein level of the IPT polypeptide in<br>
a modified plant according to the invention is less than 95%, less than 90%, less<br>
than 80%, less than 70%, less than 60%, less than 50%, less than 40%, less than<br>
30%, less than 20%, less than 10%, or less than 5% of the protein level of the<br>
same IPT polypeptide in a plant that is not a mutant or that has not been<br>
genetically modified to inhibit the expression of that IPT polypeptide. The<br>
expression level of the IPT polypeptide may be measured directly, for example, by<br>
assaying for the level of the IPT polypeptide expressed in the cell or plant, or<br>
indirectly, for example, by measuring the cytokinin synthesis activity in the cell or<br>
plant. Methods for determining the cytokinin synthesis activity of the IPT<br>
polypeptide are described elsewhere herein.<br>
In other embodiments of the invention, the activity of one or more IPT<br>
polypeptides is reduced or eliminated by transforming a plant cell with an<br>
expression cassette comprising a polynucleotide encoding a polypeptide that<br>
inhibits the activity of one or more IPT polypeptides. The cytokinin synthesis<br>
activity of an IPT polypeptide is inhibited according to the present invention if the<br>
cytokinin synthesis activity of the IPT polypeptide is statistically lower than the<br>
cytokinin synthesis activity of the same IPT polypeptide in a plant that has not<br>
been genetically modified to inhibit the cytokinin synthesis activity of that IPT<br>
polypeptide. In particular embodiments of the invention, the cytokinin synthesis<br>
activity of the IPT polypeptide in a modified plant according to the invention is less<br>
than 95%, less than 90%, less than 80%, less than 70%, less than 60%, less than<br>
50%, less than 40%, less than 30%, less than 20%, less than 10%, or less than<br>
5% of the cytokinin synthesis activity of the same IPT polypeptide in a plant that<br>
that has not been genetically modified to inhibit the expression of that IPT<br>
polypeptide. The cytokinin synthesis activity of an IPT polypeptide is "eliminated"<br>
according to the invention when it is not detectable by the assay methods<br>
described elsewhere herein. Methods of determining the cytokinin synthesis<br>
activity of an IPT polypeptide are described elsewhere herein.<br>
In other embodiments, the activity of an IPT polypeptide may be reduced or<br>
eliminated by disrupting the gene encoding the IPT polypeptide. The invention<br>
encompasses mutagenized plants that carry mutations in IPT genes, where the<br>
mutations reduce expression of the IPT gene or inhibit the cytokinin synthesis<br>
activity of the encoded IPT polypeptide.<br>
Thus, many methods may be used to reduce or eliminate the activity of an<br>
IPT polypeptide. More than one method may be used to reduce the activity of a<br>
single IPT polypeptide. In addition, combinations of methods may be employed to<br>
reduce or eliminate the activity of two or more different IPT polypeptides.<br>
Non-limiting examples of methods of reducing or eliminating the expression<br>
of an IPT polypeptide are given below.<br>
1- Polvnucteptide-Based Methods<br>
In some embodiments of the present invention, a plant cell is transformed<br>
with an expression cassette that is capable of expressing a polynucleotide that<br>
inhibits the expression of an IPT sequence. The term "expression" as used herein<br>
refers to the biosynthesis of a gene product, including the transcription and/or<br>
"translation of said gene product. For example, for the purposes of the present<br>
invention, an expression cassette capable of expressing a polynucleotide that<br>
inhibits the expression of at least one IPT sequence is an expression cassette<br>
capable of producing an RNA molecule that inhibits the transcription and/or<br>
translation of at least one IPT polypeptide. The "expression" or "production" of a<br>
protein or polypeptide from a DNA molecule refers to the transcription and<br>
translation of the coding sequence to produce the protein or polypeptide, while the<br>
"expression" or "production" of a protein or polypeptide from an RNA molecule<br>
refers to the translation of the RNA coding sequence to produce the protein or<br>
polypeptide.<br>
Examples of polynucleotides that inhibit the expression of an IPT sequence<br>
are given below.<br>
• Sense SuporBssion/CosuDDression<br>
In some embodiments of the invention, inhibition of the expression of an<br>
IPT polypeptide may be obtained by sense suppression or cosuppression. For<br>
cosuppression, an expression cassette is designed to express an RNA molecule<br>
corresponding to all or part of a messenger RNA encoding an IPT polypeptide in<br>
the "sense" orientation. Over expression of the RNA molecule can result in<br>
reduced expression of the native gene. Accordingly, multiple plant lines<br>
transformed with the cosuppression expression cassette are screened to identify<br>
those that show the greatest inhibition of IPT polypeptide expression.<br>
The polynucleotide used for cosuppression may correspond to all or part of<br>
the sequence encoding the IPT polypeptide, all or part of the 5' and/or 3'<br>
untranslated region of an IPT polypeptide transcript, or all or part of both the<br>
coding sequence and the untranslated regions of a transcript encoding an IPT<br>
polypeptide. In some embodiments where the polynucleotide comprises all or part<br>
of the coding region for the IPT polypeptide, the expression cassette is designed<br>
to eliminate the start codon of the polynucleotide so that no protein product will be<br>
transcribed.<br>
Cosuppression may be used to inhibit the expression of plant genes to<br>
produce plants having undetectable protein levels for the proteins encoded by<br>
these genes. See, for example, Broin at a/. (2002) Plant Cell 14:1417-1432.<br>
Cosuppression may also be used to inhibit the expression of multiple proteins in<br>
the same plant. See, for example, U.S. Patent No. 5,942,657. Methods for using<br>
cosuppression to inhibit the"1"expression of endogenous genes in plants are<br>
described in Flavell et al. (1994) Proc. Natl. Acad. Sci. USA 91:3490-3496;<br>
Jorgensen et al. (1996) Plant Mol. Biol. 31:957-973; Johansen and Carrington<br>
(2001) Plant Physiol. 126:930-938; Broin et al. (2002) Plant Cell 14:1417-1432;<br>
Stoutjesdijk et al (2002) Plant Physiol. 129:1723-1731; Yu et al. (2003)<br>
Phytochemistry 63:753-763; and U.S. Patent Nos. 5,034,323, 5,283,184, and<br>
5,942,657; each of which is herein incorporated by reference. The efficiency of<br>
cosuppression may be increased by including a poly-dT region in the expression<br>
cassette at a position 3' to the sense sequence and 5' of the polyadenylation<br>
signal. See, U.S. Patent. Publication No. 20020048814, herein incorporated by<br>
reference. Typically, such a nucleotide sequence has substantial sequence<br>
identity to the sequence of the transcript of the endogenous gene, optimally<br>
greater than about 65% sequence identity, more optimally greater than about 85%<br>
sequence identity, most optimally greater than about 95% sequence identity. See,<br>
U.S. Patent Nos. 5,283,184 and 5,034,323; herein incorporated by reference.<br>
'• Antisense Suppression<br>
In some embodiments of the invention, inhibition of the expression of the<br>
IPT polypeptide may be obtained by antisense suppression. For antisense<br>
suppression, the expression cassette is designed to express an RNA molecule<br>
complementary to all or part of a messenger RNA encoding the IPT polypeptide.<br>
Over expression of the antisense RNA molecule can result in reduced expression<br>
of the native gene. Accordingly, multiple plant lines transformed with the<br>
antisense suppression expression cassette are screened to identify those that<br>
show the greatest inhibition of IPT polypeptide expression.<br>
The polynucleotide for use in antisense suppression may correspond to all<br>
or part of the complement of the sequence encoding the IPT polypeptide, all or<br>
part of the complement of the 5' and/or 3' untranslated region of the IPT<br>
polypeptide transcript, or all or part of the complement of both the coding<br>
sequence and the untranslated regions of a transcript encoding the IPT<br>
polypeptide. In addition, the antisense polynucleotide may be fully complementary<br>
(i.e.. 100% identical to the complement of the target sequence) or partially<br>
complementary (i.e., less than 100% identical to the complement of the target<br>
sequence) to the target sequence. Antisense suppression may be used to. inhibit<br>
the expression of multiple proteins in the same plant. See, for example, U.S.<br>
Patent No. 5,942,657 FurtrieTmore, portions of the antisense nucleotides may be<br>
used to disrupt the expression of the target gene. Generally, sequences of at<br>
least 50 nucleotides, 100 nucleotides, 200 nucleotides, 300,400,450, 500, 550, or<br>
greater may be used. Methods for using antisense suppression to inhibit the<br>
expression of endogenous genes in plants are described, for example, in Liu at al<br>
(2002) Plant Physiol. 129:1732-1743 and U.S. Patent Nos. 5,759,829 and<br>
5,942,657, each of which is herein incorporated by reference. Efficiency of<br>
antisense suppression may be increased by including a poly-dT region in the<br>
expression cassette at a position 3' to the antisense sequence and 5' of the<br>
polyadenylation signal. See, U.S. Patent Publication No. 20020048814, herein<br>
incorporated by reference.<br>
Hi- Double-Stranded RNA Interference<br>
In some embodiments of the invention, inhibition of the expression of an<br>
IPT polypeptide may be obtained by double-stranded RNA (dsRNA) interference.<br>
For dsRNA interference, a sense RNA molecule like that described above for<br>
cosuppression and an antisense RNA molecule that is fully or partially<br>
complementary to the sense RNA molecule are expressed in the same cell,<br>
resulting in inhibition of the expression of the corresponding endogenous<br>
messenger RNA.<br>
Expression of the sense and antisense molecules can be accomplished by<br>
designing the expression cassette to comprise both a sense sequence and an<br>
antisense sequence. Alternatively, separate expression cassettes may be used<br>
for the sense and antisense sequences. Multiple plant lines transformed with the<br>
dsRNA interference expression cassette or expression cassettes are then<br>
screened to identify plant lines that show the greatest inhibition of IPT polypeptide<br>
expression. Methods for using dsRNA interference to inhibit the expression of<br>
endogenous plant genes are described in Waterhouse et al. (1998) Proc, Natl.<br>
Acad. Sci. USA 95:13959-13964, Liu et al. (2002) Plant Physiol. 129:1732-1743,<br>
and WO 99/49029, WO 99/53050, WO 99/61631, and WO 00/49035; each of<br>
which is herein incorporated by reference.<br>
/V. Hsiimin RNA Interference and Intron-Containina Hairpin<br>
SNA Interference<br>
In some embodiments of the invention, inhibition of the expression of one or<br>
more IPT polypeptides may be obtained by hairpin RNA (hpRNA) interference or<br>
intron-containing hairpin RNA (ihpRNA) interference. These methods are highly<br>
efficient at inhibiting the expression of endogenous genes. See, Waterhouse and<br>
Helliwell (2003) Nat. Rev. Genet. 4:29-38 and the references cited therein.<br>
For hpRNA interference, the expression cassette is designed to express an<br>
RNA molecule that hybridizes with itself to form a hairpin structure that comprises<br>
a single-stranded loop region and a base-paired stem. The base-paired stem<br>
region comprises a sense sequence corresponding to all or part of the<br>
endogenous messenger RNA encoding the gene whose expression is to be<br>
inhibited, and an antisense sequence that is fully or partially complementary to the<br>
sense sequence. Thus, the base-paired stem region of the molecule generally<br>
determines the specificity of the RNA interference. hpRNA molecules are highly<br>
efficient at inhibiting the expression of endogenous genes, and the RNA<br>
interference they induce is inherited by subsequent generations of plants. See, for<br>
example, Chuang and Meyerowitz (2000) Proc. Nail. Acad. Sci. USA 97:4985-<br>
4990; Stoutjesdijk et al. (2002) Plant Physiol. 129:1723-1731; and Waterhouse<br>
and Helliwell (2003) Nat. Rev. Genet. 4:29-38. Methods for using hpRNA<br>
interference to inhibit or silence the expression of genes are described, for<br>
example, in Chuang and Meyerowitz (2000) Proc. Natl. Acad. Sci. USA 97:4985-<br>
4990; Stoutjesdijk et al. (2002) Plant Physiol. 129:1723-1731; Waterhouse and<br>
Helliwell (2003) Nat. Rev. Genet. 4:29-38; Pandolfini et al. BMC Biotechnology<br>
3:7, and U.S. Patent Publication No. 20030175965; each of which is herein<br>
incorporated by reference. A transient assay for the efficiency of hpRNA<br>
constructs to silence gene expression in vivo has been described by Panstruga et<br>
al. (2003) Mol. Biol. Rep. 30:135-140, herein incorporated by reference.<br>
Alternatively, the base-paired stem region may correspond to a portion of a<br>
promoter sequence controlling expression of the gene to be inhibited.<br>
Transcriptional gene silencing (TGS) may be accomplished through use of hpRNA<br>
constructs wherein the inverted repeat of the hairpin shares sequence identity with<br>
the promoter region driving expression of a gene to be silenced. Processing of the<br>
hpRNA into short RNAs which can interact with the homologous promoter, region<br>
may trigger degradation"or methylation to result in silencing (Aufsatz et al. (2002)<br>
PNAS 99 (Suppl. 4): 16499-16506; Matte et al. (2000) EMBO J19(19):5194-5201).<br>
For ihpRNA, the interfering molecules have the same general structure as<br>
for hpRNA, but the RNA molecule additionally comprises an intron that is capable<br>
of being spliced in the cell in which the ihpRNA is expressed. The use of an intron<br>
minimizes the size of the loop in the hairpin RNA molecule following splicing, and<br>
this increases the efficiency of interference. See, for example, Smith et a/. (2000)<br>
Nature 407:319-320. In fact, Smith et al. show 100% suppression of endogenous<br>
gene expression using ihpRNA-mediated interference. Methods for using ihpRNA<br>
interference to inhibit the expression of endogenous plant genes are described, for<br>
example, in Smith et al. (2000) Nature 407:319-320; Wesley et al. (2001) Plant J.<br>
27:581-590; Wang and Waterhouse (2001) Curr. Opin. Plant Biol. 5:146-150;<br>
Waterhouse and Helliwell (2003) Nat. Rev. Genet 4:29-38; Helliwell and<br>
Waterhouse (2003) Methods 30:289-295, and U.S. Patent Publication No.<br>
20030180945, each of which is herein incorporated by reference.<br>
The expression cassette for hpRNA interference may also be designed<br>
such that the sense sequence and the antisense sequence do not correspond to<br>
an endogenous RNA. In this embodiment, the sense and antisense sequence<br>
flank a loop sequence that comprises a nucleotide sequence corresponding to all<br>
or part of the endogenous messenger RNA of the target gene. Thus, it is the loop<br>
region that determines the specificity of the RNA interference. See, for example,<br>
WO 02/00904, herein incorporated by reference.<br>
v- Amolicon-Mediated Interference<br>
Amplicon expression cassettes comprise a plant virus-derived sequence<br>
that contains all or part of the target gene but generally not all of the genes of the<br>
native virus. The viral sequences present in the transcription product of the<br>
expression cassette allow the transcription product to direct its own replication.<br>
The transcripts produced by the amplicon may be either sense or antisense<br>
relative to the target sequence (i.e., the messenger RNA for an IPT polypeptide).<br>
Methods of using amplicons to inhibit the expression of endogenous plant genes<br>
are described, for example, in Angell and Baulcombe (1997) EMBO J. 16:3675-<br>
3684, Angell and Baulcombe (1999) Plant J. 20:357-362, and U.S. Patent No.<br>
6,646,805, each of which is herein incorporated by reference.<br>
vi.<br>
In some embodiments, the polynucleotide" expressed by the expression<br>
cassette of the invention is catalytic RNA or has ribozyme activity specific for the<br>
messenger RNA of an IPT polypeptide. Thus, the polynucleotide causes the<br>
degradation of the endogenous messenger RNA, resulting in reduced expression<br>
of the IPT polypeptide. This method is described, for example, in U.S. Patent No.<br>
4,987,071, herein incorporated by reference.<br>
vii. Small Interfering RNA or Micro RNA<br>
In some embodiments of the invention, inhibition of the expression of one or<br>
more IPT polypeptides may be obtained by RNA interference by expression of a<br>
gene encoding a micro RNA (miRNA). miRNAs are regulatory agents consisting<br>
of about 22 ribonucleotides. miRNA are highly efficient at inhibiting the expression<br>
of endogenous genes. See, for example Javier et al. (2003) Nature 425: 257-263,<br>
herein incorporated by reference.<br>
For miRNA interference, the expression cassette is designed to express an<br>
RNA molecule that is modeled on an endogenous miRNA gene. The miRNA gene<br>
encodes an RNA that forms a hairpin structure containing a 22-nucleotide<br>
sequence that is complementary to another endogenous gene (target sequence).<br>
For suppression of IPT polypeptide expression, the 22-nucleotide sequence is<br>
selected from an IPT polypeptide transcript sequence and contains 22 nucleotides<br>
encoding said IPT polypeptide sequence in sense orientation and 21 nucleotides<br>
of a corresponding antisense sequence that is complementary to the sense<br>
sequence. miRNA molecules are highly efficient at inhibiting the expression of<br>
endogenous genes, and the RNA interference they induce is inherited by<br>
subsequent generations of plants.<br>
2. Polvpeotide-Based Inhibition of Gene Expression<br>
In one embodiment, the polynucleotide encodes a zinc finger protein that<br>
binds to a gene encoding an IPT polypeptide, resulting in reduced expression of<br>
the gene. In particular embodiments, the zinc finger protein binds to a regulatory<br>
region of an IPT polypeptide gene. In other embodiments, the zinc finger protein<br>
binds to a messenger RNA encoding an IPT polypeptide and prevents its<br>
translation. Methods of selecting sites for targeting by zinc finger proteins have<br>
been described, for example, in U.S. Patent No. 6,453,242, and methods for using<br>
zinc finger proteins to inhibit the expression of genes in plants are described, for<br>
example, in U.S. Patent Publication No. 20030037355; each of which is herein<br>
incorporated by reference.<br>
3. Polypeptide-Based Inhibition of Protein Activity<br>
In some embodiments of the invention, the polynucleotide encodes an<br>
antibody that binds to at least one IPT polypeptide, and reduces the cytokinin<br>
synthesis activity of the IPT polypeptide. In another embodiment, the binding of<br>
the antibody results in increased turnover of the antibody-lPT polypeptide complex<br>
by cellular quality control mechanisms. The expression of antibodies in plant cells<br>
and the inhibition of molecular pathways by expression and binding of antibodies<br>
to proteins in plant cells are well known in the art. See, for example, Conrad and<br>
Sonnewald (2003) Nature Biotech, 21:35-36, incorporated herein by reference.<br>
4. Gene Disruption<br>
In some embodiments of the present invention, the activity of an IPT<br>
polypeptide is reduced or eliminated by disrupting the gene encoding the IPT<br>
polypeptide. The gene encoding the IPT polypeptide may be disrupted by any<br>
method known in the art. For example, in one embodiment, the gene is disrupted<br>
by transposon tagging. In another embodiment, the gene is disrupted by<br>
mutagenizing plants using random or targeted mutagenesis, and selecting for<br>
plants that have reduced IPT activity.<br>
'• Transposon Tagging<br>
In one embodiment of the invention, transposon tagging is used to reduce<br>
or eliminate the cytokinin synthesis activity of one or more IPT polypeptides.<br>
Transposon tagging comprises inserting a transposon within an endogenous IPT<br>
gene to reduce or eliminate expression of the IPT polypeptide. "IPT gene" is<br>
intended to mean the gene that encodes an IPT polypeptide according to the<br>
invention.<br>
In this embodiment, the expression of one or more IPT polypeptides is<br>
reduced or eliminated by inserting a transposon within a regulatory region or<br>
coding region of the gene encoding the IPT polypeptide. A transposon that is<br>
within an exon, intron, 5' or 3' untranslated sequence, a promoter, or any other<br>
regulatory sequence of an IPT polypeptide gene may be used to reduce or<br>
eliminate the expression and/or activity of the encoded IPT polypeptide.<br>
Methods for the transposon tagging of specific genes in plants are well<br>
known in the art. See, for example, Maes et al. (1999) Trends Plant Sci. 4:90-96;<br>
Dharmapuri and Sonti (1999) FEMS Microbiol. Lett. 179:53-59; Meissner et al.<br>
(2000) Plant J. 22:265-274; Phogat et al. (2000) J. Biosci. 25:57-63; Walbot (2000)<br>
Curr. Opin. Plant Btol 2:103-107; Gai et al. (2000) Nucleic Acids Res. 28:94-96;<br>
Fitzmaurice et al. (1999) Genetics 153:1919-1928). In addition, theTUSC process<br>
for selecting Mu insertions in selected genes has been described in Bensen et al.<br>
(1995) Plant Cell 7:75-84; Mena et al. (1996) Science 274:1537-1540; and U.S.<br>
Patent No. 5,962,764; each of which is herein incorporated by reference.<br>
//. Mutant Plants with Reduced Activity<br>
Additional methods for decreasing or eliminating the expression of<br>
endogenous genes in plants are also known in the art and can be similarly applied<br>
to the instant invention. These methods include other forms of mutagenesis, such<br>
as ethyl methanesulfonate-induced mutagenesis, deletion mutagenesis, and fast<br>
neutron deletion mutagenesis used in a reverse genetics sense (with PCR) to<br>
identify plant lines in which the endogenous gene has been deleted. For<br>
examples of these methods see Ohshima et al. (1998) Virology 243:472-481;<br>
Okubara et al. (1994) Genetics 137:867-874; and Quesada et al. (2000) Genetics<br>
154:421-436; each of which is herein incorporated by reference. In addition, a fast<br>
and automatable method for screening for chemically induced mutations, TILLING<br>
(Targeting Induced Local Lesions In Genomes), using denaturing HPLC or<br>
selective endonuclease digestion of selected PCR products is also applicable to<br>
the instant invention. See McCallum et al. (2000) Nat Biotechnol. 18:455-457,<br>
herein incorporated by reference.<br>
Mutations that impact gene expression or that interfere with the function<br>
(IPT activity) of the encoded protein are well known in the art. Insertional<br>
mutations in gene exons usually result in null-mutants. Mutations in conserved<br>
residues are particularly effective in inhibiting the cytokinin synthesis activity of the<br>
encoded protein. Conserved residues of plant IPT polypeptides suitable for<br>
mutagenesis with the goal to eliminate IPT activity have been described. See, for<br>
example, Figure 1. Such mutants can be isolated according to well-known<br>
procedures, and mutations In different IPT loci can be stacked by genetic crossing.<br>
See, for example, Gruis ef al. (2002) Plant'Cell 14:2863-2882.<br>
In another embodiment of this invention, dominant mutants can be used to<br>
trigger RNA silencing due to gene inversion and recombination of a duplicated<br>
gene locus. See, for example, Kusaba et al. (2003) Plant Cell 15:1455-1467.<br>
The invention encompasses additional methods for reducing or eliminating<br>
the activity of one or more IPT polypeptides. Examples of other methods for<br>
altering or mutating a genomic nucleotide sequence in a plant are known in the art<br>
and include, but are not limited to, the use of RNA:DNA vectors, RNA:DNA<br>
mutational vectors, RNA:DNA repair vectors, mixed-duplex oligonucleotides, selfcomplementary<br>
RNA:DNA oligonucleotides, and recombinogenic<br>
oligonucleobases. Such vectors and methods of use are known in the art. See,<br>
for example, U.S. Patent Nos. 5,565,350; 5,731,181; 5,756,325; 5,760,012;<br>
5,795,972; and 5,871,984; each of which are herein incorporated by reference.<br>
See also, WO 98/49350, WO 99/07865, WO 99/25821, and Beetham ef al, (1999)<br>
Proc. Natl. Acad. Sci. USA 96:8774-8778; each of which is herein incorporated by<br>
reference.<br>
Ill- Modulating Cvtokinin Level and/or Activity<br>
As used herein, "cytokinin" refers to a class, or member of the class, of<br>
plant-specific hormones that play a central role during the cell cycle and influence<br>
numerous developmental programs. Cytokinins comprise an N8-substituted purine<br>
derivative. Representative cytokinins include isopentenyladenine (N6-(A2-<br>
isopentenyl)adenine (hereinafter, iP), zeatin (6-(4-hydroxy-3methylbut-trans-2-<br>
enylamino) purine) (hereinafter, Z), and dihydrozeatin (DZ). The free bases and<br>
their ribosides (iPR, ZR, and DZR) are believed to be the active compounds.<br>
Additional cytokinins are known. See, for example, U.S. Patent No. 5,211,738 and<br>
Keiber ef al. (2002) Cytokinins, The Arabidopsis Book, American Society of Plant<br>
Biologists, both of which are herein incorporated by reference.<br>
"Modulating the cytokinin level" includes any statistically significant<br>
decrease or increase in cytokinin level and/or activity in the plant when compared<br>
to a control plant. For example, modulating the level and/or activity can comprise<br>
either an increase or a decrease in overall cytokinin content of about 0.1%, 0.5%,<br>
1%, 3% 5%, 10%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%,<br>
70%, 75%, 80%, 85%, 90%, 95%, or greater when compared to a control plant or<br>
plant part. Alternatively, the "modulated level and/or activity of the cytokinin can<br>
include about a 0.2 fold, 0.5 fold, 2 fold, 4 fold, 8 fold, 16 fold, 32 fold or greater<br>
overall increase or decrease in cytokinin level/activity in the plant or a plant part<br>
when compared to a control plant or plant part.<br>
It is further recognized that the modulation of the cytokinin level/activity<br>
need not be an overall increase/decrease in cytokinin level and/or activity, but also<br>
includes a change in tissue distribution of the cytokinin. Moreover, the modulation<br>
of the cytokinin level/activity need not be an overall increase/decrease in<br>
cytokinins, but also includes a change in the ratio of various cytokinin derivatives.<br>
For example, the ratio of various cytokinin derivatives such as isopentenyladeninetype,<br>
zeatin-type, or dihydrozeatin-type cytokinins, and the like, could be altered<br>
and thereby modulate the level/activity of the cytokinin of the plant or plant part<br>
when compared to a control plant.<br>
Methods for assaying for a modulation in cytokinin level and/or activity are<br>
known in the art. For example, representative methods for cytokinin extraction,<br>
immunopurification, HPLC separation, and quantification by ELISA methods can<br>
be found, for example, in Faiss et a/. (1997) Plant J. &gt;/2:401-415. See, also,<br>
Werner et al. (2001) PAWS 98:10487-10492) and Dewitte et a/. (1999) Plant<br>
Physiol. 119:111-121. Each of these references are herein incorporated by<br>
reference. As discussed elsewhere herein, modulation in cytokinin level and/or<br>
activity can further be detected by monitoring for particular plant phenotypes.<br>
Such phenotypes are described elsewhere herein.<br>
In specific methods, the level and/or activity of a cytokinin in a plant is<br>
increased by increasing the level or activity of the IPT polypeptide in the plant.<br>
Methods for increasing the level and/or activity of IPT polypeptides in a plant are<br>
discussed elsewhere herein. Briefly, such methods comprise providing an IPT<br>
polypeptide of the invention to a plant and thereby increasing the level and/or<br>
activity of the IPT polypeptide. In other embodiments, an IPT nucleotide sequence<br>
encoding an IPT polypeptide can be provided by introducing into the plant a<br>
polynucleotide comprising an IPT nucleotide sequence of the invention,<br>
expressing the IPT sequence, and thereby increasing the level and/or activity of a<br>
cytokinin in the plant or plant part when compared to a control plant. In some<br>
embodiments, the IPT nucleotide construct introduced into the plant is stably<br>
incorporated into the genome of the plant. -<br>
In otner meinoas, tne level and/or activity of cytokinin in a plant is<br>
decreased by decreasing the level and/or activity of one or more of the IPT<br>
polypeptides in the plant. Such methods are disclosed in detail elsewhere herein.<br>
In one such method, an IPT nucleotide sequence is introduced into the plant and<br>
expression of the IPT nucleotide sequence decreases the activity of the IPT<br>
polypeptide, and thereby decreases the level and/or activity of a cytokinin in the<br>
plant or plant part when compared to a control plant or plant part. In other<br>
embodiments, the IPT nucleotide construct introduced into the plant is stably<br>
incorporated into the genome of the plant.<br>
As discussed above, one of skill will recognize the appropriate promoter to<br>
use to modulate the level/activity of a cytokinin in the plant. Exemplary promoters<br>
for this embodiment have been disclosed elsewhere herein.<br>
Accordingly, the present invention further provides plants having a<br>
modulated level/activity of a cytokinin when compared to the cytokinin level/activity<br>
of a control plant. In one embodiment, the plant of the invention has an increased<br>
level/activity of the iPT polypeptide of the invention, and thus has an increased<br>
level/activity of cytokinin. In other embodiments, the plant of the invention has a<br>
reduced or eliminated level of the IPT polypeptide of the invention, and thus has a<br>
decreased level/activity of a cytokinin. In certain embodiments, such plants have<br>
stably incorporated into their genome a nucleic acid molecule comprising an IPT<br>
nucleotide sequence of the invention operably linked to a promoter that drives<br>
expression in the plant cell.<br>
IV. Modulating Root Development<br>
Methods for modulating root development in a plant are provided. By<br>
"modulating root development" is intended any alteration in the development of the<br>
plant root when compared to a control plant. Such alterations in root development<br>
include, but are not limited to, alterations in the growth rate of the primary root, the<br>
fresh root weight, the extent of lateral and adventitious root formation, the<br>
vasculature system, meristem development, or radial expansion.<br>
Methods for modulating root development in a plant are provided. The<br>
methods comprise modulating the level and/or activity of the IPT polypeptide in the<br>
plant. In one method, an IPT sequence of the invention is provided to the plant.<br>
In another method, the IPT nucleotide sequence is provided by introducing into the<br>
plant a polynucleotide comprising an IPT nucleotide sequence of the invention<br>
 (which may be a fragment of a full-length IPT sequence provided), expressing said<br>
IPT sequence, and thereby modifying root development. In still other methods,<br>
the IPT nucleotide construct introduced into the plant is stably incorporated into<br>
the genome of the plant,<br>
In other methods, root development is modulated by decreasing the level or<br>
activity of the IPT polypeptide in the plant. Such methods can comprise<br>
introducing an IPT nucleotide sequence into the plant and decreasing the activity<br>
of the IPT polypeptide. In some methods, the IPT nucleotide construct introduced<br>
into the plant is stably incorporated into the genome of the plant. A decrease in<br>
cytokinin synthesis activity can result in at least one or more of the following<br>
alterations to root development, including, but not limited to, larger root meristems,<br>
increased root growth, enhanced radial expansion, an enhanced vasculature<br>
system, increased root branching, more adventitious roots, and/or an increase in<br>
fresh root weight when compared to a control plant.<br>
As used herein, "root growth" encompasses all aspects of growth of the<br>
different parts that make up the root system at different stages of its development<br>
in both monocotyledonous and dicotyledonous plants. It is to be understood that<br>
enhanced root growth can result from enhanced growth of one or more of its parts<br>
including the primary root, lateral roots, adventitious roots, etc. Methods of<br>
measuring such developmental alterations in the root system are known in the art.<br>
See, for example, U.S. Application No. 2003/0074698 and Werner et a/. (2001)<br>
PA/AS 18:10487-10492, both of which are herein incorporated by reference.<br>
As discussed above, one of skill will recognize the appropriate promoter to<br>
use to modulate root development in the plant. Exemplary promoters for this<br>
embodiment include constitutive promoters and root-preferred promoters.<br>
Exemplary root-preferred promoters have been disclosed elsewhere herein.<br>
Stimulating root growth and increasing root mass by decreasing the activity<br>
and/or level of the IPT polypeptide also finds use in improving the standability of a<br>
plant. The term "resistance to lodging" or "standability" refers to the ability of a<br>
plant to fix itself to the soil. For plants with an erect or semi-erect growth habit,<br>
this term also refers to the ability to maintain an upright position under adverse<br>
environmental conditions. This trait relates to the size, depth and morphology of<br>
the root system. In addition, stimulating root growth and increasing root mass by<br>
decreasing the level and/or activity of the IPT polypeptide at appropriate<br>
developmental stages also finds use in promoting in vitro propagation of explants.<br>
Increased root biomass and/or altered root architecture may also find use in<br>
improving nitrogen-use efficiency of the plant. Such improved efficiency may lead<br>
to, for example, an increase in plant biomass and/or seed yield at an existing level<br>
of available nitrogen, or maintenance of plant biomass and/or seed yield when<br>
available nitrogen is limited. Thus, agronomic and/or environmental benefits may<br>
ensue.<br>
Furthermore, higher root biomass production due to a decreased level<br>
and/or activity of an IPT polypeptide has an indirect effect on production of<br>
compounds produced by root cells or transgenic root cells or cell cultures of said<br>
transgenic root cells. One example of an interesting compound produced in root<br>
cultures is shikonin, the yield of which can be advantageously enhanced by said<br>
methods.<br>
Accordingly, the present invention further provides plants having modulated<br>
root development when compared to the root development of a control plant. In<br>
some embodiments, the plant of the invention has a decreased level/activity of an<br>
IPT polypeptide of the invention and has enhanced root growth and/or root<br>
biomass. In certain embodiments, such plants have stably incorporated into their<br>
genome a nucleic acid molecule comprising an IPT nucleotide sequence of the<br>
invention operably linked to a promoter that drives expression in the plant cell.<br>
V. Modulating Shoot and Leaf Development<br>
Methods are also provided for modulating vegetative tissue growth in<br>
plants. In one embodiment, shoot and leaf development in a plant is modulated.<br>
By "modulating shoot and/or leaf development" is intended any alteration in the<br>
development of the plant shoot and/or leaf when compared to a control plant or<br>
plant part. Such alterations in shoot and/or leaf development include, but are not<br>
limited to, alterations in shoot meristem development, in leaf number, leaf size,<br>
leaf and stem vasculature, internode length, and leaf senescence. As used<br>
herein, "leaf development" and "shoot development" encompasses all aspects of<br>
growth of the different parts that make up the leaf system and the shoot system,<br>
respectively, at different stages of their development, both in monocotyledonous<br>
and dicotyledonous plants. Methods for measuring such developmental<br>
alterations in the shoot and leaf system are known in the art. See, for example,<br>
Werner et at. (2001) PNAS 98:10487-10492 and U.S. Application<br>
No. 2003/0074698, each of which is herein incorporated by reference.<br>
the method tor modulating shoot and/or leaf development in a plant<br>
comprises modulating the activity and/or level of an IPT polypeptide of the<br>
invention. In one embodiment, an IPT sequence of the invention is provided. In<br>
other embodiments, the IPT nucleotide sequence can be provided by introducing<br>
into the plant a poiynucleotide comprising an IPT nucleotide sequence of the<br>
invention, expressing the IPT sequence, and thereby modifying shoot and/or leaf<br>
development. In other embodiments, the IPT nucleotide construct introduced into<br>
the plant is stably incorporated into the genome of the plant.<br>
In specific embodiments, shoot or leaf development is modulated by<br>
decreasing the level and/or activity of the IPT polypeptide in the plant. A decrease<br>
in IPT activity can result in one or more alterations in shoot and/or leaf<br>
development, including, but not limited to, smaller apical meristems, reduced leaf<br>
number, reduced leaf surface, reduced vascular tissues, shorter internodes and<br>
stunted growth, and accelerated leaf senescence, when compared to a control<br>
plant.<br>
As discussed above, one of skill will recognize the appropriate promoter to<br>
use to modulate shoot and leaf development of the plant. Exemplary promoters<br>
for this embodiment include constitutive promoters, shoot-preferred promoters,<br>
shoot meristem-preferred promoters, senescence-activated promoters, stressinduced<br>
promoters, root-preferred promoters, nitrogen-induced promoters and<br>
leaf-preferred promoters. Exemplary promoters have been disclosed elsewhere<br>
herein.<br>
Decreasing cytokinin synthesis activity in a plant generally results in shorter<br>
internodes and stunted growth. Thus, the methods of the invention find use in<br>
producing dwarf plants. In addition, as discussed above, modulation of cytokinin<br>
synthesis activity in the plant modulates both root and shoot growth. Thus, the<br>
present invention further provides methods for altering the root/shoot ratio.<br>
Shoot or leaf development can further be modulated by increasing the level<br>
and/or activity of the IPT polypeptide in the plant. An increase in IPT activity can<br>
result in one or more alterations in shoot and/or leaf development including, but<br>
not limited to, increased leaf number, increased leaf surface, increased vascular<br>
tissue, increased shoot formation, longer intemodes, improved growth, improved<br>
plant yield and vigor, and retarded leaf senescence when compared to a control<br>
plant.<br>
In one embodiment, the tolerance of a plant to flooding is improved.<br>
Flooding is a serious environmental stress that affects plant growth and<br>
productivity. Flooding causes premature senescence which results in leaf<br>
chlorosis, necrosis, defoliation, cessation of growth and reduced yield. Cytokinins<br>
can regulate senescence, and by increasing the level/activity of the IPT<br>
polypeptide in the plant, the present invention improves the tolerance of the plant<br>
to a variety of environmental stresses, including flooding. Delayed senescence<br>
may also advantageously expand the maturity adaptation of crops, improve the<br>
shelf-life of potted plants, and extend the vase-life of cut flowers.<br>
In still other embodiments, methods for modulating shoot regeneration in a<br>
callus are provided. In this method, increasing the level and/or activity of the IPT<br>
polypeptide will increase the level of cytokinins in the plant. Accordingly, lower<br>
concentrations of exogenous growth regulators (i.e., cytokinins) or no exogenous<br>
cytokinins in the culture medium will be needed to enhance shoot regeneration in<br>
callus. Thus, in one embodiment of the invention, the increased level and/or<br>
activity of the IPT sequence can be used to overcome the poor shooting potential<br>
of certain species that has limited the success and speed of transgene technology<br>
for those species. Moreover, multiple shoot induction can be induced for crops<br>
where it is economically desirable to produce as many shoots as possible.<br>
Accordingly, methods are provided to increase the rate of regeneration for<br>
transformation. In specific embodiments, the IPT sequence will be under the<br>
control of an inducible promoter (e.g., heat shock promoter, chemically inducible<br>
promoter). Additional inducible promtors are known in the art and are discussed<br>
elsewhere herein.<br>
Methods for establishing callus from explants are known. For example,<br>
roots, stems, buds, and aseptically germinated seedlings are just a few of the<br>
sources of tissue that can be used to induce callus formation. Generally, young<br>
and actively growing tissues (i.e., young leaves, roots, meristems or other tissues)<br>
are used, but are not required. Callus formation is controlled by growth regulating<br>
substances present in the medium (auxins and cytokinins). The specific<br>
concentrations of plant regulators needed to induce callus formation vary from<br>
species to species and can even depend on the source of explant. In some<br>
instances, it is advised to use different growth substances (e.g. 2, 4-D or NAA) or<br>
a combination of them during tests, since some species may not respond to a<br>
specific growth regulator. In addition, culture conditions (i.e., light, temperature,<br>
etc".) can also influence the establishment of callus. Once established, callus<br>
cultures can be used to initiate shoot regeneration. See, for example, Gurel et a/.<br>
(2001) Turk J. Bat. 25:25-33; Dodds et a/. (1995). Experiments in Plant Tissue<br>
Culture, Cambridge University Press; Gamborg (1995) Plant Cell, Tissue and<br>
Organ Culture, eds. G. Phillips; and, U.S. Application No. 20030180952, all of<br>
which are herein incorporated by reference.<br>
It is further recognized that increasing seed size and/or weight can be<br>
accompanied by an increase in the rate of growth of seedlings or an increase in<br>
vigor. In addition, modulating the plant's tolerance to stress, as discussed below,<br>
along with modulation of root, shoot and leaf development can increase plant yield<br>
and vigor. As used herein, the term "vigor" refers to the relative health,<br>
productivity, and rate of growth of the plant and/or of certain plant parts, and may<br>
be reflected in various developmental attributes, including, but not limited to,<br>
concentration of chlorophyll, photosynthetic rate, total biomass, root biomass,<br>
grain quality, and/or grain yield. In Zee mays in particular, vigor may also be<br>
reflected in ear growth rate, ear size, and/or expansiveness of silk exsertion.<br>
Vigor may relate to the ability of a plant to grow rapidly during early development<br>
and to the successful establishment, after germination, of a well-developed root<br>
system and a well-developed photosynthetic apparatus. Vigor may be determined<br>
with reference to different genotypes under similar environmental conditions, or<br>
with reference to the same or different genotypes under different environmental<br>
conditions.<br>
Accordingly, the present invention further provides plants having modulated<br>
shoot and/or leaf development when compared to a control plant. In some<br>
embodiments, the plant of the invention has an increased level/activity of the IPT<br>
polypeptide of the invention. In other embodiments, the plant of the invention has<br>
a decreased level/activity of the IPT polypeptide of the invention.<br>
VI. Modulating Reproductive Tissue Development<br>
Methods for modulating reproductive tissue development are provided. In<br>
one embodiment, methods are provided to modulate floral development in a plant.<br>
By "modulating floral development" is intended any alteration in a structure of a<br>
plant's reproductive tissue as compared to a control plant or plant part.<br>
"Modulating floral development" further includes any alteration in the timing of the<br>
development of a plant's reproductive tissue (i.e., delayed or accelerated floral<br>
development) when compared to a control plant or plant part. Macroscopic<br>
alterations may include changes in size, shape, number, or location of<br>
reproductive organs, the developmental time period during which these structures<br>
form, or the ability to maintain or proceed through the flowering process in times of<br>
environmental stress. Microscopic alterations may include changes to the types or<br>
shapes of cells that make up the reproductive organs.<br>
The method for modulating floral development in a plant comprises<br>
modulating (either increasing or decreasing) the level and/or activity of the IPT<br>
polypeptide in a plant. In one method, an IPT sequence of the invention is<br>
provided. An IPT nucleotide sequence can be provided by introducing into the<br>
plant a polynucleotide comprising an IPT nucleotide sequence of the invention,<br>
expressing the IPT sequence, and thereby modifying floral development. In some<br>
embodiments, the IPT nucleotide construct introduced into the plant is stably<br>
incorporated into the genome of the plant.<br>
As discussed above, one of skill will recognize the appropriate promoter to<br>
use to modulate floral development in the plant. Exemplary promoters for this<br>
embodiment include constitutive promoters, inducible promoters, shoot-preferred<br>
promoters, and inflorescence-preferred promoters (including developing-femaleinflorescence-<br>
preferred promoters), including those listed elsewhere herein.<br>
In specific methods, floral development is modulated by increasing the level<br>
and/or activity of the IPT sequence of the invention. Such methods can comprise<br>
introducing an IPT nucleotide sequence into the plant and increasing the activity of<br>
the IPT polypeptide. In some methods, the IPT nucleotide construct introduced<br>
into the plant is stably incorporated into the genome of the plant. An increase in<br>
the level and/or activity of the IPT sequences can result in one or more alterations<br>
in floral development including, but not limited to, accelerated flowering, increased<br>
number of flowers, and improved seed set when compared to a control plant. In<br>
addition, an increase in the level or activity of the IPT sequences can result in the<br>
prevention of flower senescence and an alteration in embryo number per kernel.<br>
See, Young et al. (2004) Plant J. 38:910-22. Methods for measuring such<br>
developmental alterations in floral development are known in the art. See, for<br>
example, Mouradov et al. (2002) The Plant Cell S111-S130, herein incorporated<br>
by reference.<br>
In other methods, floral development is modulated by decreasing the level<br>
and/or activity of the IPT sequence of the invention. A decrease in the level and/or<br>
"activity of the IPT sequence""can result in kernel abortion and infertile female<br>
inflorescence, Inducing delayed flowering or inhibiting flowering can be used to<br>
enhance yield in forage crops such as alfalfa.<br>
Accordingly, the present invention further provides plants having modulated<br>
floral development when compared to the floral development of a control plant.<br>
Compositions include plants having a decreased level/activity of the IPT<br>
polypeptide of the invention and having an altered floral development.<br>
Compositions also include plants having an increased level/activity of the IPT<br>
polypeptide of the invention wherein the plant maintains or proceeds through the<br>
flowering process in times of stress.<br>
VII. Modulating the Stress Tolerance of a Plant<br>
Methods are provided for the use of the IPT sequences of the invention to<br>
modify the tolerance of a plant to abiotic stress. Increases in the growth of<br>
seedlings or early vigor is often associated with an increase in stress tolerance.<br>
For example, faster development of seedlings, including the root system of<br>
seedlings upon germination, is critical for survival particularly under adverse<br>
conditions such as drought. Promoters that can be used in this method are<br>
described elsewhere herein, including low-level constitutive, inducible, or rootpreferred<br>
promoters, such as root-preferred promoters derived from ZmlPT4 and<br>
ZmlPTS regulatory sequences. Accordingly, in one method of the invention, a<br>
plant's tolerance to stress is increased or maintained when compared to a control<br>
plant by decreasing the level of IPT activity in the germinating seedling. In other<br>
methods, an IPT nucleotide sequence is provided by introducing into the plant a<br>
polynucleotide comprising a IPT nucleotide sequence of the invention, expressing<br>
the IPT sequence, and thereby increasing the plant's tolerance to stress. In other<br>
embodiments, the IPT nucleotide construct introduced into the plant is stably<br>
incorporated into the genome of the plant.<br>
Methods are also provided to increase or maintain seed set during abiotic<br>
stress episodes. During periods of stress (i.e., drought, salt, heavy metals,<br>
temperature, etc.) embryo development is often aborted. In maize, halted embryo<br>
development results in aborted kernels on the ear (Cheikh and Jones (1994) Plant<br>
Physiol, 106:45-51; Dietrich et al. (1995) Plant Physiol Biochem 33:327-336).<br>
Preventing this kernel loss will maintain yield. Accordingly, methods are provided<br>
to increase the stress resistance in a plant (e.g., during flowering and seed<br>
development). Increasing expression of the IPT sequence of the invention can<br>
also modulate floral development during periods of stress, and thus methods are<br>
provided to maintain or improve the flowering process in plants under stress. The<br>
method comprises increasing the level and/or activity of the IPT sequence of the<br>
invention. In one method, an IPT nucleotide sequence is introduced into the plant<br>
and the level and/or activity of the IPT polypeptide is increased, thereby<br>
maintaining or improving the tolerance of the plant under stress conditions. In<br>
other methods, the IPT nucleotide construct introduced into the plant is stably<br>
incorporated into the genome of the plant. See, for example, WO 00/63401.<br>
Significant yield instability can occur as a result of unfavorable<br>
environments during the lag phase of seed development. During this period,<br>
seeds undergo dramatic changes in ultra structure, biochemistry, and sensitivity to<br>
environmental perturbation, yet demonstrate little change in dry mass<br>
accumulation. Two important events that occur during the lag phase are initiation<br>
and division of endosperm cells and amyloplasts (which are the sites for starch<br>
deposition). It has been demonstrated that during the lag phase (around 10-12<br>
days after pollination (DAP) in maize) a dramatic increase in cytokinin<br>
concentration immediately precedes maximum rates of endosperm cell division<br>
and amyloplast formation, indicating that this hormone plays a central role in these<br>
processes and in what is called the 'sink strength* of the developing seed.<br>
Cytokinins have been demonstrated to play an important role in establishing seed<br>
size, decreasing tip kernel abortion, and increasing seed set during unfavorable<br>
environmental conditions. For example, elevated temperatures affect seed<br>
formation. Elevated temperatures can inhibit the accumulation of cytokinin,<br>
decrease endosperm ceil division and amyloplast number, and as a consequence,<br>
increase kernel abortion.<br>
Kernel sink capacity in maize is principally a function of the number of<br>
endosperm cells and starch granules established during the first 6 to 12 DAP. The<br>
final number of endosperm cells and amyloplasts formed is highly correlated with<br>
final kernel weight. (Capitanio et al., 1983; Reddy and Daynard, 1983; Jones et<br>
al., 1985, 1996; Engelen-Eigles et al., 2000). Hormones, especially cytokinins,<br>
have been shown to stimulate cell division, plastid initiation and other processes<br>
important in the establishment of kernel sink capacity (Davies, 1987). Cytokinin<br>
levels could for example be manipulated using the ZmlPT2 promoter to drive the<br>
expression of the Agrobacterium IPT gene. Similarly, endosperm- and/or pedicel-<br>
preferred promoters could be used to increase the level and/or duration of<br>
expression of ZmlPT2, which would result in an increase of cytokinin levels which<br>
would in turn increase sink strength and kernel yield. Capitano, R., Gentinetta, E.<br>
and Motto, M. (1983). Grain weight and its components in maize inbred lines.<br>
Maydica 23: 365-379. Jones, R. J., Roessler, J. and Ouattar, S. (1985). Thermal<br>
environment during endosperm cell division in maize: effects on number of<br>
endosperm cells and starch granules. Crop Science 25:830-834. Jones, R.J.,<br>
Schreiber, B.M.N. and Roessler, J. (1996). Kernel sink strength capacity in maize:<br>
Genotypic and maternal regulation. Crop Science 36:301-306. Davies, G.C.<br>
(1987).The plant hormones: their nature, occurrences and function. P 1-12. In P.J.<br>
Davies and M. Nijhoff (ed.). Plant hormones and their role in plant growth and<br>
development. Dordrecht, the Netherlands. Engelen-Eigles G., Jones, R. J. and<br>
Phillips R. L. (2000). DNA endoreduplication in maize endosperm cells: the effect<br>
of exposure to short-term high temperature. Plant, Cell and Environment 23: 657-<br>
663.<br>
Methods are therefore provided to increase the activity and/or level of IPT<br>
polypeptides in the developing inflorescence, thereby elevating cytokinin levels<br>
and allowing developing seed to achieve their full genetic potential for size,<br>
minimize seed abortion, and buffer seed set during unfavorable environments.<br>
The methods further allow the plant to maintain and/or improve the flowering<br>
process during unfavorable environments.<br>
In this embodiment, a variety of promoters could be used to direct the<br>
expression of a sequence capable of increasing the level and/or activity of the IPT<br>
polypeptide, including but not limited to, constitutive promoters, seed-preferred<br>
promoters, developing seed or kernel promoters, meristem-preferred promoters,<br>
stress-induced promoters, and inflorescence-preferred (such as developing female<br>
inflorescence promoters). In one method, a promoter that is stress insensitive and<br>
is expressed in a tissue of the developing seed during the lag phase of<br>
development is used. By "insensitive to stress" is intended that the expression<br>
level of a sequence operably linked to the promoter is not altered or only minimally<br>
altered under stress conditions. By "lag phase" promoter is intended a promoter<br>
that is active in the lag phase of seed development. A description of this<br>
developmental phase is found elsewhere herein. By "developing seed-preferred"<br>
is intended a promoter that allows for enhanced IPT expression within a<br>
developing seed. Such promoters that are stress insensitive and are expressed in<br>
a tissue of the developing seecl during the lag phase of development are known in<br>
the art and include Zag2.1 (Theissen et a/. (1995) Gene 156:155-166, Genbank<br>
Accession No. X80206), and mzE40 (Zm40) (U.S. Patent No. 6,403,862 and<br>
WO01/2178).<br>
An expression construct may further comprise nucleotide sequences<br>
encoding peptide signal sequences in order to effect changes in cytokinin level<br>
and/or activity in the mitochondria or chloroplasts. See, for example, Neupert<br>
(1997) Annual Rev, Biochem. 66:863-917; Glaser et al. (1998) Plant Molecular<br>
Biology 38:311-338; Dubyetal. (2001) The Plant J 27(6):539-549.<br>
Methods to assay for an increase in seed set during abiotic stress are<br>
known in the art. For example, plants having the increased IPT activity can be<br>
monitored under various stress conditions and compared to control plants. For<br>
instance, the plant having the increased cytokinin synthesis activity can be<br>
subjected to various degrees of stress during flowering and seed set. Under<br>
identical conditions, the genetically modified plant having the increased cytokinin<br>
synthesis activity will have a higher number of developing kernels than a control<br>
plant.<br>
Accordingly, the present invention further provides plants having increased<br>
yield or a maintained yield and/or an increased or maintained flowering process<br>
during periods of abiotic stress (drought, salt, heavy metals, temperature<br>
extremes, etc.). In some embodiments, the plants having an increased or<br>
maintained yield during abiotic stress have an increased level/activity of the IPT<br>
polypeptide of the invention. In some embodiments, the plant comprises an IPT<br>
nucleotide sequence of the invention operably linked to a promoter that drives<br>
expression in the plant cell. In some embodiments, such plants have stably<br>
incorporated into their genome a nucleic acid molecule comprising an IPT<br>
nucleotide sequence of the invention operably linked to a promoter that drives<br>
expression in the plant cell.<br>
VIII. Methods of Use for IPT promoter Polynucleotides<br>
The polynucleotides compn'sing the IPT promoters disclosed in the present<br>
invention, as well as variants and fragments thereof, are useful in the genetic<br>
manipulation of any host cell, preferably plant cell, when assembled with a DNA<br>
construct such that the promoter sequence is operably linked to a nucleotide<br>
sequence comprising a polynucleotide of interest. In this manner, the IPT<br>
"promoter polynucleoWes" of "the invention are provided in expression cassettes<br>
along with a heterologous polynucleotide sequence of interest for expression in<br>
the host cell of interest. As discussed in Example 2 below, the IPT promoter<br>
sequences of the invention are expressed in a variety of tissues and thus the<br>
promoter sequences can find use in regulating the temporal and/or the spatial<br>
expression of polynucleotides of interest.<br>
Synthetic hybrid promoter regions are known in the art. Such regions<br>
comprise upstream promoter elements of one polynucleotide operably linked to<br>
the promoter element of another polynucleotide. In an embodiment of the<br>
invention, heterologous sequence expression is controlled by a synthetic hybrid<br>
promoter comprising the IPT promoter sequences of the Invention, or a variant or<br>
fragment thereof, operably linked to upstream promoter element(s) from a<br>
heterologous promoter. Upstream promoter elements that are involved in the<br>
plant defense system have been identified and may be used to generate a<br>
synthetic promoter. See, for example, Rushton etal. (1998) Curr. Opin. Plant Biol.<br>
7:311-315. Alternatively, a synthetic IPT promoter sequence may comprise<br>
duplications of the upstream promoter elements found within the IPT promoter<br>
sequences.<br>
It is recognized that a promoter sequence of the invention may be used with<br>
its native IPT coding sequence. A DNA construct comprising an IPT promoter<br>
operably linked with its native IPT gene may be used to transform any plant of<br>
interest to bring about a desired phenotypic change, such as modulating cytokinin<br>
levels, modulating root, shoot, leaf, floral, and embryo development, stress<br>
tolerance, and any other phenotype described elsewhere herein.<br>
The promoter nucleotide sequences and methods disclosed herein are<br>
useful in regulating expression of any heterologous nucleotide sequence in a host<br>
plant in order to vary the phenotype of a plant. Various changes in phenotype are<br>
of interest including modifying the fatty acid composition in a plant, altering the<br>
amino acid content of a plant, altering a plant's pathogen defense mechanism, and<br>
the like. These results can be achieved by providing expression of heterologous<br>
products or increased expression of endogenous products in plants. Alternatively,<br>
the results can be achieved by providing for a reduction of expression of one or<br>
more endogenous products, particularly enzymes or cofactors in the plant. These<br>
changes result in a change in phenotype of the transformed plant.<br>
Genes OT interest are reflective of the commercial markets and interests of<br>
those involved in the development of the crop. Crops and markets of interest"<br>
change, and as developing nations open up world markets, new crops and<br>
technologies will emerge also. In addition, as our understanding of agronomic<br>
traits and characteristics such as yield and heterosis increase, the choice of genes<br>
for transformation will change accordingly. General categories of genes of interest<br>
include, for example, those genes involved in information, such as zinc fingers,<br>
those involved in communication, such as kinases, and those involved in<br>
housekeeping, such as heat shock proteins. More specific categories of<br>
transgenes, for example, include genes encoding important traits for agronomics,<br>
insect resistance, disease resistance, herbicide resistance, sterility, grain<br>
characteristics, and commercial products. Genes of interest include, generally,<br>
those involved in oil, starch, carbohydrate, or nutrient metabolism as well as those<br>
affecting kernel size, sucrose loading, and the like,<br>
In one embodiment, sequences of interest improve plant growth and/or crop<br>
yields. In more specific embodiments, expression of the nucleotide sequence of<br>
interest improves the plant's response to stress induced under high density growth<br>
conditions. For example, sequences of interest include agronomically important<br>
genes that result in improved primary or lateral root systems. Such genes include,<br>
but are not limited to, nutrient/water transporters and growth inducers. Examples<br>
of such genes, include but are not limited to, maize plasma membrane H+-ATPase<br>
(MHA2) (Frias Qt al. (1996) Plant Cell 8:1533-44); AKT1, a component of the<br>
potassium uptake apparatus in Arabidopisis, (Spalding et al. (1999) J Gen Physiol<br>
773:909-18); RML genes which activate cell division cycle in the root apical cells<br>
(Cheng et al. (1995) Plant Physiol 108:881); maize glutamine synthetase genes<br>
(Sukanya et al. (1994) Plant Mol Biol 26:1935-46) and hemoglobin (Duff et al.<br>
(1997) J. Biol. Chem 27:16749-16752, Arredondo-Peter et al. (1997) Plant<br>
Physiol. 115:1259-1266; Arredondo-Peter et al. (1997) Plant Physiol 174:493-500<br>
and references sited therein). The sequence of interest may also be useful in<br>
expressing antisense nucleotide sequences of genes that negatively affect root<br>
development.<br>
Additional, agronomically important traits such as oil, starch, and protein<br>
content can be genetically altered In addition to using traditional breeding<br>
methods. Modifications include increasing content of oleic acid, changing the<br>
proportions of saturated and unsaturated oils, increasing levels of lysine and<br>
sulfur, providing essential amino acids, and also modification of starch.<br>
Hordothionin protein modifications are described in U.S. Patent Nos. 5,703,049,<br>
5,885,801, 5,885,802, and 5,990,389, herein incorporated by reference. Another<br>
example is lysine and/or sulfur rich seed protein encoded by the soybean 2S<br>
albumin described in U.S. Patent No. 5,850,016, and the chymotrypsin inhibitor<br>
from barley, described in Williamson et al. (1987) Eur. J. Biochem. 165:99-106,<br>
the disclosures of which are herein incorporated by reference.<br>
Derivatives of the coding sequences can be made by site-directed<br>
mutagenesis to increase the level of preselected amino acids in the encoded<br>
polypeptide. For example, the gene encoding the barley high lysine polypeptide<br>
(BHL) is derived from barley chymotrypsin inhibitor, U.S. Application Serial No.<br>
08/740,682, filed November 1, 1996, and WO 98/20133, the disclosures of which<br>
are herein incorporated by reference. Other proteins include methionine-rich plant<br>
proteins such as from sunflower seed (Lilley et al. (1989) Proceedings of the<br>
World Congress on Vegetable Protein Utilization in Human Foods and Animal<br>
Feedstuffs, ed. Applewhite (American Oil Chemists Society, Champaign, Illinois),<br>
pp. 497-502; herein incorporated by reference); corn (Pedersen et al. (1986) J.<br>
Blot, Chem. 261:6279; Kirihara et al. (1988) Gene 71:359; both of which are herein<br>
incorporated by reference); and rice (Musumura et al. (1989) Plant Mol. Biol.<br>
12:123, herein incorporated by reference). Other agronomically important genes<br>
encode latex, Floury 2, growth factors, seed storage factors, and transcription<br>
factors.<br>
Insect resistance genes may encode resistance to pests that have great<br>
yield drag such as rootworm, cutworm, European Corn Borer, and the like. Such<br>
genes include, for example, Bacillus thuringiensis toxic protein genes (U.S. Patent<br>
Nos. 5,366,892; 5,747,450; 5,736,514; 5,723,756; 5,593,881; and Geiser et al.<br>
(1986) Gene 48:109); and the like.<br>
Genes encoding disease resistance traits include detoxification genes, such<br>
as against fumonosin (U.S. Patent No. 5,792,931); avirulence (avr) and disease<br>
resistance (R) genes (Jones et al. (1994) Science 266:789; Martin et al. (1993)<br>
Science 262:1432; and Mindrinos et al. (1994) Cell 78:1089); and the like.<br>
Herbicide resistance traits may include genes coding for resistance to<br>
herbicides that act to inhibit the action of acetolactate synthase (ALS), in particular<br>
the sulfonylurea-type herbicides (e.g., the acetolactate synthase (ALS) gene<br>
containing mutations leading to such resistance, in particular the S4 and/or Hra<br>
mufot'ions), genes coding for'resistance to herbicides that act to inhibit action of<br>
glutamine synthase, such as phosphinothricin or basta (e.g., the bar gene), or<br>
other such genes known in the art. The bar gene encodes resistance to the<br>
herbicide basta, the nptll gene encodes resistance to the antibiotics kanamycin<br>
and geneticin, and the ALS-gene mutants encode resistance to the herbicide<br>
chlorsulfuron.<br>
Sterility genes can also be encoded in an expression cassette and provide<br>
an alternative to physical emasculation. Examples of genes used in such ways<br>
include male tissue-preferred genes and genes with male sterility phenotypes<br>
such as DAM, described in U.S. Patent No. 5,750,868; 5,689,051; and 6,281,348.<br>
Other genes include kinases and those encoding compounds toxic to either male<br>
or female gametophytic development.<br>
The quality of grain is reflected in traits such as levels and types of oils,<br>
saturated and unsaturated, quality and quantity of essential amino acids, and<br>
levels of cellulose. In corn, modified hordothionin proteins are described in U.S.<br>
Patent Nos. 5,703,049, 5,885,801, 5,885,802, and 5,990,389.<br>
Commercial traits can also be encoded on a gene or genes that could<br>
increase for example, starch for ethanol production, or provide expression of<br>
proteins. Another important commercial use of transformed plants is the<br>
production of polymers and bioplastics such as described in U.S. Patent No.<br>
5,602,321. Genes such as p-Ketothiolase, PHBase (polyhydroxyburyrate<br>
synthase), and acetoacetyl-CoA reductase (see Schubert et a/. (1988) J. Bacteriol.<br>
170:5837-5847) facilitate expression of polyhyroxyalkanoates (PHAs).<br>
Exogenous products include plant enzymes and products as well as those<br>
from other sources including prokaryotes and other eukaryotes. Such products<br>
include enzymes, cofactors, hormones, and the like. The level of proteins,<br>
particularly modified proteins having improved amino acid distribution to improve<br>
the nutrient value of the plant, can be increased. This is achieved by the<br>
expression of such proteins having enhanced amino acid content.<br>
IX. Antibody Creation and Use<br>
Antibodies can be raised to a protein of the present invention, including<br>
variants and fragments thereof, in both their naturally-occurring and recornbinant<br>
forms. Many methods of making antibodies are known to persons of skill. A<br>
variety of analytic methods are available to generate a hydrophilicity profile of a<br>
protein of the present invention" Such methods can be used to guide the artisan in<br>
the selection of peptides of the present invention for use in the generation or<br>
selection of antibodies which are specifically reactive, under immunogenic<br>
conditions, to a protein of the present invention. See, e.g., J. Janin, Nature,<br>
277(1979) 491-492; Wolfenden, et al., biochemistry 20(1981) 849-855; Kyte and<br>
Doolite, J. MolBiol. 157(1982) 105-132; Rose, era/., Science 229(1985) 834-8S&amp;.<br>
The antibodies can be used to screen expression libraries for particular expression<br>
products such as normal or abnormal protein, or altered levels of the same, which<br>
may be useful for detecting or diagnosing various conditions related to the<br>
presence of the respective antigens. Assays indicating high levels of an IPT<br>
protein of the invention, for example, could be useful in detecting plants, or<br>
specific plant parts, with elevated cytokinin levels. Usually the antibodies in such<br>
a procedure are labeled with a moiety which allows easy detection of presence of<br>
antigen/antibody binding.<br>
The following discussion is presented as a general overview of the<br>
techniques available; however, one of skill will recognize that many variations<br>
upon the following methods are known.<br>
A number of immunogens are used to produce antibodies specifically<br>
reactive with a protein of the present invention. Polypeptides encoded by isolated<br>
recombinant, synthetic, or native polynucleotides of the present invention are the<br>
preferred antigens for the production of monoclonal or polyclonal antibodies.<br>
Polypeptides of the present invention are optionally denatured, and optionally<br>
reduced, prior to injection into an animal capable of producing antibodies. Either<br>
monoclonal or polyclonal antibodies can be generated for subsequent use in<br>
immunoassays to measure the presence and quantity of the protein of the present<br>
invention. Methods of producing polyclonal antibodies are known to those of skill<br>
in the art. In brief, an antigen, preferably a purified protein, a protein coupled to an<br>
appropriate carrier (e.g., GST, keyhole limpet hemanocyanin, etc.), or a protein<br>
incorporated into an immunization vector such as a recombinant vaccinia virus<br>
(see, U.S. Patent No. 4,722,848) is mixed with an adjuvant and animals are<br>
immunized with the mixture. The animal's immune response to the immunogen<br>
preparation is monitored by taking test bleeds and determining the titer of<br>
reactivity to the protein of interest. When appropriately high titers of antibody to<br>
the immunogen are obtained, blood is collected from the animal and antisera are<br>
prepared. Specific monoclonal and polyclonal antibodies will usually have an<br>
antibody binding site with an affinity constant for its cognate monovalent antigen at<br>
least between 106-107, usually at least 108, 109, 1010, and up to about 1011<br>
liters/mole. Further fractionation of the antisera to enrich for antibodies reactive to<br>
the protein is performed where desired (See, e.g., Coligan, Current Protocols in<br>
Immunology, Wiley/Greene, NY (1991); and Harlow and Lane, Antibodies: A<br>
Laboratory Manual, Cold Spring Harbor Press, NY (1989)).<br>
Antibodies, including binding fragments and single chain recombinant<br>
versions thereof, against predetermined fragments of a protein of the present<br>
invention are raised by immunizing animals, e.g., with conjugates of the fragments<br>
with carrier proteins as described above. Typically, the immunogen of interest is a<br>
protein of at least about 5 amino acids, more typically the protein is 10 amino<br>
acids in length, often 15 to 20 amino acids in length, and may be longer. The<br>
peptides are typically coupled to a carrier protein (e.g., as a fusion protein), or are<br>
recombinantly expressed in an immunization vector. Antigenic determinants on<br>
peptides to which antibodies bind are typically 3 to 10 amino acids in length.<br>
Monoclonal antibodies are prepared from hybrid cells secreting the desired<br>
antibody. Monoclonal antibodies are screened for binding to a protein from which<br>
the antigen was derived. Description of techniques for preparing such monoclonal<br>
antibodies are found in, e.g., Basic and Clinical Immunology, 4th ed., Stites et a/.,<br>
Eds., Lange Medical Publications, Los Altos, CA, and references cited therein;<br>
Harlowand Lane, Supra1, Goding, Monoclonal Antibodies: Principles and Practice,<br>
2nd ed., Academic Press, New York, NY (1986); and Kohler and Milstein, Nature<br>
256: 495-497 (1975). Summarized briefly, this method proceeds by injecting an<br>
animal with an antigen comprising a protein of the present invention. The animal<br>
is then sacrificed and cells taken from its spleen, which are fused with myeloma<br>
cells. The result is a hybrid cell or "hybridoma" that is capable of reproducing in<br>
vitro. The population of hybridomas is then screened to isolate individual clones,<br>
each of which secretes a single antibody species to the antigen. In this manner,<br>
the individual antibody species obtained are the products of immortalized and<br>
cloned single B cells generated by the animal in response to a specific site<br>
recognized on the antigenic substance.<br>
Other suitable techniques involve selection of libraries of recombinant<br>
antibodies in phage or similar vectors (see, e.g., Huse et a/., Science 246: 1275-<br>
1281 (1989); and Ward, et a/., Nature 341: 544-546 (1989); and Vaughan et a/,,<br>
Nature Biotechnology, 14: 309-314 (1996)). Also, recombinant immunoglobulins<br>
may be produced. See, Cabllly, U.S. Patent No. 4,816,567; and Queen et a/.,<br>
Proc. Nat'lAcad. Sci. 86: 10029-10033 (1989).<br>
Antibodies to the polypeptides of the invention are also used for affinity<br>
chromatography in isolating proteins of the present invention. Columns are<br>
prepared, e.g., with the antibodies linked to a solid support, e.g., particles, such as<br>
agarose, SEPHADEX, or the like, where a cell lysate is passed through the<br>
column, washed, and treated with increasing concentrations of a mild denaturant,<br>
whereby purified proteins are released.<br>
Frequently, the proteins and antibodies of the present invention will be<br>
labeled by joining, either covalently or non-covalently, a substance which provides<br>
for a detectable signal. A wide variety of labels and conjugation techniques are<br>
known and are reported extensively in both the scientific and patent literature.<br>
Suitable labels include radionucleotides, enzymes, substrates, cofactors,<br>
inhibitors, fluorescent moieties, chemiluminescent moieties, magnetic particles,<br>
and the like.<br>
Protein Immunoassavs<br>
Means of detecting the proteins of the present invention are not critical<br>
aspects of the present invention. In certain examples, the proteins are detected<br>
and/or quantified using any of a number of well-recognized immunological binding<br>
assays (see, e.g., U.S. Patents 4,366,241; 4,376,110; 4,517,288; and 4,837,168).<br>
For a genera! review of immunoassays, see also Methods in Cell Biology, Vol. 37;<br>
Antibodies in Cell Biology, Asai, Ed., Academic Press, Inc. New York (1993);<br>
Basic and Clinical Immunology 7th Edition, Stites &amp; Terr, Eds. (1991). Moreover,<br>
the immunoassays of the present invention can be performed in any of several<br>
configurations, e.g., those reviewed in Enzyme Immunoassay, Maggio, Ed., CRC<br>
Press, Boca Raton, Florida (1980); Tijan, Practice and Theory of Enzyme<br>
Immunoassays, Laboratory Techniques in Biochemistry and Molecular Biology,<br>
Elsevier Science Publishers B.V., Amsterdam (1985); Harlow and Lane, supra;<br>
Immunoassay: A Practical Guide, Chan, Ed., Academic Press, Orlando, FL<br>
(1987); Principles and Practice of Immunoassaysm, Price and Newman Eds.,<br>
Stockton Press. NY (1991); and Non-isotopic Immunoassays, Ngo, Ed., Plenum<br>
Press, NY (1988).<br>
Immunological binding assays (or immunoassays) typically utilize a<br>
"capture agent" to specifically bind to and often immobilize the analyte (in this<br>
case, a protein of the present invention). The capture agent is a moiety that<br>
specifically binds to the analyte. In certain embodiments, the capture agent is an<br>
antibody that specifically binds a protein of the present invention. The antibody<br>
may be produced by any of a number of means known to those of skill in the art as<br>
described herein.<br>
Immunoassays also often utilize a labeling agent to specifically bind to and<br>
label the binding complex formed by the capture agent and the analyte. The<br>
labeling agent may itself be one of the moieties comprising the antibody/analyte<br>
complex. Thus, the labeling agent may be a labeled protein of the present<br>
invention or a labeled antibody specifically reactive to a protein of the present<br>
invention. Alternatively, the labeling agent may be a third moiety, such as another<br>
antibody, that specifically binds to the antibody/protein complex.<br>
Throughout the assays, incubation and/or washing steps may be required<br>
after each combination of reagents. Incubation steps can vary from about 5<br>
seconds to several hours, often from about 5 minutes to about 24 hours.<br>
However, the incubation time will depend upon the assay format, analyte, volume<br>
of solution, concentrations, and the like. Usually, the assays will be carried out at<br>
ambient temperature, although they can be conducted over a range of<br>
temperatures, such as 10°C to 40°C.<br>
While the details of the immunoassays of the present invention may vary<br>
with the particular format employed, the method of detecting a protein of the<br>
present invention in a biological sample generally comprises the steps of<br>
contacting the biological sample with an antibody which specifically reacts, under<br>
immunologically reactive conditions, to a protein of the present invention. The<br>
antibody is allowed to bind to the protein under immunologically reactive<br>
conditions, and the presence of the bound antibody is detected directly or<br>
indirectly.<br>
A. Non-Competitive Assay Formats<br>
Immunoassays for detecting proteins of the present invention include<br>
competitive and noncompetitive formats. Noncompetitive immunoassays are<br>
assays in which the amount of captured analyte (i.e., a protein of the present<br>
invention) is directly measured. In one example, the "sandwich" assay, the<br>
capture agent (e.g., an antibody specifically reactive, under Immunoreactive<br>
conditions, to a protein of the present invention) can be bound directly to a solid<br>
suosiraie wnere u is immooiiizea. These immobilized antibodies then capture the<br>
protein present in the test sample. The protein thus immobilized is then bound by<br>
a labeling agent, such as a second antibody bearing a label. Alternatively, the<br>
second antibody may lack a label, but it may, in turn, be bound by a labeled third<br>
antibody specific to antibodies of the species from which the second antibody is<br>
derived. The second antibody can be modified with a detectable moiety, such as<br>
biotin, to which a third labeled molecule can specifically bind, such as enzymelabeled<br>
streptavidin<br>
B- Competitive Assay Formats<br>
In competitive assays, the amount of analyte present in the sample is<br>
measured indirectly by measuring the amount of an added (exogenous) analyte<br>
(e.g., a protein of the present invention) displaced (or competed away) from a<br>
capture agent (e.g., an antibody specifically reactive, under immunoreactive<br>
conditions, to the protein) by the analyte present in the sample. In one competitive<br>
assay, a known amount of analyte is added to the sample and the sample is then<br>
contacted with a capture agent that specifically binds a protein of the present<br>
invention. The amount of protein bound to the capture agent is inversely<br>
proportional to the concentration of analyte present in the sample.<br>
In one embodiment, the antibody is immobilized on a solid substrate. The<br>
amount of protein bound to the antibody may be determined either by measuring<br>
the amount of protein present in a protein/antibody complex, or alternatively by<br>
measuring the amount of remaining uncomplexed protein. The amount of protein<br>
may be detected by providing a labeled protein.<br>
A hapten inhibition assay is another competitive assay. In this assay a<br>
known analyte, such as a protein of the present invention, is immobilized on a<br>
solid substrate. A known amount of antibody specifically reactive, under<br>
immunoreactive conditions, to the protein is added to the sample, and the sample<br>
is then contacted with the immobilized protein. In this case, the amount of<br>
antibody bound to the immobilized protein is inversely proportional to the amount<br>
of protein present in the sample. Again, the amount of immobilized antibody may<br>
be determined by detecting either the immobilized fraction of antibody or the<br>
fraction of the antibody that remains in solution. Detection may be direct, where<br>
the antibody is labeled, or indirect, by the subsequent addition of a labeled moiety<br>
that specifically binds to the antibody, as described above.<br>
C. Generation of pooled antisera for use in immunoassavs<br>
A protein that specifically binds to, or that is specifically immunoreactive<br>
with, an antibody generated against a defined antigen is determined in an<br>
immunoassay. The immunoassay uses a polyclonal antiserum which is raised to<br>
a polypeptide of the present invention (i.e., the antigenic polypeptide). This<br>
antiserum is selected to have low cross-reactivity against other proteins, and any<br>
such cross-reactivity is removed by immunoabsorbtion prior to use in the<br>
immunoassay (e.g., by immunosorbtion of the antisera with a protein of different<br>
substrate specificity (e.g., a different enzyme) and/or a protein with the same<br>
substrate specificity but of a different form).<br>
In order to produce antisera for use in an immunoassay, a polypeptide of<br>
the present invention is isolated as described herein. For example, recombinant<br>
protein can be produced in a mammalian or other eukaryotic cell line. An inbred<br>
strain of mice is immunized with the protein using a standard adjuvant, such as<br>
Freund's adjuvant, and a standard mouse immunization protocol (see Harlow and<br>
Lane, supra). Alternatively, a synthetic polypeptide derived from the sequences<br>
disclosed herein and conjugated to a carrier protein is used as an immunogen.<br>
Polyclonal sera are collected and titered against the immunogenic polypeptide in<br>
an immunoassay, for example, a solid phase immunoassay with the immunogen<br>
immobilized on a solid support. Polyclonal antisera with a titer of 104 or greater<br>
are selected and tested for their cross reactivity against polypeptides of different<br>
forms or substrate specificity, using a competitive binding immunoassay such as<br>
the one described in Harlow and Lane, supra, at pages 570-573. Preferably, two<br>
or more distinct forms of polypeptides are used in this determination. These<br>
distinct types of polypeptides are used as competitors to identify antibodies which<br>
are specifically bound by the polypeptide being assayed for. The competitive<br>
polypeptides can be produced as recombinant proteins and isolated using<br>
standard molecular biology and protein chemistry techniques as described herein.<br>
Immunoassays in the competitive binding format are used for crossreactivity<br>
determinations. For example, the immunogenic polypeptide is<br>
immobilized to a solid support. Proteins added to the assay compete with the<br>
binding of the antisera to the immobilized antigen. The ability of the above<br>
proteins to compete with the binding of-the antisera to the immobilized protein is<br>
compared to the immunogenic polypeptide. The percent cross-reactivity for the<br>
above proteins is calculated, using standard methods. Those antisera with less<br>
than 10% cross-reactivity for a distinct form of a poiypeptide are selected and<br>
pooled. The cross-reacting antibodies are then removed from the pooled antisera<br>
by immunoabsorbtion with a distinct form of a poiypeptide.<br>
The immunoabsorbed and pooled antisera are then used in a competitive<br>
binding immunoassay as described herein to compare a second "target"<br>
poiypeptide to the immunogenic poiypeptide. In order to make this comparison,<br>
the two polypeptides are each assayed at a wide range of concentrations and the<br>
amount of each poiypeptide required to inhibit 50% of the binding of the antisera to<br>
the immobilized protein is determined using standard techniques. If the amount of<br>
the target poiypeptide required is less than twice the amount of the immunogenic<br>
poiypeptide that is required, then the target poiypeptide is said to specifically bind<br>
to an antibody generated to the immunogenic protein. As a final determination of<br>
specificity, the pooled antisera is fully immunosorbed with the immunogenic<br>
poiypeptide until no binding to the poiypeptide used in the immunosorbtion is<br>
detectable. The fully immunosorbed antisera is then tested for reactivity with the<br>
test poiypeptide. If no reactivity is observed, then the test poiypeptide is<br>
specifically bound by the antisera elicited by the immunogenic protein.<br>
D- Other Assay Formats<br>
In certain embodiments, Western blot (immunoblot) analysis is used to<br>
detect and quantify the presence of protein of the present invention in the sample.<br>
The technique generally comprises separating sample proteins by gel<br>
electrophoresis on the basis of molecular weight, transferring the separated<br>
proteins to a suitable solid support, (such as a nitrocellulose filter, a nylon filter, or<br>
derivatized nylon filter), and incubating the sample with the antibodies that<br>
specifically bind a protein of the present invention. The antibodies specifically bind<br>
to the protein on the solid support. These antibodies may be directly labeled, or<br>
may be subsequently detected using labeled antibodies (e.g., labeled sheep antimouse<br>
antibodies) that specifically bind to the antibodies.<br>
£• Quantification of Proteins.<br>
The proteins of the present invention may be detected and quantified by<br>
any of a number of means well known to those of skill in the art. These include<br>
analytic biochemical methods such as electrophoresis, capillary electrophoresis,<br>
high'" performance liquid chromatography (HPLC), thin layer chromatography<br>
(TLC), hyperdiffusion chromatography, and the like, and various immunological<br>
methods such as fluid or gel precipitin reactions, immunodiffusion (single or<br>
double), immunoelectrophoresis, radioimmunoassays (RIAs), enzyme-linked<br>
5 immunosorbent assays (ELISAs), immunofluorescent assays, and the like.<br>
F. Reduction pf Non-specific Binding<br>
One of skill will appreciate that It is often desirable to reduce non-specific<br>
binding in immunoassays and during analyte purification. Where the assay<br>
10 involves an antigen, antibody, or other capture agent immobilized on a solid<br>
substrate, it is desirable to minimize the amount of non-specific binding to the<br>
substrate. Means of reducing such non-specific binding are well known to those of<br>
skill in the art. Typically, this involves coating the substrate with a proteinaceous<br>
composition. In particular, protein compositions such as bovine serum albumin<br>
15 (BSA), nonfat powdered milk, and gelatin are widely used.<br>
G. Imiyiunoassav Labels<br>
The labeling agent can be, e.g., a monoclonal antibody, a polyclonal<br>
antibody, a binding protein or complex, or a polymer such as an affinity matrix,<br>
20 carbohydrate or lipid. Detectable labels suitable for use in the present invention<br>
include any composition detectable by spectroscopic, radioisotopic,<br>
photochemical, biochemical, immunochemical, electrical, optical or chemical<br>
means. Detection may proceed by any known method, such as immunoblotting,<br>
Western analysis, gel-mobility shift assays, fluorescent in situ hybridization<br>
25 analysis (FISH), tracking of radioactive or bioluminescent markers, nuclear<br>
magnetic resonance, electron paramagnetic resonance, stopped-flow<br>
spectroscopy, column chromatography, capillary electrophoresis, or other<br>
methods which track a molecule based upon an alteration in size and/or charge.<br>
The particular label or detectable group used in the assay is not a critical aspect of<br>
30 the invention. The detectable group can be any material having a detectable<br>
physical or chemical property, including magnetic beads, fluorescent dyes,<br>
radiolabels, enzymes, and colorimetric labels or colored glass or plastic beads, as<br>
discussed for nucleic acid labels, supra. The label may be coupled directly or<br>
indirectly to the desired component of the assay according to methods well known<br>
35 in the art. As indicated above, a wide variety of labels may be used, with the<br>
choice of label depending onlhe sensitivity required, ease of conjugation of the<br>
compound, stability requirements, available instrumentation, and disposal<br>
provisions. Means of detecting labels are well known to those of skill in the art.<br>
Non-radioactive labels are often attached by indirect means. Generally, a<br>
5 ligand molecule (e.g., biotin) is covalently bound to the molecule. The ligarid then<br>
binds to an anti-ligand (e.g., streptavidin) molecule which is either inherently<br>
detectable or covalently bound to a signal system, such as a detectable enzyme, a<br>
fluorescent compound, or a chemiluminescent compound. A number of ligands<br>
and anti-ligands can be used.<br>
10 The molecules can also be conjugated directly to signal-generating<br>
compounds, e.g., by conjugation with an enzyme or fluorophore. Enzymes of<br>
interest as labels will primarily be hydrolases, particularly phosphatases, esterases<br>
and glycosidases, or oxidoreductases, particularly peroxidases. Fluorescent<br>
compounds include fluorescein and its derivatives, rhodamine and its derivatives,<br>
15 dansyl, umbelliferone, etc. Chemiluminescent compounds include luciferin, and<br>
2,3-dihydrophthalazinediones, e.g., luminol. For a review of various labeling or<br>
signal-producing systems which may be used, see, U.S. Patent No. 4,391,904,<br>
which is incorporated herein by reference.<br>
Some assay formats do not require the use of labeled components. For<br>
20 instance, agglutination assays can be used to detect the presence of the target<br>
antibodies. In this case, antigen-coated particles are agglutinated by samples<br>
comprising the target antibodies. In this format, none of the components need be<br>
labeled and the presence of the target antibody is detected by simple visual<br>
inspection.<br>
25<br>
Assays for Compounds that Modulate Enzymatic Activity or Expression<br>
A catalytically active polypeptide of the present invention may be contacted<br>
with a compound in order to determine whether said compound binds to and/or<br>
modulates the enzymatic activity of such polypeptide. The polypeptide employed<br>
30 will have at least 20%, 30%, 40%, 50% 60%, 70% or 80% of the specific activity of<br>
the native, full-length enzyme of the present invention. Generally, the polypeptide<br>
will be present in a range sufficient to determine the effect of the compound,<br>
typically about 1 nM to 10 |*M. Likewise, the compound being tested will be<br>
present in a concentration of from about 1 nM to 10 jaM. Those of "skill will<br>
35 understand that such factors as enzyme concentration, ligand concentrations (i.e.,<br>
substrates, products, inhibitors, activators), pH, ionic strength, and temperature<br>
will be controlled so as to obtain useful kinetic data and determine the presence or<br>
absence of a compound that binds or modulates polypeptide activity. Methods of<br>
measuring enzyme kinetics are well known in the art. See, e.g., Segel,<br>
Biochemical Calculations, 2nd ed., John Wiley and Sons, New York (1976).<br>
Embodiments of the invention include the following:<br>
1. An isolated polypeptide comprising an amino acid sequence selected from<br>
the group consisting of:<br>
(a) an amino acid sequence comprising SEQ ID N0:2, 6, 9, 12, 15, 18,<br>
23, 27, 41, 43,46,49, 52, 54, 57, 59, 61, 63, 66, or 77;<br>
(b) an amino acid sequence comprising at least 85% sequence identity<br>
to SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49, 52, 54, 57,<br>
59, 61, 63, 66, or 77, wherein said polypeptide has cytokinin<br>
synthesis activity;<br>
(c) an amino acid sequence encoded by a nucleotide sequence that<br>
hybridizes under stringent conditions to the complement of SEQ ID<br>
NO:1, 3, 4, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28,<br>
40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70,<br>
71, 72, 73, 74, or 76, wherein said stringent conditions comprise<br>
hybridization in 50% formamide, 1 M NaCI, 1% SDS at 37°C, and a<br>
wash in 0.1X SSC at 60°C to 65°C, wherein said polypeptide retains<br>
cytokinin synthesis activity; and,<br>
(d) an amino acid sequence comprising at least 50 consecutive amino<br>
acids of SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49, 52,<br>
54, 57, 59, 61, 63, 66, or 77, wherein said polypeptide retains<br>
cytokinin synthesis activity.<br>
2. An isolated polynucleotide comprising a nucleotide sequence selected from<br>
the group consisting of:<br>
(a) a nucleotide sequence comprising SEQ ID N0:1, 3, 4, 5, 7, 8, 10,<br>
11,13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48,<br>
50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 7.3, 74, or 76;<br>
(b) a nucleotide sequence encoding an amino acid sequence<br>
comprising SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49,<br>
52, 54, 57, 59,61,63, 66, or 77;<br>
 (c) a nuclebtide sequence comprising at least 85% sequence identity to<br>
SEQ ID NO: 1, 3, 4, 5, 7, 8, 10,11, 13,14, 16,17, 19, 20, 21, 22, 24,<br>
26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65,<br>
69, 70, 71, 72, 73, 74, or 76, wherein said polynucleotide encodes a<br>
polypeptide having cytokinin synthesis activity;<br>
(d) a nucleotide sequence comprising at least 50 consecutive<br>
nucleotides of SEQ ID NO: 1, 3, 4, 5, 7, 8, 10, 11, 13,14, 16, 17, 19,<br>
20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58,<br>
60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76, or a complement<br>
thereof; and,<br>
(e) a nucleotide sequence that hybridizes under stringent conditions to<br>
the complement of a nucleotide sequence of a), wherein said<br>
stringent conditions comprise hybridization in 50% formamide, 1 M<br>
NaCI, 1% SDS at 37°G, and a wash in 0.1X SSC at 60°C to 65°C.<br>
An expression cassette comprising a polynucleotide of embodiment 2.<br>
The expression cassette of embodiment 3, wherein said polynucleotide is<br>
operably linked to a promoter that drives expression in a plant.<br>
A plant comprising a polynucleotide operably linked to a promoter that<br>
drives expression in the plant, wherein said polynucleotide comprises a<br>
nucleotide sequence selected from the group consisting of:<br>
(a) a nucleotide sequence comprising SEQ ID NO: 1, 3, 4, 5, 7, 8, 10,<br>
11, 13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48,<br>
50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73,74, or 76;<br>
(b) a nucieotide sequence encoding an amino acid sequence<br>
comprising SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49,<br>
52, 54, 57, 59, 61,63,66, or 77;<br>
(c) a nucleotide sequence comprising at least 85% sequence identity to<br>
SEQ ID N0:1, 3, 4, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24,<br>
26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65,<br>
69, 70, 71, 72, 73, 74, or 76, wherein said polynucleotide encodes a<br>
polypeptide having cytokinin synthesis activity;<br>
(d) a nucleotide sequence that hybridizes under stringent conditions to<br>
the complement of a nucleotide sequence of a), wherein said<br>
stringent conditions comprise hybridization in 50% formamide, 1 M<br>
NaCI, 1% SDS at 37°C, and a wash in 0.1X SSC at 60°C to 65°C,<br>
wherein said pblynucleotide encodes a polypeptide having cytokinin<br>
synthesis activity; and,<br>
(e) a nucleotide sequence comprising at least 50 consecutive<br>
nucleotidesofSEQIDNO:1,3, 4,5, 7,8, 10, 11, 13, 14, 16,17, 19,<br>
20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58,<br>
60, 62,64, 65, 69, 70, 71,72, 73, or 74 or a complement thereof.<br>
6. The plant of embodiment 5, wherein said plant has a modulated cytokinin<br>
level.<br>
7. The plant of embodiment 6, wherein said cytokinin level is modulated in a<br>
vegetative tissue, a reproductive tissue, or vegetative tissue and<br>
reproductive tissue.<br>
8. The plant of embodiment 6, wherein said cytokinin level is increased.<br>
9. The plant of embodiment 6, wherein said cytokinin level is decreased.<br>
10. The plant of embodiment 5, 6, 7, 8, or 9, wherein said promoter is a tissuepreferred<br>
promoter, a constitutive promoter, or an inducible promoter.<br>
11. The plant of embodiment 10, wherein said promoter is a root-preferred<br>
promoter, a leaf-preferred promoter, a shoot-preferred promoter, or an<br>
inflorescence-preferred promoter.<br>
12. The plant of embodiment 5, wherein said plant has modulated floral<br>
development.<br>
13. The plant of embodiment 5, wherein said plant has modulated root<br>
development.<br>
14. The plant of embodiment 13, wherein the modulated root development<br>
comprises at least one of an increase in root growth or an increase in the<br>
formation of lateral or adventitious roots.<br>
15. The plant of embodiment 5, wherein the plant has an altered shoot-to-root<br>
ratio.<br>
16. The plant of embodiment 5, wherein said plant has an increased seed size<br>
or an increased seed weight.<br>
17. The plant of embodiment 16, wherein the increase in seed size or seed<br>
weight comprises an increase in at least one of embryo size, embryo<br>
weight, cotyledon size, or cotyledon weight.<br>
18. The plant of embodiment 5, wherein vigor or biomass yield of said plant is<br>
increased.<br>
'19. The plant of emBoBimeht 5, wherein the stress tolerance of said plant is<br>
maintained or improved.<br>
20. The plant of embodiment 19, wherein the size of the plant is increased or<br>
maintained.<br>
21. The plant of embodiment 19, wherein tip kernel abortion is minimized.<br>
22. The plant of embodiment 19, wherein the seed set of said plant is increased<br>
or maintained.<br>
23. The plant of embodiment 19, 20, 21, or 22, wherein said promoter is stressinsensitive<br>
and is expressed in a tissue of the developing seed or related<br>
maternal tissue, at least during the lag phase of seed development.<br>
24. The plant of embodiment 5, wherein said plant has a decrease in shoot<br>
growth.<br>
25. The plant of embodiment 5, wherein said plant has a delayed senescence<br>
or an enhanced vegetative growth.<br>
26. The plant of any one of embodiments 5 to 9, 12 to 22, 24, and 25 wherein<br>
said polynucleotide is stably incorporated into the genome of the plant.<br>
27. A transformed seed of the plant of claim 26.<br>
28. A plant that is genetically modified at a native genomic locus, said genomic<br>
locus encoding a polypeptide selected from the group consisting of:<br>
(a) an amino acid sequence comprising SEQ ID NO: 2, 6, 9, 12,15, 18,<br>
23, 27, 41, 43, 46, 49, 52, 54, 57, 59, 61, 63, 66, or 77;<br>
(b) an amino acid sequence comprising at least 85% sequence identity<br>
to SEQ ID NO: SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46,<br>
49, 52, 54, 57, 59, 61, 63, 66, or 77, wherein said polypeptide has<br>
cytokinin synthesis activity;<br>
wherein said plant is genetically modified to increase, reduce, or eliminate<br>
the activity of said polypeptide.<br>
29. The plant of any one of embodiments 5 to 9, 12 to 22, and 24 to 28,<br>
wherein said plant is a monocot.<br>
30. The plant of embodiment 29, wherein said monocot is maize, wheat, rice,<br>
barley, sorghum, or rye.<br>
31. The plant of any one of embodiment 5 to 9,12 to 22, and 24 to 28, wherein<br>
said plant is a dicot.<br>
32. A method for reducing or eliminating the activity of a polypeptide in a plant<br>
comprising Introducing into said plant a polynucleotide comprising a<br>
nucleotide sequence comprising a fragment of SEQ ID NO: 1, 3, 4, 5, 7, 8,<br>
10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50,<br>
51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76, or<br>
comprising a sequence complementary to said fragment.<br>
33. The method of embodiment 32, wherein providing said polynucleotide<br>
decreases the level of cytokinin in said plant.<br>
34. The method of embodiment 32, wherein the activity of said polypeptide is<br>
reduced or eliminated in a vegetative tissue, a reproductive tissue, or the<br>
vegetative tissue and the reproductive tissue.<br>
35. The method of embodiment 32, wherein said introduced polynucleotide is<br>
operably linked to a tissue-preferred promoter, a constitutive promoter, or<br>
an inducible promoter.<br>
36. The method of embodiment 35, wherein said promoter is a root-preferred<br>
promoter.<br>
37. The method of embodiment 32, wherein reducing or eliminating the activity<br>
of said polypeptide modulates root development of the plant.<br>
38. The method of embodiment 37, wherein the modulated root development<br>
comprises at least one of an increase in root growth or an increase in the<br>
formation of lateral or adventitious roots.<br>
39. A method for increasing the level of a polypeptide in a plant comprising<br>
introducing into said plant a polynucleotide comprising a nucleotide<br>
sequence selected from the group consisting of:<br>
(a) a nucleotide sequence comprising SEQ ID NO: 1, 3,4, 5, 7, 8, 10,<br>
11,13, 14,16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44,45, 47, 48,<br>
50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76;<br>
(b) a nucleotide sequence encoding an amino acid sequence<br>
comprising SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49,<br>
52, 54, 57, 59, 61,63, 66, or 77;<br>
(c) a nucleotide sequence comprising at least 85% sequence identity to<br>
SEQ ID NO: 1, 3,4, 5, 7, 8, 10, 11, 13,14, 16,17, 19, 20, 21, 22, 24,<br>
26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65,<br>
69, 70, 71, 72, 73, 74, or 76, wherein said polynucleotide encodes a<br>
polypeptide having cytokinin synthesis activity;<br>
 (d)' a nucleotide" sequence comprising at least 40 consecutive<br>
nucleotides of SEQ ID NO: 1, 3, 4, 5, 7, 8, 10, 11, 13, 14,16,17,19,<br>
20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58,<br>
60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76, or a complement<br>
thereof, wherein said polynucleotide encodes a polypeptide having<br>
cytokinin synthesis activity; and,<br>
(e) a nucleotide sequence that hybridizes under stringent conditions to<br>
the complement of the nucleotide sequence of a), wherein said<br>
stringent conditions comprise hybridization In 50% formamide, 1 M<br>
NaCI, 1% SDS at 37°C, and a wash in 0.1X SSC at 60°C to 65°C,<br>
wherein said polynucleotide encodes a polypeptide having cytokinin<br>
synthesis activity.<br>
40. The method of embodiment 39, wherein expressing said polynucleotide<br>
increases the level of a cytokinin in the plant.<br>
41. The method of embodiment 39 or 40, wherein the level of the polypeptide is<br>
increased in a vegetative tissue, a reproductive tissue, or the vegetative<br>
tissue and the reproductive tissue.<br>
42. The method of embodiment 39 or 40, wherein said promoter is a tissuepreferred<br>
promoter, a constitutive promoter, or an inducible promoter.<br>
43. The method of embodiment 42, wherein said promoter is a root-preferred<br>
promoter, a leaf-preferred promoter, a shoot-preferred promoter, a seedpreferred<br>
promoter, a kernel-preferred promoter, or an inflorescencepreferred<br>
promoter.<br>
44. The method of embodiment 39 or 40, wherein the stress tolerance of said<br>
plant is maintained or improved.<br>
45. The method of embodiment 44, wherein the size of the plant is increased or<br>
maintained.<br>
46. The method of embodiment 44, wherein seed abortion is minimized.<br>
47. The method of embodiment 44, wherein the seed set of said plant is<br>
increased or maintained.<br>
48. The method of embodiment 44, wherein said promoter is stress-insensitive<br>
and is expressed in a tissue of the developing seed during the lag phase of<br>
development.<br>
49. The method of embodiment 39 or 40, wherein increasing the level -of the<br>
polypeptide increases the shoot growth of the plant.<br>
50".' The method oFem'BbS'iment 39 or 40, wherein increasing the activity of the<br>
polypeptide increases seed size or seed weight of the plant.<br>
51. The method of embodiment 50, wherein the increased seed size or seed<br>
weight comprises an increase in at least one of embryo size, embryo<br>
weight, cotyledon size, or cotyledon weight.<br>
52. The method of embodiment 39 or 40, wherein increasing the activity of the<br>
polypeptide increases plant yield or plant vigor of said plant.<br>
53. The method of embodiment 39 or 40, wherein increasing the activity of the<br>
poiypeptide modulates floral development.<br>
54. The method of embodiment 39 or 40, wherein increasing the level of the<br>
polypeptide delays senescence or increases leaf growth.<br>
55. The method of any one of embodiments 32-40, wherein said plant is a<br>
dicot.<br>
56. The method of any one of embodiments 32-40, wherein said plant is a<br>
monocot.<br>
57. The method of embodiment 56, wherein said monocot is maize, wheat, rice,<br>
barley, sorghum, or rye.<br>
58. An isolated polynucleotide comprising a nucleotide sequence comprising<br>
SEQ ID NO: 25 or 75.<br>
59. A DNA construct comprising a promoter operably linked to a nucleotide<br>
sequence of interest, wherein said promoter comprises the polynucleotide<br>
of embodiment 58.<br>
60. An expression vector comprising the DNA construct of embodiment 59.<br>
61. A plant having at least one DNA construct comprising a heterologous<br>
nucleotide sequence of interest operably linked to a promoter, wherein said<br>
promoter comprises SEQ ID NO: 25 or 75.<br>
62. The plant of claim 61, wherein said DNA construct is stably incorporated<br>
into the genome of the plant.<br>
63. The plant of embodiment 61 or 62, wherein said plant is a dicot.<br>
64. The plant of embodiment 61 or 62, wherein said plant is a monocot.<br>
65. The plant of embodiment 63, wherein said monocot is maize, wheat, rice,<br>
barley, sorghum, or rye.<br>
66. The plant of embodiment 61 or 62, wherein said DNA sequence of interest<br>
encodes a polypeptide.<br>
67. A method of regulating the expression of a nucleotide sequence of interest,<br>
said method comprising introducing into' a plant a DNA construct<br>
comprising a heterologous nucleotide sequence of interest operably linked<br>
to a promoter comprising the nucleotide sequence of embodiment 58.<br>
68. The method of claim 67, wherein said DNA construct is stably integrated<br>
into the genome of the plant.<br>
69. The method of embodiment 67 or 68, wherein said plant is a dicot.<br>
70. The method of embodiment 67 or 68, wherein said plant is a monocot.<br>
71. The method of embodiment 70, wherein said monocot is maize, wheat, rice,<br>
barley, sorghum, or rye.<br>
72. The method of embodiment 67 or 68, wherein said DNA sequence of<br>
interest encodes a polypeptide.<br>
The following examples are offered by way of illustration and not by way of<br>
limitation.<br>
EXPERIMENTAL<br>
Example 1. Cloning and Gene Characterization of ZmlPTI<br>
Below we describe the identification and characterization of an IPT<br>
polypeptide from maize designated ZmlPTI.<br>
Material and methods: A Mo17 BAG library was screened using a 3'-end<br>
fragment of the ZmlPTI cDNA from B73, identified by sequence similarity to<br>
Agrobacterium ipt. One of the positive clones was digested by H/ndlll, and<br>
subcloned into pBluescript. Recombinant plasmids were screened using colony<br>
screening and a 3'-end fragment as a probe. One positive clone was sequenced.<br>
Samples used for RT-PCR were harvested in the field from three individual plants.<br>
Five jig of total RNA was used for RT-PCR using ThermoScript RT-PCR System<br>
from invitrogen. The reverse transcribed mixture for PCR used primers designed<br>
across an intron-exon-intron junction in order to avoid amplification of genomic<br>
DNA.<br>
Results:<br>
A. . Deduced protein sequence:<br>
Two putative maize ipt ESTs were identified whose deduced amino acid<br>
sequences show similarity to the Agrobacterium (data not shown), Arabidopsis<br>
and Petunia IPT proteins (Figure 1). Full-insert sequencing of these two ESTs<br>
revealed that they were identical and the corresponding cDNA sequence was<br>
called ZmlPTI (SEQ ID NO: 22).<br>
Figure 1 provides an amino acid alignment of the ZmlPTI, Arabidopsis, and<br>
Petunia cytokinin biosynthetic enzymes. Asterisks indicate amino acids conserved<br>
in many cytokinin biosynthetic enzymes. The amino acids designated by the<br>
underline indicate a putative ATP/GTP binding site (at about amino acids 84-90).<br>
As shown in Figure 1, the deduced protein sequence of ZmlPTI contains the<br>
exact consensus sequence<br>
GxTxxGKtST]xxxxx[VLI]xxxxxxx[VLI)[VLI]xxDxxQx{57,60}[VLI][VLI]xGG[ST] (SEQ<br>
ID NO: 32) (where x denotes any amino acid residue, [ ] any one of the amino<br>
acids shown in [ ], and x{m,n} m to n amino acid residues in number) that was<br>
used by Take! et a/. (2001) J. Biol. Chem. 276:26405-26410 to isolate the<br>
Arabidopsis genes. Note that ZmlPTI also has a putative ATP/GTP binding site<br>
at about amino acids 51-58. In addition, the length of ZmlPTI is very similar to the<br>
AtlPT4 and Sho genes. In addition, the specific zinc-finger like motif<br>
(CxxCx{12,18}HxxxxxH) (SEQ ID NO:33) found in all tRNA IPTs of eukaryotes<br>
(which is necessary to bind tRNA molecules) is absent from ZmlPTI.<br>
The ZmlPTI sequence shares 21.9% amino acid sequence identity (34.1%<br>
similarity) across the full length to Sho (cytokinin biosynthetic protein from<br>
Petunia); 10.8% identity (21.2% similarity) across its full length to ipt<br>
(Agrobacterium); 24.7% identity (34.8% similarity) across its full length to AtlPTI<br>
(Arabidopsis); 35.6% identity (45.3% similarity) across its full length to AtlPT2<br>
(Arabidopsis); 22.4% identity (34.6% similarity) across its full length to AtlPT3<br>
(Arabidopsis); 20.7% identity (31.6% similarity) across its full length to AtlPT4<br>
(Arabidopsis); 22.7% identity (35.7% similarity) across its full length to AtlPT5<br>
(Arabidopsis); 21.8% identity (36.4% similarity) across its full length to AtlPT6<br>
(Arabidopsis); 23.4% identity (33.1% similarity) across its full length to AtlPT7<br>
(Arabidopsis); 26.3% identity (35.9% similarity) across its full length to AtlPTS<br>
(Arabidopsis); and, 18.9% identity (31.2% similarity) across its full length to AtlPT9<br>
(Arabidopsis).<br>
A variant of the ZmlPTI sequence is also provided. SEQ ID NOS: 22, 23,<br>
and 24 correspond to the nucleotide and amino acid sequence of ZmlPTI derived<br>
from the spliced sequence of the Mo17 genomic clone. SEQ ID NOS: 26, 27, and<br>
28 are variants of the ZmlPTI sequence derived from sequencing the full-length<br>
EST from B73. An alignment of ZmlPTI and its variant is shown in Figure 3.<br>
These sequences share 98% overall amino acid sequence identity.<br>
B. Gene structure:<br>
A Mo17 BAG library was screened using probes corresponding to the two<br>
ESTs and four identical clones were identified. An 11kb Hind\\\ fragment from one<br>
of the clones was subcloned in pBluescript and sequenced. Alignment with the<br>
full-insert sequence of the EST clone revealed the presence of six introns.<br>
Interestingly neither the AtlPT4 gene from Arabidopsis nor the Sho gene from<br>
Petunia contain introns. The genomic sequence of ZmlPTI is set forth in SEQ ID<br>
NO: 21.<br>
Example 2. Gene expression of ZmlPTI<br>
One of the identified ZrnlPTI ESTs was from a B73 embryo library, the<br>
other one was from a developing root library. In order to gain an impression of the<br>
level of expression of ZmlPTI, a search of the Lynx database was performed. A<br>
perfect tag was found in the 3'-end of the gene, 231 bp from the poly A tail start.<br>
Tissue types, number of library hits, and average pprn are presented in Table 1.<br>
Expression was found to be very low in most organs, but higher in seedling and<br>
embryo libraries. In embryo libraries, expression was higher at 10 DAP than at<br>
later development stages (Figure 4),<br>
(Table Removed)<br>
Using RT-PCR, the expression pattern of ZmlPT in various maize organs<br>
and on a kernel developmental series was tested. No amplification product could<br>
be detected after 20 cycles, confirming the very low expression of the gene.<br>
However, after 30 cycles, bands of the appropriate size were amplified (Figure 5).<br>
Figure 5A shows ZmlPTI transcripts are present in ovaries of mature plants and<br>
leaf, mesocotyl, ana roots or seedlings. This distribution indicates a bias in<br>
expression of this gene to meristematic-iike and rapidly developing tissues. In<br>
developing B73 kernels (Figure 5B), ZmlPTI transcript is strongly present from 0<br>
to 10 DAP, then decreases beyond 15 DAP. This pattern of expression of ZmlPTI<br>
correlates with the known profile of cytokinin accumulation in developing kernels,<br>
which peaks during the lag phase. This accumulation of cytokinin is thought to<br>
drive early cell division in endosperm and embryo development.<br>
In a similar manner, the ZmlPTI tag could only be detected in the cell<br>
division zone of leaves, and in leaf discs treated with BA. This distribution of<br>
transcripts indicates a bias in expression of ZmlPTI to meristematic-like and<br>
rapidly developing tissues, indicating that maize roots and developing kernels are<br>
strong sites for cytokinin synthesis.<br>
Examples. Isolation and Gene Characterization of ZmlPT2. ZmlPT4.<br>
ZrnlPTS. ZmlPT6. ZmlPTT. ZmlPTS and ZmlPT9<br>
The AtlPTI and AtlPT3 to AtlPT8 protein sequences were blasted against<br>
the six possible frames generated by the maize genomic sequences and searched<br>
for some degree of similarity. Because rice and maize genomes show a significant<br>
degree of synteny, the same method was used against rice genomic database to<br>
optimize this search. The rice sequences with an E-score of at least 200 were then<br>
used for an additional screen of the GSS maize database. Since at that time, the<br>
GSS database had not been assembled into contigs, the sequences obtained<br>
which had an E-score of at least 150 were pooled and aligned using Sequencher.<br>
Using this method, eight maize contigs encoding putative CK biosynthetic<br>
enzymes were identified (ZmlPT2 to ZmlPT9), six of them showing an open<br>
reading frame without introns. The translated proteins corresponding to these<br>
putative genes contained 320 to 380 amino acids, which correlates with the<br>
expected size for plant IPT proteins. An alignment of the corresponding proteins is<br>
presented in Figure 1. The deduced protein sequences of the new ZmlPT genes<br>
(except for ZmlPTS) contain the exact consensus sequence found in IPT proteins<br>
from different species. This sequence,<br>
GxTxxGK[ST]xxxxx[VLI]xxxxx)0([VLI][VLI]xxDxxQx{57,60}[VLI][VLI]xGG[ST3<br>
(where x denotes any amino acid residue, [ ] anyone of the amino acids shown in [<br>
], and x{m,n} m to n amino acid residues in number) (SEQ ID NO:32) is also found<br>
in ZmlPTI and was previously used by Kakimoto and Takei to isolate the<br>
Arabidopsis IPi genes. Homoiogy with other plant IPT proteins was found to be<br>
around 40%.<br>
The amino acid sequence identity and similarity to top BLAST hits across<br>
the full length of ZmlPT2, ZmlPT4, ZmlPTS, ZmlPTS, ZmlPT7, ZmlPTS and<br>
ZmlPT9 are provided below in Table 2.<br>
(Table Removed)<br>
The maize IPT sequences also have putative ATP/GTP binding sites at<br>
about arnino acids 17-24 for ZmlPT2, about amino acids 72-79 for ZmlPT4, about<br>
amino acids 57-64 for ZrnlPTS, about amino acids 55-62 for ZmlPT6, about amino<br>
acids 23-30 for ZmlPT7, and about amino acids 83-90 for ZmlPTS.<br>
The polypeptides encoded by ZmlPT polynucleotides share sequence<br>
similarity to known proteins. For example, a polypeptide encoded by nucleotides<br>
821 to 3 of ZmlPT9 shares 55% amino acid sequence identity to amino acids 48 to<br>
327 of a tRNA isopentenyltransferase from Arabidopsis thaliana (GenBank<br>
Accession No. BAB59048.1). A polypeptide encoded by nucleotides 821 to 3 of<br>
ZmlPT9 shares 55% arnino acid sequence identity to amino acids 48 to 327 of a<br>
putative IPP transferase from Arabidopsis thaliana (GenBank Accession No.<br>
AAK64114.1). A polypeptide encoded by nucleotides 821 to 3 of ZmlPT9 shares<br>
55% amino acid sequence identity to amino acids 48 to 327 of a IPP transferaselike<br>
protein from Arabidopsis thaliana (GenBank Accession No. AAM63091.1). A<br>
polypeptide encoded by nucleotides 839 to 3 of ZmlPT9 shares 36% amino acid<br>
sequence identity to amino acids 28 to 278 of a putative tRNA delta-2-<br>
isopentenylpyrophosphate transferase from Arabidopsis thaliana (GenBank<br>
Accession No. YP_008242.1). A polypeptide encoded by nucleotides 824 to 3 of<br>
ZmiPTG shares 35% aininu add sequence identity to amino acids 17 to 248 of a<br>
tRNA isopentenylpyrophosphate transferase from Streptococcus pneumoniae R6<br>
(GenBank Accession No. NP_358182.1). A polypeptide encoded by nucleotides<br>
818 to 3 of ^.miPT9 shares 34% amino acid sequence identity to amino acids 2 to<br>
231 of a tRNA delta(2)-isopentenylpyrophosphate transferase (GenBank<br>
Accession No. Q8CWS7). A polypeptide encoded by nucleotides 818 to 3 of<br>
ZmlPT9 shares 34% amino acid sequence identity to amino acids 2 to 231 of a<br>
tRNA isopentenylpyrophosphate transferase from Streptococcus pneumoniae R6<br>
(GenBank Accession No. NP_345176.1). A polypeptide encoded by nucleotides<br>
818 to 3 of ZmlPT9 shares 34% amino acid sequence identity to amino acids 31 to<br>
275 of a tRNA delta(2)-isopentenylpyrophosphate transferase from Chlamydophila<br>
caviae (GenBank Accession No. AAP05599.1). A polypeptide encoded by<br>
nucleotides 818 to 435 of ZmlPT9 shares 48% amino acid sequence identity to<br>
amino acids 6 to 133 of a tRNA delta(2)-isopentenylpyrophosphate transferase<br>
from Xylella fastidiosa 9a5c (GenBank Accession No. NP_297383.1). A<br>
polypeptide encoded by nucleotides 818 to 435 of ZmlPT9 shares 48% amino acid<br>
sequence identity to amino acids 6 to 133 of a tRNA delta(2)-<br>
isopentenylpyrophosphate transferase from Xylella fastidiosa Dixon (GenBank<br>
Accession No. Xylella fastidiosa Dixon).<br>
Example 4. Isolation of ZmlPT2 from Mo17 and B73 maize lines and<br>
molecular characterization of the ZmlPT2 gene<br>
Material and Methods:<br>
Plant materials: Maize (Zee mays) varieties B73 and Mo 17 were used in<br>
this study. Samples were harvested from field-grown plants at different stages of<br>
development and stored at -80°C. Kernel samples were harvested every five days<br>
from 0 to 25 DAP and dissected by isolating whole kernels (0 DAP), pedicel,<br>
nucellus and pericarp (5 DAP), pedicel, nucellus, endosperm/embryo sac arid<br>
pericarp (10 DAP), or pedicel, embryo, endosperm and pericarp (15, 20 and 25<br>
DAP). Tissues corresponding to 2 to 4 different ears were pooled. The series of<br>
sample harvested every DAP from 0 to 5 (whole kernels), from 6 to 15 and then<br>
20, 27 and 34 DAP (seeds without pedicel) or 20, 25, 30 and 35 DAP (pedicels)<br>
were previously used to study the expression pattern of the cytokinin oxidase 1<br>
gene (Ckx1) from corn (Brugiere et a/., 2003, supra).<br>
Arabidopsis thaliana ecotype Columbia was used for Arabidopsis<br>
transformation studies.<br>
PCR: ZmlPT2 coding sequence was PCR amplified from B73 and Mo 17<br>
genomic DNA. Primers ZmlPT2-5' f5'-ATCATCAAGACAATGGAGCACGGTG-3')<br>
'(SEQ" ID NO: 78) and Zm/PT2-3' (5'-CGTCCGCTAGCTACTTATGCATCAG-3"i<br>
(SEQ ID NO: 79) were designed based on the GSS contig sequence (coding<br>
sequence is underlined). As part of the Gateway cloning procedure, aft-flanked<br>
ZmlPT2 fragment was amplified using primers ZmlPT2-5-Gateway (5'-<br>
GGGGACAAGTTTGTACAAAAAAGCAGGCTCAATGGAGCACGGTGCCGTCGCCG-<br>
31) (SEQ ID NO: 80) and ZmlPT2-3-Gateway (51-<br>
GGGGACCACTTTGTACAAGAAAGCTGGGTCTTATGCATCAGCCACGGCGGTG-<br>
3') (SEQ ID NO: 81).<br>
In each case, a touchdown PCR was performed (GeneAmp PCR System<br>
9700), using the following cycling parameters: 94°C for 2 min (one cycle), 94°C for<br>
30 s, 65°C for 45 s and 72°C for 1 min 30 s (5 cycles, annealing temperature<br>
reduced by 1°C per cycle), 94°C for 30 s, 60°C for 45 s and 72°C for 1 min 30 s<br>
(30 cycles), 72°C for 7 min, and termination at 4°C. Pfu Ultra Hotstart DNA<br>
polymerase (Stratagene) for its very low average error rate (less than 0.5 % per<br>
500-bp fragment amplified) was used.<br>
PCR products were loaded on an agarose gel containing ethidium bromide<br>
(1:10000, v/v). Bands corresponding to ZmlPT2 gene and aft-flanked ZmlPT2<br>
gene were gel extracted using QIAquick PCR purification kit (QIAgen).<br>
DNA and RNA extraction: Genomic DNA was extracted from B73 and<br>
Mo17 plant samples at V3-4 stage according to Dellaporta et a/. (1983) Plant Mol<br>
Biol 1:19-21 and stored at -20°C. Total RNA was prepared using a hot phenol<br>
extraction procedure according to Verwoerd et a/. (1989) Nucleic Acid Res<br>
17:2362 and stored at -80°C. The kernel developmental series samples were<br>
purified using RNeasy Mini Protocol for RNA Cleanup (QIAgen) and eluted in 50 ul<br>
DEPC water. Optical Density (DO) at 260 and 280 nm was used to assess the<br>
purity of RNA preps and measure RNA and DNA concentrations.<br>
Southern blots. Northern blots, and hybridization: For Southern blots,<br>
digested genomic or BAG clones DNA were run on 0.8% agarose gel at 110V,<br>
stained after migration in a 1:10000 (v/v) ethidium bromide solution in TAE buffer,<br>
and transferred as indicated below. For Northern blots, ethidium bromide was<br>
added to denatured RNA samples and run at 80 V on 1.5% denaturing agarose<br>
gel (Brugiere et al. (2003) Plant Physiol. 732:1228-1240). Blotting was performed<br>
using Turbo-blotter (Schleicher &amp; Schuell) according to the manufacturer<br>
guidelines. After transfer, nylon membranes (Nytran plus, Schleicher &amp; Schuell)<br>
were cross-linked with a Stratalinker (Stratagene) and baked at 80°C for 30 min.<br>
Probes were labeled with" [a-32P]-dCTP using random priming (Red/prime //<br>
RandomPrime Labelling System, Amersham Biosciences) and purified with Quick<br>
Spin Columns (Roche). Hybridizations were carried out at 65°C for 16h using<br>
ExpressHyb hybridization solution (BD Biosciences) and membranes were<br>
washed under stringent conditions (O.lxSSC, 0.1% SDS) as previously described<br>
(Brugiere et a/. (2003) Plant Physiol. 132:1228-1240). Relative transcript<br>
abundance was quantified using a phosphor imager (MD860, Molecular Dynamic)<br>
with imaging software (ImageQuant, Molecular Dynamics).<br>
BAG subclonina: BAG clones were digested and subcloned in pBluescript<br>
SK+. This plasmid includes a multiple cloning site between the /acZgene and its<br>
promoter. The lacZ gene is often used as a reporter gene because it encodes a pgalactosidase,<br>
which produces a dark blue precipitate on X-gal1 enzymatic<br>
hydrolysis. The bacteria containing a plasmid in which the BAG fragment is<br>
inserted in the multiple cloning site and therefore do not synthesize this enzyme<br>
will appear white. This allows the selection of colonies containing BAG subclones<br>
that can be further screened by PCR or Southern blot.<br>
Results:<br>
Isolation of the ZmlPT2 gene from corn aenomlc DNA: Genornic DNA from<br>
two different varieties of corn, B73 and Mo17, was extracted and ZmlPT2 coding<br>
sequence was amplified by touch-down PCR. The ZmlPT2 gene amplification<br>
product was used as a probe for Southern and Northern blot experiments. The<br>
Z/77/PT2 CDS was cloned in pDONR221 (Invitrogen) and sequenced. The Mo17<br>
sequence was 100% homologous to the GSS contig sequence. The B73 and<br>
Mo17 genes were found to be 98.8% homologous at the nucleotide level. The<br>
differences between the two genes at the nucleotide level resulted in the<br>
modification of 3 amino acids at the protein level (96.1% identity). The nucleotide<br>
and amino acid sequences of ZmlPT2 are set forth in SEQ ID N0:3 and 2, and the<br>
nucleotide and amino acid sequences of a variant of ZmlPT2 are set forth in SEQ<br>
ID N0:76 and 77. An alignment of ZmlPT 2 and the variant of ZmlPT2 shows that<br>
the differences in the polypeptides ocurr at amino acid 125 (A-»L), amino acid 138<br>
(O-*R): and amino acid 193 R-»H.<br>
Mapping of the gene on the maize genome: To determine if ZmlPT2 was a<br>
single- or multi-copy gene in maize, B73 and Mo17 genomic DNA were digested<br>
with" H/ndlll, EcoRI, and"'£=coW and run on a gel, which was blotted as described<br>
in the Materials and Methods section. The membrane was hybridized with the<br>
previously extracted ZmlPT2 genomic fragment as a probe. The picture of the<br>
membrane autoradiography is presented in Figure 6.<br>
The single bands observed for each of these digestions show that ZmlPT2<br>
is most likely a single-copy gene. The size of the corresponding H/ndlll fragment<br>
was later confirmed using BAG clones. To obtain additional information regarding<br>
the physical location of the ZmlPT2 gene in the maize genome, the Oat-Maize<br>
chromosome addition lines were used (Ananiev ef a/, (1997) Proc. A/a//. Acad. Sci.<br>
94:3524-3529). A PCR was performed with the ZmlPT2-5' and ZmlPT2-3' primers<br>
as described in the Materials and Methods section (data not shown). The<br>
expected size of the amplification fragment was 995 bp. B73 genomic DNA<br>
samples were used as positive controls, while oat genomic DNA and water were<br>
used as negative ones.<br>
The data shows that the amplification of the ZmlPT2 sequence could only<br>
be seen with the chromosome 2 oat-maize addition line. This finding was verified<br>
and the position on chromosome 2 refined using a bioinformatics approach. The<br>
ZmlPT2 sequence was first used to screen the B73 and Mo17 public and<br>
proprietary Bacterial Artificial Chromosomes (BAG) libraries. Positive clones were<br>
identified and used to identify a BAG contig using FPC Contig Viewer. Using this<br>
strategy, markers were identified including several MZA markers that were<br>
physically mapped on maize chromosome 2, bin 4 (data not shown), which<br>
confirmed the experimental mapping of ZmlPT2 gene using the OMA lines.<br>
Example 5. Gene Expression of ZmlPTZ. ZmlPT4. ZmlPTS. ZmlPTS and<br>
ZmlPTS<br>
In order to gain an impression of the level of expression of the various<br>
ZmlPT sequences, a search of the Lynx database was performed. Tissue types,<br>
number of library hits, and average ppm are presented in Tables 3-7.<br>
As shown in table 3, expression of ZmlPT2 was found to be restricted to<br>
kernel tissue and to correlate with the start of cytokinin biosynthesis in the kernel<br>
as described in Bruqiere et al. (2003) Plant Physiol. 132(3): 1228-1240. Figure 9<br>
provides a graphical illustration of the ppm values for ZmlPT2 in the Lynx embryo<br>
libraries. Other ZmlPT genes have low expression, consistent with their possible<br>
function as cytokinin synthases in other tissues such as root, meristem and<br>
endosperm. See table 4-7 below.<br>
(Table Removed)<br>
Example 6. Expression of the ZmlPT2 Gene in Corn Tissues<br>
The expression pattern of ZmlPT2 in different organs at different stages of<br>
development was studied in order to provide information regarding its putative<br>
function in cytokinin biosynthesis. Based on Lynx data (discussed in Example 5),<br>
the expression of ZmlPT2 seemed to be restricted to developing kernels (Figure<br>
9), To get an overall view of ZmlPT2 expression in corn and verify Lynx data,<br>
RNA was extracted from different B73 tissues: leaf, stalk, roots, and whole kernels<br>
at 0, 5, 10, 15, 20 and 25 DAP. Forty ug of total RNA from each sample were<br>
stained with ethidiurn bromide and loaded on an agarose gel. The blot obtained<br>
was hybridized with a [a-32P]-dCTP labeled ZmlPT2 probe. In a second<br>
hybridisation, a nyclophilin probe was used as a loading control. After<br>
quantification via a phosphor-imager, the ratio of the expression of ZrnlPT2<br>
compared to cyclophilin was calculated. Cyclophilin is considered to be<br>
constitutively expressed across different organs (Marivet et a/. (1995) Mol Genet<br>
•Den"247:222-228. Results are shown in Figure 7. ZmlPT2 transcripts are<br>
detected at low levels in the leaf, stalk and roots, but at higher levels in kernels,<br>
where expression is low at 0 DAP, increases from 5 to 10 DAP and decreases<br>
from 15 to 25 DAP, This expression profile not only confirms the Lynx data but<br>
coincides with the appearance and disappearance of CK in the kernels (See,<br>
Example 5).<br>
To obtain a more precise view of the expression pattern of ZmlPT2 in<br>
kernels, levels of ZmlPT2 transcripts were measured in 0- to 5-DAP kernels with<br>
pedicels; 6- to 34-DAP kernels without pedicels; and pedicels alone, 6 to 42 DAP.<br>
See Figure 8. As previously done, the gel was loaded with 40 ug of total RNA,<br>
stained with ethidium bromide, and blotted on a nylon membrane. Membranes<br>
were hybridized with a 32P-dCTP labeled ZmlPT2 probe. The results of the<br>
Northern experiment with "seed without pedicel" samples indicate that low levels<br>
of ZmlPT2 expression are detected between 0 and 4 DAP. The expression<br>
increases from 5 to 8 DAP, peaks at 8 DAP, and decreases until 14 DAP. At later<br>
stages (15 to 34 DAP), transcript levels seem to increase again. However, it is not<br>
clear whether this is the result of diminution of cyclophilin expression. This<br>
phenomenon was also observed for Ckx1 expression (Brugiere etal. (2003) Plant<br>
Physiol ?32:1228-1240). In the Northern experiment with pedicel samples,<br>
transcript levels drastically increase from 6 to 10 DAP, peak at 10 DAP, and slowly<br>
decrease until 15 DAP. In this experiment again, transcript levels seem to rise at<br>
later stages, but it is unclear whether this is due to the diminution of cyclophilin<br>
expression. In this experiment, the sample corresponding to "seed without pedicel<br>
at 9 DAP" was used as a control to allow the comparison with the other blot. The<br>
relative expression at 9 DAP is four times as high as in the control, showing that<br>
Zm/P72 expression in the pedicel is much higher than in the rest of the seed. This<br>
difference is consistent with the fact that CK levels are nearly twice as abundant in<br>
the pedicel as in the rest of the seed (Brugiere et at. (2003) Plant Physiol<br>
732:1228-1240).<br>
The relative transcript levels were compared to ZR concentrations<br>
measured in the same samples (solid line in Figure 8). Interestingly, ZmlPT2<br>
expression nicely overlaid ZR accumulation in the pedicel, whereas in the rest of<br>
the seed, it slightly preceded the peak in ZR, including at later stages.<br>
Taken together, these results indicate that ZmlPT2 is expressed transiently<br>
during kernel development both in the pedicel and the rest of the seed, and that its<br>
pattern of expression, which parallels ZR levels in the kernel, is consistent witn<br>
ZmlPT2 role as a CK biosynthetic gene.<br>
The expression of ZmlPT2 in different kernel tissue was also studied in<br>
different kernel tissues. In this study, dissected kernel samples from 0 to 25 DAP<br>
were used. Samples were collected from the field in Johnston, Iowa, USA. Kernels<br>
were dissected into different parts (pedicel, nucelius, endosperm/embryo sac,<br>
endosperm, embryo and pericarp) depending on the stage considered. The gel<br>
was loaded with 30ug of purified total RNA, stained with ethidium bromide, and<br>
blotted onto a nylon membrane.<br>
The results shown in Figure 11 confirm that ZmlPT2 transcripts levels in<br>
pedicel are more abundant than in the rest of the seed. This is especially true at<br>
15, 20 and 25 DAP where some expression is also seen in the embryo samples.<br>
At 10 DAP however, ZmlPT2 transcripts are found in similar amounts in<br>
developing endosperm/embryo sac and pedicel. Together with the fact that 1)<br>
cytokinins are more abundant in the pedicel than the rest of the seed and that 2)<br>
transcript and activity of cytokinin oxidase in this organ is also more abundant than<br>
the rest of the seed (Brugiere et at. (2003) Plant Physiol. 732:1228-1240), these<br>
results indicate that the pedicel is most likely a major site for CK biosynthesis.<br>
Recent data presented on the expression of Arabidopsis IPT genes allows us to<br>
hypothesize that the expression of ZmlPT2 could occur in phloem cells where it<br>
could be responsible for the synthesis of CK, which would be targeted to the<br>
vascular bundles for transport to developing kernels. The presence of ZmlPT2<br>
transcripts in both developing endosperm and embryo is observed at times when<br>
cell division is the most active in these tissues. This again supports a role for<br>
ZmlPT2 as a CK biosynthetic protein, which catalyzes CK formation in fast<br>
dividing/developing tissues such as endosperm at 10 DAP and growing embryo,<br>
and could drive sink strength in the pedicel to support kernel growth.<br>
Example 7. Expression and Purlfiatlon of the 2mlPT2 Polvpeptide<br>
From E co//'<br>
Materials and Methods:<br>
Recombinant protein purification, gel electrophoresis and Western blot:<br>
BL21-AI (Invitrogen) £ col! harboring the pDEST17-ZmlPT2 (Mo17) plasmid was<br>
grown overnight. A 1/200 dilution of this culture was used to inoculate fresh LB<br>
medium and bacteria were grown at 37°C for 2h before induction with 0.2% L-<br>
arablriose arid then growni for 2"' to 4h. Bacterial protein extracts were prepared as<br>
described by the supplier and run on a 12.5% polyacrylamide gel in denaturing<br>
condition. The His-tagged protein was purified from crude protein extracts using a<br>
Ni-NTA agarose solution according to the provider's recommendations (Qiagen).<br>
After electrophoresis, proteins were either revealed by gel staining with GelCode<br>
(Pierce) or blotted onto a polyvinylidene difluoride (PVDF) membrane using an<br>
electro-transfer procedure. Western blot was carried out as described previously<br>
(Brugiere et al. (1999) Plant Cell 77:1995-2012) using an anti-poly histidine<br>
monoclonal antibody developed in mice (Sigma-AIdrich) and an anti-mice IgG<br>
antibody conjugated to alkaline phosphatase developed in goat.<br>
Results:<br>
Cloning of the ZmlPT2 coding sequence in a vector compatible with the<br>
Gateway system: In an effort to characterize the function of the ZmlPT2 protein<br>
both in vitro and in vivo, two approaches were used. The first aimed at expressing<br>
and purifying a tagged ZmlPT2 protein in E. coll, and the second at transforming<br>
Arabidopsis calli with a construct driving the over-expression of the ZmlPT2 gene<br>
under the control of the 35S promoter of the cauliflower mosaic virus. For this<br>
purpose the Gateway system for molecular cloning was used.<br>
The Gateway technology was used to build both the protein expression<br>
vector (£. colt Expression System with Gateway Technology kit, Invitrogen) and<br>
Arabidopsis transformation vector (Multisite Gateway Three-Fragment Vector<br>
Construction Kit, Invitrogen). The.first step was the addition of specific att<br>
sequences to the previously extracted ZmlPT2 fragment. This was achieved by<br>
amplifying this fragment by PCR with a pair of primers specially designed to flank<br>
the gene with the proper att sites.<br>
Once flanked with these specific sites, the new gel-extracted fragment was<br>
inserted by recombination into a donor vector (pDONR221). Recombination was<br>
catalyzed in vitro by the BP clonase. The vector generated was checked by<br>
digestion with multiple restriction enzymes and migration on agarose gel<br>
electrophoresis, The sizes of digested fragments matched with the expected<br>
length of digestion products for each enzyme. Both Mo17 and B73 ZmlPT2 genes<br>
were cloned in individual donor vectors and the inserts sequenced using M13<br>
forward and reverse primers. The Mo17 clone showed a homology of 100% with<br>
the GSS contig sequence and was therefore used to build the expression and<br>
transformation constructs.<br>
in vitro study: expression of the ZmlPT2 protein in £ coli: A tagged<br>
recombinant ZmlPT2 protein was expressed in £ coli. Besides allowing the<br>
ZmlPT2 gene to be transcrlptionally activated in £ coli via induction of the 77<br>
promoter, this approach also permitted addition of a six-histidine-tag to the Nterminal<br>
end of the recombinant ZmlPT2 protein, which could be used for its<br>
purification. The expression vector was generated by recombination of the<br>
ZmlPT2 coding sequence between pDONR221-ZmlPT2 and pDEST17<br>
(Invitrogen).<br>
The transcriptional fusjon of the 6xHis-tag with ZmlPT2 was sequenced and<br>
the expression vector was used to transform the BL21-A1 strain of £ coli. These<br>
cells contain an expression system modulated by L-arabinose, which Induces the<br>
expression of 77 RNA polymerase. Protein extracts collected at different times<br>
(2h and 4h) after 77 RNA polymerase induction were tested on denaturing<br>
polyacrylamide gel electrophoresis (SDS-PAGE). Samples that had not been<br>
induced were collected and used as negative controls, as well as GUS protein<br>
expression with and without induction. The gel was stained with Coomassie blue,<br>
which reveals the presence of proteins.<br>
After electrophoresis, induced and non-induced samples were blotted onto<br>
a PVDF membrane, A Western blot was performed to confirm that the induced<br>
protein contained a His-tag. For this purpose, we used mice antibodies raised<br>
against a poly histidine peptide. These antibodies would only recognize the<br>
tagged protein and would be in turn recognized by anti-mice IgG antibodies<br>
carrying alkaline phosphatase. The presence of the recombinant protein could<br>
therefore be characterized by a reaction transforming a colorless substrate in a<br>
purple product that precipitates on the membrane.<br>
Two bands could be observed, one of expected size (approximately 37kDa)<br>
and one of a slightly bigger size (approximately 40kDa), both of which were<br>
induced by L-arabinose. The two bands could be due to the addition of extrabasic<br>
residues that would increase the positive charge of the protein therefore<br>
altering its migration. It could also be due to the presence of a covalently bound<br>
co-factor on the E.coli expressed protein. In order to decipher between these two<br>
possibilities trypsic digestions of each purified bands could be analyzed by mass<br>
spectrometry. In order to further characterize the two bands it was necessary to<br>
partially purify the protein.<br>
Purification of the His-taaqed ZrnlPT2 protein: The presence of a 6xHis tag<br>
allowed affinity purification of this protein using a Ni-NTA resin column. The crude<br>
extract was loaded on the column and the effluent collected. After several washes<br>
of the column, the protein was eluted using a solution containing imidazol that,<br>
because of its higher affinity to the column, Is able to release the protein from the<br>
column. Samples were collected at each step of the purification and run on a gel,<br>
which was stained as previously.<br>
The experiment indicated that the protein is expressed in a soluble form<br>
since it was shown to be present in both the supernatant and the effluent, but in<br>
smaller amounts in the pellet. The protein was eluted in the second and third<br>
elution fractions. The amount of ZmlPT2 protein in the fraction corresponding to<br>
the second volume of elution was visually estimated to represent 70 to 80% of the<br>
total protein.<br>
A Western blot was carried out using the same samples. The result<br>
confirmed that although the protein is present in small amount in the pellet, most<br>
of it remains in solution (effluent). The strong signal with the effluent indicates that<br>
the amount of ZmlPT2 protein in the crude extract exceeded the column capacity.<br>
In addition, the ZmlPT2 protein was expressed using a C-terminal tag<br>
which allowed the purification of ZmlPT2 as one single band on SDS-PAGE. The<br>
purity of the fractions was close to 100%. Fractions were found to provide<br>
DMAPP:ADP and DMAPP:ATP isopentenyltransferase activity.<br>
Example 8. In vivo Study of the Over-Expression of the ZmlPT2 Gene in<br>
Arabidopsis Calli<br>
Materials and Methods:<br>
In vitro culture: Different media were used for Arabidopsis germination,<br>
callus culture and regeneration (Kakimoto (1998) J. Plant Res. 111:261-265). The<br>
media used were as follows:<br>
• 5000X CIM (callus-inducing medium) hormone mix: 2.5 mg/ml 2,4<br>
dichlorophenoxyacetic acid (2,4-D), 0.25 mg/ml kinetin and 5 mg/ml biotin<br>
dissolved in dimethyl sulfoxide (DMSO).<br>
• 500X vitamin mix: 50 mg/ml myo-inositol, 10 mg/ml thiamine-HCI,<br>
0.5 mg/ml pyridoxine-HCI, and 0.5 mg/ml nicotinic acid.<br>
• GM (germination medium): 1L of mixture comprising 4.3 g Murashige<br>
and Skoog's medium salt base (Sigma), 10 g sucrose, 2 ml 500X vitamin<br>
mix, "10 ml 5% 2-(N-m6rpholino)-ethanesulfonic acid (MES, adjusted to pH<br>
5.7 with KOH), and 3 g Phytagel (Sigma), autoclaved".<br>
• CIM (callus-inducing medium): 1L of mixture comprising 3.08 g<br>
Gamborg's B5 medium salt base (Sigma), 20 g glucose, 2 ml 500X vitamin<br>
mix, 10 ml 5% MES (adjusted to pH 5.7 with KOH), and 3 g Phytagel,<br>
autoclaved and 200 Ml 5000X CIM hormone mix added to it.<br>
• AIM (Agrobacterium infection medium): CIM from which Phytagel is<br>
omitted.<br>
• WASHM (washing medium): GM from which Phytagel is omitted,<br>
plus 100 mg/l sodium cefotaxime.<br>
Selection media for transformed calli:<br>
• GM+IBA (GIBA): GM plus 100 mg/l cefotaxime, 50 mg/l carbenicilin,<br>
3 mg/l Bialaphos and 0.3 mg/l indolebutyric acid (IBA).<br>
• GM+IBA+Z (GIBAZ): GM plus 100 mg/l cefotaxime, 50 mg/l<br>
carbenicilin, 3 mg/l Bialaphos, 0.3 mg/l indolebutyric acid (IBA), and 1 mg/l<br>
frans-zeatin (tZ).<br>
In the experiment aimed at testing the effect of auxin to cytokinin ratio on<br>
root and shoot regeneration, GM was prepared and different amounts of<br>
hormones were added. Twenty-five media containing different combinations of tZ<br>
and IBA concentrations, which were set at 0, 100, 300, 1000 and 3000 ng/ml for<br>
each hormone, were prepared.<br>
Sterilized Arabidopsis thallana seeds were germinated on GM medium and<br>
grown on continuous light at 23°C. For the above experiment, hypocotyls from 15<br>
day-old seedlings were cut with a scalpel and grown on each of the 25 media for 3<br>
weeks at 23°C under continuous light. For experiments requiring the use of callus<br>
tissue, hypocotyls were grown on CIM for 10 to 12 days in the same conditions.<br>
Arabidopsis calli transformation: Induced call! were soaked in a suspension<br>
of Agrobacterium (0.2 ODeoo) in AIM for 5 minutes. Most of the liquid was removed<br>
on filter paper, and calli were placed on CIM culture medium and grown in<br>
continuous light at 23°C for 2 days. Calli were then washed thoroughly in WASHM<br>
medium and placed on GIBA or GIBAZ medium and cultured for about 3 weeks.<br>
Cloning: In order to constitutively express ZmlPT2 in Arabidopsis using the<br>
Gateway system, a construct was built in which the gene was placed under the<br>
control of the 35S promoter of the cauliflower mosaic virus. A Gateway clone<br>
'b'ri'ta'ihing the 35S promoter was constructed using the pDONR-P4-P1R plasmid.<br>
Once these 3 elements were available, a multisite recombination was performed<br>
using the three donor vectors and a fourth vector called destination vector.<br>
The LR cionase allows an organized "three-site" recombination to occur<br>
between the plasmids carrying the promoter, gene of interest and terminator, and<br>
a binary vector containing the left and right border of the Ti plasmid and the BAR<br>
resistance gene. The resulting construct was verified by digestion with restriction<br>
enzymes, migration on agarose gel, and comparison of digested fragment sizes<br>
with expected digestion products.<br>
The final construct contained the 35S-ZmlPT2-PINII sequence and included<br>
the BAR gene. This gene is used as a selection marker for the herbicide<br>
resistance it confers to transformed plant cells.<br>
Transformation in Aarobacterium: The next step was the transformation of<br>
a plasmid containing the 35S-ZmlPT2-PINII construct in Agrobacterium<br>
tumefaciens (LBA4044). This plasmid contains the genes required for infection<br>
and delivery of the T-DNA to A. thaliana cells (vir genes). After electroporation in<br>
the bacteria (Suzuki (1999) Plant Cell Physiol 39:1258-1268), the two plasmids<br>
are able to recombine at their respective COS sites. The result of this<br>
recombination is a 48 kb plasmid called "co-integrate".<br>
Agrobacteria containing this co-integrate were checked using a quality<br>
control process. This procedure consists of extracting the co-integrate plasmid<br>
and transforming it into E. coli in order to verify it by restriction digestions. This<br>
step is necessary to screen for "mis-recombinations" of the two plasmids at the<br>
COS sites, which would result in a non-functional construct.<br>
Although many trials were attempted to transform Agrobacterium cells with<br>
the 35S-ZmlPT2-PINII construct, no colonies containing the right co-integrate<br>
plasmid could be identified. Since the 35S promoter is leaky in Agrobacterium, it<br>
was assumed that ZmlPT2 expression could be lethal for Agrobacterium. The<br>
lethality of the construct could be the result of an active degradation of an<br>
essential compound for Agrobacterium. Such a metabolite could for example be<br>
from the isoprenoid biosynthetic pathway, which includes potential substrates of<br>
CK biosynthesis, such as 4-hydroxy-3methyl-2-(E)-butenyl diphosphate (HMBPP).<br>
Analysis of microbial genomes combined with biochemical experiments<br>
established the existence of two pathways for isoprenoid synthesis, the<br>
mevalonate (MVA) and non-mevalonate (1-deoxyxylulose 5-phosphate, DXP or 2-<br>
u-methyl-u-erythritol- 4-phosphate, MEP) pathways. The DXP pathway has been<br>
found to be present in some bacteria and the chloroplasts of plants. The genes<br>
encoding the non-mevalonate pathway are present mostly in Gram-positive<br>
bacteria, HMBPP is a precursor of the non-mevalonate (MEP) pathway of<br>
isoprenoid biosynthesis and was shown to be a possible substrate for AtlPT7<br>
(Take! et al. (2003) J Plant RQS 116:265-9). Analysis of the genomic sequence of<br>
A. tumefaciens C58 showed that it encodes the enzymes of the MEP pathway but<br>
that those of the MVA pathway are absent (Wood et al. (2001) Science 294:2317-<br>
2323; Qoodner ef al. (2001) Science 294:232-2328). Based on these results we<br>
believe that HMBPP could be the substrate of the ZmlPT2 protein and that<br>
utilization of this compound by the enzyme could prevent the formation of<br>
isoprenoid, which would result in the incapacity of the bacteria to grow.<br>
To elude this problem, the same construct was built but this time using the<br>
35S promoter with the ADH1 intron to prevent the expression of ZmlPT2 gene in<br>
Agrobacterium. Using this construct, Agrobacteria carrying the right co-integrate<br>
were obtained,<br>
Results:<br>
Arabidopsis calli in culture regenerate roots or shoots depending on auxin<br>
and cytokinin levels present in the medium. As a proof of concept, Arabidopsis<br>
hypocotyls were cultured on media containing increasing levels of auxin and<br>
cytokinin. Twenty-five different combinations of tZ and IBA concentrations, at 0,<br>
100, 300, 1000 and 3000 ng/ml for each hormone, were prepared and hypocotyls<br>
transferred to the media as described above. After 3 weeks in the culture room,<br>
pictures of 2 representative calli were taken for each hormone combination.<br>
Results indicated that a higher auxin:cytokinin ratio favored root formation, while a<br>
higher cytokinin:auxin ratio favored shoot formation.<br>
This experiment confirmed that root or shoot formation is influenced by the<br>
auxin/cytokinin ratio. Auxins have a root-inducing effect whereas cytokinins<br>
induce shoot formation. Based on these results, Arabidopsis calli over-expressing<br>
a cytokinin biosynthetic gene should not be able to develop roots on a medium<br>
containing only auxin. The functionality of this assay to characterize putative<br>
cvtokinin biosynthetic genes by using the Agrobacterium tumefaciens IPT (tmr)<br>
gene has been tested. Specifically, using the Gateway cloning system, two<br>
constructs were developed aimed at over-expressing either IPT as a cytokinin<br>
biosynthetic enzyme or GUS as a control. Three weeks after transformation of<br>
•A/£fc/S6jbs7s"cal]li™'"roots could"be observed on calli transformed with the 35S-GUSPINII<br>
construct but not on calli transformed with the 35S-IPT-PINII construct. In<br>
order to demonstrate that calli were efficiently transformed, in situ GUS staining<br>
was performed. Tissue transformed with 35S-GUS-PINII contained the GUS<br>
protein as revealed by the blue color observed after incubation in a solution<br>
containing the GUS substrate. These experiments validated the use of a highthroughput<br>
assay to test the putative com CK biosynthetic genes.<br>
The 35$-ADHI-ZmlPT2-Pinll construct was transformed into 10 day-old<br>
Arabidopsis calli which were transferred onto GM medium containing either auxin<br>
or both auxin and cytokinin. Bialaphos was added to select for transformed calli.<br>
Clear phenotypes could be observed 3 weeks after transformation. Control and<br>
35S-ADHI-ZmlPT2-PINII calli grew identically on medium containing both auxin<br>
and cytokinin. As expected, control calli transformed with the 35S-GUS-PINII<br>
construct were able to regenerate roots on medium containing only auxin. On the<br>
contrary, calli transformed with the 35S-ADH1-ZmlPT2-PINII construct, like calli<br>
transformed with the 35S-IPT-PINII construct, could not form any roots on this<br>
medium and some calli were even able to regenerate shoots. Given results of the<br>
preliminary experiment described above, this implies that these calli are<br>
synthesizing CK due to the expression of the ZmlPT2 gene. In turn this decreases<br>
the auxin:cytokinin ratio, which prevents root formation. These results support the<br>
conclusion that ZmlPT2 is a cytokinin biosynthetic gene.<br>
Example 9. isolation and Sequencing of the ZmlPT2 Promoter<br>
To isolate the promoter of the ZmlPT2 gene, a high-throughput Bacterial<br>
Artificial Chromosomes (BAG) screening process was used. Five positive clones<br>
were isolated by PCR screening based on the ZmlPT2 sequence. To confirm that<br>
the gene of interest was present in the bacterial chromosome, the BAC clones<br>
were cultured and propped. The BACs obtained were digested with H/ndlll and<br>
run on an agarose gel, which was used for a Southern blot. The blot was<br>
hybridized with a [a-32P]-dCTP labeled ZmlPT2 probe. Methods for the Southern<br>
blot are described above in Example 4.<br>
The Southern blot confirmed the presence of the ZmlPT2 sequence on all<br>
BAC clones isolated. Once checked, the BACs were subcloned in pBluescript<br>
after digestion with BamHI and H/ndlll. After ligation, chemically competent E. coli<br>
were transformed and grown on ampiciliin LB medium. Positive clones were then<br>
screened by a colony hybridization method. Colonies were transferred onto a<br>
nylon membrane, which was hybridized with a [a-32P]-dCTP ZmlPT2 probe to<br>
detect the clones containing the ZmlPT2 region on their plasmid. Finally, the<br>
colonies selected were prepped and the plasmid was sent for sequencing using<br>
5'OH-oriented primers. This allowed the upstream region of ZmlPT2 up to 1354<br>
bp to be sequenced. A BAG walking strategy was employed which gave 3280 bp<br>
of promoter sequence for this gene. The sequence for the ZmlPT2 promoter is set<br>
forth in SEQ ID NO: 75. A similar strategy was followed to identify the ZmlPTI<br>
promoter set forth in SEQ ID NO: 25.<br>
Promoter sequences for ZmlPT4 through ZmlPTS, and OslPT 1 through<br>
OslPT11, may be isolated in a similar manner. Sequences provided herein for<br>
ZmlPT4 (SEQ ID NO: 5), ZmlPTS (SEQ ID NO: 8), ZmlPT6 (SEQ ID NO: 11),<br>
ZmlPT? (SEQ ID NO: 14), ZmlPTS (SEQ ID NO: 17), and ZmlPTQ (SEQ ID NO:<br>
20), OslPTI (SEQ ID NO: 47), OslPT2 (SEQ ID NO: 44), OslPTS (SEQ ID NO:<br>
62), OslPT4 (SEQ ID NO: 64), OslPT5 (SEQ ID NO: 50), OslPT6 (SEQ ID NO:<br>
55), OslPT7 (SEQ ID NO: 53), OslPTS (SEQ ID NO: 40), OslPT9 (SEQ ID NO:<br>
60), OslPTI 0 (SEQ ID NO: 58), and OslPT11 (SEQ ID NO: 42) include<br>
appropriate upstream regions useful for characterization of functional promoter<br>
sequence.<br>
Example 10. Assayingfor IPT Activity<br>
A. Synthesis of Cvtokinin bv maize or rice IPT Sequences in Bacterial<br>
Culture Medium<br>
The ability of an IPT sequence of the invention to synthesize cytokinin is<br>
assayed in a bacterial culture medium in which cytokinin is known to be secreted.<br>
Enzyme activity in E. coll is measured.<br>
£ co// strain BL21-AI (Invitrogen) containing a T7 promoter::IPT sequence<br>
(IPT cloned in pDEST17 (Invitrogen)) is cultured for 4 h at 37°C and the<br>
accumulation of the protein is induced for 12 hours at 20°C in the presence of<br>
0.2% arabinose. The microorganisms are collected by centrifugation, and after<br>
Buffer A (25 mM Tris-HCI, 50 mM KCI, 5 mM (3-mercaptoethanol, 1 mM PMSF<br>
and 20 pg/ml of leupeptin) is added to an OD600 of 100, the E. co// are disrupted<br>
by freezing and thawing. The disrupted £ co// are then centrifuged for 10 minutes<br>
at 300,000 g followed by recovery of the supernatants. 10 ul of these<br>
supernatants are mixed with Buffer A containing 60 uM DMAPP, 5 uM [3H]AMP<br>
 (7±2"(3Bq7mmbl)"'ariiaTO'"'MM''MgCl2 followed by incubation for 30 minutes at<br>
25°C. Subsequently, 50 mM of Tris-HCI (pH 9) is added to this reaction liquid<br>
followed by the addition of calf intestine alkaline phosphatase to a concentration of<br>
2 units/30 pi and incubating for 30 minutes at 37°C to carry out a<br>
dephosphatization reaction. As a result of developing the reaction liquid by C18<br>
reversed-phase thin layer chromatography (mobile phase: 50% rnethanol) and<br>
detecting the reaction products by autoradiography, formation of isopentenyl<br>
adenosine is confirmed in the reaction liquids containing extracts of £ coli having<br>
T7::IPT sequence.<br>
It is further recognized that 3H-HMBPP (4-hydroxyl-3-methyl-2-(E)-butenyl<br>
diphosphate) could also be used as a substrate in the assay described above.<br>
See, for example, Krall et a/. (2002) FEBS Letters 527:318-8, herein incorporated<br>
by reference.<br>
B. Assay for DMAPP:ATP or APR or AMP isooentenvl<br>
Transferase activity<br>
DMAPP:ATP (or ADP or AMP) isopentenyl transferase activity is measured<br>
by the method described by Blackwell and Morgan (1991) FEBS Lett 76:10-12,<br>
with some modifications. The samples to be assayed are crude extracts and<br>
purified proteins of £ coll harboring the T7 promoter.:IPT sequence. Purified<br>
proteins are diluted to appropriate concentrations with dilution buffer (25 mM<br>
Tris-HC1, pH 7.5; 5 mM 2-mercaptoethanol; 0.2 mg ml"1 bovine serum albumin).<br>
Isopentenylation reactions are started by mixing samples with an equal volume of<br>
2x assay mixture containing 25 mM Tris-HC1 (pH 7.5), 10 mM MgCb, 5 mM 2-<br>
mercaptoethanol, 60 uM DMAPP, and 2 uM [2,8-3H]ATP (120 GBq mmol-1), [2,8-<br>
3H]ADP (118 GBq mmol-1), [2-3H]AMP (72 GBq mmol-1), or [2,8-3H]adenosine<br>
(143 GBq mmol-1). After incubation for an appropriate time, 1/2 volume of calf<br>
intestine alkaline phosphatase (CIAP) mix [0.5 Tris-HC1 (pH 9.0), 10 mM MgCI2,<br>
and 1,000 units ml-1 of CIAP (Takara Shuzo Co. Ltd., Otsu, Shiga, Japan)] is<br>
added and the mixtures are incubated at 37°C for 30 min. Then, 700 ul of ethyl<br>
acetate is added and the mixtures are vortexed. After centrifugation at 17,000xg<br>
for 2 min, the organic phase is recovered and washed twice with water. The<br>
organic phase is mixed with ten volumes of scintillant, ACSII (Amersham<br>
Pharmacia Biotech, Tokyo, Japan), and radioactivity levels are measured with a<br>
liquid scintillation counter. Recovery of [2,8-3H]isopentenyladenosine (iPA) is<br>
measured and is used to calculate the amounts of the products formed. The<br>
[2,8-3H]iPA is synthesized through isopentenylation of ATP by using purified IPT<br>
sequences, followed by CIAP treatment as described above. All assays are<br>
performed in duplicate and mean values are used for calculation.<br>
To determine the Km for ATP, purified protein (2 ng ml"1 in dilution buffer) is<br>
mixed with the same volume of a 2x assay mixture containing 25 mM Tris-HC1<br>
(pH 7.5), 10 mM MgCI2, 5 mM 2-mercaptoethanol, 0.4 mM DMAPP, and ATP (2-<br>
502 uM [2,8-3H]ATP, 1.22 MBq ml-1). To determine the Km for DMAPP, purified<br>
protein (2 ng ml-1) is mixed with the same volume of a 2x assay mixture containing<br>
25 mM Tris-HC1 (pH 7.5), 10 mM MgCI2, 5 mM 2-mercaptoethanol, 0.25-200 uM<br>
DMAPP, and 200 |jM [2,8-3]ATP (7.07 GBq mmol-1). After the mixture is<br>
incubated at 24°C for 0 min or 4 min, the reaction mixtures are treated with CIAP,<br>
and then extracted with ethyl acetate as described above. Values obtained at 0<br>
min are subtracted from those at 4 min, and the resulting differences are taken as<br>
enzyme activity.<br>
To confirm that the IPT sequences catalyzed the transfer of the isopenteny!<br>
moiety to ATP, ADP, or AMP, the reaction products are analyzed by HPLC and<br>
mass spectrometry. Briefly, crude extract prepared from IPTG-induced E. coli<br>
harboring the pDEST17-IPT plasmid is incubated with Ni-NTA agarose beads.<br>
After the beads have been washed thoroughly, they are re-suspended in a<br>
solution containing 25 mM Tris-HC1 (pH 7.5), 100 mM KC1, and 5 mM 2-<br>
mercaptoethanol. The bead pellets are mixed with an equal volume of a 2x assay<br>
mixture that contains I mM unlabeled ATP and 1 mM DMAPP, and incubated at<br>
25°C for 1 h with shaking. After a brief spin, the supernatant is recovered and<br>
separated into two portions, and one portion is treated with CIAP as described<br>
before. The supernatant with or without treatment with CIAP is mixed with three<br>
volumes of acetone. The mixture is incubated at -80°C for 30 min and centrifuged<br>
at 17,000xg for 30 min to remove the proteins. The supernatants are dried under<br>
vacuum, and the residues are dissolved in methanol. Aliquots are separated by<br>
HPLC with a Chemcobond ODS-W column (Chemco, Osaka, Japan), by using the<br>
following program: 20 mM KHaPC^t for 15 min, followed by linear gradient of 0%<br>
acetonitrile and 20 mM KHaP04 to 80% acetonitrile and 4 mM KHaPCvt over 30<br>
min. The fractions are collected and dried under vacuum, and the residues are<br>
resuspended in ethanol. After centrifugation to remove any possible salt<br>
precipitates, the solutions" are subjected to fast atom bombardment mass<br>
spectrornetry (JMS-SX102 or JEOL MStation, JEOL DATUM LTD., Tokyo, Japan).<br>
C. Assaying for Shoot and Root Regeneration<br>
Transformation of Arabidopsis callus is performed as follows. Selection for<br>
transformants is made using 3mg/L of bialaphos. Arabidopsis seeds are sterilized<br>
according to Koncz et a/. (1992) Methods in Arabidopsis Research, Sinapore,<br>
River Edge, N.J., World Scientific. Seeds are placed on GM medium and grown in<br>
continuous light at 23°C for 11 days. Hypocotyl segments are cut and placed on<br>
CIM for 8 days. Calli are soaked in a suspension of Agrobacterium (0.2 ODeoo) in<br>
AIM for 5 minutes. Most of the liquid is removed on the filter paper, and the<br>
Arabidopsis is placed on CIM culture medium and grown in continuous light at<br>
23°C for 2 days. The calli are washed thoroughly in WASHM medium and placed<br>
on GM+IBA or GM+Z+IBA medium and cultured for about 3 weeks. Selection for<br>
transformants is made on 3 mg/L of bialaphos.<br>
Media recipes for the transformation protocol discussed above are as<br>
follows. SOOOxCIM hormone mix comprises 2.5 mg/ml 2,4-D (Sigma Cat. No. D<br>
6679); 0.25 mg/ml kinetin (Sigma Cat no. K 0753); and, 5 mg/ml biotin dissolved in<br>
DMSO (Sigma Cat. No. B 3399. 500x vitamin mix comprises 50 g/l myo-inositol<br>
(Sigma Cat. No. I 3011); 10 g/l thiamine-HCI (Sigma Cat. No. T 3902); 0.5 g/l<br>
pyridoxine-HCl (Sigma Cat. No. P 8666); and, 0.5 g/l nicotinic acid (Sigma Cat.<br>
No. N0765). GM (germination medium) (for 1 liter) comprises 4.3 g MS medium<br>
salt base (Sigma Cat. No. M 5524); 10 g sucrose (Sigma Cat. No. S 8501); 2 ml<br>
500x vitamin mix; 10 ml 5% MES (adjusted to pH 5.7 with KOH) (Sigma Cat. No.<br>
M 2933); and, 3 g Phytagel (Sigma Cat. No. P 8169). The mixture is autoclaved<br>
and poured in Petri dishes. CIM (callus inducing medium) comprises 3.08 g<br>
Gamborg's B5 medium salt base (Sigma Cat. No. G 5768); 20 g glucose (Sigma<br>
Cat. No. G7528); 2 ml 500x vitamin mix; 10 ml 5% MES (adjusted to pH 5.7 with<br>
KOH); and, 3 g Phytagel. The mixture is autoclaved, cooled and 200 ul of CIM<br>
hormone mix is added. The mixture is then poured into Petri dishes. AIM<br>
(Agrobacterium infection medium) comprises CIM without Phytagel. WASHM<br>
(washing medium) comprises GM from which Phytagel has been omitted, plus<br>
100mg/l of sodium cefotaxime. GM+ (selection of transformed calli) comprises<br>
GM medium that was autoclaved with the following components add via filter: 1<br>
ml of 100 mg/ml cefotaxime (Sigma Cat. No. C 7039); 1 ml of 50 mg/ml of<br>
Ca"t7NoTC!"3416); and, 3 ml of 1 mg/ml Bialaphos. GM-HBA<br>
comprises the addition of 300 M' of 1 mg/ml indolebutyric acid (IBA) (Sigma Cat.<br>
No. I 7512) to the GM media described above. GM+IBA+Z comprises the addition<br>
of 300 Ml of 1 mg/ml IBA and 1 ml of 1 mg/ml trans-Zeatin (Z) (Sigma Cat. No. Z<br>
2753) to the GM media described above.<br>
In order to examine the function of IPT, the maize IPT sequences are first<br>
selected and introduced in Arabidopsis calli under the control of the 35S promoter.<br>
Call! transformed with a control vector will exhibit normal hormone responses:<br>
root formation in the presence of only an auxin and shoot formation in the<br>
presence of a cytokinin and an auxin. By contrast, calli transformed with 35S::IPT<br>
will regenerate shoots even in the absence of exogenously applied cytokinins or in<br>
the presence of a reduced concentration of exogenously applied cytokinins. In<br>
addition, modulation in cytokinin synthesis could be assayed for changes in either<br>
direction. Representative methods include cytokinin extraction,<br>
immunopurification, HPLC separation, and quantification by ELISA methods can<br>
be found, for example, in Faiss et a/. (1997) Plant J. 72:401-415. See, also,<br>
Werner et a/. (2001) PNAS 98:10487-10492) and Dewitte ef a/. (1999) Plant<br>
Physiol. 119:111-121.<br>
D, Assaying for DMAPP:tRNA isopentenvltransferase activity<br>
Undermodified tRNA is prepared by permanganate-treatment of yeast tRNA<br>
(type X, Sigma-Aldrich Japan, Tokyo, Japan) according to the method of Kline et<br>
a/. (1969) Biochemistry 8:4361-4371. Twenty microliters of purified protein<br>
samples (20ng (protein ml"1) in dilution buffer is mixed with the same volume of 2X<br>
tRNA isopentenyltransferase assay mixture (25 mM Tris-HCI, pH 7.5; 10mM<br>
MgCIa; 5 mM 2-mercaptoethanol; 0.67 uM [1-3H]DMAPP, 555 GBq mmor1; and<br>
567 A aeo units ml"1 undermodified tRNA), and incubated at 25°C for 30 min. After<br>
160 u of 0.4 M sodium acetate and 500 pi of ethanol is added and allowed to<br>
settle on ice for 10 minutes, the tRNA precipitates are recovered by centrifugation<br>
(17,000xg for 20 minutes), washed with 80% ETOH, and dissolved in 30 ul of<br>
distilled water. These are mixed with ten volumes of ACSII, and radioactivity<br>
levels are measured.<br>
'E)(arnple11. Maintaining or Increasing Seed Set During Stress<br>
Targeted overexpression of the IPT sequences of the invention to the<br>
developing female inflorescence will elevate cytokinin levels and allow developing<br>
maize seed to achieve their full genetic potential for size, minimize tip kernel<br>
abortion, and buffer seed set during unfavorable environments. Abiotic stress that<br>
occurs during kernel development in maize has been shown to cause reduction in<br>
cytokinin levels. Under stress conditions, it is likely that cytokinin biosynthesis<br>
activity is decreased and cytokinin degradation is increased (Brugiere et al, (2003)<br>
Plant Physiol. 132(3):1228-40). Consequently, in one non-limiting method, to<br>
maintain cytokinin levels in lag phase kernels, IPT genes could be ligated to<br>
control elements that: 1) are stress insensitive; 2) direct expression of structural<br>
genes predominantly to the developing kernels; and 3) preferentially drive<br>
expression of structural genes during the lag phase of kernel development.<br>
Promoters which target expression to related maternal tissues at or around<br>
anthesis may also be employed. Alternatively, a constitutive promoter could be<br>
employed.<br>
For example, immature maize embryos from greenhouse donor plants are<br>
bombarded with a plasmid containing a sequence, chosen from ZmlPT1-9 or<br>
OslPT1-11, operably linked to the Zag2.1 promoter (Schmidt et al. (1993) Plant<br>
Cell 5:729-737) and containing the selectable marker gene BAR (Wohlleben et al.<br>
(1988) Gene 70:25-37), which confers resistance to the herbicide Bialaphos.<br>
Alternatively, the selectable marker gene is provided on a separate plasmid.<br>
Transformation is performed as follows. Media recipes follow below.<br>
The ears are husked and surface-sterilized in 30% Clorox bleach plus 0.5%<br>
Micro detergent for 20 minutes, and rinsed two times with sterile water. The<br>
immature embryos are excised and placed embryo axis side down (scutellum side<br>
up), 25 embryos per plate, on 560Y medium for 4 hours and then aligned within<br>
the 2.5cm target zone in preparation for bombardment.<br>
A plasmid vector comprising the IPT sequence operably linked to a Zag2.1<br>
promoter is made. This plasmid DNA plus plasmid DNA containing a BAR<br>
selectable marker is precipitated onto 1.1 Mm (average diameter) tungsten pellets<br>
using a CaCI2 precipitation procedure as follows: 100ul prepared tungsten<br>
particles in water; 10 |j| (1 pg) DNA in Tris EDTA buffer (1 ug total DNA); 100 \J2.5 M CaC12; and, 10 (Jl 0.1 M spermidine.<br>
Ea"6H'""r'§a§e'nt"1§""a'cldea" sequentially to the tungsten particle suspension,<br>
while maintained on the multitube vortexer. The final mixture is sonicated briefly<br>
and allowed to incubate under constant vortexing for 10 minutes. After the<br>
precipitation period, the tubes are centrifuged briefly, liquid removed, washed with<br>
500 ml 100% ethanol, and centrifuged for 30 seconds. Again the liquid is<br>
removed, and 105 ul 100% ethanol is added to the final tungsten particle pellet.<br>
For particle gun bombardment, the tungsten/DNA particles are briefly sonicated<br>
and 10 pi spotted onto the center of each macrocarrier and allowed to dry about 2<br>
minutes before bombardment.<br>
The sample plates are bombarded at level #4 in particle gun #HE34-1 or<br>
#HE34-2. All samples receive a single shot at 650 PSI, with a total often aliquots<br>
taken from each tube of prepared particles/DNA.<br>
Following bombardment, the embryos are kept on 560Y medium for 2 days,<br>
then transferred to 560R selection medium containing 3 mg/liter Bialaphos, and<br>
subcultured every 2 weeks. After approximately 10 weeks of selection, selectionresistant<br>
callus clones are transferred to 288J medium to initiate plant<br>
regeneration. Following somatic embryo maturation (2-4 weeks), well-developed<br>
somatic embryos are transferred to medium for germination and transferred to the<br>
lighted culture room. Approximately 7-10 days later, developing plantlets are<br>
transferred to 272V hormone-free medium in tubes for 7-10 days until plantlets are<br>
well established. Plants are then transferred to inserts in flats (equivalent to 2.5"<br>
pot) containing potting soil and grown for 1 week in a growth chamber,<br>
subsequently grown an additional 1-2 weeks in the greenhouse, then transferred<br>
to classic 600 pots (1.6 gallon) and grown to maturity. Plants are monitored and<br>
scored for the maintenance or increase of seed set during an abiotic stress<br>
episode. In addition, transformants under stress will be monitored for cytokinin<br>
levels (as described in Example 5c) and maintenance of kernel growth.<br>
Bombardment medium (560Y) comprises 4.0 g/l N6 basal salts (SIGMA C-<br>
1416), 1.0 ml/I Eriksson's Vitamin Mix (1000X SIGMA-1511), 0.5 mg/l thiamine<br>
HCI, 120.0 g/l sucrose, 1.0 mg/l 2,4-D, and 2.88 g/l L-proline (brought to volume<br>
with D-l H2O following adjustment to pH 5.8 with KOH); 2.0 g/l Gelrite (added after<br>
bringing to volume with D-l HaO); and 8.5 mg/l silver nitrate (added after sterilizing<br>
the medium and cooling to room temperature). Selection medium (560R)<br>
comprises 4.0 g/l N6 basal salts (SIGMA C-1416), 1.0 ml/l Eriksson's Vitamin Mix<br>
(1000X SIGMA-1511), 0.5 mg/l thiamine HCI, 30.0 g/l sucrose, and 2.0 mg/l 2,4-D<br>
"(brought toWlume flftrTDtfftjb following adjustment to pH 5.8 with KOH); 3.0 g/l<br>
Gelrite (added after bringing to volume with D-l H2O); and 0.85 mg/l silver nitrate<br>
and 3.0 mg/l bialaphos(both added after sterilizing the medium and cooling to<br>
room temperature).<br>
Plant regeneration medium (288J) comprises 4.3 g/l MS salts (GIBCO<br>
11117-074), 5.0 ml/I MS vitamins stock solution (0.100 g nicotinic acid, 0.02 g/l<br>
thiamine HCL, 0.10 g/l pyridoxine HCL, and 0.40 g/l glycine brought to volume with<br>
polished D-l H2O) (Murashige and Skoog (1962) Physiol. Plant 15:473), 100 mg/l<br>
myo-inositof, 0.5 mg/l zeatin, 60 g/I sucrose, and 1.0 ml/I of 0.1 mM abscisic acid<br>
(brought to volume with polished D-l H2O after adjusting to pH 5.6); 3.0 g/l Gelrite<br>
(added after bringing to volume with D-l H2O); and 1.0 mg/l indoleacetic acid and<br>
3.0 mg/l bialaphos (added after sterilizing the medium and cooling to 60°C).<br>
Hormone-free medium (272V) comprises 4.3 g/l MS salts (GIBCO 11117-074), 5.0<br>
ml/l MS vitamins stock solution (0.100 g/l nicotinic acid, 0.02 g/l thiamine HCL,<br>
0.10 g/l pyridoxine HCL, and 0.40 g/! glycine brought to volume with polished D-l<br>
H2O), 0.1 g/l myo-inositol, and 40.0 g/l sucrose (brought to volume with polished<br>
D-l H2O after adjusting pH to 5.6); and 6 g/l bacto-agar (added after bringing to<br>
volume with polished D-l H2O), sterilized and cooled to 60°C.<br>
Example 12: Modulating Root Development<br>
For Agrobacterium-mediated transformation of maize with a plasmid<br>
designed to achieve post-transcriptional gene silencing (PTGS) with an<br>
appropriate promoter, the method of Zhao may be employed (U.S. Patent No.<br>
5,981,840, and PCT patent publication WO98/32326, the contents of which are<br>
hereby incorporated by reference). Briefly, immature embryos are isolated from<br>
maize and the embryos contacted with a suspension of Agrobacterium capable of<br>
transferring a DNA construct. Said construct may comprise the CRWAQ81 rootpreferred<br>
promoter::ADH intron promoter operably linked to a hairpin structure<br>
made from the coding sequence of any one of the ZmlPT1-9 or OsfPT1-11<br>
polynucleotides of the invention. Other useful constructs may comprise a hairpin<br>
construct targeting the promoter of any one of the ZmlPT1-9 or OslPT1-11<br>
polynucleotides of the invention. (Aufsatz et al. (2002) PNAS 99 (Suppl. 4):16499-<br>
16506; Mette et al. (2000) EMBO J 19(19):5194-5201) The construct is<br>
ransferred to at least one cell of at least one of the immature embryos (step 1: the<br>
nfection step). In this step the immature embryos are immersed in an<br>
•Mgrooacra/70/W fO^p^rfslon toT the initiation of inoculation. The embryos are cocultured<br>
for a time with the Agrobacterium (step 2: the co-cultivation step); this<br>
may take place on solid medium. Following this co-cultivation period an optional<br>
"resting" step is contemplated. In this resting step, the embryos are incubated in<br>
the presence of at (east one antibiotic known to inhibit the growth of<br>
Agrobacterium without the addition of a selective agent for plant transformants<br>
(step 3: resting step). Next, inoculated embryos are cultured on medium<br>
containing a selective agent; growing, transformed callus is recovered (step 4: the<br>
selection step). The callus is then regenerated into plants (step 5: the<br>
regeneration step).<br>
Plants are monitored and scored for a modulation in root development. The<br>
modulation in root development includes monitoring for enhanced root growth of<br>
one or more root parts including the primary root, lateral roots, adventitious roots,<br>
etc. Methods of measuring such developmental alterations in the root system are<br>
known in the art. See, for example, U.S. Application No. 2003/0074698 and<br>
Werner et al. (2001) PNAS 18:10487-10492, both of which are herein incorporated<br>
by reference.<br>
Example 13. Modulating Senescence of a Plant<br>
A DNA construct comprising any of the ZmlPT1-9 or OslPT1-11<br>
polynudeotides operably linked to a constitutive promoter, a root-preferred<br>
promoter, or a senescence-activated promoter, such as SAG12 (Gan et al. (1995)<br>
Science 270:5244, Genbank Ace. No. U37336) is introduced into maize plants as<br>
outlined in Zhao et al. (1998) Maize Genetics Corporation Newsletter 72:34-37,<br>
herein incorporated by reference.<br>
For example, maize plants comprising the IPT sequence operably linked to<br>
the SAG12 promoter are obtained. As a control, a non-cytokinin-related construct<br>
is also introduced into maize plants using the transformation method outlined<br>
above. The phenotypes of transgenic maize plants having an elevated level of the<br>
IPT polypeptide are studied. For example, plants can be monitored for an<br>
improved vitality, shelf and vase life, and improved tolerance against infection.<br>
Plants could also be monitored for delayed senescence under various<br>
environmental1 stresses including, for example, flooding which normally results in<br>
leaf chlorosis, necrosis, defoliation, cessation of growth and reduction in yield.<br>
"fcxamdltm: saymitrEmtfrtto Transformation<br>
Soybean embryos are bombarded with a plasmid containing the IPT<br>
sequence operably linked to a ubiquitin promoter as follows. To induce somatic<br>
embryos, cotyledons, 3-5 mm in length dissected from surface-sterilized, immature<br>
seeds of the soybean cultivar A2872, are cultured in the light or dark at 26°C on<br>
an appropriate agar medium for six to ten weeks. Somatic embryos producing<br>
secondary embryos are then excised and placed into a suitable liquid medium.<br>
After repeated selection for clusters of somatic embryos that multiplied as early,<br>
globular-staged embryos, the suspensions are maintained as described below.<br>
Soybean embryogenic suspension cultures can be maintained in 35 ml<br>
liquid media on a rotary shaker, 150 rpm, at |26°C with florescent lights on a<br>
16:8 hour day/night schedule. Cultures are subcultured every two weeks by<br>
inoculating approximately 35 mg of tissue into 35 ml of liquid medium.<br>
Soybean embryogenic suspension cultures may then be transformed by the<br>
method of particle gun bombardment (Klein et al. (1987) Nature (London)<br>
327:70-73, U.S. Patent No. 4,945,050). A Du Pont Biolistic PDS1000/HE<br>
instrument (helium retrofit) can be used for these transformations.<br>
A selectable marker gene that can be used to facilitate soybean<br>
transformation is a transgene composed of the 35S promoter from Cauliflower<br>
Mosaic Virus (Odelf et al. (1985) Nature 313:810-812), the hygromycin<br>
phosphotransferase gene from plasmid pJR225 (from E. coir, Gritz et al. (1983)<br>
Gene 25:179-188), and the 3' region of the nopaline synthase gene from the<br>
T-DNA of the Ti plasmid of Agrobacterium tumefaciens. The expression cassette<br>
comprising the IPT sequence operably linked to the ubiquitin can be isolated as a<br>
restriction fragment This fragment can then be inserted into a unique restriction<br>
site of the vector carrying the marker gene.<br>
To 50 ul of a 60 mg/ml 1 urn gold particle suspension is added (in order):<br>
5 Ml DNA (1 ug/pl), 20 ul spermidine (0.1 M), and 50 ul CaCl2 (2.5 M). The<br>
particle preparation js then agitated for three minutes, spun in a microfuge for<br>
10 seconds and the supernatant removed. The ONA-coated particles are then<br>
washed once in 400 ul 70% ethartol and resuspended in 40 pi of anhydrous<br>
ethanol. The DNA/particle suspension can be sonicated three times for<br>
one second each. Five microliters of the DNA-coated gold particles are then<br>
loaded on each macro carrier disk.<br>
of a two-week-old suspension culture is placed<br>
in an empty 60x15 mm petri dish and the residual liquid removed from the tissue<br>
with a pipette. For each transformation experiment, approximately 5-10 plates of<br>
tissue are normally bombarded. Membrane rupture pressure is set at 1100psi,<br>
and the chamber is evacuated to a vacuum of 28 inches mercury. The tissue is<br>
placed approximately 3.5 inches away from the retaining screen and bombarded<br>
three times. Following bombardment, the tissue can be divided in half and placed<br>
back into liquid and cultured as described above.<br>
Five to seven days post bombardment, the liquid media may be exchanged<br>
with fresh media, and eleven to twelve days post-bombardment with fresh media<br>
containing 50 mg/ml hygromycin. This selective media can be refreshed weekly.<br>
Seven to eight weeks post-bombardment, green, transformed tissue may be<br>
observed growing from untransformed, necrotic embryogenic clusters. Isolated<br>
green tissue is removed and inoculated into individual flasks to generate new,<br>
clonally propagated, transformed embryogenic suspension cultures. Each new<br>
line may be treated as an independent transformation event. These suspensions<br>
can then be subcultured and maintained as clusters of immature embryos or<br>
regenerated into whole plants by maturation and germination of individual somatic<br>
embryos.<br>
Example 15. Sunflower Meristem Tissue Transformation<br>
Sunflower meristem tissues are transformed with an expression cassette<br>
containing the IPT sequence operably linked to a ubiquitin promoter as follows<br>
(see also European Patent Number EP 0 486233, herein incorporated by<br>
reference, and Malone-Schoneberg ef a/. (1994) Plant Science 103:199-207).<br>
Mature sunflower seed (Hetianthus annuus L) are dehulled using a single wheathead<br>
thresher. Seeds are surface sterilized for 30 minutes in a 20% Clorox bleach<br>
solution with the addition of two drops of Tween 20 per 50 ml of solution. The<br>
seeds are rinsed twice with sterile distilled water.<br>
Split embryonic axis expiants are prepared by a modification of procedures<br>
described by Schrammeijer et al. (Schrammeijer ef a/.(1990) Plant Cell Rep. 9:55-<br>
60). Seeds are imbibed in distilled water for 60 minutes following the surface<br>
sterilization procedure. The cotyledons of each seed are then broken off,<br>
producing a clean fracture at the plane of the embryonic axis. Following excision<br>
of the root tip, the expiants are bisected longitudinally between the primordial<br>
placed, cut surface up, on GBA medium consisting of<br>
Murashige and Skoog mineral elements (Murashige et al. (1962) Physiol. Plant,<br>
15: 473-497), Shepard's vitamin additions (Shepard (1980) in Emergent<br>
Techniques for the Genetic Improvement of Crops (University of Minnesota Press,<br>
St. Paul, Minnesota), 40 mg/l adenine sulfate, 30 g/l sucrose, 0.5 mg/l 6-benzylaminopurine<br>
(BAP), 0.25 mg/l indole-3-acetic acid (IAA), 0.1 rng/l gibberellic acid<br>
(GAa), pH 5.6, and 8 g/l Phytagar.<br>
The explants are subjected to microprojectile bombardment prior to<br>
Agrobacterium treatment (Sidney et al (1992) Plant Mol. Biol. 18:301-313). Thirty<br>
to forty explants are placed in a circle at the center of a 60 X 20 mm plate for this<br>
treatment. Approximately 4.7 mg of 1.8 mm tungsten microprojectiles are<br>
resuspended in 25 ml of sterile TE buffer (10 mM Tris HCI, 1 mM EDTA, pH 8.0)<br>
and 1.5 ml aliquots are used per bombardment. Each plate is bombarded twice<br>
through a 150 mrn nytex screen placed 2 cm above the samples in a PDS 1000®<br>
particle acceleration device.<br>
Disarmed Agrobacterium tumefaciens strain EHA105 is used in all<br>
transformation experiments. A binary plasmid vector comprising the expression<br>
cassette that contains the IPT gene operably linked to the ubiquitin promoter is<br>
introduced into Agrobacterium strain EHA105 via freeze-thawing as described by<br>
Holsters et al. (1978) Mol. Gen. Genet. 163:181-187. This plasmid further<br>
comprises a kanamycin selectable marker gene (i.e., nptfl). Bacteria for plant<br>
transformation experiments are grown overnight (28°C and 100 RPM continuous<br>
agitation) in liquid YEP medium (10 gm/1 yeast extract, 10 gm/l Bactopeptone, and<br>
5 gm/l NaCI, pH 7.0) with the appropriate antibiotics required for bacterial strain<br>
and binary plasmid maintenance. The suspension is used when it reaches an<br>
ODeoo of about 0.4 to 0.8. The Agrobacterium cells are pelleted and<br>
resuspended at a final ODgoo of 0.5 in an inoculation medium comprised of 12.5<br>
mM MES pH 5.7, 1 gm/l NH4CI, and 0.3 gm/l MgSO4-<br>
Freshly bombarded explants are placed in an Agrobacterium suspension,<br>
mixed, and left undisturbed for 30 minutes. The explants are then transferred to<br>
GBA medium and co-cultivated, cut surface down, at 26°C and 18-hour days.<br>
After three days of co-cultivation, the explants are transferred to 374B (GBA<br>
medium lacking growth regulators and a reduced sucrose level of 1%)<br>
supplemented with 250 mg/l cefotaxime and 50 mg/l kanamycin sutfate. The<br>
explants are cultured for two to five weeks on selection and then transferred to<br>
fresh 374B medium lacking kanamycin for one to two weeks of continued<br>
development. Explants with differentiating, antibiotic-resistant areas of growth that<br>
have not produced shoots suitable for excision are transferred to GBA medium<br>
containing 250 mg/l cefotaxime for a second 3-day phytohormone treatment. Leaf<br>
samples from green, kanamycin-resistant shoots are assayed for the presence of<br>
NPTII by ELISA and for the presence of transgene expression by assaying for<br>
cytokinin synthesis activity. Such assays are described elsewhere herein.<br>
NPTII-positive shoots are grafted to Pioneer® hybrid 6440 in vitro-grown<br>
sunflower seedling rootstock. Surface sterilized seeds are germinated in 48-0<br>
medium (half-strength Murashige and Skoog salts, 0.5% sucrose, 0.3% geJrite, pH<br>
5.6) and grown under conditions described for explant culture. The upper portion<br>
of the seedling is removed, a 1 cm vertical slice is made in the hypocotyl, and the<br>
transformed shoot inserted into the cut. The entire area is wrapped with parafilm<br>
to secure the shoot. Grafted plants can be transferred to soil following one week<br>
of in vitro culture. Grafts in soil are maintained under high humidity conditions<br>
followed by a slow acclimatization to the greenhouse environment. Transformed<br>
sectors of TO plants (parental generation) maturing in the greenhouse are<br>
identified by NPTII ELISA and/or by cytokinin synthesis activity analysis of leaf<br>
extracts while transgenic seeds harvested from NPTII-positive TO plants are<br>
identified by cytokinin synthesis activity analysis of small portions of dry seed<br>
cotyledon.<br>
Example 16. Variants of IPT<br>
A. Variant Nucleotide Sequences ofZmlPT1-9 and OslPT1-11 (SEQ ID<br>
NO: 1, 3, 4, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24, 26, 28, 40, 42, 44,<br>
45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76)<br>
That Do Not Alter the Encoded Amino Acid Sequence<br>
The ZmlPT1-9 or OslPT1-11 nucleotide sequences set forth in SEQ ID NO:<br>
1, 3, 4, 5, 7, 8, 10, 11,13,14,16, 17,19, 20, 21, 22, 24, 26, 28, 40,42, 44,45, 47,<br>
48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76 are used to<br>
generate variant nucleotide sequences having the nucleotide sequence of the<br>
open reading frame with about 70%, 75%, 80%, 85%, 90%, and 95% nucleotide<br>
sequence identity when compared to the corresponding starting unaltered ORF<br>
nucleotide sequence. These functional variants are generated using a standard<br>
"coddn table. While the nucleotide sequence of the variant is altered, the amino<br>
acid sequence encoded by the open reading frame does not change.<br>
8. Variant Amino Acid Sequences of ZmlPTI-9 and OslPT1-11<br>
Variant amino acid sequences of ZmlPT1-9 and Os!PT1-11 are generated,<br>
tn this example, one or more amino acids are altered. Specifically, the open<br>
reading frame set forth in SEQ ID NO: 1, 3, 4, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19,<br>
20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55. 56, 58, 60, 62, 64, 65,<br>
69, 70, 71, 72, 73, 74, or 76 is reviewed to determine the appropriate amino acid<br>
alteration. The selection of an amino acid to change is made by consulting a<br>
protein alignment with orthologs and other gene family members from various<br>
species. See Figure 1 and/or Figure 10. An amino acid is selected that is<br>
deemed not to be under high selection pressure (not highly conserved) and which<br>
is rather easily substituted by an amino acid with similar chemical characteristics<br>
(i.e., similar functional side-chain). Assays as outlined in Example 10 may be<br>
followed to confirm functionality. Variants having about 70%, 75%, 80%, 85%,<br>
90%, or 95% nucleic acid sequence identity to each of SEQ ID NO: 1, 3,4, 5, 7, 8,<br>
10, 11, 13,14, 16,17,19, 20, 21, 22, 24, 26, 28, 40, 42, 44, 45,47, 48, 50, 51, 53,<br>
55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, and 76 are generated using this<br>
method.<br>
B. Additional Variant Amino Acid Sequences of ZmlPTI-9<br>
andOslPT1-11<br>
In this example, artificial protein sequences are created having 80%, 85%,<br>
90%, and 95% identity relative to the reference protein sequence. This latter effort<br>
requires identifying conserved and variable regions from the alignment set forth in<br>
Figure 1 and/or Figure 10 and then the judicious application of an amino acid<br>
substitutions table. These parts will be discussed in more detail below.<br>
Largely, the determination of which amino acid sequences are altered is<br>
made based on the conserved regions among the IPT proteins or among the other<br>
IPT polypeptides. See Figures 1 and 10. Based on tine sequence alignment, the<br>
various regions of the IPT polypeptides that can likely be altered can be<br>
determined. It is recognized that conservative substitutions can be made in the<br>
conserved regions without altering function, in addition, one of skill will<br>
understand that functional variants of the IPT sequence of the invention can have<br>
minor non-conserved amino acid alterations in the conserved domain.<br>
Artificial protein sequences are then created that are different from the<br>
original in the intervals of 80-85%, 85-90%, 90-95%, and 95-100% identity.<br>
Midpoints of these intervals are targeted, with liberal latitude of plus or minus 1%,<br>
for example. The amino acids substitutions will be effected by a custom Perl<br>
script. The substitution table is provided below in Table 8.<br>
(Table Removed)<br>
First, any conserved amino acids in the protein that should not be changed<br>
are identified and "marked off' for insulation from the substitution. The start<br>
methionine will of course be added to this list automatically. Next, the changes<br>
are made.<br>
H, C, and P are not changed. The changes will occur with isoleucine first,<br>
sweeping N-terminal to C-terminal. Then leucine, and so on down the list until the<br>
desired target is reached. Interim number substitutions can be made so as not to<br>
cause reversal of changes. The list is ordered 1-17, so start with as many<br>
isoleucine changes as needed before leucine, and so on down to methionine.<br>
Clearly many amino acids will in this manner not need to be changed. L, I and V<br>
will involve a 50:50 substitution of the two alternate optimal substitutions.<br>
The variant amino acid sequences are written as output. Perl script is used<br>
to calculate the percent identities. Using this procedure, variants of ZmlPT1-9 and<br>
OslPT1-11 are generating having about 82%, 87%, 92%, and 97% amino acid<br>
identity to the starting unaltered ORF nucleotide sequence of SEQ ID NO: 1, 3, 4,<br>
5, 7, 8, 10. 11,13, 14,16, 17, 19,20,21,22,24,26,28,40,42,44,45,47,48,50,<br>
51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76.<br>
Example 17. Characterization of Rice IPT Sequences<br>
Eleven putative rice ipt sequences were identified which comprise deduced<br>
amino acid sequences showing similarity to the Arabidopsis and Petunia IPT<br>
proteins. Figure 10 provides an alignment of the amino acid sequences<br>
corresponding to Arabidopsis IPT proteins (AtlPT), the petunia IPT protein (Sho)<br>
and rice putative IPT proteins (OslPT). Asterisks indicate the positions of amino<br>
acids conserved in most IPT proteins and following the consensus sequence<br>
GxTxxGK[ST]xxxxxIVLI]xxx)c<xxivlij qx></xxivlij>
ID NO: 32) (where x denotes any amino acid residue, [ ] any one of the amino<br>
acids shown in [ ], and x{m,n} m to n amino acid residues in number) (Takei et at.<br>
(2001) J. Biol. Chem. 276:26405-26410).<br>
The presence of putative ATP/GTP-binding site (P-loop) motif (prosite<br>
PS00017: consensus [AGJ-x(4)-G-K-[STj), (SEQ ID NO: 69) is underlined. This<br>
domain is found at about amino acids 128-135 of SEQ ID NO: 54; at about amino<br>
acids 59-66 of SEQ ID NO: 66: at about amino acids 59-66 of SEQ ID NO: 63; at<br>
about amino acids 40-47 of SEQ ID NO: 61; at about amino acids 320-327 of SEQ<br>
ID NO: 43; at about amino acids 22-29 of SEQ ID NO: 49; at about amino acids<br>
315-322 of SEQ ID NO: 59; at about amino acids 32-39 of SEQ ID NO: 57; at<br>
about amino acids 41-48 of SEQ ID NO: 41; at about amino acids 25-32 of SEQ<br>
ID NO: 46; and, at about amino acids 37-44 of SEQ ID NO: 52. The presence of a<br>
putative tRNA isopentenyltransferase domain (PF01715) was found at about<br>
amino acids 59-348 of SEQ ID NO: 57 and about amino acids 69-352 of SEQ ID<br>
NO: 41.<br>
The Align X program was used on default settings to determine the overall<br>
amino acid sequence identity for the various rice IPT sequences compared with<br>
known Arabidopsis IPT sequences. Table 9 summarizes these results. Table 10<br>
provides polypeptides that share homology to the rice IPT sequences. Such<br>
(Table Removed)<br>
Example 18. IPT Activity Assay<br>
Assays were conducted to test the ability of a protein encoded by a<br>
sequence of the invention to synthesize cytokinin in a bacterial culture medium.<br>
The results confirmed that sequences of the invention encode proteins with<br>
128<br>
"isopentenyltransferase activity. The reaction catalyzed by Agrobacterium ipt is<br>
shown in Akiyoshi et al. (1984) PNAS 81(19):5994-5998.<br>
The IPT assay protocol was adapted from the following references:<br>
Kakimoto, T. (2001) Identification of plant biosynthetic enzymes as dimethylallyl<br>
diphosphate: ATP/ADP isopentenyltransferases, Plant Cell Physio! 42: 677-685.<br>
Sakakibara, H., and Takei, K. (2002) Identification of Cytokinin Biosynthesis<br>
Genes in Arabidopsis: A Breakthrough for Understanding the Metabolic Pathway<br>
and the Regulation in Higher Plants, J. Plant Growth Reguf. 21:17-23. Sakano, Y.,<br>
Okada, Y., Matsunaga, A., Suwama, T., Kaneko, T., Ito, K., Noguchi, H., and Abe,<br>
I. (2004) Molecular cloning, expression, and characterization of adenylate<br>
isopentenyltransferase from hop (Humulus lupulus L), Phytochemistry 65:2439-<br>
2446.<br>
The ZmlPT2 gene was amplified using gene-specific primers with<br>
appropriate Ndel and Notl restriction site extensions and cloned into pET28a (Nterminal<br>
tag) or pETSOb (C-terminal tag) digested by Ndel and Notl. The sequence<br>
of the resulting plasmid was verified by sequencing of the His-tag translational<br>
fusion with ZmlPT2, and BL21-Star™ E. coll competent cells (Invitrogen™) were<br>
transformed with pET28a-ZmlPT2 and pET30b-Zm(PT2. Similarly, The tzs IPT<br>
gene from Agrobacterium tumefaciens was cloned into pET28a to yield a plasmid<br>
for transformation of Rosetta2(DE3)pLysS.<br>
Recombinant his-tagged proteins were purified using a TALON™ column (BD<br>
Biosciences) according to the instructions provided by the manufacturer. Purified<br>
protein samples were used to determine Dimethylallyl diphosphate<br>
(DMAPP)::AMP and DMAPP::ATP isopentenyl transferase activities using the<br>
following protocol:<br>
« Each purified protein extract was incubated in a reaction mixture containing<br>
12.5 mM Tris-HCI (pH 7.5), 37.5 mM KCI, 5 mM MgCI2, 1 mM DMAPP and<br>
1mM AMP or ATP for 2 hours at 30°C. The reaction was stopped by boiling<br>
the samples for 5 minutes.<br>
• Half of the reaction mixture was treated with calf intestine alkaline<br>
phosphatase (CAIP) by adding one volume of 2 x CAIP reaction buffer<br>
(0.45M Tris-HCI pH 9, 10 mM MgC!2, 1000 unit of CAlP7ml) and incubating<br>
for 1 hour at 37°C.<br>
• The reaction products were separated using reversed phase HPLC (Agilent<br>
1100 system with diode-array-detector) using a C18-ODS2 column<br>
 (Phenomenex) ana a separation protocol using 0.1 M acetic acid pH 3.3<br>
(Buffer A) and acetonitrile (Buffer B) as follows:<br>
o 100% buffer A for 15 minutes,<br>
o linear gradient from 100% buffer A and 0% buffer B to 20% buffer A<br>
and 80% buffer B over 35 minutes.<br>
UV absorbance was monitored at 280nm. Product retention times were<br>
compared to standards obtained from Sigma or OlChemlm.<br>
The recombinant Tzs and ZmlPT2 proteins were first used to determine<br>
DMAPP::AMP isopentenyl transferase activity. Figure 12A and Figure 12B show<br>
HPLC chromatograms obtained for one of the substrates of the reaction, 5'-AMP<br>
(Sigma), and the expected product isopentenyladenosine S'-monophosphate<br>
(iPMP) (OlChemlm). The chromatogram obtained with the IPT (tzs) protein shows<br>
that almost all 5'-AMP substrate has been converted to iPMP (Figure 12C).<br>
Similarly, the chromatogram obtained with ZmlPT2 purified protein shows that the<br>
enzyme is able to convert 5'-AMP to iPMP but with a lower efficiency than does<br>
Agrobacterium IPT since not all the 5'-AMP has been converted (Figure 12D).<br>
Treatment of reaction products with calf intestine alkaline phosphatase<br>
(CAIP) and chromatography using HPLC confirmed the identity of the reaction<br>
product iPMP. Figure 13A and 13B show chromatograms obtained with Adenosine<br>
(Ado) (Sigma) and isopentenyladenosine (iPAR) (Sigma). As expected, after<br>
dephosphorylation of the product of each reaction, iPMP was transformed to<br>
isopentenyladenosine (iPAR) (Figure 13C and 13D) whereas remaining 5'-AMP<br>
was transformed to Ado (Figure 13D). This confirms that ZmlPT2 can metabolize<br>
5'-AMP and DMAPP into iPMP.<br>
Determination of DMAPP::ATP activity was carried out using the same<br>
reaction buffer but replacing S'-AMP by 5-ATP in the reaction mixture. Figure 14A<br>
shows the chromatogram obtained with 5-ATP. If ZmlPT2 is able to catalyze the<br>
transfer of DMAPP onto 5'ATP, the resulting product should create iPTP. The<br>
chromatogram of Figure 14B shows that all 5'-ATP was metabolized into iPTP by<br>
ZmlPT2, suggesting that ZmlPT2 uses 5'-ATP with higher efficiency than 5'-AMP.<br>
The reaction product was treated with CAIP to ascertain its identity. Such<br>
treatment should yield iPAR. After separation by HPLC, the chromatogram was<br>
compared to the chromatogram of an iPAR standard (Figure 14C). After treatment,<br>
the reaction product was transformed to iPAR (Figure 14D) therefore confirming<br>
that ZmlPT2 can metabolize 5-ATP and DMAPP into iPTP. Altogether these<br>
results prove that ZmlPT2 is a cytokinin biosynthetic enzyme preferentially using<br>
5'-ATP as a substrate.<br>
Similar experiments established that 5'-ADP is also a suitable substrate for<br>
the encoded enzyme. Taken together, these results prove that ZmlPT2 is a<br>
cytokinin biosynthetic enzyme preferentially using 5'-ATP as a substrate. Future<br>
experiments will determine the kinetic properties for each substrate using a<br>
purified ZmlPT2 protein.<br>
Example 19. Detection of the ZmlPT2 protein in developing kernels.<br>
In order to study further the expression pattern of the ZmlPT2 protein,<br>
polyclonal antibodies were used for a Western blot experiment. Poiyclonal<br>
antibodies were raised in rabbit against purified N-terminal His-tagged<br>
recombinant ZmlPT2 protein. Fifteen micrograms of proteins extracted from whole<br>
kernels harvested at different days after pollination (DAP) were run using SDSPAGE<br>
and blotted on a PVDF membrane. ZmlPT2 proteins were detected using<br>
the method of Laemmli (Nature 227:680-685, 1970) with anti-ZmlPT2 polyclonal<br>
antibodies as primary antibodies and anti-rabbit IgG antibodies raised in goat<br>
conjugated to an alkaline phosphatase as secondary antibodies.<br>
Figure 15 shows that ZmlPT2 protein levels increase from 0 to 10 DAP,<br>
peak at 10 DAP, then decrease from 10 DAP to 15 DAP and stay approximately<br>
constant thereafter. This is in agreement with Northern blot results showing that<br>
expression of the gene peaks around 10 DAP where it is strong in the pedicel and<br>
the endosperm. Although total expression of the gene decreases thereafter,<br>
expression levels remain high in the pedicel at later stages. ZmlPT2 protein levels<br>
in kernels were very high compared to other organs. Results suggest that the<br>
cytokinin activity of ZmlPT2 protein in kernels is most likely controlled at the<br>
transcriptional level. The Western blot analysis of protein levels in kernels<br>
suggests that the antibodies are very specific to ZmlPT2. Antibodies, together with<br>
in situ hybridization, will be very useful in determining the precise site of<br>
expression of the gene during kernel development.<br>
Example 20. Ectopic overexpression of ZmlPT2 in transgenic Arabidopsis.<br>
Previous examples describe the overexpression of ZmiPT2 in Arabidopsis<br>
calli. In order to study the effects of the overexpression of ZmlPT2 at the whole<br>
plant level, Arabidopsis plants were transformed with an Agmbacterium<br>
tumeraciens strain containing a plasmid comprising the construct 35S-Adhl-<br>
ZmlPT2-Pinll with the bar herbicide resistance gene as a marker. (Tho'mpson et<br>
al. (1987) EMBO J 6(9):2519-2523; White et al. (1990) Nucleic Acids Res.<br>
18(4): 1062) The simplified Abrabidopsis transformation protocol (Clough and Bent<br>
(1998) Plant J. 16:735-743) was used. Seeds were sown in flats containing soil<br>
and incubated for 2 days at 4°C to optimize germination. After 10 days in the<br>
greenhouse, transformants were selected by spraying the seedlings daily for 5<br>
days with a 1/1000 dilution of Finale™ herbicide.<br>
After selection, several plants resistant to the herbicide treatment were<br>
identified. Some plants appeared small and dark green compared to others. Leaf<br>
greenness is linked to cytokinin levels, suggesting that the dark green transformed<br>
plants have elevated levels of cytokinin. Some transgenic plants appeared more<br>
affected than others, possibly linked to the level of expression of the transgene<br>
which is known to be variable depending on position effects related to insertion in<br>
the genome.<br>
At an early stage of development, some transgenic plants showed signs of<br>
anthocyanin accumulation in leaves compared to non transgenic plants. Some<br>
transgenics had highly serrated leaves compared to wild-type Arabidopsis. This<br>
phenotype has previously been reported in Arabidopsis plants over-expressing the<br>
Agrobacterium ipt gene (van der Graaff et al. (2001) Plant Growth Regul.<br>
34(3):305-315). High levels of cytokinin are often detrimental to plant growth (van<br>
der Graaff et al., 2001) and some transgenic plants appeared to struggle in their<br>
development compared to control plants. Some transgenics appeared to have a<br>
decreased apical dominance in inflorescence stems compared to controls, which<br>
was previously reported in Arabidopsis and tobacco plants with high levels of<br>
cytokinins (van der Graaff et al., 2001; Crozier et al. (2000) Biosynthesis of<br>
hormones and ellicitor molecules. In Biochemistry and molecular biology of plants,<br>
B. Buchanan. W. Gruissem, and R.L. Jones, eds (Rockville, Maryland: American<br>
Society of Plant Biologists), pp. 850-929)<br>
Some transgenics appeared to have a "bushy" phenotype, most likely due<br>
to a larger number of leaves resulting from decreased apical dominance. Some<br>
plants also had a poor seed set due to the absence of siliques or smaller siliques<br>
with few seeds. They also displayed anthocyanin accumulation in leaves and<br>
along inflorescence stems. Plants often displayed serrated cauline leaves. The<br>
most extreme phenotype was a transgenic plant with a rosette of approximately 5<br>
mm in diameter with very small curly leaves showing signs of anthocyanin<br>
accumulation. The plant was able to flower but never yielded seeds, it also<br>
displayed an unusual abundance of large trichomes. Curly leaf phenotype was<br>
previously described in tobacco with higher cytokinin levels (Crozier et al., 2000).<br>
Thus, over-expression of the protein in Arabidopsis further confirmed the<br>
protein's function by creating a range of phenotypes in agreement with previous<br>
attempts to over-express the IPT gene in Arabidopsis and tobacco (Van der Graaff<br>
at al., 2001; Crozier et al., 2000). The phenotypes observed in several<br>
independent transgenic plants are consistent with a phenotype of cytokinin<br>
accumulation, confirming that ZmlPT2 is a cytokinin biosynthetic enzyme.<br>
Example 21. Determining gene function through Mu tagging.<br>
Gene function can be further confirmed and described by the study of<br>
mutants in which transcription and/or translation of the sequence of interest is<br>
disrupted. In certain embodiments this is accomplished through use of methods<br>
disclosed in US 5,962,764. The Trait Utility Sytstem for Corn (TUSC) is a<br>
proprietary resource for selecting gene-specific transposon insertions from a<br>
saturated collection of maize mutants created using the Mutator transposable<br>
element system. For example, effect of the ZmlPT sequences of the invention on<br>
traits such as plant sink strength may be investigated. The following methods<br>
were applied to identify and characterize a TUSC mutant for ZmlPT2.<br>
A 1495bp genomic sequence was supplied for TUSC screening to identify<br>
germinal Mutator insertions in the maize IPT2 gene (ZmlPT2). This working<br>
annotation of the gene contained a 966bp open reading frame (ORF; nt83-1048)<br>
that is uninterrupted by introns.<br>
Primary screening against TUSC DNA Pools was initiated with two<br>
ZmlPT2-specific primers (PHN79087 and PHN79088), each in combination with<br>
the Mutator terminal inverted repeat (TIR) primer as described in US 5,962,764.<br>
Primer sequences are listed below and provided as SEQ ID Nos: 82-84,<br>
respectively.<br>
PHN79087 zmIPT2-F 5'&gt; TGTTGTGTGCACAGAATCGAGCGG 
PHN79088 zmIPT2-R 5'&gt; CGTCCGCTAGCTACTTATGCATCAG 
PHNS242 MuTIR 5'&gt; AGAGAAGCCAACGCCAWCGCCTCYATTTCGTC 
Primers were validated prior to use by performing gene-specific<br>
amplification of ZmlPT2 using B73 genomic DNA and the PHN79087+79088<br>
primer combination. The control amplification product was excised from an<br>
agarose gel and used as a ^P-labeled hybridization probe for the TUSC<br>
screening.<br>
Primer Pair<br>
79087 + 79088<br>
expected (re. reference seq) (bp)<br>
1033<br>
observed B73 gDNA (bp)<br>
-1050<br>
Following successive rounds of screening the TUSC DNA template Pools<br>
and Individual samples by PCR and ZmlPT2 hybridization, prospective<br>
zmlPT2::Mu alleles were tested for their heritability through the germline. This<br>
was achieved by repeating the ZmlPT2::Mu PCR assays against DNA template<br>
prepared from 5 kernels of selfed (F2) seed from selected Individual TUSC plants.<br>
One TUSC family, PV03_13 H-07 (from Pool 26), showed strong positive results in<br>
the F2 template assay for both 79087+9242 and 79088+9242 primer<br>
combinations. These PCR products were cloned into Topo-TA vector (Invitrogen)<br>
for DNA sequence confirmation of the Mu insertion allele of zmlPT2 harbored by<br>
family PV03 13 H-07. Each PCR fragment was expected to be homologous to the<br>
ZmlPT2 locus, and also share ~71bp of Mutator TIR homology. An expected 9bp<br>
host site duplication, which is created upon insertion of Mu elements into maize<br>
genomic DNA, was also an expected outcome of the PV03 13 H-07 ZmlPT2::Mu<br>
allele.<br>
As shown in Figure 16, DNA sequence characterization of each TUSC PCR<br>
product exhibits these expected features. The 79087+9242 PCR product contains<br>
600bp of direct homology to ZmlPT2 from the left flank of the PV03 13 H-07 Mu<br>
insertion site. This product contains the PHN79087 PCR primer site. The<br>
79088+9242 PCR product contains 442bp of DNA sequence identity with ZmlPT2,<br>
representing the right flank of the Mu insertion site, and contains the PHN79088<br>
primer site. When trimmed of Mutator TIR sequences and aligned to the zmlPT2<br>
referennce sequence, these PCR fragments overlap by 9bp (nt 624-632 of the<br>
1495bp ZmlPT2 reference sequence), representing the expected 9bp host site<br>
duplication created upon insertion of Mutator into ZmlPT2.<br>
Thus, TUSC family PV03 13 H-07 contains a heritable Mutator insertion into<br>
the coding sequence (ORF) of the ZmlPT2 gene. This allele is expected to<br>
produce a null mutation or "knockout" of the ZmlPT2 locus. F2 progeny seed from<br>
PV03 13 H-07, which genetically segregates for the ZmlPT2::Mu mutation, known<br>
as zmlPT2-H07, was withdrawn from the TUSC seed bank and propagated for<br>
phenotypic and biomolecular analyses.<br>
Figure 16 graphically summarizes this TUSC result, and the corresponding<br>
sequence is provided as SEQ ID NO: 85.<br>
As added characterization, BLAST searches of the MuTIR portions of each<br>
zmlPT2::Mu PCR product were conducted to ascribe an identity for the Mu<br>
element residing at the ZmlPT2 locus. TUSC PCR products amplified with the<br>
Mutator PHN9242 primer contain 39bp of flanking TIR sequence that are specific<br>
to the resident element being amplified. BLAST results are consistent with the<br>
zmlPT2::Mu element being either a Mu4 or a Mu3 element.<br>
&gt;IPT2_TIR_L (SEQ ID NO: 86}<br>
GAGATAATTGCCATTATAGAAGAAGAGAGAAGGGGATTCGACGAAATAGAGGCGATGGCGTTGGCTTCTCT<br>
&gt;IPT2_TIR_R (SEQ ID NO: 87)<br>
AAGCCAACGCCUUiCGCCTCTATTTCGTCGAATCCCCITCTCTCTTCTTCTATAATGGCAATTATCTC<br>
In certain embodiments the nucleic acid constructs of the present invention<br>
can be used in combination ("stacked") with other polynucleotide sequences of<br>
interest in order to create plants with a desired phenotype. The polynucleotides of<br>
the present invention may be stacked with any gene or combination of genes, and<br>
the combinations generated can include multiple copies of any one or more of the<br>
polynucleotides of interest. The desired combination may affect one or more<br>
traits; that is, certain combinations may be created for modulation of gene<br>
expression affecting cytokinin activity. For example, up-regulation of cytokinin<br>
synthesis may be combined with down-regulation of cytokinin degradation. Other<br>
combinations may be designed to produce plants with a variety of desired traits,<br>
such as those previously described.<br>
All publications and patent applications mentioned in the specification are<br>
indicative of the level of those skilled in the art to which this invention pertains. All<br>
publications and patent applications are herein incorporated by reference to the<br>
same extent as if each individual publication or patent application was specifically<br>
and individually indicated to be incorporated by reference.<br>
Although the foregoing invention has been described in some detail by way<br>
of illustration and example for purposes of clarity of understanding, it will be<br>
obvious that certain changes and modifications may be practiced within the scope<br>
of the appended claims.<br><br><br><br>
THAT WHICH IS CLAIMED:<br>
1.	 	An isolated polypeptide comprising an amino acid sequence of SEQ ID NO:<br>
2, 6, 9, 12, 15, 18, 23, 27, or 77.<br>
2.	An isolated polypeptide comprising an amino acid sequence of SEQ ID NO:<br>
41, 43, 46, 49, 52, 54, 57, 59, 61, 63, or 66.<br>
3.	An isolated polypeptide comprising an amino acid sequence selected from<br>
the group consisting of:<br><br>
(a)	an amino acid sequence comprising at least 85% sequence identity<br>
to SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49, 52, 54, 57,<br>
59, 61, 63, 66, or 77, wherein said polypeptide has cytokinin<br>
synthesis activity;<br>
(b)	an amino acid sequence encoded by a polynudeotide that hybridizes<br>
under stringent conditions to the complement of a polynudeotide<br>
represented by SEQ ID NO: 1, 3, 4, 5, 7, 8, 10, 11, 13, 14, 16, 17,<br>
19, 20. 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56,<br>
58, 60, 62, 64, 65, 69, 70, 71, 72, 73, 74, or 76, wherein said<br>
stringent conditions comprise hybridization in 50% formamide, 1 M<br>
NaCI, 1% SDS at 37°C, and a wash in 0.1 X SSC at 60°C to 65°C,<br>
wherein said polypeptide retains cytokinin synthesis activity; and,<br>
(d) an amino add sequence comprising at least 50 consecutive amino adds of SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49, 52, 54, 57, 59, 61, 63, 66, or 77, wherein said polypeptide retains cytokinin synthesis activity.<br>
4	An isolated polynudeotide comprising a nucleotide sequence of SEQ ID<br>
NO: 1, 3, 5, 7, 8,10, 11,13,14, 16, 17,19, 20, 21, 22, 24, 26, 28, or 76.<br>
5	An isolated polynudeotide comprising a nucleotide sequence of SEQ ID<br>
NO: 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65, 69, 70, 71,<br>
72, 73, or 74.<br>
6.       An isolated polynudeotide comprising a nudeotide sequence selected from the group consisting of:<br>
(a) a nudeotide sequence encoding an amino acid sequence comprising SEQ ID NO: 2, 6, 9, 12, 15, 18, 23, 27, 41, 43, 46, 49, 52, 54, 57, 59, 61, 63, 66, or 77;<br><br>
(b)	a nucleotide sequence comprising at least 85% sequence identity to<br>
SEQ ID NO: 1, 3, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19, 20, 21, 22, 24,<br>
26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58, 60, 62, 64, 65,<br>
69, 70, 71, 72, 73, 74, or 76, wherein said polynucleotide encodes a<br>
polypeptide having cytokinin synthesis activity;<br>
(c)	a   nucleotide   sequence   comprising   at   least   50   consecutive<br>
nucleotides of SEQ ID NO: 1, 3, 5, 7, 8, 10, 11, 13, 14, 16, 17, 19,<br>
20, 21, 22, 24, 26, 28, 40, 42, 44, 45, 47, 48, 50, 51, 53, 55, 56, 58,<br>
60,  62,  64,  65,  69,  70,   71,  72,  73,  74,  or 76,  wherein  said<br>
polynudeotide encodes a polypeptide having cytokinin synthesis<br>
activity; and,<br>
(d)	a nudeotide sequence which represents a  polynucleotide that<br>
hybridizes  under stringent conditions to the complement of a<br>
polynudeotide  represented  by the  nucleotide  sequence of a),<br>
wherein said stringent conditions comprise hybridization in 50%<br>
formamide, 1 M NaCI, 1% SDS at 37°C, and a wash in 0.1 X SSC at<br>
60°C to 65°C.<br><br>
7.	A transgenic plant comprising a polynucleotide operably linked to a<br>
promoter that drives expression in the plant, wherein said polynucleotide<br>
comprises a nudeotide sequence of daim 6, and wherein cytokinin level in<br>
said plant is modulated relative to a control plant.<br>
8.	The plant of claim 7, wherein said cytokinin level is increased.<br>
9.	The plant of daim 7, wherein said cytokinin level is decreased.<br>
10.	The plant of daim 7, wherein said polynucleotide is operably linked to a a<br>
tissue-preferred   promoter,   a   constitutive   promoter,   or   an   inducible<br>
promoter.<br>
11.	The plant of daim 10, wherein said promoter is a root-preferred promoter, a<br>
leaf-preferred promoter, a shoot-preferred promoter, or an inflorescence-<br>
preferred promoter.<br>
12.	The plant of daim 7, wherein said cytokinin level modulation affects floral<br>
development.<br>
13.	The plant of daim 7, wherein said cytokinin level modulation affects root<br>
development,<br>
14.	The plant of daim 7, wherein the plant has an altered shoot-to-root ratio.<br>
15.	The plant of daim 7, wherein seed size or seed weight is increased.<br><br>
16.	The plant of daim 7, wherein vigor or biomass yield of said plant is<br>
increased.<br>
17.	The plant of claim 7, wherein the stress tolerance of said plant is increased.<br>
18.	The plant of claim 7, wherein said plant is maize, and tip kernel abortion is<br>
reduced.<br>
19.	The plant of claim 7, wherein said promoter is stress-insensitive and is<br>
expressed in a tissue of the developing seed or related maternal tissue at<br>
or about the time of anthesis.<br>
20.	A transformed seed of the plant of claim 7.<br>
21.	The plant of claim 7, wherein said plant is maize, wheat, rice, barley,<br>
sorghum, or rye.<br>
22.	A plant that is genetically modified at a native genomic locus, said genomic<br>
locus comprising a polynucleotide of claim 6, wherein the cytokinin level of<br>
said plant is modulated.<br>
23.	A. method of modulating cytokinin level in a plant, comprising transforming<br>
said plant with a polynucleotide of claim 6 operably linked to a promoter.<br>
24.	The method of daim 23 wherein said modulation of cytokinin level affects<br>
root growth or the shoot-to-root ratio.<br>
25.	The method of daim 23 wherein said modulation of cytokinin level affects<br>
floral development.<br>
26.	The method of claim 23 wherein said modulation of cytokinin level<br>
increases seed size or seed weight.<br>
27.	The method of claim 23 wherein said modulation of cytokinin level<br>
increases plant stress tolerance.<br>
28.	The method of daim 23 wherein said modulation of cytokinin level affects<br>
vigor or biomass yield.<br>
29.	The method of daim 23 wherein said operably-linked promoter is a tissue-<br>
preferred and/or inducible promoter.<br>
30.	The method of daim 23 wherein said promoter is stress-insensitive and is<br>
expressed in a tissue of the developing seed or related maternal tissue at<br>
or about the time of anthesis.<br>
31       The method of daim 23, wherein senescence is delayed. 32.     The method of claim 23 wherein sink strength of the seed of the plant is modulated.<br><br>
33.	The method of claim 32 wherein cytokinin level is increased in one or more<br>
of the embryo, the endosperm, and tissues proximal thereto.<br>
34.	The method of Claim 33 wherein said proximal tissue comprises the<br>
pedicel.<br>
35.	A method for modulating the rate or incidence of shoot regeneration in<br>
callus tissue, comprising expressing in said callus tissue a polynucleotide of<br>
Claim 6 operably linked to a heterologous promoter.<br>
36.	The method of Claim 35, wherein said promoter is inducible.<br>
37.	An isolated polynucleotide comprising a nucleotide sequence of SEQ ID<br>
NO: 25 or 75 or a functional fragment or variant thereof.<br>
38.	A DNA construct comprising a promoter operably linked to a nucleotide<br>
sequence of interest, wherein said promoter comprises the polynucleotide<br>
of claim 37.<br>
39.	An expression vector comprising the DNA construct of claim 38.<br>
40.	A plant comprising at least one DNA construct of claim 38.<br>
41.	Amethod of regulating the expression of a nucleotide sequence of interest,<br>
said method comprising introducing into a plant a DNA construct of claim<br>
38.<br>
42.	The method of Claim 41 wherein said nucleotide sequence of interest is<br>
transcribed to form an RNA molecule which interferes with expression of a<br>
homologous native nucleotide sequence.<br>
43.	A method of downregulating expression of ZmlPTI or ZmlPT2 in a plant,<br>
comprising transforming said plant with a construct comprising a promoter<br>
operably linked to a polynucleotide which comprises a portion of the<br>
polynucleotide of claim 37, such that a hairpin molecule is formed which<br>
corresponds to the ZmlPTI promoter or ZmlPT2 promoter.<br>
44.	The method of claim 43 wherein said promoter is tissue-preferred.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUFic3RyYWN0LSgxMi0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Abstract-(12-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUNsYWltcy0oMDUtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Claims-(05-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUNsYWltcy0oMTItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Claims-(12-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Correspondence Others-(05-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Correspondence Others-(12-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Correspondence Others-(21-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMDUtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2159-DELNP-2007-Correspondence-Others-(16-05-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1ERUxOUC0yMDA3LUZvcm0tMS0oMTYtMDUtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2159-DELNP-2007-Form-1-(16-05-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUZvcm0tMTMtKDA1LTAyLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Form-13-(05-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1ERUxOUC0yMDA3LUZvcm0tMTgtKDE2LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2159-DELNP-2007-Form-18-(16-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUZvcm0tMi0oMTItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Form-2-(12-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUZvcm0tMy0oMTItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Form-3-(12-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUZvcm0tMy0oMjEtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Form-3-(21-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LUdQQS0oMTItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-GPA-(12-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1ERUxOUC0yMDA3LUdQQS0oMTYtMDUtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2159-DELNP-2007-GPA-(16-05-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1OS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTItMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2159-delnp-2007-Petition-137-(12-09-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="258985-acrylic-coplymer-pressure-sensitive-adhesive-having-antistatic-property.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="258987-supported-metallocene-alkyl-catalyst-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>258986</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2159/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Feb-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Feb-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Mar-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PIONEER HI-BRED INTERNATIONAL, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>7100 N.W 62ND AVENUE, JOHNSTON, IA 50131-1014 USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BRUGIERE NORBERT</td>
											<td>6321 N.W. 96TH STREET, JOHNSTON, IA 50131 USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/82; A01H5/00; A01H5/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/033693</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-09-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/610,656</td>
									<td>2004-09-17</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/637,230</td>
									<td>2004-12-17</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/696,405</td>
									<td>2005-07-01</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/258986-isopentenyl-transferase-sequences-and-methods-of-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:44:19 GMT -->
</html>
