<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259083-a-vector-system by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:21:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259083:&quot;A VECTOR SYSTEM&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A VECTOR SYSTEM&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Rabies Virus compositions and methods are provided. The full-length sequence of Rabies Virus strain Evelyn-Rokitnicki-Abelseth (ERA) is disclosed. A reverse genetics system for producing recombinant ERA virus and derivatives thereof is provided, along with compositions including ERA and/or ERA derivative strain viruses, nucleic acids and/or proteins. In some instances, the compositions are immunogenic compositions useful for the pre- or post-exposure treatement of Rabies Virus.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>RABIES VIRUS COMPOSITIONS AND METHODS<br>
REFERENCE TO RELATED APPLICATION(S)<br>
This application claims the benefit of U.S. Provisional Application No, 60/727,038, filed October 14, 2005, which is incorporated herein by reference in its entirety.<br>
STATEMENT OF GOVERNMENT SUPPORT<br>
This invention was made by the Centers for Disease Control and Prevention, an agency of the United States Government. Therefore, the United States Government has certain rights in this invention.<br>
FIELD<br>
This disclosure relates to the field of virology. More specifically, this disclosure relates to compositions and methods that are useful for the production of immunogenic compositions for protecting mammals from infection by rabies virus.<br>
BACKGROUND OF THE DISCLOSURE<br>
Rabies remains one of the most dreadful infectious diseases affecting human and animals, despite significant scientific advances in its prevention and control. Rabies presents as a distinct problem in different parts of the world. In the Americas, reservoirs of rabies exist in many wild animal species, including raccoons, skunks, foxes, and bats (Rupprecht et ai, Emerg. Infect. Dis. 1(4): 107-114, 1995). Outbreaks of rabies infections in these terrestrial mammals are found in broad geographic areas across the United States. For example, raccoon rabies affects an area of more than 1 million square kilometers from Florida to Maine. Although wildlife rabies still exists in developed countries, progress has been made in control and elimination of wildlife rabies using oral immunization of wild animals.<br>
Nonetheless, rabies remains a major threat to public health and persists to cause between 50,000 and 60,000 human deaths each year (World Health Organization, April 2003). Humans get infected with the rabies virus mostly through bites from rabid domestic and wildlife animals. In developing countries, dogs are responsible for about 94% of human rabies deaths. Dog rabies is still epizootic in most countries of Africa, Asia and South America and in these countries dogs are responsible for most human deaths from the disease. Controlling rabies virus infection in domestic and wildlife animals, therefore, not only reduces the mortality in these animals but also reduces the risks of human exposure.<br>
The rabies virus is transmitted through broken skin by the bite or scratch of an infected animal. Exposure to rabies virus results in its penetration of peripheral, unmyelineated nerve endings, followed by spreading through retrograde axonal transport, replication occurring exclusively in the neurons, and finally arrival in the central nervous system (CNS). Infection of the CNS causes cellular dysfunction and death (Rupprecht &amp; Dietzschold, Lab Invest. 57:603, 1987), Since rabies<br>
virus spreads directly from cell to cell, it largely evades immune recognition (Clark &amp; Prabhakar, Rabies, In: Olson et al, eds., Comparative Pathology of Viral-Bisease. 2:165, Boca Raton, FL, CRC Press, 1985).<br>
The rabies virus (RV) is a rhabdovirus - a nonsegmented RNA virus with negative sense polarity. Within the Rhabdoviridae family, rabies virus is the prototype of the Lyssavirus genus. RV is composed of two major structural components: a nucleocapsid or ribonucleoprotein (RNP), and an envelope in the form of a bilayer membrane surrounding the RNP core. The infectious component of all rhabdoviruses is the RNP core, which consists of the negative strand RNA genome encapsidated by nucleoprotein (N) in combination with RNA-dependent RNA-polymerase (L) and phosphoprotein (P). The membrane surrounding the RNP contains two proteins: the trans-membrane glycoprotein (G) and the matrix (M) protein, located at the inner site of the membrane. Thus, the viral genome codes for these five proteins: the three proteins in the RNP (N, L and P), the matrix protein (M), and the glycoprotein (G).<br>
The molecular determinants of pathogenicity of various rabies virus strains have not been fully elucidated. RV pathogenicity was attributed to multigenic events (Yamada et al., Microbiol Immunol. 50:25-32, 2006). For example, some positions in the RV genome if mutated, affect viral transcription or replication, reducing virulence. Mutations at serine residue 389 of the phosphorylation site in the N gene (Wu et al.,J. Virol. 76:4153-4161, 2002) or GDN core sequence of the highly conserved C motif in the L gene (Schnell and Conzelmann, Virol. 214:522-530, 1995) dramatically reduced RV transcription and replication.<br>
The G protein, also referred to as spike protein, is involved in cell attachment and membrane fusion of RV. The amino acid region at position 330 to 340 (referred to as antigenic site III) of the G protein has been identified as important for virulence of certain strains of RV. Several studies support the concept that the pathogenicity of fixed RV strains is determined by the presence of arginine or lysine at amino acid residue 333 of the glycoprotein (Dietzschold et al, Proc. Natl. Acad. Sci. USA 80: 70-74, 1983; Tuffereau et al, Virol 172: 206-212, 1989).<br>
This phenomenon seems to apply at least to fixed rabies viruses such as CVS, ERA, PV, SAD-B19 and HEP-Flury strains (Anilionis et al, Nature 294:275-278, 1981; Morimoto et al, Virol. 173:465-477, 1989). For example, rabies vaccine viruses possessing an amino acid differing from Arg at position 333 of the glycoprotein are described, for instance, in WO 00/32755 (describing RV mutants in which all three nucleotides in the G protein Arg333 codon are altered compared to the parent virus, such that the Arg at position 333 is substituted with another amino acid); European Patent 350398 (describing an avirulent RV mutant SAG1 derived from the Bern SAD strain of RV, in which the Arg at position 333 of the glycoprotein has been substituted to Ser); and European patent application 583998 (describing an attenuated RV mutant, SAG2, in which the Arg at position 333 in the G protein has been substituted by Glu).<br>
Other strains, such as the RC-HL strain, possess an arginine residue at position 333 of the G, but does not cause lethal infection in adult mice (Ito et al, Microl Immunol. 38:479-482, 1994;<br>
Ito et al, J. Virol. 75:9121-9128, 2001). As such, the entire G may contribute to the virulence of RV, although the determinants or regions have not previously been identified. The G gene encodes the only protein that induces viral neutralizing antibody. At least three states of RV glycoprotein are known: the native state (N) being responsible for receptor binding; an active hydrophobic state (A) necessary in the initial step in membrane fusion process (Gaudin, J. Cell Biol. 150:601-612, 2000), and a fusion inactive conformation (I). Correct folding and maturation of the G play important roles for immune recognition. The three potential glycosylated positions in ERA G extracellular domain occur at Asn37, Asn247 and Asn319residues (Wojczyk et al, Glycobiology. 8: 121-130, 1998), respectively. Nonglycosylation of G not only affects conformation, but also inhibits presentation of the protein at the cell surface. Thus, elucidating the molecular determinants underyling pathogenicity of rabies virus presents a complex problem.<br>
SUMMARY OF THE DISCLOSURE<br>
The complete sequence of the virus strain corresponding to the fixed vaccine of Evelyn-Rokitnicki-Abelseth (ERA) for rabies virus is disclosed herein, along with methods for sequencing this and other strains of lyssavirus.<br>
A reverse genetics system for rabies virus is also described, in particular using the rabies virus strain ERA as an exemplar. Use of a T7 RNA polymerase, containing an eight amino acid nuclear localization signal (NLS) at the N terminal end facilitated virus recovery. Besides the parental ERA virus strain, several other derivative viruses are described, including ERA- (deletion of the psi-region), ERAgreenl (green fluorescent protein gene inserted in psi region), ERAgreen2 (green fluorescent protein gene inserted at the phosphoprotein and matrix protein intergenic region ), ERA2g (containing an extra copy of the glycoprotein in the psi-region), ERAg3 (with a mutation at amino acid 333 in glycoprotein), ERA2g3 (with an extra copy of altered glycoprotein at amino acid 333 in psi-region), ERA-G (from which the glycoprotein has been deleted) ERAgm (M and G genes switched in the genome), and ERAgmg (two copies of G in the rearranged ERAgm construct). The extra transcription unit was incorporated into ERA virus genome for efficient expression of Open Reading Frames (ORFs). By optimizing propagation conditions, which are described herein, rescued viruses reach titers in excess of 109 ffu/ml in either bioreactors or stationary tissue flasks.<br>
Also disclosed is a modified cell line that constitutively expresses the ERA glycoprotein. The cell line, designated BSR-G, is useful for the production of recombinant, including attenuated and/or replication deficient, rabies virus.<br>
The foregoing and other objects, features, and advantages of the invention will become more apparent from the following detailed description.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
FIG. 1A. Schematic illustration of the ERA transcription plasmid. Positions of the hammerhead ribozymes and antigenomic ERA genome are indicated graphically. Relative positions of the N, P, M G and L proteins are shown in a 5' to 3' direction.<br>
FIG. IB. Schematic diagram of the construction of the full-length ERA rabies virus genomic cDNA plasmid pTMF. RT-PCR products Fl, F2 fragments, and restriction enzyme recognition sites (Nhel, Kpnl, Blpl, Pstl and Notl)(not drawn to scale). The bar on the left indicates a RdRz-hammerhead ribozyme and the right bar indicates the HDVRz-hepatitis delta virus ribozyme. The symbol ♦ indicates that Kpnl or Pstl sites were deleted, and ▼ vertical arrow indicates that Nhel or Notl sites were left intact.<br>
FIG. 2. Schematic illustration of the proposed mechanism of NLST7 RNA polymerase autogene action by pNLST7 plasmids. The DNA-transfection reagent complex is taken into cells by endocytosis. The majority of the DNA released from lysosomes and endosomes is retained in the cell cytoplasm. A limited amount of plasmids are transferred to the nucleus: 1) through a CMV immediate early promoter, the NLST7 gene is transcribed by cellular RNA polymerasell; 2) mature NLST7 mRNA is transported from the nucleus to the cytoplasm for NLST7 RNA polymerase synthesis; 3) Newly synthesized NLST7 RNA polymerase is trans-located to the nucleus, while a trace amount of NLST7 remains in the cytoplasm; 4) NLST7 RNA polymerase initiates transcription through a pT7 promoter. By posttranscriptional modifications, additional NLST7 mRNA is produced for protein synthesis, thus increasing virus recovery efficiency.<br>
FIG. 3. Schematic diagram of the ten derivative ERA virus genomes. The size of each gene is not drawn to scale. Symbol """'denotes mutations of G at the Aa333 residue and  is the Psi-region.<br>
FIG. 4A. Analysis of recovered ERA-G virus in transfected cells, spread and growth in a BHK-G cell line. In A, the ERA-G viral foci were restrained even after seven-day post transfection with plasmids for virus recovery. In B, rescued ERA-G virus did not spread after passage in normal BSR cells. Only individual cells were stained by DFA. In C, the ERA-G virus grew well in the constitutive BHK-G cell line.<br>
FIG. 4B. Analysis of G expression in a BHK-G cell line. By indirect fluorescent staining, the ERA rabies virus G was expressed in the cytoplasm in a stable cell line BHK-G.<br>
FIG. 4C. Analysis of G mRNA in ERA-G virus- infected cells by Northern blot with a G probe. Lane 2 shows that the G gene mRNA was detected in ERA-G virus infected BHK-G cells, while virus genomic RNA was not. Lane 1 was the control of total RNA from ERA- rabies virus-infected BHK-G cells, in which both G mRNA and viral genomic RNA were detected.<br>
FIG. 5. Single-step virus growth curve All recovered rabies virus ERA strains grew to 109 or 1010 ffu/ml, but the ERA-G only reached 107ffu/ml.<br>
FIG. 6. Green foci in ERAgreenl/ERAgreen2 rabies virus-infected BSR cells<br>
Trans 1 is the transcriptional unit incorporated at the Psi and L intergenic regions. Trans2 is the transcriptional unit at the P and M intergenic region. Both ERAgreen2 and ERAgreenl expressed the GFP protein stably in virus- infected BSR cells, while the occurrence of the green foci of ERAgreen2 was 48 hours earlier after virus infection than in ERAgreenl.<br>
FIG. 7. Analysis of G mRNA expression in double G, and G, M rearranged ERA rabies viruses. In the Northern blot with a G probe, the measurement by density photometry of G mRNA in ERA2g (lane 1), ERAgm (lane 3) and ERA2g3 (lane 4) represented enhanced mRNA levels, compared with ERA- virus-infected cells (lane 2). The ratios were calculated by the use of ERA-virus as 100%.<br>
FIG. 8A. Morbidity induced by in vivo inoculation with recombinant ERA and derivatives. Three-week old mice were inoculated intramuscularly with the eight recovered viruses. At 10 days post inoculation, in ERA, ERA- and ERAgreenl groups, 50%, 50% and 20% of mice showed compatible clinical signs of rabies, but no mortality, respectively. No adverse signs were observed in the other groups.<br>
FIG. 8B. Post challenge survival in mice inoculated with recombinant ERA and derivtives. Mice surviving from the tests shown in FIG. 8A were challenged intramuscularly with a Texas dog/ coyote rabies virus. At 5 days post challenge, in the ERA and ERA- groups, 40 and 62% of mice showed signs of rabies and were euthanized, respectively. In all other groups, no signs of rabies were observed.<br>
FIG. 8C. Survival following i.e. inoculation with recombinant ERA and ERAg3 viruses. Three-week old mice were inoculated intracerebrally with ERA and ERAg3 virus strains, respectively. All mice succumbed 15 days postinoculation in the ERA group, while in the ERAg3 group, all mice survived with no clinical signs.<br>
FIG. 8D. Survival following i.e. inoculation of suckling mice. Two-day old suckling mice were inoculated intracerebrally with ERAg3 and ERA-G virus constructs, respectively. All mice succumbed in the ERAg3 group, while no mice died in the ERA-G group.<br>
FIG. 8E. Neutralizing antibody titer in mice inoculated with recombinant ERA and derivative viruses. Mouse neutralization antibody titers were determined by the RFFIT, ranging from 1.36 to 5.61 RJ per ml among the virus-inoculated groups.<br>
FIG. 9A. Survival following infection with Alabama Bat Rabies virus. Hamsters were inoculated with live Alabama Bat Rabies Virus, then treated post-exposure with either ERAg3 virus or with rabies immune globulin and commercially available inactivated RV vaccine. Survival was assessed over a more than three month period.<br>
FIG. 9B. Survival following infection with Thai Street Dog Rabies virus. Hamsters were inoculated with live Alabama Bat Rabies Virus, then treated post-exposure with either ERAg3 virus or with rabies immune globulin and commercially available inactivated RV vaccine. Survival was assessed over a more than three month period.<br>
FIG. 9C. Survival following infection with Texas Coyote Dog Rabies virus. Hamsters were inoculated with live Alabama Bat Rabies Virus, then treated post-exposure with either ERAg3 virus or with rabies immune globulin and commercially available inactivated RV vaccine. Survival was assessed over a more than three month period.<br>
SEQUENCE LISTING<br>
The nucleic and amino acid sequences listed in the accompanying sequence listing are shown using standard letter abbreviations for nucleotide bases, and three letter code for amino acids, as defined in 37 C.F.R. 1.822. Only one strand of each nucleic acid sequence is shown, but the complementary strand is understood as included by any reference to the displayed strand unless the context makes it clear that only one strand is intended. As appropriate, it will be understood that a sequence presented as DNA can be converted to RNA by replacing thiamine residues with uracils.<br>
SEQ ID NO: 1. ERA CDC wild type virus, 11,931 nucleotides<br>
1-58 nucleotides, Leader region<br>
71-1420 nucleotides, N gene<br>
1514-2404 nucleotides, P gene<br>
2496-3101 nucleotides, M gene<br>
3317-4888 nucleotides, G gene<br>
4964-5362 nucleotides, Psi-region<br>
5417-11797 nucleotides, L gene<br>
11862-11931 nucleotides, Trailer region<br>
SEQ ID NO: 2. ERACDC: 71 to 1420:450 aa, N protein.<br>
SEQ ID NO: 3. ERACDC: 1514 to 2404: 297 aa, P protein.<br>
SEQ ID NO: 4. ERACDC: 2496 to 3101:202 aa, M protein.<br>
SEQ DD NO: 5. ERACDC: 3317 to 4888:524 aa, G protein.<br>
SEQ ED NO: 6. ERACDC: 5417 to 11797:2127 aa, L protein.<br>
SEQ ID NO: 7. Recombinant ERA (rERA) recovered by reverse genetics system is 11,930 nucleotides. The specific poly (A8) tract between G gene and psi-region in wild type ERA strain was mutated to a poly (A7) tract in recombinant ERA reverse genetics system as a sequence marker. In light of this, rERA is one nucleotide shorter than wild type ERA. All the other sequence information is exactly the same.<br>
SEQ ED NO: 8. ERAg3 strain (11,930 nucleotides), amino acid in the G protein (333Aa) has been altered; the corresponding nucleic acids are at positions 4370 to 4372.<br>
SEQ ED NO: 9. ERA- (11,577 nucleotides), without the psi (pseudo-gene) region; an extra transcription unit has been introduced at nucleotide positions 4950 to 5008.<br>
SEQ ED NO: 10. ERA-2G (13,150 nucleotides), this strain has two copies of the G gene; the second copy is inserted at positions 4988 to 6559.<br>
SEQ ID NO: 11. ERAgreen (12,266 nucleotides), this strain contains the coding sequence for GFP at positions 4993 to 5673; it appears green under UV light after infection of cells or tissue.<br>
SEQ ED NO: 12. ERA-G (10,288 nucleotides), this strain has no G gene.<br>
SEQ ID NO: 13. ERA-2g3 (13,150 nucleotides); this strain has two copies of the G gene (the second of which is at positions 4988 to 6559), both of which are substituted at amino acid 333 (corresponding to nucleotide positions 4370-4372 and 6041-6043 in the shown sequence).<br>
SEQ ID NO: 14. ERA-pt (11,976 nucleotides, with an extra transcription unit after the P gene, at positions 2469 to 2521).<br>
SEQ BO NO: 15. ERA-pt-GFP (12,662 nucleotides, with GFP gene inserted after P gene at 2505 to 3185).<br>
SEQ ID NO: 16. ERAgm (11,914 nucleotides) positions of G and M genes are switched with G at positions 2505-4076 and M at positions 4122-4727, respectively.<br>
SEQ ID NO: 17. ERAg3m (11,914 nucleotides) positions of G and M genes are switched with G at positions 2505-4076 and M at positions 4122-4727, respectively. The G gene is mutated at amino acid position 333.<br>
SEQ ID NO: 18. ERAgmg (13,556 nucleotides), this strain has two copies of the G gene at positions 2505-4076 and 4943-6514, flanking the M gene at positions 4122-4727.<br>
SEQ ID NO: 19. First ten nucleotides of hammerhead ribozyme corresponding to 5' end of rabies virus ERA genome.<br>
SEQ ID NO: 20. Nucleotide sequence encoding the SV40 T antigen nuclear localization signal (NLS).<br>
SEQ ED NOs: 21-23. Artificial Kozak sequences.<br>
SEQ ID NOs: 24-57. Synthetic oligonucleotides.<br>
SEQ ID NO: 58. Amino acid sequence of G protein mutated at amino acid position 333 (from Arg to Glu).<br>
SEQ D3 NOs: 59-65. Synthetic oligonucleotides.<br>
DETAILED DESCRIPTION<br>
/.	Introduction<br>
Viral zoonoses are difficult to prevent. One major paradigm is the control of wildlife rabies by oral vaccination. All current licensed oral rabies vaccines are based on one common source. The fixed rabies virus (RV) of Evelyn-Rokitnicki-Abelseth (ERA) was derived from the Street-Alabama-Dufferin (SAD) strain, first isolated from a rabid dog in Alabama (USA) in 1935. The ERA strain was derived after multiple passages of SAD RV in mouse brains, baby hamster kidney (BFfK) cells, and chicken embryos. Repeated cloning of ERA in BFfK cells resulted eventually in a B-19 clone, which was named SAD-B19 for vaccine studies. The first RV strain recovered by reverse genetics was SAD-B19. Although SAD-B19 and ERA RV derived from the same source, different outcomes have been observed in various animal oral vaccine studies. For example, ERA did not induce obvious neutralizing antibodies in either skunks or raccoons per os, while<br>
SAD-B19 did. To elucidate potential differences between these two RV strains, a reverse genetics system for the ERA RV strain is required.<br>
Reverse genetics presents a feasible way to modify RNA viruses in defined ways. A system for reverse genetics of an initial strain of rabies virus was successfully established in 1994 (Schnell et al, The EMBO J. 13, 4195-4203, 1994). In the intervening decade, improvements to the system have been made, resulting in increased efficiency of virus recovery. This increased efficiency has facilitated the elucidation of virus pathogenicity, protein-protein, and protein-RNA interactions.<br>
Within the rabies virus genome, it has been proposed that some regions contain important signals, such as viral distal promoters region, nucleoprotein encapsidation, RNA dependent RNA polymerase L transcription initiation site, polyadenylation and termination sites. These signals are important for ensuring efficient recovery of virus and for designing an extra transcription unit for accepting an exogenous Open Reading Frame (ORF) into the rabies virus genome.<br>
This disclosure provides an efficient reverse genetics system, and describes its use to produce variants of the ERA strain virus. Modifications described herein have resulted in strains that are suitable candidates for accepting ORF expression and vaccine development.<br>
The reverse genetics system is composed of a set of plasmids. A first plasmid includes an ERA viral cRNA. In order to create authentic viral anti-genomic ends in transcribed viral cDNA, ERA genomic cDNA is flanked by a hammerhead ribozyme at the 3' end and a hepatitis delta virus ribozyme at the 5' end. The antigenomic cassette is fused to the bacteria phage T7 transcription initiation signal, which is optionally also under the control of cytomegalovirus (CMV) immediate-early promoter.<br>
The system also includes a plurality of helper plasmids that encode proteins involved in viral encapsidation. For example, the system typically includes helper plasmids that encode the viral nucleoprotein (N), phosphoprotein (P), RNA dependent polymerase (L), and optionally the viral glycoprotein (G). The system also includes a plasmid that encodes the phage T7 RNA polymerase (T7), which can be modified by the addition of a nuclear localization signal (NLS) to increase expression of the T7 polymerase in the nucleus of transfected cells. The T7 RNA polymerase expression plasmid is constructed as an "autogene," which transcribes the whole length of viral antigenomic cRNA for nucleoprotein encapsidation after transfection into cells.<br>
The reverse genetics system is useful in the design and production of immunogenic compositions for the treatment (pre and/or post exposure) of rabies virus, and for producing rabies virus ERA vectors for expressing exogenous Open Reading Frames (ORFs). For example, an extra transcription unit can be designed, tested and incorporated into the ERA genome at either the Psi-region and/or at phosphoprotein (P) -matrix (M) protein intergenic region. Essentially any ORF of interest can be expressed in the context to the ERA vector, including ORFs encoding antigens of viruses and other pathogens, such as antigens of other lyssaviruses, as well as for expressing other proteins of therapeutic interest.<br>
Thus, the methods and compositions disclosed herein are useful for the design and production of rabies virus immunogenic compositions, including compositions suitable as vaccines for the pre and/or post exposure treatment of rabies virus.<br>
//.	Abbreviations<br>
ADE	antibody-dependant enhancement<br>
Ag-ELISA	antigen-capture ELISA<br>
DNA	deoxyribosenucleic acid<br>
ERA	Rabies virus strain Evelyn-Rokitnicki-Abelseth<br>
ELISA	enzyme-linked immunoabsorbent assay<br>
G	glycoprotein<br>
i.e.	intracerebral<br>
IFA	indirect immuno-fluorescence assay<br>
i.m.	intramuscular<br>
L	RNA-dependent RNA-polymerase<br>
M	matrix protein<br>
mAb	monoclonal antibody<br>
N	nucleoprotein<br>
ORF	open reading frame<br>
P	phosphoprotein<br>
PCR	polymerase chain reaction<br>
RACE	5' rapid amplification of cDNA ends<br>
RNA	ribosenucleic acid<br>
RNP	ribonucleoprotein<br>
RT-PCR	reverse transcription-polymerase chain reaction<br>
RV	rabies virus<br>
trans 1	extra transcription unit 1<br>
trans 2	extra transcription unit 2<br>
III.	Terms<br>
Unless otherwise explained, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Similarly, unless otherwise noted, technical terms are used according to conventional usage. Definitions of common terms in molecular biology may be found in Benjamin Lewin, Genes V, published by Oxford University Press, 1994 (ISBN 0-19-854287-9); Kendrew et al. (eds.), The Encyclopedia of Molecular Biology, published by Blackwell Science Ltd., 1994 (ISBN 0-632-02182-9); and Robert A. Meyers (ed.), Molecular Biology and Biotechnology: a Comprehensive Desk Reference, published by VCH Publishers, Inc., 1995 (ISBN 1-56081-569-8).<br>
The singular terms "a," "an," and "the" include plural referents unless context clearly indicates otherwise. Similarly, the word "or" is intended to include "and" unless the context clearly indicates otherwise. Hence "comprising A or B" means including A, or B, or A and B. It is further to be understood that all base sizes or amino acid sizes, and all molecular weight or molecular mass values, given for nucleic acids or polypeptides are approximate, and are provided for description.<br>
In order to facilitate review of the various embodiments of the invention, the following explanations of specific terms are provided:<br>
Adjuvant: A substance that non-specifically enhances the immune response to an antigen. Development of vaccine adjuvants for use in humans is reviewed in Singh et al. (Nat. Biotechnol. 17:1075-1081, 1999), which discloses that, at the time of its publication, aluminum salts and the MF59 microemulsion are the only vaccine adjuvants approved for human use.<br>
Amplification: Amplification of a nucleic acid molecule (e.g., a DNA or RNA molecule) refers to use of a laboratory technique that increases the number of copies of a nucleic acid molecule in a sample. An example of amplification is the polymerase chain reaction (PCR), in which a sample is contacted with a pair of oligonucleotide primers under conditions that allow for the hybridization of the primers to a nucleic acid template in the sample. The primers are extended under suitable conditions, dissociated from the template, re-annealed, extended, and dissociated to amplify the number of copies of the nucleic acid. The product of amplification can be characterized by such techniques as electrophoresis, restriction endonuclease cleavage patterns, oligonucleotide hybridization or ligation, and/or nucleic acid sequencing.<br>
Other examples of amplification methods include strand displacement amplification, as disclosed in U.S. Patent No. 5,744,311; transcription-free isothermal amplification, as disclosed in U.S. Patent No. 6,033,881; repair chain reaction amplification, as disclosed in WO 90/01069; ligase chain reaction amplification, as disclosed in EP-A-320,308; gap filling ligase chain reaction amplification, as disclosed in U.S. Patent No. 5,427,930; and NASBA™ RNA transcription-free amplification, as disclosed in U.S. Patent No. 6,025,134. An amplification method can be modified, including for example by additional steps or coupling the amplification with another protocol.<br>
Animal: Living multi-cellular vertebrate organisms, a category that includes, for example, mammals and birds. The term mammal includes both human and non-human mammals. Similarly, the term "subject" includes both human and veterinary subjects, for example, humans, non-human primates, dogs, cats, horses, and cows.<br>
Antibody: A protein (or protein complex) that includes one or more polypeptides substantially encoded by immunoglobulin genes or fragments of immunoglobulin genes. The recognized immunoglobulin genes include the kappa, lambda, alpha, gamma, delta, epsilon, and mu constant region genes, as well as the myriad immunoglobulin variable region genes. Light chains are classified as either kappa or lambda. Heavy chains are classified as gamma, mu, alpha, delta, or epsilon, which in turn define the immunoglobulin classes, IgG, IgM, IgA, IgD and IgE, respectively.<br>
The basic immunoglobulin (antibody) structural unit is generally a tetramer. Each tetramer is composed of two identical pairs of polypeptide chains, each pair having one "light" (about 25 kDa) and one "heavy" (about 50-70 kDa) chain. The N-terminus of each chain defines a variable region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. The terms "variable light chain" (VL) and "variable heavy chain" (VH) refer, respectively, to these light and heavy chains.<br>
As used herein, the term "antibody" includes intact immunoglobulins as well as a number of well-characterized fragments. For instance, Fabs, Fvs, and single-chain Fvs (SCFvs) that bind to<br>
target protein (or epitope within a protein or fusion protein) would also be specific binding agents for that protein (or epitope). These antibody fragments are as follows: (1) Fab, the fragment which contains a monovalent antigen-binding fragment of an antibody molecule produced by digestion of whole antibody with the enzyme papain to yield an intact light chain and a portion of one heavy chain; (2) Fab', the fragment of an antibody molecule obtained by treating whole antibody with pepsin, followed by reduction, to yield an intact light chain and a portion of the heavy chain; two Fab' fragments are obtained per antibody molecule; (3) (Fab')2, the fragment of the antibody obtained by treating whole antibody with the enzyme pepsin without subsequent reduction; (4) F(ab')2, a dimer of two Fab' fragments held together by two disulfide bonds; (5) Fv, a genetically engineered fragment containing the variable region of the light chain and the variable region of the heavy chain expressed as two chains; and (6) single chain antibody, a genetically engineered molecule containing the variable region of the light chain, the variable region of the heavy chain, linked by a suitable polypeptide linker as a genetically fused single chain molecule. Methods of making these fragments are routine (see, for example, Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999).<br>
Antibodies for use in the methods and compositions of this disclosure can be monoclonal or polyclonal. Merely by way of example, monoclonal antibodies can be prepared from murine hybridomas according to the classical method of Kohler and Milstein {Nature 256:495-97, 1975) or derivative methods thereof. Detailed procedures for monoclonal antibody production are described in Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999.<br>
Antibody binding affinity: The strength of binding between a single antibody binding site and a ligand (e.g., an antigen or epitope). The affinity of an antibody binding site X for a ligand Y is represented by the dissociation constant (Kd), which is the concentration of Y that is required to occupy half of the binding sites of X present in a solution. A smaller (Kd) indicates a stronger or higher-affinity interaction between X and Y and a lower concentration of ligand is needed to occupy the sites. In general, antibody binding affinity can be affected by the alteration, modification and/or substitution of one or more amino acids in the epitope recognized by the antibody paratope.<br>
In one example, antibody binding affinity is measured by end-point titration in an Ag-ELISA assay. Antibody binding affinity is substantially lowered (or measurably reduced) by the modification and/or substitution of one or more amino acids in the epitope recognized by the antibody paratope if the end-point titer of a specific antibody for the modified/substituted epitope differs by at least 4-fold, such as at least 10-fold, at least 100-fold or greater, as compared to the unaltered epitope.<br>
Antigen: A compound, composition, or substance that can stimulate the production of antibodies or a T-cell response in an animal, including compositions that are injected or absorbed into an animal. An antigen reacts with the products of specific humoral or cellular immunity, including those induced by heterologous immunogens. In one embodiment, an antigen is a virus antigen.<br>
Attentuated: In the context of a live virus, such as a rabies virus, the virus is attenuated if its ability to infect a cell or subject and/or its ability to produce disease is reduced (for example, eliminated). Typically, an attenuated virus retains at least some capacity to elicit an immune response following administration to an immunocompetent subject. In some cases, an attenuated virus is capable of eliciting a protective immune response without causing any signs or symptoms of infection.<br>
Binding or Stable Binding: An oligonucleotide binds or stably binds to a target nucleic acid if a sufficient amount of the oligonucleotide forms base pairs or is hybridized to its target nucleic acid, to permit detection of that binding. Binding can be detected by either physical or functional properties of the target: oligonucleotide complex. Binding between a target and an oligonucleotide can be detected by any procedure known to one skilled in the art, including functional or physical binding assays. Binding can be detected functionally by determining whether binding has an observable effect upon a biosynthetic process such as expression of a gene, DNA replication, transcription, translation, and the like.<br>
Physical methods of detecting the binding of complementary strands of DNA or RNA are well known in the art, and include such methods as DNase I or chemical footprinting, gel shift and affinity cleavage assays, Northern blotting, Southern blotting, dot blotting, and light absorption detection procedures. For example, a method which is widely used, because it is so simple and reliable, involves observing a change in light absorption of a solution containing an oligonucleotide (or an analog) and a target nucleic acid at 220 to 300 nm as the temperature is slowly increased. If the oligonucleotide or analog has bound to its target, there is a sudden increase in absorption at a characteristic temperature as the oligonucleotide (or analog) and target dissociate or melt.<br>
The binding between an oligomer and its target nucleic acid is frequently characterized by the temperature (Tm) at which 50% of the oligomer is melted from its target. A higher Tm means a stronger or more stable complex relative to a complex with a lower Tm.<br>
cDNA (complementary DNA): A piece of DNA lacking internal, non-coding segments (introns) and regulatory sequences that determine transcription. cDNA is synthesized in the laboratory by reverse transcription from messenger RNA extracted from cells.<br>
Electrophoresis: Electrophoresis refers to the migration of charged solutes or particles in a liquid medium under the influence of an electric field. Electrophoretic separations are widely used for analysis of macromolecules. Of particular importance is the identification of proteins and nucleic acid sequences. Such separations can be based on differences in size and/or charge. Nucleotide sequences have a uniform charge and are therefore separated based on differences in size. Electrophoresis can be performed in an unsupported liquid medium (for example, capillary electrophoresis), but more commonly the liquid medium travels through a solid supporting medium. The most widely used supporting media are gels, for example, polyacrylamide and agarose gels.<br>
Sieving gels (for example, agarose) impede the flow of molecules. The pore size of the gel determines the size of a molecule that can flow freely through the gel. The amount of time to travel<br>
through the gel increases as the size of the molecule increases. As a result, small molecules travel through the gel more quickly than large molecules and thus progress further from the sample application area than larger molecules, in a given time period. Such gels are used for size-based separations of nucleotide sequences.<br>
Fragments of linear DNA migrate through agarose gels with a mobility that is inversely proportional to the log10 of their molecular weight. By using gels with different concentrations of agarose, different sizes of DNA fragments can be resolved. Higher concentrations of agarose facilitate separation of small DNAs, while low agarose concentrations allow resolution of larger DNAs.<br>
Epitope: An antigenic determinant. These are particular chemical groups, such as contiguous or non-contiguous peptide sequences, on a molecule that are antigenic, that is, that elicit a specific immune response. An antibody binds a particular antigenic epitope based on the three dimensional structure of the antibody and the matching (or cognate) three dimensional structure of the epitope.<br>
A "substituted epitope" comprises at least one structural substitution in the epitope, such as a substitution of one amino acid for another<br>
Hybridization: Oligonucleotides and their analogs hybridize by hydrogen bonding, which includes Watson-Crick, Hoogsteen or reversed Hoogsteen hydrogen bonding, between complementary bases. Generally, nucleic acid consists of nitrogenous bases that are either pyrimidines (cytosine (C), uracil (U), and thymine (T)) or purines (adenine (A) and guanine (G)). These nitrogenous bases form hydrogen bonds between a pyrrolidine and a purine, and the bonding of the pyrimidine to the purine is referred to as "base pairing." More specifically, A will hydrogen bond to T or U, and G will bond to C. "Complementary" refers to the base pairing that occurs between to distinct nucleic acid sequences or two distinct regions of the same nucleic acid sequence.<br>
"Specifically hybridizable" and "specifically complementary" are terms that indicate a sufficient degree of complementarity such that stable and specific binding occurs between the oligonucleotide (or its analog) and the DNA or RNA target. The oligonucleotide or oligonucleotide analog need not be 100% complementary to its target sequence to be specifically hybridizable. An oligonucleotide or analog is specifically hybridizable when binding of the oligonucleotide or analog to the target DNA or RNA molecule interferes with the normal function of the target DNA or RNA, and there is a sufficient degree of complementarity to avoid non-specific binding of the oligonucleotide or analog to non-target sequences under conditions where specific binding is desired, for example under physiological conditions in the case of in vivo assays or systems. Such binding is referred to as specific hybridization.<br>
Hybridization conditions resulting in particular degrees of stringency will vary depending upon the nature of the hybridization method of choice and the composition and length of the hybridizing nucleic acid sequences. Generally, the temperature of hybridization and the ionic strength (especially the Na+ and/or Mg++ concentration) of the hybridization buffer will determine the<br>
stringency of hybridization, though wash times also influence stringency. Calculations regarding hybridization conditions required for attaining particular degrees of stringency are discussed by Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989, chapters 9 and 11; and Ausubel et al. Short Protocols in Molecular Biology, 4th ed., John Wiley &amp; Sons, Inc., 1999.<br>
For purposes of the present disclosure, "stringent conditions" encompass conditions under which hybridization will only occur if there is less than 25% mismatch between the hybridization molecule and the target sequence. "Stringent conditions" may be broken down into particular levels of stringency for more precise definition. Thus, as used herein, "moderate stringency" conditions are those under which molecules with more than 25% sequence mismatch will not hybridize; conditions of "medium stringency" are those under which molecules with more than 15% mismatch will not hybridize, and conditions of "high stringency" are those under which sequences with more than 10% mismatch will not hybridize. Conditions of "very high stringency" are those under which sequences with more than 6% mismatch will not hybridize.<br>
"Specific hybridization" refers to the binding, duplexing, or hybridizing of a molecule only or substantially only to a particular nucleotide sequence when that sequence is present in a complex mixture (for example, total cellular DNA or RNA). Specific hybridization may also occur under conditions of varying stringency.<br>
Immune stimulatory composition: A term used herein to mean a composition useful for stimulating or eliciting a specific immune response (or immunogenic response) in a vertebrate. The immune stimulatory composition can be a protein antigen or a plasmid vector used to express a protein antigen. In some embodiments, the immunogenic response is protective or provides protective immunity, in that it enables the vertebrate animal to better resist infection with or disease progression from the organism against which the immune stimulatory composition is directed.<br>
Without wishing to be bound by a specific theory, it is believed that an immunogenic response induced by an immune stimulatory composition may arise from the generation of an antibody specific to one or more of the epitopes provided in the immune stimulatory composition. Alternatively, the response may comprise a T-helper or cytotoxic cell-based response to one or more of the epitopes provided in the immune stimulatory composition. All three of these responses may originate from naive or memory cells. One specific example of a type of immune stimulatory composition is a vaccine.<br>
In some embodiments, an "effective amount" or "immune-stimulatory amount" of an immune stimulatory composition is an amount which, when administered to a subject, is sufficient to engender a detectable immune response. Such a response may comprise, for instance, generation of an antibody specific to one or more of the epitopes provided in the immune stimulatory composition. Alternatively, the response may comprise a T-helper or CTL-based response to one or more of the epitopes provided in the immune stimulatory composition. All three of these responses may originate from naive or memory cells. In other embodiments, a "protective effective amount" of an immune<br>
stimulatory composition is an amount which, when administered to a subject, is sufficient to confer protective immunity upon the subject.<br>
Inhibiting or treating a disease: Inhibiting the full development of a disease or condition, for example, in a subject who is at risk for a disease. A specific example of diseases is rabies. "Treatment" refers to a therapeutic intervention that ameliorates a sign or symptom of a disease or pathological condition after it has begun to develop. As used herein, the term "ameliorating," with reference to a disease, pathological condition or symptom, refers to any observable beneficial effect of the treatment. The beneficial effect can be evidenced, for example, by a delayed onset of clinical symptoms of the disease in a susceptible subject, a reduction in severity of some or all clinical symptoms of the disease, a slower progression of the disease, a reduction in the number of relapses of the disease, an improvement in the overall health or well-being of the subject, or by other parameters well known in the art that are specific to the particular disease.<br>
Isolated: An "isolated" or "purified" biological component (such as a nucleic acid, peptide, protein, protein complex, or particle) has been substantially separated, produced apart from, or purified away from other biological components in the cell of the organism in which the component naturally occurs, that is, other chromosomal and extra-chromosomal DNA and RNA, and proteins. Nucleic acids, peptides and proteins that have been "isolated" or "purified" thus include nucleic acids and proteins purified by standard purification methods. The term also embraces nucleic acids, peptides and proteins prepared by recombinant expression in a host cell, as well as chemically synthesized nucleic acids or proteins. The term "isolated" or "purified" does not require absolute purity; rather, it is intended as a relative term. Thus, for example, an isolated biological component is one in which the biological component is more enriched than the biological component is in its natural environment within a cell, or other production vessel. Preferably, a preparation is purified such that the biological component represents at least 50%, such as at least 70%, at least 90%, at least 95%, or greater, of the total biological component content of the preparation.<br>
Label: A detectable compound or composition that is conjugated directly or indirectly to another molecule to facilitate detection of that molecule. Specific, non-limiting examples of labels include fluorescent tags, enzymatic linkages, and radioactive isotopes.<br>
Nucleic acid molecule: A polymeric form of nucleotides, which may include both sense and anti-sense strands of RNA, cDNA, genomic DNA, and synthetic forms and mixed polymers of the above. A nucleotide refers to a ribonucleotide, deoxynucleotide or a modified form of either type of nucleotide. The term "nucleic acid molecule" as used herein is synonymous with "nucleic acid" and "polynucleotide." A nucleic acid molecule is usually at least 10 bases in length, unless otherwise specified. The term includes single- and double-stranded forms of DNA. A polynucleotide may include either or both naturally occurring and modified nucleotides linked together by naturally occurring and/or non-naturally occurring nucleotide linkages.<br>
Oligonucleotide: A nucleic acid molecule generally comprising a length of 300 bases or fewer. The term often refers to single-stranded deoxyribonucleotides, but it can refer as well to<br>
single- or double-stranded ribonucleotides, RNA.DNA hybrids and double-stranded DNAs, among others. The term "oligonucleotide" also includes oligonucleosides (that is, an oligonucleotide minus the phosphate) and any other organic base polymer.<br>
In some examples, oligonucleotides are about 10 to about 90 bases in length, for example, 12, 13, 14, 15, 16, 17, 18, 19 or 20 bases in length. Other oligonucleotides are about 25, about 30, about 35, about 40, about 45, about 50, about 55, about 60 bases, about 65 bases, about 70 bases, about 75 bases or about 80 bases in length. Oligonucleotides may be single-stranded, for example, for use as probes or primers, or may be double-stranded, for example, for use in the construction of a mutant gene. Oligonucleotides can be either sense or anti-sense oligonucleotides. An oligonucleotide can be modified as discussed above in reference to nucleic acid molecules. Oligonucleotides can be obtained from existing nucleic acid sources (for example, genomic or cDNA), but can also be synthetic (for example, produced by laboratory or in vitro oligonucleotide synthesis).<br>
Open Reading Frame (ORF): A series of nucleotide triplets (codons) coding for amino acids without any internal termination codons. These sequences are usually translatable into a peptide/polypeptide/protein/polyprotein.<br>
It is recognized in the art that the following codons (shown for RNA) can be used interchangeably to code for each specific amino acid or termination: Alanine (Ala or A) GCU, GCG, GCA, or GCG; Arginine (Arg or R) CGU, CGC, CGA, CGG, AGA, or AGG; Asparagine (Asn or N) AAU or AAC; Aspartic Acid (Asp or D) GAU or GAC; Cysteine (Cys or C) UGU or UGC; Glutamic Acid (Glu or E) GAA or GAG; Glutamine (Gin or Q) CAA or CAG; Glycine (Gly or G) GGU, GGC, GGA, or GGG; Histidine (His or H) CAU or CAC; Isoleucine (Ile or I) AUU, AUC, or AUA; Leucine (Leu or L) UUA, UUG, CUU, CUC, CUA, or CUG; Lysine (Lys or K) AAA or AAG; Methionine (Met or M) AUG; Phenylalanine (Phe or F) UUU or UUC; Proline (Pro or P) CCU, CCC, CCA, or CCG; Serine (Ser or S) UCU, UCC, UCA, UCG, AGU, or AGC; Termination codon UAA (ochre) or UAG (amber) or UGA (opal); Threonine (Thr or T) ACU, ACC, ACA, or ACG; Tyrosine (Tyr or Y) UAU or UAC; Tryptophan (Trp or W) UGG; and Valine (Val or V) GUU, GUC, GUA, or GUG. The corresponding codons for DNA have T substituted for U in each instance.<br>
Operably linked: A first nucleic acid sequence is operably linked with a second nucleic acid sequence when the first nucleic acid sequence is placed in a functional relationship with the second nucleic acid sequence. For instance, a promoter is operably linked to a coding sequence is the promoter affects the transcription or expression of the coding sequence. Generally, operably linked DNA sequences are contiguous and, where necessary to join two protein coding regions, in the same reading frame. If introns are present, the operably linked DNA sequences may not be contiguous.<br>
Paratope: That portion of an antibody that is responsible for its binding to an antigenic determinant (epitope) on an antigen.<br>
Pharmaceutically Acceptable Carriers: The pharmaceutically acceptable carriers useful in this disclosure are conventional. Remington's Pharmaceutical Sciences, by E. W. Martin, Mack<br>
Publishing Co., Easton, PA, 15th Edition (1975), describes compositions and formulations suitable for pharmaceutical delivery of one or more therapeutic compounds or molecules, such as one or more SARS-CoV nucleic acid molecules, proteins or antibodies that bind these proteins, and additional pharmaceutical agents.<br>
In general, the nature of the carrier will depend on the particular mode of administration being employed. For instance, parenteral formulations usually comprise injectable fluids that include pharmaceutically and physiologically acceptable fluids such as water, physiological saline, balanced salt solutions, aqueous dextrose, glycerol or the like as a vehicle. For solid compositions (for example, powder, pill, tablet, or capsule forms), conventional non-toxic solid carriers can include, for example, pharmaceutical grades of mannitol, lactose, starch, or magnesium stearate. In addition to biologically-neutral carriers, pharmaceutical compositions to be administered can contain minor amounts of non-toxic auxiliary substances, such as wetting or emulsifying agents, preservatives, and pH buffering agents and the like, for example sodium acetate or sorbitan monolaurate.<br>
Polypeptide: A polymer in which the monomers are amino acid residues joined together through amide bonds. When the amino acids are alpha-amino acids, either the L-optical isomer or the D-optical isomer can be used, the L-isomers being preferred for many biological uses. The terms "polypeptide" or "protein" as used herein are intended to encompass any amino acid molecule and include modified amino acid molecules. The term "polypeptide" is specifically intended to cover naturally occurring proteins, as well as those which are recombinantly or synthetically produced.<br>
Conservative amino acid substitutions are those substitutions that, when made, least interfere with the properties of the original protein, that is, the structure and especially the function of the protein is conserved and not significantly changed by such substitutions. Examples of conservative substitutions are shown below.<br>
	Original Residue	Conservative Substitutions	<br>
Ala	Ser<br>
Arg	Lys<br>
Asn	Gin, His<br>
Asp	Glu<br>
Cys	Ser<br>
Gin	Asn<br>
Glu	Asp<br>
His	Asn; Gin<br>
lie	Leu, Val<br>
Leu	He; Val<br>
Lys	Arg; Gin; Glu<br>
Met	Leu; He<br>
Phe	Met; Leu; Tyr<br>
Ser	Thr<br>
Thr	Ser<br>
Trp	Tyr<br>
Tyr	Trp; Phe<br>
Val	He; Leu<br>
Conservative substitutions generally maintain (a) the structure of the polypeptide backbone in the area of the substitution, for example, as a sheet or helical conformation, (b) the charge or hydrophobicity of the molecule at the target site, or (c) the bulk of the side chain.<br>
Amino acids are typically classified in one or more categories, including polar, hydrophobic, acidic, basic and aromatic, according to their side chains. Examples of polar amino acids include those having side chain functional groups such as hydroxyl, sulfhydryl, and amide, as well as the acidic and basic amino acids. Polar amino acids include, without limitation, asparagine, cysteine, glutamine, histidine, selenocysteine, serine, threonine, tryptophan and tyrosine. Examples of hydrophobic or non-polar amino acids include those residues having nonpolar aliphatic side chains, such as, without limitation, leucine, isoleucine, valine, glycine, alanine, proline, methionine and phenylalanine. Examples of basic amino acid residues include those having a basic side chain, such as an amino or guanidino group. Basic amino acid residues include, without limitation, arginine, homolysine and lysine. Examples of acidic amino acid residues include those having an acidic side chain functional group, such as a carboxy group. Acidic amino acid residues include, without limitation aspartic acid and glutamic acid. Aromatic amino acids include those having an aromatic side chain group. Examples of aromatic amino acids include, without limitation, biphenylalanine, histidine, 2-napthylalananine, pentafluorophenylalanine, phenylalanine, tryptophan and tyrosine. It is noted that some amino acids are classified in more than one group, for example, histidine, tryptophan, and tyrosine are classified as both polar and aromatic amino acids. Additional amino acids that are classified in each of the above groups are known to those of ordinary skill in the art.<br>
Substitutions which in general are expected to produce the greatest changes in protein properties will be non-conservative, for instance changes in which (a) a hydrophilic residue, for example, seryl or threonyl, is substituted for (or by) a hydrophobic residue, for example, leucyl, isoleucyl, phenylalanyl, valyl or alanyl; (b) a cysteine or proline is substituted for (or by) any other residue; (c) a residue having an electropositive side chain, for example, lysyl, arginyl, or histadyl, is substituted for (or by) an electronegative residue, for example, glutamyl or aspartyl; or (d) a residue having a bulky side chain, for example, phenylalanine, is substituted for (or by) one not having a side chain, for example, glycine.<br>
Probes and primers: A probe comprises an isolated nucleic acid molecule attached to a detectable label or other reporter molecule. Typical labels include radioactive isotopes, enzyme substrates, co-factors, ligands, chemiluminescent or fluorescent agents, haptens, and enzymes. Methods for labeling and guidance in the choice of labels appropriate for various purposes are discussed, for example, in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2n ed., vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989 and Ausubel et al. Short Protocols in Molecular Biology, 4th ed., John Wiley &amp; Sons, Inc., 1999.<br>
Primers are short nucleic acid molecules, for instance DNA oligonucleotides 6 nucleotides or more in length, for example that hybridize to contiguous complementary nucleotides or a sequence to be amplified. Longer DNA oligonucleotides may be about 10, 12, 15, 20, 25, 30, or 50 nucleotides<br>
or more in length. Primers can be annealed to a complementary target DNA strand by nucleic acid hybridization to form a hybrid between the primer and the target DNA strand, and then the primer extended along the target DNA strand by a DNA polymerase enzyme. Primer pairs can be used for amplification of a nucleic acid sequence, for example, by the polymerase chain reaction (PCR) or other nucleic-acid amplification methods known in the art. Other examples of amplification include strand displacement amplification, as disclosed in U.S. Patent No. 5,744,311; transcription-free isothermal amplification, as disclosed in U.S. Patent No. 6,033,881; repair chain reaction amplification, as disclosed in WO 90/01069; ligase chain reaction amplification, as disclosed in EP-A-320 308; gap filling ligase chain reaction amplification, as disclosed in 5,427,930; and NASBA™ RNA transcription-free amplification, as disclosed in U.S. Patent No. 6,025,134.<br>
Methods for preparing and using nucleic acid probes and primers are described, for example, in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989; Ausubel et al. Short Protocols in Molecular Biology, 4th ed., John Wiley &amp; Sons, Inc., 1999; and Innis et al. PCR Protocols, A Guide to Methods and Applications, Academic Press, Inc., San Diego, CA, 1990. Amplification primer pairs can be derived from a known sequence, for example, by using computer programs intended for that purpose such as Primer (Version 0.5, © 1991, Whitehead Institute for Biomedical Research, Cambridge, MA). One of ordinary skill in the art will appreciate that the specificity of a particular probe or primer increases with its length. Thus, in order to obtain greater specificity, probes and primers can be selected that comprise at least 20, 25, 30, 35, 40, 45, 50 or more consecutive nucleotides of a target nucleotide sequences.<br>
Protein: A biological molecule, particularly a polypeptide, expressed by a gene and comprised of amino acids.<br>
Purified: The term "purified" does not require absolute purity; rather, it is intended as a relative term. Thus, for example, a purified protein preparation is one in which the subject protein is more pure than in its natural environment within a cell. Generally, a protein preparation is purified such that the protein represents at least 50% of the total protein content of the preparation.<br>
Recombinant nucleic acid: A nucleic acid molecule that is not naturally occurring or has a sequence that is made by an artificial combination of two otherwise separated segments of sequence. This artificial combination is accomplished by chemical synthesis or, more commonly, by the artificial manipulation of isolated segments of nucleic acids, e.g., by genetic engineering techniques such as those described in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989. The term recombinant includes nucleic acids that have been altered solely by addition, substitution, or deletion of a portion of a natural nucleic acid molecule.<br>
Regulatory sequences or elements: These terms refer generally to a class of DNA sequences that influence or control expression of genes. Included in the term are promoters, enhancers, locus control regions (LCRs), insulators/boundary elements, silencers, matrix attachment<br>
regions (MARs, also referred to as scaffold attachment regions), repressor, transcriptional terminators, origins of replication, centromeres, and meiotic recombination hotspots. Promoters are sequences of DNA near the 5'-end of a gene that act as a binding site for DNA-dependent RNA polymerase, and from which transcription is initiated. Enhancers are control elements that elevate the level of transcription from a promoter, usually independently of the enhancer's orientation or distance from the promoter. LCRs confer tissue-specific and temporally regulated expression to genes to which they are linked. LCRs function independently of their position in relation to the gene, but are copy-number dependent. It is believed that they function to open the nucleosome structure, so other factors can bind to the DNA. LCRs may also affect replication timing and origin usage. Insulators (also know as boundary elements) are DNA sequences that prevent the activation (or inactivation) of transcription of a gene, by blocking effects of surrounding chromatin. Silencers and repressors are control elements that suppress gene expression; they act on a gene independently of their orientation or distance from the gene. MARs are sequences within DNA that bind to the nuclear scaffold; they can affect transcription, possibly by separating chromosomes into regulatory domains. It is believed that MARs mediate higher-order, looped structures within chromosomes. Transcriptional terminators are regions within the gene vicinity where RNA polymerase is released from the template. Origins of replication are regions of the genome that, during DNA synthesis or replication phases of cell division, begin the replication process of DNA. Meiotic recombination hotspots are regions of the genome that recombine more frequently than average during meiosis.<br>
Replicon: Any genetic element (e.g., plasmid, chromosome, virus) that functions as an autonomous, self-replicating unit of DNA replication in vivo.<br>
Sample: A portion, piece, or segment that is representative of a whole. This term encompasses any material, including for instance samples obtained from an animal, a plant, or the environment.<br>
An "environmental sample" includes a sample obtained from inanimate objects or reservoirs within an indoor or outdoor environment. Environmental samples include, but are not limited to: soil, water, dust, and air samples; bulk samples, including building materials, furniture, and landfill contents; and other reservoir samples, such as animal refuse, harvested grains, and foodstuffs.<br>
A "biological sample" is a sample obtained from a plant or animal subject. As used herein, biological samples include all samples useful for detection of viral infection in subjects, including, but not limited to: cells, tissues, and bodily fluids, such as blood; derivatives and fractions of blood (such as serum); extracted galls; biopsied or surgically removed tissue, including tissues that are, for example, unfixed, frozen, fixed in formalin and/or embedded in paraffin; tears; milk; skin scrapes; surface washings; urine; sputum; cerebrospinal fluid; prostate fluid; pus; bone marrow aspirates; BAL; saliva; cervical swabs; vaginal swabs; and oropharyngeal wash.<br>
Sequence identity: The similarity between two nucleic acid sequences, or two amino acid' sequences, is expressed in terms of the similarity between the sequences, otherwise referred to as<br>
sequence identity. Sequence identity is frequently measured in terms of percentage identity (or similarity or homology); the higher the percentage, the more similar the two sequences are.<br>
Methods of alignment of sequences for comparison are well known in the art. Various programs and alignment algorithms are described in: Smith and Waterman (Adv. Appl. Math., 2:482, 1981); Needleman and Wunsch (J. Mol. Biol, 48:443, 1970); Pearson and Lipman (Proc. Natl. Acad. Sci., 85:2444, 1988); Higgins and Sharp (Gene, 13:231-AA, 1988); Higgins and Sharp (CABIOS, 5:151-53, 1989); Corpet et al. (Nuc. Acids Res., 16:10881-90, 1988); Huang et al. (Comp. Appls. Biosci, 8:155-65, 1992); and Pearson et al. (Meth. Mol. Biol, 24:307-31, 1994). Altschul et al. (Nature Genet., 6:119-29, 1994) presents a detailed consideration of sequence alignment methods and homology calculations.<br>
The alignment tools ALIGN (Myers and Miller, CABIOS A: 11-17, 1989) or LFASTA (Pearson and Lipman, 1988) may be used to perform sequence comparisons (Internet Program © 1996, W. R. Pearson and the University of Virginia, "fasta20u63" version 2.0u63, release date December 1996). ALIGN compares entire sequences against one another, while LFASTA compares regions of local similarity. These alignment tools and their respective tutorials are available on the Internet at the NCSA website. Alternatively, for comparisons of amino acid sequences of greater than about 30 amino acids, the "Blast 2 sequences" function can be employed using the default BLOSUM62 matrix set to default parameters, (gap existence cost of 11, and a per residue gap cost of 1). When aligning short peptides (fewer than around 30 amino acids), the alignment should be performed using the "Blast 2 sequences" function, employing the PAM30 matrix set to default parameters (open gap 9, extension gap 1 penalties). The BLAST sequence comparison system is available, for instance, from the NCBI web site; see also Altschul et al.,J. Mol. Biol., 215:403-10, 1990; Gish and States, Nature Genet, 3:266-72, 1993; Madden et al, Meth. Enzymoi, 266:131-41, 1996; Altschul et al., Nucleic Acids Res., 25:3389-402, 1997; and Zhang and Madden, Genome Res., 7:649-56, 1997.<br>
Orthologs (equivalent to proteins of other species) of proteins are in some instances characterized by possession of greater than 75% sequence identity counted over the full-length alignment with the amino acid sequence of specific protein using ALIGN set to default parameters. Proteins with even greater similarity to a reference sequence will show increasing percentage identities when assessed by this method, such as at least 80%, at least 85%, at least 90%, at least 92%, at least 95%, or at least 98% sequence identity. In addition, sequence identity can be compared over the full length of one or both binding domains of the disclosed fusion proteins.<br>
When significantly less than the entire sequence is being compared for sequence identity, homologous sequences will typically possess at least 80% sequence identity over short windows of 10-20, and may possess sequence identities of at least 85%, at least 90%), at least 95%, or at least 99% depending on their similarity to the reference sequence. Sequence identity over such short windows can be determined using LFASTA; methods are described at the NCSA website. One of skill in the art will appreciate that these sequence identity ranges are provided for guidance only; it is entirely<br>
possible that strongly significant homologs could be obtained that fall outside of the ranges provided. Similar homology concepts apply for nucleic acids as are described for protein. An alternative indication that two nucleic acid molecules are closely related is that the two molecules hybridize to each other under stringent conditions.<br>
Nucleic acid sequences that do not show a high degree of identity may nevertheless encode similar amino acid sequences, due to the degeneracy of the genetic code. It is understood that changes in nucleic acid sequence can be made using this degeneracy to produce multiple nucleic acid sequences that each encode substantially the same protein.<br>
Specific binding agent: An agent that binds substantially only to a defined target. Thus a protein-specific binding agent binds substantially only the defined protein, or to a specific region within the protein. As used herein, protein-specific binding agents include antibodies and other agents that bind substantially to a specified polypeptide. The antibodies may be monoclonal or polyclonal antibodies that are specific for the polypeptide, as well as immunologically effective portions ("fragments") thereof.<br>
The determination that a particular agent binds substantially only to a specific polypeptide may readily be made by using or adapting routine procedures. Examples of suitable in vitro assays which make use of the Western blotting procedure include IFA and Ag-ELISA, and are described in many standard texts, including Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999.<br>
Transformed: A "transformed" cell is a cell into which has been introduced a nucleic acid molecule by molecular biology techniques. The term encompasses all techniques by which a nucleic acid molecule might be introduced into such a cell, including transfection with viral vectors, transformation with plasmid vectors, and introduction of naked DNA by electroporation, lipofection, and particle gun acceleration.<br>
Vector: A nucleic acid molecule as introduced into a host cell, thereby producing a transformed host cell. A vector may include nucleic acid sequences that permit it to replicate in a host cell, such as an origin of replication (DNA sequences that participate in initiating DNA synthesis). A vector may also include one or more selectable marker genes and other genetic elements known in the art.<br>
Virus: Microscopic infectious organism that reproduces inside living cells. A virus typically consists essentially of a core of a single nucleic acid surrounded by a protein coat, and has the ability to replicate only inside a living cell. "Viral replication" is the production of additional virus by the occurrence of at least one viral life cycle. A virus may subvert the host cells' normal functions, causing the cell to behave in a manner determined by the virus. For example, a viral infection may result in a cell producing a cytokine, or responding to a cytokine, when the uninfected cell does not normally do so.<br>
Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, suitable methods and materials are described below.<br>
All publications, patent applications, patents, and other references mentioned herein are incorporated by reference in their entirety. In case of conflict, the present specification, including explanations of terms, will control. In addition, the materials, methods, and examples are illustrative only and not intended to be limiting.<br>
IV.	Overview of Several Embodiments<br>
Provided herein in a first embodiment is a recombinant rabies virus genome comprising the nucleic acid as set forth in SEQ ID NO: 1 (full length ERA sequence). Also provided are isolated rabies virus proteins encoded by that genome, including specific proteins comprising an amino acid sequence as set forth in SEQ ID NO: 2 (N protein); SEQ ID NO: 3 (P protein); SEQ ID NO: 4 (M protein); SEQ ID NO: 5 (G protein); or SEQ ID NO: 6 (L protein); and isolated nucleic acid molecules that encode such proteins. By way of example, such isolated nucleic acid molecules comprise a nucleotide sequence as set forth in: nucleotides 71-1423 of SEQ ID NO: 1 (N protein); nucleotides 1511-2407 of SEQ ID NO: 1 (P protein); nucleotides 2491-3104 of SEQ ID NO: 1 (M protein); nucleotides 3318-4892 of SEQ ID NO: 1 (G protein); or nucleotides 5418-11,801 of SEQ ID NO: 1 (L protein).<br>
Also provided is a recombinant virus genome with the nucleic acid sequence as set forth in SEQ ID NO: 7, which differs from SEQ ID NO: 1 by virtue of a deletion of one adenosine residue in the polyA tract between the G gene and the psi-region. SEQ ID NO: 7 also encodes the proteins shown as SEQ ID NOs: 2-6.<br>
Also provided are genomes of derivatives of the ERA strain of virus, as shown in SEQ ID NOs: 8-18. In certain embodiments, the genomes are present in a vector, such as a plasmid.<br>
Yet another described embodiment is a system for sequencing full length rabies virus genome using a method as described herein. Viral vector systems for expression of heterologous proteins are also described.<br>
Another embodiment provides compositions that comprise one or more nucleic acid molecules, or one or more proteins, provided herein. Optionally, such compositions contain a pharmaceutically acceptable carrier, an adjuvant, or a combination of two or more thereof.<br>
Also provided is a method of eliciting an immune response against an antigenic epitope in a subject, comprising introducing into the subject a composition comprising a nucleotide, peptide, or polypeptide described herein, thereby eliciting an immune response in the subject.<br>
Another aspect of the disclosure relates to a vector system for producing recombinant rabies virus. The vector system includes a first vector (transcription vector) containing a full-length rabies virus antigenomic DNA (or a derivative thereof) and a set of helper vectors containing nucleic acids that encode at least one rabies virus strain ERA protein. Expression of the vectors in a transfected host cell results in production of a live recombinant rabies virus. In certain embodiments, the antigenomic DNA is of the ERA strain (for example, SEQ ID NO: 1 or SEQ ID NO: 7) or a<br>
derivative thereof, such as one of SEQ ID NOs: 8-18. In certain embodiments, the vectors are plasmids.<br>
To facilitate recovery of full length viral RNA, the transcription vector can include, in a 5' to 3' direction: a hammerhead ribozyme; a rabies virus antigenomic cDNA; and a hepatitis delta virus ribozyme. Nucleotides of the hammerhead ribozyme are selected to be complementary to the antisense genomic sequence of the rabies virus. Transcription of the antigenomic cDNA is under the transcription regulatory control of at least one of the CMV promoter and the phage T7 RNA polymerase promoter, and commonly under the control of both of these promoters.<br>
The helper vectors typically include a vector comprising a polynucleotide sequence that encodes a rabies virus N protein; a vector comprising a polynucleotide sequence that encodes a rabies virus P protein; a vector comprising a polynucleotide sequence that encodes a rabies virus M protein; a vector comprising a polynucleotide sequence that encodes a rabies virus L protein; and a vector comprising a polynucleotide sequence that encodes a phage T7 RNA polymerase. In an embodiment,  the T7 RNA polymerase comprises a nuclear localization signal (NLS). Optionally, the vector system also includes a vector comprising a polynucleotide sequence that encodes a rabies virus G protein.<br>
Transcription of one or more of the polynucleotide sequences that encode the rabies virus P, M, L or G protein or the T7 polymerase is under the transcription regulatory control of both the CMV promoter and the T7 promoter. In contrast, transcription of the polynucleotide sequence that encodes the rabies virus N protein is under the transcription regulatory control of the T7 promoter, and transcription is cap-independent.<br>
Yet more embodiments are live rabies vaccines, each comprising a recombinant rabies virus genome as provided herein. Examples of such recombinant rabies genomes comprise the sequence shown as ERA G333 (SEQ ID NO: 13); the sequence shown as ERA 2G (SEQ ID NO: 8); and the sequence shown herein as ERA 2G333 (SEQ ID NO: 10). Optionally, the rabies vaccine is attenuated.<br>
Also provided is a method of producing a live rabies virus (for example, for use in an immunogenic composition, such as a vaccine) by introducing the vector system into a host cell. After transfection of vector system into a suitable host cell, live and optionally attenuated virus is recovered. Production and administration of a live rabies vaccine produced by such methods is also contemplated herein.<br>
Also disclosed is a method of vaccinating a subject against rabies, which method comprises administering an effective amount of the live rabies vaccine according to the provided description to a subject, such that cells of the subject are infected with the rabies vaccine, wherein an anti-rabies immune response is produced in the subject. In one embodiment, the subject is a human. In another embodiment, the subject is a non-human animal. For instance, the non-human animal in some instances is a cat, dog, rat, mouse, bat, fox, raccoon, squirrel, opossum, coyote, or wolf.<br>
In certain embodiments, the rabies vaccine is administered enterally. For instance, the enteral administration in some cases comprises oral administration. Oral administration includes administration through food-baits designed to vaccinate wild animal populations, for instance.<br>
Pharmaceutical compositions that include the described live rabies vaccines (for instance, an attenuated live rabies vaccine) and a pharmaceutically acceptable carrier or excipient are also provided.<br>
V.	Method of Sequencing Entire Lyssavirus Genome<br>
To facilitate sequencing of the full length ERA genome, a method for sequencing a full length negative strand RNA virus was developed. This method is applicable to the sequencing of a lyssavirus, such as a rabies virus, as well as other negative strand RNA viruses. Rabies virus is a single negative stranded RNA virus with a genome around 12 kb, with the range between 11,918 (Australian bat lyssavirus) and 11,940 (Mokola virus) bases. The available rabies viral nucleic acid sequences in GENBANK mainly focus on the sequences that encode proteins — nucleocapsid protein (N), glycoprotein (G), phosphoprotein (P) and matrix protein (M) genes, which are close to the 3' end of the genome. Prior phylogenetic analysis is mostly based on N and G genes. But, for remotely related rabies viral strains, RNA dependent RNA polymerase (L) gene is the most suitable candidate for phylogenetic analysis. Unfortunately, few L gene sequences are available in public gene databases. In addition, it has been proposed that both leader and trailer regions at rabies viral termini play very important roles for (regulation of) viral transcription and replication. These could be the conserved regions for nucleoprotein encapsidation or the binding sites for L/P proteins, for instance. Also the inter-genic regions among leader-N, N-P, P-M, M-G, pseudo-gene region and G-L serve as the signals for initiation of viral transcription. Thus, not only coding regions, but also non-coding regions within the viral genome, could be applied to phylogenetic analysis or evolution research. These sequences can all more readily be analyzed using the whole-genome sequencing methods provided herein.<br>
The method includes a single step reverse transcription and a two step cloning into a suitable vector. This method produces a readily sequenced genome in the vector, without the need to perform error-prone repeated RT-PCR reactions. Exploiting the inverted repeat found at the ends of the rabies genome (and the genomes of other lyssaviruses), universal primers have been designed and are described herein for use in the rapid full-genome sequencing procedures described herein.<br>
The leader and trailer regions in rabies virus contain signals for viral transcription and replication. Based on analysis the genome sequence available from GenBank, the terminal 11 nucleotides are strictly conserved in rabies viruses or rabies-related viruses, including Mokola virus. The rationale for the sequencing methods provided herein is based on the terminal 11 complementary nucleotides.   Because these two 11 nucleotide sequences are complementary, they could are not used in the follow-up PCR reactions. It will be understood that other viruses with inverted repeats can similarly be amplified using primers corresponding to the sequences of those repeats. The 11<br>
antigenome sense nucleotides were designed as reverse transcription primers for the purified ERA genome, whose integrity was verified by size comparison and Northern blots. The whole genome cDNA was also confirmed by Northern blot with N, P, M, G, L gene probes and 11 nucleotides as an oligonucleotide probe, which only bound genomic RNA, not viral mRNA.<br>
It is reasonably feasible to reverse transcribe rabies viral whole genome in one reaction, using carefully designed conserved terminal sequence-corresponding primes, provided the quality of the viral genome preparation is high.<br>
The sequence of the ERA is closely related to that of SAD, which is one of its derivatives. This is not surprising, because ERA was sent from CDC in the 1970s to Switzerland, where researchers adapted it to grow in cells, before sending it to Germany, where it was further derived, and the derivative fully sequenced in ~ 1990. Until now, rabies and rabies-related viruses have been classified into seven different types: classic rabies virus typel (ERA is included), type2 (Lagos bat), type3 (Mokola), type4 (Duvenhage), type5 (European bat lyssavirus [EBL] I, type6 (EBL II) and type7 (Australian bat virus) according to serum cross protection and genetic studies. Sequence analysis plays an important role in phylogenetics, evolution research, gene function predictive studies and other related areas, including locating viral transcription and replication regulatory regions, and hence bioinformatics towards potential therapeutic drugs.<br>
With the development of techniques in reverse transcriptional polymerase chain reaction (RT-PCR), which are known to those of ordinary skill in the art, now it is relatively easy to reverse transcribe as much as 12 kb or more of RNA to cDNA in one reaction. Under optimized conditions, PCR can amplify targets of more than 30 kb in one reaction.<br>
With the provision herein of methods for generating full-length virus genome sequences, in particularly rabies genome sequences, it now becomes practical to analyze differ strains of virus. . Effective design of attenuated virus, for instance for use in immunization or production of immune stimulatory compositions and vaccines, is also enabled using the resultant full length genomes.<br>
There is no "general" rabies virus genome, but these genomes are related. The similarities range from 60% to 100% in different types. Some regions, such as the L gene, seem more conserved, whereas others, such as the psi region, which does not code for a polypeptide, are more variable. Not only will rabies and rabies related viruses drift, but also any RNA viruses will change over time. How the viruses adapt and emerge, is an open question. For this reason, whole genome sequence analysis is important for evolutionary, pathogenicity and gene function studies.<br>
This system described herein is the first for rabies virus as concerns whole genome sequencing. It is believed to be suitable for other RNA viruses, particularly in the lyssavirus genus. At present, for rabies virus phylogenetic studies, scientists only make use of the N, P, or G genes, which are most abundant in the infected cells or tissues. It is known that for remote strain comparison, the L gene comprising more than half of the genome may be an ideal candidate site, which should be used. Unfortunately, such evolutionary comparisons are not possible due to the very limited data available, let alone the whole genome sequence. Also for viral transcription and<br>
replication studies, it is supposed that the leader and trailer regions located at the 3' and 5' extremities of the genome play important roles. The inter-genic regions are also the signals for viral trans and cis studies. All these data are quite limited, because they are not included in the mRNA. Only the whole genome sequence can provide the necessary information at this level. Whole genome sequencing is useful not only for vaccine development, it is also applicable for basic virus transcription and replication studies. It is also applicable for development siRNA and gene therapy as well.<br>
VI.	ERA Genome Sequence<br>
Using the method described herein, the unique sequence of the ERA rabies virus genome has been generated. This sequence is shown in SEQ ID NO: 1. The five proteins of the ERA rabies virus (SEQ ID NOs: 2-6) are encoded at the following positions of the genome: N, 71-1423; P, 1511-2407; M, 2491-3104; G, 3318-4892; and L, 5418-11801. The homology between ERA and SAD-B19 are: N 99.56%, P 98.65%, M 96.53%, G 99.05% and L 99.20%, respectively. One specific difference between ERA and SAD-B19 is the intergenic region between G and the pseudo-gene, with the SAD-B19 G transcription stop/polyadenylation signal destroyed.<br>
The ERA rabies virus whole genome sequence is the prerequisite for vaccine development and pathogenicity studies using reverse genetics<br>
VII.	Optimized system for production of virus<br>
Examples 6 and 7 provide an optimized set of conditions for ERA virus production, in which titers reach as high as 1010 ffu per ml. In bioreactors, the recovered virus can grow to ~109 to 1010 ffu/ml. Such high levels of production are of paramount importance for oral vaccine development, so sufficient vaccine material can be produced in a reasonable amount of time with reasonable resource allocation.<br>
The provided growth conditions can stably produce such high virus titer for both parental and recombinant ERA strains. These production data are very important for potential rabies oral vaccine development.<br>
VIII.	BSR-G Cell Line for Production of-G Virus<br>
Although strains of RV with deletions of the G protein have been previously rescued from BHK cells, this was not possible with ERA strain virus lacking the G protein. After inoculation of mice intracerebrally or intramuscularly with ERA-G, no mice died or showed any rabies symptoms.<br>
The ERA-G (without glycoprotein) can only grow in cells with the supplementation of the glycoprotein. Otherwise, the mutated virus cannot spread. To help ERA-G grow, a BSR-G cell line was established, which constitutively expressed ERA glycoprotein. Production of this cell line is described in the Examples below. This cell line is useful for recovery of RV strains such as ERA-G that are refractory to recovery in the absence of G, as well as for optimizing recovery of other strains.<br><br>
IX.	Reverse Genetics System for Engineering Rabies Virus Vaccines and Expression of<br>
Heterologous Proteins<br>
RNA cannot readily be manipulated directly by molecular biological methods. Traditional RNA virus vaccines are from naturally attenuated isolates, which are difficult to control and provide unpredictable results. Reverse genetics technology makes it possible to manipulate RNA viruses as DNA, which can be mutated, deleted or reconstructed according to deliberate designs. Every gene function can be studied carefully, independently, and in concert, which benefits vaccine development. Reverse genetics involves reverse transcription of the RNA viral genome into cDNA, and cloning into a vector, such as a plasmid. After transfection of host cells, the vector is transcribed into RNA, to be encapsidated by structural proteins, which can also be supplied by plasmids. The encapsidated RNA forms a ribonucleoprotein complex, which results in virions that can be recovered.<br>
Although three systems for rabies virus (RV) reverse genetics have been published (Schnell etal.,TheEMBOJ. 13, 4195-4203, 1994; Inoue etal., J. Virol. Method. 107, 229-236, 2003; Ito et al., Microbiol. Immunol. 47, 613-617, 2003), these systems are not readily adaptable to other strains. At present, no rabies virus strain has been recovered with the aid of helper plasmids from a different strain, even when the strains are closely related. Thus, for any specific virus strain mutation or vaccine development, a specifically tailored system must be developed.<br>
The ERA strain is a good candidate for rabies oral vaccine development, but its residual pathogenicity is obvious. During the 1970s, the ERA RV underwent extensive vaccine development (Black and Lawson, Can. J. Comp. Med. 44:169-176, 1980; Charlton, and Casey, Can. J. Vet. Res. 20:168-172, 1978; Lawson, and Crawley, Can. J. Vet. Res. 36: 339-344, 1972). Both ERA and SAD-B19 are derived from SAD. In primary oral vaccine trials, SAD-B19 was effective in both raccoons and skunks, while ERA was not. Additionally, ERA kills two-week old mice administrated intra cerebrally (i.e.), as demonstrated in animal tests. These observations raise questions of the relationship between these two RV strains and the potential effects of subtle alterations. From full viral genome sequence comparison, ERA and SAD-B19 share extremely high nucleotide identity and amino acid homologies. To clarify the genetic basis of immunogenicity and pathogenicity of these highly related strains of rabies virus, an efficient reverse genetics system was developed for ERA, which differs from reverse genetics systems previously reported for rabies virus.<br>
The rabies reverse genetics system disclosed herein is useful for a variety of purposes, including: (1) to attenuate ERA virus in a defined manner for vaccine development; (2) to produce ERA virus vectors for expressing heterologous ORFs (e.g., in the context of therapeutic compositions, such as vaccines and gene therapy); (3) to define the genetic basis of ERA RV pathogenesis; and (4) to determine the biological effects of genetic differences between the ERA and SAD viruses.<br>
The reverse genetics system has some or all of the following characteristics, illustrated schematically in FIG. 1A using the exemplary ERA strain antigenomic cDNA.<br>
This system is based a full length transcription plasmid plus a plurality of helper plasmids (e.g., five helper plasmids). The helper plasmids encode the N, P, L proteins, and optionally the G protein, as well as the T7 polymerase. Although the G protein is not necessary for virus rescue, it improves virus recovery efficiency or virus budding when included in transfection.<br>
Transcription involves both cellular RNA dependent RNA polymerase II, which is available in mammalian cells, and T7 RNA polymerase, which is supplied by pNLST7 plasmids. The dual polymerases result in virus recovery efficiency is both high and stable.<br>
In the transcription plasmid, hammerhead and hepatitis delta virus ribozymes flank a rabies virus (e.g., ERA strain) antigenomic cDNA, enabling the production of authentic 5' and 3' ends of antigenomic vRNA by transcription. The first ten nucleotides of the hammerhead sequence are designed to be complementary to the first ten nucleotides of the antisense genomic sequence. For example, the first ten nucleotides of the hammerhead sequence for the ERA antigenomic cDNA are: TGTTAAGCGT (SEQ ID NO: 19).<br>
Two modified T7 RNA polymerase constructs have been established, which support virus recovery more efficiently than the wild type T7 RNA polymerase applied previously. One T7 RNA polymerase has been mutated from the first ATG to AT. The second T7 RNA polymerase has an eight amino acid nuclear localization signal (NLS) derived from the SV40 virus large T antigen fused after the first ATG from the parental T7: ATG CCA AAA AAG AAG AGA AAG GTA GAA (SEQ ID NO: 20). The NLS is underlined. Addition of the NLS results in the T7 RNA polymerase being present predominantly in the nucleus. Following transfection mechanism of the NLS modified plasmid, the DNA/transfection reagent complex binds to surface of the cell. Through endocytosis, the complex is taken into the endosome/lysosome, and the DNA is released into the cytosol. In the absence of the NLS, the majority of the transfected plasmids are retained in the cytosol and only a small percentage of the released DNA reaches the nucleus, where it is transcribed into RNA. After protein synthesis, the NLST7 RNA polymerase is transported back to the cell nucleus, and the helper plasmids (with T7/CMV promoters) in the nucleus will be transcribed by both NLST7 and cellular polymerasell. Thus, more mRNAs of the helper plasmids and cRNA of the full-length pTMF or its derivatives were synthesized and resulted in high efficiency of virus recovery.<br>
After the initial expression of NLST7 by CMV promoter, NLST7 polymerase binds to pT7 for transcription of NLST7 gene. Through modification of the transcripts in the nucleus, more NLST7 mRNA is synthesized, resulting in more expression of NLST7 polymerase. The pT7 of the NLST7 polymerase as well as of the full length antigenomic transcription unit is under the control of the NLST7 polymerase, which acts as an "autogene." The autogene mechanism of NLST7 RNA polymerase is illustrated in FIG. 2. After expression of T7 RNA polymerase in the nucleus, the transfected T7 constructs continue to transcribe full length RNA template for N protein encapsidation and/or L protein binding, enhancing virus recovery efficiency.<br>
The T7 polymerase, and all other plasmids, except the N protein encoding plasmid pTN, are placed under control of both CMV and T7 transcriptional regulatory elements. The N protein<br>
encoding nucleic acid is under the control of a T7 promoter and is translated in cap-independent manner based on an IRES (Internal Ribosome Entry Site). Cellular RNA polymeraseII alone can help the recovery of RV if all the plasmids were cloned under the control of the CMV promoter (19). In the ERA reverse genetics system disclosed herein, only pTN is under the control of the T7 promoter and was translated in a cap-independent manner. All other constructs are under control of both CMV and the T7 transcriptional regulatory elements. Typically, in RV, N synthesis is abundant and the ratio among N, P and L is approximately 50:25:1. To mimic the wild type viral transcription and assembly in RV reverse genetics, N expression should be the highest. With the aid of NLST7 polymerase and IRES translation mode, N protein was expressed efficiently after plasmid transfection. This reduces competition for transcription with house keeping genes in host cells, because the T7 transcription initiation signal does not exist in mammalian cells, and results in increased efficiency of T7 transcription.<br>
To enhance production of viral proteins, the helper plasmids can be constructed to incorporate a Kozak sequence that has been optimized for the translation efficiency for each protein encoding sequence. Exemplary optimized Kozak sequences are shown in Table 2.<br>
 (Table Removed)<br>
Table 2: Optimized Kozak sequences.<br>
(Table Removed)<br>
CMV/T7 symbolizes the CMV promoter ahead of a pT7 promoter. The HdRz indicates a hammerhead ribozyme and HDVRz is the hepatitis delta virus ribozyme. The pTMF is the full-length transcription plasmid, and the pTN, pMP, pMG, pML and pNLST7 are helper plasmids.<br>
After five days posttransfection in the ERA reverse genetics system, the rescued virus reliably and repatably grew to l07 ffu/ml without further amplification.<br>
X.	Derivative Viruses<br>
The complete mechanism of Rabies virus pathogenicity has not been fully characterized, making rational vaccine design problematic. For example, the RV glycoprotein appears to play a role both in pathogenicity and immunogenicity of rabies virus. Mutations (such as at position 333 of the glycoprotein) result in virus that does not cause lethal infection in adult mice (Ito et al., Microl. Immunol. 38, 479-482, 1994; Ito et al.,J. Virol. 75,^9121-9128, 2001). However, overexpression of RV glycoprotein has been shown to lead to the enhancement of apoptosis and antiviral immune response (Faber et al, J. Virol. 76, 3374-3381, 2002). Thus ERA strain virus with a modified (for<br>
example, deleted, amino acid substituted) G protein could be a particular strain for vaccine development.<br>
Recombinant rabies viruses with favorable properties can be designed using the reverse genetics system disclosed herein. Exemplary recombinant viruses disclosed herein include, in addition to the parental ERA strain, ERA without Psi-region (ERA-), ERAgreenl (green fluorescent gene inserted in the Psi-region), ERAgreen2 (green fluorescent gene cloned at P-M intergenic region), ERA2g (containing an extra copy of G in Psi-region), ERAg3 (G mutated at 333 amino acid), ERA2g3 (containing an extra copy of mutated G in Psi-region), ERAgm (M and G genes switched in the genome), and ERAgmg (two copies of G in the rearranged ERAgm construct). These exemplary strains are illustrated schematically in FIG. 3.<br>
Modified strains having deleted and/or mutated glycoproteins are particularly suited for use as immunogenic compositions for pre and post exposure treatment of rabies virus because such viruses are incapable of spreading between cells and causing disease. Additionally, modified viruses such as ERA2g3, which overexpresses the G protein due to a duplication of sequences encoding a mutated glycoprotein is predicted enhance apoptosis and elicit an increased anti-viral immune response.<br>
For example, after intracerebral and intramuscular inoculation of mice with a deletion of G (ERA-G), no adverse events were observed. Moreover, the ERA-G protected mice from lethal challenge by a street RV strain. Thus, ERA-G appears to be a safer strain that ERA for vaccine development. Additionally, mutation of arginine at amino acid position 333 of the ERA G to glutamic acid (from nucleotides AGA to GAG, as in the ERAg3 and ERA2g3 strains) results in an attenuated virus. Attenuation was confirmed via animal inoculation tests. Because overexpression of RV G results in the enhancement of apoptosis and antiviral immune responses, attenuated viruses such as ERA2g3 that possess multiple copies of G are particularly favorable as vaccine candidates.<br>
The system for rabies vaccine development described herein is not limited to modifications of the G gene, but is similarly applicable to each of the viral proteins. To facilitate a systematic approach to modifying the various protein components, detailed mapping of pathogenicity can be solved by reverse genetics based on the sequence data presented herein.<br>
The reverse genetics system described herein also enables a rabies virus vector system for foreign (heterologous) gene expression. The described, non-limiting embodiment is based on the ERA virus. An extra transcription unit is shown herein to be functional in two different locations after incorporation into the ERA RV genome. In one embodiment, an extra transcription unit is incorporated in the position of the psi region (trans 1). In an alternative embodiment, an extra transcription unit is inserted into the RV P-M intergenic region.<br>
In single stranded negative RNA viruses, the 3'-distal sequences of the genome serve mainly as a transcription promoter, while the 5'-terminal sequences of the genome serve as a replication promoter (Conzelmann and Schnell, J. Virol. 68:713-719, 1994; Finke et al, J. Virol. 71:7281-7288, 1997). Thus, trans2 occupies a position that results in stronger transcription for driving ORFs<br>
expression than trans 1. Thus, the vectors disclosed herein can be used to modulate expression of a heterologous ORF to a desired level, simply by selecting the position into which the ORF is inserted in the vector. For example, when high levels of expression of a protein is desired, the trans 2 is typically an ideal position for the insertion of the heterologous ORF. Similarly, if more moderate levels of expression are desired, the heterologous ORF can be inserted into trans 1. Optimal expression levels for each ORF and for particular applications can be determined by one of skill in the art without undue experimentation.<br>
Thus, the viral vectors provided herein are excellent construct for foreign gene insertion and expression, as is demonstrated herein with respect to expression of the green fluorescent protein gene. Although the utility and efficacy of the disclosed vectors is demonstrated with respect to GFP, it should be noted that the vectors are equally suitable for expressing any gene or ORF of interest.<br>
As noted, the rabies-based heterologous expression system provided herein can be used to express any foreign (heterologous) protein(s). It is particularly contemplated, by way of example, that such heterologous genes are from another pathogenic organism, such as other pathogenic viruses, for instance SARS virus, Nipah virus, etc. In addition, the disclosed vectors can be used for delivery of other therapeutic genes, including for example, that encode proteins of therapeutic value or functional RNA molecules, such as siRNAs.<br>
XI.	Pharmaceutical and Immune Stimulatory Compositions and Uses Thereof<br>
Pharmaceutical compositions including attenuated or fixed rescued viruses, virus nucleic acid sequences or virus polypeptides comprising at least one virus epitope are also encompassed by the present disclosure. These pharmaceutical compositions include a therapeutically effective amount of one or more active compounds, such as an attenuated or fixed virus, a virus polypeptides comprising at least one virus epitope, or one or more nucleic acid molecules encoding these polypeptides, in conjunction with a pharmaceutically acceptable carrier. It is contemplated that in certain embodiments, virus nucleic acid sequences or virus polypeptides comprising multiple virus epitopes will be useful in preparing the pharmaceutical compositions of the disclosure.<br>
Disclosed herein are substances suitable for use as immune stimulatory compositions for the inhibition or treatment (either pre-exposure or post-exposure) of a virus infection, for example, a rabies virus infection.<br>
In one embodiment, an immune stimulatory composition contains an attenuated or fixed rescued (recombinant) virus. In another embodiment, the composition contains an isolated or recombinant virus polypeptide including at least one virus epitope (such as a rabies virus G protein). In a further embodiment, the immune stimulatory composition contains a nucleic acid vector that includes at least one virus nucleic acid molecule described herein, or that includes a nucleic acid sequence encoding at least one virus epitope. In a specific, non-limiting example, a nucleic acid sequence encoding at least one virus epitope is expressed in a transcriptional unit, such as those<br>
described in published PCT Application Nos. PCT/US99/12298 and PCT/US02/10764 (both of which are incorporated herein in their entirety).<br>
The immune stimulatory viruses, virus polypeptides, constructs or vectors encoding such polypeptides, are combined with a pharmaceutically acceptable carrier or vehicle for administration as an immune stimulatory composition to human or animal subjects.<br>
The immunogenic formulations may be conveniently presented in unit dosage form and prepared using conventional pharmaceutical techniques. Such techniques include the step of bringing into association the active ingredient and the pharmaceutical carrier(s) or excipient(s). In general, the formulations are prepared by uniformly and intimately bringing into association the active ingredient with liquid carriers. Formulations suitable for parenteral administration include aqueous and nonaqueous sterile injection solutions which may contain anti-oxidants, buffers, bacteriostats and solutes which render the formulation isotonic with the blood of the intended recipient; and aqueous and nonaqueous sterile suspensions which may include suspending agents and thickening agents. The formulations may be presented in unit-dose or multi-dose containers, for example, sealed ampoules and vials, and may be stored in a freeze-dried (lyophilized) condition requiring only the addition of a sterile liquid carrier, for example, water for injections, immediately prior to use. Extemporaneous injection solutions and suspensions may be prepared from sterile powders, granules and tablets commonly used by one of ordinary skill in the art.<br>
In certain embodiments, unit dosage formulations are those containing a dose or unit, or an appropriate fraction thereof, of the administered ingredient. It should be understood that in addition to the ingredients particularly mentioned above, formulations encompassed herein may include other agents commonly used by one of ordinary skill in the art.<br>
The compositions provided herein, including those for use as immune stimulatory compositions, may be administered through different routes, such as oral, including buccal and sublingual, rectal, parenteral, aerosol, nasal, intramuscular, subcutaneous, intradermal, and topical. They may be administered in different forms, including but not limited to solutions, emulsions and suspensions, microspheres, particles, microparticles, nanoparticles, and liposomes.<br>
The volume of administration will vary depending on the route of administration. By way of example, intramuscular injections may range from about 0.1 ml to about 1.0 ml. Those of ordinary skill in the art will know appropriate volumes for different routes of administration.<br>
A relatively recent development in the field of immune stimulatory compounds (for example, vaccines) is the direct injection of nucleic acid molecules encoding peptide antigens (broadly described in Janeway &amp; Travers, Immunobiology: The Immune System In Health and Disease, page 13.25, Garland Publishing, Inc., New York, 1997; and McDonnell &amp; Askari, N. Engl. J. Med. 334:42-45, 1996). Vectors that include nucleic acid molecules described herein, or that include a nucleic acid sequence encoding a virus polypeptide comprising at least one virus epitope may be utilized in such DNA vaccination methods.<br>
Thus, the term "immune stimulatory composition" as used herein also includes nucleic acid vaccines in which a nucleic acid molecule encoding a virus polypeptide comprising at least one virus epitope is administered to a subject in a pharmaceutical composition. For genetic immunization, suitable delivery methods known to those skilled in the art include direct injection of plasmid DNA into muscles (Wolff et ai, Hum. Mol. Genet. 1:363, 1992), delivery of DNA complexed with specific protein carriers (Wu et ai, J. Biol. Chem. 264:16985, 1989), co-precipitation of DNA with calcium phosphate (Benvenisty and Reshef, Proc. Natl. Acad. Sci. 83:9551, 1986), encapsulation of DNA in liposomes (Kaneda et al., Science 243:375, 1989), particle bombardment (Tang et ai, Nature 356:152, 1992; Eisenbraun et ai, DNA Cell Biol. 12:791, 1993), and in vivo infection using cloned retroviral vectors (Seeger et ai, Proc. Natl. Acad. Sci. 81:5849, 1984). Similarly, nucleic acid vaccine preparations can be administered via viral carrier.<br>
The amount of immunostimulatory compound in each dose of an immune stimulatory composition is selected as an amount that induces an immunostimulatory or immunoprotective response without significant, adverse side effects. Such amount will vary depending upon which specific immunogen is employed and how it is presented. Initial injections may range from about 1 ug to about 1 mg, with some embodiments having a range of about 10 ug to about 800 ug, and still other embodiments a range of from about 25 ug to about 500 ug. Following an initial administration of the immune stimulatory composition, subjects may receive one or several booster administrations, adequately spaced. Booster administrations may range from about 1 ug to about 1 mg, with other embodiments having a range of about 10 ug to about 750 ug, and still others a range of about 50 ug to about 500 ug. Periodic boosters at intervals of 1-5 years, for instance three years, may be desirable to maintain the desired levels of protective immunity.<br>
It is also contemplated that the provided immunostimulatory molecules and compositions can be administered to a subject indirectly, by first stimulating a cell in vitro, which stimulated cell is thereafter administered to the subject to elicit an immune response. Additionally, the pharmaceutical or immune stimulatory compositions or methods of treatment may be administered in combination with other therapeutic treatments.<br>
The preparation of food-baits containing immune stimulatory compositions is also within the ordinary skill of those in the art. For example, the preparation of food-baits containing live RV vaccines is disclosed in Wandeler et al. (Rev. Infect. Dis. 10 (suppl. 4):649-653, 1988), Aubert et al. (pp. 219-243, in Lvssaviruses (Rupprecht et ai, eds.), Springer-Verlag, New York, 1994), and Fu et al. (pp. 607-617, in New Generation Vaccines (2nd Edit.) (Levine et ai, eds.), Marcel Dekker, Inc., New York, 1997), the entire disclosures of each of which are incorporated by reference herein.<br>
XII.        Kits<br>
Also provided herein are kits useful in the detection and/or diagnosis of virus infection, for instance infection with a rabies virus or other lyssavirus. An example of an assay kit provided herein is a recombinant virus polypeptide (or fragment thereof) as an antigen and an erLzryme-conjugated<br>
anti-human antibody as a second antibody. Examples of such kits also can include one or more enzymatic substrates. Such kits can be used to test if a sample from a subject contains antibodies against a virus-specific protein. In such a kit, an appropriate amount of a virus polypeptide (or fragment thereof) is provided in one or more containers, or held on a substrate. A virus polypeptide can be provided in an aqueous solution or as a freeze-dried or lyophilized powder, for instance. The container(s) in which the virus polypeptide(s) are supplied can be any conventional container that is capable of holding the supplied form, for instance, microfuge tubes, ampoules, or bottles.<br>
The amount of each polypeptide supplied in the kit can be any appropriate amount, and can depend on the market to which the product is directed. For instance, if the kit is adapted for research or clinical use, the amount of each polypeptide provided would likely be an amount sufficient for several assays. General guidelines for determining appropriate amounts can be found, for example, in Ausubel et al. (eds.), Short Protocols in Molecular Biology, John Wiley and Sons, New York, NY, 1999 and Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999.<br>
The following examples are provided to illustrate certain particular features and/or embodiments. These examples should not be construed to limit the invention to the particular features or embodiments described.<br>
EXAMPLES Example 1: Sequencing of ERA RV<br>
This example provides a description of a method for sequencing the full length genome of a rhabdovirus, particularly in this case a rabies virus.<br>
Rabies virus strain ERA was obtained from the CDC archive and was propagated in baby hamster kidney (BHK-21) cells. Virus was harvested after four days infection at 37°C, in a 5% C02 incubator and was purified. Briefly, the cell supernatant was collected and centrifuged at 2,000 rpm for 15 minutes to remove the cell debris. The clear supernatant was subjected to further centrifugation at 18,000 rpm for 1 hour. The pellet was resuspended in PBS and subjected to rabies genomic RNA extraction.<br>
Total RNA from ERA-infected BHK-21 cells was extracted with Trizol reagent (GIBCO Invitrogen) according to the protocol recommended by the manufacturer. ERA genomic RNA was purified from the concentrated ERA virus supernatant with a high pure viral RNA kit from Roche.<br>
Integrity of the purified ERA genomic RNA was verified by gel electrophoresis and Northern blot by N, P, G, and M hybridization probes. Briefly, 5 ug of genomic RNA was loaded in a denatured RNA gel and transferred to a nylon membrane for hybridization. The probe was labeled using the Dig DNA labeling kit from Roche, according to manufacturer's instructions.<br>
The 11 conserved nucleotides from the rabies virus 5' antigenome were designed as a primer for reverse transcription. The RT reaction was carried out with a first-strand cDNA synthesis kit from Invitrogen. The complete cDNA from the ERA genome was confirmed by Northern blot using<br>
N, P, M, and G probe hybridization, as well as the 11 conserved nucleotides as oligonucleotide probes labeled by Digoxin.<br>
Two sets of primers were chosen for PCR reactions, which amplify the whole ERA genome in two contiguous fragments. One set of primers is composed of the 11 nucleotides at 5' antigenome end, Le5: ACGCTTAACAA (SEQ ID NO: 24) and BLp3: GTCGCTTGCTAAGCACTCCTGGTA (SEQ ID NO: 25). Another set contains the 11 complementary nucleotides at the 5' genome end, Le3: TGCGAATTGTT (SEQ ID NO: 26) and BLp5 CCAG GAGTGCTTAGCAAGCGACCT (SEQ ID NO: 27). The Blp3 and Blp5 primers are located in a relatively conserved region in the rabies virus genome.<br>
PCR fragments were purified and cloned into the TOPO vector purchased from Invitrogen. Sequencing was conducted in an ABI 310 sequencer and the sequence was assembled by BioEdit software or SeqMerge software from Accelrys in the GCG environment.<br>
The complete aligned sequence of the ERA genome is provided in SEQ ID NO: 1. The positions of individual protein encoding sequences are provided in Table 3, with reference to SEQ ID NO: 1. The amino acid sequences of the N, P, M, G and L proteins are provided in SEQ ED NOs: 2 through 6, respectively.<br>
Table 3: Positions of protein encoding sequences of rabies virus ERA strain<br>
(Table Removed)<br>
This method can be used for both rabies and rabies-related viruses. Rabies and rabies-related viruses have at least seven putative species types. The provided sequence method can be used also for other negative stranded RNA viruses. This is because almost all the negative-stranded RNA virus genomes have approximately 12 conserved nucleotides at both distal ends, which similarly can serve as primers for RT-PCR. The primers will of course be different for different viral species, and the sequence of specific primers can be determined by one of ordinary skill based on the teachings herein.<br>
Example 2: Construction of plasmids for a reverse genetics system for rabies virus<br>
This example describes the design and development of a Reverse Genetics System for Rabies Virus. Rabies virus strain ERA was obtained from the ATCC and was prepared as described (Wu et al, J. Virol. 76, 4153-4161, 2002). To obtain virus genome full-length virus cDNA, BSR cells (a clone of baby hamster kidney, BHK, cells) were infected with ERA strain virus and grown in Dulbecco's minimal essential medium supplemented with 10% of fetal bovine serum. Supematants were recovered and subjected to centrifugation at 22,000 g for lhour. The virus pellets were collected for viral genomic RNA purification by use of a RNA virus extraction kit purchased from Qiagen (Valencia, CA) according to the manufacturer's instructions. The integrity of viral genomic RNA was confirmed by gel electrophoresis. Viral genomic cDNA was transcribed with the first-strand cDNA synthesis kit from Invitrogen (Carlsbad, CA). The reverse transcription (RT) reaction mixture was applied to amplification by the polymerase chain reaction (PCR) for the synthesis of full-length viral genomic cDNA, N, P, G and L genes, respectively. For assembling the full-length virus genomic cDNA, a pTMF plasmid was constructed in four sequential steps as illustrated schematically in FIG. IB. Superscript III reverse transcriptase and proofreading platinum pfx polymerase (Invitrogen, Carlsbad, CA) were applied for cDNA transcript synthesis and consecutive PCR amplifications. For reverse transcription reactions, 1 µg of purified genomic RNA was used in the RT reaction mix and incubated at 50 °C for 80 min, followed by heating at 85 °C for 5 minutes to inactivate Superscript III. After the RT reaction, 1 unit of RNaseH was added to digest template RNA in the cDNA-RNA hybrids.<br>
To generate full-length virus genomic cDNA, two overlapping fragments were amplified by RT-PCR as follows: Fragmentl (Fl) was RT-PCR amplified with primers: Le5-Kpn (CCGGGTACCACGCTTAAC AACCAGATCAAAGA; SEQ ID NO: 28, Kpnl recognition site underlined) and Le3-Blp (TAGGTCGCTTGCTAAGCACTCCTGGTAGGAC; SEQ ID NO: 29, Blpl recognition site underlined). Fragment 2 (F2) was RT-PCR amplified with primers: Tr5-Blp (GTCCTACCAGGAGTGCTTAGCAAGCGACCTA: SEQ ID NO: 30, Blpl recognition site underlined) and Tr3-Pst (AAAACTGCAGACGCTTAACAAATAAACAACAAAA; SEQ ID NO: 31, Pstl recognition site underlined). After successful synthesis of the above two fragments, Fl digested by Kpnl and Blpl restriction enzymes was subjected to gel purification and cloned to pBluescriptIISK(+) phagemid (Stratagene, La Jolla, CA) to form the pSKFl plasmid. The gel purified F2 fragment, cut by Blpl and Pstl was consecutively cloned to the pSKFl plasmid to form the full-length viral antigenomic cDNA. Hammerhead ribozyme (oligol,<br>
CAAGGCTAGCTGTTAAGCGTCTGATGAGTCCGTGAGGACGAAACTATAGGAAAGGAAT TCCTATAGTCGGTACCACGCT; SEQ ID NO: 32, Nhel and Kpnl recognition sites underlined; 01igo2,<br>
AGCGTGGTACCGACTATAGGAATTCCTTTCCTATAGTTTCGTCCTCACGGACTCATCAGA CGCTTAACAGCTAGCCTTG: SEQ ID NO: 33, Kpnl and Nhel recognition sites underlined) was synthesized containing a Nhe 1 recognition site at the 5' end and a Kpn 1 site at the 3' end. This was<br>
fused ahead of the 5' end of the Fl fragment. A hepatitis delta virus ribozyme (oligo3, GACCTGCAGGGGTCGGCATGGCATCTCCACCTCCTCGCGGTCCGACCTGGGCATCCGAA GGAGGACGCACGTCCACTCGGATGGCTAAGGGAGGGCGCGGCCGCACTC; SEQ ID NO: 34 , Pstl and Notl recognition sites underlined; 01igo4,<br>
GAGTGCGGCCGCGCCCTCCCTTAGCCATCCGAGTGGACGTGCGTCCTCCTTCGGATGCC CAGGTCGGACCGCGAGGAGGTGGAGATGCCATGCCGACCCCTGCAGGTC: SEQ ID NO: 35, Notl and Pstl recognition sites underlined)(Symons, Annu. Rev. Biochem. 61: 641-671, 1992) was synthesized, having a Pstl site at its 5' end and a Notl site at its 3' end, and was fused to the 3' end of the F2 fragment. The connective Kpnl recognition site, between the hammerhead ribozyme and the Fl fragment, and the Pstl site between the F2 fragment and the hepatitis delta virus ribozyme, were deleted by site-directed mutagenesis. The full-length viral antigenomic cDNA was sandwiched by the hammerhead and hepatitis delta virus ribozymes. This was removed and cloned to the pBluescriptIISK(+) phagemid to make a pSKF construct. The full viral antigenomic cDNA with two ribozymes was fused downstream of the T7 transcription initiation site under control of the CMV immediate-early promoter in pcDNA3.1/Neo (+) plasmid (Invitrogen, Carlsbad, CA). This last step finished the construction of the pTMF plasmid.<br>
The wild type ERA viral genome includes a polyA tract of eight residues (polyA8) in the intergenic region between the G and Psi regions. To distinguish the rescued ERA (rERA) virus from the parental strain, a stretch of seven A (polyA7) was introduced to the pTMF construct by deletion of one A instead of the original polyA8. After rERA virus was recovered, RT-PCR was performed and subsequent sequence data confirmed the existence of the introduced poly A7 sequence marker.<br>
pTN plasmid: The N gene was amplified by RT-PCR with primers (5N: ACCACC17UGATGCCGACAAGATTG; SEQ ID NO: 36, Ncol recognition site and start codon underlined; and 3N: GGCCCATGG77MTGAGTCACTCGAATATGTCTT; SEQ ID NO: 37, Ncol recognition site and stop codon underlined) and cloned to the pCITE-2a(+) ( Cap-Independent Translation Enhancer) plasmid (Novagen, Madison WI).<br>
pMP plasmid: the P gene was amplified by RT-PCR with primers (5P: TTGGTACCACC^rGAGCAAGATCTTTGTCAATC; SEQ ED NO: 38, Kpnl recognition site and start codon underlined; and 3P: GGAGAGGAATTC77MGCAAGATGTATAGCGATTC; SEQ ID NO: 39, EcoRl recognition site and stop codon underlined) and cloned to the pcDNA3.1/Neo (+) plasmid.<br>
pMG plasmid: the G gene was amplified by RT-PCR with primers (5G: TTGGIACCACC4TOGTTCCTCAGGCTCTCCTG; SEQ ID NO: 40, Kpnl recognition site and start codon underlined; and 3G: AAAACTGCAGrC4CAGTCTGGTCTCACCCCCAC: SEQ ID NO: 41, Pstl recognition site and stop codon underlined) and cloned to the pcDNA3.1/Neo (+) plasmid.<br>
pML plasmid: the L gene was amplified by RT-PCR with primers (5L: ACCGCTAGCACCACC47UCTCGATCCTGGAGAGGTC: SEQ ID NO: 42, Nhel recognition site<br>
and start codon underlined; and 3L: AAAACTGCAG7TQ4CAGGCAACTGTAGTCTAGTAG; SEQ ID NO: 43, Pstl recognition site and stop codon underlined) and cloned to the pcDNA3.1/Neo (+) plasmid.<br>
pT7 plasmid: genomic DNA from bacteria BL-21(Novagene, Madison, WI) was extracted with the Dneasy Tissue Kit (Qiagen, Valencia, CA) according to the manufacturer's instructions. The T7 RNA polymerase gene was amplified from the purified genomic DNA by PCR with primers (5T7: TCGCTAGCACCACC47GAACACGATTAACATCGCTAAG: SEQ ID NO: 44, Nhel recognition site and start codon underline; and 3T7:<br>
GATGAATTC7T4CGCGAACGCGAAGTCCGACTC: SEQ ID NO: 45, EcoRl recognition site and stop codon underlined) and cloned to the pcDNA3.1/Neo (+) plasmid.<br>
pNLST7 plasmid: an eight amino acid nuclear location signal (NLS), derived from SV40 large T antigen, was added to the N terminus of the T7 RNA polymerase by PCR amplification, using the pT7 plasmid as the template, with primers (5T7NLS:<br>
TCGCTAGCCACCATGCCAAAAAAGAAGAGAAAGGTAGAAAACACGATTAACATCGCTA AGAAC; SEQ ID NO: 46, NLS underlined and 3T7 primer). The amplified fragment was designated NLST7, and was cloned to pcDNA3.1/Neo (+) to form the pNLST7 construct.<br>
pGFP plasmid: Monster Green Fluorescent Protein(GFP) plasmid phMGFP was purchased from Promega (Madison, WI). The GFP gene was amplified by PCR with primers (GFP5: AAAACTGCAGGCCACCATGGGCGTGATCAAG; SEQ ID NO: 47, Pstl recognition site and start codon underlined; and GFP3: CCGCTCGGTACCTA7T.4GCCGGCCTGGCGGG; SEQ ID NO: 48, Kpnl recognition site and stop codon underlined) and cloned to the pcDNA3.1/Neo (+) plasmid.<br>
All plasmid constructs were sequenced at least three times to confirm the absence of unexpected mutations or deletions after cloning, site-directed mutagenesis, or gene deletion. Additionally, presence of a marker sequence consisting of a poly A tract having seven adenosine residues rather than the eight residues observed in the wild type ERA genome between the glycoprotein and Psi region was confirmed.<br>
Example 3: T7 RNA polymerase expression in BSR cells<br>
This example demonstrates that addition of a nuclear localization signal to the phage T7 RNA polymerase directs expression of the polymerase in the nucleus of transfected cells. Transfection of BSR cells was performed as described by Wu, et al. {J. Virol. 76, 4153-4161, 2002). Briefly, BSR cells near 80% confluence in a six-well-plate were transfected with 0.5 µg of pT7 or pNLST7 plasmid per well, respectively. At 48 hours after transfection, cells were fixed with 80% chilled acetone for lh and dried at room temperature. Mouse monoclonal antibody against the T7 RNA polymerase and goat anti mouse IgG-FITC conjugate were successively added, and were washed in the two-step indirect fluorescent staining procedure. Results were recorded after UV microscopy. The T7 RNA polymerase expressed from pT7 without a nuclear localization signal<br>
was observed primarily in the cytosol, whereas NLST7 polymerase including a nuclear localization signal was present predominantly in the nucleus of cells. These results indicated that addition of an NLS effectively targeted the T7 KNA polymerase to the nucleus of transfected cells.<br>
Example 4: Establishment of constitutively expressed ERA glycoprotein BSR cell line<br>
This example describes the design and production of a BHK cell line that constituively expresses the ERA glycoprotein. A BHK cell line that expresses the ERA glycoprotein was constructed using the Flp-In™ system (Invitrogen, Carlsbad, CA). Briefly, Flp-In™-BHK cells (containing a single integrated Flp recombination target site) were grown to approximately 20% confluence in one six-well-plate and maintained in common DMEM medium, supplemented with 100 µg/ml of Zeocin, before transfection. The ERA G gene was amplified by PCR using pMG plasmid as template with primers EF5G5 (CACCATGGTTCCTCAGGCTCTCCTG; SEQ ED NO: 49) and EF5G3 (TCACAGTCTGGTCTCACCCCCAC; SEQ ID NO: 50), and cloned to a pEF5/FRT/V5-D-TOPO vector (Invitrogen, Carlsbad, CA) to create the pEFG construct. The pOG44 plasmid expressing Flp recombinase together with pEFG at the ratio of 10:1 was co-transfected to the Flp-In™-BHK cells. After transfection, the cells were kept in DMEM without Zeocin, but with hygromycin B at 400ug/ml. After 48 hours, the cells were split so that no more than 20% confluency occurred the next day. The cells were grown in hygromycin B selective medium at 37°C for approximately one week. The target ERA G expression was detected by indirect fluorescent staining with human anti-G monoclonal antibody and goat anti-human IgG-FITC conjugate. The cell line constitutively expressing the G was designated as BHK-G, and was used for the growth of ERA-G virus.<br>
Example 5: Defined Modification of Rabies Virus Evelyn-Rokitnicki-Abelseth (ERA) Strain<br>
In addition to the parental ERA virus strain described above, derivative virus strains were developed using the reverse genetics system disclosed herein. Several exemplary modified viruses were produced, namely ERA- (deletion of the whole psi-region), ERAgreenl (green florescent protein gene inserted in ERA viral genome psi region), ERAgreen2 (green florescent protein gene inserted in phosphoprotein and matrix protein intergenic region), ERA2g (containing an extra copy of glycoprotein in the psi-region), ERAg3 (with a mutation at amino acid 333 in glycoprotein), ERA2g3 (with an extra copy of mutated glycoprotein at Aa333 in psi-region), ERA-G (with glycoprotein deleted) ERAgm (M and G genes switched in the genome), and ERAgmg (two copies of G in the rearranged ERAgm construct) These derivatives are illustrated schematically in FIG. 3. By optimizing the growth conditions as described, all of the rescued viruses can be obtained at virus titers of 109to 1010 ffu/ml in both tissue culture flasks and bioreactors.<br>
Gene deletion and site-directed mutagenesis in the reverse genetics system<br>
Deletion of the Psi resign of the rabies virus ERA genome<br>
The complete Psi-region of the rabies virus ERA genome was deleted as follows: 3'fragment was amplified using pTMF as template by PCR with primers (5: CCCTCTGCAGTTTGGTACCGTCGAGAAAAAAACATTAGATCAGAAG; SEQ ID NO: 51, Pstl and Kpnl recognition sites underlined; and Le3-Blp primer) and was cloned to pCR-Bluntll -TOPO vector (Invitrogen, Carlsbad, CA) for the construction of pP5 plasmid. The 5'fragment was amplified using the same template by PCR with primers (SnaB5: ATGAACTTTCTACGTAAGATAGTG; SEQ ID NO: 52, SnaBl recognition site underlined; and<br>
53, Pstl recognition site underlined) was successively cloned to the above pP5 plasmid to finish the construction of the pP plasmid. The fragment recovered by SnaBl and Pstl restriction enzyme digestion from the pP plasmid substituted the counterpart in the pSKF construct to make the pSKFplasmid. The whole DNA fragment containing the ERA genomic cDNA, digested by Nhel and Notl from pSKF plasmid, was re-cloned to the pcDNA3.1/Neo (+) plasmid to finalize the construction of pTMF. For verification of the rescued strain lacking Psi, designated Era-, primers covering the Psi-region were applied in RT-PCR with total RNA from ERA-infected BSR cells. A 400bp fragment corresponding to the Psi region was amplified only from rERA virus, but not from ERA. Sequence data verified the complete deletion of the Psi-region. Deletion of the glycoprotein gene in the rabies virus ERA genome: The 5'g fragment was amplified using pSKF as template by PCR with primers (SnaB5<br>
ID NO: 54). After digestion by SnaBl and Pstl restriction enzymes, this recovered fragment was cloned to replace its counterpart in the pSKF construct. The 3'g fragment was amplified using the same template by PCR with primers (5Ag:<br>
CCTCTGCAGTTTGGTACCTTGAAAAAAACCTGGGTTCAATAG; SEQ ID NO: 55, and Le3-Blp primer), and was consecutively cloned to the modified pSKF, to replace its counterpart. The final fragment, recovered by SnaBl and Blpl restriction enzymes cut from this pSKF without the G gene, was re-cloned to pcDNA3.1/Neo (+) plasmid to form the pTMFAg construct for virus recovery.<br>
Glycoprotein gene site-directed mutagenesis:<br>
Site directed mutagenesis to introduce a three nucleotide change from AGA to GAG at amino acid position 333 of the glycoprotein was performed as previously described (Wu et al, J. Virol. 76: 4153-4161, 2002). The primers in the mutagenesis reaction were M5G primer: CTCACTACAAGTCAGTCGAGACTTGGAATGAGATC (SEQ ID NO: 56, the three mutated nucleotides in bold) and M3G primer: GACTGACTTTGAGTGAGCATCGGCTTCCATCAAGG (SEQ ID NO: 57). For the recovered strain (ERAg3), three nucleotide changes from AGA to GAG at amino acid position 333 (aa333) were confirmed by sequencing after RT-PCR with primers 5G<br>
and 3G. After confirmation by DNA sequencing, the mutated G was cloned back to the pTMF plasmid to make the pTMFg3 construct for virus recovery. The glycoprotein encoded by this mutated G gene is represented by SEQ ID NO: 58. Incorporation of an exogenous ORF into ERA rabies virus genome<br>
To express exogenous ORFs in RV, an extra transcription unit with Pstl and Kpnl recognition sites were created and incorporated at the Psi or P-M gene intergenic regions, respectively. In brief, for creation of an extra transcription unit at the Psi-region, the same steps were followed, except for the 5' fragment amplification step, the3 primer was changed to 3cis:<br>
CCAAACTGCAGCGAAAGGAGGGGTGTTAGTTTTTTTCATGATGAACCCCCCAAGGGGA GG (SEQ ID NO: 59). The final construct without the Psi-region, but with an extra transcription unit, was designated as pMTFcis. The GFP, ERA G, or mutated G at amino acid residues 333 were cloned to this transcriptional unit to form pMTFgfpl, pMTF2g, pMTFg3, pMTF2g3 constructs, respectively, for virus rescue.<br>
To incorporate an extra transcription unit to the P-M intergenic region, the cisp5 fragment was amplified using pMTF as template with primers cis55:<br>
GACTCACTATAGGGAGACCCAAGCTGGCTAGCTGTTAAG (SEQ ID NO: 60), cis53: CCAAACTGCAGCGAAAGGAGGGGTGTTAGTTTTTTTCATGTTGACTTTAGGACATCTCG G (SEQ ID NO: 61), and was cloned in substitution of its counterpart in the pMTF plasmid. The cisp3 fragment was amplified and cloned in a similar way with primers cis35: CCTTTCGCTGCAGTTTGGTACCGTCGAGAAAAAAACAGGCAACACCACTGATAAAATG AAC (SEQ ID NO: 62) and cis33: CCTCCCCTTCAAGAGGGCCCCTGGAATCAG (SEQ ED NO: 63). After assembling the cisp5 and cisp3 fragments together, the final construct was designated as pMTFcisp, for accepting ORFs. The recombinant construct containing the GFP gene was named pTMFgfp2 for virus recovery.<br>
To produce an ERA derivative, designated ERAgm, in which the glycoprotein encoding sequence was reversed in order with the matrix protein encoding sequence, the glycoprotein gene was deleted as described above. The G gene (amplified as disclosed above) was then inserted between P and M genes, yielding a rabies virus genome in the order of N-P-G-M-L. Similarly, the same strategy was applied to produce the ERAg3m derivative, in which the glycoprotein has a triple nucleotide mutation at 333 amino acid residue (from AGA to GAG) by substituting the G gene produced by site directed mutagenesis as described above. To produce the ERAgmg construct, an extra copy of glycoprotein gene was inserted between P and M genes, and made the rabies virus genome in the order of N-P-G-M-G-L.<br>
An extra transcription unit was modified and incorporated into two different regions of the ERA genome, namely psi-region and P-M intergenic region. When heterologous ORFs are incorporated into these transcription units, designated trans 1 and trans 2, respectively, efficient production of the encoded product results.<br>
Sequence of the transcription unit is:<br>
CTAACACCCCTCCTTTCGCTGCAGTTTGGTACCGTCGAGAAAAAAA (SEQ ID NO: 64, Pstl and Kpnl were underlined).<br>
Example 6: Recovery of parental and derivative viruses<br>
This example describes the recovery of parental ERA virus and exemplary derivatives using the reverse genetics system disclosed herein. BSR cells were transfected at near 80% confluence in six-well-plates with viral full length transcription plasmid pTMF (pTMF, pTMFg3, pTMF2g, pTMF2g3, pTMFgfpl, pTMFgfp2, pTMFAg, pTMFgm, or pTMFgmg, respectively) at 3ng/well, together with five helper plasmids: pTN (lµg/well), pMP   (0.5µg/well), pML (0.5µg/well), pMG (0.5µg/well) and pNLST7 (lµg/well) by TransIT-LTl reagent (Mirus, Madison, Wl) following the protocol recommended by the manufacturer. Four days after transfection, 1ml of fresh BSR cell suspension (about 5x105 cells) was added to each well. Cells were incubated at 37°C, 5%C02for 3 days. Cell supernatants were collected for virus titration.<br>
To titrate the recovered virus, monolayers of BSR cells in LAB-TEK eight-well-plates (Naperville, IL) were infected with serial 10-fold dilutions of virus supernatant and incubated at 37°C, 0.5% C02 for 48 h. Cells were fixed in 80% chilled acetone at room temperature for lh and stained with FITC-labeled anti-rabies virus N monoclonal antibody at 37°C for 30 minutes. After three rinses of the plates with PBS, stained foci were counted using direct fluorescent microscopy. Details for direct RV fluorescent assay (DFA) can be found on the world wide web at cdc.gov/ncidod/dvrd/rabies/professional/publications/DFA-diagnosis/DFA_protocol.hrm.<br>
All of the viruses except ERA-G were recovered at high titer from cultured BSR cells as indicated in Table 3. Surprisingly, rearrangement and switching of the G gene with the M gene did not hinder recovery of recombinant derivative ERA virus. Rearrangement of the G gene in the RV genomes was previously not believed feasible due to cell death from overexpression of G protein (Faber et al.,J. Virol. 76:3374-3381, 2002). However, these results demonstrate that rearrangement is possible in the ERA strain. Accordingly, it is likely that RV gene shuffling is possible not only for the G gene, but also for other genes as well.<br>
The ERA-G (without G) virus was recovered after plasmid transfection following the same procedure as for the other viral constructs rescue, but virus foci were very limited and restrained in local areas after the first round of transfection. The rescued virus was not capable of spreading further to the nearby healthy BSR cells (FIG. 4A) even after one week of incubation at 37 °C, 5%C02. Infection of normal BSR cells with the above transfection supernatants presented single cell staining in the DFA test, which suggested the recovered virus was incapable of spread. To amplify the ERA-G virus, a BHK cell line constitutively expressing ERA G was established as described in Example 4 (designated BHK-G). By indirect fluorescent assay screening, a pool of BHK cells expressing G were selected and maintained for amplification of ERA-G virus (FIG. 4B). With the aid of the BHK-G cell line, ERA-G virus grew to 107ffu/ml. Total RNA from ERA-G<br>
virus- infected BHK-G cells was extracted for Northern blot analysis (FIG. 4C) with a G gene probe. The G gene was absent in the viral genomic RNA, however G mRNA was detected, which came from infected supportive BHK-G cells. In purified ERA-G viral genomic RNA, no hybridization signal was detected by G probe, indicating the deletion of the G gene in the ERA genome.<br>
Example 7: Growth of rescued ERA virus and its derivatives to high titer in a bioreactor.<br>
In oral vaccine development, high virus titer is typically required to elicit reliable immunity after administration. This example demonstrates that the ERA virus and derivatives can be grown to high titer in a bioreactor at volumes applicable to commercial scale-up. All 10 rescued ERA viruses were amplified in a bioreactor, CELLine AD 1000 (IBS Integra Bioscience, Chur, Switzerland) to titers ranging from 107to 1010 ffu/ml. In brief, BSR cells were transfected with the exemplary antigenome transcription vectors and helper vectors, as described above. Cells were inoculated at a multiplicity of infection of 1 virion per cell, at a concentration of 10s cells/ml in one tenth the bioreactor vessel volume. Transfected cells were grown at 37 °C, 5% Co2 in DMEM supplemented with 10% fetal bovine serum. The supernatant was harvested every three to five days for between two and three harvests. The deficient ERA-G grew less well compared with other viruses, with only 108ffu/ml for the ERA-G (TABLE 3. and FIG. 5).<br>
Table 3. Full-length plasmid constructs and corresponding rescued viruses<br>
(Table Removed)<br>
Example 8: Expression of exogenous proteins from extra transcriptional units in rabies virus<br>
This example demonstrates the expression of recombinant proteins from a heterologous ORF inserted into a rabies virus vector. In this example, the ERA virus vector is used as a prototype rabies virus vector. To construct ERA virus as a vector for accepting ORFs, a conservative RV transcriptional unit between the N and P genes was modified and introduced into the ERA genome at two different locations: 1) at the psi region (trans 1), and 2) at the P-M intergenic region (trans 2). The transcriptional unit was designed to possess two unique restriction enzyme recognition sites to facilitate introduction of heterologous polynucleotide sequences:<br>
(TTTTTTTGATTGTGGGGAGGAAAGCGACGTCAAACCATGGCAGCTCTTTTTTT: SEQ ED NO: 65, Pstl and Kpnl sites underlined).<br>
In a first example, the GFP gene was cloned into this unit for virus recovery, since GFP expression could be observed directly under a UV microscope when the transfected BSR cells were still incubating. Expression of the GFP protein was directly visible by fluorescent microscopy with an excitation filter of 470+20nm. The ERAgreen2 (GFP gene inserted after P gene in RV genome-trans 2)-infected cells showed clear green foci after three days of plasmid transfection, while ERAgreenl (GFP gene inserted after G gene in the "traditional"  region-trans 1) did not present obvious green foci until five days post-transfection (FIG. 6). The introduced transcriptional unit was functional in the RV genome at both locations, although expression and accumulation was apparent more rapidly when GFP was expressed from trans 2. Thus, these results also indicate that the level of expression from a heterologous ORF can be modulated by selecting the transcription unit into which the ORF is cloned.<br>
In other examples, 1) an additional copy of ERA G; or 2) an additional copy of ERA G with an amino acid substitution at position 333, was incorporated into the ERA viral genome. The successfully rescued viruses were named ERA2g and ERA2g3, respectively. Since quantitation of viral G expression was not practical, the relative increase in expression levels of G in ERA2g and ERA2g3-infected cells was confirmed by Northern-blot with a G probe. In brief, the ERA G gene probe was labeled using the Dig DNA Labeling Kit (Roche, Indianapolis, IN) and imaged with Dig Nucleic Acid Detection Kit (Roche, Indianapolis, IN) and was measured by density spectrophotometry (FIG. 7). The tandem linked G genes in the recovered viruses were also confirmed by RT-PCR with 5G and 3G primers. A predominant band indicating a single G copy was observed at 1.5kb. In addition, a second weaker band was observed at approximately 3.0 kb indicative of the two Gs in a tandem arrangement.<br>
These results demonstrate that introduction of transcription units into the ERA genome can be used to express diverse heterologous proteins from introduced ORFs. Furthermore, expression of the protein encoded by the heterologous ORF is modulated by the position into which the ORF is inserted. Thus, ERA virus is a widely adaptable vector for the expression of recombinant proteins.<br>
Example 9: In vivo immune response to engineered viruses<br>
This example demonstrates the in vivo effects of inoculation with the engineered ERA virus and exemplary derivatives. All animal care and experimental procedures were performed in compliance with the CDC Institutional Animal Care and Use Guidelines. Eighty three-week old mice were divided into 8 groups of 10 each for intramuscular (i.m) administration of recovered viruses (106 ffu of virus per mouse). Ten healthy mice were held as uninfected mock controls. For the ERA and ERAg3 constructs, additional intracerebral (i.c) injections of the same dose of viruses were applied to another group often three-week old mice. In two-day old suckling mice, only the ERAg3 and ERA-G viruses were inoculated intracerebrally, with the same dose. Animals were<br>
checked daily for illness. Ill animals were euthanized by CO2 intoxication and brains were removed for rabies virus diagnosis. Ten days after infection, blood was collected by the retro orbital route and sera obtained for neutralizing antibody assays, following the standard rapid fluorescent focus inhibition test (RFFIT) (Smith et al, Bulletin of the World Health Organization. 48: 535-541, 1973). One monfh.after infection, surviving animals were challenged with a lethal dose of street rabies virus (dog/ coyote salivary gland homogenate) (Orciari et al, Vaccine. 19:4511-4518,2001).<br>
Mouse monoclonal antibody (Mab 523-11) against rabies virus G was maintained at CDC (Hamir et al, Vet Rec. 136, 295-296,1995) and FITC-conjugated anti-N monoclonal antibody was purchased from Centocor (Horsham, PA). T7 RNA polymerase monoclonal antibody was from Novagen (Madison, WI). Goat anti-mouse IgG-FITC conjugate was purchased from Sigma-Aldrich (St. Louis, MO). Anti-rabies virus G monoclonal antibody (Mab 1-909) was maintained at CDC and goat anti-human IgG-FITC conjugate was purchased from Sigma-Aldrich (St. Louis, MO).<br>
Among the three-week old mice inoculated intramuscularly by the eight different virus constructs, 50% of mice inoculated with ERA (rERA) or ERA-, and 20% of mice inoculated with ERAgreenl showed mild neurological signs at 10 days after inoculation. No other groups showed any sign suggestive of rabies virus infection (FIG. 8A). Sera were collected for neutralizing antibody titration before challenge. The ERA2g (5.60 IU) and ERA2g3 (5.61 IU) elicited higher titers than the single-copy G virus constructs (FIG. 8E). Mice surviving one month after inoculation were subjected to challenge with a lethal dog /coyote street virus (0.05ml, maintained at CDC for standard animal challenge tests). In the ERA and ERA- groups, 40 to 62% of the mice showed mild rabies signs, respectively, and were euthanized. All other groups survived without any signs of rabies (FIG. 8B). In the i.c groups, three-week old mice survived after ERAg3 inoculation, but succumbed after ERA injection (FIG. 8C). The ERA-G construct did not kill 2-day old suckling mouse, however ERAg3 was virulent enough to kill all infected suckling mice (FIG. 8D). Exemplary antibody titers are shown in Table 4.<br>
Table 4: Production of rabies specific antibodies<br>
(Table Removed)<br>
These data demonstrate that all of the ERA based viruses were capable of eliciting an immune response following inoculation. As expected, the parental ERA virus was virulent, resulting in substantial morbidity and mortality in infected animals. In contrast, the vaiipus exemplary derivatives elicited a protective immune response when mice were inoculated prior to challenge.<br>
In addition to the pre-exposure assessment described above, the ability of the ERA virus derivatives to elicit a protective immune response following infection with virulent rabies virus was determined. In brief, groups of hamsters were infected with one of three different strains of rabies virus (n=9 per group), and either given the recombinant vaccine (ERA-g333), or rabies immune globulin plus inactivated commercial rabies vaccines. Approximately 80-100% of control animals succumb, whereas approximately 60-100% of vaccinated animals survive as shown in FIGS. 9A-C. These results demonstrate that post-exposure administration of the derivative rabies virus confers substantial protection against different strains of rabies virus.<br>
In view of the many possible embodiments to which the principles of the disclosed invention may be applied, it will be recognized that the illustrated embodiments are only preferred examples of the invention and should not be taken as limiting the scope of the invention. Rather, the scope of the invention is defined by the following claims. We therefore claim as our invention all that comes within the scope and spirit of these claims.<br><br><br><br><br><br>
CLAIMS<br>
We claim:<br>
1.	A vector system comprising:<br>
a first vector comprising a full-length rabies virus antigenomic DNA, wherein the full-length<br>
antigenomic DNA is selected from a full-length rabies virus antigenomic DNA or a<br>
derivative thereof; and,<br>
a plurality of helper vectors comprising nucleic acids that encode at least one rabies virus strain ERA protein,<br>
wherein expression of the plurality of vectors in a transfected host cell results in production of a recombinant rabies virus.<br>
2.	The vector system of claim 1, wherein the full-length antigenomic DNA comprises a ERA strain antigenomic DNA or a derivative thereof.<br>
3.	The vector system of claim 2, wherein the full-length antigenomic DNA is selected from SEQ ID NO: 8, SEQ ID NO:9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, SEQ ID NO: 17 or SEQ ID NO: 18.<br>
4.	The vector system of any one of claims 1-3, wherein the vectors are plasmids.<br>
5.	The vector system of any one of claims 1-4, wherein the first vector comprises in a 5' to 3' direction: a hammerhead ribozyme; a rabies virus antigenomic cDNA; and a hepatitis delta virus ribozyme, wherein a plurality of nucleotides of the hammerhead ribozyme are complementary to the antisense genomic sequence of the rabies virus.<br>
6.	The vector system of claim 5, wherein transcription of the antigenomic cDNA is under the transcription regulatory control of at least one of the CMV promoter and the phage T7 RNA polymerase promoter.<br><br>
7.	The vector system of claim 6, wherein transcription of the antigenomic cDNA is under the transcription regulatory control of both of the CMV promoter and the phage T7 RNA polymerase promoter.<br>
8.	The vector system of any one of claims 1-4, wherein the plurality of helper vectors comprises:<br>
a vector comprising a polynucleotide sequence that encodes a rabies virus N protein; a vector comprising a polynucleotide sequence that encodes a rabies virus P protein;<br>
a vector comprising a polynucleotide sequence that encodes a rabies virus M protein; a vector comprising a polynucleotide sequence that encodes a rabies virus L protein; and a vector comprising a polynucleotide sequence that encodes a phage T7 RNA polymerase.<br>
9.	The vector system of claim 8, further comprising a vector comprising a polynucleotide sequence that encodes a rabies virus G protein.<br>
10.	The vector system of claim 8 or 9, wherein the T7 RNA polymerase comprises a nuclear localization signal (NLS).<br>
11.	The vector system of claim 10, wherein transcription of the polynucleotide sequence that encodes the rabies virus N protein is under the transcription regulatory control of the T7 promoter, and wherein transcription is cap-independent.<br>
12.	The vector system of any of claims 8-11, wherein transcription of one or more of the polynucleotide sequences that encode the rabies virus P, M,.L or G protein or the T7 polymerase are under the transcription regulatory control of both the CMV promoter and the T7 promoter.<br>
13.	A recombinant virus genome comprising the nucleic acid as set forth in SEQ ID NO: 1.<br>
14.	A recombinant virus genome comprising the nucleic acid as set forth in SEQ ID NO: 7, or a derivative thereof.<br>
15.	The recombinant virus genome of claim 14, wherein the derivative virus genome comprises SEQ ID NO: 8, SEQ ED NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, SEQ ID NO: 17 or SEQ ID NO: 18.<br><br>
16.	The recombinant virus genome of any one of claims 13-15, further comprising a vector.<br>
17.	The recombinant virus genome of claim 16, wherein the vector is a plasmid.<br>
18.	A recombinant virus comprising a genome as set forth in claim 14 or 15.<br>
19.	The recombinant virus of claim 18, wherein the virus is an attenuated virus.<br>
20.	A live rabies vaccine comprising at least one recombinant rabies virus genome,<br>
wherein the at least one recombinant rabies genome comprises:<br>
the sequence shown in SEQ ID NO: 8; the sequence shown in SEQ ID NO: 10; or the sequence shown in SEQ ID NO: 13.<br>
21.	The rabies vaccine of claim 20, which is attenuated.<br>
22.	An isolated protein comprising the amino acid sequence as set forth in:<br>
SEQ ID NO: 2 (N protein);<br>
SEQ ID NO: 3 (P protein); SEQ ID NO: 4 (M protein); SEQ ID NO: 5 (G protein); SEQ ID NO: 6 (L protein); or SEQ ID NO: 58 (G protein Aa333).<br>
23	An isolated nucleic acid molecule encoding any one of the proteins according to<br>
claim 22.<br>
24.	The isolated nucleic acid molecule of claim 23, comprising a nucleotide sequence<br>
as set forth in:<br>
a)	nucleotides 71-1423 of SEQ ID NO: 1 (N protein);<br>
b)	nucleotides 1511-2407 of SEQ ID NO: 1 (P protein);<br>
c)	nucleotides 2491-3104 of SEQ ID NO: 1 (M protein);<br>
d)	nucleotides 3318-4892 of SEQ ID NO: 1 (G protein);<br>
e)	nucleotides 5418-11,801 of SEQ ED NO: 1 (L protein), or<br>
f)	a nucleotide sequence that differs from one of a) through e) due solely to the degeneracy of the genetic code.<br><br>
25.	The isolated nucleic acid molecule of claim 23, comprising a nucleotide sequence as set forth in nucleotides 3317-4888 of SEQ ID NO: 8, or a nucleotide sequence that differs therefrom due solely to the degeneracy of the genetic code<br>
26.	A composition comprising at least one isolated protein of claim 22.<br>
27.	The composition of claim 26, further comprising a pharmaceutically acceptable carrier, an adjuvant, or a combination of two or more thereof.<br><br>
28.        A method for sequencing full length rabies virus genome comprising:<br>
reverse transcribing a rabies virus genome to produce a complementary DNA strand;<br>
amplifying a first portion and a second portion of the complementary DNA strand to produce a first and second amplified rabies virus genomic segment;<br>
cloning the first and second amplified rabies virus genomic, segments into a vector to produce a contiguous rabies virus antigenome; and<br>
sequencing the full length rabies virus antigenome.<br>
29. A method of producing a live rabies virus vaccine, comprising introducing the vector system of any one of claims 8-19 into a host cell, and recovering live recombinant rabies virus.<br>
30. The method of claim 30, wherein the recovered live recombinant rabies virus is suitable for use as a live rabies virus vaccine.<br>
31.	A live rabies virus vaccine produced by the method of claim 31.<br>
32	A pharmaceutical composition comprising the live rabies vaccine of claim 20 or<br>
claim 32 and a pharmaceutically acceptable carrier or.excipient.<br>
33.    The pharmaceutical composition of claim 39, wherein the rabies vaccine is attenuated.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUNsYWltcy0oMTAtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Claims-(10-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUNsYWltcy0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Claims-(27-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1ERUxOUC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3390-DELNP-2008-Correspondence Others-(10-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Correspondence-Others-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Correspondence-Others-(27-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUZvcm0tMTgtKDEyLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Form-18-(12-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUZvcm0tMi0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Form-2-(27-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1ERUxOUC0yMDA4LUZvcm0tMy0oMTAtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3390-DELNP-2008-Form-3-(10-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUZvcm0tMy0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Form-3-(27-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LUdQQS0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-GPA-(27-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM5MC1kZWxucC0yMDA4LVBldGl0aW9uLTEzNy0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3390-delnp-2008-Petition-137-(27-02-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259082-gas-barrier-laminate.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259084-a-method-for-processing-one-or-more-ordered-sequences-of-pcm-audio-frames.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259083</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3390/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Feb-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Feb-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Apr-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GOVERNMENT OF THE UNITED STATES OF AMERICA AS REPRESENTED BY THE SECRETARY OF THE DEPARTMENT OF HEALTH AND HUMAN SERVICES, CENTERS FOR DISEASE CONTROL AND PREVENTION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>DISEASE CONTROL AND PREVENTION TECHNOLOGY TRANSFER OFFICE, 4770 BUFORD HWY (K79), ATLANTA, GA 30341, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHARLES E. RUPPREHT</td>
											<td>647 MICHELLE COURT, LAWRENCEVILLE, GA 30044, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>XIANFU WU</td>
											<td>3128 BRIARCLIFF ROAD, ATLANTA, GA 30329, U.S.A</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/867 C07H21/04;</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/040134</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-10-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/727,038</td>
									<td>2005-10-14</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259083-a-vector-system by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:21:45 GMT -->
</html>
