<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259104-a-method-for-producing-biologically-active-polypeptide-having-insulinotropic-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:16:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259104:A METHOD FOR PRODUCING BIOLOGICALLY ACTIVE POLYPEPTIDE HAVING INSULINOTROPIC ACTIVITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR PRODUCING BIOLOGICALLY ACTIVE POLYPEPTIDE HAVING INSULINOTROPIC ACTIVITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The insulinotropic peptide glyexendin-4, is overexpressed in Pichia Pastor is host (with ste 13 gene disruption) using a codon optimized encoding sequence with a optional signal peptide and/or propeptide. The expression of the exogenous gene is driven by the methanol inducible promoter AOX. The purified peptide from the recombinant organism is alpha amidated at the penultimate serine. The production of glyexendin-4 may also be carried out in bacteria such as E. coli or in yeast such as Saccharomyces cereviside. The purified reombinant peptide is useful in formulating pharmaceutical compositions for teh treatment of iseases such as diabetes and obesity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHOD OF FULL LENGTH PEPTIDE EXPRESSION<br>
Field of the Invention<br>
The present invention provides a system for producing an insulinotropic peptide (e.g., exendin-4) by cloning and expressing the native peptide or an engineered form of the peptide in a genetically modified microorganisms such as yeast wherein the aminopeptidase gene is disrupted, to enable efficient expression of the full length protein or polypeptide.<br>
Background of the Invention<br>
Exendins are a family of peptides that lower blood glucose levels have some sequence similarity (53%) to GLP-1 (Goke et al9 1993 1 Biol. Chem. 268; 19650-19655; incorporated herein by reference). The exendins are found in the venom of the Gila Monster (Raufman 1996 Reg. Peptides, 61; 1-18; incorporated herein by reference) and the Mexican beaded lizard. Exendin-4 found in the venom of the Gila Monster (Heloderma suspectum) is of particular interest. Published PCT international patent application, WO 98/35033, which is incorporated herein by reference, discloses the cDNA encoding proexendin peptide, including exendin and other novel peptides, its isolation, and antibodies which specifically recognize such peptides (Pohl et al 1998, J. Biol Chem. 273; 9778-9784; incorporated herein by reference). Exendin-4 has been shown to be a strong GLP-1 agonist in isolated rat insulinoma cells. This is expected since the His-Ala domain of GLP-1 recognized by DPP-IV is not present in exendin-4, which has the sequence His-Gly instead (Goke et. al, 1993,./ Biol Chem. 268; 19650-19655; incorporated herein by reference).<br>
Exendin-4 given systematically lowers blood glucose levels by 40% in diabetic db/db mice (WO 99/07404; incorporated herein by reference). U.S. Patent 5,424,286 by Eng, which is incorporated herein by reference, discloses that a considerable portion of the N-terminal sequence is essential in order to preserve insulinotropic activity.<br>
The number of amino acid residues in a peptide sequence has a dramatic influence on the production costs of peptides made by solid phase synthetic chemistry. The cost of manufacturing a 50-mer peptide is at least 5 times greater than the cost of a 10-mer peptide.<br><br>
Solid phase peptide synthesis also requires the use of corrosive solvents such as TFS and hydrofluoric acids, a number of synthetic steps required to produce the peptide, and subsequent purification of the peptide. Therefore, there is a need for efficiently and cost effectively producing large quantities of biologically active exendin-4 and other insulinotropic peptides for the treatment of these conditions.<br>
Expression of recombinant protein or polypeptide in Pichia or other microorganisms does not always result in successful production of full length polypeptide. Often, the heterologus recombinant polypeptide/protein are subjected to cleavage/degradation by proteases intracellularly. Given the structure of a protein and the multiplicity of proteases present in the cell; specificity of the amino acid sequence recognized by or targeted by the protease is in many cases undetermined and not published. It is therefore not possible for a person skilled in the art to guess which protease would be responsible for cleavage/degradation of a specific recombinant polypeptide/protein. For eg., it has been reported that expression of murine or human endostatin in pichia led to a product which was missing C-terminal lysine (Folkman et al, , 1999 May;15(7):563-72.) When pichia KEX-1 protease homologue of Saccharomyces cerevisiae was disrupted, the pichia host secreted full length endostatin into the medium. In another study it was reported that disruption of KEX-2, but not YPS-1 gene, in pichia allowed production of mammalian gelatin, which was being cleaved at monoargylinic sites of the protein (Werten and de Wolf, Applied and Environmental Microbiology, 2005 May;71(5):2310-7).<br>
The   instant   invention   proves   the   prevention   of   in   vivo   proteolytic   cleavage   of proteins/polypeptide having the amino acids HG (His-Gly) at the N-terminus with the disruption of Saccharomyces cerevisiae STE13 gene homolog of Pichia, Very specifically the problem associated with proteolytic cleavage of Glycine-extended Exendin-4 (GlyExendin-4, Exendin-4 with an extra Glycine at the C-terminus) has been shown to be solved by disruption of Saccharomyces cerevisiae STE13 gene homolog of Pichia. GLYEXENDIN-4 HGEGTFTSDLSKQMEEEAVRLFIEWLKNGGPSSGAPPPSG (SEQ ID NO: 2)<br><br>
Objects of the Present Invention:<br>
•	The principal object of the present invention is to produce a biologically active<br>
polypeptide.<br>
•	Another object of the present invention is to produce an insulinotropic peptide, exendin -<br>
4.<br>
•	Yet another object of the present invention is to develop a method for producing the<br>
insulinotropic peptide.<br>
•	Yet another object of the present invention is to develop a method of expressing a full<br>
length polypeptide from Pichia host cell.<br>
•	Yet another object of the present invention is to develop a method of amidating the<br>
insulinotropic peptide.<br>
Statement of the Invention<br>
The present invention provides a system for preparing insulinotropic peptides by efficient expression of heterologous gene in microorganisms. This system provides for large-scale recombinant production of peptides precursor such as glyexendin-4. The system provides a way of producing large quantities of biologically active peptides without resorting to solid phase peptide synthesis. The peptides produced using the inventive system may be used in formulating pharmaceutical compositions. The peptides and compositions thereof may be used in treating diabetes mellitus, reducing gastric motility, delaying gastric emptying, preventing hyperglycemia, and reducing appetite and food intake.<br>
In producing an insulinotropic peptide by heterologous gene expression in a microorganism, the encoding polynucleotide sequence for the peptide, preferably as part of a vector, is transformed into a host cell of the microorganism. The expression of the gene encoding the insulinotropic peptide is controlled by a promoter {e.g., an inducible promoter or constitutive promoter). Preferably, the promoter is a strong promoter, more preferably a strong inducible promoter, which allows for the production of large quantities of the insulinotropic peptide. The cells transformed with the gene may be fungal (e.g., yeast (e.g., Saccharomyces cerevisiae, Pichia pastoris)). For expression in P. pastoris, the promoter used to drive the expression of the gene is preferably the AUG1 promoter, the GAP promoter (a strong constitutive promoter), or the A0X1 or A0X2 promoter (a strong inducible promoter); and the vector may be derived from a known or commercially available vector such as pPIC, pPIC3K,<br><br>
pPIC9K, or pHIL-D. For expression in S. cerevisiae, the promoter used may be the CUPI promoter, ADH promoter, the TEF1 promoter, or the GAL promoter. The polynucleotide sequence encoding the insulinotropic peptide being expressed may be optimized for expression in the particular organism (e.g., codon usage may be optimized for use in the host microorganism). The polynucleotide sequence may optionally include leader sequences, signal sequences, pre-/pro-peptide sequences, tags for processing, tags for detection or purification, fusion peptides, etc. In certain embodiments, the gene encodes a signal peptide. The signal peptide may direct the processing or secretion of the peptide. In certain embodiments, the signal peptide is the signal peptide of exendin-4 or the signal peptide of mat alpha factor. In certain embodiments, the gene encodes a propeptide, and the propeptide includes the native exendin-4 propeptide or the propeptide of mat alpha factor (see WO 98/35033; Pohl et al, J. Biol Chem. 273:9778-9784, 1998; each of which is incorporated herein by reference).<br>
The transformed host cells are selected for the presence of the vector (e.g., antibiotic resistance, ability to grown on media which does not contain a particular nutrient) and/or expression of the exogenous gene encoding the insulinotropic peptide. The selected cells are then grown under conditions suitable for expression of the gene encoding the peptide. In certain embodiments, the medium in which the cells are grown include an agent, such as methanol, which induces the expression of the gene encoding the peptide. In other embodiments, when a constitutive promoter is used, an agent for inducing the expression for the exogenous gene is not necessary. The transformed host cell produces the peptide. Preferably, peptide is properly processed by the host cell. That is, the peptide is properly folded and post-translationally modified. In certain embodiments, the peptide is secreted it into the growth medium. The recombinantly produced peptide is then isolated and purified from the host cells or from the medium, if the peptide has been secreted. The purified peptide may be further modified chemically or enzymatically (e.g., alpha amidation, cleavage).<br>
In one aspect, the insulinotropic peptide, glyexendin-4, is produced. The glyexendin-4 in Pichia pastoris involves preparing the expression construct comprising a signal sequence, a pro¬peptide, a pre-peptide, and/or a tag, which is followed by the 40 amino acid glyexendin-4 sequence in tandem. In certain embodiments, the signal sequence comprises the native exendin-4 signal sequence of 23 amino acid or a 19 amino acid alpha factor signal sequence. The propeptide comprises the native glyexendin-4 propeptide of 24 amino acids or a 66 amino acid<br><br>
propeptide of alpha factor. The tag may aid in the processing of the peptide (Kjeldsen et al. Biotechnol Appl. Biochem. 29:79-86, 1999; incorporated herein by reference). In certain embodiments, the tag is a charged amino acid sequence such as the EAEA spacer described in Example 6. A Pichia codon optimized nucleotide sequence coding for glyexendin-4 optionally with a signal peptide or propeptide is prepared. The synthetic nucleotide sequence is then cloned into an appropriate vector carrying a selectable marker gene, such as pPIC, pPIC3K, pPIC9K, or pHIL-D. The vector is transformed into Pichia pastoris cells, and transformed cells carrying the vector are selected. The selected clone is fermented on a large enough scale to produce the desired quantity of Glyexendin-4 peptide. The Glyexendin-4 peptide so produced into the broth need no further processing steps to make it a fully functional peptide unlike that produced in bacterial system (EP 978565 Ohsuye et al Suntory limited; incorporated herein by reference). The peptide is purified from the medium using protein purification techniques known in the art. The glyexendin-4 peptide is preferably not glycosylated by the Pichia host cells. Lack of glycosylation is also an advantage because the glyexendin-4 is fully functional and it leads to a simpler and easier purification process and results in improved yields of the peptide. The purified glyexendin-4 peptide may be further chemically or enzymatically modified {e.g., C-terminal alpha amidation). The exendin-4 peptide may be used in pharmaceutical compositions for administration to a subject (e.g., humans).<br>
A problem was encountered during the production of the glyexendin-4 polypeptide in Pichia pastoris. The secreted polypeptide was found to be a mixture of full length and an N-terminally clipped molecule, lacking the first two amino acids, HG. The yield of full length Exendin could be increased by knocking out the Pichia pastoris homologue of Saccharomyces cerevisiae stel3 gene, which encodes for a dipeptidyl peptidase. This study also indicated for the first time, a novel recognition site "HG" which can be cleaved by STE13 pichia homolog, which has so far not been reported earlier.<br>
Definitions<br>
"Peptide" or "protein": According to the present invention, a "peptide" or "protein" comprises a string of at least three amino acids linked together by peptide bonds. The terms "protein" and "peptide" may be used interchangeably.   Inventive peptides preferably contain<br><br>
only natural amino acids, although non-natural amino acids (i.e., compounds that do not occur in nature but that can be incorporated into a polypeptide chain) and/or amino acid analogs as are known in the art may alternatively be employed. Also, one or more of the amino acids in an inventive peptide may be modified, for example, by the addition of a chemical entity such as a carbohydrate group, a phosphate group, a farnesyl group, an isofarnesyl group, a fatty acid group, a linker for conjugation, functionalization, or other modification (e.g., alpha amindation), etc. In a preferred embodiment, the modifications of the peptide lead to a more stable peptide (e.g., greater half-life in vivo). These modifications may include cyclization of the peptide, the incorporation of D-amino acids, etc. None of the modifications should substantially interfere with the desired biological activity of the peptide. In certain embodiments, the modifications of the peptide lead to a more biologically active peptide.<br>
"Transformation": The term "transformation" as used herein refers to introducing a vector comprising a nucleic acid sequence into a host cell. The vector may integrate itself or a portion of itself into the chromosome of the cells, or the vector may exist as a self-replicating extrachromosomal vector. Integration is considered in some embodiments to be advantageous since the nucleic acid is more likely to be stably maintained in the host cell. In other embodiments, an extrachromosomal vector is desired because each host cell can contain multiple copies of the vector.<br>
Brief Description of the Drawing<br>
Figure 1         A construct for expression of GlyExendin-4 in Pichiapastoris.<br>
Figure 2        Strategy for amplification of STE13 gene fragment from Pichiapastoris genome and introduction of Ndel restriction site by PCR<br>
Figure 3	A construct for disruption of Stel3 gene (Saccharomyces cerevisiae homolog)<br>
in Pichia pastoris genome.<br>
Figure 4         PCR analysis of putative disruptants. PCR products amplified from genomic DNA of clones D2, D4, D6, D8, D10, D12 (Lanes 2-7) and GS115 (Lanel).<br>
Figure 5	HPLC chromatogram showing two peaks of glyexendin-4 - N and N-2 (i.e.N-<br>
terminally cleaved glyexendin-4).<br><br>
Figure 6         HPLC chromatogram showing N-2 impurities from a Pichia clone with DAP2 {Saccharomyces cerevisiae homolog) gene disruption.<br>
Figure 7         HPLC chromatogram showing full length GlyExendin-4 from STE13 gene disrupted Pichia pastoris host.<br>
Figure 8        ESI-MS analysis of culture supernantant from Pichia pastoris with STE13 gene disruption, showing the mass of GlyExendin-4.<br>
Detailed Description of Certain Preferred Embodiments of the Invention<br>
The instant invention of a system for recombinantly producing insulinotropic peptides is particularly useful in preparing large quantities of biologically active peptide for use in pharmaceutical compositions or for research purposes. The invention provides methods of preparing the polynucleotide and recombinant cells for producing the peptide as well as methods of recombinantly producing the peptide itself. The invention also includes polynucleotide sequences, vectors and cells useful in practicing the inventive methods, and pharmaceutical compositions and methods of using the recombinantly produced peptides.<br>
As described above, recombinantly expressing an insulinotropic peptide in a host microorganism involves preparing a polynucleotide with the peptide-encoding sequence, introducing this sequence into a vector, transforming cells of the microorganism with the vector, and selecting for cells with the vector. These selected cells are then grown under suitable conditions to produce the insulinotropic peptide, which is purified from the cells or from the medium in which the cells were growing. The purified peptide is then used in pharmaceutical compositions to treat such diseases as diabetes or obesity.<br>
After the peptide has been harvested from the fermentation, the peptide is purified. The peptide may be purified from the cells, or if the peptide is secreted, the peptide may be purified from the media. If the peptide is purified from cells, the cells are lysed, and the peptide is purified from the cell lysate. The peptide may be purified using any methods known in the art including ammonium sulfate precipitation, column chromatography (e.g. ion exchange chromatography, size exclusion chromatography, affinity chromatography), FPLC, HPLC (e.g., reverse phase, normal phase), etc. The peptide is purified to the desired level of purity. In certain embodiments, the final peptide is greater than 90% pure, 95% pure, 98% pure, or 99% pure, preferably greater than 98% pure.<br><br>
The purified peptide may be used as is, or the peptide may be further modified. In certain embodiments, the peptide is treated with a protease to cleave the peptide. In other embodiments, the peptide is phosphorylated. In yet other embodiments, the peptide is glycosylated. The peptide may be alpha-amidated. Other modifications may be performed on the peptide. The peptide may be modified chemically or enzymatically.<br>
Production of Glyexendin-4 in Pichia pastoris<br>
The production of recombinant glyexendin-4 in Pichia pastoris involves preparing the expression construct comprising a signal sequence, an optional propeptide or a tag which is followed by the 40 amino acid Glyexendin-4 sequence in tandem. The signal sequence comprises the native exendin-4 signal sequence of 23 amino acid or the 19 amino acid alpha factor signal sequence. The propeptide comprises native exendin-4 propeptide of 24 amino acids or a 66 amino acid propeptide of alpha factor.<br>
A synthetic, Pichia codon optimized Glyexendin-4 cDNA is engineered, and overlapping oligonucleotides of the sequence are prepared. The oligonucleotides are phosphorylated and annealed. PCR amplification is performed to recover the cDNA, which is cloned into the restriction sites of an appropriate vector carrying a selectable marker gene. The vector is selected from pPIC, pPIC3K, pPIC9K, or pHIL-D. These vectors include the AOX promoter to drive the expression of the Glyexendin-4 gene. The vector carrying the synthetic Glyexendin-4 cDNA is transformed into Pichia pastoris cells, and the transformed cells are plated on minimal medium. Transformed cells carrying the clone are then selected for.<br>
The transformed Pichia cells are plated on medium containing a selection agent, and positive clones are screened and/or selected. Fermentation with the positive clones are done, and the Glyexendin-4 peptide is purified from Pichia cell-free supernatant using a combination of chromatographic techniques which include ion exchange, hydrophobic, gel filtration, and/or affinity chromatography. In certain embodiments, the Glyexendin-4 peptide produced by the Pichia cells is a totally non-glycosylated product.<br>
It was observed during expression of Glyexendin-4 that some products of its degradation occurred as impurities. These impurities were determined by Mass spec analysis to be forms of Glyexendin-4 clipped at the N-terminus by 2, 6 or 8 amino acids. The proportion of (N-2) Glyexendin-4 to full-length Glyexendin-4 was approximately 1:1. The other forms were found in<br><br>
much lower amounts and accumulated as the fermentation progressed. This problem was solved by disruption of the gene encoding Saccharomyces cerevisiae stel3 homolog in Pichia.<br>
Purification and Modification of Glyexendin-4<br>
The recombinant peptide or peptide conjugate having the polypeptide sequence of glyexendin-4 is isolated and purified from the culture medium by separating the medium from the host cells, or if the peptide is produced in the cell, separating the cell debris after cell lysis. The peptide is then precipitate using salt or solvent and subjected to a variety of chromatographic procedures like ion exchange chromatography, hydrophobic interaction chromatography, affinity chromatography, and/or crystallization.<br>
The Glyexendin-4 peptide has 40 amino acids. Optionally, the 39th amino acid, serine, is amidated. The 40 amino acid peptide is amidated in vitro to yield an Glyexendin-4 peptide of 39 amino acids with the C-terminal amino acid amidated, that is, the C-terminal glycine is removed and the penultimate serine is amidated. In certain embodiments, the amidation is accomplished using a C-terminal alpha-amidating enzyme.<br>
The biological activity of Glyexendin-4 and the 39 amino acid amidated version is determined by its ability to induce the secretion of insulin in a rat insulinoma cell line as described in more detail in the Examples below.<br>
The technology of the instant Application is further elaborated with the help of following examples. However, the examples should not be construed to limit the scope of the invention. Example 1: Construction of the vector for the expression of Glyexendin-4 in Pichia pastoris<br>
The nucleotide sequence coding for Glyexendin-4 was synthesized using overlapping oligonucleotides; ten oligonucleotides were synthesized to cover the Glyexendin-4 gene in both directions. These oligonucleotides had a 14 base pair overlapping region with the next successive primer. These overlapping oligonucleotides were phospharylated, annealed, and ligated to each other to form the nucleotide sequence coding for Glyexendin-4. The ligation mix was used as a template for amplification by the polymerase chain reaction (PCR) of the nucleotide sequence coding for Glyexendin-4.<br>
PCR was performed according to standard protocols using Pfu turbo polymerase. After an initial denaturation step of 4 minutes at 94 °C, the reaction mixture was subjected to 30 amplification cycles (denaturation at 94 °C for 30 sec; annealing at 60 °C for 30 sec; extension<br><br>
at 72 °C for 60 sec.) in a thermal cycler. The 140 bp PCR fragment obtained after amplification was cloned into the Xhol and EcoRl sites of the Pichia pastoris expression vector, pPIC9K (Invitrogen, San Diego, CA) to yield the recombinant expression vector. The nucleotide sequence coding for Glyexendin-4 was cloned in frame with the Saccharomyces cerevisiae Mat alpha signal peptide present in the pPIC9K vector or its native exendin-4 signal peptide for secretion. The expression of the Glyexendin-4 gene in the pPIC9K vector is under the control of the methanol-inducible alcohol oxidase (AOX1) promoter.<br>
Example 2: Expression of Glyexendin-4 in Pichia pastoris host<br>
Transformation<br>
The plasmid having the Pichia codon optimized nucleotide insert was introduced into Pichia pastoris, GS115 strain (his49 Mut+) by electroporation as described in the Invitrogen manual. The conditions used for electroporation were charging voltage of 1.5 kV, capacitance of 25 jaF, and resistance of 200 Q. The transformed cells were plated on minimal medium (YNBD) lacking histidine and incubated at 30°C for 3 days.<br>
Screening for expression<br>
Single colonies of transformants were picked and grown in microtitre plates containing YPD (1% yeast extract, 2% Bacto-Peptone, 2% dextrose) at 30°C overnight. The pre-grown cultures were replica-plated onto YPD agar plates with Genticin (2 mg/ml) and incubated at 30°C. Clones growing on YPD-Genticin (2 mg/ml) plate were selected for carrying out expression studies.<br>
Example 3: Cloning of codon optimized Glyexendin-4<br>
A codon optimized nucleotide sequence coding for Glyexendin-4 was synthesized by polymerase chain reaction (PCR) using overlapping oligonucleotides. Ten oligonucleotides were synthesized to cover the Glyexendin-4 gene along both strands having enough base pairs overlapping region with successive primers. The overlapping oligonucleotides were phosphorylated in a reaction mixture containing lOx PNK buffer, T4 polynucleotide kinase, and dATP, and the phosphorylated oligonucleotides were ligated using T4 DNA ligase.<br>
The ligation mixture was used as a template for amplification of the Glyexendin-4 gene<br><br>
by PCR using the oligonucleotides, 5'-CTC GAG AAA AGA CAT GGA GAA GGA AC A TTT ACA TC-3'(SEQ ID NO: 6) (forward primer) and 5'-GAA TTC TTA TCC AGA TGG TGG-3' (SEQ ID NO: 7) (reverse primer). The PCR reaction was performed according to standard protocols, and amplification was done in a final volume of 50 jiL using Pfu Turbo polymerase. The amplification conditions included an initial denaturation of 4 minutes at 94 °C for one cycle, followed by 30 cycles (denaturation at 94 °C for 30 seconds; annealing at 60 °C for 30 seconds; extension at 72 °C for 60 seconds) and a final extension for 10 minutes for one cycle in a Perkin-Elmer Thermal cycler.<br>
Example 4:<br>
The 140 bp PCR fragment obtained after amplification was cloned into the vector pTZ57R using a TA cloning kit, in a reaction mix containing 3 jil of vector, 2 \xl PEG4000 PCR buffer, and 1 jal T4 DNA ligase. DH5oc competent cells were transformed, and positive clones containing the Glyexendin-4 gene fragment were screened by restriction digestion and sequencing.<br>
Example 5: Cloning of Glyexendin-4 in Pichiapastoris<br>
The Glyexendin-4 fragment in example 2, was excised using EcoRl and Xhol and sub cloned in frame into the Pichia pastoris expression vector pPIC9K vector for secretion. The expression of the Glyexendin-4 gene was placed under the control of the methanol-inducible alcohol oxidase (AOX) promoter. The expression vector was transformed into Pichia pastoris by electroporation as described in the invitrogen manual. Single colonies of transformed cells were checked for G418 antibiotics resistance, and the selected transformants were used to carry out expression studies.<br>
Example 6: Cloning of Glyexendin-4 in Pichia pastoris using a spacer sequence<br>
The codon optimized nucleotide sequence coding for Glyexendin-4 is cloned in tandem with and a spacer sequence coding for EAEA at its n-terminal end into a Pichia pastoris vector pPIC9K. The expression vector carrying the insert was transformed into Pichia pastoris. Upon expression of the polypeptide the Mat-a which ends with Lys-Arg is cleaved from the peptide by the action of Kex2 while the spacer peptide is cleaved by a Stel3 protease allowing the release of<br><br>
Glyexendin-4. The expression level of the secreted Glyexendin-4 in the medium was 300 mg/L.<br>
Example 7: Disruption of STE13 gene homolog of Saccharomyces cerevisiae in Glyexendin-4 producing Pichia pastoris clone<br>
A Pichia clone expressing Glyexendin-4, but with disruption in another Saccharomyces cerevisiae dipeptidyl peptidase gene homolog of DAP2, continued to show N-2 impurity indicating that it was not responsible for generation of this impurity. (See Fig. 5)<br>
In an attempt to remove the impurities, another dipeptidyl peptidase gene homolog (STE13) of Saccharomyces cerevisiae, was disrupted in Pichia pastoris by insertional inactivation. Putative Pichia pastoris homolog of STE13 gene (which encodes a trans-golgi dipeptidyl peptidase) of Saccharomyces cerevisiae was identified using BLASTP. The Pichia STE13 nucleotide sequence was provided by Prof. James Cregg. A ~450bp region expected to disrupt the catalytic domain of the enzyme was chosen and amplified by PCR from the genomic DNA of the Pichia pastoris strain GS115. The fragment was then cloned into a suitable vector, linearised within the region of homology and transformed into the Glyexendin-4 producing Pichia strain to achieve disruption. Selection was done on YEPD/1M Sorbitol plates amended with 100|ig/ml Zeocin. About 900 of these transformants were inoculated into YEPD in 96-well plates. After overnight growth these were replica-plated on YEPD plates and screened for the absence of STE13 by means of a plate assay. Putative disruptants identified by the assay were screened by PCR to check if the locus was disrupted. The clone that was confirmed to have STE13 locus disrupted was taken up for further analysis. Construction ofSTE13 and DAP2 disruption vectors<br>
The coding sequence of STE13 gene and the fragment chosen to provide homology for targeting have been given in Figure 1. The STE13 fragment was assembled in two PCR steps. PCR1 was done with Taq polymerase (Bangalore Genei, India) using the following primer pairs: IISTEFP1/IISTERP1 and IISTEFP2/IISTERP2. PCR cycling parameters: Initial denaturation at 94°C for 4min followed by 30 cycles of denaturation at 94°C for 30s, annealing at 60°C for 30s and extension at 72°C for 30s; Final extension was done at 72°C for lOmin. The genomic DNA of GS115 was used as the template for this PCR, which incorporated a restriction site at approximately the centre of the fragment (See Fig 2). This provided a restriction site (Ndel) to linearise the construct within the homologous region. The second PCR step was an overlap PCR<br><br>
done using XT Taq polymerase (Bangalore Genei, India), with the two amplified fragments as template to generate the 450bp fragment. Primers used for overlap PCR: HSTEFP1/IISTERP2. PCR cycling parameters: Same as PCR1 but modified to include 3 cycles of denaturation at 94°C for 45s, annealing at 37°C for 45s and extension at 72°C for 30s, after the initial denaturation step to facilitate overlap. Restriction sites were also introduced at the 5' and 3' ends of the fragments to facilitate cloning. The 450bp fragment was first cloned into pTZ57R-TA vector (Fermantas, Canada) and its identity was confirmed by sequencing. It was then subcloned into pPICZA (Invitrogen, USA) at the BamHI/Bglll sites to get STE13/Zeo. After linearization of STE13/Zeo by Ndel, the construct was transformed into the Pichia pastoris strain producing Glyexendin-4 by electroporation (Method described earlier in Example-2). Plate Assay:<br>
The assay has been described previously for Saccharomyces cerevisiae (Rendueles and Wolf, (1987) J. Bacteriol., 169 (9): 4041-4048). Briefly, the colonies were first lysed by flooding the plates with chloroform. After the chloroform evaporated completely, a mixture of the substrate, Ala-Pro-4MPNA and Fast garnet GBC in l%Tris agar was poured to form an overlay. This was incubated at room temperature for about 10-15 minutes. Colonies that did not stain red were picked up as STE13 disruptants. GS115 with intact STE13 was included as a control to check the progress of staining. PCR analysis:<br>
Genomic DNA was isolated from those transformants that were picked up as positives in the plate assay and the disruption of STE13 locus was confirmed by PCR. A forward primer upstream of the region of homology (InSteZP) and a reverse primer from the vector sequence were designed such that an amplication of ~800bp will be observed in the disruptant The undisrupted locus should not show any amplification. Of the positives obtained by PCR analysis (see figure 3), the clone D2 was taken up for further analysis. Induction of Glyexendin-4 in shake flasks:<br>
Using the following protocol Glyexendin-4 was induced and checked in shake flasks whether N terminal clipping was occurring. The following shake flask protocol was used: Inoculate culture grown in agar plates (YEPD) into YNBG (Yeast Nitrogen Base Glycerol) medium - 50ml in 250ml flask<br><br>
YNBG Composition (For 100ml)<br>
Yeast Nitrogen Base (YNB) without aa : 1.34 g in 90ml<br>
Glycerol:20%   10ml<br>
Grow overnight in YNBG medium<br>
Transfer to BYYG (50ML IN 250 ML)such that the starting OD in BGYG is 0.3<br>
BYYG Composition(FORlOOML)<br>
Bactopeptone	: lg<br>
Yeast extract	; 2g<br>
YNB without aa       : 1.34G<br>
Glycerol	: 20% 10ml<br>
lMKH2PO4,pH6 : 10ml<br>
Grow for 2 days. Spin, weigh the pellet and resuspend in induction media(BYYM) 3 g pellet in 6ml media and transfer to 100ml flask.<br>
Induction media composition : BYYM<br>
Bactopeptone	: lg<br>
Yeast extract	: 2g<br>
YNB without aa      : 1.34<br>
WATER	:   90ml<br>
Methanol	: 3 ml<br>
lMKH2PO4,pH6 : 10ml<br>
Feed methanol and Nitrogen everyday (for 2 consecutive days )and harvest on third day ,<br>
Nitrogen :0.1X of 5XYP<br>
( FOR 6ML 5X* ? - 0.1X * 6 WHICH IS EQUAL TO 120 MICROLITRES )<br>
METHANOL : 3% of the shakeflask volume ( ie 180 microlitres for 6ml)<br><br>
5x Nitogen composition : for 50ml<br>
Yeast Extract : 2.5g<br>
Bactopeptone : 5 g<br>
Glyexendin-4 produced by D2 clone when checked by HPLC and LCMS was not N-terminally<br>
clipped. This clearly shows that STE13 dipeptidyl peptidase activity was responsible for the N-<br>
terminal clipping of Glyexendin-4 (Figure 6)<br>
Example 8: Fermentation of Pichiapastoris for expression of Glyexendin-4<br>
The frozen (-70 °C) cells of Pichia sp. are cultivated in 250 ml flask containing 50 ml growth medium (1% yeast extract, 2% peptone, 10% 1M phosphate buffer of pH 6.0, 0.67% yeast nitrogen base, and 0.1% glycerol ) at 28-32 °C and 220-260 rpm. The seed cells are cultivated in 2 L fermenter containing one liter of fermentation medium consisting of 4% glycerol, 0.01% calcium sulfate, 2% potassium sulfate, 1.4% magnesium sulfate, and 0.4% potassium hydroxide. Fermentor experiments with the above medium were performed at different temperatures ranging from 20 °C to 30 °C, pH 5.0, aeration rate 0.5-2.0 wm. Agitation speed was increased gradually from 350 to 1200 rpm to maintain the dissolved oxygen above 30%. Biomass was built up to 300 g/L by 50% glycerol feeding. Aseptically 1L broth was harvested and 50 ml of broth was dispensed in each of twelve 250 ml flasks. All flasks were incubated at different temperatures ranging from 20 °C to 30 °C at 220 to 260 rpm. 100 \iL of methanol was added every day, and after five days analysis for Glyexendin-4 expression was performed.<br>
At 20 °C, assay for Glyexendin-4 after five days showed 0.0443 g/L. At 22 °C, assay after five days showed 0.0467 g/L. At 24 °C, assay after five days showed 0.0567 g/L. At 26 °C, assay after five days showed 0.0867 g/L. At 28 °C, assay after five days showed 0.0905 g/L. At 30 °C, assay after five days showed 0.0685 g/L<br>
Example 9: Fermentation of recombinant Pichia for production of Glyexendin-4<br>
The frozen (-70 °C) cells of Pichia sp are cultivated in a 250 ml flask containing 50 ml growth medium (1% yeast extract, 2% peptone, 10% 1M phosphate buffer (pH 6.0), 0.67% yeast nitrogen base, and 0.1% glycerol) at 28-32 °C and 220-260 rpm. The seed cells are cultivated in a 2 L fermenter containing one liter of fermentation medium (3.5% glycerol, 0.01% calcium<br><br>
sulfate, 1% potassium sulfate, 1% magnesium sulfate, 0.25% potassium hydroxide). Fermenter experiments with the above medium were performed at different temperatures ranging from 20 °C to 30 °C, pH 5.0, aeration rate 0.5-2.0 wm. Agitation speed was increased gradually from 350 to 1200 rpm to maintain the dissolved oxygen above 30%. Biomass was built up to 285 g/L by 50% glycerol feeding, and then methanol feeding was carried out.<br>
Samples were taken every 24 hour after methanol feeding, and Glyexendin-4 levels were determined using HPLC. At 20 °C, after five days Glyexendin-4 assay showed 0.23 g/L. At 22 °C, after five days assay showed 0.37 g/L. At 24 °C, after five days assay showed 0.67 g/L. At 26 °C after five days assay showed 0.72 g/L. At 28 °C, after five days assay showed 0.75 g/L. At 30 °C, assay showed 0.67 g/L.<br>
The purified Glyexendin-4 was characterized by ESI-MS. It showed that no N-terminal clipping and no glycosylated species. (Figure 7)<br>
Other Embodiments<br>
The foregoing has been a description of certain non-limiting preferred embodiments of the invention. Those of ordinary skill in the art will appreciate that various changes and modifications to this description may be made without departing from the spirit or scope of the present invention, as defined in the following claims.<br><br>
SEQUENCE LISTING<br> XXX<br> Production of Insulinotropic Peptides<br> 2001568-0040<br> Not yet assigned  1950-11-11<br> 13<br> Patentln version 3.2<br> 1<br> 31<br> PRT<br> Homo sapiens<br> 1<br>
His Ala Glu Gly Thr Phe Thr Ser Asp Val Ser Ser Tyr Leu Glu Gly<br>
15	10	15<br>
Gin Ala Ala Lys Glu Phe He Ala Trp Leu Val Lys Gly Arg Gly<br>
20	25	30<br> 2<br> 40<br> PRT<br> Heloderma suspectum<br> 2<br>
His Gly Glu Gly Thr Phe Thr Ser Asp Leu Ser Lys Gin Met Glu Glu<br>
15	10	15<br>
Glu Ala Val Arg Leu Phe He Glu Trp Leu Lys Asn Gly Gly Pro Ser<br>
20	25	30<br><br>
Ser Gly Ala Pro Pro Pro Ser Gly<br>
35	40<br>	3<br>	39<br>	PRT<br>	Heloderma suspectum<br> 3<br>
His Gly Glu Gly Thr Phe Thr Ser Asp Leu Ser Lys Gin Met Glu Glu<br>
15	10	15<br>
Glu Ala Val Arg Leu Phe He Glu Trp Leu Lys Asn Gly Gly Pro Ser<br>
20	25	30<br>
Ser Gly Ala Pro Pro Pro Ser<br>
35<br> 4<br> 87<br> PRT<br> Heloderma suspectum<br> 4<br>
Met Lys He lie Leu Trp Leu Cys Val Phe Gly Leu Phe Leu Ala Thr<br>
1	5	10	15<br>
Leu Phe Pro He Ser Trp Gin Met Pro Val Glu Ser Gly Leu Ser Ser<br>
20	25	30<br>
Glu Asp Ser Ala Ser Ser Glu Ser Phe Ala Ser Lys He Lys Arg His<br>
35	40	45<br>
Gly Glu Gly Thr Phe Thr Ser Asp Leu Ser Lys Gin Met Glu Glu Glu<br>
50	55	60<br><br>
Ala Val Arg Leu Phe He Glu Trp Leu Lys Asn Gly Gly Pro Ser Ser<br>
65	70	75	80<br>
Gly Ala Pro Pro Pro Ser Gly<br>
85<br> 5  141  DNA  Artificial<br><br> Exendin - 4 encoding sequence optimized for expression in Pichia pastoris.<br> 5<br>
ctcgagaaaa gacatggaga aggaacattt acatctgatt tgtctaaaca aatggaagaa    60<br>
gaagctgtta gattgtttat tgaatggttg aaaaacggag gaccatcttc tggagctcca    120<br>
ccaccatctg gataagaatt c	141<br> 6  35  DNA  Artificial<br><br> PCR primer (forward) for amplification of exendin-4 gene.<br> 6<br>
ctcgagaaaa gacatggaga aggaacattt acatc	35<br> 7  21  DNA  Artificial<br><br> PCR primer (reverse) for amplification of exendin-4 gene.<br> 7<br><br>
gaattcttat ccagatggtg g<br> 8<br> 10<br> PRT<br> Heloderma suspectum<br> 8<br>
His Gly Glu Gly Thr Phe Thr Ser Asp Leu<br>
1	5	10<br> 9<br> 12<br> PRT<br> Heloderma suspectum<br> 9<br>
His Gly Glu Gly Thr Phe Thr Ser Asp Leu Ser Lys<br>
1	5	10<br> 10<br> 8<br> PRT<br> Heloderma suspectum<br> 10<br>
Gin Met Glu Glu Glu Ala Val Arg<br>
1	5<br> 11<br> 7<br> PRT<br> Heloderma suspectum<br> 11<br>
leu phe IIe Glu Trp Leu lys	5<br><br> 12<br> 13<br> PRT<br> Heloderma suspectum<br> 12<br>
Asn Gly Gly Pro Ser Ser Gly Ala Pro Pro Pro Ser Gly<br>
1	5	10<br> 13  30  DNA  Artificial<br><br> PCR primer (forward) for amplification of STE13 homo log in Pichia<br> 13<br>
ggatccgcag ttcaacattg ggtttgagca<br> 14  28  DNA  Artificial<br><br> PCR primer (reverse) for amplification of STE13 homolog in Pichia<br> 14<br>
ccagaatcat atgccaatgt cttaagcg<br> 15  28  DNA  Artificial<br><br><br><br><br><br>
We Claim:<br>
1.   A method of producing a biologically active polypeptide having insulinotropic<br>
activity, the method comprising steps of:<br>
providing a genetically modified host cell that has certain protease gene knockout<br>
transformed with a polynucleotide vector encoding the polypeptide; and<br>
growing the host cell under suitable conditions to produce the full length<br>
polypeptide.<br>
2.	The method of claim 1 further comprising the step of isolating and thereafter alpha-<br>
amidating the isolated polypeptide.<br>
3.	The method of claim 1, wherein the host cell are yeast cells, preferably Pichia<br>
pastoris.<br>
4.	The method of claim 1, in which the host cells have a N-terminal dipeptidyl<br>
peptidase gene, STE13, disrupted.<br><br>
5.	The method of claim 1, wherein the step of growing the host cells comprises<br>
growing the cells under conditions which induce the expression of the polypeptide,<br>
6.	The method of expressing a full length polypeptide having an N-terminal<br>
recognition site His-Gly in an STE13 homolog knockout Pichia host cell<br>
7.	The method of claim 8, wherein the polypeptide is Glyexendin-4 of SEQ ID NO.2 as<br>
HGEGTFTSDLSKQMEEEAVRLFIEWLKNGGPSSGAPPPSG, and is not<br>
glycosylated by the host cell.<br>
8.	The Glyexendin-4 polypeptide of claim 11, wherein the polynucleotide sequence is<br>
ctcgagaaaa  gacatggaga  aggaacattt   acatctgatt  tgtctaaaca<br>
atggaagaa  gaagctgtta  gattgtttat   tgaatggttg  aaaaacggag<br>
gaccatcttc     tggagctcca     ccaccatctg       gataagaattc     (SEQID NO: 5).<br>
9.	A method of alpha-amidating the glyexendin -4 polypeptide, the method comprising<br>
steps of:<br>
providing glyexendin-4 polypeptide; and<br>
contacting the polypeptide with an C-terminal alpha-amidating enzyme under suitable<br>
conditions to yield an alpha-amidated exendin-4 polypeptide,<br>
10.	The method of claim 13, wherein the alpha-amidated exendin-4 polypeptide is<br>
HGEGTFTSDLSKQMEEEAVRLFIEWLKNGGPSSGAPPPS-NH2<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIEFNRU5ERUQgQ0xBSU1TICAgMjktMTEtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   AMENDED CLAIMS   29-11-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNiAgIGNvcnJlc3BvbmRlbmNlIG90aGVycyAgIDAyLTA2LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006   correspondence others   02-06-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIENPUlJFU1BPTkRFTkNFIE9USEVSUyAwOC0wMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   CORRESPONDENCE OTHERS 08-02-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIENPUlJFU1BPTkRFTkNFIE9USEVSUyAzMC0wNC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   CORRESPONDENCE OTHERS 30-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIENPUlJFU1BPTkRFTkNFIE9USEVSUy4gIDA3LTAzLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   CORRESPONDENCE OTHERS.  07-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIEVYQU1JTkFUSU9OIFJFUE9SVCBSRVBMWSBSRUNFSVZFRCAgIDI5LTExLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   EXAMINATION REPORT REPLY RECEIVED   29-11-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIEZPUk0tMSAgIDI5LTExLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   FORM-1   29-11-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIEZPUk0tMTMgICAyOS0xMS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   FORM-13   29-11-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNiAgIGZvcm0tMyAgIDAyLTA2LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006   form-3   02-06-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIEZPUk0tMyAgIDI5LTExLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   FORM-3   29-11-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIE9USEVSIFBBVEVOVCBET0NVTUVOVCAgIDI5LTExLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   OTHER PATENT DOCUMENT   29-11-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgIFBPV0VSIE9GIEFUVE9STkVZICAwNy0wMy0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006   POWER OF ATTORNEY  07-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAxNS0wNS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006  AMENDED  PAGES OF SPECIFICATION 15-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgQU1FTkRFRCBDTEFJTVMgMTUtMDUtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1057-CHE-2006  AMENDED CLAIMS 15-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDE1LTA1LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006  CORRESPONDENCE OTHERS 15-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgRk9STS0zIDE1LTA1LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006  FORM-3 15-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiAgT1RIRVJTIDE1LTA1LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006  OTHERS 15-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBBQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBDTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBDT1JSRVNQT05ERU5DRSBPVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNiBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBGT1JNIDEucGRm" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 FORM 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBGT1JNIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNiBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1DSEUtMjAwNiBTRVFVRU5DRSBMSVNUSU5HLnBkZg==" target="_blank" style="word-wrap:break-word;">1057-CHE-2006 SEQUENCE LISTING.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1057-che-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1kZXNjcmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1057-che-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1057-che-2006-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA1Ny1jaGUtMjAwNi1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">1057-che-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=TGV0dGVyIHRvIElQTyAtIE9uIGxpbmUgdXBsb2FkIC5wZGY=" target="_blank" style="word-wrap:break-word;">Letter to IPO - On line upload .pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259103-a-process-for-the-preparation-of-a-photo-fenton-catalyst-consisting-of-immobilized-fe-iii-on-hy-zeolite-support.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259105-methods-for-preparing-immunogenic-conjugates.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259104</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1057/CHE/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Feb-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Feb-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIOCON LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>20th Km Hosur Road, Electronic CityP.O. Bangalore 560 100Karnataka,Phone: +91 80 2808 2808Fax: + 91 80 2852 3423</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>Varadarajalu Lakshmi Prabha</td>
											<td>20th Km Hosur Road, Electronic City P.O. Bangalore 560 100 Karnataka,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>Suryanarayan Shrikumar</td>
											<td>20th Km Hosur Road, Electronic City P.O. Bangalore 560 100 Karnataka,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>Melarkode Ramakrishnan</td>
											<td>20th Km Hosur Road, Electronic City P.O. Bangalore 560 100 Karnataka,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>Sriram Akundi Venkata</td>
											<td>20th Km Hosur Road, Electronic City P.O. Bangalore 560 100 Karnataka,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>Sastry Kedarnath Nanjund</td>
											<td>20th Km Hosur Road, Electronic City P.O. Bangalore 560 100 Karnataka,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61P27/06</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259104-a-method-for-producing-biologically-active-polypeptide-having-insulinotropic-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:16:52 GMT -->
</html>
