<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259289-detoxifizyme-with-activity-of-transforming-aflatoxin-and-the-gene-encodes-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:33:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259289:&quot;DETOXIFIZYME WITH ACTIVITY OF TRANSFORMING AFLATOXIN AND THE GENE ENCODES THEREOF&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;DETOXIFIZYME WITH ACTIVITY OF TRANSFORMING AFLATOXIN AND THE GENE ENCODES THEREOF&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a detoxifizyme with the activity of transforming aflatoxin and the gene encodes thereof. Inventors firstly isolate and purify a novel protein, named aflatoxin-detoxifizyme (ADTZ), which has the activity of transforming aflatoxin. The primes specify to the ADTZ gene are obtained through purification and sequencing. The gene encoding of ADTZ is cloned from the total RNA of Armillariella tabescens. The recombinant protein is expressed and purified through various expression systems using genetic engineering methods. The said detoxifizyme has bioactivity of transforming AFB1, reducing mutagenic effects of AFB1. It has great potential for the manufacturing of feed or food and development of anti-tumor medicament.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DETOXIFIZYME WITH ACTIVITY OF TRANSFORMING AFLATOXIN AND THE GENE ENCODES THEREOF<br>
FIELD OF INVENTION<br>
The present invention relates to a detoxifizyme with the activity of transforming aflatoxin and the gene encodes thereof.<br>
BACKGROUND OF INVENTION<br>
Aflatoxins, a group of toxic mycotoxins, including Aflatoxin B1 (AFB1), Aflatoxin M1 (AFM1), Aflatoxin G1 (G1) etc., are produced by many species of Aspergilus. Aflatoxins are toxic and carcinogenic to animals and humans. Aflatoxins are widely present in grain, feed, and food, and the harmful effects to human beings are: (1) direct poisoning by consumption of untreated aflatoxins contaminated food; (2) poisoning by consumption of poultry, milk, etc. indirectly from untreated aflatoxins contaminated feed; (3) waste and disposal of crops or nuts contaminated with aflatoxins.<br>
Because of these harmful effects, detoxification of aflatoxins has been studied for years. Some methods of transforming aflatoxins already exist, for instance: (1) Ammonization method: this method is used for wet feed. Because of the large amount of residual ammonia, it is banned in food processing by FDA. The application on feed is also limited. (2) NaOH method (for vegetable oil): due to high equipment investment, oil consumption, and cost, the method is no longer in use (3) White soil adsorption method: no longer in use because of higher labor cost, pollution etc. (4) Extraction method (for peanut powder, cotton seed, etc): it isn't widely used because of the high cost associated with extracting, recovering the solvent. (5) Heat method (268Â°C): cost for heating and lose of flavor and nutrients make it less practical. (6) Biological method: bacteria or immobilized bacteria are used to resolve aflatoxins. Bacteria can destroy the nutrients of food, and the products and their toxicities are not well understood. Thus this application is limited to only a few types of feed and peanut oil. (7) Ultraviolet method: strong ultraviolet oxidation used to destroy aflatoxins is not consistent and high energy consuming. (8) Ultra-filtration method: it isn't practical due to high equipment cost and rigorous technical requirement. (9) Enzyme method: clone of liver cytochrome oxidase P450 in E.coli was used to transform aflatoxins (Brown DW, etc. Proc. Natl. Acad. Sc. USA. 1996).<br>
To summarize, the chemical or physical methods to transform aflatoxins often require harsh conditions, but results in lower value for the treated grain, feed, and food. These methods are often not efficient and economical, thus difficult for large scale applications. P450 enzyme method does promote the metabolism of aflatoxins, but it may also lead to higher toxicity of AFB1 to human. Because of the specificities and high efficiencies of enzymes, more research is focused on enzymes that can transform aflatoxins directly.<br><br>
SUMMARY OF THE INVENTION<br>
The present invention is generally directed to a detoxifizyme that can transform aflatoxins and gene encoding of the enzyme.<br>
This enzyme with AFB, transforming activity can be prepared from purification of crude enzyme produced from selected cells. The AFBt transforming protein can also be produced by DMA recombinant techniques. This new active protein is named Aflatoxin-detoxifizyme (ADTZ).<br>
The primers specific to the ADTZ gene can be obtained from purification and sequencing. The gene encoding of ADTZ can be cloned from the total RNA of Armillariella tabescens. The gene is a new gene that is never reported before. The recombinant protein can be expressed and purified from different expression systems, Pichia pastoris expression system, for example, using genetic engineering methods. The selected fungus, Armillariella tabescens, comes from China General Microbiological Culture Collection Center (CGMCC).<br>
Purification of ADTZ: break the fungus cell firstly, then obtain the crude protein by (NH4)2S04 precipitation method. ADTZ N-terminal peptide amino acid sequence can be obtained from mass spectrometry analysis of the target peak.<br>
Purification of ADTZ: At first we and precipitates proteins by the method of ammonium sulphate precipitate method. Secondly, we receive the purpose peak from the end short peptide of the order of amino acid.<br>
This invention relates to the extraction of total RNA of Armillariella tabescens. Throuth PCR and SMART RACE of the primers derived from the sequence of ADTZ N-terminal peptide amino acid sequence, ADTZ gene encoding can be obtained, its length is about 2.3 kb. The sequence contains a complete open reading frame, 3' and 5' non-translating regions. The ADTZ encoding cDNA contains 2088 base pairs. ADTZ mature peptide contains 695 amino acids, molecular weight: 73-77kDa (SDS-PAGE), pi: 5.3-6.8 (isoelectric focusing electrophoresis). Amino acids and DNA sequences are depicted in the Sequence Listing (SEQ ID No.1 and SEQ ID No.2). The protein claimed in the invention should be understood to include the molecular produced by elimination, substitution, modify and addition etc.<br>
This invention provides the recombinant expression carrier which comprises said gene, and the transformant obtained by a host cell transformed with said recombinant expression carrier. This invention further procides the method for the preparation of said detoxifizyme, which comprises: cultivating said transformant, and recovering the expressed detoxifizyme.<br>
This invention relates to a pair of primers to amplify the gene encoding of ADTZ mature peptide from cDNA of Armillariella tabescenes. The DNA can be cloned to eukaryotic integration type expression vectors, such as pHIL-S1. Expression plasmid pHIL-S1-ADTZ can be thus constructed from transformation of recombinant expression vector in Pichia pastoris GS115. This recombinant expression vector uses AOX as promoter. Experiments on time of cultivation and induction, lead to over 25% expression of ADTZ in total protein in soluble state.<br>
The invention relates to eukaryotic expression systems, including endocytic vectors (such as PA0815, PPIC3K, PPICZ, PHW010, PGAPZ), or excretion vectors (such as PPIC9K, PPICZa,<br><br>
PGAPZa, or other commercial vectors). For eukaryotic expression stains, Pichia pastoris KM71, MC100-3, SMD1168, SMD1165, SMD1163 can also be used as host cells.<br>
The invention relates to prokaryotic expression systems. Different expression vectors can be used, such as pET, pUCH33, or similar commercial vectors. For prokaryotic expression stains, E.coli BL21, E.coli JM109 can be used as host cells.<br>
Replication of expression vectors can be achieved following Sambrook's manual (Sambrook, et al. 2002, molecular cloning, Cold Spring Laboratory Press. USA). Preparation and transformation of E.coli DH5a may be achieved using calcium chloride protocol. Cell culture can be prepared using ampicillin (100 ug/ml) in LB media, and plasmid extracted using alkaline method.<br>
This invention relates to the optimal conditions to purify recombinant ADTZ. The expression culture can be first precipitated with (NH4)2S04. The resultant crude enzyme can be further purified by hydrophobic interaction chromatography and metal-chelating affinity chromatography to give recombinant ADTZ, which purity is greater than 95%.<br>
The invention provides the use of said detoxifizyme with activity of transforming aflatoxin for the manufacturing of feed or food. ADTZ can be used as a detoxification additive for feed, and immobilized ADTZ can be use in detoxification of peanut oil.<br>
The invention provides the use of said detoxifizyme with activity of transforming aflatoxin for the preparation of a medicament for preventing or treating tumor or cancer. ADTZ can be used in the prevention and treatment of aflatoxin induced tumors.<br>
The invention relates to methods of separation and sequencing of ADTZ gene in the first time. The gene encoding of ADTZ can be cloned by Armillariella tabescenes to expression vector to form recombinant transformant. The recombinant protein ADTZ can be expressed using this transformant. Recombinant ADTZ has similar activity in transforming AFB, as natural ADTZ from activity analysis. Recombinant ADTZ also significantly reduces mutagenic effects of AFB,. It has great potential in feed, food and pharmaceutical industries.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 shows PAGE of purified ADTZ. M: protein molecular weight standard; 1 and 2: BSA, 3: crude enzyme from ammonium sulfate precipitation. 4: purified ADTZ.<br>
Fig. 2 shows TLC of purified ADTZ on AFB, transformation. 1: AFB, analytical standard; 2 and 3: PBS buffer solution, 4: AFB, treated with deactivated ADTZ, 5: AFB, treated with purified ADTZ, 6: AFB, analytical standard.<br>
Fig. 3 shows electrophoresis of the total RNA of Armillariella tabesccns.<br>
Fig. 4 shows electrophoresis of RT-PCR product. M: DMA marker. E1: RT-PCR product.<br>
Fig. 5 shows enzyme incision of recombinant vector pTE1. M: DMA marker. 1: pTE1/Hindlll + EcoRI, 2: pTE1/EcoRI, 3: pTE1/Hindlll.<br>
Fig. 6 shows electrophoresis of 3'RACE product. M: DNA marker. E2: 3'RACE product.<br><br>
Fig. 7 shows enzyme incision of recombinant vector pTE2. M: DMA marker. 1: pTE2/Hindlll + EcoRI, 2: pTE2/EcoRI, 3: pTEZ/Hindlll.<br>
Fig. 8 shows electrophoresis of 5'RACE product. M: DMA marker. E3: 5'RACE product<br>
Fig. 9 shows enzyme incision of recombinant vector pTE3. M: DMA marker. 1: pTE3/Hindlll + EcoRI, 2: pTE3/EcoRI, 3: pTE3/Hindlll.<br>
Fig. 10 shows electrophoresis of end to end PCR product. M: DMA marker. ADTZ': PCR product.<br>
Fig. 11 shows enzyme incision of recombinant plasmid pSA. M: DMA marker. 1: pSA/Hindlll + EcoRI, 2: pSA/EcoRI, 3: pSA/Hindlll.<br>
Fig. 12 shows SDS-PAGE of expression products. 1. negative control cell culture, 2: protein molecular weight standard, 3: BSA, 4: recombinant cell culture at 24 h, 5: recombinant cell culture at 48 h, 6: recombinant cell culture at 72 h, 7: recombinant cell culture at 96 h.<br>
Fig. 13 shows TLC of purified recombinant ADTZ on AFB, transformation. 1: AFB, analytical standard, 2: AFB, treated with purified recombinant ADTZ, 3: AFB, treated with deactivated recombinant ADTZ, 4: buffer solution.<br>
Fig. 14 is schematic diagram of construction of ADTZ recombinant and homogeneous recombination in Pichia pastoris.<br>
DETAIL DESCRIPTION OF THE INVENTION<br>
Example 1 Preparation and Purification of ADTZ<br>
1.1.	Cell culture<br>
1.1.1	Strain: Armillariella tabescenes.<br>
1.1.2	The above cells were incubated in the medium (potato liquid extraction 1L,<br>
glucose 20g, KH2P04 3.0 g, MgS04-7H20 1.5 g, and trace vitamin, pH6.6) for 25 days. First<br>
through three classes incubation: 6 days, 4 days and 4 days, then fourth class: 11 days,<br>
temperature 24-28 Â°C. Cells were then collected.<br>
1.2.	Extraction of ADTZ<br>
Fresh cells were frozen in liquid nitrogen, and broken into small pieces. Phosphate buffer (1:1 W/V) was added, followed by homogenation in ice bath, ultrasound sonication to smudge cells, and centrifugation at 11000-12000 g to remove precipitate. Precipitate was collected from 20-80% saturated (NH4)2S04 fractional precipitation and suspended in phosphate buffer (0.2 mol/L, pH 6.0). Proteins were quantitated using Bradford method, enzyme activity was tested using AFB, ELISA test kit. ADTZ enzyme solution was thus produced.<br>
1.3.	Purification of ADTZ<br>
1.3.1.   Preparation of enzyme sample<br>
Enzyme sample was prepared through such steps as dialysis desalination of the crude enzyme solution in phosphate buffer (40 volumes, 0.02 mol/L, pH 6.0), and concentration by<br><br>
dialysis in PEG-20000 and micro membrane (0.45 urn) filtration. Proteins were quantitated using Bradford method.<br>
1.3.2.   Purification of ADTZ enzyme by fast protein liquid chromatography (FPLC)<br>
Columns and eluents for ion exchange chromatography and isoelectro-focusing chromatography were prepared following literature procedures (Ion Exchange Chromatography Principles and Methods. Pharmacia Co. Edited, Pharmacia Co., 1984, pp.29-31; Chromatofocusins with Poiybufferâ¢ and PBEâ¢, 6th. Experimental. Pharmacia Co. Edited, Pharmacia Co., 1984, pp.11-24). All chromatographic separations were performed on FPLC systems (Pharmacia Biotech Co., Unite States). Detail as following: 1.3.2.1 Anion exchange chromatography<br>
(1)	Reagent<br>
pH 6.0, 0.2 mol/L phosphate buffer.<br>
A: pH 6.0, 0.2 mol/L, phosphate buffer.<br>
B:  pH 6.0, 0.2 mol/L, phosphate buffer +  1 N NaCl .<br>
(2)	Column preparation<br>
DEAE-Sephadex (50 ml) was washed with 2 volumes of phosphate buffer, and kept at room temperature for 20 min. The buffer solution was removed using a micro-pump, the process was repeated 5 times. The gel slurry was poured into a glass column (20 x 30 cm) and packed at a flow rate of 0.6 ml/min.<br>
The column was washed with buffer A to equilibrate until baseline stabilized around 0. The enzyme sample was loaded onto pre-column then DEAE-Sephadex column. NaCl gradient eluting: 2 hours by buffer A, 5 hours by 0-80% buffer B and A, 2 hours by 100% buffer B, flow rate: 0.6 ml/min, inspect on UV O.D.280 nm. Effluents were collected using a fraction collector. After PEG-20000 dialyzed concentration, and desalination, different fractions of proteins were quantitated using Bradford method, their activities in transforming AFB, were also tested. The separation was repeated, only the active fractions were collected. 1.3.2.2. Electrofocusing chromatography<br>
(1)	Reagent<br>
Eluent: Poiybufferâ¢ 74(Pharmacia Co., United States, 250ml/bottle). 100 ml diluted to 1000 ml with water, store at 4Â°C.<br>
Initial buffer solution: pH 7.4, imidazole-HCl buffer (0.025 mol/L).<br>
(2)	Column<br>
Mono-pâ¢ PBE 94, 5 x 20 cm, pre-packed column (Pharmacia Co., U. S.) The active enzyme solution from anion exchange chromatography (6 ml, 3 mg/ml) was equilibrated using Poiybuffer 74 to -6.5 ml. Mono-p column was equilibrated with initial buffer solution for 2 hours, then changed to Poiybuffer 74. Enzyme solution (2 ml) was loaded onto the column, and washed with Poiybuffer 74 for 10 hours at 0.2 ml/min, inspect on UV O.D.280 nm- Effluents were collected using a fraction collector, 2 ml/tube (10 min/tube).<br><br>
Collected proteins were quantitated using Bradford method, their activities in transforming AFB1 were tested.<br>
Column was washed with 0.1 N HCl until AD -0, and washed again with 1N NaCl, and equilibrated with initial buffer overnight. The process was repeated, and active fractions were collected.<br>
1.3.2.3.	Activity testing using ELISA<br>
AFB, was treated with collected protein fractions; the remaining AFB1 was measured using ELISA method. The protein fractions were heated to 100Â°C for 10 min to prepare deactivated enzyme solutions as controls. The fractions that can lower the AFB, level are active fractions. Detail as following: (1)        Sample preparation<br>
Deactivated enzyme test mixture: AFB, (200 ul, 2.5 ng/ml in methanol) + deactivated enzyme solution (200 pi, 1.2 mg/ml).<br>
Active enzyme test mixture: AFB1 (200 ul, 2.5 ng/ml in methanol) + active enzyme solution (200 Ml, 1.2 mg/ml).<br>
Control mixture: AFB1 (200 ul, 2.5 ng/ml in methanol) + buffer solution (200 MO-<br>
Preparation of deactivated enzyme solutions: heated to 100Â°C for 10 min.<br>
The test mixtures were mixed thoroughly and reacted for 30 min at 30Â°C. After centrifugation at 3000g for 5 min, precipitate was removed. The test mixtures were tested using ELISA test kits (AgraQuantâ¢ Total Aflatoxin Assay 4/40, ROMER, United States). Remaining AFB1 was calculated based on a calibration curve. The different fractions from chromatographic separations were tested for their AFB, transforming activities, and the fractions that can reduce AFB1 level are active fractions. The result of remaining AFB, follows: test group with the active enzyme: 1.230 Â± 0.508 ng/ml, test group with the deactivated enzyme: 2.436 Â± 0.326 ng/ml, control group: 2.508 Â± 0.203 ng/ml.<br>
PAGE of the active enzyme fraction showed a single band under non-reductive condition, as shown in Fig. 1.<br>
Molecular weight of the protein is 73-76 kDa analysed by SDS PAGE, pi of the protein is 5.3-6.8 analysed by isoelectric focusing electrophoresis.<br>
Example 2<br>
Activity Measurement of purified ADTZ 2.1. Test of ADTZ activity in transforming AFB,<br>
To measure the activity of purified ADTZ, thin layer chromatography was used for the testing of remaining AFB1 after treatment with ADTZ. Detail follows: 2.1.1.   ADTZ enzyme mixture<br>
AFB1 solution (1 Ml, 0-5 MS/M^ methanol) (Alexis Biochemical's Inc., Switzerland) in a 1.5 ml centrifuge tube was evaporated under nitrogen gas. Enzyme solution (300 \i(, 0.1 mg/ml), MgS04 (0.5 \i() and PEG 200 (10 pi) were added to the tube and mixed thoroughly. The<br><br>
mixture reacted in a water bath for 1 h at 30Â°C, and then added AFB, 0.5 ul per hour until the total amount of AFB, was 2ug. After addition, the mixture reacted for another 2 h.<br>
2.1.2.	Control mixtures<br>
Control 1: AFB, solution 1 ul in a 1.5 ml centrifuge tube was evaporated under nitrogen gas. Deactivated enzyme solution 300 ul, deactivation at 100Â°C for 5 min, MgS04 0.5 ul and PEG 200 10 ul were added to the tube and mixed thoroughly. Control 2: AFB, solution 1 pi in a 1.5 ml centrifuge tube was evaporated under nitrogen gas. PBS buffer solution 300 pi (0.1 M, pH 6.6), MgS040.5 ul and PEG 200 10 pi were added to the tube and mixed thoroughly. The two control mixtures were allowed to re-act the same manner as the enzyme mixture.<br>
The enzyme mixture and control mixtures were extracted with 2 volumes CHC13 twice. CHC13 extract was evaporated under nitrogen gas at 45Â°C. The crude mixtures were re-dissolved in methanol 1 ml to get the enzyme sample and two control samples for TLC.<br>
2.1.3.	TLC measurement<br>
TLC plates (10 x 10 cm, 60 A, Whatman, United States) were freshly activated at 100Â°C for 2 h. Samples were spotted on the plates 1 cm apart, and 1 cm from the edge from left to right, 1st: AFB, (10 ul, 25 ug/ml in CHC13), 2nd, 3rd: control 2 (10 pi), 4th: control 1(10 ul), 5th: enzyme mixture (10 ul), 6th: AFB, (10 ul, 25 ug/ml in CHC13). The plate was developed in anhydrous ether, visualized with UV light (A 365 nm), and photographed (result as shown in Fig. 2). There is almost no shift among AFB1 standards (1st and 6th spots), deactivated enzyme mixture control (4th) and PBS buffer controls (2nd and 3rd), but significant shift for ADTZ enzyme mixture (product Rf = 0.95). The product following ADTZ treatment is a lot less polar than AFB,, indicating the AFB, transforming activity of purified ADTZ.<br>
2.2.	The bioactivity of ADTZ in transforming AFB,<br>
Microorganism reversal mutation assays (Ames assays) were conducted as following:<br>
2.2.1.	Test the bacterial strains<br>
Histidine Auxotroph Salmonella Typhimurium strain TA98 was stored at -85Â°C. Genotype identification and spontaneous reversal mutation quantity determination were conducted before testing to ensure the strain meet the experiment requirement.<br>
2.2.2.	Prepa ration of liver S9<br>
(a)	SD rat induction: SD rats were quarantined for one week to make sure they<br>
were healthy.   A polychlorinated biphenyl corn oil suspension was administered via stomach<br>
tube to rats (500 mg/kg).  On the fifth day, after 12h of starvation, the animals were<br>
decapitated.<br>
(b)	Liver S9 preparation: livers were collected, weighted, and perfused in situ with<br>
ice cold sterile KCl (0.15 M), and homogenated in a homogenizer. Followed by centrifugation at<br>
9000 g for 30 min, the supernatant was collected, tested and stored at -85Â°C.<br>
2.2.3.	Preparation of S9 mix<br>
The following solutions: A, B and C were mixed with S9, and stored at 4Â°C (to be used within 4 h).<br>
A: (0.2 M, coenzyme II, sterilized by filtration) 0.2 ml.<br><br>
B: (0.2 M, glucose 6-phosphate, sterilized by filtration) 0.25 ml C: (0.4 M MgCl2 20 ml + 1 .65 M KCl, 20 ml  + 0.2 mol/L phosphate buffer, pH 7.4, 500 ml + distilled water, 313 ml, mixed and sterilized by filtration) 8.55 ml. S9:  1.00ml.<br>
2.2.4.	Preparation of test mixtures<br>
In a 30 ml test sample, cone, of ADTZ was 0.2mg/ml, AFB, 0.2ng/ml, pH=6.0. The mixture reacted at 28 Â°C for 120 min, then extracted with CHC13 in the same volume for three times. The pooled CHC13 extracts were evaporated under reduced pressure at 40" C. The extraction crude was dissolved in 6.75 ml DMSO (3.75 ml and 3 ml) as enzyme test mixture. Similarly, deactivated ADTZ (pre-treated with CHC13) was used to prepare deactivated enzyme control mixture, and buffer solution was used to prepare buffer control mixture. All these samples were kept at -15Â°C.<br>
2.2.5.	Reverse mutation assay (Ames assay)<br>
The test mixtures in DMSO, S9 mix and Salmonella Typhimurium TA98 cell culture were added to the top stratum soft agar medium, mixed thoroughly, and poured onto the minimum defined Vogel selective medium at 40Â° C. The plates were incubated for 72 h at 37" C, and mutant colonies in every plate were calculated.<br>
Each sample was tested with positive control and negative control, and repeated once. The result of each sample was get from the mean value of six groups of two tests. Data were reported in number of mutant colonies, mutation rate (MR = number of mutant colonies in sample/ number of mutant colonies in negative control) and inhibition ratio ( = {1-( number of mutant colonies in sample- number of mutant colonies in negative control)/ (number of mutant colonies in AFB! control sample- number of mutant colonies in negative control)} x 100%<br>
2.2.6. Data evaluation criteria<br>
Test samples are considered Ames positive when:<br>
a), solvent controls are in normal range;<br>
b). test samples show positive at three different cone. (MR &gt;_ 2).<br>
2.2.7.Result<br>
The number of mutant colonies in active enzyme test mixtures and DMSO control samples are very similar (MR  2). The data indicates the activity of ADTZ in inhibiting mutation caused by AFB,. Data are shown in the following table.<br>
Mutation assays of AFB, treated with ADTZ enzymes<br>
Test sample	The number of mutant	MR	Inhibition rate (%)<br>
colonies /plate<br>
PBS-control	378+77	13.09	-<br>
Deactivated enzyme	359+59	12.86	:<br><br>
Enzyme	31 Â± 12	1.11	99.16<br>
AFB,control	385+97	13.75<br>
DMSO control	28 Â±5	-	-<br>
Description: mutation assays used rat liver S9 and Salmonella Typhimurium TA 98 test strain. AFB, positive control: 0.8ug /50ul DMSO/plate. Enzyme test mixtures used the same amount of AFB, and DMSO. Plates were incubated for 28 h, numbers of mutant colonies were calculated. The data shown are from avenging of 4 plates + SD.<br>
Example 3<br>
Sequencing of ADTZ Peptide<br>
Samples: active fraction collected from example 1 or active fraction further purified<br>
by PAGE. ADTZ N-terminal peptide sequencing was conducted on a Micromass Q-TOF II mass<br>
spectrometer. The sequences are as following:<br>
M1:  EAWEGFTALVDK	M2:  NKLLQDANGELENLYVR<br>
The invention relates to other peptide sequences other than the one listed above, as long as they are detected from AAALDI-MS-TOF or other chemical methods on ADTZ peptide.<br>
Example 4<br>
Extraction of the Total RNA of Armillariella tabescenes The bacterial culture of Armillariella tabescenes was placed in a Petri dish on ice. The tissues was then frozen by immersion in liquid nitrogen, and grounded to powder. The powder (100 mg) was transferred into a 1.5 ml centrifuge tube and added Trizol (1 ml). The mixture was shook vigorously and incubated for 5 min at room temperature. Chloroform (200 ul) was added, the mixture was shook vigorously for 2 min, and placed in an ice bath for 5 min. The homogenate was centrifuged at 12000g for 15 min at 2-8Â°C. The supernatant containing RNA was carefully transferred to another 1.5 ml centrifuge tube. Cooled isopropanol (500 ul) was added, and the mixture was placed in an ice bath for 20 min. The mixture was centrifuged at 12000 g for 10 min at 2-8Â°C, then supernatant was removed and the RNA pellet was washed with 75% ethanol (1ml). The sample was centrifuged at 7500 g for 5 min at 2-8Â°C. Ethanol was removed and the RNA was dried for 5-10 min at room temperature.<br>
The RNA was completely dissolved in DEPC sterile water (50 ul), and was tested by UV and electrophoresis (1.1% Agarose gel / EB 100V, 20 min) analyses before stored at -80Â°C. The result was shown in Fig. 3. From electrophoresis, 28s rRNA and 18s sRNA were clearly visible. The ratio was about 2:1. It indicates that the total RNA was not degraded.<br>
Example 5<br>
Design of ADTZ Gene Primers<br>
The invention relates to two pairs of primers (P1, P2, and G1, G2) designed according to ADTZ peptide sequence. Partial ADTZ gene sequence products were obtained from RT-PCR using QJAGEN OneStep RT-PCR kit. The RT-PCR products were TA cloned. The recombinant plasmid form TA clone was identified by Hindlll and EcoRI enzyme incisions followed by<br><br>
electrophoresis (1.5% Agarose gel). ADTZ gene partial cDNA E1 was obtained from sequencing<br>
of the recombinant plasmid. Detail follows:<br>
Primer pair 1	Primer pair 2<br>
P1: 5'-TGGGARGGNTTYACNGC-3'	G1: 5'-CARGAYGCNAAYGGNGA-3'<br>
P1: 5'-TCNCCRTTNGCRTCYTG-3'	G2: 5'-GCNGTRAANCCYTCCCA-3'<br>
The invention relates to ADTZ specific primer pairs that are not limited to the above pairs, but also any other designed from ADTZ peptide sequence. 5.1.      RT-PCR<br>
5.1.1.	Template total RNA (from Example 4) was denatured at 75Â°C for 5 min, then<br>
cooled in ice bath.<br>
5.1.2.	Master mix preparation (80u.l system)<br>
42 nl RNase-free Water<br>
16 nl SxQIAGEN One-Step RT-PCR Buffer 3.2|il dNTPMix(IOmM) 3.2 ul QIAGEN One-Step RT-PCR Enzyme Mix 64.4^1<br>
Mixed thoroughly by pipetting the master mix up and down a few times. 5.1.3. Components added in the listed order to a sterile centrifuge tube (unit: pil)<br><br>
(Table Removed)<br>
5.1.4. PCR cycles<br>
â¢	Reverse transcription: 50Â°C, 30min<br>
. Initial PCR activation step: 50 Â°C, 15min<br>
â¢	3-step cycling<br>
â¢	15 cycles:    94 Â°C, 40 sec<br>
65 Â°C 1min (-1Â°C/cycle) 72Â°C1min<br>
â¢	25 cycles:    94 Â°C 40sec<br>
50Â°C1min 72Â°C1min<br>
â¢ Final extension: 70 Â°C 10min 5.1.5.   After the PCR cycles, 5 ul sample was taken for electrophoresis.<br>
5.2.	Extraction of RT-PCR product<br>
5.2.1.	TAE electrophoretic buffer solution, and 0.8% agarose gel were prepared.<br>
5.2.2.	50ul RT-PCR product and 10x loading buffer were mixed and loaded.<br>
5.2.3.	Electrophoresis at 100V for 20 min.<br>
5.2.4.	Bands were observed with UV light after electrophoresis. Interesting bands<br>
were extracted from gel and transferred to a 1.5 ml sterile centrifuge tube.<br>
5.2.5.	800 Ml Buffer NT1 was added.<br>
5.2.6.	Swirled the NucleoTrap suspension vigorously to a homogenous mixture. 10 pi was<br>
added to the centrifuge tube.<br>
5.2.7.	The centrifuge tube was immersed in water bath at 50 CÂ° for 6 min, vortexed<br>
every two min.<br>
5.2.8.	Centrifuging at 10000 g for 30 sec at room temperature, supernatant was<br>
removed.<br>
5.2.9.	SOOut Buffer NT2 was added, and the mixture vortexed. Centrifugation at<br>
10000 g for 30 sec at room temperature, supernatant was removed. The process was repeated<br>
once.<br><br>
5.2.10.	SOOul Buffer NT3 was added, and the mixture vortexed. Centrifugation at<br>
10000 g for 30s at room temperature, supernatant was removed. The process was repeated<br>
once.<br>
5.2.11.	Centrifuging at 10000 g for 30 sec, supernatant was removed. The residue<br>
was air dried for 10-25 min.<br>
5.2.12. The precipitate was suspended in 30 ul ITE buffer (pH 8.0). This fragment was named E1. Electrophoresis of this RT-PCR product was shown in Fig 4. A new band named E1 was observed from reaction of primer pair P1 and P2 (- 800 bp).<br>
5.3.	TA clones and sequencing<br>
5.3.1.	Ligation by DNA ligase<br>
The following components were added to a 1.5 m sterile centrifuge tube 1 ul pUCm-T carrier<br>
3	ul E1 fragment(RT-PCR product)<br>
1 ul 10xbuffer<br>
1 ul T4 DNA ligase<br>
4	ul sterile water, total volume 10 ul<br>
Mixed thoroughly by pipetting up and down a few times, incubated in water bath at 22 "C for at least 4h.<br>
5.3.2.	Preparation of E.coli DH5a competent cells using CaCl2 method<br>
DH5a monoclone was incubated in 2 ml LB medium, shook at 37Â°C overnight. 50 ul of the colony was transferred to 5 ml LB medium, shook at 37Â°C for 1.5 -2 h. The culture was then cooled to 0Â°C by keeping the tube on ice for 30 min. The culture was transferred to a sterile centrifuge tube, and centrifuged at 5000 rpm for 5 min. Medium was decanted from the<br>
cell pellet. The pellet was re-suspended in 1.5 ml ice-cold CaCl2 solution, and the tube was kept on ice for 10 min. The cells were recovered by centrifugation at 5000 rpm for 5 min, and medium was decanted. The pellet was re-suspended in 200 pi ice-cold CaCl2 solution and kept at4Â°C.<br>
5.3.3.	Transformation of DH5a competent cells<br>
200 (jl of the suspension of competent cells was transferred to a centrifuge tube containing linker DMA (10 ul). The contents were mixed by swirling gently and stored on ice for 30 min. After 90 sec in a water bath at 42 CÂ°, the contents were stored on ice for 3-5 min. Added LB medium (800 ul) and the mixture was incubated at 37Â°C for 40-60 min.<br>
The transformed competent cells were spread onto agar medium containing Amp and IPTG/X-gal (200 ul /90-mm plate) and incubated at 37Â°C for 12-16 h.<br>
5.3.4.	Alkaline extraction of plasmid DNA<br>
The transformed competent cells were transferred to 2 ml LB medium containing ampicillin. The culture was shook vigorously at 37Â°C overnight.<br>
The culture (1.5 ml) was transferred to micro-centrifuge tube, and centrifuged at 12000 rpm for 2 min. The supernatant was removed.<br>
The pellet was washed with 400 pi STE solution. The contents were mixed by swirling vigorously, and centrifuged at 12000 rpm. The supernatant was removed.<br>
The pellet was added cooled solution I, and shook vigorously, then added fresh prepared solution II, mixed well and stored on ice for 3 min.<br>
Cooled solution III was added, mixed well, and stored on ice for 5 min.<br>
The mixture was centrifuged at 12000 rpm for 5 min, and the supernatant was transferred to another tube.<br>
Equal volume of phenol chloroform was added to the supernatant.<br>
The mixture was centrifuged again. The supernatant was transferred to a third tube.<br>
Cooled anhydrous ethanol (2 volume) was added to the third tube. After mixing, the tube was kept at room temperature for 40-60 min.<br>
The content was centrifuged at 12000 rpm for 10min. Supernatant was removed.<br>
The pellet was washed with 70% ethanol (200 ul). The mixture was centrifuged again at 12000 rpm for 1 min. Supernatant was removed.<br>
The pellet was air-dried for 5-10 min, then suspended in DNase-free RNase TE buffer, incubated at 30Â°C for 1 h and stored at -20Â°C.<br>
5.3.5.	Identification of recombinant plasmid pTE1 by enzymatic incisions<br>
Hindlll and EcoRI enzymatic incisions of TA clones (unit: ul)<br><br>
(Table Removed)<br>
After enzymatic incision reaction at 37Â°C for 4 h, the mixtures were analyzed by 1.5%<br>
agarose electrophoresis. Results of enzymatic incision of recombinant vector pTE1 were shown<br>
in Fig. 5. Hindlll + EcoRI two enzyme incision (sample 1) and Hindlll single enzyme incision (sample 3) all showed the same band at 400 bp, with higher intensity for sample 1. EcoRI single enzyme incision (sample 2) showed linear cleavage. These results indicate that there is a Hindlll cleavage site in E1 fragment.<br>
5.3.6. Recombinant plasmid DNA sequencing<br>
Recombinant plasmid DNA was purified by precipitation with PEG (Sambrook, et al. 1989, Molecular Cloning, Cold Spring Harbor Laboratory Press, United States). The DNA sequence of E1 fragment was determined on an ABI377 DNA sequencer using T7 and SP6 sequencing primers. The determined sequence contains P1, P2, and a Hindlll cleavage site (aagctt).<br>
Example 6 Total cDNA Sequence of ADTZ Gene<br>
Primers were designed from ADTZ gene fragment E1 as determined from example 5:<br>
S1 : 5'-TAGGCGAAGTGTCGTCGTCAATGGAA-3'<br>
S3 : 5'-GAAGTTATCGGCTTTCCAGTCAGAGGGT-3'<br>
Using S1 and S3 as primers, 3'RACE and 5'RACE were conducted using SMARTâ¢ RACE cDNA amplification Kit (COLONTECH Laboratories, Inc. Cat. No. K1811-2). RACE products were recovered from scraping from gel, and TA cloned using routine method. Recombinant plasmid DNA fragments were sequenced following Hindlll and EcoRI enzyme incision and analysed by 1.5% agarose electrophoresis. Fragments E2 and E3 were thus obtained. Vector sequences were removed using Vecscreen software. E1, E2 and E3 were assembled using DNAMAN software (Lynnon BioSoft). The complete cDNA sequence of ADTZ gene was obtained from open reading frame analysis using ORF Finder (NCBI). Detail follows:<br>
Primer 51: 5'-TAGGCGAAGTGTCGTCGTCAATGGAA-3'<br>
primer S3: 5'-GAAGTTATCGGCTTTCCAGTCAGAGGGT-3' 6.1.      3'RACE<br>
6.1.1. Preparation of 3'RACE-Ready cDNA<br>
6.1.1.1.	Template total RNA (from Example 4) was denatured at 75Â°C for 5<br>
min, then cooled in ice bath.<br>
6.1.1.2.	The following reagents were added to a 0.5 ml sterile centrifuge tube:<br>
1 nl denatured template total RNA, 1 nl 3'-CDS primer A, and 3 nl RNase free sterile water to<br>
make the total volume 5 jal.<br>
6.1.1.3.	Mixed thoroughly by pipetting up and down a few times, followed a<br>
short centrifugation step.<br>
6.1.1.4.	Incubation at 70Â°C for 2 min.<br>
6.1.1.5.	Sample was stored on ice for 2 min. After a short centrifugation step,<br>
the following reagents were added:<br>
2 nl 5 x First-Strand Buffer 1 |il DTT (20mM)<br>
1 nl dNTPMix(IOmM)<br>
1 ui PowerScript Reverse Transcriptase<br>
10 ul total volume<br>
6.1.1.6.	Mixed thoroughly by pipetting up and down a few times, followed a<br>
short centrifugation step.<br>
6.1.1.7.	Incubation at42Â°Cfor 1.5h.<br>
6.1.1.8.	Dilution with 100 ^ Tricine-EDTA.<br>
6.1.1.9.	Incubation at 72Â°C for 7 min.<br>
6.1.1.10.	Storage at-20Â°C.<br>
6.1.2.3' RACEPCR<br>
6.1.2.1. Preparation of Master Mix (100ul system) 69 [i( PCR-Grade Water 10 ul 10xAdvantage 2 PCR Buffer 2ul dNTPMix(IOmM) 2 itl 50&gt;Advantage 2 Polymerase Mix 83 ut<br>
The contents were mixed thoroughly by pipetting up and down a few times, followed a short centrifugation step.<br>
6.1.2.2.Components added to a 0.5 ml sterile centrifuge tube in the order listed (unit: ul)<br><br>
(Table Removed)<br>
6.1.2.4. After PCR cycles, 5 ul sample was used for electrophoresis, the result was shown in Fig. 6. A single band was obtained from 3'RACE (-800 bp), and named E2.<br><br>
6.1.3.	TA clone of RACE product, preparation of E.coli DH5a competent cells (CaCl2<br>
method) and alkaline extraction of plasmid DNA were conducted as described in example 5.<br>
6.1.4.	Identification of recombinant plasmid pTE2 by enzymatic incisions<br>
Hindlll and EcoRI enzymatic incisions of TA clones (unit: ul)<br><br>
(Table Removed)<br>
After enzymatic reactions at 37Â°C for 4 h, the mixtures were analyzed by 1.5% agarose electrophoresis, results as shown in Fig. 7. Hindlll + EcoRI two enzyme incision (sample 1) showed two bands at 600 bp and 300-400 bp while Hindlll (sample 3) single enzyme incision showed only one band at 300-400 bp, EcoRI (sample 2) showed linear cleavage. These results indicated that there is a Hindlll cleavage site in E2 fragment, which is close to one end of the fragment.<br>
6.1.5. Sequencing<br>
Recombinant plasmid DNA was purified by precipitation with PEG (Sambrook, et al. 1989, Molecular Cloning,  Cold Spring Harbor Laboratory Press,   United States).  The DNA sequence of E2 fragment was determined on a ABI377 DNA sequencer using T7 and SP6 sequencing primers. The E2 sequence contains a Hindlll cleavage site (aagctt) close to 3'. 6.2.   5' RACE<br>
6.2.1. Preparation of 5'RACE-Ready cDNA<br>
6.2.1.1.	Template total RNA was denatured at 75Â°C for 5 min, then cooled in<br>
ice bath.<br>
6.2.1.2.	The following reagents were added to a 0.5 ml sterile centrifuge tube:<br>
1 ul denatured template total RNA, 1 \i[ 5'-CDS primer A, 1 \i( SMART II A Oligonucleotide, and 2<br>
lil RNase free sterile water to make the total volume 5 nl.<br>
6.2.1.3.	Mixed thoroughly by pipetting up and down a few times, followed a<br>
short centrifugation step.<br>
6.2.1.4.	Incubation at 70Â°C for 2 min.<br>
6.2.1.5.	Sample was stored on ice for 2 min. After a short centrifugation step,<br>
the following reagents were added:<br>
2 nl SxFirst-Strand Buffer<br>
1 ul DTT (20mM)<br>
1 ul dNTPMix(IOmM)<br>
1 ul PowerScript Reverse Transcriptase<br>
10 nl total volume<br>
6.2.1.6.	Mixed thoroughly by pipetting up and down a few times, and a short<br>
centrifugation step.<br>
6.2.1.7.	Incubation at42Â°C for 1.5 h.<br><br>
6.2.1.8.	Dilution with 100 ul Tricine-EDTA.<br>
6.2.1.9.	Incubation at 72Â°C for 7 min.<br>
6.2.1.10.	Storage at -20Â°C.<br>
6.2.2.5' RACEPCR<br>
6.2.2.1.	Preparation of Master Mix (110^1 system)<br>
75.9 nl PCR-Grade Water<br>
11 ul 10xAdvantage 2 PCR Buffer 2.2 |il dNTPMix(IOmM) 2.2 id SQxAdvantage 2 Polymerase Mix 91.3|il<br>
The contents were mixed thoroughly by pipetting up and down for a few times, followed a short centrifugation step.<br>
6.2.2.2.	Components added in the listed order to a 0.5 ml sterile centrifuge<br>
tube (unit: ul)<br><br>
(Table Removed)<br>
6.2.2.3.	PCR cycles:<br>
â¢	94Â° C 1 min<br>
â¢	5 cycles:   94Â°C	30 sec<br>
72Â°C 4 min<br>
â¢5 cycles:   94 Â°C	30 sec<br>
70Â°C	4 min<br>
â¢25 cycles:        94Â° C	30 sec<br>
68 Â°C	4 min<br>
72Â°C	10 min<br>
6.2.2.4.	After PCR cycles, 5 ul sample was used for etectrophoresis. The result<br>
was shown in Fig. 8. A single band was obtained form 5'RACE (-1400-1800 bp), and named E3.<br>
6.2.3.	TA clone of 5' RACE product, preparation of E.coli DH5a competent cells<br>
(CaCl2 method) and alkaline extraction of plasmid DMA were conducted as described in example<br>
5.<br>
6.2.4.	Identification of recombinant plasmid pTE3 by enzymatic incisions<br>
Hindlll and EcoRI enzymatic incisions of TA clones (unit: ul)<br><br>
(Table Removed)<br>
After enzymatic reactions at 37Â°C for 4 h, the mixtures were analyzed by 1.5% agarose electrophoresis, results as shown in Fig, 9. Hindlll + EcoRI two-enzyme incision (sample 1) showed two bands at 1400-1000 bp and 300-400 bp while Hindlll (sample 3) single-enzyme incision showed only one band at 1400-1000 bp. These results indicated existence of a Hindlll cleavage sites in E3 fragment.<br>
6.2.5. Sequencing<br>
Recombinant plasmid DMA was purified by precipitation with PEG (Sambrook, et al. 1989, Molecular Cloning, Cold Spring Harbor Laboratory Press, United States). The DMA sequence of E3 fragment was determined on an ABI377 DMA sequencer using T7 and SP6 sequencing primers. The E3 sequence contains two Hindlll cleavage sites, one close to 3'and the other close to 5'. 6.3. Assembly of ADTZ cDNA sequence fragments<br>
Vector sequences were removed using Vecscreen software. E1, E2 and E3 were assembled using DNAMAN software. The complete cDNA sequence of ADTZ gene was obtained from open reading frame analysis using ORF Finder (NCBI), which contains complete open reading frame with 3' poly(A) tail, 5' and 3' non-translating regions. The results were shown in the Sequence Listing as SEQ ID No.2.<br>
BLAST and BLASTX (http://www.ncbi.nlm.nih.gov/BLAST/) were used for sequence similarity search on ADTZ cDNA sequence and calculated protein sequence. The search identified ADTZ cDNA as a new sequence. ADTZ mature peptide sequence calculated from ADTZ cDNA was identified as a new peptide from search in GENEBANK.<br>
The invention relates to not only the method described in this example, but also clone of this sequence in Armillariella tabescens cDNA data bank using probe designed from ADTZ peptide sequence.<br>
Example 7<br>
Synthesis of ADTZ Mature Peptide Gene Encoding cDNA<br>
According to 3' and 5' end cDNA sequences, a pair of primes was designed to obtain open reading frame sequence.<br>
P3: 5'-GTCGAATTCATGGCCACCACAACTGTC-3'<br>
P4: 3'-GTAACTCTCTGCTAACACTCCTAGGGAC-5'<br>
Enzyme cleavage sites EcoRI (GAATTC) and BamHI (GGATCC) were incorporated to the primers. PCR amplification was performed and PCR product was scraped off from the agar. Detail follows: 7.1.      Preparation of Master Mix (100^1 system)<br>
69 nl PCR-Grade Water<br>
10 nl 10xAdvantage 2 PCR Buffer<br>
2nl dNTPMix(IOmM)<br>
2 ui 5QxAdvantage 2 Polymerase Mix<br>
83 nl<br>
The contents were mixed thoroughly by pipetting up and down for a few times, followed a short centrifugation step. 7.2. Components added in the listed order to a 0.5 ml sterile centrifuge tube (unit: |il)<br><br><br>
(Table Removed)<br>
7.3.	PCR cycles.<br>
â¢	94 Â°C   1 min<br>
â¢	5 cycles:       94Â°C   30 sec<br>
72Â°C   4 min<br>
â¢	5 cycles:   94Â°C 30 sec<br>
72Â°C 4 min<br>
â¢	35 cycles:     94Â° C   30 sec<br>
68Â°C   4 min 72Â°C         10 min<br>
7.4.	After PCR cycles, 5 pi sample used for electrophoresis, the result was shown in Fig. 10. A<br>
single band obtained from PCR (-1800 bp) was named ADTZ1 fragment.<br>
5.      Recovery of PCR product<br>
PCR product was scraped off from the gel. The cDNA encoding ADTZ mature peptide was thus obtained. This fragment was named "ADTZ"'<br>
Example 8 Construction of Recombinant ADTZ Expression Plasmid<br>
ADTZ' from example 7 was cloned to pHIL-S1 to construct expression vector pHIL-S1-ADTZ following standard procedure (Sambrook, et al. 1989, Molecular Cloning, Cold Spring Harbor Laboratory Press, United States). The product was analyzed by enzymatic incisions and sequenced. Detail as following:<br>
As shown in Fig. 14, the construction of hybrid plasmid containing ADTZ gene was as following:<br>
Plasmid pHIL-S1 and fragment ADTZ' were cleaved by EcoRI + Baml two enzyme incisions. The mixtures were subjected to 0.8% agarose electrophoresis, and extracted from the gel. Recombinant plasmid pHIL-S1-x\DTZ was constructed from vector pHIL-S1 and ADTZ gene by T4 DMA ligase enzyme.<br>
E.coli DH5a competent cells were prepared using CaCl2 method and transformed. The transformed cells were selected, and plasmid DNA obtained from alkaline extraction. Recombinant plasmid DNA was purified by precipitation with PEG (Sambrook, et al. 1989, Molecular Cloning, Cold Spring Harbor Laboratory Press, United States). DNA sequence was determined on an ABI377 DNA sequencer using T7 and SP6 sequencing primers.<br>
Enzyme incision of recombinant plasmid pHIL-S1-^Drz (pSA) was shown in Fig. 11: BamHI and EcRI two enzyme incision (sample 1) showed a single band (-2000 bp), Hindlll single enzyme incision (sample 2) showed three bands (-1400 bp, 600 bp and 500 bp), Sacl single enzyme incision showed linear cleavage (indicating no Sacl cleavage site in the inserted fragment).<br>
Example 9 Expression of Recombinant ADTZ Gene<br>
Recombinant plasmid pHIL-S1-,4Drz and expression vector pHIL-S1 were cleaved by Sacl. The mixtures were subjected to 0.8% agarose electrophoresis. Linearized recombinant plasmid pHIL-S1-/4DTZ and vector pHIL-S1 were extracted from the gel. Mut+ transformants were selected following spheroplast transformation of Pichia pastoris GS115 (Pichia Expression kit manual, Invitorgen Inc. United States). Methanol was used as the only carbon source for the induced expression of Pichia pastoris GS115. SDS-PAGE of the incubation mixture showed clearly protein band following induced expression, while the control sample with no ADTZ gene showed no protein band. The results were shown in Fig. 12. Detail follows: Homogeneous recombination of recombinant plasmid in Pichia pastoris<br>
9.1.	Linearization of plasmids<br>
Recombinant plasmid pHIL-S1-x4D7Z (pSA) and expression vector pHIL-S1 were cleaved by Sacl. The later was used as control for the following experiment.<br>
pSA enzyme incision (120 pi): 12ml Buffer L+ 8ml Sack 100ml pSA.<br>
pHIL-S1 enzyme incision (120 pi): 12ml Buffer L+ 8ml Sack 100ml pHIL-S1.<br>
The mixtures were subjected to 0.8% agarose electrophoresis. Linearized recombinant plasmid pSA and vector pHIL-S1 were extracted from the gel.<br>
9.2.	Incubation of Pichia pastoris for spheroplast transformation<br>
9.2.1.	Inoculated 10 ml of YPD (Yeast Extract Peptone Dextrose medium) with a single<br>
colony of Pichia pastoris GS115. Grew overnight at 30Â° C in a shaking incubator (250-300 rpm).<br>
9.2.2.	Inoculated 200 ml of YPD with 5, 10, and 20 ul of the overnight culture. These<br>
samples were incubated overnight at 30Â° C in a shaking incubator (250-300 rpm).<br>
9.2.3. The three cultures were tested for OD600. The ones with OD600 = 0.2-0.3 were selected, and pelleted by centrifugation at 1500 xg for 5 min. The supernatant was discarded. The cells were used for spheroplast transformation.<br>
9.3.	Preparation of Pichia pastoris GS115 spheroplasts<br>
9.3.1.	The cell pellet was re-suspended in 20 ml sterile water, and transferred to two 10<br>
ml centrifuge tubes.<br>
9.3.2.	The cells were pelleted by centrifugation at 1500 xg for 5 min. The supernatant<br>
was discarded.<br>
9.3.3.	The cell pellet was washed with fresh prepared SED, followed by centrifugation at<br>
1500 xg for 5 min. The supernatant was discarded.<br>
9.3.4.	The cell pellet was washed with 1M Sorbitol solution, followed by centrifugation<br>
at 1500 xg for 5 min. The supernatant was discarded.<br>
9.3.5.	The cell pellet was re-suspended in 10 ml SCE.<br>
9.3.6.	Zymolyase in a tube was thawed and mixed by flicking the tube.<br>
9.3.7.	7.5 ul of Zymolyase was added and incubated for 30 min at 30Â°C.<br>
9.3.8.	The cells were pelleted by centrifugation at 1500 xg for 5 min. The supernatant<br>
was discarded.<br>
9.3.9.	The transformation mixture was washed with 1M Sorbitol solution, mixed by<br>
flicking the tube to disperse the precipitate. The cells were pelleted by centrifugation at 750<br>
xg for 5 min at room temperature. The supernatant was discarded.<br><br>
9.3.10.	The cell pellet was washed with 10 ml CaS solution, followed by centrifugation<br>
at 750 xg for 5 min. The supernatant was discarded.<br>
9.3.11.	The cell pellet was re-suspended in 0.6 ml CaS solution. The spheroplasts must<br>
be used within 30 min.<br>
9.4.	Spheroplast transformation of Pichia pastoris GS115<br>
9.4.1.	Aliquots of 100 ul each of Pichia pastoris GS115 spheroplasts were transferred to<br>
three sterile centrifuge tubes A, B and C.<br>
9.4.2.	Tube A (no DNA) negative control, tube B (added 30 ul linearized vector pHIL-S1),<br>
tube C (added 30 ul linearized recombinant plasmid pSA, incubated for 10 min at room<br>
temperature). 3 ml of PEG/CaT was prepared at the same time.<br>
9.4.3.	Aliquots of 1 ml each of PEG/CaT were added to tube A, B and C, mixed gently<br>
and incubated for 10 min at room temperature.<br>
9.4.4.	The cells were pelleted by centrifugation at 750 xg for 5 min. The supernatant<br>
was discarded.<br>
9.4.5.	The cell pellets were re-suspended in 150 ul SOS, incubated for 20 min at room<br>
temperature.<br>
9.4.6.	Aliquots of 850 ul 1M Sorbitol solution each were added to the tubes.<br>
9.4.7. The entire transformations were plated on RD solid incubation plates using a sterile spreader (200ul/plate). The plates were incubated at 28-30Â°C. Transformants appeared between 4-6 days.<br>
9.5.	Selection of Mut* transformants<br>
9.5.1.	Using a sterile toothpick, His+ transformants were patched on both MM and MD<br>
plates, the strains GS115/His+Muts Albumin and GS115/His* Mut* B-gal were also patched on the<br>
plates as controls.<br>
9.5.2.	Plates were incubated at 28-30/C for 2 days.<br>
9.5.3.	After two days, scored the plates. Mut* strains will grow normally on both plates,<br>
while Muts will grow normally only on the MD plate but little or no growth on MM plate.<br>
9.6.	Induced expression of the recombinant strains<br>
9.6.1.	Inoculated a single colony of His"Mut+ transformant in 25 ml BMG in a 250 ml<br>
baffled flask. Grew at 28-30Â°C in a shaking incubator (250-300 rpm) until the culture reached<br>
OD600= 2-6 (-16-18 h).<br>
9.6.2.	Cells were harvested by centrifugation at 1500-3000 xg for 5 min at room<br>
temperature. Supernatant was decanted and cell pellet was re-suspended in BMM to an OD600of<br>
1.0 (~100-200ml BMM). The culture was placed in a 1-litter baffled flask and returned to<br>
incubator to continue growth at 250-300 rpm at 28-30Â°C.<br>
9.6.3.	100 % methanol was added to a final concentration of 0.5 % to maintain inducted<br>
expression.<br>
9.6.4.	After 96h, the expression culture was centrifuged for 2-3 min, supernatant was<br>
transferred to a separate tube and stored at -SOT for purification of expression product.<br>
The supernatant of the culture after 96h induction was analyzed. Total mount of protein was 0.23 mg/ml. The molecular weight of the protein product is consistent with the predicted value of 76.95 kDa by BioEdit (http://www.mbio.ncsu.edu/BioEdit/bioedit.html).<br>
Example 10 Purification of Recombinant ADTZ<br>
The recombinant expression culture was precipitated with 70% saturation (NH^SC^, producing crude enzyme as precipitate. The crude enzyme was dissolved in equal volume of PBS, centrifuged. The supernatant was loaded on a hydrophobic Phenyl Sepharose column; active product was collected from gradient elution. The product was subjected to dialysis desalination and concentrated after equilibration with PBS. The concentrated crude enzyme solution was then purified by metal chelating affinity chromatography using Chelating Sepharose column. The active peak was eluted using pH gradient pH7.5-6.0 and fraction collected. Details follow:<br>
10.1.      Crude enzyme from (NH4)2S04 precipitation<br>
(NH4)2S04 powder was added to the recombinant expression culture until 40 % saturation followed by centrifugation at 10000 g for 20 min at 4Â°C. The supernatant was added more<br>
(NH4)2S04 until 70 % saturation. Crude enzyme was obtained from centrifugation at 10000 g for 20minat4Â°C.<br>
10.2.	Hydrophobic interaction chromatography<br>
ADTZ crude enzyme was dissolved in equal volume of 0.02 M PBS (pH 6.0). and centrifuged at 4000 g for 10 min at 4Â°C. Supernatant was loaded on a Phenyl Sepharose column (Pharmacia Biotech. Inc., United States) which had been washed to background using 0.02M PBS + 30% saturation (NH4)2S04 , pH 6.0. Gradient elution with A (0.02M PBS + 10% saturation (NH4)2S04 , pH 6.0 ) and B (0.02 M PBS, pH 6.0) gave an active product. The product was subjected to dialysis desalination and concentrated after equilibration with F solution (0.02 M PBS + 5 M NaCl, pH 7.5) to 1 mg/ml.<br>
10.3.	Metal chelating affinity chromatography<br>
Chelating Sepharose (Pharmacia Biotech. Inc., United States) was saturated with 0.2 M CuCl2, and then equilibrated with water and F solution (0.02 M PBS + 5 M NaCl, pH 7.5). The pooled fractions from hydrophobic interaction chromatography were loaded and purified with non-linear pH gradient using buffer G (0.02 M PBS + 0.5 M NaCl, pH 7.5-6.0, non-linear gradient incremented by 0.5 pH unit). The product peak was collected, and analyzed using SDS-PAGE.<br>
Conclusion: 58 mg purified recombinant ADTZ was obtained from I liter expression culture, purity was greater than 95%.<br>
Example 11<br>
Test of recombinant ADTZ activity<br>
Recombinant ADTZ activity was test following protocols in example 2.1. Test mixtures: (1) Enzyme mixture: 1 ul AFB, solution (0.5 ug/pl in MeOH, evaporated under nitrogen) + recombinant ADTZ enzyme solution (0.1 mg/ml, 300 ul) + 0.5 ul MgS04 + 10 ul PEG200; (2) Deactivated enzyme mixture: 1 ul AFB, solution (0.5 ug/ul in MeOH, evaporated under nitrogen) + deactivated recombinant ADTZ enzyme solution (0.1 mg/ml, 300 ul, preheated at 100Â°C for 5 min) + 0.5 ul MgS04 + 10 ul PEG200; (3) control mixture: 1 ul AFB, solution (0.5 ug/ul in MeOH, evaporated under nitrogen) + 300 [i( PBS buffer + 0.5 ul MgS04 + 10 ul PEG200. The test mixtures were mixed thoroughly, reacted for 1 h at 30Â°C, AFB, (0.5 ul x 2) was added every hour (total AFB1: 2 ug). After the addition, the reactions continued for 6 more hours. After the reaction, TLC (as described in example 1) showed new product with R/ -1 (= 0.93) for recombinant ADTZ, very similar to natural ADTZ., indicting activity of recombinant ADTZ in transforming AFB,. Result was shown in Fig. 13.<br>
Example 12<br>
Bioactivity of Recombinant ADTZ in Detoxifying AFBi The bioactivity of recombinant ADTZ was tested following protocols in example 2.2, where recombinant ADTZ was used instead of natural ADTZ. The number of mutant colonies of active recombinant ADTZ test sample is very similar to the negative control (DMSO control sample) (MR 
(MR &gt; 2), and are very close to the positive control (AFBi control sample). These results demonstrate the bioactivity of recombinant ADTZ. The results are shown in the following table.<br><br>
(Table Removed)<br><br><br>
We claim:<br>
1.	A detoxifizyme with activity of transforming aflatoxin, wherein its isoelectric point<br>
is 5.3-6.8, and its molecular weight is 73-77 kilodaltons, and its amino acid sequence is depicted<br>
in the Sequence Listing.<br>
2.	A gene comprising a nucleoside acid sequence coding for a detoxifizyme with activity<br>
of transforming aflatoxin as claimed in claim 1.<br>
3.	The gene of claim 2, wherein contains a nucleoside acid sequence depicted in the<br>
sequence listing.<br>
4.	A recombinant expression carrier comprising a gene as claimed in claim 3.<br>
5.	A transformant obtained by a host cell transformed with a recombinant expression<br>
carrier as claimed in claim 4.<br>
6.	A method for the preparation of a detoxifizyme as claimed in claim 1, which method<br>
comprising:<br>
cultivating a transformant as claimed in claim 5; and<br>
separating,   purifying   and   recovering   the   expressed   detoxifizyme   with   activity   of transforming aflatoxin.<br>
7.	The use of a detoxifizyme with activity of transforming aflatoxin as claimed in claim<br>
1 for the manufacturing of feed or food.<br>
8.	The use of a detoxifizyme with activity of transforming aftatoxin as claimed in claim<br>
1 for the preparation of a medicament for preventing or treating tumor or cancer.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUNsYWltcy0oMjQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Claims-(24-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Correspondence Others-(07-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1517-DELNP-2007-Correspondence Others-(11-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Correspondence Others-(24-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDYtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Correspondence-Others-(06-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTctMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1517-DELNP-2007-Correspondence-Others-(17-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1517-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUZvcm0tMS0oMjQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Form-1-(24-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUZvcm0tNS0oMjQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Form-5-(24-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LUdQQS0oMjQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-GPA-(24-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMjQtMDUtMjAxMyktMS5wZGY=" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Petition-137-(24-05-2013)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUxNy1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMjQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1517-delnp-2007-Petition-137-(24-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU3MS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMjAtMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1571-delnp-2007-Correspondence-others (20-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU3MS1kZWxucC0yMDA3LUZvcm0tMyAoMjAtMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1571-delnp-2007-Form-3 (20-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="259288-method-and-apparatus-for-packet-communications-in-wireless-systems.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259290-data-communication-method-and-mobile-communication-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259289</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1517/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>07-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Feb-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GUANGZHOU CO-WIN BIOENGINEERING CO., LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ROOM 901, GUANGRI MASION, NO.9, SIYOU NAN ROAD, WUYANG, XINCHENG, GUANGZHOU, GUANGDONG, 510600 CHINA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LIU, DALING</td>
											<td>DEPARTMENT OF BIOTECHNOLOGY, JINAN UNIVERSITY, 601 HUANG-PU ROAD, GUANGZHOU, GUANGDONG, 510632 CHINA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GUAN, MIN</td>
											<td>DEPARTMENT OF BIOTECHNOLOGY, JINAN UNIVERSITY, 601 HUANG-PU ROAD, GUANGZHOU, GUANGDONG, 510632 CHINA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>XIE CHUNFANG</td>
											<td>DEPARTMENT OF BIOTECHNOLOGY, JINAN UNIVERSITY, 601 HUANG-PU ROAD, GUANGZHOU, GUANGDONG, 510632 CHINA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YAO, DONGSHENG</td>
											<td>DEPARTMENT OF BIOTECHNOLOGY, JINAN UNIVERSITY, 601 HUANG-PU ROAD, GUANGZHOU, GUANGDONG, 510632 CHINA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/CN2005//000050</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-01-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>200410051120.0</td>
									<td>2004-08-17</td>
								    <td>China</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259289-detoxifizyme-with-activity-of-transforming-aflatoxin-and-the-gene-encodes-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:33:53 GMT -->
</html>
