<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259259-an-isolated-promoter-capable-of-driving-and-or-regulating-expression-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:40:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259259:&quot;AN ISOLATED PROMOTER CAPABLE OF DRIVING AND/ OR REGULATING EXPRESSION IN PLANTS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED PROMOTER CAPABLE OF DRIVING AND/ OR REGULATING EXPRESSION IN PLANTS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An isolated promoter capable of&#x27; driving and/or regulating expression, comprising: (a) An isolated nucleic acid as given in any one of SEQ ID NO 1 to 17, 19, 20, 21 and 22 or the complement of any one of SEQ ID NO 1 to 17, 19, 20, 21 and 22; or (b) an isolated nucleic having at least 90% sequence identifying with any of the DNA sequences as given in any one of the SEQ ID NO 1 to 17, 19, 20, 21 and 22; or (c) an isolated nucleic acid specifically hybridizing under stringent conditions with any of the DNA sequences as given in any one of SEQ ID NO 1 to 17, 19, 20, 21 and 22; or (d) an isolated nucleic acid as defined in any one of (a) to (c), which is interrupted by an intervening sequence; or (e) a fragment of any of the nucleic acids as defined in (a) to (d), which fragment is capable of driving and/or regulating expression.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>RICE PROMOTERS<br>
The present invention relates to the field of plant molecular biology, more particularly to   nucleic acid sequences useful for driving and/or regulating expression of an operably linked nucleic acid in plants. The isolation of these nucleic acid sequences from rice, as well as their use in driving and/or regulating expression of an operably linked, nucleic acid is disclosed. The present invention therefore concerns promoters, hybrid promote rs, genetic constructs,   expression cassettes, transformation vectors, expression vectors, host cells and transgenic plants comprising the isolated nucleic acids according to the present invention. The present invention also concerns methods for driving and/or regulating expression of a nucleic acid and methods for the production of transgenic plants.<br>
Gene expression is dependent on initiation of transcription, which is mediated via the transcription initiation complex. Gene expression is also dependent on regulation of transcription, which regulation determines how strong, when or where a gene is expressed . Said regulation of gene expression may be mediated via transcriptional control elements, which are generally embedded in the nucleic acid sequence 5'-flanking or upstream of the expressed gene. This upstream nucleic acid region is often referred to as a "promoter" since it promotes the binding, formation and/or activation of the transcription initiation complex and therefore is capable of driving and/or regulating expression of the 3' downstream nucleic acid sequence.<br>
Genetic engineering of plants with the aim of obtaining a useful plant phenotype, often involvesheteroiogous gene expression, which is generally mediated by a prompter capable of drivingand/or regulating expression of an operably linked heterologous nucleic acid, the phenotype ofthe host plant only depends on the contribution of the heterologous nucleic acid, but also on thecontribution of the specific expression pattern of the chosen promoter determining how, where andwhen that heterologous nucleic acid is expressed. Accordingly, the choice of promoter with asuitable expression pattern is of crucial importance for obtaining the suitable phenotype. A personskilled in the art will need to have available different promoters, to determine the o ptimal promoterfor a particular nucleic acid. For many different host plants, this availability is  rather limited andthere is therefore a continuing needto-provide  new premeters-with-various-expression-profttesThe nucleic acids as presented in SEQ ID NO 1 to 22 were isolated from Oryza sativa and have been found to be capable of driving and regulating expression of an operably I inked nucleic acid; their expression patterns have also been characterized. Therefore the present invention offers a collection of hitherto unknown isolated nucleic acids, which isolated nucleic acids are useful as promoters.<br>
Accordingly, the present invention provides a n isolated   promoter capable of driving and/or regulating expression, comprising:<br>
(a)	an isolated nucleic acid as given in any one of SEQ ID NO i to 22 or the complement<br>
of any one of SEQ ID NO 1 to 22; or<br>
(b)	an isolated nucleic acid having at least 90% sequence Identity with any of the DNA<br>
sequences as given in any one of SEQ ID NO 1 to 22; or<br>
(c)	an isolated nucleic acid specifically h ybridizing under stringent conditions with any of<br>
the DNA sequences as given in any one of SEQ ID NO 1 to 22; or<br>
(d)	an isolated nucleic acid as defined in any one of (a) to (c), which is interrupted by   an<br>
intervening sequence; or<br>
(e)	a fragment of any of the nu cleic acids as defined in (a) to (d), which fragment is<br>
capable of driving and/or regulating expression.<br>
The term "isolated" as used herein means being removed from its original source . Preferably, the "isolated" promoter is free of sequences (such as protein encoding sequences or other sequences at the 3' end) that naturally flank the promoter in the genomic DNA of the or ganism from which the promoter is. derived. Further preferably, the "isolated" promoter is also free of sequences that naturally flank it at the 5' end. Further preferably, the 'isolated" promoter may comprise less than about 5 kb, 4 kb, 3 kb, 2 kb, 1.5 kb, 1.2 kb, 1 kb, 0.8 kb, 0.5 kb or 0.1 kb of nucleotide sequences that naturally occur with the promoter in genomic DNA from the organism of which the promoter is derived.<br>
The present invention is not limited to the nucleic acids as presented by SEQ ID NO 1 to 22 . A person skilled in the art will recognize that variants or fragments of a nucleic acid may occur, whilst maintaining the same functionality. These varia nts or fragments may be man made (e.g. by genetic engineering) or may even occur in nature. Therefore the present invention extends to variant nucleic acids and fragments of any of SEQ ID NO 1 to 22, which variants or fragments are useful in the methods of the present invention . Such variants and fragments include:<br>
(a)	an isolated nucleic acid as given in any one of SEQ ID NO 1 to 22 or the complement<br>
of any one of SEQ ID NO 1 to 22; or<br>
(b)	an isolated nucleic acid having at least 90% sequence identity with any of the DNA<br>
sequences as given in any one of SEQ ID NO 1 to 22; or	;<br>
(c)	an isolated nucleic acid specifically hybridizing under stringent conditions with any of<br>
the DNA sequences as given in any one of SEQ ID NO'1 to 22; or<br>
(d)	an isolated nucleic acid as defined in any one of (a) to (c), which is inte trupted by an<br>
intervening sequence; or<br>
(e)	a fragment of any of the nucleic acids as defined in (a) to (d). which fragment is<br>
capable of driving and/or regulating expression.<br>
Suitable variants of any one of SEQ ID NO 1 to 22 encompass homologues which have in increasing order of preference at least 90%, 91 %. 92% 93%, 94%,: 95%, 96%, 97%, 98% or 99% sequence identity with any one of the nucleic acids as represented in SEQ ID N01 to 22 .<br>
The percentage of identity may be calculated using an alignment program. Preferably a pair wise global alignment program may be used, which implements the algorithm of Needleman -Wunsch (J. Mol. Biol. 48:443-453,1970). This algorithm maximizes the number of matches and minimizes the number of gaps. Such programs are for example GAP, Needle (EMBOSS package), stretcher (EMBOSS package) or Align X (Vector NTI suite 5.5) and may use the standard parameters (for example gap opening penalty 15 and gap extension penalty 6.66). Alternatively , a local alignment program implementing the algorithm of Smith -Waterman (Advances in Applied Mathematics 2, 482-489 (1981)) may be used. Such programs are for example Water (EMBOSS package) or matcher (EMBOSS package). "Sequence identity" as used herein is preferably calculated overthe entire length of the promoters as represented by any one of SEQ ID NO 1 to 22 . The length of these promoters is presented in Table 2.<br>
Search and identification of homolog ous nucleic acids, would be well within the realm of a person skilled in the art. Such methods, involve screening sequence databases with the sequences provided by the present invention, for example any one of SEQ ID NO 1 to 22, preferably in a computer readable form. UsefuLsequence databses,JndudeJttit^rB_natJimfed-to-Genbank— fhtto://vvww.ncbi.nlm.nih.oov/web/Genbank). the European Molecular Biology Laboratory Nucleic acid Database (EMBL) (http:Aw.ebi.ac.uk/ebi -docs/embl-db.html) or versions thereof, or the MIPS database (http://mips.gsf.de/). Different search algorithms and software for the alignment and comparison of sequences are well known in the art. Such software includes , for example GAP,<br>
BESTFIT, BLAST, FASTA and TFASTA. Preferably BLAST software is used, which calculates percent sequence identity and performs a statistical analysis of the similarity between the sequences. The suite of programs referred to as BLAST programs has 5 different implementations: three designed for nucleotide sequence queries (BLASTN, BLASTX, and TBLASTX) and two designed for protein sequence queries (BLASTP and TBLASTN) (Coulson, Trends in Biotechnology: 76-80,1994; Birren etal., GenomeAnatysis, 1:543,1997). The software for performing BLAST analysis is publicly available through the National Centre forB iotechnology Information.<br>
The sequences of the genome of Arabidopsis thaliana and the genome of OryzasaOva are now available in public databases such as Genbank . Other genomes are currently being sequehced. Therefore, it is. expected that as more sequences of the genomes of other plants become available, homologous promoters maybe identifiable by sequence alignment with any one of SEQ ID NO 1 to SEQ ID NO 22. The skilled person will readily be able to find homologous promoters from other plant species, for example from other crop plants, such as maize. Homologous promoters from other crop plants are especially useful for practising the methods of the present invention in crop plants.<br>
One example of homologues having at least 90% sequence identity with any one of SEQ ID NO to 22 are allelic variants of any one of SEQ ID N01 to 22. Allelic variants are variants of the same gene occurring in two different individuals of the same species and usually allelic variants differ by slight sequence changes. Allelic variants may encompass Single Nucleotide Polymorphisms (SNPs) as welf as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDE Ls form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms.<br>
Homologues suitable for use in the methods according to the invention may readily  be isolated from their source organism via the te chnique of PCR or hybridization. Their capability of driving and/or regulating expression may readily be determined, for example, by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the homologue.<br>
Other suitable variants of any one of SEQ ID NO 'I to 22 encompassed by the present invention are nucleic acids specifically hybridising under stringent conditions to any one of the nucleic acids of SEQ ID NO 1 to 22. The term "hybridising" means annealing to substantially homologous<br>
complementary nucleotide sequences   in a hybridization process. Tools in molecular biology 'relying on such a hybridization process include the polymerase chain reaction (PCR; and all methods based thereon), subtractive hybridisation, random primer extension, huclease S1 mapping, primer extension, reverse transcription, cDNA synthesis; differential display of RNAs, and DMA sequence determination, Northern   blotting (RNA blotting), Southern blotting (DNA blotting). The hybridisation process can also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin; Tools in molecular biology relying on such a process include the isolation of poly (A+) mRNA. The hybridisation process can furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro -cellulose or nylon membrane or immobilised by e.g. photolithography to, for example, a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chip's). Tools in molecular biology relying on such a process include RNA and DMA gel blot analysis, colony hybridisation , plaque hybridisation, in situ hybridisation and microarray hybridisation. In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, salt concentration and hybridisation buffer composition.   Conventional hybridisation conditions are described in, for example, Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York, but the skilled craftsman will appreciate that numerous different hybridisation conditions can be designed in function of the   known or the expected homology and/or length of the nucleic acid sequence.   High stringency conditions for hybridisation include high temperature and/or low sodium/salt concentration (salts include sodium as for example in NaCI and Na 3-citrate) and/or the i nclusion of formamide in the hybridisation buffer and/or lowering the concentration of compounds such as SDS (sodium dodecyl sulphate detergent) in the hybridisation buffer and/or exclusion of compounds such as dextran sulphate or polyethylene glycol (promoting molecular crowding) from the hybridisation buffer.   Specifically hybridising under stringent conditions means that the sequences have to be very similar . Specific hybrisization under stringent conditions is preferably carried out at a temperature of 60°C followed by washes in 0.1 to 1 XSSC, 0.1XSDS, and 1X SSC, 0.1X SDS.<br>
The invention also relates lo a nucleic acid molecule of at least 15 nuclsotides in length hybridizing specifically with any of the nucleic acids of the invent) on. The invention also relates to a nucleic acid molecule of at least 15 nucleotides in length specifically amplifying a nucleic acid of the invention by polymerase chain reaction.<br>
Another variant of any of SEQ ID NO 1 to 22 encompassed by the pr esent invention are nucleic acids corresponding to any one of SEQ ID NO 1 to 22   or variants thereof as described hereinabove, which are interrupted by an intervening sequence. For.example, any of the nucleic acids as presented in SEQ ID NO 1 to 22 may be interrupted by an intervening sequence. With "intervening sequences" is  meant any nucleic acid   or nucleotide, which disrupts    another sequence. Examples of intervening sequences comprise introns,   nucleic acid tags, T-DNA and mobilizable nucleic acids sequences such as transposons   or nucleic acids that can be mobilized via recombination. Examples of particular tra nsposons comprise Ac (activator), Ds (Dissociation), Spm (suppressor-Mutator,) or En. The introduction of introns into promoter s is now widely applied. The.methods according to the present invenfion may also be practised using a nucleic acid sequence according to any one of SEQ ID NO 1 to 22 provided with an intron. In case the intervening sequence is an intron, alternative splice variants of the nucleic acids according to the invention may arise. The term "alternative splice variant" as used herein encompasses variants of a nucleic acid sequence in which intervening introns have been excised, replaced or added. Such splice variants may be found in nature or may be manmade. Methods fo r making such promoters with an intron or for making the corresponding splice variants are we|I known in the art<br>
Variants interrupted by an intervenin g sequence, suitable for use in the methods according to th e invention may readily be determined for example by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the variant.<br>
The variant nucleic acids as described hereinabove may be found in nature (for example allelic variants or splice variants). Additionally and/or alternatively, variants of any one of SEQ ID N01 to 22 as described hereinabove may be manmade via techniques well known in the art involving for example mutation, substitution, insertion, deletions or derivation . T he present invention also encompasses such variants, as well as their use in the methods of the present Invention .<br>
A "mutation variant" of a nucleic acid may readily be mad e using recombinant DMA manipulation techniques or nucleotide synthesis. Examples of such techniques include site directed jm!iagenesis___yia _ M13. mutegenesis, T7-Gen in viiro mutagenesis (USB, Cleveland, OH), QuickChanga Site Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols. Alternatively, the nucleic acid of the present invention may be randomly mutated.<br>
A "substitutional variant" refers to those variants in which at least o ne residue in the nucleic acid sequence has been removed and a different residue inserted in its place.  Nucleic acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the nucleic acid sequence; insertions usualfy are of the order of about 1 to about 10 nucleic acid residues, and deletions can range from about 1  to about 20 residues.<br>
An "insertional variant" of a nucleic acid is a variant in which one or more nucleic acid residues are introduced into a predetermined site in that nucleic acid. Insertions may comprise 5' -terminal and/or 3' -terminal fusions as well as infra -sequence insertions of single or multiple nucleotides. Generally, insertions within the nucleic acid sequence will be   smaller than 5'- or 3'-terminal fusions, of the order of about 1 to 10 residues. Examples of 5' - or 3'-terminaI fusions include the coding sequences of binding domain s or activation domains of a transcripfional activator as used in the yeast two -hybrid system or yeast one-hybrid system , or of phage coat proteins, (histidine) e-tag, glutathione S-transferase-tag, protein A, maltose -binding protein, dihydrofolate reductase, Tag«100 epitope, c-myc epitope, FLAG ""-epitope, lacZ, CMP (calmodulin -binding peptide), HA epitope, protein C epitope and VSV epitope.<br>
The term "derivative" of a nucleic acid may comprise substitutions, and/or deletions and/or<br>
additions of naturally and non -naturally occurring nucleic acid residues compared to the natural<br>
nucleic acid. Derivatives may, for example, comprise methylated nucleotides, or artificial<br>
nucleofides.	•<br>
Also encompassed with in the present invention are promoters, comprising a fragment of any of the nucleic acids as presented by any one of SEQ ID NO 1 to 22 or variants thereof as described hereinabove. A "fragment" as used herein means a portion of a nucleic acid sequence. Suitable fragments useful in the methods of the present invention are functional fragment s, which retain at least one of the functional parts of the promoter and hence are still capable of driving and/or regulating expression . Examples of functional fragments of a promoter include the minimal promoter, the upstream regulat ory elements , or any combination thereof .<br>
Suitable- fragments may range from- at least about-29-base-pairs or-about-50,100,150,200,250<br>
300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950 or 100 0 base pairs, up to<br>
about the full length sequenca of the invention. T hese base pairs are typically immediately<br>
upstream of the transcription initiation start, but alternatively may be from anywhere in the promoter sequence.<br>
Suitable fragments useful in the methods of the present invention may be tested for their capability of driving and/or regulating expression by standard techniques wel| known to the skilled person, or by the following method described in the Example section .<br>
The promoters as disclosed in any one of SEQ ID NO ,1. to 22 are isolated as nucjeic acids of approximately 1.2kb from the upstream region of particular rice coding sequences (CDS). These nucleic adds may include typical elements of a promoter, which are presented in Figure 1. Generally, a promoter may comprises from coding sequence to the upstream direction: (i) an 5'UTR of pre-messenger RNA, (ii) a minimal promoter comprising the transcription, initiation element (INR) and more upstream a TATA box, and (iii) may contain regulatory elements that determine the specific expression pattern of the promoter.<br>
The term "promoter" as used herein is taken in a broad context   and refers to regulatory nucleic acid sequences capable of effecting (driving and/or regulating) expression of the sequences to which they are operably linked. A "promoter" encompasses transcriptional regulatory sequences derived from a classical genomic gene. Usually a promoter comprises a TATA box, which is capable of directing the transcription initiation complex to the appropriate transcription initiation start site. However, some promoters do not have a TATA box (TATA -less promoters^ but are still fully functional for driving and/or regulat ing expression. A promoter may additionally comprise a CCAAT box sequence and additional regulatory elements (i.e. upstream activating sequences or cis-elements such as enhancers and silencers). A "promoter" may also Include the transcriptional regulatory sequences of a classical prokaryotic gene, in which case it may Include a   -35 box sequence and/or a -10 box transcriptional regulatory sequences.<br>
"Driving expression" as used herein means promoting the transcription of a nucleic acid.<br>
"Regulating expression" as used herein means influencing the level, lime or place of transcription of a nucleic acid. The promoters of the present Invention may thus be used to increase, decrease or change in time and/or place transcription of a nucleic acid. For example, t hey may be used to limit the transcription to certain cell types, tissues or organs, or during a certain period of time, or in response to certain environmental conditions.<br>
The promoter is preferably a plant-expressible promoter. The term "plant-expressible" means being capable of regulating expression in a plant, plant cell, plant tissue and/or plant organ. Accordingly, the invention encompasses an isolated nucleic acid as mentioned above, capable of regulating transcription of an operably linked nucleic acid in a plant or in one or more particular cells, tissues or organs of a plant.<br>
The expression pattern of the promoters according to the present invention were studied in detail<br>
and it was found that many of therh were tissue-specific. Accordingly, the present invention<br>
provides "tissue-specific" promoters. The term "tissue -specific" shall be taken to indicate that<br>
expression is predominantly in a particular tissue, tissue -type, organ or any other part of the<br>
organism, albeit not necessarily exclusively in said tissue, tissue-type, organ or other part.<br>
Accordingly, the invention encompasses an isolated nucleic acid as mentioned above, capable of<br>
driving and/or regulating expression (of an operably linked nucleic acid) in a tissue-specific<br>
manner. Expression may be driven and/or regulated in the seed, embryo, scutellum, aleurone,<br>
endosperm, leaves, flower, calli, meristem,  shoot meristem, discriminating centre, shoot, shoot<br>
meristem and root In grasses the shoot meristem is located in the so -called discrimination zone<br>
from where the shoot and the leaves originate.	<br>
A tissue-specific promoter is one example of a so-called "regulated promoter": These promoters are regulated by endogenous signals such as the presence of certain transcription factors, metabolites, plant hormones, or exogenous signals, such as ag eing, stresses or nutritional status. These regulations may have an effect on one or more different levels such spatial specificity or temporal specificity. Encompassed within the present invention is a nucleic acid as described hereinabbve, which is a "regulated promoter". Examples of regulated promoters are cell -specific promoters, tissue-specific promoters, organ-specific promoters, cell cycle-specific promoters, inducible promoters or young tissue-specific promoters.<br>
Alternatively and/or additionally, some promoters of the present invention display a con stitutive expression pattern. Accordingly, the present invention provides   a promoter as described hereinabove, which is a constitutive promoter. The term "constitutive" means having no or very _Je_wsBatjaJ_pitemporal regulations. The term "constitutive expression" as used herein refers to a substantially continuously expression   in substantially all tissues of the organism.   The skilled craftsman will understand that a "constitutive promoter" is a promote r that is active during most, but not necessarily all, phases of growth and development of the organism and throughout most, but not necessarily all, parts of an organism.<br>
The "expression pattern" of a promoter is not only influenced by the spatial and te mporal aspects, but also by the level of expression. The level of expression is determined by the so -called "strength" of a promoter. Depending on the resulting expression level, a distinction   is made herein between "weak" or "strong" promoters. Gen erally by "weak promoter" is meant a promoter that drives expression of an operably finked nucleic acid at levels of about 1/10 000 transcripts to about 1/100000 transcripts to about 1/500000 transcripts . Generally, by "strong promoter" is meant a promoter that drives expression at levels of about 1/10 transcripts, to about 1/100 or to about 1/1000 transcripts.<br>
According to a particular embodiment, the invention provides an Isolated promoter as mentioned hereinabove, which is a hybrid promoter. The term "hybrid promoter" as used herein refers to a chimeric promoter made, for example, synthetically, for example by genetic engineering .Preferred hybrid promoters according to the present invention comprise a part, preferably a functional part, of one of the promoters according to the present invention and at leas t another part, preferably a functional part of a promoter. The latter part, may be a part of any promoter, including any one of the promoters according to the present invention and other promoters. One example of a hybrid promoter comprises regulatory elements) of a promoter according to the present invention combined with the minimal promoter of another promoter. Another example of a hybrid promoter is a promoter comprising additional regulatory elements to farther enhance its activity and/or to alter its spatial and/or temporal expression pattern.<br>
The present invention also provides use of a functional fragment of any one of SEQ ID N01 to 22 or variant thereof for changing the expression pattern of a promoter. In such methods , at least part of any of the nucleic acids according to the present invention are combined with at least one fragment of another promoter.<br>
Further, the invention provides a genetic consfru ct comprising:<br>
(a)	An isolated promoter as defined hereinabove<br>
(b)	A heterologous nucleic acid sequence operably linked to isolated promoter of (a), and<br>
optionally-<br>
(c)	A 3' transcription terminator<br>
The term "genetic construct" as used herein m eans a nucleic acid made by genetic engineering.<br>
The terra "operably linked" to a promoter as used herein means that the transcription is driven<br>
and/or regulated by that promoter. A person skilled in the art will understand that being operably<br>
linked to a promoter preferably means that the promoter is-positioned upstream (i.e. at the 5'-end)<br>
of the operably linked nucleic acid. The 'distance to the operably linked nucleic acid may be<br>
variable, as long as the promoter of the present invention is capable of driving and/or regulating<br>
the transcription of the operably linked nudeic acid. For example, between the promoter and the<br>
operably linked nucleic acid, there might be a cloning site, an adaptor, a transcription or<br>
translation enhancer. '	.	•<br>
The operably linked nucleic acid may be any coding or non -coding nucleic acid. The operably linked nucleic acid may be in the sense or in the anti-sense direction. Typically in the case of genetic engineering of host cells, the operabty linked nucleic acid is to be introduced into the host cell and is intended to change the phenotype of the host cell. Alternatively, the operably linked nucleic acid is an endogenous nucleic acid from the host cell.<br>
The term "heterologous" as used herein is intended to be "heterologous to the promoter of the present invention'. A nucleic acid that is heterologous to the promoter of the present invention is not naturally occurring in the nucleic acid sequences flanking the;   promoter of the present invention when it is in its biological genomic environment. While the nucleic acid may be heterologous to the promoter of the present invention, it may be homologous or native or heterologous or foreign to the plant host cell, the heterologous operably linked nucleic acid may be any nucleic acid (for example encoding any protein), provided that it comprises or it is flanked by at least one nucleotide which is normally not flanking the promoter of the present invention.<br>
The term "transcription terminator" as used in (c) refers to a DNA sequence at the end of a transcriptional unit which signals termination of transcription. Terminators are 3' -non-translated DNA sequences usually containing a polyadenylation signal, which facilitates the   addition of polyadenylate sequences to the 3' -end of a primary transcript Terminators active  in and/or isolated from viruses, yeasts, moulds, bacteria, insects, birds, mammals and plants are known and have been described in literature. Examples of terminators suitable for use in the gene tic constructs of  the present invention include the Agrobacterium tumefaciens nopaline synthase (NOS) gene terminator, the Agrobacterium tumefeciens octopins synthase (OCS) gene terminator sequence, the Cauliflower mosa ic virus (CaMV) 35S gens terminator sequence, the Qryza sativa ADP-glucose pyrophosphorylase terminator sequence (t3'Bt2), the Zea mays zein gene terminator<br>
sequence, the rbcs-1A gene terminator, and the rbcs-3A gene terminator sequences, amongst others.<br>
The present invention also provides an expression cassette, a transformation vector or a plant expression vecto r comprising a genetic construct as described above.<br>
An "expression cassette" as meant herein refs rs to a minimal genetic construct necessary for expression of a nucleic acid. A typical expression cassette compris es a promoter-gene-terminator combination. An expression cassette may additionally comprise cloning sites, for example GatewayTM recombination sites or restriction enzyme recognition sites, to allow easy cloning of the operably linked nucleic acid or to allow the easy transfer of the expression cassette into a vector. An expression cassette may further comprise 5" untranslated regions , 3' untranslated regions, a selectable marker, transcription enhancers or translation enhancers.<br>
With "transformation vector" is meant a genetic construct, which may be introduced in an organism by transformation and may be stably maintained in said organism. Some vectors may be maintened in for example Escherichia coli, A. tumefaciens, Saccharomyces cerevisiae or Schizosaccharomyces pombe, while others such as phagemids and cosmid vectors, may be maintained in bacteria and/or viruses. Transformation vectors may be multiplied in their host cell and may be isolated again therefrom to be transformed into another host cell. Vector sequences generally comprise a set of unique sites recognized by restriction enzymes, the multiple cloning site (MCS), wherein one or more non-vector sequence(s) can be inserted. Vector sequences may further comprise an origin of replication which is required for maintenance and/or replication in a specific host cell. Examples of origins of replication include, but are not li mited to, the f1 -on and colE1.<br>
"Expression vectors" form a subset of transformation vectors, which, by virtue of comprising the appropriate regulatory sequences, enabl e expression of the inserted non-vector sequence(s). Expression vectors have been described which are suitable for expression hi bacteria (e.g. £ coff), fijno' (e-9- S. cerevisiae, S. pombe, Pichia pastoris), insect cells (e.g. baculoviral expression vectors), animal ceils (e.g. COS or CHO cells) and plant cells . One suitable expression vector according to the present invention is a plant expression vector, useful for the transformation of plant cells, the stable integration in the plant genome, the maintenance in the plant cell and the expression o f the non -vector sequences in the plant cell.<br>
Typically, a plant expression vector according to (he present invention  comprises a nucleic acid of any one of SEQ ID NO 1 to 22 6r a variant thereof as described hereinabove, optionally operably linked to a second nucleic acid. Typically, a plant expressible vector according to the present invention, further comprises T-DNA regions for stable integration into the plant genome (for example the left border and the right border regions of the Ti plasmid).<br>
The genetic constructs of the invention may further   comprise a "selectable marker". As used herein, the term "selectable marker" includes any gene, which confers a phenotype to a cell in which it is expressed, to facilitate the identification and/or selection of cells that are transfected or transformed. Suitable markers,may be selected from markers that confer antibiotic or herbicide resistance. Cells containing the   genetic construct will thus survive antibiotics or herbicide . concentrations that kill untransformed cells. Examples of selectable marker genes include   genes conferring resistance to antibiotics (such as nptll encoding neomycin phosphotransferase capable of phosphorylating neomycin and kanamycin, or h pt encoding hygromycin phosphotransferase capable of phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta; arpA or gox providing resistance against glyphpsate), or genes that provide a metabolic trait (such as manA that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example beta -glucuronidase, GUS), luminescence (such as luciferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives there of). Further examples of suitable selectable marker genes include the ampicillin resistance (Ampr), tetracyoline resistance gene (Tcr), bacterial kanamycin resistance gene (Kanr), phosphinothricin resistance gene, and the chloramphenicol acefyltransferase (CAT) gene, amongst others.<br>
Furthermore, the present invention encompasses a host cell comprising an isolated promoter, or a genetic construct, or an expression cassette, or a transformation vector or an expression vector according to the invention as described hereinabove. In particular embodiment s of the invention, the host cell is selected from bacteria, algae, fungi, yeast, plant s, insect or animal host cells.<br>
In one particular embodiment, tfre invention provides a transgenic plant cell comprising an isolated promoter according to the invention, or an isolated nucleic acid, or a genetic construct, or an expression cassette, or a transformation vector or an expression vector according to the invention as described hereinabove. Preferably said plant cell is a dicot plant cell or a monocot plant cell, more preferably a cell of any of the plants as mentioned herein.  Preferably, in the transgenic plant<br>
cell according to the invention, the promoter or the genetic construct of the invention is stably integrated into the genome of the plant cell.<br>
The invention also provides a method for the production of a transgenic plant, comprising:<br>
(a)	Introducing into a plant cell an isolated promoter, for example any one of<br>
SEQ ID NO 1 to SEQ ID NO 22, or a variant or fragment thereof, or a<br>
genetic construct, or an expression cassette, or a transformation vector or<br>
an expression vector according to the present invention and as described<br>
hereinabove, and<br>
(b)	Cultivating said plant cell under conditions promo ting plant growth.<br>
"Introducing" the above mentioned isolated promoter, or genetic construct, or expression cassette, or transformation vector or expression vector, into a host cell (e.g. plant cell) is preferably achieved by transformation. The term "transformation" as used herein encompasses the transfer of an exogenous pblynubleotide into a host cell, irrespective of the method used for transfer.    In particular for plants,  tissues capable of clonal propagation, whether by brganogenesis or embryogenesis, are suitable to be transformed with a genetic construct of the present invention and a whole plant may be regenerated therefrom. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited t o. the particular plant species being transformed/ Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem). The polynucleotide may be transiently or stably introduced into a   plant cell and may be maintained non-integrated, for example, as a plasmid. Alternatively, it may be integrated into the plant genome.<br>
Transformation of a plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce the nucleic acid s of the invention into a suitable ancestor cell. Transfbrmati on methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment^ transformaliorr using viruses- or pollen-and-miefeprejeciisn^ Methods may be selected from the calcium/polyethylene glycof method for protoplasts (Krens, FA   et a/., 1882, Nature 296, 72-74; Negrutiu I. et a/., June 1987, Plant Mol. Biol. 8, 363 -373); electroporation of protoplasts (Shillito R.D. et a/., 1985 Bio/Technol 3,1099-1102); microinjection into plant material (Crossway A. et a/., 1986, Mol. Gen Genst 202,  179-185); DNA or RNA-coated particle<br>
bombardment (Klein T.M. eta!., 1987, Nature 327, 70) infection with (non -infegrative) viruses and the like. A preferred transformation metho d for the production of transgenic plant cells according to the present invention, is an Agmbacterium mediated transformation method.<br>
Transgenic rice plants comprising any one of the promoters of the present invention  are preferably produced via Agrobacterium-medlated transformation using any of the well-known methods for rice transformation, such as the ones described in any of the following: published European patent application EP 1198985 At, Aldemita and Hodges (Planta, 199, 612 -617,1996); Chan etal. (Plant Mol. fiol. 22 (3) 49i -506,1993); Hiei et al. (Plant J. 6 (2) 271 -282,1994); which disclosures are incorporated by referenca herein as if fully set forth,   In the case of com transformation, the preferred method is as described in either Ishida  et al. (Nat Biotechnol. 1996 Jun; 14(6): 745 -50) or Frame etal. (Plant Physio! 2002 May; 129(1): 13 -22), which disclosures are incorporated by reference herein as if fully set forth.<br>
Generally after transformation, plant cells or cell groupings are se lected for the presence of one or more markers which are encoded by plant -expressible genes co -transferred with the gene of interest (which could be under the control of any of the promoters of the present invention), following which the transformed materi al may be cultivated under conditions promoting plant growth.<br>
The resulting transformed plant cell may then be used to regenerate a transformed plant in a manner known to persons skilled in the art. Accordingly, the method for the production of a transgenic plant as described herelnabove, may further comprise regenerating a plant from said plant cell of (a).<br>
The present invention further provides a plant comprising a plant cell as described  hereinabove. The plants may also be able to grow, or even reach maturity Including for example fruit production, seed formation, seed ripening and seed setting.<br>
Furthermore, progeny may be produced from these seeds, which   progeny may be fertile. Alternatively 67addIionaTly,"tfielransforrned ana* regenerated plants may also produce progerT&gt;rby~ non-sexual propagation such as cloning, grafting.   The generated transformed plants may bs propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed to give homozygous<br>
second generation (or T2) transformanls, and the T2 plants further propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms.   For example, they may be chimeras of transformed cells and non -transformed cells; clonal transformanfs (e.g., all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rootst ock grafted to an untransformed scion).<br>
Following DNA transfer and growth of the transformed cells, putatively transformed plant cells or plants may be evaluated, for instance using Southern analysis, for the presence of the gene of interest, copy number and/or genomic organization. Alternatively or additionally, expression levels or expression patterns of the, newly introduced DNA may be undertaken using northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the art.<br>
The present invention clearly extends to plants obtainable by any of the methods according to the present invention, which plants comprise any of the isolated promoters or the constructs of the present invention. The present invention clearly ex tends to any plant parts and propagules of such plant The present invention extends further to encompass the progeny of a primary transformed cell, tissue, organ or whole plant that has been produced by any of the aforementioned methods, the  only  requirement being  that progeny exhibit the same genotypic and/or phenofypic characteristic(s) as those produced in the parent by the methods according to the invention. The invention also extends to harvestable parts of a plant, such as but not limited to seeds, I eaves, fruits, flowers, stem cultures, stem, rhizomes, roots, tubers, bulbs and cotton   fibers.<br>
The term "plant" or "plants" as used herein encompasses whole plants, ancestors and progeny of plants and plant parts, including seeds, shoots, stems, roots (i ncluding tubers), and plant cells, tissues and organs. The term "plant" therefore also encompasses suspension cultures, embryos, meristematic regions, callus tissue, gametophytes, sporophyfes, pollen, and microspores. Plants that are particularly useful in the methods of the invention include all plants which belong to the superfamily Viridiplantae, in particular monocotyledonous and dicotyledonous plants including a fodder or forage legume, ornamental plant, food crop, free, or shrub selected from the list comprising Acscia spp;, Acerspp.,Actindiaspp,Aessultis spp.,Agathis-austisliSrAltsisaamara, Alsophila tricolor, Andropogon spp., Arachis spp, Areca catechu, Astelia fragrans, Astragalus doer, Ba/Yosea plurijuga, Betula spp., Brasslca spp., Brugufe ra gymnorrhiza, Burkea africana, Butea frondosa, Cadaba farinosa, Calliandra spp, Camellia sinensis, Canna indica, Capsicum<br>
spp.,   Cassia spp., Centro&amp;ma pubescens, Chaenomeles spp.,Cinnamomum cassia, Coffea arabica,  Colophospermum mopane, Coronillia varia,   Cotoneaster serotina, Crataegus spp., Cucumis spp., Cupressus spp.,  Cyaihea dealbata, Cydonia obtohga, Cryptomeria japonica, Cymbopogon spp., Cynthea dealbata, Cydonia oblonga, Dalbergia morietaria, Davallia divaricata, Desmodium spp., Dicksonia squamsa,  Diheteropogon amplectens, Dioclea spp, Dolichos spp., Dorycnium rectum, Echlnochloa pyramidalis, Ehrartia spp., Eleustne coracana, Eragrestis spp., Erythrina spp., Eucalyptus spp., Euclea schimperi, Eulalia villosa, Fagopyrum spp., Feijoa sellowiana, Fragariaspp., Flemingia spp, Freycinetia banksfy Geranium ihunbergii, Ginkgo biloba, Gtycine javanica, Gliricidia spp, Gossypium hirsutum, Grevillea spp., Guibourtia coleosperma, Hedysarum spp., Hematihia altissima, Heteropogon contortus, Hordeum vulgare, Hyp arrhsnia rufa, Hypericum erectum, Hyperthelia dissoluta. Indigo /incarnate, Iris spp., Leptarrhena pyrolffdlia, Lespediza spp., Lettuca spp., Leucaena leucocephala, Loudetia simplex, Lotonus bainesii, Lotus spp.,  Macrotyloma axillare, Malus spp., Manihot esculenta, Medicago sativa, Metasequoia glyptosfroboides, Musa sapientum, Nicotianurh spp., Onobrychisspp., Omithopus spp., Oryza spp., Peltophorum africanum, Pennisetum spp., Persea grafissima'. Petunia spp., Phaseotus spp.. Phoenix canariensis, Phormium co okianum, Phqtinia spp., Picea glauca, Pinus spp., Pisum sativum, Podocarpiis totara, Pogonarthria fleckii, Pogonarthria squarrosa, Populus spp., Prosopis cineraria,  Pseudotsuga menziesii, Pterolobium stellatum, Pyrus communls,  Quercus spp., Rhaphiolepsis urn bellata, Rhopalosfylis sapida, Rhus natalensis, Ribes grossiilaria, Ribes spp., Robinia pseudoacacia,   Rosa spp.,  Rubus spp.,  Safer spp.,   Schyzachyrium  sanguineum, Sciadopitys verticillata, Sequoia sempervirens, Sequoiadendrori giganteum, Sorghum bicolor, Spinacia spp., Sporobolus fimbriatus, Stiburus alopecuroides, Stylosanlhos humilis, Tadehagi spp, Taxodium distichum, Therrieda triandra, Trifolium spp., Triticum spp., Tsuga heterophylla, Vaccinium spp., Vicia spp.VitiS vinifera, Watsonia pyramidata, Zahtedesc hia aethiopica, Zea mays,  amaranth, artichoke, asparagus, broccoli,  brussel sprout,  cabbage, canola,  carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, rice, soybean, straw, sugarbeet, sugar cane, sunflower,  tomato, squash, and tea, trees and algae amongst others. According to a preferred feature of the present invention, the plant is a crop plant such as soybean, sunflower, canola, alfalfa, rapeseed, cotton, tomato, potato, tobacco, squash, papaya, poplar, leguminosa, flax, lupinus or sorghum. According to another preferred embodiment<br>
of the present invention the plant is a monocolyledonous plant, such as sugarcane, further preferable a cereal such as rice, maize, wheat, barley, millet, rye or oats.<br>
The invention further provides a method for driving and/or regulating expression of a nucleic acid in a plant or plant cej I, comprising:<br>
a)	Operably linking a nucleic acid to an isolated nucleic acid according to the invention as<br>
described herelnabove, such as to any one of SEQ ID NO 1 to 22 or a variant or<br>
fragment thereof, and<br>
b)	Introducing the resultant genetic construct into a plant or plant cell.<br>
Preferably the operably linked nucleic acid of (a) is heterotogbus to the nudeic acids according to the present invenlion.<br>
This method may further comprise cultivating the transformed plant or plant cell under conditions promoting growth, promoting regeneration and/or prompting maturation.<br>
Furthermore, the expression of the operab ly linked nucleic acid may be driven and/or regulated in particular cells, tissues or organs of a plant Accordingly, the invention p rovides a method as described above, wherein the expression is constitutive expression or tissue -specific expression. For these embodiments, reference is made to the example section where the specific expression patterns of the promoters according to the invention are described and where different types of tissue-specific expression are detailed, .<br>
The present invention further encompasses the u se of an isolated nucleic acid as defined hereinabove to drive and/or regulate expression of an operably I inked nucleic acid.<br>
The person skilled in the art will recognize that provision of sequences SEQ ID NO 1 to 22, readily makes available the tools to isolate related promoters, which may have substantial sequence identify to any of SEQ ID ID NO 1 to 22. Additionally, provision of sequences SEQ ID NO 23 to 44 (CDS corresponding to the promoters of the present invention, see Table 1) , readily makes available the tools to iso late related promoters, of which the related CDSs may have substantial sequence identity to any of SEQ ID NO 23 to 44.   Therefore the present invention also encompasses a method for isolating nucleic acids, capable of driving and/or regulating expression of an operablylinked nucleic acidrcornprisirtg-sereening-a-ntieleie-aeid-sequence-database to find homoiogues of any of the sequences represented by SEQ ID NO 1 to 22 or SEQ ID NO 2 3 to 44. Subsequently these homologues are used to screen a library with genomic DNA, which library is for example prepared from the organism of origin of the above mentioned homologue.   The screening procedure may for example involve hybridization. Subsequently, the genomic DNA that<br>
matches the homologue, fs analysed to identify the transcription initiation site and the translation initiation site of the gene corresponding to the homologue. Finally, specific primers are designed for amplification of a nucleic acid located in the region upstream (at the 5' end) of said translation initiation site.<br>
The present invention extends to the identification of regulatory proteins that are involved in the regulation of the activity of the promoters according to the'preseht invention. Such identification may be achieved using a yeast one-hybrid system. In such a yeast one-hybrid system the sequences according to any one of'SEQ ID NO 1 to 22 are operably linked to the GAL transcription activator and transformed to a yeast cell culture. That yeast cell culture is again transformed with a library of constructs encodi ng candidate regulatory factors.<br>
The present invention will now be described with reference to the following figures in which:<br>
Figure 1 shows a general schematic representation of a promoter. Regulatory elements are sequences that may for example be responsible for special and/or temporal regulation of the promoter activity. The minimal promoter is the minimal sequence necessary and suff icient to drive expression. It includes a TATA box, which is necessary to correctly direct the RNA pblymerase II to the transcription initiation site. The transcription initiation element (INR) in eludes the transcription initiation start site. The 5' untranslated region (5'UTR) is the region that is transcribed into pre-messenger RNA and eventually Into mRNA, but is not translated into protein. The translation initiation codon is represented by the startcodon ATG.<br>
Figure 2 is a map of the vector p4581 useful for expression in plants of a ß-glucuronidase (GUS) gene under control of any one of the promoters according to the invention. This binary vector comprises a Gateway recombination cassette , suitable for the recombination cloning of any of the promoters of the present invention in front of the Escherichia coli |3-glucuronldase (GUS) gene. This cassette contains a chloramphenicol resistance gene (CamR) and the ccdB suicide gene for counter selection of non -recombined plasmids, This GUS expression cassette further comprises the double terminator sequence T-zein and T-rbcS-deltaGA. This expression cassette is located within the left border (LB repeat, LB Ti C58) and the right border (RB repeat, RB Ti C58) of the nopaline Ti plasmid. Cloned within these borders are also selectable marlcer and a screenable marker genes each under control of a constitutive promoter and a terminator sequence.   This<br>
vector also contains an origin of replication (pBR322) for bacterial replication and a bacterial selectable marker (Spe/SmeR) for bacterial selection .<br>
The following figures show the results of the GUS staining of plants or pla nt parts transformed with the reporter vector p4581 carrying a promoter according to the present invention operably linked to the reporter gene GUS ..Plants denoted "C plants" are transgenic plants grown to about 5 cm; Plants denoted "B plants" are grown to about 10 cm; and plants denoted "A plants" are grown to maturity. These A plants were used to collect different tissue samples from old leaves, young leaves and seeds. ..<br>
Figure3 shows the expression pattern of PRO0110 (RCc3, SEQ ID NO 1). GUS staining is visible in roots.<br>
Figure 4 shows the expression pattern of PRO0005 (putative beta-amylase, SEQ ID NO 2). GUS staining is visible in seeds, more specifically in the embryo or in the scutellum of the embryo.<br>
Figure 5 shows the expression pattern of PRO0009 (putative cellulose synthetase, SEQ ID NO 3). GUS staining is visible In roots.<br>
Figure 6 shows the expression pattern of PRO0058 (proteinase inhibitor Rgpi9, SEQ ID NO 4). GUS staining is visible in the seeds.<br>
Figure 7 shows the expression pattem of PRO0061 (beta expansine EXPB9, SEQ ID NO 5). GUS staining is visible in young flowers of A pla nts (A) and in other young expanding tissues of B plants (B) and C plants (C).<br>
Figure 8 shows the expression pattern of PRO0063 (putative structural protein, SEQ ID NO 6). GUS staining is visible in young tissues, for example in the call! (A) or old leaves, young leaves and seeds of "A plants" (B).<br>
Figure   0   shows   the   expression   pattern   of   PRO0081—(putative eaffeoyi-CoA—3=O=L methyltransferase, SEQ ID NO 7). GUS staining is visible in young tissues, particularly of the shoot<br>
Figure 10 shows the expression pattern of PRO0091 ( prolamine RP5, SEQ ID NO 8). GUS staining is visible in seeds (A), particularly in the endosperm , and in meristem (B).<br>
Figure 11 shows the expression pattern of PR00095 (putative amino peptidase, SEQ ID NO 9). GUS staining is visible in seeds, more particularly in the embryo.<br>
Figure 12 shows the expression pattern of PR00111 (uclacyanin 3-like protein, SEQ ID NO 10). GUS staining is visible in roots and in meristem.<br>
Figure 13 shows the expression pattern of PR00116 (26S prbteasome regulatory particle non -ATPase subunit 11, SEQ ID NO 11). GUS staining is weakly visible in the whole plant (weak constitutive) and is particularly visible in meristem.<br>
Figure 14 shows the expression pattern of PRO0117 (putative 40S ribosomal protein, SEQ ID NO 12). GUS staining is visible in the seeds, more particularly in the endosperm.<br>
Figure 15 shows the expression pattern of PR00122 (chlorophyll a/b-binding protein presursor (Cab27), SEQ ID NO 13). GUS staining is visible in the shoot.<br>
Figure 16 shows the expression pattern of PRO0123 (putative: protochlorophyllide reductase, SEQ ID NO 14). GUS staining is visible in the shoot (above-ground tissues).<br>
Figure 17 shows the expression pattern of PRO0133 (chitinase Cht-3, SEQ ID NO 15). GUS staining is visible in the roots and meristem.<br>
Figure 18 shows the expression pattern of PRO01 51 (WSI18, SEQ ID NO 16). GUS staining is visible in the calli and upper plant parts (A) as well as in the aleurone layer and embryo (B).<br>
Figure 19 shows the expression pattern of PRO0169 ( aquaporine, SEQ ID NO 17). GUS staining is visible in the whole plant (constitutive expression).<br>
Figure 20 shows the expression pattern of PR00170 (High mobility group protein, SEQ ID NO 18). GUS staining is strongly visible In the whole plant as is illustrated by the "B plants" (A), and various tissues such as old leaves, young leaves and seeds   (B) and calli (C) (constitutive expression).<br>
Figure 21 shows the expression pattern of PRO0171  (reversibly glycosylated pro tein RGP1, SEQ ID NO 19). GUS staining is visible in all plant parts (constitutive expression).<br>
Figure 22 shows the expression pattern of PRO0173   (cytosolic MDH, SEQ ID NO 20). GUS staining is visible in all p lant parts and particularly in the shoot (above-ground tissues) and seeds.<br>
Figure 23 shows the expression pattern of PRO0175 ( RAB21, SEQ ID NO 21). GUS staining is weakly visible in calli (A), meristems and young leaves, and is strongly visible in developing and maturing seeds (B) more particularly in the embryo.<br>
Figure 24 shows the expression pattern of PRO0177 (Cdc2-1, SEQ ID NO 22). GUS staining is weakly visible in meristem and in leaf sheets.<br>
Examples<br>
The promoters according to the present invention were isolated as DNA regions spanning about 1.2 kb of the sequence upstream of the translation initiation codon (i.e. first ATG,   which codon was excluded) from various rice genes. For determinate- n of their nucleic acid sequence and their expression pattern, the following procedure was followed:   First in silica studies on genomic rice sequences were performed. However, procedures based on automated prediction programs to locate promoter-like nucleic acid sequence are highly error prone, even for the localization the best-characterized promoter control elements such as the TATA box and the transcription initiation element (INR), Also, in silica determination of expression pattern   is extremely speculative. Therefore, to obtain unambiguous data about the; nucleic acid sequence and the expression pattern of the promoter s, in vivo studies were performed encompassing (i) isolation of the promoter nucleic acid sequence ; (ii) operably linking a reporter gene to the promoter and introducing the resulting genetic construct into a host organisms ; (iii) growing the transformed host cell under conditions allowing expression of the reporter gene, and (iv) determination of the reporter gene activity in the differen t tissues of the host organism. These methods are now described in more detail;<br>
Example "1.  Identification and Isolation of the promoters<br>
Identification of rice ESTs, 819 corresponding genss and their location in the rice genome<br>
Sequence databases, comprising rice sequences, were searched for rice expressed sequence lags (ESTs). Subsequently an "in silica" Northern-blot was performed to allow identification of EST families that are strongly expressed or that are specific for a particular organ. This analysis included normalization of the numbers of ESTs isolated from different plant organs.   The ESTs families with ah interesting distribution among source cDNA libraries  were selected for further analysts and sequence homology searches. After sequence homology searches in combination with scanning scientific data, the genes that correspond to those families of EST s were ide ntified from sequence databases and a (putative) function and corresponding gene name was given (see Table 1). Subsequently , the- corresponding promoter region was isolated by the following procedure. In 3 first step the TIGR database was searched to find a tentative contig corresponding to an EST family. Sequence homology was found using standard computer programs , such as Blast N using standard parameters (typically G Cost to open a gap =5, E Cost to extend a gap = 2, q Penalty for a mismatch in the blast portion of run = -3, r Reward for a match in the blast portion of run   = 1, e Expectation value = 10.0, W Word size =   11, v Number of one-line descriptions = 100. b Number of alignments to show = 100, Matrix = BLOSUM62). The TIGR database (The Institute for Genomic Research), provides   Tentative Contigs (TC) which are sequence predictions based on contig building from all known EST,   from all known cDNA and from reconstructed mRNA. The  TCs used for identification of the promoters of the present invention are represented in Table 1. In a second step these TCs were used to  locate the corresponding gene on a genomic sequence, which gene comprises the coding region as well as the promoter region.  Generally, these genomic sequences were BAC clones, which   are represented herein by their Genbank accession number (see Table 1). From these BAC clones the sequence identity of the promoter region could be determined.<br>
Table 1: list of rice promoters of the present invention. The promoter sequences are represented herein by their SEQ ID NO and promoter number (PRO). The coding sequences (CDS) naturally driven by a promoter of the present invention are represented by their name, by SEQ ID NO and by Tentative contig (TC) accession number of the TIGR database. The Genomic sequences (BAC clones or genes) comprising a promoter region of the present invention are represented by their GenbanK accession nurnber.<br>
(Table Removed)<br>
Identification and isolation of the promoter regions of rice genes<br>
Starting from the sequence information of (he gene s and their location in the rice genome, the promoter regions of these genes were isolated as the DNA region spanning about 1.2 kb upstream of the translation initiation codon (Le. first ATG), which codon was excluded. When an JnteTvening sequsricesuch as an intron, was present in the-5 urftranslated-regior^of-the-gene^the-. isolated DNA region was taken as the region spanning about 1.2 kb plus the length of that intervening sequence. The promoter regions were isolated from genomic DNA of Oryza sativa Japonica or exceptionally from Oryza sativa fndica via PCR using specific primers. These specific primers comprise AttB recombination sites, suitable for recombination cloning of the isolated<br>
promoter region These specific primers are herein represented as SEQ ID NO 45 to 88 and are listed in Table 2. Conditions for PCR were as follows: 1 cycle of 2 min at 94°C, 35 cycles of 1 min at 94°C, 1 min at 58°C and 2 min at 68°C, and 1 cycle of 5 min at 68°C. The length of the expected PCR fragment is also indicated in Table 2. The corresponding PCR fragment was purified from the PCR reaction mix via gele electrophoresis and subsequent purification using Zymoclean Gel DNA Recovery Kit (Zymo Research, Orange, California).<br>
Table 2: Overview of the primers used to isolate the rice promoters of the present invention and the length of the rice promoter regions.<br><br>
(Table Removed)<br>
Example 2.   Cloning of promoter-GUS reporter vectors for plant transformation<br>
The purified PCR fragments of Example 1, corresppnding to the promoter regions of the present invention, were cloned into the pDOMR201 entry plasmid of the GatewayTM system (Life Technologies) using the "BP recombination reaction". T he identity and base pair composition of the cloned insert was confirmed by sequencing and additionally, the resulting plasmid was tested via restriction digests.<br>
In order to clone each of the promoters of the present invention in front of a reporter gene, each entry clone of Example 1 was subsequently used in an "LR recombination reaction" (GatewayTM) with the destination vector p4581. This destination vector was designed to operably link each promoter of the present invention to the Escherichia coli beta-glucuronidase (GUS) gene via the substitution of the Gateway recombination cassette in front of the GUS gene.  Furthermore this destination vector is suitable for transformation of plants and comprises within the T -DMA left and right borders the resulting promoter-GUS cassette and selectable marker and screenable marker cassettes (see Rgure 2). The resulting reporter vectors, comprising a promoter of the present invention operably linked to GUS,   are subsequently transformed into Agrobacterium strain LBA4044 and subsequently into rice plants using standard transformation techniques .<br>
Example 3.   Expression pattern s of the promoter-GUS reporter cassette in plants Growth and harvest of transgenic plants or plant pa rts at various stages (C plants, B plants and A plants)<br>
For each promoter-GUS reporter construct, 3 TO transgenic rice plants were generated from transformed cells. Plant growth was performed under normal conditions. The first transgenic plant was sacrificed for GUS staining when it had reached a size of about 5 cm, which plant is named herein "C plant". The second transgenic plant was sacrificed for GUS staining when it had reached a size of about 10cm, which plant is named herein "B plant". The third transgenic plant was kept for seed production and is named herein "A plant". GUS staining was performed on complete C and B plants. On A plants, GUS staining was performed on leaf pieces, flowers and section of seeds at various developmental stages. A plants were allowed to set seed, which seeds were used after harvest for confirmation of the expression pattern in T1 plants.<br>
GUS staining<br>
The sacrificed plants or plant parts were covered with 90% ice-cold acetone and incubated for 30 min at 4 °C. After 3 washes of 5 min with Tris buffer [15,76 g Trizma HCI (Sigma T3253) + 2,922 g<br>
NaCI in 11 bidi, adjusted to pH 7,0 with NaOH], the material was covered by a Tris/ferricyanate/X -Glue solution [9,8 ml Tris buffer + 0,2 ml ferricyanate stock (0,33 g Potassium ferricyanate (Sigma P3667) in 10 ml Tris buffer)+ 0,2 ml X -Glue stock (26,i mg X-Gluc (Europa Bioproducts ML 113A) in 500 ul DMSO)]. Vacuum Infiltration was applied for 15 to 30 minutes. The   plants or plant parts were incubated for up to 16 hours at 37 °C until development of blue   colour was visible. The samples were washed 3 times for 5 minutes with Tris buffer. Chlorophyll was extracted in ethanol series of 50 %, 70% and 90% (each for 30 minutes).<br>
Expression patterns of the promoters of ths present Invention<br>
The expression patterns of trie rice promoters of the present invention are summarized in Table 3.<br>
Table 3: expression patterns of the rice promoters of the present invention<br><br>
(Table Removed)<br>
The following paragraphs describe the observed expression patterns of the promoters of the present invention in more detail. The observations are based on the visual inspection of the GUS stained tissues as described above. It is to be understood that for some promoters expression may be weak and that expression in certain tissues may only be visible with very sensitive detection methods.<br>
PRO0110-SEQ ID NO1-RCc3<br>
1 construct (OS1432), which is a reporter vector as described in Example 2 comprising PR00110<br>
was investigated. 25 calli, 14 C, 21 B plants and 21 A plants were analysed. There was no<br>
expression visible in calli, but strong expression in roots of C plants  (93%) and of B plants (81%)<br>
was observed. No expression in the shoots of A plants was observed. Therefore the RCc3<br>
promoter PRO0110 is suitable for strong expression in root s.<br>
PRO0005 - SEQ ID NO 2 - putative beta-amyfase<br>
1 construct (OS1365) was investigated. 28 calli, 24 B plants and 22 A plants were analysed.<br>
Occasional expression in calli (7%) was observed as well as occasional weak expression in root s<br>
(4%) and shoots (12%) of B plants, expression in the scutellum of embryos of A plants (43%) and<br>
occasional expression in leaves (5%) of A plants. This promoter is therefore   suitable for<br>
expression in embryo, more preferably in the scutellum of the embryo. This region of the embryo<br>
is also referred to as th e transfer layer of the embryo. This promoter may have some leakiness in<br>
other tissues.	,<br>
PRO0009 - SEQ ID NO 3- putative cellulose synthase<br>
1 construct (OS1461) was investigated. 20 calli, 20 C, 20 B plants and 20 A plants were analysed.<br>
Occasional expression in calli (20 %) was observed as well as weak expression in roots (55%) of<br>
C plants, occasional expression in young leaves (10%) of C plants and weak expression in the<br>
roots (25%) of B plants. No expression in leaves of A or B plants was observed. Therefore this<br>
promoter is suitable for expression in  roots. This promoter may show some leakiness in the<br>
leaves.<br>
PRO0058 - SEQ ID NO 4-protelnase inhibitor Rgpi9<br>
1 construct (OS1370) was investigated. 13 B plants and 12 A plants were analysed. No<br>
expression was observed in B plants. In A plants, no expression was observed in the leaves, but<br>
there was strong expression tn endosperm antf e~mbryo (58^42%). ThereforertrnVprortTOter<br>
PRO0058 is suitable for expression in seeds.<br>
PRO0061 -SEQ ID NO 5- beta expansine EXPB9<br>
2 constructs (OS1441 and OS1460) were investigated. 20 calli, 32 C, 32 B plants and 32 A plants<br>
were analysed. Weak expression was observed in the leaves of C and B plants. In A plants<br>
expression in the flowers was observed (44%), more particularly in lemma of young spikelets. It was concluded that the promoter PRO0061 is suitable for expression in young tissue, more preferably in young, developing or expanding tissue, more preferably in green tissue. PRO0063 -SEQID NO 6-putative structural protein<br>
1 construct (OS1446) was investigated. 13 calli, 13 C, 13 B plants and 12 A plants were analysed. In calli, weak expression was detected (92%). In C plants, there was no expression in roots and there was weak expression in some leaves (46%). In B plants, there was no expression in roots and weak expression in young tillers (78%) o r young leaves (54%), but no expression in old leaves. In A plants, there was occasional expression in young leaves (17%) and expression in embryo and scutellum (42%). Therefore it was  concluded that this promoter Is active in the above-ground tissues, such as leaf, stem and seed. These data demonstrate that the promoter is suitable for expression in calli and in the shoot, and for expression in young tissues and seeds. PRO0081 -SEQID NO 7-putative caffeoyl-CoA 3-O-methyltransferase 1 construct (OS1419) was investigated. 20 calli, 20 C, 20 B plants and 20 A plants were analysed. No expression was observed in Calli. Expression was observed in C plants,' more particularly weak expression in root cylinder (40%) an d weak expression in young leaves (80%) and in old leaves. Expression was also observed in B plants, more particularly   weak expression in roots (25%) and weak expression in young leaves (80%).   Expression was also observed in young leaves (50%) of A plants. It was concluded that promoter PRO0081 is suitable for expression in above-ground tissue s, preferably in the shoot. This promoter may have some leakage of expression in roots.<br>
PRO0091 - SEQ ID NO 8- prolamine RP5<br>
1 construct (OS1558) was investigated. 12 C, 12 B plants and 12 A plants were analysed. Weak expression was observed in the discrimination centre (50%) of C plants and in the discrimination centre (58%) of B plants. Strong expression was observed in endosperm (55%) of A plants. This promoter was found to be useful for strong expres sion In the  endosperm, with leakiness in meristem, preferably the shoot meristem or discrimination centre. PRO0095 • SEQ ID NO 9- putative meth ionine aminopeptidase<br>
1 construct (OS1423) was investigated. 16 calli, 14C, 14 B plants and 16 A plants were analysed. Some expression was observed in root-tips (36 %) of C plants and in the embryo (38%) of A plants, but not in endosperm of A plants. It was concluded that PRO0095 is suitable for expression in embryo.<br>
PRQQ111 - SEQ ID MO 10- uclacyanin S-Iika protein<br>
1 construct (OS1421) was investigated. 22 calli, 21 C, 22 B pla nts and 21 A plants were analysed. Weak expression was observed in the discrimination centre and meristems (77%) of B plants. It<br>
was concluded that promoter PROQ111 is suitable for weak expression in the meristern, preferably<br>
in shoot meristern or discrimination centre.	,<br>
PRO0116 - SEQ ID NO 11-26S profsasome regulatory particle non -ATPase subunlt 11<br>
1 construct (OS1679) was investigated. 13 C, 14 B plants and A plants were analysed.  Weak<br>
expression was observed in meristem/discrimination centre of C plants (38%) and of B plants<br>
(71%) and in young leaf sheaths of C plants (77%) and of B plants (21%). It was concluded that<br>
promoter PRO0116 is suitable for expression in meristern, preferably in shoot meristern or<br>
discrimination centre.<br>
PRO0117 - SEQ ID NO 12- putativa40S ribosomal protein<br>
1 construct (OS1425) was investigated. 9 qalli, 9 C, 9 B plants and 9 A plants were analysed.<br>
Occasional weak expression was observed in roots (22%) and in young lea f blade s (44%) of C<br>
plants. Expression was mainly, observed in endosperm (37%) of A plants. Therefore, promoter<br>
PRO117 was found to be suitable for expression in endosperm and may have some leakiness in<br>
young leaves.<br>
PRO0122 - SEQ ID NO 13- chlorophyll afb-binding protein presursor (Cab27)<br>
1 construct (OS1675) was investigated. 38 calli, 38 C, 38 B plants and 15 A plants were analysed.<br>
Very weak expression was observed in the discrimination centre and young leaf sheaths of C<br>
plants. It was concluded that this promoter PRO0122 is suitable for weak expression in shoots.<br>
PRO0123 - SEQ ID NO 14-putative protochlorophyltide reductase<br>
1 construct (OS1433) was investigated, 21 calli, 18 C, 19 B plants and 18 Aplants were analysed.<br>
Strong expression was observed in shoots (33-68%) of C plants and B plants (63-79%). In B<br>
plants there was also occasional expression in roots. In A plants, again strong expression in<br>
young leaves (73%) was observed, as well as occasional expression in old leaves (39%). It was<br>
concluded that this promoter is suitable far. strong expression in shoots, preferably hi leaves.<br>
PRO0133 - SEQ ID NO 15- chltlnase Cht-3<br>
1 construct (OS1687) was investigated. 15 calli, 12 C, 16 B plants and 12 A plants were analysed.<br>
Weak expression was observed in calli (66%) and in the discrimination centre/meristem (50%) of<br>
B plants. It was concluded that promoter PR00133 is suitable for weak expression in meristern,<br>
preferably in shoot meristern or discrimination centre.<br>
PRO0151 - SEQ ID NO 13- WSI18<br>
1 construct (OS1458) was investigated. 22 cW,T6T^TffB"pIanissrfior13 A plants^verewalyseo*:—<br>
Strong expression was observed in calli (91%) and weak expression in shoot s of C plants (62%).<br>
In A plants there was very strong expression in tha aleurone layer and in the embryo (46%). It was<br>
concluded that promoter PR00151  is suitable for strong expression in call! and in seeds, more<br>
particularly in the aleurone layer and in ihe embryo of the seeds.<br>
PR00169 - SEQ ID NO 17- aquaporlne<br>
1 construct (OS1911) was investigated . 11 calli, 10 C plants, B plants and A plants were analysed.<br>
Some expression (55%) was observed in calli and in roots (30% ) of C plants. Furthermore, g ood<br>
expression was observed in shoot tissues (80%) of C plants and in young leaves of B plants.  |t<br>
was concluded that this promoter is suitable for constituti  ve expression, preferably constitutive in<br>
young plants.<br>
PR0170 - SEQ ID NO 18- High mobility group protein<br>
1 construct (OS1434) was investigated. 23 calli, 21 C, 21 B plants and 14 A plants were analysed.<br>
Expression was observed in calli (52%) and in roots (51%) of C plants. Moreover, strong<br>
expression was observed in young leaves (81%) of C plants, in roots (86%) of B plants and in<br>
young leaves (86%) of B plants. In A plants there was strong expression in young  leaves (75%),<br>
old jeaves (43%), embryo and aleurone but a weaker expression in endosperm (82%).  It was<br>
concluded that promoter PRO170 is suitable for strong constitutive expression.<br>
PROQ171 -SEQIDNO 19-reversiblyglycosylated protein RGP1<br>
1 construct (OS1762) was investigated. 18 calli, 11 C and 13 B plants were analysed. Strong<br>
expression was observed in calli (44%) and in all tissues (27%) of  C plants. In all tissues of B<br>
plants (16%), expression was somewhat weaker but most pronounced the in discrimination<br>
centres (46%). It was concluded that promoter PRO0171 is suitable for constitutive expression.<br>
PRO0173-SEQbNO20-cytosolicMDH<br>
1 construct (OS1435J was investigated. 17 calli, 17 C, 17 B plants and 15 A plants were analysed.<br>
Occasional expression (12%) was observed in calli and weak expression was observed in upper<br>
parts (24-69%) of C plants as well as in young leaves (41%) of B plants. In A plants, expression in<br>
(eaves (33%) was observed and strong expression in seeds (38%), but not in the root. It was<br>
concluded that the promoter PRO 0173 is suitable for expression in   above-ground tissues<br>
especially for constitutive expression in the shoot and especially in the seeds.<br>
PRO0175 - SEQ ID NO 21- RAB21<br>
1 construct (OS1436) was investigated. 16 calli, 12 C, 15 B plants and 15 A plants were analysed.<br>
Expression was observed in some calli (31%), in the discrimination centres (42%) of C plants and<br>
in young leaves (25-58%) of C plants and A plants (15 %). Furthermore, very strong expression<br>
wa&amp;observed in aleuronearjd_ernbryc^(60%) of a plant. it was concluded that promoter PRO0175<br>
is suitable for strong expression in calli and in seeds, more particularly in developing/maturing<br>
seeds, more particularly in the aleurone layer and in the embryo of the seeds.<br>
PRO0177 - SEQ ID NO 22- Cdc2-1<br>
1 construct (OS1436) was investigated .16calli, 12 C, 15 B plants and 15 A plants were.analysed. Expression was observed in some of the calli (31%), in the discrimination centre (42%) of C plants, in young leaves (25-58%) of C plants and occasionally in young leaves (15 %) of A plants. Moreover, very strong expression was observed in aleurone and embryo (60%) of seeds from A plants. It was concluded that this promoter is suitable for specific expression in seeds, more particularly in developing/maturing seeds.<br>
Example 4.   Stability of the expression patterns of the promoters of the present invention in Further generations<br>
The above-mentioned analyses were performed on TO plants originating from the transformed tissues. The stability of promoter activity in the next generations or progeny plants of the original TO plant, the so-called T1 and T2 plants, was evaluated as follows.The TO plant transformed with the reporter constructs as mentioned in the above paragraphs of Example 2, were grown until maturity (A plants), of which the seeds (T1 seeds) were harvested and sown to generate progeny T1 plants. These plants were analysed as described above in Example 3 and the A T1 plants were allowed to reach maturity and to set T2 seeds.<br>
The expression pattern of the promoters of the present invention was studied in TO plants, T1 seeds, T1 plants and T2 seeds and in all the tissues (including seeds and seed tissues ) as described ,in Example 3. The specific expression pattern s as reported from the TO and T1 seeds and described in Example 3 were confirmed in the following T1 generation and T2 seeds . It is concluded that the expression pattern of the promoter s of the present are stably inherited in plants of subsequent generations.<br>
Example 5.  Stability of expression patterns  of the promoters of the present invention In other pla nts<br>
The above-mentioned plant analyses were performed on rice plants. This choice was based on the practical consideration that plant genetic engineering Is most profitable for crop plants.   Also in other crppjlants, such as_for example Zea Mays,  the reporter constructs comprising the promoters according to the present i nvention are introduced and transformed plant are evaluated as described hereinabovs. The expression patterns of the promoters according to the present invention are conserved among plants.   Therefore, the promoters according to the present<br>
invention are also suitable for driving and/or regulating expression of an operably linked nucleic acid in monocots, such as com.<br>
For many other purposes such as research and horticulture, (small) herbs are  being genetically modified, which involves the use of promoters. Therefore the reporter constructs comprising the promoters according to the present invention are introduced into other plants species such as for example Arabidopsis thaliana' and transformed plants are evaluated as described herein above. The expression patterns of the promoters according to the present invention are conserved among plants. Therefore, the promoters according to the present invention are also suitable for driving and/or regulating expression of a h operably linked nucleic acid in other plant species such as for example dicots, such as Arabidopsis.<br><br><br><br><br>
1 . An isolated 13r017iotcr capable ol'driving andlor regulating expression in gsccn tissue ofti<br>
plant. comprising:<br>
( a ) An isolated n~lclcic acid as givcn in any one of' SI1Q Ill YO 5 or 14 or thc<br>
complcmcn~o l'any one t,l.SI-Q 11) YO5 or 1.1: or<br>
(b) An isolatcd nnclcic having at least OO'XI scq~~clicidec ntil'ying \virh any ol' tlic<br>
IIYA scclucnccs as givcn in any one of'tlic SliQ I I ) NO 5 or 14: or<br>
(e) A11 isolatcd nuclcic acid specifically liybridizi~igu nder stringent conditiolis s~lcll<br>
as hcrcin dcscribcd with any of the 1)SA scclucnccs as give11 in any one 01' SI.:C)<br>
I l l \O 5 or 14: or<br>
(d) An isolatcd nuclcic acid as dclincd in any one ol'(u) lo (c). \\hiell is i~~tel-1-uprcb! d<br>
an intervening scqucncc sucli as hcrcin described: or<br>
(e) A fi-agment of at lcast 200 bases comprising any ol'tlic nuclcic acids as dclincci in<br>
(a) to (d). \\lliicli fi-agmcnt is capable of driving andlor rcg~~latinegx pression.<br>
2. A pso~iiotu' ;I\ claimed in claim I . \z hicli is a hybrid promoter comprising :it Ic:~st onc<br>
part ol'a pronlotcr ,is clain~cd in claim 1 and colnprisitlg another past ol'a p r o l ~ i o ~ c ~<br>
7<br>
.) . A genetic conslruct comprising:<br>
(a) An isolatcd promotes as clailiicd in claitii I; and<br>
(b) A hctcrologous nuclcic acid scclucncc s~lcha s licrcin dcscribcd opcrablq linked to<br>
said promoter oI'(a): and optionally.<br>
(c) A 3' transcription tcrinina~ors uch as herein dcscribcd.<br>
z 1 An cxprcssion cassette comprising a gcne~icc onslruct as claimed in claim 3<br>
5. A transfbsmatio~iv cctol- comprising a genetic construcl as claimed in claim 3.<br>
0. An cxpscssion vcc~orc omprising a gcnctic construct as claimed in claim -3.<br>
7. A host ccll comprising an isolatcd ~?ronlotcar s claimcd in claim I , or gcnctic construct as<br>
clai~ncd in claim 3, or an csprcssion cassette as claimed in claim 4. or a rrans1i)rmation<br>
vcctor as claitncd in claim 5. or an csprcssion vcctor as claitncd in claim 0. \\~lic~-csitali d<br>
host cell is a bacrcrial or yeas1 ccll.<br>
8. Method l'or dsiving andlor regulating cxprcssion of' a ti~lclcic acid it1 grccn tissue of' :I<br>
plant. comprising:<br>
(a) opcrably linking said nuclcic acid to any onc of' ~ h cis olated n~~clciacc id5 as<br>
claimcd in claim I . and<br>
(b) introducing the resultant gcnctic consiruct into a plant or platit ccll.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQWJzdHJhY3QtKDMxLTA3LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Abstract-(31-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">564-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ2xhaW1zLSgwNS0wMi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Claims-(05-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LURFTE5QLTIwMDgtQ2xhaW1zLSgzMS0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-DELNP-2008-Claims-(31-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgwNS0wMi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence Others-(05-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNC0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence Others-(14-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNC0xMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence Others-(14-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNy0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence Others-(17-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyNy0xMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence Others-(27-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgzMS0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence Others-(31-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgxNC0wOC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Correspondence-Others-(14-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">564-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtRm9ybS0xLSgxNC0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Form-1-(14-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtRm9ybS0xMy0oMTQtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Form-13-(14-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtRm9ybS0xMy0oMjktMDQtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Form-13-(29-04-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtRm9ybS0xOCAoMTgtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Form-18 (18-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtRm9ybS0zLSgxNC0xMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Form-3-(14-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtRm9ybS0zLSgzMS0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Form-3-(31-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtR1BBLSgxNC0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-GPA-(14-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtR1BBLSgxNy0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-GPA-(17-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtR1BBLSgzMS0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-GPA-(31-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">564-delnp-2008-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTY0LWRlbG5wLTIwMDgtUGV0aXRpb24tMTM3LSgzMS0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">564-delnp-2008-Petition-137-(31-07-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259258-a-method-of-tunnel-end-point-creation-using-simple-network-management-protocol.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259260-an-inkjet-printing-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259259</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>564/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>07-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Jan-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROP DESIGN N. V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3, B-9052 ZWIJNAARDE, BELGIUM.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YVES HATZFELD</td>
											<td>18C RUE DES DONDAINES, F-59000 LILLE, FRANCE.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WILLEM BROEKAERT</td>
											<td>KLUIZENBISSTRAAT 26, B-1700 DILBOOK, BELGIUM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>NA</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2004/050081</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>03075331.3</td>
									<td>2003-02-04</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259259-an-isolated-promoter-capable-of-driving-and-or-regulating-expression-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:40:51 GMT -->
</html>
