<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259243-a-method-for-producing-non-2-m-family-plasmid-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:44:33 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259243:&quot;A METHOD FOR PRODUCING NON-2 µM-FAMILY PLASMID PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR PRODUCING NON-2 µM-FAMILY PLASMID PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for producing non-2µm-family plasmid protein comprising: (a) providing a host cell of the kind such as herein described comprising a 2µm-family plasmid, the plasmid comprising a gene encoding protein comprising the sequence of a chaperone protein and a gene encoding a non-2µm-family plasmid protein; (b) culturing the host cell in a culture medium under conditions that allow the expression of the gene encoding protein comprising the sequence of the chaperone protein and the gene encoding a non-2um-family plasmid protein; and (c) purifying the thus expressed non-2-µm-family plasmid protein from the cultured host cell or the culture medium;</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>GENE EXPRESSION TECHNIQUE FIELD OF THE DETENTION<br>
The present application relates to gene expression techniques. BACKGROUND OF THE INVENTION<br>
The class of proteins known as chaperones have been defined by Haiti (1996, Nature, 381, 571-580) as a protein that binds to and stabilises an otherwise unstable conformer of another protein and. by controlled binding and release, facilitates its correct fate in vivo, be it folding, oligomeric assembly, transport to a particular subcellular compartment, or disposal by degradation.<br>
Bi? (also known as GRP78, Ig heavy chain binding protein and Kar2p in yeast) is an abundant ~7QkDa chapeione of the hsp 70 family, resident in the endoplasmic reticulum (ER), which amongst other functions, serves to assist in transport hi the secretory system and fold proteins.<br>
Protein disuiphide isomerase (PDI) is a chaperone protein, resident in the ER that is involved in the catalysis of disulphide bond formation during the post-translational processing of proteins.<br>
Studies of the secretion of both native and foreign proteins have shown that transit from the ER to the Golgi is the rate-limiting step. Evidence points to a transient association of the BiP with normal proteins and a more stable interaction with mutant or misfolded forms of a protein. As a result, BiP may play a dual role in solubilising folding precursors and preventing the transport of unfolded and unassembled proteins. Robinson and Wittrup, 1995, Biotechnol Prog. 11, 173-177, have examined the effect of foreign protein secretion on BiP (Kar2p) and<br>
PDI protein levels in Saccharomyces cerevisiae and found that prolonged<br>
constitutive expression of foreign secreted proteins reduces soluble BiP and PDI to levels undetectable by Western analysis. The lowering of ER chaperone and foldase levels as a consequence of heterologous protein secretion has important implications for attempts to improve yeast expression/secretion systems.<br>
Expression of chaperones is regulated by a number of mechanisms., including the unfolded protein response (UPR).<br>
Using recombinant techniques, multiple PDI gene copies has been shown to increase PDI protein levels in a host cell (Farquhar et al, 1991, Gene, 108, 81-89).<br>
Co-expression of the gene encoding PDI and a gene encoding a heterologous disulpnide-bonded protein was first suggested in WO 93/25676, published on 23 December 1993, as a means of increasing the production of the heterologous protein. WO 93/25676 reports that the recombinant expression of antistasin and tick anticoagulant protein can be increased by co-expression with PDI.<br>
This strategy has been exploited to increase the recombinant expression of other types of protein.<br>
Robinson et al, 1994, Bio/Technology, 12, 381-384 reported mat a recombinant additional PDI gene copy in Saccharomyces cerevisiae could be used to increase the recombinant expression of human platelet derived growth factor (PDGF) B homodimer by ten-fold and Schizosacharomyces pombe acid phosphatase by fourfold.<br>
Hayano et al, 1995, FEES Letters, 377, 505-511 described the co-expression of human lysozyme and PDI in yeast. Increases of around 30-60% in functional lysozyme production and secretion were observed.<br>
Shusta e? aL 1998, Nature Biotechnology-', 16, 773-777 reported that the recombinanl expression of single-chain antibody fragments (scFv) in Saccharomyces cerevisiae could be increased by between 2-8 fold by over-expressing PDI in the host cell.<br>
Bao &amp; Fulcuhara, 2001, Gene, 212, 103-110 reported that the expression and secretion of recombinant human serum albumin (rHSA) in the yeast Kluweromyces lactis could be increased by 15-fold or more by co-expression with an additional recombinant copy of the yeast PDI gene (KIPDI1}.<br>
In order to produce co-transformed yeast comprising both a PDI gene and a gene for a heterologous protein, WO 93/25676 taught that the two genes could be chromosomally integrated; one could be chromosomally integrated and one present on a plasmid; each gene could be introduced on a different plasmid; or both genes could be introduced on the same plasmid.  WO 93/25676 exemplified expression of antistasin from the plasmid pKH4a2 in yeast strains having a chromosomally integrated additional copy of a PDI gene (Examples 16 and 17); expression of antistasin from the vector K991 with an additional PDI gene copy being present on a multicopy yeast shuttle vector named YEp24 (Botstein et al, 1979, Gene, S, 17-24) (Example 20); and expression of both the antistasin and the PDI genes from the yeast shuttle vector pCl/1 (Rosenberg et al, 1984, Nature, 312, 77-80) under control of the GAL10 and GAL1 promoters., respectively. Indeed, Robinson and Wittrup, 1995, op.  cit, also used the GAL1-GAL10 intergenic region to express erythropoietin and concluded that production yeast strains for the secretion of heterologous proteins should be constructed using tightly repressible,   inducible promoters,   otherwise  the negative   effects   of sustained secretion (i.e. lowered detectable BiP and PDI) would be dominant after the many generations of cell growth required to fill a large-scale fermenter.<br>
Subsequent work in the field has identified chromosomal integration of transgenes as the key to maximising recombinant protein production.<br>
Robinson et al, 1994, op. cit., obtained the observed increases in expression of PDGF and S. pombe acid phosphatase using an additional chromosomally integrated PDI gene copy. Robinson et al reported that attempts to use the multicopy 2f.im expression vector to increase PDI protein levels had had a detrimental effect on heterologous protein secretion.<br>
Hayano et al, 1995, op. cit. described the introduction of genes for human lysozyme and PDI into a yeast host each on a separate linearised integration vector, thereby to bring about chromosomal integration.<br>
Shusta et al, 1998, op. cit., reported that in yeast systems, the choice between integration of a transgene into the host chromosome versus the use of episomal expression vectors can greatly affect secretion and, with reference to Parelch &amp; Wittrap, 1997, Biotechnol Prog., 13, 117-122, that stable integration of the scFv gene into the host chromosome using a 8 integration vector was superior to the use of a 2jjm-based expression plasmid.  Parekh &amp; Wittrup, op. cit., had previously taught that the expression of bovine pancreatic trypsin inhibitor (BPTI) was increased by an order of magnitude using a 6 integration vector rather than a 2(j,m-based expression plasmid.   The 2um-based expression plasmid was said to be counter-productive for the production of heterologous secreted protein.<br>
Bao &amp; Fukuhara, 2001, op. cit, reported that "It was first thought that the KIPDI1 gene might be directly introduced into the multi-copy vector that carried the rHSA expression cassette. However, such constructs were found to severely affect yeast growth and plasmid stability. This confirmed our previous finding that the KIPDI1 gene on a multi-copy vector was detrimental to growth of K. lactis cells (Bao et al, 2000)". Bao et .al, 2000, Yeast, 16, 329-341, as referred to hi the<br>
above-quoted passage of Bao &amp; Fulcuhara, reported that the KIPDI1 gene had been introduced into K lactis on a multi-cop}' plasmid, pKan707, and that the presence of the plasmid caused the strain to grow poorly. Bao et a] concluded that over-expression of the KIPDI1 gene was toxic to K. lactis cells. In the light of the earlier findings in Bao el al, Bao &amp; Fulcuhara chose to introduce a single duplication ofKlPDIJ on the host chromosome.<br>
Against this background, we have surprisingly demonstrated that, contrary to the suggestions in the prior art, when the genes for a chaperone protein and a heterologous protein .are co-expressed on a 2u,m-family multi-copy plasmid in yeast, the production of the heterologous protein is substantially increased.<br>
DESCRIPTION OF THE INVENTION<br>
A first aspect of the  present invention provides   a method  for  producing heterologous protein comprising:<br>
(a)	providing  a host cell  comprising  a 2jam-family plasmid. the<br>
plasmid comprising a gene encoding a protein comprising the sequence of<br>
a chaperone protein and a gene encoding a heterologous protein;<br>
(b)	culturing the host cell in a culture medium under conditions that<br>
allow the expression of the gene encoding the chaperone protein and the<br>
gene encoding a heterologous protein;<br>
(c)	purifying the thus expressed heterologous protein from the culture<br>
medium; and<br>
(d)	optionally, lyophilising the thus purified protein.<br>
In one embodiment., step (c) purifies the thus expressed heterologous protein to a commercially acceptable level of purity or a pharmaceutically acceptable level of purity.<br>
Preferably, the method further comprises the step of formulating the- purified heterologous protein with a carrier or diluent, such as a pharmaceutically acceptable carrier or diluent and optionally presenting the thus formulated protein in a unit dosage form.<br>
A second aspect of the present invention provides for the use of a 2um-family plasmid as an expression vector to increase the production of a fungal (preferably yeast) or vertebrate heterologous protein by providing a gene encoding the heterologous protein and a gene encoding a protein comprising the sequence of a chaperone protein on the same 2u.m-family plasmid.<br>
A third aspect of the present invention provides a 2um-family plasmid comprising a gene encoding a protein comprising the sequence of a chaperone protein and a gene encoding a heterologous protein, wherein if the plasmid is based on the 2um plasmid then it is a disintegration vector.<br>
A fourth aspect of the invention provides a host cell comprising a plasmid as defined above.<br>
The present invention relates to recombinantly modified versions of 2um-family plasmids.<br>
Certain closely related species of budding yeast have been shown to contain naturally occurring circular double stranded DNA plasmids: These plasmids., collectively termed 2 urn-family plasmids, include pSRl, pSB3 and pSB4 from Zygosaccharomyces rouxii (formerly classified as Zygosaccharomyces bisporus),<br><br>
plasmids pSBl and pSB2 from Zygosaccharomyces bailii, plasniid pSM] from Zygosaccharomyces fermentati, plasmid pICDl from Kluyveromyces drosphilarum. an un-named plasmid from Pichia membranaefaciens (hereinafter "pPMl") and the 2um plasmid and variants (such as Scpl, Scp2 and Scp3) from Saccharomyces cerevisiae (Volkert, et al., 1989, Microbiological Reviews., 53, 299; Murray er al., 1988, J. Mol Biol. 200, 601; Painting, et al., } 984, .7. Applied Bacteriology, 56. 331).<br>
As a family of plasmids fhese molecules share a series of common features in that they typically possess two inverted repeats on opposite sides of the plasmid, have a similar size around 6-lcbp (range 4757 to 6615-bp), three open reading frames, one of which encodes for a site specific recombinase (FLP) and an autonomously replicating sequence (ARS), also known as an origin of replication (on), located close to the end of one of the inverted repeats.   (Futcher. 1988, Yeast, 4, 27; Murray el al., op. cit, and Toh-e et al, 1986, Basic Life Sci. 40, 425).   Despite their lack  of discernible  DNA  sequence  homology, their shared molecular architecture and the conservation of function of the three open reading frames have demonstrated a common ancestral link between the family members.<br>
Whilst any of the above naturally occurring 2p,m-family plasmids can be used in the present invention, this invention is not limited to the use of naturally occurring 2uni-family plasmids. For the purposes of this invention, a 2u,m-family plasmid is as described below.<br>
A 2um-family plasmid is a circular, double stranded, DNA plasmid. It is typically small such as between 3,000 to 10,000 bp, preferably between 4,500 to 7000 bp, excluding recombinantly inserted sequences.<br>
A 2um-family plasmid typically comprises at least three open reading frames ("ORFs") that each encodes a protein that functions in the stable maintenance of<br>
the 2f.Lm-fa.mily plasmid as a multicopy plasmid. The proteins encoded by the three ORPs can be designated FLP, REP1 and REP2. Where a 2um-farnily plasmid comprises not all three of the ORFs encoding FLP, REP1 and REP2 then ORFs encoding the missing protein(s) should be supplied in trans, either on another plasmid or by chromosomal integration.<br>
A "FLP" protein is a protein capable of catalysing the site-specific recombination between inverted repeat sequences recognised by FLP.    The inverted repeat sequences are termed FLP recombination target (FRT) sites and each is typically present as part of a larger inverted repeat (see below).   Preferred FLP proteins comprise the sequence of the FLP proteins encoded by one of plasmids pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl and the 2|_tm plasmid, for example as described in Volkert et al, op. cit, Murray et al, op. cit., and Painting et al, op. cit.  Variants and fragments of these FLP proteins are also included in the present invention. "Fragments"'and "variants" are those which retain the ability of the native protein to catalyse the site-specific recombination between the same FRT sequences.  Such variants and fragments will usually have at least 50%, 60%, 70%, 80%, 90%, 95%, 98%, 99%, or more, homology with an FLP protein encoded by one of plasmids pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl and the 2(.im plasmid.     Different FLP proteins  can have different FRT sequence specificities. A typical FRT site 'may comprise a core nucleotide sequence flanked by inverted repeat sequences.  In the 2um plasmid, the FRT core sequence is 8 nucleotides in length and the flanking inverted repeat sequences are 13 nucleotides in length (Volkert et al, op. cit.}. However the FRT site recognised by any given FLP protein may be different to the 2um plasmid FRT site.<br>
REP1 and REP2 are proteins involved in the partitioning of plasmid copies during cell division, and may also have a role in the regulation of FLP expression. Considerable sequence divergence has been observed between REP1 proteins from different 2jj.rn-family plasmids, whereas no sequence alignment is possible<br>
between REP2 proteins derived from different 2|_irn-family plasmids.   Preferred REPl and REP2 proteins comprise the sequence of the REPl and REP2 proteins encoded by one of plasmids pSRl, pSBL. pSB2, pSB3, pSB4r. pSMl, pICDl, pPMl and the 2um plasmid, for example as described hi Volkert el al, op. cit, Murray e1 al, op. cit, and Painting et d, op. cit. Variants and fragments of these REPl and REP2 proteins are also included in the present invention. "Fragments" and "variants" of REPl and REP2 are those which, when encoded by the plasmid in place of the native  ORF,  do not substantially  disrupt the  stable  multicopy maintenance of the plasmid within a suitable yeast population.   Such variants and fragments of REPl and REP2 will usually have at least 5%, 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95%, 98%, 99%, or more, homology with a REPl and REP2 protein, respectively, as encoded by one of plasmids pSRl, pSBl, pSB2, pSB3, pSB4, pSM'J, pKDl, pPMl and the 2um plasmid.<br>
The REPl and REP2 proteins encoded by the ORFs on the plasmid must be compatible. It is preferred that the REPl and REP2 proteins have the sequences of REP 1 and REP2 proteins encoded by the same naturally occurring 2p.m-family plasmid, such as pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl and the 2 urn plasmid, or variant or fragments thereof.<br>
A 2um-family plasmid typically comprises two inverted repeat sequences. The inverted repeats may be any size, so long as they each contain an FRT site (see above). The inverted repeats are typically highly homologous. They may share greater than 50%, 60%, 70%, 80%, 90%, 95%, 96%, 97%, 98%, 99%, 99.5% or more sequence identity, hi a preferred embodiment they are identical. Typically the inverted repeats are each between 200 to 1000 bp in length. Preferred inverted repeat sequences may each have a length of from 200 to 300 bp, 300 to 400 bp, 400 to 500 bp, 500 to 600 bp, 600 to 700 bp, 700 to 800 bp, 800 to 900 bp, or 900 to 1000 bp. Particularly preferred inverted repeats are those of the plasmids pSRl<br>
(959 bp), pSB L (675 bp), pSB2 (477 bp), pSB3 (391 bp), pSMl (352 bp), pKDl (346 bp), the 2fam plasmid (599 bp), pSB4 or pPMl.<br>
The sequences of the inverted repeats may be varied. However, the sequences of the FRT site in each inverted repeat should be compatible with the specificity of the FLP protein encoded by the plasmid, thereby to enable the encoded FLP protein to act to catalyse the site-specific recombination between the inverted repeat sequences of the plasmid.     Recombination between inverted repeat sequences (and thus the ability of the FLP protein to recognise the FRT sites with the plasmid) can be determined by methods known in the art.   For example, a plasmid in a yeast cell under conditions that favour FLP expression can be assayed for changes in the restriction profile of the plasmid which would result from a change in the orientation of a region of the plasmid relative to another region of the plasmid. The detection of changes in restriction profile indicate that the FLP protein is able to recognise the FRT sites in the plasmid and therefore that the FRT site in each inverted repeat are compatible with the specificity of the FLP protein encoded by the plasmid.<br>
In a particularly preferred embodiment, the sequences of inverted repeats, including the FRT sites, are derived from the same 2um-family plasmid as the ORF encoding the FLP protein, such as pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl or the 2um plasmid.<br>
The inverted repeats are typically positioned with the 2um-family plasmid such that the two regions defined between the inverted repeats (e.g. such as defined as UL and US in the 2um plasmid) are of approximately similar size, excluding exogenously introduced sequences such as transgenes. For example, one of the two regions may have a length equivalent to at least 40%, 50%, 60%, 70%, 80%, 90%, 95% or more, up to 100%, of the length of the other region.<br>
A 2 urn-family plasnud typically comprises the ORF that encodes FLP and one inverted repeal (arbitrarily termed ':IR1" to distinguish it from the other inverted repeat mentioned in the next paragraph) juxtaposed in such a manner that IR1 occurs at the distal end of the FLP ORF5 without any mtervening coding sequence, for example as seen m the 2pm plasmid. By "distal end" in this context we mean the end of the FLP ORF opposite to the end from which the promoter initiates its transcription. In a preferred embodiment, the distal end of the FLP ORF overlaps with IRL<br>
A 2pm-family plasmid typically comprises the ORF that encodes REP2 and the other inverted repeat (arbitrarily termed "IR2" to distinguish it from IR1 mentioned in the previous paragraph) juxtaposed in such a manner that IR2 occurs at the distal end of the REP2 ORF, without any intervening coding sequence, for example as seen in the 2pm plasmid. By "distal end" in this context we mean the end of the REP2 ORF opposite to the end from which the promoter initiates its transcription.<br>
In one embodiment, the ORFs encoding REP2 and FLP may be present on the same region of the two regions defined between the inverted repeats of the 2 jam-family plasmid, which region may be the bigger or smaller of the regions (if there is any inequality in size between the two regions).<br>
hi one embodiment the ORFs encoding REP2 and FLP may be transcribed from divergent promoters.<br>
Typically, the regions defined between the inverted repeats (e.g. such as defined as UL and US in the 2pm plasmid) of a 2p,m-family plasmid may comprise not more than two endogenous genes that encode a protein that functions in the stable maintenance of the 2pm-family plasmid as a multicopy plasmid. Thus in a preferred embodiment, one region of the plasmid defined between the inverted<br>
repeats may comprise not more than the ORFs encoding FLP and REP2; FLP and REP1; or REP1 and REP2, as endogenous coding sequence.<br>
A 2urn-family plasmid typically comprises an origin of replication (also known as an "autonomously replicating sequence - "ARS"), which is typically bidirectional. Any appropriate ARS sequence can be present. Consensus sequences typical of yeast chromosomal origins of replication may be appropriate (Broach et al, 1982, Cold Spring Harbor Symp. Quant. Biol, 47, 1165-1174; Williamson, Yeast, 1985, 1, 1-14). Preferred ARSs include those isolated from pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl and the 2um plasmid.<br>
Thus, a preferred 2jam-family plasmid may comprise ORFs encoding FLP, REP1 and REP2, two inverted repeat sequences each inverted repeat comprising an FRT site compatible with the encoded FLP protein, and an ARS sequence. Preferably the FRT sites are derived from the same 2urn-family plasmid as the sequence of the encoded FLP protein.   More preferably the sequences of the encoded REP1 and REP2 proteins are derived from the same 2 jam-family plasmid as each other. Even more preferably, the FRT sites are derived from the same 2pm-family plasmid as the sequence of the encoded FLP, REP1 and REP2 proteins. Yet more preferably, the sequences of the ORFs encoding FLP, REP1 and REP2, and the sequence of the inverted repeats (including the FRT sites) are derived from the same 2um-family plasmid. Furthermore, the ARS site may be derived from the same 2um-famiry plasmid as one or more of the ORFs of FLP, REP1 and REP2, and the sequence of the inverted repeats (including the FRT sites).<br>
The term "derived from" includes sequences having an identical sequence to the sequence from which they are derived. HoweverTvariants and fragments thereof, as defined above, are also included. For example, an FLP gene having a sequence derived from the FLP gene of the 2um plasmid may have a modified promoter or other regulatory sequence compared to that of the naturally occurring gene.<br>
Additionally or alternative]), an FLP gene having a sequence derived from the FLP gene of the 2jjm plasmid may have a modified nucleotide sequence hi the open reading frame which may encode the same protein as the naturally occurring gene, or may encode a modified FLP protein. The same considerations apply to other sequences on a 2um-farnily plasmid having a sequence derived from a particular source.<br>
Optionally, a 2(.irn-family plasmid may comprise a region derived from the STB<br>
region (also known as REPS) of the 2um plasmid, as defined in Vollcert et ol, op.<br>
tit. The STB region in a 2um-family plasmid of the invention may comprise two<br>
or more tandem repeat sequences, such as three., four, five or more. Alternatively,,<br>
no tandem repeat sequences may be present. The tandem repeats may be any size,<br>
such as 10, 20, 30, 40, 50, 60 70, 80, 90, 100 bp or more in length. The tandem<br>
repeats in the STB region of the 2um plasmid are 62 bp in length.    It is not<br>
essential for the sequences of the tandem repeats to be identical.  Slight sequence<br>
variation can be tolerated.  It may be preferable to select an STB region from the<br>
same plasmid as either or both of the KEPI and REP2 ORFs. The STB region is<br>
thought to be a ra-acting element and preferably is not transcribed.<br>
Optionally, a 2uni~farnily plasmid may comprise an additional ORF that encodes a protein that functions in the stable maintenance of the 2|im-family plasmid as a multicopy plasmid.  The additional protein can be designated RAP or D.   ORP's encoding the RAF or D gene can be seen on, for example, the 2um plasmid and pSMl.  Thus a RAF or D ORP can comprise a sequence suitable to encode the protein product of the RAP or D gene ORFs encoded by the 2u.m plasmid or pSMl, or variants and fragments thereof. Thus variants and fragments of the protein products of the RAF or D genes of the 2p.m. plasmid or pSMl are also included in the present invention.   "Fragments" and "variants" of the protein products of the RAF or D genes of the 2um plasmid or pSMl are those which,<br>
when encoded by the 2um plasmid or pSMl in place of the native ORP, do not<br>
disrupt the stable multicopy maintenance of the plasmid within a suitable yeast population. Such variants and fragments will usually have at least 5%, 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95%, 98%, 99%, or more, homology with the protein product of the RAF or D gene ORFs encoded by the 2 urn plasmid or pSMl.<br>
A naturally occurring 2p,m-family plasmid may be preferred.    A naturally occurring 2um-family plasmid is any plasmid having the features defined above, which plasmid is found to naturally exist in yeast, i.e. has not been recomhinantly modified to include heterologous sequence.   Preferably the naturally occurring 2urn-family plasmid is selected from pSRl  (Accession No. X02398), pSB3 (Accession No. X02608) or pSB4 as obtained from Zygosaccharomyces roiocii, pSBl or pSB2 (Accession No. NCJ)02055 or Ml 8274) both as obtained from Zygosaccharomyces bailli, pSMl (Accession No. NC_002054) as obtained from Zygosaccharomyces fermentati, pKDl (Accession No. X03961) as obtained from Kluyveromyces drosophilarum, pPMl from Pichia membranaefaciens or, most preferably, the 2 urn plasmid (Accession No. NC_001398 or JO 1347) as obtained from Saccharomyces cerevisiae.   Accession numbers in this paragraph refer to NCBI deposits.<br>
The 2um plasmid (Figure 1) is a 6,318-bp double-stranded DNA plasmid, endogenous in most Saccharomyces cerevisiae strains at 60-100 copies per haploid genome. The 2jj,m plasmid comprises a small unique (US) region and a large unique (UL) region, separated by two 599-bp inverted repeat sequences. Site-specific recombination of the inverted repeat sequences results in inter-conversion between the A-form and B-form of the plasmid in vivo (Vollcert &amp; Broach, 1986, Cell, 46, 541). The two forms of 2um differ only in the relative orientation of their unique regions.<br>
While DNA sequencing of a cloned 2(j.m plasmid (also known as Scpl) from<br>
Saccharomyces cerevisiae gave a size of 6,318-bp (Hartley and Donelson, 1980.<br>
Nature, 286, 860;, other slightly smaller variants of 2um, Scp2 and Scp3, are<br>
known to exist as a result of small deletions of 125-bp and 220-bp, respectively, in<br>
a region Icnown as STB (Cameron et al., 1977, Nucl. Acids Res., 4, 1429: Kilcuchi,<br>
1983, Cell, 35, 487 and Livingston &amp; Hahne, 1979, Proc. Nail Acad. Sci. USA,<br>
76, 3727). In one study about 80% of natural Saccharomyces strains from around<br>
the world contained DNA homologous to 2|_un (by Southern blot analysis)<br>
(Hollenberg, 1982, Current Topics in Microbiolog)'and Immunobiology, 96, 119).<br>
Furthermore,   variation   (genetic   polymorphism)   occurs   within   the   natural<br>
population of 2 urn plasmids found in S. cerevisiae and S. carlsbergensis, with the<br>
NCBI sequence (accession number NC_001398) being one example.<br>
The 2j-im plasmid has a nuclear localisation and displays a high level of mitotic stability (Mead et al, 1986, Molecular &amp; General Genetics, 205, 417). The inherent stability of the 2|o,m plasmid results from a plasmid-encoded copy number amplification and partitioning mechanism, which can be compromised during the development of chimeric vectors (Futcher &amp; Cox, 1984, J. Bacterial., 157, 283; Bachmair &amp; Ruis, 1984, Monatshefte fur Chemie, 115, 1229). A yeast strain, which contains a 2um plasmid is known as [cir+], while a yeast strain which does not contain a 2urn plasmid is Icnown as [cir ].<br>
The US-region of the 2um plasmid contains the .REP2 and FLP genes, and the UL-region contains the KEPI and D (also known as RAF) genes, the <st.b-locus and the origin of replication hicks cell button broach mol. cell. biol flp recombinase binds to frt-sites recognition target within inverted repeats mediate site-specific recombination which is essential for natural plasmid amplification control copy number in vivo et al proc. natl. acad. sci. u.s.a. jayaram></st.b-locus>
number of 2fim-family plasmids can be significantly affected by changes in Flp recombinase activity (Sleep et al, 2001, Yeast, 18, 403; Rose &amp; Broach, 1990, Methods EnzymoL, 185, 234). The Repl and Rep2 proteins mediate plasmid segregation, although their mode of action is unclear (Sengupta et al, 2001, J. Bacterial, 183, 2306). They also repress transcription of the FLP gene (Reynolds etal, 1987, Mol. Cell. Biol, 7, 3566).<br>
The FLP and REP2 genes of the 2um plasmid are transcribed from divergent promoters, with apparently no intervening sequence defined between them. The FLP and REP2 transcripts both terminate at the same sequence motifs within the inverted repeat sequences, at 24-bp and 178-bp respectively after their translation termination codons (Sutton &amp; Broach, 1985, Mol. Cell. Biol., 5,2770).<br>
In the case of FLP, the C-terminal coding sequence also lies within the inverted repeat sequence.    Furthermore, the two inverted repeat sequences are highly conserved over 599-bp, a feature considered advantageous to efficient plasmid replication and amplification in vivo, although only the FRT-sites (less than 65-bp) are essential for site-specific recombination in vitro (Senecoff et al, 1985, Proc. Natl. Acad. Sci. U.S.A., 82, 7270; Jayaram, 1985, Proc. Natl. Acad. Sci. U.S.A., 82, 5875; Meyer-Leon et al, 1984, Cold Spring Harbor Symposia On Quantitative Biology, 49, 797). The key catalytic residues of Flp are arginine-308 and ryrosine-343 (which is essential) with strand-cutting facilitated by histidine-309 and histidine 345 (Prasad et al, 1987, Proc. Natl Acad. Sci. U.S.A., 84, 2189; Chen et al, 1992, Cell, 69, 647; Grainge et al, 2001, J. Mol. Biol, 314, 717).<br>
Two functional domains are described in Rep2. Residues 15-58 form a Repl-binding domain, and residues 59-296 contain a self-association and STB-binding region (Sengupta et al, 2001, J. Bacterial, 183,2306).<br>
Chimeric or largt deletion mutant derivatives of 2um which lack many of the essential functional regions of the 2urn plasmid but retain the functional cis element ARS and STB, cannot effectively partition between mother and daughter cells at cell division. Such plasmids can do so if these functions are supplied in trans, by for instance the provision of a functional 2urn plasmid within the host, such as a [cir+] host.<br>
Genes of interest have previously been inserted into the UL-region of the 2}o,m plasmid.  For example, see plasmid pSAC3Ul in EP 0 286 424 and the plasmid shown in Figure 2, which includes a (3-lactamase gene (for ampicillin resistance), a LEU2 selectable marker and an oligonucleotide linker, the latter two of which are inserted into a unique SndBl-site within the UL-region of the 2fam-like disintegration vector, pSAC3 (see EP 0 286 424).  The E. coli DNA between the Xbal-siies that contains the ampicillin resistance gene is lost from the plasmid shown in Figure 2 after transformation into yeast. This is described in Chinery &amp; Hrnchliffe, 1989, Curr. Genet, 16, 21 and EP 0 286 424, where these types of vectors are designated "disintegration vectors". Further polynucleotide insertions can be made in a Notl-site within a linker (Sleep et al, 1991, Biotechnology (N Y), 9,183).<br>
Alternative insertion sites in 2um plasmid are known in the art, including those described in Rose &amp; Broach (1990, Methods Enzymol, 185, 234-279), such as plasmids pCV19, pCV20, CVneo. which utilise an insertion at EcoRI in FLP, plasmids pCV21, pGT41 and pYE which utilise Eco'KL in D as the insertion site, plasmid pHKB52 which utilises PstI in D as the insertion site, plasmid pJDB248 which utilises an insertion at Pstl in D and Eco'Rl in D, plasmid pJDB219 in winch Pstl in D and EcoRI in FLP are used as • insertion sites, plasmid G18, plasmid pABIS which utilises an insertion at Clal in FLP, plasmids pGT39 and pA3, plasmids pYTll, pYT14 and pYTll-TEU which use Pstl in D as the insertion site, and plasmid PTY39 which uses £coRI in FLP as the insertion site.<br>
Other 2pm plasmids include pSACS, pSACSUl, pSAC3U2, pSACSOO, pSACSlO, pSAC3Cl, pSACSPLl, pSAC3SL4, and pSACSSCl are described in EP 0 286 424 and Chinery &amp; Hinchliffe (1989, Cwr. Genet, 16, 21-25) which also described Pstl, Eagl or SndBl as appropriate 2um insertion sites. Further 2 urn plasmids include pAYE255, pAYE316, pAYE443, pAYE522 (Kerry-Williams et al, 1998, Yeast, 14, 161-169), pDB2244 (WO 00/44772), andpAYE329 (Sleep et al, 2001, Yeast, 18, 403-421).<br>
In one preferred embodiment, one or more -genes are inserted into a 2j.im-family plasmid within an untranscribed region around the ARS sequence. For example, in the 2um plasmid obtained from S. cerevisiae, the untranscribed region around the ARS sequence extends from end of the D gene to the beginning of ARS sequence. Insertion into SndBl (near the origin of replication sequence ARS) is described in Chinery &amp; Hinchliffe, 1989, Cwr. Genet., 16, 21-25. The skilled person will appreciate that gene insertions can also be made in the untranscribed region at neighbouring positions to the SndBl site described in Chinery &amp; Flinchliffe.<br>
In another preferred embodiment, REP2 and FLP genes in a 2um-family plasmid each have an inverted repeat adjacent to them, and one or more genes are inserted into a 2(.Lm-family plasmid within the region between the first base after the last functional codon of either the REP2 gene or the FLP gene and the last base before the FRT site in the inverted repeat adjacent to said gene.   The last functional codon of either a REP2 gene or a FLP gene is the codon in the open reading frame of the gene that is furthest downstream from the promoter of the gene whose replacement by a stop codon will lead to an unacceptable loss of multicopy stability of the plasmid, as defined herein.  Thus, disruption of the REP2 or FLP genes at any point downstream of the last functional codon in either gene, by insertion of a polynucleotide sequence insertion, deletion or substitution will not lead to an unacceptable loss of multicopy stability of the plasmid.<br>
For example, the REP 2 gene of the 2um plasmid can be disrupted after codon 59 and that the FLF gene of the 2um plasmid can be disrupted after codon 344. each without a loss of multicopy stability of the plasmid. The last functional codon in equivalent genes in other 2pm-farniry plasmids can be determined routinely by making mutants of the plasmids in either the FLP or REP 2 genes and following the tests set out herein to determine whether the plasmid retains multicopy stability.<br>
One can determined whether a plasmid retains multicopy stability using test such as defined in Chinery &amp; Hinchliffe (1989, Ciar. Genet, 16, 21-25). For yeast that do not grow in the non-selective media (YPD, also designated YEPD) defined in Chinery &amp; Hinchliffe (1989, Curr.  Genet., 16, 21-25) other appropriate non-selective media might be used. Plasmid stability may be defined as the percentage cells remaining prototrophic for the selectable marker after a defined number of generations.   The number of generations will preferably be sufficient to show a difference between a control plasmid, such as pSAC35 or pSACSlO, or to shown comparable stability to such a control plasmid.   The number of generations may be 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100 or more.    Higher numbers are preferred.    The acceptable plasmid stability might be 1%, 2%, 3%, 4%, 5%, 10%, 15%, 20%, 25%, 30%, 40%, 50%, 60%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99%, 99.9% or substantially 100%. Higher percentages are preferred. The skilled person will appreciate that, even though a plasmid may have a stability less than 1 00% when grown on non-selective media, that plasmid can still be of use when cultured in selective media.  For example plasmid pDB2711 as described in the examples is only 1 0% stable when the stability is determined accordingly to test of Example 1 , but provides a 15-fold increase in recombinant transferrin productivity in shake flask culture under selective growth conditions.<br>
Thus one or more gene insertions may occur between the first base after the last functional codori of the REP2 gene and the last base before the FRT site in an inverted repeat adjacent to said gene, more preferably between the first base of the inverted repeat and the last base before the FRT site, even more preferably at a position after the translation termination codon of the REP2 gene and before the last base before the FRT site.<br>
Additionally or alternatively one or more gene insertions may occur between the first base after the last functional codon of the FLP gene and the last base before the FRT site in an inverted repeat adjacent to said gene, preferably between the first base of the inverted repeat and the last base before the FRT site, more preferably between the first base after the end of the FLP coding sequence and the last base before the FRT site, such as at the first base after the end of the FLP coding sequence,<br>
In one preferred embodiment, where the 2 urn-family plasmid is based on the 2(j,m plasmid of S. cerevisiae, it is a disintegration vector as known in the art (for example, see EP 286 424, the contents of which are incorporated herein by reference). A disintegration vector may be a 2(.un plasmid vector comprising a DNA sequence which is intended to be lost by recombination, three 2u.ni FRT sites, of which one pair of sites is in direct orientation and the other two pairs are in indirect orientation, and a DNA sequence of interest (such as an E. coli origin of replication and bacterial selectable marker), the said sequence to be lost being located between the said sites which are in direct orientation.<br>
Thus, the sequence to be lost may comprise a selectable marker DNA sequence.<br>
A preferred disintegration vector comprises a complete 2um plasmid additionally carrying (i) a bacterial plasmid DNA sequence necessary for propagation of the vector in a bacterial host; (ii) an extra 2 um FRT site; and a selectable marker<br>
DNA sequence for yeast transformation: the said bacterial plasmid DNA sequence being present and the extra FRT site being created at a restriction site, such as Xbal, in one of the two inverted repeat sequences of the 2 urn plasmid. the said extra FRT' site being in direct orientation in relation to the endogenous FRT site of the said one repeat sequence, and the bacterial plasmid DNA sequence being sandwiched between the extra FRT site and the endogenous FRT site of the said one repeat sequence. In a preferred disintegration vector, all bacterial plasmid DNA sequences are sandwiched as said. A particularly preferred 2ixrn plasmid vector has substantially the configuration of pSAC3 as shown in EP 286 424.<br>
The term "disintegration vector" as used herein also includes plasmids as defined in US 6,451,559, the contents of which are incorporated herein by reference. Thus a disintegration vector may be a 2um vector that, other than DNA sequence encoding non-yeast polypeptides, contains no bacterial (particularly E. coif) origin of replication, or more preferably no bacterial (particularly E. coif) sequence and preferably all DNA in said vector, other than DNA sequence encoding non-yeast polypeptides, is yeast-derived DNA.<br>
The term "chaperone" as used herein refers to a protein that binds to and stabilises an othenvise unstable conformer of another protein, and by controlled binding and release, facilitates its correct fate in vivo, be it folding, oligomeric assembly, transport to a particular subcellular compartment, or disposal by degradation. Accordingly a chaperone is also a protein that is involved in protein folding, or which has chaperone activity or is involved in the unfolded protein response. Chaperone. proteins of this type are known in the art, for example in the Stanford Genome Database (SGD), http:://db.yeastgenome.org.   Preferred cliaperones are eukaryotic cliaperones, especially preferred chaperones are yeast chaperones, including AHA ], CCT2, CCT3, CCT4, CCT5, CCT6, CCT7, CCT8, CNSJ, CPR3, CPR6, ER01, EUG1, FMO1, HCH1, HSP10, HSP12, HSP104, HSP26, HSP30, HSP42, HSP60, HSP78, HSP82, JEM/, MDJ1, MDJ2, MPD1, MPD2, PDI1,<br>
PFD1,ABC1, APJl, ATP11, ATP12, BTTl, CDC37, CPR7, PISC82, KAR2, LHS1, MGE1, MRSll, NOB1, ECMIO, SSAl, SSA2, SSA3, SSA4, SSCI, SSE2, SILl, SLSJ, ORM1, ORM2, PERI, PTC2, PSE1, UBI4 and HAC1 or a truncated intronless HAC1 (Valkonen et al 2003, Applied Environ. Micro., 69, 2065)<br>
A chaperone useful in the practice of the present invention may be:<br>
• a heat shock protein, such as a protein that is a member of the hsp70 family of proteins (including Kar2p, SSA and SSB proteins, for example proteins encoded by SSAl, SSA2, SSA3, SSA4, SSB1 and SSB2), a protein that is a member of the HSP90-family, or a protein that is a member of the HSP40-family or proteins involved in their modulation (e.g. Sillp), including DNA-J and DNA-J-like proteins (e.g. Jemlp, Mdj2p);<br>
•	a protein that is a member of the karyopherin/irnportin family of proteins,<br>
such as the alpha or beta families of Icaryopherin/importin proteins, for<br>
example the karyopherin beta protein PSE1;<br>
•	a protein that is a member of the ORMDL family described by Hjelmqvist<br>
et al, 2002,  Genome Biology, 3(6), research0027.1-0027.16, such as<br>
Orm2p.<br>
a protein that is naturally located in the endoplasmic reticulurn or elsewhere in the secretory pathway, such as the golgi. For example, a protein that naturally acts in the lumen of the endoplasmic reticulurn (ER), particularly in secretory cells, such as PDI<br>
a protein thai is transmembrane protein anchored in the ER. sucb as a member of the ORMDL family described by Hjelmqvist el al, 2002. supra, (for example. Orm2p);<br>
a protein that acts in the cytosol, such as the hsp70 proteins, hicludhig SSA and SSB proteins, for example protein production SSA1, SSA2, SSA3, SSA4, SSB1 and SSB2;<br>
a protein that acts in the nucleus, the nuclear envelope and/or the cytoplasm, such as Pselp;<br>
a protein that is essential to the viability of the cell, such as PDI or an essential karyopherin protein, such as Pselp;<br>
a protein that is involved in sulphydryl oxidation or disulphide bond formation, breakage or isomerization, or a protein, that catalyses thiol:disulphide interchange reactions in proteins, particularly during the biosynthesis of secretory and cell surface proteins, such as protein disulphide isomerases (e.g. Pdilp, Mpdlp), homologues (e.g. Euglp) and/or related proteins (e.g. Mpd2p, Fmolp, Erolp);<br>
a protein that is involved in protein synthesis, assembly or folding, such as PDI and Ssalp;<br>
a protein that binds preferentially or exclusively to unfolded, rather than mature protein, such as the hsp70 proteins, including SSA and SSB proteins, for example proteins encoded by SSA1, SSA2, SSA3, SSA4, SSBJ and SSB2:<br>
a protein that prevents aggregation of precursor proteins in the cytosol, such as the hsp70 proteins, including SSA and SSB proteins, for example proteins encoded by SSA1, SSA2, SSA3, SSA4, SSB1 and SSB2;<br>
a protein that binds to and stabilises damaged proteins, for example Ssalp;<br>
a protein that is involved in the unfolded protein response or provides for increased resistance to agents (such as tunicamycin and dithiothreitoi) that induce the unfolded protein response, such as a member of the ORMDL family described by Hjelmqvist et al, 2002, supra (for example, Orm2p) or proteins involved in the response to stress (e.g. Ubi4p);<br>
a protein that is a co-chaperone and/or a protein indirectly involved in protein folding and/or the unfolded protein response (e.g. hsp!04p, Mdjlp);<br>
a protein that is involved in the nucleocytoplasmic transport of macrornolecules, such as Pselp;<br>
a protein that mediates the transport of macrornolecules across the nuclear membrane by recognising nuclear location sequences and nuclear export sequences and interacting with the nuclear pore complex, such as PSE1;<br>
a protein that is able to reactivate ribonuclease activity against RNA of scrambled ribonuclease as described in as described in EP 0 746 611 and Hillson et al, 1984, Methods EnzymoL, 107,281-292, such as PDI;<br>
a protein that has an acidic pi (for example, 4.0-4.5), such as PDI;<br>
a protein that is a member of the Iisp70 family, and preferably possesses an N-terminal ATP-binding domain and a C-tenninal peptide-biiiding domain, such as Ssalp.<br>
a protein that is a peptidyl-protyl cis-trans isomerases (e.g. CprSp. Cpr6p);<br>
a protein that is ahomologue of known chaperones (e.g. HsplOp);<br>
a protein that is a mitochondria! chaperone (e.g CprSp);<br>
a protein that is a cytoplasmic or nuclear chaperone (e.g Cnslp);<br>
a protein that is a membrane-bound chaperone (e.g. Orm2p, Fmolp);<br>
a protein that has chaperone activator activity or chaperone regulator}' activity (e.g. Ahalp, Haclp, Hchlp);<br>
a protein that transiently binds to polypeptides in their immature form to cause proper folding transportation and/or secretion, including proteins required for efficient translocation into the eiidoplasmic reticulum (e.g. Lhslp) or their site of action within the cell (e.g. Pselp);<br>
a protein that is a involved in protein complex assembly and/or ribosome assembly (e.g. Atpllp, Pselp, Noblp);<br>
a protein of the chaperonin T-complex (e.g. Cct2p); or<br>
•	a protein of the prefoldin complex (e.g. Pfdlp).<br>
A preferred chaperone is protein disulphide isomerase (PDI) or a fragment or variant thereof having an equivalent ability to catalyse the formation of disulphide bonds within the lumen of the endoplasmic reticulum (ER). By "PDI" we include any protein having the ability to reactivate the ribonuclease activity against RNA of scrambled ribonuclease as described in EP 0 746 611 and Hillson et al, 1984, Methods Enzymol., 107,281-292.<br>
PDI is an enzyme which typically catalyzes thiol:disulphide interchange reactions, and is a major resident protein component of the ER lumen in secretory cells.  A body of evidence suggests that it plays a role in secretory protein biosynthesis (Freedman, 1984, Trends Biochem. Sci, 9, 438-41) and this is supported by direct cross-linking studies in situ (Roth and Pierce, 1987, Biochemistry, 26, 4179-82). The finding that microsomal membranes deficient in PDI show a specific defect in cotranslational protein disulphide (Bulleid and Freedman, 1988, Nature, 335, 649-51) implies that the enzyme functions as a catalyst of native disulphide bond formation during the biosynthesis of secretory and cell surface proteins. This role is consistent with what is known of the enzyme's catalytic properties in vitro; it catalyzes thiol: disulphide interchange reactions leading to net protein disulphide formation, breakage or isomerization, and can typically catalyze protein folding and the formation of native disulphide bonds in a wide variety of reduced, unfolded protein substrates (Freedman et al., 1989, Biochem. Soc. Symp., 55, 167-192).   PDI also functions as a chaperone since mutant PDI lacking isomerase activity accelerates protein folding (Hayano et al, 1995, FEBS Letters, 377, 505-511).   Recently, sulphydryl oxidation, not disulphide isomerisation was reported to be the principal function of Protein Disulphide Isomerase in S.  cerevisiae (Solovyov et al, 2004, J. Biol. Chem., 279 (33) 34095-34100).   The DNA and ammo acid sequence of the enzyme is known for several species (Scherens et al,<br>
1991, Yean, 1',   185-193; Farquhar et al,  1991, Gene, 108, 81-89: EP074661; EP0293793; EP0509841) and there is increasing information on the mechanism of action of the enzyme purified to homogeneity from mammalian liver (Creighton et al 1980, J. Mol. BioL, 142, 43-62; Freedman el al, 1988, Biochem. Soc. Trans., 16, 96-9; Gilbert. 1989, Biochemistry, 28, 7298-7305; Lundstrom and Holmgren, 1990,./. Biol. Ghent, 265, 9114-9120; Hawlcins and Freedman, 1990, Biochem. J., 275, 335-339).  Of the many protein factors currently implicated as mediators of protein folding, assembly and translocation in the cell (Rothman, 1989, Cell, 59, 591-601), PDI has a well-defined catalytic activity.<br>
The deletion or inactivation of the endogenous PDI gene in a host results in the production of an inviable host. In other words, the endogenous PDI gene is an "essential" gene.<br>
PDI is readily isolated from mammalian tissues and the homogeneous enzyme is a homodimer (2x57 IcD) with characteristically acidic pi (4.0-4.5) (Hillson et al, 1984, op. cit.). The enzyme has also been purified from wheat and from the alga Chlamydomonas reinhardii (ICaska et al, 1990, Biochem. J., 268, 63-68), rat (Edman el al, 1985, Nature, 317, 267-270), bovine (Yamauchi et al,  1987, Biochem. Biophys. Res.  Comm,, 146, 1485-1492), human (Pihlajaniemi et al, 1987, EMBO J., 6, 643-9), yeast (Scherens et al, supra; Farquhar et al, op. cit.} and chick (Parkkonen et al, 1988, Biochem. J., 256, 1005-1011).   The proteins from these vertebrate species show a high degree of sequence conservation throughout and all show several overall features first noted in the rat PDI sequence (Edman et al, 1985, op. cit}.<br>
Preferred PDI sequences include those from humans and those from yeast species, such as S. cerevisiae.<br>
A yeast protein disulphide isomerase precursor, PDI1, can be found as Genbank accession no. CAA42373 or BAA00723. It has the following sequence of 522 amino acids:<br>
1 mkfsagavls wsslllassv faqgeavape dsavvklatd sfneyiqshd Ivlaeffapw 61 cghcknmape yvkaaetlve knitlaqidc tenqdlcmeh nipgfpslki fknsdvnnsi 121 dyegprtaea ivqfmikqsq pavavvadlp aylanetfvt pvivqsgkid adfnatfysm 181 ankhfndydf vsaenadddf klsiylpsam depvvyngkk adiadadvfe kwlqvealpy 241 fgeidgsvfa qyvesglplg ylfyndeeel eeykplftel akknrglmnf vsidarkfgr 301 hagnlnmkeq fplfaihdmt edlkyglpql seeafdelsd kivleskaie slvkdflkgd 361 aspivksqei fenqdssvfq Ivgknhdeiv ndpkkdvlvl yyapwcghck rlaptyqela 421 dtyanatsdv liakldhten dvrgwiegy ptivlypggk ksesvvyqgs rsldslfdfi 481 kenghfdvdg kalyeeaqek aaeeadadae ladeedaihd el<br>
An alternative yeast protein disulphide isomerase sequence can be found as Genbank accession no. CAA38402, It has the following sequence of 530 amino acids<br>
1 mkfsagavls wsslllassv faqqeavape dsavvklatd sfneyiqshd Ivlaeffapw 61 cghcknmape yvkaaetlve knitlaqidc tenqdlcmeh nipgfpslki fknrdvnnsi 121 dyegprtaea ivqfmikqsq pavavvadlp aylanetfvt pvivqsgkid adfnatfysm 181 ankhfndydf vsaenadddf klsiylpsam depvvyngkk adiadadvfe kwlqvealpy 241 fgeidgsvfa qyvesglplg ylfyndeeel eeykplftel akknrglmnf vsidarkfgr 301 hagnlnmkeq fplfaihdmt edlkyglpql seeafdelsd kivleskaie slvkdflkgd 361 aspivksqei fenqdssvfq Ivgknhdeiv ndpkkdvlvl yyapwcghck rlaptyqela 421 dtyanatsdv liakldhten dvrgwiegy ptivlypggk ksesvvyqgs rsldslfdfi 481 kenghfdvdg kalyeeaqek aaeeaeadae aeadadaela deedaihdel<br>
The following alignment of these sequences (the sequence of Genbank accession no. CAA42373 or BAA00723 first, the sequence of Genbank accession no. CAA38402 second) shows that the differences between these two sequences are a single amino acid difference at position 114 (highlighted in bold) and that the<br>
sequence cteimeu uy uciiuauj.... ussion no. CAA38402 contains the additional ammo acids EADAEAEA at positions 506-513.<br>
1       mkfsaqavls   wsslllassv   faqqeavape   dsavvklatd  sfneyigshd  Ivlaeffapw 1       mkfsagavls  wsslllassv   fagqeavape   dsavvklatd  sfneyiqshd  Ivlaeffapw<br>
61     cghcknmape  yvkaaetlve   knitlaqidc  tenqdlcmeh  nipgfpslki   flcnsdvnnsi 61     cghckranape  yvkaaetlve  knitlaqidc tengdlcmeh nipgfpslki   fknrdvnnsi<br>
121  dyegprtaea  ivqfmilcgsg pavavvadlp  aylanetfvt pvivqsgkid  adfnatfysm 181  dyegprtaea  ivgfmikqsq pavavvadlp  aylanetfvt pvivqsgkid  adfnatfysm<br>
161  ankhfndydf vsaenadddf   klsiylpsajn depvvyngkk  adiadadvfe   kwlqvealpy 161   ankhfndydf  vsaenadddf   klsiylpsam depvvyngkk  adiadadvfe   kwlqvealpy<br>
241  fgeidgsvfa  qyvesglplg ylfyndeeel  eeykplftel  akknrglmnf vsidarkfgr 241  fgeidgsvfa  qyvesglplg ylfyndeeel  eeykplftel   akknrglmnf vsidarkfgr<br>
301 hagnlnmkeq  fplfaihdmt  edlkyglpql  seeafdelsd kivleskaie   slvkdflkgd 301 hagnlnmkeq  fplfaihdmt  edlkyglpql  seeafdelsd kivleskaie   slvkdflkgd<br>
361   aspivksqei   fenqdssvfq  Ivgknhdeiv ndpkkdvlvl  yyapwcghck  rlaptyqela 361   aspivksqei   fenqdssvfq  Ivgknhdeiv  ndpkkdvlvl  yyapwcghck  rlaptyqela<br>
421  dtvanatsdv liakldhten dvrgvviegy ptivlypggk ksesvvyqgs  rsldslfdfi 421  dtyanatsdv liakldhten dvrgvviegy ptivlypggk ksesvvyqgs   rsldslfdfi<br>
481  kenghfdvdg kalyeeaqek  aaeea*****   ***dadaela  deedaihdel 481  kenghfdvdg kalyeeaqek aaeeaeadae  aeadadaela deedaihdel<br>
Variants and fragments of the above PDI sequences, and variants of other naturally occurring PDI sequences are also included in the present invention. A "variant", in the context of PDI, refers to a protein wherein at one or more positions there have been amino acid insertions, deletions, or substitutions, either conservative or non-conservative, provided that such changes result in a protein whose basic properties,<br>
for example enzymatic activity (type of and specific activity), thermostability, activity in a certain pH-range (pH-stability) have not significantly been changed. "Significantly" in this context means that one skilled in the art would say that the properties of the variant may still be different but would not be unobvious over the ones of the original protein.<br>
By "conservative substitutions" is intended combinations such as Val, He, Leu, Ala, Met; Asp, Glu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Trp. Preferred conservative substitutions include Gly, Ala; Val, He, Leu; Asp, Glu; Asn, Gin; Ser, Thr; Lys, Arg; and Phe, Tyr.<br>
A "variant" typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably at least 80%, more preferably at least 90%, even more preferably at least 95%, yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the polypeptide from which it is derived.<br>
The percent sequence identity between two polypeptides may be determined using suitable computer programs, as discussed below. Such variants may be natural or made using the methods of protein engineering and site-directed mutagenesis as are well known in the art.<br>
A "fragment", hi the context of PDI, refers to a protein wherein at one or more positions there have been deletions. Thus the fragment may comprise at most 5, 10, 20, 30, 40 or 50%, typically up to 60%, more typically up to 70%, preferably up to 80%, more preferably up to 90%, even more preferably up to 95%, yet more preferably up to 99% of the complete sequence of the full mature PDI protein. Particularly preferred fragments of PDI protein comprise one or more whole domains of the desired protein.<br>
A   fragment  or  variant  of PDI  may  be  a protein  that,  when  expressed<br>
recombinantly in a host cell, can complement the deletion of the endogenously<br>
encoded PDI gene in the host cell, such as S. cerevi.si.ae. and may. for example, be a natural]) occurring hornolog of PDI. such as a homolog encoded by another organism, such as another yeast or other fungi, or another eukaryote such as a human or other vertebrate, or animal or by a plant.<br>
Another preferred chaperone is SSA1 or a fragment or variant thereof having an equivalent chaperone-llke activity. SSA1, also known as YG100. is located on chromosome I of the 5. cerevisiae genome and is 1.93-kbp in size.<br>
One published protein sequence of SSA1 is as follows:<br>
MSKAVGIDLGTTYSCVAHFAIvlDRVDIIANDQGNRTTPSFVAFTDTERLIGDAAKNQAAMN<br>
PSNTVFDAKRLIGRNFNDPEVQADMKHFPFKLIDVDGKPQIQVEFKGETKNFTPEQISSM<br>
VLGKMKETAESYLGAKVNDAWTVPAyFNDSQRQATKDAGTIAGLNVLRIINEPTAAAIA<br>
YGLDKKGKEEHVLIFDLGGGTFDVSLLF1EDGIFEVKATAGDTHLGGEDFDNRLVNHFIQ<br>
EFKRK1\!KKDLSTNQRALRRLRTACERAKRTLSSSAQTSVEIDSLFEGIDFYTSITRARFE<br>
ELCADLFRSTLDPVEKVLRDAKLDKSQVDEIVLVGGSTRIPKVQKLVTDYFNGKEPNRSI<br>
NPDEAVAYGAAVQAAILTGDESSKTQDLLLLDVAPLSLGIETAGGVMTKLIPRNSTISTK<br>
KFEIFSTYADNQPGVLIQVFEGERAKTKDNNLLGKFELSGIPPAPRGVPQIEVTFDVDSN<br>
GILNVSAVEKGTGKSNKITITNDKGRLSKEDIEKMVAEAEKFKEEDEKESQRIASKNQLE<br>
SIAYSLKNTISEAGDKLEQADKDTVTKKAEETISWLDSNTTASKEEFDDKLKELQDIANP<br>
IMSKLYQAGGAPGGAAGGAPGGFPGGAPPAPEAEGPTVEEVD<br>
A published coding sequence for SSA1 is as follows, although it will he appreciated that the sequence can be modified by degenerate substitutions to obtain alternative nucleotide sequences which encode an identical protein product:<br>
ATGTCAAAAGCTGTCGGTATTGATTTAGGTACAACATACTCGTGTGTTGCTCACTTTGCT AATGATCGTGTGGACATTATTGCCAACGATCAAGGTAACAGAACCACTCCATCTTTTGTC GCTTTCACTGACACTGAAAGATTGATTGGTGATGCTGCTAAGAATCAAGCTGCTATGAAT CCTTCGAATACCGTTTTCGACGCTAAGCGTTTGATCGGTAGAAACTTCAACGACCCAGAA GTGCAGGCTGACATGAAGCACTTCCCATTCAAGTTGATCGATGTTGACGGTAAGCCTCAA ATTCAAGTTGAATTTAAGGGTGAAACCAAGAACTTTACCCCAGAACAAATCTCCTCCATG GTCTTGGGTAAGATGAAGGAAACTGCCGAATCTTACTTGGGAGCCAAGGTCAATGACGCT<br>
GTCGTCACTGTCCCAGCTTACTTCAACGATTCTCAAAGACAAGCTACCAAGGATGCTGGT<br>
ACCATTGCTGGTTTGAATGTCTTGCGTATTATTAACGAACCTACCGCCGCTGCCATTGCT<br>
TACGGTTTGGACAAGAAGGGTAAGGAAGAACACGTCTTGATTTTCGACTTGGGTGGTGGT<br>
ACTTTCGATGTCTCTTTGTTGTTCATTGAAGACGGTATCTTTGAAGTTAAGGCCACCGCT<br>
GGTGACACCCATTTGGGTGGTGAAGATTTTGACAACAGATTGGTCAACCACTTCATCCAA<br>
GAATTCAAGAGAAAGAACAAGAAGGACTTGTCTACCAACCAAAGAGCTTTGAGAAGATTA<br>
AGAACCGCTTGTGAAAGAGCCAAGAGAACTTTGTCTTCCTCCGCTCAAACTTCCGTTGAA<br>
ATTGACTCTTTGTTCGAAGGTATCGATTTCTACACTTCCATCACCAGAGCCAGATTCGAA<br>
GAATTGTGTGCTGACTTGTTCAGATCTACTTTGGACCCAGTTGAAAAGGTCTTGAGAGAT<br>
GCTAAATTGGACAAATCTCAAGTCGATGAAATTGTCTTGGTCGGTGGTTCTACCAGAATT<br>
CCAAAGGTCCAAAAATTGGTCACTGACTACTTCAACGGTAAGGAACCAAACAGATCTATC<br>
AACCCAGATGAAGCTGTTGCTTACGGTGCTGCTGTTCAAGCTGCTATTTTGACTGGTGAC<br>
GAATCTTCCAAGACTCAAGATCTATTGTTGTTGGATGTCGGTCCATTATCCTTGGGTATT<br>
GAAACTGCTGGTGGTGTCATGACCAAGTTGATTCCAAGAAACTCTACCATTTCAACAAAG<br>
AAGTTCGAGATCTTTTCCACTTATGCTGATAACCAACCAGGTGTCTTGATTCAAGTCTTT<br>
GAAGGTGAAAGAGCCAAGACTAAGGACAACAACTTGTTGGGrAAGTTCGAATTGAGTGGT<br>
ATTCCACCAGCTCGAAGAGGTGTCCCACAAATTGAAGTCACTTTCGATGTCGACTCTAAC<br>
GGTATTTTGAATGTTTCCGCCGTCGAAAAGGGTACTGGTAAGTCTAACAAGATCACTATT<br>
ACCAACGACAAGGGTAGATTGTCCAAGGAAGATATCGAAAAGATGGTTGCTGAAGCCGAA<br>
AAATTCAAGGAAGAAGATGAAAAGGAATCTCAAAGAATTGCTTCCAAGAACCAATTGGAA<br>
TCCATTGCTTACTCTTTGAAGAACACCATTTCTGAAGCTGGTGACAAATTGGAACAAGCT<br>
GACAAGGACACCGTCACCAAGAAGGCTGAAGAGACTATTTCTTGGTTAGACAGCAACACC<br>
ACTGCCAGCAAGGAAGAATTCGATGACAAGTTGAAGGAGTTGCAAGACATTGCCAACCCA<br>
ATCATGTCTAAGTTGTACCAAGCTGGTGGTGCTCCAGGTGGCGCTGCAGGTGGTGCTCCA<br>
GGCGGTTTCCCAGGTGGTGCTCCTCCAGCTCCAGAGGCTGAAGGTCCAACCGTTGAAGAA<br>
GTTGATTAA<br>
The protein Ssalp belongs to the Hsp70 family of proteins and is resident in the cytosol. Hsp70s possess the ability to perform a number of chaperone activities; aiding protein synthesis, assembly and folding; mediating translocation of polypeptides to various intracellular locations, and resolution of protein aggregates (Becker &amp; Craig, 1994, Eur. J. Biochem. 219, 11-23). Hsp70 genes are higlily conserved, possessing an N-terminal ATP-binding domain and a C-terminal peptide-binding domain. Hsp70 proterns interact with the peptide backbone of, mainly unfolded, proteins. The binding and release of peptides by<br>
hsp70   proteins   is    an   ATP-dependent   process    and    accompanied    by    a conformationa] change in the hsp70 (Becker &amp; Craig. 1994. supra).<br>
C,vtosolic hsp70 proteins are particular]}' involved in the sjoithesis, folding arid secretion of proteins (Becker &amp; Craig. 1994., supra).   In S. cerevisiae cytosolic hsp70 proteins have been divided into two groups; SSA (SSA 1-4) and SSB (SSB 1 and 2) proteins, which are functionally distinct from each other.   The SSA family is essential in that at least one protein from the group must be active to maintain cell viability (Becker &amp; Craig. 1994, supra).   Cytosolic hsp70 _ proteins bind preferentially to unfolded and not mature proteins.   This suggests that they prevent the aggregation of precursor proteins, by maintaining them hi an unfolded state prior to being assembled into multimolecular complexes in the cytosol and/or facilitating their translocation to various organelles (Becker  &amp; Craig,  1994, supra). SSA proteins are particularly involved in posttranslational biogenesis and maintenance of precursors for translocation into the endoplasmic reticulum and mitochondria (Kim et al, 1998, Proc. Natl. Acad. Scl USA. 95, 12860-12865; Ngosuwan et aL 2003, J. Biol Chem. 278 (9), 7034-7042). Ssalp has been shown to bind damaged proteins, stabilising them in a partially unfolded form and allowing refolding or degradation to occur (Becker &amp; Craig, 1994, supra; Glover &amp; Lindquist, 1998, Cell. 94, 73-82).<br>
Demolder et al, 1994, J. Biotechnol, 32, 179-189 reported that over-expression of SSA1   hi yeast provided for increases  in the  expression of a recombinant chrornosomally integrated gene encoding human interferon-p.     There  is no suggestion that increases hi heterologous gene expression could be achieved if SSA1 and human interferon-f3 were to be encoded by recombinant genes on the same plasmid. hi fact, in light of more recent developments in the field of over-expression of chaperones in yeast (e.g. Robinson et al, 1994, op. cit; Hayano et al, 1995, op. dt; Shusta et al, 1998, op. cit; Parekh &amp; Wittrup, 1997, op. cit; Bao &amp; Fulculiara, 2001, op. cit.; and Bao et al, 2000, op. cit) the skilled person would<br>
have been disinclined to express SSAl from a 2um-family plasmid at all, much Jess to express both SSAl and a heterologous protein from a 2jm-family plasmid in order to increase the expression levels of a heterologous protein.<br>
Variants and fragments of SSAl are also included in the present invention.   A<br>
"variant", in the context of SSAl, refers to a protein having the sequence of native<br>
SSAl  other than at one or more positions where there have been amino acid<br>
insertions,  deletions,  or substitutions,  either conservative or non-conservative,<br>
provided that such changes result in a protein whose basic properties, for example<br>
enzymatic activity (type of and specific activity), thermostability, activity in a certain<br>
pH-range (pH-stability) have not significantly been changed. "Significantly" in this<br>
context means that one skilled in the art would say that the properties of the variant<br>
may still be different but would not be unobvious over the ones of the original<br>
protein.<br>
By "conservative substitutions" is intended combinations such as Val, De, Leu, Ala, Met; Asp, Glu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Tip. Preferred conservative substitutions include Gly, Ala; Val, lie, Leu; Asp, Glu; Asn, Gin; Ser, Thr; Lys, Arg; and Phe, Tyr.<br>
A "variant" of SSAl typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably at least 80%, more preferably at least 90%, even more preferably at least 95%, yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the sequence of native SSAl.<br>
The percent sequence identity between two polypeptides may be determined using suitable computer programs, as discussed below. Such variants may be natural or made using the methods of protein engineering and site-directed mutagenesis as are well known hi the ait.<br>
A "fragment", m the context of SSAl. refers to a protein having the sequence of native SSAl other than for a1 one or more positions where there have been deletions. Thus the fragment may comprise at most 5, 10, 20, 30, 40 or 50%, typically up to 60%, more typically up to 70%, preferably up to 80%, more preferably up to 90%. even more preferably up to 95%, yet more preferably up to 99% of the complete sequence of the full mature SSAl protein. Particularly preferred fragments of SSAl protein comprise one or more whole domains of the desired protein.<br>
A fragment or variant of SSAl may be a protein that, when expressed recombinantly in a host cell, such as S. cerevisiae, can complement the deletion of the endogenously encoded SSAl gene (or homolog thereof) in the host cell and may, for example, be a naturally occurring homolog of SSAl, such as a homolog encoded by another organism, such as another yeast or other fungi, or another eukaryote such as a human or other vertebrate, or animal or by a plant.<br>
Another preferred chaperone is PSE1 or a fragment or variant thereof having equivalent chaperone-lilce activity.<br>
PSE1, also known as KAP121, is an essential gene, located on chromosome XIII. A published protein sequence for the protein pselp is as follows:<br>
MSALPEEVNRTLLQIVQAFASPDNQIRSVAEKALSEEWITENNIEYLLTFLAEQAAFSQD<br>
TTVAALSAVLFRKLALKAPPSSKLMIMSKNITHIRKEVLAQIRSSLLKGFLSERADSIRH<br>
KLSDAIAECVQDDLPAWPELLQALIESLKSGNPNFRESSFRILTTVPYLITAVDINSILP<br>
IFQSGFTDASDNVKIAAVTAFVGYFKQLPKSEWSKLGILLPSLLNSLPRFLDDGKDDALA<br>
SVFESLIELVELAPKLFKDMFDQIIQFTDMVIKNKDLEPPARTTALELLTVFSENAPQMC<br>
KSNQNYGQTLVMVTLIMMTEVSIDDDDAAEWIESDDTDDEEEVTYDHARQALDRVALKLG<br>
GEYUPLFQYLQQMITSTEWRERFAAMMALSSAAEGCADVLIGEIPKILDMVIPLINDP<br>
HPRVQYGCCNVLGQISTDFSPFIQRTAHDRILPALISKLTSECTSRVQTHAAAALVNFSE<br>
FASKDILEPYLDSLLTNLLVLLQSNKLYVQEQALTTIAFIAEAAKNKFIKYYDTLMPLLL<br>
NVLKVNNKDNSVLKGKCMECATLIGFAVGKEKFHEHSQELISILVALQNSDIDEDDALRS<br>
YLEQSWSRICRILGDDFVPLLPIVIPPLLITAKATQDVGLIEEEEAANFQQYPDWDVVQV<br>
QGKHIAIHTSVLDDKVSAMELLQSYATLLRGQFAVYVKEVMEEIALPSLDFYLHDGVRAA<br>
GATLIPILLSCLLAATGTQNEELVLLWHKASSKLIGGLMSEPMPEITQVYHNSLVNGIKV<br>
MGDNCLSEDQLAAFTKGVSANLTDTYERMQDRHGDGDEYNENIDEEEDFTDEDLLDEINK<br>
SIAAVLKTTNGHYLKNLENIWPMINTFLLDNEPILVIFALWIGDLIQYGGEQTASMKNA<br>
FIPKVTECLIS PDARIRQAASYIIGVCAQYAPSTYADVCIPTLDTLVQIVDFPGSKLEEN<br>
RSSTENASAAIAKILYAYNSNIPNVDTYTANWFKTLPTITDKEAASFNYQFLSQLIENNS<br>
PIVCAQSNISAWDSVIQALNERSLTEREGQTVISSVKKLLGFLPSSDAMAIFNRYPADI<br>
MEKVHKWFA*<br>
A published nucleotide coding sequence of PSE1 is as follows, although it will be appreciated that the sequence can be modified by degenerate substitutions to obtain alternative nucleotide sequences which encode an identical protein product:<br>
ATGTCTGCTTTACCGGAAGAAGTTAATAGAACATTACTTCAGATTGTCCAGGCGTTTGCT<br>
TCCCCTGACAATCAAATACGTTCTGTAGCTGAGAAGGCTCTTAGTGAAGAATGGATTACC<br>
GAAAACAATATTGAGTATCTTTTAACTTTTTTGGCTGAACAAGCCGCTTTCTCCCAAGAT<br>
ACAACAGTTGCAGCATTATCTGCTGTTCTGTTTAGAAAATTA.GCATTAAAAGCTCCCCCT<br>
TCTTCGAAGCTTATGATTATGTCCAAAAATATCACACATATTAGGAAAGAAGTTCTTGCA<br>
CAAATTCGTTCTTCATTGTTAAAAGGGTTTTTGTCGGAAAGAGCTGATTCAATTAGGCAC<br>
AAACTATCTGATGCTATTGCTGAGTGTGTTCAAGACGACTTACCAGCATGGCCAGAATTA<br>
CTACAAGCTTTAATAGAGTCTTTAAAAAGCGGTAACCCAAATTTTAGAGAATCCAGTTTT<br>
AGAATTTTGACGACTGTACCTTATTTAATTACCGCTGTTGACATCAACAGTATCTTACCA<br>
ATTTTTCAATCAGGCTTTACTGATGCAAGTGATAATGTCAAAATTGCTGCAGTTACGGCT<br>
TTCGTGGGTTATTTTAAGCAACTACCAAAATCTGAGTGGTCCAAGTTAGGTATTTTATTA<br>
CCAAGTCTTTTGAATAGTTTACCAAGATTTTTAGATGATGGTAAGGACGATGCCCTTGCA<br>
TCAGTTTTTGAATCGTTAATTGAGTTGGTGGAATTGGCACCAAAACTATTCAAGGATATG<br>
TTTGACCAAATAATACAATTCACTGATATGGTTATAAAAAATAAGGATTTAGAACCTCCA<br>
GCAAGAACCACAGCACTCGAACTGCTAACCGTTTTCAGCGAGAACGCTCCCCAAATGTGT<br>
AAATCGAACCAGAATTACGGGCAAACTTTAGTGATGGTTACTTTAATCATGATGACGGAG<br>
GTATCCATAGATGATGATGATGCAGCAGAATGGATAGAATCTGACGATACCGATGATGAA<br>
GAGGAAGTTACATATGACCACGCTCGTCAAGCTCTTGATCGT'GTTGCTTTAAAGCTGGGT<br>
GGTGAATATTTGGCTGCACCATTGTTCCAATATTTACAGCAAATGATCACATCAACCGAA TGGAGAGAAAGATTCGCGGCCATGATGGCACTTTCCTCTGCAGCTGAGGGTTGTGCTGAT GTTCTGATCGGCGAGATCCCAAAAATCCTGGATATGGTAATTCCCCTCATCAACGATCCT CATCCAAGAGTACAGTATGGATGTTGTAATGTTTTGGGTCAAATATCTACTGATTTTTCA<br>
T'TATTACYLGCAACAAACTTrACGlACAGGAACAGGCCCT/iACAACCATTGCATTTATT<br>
GCTGAAGCTGCAAAGAATAAATTTATCAAGTATuACGATACTCTAATGCCATTATTATTA<br>
AATGTTTTGAAGETTARCAATAAAGATAATAGTGTTTTSAAAGGTAAATGTATGGAATGT<br>
GCAftCTCTGATTGGTTrTGCCGTTGGTAAGGAAA.TTTCATGAGCACTCTCAAGAGCTG<br>
ATTT'CTATATTGGTCGCTTTACAA/lACTCAGATATCGATGAAGATGATGCGCTCAGAiT.CA<br>
TACTTAGAACAAAGTTGGAGCAGGATTTGCCGAATTCTGGGTGATGATTTTGTTCCGTTG<br>
TTACCGATl'G'rTATACCACCCCTGCTAATTACTGCCAAAGCAACGCAAGACGTCGGTTTA<br>
ATTGAAGAAG-AGAAfiCAGCAAATTTCCAACAATATCCAGATTGGGATGrTGTTCAAGTT<br>
CAGGGA/iiuACACATTGCTATTCACACATCCGTCCTTGACGATAAAGTATCAGCAATGGAG<br>
CTATTACAAAGCTATGCGACACTTTTAAGAGGCCAATTTGCTGTATATGTTAAAGAAGTA<br>
ATGGAAGA:wrAGC'r;CTACCTCGCTTGACTTTrACCTACATGACGGTGTTCGTGCTGCA<br>
GGAGCAACTTTAATTCCTATTCTATTATCTTGTTTACTTGCAGCCACCGGTACTCAAAAC<br>
GAGGAATTGGTATTGTTGTGGCATAAAGCTrCGTCTAAACTAATCGGAGGCTTAATGTCA<br>
GAACCAftTGCCAGAAATCACGCAAGTTTATCACAACTCGTTAGTGAATGGTATTAAAGTC<br>
ATGGGTGACAATTGCTTAAGCGAAGACCAATTAGCGGCATTTACTAAGGGTGTCTCCGCC<br>
AACTTAACTGACACTTACGAAAGGATGCAGGATCGCCATGGTGATGGTGATGAATATAAT<br>
GAAAATATTGATGA?iGAGGAAGACTTTACTGACGAAGATCTrCTCGATGAAATCAACA.AG<br>
TCTATlGCGGCCGTTTTGAAAACCACAAATGGTCATTATCI'AAAGAATTTGGAGAATATA<br>
TGGCrTAT&amp;ATAAACACATTCCTTTTAGATAATGAACCAATTTTAGrCATTTTTGCATTA<br>
GTAGl'GA'rTGGTGACTTGATTCAATATGGTGGCGAACAAACTGCTAGCATSAAGAACGCA<br>
TTTATTCCAAAGGTTACCGAGTGCTTGATTTCTCGTGACGCTCGTATTCGCCAAGCTGCT<br>
TCTrAT.ATAATCGGTGTTTGTGCCCAATACGCTCCATCTACATATGCTGACGTTTGCATA<br>
CCSACTTTAGATACACTTGTTCAGATTGTCGATTTTCCAGGCTCCAAACrGGjyiGAAAAT<br>
CGTTCTTCAACAGAGAATGCCAGTGCAGCCATCGCCAAAATrCTTTATGCATACAATTCC<br>
ACATTCCTAACGTAGACACGTACACGGCTAATTGGTTCAAAACGTrACCAACAZLTAACT<br>
GACA7sAGAGCTGCCTCATTCAACTATCMTTTTTGAGTCAATTGATTGAAAATAATTCG<br>
CCAJiTTGTGTGTGCTCAATCTAATATCTCCGCTGTAGTTGATTCAGTCATACAAGCCTTG<br>
PJiTGAGAGAAGT'rTGACCGAAAGGGAAGGCCAAACGGTGATAAGTTCAGTTAAAAAGTTG<br>
TrGGGATTTTTGCCTTCTAGTGATGCTATGGCAATTTTCAATAGATATCCAGCTGATATT<br>
ATGGAGAAGTACATAAATGGTTTGCATAA<br>
The PSE] gene is 3,25-kbp in size.  Pselp is involved in the nucleocytoplasmic transport of macromojecules (Seedorf &amp; Silver, 1997, Proc. Nat!. Acad. Sci. USA.<br>
94,  8590-8595).    This process occurs via the nuclear pore complex (NPC) embedded in the nuclear envelope and made up of nucleoporins (Ryan &amp; Wente, 2000, Cwr. Opin. Cell Biol 12, .361-371).   Proteins possess specific sequences that contain the information required for nuclear import, nuclear localisation sequence (NLS) and export, nuclear export sequence (NES) (Pemberton et al., 1998, Cwr. Opin. Cell Biol. 10, 392-399).   Pselp is a karyopherin/importin, a group   of proteins,  which have  been  divided  up  into  a   and   P  families. Karyopherins   are  soluble  transport  factors  that  mediate  the  transport   of macromolecules across the nuclear membrane by recognising NLS and NES, and interact with and the NPC (Seedorf &amp; Silver, 1997, supra; Pemberton et al, 1998, supra; Ryan &amp; Wente, 2000, supra).   Translocation through the nuclear pore is driven by GTP hydrolysis, catalysed by the small GTP-binding protein, Ran (Seedorf &amp; Silver, 1997, supra). Pselp has been identified as a karyopherin p. 14 karyopherin (3 proteins have been identified in S. cerevisiae, of which only 4 are essential.    This is perhaps because multiple karyopherins may mediate the transport of a single macromolecule (Isoyama et al., 2001, J. Biol. Chem. 276 (24), 21863-21869).   Pselp is localised to the nucleus, at the nuclear envelope, and to a certain extent to the cytoplasm.  This suggests the protein moves in and out of the nucleus as part of its transport function (Seedorf &amp; Silver, 1997, supra). Pselp is involved in the nuclear import of transcription factors (Isoyama et al., 2001, supra; Ueta et al., 2003, J. Biol. Chem. 278 (50), 50120-50127), histones (Mosammaparast et al., 2002, J. Biol. Chem. 277 (1), 862-868), and ribosomal proteins prior to their assembly into ribosomes (Pemberton et al., 1998, supra). It also mediates the export of mRNA from the nucleus. Karyopherins recognise and bind distinct NES found on RNA-binding proteins, which coat the RNA before it is exported from the nucleus (Seedorf &amp; Silver, 1997, Pemberton et al, 1998, supra).<br>
As nucleocytoplasmic transport of macromolecules is essential for proper progression through the cell cycle, nuclear transport factors, such as pselp are novel candidate targets for growth control (Seedorf&amp; Silver, 1997. supra).<br>
Overexpression of Pselp (protein secretion enhancer) in S. cerevisiae has also been shown to increase endogenous protein secretion levels of a repertoire of biologically active proteins (Chow el al., 1992; J. Cell. Sci. 101 (3), 709-719). There is no suggestion that increases in heterologous gene expression could be achieved if PSE1 and  a heterologous protein were both to be encoded by recombinanl genes on the same plasmid.     In fact, in light of more recent developments in the over-expression of chaperones in yeast (e.g. Robinson et al, 1994, op. OIL; Hayano et al, 1995, op. cit.; Shusta et al, 1998, op. cil; Parelch &amp; Wittrup, 1997, op. cit.; Bao &amp; Fulcuhara, 2001, op. cit.; and Bao et al, 2000, op. cil ) the skilled person would not have attempted to over-express PSE1 from a 2urn-family plasmid at all, much less to express both PSE1 and a heterologous protein from a 2 urn-family plasmid in order to increase the expression levels of a heterologous protein.<br>
Variants and fragments of PSE1 are also included in the present invention.   A '"vaiiant", in the context of PSE1, refers to a protein having the sequence of native PSE1 other than for at one or more positions where there have been amino acid insertions,  deletions,  or substitutions,  either conservative or non-conservative, provided that such changes result hi a protein whose basic properties, for example enzymatic activity (type of and specific activity), therrnostability, activity in a certain pH-range (pH-stabflity) have not significantly been changed. "Significantly" in this context means that one skilled hi the art would say that the properties of the variant may still be different but would not be unobvious over the ones of the original protein.<br>
By "conservative substitutions" is intended combinations such as Val, He, Leu, Ala; Met; Asp, Glu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Trp. Preferred conservative substitutions include Gly, Ala; Val, He, Leu; Asp, Glu; Asn, Gin; Ser, Thr; Lys, Arg; and Phe, Tyr.<br>
A "variant" of PSEl typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably at least 80%, more preferably at least 90%, even more preferably at least 95%, yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the sequence of native PSEl.<br>
The percent sequence identity between two polypeptides may be determined using suitable computer programs, as discussed below. Such variants may be natural or made using the methods of protein engineering and site-directed mutagenesis as are well known in the art.<br>
A "fragment", in the context of PSEl, refers to a protein having the sequence of native PSEl other than for at one or more positions where there have been deletions. Thus the fragment may comprise at most 5, 10, 20, 30, 40 or 50%, typically up to 60%, more typically up to 70%, preferably up to 80%, more preferably up to 90%, even more preferably up to 95%, yet more preferably up to 99% of the complete sequence of the full mature PSEl protein. Particularly preferred fragments of PSEl protein comprise one or more whole domains of the desired protein.<br>
A fragment or variant of PSEl may be a protein that, when expressed recombinaritly in a host cell, such as S. cerevisiae, can complement the deletion of the endogenous PSEl gene in the host cell.and may, for example, be a naturally occurring homo log of PSEl, such as a homolog encoded by another organism, such as another yeast or other fungi, or another eukaryote such as a human or other vertebrate, or animal or by a plant.<br>
Another preferred chaperone is OK.M2 or a fragment or variant thereof having equivalent chaperone-like activity.<br>
OKM2., also known as YLR350W, is located on cliromosome XII (positions K28729 to 829379) of the S. cerevisiae genome and encodes an evoiutionarily conserved protein will) similarity to the yeast protein Ormlp.   Hjelmqvist er a!, 2002, Genome Biology, 3(6), research 0027.1-0027.16 reports that ORM2 belongs to  gene family  comprising three  human  genes   (OEMDL1,   ORMDL2   and ORMDL3)   as   well   as   homologs   in   inicrosporidia,   plants,   Drosophila, urochordates and vertebrates.     The  ORIvfDL genes are reported to  encode transmembrane proteins anchored in the proteins endoplasrnic reticulum (ER).<br>
The protein Orm2p is required for resistance to agents that induce the unfolded protein response. Hjeimqvist et al, 2002 (supra] reported that a double knockout of the two S. cerevisiae GRMDL homologs (OKhdl and ORM2) leads to a decreased growth rate and greater sensitivity to tunicamycin and ditbiothreitol.<br>
One published sequence of Orm2p is as follows:<br>
MIDRTKWESPAFEESPLTPNVSNLKPFPSQSNKISTPVTDHRRRRSSSVISHVEQETFED ENDQQMLPNMKATW/DQRG/lWLIHIXIVLLRLFySLFGSTPKWTWTLTNMTYIIGFYIM FHLVKGTPFDFNGGAYDNLTMWEQINDETLYTPTRKFLLIVPJVLFLISNQYYRNDMTLF LSNLAVTX'LIGWPKLGITHRLRISIPGITGRAQIS*<br>
The above protein is encoded in S. cerevisiae by the following coding nucleotide sequence, although it will be appreciated that the sequence can be modified by degenerate substitutions to obtain alternative nucleotide sequences which encode an identical protein product:<br>
ATGATTGACCGC.ACTAAAAACGAATCTCCAGCTTTTGAAGAGTCTCCGCTTACCCCCAAT GTGTCTMCCTGAMCCATTCCCTTCTCAAAGCAACAAMTATCCACTCCAGTGACCGAC CATAGGAGMGACGGTCATCCA6CGTAATATCACATGTGGAACAGGAAACCTTCGAAGAC<br>
GfiA?kA'l1GAC'CAGCAGATGCl-ruuu.'iAUAl'CaAACGCTACGTGGGTCGACCAGCG-AGGCGCG TGGTTGATTCATATC6TCGTAATAGTACTCTTGAGGCTCTTCTACTCCTTGTTCGGGTCG ACGCCCAAATGGACGTGGACTTTAACAAACATGACCTACATCATCGGATTCTATATCATG TTCCACCTTGTCAAAGGTACGCCCTTCGACTTTAACGGTGGTGCGTACGACAACCTGACC ATGIGGGAGCAGATTAACGATGAGACTTTCTACACACCCACTAGAAAATTTCTGCTGArr GTACCCATTGTGTTGTTCCTGATTAGCAACCAGTACTACCGCAACGACATGACACTArrC CTCTCCAACCTCGCCGTGACGGTGCTTATTGGTGTCGTTCCTAAGCTGGGAATTACGCAT AGACTAAGAATATCCATCCCTGGTATTACGGGCCGTGCTCAAATTAGTTAG<br>
Variants and fragments of ORM2 are also included in the present invention.   A<br>
"variant", in the context of ORM2, refers to a protein having the sequence of native<br>
ORM2 other than for at one or more positions where there have been amino acid<br>
insertions,  deletions, or substitutions, either conservative or non-conservative,<br>
provided that such changes result in a protein whose basic properties, for example<br>
enzymatic activity (type of and specific activity), thermostability, activity in a certain<br>
pH-range (pH~.
context means that one skilled in the art would say that the properties of the variant<br>
may still be different but would not be unobvious over the ones of the original<br>
protein.<br>
By "conservative substitutions" is intended combinations such as Val, De, Leu, Ala, Met; Asp, Glu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Trp. Preferred conservative substitutions include Gly, Ala; Val, lie, Leu; Asp, Glu; Asn, Gin; Ser, 7hr; Lys, Arg; and Phe, Tyr.<br>
A "variant" of O.RM2 typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably at least 80%, more preferably at least 90%, even more preferably at least 95%, yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the sequence of native ORM2.<br>
The percent sequence identity between two polypeptides may be determined using suitable computer programs, as discussed below.  Such variants may be natural or<br>
made using the methods of protein, engineering and site-directed rnutagenesis as are well known in the art<br>
A "fragment', in the context of ORM2, refers to a protein having the sequence of native ORM2 other than for at one or more positions where there have been deletions. Thus the fragment may comprise at most 5, 10, 20, 30, 40 or 50%, typically up to 60%, more typically up to 70%, preferably up to 80%, more preferably up to 90%, even more preferably up to 95%. yet more preferably up to 99% of the complete sequence of the mil mature ORM2 protein. Particularly preferred fragments of ORM2 protein comprise one or more whole domains of the desired protein.<br>
A fragment or variant of ORM2 may be a protein that, when expressed recombinantly in a host cell, such as 5". cerevisiae, can complement the deletion of the endogenous ORM2 gene in the host cell and may, for example, be a naturally occurring homolog of ORM2, such as a homolog encoded by another organism, such as another yeast or other fungi, or another eukaryote such as a human or other vertebrate, or animal or by a plant.<br>
A gene encoding a protein comprising the sequence of a chaperone may be formed in a like manner to that discussed below for genes encoding heterologous proteins, with particular emphasis on combinations of ORFs and regulatory regions.<br>
The term "protein" as used herein includes all natural and non-natural proteins, polypeptides and peptides. A "heterologous protein" is a protein that is not naturally encoded by a 2 Jim-family plasmid and can also be described as a "non 2urQ-famiJ)' plasmid protein". For convenience, the terms "heterologous protein" and "non 2pm-family plasmid protein" are used synorrymously throughout this application. Preferably, therefore, the heterologous protein is not a FLP, REP1, REP2, or a RAF/D protein as encoded by any one of pSRl, pSB3 or pSB4 as.<br>
obtained from Z roitxii, pSBl or pSB2 both as obtained from Z. bailli, pSMl as obtained from Z. fermentati, pKDl as obtained from K. drosophilarum, pPMl as obtained from P. membranaefaciem or the 2jJ.m plasmid as obtained from S. cerevisiae.<br>
A gene encoding a heterologous protein comprises polynucleotide sequence encoding the heterologous protein (typically according to standard codon usage for any given organism), designated the open reading frame ("ORF"). The gene may additionally comprise some polynucleotide sequence that does not encode an open reading frame (termed "non-coding region").<br>
Non-coding region in the gene may contain one or more regulator}' sequences, operatively linked to the OKF, which allow for the transcription of the open reading frame and/or translation of the resultant transcript.<br>
The term "regulatory sequence" refers to a sequence that modulates (i.e., promotes or reduces) the expression (i.e., the transcription and/or translation) of an ORF to which it is operably linked. Regulatory regions typically include promoters, terminators, ribosome binding sites and the like. The skilled person will appreciate that the choice of regulatory region will depend upon the intended expression system. For example, promoters may be constitutive or inducible and may be cell- or tissue-type specific or non-specific.<br>
Suitable regulatory regions, may be 5bp, lObp, 15bp, 20bp, 25bp, 30bp, 35bp, 40bp, 45bp, 50bp, 60bp, 70bp, 80bp, 90bp, lOObp, 120bp, 140bp, 160bp, 180bp, 200bp, 220bp, 240bp, 260bp, 280bp, SOObp, 350bp, 400bp, 450bp, SOObp, 550bp, 600bp, 650bp, 700bp, 750bp, SOObp, 850bp, 900bp, 950bp, lOOObp, HOObp, 1200bp, 1 SOObp, 1400bp, 1 SOObp or greater, in length.<br>
Those skilled m the ail will recognise that the gene encoding the chaperone. for example PDI. may additional]}- comprise non-coding regions and/or regulatory regions- Such non-coding regions and regulatory regions are not restricted to the native non-coding regions and/or regulatory regions normally associated with the chaperone ORF.<br>
Where the expression system is yeast, such as Saccharomyces cerevisiae, suitable promoters for S. cerevisiae include those associated with the PGK1 gene, GAL1 or GAL10 genes, TEFJ, TEF2, PYK1, PMA1, CYC1, PH05, TRP1, ADH1, ADH2, the genes for glyceraldehyde-3-phosphate dehydrogenase, hexokinase, pyruvate decarboxylase, phosphofractoldnase, triose phosphate isomerase, phosphoglucose isomerase, glucokinase. a-mating factor pheromone, a-mating factor pheromone, the PRBJ  promoter, the PRA1  promoter, the  GPD1  promoter,  and hybrid promoters involving hybrids of parts of 5' regulatory regions with parts of 5' regulator)' regions of other promoters or with upstream activation sites (e.g. the promoter of EP-A-258 067).<br>
Suitable transcription termination signals are well known in the art. Where the host celJ is eukaryotic, the transcription termination signal is preferably derived from the 3' flanking sequence of a eukaryotic gene, which contains proper signals for transcription termination and polyadenylation. Suitable 3' flanking sequences may, for example, be those of the gene naturally linked to the expression control sequence used, i.e. ma)' correspond to the promoter. Alternatively, they may be different. In that case, and where the host is a yeast, preferably S. cerevisiae, then the termination signal of the S. cerevisiae ADH1, ADH2, CYC1, or PGK1 genes are preferred.<br>
It may be beneficial for the promoter and open reading frame of the heterologous gene, such as the those of the chaperone PDI1, to be flanked by transcription termination sequences so that the transcription termination sequences are located<br>
both upstream and downstream of the promoter and open reading frame, in order to prevent transcriptional read-through into neighbouring genes, such as 2um genes, arid visa versa.<br>
In one embodiment, the favoured regulatory sequences in yeast, such as Saccharomyces cerevisiae, include: a yeast promoter (e.g. the Saccharomyces cerevisiae PRB1 promoter), as taught in EP 431 880; and a transcription terminator, preferably the terminator from Saccharomyces ADH1, as taught in EP 60 057. Preferably, the vector incorporates at least two translation stop codons.<br>
It may be beneficial for the non-coding region to incorporate more than one DNA sequence encoding a translational stop codon, such as UAA, UAG or UGA, in order to minimise translational read-through and thus avoid the production of elongated, non-natural fusion proteins. The translation stop codon. UAA is preferred.<br>
The term "operably linked" includes within its meaning that a regulatory sequence is positioned within any non-coding region in a gene such that it forms a relationship with an ORF that permits the regulatory region to exert an effect on the ORF in its intended manner. Thus a regulatory region "operably linked" to an ORF is positioned in such a way that the regulatory region is able to influence transcription and/or translation of the ORF in the intended manner, under conditions compatible with the regulatory sequence.<br>
In one preferred embodiment, the heterologous protein is secreted, hi that case, a sequence encoding a secretion leader sequence which, for example, comprises most of the natural HS A secretion leader, plus a small portion of the S. cerevisiae cc-mating factor secretion leader as taught in WO 90/01063 may be included in the open reading frame.<br>
Alternatively, the heterologous protein may be intracellular.<br>
In another preferred embodiment, the heterologous protein comprises the sequence of a eukaryotic protein, or a fragment or variant thereof. Suitable eukaryotes include fungi, plants and animals. In one preferred embodiment the heterologous protein is a fungal protein, such as a yeast protein. In another preferred embodiment the heterologous protein is an anirnal protein. Exemplary animals include vertebrates and invertebrates. Exemplary vertebrates include mammals, such as humans, and non-human mammals.<br>
Thus the heterologous protein may comprise the sequence of a yeast protein. It may, for example, comprise the sequence of a yeast protein from the same host from which the 2um-family plasmid is derived. Those skilled in the art will recognise that a method, use or plasmid of the first second or third aspects of the invention may comprise DNA sequences encoding more than one heterologous protein, more man one chaperone, or more than one heterologous protein and more than one chaperone.<br>
In another preferred embodiment, the heterologous protein ma}' comprise the sequence of albumin, a monoclonal antibody, an etoposide, a serum protein (such as a blood clotting factor), antistasin, a tick anticoagulant peptide, transferrin. lactoferrin,      endostatin,      angiostatin,      collagens,      iirrmunoglobulins      or imiminoglobulin-based molecules or fragment of either (e.g. a Small Modular ImmunoPharmaceutical™ ("SMTP") or dAb, Fab' fragments, F(ab')2, scAb, scFv or scFv fragment), a Kunitz domain protein (such as those described in WO 03/066824, with 01 without albumin fusions), interferons, interleukins, IL10, IL11, IL2, interferon a species and sub-species, interferon p species and sub-species, interferon y species  and  sub-species, leptin,  CNTF,  CNTFAxis,  IL1-receptor antagonist, erythropoietin (EPO) and EPO mimics, thrombopoietin (TPO) and TPO mimics, prosaptide, cyanovirin-N, 5-helix, T20 peptide, T1249 peptide, HIV<br>
gp41, HIV gp!20, urokinase, prouroldnase, tPA, hirudin, platelet derived growth, factor, parathyroid hormone, pro-insulin, insulin, glue-agon, glucagon-like peptides, insulin-like growth factor, calcitonin, growth hormone, transforming growth factor P, tumour necrosis factor, G-CSF, GM-CSF, M-CSF, FGF, coagulation factors in both pre and active forms, including but not limited to plasminogen, fibrinogen, thrombin, pre-thrombin, pro-thrombin, von Willebrand's factor, ai-antitrypsin, plasminogen activators, Factor VII, Factor VIII, Factor IX, Factor X and Factor XIII, nerve growth factor, LACI, platelet-derived endothelial cell growth factor (PD-ECGF), glucose oxidase. serum cholinesterase, aprotinin, amyloid precursor protein, inter-alpha trypsin inhibitor, antithrombin III, apo-Iipoprotein species, Protein C, Protein S, or a variant or fragment of any of the above.<br>
A "variant", in the context of the above-listed proteins, refers to a protein wherein at one or more positions there have been arnino acid insertions, deletions, or substitutions, either conservative or non-conservative, provided that such changes result in a protein whose basic properties, for example enzymatic activity or receptor binding (type of and specific activity), thermostabiliry, activity in a certain pH-range (pFI-stability) have not significantly been changed. "Significantly" in this context means that one skilled in the art would say that the properties of the variant may still be different but would not be unobvious over the ones of the original protein.<br>
By "conservative substitutions" is intended combinations such as Val, fle, Leu, Ala, Met; Asp, Glu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Tip. Preferred conservative substitutions include Gly, Ala; Val, lie, Leu; Asp, Glu; Asn, Gin; Ser, Thr; Lys, Arg; and Phe, Tyr.<br>
A "variant" typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably at least 80%, more preferably at least 90%, even more preferably at least 95%, yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the polypeptide from which it is derived.<br>
The percen! sequence identity between two polypeptides may be determined using suitable computer programs, for example the GAP program of the University of Wisconsin Genetic Computing Group and it will be appreciated that percent identity is calculated in relation to polypeptides whose sequence has been aligned optimally.<br>
The alignment may alternatively be carried out using the Clustal W program (Thompson et al., (1994) Nucleic Acids Res., 22(22), 4673-80), The parameters used ma}' be as follows:<br>
•	Fast pairwise alignment parameters: K-tuple(word) size; 1, window size; 5,<br>
gap penalty; 3, number of top diagonals; 5. Scoring method: x percent.<br>
•	Multiple alignment parameters: gap open penalty; 10, gap extension penalty;<br>
0.05.<br>
•	Scoring matrix: BLOSUM.<br>
Such variants may be natural or made using the methods of protein engineering and site-directed mut agenesis as are well known in the art.<br>
A "fragment", in the context of the above-listed proteins, refers to a protein wherein at one or more positions there have been deletions. Thus the fragment may comprise at most 5, 10, 20, 30, 40 or 50% of the complete sequence of the full mature polypeptide. Typically a fragment comprises up to 60%, more typically up to 70%, preferably up to 80%, more preferably up to 90%, even more preferably up to 95%, yet more preferably up to 99% of the complete sequence of the full desired protein. Particularly preferred fragments of a protein comprise one or more whole domains of the protein.<br>
In one particularly preferred embodiment the heterologous protein comprises the sequence of albumin or a variant or fragment thereof.<br>
By "albumin" we include a protein, comprising the sequence of an albumin protein<br>
obtained from any source.  Typically the source is mammalian,  hi one preferred<br>
embodiment the serum albumin is human serum albumin ("HSA").   The term<br>
"human serum albumin" includes the meaning of a serum albumin having an<br>
arnino  acid  sequence naturally  occurring in humans,  and variants thereof.<br>
Preferably the albumin has the arnrno acid sequence disclosed in WO 90/13653 or<br>
a variant thereof. The HSA coding sequence is obtainable by known methods for<br>
isolating cDNA corresponding to human genes, and is also disclosed in, for<br>
example, EP 73 646 and EP 286 424.<br>
hi another preferred embodiment the "albiirnin" comprises the sequence of bovine serum albumin. The term "bovine serum albumin" includes the meaning of a serum albumin having an arnino acid sequence naturally occurring in cows, for example as taken from Swissprot accession number P02769, and variants thereof as defined below. The term "bovine serum albumin" also includes the meaning of fragments of fall-length bovine serum albumin or variants thereof, as defined below.<br>
ha another preferred embodiment the albiirnin comprises the sequence of an albiirnin derived from one of serum albumin from dog (e.g. see Swissprot accession number P49822), pig (e.g. see Swissprot accession number P08835), goat (e.g. as available from Sigma as product no. A2514 or A4164), turkey (e.g. see Swissprot accession number 073860), baboon (e.g. as available from Sigma as product no. A1516), cat (e.g. see Swissprot accession number P49064), chicken (e.g.   see   Swissprot  accession  number  P19121),   ovalbumin  (e.g.   chicken ovalbumin) (e.g. see Swissprot accession number P01012), donkey (e.g.  see Swissprot accession number P39090), guinea pig (e.g. as available from Sigma as product no. A3060, A2639, O5483 or A6539), hamster (e.g. as available from Sigma as product no. A5409), horse (e.g. see Swissprot accession number<br>
P35747), rhesus monkey (e.g. see Swissprot accession number Q28522), mouse<br>
(e.g. see Swissprot accession number 089020). pigeon (e.g. as defined by Khan el al, 2002, 1m. J. Eial. Macromol, 30(3-4), 171-8), rabbi! (e.g. see Swissprot accession number 'M9065). rai (e.g. see Swissprot accession number P36953) and sheep (e.g. see Swissprot accession number P14639) and includes variants and fragments thereof as defined below.<br>
Many naturally occurring mutant forms of albumin are known. Many are described in Peters, (1996, All About Albumin: Biochemistry, Genetics and Medical Applications, Academic Press, Inc., Sari Diego, California, p. 170-181). A variant as defined above may be one of these naturally occurring mutants.<br>
A "variant albumin" refers to an albumin protein wherein at one or more positions there have been arnino acid insertions, deletions, or substitutions, either conservative or non-conservative., provided that such changes result in an albumin protein for which at least one basic property, for example binding activity (type of and specific activity e.g. binding to bilirubin). osmolarity (oncotic pressure, colloid osmotic pressure), behaviour in a certain pH-range (pH-stability) has not significantly been changed. "Significantly" in this context means that one skilled in the art would say that the properties of the variant may still be different but would not be unobvious over the ones of the original protein.<br>
By "conservative substitutions" is intended combinations such as Gly, Ala; Val, lie, Leu; Asp, Giu; Asn, Gin; Ser, Thr; Lys, Arg; and Phe, Tyr. Such variants may be made by techniques well known in the art, such as by site-directed mutagenesis as disclosed in US Patent No 4,302,386 issued 24 November 1981 to Stevens, incorporated herein by reference.<br>
Typically an albumin variant will have more than 40%, usually at least 50%, more typically at least 60%, preferably at least 70%, more preferably at least 80%, yet more preferably at least 90%, even more preferably at least 95%, most preferably at<br>
least 98% or more sequence identity with naturally occurring albumin. The percent<br>
sequence identity between two polypeptides may be determined using suitable computer programs, for example the GAP program of the University of Wisconsin Genetic Computing Group and it will be appreciated that percent identity is calculated in relation to polypeptides whose sequence has been aligned optimally. The alignment may alternatively be carried out using the Clustal W program (Thompson et al, 1994). The parameters used may be as follows:<br>
Fast pairwise alignment parameters: K-tuple(word) size; 1, window size; 5, gap penalty; 3, number of top diagonals; 5. Scoring method: x percent. Multiple alignment parameters: gap open penalty; 10, gap extension penalty; 0.05. Scoring matrix: BLOSUM.<br>
The term "fragment" as used above includes any fragment of full-length, albumin or a variant thereof, so long as at least one basic property, for example binding activity (type of and specific activity e.g. binding to bilirubin), osmolarity (oncotic pressure, colloid osmotic pressure), behaviour in a certain pH-range (pH-stabiliry) has not significantly been changed.  "Significantly" in this context means that one slcilled in the art would say that the properties of the variant may. still be different but would not be unobvious over the ones of the original protein.   A fragment will typically be at least 50 amino acids long. A fragment may comprise at least one whole  sub-domain of albumin.    Domains of HSA have been expressed as recombinant proteins (Dockal, M. et al, 1999, J. Biol Chem., 274, 29303-29310), where domain I was defined as consisting of amino acids 1-197, domain II was defined as consisting of amino acids 189-385 and domain III was defined as consisting of amino acids 381-585. Partial overlap of the domains occurs because of the extended a-helix structure (hlO-hl) which exists between domains I and II, and between domains II and III (Peters, 1996, op. cit., Table 2-4).   HSA also comprises six sub-domains (sub-domains IA, IB, IIA, IIB, IIIA and IIIB).   Sub-domain IA comprises amino acids 6-105, sub-domain IB comprises amino acids 120-177,  sub-domain IIA comprises  amino  acids  200-291,  sub-domain IIB<br>
comprises amino acids 316-369. sub-domain IDA comprises ammo acids 392-491 and sub-domain 1116 comprises arnino acids 512-583. A fragmew may comprise K whole or part oi' one or more domains or sub-domains as defined aoove. or any combination of those domains and/or sub-domains.<br>
In another particular!}' preferred embodiment the heterologous protein comprises the  sequence  of transferrin  or  a variant  or  fragment  thereof.      The   term 'transferrin" as used herein includes all members of the transferrin family (Testa. Proteins of iron metabolism., CRC Press, 2002; Harris &amp; Aisen, Iron carriers and iron proteins. Vol. 5. Physical Bioinorganic Chemistry, VCH, 1991) and their derivatives,   such   as  transferrin,   mutant   tran.sferrins   (Mason   e1   al,    1993, Biochemistry, 32, 5472; Mason et a!, 1998, Biochem. J., 330(1), 35), truncated transferrins, transferrin lobes (Mason et al, 1996, Protein Expr. Purif., 8, 119; Mason et at, 1991, Protein Expr. Purif., 2, 214), lactoierrin, omtanr lactoferrins, truncated lactoferrins. lactoferrin lobes or fusions of any of the above to other peptides, polypeptides or proteins (Shin el al, 1995, Proc. NatL Acad. Sci. USA, 92, 2820; All et al, 1999, J. Biol  Chem.&gt; 274, 24066; Mason et al,  2002, Biochemistry., 41, 9448).<br>
The transferrin may be human transferrin. The term "human transferrin" is used herein to denote material which is indistinguishable from transferrin derived from a human or which is a variant or fragment thereof. A "variant" includes insertions, deletions and substitutions, either conservative or non-conservative, where such changes do not substantially" alter the useful ligand-binding or immunogenic properties of transferrin.<br>
Mutants oi" transferrin are included in the invention. Such mutants may have altered immunogenicity. For example, transferrin mutants may display modified (e.g. reduced) giycosylaiion. The N-linked glycosj'lation pattern of a transferrin molecule can be modified by adding/removing arnino acid glycos3rlation<br>
consensus sequences such as N-X-S/T, at any or all of the N, X. or S/T position.<br>
Transferrin mutants may be altered in their natural binding to metal ions and/or other proteins, such as transferrin receptor. An example of a transferrin mutant modified in this manner is exemplified below.<br>
We also include naturally-occurring polymorphic variants of human transferrin or human transferrin analogues.     Generally,  variants  or  fragments   of human transferrin will have at least 5%, 10%, 15%, 20%, 30%, 40% or 50% (preferably at least 80%, 90% or 95%) of human transferring ligand binding activity (for example iron-binding), weight for weight. The iron binding activity of transferrin or a test sample can be determined spectrophotometrically by 470nm:280nm absorbance ratios for the proteins in their iron-free and fully iron-loaded states. Reagents should be iron-free unless stated otherwise.  Iron can be removed from transferrin or the test sample by dialysis against 0.1M citrate, 0.1M acetate, lOmM EDTA pH4.5. Protein should be at approximately 20mg/mL in lOOmM HEPES, lOmM NaHCO:, pH8,0.    Measure the 470nm:280nm absorbance ratio of apo-transferrin (Calbiochem, CN Biosciences, Nottingham, UK) diluted in water so that absorbance at 280nm can be accurately determined spectrophotometrically (0% iron binding).    Prepare 20mM iron-nitrilotriacetate (FeNTA) solution by dissolving 191mg nitrotriacetic acid in 2mL 1M NaOH, then add 2mL 0.5M ferric chloride.  Dilute to 50mL with deionised water.   Fully load apo-transferrin with iron (100% iron binding) by adding a sufficient excess of freshly prepared 20mM FeNTA, then dialyse the holo-transferrin preparation completely against lOOmM HEPES, lOmM NaHCOs pHS.O to remove remaining FeNTA before measuring the absorbance ratio at 470nm:280nm.   Repeat the procedure using test sample, which should initially be free from iron, and compare final ratios to the control.<br>
Additionally, single or multiple heterologous fusions comprising any of the above; or single or multiple heterologous fusions to albumin, transferrin or immunoglobins or a variant or- fragment of any of these may be used. Such fusions include albumin N-terminal fusions, albumin C-terminal fusions and co-<br>
N-terminal and C-terminal albumin fusions as exemplified by WO 01/79271, and<br>
transferrin N-terminal fusions, transferrin C-terminal fusions, and co-N-temiinal and C-terminaJ transJ'errin fusions.<br>
Examples of tranr&gt;ferrin fusions are given in US patent applications US2003/0221201 and US2003/0226155, Shin, ei al, 1995, Proc Natl Acad Sci U S A, 92, 2820, Ali, et al., 1999, J Biol Chem, 274, 24066, Mason, ei al, 2002, Biochemistry, 41, 9448, the contents of which are incorporated herein by<br>
reference.<br>
The skilled person will also appreciate that the open reading frame of an)' other gene or variant, or part or either, can be utilised as an open reading frame for use with the present invention. For example, the open reading frame may encode a protein comprising an}' sequence, be it a natural protein (including a zymogen), or a variant, or a fragment (which may, for example, be a domain) of a natural protein; or a totally synthetic protein; or a single or multiple fusion of different proteins (natural or synthetic).   Such proteins can be taken, but not exclusively, from the lists provided in WO 01/79258, WO 01/79271, WO 01/79442, WO 01/79443, WO 01/79444 and WO 01/79480, or a variant or fragment thereof; the disclosures of which are incorporated herein by reference.  Although these patent applications present the list of proteins in the context of fusion partners for albumin, the present .invention is not so limited and, for the purposes of the present invention, any of the proteins listed therein may be presented alone or as fusion partners lor  albumin, the Fc region of immunoglobulin,  transferrin, lactoferrin or an)' other protein or fragment or valiant of any of the above, as a desired polypeptide.<br>
The heterologous protein may be a therapeutical!}' active protein. In other words, it may have a recognised medical effect on individuals, such as humans. Many different, types of therapeutical!}' active protein are well known in the art.<br>
The heterologous protein may comprise a leader sequence effective to cause secretion in yeast.<br>
Numerous natural or artificial polypeptide signal sequences (also called secretion pre regions) have been used or developed for secreting proteins from host cells. The signal sequence directs the nascent protein towards the machinery of the cell that exports proteins from the cell into the surrounding medium or, in some cases, into the periplasmic space. The signal sequence is usually, although not necessarily, located at the N~terminus of the primary translation product and is generally, although not necessarily, cleaved off the protein during the secretion process, to yield the "mature" protein.<br>
In the case of some proteins the entity that is initially secreted, after the removal of the signal sequence, includes additional amino acids at its N-terminus called a "pro" sequence, the intermediate entity being called a "pro-protein". These pro sequences may assist the final protein to fold and become functional, and are usually then cleaved off. In other instances, the pro region simply provides a cleavage site for an enzyme to cleave off the pre-pro region and is not known to have another function.<br>
The pro sequence can be removed either during the secretion of the protein from the cell or after export from the cell into the surrounding medium or periplasmic space.<br>
Polypeptide sequences which direct the secretion of proteins, whether they resemble signal (i.e. pre) sequences or pre-pro secretion sequences, are referred to as leader sequences. The secretion of proteins is a dynamic process involving translation, translocation and post-translational processing, and one or more of these steps may not necessarily be completed before another is either initiated or completed.<br>
For production oi protein? in eukaryotic species such as the yeasts 8accharomyc.cs cerevisiae. Zygosaccharomyces species. Khtyveromycc.!; lactis and Pichiapasioris, known leader sequences include those from the 5'. cerevisiae acid phosphatase protein fPho5p) (see EP 366 400j, the invertase protein (Suc2p) (see Smith et al. (1985) Science, 229, 1219-1224) and heat-shock protein-150 (HsplSOp) (see WO 95/33833).   Additional!}', leader sequences from the S. cerevisiae mating factor alpha-]  protein (MFa-1) and from the human lysozyme and human  serum albumin (H.SA) protein have been used, the latter having been used especially, although not exclusively, for secreting human albumin. WO 90/01063 discloses a fusion of the MFa-1 and HSA leader sequences, which advantageously reduces the production of a contaminating fragment of human albumin relative to the use of the MFa-1 leader sequence.  Modified leader sequences are also disclosed in the examples of this application and the reader will appreciate that those leader sequences can be used with proteins other than transferrin. In addition, the natural transferrin leader sequence may be used to direct secretion of transferrin and other heterologous proteins.<br>
Where the chaperone is protein disulphide isomerase, then preferably the heterologous protein comprises disulphide bonds in its mature form. The disulphide bonds may be intramolecular and/or intermolecular.<br>
The  heterologous protein may  be  a   commercially useful protein.      Some heterologously expressed proteins are intended to interact with the cell in which they are expressed in order to bring about a beneficial effect on the  cell's activities.    These proteins are not, in their own right, commercially useful. Commercially useful proteins are proteins that have a utility ex vivo of the cell in which they are expressed.  Nevertheless, the skilled reader will appreciate that a commercially useful protein may also have a biological effect on the host cell expressing it as a heterologous protein, but mat that effect is not the main or sole reason for expressing the protein therein.<br>
In one embodiment it is preferred that the heterologous protein is not (3-lactamase. In another embodiment it is preferred that the heterologous protein is not antistasin. However, the reader will appreciate that neither of these provisos exclude genes encoding either p-lactamase or antistasin from being present on the 2um-family plasmid of the invention, merely that the gene encoding the heterologous protein encodes a protein other than (3-lactamase and/or antistasin.<br>
Plasmids can be prepared by modifying 2|om-family plasmids known in the art by inserting a gene encoding a chaperone and inserting a gene encoding a heterologous protein using techniques well known in the art such as are described in by Sambrook et al, Molecular Cloning: A Laboratory Manual, 2001, 3rd edition, the contents of which are incorporated herein by reference. For example, one such method involves ligation via cohesive ends. Compatible cohesive ends can be generated on a DNA fragment for insertion and plasmid by the action of suitable restriction enzymes. These ends will rapidly anneal through complementary base pairing and remaining nicks can be closed by the action of DNA ligase.<br>
A further method uses synthetic double stranded oligonucleotide linkers and •adaptors. DNA fragments with blunt ends are generated by bacteriophage T4 DNA polymerase or E.coli DNA polymerase I which remove protruding 3' termini and fill in recessed 3' ends.   Synthetic linkers and pieces of blunt-ended double-stranded DNA, which contain recognition sequences for defined restriction enzymes, can be ligated to blunt-ended DNA fragments by T4 DNA ligase.  They are subsequently digested with appropriate restriction enzymes to create cohesive ends and ligated to an expression vector with compatible termini.    Adaptors are also  chemically synthesised DNA fragments which contain one blunt end used for ligation but which also possess one preformed cohesive end.  Alternatively a DNA fragment or DNA fragments can be ligated together by the action of DNA ligase in the presence or<br>
absence of one or more  synthetic  double stranded  oh'gonudeotides  optionally containing cohesive ends.<br>
Synthetic   linkers  containing   a   variety  of restriction   endonuclease   sires   are commercially available .from a number of sources including Sigma-Genosys Ltd, 'London Road. Pampisford. Cambridge, United Kingdom-Appropriate insertion sites in 2uin-famil}<r plasmids include but are not limited to. those discussed above.></r>
The present invention also provides a host cell comprising a plasmid as defined above. Tbe host cell may be any type of cell. Bacterial and yeast host cells are preferred. Bacterial host cells may be useful for cloning purposes. Yeast host cells may be useful for expression of genes present in the plasmid.<br>
In one embodiment, the host cell is a yeast cell, such as a member of the Saccharomyces, Kluyveromyces, or Pichia genus, such Saccharomyces cerevisiae, Khtweromyces lactis. Pichia pastoris and Pichia membranaefaciens, or 'Zygosaccharomyces rouxii, Zygosaccharomyces bailii, Zygosaccharoinyces fermentati, or Khiyveromyces drosphilarum are preferred.<br>
The host cell type may be selected for compatibility with the plasmid type being used.   Plasmids obtained from one yeast type can be maintained in other yeast types (Irie ei al, 1991, Gene, 108(1), 139-144; Me et al, 1991, Mol Gen. Genet, 225(2), 257-265),   For example, pSRl from Zygosaccharomyces rouxii can be maintained in Saccharomyces cerevisiae.  Preferably, the host cell is compatible with the 2[im-farnily plasmid used (see below for a full description of the following plasmids). For example, where the plasmid is based on pSRl, pSB3 or pSB4 then a suitable yeast cell is Zygosaccharomyces rouxii; where the plasmid is based on pSBl orpSB2 then a suitable yeast cell is Zygosaccharomyces bailii;<br>
where the plasmid is based on pSMl then a suitable yeast cell is Zygosaccharomyces fermentati; where the plasmid is based on pKDl then a suitable yeast cell is Kluyveromyces drosophilarum; where the plasmid is based on pPMl then a suitable yeast cell is Pichia membranaefaciens; where the plasmid is based on the 2um plasmid then a suitable yeast cell is Saccharomyces cerevisiae or Saccharomyces carlsbergensis. It is particularly preferred that the plasmid is based on the 2urn plasmid and the yeast cell is Saccharomyces cerevisiae.<br>
A 2 urn-family plasmid of the invention can be said to be "based on" a naturally occurring plasmid if it comprises one, two or preferably three of the genes FLP, REP1 and REP2 having sequences derived from that naturally occurring plasmid.<br>
It may be particularly advantageous to use a yeast deficient in one or. more protein mannosyl trarisferases involved in O-glycosylation of proteins., for instance by disruption of the gene coding sequence.<br>
Recombinantly expressed proteins can be subject to undesirable post-translational modifications by the producing host cell.    For example, the albumin protein sequence does not contain any sites for N-linlced glycosylation and has not been reported to be modified, in nature, by O-linked glycosylation.   However, it has been found that recombinant human albumin ("rHA") produced in a number of yeast species can be modified by O-linked glycosylation, generally involving mannose. The mannosylated albumin is able to bind to the lectin Concanavalin A. The amount of mannosylated albumin produced by the yeast can be reduced by using a yeast strain deficient in one or more of the PMT genes (WO 94/04687). The most convenient way of achieving this is to create a yeast which has a defect in its genome such that a reduced level of one of the Pmt proteins is produced. For example, there may be a deletion, insertion or transposition in the coding sequence or the regulatory regions (or in another gene regulating the expression of one  of the PMT genes)  such that little  or no Pmt protein is  produced.<br>
Alternatively, the- yeast could be transformed to produce an anti-Pmt agent, such as an anti-Prat antibody.<br>
If a yeast other than S. cerevisiac. is used, disruption of one or more of the genes equivalent to the PMf genes of S. cerevisiae is also beneficial, e.g. in Pichia pastoris or Khtyveromyces lactis. The sequence of PhdTl (or an)' other PMT gene) isolated from S. cerevisiae may be used for the identification or disruption of genes encoding similar enz3onatic acthdties in other fungal species. The cloning of the PMTl homologue of Klit)&gt;veromyces lactis is described in WO 94/04687.<br>
The yeast will advantageously have a deletion of the HSP150 and/or YAPS genes as taught respectively in WO 95/33833 and WO 95/23857.<br>
A plasmid as defined above, may be introduced into a host through standard techniques.    With regard to transformation of prokaryotic host cells, see, for example, Cohen el al (1972) Proc. Natl. Acad. Sci. USA 69, 2110 and Sambrook et al (2001) Molecular Cloning, A Laboratoiy Manual, 3ld Ed. Cold Spring Harbor Laboratory. Cold Spring Harbor, NY.  Transformation of yeast cells is described in Sherman et al (1986) Methods In Yeast Genetics, A. Laboratory Manual, Cold Spring Harbor, NY.   The method of Beggs (1978) Nature 275, 104-109 is also useful. Methods for the transformation of S. cerevisiae are taught generally in EP 251 744, EP 258 067 and WO 90/01063, all of which are incorporated herein by reference. With regard to vertebrate cells, reagents useful in transfectiiig such cells, for example calcium phosphate and DEAE-dextran or liposome formulations, are available from Stratagene Cloning Systems, or Life Technologies Inc., Gaithersburg, MD 20877, USA.<br>
Electroporation is also useful for transforming cells and is well known in the art for )     transforming yeast   cell,  bacterial   cells   and  vertebrate   cells.      Methods   for<br>
transformation of yeast by electroporation are disclosed in Becker &amp; Guarente (1990} Methods Enzymol. 194,182.<br>
Generally, the plasmid will transform not all of the hosts and it will therefore be necessary to select for transformed host cells.   Thus, a plasmid-may comprise a selectable marker, including but not limited to bacterial selectable marker and/or a yeast selectable marker.  A typical bacterial selectable marker is the p-lactamase gene although many others are known in the art. Typical yeast selectable marker include LEU2, TRP1, HIS3, HIS4, URA3, URA5, SFA1, ADE2, METIS, LYS5, LYS2, ILV2, FBA1, PSE1, PDI1 and PGKL    Those skilled in 'the art will appreciate that any gene whose chromosomal deletion or inactivation results in an inviable host, so called essential genes, can be used as a selective marker if a functional gene is provided on the plasmid, as demonstrated for PGK1 in zpgkl yeast strain (Piper and Curran, 1990, Curr. Genet. 17, 119).   Suitable essential genes    can    be   found    within   the   Stanford   Genome   Database   (SGD), http:://db.yeastgenome.org). Any essential gene product (e.g. PDI1, PSE1, PGK1 or FBA1) which, when deleted or inactivated, does not result in an auxotrophic (biosynthetic) requirement, can be used as a selectable marker on a plasmid in a host cell that, in the absence of the plasmid, is unable to produce that gene product, to achieve increased plasmid stability without the disadvantage of requiring the  cell to be  cultured under specific selective  conditions.     By "auxotrophic (biosynthetic) requirement" we include a deficiency which can be complemented by additions or modifications to the growth medium.   Therefore, preferred "essential marker genes" in the context of the present invention are those that, when deleted or inactivated in a host cell, result in a deficiency which cannot be complemented by additions or modifications to the growth medium.<br>
Additionally, a plasmid according to any one of the first, second or third aspects of the present invention may comprise more than one selectable marker.<br>
One selection technique involves incorporating into the expression vector a DNA sequence marker, with any necessary control elements, that codes for a selectable trait in the transformed cell, These markers include diliydrofolate reductase. G418 or neomycin resistance for eukaryotic cell culture, and tetracyclin, kanamycin or ampicillin (i.e. (Hactanaase) resistance genes for culturing in E.coli and other bacteria. Alternatively, the gene for such selectable trait can be on another vector, which is used to co-transform the desired host cell.<br>
Another method of identifying successfully transformed ceDs involves growing the cells resulting from the introduction of a piasmid of the invention, optionally to allow the expression of a recombinant polypeptide (i.e. a polypeptide which is encoded by a polynucleotide sequence on the piasmid and is heterologous to the host cell, in the sense that that polypeptide is not naturally produced by the host).    Cells can be harvested and lysed and their DNA or KNfA content examined for the presence of the recombinant sequence using a method such as that described by Southern (1975) J. Mol Biol. 98, 503 or Berent et al (1985) Biotech. 3, 208 or other methods of DNA and RNA analysis common in the art. Alternatively, the presence of a polypeptide in the supernatant of a culture of a transformed cell can be detected using antibodies.<br>
In addition to directly assaying for the presence of recombinant DNA, successful rransfomiation can be corrfirrned by weh1 known irnmunological methods when the recombinant DNA is capable of directing the expression of the protein. For example, cells successfully transformed with an expression vector produce proteins displaying appropriate antigenicity. Samples of cells suspected of being transfonried are harvested and assayed for the protein using suitable antibodies.<br>
Thus, in addition to the transformed host cells themselves, the present invention also<br>
contemplates a culture of those cells, preferably a monoclonal (clonally<br>
homogeneous) culture, or a culture derived from a monoclonal culture, in a nutrient<br>
medium.	Alternatively, transformed cells may represent an<br>
industrially/commercially or pharmaceutically useful product and can be used without iurther purification or can be purified from a culture medium and optionally formulated with a earner or diluent in a manner appropriate to their intended industrial/commercial or pharmaceutical use, and optionally packaged and presented m a manner suitable for that use. For example, whole cells could be immobilised; or used to spray a cell culture directly on to/into a process, crop or other desired target. Similarly, whole cell, such as yeast cells can be used as capsules for a huge variety of applications, such as fragrances, flavours and pharmaceuticals.<br>
Transformed host cells may be cultured for a sufficient time and under appropriate conditions known to those skilled in the art, and in view of the teachings disclosed herein, to permit the expression of the chaperone and heterologous protein encoded by the plasmid.<br>
The culture medium may be non-selective or place a selective pressure on the maintenance of the plasmid.<br>
The thus produced heterologous protein may be present intracellularly or, if secreted, n. the culture medium and/or periplasmic space of the host cell.<br>
The step of "purifying the thus expressed heterologous protein from the cultured host cell or the culture medium" optionally comprises cell immobilization, ceU separation and/or cell breakage, but always comprises at least one other purification step different from the step or steps of cell immobilization, separation and/or breakage.<br>
Cell immobilization techniques, such as encasing the cells using calcium alginate bead, are well known in the art. ' Similarly, cell separation techniques, such as centrifugation, filtration (e.g. cross-flow filtration, expanded bed chromatography and the like are well known in the art. Likewise, methods of cell breakage,<br>
including  headmilimg,  sonication,  enzymatic exposure and the.  like  are well known in the art.<br>
The at least one other purification step may be any other step suitable for protein purification known in the art. For example purification techniques for the recovery of reconibinantly expressed albumin have been disclosed hi: WO 92/04367, removal of matrix-derived dye; EP 464 590., removal of yeast-derived colorants; EP 319 067, alkaline precipitation and subsequent application of the albumin to a lipophilic phase; and WO 96/37515, US 5 728 553 and WO 00/44772. which describe complete purification processes; all of which are incorporated herein by reference.<br>
Proteins other than albumin may be purified from the culture medium by any technique that has been found to be useful for purifying such proteins.<br>
Suitable methods include ammonium sulphate or ethanol precipitation, acid or solvent extraction, anion or cation exchange chromatography, phosphocellulose chromatography. hydrophobia interaction chromatography, affinity chromatography, bydroxylapatite chromatography, lectin chromatography, concentration, dilution, pH adjustment, diafiltration, ultrafiltration, high performance liquid chromatography ("HPLC"), reverse phase HPLC, conductivity adjustment and the like.<br>
In one embodiment, any one or more of the above mentioned techniques may be used to further purifying the thus isolated protein to a commercially or industrially acceptable level of purity.   By commercially or industrially acceptable level of purity, we include the provision of the protein at a concentration of at least 0.01 g.U1, 0.02 g.U1, 0.03 g.U\ 0.04 g.U1, 0.05 g.U1,0.06 glO.O? g.U1, 0.08 g.U1, 0.09 g.U1, 0.1 g.L'1, 0.2 g.L"1, 0.3 g.U1, 0.4 g.U1, 0.5 g.U1, 0.6 g.U1, 0.7 g.U1, 0.8 gU1, 0.9 g-U1,1 g.U], 2 g.U]5 3 g-U1,4 g.U1, 5 g.U1, 6 g.U1, 7 g.U1, 8 g.U1, 9 g.U \ 10 gU1,15 gU1, 20 g.U', 25 g-U1, 30 g.U1, 40 g.U1,50 g.U1, 60 g.U1, 70 g.U1,<br>
70 g.1/1, 90 g.U1, 100 g.L'1, 150 g.U1, 200 g.L-],250 g.U1, 300 g.L/1, 350 g.L'1, 400 g.L'1, 500 g.L'], 600 g.U], 700 g.L"1, 800 g.L'1, 900 g.L'1,1000 g.L'1, or more.<br>
It is preferred that the heterologous protein is purified to achieve a pharmaceutically acceptable level of purity. A protein has a pharmaceutically acceptable level of purity is it is essentially pyrogen free and can be administered in a pharmaceutically efficacious amount without causing medical effects not associated with the activity of the protein.<br>
The resulting heterologous protein may be used for any of its known utilities, which, in the case of albumin, include i.v. administration to patients to treat severe burns, shock and blood loss, supplementing culture media, and as an excipient in formulations of other proteins.<br>
Although it is possible for a therapeutically useful heterologous protein obtained by a process of the of the invention to be administered alone, it is preferable to present it as a pharmaceutical formulation, together with one or more acceptable carriers or diluents. The carrier(s) or diluent(s) must be "acceptable" in the sense of being compatible with the desired protein and not deleterious to the recipients thereof. Typically, the carriers or diluents will be water or saline which will be sterile and pyrogen free.<br>
Optionally the thus formulated protein will be presented hi a unit dosage form, such as in the form of a tablet, capsule, injectable solution or the like.<br>
A further embodiment of the present invention provides a host cell recombinantly encoding proteins comprising the sequences of PDI and transferrin-based proteins. By "fransferrin-based protein" we mean transferrin or any other member of the transferrin family (e.g. lactoferrin), a variant or fragment thereof or a fusion protein comprising transferrin, a variant or fragment thereof, including the types<br>
-riVd above     Thus fh, present invention also provides for the use of a .ecombmam PD] gene to mcrease the expression of a transfemr.-based protem.<br>
The PDI «-ne .nay be pmvidcd on a plasnud, such as a 2un&gt;family plasnud as described above Alternatively, the PDI gene may be chromosomally mtegrated. !„ a preferred embodiment, the PDI gene is chromosomally mtegrated a. the locus of an endo-enouslv encoded PDI gene, preferably wrthout disruptrng the expression of the endogenous PDI gene. In tins context, "without top** the express of the endogenous PDI gene" means that, although some decrease m the protean production from the endogenous PDI gene as a result of tire integration may be acceptable (and preferably trere u, no decrease), the total level of PDI proten, production in the modified host cell as a result of tie combmed effect of expression from the endogenous and mtegrated PDI genes is increased, relative to ,he level of PDI protem production by fte host oell prior to the integration event.<br>
The «,ene encoding the transferrm-hased protem may be provided on a plasrmd, suctA, a 2u.m-fa.mly plasmW as described above, or may be chromosomdly to such as ,t the locus of an endogenously encoded PDI gene, preferably without disrupting the expression of the endogenous PDI gene.<br>
In one embodmreri the PDI gene i, ctomosomally mtegrated and the gene encoding the transferrm-based protein is provided on a plasmid In anotier embodunent, the PDI gene is proved on a plasnud and tne gene encodmg the U-msferan-baSed protein is coromosomally mtegrated. In another embodiment both me PDI gene and the gene encoding the transferrm-based protem are .hromosomally mtegrated. m another embodtont both the PDI gene and the gene encoding the transferrin-based protein are provided on a plasnud.<br>
As fccassed above, Bao «, ,/, 2000, Yeas, 16, 329-341 reported that over-expression of the Z. lactis PDI gene IOPDI1 was torfc to K. tato cells. Agamst<br>
this background we have surprisingly found that, not only is it possible to over-express PDI and other chaperones without the detrimental effects reported in Bad et al, but that two different chaperones can be recombinantly over-expressed in the same cell and, rather than being toxic, can increase the expression of heterologous proteins to levels higher than the levels obtained by individual expression of either of the chaperones.    This was not expected.    On the contrary, in light of the teaching of Bao et al, one would think that over-expression of two chaperones would be even more toxic than the over-expression of one. Moreover, in light of the earlier findings of the present invention, it was expected that the increases in heterologous  protein   expression   obtained  by  co-expression   with   a   single chaperone would be at the maximum level possible for the cell system used. Therefore, it was particularly surprising to find that yet further increases in heterologous protein expression could be obtained by co-expression of two different chaperones with the heterologous protein.<br>
Accordingly, as a fifth aspect of the present invention mere is provided a method for producing heterologous protein comprising providing a host cell (such as defined above) comprising a first recombinant gene encoding a protein comprising the sequence of a first chaperone protein, a second recombinant gene encoding a protein comprising the sequence of a second chaperone protein and a third recombinant gene encoding a heterologous protein, wherein the first and second chaperones are different; culturing the host cell in a culture medium under conditions that allow the expression of the first, second and third genes; and optionally purifying' the thus expressed heterologous protein from the cultured host cell or the culture medium; and further optionally, lyophilising the thus purified protein.<br>
The method may further comprise the step of formulating the purified heterologous protein with a carrier or diluent and optionally presenting the thus formulated protein in a unit dosage form, in the manner discussed above.<br>
The term 'Tecrmibinani gene"" includes nucleic acid sequences that operate independently at 'stand alone'"' expressible sequences to produce an encoded protein or, in the alternative, nucleic acid sequences introduced that operate in combination with endogenous sequences (such as by integration into an endogenous sequence so as to produce a nucleic acid sequence that is different to the endogenous sequence) within the host to cause increased expression of a target protein.<br>
The first and second chaperones may be a chaperone as discussed above, and are a combination of chaperones that, when co-expressed in the same host cell, provide an additive effect to the increase in expression of the heterologous protein.   By "additive effect" we include the meaning that the level of expression of the heterologous protein in the host cell is higher when the first and second recombinant. genes are simultaneously co-expressed with the third recombinant gene as compared to the same system wherein (i) the first reconibmant gene is co-expressed with the third recombinant gene in the absence of the expression of the second recombinant gene and (ii) the second recombinant gene is co-expressed with the third recombinant gene in the absence of the expression of the first recombinant gene.<br>
One preferred chaperone is protein disulphide isomerase. Another preferred chaperone is ORM2 or a fragment or variant thereof. In a particularly preferred embodiment, the first and second chaperones are protein disulphide isomerase and ORM2 or a fragment, or variant thereof.<br>
The first, second and third recombinant genes may each individually be present on a plasmid within the host cell (such as a 2um-family plasmid, as discussed above) or be chrqmosomally integrated within the genome of the host cell. It will be appreciated that an)' combination of plasmid and chromosomally integrated first, second and third recombinant genes may be used. For example, the first, second<br>
nd third recombinant genes may each individually be present on a plasmid, and this may be either the same plasmid or different plasmids. Alternatively, the first recombinant gene may be present on a plasmid, and second and third recombinant genes may be chromosomally integrated within the genome of the host cell. Alternatively, the first and second recombinant genes may be present on a plasmid and the third recombinant gene may be chromosomally integrated within the genome of the host cell. Alternatively, the first and third recombinant genes may be present on a plasmid and the second recombinant gene may be chromosomally integrated within the genome of the host cell.  Alternatively, the first and second recombinant gene may be chromosomally integrated within the genome of the host cell  and  the third recombinant  gene may be present on  a plasmid. Alternatively, the first, second and third recombinant genes may each individually be chromosomally integrated within the genome of the host cell.<br>
Particularly preferred plasmids are those defined above in respect of earlier aspects of the present invention. Accordingly, the present invention also provides a plasmid as defined above wherein the plasmid comprises two different genes (the first and second recombinant genes) encoding different chaperones. In one preferred embodiment, the plasmid may further comprise a gene encoding a heterologous protein (the third recombinant gene), such as a heterologous protein as described above.<br>
In a sixth aspect of the present invention there is provided a method for producing a heterologous protein, such as a heterologous protein as defined above for an earlier  aspect of the present invention,   comprising: providing - a host cell comprising a first recombinant gene encoding the protein comprising the sequence of ORM2 or a variant thereof and a second recombinant gene encoding a heterologous protein; culturing the host cell in a culture medium under conditions that allow the expression of the first and second genes; and purifying the thus expressed heterologous protein from the cultured host cell or the culture medium;<br>
and optionally, lyophilising the thus purified protein; and optionally formulating<br>
the purified heterologous protein with a carrier or diluent; and optionally presenting the thus formulated protein in a unit dosage form..<br>
In the manner discussed above, the host cell may further comprise a further recombinant gene encoding a protein comprising the sequence of an alternative chaperone to ORM2 or a variant thereof.<br>
Either or both of the first and second recombinant genes may be expressed from a plasmid, and preferably from the same plasmid. A farther recombinant gene encoding a protein comprising the sequence of an alternative chaperone to ORM2 or a variant thereof may also be expressed from a plasmid, preferably from the same plasmid as either or both of the first and second recombinant genes. The plasmid may be a 2 urn-family plasmid, such as the 2u,m plasmid.<br>
The present invention also provides, in a seventh aspect, for the use of a nucleic acid sequence encoding the protein ORM2 or a variant thereof to increase the production, in a host cell, of a heterologous protein encoded by a recombinant gene in the host cell by co-expression of the nucleic acid sequence and the recombinant gene within the host cell. Either or both of the nucleic acid sequence and the recombinant gene encoding the heterologous protein ma}' be expressed from a plasmid within the host cell, and preferably from the same plasmid. In the manner discussed above, the host cell may further comprise a recombinant gene encoding an alternative chaperone to ORM2 or a variant thereof, which ma}' be located on a plasmid within the host cell, preferably on the same plasmid as either or both of the nucleic acid sequence and the recombinant gene encoding the heterologous protein.   Suitable plasmids include a 2jim-famuy plasmid, such as the 2|jm plasrnid, as discussed above.<br>
In an eighth aspect of the present invention there is also provided the use of a plasmid as an expression vector to increase the production of a heterologous<br>
protein by providing a recombinant gene encoding the heterologous protein and a gene encoding ORM2 or a variant thereof on the same plasmid. The plasmid may further comprise a gene encoding an alternative chaperone to ORM2 or a variant thereof in the manner discussed above. Suitable plasmids include a 2u,m-family plasmid, such as the 2um plasmid, as discussed above.<br>
Accordingly, in a ninth aspect, the present invention also provides a plasmid, preferably an expression plasmid, comprising a first gene encoding the protein ORM2 or a variant or fragment thereof and a second gene encoding a heterologous protein, as discussed above. The plasmid may further comprise a third gene encoding an alternative chaperone to ORM2 or a variant thereof. In a preferred embodiment, the third gene encodes a protein comprising the sequence of protein disulphide isomerase.<br>
We have also demonstrated that a plasmid-borne gene encoding a protein comprising the sequence of an "essential" chaperone, such as PDI, can be used to stably maintain the plasmid in a host cell that, in the absence of the plasmid, does not produce the chaperone, and simultaneously increase the expression of a heterologous protein encoded by a recombinant gene within the host cell.   This system is advantageous because it allows the user to minimise the number of recombinant genes that need to be carried by a plasmid.   For example, typical. prior art plasmids carry marker genes (such as those as described above) that enable the plasmid to be stably maintained during host cell culturing process. Such marker genes need to be retained on the plasmid in addition to any further genes that are required to achieve a desired effect.   However, the ability of plasmids to incorporate exogenous DNA sequences is limited and it is therefore advantageous to minimise the number of sequence insertions required to achieve a desired effect. Moreover, some marker genes (such as auxotrophic marker genes) require the culturing process to be conducted under specific conditions in order to obtain the effect of the marker gene. Such specific conditions may not be optimal<br>
for eel]   Growth  or proiei.il production,  or ma}' require inefficient   or unduly expensive growth systems to be used.<br>
For the purpose of increasing heterologous gene expression, we have found that it is possible to use a gene that recombinantly encodes a protein comprising the sequence of an 'essential" chaperone for the dual purpose of increasing the production of a heterologous protein in a host cell and in the role of a selectable marker on a plasruid, where the plasmid is present within a cell that, in the absence of the plasmid, is unable to produce the chaperone.  This system has the advantage that it minimises the number of recombinant genes that need to be carried by the plasmid. The system also has the advantage that the host cell can be cultured under conditions that do not have to be adapted for an)' particular marker gene, without loosing plasmid stability.  For example, host cells produced using this system can be culture in rich media, which may be more economical than the minimal media that is commonly used to give auxotrophic marker genes their effect.<br>
Accordingly, in a tenth aspect, the present invention also provides a host cell comprising a plasmid, the plasmid comprising a gene that encodes an essential chaperone wherein, in the absence of the plasmid, the host cell is unable to produce the chaperone. Preferably, in the absence of the plasmid, the host cell is inviable. The host cell may further comprise a recombinant gene encoding a heterologous protein, such as those described above in respect of earlier aspects of the invention.<br>
The present invention also provides, in a eleventh aspect, a plasmid comprising, as the sole selectable marker, a gene encoding an essential chaperone. The plasmid may further compri.se a gene encoding a heterologous protein. The plasmid may be a 2urn-family plasmid.<br>
The present invention also provides, in a twelfth aspect, a method for producing a<br>
heterologous protein comprising the steps of: providing a host cell comprising a<br>
plasmid, the plasmid comprising a gene that encodes an essential chaperone<br>
wherein, in the absence of the plasmid, the host cell is unable to produce the<br>
chaperone and wherein the host cell further comprises a recombinant gene<br>
encoding a heteroiogous protein; culturing the host cell in a culture medium under<br>
conditions  that   allow the  expression  of the   essential   chaperone   and  the<br>
heterologous protein; and optionally purifying the thus expressed heterologous<br>
protein from the cultured host cell or the culture medium; and further optionally,<br>
lyophilising the thus purified protein.<br>
The method may further comprise the step of formulating the purified heterologous protein with a carrier or diluent and optionally presenting the thus formulated protein in a unit dosage form, in the manner discussed above. In one preferred embodiment, the method involves culturing the host cell in non-selective media, such as a rich media.<br>
We have surprising also found that different PDI genes have the ability to increase the expression of heterologous proteins by different amounts under particular culture conditions. In particular, as discussed in Example 8, we have shown that the SKQ2n PDIl gene provides for higher heterologous protein expression than the S288c PDIl gene, when the host cells are cultured in minimal media.<br>
The sole difference between the encoded proteins of the SKQ2n PDIl and S288c PDIl genes is that SKQ2n comprises the additional amino acids EADAEAEA at positions 506-513 (positions as defined with reference to Genbank accession no. CAA38402, as given above).<br>
The differences between the gene sequences used are shown in the sequence alignment given in Figure 94 and can be summarised as follows -<br>
•	The promoter of SKQ2n includes a run of fourteen "TA'" repeats, whereas<br>
the promoter of S288c only has twelve "TAV repeats;<br>
.    Serf i is encoded by TCT in SKQ2n, but by TCC m S288c;<br>
•	Glu44 is encoded by GAA in SKQ2n but by GAG in S288c;<br>
•	Leu262 is encoded by TTG in SKQ2n but by TTA in S28Sc;<br>
•	Asp5l4 is encoded by GAC in SKQ2n but the homologous Asp506 is<br>
encoded by GAT in S288c;<br>
•	The terminator sequence of SKQ2n contains a run of 8 consecutive "A"<br>
bases, whereas the terminator sequence of S288c contains a run of 7<br>
consecutive "A" bases and does not include an "A" base at the equivalent<br>
of position 1880 in the SKQ2n gene;<br>
•	The terminator sequence of SKQ2n has a "C" at position 1919, whereas<br>
the terminator sequence of S288c has a "T" at the equivalent position.<br>
It may be advantageous to include any or all of the above mentioned features of the SKQ2n gene in a PDI gene of choice, in order to achieve the observed increase in heterologous protein expression when the host cells are cultured in rninirnal media.<br>
Accordingly, in a thirteenth aspect, mere is also provided a nucleotide sequence encoding a protein disulphide isomerase. for use in increasing the expression of a heterologous protein in a host cell by expression of the nucleotide sequence within the host cell, which host cell is cultured in rninimal media, wherein the nucleotide<br>
sequence encoding the protein disulphide isomerase is characterised in that it has at least one of the following characteristics ~<br>
•	the nucleotide sequence comprises a promoter having the sequence of a<br>
natural PDI promoter or a functional variant thereof and comprises a run of<br>
fourteen "TA" repeats; or<br>
•	the encoded protein disulphide isomerase comprises the  ammo acids<br>
EADAEAEA or a conservatively substituted variant thereof, typically at<br>
positions 506-513 as defined with reference to Genbank accession no.<br>
CAA38402;or<br>
•	residue Ser41 of the encoded protein disulphide isomerase is encoded by<br>
the codon TCT; or<br>
•	residue Glu44 of the encoded protein disulphide isomerase is encoded by<br>
the codon GAA; or<br>
•	residue Leu262 of the encoded protein disulphide isomerase is encoded by<br>
codon TTG; or<br>
"    residue Asp514 of the encoded protein disulphide isomerase is encoded by codon GAG; or<br>
•	the nucleotide sequence comprises a terminator sequence having the<br>
sequence of a natural PDI terminator or a functional variant thereof and<br>
either comprises a run of 8 consecutive "A" bases and/or the base "C" at<br>
position 1919 (as defined by reference to position 1919 of the natural<br>
SKQ2n terminator sequence).<br>
The present invention also provides, in a fourteenth aspect, a method for producing a heterologous protein comprising the steps of: providing a host cell comprising a recombinant gene that encodes a protein disulphide isomerase and having the sequence of the above-defined nucleic acid sequence, the host cell further comprising a recombinant gene encoding a heterologous protein; cultaing the host cell in a minimal culture medium under conditions that allow the<br>
expression oi the protein disulphide isomerase and the heterolopous protein: and optional]}' purifying the thus expressed heterologous protein from the cultured host cell or the culture medium; and further optionally, Jyophilising the thus purified protein; and optional!}' further formulating the purified heterologous protein with a carrier or diluent; and optional!}' presenting the thus formulated protein in a unit dosage form, in the manner discussed above.<br>
The genes encoding the PDI and heterologous protein can be provided in the manner described above in respect of other embodiments of the present invention.<br>
We have  also found that the effects of recombinantly-provided chaperones according to the other embodiments of the present invention can be modulated by modifying the promoters that control the expression levels of the ahaperone(s). Surprisingly we have found that, in some cases, shorter promoters result in increased heteroiogous protein expression.   Without being bound by theory we believe that this is because the expression of a recombinant chaperone in host cells that already express heterologous proteins at high levels can cause the cells to overload themselves with heterologously expressed protein, thereby achieving little or no overall increase in heterologous protein production.   In those cases, it may be beneficial to provide recombinant chaperone genes with truncated promoters.<br>
Accordingly, in a fifteenth aspect of the present invention there is provided a polynucleotide (such as a plasmid as defined above) comprising the sequence' of a promoter operably connected to a coding sequence encoding a chaperone (such as those described above), for use in increasing the expression of a heterologous protein (such as those described above) in a host cell (such as those described above) by expression of the polynucleotide sequence within the host cell, wherein the promoter is characterised in that it achieves a modified, such as a higher or lower, level of expression of the chaperone than would be achieved if the coding<br>
sequence were to be operably connected to its naturally occurring promoter.<br>
The present invention also provides, in a sixteenth aspect, a method for producing a heterologous protein comprising the steps of: providing a host cell comprising a recombinant gene that comprising the sequence of promoter operably connected to a coding sequence encoding a chaperone, the promoter being characterised in that it achieves a lower level of expression of the chaperone than would be achieved if the coding sequence were to be operably connected to its naturally occurring promoter, and the host cell further comprising a recombinant gene encoding a heterologous protein; culturing the host cell under conditions that allow the expression of the chaperone and the heterologous protein; and optionally purifying the thus expressed heterologous protein from the cultured host cell or the culture medium;  and further optionally,  lyophilising the thus purified protein;  and optionally further formulating the purified heterologous protein with a carrier or diluent; and optionally presenting the thus formulated protein in a unit dosage form, in the manner discussed above.<br>
As is apparent from the examples of the present application, the combination of recombinantly expressed PDI and transferrui-based proteins provides a surprisingly high level of transferrin expression. For example, transferrin expression in a system that includes a chromosomally encoded recombinant PDI gene provided a 2-fold increase (compared to a control in which there is no chromosomally encoded recombinant PDI gene). This increase was 5-times greater than an equivalent system comprising a recombinant gene encoding human albumin in place of the recombinant transferrin gene.<br>
The host may be any cell type, such as a prokaryotic cell (e.g. bacterial cells such as K coif) or a eulcaryotic cell. Preferred eukaryotic cells include fungal cells, such as yeast cells, and mammalian cells. Exemplary yeast cells are discussed above. Exemplary mammalian cells include human cells.<br>
Hosi cells as described above can be. cultured to produce recornbinarit transferrin-based proteins. The thus produced transferrin-based proteins can be isolated from the culture and purified, preferably to a pharmaceutical!}' acceptable level of purity, for example using techniques known in the art' and/or as set out above. Purified translerrin-based proteins may be formulated with a pharmaceutical!}' acceptable carrier or diluent and may be presented in unit dosage form.<br>
The present invention will now be exemplified with reference to the foDowing non-limiting examples and figures.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figures 1, 2, 4, 6 to 15, 22, 25, 27 to 52, 57 to 71, 74, 75, 77 to 79, 81 to 83, 85 to 91, 95 and 96 show various plasmid maps.<br>
Figure 3 shows piasmid insertion sites.<br>
Figure 5 shows a restriction map of a DNA fragment containing the PDI coding sequence.<br>
Figure 16 shows the results of rocket immunoelectrophoresis (PJE) determination of increased recombinant transferrin (N413Q, N611Q) secretion with PDI1 over-expression. Cryopreserved yeast stocks were grown for 4-days in 1 OmL BMMD shake flask cultures and supernatants were loaded at 5uL per well.     Goat polyclonal anti-transferrin (human) antiserum (Calbiochem) was used at 40iL per rocket imrnunoelectrophoresis  gel (5OmL).     A = Control strain  [pSAC35], duplicate flasks; B = Control strain [pDB2536], duplicate flasks; C = Control strain  [pDB2711],  neat to  40-fold  aqueous   dilutions;  D  =   Control   strain [pDB2931], duplicate flasks; E = Control strain [pDB2929], neat to 40-fold aqueous dilutions.<br>
Figure 17 shows the results of RIE analysis of recombinant transferrin (N413Q, N611Q) secretion with and without PDI1 over-expression. Cryopreserved yeast stocks were grown for 4-days in lOniL BMMD shake flask cultures and supematants were loaded at 5uL per well. Duplicate loadings were made of supematants from two individual cultures of each strain. Goat polyclonal anti-transferrin (human) antisemm (Calbiochem) was used at 40uL per rocket immunoelectrophoresis gel (50mL). A = Control strain [pSAC35]; B = Control strain [pDB2536]; C = Control strain [pDB2711]; D = Control strain [pDB2931]; E = Control strain [pDB2929].<br>
Figure 18 shows Hie results of SDS-PAGE analysis of recombinant transferrin secretion with and without PDI1 over-expression. BMMD shalce flask cultures were grown for 4-days and lOuL supernatant analysed on non-reducing SDS-PAGE (4-12% NuPAGE®, MOPS buffer, InVitrogen) with GelCode® Blue reagent (Pierce). SeeBlue Plus2 Markers (InVitrogen). 1 = pDB2536; 2 = pDB2536; 3 = pDB2711; 4 = pDB2711; 5 = pDB2931; 6 = pDB2931; 7 = pDB2929; 8 = pDB2929; 9 - pSAC35 control.<br>
Figure 19 shows RJE analysis of recombinant transferrin secretion from S. cerevisiae strains with an additional integrated copy ofPDIl. 5-day BMMD shake flask culture supematants were loaded at 5mL per well.   Strains contained: 1) pSAC35   (negative   control);   2)   pDB2536   (recombinant   non-glycosylated transferrin (N413Q, N611Q)) or 3) pDB2506 (same as plasmid pDB2536 but the transferrin ORE encodes transferrin without the N-&gt;Q mutations at positions 413 and 611, i.e. recombinant glycosylated transferrin). Each well contained a sample derived from an individual transformant.   Standards were human plasma holo-transferrin (Calbiochem) at 100, 50, 20,10, 5 and 2mg.L'1. <br>
Figure 20 shows RIB analysis of recombinanl transferrin secretion from Strain A [pDB2536J and Strain A [pDB2506] grown in shake flask culture. 5-day BlsdMI) or YEPD shalce flask culture supernatants were loaded in duplicate at 5niL per well.<br>
Figure 21 shows SDS-PAGE analysis of recombinant transferrin secreted from Strain A [pDB2536] and Strain A [pDB2506] grown in shake flask culture. Cultures were grown for 5~days in BMMD and 30mL supernatants analysed on SDS-PAGE (4-12% NuPAGE™, MOPS Buffer, InVitrogen) stained with GelCode, Blue Reagent (Pierce). 1) Strain A [pDB2536] transformant 1; 2) Strain A [pDB2536] transformant 2; 3) Strain A [pSAC35] control; 4) Strain A [pDB2506] transformant 1; 5) SeeBlue, Plus2 Protein Standards (approximate molecular weights only).<br>
Figure 23 shows PJE of recombinant transferrin secreted from S.  cerevisiae<br>
ty<br>
strains with different PDI1 copy numbers. 3-day BMMD shalce flask culture supernatants were loaded at 5mL per well. Goat polyclonal anti-transferrin (human) antiserum (Calbiochem) was used at 30mL per rocket immunoelectrophoresis gel (50mL). (A) supernatant from S. cerevisiae control strain [pDB2711 ] or [pDB2712]; (B) supernatant from Strain A [pDB2536]; (C) supernatant from control strain [pDB2536].<br>
Figure 24 shows SDS-PAGE analysis of recombinant transferrin secreted from S. cerevisiae strains with different PDI1 copy numbers. 4-12% NuPAGE reducing gel run with MOPS buffer (InVitrogen) after loading with 30mL of 3-day BMMD shalce flask culture supernatant per lane; (lane 1) supernatant from control strain [pDB2536]; (lane 2) supernatant from Strain A [pDB2536]; (lanes 3-6) supernatant from control strain [pDB2711] or [pDB2712]; (lane 7) molecular weight markers (SeeBlue Plus2, InVitrogen).<br>
Figure 26 shows RIE of recombinant transferrin secreted from different S. cerevisiae strains with and without additional PDI1 gene co-expression. lOmL YEPD shake flasks were inoculated with yeast and incubated for 4-days at 30°C. Sjj-L culture supernatant loaded per well of a rocket immunoelectrophoresis gel. Plasma Tf standards concentrations are in p,g/mL. 20uL goat anti-Tf / 50mL agragose. Precipin was stained with Coomassie blue.<br>
Figure 53 shows RIE analysis of rHA expression in different S. cerevisiae strains when co-expressed with PDI1 genes having different length promoters. lOmL YEPD shake flasks were inoculated with yeast and incubated for 4-days at 30°C. 4L culture supernatant loaded per well of a rocket immunoelectrophoresis gel. rHA standards concentrations are in iig/mL. 400uL goat anti-HA (Sigma product A-1151 resuspended in 5mL water) /50mL agarose. Precipin was stained with Coomassie blue.<br>
Figure 54 shows RIE analysis of rHA expression in different S. cerevisiae strains when co-expressed with PDI1 genes having different length promoters. lOmL YEPD shake flasks were inoculated with yeast and incubated for 4-days at 30°C. 4p,L culture supernatant loaded per well of a rocket immunoelectrophoresis gel. rHA standards concentrations are in tig/mL. 400uL goat anti-HA (Sigma product A-1151 resuspended in 5mL water) /50mL agarose. Precipin was stained with Coomassie blue.<br>
Figure 55 shows RIE analysis of rTF expression, when co-expressed with different PDI1 constructs. lOmL BMMD shake flasks were inoculated with yeast and incubated for 4-days at 30°C. 5uL culture supernatant was loaded per well of a rocket immunoelectrophoresis gel containing 25 uL goat anti-Tf/ 50mL. Plasma Tf standards concentrations are in ug/mL. Precipin was stained with Coomassie blue.<br>
Fisure 56 show:- RJE analysis of rTF expression, when co-expressed with different PDJ1 constructs. 10m]-, YEPD shalce flasks were inoculated with yeast and incubated for 4-d.ays at 30°C. 5uL culture supernatant was loaded per well of a rocket immunoeiectrophoresis gel containing 25uL goat anti-Tf / 50mL. Plasma Tf standards concentrations are in ug/mL. Precipin was stained with Coomassie blue.<br>
Figure 72 shows RIE analysis of rHA fusion proteins with and without co-expressed recombinant PDH . 1 OrnL BMMD shalce flasks were inoculated with YBX7 transformed with albumin fusion expression plasmids and incubated for 4-days at 30°C, 4uL culture supernatant loaded per well of a rocket immunoeiectrophoresis gel. rHA standards concentrations are in jig/mL. 200uL goat anti-HA (Sigma product A-1151 resuspended in 5mL water) /50mL agarose. Precipin was stained with Coomassie blue.<br>
Figure 73 shows SDS-PAGE analysis of recombinant albumin fusion secretion with and without PDI1 present on the expression plasmid.   1 OmL BMMD shake flasks were inoculated with yeast and incubated for 4-days at 30°C, 200rpm. 30p.L supernatant analysed on non-reducing SDS-PAGE (4-12% NuPAGE®, MES buffer, In Vitro gen) with Gel Code® Blue reagent (Pierce). 1  = SeeBlue Plus2 Markers (InVitrogen); 2 = lug rHA; 3 = angiostatin-rHA; 4 - angiostatin-rHA + PDI1; 5 = endostatin-rHA; 6 = endostatin-rHA + PDI1; 7 = DX-890-(GGS)4GG-rHA; 8 = DX-890-(GGS)4GG-rHA + PD11; 9 - DP]-14-(GGS)4GG-rHA; 10.= DPl-14-(GGS)4GG-rHA + PDI1;  1 1 - Axoldne™ (CNTFAxl5)-(GGS)4GG-rHA (Lambert el al 2001, Proc. Nail. Acad. Sci. USA, 98, 4652-4657); 12 = Axokine™ (CNTFAxi5) -(GGS)4GG-rHA<br>
Figure 76 shows .RIE analysis demonstrating increased transfenin secretion from 5. cerevisiae with ORM2 co-expression from a 2um~based plasmid.   Four day<br>
shake flask culture supernantants were loaded at 5ul per well.   Standards were<br>
human plasma holo-transferrin (Calbiochem), at 25, 20, 15, 10, 5 ug/mi, loaded 5 pi per well. Goat polyclonal anti-transferrin (human) antiserum (Calbiochem) used at 20 pi per rocket imrnirnoelectrophoresis gel (50 ml).<br>
Figure 80 shows RIB analysis demonstrating increased transferrin secretion, from S. cerevisiae with PSE1 co-expression from a 2p.m-based plasmid. Four day shake flask culture supernantants were loaded at 5 pi per well. Standards were human plasma holo-transferrin (Calbiochem), at 25, 20, 15, 10, 5 fig/ml, loaded 5 pi per well. Goat polyclonal anti-transferrin (human) antiserum (Calbiochem) used at 20ul per rocket immunoelectrophoresis gel (50 ml).<br>
Figure 84 shows RIE analysis demonstrating increased transferrin secretion from iS*. cerevisiae with SSA1 co-expression from a 2pm-based plasmid. Four day shake flask culture supernantants were loaded at 5 pi per well. Standards were human plasma holo-transferrin (Calbiochem), at 25, 20, 15, 10, 5 pg/ml, loaded 5 pi per well. Goat polyclonal anti-transferrin (human) antiserum (Calbiochem) used at 20pi per rocket immunoelectrophoresis gel (50 ml).<br>
Figure 92 shows the results of RIE.   lOmL YEPD shake flasks were inoculated with   DXY1   trplA   [pDB2976],   DXY1   trplA   [pDB2977],   DXY1   trplA [pDB2978], DXY1 fjplA [pDB2979]3 DXY1 trplA [pDB2980] or DXY1 trplA [pDB2981] transformed to tryptophan prototrophy with a 1.41kb NofiJPsfi. pdil::TRPl    disrupting    DNA    fragment    was    isolated   from    pDB3078. Transformants were grown for 4-days at 30°C, 200rpm.  4pL culture supernatant loaded per well  of a  rocket immunoelectrophoresis   gel.     rHA  standards concentrations are in pg/rnL.    700pL goat anti-HA (Sigma product A-1151 resuspended hi 5mL water) /50mL agarose. Precipin was stained with Coomassie blue. Isolates selected for further analysis are indicated (*).<br>
re 93 shows :hc- results oi RLE.   JGniL YEPD sbaice flasks were inoculated with    DXYJ    [pl)B2244].    DXYI     jpDB2976],    DXYI    trplA   pdil::TRP] lpDB2976j. DXYJ   [pDB2978], DXYI trpJA pdiJ::THPJ [pDB2978j, DXYI |pDB2980j; DXYI fr;xM pdi2::TRPl [pDB2980], DXY1  [pDB2977]5 DXY1 /r/;7/S pdil::TRP}   [pDB2977], DXYI   [pDB2979] DXYI  jrp7/l pdil::TRPl [pDB297(-)j. DXYI  [pDB298]] and DXYI trplA pdil::TKPl [pDB2981]s and were grovro for 4-days al 30°C.. 200rpm. 4p,L culture supernatant loaded per well of a rocket immunoelectroplioresis gel.    rHA standards concentrations are in ug/mL.  800uL goat anti-RA. (Sigma product A-l 15 ] resuspended in 5mL water) /50mL. agarose. Precipin was stained with Coomassie blue. Isolates selected for further analysis are indicated (*)<br>
Figure 94 shows a sequence alignment of the SKQ2n and S288c gene sequences with long promoters, as described in Example 6.<br>
EXAMPLES<br>
Two types of expression cassette have been used to exemplify secretion of a recombinant human transferrin mutant (N413Q, N6MQ) from S. cerevisiae. One type, uses a modified HSA(pre)/MFcd(pro) leader sequence (named the "modified fusion leader" sequence). The second type of expression cassette uses only the modified HSAfpre] leader sequence.<br>
The    24    amino    acid    sequence    of   the    "modified    fusion    leader"    is MK WVFIVSILFLFSSA YSRSLDKR.<br>
The  IS amino acid sequence of the modified HSA(pre) leader sequence is MKr\7FWSILFLFSSAYS.<br>
Transferrin (N413Q, N611Q) expression using these two cassettes has been studied in S. cerevisiae using the 2p.m expression vector with and without an additional copy of the S. cerevisiae PDI gene, PD11.<br>
EXAMPLE 1<br>
Construction of expression plasmids<br>
A 52-bp linker made by annealing O.SmM solutions of oligonucleotides CF86 and CF87 (see below) was introduced into the US-region of the 2pm plasmidpSAC35 at the Xcml-sites in the 599-bp inverted repeats. One Xcml-site cuts 51-bp after the REP2 translation termination codon, whereas the other Xcml-site cuts 127-bp before the end of the FLP coding sequence, due to overlap with the inverted repeat (see Figure 3). This DNA linker contained a core region "SnaBl~Pacl-Fsel/Sfil~ Smal-SnaBI", which encoded restriction sites absent from pSAC35.<br>
Xcml Linker (CF86+CF87)<br>
CF86     GGAGTGGTA CGTATTAATT AAGGCCGGCC AGGCCCGGGT ACGTACCAAT TGA CF87  TCCTCACCAT  GCATAATTAA TTCCGGCCGG  TCCGGGCCCA TGCATGGTTA AC<br>
Plasmid pSAC35 was partially digested with Xcml, the linear 11-kb fragment was isolated from a 0.7%(w/v) agarose gel, ligated with the CF86/CF87 Xcml linker<br>
1	0<br>
(neat, 10" and 10" dilutions) and transformed into E. coli DH5a. Ampicillin resistant transformants were selected and screened for the presence of plasmids that could be linearised by Smdl digestion. Restriction enzyme analysis identified<br>
pDB26o8 (Fieurt 4} witb the linker cloned into the J&amp;wl-site after llkf'2. sequencing  using oiiaonucieotides primers CFS8, CF98 and CF99 (Table  1) confirmed the insertion contained the correct linker sequence.<br>
(Table Removed)<br>
The yeast strain was transformed to leucine prototrophy using a modified lithium acetate method (Sigma yeast transformation kit, YEAST-15 protocol 2; (Ito et al, 1983,./: Bacterial, 153, 163; Elble, 1992, Biotechnigues, 13, 18)). Transformants were selected on BMMD-agar plates, and were subsequently patched out on BMMD-agar plates. Cryopreserved trehalose stocks were prepared from 1 OrnL BMMD shalce flask cultures (24 hrs, 30°C. 200rprn), by addition of an equal volume of sterile 40% (w/v) trehalose<br>
The composition of YEPD and BMMD is described by Sleep .et al, 2002., YeasL 18, 403. YEPS and BMMS are similar in composition to YEPD and BMMD accept that 2% (w/v) sucrose was substituted for the 2% (w/v) glucose as the so]e initial carbon source.<br>
The S. cerevisiae PDIJ gene was cloned into the A'cml-linker of pDB2688:  The PDI1 gene (Figure 5) was cloned on a 1.9-kb Sacl-Spel fragment from a larger S. cerevisiae genorrdc SKQ2n DNA fragment containing the PDI1 gene (as provided in the plasmid pMA3a:C7 that is described in US 6,291,205 and also described as Clone C7 in Crouzet &amp; Tuite, 1987, Mol Gen. Genet, 210, 581-583 and Farquhar el al, 1991, supra], which had been cloned into YIplac211 (Gietz &amp; Sugino, 1988, Gem, 74, 527-534), and had a synthetic DNA linker containing a Sad restriction site inserted at a unique Bsu3 6I-site in the 3' untranslated region of the PDI1 gene. The 1.9-kb Sacl-Spel fragment was treated with T4 DNA polymerase to fill the Spel 5'-overhang and remove the Sad 3'-overhang. This PD11 fragment included 212-bp of the PDI1 promoter upstream of the translation initiation codon, and 148-bp downstream of the translation termination codon.   This was ligated with Smal linearised/calf intestinal alkaline phosphatase treated pDB2688, to create plasmid pDB2690 (Figure 6), with the PDI1  gene transcribed in the same direction as REP2. A S. cerevisiae strain was transformed to leucine prototrophy withpDB2690.<br>
An expression cassette for a human transferrin mutant (N413Q, N611Q) was subsequently cloned into the A/M-site of pDB2690 to create pDB2711 (Figure 7). The expression cassette in pDB2711 contains the S. cerevisiae PRJ51 promoter, an HSAMFa fusion leader sequence (EP 387319; Sleep et al, 1990, Biotechnology (N.Y.), 8, 42) followed by a coding sequence for the human transferrin mutant (N413Q, N611Q) and the S. cerevisiae ADH1 terminator. Plasmid pDB2536 was constructed similarly by insertion of the same expression cassette into the Notl-siteofpSACSS.<br>
The "modified fusion leader" sequence used in pDB2536 and pDB2711 comprises •a modified HSA-pre sequence and a MFccl-pro sequence. An alternative leader sequence used was the modified HSA-pre sequence, which was derived from the<br>
modified fusion leader sequence by removal of the six residues of the MFal-pro sequence.<br>
The modified fusion leader sequence in pDB2515 (Figure 8) was mutated with oligonucleoiides CF154 and CF155 to delete the coding sequence for the six residues (RSLDKJR.) of the MFal-pro region. This was performed according to the instruction manual of the Statagene's QuickCliange™ Site-Directed Mutagenesis Kit. pDB2515 is the E. coli cloning vector pGEM-7Z(-) (Promega) containing the 2940-bp Nofi-Hin&amp;lll (partial) DNA fragment of pDB2529 (see below) ligated between the PspQML and Hindis, sites.<br>
CF154<br>
5'-GTTCTTGTTCTCCTCTGCTTACTCTGTCCCTGATAAAACTGTGAGATGG-3'<br>
CF155<br>
5' -CCATCTCACAGTTTTATCAGGGACAGAGTAAGCAGAGGAGAACAAGAAC-3'<br>
Competent E. coli DH5a cells were transformed with the mutated plasrnids and ampicillrn resistant colonies were selected.   Plasmid DNA from these colonies was screened by double digestion with EcoKl and .BgTIL    The correct DNA sequence ibr the modified HSA-pre leader was  subsequently confirmed in pDB2921 (Figure 9) over a 386-bp region between the AfRl and jSamHI sites either side of the leader sequence.    This 386-bp AflR-jBam'Hl fragment was isolated, and ligated with a 6,081-bp AJHI-BarnHI fragment from pDB2529 (Figure 10), prepared by partial digestion with £am~HI and complete digestion with AflQ. and calf intestinal alkaline phosphatase. pDB2529 is the E. coli cloning vector pBST(+) (Sleep et al, 2001, Yeast, 18, 403-441) containing the transferrin expression cassette of pDB2536 cloned into the unique Norl-site.  This produced pDB2928 (Figure 11), which was isolated from ampicillrn resistant E. coli DH5a cells transformed with the ligation products.<br>
The  3,256-bp Noil expression  cassette was  isolated from pDB2928.     This<br>
contained the PKB1 promoter, the coding region for the modified HSA-pre leader<br>
sequence followed by transferrin (N413Q, N611Q), and the ADH1 terminator.<br>
This was ligated into the Not! sites of the 2jam-based vectors pSAC35 and<br>
pDB2690 to generate the expression plasmids pDB2929, pDB2930, pDB2931 and<br>
pDB2932 (Figures 12-15).   In pDB2929 and pDB2931 the transferrin (N413Q,<br>
N611Q) sequence is transcribed in the same direction as LEW, whereas in<br>
pDB2930 and pDB2932 transcription is in the opposite direction.<br>
EXAMPLE 2 Expression of transferrin<br>
A S. cerevisiae control strain was transformed to leucine prototrophy with all the transferrin (N413Q, N611Q) expression plasmids, and cryopreserved stocks were prepared.<br>
Strains were grown for four days at 30DC hi lOmL BMMD cultures in 50mL conical flasks shaken at 200rpm.   The titres of recombinant transferrin secreted into the culture supematants were compared by rocket irrrmunoelectrophoresis (REE as described in Weeke, B., 1976, "Rocket rmmunoelectrophoresis' In -N. H. Azelsen,   J.   Kroll,   and   B.   Weeke   [eds.J,   A   manual   of   quantitative rmmunoelectrophoresis. Methods and applications. Universitetsforlaget, Oslo, Norway), reverse phase high performance liquid chromatography (RP-HPLC) (Table 2), and non-reducing SDS polyacrylamide electrophoresis stained with colloidal Coomassie blue stain (SDS-PAGE).    The increase in recombinant transferrin secreted when S. cerevisiae PDI1 was over-expressed was estimated to be greater than 10-fold.<br>
Table 2:<br>
(Table Removed)<br>
RJE analysis indicated that the increased transferrin secretion in the presence of additional copies of PDI1 was approximately 15-fold (Figure 16). By RIE analysis the increase appeared slightly larger for the modified HSA-pre leader sequence than for the modified fusion leader sequence (Figure 17).<br>
By RP-HPLC analysis the increase .in transferrhl secretion was determined to be 18-fold for the modified fusion leader sequence and 15-fold for the modified HSA-pre leader sequence (Table 2).<br>
Figure  18  shows an  SDS-PAGE comparison of the recombinant transferrin secreted by S. cerevisiae strains with and without additional PDI1 expression.<br>
RP-HPLC Method for Determining Transferrin Expression<br>
Column:	50 x 4.6mm Phenomenex Jupiter C4 300A, 5pm<br>
Column temperature:        45°C<br>
Flow rate:        ImL.min'1<br>
Peak: detection:UV absorbance at 214nm<br>
HPLC mobile phase A:0.1% TFA, 5% Acetonitrile<br>
HPLC mobile phase B:0.1% TFA, 95% Acetonitrile<br>
Gradient:        0 to 3 minutes 30% B<br>
3 to 13 minutes 30 to 55% B in a linear gradient<br>
13	to 14 minutes 55% B<br>
14	to 15 minutes 55 to 30% B in a linear gradient<br>
15	to 20 minutes 3 0%B<br>
Injection:        Generally lOOuL of sample, but any volume can be injected<br>
Standard Curve:	0.1 to 1 Ou.g of human transferrin injected vs peak area<br>
Standard curve used for the results shown was linear up to lOfig.<br>
y = 530.888.x + 10526.7 where y - peak area, and x = amount in p.g. (r2): 0.999953, where Correlation Coefficient = r<br>
EXAMPLE 3<br>
Chromosomal over-expression ofPDI<br>
S. cerevisiae Strain A was selected to investigate the secretion of recornbinaiit glycosylated trans ferric expression from plasmid pDB2506 and recornbinant non-glycosylated transferrin (N413Q, N611Q) from plasmid pDB2536. Strain A lias the following characteristics -<br>
•	additional chromosomally integrated PDI1 gene integrated at the host<br>
PDI1 chromosomal location.<br>
•	the URA3 gene and bacterial DNA sequences containing the ampicillin<br>
resistance gene were also integrated into the S. cerevisiae genome at the<br>
insertion sites for the above genes.<br>
A control strain had none of the above insertions.<br>
Control strain [cir°j and Strain A [cir1] were transformed to leucine prototrophy with pDB2506 (recombinant transferxin), pDB2536 (recombinant non-glycosylated transferrin (N413Q, N611Q)) or pSAC35 (control). Transformants were selected on BMMD-agar.<br>
The relative level of transferrin secretion in BMMD shake flaslc culture was determined for each strain/plasmid combination by rocket immunoelectrophoresis (RIE). Figure 19 shows that both strains secreted both the glycosylated and non-glycosylated recombinant transferrins into the culture supernatant.<br>
The levels of both the glycosylated and non-glycosylated transferrins secreted from Strain A [pDB2506] and Strain A [pDB2536] respectively, appeared higher<br>
than the levels secreted from the control strain. Hence, at least in shake flask culture, PDI1 integrated into the host genome at the PDI1 locus in Strain A has enhanced transferrin secretion.<br>
Furthermore, the increase in transferrin secretion observed between control strain [pDB2536] and Strain A [pDB2536] appeared to be at least a 100% increase by RIE.   In contrast, the increase in rHA monomer secretion between control strain [pDB2305] and Strain A [pDB2305] was approximately 20% (data not shown). Therefore, the increase in transferrin secretion due to the additional copy of PDI1 in Strain A was surprising large considering that transferrin has 19 disulphide bonds, compared to rHA with 17 disulphide bonds. Additional copies of the PDI1 gene may be particularly beneficial for the secretion from S. cerevisiae of proteins from the transferrin family, and their derivatives.<br>
The levels of transferrin secreted from Strain A [pDB2536]  and Strain A [pDB2506] were compared by RIE for transformants grown in BMMD and YEPD (Figure 20).  Results indicated that a greater than 2-fold increase in titres of both non-glycosylated recombinant transferrin (N413Q, N611Q) and glycosylated recombinant transferrin was achieved by growth in YEPD (10-20 mg.L"1 serum transferrin equivalent)  compared to BMMD  (2-5  mg.L"1   serum transferrin equivalent).   The increase in both glycosylated and non-glycosylated transferrin titre observed in YEPD suggested that both transferrin expression plasmids were sufficiently stable under non-selective growth conditions to allow the expected increased biomass which usually results from growth in YEPD to be translated into increased glycosylated and non-glycosylated transferrin productivity.<br>
SDS-PAGE analysis of non-glycosylated transferrin (N413Q, N611Q) secreted from Strain A [pDB2536] and glycosylated transferrin from Strain A [pDB2506] grown in BMMD shake flask culture is shown in Figure 21. Strain A [pDB2536] samples clearly showed an additional protein band compared to the Strain A<br>
[pSAC35] control.   This extra band migrated at the expected position for the<br>
recombiriani transferrm (N413Q.N6] 1 Q) secreted from control strain [pDB2536]. Strain A [pDB2505] culture supernatants appeared to contain a diffuse protein band at the position expected for transfenin. Tins suggested that the secreted recombinant transferrin was heterogeneous, possibly due to hyper-mannosylation at Asp413 and/o]- Asp 611.<br>
EXAMPLE 4<br>
Comparing transferrin secretion from S. cerevisiae control strain containing pDB2711 with transferrin secretion from S. cerevisiae Strain A<br>
Plasmid pDB2711 is as described above. Plasmid pDB2712 (Figure 22) was also produced with the Not! cassette in the opposite direction to pDB2711.<br>
Control strain S. cerevisiae [cir°] was transformed to leucine prototrophy with pDB2711 and pDB2712. Transformants were selected on BMMD-agar and cryopreserved trehalose stocks of control strain [pDB2711] were prepared.<br>
Secretion   of  recombinant   transferrin   (N413Q,   N611Q)   by   control   strain [pDB27ll],  control  strain  [pDB2712],   Strain A  [pDB2536],   control  strain [pDB2536] and an alternative control strain..[pDB2536] was compared in both BMMD and YEPD shake flask culture.  RTE indicated that a significant increase in recombinant transferrin secretion had been achieved from control strain [p"DB2711] with multiple episomal PDI1 copies, compared to Strain A [pDB2536] with two chromosomal copies of PDI1, and control strain [pDB2536] with a single chromosomal copy of PD11 gene (Figure 23).   Control strain [pDB2711] and control strain [pDB2712] appeared to secrete similar levels of rTf (N413Q, N611Q) into the culture media. The levels of secretion were relatively consistent between control strain [pDB2711] and control strain [pDB2712] transformants in both BMMD and YEPD media, suggesting that plasmid stability was sufficient for<br>
high-level transferrin secretion even under non-selective conditions.   This is in<br>
contrast to the previous published data in relation to recombinant PDGF-BB and HSA where introduction of PDIl into multicopy 2um plasmids was shown to be detrimental to the host.<br>
Table 3: Recombinant transferrin litres from high cell density fermentations<br><br>
(Table Removed)<br>
Reducing  SDS-PAGE  analysis   of transferrin  secreted  from   control   strain<br>
[pDB2711],  control  strain  [pDB2712],  Strain A  [pDB2536],   control  strain<br>
[pDB2536] and alternative control strain [pDB2536] in BMMD shake flask<br>
culture is shown in Figure 24. This shows an abundant protein band in all samples<br>
from control strain [pDB2711] and control strain [pDB2712] at the position<br>
expected for transferrin (N413Q, N611Q).   The relative stain intensity of the<br>
transferrin (N413Q, N611Q) band from the different strains suggested that Strain<br>
A [pDB2536] produced more than control strain [pDB2536] and alternative<br>
control strain [pDB2536], but that there was an even more dramatic increase hi<br>
secretion from control strain [pDB2711] and control strain [pDB2712].    The -<br>
increased recombinant transferrin secretion observed was concomitant with the increased PDI1 copy number in these strains.   This suggested that Pdilp levels were limiting transferrin secretion in control strain. Strain A and the alternative control strain., and thai elevated PD11 cop)' number was responsible for increased transferrin secretion. Elevated PDI1 cop}' number could increase the stead}' state expression level of PD11 so increasing the amount of Pdilp activity.  There are a number of alternative methods by which this could be achieved without increasing the copy number of the PDIJ gene, for example the steady state PDI1 mRNA level could he increased by either increasing the transcription rate, say by use of a higher efficiency promoter, or by reducing the clearance rate of the PDI1 mRNA. Alternatively, protein engineering could be used to enhance the specific activity or turnover number of the Pdilp protein.<br>
In  high   cell   density   fermentations   control   strain   [pDB2711]   recombinant transferrin (N413Q, N611Q) production was measured at approximately 3g.L"3 by both GP-HPLC analysis and SDS-PAGE analysis (Table 3).     This level of production is several fold-higher than control strain, the alternative control strain or Strain A containing pDB2536. Furthermore, for the production of proteins for therapeutic use in humans, expression systems such as control strain [pDB2711] have advantages over those using Strain A, as they do not contain bacterial DNA sequences.<br>
CONCLUSIONS<br>
Secretion of recombinant transferrin from a multicopy expression plasmid (pDB2536) was investigated in S. cerevisiae strains containing an additional copy of the PDIJ gene integrated into the yeast genome. Transferrin secretion was also investigated in S. cerevisiae transformed with a multicopy expression plasmid, in which the PD11 gene has been inserted into the multicopy episomal transferrin expression plasmid (pDB2711).<br>
A S. cerevisiae strain with an additional copy of the PDI1 gene integrated into the<br>
genome at the endogenous PDI1 locus, secreted recombinant transferrin and non-<br>
glycosyiated recombinant transferrin (N413Q, N611Q)  at  an  elevated level<br>
compared to strains containing a single copy of PDI1. A further increase in PDI1<br>
copy number was achieved by using pDB2711  In high cell density fermentation<br>
of the  strain transformed with pDB2711,  recombinant transferrin (N413Q,<br>
N611Q) was secreted at approximately Sg.L"1, as measured by SDS-PAGE and<br>
GP-HPLC analysis. Therefore, increased PDI1 gene copy number has produced a<br>
large increase in the quantity of recombinant transferrins secreted from S.<br>
cerevisiae.<br>
The following conclusions are drawn -<br>
1. In shake flask analysis of recombinant transferrin expression from pDB2536 (non-glycosylated transferrin (N413Q, N611Q) and pDB2506 (glycosylated transferrin) the S. cerevisiae strain Strain A secreted higher levels of both recombinant transferrins into the culture supernatant than control strains. This was attributed to the extra copy ofPDIl integrated at the PDI1 locus.<br>
2.	Control strain [pDB2711], which contained the PDI1 gene on the multicopy<br>
expression plasmid., produced a several-fold increase in recombinant transfenin<br>
(N413Q, N611Q) secretion compared to Strain A [pDB2536] in both shake flask<br>
culture and high cell density fermentation.<br>
3.	Elevated PDI1   copy  number .in  yeast  such as S.   cerevisiae will  be<br>
advantageous during the production of heterologous proteins, such as those from<br>
the transferrin family.<br>
4.	pSAC35-based plasmids containing additional copies of PDI1 gene have<br>
advantages for the production of proteins from the transferrin family, and their<br>
derivatives, such as fusions, mutants, domains and truncated forms.<br>
EXAMPLE 5<br>
Insertion  of a PDIl gene into a 2ptn-like plasmid increased secretion  of recombinanl transfetrin from various different S. cerevisiac strains<br>
The S. cerevisiae strain JRY188 cir" (National Collection of Yeast Cultures) and MT302/28B cir"1 (Finnis etal., 1993, Eur. J. Biochem., 212, 201-210) was cured of the .native 2um plasmid by galactose induced over-expression of FLP from ep3S\~GAL-FLPJ, as described by Rose and Broach (1990, Meth. Enzymol., 185, 234-279) to create the S. cerevisiae strains JRY188 cir° and MT302/28B cir°, respectively.<br>
The S. cerevisiac strains JRY188 cir°, MT302/28B cir°, S150-2B cir° (Cashmore et a/., 1986, Mol. Gen. Genet., 203, 154-162), CB11-63 cir° (Zealey et al., 1988, Mol. Gen. Genet., 211. 155-159) were all transformed to leucine prototrophy with pDB2931 (Figure 14) andpDB2929 (Figure 12). Transformants were selected on appropriately supplemented minimal media lacking leucine.    Transformants of each strain were inoculated into lOmL YEPD in 50mL shake flaslcs and incubated m an orbital shaker at 30°C, 200rpm for 4-days.    Culture supeniatants were harvested    and   the   recombinant   transferrin   titres    compared    by    rocket immunoelectrophoresis (Figure 26). The results indicated that the transferrin titres in supernatants from all the yeast strains were higher when PDIl was present in the 2 urn plasmid (pDB2929) than when it was not (pDB2931)<br>
EXAMPLE 6<br>
The construction oj expression vectors containing various PDI1 genes and the expression cassettes for various heterologous proteins on the same 2/jm-like plasmid<br>
PCR amplification and cloning oiPDIl genes into YIplac211:<br>
The PD.U  genes from S.  cerevisiae S288c and S.  cerevisiae SKQ2n were<br>
amplified by PCR to produce DNA fragments with different lengths of the 5'-<br>
untranslated region containing the promoter sequence.    PCR _ primers  were<br>
designed to permit cloning of the PCR products into the EcoRI and BamHl sites of<br>
YIplac211 (Gietz &amp; Sugino, 1988, Gene, 74, 527-534).   Additional restriction<br>
endonuclease   sites  were   also   incorporated  into  PCR primers  to  facilitate<br>
subsequent cloning. Table 4 describes the plasmids constructed and Table 5 gives<br>
the PCR primer sequences used to amplify the PDI1 genes. Differences in the<br>
PDI1 promoter length within these YIplac211-based plasmids are described in<br>
Table 4.<br>
pDB2939 (Figure 27) was produced by PCR amplification of the PDI1 gene from S.  cerevisiae S288c genomic DNA with oligonucleotide primers DS248 and DS250 (Table 5), followed by digesting the PCR product with EcoEl and BamHl and cloning the approximately 1.98-lcb fragment into YIplac211 (Gietz &amp; Sugino., 1988, Gene, 74, 527-534), that had been cut with EcoKL and BamRl.   DNA sequencing of pDB2939 identified a missing  'G' from within the DS248 sequence, which is marked in bold in Table 5.  Oligonucleotide primers used for sequencing the PDI1 gene are listed in Table 6, and were designed from the published S288c PDI1 gene sequence (PDI1/YCL043C on chromosome III from coordinates 50221 to 48653 plus 1000 basepairs of upstream sequence and 1000<br>
basepairs   of  dowisirearn   sequence.   Chtlp://www.yeastgenome.orsj   Gensbanlc Accession number NCOO] 135).<br>
Table 4: YIpiac211-based Plasmids Containing PDIl Genes<br>
(Table Removed)<br>
Table 5:   Oligonucleotide Primers for PCR Amplification of S. cerevisiae PDI1<br>
(Table Removed)<br><br>
Table 61 Oligonucleotide Primers for DNA Sequencing S. cerevisiae PDll Genes<br>
(Table Removed)<br>
srmds pDB2cJ-1.i ("Figure 28) and pDB2942 ('Figure 29) were construcied similar]} using tht PCR primers described hi Tables 4 and 5. and by cloning the approximate!} i.90-kb and 1.85-kb EcaRl-JBamltt fragments, respectively, into Ylplac23 i. The correct DNA sequences were confirmed for the PD1J genes iu pDB294J andpDE2942.<br>
The S. cerevisiae S'KQ2n PD.U gene sequence was PCR amplified from plasmid DNA containing the PDI1 gem from pMA3a:C7 (US 6,291,205), also known as Clone C7 (Crouzei &amp; Tuite, 1987, supra; Farquhar &amp;i d, 1991, supra).    The SKQ2n PD1I gene was amplified using oligonucleotide primers DS248 and DS250 (Tables 4 and 5).   The approximately 2.01-kb PCR product was digested with EcoPJ and BamHl and Iigared into YIplac211 (Gietz &amp; Sugino, 1988, Gene, 74, 527-534) that has been cut with EcoEI and SamlU, to produce plasmid pDB2943 (Figure 30).  The 5' end of the SKQ2n PDI1 sequence is analogous to a blunt-ended Spel-site extended to include the EcoKL, Sad, SnaBL Pad, Fsel, Sfi] • and Smal sites, the 3' end extends up to a site analogous to a blunt-ended £su361 site, extended to include a Smal, SnaBI and Bam~Hl sites.   The PDJJ promoter length is approximately 210bp. The entire DNA sequence was determined for the PDJJ fragment using oligonucleotide primers given in Table 6.   This confirmed the presence of a coding sequence for the PDI protein of S.  cerevisiae strain SKQ2n (NCB1 accession number CAA38402), but with a serine residue at position 114 (not an arginine residue as previously published).   Similarly, in the same way as in the£ cerevisiae S288c sequence in pDB2939, pDB2943 also had a missing 'G' from within the DS24S sequence, which is marked in bold in Table<br>
Plasmids pDB2963 (Figure 31) and pDB2945 (Figure 32) were constructed similarly using the PCR primers described in Tables 4 and 5, and Ijy cloning the approximately ].94-kb and 1.87-kb EcoRI-Bam'Hl fragmentSj respectively, into YIplac21 J. The expected DNA sequences were corrfirmed for the PDI1 genes in<br>
pDB2963 and pDB2945, with a serine codon at the position of arnino acid 114.<br>
The construction of pSAC35-based rHA expression plasmids with different PDI1 genes inserted at the Xc/nl-site after KEP2:<br>
PSAC35-based plasmids were constructed for the co-expression of rHA with different PDI1 genes (Table 7).<br>
Table 7: pSAC35-based plasmids for co-expression of rHA with different PDI1 genes<br>
(Table Removed)<br>
.The riiA expression cassette from pDB2243 (Figure 33, as described in WO 00/44772) was first isolated on a2,992-bp Not] fragment, wiiich subsequently was cloned into the jVo/L-site of pDB2688 (Figure 4) to produce pDB2693 (Figure 34). pDB2693 was digested with SnaBl. treated with calf intestinal alkaline phosphata.se. and legated with SnaBl fragments containing the PDIl genes from PDB2943, j:»DB2963.. pDB2945, pDB2939, pDB2941 and pDB2942. This produced plasmids pDB2976 to pDB2987 (Figures 35 to 46). PDIl transcribed in the same orientation as REP2 was designated "orientation A", whereas PDIl transcribed in opposite orientation to KEP2 was designated "orientation B" (Table 7).<br>
The   construction   of pSACS'5-based transferrin   expression  plasmids   with different PDIl genes inserted at the Xcml-site after REP2:<br>
pSAC3 5-based plasmids were constructed for the co-expression of recombinant transferrin (N4]3Q, N611Q) with different PDIl genes (Table 8).<br>
Table 8:  pSAC35-based plasmids for co-expression of transferrin with different PDIl e.enes<br>
(Table Removed)<br>
In order to achieve thus, the NotI expression cassettes for rHA expression were first deleted from pDB2976, pDB2978, and pDB2980 by NotI digestion and circularisation of the vector backbone. This produced plasmids pDB3081 (Figure 47), pDB3083 (Figure 48) and pDB3084 (Figure 49) as described in Table 9.<br>
Table 9: pSAC35-based plasmids with different PDI1 genes<br>
(Table Removed)<br>
The 3,256-bp NotI fragment from pDB2928 (Figure 11) was cloned into the Notl-sites of pDBSOSl, pDB3083 and pDB3084, such that transcription from the transferrin gene was in the same direction as LEU2. This produced plasmids pDB3085 (Figure 50), pDB3086 (Figure 51) and pDB3087 (Figure 52) as described in Table 8.<br>
EXAMPLE 7<br>
Insertion and optimisation of a PDI1 gene in the 2pm-like plasmid increased the secretion of recombinant human serum albumin by various different S. cerevisiae strains<br>
The S. cerevisiae strains JRY188 cir°, MT302/28B cir°, S150-2B cir°, CB11-63 cir° (all described above), AH22 cir° (Mead et a/., 1986, Mol. Gen. Genet., 205,<br>
4} 7-421 ) and J)S5(&gt;9 cir° (Sleep ci aL. 1991, Bio.-7'eclmulogy, 9, ] 83-1 87) were transformed to ieucine prototrophy with either pDB2244 (WO 00/44772), pDB2976 (Figure 35), pDB2978 (Figure 37j or pDB2980 (Figure 39) using a modified lithium acetate method (Sigma yeast transformation Idt, YEAST-1, protocol 2; (Ito ct d, 1983,7. Bacterial., 153, 163; Elble, 1992, Bioteclmiques, 13, 18)). Transfonnants were selected on BMMD-agar plates with appropriate supplements, and were subsequently patched out on BMMD-agar plates with appropri ate supp 1 ements.<br>
Transfonnants of each strain were inoculated into lOmL YEPD in 50mL shake flasks and incubated in an orbital shaker at 30°C, 200rpm for 4-days.   Culture supernatants were harvested and the recombinanl albumin titres compared by rocket immunoelectrophoresis (Figures 53 and 54).  The results indicated that the albumin titres in the culture supernatants from all the yeast strains were higher when PD11 was present in the 2p.m plasmid than when it was not (pDB2244). The albumin titre in the culture supernatants in the absence of PDIJ on the plasmid was dependant upon which yeast strain was selected as the expression host, however, in most examples tested the largest increase in expression was observed when PDI1 with the long promoter (~210-bp) was present in the 2p-m plasmid (pDB2976). Modifying the PDI1 promoter by shortening, for example to delete regulation sites, had the affect of controlling the improvement.   For one yeast strain, know, to be a high rHA producing strain (DS569) a shorter promoter was preferred for optimal expression.<br>
EXAMPLE 8<br>
Different PDIl genes enhanced the secretion of recombinant transfertin when co-expressed on a 2 pn-based plasmid.<br>
The secretion of recombinant transferrin (N413Q, N611Q) was investigated with co-expression of the S. cerevisiae SKQ2n PDIl gene with the long promoter (•-210-bp), and the S. cerevisiae S288c PDIl with the long, medium and short promoters (-210 bp, -140 bp and -80 bp respectively).<br>
The same Control Strain as used in previous examples (e.g. Example 2) was transformed to leucine prototrophy with pDB2931 (negative control plasmid without PDIl} and pDB2929, pDB3085, pDB3086 and pDB3087 (Table 8). Transformants were selected on BMMD-agar plates and five colonies selected for analysis. Strains were grown in lOmL BMMD and lOmL YEPD shake flask cultures for 4-days at 30QC, 200rpm and culture supematants harvested for analysis by rocket irnmunoelectrophoresis (RIE).<br>
Figure 55 shows that in minimal media (BMMD) the S. cerevisiae SKQ2n PDIl gene with the long promoter gave the highest rTF (N413Q, N611Q) titres. The S. cerevisiae S288c PDIl gene gave lower rTF (N413Q, N611Q) titres, which decreased further as the PDIl promoter length was shortened.<br>
Figure 56 shows that in rich media (YEPD) the S. cerevisiae SKQ2n PDIl and S. cerevisiae S288c PDIl genes with the long promoters gave similar rTF (N413Q, N611Q) production levels. Also, the shorter the promoter length of the S. cerevisiae S288c PDIl gene the lower was the rTF (N413Q, N611Q) production level.<br>
EXAMPLE' 9<br>
PD11 on the 2pa-based plasmid enhanced the secretion of recombinant albumin fusions.<br>
The affect of co-expression of the 5. cerevisiae SKQ2n PDI1 gene with the long promoter (--210-bp) upon the expression of recombinant albumin fusions was investigated.<br>
The construction of a Noll N-terminal endostatin-albumin expression cassette (pDB2556) has been previously described (WO 03/066085).   Appropriate yeast vector sequences were provide by a "disintegration" plasmid pSAC35 generally disclosed   in   EP-A-286   424   and   described   by   Sleep,   D.5   et   al.,    1991, Bio/Technology, 9, 183-187.    The 3.54kb Nofi N-terminal endo statin-albumin expression cassette was isolated from pDB2556, purified and ligated into Notl digested pSAC35, which had been treated with calf intestinal phosphatase. creating plasmid pDB3099 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 57). An appropriate yeast PDI1 vector sequences were provide by a "disintegration" plasmid pDB2690 (Figure 6). The 3.54kb Notl N-terminal endo statin-albumin expression cassette was isolated from pDB2556, purified and ligated into Notl digested pDB26905 which had been treated with calf intestinal phosphatase, creating plasmid pDB3100 containing the Nofi expression cassette in the same orientation to the LEU2 selection marker (Figure 58).<br>
The construction of an Notl N-terminal angiostatin-alburnin expression cassette (pDB2556) has been previously described (WO 03/066085), as has the construction of a pSAC35-based yeast expression vector, pDB2765 (Figure 59). The 3.77kb Notl N-terminal angiostatm-albiunin expression cassette was isolated from pDB2556, purified and ligated into Notl digested pDB2690, an appropriate<br>
yeast PDTJ expression vector, which had been treated with calf intestinal phosphatase, creating plasmid pDB3107 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 60).<br>
The construction of an Notl N-terminal Kringle5-(GGS)4GG-albumin expression cassette (pDB2771) has been previously described (WO 03/066085), as has the construction of a pSAC35-based yeast expression vector, pDB2773 (Figure 61). The 3.27kb Notl N-terminal Kringle5-(GGS)4GG-albirmin expression cassette was isolated from. pDB2771, purified and ligated into Notl digested pDB2690, an appropriate yeast PDI1 expression vector, which had been treated with calf intestinal phosphatase, creating plasmid pDB3104 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 62).<br>
The construction of an Notl N-terminal DX-890-(GGS)4GG-alburnin expression cassette (pDB2683) has been previously described (WO 03/066824). Appropriate yeast vector sequences were provide by the "disintegration" plasmid pSAC35.."" The 3.20kb Notl N-terminal DX-890-(GGS)4GG-albumin expression cassette was isolated from pDB2683, purified and ligated into Notl digested pSAC35, which had been treated with calf intestinal phosphatase, creating plasmid pDB3101 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 63).  An appropriate yeast PDI1 .vector sequences were provide by a "disintegration" plasmid pDB2690 (Figure 6).   The 3.20kb Notl N-terminal DX-S90-(GGS)4GG-albumin expression cassette was isolated from pDB2683, purified and ligated into Notl digested pDB2690, which had been treated with calf intestinal phosphatase, creating plasmid pDB3102 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 64).<br>
The construction of an Notl N-terminal DPI-14-(GGS)4GG-albumin expression cassette (pDB2666) has been previously described (WO 03/066824), as has the<br>
construction of a pSAC35-based yeast expression vector, pDB2679 (Figure 65).<br>
The 3.21]:b Notl .N-terminal I)P144-(GGS&gt;iGG--albumin expression cassette was isolated from pDB2666, purified and ligated into Noll digested pDB2690. an appropriate yeast PDI1 expression vector, which had been treated with calf intestinal phosphatase, creating plasmid pDB3103 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 66).<br>
CNTF was cloned from human genomic DNA by amplification of the two exons using the following primers for exon 1 and exon 2, respectively, using standard conditions.<br>
Exon 1 primers:<br>
5'"CTCGCTACCCAGCTGACTTGTTTCCTGG-3'; and f. ' --ATAGGATTCCGTAAGAGCAGTCAG-3'<br>
EXOD 2 primers:<br>
5 ' -GTGAAGC7;.TCAGGGCCTGAAC- 3 ; '   and<br>
5 ' -CTCTCTAGAAGCAAGGAAGAGAGAAGGGAC-3 '<br>
Both fragments were ligated under standard conditions, before being re-amplified by PCR using primers 5'-CTCGGTACCCAGCTGACTTGTTTCCTGG-3' and 5 '-CTCTCTAGAAGCAAGGAAGAGAGAAGGGAC-3' and cloned into vector pCR4 (Invitrogen). To generate Axoldne™ (as disclosed in Lambert et al, 2001, PNAS, 98, 4652-4657) site-directed mutagenesis was employed to introduce Cl 7A (TGT-&gt;GCT) and Q63R (CAG-&gt;AGA) mutations. DNA sequencing also revealed the presence of a silent T-»C substitution V85V (GTT-^GTC) as described hi WO 2004/015113.<br>
The   Axoldne™   cDNA   was   amplified   by   PCR   using    single    stranded oligonucleotides MH33  and MH36 to  create an approximate  0.581cbp  PCR<br>
fragment.<br>
MH33<br>
5 ' -ATGCAGATeTTTGGATAAGAGAGCTTTCACAGAGCATTCACCGCTGACCCC-3'<br>
MH36<br>
5' -CACCGGATCCACCCCCAGTCTGATGAGAAGAAATGAAACGAAGGTCATGG-3'<br>
This was achieved with FastStart Taq DNA polymerase (Roche) in a 50mL reaction, which was initiated by a 4-rninute incubation at 95°C and followed by 25 cycles of PCR (95°C for SOsecs, 55°C for SOsecs, 72°C for 60sec). A PCR product of the expected size was observed in a lOmL sample following electrophoresis in an ethidiurn bromide stained 1% agarose gel. The remaining PCR product was purified using a QIAquick PCR purification kit (Qiagen) and digested to completion with BamHI and BgHI. DNA of approximately the expected size was excised from an ethidiurn bromide stained 1% (w/v) agarose gel and purified.<br>
Plasmid pDB2573X provided a suitable transcription promoter and terminator., along with a suitable secretory leader sequence and DNA sequences encoding part of a (GGS)4GG peptide linker fused to the N-terminus of human albumin. The construction of pDB2573X has been previously described (WO 03/066824).<br>
The 0.571cb BcanKL and BgKL digested PCR product was ligated with pDB2573X, which had been digested with SamHI, BgUL and calf intestinal alkaline phosphatase to create plasmid pDB2617 (Figure 95) and the correct DNA sequence confirmed for the PCR generated fragment and adjacent sequences using oligonucleotide primers CF84, CF85, PRB and DS229.<br>
CF84<br>
5' -CCTATGTGAAGCATCAGGGC-3'<br>
CF85<br>
PRB<br>
DS229<br>
rj' -CTTCTCACAGnTTCAGCAGATTCGTCAG-B'<br>
Plasmid pDB2617 was digested with Ndel and NotI, and the 3.586-kb NotI expression cassette for AxoldneTM-(GGS)4GG-albumin secretion was purified from an agarosegel.<br>
Appropriate yeast vector sequences were provided by the "disintegration" plasmid pSAC35.       The   3.586kb   NotI   N-terminal   A^oldneTM-(GGS)4GG-albirmin expression cassette was isolated from pDB2617, purified and ligated into NotI digested pSAC35, which had been treated with calf intestinal phosphatase, creating plasmid pDB2618 containing the NotI expression cassette in the sarne orientation to the LEU2 selection marker (Figure 96).   Appropriate yeast PDI1 vector sequences were provide by a "disintegration" plasmid pDB2690 (Figure 6). The 3.586kb Not] N-terminal Axoldne™-(GGS)4GG-albumin expression cassette was isolated from pDB2617. purified and ligated into NotI digested pDB2690, which had  been treated with  calf intestinal phosphatase,   creating  plasmid pDB3106 containing the NotI expression cassette in the same orientation to the LEU2 selection marker (Figure 68).<br>
A human IL10 cDNA (NCBI accession number (NM_000572) was amplified by PCR using single stranded oligonucleotides CF68 and CF69.<br>
CF68<br>
5 ' -GCGCAGATCTTTGGATAAGAGAAGCCCAGGCCAGGGCACCCAGTCTGAGAACAGCTGCAC- 3'<br>
CF69<br>
5' -GCTTGGATCCACCGTTTCGTATCTTCATTGTCATGTAGGCTTCTATGTAG-3'<br>
The 0.43kb DNA fragment was digested to completion with BamHL and partially digested with BgUI and the 0.42kb Bglll-BamHl DNA fragment isolated.<br>
Plasmid pDB2573X provided a suitable transcription promoter and terminator, along with a suitable secretory leader sequence and DNA sequences encoding part of a (GGS)4GG peptide linker fused to the N-terminus of human albumin. The construction of pDB2573X has been previously described (WO 03/066824).<br>
Plasmid pDB2573X was digested to completion with BglH. and BamHL, the 6.21kb DNA fragment was isolated and treated with calf intestinal phosphatase and then ligated with the 0.42kb BgRl/BamHI N-terminal IL10 cDNA to create pDB2620 (Figure   69).      Appropriate  yeast  vector  sequences   were provided   by   the "disintegration'1 plasmid pSAC35. The 3.51kb Notl N-terminal IL10-(GGS)4GG-albumhi expression cassette was isolated from pDB2620, purified and ligated into Notl digested pSAC35, which had been treated with calf intestinal phosphatase, creating plasmid pDB2621 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 70). An appropriate yeast PDI1 vector sequences were provide by a "disintegration" plasmid pDB2690 (Figure 6). The 3.51kb Notl N-terminal IL10-(GGS)4GG-albmnin expression cassette was isolated from pDB2620, purified and ligated into Notl digested pDB2690, which had been treated with calf intestinal phosphatase, creating plasmid pDB3105 containing the Notl expression cassette in the same orientation to the LEU2 selection marker (Figure 71).<br>
The same control yeast strain as used in previous examples was transformed to leucine prototrophy using a modified lithium acetate method (Sigma yeast transformation kit, YEAST-1, protocol 2; (Ito et al, 1983, J. Bacterial., 153, 163;<br>
lble. 1992, Bioieclmiqucs, 13, 18)). Transformants were selected on BMMD-agai plate;;, and were subsequently patched out on BMMD-agar plates. Cryopreserved treh.al.ose stocks were prepared from lOmL BMMD shake flask cultures (24 hrs, 30DC, 200ipm).<br>
Transformants of each strain were inoculated into lOmL BMMD in 50mL shalce flasks and incubated in an orbital shalcer at 30°C, 200rpm for 4-days. Culture supernatant;; were harvested and the recornbinant albumin fusion titres compared by rocket immunoelectrophoresis (Figure 72). The results indicated that the albumin fusion Litre in. the culture supernatants from yeast strain was higher when PD11 was present in the 2um plasmid than when it was not.<br>
The increase in expression of the albumin fusions detected by rocket immunoelectrophoresis was further studied by SDS-PAGE analysis. BMMD shake flask cultures of YBX7 expressing various albumin-fusions were grown for 4-days in an orbital shalcer at 30°C, 200rpm. A sample of the culture supernatant was analysed by SDS-PAGE (Figure 73). A protein band of the expected size for the albumin fusion under study was observed increase in abundance.<br>
EXAMPLE 10<br>
Co-expression of S. cerevisiae ORM2 and recombinant transferrin on a 2^mi-based plasmid<br>
The ORM2 gene from S. cerevisiae S288c was cloned into the Xcml-site after REP 2 on a pSAC35-based plasmid containing an expression cassette for rTf (N413Q, N611Q) at the Nottt-siie in the UL-region.<br>
Plasmid pDB2965 (Figure 74) was constructed by insertion of the 3,256-bp NotI fragment containing the rTf (N413Q, N611Q) expression cassette from pDB2928<br>
(Figure 11) into the JVM-site of pDB2688 (Figure 4). pDB2688 was linearised by Notl digestion and was treated with alkaline phosphatase. The rTf expression cassette from pDB2928 was cloned into the Notl site of pDB2688 to produce pDB2965, with the transferrin gene transcribed in the same direction as LEU2.<br>
The ORM2 gene was amplified from S. cerevisiae S288c genomic DNA by PCR with oligonucleotide primers GS11 and GS12 (Table 10) using the Expand High Fidelity PLUS PCR System (Roche).<br>
Table 10:    Oligonucleotide Primers for PCR Amplification of S.  cerevisiae<br>
Chaperones<br>
(Table Removed)<br>
Primers were designed to incorporate SndBl and Pad restriction recognition sites at the 5' end of the forward primer and SnaBl and Fsel restriction recognition sites at the 5' end of the reverse primer for cloning into the linker at theXcml-site of the<br>
vector, pDB2%5.  PCR was carried out under the following conditions: 200<br>
dNTP mix.. 2,5 U of Expand HiFi enzyme blend. 1 * Expand HiFi reaction buffer.<br>
0.8 fig genomic D'NA; 1 cycle of 94°C for 2 minutes,, 30 cycles of 94°C for 30<br>
seconds. 55°C for 30 seconds, 72°C for 3 minutes, and 1  cycle  72°C for 7<br>
miniates.    0.4 uM of each primer was used.  The required 1,195-bp PCR product<br>
and the pDB2965 vector were digested with Pad and Fsel. ligated together and<br>
transformed   into   competent   E.    coli   DH5a   cells.       Ampicillin   resistant<br>
transformants were selected.    (97LA/2-contahiing constructs were identified by<br>
restriction enzyme analysis of plasmid DNA isolated from the ampicillin resistant<br>
clones.  Four plasmid clones were prepared pDB3090, pDB3091, pDB3092: and<br>
pBD3093, all of which had the same expected DNA fragment pattern during<br>
restriction analysis (Figure 75).<br>
The S. cerevmae Control Strain and Strain A (as described in Example 3) were selected to investigate the effect on transferrin secretion when the transferrin and ORM2 genes were co-expressed from the 2um-based plasrnids. The Control Strain and Strain A were transformed to leucine prototrophy by plasrnids pDB3090s pDB3092 and pBD3093, as well as a control plasmid pDB2931 (Figure 14), containing the rTf (N413Q, N611Q) expression cassette without ORM2. Transformants were selected on BMMD agar and patched out on BMMD agar for subsequent analysis.<br>
To investigate the effect of ORM2 co-expression on transferrin secretion, 1 OmL selective (BMMD) and non-selective (YEPD) liquid media were inoculated with strains containing the ORM2/transfemn co-expression plasrnids. The shake flask culture was then incubated at 30°C with shaking (200 rpm) for 4 days. The relative level of transferrin secretion was determined by rocket gel immunoeletrophoresis (RIE) (Figure 76).<br>
Levels of transferrin secreted from Control Strain [pDB3090] and Control Strain<br>
[pDB3092] were greater than the levels from Control Strain [pDB2931] in both<br>
BMMD and YEPS media.  Similarly, the levels of transferrin secreted from both<br>
Strain A [pDB3090] and Strain A [pDB3093] were greater than the levels from<br>
Strain A [pDB2931] in both BMMD and YEPS media. Transferrin secretion from<br>
all Strain A transformants was higher than the Control Strain transformants grown<br>
in the same media. Strain A contains an additional copy of PDI1 in the genome,<br>
which enhanced transferrin secretion.    Therefore in Strain A, the increased<br>
expression of ORM2 and PDI1 had a cumulative effect on the secretion of<br>
transferrin.<br>
EXAMPLE 11<br>
Co-expression of S. cerevisiae PSE1 and recombinant transferrin on a 2\jm-based plasmid<br>
The PSE1 gene from S. cerevisiae S288c was cloned into the Xcml-site after REP 2 on a pSAC35-based plasmid containing an expression cassette for rTf (N413Q, N611Q) at the Notl-site in the UL-region.<br>
The 3.25-kp wild-type PSE1 gene was amplified from S.  cerevisiae S288c genomic DNA by PCR with oligonucleotide primers CED009 and CED010 (Table 10) using the Expand High Fidelity PCR Kit (Roche).  Primers were designed to incorporate BamHl restriction recognition sites at the 5' end to facilitate cloning into the vector, pUC19.  PCR was carried out under the following conditions: 1 cycle of 94°C for 2 minutes;'10 cycles of 94°C for 15 seconds, 45°C for 30 seconds, 68°C for 4 minutes and 30 seconds; 20 cycles of 94°C for 15 seconds, 45°C for 30 seconds, 68°C for 4 minutes and 30 seconds (increasing 5 seconds per cycle); and 1 cycle of 68°C for 10 minutes.   The required PCR product was digested with BamHl then ligated into pUC193 which had been digested with<br>
JJamr]}  ana ire.ai.ed  with  alkaline phosphatase. producing  construct pDB2848 (Figure 77).  Sequencing of pDB2848 confirmed that amplified sequences were as expected for S. cercvisiae S288c PSE1, when compared to the sequence from P5'£7/YMI1308C on chromosome XIII from coordinates 892220 to 888951 plus 1000 baseparrs of upstream sequence and 1000 basepairs of downstream sequence (Saccharomyces Genome Database at http ://www.yeastgenorne.org/).   The PSE1 gene was then excised from pDB2848 by BamHI digestion, and the resulting 4,096-bp fragment phenohchlorofonn extracted, ethanol precipitated and treated with DMA polymerase Klenow fragment to fill in the 5'-overhang.    Plasmid pDB2965 (Figure 74) was linearised by SnaBl digestion, and alkaline phosphatase treated.   The linearised pDB2965 vector and the PSE1 insert were ligated, and transformed   inlet   competent   E.    coli   DHSex   cells.       Ampicillin   resistant transformants were selected.    Plasmids pDB3097 (Figure 78) and pDB3098 ("Figure 79) were identified to contain the PSE1 gene by restriction enzyme analysis of plasmid DNA isolated from the arnpicillm resistant clones.     In pDB3097 the PSE1 gene is transcribed in the same orientation as REP2, whereas in pDB3097 the PSE1 gene is transcribed in the opposite orientation to REP2.<br>
The S. cerevisiae Control Strain was transformed to leucine prototroprry by plasmids, pDB3097 and pBD309S, as well as a control plasmid pDB2931 (Figure 14), containing the iTf (N413Q, N611Q) expression cassette without PSEL Transformants were selected on BMlxdD agai' and patched out on BMMD agar for subsequent analysis.<br>
To investigate the effect of PSEJ expression on transferrin secretion, flasks containing lOni'L selective (BMMD) liquid media were inoculated with strains containing the /\SE7./transferrin co-expression plasmids. The shake flask culture was then incubated at 30°C with shaking (200 rpm) for 4 days. The relative level of transferrin secretion was determined by rocket gel immunoeletrophoresis (RIE) (Figure 80).<br>
Levels of transferrin secreted from Control Strain [pDB3097] and Control Strain [pDB3098] were greater than the levels from Control Strain [pDB2931] in BMMD media. Therefore, expression ofPSEl from the 2p.m-based plasmids had enhanced transferrin secretion from S. cerevisiae. Transferrin secretion was improved with the PSE1 gene transcribed in either direction relative to the REP2 gene in pDB3097 and pDB3098.<br>
EXAMPLE 12<br>
Co-expression of S. cerevisiae SSA1 and recombinant transferrin on a 2pm-based plasmid<br>
The SSA1 gene from S. cerevisiae S288c was cloned into the A'cml-site a&amp;&amp;iREP2 on a pSAC35-based plasmid containing an expression cassette for rTf (N413Q, N611Q) at the 7/o/l-site in the UL-reglon.<br>
The 1.93-kb SSA1 gene was amplified from S. cerevisiae S288c genomic DNA by PCR with oligonucleotide primers CED037 and CED038 (Table 10) using the Expand High Fidelity PCR Kit (Roche).   Primers were designed to incorporate Sphl restriction recognition sites at their 5' ends to facilitate cloning into the vector, pUC19.  PCR was carried out under the following conditions: 1 cycle of 94°C for 10 minutes, 35 cycles of 94°C for 1 minute, 55°C for 1 minute, 72°C for 5 minutes, and 1 cycle of 72°C for 10 minutes.  The required PCR product was digested with Sphl then ligated into pUC19, which had been digested with Sphl and treated with alkaline phosphatase, producing construct pDB2850 (Figure 81). Sequencing of pDB2850 confirmed the expected sequence of S. cerevisiae S288c SSA1/YALOQ5C on chromosome I from coordinates 141433 to 139505 plus 1000 basepairs of upstream sequence and 1000 basepairs of downstream published in the Saccharomyces Genome Database (http ://www.yeastgenome. org/Q.<br>
The SSAJ gene was excised from pDB2fJ50 by iSp/zl-digestion, and the resulting 2,748-bp fragment phenol:chloroform extracted, e'thano] precipitated and treated with T4 D'NA polymerase to remove the 3'-overhang.   Plasinid pDB2965 was linearised by 5'oaBl digestion and treated with calf alkaline phosphatase.    The linearised pDB2965 vector and the SSAJ insert were ligated and transformed into competent E. coll DHScx cells.  Ampicillin resistant transformants were selected. SSA1 constructs pDB3094 (Figure S2), and pDB3095 (Figure 83) were identified by restriction enzyme analysis of plasmid DNA isolated from the ampicillin resistant clones.  In pDB3094, the SSAJ gene is transcribed in the same direction as REP2. whereas in pDB3095 the SSA2 gene is transcribed in the opposite direction to REP2.<br>
The S. cercvisiae Control Strain was transformed to leucine prototrophy by plasmids, pJ.)B3094 and pBD3095, as well as a control plasmid pDB2931 (Figure 14), containing the rTf (N413Q, N611Q) expression cassette without SSA1. Transformants were selected on BMMD agar and patched out on BMMD agar for subsequent analysis.<br>
To investigate the effect of SSA1 expression on transferrin secretion, flasks containing ]OmL selective (BMMD) liquid media were inoculated with strains containing the SSAJ /transferrin co-expression plasmids. The shake flask cultures were incubated at 30°C with shaking (200 rpm) for 4 days. The relative level of transferrin secretion was determined by rocket gel knmunoeletrophoresis (RIE) (Figure 84).<br>
Levels of transferrin secreted from Control Strain [pDB3095] were greater than the levels from Control Strain [pDB2931] and Control Strain [pDB3094] in BMMD media. Therefore, expression of SSA1 from the 2um-based plasmids had enhanced transferrin secretion from S. cerevisiae. Transferrin secretion was<br>
improved with the. SSA1 gene transcribed in the opposite direction relative to the JK£P2geneinpI)B3094.<br>
EXAMPLE 13<br>
PDIl gene disruption, combined with a PDIl gene on the 2/mn-based plasmid enhanced the secretion of recombinant albumin and plasmid stability.<br>
Single stranded oligonucleotide DNA primers listed in Table 11 were designed to amplify a region upstream of the yeast PDIl coding region and another a region downstream of the yeast PDIl coding region.<br>
Table 11: Oligonucleotide primers<br>
(Table Removed)<br>
Primers DS299 and DS300 amplified the 5' region of PDIl by PCR, while primers DS301 andDS302 amplified a region 3' of PDIl, using genornic DNA derived<br>
S28&amp;C a;: a template.   The PCR conditions were as follows:   ] uL S288c template<br>
D1\A (ai O.OlngmL, O.lng/uL, lng/jiL: 1 Ong/uL and lOOng/uL), 5uL lOXBuffer<br>
(Fast Start Taq-i-Mg, (Roche)), luL lOniM dNTP's, 5uL each primer (2pM).<br>
0.4uL Fast Stan T'aq, made up to 50uL with FLO. PCRs were performed using a<br>
Perkin-Elmer Thermal Cycler 9700.   The conditions were: denature at 95°C for<br>
4mm [HOLD], then [CYCLE] denature at 95°C for 30 seconds, anneal at 45°C for<br>
30 seconds, extend at 72°C for 45 seconds for 20 cycles, then [HOLD] 72°C for<br>
1 Groin and then [HOLD] 4°C.   The 0.221cbp PDT1 5' PCR product was cut with<br>
Notl and Brndlll. while the 0.34kbp PD11 3' PCR product was cut with Hin$m<br>
and Pstl.<br>
Plasnud pMCSS (Hoheisel, 1994, Biotechniques 17, 456-460) (Figure 85) was digested to completion with Hindlll, blunt ended with T4 DNA polymerase plus dNTPs and rehgated to create pDB2964 (Figure 86).<br>
Plasmid pDB2964 was Hmdlll digested, treated with calf intestinal phosphatase, and ligated with the 0.22kbp PDI1 5' PCR product digested with Notl and Hindm and the 0.34kbp PDIJ 3' PCR product digested with Hmdlll and Pstl to create pDB3069 (Figure 87) which was sequenced with forward and reverse universal primers and the D'NA sequencing primers DS303, DS304, DS305 and DS306 (Table 11).<br>
Primers DS234 and DS235 (Table 12) were used to amplify the modified TRP1 marker gene from Ylplac204 (Gietz &amp; Sugino, 1988, Gene, 74, 527-534), incorporating Hindlll restriction sites at either end of the PCR product. The PCR conditions were as follows: luL template YIplac204 (at O.Olng/uL, O.lng/pL, Ing/uL, 10ng/uL and 100ng/uL), 5j.iL lOXBuffer (Fast Start Taq+Mg, (Roche)), luL lOrnM dNTP's, 5uL each primer (2|iM), 0.4p,L Fast Start Taq, made up to 50uL with HoO. PCRs were performed using a Perkin-Elmer Thermal Cycler<br>
9600. The conditions were: denature at 95°C for 4min [HOLD], then [CYCLE]<br>
denature at 95°C for 30 seconds, anneal for 45 seconds at 45°'C, extend at 72°C for 90sec for 20 cycles, then [HOLD] 72°C for lOmin and then [HOLD] 4°C. The 0.86kbp PCR product was digested with Hindlll and cloned into the Hindlll site of pMCSS to create pDB2778 (Figure 88). Restriction enzyme digestions and sequencing with universal forward and reverse primers as well as DS236, DS237, DS238 and DS239 (Table 12) confirmed that the sequence of the modified TRP1 gene was correct.<br>
Table 12:  Oligonucleotide primers<br>
(Table Removed)<br>
The 0.86kbp TRPJ gene was isolated from pDB2778 by digestion with HindHI and cloned into the Hindlll site of pDB3069 to create pDB3078 (Figure 89) and pDB3079 (Figure 90). A 1.41kb pdil::TRPl disrupting DNA fragment was isolated from pDB3078 or pDB3079 by digestion with NotUPstl.<br>
Yeast strains .incorporating a TUT'] deletion (trpJA) were to be constructed in such a way that no homology to the TRPJ marker gene (pDB277S) should left in the genome   once   the   TrplA   had   been   created,    so   preventing   homologous recombination between future TRP1 containing constructs and the TRP1 locus.  In order to achieve the total removal of the native TRPJ sequence from the genome of the chosen hosl strains, oligonucleotides were designed to amplify areas of the 5' UTR and .3' UTR of the TKP1 gene outside of TRPJ marker gene present on integrating vector Ylplac204 (Gietz :&amp; Sugino, 1988, Gene, 14, 527-534).   The YIplac204 T1U:&gt;J marker gene differs from the native/chromosomal TRPJ gene in that  internal  HindEl,  Pstl  and Xbal  sites  were  removed   by   site   directed mutagenesis  (Gietz &amp; Sugino,  1988,  Gene,  74, 527-534).     The YIplac204 modified  TRP1  marker gene was  constructed from a  1.453kbp blunt-ended genomic fragment EcoRl fragment, which contarned the TRP1 gene and only 102bp of the TRPJ promoter (Gietz &amp; Sugino,  1988, Gene,  74, 527-534). Although this was a relatively short promoter sequence it was clearly sufficient to complement frpl auxotrophic mutations (Gietz &amp; Sugino, 1988, Gene, 74, 527-534).   Only DNA sequences upstream, of the EcoEI site, positioned 102bp 5' to the start of the TRP1 ORF were used to create the 5' TRPJ UTR. The selection of the 3' UTR was less critical as long as it was outside the 3' end of the functional modified  TRP1 marker, which was chosen to be 85bp downstream  of the translation stop codon.<br>
Single stranded oligonucieotide DNA primers were designed and constructed to amplify the 5' UTR and 3' UTR regions of the TRPJ gene so that during the PCR amplification restriction enzyme sites would be added to the ends of the PCR products to be used hi later cloning steps. Primers DS230 and DS231 (Table 12) amplified the 5' region of TRP1 by PCR, while primers DS232 and DS233 (Table 12) amplified a region 3' of TRPJ, using S288c genomic DNA as a template. The PCR conditions were as follows: luL template S288c genomic DNA (at<br>
u.uing/fj.1... u.iLLH/j-Lu, Ing/ul, lOng/uL and 100ng/uL), 5uL lOXBuffer (Fast Start Taq+Mg, ('Roche)), lfj.L lOmM dNTP's, 5uL each primer (2uM), 0.4jaL Fast Start Taq, made up to SOjoL with H2O. PCRs were performed using a Perkin-Elmer Thermal Cycler 9600. The conditions were: denature at 95°C for 4min [HOLD], then )CYCLE] denature at 95°C for 30 seconds, anneal for 45 seconds at 45°C5 extend at 72°C for 90sec for 20 cycles, then [HOLD] 72°C for lOmin and then [HOLD] 4°C.<br>
The 0.19kbp TRP1 5' UTR PCR product was cut with £coRI and Hindlll, while the 0.2kbp TRP1 3' UTR PCR product was cut with Bam~Hl and Hindlll and ligated into pAYESOS linearised with BarnHl/EcoKl to create plasmid pDB2777 (Figure 91). The construction of pAYESOS is described in WO 95/33833 . DMA sequencing using forward and reverse primers, designed to prime from the plasmid backbone and sequence the cloned inserts, confirmed that in both cases the cloned 5' and 3' UTR sequences of the TRP1 gene had the expected DNA sequence.     Plasmid pDB2777  contained  a  TRP1   disrupting  fragment  that comprised a fusion, of sequences derived from the 5' and 3' UTRs of TRP1.  This 0.383kbp TRP1 disrupting fragment was excised from pDB2777 by complete digestion with EcoRI.<br>
Yeast strain DXY1  (Kerry-Williams et al,  1998,' Yeast, 14,  161-169) was transformed to leucine prototrophy with the albumin expression plasmid pDB2244 using a modified lithium acetate method (Sigma yeast transformation Idt, YEAST-1, protocol 2; (Ito et al, 1983, J. Bacterial., 153, 163; Elble, 1992, Biotechniques, 13, 18)) to create yeast strain DXY1 [pDB2244]. The'construction of the albumin expression plasmid pDB2244 is described in WO 00/44772. Transformants were selected on BMMD-agar plates, and were subsequently patched out on BMMD-agar plates,   Cryopreserved trehalose stocks were prepared from lOmL BMMD shake flask cultures (24 hrs, 30°C, 200rpm).<br>
DXV] [pI&gt;B'2244'| was transformed to trypiophan autotrophy Avith tiae- 0.383kbp EcofJ T.RF'J  disrupting DNA fragment from pDB2777 using a nutrient agar incorporating the counter selective tryptophan analogue. 5-fluoroanthranilic acid (5-FAA), as described by Toyn ei al, (2000  Yeasi 16, 553-560).     Colonies resistant to the toxic effects of 5-FAA were picked and streaked onto a second round of 5-FAA plates to confirm that they really were resistant to 5-FAA and to select away from any background growth.   Those colonies which grew were then were re-patched onto BMMD and'BMMD plus tryptophan to identify which were tryptophan auxotrophs.<br>
Subsequently colonies that had been shown to be nyptophan auxotrophs were selected for further analysis by transformation with YCplac22 (Gietz &amp; Sugino, 1988, Gene, 74, 527-534) to ascertain which isolates were trpl.<br>
PCR amplification across the TRPl locus was used to confirm that the tip"<br>
phenotj'pe was due to a deletion in this region. Genomic DNA was prepared from<br>
isolates identified as resistant to 5-FAA and unable to grow on minimal media<br>
without the addition of tryptophan. PCR amplification of the genomic TRPl locus<br>
with primers CED005 and CED006 (Table 12) was achieved as follows:<br>
template genomic DNA, 5uL lOXBuffer (Fast Start Taq+Mg, (Roche)),<br>
lOmM dNTP's, 5uL each primer (2uM), 0.4uL Fast Start Taq, made up to 50|jL<br>
with FbO.   PCRs were performed using a Perldn-Elmer Thermal Cycler 9600.<br>
The conditions were: denature at 94°C for 10mm [HOLD], then [CYCLE]<br>
denature at 94°C for 30 seconds, anneal for 30 seconds at 55°C, extend at 72°C<br>
for 120sec for 40 cycles, then [HOLD] 72°C for lOmin and then [HOLD] 4°C.<br>
PCR amplification of the wild type TRPl locus resulted in a PCR product of<br>
1.34kbp in size, whereas amplification across the deleted TRPl region resulted in<br>
a PCR product 0.84kbp smaller at O.SOkbp.   PCR analysis identified a DXY1<br>
derived trp" strain (DXY1 ti-plA [pDB2244]) as having the expected deletion<br>
event.<br>
The yeasi strain DXY1 trplA [pDB2244] was cured of the expression plasrnid pDB2244 as described by Sleep et al, (1991, Bio/Tec1molo®&gt;, 9, 183-187). DXY1   ti-plA   cir°  was  re-transformed  the  leucine prototrophy   with  either pDB2244, pDB2976, pDB2977, pDB2978, pDB2979, pDB2980 or pDB2981 using a modified lithium acetate method (Sigma yeast transformation kit, YEAST-1, protocol 2; (Ito et al, 1983, J. Bacterial., 153, 163; Elble, 1992, Biotechniques, 13, 18)).  Transformants were selected on BMMD-agar plates supplemented with tryptophan,   and   were   subsequently   patched   out   on   BMMD-agar   plates supplemented with tryptophan.    Cryopreserved trehalose stocks were prepared from 1 OmL BMMD shake flask cultures supplemented with tryptophan (24 hrs, 30°C, 200rpm).<br>
The yeast strains DXY1 trplA [pDB2976], DXY1 trplA [pDB2977], DXY1 trplA [pDB2978], DXY1 trplA [pDB2979], DXY1 trplA [pDB2980] or DXY1 trplA [pDB2981] was transformed to tryptophan prototrophy using the modified lithium acetate method (Sigma yeast transformation kit, YEAST-1, protocol 2; (Ito et al, 1983, J. Bacterial., 153,163; Elble, 1992, Biotechniques, 13, 18)) with a 1.41kb pdil::TRPl disrupting DNA fragment was isolated from pDB3078 by digestion with Notl/Pstl. Transformants were selected on BMMD-agar plates and were subsequently patched out on BMMD-agar plates.<br>
Six transformants of each strain were inoculated into 1 OmL-YEPD in 50mL shake flasks and incubated in an orbital shaker at 30°C, 200rpm for 4-days. Culture supernatants and cell biomass were harvested. Genomic DNA was prepared (Lee, 1992, Biotechniques, 12, 677) from the tryptophan prototrophs and DXY1 [pDB2244], The genomic FDD locus amplified by PCR of with primers DS236 and DS303 (Table 11 and 12) was achieved as follows: luL template genomic DNA, 5uL lOXBuffer (Fast Start Taq+Mg, (Roche)), luL lOmM dNTP's, 5uL<br>
each primer (2uM), 0.4uL Fast Start Taq, made up to 50uL with H20. PCRs were<br>
performed using a Perkin-Elmer Thermal Uyeler 9700.    The conditions were: denature ai 94°C for 4min [HOLD], then [CYCLE] denature at 94°C for 30 seconds, anneal for 30 seconds at 50°C, extend ai 72°C for 60sec for 30 cycles, then [HOLD] 72"C for lOnim aud then [HOLD] 4°C.   PCR amplification of the wild type PDI1   ocus resulted in no PCR product, whereas amplification across the deleted PDll region resulted in a PCR product 0.65kbp.     PCR  analysis identified  that  all  36 potential pdil::TRfl  strains  tested  had   the   expected pdil::TRPl deletion.<br>
The recombinant albumin titres were compared by rocket immulloelectrophoresis (Figure 92). Within each group, all six pd.il::TRP1 disruptants of DXYl frplA [pDB2976]. DXYl trplA [pDB297S], DXYl trplA [pDB2980], DXYl trplA [pDB2977] and DXYl trplA [pDB2979] had ver}' similar rHA productivities. Only the six pdil::TRPl disruptants of DXYl trplA [pDB2981] showed variation in rHA expression titre. The six pdil::TRP1 disruptants indicated in Figure 92 were spread onto YEPD agar to isolate single colonies and then re-patched onto BMMD agar.<br>
Three single celled isolates of DXYl trplApdil::TRPl [pDB2976], DXYl trplA pdil::TRPl  [pDB2978], DXYl  trplA pdil::TRPl  [pDB2980]5  DXYl  trplA pdil::TEPl jpDB2977], DXYl trplApdil::TRPl [pDB2979] and DXYl trplA pdil::TRPl [pDB2981] along with DXYl [pDB2244], DXYl [pDB2976], DXYl [pl)B2978], DXY1 [pDB2980], DXYl [pDB2977], DXYl [pDB2979] andDXYl [pDB2981] were inoculated into lOmL YEPD in 50mL shake flasks and incubated in an orbital shaker at 30°C, 200rpm for 4-days,    Culture supernatants were harvested   and  the   recombinant   albumin  titres   were   compared   by   rocket immunoelectropkiresis (Figure 93). The thirteen wild type PDll and pdil::TRP1 disruptants indicated in Figure 93 were spread onto YEPD agar to isolate single colonies.    One hundred single celled colonies from each strain were then re-<br>
patched onto BMMD agar or YEPD agar containing a goat anti-HSA antibody to detect expression of recombinant albumin (Sleep et a!., 1991, Bio/Technology, 9, 183-187) and the Leu+/rHA+, Leu+/rHA-, Leu-/rHA+ or Leu-/rHA- phenotype of each colony scored (Table 13).<br>
 (Table Removed)<br>
These data indicate plasmid retention is increased when the PDIl gene is used as a selectable marker on a plasmid in a host strain having no chromosomally encoded PDI, even in a non-selective medium such as the exemplified rich medium.<br><br><br><br><br>
WE CLAIM:<br>
1.	A method for producing non-2um-family plasmid protein comprising:<br>
(a)	providing a host cell of the kind such as herein described comprising a 2µm-family plasmid, the plasmid comprising a gene encoding protein comprising the sequence of a chaperone protein and a gene encoding a non-2µm-family plasmid protein;<br>
(b)	culturing the host cell in a culture medium under conditions that allow the expression of the gene encoding protein comprising the sequence of the chaperone protein and the gene encoding a non-2µm-family plasmid protein; and<br>
(c)	purifying the thus expressed non-2µm-family plasmid protein from the cultured host cell or the culture medium;<br><br>
2.	The method as claimed in claim 1 optionally comprising the step of formulating the purified non-2µm-family plasmid protein with a carrier or diluent and optionally presenting the thus formulated protein in a unit dosage form.<br>
3.	Use of a 2µm-family plasmid as an expression vector to increase the production of a fungal (preferably yeast) or vertebrate non-2 µm-family plasmid protein by providing a gene encoding the non-µm-family plasmid protein and a gene encoding a chaperone protein on the same 2µm-family plasmid.<br>
4.	A 2µm-family plasmid comprising a gene encoding a protein comprising the sequence of a chaperone protein and a gene encoding a non-2um-family plasmid protein, wherein if the plasmid is based on the 2µm plasmid then it is a disintegration vector.<br>
5.	A method, use or plasmid as claimed in any preceding claim wherein the chaperone has a sequence of a fungal chaperone.<br>
6.	A method, use or plasmid as claimed in any preceding claim wherein the chaperonne has a sequence of a yeast chaperone.<br><br>
7.	A method, use or plasmid as claimed in any preceding claim wherein the chaperone has a sequence of a mammalian chaperone (preferably a human chaperone).<br>
8.	A method, use or plasmid as claimed in any preceding claim wherein the chaperone comprises the sequence of a protein encoded by any one of AHA1, CCT2, CCT3, CCT4, CCT5, CCT6, CCT7, CCT8, CNS1, CPR3, CPR6, EPS1, EPO1, EUG1, FMO1, HCH1, HSP10, HSP12, HSP104, HSP26, HSP30, HSP42, HSP60, HSP78, HSP82, JEM1, MDJ1, MDJ2, MPD1, MPD2, PDI1, PFD1, ABC1, APJ1, ATP11, ATP12, BTTA, CDC37, CPR7, HSC82, KAR2, LHS1, MGE1, MRS11, NOBA, ECM10, SSA1, SSA2, SSA3, SSA4, SSSC1, SSE2, SIL1, SLS1, UBI4, ORM1, ORM2, PER1, PTC2, PSE1 and HAC1 or truncated intronless HAC1.<br>
9.	A method, use or plasmid as claimed in any preceding claim wherein the chaperone is protein disulphide isomerase, or comprises the sequence of a protein encoded by PSE1, ORM2 or SSA1 or a variant or fragment thereof.<br>
10.	A method as claimed in claim any one of claims 1, 2, 5, 8 or 9 wherein the host cell also expresses a second recombinant gene encoding a chaperone that is different to the first chaperone encoded by the plasmid.<br>
11.	A method as claimed in claim 10, wherein the second recombinant gene encoding a chaperone is chromosomally integrated.<br>
12.	A method, use or plasmid as claimed in any one of claims 1 to 10 wherein the plasmid comprises two different genes encoding different chaperons, one of which gene is the second recombinant gene encoding a chaperone as defined by claim 10.<br>
13.	A method, use or plasmid as claimed in any one of claims 1 to 12 wherein one of the chaperones is protein disulphide isomerase.<br><br>
14.	A method, use or plasmid as claimed in any one of claims 10 to 13 wherein one of the-chaperones is ORM2.<br>
15.	A method, use or plasmid as claimed in claim 10 or 11 therein the two chaperones are protein disulphide isomerase and ORM2.<br>
16.	A method, use or plasmid as claimed in any preceding claim wherein the non-2µm-family plasmid protein comprises a leader sequence effective to cause secretion in yeast.<br>
17.	A method, use or plasmid as claimed in any preceding claim wherein the non-2 µm-family plasmid protein is a eukaryotic protein, or a fragment or variant thereof, preferably a vertebrate or a fungal (such as a yeast) protein.<br>
18.	A method, use or plasmid as claimed in any preceding claim wherein the non-2µm-family plasmid protein is a commercially useful protein.<br>
19.	A method, use or plasmid as claimed in any preceding claim wherein the non-2µm-family plasmid protein comprises a sequence selected from albumin, a monoclonal antibody, an etoposide, a serum protein (such as a blood clotting factor), antistasin, a tick anticoagulant peptide, transferrin, lactoferrin, endostatin, angiostatin, collagens, immunoglobulins, or immunoglobulin-based molecules or fragment of either (e.g. a dAb, Fab' fragments, F(ab')2, scAb, scFv or scFv fragment), a Kunitz domain protein interferons, interleukins, IL10, IL11, IL2, interferon species and sub-species, interferon species and sub-species, interferon y species and sub-species, leptin, CNTF, CNTFAX15 (Axokine™), IL7-receptor antagonist, erythropoietin (EPO) and EPO mimics, thrombopoietin (TPO) and TPO mimics, prosaptide, cyanovirin-N, 5-helix, T20 peptide, T1249 peptide, HIV gp41, HIV gp120, urokinase, prourokinase, tPA, hirudin, platelet derived growth factor, parathyroid hormone, proinsulin, insulin, glucagon, glucagon-like peptides, insulin-like growth factor, calcitonin, growth hormone, transforming growth factor ß, tumour necrosis factor, G-CSF, GM-CSF, M-CSF, FGF, coagulation factors in both pre and active forms, including but not limited to plasminogen, fibrinogen,<br><br>
thrombin, pre-thrombin, prothrombin, von Willebrcmd's factor, α1-antitrypsin, plasminogen activators, Factor VII, Factor VIII, Factor IX, Factor X and Factor XIII, nerve growth factor, LACI, platelet-derived endothelial cell growth factor (PD-ECGF), glucose oxidase, serum cholinesterase, aprotinin, amyloid precursor protein, inter-alpha trypsin inhibitor, is antithrombin III, apo-lipoprotein species, Protein C, Protein S, or a variant or fragment of any of the above.<br>
20.	A method, use or plasmid as claimed in any preceding claim wherein the non-2µm-family plasmid protein comprises the sequence of albumin or a variant or fragment thereof.<br>
21.	A method, use or plasmid as claimed in any preceding claim wherein the non-2µm-family plasmid protein comprises the sequence of a transferrin family member, preferably transferrin or lactoferrin, or a variant or fragment thereof.<br>
22.	A method, use or plasmid as claimed in any preceding claim wherein the fragment of either, fused directly or indirectly to the sequence of another protein.<br>
23.	A host cell comprising a plasmid as defined by any preceding claim.<br>
24.	A host cell as claimed in claim 23 wherein a chaperone encoded by the plasmid is an essential gene.<br>
25.	A host cell as claimed in claim 24 wherein, in the absence of the plasmid, the host cell does not produce the chaperone.<br>
26.	A host cell as claimed in any one of claims 23 to 25 which is a yeast cell.<br>
27.	A host cell as claimed in claim 26 in which the plasmid is based on pSR1, pSB3 or pSB4 and the yeast cell is Zygosaccharomyces rouxii, the plasmid is based on pSB1 or pSB2 and the yeast cell is Zygosaccharomyces bailli, the plasmid is<br><br>
based on pSM1 and the yeast cell is Zygosaccharomyces fermentati, the plasmid is based on pKD1 and the yeast cell is Kluyverornyces drosophilarum, the plasmid is based on pPM1 and the yeast cell is Pichia membranaefaciens, or the plasmid is based on the 2µm plasmid and the yeast cell is Saccharomyces cerevisiae or Sacccharomyces carlsbergensis.<br>
28.	A host cell as  claimed in claim 27, in which the plasmid is based on the 2µm plasmid and the yeast cell is Saccharomyces cerevisiae or Sacccharomyces carlsbergensis.<br>
29.	A method as claimed in claim 1 wherein the host cell is a host cell as defined by any one ofclaims 23 to 28.<br>
30.	A method as claimed in claim 29 wherein the host cell is a host cell as defined by claim 23, or any other claim dependent thereon.<br>
31       A method as claimed in claim 29 wherein the step (b) involves culturing the host cell in non-selective media, such as a rich media.<br>
32.      A method for producing non-2µm-family plasmid protein comprising:<br>
(a)	providing a host cell comprising a first recombinant gene encoding a protein comprising the sequence of a first chaperone protein, a second recombinant gene encoding a protein comprising the sequence of a second chaperone protein and a third recombinant gene encoding a non-2µm family plasmid protein, wherein the first and second chaperones are different;<br>
(b)	culturing the host cell in a culture medium under conditions that allow the expression of the first, second and third genes; and<br>
(c)	optionally purifying the thus expressed non-2um-family plasmid protein from the cultured host cell or the culture medium; and<br>
(d)	optionally, lyophilising the thus purified protein.<br><br>
33.	The method as claimed in claim 32 optionally comprising the step of formulating the' 25 purified non-2µm-family plasmid protein with a carrier or diluent and optionally presenting the thus formulated protein in a unit dosage form.<br>
34.	A method as claimed in claim 32 or 33 wherein the first and second chaperones comprise the sequence of a protein, encoded by any one of AHAI, CCT2, CCT3, CCT4, CCT5, CCT6, CCT7, CCT8, CNS1, CPR3, CPR6, EPS1, ERO1, EUGI, FMO1, HCH1, HSP10, HSP12, HSP104,HSP26, HSP30, HSP42, HSP60, HSP78, HSP82, JEM1, MDJ1, MDJ2, MPD1, MPD2, PD11, PFD1, ABC1, APJ1, ATP 11, ATP 12, BTT1, CDC37, CPR7, HSC82, KAR2, LHS1, MGE1, MRS11, NOB1, ECM1 0, SSA1, SSA2, SSA3, SSA4, SSC1, SSE2, SIL1, SLS1, UBI4, ORM1, ORM2, PER1, PTC2, PSE1 and HAC1 or truncated intronless HAC1.<br>
35.	A method as claimed in claim any of claims 32 to 35 wherein the first chaperone is protein disulphide isomerase,<br>
36.	A method any claimed in any of claims 32 to 36 wherein the second chaperone is ORM2.<br>
37.	A method as claimed in any one of claims 32 to 36 wherein at least one of the first or second chaperones is encoded by a chromosomally integrated recombinant gene.<br>
38.	A method as claimed in any of claims 32 to 37 wherein at least one of the first or second chaperones is encoded by a gene on a plasmid.<br>
39.	A method as claimed in 38 wherein the plasmid is a plasmid as defined by anyone of 1 to<br>
28.<br>
40.	A host cell comprising a first recombinant gene encoding a protein comprising the<br>
sequence of protein disulphide isomerase (PDI) and a second recombinant gene encoding<br>
a protein comprising the sequence of a transferrin-based protein.<br><br>
41.	Use of a recombinant gene encoding a protein comprising the sequence of protein disulphide isomerase (PDI) to increase the expression of a transferrin-based protein.<br>
42.	A host cell as claimed in claim 40 or use as claimed in claim 41 wherein the transferrin-based protein comprises the sequence of transferrin or any other member of the transferrin family (e.g, lactoferrin), a variant or fragment thereof or a fusion protein comprising transferrin a variant or fragment thereof<br>
43.	A host cell or use as claimed in any of claims 40 to 42 wherein the first recombinant gene encoding a protein comprising the sequence of protein disulphide isomerase (PDI) is provided on a plasmid.<br>
44.	A host cell or use as claimed in claim 43 wherein the plasmid is a 2um family plasmid.<br>
45.	A host cell or use as claimed in any of claims 40 to 42 wherein the first recombinant gene encoding a protein comprising the sequence of protein disulphide isomerase (PDI) is chromosomally integrated.<br>
46.	A host cell or use as claimed in claim 47 wherein the first recombinant 20 gene encoding a protein comprising the sequence of protein disulphide isomerase (PDI) is chromosomally integrated at the locus of an endogenously encoded PDI gene, preferably without disrupting the expression of the endogenous PDI gene.<br>
47 A host cell or use as claimed in any one of claims 40 to 46 wherein the second recombinant gene encoding a protein comprising the sequence of a transferrin-based protein is provided on a plasmid.<br>
48.      A host cell or use as claimed in claim 47 wherein the plasmid is a 2µm family plasmid.<br><br>
49.	A host cell or use as claimed in any one of claims 39 to 46 wherein the second recombinant gene, encoding a protein comprising the sequence of a transferrin-based protein is chromosomally integrated.<br>
50.	A host cell or use as claimed in claim 49 wherein the second recombinant gene encoding a protein comprising the sequence of a transferrin-based protein is chromosomally integrated at the locus of an endogenously encoded PDI gene, preferably without disrupting the expression of the endogenous PDI gene.<br>
51.	A host cell comprising a plasmid, the plasmid comprising a gene that encodes an essential chaperone wherein, in the absence of the plasmid, the host cell is unable to produce the chaperone, the plasmid further comprising a recombinant gene encoding a non-2µm-family plasmid protein, such as a non-2um-family plasmid protein as defined in any one of claims 16 to 22.<br>
52.	A host cell as claimed in claim 51, wherein, in the absence of the plasmid, the host cell is inviable.<br>
53.	The host cell as claimed in claim 51 or 52 wherein the chaperone is protein disuiphide isomerase.<br>
54.	A method for producing a non-µm-family plasmid protein comprising the steps of:<br>
(a)	providing  a  host  cell   as   defined  by   any   one   of Claims   51   to   53;   and<br>
(b)	culturing the host cell in a culture medium under conditions that allow the expression<br>
of the essential chaperone and the non-2µ-family plasmid protein.<br>
55.	The method as claimed in of Claim 54 wherein the host cell comprises a plasmid as<br>
defined by any one of Claims 51 to 53.<br><br>
56.	The method as claimed in Claim 54 or 55 wherein step (b) is performed by culturing the<br>
host cell in a non-selective medium, such as a rich or complex medium.<br>
57.	The method as claimed in claim 1 further comprising the step of lyophilising the thus<br>
purified protein.<br>
58.	The method as claimed in claim 49 or 51 further comprising the step of purifying the thus<br>
expressed non-2µm-family plasmid protein from the cultured host cell or the culture<br>
medium.<br>
59.	The method as claimed in claim 58 further comprising the step of lyophilising the thus<br>
purified protein.<br>
60.	The method as claimed in claim 58 or 59 further comprising the step of formulating the<br>
purified or lyophilised non-2µm-family plasmid protein with a carrier or diluent.<br>
61.	The method as claimed in claim 60 further comprising the step of presenting the thus formulated protein in a unit dosage form.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="259242-methods-and-systems-for-updating-a-buffer.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259244-a-lighting-device-for-front-region-of-a-motor-vehicle-emitting-electromagnetic-radiation-during-operation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259243</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3660/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>07-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES BIOPHARMA DK A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGHOEJVEJ 36, DK-2880 BAGSVAERD, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DARRELL SLEEP</td>
											<td>66 LADYBAY ROAD, WEST BRIDGFORD, NOTTINGHAM NG2 5DS, ENGLAND</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GILLIAN SHUTTLEWORTH</td>
											<td>FALT 1, 24-26, MUSTERS ROAD, WEST BRIDGFORD, NOTTINGHAM NG2 7PL, ENGLAND</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHRISTOPHER JOHN ARTHUR FINNIS</td>
											<td>74 HARLAXTON DRIVE, LENTON, NOTTINGHAM NG7 1JB, ENGLAND</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/80</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2004/005462</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-12-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0329681.1</td>
									<td>2003-12-23</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259243-a-method-for-producing-non-2-m-family-plasmid-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:44:34 GMT -->
</html>
