<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259313-reagents-and-methods-for-improving-reproducibility-and-reducing-mispriming-in-pcr-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:28:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259313:&quot;REAGENTS AND METHODS FOR IMPROVING REPRODUCIBILITY AND REDUCING MISPRIMING IN PCR AMPLIFICATION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;REAGENTS AND METHODS FOR IMPROVING REPRODUCIBILITY AND REDUCING MISPRIMING IN PCR AMPLIFICATION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An additive for preventing misprinting in polymerase chain reaction (PCR) amplifications and assays comprising a hairpin oligonucleotide having a stem duplex greater than six nucleotides in length and a stabilized stem terminus. The additive improves PCR amplifications, including LATE-PCR amplifications when added to initial amplification reaction mixtures. It can be included in oligonucleotide sets and in kits for PCR amplification and assays.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TECHNICAL FIELD<br>
This invention relates to nucleic acid amplification reactions and assays utilizing the polymerase chain reaction (PCR), including homogeneous assays, both real-time and end-point.<br>
BACKGROUND<br>
Nucleic acid amplification employing the polymerase chain reaction (PCR) is well known, as are assays that include PCR amplification. See U.S. Patents 4,683,202, 4,683,195 and 4,965,188, and, generally, PCR PROTOCOLS, a guide to Methods and Applications, Innis et al. eds., Academic Press (San Diego, CA(USA) 1990). Homogeneous PCR assays that do not require washing to remove unbound detector reagents or probes and thus can be performed without opening amplification reaction vessels are also well known. Homogeneous PCR assays include both end-point assays, in which amplified product is detected at the end of the amplification reaction, and real-time assays, in which amplified product is detected during some or all of the thermal cycles as the reaction proceeds. See U.S. Patents 5,994,056, 5,487,972, 5,925,517 and 6,150,097.<br>
PCR amplification reactions generally are designed to be symmetric, that is, to make double-stranded amplicons by utilizing a forward primer and a reverse primer that are "matched"; that is, they have melting temperatures that are as close as possible, and they are added to the reaction in equimolar concentrations. A technique that has found limited use for making single-stranded DNA directly in a PCR reaction is "asymmetric PCR." Gyllensten and Erlich, "Generation of Single-Stranded DNA by the Polymerase Chain Reaction and Its Application to Direct Sequencing of the HLA-DQA Locus," Proc. Natl. Acad. Sci. (USA) 85: 7652-7656 (1988); and U.S. Patent 5,066,584. Asymmetric PCR differs from symmetric PCR in that one of the primers is added in limiting amount, typically 1-20 percent of the concentration of the other primer.<br>
More recently we have developed a non-symmetric PCR amplification method known as "Linear-After-The-Exponential" PCR or, for short, "LATE-PCR." See Sanchez<br>
et al. (2004) PNAS 101: 1933-1938, Pierce et al. (2005) PNAS 102: 8609-8614, and published international patent application WO 03/054233 (3 July 2003), which is incorporated herein by reference in its entirety. LATE-PCR takes into account the actual melting temperatures of PCR primers at the start of amplification, referred to as Tm[0]. Tm[Q] can be determined empirically, as is necessary when non-natural nucleotides are used, or calculated according to the "nearest neighbor" method (Santa Lucia, J. (1998) PNAS (USA) 95: 1460-1465; and Allawi, H. T. and Santa Lucia, J. (1997) Biochem. 36: 10581-10594) using a salt concentration adjustment. In our work we have utilized 0.07M monovalent salt concentration.<br>
An undesirable feature of PCR amplifications, reduced in the case of LATE-PCR, is scatter among replicates. Following the exponential phase of the amplification, replicate amplifications followed in real time diverge and plateau at different levels. Scatter indicates that replicates do not have the same reaction kinetics and reduces accuracy. This is a problem for PCR assays generally, but particularly for end-point assays and assays that depend upon the slope of signal during the linear phase.<br>
Another significant problem with PCR amplifications is mispriming, which we believe is manifest in at least three types: Type 1, mispriming that occurs during preparation of reaction mixtures prior to the start of amplification; Type 2, mispriming that occurs during amplification if cycle temperatures include any temperature significantly below the melting temperature of a primer; and Type 3, mispriming that occurs in the late stages of a PCR amplification that is continued after a high concentration of amplicon has been made. Several approaches have been used to address the first type of mispriming. One approach is to modify the polymerase chemically so that it is inactive until heated to a high temperature such as 95 °C. See U.S. Patents 5,677,152 and 5,773,258. Another approach is to bind an antibody to the polymerase to inhibit the polymerase until the reaction is heated to a high temperature such as 95 °C to irreversibly denature the antibody. See U.S. Patent 5,338,671. Yet another approach is to include an aptamer in the reaction mixture. See Doug and Jayasena (1996), J. Mol. Biol. 264: 268 - 278 and U.S. patent 6,020,130. An aptamer is a single-stranded oligonucleotide approximately 30 nucleotides in length that binds to a polymerase and inhibits its ability to extend a recessed 3' end at low temperatures. Aptamers are not<br>
irreversibly denatured at 95 °C, a typical highest temperature for a PCR cycle. Kainz et al. (2000) Biotechniques 28: 278-282 reported that the addition to PCR reaction mixtures of double-stranded DNA fragments having lengths of 16-21 nucleotides in certain amounts inhibit polymerases at temperatures below typical PCR extension temperatures and suppress synthesis of non-specific products. DNA fragments are not irreversibly denatured during PCR cycling. Eppendorf-5 Prime, Inc. markets a proprietary ligand that is said to bind to Taq polymerase in a temperature-dependent manner and to inhibit its binding to double-stranded DNA at temperatures below about 50 °C. Despite these many attempts, mispriming remains a problem with PCR amplifications.<br>
Another manifestation of mispriming during PCR amplification is known as primer-dimer formation and amplification. According to this phenomenon one primer hybridizes to the other primer or to itself and then undergoes extension of the 3' end to generate a small double-stranded amplicon, which can then amplify further or can multimerize and amplify further. Primer-dimer formation can occur in the absence of target.<br>
Quantitative analysis of PCR amplifications has been enabled by real-time detection methods, as the PCR cycle at which fluorescent signal becomes visible above the threshold cycle or CT of reactions is indicative of starting target concentrations. End-point analyses are semi-quantitative at best, due in part to scatter among replicates as the reaction exits exponential amplification. Electrophoretic analysis of double-stranded amplicons is semi-quantitative, and may utilize fluorescently labeled primers. End-point analysis utilizing fluorescently labeled probes, either allele-discriminating probes or mismatch-tolerant probes, are also semi-quantitative at best. By reducing scatter and producing single-stranded product, LATE-PCR offers significant improvement in end-point analysis, but scatter among replicates is often not completely eliminated, leaving quantitative multiplex detection less accurate than desired.<br>
An aspect of this invention is a class of reagent additives to improve product specificity and to eliminate the effects of mispriming in PCR amplification reactions. These additives out-perform existing "hot-start" methodologies in all types of PCR and can be used to prevent the accumulation of undesired products, including primer-dimers<br>
and misprimed amplicons, both at early stages of the reaction and during LATE-PCR reactions having many cycles (typically 60 cycles and more).<br>
Another aspect of this invention is PCR amplification and assay methods, both symmetric PCR or non-symmetric PCR, including but not limited to LATE-PCR, and kits, partial kits, and oligonucleotide sets that include such reagent additives.<br>
SUMMARY<br>
Reagents according to this invention are additives that are capable of preventing one or more manifestations of mispriming in at least some PCR amplifications. By "prevent a manifestation" we mean that a product or products of mispriming are not detected at the end of a reaction by techniques described herein, namely fluorescent DNA dye, gel elecrrophoresis, DNA sequencing and melting-point analysis. Reagents according to this invention may be included in PCR amplification mixtures prior to the start of amplification at relatively low concentration, less than Imicromolar (uM), that is, lO00nM, preferably not more than 650 nanomolar (nM), more preferably not more than 300 nM, and most preferably 50-250 nM, even when utilizing polymerases that have both polymerization activity and 5' - to - 3' exonuclease activity. Reagents according to this invention are modified single-stranded oligonucleotides. Oligonucleotides that may be utilized to construct reagents according to this invention are oligonucleotides broadly. They may be DNA, RNA or mixed DNA-RNA. They may contain modified nucleotides, non-natural nucleotides, for example, 2'O-methyl ribonucleotides, non-natural internucleotide linkages, non-nucleotide linkers, PNA, LNA and added chemical moieties such as capped nucleotides described by Glenn Research.<br>
Reagents according to this invention are single-stranded oligonucleotides that form a stem-and-loop structure, commonly referred to as a "hairpin" structure, in a PCR amplification reaction mixture, although they can also be comprised of a stem-and-loop structure wherein the loop is not comprised of nucleotides. In such a structure, a central portion of the molecule remains single-stranded (not hybridized) (the loop) and the ends hybridize to one another to form the stem. A stem may be blunt ended or one strand may extend beyond the end of the other. A stem may comprise a continuous double-stranded region, or it may include an internal mismatch, causing a bulge. The end of the stem<br>
formed by the termini of the oligonucleotide is stabilized so as to be more tightly bound than a DNA-DNA hybrid. We characterize the stem of a reagent according to this invention by reference to its melting temperature, or Tm. The melting temperature is that temperature in degrees centigrade at which 50% of the complementary sequences of the stem are not-hybridized (open configuration) and 50% of the complementary sequences are hybridized or partially hybridized to self (closed configuration). In this application the "calculated Tm" of a stem means the calculated melting temperature of the stem portion of the corresponding, unstabilized complete DNA oligonucleotide obtained using the M-fold program: Zucker, M. (2003). "Mfold Web Server for Nucleic Acid Folding and Hybridization Prediction." Nucl. Acids Res. 31: 3406-3415, assuming a sodium concentration of 70 millimolar (mM) and a magnesium concentration of 3 mM. In the case of preferred embodiments that have added quencher pairs, preferably non-fluorescent quenchers such as Dabcyl and Black Hole Quenchers that absorb light but emit absorbed energy as heat, Tm is the calculated Tm of the DNA hairpin without the quenchers. In the case of embodiments that have capped nucleotides at their termini, Tm is the calculated Tm of the DNA hairpin containing the equivalent uncapped nucleotides. In the case of 2'-O-methyl ribonucleotide inclusions at a stem end, Tm is the calculated Tm of the all-DNA hairpin containing deoxyribonucleotide analogs of the 2'-O-methyl ribonucleotides. We utilize this methodology for the practical reason that it is difficult to obtain the actual melting point of a stabilized hairpin, recognizing that the actual melting point will be several degrees higher than the calculated Tm due to the stabilizing modification. Reagents according to this invention have a calculated stem Tm that does not exceed 94 °C, and preferably is in the range of 50-85 °C. For some embodiments we prefer a calculated stem Tm that is higher than the primer annealing temperature (typically 55-72 °C of an amplification reaction, that is, in the range of 72-85 °C), but in other embodiments we prefer a calculated stem Tm that is lower than the primer annealing temperature, that is, in the range of 50-71 °C. The complementary sequences of the stem open, become single-stranded, at 95 °C, the temperature used for the strand-melting during PCR cycling. Further, the stem of reagents according to this invention has a length exceeding six nucleotides at temperatures low enough for formation of a closed, self hybridized, confirmation. Currently our preferred embodiments form stems that are<br><br>
9-12 base pairs long, preferably 9-11 base pairs, when closed, more preferably 9-12, most preferably 9-11, continuous complementary base pairs without internal mismatches, most preferably blunt-ended stems that are 9-12 base pairs long and perfectly complementary. The 3' terminus of the reagent is non-extendable by a DNApolymerase in the amplification reaction, so the reaction is not a PCR primer. Any overhanging 3' end is blocked to prevent extension, for example, by adding a phosphate or some blocking chemical moiety.<br>
The length of the loop can be varied considerably. If a loop consists of nucleotides and inter-nucleotide linkages, it is at least three nucleotides in length. Further, if it is just three nucleotides long, it contains a thymidine residue. Our currently preferred nucleotide loops have lengths in the range of 3-22 nucleotides. Loops may also be non-nucleotide chemical linkers, for example, alkylene chains. For carbon-chain linkers we prefer a length of 3-6 carbon atoms, and a most preferred length of 3 carbon atoms. In alkylene carbon chains the remaining two valence electrons of each member of the carbon chain are also engaged in covalent linkages. Such linkages can be to hydrogen atoms, to short-chain alkyl or alkylene groups, to substituents for linking the reagent to a solid surface. Non-oligonucleotide loops comprising chemical linkers are not limited to hydrocarbon chains and may include hetero (not carbon or hydrogen) atoms. We prefer that chemical linkers be electrically neutral so as not to bind to the polymerase. Because the activity of the reagent depends on the closed confirmation of the stem, it is preferred that the composition of the loop, for reagents having loops of more than 5 nucleotides, not be extensively complementarity to any other sequence or sequences in the reaction, or generated by the reaction. A reagent according to this invention is not a hybridization probe for any product of the amplification reaction and does not signal product accumulation.<br>
As indicated, reagents according to this invention include stabilization of the end of the stem remote from the loop such that the end of the stem is more tightly bound than the end of a DNA-DNA hybrid. In our currently most preferred embodiments, the stem is blunt ended, and the terminus of the stem is modified in a manner that inhibits partial f    strand separation compared to a natural DNA-DNA hybrid. We demonstrate below the effect of adding stabilizing interactive chemical moieties to the ends of the stem, both a<br>
pair of Dabcyl moieties covalently attached to the 3' and 5' nucleotides of the stem by means of a commercial linker and a pair of commercially available Black Hole™ quenchers (proprietary quenchers marketed by Biosearch Technologies, Novato, CA, U.S.A.). We also demonstrate the effect of utilizing strongly binding non-natural nucleotides, 2' O-methyl ribonucleotides, at the end of the stem. Preferred stabilizing moieties are non-fluorescent so that the reagents do not contribute background fluorescence to amplifications and amplification assays. Unstabilized oligonucleotide DNA hairpins of reagents according to this invention, added at concentrations below 1 uM, need not substantially reduce mispriming if not modified as described to stabilize the hydrogen-bonded stem terminus, and in many cases do not do so. Thus, stabilization is essential in reagents according to this invention.<br>
Hairpin reagents according to this invention may be described as molecules that contain two complementary nucleotide oligomeres that are held together such that they can form a stem of greater than 6 base pairs, and the open end of the stem is chemically modified by one means or another such that the tendency of the stem to unwind or "fray" at its open end is suppressed. Although a thorough understanding of the molecular mechanism of action awaits further analysis via enzyme kinetics in the presence and absence of reagents according to this invention, as well as structural analysis of the interaction of such reagents with the polymerase via electron microscopy, nuclear magnetic resonance, and X-ray crystallography, a partial understanding of mechanism can be anticipated based on the information presented here and in the scientific literature. The literature teaches that Taq polymerase, like other DNA polymerases, is comprised of a synthetic domain and a 5'exonucleolytic domain. The 5'exonucleolytic domain is not present in the Stoffel fragment. The literature further teaches that the synthetic domain carries out DNA polymerization by first binding to and sliding along double stranded DNA. The synthetic domain has a shape that has been likened to an "open hand" which, in the presence of dNTP's, undergoes a shape change at the polypeptide level upon contact with double stranded DNA. As a result the "fingers" of the "hand" "close" around the double stranded DNA molecule. Mismatched sequences within a double-stranded molecule cause the polymerase to back up for some nucleotides and substitute the correct base using the 3'editor function of the synthetic domain. Synthesis of a new<br>
DNA strand occurs as the enzyme reads the template strand and extends the 3' end of the complementary primer. If the enzyme encounters a 5' tail of an oligonucleotide already bound to the template strand the polymerase can invade the region of the bound oligonucleotide by 1-2 nucleotides and cleave the resulting 5' tail by means of the 5'exonuclease domain of the enzyme.<br>
Based on this information it can be hypothesized that the double-stranded stem of a reagent according to this invention functions, at least in part, by binding to the synthetic domain of the polymerase in the open conformation and causing it to close. The synthetic activity of the polymerase is thereby inhibited. Thus, while not wishing to be bound by any theory, it can be postulated that the First Mode of action is as a temperature-dependent inhibitor of the synthetic activity of the polymerase. The evidence presented below indicates that at least some versions of the reagents are not released by the enzyme at the melting temperature of their double-stranded regions, e.g., stems, but remain bound at higher temperatures, including especially typical PCR extension temperatures. During the denaturation (or strand-melting) step of a PCR cycle, however, reagents of this invention become unbound from DNA polymerase. During the denaturing step of PCR cycles (also referred to as the strand-melting step), typically above 90°C, the stems of hairpin reagents according to this invention melt apart, and they bind again to the polymerase only if and when the temperature is dropped sufficiently for the double-stranded region to reform. Therefore, one may choose to design a reagent according to this invention that is double-stranded only until the first denaturation, that is, is double-stranded when added to a PCR reaction mixture prior to amplification (commonly at room temperature) but is thereafter maintained above the Tm of its stem. For such an embodiment the primer annealing temperature of all amplification cycles and<br>
any LATE-PCR low- temperature detection temperature are higher than the calculated<br>
I stem Tm, preferably by at least 5 °C. Alternatively, one may choose to design a reagent<br>
according to this invention that again becomes double-stranded later during amplification. This can be accomplished by utilizing a stem Tm (calculated as described ) that is higher than the primer annealing temperature used in all or some amplification cycles. Alternatively, in LATE-PCR amplification, this can be accomplished by utilizing a stem Tm that is below the primer - annealing temperature but higher than a low's<br>
temperature detection temperature. The former design type generally will not inhibit polymerization, as indicated by a delay in CT, whereas utilizing a primer annealing temperature below the stem Tm generally will result in a modest delay of the Cjof 1-3 amplification cycles.<br>
Further, while not wishing to be bound by any theory, it can be postulated that a Second Mode of action of reagents according to this invention is as a temperature-dependent inhibitor of the 5'exonucleolytic activity of the polymerase. We show below that reagents according to this invention inhibit the 5' - to - 3' exonuclease activity of the polymerase enzyme at temperatures up to at least 55 °C and do so without unduly suppressing the capacity of the enzyme to extend a DNA strand by polymerization of its 3' end (as PCR primers are extended in PCR amplifications).<br>
In addition, while not wishing to be bound by any theory, it can be postulated that a Third Mode of action of reagents according to this invention is as a temperature-dependent ligand that binds the polymerase causing the polypeptides of the polymerase to change shape. Once the ligand is released from the enzyme, the enzyme remains in the altered shape for some time before returning to its former shape. While in the altered shape the enzyme preferentially binds to completely complementary primer-template hybrids, as compared to partial mismatched primer-template hybrids.<br>
Based on this model it can be anticipated that the capacity of the stem of reagents according to this invention to change the shape of the synthetic domain of the polymerase is different from the capacity of reagents according to this invention to inhibit either the synthetic function of the polymerase (First Mode, above) or the 5'exonuclease function of the polymerase (Second Mode, above). Both the First Mode and the Second Mode of action actually depend on the rate at which the bound reagent is released by the polymerase. Reagents according to this invention which are released rapidly preferentially act by the Third Mode without significantly inhibiting the synthetic activity of the polymerase., even at high concentration. In contrast, reagents according to this invention which are released from the polymerase slowly are likely to be inhibitory at high concentration. By way of example, neither compound 12-CJDD comprised of a stem and a carbon-linker loop of three methylene (-Cflj-) groups, nor compound 12-C3 C3DD comprised of a stem and a carbon-linker loop of six methylene (-C7/2-) groups<br>
only partially inhibit PCR amplification at concentrations of 3000 nM. In contrast, reagents according to this invention having loops comprised of nucleotides typically inhibit PCR amplification completely a concentration of 
As persons familiar with protein-nucleic acid interactions will appreciate, the action of a reagent according to this invention may be by more than one of Modes 1-3, which are not mutually exclusive, but rather are relative. As described below, we have devised a quantitative test (Example 14) that can be used to determine the extent to which a particular molecular structure linking the two oligonucleotides comprising the stem causes the complete compound to act as an enzyme inhibitor (via the First Mode or the Second Mode) or primarily as an enhancer of enzyme specificity, the Third Mode of action.<br>
Based on the above model it can be further anticipated that one or more polymerase molecules can be bound to an equal number of molecules of a reagent according to this invention, which, in turn, are covalently linked to a larger moiety such as bead, particle, or material made of a solid material. The solid could thereby be considered to be "loaded" with polymerase. The linkage of reagents according to this invention to the solid could be temporary or could be cleavable by a variety of means known in the art. Cleavage of the linkage would release the reagent or the reagent-polymerase complex into solution (subsequently the reagent could be released from the polymerase).<br>
Design of reagents according to this invention is within the skill of the art. It will be appreciated that the melting temperature of a stem can be adjusted considerably by varying its G-C content. For example, stems of two preferred embodiments described below both have a length of nine nucleotides, but they have calculated Tm's that differ by 25 °C (81 °C and 56 °C). We commonly use the M-fold computer software program cited above for calculation of stem Tm, in conjunction with one or more possible loop sequences. Loop sequence has not otherwise been found to be significant in mispriming reagents according to this invention. In our designing, we utilize a sequence that forms a perfect blunt-ended stem, that is, a stem with no internal mismatches and no terminal overhang. One could then, by simple trial, evaluate the effect of introducing an internal<br><br>
mismatch (a terminal mismatch is destabilizing and not acceptable, as shown in Example 1) or a short extension of one or, at most, two nucleotides beyond the double-stranded region.<br>
Construction of reagents according to this invention is within the skill of the art. For example, oligonucleotide sequences can be prepared on an oligonucleotide synthesizer. Stabilizing moieties can be included using known methods. For example, Dabcyl's can be added conveniently by starting with a dabcylated column (Glen Research) and concluding synthesis with a Dabcyl-modified nucleotide. Non-natural nucleotides can be used as the first, and penultimate, and last nucleotides of the synthesis.<br>
Once a candidate reagent is designed and constructed, the melting temperature of its stem can be approximated empirically in some cases by adding fluorescent DNA-binding dye and performing a melt analysis while stimulating the dye (recognizing that the dye itself has an effect on Tm). Also, practical useful information regarding the actual melting temperature of a stabilized stem of a reagent according to this invention can in many cases be obtained inferentially by performing real-time PCR amplifications utilizing different annealing temperatures. Adjustments in stem length or G-C content may be made, if needed, to achieve a desired Tm. We then evaluate the effects of a candidate reagent on various manifestations of mispriming and amplification efficiency by determining its performance in a rigorous PCR amplification such as described below in Example 1. A candidate reagent is judged to be a reagent according to this invention if, when included in the reaction mixture of Example 1 at some concentration below lO00nM, preferably not more than 650nM, a clean amplicon is obtained (See FIG. 1). We evaluate candidate assays using the assay of Example 1 except with the intended target and intended primers. However, amplifications and assays (amplification plus detection) according to this invention are not limited to the conditions or procedures of Example 1. While a reagent according to this invention must inhibit at least one manifestation of mispriming when added at a concentration below lO00nM compared to a polymerase concentration (Example 1) of 1.25units per 25 yl of reaction mixture, the reagent can be used at any concentration relative to the polymerase concentration at which it is effective; that is, any concentration (or relative concentration) at which it prevents mispriming but does not substantially prevent amplification. Utilizing the intended primers in the<br>
evaluation will reveal unintended consequences, such as failure to block the 3' end of the reagent or an overlooked complementarity.<br>
This invention includes PCR amplification reactions and assays that include PCR amplification reactions, including reactions wherein the amplification reaction mixture includes a thermostable DNA polymerase having both polymerization activity and 5'-3' exonuclease activity, such as Taq DNA polymerase, and wherein at least one reagent according to this invention is included in the amplification reaction mixture. Many reagents according to this invention inhibit the activity of the polymerase and are added at a concentration relative to the polymerase concentration within the ranges described above prior to or during thermal cycling that is, not more than lO00nM, preferably not more than 650nM for a reaction containing 1.25 units of DNA polymerase per 25 ul of reaction volume. Some reagents according to this invention, while effective at concentrations below lO00nM and preferably at concentrations below 650nM, inhibit the activity of the polymerase to a lesser degree and can be added at concentrations up to IS00nM or even 3000nM for such a polymerase concentration. In assays containing more than one reagent according to this invention, each reagent can have a stem with its own Tm and can be added at its own concentration, such that different reagents function in different portions of the steps in the amplification process. If a reagent of this invention is added during thermal cycling, it should be added without aspirating the reaction mixture into the work area. Such amplification reactions include symmetric PCR amplifications, asymmetric PCR amplifications and LATE-PCR amplifications, any of which may further include reverse transcription, if RNA targets are involved. PCR amplifications may be used to prepare amplified product for any purpose, for example, as a starting material for dideoxy sequencing. PCR amplifications may be combined with detection of amplified product in an assay, including particularly homogeneous assays employing labeled primers, labeled probes or fluorescent DNA-binding dyes such as SYBR Green or ethidium bromide. The assay can be a real-time assay in which detection readings are taken during multiple amplification cycles or an end-point assay in which detection is performed after the completion of amplification. It may be qualitative or quantitative, including but not limited to a quantitative end-point assay. The assay can be<br>
designed to amplify a single double-stranded or single-stranded product, or more than one double-stranded products without or with related single-stranded products.<br>
As used in this application, "LATE-PCR" means a non-symmetric DNA amplification employing the polymerase chain reaction (PCR) process utilizing one oligonucleotide primer (the "Excess Primer") in at least five-fold excess with respect to the other primer (the "Limiting Primer"), which itself is utilized at low concentration, up to 200 nM, so as to be exhausted in roughly sufficient PCR cycles to produce fluorescently detectable double-stranded amplicon, wherein the concentration-adjusted melting temperature of the Limiting Primer at the start of amplification, Tm[oj, is higher than or not more than 5 °C below the concentration-adjusted melting temperature of the Excess Primer at the start of amplification, Tm[o]X, preferably 3-10 °C higher; and wherein thermal cycling is continued for multiple cycles after exhaustion of the Limiting Primer to produce single-stranded product, namely, the extension product of the Excess Primer. LATE-PCR assays may include a low temperature detection step in which the temperature is reduced below the primer annealing temperature during at least some cycles of linear amplification. Preferably such a step occurs following extension before strand melting.<br>
This invention also includes complete PCR kits, partial kits, and oligonucleotide sets. A complete PCR amplification kit includes at least all the reagents for carrying out a PCR amplification or assay, including at least one pair of PCR primers, dNTP's, reaction buffer, thermostable DNA polymerase, preferably a polymerase having 5'- to - 3' exonuclease activity, and at least one reagent according to this invention. A complete kit for a homogeneous PCR assay further includes any additional detection reagents that are needed, for example, a fluorescent DNA dye or fluorescently labeled probes. A complete kit of either type preferably includes reagents for sample preparation and may include, in some embodiments, a reverse transcriptase. Partial kits according to this invention omit at least some ingredients of a complete kit but include at least the thermostable DNA polymerase (and, if needed, reverse transcriptase) and at least one reagent according to this invention. For example, products known commercially as "master mix" or "basic kit" typically omit PCR primers and probes. A preferred partial kit includes all reagents needed for an amplification or assay except for sample preparation. Oligonucleotide sets<br>
according to this invention include at least one pair of PCR primers and at least one reagent according to this invention. They may further include oligonucleotide probes or sequencing primers.<br>
The details of one or more embodiments of the invention are set forth in the accompanying drawings and the description below. Other features, objects, and advantages of the invention will be apparent from the description and drawings, and from the claims.<br>
DESCRIPTION OF DRAWINGS<br>
FIG. 1 shows a gel electrophoresis analysis of samples from a qualitative rigorous PCR assay for mispriming errors designed to test the inhibitory activity of the reagents described in this invention.<br>
FIG. 2 shows that reagents according to this invention enable PCR-based realtime and end-point assays.<br>
FIG. 3 shows a DNA sequence chromatograph obtained from single-stranded DNA products generated after 70 cycles of LATE-PCR amplification in the absence of mispriming errors using the reagents described in this invention.<br>
FIG. 4 shows the effect of compound 9-3 DD on a pentaplex LATE-PCR amplification.<br>
FIG. 5 shows evidence that once the reagents according to the invention bind Taq polymerase they are not released from the polymerase until PCR cycling temperatures reach the denaturation step.<br>
FIG. 6 shows a direct comparison of reagents according to this invention for their effect on DNA polymerase activity at 25 °C.<br>
FIG. 7 illustrates an assay for assessing the effect of the reagents according to this invention on a polymerase's exonuclease activity in the absence of polymerase activity.<br>
FIG. 8 illustrates the capacity of a reagent according to this invention to prevent mispriming in a symmetric PCR assay.<br>
FIG. 9 illustrates an assay for quantifying exonuclease inhibition.<br>
FIG. 10 shows the dose-dependent effect of a reagent according to this invention on formation of primer-dimers and primer oligomers in a LATE-PCR amplification.<br>
FIG. 11 shows the effect of a low concentration of reagent 9-22 DD on duplex LATE-PCR amplifications of two target sequences with two primer pairs.<br>
FIG. 12 shows the effect of varying the stem composition of reagent 9-3 DD on a duplex LATE-PCR amplification of two target sequences with two primer pairs.<br>
FIG. 13 shows the effect of varying the concentration of reagent 9-3b DD on a duplex LATE-PCR amplification of two target sequences with two primer pairs.<br>
FIG. 14 shows the effect of increasing concentrations of reagent 9-3 DD on the efficiency of a LATE-PCR amplification in terms of CT values.<br>
FIG. 15 shows the effect of increasing concentrations of reagent 9-3 DD on the efficiency of a LATE-PCR amplification in terms of fluorescent signal slope and final fluorescence.<br>
FIG. 16 shows products of a multiplex reaction carried out with and without a mixture of compounds according to this invention.<br>
FIG. 17 shows the results of a test for the action of compound 12-3DD.<br>
FIG. 18 shows the results of a test for the action of compound 12-C3DD.<br>
Like reference symbols in the various drawings indicate like elements.<br>
DETAILED DESCRIPTION<br>
As indicated above, we have identified what we believe to be three different types and causes of mispriming errors. To assess mispriming of all three types we have developed the rigorous assay of Example 1 below. Amplification begins with sheared DNA, which promotes mispriming. Physical handling, for example, drawing DNA into a pipette, tends to shear DNA. Further, because only Type 1 mispriming can be prevented by "hot start" reagents and methods, we have included in the assay many temperature cycles (at least 60). In addition we have included a low-temperature detection step utilized in some LATE-PCR assays. Further, we also recognize formation of primer-dimers and their further oligomerization and amplification as an additional type of mispriming. Primer-dimer formation and amplification can occur both in the presence and absence of added template DNAs, and we have also developed assays for these phenomena.<br>
Whereas hot-start enzymes are effective only prior to the start of amplification due to irreversible changes that occur upon heating to high temperature, for example, 95 °C, mispriming reagents according to this invention, are not irreversibly denatured and could have positive or negative effects, or both, later in the reaction, if a low temperature step is included.<br>
One can design stems to have melting temperatures, Tm's, above or below particular cycle temperatures, with different impacts on amplification. In designing stems one preferably takes into account that melting is a dynamic phenomenon occurring over a temperature range, with Tm specifying the temperature at which fifty percent of molecules are in double-stranded form and fifty percent are in single-stranded form. We believe that the binding of some closed stems to polymerase shifts the equilibrium among molecules that remain unbound to polymerase in favor of more stems closing. If we wish the vast majority of stems to be closed at a particular cycle temperature, such as the primer annealing temperature, we generally start with an unmodified oligonucleotide stem having a calculated Tm at least 5 °C above that temperature, recognizing that stabilizing stem modification will raise the actual Tm by several degrees. Conversely, if we wish the vast majority of stems to be open at a particular cycle temperature, we generally start with an unmodified oligonucleotide stem having a calculated Tm of at least 5 °C below, preferably at least about 10 °C below, that particular cycle temperature, again recognizing that stabilizing stem modification likely will raise the actual Tm by several degrees.<br>
Utilizing a particular reagent according to this invention with differing primer annealing temperatures can have differing effects on amplification efficiency, as we demonstrate below in Example 5. One of our currently calculated embodiments (compound 9-22 DD) discussed below has a calculated Tm of 81 °C. It will close during every typical PCR thermal cycle as the temperature is lowered from the strand-melting temperature (for example, 95 °C) to the primer annealing temperature (for example, 55 °C). A reagent with such a stem will have an effect during each cycle of amplification. Another of our currently preferred embodiments (compound 9-3 DD) has a calculated Tm of 56 °C. When PCR amplification mixtures are prepared at room temperature, this reagent has a closed stem and binds to the polymerase. It remains bound as the<br>
temperature is raised at the commencement of amplification. Polymerase-bound molecules of 9-3DD become unbound and open during the initial high-temperature step, and do not bind to the polymerase again unless the temperature is lowered sufficiently for the stem of the unbound molecules to close. Thus, by keeping the lowest cycle temperature at 65 °C, the stem does not reform. Under these conditions this embodiment will function in that amplification as a hot-start reagent to prevent mispriming that occurs prior to the start of amplification. However, if a low-temperature detection step is included, for example 40 °C incubation following extension in a LATE-PCR cycle to permit hybridization of a low-temperature molecular beacon or other labeled probe to the single-strand accumulating during the linear phase of amplification, the stem having a calculated melting temperature of 56 °C stem will close, permitting the reagent to bind to the polymerase again until a high-temperature of strand melting is achieved.<br>
Example 1 below describes a rigorous LATE-PCR assay that we utilize to evaluate the performance of reagents according to this invention. FIG. 1 shows that mispriming errors occur not only with "regular" Taq DNA polymerases, that is, thermostable DNA polymerases with 5'- to - 3' exonuclease activity but without hot-start modification (lanes b and lanes c), but also with hot-start Taq polymerases (lanes d and lanes e). FIG. 1 further shows the elimination of non-specific products formed due to mispriming errors, and the presence of only the desired specific product, obtained by the addition of reagent 9-22 DD at 300 nM concentration, not only to amplifications with hot-start polymerases (lanes h and lanes i), but also to amplifications with regular Taq DNA polymerases (lanes f and lanes g). The effectiveness of compounds 9-22 DD and 9-3 DD in suppressing mispriming was assessed utilizing the assay described in Example 1. Both were found to suppress Type 1 mispriming. The low calculated Tm of the stem of 9-3 DD (56 °C) permits it to be used as simply an effective hot-start reagent by utilizing a PCR amplification protocol with all cycle temperatures 60 °C and above. By staying above the stem's melting point the stem does not reform as amplification proceeds. Compound 9-3 DD has no inhibitory effect on polymerization efficiency when so used. Example 5 below is instructive in this regard. It compares amplifications utilizing reagent 9-3 DD and annealing temperatures of 65 °C and 55 °C. When 65 °C was utilized, no effect on polymerization efficiency, as reflected by the CT, was found. When<br>
55 °C was utilized, however, the CT was delayed. This is evidence that if the temperature is lowered sufficiently for the stem to close and the polymerase to be engaged, engagement continues as the temperature is raised for primer extension, even if the primer extension temperature is above the stem Tm. On the other hand, the high Tm of the stem of 9-22 DD (81 °C) permits the stem to reform into a hairpin during the primer annealing step of every PCR cycle. It prevents the manifestation not only Type 1 misprinting, but also Type 2 mispriming, Type 3 mispriming, and Primer-Dimer formation. It does affect polymerization efficiency somewhat, however, as evidenced by a 1.5 cycle delay in the threshold cycle (Or) of the PCR amplification followed in real time with SYBR Green 1 or a target-specific molecular beacon probe when used at a 100 nM concentration.<br>
Table I in Example 1 reports the application of that rigorous assay to a series of additives, and further reports the minimum concentration needed to achieve prevention of manifestations of mispriming, that is, the avoidance of detectable levels of non-specific products formed due to mispriming errors, and the presence of only the desired specific amplification product such as is shown in lanes f and g of FIG. 1, using regular Taq DNA polymerase rather than a hot-start version. In Table I compounds are identified by our nomenclature system. Unmodified hairpin molecules are identified by the stem loop length. For example, compound 6-22 has a stem six nucleotides long and a loop twenty-two nucleotides long. Because the loop identifier begins with a number or is only a number ("22" or "3b") the loop is nucleotides. Non-nucleotide loops are specified further. Compound 12-C3C3 has a stem twelve nucleotides long and a loop that is a six-carbon chain, that is, a chain of six methylene (CH2) groups. Various modifications were made to one or both ends of the basic hairpins. These are designated by suffixes, where 3D is a Dabcyl (5'-Dimethoxytrityloxy-5-[(N-4'-carboxy-4-(dimethylamino)-azobenzene)-aminohexyl-3-acrylimido]-2'-deoxyUridine-3l-[(2-cyanoethyl)-(N,N-diisopropyl)]-phosphoramidite) added to the 3' terminal nucleotide, 5D is a Dabcyl added to the 5' terminal nucleotide, DD is a Dabcyl added to both terminal nucleotides, BHQBHQ is a Black Hole Quencher added to both terminal nucleotides, FF is a fluorophore (in this case FAM) added to both terminal nucleotides, AA is an adenosine<br>
(A) added to both terminal nucleotides, and TT is a thymidine (T) added to both terminal nucleotides.<br>
As well as additions, several modifications were made to the basic stems. The suffix 2'OM4 indicates that four terminal nucleotides, the last two on each stem oligonucleotide, or arm, were changed from deoxyribonucleotides to 2'-O-Methyl nucleotides. To change the melting temperature of stems, we changed the G-C content. Compounds in which the stem sequence, but not the stem length, was changed are denoted by a lower-case letter designator preceding the terminal-modifier suffix, for example, 9-3b DD or 9-3i DD. The nucleotide sequences, showing also carbon-chain loops were used, for the various compounds in Table I are presented in Table II. Table I is instructive in several regards. It shows, for example, that in that assay unmodified DNA hairpin oligonucleotides either are not effective or are effective only at high concentrations, 1000 nM or higher. Oligomer 9-22 (stem 9 nucleotides long, loop 22 nucleotides long) was effective only at 3000 nM concentration, the amount reported for mispriming suppression by double-stranded DNA. Oligomers 6-22 (stem 6 nucleotides long, loop 22 nucleotides long) and 9-5 (stem 9 nucleotides long, loop 5 nucleotides long) were not effective even at 3000 nM concentration. However, when the 9-22 oligomer was modified by adding Dabcyl moieties to both ends of the stem to create compound 9-22 DD, a preferred reagent according to this invention, only 50 nM concentration was required to prevent manifestations of mispriming. When the 9-3 oligomer was similarly modified, only 100 nM concentration was required.<br>
Table I shows the effect of stem length. Whereas the several oligomers having stems nine and twelve nucleotides eliminated mispriming at low concentration when modified with a pair of interacting Dabcyl quenchers, oligomer 6-22 did not so respond to modification: composition 6-22 DD, not a reagent according to this invention, was effective only at high (3000 nM) concentration. Reagents according to this invention have stems longer than six nucleotides and shorter than fourteen base pairs in most cases. Lengths shorter than fourteen base pairs are currently preferred.<br>
Table I also shows the effectiveness of other modifications that increase the binding strength at the end of the stem. Addition of a pair of interactive Black Hole™ quenchers to the stem of oligomer 9-5 produced reagent 9-5 BHQBHQ, of which only<br>
100 nM concentration was required to yield elimination of non-specific products formed due to mispriming errors, and the presence of only the desired specific amplification product. Substitution of two 2'-0-methyl ribonucleotides at each end of oligomer 9-5 produced reagent 9-5 2'OM4, for which only 50 nM concentration was required.<br>
Table I shows that the length of the oligonucleotide loop can be varied considerably. Loop lengths of three nucleotides (reagent 9-3 DD), five nucleotides (reagent 9-5 DD), and twenty-two nucleotides (reagent 9-22 DD) all resulted in reagents according to this invention that prevented mispriming at concentrations of 100 nM or less. At the same time Table I shows that some trial and error adjustment is needed for optimization. If a pair of 2'-0-methyl ribonucleotides is substituted for the Dabcyl quenchers in the stabilizing modification of oligomer 9-5 (comparing reagent 9-5 2'OM4 to reagent 9-5 DD), the level of effectiveness is unchanged: only 50 nM concentration is required. However, making the same change to modified oligomer 9-3 (comparing reagent 9-3 2'OM4 to reagent 9-3 DD), the level of effectiveness, while still good, was diminished: 300 nM concentration was required rather than 50 nM concentration. The same data shows that increasing the loop length of reagent 9-3 2'OM4 by two nucleotides to create reagent 9-5 2'OM4 improved effectiveness: only 50 nM concentration was required rather than 300 nM concentration. Candidate reagents can be routinely evaluated and adjusted with the aid of a rigorous assay such as the one described in Example 1.<br>
Table I also shows that compounds comprised of two complementary oligonucleotides held together in opposite polarity by a non-nucleotide bridge (loop) are also active when the free 3' and 5' ends of the oligonucleotides are modified according to this invention (in the tested embodiments by Dabcyl moieties). Table 1 shows that the size of a non-nucleotide bridge is not critical. The bridges described in Table I are chemical linkers comprised of linear chains of 3-6 carbon atoms, but many other variations on non-nucleotide bridges (loops) could be constructed by persons skilled in the art. The data in Table I suggest that a bridge of a 3 carbon-atom chain is better than one of a 6 carbon-atom chain for the same oligonucleotide stem (active at a lower concentration) and, while not rigorously tested, we hypothesize that this difference reflects the relative flexibility of the bridges and their consequent effect or ability to raise<br>
or lower the melting temperature of the stem. We utilize chemically neutral linkers (uncharged), which we prefer.<br>
Different applications of reagents and methods according to this invention are exemplified in Examples 2-4 below. Example 2 describes the utilization of reagent 9-22 DD in a LATE-PCR assay that is a real-time assay but is shown to be suitable for an end-point assay as well. The assay of Example 2 utilizes a detection technique that is the subject of our simultaneously filed United States Provisional patent application no. 60/619,654 titled Primers, Probes and Methods for Nucleic Acid Amplification, which is incorporated herein by reference in its entirety. That detection technique includes adding both a fluorescent DNA dye such as SYBR Gold, which fluoresces when bound to double-stranded DNA, and a fluorophore-labeled hybridization probe complementary to single-stranded amplicon produced in a LATE-PCR amplification following exhaustion of the limiting primer, where the fluorophore is stimulated by emission from the dye; stimulating the dye; detecting emissions from both the dye and the fluorophore; and calculating the ratio of the fluorophore signal to the dye signal. Example 2 thus shows the compatibility of reagents according to this invention with fluorescent probes and fluorescent dyes, and effective prevention of mispriming when used with either or both.<br>
Example 4 describes several multiplex LATE-PCR amplifications utilizing two, three, four and even five primer pairs, and shows amplification of the correct amplicons in all cases when reagent 9-3 DD was added to the reaction mixture. Although amplified products were analyzed by gel electrophoresis for the purpose of Example 4, the multiplex method is also applicable to multiplex assays, including qualitative and quantitative assays, such as real-time assays, employing other detection means such as fluorescent hybridization probes for intended amplicons. For real-time LATE-PCR assays, we prefer to utilize a low- temperature detection step following the step of primer extension and low temperature probes as illustrated, for example, in Example 3.<br>
Example 13 illustrates that amplification reactions with amplification reaction mixtures containing more than one version of the reagents according to this invention can also be utilized. For instance, an amplification mixture can contain both a compound of this invention with a high melting temperature and another compound of this invention with a low melting temperature, each added at its own optimized concentration.    The<br>
stem of compound 9-22DD with a calculated Tm of 81 °C is in the closed conformation at the start of the reaction and soon after the temperature decreases during each thermal cycle, while compound 12-3DD with a calculated Tm of 58°C is only closed at relatively low temperatures. It therefore acts as a "hotstart" to prevent mispriming when the reaction mixture is being prepared at room temperature, but during the subsequent thermal cycles of a PCR amplification having an annealing temperature of 60°C or higher does not close and bind to the polymerase. Mixtures of reagents are particularly useful for construction of multiplex reactions in which multiples pairs of primers spanning a range of annealing temperatures are combined for amplification of multiple amplicons. The mixture of reagents used in Example 13 is better than either single reagent alone.<br>
Example 3 describes a LATE-PCR amplification utilizing reagent 9-3 DD to prepare amplified single-stranded product that is sufficient in amount and sufficiently free of non-specific products arising from mispriming and to be suitable for use as a starting material for sequencing. The amplification was extended to seventy cycles to insure sufficient starting material. We have found that some amplicons are more prone to Type 2 and Type 3 mispriming errors than are other amplicons. In amplifications where such mispriming is more likely, we prefer the stem of the reagent according to this invention to close when the cycling temperature drops from strand-melting step to the primer-annealing step. This can be ensured by adjusting the stem Tm, the annealing temperature, or both. In related work we have found that product dilution is a simple clean-up method for use with sequencing, as disclosed in our simultaneously filed United States Provisional patent application identified above.<br>
We have investigated the functioning and effectiveness of embodiments of reagents of this invention as compared to unmodified DNA hairpin-forming oligomers, and one another. We devised a DNA polymerization assay to determine the extension of a labeled primer by Taq DNA polymerase in the presence of a test reagent. The assay utilizes primers that are labeled with fluorophores that are excited by emission from a fluorescent dye, as disclosed in our simultaneously filed United States Provisional patent application identified above. The assay mixture comprises 0.5 uM concentration of a synthetic oligonucleotide DNA template, 1.5uM concentration of a DNA primer complementary to the 3' end of the template and labeled with Cy5 at its 5' end, PCR<br>
buffer, MgCh, Taq DNA polymerase, a 1:40,000 dilution of SYBR Green fluorescent DNA dye, and the test reagent. Controlled initiation of the reaction is achieved by addition of dNTP's. The reaction is isothermal: it proceeds at a prescribed temperature for a prescribed amount of time. For evaluating the effect of the test compound on the 5'-3' exonuclease activity of the DNA polymerase, we add to the reaction mixture a "blocker", namely an oligonucleotide that is complementary to the 5' end of the template, that is blocked at its 3' end with a phosphate, and that is labeled with ROX at its 5' end. For evaluating the effect of the test compound on the polymerization activity of the DNA polymerase, the blocker is not used. To facilitate analysis of reaction products by melting-curve analysis, the template, primer and blocker are designed so that the following hybrids are readily distinguishable by their melting temperatures: primer-template, blocker-template, full-length extension product-template, and partial extension product (up to the blocker) - template. Real-time analysis is obtained by periodically exciting the dye and periodically monitoring the fluorescence emission from the dye and from the two fluorophores, both of which are excited indirectly by emission from the dye. Increase in SYBR Green I fluorescence indicates polymerization. Decrease in ROX fluorescence indicates degradation of the blocker. End-point analysis is obtained by stopping the reaction by adding 12 mM EDTA, adjusting the SYBR Green I dilution to 1:14,200, and adjusting the test compound concentration to 800 nM; and then performing a standard melting curve analysis, while stimulating the dye and monitoring fluorescence from each of the two fluorophores.<br>
We evaluated compounds 9-22, 9-22 DD and 9-3 DD with incubations at 55 °C for 60 minutes. We also included a control with no test compound. Results from the control incubated with blocker present but no test compound indicate that Taq DNA polymerase extended the primer through the blocker region, although not completely under these conditions, producing product with nearly a single melting peak according to Cy5 fluorescence. Thus, the enzyme exhibited both polymerization activity and 5'-3' exonuclease activity. Results with compound 9-22 DD present at 300 nM concentration or 1000 nM concentration showed production of partial extension product and an effect on the amount of residual blocker in a dose-dependent manner. Thus, compound 9-22 DD inhibited the 5'-3' exonuclease activity of the polymerase at 55 °C. There was a<br>
decrease in the production of full-length product, particularly at 1000 nM concentration, suggesting that at the high concentration compound 9-22 DD begins to inhibit polymerization.<br>
Another assay that we have utilized to investigate the properties of reagents according to this invention is described below in Example 6. That assay detects the ability of a test reagent to inhibit extension by Taq DNA polymerase at 25 °C. Results reported in Example 7 show that, whereas unmodified hairpin oligonucleotides 9-22 and 9-3 had no effect relative to a control, reagents 9-22 DD and 9-3 DD inhibited primer extension in a dose-dependent manner. Thus, even at a temperature sufficiently low for reagents according to this invention to inhibit polymerization activity at low concentrations, their unmodified analogs did not have that property at the same concentrations.<br>
We have utilized two assays to investigate the inhibitory effect of reagents according to this invention on 5'-to-3' exonuclease activity of DNA polymerase. One assay is described below in Example 7. The assay measures inhibition of 5'- to - 3' exonuclease activity during thermal cycling. FIG. 7 shows that reagent 9-22 DD inhibits that exonuclease activity in a dose-dependent manner at low concentrations ranging from 50nMto300nM.<br>
The second assay is described below in Example 9. It measures quantitatively the inhibition of 5'-to-3' exonuclease activity at 25 °C. FIG. 9 shows dose-dependent inhibition by reagent 9-22 DD at concentrations from 50 nM to lO00nM.<br>
Yet another assay described below in Example 6 measures reagent inhibition of primer extension carried out by the Stoffel Fragment of the Taq DNA Polymerase. As is the case with the intact Taq polymerase, inhibition of extension required the presence of modified ends on the oligonucleotide stems and was not significantly affected by the length of the loop between the stem. However, unlike the case of intact Taq polymerase, the Stoffel Fragment is inhibited by compound 6-22DD, a reagent having a stem of only six nucleotides. We conclude that the Stoffel fragment does not interact with the reagent in the same way as does the intact polymerase.<br>
Reagents according to this invention are useful for prevention of mispriming in symmetric PCR assays, as demonstrated below in Example 8. The assay measured<br>
amplification of a sequence in the cystic fibrosis gene using ah equimolar pair of primers having very similar Tm values.<br>
PCR amplifications and assays currently in use typically include preparation of amplification reaction mixtures at room temperature, thus requiring inclusion of reagents according to this invention in the initial mixture to inhibit Type 1 mispriming. However, that need not be the case. Certain automated methods, for example, microfluidics methods, permit addition of polymerase to a reaction mixture at elevated temperature for a hot start, thereby bypassing Type 1 mispriming. With such methods reagents according to this invention could be added after the start of amplification.<br>
Primer-dimer formation is a manifestation of mispriming that can take place in complete reactions, i.e. those containing all of the components required for amplification plus an initial target sequences, as well as in reactions that do not contain a target sequences. Primer-dimers are typically short double-stranded DNA sequences formed by mispriming of one or more primers in a reaction by hybridization to and extension along either another copy of the same single-stranded primer or some other primer present in the reaction. Primer-dimer formation in the presence of a target sequence is observed as an accumulation of primer-dimers in addition to accumulation of the expected amplicon. Primer-dimer formation in the absence of a target sequence is observed as accumulation of the primer-dimer without accumulation of a specific amplicon. Primer-dimers can also form oligomers that are longer in sequence than the basic primer-dimer because they contain additional, concatenated copies of one or both of the primers. The process of oligomerization is not well understood but is easily detected by gel electrophoresis or melting point analysis of the reaction products. Because of primer-dimer formation the amount of the expected amplicon produced over a given number of thermal cycles is reduced as one or both of the primers is consumed in generation and accumulation of primer-dimers. Elimination of primer-dimers is therefore desirable because it increases both the specificity and yield of the correct product. These features of primer-dimers, as well as their elimination by addition of the reagents described in this invention, are illustrated in Example 10.<br>
The probability of primer-dimer formation is increased by sequence homology between the 3' end of one primer and sequences internal to the same primer or a different<br>
primer, in addition to many other factors, for example, elevation of primer concentration, increasing the magnesium concentration, increasing the length of the primer, lowering the annealing temperature of the thermal cycle, and increasing the total number of primers included in reaction. In addition, it is well known in the art that the probably of primer-dimer formation is much higher in reactions employing non-hot-start polymerases as compared to those employing hot-start polymerases. This is because primer-primer hybridization and extension can occur at relatively low temperatures when the components of the reaction are being mixed. Hot-start enzymes reduce, but do not entirely eliminate generation and accumulation of primer-dimers. These features of primer-dimers, as well as their elimination via addition of the reagents described in this invention, are illustrated in Example 11.<br>
Formation of primer-dimers can also have subtle effects on the kinetics of a reaction, and their elimination by addition of the reagents described herein can be observed and optimized kinetically. Example 12 describes a LATE-PCR amplification comprised of two pairs of primers and generating two single-stranded products. The kinetic accumulation of one of those products was detected by hybridization to a fluorescent probe, and is shown in FIG. 13 and FIG. 14. The data demonstrate that the linearity of the kinetics is affected by both the precise composition of the stem of the reagent, because of its effect on melting temperature, and by the concentration of the reagent. Moreover, optimal and suboptimal use of reagents can alter the rate of linear amplification in a LATE-PCR and thereby the magnitude of the signal at the end of the reaction without affecting the CT value of the reaction. FIG. 15 and FIG. 16 demonstrate that amplification efficiency was the same as judged by CT values over a range of template starting concentrations when compound 9-3DD was used at 150 and 300 nM, but the magnitude of the signals arising from 100 and 1000 template starting numbers in the presence of 300 nM 9-3DD was higher than those obtained using 150 nM 9-3DD, probably because of the absence of primer-dimer formation.<br>
EXAMPLES<br>
Example 1. Rigorous Assay for Assessing Mispriming Errors In order to assess Type 1, Type 2 and Type 3 rnisprimiiig, in PCR amplification reactions, we perform a LATE-PCR amplification utilizing sheared genomic DNA and investigate the products by melting-point analysis and by gel electrophoresis. A particular amplification that we have used is the following:<br>
A.	Substrate: 10-10,000 genome equivalents of sheared genomic DNA. The<br>
DNA was purchased commercially and processed by freezing and thawing genomic DNA<br>
multiple times or by any other similar method known by those skilled in the art for<br>
shearing DNA.<br>
B.	PCR Amplification Mixture Base (see Sanchez et al. (1994) PNAS 101:<br>
1933-1938):<br>
Substrate: 2000 genomes of sheared genomic DNA<br>
IX PCR buffer<br>
Mg+2: at 3 millimolar (mM)<br>
dNTPs: 250 micromolar (jiM) concentration of each of the four dNTP's<br>
Excess primer: sequence 5' CTTTGATGACGCTTCTGTATCTA 3'(SEQ<br>
ID NO: 13) at 1000 nanomolar (nM)<br>
Limiting primer: sequence 5' CCTGGATTATGCCTGGCACCAT 3'<br>
(SEQIDNO: 14)at50nM<br>
DNA polymerase: 1.25 units per 25 microliters (u.1) reaction mixture.<br>
C.	Amplification Protocol:<br>
Low-temperature incubation: 35 min. at room temperature. PCR Amplification protocol: high-temperature soak at 95 °C for 15 min; 10 cycles at 95 °C for 10 sec, 55 °C for 30 sec; and 70 °C for 30 sec, followed by 70 cycles of 95 °C for 10 sec, 50 °C for 30 sec, and 70 °C for 30 sec.<br>
A base-line case for mispriming was established by amplifications utilizing two thermostable DNA polymerases, Promega Taq and Invitrogen Taq without use of any<br>
hot-start methodology. Additional amplifications were performed utilizing two different commercially hot-start DNA polymerases, Qiagen Hot Star Taq and Platinum Taq (Invitrogen). Finally, additional amplifications were performed with the addition of 300 nM of a currently preferred embodiment of a reagent according to the invention. One embodiment used in this example is a reagent that we refer to as composition 9-22 DD (in our nomenclature "9" is the stem length, "22" is the loop length, and "DD" signifies the stabilizing modification, in this case a pair of Dabcyl quenchers). Compound 9-22 DD has a calculated Tm of 81°C. It has the sequence shown in Table II that is modified by the addition of 5' terminal and 3' terminal Dabcyl moieties.<br>
Gel electrophoresis results of the various amplifications, with ethidium bromide staining, are shown in FIG.l, which includes size markers (100 base-pair differences) in the unlabeled marginal columns. Duplicates were run, so there are two lanes for the product of each LATE-PCR amplification. Lanes a, a base-line case in which the DNA was omitted; lanes b, a base-line case in which the DNA polymerase was Promega Taq; lanes c, a base-line case in which the DNA polymerase was Invitrogen Taq; lanes d, substitution of the hot-start enzyme Qiagen Hot Star Taq; lanes e, substitution of the hot-start enzyme Platinum Taq; lanes f, amplification utilizing Promega Taq polymerase with the addition of compound 9-22 DD; lanes g, amplification utilizing Invitrogen Taq polymerase and compound 9-22 DD; lanes h, amplification utilizing Qiagen Hot Star Taq DNA polymerase and compound 9-22 DD; lanes i, amplification utilizing Platinum Taq polymerase and compound 9-22DD.<br>
From FIG. 1 it can be seen that regular Taq DNA polymerases that have both polymerization activity and 5'-3' exonuclease activity (lanes b and lanes c) yielded a range of product sizes, almost none of which was the desired amplicon defined by the primer pair, which is the product seen in lanes f-i. Switching to modified "hot-start" polymerases (lanes d-e) helped but still did not eliminate non-specific products formed due to mispriming errors. When compound 9-22 DD was present at 300 nM concentration, however, the desired amplicon was obtained without non-specific products formed due to mispriming errors both when the enzyme was a regular Taq DNA polymerase (lanes f-g) and when the enzyme was a hot-start Taq DNA polymerase (lanes h-i).<br>
Using the rigorous assay of this example, we have compared a number of unmodified and modified DNA hairpin molecules without terminal stabilization; with terminal destabilization as one obtains by adding identical nucleotides to the end of each arm; with terminal stabilization according to this invention; and with non-stabilizing terminal additions. To make the comparison quantitative, we performed dosage response tests using the assay for each compound that exhibited mispriming suppression activity in order to determine the minimum concentration needed to yield elimination of nonspecific products formed due to mispriming errors, and the presence of only the desired specific product, obtained as shown in FIG. 1 with regular Taq DNA polymerase. The results are shown in Table I, which includes the calculates stem Tm for each reagent tested. Sequences of the various compounds reported in Table I are set forth in Table II.<br><br>
(Table Removed)<br>
Table I compares several DNA hairpins, with and without modifications and identifies several terminally stabilized hairpin molecules that are reagents according to this invention (clean amplicon without products of mispriming at less than lO00nM concentration) or that are preferred reagents according to this invention (clean amplicon without products of mispriming at no more than 650nM concentration). The unmodified hairpins are identified by two numbers, the first being the stem length and the second being the loop length. Compounds according to this invention appearing in Table I have several stabilizing modifications: a 5' Dabcyl quencher and a 3' Dabcyl quencher covalently attached to the terminal nucleotides by means of a commercial linker (signified by the suffix "DD"), a 3' Black Hole™ quencher and a 5' Black Hole ™ quencher similarly attached (signified by the suffix "BHQBHQ"), and two 3' 2'-O-methyl ribonucleotides and two 5' 2'-O-methyl ribonucleotides substituted for deoxyribonucleotides at the stem terminus (signified by "2'OM4"). Destabilizing modifications to the stem termini included substituting a pair of A's or T's at the 3' and 5' ends of the molecule. Adding a pair of FAM fluorophores, which are not believed to interact with one another in a stem-stabilizing fashion, was destabilizing, as was adding a single 5' Dabcyl. However, adding a single 3' Dabcyl slightly stabilized the 9-22<br>
hairpin, as shown by a drop in the concentration required. Hairpin 9-22, suppressed<br>
misprinting only at high (3000 nM) concentration, but 9-22 DD did so at low concentration (50 nM).<br>
Another of our presently preferred hairpins, compound 9-3, suppressed mispriming only at high (1000 nM) concentration, but 9-3 DD and 9-3 2'OM4 did so at much lower concentration (100-300 nM). Yet another embodiment, 9-5 DD, 9-5 BHQBHQ and 9-5 2'OM4 did so also at low concentrations (50-100 nM). Oligonucleotide 9-5 has the same stem as oligonucleotide 9-22 but a shorter loop whose nucleotides are A's and T's. The calculated stem Tm of oligonucleotide 9-5 is 93 °C, some 12 °C higher than the calculated stem Tm of oligonucleotide 9-22 due to the influence of the loops.<br>
Similar results were obtained with an even longer stem. Compound 12-3 did not suppress mispriming, but compound 12-3DD was effective at only 50nM concentration.<br>
Table I shows that stabilized stems can be modified to change the calculated stem melting temperature (Tm). For example, compound 9-3DD, Tm 56°C, was effective at l00nM concentration. By altering the G-C content of the stem we created compounds 9-3bDD (Tm 62°C) and 9-3iDD (Tm 68°C). Both were also effective at l00nM concentration. Compound 12-3DD, Tm 58°C, was effective at 50nM concentration. Compounds with altered stems, namely, compound 12-3bDD, Tm 63°C, and compound 12-3cDD, Tm 68°C, were effective at a l00nM concentration, still very good.<br>
Example 2. LATE-PCR End-Point Assay<br>
Stochastic variation in reaction kinetics among replicate samples results in scatter of signals from hybridization probes and hampers end-point analysis. LATE-PCR significantly reduces that problem. Utilization of a reagent according to this invention further improves the ability to utilize end-point detection. FIG. 2 presents results obtained in amplifying an amplicon that includes the G269 allele of the hexosaminidase A gene responsible for Tay Sachs disease. The reaction mixture contained 0.6 uM hybridization probe and either 1000 genomes homozygous wild type DNA target (+/+) or heterozygous DNA target (G269/+). Replicate samples of each type were amplified as follows: Ix PCR buffer, 3 mM MgCl2, 250 uM of each dNTP, 25 nM Limiting Primer having a sequence 5' CGAGGTCATTGAATACGCACGGCTCC 3' (SEQ ID NO: 15), 1000 nM Excess Primer having a sequence 5' TAACAAGCAGAGTCCCTCTGGT 3'<br>
(SEQ ID NO: 16), 1.25 units Platinum (hot-start) Taq polymerase, 0.6 ^.M Cy5-labeled hybridization probe having a sequence 5' Cy5- GGGACCAGGTAAGAA - phosphate 3' (SEQ ID NO: 17), and a 1:40,000 dilution SYBR Gold I in the presence or absence of 100 nM of embodiment 9-22 DD in 25ul reactions. PCR cycle parameters were 95 °C for 3 minutes; 25 cycles at 95 °C for 10 sec, 65 °C for 20 sec, and 72 °C for 20 sec; and 30 cycles at 95 °C for 10 sec, 65 °C for 20 sec, 72 °C for 20 sec, 55 °C for 20 second, and 45 °C for 20 sec with fluorescence acquisition at 72°C for SYBR Gold, and fluorescence acquisition at 55 °C and 45 °C for Cy5. To correct for tube-to-tube variations in reaction kinetics, hybridization signals were normalized by the ratio of Cy5 signals at 55 °C to SYBR Gold signals at 72 °C. One consequence of such normalization is that any CT delay is masked.<br>
FIG.2, lines identified by circle 21 and lines identified by circle 22, show the results of multiple replicates without addition of a reagent according to this invention but with hot-start polymerase. Homozygous replicates (circle 21) and heterozygous replicates (circle 22) can be distinguished by the slopes of their linear plots. Scatter among replicates and mispriming errors in these samples, evidenced by the drop in hybridization probe signal (the "hook effect") after about 40 cycles precludes end-point identification of these samples. FIG. 2, lines identified by circles 23-24, show the results with addition of reagent according to this invention, namely, 100 nM 9-22 DD. Under these conditions mispriming errors do not occur, linear kinetics are preserved in the course of the assay, scatter is reduced, the hook effect is avoided, and end-point identification of homozygous and heterozygous samples is possible at 50 cycles.<br>
Example 3. Preparing DNA Samples for Dideoxy Sequencing LATE-PCR is a non-symmetric amplification method that potentially generates large amounts of single-stranded DNA suitable for DNA sequencing when the amplification reaction is carried out over 60 and more thermal cycles. However, such large numbers of amplification cycles encourage the appearance of products of mispriming, believed to include the formation of Type 2 and Type 3 mispriming errors, that manifest themselves by accumulation of detectable non-specific products that eventually become the predominant species in the reaction. Reagents according to this<br>
invention allow the synthesis of large amounts of single-stranded DNA suitable for DNA sequencing in the absence of non-specific products when using 70 and more cycles of LATE-PCR amplification. Samples suitable for dideoxy sequencing by capillary gel electrophoresis were prepared as follows.   LATE-PCR reaction mixture was first prepared in 25 ul containing IX PCR buffer, 3 mM MgCh, 1000 nM Excess Primer having the sequence 5' GCCAGGGGTTCCACTACGTAGA 3' (SEQ ID NO: 18, 25 nM Limiting Primer having the sequence 5' CCGCCCTTCTCTCTGCCCCCTGGT 3' (SEQ ID NO: 19), 1.25 units Platinum Taq polymerase, 600 nM reagent 9-3 DD and 250 uM of each dNTP. Compound 9-3 DD has a Dabcyl quencher added to each terminal nucleotide of oligonucleotide 9-3. Amplification was carried out at 95°C for 3 minutes; 10 cycles at 95 °C for 10 sec, 65 °C for 20 sec, and 72 °C for 20 sec; and 60 cycles at 95 °C for 10 sec, 65 °C for 20 sec, 72 °C for 20 sec; and 45 °C for 20 sec. Parallel reactions were performed with either 1:40,000 dilution SYBR Green or 2.4 uM Low Tm molecular beacon having the sequence 5' FAM-CGTGCGCTCTGGTAAGGGTTTGCACG-Dabcyl 3' (SEQ ID NO: 20) to monitor double-stranded and single-stranded DNA synthesis separately. Each sample contained 6 ng (about 1000 genomes) of human DNA. Further, a control was run without addition of compound 9-3 DD.<br>
Results of the amplification reactions were that the control with hot-start Taq DNA polymerase but not compound 9-3 DD manifested significant mispriming, as evidenced by a secondary rise in SYBR Green signal during the late stages of amplification, after the Limiting Primer had been depleted. Because the product was such a mixture, it was not possible to sequence the control. On the other hand, results of the amplification with compound 9-3 DD added showed the presence of only the desired specific product without a delayed rise in the SYBR Green signal through 70 cycles. We calculated, from the Limiting Primer concentration and an assumed efficiency of linear amplification of 50%, that the reaction generated 250 fmoles/ul of clean product. That product was submitted to the Brandeis University dideoxy sequencing facility for sequencing.<br>
The sequencing reaction was performed using a standard protocol, as was capillary electrophoresis utilizing a Beckman CEQ 2000 DNA Sequencer. The sequencing reaction was performed with a 1/5 aliquot of the amplified product, that is,<br>
with only 50 fmoles of product. The machine-produced sequence is presented in FIG. 3, which shows the sequence along the upper portion of the chart, the clear, unambiguous product peaks from which the sequence was derived, and, at the bottom a very low level of background signal. We confirmed the correctness of the machine-produced sequence by comparing it to the known sequence of the amplicon available from GenBank Accession No. NT 010235.<br>
Example 4. Multiplexed PCR Amplification<br>
Suppression of mispriming errors is particularly critical in multiplexed reactions where multiple products are amplified simultaneously. The presence of several primer pairs and amplification products increases the probability of mispriming interactions between and among these reactive species. Multiplexed reactions composed of increasing numbers of amplicons were carried out in the absence or presence of compound 9-3 DD.   LATE-PCR reactions, 25|il volume, consisting of sample, IX PCR buffer, 3 mM MgCl2, 100 uM of each dNTP, 1000 nM Excess Primer, 50 nM Limiting Primer, and 1.25 units of non-hot-start Promega Taq polymerase were set up in the absence or presence of 300 nM reagent 9-3 DD for different combinations of primer pairs as indicated below. Samples were amplified using a thermal cycling profile of 95 °C for 3 minutes; 10 cycles at 95 °C for 10 sec, 65 °C for 30 sec, and 70 °C for 30 sec; and 40 cycles at 95 °C for 10 sec, 50 °C for 30 sec, 70 °C for 30 sec. At the end of the reaction the samples were analyzed by gel electrophoresis in a 3.5% agarose gel in 0.5 xTBE. The intended amplicons and primers pairs tested were as following<br>
Reaction A: two regions TSD 1278 + TSD 1421 of the Hex-A gene associated<br>
with Tay Sachs disease.<br>
TSD 1278 Excess Primer: 5' GCCAGGGGTTCCACTACGTAGA 3' (SEQ ID NO: 21)<br>
TSD 1278 Limiting Primer: 5' CCGCCCTTCTCTCTGCCCCCTGGT 3'(SEQ ID NO: 22)<br>
TSD 1421 Excess Primer: 5' CCGGGTCTCTAAGGGAGAACTCCT 3'(SEQ ID NO: 23)<br>
TSD 1421 Limiting Primer: 5' CCGGCCGACAACACAAACCTGGTCC 3'(SEQIDNO:24)<br>
Reaction B: TSD 1278 amplicon, TSD 1421 amplicon, and a region of the CFTR<br>
gene, the CF exon 10 amplicon.<br>
This reaction contained the same as primer pairs at Reaction A plus<br>
CF ex 10 Excess Primer: 5' GCTTTGATGACGCTTCTGTATCTA 3' (SEQ ID NO: 25)<br>
CF ex 10 Limiting Primer: 5' CAGTTTTCCTGGATTATGCCTGGCACCAT 3' (SEQ ID NO: 26)<br>
Reaction C: TSD 1278 amplicon, TSD 1421 amplicon, CF exon 10 amplicon, and<br>
another region of the CFTR gene, the CF exon 11 amplicon.<br>
This reaction contained the same as primer pairs at Reaction B plus<br>
CF ex 11 Excess Primer: 5' TCGAAGTTTGCAGAGAAAGACAAT 3' (SEQ ID NO: 27)<br>
CF ex 11 Limiting Primer: 5' TGACGTTTACAGCGAATGCTTGCTAGACCAAT 3' (SEQ ID NO: 28)<br>
Reaction D: TSD 1278 amplicon, TSD 1421 amplicon, CF exon 10 amplicon, CF<br>
exl 1 amplicon, and a region of the human beta globin gene.<br>
This reaction contained the same as primer pairs at Reaction C plus<br>
Beta Globin Excess Primer: 5' TGGGTTTCTGATACGCACTGACTCTCTC 3' (SEQ ID NO: 29)<br>
Beta Globin Limiting Primer: 5' GGCCATCACTAAAGGCACCGAGCACT 3' (SEQ ID NO: 30)<br>
Electrophoresis gels were stained with ethidium bromide to detect double-stranded products produced during the exponential phase of the LATE-PCR reaction. FIG. 4 shows pertinent portions of gels of products from multiplexed reactions A through D. Arrows point to specific double-stranded amplification products, unnumbered bands<br>
correspond to specific single-stranded DNA products with secondary structure, and asterisks identify non-specific products. Gel pairs A-D correspond to reactions A-D, respectively. Each gel labeled "+" is product from amplification including compound 9-3 DD. Each gel labeled "-" is product from amplification without compound 9-3 DD.<br>
Gel pair A shows that, in the absence of 9-3 DD compound region TSD 1278 (arrow 41) did not amplify but region TSD 1421 (arrow 42) did amplify, along with a large number of non-specific products, predominantly or entirely double-stranded, that appeared in the gel as a smear. Addition of the 9-3 DD compound reduced the background of non-specific products and allowed amplification of both TSD 1278 (arrow 41) and TSD 1421 (arrow 42) amplified more cleanly, showing that several manifestations of mispriming were prevented.<br>
Gel pair B shows that in the absence of 9-3 DD compound regions TSD 1278, TSD 1421, and CF exon 10 (arrow 43) amplified along with a large number of nonspecific products that appeared in the gel as a smear. Addition of the 9-3 DD compound eliminated the background non-specific products, and all three expected amplicons were obtained.<br>
Gel pair C shows that in the absence of 9-3 DD compound regions TSD 1278, TSD 1421, CF exon 10, and CF exon 1 1 (arrow 44) amplified along with a non-specific product indicated by the asterisk. Addition of the 9-3 DD compound eliminated synthesis of the non-specific product, and all four expected amplicons were obtained.<br>
Gel pair D shows that, in the absence of 9-3 DD compound the selected region of the beta globin gene (arrow 45) did not amplify but TSD 1278, TSD 1421, CF exon 10, and CF exon 1 1 did amplify, along with a non-specific product indicated by the asterisk. Addition of the 9-3 DD compound eliminated synthesis of the non-specific product and allowed amplification of all five expected amplicons.<br>
We have also demonstrated the use of a compound according to this invention in multiplex amplification and detection reactions when only one target is present in any given test sample, such as may occur when screening for an infecting pathogen. In such a case multiple pairs of primers (we utilized multiple pairs of Excess Primers and Limiting Primers for LATE-PCR amplifications) are present, but only one primer pair will be active for a particular sample. As indicated by SYBR green fluorescence and<br>
post-amplification electrophoretic analysis, that is in fact what occurred, demonstrating that compounds of this invention effectively inhibit mispriming in multiplex reactions when fewer than all substrates are present.<br>
Example 5. Relation between Stem Tn^and Annealing Temperature<br>
Two different LATE-PCR amplification protocols were compared utilizing the same amplification mixture, including target. Each amplification was performed both with 600 nM compound 9-3 DD and without compound 9-3 DD. As indicated earlier the stem Tm of compound 9-3 DD (that is, the calculated melting temperature of unmodified oligonucleotide 9-3) is 56 °C. Hot-start Platinum Taq DNA polymerase was used. The two amplifications differed primarily in the primer annealing temperature that was employed, either 65 °C (above the stem Tm) or 55 °C (below the stem Tm). The amplification cycle parameters were:<br>
Profile A: 95 °C for 3 minutes; ten cycles of 95 °C for 10 sec, 65 °C for 20 sec, 72 °C for 20 sec; sixty cycles of 95 °C for 15 sec, 55 °C for 20 sec, 72 °C for 20 sec, 50 °C for 20 sec.<br>
Profile B: same as profile A, except that the last sixty cycles were 95 °C for 10 sec, 65 °C for 20 sec, 72 °C for 20 sec, 45 °C for 20 sec.<br>
The amplification reactions were monitored in real time using SYBR Green I fluorescent dye to monitor the synthesis of double-stranded product, which plateaus in a LATE-PCR reaction following exhaustion of Limiting Primer, unless mispriming causes a variation in the later portion of an amplification. Readings were taken during the low-temperature detection step.<br>
Results are presented in FIG. 5. Panel A shows the fluorescence readings from replicate samples of Profile A, when the 55 °C annealing temperature was utilized after ten cycles of extension, both with (circle 52) and without (circle 51) inclusion of compound 9-3 DD. Panel B shows the fluorescence readings from replicate samples of Profile B, when the 65 °C annealing temperature was used after ten cycles of extension, both with (circle 54) and without (circle 53) inclusion of compound 9-3 DD.<br>
FIG. 5 shows that when a primer annealing temperature below the stem Tm of the reagent of this invention was used (Panel A), a delay of several cycles in the threshold<br>
cycle, CT, was observed. However, when the primer annealing temperature was maintained significantly above the stem Tm throughout the exponential PCR phase (Panel B), little, if any, CT delay was observed. It was noted that for the particular amplicon utilized in this amplification, no Type 2/Type 3 mispriming was observed with or without compound 9-3 DD when hot-start Taq was used, showing the unpredictability of the occurrence of such mispriming errors. This example demonstrates that the stem of reagent 9-3 DD is closed when the temperature of the reaction drops to 55 °C and that the reagent remains bound to the polymerase during the higher-temperature extension step of the reaction.<br>
Example 6. Effect on DNA polvmerase activity at 25 °C<br>
A series of primer extension experiments was run at 25 °C to evaluate the effect of reagents according to this invention and their unmodified oligonucleotide analogs. Each reaction mixture included a template, a primer and Taq DNA polymerase. The reaction was carried out for two hours in each case. The primer was fluorescently labeled with Cy5. SYBR Green fluorescent dye was added to the reaction mixture. Following the extension reactions, melt curves were run, in which the dye was stimulated and fluorophore emission due to FRET transfer from the dye was read, as disclosed in our concurrently filed United States Provisional patent application titled Primers, Probes and Methods for Nucleic Acid amplification. Results are presented in FIG. 6.<br>
FIG. 6, panel A, includes melting curve 61, obtained when extension was precluded by omitting dNTPs from the reaction mixture. Curve 61 is thus the melting curve of the primer-target hybrid, showing a Tmof 58 °C. Panel A also includes several controls which contained dNTPs, either alone or with unmodified oligonucleotide 9-3 (300 nM, 1000 nM) or 9-22 (50 nM, 100 nM, 300 nM). Curves identified by circle 62 for the controls show extension in all cases, that is, the primer melting peak at 58 °C disappeared. FIG. 6, panel B shows a dose-dependent effect of including reagent 9-22 DD in the reaction mixture: curve 63, 50 nM; curve 64, 100 nM; curve 65, 300 nM. The larger the peak at 58 °C, the greater the effect of preventing extension by polymerization under the isothermal test conditions. FIG. 6, panel C shows a dose-dependent effect of including reagent 9-3 DD in the reaction mixture: curve 66, 300 nM; curve 67, 1000 nM.<br>
Again, the larger the peak at 58 °C, the greater the effect of preventing extension by polymerization under the test conditions.<br>
We also investigated the effect on the Stoffel Fragment (Lawyer et al. (1993) PCR Methods and Applications 2: 275-287), a DNA polymerase lacking 5'-3' exonuclease activity. We compared various compounds listed on Table 1 with a negative control containing only template, fluorescently labeled primer, the Stoffel Fragment and dNTPs. In this series, extension was performed at 40°C and SYBR Green readings were taken every 20 seconds for more than 30 minutes. Curves were compared by eye, that was reliable, because either the curve obtained tracked the control almost precisely or else the curve obtained differed markedly from the control. Test compounds were added at 50nM, l00nM and 300nM concentration. Unmodified hairpins 6-22, 9-3, 9-5 and 9-22 did not suppress elongation by the Stoffel Fragment at any concentration. Compound 6-22DD suppressed elongation at all three concentrations, as did compounds 9-3DD, 9-5DD and 9-22DD. The following compounds did not suppress elongation at 50M or l00nM concentration but did suppress elongation at 300nM concentration: 9-22-5D, 9-22-3D, 9-5 2'OM4, 9-5BHBHQ, and 9-32'OM4.<br>
Example 7. Effect on Tag Exonuclease Activity in the Absence of Polymerase Activity<br>
We have discovered an assay that demonstrates inhibition of 5'- to - 3' exonuclease activity in the absence of DNA synthesis. The assay uses as a substrate a specific double-stranded DNA molecule where one strand is labeled with a FAM fluorophore at the 5' end and the other strand is labeled at the 3' end with a Dabcyl quencher. The fluorophore is in close proximity with the quencher, and fluorescence does not occur when the fluorophore is stimulated. Strand denaturation and annealing upon heating and cooling causes cleavage of the FAM-labeled strand and fluorophore release. We believe that cycling somehow generates a substrate for the 5'- to - 3' exonuclease activity of Taq polymerase. Fluorescence increase therefore provides a measure of Taq polymerase 5'- to - 3' exonuclease activity.<br>
The 25 ul reaction mixture contains 300 nM double-stranded DNA template in IX PCR buffer, 3 mM MgC^, 1.25 units, (U), Taq polymerase, and the presence or absence<br><br>
of an appropriate concentration of 9-22 DD compound. The reactions do not contain any dNTP nor any other nucleic acid target besides the double-stranded DNA. The sequences of the complementary strands of the double-stranded DNA template are 5' FAM-AGTGTGATGATGGTGAGG -phosphate 3' (SEQ ID NO: 31) and 5' -ACTTTCAACTCTGTCT 3' -Dabcyl (SEQ ID NO: 32). Samples are denatured at 95°C for 3 minutes and then subjected to cycles of 95°C for 10 sec, 67°C for 30 sec, 72°C for 30 sec, and 45°C for 20 sec. Fluorescence is acquired during each cycle at 45°C.<br>
Fluorescence readings from performance of the assay on several samples are reported in FIG. 7, in which readings are normalized to begin at the same background fluorescence. Curve 76 is a control showing the fluorescence with no Taq polymerase present. Taq polymerase was included in the remaining samples. Curve 71 shows the fluorescence increase resulting when reagent 9-22 DD was not added. Fluorescence rose steadily over thirty cycles. Curves 72-75 show the fluorescence increase with reagent 9-22 DD included at concentrations of 50 nM, 100 nM, 200 nM and 300 nM, respectively. Addition of the 9-22 DD compound to the above reaction reduced the observed fluorescence increase in a dose-dependent manner.<br>
Example 8. Symmetric PCR Amplification<br>
We performed a real-time PCR amplification assay utilizing sheared genomic DNA and a pair of primers of similar Tm in equimolar concentration. We performed replicate assays without and with addition of 400 nM reagent 9-3 DD. The effects of the reagent were evaluated in terms of the kinetics of total DNA accumulation via staining with SYBR Green, as well as by gel electrophoresis with ethidium bromide staining.<br>
The intended target for amplification was an allele of the CFTR gene that occurs in intron 19 as a single base transition (GenBank Accession No. AC000061). The 25 ul reaction mixture contained a substrate of 120 picograms, (pg), of sheared human genomic DNA, Ix PCR buffer, 5 mM MgCl2,250 uM of each dNTP, a 1:40,000 dilution of SYBR Green, 1000 nM forward primer: sequence 5' TAATTACAAGAGTCTTCCAT 3' ( SEQ ID NO: 33), Tm 56.6 °C, and 1000 nM reverse primer: sequence 5' CATGAATAGAACATTTCCTT 3' (SEQ ID NO: 34), Tm 56.3 °C, and 1.25 units of non-hot-start Invitrogen Taq polymerase in the absence or presence of 400 nM of reagent 9-3<br>
DD. Samples were amplified using a thermal cycling profile of 95 °C for 3 minutes; 60 cycles at 95 °C for 10 sec, 55 °C for 30 sec, and 72 °C for 30 sec with SYBR Green I fluorescence monitored at 72 °C, and finally a temperature gradient of 54 °C to 96 °C at 1 °C increments for 30 sec. At the end of the reaction several randomly selected samples from each set of reactions were analyzed by gel electrophoresis in a 3.0% agarose gel in 0.5 x Tris, Boric Acid, EDTA Solution (TBE).<br>
FIG. 8 presents the results obtained. The amplification plot in panel A of FIG. 8 shows the SYBR Green I fluorescence for 60 cycles with curves identified by circle 81 being the replicate samples that did not contain the reagent 9-3 DD and curves identified by circle 82 being the replicate samples with 400nM of 9-3 DD. The kinetics of fluorescent signals arising from the two sets of samples demonstrates that total double-stranded DNA accumulated somewhat more rapidly in the absence of reagent 9-3 DD than in the presence of the reagent, but the total amount of double-stranded DNA accumulated by 60 cycles is virtually identical in the two sets of reactions.<br>
Panel B of FIG. 8 shows the electrophoretic gels with ethidium bromide staining. On the left, lane a is the size marker (50 base-pair differences), and lanes b through i are samples without reagent 9-3 DD. On the right, lanes j thru q are samples with 400nM 9-3 DD, and lane r is the size marker. The results show that reactions that did not contain the reagent according to this invention generated the correct product as well as many nonspecific products of higher molecular weight, while the reactions that contained reagent 9-3 DD generated only the correct product. Moreover, reactions that contained reagent 9-3 DD generated about twice as much of the correct product as reactions that did not, as judged by the relative intensities of the correct product band in the gels.<br>
In combination the kinetic analysis and the electrophoretic analysis reveal that reactions 81 that generate non-specific products produce fluorescent signals sooner than reactions 82 that generate only the correct product. SYBR Green dye intercalates into double stranded DNA without regard to sequence specificity. Thus the sigmoidal kinetics of the reactions 82, as compared to the more rectilinear kinetics in reactions 81 can be used to judge whether a symmetric reaction is or is not accumulating only the correct product.<br>
Example 9. Quantification of Exonuclease Inhibition<br>
This example describes a rigorous assay for measuring the inhibitory effect of a reagent according to this invention on the 5'-3' exonuclease activity of Taq DNA polymerase. This assay uses a similar primer-template system described by M. W. Kaiser, N. Lyamicheva, W. Ma, C. Miller, B. Neri, L. Fors, and V. I. Lyamicheva in J. Bio. Chem., 274. pp. 21387-21394 (1999), except that the template is labeled with Cy5 at the 5' end and the assay is carried out in the presence of SYBR Green I, a DNA dye that fluoresces when bound to double-stranded DNA. The template for this assay consists of the oligonucleotide sequence<br>
5'-Cy5-<br>
AAAACGCTGTCTCGCTGAAAGCGAGACAGCGAAAGACGCTCGT-3' (SEQ ID NO: 35) annealed to a primer with the nucleotide sequence:<br>
5'-ACGAGCGTCTTTC-3' ( SEQ ID NO: 36). The longer oligonucleotide forms a hairpin structure with two single-stranded tails of different lengths. The shorter 5' tail of the template contains the Cy5 fluorophore and consists of 4 adenosine residues; the longer 3' tail of the template serves as the target for the primer oligonucleotide such that the primer sits directly in front of the 5' template tail with one base pair overlap. Addition of SYBR Green I to this template-primer complex results in SYBR Green I binding to the double-stranded DNA regions of the complex. Upon SYBR Green I excitation with a 480 nm laser, fluorescence energy from the bound DNA dye is completely absorbed by Cy5, which is believed to occur via fluorescence resonance energy transfer, and there is no detectable fluorescence at the maximum SYBR Green emission wavelength (maximum emission wavelength: 521 nm). Hydrolysis of the 5' tail by the 5'-3' exonuclease activity of Taq polymerase results in removal of the Cy5 moiety from the template-primer complex and restoration of detectable SYBR Green I fluorescence. The kinetics of SYBR Green I fluorescence restoration in the presence of Taq DNA polymerase provides a quantitative measure of 5'-3' exonuclease activity. Reagents according to this invention inhibit the 5'-3' exonuclease activity of Taq polymerase and slow down SYBR Green I fluorescence increase.<br>
The assay consisted of 0.5 u-M of the above template and 1.5 uM of the above primer mixed in IX PCR buffer, 3mM MgCl2, a 1:40000 dilution of a commercial SYBR<br>
Green I stock (Molecular Probes, Eugene, OR), 1.25 U Taq polymerase (Invitrogen, Calrsbad, CA) in the absence or the presence of 9-22 DD compound at 50nM, l00nM, 300nM, or lO00nM in a volume of 25 microliters (ul). There was no addition of dNTPs in the reaction mix, because the assay does not rely on DNA polymerase activity. The reaction mixture minus the template and primer was set up at 25 °C to promote the interaction between the 9-22 DD compound and Taq DNA polymerase. The sample was then placed on ice, supplemented with template and primers, and kept on ice until the start of the reaction. Negative controls lacked Taq DNA polymerase. The reaction was initiated by incubating the sample at 25 °C in an ABI Prism Sequence Detector 7700 for 60 minutes with fluorescence collection in the SYBR Green I channel every 30 seconds. Three different trials were averaged for each control and for each concentration of reagent 9-22 DD.<br>
FIG. 9 presents the results of the assay. Detectable SYBR Green I fluorescence provides a measure of the extent of 5'-3' exonuclease activity. The "No Taq" control shows no SYBR fluorescence signal consistent with complete absorption of SYBR Green I fluorescence by Cy 5 and complete absence of 5'-3' exonuclease activity (line 96). Addition of Taq polymerase restores SYBR Green I fluorescence and provides a baseline for the maximum levels of 5'-3' exonuclease activity under the assay conditions (line 91). Addition of 9-22 DD compound to the reaction mix slows down restoration of SYBR-Green I fluorescence in a dosage-dependent manner due the inhibitory effect of 9-22 DD on the 5'-3' exonuclease activity of Taq polymerase (line 92, 50 nM; line 93, 100 nM; line 94, 300 nM; line 95, 1000 nM). Table III below quantifies the percentage inhibition of 5'-3' exonuclease activity by various concentrations of the 9-22 DD compound based on the relative SYBR Green I fluorescence levels achieved after 60 minute incubation at 25 °C.<br>
TABLE III<br><br>
Fluorescence<br>
Reaction<br>
% 5'-3* Exonuclease Inhibition<br>
(Table Removed) <br><br>
A series of LATE-PCR amplification reactions were set up in the presence or absence of 100 genomes human placental DNA (Sigma, St Louis, MO) in a final volume of 25 ul. In this experiment, LATE-PCR amplification reactions consisted of IX PCR buffer (Invitrogen, Carlsbad, CA), 3 mM MgCl2, 0.25 mM dNTP, 1000 nM Excess Primer, 50 nM Limiting Primer, 0.25 uM FAM-labeled probe strand, 0.3 uM Dabcyl-labeled reverse complement strand, 1.25 units Platinum Taq DNA polymerase (Invitrogen, Carlsbad, CA). The sequence of the primers and probes were as follows:<br>
Excess Primer: 5' GTTTCTTTGCTGCCGTGTTC 3' (SEQ ID NO: 37)<br>
Limiting Primer: 5' CCCCAGAGACCCCAGTTGCTAACCAGAC 3'(SEQ ID NO: 38)<br>
FAM-labeled probe strand: 5' [TET] AGACAGAGTTGAAAGTCAGG [Phos] 3' (SEQ ID NO: 39)<br>
Dabcyl-labeled reverse complement strand: 5' ACTTTCAACTCTGTCT [Dabcyl] 3' (SEQ ID NO: 40)<br>
These primers and probes amplify and detect a 488 base-pair (bp) amplicon that encompasses exons 5 and 6 of the human p53 gene.<br>
Amplification was carried out in an ABI Prism 7700 Sequence Detector (Applied Biosystems,   CA) with a thermal profiles that consisted of 1 cycle at 95 °C for 3 minutes; 25 cycles at 95 °C for 10 sec, 64 °C for 30 sec, 75 °C for 30 sec; and 35 cycles at 95 °C<br>
for 10 sec, 64 °C for 30 sec, 75 °C for 30 sec, 45 °C for 20 sec with fluorescence detection in the TET channel during the 45 °C step.<br>
The resulting amplification products were analyzed by gel electrophoresis in a 3% agarose gel in 0.5X TBE buffer for 2 hours and stained with ethidium bromide. Results are shown in FIG. 10. The samples in lanes 3, 5, 7, 9, 11 had been prepared without genomic DNA. The samples in lanes 2, 4, 6, 8, 10 had been prepared with genomic DNA. Lane 1 contained elecrrophoretic size markers in a 100 base-pair ladder. Reagent<br>
9-22 DD was added to the starting reactions as follows: lanes 2 and 3, 0 nM; lanes 4 and<br>
i<br>
5, 50 nM; lanes 6 and 7, 100 nM; lanes 8 and 9, 200 nM; lanes 10 and 11, 300 nM.<br>
Adjacent to the right side of the gel we have marked the size of correct product and also sizes that we infer to be primer-dimers and oligomers of primer. The results presented in FIG. 10 demonstrate that in reactions initiated with genomic DNA increasing the concentration of reagent 9-22 DD acts in a dosage-dependent manner to prevent manifestation of non-specific products, including primer-dimers and primer oligomers and thereby increases both the specificity and yield of the correct products. Increasing concentrations of reagent 9-22 DD also prevent manifestation of primer-dimers and primer oligomers in reactions that do not contain genomic DNA.<br>
Example 11. Preventing Primer-Dimer Formation and in a Duplex Reaction A series of LATE-PCR amplification reactions was set up in the presence or absence of 100 genomes human placental DNA (Sigma, St Louis, MO) in a final volume of 25 pi. In this experiment, LATE-PCR amplification reactions consisted of IX PCR buffer (Invitrogen, Carlsbad, CA), 3 mM MgCl2, 0.20 mM dNTP. Taq polymerase was used at 1.25 units in every sample. A first amplification target sequence (Product 1) was a portion of exon 11 of the Cystic Fibrosis Gene and was amplified with a Limiting Primer: 5' GACGTTTACAGCGAATGCTTGCTAGACCAAT 3' (SEQ ID NO: 41) at 100 nM and an Excess Primer: 5' TCCAAGTTTGCAGAGAAAGACAAT 3' (SEQ ID NO: 42) at 2,000 nM. A second amplification target sequence (Product 2) was a portion of exon 10 of the Cystic Fibrosis Gene and was amplified with a Limiting Primer: 5' CAGTTTTCCTGGATTATGCCTGGCACCAT 3' (SEQ ID NO: 43) at 50 nM and an Excess Primer: 5' GCTTTGATGACGCTTCTGTATCTA 3' (SEQ ID NO: 44) at 1000<br>
nM.<br>
Amplification was carried out in an ABI Prism 7700 Sequence Detector (Applied Biosystems,   CA) 2 min at 95 °C, followed by 25 cycles of 95 °C for 10 sec, 56 °C for 15 sec, 70 °C for 20 sec, followed by 50 cycles of 95 °C for 10 sec, 56 °C for 15 sec, 70 °C for 20 sec, and 45 °C for 30 sec with fluorescence acquisition.<br>
The resulting amplification products were analyzed by gel electrophoresis in a 3% agarose gel in 0.5X TBE buffer for 2 hours and stained with ethidium bromide. Results are shown in FIG. 11. Reactions analyzed in lanes 1-6 utilized a non-hot-start Taq polymerase, while reactions analyzed in lanes 7-12 utilized a Taq polymerase plus a hot-start antibody. Reagent 9-22 DD was added to analyzed in lanes reactions 4-6 and 10-12 at l00nM.<br>
Adjacent to the right side of the gel we have added our interpretation of the identities of the amplification products, including single-stranded amplicons (ssDNA) and double-stranded amplicons (ds DNA) of Product 1 and Product 2, as well as shorter products we infer to be primer-dimers. Other non-specific products are marked with an asterisk. Amplifications with non-hot-start polymerase and no reagent according to this invention, lanes 1-3, produced primer-dimers and other non-specific products. Amplifications with hot-start polymerase and no reagent according to this invention, lanes 7-9, were on the whole somewhat cleaner but still showed a manifestation of mispriming. Addition of reagent according to this invention to the hot-start amplifications, lanes 10-12, produced only the intended products. Addition of reagent according to this invention to the non-hot-start amplifications, lanes 4-6, eliminated primer-dimers and in two of the three repeats (lane 4 and lane 6), all other non-specific products as well, with only one repeat (lane 5) manifesting a non-specific product. The results presented in FIG. 11 demonstrate that a low concentration (100 nM) of reagent 9-22 DD was sufficient to prevent formation of primer-dimers when either non-hot-start Taq polymerase or not-start Taq polymerase was used, and further that that low concentration of reagent 9-22 DD was nearly sufficient to prevent all manifestations of mispriming even when used with non-hot-start Taq polymerase in amplifications of two targets with two primer pairs.<br>
Example 12 Optimizing the Kinetics of a Duplex Real-time PCR and a Real-time PCR by Preventing Primer-Dimer Formation<br>
We designed a duplex real-time LATE-PCR assay for simultaneous amplification of sequences within exons of the murine Oct4 and Xist genes (GenBank Accession Number NM_013633 and L04961, respectively). Each reaction was run in a final volume of 50 jjl and contained the following reagents: 1 X PCR buffer (Invitrogen, Carlsbad, CA) comprised by 20 mM Tris-HCl, pH 8.4, and 50 mM KC1, 3mM MgCl2, 0.4 mM of each dNTP, 50 nM Oct4 Limiting Primer having the sequence 5' TGGCTGGACACCTGGCTTCAGACT 3' (SEQ ID NO: 45), 2 uM Oct4 Excess Primer having the sequence 5' CAACTTGGGGGACTAGGC 3' (SEQ ID NO: 46), 100 nM Xist Limiting Primer having the sequence 5' GGTCGTACAGGAAAAGATGGCGGCTCAA 3' (SEQ ID NO: 47), 2 uM Xist Excess Primer having the sequence 5' TGAAAGAAACCACTAGAGGGCA 3' (SEQ ID NO: 48), 1 pM Low-Tm Oct4 molecular beacon having the sequence 5' TET-CCG CCT GGG ATG GCA TAG TGT GGA AGG CGG-Dabcyl 3' (SEQ ID NO: 49) and 2 units of antibody-complexed Platinum® Taq DNA polymerase (Invitrogen, Carlsbad, CA). Either compound 9-3 DD or compound 9-3bDD was also included in the PCR mixture, at the concentrations specified below. A molecular beacon for the detection of Xist amplicons was not added in this example. Each assay also contained the reagents necessary for cell lysis and reverse transcription according to the PurAmp protocol (see Hartshorn et al. (2005) BMC Biotechnol 5: 2, in a 2.5 pi volume. In this duplex LATE-PCR, the final concentrations of such reagents were the following: 2.5 mM Tris acetate, pH 8.4, 3.75 mM potassium acetate and 0.4 mM magnesium acetate (diluted cDNA Synthesis Buffer, ThermoScript™ RT-PCR System, Invitrogen, Carlsbad, CA), additional 50 uM of each dNTP, 0.13 mM guanidine isothiocyanate, 6.7 /xM /2-mercapto-ethanol, 0.7 pM sodium citrate, pH 7.0, 0.7 x 104 % (vol/vol) dimethylsulfoxide and 0.2 x 10'4 % sarcosyl.<br>
Mouse genomic DNA (Sigma, St Louis, MO) was also added to each assay and provided the templates for PCR amplification. The number of genomes added to each tube was calculated based on a 6 pg/genome size (see Vendrely and Vendrely (1949) Experientia 5: 327-329).<br>
Amplification was carried out in an ABI Prism 7700 Sequence Detector (Applied Biosystems, CA) with a thermal profile comprised by 1 cycle at 95°C for 5 minutes; 15 cycles at 95 °C for 10 sec, 63 °C for 20 sec, and 72 °C for 30 sec; and 40 cycles at 95 °C for 15 sec, 55 °C for 25 sec, 72 °C for 35 sec, and 45 °C for 30 sec, with fluorescence acquisition at 45 °C in the TET channel.<br>
At the end of the PCR, amplification products were analyzed by gel electrophoresis in a 3% agarose gel in 0.5X TBE buffer for 2 hours and stained with ethidium bromide. Both the double-stranded and the single-stranded products, of the expected sizes, were visible for the two co-amplified genes, indicating efficient duplex LATE-PCR.<br>
FIG. 12 illustrates the effect of varying the stem composition of reagent 9-3DD on the kinetics of the duplex LATE-PCR described above, containing two sets of primers for the amplification of two non-homologous sequences within the Oct4 and Xist genes. The figure shows the fluorescent signals generated by accumulating Oct4 amplicons through hybridization with the TET-Oct4 molecular beacon. The results demonstrate that the Cj values are very similar in the presence of 300 nM 9-3DD (lines with triangles) or the same concentration of its modified form 9-3bDD (heavy lines without triangles) at each genome concentration analyzed (10 genomes, circle 122; 100 genomes, circles 123 ; 1000 genomes, circle 124. The kinetics of the real-time fluorescent signals is, instead, affected by the composition of the reagent's stem. Compound 9-3bDD has a higher- Tm stem compared to 9-3DD, thus, we believe, optimally preventing primer-dimer formation, which in turns results in very linear and parallel signals at all genome concentration tested. In the presence of the less stringent compound 9-3DD, instead, some of the fluorescent signals have a steeper slope but others start plateauing early in the reaction, indicating random formation of primer dimers. Signals generated at different template concentrations with compound 9-3 DD are less parallel than those obtained in the presence of compound 9-3bDD (with the modified stem). Linear amplification generating perfectly parallel signals (having a constant slope) is ideally desirable and particularly relevant for end-point types of analysis.<br>
Agarose gel analysis of the samples containing 9-3DD revealed bands, particularly visible at higher template numbers, that may be primer dimers and very small<br>
bands at lower template numbers are consistent with primers. Such bands did not appear on the analysis of the samples containing 9-3bDD.<br>
FIG. 13 illustrates the effect of varying the concentration of reagent 9-3bDD in the duplex LATE-PCR described in this example and also used for FIG. 12. In this case, lowering the concentration of reagent 9-3bDD from 300 nM (heavy lines without triangles) to 200 nM (lines with triangles) again affects the slope of linear amplification due to primer-dimer formation, as confirmed by agarose gel analysis. As a result, some of the samples containing low initial template numbers (10 genomes, circle 132) have higher fluorescence at the last cycle ("end point") than samples containing higher initial template numbers (100 genomes, circle 133 and 1000 genomes, circle 134).<br>
The concentration effect of the reagents described in this patent application on PCR efficiency was also tested in a LATE-PCR amplifying one template with one pair of primers. The amplified template was the same murine Oct4 sequence used in the duplex described for FIG. 12 and FIG. 13 and the primers and the molecular beacon sequences were also the same as in that reaction.<br>
Each reaction was run in a final volume of 100 ul and contained the following reagents: 1 X PCR buffer (Invitrogen, Carlsbad, CA) comprised by 20 mM Tris-HCl, pH 8.4, and 50 mM KC1, 3mM MgCl2) 0.25 mM of each dNTP, 50 nM Oct4 Limiting Primer, 2 |j,M Oct4 Excess Primer, 1 uM Low-Tm TET-Oct4 molecular beacon, and 2 units of antibody-complexed Platinum® Taq DNA polymerase (Invitrogen, Carlsbad, CA). Compound 9-3DD was also included in the PCR mixture, at concentrations of 150, or 300 or 450 nM. As in the case of the aforementioned duplex (see FIG. 12 and FIG. 13), each assay also contained the reagents necessary for cell lysis and reverse transcription according to the PurAmp protocol, in a 10.5 jol volume. In this LATE-PCR, the final concentrations of such reagents were the following: 5 mM Tris acetate, pH 8.4, 7.5 mM potassium acetate and 0.8 mM magnesium acetate (diluted cDNA Synthesis Buffer), 1 ng/jil Random Hexamers and additional lOOuM of each dNTP (all components of a ThermoScript™ RT-PCR System, Invitrogen, Carlsbad, CA), 0.4 mM guanidine isothiocyanate, 20 uM |3-mercapto-ethanol, 2 |oM sodium citrate, pH 7.0, 2 x 10"4 % (vol/vol) dimethylsulfoxide and 0.5 x 10"4 % sarcosyl.<br>
Mouse genomic DNA was also added to each assay and provided the templates for PCR amplification, as specified for the duplex LATE-PCR described in this example. Amplification was again carried out in an ABI Prism 7700 Sequence Detector with the same thermal profile detailed for the duplex reaction. At the end of the PCR, amplification products were analyzed by gel electrophoresis in a 3% agarose gel in 0.5X TBE buffer for 2 hours and stained with ethidium bromide. Both the double-stranded and the single-stranded products were visible on the gel and had the expected sizes, indicating efficient amplification by LATE-PCR.<br>
The effects of increasing concentrations of compound 9-3 DD on the amplification efficiency of the Oct4 template were also tested. The Cj values were obtained from realtime PCR assays containing 5 genomes (average of two replicates for each 9-3DD concentration tested), 20 genomes (average of two replicates for each 9-3DD concentration tested), 100 genomes, and 1000 genomes performed with each of the 9-3DD concentrations tested: 150nM, 300 nM, 450 nM. The linear regression for each CT point series at each concentration is shown in FIG. 14, where triangles are 450nM concentration, open squares are 300nM, and filled squares are 150nM. It is immediately apparent from this plot that 450 nM 9-3DD greatly delays the appearance of the fluorescent signals above the threshold, also altering the dose-dependence of the PCR amplification (see the points at 10 gene copies and 40 gene copies, distant from the line of regression). The CT values obtained using 150 nM or 300 nM 9-3DD, on the other hand, are very similar and demonstrate a linear relationship between template copy number and fluorescent signal first appearance.<br>
Analysis of the real-time patterns of fluorescence, however, highlights a difference between the two conditions (concentrations) as illustrated in FIG. 15. The slope of the curves generated by either 100 genomes, circle 151, or 1000 genomes, circle 152, is steeper when using 300 nM 9-3DD (heavy lines with no triangles) than when using 150 nM 9-3DD (lines with triangles), suggesting that the more stringent conditions, 300nM, eliminate primer dimers, thus increasing amplification efficiency.<br>
Example 13. Use of Multiple Reagents AccordinR to this Invention.<br>
LATE-PCR reaction mixtures were prepared to amplify five different target sequences of human genomic DNA starting material individually and to amplify all five together in a multiplex reaction. Each target sequence required its own pair of limiting and excess primers. The target sequences were (1) a 191-base pair Globin allele, (2) a 312 base-pair Tay Sachs G269 allele, (3) a 452 base-pair Tay Sachs 1278 and 1421 allele, (4) a 549 base-pair segment of Mitochondrial hypervariable region 1, and (5) a 611 base-pair segment comprising p53 gene exons 7 and 8. The 25 ul reaction mixtures contained Ix PCR buffer (Invitrogen), 0.4mM dNTPs, 3mM Mg++, 0.24 x SYBR Green and 1.5 units of Taq DNA polymerase (Invitrogen). Primers were added at a concentration of 50nM for Limiting Primer and lO00nM for Excess Primer. All five primer pairs were added to the multiplex reactions. Thermal cycling was forty-five cycles of 95°C for 10 sec, 64°C for 20 sec, and 72°C for 1 min.<br>
Both the reaction mixtures with individual primer pairs and individual targets, and the reaction mixtures with all five primer pairs and all five targets were amplified in the presence of a combination of 25nM compound 9-22DD and l00nM compound 12-3DD. The pentaplex reaction mixture was amplified also without addition of any reagent according to this invention. Reaction products were examined by gel electrophoresis. Gel electrophoresis results of the various amplifications are shown in FIG. 16, which includes size markers (100 base-pair differences) in lane r. Central lanes g-k are the products of individual amplifications of target sequences (1) - (5), respectively. Lanes 1-q are the products of six replicate pentaplex amplifications that included the mixture of compound 9-22DD and compound 12-3DD. Lanes a-f are the products of six replicate pentaplex amplifications that included no reagent according to this invention. FIG. 16 shows that the mixture of compounds enhanced amplification (more and cleaner desired amplicons) as compared to the multiplex reaction without addition of any compound, demonstrating that mixtures of reagents may be used in kits, amplifications and assays of this invention.<br>
Example 14. Testing Modes of Action<br>
We have devised a test to investigate quantitatively the mispriming reduction (specificity improvement) and polymerase-inhibiting effects of reagents according to this invention, which we believe distinguishes First Mode from Third Mode action. The test is a PCR amplification assay essentially similar to the assay described in Example 1, with the following exceptions: the amplification reaction mixture contains half (1000 genomes) of the sheared genomic DNA, and the final portion of thermal cycling is reduced from 70 cycles to 40 cycles. Amplifications are performed with the addition of varying amounts of reagent according to this invention, typically 25nM, 50nM, l00nM, 300nM, IS00nM and SO00nM.   The test includes two analyses: first, for specificity, a melting curve of the amplified product; and, second, for inhibition, a real-time fluorescence curve of double-stranded product being synthesized.<br>
FIGS. 17-18 present portions of the analytical results for two reagents according to this invention, compound 12-3DD and compound 12-C3DD, respectively. In each figure Column I presents melting curves for several selected concentrations of reagent, as indicated. In each figure Column II presents real-time florescence curves for the same concentrations, as indicated. Referring to FIG. 17, Column I, melting curves showed that compound 12-3DD achieved high specificity (avoiding mispriming) at concentrations of l00nM and above (the lowest concentration, l00nM, being slightly higher than found for the reaction described in Example 1). Referring to FIG. 17, Column II, real-time fluorescence curves showed that polymerase inhibition increased progressively as the concentration of compound 12-3DD was increased above l00nM until the reaction was essentially shut down at concentrations above lO00nM. Inhibition is reflected in the lower amount of total double-stranded product(s) generated (including the desired specific product and non-specific products of mispriming) indicated by the plateau florescence level, and by the delay in the CT value (19.6 for l00nM, 21.1 for 300nM, and nonexistent, that is, at least 40 for 1500nM and 3000nM). Referring to FIG. 18, Column I, melting curves showed that compound 12-3DD achieved high specificity at concentrations of 300nM and above (the lowest concentration, 300nM, being the same as found for the reaction described in Example 1). Referring to FIG. 18, Column II, realtime fluorescence curves showed that polymerase inhibition increased progressively as<br>
the concentration of compound 12-C3DD was increased above 300nM (Or values were 18.8 for l00nM, 18.8 for 300nM, 21.1 for IS00nM and 24.5 for 3000nM) but that the reaction was not shut down even at a concentration of 3000nM.<br>
Comparing FIG. 18 and FIG. 18, it can be seen that the overall performance of comp9ound 12-C3DD is less concentration dependent than the overall performance of compound 12-3DD. Thus, for this particular amplification reaction, the CT delay and plateau florescence decrease resulting as compound 12-C3DD concentration goes from 300nM to IS00nM are roughly the same as result as compound 12-3DD concentration goes from l00nM to 300nM.<br>
A number of embodiments of the invention have been described. Nevertheless, it will be understood that various modifications may be made without departing from the spirit and scope of the invention. Accordingly, other embodiments are within the scope of the following claims.<br><br><br><br><br>
What is claimed is:<br>
1.	A reagent capable of preventing at least one manifestation of mispriming in a<br>
polymerase chain reaction (PCR) amplification to produce at least one amplified DNA<br>
product when added at a concentration of not more than 650 nM to a PCR amplification<br>
mixture that includes 1.25 units of a thermostable DNA polymerase per 25p.l of reaction<br>
mixture, said reagent being a non-extendable oligonucleotide that has a stem-loop<br>
structure, that is not a hybridization probe for said at least one amplified DNA product,<br>
that has a stem that is greater than six nucleotides in length, that is stabilized at its<br>
terminus away from the loop, and that has a calculated stem melting temperature (Tm)<br>
below 94 °C.<br>
2.	The reagent according to claim 1 wherein the loop is an oligonucleotide<br>
comprising at least three nucleotides.<br>
3.	The reagent according to claim 1 wherein the loop is a non-nucleotide<br>
chemical linker.<br>
4.	The reagent according to any of claims 1-3 wherein the stem is terminally<br>
stabilized by interacting chemical moieties that are covalently attached to the 3' and 5'<br>
nucleotides of the stem.<br>
5.	The reagent according to claim 4 wherein neither of the interacting chemical<br>
moieties is fluorescent.<br>
6.	The reagent according to any of claims 1-5 wherein the portion of the stem that<br>
is double-stranded at temperatures below its Tm has a length of 9-12 base pairs.<br>
7.	The reagent according to any of claims 1-6 wherein the calculated stem<br>
melting temperature is in the range of 72-85 °C.<br>
8.	The reagent according to ay of claims 1-6 wherein the calculated stem melting<br>
temperature is the range of 50-71 °C.<br>
9.	A method for preventing at least one manifestation of mispriming during a<br>
polymerase chain reaction (PCR) amplification to produce at least one amplified DNA<br>
product from an amplification reaction mixture that includes deoxynucleotide phosphates<br>
(dNTPs) and a thermostable DNA polymerase, comprising adding to said reaction<br>
mixture at least one reagent according to any of claims 1-8.<br>
10.	The method according to claim 9 wherein the reagent has an oligonucleotide<br>
loop and the concentration of said reagent is not more than 300 nM.<br>
11.	The method according to claim 9 wherein the reagent has a loop that is a non-<br>
nucleotide chemical linker and the reagent is added to the reaction mixture at a<br>
concentration of not more than l000nM.<br>
12.	The method according to any of claims 9-11 wherein the amplification is a<br>
linear-after-the-exponential PCR (LATE-PCR) assay including a low-temperature<br>
detection step performed at a temperature lower than said Tm.<br>
13.	The method according to any of claims 9-12 wherein the amplification is a<br>
symmetric PCR assay.<br>
14.	The method according to any of claims 9-13 wherein the amplification is<br>
followed by end-point detection.<br>
15.	A kit of reagents for performing a polymerase chain reaction (PCR)<br>
amplification comprising a thermostable DNA polymerase , at least one pair of PCR<br>
primers, dNTP's, and at least one reagent according to any of claims 1-8.<br>
16.	A kit according to claim 15 further comprising reagents for nucleic acid<br>
isolation.<br>
17.	A set of oligonucleotides for performing a polymerase chain reaction (PCR)<br>
amplification comprising at least one pair of PCR primers and at least one reagent<br>
according to any of claims 1-8.<br>
18.	The set according to claim 17 further including at least one fluorescently<br>
labeled probe that hybridizes to the amplified product defined by said at least one pair of<br>
PCR primers.<br>
19.	The set according to claim 17 or claim 18 wherein the at least one primer pair<br>
includes a limiting primer and an excess primer in a ratio appropriate for performing a<br>
linear-after-the-exponential PCR (LATE-PCR) amplification.<br>
20.	The set according to any of claims 17-19 wherein said amplified product is<br>
single-stranded.<br>
21.	A reagent capable of preventing manifestation of mispriming in a PCR<br>
amplification, a method thereof and a kit and a set of reagents performing a PCR<br>
substantially as herein described with reference to forgoing description, examples and<br>
diagrams.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUFic3RyYWN0LSgyMS0wMi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Abstract-(21-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUNsYWltcy0oMTgtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Claims-(18-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUNsYWltcy0oMjEtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Claims-(21-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Correspondence Others-(07-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Correspondence Others-(11-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Correspondence Others-(18-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Correspondence Others-(21-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUZvcm0tMS0oMDctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Form-1-(07-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUZvcm0tMTMtKDEwLTA4LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Form-13-(10-08-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUZvcm0tMTgtKDI2LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Form-18-(26-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUZvcm0tMy0oMTEtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Form-3-(11-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LUdQQS0oMDctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-GPA-(07-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LXBjdC0xMDYucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-pct-106.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LXBjdC0zMTEucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-pct-311.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyMS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTEtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3621-delnp-2007-Petition-137-(11-11-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259312-a-granular-starch-hydrolyzing-enzyme-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259314-battery-for-mobile-communication-terminal.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259313</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3621/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>11/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-May-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BRANDEIS UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>415 SOUTH STREET, MS115, WALTHAM, MASSACHUSETTS, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PIERCE, KENNETH</td>
											<td>52 WALNUT STREET, NATICK, MA 01760, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WANGH, LAWRENCE, J.</td>
											<td>20 DUFFIELD ROAD, AUBURNDALE, MA 02466, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RICE, JOHN</td>
											<td>268 COMMON STREET, QUINCY, MA 02169, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SANCHEZ, AQUILES, J</td>
											<td>14 FOSTER DRIVE, FRAMINGHAM, MA 01701, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SALK, JESSE</td>
											<td>3011 WEBSTER POINT ROAD N.E. SEATTLE, WA 98105, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>REIS, ARTHUR</td>
											<td>28 WILLIAMS STREET, ARLINGTON STREET, ARLINGTON, MA 02476-5624, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/037688</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/619,670</td>
									<td>2004-10-18</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259313-reagents-and-methods-for-improving-reproducibility-and-reducing-mispriming-in-pcr-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:28:18 GMT -->
</html>
