<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259562-a-method-for-rapid-isolation-of-rna-and-a-kit-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:27:33 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259562:A METHOD FOR RAPID ISOLATION OF RNA AND A KIT THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR RAPID ISOLATION OF RNA AND A KIT THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for rapid isolation of RNA. More particularly, it relates to a method for isolation of RNA using two-solution system. The present invention also relates to a RNA isolation kit.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD FOR RAPID ISOLATION OF RNA AND A KIT THEREOF<br>
Field of invention:<br>
The present invention relates to a method for rapid isolation of RNA. More particularly, it relates to a method for isolation of RNA using two-solution system. The present invention also relates to a RNA isolation kit. Background and prior art of invention:<br>
Extensive research has been undertaken in the field of molecular biology to facilitate the deciphering of underlying mechanisms of gene expression, signal transduction, gene regulation and transcriptome analysis. This involves a whole gamut of techniques such as reverse transcription polymerase chain reaction (hereinafter, referred to as RT-PCR), northern hybridization, construction of cDNA libraries and in vitro translation. Substantially pure and undegraded RNA is a fundamental requisite for all the above-mentioned techniques.<br>
Several compositions and procedures for isolation of RNA have been described as mentioned below:<br>
Current Protocols in Molecular Biology (Ausubel, F.M., Brent, R., Kingston, R. E., Moore, D.D., Seidman, J.G., Smith, J.A. and Struhl, K. Eds, 1994. John Wiley and Sons, Inc. USA) described a RNA isolation protocol, which involves cell lysis and protein removal by phenol/SDS, followed by selective precipitation of RNA using lithium chloride. In brief, the tissue is ground in liquid nitrogen using mortar pestle followed by transfer to grinding buffer (0.18 M Tris, 0.09 M LiCl, 4.5 mM EDTA and 1% SDS, pH 8.2) containing phenol saturated with 0.2M Tris , 0.1 M LiCl and 5 mM EDTA solution (pH 8.2) in a ratio of 3:1. After homogenization in a polytron, proteins are removed from the aqueous phase by several re-extractions with saturated phenol and chloroform. The aqueous phase so obtained is precipitated several times with 8 M LiCl. Because of repeated precipitation steps, the procedure becomes time consuming, tiring and expensive.<br>
Cox, R.A. (in Methods in Enzymology 1968, Grosmann, L. and Moldave, K. Eds. Vol.12 B, pp. 120-129, Academic Press, Orlando, FL) described RNA isolation method using guanidine hydrochloride. Guanidine hydrochloride is a strong inhibitor of ribonucleases. However, the procedure is time-consuming. Guanidine salts employed for the isolation of RNA are extremely poisonous. Moreover, several plant tissues are recalcitrant to extraction in guanidinium salts. Negligible or no RNA is obtained from certain tissues by extraction in guanidine [R.C. Bugos, V.L. Chiang, X-H. Zhang, E.R. Campbell, G.K. Podila, W.H. Campbell. RNA isolation from plant tissues recalcitrant to extraction in guanidine, Biotechniques 19 (1995) 734-737].<br>
Chirgwin, J.M., Przybyla, A.E., Macdonald, R.J.,&amp; Rutter, W.J. (1979) Biochemistry 18: 5294-5299, described a method for isolation of RNA using strong denaturant guanidine thiocyanate, in which both cation and anion are potent chaotropic agents. In this method, the tissue is homogenized in a solution containing guanidine thiocyanate (4M), sodium N-laurylsarcosine (0.5%), sodium citrate, pH 7.0 (25mM) and beta-mercaptoethanol (0.1M). The supernatant is acidified with 1M acetic acid, and RNA is precipitated with 0.75 volume of absolute alcohol at -20°C. Further steps involve dissolution of RNA pellet obtained after centrifugation in guanidine chloride solution (7.5M), buffered with sodium citrate pH 7.0 containing 5mM dithiothreitol. Re-precipitation is done in acetic acid and ethanol for 3-4h at -20°C. Another dissolution and re-precipitation step is involved for isolation of RNA. A modification of this procedure involves separating RNA from the homogenate by ultracentrifugation through a cesium chloride gradient. This method also uses toxic gunanidine salt and is very time consuming. Wallace, D.M. (in Methods in Enzymology, 152:33-41; 1987) described phenol-based extractions of RNA from biological tissues. Using warm buffer-saturated phenol, aqueous phase obtained was re-extracted with chloroform-isoamyl alcohol and buffer-saturated phenol. Re-extraction is repeated and RNA is precipitated with ethanol by incubating overnight. The procedure is time consuming as it requires overnight precipitation procedures. Further, there is no protection to RNA from RNases in the aqueous phase US Patent 4,843,155 and Chomczynski, P. and Sacchi, N. (1987) Anal. Biochem. 162:156-159 described a procedure for simultaneous isolation of RNA, DNA and protein. This is also termed as acid guanidinium-phenol-chloroform (hereinafter called as, AGPC)<br>
method. In this procedure, tissue is homogenized in a solvent consisting of 4M guanidinium thiocyanate, 25mM sodium citrate, pH 7; 0.5% sarkosyl, 0.1M 2-mercaptoethanol. Phase separation is done by addition of phenol saturated with water and 0.2M sodium acetate, pH 4.0, and chloroform-isoamyl alcohol mixture (49:1) by vigorous mixing and centrifugation. RNA compartmentalizes to the aqueous phase whereas DNA and proteins are present in the organic and the interphase. Precipitation is done by the addition of equal volume of isopropanol, the resulting RNA pellet is dissolved in homogenization solution and precipitated with 1 volume of isopropanol at -20°C for 1 hour. Washing is done in 70% ethanol and RNA pellet is dissolved in 0.5% SDS. This procedure requires at least 4 hours for RNA isolation.<br>
Method developed by Chomczynski, P. and Sacchi, N. (1987) Anal. Biochem. 162:156-159 involving modification of guanidinium thiocyanate method of Chirgwin, J.M., Przybyla, A.E., Macdonald, R.J.,&amp; Rutter, W.J.(1979) Biochemistry 18: 5294-5299, using guanidine thiocyanate-phenol-cholroform at acidic pH is rapid but this AGPC method usually yields very high amounts of genomic DNA contamination. Siebert, P.D. and Chenchik, A. (1993) Nucl. Acid Res. 21(8): 2019-2020 made simple modifications of AGPC method by addition of a selective RNA precipitation step. This involved addition of one-third volume of 95% ethanol following lysis in guanidine thiocyanate. Isopropanol precipitation step was also reduced from lh to 30 min. DNA contamination was significantly reduced. But still the method required 3-4h for RNA isolation and is not suitable for tissues resistant to guanidine salts.<br>
United States Patent 5,945,515 by Chomczynski, P. (1999) disclosed a solution for simultaneous isolation of RNA, DNA and proteins. The solution consisted of guanidinium thioicyanate in 40-60% phenol, glycerol as phenol solubilizer and a buffer to maintain the solvent pH at or about 4. The mixture is a homogenous mixture (monophasic). Phase separation is effected by the addition of chloroform at 10% that results in partitioning of RNA in the aqueous phase. Proteins and DNA are concentrated in the organic or the interphase. RNA precipitation is carried out by the addition of equal volume of isopropanol to the aqueous phase. The RNA pellet is obtained by centrifugation that is washed with 70% ethanol and allowed to dry. The presence of very<br>
high concentration (2-5M) of guanidinium salts makes the solution extremely hazardous to health. Further, tissues recalcitrant to guanidinium salts do not yield any RNA. US Patent 5,777,099 by Mehra, M. (1998) describes a method for separation of RNA from liquid samples of biological origin, by contacting with a biphasic solution wherein the upper phase is phenol and the lower phase is aqueous phase containing guanidinium salt, a buffer, and urea. Separation of aqueous phase is effected by addition of water-insoluble solvent such as chloroform. RNA is recovered from the resulting aqueous phase. The presence of guanidinium salts makes the solution extremely hazardous to health. Further, tissues recalcitrant to guanidinium salts do not yield any RNA. U.S. Pat. No. 5,010,183, by Macfarlane, D. E. (1991) describes the ability of cationic surfactants to lyse cells and simultaneously precipitate RNA and DNA from solution. This method differs fundamentally from those described above in that its first step renders the RNA insoluble, whereas in the above described methods the first step is to solubilize RNA. In this method, a 2% solution of the surfactant benzyldimethyl-n-hexadecylammonium chloride together with 40% urea and other additives are added to a cell suspension, and the mixture is centrifuged. The pellet is resuspended in ethanol, from which the RNA and DNA is precipitated by the addition of a salt. In attempts to apply this method to blood, the inventor himself found that the use of the latter surfactant and other commercially available surfactants results in inefficient precipitation of RNA and incomplete lysis of blood cells.<br>
US patent No. 5,300,635 by Macfarlane, D. E. (1994) discloses a novel method for isolating nucleic acids from biological samples, most particularly blood, using selected quaternary amine surfactants. The selected quaternary amine is produced through the reaction of a quaternary amine hydroxide and an acid of the group consisting of phosphoric, sulfuric, formic, acetic, propionic, oxalic, malonic, succinic and citric. The quaternary amine is either an acyltrimethylammonium or an acylbenzyldimethylammonium, where the acyl group contains 12, 14, 16 or 18 carbons. This method employs 4M guanidinium isothiocyanate solution for dissociation of nucleic acids from nucleic acid/ quaternary amine complex which is quite hazardous. Further, the method is mostly suitable for isolation of RNA from blood and animal tissue, whereas no experiment has been conducted on plant tissue.<br>
Feramisco, J.R., Helfman, D.M., Smart, J.E., Burridge, K., and Thomas, G.P. (in Molecular Cloning (Maniatis, T., Fritsch, E. F., and Sambrook, J., Eds. (1982), PP. 194-195, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY) reports another RNA isolation procedure wherein RNA-containing samples are homogenized in a solution of 4M guanidinium thiocyanate, 20% sodium lauryl sarcosine, and 2-mercaptoethanol. Following homogenization, an equal volume of heated phenol (60°C) and sodium acetate pH 5.2 are added to the homogenate. Next, an equal amount of chloroform is added, and the mixture is cooled and centrifuged. The aqueous phase is decanted and re-extracted several times with a phenol/chloroform solution in order to maximize the yield. The procedure requires a skilled technician and takes four to five hours to complete and also employs guanidinium thiocyanate which is quite hazardous.<br>
United States Patent No. 20040019196 by Bair, R J Jr., Heath, E M., Meehan, H, Paulsen, K E, Wages, J M. Jr.(2004) describes a method for isolation of RNA from blood, mammalian cells, plant tissue, bacteria and fungi. Initially, 200 uL of Lysing/Binding Solution (4 LiCl, 5% Triton X-100, 5% DGME (diethylene glycol monoethyl ether), 10 mM EDTA, 10 mM TCEP (Tris (carboxyethyl) phosphine), 1% sodium tungstate, in 100 mM TRIZMA at pH 8.8) was added for each 30 mg of tissue sample in a tube. Tissue was homogenized in a roto-stator at low speed and then the speed was increased to complete homogenization for an additional minute. After homogenization, 200 µl of homogenized lysate was added to a pre-clear column (Gentra). Depending upon the tissue and extent of homogenization, the pre-clear column trapped particulates, while allowing the bulk of the applied lysis volume containing the RNA to wash through the filter. After centrifugation in the pre-clear column, the cleared lysate at the bottom of the pre-clear tube was vortexed briefly. Then, the entire volume of cleared lysate (.about.200 uL) was pipetted onto the purification column, which contained a borosilicate glass fiber membrane (Whatman D glass fiber membrane) within a basket and placed inside a 2 ml microfuge tube. The microfuge tube was then spun at maximum speed in a microcentrifuge for 1 minute. The microfuge tube was then turned 180 degree in the microcentrifuge and centrifuged for an additional 2 minutes. After centrifugation of lysate and subsequent binding of RNA to the borosilicate membrane surface, 200 uL of Wash I Solution (5 M LiCl and 55% ethanol) was added to the column material and spun<br>
at maximum speed in a microcentrifuge for 1 minute. The basket containing the membrane was then transferred to a new microfuge tube and 200 uof Wash II Solution (5 mM EDTA, 70% ethanol, in 100 mM Tris HC1 at pH 7.6) was added to the column material and spun at maximum speed in a microcentrifuge for 1 minute. The Wash II Solution addition and centrifugation steps were repeated once. To elute the RNA from the solid support, the basket containing the membrane was transferred to a new microfuge tube and 50 uL of DEPC-treated water was added to the column material and spun at maximum speed for 1 minute. The patent doesn't not provide any experiment /data on isolation of RNA from plant tissues which are usually very difficult due to presence of polysaccharides and other contaminating material. The use of lithium chloride and other accessories renders this method expensive. United States Patent No. 5,973,137 by Heath, E M. (1999) describes a method for isolation of RNA from human whole blood, plant and animal tissues, cultured cells, body fluids, yeast, and bacteria. Inventor has used a "Cell Lysis Reagent," which includes an anionic detergent (salts e.g., sodium, potassium, and lithium salts of dodecyl sulfate as well as N-lauroyl sarcosine; 1.8-2.2% weight/volume) dissolved in water buffered with sodium citrate and citric acid at concentration of 66-70 mM and 130-134 mM, concentration, respectively. In addition to the anionic detergent and buffer, the reagent includes a chelating agent such as ethylene diamine tetraacetate (EDTA) as a preferred reagent in an amount effective to reduce DNase activity. The second reagent, referred to therein as a "Protein-DNA Precipitation Reagent," includes a sodium or potassium salt such as sodium chloride, sodium acetate, potassium chloride and potassium acetate in a relatively high salt concentration (3.8-4.2 M) dissolved in water. Inventor has used anionic detergent such as sodium dodecyl sulfate as inhibitor of RNAses. RNAses is a major problem in the tissues, particularly in old and stressed tissues. Anionic detergents are mild inhibitors of RNAses (Wallace, D.M., in Methods in Enzymology, 152:33-41; 1987; please refer to the 7th line from the top on page 37) and hence would not be suitable for the tissues with high RNAses. Secondly, a very high salt concentration (3.8 to 4.2 M) used in the method would cause non-specific RNA aggregation leading to loss of, particularly, the minor RNA species (Wallace, D.M., in Methods in Enzymology, 152:33-41; 1987; please refer to the 2nd last para from the bottom on page 36).<br>
The drawbacks in the prior art are:<br>
The methods use guanidinium salts (Guanidine hydrochloride and guanidine thiocyanate)<br>
as a major constituent. Guanidine hydrochloride is extremely hazardous and rated as<br>
harmful by CHIP (UK Chemicals Hazard information and Packaging) and toxic by HCS<br>
(US Hazard Communication standards). Oral Rat LD50 = 475 mg/kg.<br>
Tissues recalcitrant to guanidine salts do not yield RNA or the yield is too poor to allow<br>
any further use<br>
Available reagents in the market are very expensive.<br>
Non-guanidine hydrochloride based procedures are very lengthy.<br>
Solutions/protocols not involving the use of guanidine salts are not suitable for tissues<br>
with high RNAses and also would lead to loss to RNA species due to the high salt<br>
concentration used to inhibit RNase activity.<br>
Some of the methods use cesium chloride for isolation of RNA on ultracentrifuge, which<br>
is a very expensive instrument. Also, skilled personnel are required to operate the<br>
ultracentrifuge taking all safety precautions, thus, the procedure becomes very expensive<br>
and time consuming.<br>
Objects of invention:<br>
The main object of the present invention is to provide a method for rapid isolation of<br>
RNA.<br>
Another object of the present invention is to provide a method for isolation of RNA using<br>
two-solution system.<br>
Further, another object of the present invention is to provide a cost effective, less<br>
hazardous, two-solution system for rapid isolation of RNA.<br>
Yet another object of the present invention is to provide a two a RNA isolation kit<br>
suitable for downstream applications employed in molecular biology includes synthesis<br>
of a complementary DNA (cDNA) using RNA as a substrate and performing northern<br>
hybridization, polymerase chain reaction, cloning of genes, preparation of probe and<br>
construction of a cDNA library.<br>
Brief description of figures:<br>
Figure 1 represents comparison of total RNA isolated from different tissues from various<br>
species using different RNA isolation methods, where P, G, T, and R stand for present<br>
method, guanidine-HCl, Trizol and RNeasy methods, respectively.<br>
Figure 2 represents total RNA isolated from leaf and root of Arabidopsis thaliana cv. col-<br>
0. A represents RNA isolated from leaf and B represents RNA isolated from root.<br>
Figure 3 represents total RNA isolation from leaf of Arabidopsis thaliana cv. col-0 using<br>
different quantities of phase separation solution. A, B, C, and D contains 200, 400, 600<br>
and 800 uL of solution II respectively.<br>
Figure 4 represents total RNA isolated from Solarium tuberosum, Capsicum annum,<br>
Rheum, Picrorhiza leaves and roots. A is RNA from Solanum tuber, B is RNA from fruit<br>
of Capsicum, C is RNA from Rheum leaves, D is RNA from Picrorhiza leaves and E is<br>
RNA from Picrorhiza roots.<br>
Figure 5 represents total RNA isolated from vegetative Bud and third leaf of tea. M<br>
represents RNA marker; A, RNA isolated from vegetative bud; B, RNA isolated from<br>
third leaf<br>
Figure 6 represents RT-PCR based amplification of the RNA of Arabidopsis thaliana cv.<br>
col-0 using actin primers. M represents 100 bp marker; C represents control reaction<br>
without template; A represents amplification with cDNA of Arabidopsis.<br>
Figure 7 represents RT-PCR based amplification of the RNA of Rheum using actin<br>
primers.<br>
Figure 8 represents RT-PCR based amplification of the RNA of Arabidopsis thaliana cv.<br>
col-0 using wrky primers. M represents 100 b.p. marker. A represents control reaction<br>
with template and B showing amplification of 500 base pairs.<br>
Figure 9 represents RT-PCR based amplification of Rheum RNA with wrky specific<br>
primers. Where M is 100 b.p. marker, A is control reaction without template and B is<br>
amplification product of 700 base pairs.<br>
Figure 10 represents RT-PCR based amplification of the RNA of Vegetative bud and<br>
third leaf from tea with DFR specific primers. Where M is 100 base pairs marker, A is<br>
amplification in vegetative bud and B is amplification in third leaf.<br>
Figure 11 represents northern hybridization with RNA from tea bud (A) and third leaf (B)<br>
using 950 base pairs DFR fragment as probe.<br>
Summary of invention:<br>
The present invention deals with a cost effective and less hazardous method for rapid isolation of RNA using two solution system. It also provides a rapid RNA isolation kit. It provides RNA from a range of tissues that can be utilized for various downstream applications like RT-PCR, northern hybridizations etc.<br>
Detailed description of invention:<br>
Accordingly, the present invention provides a method for rapid isolation of RNA, wherein the said method comprising the steps of:<br>
a)	grinding 10 to 100 mg tissue in liquid nitrogen to make it fine powder;<br>
b)	adding 1 to 2ml of solution I in the powdered sample obtained from step (a) followed by homogenizing to make a fine powder;<br>
c)	adding   600 to 800ul of solution II in powdered sample obtained from step<br>
(b);<br>
d)	adding 150 to 200ul of chloroform in the above said solution obtained from step (c) followed by vortexing and kept it at room temperature for 10 min to separate the layer;<br>
e)	transferring upper layer obtained from step (d) into fresh tube;<br>
f)	adding isopropanol into above layer obtained from step (e) in the ratio 5:3 to 5:4 followed by vortexing and kept it at room temperature for 10 min;<br>
g)	centrifuging the solution obtained from step (f) for 5 to 10 min at about 4 degree C to get the desired RNA pellet;<br>
h) washing the resultant RNA pellets obtained from step (g) with 70% ethanol<br>
followed by air drying; i)   dissolving the washed pellet obtained from step (h) in appropriate amount of DEPC treated water. In an embodiment of the present invention, the said tissue is selected from the group consisting of tubers of Solarium tuberosum, fruits of Capsicum annum, Rheum leaves, leaves and roots of Picrorhiza kurroa, leaves and roots of Arabidopsis thaliana.<br>
In another embodiment of the present invention, the said solution I comprises of saturated phenol of pH 6.0-6.8, anionic detergent, acetate salt and a chelator wherein the ratio of the individual ingredient is in the range of 10 : 0.003 : 0.02 : 1.0 to 10 : 0.03 : 0.08 : 2.0 respectively.<br>
In further another embodiment of the present invention, saturated phenol used is preferably of a pH less than 7.<br>
In yet another embodiment of the present invention, the anionic detergent used is selected from the group of a dodecyl sulphate salt or N-lauroyl sarcosine.<br>
In still another embodiment of the present invention, the anionic detergent used is in an amount of about 0.3-1% weight/volume most preferably about 0.5-0.6% W/V. In still another embodiment of the present invention, the acetate salt used is selected from the group of sodium or potassium.<br>
In still another embodiment of the present invention, the acetate salt used is in concentration of 0.2-0.8 M.<br>
In still an embodiment of the present invention, the chelating agent used is selected from the group of disodium and dipotassium of ethylene diamine tetraacetic acid In still another embodiment of the present invention,  the  solution II  comprising diethylpyrocarbonate in deionised water having a conductivity of 17-18.2 mega-ohms. In still another embodiment of the present invention, the diethylpyrocarbonate used is in a concentration of about 0.1% volume /volume.<br>
In still another embodiment of the present invention, the isolated RNA is tested for downstream applications in molecular biology. Further, the present invention also provides a rapid RNA isolation kit comprising:<br>
a)	solution I;<br>
b)	solution II;<br>
c)	instructions for using the solutions.<br>
Present invention provides a cost effective, less hazardous, two-solution system for rapid isolation of RNA which includes solution I comprising of buffer saturated phenol (pH less than 7), SDS(0.1-1%), EDTA(10-20mM), sodium acetate(0.3-0.8M), and solution II comprising of DEPC(0.001-0.1%) treated deionized water. Solution system can be used<br>
to isolate RNA from diverse tissues. Time required in whole process of isolation of RNA is less than an hour and it provides pure and high quality RNA.<br>
The optimum quantity of solution II was standardized using the leaves of Arabidopsis thaliana (a model plant system in plant biology). The developed procedure was employed for a range of plant materials which included tubers of Solanum tuberosum (carbohydrate rich), fruits of Capsicum annum, Rheum leaves (recalcitrant to guanidine containing solutions), leaves and roots of Picrorhiza kurroa, leaves and roots of Arabidopsis thaliana, and vegetative buds and third leaf of Camellia sinensis (high polyphenol containing tissue). Comparative data of yield of isolated RNA using present protocol and other protocol is given in table 1.<br>
Table 1<br>
Comparative data of yield of isolated RNA<br><br><br>
(Table Removed)<br>
The following examples are given by way of illustration of the present invention and should not be construed to limit the scope of present invention<br>
EXAMPLE 1<br>
Preparation of solution for isolation of RNA<br>
To prepare solution I, phenol was saturated with Tris-(hydroxymethyl)-aminomethane (Tris) buffer to a pH of 6.7+0.2. Thereafter, 0.3-1% sodium dodecyl sulphate, 0.2-0.8M sodium acetate and 10-20 mM EDTA was added.<br>
The solution II was prepared separately wherein 0.1% DEPC (final concentration) was added to deionised water having a conductivity of 17-18.2 mega-ohms. Solution was autoclaved after leaving overnight.<br>
EXAMPLE 2<br>
To isolate total RNA, the following protocol are applied:<br>
RNA isolation using Guanidine hydrochloride-based procedure:<br>
(Singh, G., Kumar, S. and Singh, P. 2003. A quick method to isolate RNA from wheat and other carbohydrate-rich seeds. Plant Molecular Biology Reporter 21: 93a-93f)<br>
1.	Tissue (1 g) was ground in liquid nitrogen to fine powder. Powder was transferred into a new mortar containing 5 ml of the GH buffer [8 M guanidine hydrochloride, 20 raM ethylene diamine tetraacetic acid (EDTA), 20 mM MES {2-(N-morpholino)ethanesulfonic acid}, beta mercaptoethanol, 200 mM; pH 7.0] and was ground further.<br>
2.	Resulting homogenate was transferred to an oak-ridge tube containing equal volume of PCI (Phenokchloroform: isoamylalcohol).<br>
3.	Phases were emulsified by vortexing and separated by centrifugation at 10,000 rpm for 20 min (7°C).<br>
4.	Upper aqueous phase was transferred to a fresh oak-ridge tube and extracted with the equal volume of CI.<br>
5.	Resulting upper aqueous phase was transferred to a Corex tube and RNA was precipitated by adding 0.2 volume of 1 M acetic acid and 0.7 volume of chilled ethanol.<br>
6.	The tubes were kept at -72°C for 3 h.<br>
7.	Precipitate was pelleted followed by spin at 10,000 rpm for 10 min at 4°C. Pellet was washed thrice using 5 ml of 3 M sodium acetate (pH 5.2) followed by final washing with 70% chilled ethanol.<br>
8.	Pellet was dried and dissolved in minimum volume of DEPC-treated ADW.<br>
RNA isolation usingTRIzol Reagent:<br>
1.	Tissue (100 mg) was ground in liquid nitrogen to fine powder.<br>
2.	Added 1 ml of TRIzol reagent and further ground to a fine powder using pestle and mortar.<br>
3.	Incubated the homogenized samples for 5 min at 15 to 30°C. Added 0.2 ml of chloroform per ml of TRIzol reagent. The tubes were shaken vigorously for 15 seconds and incubated at 15 to 30°C for 2 to 3 min.<br>
4.	Centrifuged the samples at 12,000 X g for 15 min at 4°C.<br>
5.	Transferred the aqueous phase to a fresh tube.<br>
6.	The RNA was precipitated from the aqueous phase by mixing with isopropyl alcohol. Added 0.5 ml of isopropyl alcohol per 1 ml of TRIzol reagent used for initial homogenization.<br>
7.	Incubated the samples at 15 to 30°C for 10 min.<br>
8.	Centrifuged the samples at 12,000 X g for 10 min at 4°C.<br>
9.	Removed the supernatant. Washed the RNA pellet with 75% ethanol and air dry.<br>
10.	Dissolved RNA in 30-50 ul of RNase-free water.<br>
RNeasy Plant Mini Protocol for RNA isolation: (QIAGEN):<br>
1.	Tissue (100 mg) was ground under liquid nitrogen to a fine powder using a pestle and mortar. Transferred the tissue powder to a 2 ml tube. Do not allow the sample to thaw.<br>
2.	Added 450 ul of buffer RLC (containing guanidine hydrochloride) to maximum of 100 mg tissue powder. Vortexed vigorously. Ensure lOul p-ME is added to 1 ml of buffer RLC before use.<br>
3.	Applied lysate to the QIAshredder spin column sitting in a 2 ml collection tube, and centrifuged for 2 min at maximum speed. Transferred the flow-through fraction from QIAshredder to a new tube without disturbing the cell debris pellet in the collection tube.<br>
4.	Added 0.5 volumes ethanol to the cleared lysate and mixed well by pipetting.<br>
5.	Applied sample, including any precipitate, which may have formed, onto an RNeasy mini spin column sitting in a 2 ml collection tube. Centrifuged for 15 sec at 10,000 rpm.<br>
6.	Added 700 ul of buffer RW onto the RNeasy column, and centrifuged for 15 sec at 10,000 rpm to wash. Discarded the flow-through and collection tube.<br>
7.	Transferred RNeasy column into a new 2 ml collection tube. Added 500ul buffer RPE onto the RNeasy column and centrifuged for 15 sec at 10,000 rpm.<br>
8.	Added 500ul buffer RPE to RNeasy column, and centrifuged for 2 min at maximum speed to dry the RNeasy membrane.<br>
9.	Placed the RNeasy spin column in a new 2 ml collection tube, and discarded the old collection tube with filtrate. Centrifuged at full speed for 1 min.<br>
10.	Transferred RNeasy column into a new 1.5 ml collection tube, and added 50ul of RNase-free water directly onto the RNeasy membrane. Centrifuged for 1 min at 10,000 rpm to elute the RNA.<br>
Isolation of RNA using present protocol:<br>
a.	Leaves of Arabidopsis thaliana (100 mg) were ground in liquid nitrogen using<br>
mortar and pestle to make fine powder.<br>
b.	Added 2 ml of solution I.<br>
c.	Homogenized to make a fine powder in pestle.<br>
d.	Added 800ul solution II<br>
e.	Transferred sample into 2.0 ml eppendorf tubes.<br>
f.	Added 200ul of chloroform.<br>
g.	Vortexed and left at room temperature for 10 minutes,<br>
h.   Centrifµged for 10 minutes at room temperature<br>
i.   Transferred upper aqueous phase into a new 2.0 ml eppendorf tube.<br>
j.   Added 0.6 Volumes of isopropanol<br>
k.   Vortexed and left at room temperature for 10 minutes.<br>
1.    Centrifµged for five minutes at 4°C.<br>
m. Resulting RNA pellet was washed with 70% ethanol, air-dried and dissolved in appropriate amount of DEPC treated water. RNA was quantified by scanning absorbance between 220 to 320 nm; the purity was determined by calculating the ratio of absorbance measured at 260 and 280 nm. A value &gt;1.8 at 260/280 nm was considered ideal for the purpose of present investigation. The formula used to calculate RNA concentration and yield was as follows: Concentration of RNA (µg/ml) = A260 (Absorbance at 260 nm) X 40 X dilution factor Total yield (µg) = concentration X volume of stock RNA sample. The quality of RNA isolated was assayed on 1.2% agarose gel containing formaldehyde. To check the integrity of RNA, 5-6 µg of RNA in 4.5 ul of DEPC treated autoclaved water was diluted with 15.5 ul of Ml solution (2 ul of 5X MOPS buffer, 3.5 ul of formaldehyde and 10 ul of formamide [5X MOPS buffer: 300mM sodium acetate, 10mM MOPS (3-{N-morpholino propane sulphonic acid}, 0.5 mM ethylene diamine tetra-acetic acid (EDTA)] and incubated for 15 minutes at 65 degree C. RNA was loaded onto 1.2% formaldehyde agarose-gel after adding 2µl of formaldehyde- gel loading buffer [50% glycerol, ImM EDTA (pH 8.0), 0.25% bromophenol blue, 0.25% xylene cyanol FF] and electrophoresed at 72 volts in IX MOPS buffer (60mM sodium acetate, 2mM MOPS and<br>
0.1 mM EDTA),as described by Sambrook, J., Fritsch, E.F., Maniatis, T., (1989) ( in Molecular Cloning: A Laboratory Manual, Cold Spring harbor Laboratory Press, Plainview, NY). Using 100 mg of leaves 153 µg RNA was obtained (Table 2). Figure 2 shows that the isolated RNA was intact as it revealed intact bands.<br>
Table 2 Yield of RNA from different tissues<br><br>
(Table Removed)<br>
Comparative data of yield of isolated RNA using present protocol and other protocol mentioned in example 2 is given in table 1.<br>
EXAMPLE 3<br>
The amount of solution II added to the homogenate was varied to study its effect on RNA quantity and quality. Arabidopsis thaliana leaves were used as starting material and the protocol was followed essentially as described except that 200ul, 400uA, 600ul and 800ul of solution II was added to the homogenate. It is evident from Table 2 and Figure 3 that 800ul of solution II yields the best results.<br>
Table3 Effect of different volume of solution II on RNA yield.<br><br>
(Table Removed)<br>
EXAMPLE 4<br>
Developed solution was used to isolate RNA from diverse plant species. The samples used were Solanum tuberosum tubers (carbohydrate rich) (Figure 4-A), Capsicum annum fruits (Figure 4-B), Rheum leaves (recalcitrant to guanidine containing solutions) (Figure 3-C), Picrorhiza leaves (Figure 4-D) and roots (Figure 4-E), Arabidopsis leaves and roots (Figure 2), and Camellia sinensis (Tea) vegetative buds and third leaf (rich in ployphenols) (Figure 5). The procedures used for isolation, quantity and quality assessment of RNA are the same as described in example 2. Depending upon the tissue used, the yield can be as high as 215 µg/100 mg of tissue. Table 1 describes the yield obtained from different tissues.<br>
EXAMPLE 5<br>
To assess the suitability of RNA for downstream applications, a reverse transcription (RT) polymerase chain reaction (PCR) was performed using the RNA from Arabidopsis roots and shoots, and Rheum leaves. Total RNA (2 µg) was digested with DNase I (Cat. No. 18068-015 Invitrogen, U.S.A.), reverse-transcribed with superscript II reverse transcriptase (Cat No. 18064-022, Invitrogen, U.S.A.) to synthesize cDNA using oligo dT primer (Cat. No. 18418-012, Invitrogen, U.S.A.). In both the above reactions, manufacturer's instructions were followed. For PCR amplification, oligonucleotide primers specific to beta actin were used to amplify this DNA during 35 cycles, where a<br>
(Table Removed)<br>
cycle was defined as 94 degree C for30 seconds, 52 degree C for 1 minute, and 72 degree for 1 minute. The composition of reaction mixture was as follows:<br><br>
The amplified DNA was electrophoresed throµgh 1.5% agarose gel. A band of 500 base pairs was amplified amplification from cDNA of Arabidopsis (Figure 6) and Rheum (Figure 7).<br>
Using cDNA synthesized from Arabidopsis and Rheum RNA another PCR was carried out with oligonucleotide primers specific for transcription factor, wrky. The cycling parameters for wrky primers to amplify DNA for 35 cycles were 94 degree C for 30 seconds, 48 degree C for 45 seconds and 72 degree for 1 minute. The amplified products were visualized on 1.5% agarose gel. The amplification products of 500 base pairs (Figure No. 8) and 700 base pairs (Figure No. 9) were obtained in Arabidopsis and Rheum, respectively. Sequencing confirmed them to be the wrky fragments. Complementary DNA was also synthesized using RNA isolated from tea buds and third leaf essentially as described elsewhere. For PCR amplification, oligonucleotide primers specific for dihydroflavonol-4- reductase (DFR) were used for amplification during 35 cycles, where cycling parameters were defined as 94 degree C for 30 sec, 52 degree C for 40 sec, and 72 degree for 2 minutes. The amplified DNA was electrophoresed throµgh 1.5% agarose gel at 80 volts for 2 hours. As evident from Figure 10, a band of<br>
950 base-pair was amplified and further sequencing confirmed it to be the fragment of<br>
DFR.<br>
Thus, the experiments explicitly showed that isolated RNA is suitable for RT-PCR<br>
analysis, a down-stream application.<br>
EXAMPLE 6<br>
To further assess the suitability of RNA for down-stream applications, northern blotting was performed using RNA from tea bud and third leaf using the 950 base pairs amplification product as obtained in example 5 as a probe. After visualizing the amplified products on 1.5% agarose gel. The band was excised from the gel and DNA was eluted using QIAEX II Gel Extraction Kit from M/s Qiagen, Germany, following the manufacturer's instructions.<br>
Purified fragment was radiolabeled with a- [32P] dATP (4000 Ci/mmole) using HotPrime Kit (M/s GenHunter Corporation, Nashville, U.S.A.). Unincorporated radionucleotides were removed using QIAquick Nucleotide Removal Kit (QIAGEN, Germany). Radio-labelled probe was used for hybridization experiments.<br>
For blotting, 15 µg of RNA was elecrophoresed on 1.0% formaldehyde agarose gel essentially as described in example 2. Following electrophoresis, the gel was washed twice with DEPC treated autoclaved water for 15 minutes each with shaking. Gel was then washed twice with 10 X SSPE (10X SSPE: 1.5 M sodium chloride, 115 mM NaH2PC4, 10mM EDTA) for 20 minutes each, with shaking. Nylon membrane was wetted in DEPC water and then soaked in 10 X SSPE for 5 minutes with gentle shaking. RNA was then vacuum-blotted (Using pressure of 40 mbar) onto nylon membrane using DEPC treated 10X SSPE as transfer medium. Transfer was carried out for 4 hours. Pressure was increased to 70 mbar for 15 minutes before removing the gel from the vacuum blotter. The location of RNA marker was marked on the nylon surface under a UV light source. Membrane was dried and baked at 80 degree C for two hours. After a brief rinse in 5X SSPE membrane was soaked into prehybridization solution (50% formamide, 0.75 M NaCl, 50mM sodium phosphate, pH 7.4, 0.5 mM EDTA, 0.1 % Ficoll-400, 0.1% bovine serum albumin, 0.1% polyvinylpyrollidone, 0.1% SDS solution and 150 µg of freshly boiled salmon sperm DNA) for five hours.<br>
Radiolabeled probe was denatured by boiling for 10 minutes followed by addition to the prehybridization solution. Hybridization was carried out for 16 hours. Solution was removed and the membrane was washed twice with IX SSC (20X SSC; 3M sodium acetate and 0.3M sodium citrate dihydrate, pH 7.0) containing 0.1% SDS at room temperature for 15 minutes each. Final washing was done at 60 degree C (Using pre-warmed 0.25X SSC containing 0.1%) SDS for 15 minutes. Membrane was wrapped in saran wrap and exposed to X-ray film for 48 hours.<br>
Figure 11 shows that probe did give signal. This result further confirmed that isolated RNA is suitable for down stream applications.<br>
Advantages:<br>
The main advantages of the present invention are:<br>
1.	Present invention provides good quality of RNA.<br>
2.	Good quality and quantity of RNA can be isolated using present protocol.<br>
3.	This method can be applied for various tissue system.<br>
4.	Present invention provides cost effective and rapid method for RNA isolation.<br>
5.	The invention is suitable for tissues recalcitrant to guanidine salts<br><br><br><br><br><br><br><br><br><br><br>
We Claim:<br>
1.	A RNA isolation system comprising of solution I and solution II, wherein the said solution I comprises of saturated phenol of pH 6.0-6.8, anionic detergent, acetate salt and a chelator wherein the ratio of the individual ingredient is in the range of 10: 0.003: 0.02: 1.0 to 10: 0.03: 0.08: 2.0 respectively and solution II comprises of diethylpyrocarbonate in deionised water having a conductivity of 17-18.2 mega-ohms.<br>
2.	A solution as claimed in claim 1, wherein the phenol is saturated with the Tris-(hydroxymethyl)-aminomethane (Tris) buffer.<br>
3.	A solution as claimed in claim 1, wherein the anionic detergent used is selected from the group of a dodecyl sulphate salt or N-lauroyl sarcosine.<br>
4.	A solution as claimed in claim 1, wherein the acetate salt used is selected from the group of sodium or potassium.<br>
5.	A solution as claimed in claim 1, wherein the chelating agent used is selected from the group of disodium and dipotassium of ethylene diamine tetraacetic acid.<br>
6.	A solution as claimed in claim 1, wherein the diethylpyrocarbonate used is in a concentration of about 0.1% volume /volume.<br>
7.	A method for rapid isolation of RNA, wherein the said method comprising the steps of:<br><br>
a)	grinding the plant tissue in liquid nitrogen to make it fine powder;<br>
b)	adding solution I in the powdered sample obtained from step (a) in the ratio ranging from 1:10 to 1:50 followed by homogenizing to make it a fine powder;<br>
c)	adding solution II in powdered sample obtained from step (b); wherein the concentration of solution I to solution II used is in the ratio ranging from 5:3 to 5:2;<br>
d)	adding chloroform to the above said solution obtained from step (c) followed by vortexing and kept it at room temperature for 10 min to separate the layer;<br>
e)	transferring upper layer obtained from step (d) into fresh tube;<br>
f)	adding isopropanol into above layer obtained from step (e) in the ratio of 5:3 to 5:4 followed by vortexing and kept it at room temperature for 10 min;<br>
g)	centrifuging the solution obtained from step (f) for 5 to 10 min at about 4°C to get the desired RNA pellet;<br>
h)  washing the resultant RNA pellets obtained from step (g) with 70% ethanol<br>
followed by air drying; i)   dissolving the washed pellet obtained from step (h) in appropriate amount of<br>
DEPC treated water.<br><br>
8.	A method as claimed in claim 7, wherein the said tissue is selected from the group<br>
consisting of tubers of Solarium tuberosum, fruits of Capsicum annum, Rheum leaves,<br>
leaves and roots of Picrorhiza kurroa, leaves and roots of Arabidopsis thaliana.<br>
9.	Rapid RNA isolation kit comprising:<br>
a)	solution I as claimed in claim 1 ;<br>
b)	solution II as claimed in claim 1 ;<br>
c)	instructions for using the solutions.<br>
10.	A method for rapid isolation of RNA and a kit thereof substantially as herein described with reference to the examples and drawings accompanying this specification.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="259561-method-for-manufacturing-an-electronic-module.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259563-multimedia-optical-disk-reproduction-apparatus-and-method-for-achieving-variable-scene-development-based-on-interactive-control.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259562</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>885/DEL/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC &amp; INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ANUSANDHAN BHAWAN, RAFI MARG, NEW DELHI-110 001, INDIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KASHMIR SINGH</td>
											<td>INSTITUTE OF HIMALAYAN BIORESOURCE TECHNOLOGY, PALAMPUR, INDIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JYOTI RAIZADA</td>
											<td>INSTITUTE OF HIMALAYAN BIORESOURCE TECHNOLOGY, PALAMPUR, INDIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ARTI RANI</td>
											<td>INSTITUTE OF HIMALAYAN BIORESOURCE TECHNOLOGY, PALAMPUR, INDIA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>PRADEEP KUMAR BHARDWAJ</td>
											<td>INSTITUTE OF HIMALAYAN BIORESOURCE TECHNOLOGY, PALAMPUR, INDIA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SANJAY KUMAR</td>
											<td>INSTITUTE OF HIMALAYAN BIORESOURCE TECHNOLOGY, PALAMPUR, INDIA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>SANJAY GHWANA</td>
											<td>INSTITUTE OF HIMALAYAN BIORESOURCE TECHNOLOGY, PALAMPUR, INDIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259562-a-method-for-rapid-isolation-of-rna-and-a-kit-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:27:34 GMT -->
</html>
