<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259627-modified-microorganisms-with-inactivated-lactate-dehydrogenase-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:08:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259627:&quot;MODIFIED MICROORGANISMS WITH INACTIVATED LACTATE DEHYDROGENASE GENE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;MODIFIED MICROORGANISMS WITH INACTIVATED LACTATE DEHYDROGENASE GENE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Modified microorganisms are prepared by inactivation of the endogenous lactate dehydrogenase gene. The microorganisms are deposited under NCIMB Accession Nos. 41277, 41278, 41280 and 41281</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MODIFIED MICROORGANISMS WITH INACTIVATED LACTATE<br>
DEHYDROGENASE GENE<br>
Field of the Invention<br>
This invention relates to the identification of microorganisms which may be adapted for the production of ethanol as a product of bacterial fermentation.   In particular, the invention relates to ethanol production by thermophilic bacteria. Background to the Invention<br>
Bacterial metabolism can occur through various different mechanisms depending on the bacterial species and environmental conditions. Hetrotrophic bacteria, which include all pathogens, obtain energy from oxidation of organic compounds, with carbohydrates (particularly glucose), lipids and protein being the most commonly oxidised compounds. Biologic oxidation of these organic compounds by bacteria results in synthesis of ATP as the chemical energy source. The process also permits generation of more simple organic compounds (precursor molecules), which are required by the bacterial cell for biosynthetic reactions. The general process by which bacteria metabolise suitable substrates is glycolysis, which is a sequence of reactions that converts glucose into pyruvate with the generation of ATP. The fate of pyruvate in the generation of metabolic energy varies depending on the microorganism and the environmental conditions. There are three principle reactions of pyruvate.<br>
First, under aerobic conditions, many micro-organisms will generate energy using the citric acid cycle and the conversion of pyruvate into acetyl coenzyme A, catalysed by pyruvate dehydrogenase (PDH).<br>
Second, under anaerobic conditions, certain ethanologenic organisms can carry out alcoholic fermentation by the decarboxylation of pyruvate into acetaldehyde, catalysed by pyruvate decarboxylase (PDC) and the subsequent reduction of acetaldehyde into ethanol by NADH, catalysed by alcohol dehydrogenase (ADH).<br>
A third process is the conversion of pyruvate into lactate which occurs through catalysis by lactate dehydrogenase (LDH).<br>
There has been much interest in using micro-organisms for the production of ethanol using either micro-organisms that undergo anaerobic fermentation naturally or through the use of recombinant micro-organisms which incorporate the pyruvate decarboxylase and alcohol dehydrogenase genes. Although there has been some success in producing ethanol by using these micro-organisms, fermentation is often compromised by the increased concentration of the ethanoi, especially where the micro-organism has a low level of ethanol tolerance.<br>
Thermophilic bacteria have been proposed for ethanol production, and their use has the advantage that fermentation can be carried out at elevated temperatures which allows the ethanol produced to be removed as vapour at temperatures above 50Â°C; this also permits fermentation to be carried out using high sugar concentrations. However, finding suitable thermophilic bacteria which can produce ethanol efficiently is problematic.<br>
WO01/49865 discloses a Gram-positive bacterium which has been transformed with a heterologous gene encoding pyruvate decarboxylase and which has native alcohol dehydrogenase function, for the production of ethanol. The bacterium is a thermophilic Bacillus and the bacterium may be modified by the inactivation of the lactate dehydrogenase gene using transposon insertion. The bacteria disclosed in WO01/49865 are all derived from Bacillus Strain LLD-R, a sporulation deficient strain that arose spontaneously from culture, and in which the Idh gene has been inactivated by spontaneous mutation or by chemical mutagenesis. Strains LN and TN are disclosed as improved derivatives of strain LLD-R. However, all strains contain a Hae III type restriction systems that impedes plasmid transformation and therefore prevents the transformation within un-methylated DNA.<br>
WO01785966 discloses microorganisms that are prepared by in vivo methylation to overcome the restriction problems. This requires transformation with Hae III methyltransferase from Haemophilus aegyptius into strains LLD-R, LN and TN. However, strains LLD-R, LN and TN are unstable mutants and spontaneously revert to lactate-producing wild-type strains, particularly at low pH and in high sugar concentrations. This results in fermentation product changes from ethanol to lactate, making the strains unsuitable for ethanol production.<br>
WO02/29030 discloses that strain LLD-R and its derivatives include a naturally-occurring insertion element (IE) in the coding region of the Idh gene. Transposition of this into (and out of) the Idh gene and subsequent gene inactivation is unstable, resulting in reversion. The proposed solution to this was to integrate plasmid DNA into the IE sequence.<br>
Therefore, in the art, the production of microorganisms for ethanol production relies on modifying laboratory-produced chemically mutated Bacillus microorganisms, treating these with in vivo methylation procedures and further modifying the microorganisms to integrate plasmid DNA into the IE sequence. The procedure is complex, uncertain and there are also regulatory issues on how the strains can be used.<br>
There   is  therefore   a   need  for  improved  microorganisms  for  ethanol production. Summary of the Invention<br>
According to a first aspect of the present invention, a thermophilic microorganism designated herein under any of NCIMB numbers 41277, 41278, 41279, 41280 and 41281 is modified to permit the increased production of ethanol, the modification being the inactivation of the lactate dehydrogenase gene of a wild-type thermophilic microorganism.<br>
According to a second aspect of the present invention, a method for the production of ethanol comprises culturing a microorganism according to the definition provided above under suitable conditions in the presence of a C5 or C6 sugar.<br>
According to a third aspect of the present invention, a method for the modification of a microorganism to increase the production of ethanol comprises obtaining a thermophilic microorganism defined as any of NCIMB Accession Nos. 41277, 41278, 41279, 41280 and 41281 and deleting the lactate dehydrogenase gene from the microoganism. Description of the Drawings<br>
The present invention is described with reference to the accompanying drawing, wherein;<br>
Figure 1 is a graphic illustration of the plasmid maps for puB190 and puB190-Idh. Description of the Invention<br>
The present invention is based on the identification of wild-type microorganisms with desirable ethanol-producing properties and the modification of the wild-type thermophilic microorganism to disrupt the expression of the lactate dehydrogenase gene.<br>
Inactivating the lactate dehydrogenase gene helps to prevent the breakdown of pyruvate into lactate, and therefore promotes (under appropriate conditions) the breakdown of pyruvate into ethanol using pyruvate decarboxylase and alcohol dehydrogenase, or equivalent enzymatic routes.<br>
The wild-type microorganism of the invention may be any of the thermophilic microorganisms deposited under NCIMB number 41277, 41278, 41279, 41280 and 41281.<br>
The microorganism selected for modification is said to be "wild-type", i.e. it is not a laboratory-produced mutant. The microorganism was isolated from environmental samples which contained thermophiles. The isolated wild-type<br>
microorganism was chosen due to its surprising ability to produce ethanol. However, unmodified, lactate is likely to be the major fermentation product. The isolate was also selected for the ability to grow on hexose and/or pentose sugars at thermophilic temperatures.<br>
The microorganism of the invention has certain desirable characteristics which permit the microorganism to be used in a fermentation process. The microorganism has no restriction system, thereby avoiding the need for in vivo methylation. In addition, the microorganism is stable to at least 3% ethanol and has the ability to utilise C5 and C6 sugars as a substrate, including cellubiose and starch. The microorganism is also transformable at a high frequency. Furthermore, the microorganism has a growth rate in continuous culture of above 0.3 hours"1.<br>
The microorganism is a thermophile and grows in the temperature range of 40Â°C - 85Â°C. Preferably, the microorganism grows within the temperature range 50Â°C-70Â°C. In addition, the microorganism grows in conditions of pH 6.5 or below, in particular pH6.5-pH4.5.<br>
The nucleic acid sequence for lactate dehydrogenase is now known. Using this sequence, it is possible for the skilled person to target the lactate dehydrogenase gene to achieve inactivation of the gene through different mechanisms. It is preferred if the lactate dehydrogenase gene is inactivated either by the insertion of a transposon, or, preferably, by the deletion of the gene sequence or a portion of the gene sequence. Deletion is preferred, as this avoids the difficulty of reactivation of the gene sequence which is often experienced when transposon inactivation is used. In a preferred embodiment, the lactate dehydrogenase gene is inactivated by the integration of a temperature-sensitive plasmid (plasmid pl)B190-ldh; deposited as NCIMB Accession No. 41276), which achieves natural homologous recombination or integration between the plasmid and the microorganism's chromosome. Chromosomal integrants can be selected for on the basis of their resistance to an antibacterial agent (kanamycin). The integration into the lactate dehydrogenase gene may occur by a single cross-over recombination event or by a double (or more) cross-over recombination event. A double cross-over event is preferred. The resulting mutant is stable without antibiotic selection.<br>
Micro-organisms to be used in the invention have been deposited as NCIMB 41277, 41278, 41279, 41280 and 41281, NCIMB Ltd, Ferguson Building, Craibstone Estate, Bucksburn, Aberdeen, AB21 9YA, Scotland.<br>
The microorganisms of the invention may be cultured under conventional culture conditions, depending on the thermophilic microorganism chosen. The choice<br>
of substrates, temperature, pH and other growth conditions can be selected based on known culture requirements, for example see WO01/49865 and WO01/85966.<br>
The present invention will now be described, by way of example only, with reference  to   the   accompanying  drawings,   in  the  following   examples.     The microoganism used to demonstrate the modifications to the Idh gene is Geobacillus 11955, although the methods disclosed herein are suitable for use with any of the microorganisms defined herein, as shown in Example 5. Inactivation of the LDH gene Example 1 - Single crossover mutation Development of the LDH knockout vector<br>
A partial Idh gene fragment of approx 800 bp was subcloned into the temperature-sensitive delivery vector pUB190 using H/ndlll and Xbal resulting in a 7.7 kb plasmid pUB190-ldh (Figure 1 and SEQ ID NO.1). Ten putative Â£ coli JM109 (pUB190-ldh) transformants were verified by restriction analysis and two cultures used to produce plasmid DNA for transformation purposes. Digestion of pUB190-ldh with Hind]\\ and Xbal releases the expected Idh fragment. Transformation of Geobacillus thermoglucosidasius 11955 with pUB190-ldh<br>
Transformants were obtained with all three plasmids tested after 24-48 hrs at 54Â°C with kanamycin selection. The Idh transformants were purified from Geobacillus thermoglucosidasius (Gt) 11955 and verified by restriction analysis using H/ndlll and Xbal. LDH Gene Knockout<br>
Gene knockout was performed by integration of a temperature-sensitive plasmid into the Idh gene on the chromosome.<br>
Plasmid pUB190-ldh replicates at 54Â°C in Gt11955 but not at 65Â°C. Selection was maintained with kanamycin (kan) (12ug/ml). The growth temperature was then increased to 65Â°C (the plasmid is no longer replicative). Natural recombination or integration occurs between the plasmid and chromosome. Chromosomal integrants were selected for by their resistance to kanamycin. Integration was directed towards the Idh gene since an homologous sequence resides on the plasmid. Targeted integration into the Idh gene occurred by a process known as single cross-over recombination. The plasmid becomes incorporated into the Idh gene resulting in an inactive Idh gene. Tandem repeats may occur if several copies of the plasmid integrate. Methodology and Results<br>
Two different methods were attempted for integration:<br>
Method 1: 4 x 50rnl TCP (kan) cultures were grown at 54Â°C for 12-18 hours. The cells were pelleted by centrifugation and resuspended in 1ml of TOP. The resuspension was plated (5 x 200ml) on TGP (kan) plates and incubated overnight at 68Â°C. Integrants were picked and plated onto a 50-square grid on fresh TGP (kan) plates and incubated o/n at 68Â°C.<br>
Method 2: 1 x 50ml TGP (km) cultures was grown at 54Â°C for 12-18 hours. 1 ml of the culture was used to inoculate 50ml of fresh TGP (kan) cultures which was grown at 68Â°C for 12-18 hours. This was sub-cultured the following day into 50ml of fresh TGP (kan) cultures and grown at 68Â°C for another 12-18 hours. The culture was plated out on TGP (km) plates and incubated at 68Â°C overnight. Confluent growth was obtained on the plates. Single colonies were plated onto a 50-square grid on fresh TGP (kan) plates and incubated overnight at 68Â°C. Screening<br>
The putative integrants were screened for Idh gene knockout using the following:<br>
1)	A plate screen<br>
Replica plating of several hundred integrants onto SAM2 plates (with kan) at 68Â°C. Lactate negative cells produce less acid and may have a growth advantage over the wild type on fermentative media without buffers.<br>
2)	A PCR Screen<br>
Colony PCR was used to determine whether the plasmid has integrated into the Idh gene. By choosing primers that flank the integration site, it was possible to determine whether Idh gene integration had occurred (no PCR fragment was amplified for inserts).<br>
3)	Lactate Assay<br>
This assay determines whether the integrants produce lactate when grown overnight in SAM2 (kan) at 68Â°C. The culture supernatant was tested for the concentration of lactate with the Sigma lactate reagent for lactae determination. Lactate negative integrants were further characterised by PCR and evaluated in a fermenter for stability. Electroporation protocol for Geobacillus thermQglucosidasiusNClMB 11955<br>
A frozen stock of NCIMB 11955 was made by growing an overnight culture in TGP medium (250rpm at 55Â°C, 50ml volume in a 250ml conical flask, OD60o), adding an equal volume of 20% glycerol and dividing into 1ml aliqouts and storing in cryotubes at -80Â°C. 1ml of this stock was used to inoculate 50ml of TGP in a 250m! conical flask, incubated at 55Â°C, 250rpm, until the culture reaches OD6001.4.<br>
The flask was cooled on ice for 10 minutes, then the culture centrifuged for 20 minutes at 4000rpm at 4Â°C. The pellet was resuspended in 50ml ice-cold electroporation medium and centrifuged at 4,000rpm for 20 minutes. Three further washes were carried out, 1 x 25ml and 2 x 10ml, and then the pellet resuspended in 1.5ml ice-cold electroporation medium and divided into 60jj| aliquots.<br>
For the electroporation, 1-2jal of DNA was added to 60p.l of electrocompetent cells in an eppendorf tube kept on ice, and gently mixed. This suspension was transferred to a pre-cooled electroporation cuvette (1mm gap) and electroporated at 2500V, 10jaF capacitance and 600Q resistance.<br>
Immediately after the pulse, 1ml TCP was added, mixed, and the suspension transferred to a screw top tube and incubated at 52Â°C for 1 hour in a shaking waterbath. After incubation the suspension was either plated directly (eg 2 x 0.5ml) or centrifuged at 4,000rpm for 20 minutes, resuspended in 200|o.l-500(il TCP, and plated on TGP agar containing the appropriate antibiotic.<br>
(Table Removed)<br>
Inactivation of the LDH gene. Double cross-over mutation<br>
Primers were designed based on the available 11955 LDH sequence. The knock-out strategy was based on the generation of internal deletions within the LDH gene by two approaches.<br>
In strategy 1, two existing unique restriction sites near the middle of the LDH coding sequence were exploited to generate a deletion. A single large per product was generated from genomic DNA covering most of the available LDH sequence, and cloned into the Smal site in the multiple cloning site of pUC19. The pUC19 clone was then digested sequentially with SsfEII and BsrGl and religated after Klenow<br>
digestion, to generate an internal deletion in the LDH gene between BstEII and BsrGI.<br>
In strategy 2 (see Example 2) the LDH gene was cloned as 2 PCR products, introducing Not\ sites on the oligonucleotide primers to allow the 2 PCR products to be ligated together in pUC19, with the generation of a deletion in the middle of the LDH sequence.<br>
The two LDH genes with the internal deletions were subcloned into 3 potential delivery systems for Geobacillus.<br><br>
(Table Removed)<br>
The delivery vectors were transformed into 11955 by electroporation. Genetic    strategy    Information:    Development    of    delivery    systems    for homologous recombination.<br>
To generate knockouts, an efficient system is required to deliver a mutated gene into the target organism and select for integration into the genome by homologous recombination with the target "wild-type" gene. In principle, this could be achieved by introducing the DMA on an E.coli vector without a Gram positive replicon but which carries a Gram negative selectable marker. This requires a high transformation efficiency. The electroporation method developed for Geobacillus 11955 generates 3x104 transformants per ug of DNA with pNW33N. The Gram positive replicon is derived from pBC1 in the BGSC catalogue, and from pTHT15 in the sequence database.<br>
The cat gene on pNW33N is used for selection in both E.coli and Geobacillus. Temperature-sensitive mutants  of pNW33N were generated  by passaging the plasmid through the Statagene XL1r ed mutator strain. Example 2 Generation of an LDH mutant by gene replacement<br>
A further strategy was designed to generate a stable mutation of the LDH gene in Geobacillus thermoglucosidasius NCIMB 11955 by gene replacement. This<br><br>
strategy involved the generation of a 42bp deletion close to the middle of the coding sequence, and the insertion at this position of 7bp introducing a novel Notl restriction site. This inserted sequence was intended to cause a frame-shift mutation downstream.<br>
The strategy involved generating the deletion using 2 PCR fragments using oligo primers introducing the novel Not\ site. Primers were designed based on the partial sequence of the LDH coding region from 11955. The sequence of the primers used is shown below. Fragment 1:<br>
Primer 1 (forward); GGAATTCCCTTATGAACCAAGGAATAGCA (underlined sequence indicates bases introduced to generate a novel Hco/3! sitel; SEQ ID NO. 3)<br>
Primer     2      (reverse);      GCGGCCGCACCCGCTCTTTCGGTAACCCGCT (underlined sequence indicates bases introduced to generate a novel Not\ site; SEQ ID NO. 4). Fragment 2:<br>
Primer 3 (forward); GCGGCCGCTTGCTAAGTGAATATTTTCAAGT (underlined sequence indicates bases introduced to generate a novel Nofi site; SEQ ID NO. 5).<br>
Primer 4 (reverse); CTGCAGCGTCAATTCCATCACTTCACGA (underlined sequence indicates bases introduced to generate a novel Pst\ site; SEQ ID NO. 6). Preparation of genomlc DNA.<br>
Genomic DNA was prepared from 11955 to serve as template for PCR. Cells from a 20ml overnight culture of 11955 grown in TGP medium at 52Â°C were collected by centrifugation at 4,000 rpm for 20mins. The cell pellet was resuspended in 5ml of STE buffer 0.3M sucrose, 25mM Tris HCI and 25mM EDTA, adjusted to pH 8.0 containing 2.5mg of lysozyme and 50ul of 1mg/ml ribonuclease A. This was incubated for 1 hour at 30Â°C, then 5mg of proteinase K was added and 50ul of 10% SDS followed by incubation at 37Â°C for 1 hour. The lysed culture was then extracted sequentially with equal volumes of phenol/chloroform followed by chloroform before precipitation with isopropanol. After washing twice with ice-cold 70% ethanol, the DNA pellet was redissolved in 0.5ml TE buffer. Generation of LDH deletion construct.<br>
PCR was carried out using a Robocycler Gradient 96 (Stratagene) and the reaction conditions were as follows: Cycle 1; denaturation at 95Â°C for 5 min, annealing at 47Â°C for 1 min, extension at 72Â°C for 2 min, Cycle2-30; denaturation at 95Â°C for 1 min, annealing at 47Â°C for 1 min, extension at 72Â°C for 2 min, and a further<br>
incubation at 72Â°C for 5 min. The enzymes used were an equal mixture of Pfu polymerase (Promega) and Taq polymerase ( New England Biolabs, NEB ). The buffer and dNTPs composition and concentration used was that recommended for Pfu by the suppliers. The PCR products obtained using genomic DNA from NCIMB 11955 as template were purified by agarose gel electrophoresis and eluted from the agarose gel by using the QIAquick Gel Extraction Kit (Qiagen). The purified PCR products were ligated to pUC19 (New England Biolabs) previously digested with Smal and the ligation mixture was used to transform Escherichia coli DH10B (Invitrogen). Ampicillin resistant colonies were selected and the contained plasmids were isolated and characterised by restriction analysis, and the orientation of the inserts was established.<br>
A plasmid (pTM002) with fragment 2 inserted into pUC19 (with the novel Psfl site introduced on primer 4 closest to the Psfl site in the multiple cloning site (mcs) of pUC19) was digested with Not\ and Psfl. The resulting fragment (approximately 0.4kb) was ligated into a pUC19 plasmid (pTMOOl) bearing fragment 1 (with the novel EcoRI site introduced on oligo 1 closest to the EcoR\ site in the mcs of pUC19) digested with Not\ and Psfl to linearise the plasmid. The ligation mixture was used to transform E.coli DH10B. Ampicillin-resistant colonies were selected and the contained plasmids were isolated and characterised by "restriction analysis. A plasmid (pTMOOS) with the expected restriction pattern for the desired construct (the LDH coding region carrying the deletion and introduced A/ofl site) was identified and verified by sequencing using M13mp18 reverse and forward primers.<br>
The mutated LDH gene was excised from pTM003 by digestion with H/ncflll and EcoR\ and purified by agarose gel electrophoresis followed by elution from the agarose gel using the QIAquick Gel Extraction Kit (as an approximately 0.8kb fragment). This fragment was treated with Klenow polymerase (NEB, according to manufacturers instructions) to generate blunt ends and introduced into the pUB190 vector. This was achieved by blunt-end ligation with pUB190 linearised by digestion with Xbal and then Klenow-treated followed by gel-purification as before. The ligation mixture was used to transform E.coli SCS110 (Stratagene). Ampicillin resistant colonies were selected and the contained plasmids were isolated and characterised by restriction analysis. A plasmid (pTM014) with the expected restriction pattern for the desired construct was identified and used to transform NCIMB 11955 by electroporation using the electroporation protocol as described in Example 1. Generation and characterization of a gene-replacement LDH mutant by double-crossover.<br>
A presumptive primary integrant of pTM014 obtained in this fashion (strain TM15) was used to obtain double recombinants (gene replacement). This was achieved by serial sub-culture of TM15 in TOP medium without kanamycin. Five successive shaken cultures were used, alternating between 8 hours at 54Â°C and 16 hours at 52Â°C, using 5ml TOP in 50ml tubes (Falcon) at 250rpm, 1% transfer at each stage. After these 5 passages, the resulting culture was serially diluted in TGP and 100ul samples plated on TGP agar plates for incubation at 54Â°C. Replica-plating of the resultant colonies onto TGP agar containing 12yg/ml kanamycin was used to identify kanamycin-sensitive colonies. After streaking to single colonies on agar to purify, these kanamycin sensitive derivatives were tested for lactate production, and as expected, proved a mixture of LDH+ and LDH". One LDH" derivative, TM89, was further characterized by PCR and Southern blots.<br>
Genomic DNA was prepared from TM15 (primary integrant) and TM89 (presumptive double recombinant LDH"), and used as template for PCR using primers 1 and 4, using the conditions described above. Genomic DNA from 11955 was used as control. The PCR products (approx. 0.8 kb bands were obtained from all 3 templates) were purified by agarose gel electrophoresis and eluted from the agarose gel using the QIAquick Gel Extraction Kit. Samples were digested with Not\ and run on a 0.7% agarose gel to visualize products. The PCR product of 11955 showed no evidence of Not\ digestion, as expected, whereas the PCR product of TM89 gave 2 bands of around 0.4kb, indicating the replacement of the wild-type gene with the mutated allele. Notl digestion of the PCR product of TM15, the primary integrant, gave predominantly the 2 bands seen with TM89, with a trace of the uncut (0.8kb) band. This can be explained by the result obtained with Southern blotting of the TM15 genomic DNA.<br>
Genomic DNA of 11955, TM15 and TM89 was digested with Not\, Pst\ and Not\, and Hind\\\ and A/ofl, and subjected to agarose gel electrophoresis. The DNA was transferred onto a positively-charged nylon membrane (Roche) and hybridized with a DlG-labelled probe generated by PCR of the 11955 LDH gene using primers 1 and 4 with DIG-labeled dNTps, following the suppliers instructions (Roche Molecular Biochemicals DIG application manual). The hybridizing bands were visualized using the detection kit supplied (Roche). The Southern blot showed evidence of a much-amplified band of approx. 7.5kb in the A/ofl digest of TM15, with similarly-amplified bands of approximately 7 and 0.4kb in the H/ncflll/A/ofl and Pst\/Not\ digests of TM15, indicating integration of multiple tandem copies of pTM014 integrated at the LDH locus in this primary integrant. With all 3 restriction digests, TM89 showed evidence<br>
of a different restriction pattern showing an extra hybridizing band compared to<br>
11955, consistent with gene replacement.<br>
Example 3<br>
Ethanol production by the wild-type thermophile<br>
Reproducible growth and product formation was achieved in fed-batch and continuous cultures for the wild-type thermophile.    Tables 1, 2 and 3 show the conditions used in the fermentation process. Table 1<br><br>
(Table 2 Removed)<br>
Table 2<br>
Sulphate Trace Elements Stock Solution<br><br>
Del-    H2O    (final) | 1000 ml  |10 litres<br>
Table 3<br>
(Table 3 Removed)<br>
Fermenter Conditions<br><br>
Table 4<br><br>
(Table 4 Removed)<br>
Example 4<br>
Increasing ethanol production by thermophiles<br>
In order to achieve the target ethanol yields and productivity from the thermophile, lactic acid production was minimised through the knock-out of L-Lactate Dehydrogenase (LDH) activity by inactivation of the Idh gene. There were two approaches taken to inactivate the Idh gene: a single crossover recombination of marker DNA into the Idh region of the chromosome, preventing its transcription, or a double recombination of homologous regions of DNA into the Idh gene to create a mutation within the gene region, rendering it non-functional.<br>
The single cross-over approach rapidly generated  LDH-negative mutants which showed an increase in ethanol production when compared to the wild-type strain. Improvements in ethanol production by the LDH mutants<br>
The LDH-negative mutants were grown in fed-batch cultures, in the established minimal media to measure the increase in ethanol production resulting from the knock-out of LDH activity. The change in the metabolite profiles of the LDH mutants are shown in Table 5, compared to the optimum ethanol production from the wild-type strain. Table 6 shows the increase of ethanol production caused by the knock-out of LDH activity.<br>
Table 5<br><br>
(Table 5 Removed)<br>
Table 6<br><br>
(Table 6 Removed)<br>
The LDH mutants produced significantly higher yields of ethanol than the wild-type thermophile, demonstrating the successful rerouting of the metabolism of these thermophiles. Further optimisation of culture conditions and media constituents for the LDH mutant strains will result in increased ethanol yields.<br>
Example 5<br>
Using the methods outlined in Example 2, the idh modification was made to isolate NCIMB41277. The same transformation protocol was used, but transformation efficiency was optimised by incubation at 55Â°C in recovery (as<br><br>
opposed to 52Â°C in Example 2).   The serial sub-culturing of the primary integrants was carried out in 2TY media to obtain double cross-overs with the idh-negative phenotype.  A stable knockout strain 41277K04 was characterised in fed-batch and continuous fermentation.   The results are shown in Table 7.   This shows that the mutant 41227KO4 has significantly increased levels of ethanol production compared to the wild-type microorganism.   The mutant is also stable in culture and does not revert back to the wild-type. Table 7 Comparison between NCIMB 41277 and 41277KO4<br><br>
(Table 7 Removed)<br>
The ethanol data for all wild-type isolates are shown in Table 8, demonstrating the ability of the isolates to produce ethanoi in the wild-type form. This is unexpected as most wild-type microorganisms of this type will not produce ethanol.<br>
Table 8<br>
Ethanol data for isolates<br>
(Table 8 Removed)<br><br><br><br><br><br>
We Claim:<br>
1.	A microorganism deposited under NCIMB number 41277, 41278, 41279, 41280 and 41281 modified to inactivate the endogenous lactate dehydrogenase gene.<br>
2.	A microorganism as claimed in claim 1, wherein the lactate dehydrogenase gene has been deleted.<br>
3.	A method for the production of ethanol, comprising culturing a microorganism as claimed in claim 1 or claim 2, under suitable conditions in the presence of a C5 or C6 sugar.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LTkyNTItZGVsbnAtMjAwNy1BYnN0cmFjdC0oMDMtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-9252-delnp-2007-Abstract-(03-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LTkyNTItZGVsbnAtMjAwNy1Gb3JtLTItKDAzLTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-9252-delnp-2007-Form-2-(03-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LTkyNTItZGVsbnAtMjAwNy1Gb3JtLTMtKDAzLTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-9252-delnp-2007-Form-3-(03-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNsYWltcy0oMTAtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Claims-(10-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1ERUxOUC0yMDA3LUNsYWltcy0oMjYtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-DELNP-2007-Claims-(26-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDEtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(01-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(07-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDgtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(08-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(09-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTItMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(12-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTItMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(12-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(14-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(17-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(22-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(25-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-DELNP-2007-Correspondence Others-(26-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence Others-(26-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjctMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-DELNP-2007-Correspondence Others-(27-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence-Others-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDYtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence-Others-(06-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTAtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence-Others-(10-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjEtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence-Others-(21-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMzAtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Correspondence-Others-(30-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUZvcm0tMy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Form-3-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUZvcm0tMy0oMDktMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Form-3-(09-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LUZvcm0tMy0oMzAtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Form-3-(30-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1ERUxOUC0yMDA3LUdQQS0oMjctMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-DELNP-2007-GPA-(27-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMDYtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Petition-137-(06-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMDktMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Petition-137-(09-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI1Mi1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMzAtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9252-delnp-2007-Petition-137-(30-05-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259626-an-oxygen-sensor-readiness-module-and-a-method-of-determining-the-readiness.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259628-system-for-simplifying-the-programmable-memory-to-logic-interface-in-field-programmable-gate-array-fpga.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259627</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>9252/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Nov-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>TMO RENEWABLES LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CRIPPS, ROGER</td>
											<td>TMO RENEWABLES LIMITED, 40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LEAK, DAVID</td>
											<td>TMO RENEWABLES LIMITED, 40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RUDD, BRIAN</td>
											<td>NOVACTA BIOSYSTEMS LIMITED, BIO PARK HERTFORDSHIRE, BROADWATER ROAD, WELWYN GARDEN CITY, HERTS AL7 3AX, U.K.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ATKINSON, ANTHONY</td>
											<td>TMO RENEWABLES LIMITED, 40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>THOMPSON, ANN</td>
											<td>3-25 BYRON AVENUE, TAKAPUNA, AUCKLAND, NEW ZEALAND.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ELEY, KIRSTIN</td>
											<td>TMO RENEWABLES LIMITED, 40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>TAYLOR, MARK</td>
											<td>TMO RENEWABLES LIMITED, 40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>BAKER, SIMON</td>
											<td>TMO RENEWABLES LIMITED, 40 ALAN TURING ROAD, THE SURREY RESEARCH PARK, GUILDFORD, SURREY GU2 7YF, U.K.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2006/002087</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-06-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0511602.5</td>
									<td>2005-06-07</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259627-modified-microorganisms-with-inactivated-lactate-dehydrogenase-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:08:06 GMT -->
</html>
