<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259687-process-for-producing-rose-plant-cell-with-modified-color by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:50:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259687:PROCESS FOR PRODUCING ROSE PLANT CELL WITH MODIFIED COLOR</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PROCESS FOR PRODUCING ROSE PLANT CELL WITH MODIFIED COLOR</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to A process for producing a rose, characterized in that the intrinsic metabolic pathway of rose is artificially suppressed and that a gene coding for flavonoid 3&#x27;,5&#x27;-hydroxylase derived from pansy is expressed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION <br>
METHOD   FOR   PRODUCING  ROSE  WITH ALTERED—PETAL  COLORS -.<br>
Technical Field<br>
The present invention relates to a new method for producing a rose with altered petal colors.  More specifically, it relates to a method for producing a rose by artificially inhibiting the endogenous metabolic pathway of rose, and expressing the gene coding for pansy flavonoid 3',5'-hydroxylase and the gene coding for dihydroflavonol reductase which reduces dihydromyricetin.<br>
Background Art<br>
Flower petals perform the role of attracting pollinators such as insects and birds, which transport plant pollen, and therefore flower colors, shapes, patterns and odors have evolved in tandem with pollinators (Honda, T. et al., Gendai Kagaku, May, 25-32(1998)).  Probably as a result of this, it is rare for a single species of flower to exhibit several different colors, and for example, rose or carnation varieties exhibiting violet to blue colors do not exist, while iris or gentian varieties exhibiting bright red colors do not exist.  Because flower color is the most important aspect of petals for purposes of appreciation as well, flowers of different colors have traditionally been bred by crossbreeding.  The rose, known as the "queen of flowers" and having high commercial value, has also been crossbred throughout the world.<br>
For example, the current yellow rose cultivar was created by crossbreeding of Rosa foetida, originating from western Asia, with a non-yellow rose variety. However, because flower color is determined by the genetic capacity of the plant, there has been a limit to the flower colors that can currently be produced in cross-bred strains whose available genetic sources are<br>
restricted (Tanaka et al. Plant Cell Physiol. 39, 1119-1126, 1998; Mol et al. Curr. Opinion Biotechnol. 10, 198-201 1999).  Among these, the cultivation of blue roses has been thought impossible and has been considered the "holy grail" of colors (Oba, H., "Bara no Tanjo", 1997, Chukoshinsho; Suzuki, M. , "Shokubutsu Bio no Mahou: Aoi Bara mo Yume dewanakunatta", 1990, Kodansha Bluebacks; Saisho, H., "Aoi Bara", 2001, Shogakkan).<br>
Although "blue rose" varieties currently exist, these are actually pale violet roses.  The first improved variety of "blue rose" by crossbreeding is said to have been the light-violet shaded grey-colored "Grey Pearl" created in 1945.  The light-violet pink-colored "Staring Silver" was later created in 1957, and these varieties were crossed to produce several pale violet roses such as "Blue Moon" (1964) and "Madam Violet" (1981).  These pale violet roses and other roses were then utilized in further breeding to create light-grey-colored roses such as "Seiryu" (1992) and "Blue Heaven" (2002), which were hailed as new types of "blue roses".<br>
However, these flower colors are not actually blue but merely greyish-dull pink, and despite many years of breeding efforts, there is still no example of a truly "blue" rose.  In horticultural industry, the group of colors from violet to blue is generally considered "blue" according to the RHSCC (The Royal Horticultural Society Colour Chart).  It is an aim of the present invention to create rose plants having flower colors falling within the "violet group", "violet-blue" group and "blue group" according to the Royal Horticultural Society Colour Chart.<br>
Flower colors derive mainly from the three compound groups of anthocyanins, carotenoids and betalains, but it is the anthocyanins, having the widest absorption wavelength range (from orange to blue), that are responsible for blue color.  Anthocyanins belong to the flavonoid family and are biosynthesized by the metabolic<br>
pathway shown in Fig. 1.  Anthocyanins are normally localized in the vacuoles of epithelial cells.  The color shade of anthocyanins (i.e. flower color) depends largely on the structure of the anthocyanins, with more numerous hydroxyl groups on the B ring resulting in a bluer color. Hydroxylation of the B ring is catalyzed by flavonoid 3'-hydroxylase (F3'H) and flavonoid 3 ', 5'-hydroxylase (F3'5'H).  Absence of F3 ' H and F3'5'H activity leads to synthesis of pelargonidin (orange to red colors), presence of F3'H activity leads to synthesis of cyanidin (red to rouge colors) and presence of F3'5'H activity leads to synthesis of delphinidin (violet color).<br>
These anthocyanidins are modified with sugars and acyl groups to produce an assortment of anthocyanins. Generally speaking, a larger number of modifying aromatic acyl groups correlates to bluer anthocyanins. Anthocyanins also produce quite different colors depending on the vacuole pH and the copresent flavonols and flavones or metal ions (Saito, N., Tanpakushitsu Kakusan Kouso, 47 202-209, 2002; Broullard and Dangles, In the flavonoids: Advances in Research since 1986 (Ed. by Harborne) Capmann and Hall, London pp.565-588; Tanaka et al. Plant Cell Physiol. 39 1119-1126, 1998; Mol et al., Trends in Plant Science 3, 212-217, 1998; Mol et al., Curr. Opinion Biotechnol. 10, 198-201 1999).<br>
Rose flower petal anthocyanins are derivatives of pelargonidin, cyanidin and peonidin, whereas no delphinidin derivatives are known (Biolley and May, J. Experimental Botany, 44, 1725-1734 1993; Mikanagi Y., Saito N., Yokoi M. and Tatsuzawa F. (2000) Biochem. Systematics Ecol. 28:887-902).  This is considered to be the main reason for the lack of blue roses.  Existing roses have been created by crossbreeding of crossable related rose species (R. multiflora, R. chinensis, R. gigantean, R. moschata, R. gallica, R. whichuraiana, R. foetida, etc.).<br>
The fact that no blue rose has been achieved in<br>
spite of repeated efforts at crossbreeding is attributed to the lack of delphinidin production ability by rose-related varieties.  Production of delphinidin in rose petals would require expression of F3'5'H in the petals as mentioned above, but F3'5'H is believed to be non-expressed in the petals of rose and rose-related varieties.  Thus, it is likely impossible to obtain a blue rose by accumulating delphinidin in the petals through crossbreeding.  It is known that trace amounts of the blue pigment rosacyanin are found in rose petals and its chemical structure has been determined (Japanese Unexamined Patent Publication No. 2002-201372), but no reports are known regarding augmentation of rosacyanin to create a blue rose, and no findings have been published on the rosacyanin biosynthesis pathway or the relevant enzymes or genes.<br>
Examples of blue or violet colors produced by biological organisms also include indigo plant-produced indigo (for example, Appl. Microbiol. Biotechnol. Feb. 2003, 60(6}:720-5) and microbially-produced violacein (J. Mol. Microbiol. Biotechnol. Oct. 2000 2(4):513-9; Org. Lett., Vol.3, No.13, 2001, 1981-1984), and their derivation from tryptophan and their biosynthetic pathways have been studied.<br>
Blue pigments based on gardenia fruit-derived iridoid compounds (S. Fujikawa, Y. Fukui, K. Koga, T. Iwashita, H. Komura, K. Nomoto, (1987) Structure of genipocyanin Gl, a spontaneous reaction product between genipin and glycine. Tetrahedron Lett. 28 (40), 4699-700; S. Fujikawa, Y. Fukui, K. Koga, J. Kumada, (1987), Brilliant skyblue pigment formation from gardenia fruits, J. Ferment. Technol. 65 (4), 419-24) and lichen-derived azulenes (Wako Pure Chemical Industries Co., Ltd.) are also known, but no reports are known of expressing these in plant flower petals to produce blue-colored flowers.<br>
It has been expected that a blue rose could be created by transferring the F3'5'H gene expressed by<br>
other plants into rose and expressing it in rose petals (Saisho, H., "Aoi Bara", 2001, Shogakkan).  The F3'5'H gene has been obtained from several plants including petunia, gentian and Eustoma russellianum (Holton et al. Nature 366, 276-279, 1993; Tanaka et al. Plan Cell Physiol. 37, 711-716 1996; W093/18155).  There are also reports of transformed varieties of rose (for example, Firoozababy et al. Bio/Technology 12:883-888 (1994); US 5480789; US 5792927; EP 536,327 Al; US 20010007157 Al).<br>
Actual transfer of the petunia F3'5'H gene into rose has also been reported (WO93/18155, W094/28140) .<br>
However, it has not been possible to obtain a blue rose, and it is believed that obtaining a blue rose will require a modification which alters the metabolism of flower pigments suited for rose.<br>
On the other hand, it has been confirmed that transfer of the F3'5'H gene into red carnation, which produces pelargonidin instead of delphinidin, leads to accumulation of both pelargonidin and delphinidin, but that the flower color is only altered to a slightly purplish red (W094/28140).  This result suggests that it is not possible to obtain a "blue" carnation simply by expression of F3'5'H, and that it is necessary to inhibit the metabolic pathway to endogenous synthesis of pelargonidin by carnation.<br>
In order to avoid competition with the carnation endogenous metabolic pathway (reduction of dihydrokaempferol (DHtf) by dihydroflavonol reductase (DFR)), a variety lacking DFR was selected from among white carnations.  The F3'5'H gene and petunia DFR (which is known to efficiently reduce dihydromyricetin (DHM) without reducing DHK) gene were transferred into carnation.  This resulted in one case of successfully obtaining a recombinant carnation with a delphinidin content of about 100% and a blue-violet flower color previously not found in carnation (Tanpakushitsu Kakusan Kouso, Vol.47, No.3, p228, 2002).  Thus, further<br>
modification was necessary to realize a blue carnation flower, in addition to accumulating delphinidin by expression of the F3'5'H gene.<br>
DFR has already been cloned from several plants (petunia, tobacco, rose, Torenia, snapdragon, transvaal daisy, orchid, barley, corn, etc.) (Meyer et al., Nature 330, 677-678, 1987; Helariutta et al., Plant Mol. Biol. 22, 183-193 1993; Tanaka et al., Plant Cell Physiol. 36, 1023-1031; Johnson et al., Plant J. 19, 81-85, 1999). Substrate specificity of the DFR gene differs depending on the plant variety, and it is known that the petunia, tobacco and orchid DFR genes cannot reduce DHK, whereas the petunia DFR gene most efficiently reduces DHM among the dihydroflavonols (Forkmann et al., Z. Naturforsch. 42c, 1146-1148, 1987; Johnson et al. Plant J. 19, 81-85, 1999).  Nevertheless, no cases have been reported for expression of these DFR genes in rose.<br>
As a means of avoiding competition with the endogenous metabolic pathway or between the enzyme and the exogenous gene-derived enzyme such as F3'5'H, as mentioned above, the gene may be transferred into a variety lacking the gene.  Also, it is known that expression of the target gene can be artificially inhibited by deletion methods involving homologous recombination of the target gene, but because of the low frequency of homologous recombination and the limited number of suitable plant varieties, this has not been implemented in practice (for example, Nat. Biotechnol. 2002, 20:1030-4).<br>
Inhibition methods on the transcription level include the antisense method using antisense RNA transcripts for mRNA of the target gene (van der Krol et al., Nature 333, 866-869, 1988), the sense (cosuppression) method using transcripts of RNA equivalent to mRNA of the target gene (Napoli et al., Plant Cell 2, 279-289, 1990) and a method of using duplex RNA transcripts corresponding to mRNA of the target gene<br>
(RNAi method; Waterhouse et al., Pro. Natl. Acad. Sci. USA 95, 13959-13964, 1998).<br>
Numerous successful examples of these three methods have been published.  For rose, cosuppression of chalcone synthase (CHS) gene which is necessary for synthesis of anthocyanins was reported to successfully alter flower color from red to pink (Gutterson HortScience 30:964-966 1995), but this CHS suppression is incomplete and therefore it has not been possible to totally suppress anthocyanin synthesis to obtain a white flower stock.<br>
Patent document 1: Japanese Unexamined Patent Publication No. 2002-201372<br>
Patent document 2: W093/18155<br>
Patent document 3: USP 5480789<br>
Patent document 4: USP 5792927<br>
Patent document 5: EP 536 327 Al<br>
Patent document 6: US 20010007157 Al<br>
Patent document 7: W094/28140<br>
Non-patent document 1:  Honda T. et al. Gendai Kagaku, May, 25-32(1998)<br>
Non-patent document 2: Tanaka et al. Plant Cell Physiol. 39, 1119-1126, 1998<br>
Non-patent document 3: Mol et al. Curr. Opinion Biotechnol. 10, 198-201 1999<br>
Non-patent document 4: Oba, H., "Bara no Tanjo", 1997, Chukoshinsho<br>
Non-patent document 5: Suzuki, M., "Shokubutsu Bio no Mahou: Aoi Bara mo Yume dewanakunatta", 1990, Kodansha Bluebacks<br>
Non-patent document 6: Saisho, H., "Aoi Bara", 2001, Shogakkan<br>
Non-patent document 7: Saito, N., Tanpakushitsu Kakusan Kouso, 47 202-209, 2002<br>
Non-patent document 8: Broullard et al. In the flavonoids: Advances in Research since 1986 (Ed by Harborne) Capmann and Hall, London pp565-588<br>
Non-patent document 9: Tanaka et al. Plant Cell<br>
Physiol. 39 1119-1126, 1998<br>
Non-patent document 10:	Mol et al, Trends in Plant<br>
Science 3, 212-217 1998<br>
Non-patent document 11:	Mol et al. Curr. Opinion<br>
Biotechnol. 10, 198-201 1999<br>
Non-patent document 12:	Biolley and May, J.<br>
Experimental Botany, 44, 1725-1734 1993<br>
Non-patent document 13:	Mikanagi Y, et al. (2000)<br>
Biochem Systematics Ecol. 28:887-902<br>
Non-patent document 14:	Appl. Microbiol. Biotechnol.<br>
2003 Feb;60(6):720-5<br>
Non-patent document 15:	J. Mol. Microbiol.<br>
Biotechnol. 2000 Oct; 2 (4) :	513-9<br>
Non-patent document 16:	Org. Lett., Vol. 3, No. 13,<br>
2001, 1981-1984<br>
Non-patent document 17:	S. Fujikawa, et al. (1987)<br>
Tetrahedron Lett. 28 (40), 4699-700<br>
Non-patent document 18:	S. Fujikawa, et al. (1987)<br>
J. Ferment. Technol. 65 (4),	419-24<br>
Non-patent document 19:	Holton et al. Nature 366,<br>
276-279, 1993<br>
Non-patent document 20:	Tanaka et al. Plant Cell<br>
Physiol. 37, 711-716 1996<br>
Non-patent document 21:	Firoozababy et al.<br>
Bio/Technology 12:883-888 (1994)<br>
Non-patent document 22:	Tanpakushitsu Kakusan Kouso,<br>
Vol.47, No.3, p228, 2002<br>
Non-patent document 23:	Meyer et al. Nature 330,<br>
677-678, 1987<br>
Non-patent document 24:	Helariutta et al. Plant Mol.<br>
Biol. 22 183-193 1993<br>
Non-patent document 25:	Tanaka et al. Plant Cell<br>
Physiol. 36, 1023-1031<br>
Non-patent document 26:	Johnson et al. Plant J. 19,<br>
81-85, 1999<br>
Non-patent document 27:	Forkmann et al. Z.<br>
Naturforsch. 42c, 1146-1148,	1987<br>
Non-patent document 28: Nat Biotechnol 2002, 20:1030-4<br>
Non-patent document 29: van der Krol et al. Nature 333, 866-869, 1988<br>
Non-patent document 30: Napoli et al. Plant Cell 2, 279-289, 1990<br>
Non-patent document 31: Waterhouse et al. Pro. Natl. Acad. Sci. USA 95, 13959-13964 1998<br>
Non-patent document 32: Gutterson HortScience 30:964-966 1995<br>
Non-patent document 33:  Suzuki, S., "Bara, Hanazufu", Shogakkann, p.256-260, 1990<br>
Disclosure of the Invention<br>
As mentioned above, rose flower colors have been successfully altered by transferring the F3'5'H gene into rose and expressing it in the petals.  In carnation, the F3'5'H gene and petunia DFR gene have been expressed in DFR-deficient varieties to create blue-violet carnations. However, a "blue rose" has not yet been created.  It is therefore an object of the present invention to provide a rose which blossoms with a blue flower.<br>
The invention thus provides (1) a method for producing a rose characterized by artificially suppressing the rose endogenous metabolic pathway and expressing the pansy gene coding for flavonoid 3',5'-hydroxylase.<br>
The invention further provides (2) a method for producing a rose characterized by artificially suppressing the rose endogenous metabolic pathway, and expressing the pansy gene coding for flavonoid 3',5'-hydroxylase and the gene coding for dihydroflavonol reductase.<br>
The invention still further provides (3) a method for producing a rose characterized by artificially suppressing expression of rose endogenous dihydroflavonol reductase, and expressing the pansy gene coding for<br>
flavonoid 3',5'-hydroxylase and the gene coding for dihydroflavonol reductase derived from a plant other than rose.<br>
The invention still further provides (4) a method for producing a rose characterized by artificially suppressing expression of rose endogenous flavonoid 3'-hydroxylase and expressing the pansy gene coding for flavonoid 3',5'-hydroxylase.<br>
The aforementioned pansy gene coding for flavonoid 3',5'-hydroxylase is, for example, the gene listed as SEQ ID NO: 1 or SEQ ID NO: 3.  The gene coding for dihydroflavonol reductase is preferably derived from iris, Nierembergia, petunia, orchid, gentian or Eustoma russellianum.<br>
The invention still further provides (5) a rose obtained by the production method according to any one of (1) to (4) above, or a progeny or tissue thereof having the same properties as the rose.<br>
The invention still further provides (6) a rose obtained by the production method according to any one of (1) to (4) above, or a progeny or tissue thereof, wherein the petal color of the rose is violet, blue-violet or blue.<br>
The invention further provides (7) a rose according to (6) above, or a progeny or tissue thereof, wherein the petal color of the rose belongs to the "Violet group", "Violet-Blue" group or "Blue group" according to the Royal Horticultural Society Colour Chart (RHSCC).<br>
The invention further provides (8) a rose according to (7) above, or a progeny or tissue thereof, wherein the petal color of the rose belongs to "Violet group" 85a or 85b according to the Royal Horticultural Society Colour Chart (RHSCC).<br>
Brief Description of the Drawings<br>
Fig. 1 shows the flavonoid biosynthesis pathway.<br>
CHS: Chalcone synthase, CHI: Chalcone isomerase<br>
FNS: Flavone synthase, F3H: Flavanone 3-hydroxylase F3'H: Flavonoid 3'-hydroxylase<br>
F3'5'H: Flavonoid 3'5'-hydroxylase, FLS: Flavonol synthase<br>
DFR: Dihydroflavonol 4-reductase<br>
ANS: Anthocyanidin synthase, AS: Aurone synthase C2'GT: Chalcone 2'-glucosyl transferase Fig. 2 shows the structure of plasmid pBERDl. Fig. 3 shows the structure of plasmid pBPDBP2. Fig. 4 shows the structure of plasmid pBPDBPS. Fig. 5 shows the structure of plasmid pSPB461. Fig. 6 shows the structure of plasmid pSPB472. Fig. 7 shows the structure of plasmid pSPB130. Fig. 8 shows the structure of plasmid pSPB919. Fig. 9 shows the structure of plasmid pSPB920. Fig. 10 shows the structure of plasmid pSPB1106.<br>
Best Mode for Carrying Out the Invention Several reasons may be postulated for a lack of blue color in rose even with production of delphinidin.  The stability, solubility and color of anthocyanins varies depending on modification with acyl groups and sugars. Specifically, it is known that an increased number of aromatic acyl groups results in greater blueness.  Also, formation of complexes between flavonol and flavone copigments and anthocyanins produce a blue color and shift the maximum absorption wavelength toward the longer wavelength end while also increasing the absorbance. Anthocyanin color is also dependent on pH.  Since a lower pH tends toward redness and a more neutral pH produces blueness, the flower color depends on the pH of the vacuoles in which the anthocyanins are localized.  In addition, formation of metal chelates in the copresence of metal ions such as A13+ and Mg2+ can significantly affect flower color as well.  Trial and error and assiduous research led to the proposal for a modification whereby the proportion of delphinidin in flower petals is<br>
increased.<br>
First, it was attempted to create a blue rose by the same method used to create a blue-violet carnation. Specifically, it was attempted to analyze white rose variety 112 and identify a DFR-deficient line, but unlike carnation, no completely DFR-deficient line could be obtained.  This is presumably due to the fact that carnation is diploid while ordinarily cultivated rose is tetraploid, such that it is difficult to find a line deficient in a single gene.<br>
Next, the pansy F3'5'H gene and petunia DFR gene were transferred into the white flower variety Tineke and accumulation of delphinidin was detected, but the amount was minimal and a blue rose was not obtained.<br>
According to the present invention, the DFR gene, an enzyme participating in the rose endogenous flavonoid synthesis pathway, is artificially suppressed by a gene engineering technique, and the pansy F3'5'H gene is expressed while a dihydromyricetin-reducing DFR gene is also expressed, in order to increase the delphinidin content to roughly 80-100% of the total anthocyanidins in the flower petals, thereby allowing realization of a blue rose.<br>
The dihydromyricetin-reducing DFR genes used in this case were derived from iris (Iridaceae), Nierembergia (Solanaceae) and petunia (Solanaceae), but as other dihydromyricetin-reducing DFR gene sources there may be mentioned non-pelargonidin-accumulating plants such as tobacco (Solanaceae), cyclamen (Primulaceae), delphinium (.Ranunculaceae) , orchid (Orchidaceae) , gentian (Gentianaceae), Eustoma russellianum (Gentianaceae) and the like (Forkmann 1991, Plant Breeding 106, 1-26; Johnson et al., Plant J. 1999, 19, 81-85).  The DFR genes used for the present invention are genes that preferentially reduce dihydromyricetin.<br>
According to the invention, the flavonoid 3'-hydroxylase (F3'H) gene, an enzyme participating in the<br>
rose endogenous flavonoid synthesis pathway, is artificially suppressed by a gene engineering technique, and the pansy F3'5'H gene is expressed, in order to increase the delphinidin content to roughly 80-100% of the total anthocyanidins in the flower petals, thereby allowing realization of a blue rose.<br>
The roses obtained according to the invention have hitherto non-existent flower colors, and the invention can provide roses with flower colors belonging not only to the red-purple group, purple group and purple-violet group but also to the violet group, violet-blue group and blue group, according to the Royal Horticultural Society Colour Chart.<br>
Examples<br>
The present invention will now be explained in greater detail by the following examples.  Unless otherwise specified, the molecular biological protocols used were based on Molecular Cloning (Sambrook and Russell, 2001, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York).<br>
Example 1. Flower color measuring method<br>
The flower petal color shade was evaluated by measurement using a CM2022 spectrophotometric colorimeter (Minolta Japan) with a 10° visual field and a D65 light source, and analysis using SpectraMagic color control software (Minolta Japan).  The Royal Horticultural Society Colour Chart (RHSCC) number is the nearest color as compared against Color Classification System Version 2.1.1 (The Japan Research Institute Co., Ltd.; Japanese Unexamined Patent Publication No. 2002-016935), based on the color value (CIE L*a*b* color system) obtained by visual discrimination and measurement with the device mentioned above.  This system may be used for objective selection of the nearest RHSCC number.<br>
Upon measuring the color shades of flower petals of cultivars conventionally referred to as "blue roses" and<br>
determining the nearest colors according to the RHSCC by this method, it was determined that Blue Moon and Madam Violet were 186d (Greyed-Purple group), Lavande was 186c (Greyed-Purple group), Seiryu was 189d (Greyed-Green group) and Blue Heaven was 198d (Greyed-Green group). These cultivars are called blue roses but are classified in "Grey" groups according to RHSCC number and therefore do not exhibit the blue color which is the object of the present invention.<br>
Example 2. Flavonoid analysis<br>
1)	Extraction of flower petal color<br>
A 0.5 g portion of freeze-dried rose petals was subjected to extraction in 4 ml of 50% acetonitrile (CH3CN) containing 0.1% TFA for 20 minutes under ultrasonic vibration and then filtered with a 0.45 ^m filter.  High-performance liquid chromatography (HPLC) of the anthocyanins in the extract was conducted under the following conditions.  Isocratic elution was carried out using an RSpak DE-413L (4.6 mm(|&gt; x 25 cm, Shoko Co., Ltd.) column with a flow rate of 0.6 ml/min, and a mobile phase at a linear concentration gradient of 10%-»50% CH3CN/H20 containing 0.5% trifluoroacetic acid (TFA) for 15 minutes followed by 50% CH3CN/H20 containing 0.5% TFA for 10 minutes.  Detection was performed using an SPD-M10A photodiode array detector (Shimadzu Laboratories), with detection in the wavelength range of 600-250 nm and calculation of the abundance ratio of each anthocyanin based on the 520 nm absorbance area.<br>
2)	Anthocyanidin analysis<br>
A 0.2 ml portion of the filtrate was dried completely under reduced pressure in a glass test tube and dissolved in 0.2 ml of 6N hydrochloric acid (HC1) , and subjected to hydrolysis at 100°C for 20 minutes.  The hydrolyzed anthocyanidins were extracted with 0.2 ml of 1-pentanol, and the organic layer was analyzed by HPLC<br>
under the following conditions.  The column used was an ODS-A312 (6 mm x 15 cm, YMC Co., Ltd.), and elution was performed at a flow rate of 1 ml/min using a CH3COOH:CH3OH:H20 = 15:20:65 solution as the mobile phase.<br>
Detection was performed by spectral measurement at 600-400 nm using an SPD-M10A photodiode array detector (Shimadzu Laboratories) , identification based on absorption maximum (Xmax) and retention time (RT), and quantitation based on 520 nm absorbance area.  The retention time and Xmax of delphinidin and cyanidin under these HPLC conditions were 4.0 min, 5.2 min and 534 nm, 525 nm, respectively.  Delphinidin hydrochloride and cyanidin hydrochloride purchased from Funakoshi Co., Ltd. were used as samples for identification and quantitation.<br>
3) Flavonol analysis<br>
A 0.2 ml portion of the flower petal-extracted filtrate was dried to hardness under reduced pressure in a 1.5 ml Eppendorf tube and dissolved in 0.2 ml of 0.1 M potassium phosphate buffer (KPB) at pH 4.5, and then 6 units of (i-glucosidase (Shinnihon Kagaku Co., Ltd.) and 1 unit of naringenase (Sigma Chemical Co., MO, USA) were added and the mixture was kept at 30°C for 16 hours. After the reaction, 0.2 ml of 90% CH3CN was added to the enzyme reaction solution to terminate the reaction.  The solution was filtered with a 0.45 um filter and subjected to HPLC under the following conditions.<br>
Isocratic elution was carried out using a Develosil C30-UG-5 (4.6 mm<f> x 15 cm, Nomura Chemical Co., Ltd.) column with a flow rate of 0.6 ml/min, and a mobile phase at a linear concentration gradient of 18%-»63% CH3CN/H20 containing 0.1% TFA for 10 minutes followed by 63% CH3CN/H20 containing 0.1% TFA for 10 minutes.  Detection was performed using an SPD-M10A photodiode array detector, with detection in the wavelength range of 400-250 nm.  The R.T. and Xmax of kaempferol and quercetin under these conditions were 11.6 min, 365 nm and 10.3<br>
min, 370 nm, respectively.  Kaempferol and quercetin purchased from Funakoshi Co., Ltd. were used as samples for quantitation based on the A330 nm area.<br>
Example 3. pH measurement method<br>
Approximately 2 g of rose petals frozen at -80°C for 1 hour or longer was pressed with a homogenizer to obtain the petal juice.  The pH was measured by connecting a 6069-10C microelectrode (Horiba Laboratories) to a pH meter (F-22, Horiba Laboratories).<br>
Example 4. Transformation of rose<br>
Several methods have been reported for<br>
transformation of roses (for example, Firoozababy et al. Bio/Technology 12:883-888 (1994); US 5480789; US 5792927; EP 536,327 Al; US 20010007157 Al), and transformation may be carried out by any of these techniques.  Specifically, rose calli taken from aseptic seedling leaves were immersed for 5 minutes in a bacterial suspension of Agrobacterium tumefaciens AglO (Lazo et al., Bio/Technology 9:963-967, 1991), the excess bacterial suspension was wiped off with sterile filter paper, and the calli were transferred to subculturing medium and cocultivated for 2 days in a dark room.<br>
After subsequently rinsing with MS liquid medium containing 400 mg/L carbenicillin, the calli were transferred to selection/elimination medium prepared by adding 50 mg/L kanamycin and 200 mg/L carbenicillin to subculturing medium.  Upon repeating transfer and cultivation of the portions which grew normally in selection medium without growth inhibition, the kanamycin-resistant calli were selected out.  The kanamycin-resistant transformed calli were cultivated in redifferentiation medium containing 50 mg/L kanamycin and 200 mg/L carbenicillin to obtain kanamycin-resistant shoots.  The obtained shoots were rooted in 1/2MS medium and then habituated.  The habituated plants were potted and then cultivated in a closed greenhouse until blooming.<br>
Example 5. Obtaining rose flavonoid gene A cDNA library derived from Kardinal rose variety flower petals was screened using the petunia DFR gene (described in W096/36716) as the probe, to obtain rose DFR cDNA was which designated as pCGP645.  The details have already been reported (Tanaka et al., Plant Cell Physiol. 36, 1023-1031 1995).<br>
Likewise, the same library was screened with the petunia chalcone synthase-A (CHS-A) gene (Koes et al., Gene (1989) 81, 245-257) and the anthocyanidin synthase (ANS) gene (Martin et al., Plant J., (1991) 1, 37-49) according to a publicly known procedure (Tanaka et al., Plant Cell Physiol. 36, 1023-1031 1995), to obtain rose chalcone synthase (CHS) and anthocyanidin synthase (ANS) homologs which were designated as pCGP634 and pCGP1375, respectively.  The nucleotide sequence for rose CHS is listed as SEQ ID NO: 5, and the nucleotide sequence for rose ANS is listed as SEQ ID NO: 6.<br>
Example 6. Screening for white rose For creation of a blue cultivar by gene<br>
recombination, cultivars lacking only the DFR gene may be selected, in order to avoid competition between the endogenous anthocyanin synthesis pathway and the introduced genes (particularly the F3'5'H gene), and the petunia DFR gene and F3'5H gene transferred into those cultivars (W096/36716).<br>
A screening was conducted among the numerous existing white rose varieties, for those lacking only the DFR gene and normally expressing other anthocyanin biosynthesis enzyme genes.  The cause of flower color whitening is believed to be occasional mutation or deletion of structural genes involved in anthocyanin biosynthesis, and occasional loss of transcription regulating factors which control transcription of structural genes involved in anthocyanin biosynthesis. Roses lacking DFR gene mRNA were examined according to the method described in W096/36716.<br>
First, 112 primarily white rose lines were analyzed for flavonoid composition of the flower petals by the method described in Example 1, and lines with high accumulation of flavonols were selected.  The pH of each petal juice was then measured and 80 cultivars with relatively high pH values were chosen as primary candidates.<br>
RNA was then extracted from petals of these cultivars.  The RNA extraction was accomplished by a publicly known method (Tanaka et al., Plant Cell Physiol. 36, 1023-1031, 1995).  The obtained RNA was used to examine the presence or absence of mRNA corresponding to the rose DFR gene (Tanaka et al., Plant Cell Physiol. 36, 1023-1031, 1995) and the rose anthocyanidin synthase (ANS) gene.  RT-PCR was performed and eight cultivars (WKS-11, 13, 22, 36, 43, White Killarney, Tsuru No.2, Tineke) having low endogenous expression of DFR mRNA and normal ANS mRNA levels were selected.<br>
RT-PCR was carried out with a Script First-strand Synthesis System for RT-PCR (Invitrogen) using RNA obtained from petals of each cultivar.  The DFR mRNA was detected using DFR-2F (5'-CAAGCAATGGCATCGGAATC-3') (SEQ ID NO: 13) and DFR-2B (5'-TTTCCAGTGAGTGGCGAAAGTC-3') (SEQ ID NO: 14) primers, and the ANS mRNA was detected using ANS-2F (5'-TGGACTCGAAGAACTCGTCC-3') (SEQ ID NO: 15) and ANS-2B (5'-CCTCACCTTCTCCCTTGTT-3') (SEQ ID NO: 16) primers.<br>
These eight cultivars showed lower levels of DFR mRNA and normal levels of ANS mRNA in Northern blotting (Table 1), and their cultivating properties were excellent.  Two of the transformable cultivars (Tineke, WKS36) were decided on for actual transfer of the delphinidin-producing construct.<br><br>
(Table Removed) <br>
Example 7. Transfer of rose DFR gene into Tineke Plasmid pE2113 (Mitsuhara et al., Plant Cell Physiol. 37, 45-59, 1996) comprises the enhancer sequence repeat-containing cauliflower mosaic virus 35S (E1235S) promoter and the nopaline synthase terminator.  This plasmid was digested with SacI and the ends were blunted using a Blunting Kit (Takara).  The DNA fragment was ligated with an 8 bp Sail linker (Takara) and the obtained plasmid was designated as pUE5.<br>
Plasmid pUE5 was digested with Hindlll and EcoRI to obtain an approximately 3 kb DNA fragment, which was introduced into pBin!9 (Bevan M., Binary Agrobacterium Vector for plant transformation. Nucl. Acid Res. 12. 8711-21, 1984) previously digested with Hindlll and EcoRI, to obtain plasmid pBE5.  Next, pCGP645 was digested with BamHI and Xhol to obtain a DNA fragment containing full-length rose DFR cDNA.  This was ligated with pBE5 digested with BamHI and Xhol to construct pBERDl (Fig. 2).  The plasmid was transferred into<br>
Agrobacterium tumefaciens AglO.<br>
Plasmid pBERDl (Fig. 2) was transferred into the white rose cultivar "Tineke", and 18 transformants were obtained.  Flower color was altered in six of the obtained transformants.  Pigment analysis of two plants in which a clear color change from white to pink was observed confirmed accumulation of cyanidin and pelargonidin in both (Table 2).  These results suggested that the Tineke cultivar is a cultivar lacking the DFR gene.<br>
(Table Removed)<br>
Example 8. Transfer of pansy F3'5'H gene (#18) and petunia DFR gene into Tineke<br>
RNA was extracted from young budding pansy (Black Pansy variety) petals by the method of Turpen and Griffith (BioTechniques 4:11-15, 1986), and Oligotex-dT (Qiagen) was used for purification of polyA+RNA.  This polyA+RNA and a XZAPII/Gigapackll Cloning Kit (Stratagene) were used to construct a cDNA library from the young budding pansy petals.  After transferring approximately 100,000 pfu of phage plaques grown on an NZY plate onto a Colony/PlaqueScreen (DuPont), treatment was conducted by the manufacturer's recommended protocol.  The plaques were 32P-labeled and screened using petunia HflcDNA (pCGP602, Holton et al., Nature, 366, p276-279, 1993) as the probe.<br>
The membrane was subjected to pre-hybridization for 1 hour at 42°C in hybridization buffer (10% (v/v) formamide, 1 M NaCl, 10% (w/v) dextran sulfate, 1% SDS), and then the 32P-labeled probe was added to 1 x 106 cpm/ml<br>
and hybridization was performed for 16 hours at 42°C.  The membrane was then rinsed for 1 hour in 2xSSC, 1% SDS at 42°C, fresh rinsing solution was exchanged, and rinsing was again performed for 1 hour.  The rinsed membrane was exposed on a Kodak XAR film together with an intensifying screen, and the hybridization signal was detected.<br>
The results of cDNA analysis demonstrated that the two obtained cDNA had high identity with petunia Hfl. The two cDNA types were designated as pansy F3'5'H cDNA, BP#18 (pCGP1959) and BP#40 (pCGP1961).  The nucleotide sequence for #18 is listed as SEQ ID NO: 1, and its corresponding amino acid sequence is listed as SEQ ID NO: 2, the nucleotide sequence for #40 is listed as SEQ ID No. 3, and its corresponding amino acid sequence is listed as SEQ ID NO: 4.  BP#18 and BP#40 have 82% identity on the DNA level.  Also, BP#18 and BP#40 both exhibit 60% identity with petunia Hfl and 62% identity with petunia Hf2 (Holton et al., Nature, 366, p276-279, 1993), on the DNA level.<br>
Separately, plasmid pUE5 was digested with EcoRI and the ends were blunted using a Blunting Kit (Takara), and the obtained DNA fragment was ligated with an 8 bp HindiII linker (Takara), producing a plasmid which was designated as pUESH.  There was recovered an approximately 1.8 kb DNA fragment obtained by subjecting plasmid pGGP1959 containing pansy F3'5'H #18 cDNA to complete digestion with BamHI and partial digestion with Xhol.  The plasmid obtained by ligation of this with pUESH digested with BamHI and Xhol was designated as pUEBPlS.<br>
Separately, a DNA fragment containing petunia DFR cDNA was recovered by digestion of pCGP1403 (W096/36716) with BamHI and Xhol, and this DNA fragment was ligated with pBE5 that had been digested with BamHI and Xhol, to prepare pBEPD2.  Next, pUEBPIS was partially digested with Hindlll and an approximately 2.8 kb DNA fragment was recovered containing the E1235S promoter, pansy F3'5'H<br>
#18 cDNA and the nos terminator.  This fragment was ligated with a DNA fragment obtained by partial digestion of pBEPD2 with Hindlll to obtain a binary vector plasmid pBPDBP2 (Fig. 3).  This plasmid was introduced into Agrobacterium tumefaciens AglO.<br>
Plasmid pBPDBP2 (Fig. 3) was transferred into the white rose cultivar "Tineke", and 40 transformants were obtained.  Flower color was altered in 23 of the obtained transformants, and pigment analysis confirmed accumulation of delphinidin in 16 of the 19 analyzed transformants (Table 3).  The delphinidin content was 100% at maximum (average: 87%), but the maximum amount of pigment was very low at 0.035 mg per gram of petals and the flower color was only altered from RHS Color Chart 158d (Yellow-White group) to 56a (Red group) or 65b (Red-Purple group), while no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed) <br>
Example 9. Transfer of pansy F3'5'H gene (#40) and petunia DFR gene into Tineke<br>
Plasmid pE2113 (Mitsuhara et al., Plant Cell Physiol. 37, 45-59, 1996) was digested with HindiII and Xbal to obtain an approximately 800 bp DNA fragment, which was ligated with pBin!9 (Bevan M., Binary Agrobacterium Vector for plant transformation. Nucl. Acid Res. 12. 8711-21, 1984) previously digested with Hindlll and Xbal.  The obtained plasmid was designated as PCGP1391.  Another plasmid, pCGP669 (W094/21840), contains the petunia chalcone synthase A (CHS-A) gene promoter.  This plasmid was digested with EcoRI, blunted and then digested with Hindlll.<br>
The approximately 700 bp DNA fragment was ligated with pCGP1391 that had been digested with Hindlll and SnaBI, and the obtained plasmid was designated as<br>
pCGP1707.  Also, there was recovered an approximately 1.8 kb DMA fragment obtained by subjecting plasmid pCGP1961 containing pansy F3'5 ' H #40 cDNA to complete digestion with BamHI and partial digestion with Xhol.  The plasmid obtained by ligation of this with pUESH digested with BamHI and Xhol was designated as pUEBP40.  Plasmid pUEBP40 was digested with EcoRV and Xbal and an approximately 5.5 kb DNA fragment was recovered.<br>
This fragment was ligated with an approximately 700 bp fragment obtained by digesting plasmid pCGP!707 with Hindlll, blunting the ends and further digesting with Xbal, to obtain plasmid pUFBP40.  Next, pUFBP40 was partially digested with Hindlll and an approximately 3.4 kb DNA fragment was recovered containing the cauliflower 35S promoter enhancer, CHS-A promoter, pansy F3'5'H #40 cDNA and the nos terminator.  This fragment was ligated with a DNA fragment obtained by partial digestion of pBEPD2 with Hindlll to obtain a binary vector plasmid pBPDBPS (Fig. 4).  This plasmid was introduced into Agrobacterium tumefaciens AglO.<br>
Plasmid pBPDBPB (Fig. 4) was transferred into the white rose cultivar "Tineke", and 53 transformants were obtained.  Flower color was altered in 17 of the obtained transformants, and pigment analysis confirmed accumulation of delphinidin in 8 of the 9 analyzed transformants (Table 4).  The delphinidin content was 93% at maximum (average: 79%), but the maximum amount of pigment was very low at 0.014 mg per gram of petals and the flower color was only altered from RHS Color Chart 158d (Yellow-White group) to 56a (Red group) or 65b (Red-Purple group), while no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.  This suggested that the Tineke variety is not a variety lacking only the DFR gene.<br>
(Table Removed)<br>
Example 10. Transfer of pansy F3'5'H gene (#18) and petunia DFR gene into WKS36<br>
Plasmid pBPDBP2 (Fig. 3) was transferred into the white rose "WKS36", and 138 transformants were obtained. Flower color was altered in 10 of the obtained transformants, and accumulation of delphinidin was confirmed in all of the plants (Table 5).  The delphinidin content was 91% at maximum (average: 60%), but the maximum amount of pigment was very low at 0.033 mg per gram of petals and the flower color was only altered to very light pink, while no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.  This suggested that the WKS36 variety is not a variety lacking only the DFR gene.<br>
(Table Removed)<br>
Example 11. Transfer of pansy F3'5'H gene (#18) and petunia DFR gene into WKS36<br>
A plasmid obtained by replacing the AscI site of plasmid pUCAP (van Engelen et al., Transgenic Research 4, 288-290, 1995) with Pad linker was designated as pUCPP. Separately, an expression cassette prepared by linking the rose chalcone synthase promoter, pansy F3'5'H #18 cDNA and nos terminator was obtained in the following manner.<br>
Chromosomal DNA was extracted from young leaves of the Kardinal rose cultivar (Tanaka et al., Plant Cell Physiol. 36, 1023-1031, 1995).  An approximately 100 fig portion of DNA was partially digested with Sau3AI, and approximately 20-kb DNA fragments were recovered by sucrose density gradient.<br>
These were ligated with lambda phage EMBL3 (for example, Stratagene) that had been digested with BamHI, and a chromosomal DNA library was prepared by the manufacturer's recommended protocol.  The library was screened by a publicly known method (Tanaka et al., Plant Cell Physiol. 36, 1023-1031, 1995) using rose chalcone synthase cDNA (DNA database: GenBank Accession No. AB038246) as the probe.  Among the obtained chalcone<br>
synthase chromosome clones, there existed lambda CHS20 which included an approximately 6.4 kb DNA sequence upstream from the start codon of chalcone synthase.  The approximately 2.9 kb DNA fragment obtained by digestion of lambda CHS20 with Hindlll and EcoRV includes the chalcone synthase promoter region.<br>
This fragment was ligated with a fragment obtained by digestion of pUC19 (Yanisch-Perron C et al., Gene 33:103-119, 1985) with Hindlll and Smal.  This was designated as pCGPlllS.  The sequence of the chalcone synthase promoter region included therein is listed as SEQ ID NO: 21.  An approximately 2.9 kb DNA fragment obtained by digestion of pCGPlllS with Hindlll and Kpnl was ligated with a DNA fragment obtained by digestion of pJBl (Bodeau, Molecular and genetic regulation of Bronze-2 and other maize anthocyanin genes. Dissertation, Stanford University, USA, 1994) with Hindlll and Kpnl to obtain pCGP197.<br>
Separately, an approximately 300 bp DNA fragment containing the nopaline synthase terminator, obtained by digestion of pUE5 with SacI and Kpnl, was blunted and linked with pBluescriptSK- which had been digested with EcoRV and BamHI and blunted.  A plasmid of those obtained in which the 5' end of the terminator was close to the Sail site of pBluescriptSK- was designated as pCGP1986. A DNA fragment obtained by digesting pCGP1986 with Xhol, blunting the ends and further digesting with Sail was linked with a DNA fragment obtained by digesting pCGP197 with Hindlll, blunting the ends and further digesting with Sail, to obtain pCGP2201.<br>
Next, a DNA fragment obtained by digesting pCGP2201 with Sail and blunting the ends was linked with an approximately 1.7 kb DNA fragment (containing the pansy flavonoid 3' , 5'-hydroxylase gene) obtained by digesting pCGP1959 with BamHI and Kpnl and blunting the ends.  A plasmid of those obtained in which the rose chalcone synthase promoter had been inserted in a direction<br>
allowing transcription of the pansy flavonoid 3',5'-hydroxylase gene in the forward direction was designated as pCGP2203.  Plasmid pCGP2203 was recovered by digestion with Hindlll and Sacl.  The DNA fragment was cloned at the Hindlll and Sacl sites of pUCPP, and the resulting plasmid was designated as pSPB459.  Next, plasmid pE2113 was digested with SnaBI and a BamHI linker (Takara) was inserted to obtain a plasmid designated as pUE6.<br>
An approximately 700 bp DNA fragment obtained by digestion of pUE6 with Hindlll and BamHI was linked with an approximately 2.2 kb DNA fragment obtained by digestion of pCGP1405 (W096/36716) with BamHI and Bglll and with the binary vector pBinplus (van Engelen et al., Transgenic Research 4, 288-290, 1995) digested with Hindlll and BamHI, to obtain pSPB460.  An approximately 5 kb DNA fragment obtained by digestion of pSPB459 with PacI was introduced into the PacI site of pSPB460 to obtain pSPB461 (Fig. 5) having the petunia DFR and pansy F3'5'H #18 genes linked in the forward direction on the binary vector.  This plasmid is modified for constitutive expression of the petunia DFR gene in plants and specific transcription of the pansy F3'5'H #18 gene in flower petals.  The plasmid was transferred into Agrobacterium tumefaciens AglO.<br>
Plasmid pSPB461 (Fig. 5) was transferred into the white rose "WKS36", and 229 transformants were obtained. Flower color was altered in 16 of the obtained transformants, and accumulation of delphinidin was confirmed in all 12 of the pigment-analyzed plants (Table 6).  The delphinidin content was 79% at maximum (average: 58%), but the amount of pigment was very low at 0.031 mg per gram of petals and the flower color was only altered to very light pink, while no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.  This suggested that the WKS36 variety is not a variety lacking only the DFR gene.<br>
(Table Removed)<br>
Example 12. Transfer of pansy F3'5'H gene (t!8), petunia DFR gene and perilla anthocyanin 3-glucoside acyltransferase gene into WKS36<br>
A gene comprising a start codon added to the perilla hydroxycinnamoyl CoA: anthocyanin 3-glucoside acyltransferase (3AT) gene was designated as pSAT208F (Yonekura-Sakakibara et al., Plant Cell Physiol. 41, 495-502, 2000).  An approximately 3.9 kb DNA fragment obtained by digestion of pSPB580 (PCT/AU03/00079) with BamHI and Xhol was linked with an approximately 1.8 kb DNA fragment obtained by digestion of pSAT208F with BamHI and Xhol.<br>
The obtained plasmid was digested with AscI, and a DNA fragment was recovered containing the E1235S promoter, the perilla SAT gene and the petunia phospholipid transfer protein terminator.  The DNA fragment was inserted into the AscI site of pSPB461 to obtain plasmid pSPB472 (Fig. 6) having the perilla 3AT, petunia DFR and pansy F3'5'H #18 gene transcription directions in the forward direction.  This plasmid is modified for constitutive expression of the perilla SAT<br><br>
gene and the petunia DFR gene in plants and specific transcription of the pansy F3'5'H #18 gene in flower petals.  The plasmid was transferred into Agrobacterium tumefaciens AglO.<br>
Plasmid pSPB472 (Fig. 6) was transferred into the white rose "WKS36", and 75 transformants were obtained. Flower color was altered in four of the obtained transformants, and accumulation of delphinidin was confirmed in all three of the pigment-analyzed plants (Table 7).  The delphinidin content was 67% at maximum (average: 49%), but the amount of pigment was very low at 0.011 mg per gram of petals and the flower color was only altered to very light pink, while no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.  This suggested that the WKS36 variety is not a variety lacking only the DFR gene.<br>
(Table Removed)<br>
 Thus, despite screening of several white roses, it was not possible to obtain a cultivar lacking only the DFR gene.  In other words, it was not possible to obtain a blue rose by the method for creation of blue carnation (W094/28140).<br>
Example 13. Inhibition of rose DFR gene by cosuppression<br>
Plasmid pBERDl was transferred into the pale violet rose "Lavande", and 26 transformants were obtained. However, none of the plants exhibited altered flower color, suggesting that it is difficult to inhibit the<br><br>
rose endogenous DFR gene by cosuppression.<br>
Example 14. Screening for colored roses Cultivars for creation of blue roses were then selected from among colored roses.  After visually selecting 136 lines from colored rose cultivars with relatively blue shades, 89 of the lines were subjected to pigment analysis.  The values obtained for the examined colored roses are shown in Tables 8 to 10.<br>
(Table Removed)<br>
Example 15. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into Lavande<br>
Modification of anthocyanins with aromatic acyl groups can stabilize the anthocyanins and produce a bluer color (for example, W096/25500).  The following experiment was conducted with the goal of producing acylated delphinidin-type anthocyanins.<br>
RNA was obtained from Torenia Summer Wave flower petals, and polyA+RNA was prepared therefrom.  A cDNA library was prepared from the polyA+RNA with A.ZAPII (Stratagene) as the vector, using a directional cDNA library preparation kit (Stratagene) according to the manufacturer's recommended protocol.  The major anthocyanin of Torenia is modified with an aromatic acyl group at the 5-position glucose (Suzuki et al., Molecular Breeding 2000 6, 239-246), and therefore anthocyanin acyltransferase is expressed in Torenia petals.<br>
Anthocyanin acyltransferase includes the conserved amino acid sequence Asp-Phe-Gly-Trp-Gly-Lys, and corresponding synthetic DNA can be used as primer to obtain the anthocyanin acyltransferase gene (W096/25500). Specifically, 10 ng of single-stranded cDNA synthesized for construction of the Torenia cDNA library was used as template, and 100 ng of ATC primer (51-<br>
GA(TC)TT(TC)GGITGGGGIAA-3', I: inosine) (SEQ ID NO: 17) and 100 ng of oligo dT primer  (SEQ ID NO: 18) were used as primers for PCR with Taq polymerase (Takara, Japan), under the manufacturer's recommended conditions.<br>
The PCR was carried out in 25 cycles of reaction with one cycle consisting of 1 minute at 95°C, 1 minute at 55°C and 1 minute at 72°C.  The approximately 400 bp DNA fragment that was obtained was recovered with Gene Clean II (BIO,101. Inc.) according to the manufacturer's recommended protocol, and was subcloned in pCR-TOPO. Determination of the nucleotide sequence revealed a sequence homologous to the gentian acyltransferase gene (Fujiwara et al., 1998, Plant J. 16 421-431).  The nucleotide sequence was determined by the Dye Primer method (Applied Biosystems), using Sequencer 310 or 377 (both by Applied Biosystems).<br>
The DNA fragment was labeled with DIG using a DIG-labeling detection kit (Japan Roche), and used for<br>
screening of a Torenia cDNA library by plaque hybridization according to the manufacturer's recommended protocol.  Twelve of the obtained positive signal clones were randomly selected, the plasmids were recovered, and their nucleotide sequences were determined.  These exhibited high homology with anthocyanin acyltransferase. The total nucleotide sequence of the cDNA in the clone designated as pTAT7 was determined.  The nucleotide sequence is listed as SEQ ID NO: 7, and the corresponding amino acid sequence is listed as SEQ ID NO: 8.<br>
After digesting pBE2113-GUS (Mitsuhara et al., Plant Cell Physiol. 37, 45-59, 1996) with SacI, the ends were blunted and an 8 bp Xhol linker (Takara) was inserted. An approximately 1.7 kb DNA fragment obtained by digesting pTAT7 with BamHI and Xhol was inserted at the BamHI and Xhol sites of this plasmid, to obtain pSPB120. After digesting pSPB120 with SnaBI and BamHI, the ends were blunted and ligation was performed to obtain pSPB120'.  Separately, plasmid pCGP1961 containing pansy F3'5'H #40 cDNA was completely digested with BamHI and then partially digested with Xhol to obtain an approximately 1.8 kb DNA fragment which was recovered and ligated with pUESH previously digested with BamHI and Xhol, to obtain a plasmid which was designated as pUEBP40.<br>
After digesting pUEBP40 with SnaBI and BamHI, the ends were blunted and ligation was performed to obtain pUEBP40'.  This plasmid pUEBP40' was partially digested with Hindlll to obtain an approximately 2.7 kb DNA fragment which was recovered and linked with a DNA fragment obtained by partial digestion of pSPB120' with Hindlll.  Of the obtained plasmids, a binary vector having the neomycin phosphotransferase gene, pansy F3'5'H #40 gene and Torenia SAT gene linked in that order in the same direction from the right border sequence on the binary vector, was designated as pSPB130 (Fig. 7).  This plasmid is modified for constitutive expression of the<br>
pansy F3'5'H #40 gene and the Torenia SAT gene in plants and specific transcription of the genes in the flower petals.  The plasmid was transferred into Agrobacterium tumefaciens AglO.<br>
Plasmid pSPB130 (Fig. 7) was transferred into the pale violet rose variety "Lavande", and 41 transformants were obtained.  Accumulation of delphinidin was confirmed in 20 of the 32 pigment-analyzed plants (Tables 11 and 12).  The delphinidin content was 71% at maximum (average: 36%).  The flower color was altered from RHS Color Chart 186c (Greyed-Purple group) to 79d (Purple group) .  The proportion of acylated anthocyanins was only about 30% of the total anthocyanins.  Upon spectral measurement of the acylated anthocyanins, the maximum absorption wavelength had shifted toward longer wavelength by 4 nm from delphinidin 3,5-diglucoside, but because of the low proportion among the total anthocyanins, no clear effect was achieved for the flower color.<br>
(Table Removed)<br>
Example 16. Transfer of pansy F3'5'H gene (#40) and<br>
Torenia anthocyanin 5-acyltransferase gene into WKS100 Plasmid pSPB130 (Fig. 7) was transferred into the pale violet rose variety "WKS100", and 146 transformants were obtained.  Accumulation of delphinidin was confirmed in 56 of the 63 pigment-analyzed plants (Tables 13-15). The delphinidin content was 95% at maximum (average: 44%).  The flower color was altered from RHS Color Chart 56d (Red group) to 186d (Greyed-Purple group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed)<br>
Example 17. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS116<br>
Plasmid pSPB130 (Fig. 7) was transferred into the pale violet rose variety "WKS116", and 282 transformants were obtained.  Accumulation of delphinidin was confirmed in 33 of the 36 pigment-analyzed plants (Tables 16 and 17).  The delphinidin content was 80% at maximum (average: 73%).  The flower color was altered from RHS Color Chart 196d (Greyed-Green group) to 186d (Greyed-Purple group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
 (Table Removed)<br>
Example 18. Transfer of pansy F3' 5' H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS124<br>
Plasmid pSPB130 (Fig. 7) was transferred into the pale orange rose variety "WKS124", and 50 transformants were obtained.  Accumulation of delphinidin was confirmed in 13 of the 15 pigment-analyzed plants (Table 18).  The delphinidin content was 95% at maximum (average: 82%). The flower color was altered from RHS Color Chart 52d (Red group) to 71c (Red-Purple group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed)<br>
Example 19. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS132<br>
Plasmid pSPB130 (Fig. 7) was transferred into the bright red rose variety "WKS132", and 24 transformants were obtained.  Accumulation of delphinidin was confirmed in 6 of the 7 pigment-analyzed plants (Table 19).  The delphinidin content was 43% at maximum (average: 12%). The flower color was altered from RHS Color Chart 57a (Red-Purple group) to 66a (Red-Purple group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed)<br>
 Example 20. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS133<br>
Plasmid pSPB130 (Fig. 7) was transferred into the dark red-violet rose variety "WKS133", and 16 transformants were obtained.  Accumulation of delphinidin was confirmed in all eight of the pigment-analyzed plants (Table 20).  The delphinidin content was 34% at maximum (average: 11%).  The flower color was altered from RHS Color Chart 53a (Red group) to 61a (Red-Purple group). However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br><br>
(Table Removed)<br>
Example 21. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS137<br>
Plasmid pSPB130 (Fig. 7) was transferred into the dark red-violet rose variety "WKS137", and 20 transformants were obtained.  Accumulation of delphinidin was confirmed in all 17 of the pigment-analyzed plants (Table 21).  The delphinidin content was 1.3% at maximum (average: 0.4%).  No alteration in flower color was observed from RHS Color Chart 61b (Red-Purple group).<br>
(Table Removed)<br>
Example 22. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS140<br>
Plasmid pSPB130 (Fig. 7) was transferred into the pale violet rose variety "WKS140", and 197 transformants were obtained.  Accumulation of delphinidin was confirmed in 37 of the 45 pigment-analyzed plants (Tables 22 and 23).  The delphinidin content was 94% at maximum (average: 47%).  The flower color was altered from RHS Color Chart 186d (Greyed-Purple group) to 79d (Purple group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed)<br>
Example 23. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS77<br>
Plasmid pSPB130 (Fig. 7) was transferred into the dark red-purple rose variety "WKS77", and 35 transformants were obtained.  Accumulation of delphinidin was confirmed in all 17 of the pigment-analyzed plants (Table 24).  The delphinidin content was 57% at maximum (average: 33%).  The flower color was altered from RHS Color Chart 57a (Red-Purple group) to 71a (Red-Purple group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed)<br>
Example 24. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS82<br>
Plasmid pSPB130 (Fig. 7) was transferred into the pale violet rose variety "WKS82", and 89 transformants were obtained.  Accumulation of delphinidin was confirmed in all 44 of the pigment-analyzed plants (Tables 25 and 26).  The delphinidin content was 91% at maximum (average: 49%).  The flower color was altered from RHS Color Chart 186d (Greyed-Purple group) to 80c (Purple-Violet group).  However, no color of the Violet group, Violet-Blue group or Blue group according to the RHSCC was achieved and the target blue rose could not be obtained.<br>
(Table Removed)<br>
Example 25. Transfer of pansy F3'5'H gene (#40) and Torenia anthocyanin 5-acyltransferase gene into WKS91<br>
Plasmid pSPB130 (Fig. 7) was transferred into the light orange rose variety "WKS91", and 10 transformants were obtained.  Accumulation of delphinidin was confirmed in only one of the two pigment-analyzed plants (Table 27).  The delphinidin content was 2% at maximum.  No alteration in flower color was observed from RHS Color Chart 43c (Red group).<br>
(Table Removed)<br>
Example 26. Expression of pansy F3'5'H gene (#40) and iris DFR gene and suppression of rose endogenous DFR gene in Lavande<br>
RNA was obtained from blue iris petals of cut flowers, and polyA+RNA was prepared therefrom.  A cDNA<br>
library was prepared from the polyA+RNA with XZAPII (Stratagene) as the vector, using a cDNA library preparation kit (Stratagene) according to the manufacturer's recommended protocol.  An iris DFR gene fragment was prepared by the same method as reported for obtaining gentian DFR gene fragment (Tanaka et al. Plant Cell Physiol. 37, 711-716 1996).<br>
The approximately 400 bp DNA fragment obtained was recovered with Gene Clean according to the manufacturer's recommended protocol, and was subcloned in pCR-TOPO. Determination of the nucleotide sequence revealed a sequence homologous to the rose DFR gene.  The DNA fragment was used for screening of the iris cDNA library,<br>
and iris DFR cDNA including the full-length amino acid sequence was obtained.  The total nucleotide sequence of the cDNA in the clone designated as pSPB906 was determined.  The nucleotide sequence is listed as SEQ ID NO: 9, and the corresponding amino acid sequence is listed as SEQ ID NO: 10.<br>
Next, an approximately 3.9 kb DNA fragment obtained by digestion of pSPBSSO with BamHI and Xhol was linked with an approximately 1.5 kb DNA fragment obtained by digestion of pSPB906 with BamHI and Xhol, and the obtained plasmid was designated as pSPB909.<br>
A vector for transcription of double-stranded RNA for the rose DFR cDNA in plants was prepared in the following manner.  An approximately 3.5 kb DNA fragment (including Macl promoter, rose DFR cDNA and mas terminator) obtained by partial digestion of pCGP1364 (Tanaka et al., Plant Cell Physiol. (1995) 36, 1023-1031) with PstI was inserted at the PstI site of pUC19 (Yanisch-Perron C et al., Gene 33:103-119, 1985) to obtain plasmids, among which a plasmid having the Hindlll site of pUC19 near the Macl promoter was designated as pCGP1394.<br>
Next, an approximately 1.4 kb DNA fragment obtained by digestion of pCGP1394 with Hindlll and SacII was ligated with an approximately 1.9 kb DNA fragment obtained by digestion of pCGP1394 with PstI, blunting of the ends and further digestion with SacII, and with a binary vector fragment obtained by digestion of pBinPLUS with SacI, blunting of the ends and further digestion with Hindlll, to obtain pSPB!85.  Plasmid pSPBlSS was digested with Xbal, blunted and ligated with a Sail linker to obtain pSPB521.  An approximately 700 bp DNA fragment obtained by digestion of pUE6 with Hindlll and BamHI was ligated with a binary vector DNA fragment obtained by digestion of pSPB521 with Hindlll and SacI and with a GUS gene fragment obtained by digestion of pE2113 with BamHI and SacI, to obtain pSPB528.<br>
Plasmid pSPB528 is a binary vector having a structural gene inserted between the enhancer-containing cauliflower mosaic virus 35S promoter and the manopine synthase terminator, which is expressible in plants. Also, in order to shorten the 5'-end non-translated sequence of rose DFR cDNA in pCGP645, plasmid pCGP645 was digested with Smal and Pvul, blunted and re-ligated to obtain pCGP645s.<br>
The 5'-end sequence of rose DFR cDNA was obtained by PCR amplification using pCGP645s as the template and a reverse primer and the synthetic primer RDF310 (5'-CCCTCGAGCCCTTGATGGCCTCGTCG-3') (SEQ ID NO: 19) as the primers, and was cloned in pCRTOPO.  The DNA nucleotide sequence was determined and absence of errors by PCR was confirmed.  This plasmid was designated as pSPB569. Also, a rose DFR cDNA 5'-end sequence with a different length was obtained by amplification using pCGP645s as the template and a reverse primer and the synthetic primer RDF830 (5'-GGGTCGACGCGGCCCTCTGCTTTCGG-3') (SEQ ID NO: 20) as the primers, and was cloned in pCRTOPO.  The DNA nucleotide sequence was determined and absence of errors by PCR was confirmed.<br>
This plasmid was designated as pSPB570.  A binary vector DNA fragment obtained by digestion of pSPB528 with BamHI and SacI, and an approximately 0.3 kb DNA fragment obtained by digestion of pSPB569 with SacI and Xhol, were ligated with a DNA fragment obtained by digestion of pSPB570 with BamHI and Sail, to obtain pSPB572.  This vector is designed for transcription of double-stranded RNA for rose DFR cDNA in plants.<br>
Plasmid pUE6 was digested with SacI and blunted, and a Sail linker was inserted to obtain pUE8.  A DNA fragment obtained by digesting pUE8 with Hindlll and EcoRI was introduced at the Hindlll and EcoRI sites of pBinPLUS to obtain plasmid pSPB189.  An approximately 3.7 kb DNA fragment obtained by digestion of pSPB189 with BamHI and Sail was ligated with an approximately 1.8 kb<br>
DNA fragment obtained by complete digestion of pCGP1961 with BamHI followed by partial digestion with Xhol, to obtain plasmid pSPB567.  After PacI digestion and dephosphorylation treatment of pSPB572, it was linked with an approximately 2.8 kb DNA fragment obtained by digestion of pSPB567 with PacI, and a plasmid with transcription of the nptll gene and pansy F3'5'H #40 in the same direction was selected and designated as pSPB905.<br>
After AscI digestion and dephosphorylation treatment of pSPB905, it was linked with an approximately 2.5 kb DNA fragment obtained by digestion of pSPB909 with AscI, and a plasmid with transcription of the iris DFR gene in the same direction as the nptll gene was obtained and designated as pSPB919 (Fig. 8).  This plasmid is expected to allow transcription of the iris DFR gene and pansy F3'5'H #40 gene in rose, while suppressing expression of the rose DFR gene due to transcription of double-stranded RNA.  The plasmid was transferred into Agrobacterium tumefaciens AglO.<br>
Plasmid pSPB919 (Fig. 8) was transferred into the pale violet rose variety "Lavande", and 87 transformants were obtained.  Accumulation of delphinidin was confirmed in 31 of the 38 pigment-analyzed plants (Tables 28 and 29).  The delphinidin content was 100% at maximum (average: 76%).  The flower color was altered from RHS Color Chart 186c (Greyed-Purple group) to 85a,b (Violet group).<br>
RNA was extracted from rose petals in the same manner as explained above, and after separating the RNA by agarose gel electrophoresis, it was transferred onto Hybond N (Amersham) (for example, Tanaka et al., 1995). The mRNA was detected using a DIG Northern Starter Kit (Roche) by the manufacturer's recommended protocol.  The rose DFR mRNA was detected using pCGP645 (Tanaka et al., Plant Cell Physiol. 36, 1023-1031, 1995) as template and a T7 primer transcript as the probe.<br>
Detection of pansy F3'5'H #40 mRNA was accomplished using pCGP1961 as template and a T7 primer transcript as the probe.  Detection of iris DFR mRNA was accomplished using pSPB906 as template and a T7 primer transcript as the probe.  Pansy F3'5'H #40 and iris DFR gene mRNA were detected in the altered-color roses.  On the other hand, rose DFR mRNA was significantly reduced compared to the host and a band was detected at the low molecular weight position, indicating decomposition of the rose DFR mRNA.<br>
(Table Removed)<br>
Example 27. Expression of pansy F3'5'H gene (#40) and Nierembergia DFR gene, and suppression of rose endogenous DFR gene in Lavande<br>
RNA was obtained from petals of the Nierembergia hybrida cultivar Fairy Bell Patio Light Blue (Suntory Flowers Co., Ltd.), and polyA+RNA was prepared therefrom. A cDNA library was prepared from the polyA+RNA with XZAP (Stratagene) as the vector, using a cDNA library synthesis kit (Stratagene) according to the manufacturer's recommended protocol.  The cDNA library was screened using DIG-labeled petunia DFR cDNA (from pCGP1405).<br>
The screening conditions were according to the plaque hybridization method using a DIG-labeling system, according to the manufacturer's recommended protocol. However, the formaldehyde concentration was 30% for the pre-hybridization and hybridization buffers, and<br>
hybridization was carried out overnight at 37°C.  The membrane was rinsed at 55°C in SxSSC containing 1% SDS.<br><br>
Plasmids were recovered from 20 plaques among the numerous positive signals, and their nucleotide sequences were determined using Reverse Primer (Takara) .  These exhibited high homology with the DFR genes of other plants including petunia.  The total nucleotide sequence of the cDNA in the clone designated as pSPB709 was determined.  The nucleotide sequence is listed as SEQ ID NO: 11, and the corresponding amino acid sequence is listed as SEQ ID NO: 12<br>
An approximately 3.9 kb DNA fragment obtained by digestion of pSPB580 with BamHI and Xhol was linked with an approximately 1.5 kb DNA fragment obtained by digestion of pSPB709 with BamHI and Xhol, to obtain plasmid pSPB910.  After AscI digestion and dephosphorylation treatment of pSPB910, it was linked with an approximately 2.5 kb DNA fragment obtained by digestion of pSPB910 with AscI, and a plasmid with transcription of the Nierembergia DFR gene in the same direction as the nptll gene was obtained and designated as pSPB920 (Fig. 9).  This plasmid is expected to allow transcription of the Nierembergia DFR gene and pansy F3'5'H #40 gene in rose, while suppressing expression of the rose DFR gene due to transcription of double-stranded RNA.  The plasmid was transferred into Agrobacterium tumefaciens AglO.<br>
Plasmid pSPB920 (Fig. 9} was transferred into the pale violet rose variety "Lavande", and 56 transformants were obtained.  Accumulation of delphinidin was confirmed in 23 of the 24 pigment-analyzed plants (Table 30).  The delphinidin content was 100% at maximum (average: 43%). The flower color was altered from RHS Color Chart 186c (Greyed-Purple group) to 85b (Violet group).<br>
(Table Removed)<br>
Example 28. Inheritance of traits to progeny Cross-breeding was carried out using a transformant (LA/919-2-13) obtained by transfer of pSPB919 (Fig. 8) into the pale violet rose variety "Lavande" as the pollen parent and non-recombinant WKS77 or WKS133 as the maternal parent (Suzuki, S., "Bara, Hanazufu", Shogakkann, p.256-260, 1990).  Fruit was collected on the 100th day after pollination.  Seed production was accomplished by first peeling the fruit, harvesting the achene, peeling the achene, and then removing the germ and embedding it on moistened filter paper in a dish. The water used for seed production was sterilized water<br>
containing 1 ml/1 PPM™ (Plant Preservative Mixture, Plant Cell Technology, Inc.) and 50 mg/1 kanamycin, and seedlings were raised by potting only the normally budded plants.<br>
Accumulation of delphinidin was confirmed in all 40 of the pigment-analyzed transformant progeny (Tables 31 and 32).  The delphinidin content was 99% at maximum (average: 46%) .<br>
(Table Removed)<br>
Example 29. Expression of pansy F3'5'H #40 gene and iris DFR gene and suppression of rose endogenous DFR gene in WKS140<br>
Plasmid pSPB919 was transferred into the pale violet rose variety "WKS140", and 89 transformants were obtained.  Accumulation of delphinidin was confirmed in 74 of the 79 pigment-analyzed plants.  The delphinidin content was 100% at maximum (average: 68%).  The flower color was altered from RHS Color Chart 186d (Greyed-Purple group) to primarily 84c (Violet group).<br>
(Table Removed)<br>
Example 30. Expression of pansy F3'5'H #40 gene and iris DFR gene and suppression of rose endogenous DFR gene in WKS77<br>
Plasmid pSPB919 was transferred into the dark red-purple rose variety "WKS77", and 50 transformants were obtained.  Accumulation of delphinidin was confirmed in 21 of the 23 pigment-analyzed plants.  The delphinidin content was 81% at maximum (average: 19%).  The flower color was altered from RHS Color Chart 57a (Red-Purple group) to 77b (Purple group).<br>
(Table Removed)<br>
Example 31. Expression of pansy F3'5'H #40 gene and Nierembergia DFR gene and suppression of rose endogenous DFR gene in WKS77<br>
Plasmid pSPB920 was transferred into the dark red-purple rose variety "WKS77", and 30 transformants were obtained.  Accumulation of delphinidin was confirmed in 26 of the 27 pigment-analyzed plants.  The delphinidin content was 98% at maximum (average: 60%) .  The flower color was altered from RHS Color Chart 57a (Red-Purple group) to 77b (Purple group).<br>
(Table Removed)<br>
Example 32. Expression of pansy F3'5'H #40 gene and petunia DFR gene and suppression of rose endogenous DFR gene in WKS77<br>
Plasmid pSPB921 was transferred into the dark red-purple rose variety "WKS77", and 15 transformants were obtained.  Accumulation of delphinidin was confirmed in 12 of the 13 pigment-analyzed plants.  The delphinidin content was 98% at maximum (average: 60%).  The flower color was altered from RHS Color Chart 57a (Red-Purple group) to 72b (Red-Purple group).<br>
(Table Removed)<br>
Example 33. Inheritance of traits to progeny Cross-breeding was carried out in the same manner as Example 28, using a transformant (LA/919-4-10) obtained by transfer of pSPB919 into the pale violet rose variety "Lavande" as the pollen parent and the non-recombinant rose variety "Black Baccara" as the maternal parent. Fruit was collected on the 100th day after pollination. Seed production was accomplished by first peeling the fruit, harvesting the achene, peeling the achene, and then removing the germ and embedding it on moistened filter paper in a dish.  The water used for seed production was sterilized water containing 1 ml/1 PPM™ (Plant Preservative Mixture, Plant Cell Technology, Inc.) and 50 mg/1 kanamycin, and seedlings were raised by potting only the normally budded plants.<br>
Accumulation of delphinidin was confirmed in all 18 of the pigment-analyzed transformant progeny.  The delphinidin content was 99.8% at maximum (average: 98.7%) (Table Removed)<br>
Example 34. Expression of pansy F3'5'H #40 gene and suppression of rose endogenous F3' H gene in WKS77<br>
Plasmid pSPB1106 (Fig. 10) was transferred into the dark red-purple rose variety "WKS77", and 40 transformants were obtained.  Accumulation of delphinidin was confirmed in all 26 of the pigment-analyzed plants. The delphinidin content was 80.0% at maximum (average: 30.5%).  The flower color underwent a major alteration from RHS Color Chart 57a (Red-Purple group) to 83d (Violet group).<br>
(Table Removed)<br>
Example 35. Expression of pansy F3'5'H #40 gene and suppression of rose endogenous F3'H gene in Lavande<br>
Plasmid pSPB1106 was transferred into the pale violet rose variety "Lavande", and 40 transformants were obtained.  Accumulation of delphinidin was confirmed in 23 of the 25 pigment-analyzed plants.  The delphinidin content was 98.3% at maximum (average: 46.9%).<br>
(Table Removed)<br>
These results demonstrate that the transferred exogenous gene was inherited and expressed by the progeny, and that the trait of delphinidin production which is not found in ordinary rose petals was successfully inherited by the rose progeny.  Thus, this gene can be used for cross-breeding cultivation of roses with altered colors to create roses with new colors including blue and purple.<br>
Industrial Applicability<br>
By artificially suppressing function of the endogenous metabolic pathway such as, for example, expression of dihydroflavonol reductase, in rose, and<br>
expressing the gene coding for pansy flavonoid 3',5'-hydroxylase and a gene coding for dihydroflavonol reductase from species other than rose, it is possible to create blue to violet roses.  These genes are inherited by subsequent generations, and the blue rose trait can be utilized for cross-breeding.<br><br><br><br><br>
We Claim:<br>
1.	A method for producing a rose characterized by artificially suppressing the rose<br>
flavonoid synthesis pathway and expressing the pansy gene coding for flavonoid<br>
3',5'-hydroxylase having the nucleotide sequence shown in SEQ ID NO. 1 or 3,<br>
wherein the rose flavonoid synthesis pathway is suppressed by<br>
(i)        artificially suppressing expression of rose endogenous dihydroflavonol<br>
reductase; or <br>
           (ii)       artificially  suppressing expression of rose  endogenous  flavonoid 3'-<br>
hydroxylase.<br>
2.	A method for producing a rose as claimed in claim l(i), additionally comprising<br>
expression of the gene coding for dihydroflavonol reductase derived from a plant<br>
other than rose.<br>
3.	A method as claimed in claim 2, wherein the plant other than rose is iris, Nierembergia or petunia.</f></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1ERUxOUC0yMDA2LUFic3RyYWN0LSgxOC0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1372-DELNP-2006-Abstract-(18-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUFic3RyYWN0LSgxOC0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Abstract-(18-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUFtZW5kIFNwZWNpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Amend Specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUFzc2lnbm1lbnQtKDI4LTA3LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Assignment-(28-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1ERUxOUC0yMDA2LUNsYWltcy0oMTgtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-DELNP-2006-Claims-(18-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUNsYWltcy0oMTgtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Claims-(18-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Correspondence Others-(05-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-DELNP-2006-Correspondence Others-(18-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Correspondence Others-(18-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTktMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Correspondence Others-(19-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Correspondence Others-(28-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Correspondence-Others-(22-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUZvcm0tMS0oMjgtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Form-1-(28-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUZvcm0tMTMtKDA1LTEwLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Form-13-(05-10-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUZvcm0tMi0oMTgtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Form-2-(18-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1ERUxOUC0yMDA2LUZvcm0tMy0oMTgtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-DELNP-2006-Form-3-(18-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUZvcm0tMy0oMTktMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-Form-3-(19-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LUdQQS0oMjgtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-GPA-(28-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1kZWxucC0yMDA2LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">1372-delnp-2006-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3Mi1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMTgtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1372-DELNP-2006-Petition-137-(18-05-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259686-onics-metal-clad-wire-wound-dynamic-braking-resistor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259688-wafer-type-direct-acting-valve.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259687</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1372/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SUNTORY HOLDINGS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1-40, DOJIMAHAMA 2-CHOME, KITA-KU, OSAKA-SHI, OSAKA 530-8203, JAPAN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YOSHIKAZU TANAKA</td>
											<td>2-7-4, OHGINOSATO, OTSU-SHI, SHIGA 520-0246, JAPAN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YUKO FUKUI</td>
											<td>1-2A-1007, AKUTAGAWACHO, TAKATSUKI-SHI, OSAKA 569-1123, JAPAN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JUNICHI TAGAMI</td>
											<td>5-15-10-303, TONDACHO, TAKATSUKI-SHI, OSAKA 569-0814, JAPAN</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YUKIHISA KATSUMOTO</td>
											<td>1-9-5-402, YAMAZAKI, SHIMAMOTO-CHO, MISHIMA-GUN, OSAKA 618-0001, JAPAN</td>
										</tr>
										<tr>
											<td>5</td>
											<td>MASAKO MIZUTANI</td>
											<td>53-60, KATSURAINUICHO, NISHIKYO-KU, KYOTO-SHI, KYOTO 615-8086, JAPAN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2004/011958</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2004-192034</td>
									<td>2004-06-29</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>2</td>
									<td>2003-293121</td>
									<td>2003-08-13</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259687-process-for-producing-rose-plant-cell-with-modified-color by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:50:25 GMT -->
</html>
