<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259641-genotoxic-testing by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:03:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259641:&quot;GENOTOXIC TESTING&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;GENOTOXIC TESTING&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for detecting for the presence of an agent that putatively causes or potentiates DNA damage comprising subjecting a cell (containing a DNA sequence encoding a reporter protein operatively linked to a human GADD45a gene promoter and a human GADD45a gene regulatory element arranged to activate expression of the DNA sequence in response to DNA damage) to an agent; and monitoring the expression of the reporter protein from the cell. The invention also concerns expression cassettes, vectors and cells which may be used according to such a method and also modified media that may be employed in fluorscence assays and in preferred embodiments of the method of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to methods for detecting agents that cause or potentiate DNA damage, and to molecules and transfected cell lines that may be employed in such methods. In particular, the invention relates to biosensors for detecting DNA damage in human cell cultures.<br>
DNA damage is induced by a variety of agents such as ultraviolet light, X rays, free radicals, methylating agents and other mutagenic compounds. DNA damage can also be caused indirectly either by agents that affect enzymes and proteins which interact with DNA (including polymerases and topoisomerases) or by promutagens (agents that can be metabolised to become mutagenic). Any of these agents may cause damage to the DNA that comprises the genetic code of an organism and cause mutations in genes. In animals, such mutations can lead to carcinogenesis or may damage the gametes to give rise to congenital defects in offspring. Such DNA damaging agents can be collectively known as genotoxins.<br>
These DNA damaging agents may chemically modify the nucleotides that comprise DNA and may also break the phosphodiester bonds that link the nucleotides or disrupt association between bases (T-A or C-G). To counter the effect of these DNA damaging agents cells have evolved a number of mechanisms. For example, the SOS response in E. coli is a well-characterised cellular response induced by DNA damage in which a series of proteins are expressed, including DNA repair enzymes, which repair the damaged DNA. In mammalians, nucleotide excision repair and base excision repair mechanisms play a prominent role in DNA damage repair, and are the primary mechanism for removal of bulky DNA adducts and modified bases.<br>
There are numerous circumstances when it is important to identify what agents may cause or potentiate DNA damage. It is particularly important to detect agents that cause DNA damage when assessing whether it is safe to expose a person to these agents. For instance, a method of detecting these agents may be used as a genotoxicity assay for screening compounds that are candidate medicaments, food additives or cosmetics to assess whether or not the compound of interest induces DNA damage. Alternatively,<br><br>
methods of detecting DNA damaging agents may be used to monitor for contamination of water supplies with pollutants that contain mutagenic compounds.<br>
Various methods, such as the Ames Test, the in vitro micronucleus test and the mouse lymphoma assay (MLA), for determining the toxicity of an agent are known but are unsatisfactory for a number of reasons. For instance, incubation of samples can take many weeks, when it is often desirable to obtain genotoxic data in a shorter time frame. Furthermore, many known methods of detecting DNA damage (including the Ames Test and related methods) assay lasting DNA damage, as an endpoint, either in the form of mis-repaired DNA (mutations and recombinations) or unrepaired damage in the form of fragmented DNA. However, most DNA damage is repaired before such an endpoint can be measured and lasting DNA damage only occurs if the conditions are so severe that the repair mechanisms have been saturated.<br>
An improved genotoxic test is disclosed in WO 98/44149, which concerns recombinant DNA molecules comprising a Saccharomyces cerevisiaie regulatory element that activates gene expression in response to DNA damage operatively linked to a DNA sequence that encodes a light emitting reporter protein, such as Green Fluorescent Protein (GFP). Such DNA molecules may be used to transform a yeast cell for use in a genotoxic test for detecting for the presence of an agent that causes or potentiates DNA damage. The cells may be subjected to an agent and the expression of the light emitting reporter protein (GFP) from the cell indicates that the agent causes DNA damage. The genotoxic tests described in WO 98/44149 detect the induction of repair activity that can prevent an endpoint being reached. The method described in WO 98/44149 may therefore be used to detect for the presence of DNA damaging agents.<br>
US 6,344,324 discloses a recombinant DNA molecule comprising the regulatory element of the hamster GADD153 upstream promoter region that activates gene expression in response to a wide range of cellular stress conditions, linked to a DNA sequence that encodes GFP. This reporter system is carried out in a human head and neck squamous-cell carcinoma cell line. However, problems associated with this reporter system are that it requires at least a four day treatment period at test agent concentrations that result in less than 10% cell survival, followed by analysis of fluorescence by flow cytometry. In addition, the biological relevance of any gene induction when tested with agents at this<br><br>
level of toxicity is debatable. Furthermore, this development does not disclose a means of specifically monitoring for the presence of agents that may cause or potentiate DNA damage, and the mechanism ofGADD153 induction remains unclear. Hence, this system is of very limited use as a human DNA damage biosensor.<br>
Therefore, it is an aim of embodiments of the present invention to address problems associated with the prior art, and to provide an improved biosensor for detecting DNA damage in human cell cultures.<br>
According to a first aspect of the present invention, there is provided an expression cassette comprising a DNA sequence encoding a reporter protein, which DNA sequence is operatively linked to a human GADD45a gene promoter and a human GADD45a gene regulatory element arranged to activate expression of the DNA sequence in response to DNA damage.<br>
By the term "regulatory element", we mean a DNA sequence that regulates the transcription of a gene with which it is associated, i.e. the DNA sequence encoding the reporter protein.<br>
By the term "operatively linked", we mean that the regulatory element is able to induce the expression of the reporter protein.<br>
According to a second aspect of the invention, there is provided a recombinant vector comprising an expression cassette according to the first aspect.<br>
According to a third aspect of the invention, there is provided a cell containing a recombinant vector in accordance with the second aspect of the present invention.<br>
According to a fourth aspect of the present invention, there is provided a method of detecting for the presence of an agent that causes or potentiates DNA damage comprising subjecting a cell in accordance with the third aspect of the present invention to an agent; and monitoring the expression of the reporter protein from the cell.<br><br>
The method of the fourth aspect of the invention represents a novel cost-effective genotoxicity screen that may be used to provide a pre-regulatory screening assay for use by the pharmaceutical industry and in other applications where significant numbers of agents or compounds need to be tested. It provides a higher throughput and a lower compound consumption than existing in vitro and in vivo mammalian genotoxicity assays, and is sensitive to a broad spectrum of mutagens.<br>
The method of the fourth aspect of the invention is suitable for assessing whether or not an agent may cause DNA damage. It is particularly useful for detecting agents that cause DNA damage when assessing whether it is safe to expose a person to DNA damaging agents. For instance, the method may be used as a genotoxicity assay for screening whether or not known agents, such as candidate medicaments, pharmaceutical and industrial chemicals, pesticides, fungicides, foodstuffs or cosmetics, induce DNA damage. Alternatively, the method of the invention may be used to monitor for contamination of water supplies, leachates and effluents with pollutants containing DNA damaging agents.<br>
The method of the fourth aspect of the invention may also be used for assessing whether an agent may potentiate DNA damage. For example, certain agents can cause accumulation of DNA damage by inhibiting DNA repair (for instance by preventing expression or function of a repair protein) without directly inflicting DNA damage.<br>
Surprisingly, the use of a human GADD45a gene regulatory element in addition to the human GADD45a gene promoter in the expression cassette according to the first aspect of the invention radically enhances the response of the cassette to genotoxic stress and, hence, DNA damage in the cell according to the third aspect. Advantageously, the cassette can be analysed for expression of the reporter protein within or after only 24 hours simply by assaying for the protein in a test culture. The cells may be subjected to the test agent or compound, and expression of the reporter protein in the cell indicates whether the test agent causes DNA damage.<br>
The inventors have found that DNA encoding a human GADD45a gene promoter and a human GADD45a gene regulatory element may be operatively linked a reporter protein to form a cassette according to the first aspect of the invention and then<br><br>
advantageously used in a genotoxic test according to the fourth aspect of the invention. Such cassettes may comprise the whole of the GADD45a gene (including coding sequences) provided that it is operatively linked to DNA encoding a light emitting reporter. For instance cassettes may be made according to the first aspect of the invention comprising the whole of, or substantially all of, the GADD45a gene (comprising regulatory elements and promoter) with DNA encoding a reporter inserted 3' of the GADD45a promoter (e.g. within the GADD45a coding sequence or at the 3' of the coding sequence), end of the arranged to activate expression of the DNA sequence in response to DNA damage.<br>
Preferably, the human GADD45a gene promoter sequence induces RNA polymerase to bind to the DNA molecule and start transcribing the DNA encoding the reporter protein. It is preferred that the promoter sequence comprises the human GADD45a gene promoter sequence and the 5' untranslated region. The promoter sequence may be obtained from the pHG45-HC plasmid, which is illustrated in Figure 3. The promoter sequence can be seen in each of the expression cassettes in accordance with the invention in Figure 9, and the nucleotide sequence of the GADD45a gene promoter is shown as bases 4-2254 in SEQ ID No.s 1, 2, 3, 4 &amp; 5 in the sequence listing. It will be appreciated that the promoter may comprise each of the bases 4 -2254 or alternatively may be a functional derivative or functional fragment thereof. Functional derivatives and functional fragments may be readily identified be assessing whether or not transcriptase will bind to a putative promoter region and will then lead to the transcription of the marker protein. Alternatively such functional derivatives and fragments may be examined by conducting mutagenesis on the GADD45a promoter, when in natural association with theGADD45a gene, and assessing whether or not GADD45a expression may occur.<br>
The regulatory element in the expression cassette according to the invention may comprise sequences downstream of the GADD45a gene promoter sequence. The regulatory element may comprise functional DNA sequences such as those encoding translation initiation sequences for ribosome binding or DNA sequences that bind transcription factors which promote gene expression following DNA damage. The regulatory element in the expression cassette according to the invention may comprise at least one exon of the GADD45a gene. For example, the regulatory element may comprise Exon 1, Exon 2, Exon 3, and/or Exon 4 of the GADD45a gene, or at least a region thereof,<br><br>
or any combination thereof. Hence, the regulatory element may comprise any combination of the four exons of the GADD45a gene, or at least a region thereof.<br>
In a preferred embodiment, the regulatory element comprises at least a region of Exon 1 of the GADD45a gene, and preferably at least a region of Exon 3 of the GADD45a gene, and more preferably, at least a region of Exon 4 of the GADD45a gene. It is especially preferred that the regulatory element comprises all of Exon 1 of the GADD45a gene, and preferably at least a region of Exon 3 of the GADD45a gene, and more preferably, all of Exon 4 of the GADD45a gene.<br>
Preferred regulatory elements are illustrated in the each of the expression cassettes in accordance with the invention in Figure 9. The nucleotide sequence of Exon 3 of the GADD45a gene is shown as bases 3405-3642 in SEQ ID No.s 2, 3 &amp; 4 in the sequence listing. The nucleotide sequence of the preferred region of Exon 3 of the GADD45a gene is shown as bases 3503-3642 in SEQ ID No.4. The nucleotide sequence of Exon 4 of the GADD45a gene is shown as bases 4716-5391 in SEQ ID No.s 2, 3 &amp; 4 in the sequence listing.<br>
Alternatively, or additionally, the regulatory element may comprise a non-coding DNA sequence, for example, at least one intron of the GADD45a gene. For example, the regulatory element may comprise Intron 1, Intron 2, and/or Intron 3 of the GADD45a gene, or at least a region thereof, or any combination thereof. Hence, the regulatory element may comprise any combination of the three introns of the GADD45a gene, or at least a region thereof.<br>
In a preferred embodiment, the regulatory element in the expression cassette according to the invention comprises at least a region of Intron 3 of the GADD45a gene. Intron 3 of GADD45a is illustrated in Figure 9, and the nucleotide sequence of Intron 3 of the GADD45a gene is shown as bases 3643-4715 in SEQ ID No.s 2, 3 &amp; 4 in the sequence listing.<br>
In a preferred embodiment, the expression cassette in accordance with the invention comprises the promoter sequence of the GADD45a. gene and also gene regulatory elements found within Intron 3 of the genomic GADD45a gene sequence itself.<br><br>
While the inventors do not wish to be bound by any hypothesis, they believe that Intron 3 of the GADD45a gene, contains a putative p53 binding motif, and that it is this p53 motif which surprisingly enhances the response of the expression cassette to genotoxic stress. The putative p53 binding motif is shown as nucleotide bases 3830-3849 of SEQ ID No.s 2, 3 and 4.<br>
The inventors also believe that Intron 3 of the GADD45a. gene may contain a putative TRE motif, which may encode a AP-1 binding site. The putative TRE motif is shown as nucleotide bases 3879-3885 of SEQ ID No.s 2, 3 and 4. Hence, while the inventors do not wish to be bound by any hypothesis, they postulate that this putative AP-1 binding site may also contribute to the improved response to genotoxic agents.<br>
It is preferred that the expression cassette comprises at least the p53 binding motif and/or the AP-1 binding motif from Intron 3 of the GADD45a gene.<br>
The regulatory element may comprise a 3' untranslated (UTR) region of the GADD45a gene, the nucleotide sequence of which is shown as bases 4830-5391 in SEQ ID No.s 2, 3, and 4. While the inventors do not wish to be bound by any hypothesis, they believe that this 3' UTR may be involved with stabilisation of mRNA cassette, and hence, may be surprisingly important when used with the rest of the regulatory element, such as Intron 3.<br>
By the term "reporter protein", we mean a protein which when expressed in response to the regulatory element of the DNA molecule of the invention, is detectable by means of a suitable assay procedure. Preferably, the reporter protein comprises a light emitting reporter protein. The DNA sequence that encodes a light emitting reporter protein may code for any light emitting protein, for example Luciferase or Green Fluorescent Protein. However, it is preferred that the DNA sequence codes for a protein that is substantially fluorescent.<br>
Preferred DNA sequences that encode a light emitting reporter protein code for Green Fluorescent Protein (GFP), and light emitting derivatives thereof. GFP is from the jelly fish Aequorea victoria and is able to absorb blue light and re-emits an easily detectable green light and is thus suitable as a reporter protein. GFP may be<br><br>
advantageously used as a reporter protein because its measurement is simple and reagent free and the protein is non-toxic.<br>
Derivatives of GFP include DNA sequences encoding for polypeptide analogues or polypeptide fragments of GFP, which are able to emit light. Many of these derivatives absorb and re-emit light at wavelengths different to GFP found endogenously in Aequorea victoria.<br>
For instance, preferred expression cassettes according to the first aspect of the invention comprise a DNA sequence that encodes the human enhanced GFP (hEGFP), also known as GFPmutl variant of GFP. This consists of the GFP wild type containing a double amino acid substitution of Phe-64 to Leu, and Ser-65 to Thr. The coding sequence of the hEGFP gene also contains more than 190 silent base changes, which correspond to human codon usage preferences relative to the native Aequorea victoria sequence. This is particularly advantageous for use in a human DNA damage reporter system. The hEGFP sequence may be obtained from the pEGFP-Nl plasmid (e.g. obtained from Clontech), which is shown in Figure 1. The hEGFP sequence can be seen in each of the expression cassettes in accordance with the invention in Figure 9, and the nucleotide sequence of the hEGFP is shown as bases 2550-3278 in SEQ ID No.s 1, 2, 3, 4 &amp; 5 in the sequence listing.<br>
GFP produces a good quantum yield of fluorescence and matches the output of argon ion lasers used in fluorescence activated cell sorters. Cells according to the third aspect of the invention, which contain DNA molecules coding hEGFP, may be used according to the method of the fourth aspect of the invention.<br>
Hence, preferred expression cassettes according to the invention comprise a human GADD45a gene regulatory element and human GADD45a gene promoter operatively linked to a DNA sequence encoding a GFP, or light emitting derivative or analogues thereof (e.g. YFP etc). Most preferred expression cassettes comprise a human GADD45a gene promoter operatively linked to a DNA sequence encoding a GFP or light emitting derivative thereof, and Intron 3 of the GADD45a gene.<br><br>
In a first embodiment, the expression cassette according to the first aspect is preferably 5-31, (the cassette illustrated in Figure 9 as GD531). The nucleotide sequence of expression cassette 5-31 is shown in the sequence listing as SEQ ID No.2.<br>
In a second embodiment, the expression cassette according to the first aspect is preferably 5-32 (the cassette illustrated in Figure 9 as GD532). The nucleotide sequence of expression cassette 5-32 is shown in the sequence listing as SEQ ID No.3.<br>
In a third embodiment, the expression cassette according to the first aspect is preferably 5-33 (the cassette illustrated in Figure 9 as GD533) The nucleotide sequence of expression cassette 5-33 is shown in the sequence listing as SEQ ID No.4.<br>
The recombinant vector according to the second aspect of the present invention may for example be a plasmid, cosmid or phage. Such recombinant vectors are of great utility when replicating the expression cassette. Furthermore, recombinant vectors are highly useful for transfecting cells with the expression cassette, and may also promote expression of the reporter protein.<br>
Recombinant vectors may be designed such that the vector will autonomously replicate in the cytosol of the cell or can be used to integrate into the genome. In this case, elements that induce DNA replication may be required in the recombinant vector. Suitable elements are well known in the art, and for example, may be derived from pCEP4 (Invitrogen, 3 Fountain Drive, Inchinnan Business Park, Paisley, PA4 9RF, UK) pEGFP-Nl (BD Biosciences Clontech UK, 21 In Between Towns Road, Cowley, Oxford, OX4 LY, United Kingdom) or pCI and pSI (Promega UK ltd, Delta house, chilworth Science Park, Southampton SO16 7NS, UK).<br>
Such replicating vectors can give rise to multiple copies of the DNA molecule in a transformant and are therefore useful when over-expression (and thereby increased light emission) of the reporter protein is required. In addition, it is preferable that the vector is able to replicate in human, primate and/or canine cells. It is preferred that the vector comprises an origin of replication, and preferably, at least one selectable marker. The selectable marker may confer resistance to an antibiotic, for example, hygromycin or neomycin. Hence, a suitable element is derived from the pCEP4 plasmid (Invitrogen, 3<br><br>
Fountain Drive, Inchinnan Business Park, Paisley, PA4 9RF, UK), which is illustrated in Figure 2.<br>
In a first embodiment, the recombinant vector according to the second aspect is preferably pEP-GD531, as illustrated in Figure 5.<br>
In a second embodiment, the recombinant vector according to the second aspect is preferably pEP-GD532, as illustrated in Figure 6.<br>
In a third embodiment, the recombinant vector according to the second aspect is preferably pEP-GD533, as illustrated in Figure 7.<br>
According to the second aspect of the invention the recombinant vector is incorporated within a cell. It is preferred that the cell is eukaryotic. Such host cells may be mammalian derived cells and cell lines. Preferred mammalian cells include human, primate, murine or canine cells. The host cells may be lymphoma cells or cell lines, such as mouse lymphoma cells. The host cells may be immortalised, for example, lymphocytes.<br>
Preferred host cells are human cell lines. Preferably, the host cells are human cell lines having a fully functional p53, for example, ML-1 (a human myeloid leukaemia cell line with wild-type p53), TK6 (a human lymphoblastoid cell line with wild-type p53). However, host cell lines of WI-L2-NS and WTKl (both of which are sister lines of TK6 and have mutant p53 proteins) are also envisaged. These cell lines may all be found at the European Collection of Cell Cultures (ECACC General Office, CAMR, Porton Down, Salisbury, Wiltshire, SP4 OJG, United Kingdom).<br>
The inventors have found that TK6 human cells are particularly preferred cell lines for use according to the method of the invention. While the inventors do not wish to be bound by any hypothesis, they believe that TK6 cells are most useful because they have a fully functional p53.<br>
Host cells used for expression of the protein encoded by the DNA molecule are ideally stably transfected, although the use of unstably transfected (transient) cells is not precluded.<br><br>
Transfected cells according to the third aspect of the invention may be formed by following procedures described in the Example. The cell is ideally a human cell line, for example TK6. Such transfected cells may be used according to the method of the fourth aspect of the invention to assess whether or not agents induce or potentiate DNA damage. GFP expression is induced in response to DNA damage and the light emitted by GFP may be easily measured using known appropriate techniques.<br>
Most preferred cells according to the third aspect of the invention are TK6 cells transformed with the vector pEP-GAD532. These cells are referred to herein as GenTK-T01.<br>
It is also envisaged that the expression cassette according to the invention may be integrated into the genome of a host cell. The skilled technician will appreciate suitable methods for integrating the cassette into the genome. For example, the expression cassette may be harboured on a retroviral vector, which in combination with a packaging cell line may produce helper-free recombinant retrovirus, which may then be introduced into the host cell. The cassette may then integrate itself into the genome. Examples of suitable helper-free retroviral vector systems include the pBabePuro plasmid with the BING retroviral packaging cell line [Kinsella and Nolan, 1996. Episomal Vectors Rapidly and Stably Produce High-Titer Recombinant Retroviruses. Human Gene Therapy. 7:1405-1413.]<br>
The method of the fourth aspect of the invention is particularly useful for detecting agents that induce DNA damage at low concentrations. The methods may be used to screen compounds, such as candidate medicaments, food additives or cosmetics, to assess whether it is safe to expose a living organism, particularly people, to such compounds. Alternatively, the method of the fourth aspect of the invention may be employed to detect whether or not water supplies are contaminated by DNA damaging agents or agents that potentiate DNA damage. For instance, the methods may be used to monitor industrial effluents for the presence of pollutants that may lead to increased DNA damage in people or other organisms exposed to the pollution.<br><br>
The method of the invention is preferably performed by growing cells transfected with a recombinant vector according to the second aspect of the invention (such as pEP-GD531, pEP-GD532, or pEP-GD533), incubating the cells with the agent which putatively causes DNA damage for a predetermined time and monitoring the expression of the light emitting reporter protein directly from a sample of the cells.<br>
Cells are preferably grown in low fluorescence growth medium. This can obviate the need to wash the cells before measurements are made and therefore reduce the number of steps in the method further. For instance, preferred human cells according to the third aspect of the invention may be grown in a low fluorescent media. Suitable media may be a modified RPMI 1640 media and most preferably contains either low concentrations of, or preferably, no riboflavin or phenol red. The inventors have found that the use of such media is surprisingly useful in assays of fluorescent markers such as GFP. The use of such media reduces the "signal to noise ratio" when measuring fluorescence. Therefore, according to a fifth aspect of the invention there is provided a media that does not contain, or contains reduced levels (relative to standard cell culture media) of riboflavin or phenol red for use in fluorescent assays. Most preferably the media contains no riboflavin and no phenol red.<br>
It is preferred that the media is modified such that it does not contain riboflavin. The use of such a modified media would be counter-intuitive to a skilled person because it would be understood that riboflavin is usually required in media. This is because most cells require riboflavin to be able to grow and survive. The inventors have realised that cells, which have been grown in conventional growth media containing fiboflavin, may be removed from such media; placed in media without riboflavin; and will remain viable for the purposes of conducting a reliable fluorescence assay. Cells left in conventional media (containing riboflavin and/or phenol red) tend to provide less satisfactory fluorescence values when tested according to the method of the fourth aspect of the invention (particularly when the reporter protein is GFP or a derivative thereof).<br>
It is preferred that media is used according to the fifth aspect of the invention when conducting assays according to the fourth aspect of the invention. It is most preferred such media is used when performing a genotoxic test with GFP as the reporter.<br><br>
A preferred media with low autofluorescence is a modified RPMI 1640 media that does not contain riboflavin or Phenol red. A most preferred media is disclosed in table 1 of Example 1.<br>
Other media that could be modified for low autofluorescence include Dulbecco's Modified Eagle media (D-MEM), F-10 Nutrient mixture (Ham), F-12 Nutrient mixture (Ham), and Fischer's medium (GIBCO, Invitrogen, Paisley, UK). The most appropriate medium may be used for different cell types that could be used as a host for recombinant reporter vectors.<br>
According to a preferred embodiment of the method of the invention TK6 cells may be transfected with pEP-GD531, pEP-GD532, or pEP-GD533, and grown in RPMI medium containing no riboflavin. A putative DNA damaging agent (e.g. a food additive or potential medicament or an agent contained within a water sample or effluent sample) may then be added to the medium containing the cells. The cells are then allowed to grow for a defined period of time after which a sample of the cells is removed and fluorescence measured therefrom.<br>
Suitable methods of fluorescence detection and quantitation will be known to the skilled technician, and a method is described in the Examples. A preferred method of fluorescence detection is described in US 6, 509, 161. This method is particularly useful when the light emitting report is GFP or a derivative thereof.<br>
According to a preferred embodiment of the method of the invention, fluorescence and absorbance readings may be recorded from TK6 cells transfected with pEP-GD531, pEP-GD532, or pEP-GD533, for example, from the well of a microplate. An example of a suitable microplate is a 96 well, black, clear-bottomed microplates, manufactured by Matrix ScreenMates, Cat. No. 4929, Apogent Discoveries, USA, or Corning (BV, Netherlands: Cat. No. 3651). Fluorescence and absorbance measurements may be recorded using a suitable microplate reader, for example, the Tecan Ultra-384 (Tecan UK Ltd.) microplate reader: excitation 485 nm / emission 535 nm with an additional dichroic mirror (reflectance 320 nm - 500 nm, transmission 520 nm - 800 nm). Absorbance may be measured through a suitable filter, for example, a 620 nm filter.<br><br>
Most preferred protocols for conducting the method of the fourth aspect of the invention are described in Examples 1, 3 and 4.<br>
From the fluorescence and absorbance measurements, "brightness values" and fluorescence induction ratios for cells according to the third aspect of the invention can be made as described in the examples.<br>
There is background ("constitutive") expression of GFP from the GADD45a-EGFP constructs, thus the higher the cell density, the more fluorescent the culture. In order to correct for fluorescence increase that is consequent on growth, the fluorescence data (GFP) are divided by absorbance data (cell density) to give 'brightness units', i.e. the measure of average fluorescence per cell. This is independent of culture density. Accordingly, measurement of absorbance may be used primarily for normalisation of fluorescence signals rather than a measurement of the toxicity of the DNA damaging agent. Accordingly, it is envisaged that a secondary assay may be used in conjunction with the absorbance measurement in order to determine toxicity via cell viability and apoptosis. For example, using the Biovison Bioluminescence Cytotoxicity Assay (Biovison Incoperated, 2455-D Old Middlefield Way, Mountain View, California 94043, USA), or the Vybrant® Apoptosis Assay Kit (Molecular probes Inc., 29851 Willow Creek Road, Eugene, OR 97402, USA).<br>
Occasionally, compounds show fluorescence themselves or induce cellular auto-fluorescence. Accordingly, when evaluating increased GFP expression the brightness values of a control cell line is subtracted from the brightness values of a reporter cell line. This removes the interference from the data. It is therefore preferred that methods according to the fourth aspect of the invention employ cells containing control vectors based upon reporter vectors according to the second aspect of the invention. Such control vectors may correspond to the reporter vector, but with a non-sense mutation introduced into the light emitting reporter gene.<br>
The recombinant control vector may preferably be the vectors described as pEP-GD500C or pEP-GD532C, as illustrated in Figures 18 and 19 respectively.<br><br>
Preferred methods according to the fourth aspect of the invention will utilise cells according to the third aspect of the invention (e.g. GenTK-TOl) amd a control cell line based upon TK6 cells transformed with the vector pEP-GD532C (referred to herein asGenTK-COl).<br>
It will be appreciated that some non-genotoxic compounds can be chemically altered by cellular metabolism. In mammals this process is often called metabolic activation (MA). MA can convert certina non-genotoxic compounds (for example promutagens) into genotoxic compounds. Most frequently MA occurs in the liver. For this reason it is often preferred that genotoxicity tests are adapted such that assays of test compound are carried out in the presence and absence of liver extracts that are capable of metabolising a compound is if it were being metabolised in vivo. Example 4 illustrates a preferred method according to the fourth aspect of the invention which utilises a liver extract (known to the skilled person) called S9. Inclusion of such an extract allows assays to detect compounds that only become genotoxic after passage through the liver.<br>
All of the features described herein (including any accompanying claims, abstract and drawings), and/or all of the steps of any method or process so disclosed, may be combined with any of the above aspects in any combination, except combinations where at least some of such features and/or steps are mutually exclusive.<br>
Embodiments of the invention will now be further described, by way of example only, with reference to the following Examples and Figures in which:-<br>
Figure 1 shows a restriction map of vector pEGFP-Nl; Figure 2 shows a restriction map of vector pCEP4; Figure 3 shows a restriction map of vector pHG45-HC; Figure 4 shows a restriction map of vector pEP-GD500;<br>
Figure 5 shows a restriction map of vector pEP-GD531, a preferred recombinant vector according to the present invention;<br><br>
Figure 6 shows a restriction map of vector pEP-GD532, a preferred recombinant vector according to the present invention;<br>
Figure 7 shows a restriction map of vector pEP-GD533, a preferred recombinant vector according to the present invention;<br>
Figure 8 shows a restriction map of vector pEP-GD534;<br>
Figure 9 shows embodiments of reporter expression cassettes according to the invention;<br>
Figure 10 shows the effect of MMS induction on GADD45a reporter cell lines;<br>
Figure 11 shows the effect on brightness by MMS induction on GADD45a reporter cell lines;<br>
Figure 12 shows methyl methanesulphonate induced fluorescence measured directly from a reporter cell line cultured in either RPMI 1640 with serum (A) or modified low autofiuorescence medium with serum (B) as discussed in Example 2;<br>
Figure 13 shows the growth of a human lymphoblastoid cell line in RPMI 1640 or modified low autofiuorescence medium in Example 2;<br>
Figure 14 shows the effect of MMS on relative cell density in a 96 well plate assay after 48 hours incubation in Example 3;<br>
Figure 15 shows the effect of MMS on florescence induction ratios in a 96 well plate assay in Example 3;<br>
Figure 16 shows the effect of Cyclophosphamide with and without S9 on the relative cell densitys of GenTK-COl cells in Example 4;<br>
Figure 17 shows the effect of Cyclophosphamide with and without S9 on the fluorescence induction in GenTK-COl cells in Example 4;<br><br>
Figure 18 shows a restriction map of a control vector named pEP-GD500C; and Figure 19 shows a restriction map of a control vector named pEP-GD532C.<br>
Example 1<br>
The following Example outlines the components that have been used in the construction of a series of GADD45a-EGFP Human cell culture DNA damage biosensors/reporters according to the first and second aspects of the invention. In addition, this Example describes the construction of the biosensors (cells according to the third aspect of the invention), and illustrates how they respond to a genotoxic agent, for example, methanesulphonic acid methyl ester (MMS) when utilised in a method according to the fourth aspect of the invention.<br>
1.1 System Components:-<br>
(H The Promoter - From GADD45a<br>
The source of the GADD45a promotor is the plasmid pHG45-HC from Albert J. Fornace Jr., M.D. (National Cancer Institute, Building: 37, Room: 6144, National Institutes of Health, Bethesda, Maryland 20892, USA), which is illustrated in Figure 3.<br>
(ii) The Fluorescent Protein - Clontech's EGFP<br>
The source of the Green fluorescent protein is the Clontech plasmid pEGFP-Nl (BD Biosciences Clontech UK, 21 In Between Towns Road, Cowley, Oxford, OX4 LY, United Kingdom), which is illustrated in Figure 1. pEGFP-Nl encodes a red-shifted variant of wild-type GFP that has been optimised for brighter fluorescence and higher expression in mammalian cells. (Excitation maximum = 488 nm; emission maximum = 507 nm.) The plasmid pEGFP-Nl encodes the GFPmutl variant, which contains the double-amino-acid substitution of Phe-64 to Leu and Ser-65 to Thr. The coding sequence of the EGFP gene contains more than 190 silent base changes, which correspond to human codon-usage preferences.<br><br>
1.2 Construction of Biosensor Reporter cassettes<br>
The construction of a set of recombinant vectors each containing a GADD45a-EGFP reporter cassette was carried out in several stages using the Invitrogen pCEP4 plasmid as a backbone, which is illustrated in Figure 2. Plasmid pCEP4 is an episomal mammalian expression vector that uses the cytomegalovirus (CMV) immediate early enhancer/promoter for high level transcription of recombinant genes inserted into the multiple cloning site. The Epstein-Barr Virus replication origin (oriP) and nuclear antigen (encoded by the EBNA-1 gene) is carried by this plasmid to permit extrachromosomal replication in human, primate and canine cells. pCEP4 also carries the hygromycin B resistance gene for stable selection in transfected cells.<br>
Step 1: (Insertion of an EGFP module into pCEP4) - An EGFP module (consisting of the Human optimised Green Fluorescent Protein and the SV40 poly adenylation signal) was amplified by PCR from the Clontech plasmid pEGFP-N\ (BD Biosciences Clontech UK, 21 In Between Towns Road, Cowley, Oxford, OX4 LY, United Kingdom), which is shown in Figure 1. The EGFP gene is flanked by Bgl \\-Asc I (5') and Pac l-XhoIQ') sites. This was inserted into Bgl II- Xho I cut pCEP4 (Invitrogen), which is shown in Figure 2. This step also resulted in the removal of the CMV promoter from pCEP4.<br>
Step 2: (Insertion of GADD45a promoter) - The GADD45a 5' promoter sequence was PCR amplified (from a plasmid pHG45-HC, kindly donated by Albert J. Fornace Jr., M.D. (National Cancer Institute, Building: 37, Room: 6144, National Institutes of Health, Bethesda, Maryland 20892, USA), illustrated in Figure 3, from -2253bp up to and including the 5' UTR (untranslated region) and start codon (+298bp) with flanking Bgl II (5') and AscI I (3') sites. This was inserted into the plasmid resulting from step 1. The resultant plasmid was named pEP-GD500, which is shown in Figure 4.<br>
Step 3: (Insertion of GADD45a gene sequence) - Various GADD45a gene sequences (3' end of gene) were amplified with cloning sites by PCR and cloned into the plasmid resulting from step 2 (pEP-GD500) to give four further reporter plasmids.<br>
These plasmids were named:-<br>
(i) pEP-GD531 (3' sequence from +1034 to +3149) - as shown in Figure 5. This plasmid<br>
includes all of Intron 3 of the Human GADD45a gene, including the p53 binding motif;<br><br>
(ii) pEP-GD532 (3' sequence from +1147 to +3149) - as shown in Figure 6. This plasmid includes all of Intron 3 of the Human GADD45agene, including the p53 binding motif; (iii) pEP-GD533 (3' sequence from +1248 to +3149) - as shown in Figure 7. This plasmid includes all of Intron 3 of the Human GADD45a gene, including the p53 binding motif; and<br>
(iv) pEP-GD534 (3' sequence from +3155 to +3149) - as shown in Figure 8. This plasmid does not include all of Intron 3 of the Human GADD45a gene, and does not include the p53 binding motif. This plasmid includes the GADD45a3' untranslated region.<br>
Cloning sites used for step 3:-<br>
pEP-GD531: PCR product flanked by Not I (5s) and Xho I (3'). Cloned into Not I-Xhol cut pEP-GD500.<br>
pEP-GD532: PCR product flanked by EcoRV (5') and Xho I (3'). Cloned into Hpa I -Xho I cut pEP-GD500.<br>
pEP-GD533: PCR product flanked by Pac I (5') andXho I (3'). Cloned intoPac I-Xho I cut pEP-GD500.<br>
pEP-GD534: PCR product flanked by Not I (5') and Xho I (3'). Cloned into Not I -Xho I cut pEP-GD500.<br>
Step 4: (creation of control vectors) - Reporter plasmids were digested with Asc I and the 5' overhangs filled in with Klenow enzyme. The modified DNA fragments were then recirculised. This introduced a 4 bp frameshift in the EGFP gene. The resultant protein is non-functional. The resultant control plasmids are suffixed with the letter C and illustrated in Figures 18 and 19.<br>
The DNA sequences of each of the expression cassettes in the plasmids pEP-GD500, pEP-GD531, pEP-GD532, pEP-GD533, and pEP-GD534 are shown in the sequence listing as SEQ ID No.l, SEQ ID No.2, SEQ ID No.3, SEQ ID No.4, and SEQ ID No.5, respectively.<br><br>
Referring to Figure 9, there are shown the series of reporter cassettes in accordance with the invention each aligned underneath the Human GADD45a gene. The GADD45a -EGFP reporter gene fusions of plasmids pEP-GD531, pEP-GD532, and pEP-GD533 were constructed in such a way that the overall length of the GADD45a -EGFP fusion is identical to the native GADD45a gene, as shown in Figure 9. In plasmids pEP-GD531 and pEP-GD532, the GADD45a DNA downstream of the hEGFP is transcribed into mRNA as well as the hEGFP itself, and is subject to splicing. In plasmid pEP-GD533, only the hEGFP gene is transcribed into mRNA. The transcription is stopped at the SV40 polyadenylation signal before the GADD45a gene sequence. These plasmids are in accordance with the first aspect of the invention. The pEP-GD534 cassette is much shorter and does not include the p53 motif in Intron 3 of the GADD45a gene.<br>
1.3 Cell line selection and transfection<br>
When considering a cell line as a host for a new mammalian DNA-damage reporter system, it was important to consider the ability of the chosen cell line to respond to genotoxic damage in a way that would allow the required response of the reporter. The p53 status and effects on downstream genes of two cells lines commonly used in molecular biology and/or genetic toxicology were explored and this work is described herein. These cell lines were:- (i) TK6, a human lymphoblastoid cell line (wt p53); and a sister line of TK6, (ii) WI-L2-NS (mutant p53 protein).<br>
A dysfunctional p53 protein can lead to increased radioresistance (i.e. increased survival) and an increased sensitivity to mutagenic and clastogenic effects of radiation. While these properties make a cell line suitable for detecting mutation endpoints, the same properties imply that the induction of surrogate biomarkers for DNA damage in a cell line with a mutant p53 status would be at best poor. To this end, the TK6 cell line was chosen for the biosensor (i.e. a cell according to the third aspect of the invention). It is believed that additional data of scientific interest may be gained from the use of its sister cell lines with the recombinant GADD45a -EGFP reporter vector.<br>
TK6 cells are transfected with the vectors by electroporation using a method adapted from Xia and Liber [Methods in Molecular biology, Vol.48: Animal Cell Electroporation and Electrofusion Protocols, 1995. Edited by J.A. Nickoloff. Humana<br><br>
Press Inc., Totowa, NJ, USA, Pages 151-160], and clones bearing the reporter plasmids are selected over two weeks with Hygromycin B.<br>
TK6 cells transformed with pEP-GAD532 are referred to herin as GenTK-TOl. An equivalent control cell line, termed GenTK-COl, was also produced in which the GFPreporter gene was deliberately placed out of frame. Accordingly GFP could not be expressed from this control cell line.<br>
1.4 The Assay media - modified RPMI<br>
Conventional media may be used in the methods of the invention. However the inventors have found that many traditional cell culture media are autofluorescent at the wavelengths associated with EGFP excitation (488nm max) and emission (520nm max). In this work, the major contributing factor of this fluorescence was found to be the vitamin riboflavin. It is therefore preferred that a modified low autofluorescent media is used.<br>
A number of such media may be produced and, by way of example, the inventors developed a modified low autofluorescent RPMI 1640 media (see Table 1). This media represents a preferred media for use according to the fifth aspect of the invention that does not contain any riboflavin, and may be used with the biosensor, to facilitate direct fluorescence reading of biosensor cultures.<br><br>
Table 1: working concentrations of components of a low autofluorescent mammalian cell culture media.<br><br>
INORGANIC SALTS:<br>
Ca(N03)2 • 4H20<br>
KC1<br>
MgSO4(anhyd.)<br>
NaCl<br>
NaHCO3<br>
Na2HPO4(anhyd.)<br>
OTHER COMPONENTS:<br>
D-Glucose Glutathione (reduced)<br>
AMINO ACIDS:<br>
L-Arginine HC1<br>
L-Asparagine (free base)<br>
L-Aspartic Acid<br>
L-Cystine-2HCl<br>
L-Glutamic Acid<br>
Glycine<br>
L-Histidine (free base)<br>
L-Hydroxyproline<br>
L-lsoleucine<br>
L-Leucine<br>
L-Lysine • HC1<br>
L-Methionine<br>
L-Phenylalanine<br>
L-Proline<br>
L-Serine<br>
L-Threonine<br>
L-Tryptophan<br>
L-Tyrosine (disodium salt)<br>
L-Valine<br>
VITAMINS:<br>
D-Biotin<br>
D-Ca Pantothenate<br>
Choline Chloride<br>
Folic Acid<br>
i-lnositol<br>
Nicotinamide<br>
Para-aminobenzoic Acid<br>
Pyridoxal HC1<br>
Thiamine HC1<br>
Vitamin B[2	<br><br>
100.00 400.00 48.84 6000.00 2000.00 800.00<br>
2000.00 1.00<br>
241.86 50.00 20.00 65.20 20.00 10.00 15.00 20.00 50.00 50.00 40.00 15.00 15.00 20.00 30.00 20.00 5.00 28.83 20.00<br>
0.20 0.25 3.00 1.00 35.00 1.00 1.00 1.00 1.00 0.005<br><br>
1.5	The method of Analysis - Microplate reader<br>
Fluorescence and absorbance data was collected from microplates using the Tecan Ultra-384 (Tecan UK Ltd.) microplate reader: excitation 485 nm / emission 535 nm with an additional dichroic mirror (reflectance 320 nm - 500 nm, transmission 520 nm - 800 nm). Absorbance was measured through a 620 nm filter. The data were transported into a Microsoft Excel spreadsheet, and converted to graphical data. The absorbance data give an indication of reduction in proliferative potential and these data were normalised to the untreated control (=100% growth). Fluorescence data were divided by absorbance data to give 'brightness units', the measure of average GFP induction per cell. These data were normalised to the untreated control (=1). In this way, one can distinguish between a small number of highly fluorescent cells and a large number of weakly fluorescent cells. In order to correct for induced cellular autofluorescence and intrinsic compound fluorescence, the brightness values for the untransfected TK6 cell line were subtracted from those of the five cell lines bearing a reporter plasmid. This makes visual assessment of the data more reliable. All the data were checked with and without this correction. From this "brightness" data the fluorescence induction ratios were calculated.<br>
1.6	MMS Induction of the Biosensor<br>
As an example of reporter induction, six cell lines (TK6, without a plasmid, and bearing one of the five reporter plasmids) were treated with varying concentrations of MMS. MMS is known to those skilled in he art as a genotoxic and cytotoxic agent and may be used as a positive control when using the method of the invention.<br>
IxlO6 cells in 1 ml of modified RPMI 1640 media (see 1.4 above)+ 10% horse Serum were incubated overnight (37°C, 5% CO2, 100% humidity) with Oug/ml, 6.25ug/ml, 12.5ug/ml, and 25ng/ml MMS.<br>
After incubation, the cells were washed in PBS, resuspended in 300ul PBS, and then split into two wells of a 96 well, black, clear-bottomed microplates. For example Matrix ScreenMates, Cat. No. 4929, Apogent Discoveries, USA, or Corning (BV, Netherlands: Cat. No.3651). Fluorescence and absorbance readings were recorded and fluorescence induction ratios were calculated.<br><br>
This was repeated three times and the average inductions and standard errors were calculated. The results are illustrated in Figure 10.<br>
Referring to Figure 11, there is shown unprocessed brightness values from the five reporter cell lines and the parent untransfected cell line TK6. Brightness values were calculated for each cell line-MMS combination by measuring the total GFP fluorescence and then normalising this with absorbance at 620nm. Hence, it will be appreciated that each of the plasmids pEP-GD531, pEP-GD532, and pEP-GD533 show an increase in background fluorescence as compared to the untransfected parent cell line TK6, and these cell lines showed a substantial degree of dose dependent increase in fluorescence in the human cell line TK6 when treated with MMS. In contrast while the cell lines bearing the plasmid pEP-GD500 or pEP-GD534 show an increase in background fluorescence as compared to TK6, this fluorescence is not further induced by treatment with MMS. While the inventors do not wish to be bound by any hypothesis, they believe that the addition of the GADD45a 3' untranslated region in plasmid pEP-GD534 stabilises the resultant mRNA, leading to a higher background level of fluorescence in cell lines bearing this plasmid as compared to cell lines bearing the plasmid pEP-GD500.<br>
In conclusion, the present invention uses the regulatory sequences of the human DNA-Damage inducible gene GADD45a to control the production of EGFP in the p53 wild-type human lymphoblastiod cell line TK6. The human gene GADD45a stimulates DNA excision repair in vitro and inhibits entry of cells into S phase. The reporters in accordance with the invention not only use the upstream promoter of the GADD45a gene, but also incorporate the gene regulatory elements found within the genomic gene sequence itself. This includes a putative p53 binding site in the third Intron of the GADD45a gene. Surprisingly, the use of these sequences enhances the response of the biosensors to genotoxic stress. This reporter system can be analysed for fluorescence induction after only 24 hours by simply measuring the fluorescence of the test culture. The cells may be subjected to an agent, and the expression of the light emitting reporter protein (GFP) from the cell indicates that the agents cause DNA damage. It is believed that this biosensor has applications in short term pre-regulatory drug screening.<br><br>
Example 2<br>
Further experiments were conducted to evaluate the effects of autofluorescence caused by conventional media in order that preferred media may be developed for use in the method according to the fourth aspect of the invention.<br>
The constructs, cells and protocols described in Example 1 were used unless indicated to the contrary.<br>
As-well-as riboflavin (see 1.4) the inventors established that a second major contribution factor to cell culture media autofluorescence was phenol red (a pH indicator dye commonly used in culture media). The fluorescence of phenol red also varies with pH. Therefore it is preferred that media used according to the invention is formulated as a modified low autofluorescence medium whereby phenol red or riboflavin, but preferably both phenol red and riboflavin, is omitted (see table 1).<br>
Referring to figure 12, the use of conventional RPMI 1640 medium decreased fluorescence signal to noise ratios, and increased variability of fluorescence measurements. Accordingly the use of normal RPMI 1640 media was not optimal for direct fluorescence readings. In contrast, the use of a modified low autofluorescent cell culture media (corresponding to normal media save riboflavain and phenol red was omitted) facilitated the direct measurement of genotoxin induced fluorescence readings.<br>
Referring to figure 13, the omission of both riboflavin and phenol red from media was found to result in a low autofluorescence medium that did not affect cell growth over 96 hours, and four cell divisions. Accordingly such a media is a preferred media for use according to the fifth aspect of the invention.<br>
It will therefore be appreciated that it is preferred that media used in the method according to the fourth aspect of the invention may comprise conventional media (e.g. RPMI 1640) but is preferably modified, according to the fifth aspect of the invention, such that it does not contain Phenol Red or riboflavin.<br><br>
Example 3<br>
Further development work was conducted to develop a most preferred protocol for conducting the method according to the fourth aspect of the invention in 96 well plates.<br>
The constructs, cells and protocols described in Example 1 were used unless indicated to the contrary.<br>
A preferred method of testing for DNA damage according to the fourth aspect of the invention comprises the steps of : (1) preparing a microplate for use in an assay; (2) conducting the assay in the microplates; (3) collecting and analysing the data; and (4) making a judgment on DNA damage and the consequences.<br>
Details of steps (1) - (4) for conducting a most preferred test for DNA damage are given below.<br>
3.1	Microplate preparation.<br>
Assays were carried out in 96 well, black, clear-bottomed microplates. For example Matrix ScreenMates, Cat. No. 4929, Apogent Discoveries, USA, or Corning (BV, Netherlands: Cat. No. 3651). A number of alternative microplates were assessed, though the variable background absorbance and fluorescence both within and between plates from individual manufacturers were found to be variable and in several cases unacceptable. It will therefore be appreciated that microplates used according to the invention preferably have consistent absorbance and fluorescence between plates and batches thereof. It will.be appreciated that a skilled person should give careful consideration when selecting a suitable microplate.<br>
The assay plates can be filled using a liquid handling robot. For example the MicroLabS single probe, from Hamilton GB Ltd., Birmingham or a Genesis 8-probe robot (Tecan UK Ltd. Theale. UK). Microplates can also be filled rapidly and effectively using a multi-channel pipette.<br>
3.2	Assay<br>
The following standard protocol may be followed. A 2 mM or lOOOug/ml (whichever is lowest) stock of a test chemical, or sample containing an agent that<br><br>
putatively caused DNA damage, was prepared in 2% v/v aqueous DMSO, and used to make 2 identical dilution series across a 96 well microplate and a 'control' (see below). To achieve this, 150 microlitres of the test chemical solution were put into 2 microplate wells. Each sample was serially diluted by transferring 75 microlitres into 75 microlitres of 2% DMSO, mixing, and then taking 75 microlitres out and into the next well. This produced 9 serial dilutions of 75 microlitres each. The final top concentration of test chemical/sample is 1 mM or 500 fig/ml on the microplate.<br>
Controls were added as follows:<br>
a.	Test compound/ sample containing agent in assay medium alone, to provide<br>
information on compound absorbance/fluorescence<br>
b.	Human cell cultures diluted with 1% DMSO alone, to give a measure of<br>
maximum proliferative potential<br>
c.	MMS as a genotoxicity and cytotoxicity control: 'high' = 50 |iig/ml, 'low' = 10<br>
fig/ml v/v<br>
d.	2% DMSO diluent alone, to confirm lack of diluent absorbance/fluorescence<br>
e.	Growth medium alone, to confirm sterility/lack of contamination<br>
Exponentially growing cultures of cells lines according to the invention (e.g. GenTK-COl and GenTK-TOl) were washed in D-PBS and suspended at a density of 2 x 106 cells/ml in double strength low autofluorescent assay medium according to the invention. 75 microlitres of the cell suspension were added to each well of the diluted chemical: GenTK-COl to one series and GenTK-TOl to the second, and to appropriate standards and controls. After the plates were filled, they were sealed using either a gas permeable membrane (for example Breath-easy, Diversified Biotech, USA) or a plastic lid, and then incubated without shaking, for 24 hours at 37°C, 5% CO2, 95% humidity.<br>
3.3 Data collection and handling.<br>
Following 24 hour incubation, fluorescence and absorbance data were collected from the microplates. Two different microplate readers which combine fluorescence and absorbance functionality have been used, and comparable data were obtained. These were a Tecan Ultra-384 (Tecan UK Ltd.): excitation 485 nm / emission 535 nm with an additional dichroic mirror (reflectance 320 nm - 500 nm, transmission 520 nm - 800 nm), and a Wallac Victor2 (PerkinElmer Life and Analytical Sciences, Monza, Italy): excitation<br><br>
485 nm / emission 535 nm. Absorbance was measured through a 620 nm filter in both instruments. Microplates were re-sealed using either a gas permeable membrane (for example Breath-easy, Diversified Biotech, USA) or a plastic lid, and then incubated without shaking, for a further 24 hours at 37°C, 5% CO2, 95% humidity. Further absorbance and fluorescence measurements were then collected. The data were transported into a Microsoft Excel spreadsheet, and converted to graphical data (see typical data in Figs 14 and 15 for Example 2). Data processing is minimal: absorbance data give an indication of reduction in proliferative potential and these data were normalised to the untreated control (=100% growth). Fluorescence data were divided by absorbance data to give 'brightness units', the measure of average GFP induction per cell. These data were normalised to the untreated control (=1). In this way, one can distinguish between a small number of highly fluorescent cells and a large number of weakly fluorescent cells. In order to correct for induced cellular autofluorescence and intrinsic compound fluorescence, the brightness values for the GenTK-COl cell line were subtracted from those of GenTK-T01. This makes visual assessment of the data more reliable. All the data were checked with and without this correction, and the decision (see below), on whether or not a compound was classified as being genotoxic, was not affected.<br>
3.4 Decision thresholds.<br>
It is useful to have clear definitions of positive and negative results from routine assays and such definitions have been derived, taking into account the maximum noise in the system and data from chemicals where there is a clear consensus on genotoxicity and mechanism of action. Naturally it is also possible for users to inspect the numerical and graphical data and draw their own conclusions. For example an upward trend in genotoxicity data that did not cross the threshold might still distinguish two compounds. The decision thresholds were set as follows:<br>
The cytotoxicity threshold is set at 80 % of the cell density reached by the untreated control cells. This is greater than 3 times the standard deviation of the background. A positive cytotoxicity result (+) is concluded if 1 or 2 compound dilutions produce a final cell density lower than the 80% threshold. A strong positive cytotoxicity result positive (++) is concluded when either (i) three or more compound dilutions produce a final cell density lower than the 80% threshold or (ii) at least one compound dilution produces a final cell density lower than a 60% threshold. A negative result (-) is concluded when no<br><br>
compound dilutions produce a final cell density lower than the 80% threshold. The lowest effective concentration (LEG) is the lowest test compound concentration that produces a final cell density below the 80% threshold.<br>
The compound absorbance control allows a warning to be generated if a test compound is significantly absorbing. If the ratio of the absorbance of the compound control well to a well filled with media alone is &gt; 2, there is a risk of interference with interpretation. The cytotoxicity controls indicate that the cell lines are behaving normally. The 'high' MMS standard should reduce the final cell density to below the 80% threshold, and should be a lower value than the 'low' standard.<br>
The genotoxic threshold is set at a relative GFP induction of 1.5 (i.e. a 50% increase). This is greater than 3 times the standard deviation of the background. A positive genotoxicity result (+) is concluded if 1 or 2 compound dilutions produce a relative GFP induction greater than the 1.5 threshold. A strong positive genotoxicity result (++) is concluded if either (i) three or more compound dilutions produce a relative GFP induction greater than the 1.5 threshold or (ii) at least one compound dilution produces a relative GFP induction greater than a 1.8 threshold. A negative genotoxicity result (-) is concluded where no compound dilutions produce a relative GFP induction greater than the 1.5 threshold. The LEG is the lowest test compound concentration that produces a relative GFP induction greater than the 1.5 threshold. The genotoxic controls demonstrate that the cell lines are responding normally to DNA damage. The 'high' MMS standard must produce a fluorescence induction &gt; 2, and be a greater value than the 'low' MMS standard. Anomalous brightness data is generated when the toxicity leads to a final cell density less than 30% that of the blank. Genotoxicity data is not calculated above this toxicity threshold. Compounds that tested negative for genotoxicity, were re-tested up to lOmM or 5000 ng/ml, or to the limit of solubility or cytotoxicity.<br>
The compound fluorescence control allows a warning to be generated when a compound is highly auto-fluorescent. If the ratio of the fluorescence of the compound control well to a media filled well is &gt;5, there is a risk of interference with interpretation. In these cases, fluorescence polarisation can be used to distinguish GFP from other sources of fluorescence (Knight et al., 2000, 2002). The Tecan instrument has this facility. Occasionally, compounds though not fluorescent themselves, induce cellular auto-<br><br>
fluorescence. This is apparent from rising brightness in the control (GenTK-COl) cell line in the absence of fluorescence from the chemical-only control. The routine subtraction of GenTK-COl from GenTK-TOl data removes this interference from the data.<br>
Example 4<br>
Further development work was conducted to develop another most preferred protocol for conducting the method according to the fourth aspect of the invention in 24 well plates with and without S9 metabolic activation.<br>
S9 is a liver extract (known to the skilled person) that makes it possible to discriminate between genotoxic compounds and some non-genotoxic compounds that may be chemically altered by hepatocyte metabolism to generate a genotoxic compound in vivo.<br>
The constructs, cells and protocols described in Example 1 were used unless indicated to the contrary.<br>
A preferred method of testing for DNA damage according to the fourth aspect of the invention comprises the steps of: (1) preparing a 24 well plate for use in an assay with metabolic activation; (2) conducting the assay in 24 well plates and processing samples in to 96 well plates; (3) collecting and analysing the data; and (4) making a judgment on DNA damage and the consequences.<br>
Details of steps (1) - (4) for conducting a most preferred test for DNA damage are given below.<br>
4.1 Microplate preparation.<br>
Assays were carried out in 24 well plates (Corning BV, Schiphol-Rijk The Netherlands) and 96 well, black, clear-bottomed microplates. For example Matrix ScreenMates, Cat. No. 4929, Apogent Discoveries, USA, or Corning (BV, Netherlands: Cat. No. 3651). A number of alternative microplates were assessed, though the variable background absorbance and fluorescence both within and between plates from individual manufacturers were generally unacceptable, leading to the currently preferred choice. It will therefore be appreciated that microplates used according to the invention preferably have consistent absorbance and fluorescence between plates and batches thereof.<br><br>
The assay plates can be filled using a liquid handling robot. For example the MicroLabS single probe, from Hamilton GB Ltd., Birmingham or a Genesis 8-probe robot (Tecan UK Ltd. Theale. UK). Microplates can also be filled rapidly and effectively using a multi-channel pipette.<br>
4.2 Assay<br>
The following standard protocol may be followed. To each row of a 24 well plate 1ml of the following Cell line media combinations were added to each well: i) 1 xio6 cells/ml GenTK-COl in RPMI1640 media ii) 1 xlO6 cells/ml GenTK-TOl in RPMI 1640 media iii) 1 xio6 cells/ml GenTK-COl in RPMI 1640 media + 10% S9 mix iv) 1 xio6 cells/ml GenTK-TOl in RPMI 1640 media +10% S9 mix<br>
The RPMI 1640 media is preferably a modified media as defined by the fifth aspect of the invention (e.g. the media disclosed in Table 1).<br>
A lOOmg/ml stock of a test chemical, or sample containing an agent that putatively caused DNA damage, was prepared in 100% v/v aqueous DMSO, and used to make 4 identical dilution series across a 24 well microplate and a 'control' (see below). To achieve this, 200 microlitres of the test chemical solution were put into a sterile 7ml glass vial or 1.5 ml microfuge tube. Each sample was serially diluted by transferring 100 microlitres into 100 microlitres of 100% DMSO, mixing, and then taking 100 microlitres out and into the next well. This produced 3 serial dilutions of 100 microlitres each. To each well of a 24 well plate The final top concentration of test chemical/sample is 1 mg/ml on the 24 well plate.<br>
Controls were added as follows:<br>
a.	Human cell cultures diluted with lOul 100% DMSO alone, to give a measure<br>
of maximum proliferative potential<br>
b.	Cyclophosphamide as a positive genotoxicity and cytotoxicity control at 30<br>
in the presence of S9.<br><br>
After the plates were filled, they were sealed using either a gas permeable membrane (for example Breath-easy, Diversified Biotech, USA) or a plastic lid, and then incubated without shaking, for 24 hours at 37°C, 5% CO2, 95% humidity. Following 24 hours incubation cells were transferred to 1.5 ml microfuge tubes and collected by centrifugation at 2500 rcf for 5 minutes. Cells were washed in 500 ul pre-warmed D-PBS (Sigma-Aldrich, Gillingham, UK) and collected by centrifugation as before. Cells were resuspended in 150 ul D-PBS, and cell suspensions transferred to a well of an optically clear bottom, black polystyrene 96 well microplate (Matrix Technologies, Wilmslow, UK). As blanks, 150 ul of D-PBS were added to 2 wells.<br>
4.3	Data collection and handling.<br>
After transferring cell suspensions to 96 well plates fluorescence and absorbance data were collected. Two different microplate readers which combine fluorescence and absorbance functionality have been used, and comparable data were obtained. These were a Tecan Ultra-384 (Tecan UK Ltd.): excitation 485 nm / emission 535 nm with an additional dichroic mirror (reflectance 320 nm - 500 nm, transmission 520 nm - 800 nm), and a Wallac Victor2 (PerkinElmer Life and Analytical Sciences, Monza, Italy): excitation 485 nm / emission 535 nm. Absorbance was measured through a 620 nm filter in both instruments. The data were transported into a Microsoft Excel spreadsheet, and converted to graphical data (see typical data in Figs 16 and 17 for Example 3). Data processing is minimal: absorbance data give an indication of reduction in proliferative potential and these data were normalised to the untreated control (=100% growth). Fluorescence data were divided by absorbance data to give 'brightness units', the measure of average GFP induction per cell. These data were normalised to the untreated control (=1). In this way, one can distinguish between a small number of highly fluorescent cells and a large number of weakly fluorescent cells. In order to correct for induced cellular autofluorescence and intrinsic compound fluorescence, the brightness values for the GenTK-COl cell line were subtracted from those of GenTK-TOl. This makes visual assessment of the data more reliable. All the data were checked with and without this correction, and the decision (see below), on whether or not a compound was classified as being genotoxic, was not affected.<br>
4.4	Decision thresholds.<br>
It is useful to have clear definitions of positive and negative results from routine assays and such definitions have been derived, taking into account the maximum noise in<br><br>
the system and data from chemicals where there is a clear consensus on genotoxicity and mechanism of action. Naturally it is also possible for users to inspect the numerical and graphical data and draw their own conclusions. For example an upward trend in genotoxicity data that did not cross the threshold might still distinguish two compounds. The decision thresholds were set as follows:<br>
The cytotoxicity threshold is set at 80 % of the cell density reached by the untreated control cells. This is greater than 3 times the standard deviation of the background. A positive cytotoxicity result (+) is concluded if 1 or 2 compound dilutions produce a final cell density lower than the 80% threshold. A strong positive cytotoxicity result positive (++) is concluded when either (i) three or more compound dilutions produce a final cell density lower than the 80% threshold or (ii) at least one compound dilution produces a final cell density lower than a 60% threshold. A negative result (-) is concluded when no compound dilutions produce a final cell density lower than the 80% threshold. The lowest effective concentration (LEG) is the lowest test compound concentration that produces a final cell density below the 80% threshold.<br>
The compound absorbance control allows a warning to be generated if a test compound is significantly absorbing. If the ratio of the absorbance of the compound control well to a well filled with media alone is &gt; 2, there is a risk of interference with interpretation. The cytotoxicity controls indicate that the cell lines are behaving normally.<br>
The genotoxic threshold is set at a relative GFP induction of 1.3 (i.e. a 30% increase). This is greater than 3 times the standard deviation of the background. A positive genotoxicity result (+) is concluded if 1 or 2 compound dilutions produce a relative GFP induction greater than the 1.3 threshold. A strong positive genotoxicity result (++) is concluded if either (i) three or more compound dilutions produce a relative GFP induction greater than the 1.3 threshold or (ii) at least one compound dilution produces a relative GFP induction greater than a 1.6 threshold. A negative genotoxicity result (-) is concluded where no compound dilutions produce a relative GFP induction greater than the 1.3 threshold. The LEG is the lowest test compound concentration that produces a relative GFP induction greater than the 1.3 threshold. The genotoxic controls demonstrate that the cell lines are responding normally to DNA damage. The Cyclophosphomide standard must produce a fluorescence induction &gt; 1.3 with S9, and not produce a fluorescence induction<br><br>
&gt; 1.1 without S9. Anomalous brightness data is generated when the toxicity leads to a final cell density less than 30% that of the blank. Genotoxicity data is not calculated above this toxicity threshold. Compounds that tested negative for genotoxicity, were re-tested up to lOmM or 5000 ng/ml, or to the limit of solubility or cytotoxicity.<br><br>
We Claim :<br>
1.	An expression cassette comprising a DNA sequence encoding a reporter protein, which<br>
DNA sequence is operatively linked to a human GADD45a gene promoter and a<br>
human GADD45a gene regulatory element arranged to activate expression of the DNA<br>
sequence in response to DNA damage.<br>
2.	An expression cassette according to claim 1, wherein the regulatory element comprises<br>
Exon 1, Exon 2, Exon 3, and/or Exon 4 of the GADD45a gene, or at least a region<br>
thereof, or any combination thereof.<br>
3.	An expression cassette according to claim 2, wherein the regulatory element comprises<br>
at least a region of Exon 1 of the GADD45a gene, at least a region of Exon 3 of the<br>
GADD45a gene, and at least a region of Exon 4 of the GADD45a gene.<br>
4.	An expression cassette according to any preceding claim, wherein the regulatory<br>
element comprises Intron 1, Intron 2, and/or Intron 3 of the GADD45a gene, or at least<br>
a region thereof, or any combination thereof.<br>
5.	An expression cassette according to claim 4, wherein the regulatory element comprises<br>
at least a region of Intron 3 of the GADD45a gene.<br>
6.	An expression cassette according to claim 5, wherein the regulatory element comprises<br>
a putative p53 binding motif.<br>
7.	An expression cassette according to either claim 5 or claim 6, wherein the regulatory<br>
element comprises a putative AP-1 motif.<br>
8.	An expression cassette according to any preceding claim, wherein the DNA sequence<br>
encodes a light emitting reporter protein.<br>
9.	An expression cassette according to any preceding claim, wherein the DNA sequence<br>
encodes Green Fluorescent Protein (GFP), and light emitting derivatives thereof.<br><br>
10.	An expression cassette according to any preceding claim, wherein the DNA sequence<br>
encodes a human enhanced GFP (hEGFP), which consists of the GFP wild type<br>
containing a double amino acid substitution of Phe-64 to Leu, and Ser-65 to Thr.<br>
11.	An expression cassette 5-31, substantially as illustrated in Figure 9.<br>
12.	An expression cassette 5-32, substantially as illustrated in Figure 9.<br>
13.	An expression cassette 5-33, substantially as illustrated in Figure 9.<br>
14.	A recombinant vector comprising an expression cassette according to any one of<br>
claims 1-13.<br>
15.	A recombinant vector according to claim 14, wherein the vector comprises DNA from<br>
the pCEP4 plasmid.<br>
16.	A recombinant vector PEP-GD531, substantially as illustrated in Figure 5.<br>
17.	A recombinant vector PEP-GD532, substantially as illustrated in Figure 6.<br>
18.	A recombinant vector PEP-GD533, substantially as illustrated in Figure 7.<br>
19.	A cell containing a recombinant vector according to any one of claims 15-18.<br>
20.	A cell according to claim 19, wherein the cell is a human cell.<br>
21.	A cell according to claim 20, wherein the cell is a human cell having a fully functional<br>
P53.<br>
22.	A cell according to claim 21, wherein the cell is a TK6 human cell line.<br>
23.	A method of detecting for the presence of an agent that causes or potentiates DNA<br>
damage comprising subjecting a cell according to any one of claims 19-22 to an agent;<br>
and monitoring the expression of the reporter protein from the cell.<br><br>
24.	The method according to claim 23, wherein the agent is further screened to assess<br>
whether it is safe to expose a living organism to the agent.<br>
25.	The method according to either claim 23 or claim 24, wherein the agent is a candidate<br>
medicament, food additive or cosmetic.<br>
26.	The method according to any one of claims 23-25, wherein the expression of a light<br>
emitting reporter protein from the cell is monitored.<br>
27.	The method according to claim 26, wherein the light emitting reporter protein is Green<br>
Fluorescent Protein.<br>
28.	The method according to claim 27, comprising growing cells transfected with a<br>
recombinant vector according to any one of claims 14-18, incubating the cells with the<br>
agent for a pre-determined time, and monitoring the expression of the Green<br>
Fluorescent Protein directly from a sample of the cells.<br>
29.	The method according to any one of claims 23 - 28, wherein the cells are grown in a<br>
low fluorescence growth medium.<br>
30.	The method according to claim 29, wherein the low fluorescence growth medium is an<br>
RPMI medium containing no riboflavin.<br>
31.	The method according to claim 29 or 30, wherein the low fluorescence growth medium<br>
is an RPMI medium containing no phenol red.<br>
32.	The method according to any one of claims 26 - 31, wherein fluorescence is<br>
determined by comparing fluorescence from a cell according to a third aspect of the<br>
invention with fluorescence from a control cell comprising a vector according to the<br>
second aspect of the invention in which the nucleic acid encoding the light emitting<br>
report is place out of frame.<br><br>
33.	The use of a cell culture media that does not contain, or contains reduced levels relative<br>
to standard cell culture media, riboflavin or phenol red in fluorescent assays.<br>
34.	The use according to claim 33 wherein the media is a media as defined by claims 30 or<br>
31.<br>
35.	The use according to claim 33 or 34 wherein the fluorescent assay is the method<br>
defined by any one of claims 23 - 32.<br>
36.	An expression cassette comprising a DNA sequence encoding a reporter protein and<br>
the use of a cell culture media substantially as herein described with reference to the<br>
accompanying drawings and as illustrated in the foregoing examples.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LUNsYWltcy0oMDEtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-Claims-(01-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LUNsYWltcy0oMTMtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-Claims-(13-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDEtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-Correspondence Others-(01-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-Correspondence Others-(13-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-Correspondence Others-(17-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1ERUxOUC0yMDA2LUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7291-DELNP-2006-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1ERUxOUC0yMDA2LUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7291-DELNP-2006-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzI5MS1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMDEtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7291-delnp-2006-Petition-137-(01-03-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259640-vaccines-against-aids-comprising-cmv-r-nucleic-acid-constructs.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259642-an-antibody-polypeptide-that-specifically-and-monovalently-binds-to-cd40l.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259641</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7291/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Dec-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENTRONIX LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>THE FAIRBAIRN BUILDING 72 SACKVILLE STREET MANCHESTER M60 1QD GREAT BRITAIN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HASTWELL PAUL</td>
											<td>32 COTEFIELD DRIVE LEIGHTON BUZZARD BEDFORDSHIRE LU7 3DS GREAT BRITAIN.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WALMSLEY RICHARD</td>
											<td>FACULTY OF LICE SCIENCES G10 JACKSON&#x27;s MILL THE UNIVERSITY OF MANCHESTER PO BOX 88 SACKVILLE STREET MANCHESTER M60 1QD GREAT BRITAIN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2005/001913</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-05-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0411198.5</td>
									<td>2004-05-20</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259641-genotoxic-testing by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:03:54 GMT -->
</html>
