<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/259915-a-recombinant-eukaryotic-host-cell-producing-human-like-glycoproteins by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:41:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 259915:&quot;A Recombinant Eukaryotic Host cell producing Human-like Glycoproteins”</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A Recombinant Eukaryotic Host cell producing Human-like Glycoproteins”</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Galactosylated Glycoproteins in Lower Eukaryotes The present invention provides a novel lower eukaryotic host cell producing human-like glycoproteins characterized as having a terminal p-galactose residue and essentially lacking fucose and sialic acid residues. The present invention also provides a method for catalyzing the transfer of a galactose residue from UDP-galactose onto an acceptor substrate in a recombinant lower eukaryotic host cell, which can be used as a therapeutic glycoprotein.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to Production of Galactosylated Glycoproteins in Lower Eukaryotes<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
This application is a continuation-in-part of U.S. Application No. 10/371,877, filed on Feb. 20, 2003, which is a continuation-in-part of U.S. Application No. 09/892,591, filed June 27, 2001, which claims the benefit under 35 U.S.C. §119(e) of U.S. Provisional Application No. 60/214,358, filed June 28, 2000, U.S. Provisional Application No. 60/215,638, filed June 30, 2000, and U.S. Provisional Application No. 60/279,997, filed March 30, 2001, each of which is incorporated herein by reference in its entirety. This application is also a continuation-in-part of PCT/US02/41510, filed on December 24, 2002, which claims the benefit of U.S. Provisional Application No. 60/344,169, filed Dec. 27, 2001, each of which is incorporated herein by reference in its entirety. This application also claims priority to U.S. Provisional Application No. 60/562,424, filed April 15,2004, which is incorporated herein by reference in its entirety. FIELD OF THE INVENTION<br>
The present invention relates to the field of protein glycosylation engineering in lower eukaryotes, specifically the production of glycoproteins having terminal galactose residues. The present invention further relates to novel host cells comprising genes encoding enzymes involved in galactosyltransfer on glycans and production of glycoproteins that are particularly useful as therapeutic agents.<br>
BACKGROUND OF THE INVENTION<br>
Yeast and filamentous fungi have both been successfully used for the production of recombinant proteins, both intracellular and secreted (Cereghino, J. L. and J. M. Cregg 2000 FEMS Microbiology Reviews 24(1): 45-66; Harkki, A., et al. 1989 Bio-Technology 7(6): 596; Berka, R. M., et al. 1992 Abstr.Papers Amer. Chem.Soc.203: 121-BIOT; Svetina, M., et al. 2000 J. Biotechnol. 76(2-3): 245-251). Various yeasts, such as K. lactis, Pichia pastoris, Pichia methanolica, and Hansenula polymorpha, have played particularly important roles as eukaryotic expression systems because they are able to grow to high cell densities and secrete large quantities of recombinant protein. Likewise, filamentous fungi, such as Aspergillus niger, Fusarium sp, Neurospora crassa and others, have been used to efficiently produce glycoproteins in industrial scale. However, glycoproteins expressed in any of these eukaryotic microorganisms differ substantially in JV-glycan structure from those in animals. This has prevented the use of yeast or filamentous fungi as hosts for the production of glycosylated therapeutic proteins.<br>
Currently, expression systems such as yeast, filamentous fungi, plants, algae and insect cell lines (lower eukaryotes) are being investigated for the production of therapeutic proteins, which are safer, faster and yield higher product titers than mammalian systems. These systems share a common secretory pathway in N-linked oligosaccharide synthesis. Recently, it was shown that the secretory pathway of P. pastoris can be genetically re-engineered to perform sequential glycosylation reactions that mimic early processing of N-glycans in humans and other higher mammals (Choi et al., Proc Natl AcadSci USA. 2003 Apr 29;100(9):5022-7). In addition, production of human glycoproteins with complex N-glycans lacking galactose through re-engineering the secretory pathway in yeast P. pastoris has been shown (Hamilton et al., Science. 2003<br>
Aug 29;301(5637):1244-6). In mammalian cells, further maturation involves galactose transfer. Consequently, the maturation of complex glycosylation pathways from yeast and lower eukaryotes requires the functional expression of pl,4-galactosyltransferase.<br>
Recombinant expression of UDP-Gal: pGlcNAc Pl,4-galactosyltransferase (pl,4GalT) has been demonstrated in mammalian cells, insect cells (e.g., Sf-9) and yeast cells. A cDNA encoding a soluble form of the human pl,4-galactosyltransferase I (EC 2.4.1.22) (lacking the endogenous Type II membrane domain) has also been expressed in the methylotrophic yeast P. pastoris. Malissard et al. Biochem Biophys Res Commun. 2000 Jan 7;267(l):169-73. Additionally, gene fusions encoding ScMntlp fused to the catalytic domain of a human pl,4-galactosyltransferase (Gal-Tf) have been expressed showing some activity of the enzyme in the yeast Golgi albeit at very low conversion efficiency. Schwientek et al., J Biol Chem. 1996 Feb 16;271(7):3398-405. Thus, targeting a pl,4-galactosyltranferase (pl,4GalT) to the secretory pathway of a host that produces glycans containing terminal GlcNAc is expected to result in some galactose transfer. However the formation of complex glycans in higher eukaryotes involves the action of mannosidase II which in mammalian cells has been found to act in competition with GalTI (Fukuta et al., Arch Biochem Biophys. 2001 Aug l;392(l):79-86). The premature action of GalT is thus expected to prevent the formation of complex galactosylated glycoproteins in the secretory pathway and yield mostly hybrid glycans.<br>
The JV-glycans of mammalian glycoproteins typically include galactose, fucose, and terminal sialic acid. These sugars are not usually found on glycoproteins produced in yeast and filamentous fungi. In humans, nucleotide sugar precursors (e.g. UDP-Af-acetylglucosamine, UDP-W-acetylgalactosamine, CMP-N-acetylneuraminic acid, UDP-<br>
galactose, GDP-fucose, etc.) are synthesized in the cytosol and transported into the Golgi, where they are incorporated into N-glycans by glycosyltransferases (Sommers and Hirschberg, 1981 J. Cell Biol. 91(2): A406-A406; Sommers and Hirschberg 1982 J. Biol. Chem. 257(18): 811-817; Perez and Hirschberg 1987 Methods in Enzymology 138: 709-715).<br>
Glycosylation engineering in heterologous protein expression systems may involve expression of various enzymes that are involved in the synthesis of nucleotide sugar precursors. The enzyme UDP-galactose 4-epimerase converts the sugar nucleotide UDP-glucose to UDP-galactose via an epimerization of C4. The enzyme has been found in organisms that are able to use galactose as its sole carbon source. Recently, the bifunctional enzyme, Gall Op, has been purified in Saccharomyces cerevisiae having both a UDP-glucose 4-epimerase and aldose 1-epimerase activity. Majumdar et al., Eur J Biochem. 2004 Feb;271(4):753-759.<br>
The UDP-galactose transporters (UGT) transport UDP-galactose from the cytosol to the lumen of the Golgi. Two heterologous genes, gmal2(+) encoding alpha 1,2-galactosyltransferase (alpha 1,2 GalT) from Schizosaccharomyces pombe and (hUGT2) encoding human UDP-galactose transporter, have been functionally expressed in S. cerevisiae to examine the intracellular conditions required for galactosylation. Correlation between protein galactosylation and UDP-galactose transport activity indicated that an exogenous supply of UDP-Gal transporter, played a key role for efficient galactosylation in S. cerevisiae (Kainuma, 1999 Glycobiology 9(2): 133-141). Likewise, a UDP-galactose transporter from S. pombe was cloned (Aoki, 1999 J.Biochem. 126(5): 940-950; Segawa, 1999 Febs Letters 451(3): 295-298).<br>
Glycosyltransfer reactions typically yield a side product which is a nucleoside diphosphate or monophosphate. While monophosphates can be directly exported in exchange for nucleoside diphosphate sugars by an antiport mechanism, diphosphonucleosides (e.g. GDP) have to be cleaved by phosphatases (e.g. GDPase) to yield nucleoside monophosphates and inorganic phosphate prior to being exported. This reaction is important for efficient glycosylation; for example, GDPase from S. cerevisiae has been found to be necessary for mannosylation. However that GDPase has 90% reduced activity toward UDP (Berninsone et al., 1994 J. Biol. Chem. 269(1):207-211). Lower eukaryotes typically lack UDP-specific diphosphatase activity in the Golgi since they do not utilize UDP-sugar precursors for Golgi-based glycoprotein synthesis. S. pombe, a yeast found to add galactose residues to cell wall polysaccharides (from UDP-galactose) has been found to have specific UDPase activity, indicating the potential requirement for such an enzyme (Berninsone et al., 1994).<br>
UDP is known to be a potent inhibitor of glycosyltransferases and the removal of this glycosylation side product may be important to prevent glycosyltransferase inhibition in the lumen of the Golgi (Khatara et al., 1974). See Berninsone, P., et al. 1995. J. Biol. Chem. 270(24): 14564-14567; Beaudet, L., et al. 1998 Abe Transporters: Biochemical, Cellular, and Molecular Aspects. 292: 397-413.<br>
What is needed, therefore, is a method to catalyze the transfer of galactose residues from a sufficient pool of UDP-galactose onto preferred acceptor substrates for use as therapeutic glycoproteins. SUMMARY OF THE INVENTION<br>
The present invention provides a novel lower eukaryotic host cell producing human-like glycoproteins characterized as having a terminal galactose residue and essentially lacking fucose and sialic acid residues on the glycoprotein. In one embodiment, the present invention provides a recombinant lower eukaryotic host cell producing human-like glycoproteins, the host comprising an isolated nucleic acid molecule encoding p-galactosyltransferase activity and at least an isolated nucleic acid molecule encoding UDP-galactose transport activity, UDP-galactose C4 epimerase activity, galactokinase activity or galactose-1-phosphate uridyl transferase. The present invention also provides a recombinant lower eukaryotic host cell producing human-like glycoproteins, the host cell capable of transferring p-galactose residue onto an N-linked oligosaccharide branch of a glycoprotein comprising a terminal GlcNAc residue, the N-linked oligosaccharide branch selected from the group consisting of GlcNAcpl,2-Manal,3; GlcNAcpl,4-Manal,3; GlcNAcpl,2-Manal,6; GlcNAcpl,4-Manal,6; and GlcNAcpl,6-Manal,6 on a trimannose core. In another embodiment, the present invention provides a recombinant lower eukaryotic host cell that produces glycoproteins that are acceptor substrates for sialic acid transfer.<br>
In another aspect of the invention, herein is provided a composition comprising a human-like glycoprotein characterized as having a terminal p-galactose residue and essentially lacking fucose and sialic acid residues on the glycoprotein. In one embodiment, the glycoprotein comprises N-linked oligosaccharides selected from the group consisting of: GalGlcNAcMan3GlcNAc2, GalGlcNAc2Man3GlcNAc2, Gal2GlcNAc2Man3GlcNAc2, GalGlcNAc3Man3GlcNAc2, Gal2GlcNAc3Man3GlcNAc2, Gal3GlcNAc3Man3GlcNAc2, GalGlcNAc4Man3GlcNAc2, Gal2GlcNAc4Man3GlcNAc2,<br>
Gal3GlcNAc4Man3GlcNAc2, Gal4GlcNAc4Man3GlcNAc2 GalGlcNAcMan5GlcNAc2, GalGlcNAc2Man5GlcNAc2, Gal2GlcNAc2Man5GlcNAc2, GalGlcNAc3Man5GlcNAc2, Gal2GlcNAc3Man5GlcNAc2 and Gal3GlcNAc3Man5GlcNAc2.<br>
In another embodiment, a method is provided for producing human-like glycoproteins in a lower eukaryotic host cell the method comprising the step of producing UDP-galactose above endogenous levels.<br>
In yet another embodiment, a method is provided for producing human-like glycoprotein composition in lower eukaryotic host cell comprising the step of transferring a galactose residue on a hybrid or complex glycoprotein in the absence of fucose and sialic acid residues.<br>
In accordance with the methods of the present invention, at least 10%, preferably 33%, more preferably 60% or greater galactosylated glycoprotein composition is produced.<br>
The present invention further provides a recombinant lower eukaryotic host cell expressing GalNAc Transferase activity.<br>
The present invention also provides a recombinant lower eukaryotic host cell expressing a gene encoding heterologous UDPase activity.<br>
Additionally, the present invention provides an isolated polynucleotide comprising or consisting of a nucleic acid sequence selected from the group consisting of:<br>
(a)SEQIDNO:14;<br>
(b) at least about 90% similar to the amino acid residues of the donor nucleotide binding site of SEQ ID NO: 13;<br>
(c)	a nucleic acid sequence at least 92%, at least 95%, at least 98%, at least 99%<br>
or at least 99.9% identical to SEQ ID NO: 14;<br>
(d)	a nucleic acid sequence that encodes a conserved polypeptide having the<br>
amino acid sequence of SEQ ID NO: 13;<br>
(e)	a nucleic acid sequence that encodes a polypeptide at least 78%, at least 80%,<br>
at least 85%, at least 90%, at least 95%, at least 98%, at least 99% or at least 99.9%<br>
identical to SEQ ID NO: 13;<br>
(f)	a nucleic acid sequence that hybridizes under stringent conditions to SEQ ID<br>
NO:13;and<br>
(g)	a nucleic acid sequence comprising a fragment of any one of (a) - (f) that is at<br>
least 60 contiguous nucleotides in length.<br>
Herein is also provided a modified polynucleotide comprising or consisting of a nucleic acid sequence selected from the group consisting of the conserved regions of SEQ ID NO: 48 - SEQ ID NO: 52 wherein the encoded polypeptide is involved in catalyzing the interconversion of UDP-glucose and UDP-galactose for production of galactosylated glycoproteins. BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1A-1B depicts the construction of a plasmid map of the integration vector pXB53 encoding hGalTI.<br>
Figure 2 depicts the construction of a plasmid map of the integration vector pRCD425 encoding the &amp; pombe Gal epimerase (SpG&amp;lE) and hGalTI.<br>
Figure 3A-3B depicts the construction of a plasmid map of the integration vector pSH263 encoding the D. melanogaster UDP-galactose Transporter (DwUGT).<br>
Figure 4 depicts the construction of a plasmid map of the integration vector pRCD465<br>
encoding hGalTI, SpGalE and D/wUGT.<br>
Figure 5 depicts the construction of a plasmid map of the integration vector pRCD461<br>
encoding the ScMnnZ/S/jGalE/hGalTI fusion protein.<br>
Figure 6A depicts the amino acid sequence of SpGalE.   Figure 6B depicts the coding<br>
sequence ofSpGALE.<br>
Figure 7 shows a sequence alignment of S. pombe, human, E. coli and S. cerevisiae<br>
epimerases.<br>
Figure 8A is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
RDP30-10 (RDP27 transformed with pRCD257) displaying a peak at 1342 m/z [A]<br>
corresponding to the mass of the Af-glycan GlcNAc2Man3GlcNAc2.<br>
Figure 8B is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
RDP37 (RDP30-10 transformed with pXB53) displaying a peak at 1505 m/z [B], which<br>
corresponds to the mass of the JV-glycan GalGlcNAc2Man3GlcNAc2 and a peak at 1662<br>
m/z [C], which corresponds to the mass of Gal2GlcNAc2Man3GlcNAc2.<br>
Figure 9A is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
YSH-44 transformed with pXB53 displaying a peak at 1501 m/z [B], which corresponds<br>
to the mass of the N-glycan GalGlcNAc2Man3GlcNAc2 and a peak at 1339 m/z [A],<br>
which corresponds to the mass of GlcNAc2Man3GlcNAc2.<br>
Figure 9B is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
YSH-44 transformed with pXB53 and pRCD395 displaying a peak at 1501 m/z [B],<br>
which corresponds to the mass of the N-glycan GalGlcNAcaMansGlcNAca; a peak at<br>
1663 m/z [C], which corresponds to the mass of GabGlcNAcaManaGlcNAca; and a peak at 1339 m/z [A], which corresponds to the mass of GlcNAc2Man3GlcNAc2. Figure 10A is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in RDP 39-6 (P. pastoris PBP-3 (US Pat. Appl. No. 20040018590)) transformed with pRCD352 and pXB53 displaying a predominant peak at 1622 m/z [K], which corresponds to the mass of the N-glycan GalGlcNAcMansGlcNAca and a peak at 1460 m/z [H], which corresponds to the mass of GlcNAcMansGlcNAc2. Figure 10B is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in RDP 39-6 after a 1,2 and pl,4-galactosidase digest displaying a predominant peak at 1461 m/z [H], which corresponds to the mass of the N-glycan GlcNAcMansGlcNAc2-Figure 11 is a MALDI-TOF-MS analysis of N-glycans isolated from K3 produced in various P. pastoris strains comparing the UDP-galactose transport activities. Panel A shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD425 encoding Mnn2(s)/hGalTI and SpGalE, which was designated RDP52. Panel B shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD425 and pRCD393 encoding SpUGT, which was designated as RDP69. Panel C shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD425 and pSH262 encoding hUGT2, which was designated as RDP70. Panel D shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD425 and pSH264 encoding hUGTI, which was designated as RDP71. Panel E shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD425 and pSH263 encoding DwUGT, which was designated RDP57.<br>
Figure 12 is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in various P. pastor is strains comparing the (3-1,4-galactosyltransferase activities. Panel A shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD425 and pSH263 encoding DmUGT, which was designated as RDP57. Panel B shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD440 encoding Mnn2(s)/hGalTII and SpGalE and pSH263 encoding DmUGT, which was designated as RDP72. Panel C shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD443 encoding Mnn2(s)/hGalTIII and SpGalE and pSH263 encoding DmUGT, which was designated as RDP73.<br>
Figure 13 is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in various P. pastoris strains comparing epimerase activities. Panel A shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pRCD424 encoding Mnn2(s)/ hGalTI and ScGallO and pSH263 encoding DmUGT, which was designated as RDP65. Panel B shows the N-glycan profile of P. pastoris YSH-44 sequentially transformed with vectors pSH263 encoding DmUGT and pRCD425, which was designated as RDP74. Panel C shows the N-glycan profile of P. pastoris YSH-44 sequentially transformed with vectors pRCD425 and then pSH263 encoding DmUGT, which was designated as RDP63. Panel D shows the N-glycan profile of P. pastoris YSH-44 transformed with vectors pXB53 and pRCD438 encoding Mnn2(s)/hGalTI and hGalE and pSH263 encoding DmUGT, which was designated as RDP67.<br>
Figure 14A is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in RDP80 (P. pastoris YSH-44 transformed with pRCD465) displaying a predominant<br>
peak   at   1663   m/z   [C],   which   corresponds   to   the   mass   of   the   N-glycan<br>
Figure 14B is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
RDP80 (P. pastoris YSH-44 transformed with pRCD465) after pl,4-galactosidase digest<br>
displaying a predominant peak at 1340 m/z [A], which corresponds to the mass of the N-<br>
glycan GlcNAciMansGlcNAca.<br>
Figure 14C is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
RDP80 and incubated with sialyltransferase in vitro in the presence of CMP-NANA,<br>
displaying a predominant peak at 2227 m/z [X], which corresponds to the mass of the N-<br>
glycanNANA2Gal2GlcNAc2Man3GlcNAc2.<br>
Figure     ISA     is     a     MALDI-TOF-MS     analysis     depicting     the     N-glycan<br>
GlcNAc2Man3GlcNAc2 [A] released from K3 produced in P. pastoris YSH-44 (control).<br>
Figure 15B is a MALDI-TOF-MS analysis of N-glycans released from K3 produced in<br>
RDP86 (P. pastoris YSH-44 transformed with pRCD461 (Mnn2(s)/SpGalE/hGalTI<br>
fusion) displaying a predominant peak at 1679 m/z [C], which corresponds to the mass<br>
of the N-glycan Gal2GlcNAc2Man3GlcNAc2.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Unless otherwise defined herein, scientific and technical terms used in connection with the present invention shall have the meanings that are commonly understood by those of ordinary skill in the art. Further, unless otherwise required by context, singular terms shall include pluralities and plural terms shall include the singular. The methods and techniques of the present invention are generally performed according to conventional methods well known in the art. Generally, nomenclatures used in<br>
connection with, and techniques of biochemistry, enzymology, molecular and cellular biology, microbiology, genetics and protein and nucleic acid chemistry and hybridization described herein are those well known and commonly used in the art. The methods and techniques of the present invention are generally performed according to conventional methods well known in the art and as described in various general and more specific references that are cited and discussed throughout the present specification unless otherwise indicated. See, e.g., Sambrook et al. Molecular Cloning: A Laboratory Manual, 2d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989); Ausubel et al., Current Protocols in Molecular Biology, Greene Publishing Associates (1992, and Supplements to 2002); Harlow and Lane Antibodies: A Laboratory Manual Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1990); Introduction to Glycobiology, Maureen E. Taylor, Kurt Drickamer, Oxford Univ. Press (2003); Worthington Enzyme Manual, Worthington Biochemical Corp. Freehold, NJ; Handbook of Biochemistry: Section A Proteins Vol I 1976 CRC Press; Handbook of Biochemistry: Section A Proteins Vol II 1976 CRC Press; Essentials of Glycobiology, Cold Spring Harbor Laboratory Press (1999). The nomenclatures used in connection with, and the laboratory procedures and techniques of, biochemistry and molecular biology described herein are those well known and commonly used in the art.<br>
All publications, patents and other references mentioned herein are incorporated by reference.<br>
The following terms, unless otherwise indicated, shall be understood to have the following meanings:<br><br>
As used herein, the term "K3" refers to the kringle 3 domain of human plasminogen.<br>
As used herein, the term "N-glycan" refers to an N-linked oligosaccharide, e.g., one that is attached by an asparagine-N-acetylglucosamine linkage to an asparagine residue of a polypeptide. N-glycans have a common pentasaccharide core of Man3GlcNAc2 ("Man" refers to mannose; "Glc" refers to glucose; and "NAc" refers to N-acetyl; GlcNAc refers to N-acetylglucosamine). N-glycans differ with respect to the number of branches (antennae) comprising peripheral sugars (e.g., GlcNAc, galactose, fucose and sialic acid) that are added to the Man3GlcNAc2 ("Man3") core structure. N-glycans are classified according to their branched constituents (e.g., high mannose, complex or hybrid). A "high mannose" type N-glycan has five or more mannose residues. A "complex" type N-glycan typically has at least one GlcNAc attached to the 1,3 mannose arm and at least one GlcNAc attached to the 1,6 mannose arm of a "trimannose" core. The "trimannose core" is the pentasaccharide core having a Man3 structure. It is often referred to as "paucimannose" structure. Complex N-glycans may also have galactose ("Gal") residues that are optionally modified with sialic acid or derivatives ("NeuAc", where "Neu" refers to neuraminic acid and "Ac" refers to acetyl). Complex N-glycans may also have intrachain substitutions comprising "bisecting" GlcNAc and core fucose ("Fuc"). Complex N-glycans may also have multiple antennae on the "trimannose core," often referred to as "multiple antennary glycans." A "hybrid" N-glycan has at least one GlcNAc on the terminal of the 1,3 mannose arm of the trimannose core and zero or more mannoses on the 1,6 mannose arm of the trimannose core.<br>
Abbreviations used herein are of common usage in the art, see, e.g., abbreviations of sugars, above. Other common abbreviations include "PNGase", which refers to peptide N-glycosidase F (EC 3.2.2.18); "GalT", which refers to Galactosyl transferase, "pl,4GalT", which refers to UDP-galactose: p-JV-acetylglucosamine pi,4-galactosyltransferase. p-Galactosyltransferases from various species are abbreviated as follows: "hGalT" refers to human pl,4-galactosyltransferase, "bGalT" refers to bovine Pl,4-galactosyltransferase, "A7GalT" refers to Xenopus leavis pl,4-galactosyltransferase and "CeGalT" refers to C. elegans pl,4-galactosyltransferase. "GalNAcT" refers to UDP-GalNAc - GlcNAc p-l,4-N-acetylgalactosaminyltransferase.<br>
As used herein, the term "UGT" refers to UDP-galactose transporter. The term "S/?GalE" refers to S. pombe UDP-galactose 4-epimerase, "hGalE" refers to human UDP-galactose 4-epimerase, "ScGallO" refer to S. cerevisiae UDP-galactose 4-epimerase and "£cGalE" refers to E. coli UDP-galactose 4-epimerase.<br>
As used herein, the term "UDP-Gal" refers to UDP-galactose and the term "UDP-GalNAc" refers to UDP-N-acetylgalactosamine.<br>
N-linked glycoproteins contain an N-acetylglucosamine residue linked to the amide nitrogen of an asparagine residue in the protein. The predominant sugars found on glycoproteins are glucose, galactose, mannose, fucose, N-acetylgalactosamine (GalNAc), N-acetylglucosamine (GlcNAc) and sialic acid (e.g., N-acetyl-neuraminic acid (NANA)). The processing of the sugar groups occurs cotranslationally in the lumen of the ER and continues in the Golgi apparatus for N-linked glycoproteins.<br>
As used herein, the term "human-like" glycoprotein refers to modified N-glycans covalently attached to a protein that are similar to the glycoproteins found in the human<br>
N-linked oligosaccharide synthesis. Complex and hybrid N-glycans are intermediates found in human glycosylation. Common to these intermediates is the MansGlcNAc2 core structure also referred to as the paucimannose core, pentasaccharide core or simply Man3 or Mans. Human-like glycoproteins, therefore, have at least the Man3 core structure.<br>
As used herein, the term "initiating 1,6 mannosyltransferase activity" refers to yeast specific glycan residues typically added to the Manal,3 arm of the trimannose core in outer chain formation initiated by Ochlp with an al,6 linkage.<br>
The mole % transfer of galactose residue onto N-glycans as measured by MALDI-TOF-MS in positive mode refers to mole % galactose transfer with respect to mole % total neutral N-glycans. Certain cation adducts such as K+ and Na+ are normally associated with the peaks eluted increasing the mass of the N-glycans by the molecular mass of the respective adducts.<br>
As used herein, the term "secretion pathway" refers to the assembly line of various glycosylation enzymes to which a lipid-linked oligosaccharide precursor and an N-glycan substrate are sequentially exposed, following the molecular flow of a nascent polypeptide chain from the cytoplasm to the endoplasmic reticulum (ER) and the compartments of the Golgi apparatus. Enzymes are said to be localized along this pathway. An enzyme X that acts on a lipid-linked glycan or an N-glycan before enzyme Y is said to be or to act "upstream" to enzyme Y; similarly, enzyme Y is or acts "downstream" from enzyme X.<br>
As used herein, the term "mutation" refers to any change in the nucleic acid or amino acid sequence of a gene product, e.g., of a glycosylation-related enzyme.<br>
The term "polynucleotide" or "nucleic acid molecule" refers to a polymeric form of nucleotides of at least 10 bases in length. The term includes DNA molecules (e.g., cDNA or genomic or synthetic DNA) and RNA molecules (e.g., mRNA or synthetic RNA), as well as analogs of DNA or RNA containing non-natural nucleotide analogs, non-native interaucleoside bonds, or both. The nucleic acid can be in any topological conformation. For instance, the nucleic acid can be single-stranded, double-stranded, triple-stranded, quadruplexed, partially double-stranded, branched, hairpinned, circular, or in a padlocked conformation. The term includes single and double stranded forms of DNA.<br>
Unless otherwise indicated, a "nucleic acid comprising SEQ ID NO:X" refers to a nucleic acid, at least a portion of which has either (i) the sequence of SEQ ID NO:X, or (ii) a sequence complementary to SEQ ID NO:X. The choice between the two is dictated by the context. For instance, if the nucleic acid is used as a probe, the choice between the two is dictated by the requirement that the probe be complementary to the desired target.<br>
An "isolated" or "substantially pure" nucleic acid or polynucleotide (e.g., an RNA, DNA or a mixed polymer) is one which is substantially separated from other cellular components that naturally accompany the native polynucleotide in its natural host cell, e.g., ribosomes, polymerases, and genomic sequences with which it is naturally associated. The term embraces a nucleic acid or polynucleotide that (1) has been removed from its naturally occurring environment, (2) is not associated with all or a portion of a polynucleotide in which the "isolated polynucleotide" is found in nature, (3) is operatively linked to a polynucleotide which it is not linked to in nature, or (4) does not occur in nature. The term "isolated" or "substantially pure" also can be used in reference<br>
to recombinant or cloned DNA isolates, chemically synthesized polynucleotide analogs, or polynucleotide analogs that are biologically synthesized by heterologous systems.<br>
However, "isolated" does not necessarily require that the nucleic acid or polynucleotide so described has itself been physically removed from its native environment. For instance, an endogenous nucleic acid sequence in the genome of an organism is deemed "isolated" herein if a heterologous sequence (i.e., a sequence that is not naturally adjacent to this endogenous nucleic acid sequence) is placed adjacent to the endogenous nucleic acid sequence, such that the expression of this endogenous nucleic acid sequence is altered. By way of example, a non-native promoter sequence can be substituted (e.g., by homologous recombination) for the native promoter of a gene in the genome of a human cell, such that this gene has an altered expression pattern. This gene would now become "isolated" because it is separated from at least some of the sequences that naturally flank it.<br>
A nucleic acid is also considered "isolated" if it contains any modifications that do not naturally occur to the corresponding nucleic acid in a genome. For instance, an endogenous coding sequence is considered "isolated" if it contains an insertion, deletion or a point mutation introduced artificially, e.g., by human intervention. An "isolated nucleic acid" also includes a nucleic acid integrated into a host cell chromosome at a heterologous site, a nucleic acid construct present as an episome. Moreover, an "isolated nucleic acid" can be substantially free of other cellular material, or substantially free of culture medium when produced by recombinant techniques, or substantially free of chemical precursors or other chemicals when chemically synthesized.<br>
As used herein, the phrase "degenerate variant" of a reference nucleic acid sequence encompasses nucleic acid sequences that can be translated, according to the standard genetic code, to provide an amino acid sequence identical to that translated from the reference nucleic acid sequence.<br>
The term "percent sequence identity" or "identical" in the context of nucleic acid sequences refers to the residues in the two sequences which are the same when aligned for maximum correspondence. The length of sequence identity comparison may be over a stretch of at least about nine nucleotides, usually at least about 20 nucleotides, more usually at least about 24 nucleotides, typically at least about 28 nucleotides, more typically at least about 32 nucleotides, and preferably at least about 36 or more nucleotides. There are a number of different algorithms known in the art which can be used to measure nucleotide sequence identity. For instance, polynucleotide sequences can be compared using FASTA, Gap or Bestfit, which are programs in Wisconsin Package Version 10.0, Genetics Computer Group (GCG), Madison, Wisconsin. FASTA provides alignments and percent sequence identity of the regions of the best overlap between the query and search sequences (Pearson, 1990, (herein incorporated by reference). For instance, percent sequence identity between nucleic acid sequences can be determined using FASTA with its default parameters (a word size of 6 and the NOPAM factor for the scoring matrix) or using Gap with its default parameters as provided in GCG Version 6.1, herein incorporated by reference.<br>
The term "substantial homology" or "substantial similarity," when referring to a nucleic acid or fragment thereof, indicates that, when optimally aligned with appropriate nucleotide insertions or deletions with another nucleic acid (or its complementary strand),<br>
there is nucleotide sequence identity in at least about 50%, more preferably 60% of the nucleotide bases, usually at least about 70%, more usually at least about 80%, preferably at least about 90%, and more preferably at least about 95%, 96%, 97%, 98% or 99% of the nucleotide bases, as measured by any well-known algorithm of sequence identity, such as FASTA, BLAST or Gap, as discussed above.<br>
Alternatively, substantial homology or similarity exists when a nucleic acid or fragment thereof hybridizes to another nucleic acid, to a strand of another nucleic acid, or to the complementary strand thereof, under stringent hybridization conditions. "Stringent hybridization conditions" and "stringent wash conditions" in the context of nucleic acid hybridization experiments depend upon a number of different physical parameters. Nucleic acid hybridization will be affected by such conditions as salt concentration, temperature, solvents, the base composition of the hybridizing species, length of the complementary regions, and the number of nucleotide base mismatches between the hybridizing nucleic acids, as will be readily appreciated by those skilled in the art. One having ordinary skill in the art knows how to vary these parameters to achieve a particular stringency of hybridization.<br>
In general, "stringent hybridization" is performed at about 25°C below the thermal melting point (Tm) for the specific DNA hybrid under a particular set of conditions. "Stringent washing" is performed at temperatures about 5°C lower than the Tm for the specific DNA hybrid under a particular set of conditions. The Tm is the temperature at which 50% of the target sequence hybridizes to a perfectly matched probe. See Sambrook et al., supra, page 9.51, hereby incorporated by reference. For purposes herein, "high stringency conditions" are defined for solution phase hybridization as<br>
aqueous hybridization (i.e., free of formamide) in 6X SSC (where 20X SSC contains 3.0 M NaCl and 0.3 M sodium citrate), 1% SDS at 65oC for 8-12 hours, followed by two washes in 0.2X SSC, 0.1% SDS at 65oC for 20 minutes. It will be appreciated by the skilled worker that hybridization at 65°C will occur at different rates depending on a number of factors including the length and percent identity of the sequences which are hybridizing.<br>
The nucleic acids (also referred to as polynucleotides) of this invention may include both sense and antisense strands of RNA, cDNA, genomic DNA, and synthetic forms and mixed polymers of the above. They may be modified chemically or biochemically or may contain non-natural or derivatized nucleotide bases, as will be readily appreciated by those of skill in the art. Such modifications include, for example, labels, methylation, substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoramidates, carbamates, etc.), charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), pendent moieties (e.g., polypeptides), intercalators (e.g., acridine, psoralen, etc.), chelators, alkylators, and modified linkages (e.g., alpha anomeric nucleic acids, etc.) Also included are synthetic molecules that mimic polynucleotides in their ability to bind to a designated sequence via hydrogen bonding and other chemical interactions. Such molecules are known in the art and include, for example, those in which peptide linkages substitute for phosphate linkages in the backbone of the molecule.<br>
The term "mutated" when applied to nucleic acid sequences means that nucleotides in a nucleic acid sequence may be inserted, deleted or changed compared to a<br>
reference nucleic acid sequence. A single alteration may be made at a locus (a point mutation) or multiple nucleotides may be inserted, deleted or changed at a single locus. In addition, one or more alterations may be made at any number of loci within a nucleic acid sequence. A nucleic acid sequence may be mutated by any method known in the art including but not limited to mutagenesis techniques such as "error-prone PCR" (a process for performing PCR under conditions where the copying fidelity of the DNA polymerase is low, such that a high rate of point mutations is obtained along the entire length of the PCR product. See, e.g., Leung, D. W., et al., Technique, I, pp. 11-15 (1989) and Caldwell, R. C. &amp; Joyce G. F., PCR Methods Applic., 2, pp. 28-33 (1992)); and "oligonucleotide-directed mutagenesis" (a process which enables the generation of site-specific mutations in any cloned DNA segment of interest. See, e.g., Reidhaar-Olson, J. F. &amp; Sauer, R. T., et al., Science, 241, pp. 53-57 (1988)).<br>
The term "vector" as used herein is intended to refer to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a "plasmid", which refers to a circular double stranded DNA loop into which additional DNA segments may be ligated. Other vectors include cosmids, bacterial artificial chromosomes (BAG) and yeast artificial chromosomes (YAC). Another type of vector is a viral vector, wherein additional DNA segments may be ligated into the viral genome (discussed in more detail below). Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., vectors having an origin of replication which functions in the host cell). Other vectors can be integrated into the genome of a host cell upon introduction into the host cell, and are thereby replicated along with the host genome. Moreover, certain preferred vectors are capable of directing<br>
the expression of genes to which they are operatively linked. Such vectors are referred to herein as "recombinant expression vectors" (or simply, "expression vectors").<br>
"Operatively linked" expression control sequences refers to a linkage in which the expression control sequence is contiguous with the gene of interest to control the gene of interest, as well as expression control sequences that act in trans or at a distance to control the gene of interest.<br>
The term "expression control sequence" as used herein refers to polynucleotide sequences which are necessary to affect the expression of coding sequences to which they are operatively linked. Expression control sequences are sequences which control the transcription, post-transcriptional events and translation of nucleic acid sequences. Expression control sequences include appropriate transcription initiation, termination, promoter and enhancer sequences; efficient RNA processing signals such as splicing and polyadenylation signals; sequences that stabilize cytoplasmic mRNA; sequences that enhance translation efficiency (e.g., ribosome binding sites); sequences that enhance protein stability; and when desired, sequences that enhance protein secretion. The nature of such control sequences differs depending upon the host organism; in prokaryotes, such control sequences generally include promoter, ribosomal binding site, and transcription termination sequence. The term "control sequences" is intended to include, at a minimum, all components whose presence is essential for expression, and can also include additional components whose presence is advantageous, for example, leader sequences and fusion partner sequences.<br>
The term "recombinant lower eukaryotic host cell" (or simply "host cell"), as used herein, is intended to refer to a cell into which a recombinant vector has been introduced.<br>
It should be understood that such terms are intended to refer not only to the particular subject cell but to the progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term "host cell" as used herein. A recombinant host cell may be an isolated cell or cell line grown in culture or may be a cell which resides in a living tissue or organism. A recombinant host cell includes yeast, fungi, collar-flagellates, microsporidia, alveolates (e.g., dinoflagellates), stramenopiles (e.g, brown algae, protozoa), rhodophyta (e.g., red algae), plants (e.g., green algae, plant cells, moss) and other protists.<br>
The term "peptide" as used herein refers to a short polypeptide, e.g., one that is typically less than about 50 amino acids long and more typically less than about 30 amino acids long. The term as used herein encompasses analogs and mimetics that mimic structural and thus biological function.<br>
The term "polypeptide" encompasses both naturally-occurring and non-naturally-occurring proteins, and fragments, mutants, derivatives and analogs thereof. A polypeptide may be monomeric or polymeric. Further, a polypeptide may comprise a number of different domains each of which has one or more distinct activities.<br>
The term "isolated protein" or "isolated polypeptide" is a protein or polypeptide that by virtue of its origin or source of derivation (1) is not associated with naturally associated components that accompany it in its native state, (2) when it exists in a purity not found in nature, where purity can be adjudged with respect to the presence of other cellular material (e.g., is free of other proteins from the same species) (3) is expressed by a cell from a different species, or (4) does not occur in nature (e.g., it is a fragment of a<br>
polypeptide found in nature or it includes amino acid analogs or derivatives not found in nature or linkages other than standard peptide bonds). Thus, a polypeptide that is chemically synthesized or synthesized in a cellular system different from the cell from which it naturally originates will be "isolated" from its naturally associated components. A polypeptide or protein may also be rendered substantially free of naturally associated components by isolation, using protein purification techniques well known in the art. As thus defined, "isolated" does not necessarily require that the protein, polypeptide, peptide or oligopeptide so described has been physically removed from its native environment.<br>
The term "polypeptide fragment" as used herein refers to a polypeptide that has an amino-terminal and/or carboxy-terminal deletion compared to a full-length polypeptide. In a preferred embodiment, the polypeptide fragment is a contiguous sequence in which the amino acid sequence of the fragment is identical to the corresponding positions in the naturally-occurring sequence. Fragments typically are at least 5, 6, 7, 8, 9 or 10 amino acids long, preferably at least 12, 14, 16 or 18 amino acids long, more preferably at least 20 amino acids long, more preferably at least 25, 30, 35,40 or 45, amino acids, even more preferably at least 50 or 60 amino acids long, and even more preferably at least 70 amino acids long.<br>
A "modified derivative" refers to polypeptides or fragments thereof that are substantially homologous in primary structural sequence but which include, e.g., in vivo or in vitro chemical and biochemical modifications or which incorporate amino acids that are not found in the native polypeptide. Such modifications include, for example, acetylation, carboxylation, phosphorylation, glycosylation, ubiquitination, labeling, e.g., with radionuclides, and various enzymatic modifications, as will be readily appreciated<br><br>
by those well skilled in the art. A variety of methods for labeling polypeptides and of substituents or labels useful for such purposes are well known in the art, and include radioactive isotopes such as 125I, 32P, 35S, and 3H, ligands which bind to labeled antiligands (e.g., antibodies), fluorophores, chemiluminescent agents, enzymes, and antiligands which can serve as specific binding pair members for a labeled ligand. The choice of label depends on the sensitivity required, ease of conjugation with the primer, stability requirements, and available instrumentation. Methods for labeling polypeptides are well known in the art. See Ausubel et al., 1992, hereby incorporated by reference.<br>
The term "fusion protein" refers to a polypeptide comprising a polypeptide or fragment coupled to heterologous amino acid sequences. Fusion proteins are useful because they can be constructed to contain two or more desired functional elements from two or more different proteins. A fusion protein comprises at least 10 contiguous amino acids from a polypeptide of interest, more preferably at least 20 or 30 amino acids, even more preferably at least 40, 50 or 60 amino acids, yet more preferably at least 75, 100 or 125 amino acids. Fusion proteins can be produced recombinantly by constructing a nucleic acid sequence which encodes the polypeptide or a fragment thereof in frame with a nucleic acid sequence encoding a different protein or peptide and then expressing the fusion protein. Alternatively, a fusion protein can be produced chemically by crosslinking the polypeptide or a fragment thereof to another protein.<br>
The term "non-peptide analog" refers to a compound with properties that are analogous to those of a reference polypeptide. A non-peptide compound may also be termed a "peptide mimetic" or a "peptidomimetic". See, e.g., Jones, (1992) Amino Acid and Peptide Synthesis, Oxford University Press; Jung, (1997) Combinatorial Peptide and<br>
Nonpeptide Libraries: A Handbook John Wiley; Bodanszky et al., (1993) Peptide Chemistry--A Practical Textbook, Springer Verlag; "Synthetic Peptides: A Users Guide", G. A. Grant, Ed, W. H. Freeman and Co., 1992; Evans et al. J. Med. Chem, 30:1229 (1987); Fauchere, J. Adv. Drug Res. 15:29 (1986); Veber and Freidinger TINS p.392 (1985); and references sited in each of the above, which are incorporated herein by reference. Such compounds are often developed with the aid of computerized molecular modeling. Peptide mimetics that are structurally similar to useful peptides of the invention may be used to produce an equivalent effect and are therefore envisioned to be part of the invention.<br>
A "polypeptide mutant" or "mutein" refers to a polypeptide whose sequence contains an insertion, duplication, deletion, rearrangement or substitution of one or more amino acids compared to the amino acid sequence of a native or wild type protein. A mutein may have one or more amino acid point substitutions, in which a single amino acid at a position has been changed to another amino acid, one or more insertions and/or deletions, in which one or more amino acids are inserted or deleted, respectively, in the sequence of the naturally-occurring protein, and/or truncations of the amino acid sequence at either or both the amino or carboxy termini. A mutein may have the same but preferably has a different biological activity compared to the naturally-occurring protein.<br>
A mutein has at least 70% overall sequence homology to its wild-type counterpart. Even more preferred are muteins having 80%, 85% or 90% overall sequence homology to the wild-type protein. In an even more preferred embodiment, a mutein exhibits 95% sequence identity, even more preferably 97%, even more preferably 98%<br>
and even more preferably 99%, 99.5% or 99.9% overall sequence identity. Sequence homology may be measured by any common sequence analysis algorithm, such as Gap or Bestfit.<br>
Preferred amino acid substitutions are those which: (1) reduce susceptibility to proteolysis, (2) reduce susceptibility to oxidation, (3) alter binding affinity for forming protein complexes, (4) alter binding affinity or enzymatic activity, and (5) confer or modify other physicochemical or functional properties of such analogs.<br>
As used herein, the twenty conventional amino acids and their abbreviations follow conventional usage. See Immunology - A Synthesis (2nd Edition, E.S. Golub and D.R. Gren, Eds., Sinauer Associates, Sunderland, Mass. (1991)), which is incorporated herein by reference. Stereoisomers (e.g., D-amino acids) of the twenty conventional amino acids, unnatural amino acids such as a-, a-disubstituted amino acids, N-alkyl amino acids, and other unconventional amino acids may also be suitable components for polypeptides of the present invention. Examples of unconventional amino acids include: 4-hydroxyproline, y-carboxyglutamate, e-N,N,N-trimethyllysine, e-N-acetyllysine, O-phosphoserine, N-acetylserine, N-formylmethionine, 3-methylhistidine, 5-hydroxylysine, s-N-methylarginine, and other similar amino acids and imino acids (e.g., 4-hydroxyproline). In the polypeptide notation used herein, the left-hand direction is the amino terminal direction and the right hand direction is the carboxy-terminal direction, in accordance with standard usage and convention.<br>
A protein has "homology" or is "homologous" to a second protein if the nucleic acid sequence that encodes the protein has a similar sequence to the nucleic acid sequence that encodes the second protein. Alternatively, a protein has homology to a<br>
second protein if the two proteins have "similar" amino acid sequences. (Thus, the term "homologous proteins" is defined to mean that the two proteins have similar amino acid sequences). In a preferred embodiment, a homologous protein is one that exhibits 60% sequence homology to the wild type protein, more preferred is 70% sequence homology. Even more preferred are homologous proteins that exhibit 80%, 85% or 90% sequence homology to the wild type protein. In a yet more preferred embodiment, a homologous protein exhibits 95%, 97%, 98% or 99% sequence identity. As used herein, homology between two regions of amino acid sequence (especially with respect to predicted structural similarities) is interpreted as implying similarity in function.<br>
When "homologous" is used in reference to proteins or peptides, it is recognized that residue positions that are not identical often differ by conservative amino acid substitutions. A "conservative amino acid substitution" is one in which an amino acid residue is substituted by another amino acid residue having a side chain (R group) with similar chemical properties (e.g., charge or hydrophobicity). In general, a conservative amino acid substitution will not substantially change the functional properties of a protein. In cases where two or more amino acid sequences differ from each other by conservative substitutions, the percent sequence identity or degree of homology may be adjusted upwards to correct for the conservative nature of the substitution. Means for making this adjustment are well known to those of skill in the art (see, e.g., Pearson et al., 1994, herein incorporated by reference).<br>
The following six groups each contain amino acids that are conservative substitutions for one another: 1) Serine (S), Threonine (T); 2) Aspartic Acid (D), Glutamic Acid (E); 3) Asparagine (N), Glutamine (Q); 4) Arginine (R), Lysine (K); 5)<br>
Isoleucine (I), Leucine (L), Methionine (M), Alanine (A), Valine (V), and 6) Phenylalanine (F), Tyrosine (Y), Tryptophan (W).<br>
Sequence homology for polypeptides, which is also referred to as percent sequence identity, is typically measured using sequence analysis software. See, e.g., the Sequence Analysis Software Package of the Genetics Computer Group (GCG), University of Wisconsin Biotechnology Center, 910 University Avenue, Madison, Wisconsin 53705. Protein analysis software matches similar sequences using measure of homology assigned to various substitutions, deletions and other modifications, including conservative amino acid substitutions. For instance, GCG contains programs such as "Gap" and "Bestfit" which can be used with default parameters to determine sequence homology or sequence identity between closely related polypeptides, such as homologous polypeptides from different species of organisms or between a wild type protein and a mutein thereof. See, e.g., GCG Version 6.1.<br>
A preferred algorithm when comparing a inhibitory molecule sequence to a database containing a large number of sequences from different organisms is the computer program BLAST (Altschul, S.F. et al. (1990) J. Mol. Biol. 215:403-410; Gish and States (1993) Nature Genet. 3:266-272; Madden, T.L. et al. (1996) Meth. Enzymol 266:131-141; Altschul, S.F. et al. (1997) Nucleic Acids fos.25:3389-3402; Zhang, J. and Madden, T.L. (1997) Genome Res. 7:649-656), especially blastp or tblastn (Altschul et al., 1997). Preferred parameters for BLASTp are: Expectation value: 10 (default); Filter: seg (default); Cost to open a gap: 11 (default); Cost to extend a gap: 1 (default); Max. alignments: 100 (default); Word size: 11 (default); No. of descriptions: 100 (default); Penalty Matrix: BLOWSUM62.<br>
The length of polypeptide sequences compared for homology will generally be at least about 16 amino acid residues, usually at least about 20 residues, more usually at least about 24 residues, typically at least about 28 residues, and preferably more than about 35 residues. When searching a database containing sequences from a large number of different organisms, it is preferable to compare amino acid sequences. Database searching using amino acid sequences can be measured by algorithms other than blastp known in the art. For instance, polypeptide sequences can be compared using FASTA, a program in GCG Version 6.1. FASTA provides alignments and percent sequence identity of the regions of the best overlap between the query and search sequences (Pearson, 1990, herein incorporated by reference). For example, percent sequence identity between amino acid sequences can be determined using FASTA with its default parameters (a word size of 2 and the PAM250 scoring matrix), as provided in GCG Version 6.1, herein incorporated by reference.<br>
The term "domain" as used herein refers to a structure of a biomolecule that contributes to a known or suspected function of the biomolecule. Domains may be coextensive with regions or portions thereof; domains may also include distinct, noncontiguous regions of a biomolecule. Examples of protein domains include, but are not limited to, an Ig domain, an extracellular domain, a transmembrane domain, and a cytoplasmic domain.<br>
As used herein, the term "molecule" means any compound, including, but not limited to, a small molecule, peptide, protein, sugar, nucleotide, nucleic acid, lipid, etc., and such a compound can be natural or synthetic.<br>
Throughout this specification and its embodiments, the word "comprise" or variations such as "comprises" or "comprising", will be understood to refer to the inclusion of a stated integer or group of integers but not the exclusion of any other integer or group of integers.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention pertains. Exemplary methods and materials are described below, although methods and materials similar or equivalent to those described herein can also be used in the practice of the present invention and will be apparent to those of skill in the art. All publications and other references mentioned herein are incorporated by reference in their entirety. In case of conflict, the present specification, including definitions, will control. The materials, methods, and examples are illustrative only and not intended to be limiting. Engineering Hosts To Produce Human-Like Galactosylated Glycoproteins<br>
The present invention provides a recombinant lower eukaryotic host cell producing human-like glycoproteins wherein the glycoproteins are characterized as having a terminal p-galactose residue and essentially lacking fucose and sialic acid. In one embodiment, the present invention provides a lower eukaryotic host cell comprising an isolated nucleic acid molecule encoding UDP-galactose: p-A^-acetylglucosamine pl,4-galactosyltransferase (pl,4GalT) in combination with at least a second isolated nucleic acid molecule encoding a UDP-galactose transporter, an isolated nucleic acid encoding a UDP-galactose 4-epimerase, an isolated nucleic acid encoding galactokinase or a galactose-1-phosphate uridyl transferase. In another embodiment, pl,4GalT is expressed<br>
in combination with an isolated nucleic acid molecule encoding a UDP-galactose transporter and an isolated nucleic acid molecule encoding a UDP-galactose 4-epimerase. Variants and fragments of the nucleic acid sequences encoding the above enzymes, recombinant DNA molecules and expression vectors comprising the enzymes for transformation are also provided.<br>
In one aspect of the present invention, a method is provided to produce a humanlike glycoprotein in a lower eukaryotic host cell comprising the step of catalyzing the transfer of a galactose residue from UDP-galactose onto an acceptor substrate in a P-linkage by expression of a pl,4GalT activity and introducing into the host a UDP-galactose 4-epimerase activity, galactokinase activity, a galactose-1-phosphate uridyl transferase activity or a UDP-galactose transport activity. The acceptor substrate is preferably an oligosaccharide composition comprising a terminal GlcNAc residue, for example, GlcNAcpl,2-Manal,3; GlcNAcpl,4-Manal,3; GlcNAcpl,2-Manal,6; GlcNAcpl,4-Manal,6; or GlcNAcpl,6-Manal,6 branch on a trimannose core.<br>
The acceptor substrate is more preferably a complex glycan (e.g., GlcNAc2Man3GlcNAc2), a hybrid glycan (e.g., GlcNAcMansGlcNAc2) or a multiple antennary glycan (e.g., GlcNAc4Man3GlcNAc2) that is covalently linked (N-linked) to a protein of interest. The p-galactose residue is transferred onto the acceptor substrate comprising a hydroxy group at carbon 4 of 2-acetamido-2-deoxy-D-glucose (GlcNAc) forming a p-glycosidic linkage. The N-linked acceptor substrates comprising a terminal GlcNAc residue capable of accepting a galactose residue include, without limitation, GlcNAcMan3GlcNAc2, GlcNAc2Man3GlcNAc2, GlcNAc3Man3GlcNAc2, GlcNAc4Man3GlcNAc2, GlcNAc5Man3GlcNAc2 GlcNAc6Man3GlcNAc2,<br>
GlcNAcMan4GlcNAc2, GlcNAcMan5GlcNAc2, GlcNAc2Man5GlcNAc2 and<br>
GlcNAc3Man5GlcNAc2.<br>
Cloning of pl.4-Galactosvltransferase genes<br>
The human p-l,4-galactosyltransferase I gene (hGalTI, Genbank AH003575) was PCR amplified from human kidney cDNA (marathon ready cDNA, Clontech) using primers RCD192 (SEQ ID NO:1) and RCD186 (SEQ ID NO:2). This PCR product was cloned in pCRl.l (Invitrogen) cloned and sequenced. From this clone, a PCR overlap mutagenesis was performed. The 5' end of the gene up to the NotI site was amplified using primers RCD198 (SEQ ID NO:3) and RCD201 (SEQ ID NO:4) and the 3' end was amplified with primers RCD200 (SEQ ID NO:5) and RCD199 (SEQ ID NO:6). The products were overlapped together with primers RCD198 (SEQ ID NO:3) and RCD199 (SEQ ID NO:6) to resynthesize the ORF with the wild-type amino acid (except for an N-terminal deletion of 43 amino acids) sequence while eliminating the Notl site. The new truncated hGalTI PCR product was cloned in pCRZ.l and sequenced. The introduced AscllPacl sites were then used to subclone the fragment into plasmid pRCD259 (Figure 1), a PpURA3/HYGR roll-in vector creating pRCD260 (Figure 1) (Example 4).<br>
The same strategy was applied in cloning the human pl,4GalTII and the human pl,4GalTIII. Example 4 describes using gene-specific primers to amplify the human pl,4-galactosyltransferase II and III genes by PCR and cloning it then into a vector. Expression of pl,4-Galactosyltransferase Activity in a Lower Eukarvote<br>
A gene encoding pl,4GalT activity or a recombinant nucleic acid molecule encoding pl,4-galactosyltransferase activity, a gene fusion encoding pl,4GalT activity (e.g., pXB53) (Figure 1) or expression from a nucleic acid molecule encoding pl,4-<br>
galactosyltransferase (Genbank AH003575) is introduced and expressed in a lower eukaryotic host cell (e.g. P. pastoris) to produce galactosylated glycoproteins. Alternatively, by activation of a p-galactosyltransferase activity, a lower eukaryotic host cell is engineered to produce galactosylated glycoforms. A catalytically active (31,4-galactosyltransferase domain or a part thereof catalyzes the transfer of a galactose residue from UDP-galactose onto the terminal GlcNAc residue of an oligosaccharide acceptor substrate (e.g. GlcNAc2Man3GlcNAc2) forming a pl,4Gal glycosidic linkage. Complex galactosylated N-glycans thaf are produced according to the present invention essentially lack fucose and sialic acid (e.g., GahGlcNAciMansGlcNAca). Such a glycoprotein composition comprising complex galactosylated, afucosylated and asialylated N-glycans are useful as therapeutic agents.<br>
The newly formed substrates are also preferable precursors in the formation of sialylated glycoproteins produced in a lower eukaryotic host. The present invention, thus provides a method for producing human-like glycoproteins wherein the glycoproteins are characterized as having a terminal galactose residues that are acceptor substrates for the transfer of sialic acid in a lower eukaryote. Combinatorial DNA library of (31.4-galactosvltransferase<br>
In a related aspect of the invention, a combinatorial DNA library of Pl,4-galactosyltransferase and yeast targeting sequence transmembrane domains is created and expressed in a lower eukaryotic host cell as described in WO 02/00879.<br>
Accordingly, a sub-library of hGalTI (e.g. Genbank Accession No. X55415) fused to a sub-library of targeting peptides of lengths: short, medium and long as described in WO 02/00879 is generated. The targeting peptide sub-library includes nucleic acid<br>
sequences encoding targeting signal peptides that result in localization of a protein to a particular location within the ER, Golgi, or trans Golgi network. These targeting peptides may be selected from the host organism to be engineered as well as from other related or unrelated organisms. Generally such sequences fall into three categories: (1) N-terminal sequences encoding a cytosolic tail (ct), a transmembrane domain (tmd) and part or all of a stem region (sr), which together or individually anchor proteins to the inner (lumenal) membrane of the Golgi; (2) retrieval signals which are generally found at the C-terminus such as the HDEL or KDEL tetrapeptide; and (3) membrane spanning regions from various proteins, e.g., nucleotide sugar transporters, which are known to localize in the Golgi.<br>
The targeting peptides are indicated herein as short (s), medium (m) and long (1) relative to the parts of a type II membrane protein. The targeting peptide sequence indicated as short (s) corresponds to the transmembrane domain (tmd) of the membrane-bound protein. The targeting peptide sequence indicated as long (1) corresponds to the length of the transmembrane domain (tmd) and the stem region (sr). The targeting peptide sequence indicated as medium (m) corresponds to the transmembrane domain (tmd) and approximately half the length of the stem region (sr). The catalytic domain regions are indicated herein by the number of nucleotide deletion with respect to its wild-type glycosylation enzyme.<br>
In one embodiment, the library was transformed into P. pastoris and the transformants were selected on minimal medium containing hygromycin. The activity of Pl,4-galactosyltransferase I fused to various leader sequences (as described below) was<br>
analyzed via production of galactosylated N-glycans as a readout using MALDI-TOF MS in positive mode. fi-Galactosyltransferase Fusion Constructs<br>
A library of the isolated yeast targeting sequence transmembrane domains (consisting of 48 leader sequences (WO 02/00879)) was ligated into the Notl/Ascl sites on pRCD260 located upstream of the hGalTI gene to create plasmids pXB20-pXB67 (each plasmid carrying one leader sequence).<br>
A representative example of a GalT fusion construct derived from a combinatorial DNA library of the invention is pXB53 (Figure 1), which is a truncated S. cerevisiae Mnn2(s) targeting peptide (1-108 nucleotides of MNN2 from Genbank NP_009571) ligated in-frame to a 43 N-terminal amino acid deletion of a human pi,4-galactosyltransferase I (Genbank AH003575). The nomenclature used herein, thus, refers to the targeting peptide/catalytic domain region of a glycosylation enzyme as S. cerevisiae Mnn2(s)/hGalTI A43. The encoded fusion protein alone, however, is insufficient to produce N-glycans having predominantly galactosylated glycans as shown in Figure 9A. Although a peak consistent with the mass of the N-glycan GalGlcNAc2Man3GlcNAc2 [B] is shown with the introduction of hGalTI in P. pastoris YSH-44, subsequent digest of the sample shows that this peak is recalcitrant to P-1,4-galactosidase (Example 7).<br>
In addition, p-l,4-galactosyltransferase activity may be specific to a particular protein of interest. Thus, it is to be further understood that not all targeting peptide/ galactosyltransferase catalytic domain fusion constructs function equally as well to produce the proper glycosylation on a glycoprotein of interest. Accordingly, a protein of<br>
interest may be introduced into a host cell transformed with a combinatorial DNA library to identify one or more fusion constructs which express a galactosyltransferase activity optimal for the protein of interest. One skilled in the art will be able to produce and select optimal fusion construct(s) using the combinatorial DNA library approach described herein.<br>
It is apparent, moreover, that other such fusion constructs exhibiting localized active galactosyltransferase catalytic domains (or more generally, domains of any enzyme) may be made using techniques described herein. It will be a matter of routine experimentation for one skilled in the art to make and use the combinatorial DNA library of the present invention to optimize, for example, Gal2GlcNAc2Man3GlcNAc2 production from a library of fusion constructs in a particular expression vector introduced into a particular host cell. Production of Galactosvlated N-glvcans In Genetically Altered P. pastor is<br>
The human-like galactosylated glycoproteins produced according to the method of present invention include GalGlcNAcMan3GlcNAc2, GalGlcNAc2Man3GlcNAc2, Gal2GlcNAc2Man3GlcNAc2, GalGlcNAc3Man3GlcNAc2, Gal2GlcNAc3Man3GlcNAc2, Gal3GlcNAc3Man3GlcNAc2, GalGlcNAc4Man3GlcNAc2, Gal2GlcNAc4Man3GlcNAc2, Gal3GlcNAc4Man3GlcNAc2, Gal4GlcNAc4Man3GlcNAc2, GalGlcNAcMan5GlcNAc2, GalGlcNAc2Man5GlcNAc2, Gal2GlcNAc2Man5GlcNAc2, GalGlcNAcMan5GlcNAc2, GalGlcNAc2Man5GlcNAc2, Gal2GlcNAc2Man5GlcNAc2, GalGlcNAc3Man5GlcNAc2, Gal2GlcNAc3Man5GlcNAc2 and Gal3GlcNAc3Man5GlcNAc2.<br>
In one embodiment of the invention, the plasmid pXB53 comprising MNN2(s)/hGalTI was transformed in P. pastoris RDP30-10, host producing<br>
GlcNAc2Man3GlcNAc2 (Example 5). The catalytically active p-galactosyltransferase domain catalyzes the transfer of a galactose residue onto an acceptor substrate having a terminal GlcNAc residue (e.g. GlcNAc2Man3GlcNAc2) to produce a galactosylated glycoform. Using MALDI-TOF MS, the N-glycans released from the reporter protein from P. pastoris RDP37 showed a peak at 1505 m/z, which corresponds to the mass of GalGlcNAc2Man3GlcNAc2 [B] (Figure 8B). Transfer of a galactose residue by the fusion construct comprising human S. cerevisiae Mnn2(s)/pl,4-galactosyltransferase onto the acceptor substrate GlcNAc2Man3GlcNAc2 producing GalGlcNAc2Man3GlcNAc2 was shown to be about 10-20%. Figure 8B shows the corresponding mass of Gal2GlcNAc2Man3GlcNAc2 at 1662 m/z [C]. Transfer of two galactose residues onto the GlcNAc2Man3GlcNAc2 substrate producing Gal2GlcNAc2Man3GlcNAc2 was, therefore, evident. Accordingly, the host of the present invention exhibits at least 10 mole % of galactosyl moiety on a human-like N-glycan.<br>
It is recognized that GalTI is capable of transferring a second galactose residue onto an acceptor substrate having a second terminal GlcNAc residue in a host producing complex (e.g., biantennary) glycans. For example, a Mnn2(s)/hGalTI fusion, which is capable of capping the terminal GlcNAc with a galactose residue on the GlcNAcpl,2 Manal,3 arm of the glycan GlcNAc2Man3GlcNAc2, can form at least one additional P-glycosidic linkage on the other arm exposed with a terminal GlcNAc residue (e.g., GlcNAcpl,2 Manal,6), thereby, producing a galactosylated glycoform without the expression of subsequent galactosyltransferases. Figure 12 displays the MALDI-TOF MS exhibiting a peak at 1663 m/z [C], which corresponds to Gal2GlcNAc2Man3GlcNAc2- The results show that substrate specificity for a particular<br>
pl,4-GalT is not limited to catalyzing the transfer of galactose residues on only the designated arm of the glycan, hence, a second galactosyltransferase may be obviated. Accordingly, in one embodiment of the present invention, expression of only one pi,4-GalT activity is capable of producing mono-, bi-, tri- or tetra-antennary galactosylated glycoforms. In such an embodiment, all glycosidic linkages between the galactose residue and the GlcNAc residue on the glycan would be the same. For instance, expression of hGalTl in a host producing biantennary glycans would exhibit two terminal Galpl,4 - GlcNAcpl,2 linkages.<br>
Alternatively, a different p-galactosyltransferase activity (e.g. hGalT II) or a catalytically active part thereof is expressed in a lower eukaryotic host cell. In one embodiment, a vector pRCD440 comprising the MNN2(s)l hGalTII and SpGALE and the vector pSH263 (Figure 3B) comprising jDwUGT was transformed into a host P. pastoris YSH-44 (Figure 12B). The N-glycan analysis of the transformants showed the production of the Gal2GlcNAc2Man3GlcNAc2 glycoform indicating that hGalTII transferred both galactose residues onto the acceptor substrate (Figure 12B). Bi-galactosylated structures (Gal2GlcNAc2Man3GlcNAc2) are predominant. Transfer of galactosyl moiety with respect to % neutral glycans was approximately 75%.<br>
In yet another embodiment, a sequence encoding the hGalTIII is expressed in a lower eukaryotic host cell. Figure 12C shows galactose transfer of the combined mono-and bi-galactosylated glycans to be about 50 to 60 mole %. Comparison of hGalTl, hGalTII and hGalTIII show various level of galactose transfer (Figure 12A-C). The N-glycan profile from P. pastoris RDP71 (Figure 12A) shows that the transfer of galactose<br>
residue by the expression of hGalTI is optimal (about 80 mole %) for the K3 reporter<br>
protein.<br>
Expression of Additional pl.4-Galactosvltransferases<br>
In another embodiment, hGalTI and hGalTII are sequentially localized and expressed using medial and late Golgi targeting sequences, respectively. For example, the hGalTI is localized in the medial Golgi whereas the hGalTII is localized in the late Golgi. Alternatively, to avoid substrate competition with Mannosidase II, in another embodiment, late Golgi leaders are used for p-galactosyltransferases.<br>
Expression of galactosyltransferase activities usually generates both mono- and bi-galactosylated glycans. Multiple antennary galactosylated glycoforms in addition to mono-galactosylated glycoforms are generally produced in host cells expressing galactosyltransferase activity.<br>
It will be a matter of routine experimentation for a skilled artisan to optimize galactosyltransferase activity or expression of the gene encoding the protein by using various promoters and various expression vectors in a recombinant host cell. Tailored Galactosvlated Glvcosidic Linkages in the Production of N-Glvcans<br>
In another feature of the invention, production of multiple antennary galactosylated glycoproteins using different GalTs result in different p-glycosidic linkages. In one embodiment, desired p-glycosidic linkages of preference are generated in a lower eukaryotic host cell. For example, any one of the pl,4GalT family (e.g., hGalTI, hGalT2, hGalT3, hGalT4, hGalTS, hGalT6, hGalT7, bGalTI, A7GalT, CeGalTII) is expressed for the production of galactosylated glycoproteins characterized as having a pl,4Gal glycosidic linkage.<br>
Alternatively, by expressing other galactosyltransferases, such as, pl,3GalT or pl,6GalT activities (enzyme, homologs, variants, derivatives and catalytically active fragment thereof) in a lower eukaryotic host cell (e.g. P. pastoris), a galactose residue is transferred onto an intermediate oligosaccharide acceptor substrate forming a specifically desired PGal-glycosidic linkage. Various terminal galactose linkages (e.g., 01,3, 01,4; or {31,6) are formed as a result of the expression of a desire p-galactosyltransferase activity. GalNAcT Expression in Lower Eukarvotes<br>
GalNAc capped glycans have been observed on specific proteins in human. In another aspect of the present invention, a gene encoding GalNAc Transferase (GalNAcT) is expressed in a lower eukaryotic host cell, which transfers GalNAc residues onto a substrate having a terminal GlcNAc residue. In one embodiment, a gene encoding C. elegans GalNAcT (Genbank AN NP_490872) catalyzes the transfer of a GalNAc residue onto a substrate having a terminal GlcNAc residue extending the oligosaccharide branch of the glycans produced in a host cell. Enhanced Galactosvl Transfer<br>
The hGalTI expression comparison as shown in Figure 12 indicates that P-galactosyltransferase expression alone may not be sufficient in the formation of PGal-glycosidic linkages on acceptor substrates in a lower eukaryote. The transfer of a galactose residue is enhanced by the addition of a heterologous gene encoding an epimerase or galactokinase, a galactose-1-phosphate uridyl transferase and/or a gene encoding a UGT. Sufficient quantity of galactosylated glycoforms (e.g., Gal2GlcNAc2Man3GlcNAc2) is desirable as therapeutic glycoprotein. Accordingly, it is a feature of the present invention to enhance galactosyl transfer onto glycans by additional<br>
expression of a transport activity and/or to elevate endogenous UDP-galactose levels. In one embodiment, an epimerase activity is introduced in a host cell to increase UDP-galactose levels. In another embodiment, increased UDP-galactose level is mediated by galactokinase or a galactose-1-phosphate uridyl transferase activity. The present invention, therefore, provides a method to enhance galactosyltransfer by introducing and expressing a p-galactosyltransferase activity in combination with either a UDP-Gal transport activity and/or by elevating endogenous UDP-galactose levels via an epimerase or galactokinase or a galactose-1-phosphate uridyl transferase.<br>
Cloning and Expression of UDP-Galactose Transporter (UGT) in Lower Eukaryotic Hosts in the Production of Human-like Glycoproteins<br>
Herein the specification, is also disclosed a method to introduce and express a gene encoding a UDP-galactose transporter in a lower eukaryotic cell (e.g. P. pastoris) for the production of human-like galactosylated glycoproteins. Cloning and Expression of S. pombe UDP-galactose transporter<br>
Gene-specific primers were designed to complement the homologous regions of the S. pombe UDP-galactose transporter gene (Genbank AL022598) and PCR amplified from S. pombe genomic DNA (ATCC24843) eliminating a single intron. Primers RCD164 (SEQ ID NO:7) and RCD177 (SEQ ID NO:8) were used to amplify the 5' 96bp of the gene. Primers RCD176 (SEQ ID NO:9) and RCD165 (SEQ ID NO:10) were used to amplify the 3' 966bp. Primers RCD164 (SEQ ID NO:7) and RCD165 (SEQ ID NO: 10) were used to overlap the two amplified products into a single PCR fragment containing one contiguous ORF with NotI and Pad sites introduced at the ends. The PCR product was cloned into pCRZ.l TA (Invitrogen) and sequenced. The gene product<br>
was subcloned into plasmid pJN335 containing the P. pastoris GAPDH promoter (Example 2).<br>
Accordingly, in one embodiment, a plasmid pRCD257 encoding the S. pombe UDP-galactose transporter (Genbank AB023425) is constructed and expressed in a host producing terminal GlcNAc residues (P. pastoris RDP-27 (e.g. GlcNAcMan3GlcNAc2)). Cloning and Expression of Various UDP-galactose transporters<br>
In a preferred embodiment, the gene encoding the D. melanogaster UDP-galactose transporter is introduced and expressed in a lower eukaryotic host cell. The D. melanogaster UGT was PCR amplified from a D. melanogaster cDNA library (UC Berkeley Drosophila Genome Project, ovary X-ZAP library GM) and cloned into the pCR2.1 PCR cloning vector and sequenced. Primers DmUGT-5' (SEQ ID NO: 11) and DmUGT-3' (SEQ ID NO: 12) were used to amplify the gene introducing NotI and Pad sites. The NotI and Pad sites were used to subclone this gene fused downstream of the PpOCHl promoter at the Notl/Pad sites in pRCD393 creating pSH263 (Figure 3B). Example 2 describes cloning of various other UDP galactose transporters.<br>
Figure 11 shows UDP-transporter activity in comparison for enhanced galactose transfer. As the best mode of the present invention, the UDP-galactose transporter isolated from D. melanogaster is expressed in P. pastoris. The activity of the human GalTI gene fusion co-expressed with the D. melanogaster UDP-galactose transporter (DmUGT) is shown in Figure HE. Surprisingly, host cells expressing the D. melanogaster UGT produce predominantly galactosylated glycoforms, whereas, UGTs from S. pombe (Figure HB), human I (Figure 11C) and human II (Figure 11D) showed less than optimal transfer. A significant increase in the production of a bi-galactosylated,<br>
afucosylated and asialylated glycoform Gal2GlcNAc2Man3GlcNAc2 is produced.   The<br>
uniform   peak   at   1664   m/z   [C]   corresponds   to   the   mass   of   the   glycan<br>
Gal2GlcNAc2Man3GlcNAc2.    A host cell (e.g., P. pastoris) expressing the DmUGT<br>
exhibits at least 90 mole % galactose transfer in comparison to other UDP-galactose<br>
transporters.<br>
UDP-Galactose Transporter Polypeptides<br>
The invention additionally provides various combination of transporter-transferase fusions expressed in a lower eukaryotic host cell (e.g., P. pastoris). Accordingly, in one embodiment, the present invention provides a lower eukaryotic host comprising a UDP-galactose transporter fused in-frame to a catalytically active (3-galactosyltransferase domain. In another embodiment, the host cell producing humanlike glycoproteins comprises a UDP-galactose transporter isolated from S. potnbe and S. cerevisiae Mnn2(s) targeting peptide fused in-frame to hGalTI catalytic domain. Expression of UDP-Galactose 4-Epimerase in Lower Eukaryotic Hosts in the Production of Human-like Glycoproteins<br>
In another aspect of the invention, a method is provided for producing a humanlike glycoprotein in a lower eukaryote (e.g. P. pastoris) by expressing a 01,4-galactosyltransferase activity and at least a UDP-galactose 4-epimerase activity (enzyme, homologs, variants, derivatives and catalytically active fragment thereof). The epimerase is an enzyme that catalyzes the interconversion of UDP-galactose and UDP-glucose. Using well known techniques in the art, gene-specific primers are designed to complement the homologous regions of an epimerase gene (e.g. ScGALlO, SpGALE, hGALE) and PCR amplified (Example 3). In one embodiment, a gene encoding the S.<br>
cerevisiae Gal 10 activity or a recombinant nucleic acid molecule encoding an epimerase or expression from a nucleic acid molecule encoding an epimerase activity is introduced and expressed in a lower eukaryotic host cell (e.g. P. pastoris) to produce human-like glycoproteins characterized as having a terminal p-galactose residue. Alternatively, by activation of an epimerase activity, a host cell is engineered to produce increased levels of galactosylated glycoforms. Expression of UDP-galactose 4-epimerase in the Production of Complex N-glvcans<br>
In one embodiment, a gene encoding an epimerase activity is expressed to convert UDP-glucose to UDP-galactose, generating an increased level of UDP-galactose for galactosyltransfer in host cells. The expression of an epimerase activity in addition to a p-l,4-galactosyltransferase activity increases production of galactosylated N-glycans. Figure 9B shows a yeast strain producing complex glycans (e.g., P. pastoris YSH-44) transformed with a Mnn2(s)/hGalTI fusion in combination with pRCD395, a plasmid encoding ScGallO. The addition of the ScGallO epimerase increases the available UDP-galactose for galactose transfer. A peak at 1501 m/z [B] corresponds to the transfer of one galactose residue on the glycan GlcNAc2Man3GlcNAc2 and a peak at 1663 m/z [C] corresponds to the transfer of two galactose residues on the glycan GlcNAc2Man3GlcNAc2. Preferably, at least 60 mole % of galactose is transferred with respect to % total neutral glycans. Accordingly, in one embodiment, a (3-1,4-galactosyltransferase activity in combination with an epimerase activity is expressed in a host cell to produce galactosylated glycoproteins (Example 7). Expression of UDP-galactose 4-epimerase in the Production of Hybrid N-glvcans<br>
In another embodiment, the introduction and expression of ScGALlO increases galactose transfer on a hybrid glycoprotein in a lower eukaryote (Example 6). Figure 10A shows the P. pastoris strain RDP39-6 expressing an Mnn2(m)/hGalTI fusion in combination with the ScGallO epimerase producing hybrid galactosylated N-glycans. The N-glycan analysis shows peak at 1622 m/z [K], which corresponds to the mass of the glycan GalGlcNAcMansGlcNAc2 confirming transfer of one galactose residue, and a peak at 1460 m/z [H], which corresponds to the mass of the hybrid glycan GlcNAcMansGlcNAc2. Subsequent pl,4-galactosidase digest confirms presence of a single galactose residue (Figure 10B). Preferably, at least 70 mole % of galactose transfer is detected with respect to % total neutral glycans.<br>
Still other epimerases are expressed in a host cell to increase galactose transfer. Example 3 describes construction of epimerase constructs and Figure 13 shows the activity of various epimerases in the production of human-like N-glycans. The expression of ScGallO along with Mnn2(s)/hGalTI and DniUGT in Figure 13A shows a predominant bi-galactosylated glycoform Gal2GlcNAc2Man3GlcNAc2. Similarly, the transformation of SpGalE, Mnn2(s)/hGalTI and the DmUGT in either order results in the production of the bi-galactosylated glycoform (Figure 13B and C). The addition of hGalE has the same effect (Figure 13D). Preferably, the epimerase is selected from the group consisting of S. cerevisiae UDP-galactose 4-epimerase, S. pombe UDP-galactose 4-epimerase, E. coli UDP-galactose 4-epimerase and H. sapiens UDP-galactose 4-epimerase. It is contemplated that other epimerases, without limitation, can be selected and expressed in the host cell as well.<br>
Nucleic acid sequences encoding SpGALE<br>
The present invention additionally provides isolated nucleic acid molecules that include the GALE gene from S. pombe and variants thereof. The full-length nucleic acid sequence for this gene, which encodes the enzyme UDP-galactose 4-epimerase, has already been sequenced and identified as set forth in Genbank NC_003423. Primers used to amplify SpGALE from S. pombe genomic DNA revealed a 175bp intron, which was eliminated (Example 3). Included within the cloned genomic sequence is a coding sequence for S. pombe UDP-galactose 4-epimerase. The encoded amino acid sequence is also set forth as SEQ ID NO: 13. The SpGALE gene is particularly useful in generating a sufficient pool of UDP-galactose for galactose transfer onto N-glycans in a host cell. Expression of the SpGALE gene in a lower eukaryote provides increased and efficient galactose transfer in N-linked oligosaccharide synthesis.<br>
In one embodiment, the invention provides an isolated nucleic acid molecule having a nucleic acid sequence comprising or consisting of a SpGALE coding sequence as set forth in SEQ ID NO: 14, and homologs, variants and derivatives thereof. In a further embodiment, the invention provides a nucleic acid molecule comprising or consisting of a sequence which is a variant of the SpGALE gene having at least 53% identity to the wild-type gene. The nucleic acid sequence can preferably have at least 70%, 75% or 80% identity to the wild-type gene. Even more preferably, the nucleic acid sequence can have 85%, 90%, 95%, 98%, 99%, 99.9% or even higher identity to the wild-type gene.<br>
In another embodiment, the nucleic acid molecule of the invention encodes a polypeptide having the amino acid sequence of SEQ ID NO: 13. Also provided is a<br>
nucleic acid molecule encoding a polypeptide sequence that is at least 60% identical to SEQ ID NO: 13. Typically the nucleic acid molecule of the invention encodes a polypeptide sequence of at least 70%, 75% or 80% identity to SEQ ID NO: 13. Preferably, the encoded polypeptide is 85%, 90% or 95% identical to SEQ ID NO: 13, and the identity can even more preferably be 98%, 99%, 99.9% or even higher. Epimerase Conserved Regions involved in the interconversion of UDP-Glucose and UDP-Galactose for the Production of Galactosylated Glvcoproteins<br>
Sequence alignment of epimerases from S. pombe, human, E. coli and the first 362 amino acid residues of S. cerevisiae shows highly conserved regions indicating the presence of several motifs and a potential active site (Figure 7) (Example 11). In one embodiment, the invention encompasses a polypeptide comprising the amino acid sequence of SEQ ID NO: 13, which has a potential UDP-galactose or UDP-glucose binding motif at<br>
9-VLVTGGXGYIGSHT-22 (SEQ ID NO:48), 83-VIHFAGLKAVGESXQXPLXYY-103 (SEQ ID NO:49), 127-FSSSATVYGX-136 (SEQ IDNO:50),<br>
184-LRYFNPXGAHXSGXXGEDPXGIPNNLXPYXXQVAXGRX-221      (SEQ      ID NO:51),or<br>
224-LXXFGXDYXXXDGTXXRDYIHVXDLAXXHXXAX-256 (SEQ ID NO:52). In another preferred embodiment, the amino acid residue at position 15 of the first sequence is selected from the group consisting of S and A.<br>
In another preferred embodiment, the amino acid residue at position 96 of the second sequence is selected from the group consisting of T and V.<br>
In another preferred embodiment, the amino acid residue at position 98 of the second<br>
sequence is selected from the group consisting of V, K and I.<br>
In another preferred embodiment, the amino acid residue at position 101 of the second<br>
sequence is selected from the group consisting of S, D, E and R.<br>
In another preferred embodiment, the amino acid residue at position 136 of the third<br>
sequence is selected from the group consisting of D and N.<br>
In another preferred embodiment, the amino acid residue at position 190 of the fourth<br>
sequence is selected from the group consisting of G, T, V and I.<br>
In another preferred embodiment, the amino acid residue at position 194 of the fourth<br>
sequence is selected from the group consisting of P and A.<br>
In another preferred embodiment, the amino acid residue at position 197 of the fourth<br>
sequence is selected from the group consisting of E, C, D and L.<br>
In another preferred embodiment, the amino acid residue at position 198 of the fourth<br>
sequence is selected from the group consisting of L, I and M.<br>
In another preferred embodiment, the amino acid residue at position 203 of the fourth<br>
sequence is selected from the group consisting of L and Q.<br>
In another preferred embodiment, the amino acid residue at position 210 of the fourth<br>
sequence is selected from the group consisting of L and M.<br>
In another preferred embodiment, the amino acid residue at position 213 of the fourth<br>
sequence is selected from the group consisting of I, V and M.<br>
In another preferred embodiment, the amino acid residue at position 214 of the fourth<br>
sequence is selected from the group consisting of A and S.<br>
In another preferred embodiment, the amino acid residue at position 218 of the fourth<br>
sequence is selected from the group consisting of V and I.<br>
In another preferred embodiment, the amino acid residue at position 221 of the fourth<br>
sequence is selected from the group consisting of L and R.<br>
In another preferred embodiment, the amino acid residue at position 225 of the fifth<br>
sequence is selected from the group consisting of N, A and Y.<br>
In another preferred embodiment, the amino acid residue at position 226 of the fifth<br>
sequence is selected from the group consisting of V and I.<br>
In another preferred embodiment, the amino acid residue at position 229 of the fifth<br>
sequence is selected from the group consisting of D and N.<br>
In another preferred embodiment, the amino acid residue at position 232 of the fifth<br>
sequence is selected from the group consisting of P and D.<br>
In another preferred embodiment, the amino acid residue at position 233 of the fifth<br>
sequence is selected from the group consisting of T and S.<br>
In another preferred embodiment, the amino acid residue at position 234 of the fifth<br>
sequence is selected from the group consisting of S, E and R.<br>
In another preferred embodiment, the amino acid residue at position 238 of the fifth<br>
sequence is selected from the group consisting of P and G.<br>
In another preferred embodiment, the amino acid residue at position 239 of the fifth<br>
sequence is selected from the group consisting of I and V.<br>
In another preferred embodiment, the amino acid residue at position 246 of the fifth<br>
sequence is selected from the group consisting of C, V and M.<br>
In another preferred embodiment, the amino acid residue at position 250 of the fifth<br>
sequence is selected from the group consisting of E, K and D.<br>
In another preferred embodiment, the amino acid residue at position 251 of the fifth<br>
sequence is selected from the group consisting of A and G.<br>
In another preferred embodiment, the amino acid residue at position 253 of the fifth<br>
sequence is selected from the group consisting of V and I.<br>
In another preferred embodiment, the amino acid residue at position 254 of the fifth<br>
sequence is selected from the group consisting of A and V.<br>
In another preferred embodiment, the amino acid residue at position 256 of the fifth<br>
sequence is selected from the group consisting of L and M.<br>
Isolated Polypeptides<br>
According to another aspect of the invention, isolated polypeptides (including muteins, allelic variants, fragments, derivatives, and analogs) encoded by the nucleic acid molecules of the invention are provided. In one embodiment, the isolated polypeptide comprises the polypeptide sequence corresponding to SEQ ID NO: 13. In an alternative embodiment of the invention, the isolated polypeptide comprises a polypeptide sequence at least 60% identical to SEQ ID NO: 13. Preferably the isolated polypeptide of the invention has at least 70%, 75% or 80% identity to SEQ ID NO: 13. More preferably, the identity is 85%, 90% or 95%, but the identity to SEQ ID NO: 13 can be 98%, 99%, 99.9% or even higher.<br>
According to other embodiments of the invention, isolated polypeptides comprising a fragment of the above-described polypeptide sequences are provided.<br>
These fragments preferably include at least 20 contiguous amino acids, more preferably at least 25, 30, 35, 40, 45, 50, 60, 70, 80, 90,100 or even more contiguous amino acids.<br>
The polypeptides of the present invention also include fusions between the above-described polypeptide sequences and heterologous polypeptides. The heterologous sequences can, for example, include heterologous sequences designed to facilitate purification and/or visualization of recombinantly-expressed proteins. Other non-limiting examples of protein fusions include those that permit display of the encoded protein on the surface of a phage or a cell, fusions to intrinsically fluorescent proteins, such as green fluorescent protein (GFP), and fusions to the IgG Fc region. UDP-Galactose 4-Epimerase / pl,4-Galactosvltransferase Fusion Poivpeptides<br>
In a further aspect of the invention, a gene fusion encoding a polypeptide comprising epimerase and galactosyltransferase activities is generated. In one embodiment, a fusion polypeptide comprising a UDP-galactose 4-epimerase and pi,4-GalTI is generated and introduced in a host cell. In a more preferred embodiment, the fusion polypeptide further comprises a leader sequence. For example, a library of leader sequences encoding targeting peptides is ligated in-frame to SpGalE/hGalTI fusion. In an even more preferred embodiment, the fusion polypeptide comprises ScMnn2(s) leader, SpGalE epimerase, and hGalTI. The fusion polypeptide is inserted into a yeast integration plasmid comprising a HYG marker. An example of an epimerase-galactosyltransferase integration plasmid designated pRCD461 is shown in Figure 5 (Example 8). The epimerase-galactosyltransferase fusion transformant produces approximately 70% galactosylated human-like glycoprotein Gal2GlcNAc2Man3GlcNAc2 (Figure 15B).<br>
B1.4-Galactosyltransferase: UDP-Galactose 4-Epimerase; UDP-Galactose Transporter Polypeptides<br>
In another aspect of the present invention, a single construct encoding polypeptides comprising a p-galactosyltransferase, epimerase and UDP-galactose transporter activities is generated. In one embodiment, a plasmid comprising human pl,4GalT, SpGdE and DmUGT ("triple") is constructed (Example 9). In a preferred embodiment, the transferase polypeptide further comprises a leader sequence, for example, 5'cMnn2(s) ligated in-frame to hGalTI. All three polypeptides are inserted into a yeast integration plasmid containing a KAN* marker, preferably with their own promoters and terminators. An example of this "triple" integration plasmid, designated pRCD465, is shown in Figure 4. In one embodiment, the "triple" integration plasmid comprising the fusion polypeptide is introduced and expressed in a host cell producing terminal GlcNAc residues. P. pastoris YSH-44 was transformed with the "triple" integration plasmid and was denoted RDP80.<br>
To evaluate whether the N-glycans produced in strain RDP80 are the predicted Gal2GlcNAc2Man3GlcNAc2 species, purified K3 secreted from RDP80 was incubated with sialyltransferase in vitro in the presence of CMP-NANA and the resulting N-glycans were released. The MALDI-TOF MS analysis of the N-glycans displayed a predominant peak at 2227 m/z [X], which corresponds to the mass of the complex, terminally sialylatedN-glycanNANA2Gal2GlcNAc2Man3GlcNAc2 (Figure 14C). Alternative Production of UDP-Gal<br>
As described previously, the transfer of galactose residues onto N-glycans requires a pool of activated galactose (UDP-Gal). One way to generate such a pool above<br>
endogenous levels in a lower eukaryote is the expression of a UDP-galactose 4 epimerase. An alternative route includes the expression of three separate genes: a plasma membrane galactose permease, a galactokinase, and a galactose-1-phosphate uridyl transferase in the absence of UDP-galactose 4 epimerase. Expression of the other three genes of the LeLoir pathway in the absence of the UDP-galactose 4 epimerase, with an exogenous source of galactose, would serve to elevate the endogenous levels of UDP-galactose (Ross et al, 2004). Furthermore, in this embodiment the absence of UDP-galactose 4 epimerase allows the levels of UDP-galactose to be modulated by controlling the exogenous concentration of galactose because the UDP-galactose generated cannot be metabolized apart from addition to substrates such as N-glycans.<br>
A galactose permease is a plasma membrane hexose transporter, which imports galactose from an exogenous source. In one embodiment, the galactose permease gene from S. cerevisiae, GAL2 (Genbank: M81879), or any gene encoding a plasma membrane hexose transporter capable of importing galactose is used.<br>
A galactokinase is an enzyme that catalyzes the first step of galactose metabolism, namely the phosphorylation of galactose to galactose-1-phosphate. In another embodiment, the GAL1 gene from S. cerevisiae (Genbank: X76078) is used.<br>
Galactose-1-phosphate uridyl transferase catalyzes the second step of galactose metabolism, which is the conversion of UDP-glucose and galactose-1-phosphate to UDP-galactose and glucose-1-phosphate. In another embodiment, any gene encoding galactose-1-phosphate uridyl transferase activity can be used, including S. cerevisiae GAL7 (Genbank: M12348).<br>
In a preferred embodiment, the UDP-galactose 4 epimerase encoding gene is deleted from a lower eukaryote capable of metabolizing galactose via the LeLoir pathway.<br>
In a more preferred embodiment, galactose permease, galactokinase, and galactose-1-phosphate uridyl transferase encoding genes are expressed in a lower eukaryotic host cell that is gal (-) and does not express the genes of the LeLoir pathway endogenously (Hittinger et al, 2004).<br>
The advantage of this alternative embodiment is that the absence of UDP-galactose 4-epimerase allows specific control of internal UDP-galactose concentration by the modulation of external galactose at levels below growth inhibitory concentrations. Increased Galactosylated N-glycans Production in Genetically Altered Yeast Cells<br>
Methods to produce human-like N-glycans in yeast and fungal hosts are provided in WO00200879A3 and WO 03056914A1 and are incorporated herein. The skilled artisan recognizes that routine modifications of the procedures disclosed herein in combination with the above methods may provide improved results in the production of the glycoprotein of interest.<br>
In accordance with the methods of the present invention, P. pastoris transformed with at least a p-galactosyltransferase fusion construct pXB53 (Example 4) (Figure 12) produces complex galactosylated glycans in a detectable moiety. At least 10% of galactose residue is transferred onto a glycoprotein in a host cell. In another embodiment, at least 40% of galactose residue is transferred onto a glycoprotein in a host cell. The expression of an epimerase also increases the level of galactose transfer (Figure 13). Preferably, at least 60% of galactose residue is transferred onto a<br>
glycoprotein in a host cell. The expression of another heterologous glycosylation enzyme, such as UGT, further enhances the cellular production of the desired galactosylated glycoproteins. Surprisingly, expression of one such transporter, the DmUGT increases galactose transfer dramatically (Figure 11). In the best mode of the embodiment, a host cell transformed with the DmUGT shows at least 90% or higher galactose transfer.<br>
Preferably, the temperature of the yeast host cell is kept at 37°C to match the temperature optimum of the enzyme.<br>
Additionally, the method also includes isolating these glycoproteins. Expression of UDPase Activity<br>
As described in WO 02/00879, in humans, nucleotide sugar precursors (e.g. UDP-N-acetylglucosamine, UDP-N-acetylgalactosamine, CMP-N-acetylneuraminic acid, UDP-galactose, etc.) are generally synthesized in the cytosol and transported into the Golgi, where they are attached to the core oligosaccharide by glycosyltransferases. To replicate this process in lower eukaryotes, sugar nucleoside specific transporters have to be expressed in the Golgi to ensure adequate levels of nucleoside sugar precursors (Sommers, 1981; Sommers, 1982; Perez, 1987). A side product of the transfer of sugars onto N-glycans is either a nucleoside diphosphate or monophosphate. While monophosphates can be directly exported in exchange for nucleoside triphosphate sugars by an antiport mechanism, diphospho nucleosides (e.g. GDP) have to be cleaved by phosphatases (e.g. GDPase) to yield nucleoside monophosphates and inorganic phosphate prior to being exported. This reaction appears to be important for efficient glycosylation, as GDPase from S. cerevisiae has been found to be necessary for mannosylation.<br><br>
However, the enzyme only has 10% of the activity towards UDP (Berainsone, 1994). Lower eukaryotes often do not have UDP specific diphosphatase activity in the Golgi since they do not utilize UDP-sugar precursors for glycoprotein synthesis in the Golgi.<br>
Engineered yeast strains contain multiple transferase enzymes that utilize UDP-GlcNAc or UDP-galactose as a substrate. This requires the engineering of suitable substrate pools in the yeast Golgi, which in most species does not contain these substrates. However, the endproducts of a transferase reaction utilizing UDP-GlcNAc or UDP-galactose include free UDP. This UDP acts as a potent inhibitor of most transferases that utilize these sugar nucleotides. S. cerevisiae expresses two Golgi proteins with nucleoside diphosphatase activity. One, ScGDAl, is highly specific for GDP (Abeijon et al, 1993). The second, ScYNDl, is an apyrase and thus capable of hydrolyzing both nucleoside tri- as well as di-phosphates and is equally specific for ADP/ATP, GDP/GTP and UDP/UTP (Gao et al, 1999). However, because of the lack of UDP conjugated sugars in the wild-type Golgi and the concomitant lack of transferase enzymes producing UDP as an end product, the possible elevated accumulation of UDP in engineered yeast strain is a significant concern.<br>
Because transfer of galactose residues from the cytosol to the Golgi can be hampered by the lack of UDPase, genetic manipulation to express UDPase may be required for efficient galactose transfer in a lower eukaryotic host cell. Accordingly, in another aspect of the present invention, a method is provided to express, preferably overexpress, a gene encoding for the UDPase. It is contemplated that overexpression of a gene encoding for the UDPase activity increases the availability of the sugar nucleotide UDP-galactose required for galactose transfer onto the acceptor substrates in the Golgi.<br>
To raise the level of UDPase activity in the Golgi of a yeast, several possibilities exist. In one embodiment, a gene encoding UDPase activity, e.g., ScGDAl (NP_010872) is overexpressed, which has some (about 10%) activity towards UDP. In another embodiment, a gene encoding nucleoside diphosphatase activity, e.g., ScYNDl (NP_010920) is overexpressed, which has a higher activity towards UDP compared to GDP, though is not specialized for nucleotide diphosphates. Furthermore, in another embodiment, to achieve the goal of higher UPDase activity in P. pastoris, the S. cerevisiae GDA1 or YND1 is expressed or the P. pastoris homologs of these genes are overexpressed, which are readily identifiable via BLAST homology searches.<br>
Additionally, organisms that utilize these sugar nucleotides are able to convert them to UMP via the action of a nucleotide diphosphatase specific for UDP. An example is the human uridine diphosphatase (UDPase) identified by Wang and Guidotti (AF016032). However, this protein contains two putative transmembrane domains, one at the C-terminus and one at the N-terminus. Accordingly, localization of this protein in the yeast Golgi thus requires fusing the catalytic domain of this protein with a yeast targeting domain.<br>
Other yeasts including K. lactis and S. pombe utilize UDP-sugars in their Golgi to add GlcNAc and galactose, respectively, to their N-glycans. Both K. lactis and S. pombe express homologs of ScGDAl, designated KIGDA1 (Lopez-Avalos et al, 2001; CAC21576) and Spgdal (D'Alessio et al, 2003; NP_593447), respectively, which also have UDPase activity. In case UDP accumulates in engineered yeast strains and proves to be detrimental to the engineered transferases, expression of any or more of these proteins serves to boost UDPase activity to acceptable levels.<br>
Binding affinity to asialoglycoprotein receptors (ASGR)<br>
Another feature of the invention provides less binding affinity to ASGR , which are known to clear asialylated glycoproteins and reduce half-life of a therapeutic protein in the circulatory system. Previous work has shown that glycans having biantennary structures are cleared out less rapidly than glycans having tri or tetra-antennary structures (Stockert, Physiol Rev. 1995 Jul;75(3):591-609). In one aspect of the present invention provides glycans on the protein of interest having a single glycoform (e.g., bi-antennary structures) characterized as having terminal galactose residues. Such bi-antennary structures are not readily produced in mammalian cells because of other GnTs that catalyze tri- and tetra-antennary branching reactions. By capping the substrates having terminal GlcNAc residues with galactose residues, other GnTs (e.g. GnT IV, GnT V) are not present to catalyze the transfer of GlcNAcs onto the galactosylated substrates. Accordingly, the present invention provides methods for producing asialylated glycoproteins having less binding affinity to ASGR in comparison to those glycoproteins produced in mammalian hosts. In a more preferred embodiment, the asialylated glycoprotein is characterized by its increased circulatory half-life and bioactivity in vivo in comparison to heterogeneous glycoproteins produced in mammals. Integration Sites<br>
It is preferable to integrate the nucleic acids encoding the UGT, epimerase and pl,4GalT in a locus that is responsible for mannosyltransferases such as 1,3 mannosyltransferases (e.g. MNN1 in S. cerevisiae) (Graham, 1991), 1,2 mannosyltransferases (e.g. KTRIKRE family from S. cerevisiae), 1,6 mannosyltransferases (OCH1 from S. cerevisiae or P. pastoris), mannosylphosphate<br>
transferases and their regulators {MNN4, PNO1 and MNN6 from S. cerevisiae), vacuolar proteinase A (PEP4), vacuolar protease B (PRB1) GPI-anchored aspartic protease (YPS1) and additional enzymes that are involved in aberrant, immunogenic, i.e. non-human glycosylation reactions.<br>
The mutants with the disrupted locus give rise to a viable phenotype with reduced enzyme activity or eliminated enzyme activity completely. Preferably, the gene locus encoding the initiating a-1,6 mannosyltransferase activity is a prime target for the initial integration of genes encoding glycosyltransferase activity. In a similar manner, one can choose a range of other chromosomal integration sites that, based on a gene disruption event in that locus, are expected to: (1) improve the cell's ability to glycosylate in a more human-like fashion, (2) improve the cell's ability to secrete proteins, (3) reduce proteolysis of foreign proteins and (4) improve other characteristics of the process that facilitate purification or the fermentation process itself.<br>
In an especially preferred embodiment, library DNA is integrated into the site of an undesired gene in a host chromosome, effecting the disruption or deletion of the gene. For example, integration into the sites of the OCH1, MNN1, or MNN4 genes allows the expression of the desired library DNA while preventing the expression of enzymes involved in yeast hypermannosylation of glycoproteins. In other embodiments, library DNA may be introduced into the host via a nucleic acid molecule, plasmid, vector (e.g., viral or retroviral vector), chromosome, and may be introduced as an autonomous nucleic acid molecule or by homologous or random integration into the host genome. In any case, it is generally desirable to include with each library DNA construct at least one selectable marker gene to allow ready selection of host organisms that have been stably<br>
transformed.   Recyclable marker genes such as URA5 (Yeast. 2003 Nov;20(15):1279-90.), which can be selected for or against, are especially suitable. Generating Additional Sequence Diversity<br>
The method of this embodiment is most effective when a nucleic acid, e.g., a DNA library transformed into the host contains a large diversity of sequences, thereby increasing the probability that at least one transformant will exhibit the desired phenotype. Single amino acid mutations, for example, may drastically alter the activity of glycoprotein processing enzymes (Romero et al., 2000). Accordingly, prior to transformation, a DNA library or a constituent sub-library may be subjected to one or more techniques to generate additional sequence diversity. For example, one or more rounds of gene shuffling, error prone PCR, in vitro mutagenesis or other methods for generating sequence diversity, may be performed to obtain a larger diversity of sequences within the pool of fusion constructs. Codon Optimization<br>
It is also contemplated that the nucleic acids of the present invention may be codon optimized resulting in one or more changes in the primary amino acid sequence, such as a conservative amino acid substitution, addition, deletion or combination thereof. Expression Control Sequences<br>
In addition to the open reading frame sequences described above, it is generally preferable to provide each library construct with expression control sequences, such as promoters, transcription terminators, enhancers, ribosome binding sites, and other functional sequences as may be necessary to ensure effective transcription and translation of the fusion proteins upon transformation of fusion constructs into the host organism.<br>
Suitable vector components, e.g., selectable markers, expression control sequences (e.g., promoter, enhancers, terminators and the like) and, optionally, sequences required for autonomous replication in a host cell, are selected as a function of which particular host cell is chosen. Selection criteria for suitable vector components for use in a particular mammalian or a lower eukaryotic host cell are routine. Preferred lower eukaryotic host cells of the invention include Pichia pastoris, Pichia finlandica, Pichia trehalophila, Pichia koclamae, Pichia membranaefaciens, Pichia minuta (Ogataea minuta, Pichia lindneri), Pichia opuntiae, Pichia thermotolerans, Pichia salictaria, Pichia guercuum, Pichia pijperi, Pichia stiptis, Pichia methanolica, Pichia sp., Saccharomyces cerevisiae, Saccharomyces sp., Hansenula polymorpha, Kluyveromyces sp., Kluyveromyces lactis, Candida albicans, Aspergillus nidulans, Aspergillus niger, Aspergillus oryzae, Trichoderma reesei, Chrysosporium lucknowense, Fusarium sp., Fusarium graminewn, Fusarium venenatum, Physcomitrella patens and Neurospora crassa.<br>
Where the host is Pichia pastoris, suitable promoters include, for example, the AOX1, AOX2, GAPDH, OCH1, SEC4, D2 and P40 promoters. Selectable Markers<br>
It is also preferable to provide each construct with at least one selectable marker, such as a gene to impart drug resistance or to complement a host metabolic lesion. The presence of the marker is useful in the subsequent selection of transformants; for example, in yeast the URA5, URA3, HIS4, SUC2, G418, BLA, or SH BLE genes may be used. A multitude of selectable markers are known and available for use in yeast, fungi, plant, insect, mammalian and other eukaryotic host cells.<br>
Transformation<br>
In yeast, any convenient method of DNA transfer may be used, such as electroporation, the lithium chloride method, or the spheroplast method. In filamentous fungi and plant cells, conventional methods include particle bombardment, electroporation and agrobacterium mediated transformation. To produce a stable strain suitable for high-density culture (e.g., fermentation in yeast), it is desirable to integrate the fusion constructs into the host chromosome. In a preferred embodiment, integration occurs via homologous recombination, using techniques well-known in the art. Preferably, stable genetic modification of P. pastoris occurs via a double cross-over event. Nett et al.,Yeast. 2003 Nov;20(15):1279-90. For example, the heterologous enzyme activities are provided with flanking sequences homologous to sequences of the host organism and successively transformed reusing a single marker. In this manner, integration occurs at a defined site in the host genome using a recyclable marker. Screening and Selection Processes<br>
After transformation of the host strain with the heterologous enzymes, transformants displaying a desired glycosylation phenotype are selected. Selection may be performed in a single step or by a series of phenotypic enrichment and/or depletion steps using any of a variety of assays or detection methods. Phenotypic characterization may be carried out manually or using automated high-throughput screening equipment. Commonly, a host microorganism displays protein W-glycans on the cell surface, where various glycoproteins are localized.<br>
One may screen for those cells that have the highest concentration of terminal GlcNAc on the cell surface, for example, or for those cells which secrete the protein with<br>
the highest terminal GlcNAc content. Such a screen may be based on a visual method, like a staining procedure, the ability to bind specific terminal GlcNAc binding antibodies or lectins conjugated to a marker (such lectins are available from E.Y. Laboratories Inc., San Mateo, CA), the reduced ability of specific lectins to bind to terminal mannose residues, the ability to incorporate a radioactively labeled sugar in vitro, altered binding to dyes or charged surfaces, or may be accomplished by using a Fluorescence Assisted Cell Sorting (FACS) device in conjunction with a fluorophore labeled lectin or antibody (Guillen, 1998).<br>
Accordingly, intact cells may be screened for a desired glycosylation phenotype by exposing the cells to a lectin or antibody that binds specifically to the desired N-glycan. A wide variety of oligosaccharide-specific lectins are available commercially (e.g., from EY Laboratories, San Mateo, CA). Alternatively, antibodies to specific human or animal Af-glycans are available commercially or may be produced using standard techniques. An appropriate lectin or antibody may be conjugated to a reporter molecule, such as a chromophore, fluorophore, radioisotope, or an enzyme having a chromogenic substrate (Guillen et al., 1998. Proc. Natl. Acad. Sci. USA 95(14): 7888-7892)).<br>
Screening may then be performed using analytical methods such as spectrophotometry, fluorimetry, fluorescence activated cell sorting, or scintillation counting. In other cases, it may be necessary to analyze isolated glycoproteins or A/-glycans from transformed cells. Protein isolation may be carried out by techniques known in the art. In a preferred embodiment, a reporter protein is secreted into the medium and purified by affinity chromatography (e.g. Ni-affinity or glutathione -S-<br>
transferase affinity chromatography). In cases where an isolated 7V-glycan is preferred, an enzyme such as endo-p-Af-acetylglucosaminidase (Genzyme Co., Boston, MA; New England Biolabs, Beverly, MA) may be used to cleave the N-glycans from glycoproteins. Isolated proteins or W-glycans may then be analyzed by liquid chromatography (e.g. HPLC), mass spectroscopy, or other suitable means. U.S. Patent No. 5,595,900 teaches several methods by which cells with desired extracellular carbohydrate structures may be identified. In a preferred embodiment, MALDI-TOF mass spectrometry is used to analyze the cleaved N-glycans.<br>
Prior to selection of a desired transformant, it may be desirable to deplete the transformed population of cells having undesired phenotypes. For example, when the method is used to engineer a functional mannosidase activity into cells, the desired transformants will have lower levels of mannose in cellular glycoprotein. Exposing the transformed population to a lethal radioisotope of mannose in the medium depletes the population of transformants having the undesired phenotype, i.e. high levels of incorporated mannose (Huffaker TC and Robbins PW., Proc Natl AcadSci USA. 1983 Dec;80(24):7466-70). Alternatively, a cytotoxic lectin or antibody, directed against an undesirable W-glycan, may be used to deplete a transformed population of undesired phenotypes (e.g., Stanley P and Siminovitch L. Somatic Cell Genet 1977 Jul;3(4):391-405). U.S. Patent No. 5,595,900 teaches several methods by which cells with a desired extracellular carbohydrate structures may be identified. Repeatedly carrying out this strategy allows for the sequential engineering of more and more complex glycans in lower eukaryotes.<br>
To detect host cells having on their surface a high degree of the human-like N-glycan intermediate Ga^GlcNAcaManaGlcNAci, for example, one may select for transformants that allow for the most efficient transfer of Galactose by GalT from UDP-Galactose in an in vitro cell assay. This screen may be carried out by growing cells harboring the transformed library under selective pressure on an agar plate and transferring individual colonies into a 96-well microtiter plate. After growing the cells, the cells are centrifuged, the cells resuspended in buffer, and after addition of UDP-Galactose and GalT, the release of UDP is determined either by HPLC or an enzyme linked assay for UDP. Alternatively, one may use radioactively labeled UDP-Galactose and GalT, wash the cells and then look for the release of radioactive Galactose by P-galactosidase. All this may be carried manually or automated through the use of high throughput screening equipment. Transformants that release more UDP, in the first assay, or more radioactively labeled Galactose in the second assay, are expected to have a higher degree of Gal2GlcNAc2Man3GlcNAc2 on their surface and thus constitute the desired phenotype. Similar assays may be adapted to look at the N-glycans on secreted proteins as well.<br>
Alternatively, one may use any other suitable screen such as a lectin binding assay that is able to reveal altered glycosylation patterns on the surface of transformed cells. In this case the reduced binding of lectins specific to terminal mannoses may be a suitable selection tool. Galantus nivalis lectin binds specifically to terminal a-1,3 mannose, which is expected to be reduced if sufficient mannosidase II activity is present in the Golgi. One may also enrich for desired transformants by carrying out a chromatographic separation step that allows for the removal of cells containing a high terminal mannose<br>
content. This separation step would be carried out with a lectin column that specifically binds cells with a high terminal mannose content (e.g., Galantus nivalis lectin bound to agarose, Sigma, St.Louis, MO) over those that have a low terminal mannose content. Host Cells<br>
Although the present invention is exemplified using P. pastor is as a host organism, it is understood by those skilled in the art that other eukaryotic host cells, including other species of yeast and fungal hosts, may be altered as described herein to produce humanlike glycoproteins. Such hosts include preferably Pichia Jinlandica, Pichia trehalophila, Pichia koclamae, Pichia membranaefaciens, Pichia minuta (Ogataea minuta, Pichia lindneri), Pichia opuntiae, Pichia thermotolerans, Pichia salictaria, Pichia guercuum, Pichia pijperi, Pichia stiptis, Pichia methanolica, Pichia sp., Saccharomyces cerevisiae, Saccharomyces sp., Hansenula polymorpha, Kluyveromyces sp., Kluyveromyces lactis, Candida albicans, Aspergillus nidulans, Aspergillus niger, Aspergillus oryzae, Trichoderma reesei, Chrysosporium lucknowense, Fusarium sp., Fusarium gramineum, Fusarium venenatum, Physcomitrella patens and Neurospora crassa.<br>
The techniques described herein for identification and disruption of undesirable host cell glycosylation genes, e.g. OCH1, is understood to be applicable for these and/or other homologous or functionally related genes in other eukaryotic host cells such as other yeast and fungal strains (See WO 02/00879). Additionally, other preferred host cells are deficient in Alg3p encoding for Dol-P-Man:MansGlcNAc2-PP-Dol mannosyltransferase activity (See WO 03/056914).<br>
Preferred host cells are yeast and filamentous fungal hosts, which inherently lack pl,4-galactose linkages, fucose, and terminal sialic acid. Unlike the W-glycans of<br><br>
mammalian glycoproteins these sugars are not usually found on glycoproteins produced in yeast and filamentous fungi. The present invention provides methods for engineering host cells to produce galactose residues onto glycoproteins and essentially lack fucose and sialic acid residues on the glycoproteins. In another embodiment, those host cells that produce fucose or sialic acid can be modified to have reduced or eliminated fucosyltransferase activity or sialyltransferase activity. The glycoprotein compositions produced from the host of the present invention are, therefore, essentially free of fucose and sialic acid residues. A significant advantage of the present invention is that the host cells produce galactosylated, fucose-free and sialic acid-free glycoproteins without ex vivo modification with fucosidase and sialidase treatment.<br>
Other preferred host cells include fungal hosts that lack mannosylphosphorylation with respect to glycans (USSN 11/020,808). Still other preferred host cells include fungal hosts that lack p-mannosylation with respect to glycans (USSN 60/566,736).<br>
Another aspect of the present invention thus relates to a non-human eukaryotic host strain expressing glycoproteins comprising modified W-glycans that resemble those made by human-cells. Performing the methods of the invention in species other than yeast and fungal cells is thus contemplated and encompassed by this invention. It is contemplated that a combinatorial nucleic acid library of the present invention may be used to select constructs that modify the glycosylation pathway in any eukaryotic host cell system. For example, the combinatorial libraries of the invention may also be used in plants, algae and insects, and in other eukaryotic host cells, including mammalian and human cells, to localize proteins, including glycosylation enzymes or catalytic domains thereof, in a desired location along a host cell secretory pathway. Preferably,<br>
glycosylation enzymes or catalytic domains and the like are targeted to a subcellular location along the host cell secretory pathway where they are capable of functioning, and preferably, where they are designed or selected to function most efficiently.<br>
Examples of modifications to glycosylation which can be affected using a method according to this embodiment of the invention are: (1) engineering a eukaryotic host cell to trim mannose residues from MangGlcNAc2 to yield a ManjGlcNAca JV-glycan; (2) engineering eukaryotic host cell to add an Af-acetylglucosamine (GlcNAc) residue to MansGlcNAc2 by action of GlcNAc transferase I; (3) engineering a eukaryotic host cell to functionally express an enzyme such as an JV-acetylglucosaminyl Transferase (GnTI, GnTII, GnTIII, GnTIV, GnTV, GnTVI, GnTIX), mannosidase II, fucosyltransferase (FT), galactosyl transferase (GalT) or a sialyltransferase (ST).<br>
By repeating the method, increasingly complex glycosylation pathways can be engineered into a target host, such as a lower eukaryotic microorganism. In one preferred embodiment, the host organism is transformed two or more times with DNA libraries including sequences encoding glycosylation activities. Selection of desired phenotypes may be performed after each round of transformation or alternatively after several transformations have occurred. Complex glycosylation pathways can be rapidly engineered in this manner. Target Glycoproteins<br>
The methods described herein are useful for producing glycoproteins, especially glycoproteins used therapeutically in humans. Glycoproteins having specific glycoforms may be especially useful, for example, in the targeting of therapeutic proteins. For example, mannose-6-phosphate has been shown to direct proteins to the lysosome, which<br>
may be essential for the proper function of several enzymes related to lysosomal storage disorders such as Gaucher's, Hunter's, Hurler's, Scheie's, Fabry's and Tay-Sachs disease, to mention just a few. Likewise, the addition of one or more sialic acid residues to a glycan side chain may increase the lifetime of a therapeutic glycoprotein in vivo after administration. Accordingly, host cells (e.g., lower eukaryotic or mammalian) may be genetically engineered to increase the extent of terminal sialic acid in glycoproteins expressed in the cells. Alternatively, sialic acid may be conjugated to the protein of interest in vitro prior to administration using a sialic acid transferase and an appropriate substrate. Changes in growth medium composition may be employed in addition to the expression of enzyme activities involved in human-like glycosylation to produce glycoproteins more closely resembling human forms (S. Weikert, et al., Nature Biotechnology, 1999, 17, 1116-1121; Werner, Noe, et al 1998 Arzneimittelforschung 48(8):870-880; Weikert, Papac et al., 1999; Andersen and Goochee 1994 Cur. Opin. Biotechnol. 5: 546-549; Yang and Butler 2000 Biotechnol.Bioengin. 68(4): 370-380). Specific glycan modifications to monoclonal antibodies (e.g. the addition of a bisecting GlcNAc) have been shown to improve antibody dependent cell cytotoxicity (Umana P., et al. 1999), which may be desirable for the production of antibodies or other therapeutic proteins.<br>
Therapeutic proteins are typically administered by injection, orally, pulmonary, or other means. Examples of suitable target glycoproteins which may be produced according to the invention include, without limitation: erythropoietin, cytokines such as interferon-a, interferon-p, interferon-y, interferon-o, and granulocyte-CSF, GM-CSF, coagulation factors such as factor VIII, factor IX, and human protein C, antithrombin III,<br>
thrombin, soluble IgE receptor a-chain, IgG, IgG fragments, IgG fusions, IgM, interleukins, urokinase, chymase, and urea trypsin inhibitor, IGF-binding protein, epidermal growth factor, growth hormone-releasing factor, annexin V fusion protein, angiostatin, vascular endothelial growth factor-2, myeloid progenitor inhibitory factor-1, osteoprotegerin, oc-1-antitrypsin, a- feto proteins, DNase II, kringle 3 of human plasminogen, glucocerebrosidase, TNF binding protein 1, follicle stimulating hormone, cytotoxic T lymphocyte associated antigen 4 - Ig, transmembrane activator and calcium modulator and cyclophilin ligand, soluble TNF receptor Fc fusion, glucagon like protein 1 and IL-2 receptor agonist. Secretory Signal Sequence<br>
It is also preferred to associate a nucleic acid sequence encoding a secretory signal with a sequence of interest encoding the glycoprotein. The term "secretory signal sequence" denotes a DNA sequence that encodes a polypeptide (a "secretory peptide") that, as a component of a larger polypeptide, directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized. The larger polypeptide is commonly cleaved to remove the secretory peptide during transit through the secretory pathway. To direct a polypeptide into the secretory pathway of a host cell, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) is provided in an expression vector. The secretory signal sequence may be that of, without limitation, a wild-type sequence related to a glycoprotein, sequence encoding S. cerevisiae Suc2 signal sequence, sequence encoding Pichia Pho2 signal sequence, sequence encoding Pichia Prcl signal sequence, sequence encoding S. cerevisiae alpha-mating factor (aMF) signal sequence, sequence encoding bovine lysozyme C signal<br>
sequence. The secretory signal sequence is operably linked to a nucleic acid sequence, i.e., the two sequences are joined in the correct reading frame and positioned to direct the newly synthesized polypeptide into the secretory pathway of the host cell. Secretory signal sequences are commonly positioned 5' to the DNA sequence encoding the polypeptide of interest, although certain secretory signal sequences may be positioned elsewhere in the DNA sequence of interest (See, e.g., Welch et al., U.S. Pat. No. 5,037,743; Holland et al., U.S. Pat. No. 5,143,830).<br>
Alternatively, the secretory signal sequence contained in the polypeptides of the present invention is used to direct other polypeptides into the secretory pathway. The present invention provides for such fusion polypeptides. The secretory signal sequence contained in the fusion polypeptides of the present invention is preferably fused amino-terminally to an additional peptide to direct the additional peptide into the secretory pathway. Such constructs have numerous applications known in the art. For example, these novel secretory signal sequence fusion constructs can direct the secretion of an active component of a normally non-secreted protein, such as a receptor. Such fusions may be used in vivo or in vitro to direct peptides through the secretory pathway.<br>
Glycoproteins produced by the methods of the present invention can be isolated by techniques well-known in the art. The desired glycoproteins are purified and separated by methods such as fractionation, ion exchange, gel filtration, hydrophobic chromatography and affinity chromatography.<br>
The following are examples which illustrate the compositions and methods of this invention. These examples should not be construed as limiting: the examples are included for the purposes of illustration only.<br>
EXAMPLE 1<br>
Construction of promoter cassettes and expression vectors<br>
The SOObp promoter for the PpOCHl gene was amplified using primers RCD48 (SEQ ID NO: 15) (5'-TATGCGGCCGCGGCTGATGATATTTGCTACGA-3') and RCD134 (SEQ ID NO:16) (5'-CCTCTCGAGTGGACACAGGAGACTCAGAAACAG-3') and the 400bp promoter for the PpSEC4 gene was amplified using primers RCD156 (SEQ ID NO:17) (5'-CTTCTCGAGGAAGTAAAGTTGGCGAAACTT-3') and RCD157 (SEQ ID NO:18) (5'-CTTAGCGGCCGCGATTGTTCGTTTGAGTAGTTT-3'). The PCR products were cloned into the pCRl.l cloning vector (Invitrogen) and sequenced. The OCH1 and SEC4 promoters were then subcloned into the vector pJN261 (Nett et al., Yeast. 2003 Nov;20(15):1279-90) in place of the GAPDH promoter using the introduced XhollNotl restriction sites to create plasmids pRCD360 and pRCD362, respectively.<br>
The PpHIS3 promoter was PCR amplified using primers RCD152 (SEQ ID NO:19) (5'-CTTCTCGAGGGCATTCAAAGAAGCCTTGGG-3') and RCD153 (SEQ ID NO:20) (5'-CTTAGCGGCCGCTGAGTGGTCATGTGGGAACTT-3'), cloned into plasmid pCR2.1 and sequenced. The XhollNotl sites were then used to subclone the PpHISS promoter into plasmid pTA18 to replace the PpPMAl strong promoter with the weaker PpHISS promoter and create plasmid pRCD351, which is a NATR plasmid that rolls into the PpHISS promoter.<br>
A portion of the PpHISS gene was amplified using primers RCD301 (SEQ ID NO:21) (5'-CCTGGATCCAACAGACTACAATGACAGGAG-3') and RCD302 (SEQ ID NO:22) (5'-CCTGCATGCCTCGAGCTTGCCGGCGTCTAAATAGCCGTTGAAG-<br>
3') and inserted into pUC19 using the BamHl/Sphl restriction sites to create plasmid pRCD391. This vector contains a 1.2 Kb portion of the PpHISS locus as well as Xhol and NgoMlV sites engineered into the primer RCD302 (SEQ ID NO:22). The G418R gene was inserted as a Bglll/Sacl fragment from pUG6 (Wach et al., 1994) into the BamHl/Sacl sites of pRCD391 to create pRCD392.<br>
A 1.2 Kb portion of the PpTRPl gene was amplified from P. pastoris genomic DNA with primers RCD307 (SEQ ID NO.23) (5'-CCTGTCGACGCTGCCGGCAAGCTCGAGTTTAAGCGGTGCTGC-3') and RCD308 (SEQ ID NO:24) (5'-CCTGGATCCTTTGGCAAAAACCAGCCCTGGTGAG-3'). The amplified fragment was inserted into pUC19 using BamHI/Sall sites to create plasmid pRCD399. The PAT gene conferring resistance to phosphinothricin was released from plasmid pAG29 (Goldstein and McCusker, 1999) using Bglll/Sacl and inserted into pRCD399 digested with BamHl/Sacl to create the PpTRPlfPAl roll-in plasmid pRCD401. EXAMPLE 2<br>
Cloning of Galactose Transporters Schizosaccharomyces pombe UDP Galactose Transporter<br>
The S. pombe gene encoding the UDP Galactose Transporter (SpGMSl+,<br>
Genbank AL022598) referred to as SpUGT was PCR amplified from S. pombe genomic<br>
DNA (ATCC24843) in two pieces to eliminate a single intron. Primers RCD164 (SEQ<br>
ID NO:7) (5'-CCTTGCGGCCGCATGGCTGTCAAGGGCGACGATGTCAAA-3') and<br>
RCD177	(SEQ	ID	NO:8)	(5'-<br>
ATTCGAGAATAGTTAAGTGTCAAAATCAATGCACTATTTT-3')   were   used   to<br>
amplify the 5' 96bp of the gene and primers RCD176 (SEQ ID NO:9) (5'-AAAATAGTGCATTGATTTTGACACTTAACTATTCTCGAAT-3') and RCD165 (SEQ ID NO:10) (5'-CCTTTTAATTAATTAATGCTTATGATCAACGTCCTTAGC-3') to amplify the 3' 966bp. Subsequently, primers RCD164 (SEQ ID NO:7) and RCD165 (SEQ ID NO: 10) were used to overlap the two amplified products into a single PCR fragment comprising one contiguous open reading frame with Notl and Pad sites introduced at the ends. This PCR product was cloned into the pCR2.1 vector (Invitrogen) and sequenced. The Notl and Pad sites were then used to subclone this gene into plasmid pJN335, which contains a cassette that fuses a gene downstream of the P. pastoris GAPDH promoter. The 400bp PpOCHl transcriptional terminator was then PCR amplified using primers RCD202 (SEQ ID NO:25) (5'-TCCTTAATTAAAGAAAGCTAGAGTAAAATAGAT-3') and RCD203 (SEQ ID NO:26) (5'-TCCCTCGAGGATCATGTTGATCAACTGAGACCG-3') and cloned into pCR2.1. Subsequently a triple ligation was performed to insert the GAPDH promoter/SpUGT gene fusion as an XhollPacl fragment and the PpOCHl-TT as a Pad/Xhol fragment into a single Xhol site in plasmid pTA18 to create plasmid pRCD257. The new plasmid, pRCD257, is a NATR containing vector that contains the GAPDH-SpGALE-OCHITT fusion along with a second cassette that contains a truncated version of the human GnTII gene fused to the ScVANl transmembrane domain driven by the PpPMAI promoter.<br>
The SpUGT gene was also inserted into the Notl/Pad sites of pRCD360 with the OCH1 promoter and pRCD362 with the SEC4 promoter to create plasmids pRCD385 and pRCD387, respectively. The POcm-SpUGT-PpCYClTT cassette from pRCD385<br>
and PSEC4-SpUGT-PpCYClTI cassette from pRCD387 were inserted into the pRCD392 HIS3/G418R roll-in vector using Xhol/NgoMlV to create P. pastoris ///5'3/G418R roll-in expression plasmids pRCD393 and pRCD394, respectively. Drosophila melanogaster UDP Galactose Transporter<br>
The D. melanogaster gene encoding the UDP Galactose Transporter (Genbank BAB62747) referred to as DmUGTwas PCR amplified from a D. melanogaster cDNA library (UC Berkeley Drosophila Genome Project, ovary X-ZAP library GM) and cloned into the pCR2.1 PCR cloning vector and sequenced. Primers DmUGT-5' (SEQ ID<br>
GGCTCGAGCGGCCGCCACCATGAATAGCATACACATGAACGCCAATACG-3') and DmUGT-3' (SEQ ID NO:12) (5'-<br>
CCCTCGAGTTAATTAACTAGACGCGCGGCAGCAGCTTCTCCTCATCG-3') were used to amplify the gene, which introduced Notl and Pad sites at the 5' and 3' ends, respectively. The Notl and Pad sites were then used to subclone this gene fused downstream of the PpOCHl and promoter at the Notl/PacI sites in pRCD393 to create plasmid pSH263. Homo sapiens UDP Galactose Transporter<br>
The H. sapiens genes encoding the UDP Galactose Transporter 1 (Genbank #BAA95615) and UDP Galactose Transporter 2 (Genbank #BAA95614) referred to as hUGTl and hUGT2, respectively, were amplified from human prostate cDNA (marathon ready dDNA, Clontech). The HUGT1 gene was amplified with primers hUGTl-5' (SEQ IDNO:27)(5'-GGCTCGAGCGGCCGCCACCATGGCAGCGGTTGGGGCTGGTGGTTC-3') and<br>
hUGTl-3' (SEQ ID NO:28) (5'-<br>
CCCTCGAGTTAATTAATCACTTCACCAGCACTGACTTTGGCAG-3') and the hUGT2 gene was amplified with primers MJGT2-5' (SEQ ID NO:29) (5'-GGCTCGAGCGGCCGCCACC ATGGCAGCGGTTGGGGCTGGTGGTTC-3') and MJGT2-3' (SEQ ID NO:30) (5'-<br>
CCCTCGAGTTAATTAACTAGGAACCCTTCACCTTGGTGAGCAAC-3'). The PCR products were cloned into the pCR2.1 vector (Invitrogen, Carlsbad, CA) and sequenced. The hUGTl and hUGT2 genes were subsequently inserted into pRCD393 downstream of the PpOCHl promoter using NotllPacl to create plasmids pSH264 and pSH262, respectively. EXAMPLE 3<br>
Cloning UDP-Galactose-4-Epimerase Genes S. cerevisiae UDP-galactose 4-epimerase<br>
The S. cerevisiae gene encoding UDP-galactose 4-epimerase (ScGALlO) was PCR amplified from S. cerevisiae genomic DNA using primers RCD270 (SEQ ID NO:31) (5'-TAGCGGCCGCATGACAGCTCAGTTACAAAGTGAAAG-3') and RCD271 (SEQ ID NO:32) (5'-<br>
CGTTAATTAATCAGGAAAATCTGTAGACAATCTTGG-3'). The resulting PCR product was cloned into pCRZ.l and sequenced.<br>
The ScGALlO gene was then subcloned using the NotllPacl sites into plasmids pRCD393 and pRCD394 to create plasmids pRCD395 and pRCD396, respectively and also into plasmids pRCD402 and pRCD403 to create plasmids pRCD404 and pRCD405, respectively. Plasmids pRCD402 and pRCD403 are expression vectors<br><br>
containing the P. pastoris OCH1 and SEC4 promoters, respectively, and the PpCYCl<br>
terminator and convenient restriction sites that were used to fuse the epimerases with<br>
these promoters and create a cassette that could be collectively moved into another<br>
plasmid.<br>
Homo sapiens UDP-galactose 4-epimerase<br>
The H. sapiens gene encoding UDP-galactose 4-epimerase (Thoden et al., (2001) JBC Vol. 276 (18) 15131-15136.), referred to as hGALE was PCR amplified from human kidney cDNA (marathon ready cDNA, Clontech) using primers GD7 (SEQ ID N0:33) and GD8 (SEQ ID NO:34) with NotI and Pad sites respectively, cloned into pCR2.1 and sequenced. The hGALE gene was then subcloned using Notl/Pacl sites into plasmids pRCD406 and pRCD407 to create plasmids pRCD427 and pRCD428, respectively. S. pombe UDP-galactose 4-epimerase<br>
Primers GALE2-L (SEQ ID NO.35) and GALE2-R (SEQ ID NO:36) were used to amplify the SpGALE gene from S. pombe (ATCC24843) genomic DNA. The amplified product was cloned into pCRZ.l and sequenced. Sequencing revealed the presence of an intron (175bp) at the +66 position.<br>
To eliminate the intron, upstream primer GDI (SEQ ID NO:37) (94 bases) was designed. It has a NotI site, 66 bases upstream of the intron, followed by 20 bases preceding the intron. GD2 (SEQ ID NO:38) is the downstream primer and has a Pad site. Primers GDI (SEQ ID NO:37) and GD2 (SEQ ID N0:38) were used to amplify the SpGALE intronless gene from the pCRZ.l subclone and the product cloned again into pCR2.1 and sequenced. EXAMPLE 4<br>
Cloning of p-l,4-Galactosyltransferase Genes<br>
Homo sapiens p-1.4-galactosvrtransferase I<br>
The H. sapiens |3-l,4-galactosyltransferase I gene (HGalTI, Genbank AH003575) was PCR amplified from human kidney cDNA (marathon ready cDNA, Clontech) using primers RCD192 (SEQ IDNO:1) (5'-GCCGCGACCTGAGCCGCCTGCCCCAAC-3') and RCD186 (SEQ ID NO:2) (5'-CTAGCTCGGTGTCCCGATGTCCACTGT-3'). This PCR product was cloned into pCRZ.l vector (Invitrogen, Carlsbad, CA) and sequenced. From this clone, a PCR overlap mutagenesis was performed for three purposes: 1) to remove a Notl site within the open reading frame while maintaining the wild-type protein sequence, 2) to truncate the protein immediately downstream of the endogenous transmembrane domain, 3) and to introduce Ascl and Pad sites at the 5' and 3' ends for modular cloning. To do this, the 5' end of the gene up to the Notl site was amplified using primers RCD198 (SEQ ID NO:3) (5'-<br>
CTTAGGCGCGCCGGCCGCGACCTGAGCCGCCTGCCC-3') and RCD201 (SEQ ID NO:4) (5'-GGGGCATATCTGCCGCCCATC-3') and the 3' end was amplified with primers RCD200 (SEQ ID NO:5) (5'-GATGGGCGGCAGATATGCCCC-3') and RCD199 (SEQ ID NO:6) (5'-<br>
CTTCTTAATTAACTAGCTCGGTGTCCCGATGTCCAC-3'). The products were overlapped together with primers 198 and 199 to resynthesize the ORF with the wild-type amino acid sequence while eliminating the Notl site. The new truncated /iGa/77PCR product was cloned into pCR2.1 vector (Invitrogen, Carlsbad, CA) and sequenced. The introduced AscllPacl sites were then used to subclone the fragment into plasmid pRCD259, which is a PpURA3IHYGR roll-in vector, to create pRCD260  A library of<br>
yeast targeting sequence transmembrane domains as described in WO 02/00879, which is incorporated by reference, was ligated into the Notl/Ascl sites located upstream of the hGalTl gene to create plasmids pXB20-pXB67. Homo sapiens p-l,4-galactosyltransferase II<br>
A truncated form of the H. sapiens p-l,4-galactosyltransferase II gene (hGalTII, Genbank AF038660) was PCR amplified from human kidney cDNA (marathon ready cDNA, Clontech) using primers RCD292 (SEQ ID NO:39) (5'-CTTAGGCGCGCCCAGCACCTGGCCTTCTTCAGC-3') and RCD293 (SEQ ID NO:40) (5'-CTTGTTAATTAATCAGCCCCGAGGGGGCCACGACGG-3')&gt; cloned into plasmid pCRl.l and sequenced. This truncated clone, which eliminates part of the gene encoding the N-terminal transmembrane domain, was subcloned using the introduced Ascl/Pacl sites into vector pXB53 in place of hGalTl to create plasmid pRCD378. The plasmid, containing the gene fusion of the truncated hGalTII with the transmembrane domain/leader sequence-encoding portion of the S. cerevisiae MNN2 gene is driven by the PpGAPDH promoter. Homo sapiens p-l,4-galactosvltransferase III<br>
A truncated form of the H. sapiens p-l,4-galactosyltransferase III gene (hGalTIII, Genbank AF038661) was PCR amplified from human kidney cDNA (marathon ready cDNA, Clontech) using primers RCD294 (SEQ ID NO.41) (5'-CTTAGGCGCGCCCGAAGTCTCAGTGCCCTATTTGGC-3') and RCD295 (SEQ ID NO-.42) (5'-CTTGTTAATTAATCAGTGTGAACCTCGGAGGGCTGT-3'), cloned into plasmid pCR2.1 and sequenced. This truncated clone, which eliminates part of the gene encoding the N-terminal transmembrane domain, was subcloned using the introduced<br>
AscllPacl sites into vector pXB53 in place of hGalTI to create plasmid pRCD381. This<br>
plasmid now contains a gene fusion of the truncated HGalTIII with the transmembrane<br>
domain/leader sequence-encoding portion of the 5". cerevisiae MNN2 gene driven by the<br>
PpGAPDH promoter.<br>
EXAMPLE 5<br>
Expression of hGalTI with SpUGTin a strain producing complex N-glycans<br>
The pRCD257 plasmid containing the human GnTII gene and the SpGMSl+ gene (SpUGT) was introduced into strain RDP27. RDP27 is a mutant strain of P. pastoris that has ochl and alg3 deletions, and that has been transformed with plasmids pSH49 and pPB104 which contain active fusion constructs of mouse Mannosidase IB and human GnTI, respectively as well as plasmid pPB103, which contains the K. lactis UDP-GlcNAc transporter and pBK64 which contains the reporter protein K3 (Choi et al. 2003). After selection on nourseothricin, 16 transformants were selected to determine the glycosylation of the expressed reporter protein K3. In two of these transformants, the expected complex human glycosylation structure GlcNAciManaGlcNAci was observed and these strains were designated RDP30-10 (Figure 8A) and RDP30-13. A portion of the hGalTI gene/leader fusion plasmid library was transformed into strain RDP30-10 and transformants were selected on minimal medium containing hygromycin. N-glycans released from K3 secreted by the resulting strains were analyzed on MALDI-TOF MS. A molecular shift in mass consistent with the addition of one galactose sugar was observed on N-glycans from transformants of two different leader constructs, pXB53 and pXB65. The first, pXB53 consists of hGalTI fused to the ScMnn2(s) leader (referred to here as 5'cMnn2(s)/hGalTI) and the other was a<br>
fusion with the ScMnnl(m) leader.   Analysis of the N-glycans released from K3 from RDP37 (RDP30-10 transformed with pXB53) by MALDI-TOF revealed approximately 10-20% GlcNAc2Man3GlcNAc2 being converted to GalGlcNAc2Man3GlcNAc2 and a lesser amount (1-2%) to Gal2GlcNAc2Man3GlcNAc2 (pXB53, Figure 8B).   A lesser amount   of conversion   (3-5%)   to   GalGlcNAc2Man3GlcNAc2   but   no   observable Gal2GlcNAc2Man3GlcNAc2 was observed for the second fusion (pXB65). EXAMPLE 6 Expression ofhGalTIand ScGALlO in a strain producing hybrid N-glycans<br>
The ScGALlO gene encoding UDP-galactose 4-epimerase was subcloned with NotllPacl into the NATR vectors pTA18 and pRCD351 in place of hGnTII, which inserts the epimerase gene in front of the strong PMAI promoter and the weaker PpHIS3 promoter, respectively, to create plasmids pRCD331 (PpMAi-ScGALlO) and pRCD352 (Pftiss-ScGALlO), respectively. The plasmids were linearized (pRCD331 with Sad in the PpPMAl promoter and pRCD352 with 5g/II in the PpHIS3 promoter) and transformed into strain PBP-3 (US Pat. Appl. No. 20040018590). Strain PBP-3 is a mutant strain of P. pastoris, which has an ochl deletion and has been transformed with plasmids pSH49 and pPB104 which contain active fusion constructs of mouse Mannosidase IB and human GnTI, respectively as well as plasmid pPB103, which contains the K. lactis UDP-GlcNAc transporter and plasmid pBK64 which contains the reporter protein K3 (Choi et al.). This strain produces hybrid N-glycans of the structure GlcNAcMansGlcNAc2 on secreted proteins. Resulting transformants selected on YPD medium containing Nourseothricin were analyzed by PCR with primers RCD285 (SEQ ID NO:43) (5'-TACGAGATTCCCAAATATGATTCC-3') and RCD286 (SEQ ID NO:44) (5s-<br>
ATAGTGTCTCCATATGGCTTGTTC-3') and by expressing the reporter protein K3 and analyzing the released N-glycans to ensure that the strains maintained the hybrid GlcNAcMansGlcNAca glycan structure. One strain transformed with the pRCD352 (Pmss-ScGALlO) construct was designated RDP38-18. This strain was transformed with the plasmid pXB53 (containing the Mnn2(s)/hGalTI fusion construct and the HYGR and PpURAS genes) after linearization with Sail (located in PpURA3). Transformants were selected on YPD medium with Hygromycin and screened by expressing K3 and determining the size of the N-glycans. A large portion (~2/3) of the N-glycans released from K3 purified from RDP39-6 strains (Figure 10A) contained one additional hexose (HexGlcNAcMansGlcNAca) as compared with those from RDP38-18, which were mostly GlcNAcMansGlcNAci. Furthermore, the additional hexose residue could be removed by subsequent incubation with soluble (5-1,4-galactosidase, but not oc-1,3-galactosidase or a-l,2-mannosidase, indicating that the addition of a single galactose to the terminal GlcNAc with a specific linkage (P-1,4) was catalyzed by hGalTI in this strain.<br>
EXAMPLE 7 Expression ofhGalTI and ScGALlO in a strain producing complex N-glycans<br>
The P. pastoris strain YSH-44 was constructed, which displays complex N-glycans with a GlcNAc2Man3GlcNAc2 structure. YSH-44 is a mutant strain of P. pastoris deleted for ochl and transformed with plasmids pSH49, pPB104, pKD53, and pTC53 which contain active fusion constructs of mouse Mannosidase IB, human GnTI, D. melanogaster Mannosidase II, and human GnTII, respectively as well as plasmid pPB103, which contains the K. lactis UDP-GlcNAc transporter and plasmid pBK64<br>
which contains the reporter protein K3 (Hamilton et al., Science. 2003 Aug 29;301(5637): 1244-6.). This strain was transformed with the pXBS3 plasmid containing a Mnn2(s)/hGalTI fusion construct and transformants were selected on YPD medium with hygromycin. Several transformants were analyzed by purifying K3 and analyzing the released N-glycans by MALDI-TOF MS. Each of the transformants analyzed yielded a majority of N-glycans with a GlcNAciManaGlcNAca structure and a minority (-5%) consistent with a single hexose addition (YSH-71). However, although this peak always correlated with the introduction of hGalTI, it was completely recalcitrant to (i-1,4-galactosidase. Subsequently, several of these strains were transformed with plasmids pRCD395 and pRCD396 (PpHIS3/G4lSK plasmids containing POcm-ScGAL10 and PsEC4-ScGAL10, respectively) after linearization with Bglll, selected on G418, and the resulting strains were named YSH-83 and YSH-84, respectively. N-glycans released from secreted K3 were analyzed by MALDI-TOF MS.. The resulting transformants were selected on YPD medium containing G418 and N-glycans released from purified, secreted K3 from these strains were analyzed by MALDI-TOF MS. A majority of N-glycans from these transformants were of three structures, Gal2GlcNAc2Man3GlcNAc2 (-0-25%) or GalGlcNAc2Man3GlcNAc2 (-40-50%), with the rest of the N-glycans retaining the GlcNAc2Man3GlcNAc2 structure displayed by the parental YSH-44 strain. The relative amount of N-glycans remained unchanged irrespective of whether the ScGALlO epimerase gene was driven by the PpOCHl promoter (YSH-83) or the PpSEC4 promoter (YSH-84). Figure 9B shows a MALDI-TOF MS of the N-glycans released from YSH-84. EXAMPLE 8<br>
onstruction of a Epimerase/Transferase Fusion Construct<br>
The SpGALE gene was amplified using primers RCD326 (SEQ ID NO:45) (5'-CTTGGCGCGCCATGACTGGTGTTCATGAAGGGACT-3') and RCD329 (SEQ ID N0:46) 5'-CCTGGATCCCTTATATGTCTTGGTATGGGTCAG-3'). cloned into the pCR2.1 vector (Invitrogen) and sequenced. A truncated portion of the /jGa/77 gene eliminating the first 43 amino acid (hGalTIA43) was amplified using primers RCD328 (SEQ IDNO:47) (5'-CTTGGATCCGGTGGTGGCCGCGACCTGAGCCGCCTGCCC-3') and RCD199 (SEQ ID NO:48) (5'-<br>
CTTCTTAATTAACTAGCTCGGTGTCCCGATGTCCAC-3') cloned into the pCR2.1 vector (Invitrogen) and sequenced. The SpGALE clone was then digested with Ascl/BamHl and the hGalTI clone digested with BamHl/Pacl and both were inserted into pRCD452 digested with AscI/PacI. The plasmid pRCD452 contains the G418 resistance marker and GAPDH/CYCl cassette with the ScMNN2(s)/hGalTI fusion. The AscllBamUl SpGALE and BamHl/Pacl hGalTIA43 fragments were ligated in place of the Ascl/Pacl released hGalTI io create pRCD461. This new plasmid, pRCD461 contains a ScMNN2(s)/SpGALE/hGalTI fusion where the S/?GalE and hGalTI proteins are encoded in a single polypeptide separated by a four amino acid (GSGG) linker containing the BamHl site, and driven by the PpGAPDH promoter. EXAMPLE 9<br>
Expression of a Galactosyl transferase, epimerase and transporter in a strain producing complex N-glycans<br>
Plasmids pXB53, containing the active hGalTI-53 gene fusion, and pRCD378, containing an hGalTII-53 fusion, were linearized with Xhol adjacent to the HYGR marker<br>
and blunted with T4 DNA polymerase (New England Biolabs, Beverly, MA). Plasmid pRCD381, containing a hGalTIII-53 gene fusion, was linearized with HindHI adjacent to the URA3 gene and blunted with T4 polymerase. The three epimerase genes ScGALlO, SpGALE and hGALE were then digested from plasmids pRCD404, pRCD406, and pRCD427, respectively, with XhollSphl, blunted with T4 DNA polymerase, and inserted into the three linearized transferase plasmids. This generated nine new double transferase/epimerase HYGR plasmids: pRCD424 with hGalTI-53 and ScGALJO, pRCD425 with hGalTI-53 and SpGALE, pRCD438 with hGalTI-53 and hGALE, pRCD439 with hGalTII-53 and ScGALlO, pRCD440 with hGalTII-53 and SpGALE, pRCD441 with hGalTII-53 and hGALE, pRCD442 with hGalTIII-53 and ScGALJO, pRCD443 with hGalTIII-53 and SpGALE, and pRCD447 with hGalTIII-53 and hGALE. Subsequently, the strain YSH44 was transformed sequentially with these double HYGR plasmids (linearized with Xbal) and the G418R plasmids pRCD393, pSH262, pSH263 and pSH264 containing the SpUGT, hUGT2, DmUGT, and HUGTJ UDP-Gal transporter encoding genes, respectively (linearized with Agel). Thus, a series of strains was created that each contained a different combination of transferase, epimerase and transporter. First, the different UDP-Gal transporters were compared in strains that contained hGalTI-53 and SpGALE. The introduction of the DmUGT gene resulted in virtually all of the complex glycans having two terminal galactose residues (Gal2GlcNAc2Man3GlcNAc2), whereas the other three transporter genes resulted in a profile of complex glycans virtually identical to that obtained with only the transferase and epimerase (Figure 11A-11E). Second, the epimerase genes were compared in strains with the hGalTI-53 fusion and active DmUGT gene by introducing pSH263 into strains with pRCD424, pRCD425<br>
or pRCD438. The combinations of Gal genes with each of the three epimerase genes were equivalent in generating Gal2GlcNAc2Man3GlcNAc2 complex N-glycans on secreted K3. Finally, the three human transferase fusion constructs HGalTI-53, hGalTII-53, and hGalTIII-53 were compared in strains with DmUGTand SpGALE by introducing pRCD425, pRCD440 and pRCD443 into strains transformed with pSH263. Here, hGalTII-53 was slightly less efficient in transferring Gal as approximately 10% of the complex N-glycans in the strain with HGalTI-53 had only a single galactose (GalGlcNAc2Man3GlcNAc2) where as all the observable complex N-glycans in the strain with hGalTI-53 were bi-galactosylated Gal2GlcNAc2Man3GlcNAc2 (Figure 12A - 12B). Moreover, hGalTIII-53, was significantly less efficient than either hGalTI-53 or hGalTII-53 as 60-70% of the complex N-glycans contained 0-1 galactose residues (GlcNAc2Man3GlcNAc2 or GalGlcNAc2Man3GlcNAc2) whereas only 30-40% were Gal2GlcNAc2Man3GlcNAc2 (Figure 12A - 12C). EXAMPLE 10<br>
Expression of a Galactosyl transferase, epimerase and transporter using a single plasmid construct<br>
The G418R plasmid containing Pocm-DmUGT, pSH263, was linearized by digesting with Sad, then blunted with T4 DNA polymerase (New England Biolabs). The PSEC4-SpGALE gene was digested from plasmid pRCD405 with Xhol/Sphl and blunted with T4 DNA polymerase. The blunt SpGALE was then inserted into the blunt Sad site of pSH263 to create plasmid pRCD446, which is a double transporter/epimerase G418R plasmid. pRCD446 was then linearized with £coRI and blunted with T4 DNA polymerase. The PcAPDHScMNN2(syhGalTI fusion construct was<br><br>
released from pXB53 with BgllllBamHI and blunted with T4 DNA polymerase. The blunt ScMNN2(s)/hGalTl was then inserted into the blunt EcoRI site of pRCD446 to create plasmid pRCD465, which is a triple G418R plasmid containing ScMNN2(s)/hGalTI, SpGALE, and DmUGT. P. pastoris YSH-44, transformed with pRCD465 was designated RDP80. The N-glycan profile showed a single peak at 1663 m/z corresponding to the mass of Gal2GlcNAc2Man3GlcNAc2 [C] (Figure 14A).<br>
The HYGR plasmid containing hGalTI-53 and SpGALE, pRCD425, was linearized with Aflil and blunted with T4 DNA polymerase. The DmUGT gene was released from pSH263 with Notl/PacI and inserted into plasmid pRCD405 digested with NotllPacl to create plasmid pRCD468, which contains a PSEC4-DmUGT-CYCl-TT fusion that can be released as a single cassette. pRCD468 was digested with Xhol/Sall to release the DmUGT cassette and blunted with T4 DNA polymerase. The blunted DmUGT w-as inserted into the blunt Aflil site of pRCD425 to create plasmid pRCD466, which is a HYGR triple plasmid with hGalTI-53, SpGALE, and DmUGT.<br>
The HYGR plasmid containing hGalTI-53 and hGALE pRCD438 was linearized with Aflll and blunted with T4 DNA polymerase. pRCD468 was digested with XhollSaH to release the DmUGT cassette and blunted with T4 DNA polymerase. The blunted DmUGT was inserted into the blunt Aflll site of pRCD438 to create plasmid pRCD467, which is a HYGR triple plasmid with hGalTI-53, hGALE, and DmUGT. In vitro ft-galactosidase Digest<br>
N-glycans (2^g) from P. pastoris strain RDP80 were incubated with 3mU (31,4 galactosidase (QA bio, San Mateo, CA) in 50 mM NH4HCO3, pH6.0 at 37°C for 16-20 hours. N-glycan analysis in Figure 14B shows a predominant peak at 1430 m/z [A],<br>
which corresponds to the mass of the N-glycan GlcNAcaMansGlcNAci, confirming galactose transfer in Figure 14A. In vitro Sialvltransferase Reaction<br>
K3 purified from strain RDP80 (200 ng) was incubated with 50 ug CMP-sialic acid and 15 mil rat recombinant a(2,6)-(N)-sialyltransferase (Calbiochem) in 50 mM NH4HCO3, pH6.0 at 37°C for 16-20 hours. N-glycan were then released by PNGaseF digest and detected on MALDI-TOF MS. The spectrum of the glycans showed an increase in mass following sialyltransferase treatment (Figure 14C) when compared with those from RDP80 (Figure 14A). The spectrum as shown in Figure 14C depicts a predominant peak at 2227 m/z [X], which corresponds to the mass of the N-glycan NANA2Gal2GlcNAc2Man3GlcNAc2 further confirming that the N-glycans produced by strain RDP80 is human-type Gal2GlcNAc2Man3GlcNAc2. Example 11 Epimerase sequence alignment<br>
Sequence alignment of epimerases was performed using CLUSTAL. The nucleotide sequences and/or amino acid sequences of the Sequence Listing were used to query sequences in the GenBank, SwissProt, BLOCKS, and Pima II databases. These databases, which contain previously identified and annotated sequences, were searched for regions of homology using BLAST (Basic Local Alignment Search Tool). (See, e.g., Altschul, S. F. (1993) J. Mol. Evol 36:290-300; and Altschul et al. (1990) J. Mol. Biol. 215:403-410.)<br>
BLAST produced alignments of both nucleotide and amino acid sequences to determine sequence similarity.  Other algorithms could have been used when dealing with primary<br>
sequence patterns and secondary structure gap penalties. (See, e.g., Smith, T. et al. (1992) Protein Engineering 5:35-51.) EXAMPLE 12 Materials<br>
MOPS, sodium cacodylate, manganese chloride, UDP-galactose and CMP-N-acetylneuraminic acid were from Sigma. TFA was from Aldrich. (31,4-galactosyltransferase from bovine milk were from Calbiochem. Protein N-glycosidase F, mannosidases, and oligosaccharides were from Glyko (San Rafael, CA). DEAE ToyoPearl resin was from TosoHaas. Metal chelating "HisBind" resin was from Novagen (Madison, WI). 96-well lysate-clearing plates were from Promega (Madison, WI). Protein-binding 96-well plates were from Millipore (Bedford, MA). Salts and buffering agents were from Sigma (St. Louis, MO). MALDI matrices were from Aldrich (Milwaukee, WI). Shake-flask cultivations<br>
A single colony was picked from an YPD plate (
ml baffled volumetric flask. After 24 hours, the expression medium flask is harvested by centrifugation (3000g for 10 minutes) and the supernatant analyzed for K3 production. Bioreactor Cultivations<br>
A 500ml baffled volumetric flask with 150ml of BMGY media was inoculated with 1 ml of seed culture (see flask cultivations). The inoculum was grown to an ODeoo of 4-6 at 24° C (approx 18 hours). The cells from the inoculum culture was then centrifuged and resuspended into 50ml of fermentation media (per litre of media: CaSO4.2H2O O.SOg, K2SO4 6.00g, MgSO4.7H2O 5.00g, Glycerol 40.0g, PTMi salts 2.0ml, Biotin 4xlO"3g, H3PO4 (85%) 30ml, PTMI salts per litre: CuSO4.H2O 6.00, Nal 0.08g, MnSO4.7H2O 3.00g, NaMoO4.2H2O 0.20g, H3B03 0.02g, CoCl2.6H2O O.SOg, ZnCl2 20.0g, FeSO4.7H2O 65.Og, Biotin 0.20g, H2S04 (98%) 5.00ml).<br>
Fermentations were conducted in 3 litre dished bottom (1.5 litre initial charge volume) Applikon bioreactors. The fermentors were run in a fed-batch mode at a temperature of 24° C, and the pH was controlled at 4.5 ±0.1 using 30% ammonium hydroxide. The dissolved oxygen was maintained above 40% relative to saturation with air at 1 atm by adjusting agitation rate (450-900 rpm) and pure oxygen supply. The air flow rate was maintained at 1 wm. When the initial glycerol (40g/l) in the batch phase is depleted, which is indicated by an increase of DO, a 50% glycerol solution containing 12 ml/1 of PTMi salts was fed at a feed rate of 12 ml/l/h until the desired biomass concentration was reached. After a half an hour starvation phase, the methanol feed (100% Methanol with 12 ml/1 PTMi) is initiated. The methanol feed rate is used to control the methanol concentration in the fermentor between 0.2 and 0.5%. The methanol concentration is measured online using a TGS gas sensor (TGS822 from Figaro<br>
Engineering Inc.) located in the offgass from the fermentor. The fermentors were sampled every eight hours and analyzed for biomass (ODeoo, wet cell weight and cell counts), residual carbon source level (glycerol and methanol by HPLC using Aminex 87H) and extracellular protein content (by SDS page, and Bio-Rad protein assay). Reporter protein expression, purification and release of N-linked glycans<br>
The K3 domain, under the control of the alcohol oxidase 1 (AOX1) promoter, was used as a model protein and was purified using the 6xHistidine tag as reported previously (Choi et al., Proc Natl Acad Sci USA. 2003 Apr 29;100(9):5022-7). The glycans were released and separated from the glycoproteins by a modification of a previously reported method (Papac and Briggs 1998). After the proteins were reduced and carboxymethylated, and the membranes blocked, the wells were washed three time with water. The protein was deglycosylated by the addition of 30 ul of 10 mM NFLjHCOa pH 8.3 containing one milliunit of N-glycanase (Glyko). After 16 hr at 37°C, the solution containing the glycans was removed by centrifugation and evaporated to dryness. Protein Purification<br>
Kringle 3 was purified using a 96-well format on a Beckman BioMek 2000 sample-handling robot (Beckman/Coulter Ranch Cucamonga, CA). Kringle 3 was purified from expression media using a C-terminal hexa-histidine tag. The robotic purification is an adaptation of the protocol provided by Novagen for their HisBind resin. Briefly, a 150uL (uL) settled volume of resin is poured into the wells of a 96-well lysate-binding plate, washed with 3 volumes of water and charged with 5 volumes of 50mM NiSO4 and washed with 3 volumes of binding buffer (5mM imidazole, 0.5M NaCl, 20mM Tris-HCL pH7.9). The protein expression media is diluted 3:2, media/PBS<br>
(60mM PO4, 16mM KC1, 822mM NaCl pH7.4) and loaded onto the columns. After draining, the columns are washed with 10 volumes of binding buffer and 6 volumes of wash buffer (30mM imidazole, 0.5M NaCl, 20mM Tris-HCl pH7.9) and the protein is eluted with 6 volumes of elution buffer (1M imidazole, 0.5M NaCl, 20mM Tris-HCl pH7.9). The eluted glycoproteins are evaporated to dryness by lyophilyzation. Release of N-linked Glycans<br>
The glycans are released and separated from the glycoproteins by a modification of a previously reported method (Papac, et al. A. J. S. (1998) Glycobiology 8, 445-454). The wells of a 96-well MultiScreen IP (Immobilon-P membrane) plate (Millipore) are wetted with lOOuL of methanol, washed with 3X150uL of water and 50uL of RCM buffer (8M urea, 360mM Tris, 3.2mM EDTA pH8.6), draining with gentle vacuum after each addition. The dried protein samples are dissolved in 30uL of RCM buffer and transferred to the wells containing lOuL of RCM buffer. The wells are drained and washed twice with RCM buffer. The proteins are reduced by addition of 60uL of 0.1 M DTT in RCM buffer for Ihr at 37oC. The wells are washed three times with 300uL of water and carboxymethylated by addition of 60uL of 0.1 M iodoacetic acid for 30min in the dark at room temperature. The wells are again washed three times with water and the membranes blocked by the addition of lOOuL of 1% PVP 360 in water for Ihr at room temperature. The wells are drained and washed three times with 300uL of water and deglycosylated by the addition of 30uL of lOmM NfyHCOs pH 8.3 containing one milliunit of N-glycanase (Glyko). After 16 hours at 37°C, the solution containing the glycans was removed by centrifugation and evaporated to dryness.<br>
Miscellaneous: Proteins were separated by SDS/PAGE according to Laemmli (Laemmli<br>
1970).<br>
Matrix Assisted Laser Desorption lonization Time of Flight Mass Spectrometry<br>
Molecular weights of the glycans were determined using a Voyager DE PRO linear MALDI-TOF (Applied Biosciences) mass spectrometer using delayed extraction. The dried glycans from each well were dissolved in 15uL of water and O.SuL spotted on stainless steel sample plates and mixed with O.SuL of S-DHB matrix (9mg/mL of dihydroxybenzoic acid, Img/mL of 5-methoxysalicilic acid in 1:1 water/acetonitrile 0.1% TFA) and allowed to dry.<br>
Ions were generated by irradiation with a pulsed nitrogen laser (337nm) with a 4ns pulse time. The instrument was operated in the delayed extraction mode with a 125ns delay and an accelerating voltage of 20kV. The grid voltage was 93.00%, guide wire voltage was 0.10%, the internal pressure was less than 5 X 10-7 torr, and the low mass gate was 875Da. Spectra were generated from the sum of 100-200 laser pulses and acquired with a 2 GHz digitizer. MansGlcNAc2 oligosaccharide was used as an external molecular weight standard. All spectra were generated with the instrument in the positive ion mode. The estimated mass accuracy of the spectra was 0.5%.<br><br>
WE CLAIM<br>
1.	A recombinant lower eukaryotic host cell producing human-like glycoproteins<br>
characterized as having a terminal P-galactose residue and essentially lacking fucose and<br>
sialic acid residues on the glycoprotein.<br>
2.	The   host   cell   of claim   1   wherein  the   host   cell   expresses   pi,4-<br>
galactosyltransferase activity.<br>
3.	The host cell of claim 1 wherein the host cell expresses a UDP-galactose<br>
transport activity.<br>
4.	The host cell of claim 1 wherein the host cell exhibits an elevated level of<br>
UDP-galactose.<br>
5.	A recombinant lower eukaryotic host cell producing human-like glycoproteins,<br>
the host comprising an isolated nucleic acid molecule encoding p-galactosyltransferase<br>
activity and at least an isolated nucleic acid molecule encoding UDP-galactose transport<br>
activity, UDP-galactose C4 epimerase activity, galactokinase activity or a galactose-1-<br>
phosphate uridyl transferase activity.<br>
6.	The host of claim 3 or 5 wherein the UDP-galactose transport activity is<br>
encoded by a gene selected from the group consisting of: SpUGT, hUGTl, HUGT2, and<br>
DmUGT.<br>
7.	The host of claim 4 or 5 wherein the UDP-galactose C4 epimerase activity is<br>
encoded by a gene selected from the group consisting of: SpGALE, ScGALlO and<br>
hGALE.<br>
8.	A recombinant lower eukaryotic host cell producing human-like glycoproteins,<br>
the host cell capable of transferring p-galactose residue onto an N-linked oligosaccharide<br>
branch of a  glycoprotein  comprising  a terminal  GlcNAc  residue,  the N-linked<br>
oligosaccharide branch selected from the group consisting of GlcNAcpl,2-Manal,3;<br>
GlcNAcpl,4-Manal,3; GlcNAcpl,2-Manal,6; GlcNAcpl,4-Manal,6; and GlcNAcpl,6-<br>
Manal,6 on a trimannose core.<br>
9.	A recombinant lower eukaryotic host cell produced in claim 1 wherein the host<br>
cell produces glycoproteins that are acceptor substrates for sialic acid.<br>
10.	The host of any one of claims 1, 5, or 8 wherein said host cell is impaired in<br>
initiating 1,6 mannosyltransferase activity with respect to the glycan on the glycoprotein.<br>
11.	The host of any one of claims 1, 5, or 8 wherein the host cell is diminished or<br>
depleted in dolichyl-P-Man: Man5GlcNAc2-PP-dolichyl alpha-1,3 mannosyltransferase<br>
activity.<br>
12.	The host of any one of claims 1, 5, or 8 wherein said host cell expresses a<br>
mannosidase activity selected from the group consisting of an a-l,2-mannosidase I<br>
activity, mannosidase II activity, mannosidase IIx activity and class III mannosidase<br>
activity.<br>
13.	The host of any one of claims 1, 5, or 8 wherein said host cell expresses a<br>
GnT activity selected from the group consisting of GnTI, GnTII, GnTIII, GnTIV, GnTV,<br>
GnTVI and GnTIX.<br>
14.	The host of any one of claims 1, 5, or 8 wherein the host cell is selected from<br>
the group consisting of Pichia pastoris, Pichia finlandica, Pichia trehalophila, Pichia<br>
koclamae, Pichia membranaefaciens, Pichia minuta (Ogataea minuta, Pichia lindneri),<br>
Pichia opuntiae, Pichia thermotolerans, Pichia salictaria, Pichia guercuum, Pichia<br>
pijperi, Pichia stiptis, Pichia methanolica, Pichia sp.,  Saccharomyces cerevisiae,<br>
Saccharomyces sp., Hansenula polymorpha, Kluyveromyces sp., Kluyveromyces lactis,<br>
Candida   albicans,   Aspergillus   nidulans,   Aspergillus   niger,   Aspergillus   oryzae,<br>
Trichoderma reesei, Chrysosporium lucknowense, Fusarium sp., Fusarium gramineum,<br>
Fusarium venenatum, Physcomitrella patens and Neurospora crassa.<br>
15.	A composition comprising a human-like glycoprotein characterized as having<br>
a terminal p-galactose residue and essentially lacking fucose and sialic acid residues on<br>
the glycoprotein.<br>
16.	The composition of claim 15 wherein the glycoprotein comprises N-linked<br>
oligosaccharides selected from the group consisting of: GalGlcNAcMan3GlcNAc2,<br>
GalGlcNAc2Man3GlcNAc2, Gal2GlcNAc2Man3GlcNAc2, GalGlcNAc3Man3GlcNAc2,<br>
Gal2GlcNAc3Man3GlcNAc2, Gal3GlcNAc3Man3GlcNAc2, GalGlcNAc4Man3GlcNAc2,<br>
Gal2GlcNAc4Man3GlcNAc2, Gal3GlcNAc4Man3GlcNAc2, Gal4GlcNAc4Man3GlcNAc2<br>
GalGlcNAcMan5GlcNAc2, GalGlcNAc2Man5GlcNAc2, Gal2GlcNAc2Man5GlcNAc2,<br>
GalGlcNAc3Man5GlcNAc2, Gal2GlcNAc3Man5GlcNAc2 and Gal3GlcNAc3Man5GlcNAc2.<br>
17.	The composition of claim 15 wherein the glycoprotein is selected from the<br>
group  consisting  of:   erythropoietin,  cytokines  such  as  interferon-a,  interferon-p,<br>
interferon-y, interferon-co, and granulocyte-CSF, GM-CSF, coagulation factors such as<br>
factor VIII, factor IX, and human protein C, antithrombin III, thrombin, soluble IgE<br>
receptor a-chain, IgG,  IgG fragments,  IgG fusions, IgM,  interleukins, urokinase,<br>
chymase, and urea trypsin inhibitor, IGF-binding protein, epidermal growth factor,<br>
growth hormone-releasing  factor,  annexin V  fusion protein,  angiostatin,  vascular<br>
endothelial growth factor-2, myeloid progenitor inhibitory factor-1, osteoprotegerin, cc-1-<br>
antitrypsin,   a-   feto   proteins,   DNase   II,   kringle   3   of   human   plasminogen,<br>
glucocerebrosidase, TNF binding protein 1, follicle stimulating hormone, cytotoxic T<br>
lymphocyte associated antigen 4 - Ig, transmembrane activator and calcium modulator<br>
and cyclophilin ligand, soluble TNF receptor Fc fusion, glucagon like protein 1, IL-2<br>
receptor agonist.<br>
18.	A method for producing human-like glycoproteins in a lower eukaryotic host<br>
cell the method comprising the step of producing UDP-galactose above endogenous<br>
levels.<br>
19.	The host cell produced by the method of claim 18.<br>
20.	A method for producing human-like glycoprotein composition in lower<br>
eukaryotic host cell comprising the step of transferring a galactose residue on a hybrid or<br>
complex glycoprotein in the absence of fucose and sialic acid residues.<br>
21.	The method of claim 20, wherein the galactose residue is transferred onto a<br>
glycoprotein selected from the group consisting of: GlcNAcMan3GlcNAc2,<br>
GlcNAc2Man3GlcNAc2, GlcNAc3Man3GlcNAc2, GlcNAc4Man3GlcNAc2,<br>
GlcNAc5Man3GlcNAc2 GlcNAc6Man3GlcNAc2, GlcNAcMan4GlcNAc2,<br>
GlcNAcMansGlcNAc2, GlcNAc2Man5GlcNAc2 and GlcNAc3Man5GlcNAc2.<br>
22.	The method of claim 20 wherein the transferring step further comprises<br>
expressing a gene encoding a p-galactosyltransferase activity or a catalytically active<br>
fragment thereof.<br>
23.	The method of claim 22 wherein the galactosyltransferase activity is selected<br>
from the group consisting of: human GalT I, GalT II, GalT III, GalT IV, GalT V, GalT<br>
VI, GalT VII, bovine GalTI, X. leavis GalT and C. elegans GalTII.<br>
24.	The method of claim 20 wherein the transferring step further comprises<br>
expressing a UDP-Galactose Transport activity.<br>
25.	The method of claim 24 wherein the UGT is selected from the group<br>
consisting of S. pombe UGT, human UGT1, human UGT2, and D. melanogaster UGT.<br>
26.	The method of claim 20 wherein the transferring step further comprises<br>
expressing a gene encoding a UDP-galactose C4 epimerase activity.<br><br>
27.	The method of claim 26 wherein the epimerase activity is selected from the<br>
group consisting of: S.pombe GalE, S. cerevisiae Gal 10 and human GalE.<br>
28.	The method of any one of claims 20 -27 wherein at least 33% galactosylated<br>
glycoprotein composition is produced.<br>
29.	The method of any one of claims 20 - 27 wherein at least 60% galactosylated<br>
glycoprotein composition is produced.<br>
30.	The method of any one of claims 20 - 27 wherein at least 90% galactosylated<br>
glycoprotein composition is produced.<br>
31.	The glycoprotein composition produced by any one of the claims 20 - 27.<br>
32.	The glycoprotein composition produced by any one of claims 20 - 27 wherein<br>
the glycoprotein composition is an acceptor substrate for sialic acid.<br>
33.	The  glycoprotein  composition  produced  by  claim   31,  wherein  the<br>
glycoprotein is selected from the group consisting of: erythropoietin, cytokines such as<br>
interferon-a, interferon-p,   interferon-y, interferon-co, and granulocyte-CSF, GM-CSF,<br>
coagulation factors such as factor VIII, factor IX, and human protein C, antithrombin III,<br>
thrombin,  soluble IgE  receptor cc-chain, IgG,  IgG fragments,  IgG  fusions, IgM,<br>
interleukins,  urokinase,  chymase,  and urea trypsin inhibitor,  IGF-binding protein,<br>
epidermal growth factor, growth hormone-releasing factor, annexin V fusion protein,<br>
angiostatin, vascular endothelial growth factor-2, myeloid progenitor inhibitory factor-1,<br>
osteoprotegerin,  oc-1-antitrypsin, a- feto proteins, DNase II, kringle 3  of human<br>
plasminogen, glucocerebrosidase, TNF binding protein 1, follicle stimulating hormone,<br>
cytotoxic T lymphocyte associated antigen 4 - Ig, transmembrane activator and calcium modulator and cyclophilin ligand, soluble TNF receptor Fc fusion, glucagon like protein 1, IL-2 receptor agonist.<br>
34.	The   glycoprotein  composition  produced  in   claim   31,   wherein  the<br>
glycoprotein is produced from a host cell selected from the group consisting of Pichia<br>
pastoris,    Pichia   fmlandica,    Pichia    trehalophila,    Pichia    koclamae,    Pichia<br>
membranaefaciens, Pichia minuta (Ogataea minuta, Pichia lindneri), Pichia opuntiae,<br>
Pichia thermotoleram, Pichia salictaria, Pichia guercuum, Pichia pijperi, Pichia stiptis,<br>
Pichia   methanolica,   Pichia   sp.,   Saccharomyces   cerevisiae,   Saccharomyces   sp.,<br>
Hansenula polymorpha, Kluyveromyces sp., Kluyveromyces lactis, Candida albicans,<br>
Aspergillus  nidulans,  Aspergillus  niger,   Aspergillus  oryzae,   Trichoderma  reesei,<br>
Chrysosporium lucknowense, Fusarium sp., Fusarium graminewn, Fusarium venenatum,<br>
Physcomitrella patens and Neurospora crassa.<br>
35.	A recombinant lower eukaryotic host cell expressing GalNAc Transferase<br>
activity.<br>
36.	A recombinant lower eukaryotic host cell expressing a gene encoding<br>
heterologous UDPase activity.<br>
37.	An isolated polynucleotide comprising or consisting of a nucleic acid<br>
sequence selected from the group consisting of:<br>
(a)	SEQ ID NO: 14;<br>
(b)	at least about 90% similar to the amino acid residues of the donor nucleotide<br>
binding site of SEQ ID NO: 13;<br>
(c)	a nucleic acid sequence at least 92%, at least 95%, at least 98%, at least 99%<br>
or at least 99.9% identical to SEQ ID NO: 14;<br>
(d)	a nucleic acid sequence that encodes a conserved polypeptide having the<br>
amino acid sequence of SEQ ID NO: 13;<br>
(e)	a nucleic acid sequence that encodes a polypeptide at least 78%, at least 80%,<br>
at least 85%, at least 90%, at least 95%, at least 98%, at least 99% or at least 99.9%<br>
identical to SEQ ID NO: 13;<br>
(f)	a nucleic acid sequence that hybridizes under stringent conditions to SEQ ID<br>
NO:13;and<br>
(g) a nucleic acid sequence comprising a fragment of any one of (a) - (f) that is at least 60 contiguous nucleotides in length.<br>
38.	A modified polynucleotide comprising or consisting of a nucleic acid<br>
sequence selected from the group consisting of the conserved regions of SEQ ID NO: 48<br>
- SEQ ID NO: 52 wherein the encoded polypeptide is involved in catalyzing the<br>
interconversion of UDP-glucose and UDP-galactose for production of galactosylated<br>
glycoproteins.<br>
39.	An invention substantially such as herein before described.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUFic3RyYWN0LSgxNC0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Abstract-(14-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWFzc2lnbm1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-assignments.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNsYWltcy0oMTQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Claims-(14-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNsYWltcy0oMjgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Claims-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNvcnJlcG9uZGVuY2UgT3RoZXJzLSgyOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Correpondence Others-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Correspondence Others-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Correspondence Others-(14-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Correspondence Others-(22-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Correspondence Others-(25-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjctMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Correspondence Others-(27-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUZvcm0tMy0oMTEtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Form-3-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUZvcm0tMy0oMjctMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Form-3-(27-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUZvcm0tMy0oMjgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-Form-3-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LUdQQS0oMjgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-GPA-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU5NS1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">6595-delnp-2006-pct-search report.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="259914-method-for-treating-a-prebleached-chemical-pulp.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="259916-multiple-packaging-of-wrappers-containing-products-for-medical-use.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>259915</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6595/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Nov-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GLYCOFI, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SUITE 200,21 LAFAYETTE ST., LEBANON NH 03766. US US NATIONAL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DAVIDSON, ROBERT</td>
											<td>37 LANDING ROAD # 2, ENFIELD, NH 03748</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GERNGROSS, TILLMAN</td>
											<td>2 PARKWAY, HANOVER, NH 03755</td>
										</tr>
										<tr>
											<td>3</td>
											<td>WILDT, STEFAN</td>
											<td>32 PARKHURST ST., LEBANON, NH 03766</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CHOI, BYUNG-KWON</td>
											<td>24 HAZEN ST., NORWICH, VT 05055</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NETT, JUERGEN</td>
											<td>PO BOX 808, GRANTHAM, NH 03753</td>
										</tr>
										<tr>
											<td>6</td>
											<td>BOBROWICZ, PIOTR</td>
											<td>292 WOODHAVEN DR. 9B, WHITE RIVER JUNCTION, VT 05001</td>
										</tr>
										<tr>
											<td>7</td>
											<td>HAMILTON, STEPHEN</td>
											<td>98 ALGONQUIN ROAD, ENFIELD, NH 03748</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 1/19</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2005/051249</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-04-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/562,424</td>
									<td>2004-04-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/259915-a-recombinant-eukaryotic-host-cell-producing-human-like-glycoproteins by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:41:23 GMT -->
</html>
