<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260090-high-throughput-testing-for-presence-of-microorganisms-in-a-biological-sample by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:45:58 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260090:HIGH THROUGHPUT TESTING FOR PRESENCE OF MICROORGANISMS IN A BIOLOGICAL SAMPLE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">HIGH THROUGHPUT TESTING FOR PRESENCE OF MICROORGANISMS IN A BIOLOGICAL SAMPLE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Provided are methods and apparatus for high throughput testing of biological samples that may or may not comprise microorganisms. The methods include the use of a diagnostic multiplex¬ing panel (DMP) specifically designed for the simultaneous identification of a plurality of po¬tential microorganisms that may be present in the biological sample via a primer extension reaction directed a highly conserved nucleic acid sequences in the microorganisms under test. The biological sample is typically immobilised on a solid substrate at a first location before being transferred to a second location for analysis using the DMP. The methods and apparatus of the invention are particularly suited to diagnosis of the presence of infectious pathogens in the biological sample, for example for diagnosis of sexually transmitted infection. Figure 1 is the representative Figure.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD<br>
The invention relates to high throughput multiplex testing for microorganisms that may be present in a biological sample using nucleic acid based enzymatic techniques. More specifically, the invention is directed towards identification of pathogenic microorganisms.<br>
BACKGROUND<br>
Since the identification in the nineteenth century of microorganisms as one of the major sources of morbidity and mortality, efforts have continued to monitor and control the spread of infectious disease. The earliest efforts were made by pioneers in the young science of epidemiology such as Dr John Snow, who famously removed the handle of London's Broad Street Pump after he identified it as the source of the City-wide cholera outbreak. In the twentieth century, the advent of antibiotics, mass vaccination and antiviral treatments has offered an unprecedented level of control over the spread of such diseases, in the developed world at least. Nevertheless, infectious disease still remains one of the main causes of death worldwide, with an unrelenting succession of 'new' microbial killers seemingly emerging every year. MRSA, SARS, avian flu, HIV and malaria represent but a few of the many infectious pathogens causing alarm and concern around the world.<br>
International health organisations such as the UN and the WHO consistently express concern over the unrestrained use of antibiotic compounds, leading to increasing levels of antibiotic resistance amongst many microbial species. In addition sexually transmitted infections (STIs) represent one of the greatest infectious disease problems in the world today and in some regions, particularly Africa and the former Soviet Union, are at epidemic level. According to one study (Adler, M., 2005, Why sexually transmitted infections are important. In:<br>
ABC of Sexually Transmitted Infections. 5th ed. BMJ Books) the number of reported infected people in Western Europe was 17 million, in the USA it was 15 million, in Africa 70 million, and globally around 400 million.<br>
Combating sexually transmitted infections and HIV/AIDS remains uppermost on the world's governmental health agendas and the need for improvement in access to health services plus the creation and provision of diagnostic services available for all who need them are clear. Many STIs are asymptomatic and can only be diagnosed through testing, however routine screening programmes are extremely rare, social stigma high, funding inadequate and public awareness limited.<br>
The effects of infectious diseases are not limited to the human population, severe economic damage is caused by outbreaks of diseases in livestock and plant crops. Outbreaks of swine fever in pigs, foot and mouth disease in cattle and avian flu can rapidly spread and decimate the agricultural output and the economy of a country. In the UK in 2001, an outbreak of foot and mouth disease lead to the culling of over seven million cattle and sheep as well as the effective 'closure' of rural areas.<br>
Currently available systems for screening for the presence of microorganisms in a sample taken from a host (such as a patient, animal or plant source) are low throughput. In the clinical environment, usually only one microorganism is tested for per sample unless there are medical indications that the host may suffer from several microbial diseases. This means that it is usually only possible to detect a single infection per test, which has a significant impact on the price and the testing time. In addition, in an asymptomatic host it is often difficult to decide which microorganism the patient should be tested for. Negative results for two or three infectious organisms can provide a false sense of security.<br>
Modern detection systems used in clinical practice employ DNA based assays for detection of microorganism infection. The most common methodologies are based on Polymerase Chain Reaction (PCR), Ligase Chain Reaction, Strand<br>
Displacement Amplification, Transcription Mediated Amplification, Sequence Based Amplification Assay and several others. These approaches are also low throughput, time consuming, require a large amount of hands-on time and are difficult to automate. In addition to this, there remains no reliable approach which could be used to detect multiple pathogens in a single test, using a single sample taken from the host organism.<br>
Hence, it would be desirable to provide a means for reliably and cheaply testing for a plurality of microorganisms that may be present in a biological sample. In particular it would be desirable to provide means for effecting testing that can allow for reliable biological sample collection in home or otherwise in the field.<br>
SUMMARY<br>
In broad terms, the present invention overcomes the aforementioned problems in the prior art by providing methods and apparatus for high throughput testing of biological samples that may or may not comprise microorganisms. The methods include the use of a diagnostic multiplexing panel (DMP) specifically designed for the simultaneous identification of a plurality of potential microorganism species that may or may not be present in the biological sample.<br>
In a first aspect, the invention provides a method for determining whether one or more specified microorganisms are present within a biological sample that potentially comprises the microorganisms comprising:<br>
(a)	immobilizing the biological sample on and/or within a solid substrate at a first location;<br>
(b)	transferring the immobilized biological sample to at least a second location and performing a extraction step on the solid substrate so as to extract any microorganism DNA immobilized on and/or within the solid substrate;<br>
(c)	performing a nucleic acid amplification step on microorganism DNA extracted in step (b), wherein the amplification step is directed towards amplification of at least one highly conserved sequence, from one or more specific microorganisms, and wherein amplified sequences are designated as target sequences,<br>
(d)	combining the target sequences with a plurality of primer sequences comprised within a diagnostic multiplexing panel (DMP), wherein each primer sequence facilitates genotyping of the target sequence;<br>
(e)	performing a primer extension reaction on the combination of target sequences and the DMP present in (d), thereby producing a DMP reaction product; and<br>
(f)	analysing the reaction product so as to determine genotype of any target sequences that are present and correlating the genotype of the target sequences in the reaction product with the identification of specified microorganisms present in the biological sample.<br>
In a second aspect, the invention provides a DMP, suitable for use in genotyping pathogenic microorganisms known to cause at least one infectious disease that may be present within a biological sample, the DMP comprising a plurality of primer sequences directed at identification of at least two or more SNPs present in a highly conserved allele of at least one microorganism known to cause an infectious disease when used in a primer extension reaction.<br>
In a third aspect, the invention provides a microorganism testing kit suitable for personal use by a user located in a first location, the kit comprising a testing surface located within a sealable chamber, the testing surface further comprising a solid substrate that is capable of immobilizing a biological sample either within it or upon its surface, and wherein once a biological sample is deposited upon the testing surface, the chamber is sealed around the testing surface such that the testing kit can be despatched to a second location for<br>
analysis to determine whether one or more microorganisms are present in the biological sample.<br>
In a fourth aspect, the invention provides a method of treating an animal, including a human, that is suspected of carrying one or more infectious microorganisms, comprising obtaining a biological sample from the animal, testing the biological sample according to the method methods described herein, thereby diagnosing whether the animal is Infected with one or more infectious microorganisms, and administering treatment to the animal, which treatment is configured appropriately in light of the Information regarding the type(s) of infectious microorganisms found to be present in the biological sample. Optionally, the treatment is further configured appropriately according to information regarding the anti-biotlic; resistance status of one or more of, the infectious microorganisms found to be present in the biological sample. These and other uses, features and advantages of the invention should be apparent to those skilled In the art from the teachings provided herein.<br>
DRAWINGS<br>
Figure 1 shows a schematic representation of highly conserved DNA consensus sequences - the consensus sequences were generated by comparison of several strains of the same species * the locations at which primers for the amplification or primer extension reactions are selected for use in a DMP of the invention are shown. For each primer, the first two letters identify the organism, the letter after identifies the target sequence for the species (1-3); IsiPCRP denotes the first PCR primer; 2ndPCRP denotes the second PCR primer; El or £2 denotes an extension primer; for UreapJasma there are two sites for which the primers were designed (UU1 and UU2). The identity of the primera is set out in Table 1. The organisms are (a) Candida albicans; (b) Chlamydia trachomatis; (c) Gardnerella vaginalis; (d) Mycoplasma genitalium; (e) Mycoplasma homlnis; (f) Neisseria gonorrhoea; (g) Treponema pallidum; (h) Trichomonas vaginalis; (i) Ureaplasrna urealyticum.<br>
DETAILED DESCRIPTION<br>
In setting forth the invention! a number of definitions are provided that wilt assist in the understanding of the invention. For the avoidance of doubt, ail references cited herein are incorporated by reference in their entirety. Unless otherwise defined, ail technical and scientVk: terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Where common molecular biology techniques are described<br>
it is expected that a person of skill in the art would have knowledge of such techniques, for example from standard texts such as Sambrook J. et al, (2001) Molecular Cloning; a Laboratory Manual, Cold Spring Harbor Press, Cold Spring Harbor, NY.<br>
The term "specified microorganisms" as used herein is intended to denote one or more specified species of microorganism that may or may not be present in a biological sample. The specified microorganisms are suitably of viral, bacterial, fungal (including unicellular yeast) and/or protozoan (including Plasmodium) origin. Typically, the specified microorganisms will be pathogenic to an animal host at some point in their life cycle. However, the present invention is adequate for testing for species of microorganisms that exhibit dormancy, commensal infection or sub-clinical infection.<br>
The term "biological sample" as used herein is intended to encompass samples that may contain one or more of specified microorganisms that are tested for according to the present invention. Depending upon the intention of the DMP, the biological sample may be obtained from a human patient, a non-human animal, from a plant or from a foodstuff. In the latter case the DMP will be intended for the purpose of determining contamination of the foodstuff by food-borne pathogens. The samples can suitably include, for example, urine; faeces; vaginal, nasal, rectal or oral swabs; blood, saliva; and/or sputum; semen; vaginal or urethral discharges and swabs thereof; tears (i.e. lacrimal secretions); biopsy tissue samples; and swabs of surfaces upon which any of the aforementioned secretions and substances may have been deposited. Whilst the biological sample of the present invention may contain cells, tissue and/or DNA originating from a host organism, e.g. a human patient, the intension of the invention is to test for the presence of microorganisms present within that host - not to test the host's own DNA.<br>
The solid substrate of the invention is suitably selected from an absorbent fibrous material impregnated with one or more reagents that act to immobilize and inactivate any microorganisms present in the biological sample, or even more simply to cause immobilization of nucleic acid contained within the<br>
microorganisms. Such reagents may include detergent compounds: anionic or catjonic detergents, chelating agents (e.g. EDTA), urea and/or uric acid. The solid substrate itself may incorporate an absorbent a material selected from a cellulose-based paper (e.g. blotting paper); a microfibrous membrane; a glass-fibre material; a polymeric fibre material (e.g. a nylon filter membrane); a woven fabric; and a non-woven fabric. Suitable solid substrates are described, for example, US Patent Nos. 5498562, 5756126, 5807527, 5939259, 5972386, 5985327, 6168922, 6746841 and 6750059. In specific embodiments of the present invention solid substrate includes filter paper treated with Whatman FTA® or Whatman FTA Elute® reagent, such as Whatman FTA® paper or Whatman FTA® Etute paper.<br>
A significant advantage of the method of the present invention is that a solid substrate treated with reagent, such as Whatman FTA Elute® reagent, can be used by an individual in a non-clinical setting. For instance, it is envisaged that the initial sample collection phase of the method of the invention could be carried out by an individual purchasing a kit and simply wetting the solid sold substrate with, say, a sample of urine or saliva. The solid substrate will immobilize any microorganisms present in the urine or saliva, such that infectious pathogens are safely rendered non-infectious and the sample can be transmitted to a testing location without the need for expensive handling (i.e. refrigeration, or additional chemical fixing), for instance by regular postal services. The immobilized microorganism DNA can foe easily extracted from the solid substrate in the testing location, typically by using a simple heat and water elution step. The invention, thus, provides a system in which biological sample collection can even be performed at home or in the field, samples can be stored indefinitely and then tested at a later date. The benefits of this arrangement are significant as to-date most diagnostic testing is hampered by the need for sample collection to be performed in a clinical environment, which reduces uptake amongst the population as a whole.<br>
A suitable home testing kit according to an embodiment of the present invention, includes a sealable chamber that encloses a testing surface. The kit<br>
includes instructions for use, which direct the user to place a biological sample -such as a drop of urine - on the testing surface. The testing surface comprises a solid substrate of the type described above. After the sample is deposited on the testing surface the seaiable chamber can be closed such that it encapsulates and protects the testing surface from further interference. In the seated state the kit remains secure from outside interference or contamination and can be despatched to a testing facility located remotely from the user's home. At the testing facility the seaiable chamber can be opened (if necessary by breaking the chamber open) allowing access to the testing surface for analytical purposes according to the method of the invention.<br>
A "nucleic acid sequence* is a single or double stranded covalently-linked sequence of nucleotides in which the 3' and 5" ends on each nucleotide are joined by phospbodiester bonds. The nucleic acid sequences are typically polynucleotides that may be made up of deoxyribonucleotide bases or ribonucleotide bases. Polynucleotides include DNA and RNA, and may be manufactured synthetically in vitro or isolated from natural sources. Sizes of nucleic acid sequences are typically expressed as the number of base pairs (bp) for double stranded polynucleotides, or in the case of single stranded polynucleotides as the number of nucleotides (nt). One thousand bp or nt equal a kilobase (kb). Polynucleotides of Jess than around 40 nucleotides in length are typically called "oligonucleotides". The primer sequences utilised in the present invention for the nucleic acid amplification and primer extension steps are single stranded oligonucleotides.<br>
The term "nucleic acid amplification reaction" as used herein denotes any of a number of related enzymatic techniques that utilise a thermostable DNA polymerase to amplify a specified sequence of DNA using serial rounds of primer extension, denaturation and hybridisation. Typically, PCR is the preferred nucleic acid amplification reaction used in the method of the present invention.<br>
The term "primer extension reaction" is intended to denote a reaction in which nucleic acid primers are designed to hybridize with a target sequence and be<br>
enzymaticaliy extended by adding one or more nucleotides to the 3'-end of the primer. The primers hybridise at a position on a given target sequence that is immediately 5' to or a few bases upstream of the position of a polymorphism, such as a single nucleotide polymorphism. In embodiments of the invention where the primer hybridises on base upstream of a known SNP site, single-base primer-extension acts on primer-target sequence hybrids to add a single chain-terminating nucleotide, often a dJdeoxynucfeotide. The only one of four nucleotides that will extend the primer is the one that is complementary to the sequence on the target strand. The identity of the added nucleotide is determined during the analysis phase of the method of the invention, described in more detail below.<br>
The term "polymorphic allele" is used herein to denote any two or more alternative forms of genetic sequence occupying the same chromosomal locus and controlling the same inherited characteristic. Allelic variation arises naturally though mutation, and may result in phenotypic polymorphism within populations or may result in a conservative (non-phenotypic) polymorphism. Gene mutations typically result in an altered nucleic acid sequence. As used herein. the phenomenon of allelic polymorphism is utilised in respect of single nucleotide polymorphisms (SNPs), insertions, deletions, inversions and substitutions, all of which can occur even in genes that are highly conserved in a given species. SNPs are polymorphisms where the alleles differ by the replacement/substitution of a single nucleotide in the DNA sequence at a given position in the genome. In highly conserved genes, such as 16S rRNA in bacteria, SNPs are highly species and strain specific, thereby allowing accurate genotyping information to be obtained. Other highly conserved regions in microorganisms include the bacterial 32S rRNA gene, yeast 16S and 18S rRNA genes and viral polymerase genes. Nevertheless, it is within the remit of the skilled person to utilise bioinformatics techniques identify SNPs in other alternative conserved regions of the genome for a given microorganism. Polymorphisms, such as those described above, may be linked to specific phenotypic traits in the organism under test. For instance, antibiotic resistance is associated with mutation and, thus, polymorphism. Nevertheless, the method of the present invention is not restricted to identification of only polymorphic<br><br>
positions in the genomes of the microorganisms under test. Where competitor control sequences are used for a given conserved target sequence, the term "polymorphic aiieie" is used loosely to denote the variation at a given part of the sequence between the wild type sequence (that being tested for) and the artificial competitor sequence. In this instance it will be appreciated that the so-called polymorphism is simply to assist in differentiation of the reaction products of primer extension on the competitor template versus the wild type template, as the respective reaction products will have differing relative molecular masses.<br>
The present invention is based in part upon a method for reliable and high throughput testing of one or more biological samples for the presence of microorganisms in that sample. The high throughput analysis is enabled by the use of a diagnostic multiplexing panel (DMP) that is directed towards genotyping of a plurality of microorganisms that are potentially present in the biological sample. The DMP provides a combination of primers that each specifically hybridise with a highly conserved sequence in DNA that is isolated from microorganisms that may be present within a biological sample. The DMP allows for simultaneous primer extension reactions to identify if one or more of a plurality of organisms are potentially present in a single sample. The DMPs of the present invention may suitably be directed at particular therapeutic or diagnostic areas, wherein the microorganisms being tested for fall broadly within a disease area or type. In an example of the invention in use described in more detail below, a DMP is assembled directed at diagnosis of the presence of sexually transmitted infection in biological samples taken from human patients. This form of DMP can suitably test for the presence of bacterial pathogens such as Mycoplasma spp.; Chlamydia spp.; Ureaplasma spp; Neisseria spp.; Gardnereila spp.; Trichomonas spp.; Treponema spp; or the yeast Candida albicans; or viral pathogens such as: cytomegalovirus (CMV); hepatitis viruses (e.g. HAV, HBV and HCV etc.); human immunodeficiency viruses (HIV); human papilloma viruses (HPV); herpes simplex viruses (HSV); Molluscum contagiosum virus (MCV); influenza virus; Epsteln-Barr virus (EBV) and varicella-zoster virus (VZV).<br>
Other disease areas to which DMPs are suitably directed include: food poisoning; tuberculosis; viraliy induced cancer; encephalitis; malaria; hepatitis; meningitis; leishmaniasis; African trypanosomiasis; pneumonia; plague; SARS; MRSA; rabies; anthrax; Rift valley fever tularemia; shigella; botulism; yellow fever; Q fever; eoola; dengue fever; West Nile fever; dysentery; influenza; measles; and typhus.<br>
The invention further enables detection of sequences that confer antibiotic sensitivity in bacterial pathogens by including such sequences in the DMP testing design. This allows speeding up the commencement of treatment of individuals found to be harbouring such pathogens by removing the additional separate step of microbiological identification of antibiotic sensitivity. Furthermore, the invention may provide information about the progression of some diseases by determining the concentration of detected pathogens, which in many cases reflects the progress of the disease. Concentration may include, for example, an assessment of viral load. Quantitative information can be obtained from the primer extension phase of the reaction, for instance, by inclusion of a competitor sequence to a given target sequence, which competitor sequence contains an introduced polymorphism at a specified position in its sequence compared to the target sequence. The competitor sequence can suitably include an alternative nucleotide at the position of a known SNP but which is otherwise identical, (f the competitor is supplied during the nucleic acid amplification stage at a known concentration (or copy number) then is can serve as a benchmark for quantifying concentration of the polymorphism-containing target sequence from the microorganism of interest. In a specific embodiment of the invention it is possible to provide additional competitor sequences at different concentrations (e.g. low, medium and high concentration) all with an introduced sequence variation directed at-a specific site in a target sequence to enable more accurate quantification of the microorganism concentration in the original biological sample. Quantification aside, inclusion of competitor sequences also provides an internal control for all the enzymatic steps In the diagnostic method of the invention.<br>
The DMP of the invention is suitably provided as a plurality of appropriately plexed primers in solution. However, the DMP can also comprise primers that are immobilized on a solid surface such as in the form of a microarray. The solid surface can suitably be in the form of a silicon substrate or a glass substrate.<br>
Resolution of the DMP reaction products following primer extension can be achieved using a number of technologies, including mass spectrometry (e.g. MALDI-TOF), electrophoresis (e.g. capillary electrophoresis), DNA microarray (e.g. Affymetrix's GeneChip™ or printed DNA arrays), via incorporation of fluorescently labelled nucleotides (e.g. Beckman Coulter's SNPstream® or Applied Biosystems' SNPIexa), or other labels (e.g. antigen, biotin, or a radiolabel). The preferred method for resolution of the primer extension products involves determination according to relative molecular weight, both mass spectrometry and capillary electrophoresis are favoured for this.<br>
In one specific embodiment of the present invention, each primer comprised within the DMP varied by overall nucleotide length such that no two primers were of the same relative molecular weight either before or after the primer extension reaction. The products of the reaction are purified in order to optimise mass spectrometric analysis. After purification the products were spotted onto an appropriate element, typically a silicon chip incorporating high-density, photo-resistant array of mass spectrometry analysis sites (e.g. a SpectroCHIP®) and analysed on a matrix assisted laser desorption/ionisation-time-of-flight (MALDI-TOF) mass spectrometer (e.g. Sequenom's MassArray® Mass Spectrometer as described in US Patent Nos. 6500621, 6300076, 6258538, 5869242, 6238871, 6440705, and 6994969). The results of mass spectrometric analysis will be processed using an appropriate software package, so as to provide information on the presence or absence of primer extension products that are correlated to the presence or absence of particular specified microorganisms in the biological sample.<br>
The invention is further illustrated by the following non-limiting example.<br>
!-
The present inventors have developed a cost effective, robust and highly accurate test, which has the potential to determine any number of infections in a single sample. The test comprises two parts - a simple home collection kit (utilising Whatman FTA Elute® paper as the sample carrying substrate) and a novel sexually transmitted infection Multiplex Panel (STIMP) as the Diagnostic Multiplex Panel (DMP) which can determine whether any given individual is infected with one or several sexually transmitted bacterial, viral, protozoan and/or fungal pathogens using DNA based technology.<br>
The present example demonstrates that.<br>
•	Whatman FTA Elute®  paper is an adequate carrier of bacterial, fungal<br>
and protozoan pathogens derived from a human urine sample.<br>
«    compounds present within Whatman FTA Elute^  paper or urine do not interfere with down stream enzymatic testing processes.<br>
•	the novel STIMP functions as a DMP and is able to detect individual<br>
pathogens from a pathogen mixture.<br>
MATERIALS &amp; METHODS<br>
• CLINICAL SAMPLES<br>
A total of 44 samples were obtained from patients attending a private GUM (Genito-Urinary Medicine) clinic located in the Ukraine.<br>
Patients in the experimental group were asked to provide a first void urine sample (approximately 30ml-50ml) in a sterile collection pot upon attending the consultation. All samples were provided with informed patient consent and all ethical requirements and regulations (including the method of sample collection)<br>
ere met and carried out accordingiy as stipulated by the Ukrainian Department of Health.<br>
Each urine sample was transferred onto a Whatman FTA Eiute® (Whatman pic, Brentford, UK) sample card by dipping a sterile foam tipped applicator (Puritan®,Maine, USA) once into the urine sample pot and then blotting four times in four separate areas on the card.<br>
After sample transfer each individual Whatman FTA Bute® card was dried at room temperature until completely dry. To prevent cross contamination dry sample cards were placed individually inside a self-seal polythene bag and further stored at room temperature.<br>
Batch 1 and 2 samples (31 and 13 separate samples respectively) were then shipped in two consignments (via Federal Express® courier service) to the testing location at a laboratory in Germany. Batch 1 sample collection was undertaken during the period 12 December - 26 December 2006 inclusive. Batch 2 sample collection was undertaken during the period 5 January - 12 January 2007 inclusive.<br>
All samples were analysed in parallel by an independent local laboratory (Kiev, Ukraine) which specialises in Sexually Transmitted Infection testing utilising conventional DNA based detection techniques as recommended by the Ukrainian Department of Health.<br>
AH samples were tested for presence of the following microorganisms:<br>
Candida albicans Chlamydia trachomatis Gardnerelta vaginalis Mycoplasma genitalium Mycoplasma hominls Neisseria gonorrhoeae<br>
Trichomonas vaginalis Treponema pallidum Ureaplasma urealyticum<br>
The results obtained from the local testing latioratory (designated as the 'clinic') were then compared using the results obtained utilising the STIMP/DMP method (designated as the lab).<br>
• EXPERIMENTAL PROCEDURES<br>
DNA EXTRACTION<br>
To account for differences in DNA concentration six 8mm sample disks from each Whatman FTA Elute® sample card were excised using a hand held punching device in the following sequence - 1 x circle per 2 ml tube, 2 X circles per 2 ml tube, 3 x circles per 2 ml tube. The punching device was cleaned after excising of each sample card by punching through clean filter paper three times followed by punching through filter paper soaked with 70% EtOH a further three times. To account for cross contamination a clean FTA Elute* card was then punched once and DNA extracted as below (tube number 4),<br>
DNA extraction was performed using Whatman's FTA Elute card® DNA extraction protocol in our modification in order to account for a 6mm sample disk punch as opposed to the recommended 3mm sample disk punch. One, two and three 6mm sample disks or Excised Paper Fragments (EPF) were placed into separate 2ml round bottom Eppendorf tubes to which 0.7ml, 1.4ml and 2.1ml of ddH2O was added respectively.<br>
Each sample was vortexed for 5 seconds three times and the sample disks transferred into separate clean 0.5ml Eppendorf tubes.<br>
50ul, 80pl and 100µl of ddH20 were added to each tube containing one, two and three sample disks respectively after which the tubes were incubated at<br><br>
95C for 30 minutes. After incubation the samples were spun at 12000g for 2 minutes and stored at +4°C. For PCR amplification 1ul of each sample was used directly or as a 1:1 dilution with ddH2O<br>
POSITIVE CONTROLS<br>
The following cell lines obtained from the National Collection of Type Cultures (NCTC) were used as positive controls.<br>
Cell fine No. Species<br>
NC12700	N. gonorrhoeae<br>
N C11148	N. gonorrhoeae<br>
NC08448	N. gonorrhoeae<br>
NC10177	U. ureaiyticum<br>
NC10111	M.hominis<br>
NC10915	G. vaginalis<br>
NCPF3179	C. albicans<br>
NC10195	M. geniialium<br>
The cells were diluted with 500pl of fresh urine and then 50ul from each sample was taken to make positive controls as follows.<br>
Positive control No. 1<br>
NC12700	N. gonorrhoeae<br>
NC10177	U. ureaiyticum<br>
NC10111	M.hominis<br>
NC10915	G. vaginalis<br>
NCPF3179	G albicans<br>
NC10195	M.genitalium<br>
Positive control No. 2<br>
NC11148	N. gonorrhoeae<br>
NC10177	U. ureaiyticum<br>
NC10111	M. hominis<br>
NC10915	G. vaginalis<br>
NCPF3179	C. albicans<br>
NC10195	M. genitalium<br>
Positive control No. 3<br>
NC08448	N. gonorrhoeae<br>
NC10177	U. ureaiyticum<br>
NC10111	M. hominis<br>
NC10915	G. vaginalis<br>
NCPF3179	C. albicans<br>
NCI 0195	M. genitalium<br>
The positive controls and the fresh urine sample used for diluting the cells were pipetted onto individual FTA Bute® cards as recommended by Whatman (50ul per ca 1cm2). After application of the samples the cards were dried at 60°C for one hour and DNA extracted as above.<br>
In addition an aliquot of each positive control and the fresh urine sample (used for diluting the cells) were used both directly and at 1:5 dilution in ddH2O for PCR amplification.<br>
At the time of the experiment we were unable to obtain positive controls for C, trachomatis, T, vaginalis or T, pallidum,<br>
ASSAY DESIGN<br>
The DMP assay design is based on DNA sequences highly conserved between different strains of each species of interest. To account for high variability, three<br>
different areas in the conserved regions were used to design three assays for each pathogen. For the all pathogens the conserved region chosen was the 16S rRNA gene, except for C. albicans where the conserved region chosen was the 18S rRNA gene (see Fig. 1).<br>
For U. urealyiicum two different conserved areas were used to design six assays. The exact sequences for PCR and primer extension (PE) primers are shown in Table 1.<br>
The assay identifies the presence of the following bacterial, fungal and protozoan species in the sample.<br>
C, albicans C. trachomatis G, vaginalis M, genitalium M. hominis N. gonorrhoeae T. vaginalis T. pallidum U. ureatyticum<br>
Two types of assays were performed. The first type included control competitor sequences for each target, the second did not contain any competitors. The role of the competitor is to serve as the internal positive control for PCR, PE and other enzymatic reactions as well as chip spotting. In a sample with a competitor ft would be expected to see at least a signal for the competitor even if the sample did not contain the corresponding DNA target from the pathogen. The competitors were designed to be identical to the target DNA sequence with a known single nucleotide difference at the site of the SNP of interest, the sequences of the competitors are shown in Table 1. The competitors were added at an approximate amount of 30 copies per target per PCR amplification.<br>
In the samples analysed without competitors a signal was expected only when the target DNA from the pathogen was present in the sample. Targets from the same multiplex serve as internal positive controls for each other, however if the signal is absent for all targets amplified together it is impossible to conclude whether the sample does not contain any target pathogen DNA or that one or several enzymatic reactions failed.<br>
PCR AMPLIFICATION<br>
PCR amplification was performed in 5µl reaction volume containing 1.25x HotStar® PCR Buffer, 1.625mM MgCI2, 0.04mM of each dNTP, 0.1 µM of each primer and 0.1 U of HotStarTaq®. Cycling parameters were as follows:<br>
95°C	15min<br>
94°C	20 sec<br>
56°C	30 sec<br>
72°C	1 min<br>
For 45 cycles<br>
72X	3 min<br>
4°C	5 min<br>
15ºC	Forever<br>
Amplification was performed on an MJR Tetrad Thermo Cycler,<br>
SHRIMP ALKALINE PHOSPHATE (SAP) TREATMENT<br>
After PCR amplification the following was added to each tube: 1.53µl of H20, 0.17µl of TS buffer and 0.30µl of SAP. The reaction was performed using the following parameters:<br>
37ºC   20 min 85ºC   5 min 4°C    Forever<br>
The reaction was performed on an MJR Tetrad Thermo Cycler. PRIMER EXTENSION REACTION<br>
Primer extension was carried out according to the iPLEXTM protocol (Sequenom, Inc., San Diego, CA, USA). After incubation 2µl of the tPLEX primer extension cocktail containing 1x iPLEX buffer, 1x iPLEX termination mix, 0.625uM of each extension primer and 1x iPLEX enzyme was added to each tube. iPLEX reaction was performed using the following parameters:<br>
94°C   30 sec<br>
94°C   5 sec<br>
52°C   5 sec<br>
80ºC   5 sec<br>
Go to step 3x5 times<br>
Go to step 2 X 40 times<br>
72X   3 min<br>
4°C     Forever<br>
The reaction was performed on an MJR Tetrad Thermo Cycler.<br>
DESALTING<br>
16pl of H2O and 6mg of SpectroCLEAN® resin was added to the above mixture and rotated on a circular shaker for 30 minutes and then centrifuged for 5 min at 3,000 g to precipitate the resin.<br>
SAMPLE SPOTTING<br>
9 nanolftres of each sample were spotted onto a 384 SpectroCHIP® using a MassARRAY® Nanospotter robot as per the manufacturers instructions. Each chip was then read on a MALDI-TOF mass spectrometer, a MassARRAY®<br>
Compact Analyser, and data collected using Typer v3.3 (or above) and stored in the database,<br>
GENOTYPE SCORING &amp; DETERMINATION OF INFECTION STATUS OF SAMPLE<br>
For each assay (DNA target) a specific competitor was designed to differ from the target DNA by an artificially introduced single nucleotide polymorphism (SNP). The latter was designated as the MUT (Mutant) genotype by the genotyping software. The pathogenic SNP was designated as C (Control) genotype. When both competitor and the target DNA were present in the reaction a heterozygous genotype designated as either C.MUT or MUT.C was scored by the software. The order of the alleles in the heterozygous genotype reflects the relative proportion of corresponding DNA in the reaction.<br>
The sample was considered to be infected when at least two out of the three assays indicated the presence of pathogenic DNA in a single DNA sample. As three different DNA samples were obtained from each patient sample card it was expected that all three samples should show identical results (NB. because each sample potentially contained different copy numbers of a pathogen, DNA samples derived from two and three sample disks were expected to produce more reliable results due to increased amounts of pathogenic DNA in comparison with a DNA sample extracted from a single sample disk). Occasionally only one or two assays indicated the presence of pathogenic DNA in a patients sample while the majority of assays for this pathogen on all DNA samples were negative. These results were thought to be artefacts (most probably due to cross contamination) however the samples were scored as positive and indicated by the presence of a 'x' cross next to the result within the summary results table (Table 2).<br>
.    RESULTS &amp; DISCUSSION<br>
Whatman FTA Elute® paper<br>
To investigate whether any compounds present within Whatman FTA Elute® paper or urine can interfere with down stream enzymatic processes a series of mock samples containing an aliquot of known good quality DNA was diluted 1:1 with eluant obtained after extracting one, two and three sample disks or with fresh urine. The samples were then used for PCR and other downstream enzymatic reactions. For alt the samples good quality PCR products and mass spectrum was obtained (data not presented). This indicates that the chosen extraction method and washing protocol are adequate for MALDl-TOF analysis and also show that substances in human urine do not significantly affect the reliability of the DMP analysis.<br>
To determine whether Whatman FTA Elute® paper is able to capture and preserve pathogenic DNA derived from a urine sample several DNA samples (extracted from the cards) belonging to the patients identified as positive by the local laboratory analysis (clinic) were amplified in a 50µl reaction using the primers from the STIMP/DMP and the products separated on 2% agarose gel in 1xTBE buffer (see Fig. 1). In all cases a good signal for the correct size amplicons was observed indicating the presence of pathogenic DNA on Whatman FTA Elute® paper in amounts sufficient for reliable PCR amplification.<br>
The same experiment was performed using positive control urine samples and similar results were observed (data not presented).<br>
These results confirm that cellulose based products such as Whatman FTA Elute® paper can be used as a suitable solid substrate both collect and transport patient samples and show no visible time dependant deterioration. Nevertheless, it is envisaged that crude urine samples as well as biological samples immobilised on other types of substrate can also be used for testing utilising the present STIMP/DMP method.<br>
STIMP/DMP METHODOLOGY<br>
All the samples were tested in reactions with or without competitor control sequences. The experiment in which competitors were included produced signals for competitor DNA, in most cases including positive controls.<br>
In this section the results obtained from the experiments when competitors were not included in the reaction are discussed.<br>
The results of the experiments are presented in Table 2.<br>
POSITIVE CONTROLS<br>
To confirm whether the STIMP/DMP method can be used for detecting individual pathogens from a pathogen mixture three positive controls obtained by mixing sexually transmitted pathogens obtained from the NCTC were created using urine as a medium (see above). The positive control samples were analysed both as urine samples and as Whatman FTA Elute® paper samples (Table 3). In all cases the presence of each pathogen within the sample was detected independent of its nature. Positive controls differed by the strain of N. gonorrhoeae present in them. In all the assays for this species the STIMP/DMP method was able to detect the presence of N. gonorrhoeae independent of the strain type. This shows that the regions of DNA chosen for developing the N. gonorrhoeae assays do not show strain-dependant specificity and can be used to detect the presence of any known A/, gonorrhoeae species in the sample.<br>
CLINICAL SAMPLES<br>
When analysing the clinical samples the STIMP/DMP lab results confirmed the results obtained and analysed independently from the clinic in all cases (Table 2). Several samples were also identified as positives for infections not detected by the testing laboratory. However, because the pathogen was not detected by all assays and because only one (rarely two) DNA samples had a positive signal these results should be treated as artefacts of this pilot study.   In most<br>
cases this occurred when the patient whose sample was on the previously punched sample card was positive for a particular infection (e.g. patient 2 &amp; 3 -U. urealyticum, patients 11 &amp; 12 - M. genitalium etc). The most likely explanation for these artefacts is the carrying over of DNA from the previous sample. This is supported by the fact that contamination control samples (clean Whatman FTA Elute® card) used in between patient samples were also found to be positive for the same infections. Clearly this demonstrates the high sensitivity of the DMP of the invention and future experimental procedures will be optimised to avoid cross contamination.<br>
Three different DNA samples were collected from each patient sample card. The samples contained different copy number of pathogenic ONA (when present), in the majority of cases it was possible to identify the infection in DNA samples from positive patients extracted from two or more sample disks.<br>
For most DNA samples from positive patients all three assays identified infection. In a very small number of instances one of the three assays did not produce good quality signal and was flagged by the genotyping software.<br>
It is also possible to determine the copy number of target DNA present in the reaction by titrating the amount of competitor or including three different competitors at known initial concentrations in the same reaction. The latter approach is recommended for pathogens whose level of infectious load is important from a clinical perspective, for example in individuals infected with HIV or G. vaginalis.<br><br>
Table I<br><br>
(Table Removed)<br><br><br>
Tabte 1 (cont'd)<br><br><br>
Table3 -Positive Control data<br><br><br>
(Table Removed)<br><br><br>
Although particular embodiments of the invention have been disclosed herein in detail, this has been done by way of example and for the purposes of illustration only. The aforementioned embodiments are not intended to be limiting with respect to the scope of the appended claims, which follow, it is contemplated by the inventors that various substitutions, alterations, and modifications may be made to the invention without departing from the spirit and scope of the invention as defined by the claims,<br><br><br><br><br><br><br><br>
We claim:<br>
m<br>
^ 1. A method for determining whether two or more species of specified<br>
microorganisms are present within a biological sample that potentially<br>
comprises the microorganisms comprising:<br>
(a) immobilizing the biological sample on and/or within a solid substrate I at a first location;<br>
(b) transferring the immobilized biological sample to at least a second<br>
location and performing a extraction step on the solid substrate so as to<br>
extract any microorganism DNA immobilized on and/or within the solid<br>
substrate;<br>
(c) performing a nucleic acid amplification step on microorganism DNA<br>
extracted in step (b), wherein the amplification step comprises a plurality<br>
of amplification primers that are directed towards amplification of a<br>
plurality of highly conserved sequences, from the two or more specific<br>
microorganisms, and wherein amplified sequences are designated as<br>
target sequences;<br>
(d) combining the target sequences with a plurality of primer sequences<br>
comprised within a diagnostic multiplexing panel (DMP), wherein each<br>
primer sequence facilitates genotyping of the target sequence;<br>
(e) performing a primer extension reaction on the combination of target<br>
sequences and the DMP present in (d), thereby producing a DMP<br>
reaction product; and<br>
(f) analysing the reaction product so as to determine genotype of any<br>
target sequences that are present and correlating the genotype of the<br>
target sequences in the reaction product with the identification of<br>
specified microorganisms present in the biological sample;<br>
34<br>
wherein the specified microorganisms are selected from one or more of<br>
the group consisting of: Mycoplasma spp., Chlamydia spp., Ureaplasma<br>
^ spp., Neisseria spp., Gardnerella spp.. Trichomonas spp., Treponema<br>
spp., and Candida albicans; and<br>
wherein the DMP comprises one or more primer sequences selected<br>
from SEQ ID NOs: 31-45, and/or one or more primer sequences selected<br>
from SEQ ID NOs: 76-90.<br>
2. The method as claimed in claim 1, wherein the first location is remote<br>
from the second location.<br>
I 3. The method as claimed in claims 1 and 2, wherein the biological sample<br>
comprises at least one of the group consisting of: urine; saliva; blood; sputum;<br>
semen; faeces; a nasal swab; tears; a vaginal swab; a rectal swab; a cervical<br>
smear; a tissue biopsy; and a urethral swab.<br>
4. The method as claimed in any previous claims, wherein the biological<br>
sample is immobilized on and/or within a solid substrate at the first location, and<br>
an extraction step is performed on the solid substrate so as to extract any<br>
microorganism DNA immobilized on and/or within the solid substrate at the<br>
second location.<br>
5. The method as claimed in claim 4, wherein the solid substrate comprises<br>
an absorbent fibrous material impregnated with one or more reagents that act to<br>
immobilize and inactivate any microorganisms present in the biological sample.<br>
6. The method as claimed in claim 5, wherein the solid substrate comprises<br>
a material selected from a cellulose-based paper; a microfibrous membrane; a<br>
glass-fibre material; a polymeric fibre material; a woven fabric; and a non-woven<br>
fabric.<br>
35<br>
7. The method as claimed in any previous claim, wherein the specified<br>
microorganisms are selected from one or more of the group consisting of:<br>
" bacteria; fungi; viruses; and protozoa.<br>
8. The method as claimed in any previous claims, wherein the two or more<br>
microorganisms include human pathogens that are the causative agents in one<br>
or more of the diseases selected from the group consisting of: sexually<br>
transmitted infection; food poisoning; tuberculosis; virally induced cancer;<br>
encephalitis; malaria; hepatitis; meningitis; leishmaniasis; African<br>
trypanosomiasis; pneumonia; plague; SARS; MRSA; rabies; anthrax; Rift valley<br>
fever; tularemia; shigella; botulism; yellow fever; Q fever; ebola; dengue fever;<br>
West Nile fever; dysentery; influenza; measles; and typhus.<br>
9. The method as claimed in claims 7 and 8, wherein the viruses are further<br>
selected from the group consisting of: cytomegalovirus (CMV); hepatitis A virus<br>
(HAV); hepatitis B virus (HBV); hepatitis C virus (HCV), hepatitis E virus (HEV),<br>
hepatitis G and GB virus (GBV-C); human immunodeficiency viruses (HIV);<br>
human papilloma viruses (HPV); herpes simplex viruses (HSV); Molluscum<br>
contagiosum virus (MCV); influenza virus; Epstein-Barr virus (EBV) and<br>
varicella-zoster virus (VZV).<br>
10. The method as claimed in any previous claims, wherein the DMP is<br>
directed towards identification of alleles from a combination of microorganisms<br>
potentially present in the biological sample, wherein the combination includes<br>
bacteria, viruses and fungi.<br>
11. The method as claimed in claim 10, wherein the DMP is directed towards<br>
identification of microorganisms that are associated with a particular type of<br>
disease.<br>
12. The method as claimed in claim 11, wherein the type of disease is<br>
sexually transmitted infection. i<br>
i<br>
36<br>
13. The method as claimed in claim 12, wherein the DMP comprises primer<br>
sequences that hybridise with one or more target sequences obtained from<br>
^ microorganisms selected from the group consisting of: Mycoplasma genitalum;<br>
I Mycoplasma hominis; Chlamydia trachomatis.; Ureaplasma urealyticum;<br>
Neisseria gonorrhoea; Gardnerella vaginalis; Trichomonas vaginalis.;<br>
Treponema pallidum; CMV; HAV; HBV; HCV; HEV. GBV-C, HIV-1; HIV-2; HPV;<br>
HSV-1; HSV-2; MCV; VZV; EBV; and Candida albicans.<br>
14. The method as claimed in any previous claims, wherein the highly<br>
conserved polymorphic allele comprises all or a part of a microorganism gene<br>
selected from: a bacterial 16S rRNA; a bacterial 32S rRNA; a yeast 16S rRNA;<br>
a yeast 18S rRNA; and a viral polymerase.<br>
15. The method as claimed in any previous claims, wherein the highly<br>
conserved sequence comprises a polymorphic allele selected from the group<br>
consisting of: a single nucleotide polymorphism (SNP); an insertion; a deletion;<br>
an inversion; and a substitution.<br>
i 16. The method as claimed in any previous claims, wherein if two or more<br>
specified microorganisms are present in the biological sample the primer<br>
extension reaction produces a DMP reaction product comprising at least two<br>
extended primer sequences each of a known predetermined molecular weight<br>
that is different to the other.<br>
17. The method as claimed in claim 16, wherein DMP reaction product(s) are :<br>
analysed using a technique that resolves extended primer sequences according<br>
to their molecular weight.<br>
18. The method as claimed in f claim 17, wherein DMP reaction product(s)<br>
are analysed using a technique selected from: MALDI-TOF mass spectrometry;<br>
and/or capillary electrophoresis.<br>
19. The method as claimed in any of claims 1 to 15, wherein the primer<br>
extension reaction comprises a primer labelling reagent such that if one or more<br>
37<br>
specified microorganisms is present in the biological sample the primer<br>
extension reaction incorporates the labelling reagent into the primer extension<br>
^ product, thereby producing a DMP reaction product comprising the labelling<br>
reagent.<br>
20. The method as claimed in claim 19, wherein the primer labelling reagent<br>
comprises a label selected from: a radiolabel; a fluorescent label; and an f<br>
antigen.<br>
21. The method as claimed in claims 19 and 20, wherein DMP reaction<br>
product(s) are analysed using a technique that identifies presence of an<br>
incorporated labelling reagent in the primer extension product.<br>
22. The method as claimed in any previous claims, wherein the plurality of<br>
primer sequences comprised within the DMP are immobilized on a solid<br>
support.<br>
;<br>
23. The method as claimed in claim 22, wherein the solid support is selected<br>
from one of: glass; and silicon.<br>
24. The method as claimed in any previous claims, wherein the nucleic<br>
amplification step comprises a plurality of amplification primers that are directed<br>
towards amplification of a plurality of highly conserved sequences from one or<br>
more specified microorganisms.<br>
•<br>
25. The method as claimed in claim 24, wherein the plurality of amplification<br>
primers comprise one or more primer pairs selected from the group consisting<br>
of SEQ ID NOS: 1/ 2; 3/ 4; 5/6; 7/8; 9/10; 11/12; 13/14; 15/16; 17/18; 19/20;<br>
21/22; 23/24; 25/26; 27/28; and 29/30.<br>
26. The method as claimed in claim 24, wherein the plurality of amplification<br>
primers comprise one or more primer pairs selected from the group consisting<br>
of SEQ ID NOS: 46/47; 48/49; 50/51; 52/53; 54/55; 56/57; 58/59; 60/61; 62/63;<br>
64/65; 66/67; 68/69; 70/71; 72/73; and 74/75.<br>
38<br>
27. The method ds claimed in any previous claims, wherein the DMP<br>
comprises one or more primer sequences selected from SEQ ID NOS: 31-45.<br>
28. The method as claimed in any previous claims, wherein the DMP<br>
comprises one or more primer sequences selected from SEQ ID NOS: 76-90.<br>
29. The method as claimed in any previous claims, wherein one or more<br>
control competitor sequences are combined with the target sequences prior to<br>
the nucleic acid amplification step of part (c), wherein each competitor<br>
•<br>
sequence is identical to a corresponding target sequence except that the<br>
competitor comprises a sequence variation at a specified position compared to<br>
the corresponding target sequence.<br>
30. The method as claimed in claim 29, wherein the sequence variation<br>
comprises an artificially introduced SNP.<br>
31. The method as claimed in any previous claims, wherein one or more<br>
control sequences are combined with the target sequences prior to the nucleic<br>
acid amplification step of part (c), and wherein the one or more control<br>
sequences comprise a sequence of DNA selected from: a species unrelated to<br>
that of the biological sample; a species unrelated to the microorganism(s) being<br>
tested for; and a synthetic DNA sequence, and wherein the nucleic acid<br>
amplification step and the DMP comprise corresponding primer sequences that J<br>
specifically hybridise with each of the one or more control sequences.<br>
32. The method as claimed in any previous claims, wherein the biological<br>
sample is obtained from a human.<br>
39 I<br>
33. The method as claimed in any of claims 1 to 31, wherein the biological<br>
sample is obtained from one of the group consisting of: a non-human animal; a<br>
W plant; and a foodstuff.<br>
Dated this 22""^ day of August, 2008<br>
(CHETAN CHADHA)<br>
PATENT AGENT<br>
40</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNsYWltcy0oMTktMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Claims-(19-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNsYWltcy0oMjctMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Claims-(27-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc29wbmRlbmNlIE90aGVycy0oMTktMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Corresopndence Others-(19-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence Others-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence Others-(16-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence Others-(24-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence Others-(25-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMjYtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence-others (26-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTMtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence-Others-(13-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLW90aGVycy0oMTUtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence-others-(15-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence-Others-(27-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Correspondence-Others-(29-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1ERUxOUC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7202-DELNP-2008-Correspondence-Others-(29-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1ERUxOUC0yMDA4LUZvcm0tMS0oMjktMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7202-DELNP-2008-Form-1-(29-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMTggKDI2LTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-18 (26-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1ERUxOUC0yMDA4LUZvcm0tMjYtKDI5LTEyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">7202-DELNP-2008-Form-26-(29-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMTMtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(13-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMTUtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(15-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMTUtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMTYtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(16-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMjUtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(25-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMjctMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(27-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUZvcm0tMy0oMjktMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Form-3-(29-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LUdQQS0oMTktMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-GPA-(19-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LVBldGl0aW9uLTEzNy0oMjctMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Petition-137-(27-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzIwMi1kZWxucC0yMDA4LVBldGl0aW9uLTEzOC0oMjctMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7202-delnp-2008-Petition-138-(27-08-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260089-a-method-of-producing-a-polymer-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260091-a-cap-having-storage-space.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260090</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7202/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Mar-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Aug-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>STIRUS GLOBAL SOLUTIONS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>4TH FLOOR, 5-7 JOHN PRINCES STREET, LONDON W1G 0JN, U.K</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SEMIKHODSKII, ANDREI</td>
											<td>134 SOMERSET ROAD, WIMBLEDON SW19 5HP, U.K</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GREEN, SIMON</td>
											<td>134 SOMERSET ROAD, WIMBLEDON SW19 5HP, U.K</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2007/000195</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-01-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0601302.3</td>
									<td>2006-01-23</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260090-high-throughput-testing-for-presence-of-microorganisms-in-a-biological-sample by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:45:59 GMT -->
</html>
