<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260273-process-for-biochemical-synthesis-of-1-4-butanediamine by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:21:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260273:PROCESS FOR BIOCHEMICAL SYNTHESIS OF 1,4-BUTANEDIAMINE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PROCESS FOR BIOCHEMICAL SYNTHESIS OF 1,4-BUTANEDIAMINE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a process for biochemical synthesis of 1,4-butanediamine in a microorganism having an increased level of an ornithine decarboxylase activity (increased ODC activity) as compared to the native level of the ornithine decarboxylase activity, wherein in the microorganism also an increased activity of N-acetylglutamate formation is present as compared to the native level of activity of N-acetylglutamate formation in the microorganism and wherein 1,4-butanediamine produced in the microorganism is excreted into a fermentation broth, and is recovered from the fermentation broth. The invention also relates to vectors, plasmids and hosts carrying a corresponding increased ODC activity and an increased activity of N-acetylglutamate formation.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BIOCHEMICAL SYNTHESIS OF 1,4-BUTANEDIAMINE<br>
The present invention relates to a new process for biochemical synthesis of 1,4-butanediamine (CAS number 110-60-1; a compound also referred to as tetramethylenediamine; in biochemical literature it is also being referred to as putrescine) in a microorganism having an increased level of an ornithine decarboxylase activity as compared to the native level of the ornithine decarboxylase activity. Ornithine decarboxylase hereinafter also will be referred to as ODC. Generally such microorganisms having ODC activity are known to be capable of producing polyamines such as spermidine and spermine, which are the common names for respectively the products N-(3-aminopropyl)-1,4-butanediamine and N,N'-bis-(3-aminopropyl)-1,4-butanediamine. Such compounds, as well as various short linear diamines themselves such as, for instance, 1,4-butanediamine and 1,5-pentanediamine (also referred to as cadaverine), are often referred to in biochemical studies as polyamines, even though from a strictly chemical definition of polyamines a higher number of amino groups would be expected. For the purposes of the present patent application, however, the term polyamines is being used in its biochemical meaning and therefore includes 1,4-butanediarnine.<br>
The compound 1,4-butanediamine is an important raw material for the production of some of the major engineering plastics: polyamide-4,6, either in the form of a homopolymer, or copolymerised, for example, with about 5 wt.% of polyamide-6 monomer (caprolactam). The homopolymer polyamide-4,6 (nylon-4,6) was described as early as 1938(US-A-2,130,948, Carofhers). It is the polycondensation product of the monomers 1,4-butanediamine and adipic acid. Presently, especially compounds of polyamide-4,6 are being produced and sold by DSM in the Netherlands under the trade name STANYL®.<br>
For the synthesis of 1,4-butanediamine a number of chemical routes are known. All these chemical routes suffer from the disadvantage that starting materials have to be obtained form sources that are considered to be non-renewable. There exists, however, a substantial need for providing new and feasible routes for the synthesis of 1,4-butanediamine starting from renewable carbon sources and using biochemical processes (also referred to as "biotransformation") in living cells. In general, polyamines are considered to be toxic for any cell or microorganism used in biochemical eduction. Therefore, until now such new routes by biochemical synthesis, however, were believed to be unattractive.<br>
This can for instance be seen from the following references: Fukuchi et al., J. Biol. Chem., Vol.270 (1995), pages 18831-18835; and Suzuki et al., Proc. Natl. Acad. Sci. USA, Vol.91 (1994), pages 8930-8934.<br>
Fukuchi clearly describes the decrease in cell viability (and of synthesis of almost all kinds of proteins) due to the accumulation of spermidine in spermidine acetyltransferase-deficient cells of £ coli (i.e. in cells lacking the acetyltransferase SpeG). It is to be noticed that Limsuwum ef a/. (J. Bacteriol. Vol.182 (2000), pages 5373-5380) have shown that at low temperatures such problems can be overcome by overexpression of the dedicated gene speG. Spermidine is a product that is being produced in cells from 1,4-butanediarnine as an intermediate. Accordingly, biosynthesis of 1,4-butanediamine inevitably also leads to formation of spermidine.<br>
Suzuki ef a/, on the one hand also demonstrate (in mice cells), that overexpression of ODC results in accumulation of polyamines, especially of spermidine, and that- upon addition of small amounts of spermidine - already cell death is observed even in cells that are not deficient in speG. Suzuki et a/, suggest that this lowered cell viability is due to an insufficient feedback inhibition of ODC by antizymes and can be overcome by overproduction of a suitable antizyme. Such overproduction of antizymes then also would lower the production of polyamines in the cells and is therefore not feasible for DAB production.<br>
Further, as Kashiwagi et a/, described in J. Bacteriol. Vol.170 (1988), pages 3131-3135, the contents of polyamines in E. coll can be adjusted by overexpression of an omfthine decarboxylase (ODC) encoding gene, in particular of the constitutively expressed speC. For their experiments the plasrnid pODC as produced by Boyle et al. (Methods in Enzymology, Vol.94 (1983), pages 117-121, was used in the cloning. Clearly, such overproduction of ornithine decarboxylase did not lead to strongly increased levels of 1,4-butanediamine content in the cells. At a 70-fold level of ODC no more than a 20% increase of 1,4-butanediamine content in the cells was observed, independent of the amount of ornithine added to the cells. On the other hand, however, cells grown in the presence of ornithine were shown to exhibit increased excretion of 1,4-butanediamine. At the 70-fold level of ODC in total about 8,5 times higher production of 1,4-butanediamine (the titer of 1,4-butanediamine produced was approximately 20-25 mg/I, in total for in- and external concentrations, i.e. an extremely low concentration) was found. The authors suggested that such rather low efficiency of 1,4-butanediamine production might be due to shortage of ornithine, and tried to solve this by external feeding of ornithine, but only reached a minor improvement. Accordingly, it would seem to be impossible to provide biochemical<br>
synthesis processes for the production of 1,4-butanediamine at significantly higher levels than 30 mg/l.<br>
These studies mentioned above, moreover, were not directed to synthesis of polyamines (including 1,4-butanediamine) as such, but rather tried to get more insights in the physiological functions of polyamines at the molecular level. At higher levels of ornithine in the cells presumably also more arginine would be present in the cells. According to the teaching of Kashiwagi, such higher amounts of arginine should have a substantial negative effect on 1,4-butanediamine formation.<br>
EP-A-0726240 until now is one of the very few patent references relating to the biochemical synthesis of polyamines, including 1,4-butanediamine. However, it describes the production of, inter a//a,1,4-butanediamine by fermentation of natural products containing proteins as a major component. In said process, the natural products are first treated by subjecting them to partial or total degradation, and any undesirable compounds (e.g. Hg, Cr, As, Cd, Se and Pb), cell growth inhibitors, pesticides, antibiotics, detergents, soaps, fats, oils, cyanides and phenols are then removed before the fermentation step. The putrescine and other diamines produced in such a way are being (re-)used as fertilizers and manures, but contain such large number of other substances that they are unsuitable as a raw material for the production of, for example, polyamide-4,6.<br>
Accordingly, there remains a need for an efficient alternative biosynthetic route for the synthesis of 1,4-butanediamine at significantly higher titers than about 30 mg/l, preferably even without the need for external feeding of (expensive) ornithine. This need for improved availability of 1,4-diaminobutane is based on its intended use as a starting material, for instance, for the production of polyamide-4,6. In general, the routes to 1,4-butanediamine as are known until today are quite laborious and troublesome, and may lead to a quality of said product which - without further purification - is difficult to be used in the production of nylons. The known chemical routes to 1,4-butanediamine require relatively expensive starting materials and reactants (including reactants that are difficult to handle), and relatively severe reaction conditions of temperature and pressure in a multi-step and multi-reactor design, as well as the use of expensive catalyst systems. Accordingly there remains a need for alternative routes to 1,4-butanediamine, preferably from much less expensive raw materials and avoiding problems of handling reactants like hydrocyanic acid. It is well known that naturally growing, and thus renewable, materials from agricultural production are the basis for carbon sources such as glucose (or other appropriate carbon sources and mixtures thereof) that can be used in fermentation. Such<br>
renewable materials are relatively cheap and abundantly available. In general, it is considered to be very advantageous if renewable materials can be used as starting materials for all kinds of chemical materials.<br>
It is thus an aim of the present invention to provide improved possibilities for large-scale industrial production of 1,4-butanediamine by biotransformation.<br>
The present inventors surprisingly have found that this aim is achieved with a new process for biochemical synthesis of 1,4-butanediamine in a microorganism having an increased level of an ornithine decarboxylase activity (increased ODC activity) as compared to the native level of ornithine decarboxylase activity, wherein in the microorganism also an increased activity of N-acetylglutamate formation is present as compared to the native level of activity of N-acetylglutamate formation in the microorganism and that 1,4-butanediamine produced in the microorganism is excreted into a fermentation broth, and is recovered from the fermentation broth.<br>
As meant in the present patent application, the term "biochemical synthesis" (a term that, in the context of this patent application, alternatively is referred to as "biotransformation") includes not only processes which involve - besides a number of purely chemical reaction steps - one or more biocatalytic reactions using whole cells of suitable production strains, but also purely biochemical processes using whole cells of suitable production strains. Such purely biochemical processes, respectively, are referred to as fermentations in case the biochemical synthesis starts from a suitable carbon source, or are referred to as precursor fermentations in case the biosynthesis starts from an intermediate product already having a carbon skeleton from which the target molecule to be synthesized can be obtained. The processes may be carried out either under aerobic or under anaerobic conditions.<br>
The biocatalytic reactions in the biochemical synthesis of the present invention can be carried out either in vivo or in vitro. Generally, in vivo processes are processes carried out when using living cells (the term "living cells" thereby also including so-called resting cells); in vitro processes, on the other hand, usually are being carried out using cell lysates or (partly) purified enzymes. The biochemical synthesis according to the present invention is carried out in a microorganism. This can be done using whole cells of suitable production strains, but also may be carried out using permeabilized cells; the differentiation between in vivo and in vitro, however, does not make much sense for processes being carried out with permeabilized cells or immobilized host cells. It will be evident, however, that individual biocatalytic steps from<br>
the process of the invention, when carried out, for instance, by using, immobilized enzymes, etc. are considered equivalent to such steps in the biochemical synthesis as meant in the context of the present application.<br>
Omithine decarboxylases (i.e. enzymes having omithine<br>
decarboxylation activity, or ODCs) are enzymes classified in class EC. 4.1.1.17. The level of activity of an omithine decarboxyiase, if overproduced, can easily be compared with the native (i.e. non-overproduced) level of omithine decarboxyiase activity under standard conditions (at 37 °C in the presence of ornrthine and PLP) within cell free extracts using the Sigma Diagnostics carbon dioxide detection kit (Sigma); assay described in Osterman, A. L. et at. 1994, Biochemistry 33, p. 13662-13667. The skilled man, accordingly, can easily establish whether the ODC used has an increased level of omithine decarboxyiase activity (increases ODC activity) as compared to the native level of the omithine decarboxyiase activity in the microorganism used by determination of the protein content, or by determining the RNA level. Various standard procedures for determination of protein content, for instance colorimetric as well as spectroscopic methods, are described in Lottspeich and Zorbas, Bioanaiytik, Spektrum Akademischer Verlag GmbH, Heidelberg / Berlin, ISBN 3-8274-0041-4 (1998), Chapters 3, 5, 21,22 and 24. Methods for determination of protein level as well as RNA level, for instance northern hybridization, RT-PCR, and many other methods, are described in J. Sambrook, E.F. Fritsch and T. Maniatis, Molecular Cloning, A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory Press, ISBN 0-87969-309-6 (1989). Many other standard procedures, however, are known to the skilled man in this analytic field and do not need to be mentioned here.<br>
Suitable omithine decarboxylases that can be used in the process of the invention, are all enzymes and mutants thereof, that are capable of decarboxylating omithine. Any such enzyme may be used in the process of the invention, at an increased level of activity, i.e. in overproduced form. Such increased level of activity can be achieved by any means known to the skilled man, for instance by means of increasing the gene copy number, or by increasing the endogenous activity or structure of the enzymes by means of mutations, or by using deregulated enzymes. However, and most preferably, it also can be achieved by means of overexpressing an omithine decarboxyiase gene with increased transcriptiona! and/or translational efficiency. In addition it is to be noticed, that the term "increased level of activity" as used herein for any specifically named enzyme activity is also intended to encompass such situations where the activity of such enzyme activity, for instance an omithine decarboxyiase, is<br>
not present at all in the natural source of the microorganism wherein the reaction is taking place, but is introduced therein purposively by genetic modification.<br>
In the process according to the invention, an increased level of activity of N-acetylglutamate formation (as further defined hereinbelow, when the discussion of the pertaining dependent claims is presented) needs to be present as compared to the native level of activity of N-acetylglutamate formation in the microorganism. Comparison of the increased and native activity levels of N-acetylglutamate formation can easily be done, similar to such determination for the ODCs, with appropriate test reactions under standard conditions (assay described in Abadjieva, A., 2001, J. Biol. Chem., 276, p. 42869-42880) within cell free extracts by a radioassay using L-[14C]glutamate and acetyl-CoA as substrates.<br>
1,4-Butanediamine is, according to the present invention, produced in the microorganism with increased ODC and N-acetylglutamate formation activity by biotransformation, and is excreted into the fermentation broth surrounding the microorganism. The 1,4-butanediamine is excreted into and recovered from the fermentation broth.<br>
According to the present invention, thus, an improved biochemical process for the synthesis of 1,4-butanediamine is provided, and the resulting 1,4-butanediamine is excellently suitable as raw material, for instance, for the production of poiyamide-4,6.<br>
The formation of 1,4-butanediamine in the high amounts as produced according to the invention is most surprising, because of the fact that an increased level of ODC activity (together with the increased activity of formation of N-acetylglutamate), without any additional measures being taken for avoiding the negative effects of polyamine formation on the viability of the cells, would be expected to result in death of cells.<br>
As mentioned above, any ODC enzyme may be used, .at an increased level of activity, i.e. in overproduced form, in the process of the invention.<br>
According to the present invention all ornithine decarboxylases can be used that have sufficient, i.e. at least 30%, more preferably at least 45%, and most preferably at least 65% identity with the ODC from the E. coli reference enzyme, and are capable of catalyzing the ornithine decarboxylation reaction. Many ODCs are known having such relatively high level of identity with the E. coli reference enzyme.<br>
Determining identity percentages with reference enzymes can be performed by methods known to the skilled man, for instance by using the protein sequence of the reference enzyme as a "query sequence" to perform a search against<br>
public databases to, for example, identify other family members or related sequences. Such searches can be performed using BLAST programs (version 2.2) using the default parameters of respective program. See http://www.ncbi.nlm.nih.gov.<br>
Preferably, the increased ODC activity is achieved by overexpression of an ornithine decarboxylase encoding gene speF or speC (each belonging to E.G. 4.1.1.17) originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, and Shewanella. The ornithine decarboxylase speF is an inducible ornithine decarboxylase; ornithine decarboxylase speC is a constitutive ornithine decarboxylase.<br>
More preferably, the ornithine decarboxylase encoding gene is originating from one of the species selected from the group consisting of Escherichia co//, Shigella flexneri, Salmonella typhimutium, Yersinia pestis, and Shewanella oneidensis. Until now speC has been investigated in literature much more than speF. Most surprisingly, however, and most preferably, best results according to the present invention are achieved when the ornithine decarboxylase encoding gene is speF, more particularly speF originating from one of the species selected from the group consisting of Escherichia coll, Salmonella typhimutium, and Shewanella oneidensis. When compared to results with overexpression of the constitutive speC ornithine decarboxylase encoding gene, by far the best results according to the present invention indeed are being achieved when using speF.<br>
In the context of the present application, any gene being homologous with any of the abovementioned ornithine decarboxylases and coding for enzymes having ornithine decarboxylase activity sufficiently comparable to the ornithine decarboxylases shown, is suitable in the process of the invention. Such equivalent genes can suitably be obtained by means of any appropriate cloning strategy known to the skilled man, for instance, by the methods described in the experimental part hereof. Alternatively, such equivalent ornithine decarboxylase genes can also be obtained by purposive construction.<br>
The term activity of N-acetylgiutamate formation represents, in the context of the present patent application, any enzyme activity, whether due to a single enzyme or a combination of enzymes, capable of leading to intracellular formation of N-acetylglutamate.<br>
According to the present invention, in particular, all N-acetylglutamate syntheses can be used that have sufficient, i.e. at least 30%, more preferably at least 45%, even more preferably at least 60%, and most preferably at least 75% identity with the N-acetylglutamate synthase from the £. co// reference enzyme, and are capable of<br>
catalyzing the N-acetylglutarnate formation reaction. Many N-acetylglutamate syntheses are known having such relatively high level of identity with the £ coli reference enzyme.<br>
Preferably, increased activity of N-acetylglutamate formation is achieved by overexpression of either an N-acetylglutamate synthase encoding gene argA (belonging to E.G. 2.3.1.1) and/or an N2-acetyl-L-ornithine: L-glutamate N-acetyl transferase encoding gene argJ (belonging to E.G. 2.3.1.35). It will be evident, that any gene coding for an enzyme or mutant thereof having the same functionality of one of the enzymes as mentioned here, will be considered to be equivalent with such enzyme in one of the classes E.G. 2.3.1.1 or 2.3.1.35.<br>
In one of the preferred embodiments of the present invention, the increased activity of N-acetylglutamate formation is achieved by overexpression of an N-acetylglutamate synthase encoding gene argA (belonging to E.G. 2.3.1.1) originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yers/n/a, Photorhabdus, and Buohnera. These ArgA enzymes require the presence of (or supply of) co-enzyme A for being active in their N-acetylglutamate formation.<br>
In this embodiment the increased activity of N-acetylglutamate formation is achieved by overexpression of an N-acetylglutarnate synthase encoding gene argA originating from one of the species selected from the group consisting of Escherichia coli, Shigella tiexneri, Salmonella enterica, Yersnia pestis, Photorhabdus luminescens, and Buchnera aphidicola.<br>
In another preferred embodiment of the present invention, the increased activity of N-acetylglutamate formation is achieved by overexpression of an N2-acetyl-L-ornithine:L-glutamate N-acetyl transferase encoding gene argJ (belonging to E.G. 2.3.1.35) originating from one of the genera selected from the group consisting of Bacillus, Listeria, Oceanobacillus, Staphylococcus, Lactobacillus, Corynebacterium, Mycobacterium, Thermobifida, Streptomyces, and Bifidobacterium.<br>
Suitable N2-acetyl-L-omithine: L-glutamate N-acetyl transferases that can be used in the process according to the invention are<br>
N2-acetyl-L-ornithine:L-glutamate N-acetyl transferases that have sufficient, i.e. at least 20%, more preferably at least 30%, and most preferably at least 40% identity with the N2-acetyl-L-ornithine: L-glutamate N-acetyl transferase from the Bacillus reference species, and are capable of catalyzing the N2-acetyl-L-ornithine:L-glutamate N-acetyl transfer reaction. Many the N2-acetyl-L-ornithine:L-glutamate N-acetyl transferases are known having such level of identity with the corresponding Bacillus reference enzyme.<br>
Contrary to the ArgA enzymes, the ArgJ enzymes do not require any presence of (or supply of) co-enzyme A for being active in their N-acetylglutamate formation. Accordingly, these ArgJ enzymes are clearly preferred above the ArgA enzymes for use in industrial applications.<br>
In this other preferred embodiment of the present invention, the increased activity of N-acetylglutamate formation is achieved by overexpression of an N2-acety!-L-ornithine:L-glutamate N-acetyl transferase encoding gene argJ originating from one of the species selected from the group consisting of Bacillus cereus, Listeria monocytogenes, Oceanobacillus iheyensis, Staphylococcus epidermis, Lactobacillus plantarum, Lactobacillus lactis, Corynebacterium glutamicum, Mycobacterium leprae, Thermobifida fusca, Streptomyces coelicor, and Bifidobacterium bngum.<br>
In the context of the present application, any gene being homologous with any of the abovementioned N-acetylglutamate formation activity and coding for enzymes having N-acetylglutamate formation activity sufficiently comparable to the N-acetylglutamate formation enzymes shown, is suitable in the process of the invention. Such equivalent genes can suitably be obtained by means of any suitable cloning strategy known to the skilled man, for instance, by the methods described in the experimental part hereof. Alternatively, such equivalent N-acetylglutamate formation genes can also be obtained by purposive construction.<br>
In a further preferred embodiment of the present invention, the<br>
process for the biochemical synthesis of 1,4-butanediarnine is carried out in a<br>
microorganism wherein, additionally, also an increased enzyme activity is obtained for<br>
at least two other enzymes by means of overexpression of either<br>
(i)	an arginine decarboxylase encoding gene speA (belonging to E.C.<br>
4.1.1.19) and an agmatinase encoding gene speB (belonging to E.C. 3.5.3.11;<br>
also referred to as agmatine ureahydrolase encoding gene); or<br>
(ii)	an arginine decarboxylase encoding gene spe/A (belonging to E.G.<br>
4.1.1.19), and an agmatine iminohydrolase encoding gene aguA (belonging to<br>
E.G. 3.5.3.12; also referred to as agmatine deiminase encoding gene), and an<br>
N-carbamoylputrescine amidohydrolase encoding gene aguB (belonging to E.C.<br>
3.5.1.53), and optionally also an agmatinase encoding gene speB (belonging to<br>
E.C. 3.5.3.11).<br>
The advantage of this further preferred embodiment is that 1,4-diaminobutane is formed in even higher amounts.<br>
Overexpression as meant herein, can be achieved by any method known to the skilled man; for instance by increasing the translational and/or<br>
transcriptional efficiency of the respective gene, but also by any other known methods such as increasing the gene copy number, or by increasing the endogenous activity or structure of the enzymes by means of mutations, or by using deregulated enzymes. As meant in part (i) of the further preferred embodiment mentioned here above hereof the combination of SpeA and SpeB is intended to represent any functional combination (whether in a combined fusion protein, or as separate enzyme activities) of SpeA and SpeB. In fact, this combination also might be designated as SpeAB. Part (ii) hereof represents, that in such combinations of SpeA and SpeB, the SpeB-part itself may be replaced by any functional combination (whether in a combined fusion protein, or as separate enzyme activities) of AguA and AguB.<br>
Janowte et al, FEBS Letters 544 (2003), 258-261, have described that agmatine deiminase AguA is involved in the arginine decarboxyiase pathway in higher plants. It is further known from Nakada et al., Microbiology, 149 (2003), 707-714, that the conversions catalyzed by SpeB also can be catalyzed by enzymes occurring in plants, namely by the combined action of agmatine deiminase AguA and A/-carbamoyl-putrescine amidohydrolase AguB. Accordingly, instead of, or even in combination with, SpeB in the context of the present invention also AguA and AguB can be used. Sources for such aguA and aguB genes could be Arabidopsis thaliana and Lycopersicon esculentum, but comparable genes can be found in mutants of Pseudomonas aeroginosa.<br>
In the context of the present application, any gene being homologous with any of the abovementioned arginine decarboxylases, respectively agmatinases, or agmatine iminohydrolases or N-carbamoylputrescine amidohydrolases, and coding for such respective enzymes having arginine decarboxyiase (respectively agmatinase, or agmatine iminohydrolase or N-carbamoylputrescine amidohydrolase) activity sufficiently comparable to the respective enzymes - as the case may be - is suitable in this further preferred embodiment of the process of the invention. Such equivalent genes suitably can be obtained by means of any suitable cloning strategy known to the skilled man, for instance, by the methods described in the experimental part hereof. Alternatively, such equivalent genes also can be obtained by purposive construction.<br>
Accordingly, in this preferred embodiment of the process of the present invention, also additional combinations of overexpressed genes are being used, namely genes encoding for (i) arginine decarboxyiase and agmatinase, or (ii) arginine decarboxyiase and agmatine iminohydrolase and N-carbamoylputrescine amidohydrolase, and optionally agmatinase.<br>
In this preferred embodiment of the process according to the present<br>
invention, in particular, all arginine decarboxylases can be used that have sufficient, i.e. at least 30 %, more preferably at least 45% identity, and most preferably at least 65% identity with the arginine decarboxylase from the E. coli reference enzyme, and are capable of catalyzing the arginine decarboxytation reaction. Many arginine decarboxylases are known having such relatively high level of identity with the £. coli reference enzyme.<br>
Moreover, in said embodiment, in particular all agmatinases can be used that have sufficient, i.e. at least 30%, more preferably at least 45%, and most preferably at least 60% identity with the agmatinase from the E. coli reference enzyme, and are capable of catalyzing the agmatinase reaction. Many agmatinases are known having such relatively high level of identity with the £ coli reference enzyme.<br>
Further, in said embodiment, in particular all agmatine<br>
iminohydrolases and/or N-carbamoylputrescine amidohydrolases can be used in the process that have sufficient, i.e. at least 20%, more preferably at least 30%, and most preferably at least 40%, identity with the agmatine iminohydrolase and/or the N-carbamoylputrescine amidohydrolase from the Pseudomonas reference enzymes, and are capable of catalyzing the agmatine iminohydrolase, respectively the N-carbamoylputrescine amidohydrolase reaction. Many agmatine iminohydrolases and N-carbamoylputrescine amidohydrolases are known having such relatively high level of identity with the Pseudomonas reference enzymes.<br>
The overexpressed arginine decarboxylase encoding gene in the above preferred embodiment of the invention is preferably an arginine decarboxylase gene speA originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, Pasteurella, and Neisseria. More preferably, the overexpressed arginine decarboxylase encoding gene is preferably an arginine decarboxylase gene speA originating from one of the species selected from the group consisting of Escherichia coli, Shigella flexneri, Salmonella enterica, Yersinia pestis, Pasteurella multocida, and Neisseria meningitidis.<br>
In this preferred embodiment of the invention, moreover, the overexpressed agmatinase encoding gene preferably is an agmatinase gene speB originating from one of the genera selected from the group consisting of Escherichia, Salmonella, Proteus, Photorhabdus, Vibrio, and Neisseria. More preferably, the overexpressed agmatinase encoding gene is an agmatinase gene speB originating from one of the species selected from the group consisting of Escherichia coli, Salmonella enterica, Proteus mirabilis, Photorhabdus luminescens, Vibrio cholerae, and Neisseria meninaitidis.<br>
In this further preferred embodiment of the invention, further, the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoylputrescine amidohydrolase encoding gene is preferably an agmatine iminohydrolase gene aguA and/or an N-carbamoylputrescine amidohydrolase gene aguB originating from one of the genera selected from the group consisting of Pseudomonas, Streptococcus, Streptomyces, Azotobacter, Arabidopsis, Novosphingobium, and Bacillus. More preferably, the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoylputrescine amidohydrolase encoding gene is an agmatine iminohydrolase gene aguA and/or an N-carbamoylputrescine amidohydrolase gene aguB originating from one of the species selected from the group consisting of Pseudomonas aeruginosa, Streptococcus mutans, Streptomyces avermitiiis, Azotobacter vinelandii, Arabidopsis thaliana, Novosphingobium aromaticivorans, and Bacillus cereus.<br>
The process of the invention may be carried out in any suitable host organism. The hosts may be selected from the groups of production organisms (or cells) generally known to the skilled man in biosynthesis. Such organisms may be from eukaryotic origin, or - as is more preferred - from prokaryotic origin. Eukaryotic cells, for instance, can be cells from plants and fungi, and from various other groups, which other groups collectively are referred to as "Protista".<br>
It is particularly preferred, that the process according to the invention is carried out in a host organism selected from the group consisting of Saccharomyces sp., Bacillus sp., Corynebacterium sp., Escherichia sp. and Pichia sp.<br>
In the process of the invention, it is especially preferred that the microorganism to be used as a host is able to produce the amino acids omithine and/or arginine. For most natural microorganisms this requirement is fulfilled because usually such capability is available in all wild type strains, since arginine represents an essential amino acid.<br>
Of these species, Escherichia sp. are preferred because they are easy to handle by genetic manipulation in order to provide strains with the desired overexpressed enzyme activities. Moreover, Escherichia sp. already in nature contain almost each of the abovementioned enzyme activities (i.e. apart from the agu genes from plants), so that most of the overexpressed genes can be used as homologous genes. Also, Corynebacterium sp. (though lacking a natural ornithine decarboxylase) is particularly preferred because it is a suitable glutamate production strain that can be handled easily in fermentation processes.<br>
In the process of the present invention glutamate is a very suitable<br>
precursor. Accordingly, the process is preferably being carried out in a host strain capable of formation of glutamate (for instance, Corynebacterium glutamicum).<br>
Best results are being achieved when the process according to the invention is carried out in a host organism from the group consisting of Saccharomyces cerevisiae, Corynebacterium sp. and Escherichia sp. wherein, apart from the increased level of activity of an ornithine decarboxylase and of N-acetyl giutamate formation, at least also the level of activity of an arginine decarboxylase in combination with an agmatinase and/or an agmatine iminohydrolase and an N-carbamoylputrescine amidohydrolase enzyme is increased. As meant herein, for each of the enzymes mentioned the increased level of activity is compared with the native level of activity of the respective said enzyme activity in the host organism.<br>
It will be clear that the process of the invention is preferably carried out under reaction conditions that are also usual as fermentation conditions. The process, therefore can be carried out batch-wise, but also - if so desired - fed-batch. It may be convenient to ensure that the organism used as host organism has, or is provided with, a suitable exporter system for the 1,4-diaminobutane formed: Preferably such exporter system is a native one.<br>
The present invention, of course, finally also relates to all vectors, plasmids and hosts carrying an increased level of an ornithine decarboxylase activity (increased ODC activity) as compared to the native level of the ornithine decarboxylase activity, and an increased activity of N-acetylglutamate formation as compared to the native level of activity of N-acetylglutamate formation. In particular, for the preferred embodiments, the present invention also relates to all vectors, plasmids and hosts additionally carrying an increased level of activity of one or more of the other aforementioned enzyme activities according to the attached claims.<br>
The invention will now be elucidated by means of some experimental results, which by no means are intended to limit the scope of the invention.<br>
EXPERIMENTAL PART<br>
General procedures<br>
Standard procedures were applied for all DNA manipulations (Sambrook, J. ef a/. (1989), Molecular cloning: a laboratory manual, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York). DNA was amplified from chromosomal DNA of £ coli LJ110 (Zeppenfeld, et al. (2000), J Bactehol. 182, 4443-4452), Bacillus subtilis  ATCC10783, or Corynebacterium glutamicum<br>
ATCC13032 if not indicated otherwise. PCR amplification was performed using the proof-reading enzymes SAWADYPwo-DNA-Polymerase (Peqlab Biotechnologie GmbH, Erlangen, Germany) or Platinum Pfx DNA Polymerase (Invitrogen, Karlsruhe, Germany) following the manufacture's protocol, whereas the verification of the constructed strains was carried out by colony PCR utilizing the Taq polymerase READYMIX (Sigma, Taufkirchen, Germany). Restriction sites for subsequent cloning as well as further mutations were introduced with oligonucleotides purchased from MWG-Biotech (Ebersberg, Germany). DNA fragments were purified with the MinElute Gel Extraction Kit (Qiagen, Hilden, Germany) following the manufacture's protocol. Preparation of plasmid DNA was accomplished by the utilization of QIAprep spin Miniprep Kit (Qiagen, Hilden, Germany). Verification of the constructed plasmids was carried out by restriction analysis and subsequent sequencing (Agowa, Berlin, Germany).<br>
For high-level expression of genes, the vector pJF119EH (Furste, J. P. et a/. (1986), Gene 48,119-131) was used suitable for IPTG-induced protein production based on the Isopropyl-beta-D-thiogalactopyranoside (IPTG) inducible tac promotor and the lac represser system (lacIQ).<br>
Construction of plasmids<br>
(i)     Construction of the plasmid pDAB2 (pJF119EH-speF)<br>
The (inducible, biodegradative) orn'rthine decarboxyiase encoding gene speF of £ co// LJ110 (Zeppenfeld, et a/., see general procedures) was cloned into the expression vector pJF119EH (Furste, J. P. et a/. (1986), Gene 48,119-131). This vector allows a high-level protein production based on the transcriptional control of cloned genes under the isopropyl-p-D-thiogalactopyranoside (IPTG) inducible tac promotor and the lac represser system (laclQ). For construction of the expression piasmid pDAB2 (pJF119EH-speF) the coding gene was cloned with original RBS (ribosomal binding site), start and stop codon.<br>
The 2247 bp speF-containing DNA fragment was amplified from chromosomal DNA of E. co//LJ110 (accession number AE000172; nucleotides 10242 -12468) using the following oligonucleotides:<br>
5'-GAC CTG CTG 6TA CCT AAA ATA AAG AGA TGA AA-3'        [SEQ ID: No.1]<br>
(mutations in bold, Kpnl restriction site in italics)<br>
and<br>
5'-TCG ATC TAG ACT GAG TCA TAA TTT TTC CCC-31	[SEQ ID: No.2]<br>
(mutations in bold, Xbal restriction site in italics).<br>
The fragment was terminally modified with the restriction<br>
endonucleases Kpnl and Xbal and ligated into the expression vector pJF119EH, which was cut in the same manner. After transformation in £ coll DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the obtained plasmid pDAB2 (pJF119EH-speF, 7502 bp) was verified by restriction analysis and subsequent sequencing.<br>
(it)     Construction of the plasmid pDAB4 (pJF119EH-speC)<br>
The (constitutive, biosynthetic) ornithine decarboxylase encoding gene speC of £ coli LJ110 (Zeppenfeld, et al., see general procedures) was cloned into the expression vector pJF119EH (Furste, J. P. et al. (1986), see (i)), allowing a strong gene expression based on the transcriptional control under the isopropyl-(3-D-thiogalactopyranoside (IPTG) inducible tac promoter and the lac represser system (laclQ). Therefore, the coding gene was cloned with original start and stop codon. Since, no conserved ribosomal binding site (RBS) could be determined for speC utilizing in silico studies, an optimized RBS was introduced 7 bp upstream of the speC start codon by mutagenesis.<br>
The 2235 bp speC-containing DNA fragment was amplified from chromosomal DNA of E. coli LJ110 (accession number AE000379; nucleotides 2650 -4867) using the following oligonucleotides:<br>
5-GAG CTC TAG ACC ACTTTG AGG AAT ATC T-3'	[SEQ ID: No.3]<br>
(mutations in bold, Xbal restriction site in italics) and<br>
5'-TTT TGC ATG CTT ACT TCA ACA CAT AAC CGT AC-3'	[SEQ ID: No.4]<br>
(mutations in bold, Sph\ restriction site in italics).<br>
After terminal modification with the endonucleases XJbal and Sprtl, the PCR product was ligated into plasmid pJF119EH, which was cut in the same manner. After transformation in £ coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDAB4 (pJF119EH-speC, 7491 bp) was carried out by restriction analysis and subsequent sequencing.<br>
Oii)    Construction of the plasmid pDAB1 (pJF119EH-argX\)<br>
The N-acetylglutamate synthase encoding gene argA of E coli LJ110<br>
(Zeppenfeld, etal., see general procedures) was cloned into the expression vector pJF119EH (FOrste, J. P. ef a/. (1986), see (i)). The gene was cloned with original RBS (ribosomal binding site) and stop codon. However, the translation start was altered from GTG to ATG.<br>
The 1365 bp argA-coding DNA fragment was amplified from chromosomal DNA of E. coli LJ110 (accession number AE000365; nucleotides 3312 -4643) using the following oligonucleotides:<br>
5-ATA AGA ATTCAA AGA GGTGTG CCA TGG TAA AG-3'        [SEQ ID: No.5] (mutations in bold, EcoRI restriction site in italics) and<br>
5'-TTT TGG TAC CTT ACC CTA AAT CCG CCA T-31	[SEQ ID: No.6]<br>
(mutations in bold, Kpnl restriction site in italics).<br>
The fragment was terminally modified using the restriction<br>
endonucleases EcoRl and Kpnl, and subsequently ligated into the expression plasmid pJF119EH, which was cut in the same manner. After transformation in E. coll DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l arnpicillin. After preparation, the verification of the obtained plasmid pDAB1 (pJF119EH-a/pA 6627 bp) was carried out by restriction analysis and subsequent sequencing.<br>
(iv)    Construction of the plasmid pDABS (pJF11 QB-\-argA-speF)<br>
In order to allow in-parallel production of the ornithine decarboxylase SpeF and the N-acetylglutamate synthase ArgA, the speF encoding gene of E. coli LJ110 (Zeppenfeld, et a/., see general procedures) was cloned into the argA-expression vector pDAB1 (see (iii)).<br>
The speF-containing DNA fragment (2225 bp) was cut out from the constructed plasmid pDAB2 (see A.1) by digestion with the endonucleases Kpnl and Xbal and ligated into the argA containing plasmid pDAB1 (see (iii)), cut likewise. After transformation in E. coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the obtained plasmid pDABS (pJF119EH-arg,4-speF, 8841 bp) for in-parallel SpeF and ArgA production was verified by restriction analysis.<br>
(v)    Construction of the plasmid pDAB6 (pJF119EH-arg/4-speC)<br>
In order to allow in-parallel production of the ornithine decarboxylase<br>
SpeC and the N-acetylglutamate synthase ArgA, the speC encoding gene of Ecoli LJ110 (Zeppenfeld, et a/., see general procedures) was cloned into the argA-expression vector pDAB1 (see (iii)1).<br>
By digestion of the constructed plasmid pDAB4 (see (ii)) with the endonucleotides Xba\ and Sphl, the 2225 bp DNA fragment containing the speC gene with optimized RBS was separated. Subsequently, the fragment was ligated into the arg/4-containing plasmid pDAB1 (see (i)), which was cut in the same manner. After transformation in £ co//DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDABS (pJF119EH-arg/4-speC, 8830 bp) allowing the in-parallel expression of speC and argA was carried out by restriction analysis and subsequent sequencing.<br>
(vi)    Construction of the plasmid pDAB7 (pJF119EH-speAB)<br>
The arginine decarboxylase encoding gene speA as well as speB coding for the agmatinase of E coli LJ110 (Zeppenfeld, et a/., see general procedures) were cloned into the expression vector pJF119EH (FQrste, J. P. et a/. (1986), see (i)). This way, the original operon structure of the genes as well as RBS, start and stop codon were maintained.<br>
The 3079 bp .speAB-containing DNA fragment was amplified from chromosomal DNA of £. coli LJ110 (accession number AE000377; nucleotides 1190 -4247) using the following oiigonucleotides:<br>
5-ACA CTT TCTAGA ATA ATTTGA GGT TCG CTA TG-3'	[SEQ ID: No.7]<br>
(mutations in bold, Xbal restriction site in italics) and<br>
5'-CAT GGC ATG CGG TGC TTA CTC G-3'	[SEQ ID: No.8]<br>
(mutations in bold, Sph1 restriction site in italics).<br>
After terminal modification with the restriction endonucleases Xbal and Sphl, the DNA fragment was ligated into the expression plasmid pJF119EH, which was cut likewise. After transformation in E. coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDAB7 (pJF119EH-spe/4B, 8339 bp) was carried out by restriction analysis and subsequent sequencing.<br>
(vii)   Construction of the plasmid pDABB (pJF119EH-speF-spe/AS)<br>
in order to allow in-parallel production of the ornithine decarboxylase SpeF, the arginine decarboxylase SpeA and the agmatinase SpeB, the speAB genes of £ co// LJ110 (Zeppenfeld, et a/., see general procedures) were cloned into the speF-expression vector pDAB2 (see (i)).<br>
By digestion of the plasmid pDAB7 (see (vi)) with the restriction endonucleases Xbal and Sph\, the 3067 bp comprising speAB gene-operon was separated and ligated into the speF containing plasmid pDAB2 (see (i)), which was cut in the same manner. After transformation in £. coll DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the obtained plasmid pDAB8(pJH119EH-speFAB, 10547 bp) allowing the in-parallel production of SpeFAB was verified by restriction analysis.<br>
(viii) Construction of the plasmid pDAB10 (pJF11 QEH-argA-speF-speAB)<br>
In order to allow in-parallel production of the ornithine decarboxyiase SpeF, the arginine decarboxylase SpeA, the agmatinase SpeB and the N-acetylglutamate synthase ArgA, the speAB genes of £. coll LJ110 (Zeppenfeld, et ai, see general procedures) were cloned into the argA-speF-expression vector pDABS (see (iv)).<br>
By digestion of the plasmid pDAB7 (see (vi)) with the restriction endonucleases Xba\ and Sphl, the 3067 bp comprising speAB gene-operon was separated and ligated into the argA-speF containing plasmid pDABS (see (iv)), which was cut in the same manner. After transformation in £. co// DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the obtained plasmid pDABIO (pJH119EH-argA-speFAB, 11886 bp) allowing the in-parallel production of ArgA and SpeFAB was verified by restriction analysis.<br>
(ix)    Construction of the plasmid pDAB37 (pJF119EH-argJBs-speF)<br>
In order to allow in-parallel production of the ornithine decarboxylase SpeF and the N2-acetyl-L-ornithine:L-glutamate N-acetyl transferase encoding gene ArgJ from Bacillus subtilis ATCC 10783, the argJ encoding gene of B. subtilis was cloned into the speF-expression vector pDABS (see (iv)) by replacing the present argA gene. The gene was cloned with original RBS and stop codon.<br>
The 1279 bp argJ-containing DNA fragment was amplified from<br>
chromosomal DNA of Bacillus subtilis ATCC 10783 (accession number Z99109<br>
(fi. subtilis subsp. subtilis str. 168, complete genome (section 6 of 21)); nucleotides<br>
184321 -185600) using the following oiigonucleotides:<br>
5'-TCA CGC GM TTC ATC CAT AGA ACG GGA GAG-3'	[SEQ ID: No.9]<br>
(mutations in bold, EcoRi restriction site in italics)<br>
and<br>
5-CTT CAT TTC GGTACC CTT TAT TAC GTG CGA TAG CTC-3'     [SEQ ID: No. 10]<br>
(mutations in bold, Kpn\ restriction site in italics). The oiigonucleotides were<br>
constructed according to the sequence of argj as present in the genome of the strain<br>
B. subtilis subsp. subtilis str. 168.<br>
The amplified DNA-fragment and plasmid pDABS were restricted with the endonucleases EcoRl and Kpnl. In case of pDAB5, two fragments of 1355 bp and 7486 bp were obtained. The 7486 bp fragment comprising vector pJF119EH and the gene speF was isolated and iigated with amplified DNA-fragment. After transformation in £ coll DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDAB37 (pJF119EH-a^(JBs-speC, 8749 bp) was carried out by restriction analysis and subsequent sequencing. Sequence analyses revealed, that the argJ gene cloned from B. subtilis ATCC 10783 differs from the argj gene reported for B. subtilis subsp. subtilis str. 168. In comparison to protein ArgJ of B. subtilis subsp. subtilis str. 168 (accession number CAB12961) the protein ArgJ encoded from the argJ gene present on plasmid pDAB37 shows the following exchanges: H72Q, P74A, T75A, L95I, F105L, G110D, H134Q, E142Q, A169T, R1B1A, T216I, A242G, D255E, N353H, I363L, A380D, D383E.<br>
(x)    Construction of the plasmid pDAB38 (pJF119EH-argJCg-speF)<br>
In order to allow in-parallel production of the omithine decarboxylase<br>
SpeF and the N2-acetyl-L-ornithine:L-glutamate N-acetyl transferase encoding gene<br>
ArgJ from Corynebacterium glutamicum, the argJ encoding gene of C. glutamicum<br>
ATCC 13032 was cloned into the speF-expression vector pDABS (see (iv)) by replacing<br>
the present argA gene. The gene was cloned with original RBS and stop codon.<br>
The 1219 bp argj-containing DNA fragment was amplified from<br>
chromosomal DNA of C. glutamicum ATCC 13032 (accession number NC 006958 (C.<br>
glutamioum ATCC 13032, complete genome); nucleotides 1466650-1467869) using<br>
the following oiigonucleotides:<br>
5'-ACA CAT CGA ATT GAG TAG GAG TTC CAC ATG G-3'	[SEQ ID: No. 11]<br>
(mutations in bold, EcoRl restriction site in italics)<br>
and<br>
5'-AGT GCT GGT ACC TTT TM GAG CTG TAG GC 3'       [SEQ ID: No.12]<br>
(mutations in bold, Kpn\ restriction site in italics).<br>
The amplified DNA-fragment and plasmid pDABS were restricted with the endonucleases EcoRl and Kpnl. In case of pDABS, two fragments of 1355 bp and 7486 bp were obtained. The 7486 bp fragment comprising vector pJF119EH and the gene speF was isolated and ligated with the amplified DNA-fragment. After transformation in£ coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDABSB (pJF119EH-aroJCg-speF, 8679 bp) was carried out by restriction analysis and subsequent sequencing.<br>
(xi)    Construction of the plasmid pDABS (pJF119EH-speCnrbs)<br>
The (constitutive, biosynthetic) ornithine decarboxylase encoding gene speC of £ coli LJ110 (Zeppenfeld, at a/., see general procedures) was cloned into the expression vector pJF119EH (Furste, J. P. etal. (1986), Gene 48, 119-131), allowing a strong gene expression based on the transcriptional control under the isopropyl-p-D-thiogalactopyranoside (IPTG) inducible tac promotor and the lac represser system (laclQ). Therefore, the coding gene speC was cloned with original RBS, start and stop codon.<br>
The 2235 bp speCnRss -containing DNA fragment was amplified from chromosomal DNA of £ coll LJ11Q (accession number AE000379; nucleotides 2650 -4867) using the following oligonucleotides:<br>
5'-GAG CTC TAG ACC AGT TTG ACC CAT ATC T-3'	[SEQ ID: No. 13]<br>
(mutations in bold, Xbal restriction site in italics) and<br>
5'-TTT TGC ATG CTT ACT TCA ACA CAT AAC CGT AC-3'	[SEQ ID: No. 14]<br>
(mutations in bold, Sp/il restriction site in italics).<br>
After terminal modification with the endonucleases Xba\ and Sph\, the PCR product was ligated into plasmid pJF119EH, which was cut in the same manner. After transformation in E. coli DH5 cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDAB3 (pJF119EH-speCnRbs, 7491 bp) was carried out by restriction analysis and subsequent sequencing.<br>
Comparative Experiment A:<br>
Production of 1,4-butanediamine via the sole overexpression of ornithine<br>
decarboxylase SQeC, with gene expression induced by IPTG (shake flask)<br>
The influence of overexpression of ornithine decarboxylase encoding gene speC on DAB production was investigated within the £ coll host strain LJ110 (Zeppenfeld, ef al., see general procedures) carrying the plasmid pDABS (see (ix)).<br>
This strain was tested in shake flask experiments utilizing minimal salt medium consisting of MgS047H20 (300 mg/l), CaCI2-2H2O (15 mg/l), KH2PO4 (3 g/l), K2HP04 (12 g/l), NaCl (100 mg/l), (NH4)2S04 (5 g/l), Na citrate-3H20 (1 g/l), FeS047H20 (75 mg/l), thiamine-HCI (vitamin B1) (5 mg/l) as well as the trace elements AI2(S04)3-18H20 (3 mg/l), CoCI2-6H20 (1.05 mg/l), CuSO4-5H20 (3.75 mg/l), H3BO3 (0.75 mg/l), MnCI2-4H20 (30 mg/l), Na2MoO4-2H2O (4.5 mg/l), NiSO4-6H2O (3 mg/l) and ZnS047H2O (22.5 mg/l). A stock solution of glucose (500 g/l) was autoclaved separately and added to the sterilized medium up to a final concentration of 10 g/l.<br>
A preculture of minimal salt medium containing 100 mg/l ampicilline was inoculated with 1 - 5 pi/ml stock solution and incubated at 33 °C, 180 rpm for 16 h up to an OD620 of 2. 5 ml of this culture was subsequently used for inoculation of the main culture consisting of 50 ml of the same medium, which was incubated for 24 h at 33 °C and 180 rpm. Since the cells reached an OD62onni of 1.5 (after ~ 7 h), gene expression was induced by the addition of 10 uM IPTG.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, diluted supernatant was analyzed by HPLC. Here, the contained amines were detected as orfbo-phthaldialdehyde (OPA) derivatives at 230 nm on a Hewlett-Packard 1100 Series instrument, using a C1B-reverse phase column (.Nucleosil 120-5 C18, Macherey &amp; Nagel, Duren, Germany) equilibrated to 50% buffer B (buffer A, 0.1 M sodium acetate pH 7.2; buffer B methano!). For separation, the following gradient was applied: 1 - 7 min linear gradient from 50% to 75% buffer B with a flow rate of 0.5 ml/min, 7 -13 min 75% to 85% buffer B with a flow rate of 0.5 ml/min, 13-14.5 min 85% to 50% buffer B with a low rate of 1 ml/min, 14.5-17 min 50% buffer B with a flow rate of 1 ml/min and 17 - 20 min at 50% buffer B with a flow rate of 0.5 ml/min.<br>
By the utilization of standard substances for calibration, the following DAB concentrations could be determined (see Table 1) and verified by NMR spectroscopy.<br>
Strain used	Gene       DAB concentration<br>
expressed	[mg/l]<br>
LJ110pDAB3        speC	50<br>
Table 1: DAB formation utilizing ODC overproduction in £. coli<br>
Examples: Improvement of 1,4-butanediamine production by increased ornithine decarboxvlation activity combined with increased N-acetvlalutamate formation activity<br>
Example 1.         Production of 1,4-butanediamine utilizing ODC as well as ArgA overproduction (shake flask)<br>
The influence of the in-parallel production of N-acetylglutamate synthase ArgA, catalyzing the first step in ornithine biosynthesis starting from glutamate, and ornithine decarboxylases SpeF or SpeC on DAB production was investigated within the £ coll host strain LJ110 (Zeppenfeld, ef a/., see general procedures) carrying the plasmid pDAB5 (see (iv)) or pDAB6 (see (v)).<br>
These strains were tested in shake flask experiments in minimal salt medium (see Comparative Experiment A). Therefore, a preculture of minimal salt medium containing 100 mg/l ampicilline was inoculated with 1 - 5 ul/ml stock solution and incubated at 33 °C, 180 rpm for 16 h up to an optical density at 620 nm of 2. 5 ml of this culture was subsequently used for inoculation of the main culture consisting of 50 ml of the same medium, which was incubated for 24 h at 33 °C and 180 rpm. Since the cells reached an OD620nm of 1 -5 (after ~ 7 h), gene expression was induced by the addition of 10 uM IPTG.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, the supernatant was analyzed by HPLC (see Comparative Experiment A). By the utilization of standard substances for calibration, the following DAB concentrations could be determined (see Table 2).<br>
Used strain        expressed        DAB concentration<br>
genes	[mg/l]   '<br>
LJ.110pDAB5      argA speF	893<br>
LJ110 pDAB6      argA speC	1064<br>
Table 2: DAB formation utilizing in-parallel ArgA and ODC overproduction in E. coli<br>
Example 2.         Production of 1,4-butanediamine utilizing QDC as well as ArgJ overproduction (shake flask)<br>
The influence of the in-parallel production of the<br>
N2-acetyl~L-ornitriine:L-glutamate N-acetyl transferase encoding gene ArgJ either from C. glutamicum or 6. subtilis, catalyzing the formation of N-acetylglutamate and ornithine from L-glutamate and N-acetylornithine, and omtthine decarboxylases SpeF or SpeC on DAB production was investigated within the £. coll host strain LJ110 (Zeppenfeld, et a/., see general procedures) carrying the plasmid pDAB37 (see (ix)) or pDAB38 (see (x)).<br>
These strains were tested in shake flask experiments in minimal salt medium (see Comparative Experiment A). Therefore, a preculture of minimal salt medium containing 100 mg/l ampicilline was inoculated with 1 - 5 pi/ml stock solution and incubated at 33 °C, 180 rpm for 16 h up to an optical density at 620 nm of 2. 5 ml of this culture was subsequently used for inoculation of the main culture consisting of 50 ml of the same medium, which was incubated for 24 h at 33 °C and 180 rpm. Since the cells reached an OD620nm of 1.5 (after ~ 7 h), gene expression was induced by the addition of 50 uM IPTG.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, the supernatant was analyzed by NMR. A sample of the culture supernatant was adjusted to pH 5.8, lyophilized, and redissolved in D2O. 600 MHz 1H-NMR at 323 K showed the expected resonance spectrum and spiking with a small amount of DAB confirmed the presence of DAB. By the utilization of standard substances for calibration, the following DAB concentrations could be determined (see (Table Remove)from ornithine as welt as arginine, the influence of combined overproduction of the ornithine decarboxylase SpeF, the arginine decarboxylase SpeA and the agmatinase SpeB was investigated. In addition, in order to ensure a proper precursor supply, these investigations were combined in a further experiment with the overproduction of the N-acetylglutamate synthase ArgA, catalyzing the first step in ornithine biosynthesis starting from giutamate.<br>
Therefore, shake flask cultivations were carried out in minimal salt medium (see A.3) by the utilization of the £ co//host strain LJ110 (Zeppenfeld, et al, see general procedures) carrying the plasmids pDABB and pDABIO, respectively (see 2.2 and 2.3). Therefore, a preculture of minimal salt medium containing 100 mg/l ampicilline was inoculated with 1 - 5 pi/ml stock solution and incubated at 33 °C, 180 rpm for 16 h up to an optical density at 620 nm of 2. 5 ml of this culture was subsequently used for inoculation of the main culture consisting of 50 ml of the same medium, which was incubated for 24 h at 33 °C and 180 rprn. Since the cells reached an ODEaonm of 1.5 (after ~ 7 h), gene expression was induced by the addition of 10 IPTG.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, the supernatant was analyzed by HPLC (see Comparative Experiment A). By the utilization of standard substances for calibration, the following DAB concentrations could be determined (see Table 4).<br>
Used strain	expressed	DAB concentration<br>
gene	[rng/l]<br>
LJ110pDAB8	speFAB	1025<br>
LJ110pDAB10      argA speFAB                1433<br>
Table 4: DAB formation starting from ornithine as well as arginine in E. co//<br><br><br>
CLAIMS<br>
1.	Process for biochemical synthesis of 1,4-butanediamine in a microorganism<br>
having an increased level of an ornithine decarboxylase activity (increased<br>
ODC activity) as compared to the native level of the ornithine decarboxylase<br>
activity,<br>
characterized in that<br>
in the microorganism also an increased activity of N-acetyiglutamate formation is present as compared to the native level of activity of N-acetylglutamate formation in the microorganism and that 1,4-butanediamine produced in the microorganism is excreted into a fermentation broth, and is recovered from the fermentation broth.<br>
2.	Process according to claim 1,<br>
wherein<br>
the increased ODC activity is achieved by overexpression of an ornithine decarboxylase encoding gene speFar speC (each belonging to E.C. 4.1.1.17) originating from one of the genera selected from the group consisting of Esch&amp;richia, Shigella, Salmonella, Yersinia, and Shewanella.<br>
3.	Process according to claim 2,<br>
wherein<br>
the ornithine decarboxylase encoding gene is originating from one of the species selected from the group consisting of Escherichia coll, Shigella flexneri, Salmonella typhimutium, Yersinia pestis, and Shewanella oneidensis. A.         Process according to any of claims 1 to 3, wherein<br>
the ornithine decarboxylase encoding gene is speF originating from one of the species selected from the group consisting of Escherichia coli, Salmonella typhimutium,and Shewanella oneidensis.<br>
5.	Process according to any of claims 1 to 4,<br>
wherein<br>
the increased activity of N-acetylglutamate formation is achieved by overexpression of either an N-acetylglutamate synthase encoding gene argA (belonging to E.C. 2.3.1.1) and/or an N2-acetyl-L-ornithine:L-glutamate N-acetyl transferase encoding gene argj (belonging to E.C. 2.3.1.35).<br>
6.	Process according to claim 5,<br>
wherein<br>
the increased activity of N-acetylglutamate formation is achieved by overexpression of an N-acetylglutamate synthase encoding gene argA originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, Photorhabdus, and Buchnera.<br>
7.	Process according to claim 6,<br>
wherein<br>
the increased activity of N-acetylglutamate formation is achieved by overexpression of an N-acetylglutamate synthase encoding gene argA originating from one of the species selected from the group consisting of Escherichia coli, Shigella flexneri, Salmonella enterica, Yersinia pesf/s, Photorhabdus luminescens, and Buchnera aphidicola.<br>
8.	Process according to claim 5,<br>
wherein<br>
the increased activity of N-acetylglutamate formation is achieved by overexpression of an N2-acetyl-L-orntthine:L-giutamate N-acetyl transferase encoding gene argJ (belonging to E.G. 2.3.1.35) originating from one of the genera selected from the group consisting of Bacillus, Listeria, Oceanobacillus, Staphylococcus, Lactobacillus, Corynebacterium, Mycobacterium, Thermobifida, Streptomyces, and Bifidobacterium.<br>
9.	Process according to claim 8,<br>
wherein<br>
the increased activity of N-acetylglutamate formation is achieved by overexpression of an N2-acetyl-L-omithine:L-glutamate N-acetyl transferase encoding gene originating from one of the species selected from the group consisting of Bacillus cereus, Listeria monocytogenes, Oceanobacillus iheyensis, Staphylococcus epidermis, Lactobacillus plantarum, Lactobacillus lactis, Corynebacterium glutamicum, Mycobacterium leprae, ThermobiFida fusca, Streptomyces coelbor, ar\6-Bifidobacterium longum.<br>
10.	Process according to any of claims 1 to 9,<br>
wherein<br>
additionally also an increased enzyme activity is obtained for at least two other<br>
enzymes by means of overexpression of either<br>
(i) an arginine decarboxylase encoding gene spe/A (belonging to E.G. 4.1.1.19)<br>
and an agmatinase encoding gene speB (belonging to E.G. 3.5.3.11; also<br>
referred to as agmatine ureahydrolase encoding gene); or<br>
(ii) an arginine decarboxylase encoding gene speA (belonging to E.G.<br>
4.1.1.19), and an agmatine iminohydrolase encoding gene aguA (belonging to E.C. 3.5.3.12; also referred to as agmatine deiminase encoding gene), and an N-carbamoylputrescine amidohydrolase encoding gene aguB (belonging to E.C. 3.5.1.53), and optionally also an agmatinase encoding gene speB (belonging to E.C. 3.5.3.11).<br>
11.	Process according to claim 10,<br>
wherein<br>
the overexpressed arginine decarboxylase encoding gene is an arginine decarboxylase gene speA originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, Pasteurella, and Neisseria.<br>
12.	Process according to claim 11,<br>
wherein<br>
the overexpressed arginine decarboxylase encoding gene is an arginine decarboxylase gene speA originating from one of the species selected from the group consisting of Escherichia coll, Shigella flexneri, Salmonella enterica, Yersinia pestis, Pasteurella multocida, and Neisseria meningitidis.<br>
13.	Process according to claim 10,<br>
wherein<br>
the overexpressed agmatinase encoding gene is an agmatinase gene speB originating from one of the genera selected from the group consisting of Escherichia, Salmonella, Proteus, Photorhabdus, Vibrio, and Neisseria.<br>
14.	Process according to claim 13,<br>
wherein<br>
the overexpressed agmatinase encoding gene is an agmatinase gene speB originating from one of the species selected from the group consisting of Escherichia coll, Salmonella enterica, Proteus mirabilis, Photorhabdus luminesoens, Vibrio cholerae, and Neisseria meningitidis.<br>
15.	Process according to claim 10,<br>
wherein<br>
the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoylputrescine amidohydrolase encoding gene is an agmatine iminohydrolase gene aguA and/or an N-carbamoylputrescine amidohydrolase gene aguB originating from one of the genera selected from the group consisting of Pseudomonas, Streptococcus, Streptomyces, Azotobacter, Arabidopsis, Novosphingobium, and Bacillus.<br>
16.	Process according to claim 15,<br>
wherein<br>
the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoylputrescine amidohydrolase encoding gene is an agmatine iminohydrolase gene aguA and/or an N-carbamoylputrescine amidohydrolase gene aguB originating from one of the species selected from the group consisting of Pseudomonas aeruginosa, Streptococcus mutans, Streptomyces avermitilis, Azotobacter vinelandii, Arabidopsis thaiiana, Novosphingobium aromaticivorans, and Bacillus cereus.<br>
17.	Process according to any of claims 1 to 16,<br>
wherein<br>
the process is carried out in a host organism selected from the group consisting of Saccharomyces sp., Bacillus sp., Corynebacterium sp., Escherichia sp. and Pichie sp.<br>
18.	Process according to any of claims 1 to 17,<br>
wherein<br>
the process is carried out in a host organism selected from the group of Saccharomyces cerevisiae, Corynebacterium sp. and Escherichia sp. and in that, apart from the increased level of activity of an ornithine decarboxylase and of N-acetylglutamate formation, at least also the level of activity of an arginine decarboxylase in combination with an agmatinase and/or an agmatine iminohydrolase and an N-carbamoylputrescine amidohydrolase is increased.<br>
Vectors, plasmids and hosts carrying, an increased level of an ornithine<br>
decarboxylase activity (increased ODC activity) as compared to the native<br>
level of the omrthine decarboxylase activity according to any of claims 1-4,<br>
and an increased activity of N-acetylglutamate formation as compared to the<br>
native level of activity of N-acetylglutamate formation according to any of<br>
claims 1 and 5-9.<br>
Vectors, plasmids and hosts according to claim 19, and further carrying an<br>
increased level of activity of one or more of the further enzyme activities<br>
according to any of claims 10-18.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctQWJzdHJhY3QtKDI1LTAyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Abstract-(25-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">479-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctQ2xhaW1zLSgyNS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Claims-(25-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">479-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyMC0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Correspondence Others-(20-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyMi0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Correspondence Others-(22-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyNS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Correspondence Others-(25-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LURFTE5QLTIwMDctQ29ycmVzcG9uZGVuY2UtT3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">479-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">479-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctRm9ybS0xOC0oMjEtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Form-18-(21-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">479-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctRm9ybS0zLSgyMi0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-Form-3-(22-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LURFTE5QLTIwMDctRm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">479-DELNP-2007-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">479-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctR1BBLSgyMC0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-GPA-(20-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctR1BBLSgyNS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-GPA-(25-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">479-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc5LWRlbG5wLTIwMDctcGN0LW5vdGlmaWNhdGlhbi5wZGY=" target="_blank" style="word-wrap:break-word;">479-delnp-2007-pct-notificatian.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260272-a-differential-pressure-regulator-for-an-automotive-fuel-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260274-fat-based-confectionery-product.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260273</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>479/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>16/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>18-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jan-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, NL - 6411 TE HEERLEN, THE NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NOSSIN, PETRUS MARTINUS MATHEUS</td>
											<td>SCHOOR 11, NL,-6031 SC NEDERWEERT, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RAEVEN, LEON JEAN RENIER MARIE</td>
											<td>LANGE VOER 25, NL-6132 HS SITTARD, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KREMER, SUSANNE MARIA</td>
											<td>EDERENERSTRASSE 32, 52441 LINNICH-GEREONSWEILER, GERMANY.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>EPPELMANN, KATRIN</td>
											<td>4461 HEMMINGWAY DRIVE, KALAMAZOO, MICHIGAN 49009, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 13/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/007606</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-07-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04077046.3</td>
									<td>2004-07-15</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260273-process-for-biochemical-synthesis-of-1-4-butanediamine by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:21:10 GMT -->
</html>
