<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260246-biochemical-synthesis-of-1-4-butanediamine by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:12:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260246:BIOCHEMICAL SYNTHESIS OF 1,4-BUTANEDIAMINE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">BIOCHEMICAL SYNTHESIS OF 1,4-BUTANEDIAMINE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a process for biochemical synthesis of 1,4-butanediamine in a microorganism having an increased level of an ornithine decarboxylase activity (increased ODC activity) as compared to the native level of the ornithine decarboxylase activity, wherein the increased ODC activity is obtained by means of overexpression of an ornithine decarboxylase encoding gene with increased translational and/or transcriptional efficiency, and wherein 1,4-butanediamine produced in the microorganism is excreted into a fermentation broth, and is recovered from the fermentation broth. In preferred embodiments also increased enzyme activity is obtained by of overexpression of either (i) an arginine decarboxylase encoding gene speA and an agmatinase encoding genespeB; or (ii) an arginine decarboxylase encoding gene speA and an agmatine iminohydrolase encoding gene aguA.and an N-carbamoylputrescine amidohydrolase encoding geneaguB, and optionally also an agmatinase encoding gene speB. The invention also relates to vectors, plasmids and hosts carrying, at an increased level of activity, one or more of the enzyme activities as mentioned.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BIOCHEMICAL SYNTHESIS OF 1.4-BUTANEDIAMINE<br>
The present invention relates to a new process for biochemical synthesis of 1,4-butanediamine (CAS number 110-60-1; a compound also referred to as tetramethylenediamine; in biochemical literature it is also being referred to as putrescine) in a microorganism having an increased level of an ornithine decarboxylase activity as compared to the native level of the ornithine decarboxylase activity. Ornithine decarboxylase hereinafter also will be referred to as "ODC". An "increased level of ornithine decarboxylase activity" is hereinafter also referred to as "increased ODC activity". Generally microorganisms having ODC activity are known to be capable of producing polyamines such as spermidine and spermine, which are the common names for respectively the products N-(3-aminopropyl)-1,4-butanediamine and N,N'-bis-(3-aminopropyl)-1,4-butanediamine. Such compounds, as well as various short linear diamines themselves such as, for instance, 1,4-butanediamine and 1,5-pentanediamine (also referred to as cadaverine), are often referred to in biochemical studies as polyamines, even though from a strictly chemical definition of polyamines a higher number of arnino groups would be expected. For the purposes of the present patent application, however, the term polyamines is being used in its biochemical meaning and therefore includes 1,4-butanediamine.<br>
The compound 1,4-butanediamine is an important raw material for the production of some of the major engineering plastics: polyamide-4,6, either in the form of a homopolymer, or copolymerised, for example, with about 5 wt.% of polyamide-6 monomer (caprolactam). The homopolymer polyamide-4,6 (nylon-4,6) was described as early as 1938 (US-A-2,130,948, Carothers). It is the polycondensation product of the monomers 1,4-butanediamine and adipic acid. Presently, especially compounds of polyamide-4,6 are being produced and sold by DSM in the Netherlands under the trade name STANYL®.<br>
For the synthesis of 1,4-butanediamine a number of chemical routes are known. All these chemical routes suffer from the disadvantage that starting materials have to be obtained form sources that are considered to be non-renewable. There exists, however, a substantial need for providing new and feasible routes for the synthesis of 1,4-butanediamine starting from renewable carbon sources and using biochemical processes (also referred to as "biotransformation") in living cells. In general, polyamines are considered to be toxic for any cell or microorganism used in<br>
biochemical production. Therefore, until now such new routes by biochemical synthesis, however, were believed to be unattractive.<br>
This can for instance be seen from the following references: Fukuchi et a/., J. Biol. Chem., Vol.270 (1995), pages 18831-18835; and Suzuki etal., Proc. Natl. Acad. Sci. USA, Vol.91 (1994), pages 8930-8934.<br>
Fukuchi clearly describes the decrease in cell viability (and of synthesis of almost all kinds of proteins) due to the accumulation of spermidine in spermidine acetyltransferase-deficient cells of £ coli (i.e. in cells lacking the acetyltransferase SpeG). Spermidine is a product that is being produced in cells from 1,4-butanediamine as an intermediate. Accordingly, biosynthesis of 1,4-butanediamine inevitably also leads to formation of spermidine.<br>
Suzuki on the one hand also demonstrates (in mice cells), that overproduction of ODC results in accumulation of polyamines, especially of spermidine, and that - upon addition of small amounts of spermidine - already cell death is observed even in cells that are not deficient in speG.<br>
It is to be noticed that Limsuwum et al. (J. Bacteriol. Vol.182 (2000), pages 5373-5380) have shown that at low temperatures such problems can be overcome by overexpression of the dedicated gene speG. Suzuki et al. (cited above) suggest that the lowered cell viability is due to an insufficient feedback inhibition of ODC by antizymes and can be overcome by overproduction of a suitable antizyme. Such overproduction of antizymes then also would lower the production of polyamines in the cells and is therefore not feasible for DAB production.<br>
Further, as Kashiwagi et al. described in J. Bacteriol. Vol.170 (1988), pages 3131-3135, the contents of polyamines in £ co//can be adjusted by overexpression of an ornithine decarboxylase (ODC) encoding gene, in particular of the constitutively expressed speC. For their experiments the plasmid pODC as produced by Boyle et al. (Methods in Enzymology, Vol.94 (1983), pages 117-121, was used in the cloning. As taught by Kashiwagi ef al., even a 70-fold overproduction of ornithine decarboxylase SpeC under the native transcriptional and translational control (i.e. using the native genetic elements consisting of ribosomal binding site and promoter) did only lead to slightly increased levels of the sum of intra- and extracellular 1,4-butanediamine content. As can be seen in the cited Kashiwagi reference, the said authors were unable to reach higher production levels of 1,4-butanediamine than about 25 mg/l (without ornithine feeding). Moreover, they showed that overproduction of ODC in the cells did lead to a strong decrease in ornithine content in the cells (from about 65 umol/l to less than 1 umol/l) and concluded that the cells became ornithine deficient<br>
when ODC was overproduced. Kashiwagi et al. tried to remove the observed limitation in ornithine content by external feeding of ornithine, but - although a slight improvement was reached - production levels of 1,4-butanediamine did not get higher than about 30 mg/l. Due to the described restriction of the precursor supply by ODC overproduction and because, moreover, it would be expected that higher levels of proteins like ODC would cause increasing effects of toxicity in the cells due to the presence of higher amounts of polyamines, the skilled man, in view of the above references, would assume, that it would be impossible to provide biochemical synthesis processes for the production of 1,4-butanediamine at significantly higher levels than 30 mg/l.<br>
EP-A-0726240 until now is one of the very few patent references relating to the biochemical synthesis of polyamines, including 1,4-butanediamine. However, it describes the production of, inter alia, 1,4-butanediamine by fermentation of natural products containing proteins as a major component. In said process, the natural products first are treated by subjecting them to partial or total degradation, and any undesirable compounds (e.g. Hg, Cr, As, Cd, Se and Pb), cell growth inhibitors, pesticides, antibiotics, detergents, soaps, fats, oils, cyanides and phenols are then removed before the fermentation step. The putrescine and other diamines produced in such a way are being (re-)used as fertilizers and manures, but contain such large number of other substances that they are unsuitable as a raw material for the production of, for example, polyamide-4,6.<br>
Accordingly, there remains a need for an efficient biosynthetic route for the synthesis of 1,4-butanediamine at significantly higher titers than about 30 mg/l, preferably even without the need for external feeding of (expensive) ornithine. This need for improved availability of 1,4-butanediamine mainly is based on its intended use as a starting material, for instance, for the production of polyamide-4,6. In general, the routes to 1,4-butanediamine as are known until today are quite laborious and troublesome, and may lead to a quality of said product which - without further purification - is difficult to be used in the production of nylons. The known chemical routes to 1,4-butanediamine require relatively expensive starting materials and reactants (including reactants that are difficult to handle), and relatively severe reaction conditions of temperature and pressure in a multi-step and multi-reactor design, as well as the use of expensive catalyst systems. Accordingly there remains a need for alternative routes to 1,4-butanediamine, preferably from much less expensive raw materials and avoiding problems of handling reactants like hydrocyanic acid. It is well known that naturally growing, and thus renewable, materials from agricultural<br>
production are the basis for carbon sources such as glucose (or other appropriate carbon sources and mixtures thereof) that can be used in fermentation. Such renewable materials are relatively cheap and abundantly available. In general, it is considered to be very advantageous if renewable materials can be used as starting materials for all kinds of chemical materials.<br>
It is thus an aim of the present invention to provide improved possibilities for large-scale industrial production of 1,4-butanediamine by biotransformation.<br>
The present inventors surprisingly have found that this aim is achieved with a new process for biochemical synthesis of 1,4-butanediamine in a microorganism having an increased level of an ornithine decarboxylase activity (increased ODC activity) as compared to the native level of ornithine decarboxylase activity, wherein the increased ornithine decarboxylase activity is obtained by means of overexpression of an ornithine decarboxylase encoding gene with increased transcriptional and/or translational efficiency, and that 1,4-butanediamine produced in such microorganism by biotransformation is excreted into a fermentation broth, and is recovered from the fermentation broth.<br>
According to the present invention, thus, an improved biochemical process for the synthesis of 1,4-butanediamine is provided, and the resulting 1,4-butanediamine is excellently suitable as raw material, for instance, for the production of polyamtde-4,6.<br>
As meant in the present patent application, the term "biochemical synthesis" (a term that, in the context of this patent application, alternatively is referred to as "biotransformation") includes not only processes which involve - besides a number of purely chemical reaction steps - one or more biocatalytic reactions using whole cells of suitable production strains, but also purely biochemical processes using whole cells of suitable production strains. Such purely biochemical processes, respectively, are referred to as fermentations in case the biochemical synthesis starts from a suitable carbon source, or are referred to as precursor fermentations in case the biosynthesis starts from an intermediate product already having a carbon skeleton from which the target molecule to be synthesized can be obtained. The processes may be carried out either under aerobic or under anaerobic conditions.<br>
The biocatalytic reactions in the biochemical synthesis of the present invention can be carried out either in vivo or in vitro. Generally, in vivo processes are processes carried out when using living cells (the term "living cells" thereby also including so-called resting cells); in vitro processes, on the other hand, usually are<br>
being carried out using cell lysates or (partly) purified enzymes. The biochemical synthesis according to the present invention is carried out in a microorganism. This can be done using whole cells of suitable production strains, but also may be carried out using permeabilized cells; the differentiation between in vivo and in vitro, however, does not make much sense for processes being carried out with permeabilized cells or immobilized host cells. It will be evident, however, that individual biocatalytic steps from the process of the invention, when carried out, for instance, by using immobilized enzymes, etc. are considered equivalent to such steps in the biochemical synthesis as meant in the context of the present application.<br>
Ornrthine decarboxylases (i.e. enzymes having ornithine<br>
decarboxylation activity, or ODCs) are enzymes classified in class E.G. 4.1.1.17. The level of activity of an ornithine decarboxylase, if overproduced, can easily be compared with the native (i.e. non-overproduced) level of ornithine decarboxylase activity under standard conditions (at 37 °C in the presence of ornithine and PLP) within cell free extracts using the Sigma Diagnostics carbon dioxide detection kit (Sigma); assay described in Osterman, A. L. et a/. 1994, Biochemistry 33, p. 13662-13667. The skilled man, accordingly, can easily establish whether the ODC used has an increased level of ornithine decarboxylase activity (increased ODC activity) based on increased translational and/or transcriptional efficiency as compared to the native level of the ornithine decarboxylase activity in the microorganism used by determination of the protein content, or by determining the RNA level. Various standard procedures for determination of protein content, for instance colorimetric as well as spectroscopic methods, are described in Lottspeich and Zorbas, Bioanalytik, Spektrum Akademischer Verlag GmbH, Heidelberg / Berlin, ISBN 3-8274-0041-4 (1998), Chapters 3, 5, 21, 22 and 24. Methods for determination of protein level as well as RNA level, for instance northern hybridization, RT-PCR, and many other methods, are described in J. Sambrook, E.F. Fritsch and T. Maniatis, Molecular Cloning, A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory Press, ISBN 0-87969-309-6 (1989). Many other standard procedures, however, are known to the skilled man in this analytic field and do not need to be mentioned here.<br>
Suitable ornithine decarboxylases that can be used in the process of the invention are all enzymes and mutants thereof, that are capable of decarboxylating ornithine. Any such enzyme may be used in the process of the invention, at an increased level of activity, i.e. in overproduced form by means of overexpressing an ornithine decarboxylase gene with increased transcriptional and/or translational efficiency. In addition it is to be noticed, that the term "increased level of activity" as<br>
used herein for any specifically named enzyme activity is also intended to encompass such situations where the activity of such enzyme activity, for instance an ornithine decarboxylase, is not present at all in the natural source of the microorganism wherein the reaction is taking place, but is introduced therein purposively by genetic modification with increased transcriptional and/or translational efficiency.<br>
As mentioned above, in the biochemical synthesis of<br>
1,4-butanediamine any ODC enzyme may be used, of which the increased ODC activity, is obtained by means of overexpression of an ODC encoding gene with increased translational and/or transcriptional efficiency. Preferably, the increased translational and/or transcriptional efficiency is obtained by the use of a strong, regulated promoter, preferably by use of a strong inducible promoter.<br>
Most preferably, the increased translational and/or transcriptional efficiency is obtained by the use of an isopropyl-p-D-thiogalactopyranoside (IPTG) inducible strong promoter. Suitable strong promoters are described in J. Sambrook, E.F. Fritsch andT. Maniatis, Molecular Cloning, A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory Press, ISBN 0-87969-309-6 (1989).<br>
In particular, the increased translational and/or transcriptional efficiency is obtained by the use of a promoter selected from the group consisting of T7, T5, ptac, and p/ac promoters. It will be clear to the man skilled in the art, that the optimum choice of the promoter will be dependent of the host to be used and the reaction conditions to be applied.<br>
In a very preferred embodiment of the invention, the ornithine decarboxylase encoding gene has a Ribosomal Binding Site (RBS) located up-stream of the coding region of the said gene which RBS is adapted to achieve better recognition of RNA-template by the ribosomes. Adaptation of the RBS may be done by any method known to the skilled man, and will take into account specific properties of the host used, etc.<br>
Most preferably, the overexpressed ornithine decarboxylase encoding gene is an ornithine decarboxylase speF or speC gene (each belonging to E.G. 4.1.1.17). Until now SpeC has been investigated in literature much more than SpeF. Most surprisingly, however, and most preferably, best results according to the present invention are achieved when it is an ornithine decarboxylase speF gene.<br>
It is particularly preferred that the overexpressed ornithine decarboxylase encoding gene used in the process according to he invention is an ornithine decarboxylase gene speF or speC originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, and<br>
Shewanella. The omrthine decarboxylase speF is an inducible ornithine decarboxylase; ornithine decarboxylase speC is a constitutive ornithine decarboxylase.<br>
More preferably, the overexpressed ornithine decarboxylase encoding gene is an ornithine decarboxylase gene originating from one of the species selected from the group consisting of Escherichia coll, Shigella flexneri, Salmonella typhimutium, Yersinia pestis, and Shewanella oneidensis. Most preferably, the overexpressed ornithine decarboxylase encoding gene is speF originating from one of the species selected from the group consisting of Escherichia coli, Salmonella typhimutium, and Shewanella oneidensis. When compared to results with overexpression of the constitutive ornithine decarboxylase encoding gene speC, by far the best results according to the present invention indeed are being achieved when using speF.<br>
According to the present invention, in particular, all ornithine decarboxylases can be used that have sufficient, i.e. at least 30%, more preferably at least 45%, and most preferably at least 65% identity with the ODC from the £ coli reference enzyme, and are capable of catalyzing the ornithine decarboxylation reaction. Many ODCs are known having such relatively high level of identity with the £. coli reference enzyme.<br>
Determining identity percentages with reference enzymes can be performed by methods known to the skilled man, for instance by using the protein sequence of the reference enzyme as a "query sequence" to perform a search against public databases to, for example, identify other family members or related sequences. Such searches can be performed using BLAST programs (version 2.2) using the default parameters of respective program. See http://www.ncbi.nlm.nih.gov.<br>
According to the present invention, thus, an improved biochemical process for the synthesis of 1,4-butanediamine is provided, and the resulting 1,4-butanediamine is excellently suitable as raw material for the production of polyamide-4,6 and/or other polyamides.<br>
It will be clear, that - in the context of the present invention - any gene being homologous with any of the genes coding for the abovementioned ornithine decarboxylases and coding for enzymes having ornithine decarboxylase activity sufficiently comparable to the ornithine decarboxylases shown, is to be considered an equivalent thereof and suitable in the process of the invention. Such equivalent genes can suitably be obtained by means of any appropriate cloning strategy known to the skilled man, for instance, by the methods described in the experimental part hereof.<br>
Alternatively, such equivalent ornithine decarboxylase genes can also be obtained by purposive construction.<br>
In a further preferred embodiment of the present invention, the process for the biochemical synthesis of 1,4-butanediamine is carried out in a microorganism wherein, additionally to the increased ODC activity also increased enzyme activity is obtained for at least two other enzymes by means of overexpression of either (i)       an arginine decarboxylase encoding gene speA (belonging to E.G. 4.1.1.19)<br>
and an agmatinase encoding gene speB (belonging to E.G. 3.5.3.11; also<br>
referred to as agmatine ureahydrolase encoding gene); or (ii)      an arginine decarboxylase encoding gene speA (belonging to E.G. 4.1.1.19),<br>
and an agmatine iminohydrolase encoding gene aguA (belonging to E.G.<br>
3.5.3.12; also referred to as agmatine deiminase encoding gene), and an<br>
N-carbamoylputrescine amidohydrolase encoding gene aguB (belonging to E.G.<br>
3.5.1.53), and optionally also an agmatinase encoding gene speB (belonging to<br>
E.G. 3.5.3.11).<br>
Overexpression as meant herein for these additionally increased enzyme activities, can be achieved by any method known to the skilled man; for instance by increasing the translational and/or transcriptional efficiency of the respective gene, but also by any other known methods such as increasing the gene copy number, or by increasing the endogenous activity or structure of the enzymes by means of mutations, or by using deregulated enzymes. As meant in part (i) of the further preferred embodiment mentioned here above the combination of SpeA and SpeB is intended to represent any functional combination (whether in a combined fusion protein, or as separate enzyme activities) of SpeA and SpeB. In fact, this combination also might be designated as SpeAB. Part (ii) hereof represents, that in such combinations of SpeA and SpeB, the SpeB-part itself may be replaced by any functional combination (whether in a combined fusion protein, or as separate enzyme activities) of AguA and AguB.<br>
Janowitz et al., FEBS Letters 544 (2003), 258-261, have described that agmatine deiminase AguA is involved in the arginine decarboxylase pathway in higher plants. It is further known from Nakada et al., Microbiology, 149 (2003), 707-714, that the conversions catalyzed by SpeB also can be catalyzed by enzymes occurring in plants, namely by the combined action of agmatine deiminase AguA and A/-carbamoylputrescine amidohydrolase AguB. Accordingly, instead of, or even in combination with, SpeB in the context of the present invention also AguA and AguB<br>
can be used. Sources for such aguA and aguB genes could be Arabidopsis thaliana and Lycopersicon esculentum, but comparable genes can be found in mutants of Pseudomonas aeroginosa.<br>
It will be clear, that - in the context of the present invention - any gene being homologous with any of the genes coding for the abovementioned arginine decarboxylases, respectively agmatinases, or agmatine iminohydrolases or N-carbamoylputrescine amidohydrolases, and coding for such respective enzymes having arginine decarboxylase (respectively agmatinase, or agmatine iminohydrolase or N-carbamoylputrescine amidohydrolase) activity sufficiently comparable to the respective enzymes - as the case may be - is to be considered an equivalent thereof and suitable in this further preferred embodiment of the process of the invention. Such equivalent genes suitably can be obtained by means of any suitable cloning strategy known to the skilled man, for instance, by the methods described in the experimental part hereof. Alternatively, such equivalent genes also can be obtained by purposive construction.<br>
Accordingly, in this preferred embodiment of the process of the present invention, also additional combinations of overexpressed genes are being used, namely genes encoding for (i) arginine decarboxylase and agmatinase, or (ii) arginine decarboxylase and agmatine iminohydrolase and N-carbamoylputrescine amidohydrolase, and optionally agmatinase.<br>
In this further preferred embodiment of the invention, the overexpressed arginine decarboxylase encoding gene is preferably an arginine decarboxylase gene speA originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, Pasteurella, and Neisseria. More preferably, the overexpressed arginine decarboxylase encoding gene is an arginine decarboxylase gene speA originating from one of the species selected from the group consisting of Escherichia coll, Shigella flexneri, Salmonella enterica, Yersinia pestis, Pasteurella multocida, and Neisseria meningitidis.<br>
According to the present invention, in particular, all arginine decarboxylases can be used that have sufficient, i.e. at least 30%, more preferably at least 45%, and most preferably at least 65%, identity with the arginine decarboxylase from the £ coll reference enzyme, and are capable of catalyzing the arginine decarboxylation reaction. Many arginine decarboxylases are known having such relatively high level of identity with the £. coli reference enzyme.<br>
In this further preferred embodiment of the invention, the overexpressed agmatinase encoding gene is an agmatinase gene speB originating<br>
from one of the genera selected from the group consisting of Escherichia, Salmonella, Proteus, Photorhabdus, Vibrio, and Neisseria. More preferably, the overexpressed agmatinase encoding gene is an agmatinase gene speB originating from one of the species selected from the group consisting of Escherichia coli, Salmonella enterica, Proteus mirabifis, Photorhabdus luminescens, Vibrio cholerae, and Neisseria meningitidis.<br>
According to the present invention, in particular, all agmatinases can be used that have sufficient, i.e. at least 30%, more preferably at least 45%, and most preferably at least 60%, identity with the agmatinase from the £ coll reference enzyme, and are capable of catalyzing the agmatinase reaction. Many agmatinases are known having such relatively high level of identity with the E. coli reference enzyme.<br>
In this further preferred embodiment of the invention, moreover, the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoylputrescine amidohydrolase encoding gene is preferably an agrnatine iminohydrolase gene aguA and/or an N-carbamoylputrescine amidohydrolase gene aguB originating from one of the genera selected from the group consisting of Pseudomonas, Streptococcus, Streptomyces, Azotobacter, Arabidopsis, Novosphingobium, and Bacillus. More preferably, the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoyiputrescine amidohydrolase encoding gene is an agmatine iminohydrolase gene aguA and/or an N-carbamoylputrescine amidohydrolase gene aguB originating from one of the species selected from the group consisting of Pseudomonas aeruginosa, Streptococcus mutans, Streptomyces avermitilis, Azotobacter vinelandii, Arabidopsis thaliana, Novosphingobium aromaticivorans, and Bacillus cereus.<br>
According to the present invention, in particular, all agmatine iminohydrolases and/or N-carbamoyiputrescine amidohydrolases can be used that have sufficient, i.e. at least 30%, and most preferably at least 40%, identity with the agmatine iminohydrolase and/or the N-carbamoylputrescine amidohydrolase from the Pseudomonas reference enzyme, and are capable of catalyzing the agmatine iminohydrolase, respectively the N-carbamoylputrescine amidohydrolase reaction. Many agmatine iminohydrolases and/or the N-carbamoylputrescine amidohydrolases are known having such relatively high level of identity with the Pseudomonas reference enzyme.<br>
It is preferred, that the process according to the invention is being carried out whilst ensuring an increased intraceilular level of ornithine. This can, for instance, be achieved by externally feeding of ornithine.<br>
The process of the invention may be carried out in any suitable host organism. The hosts may be selected from the groups of production organisms (or cells) generally known to the skilled man in biosynthesis. Such organisms may be from eukaryotic origin, or - as is more preferred - from prokaryotic origin. Eukaryotic cells, for instance, can be cells from plants and fungi, and from various other groups, which other groups collectively are referred to as "Protista".<br>
It is particularly preferred, that the process according to the invention is carried out in a host organism selected from the group consisting of Saccharomyces sp., Bacillus sp., Corynebacterium sp., Escherichia sp. and Pichia sp.<br>
In the process of the invention, it is especially preferred that the microorganism to be used as a host is able to produce the amino acids ornithine and/or arginine. For most natural microorganisms this requirement is fulfilled because usually such capability is available in all wild type strains, since arginine represents an essential amino acid.<br>
Of these species, Escherichia sp. are preferred because they are easy to handle by genetic manipulation in order to provide strains with the desired overexpressed enzyme activities. Moreover, Escherichia sp. already in nature contain almost each of the abovementioned enzyme activities (i.e. apart from the agu genes from plants), so that most of the overexpressed genes can be used as homologous genes. Also, Corynebacterium sp. (though lacking a natural ornithine decarboxylase) is particularly preferred because it is a suitable glutamate production strain that can be handled easily in fermentation processes.<br>
In the process of the present invention glutamate is a very suitable precursor. Accordingly, the process is preferably being carried out in a host strain capable of formation of glutamate (for instance, Corynebacterium glutamicum).<br>
Best results are being achieved when the process according to the invention it is carried out in a host organism from the group consisting of Saccharomyces cerevisiae, Corynebacterium sp. and Escherichia sp. wherein, apart from the activity of an ornithine decarboxylase, arginine decarboxylase and at least agmatinase or agmatine iminohydrolase and N-carbamoylputrescine amidohydrolase enzyme activities is being present in the host microorganism at an increased activity level as compared with the native level of the said enzyme activity is homologous to the host microorganism.<br>
It will be clear that the process of the invention is preferably carried out under reaction conditions that are also usual as fermentation conditions. The process, therefore can be carried out batch-wise, but also - if so desired - fed-batch. It<br>
may be convenient to ensure that the organism used as host organism has, or is provided with, a suitable exporter system for the 1,4-diaminobutane formed: Preferably such exporter system is a native one.<br>
The present invention, of course, also encompasses all vectors, plasmids and hosts carrying, at an increased level of activity, one or more of the aforementioned enzyme activities according to the attached claims.<br>
The invention will now be elucidated by means of some experimental results, which by no means are intended to limit the scope of the invention.<br>
EXPERIMENTAL PART<br>
General procedures<br>
Standard procedures were applied for all DNA manipulations (Sambrook, J. et a/. (1989), Molecular cloning: a laboratory manual, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York). DNA was amplified from chromosomal DNA of £ CO//LJ110 (Zeppenfeld, et a/. (2000), J Bacteriol. 182, 4443-4452) if not indicated otherwise. PCR amplification was performed using the proof-reading enzymes SAWADYPwo-DNA-Polymerase (Peqlab Biotechnologie GmbH, Erlangen, Germany) or Platinum Pfx DNA Polymerase (Invitrogen, Karlsruhe, Germany) following the manufacture's protocol, whereas the verification of the constructed strains was carried out by colony PCR utilizing the Taq poiymerase READYMIX (Sigma, Taufkirchen, Germany). Restriction sites for subsequent cloning as well as further mutations were introduced with oligonucleotides purchased from MWG-Biotech (Ebersberg, Germany). DNA fragments were purified with the MinElute Gel Extraction Kit (Qiagen, Hilden, Germany) following the manufacture's protocol. Preparation of plasrnid DNA was accomplished by the utilization of QIAprep spin Miniprep Kit (Qiagen, Hilden, Germany). Verification of the constructed plasmids was carried out by restriction analysis and subsequent sequencing (Agowa, Berlin, Germany).<br>
Construction of plasmids<br>
(i)     Construction of the plasmid pDABS (pJF119EH-speCnra3S)<br>
The (constitutive, biosynthetic) ornithine decarboxylase encoding gene speC of £ coll LJ110 (Zeppenfeld, et a/., see general procedures) was cloned into the expression vector pJF119EH (Furste, J. P. et al. (1986), Gene 48,119-131), allowing a strong gene expression based on the transcriptional control under the<br>
isopropyl-p-D-thiogalactopyranoside (IPTG) inducible tac promotor and the lac represser system (laclQ). Therefore, the coding gene speC was cloned with original RBS, start and stop codon.<br>
The 2235 bp speCnRBs -containing DMA fragment was amplified from chromosomal DNA of £ coll LJ110 (accession number AE000379; nucleotides 2650 -4867) using the following oligonucleotides:<br>
5'-GAG C7C TAG ACC AGT TTG ACC CAT ATC T-3'	[SEQ ID: No.1]<br>
(mutations in bold, Xba\ restriction site in italics) and<br>
5'-TTTTGC ATG CTT ACT TCA ACA CAT AAC CGT AC-3'	[SEQ ID: No.2]<br>
(mutations in bold, Sph\ restriction site in italics).<br>
After terminal modification with the endonucleases Xba\ and Sphl, the PCR product was ligated into plasmid pJF119EH, which was cut in the same manner. After transformation in £. coli DH5oc cells (Invttrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDAB3 (pJF119EH-speCnRBS, 7491 bp) was carried out by restriction analysis and subsequent sequencing.<br>
(ii)    Construction of the plasmid pDAB4 (pJF119EH-speCBRBS)<br>
The (constitutive, biosynthetic) ornithine decarboxylase encoding gene speC of £ coli LJ110 (Zeppenfeld, et a/., see general procedures) was cloned into the expression vector pJF119EH (Furste, J. P. etal. (1986), Gene 48, 119-131), allowing a strong gene expression based on the transchptional control under the isopropyl-p-D-thiogalactopyranoside (IPTG) inducible tac promotor and the lac represser system (laclQ). Therefore, the coding gene speC was cloned with original start and stop codon. Since, no conserved ribosomal binding site (RBS) could be determined for speC utilizing in silico studies, the RBS, located 7 bp upstream of the speC start codon, was adapted to the consensus sequence of £ coli by site-directed mutagenesis.<br>
The 2235 bp speCaRBs -containing DNA fragment was amplified from chromosomal DNA of £ coli LJ110 (accession number AE000379; nucleotides 2650 -4867) using the following oligonucleotides:<br>
5'-GAG CTC TAG ACC AGT TTG AGO AAT ATC T-3'	[SEQ ID: No.3]<br>
(mutations in bold, Xba\ restriction site in italics) and<br>
5'-TTT TGC ATG CTT ACT TCA ACA CAT AAC CGT AC-3'	[SEQ ID: No.2]<br>
(mutations in bold, Sph\ restriction site in italics).<br>
After terminal modification with the endonucleases Xbal and Sph\, the PCR product was ligated into plasrnid pJF119EH, which was cut in the same manner. After transformation in £. co//DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l arnpiclllin. After preparation, the verification of the obtained plasmid pDAB4 (pJF119EH-speCafl0S, 7491 bp) was carried out by restriction analysis and subsequent sequencing.<br>
(iii)    Construction of the piasmid pDAB2 (pJF119EH~speF)<br>
The (inducible, biodegradative) ornithine decarboxylase encoding gene speF of E. coli LJ110 (Zeppenfeld, et al., see general procedures) was cloned into the expression vector pJF119EH (Furste, J. P. et al. (1986), Gene 48, 119-131). This vector allows a high-level protein production based on the transcriptionai control of cloned genes under the isopropyl-(3-D-thiogalactopyranoside (JPTG) inducible tac promotor and the lac represser system (lac!Q). For construction of the expression plasmid pDAB2 (pJFUQEH-spe/1) the coding gene speFwas cloned with original RBS (ribosomai binding site), start and stop codon.<br>
The 2247 bp speF-containing DNA fragment was amplified from chromosomal DNA of £ co//LJ11Q (accession number AE000172; nucleotides 10242 -12468) using the following oligonucleotides:<br>
5'-GAC CTG CTG GTA CCT AAA ATA AAG AGA TGA AA-3'        [SEQ ID: No.4] (mutations in bold, Kpn\ restriction site in italics) and<br>
5'-TCG ATC TAG ACT GAC TCA TAA TTT TTC CCC-31	[SEQ ID: No,5]<br>
(mutations in bold, Xbal restriction site in italics).<br>
The fragment was terminally modified with the restriction<br>
endonucleases Kpnl and X6al and ligated into the expression vector pJF119EH, which was cut in the same manner. After transformation in £. coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mgll ampicillin. After preparation, the obtained plasmid pDAB2 (pJF119EH-speF, 7502 bp) was verified by restriction analysis and subsequent sequencing.<br>
(iv)   Construction of the plasmid pDAB7 (pJF119EH~spa4B)<br>
The arginine decarboxylase encoding gene speA as well as speS<br>
coding for the agmatinase of £ coli LJ110 (Zeppenfeld, et al., see general procedures) were cloned into the expression vector pJF119EH (Ftirste, J. P. et al. (1986), Gene 48, 119-131) allowing a high-level protein production based on the transcriptional control of cloned genes under the isopropyl-p-D-thiogalactopyranoside (IPTG) inducible tac promotor and the lac represser system (laclQ). This way, the original operon structure of the genes as well as RBS, start and stop codon were maintained.<br>
The 3079 bp spe/4B-containing DNA fragment was amplified from chromosomal DNA of £. coli LJ110 (accession number AE000377; nucleotides 1190 -4247) using the following oligonucleotides:<br>
5-ACA CTT TCTAGA ATA ATT TGA GGTTCG CTA TG-31	[SEQ ID: No.6]<br>
(mutations in bold, Xibal restriction site in italics) and<br>
5-CAT GGC ATG CGGTGC TTA CTC G-3'	[SEQ ID: No.7]<br>
(mutations in bold, Sph\ restriction site in italics).<br>
After terminal modification with the restriction endonucleases Xba\ and Sph\, the DNA fragment was ligated into the expression plasrnid pJF119EH, which was cut likewise. After transformation in £ coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the verification of the obtained plasmid pDAB7 (pJF119EH-sp©/4B, 8339 bp) was carried out by restriction analysis and subsequent sequencing.<br>
(v)    Construction of the plasmid pDAB8 (pJF119EH-speF-speAB)<br>
In order to allow in-parallel production of the ornithine decarboxylase SpeF, the arginine decarboxylase SpeA and the agmatinase SpeB, the speAB genes of £ coli LJ110 (Zeppenfeld, et al., see general procedures) were cloned into the speF-expression vector pDAB2 (see iii)).<br>
By digestion of the plasmid pDAB7 (see iv)) with the restriction endonucleases Xba\ and Sp/?l, the 3067 bp comprising speAB gene-operon was separated and ligated into the speF containing plasmid pDAB2 (see (iii)), which was cut in the same manner. After transformation in £ coli DH5a cells (Invitrogen, Karlsruhe, Germany), transformants were selected on LB agar plates containing 100 mg/l ampicillin. After preparation, the obtained plasmid pDAB8 (pJH119EH-speF/A6, 10547 bp) allowing the in-parallel production of SpeFAB was verified by restriction analysis.<br>
Example 1: Improvement of DAB production by overexpression of ornithine decarboxvlase encoding genes with increased translational and/or transcriptional efficiency<br>
Example 1.1 Production of 1,4-butanediamine via the overproduction of ornithine decarboxylases (shake flask)<br>
The influence of overexpression of ornithine decarboxylase encoding genes speF or speC (with increased translational and/or transcriptional efficiency) on DAB production was investigated within the £ coli host strain LJ110 (Zeppenfeld, et a/., see general procedures) carrying the plasmid pDAB2 (see (iii)), or pDABS (see (i)), or pDAB4 (see (ii)).<br>
These strains were tested in shake flask experiments utilizing minimal salt medium consisting of MgS04-7H20 (300 mg/l), CaCI2-2H20 (15 mg/l), KH2PO4 (3 g/l), K2HP04 (12 g/l), IslaCI (100 mg/l), (NH4)2S04 (5 g/l), Na citrate-3H20 (1 g/l), FeS04-7H20 (75 mg/l), thiamine-HCI (vitamin B1) (5 mg/l) as well as the trace elements AI2(S04)3-18H20 (3 mg/l), CoCI2-6H20 (1.05 mg/l), CuS04-5H20 (3.75 mg/l), H3BO3 (0.75 mg/l), MnCI2-4H20 (30 mg/l), Na2Mo04-2H2O (4.5 mg/l), NiS04-6H2O (3 mg/l) and ZnS04-7H20 (22.5 mg/l). A stock solution of glucose (500 g/l) was autoclaved separately and added to the sterilized medium up to a final concentration of 10 g/l.<br>
A preculture of minimal salt medium containing 100 mg/l ampicilline was inoculated with 1 - 5 ul/ml stock solution and incubated at 33 "C, 180 rpm for 16 h up to an OD62o of 2. 5 ml of this culture was subsequently used for inoculation of the main culture consisting of 50 ml of the same medium, which was incubated for 24 h at 33 °C and 180 rpm. Since the cells reached an OD62onm of 1.5 (after ~ 7 h), gene expression was induced by the addition of 50 pM IPTG.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, diluted supernatant was analyzed by HPLC. Here, the contained amines were detected as ort/io-phthaldialdehyde (OPA) derivatives at 230 nm on a Hewlett-Packard 1100 Series instrument, using a C18-reverse phase column (Nucleosil 120-5 cib, Macherey &amp; Nagel, Duren, Germany) equilibrated to 50% buffer B (buffer A, 0.1 M sodium acetate pH 7.2; buffer B methanol). For separation, the following gradient was applied: 1 - 7 min linear gradient from 50% to 75% buffer B with a flow rate of 0.5 ml/min, 7 -13 min 75% to 85% buffer B with a flow rate of 0.5 ml/min, 13-14.5 min 85% to 50% buffer B with a low rate of 1 ml/min, 14.5-17 min 50% buffer B with a flow rate of 1 ml/min and 17 - 20 min at 50% buffer B with a flow rate of 0.5 ml/min.<br>
By the utilization of standard substances for calibration, the following DAB concentrations could be determined (see Table 1) and verified by NMR spectroscopy.<br>
Strain used         Gene       DAB concentration	Remarks<br>
expressed	[mg/l]<br>
LJ110pDAB2       speF	869	IPTG induction; nRBS<br>
LJ110pDAB3    speC^s	~ 50	IPTG induction; nRBS<br>
LJ110pDAB4    speCaRss	715	Adapted RBS + IPTG induction<br>
Table 1: DAB formation utilizing ODC overproduction in E. coli<br>
Example 1.2 Prpduction of 1,4-butanediamine via the overproduction of ornithine decarboxvlase within fed batch (21) cultivation<br>
The potential of fermentative DAB production under fed batch conditions was investigated by the utilization of the high-level DAB producer strain LJ110 pDAB2 within a labfors bioreactor (Infers, Einsbach, Germany). Since, the DAB producing strain is not amino acid dependent for its growth, a developed protocol for phosphate limited cultivation was applied in order to restrict the cell growth. Therefore, a phosphate limited minimal salt medium was used consisting of MgSGy7H2O (3 g/l), CaCI2'2H2O (15 mg/l), KH2P04 (400 mg/l), NaCI (1 g/l), (NH4)2S04 (5 g/l) as well as the trace elements AI2(S04)3-18H20 (3 mg/l), CoCI2-6H2O (1.05 mg/l), CuS04-5H2O (3.75 mg/l), H3B03 (0.75 mg/l), MnCI2-4H20 (30 mg/l), Na2MoO4-2H20 (4.5 mg/l), NiS04-6H20 (3 mg/l) and ZnSO4-7H20 (22.5 mg/l). After autoclaving, Na citrate-3H20 (1.5 g/l), FeSO4-7H2O (112.5 mg/l), thiamine-HCI (vitamin B1) (7.5 mg/l), ampicilline (100 mg/l) and glucose (10 g/l) was added under sterile conditions into the bioreactor.<br>
A preculture of minimal salt medium (see 1.1) containing 100 mg/l ampicilline was inoculated with 1 - 5 ul/ml stock solution and incubated at 33 °C, 180 rpm for 16 h up to an optical density at 620 nm of 2. Following, this culture was subsequently used for 1:10 inoculation of the main culture consisting of 2 I phosphate limited minimal salt medium. During cultivation, the temperature was kept constant at 33 °C and the pH was controlled to 6.8 ± 0.1 by the addition of 5 N KOH. During growth, a stable agitation speed of 1500 rpm was used. In order to avoid oxygen limitation, the gas flow into the vessel was increased from 2.5-10 l/min during cultivation. Antifoam Dehysan was added as needed.<br>
The cells were induced with 50 uM IPTG at an OD620 of 5 followed by<br>
a combined glucose (500 g/l) / ammonia sulfate (200 g/l) feed, whereas the feeding rate was adapted in order to receive stable concentrations of 10 g/l glucose and 1.5 g/l ammonia. Two hours after induction, phosphate feed consisting of 18 g/l KH2PO4 with a feeding rate of 7 ml/h was started for 8 h. This way, cell growth could be restricted to an OD620 of ~ 50.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, supernatant was analyzed by HPLC (see 1.1) and a DAB amount of 5.1 g/l (0.403 g/gBDW; BDW means Biomass Dry Weight) was determined and verified by NMR spectroscopy.<br>
Example 2: Production of 1,4-butanediamine within batch starting from ornithine as well as arginine (shake flask)<br>
For demonstrating further improvement of DAB formation starting from ornithine as well as arginine, the influence of combined overproduction of the ornithine decarboxylase SpeF(with increased transcriptional efficiency), the arginine decarboxylase SpeA and the agmatinase SpeB was investigated.<br>
Therefore, shake flask cultivations were carried out in minimal salt medium (see 1.1) by the utilization of the E. coli host strain LJ110 (Zeppenfeld, ef a/., see general procedures) carrying the plasmid pDABS (see (v)). Therefore, a preculture of minimal salt medium containing 100 mg/l ampicilline was inoculated with 1-5 pi/ml stock solution and incubated at 33 °C, 180 rpm for 16 h up to an optical density at 620 nm of 2. 5 ml of this culture was subsequently used for inoculation of the main culture consisting of 50 ml of the same medium, which was incubated for 24 h at 33 °C and 180 rpm. Since the cells reached an OD620nm of 1.5 (after ~ 7 h), gene expression was induced by the addition of 10 uM IPTG.<br>
In order to observe the time-dependent DAB production, samples were taken at different time points during cultivation. After separation of the cells utilizing centrifugation, the supernatant was analyzed by HPLC (see 1.1). By the utilization of standard substances for calibration, the following DAB concentrations could be determined (see Table 3).<br>
d strain	expressed	DAB concentration<br>
genes	[mg/l]<br>
LJ110pDAB8	speFAB	1025<br>
Table 2: DAB formation starting from ornithine as well as arginine in E. coli<br><br><br>
CLAIMS<br>
1.	Process for biochemical synthesis of 1,4-butanediamine in a microorganism<br>
having an increased level of an ornithine decarboxylase activity (increased<br>
ODC activity) as compared to the native level of the ornithine decarboxyiase<br>
activity,<br>
characterized in that<br>
the increased ornithine decarboxylase activity is obtained by means of overexpression of an ornithine decarboxylase encoding gene with increased translational and/or transcriptional efficiency, and that 1,4-butanediamine produced in the microorganism is excreted into a fermentation broth, and is recovered from the fermentation broth.<br>
2.	Process according to claim 1,<br>
wherein<br>
the increased translational and/or transcriptional efficiency is obtained by the use of a strong, regulated promoter, preferably by use of a strong inducible promoter.<br>
3.	Process according to claim 2,<br>
wherein<br>
the increased translational and/or transcriptional efficiency is obtained by the use of an isopropyl-p-D-thiogalactoside (IPTG) inducible strong promoter.<br>
4.	Process according to any of claims 1 to 3,<br>
wherein<br>
the increased translational and/or transcriptional efficiency is obtained by the use of a promoter selected from the group consisting of T7, T5, ptac, and p/ac promoters.<br>
5.	Process according to any of claims 1 to 4,<br>
wherein<br>
the ornithine decarboxylase encoding gene has a Ribosomal Binding Site (RBS) located up-stream of the coding region of the said gene which RBS is adapted to achieve better recognition of RNA-tempiate by the ribosomes.<br>
6.	Process according to any of claims 1 to 5,<br>
wherein<br>
the overexpressed ornithine decarboxylase encoding gene is an ornithine decarboxylase speF or speC gene (each belonging to E.G. 4.1.1.17).<br>
7.	Process according to claim 6,<br>
wherein<br>
the overexpressed omithine decarboxylase encoding gene is an ornithine decarboxylase speF gene.<br>
8.	Process according to any of claims 6 or 7,<br>
wherein<br>
the overexpressed ornithine decarboxylase encoding gene is an ornithine decarboxylase gene speF or speC originating from one of the genera selected from the group consisting of Escherichia, Shigella, Salmonella, Yersinia, and Shewanella.<br>
9.	Process according to claim 8,<br>
wherein<br>
the overexpressed ornithine decarboxylase encoding gene is an ornithine decarboxylase gene originating from one of the species selected from the group consisting of Escherichia coll, Shigella flexneri, Salmonella typhimutium, Yersinia pestis, and Shewanella oneidensis.<br>
10.	Process according to claim 9,<br>
wherein<br>
the overexpressed ornithine decarboxylase encoding gene is speF originating from one of the species selected from the group consisting of Escherichia coll, Salmonella typhimutium, and Shewanella oneidensis.<br>
11.	Process according to any of claims 1 to 10,<br>
wherein<br>
additionally to the increased ODC activity also increased enzyme activity is obtained for at least two other enzymes by means of overexpression of either (i) an arginine decarboxylase encoding gene speA (belonging to E.C. 4.1.119) and an agmatinase encoding gene speB (belonging to E.C. 3.5.3.11; also referred to as agmatine ureahydrolase encoding gene); or (ii) an arginine decarboxylase encoding gene speA (belonging to E.G. 4.1.1.19), and an agmatine iminohydrolase encoding gene aguA (belonging to E.G. 3.5.3.12; also referred to as agmatine deiminase encoding gene), and an N-carbamoylputrescine amidohydrolase encoding gene aguB (belonging to E.C. 3.5.1.53), and optionally also an agmatinase encoding gene speB (belonging to E.C. 3.5.3.11).<br>
12.	Process according to claim 11,<br>
wherein<br>
the overexpressed arginine decarboxylase encoding gene is an arginine<br>
decarboxylase gene speA originating from one of the genera selected from the group consisting of Escherichia, Shigelia, Salmonella, Yersinia, Pasteurella, and Neisseria.<br>
13.	Process according to claim 12,<br>
wherein<br>
the overexpressed arginine decarboxylase encoding gene is an arginine decarboxylase gene spe/A originating from one of the species selected from the group consisting of Escherichia coii, Shigella flexneri, Salmonella enterica, Yersinia pestis, Pasteurella multocida, and Neisseria meningitidis.<br>
14.	Process according to claim 13,<br>
wherein<br>
the overexpressed agmatinase encoding gene is an agmatinase gene speB originating from one of the genera selected from the group consisting of Escherichia, Salmonella, Proteus, Photorhabdus, Vibrio, and Neisseria.<br>
15.	Process according to claim 14,<br>
wherein<br>
the overexpressed agmatinase encoding gene is an agmatinase gene speB originating from one of the species selected from the group consisting of Escherichia coli, Salmonella enterica, Proteus mirabilis, Photorhabdus luminescens, Vibrio cholerae, and Neisseria meningitidis.<br>
16.	Process accordtng to claim 15,<br>
wherein<br>
the overexpressed agmatine iminohydrolase encoding gene and/or the overexpressed N-carbamoylputrescine arnidohydroiase encoding gene is an agmatine iminohydrolase gene aguA and/or an N-carbamoylputrescine arnidohydroiase gene aguB originating from one of the genera selected from the group consisting of Pseudomonas, Streptococcus, Streptomyces, Azotobacter, Arabidopsis, Novosphingobium, and Bacillus.<br>
17.	Process according to claim 16,<br>
wherein<br>
the overexpressed agmatine iminohydroiase encoding gene and/or the overexpressed N-carbarnoylputrescine arnidohydroiase encoding gene is an agmatine iminohydrolase gene aguA and/or an N-carbamoyiputrescine arnidohydroiase gene aguB originating from one of the species selected from the group consisting of Pseudomonas aeruginosa, Streptococcus mutans, Streptomyces avermitilis, Azotobacter vinelandii, Arabidopsis thaliana,<br>
Navasphingobium aromaticivorans, and Bacillus cereus.<br>
18.	Process according to any of claims 1 to 17,<br>
wherein<br>
the process is being carried out whilst ensuring an increased intraceliular level of ornithine.<br>
19.	Process according to any of claims 1 to 18,<br>
wherein<br>
the process is carried out in a host organism selected from the group consisting of Saccharomyces sp., Bacillus sp., Corynebacterium sp., Escherichia sp. and Pichia sp.<br>
20.	Process according to any of claims 1 to 19,<br>
wherein<br>
the process is carried out in a host organism selected from the group consisting of Saccharomyces cerevisiae, Corynebacterium sp. and Escherichia sp. and in that, apart from the increased levei of activity of an ornrthine decarboxyiase, at least also the level of activity of an arginine decarboxylase in combination with an agmatinase and/or an agmatine iminohydrolase and an N-carbamoylputrescine amidohydrolase is increased.<br>
21.	Vectors, plasmids and hosts carrying, at an increased level of activity, one or<br>
more of the enzyme activities as are mentioned in one or more of claims 1-20.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">478-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctQ2xhaW1zLSgyMC0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Claims-(20-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctQ2xhaW1zLSgyNC0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Claims-(24-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">478-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyMC0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Correspondence Others-(20-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyNC0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Correspondence Others-(24-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2Utb3RoZXJzICgyMS0wNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Correspondence-others (21-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyNC0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Correspondence-Others-(24-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LURFTE5QLTIwMDctQ29ycmVzcG9uZGVuY2UtT3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">478-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">478-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctRm9ybS0xOCAoMjEtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Form-18 (21-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">478-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctRm9ybS0zLSgyNC0wOC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Form-3-(24-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LURFTE5QLTIwMDctRm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">478-DELNP-2007-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">478-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">478-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctcGN0LW5vdGlmaWNhdGlhbi5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-pct-notificatian.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDc4LWRlbG5wLTIwMDctUGV0aXRpb24tMTM3LSgyNC0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">478-delnp-2007-Petition-137-(24-01-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260245-gps-based-programmable-search-and-rescue-beacon.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260247-ophthalmic-compositions-containing-mucoadhesive-polysaccharides-able-to-promote-corneal-re-epthelization.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260246</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>478/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>16/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>18-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>15-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jan-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, NL-6411 TE HEERLEN, THE NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NOSSIN, PETRUS MARTINUS MATHEUS</td>
											<td>SCHOOR 11, NL - 6031 SC NEDERWEERT, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KREMER, SUSANNE MARIA</td>
											<td>EDERENERSTRASSE 32, 52441 LINNICH-GEREONSWEILER, GERMANY.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>WUBBOL TS. MARCEL GERHARDUS</td>
											<td>PREVOTLAAN 9, NL - 6132 BM SITTARD, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>EPPELMANN, KATRIN</td>
											<td>4461 HEMMINGWAY DRIVE, KALAMAZOO, MICHIGAN 49009, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 13/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/007608</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-07-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04077047.1</td>
									<td>2004-07-15</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260246-biochemical-synthesis-of-1-4-butanediamine by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:12:39 GMT -->
</html>
