<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260243-recombinant-mva-viruses-expressing-clade-c-modified-hiv-env-hiv-gag-and-pol-genes by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:11:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260243:&quot;RECOMBINANT MVA VIRUSES EXPRESSING CLADE C MODIFIED HIV ENV, HIV GAG AND POL GENES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;RECOMBINANT MVA VIRUSES EXPRESSING CLADE C MODIFIED HIV ENV, HIV GAG AND POL GENES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>1 ABSTRACT A RECOMBINANT MVA VIRUSES EXPRESSING CLADE, A/G, CLADE B AND CLADE C MODIFIED HIV ENV, GAG AND POL GENES A recombinant MVA virus comprising a sequence encoding an HIV Env antigen inserted into deletion site II of an MVA wherein the HIV Env antigen comprises gpl20 and the membranespanning and ectodomain of gp41 but lacking part or all of the cytoplasmic domain of gp41, and a sequence encoding HIV Gag and Pol antigens inserted into deletion site III of an MVA, wherein the sequence encoding the HIV Gag and Pol antigens comprises safety mutations corresponding to clade B D185N, W266T, and E478Q to inactivate reverse transcriptase and delete the integrase gene; and wherein the sequences are under the control of promoters compatible with poxvirus expression systems.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>RECOMBINANT MVA VIRUSES EXPRESSING CLADE A/G, CLADE B, AND CLADE C MODIFIED HIV ENV, GAG AND POL GENES<br>
Cross-Reference to Related Applications<br>
This application claims the benefit if US Provisional Patent Application No.: 60/604,918 filed August 27, 2004, the entire disclosure of which is hereby expressly incorporated by reference.<br>
Field of the Invention<br>
The invention provides modified vaccinia Ankara (MVA), a replication-deficient strain of vaccinia virus, expressing human immunodeficiency virus (HIV) env, gag, sndpol genes.<br>
Description of the Related Art<br>
Many novel candidates and approaches have been developed in the pursuit to identify an effective human immunodeficiency virus (HTV) vaccine. The induction of strong cell-mediated immunity and broadly reactive anti-envelope antibodies that can neutralize primary HTV-1 may be necessary in the making of an effective HTV-1 vaccine, and the ideal vaccine may need to induce both T and B cell responses. Plasmid DNA vaccination can elicit both humoral and cellular immune responses (Tang et al. 1992 Nature 356: 152-154; Ulmer et al. 1993 Science 259:1745-1749; and Wolff ef al. 1990 Science 247:1465-1468) that protect nonhuman primates against challenges with nonpathogenic AIDS viruses (Boyer et al 1997 Nat. Med. 3:526-532; and Letvin et al. 1997 PNAS U.S.A. 94:9378-9383.) and afford modest protection against disease from pathogenic simian immunodeficiency virus (SW) (Egan et al. 2000 J. Virol. 74:7485-7495; and Lu et al, 1996 J. Virol. 70: 3978-3991).<br>
Vaccines that have been designed to raise cellular immunity can in some cases control virulent viral challenges and prevent the development of AIDS in rhesus macaques (Amara et al 2001 Science 292: 69-74; Barouch et al. 2000 Science 290: 486-492; Barouch et al. 2001 J. Virol. 75:5151-5158; Rose et al 2001 Cell 106:539-549; and Shiver et al. 2002 Nature 415: 331-335). Priming the immune response with plasmid DNA followed by a recombinant poxvirus booster such as the modified vaccinia virus Ankara (Amara et al 2001 Science 292:69-74 and Robinson et al 2000 AIDS Rev. 2:105-110) or a VLP-protein boost plus IL-12/GM-CSF (O'Neill et al 2003 AIDS Res. Hum.<br>
Retrovir. 19: 883-890 and O'Neill et al. 2002 J. Med. Primatol. 31: 217-227) are two of the many different approaches to raising cellular immunity. Including multiple HIV-1 gene regions on the same vector augments the DNA prime-MVA boost approach, as both gag and env responses are important to protection against virus challenge (Arnara et al. 2002 J. Virol. 76:6138-6146). Lacking this potential advantage are HTV-1 DNA vaccines that encode only Gag or encode Gag, Env, and other viral proteins on separate DNA constructs (Barouch et al. 2000 Science 290: 486^92; Barouch et al. 2002 Nature 415:335-339 and Kang et al 1999 Biol. Chem. 380: 353-364). Escape by HIV is possible when the immune response is driven by a single dominant epitope (Barouch et al. 2000 Science 290: 486-492; Barouch et al. 2002 Nature 415:335-339 and Mortara et al. 1998 J. Virol. 72: 1403-1410); thus, a multi-epitope or multi-protein response appears advantageous.<br>
It is well established mat VLP assembly and release from cells depend on proper intracellular protease regulation to preserve the Gag polyprotein (Gottlinger et al. 1989 PNAS USA 86: 5781-5785; Karacostas et al. 1993 Virology 193:661-671 and Peng et al. 1989 J. Virol. 63:2550-2556) and may be observed upon transfection of Gag-only expression systems (Huang et al. 2001 /. Virol 75: 4947-4951; Kang et al 1999 Biol Chem. 380: 353-364 and Schneider et al. 1997 J. Virol. 71:4892-4903). Previous studies demonstrated that alteration of the 25th residue of protease, from Asp (D) to Asn (N), resulted in complete loss of protease activity (Gottlinger et al 1989 PNAS USA 86: 5781-5785; Kohl et al. 1988 PNAS USA 85:4686-4690 and Loeb et al. 1989 J. Virol. 63: 111-121). Furthermore, Jacobsen et al. (1995 Virology 206:527-534) demonstrated that mutagenesis of protease at positions 48 (G48V) and 90 "(M90L) led to less efficient enzymatic activity and delayed processing of the gag and gag-pol polyproteins. Protease mutations at either amino acid positions 48 and 90 delay, but do not abolish, protease enzymatic activity, unlike the D25N mutation, and permit production of infectious virus when present within an otherwise wild-type provirus. A recent study suggested that the two protease mutations at positions 48 and 90 are not limiting, not only in terms of high-level production of HIV-1 proteins, but also with regard to assembly of VLPs (Ellenberger et al. 2004 Virology 319: 118-130).<br>
One of the most promising HIV vaccines is the heterologous prime-boost approach. The prime may consist of a recombinant plasmid DNA or viral vector-expressing HTV proteins. The heterologous boost would be a recombinant viral vector (poxvirus or adenovirus) or second viral vector, optimizing the boost for the vaccine insert immunogen<br>
and not the vector itself. Using the same recombinant viral vector for the prime and boost may lead to a diminished immune response similar to preexisting immunity to the recombinant vector (Vogels et al. 2003 J. Virol 77: 8263-8271). Priming with a recombinant plasmid DNA and boosting with a recombinant MVA enhances the cellular responses to a common immunogen contained in both vectors (Amara et al. 2001 Science 292: 69-74; Hanke et al. 1998 Vaccine 16: 439-445; Schneider et al 1998 Nat. Med. 4: 397-402; and Schneider et al. 2001 Vaccine 19: 4595-4602). At the peak effector phase of the immune response to the vaccine protocol, induced CDS cells can reach very high frequencies of total CDS cells (Allen et al 2000 J. Immunol 164:4968-4978; Amara et al 2001 Science 292: 69-74; and Horton et al 2002 J. Virol 76: 7187-7202).<br>
Summary of the Invention<br>
The invention is related to a pharmaceutical composition comprising a recombinant MVA virus expressing an HTV env, gag, and pol gene or modified gene thereof for production of an HTV Env, Gag, and Pol antigen by expression from said recombinant MVA virus, where the HIV env gene is modified to encode an HIV Env protein composed of gp!20 and the membrane-spanning and ectodomain of gp41 but lacking part or all of the cytoplasmic domain of gp41, and a pharmaceutically acceptable carrier; where the HIV env, gag, or pol gene or modified gene thereof is taken from clade AG and said HIV env gene or modified gene thereof has SEQ ID NO: 1 or a sequence having at least about 97%, 98%, 99% or 99.9% identity thereto, and the HTV gag sxidpol gene(s) or modified gene(s) thereof has SEQ ID NO: 2 or a sequence having at least about 97%, 98%, 99% or 99.9% identity thereto; or the HIV env, gag, or pol gene or modified gene thereof is taken from clade B and the HTV env gene or modified gene thereof has SEQ ID NO: 3 or a sequence having at least about 97%, 98%, 99% or 99.9% identity thereto, and the HW gag mdpol gene(s) or modified gene(s) thereof has SEQ ID NO: 4 or a sequence having at least about 97%, 98%, 99% or 99.9% identity thereto; or the HTV env, gag, or pol gene or modified gene thereof is taken from clade C and the HTV env gene or modified gene thereof has SEQ ID NO: 5 or a sequence having at least about 97%, 98%, 99% or 99.9% identity thereto, and the HTV gag and pol gene(s) or modified gene(s) thereof has SEQ ID NO: 6 or a sequence having at least about 97%, 98%, 99% or 99.9% identity thereto; and related methods of making and methods of using thereof.<br>
Brief Description of the Drawings<br>
Figure 1. Phylogenetic relationships of HIV-1 and HTV-2 based on identity ofpol gene sequences. SIVcpz and SlVsmm are subhuman primate lentiviruses recovered from a chimpanzee and sooty mangabey monkey, respectively.<br>
Figure 2. Phylogenetic relationships of HTV-1 groups M, N and O with four different SrVcpz isolates based on full-length pol gene sequences. The bar indicates a genetic distance of 0.1 (10% nucleotide divergence) and the asterisk positions group N HTV-1 isolates based on env sequences.<br>
Figure 3. Tropic and biologic properties of HTV-1 isolates.<br>
Figure 4. HTV-encoded proteins. The location of the HTV genes, the sizes of primary translation products (in some cases polyproteins), and the processed mature viral proteins are indicated.<br>
Figure 5. Schematic representation of a mature HTV-1 virion.<br>
Figure 6. Linear representation of the HTV-1 Env glycoprotein. The arrow indicates the site of gp!60 cleavage to gp!20 and gp41. In gp!20, cross-hatched areas represent variable domains (Vi to Vs) and open boxes depict conserved sequences (Ci to Cs). In the gp41 ectodomain, several domains are indicated: the N-terminal fusion peptide, and the two ectodomain helices (N- and C-helix). The membrane-spanning domain is represented by a black box. In the gp41 cytoplasmic domain, the Tyr-X-X-Leu (YXXL) endocytosis motif (SEQ ID NO: 13) and two predicted helical domains (helix-1 and -2) are shown. Amino acid numbers are indicated.<br>
Figure 7. pJD-5 transfer vector.<br>
Figure 8. pJD-6 transfer vector.<br>
Figure 9. Flow chart "Construction of recombinant MVA 65A/G".<br>
Figure 10. MVA 65A/G expression of HIV Env.<br>
Figure 11. MVA 65A/G Expression of HTV GagPol.<br>
Figure 12. MVA 65A/G Env sequence (SEQ ID NO: 1).<br>
Figure 13. MVA 65A/G GagPol Sequence (SEQ ID NO: 2).<br>
Figure 14. pJD-16 transfer vector.<br>
Figure 15. Enhanced Envelope expression from modified MVA 65A/G.<br>
Figure 16. Gag expression from MVA 65A/G constructs.<br>
Figure 17. Clade A/G gp!40 ELISA titers from immunized mice.<br>
Figure 18. Clade A p24 peptide ICS response.<br>
Figure 19. pLAS-1 HXB2/BH10 Gag Pol transfer vector. Figure 20. pLAS-2 ADA Env transfer vector.<br>
Figure 21. Flow chart "Construction of recombinant MVA 62B".<br>
Figure 22. Expression of HW Env and gag from MVA 62B.<br>
Figure 23. MVA 62B AD A Env sequence (SEQ ID NO: 3).<br>
Figure 24. HXB2/BH10 Gag Pol Sequence in MVA 62B (SEQ ID NO: 4).<br>
Figure 25.  Intracellular cytokine staining (ICS) Env and gag responses in MVA i2B-immunized mice.<br>
Figure 26. pLW-66 transfer vector.<br>
Figure 27. pJD-17 transfer vector.<br>
Figure 28. Enhanced expression of modified MVA 62B constructs.<br>
Figure 29. Expression of Gag from the modified MVA 62B constructs.<br>
Figure 30. Enhanced gpl40 ELISA titers with modified constructs in mice,<br>
Figure 31. Enhanced ICS Env response with modified constructs.<br>
Figure 32. ICS Gag response.<br>
Figure 33. pLAS-6 transfer plasmid for MVA 56.<br>
Figure 34. Env and GagPol expression by MVA recombinants MVA 48 and 56.<br>
Figure 35. pDC-3 (LAS-1 C-IN3 gag pol) plasmid.<br>
Figure 36. pJD-15 (LAS-2 C-IN3 env) plasmid.<br>
Figure 37. Flow chart "Construction of recombinant MVA/HIV 71C".<br>
Figure 38. MVA 71C expression of HIV Env and Gag Pol.<br>
Figure 39. MVA/HIV 71C Env sequence (SEQ ID NO: 5).<br>
Figure 40. MVA/HIV 71C Gag Pol sequence (SEQ ID NO: 6).<br>
Figure 41. Analysis of HIV-1-like particles. (A) immunoblot analysis of sucrose-gradient-purified VLP using rabbit polyclonal anti-p24. The positions of HIV precursors and products are indicated on the right. Fraction 9 from each gradient was separated by SDS-PAGE, transferred to nitrocellulose, and analyzed by irnmunoblotting. Vaccine construct pGAl/IC25 (complete ablation of HIV protease activity) is included as reference (Ellenberger, D. et al 2004 Virology 319:118-130). (B) summary of protease mutations introduced in vaccine constructs and demonstrable expression products. (C) transmission electron microscopy of VLPs and inrracellular protein aggregates in 293T cells transfected with vaccine constructs IC1-90 and IC48.<br>
Figure 42. Temporal IFN-y ELISPOT responses for pGAl/ICl-90- and pGAl/IC48 DNA-primed groups. Sixteen animals were tested against 19 peptide pools. Symbols designate the total ELISPOT response of each individual animal. The bold bar represents the arithmetic mean of each designated group. Peak ELISPOT response (+42 weeks; 1 week after the MVA booster), memory response (8 weeks after MVA), and late memory (26 weeks after MVA) are indicated. The background value (2x average of negative control plus 10) was subtracted from the raw counts of the peptide wells before conversion to 1 million PBMC.<br>
Figure 43. IFN-y ELISPOT responses at 1 week after the MVA booster. Numbers indicate the arithmetic means for the response to the indicated gene region. Designations below the panels indicate the gene and peptide pool used for stimulation. Designations to the right of the panels indicate the DNA prime group. Symbols indicate the individual animals. The background value was subtracted before conversion to 1 million PBMC.<br>
Figure 44. Breadth of peak ELISPOT responses for IC1-90- and IC48-primed groups. Eight animals per vaccine group were tested against 19 peptide pools (5 Gag, 6 Pol, and 7 Env, and 1 Tat). ELISPOT responses recorded as positive for each peptide pool per animal are indicated.<br>
Figure 45. ELISPOT responses determined 8 weeks after the MVA booster. Designations below the panels indicate individual animals and the DNA prime group (8 animals/group). Stacked bars indicate ELISPOT response to individual peptide pools for each animal. White bars represent Gag peptide pools, filled dark bars represent Pol peptide pools, and light-colored gray bars represent Env peptide pools. The background value was subtracted before conversion to 1 million PBMC.<br>
Figure 46. Responding CDS and CD4 cells determined by IFN-y and IL2 intracellular cytokine staining. (A) Percent-specific CDS cells responding to Gag and Env for individual animals at peak (1 week after the MVA booster). The bold bar indicates the arithmetic mean height for the response in each group. Designations above the panels specify the measured cytokine (IFN-y or IL2) and designations below the panels specify the DNA prime group. Symbols indicate the individual animals. (B) Percent-specific CD4 cells responding to Gag and Env for individual animals at peak (1 week after the MVA booster). Designations above and below the panels are the same as in panel A.<br>
Figure 47. Modifications to HTV genes. In the HIV env gene, vaccinia early termination signals were removed by making silent mutations and part of the gp 41<br>
cytoplasmic tail was truncated. In the pol gene, three mutations were made in RT and the integrase was deleted.<br>
Figure 48. Recombinant MVA/HIV construction. Single recombtnant viruses were made by homologous recombination of MVA and MVA env or gag shuttle plasmids containing transient GFP, and expression of the HTV gene controlled by the early/late modified H5 vaccinia promoter. The double recombinant was made by infecting both single recombinants at an MOI of 5 pfu/cell, and plating out the harvest to pick individual plaques. The plaques were characterized by immunostaining and those plaques expressing both Env and Gag proteins were plaque purified further to obtain the double recombinant.<br>
Deposit of Microorganism<br>
The following microorganism has been deposited in accordance with the terms of the Budapest Treaty with the American Type Culture Collection (ATCC), Manassas, VA, on the date indicated:<br>
MVA 1974/NIH Clone 1 was deposited as ATCC Accession No.: PTA-5095 on March 27, 2003 with the American Type Culture Collection (ATCC), 10801 University Blvd., Manassas, VA 20110-2209, USA. This deposit was made under the provisions of the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure and the Regulations thereunder (Budapest Treaty). This assures maintenance of a viable culture of the deposit for 30 years from date of deposit. The deposit will be made available by ATCC under the terms of the Budapest Treaty, and subject to an agreement between Applicant and ATCC which assures permanent and unrestricted availability of the progeny of the culture of the deposit to the public upon issuance of the pertinent U.S. patent or upon laying open to the public of any U.S. or foreign patent application, whichever comes first, and assures availability of the progeny to one determined by the U.S. Commissioner of Patents and Trademarks to be entitled thereto according to 35 USC § 122 and the Commissioner's rules pursuant thereto (including 37 CFR § 1.14). Availability of the deposited strain is not to be construed as a license to practice the invention in contravention of the rights granted under the authority of any government in accordance with its patent laws.<br>
Detailed Description of the Preferred Embodiment Recombinant MVA Virus<br>
Vaccinia virus, a member of the genus Orthopoxvirus in the family of Poxviridae, was used as live vaccine to immunize against the human smallpox disease. Successful worldwide vaccination with vaccinia virus culminated in the eradication of variola virus, the causative agent of the smallpox ("The global eradication of smallpox. Final report of the global commission for the certification of smallpox eradication". History of Public Health, No. 4, Geneva: World Health Organization, 1980). Since that WHO declaration, vaccination has been universally discontinued except for people at high risk of poxvirus infections (e.g. laboratory workers).<br>
More recently, vaccinia viruses have also been used to engineer viral vectors for recombinant gene expression and for the potential use as recombinant live vaccines (Mackett, M. et al 1982 PNAS USA 79:7415-7419; Smith, G.L. et al. 1984 Biotech Genet Engin Rev 2:383-407). This entails DNA sequences (genes) which code for foreign. antigens being introduced, with the aid of DNA recombination techniques, into the genome of the vaccinia viruses. If the gene is integrated at a site in the viral DNA which is non-essential for the life cycle of the virus, it is possible for the newly produced recombinant vaccinia virus to be infectious, mat is to say able to infect foreign cells and thus to express the integrated DNA sequence (EP Patent Applications No. 83,286 and No. 110,385). The recombinant vaccinia viruses prepared in this way can be used, on the one hand, as live vaccines for the prophylaxis of infectious diseases, on the other hand, for the preparation of heterologous proteins in eukaryotic cells.<br>
For vector applications health risks would be lessened by the use of a highly attenuated vaccinia virus strain. Several such strains of vaccinia virus were especially developed to avoid undesired side effects of smallpox vaccination. Thus, the modified vaccinia Ankara (MVA) has been generated by long-term serial passages of the Ankara strain of vaccinia virus (CVA) on chicken embryo fibroblasts (for review see Mayr, A. et al 1975 Infection 3:6-14; Swiss Patent No. 568,392). The MVA virus is publicly available from American Type Culture Collection as ATCC No.: VR-1508. MVA is distinguished by its great attenuation, that is to say by diminished virulence and ability to replicate in primate cells while maintaining good immunogenicity. The MVA virus has been analyzed to determine alterations in the genome relative to the parental CVA strain. Six major deletions of genomic DNA (deletion I, n, HI, IV, V, and VI) totaling 31,000 base pairs have<br>
been identified (Meyer, H. et al. 1991 J Gen Virol 72:1031-1038).   The resulting MVA virus became severely host cell restricted to avian cells.<br>
Furthermore, MVA is characterized by its extreme attenuation. When tested in a variety of animal models, MVA was proven to be avirulent even in immunosuppressed animals. More importantly, the excellent properties of the MVA strain have been demonstrated hi extensive clinical trials (Mayr A. et al. 1978 Zentralbl Bakteriol [B] 167:375-390; Stickl et al. 1974 Dtsch Med Wschr 99:2386-2392). During these studies in over 120,000 humans, including high-risk patients, no side effects were associated with the use of MVA vaccine.<br>
MVA replication in human cells was found to be blocked late in infection preventing the assembly to mature infectious virions. Nevertheless, MVA was able to express viral and recombinant genes at high levels even in non-permissive cells and was proposed to serve as an efficient and exceptionally safe gene expression vector (Sutter, G. and Moss, B. 1992 PNAS USA 89:10847-10851). Additionally, novel vaccinia vector vaccines were established on the basis of MVA having foreign DNA sequences inserted at the site of deletion Ef within the MVA genome (Sutter, G. et al. 1994 Vaccine 12:1032-1040).<br>
The recombinant MVA vaccinia viruses can be prepared as set out hereinafter. A DNA-construct which contains a DNA-sequence which codes for a foreign polypeptide flanked by MVA DNA sequences adjacent to a naturally occurring deletion, e.g. deletion HI, or other non-essential sites, within the MVA genome, is introduced into cells infected with MVA, to allow homologous recombination. Once the DNA-construct has been introduced into the eukaryotic cell and the foreign DNA has recombined with the viral DNA, it is possible to isolate the desired recombinant vaccinia virus in a manner known per se, preferably with the aid of a marker. The DNA-construct to be inserted can be linear or circular. A plasmid or polymerase chain reaction product is preferred. The DNA-construct contains sequences flanking the left and the right side of a naturally occurring deletion, e.g. deletion HI, within the MVA genome. The foreign DNA sequence is inserted between the sequences flanking the naturally occurring deletion. For the expression of a DNA sequence or gene, it is necessary for regulatory sequences, which are required for the transcription of the gene, to be present on the DNA. Such regulatory sequences (called promoters) are known to those skilled in the art, and include for example those of the vaccinia 11 IcDa gene as are described in EP-A-198,328, and those of the 7.5 kDa gene (EP-A-110,385). The<br>
been identified (Meyer, H. et al. 1991 J Gen Virol 72:1031-1038).   The resulting MVA virus became severely host cell restricted to avian cells.<br>
Furthermore, MVA is characterized by its extreme attenuation. When tested in a variety of animal models, MVA was proven to be avirulent even in immunosuppressed animals. More importantly, the excellent properties of the MVA strain have been demonstrated in extensive clinical trials (Mayr A. et al. 1978 Zentralbl Bakteriol [B] 167:375-390; Stickl et al. 1974 Dtsch Med Wschr 99:2386-2392). During these studies in over 120,000 humans, including high-risk patients, no side effects were associated with the use of MVA vaccine.<br>
MVA replication in human cells was found to be blocked late in infection preventing the assembly to mature infectious virions. Nevertheless, MVA was able to express viral and recombinant genes at high levels even in non-permissive cells and was proposed to serve as an efficient and exceptionally safe gene expression vector (Sutter, G. and Moss, B. 1992 PNAS USA 89:10847-10851). Additionally, novel vaccinia vector vaccines were established on the basis of MVA having foreign DNA sequences inserted at the site of deletion m within the MVA genome (Sutter, G. et al. 1994 Vaccine 12:1032-1040).<br>
The recombinant MVA vaccinia viruses can be prepared as set out hereinafter. A DNA-construct which contains a DNA-sequence which codes for a foreign polypeptide flanked by MVA DNA sequences adjacent to a naturally occurring deletion, e.g. deletion ffi, or other non-essential sites, within the MVA genome, is introduced into cells infected with MVA, to allow homologous recombination. Once the DNA-construct has been introduced into the eukaryotic cell and the foreign DNA has recombined with the viral DNA, it is possible to isolate the desired recombinant vaccinia virus in a manner known per se, preferably with the aid of a marker. The DNA-construct to be inserted can be linear or circular. A plasmid or polymerase chain reaction product is preferred. The DNA-construct contains sequences flanking the left and the right side of a naturally occurring deletion, e.g. deletion ffl, within the MVA genome. The foreign DNA sequence is inserted between the sequences flanking the naturally occurring deletion. For the expression of a DNA sequence or gene, it is necessary for regulatory sequences, which are required for the transcription of the gene, to be present on the DNA. Such regulatory sequences (called promoters) are known to those skilled in the art, and include for example those of the vaccinia 11 kDa gene as are described in EP-A-198,328, and those of the 7.5 kDa gene (EP-A-110,385). The<br>
All of the HTV-1 group M subtypes can be found in Africa. Clade A viruses are genetically the most divergent and were the most common HIV-1 subtype in Africa early in the epidemic. With the rapid spread of HIV-1 to southern Africa during the mid to late 1990s, clade C viruses have become the dominant subtype and now account for 48% of HIV-1 infections worldwide. Clade B viruses, the most intensively studied HTV-1 subtype, remain the most prevalent isolates in Europe and North America.<br>
High rates of genetic recombination are a hallmark of retroviruses. It was initially believed that simultaneous infections by genetically diverse virus strains were not likely to be established in individuals at risk for HTV-1. By 1995, however, it became apparent that a significant fraction of the HIV-1 group M global diversity included interclade viral recombinants. It is now appreciated that HIV-1 recombinants will be found in geographic areas such as Africa, South America, and Southeast Asia, where multiple HIV-1 subtypes coexist and may account for more than 10% of circulating HTV-1 strains. Molecularly, the genomes of these recombinant viruses resemble patchwork mosaics, with juxtaposed diverse HIV-1 subtype segments, reflecting the multiple crossover events contributing to their generation. Most HIV-1 recombinants have arisen in Africa and a majority contains segments originally derived from clade A viruses. In Thailand, for example, the composition of the predominant circulating strain consists of a clade A gag plus pol gene segment and a clade E env gene. Because the clade E env gene in Thai HIV-1 strains is closely related to the clade E env present in virus isolates from the Central African Republic, it is believed that the original recombination event occurred in Africa, with the subsequent introduction of a descendent virus into Thailand. Interestingly, no full-length HTV-1 subtype E isolate (i.e., with subtype E gag, pol, and env genes) has been reported to date.<br>
The discovery that a and P chemokine receptors function as coreceptors for virus fusion and entry into susceptible CD4+ cells-has led to a revised classification scheme for HTV-1 (Fig. 3). Isolates can now be grouped on the basis of chemokine receptor utilization in fusion assays in which HTV-1 gp!20 and CD4+ coreceptor proteins are expressed in separate cells. As indicated in Figure 3, HTV-1 isolates using the CXCR4 receptor (now designated X4 viruses) are usually T cell line (TCL)-tropic syncytium inducing (SI) strains, whereas those exclusively utilizing the CCR5 receptor (R5 viruses) are predominantly macrophage (M)-tropic and non-syncytium inducing (NST). The dual-tropic R5/X4 strains,<br>
which may comprise the majority of patient isolates and exhibit a continuum of tropic phenotypes, are frequently SI.<br>
As is the case for all replication-competent retroviruses, the three primary HTV-1 translation products, all encoding structural proteins, are initially synthesized as polyprotein precursors, which are subsequently processed by viral or cellular proteases into mature particle-associated proteins (Fig. 4). The 55-kd Gag precursor Pr55Gag is cleaved into the matrix (MA), capsid (CA), nucleocapsid (NC), and p6 proteins. Autocatalysis of the 160-kd Gag-Pol polyprotein, Prl60Gag"p°', gives rise to the protease (PR), the heterodimeric reverse transcriptase (RT), and the integrase (IN) proteins, whereas proteolytic digestion by a cellular enzyme(s) converts the glycosylated 160-lcd Env precursor gp!60 to the gp!20 surface (SU) and gp41 transmembrane (TM) cleavage products. The remaining six HTV-1-encoded proteins (Vif, Vpr, Tat, Rev, Vpu, and Nef) axe the primary translation products of spliced mRNAs. Gag<br>
The Gag proteins of HIV, like those of other retroviruses, are necessary and sufficient for the formation of noninfectious, virus-like particles. Retroviral Gag proteins are generally synthesized as polyprotein precursors; the HIV-1 Gag precursor has been named, based on its apparent molecular mass, Pr55Gag. As noted previously, the mRNA for Pr55Gas is the unspliced 9.2-kb transcript (Fig. 4) that requires Rev for its expression in the cytoplasm. When ihepol ORF is present, the viral protease (PR) cleaves Pr55Gag during or shortly after budding from the cell to generate the mature Gag proteins p!7 (MA), p24 (CA), p7 (NC), and p6 (see Fig. 4). In the virion, MA is localized immediately inside the lipid bilayer of the viral envelope, CA forms the outer portion of the cone-shaped core structure in the center of the particle, and NC is present in the core in a ribonucleoprotein complex with the viral RNA genome (Fig. 5).<br>
The HTV Pr55Ga8 precursor oligomerizes following its translation and is targeted to the plasma membrane, where particles of sufficient size and density to be visible by EM are assembled. Formation of virus-like particles by Pr55Gag is a self-assembly process, with critical Gag-Gag interactions taking place between multiple domains along the Gag precursor. The assembly of virus-like particles does not require the participation of genomic RNA (although the presence of nucleic acid appears to be essential), encoded enzymes, or Env glycoproteins, but the production of infectious virions requires the<br>
encapsidation of the viral RNA genome and the incorporation of the Env glycoproteins and<br>
the Gag-Pol polyprotein precursor Prl60Gag"po1.<br>
Pol<br>
Downstream of gag lies the most highly conserved region of the HIV genome, the pol gene, which encodes three enzymes: PR, RT, and IN (see Fig. 4). RT and IN are required, respectively, for reverse transcription of the viral RNA genome to a double-stranded DNA copy, and for the integration of the viral DNA into the host cell chromosome. PR plays a critical role late in the life cycle by mediating the production of mature, infectious virions. The pol gene products are derived by enzymatic cleavage of a 160-kd Gag-Pol fusion protein, referred to as Prl60Gas"Po1. This fusion protein is produced by ribosomal frameshifting during translation of Pr55Gag (see Fig, 4). The frame-shifting mechanism for Gag-Pol expression, also utilized by many other retroviruses, ensures that the/JoZ-derived proteins are expressed at a low level, approximately 5% to 10% that of Gag. Like Pr55Gag, the N-terminus of Prl60Gag"po1 is myristylated and targeted to the plasma membrane. Protease<br>
Early pulse-chase studies perfomied with avian retroviruses clearly indicated that retroviral Gag proteins are initially synthesized as polyprotein precursors that are cleaved to generate smaller products. Subsequent studies demonstrated that the processing function is provided by a viral rather than a cellular enzyme, and that proteolytic digestion of the Gag and Gag-Pol precursors is essential for virus infectivity. Sequence analysis of retroviral PRs indicated that they are related to cellular "aspartic" proteases such as pepsin and renin. Like these cellular enzymes, retroviral PRs use two apposed Asp residues at the active site to coordinate a water molecule that catalyzes the hydrolysis of a peptide bond in the target protein. Unlike the cellular aspartic proteases, which function as pseudodimers (using two folds within the same molecule to generate the active site), retroviral PRs function as true dimers. X-ray crystallographic data from HIV-1 PR indicate that the two monomers are held together in part by a four-stranded antiparallel p-sheet derived from both N- and C-terminal ends of each monomer. The substrate-binding site is located within a cleft formed between the two monomers. Like their cellular homologs, the HTV PR dimer contains flexible "flaps" that overhang the binding site and may stabilize the substrate within the cleft; the active-site Asp residues lie in the center of the dimer. Interestingly, although some limited amino acid homology is observed surrounding active-site residues, the<br>
primary sequences of retroviral PRs are highly divergent, yet their structures are remarkably<br>
similar.<br>
Reverse Transcriptase<br>
By definition, retro-viruses possess the ability to convert their single-stranded RNA genomes into double-stranded DNA during the early stages of the infection process. The enzyme that catalyzes this reaction is RT, in conjunction with its associated RNaseH activity. Retroviral RTs have three enzymatic activities: (a) RNA-directed DNA polymerization (for minus-strand DNA synthesis), (b) RNaseH activity (for the degradation of the tRNA primer and geuomic RNA present in DNA-RNA hybrid intermediates), and (c) DNA-directed DNA polymerization (for second- or plus-strand DNA synthesis).<br>
The mature HIV-1 RT holoenzyme is a heterodimer of 66 and 51 ltd subunits. The 51-kd subunit (p51) is derived from the 66-kd (p66) subunit by proteolytic removal of the C-terminal 15-kd RNaseH domain of p66 by PR (see Fig. 4). The crystal structure of HTV-1 RT reveals a highly asymmetric folding in which the orientations of the p66 and p51 subunits differ substantially. The p66 subunit can be visualized as a right hand, with the polymerase active site within the palm, and a deep template-binding cleft formed by the palm, fingers, and thumb subdomains. The polymerase domain is linked to RNaseH by the connection subdomain. The active site, located in the palm, contains three critical Asp residues (110, 185, and 186) in close proximity, and two coordinated Mg2+ ions. Mutation of these Asp residues abolishes RT polymerizing activity. The orientation of the three active-site Asp residues is similar to that observed in other DNA polymerases (e.g., the Klenow fragment of E. coli DNApo/I). The p51 subunit appears to be rigid and does not form a polymerizing cleft; Asp 110, 185, and 186 of this subunit are buried within the molecule. Approximately 18 base pairs of the primer-template duplex lie in the nucleic acid binding cleft, stretching from the polymerase active site to the RNaseH domain.<br>
In the RT-primer-template-dNTP structure, the presence of a dideoxynucleotide at the 3' end of the primer allows visualization of the catalytic complex trapped just prior to attack on the incoming dNTP. Comparison with previously obtained structures suggests a model whereby the fingers close in to trap the template and dNTP prior to nucleophilic attack of the 3'-OH of the primer on the incoming dNTP. After the addition of the incoming dNTP to the growing chain, it has been proposed that the fingers adopt a more open configuration, thereby releasing the pyrophosphate and enabling RT to bind the next<br>
dNTP.     The  structure of the HTV-1 RNaseH has also  been determined by x-ray crystallography; this domain displays a global folding similar to that of E. coli RNaseH. Integrase<br>
A distinguishing feature of retrovirus replication is the insertion of a DNA copy of the viral genome into the host cell chromosome following reverse transcription. The integrated viral DNA (the provirus) serves as the template for the synthesis of viral RNAs and is maintained as part of the host cell genome for the lifetime of the infected cell. Retroviral mutants deficient in the ability to integrate generally fail to establish a productive infection.<br>
The integration of viral DNA is catalyzed by integrase, a 32-kd protein generated by PR-mediated cleavage of the C-terminal portion of the HTV-1 Gag-Pol polyprotein (see Fig.<br>
4).<br>
Retroviral IN proteins are composed of three structurally and functionally distinct domains: an N-terminal, zinc-finger-containing domain, a core domain, and a relatively nonconserved C-terminal domain. Because of its low solubility, it has not yet been possible to crystallize the entire 288-amino-acid HTV-1 IN protein. However, the structure of all three domains has been solved independently by x-ray crystallography or NMR methods. The crystal structure of the core domain of the avian sarcoma virus IN has also been determined. The N-terminal domain (residues 1 to 55), whose structure was solved by NMR spectroscopy, is composed of four helices with a zinc coordinated by amino acids His-12, His-16, Cys-40, and Cys-43. The structure of the N-terminal domain is reminiscent of helical DNA binding proteins that contain a so-called helix-turn-helix motif; however, in the HIV-1 structure mis motif contributes to dimer formation. Initially, poor solubility hampered efforts to solve the structure of the core domain. However, attempts at crystallography were successful when it was observed that a Phe-to-Lys change at DSf residue 185 greatly increased solubility without disrupting in vitro catalytic activity. Each monomer of the HIV-1 IN core domain (IN residues 50 to 212) is composed of a five-stranded (3-sheet flanked by helices; this structure bears striking resemblance to other polynucleotidyl transferases including RNaseH and the bacteriophage MuA transposase. Three highly conserved residues are found in analogous positions in other polynucleotidyl transferases; in HTV-1 IN these are Asp-64, Asp-116 and Glu-152, the so-called D,D-35-E motif. Mutations at these positions block HTV IN function both in vivo and in vitro. The close proximity of these three amino acids in the crystal structure of both avian sarcoma<br>
virus and HTV-1 core domains supports the hypothesis that these residues play a central role in catalysis of the polynucleotidyl transfer reaction that is at the heart of the integration process. The C-terminal domain, whose structure has been solved by NMR methods, adopts a five-stranded p-barrel folding topology reminiscent of a Src homology 3 (SH3) domain. Recently, the x-ray structures of SIV and Rous sarcoma virus IN protein fragments encompassing both the core and C-terminal domains have been solved. Env<br>
The HIV Env glycoproteins play a major role in the virus life cycle. They contain the determinants that interact with the CD4 receptor and coreceptor, and they catalyze the fusion reaction between the lipid bilayer of the viral envelope and the host cell plasma membrane. In addition, the HIV Env glycoproteins contain epitopes that elicit immune responses that are important from both diagnostic and vaccine development perspectives.<br>
The HIV Env glycoprotein is synthesized from the singly spliced 4.3-kb Vpu/Env bicistronic mRNA (see Fig. 4); translation occurs on ribosomes associated with the rough endoplasmic reticulum (ER). The 160-kd polyprotein precursor (gp!60) is an integral membrane protein that is anchored to cell membranes by a hydrophobic stop-transfer signal in the domain destined to be the mature TM Env glycoprotein, gp41 (Fig. 6). The gp!60 is cotranslationally glycosylated, forms disulfide bonds, and undergoes oligomerization in the ER. The predominant oligomeric form appears to be a trimer, although dimers and tetramers are also observed. The gp!60 is transported to the Golgi, where, like other retroviral envelope precursor proteins, it is proteolytically cleaved by cellular enzymes to the mature SU glycoprotein gp!20 and TM glycoprotein gp41 (see Fig. 6). The cellular enzyme responsible for cleavage of retroviral Env precursors following a highly conserved Lys/Arg-X-Lys/Arg-Arg motif is furin or a furin-like protease, although other enzymes may also catalyze gp!60 processing. Cleavage of gp!60 is required for Env-induced fusion activity and virus infectivity. Subsequent to gp!60 cleavage, gpl20 and gp41 form a noncovalent association that is critical for transport of the Env complex from the Golgi to the cell surface. The gp!20-gp41 interaction is fairly weak, and a substantial amount of gp!20 is shed from the surface of Env-expressing cells.<br>
The HIV Env glycoprotein complex, in particular the SU (gp!20) domain, is very heavily glycosylated; approximately half the molecular mass of gp!60 is composed of oligosaccharide side chains. During transport of Env from its site of synthesis in the ER to the plasma membrane, many of the side chains are modified by the addition of complex<br>
sugars. The numerous oligosaccharide side chains form what could be imagined as a sugar cloud obscuring much of gpl20 from host immune recognition. As shown in Figure 6, gp!20 contains interspersed conserved (Q to C5) and variable (Vi to V5) domains. The Cys residues present in the gp!20s of different isolates are highly conserved and form disulfide bonds that link the first four variable regions in large loops.<br>
A primary function of viral Env glycoproteins is to promote a membrane fusion reaction between the lipid bilayers of the viral envelope and host cell membranes. This membrane fusion event enables the viral core to gain entry into the host cell cytoplasm. A number of regions in both gp!20 and gp41 have been implicated, directly or indirectly, in Env-mediated membrane fusion. Studies of the HAa hemagglutinin protein of the orthomyxoviruses and the F protein of the paramyxoviruses indicated that a highly hydrophobic domain at the N-terminus of these proteins, referred to as the fusion peptide, plays a critical role in membrane fusion. Mutational analyses demonstrated that an analogous domain was located at the N-tenninus of the HTV-1, HIV-2, and SIV TM glycoproteins (see Fig. 6). Nonhydrophobic substitutions within this region of gp41 greatly reduced or blocked syncytium formation and resulted in the production of noninfectious progeny virions.<br>
C-terminal to the gp41 fusion peptide are two amphipathic helical domains (see Fig. 6) which play a central role in membrane fusion. Mutations in the N-terminal helix (referred to as the N-helix), which contains a Leu zipper-like heptad repeat motif, impair infectivity and membrane fusion activity, and peptides derived from these sequences exhibit potent antiviral activity in culture. The structure of the ectodomain of HTV-1 and SIV gp41, the two helical motifs in particular, has been the focus of structural analyses in recent years. Structures were determined by x-ray crystallography or NMR spectroscopy either for fusion proteins containing the helical domains, a mixture of peptides derived from the N- and C-helices, or in the case of the SIV structure, the intact gp41 ectodomain sequence from residue 27 to 149. These studies obtained fundamentally similar trimeric structures, in which the two helical domains pack in an antiparallel fashion to generate a six-helix bundle. The N-helices form a coiled-coil in the center of the bundle, with the C-helices packing into hydrophobic grooves on the outside.<br>
In the steps leading to membrane fusion CD4 binding induces conformation changes in Env that facilitate coreceptor binding. Following the formation of a ternary gp!20/CD4/coreceptor complex, gp41 adopts a hypothetical conformation that allows the<br>
fusion peptide to insert into the target lipid bilayer.  The formation of the gp41 six-helix bundle (which involves antiparallel interactions between the gp41 N- and C-helices) brings the viral and cellular membranes together and membrane fusion takes place, Use of Recombinant MVA Virus To Boost CD+8 Cell Immune Response<br>
The present invention relates to generation of a CD8+ T cell immune response against an antigen and also eliciting an antibody response. More particularly, the present invention relates to "prime and boost" immunization regimes in which the immune response induced by administration of a priming composition is boosted by administration of a boosting composition. The present invention is based on inventors' experimental demonstration that effective boosting can be achieved using modified vaccinia Ankara (MVA) vectors, following priming with any of a variety of different types of priming compositions including recombinant MVA itself.<br>
A major protective component of the immune response against a number of pathogens is mediated by T lymphocytes of the CD8+ type, also known as cytotoxic T lymphocytes (CTL). An important function of CD8+ cells is secretion of gamma interferon (DFNy), and this provides a measure of CD8+ T cell immune response. A second component of the immune response is antibody directed to the proteins of the pathogen.<br>
The present invention employs MVA which, as the experiments described below show, has been found to be an effective means for providing a boost to a CD8+ T cell immune response primed to antigen using any of a variety of different priming compositions and also eliciting an antibody response.<br>
Remarkably, the experimental work described below demonstrates that use of embodiments of the present invention allows for recombinant MVA virus expressing an HIV antigen to boost a CD8+ T cell immune response primed by a DNA vaccine and also eliciting an antibody response. The MVA was found to induce a CDS T cell response after intramuscular immunization.<br>
Based on previous work (Amara et al 2001 Science 292:69-74), it is predicted that non-human primates immunized with plasmid DNA and boosted with the MVA would effectively protect against intramucosal challenge with live virus. Advantageously, the inventors contemplate that a vaccination regime using intradermal, intramuscular or mucosal immunization for both prime and boost can be employed, constituting a general immunization regime suitable for inducing CD8+ T cells and also eliciting an antibody response, e.g. in humans.The present invention in various aspects and embodiments employs an MVA vector encoding an HIV antigen for boosting a CD8+ T cell immune response to the antigen primed by previous administration of nucleic acid encoding the antigen and also eliciting an antibody response.<br>
A general aspect of the present invention provides for the use of an MVA vector for boosting a CD8+ T cell immune response to an HIV antigen and also eliciting an antibody response.<br>
One aspect of the present invention provides a method of boosting a CD8+ T cell immune response to an HTV antigen in an individual, and also eliciting an antibody response, the method including provision in the individual of an MVA vector including nucleic acid encoding the antigen operably linked to regulatory sequences for production of antigen in the individual by expression from the nucleic acid, whereby a CDS"1" T cell immune response to the antigen previously primed in the individual is boosted.<br>
An immune response to an HIV antigen may be primed by immunization with plasrnid DNA or by infection with an infectious agent.<br>
A further aspect of the invention provides a method of inducing a CD8+ T cell immune response to an HTV antigen in an individual, and also eliciting an antibody response, the method comprising administering to the individual a priming composition comprising nucleic acid encoding the antigen and then administering a boosting composition which comprises an MVA vector including nucleic acid encoding the antigen operably linked to regulatory sequences for production of antigen in the individual by expression from the nucleic acid.<br>
A further aspect provides for use of an MVA vector, as disclosed, in the manufacture of a medicament for administration to a mammal to boost a CD8+ T cell immune response to an HTV antigen, and also eliciting an antibody response. Such a medicament is generally for administration following prior administration of a priming composition comprising nucleic acid encoding the antigen.<br>
The priming composition may comprise any viral vector, such as a vaccinia virus vector such as a replication-deficient strain such as modified vaccinia Ankara (MVA) or NYVAC (Tartaglia et al 1992 Virology 118:217-232), an avipox vector such as fowlpox or canarypox, e.g. the strain known as ALVAC (Paoletti et al 1994 Dev Biol Stand 82:65-69), or an adenovirus vector or a vesicular stomatitis virus vector or an alphavirus vector.<br>
The priming composition may comprise DNA encoding the antigen, such DNA preferably being in the form of a circular plasmid that is not capable of replicating in mammalian cells. Any selectable marker should not be resistance to an antibiotic used clinically, so for example Kanamycin resistance is preferred to Ampicillin resistance. Antigen expression should be driven by a promoter which is active in mammalian cells, for instance the cytomegalovirus immediate early (CMV IE) promoter.<br>
In particular embodiments of the various aspects of the present invention, administration of a priming composition is followed by boosting with a boosting composition, or first and second boosting compositions, the first and second boosting compositions being the same or different from one another. Still further boosting compositions may be employed without departing from the present invention. In one embodiment, a triple immunization regime employs DNA, then adenovirus as a first boosting composition, then MVA as a second boosting composition, optionally followed by a further (third) boosting composition or subsequent boosting administration of one or other or both of the same or different vectors. Another option is DNA then MVA then adenovirus, optionally followed by subsequent boosting administration of one or other or both of the same or different vectors.<br>
The antigen to be encoded in respective priming and boosting compositions (however many boosting compositions are employed) need not be identical, but should share at least one CDS"1" T cell epitope. The antigen may correspond to a complete antigen, or a fragment thereof. Peptide epitopes or artificial strings of epitopes may be employed, more efficiently cutting out unnecessary protein sequence in the antigen and encoding sequence in the vector or vectors. One or more additional epitopes may be included, for instance epitopes which are recognized by T helper cells, especially epitopes recognized in individuals of different HLA types. Priming and boosting may be administered intradermally, intramuscularly or mucosally.<br>
An HTV antigen of the invention to be encoded by a recombinant MVA virus includes polypeptides having immunogenic activity elicited by an amino acid sequence of an HIV Env, Gag, Pol, Vif, Vpr, Tat, Rev, Vpu, or Nef amino acid sequence as at least one CD8+ T cell epitope. This amino acid sequence substantially corresponds to at least one 10-900 amino acid fragment and/or consensus sequence of a known HIV Env or Pol; or at least one 10-450 amino acid fragment and/or consensus sequence of a known HTV Gag; or<br>
at least one 10-100 amino acid fragment and/or consensus sequence of a known HTV Vif, Vpr, Tat, Rev, Vpu, or Nef.<br>
Although a Ml length Env precursor sequence is presented for use in the present invention, Env is optionally deleted of subsequences. For example, regions of the gp!20 surface and gp41 transmembrane cleavage products can be deleted.<br>
Although a full length Gag precursor sequence is presented for use in the present invention, Gag is optionally deleted of subsequences. For example, regions of the matrix protein (pi 7), regions of the capsid protein (p24), regions of the nucleocapsid protein (p7), and regions of p6 (the C-terminal peptide of the Gag polyprotein) can be deleted.<br>
Although a full length Pol precursor sequence is presented for use in the present invention, Pol is optionally deleted of subsequences. For example, regions of the protease protein (plO), regions of the reverse transcriptase protein (p66/p51), and regions of the integrase protein (p32) can be deleted.<br>
Such an HTV Env, Gag, or Pol can have overall identity of at least 50% to a known Env, Gag, or Pol protein amino acid sequence, such as 50-99% identity, or any range or value therein, while eliciting an immunogenic response against at least one strain of an HTV.<br>
Percent identify can be determined, for example, by comparing sequence information using the GAP computer program, version 6.0, available from the University of Wisconsin Genetics Computer Group (UWGCG). The GAP program utilizes the alignment method of Needleman and Wunsch (J Mol Biol 1970 48:443), as revised by Smith and Waterman (Adv Appl Math 1981 2:482). Briefly, the GAP program defines identity as the number of aligned symbols (i.e., nucleotides or amino acids) which are identical, divided by the total number of symbols in the shorter of the two sequences. The preferred default parameters for the GAP program include: (1) a unitary comparison matrix (containing a value of 1 for identities and 0 for non-identities) and the weighted comparison matrix of Gribskov and Burgess (Nucl Acids Res 1986 14:6745), as described by Schwartz and Dayhoff (eds., Atlas of Protein Sequence and Structure, National Biomedical Research Foundation, Washington, D.C. 1979, pp. 353-358); (2) a penalty of 3.0 for each gap and an additional 0.10 penalty for each symbol in each gap; and (3) no penalty for end gaps.<br>
ha a preferred embodiment, an Env of the present invention is a variant form of at least one HIV envelope protein. Preferably, the Env is composed of gpl20 and the<br>
membrane-spanning and ectodomain of gp41 but lacks part or all of the cytoplasmic domain of gp41.<br>
Known HTV sequences are readily available from commercial and institutional HTV sequence databases, such as GENBANK, or as published compilations, such as Myers et al. eds., Human Retroviruses and AIDS, A Compilation and Analysis of Nucleic Acid and Amino Acid Sequences, Vol. I and II, Theoretical Biology and Biophysics, Los Alamos, N. Mex. (1993), or http://hiv-web.lanl.gov/.<br>
Substitutions or insertions of an HIV Env, Gag, or Pol to obtain an additional HTV Env, Gag, or Pol, encoded by a nucleic acid for use in a recombinant MVA virus of the present invention, can include substitutions or insertions of at least one amino acid residue (e.g., 1-25 amino acids). Alternatively, at least one amino acid (e.g., 1-25 amino acids) can be deleted from an HIV Env, Gag, or Pol sequence. Preferably, such substitutions, insertions or deletions are identified based on safety features, expression levels, immunogenicity and compatibility with high replication rates of MVA.<br>
Amino acid sequence variations in an HIV Env, Gag, or Pol of the present invention can be prepared e.g., by mutations in the DNA. Such HIV Env, Gag, or Pol include, for example, deletions, insertions or substitutions of nucleotides coding for different amino acid residues within the amino acid sequence. Obviously, mutations that will be made in nucleic acid encoding an HPV Env, Gag, or Pol must not place the sequence out of reading frame and preferably will not create complementary domains that could produce secondary mRNA structures.<br>
HIV Env, Gag, or Pol-encoding nucleic acid of the present invention can also be prepared by amplification or site-directed mutagenesis of nucleotides in DNA or RNA encoding an HIV Env, Gag, or Pol and thereafter synthesizing or reverse transcribing the encoding DNA to produce DNA or RNA encoding an HTV Env, Gag, or Pol, based on the teaching and guidance presented herein.<br>
Recombinant MVA viruses expressing HIV Env, Gag, or Pol of the present invention, include a finite set of HIV Env, Gag, or Pol-encoding sequences as substitution nucleotides that can be routinely obtained by one of ordinary skill in the art, without undue experimentation, based on the teachings and guidance presented herein. For a detailed description of protein chemistry and structure, see Schulz, G.E. et al., 1978 Principles of Protein Structure, Springer-Verlag, New York, N.Y., and Creighton, T.E., 1983 Proteins: Structure and Molecular Properties, W. H. Freeman &amp; Co., San Francisco, CA. For a<br>
presentation of nucleotide sequence substitutions, such as codon preferences, see Ausubel et al. eds. Current Protocols in Molecular Biology, Greene Publishing Assoc., New York, N.Y. 1994 at §§ 'A.l.l-A.1.24, and Sambrook, J. et al. 1989 Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold. Spring Harbor, N.Y. at Appendices C and D.<br>
Thus, one of ordinary skill in the art, given the teachings and guidance presented herein, will know how to substitute other amino acid residues in other positions of an HIV env, gag, or pol DNA or RNA to obtain alternative HTV Env, Gag, or Pol, including substirutional, deletional or insertional variants.<br>
Within the MVA vector, regulatory sequences for expression of the encoded antigen will include a natural, modified or synthetic poxvirus promoter. By "promoter" is meant a sequence of nucleotides from which transcription may be initiated of DNA operably linked downstream (i.e. in the 3' direction on the sense strand of double-stranded DNA). "Operably linked" means joined as part of the same nucleic acid molecule, suitably positioned and oriented for transcription to be initiated from the promoter. DNA operably linked to a promoter is "under transcriptional initiation regulation" of the promoter. Other regulatory sequences including terminator fragments, polyadenylation sequences, marker genes and other sequences may be included as appropriate, in accordance with the knowledge and practice of the ordinary person skilled in the art: see, for example, Moss, B. (2001). Poxviridae: the viruses and their replication. In Fields Virology, D.M. Knipe, and P.M. Howley, eds. (Philadelphia, Lippincott Williams &amp; Wilkins), pp. 2849-2883. Many known techniques and protocols for manipulation of nucleic acid, for example in preparation of nucleic acid constructs, mutagenesis, sequencing, introduction of DNA into cells and gene expression, and analysis of proteins, are described in detail in Current Protocols in Molecular Biology, 1998 Ausubel et al. eds., John Wiley &amp; Sons.<br>
Promoters for use in aspects and embodiments of the present invention must be compatible with poxvirus expression systems and include natural, modified and synthetic sequences.<br>
Either or both of the priming and boosting compositions may include an adjuvant, such as granulocyte macrophage-colony stimulating factor (GM-CSF) or encoding nucleic acid therefor.<br>
-Administration of the boosting composition is generally about 1 to 10 months after administration of the priming composition, preferrably about 1 to 6 months, preferably about 1 ro 4 months, preferably about 1 to 3 months.<br>
Preferably, administration of priming composition, boosting composition, or both priming and boosting compositions, is intradermal, intramuscular or mucosal immunization.<br>
Administration of MVA vaccines may be achieved by using a needle to inject a suspension of the virus. An alternative is the use of a needleless injection device to administer a virus suspension (using, e.g., Biojector™ needleless injector) or a resuspended freeze-dried powder containing the vaccine, providing for manufacturing individually prepared doses that do not need cold storage. This would be a great advantage for a vaccine that is needed in rural areas of Africa.<br>
MVA is a virus with an excellent safety record in human immunizations. The generation of recombinant viruses can be accomplished simply, and they can be manufactured reproducibly in large quantities. Intradermal, intramuscular or mucosal administration of recombinant MVA virus is therefore highly suitable for prophylactic or therapeutic vaccination of humans against ADDS which can be controlled by a CD8+ T cell response.<br>
The individual may have AIDS such that delivery of the antigen and generation of a CD8+ T cell immune response to the antigen is of benefit or has a therapeutically beneficial effect.<br>
Most likely, administration will have prophylactic aim to generate an immune response against HTV or AIDS before infection or development of symptoms.<br>
Components to be administered in accordance with the present invention may he formulated in pharmaceutical compositions. These compositions may comprise a pharmaceutically acceptable excipient, carrier, buffer, stabilizer or other materials well known to those skilled in the art. Such materials should be non-toxic and should not interfere with the efficacy of the active ingredient. The precise nature of the carrier or other material may depend on the route of administration, e.g. intravenous, cutaneous or subcutaneous, nasal, intramuscular, intraperitoneal routes.<br>
As noted, administration is preferably intradermal, intramuscular or mucosal. Physiological saline solution, dextrose or other saccharide solution or glycols such as ethylene glycol, propylene glycol or polyethylene glycol may be included.<br>
For intravenous, cutaneous, subcutaneous, intramuscular or mucosal injection, or injection at the site of affliction, the active ingredient will be in the form of a parenterally acceptable aqueous solution which is pyrogen-free and has suitable pH, isotonicity and stability. Those of relevant skill in the art are well able to prepare suitable solutions using, for example, isotonic vehicles such as Sodium Chloride Injection, Ringer's Injection, Lactated Ringer's Injection. Preservatives, stabilizers, buffers, antioxidants and/or other additives may be included as required.<br>
A slow-release formulation may be employed.<br>
Following production of MVA particles and optional formulation of such particles into compositions, the particles maybe administered to an individual, particularly human or other primate. Administration may be to another mammal, e.g. rodent such as mouse, rat or hamster, guinea pig, rabbit, sheep, goat, pig, horse, cow, donkey, dog or cat.<br>
Administration is preferably in a "prophylactically effective amount" or a "therapeutically effective amount" (as the case may be, although prophylaxis may be considered therapy), this being sufficient to show benefit to the individual. The actual amount administered, and rate and time-course of administration, will depend on the nature and severity of what is being treated. Prescription of treatment, e.g. decisions on dosage etc, is within the responsibility of general practitioners and other medical doctors, or in a veterinary context a veterinarian, and typically takes account of the disorder to be treated, the condition of the individual patient, the site of delivery, the method of administration and other factors known to practitioners. Examples of the techniques and protocols mentioned above can be found in Remington's Pharmaceutical Sciences, 16th edition, 1980, Osol, A. (ed.).<br>
In one preferred regimen, DNA is administered at a dose of 250 p,g to 2.5 mg/injection, followed by MVA at a dose of 106 to 109 infectious virus particles/injection.<br>
A composition may be administered alone or in combination with other treatments, either simultaneously or sequentially dependent upon the condition to be treated.<br>
Delivery to a non-human mammal need not be for a therapeutic purpose, but may be for use in an experimental context, for instance in investigation of mechanisms of immune responses to an antigen of interest, e.g. protection against HIV or AIDS.<br>
Further aspects and embodiments of the present invention will be apparent to those of ordinary skill in the art, in view of the above disclosure and following experimental<br>
exemplification, included byway of illustration and not limitation, and with reference to the attached figures.  <br>
20<br><br>
EXAMPLE 1<br>
Construction of plasmid shuttle vectors LAS-1 and LAS-2 used to make MVA/HIV<br>
recombinants.<br>
1.	Plasmid shuttle vector LAS-1 - This plasmid shuttle vector, which inserts<br>
into deletion IE of the MVA genome, was constructed in the following steps: The Pll<br>
vaccinia virus promoter and the Escherichia coli glucuronidase (GUS) gene were removed<br>
from pLW-51 (Wyatt, L.S. et al. 2004 AIDS Res. Human Retroviruses 20:645-653) byAsd<br>
and Sad digestion.  The Pll vaccinia promoter and green fluorescent protein (GFP) with<br>
Ascl and Sad ends, obtained by PCR amplification from pLW-44 (Bisht, H. et al. 2004<br>
PNAS USA 101:6641-6646), were ligated into the vector making pLW-51GFP. The psynH<br>
promoter was removed from pLW-51GFP by Xliol and Notl digestion, blunted, and<br>
religated to make LAS-1.<br>
2.	Plasmid shuttle vector LAS-2 -  This plasmid shuttle vector, which inserts<br>
into deletion n of the MVA genome, was constructed in the following manner: pLW-37<br>
was made by annealing 2 oligos comprising complementary strands of the mH5 promoter<br>
(Wyatt, L.S. et al. 1996 Vaccine 14:1451-58) with Hindi site at the beginning and Smal<br>
end and inserting this into the Smal site of pLW-16 (Wyatt, L.S. et al. 1999 Vaccine<br>
18:392-397).   The last 217 bases of MVA flank 2 were duplicated with Kpnl, Ascl, and<br>
Sad sites on the end of the inside oligo and Kpnl site on the end of the outside oligo, and<br><br>
inserted into the Kpnl site of pLW-37.  This plasmid, designated pLW-37a, was digested with Ascl and Sad, and Pll vaccinia promoter and green fluorescent protein (GFP) with Ascl and Sad ends (obtained by PCR amplification of Pll promoter and GFP from pLW-44 as described above) was inserted into it. The resulting plasmid was designated pLAS-2. I. pLW-51<br>
The plasmid transfer vector pLW-51 was constructed as follows. The mH5 promoter (Wyatt et al. 1996 Vaccine 14:1451-1458) was inserted into plasmid pGOl (Sutter and Moss. 1992 PNAS USA 89:10847-10851), which contains 926 and 530 bp of DNA that flank deletion IE (del ID). Gag-pol sequences from HTV clade B (strain BH10) DNA in plasmid GA2/JS2 (GenBank accession #AF42688) (Smith et al, 2004 AIDS Res Human Retroviruses 20:654-665) were amplified using the polymerase chain reaction (PCR) and inserted into the TA cloning plasmid pCR2.1 (Invitrogen Corp., Carlsbad, CA). The first 1872 nucleotides of the HTV strain BH10 gag-pol open reading frame (ORF) were replaced with the corresponding portion of the HXB2 gag ORF to enhance virus-like particle (VLP) formation. This chimeric gag-pol ORF was inserted into the modified pGOl plasmid after the mH5 promoter. The last 280 bp of the left MVA flank was duplicated and the Pll vaccinia virus promoter and the Escherichia coli glucuronidase (GUS) gene were inserted between the two direct repeats in order to implement a transient markerstabilization recombinant virus isolation protocol (Wyatt et al. 2004 AIDS Res. Human Retrovir. 20:645-653). Plasmid pGOl<br>
Plasmids. Sequences of MVA DNA flanking the site of a 3500-bp deletion in the<br>
Hindm A fragment of the MVA genome were amplified by PCR and cloned into pGEM 4Z<br>
(Promega). The primers for the left 900-bp DNA flank were 5'-<br>
CAGCAGGAATTCGTTGGTGGTCGCCATGGATGGTGT-3' (SEQ ID NO: 14) and 5'-<br>
GGGGGGGGTACCTACCAGCCACCGAAAGAG-3' (SEQ ID NO: 15) (sites for<br>
restriction enzymes EcoRl and Kpn I are underlined). The primers for the right 600-bp<br>
DNA	flank	were	5'-<br>
GGGGGGCTGCAGTTTGGAAAGTTTTATAGGGGGGGGCTGCAGTTTGGAAAGTT TTATAGG-3' (SEQ ID NO: 16) and 5'-GGGGGGAAGCTTAACTAGTTTCTGGTG-3' (SEQ ID NO: 17) (sites for the restriction enzymes Pst I and Hindis, are underlined). Between these flanlcs of MVA DNA, the Escherichia coli lacZ gene under control of the<br>
vaccinia virus late promoter PI 1 and the E. coli gpt gene under control of the vaccinia virus early/late promoter P7.5 were cloned (Sutter and Moss 1992 PNAS USA 89:10847-10851). n. pLW44<br>
pLW44 contains a gene encoding enhanced GFP regulated by the vaccinia virus PI 1 late promoter. m. pLW-16<br>
A new plasmid transfer vector, pLW-17, was constructed to allow insertion into deletion n located on the left side of the MVA genome (H. Meyer, et al. 1991J. Gen. Virol. 72:1031-1038) as follows. Flank 1 was prepared by PCR using primers beginning at nucleotide 52 preceding the left side of deletion n, cloned into a T-A cloning vector, digested with SphI, and cloned into the Sphl site of pGEM-4Z (Promega, Madison, WI). Flank 2 was prepared by PCR using primers containing EcoSI and Kpnl compatible ends from the right side of deletion n and cloned into EcdKL and Kpnl sites of pGEM-4Z. This plasmid containing the two flanking regions of deletion II was designated pLW-16. The modified H5 promoter was excised from pLW-9 by digestion with Smal and Pstl and cloned into Smal and Pstl sites of pLW-16, resulting in the new plasmid transfer vector, pLW-17. The F coding sequence of RSV-A2 was excised from a plasmid kindly provided by P. Collins and blunt ligated into the Smal site of pLW-17 (Wyatt, L.S. et al. 1999 Vaccine 18:392-397).<br>
EXAMPLE 2<br>
MVA recombinants expressing Clade A Env, Gag, and Pol MVA 65A/G Construction and Characterization<br>
This example describes the construction of a modified vaccinia virus Ankara (MVA) recombinant virus, MVA 65A/G, expressing clade A/G HTV strain 928 Env and Gag Pol. The salient features of this recombinant virus are:<br>
1)	A transient screening marker of green fluorescent protein (GFP) was used in<br>
construction of MVA 65A/G, so that the GFP is eliminated in the final virus product.<br>
2)	The A/G env gene is inserted into del n of MVA genome and the A/G gag<br>
pol is inserted in del m.<br>
3)	Both env and gag pol of MVA 65A/G are controlled by vaccinia PmH5<br>
promoter.<br>
4)	The MVA virus used to make the recombinant MVA 65A/G is MVA<br>
1974/Nffl Clone 1.<br>
Plasmid Transfer Vectors<br>
The plasmid transfer vectors, pJD5 and pJD6 (Figure 7 and Figure 8 respectively), were used to make single recombinant MVAs by homologous recombination. Each of these plasmids carry the transient GFP marker, and were constructed as follows:<br>
1. Gag/pol gene sequence of HIV A/G 928 (from pGA2/928/928+3RT) containing three mutations to inactivate reverse transcriptase activity (corresponding to that given for Clade B recombinant) with the integrase removed was cloned into pCR2.1 using FailSafe PCR kit. The gag/pol gene was cloned into MVA deletion in transfer vector pLAS-1 (containing transient screening marker GFP flanked by two direct repeats to eliminate GFP in the final virus) via Sail restriction endonuclease site, and gene expression was under control of the early/late modified H5 promoter. Sequences of the HTV A/G 928 gag/pol gene was confirmed by DNA sequencing, and this construct was designated pJD5 gagpol(Fig. 7).<br>
2. Env gene sequence of HIV A/G 928 (removed from pGAl/928/928 from Harriet Robinson) was truncated via PCR removal of 114 amino acids in the cytoplasmic tail of gp41 and cloned into pCR2.1. Silent mutations to eliminate 2 early poxvirus termination 5TNT signals were made (QuikChange Kit, Stratagene). The modified, truncated env gene was cloned into MVA deletion n transfer vector pLAS-2 (containing transient screening marker GFP flanked by two direct repeats to eliminate GFP in the final virus), with gene expression under control of the early/late modified H5 promoter. Sequence of the HIV A/G 928 env gene was confirmed by DNA sequencing, and this construct was designated pJD6 (Fig. 8). Single recombinant MVA construction<br>
1.	MVA 1974/NIH Clone 1 virus, which was derived from an MVA received<br>
from Anton Mayr (Munich, Germany) at passage 572 from 2/22/74 by terminally diluting 3<br>
times at the National Institutes of Health, was used for recombination.<br>
2.	Secondary CEF made from 10 day old SPAFAS  fertile chicken eggs<br>
(distributed by B&amp;E Eggs, Stevens, PA) in six-well culture plates were infected at an MOI<br>
of .05 of MVA and transfected with approximately 1.5 ^g of JDS or JD6 (Fig. 9).  These<br>
two virus constructions were carried in like manner as described below, but independently.<br>
3.	Following a two day incubation at 37°C, the viruses were harvested, frozen<br>
and thawed 3x, and plated out on CEF.<br>
4.	At 3 days post-infection, those foci of infections that expressed GFP<br>
(indicating that recombination had occurred between the plasmid and the infecting virus)<br>
were picked and replated on CEF. Again, those foci that expressed GFP were picked. This<br>
process was repeated once more.<br>
5.	These 3x plaque-purified GFP-expressing foci were plated out in duplicate<br>
CEF plates and two plates were analyzed for GFP fluorescence and Gag or Env expression<br>
with T24 monoclonal antibody (envelope expression) or mAb 3537 (183-H12-5C, NIH<br>
AIDS Research and Reference Reagent Program) specific for Gag.   Individual foci were<br>
picked from the duplicate replicate plate of those samples with little or no GFP<br>
fluorescence as well as mostly Gag or Env staining foci.<br>
6.	These foci were again plated out in duplicate, and analyzed the same way 1-<br>
2  additional times.   The resulting recombinant viruses derived expressed either Gag Pol<br>
(vJD5gag/pol) or Env (vJD6env) proteins but had deleted the GFP gene by recombination<br>
of the double repeats.<br>
Double Recombinant MVA Construction<br>
1.	To make the double MVA recombinant, MVA 65A/G, the two single<br>
recombinant MVA viruses, vJDSgag pol and vJD6 env, expressing Gag Pol and Env<br>
respectively, were used to infect CEF cells together at a MOI of 5, grown at 37C for 2 days<br>
and harvested (Fig. 9).<br>
2.	Duplicate plates of the harvest were infected at different dilutions, and one<br>
plate was fixed and stained with T24 mAb and number of foci of infected cells that had<br>
rounded up CPE (characteristic of Gag Pol recombinants) were scored in order to assure<br>
there were double recombinants in the population.    Then, foci with the rounded up<br>
morphology were picked from the unstained duplicate plate.<br>
3.	These   foci  were   plated   out   at  various   dilutions   in  triplicate   and<br>
immunostained for Env expression (T24 mAb) and Gag expression (3537 mAb).   Those<br>
foci that stained for both Env and Gag expression were plaque purified 5 additional times.<br>
4.	The virus was expanded in CEF cells and a LVD Seed stock of MVA/HTV<br>
65A/G 5.5 xlO8 pfu/ml was produced.   Mycoplasma testing by ATCC was negative.<br>
Sterility testing by BioReliance was negative.   DNA sequencing confirmed both the env<br>
and the gag pol ORFs, as well as the deletion of the GFP reporter gene.<br>
Characterization of MVA Recombinant Virus. MVA 65A/G<br>
1.	Aliquots of MVA 65A/G infected BS-C-1 cell lysates were analyzed by<br>
radioimmunoprecipitation (RJP) with monoclonal antibodies T24 and 3537 for expression<br>
of Envelope and Gag proteins, respectively (Fig. 10 and Fig. 11). Bands of the correct size<br>
corresponding to each of these proteins were detected.   Early expression of recombinant<br>
proteins was confirmed by infecting the cells in the presence of ARA-C. The recombinant<br>
virus was shown to produce gag particles in the supernatant of infected cells by pelleting<br>
the 35S labeled particles on a 20% sucrose cushion.<br>
2.	Repeated passage of MVA 65A/G and analyzing the resultant virus has<br>
confirmed that MVA 65A/G is stable through 10 passages of the LVD Seed stock.<br>
3.	Sequencing a region of the MVA/HIV 62 genome consisting of the HIV<br>
inserts as well as a 1000 base pair region of the MVA genome flanking either sides of the<br>
HTV confirmed that the GFP gene has been deleted and the sequence of the HTV inserted<br>
genes are correct. Figures 12 and 13 show the sequence of the env and gag pol genes in the<br>
plasmids.<br>
4.	Immunogenicity was assessed by 107 pfu of purified MVA 65A7G inoculated<br>
into mice at 0 and 4 weeks.   ELISA serum antibodies were assayed in a capture ELISA<br>
utilizing gp!40 clade A env as the antigen.    Table 1 shows the clade A env is<br>
immunogenic.<br>
Summary<br>
In summary, we have made a recombinant MVA virus, MVA 65A/G, which expresses the A/G 928 modified, truncated envelope and mutated gag pol. The MVA double recombinant virus was made by homologous recombination of single MVA recombinants, one expressing Env and one expressing Gag Pol. These single MVA recombinants were made using a transiently expressed GFP marker that was deleted in the final viruses. MVA/HTV 65A/G was shown to be stable through repeated passage of the LVD Seed Stock.<br>
Modification to MVA 65A/G Construct to Increase Expression and Immunogenicity<br>
The immunogenicity of MVA 65A/G recombinant virus as measured by ELISA utilizing MVA clade A gp!40 env as the antigen showed that MVA 65A/G envelope was not as immunogenic as hoped (See Table 1). This example describes the modification to the MVA 65A/G plasmid transfer vector, pJD-6, which was made to increase expression<br>
and immunogenicity of the A/G envelope, and its comparison to the original MVA 65 A/G virus.<br>
ELISA titers from serum of five mice individually assayed which were immunized twice with 107 PFU of the designated virus. Serum antibody responses were assayed by a 2 day ELISA utilizing secreted Clade A 928 gp!40, commercial p24, and purified vaccinia virus as the antigens for the Clade A ELISA response.<br>
Modification to pJD-6<br>
The plasmid transfer vector, pJD-6 was modified in the following way:<br>
1.	The A/G envelope gene was PCR amplified from pJD-6 with oligos which<br>
incorporated a Smal site at the 5' end of the clade A/G and a NotI site at the 3' end of the<br>
env gene. This insert was cloned into the Smal and NotI site of the pLAS-2 insertion vector<br>
making a transfer vector, JD-16 (Fig. 14).  This plasmid differed from pJD-6 (which had<br>
been used to make MVA 65 A/G recombinant virus) in only one respect, the intervening<br>
sequences between the end of the promoter and the start site of the env gene within the<br>
multiple cloning region (see #2 for further explanation).<br>
In pJD-6, the A/G 928 env was cloned into the multiple cloning site of<br>
pLAS-2 using the NotI restriction site. In this arrangement there is an intervening initiation<br>
codon between the promoter and the initiation codon of the envelope (part of the Sphl<br>
restriction enzyme site). In pJD-16, the A/G 928 env was cloned into the multiple cloning<br>
site of pLAS-2 using the Smal site at the beginning of the gene and Not I at the end of the<br>
gene, thus eliminating this intervening initiation codon.  Thus, the intervening sequences<br>
between the end of the promoter and the start site of the env gene within the multiple<br>
cloning region are the only Double Recombinant MVA Construction<br>
To make the double recombinant MVA virus with the JD-16 plasmid, the single MVA recombinant virus expressing the gag pol of the A/G virus, vJDSgag pol, as described above in the original MVA 65A/G construct, was used to infect CEF, and pJD-16 was transfected into this infected culture. Virus isolation proceeded as described above for the original MVA/HIV 65A/G virus and the resulting double recombinant virus (Clone Dl.3266-15), was designated MVA/HIV 65A/G(SmaI).<br>
Characterization and Comparison of the original MVA/HTV 65A/G with the modified MVA/HTV 65A/G(SmaD<br>
1.	Aliquots of MVA/HIV 65A/G and MVA/HTV 65A/G(SmaI) infected BS-C-<br>
1 cells were analyzed by RIP with monoclonal Antibodies T24 and T32 mAb for the env<br>
(Fig. 15) and 3537 (183-H12-5C, NTH AIDS Research and Reference Reagent Program)<br>
mAb for the gag expression (Fig. 16).   Utilizing Image Quant program, the amount of<br>
protein immunoprecipitated with T24 from each construct was quantitated.    and Env<br>
expression was 2 fold higher with modified 65A/G(Smar) compared to MVA 65A/G.<br>
MVA 65A/G (Smal) was shown to produce gag particles in the supernatant of infected cells<br>
by pelleting 35S labeled particles through a 20% sucrose cushion.<br>
2.	Immunogenicity of the modified 65A/G(SmaI) virus was assessed in mice<br>
immunized 2x with either MVA 65 A/G or modified MVA 65A/G(SmaI) at 107 dose given<br>
M three weeks apart.   The average ELISA titer from serum of five mice (individually<br>
assayed) with the designated virus was determined.    Serum antibody responses were<br>
assayed by a 2 day ELISA utilizing secreted Clade A 928 strain gp!40 as the antigen. The<br>
results in Fig. 17 indicate that the modified construct (MVA 65A/G (SmaT) Env is about 40<br>
times more immunogenic than the original MVA 65A/G construct.<br>
The 2 different viruses were also tested for their ability to induce a cellular immune response of the Gag protein as measured by intracellular cytokine staining. Mice (5/group) were immunized twice with 107 pfu of the designated viruses. Splenocytes from individual<br>
mice were assayed directly ex vivo after overnight culture with 2 different gag p24 peptide-pulsed P815 cells. CD8+ IFN-y+ cells were enumerated by flow cytometry. As shown in Fig. 18, both of the constructs had similar intracellular cytokine staining (TCS) response to the gag peptides. This would be expected as the gag virus used to make the 2 constructs was identical.<br>
3. Repeated passage of MVA 65A/G(SmaI) and analyzing the expression of the resultant virus has confirmed that the envelope expression of modified MVA 65A/G(SmaI) is highly unstable as shown in <br>
Thus, removal of the intervening codon initiation site, through recloning of the envelope into a closer site to the promoter, made a virus which expressed larger quantities of env and was much more immunogenic; however it could not be pursued as a candidate vaccine because it was highly unstable (Table 3).<br>
TableS. EXAMPLES MVA recombinants expressing Clade B Env, Gag, and Pol<br>
MVA/HIV 62B Construction and Characterization<br>
This example describes the construction of a modified vaccinia virus Ankara (MVA) recombinant virus, MVA/HIV 62B, expressing clade B HIV strains ADA Env and chimeric HXB2/BH10 Gag Pol. This virus differs from an earlier MVA clade B recombinant, MVA/HTV 48, (which also expresses identical HTV strain ADA Env and HXB2/BH10 Gag Pol) in 4 ways:<br>
1)	MVA/HTV 62B uses a transient screening marker of green fluorescent protein<br>
(GFP) instead of GUS screening marker used in MVA/HIV 48.<br>
2)	The env gene is inserted into del n of MVA genome and the gagpol is inserted<br>
in del HI. In MVA/HTV 48, both env and gagpol are inserted into del HI.<br>
3)	Both env and gag pol of MVA/HTV 62B are controlled by PmH5, the same<br>
promoter controlling gagpol in MVA/HTV 48.<br>
4)	The MVA virus used to make the recombinant MVA/HIV 62B is MVA<br>
1974/NIH Clone 1 instead of MVA 1983/NIH Clone 1 used to make MVA/HIV 48.<br>
Plasmid Transfer Vectors<br>
The plasmid transfer vectors, pLAS-1 HXB2/BH10 Gag Pol, and pLAS-2 ADA Env (Fig. 19 and Fig. 20 respectively), were used to make single recombinant MVAs by homologous recombination. Each of these plasmids carry the transient GFP marker, and were constructed as follows:<br>
1.	The clade B gag pol was truncated so that the integrase was remove and was<br>
cloned into the plasmid so that it was controlled by the mH5 promoter.    This gene<br>
contained the complete HXB2 sequence of the gag. The pol gene has reverse transcriptase<br>
safety mutations in amino acid 185 within the active site of RT, in amino acid 266 which<br>
inhibits strand transfer activity, and at amino acid 478 which inhibits the Rnase H activity.<br>
In addition, the integrase gene is deleted past EcoRI site.<br>
2.	pLAS-1 was constructed   from pLW-51GFP by cutting out the Psyn E<br>
vaccinia promoter, blunting and religating. Clade B HXB2/BH10 gagpol sequences from<br>
MVA/HIV 48 was cloned into pLAS-1, making the plasmid transfer vector, pLAS-1<br>
HXB2/BH10 Gag Pol (Fig. 19).<br>
3.	The ADA envelope is a truncated version with silent 5TNT mutations. The<br>
envelope was truncated in the cytoplasmic tail of the gp41 gene, deleting 115 amino acids<br>
of the cytoplasmic tail. This truncation was shown by us to increase the amount of envelope protein on the surface of infected cells and enhance irnmunogenicity of the envelope protein in mice, and stability of the recombinant virus in tissue culture.<br>
4.        pLAS-2 was constructed by a insertion of a direct repeat of the MVA flank as described under LAS-2 construction followed by an insertion of GFP controlled by PI 1 vaccinia promoter.  Addition of the above ADA modified env sequences from MVA/HTV 48 made plasmid transfer vector pLAS-2 ADA Env (Fig. 20). Single recombinant MVA construction<br>
1.	MVA 1974/NIH Clone 1 virus, which was derived from an MVA received<br>
from Anton Mayr (Munich, Germany) at passage 572 from 2/22/74 by terminally diluting 3<br>
times at the National Institutes of Health, was used for recombination.<br>
2.	Secondary CEF cells were infected at an MOI of 0.05 of MVA and<br>
transfected with 2 p,g of pLAS-1 HXB2/BH10 Gag Pol or 2 ug of pLAS-2 ADA Env (Fig.<br>
21).   These two virus constructions were carried in like manner as described below, but<br>
independently.<br>
3.	Following a two day incubation at 37°C, the viruses were harvested, frozen<br>
and thawed 3x, and plated out on CEF.<br>
4.	At 3 days post-infection, those foci of infections that exhibited GFP<br>
fluorescence (indicating that recombination had occurred between the plasmid and the<br>
infecting virus) were picked and replated on CEF.  Again, those foci that expressed GFP<br>
were picked.<br>
5.	These 2x plaque-purified GFP-expressing foci were plated out in triplicate<br>
CEF plates and two plates were analyzed for GFP fluorescence and Gag or Env expression.<br>
Individual foci were picked from the 3rd replicate plate of those samples with little or no<br>
GFP fluorescence as well as mostly Gag or Env staining foci.<br>
6.	These foci were again plated out in triplicate, and analyzed the same way 2<br>
additional times (3x for the Gag Pol construct).  The resulting viruses derived expressed<br>
either Gag Pol (MVA 60) or Env (MVA 61) proteins but had deleted the GFP gene by<br>
recombination of the double repeats.<br>
Double Recombinant MVA Construction<br>
1.        To   make the double MVA recombinant, MVA/HTV 62B, the two single recombinant MVA viruses,    MVA 60 and MVA 61, expressing Gag Pol and Env<br>
respectively, were used to infect CEF cells together at a MOI of 5, grown at 37°C for 2 days and harvested (Fig. 21).<br>
2.	Duplicate plates of the harvest were infected at different dilutions, and one<br>
plate was fixed and stained with T8 mAb and number of foci of infected cells that had<br>
rounded up CPE (characteristic of Gag Pol recombinants) were scored in order to assure<br>
there were double recombinants in the population.    Then, foci with the rounded up<br>
morphology were picked from the unstained duplicate plate.<br>
3.	These   foci   were   plated   out   at   various   dilutions   in  triplicate   and<br>
immunostained for Env expression (T8 mAb) and Gag expression (3537 mAb, 183-H12-<br>
5C, NIH AIDS Research and Reference Regent Program). Those foci that stained for both<br>
Env and Gag expression were plaque purified further.<br>
4.	The virus was expanded in CEF cells and a LVD Seed stock of MVA/HTV<br>
62 was produced.    Mycoplasma testing by ATCC was negative.    Sterility testing by<br>
BioReliance was negative.<br>
Characterizatiori of MVA Recombinant Virus. MVA/HIV 62B<br>
1.	Aliquots of MVA/HTV 62B infected BS-C-1 cell lysates were analyzed by<br>
radioimmunoprecipitation (RIP) with monoclonal antibodies T8 and 3537  for expression<br>
of Envelope and Gag proteins, respectively (Fig. 22).    Bands of the  correct size<br>
corresponding to each of .these proteins were detected.   Early expression of recombinant<br>
proteins were confirmed by infecting the cells in the presence of ARA-C. The recombinant<br>
virus was shown to produce gag particles in the supernatant of infected cells by pelleting<br>
the 35S labeled particles on a 20% sucrose cushion.<br>
2.	Repeated passage of MVA/HTV 62B and analyzing the resultant virus has<br>
confirmed that MVA 62B is relatively stable through 10 passages of the LVD Seed stock.<br>
3.	Sequencing a region of the MVA/HIV 62B genome consisting of the HTV<br>
inserts as well as a 1000 base pair region of the MVA genome flanking either sides of the<br>
HIV inserts confirmed that the GFP gene has been deleted and the sequence of the HIV<br>
inserted genes are correct. Sequence of ADA Env is given in Fig. 23. Sequence of Gag Pol<br>
is given in Fig. 24.<br>
4.	Immunogenicity was assessed by 107 pfu of purified MVA 62B inoculated<br>
into mice at 0 and 4 weeks.  ELISA serum antibodies were assayed in a capture ELISA<br>
utilizing gp!40 ADA env as the antigen.  Table 4 shows the ADA env is immunogenic.<br>
Fig. 25 shows the ICS env and gag responses of mice to MVA 62B.<br>
ELISA titers from serum of five mice individually assayed which were immunized twice with 107 PFU of the designated virus. Serum antibody responses were assayed by a 2 day ELISA utilizing secreted Clade B ADA gp!40, commercial p24s and purified vaccinia virus as the antigens for the Clade B ELISA response.<br>
Summary<br>
In summary, we have made a recombinant MVA virus, MVA/HTV 62B, which expresses ADA modified, truncated Envelope and HXB2/BH10 Gag Pol. The MVA double recombinant virus was made by homologous recombination of single MVA recombinants, one expressing Env and one expressing Gag Pol. These single MVA recombinants were made using a transiently expressed GFP marker that was deleted in the final viruses. The MVA 62 B virus was shown to make particles and was immunogenic in mice. MVA/HTV 62B was shown to be stable through repeated passage of the LVD Seed Stock.<br>
Modifications to MVA 62B Construct to Increase Expression and Immunogenicity<br>
The immunogenicity of MVA 62B in mice as measured by ELISA utilizing MVA clade B gp!40 as the antigen (Table 4) showed that MVA 62B envelope was not as immunogenic as hoped. This example describes the modifications to the MVA 62B plasmid transfer vector, pLAS-2 ADA Env, which were made to increase expression and immunogenicity of the ADA envelope in the final MVA 62 B double recombinant viruses. Plasmid Transfer Vector Modification<br>
The plasmid transfer vector, pLAS-2 ADA Env was modified in the following ways:<br>
1. The ADA env was cut out of JD-9 ADA Env by treating with the restriction enzyme, Xmal (an isoschizmer of Smal which cuts making overlapping ends compared to the blunt cut of Smal), and inserted into the Xmal site in LAS-2 making plasmid pLAS ADA Env (Smal) or as it was designated pLW-66 (Fig. 26). This plasmid differed from pLAS-2 ADA Env (which had been used to make MVA 62 B recombinant virus) in only one significant respect, the intervening sequences between the end of the promoter and the start site of the env gene within the multiple cloning region. (It also differed from pLAS-2 ADA Env in the cloning region of the end of env gene because of the way it was cloned into the plasmid, but this is not of consequence to the expression of the gene.) (see #3 for further explanation).<br>
2.	pLAS-2 ADA Env was digested with restriction enzyme Sal I followed by<br>
Xhol to cut out the SphI site (which contains an initiation codon) and religated together to<br>
make pJD-17 (Fig. 27).  This plasmid differed from pLAS-2 ADA Env (which had been<br>
used to make MVA 62 B recombinant virus) in only one respect, the intervening sequences<br>
between the end of the promoter and the start site of the env gene within the multiple<br>
cloning region (see #3 for further explanation).<br>
3.	In plasmid pLAS-2 ADA Env, the ADA envelope had been cloned into the<br>
Not I site generating the following nucleotide sequence between the promoter. In pLW-66,<br>
the ADA env was cloned into the Xrnal site (Smal) generating the following sequence<br>
between the vaccinia mHSpromoter and the ADA env initiation codon as depicted below.<br>
In pJD-17 the SphI site had simply been removed. This all shown below:<br><br>
MVA 60, as described above, was used to infect CEF cells, and pLW-66 or pJD -17 was transfected into these infected CEF cultures. Virus isolation proceeded as described above for the original MVAyHIV 62B virus and the two clones of each resulting double recombinant viruses were designated as follows:<br>
Characterization and Comparison of the original MVA/HIV62B  with the modified MVA/HIV 62B viruses<br>
1.	Aliquots of MVA/HTV 62B and 4 modified MVA 62B virus infected BS-C-<br>
1 cells were analyzed by REP with monoclonal antibody T8 for the env (Fig. 28) and 3537<br>
mAb for the gag expression (Fig. 29).   Utilizing Image Quant program, the amount of<br>
protein immunoprecipitated with T8 from each construct was quantitated.    and Env<br>
expression was 2.6-3.3 fold higher with the modified constructs compared to MVA 62B.<br>
All of the MVA62B constructs were shown to produce gag particles in the supernatant of<br>
infected cells by pelleting 35S labeled particles through a 20% sucrose cushion.<br>
2.	Immunogenicity of the modified MVA62B constructs were assessed in mice<br>
immunized 2x with either the original or modified MVA 62B viruses at 107 dose given IM<br>
three weeks apart.   Fig. 30 shows the   average ELISA titer from serum of five mice<br>
(individually assayed). Serum antibody responses were assayed by a 2 day ELISA utilizing<br>
secreted Clade B ADA gp!40 as the antigen.   The results indicate that the modified<br>
constructs are 11-32 fold more immunogenic than the original construct in this assay of the<br>
envelope immunogenicity.<br>
The different viruses were also tested for their ability to induce a cellular immune response to the Env or Gag protein as measured by intracellular cytokine staining (ICS). Mice (5/group) were immunized twice with 107 pfu of the designated viruses. Splenocytes from individual mice were assayed directly ex vivo after overnight culture with the env peptide or gag p24 peptide-pulsed P815 cells. CDS4" IFN-y+ cells were enumerated by flow cytometry.<br>
As shown in Fig. 31, the modified 62B constructs all made an enhanced ICS Env response compared to the original. All were 3-5 fold higher than original MVA 62B.<br>
As shown in Fig. 32, all of the MVA 62B constructs whether original or modified had similar ICS response to the gag pep tide (no more than a two fold variation). This would be expected as the gag virus used to make the modified MVA 62B constructs was identical.<br>
3.        Repeated passage of the modified MVA/HIV 62B viruses and analyzing the resultant virus has confirmed that MVA 62B is relatively stable through 10 passages of the LVD Seed stock. Summary<br>
Thus, removal of the intervening codon initiation site, through recloning of the envelope into a closer site to the promoter, made viruses which expressed larger amounts of env and were much more immunogenic. These viruses are being pursued as candidate vaccine viruses (Table MVA 56 Construction and Characterization<br>
This example describes the construction of a modified vaccinia virus Ankara (MVA) recombinant virus, MVA/HIV clade B expressing HIV strain ADA env and chimeric HXB2/BH10 gag pol. This virus differs from an earlier MVA recombinant, MVA/HIV 48, (which also expresses the HIV strain ADA env and the HXB2/BH10 gag pol) in 3 ways:<br>
1. MVA/HIV 56 uses a transient screening marker of green fluorescent protein (GFP) instead of the GUS screening marker used in MVA/HIV 48.<br>
2.	The ADA env of MVA/HIV 56 is controlled by a new modified vaccinia<br>
vims promoter, Pm2H5, which allows more early expression of ADA env than the Psyn n<br>
promoter used to express the ADA env in MVA/HIV 48. (The gag pol is controlled by the<br>
vaccinia virus mHSpromoter, the same promoter controlling the gag pol in MVA 48.)<br>
3.	The MVA virus used to make the recombinant MVA/HIV 56 is MVA<br>
1974/NIH Clone 1 instead of MVA 1983/NIB Clone 1, used to make MVA/HIV 48.<br>
It differs from later constructs of original MVA 62B and modified MVA 62B in that the env and gag genes are controlled by double vaccinia virus promoters which are in tandem, like in MVA 48, and insert into only deletion HE of the MVA genome. This gives this type of construct a construction advantage over the MVA 62B constructs since one has to go through only one set of plaque purifications as compared to the construction regime of the MVA 62B constructs.<br>
All clade B constructs described have the same modified ADA env and modified HXB2/BH10gagpol. Plasmid Transfer Vector<br>
The plasmid transfer vector, pLAS-6, used to make MVA/HIV 56 by homologous recombination was constructed as follows:<br>
1.	The GUS screening marker was cut out of the plasmid, pLW-48 (see WO<br>
002/072759), and replaced with the GFP screening marker run by the vaccinia virus Pll<br>
promoter (pLW-51 GFP).  This GFP screening marker was used transiently to 2x plaque-<br>
purify recombinant virus, but was deleted in the final recombinant by homologous<br>
recombination (because GFP was flanked by two direct repeats).<br>
2.	The Psyn H vaccinia promoter was cut out of pLW-51 GFP by Xhol and NotI<br>
digestion and replaced with a new further modified H5 vaccinia virus promoter, pm2H5, in<br>
the same sites. This new plasmid was named pLAS-5.<br>
3.	The ADA env, was removed from pLW-48 by NotI enzyme digestion of the<br>
plasmid and cloned into the NotI site of pLAS-5 controlled by the vaccinia virus m2H5<br>
promoter.   The HXB2/BH10 gag pol gene was removed from pLW-48 by Sail digestion<br>
and placed in the Sail site under the control of the mH5 promoter to make the plasmid<br>
transfer vector, pLAS-6 (Fig 33), which was used to make recombinant virus MVA/HIV 56. The recombinant HIV genes were shown by transfection to be expressed. Recombinant MVA/HIV 56 Construction<br>
1.	MVA 1974/N1H Clone 1 virus, which was derived from an MVA received<br>
from Anton Mayr (Munich, Germany) at passage 572 from 2/22/74 by terminally diluting 3<br>
times at the National Institutes of Health, was used for recombination.<br>
2.	Secondary CEF cells were infected at an MOI of 0.05 of MVA and<br>
transfected with 2 jag of pLAS-6, the plasmid described above.   Following a two day<br>
incubation at 37°C, the virus was harvested, frozen and thawed 3x, and plated out on CEF.<br>
3.	At 3 days, those foci of infections that expressed GFP (indicating that<br>
recombination had occurred between the plasmid and the infecting virus), were picked and<br>
replated on CEF. Again, those foci that expressed GFP were picked.<br>
4.	These 2x plaque-purified GFP-expressing foci were plated out in triplicate<br>
CEF plates and two plates were analyzed for GFP fluorescence and gag pol expression.<br>
Individual foci were picked from the 3rd replicate plate of those samples with little or no<br>
GFP staining as well as mostly gag staining foci.<br>
5.	These foci were again plated out in triplicate, and analyzed the same way 2<br>
additional times. The resulting virus derived expressed the gag protein but had deleted the<br>
GFP gene by recombination of the double repeats.   By immunostaining, this virus also<br>
expressed the env protein.  The virus was expanded in CEF cells and a LVD Seed stock<br>
was produced and sterility and mycoplasma tested by BioReliance and ATCC, respectively.<br>
Characterization of MVA Recombinant Virus. MVA/HIV 56<br><br>
1.	Aliquots   of MVA/HIV   56   infected   cell   lysates   were   analyzed  by<br>
radioimmunoprecipitation (RIP) with monoclonal antibodies for expression of both the<br>
envelope and gag proteins, both in the presence and absence of ARA-C. In RIP, bands of<br>
the correct size corresponding to each of these proteins was detected, both in the presence<br>
and absence of ARA-C indicating both early and late expression of recombinant protein<br>
(Fig. 34).<br>
2.	MVA/HIV 56 was shown to produce gag particles in the supernatant of<br>
infected cells by pelleting the 35S labeled particles from supernatant of MVA/HIV 56-<br>
infected BS-C-lcells on a 20% sucrose cushion and analyzing by PAGE.<br>
3.	Sequencing a region of the MVA/HTV 56 genome consisting of the HTV<br>
inserts as well as a 1000 base pair region of the MVA genome flanking either side of the<br>
HTV inserts confirmed that the GFP gene had been deleted and the sequence of the HIV inserted genes was correct.<br>
4.	Repeated passage of the modified MVA/HTV 62B viruses and analyzing the<br>
resultant virus has confirmed that MVA 62B is relatively stable through 5 passages of the<br>
LVD Seed stock.<br>
5.	Immunogenicity of MVA 56 was assessed in mice immunized 2x with 107<br>
pfu three weeks apart.   Fig. 28 shows the   average ELISA titer from serum of five mice<br>
(individually assayed). Serum antibody responses were assayed by a 2 day ELISA utilizing<br>
secreted Clade B ADA gp!40 as the antigen. The results indicate that MVA 56 is able to<br>
elicit serum antibodies. MVA 56 was also tested for its ability to induce a cellular immune<br>
response of the Env or Gag protein as measured by ICS. Mice (5/group) were immunized<br>
twice with 10 pfu of MVA 56. Splenocytes from individual mice were assayed directly ex<br>
vivo after overnight culture with the env peptide or gag p24 peptide-pulsed P815 cells.<br>
CD8+ IFN-y+ cells were enumerated by flow cytometry.   As shown in Fig. 29, MVA 56<br>
induces an env cellular immune response. As shown in Fig. 29, MVA 56 also induces an<br>
ICS gag p24 response, similar to the other clade B constructs.<br>
Summary<br>
In summary, we have made a recombinant MVA virus, MVA/HTV 56, which has high expression of ADA modified truncated envelope and HXB2/BH10 gag pol and inserts into deletion in of the MVA genome. The MVA recombinant virus was made using a transiently expressed GFP marker that was deleted in the final virus. High expression of the ADA envelope was possible because of a new hybrid early/late promoter, Pm2H5. The MVA 56 recombinant malces gag particles which has been shown by pelleting the particles through sucrose and analyzing by PAGE. Sequencing of the recombinant region of the MVA genome confirmed the absence of GFP and the correct sequence of the HIV genes. Env ELISA confirmed the immunogenicity of the ADA envelope and cellular response to both the envelope and p24 gag were detected.<br>
EXAMPLE 4<br>
MVA recombinant expressing Clade C Env, Gag. Pol MVA/HIV 71C Construction and Characterization<br>
This example describes the construction of a modified vaccinia virus Ankara (MVA) recombinant virus, MVA/HTV 71C, expressing an Indian clade C HIV INS Env and Gag Pol. The salient features of this recombinant virus are:<br>
1.	A transient screening marker of green fluorescent protein (GFP) was used in<br>
construction of MVA/HTV 71C, so that the GFP is eliminated in the final virus product.<br>
2.	The 71C env gene is inserted into del n of MVA genome and the 71C gag<br>
pol is inserted in del IE.<br>
3.	Both env and gag pol of MVA 71C are controlled by vaccinia mH5<br>
promoter.<br>
4.	The MVA virus used to make the recombinant MVA/HIV 71C is MVA<br>
1974/NIH Clone 1.<br>
Plasmid Transfer Vectors<br>
The plasmid transfer vectors, pDC3 and pJDIS, Figure 35 and Figure 36 respectively, were used to make the double recombinant MVA by homologous recombination. Each of these plasmids carries the GFP marker, flanked on either side by direct repeats to eliminate GFP in the final virus product. The plasmids were constructed as follows:<br>
1.	The gene sequence of HIV C INS gag pol (from Indian isolate GenBank<br>
#AF286231 from Harriet Robinson, Emory University) containing three mutations to<br>
inactivate  reverse transcriptase  activity (corresponding  to  that  given  for  clade B<br>
recombinant) with the integrase removed was cloned into pCR2.1 using FailSafe PCR kit.<br>
The gag/pol gene was cloned into MVA deletion ffi transfer vector pLAS-1 (containing<br>
screening marker GFP flanked by two direct repeats to eliminate GFP in the final virus) via<br>
Smal restriction endonuclease site, and gene expression was under control of the early/late<br>
modified H5 promotor.  Sequence of the HIV C DST3 gag pol gene was confirmed by DNA<br>
sequencing, and this construct was designated pDC3 (Fig. 35).<br>
2.	The gene sequence of HIV C BSD env (from Indian isolate GenBank<br>
#AF286231 from Harriet Robinson, Emory University) was truncated via PCR removal of<br>
120 amino acids in the cytoplasmic tail of gp41 and cloned into pCR2.1.  Silent mutations<br>
to eliminate 2 early poxvirus termination 5TNT signals were made (QuikChange Kit,<br>
Stratagene).  The modified, truncated env gene was cloned into MVA deletion n transfer<br>
vector pLAS-2 (containing screening marker GFP flanked by two direct repeats to eliminate<br>
GFP in the final virus) in the Not I site followed by removal of the Spbl site within the<br>
cloning region by cutting sequentially with Sail, then Xhol and ligating the ends to make<br>
pJD-15.   Gene expression in this plasmid is under control of the early/late modified H5<br>
promoter. Sequence of the HIV C INS env gene was confirmed by DNA sequencing, in this construct was designated pJD-15 (Fig. 36). Single recombinant Glade C MVA/HTV gag pol<br>
1.	MVA 1974/NIH Clone 1 virus, which was derived from an MVA received<br>
from Anton Mayr (Munich, Germany) at passage 572 from 2/22/74 by terminally diluting 3<br>
times at the National Institutes of Health, was used for recombination.<br>
2.	Secondary CEF made from 10 day old SPAFAS fertile chicken eggs<br>
(distributed by B&amp;E Eggs, Stevens, PA) in six-well culture plates were infected at an MOI<br>
of 0.05 of MVA and transfected with approximately 3.0 ug of pDC3 (Fig. 37).<br>
3.	Following a two day incubation at 37°C, the virus was harvested, frozen and<br>
thawed 3x, and plated out on CEF plates.<br>
4.	At 3 days post-infection, those foci of infections that expressed GFP<br>
(indicating that recombination had occurred between the plasmid and the infecting virus)<br>
were picked and replated on CEF. Again, those foci that expressed GFP were picked.<br>
5.	These 2x plaque-purified GFP-expressing foci were plated out in duplicate<br>
CEF plates and two plates were analyzed for GFP fluorescence and for Gag expression with<br>
MAb 183-H12-5C (3537) specific for Gag. Individual foci were picked from the duplicate<br>
replicate plate of those samples with little or no GFP staining as well as mostly Gag<br>
staining foci.<br>
6.	These foci were again plated out in duplicate, and analyzed the same way 3<br>
additional times.   The resulting recombinant virus expressed Gag Pol proteins but had<br>
deleted the GFP gene by recombination of the double repeats. A seed stock was made, first<br>
by growing up the final picked plaques in 1 well of 6 well plate, then T-25 flasks, followed<br>
by growth in T-150 flasks.  This vims, labeled MVA 68C (LAS1 gag pol), was used to<br>
make the double recombinant.<br>
Double Recombinant MVA Construction<br>
1.	To make the double MVA recombinant expressing both the Env and the Gag<br>
Pol proteins, single recombinant MVA virus expressing Gag Pol (MVA 68C), was used to<br>
infect CEF cells at a MOI of 0.05, followed by transfection of 1 fig of pJD 15, and grown at<br>
37°C for 2 days and harvested (Fig. 37).<br>
2.	At 3 days post-infection, those foci of infections mat expressed GFP<br>
(indicating that recombination had occurred between the plasmid and the infecting virus)<br>
were picked and replated on CEF. Again, those foci that expressed GFP were picked.<br>
3.	These 2x plaque-purified GFP-expressing foci were plated out in duplicate<br>
CEF plates and two plates were analyzed for GFP fluorescence and  for Env expression<br>
with T-43 mAb specific for Env protein.  Individual foci were picked from the duplicate<br>
replicate plate of those samples with little or no GFP fluorescence as well as mostly Env<br>
staining foci.<br>
4.	These foci were again plated out in duplicate, and analyzed the same way 4<br>
additional times.   The resulting double recombinant virus expressed Env and Gag Pol<br>
proteins but had deleted the GFP gene by recombination of the double repeats.  An LVD<br>
seed stock was made, first by growing up the final picked plaques in 1 well of 6 well plate,<br>
then T-25 flask, followed by growth in T-150 flasks.  This virus, labeled MVA/HTV 71C<br>
(lab. name-Clone 2 C4 6.21121), expressed both Env and Gag by immunostaining.<br>
Characterization of MVA Double Recombinant Virus. MVA/HIV 71C<br><br>
1.	Aliquots of MVA/HTV 71C infected BS-C-1 cell lysates were analyzed by<br>
radioimmunoprecipitation (RIP) with monoclonal antibodies T-43 and 3537 for expression<br>
of Envelope and Gag proteins, respectively (Fig.  38).     Bands of the correct size<br>
corresponding to each of these proteins were detected. Expression occurred in the presence<br>
of Ara-C indicating the functionality of the early part of the mH5 promoter.    The<br>
recombinant virus was shown to produce gag particles in the supernatant of infected cells<br>
by pelleting the 35S labeled particles on a 20% sucrose cushion.  The functionality of the<br>
expressed Clade C envelope had been shown with the identical C sequence in previous<br>
constructs in fusion assay.<br>
2.	Repeated passage of MVA/HIV 71C and analyzing the resultant virus has<br>
confirmed that it is relatively stable through 9 passages of the LVD Seed stock.<br>
3.	Sequencing a region of the MVA/HTV 71C genome consisting of the HIV<br>
inserts as well as a 1000 base pair region of the MVA genome flanking either sides of the<br>
HIV inserts confirms that the GFP genes have been deleted and the sequence of the HIV<br>
inserted genes are correct which include the three mutations in the reverse transcriptase<br>
gene. Figures 39 and 40 show the sequence of the env and gagpol genes in the plasmids.<br>
4.	The immunogenicity of a dose 107 pfu of purified MVA/HIV 71C inoculated<br>
into mice at 0 and 4 weeks by the intramuscular route has been assessed.  ELISA serum<br>
antibodies were assayed in a capture ELISA utilizing gp!40 clade C env as the antigen.<br>
Table 6 shows that the clade C env is immunogenic.<br>
immunize 5 mice each with 107 dose given IM at 0 and 4 weeks. Bled at 2 weeks post<br>
2nd immunization;<br>
reciprocal end point dilution.<br>
EXAMPLE 5<br>
Comparative immunogenicitv in rhesus monkeys of multi-protein HTV-1 (CRF02 AG) DNA/MVA vaccines expressing mature and immature VLPs<br>
We developed an ADDS vaccine for Western and West-Central Africa founded on HIV-1 subtype CRF02_AG. Rhesus macaques were primed with Gag-Pol-Env-expressing plasmid DNA and boosted with a recombinant modified vaccinia virus Ankara (rMVA), expressing matched proteins. Two DNA vaccine constructs (1C 1-90 and IC48) that differed by point mutations in gag and pol were compared. IC1-90 produces primarily immature (core comprises unprocessed PrSSGag) HIV-like particles (VLPs) and IC48 produces mature VLP with processed PrSSGag, immature VLP, and intracellular protein<br>
aggregates.   Both vaccines raised significant cellular responses for Gag, Pol, and Env. Approximate twofold higher ELISPOT responses to Gag and Env epitopes were observed for IC48 animals than for IC1-90 animals at the peak post-MVA effector (P = 0.028) and late memory (P = 0.051) phases, respectively.   Greater breadth for IC48-primed animals was observed than for ICl-90-primed animals at peak response (P = 0.03).   Our results indicated that the vaccines elicited high frequency T cell responses and primed anti-Env antibody.   They also suggest that expression of different forms of VLP has a significant effect on elicited cellular and humoral immunity. Results Vaccine immunogens<br>
To assess the assembly of VLPs, transfection culture supernatants were banded through a sucrose gradient and subsequent fractions were examined for the presence of Gag.  Peak VLP fractions were analyzed by immunoblotting using rabbit polyclonal anri-p24 antibody (Fig. 41 A). Only two Gag polyproteins, p55 and p41, were detected for IC1-90. Three forms of Gag (p55, p41, and p24) were observed for IC48. The presence of Gag p24 indicates cleavage of the Gag polyprotein.   As previously demonstrated, complete inhibition of HIV protease activity by saquinavir (HIV-1 protease inhibitor) or vaccine construct IC25 (genetic, modification of protease activity) results in the Gag protein being primarily p55, indicating no cleavage of the Gag polyprotein (Ellenberger, D. et al. 2004 Virology 319:118-130). A summary of the vaccine constructs with corresponding protease mutations which result in the production of assorted products, including HIV protein aggregates and VLPs containing complete to partial cleavage of Gag, is shown in Fig. 41B. To confirm partial inhibition of protease activity permitted VLP assembly and possible particle maturation, cells transfected with protease-mutant vaccine constructs were examined by electron microscopy (Fig. 41C). For IC1-90, immature VLPs were seen (Fig. 41C). These particles were uniformly sized, abundant in number, and similar in appearance (electron-dense shells and lucent centers) to those formed by expression in the presence of saquinavir and IC25 (Ellenberger, D. et al, 2004 Virology 319:118-130).    Budding structures and numerous VLPs were also observed for IC48 (Fig. 41C).   However, two distinctive particle morphologies were observed: VLPs with a spherical protein shell (immature) and other particles with electron dense centers (mature). Furthermore, the VLP with electron-dense centers had a lean spherical protein shell, suggesting cleavage of Gag p55 and condensing of the viral core, comparable to mature HIV particles.    IC48-<br>
transfected cells were also observed to contain intracelhilar aggregates of HTV proteins (Fig. 41C).<br>
Total Gag expression was detected by antigen-capture ELISA following 293T cell transfection; 1C 1-90 expression appeared sharply diminished as compared with IC48, 131, and 878 ng/ml, respectively. In the supernatant and cell lysate, Gag protein levels were 77 and 54 ng/ml (IC1-90) and 624 and 254 ng/ml (IC48), respectively. However, as previously reported (Schneider, R, et al. 1997 J. Virol 71:4892-4903), the detection efficiency of Gag p55 was substantially less than that of Gag p24. Therefore, quantification of total Gag protein of transfection supematants by Western blot analysis as measured by phosphorimager demonstrated that IC1-90 expressed Gag protein to a level similar to IC48, 1771, and 1570 relative units. For IC1-90, the Gag protein was primarily p55 (61% of the rmmunoreactive protein). Production of Env protein was also detected hy ELISA, and IC1-90 and IC48 Env levels were similar, 696 and 665 ng/ml, respectively. A greater percentage of total Env protein was detected in the supernatant for both constructs. Preclinical safety testing<br>
Three DNA primes at 0, 8, and 26 weeks and a single rMVA booster at 41 weeks were administered in 1 ml by IM needle injections. DNA pruning was 0.6 mg of either IC1-90 or IC48 per injection and MVA booster was at 1 x 10^ pfu. All 16 macaques received identical aliquots only differing by the priming agent, IC1-90, or IC48. Physical examinations, daily observations, measuring for weight, hematology, and clinical chemistry were completed. No local redness or indurations at the injection site were observed. ELISPOT analysis<br>
Vaccine-elicited T cells expanded rapidly after the MVA booster (Fig. 42). Prior to the MVA booster, HIV-1-specific ELISPOT responses were at, or below, our background for detection following each of the DNA primes. Temporal ELISPOT analyses revealed that the highest ELISPOT response was 1 week after the MVA booster (Peak). Frequencies up to 5300 spots per one million peripheral blood mononuclear cells (PBMC) were found at the peak response. This peak of specific cells underwent a threefold contraction into the DNA/MVA memory pool (8 weeks post-MVA). T cell responses contracted an additional threefold at 26 weeks post-peak (late memory). Regardless of the immune phase (peak, memory, or late-memory) total ELISPOTS were greater for IC48-primed animals than IC1-90-primed animals; the T cell responses for IC48 animals were 2.2-fold greater (arithmetic mean) than IC1-90 animals. A random effects model using a linear regression of the<br>
aggregate ELISPOT values on group and time demonstrates that IC48 is higher than IC1-90, group effect has P value = 0.002 in the model. Furthermore, at peak response (P = 0.028) and late memory (P = 0.051), significantly higher responses by IC-48-primed animals were observed.<br>
At the peak response, both of tested plasmid DNAs successfully primed the MVA booster for an anti-viral IFN-y ELISPOT response to Gag, Pol, and Env sequences. The analyses revealed greater than twofold differences between the IC48 and IC1-90 primes (Fig. 43). The twofold difference was significant and observed in all tested gene regions of the virus, including Gag (P = 0.015), Pol (P = 0.047), and Env (P = 0.0003). For Pol and Env, the total ELISPOT mean responses were less than the response observed in Gag, but the twofold difference is evident. The limited Tat response was expected as Tat was not a common immunogen in the recombinant plasmid and rMVA.<br>
A broad ELISPOT response for both Gag and Env was present 1 week after the MVA booster (Fig. 44). Greater breadth of response to peptide pools for IC48-primed animals was observed than ICl-90-primed animals (P = 0.03). At the peak response, each of the 5 pools of Gag peptides and 7 pools of Env peptides was recognized by at least 3 of the immunized animals. On average per animal, 2.8 Gag and 2.9 Env peptide pools stimulated a cellular response as measured by ELISPOT. Much lower frequencies of responses were elicited for PR, RT, and Tat, but these pools were recognized by at least one or more of the animals. Four of the 16 animals recognized the one pool of PR peptides, 7 recognized one of the 5 pools of RT peptides, and one animal's cells were stimulated by the pool of Tat peptides.<br>
Consistent with the height of the ELISPOT response, T cells from IC48-primed animals recognized more peptide pools than observed with ICl-90-prhned animals (Table 7). hi the IC1-90 group, 6 of 8 animals responded to at least one Gag pool with an overall average of 2.5 of 5 pools being recognized (Table 7, Fig. 44). For the IC48 group, 7 of 8 animals responded to at least two Gag pools with an overall average of 3.1 pools being recognized. The Env response was similar in the IC48 group where 7 of 8 animals responded to at least two pools. In the IC1-90 group, only 6 of 8 animals responded to Env, recognizing at least one Env pool. In the memory phase of the immune response, less peptide pools stimulated T cell responses in all the vaccinated animals, but IC48 animals responded to more peptide pools than the IC1-90 group (5.3 vs. 3 pools), respectively.<br>
A broad response for both Gag and Env was retained 8 weeks post-MVA booster (Fig. 45). In the memory phase, IC48-primed animals responded to more epitopes (P = 0.004) than ICl-90-primed animals. Frequencies up to 2125 spots per million PBMC were found at the memory phase. Fifteen of the 16 animals recognized at least one pool of peptides. Monkeys AC36 and M509, initially nonresponders at 1-week post-MVA, were shown to recognize at least one Gag pool at 8 weeks post-MVA. Animal O2L remained nonresponsive to the peptide pools. At the memory response, each of the 5 pools of Gag peptides and 5 of 7 Env peptide pools were recognized by at least one of the immunized animals. Intracellular cytokine analysis<br>
To determine the frequencies of responding CD4 and CDS T cells, intracellular cytokine assays were perfonned (Fig. 46). Peptide induction of IFN-y and IL-2 expression in CD4 and CDS lymphocytes was measured, and both tested plasmid DNAs successfully primed the MVA booster. One week following the MVA booster, peak responses were observed as seen by ELISPOT analyses. Regardless of the gene region, the response by IC48-primed animals was significantly greater than ICl-90-primed animals: Env (P =<br>
0.006) and Gag (P = 0.026). CDS cells specific for Gag epitopes expanded to frequencies as high as 2.6% of total CDS T cells in one animal. Intracellular cytokine staining for IFN-y expression and IL-2 revealed an approximate fourfold difference to Gag peptides between the DNA plasmids; IC48 primed more CDS cells than IC1-90. The geometric mean of percent-specific CDS cells was 0.54 vs. 0.14 for the IC48 group and IC1-90 group, respectively. A twofold difference was detected in the Env region (0.28 vs. 0.14). The same ratios were observed regardless of the expressed cytokine.<br>
In several animals, CD4 cells specific for Gag epitopes expanded to frequencies as high as 0.5% of the total CD4 T cells (Fig. 46). Statistical analyses revealed that a significantly greater number of CD4 cells specific for Env epitopes were observed for the IC48-primed animals than 1C 1-90 animals (P = 0.006). However, staining for IFN-y expression and IL-2 revealed a fourfold difference (0.39 vs. 0.11 geometric means) to Env peptides and twofold difference (0.31 vs. 0.15) for Gag peptides between the DNA plasmids. Overall, CD4 cell responses were significantly greater than CDS cell responses (P 
Based on Fig. 46 data, geometric means were determined to compare specific cells (CD4 and CDS), cytokmes expressed (IFN-y and IL-2), and HTV gene regions. Peptide induction of IL-2 expression in CD4 lymphocytes was twofold greater than the percent-specific CDS lymphocytes (0.113 vs. 0.056 geometric mean). However, CDS and CD4 cells were similar for IFN-Y expression (0.089 vs. 0.078 geometric mean). In the Gag region, IFN-y expression was significantly greater than IL-2 expression (P = 0.013), but CD4-EL2, CD8-IL2, and CDS-IFN-y expressions were significantly greater than CD4-IFN (P = 0.032).<br>
The average number of peptide pools recorded as positive by intracellular cytokine staining to include both IFN-y and IL-2 expression were found to be relatively comparable between DNA priming groups regardless of the gene region (Table 7). At the peak immune response, 1C 1-90 animals would require a 31% and 11% increase in peptide pool responses to equal the response of IC48 animals for CDS and CD4 cells, respectively. However, threefold more peptide pools induced IFN-y and IL-2 expression in CD4 cells than CDS cells (6.25 vs. 2.2 pools). This same threefold ratio was observed for both IC1-90- and IC48-primed animals.<br>
Antibody responses<br>
The antibody response was strongly boosted by the MVA. By Western blot analysis, negligible binding antibody was elicited by DNA priming, but 11 of 16 animals (6 IC1-90 and 5 IC48 monkeys) showed Env reactivity against an HIV-1 subtype AG virus lysate 3 weeks following the MVA. All 11 reactive animals produced antibody which detected Env gp!20 and only one animal responded to Gag p24. There was no difference between the two groups of primed animals.<br>
Anti-Env titers were determined for the vaccinated animals by solid-phase Env ELISA. At 3 weeks post-MVA, the average reciprocal titers were 1075(±1569) and 2119(±1898) for the IC1-90- and IC48-primed animals, respectively. The titers fell to 953(±979) and 850(±563), respectively, at 8 weeks after the booster. There was no substantial difference observed between the two groups of animals.<br>
Tests for neutralizing antibody revealed very low levels of activity for the primary isolate used for the vaccine; only one animal scored positive.   The primary isolate, a recombinant HTV-1 subtype AG, was relatively resistant to neutralization by many other sera and neutralizing monoclonal antibodies. The neutralizing activity of the plasma also did not score for HIV-l-MN (subtype B), a laboratory-adapted isolate against HIV-1. Materials and methods DNA immunogens<br>
DNA constructs were made in the pGAl expression vector (GenBank accession no.: AF425297) that includes intron A in the promoter cassette and uses the CMV immediate-early promoter and the bovine growth hormone polyadenylation sequence to express RNAs (Chapman, B.S. et al. 1991 PNAS USA 19:3979-3986 and Ross, T.M. et al. 2000 Nat. Immunol. 1:127-131). An incident recombinant HIV-1 subtype AG (CRF02_AG) strain IC0928 was reverse transcribed (GenBank accession no.: AY227361 and AY227362), and fragments produced by DNA PCR were cloned into the pGAl expression vector (Ellenberger, D.L. et al. 2002 Virology 302:155-163 and Ellenberger, D. et al. 2004 Virology 319:118-130). Construction and description of pGAl/IC48 (IC48) have been previously described (Ellenberger, D. et al. 2004 Virology 319:118-130). pGAl/ICl-90 (1C 1-90) differs from IC48 by point mutations in gag and pol. Briefly, 1C 1-90 lacks the introduced point mutations in gag NC zinc fingers and pol RT as previously described for IC48 (Ellenberger, D. et al. 2004 Virology 319:118-130, Smith, J.M. et al. 2004 AIDS Res. Hum. Retrovir. 20:1335-1347 and Smith, J.M. et al. 2004 AIDS Res. Hum. Retrovir.<br>
20:654-665). IC1-90 and IC48 also differ in their protease sequence where there are two substitutions in 1C 1-90, an Arg-to-Gly substitution at position 70 and Met-to-Leu substitution at position 90. IC48 has a Gly-to-Val substitution at position 48 as previously described. The protease mutations were made using a site-directed mutagenesis kit (Stratagene, La Jolla, CA) following the manufacturer's protocol. All mutations were confirmed by sequencing. Recombinant MVA immunogen<br>
A recombinant MVA expressing gag, pol, and env regions derived from IC0928 is described above. Briefly, it was constructed with gag-pol inserted into deletion TH and a truncated env into del EL as described previously for MVA/SHIV89.6 (Earl, P.L. et al. 2002 Virology 294:270-281) with the modification that a transient GFP marker for obtaining recombinant virus was used, similar to the transient GUS described by Wyatt et al. (Wyatt, L.S. etal. 2004 AIDS Res. Hum. Retrovir. 20:645-653). Cell line and transient transfections<br>
Human 293T cells, a human kidney-derived cell line, were maintained in Dulbecco's modified Eagle medium (DMEM) supplemented with 10% fetal bovine serum. 293T cells were added at 106 cells/well of Costar 6-well plates in 2 ml of DMEM growth medium and incubated for 24 h at 37°C in a 5% C02 humidified atmosphere. After 24 h, LipofectAmine2000 reagent (Invitrogen, Carlsbad, CA) and plasmid DNA were mixed according to the manufacturer's protocol and added to each well. Supernatants were harvested 24 or 48 h following the addition of transfection reagents. Purification of particles<br>
Following 24 h of. transient transfection of 293T cells, culture supernatants were recovered, clarified by centrifugation at 200 x g for 5 min, and 1 ml was layered on top of 10-50% sucrose gradients. Gradients of 10 ml consisting of 2 ml each of 50%, 40%, 30%, 20%, and 10% sucrose solutions were added to ultraclear centrifuge tubes from bottom to top. Gradients were centrifuged at 40,000 rpm for 16 h in SW41T1 rotor. Fractions of 1 ml were recovered, top to bottom. Fraction aliquots were analyzed by HIV-1 p24-antigen-capture assay. Electron microscopy<br>
Transfected 293T cells were fixed in the multi-well plate using 1 ml of 2.5% glutaraldehyde in 0.1 M cacodylate buffer for 2 h at 4 °C. After three washes with the same buffer, 1.0% osmium tetroxide hi 0.1 M cacodylate buffer was added, incubated for 1 h,<br>
dehydrated through aii ethanol series, and embedded with Eponate 12 resin.   Following polymerization of the resin, the cells were sectioned en face, stained with 4% uranyl acetate in water and lead citrate, and observed on a Hitachi H-7500 transmission electron microscope. Antigen-capture assay<br>
Assay was performed using an HTV-1 antigen-capture EIA kit (Coulter, Hialeah, FL) according to the manufacturer's instructions. Env ELISA<br>
Pooled human immunoglobulin anti-HTV (catalog no. 3957) was obtained from the National Institute of Allergy and Infectious Diseases AIDS Research and Reference Reagent Program (Rockville, MD).    Env ELISA was done as previously described (Ellenberger, D. etal 2004 Virology 319:118-130). Macaques and immunogenicity trial<br>
Sixteen young adult male outbred rhesus macaques (Macaco, mulatto) of Indian origin were randomized by weight into two vaccine groups of 8. The IC1-90 group received 0.6 mg of IC1-90 DNA and the IC48 group received 0.6 mg of IC48 DNA intramuscularly (IM) at weeks 0, 8, and 26. DNA immunizations were delivered in phosphate-buffered saline with a needle and syringe. A total of 3 injections were delivered to the upper lateral right thigh in a volume of 1 ml/injection. Fifteen weeks following the 3rd DNA immunization (week 41), all animals received 1 x 10^ pfu of a modified vaccinia Ankara virus (MVA/HIV) booster EVI using a needle and syringe in the upper lateral right thigh. The MVA immunization was delivered in phosphate-buffered saline in a volume of 1 ml. Injection sites were monitored for local inflammatory reactions. Humoral responses<br>
Western blot analysis of whole virus lysate and sucrose-purified VLP was performed as previously described (Sambrook, J. et al. 1989 Molecular Cloning: A Laboratory Manual (2nd ed.), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY). Whole virus lysate (IC0928) was run on 4-15% gradient SDS gel and transferred to nitrocellulose. Plasma samples were diluted 1:100 in blocking buffer. The secondary antibody was anti-human IgG (Fab)-phosphatase and detection was completed by using the BCIP/NBT phosphatase substrate system (KPL, Gaithersburg, MD) according to manufacturer's instructions. Analysis of sucrose-purified VLP was completed using rabbit polyclonal anti-p24 (catalog no. 4250) which was obtained from the National Institute of<br>
Allergy and Infectious Diseases AIDS Research and Reference Reagent Program. HIV-1 Gag protein bands were visualized using the ECL Western blotting detection reagent (Amersham Pharmacia Biotech, Piscataway, NJ) as described by the manufacturer. Expression levels were determined using a phosphoimager.<br>
To measure the levels of Env IgG in sera of immunized monkeys, 96-well U-bottom plates were coated with sheep antibody to the C-terminus of gp!20 followed by purified CRF02_AG gp!40 as previously described (Earl, P.L. et al. 2002 Virology 294:270-281). The time of incubation of serum dilutions was increased from 2 h to overnight for enhanced sensitivity of the ELISA.<br>
Neutralization was measured as a function of reductions in luciferase reporter gene expression after a single round of infection in TZM-bl cells. TZM-bl cells were obtained from the NTH AIDS Research and Reference Reagent Program, as contributed by John Kappes and Xiaoyun Wu. Briefly, 200 TCID50 of virus was incubated with serial threefold dilutions of serum sample in triplicate in a total volume of 150 ul for 1 h at 37 °C in 96-well flat-bottom culture plates. Freshly trypsinized cells (10,000 cells in 100 nl of growth medium containing 75 (J.g/ml DEAE dextran) were added to each well. Indinavir was added at a final concentration of 1 [XM to prevent replication in the case of HTV-1 MN, One set of control wells received cells + virus (virus control) and another set received cells only (background control). After a 48-h incubation, 100 ul of cells was transferred to a 96-well black solid plates (Costar) for measurements of luminescence using Bright Glo substrate solution as described by the supplier (Promega). An assay stock of virus pseudotyped with gp!60 from the CRF02_AG vaccine strain was prepared by transfection in 293T cells and titrated in TZM-bl cells. The assay stock of HIV-1 MN was prepared in H9 cells. Peptide pools<br>
HIV-1 recombinant subtype CRF02_AG peptides were ISmers overlapping by 11 derived from the vaccine strain.     Peptides were assembled into  pools containing approximately 25 peptides. Peptides were dissolved in DMSO at 50-100 mg/ml and stock solutions maintained at -70 °C. Peptide working solutions were kept at -20 °C for 1 week. T cell assays<br>
ELISPOT assays were done as previously described (Amara et al., 2001 and Amara et al., 2002), except that 2 pg/ml of antibody to human CD28 and CD49d (Becton Dickinson, San Jose, CA) was included in incubations (Waldrop, S.L. et al. 1998 J. Immunol. 161:5284-5295). Purified PBMC were plated in duplicate in 96-well plates at 2<br>
x 105 cells/well with peptide pools used at a final concentration of 10 (J.g/ml for each peptide. Four wells received only media. Plates were incubated at 37 °C, 5% COz for 36 h, and spots counted using an ELISPOT reader. Background was set at 2 the average of the negative control wells plus 10. This baclcground value was subtracted from the raw counts of the peptide wells before conversion to 1 million PBMC. Only ELISPOT counts of twice the background (1 00) were considered significant.<br>
Intracellular cytokine (ICC) staining assays for IFN-y and IL-2 have been described previously (Amara, R.R. el al. 2001 Science 292:69-74 and Amara, R.R. et al 2002 J. Virol. 76:6138-6146). For those assays, one million PBMC were resuspended in 100 (il of RPMI in a 5-ml polypropylene tube and stimulated with 100 ug/ml of the peptide pool in the presence of anti-CD28 and anti-CD49d antibody at a final concentration of 1 ug/ml. The cells were incubated at 37 °C, in a tissue culture incubator at an angle of 5° in 100 jol. At 2 h, 900 ul of medium with 10 |Ag of Brefeldin A was added and the incubation continued for an additional 4 h. Cells were then stained for CD3, CDS, CD69, IFN-y, and IL-2 fixed with 1% formaldehyde in PBS, and acquired within 24 h on a FACScaliber (Becton Dickinson, San Jose, CA). Approximately 150,000 lymphocytes were acquired for each sample. Data were analyzed using FloJo software (Tree Star, Inc., San Carlos, CA). Statistical analysis<br>
The Wilcoxon test for independent samples and signed-rank test were used for comparison of immunogenicity tests from groups primed with the two DNA vaccines. A mixed-effects model using a linear regression of the aggregate ELISPOT values on group (IC-48 or IC1-90) and time with monkeys as the random effect was used. Also, a mixed-effects linear regression model was used for testing responses of CD4 versus CDS T cells, T cell responses against Gag versus Env, and IFN-y and EL-2 expression.<br>
While the present invention has been described in some detail for purposes of clarity and understanding, one skilled in the art will appreciate that various changes in form and detail can be made without departing from the true scope of the invention. All figures, tables, and appendices, as well as patents, applications, and publications, referenced to above, are hereby incorporated by reference.<br><br><br><br>
We Claim:<br>
1. A recombinant MVA virus comprising a sequence encoding an HIV Env antigen<br>
inserted into deletion site II of an MVA wherein the HIV Env antigen comprises gpl20 and<br>
the membrane-spanning and ectodomain of gp41 but lacking part or all of the cytoplasmic<br>
domain of gp41, and a sequence encoding HIV Gag and Pol antigens inserted into deletion<br>
site III of an MVA, wherein the sequence encoding the HIV Gag and Pol antigens comprises<br>
safety mutations corresponding to clade B D185N, W266T, and E478Q to inactivate reverse<br>
transcriptase and delete the integrase gene; and wherein the sequences are under the control<br>
of promoters compatible with poxvirus expression systems.<br>
2. The recombinant MVA virus as claimed in claim 1, wherein the sequence encoding<br>
the HIV Env antigen comprises a sequence that is at least 97% identical to SEQ ID N0:1 and<br>
the sequence encoding the HIV Gag and Pol antigens comprises a sequence that is at least<br>
97% identical to SEQ ID NO: 2.<br>
3. The recombinant MVA virus as claimed in claim 1, wherein the sequence encoding<br>
the HIV Env antigen comprises a sequence that is at least 97% identical to SEQ ID NO:3 and<br>
the sequence encoding the HIV Gag and Pol antigens comprises a sequence that is at least<br>
97% identical to SEQ ID NO: 4.<br>
4. The recombinant MVA virus as claimed in claim 1, wherein the sequence encoding<br>
the HIV Env antigen comprises a sequence that is at least 97% identical to SEQ ID NO:5 and<br>
the sequence encoding the HIV Gag and Pol antigens comprises a sequence that is at least<br>
97% identical to SEQ ID NO: 6.<br>
5 The recombinant MVA virus as claimed in claim 1 wherein the sequence between the<br>
first promoter and the initiation codon of the seqnce encoding the HIV Env antigen comprises<br>
the sequence ATG.<br>
6. The recombinant MVA virus as claimed in any of the preceding claims, wherein said<br>
recombinant MVA virus is MVA 1974/NIH Clone 1.<br>
7. The recombinant MVA vims as claimed in any of the preceding claims, wherein said<br>
recombinant MVA virus additionally expresses an additional HIV gene or modified gene<br>
thereof for production of an HIV antigen by expression fi-om said recombinant MVA virus,<br>
wherein said additional HIV gene is a member selected from the group consisting of vif, vpr,<br>
tat, rev, vpu, and nef.<br>
8. The recombinant MVA virus as claimed in any of the preceding claims, wherein said<br>
HIV env, gag, or pol gene or modified gene thereof is vmder transcriptional regulation of the<br>
mH5 promoter.<br>
9. The recombinant MVA virus as claimed in any of the preceding claims, wherein said<br>
HIV env, gag, or pol gene or modified gene thereof is under transcriptional regulation of a<br>
promoter, and wherein an intervening sequence between the end of the promoter and the start<br>
site of the env gene, or modified gene thereof, lacks an intervening initiation codon.<br>
10. The recombinant MVA virus as claimed in claim 9, wherein the intervening sequence<br>
hasSEQIDNO:9orll.<br>
11. The recombinant MVA virus as claimed in claim 1 wherein the promoter is m2H5<br>
having SEQ ID NO: 12.<br>
12. A method of making a recombinant MVA virus as claimed in any of claims 1-10<br>
comprising preparing a plasmid transfer vector encoding an HIV end, gag, and pol gene or<br>
modified gene thereof, wherein said HIV env gene is modified to encode an HIV Env protein<br>
composed of gpl20 and the membrane-spaiming and ectodomain of gp41 but lacking part or<br>
all of the cytoplasmic domain of gp41, and recombining said plasmid transfer vector with a<br>
MVA virus to produce a recombinant MVA virus of any of claims 1-10.<br>
Dated: 08/03/2007 Cl^^<br>
[SWARfepKtjMAR]<br>
OF REMFRY &amp; SAGAR<br>
ATTORNEY FOR THE APPLICANTS</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUFic3RyYWN0LSgyMS0wOC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Abstract-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUFzc2dpbm1lbnQtKDE1LTAzLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Assginment-(15-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1ERUxOUC0yMDA3LUFzc2lnbm1lbnQtKDAxLTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1814-DELNP-2007-Assignment-(01-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNsYWltcy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Claims-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNsYWltcy0oMjQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Claims-(24-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence Others-(20-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence Others-(24-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDEtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1814-DELNP-2007-Correspondence-Others-(01-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDUtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence-Others-(05-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTUtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence-Others-(15-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLW90aGVycy0oMjEtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence-others-(21-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence-Others-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-DELNP-2007-Correspondence-Others-(22-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjQtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Correspondence-Others-(24-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUZvcm0tMS0oMjAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Form-1-(20-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUZvcm0tMTgtKDIxLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Form-18-(21-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUZvcm0tMi0oMjQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Form-2-(24-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUZvcm0tMy0oMDUtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Form-3-(05-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUZvcm0tMy0oMjAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Form-3-(20-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUZvcm0tMy0oMjQtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Form-3-(24-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LWZvcm0tNi0oMDEtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-form-6-(01-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1ERUxOUC0yMDA3LUdQQS0oMDEtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1814-DELNP-2007-GPA-(01-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1ERUxOUC0yMDA3LUdQQS0oMjItMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-DELNP-2007-GPA-(22-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LUdQQS0oMjQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-GPA-(24-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTUtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Petition-137-(15-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-Petition-137-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXVyLUFic3RyYWN0LSgyMS0wOC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-ur-Abstract-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXVyLUNsYWltcy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-ur-Claims-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXVyLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-ur-Correspondence-Others-(21-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgxNC1kZWxucC0yMDA3LXVyLVBldGl0aW9uLTEzNy0oMjEtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1814-delnp-2007-ur-Petition-137-(21-08-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260242-a-multi-hop-wireless-mesh-network-medium-access-control-protocol.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260244-an-improved-process-for-producing-leathers-in-more-than-one-tone.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260243</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1814/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>16/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>18-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>15-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Mar-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GOVERNMENT OF THE UNITED STATES OF AMERICA AS REPRESENTED BY THE SECRETARY, DEPARTMENT OF HEALTH AND HUMAN SERVICES</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NATIONAL INSTITUTE OF HEALTH, OFFICE OF TECHNOLOGY TRANSFER, 6011 EXECUTIVE BOULEVARD, SUITE 325, ROCKVILLE, MD 20852-3804, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BERNARD MOSS</td>
											<td>10301 DICKENS AVENUE, BETHESDA, MA 20814, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LINDA S.WYATT</td>
											<td>11201 HUNTOVER DRIVE, ROCKVILLE, MD 20852, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HARRIET L.ROBINSON</td>
											<td>1051 MASON WOODS DRIVE, ATLANTA, GEORGIA 30329, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/16</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/030977</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-08-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/604,918</td>
									<td>2004-08-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260243-recombinant-mva-viruses-expressing-clade-c-modified-hiv-env-hiv-gag-and-pol-genes by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:11:42 GMT -->
</html>
