<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260329-compositions-against-sars-coronavirus-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:38:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260329:COMPOSITIONS AGAINST SARS-CORONAVIRUS AND USES THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">COMPOSITIONS AGAINST SARS-CORONAVIRUS AND USES THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides compositions of binding molecules specifically binding to a coronavirus such as SARS-CoV and capable of neutralizing an infection caused by the virus. The compositions are suitable for diagnosing, preventing and/or treating a condition resulting from a coronavirus such as SARS-CoV.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
TITLE OF THE INVENTION Compositions against SARS-coronavirus and uses thereof<br>
FIELD OF THE INVENTION The invention relates to medicine. In particular the invention relates to compositions comprising binding molecules capable of specifically binding to and neutralizing SARS-coronavirus (SARS-CoV). The compositions are useful in the diagnosis of SARS-CoV and the prophylaxis and/or treatment of a condition resulting from SARS-CoV.<br>
BACKGROUND OF THE INVENTION Recently a new and in several cases deadly clinical syndrome was observed in the human population, now called severe acute respiratory syndrome (SARS) (Holmes, 2003). The syndrome is caused by a novel coronavirus (Ksiazek et  al-, 2003), referred to as the SARS-CoV. The genome sequence of SARS-CoV has been determined (Rota et al., 2003; Marra et  al., 2003). However, much remains to be learnt about this virus, and means and methods for diagnostics, prophylaxis and/or treatment of the virus and the syndrome are needed. The present invention provides means and methods for use in diagnostics, prevention and/or treatment of SARS-CoV.<br>
DESCRIPTION OF THE FIGURES Figure 1 shows results from an ELISA, wherein the binding of the single-chain phage antibodies called SC03-014 and SC03-022 to an immobilized UV-inactivated SARS-CoV preparation (left column) or immobilized FBS (right column) was measured. The binding of the control single-chain phage antibody called SC02-006 is also shown. On the y-axis the absorbance at 4 92 nm is shown.<br><br>
figure 2 shows an ELISA binding of IgGs CR03-014, CR03-022, control IgG and no IgG to an inactivated SARS-CoV preparation. Dn the Y-axis the absorbance at 4 92 nm is shown.<br>
figure 3 shows a FACS binding of the scFv phage antibodies 3C03-014, SC03-022 and a control scFv phage antibody to the full length S protein expressed on HEK293T cells (left column) Dr mock transfected HEK293T cells (right column). On the Y-axis the mean fluorescense intensity is shown.<br>
figure 4 shows an ELISA binding of the antibodies CR03-014, ZRO3-022 and a control antibody to the S565 fragment (amino acids 1-565 of the S protein of SARS-CoV; left column), S318-510 fragment (amino acids 318-510 of the S protein of SARS-3oV; middle column) and an irrelevant control myc-tagged antigen (right column). On the Y-axis the absorbance at 4 92 nm Ls shown.<br>
figure 5 shows an ELISA binding of dilutions of antibodies :R03-014, CR03-022 and a control antibody to the S565 fragment Df the S protein of SARS-CoV. On the Y-axis the absorbance at i92 nm and on the X-axis the amount of antibody in pg/ml is shown.<br>
figure 6 shows a competition ELISA for binding to the S565 fragment. Figure 6A shows competition between biotinylated antibody CR03-014 without competing antibody or with 1, 5 or 25 ug/ml competing antibody CR03-014, CR03-022 or control antibody. Figure 6B shows competition between biotinylated antibody CR03-022 with or without the competing antibodies as iescribed above. On the Y-axis the % of maximal binding is<br><br>
shown and on the X-axis the amount of the competing antibody in jjg/ml.<br>
Figure 7 shows a sandwich ELISA using anti-S protein antibodies. Immobilized antibodies CR03-014 and CR03-022 were used to capture S protein fragment S318-510. Bound fragment was detected using biotinylated antibody CR03-014, CR03-022 or control antibody. On the Y-axis the absorbance at 4 92 nm is shown.<br>
Figure 8 shows binding of the monoclonal anti-SARS-antibodies CR03-014 and CR03-022 to the amino acid region of 318-510 of the S protein of the SARS-CoV strain Frankfurt 1 (called WT S318-510) and naturally occuring variants of the WT S318-510 fragment (variant A, mutation K344R; variant B, mutation S353F; variant C, mutation R426G and N437D; variant D, mutation Y436H; variant E, mutation Y442S; variant F, mutation N479S; variant G, mutation K344R, F360S, N479K and T487S; variant H, mutation K344R, F360S, L472P, D480G, and T487S; variant I, mutation K344R and F501Y). The control is an irrelevant myc-His tagged protein. On the Y-axis the absorbance at 492 nm is shown.<br>
Figure 9 shows the comparison of the nucleotide and amino acid sequences of the SARS-CoV wild-type strain (SARS-CoV strain HKU 39849) and escape viruses of antibody CR03-014. Virus-infected cells were harvested 2 days post-infection and total RNA was isolated. cDNA was generated and used for DNA sequencing. Regions containing mutations are shown and the mutations are indicated in bold. Numbers above amino acids indicate amino acids numbers from S protein including signal<br><br>
peptide. The sequences in Figure 9 are also represented by SEQ ID Nos:118-121.<br>
Figure 10 shows binding of the monoclonal anti-SARS-antibodies CR03-014 and CR03-022 to the amino acid region of 318-510 of the S protein of the SARS-CoV strain Frankfurt 1 (called FRA1 S318-510) and an escape variant of antibody CR03-014 harboring a P462L substitution. On the Y-axis the absorbance at 492 nm is shown.<br>
DESCRIPTION OF THE INVENTION Herebelow follow definitions of terms as used in the invention<br>
DEFINITIONS<br>
Binding molecule<br>
As used herein the term "binding molecule" refers to an intact immunoglobulin including monoclonal antibodies, such as chimeric, humanised or human monoclonal antibodies, or to an antigen-binding and/or variable domain comprising fragment of an immunoglobulin that competes with the intact immunoglobulin for specific binding to the binding partner of the immunoglobulin, e. g. the SARS-CoV. Regardless of structure, the antigen-binding fragment binds with the same antigen that is recognised by the intact immunoglobulin. An antigen-binding fragment can comprise a peptide or polypeptide comprising an amino acid sequence of at least 2 contiguous amino acid residues, at least 5 contiguous amino acid residues, at least 10 contiguous amino acid residues, at least 15 contiguous amino acid residues, at least 2 0 contiguous amino acid residues, at least 25 contiguous amino acid residues, at least 30 contiguous amino acid residues, at least 35 contiguous<br><br>
amino acid residues, at least 40 contiguous amino acid residues, at least 50 contiguous amino acid residues, at least 60 contiguous amino residues, at least 70 contiguous amino acid residues, at least contiguous 8 0 amino acid residues, at least contiguous 90 amino acid residues, at least contiguous 100 amino acid residues, at least contiguous 125 amino acid residues, at least 150 contiguous amino acid residues, at least contiguous 175 amino acid residues, at least 200 contiguous amino acid residues, or at least contiguous 250 amino acid residues of the amino acid sequence of the binding molecule.<br>
The term "binding molecule", as used herein includes all immunoglobulin classes and subclasses known in the art. Depending on the amino acid sequence of the constant domain of their heavy chains, binding molecules can be divided into the five major classes of intact antibodies: IgA, IgD, IgE, IgG, and IgM, and several of these may be further divided into subclasses (isotypes), e.g.,   IgAl, IgA2, IgGl, IgG2, IgG3 and IgG4.<br>
Antigen-binding fragments include, inter alia,   Fab, F(ab'), F(ab?)2, Fv, dAb, Fd, complementarity determining region (CDR) fragments, single-chain antibodies (scFv), bivalent single-chain antibodies, single-chain phage antibodies, diabodies, triabodies, tetrabodies, (poly)peptides that contain at least a fragment of an immunoglobulin that is sufficient to confer specific antigen binding to the (poly)peptide, etc. The above fragments may be produced synthetically or by enzymatic or chemical cleavage of intact immunoglobulins or they may be genetically engineerd by recombinant DNA techniques. The methods of production are well known in the art and are described, for example, in Antibodies: A Laboratory Manual, Edited by: E. Harlow and D,<br><br>
Lane (1988), Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, which is incorporated herein by reference. A binding molecule or antigen-binding fragment thereof may have one or more binding sites. If there is more than one binding site, the binding sites may be identical to one another or they may be different.<br>
The binding molecule can be a naked or unconjugated binding molecule but can also be part of an immunoconjugate. A naked or unconjugated binding molecule is intended to refer to a binding molecule that is not conjugated, operatively linked or otherwise physically or functionally associated with an effector moiety or tag, such as inter alia  a toxic substance, a radioactive substance, a liposome, an enzyme. It will be understood that naked or unconjugated binding molecules do not exclude binding molecules that have been stabilized, multimerized, humanized or in any other way manipulated, other than by the attachment of an effector moiety or tag. Accordingly, all post-translationally modified naked and unconjugated binding molecules are included herewith, including where the modifications are made in the natural binding molecule-producing cell environment, by a recombinant binding molecule-producing cell, and are introduced by the hand of man after initial binding molecule preparation. Of course, the term naked or unconjugated binding molecule does not exclude the ability of the binding molecule to form functional associations with effector cells and/or molecules after administration to the body, as some of such interactions are necessary in order to exert a biological effect.<br>
Biological  sample<br>
As used herein, the term "biological sample" encompasses a<br>
variety of sample types, including blood and other liquid<br><br>
samples of biological origin, solid tissue samples such as a biopsy specimen or tissue cultures, or cells derived therefrom and the progeny thereof. The term also includes samples that have been manipulated in any way after their procurement, such as by treatment with reagents, solubilization, or enrichment for certain components, such as proteins or polynucleotides. The term encompasses various kinds of clinical samples obtained from any species, and also includes cells in culture, cell supernatants and cell lysates.<br>
Complementarity determining regions   (CDR)<br>
The term "complementarity determining regions" as used herein means sequences within the variable regions of binding molecules, such as immunoglobulins, that usually contribute to a large extent to the antigen binding site which is complementary in shape and charge distribution to the epitope recognised on the antigen. The CDR regions can be specific for linear epitopes, discontinuous epitopes, or conformational epitopes of proteins or protein fragments, either as present on the protein in its native conformation or, in some cases, as present on the proteins as denatured, e.g., by solubilization in SDS. Epitopes may also consist of posttranslational modifications of proteins.<br>
Expression-regulating nucleic acid seguence<br>
The term "expression-regulating nucleic acid sequence" as used herein refers to polynucleotide sequences necessary for and/or affecting the expression of an operably linked coding sequence in a particular host organism. The expression-regulating nucleic acid sequences, such as inter alia   appropriate transcription initiation, termination, promoter, enhancer sequences; repressor or activator sequences; efficient RNA<br><br>
processing signals such as splicing and polyadenylation signals; sequences that stabilize cytoplasmic mRNA; sequences that enhance translation efficiency (e.g., ribosome binding sites); sequences that enhance protein stability; and when desired, sequences that enhance protein secretion, can be any nucleic acid sequence showing activity in the host organism of choice and can be derived from genes encoding proteins, which are either homologous or heterologous to the host organism. The identification and employment of expression-regulating sequences is routine to the person skilled in the art.<br>
Functional  variant<br>
The term "functional variant", as used herein, refers to a binding molecule that comprises a nucleotide and/or amino acid sequence that is altered by one or more nucleotides and/or amino acids compared to the nucleotide and/or amino acid sequences of the parent binding molecule and that is still capable of competing for binding to the binding partner, e.g. SARS-CoV, with the parent binding molecule. In other words, the modifications in the amino acid and/or nucleotide sequence of the parent binding molecule do not significantly affect or alter the binding characteristics of the binding molecule encoded by the nucleotide sequence or containing the amino acid sequence, i.e. the binding molecule is still able to recognize and bind its target. The functional variant may have conservative sequence modifications including nucleotide and amino acid substitutions, additions and deletions. These modifications can be introduced by standard techniques known in the art, such as site-directed mutagenesis and random PCR-mediated mutagenesis, and may comprise natural as well as non-natural nucleotides and amino acids.<br><br>
Conservative amino acid substitutions include the ones in which the amino acid residue is replaced with an amino acid residue having similar structural or chemical properties. Families of amino acid residues having similar side chains have been defined in the art. These families include amino acids with basic side chains (e.g./ lysine, arginine, histidine), acidic side chains {e.g.,   aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cystine, tryptophan), nonpolar side chains (e.g.,   alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). It will be clear to the skilled artisan that other classifications of amino acid residue families than the one used above can also be employed. Furthermore, a variant may have non-conservative amino acid substitutions, e.g., replacement of an amino acid with an amino acid residue having different structural or chemical properties. Similar minor variations may also include amino acid deletions or insertions, or both. Guidance in determining which amino acid residues may be substituted, inserted, or deleted without abolishing immunological activity may be found using computer programs well known in the art.<br>
A mutation in a nucleotide sequence can be a single alteration made at a locus (a point mutation), such as transition or transversion mutations, or alternatively, multiple nucleotides may be inserted, deleted or changed at a single locus. In addition, one or more alterations may be made at any number of loci within a nucleotide sequence. The mutations may be performed by any suitable method known in the art.<br><br>
Host<br>
The term "host", as used herein, is intended to refer to an organism or a cell into which a vector such as a cloning vector or an expression vector has been introduced. The organism or cell can be prokaryotic or eukaryotic. It should be understood that this term is intended to refer not only to the particular subject organism or cell, but to the progeny of such an organism or cell as well. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent organism or cell, but are still included within the scope of the term "host" as used herein.<br>
Human<br>
The term "human", when applied to binding molecules as defined herein, refers to molecules that are either directly derived from a human or based upon a human sequence. When a binding molecule is derived from or based on a human sequence and subsequently modified, it is still to be considered human as used throughout the specification. In other words, the term human, when applied to binding molecules is intended to include binding molecules having variable and constant regions derived from human germline immunoglobulin sequences based on variable or constant regions either or not occuring in a human or human lymphocyte or in modified form. Thus, the human binding molecules may include amino acid residues not encoded by human germline immunoglobulin sequences, comprise substitutions and/or deletions (e.g., mutations introduced by for instance random or site-specific mutagenesis in   vitro  or by somatic mutation in   vivo)-   "Based on" as used herein refers<br><br>
to the situation that a nucleic acid sequence may be exactly copied from a template, or with minor mutations, such as by error-prone PCR methods, or synthetically made matching the template exactly or with minor modifications. Semisynthetic molecules based on human sequences are also considered to be human as used herein.<br>
Isolated<br>
The term "isolated", when applied to binding molecules as defined herein, refers to binding molecules that are substantially free of other proteins or polypeptides, particularly free of other binding molecules having different antigenic specificities, and are also substantially free of other cellular material and/or chemicals. For example, when the binding molecules are recombinantly produced, they are preferably substantially free of culture medium, and when the binding molecules are produced by chemical synthesis, they are preferably substantially free of chemical precursors or other chemicals, i.e., they are separated from chemical precursors or other chemicals which are involved in the synthesis of the protein. The term "isolated" when applied to nucleic acid molecules encoding binding molecules as defined herein, is intended to refer to nucleic acid molecules in which the nucleotide sequences encoding the binding molecules are free of other nucleotide sequences, particularly nucleotide sequences encoding binding molecules that bind binding partners other than SARS-CoV. Furthermore, the term "isolated" refers to nucleic acid molecules that are substantially separated from other cellular components that naturally accompany the native nucleic acid molecule in its natural host, e.g., ribosomes, polymerases, or genomic sequences with which it is naturally associated. Moreover, "isolated" nucleic<br><br>
acid molecules, such as a cDNA molecules, can be substantially free of other cellular material, or culture medium when produced by recombinant techniques, or substantially free of chemical precursors or other chemicals when chemically synthesized.<br>
Monoclonal antibody<br>
The term "monoclonal antibody" as used herein refers to a preparation of antibody molecules of single molecular composition. A monoclonal antibody displays a single binding specificity and affinity for a particular epitope. Accordingly, the term "human monoclonal antibody" refers to an antibody displaying a single binding specificity which have variable and constant regions derived from or based on human germline immunoglobulin sequences or derived from completely synthetic sequences. The method of preparing the monoclonal antibody is not relevant.<br>
Nucleic acid molecule<br>
The term "nucleic acid molecule" as used in the present invention refers to a polymeric form of nucleotides and includes both sense and antisense strands of RNA, cDNA, genomic DNA, and synthetic forms and mixed polymers of the above. A nucleotide refers to a ribonucleotide, deoxynucleotide or a modified form of either type of nucleotide. The term also includes single- and double-stranded forms of DNA. In addition, a polynucleotide may include either or both naturally-occurring and modified nucleotides linked together by naturally-occurring and/or non-naturally occurring nucleotide linkages. The nucleic acid molecules may be modified chemically or biochemically or may contain non-natural or derivatized nucleotide bases, as will be readily<br><br>
appreciated by those of skill in the art. Such modifications include, for example, labels, methylation, substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoramidates, carbamates, etc.), charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), pendent moieties (e.g., polypeptides), intercalators (e.g., acridine, psoralen, etc.), chelators, alkylators, and modified linkages (e.g., alpha anomeric nucleic acids, etc.). The above term is also intended to include any topological conformation, including single-stranded, double-stranded, partially duplexed, triplex, hairpinned, circular and padlocked conformations. Also included are synthetic molecules that mimic polynucleotides in their ability to bind to a designated sequence via hydrogen bonding and other chemical interactions. Such molecules are known in the art and include, for example, those in which peptide linkages substitute for phosphate linkages in the backbone of the molecule. A reference to a nucleic acid sequence encompasses its complement unless otherwise specified. Thus, a reference to a nucleic acid molecule having a particular sequence should be understood to encompass its complementary strand, with its complementary sequence. The complementary strand is also useful, e.g., for antisense therapy, hybridization probes and PCR primers.<br>
OperaJbiy linked<br>
The term "operably linked" refers to two or more nucleic acid sequence elements that are usually physically linked and are in a functional relationship with each other. For instance, a promoter is operably linked to a coding sequence if the promoter is able to initiate or regulate the transcription or<br><br>
expression of a coding sequence, in which case the coding sequence should be understood as being "under the control of" the promoter.<br>
Pharmaceutically acceptable  excipient<br>
By "pharmaceutically acceptable excipient" is meant any inert substance that is combined with an active molecule such as a drug, agent, or binding molecule for preparing an agreeable or convenient dosage form. The "pharmaceutically acceptable excipient" is an excipient that is non-toxic to recipients at the dosages and concentrations employed and is compatible with other ingredients of the formulation comprising the drug, agent or binding molecule.<br>
Specifically Binding<br>
The term "specifically binding", as used herein, in reference to the interaction of a binding molecule, e.g. an antibody, and its binding partner, e.g. an antigen, means that the interaction is dependent upon the presence of a particular structure, e.g. an antigenic determinant or epitope, on the binding partner. In other words, the antibody preferentially binds or recognizes the binding partner even when the binding partner is present in a mixture of other molecules or organisms. The binding may be mediated by covalent or non-covalent interactions or a combination of both. In yet other words, the term "specifically binding" means<br>
immunospecifically binding to an antigen or a fragment thereof and not immunospecifically binding to other antigens. A binding molecule that immunospecifically binds to an antigen may bind to other peptides or polypeptides with lower affinity as determined by, e.g., radioimmunoassays (RIA), enzyme-linked immunosorbent assays (ELISA), BIACORE, or other assays known<br><br>
in the art. Binding molecules or fragments thereof that immunospecifically bind to an antigen may be cross-reactive with related antigens. Preferably, binding molecules or fragments thereof that immunospecifically bind to an antigen do not cross-react with other antigens.<br>
Therapeutically effective  amount<br>
The term "therapeutically effective amount" refers to an amount of the binding molecule as defined herein that is effective for preventing, ameliorating and/or treating a condition resulting from infection with SARS-CoV.<br>
Treatment<br>
The term "treatment" refers to therapeutic treatment as well as prophylactic or preventative measures to cure or halt or at least retard disease progress. Those in need of treatment include those already inflicted with a condition resulting from infection with SARS-CoV as well as those in which infection with SARS-CoV is to be prevented. Subjects partially or totally recovered form infection with SARS-CoV might also be in need of treatment. Prevention encompasses inhibiting or reducing the spread of SARS-CoV or inhibiting or reducing the onset, development or progression of one or more of the symptoms associated with infection with SARS-CoV.<br>
Vector<br>
The term "vector" denotes a nucleic acid molecule into which a second nucleic acid molecule can be inserted for introduction into a host where it will be replicated, and in some cases expressed. In other words, a vector is capable of transporting a nucleic acid molecule to which it has been linked. Cloning as well as expression vectors are contemplated by the term<br><br>
"vector", as used herein. Vectors include, but are not limited to, plasmids, cosmids, bacterial artificial chromosomes (BAC) and yeast artificial chromosomes (YAC) and vectors derived from bacteriophages or plant or animal (including human) viruses. Vectors comprise an origin of replication recognised by the proposed host and in case of expression vectors, promoter and other regulatory regions recognised by the host. A vector containing a second nucleic acid molecule is introduced into a cell by transformation, transfection, or by making use of viral entry mechanisms. Certain vectors are capable of autonomous replication in a host into which they are introduced {e.g.,   vectors having a bacterial origin of replication can replicate in bacteria). Other vectors can be integrated into the genome of a host upon introduction into the host, and thereby are replicated along with the host genome.<br>
SUMMARY OF THE INVENTION The invention provides compositions of binding molecules capable of specifically binding to SARS-CoV and having SARS-CoV neutralizing activity. In a preferred embodiment, said binding molecules are human binding molecules. The invention further provides for the use of the compositions of the invention in the prophylaxis and/or treatment of a subject having, or at risk of developing, a condition resulting from SARS-CoV. Besides that, the invention pertains to the use of the compositions of the invention in the diagnosis/detection of SARS-CoV.<br>
DETAILED DESCRIPTION OF THE INVENTION<br><br>
In a first aspect the present invention encompasses compositions comprising at least two binding molecules. In an embodiment the binding molecules are immunoglobulins. Fragments of immunoglobulins still having the desired functionality and/or activity of the complete immunoglobulin are also considered immunoglobulins according to the invention. Preferably, the at least two binding molecules, i.e. immunoglobulins, are capable of specifically binding to a coronavirus. Coronaviruses include, but not limited to, avian infectious bronchitis virus, avian infectious laryngotracheitis virus, enteric coronavirus, equine coronavirus, coronavirus Group 1 species such as for instance human coronavirus 229E or human coronavirus NL63, coronavirus Group 2 species such as human coronavirus OC43 or chicken enteric coronavirus, coronavirus Group 3 species, human enteric coronavirus 4408, and SARS-CoV. The compositions can be administered to a mammal to treat, prevent or ameliorate one or more symptoms associated with a coronavirus infection.<br>
In an embodiment the invention relates to synergistic compositions, i.e. compositions exhibiting synergistic coronavirus neutralizing activity. In.other words, the compositions comprise at least two binding molecules, i.e. immunoglobulins, that are capable of specifically binding to a coronavirus and that have coronavirus neutralizing activity, characterized in that the binding molecules act synergistically in neutralizing coronavirus. As used herein, the term "synergistic" means that the combined effect of the binding molecules when used in combination is greater than their additive effects when used individually. In other words, the neutralizing activity of the composition is greater than the sum of the neutralizing activity of each immunoglobulin alone. In an embodiment none of the binding molecules, i.e.<br><br>
immunoglobulins, present in the synergistic coronavirus neutralizing activity exhibiting compositions may have coronavirus neutralizing activity when used as an individual binding molecule. Alternatively, one binding molecule of the at least two binding molecules in the compositions exhibiting synergistic coronavirus neutralizing activity may have coronavirus neutralizing activity when used individually. In a preferred embodiment both of the at least two binding molecules, i.e. immunoglobulins, have coronavirus neutralizing activity when used individually. In an embodiment one of the at least two binding molecules in the synergistic coronavirus neutralizing activity exhibiting compositions may bind to a coronavirus and the other binding molecule may bind to a cell associated receptor of the coronavirus. Alternatively, both binding molecules may bind to either the coronavirus or cell associated receptor.<br>
In a preferred embodiment of the invention the coronavirus is a SARS-CoV including animal or human SARS-CoV. Preferably, the SARS-CoV is a human SARS-CoV. In another aspect the invention thus provides compositions comprising at least two binding molecules, i.e. immunoglobulins, capable of specifically binding to SARS-CoV and preferably having SARS-CoV neutralizing activity. The compositions preferably exhibit synergistic SARS-CoV neutralizing activity. In other words, the compositions comprise at least two binding molecules, i.e. immunoglobulins, having SARS-CoV neutralizing activity, characterized in that the binding molecules act synergistically in neutralizing SARS-CoV. The SARS-CoV neutralizing activity of the composition is greater than the sum of the neutralizing activity of each immunoglobulin alone. In a preferred embodiment of the invention, the binding molecules in the compositions act synergistically in<br><br>
neutralizing a plurality of SARS-CoV strains (see Table 1 for a list of several known SARS-CoV genome sequences and S protein genes). In another embodiment each of the immunoglobulins in the composition is capable of neutralizing a plurality of (different) SARS-CoV strains, preferably human SARS-CoV strains. In another embodiment at least one of the binding molecules, i.e. immunoglobulins, of the compositions of the invention is capable of neutralizing an animal SARS-CoV. The binding molecules in the compositions of the invention may neutralize coronavirus infectivity, such as SARS-CoV infectivity, by several modes of action including, but not limited to, preventing the attachment of the coronavirus to possible receptors on host cells, inhibition of the release of RNA into the cytoplasm of the cell, prevention of RNA transcription or translation, or inhibition or downregulation of coronavirus replication. Furthermore, the binding molecules may act by fixing complement or be capable of assisting in the lysis of enveloped coronavirus. They might also act as opsonins and augment phagocytosis of coronavirus either by promoting its uptake via Fc or C3b receptors or by agglutinating the coronavirus to make it more easily phagocytosed. The binding molecules in the compositions may have similar modes of action or may have different modes of action. In a specific embodiment, the compositions neutralize coronavirus, such as SARS-CoV, infectivity by at least 10%, at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, at least 90%, at least 95%, or at least 99% relative to infection of host cells by the coronavirus in the absence of said compositions. Assays for measuring virus neutralizing activity are known to the skilled person. Examples of such assays are described below.<br><br>
The binding molecules, i.e. immunoglobulins, in the compositions of the invention may be capable of specifically binding to a coronavirus, such as SARS-CoV, in activated or inactivated/attenuated form. Methods for<br>
inactivating/attenuating viruses are well known in the art and include, but are not limited to, heat inactivation, inactivation by UV irradiation, inactivation by gamma irradiation. The binding molecules may also be capable of specifically binding to one or more fragments of the coronavirus including inter alia  a preparation of one or more proteins and/or (poly)peptides derived from the coronavirus. Preferably, the fragment at least comprises an antigenic determinant recognised by the binding molecules of the invention. An "antigenic determinant" as used herein is a moiety, such as a coronavirus (such as SARS-CoV), (poly)peptide, protein, glycoprotein, analog or fragment thereof, that is capable of binding to a binding molecule of the invention with sufficiently high affinity to form a detectable antigen-binding molecule complex.<br>
in an embodiment the binding molecules, i.e. immunoglobulins, are capable of specifically binding to surface accessible proteins of a coronavirus which include, but are not limited to, inner and outer membrane proteins, proteins adhering to the cell wall, and potential secreted proteins. Relevant proteins of SARS-CoV in that respect are inter alia   the spike (S) protein, the membrane (matrix) protein, the (small) envelope protein, Orf 3, Orf 4, Orf 7, Orf 8, Orf 9, Orf 10 and Orf 14. The amino acid sequence of proteins and potential proteins of various known strains of coronaviruses, such as SARS-CoV, can be found in the EMBL-database and/or other databases. For instance the complete genome of the SARS coronavirus Urbani can be found in the<br><br>
EMBL-database under accession number AY278741, the complete genome of the SARS coronavirus HSR 1 can be found under accession number AY323977, the complete genome of the SARS coronavirus Frankfurt 1 can be found under accession number AY291315 and the complete genome of the SARS coronavirus TOR2 can be found under accession number AY274119.<br>
In an embodiment at least one of the binding molecules, i.e. immunoglobulins, in the compositions of the invention is capable of specifically binding to the S protein of SARS-CoV. The other binding molecule may bind to a receptor of SARS-CoV present on or associated with target cells. An example of such a receptor is the ACE-2 receptor (see Li et al.,   2003). In another embodiment all binding molecules in the compositions of the invention are capable of specifically binding to the S protein of SARS-CoV.<br>
In yet another embodiment at least one of the binding molecules in the compositions of the invention is capable of specifically binding to the extracellular domain of the S protein of SARS-CoV. This domain consists of amino acids 15-1195 of the S protein. In a specific embodiment at least one binding molecule in the compositions of the invention is capable of specifically binding to amino acids 318-510 of the S protein of SARS-CoV. The neutralizing binding molecules, i.e. immunoglogulins, in the compositions of the invention may react with overlapping, competing epitopes, but preferably they react with different/distinct, non-competing epitopes of the coronavirus, such as SARS-CoV.<br>
Another aspect of the invention are compositions comprising at least two binding molecules capable of specifically binding to a coronavirus, such as SARS-CoV, wherein the binding molecules are capable of reacting with different, non-competing epitopes of the coronavirus.<br><br>
Preferably, the coronavirus is a human coronavirus, more preferably the coronavirus is SARS-CoV. Compositions comprising at least two binding molecules wherein each binding molecule binds to a different epitope or site on a virus are more suitable for preventing the escape of resistant variants of the virus compared to compositions comprising at least two binding molecules wherein each binding molecule binds to an overlapping epitope or site on the virus.<br>
In a specific embodiment the different, non-competing epitopes recognised by the binding molecules, i.e. immunoglobulins, in the compositions of the invention are located on the S protein of SARS-CoV, particularly the extracellular domain of the S protein, more particularly within amino acids 318-510 of the S protein. In another aspect, at least one of the binding molecues, i.e. immunoglobulins, of the compositions of the invention is capable of reacting with amino acids 318-510 of the S protein of a human and an animal SARS-CoV. In another embodiment at least one of the binding molecues, i.e. immunoglobulins, of the compositions of the invention reacts with an epitope comprising the amino acid sequence of SEQ ID NO:128. The epitope may consist of 11, 11 to 15, 11 to 20, 11 to 25, 11 to 30, 11 to 35, 11 to 40, 11 to 45 or even more amino acids. In another aspect at least one of the binding molecules, i.e. immunoglobulins, in the compositions of the invention is capable of reacting with amino acids 318-510 of the S protein of a SARS-CoV, wherein the amino acid at position 479 is an amino acid other than asparagine, to a similar extent as with amino acids 318-510 of the S protein of a SARS-CoV, wherein the amino acid at position 479 is an asparagine. In other words, substitution of the amino acid at position 479 does not dramatically influence the binding of at least one of the<br><br>
immunoglobulins in the compositions of the invention to amino acids 318-510 of the S protein of SARS-CoV. "To a similar extent" as defined above means that the binding molecule binds to amino acids 318-510 of the S protein of a SARS-CoV, wherein the amino acid at position 47 9 is an amino acid other than asparagines, in an amount of at least 60%, at least 70%, at least 75%, at least 80%, at least 85%, at least 90%, at least 95%, preferably at least 100%, compared to the binding of the binding molecule to amino acids 318-510 of the S protein of a SARS-CoV, wherein the amino acid at position 47 9 is an asparagine. Binding can be measured by methods well known to a person skilled in the art such as for instance ELISA.<br>
The binding molecules in the compositions according to the invention are preferably human binding molecules, i.e. immunoglobulins. They can be intact immunoglobulin molecules such as polyclonal or monoclonal antibodies, in particular human monoclonal antibodies, or the binding molecules can be antigen-binding fragments including, but not limited to, Fab, F (ab?) , F (abf ) 2, Fv, dAb, Fd, complementarity determining region (CDR) fragments, single-chain antibodies (scFv), bivalent single-chain antibodies, single-chain phage antibodies, diabodies, triabodies, tetrabodies, and (poly)peptides that contain at least a fragment of an immunoglobulin that is sufficient to confer specific antigen binding to the SARS-CoV or fragment thereof- The human binding molecules are preferably human monoclonal antibodies. The binding molecules in the compositions can be in non-isolated or isolated form. The compositions may further comprise at least one other therapeutic agent. Preferably, the therapeutic agent is useful in the prophylaxis and/or treatment of a condition resulting from a coronavirus such as for instance SARS-CoV.<br><br>
Typically, binding molecules can bind to their binding partners, i.e. a coronavirus or fragments thereof, with an affinity constant (Kd-value) that is lower than 0.2,10~4 M, 1.0,10~5 M, 1.0,10"6 M, 1.0,10~7 M, preferably lower than 1.0,10" 8 M, more preferably lower than 1.0,10~9 M, more preferably lower than 1.0,10~10 M, even more preferably lower than 1.0,10" 11 M, and in particular lower than 1.0,10~12 M. The affinity constants can vary for antibody isotypes. For example, affinity binding for an IgM isotype refers to a binding affinity of at least about 1.0,10~7 M. Affinity constants can for instance be measured using surface plasmon resonance, i.e. an optical phenomenon that allows for the analysis of realtime biospecific interactions by detection of alterations in protein concentrations within a biosensor matrix, for example using the BIACORE system (Pharmacia Biosensor AB, Uppsala, Sweden).<br>
The binding molecules may bind to a coronavirus in soluble form such as for instance in a sample or may bind to a coronavirus bound or attached to a carrier or substrate, e.g., microtiter plates, membranes and beads, etc. Carriers or substrates may be made of glass, plastic {e.g.,   polystyrene), polysaccharides, nylon, nitrocellulose, or teflon, etc. The surface of such supports may be solid or porous and of any convenient shape. Furthermore, the binding molecules may bind to a coronavirus in purified/isolated or non-purified/non-isolated form.<br>
In a preferred embodiment, the binding molecules, i.e. immunoglobulins, of the compositions according to the invention comprise at least a CDR3 region, preferably a heavy chain CDR3 region, comprising the amino acid sequence of SEQ ID NO:l or. SEQ ID NO:2. In another embodiment the binding molecules comprising the heavy chain CDR3 region of SEQ ID<br><br>
N0:1 further comprise a light chain CDR3 region comprising the amino acid sequence of SEQ ID NO:129. In another embodiment, the binding molecules comprising the heavy chain CDR3 region of SEQ ID N0:1 further comprise a heavy chain CDR1, heavy chain CDR2, light chain CDR1 and light chain CDR2 region comprising the amino acid sequence of SEQ ID NO:130, SEQ ID NO:131, SEQ ID NO:132, and SEQ ID NO:133, respectively. In another embodiment the binding molecules comprising the heavy chain CDR3 region of SEQ ID NO:2 further comprise a light chain CDR3 region comprising the amino acid sequence of SEQ ID NO:134. In another embodiment, the binding molecules comprising the heavy chain CDR3 region of SEQ ID NO:2 further comprise a heavy chain CBR1, heavy chain CDR2, light chain CDR1 and light chain CDR2 region comprising the amino acid sequence of SEQ ID NO:135, SEQ ID NO:136, SEQ ID NO:137, and SEQ ID NO:138, respectively. In yet another embodiment, the binding molecules according to the invention comprise a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:4 or SEQ ID NO:6. In an embodiment the binding molecules, i.e. immunoglobulins, in the compositions of the invention comprise at least one CDR region of a binding molecule comprising a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:4 or SEQ ID NO:6. In another embodiment they comprise two, three, four, five or even all six CDR regions.<br>
In a further embodiment, the binding molecules according to the invention comprise a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:4 and a light chain variable region comprising the amino acid sequence of SEQ ID NO:8, or a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:6 and a light chain variable region comprising the amino acid sequence of SEQ ID NO:10.<br><br>
In a preferred embodiment the binding molecules having coronavirus, such as SARS-CoV, neutralizing activity are administered in IgGl or IgA format.<br>
In another aspect of the invention the compositions may comprise at least one functional variant of a binding molecule as defined herein. The compositions may also consist of only functional variants of binding molecules as herein described. Molecules are considered to be functional variants of a binding molecule, if the variants are capable of competing for specifically binding to a coronavirus, such as SARS-CoV, or a fragment thereof with the parent binding molecules. In other words, when the functional variants are still capable of binding to the coronavirus, such as SARS-CoV, or a fragment thereof. Preferably, the functional variants are capable of neutralizing coronavirus, such as SARS-CoV, infectivity and should together with the other binding molecule (or other functional variant) or other binding molecules (or other functional variants) form a composition exhibiting synergistic coronavirus, such as SARS-CoV, neutralizing activity. The neutralizing activity of a functional variant may either be higher or be lower compared to the parent binding molecules.<br>
Functional variants include, but are not limited to, derivatives that are substantially similar in primary structural sequence, but which contain e.g. in  vitro  or in vivo  modifications, chemical and/or biochemical, that are not found in the parent binding molecule. Such modifications are well known to the skilled artisan and include inter alia acetylation, acylation, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, glycosylation, methylation, pegylation, proteolytic processing, phosphorylation, and the like.<br><br>
Alternatively, functional variants can be binding molecules as defined in the present invention comprising an amino acid sequence containing substitutions, insertions, deletions or combinations thereof of one or more amino acids compared to the amino acid sequences of the parent binding molecules. Furthermore, functional variants can comprise truncations of the amino acid sequence at either or both the amino or carboxy termini. Functional variants may have the same or different, either higher or lower, binding affinities compared to the parent binding molecule but are still capable of binding to a coronavirus, such as SARS-CoV, or a fragment' thereof and preferably still capable of neutralizing coronavirus, such as SARS-CoV, infectivity. For instance, functional variants may have increased or decreased binding affinities for a coronavirus, such as SARS-CoV, or a fragment thereof compared to the parent binding molecules. Preferably, the amino acid sequences of the variable regions, including, but not limited to, framework regions, hypervariable regions, in particular the CDR3 regions, are modified. Generally, the light chain and the heavy chain variable regions comprise three hypervariable regions, comprising three CDRs, and more conserved regions, the so-called framework regions (FRs). The hypervariable regions comprise amino acid residues from CDRs and amino acid residues from hypervariable loops. Functional variants intended to fall within the scope of the present invention have at least about 50% to about 99%, preferably at least about 60% to about 99%, more preferably at least about 70% to about 99%, even more preferably at least about 80% to about 99%, most preferably at least about 90% to about 99%, in particular at least about 95% to about 99%, and in particluar at least about 97% to about 99% amino acid sequence homology with the parent binding molecules as defined herein. Computer<br><br>
algorithms such as inter alia  Gap or Bestfit known to a person skilled in the art can be used to optimally align amino acid sequences to be compared and to define similar or identical amino acid residues. Functional variants can be obtained by altering the parent binding molecules or parts thereof by general molecular biology methods known in the art including, but not limited to, error-prone PCR, oligonucleotide-directed mutagenesis and site-directed mutagenesis.<br>
In yet a further aspect, the invention includes compositions comprising at least one immunoconjugate, i.e. a molecule comprising at least one binding molecule or functional variant thereof as defined herein and further comprising at least one tag. Also contemplated in the present invention are compositions consisting of immunoconjugates. The compositions may further comprise another molecule, such as a therapeutic agent or immunoconjugate having a different specificity. In a further embodiment, the immunoconjugates of the invention may comprise more than one tag. These tags can be the same or distinct from each other and can be joined/conjugated non-covalently to the binding molecules. The tag(s) can also be joined/conjugated directly to the binding molecules through covalent bonding. Alternatively, the tag (s) can be joined/conjugated to the binding molecules by means of one or more linking compounds. Techniques for conjugating tags to binding molecules are well known to the skilled artisan.<br>
The tags of the immunoconjugates of the present invention may be therapeutic agents, but preferably they are detectable moieties/agents. Compositions comprising immunoconjugates comprising a detectable agent can be used diagnostically to, for example, assess if a subject has been infected with a coronavirus, such as SARS-CoV, or monitor the development or progression of a coronavirus, such as SARS-CoV, infection as<br><br>
part of a clinical testing procedure to, e.g.,   determine the efficacy of a given treatment regimen. However, they may also be used for other detection and/or analytical and/or diagnostic purposes. Detectable moieties/agents include, but are not limited to, enzymes, prosthetic groups, fluorescent materials, luminescent materials, bioluminescent materials, radioactive materials, positron emitting metals, and nonradioactive paramagnetic metal ions. The tags used to label the binding molecules for detection and/or analytical and/or diagnostic purposes depend on the specific<br>
detection/analysis/diagnosis techniques and/or methods used such as inter alia   immunohistochemical staining of (tissue) samples, flow cytometric detection, scanning laser cytometric detection, fluorescent immunoassays, enzyme-linked immunosorbent assays (ELISA's), radioimmunoassays (RIA's), bioassays (e.g., neutralisation assays), Western blotting applications, etc. For immunohistochemical staining of tissue samples preferred labels are enzymes that catalyze production and local deposition of a detectable product. Furthermore, the compositions of the invention can also be attached to solid supports, which are particularly useful for in  vitro immunoassays or purification of a coronavirus or a fragment thereof. Such solid supports might be porous or nonporous, planar or nonplanar. The binding molecules of the present invention or functional fragments thereof can be fused to marker sequences, such as a peptide to facilitate purification. Alternatively, an antibody can be conjugated to a second antibody to form an antibody heteroconjugate. In another aspect the binding molecules of the invention may be conjugated/attached to one or more antigens. Preferably, these antigens are antigens which are recognised by the immune<br><br>
system of a subject to which the binding molecule-antigen conjugate is administered.<br>
Next to producing immunoconjugates chemically by conjugating, directly or indirectly via for instance a linker, the immunoconjugates can be produced as fusion proteins comprising the binding molecules of the invention and a suitable tag. Fusion proteins can be produced by methods known in the art such as, e.g.,   recombinantly by constructing nucleic acid molecules comprising nucleotide sequences encoding the binding molecules in frame with nucleotide sequences encoding the suitable tag(s) and then expressing the nucleic acid molecules.<br>
It is another aspect of the present invention to provide a nucleic acid molecule encoding at least a binding molecule or functional fragment thereof present in the compositions according to the invention. Such nucleic acid molecules can be used as intermediates for cloning purposes, e.g. in the process of affinity maturation described above. In a preferred embodiment, the nucleic acid molecules are isolated or purified.<br>
The skilled man will appreciate that functional variants of these nucleic acid molecules are also intended to be a part of the present invention. Functional variants are nucleic acid sequences that can be directly translated, using the standard genetic code, to provide an amino acid sequence identical to that translated from the parent nucleic acid molecules.<br>
Preferably, the nucleic acid molecules encode binding molecules comprising a CDR3 region, preferably a heavy chain CDR3 region, comprising the amino acid sequence of SEQ ID NO:1 or SEQ ID NO:2. In another embodiment the nucleic acid molecules of the invention encode binding molecules comprising a light chain CDR3 region comprising the amino acid sequence<br><br>
of SEQ ID NO: 129 or SEQ ID NO: 134. In yet another embodiment the nucleic acid molecules of the invention encode binding molecules comprising a heavy chain CDR1 region comprising the amino acid sequence of SEQ ID NO:130 or SEQ ID NO:135; a heavy chain CDR2 region comprising the amino acid sequence of SEQ ID NO:131 or SEQ ID NO:136; a light chain CDR1 region comprising the amino acid sequence of SEQ ID NO:132 or SEQ ID NO:137; and/or a light chain CDR2 region comprising the amino acid sequence of SEQ ID NO:133 or SEQ ID NO:138.<br>
Even more preferably, the nucleic acid molecules encode binding molecules comprising a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:4 or SEQ ID NO: 6.<br>
In yet another embodiment, the nucleic acid molecules encode binding molecules comprising a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:4 and a light chain variable region comprising the amino acid sequence of SEQ ID NO:8 or they encode a heavy chain variable region comprising the amino acid sequence of SEQ ID NO:6 and a light chain variable region comprising the amino acid sequence of SEQ ID NO:10.<br>
In a specific embodiment of the invention the nucleic acid molecules encoding the heavy chain variable region of the binding molecules of the invention comprise the nucleotide sequence of SEQ ID NO:3 or SEQ ID NO:5.<br>
In yet another specific embodiment of the present invention, the nucleic acid molecules encoding the light chain variable region of the binding molecules of the invention comprise the nucleotide sequence of SEQ ID NO:7 or SEQ ID NO:9. The nucleic acid molecules of the invention may even contain the nucleotide sequences or parts thereof of the at<br><br>
least two binding molecules present in the compostions of the invention.<br>
It is another aspect of the invention to provide vectors, i.e. nucleic acid constructs, comprising one or more nucleic acid molecules according to the present invention. Vectors can be derived from plasmids such as inter alia   F, Rl, RP1, Col, pBR322, TOL, Ti, etc; cosmids; phages such as lambda, lambdoid, M13, Mu, PI, P22, Qp, T-even, T-odd, T2, T4, T7, etc; plant viruses; or animal viruses. Vectors can be used for cloning and/or for expression of the binding molecules of the invention and might even be used for gene therapy purposes. Vectors comprising one or more nucleic acid molecules according to the invention operably linked to one or more expression-regulating nucleic acid molecules are also covered by the present invention. The binding molecules present in the compositions of the invention may be expressed on separate vectors but may also be expressed on the same vector. The choice of the vector(s) is dependent on the recombinant procedures followed and the host used. Introduction of vectors in host cells can be effected by inter alia  calcium phosphate transfection, virus infection, DEAE-dextran mediated transfection, lipofectamin transfection or electroporation. Vectors may be autonomously replicating or may replicate together with the chromosome into which they have been integrated. Preferably, the vectors contain one or more selection markers. The choice of the markers may depend on the host cells of choice, although this is not critical to the invention as is well known to persons skilled in the art. They include, but are not limited to, kanamycin, neomycin, puromycin, hygromycin, zeocin, thymidine kinase gene from Herpes simplex virus (HSV-TK), dihydrofolate reductase gene from mouse (dhfr). Vectors comprising one or more nucleic acid<br><br>
molecules encoding the binding molecules as described above operably linked to one or more nucleic acid molecules encoding proteins or peptides that can be used to isolate the binding molecules are also covered by the invention. These proteins or peptides include, but are not limited to, glutathione-S-transferase, maltose binding protein, metal-binding polyhistidine, green fluorescent protein, luciferase and beta-galactosidase.<br>
Hosts containing one or more copies of the vectors mentioned above are an additional subject of the present invention. Preferably, the hosts are host cells. Host cells include, but are not limited to, cells of mammalian, plant, insect, fungal or bacterial origin. Bacterial cells include, but are not limited to, cells from Gram positive bacteria such as several species of the genera Bacillus,   Streptomyces  and Staphylococcus  or cells of Gram negative bacteria such as several species of the genera Escherichia, such as E.   coli, and Pseudomonas-   In the group of fungal cells preferably yeast cells are used. Expression in yeast can be achieved by using yeast strains such as Inter alia   Pichia pastoris, Saccharomyces   cerevisiae  and Hansenula polymorpha. Furthermore, insect cells such as cells from Drosophila and Sf9 can be used as host cells. Besides that, the host cells can be plant cells expression systems using mammalian cells such as Chinese Hamster Ovary (CHO) cells, COS cells, BHK cells or Bowes melanoma cells are preferred in the present invention. Mammalian cells provide expressed proteins with posttranslational modifications that are most similar to natural molecules of mammalian origin. Since the present invention deals with molecules that may have to be administered to humans, a completely human expression system would be particularly preferred. Therefore, even more<br><br>
preferably, the host cells are human cells. Examples of human cells are inter alia  HeLa, 911, AT1080, A549, 293 and HEK293T cells. Preferred mammalian cells are human retina cells such as 911 cells or the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6® (PER.C6 is a registered trademark of Crucell Holland B.V.). For the purposes of this application "PER.C6" refers to cells deposited under number 96022940 or ancestors, passages upstream or downstream as well as descendants from ancestors of deposited cells, as well as derivatives of any of the foregoing. In preferred embodiments, the human producer cells comprise at least a functional part of a nucleic acid sequence encoding an adenovirus El region in expressible format. In even more preferred embodiments, said host cells are derived from a human retina and immortalised with nucleic acids comprising adenoviral El sequences, such as the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6®, and derivatives thereof. Production of recombinant proteins in host cells can be performed according to methods well known in the art. The use of the cells marketed under the trademark PER.C6® as a production platform for proteins of interest has been described in WO 00/63403 the disclosure of which is incorporated herein by reference in its entirety.<br>
Methods of producing binding molecules or functional variants are well known to the skilled artisan. One method comprises the steps of a) culturing a host as defined above under conditions conducive to the expression of the binding<br><br>
molecules or functional variants, and b) optionally, recovering the expressed binding molecules or functional variants. The expressed binding molecules or functional variants thereof can be recovered from the cell free extract, but preferably they are recovered from the culture medium-Methods to recover proteins, such as binding molecules, from cell free extracts or culture medium are well known to the man skilled in the art. Binding molecules or functional variants thereof as obtainable by the above described method are also a part of the present invention. Alternatively, next to the expression in hosts, such as host cells, the binding molecules or functional variants thereof can be produced synthetically by conventional peptide synthesizers or in cell-free translation systems using RNA nucleic acid derived from DNA molecules according to the invention. Binding molecule or functional variants thereof as obtainable by the above described synthetic production methods or cell-free translation systems are also a part of the present invention. In yet another alternative embodiment, binding molecules according to the present invention, preferably human binding molecules specifically binding to a coronavirus, such as SARS-CoV, or a fragment thereof, may be generated by transgenic non-human mammals, such as for instance transgenic mice or rabbits, that express human immunoglobulin genes. Preferably, the transgenic non-human mammals have a genome comprising a human heavy chain transgene and a human light chain transgene encoding all or a portion of the human binding molecules as described above. The transgenic non-human mammals can be immunized with a purified or enriched preparation of a coronavirus, such as SARS-CoV, or a fragment thereof. Protocols for immunizing non-human mammals are well established in the art. See Using Antibodies: A Laboratory<br><br>
Manual, Edited by: E. Harlow, D. Lane (1998), Cold Spring Harbor Laboratory, Cold Spring Harbor, New York and Current Protocols in Immunology, Edited by: J.E. Coligan, A.M. Kruisbeek, D.H. Margulies, E.M. Shevach, W. Strober (2001), John Wiley &amp; Sons Inc., New York, the disclosures of which are incorporated herein by reference. Immunization protocols often include multiple immunizations, either with or without adjuvants such as Freund's complete adjuvant and Freund's incomplete adjuvant, but may also include naked DNA immunizations. In another embodiment, the human binding molecules are produced by B cells or plasma cells derived from the transgenic animals or human subjects that have been exposed to SARS-CoV. In yet another embodiment, the human binding molecules are produced by hybridomas which are prepared by fusion of B cells obtained from the above described transgenic non-human mammals or human subjects to immortalized cells. B cells, plasma cells and hybridomas as obtainable from the above described transgenic non-human mammals or human subjects and human binding molecules as obtainable from the above described transgenic non-human mammals or human subjects are also a part of the present invention.<br>
Methods of identifying binding molecules, preferably human binding molecules such as human monoclonal antibodies or fragments thereof, or nucleic acid molecules encoding the binding molecules may comprise the steps of a) contacting a phage library of binding molecules, preferably human binding molecules, with a coronavirus, such as SARS-CoV, or a fragment thereof, b) selecting at least once for a phage binding to the coronavirus or the fragment thereof, and c) separating and recovering the phage binding to the coronavirus or the fragment thereof. The selection step may be performed by<br><br>
contacting a phage library with a coronavirus which is inactivated. The coronavirus may be isolated or non-isolated, e.g. present in serum and/or blood of an infected individual. Alternatively, the selection step may be performed in the presence of a fragment of a coronavirus such as an extracellular part of the coronavirus (such as SARS-CoV), one or more proteins or (poly)peptides derived from a coronavirus, fusion proteins comprising these proteins or (poly)peptides, and the like. Phage display methods for identifying and obtaining binding molecules, e.g. antibodies, are by now well-established methods known by the person skilled in the art. They are e.g. described in US Patent Number 5,696,108; Burton and Barbas, 1994; de Kruif et al.,   1995b; and Phage Display: A Laboratory Manual. Edited by: CF Barbas, DR Burton, JK Scott and GJ Silverman (2001), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York. All these references are herewith incorporated herein in their entirety. For the construction of phage display libraries, collections of human monoclonal antibody heavy and light chain variable region genes are expressed on the surface of bacteriophage, preferably filamentous bacteriophage, particles, in for example single-chain Fv (scFv) or in Fab format (see de Kruif et al., 1995b). Large libraries of antibody fragment-expressing phages typically contain more than 1.0,109 antibody specificities and may be assembled from the immunoglobulin V regions expressed in the B lymphocytes of non-immunized or immunized individuals. In a specific embodiment the phage library of binding molecules, preferably scFv phage library, is prepared from RNA isolated from cells obtained from a subject that has been vaccinated or exposed to a coronavirus, such as SARS-CoV. RNA can be isolated from inter alia  bone marrow or peripheral blood, preferably peripheral blood<br><br>
lymphocytes. The subject can be an animal vaccinated or exposed to a coronavirus, but is preferably a human subject which has been vaccinated or has been exposed to a coronavirus. Preferably, the human subject has recovered from the coronavirus.<br>
Alternatively, phage display libraries may be constructed from immunoglobulin variable regions that have been partially assembled in   vitro  to introduce additional antibody diversity in the library (semi-synthetic libraries). For example, in vitro  assembled variable regions contain stretches of synthetically produced, randomized or partially randomized DNA in those regions of the molecules that are important for antibody specificity, e. g.   CDR regions. Coronavirus specific phage antibodies can be selected from libraries by immobilising a coronavirus (in inactivated or active form) or target antigens such as antigens from a coronavirus on a solid phase and subsequently exposing the coronavirus (in inactivated or active form) or target antigens to a phage library to allow binding of phages expressing antibody fragments specific for the solid phase-bound antigen(s). Non-bound phages are removed by washing and bound phages eluted from the solid phase for infection of Escherichia   coli (E.coli)   bacteria and subsequent propagation. Multiple rounds of selection and propagation are usually required to sufficiently enrich for phages binding specifically to the coronavirus (in inactivated or active form) or target antigen(s). If desired, before exposing the phage library to a coronavirus (in inactivated or active form) or target antigens the phage library can first be subtracted by exposing the phage library to non-target antigens bound to a solid phase. Phages may also be selected for binding to complex antigens such as complex mixtures of coronavirus proteins or<br><br>
(poly)peptides or host cells expressing one or more proteins or (poly)peptides of a coronavirus. Antigen specific phage antibodies can be selected from the library by incubating a solid phase with bound thereon a preparation of an inactivated coronavirus with the phage antibody library to let for example the scFv or Fab part of the phage bind to the proteins/polypeptides of the coronavirus preparation. After incubation and several washes to remove unbound and loosely attached phages, the phages that have bound with their scFv or Fab part to the preparation are eluted and used to infect Escherichia   coli  to allow amplification of the new specificity. Generally, one or more selection rounds are required to separate the phages of interest from the large excess of non-binding phages. Alternatively, known proteins or (poly)peptides of the coronavirus can be expressed in host cells and these cells can be used for selection of phage antibodies specific for the proteins or (poly)peptides. A phage display method using these host cells can be extended and improved by subtracting non-relevant binders during screening by addition of an excess of host cells comprising no target molecules or non-target molecules that are similar, but not identical, to the target, and thereby strongly enhance the chance of finding relevant binding molecules (This process is referred to as the Mabstract™ process. Mabstract™ is a pending trademark application of Crucell Holland B.V., see also US Patent Number 6,265,150 which is incorporated herein by reference). An example of a coronavirus against which binding molecules can be found using the above described method of identification is SARS-CoV.<br>
A method of obtaining a binding molecule, preferably a human binding molecule or a nucleic acid molecule encoding such a binding molecule may comprise the steps of a)<br><br>
performing the above described method of identifying binding molecules, preferably human binding molecules such as human monoclonal antibodies or fragments thereof, or nucleic acid molecules encoding the binding molecules, and b) isolating from the recovered phage the human binding molecule and/or the nucleic acid encoding the human binding molecule. Once a new monoclonal phage antibody has been established or identified with the above mentioned method of identifying binding molecules or nucleic acid molecules encoding the binding molecules, the DNA encoding the scFv or Fab can be isolated from the bacteria or phages and combined with standard molecular biological techniques to make constructs encoding bivalent scFvs or complete human immunoglobulins of a desired specificity (e.g. IgG, IgA or IgM). These constructs can be transfected into suitable cell lines and complete human monoclonal antibodies can be produced (see Huls et  al.,   1999; Boel et  al., 2000).<br>
In addition to the at least two binding molecule, the compositions of the invention may comprise inter alia stabilising molecules, such as albumin or polyethylene glycol, or salts. Preferably, the salts used are salts that retain the desired biological activity of the binding molecules and do not impart any undesired toxicological effects. Examples of such salts include, but are not limited to, acid addition salts and base addition salts. Acid addition salts include, but are not limited to, those derived from nontoxic inorganic acids, such as hydrochloric, nitric, phosphoric, sulfuric, hydrobromic, hydroiodic, phosphorous and the like, as well as from nontoxic organic acids such as aliphatic mono- and dicarboxylic acids, phenyl-substituted alkanoic acids, hydroxy alkanoic acids, aromatic acids, aliphatic and aromatic sulfonic acids and the like. Base addition salts include, but<br><br>
are not limited to, those derived from alkaline earth metals, such as sodium, potassium, magnesium, calcium and the like, as well as from nontoxic organic amines, such as N,NT-dibenzylethylenediamine, N-methylglucamine, chloroprocaine, choline, diethanolamine, ethylenediamine, procaine and the like. If necessary, the binding molecules of the invention may be coated in or on a material to protect them from the action of acids or other natural or non-natural conditions that may inactivate the binding molecules.<br>
In yet a further aspect, the invention provides compositions comprising at least two nucleic acid molecule encoding binding molecules as defined in the present invention. The compositions may comprise aqueous solutions such as aqueous solutions containing salts (e.g., NaCl or salsts as described above), detergents (e.g., SDS) and/or other suitable components.<br>
Furthermore, the present invention pertains to pharmaceutical compositions comprising a composition according to the invention. The pharmaceutical composition of the invention further comprises at least one pharmaceutically acceptable excipient.<br>
A pharmaceutical composition according to the invention can further comprise at least one other therapeutic, prophylactic and/or diagnostic agent. Preferably, said further therapeutic and/or prophylactic agents are agents capable of preventing and/or treating an infection and/or a condition resulting from a coronavirus, such as SARS-CoV. Therapeutic and/or prophylactic agents include, but are not limited to, anti-viral agents. Such agents can be binding molecules, small molecules, organic or inorganic compounds, enzymes, polynucleotide sequences etc.<br><br>
Examples of anti-viral agents are well known to the skilled artisan. Agents that are currently used to treat patients infected with for instance SARS-CoV are interferon-alpha, steroids and potential replicase inhibitors. Furthermore, patients infected with SARS-CoV are currently treated by transfusion of serum produced from blood donated by recovering/recovered SARS patients who have produced antibodies after being exposed to the virus. Agents capable of preventing and/or treating an infection with SARS-CoV or other coronavirus and/or a condition resulting from SARS-CoV or other coronavirus that are in the experimental phase might also be used as other therapeutic and/or prophylactic agents useful in the present invention.<br>
The pharmaceutical compositions of the invention can be tested in suitable animal model systems prior to use in humans. Such animal model systems include, but are not limited to, ferrets, mice, rats, chicken, cows, monkeys, pigs, dogs, rabbits, etc.<br>
Typically, pharmaceutical compositions must be sterile and stable under the conditions of manufacture and storage. The compositions of the present invention can be in powder form for reconstitution in the appropriate pharmaceutically acceptable excipient before or at the time of delivery. In the case of sterile powders for the preparation of sterile injectable solutions, the preferred methods of preparation are vacuum drying and freeze-drying (lyophilization) that yield a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof.<br>
Alternatively, the compositions of the present invention can be in solution and the appropriate pharmaceutically acceptable excipient can be added and/or mixed before or at<br><br>
the time of delivery to provide a unit dosage injectable form. Preferably, the pharmaceutically acceptable excipient used in the present invention is suitable to high drug concentration, can maintain proper fluidity and, if necessary, can delay absorption.<br>
The choice of the optimal route of administration of the pharmaceutical compositions will be influenced by several factors including the physico-chemical properties of the active molecules within the compositions, the urgency of the clinical situation and the relationship of the plasma concentrations of the active molecules to the desired therapeutic effect. For instance, if necessary, the compositions of the invention can be prepared with carriers that will protect them against rapid release, such as a controlled release formulation, including implants, transdermal patches, and microencapsulated delivery systems. Biodegradable, biocompatible polymers can inter alia  be used, such as ethylene vinyl acetate, polyanhydrides, polyglycolic acid, collagen, polyorthoesters, and polylactic acid. Furthermore, it may be necessary to coat the compositions with, or co-administer the compositions with, a material or compound that prevents the inactivation of the binding molecules in the compositions. For example, the binding molecules of the compositions may be administered to a subject in an appropriate carrier, for example, liposomes, or a diluent.<br>
The routes of administration can be divided into two main categories, oral and parenteral administration. These two categories include several routes of administration well known to the skilled person. The preferred administration route is intravenous, particularly preferred is intramuscular.<br><br>
Oral dosage forms can be formulated in several formulations and may contain pharmaceutically acceptable excipients including, but not limited to, inert diluents, granulating and disintegrating agents, binding agents, lubricating agents, preservatives, colouring, flavouring or sweetening agents, vegetable oils, mineral oils, wetting agents, and thickening agents.<br>
The pharmaceutical compositions of the present invention can also be formulated for parenteral administration. Formulations for parenteral administration can be inter alia in the form of aqueous or non-aqueous isotonic sterile nontoxic injection or infusion solutions or suspensions. The solutions or suspensions may comprise agents that are nontoxic to recipients at the dosages and concentrations employed. Such agents are well known to the skilled artisan and include 1,3-butanediol, Ringer's solution, Hank's solution, isotonic sodium chloride solution, oils or fatty acids, local anaesthetic agents, preservatives, buffers, viscosity or solubility increasing agents, water-soluble antioxidants, oil-soluble antioxidants, and metal chelating agents.<br>
In a further aspect, the pharmaceutical compositions of the invention can be used as a medicament. So, a method of treatment and/or prevention of a coronavirus infection using the pharmaceutical compositions of the invention is another part of the present invention. The (pharmaceutical) compositions of the invention can inter alia  be used in the diagnosis, prophylaxis, treatment, or combination thereof, of one or more conditions resulting from a coronavirus. They are suitable for treatment of yet untreated patients suffering from a condition resulting from a coronavirus and patients who have been or are treated from a condition resulting from a<br><br>
coronavirus. They protect against further infection by a coronavirus and/or will retard the onset or progress of the symptoms associated with a coronavirus. They may even be used in the prophylaxis of conditions resulting from a coronavirus in for instance people exposed to the coronavirus such as hospital personnel taking care of suspected patients. Preferably, the (pharmaceutical) compositions can be used in a method to detect, prevent, and/or treat a human coronavirus, such as SARS-CoV, infection.<br>
The above mentioned compositions and pharmaceutical compositions may be employed in conjunction with other molecules useful in diagnosis, prophylaxis and/or treatment of a coronavirus infection. They can be used in  vitro,   ex  vivo  or in  vivo-   For instance, the pharmaceutical compositions of the invention can be co-administered with a vaccine against a coronavirus, such as SARS-CoV. Alternatively, the vaccine may also be administered before or after administration of the pharmaceutical compositions of the invention. Administration of the pharmaceutical compositions of the invention with a vaccine might be suitable for postexposure prophylaxis and might also decrease possible side effects of a live-attenuated vaccine in immunocompromised recipients.<br>
The binding molecules are typically formulated in the compositions and pharmaceutical compositions of the invention in a therapeutically or diagnostically effective amount. Dosage regimens can be adjusted to provide the optimum desired response (e.g./ a therapeutic response). A suitable dosage range may for instance be 0.05-100 mg/kg body weight, preferably 0.1-15 mg/kg body weight.<br>
Typically, the molar ratio of the two binding molecules in the compositions and pharmaceutical compositions of the invention may vary from 1:100 to 100:1, preferably from 1:50<br><br>
to 50:1, more preferably from 1:25 to 25:1, particularly 1:10 to 10:1, and more particularly 1:5 to 5:1. Furthermore, for example, a single bolus may be administered, several divided doses may be administered over time or the dose may be proportionally reduced or increased as indicated by the exigencies of the therapeutic situation. The molecules and compositions according to the present invention are preferably sterile. Methods to render these molecules and compositions sterile are well known in the art. The other molecules useful in diagnosis, prophylaxis and/or treatment can be administered in a similar dosage regimen as proposed for the binding molecules of the invention. If the other molecules are administered separately, they may be adminstered to a patient prior to (e.g., 2 minutes, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 4 5 minutes, 60 minutes, 2 hours, 4 hours, 6 hours, 8 hours, 10 hours, 12 hours, 14 hours, 16 hours, 18 hours, 20 hours, 22 hours, 24 hours, 2 days, 3 days, 4 days, 5 days, 7 days, 2 weeks, 4 weeks or 6 weeks before), concomitantly with, or subsequent to (e.g., 2 minutes, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 45 minutes, 60 minutes, 2 hours, 4 hours, 6 hours, 8 hours, 10 hours, 12 hours, 14 hours, 16 hours, 18 hours, 20 hours, 22 hours, 24 hours, 2 days, 3 days, 4 days, 5 days, 7 days, 2 weeks, 4 weeks or 6 weeks after) the administration of one or more of the binding molecules or pharmaceutical compositions of the invention. The exact dosing regimen is usually sorted out during clinical trials in human patients.<br>
Human binding molecules and pharmaceutical compositions comprising the human binding molecules are particularly useful, and often preferred, when to be administered to human beings as in   vivo  therapeutic agents, since recipient immune response to the administered antibody will often be<br><br>
substantially less than that occasioned by administration of a monoclonal murine, chimeric or humanised binding molecule. In another aspect, the invention concerns the use of<br>
(pharmaceutical) compositions according to the invention in the preparation of a medicament for the diagnosis, prophylaxis, treatment, or combination thereof, of a condition resulting from a coronavirus. Preferably, the coronavirus is a human coronavirus such as SARS-CoV.<br>
Next to that, kits comprising at least one composition according to the invention or at least one pharmaceutical composition according to the invention are also a part of the present invention. Optionally, the above described components of the kits of the invention are packed in suitable containers and labeled for diagnosis, prophylaxis and/or treatment of the indicated conditions. The binding molecules in the<br>
(pharmaceutical) compositions may be packaged individually. The above-mentioned components may be stored in unit or multi-dose containers, for example, sealed ampules, vials, bottles, syringes, and test tubes, as an aqueous, preferably sterile, solution or as a lyophilized, preferably sterile, formulation for reconstitution. The containers may be formed from a variety of materials and may have a sterile access port (for example the container may be an intravenous solution bag or a vial having a stopper pierceable by a hypodermic injection needle). The kit may further comprise more containers comprising a pharmaceutically acceptable buffer- It may further include other materials desirable from a commercial and user standpoint, including other buffers, diluents, filters, needles, syringes, culture medium for one or more of the suitable hosts. Associated with the kits can be instructions customarily included in commercial packages of therapeutic, prophylactic or diagnostic products, that contain<br><br>
information about for example the indications, usage, dosage, manufacture, administration, contraindications and/or warnings concerning the use of such therapeutic, prophylactic or diagnostic products.<br>
The invention further pertains to a method of detecting a SARS-CoV in a sample, wherein the method comprises the steps of a) contacting a sample with a diagnostically effective amount of compositions or pharmaceutical compositions according to the invention, and b) determining whether the compositions or pharmaceutical compositions specifically bind to a molecule of the sample. The sample may be a biological sample including, but not limited to blood, serum, urine, tissue or other biological material from (potentially) infected subjects, or a nonbiological sample such as water, drink, etc. The (potentially) infected subjects may be human subjects, but also animals that are suspected as carriers of a coronavirus, such as SARS-CoV, might be tested for the presence of the coronavirus using the compositions or pharmaceutical compositions. The sample may first be manipulated to make it more suitable for the method of detection. Manipulation mean inter alia   treating the sample suspected to contain and/or containing the coronavirus in such a way that the coronavirus will disintigrate into antigenic components such as proteins, (poly)peptides or other antigenic fragments. Preferably, the compositions or pharmaceutical compositions are contacted with the sample under conditions which allow the formation of an immunological complex between the binding molecules in the compositions or pharmaceutical compositions and the coronavirus or antigenic components thereof that may be present in the sample. The formation of an immunological complex, if any, indicating the presence of the coronavirus in the sample, is then detected and measured by<br><br>
suitable means. Such methods include, inter alia,   homogeneous and heterogeneous binding immunoassays, such as radioimmunoassays (RIA), ELISA, immunofluorescence, immunohistochemistry, FACS, BIACORE and Western blot analyses.<br>
Preferred assay techniques, especially for large-scale clinical screening of patient sera and blood and blood-derived products are ELISA and Western blot techniques. ELISA tests are particularly preferred and well known to persons skilled in the art.<br>
In a further aspect, the invention provides a method of screening a binding molecule or a functional variant of a binding molecule for specific binding to a different, non-overlapping epitope of a coronavirus such as SARS-CoV as the epitope bound by a binding molecule or functional variant of the invention, wherein the method comprises the steps of a) contacting a binding molecule or a functional variant to be screened, a binding molecule or functional variant of the invention and a coronavirus or fragment thereof, b) measure if the binding molecule or functional variant to be screened is capable of competing for specifically binding to the coronavirus or fragment thereof with the binding molecule or functional variant of the invention. If the binding molecule or functional variant to be screened is not capable of competing for specifically binding to the coronavirus or fragment thereof with the binding molecule or functional variant of the invention, it most likely binds to a different, non-overlapping epitope. In a further step it may be determined if the screened binding molecules that bind to a different, non-overlapping epitope compared to the binding molecules of the invention have coronavirus neutralizing activity. In yet a further step it can be determined if the screened binding molecules that bind to a different, non-<br><br>
overlapping epitope compared to the binding molecules of the invention and have coronavirus neutralizing activity form together with the binding molecules of the invention a composition exhibiting synergistic coronavirus neutralizing activity. Assays to screen for non-competing binding molecules and measure (synergistic) neutralizing activity are well known to the skilled person.<br>
EXAMPLES To illustrate the invention, the following examples are provided. The examples are not intended to limit the scope of the invention in any way.<br>
Example  1<br>
Construction   of   a scFv phage   display   library   using peripheral<br>
blood lymphocytes   of a patient  having been   exposed  to  SARS~CoV<br>
Lymphocytes were obtained from a patient recovered from SARS-CoV (see Rickerts et al.   2003) and frozen in liquid nitrogen. The lymphocytes were quickly thawed in a 37°C water bath and transferred to wet-ice. The lymphocytes were diluted with cold DMEM culture medium to a final volume of 50 ml in a 50 ml tube and centrifuged for 5 minutes at 350xg. The obtained cell pellet was suspended in 5 ml DMEM and cell density was determined microscopically using trypan-blue exclusion to visualize dead cells. All cells (~5xl06) were spun again for 5 minutes at 350xg, decanted and suspended in residual fluid (DMEM). Total RNA was prepared from these cells using organic phase separation (TRIZOL™) and subsequent ethanol precipitation- The obtained RNA was dissolved in DEPC treated ultrapure water and the concentration was determined by OD 260 nm measurement. Thereafter, the RNA was diluted to a concentration of 100 ng/ML. Next, 1 ]iq  of RNA was converted<br><br>
into cDNA as follows: To 10 ML   total RNA, 13 ML   DEPC treated ultrapure water and 1 ML  random hexamers (500 ng/jal) were added and the obtained mixture was heated at 65°C for 5 minutes and quickly cooled on wet-ice. Then, 8 jjl 5X First-Strand buffer, 2 ML  dNTP (10 mM each), 2 \il   DTT (0.1 M) , 2 ML Rnase-inhibitor (40 U/ul) and 2 \i±   Superscript™III MMLV reverse transcriptase (200 U/JJI) were added to the mixture, incubated at room temperature for 5 minutes and incubated for 1 hour at 50°C. The reaction was terminated by heat inactivation, i.e. by incubating the mixture for 15 minutes at 75°C.<br>
The obtained cDNA products were diluted to a final volume of 200 pi with DEPC treated ultrapure water. The OD 260 run of a 50 times diluted solution (in 10 mM Tris buffer) of the dilution of the obtained cDNA products gave a value of 0.1.<br>
5 to 10 \xl   of the diluted cDNA products were used as template for PCR amplification of the immunoglobulin gamma heavy chain family and kappa or lambda light chain sequences using specific oligonucleotide primers (see Tables 2-9). PCR reaction mixtures contained, besides the diluted cDNA products, 25 pmol sense primer and 2 5 pmol anti-sense primer in a final volume of 50 jil of 20 mM Tris-HCl (pH 8.4), 50 mM KC1, 2.5 mM MgCl2, 250 ]M  dNTPs and 1.25 units Taq polymerase. In a heated-lid thermal cycler having a temperature of 96CC, the mixtures obtained were quickly melted for 2 minutes, followed by 30 amplification cycles of: 30 seconds at 96°C, 30 seconds at 60°C and 60 seconds at 72°C. In a first round amplification, each of nine sense directed primers (see Table 2; covering all families of heavy chain variable regions) was combined with an IgG specific constant region anti-sense primer called HuCIgG 5'-GTC CAC CTT GGT GTT GCT GGG CTT-3' (SEQ ID NO:87) yielding nine products of about 650 basepairs.<br><br>
These products were purified on a 2% agarose gel and isolated from the gel using Qiagen gel-extraction columns. 1/10 of each of the isolated products was used in an identical PCR reaction as described above using the same nine sense primers (covering all families of heavy chain variable regions), whereby each sense primer was combined with one of the four J-region specific anti-sense primers (see Table 3). This resulted in 36 products of approximately 350 basepairs. The products obtained were purified on a 2% agarose gel and isolated from the gel using Qiagen gel-extraction columns. In a third round, 1/10 of each of the isolated products was subjected to re-amplification with the same set of primers as in the second round with the proviso that the primers used were extended with restriction sites (see Table 4) to enable directed cloning in the phage display vector pDV-C05 (see SEQ ID NO:88). This resulted again in 36 products. These products were pooled per used (VH) sense primer into nine fractions. In the next step, 2.5 jig of pooled fraction and 100 \ig  pDV-C05 vector were digested with Ncol and Xhol and purified by gel. Thereafter, a ligation was performed overnight at 16°C as follows. To 500 ng pDV-C05 vector 70 ng pooled fraction was added in a total volume of 50 \xl  ligation mix containing 50 mM Tris-HCl (pH 7.5), 10 mM MgCl2, 10 mM DTT, 1 mM ATP, 25 pg/ml BSA and 2.5 jjl T4 DNA Ligase (400 u/^1) . This procedure was followed for each pooled fraction. The ligation mixes were purified by phenol/chloroform, followed by a chloroform extraction and ethanol precipitation, methods well known to the skilled artisan. The DNA obtained was dissolved in 50 \il ultrapure water and per ligation mix two times 2.5 ML   aliquots were electroporated into 40 \il  of TGI competent E.   coli bacteria according to the manufacturer's protocol (Stratagene) . Transformants were grown overnight at 37 °C in a<br><br>
total of 27 dishes (three dishes per pooled fraction; dimension of dish: 240 mm x 240 mm) containing 2TY agar supplemented with 50 jjg/ml ampicillin and 4.5% glucose. A (sub)library of variable heavy chain regions was obtained by scraping the transformants from the agar plates. This (sub)library was directly used for plasmid DNA preparation using a Qiagen™ kit.<br>
The light chain immunoglobulin sequences were amplified from the same cDNA preparation in a similar three round PCR procedure and identical reaction parameters as described above for the heavy chain regions with the proviso that the primers depicted in Tables 5-9 were used. The first amplification was performed using a set of seventeen light chain variable region sense primers (eleven for the lambda light chain (see Table 5) and six for the kappa light chain (see Table 6)) each combined with an anti-sense primer recognizing the C-kappa called HuCk 5'-ACACTCTCCCCTGTTGAAGCTCTT-3' (see SEQ ID NO:89) or Olambda constant region HuCX2 5'-TGAACATTCTGTAGGGGCCACTG-3' (see SEQ ID NO:90) or HuCA7 5'-AGAGCATTCTGCAGGGGCCACTG-3' (see SEQ ID NO:91) (the HuCA2 and HuCA7 anti-sense primers were mixed to equimolarity before use), yielding 17 products of about 600 basepairs. These products were purified on a 2% agarose gel and isolated from the gel using Qiagen gel-extraction columns. 1/10 of each of the isolated products was used in an identical PCR reaction as described above using the same seventeen sense primers, whereby each lambda light chain sense primer was combined with one of the three Jlambda-region specific anti-sense primers (see Table 7) and each kappa light chain sense primer was combined with one of the five Jkappa-region specific anti-sense primers (see Table 8). This resulted in 63 products of approximately 350 basepairs. The products obtained were purified on a 2% agarose gel and isolated from the gel<br><br>
using Qiagen gel-extraction columns. In a third round, 1/10 of each of the isolated products was subjected to re-amplification with the same set of primers as in the second round with the proviso that the primers used were extended with restriction sites (see Table 9) to enable directed cloning in the heavy chain (sub)library vector. This resulted again in 63 products. These products were pooled to a total of 10 fractions. This number of fractions was chosen to maintain the natural distribution of the different light chain families within the library and to over or under represent certain families. The number of alleles within a family was used to determine the percentage of representation within a library (see Table 10). Next, the fractions were digested with Sail and NotI and ligated in the heavy chain (sub)library vector, which was cut with the same restriction enzymes, using the same ligation procedure and volumes as described above for the heavy chain (sub)library. Ligation purification and subsequent transformation of the resulting definitive library was also performed as described above for the heavy chain (sub)library. The transformants were grown in 30 dishes (three dishes per pooled fraction; dimension of dish: 240 mm x 240 mm) containing 2TY agar supplemented with 50 jig/ml ampicillin and 4.5% glucose. All bacteria were harvested in 2TY culture medium containing 50 |jg/ml ampicillin and 4.5% glucose, mixed with glycerol to 15% (v/v) and frozen in 1.5 ml aliquots at -80°C.<br>
Example 2<br>
Selection  of phage  carrying single-chain  Fv fragments<br>
specifically recognizing SARS-CoV<br>
Antibody fragments were selected using antibody phage display libraries and technology, essentially as described in<br><br>
US patent 6,265,150 and in WO 98/15833, both of which are incorporated herein in their entirety. All procedures were performed at room temperature unless stated otherwise. An inactivated SARS-CoV preparation (Frankfurt 1 strain) was prepared as follows. Medium from Vero cells which were infected with SARS-CoV strain Frankfurt 1 was harvested as soon as cyotopathic effect (CPE) was observed. Cell debris was removed by centrifugation of the harvested medium for 15 minutes at 3000 rpm. The obtained supernatant was collected, spun again for 15 minutes at 3000 rpm and transferred to a clean tube. Subsequently, ultracentrifuge tubes were filled with 10 ml sterile 25% glycerol in PBS. 20 ml of the cleared supernatant was gently applied on the glycerol cushion and the tubes were spun for 2 hours at 20,000 rpm at 4°C. The supernatant was discarded and the virus pellets were resuspended in 1 ml TNE buffer (10 mM Tris-HCl pH 7.4, 1 mM EDTA, 200 mM NaCl) and stored at -80°C. Next, the resuspended virus pellets were gamma-irradiated at 45kGy on dry ice. Subsequently, they were tested for the absence of infectivity in cell culture. If absence of infectivity was established, the thus obtained inactivated SARS-CoV preparation was used for selection of single-chain phage antibodies specifically binding to SARS-CoV.<br>
The inactivated virus preparation and heat-inactivated fetal bovine serum (FBS) were coated overnight at 4°C onto the surface of separate Maxisorp™ plastic tubes (Nunc). The tubes were blocked for two hours in 3 ml PBS containing 2% FBS and 2% fat free milk powder (2% PBS-FM). After two hours the FBS-coated tube was emptied and washed three times with PBS. To this tube, 500 ML   (approximately 1013 cfu) of a phage display library expressing single-chain Fv fragments (scFvs) essentially prepared as described by De Kruif et   al.    (1995a)<br><br>
and references therein (which are incorporated herein in their entirety), 500 \il   4% PBS-FM and 2 ml 2% PBS-FM were added. The tube was sealed and rotated slowly at room temperature for two hours. Subsequently, the obtained blocked phage library (3 ml) was transferred to a SARS-CoV preparation-coated tube that had been washed three times with PBS. Tween-2 0 was added to a final concentration of 0.05% and binding was allowed to proceed for two hours on a slowly rotating wheel at room temperature or at 37°C. The tube was emptied and washed ten times with PBS containing 0.05% Tween-20, followed by washing ten times with PBS. 1 ml glycine-HCL (0.05 M, pH 2.2) was added to elute bound phages, and the tube was rotated slowly for ten minutes. For neutralisation purposes, the eluted<br>
phages were added to 500 fil 1 M Tris-HCl pH 7.4. To this mixture, 5 ml of exponentially growing XL-1 blue bacterial culture was added. The obtained culture was incubated for thirty minutes at 37°C without shaking. Then, the bacteria were plated on TYE agar plates containing ampicillin, tetracycline and glucose. After overnight incubation of the plates at 37°C, the colonies were scraped from the plates and used to prepare an enriched phage library, essentially as described by De Kruif et al.    (1995a) and WO 02/103012 (both are incorporated by reference herein). Briefly, scraped bacteria were used to inoculate 2TY medium containing ampicillin, tetracycline and glucose and grown at a temperature of 37°C to an OD 600 nm of -0.3. CT or VCSM13 helper phages were added and allowed to infect the bacteria after which the medium was changed to 2TY containing ampicillin, tetracycline and kanamycin. Incubation was continued overnight at 30°C. The next day, the bacteria were removed from the 2TY medium by centrifugation after which the phages in the obtained supernatant were precipitated using<br><br>
polyethylene glycol 6000/NaCl. Finally, the phages were dissolved in a small volume of PBS containing 1% BSA, filter-sterilized and used for a next round of selection. The selection/re-infection procedure was performed two or three times. After each round of selection, individual E.coli colonies were used to prepare monoclonal phage antibodies. Essentially, individual colonies were grown to log-phase and infected with VCSM13 helper phages after which phage antibody production was allowed to proceed overnight. Phage antibody containing supernatants were tested in ELISA for binding activity to the SARS-CoV preparation which was coated to 96-well plates. In the above-described selection, the phage antibody called SC03-014 was obtained. ScFvs of the phage antibody SC03-014 were produced as described before in De Kruif et al.    (1995a and 1995b) and references therein (which are incorporated herein in their entirety). The buffer of the scFvs was adjusted to 1 x PBS.<br>
Additionally, antibody fragments were selected from the immune phage display library expressing single chain Fv fragments (scFvs) (see Example 1 for the construction of this library) essentially as described supra.   For the selection described below an UV-inactivated SARS-CoV preparation was used. UV-inactivated SARS-CoV (Frankfurt 1 strain) was prepared as follows. Medium from Vero cells which were infected with 0.1. moi (multiplicity of infection) SARS-CoV strain Frankfurt 1 was harvested as soon as cyotopathic effect (CPE) was observed. Cells were once frozen at -20°C and thawed. Cell debris was removed by centrifugation of the harvested medium for 15 minutes at 3000 rpm. The obtained supernatant was collected, spun again for 15 minutes at 3000 rpm and transferred to a clean tube. Subsequently, ultracentrifuge tubes were filled with 10 ml sterile 25% v/v<br><br>
glycerol in PBS. 20 ml of the cleared supernatant was gently applied on the glycerol cushion and the tubes were spun for 2 hours at 20,000 rpm at 4CC in a Beckman SW28 rotor. The supernatant was discarded and the virus pellets were resuspended in 1 ml TNE buffer (10 mM Tris-HCl pH 7.4, 1 mM EDTA, 200 mM NaCl) and stored at -80°C. Next, the resuspended virus pellets were UV-irradiated at 4°C for 15 minutes (UV-B radiation 280-350 nm; Amax 306 nm). Subsequently, they were tested for the absence of infectivity in cell culture. If absence of infectivity was established, the thus obtained inactivated SARS-CoV preparations were used for further experiments.<br>
In contrast to the selections described supra  no pre-subtraction using heat-inactivated fetal bovine serum coated Maxisorp™ tubes (Nunc) was performed. To the SARS-CoV coated tubes, 500 jil (approximately 1013 cfu) of the immune phage display library expressing single chain Fv fragments (scFvs), one volume of 4% PBS-FM and Tween-20 to a final concentration of 0.05% was added.<br>
For the immune phage display library selections which consisted of a single selection round only, binding was allowed to proceed at 37 °C or room temperature on a slowly rotating wheel at 37°C followed by an incubation of 30 minutes without agitation. The following selections and washes were performed: incubation at 37°C, washing 5 times with PBS containing 0.05% Tween-20 (PBST) and 5 times with PBS; incubation at 37°C, washing 10 times with PBST and 10 times with PBS; incubation at room temperature, washing 10 times with PBST and 10 times with PBS. Bound phages were eluted and processed as described above. Phages derived from individual colonies were tested in ELISA for binding activity to SARS-CoV coated to 96-well plates. In the selections from the immune<br><br>
phage display library the phage antibody called SC03-022 was obtained.<br>
Example  3<br>
Validation   of  the  SARS-CoV specific  single-chain phage<br>
antibodies<br>
Selected single-chain phage antibodies that were obtained in the screens described above, were validated in ELISA for specificity, i.e. binding to the UV-inactivated SARS-CoV preparation prepared as described supra.   Additionally, the single-chain phage antibodies were also tested for binding to 10% FBS. For this purpose, the UV-inactivated SARS-CoV preparation or 10% FBS preparation was coated to Maxisorp™ ELISA plates. After coating, the plates were blocked in 2% PBS-FM. The selected single-chain phage antibodies were incubated in an equal volume of 4% PBS-FM to obtain blocked phage antibodies. The plates were emptied, washed three times with PBS, after which the blocked phage antibodies were added. Incubation was allowed to proceed for one hour, the plates were washed in PBS containing 0.05% Tween-20 and bound phage antibodies were detected (using OD 492 nm measurement) using an anti-M13 antibody conjugated to peroxidase. As a control, the procedure was performed simultaneously using no single-chain phage antibody or control single-chain phage antibody directed against thyroglobulin (SC02-006) (see De Kruif et al. 1995a and 1995b). Both controls served as a negative control. As shown in Figure 1, the selected phage antibodies called SC03-014 and SC03-022 displayed significant binding to the immobilized UV-inactivated SARS-CoV preparation, while no binding to FBS was observed.<br>
Example   4<br><br>
Characterization   of  the  scFvs   specific  for  SARS-CoV.<br>
From the selected specific single chain phage antibody (scFv) clones plasmid DNA was obtained and nucleotide sequences were determined according to standard techniques. The nucleotide sequences of the scFvs (including restriction sites for cloning) called SC03-014 and SC03-022 are shown in SEQ ID NO:92 and SEQ ID NO:94, respectively. The amino acid sequences of the scFvs called SC03-014 and SC03-022 are shown in SEQ ID NO:93 and SEQ ID NO:95, respectively. The VH and VL gene identity (see Tomlinson IM, Williams SC, Ignatovitch 0, Corbett SJ, Winter G. V-BASE Sequence Directory. Cambridge United Kingdom: MRC Centre for Protein Engineering (1997)) and heavy chain CDR3 compositions of the scFvs specifically binding the SARS-CoV preparation are depicted in Table 11.<br>
Example  5<br>
Construction   of fully human   immunoglobulin molecules   (human monoclonal   anti-SARS-CoV antibodies)   from   the  selected anti-SARS-CoV single   chain  Fvs<br>
Heavy and light chain variable regions of the scFvs called SC03-014 and SC03-022 were PCR-amplified using oligonucleotides to append restriction sites and/or sequences for expression in the IgG expression vectors pSyn-C03-HCvl (see SEQ ID No:96) and pSyn-C05-CK (see SEQ ID No:97), respectively. The VL gene of scFv SC03-014 was amplified using oligonucleotides 5K-I acctgtctcgagttttccatggctgacatccagat gacccagtctccatcctcc (SEQ ID NO:98) and sy3K-C gctgggggcggccac ggtccgtttgatctccaccttggtccc (SEQ ID NO:99) and the PCR product cloned into vector pSyn-C05-CK. The VL gene of scFv SC03-022 was amplified using oligonucleotides 5K-J acctgtctcgagt tttccatggctgacatcgtgatgacccagtctccag (SEQ ID NO:100) and sy3K-F gctgggggcggccacggtccgcttgatctccaccttggtccc (SEQ ID NO:101)<br><br>
and the PCR product cloned into vector pSyn-C05-CK. Nucleotide sequences for all constructs were verified according to standard techniques known to the skilled artisan. VH genes of scFv SC03-014 were amplified using oligonucleotides 5H-B acctgtcttgaattctccatggccgaggtgcagctggtggagtctg (SEQ ID NO:102) and sy3H-A gcccttggtgctagcgctggagacggtcaccagggtgccctggcccc (SEQ ID NO:103). VH genes of scFv SC03-022 were amplified using oligonucleotide set 5H-H acctgtcttgaattctccatggccgaggtgcag ctggtgcagtctgg (SEQ ID NO:104) and sy3H-C gcccttggtgctagcgct ggagacggtcacggtggtgccctggcccc (SEQ ID NO:105). Thereafter, the PCR products were cloned into vector pSyn-C03-HCYl and nucleotide sequences were verified according to standard techniques known to the skilled person in the art.<br>
The resulting expression constructs pgG103-014C03 and pgG103-022C03 encoding the anti-SARS-CoV human IgGl heavy chains were transiently expressed in combination with the pSyn-C05-VkI (VL SC03-014) and pgG103-022C05 (VL SC03-022), respectively in HEK293T or PER.C6® cells and supernatants containing IgGl antibodies were obtained. The nucleotide sequences of the heavy chains of the antibodies called CR03-014 and CR03-022 are shown in SEQ ID NO:106 and SEQ ID NO:108, respectively. The amino acid sequences of the heavy chains of the antibodies CR03-014 and CR03-022 are shown in SEQ ID NO:107 and SEQ ID NO:109, respectively.<br>
The nucleotide sequences of the light chain of antibodies CR03-014 and CR03-022 is shown in SEQ ID NO:110 and SEQ ID NO:112, respectively. The amino acid sequences of the light chain of antibodies CR03-014 and CR03-022 is shown in SEQ ID NO:111 and SEQ ID NO:113. Subsequently, the recombinant human monoclonal antibodies were purified over protein-A columns and size-exclusion columns using standard purification methods<br><br>
used generally for immunoglobulins (see for instance WO 00/63403 which is incorporated by reference herein).<br>
Example   6<br>
Screening assay for SARS-CoV neutralizing activity of<br>
recombinant  human  anti-SARS-CoV antibodies<br>
The SARS-CoV neutralization assay was performed on Vero cells (ATCC CCL 81). The SARS-CoV strains used in the neutralization assay were the Frankfurt 1 strain (for the complete genome of this strain see EMBL-database accession # AY291315) (Rickerts et   al.   2003). Virus stocks of the strains were used in a titer of 4xl03 TCID50/ml (50% tissue culture infective dose per ml), with the titer calculated according to the method of Spearman and Kaerber which is known to the average skilled person. Recombinant human anti-SARS-CoV antibodies produced as described above were screened by serially 2-fold-dilution of the undiluted material (2.5 mg/ml) in PBS starting from 1:4 (dilution range 1:4 - 1:512). A neutralization titer of ^ 1:4 was regarded as specific evidence of reactivity of the antibodies against SARS-CoV in the screening assay. Convalescent serum from a SARS-patient was used as a positive control for the neutralization assay.<br>
In general, the neutralization assay worked as follows. 25 JJI of the respective antibody dilutions were mixed with 25 \il   of virus suspension (= approx. 100 TCID50/25 ML)   and incubated for one hour at 37°C. The suspension was then pipetted in triplicate into 96-we 11 plates. Next, 50 \il   of a freshly trypsinized and homogenized suspension of Vero cells (1:3 split of the confluent cell monolayer of a T75-flask), resuspended in DMEM containing 10% w/v FCS and antibiotics, were added. The inoculated cells were cultured for 3-4 days at 37°C and observed daily for the development of cytopathic<br><br>
effect (CPE). CPE was compared to the positive control (virus inoculated cells) and negative controls (mock-inoculated cells. The complete absence of CPE in an individual cell culture was defined as protection (= 100% titer reduction). The highest antibody/serum dilution giving protection in 66% percent of wells was defined as the neutralizing antibody titer. The experiment was performed three times in triplicate (see Tables 12A, B and C). The IgGs CR03-014, CR03-022, a negative control IgGl and a positive control serum from a SARS-patient were tested for SARS-CoV neutralizing activity. It is clear from Table 12A, B and C that the IgGs CR03-014 and CR03-022 displayed significant neutralizing activity. The CR03-014 IgG neutralized the Frankfurt 1 strain at titers of 128 (n=l) or 256 (n=2) in the above described assay. The CR03-022 IgG neutralized the Frankfurt 1 strain at titers of 32 (n=l) or 64 (n=2). These titers correspond to final antibody neutralization concentrations in the tissue culture well of 2.4 pg/ml (n=2) and 4.9 pg/ml (n=l) for CR03-014 and 9.8 pg/ml (n=2) and 19.5 pg/ml (n=l) for CR03-022. In the light of these concentrations both neutralizing antibodies may be suitable in the prophylaxis and/or treatment of a condition resulting from a SARS-CoV infection.<br>
Additionally, different SARS-CoV strains were used to evaluate the potency and breadth of protection of the anti-SARS-CoV antibodies. The SARS-CoV strains HKU-36, HKU-39849, HKU-66, HKU-61567, GZ43 and GZ50 were passaged on FRhK-4 cells for two or three times before testing (see Table 13). Strain HKU-61644 was passaged on Vero cells and tested after passage 1 and 15. The SARS-CoV neutralization assay was performed on FRhK-4 cells as follows. A 500 pi stock solution (100 pg/ml) of antibody was prepared in maintenance medium (MM, MEM supplemented with 1% w/v FCS). From this stock solution 2-<br><br>
fold-serially dilutions were prepared. 220 p.1 stock solution (100 jig/ml) was added in duplo in a 96-well plate from which 110 pi was taken and mixed with 110 pi MM in each of the nine subsequent wells. 110 jil of the tenth well was discarded. This resulted in ten wells containing 110 ML   0.2-100 jjg/ml antibody. 110 jil of the antibody dilution was mixed with 110 \xl   of the different SARS-CoV isolates at a concentration of 2000 TClD50/ml with the titer calculated according to the method of Reed and Muench which is known to the skilled artisan. At this stage, in a 220 \xl  volume, antibody concentrations varied from 0.1 to 50 pg/ml in the presence of 1000 TCID50/ml SARS-CoV. The 96-well plate containing the antibody/virus mixtures was preincubated for 1-2 hours at 37 °C. 100 ML   of the antibody /virus mixtures were added in quadruplicate to wells from a second 96-well tissue culture plate containing confluent FRhK-4 cells in 100 ML  MM and incubated at 37°C. During this final incubation step, 100 TCID50 of SARS-CoV was present in the presence of antibody concentrations varying from 0.05 to 25 pg/ml. The cells were cultured at 37°C and observed for the development of CPE at 72 and 96 hours. CPE was compared to a positive control (SARS-CoV inoculated cells) and a negative control (cells incubated with MM only). The antibody neutralization titer was determined as the concentration of antibody which gave 100% protection of the quadruplicate cell cultures. The monoclonal anti-SARS-CoV antibody CR03-014 completely neutralized 100 TCID50 of all tested SARS-CoV isolates at a concentration of 12.5 pg/ml (see Table 13). This indicates that antibody CR03-014 is able to neutralize a variety of SARS-CoV isolates.<br>
In an additional experiment the SARS-CoV neutralization assay was performed as described for the Frankfurt 1 strain supra  to determine synergy between SARS-CoV neutralizing<br><br>
antibodies CR03-014 and CR03-022. Stock solutions of antibody CR03-014 and CR03-022 of approximately similar potency were mixed in different ratios. To compensate for an estimated 4 times higher potency of CR03-014 compared to CR03-022, the CR03-014 antibody stock solution of 2.5 mg/ml was diluted 4-fold to 625 pg/ml. Subsequently, antibody CR03-014 and CR03-022 were mixed in the following ratios (mixture A: CR03-014 0%, CR03-022 100%; mixture B: CR03-014 10%, CR03-022 90%; mixture C: CR03-014 50%, CR03-022 50%; mixture D: CR03-014 90%, CR03-022 10%; and mixture E: CR03-014 100%, CR03-022 0%). When the antibodies in the mixtures have an additive effect, the mixtures should neutralize SARS-CoV at the same titer as the individual antibodies present in the mixtures. When the antibodies in the mixtures have a synergistic effect, the mixtures should neutralize SARS-CoV at a higher titer as the individual antibodies present in the mixtures. The neutralization assay was performed twice in triplicate wells as described above. The results of both assays were combined. Protection of at least 66% percent of the wells (4 of the 6 wells tested) was defined as the neutralizing antibody titer. The neutralization titers of the separate mixtures are shown in Table 14. From Table 14 can be deducted that the mixtures had the following titers: mixture A, 64; mixture B, 256; mixture C, &gt;1024; mixture D, 256; and mixture E, 16. From this can be concluded that, when both antibodies were tested in combination (mixtures B-D), the neutralizing titers were higher than those for antibody CR03-014 and CR03-022 individually. Together, these data indicate that antibodies CR03-014 and CR03-022 exhibit synergistic SARS-CoV neutralizing activity.<br>
In yet another embodiment the SARS-CoV neutralization assay showing synergy between the anti-SARS-CoV antibodies was<br><br>
performed on FRhK-4 cells (ATCC CRL-1688) as follows. The SARS-CoV strain called HK-39849 (GenBank accession number AY278491) was used in a titer of 2xl03 TCID50/ml as calculated according to the method of Reed and Muench known to the average skilled person. The human anti-SARS-CoV antibodies were screened by serially 1.46-fold-dilution in maintenance medium (MM) (1% w/v FCS in MEM with antibiotic) starting at a concentration of 200 |ig/ml (dilution range 200 - 6.7 pg/ml) in duplo. Four different compositions were tested: antibody CR03-014 individually, antibody CR03-022 individually, control IgGl antibody, and antibodies CR03-014 and CR03-022 in combination (start concentration 200 pg/ml of each antibody). 110 ML of virus suspension was mixed with 110 jjl of the respective recombinant human anti-SARS-CoV antibody dilution and incubated for one hour at 37°C. 100 pi of this suspension was then pipetted two times in duplicate into 96-well plates containing an 80% confluent monolayer of FRhK-4 cells in 100 ML  MM. The FRhK-4 cells were cultured at 37°C and observed after 3-4 days for the development of CPE. CPE was compared to the positive control (virus inoculated cells) and negative controls (mock-inoculated cells or cells incubated with recombinant antibody only). The complete absence of CPE in an individual cell culture was defined as protection (= 100% titer reduction). The concept of the combination index (CI) was used to quantitate synergistic effects as described previously (Chou and Talalay, 1984). According to the concept a combination of agents that produce an additive effect the sum of the ratios of their concentrations in the mixture (cmixt) to the concentrations of agents that individually have the same effect as the mixture (ceffect) is 1. This sum is the CI. When this sum is lower than 1, the agents act in synergy.<br><br>
For a two-component system, as in the present study, CI is calculated as follows:<br><br>
Clmixt is the concentration of the first component in the mixture which leads to a certain level of inhibition (f), cleffect is that concentration of the first component which alone (in the absence of the second component) will result in the same inhibitory effect as the mixture of the two components, and c2mixt and c2effect are the corresponding concentrations for the second component.<br>
To determine the CI for antibodies CR03-014 and CR03-022 the neutralization assay was performed as described above. Complete neutralization of 100 TCID50 of strain HKU39849 was reached at 7.6 jig/ml for CR03-014, 50.0 jig/ml for CR03-022 and 2.4 pg/ml of each antibody when added in combination. This outcome results in a CI of 2.4/7.6 + 2.4/50.0= 0.36. Fifty percent neutralization was achieved at 5.9 pg/ml for CR03-014, 30.2 pg/ml for CR03-022 and 1.7 pg/ml of each antibody when added in combination. This results in a CI of 1.7/5.9 + 1.7/30.2= 0.34. Thus, at both 50 and 100% neutralization CI values lower than 1 were obtained for a mixture of CR03-014 and CR03-022, which indicates that both antibodies act in synergy against SARS-CoV.<br><br><br>
An ELISA to detect binding of anti-SARS antibodies to SARS-CoV was performed as follows. Wells of ELISA plates were coated overnight with UV-inactivated SARS-CoV preparation in 50 mM bicarbonate buffer pH 9.6. The wells of the plates were washed three times with PBS containing 0.05% Tween and blocked for 2 hours at 37°C with PBS containing 1% BSA. Next, the antibodies diluted in PBS containing 1% BSA were incubated for 1 hour at room temperature. The wells were washed three times with PBS containing 0.05% Tween and incubated for 1 hour at room temperature using a murine anti-Hu-IgG HRP conjugate. Development was done with O-phenylenediamine substrate, the reaction was stopped by the addition of 1 M H2S04 and the absorbance was measured at 4 92 nm. As shown in Figure 2 antibodies CR03-014 and CR03-022 were both capable of binding an inactivated SARS-CoV preparation in ELISA in contrast to a negative control IgG which was directed against an irrelevant antigen.<br>
To detect the target of the antibodies CR03-014 and CR03-022 another binding assay was used. Single chain phage antibodies SC03-014 and SC03-022 were analyzed for their ability to bind HEK293T cells that recombinantly express proteins of SARS-CoV. To this end HEK293T cells were transfected with a plasmid carrying a cDNA sequence encoding the spike (S) protein from SARS-CoV strain Frankfurt 1 or with control vector. For flow cytometry analysis, single-chain phage antibodies were first blocked in an equal volume of 4% PBS-M for 15 minutes at 4°C prior to the staining of the transfected HEK293T cells. The blocked phage antibodies were added to mock transfected HEK293T cells and HEK293T cells transfected with the SARS-CoV S protein. The binding of the single chain phage antibodies to the cells was visualized using a biotinylated anti-M13 antibody (Santa Cruz<br><br>
Biotechnology) followed by streptavidin-phycoerythrin (Caltag). As shown in Figure 3, the single chain phage antibodies SC03-.014 and SC03-022 were capable of binding spike transfected HEK293T cells, whereas no binding to mock transfected HEK293T cells was observed. A control single chain phage antibody did neither recognize the spike transfected HEK293T cells nor the mock transfected HEK293T cells. These data suggest that both antibodies are directed against the S protein of SARS-CoV.<br>
To further localize the binding sites of the antibodies within the S protein, an assay was performed wherein the antibodies were analyzed for their ability to bind to portions of the S protein of SARS-CoV. The nucleotide and amino acid sequence of the S protein is shown in SEQ ID NO:114 and SEQ ID NO:115, respectively. DNA coding for the N-terminal 565 amino acids (portion called S565) was cloned as a KpnI-BamB.1 fragment in pAdapt (Havenga et al.,   2001) that was modified by insertion of the polylinker of the vector called pcDNA3.1/myc-His C (Invitrogen) (vector called pAdapt/myc-His C). A fragment corresponding to amino acid residues 318-510 of the S protein (portion called S318-510) was amplified on S gene cDNA using the oligonucleotide primers: £coRIspikeFor318 5'-cctggaattctccatggccaacatcaccaacc-3' (SEQ ID NO:116) and XbaIspikeRev510 5'-gaagggccctctagacacggtggcagg-3' (SEQ ID NO:117). The resulting fragment was digested with EcoRI-Xbal and cloned into pHAVT20/myc His A to yield pHAVT2 0/myc-His A S318-510. In this vector expression of fragment S318-510 fused to the HAVT20 leader sequence was under control of the human, full-length, immediate-early CMV promoter. DNA transfections were performed in HEK293T cells for transient expression using standard techniques. The S protein derived fragments were used directly from culture supernatant or were purified from<br><br>
culture supernatant using Ni-NTA (Qiagen). An ELISA to evaluate binding of antibodies to the S protein derived fragments was performed as follows. Wells of ELISA plates were coated overnight with 5 jig/ml anti-myc antibody in 50 mM bicarbonate buffer pH 9.6. The wells of the plates were washed three times with PBS containing 0.05% Tween and blocked for 2 hours at 37°C with PBS containing 1% BSA. The wells coated with anti-myc antibody were incubated with the myc-tagged fragments S565 or S318-510 diluted in PBS containing 1% BSA for 1 hour at room temperature. The wells-were washed three times with PBS containing 0.05% Tween. Next, the antibodies CR03-014, CR03-022 or control antibody diluted in PBS containing 1% BSA were incubated for 1 hour at room temperature. Detection of bound antibody was performed as described supra-   As shown in Figure 4, antibodies CR03-014 and CR03-022 were both capable of binding to the S565 and S318-510 fragment, but not to an irrelevant control myc-tagged antigen. A control antibody did not bind any of the fragments.<br>
In order to rank the affinities of the antibodies for binding to the S565 fragment, a titration of IgG concentration was performed (by diluting the antibodies in PBS containing 1% ELK), followed by an ELISA as described above. Titration of the monoclonal antibodies showed that CR03-014 and CR03-022 bound S565 with approximately similar affinities (see Figure 5). To investigate changes of affinities as a possible mechanism of synergy, the KD for CR03-014 and CR03-022 binding sequentially or simultaneously to recombinant receptor binding domain fragment S318-510 was investigated by means of BIAcore analysis. Surface plasmon resonance analyses were performed at 25°C on a BIAcore3000™. CM5 sensorchips and running buffer HBS-EP were from Biacore AB (Uppsala Sweden). Recombinant S318-510 fragment was immobilized to CM5 chips using an amine<br><br>
coupling procedure resulting in a response level of approximately 1,000 resonance units (RU). Kinetic analysis was performed to determine the association rate (ka), dissociation rate (kd) constants and the affinity (KD) of the monoclonal antibodies. Therefore, a concentration series of 0.4 to 250 nM IgG was prepared using two-fold dilutions in HBS-EP. Samples were injected in duplicate at a flow rate of 30 jil/min (injection time = 2 min, dissociation time = 5 min). The sensor chip surface was regenerated with a pulse of 5 ML   5 nM NaOH. Biacore evaluation software (BIAevalution, July 2 001) was used to fit the association and dissociation curves of all concentrations injected. The individual KD for CR03-014 and CR03-022 was determined as 16.3 nM and 0.125 nM, respectively, the KD for the antibodies binding simultaneously as 5.71 nM and for binding of CR03-014 to CR03-022 saturated S318-510 as 14.8 nM. Compared to the dose reduction indices of 3 and 2 0 for CR03-014 and CR03-022, respectively, neither simultaneous nor sequential binding of the monoclonal antibodies resulted in changes of KD which could explain their synergistic neutralizing action through cooperative binding.<br>
To further explore the antibody binding sites within the S protein, a competition ELISA on immobilized S318-510 fragment was performed. Captured S318-510 was incubated with non-saturating amounts of biotinylated antibody without competing antibody or in the presence of 1, 5, and 25 pg/ml of competing antibody (antibody CR03-014, CR03-022 or control antibody). Bound biotinylated antibody was detected with streptavidin-conjugated-HRP (BD Pharmingen) and developed as described above. Figure 6A shows that binding of monoclonal antibody CR03-014 was unaffected in the presence of excess unlabeled monoclonal antibody control or antibody CR03-022. Figure 6B shows that binding of antibody CR03-022 was<br><br>
unaffected in the presence of excess unlabeled monoclonal antibody control or antibody CR03-014. As expected, binding of both biotinylated CR03-014 and CR03-022 was effectively reduced by their unlabeled counterparts (see Figures 6A and B). These results demonstrate that the antibodies CR03-014 and CR03-022 do not compete with each other for binding to the S318-510 fragment and recognize different/distinct, non-competing epitopes.<br>
To confirm these findings, an antibody sandwich ELISA was performed. Antibodies CR03-014 and CR03-022 were coated overnight to microtiter wells at 5 jjg/ml in 50 mM bicarbonate buffer pH 9.6. Capture of the S318-510 fragment, binding of biotinylated antibodies and subsequent development of the ELISA reaction was performed as described supra-   Figure 7 indicates that CR03-014-captured S318-510 could be bound by biotinylated CR03-022, but not by CR03-014. CR03-022-captured S318-510 could only be bound by biotinylated CR03-014 and not CR03-022. This indicates that CR03-014 and CR03-022 are able to bind simultaneously to different epitopes within the S derived fragment S318-510 and furthermore indicates that both antibodies bind to different non-competing epitopes.<br>
Example  8<br>
Construction  and evaluation  of binding of  the monoclonal  anti-<br>
SARS-CoV antibodies   to  variant   S318-510  fragments.<br>
The diversity within the region 318-510 of the S protein was determined as follows. A list containing more than 14 6 genomes or genes encoding complete human SARS-CoV or fragments thereof was compiled. In 114 cases, an open reading frame encoding for full-length spike (S) protein was identified. Alignment of the spike amino acid residues 318-510 revealed 30 spike proteins, in which the region 318-510 was not identical<br><br>
to that of the spike protein of strain Frankfurt 1 (see Genbank accession number AY291315), which was used herein as wild-type. The mutations, strains and Genbank numbers are depicted in Table 15. To investigate if CR03-014 and CR03-022 bound the S protein of all currently known human SARS-CoV isolates, eight recombinant spike 318-510 fragments (variants A-F and variants H and I) harbouring the different amino acid substitutions as shown in Table 15 were generated. In addition, a 318-510 fragment (variant G) corresponding to the sequence of four animal SARS-like CoVs (Genbank accession numbers AY304486 - AY304489; see also Table 1, SARS-CoV SZ3, SZ13, SZ16 and SZ1, respectively) was generated. The four SARS-like CoVs, which were isolated from raccoon dogs and palm civet cats, contain the amino acid substitutions K344R, F360S, N479K and T487S (see Guan et al.   2003). To this end, the above substitutions were introduced in the pHAVT20/myc-His A S318-510 vector using the Stratagene's QuikChange II site-directed mutagenesis kit according to the manufacturer's instructions. In case a sequence contained multiple amino acid substitutions, the process of mutagenesis, sequence analysis and confirmation was repeated for every single substitution. To rule out the introduction of additional mutations in the plasmid outside the gene of interest, the mutated (592 bp EcoRI-Xbal) fragment was recloned in EcoRI-Xbal cut pHAVT2 0/myc-His A. The resulting plasmids were transfected into HEK293T cells, and binding of CR03-014 and CR03-022 was evaluated by means of ELISA as described supra - As shown in Figure 8, CR03-014 was capable of binding to variants A-E and variants H and I to a similar extent as to the wild-type fragment. Binding of CR03-014 to variant F (N479S substitution) was substantially lower than binding to the other fragments. No binding of CR03-014 to fragment G (K344R,<br><br>
F360S, N479K and T487S substitutions) was observed. Antibody CR03-022 was capable of binding all variant S318-510 fragments to a similar extent as the wild-type (non-mutated) S318-510 fragment. Together this indicates that residue N479 is involved in binding of CR03-014, either directly by being part of the binding site of CR03-014 or indirectly by being important for the correct conformation of the binding site of CR03-014 within the S protein. Since, antibody CR03-022 is capable of binding to recombinant fragments composed of amino acid residues 318-510 of all human SARS-CoV isolates (as described in Table 15) and in addition is also capable of binding to animal SARS-like CoV, CR03-022 is suitable for treatment and/or protection against SARS-CoV isolates in general.<br>
Particularly suitable for treatment and/or protection against human SARS-CoV isolates is a combination/cocktail comprising both antibodies, CR03-014 and CR03-022, as both antibodies are capable of specifically binding to human SARS-CoV and the antibodies act synergistically in neutralizing human SARS-CoV. In other words, the combination/cocktail of CR03-014 and CR03-022 comprises synergistic human SARS-CoV neutralizing activity. An additional advantage of such a combination/cocktail is its capability of neutralizing human SARS-CoV as well as animal SARS-like CoV.<br>
Example  9<br>
Generation   of CR03-014  and CR03-022  escape  viruses   of SARS-CoV To further elucidate the epitopes recognized by the human monoclonal antibodies CR03-014 and CR03-022, escape viruses of CR03-014 and CR03-022 were generated. The process for generating escape viruses of CR03-014 is given infra-   The process for generating escape viruses of CR03-022 was<br><br>
identical with the proviso that 60 fig/ml antibody instead of 20 fig/ml was used in all respective steps. Serial dilutions (0.5 ml) of SARS-CoV strain HKU 39849 (dilutions ranging from 10"1 -10"8) were incubated with a constant amount (20 jig/ml giving a -3 log reduction of TCID50/ml) of antibody CR03-014 (0.5 ml) for 1 hour at 37°C/5% C02 before addition to wells containing FRhK-4 cells. The virus was allowed to attach to the cells for 1 hour at 37°C/5% C02, then removed and cells were washed twice with medium. Finally, cells were incubated for 2 days in the presence of selecting antibody CR03-014 at 20 fig/ml in 0.5 ml medium. Then, medium of wells with highest virus dilution showing CPE (cytopathic effect) containing potential escape<br>
viruses was harvested and stored at 4°C until further use. Subsequently, virus samples were freeze/thawed once and serial dilutions were prepared in medium. Virus dilutions were added to wells containing FRhK-4 cells and incubated for 1-2 hours<br>
at 37°C/5% C02 in the presence of CR03-014 at 20 jag/ml. Wells were then overlayed with agarose containing CR03-014 at 2 0<br>
jig/ml and incubated for 3-5 days at 37°C/5% C02 - Individual escape virus plaques were picked using a Pasteur pipet, freeze/thawed once, and the escape viruses were amplified on FRhK-4 cells- To analyse the escape viruses the following experiments were performed.<br>
Firstly, to identify possible mutations in the SARS-CoV spike protein of each of the escape viruses the nucleotide sequence of the SARS-CoV spike open reading frame (ORF) was determined. Viral RNA of each of the escape viruses and wild-type SARS-CoV virus was isolated and converted into cDNA by standard RT-PCR. Subsequently, the cDNA was used for nucleotide sequencing of the SARS-CoV spike ORF in order to identify mutations. Figure 9 shows the results of the<br><br>
sequencing data for the five E014 escape viruses obtained. All escape viruses contained a nucleotide mutation at position 1385 (C to T), which resulted in an amino acid mutation P to L at position 462 in the spike protein. Apparently, P462L resulted in loss of epitope recognition and, subsequently, loss of neutralization of SARS-CoV by CR03-014. This indicates that next to amino acid 479, also amino acid 462 is involved in the binding of CR03-014 to the S protein of SARS-CoV. The results of the sequencing data for the five E022 escape viruses obtained -were as follows. Four out of five escape viruses contained a nucleotide mutation at position 2588 (C to T), which resulted in an amino acid mutation T to I at position 863 in the spike protein.<br>
Secondly, the neutralization index (NI) was determined for each of the E014 and E022 escape viruses. A virus was defined as an escape variant, if the neutralization index was 
escape viruses with the proviso that 60 jig/ml instead of 20<br>
jig/ml monoclonal antibody was used in all respective steps. The neutralization index was determined by subtracting the number of infectious virus particles (in TCID50/ml) produced in FRhK-4 cell cultures infected with virus plus monoclonal antibody (20<br>
jig/ml) from the number of infectious virus particles (in TCID50/ml) produced in FRhK-4 cell cultures infected with virus alone ( [log TCID50/ml virus in absence of monoclonal antibody minus log TCID50/ml virus in presence of monoclonal antibody]). An index lower than 2.5 logs was considered as evidence of escape. Therefore, each escape virus and wild-type<br>
SARS-CoV (100 TCID50) was incubated for 1 hour at 37°C/5% C02<br>
with 20 jig/ml of CR03-014 before addition to FRhK-4 cells. The<br><br>
virus was allowed to attach to the cells for 1 hour at 37°C/5% C02 after which the inoculum was removed and cells were washed twice with medium before being replenished with medium containing 20 jig/ml of CR03-014. After a 2 day incubation at<br>
37°C/5% C02 the medium was harvested and the TCID50/ml of each virus was determined- As shown in Table 16, the concentration of antibody used to determine the NI resulted in a -3 log reduction of virus titer when performed on the wild-type SARS-CoV virus. Thus, wild type SARS-CoV was neutralized by CR03-014 as judged by the NI of 3.3. In contrast, the NI for each escape virus was 2.5 and thus it was concluded that each of the escape viruses was still neutralized by CR03-022. The amino acid substitution in four of the five E022 escape viruses apparently does not play a role in neutralization of SARS-CoV by CR03-022. It might have been induced non-specifically during the course of the experiment. This agrees with finding by Poon et al. (2005) who observed the mutation at position 8 63 (T to I) when SARS-CoV was passaged multiple times in FRhK-4 cells. The neutralizing epitope of CR03-022 could not be determined by means of generating escape viruses. This may be caused by the functional constraints of the binding region on the S protein. A mutation occurring in this region may be detrimental to the stability of the virus and could therefore not be isolated in the experiments described above.<br>
In order to confirm lack of binding of antibody CR03-014 to the escape variant, a recombinant S318-510 fragment harbouring the P to L substitution at position 4 62 was<br><br>
constructed essentially as described supra.   DNA transfection of the resulting plasmid was performed in HEK293T cells for transient expression and the fragments were directly used from culture supernatant. The ELISA was performed as described supra-   Briefly, the fragments were captured on anti-myc coated microtiter wells. Subsequently, antibodies CR03-014 and CR03-022 were added and binding of the antibodies was detected using an anti-human IgG HRP-conjugate. As shown in Figure 10, antibody CR03-014 was not able to bind the S318-510 fragment carrying a P to L substitution at position 4 62. Binding of CR03~022 was not affected by this amino acid substitution. This further indicates that antibody CR03-022 is capable of binding to a different/distinct, non-competing epitope on the S protein and suggests that CR03-022 might be used to compensate potential lack of neutralization of SARS-CoV variants by CR03-014.<br>
Furthermore, cross-neutralization assays were performed on E14 escape viruses. Table 18 clearly shows that CR03-022 neutralized the E14 escape viruses to a similar level as wild-type virus further illustrating that CR03-022 binds to a different epitope compared to CR03-014, which no longer neutralized the E14 escape viruses. This result is in agreement with the ELISA data shown in Figure 10. The reverse experiment could not be performed as escape viruses of CR03-022 could not be generated. From the above can be concluded that the combination of CR03-014 and CR03-022 in a cocktail prevents the escape of neutralization-resistant SARS-CoV variants and is therefore an ideal immunoglobulin preparation for SARS-CoV prophylaxis and therapy.<br><br><br>
It is known that in certain diseases caused by coronaviruses prior immunity or passive antibody increases the severity of the disease. In feline infectious peritonitis, the macrophage is the main target cell for virus replication and anti-viral antibodies enhance replication of the virus in macrophage cultures in  vitro. The macrophage is also a prominent cell seen in the cell infiltrates of lungs of patients dying of SARS-CoV (Nicholls et al.   2003). This has led to concern whether antibody dependent enhancement (ADE) may be relevant in the pathogenesis of SARS-CoV. To investigate this it was tested whether ADE occurred when macrophages were infected with SARS-CoV in the presence of the neutralizing anti-SARS-CoV monoclonal antibody CR03-014, the non-neutralizing anti-SARS-CoV monoclonal antibody CR-03-015, the monoclonal antibody called CR-JA (an monoclonal antibody against rabies which is used herein as a control antibody), convalescent serum from an individual exposed to SARS-CoV and serum from a healthy individual.<br>
Human peripheral blood mononuclear cells (PBMCs) were obtained from leukocyte-rich buffy coats of healthy blood donors (The Hong Kong Red Cross Blood Transfusion Service, Hong Kong). The PBMCs were separated by Ficoll-Paque gradient centrifugation (Pharmacia Biotech, Uppsala, Sweden). To isolate monocytes 2 x 107 PBMCs were allowed to adhere onto petridishes (Greiner, Frickenhausen, Germany) for 1 hour in RPMI 164 0 medium supplemented with 2 0 mM HEPES, 2 mM glutamine, 0.6 ]iq/ml  penicillin, and 60 ]ig/ml   streptomycin and 5% heat-inactivated autologous plasma. After washing with<br><br>
medium, the adherent monocytes were detached by pipeting and re-seeded into 24 well plates at a density of 2,105 cells per well in supplemented RPMI 1640 medium.<br>
To check the purity of the monocyte preparations monocytes were also seeded and allowed to adhere onto glass coverslips. The purity of the monocytes on the glass coverslips was confirmed by staining with a CD14 R-phycoerythrin (R-PE)-conjugated mouse anti-human monoclonal antibody (BD Biosciences, San Diego, U.S.A.).<br>
Medium in the monocyte cultures was replaced every 2-3 days and the cells were allowed to differentiate for 14 days in  vitro.   Differentiation of monocytes into macrophages was confirmed by the typical morphology of macrophages. The obtained primary human monocyte-derived macrophages were used in further experiments. Two days prior to the ADE experiments, the supplemented RPMI 164 0 medium was exchanged into Macrophage Serum Free medium (SFM) (Invitrogen, Carlsbad, CA, U.S.A.).<br>
To investigate the effect of sub-neutralizing doses of antibody on viral infection in macrophages, 300 pi of serial 10-fold dilutions in MM medium (MEM including 1% FCS and 0.6 ]ig/ml  penicillin, and 60 jag/ml streptomycin) of the monoclonal antibodies CR03-014, CR03-015 and CR-JA, convalescent serum from a SARS-CoV exposed individual and serum from a healthy individual was mixed with 300 jil of SARS-CoV. MM medium mixed with SARS-CoV served as the virus control. The virus/monoclonal antibody mixtures and virus/serum mixtures were incubated for one hour at 37 °C. Then, 250 ML  of the mixtures was added to duplicate wells containing macrophages. After one hour of virus adsorption at 37 °C, the virus inoculum was removed, infected cells were washed with macrophage SFM culture medium and incubated in macrophage SFM medium<br><br>
supplemented with 0.6 pg/ml penicillin and 60 pg/ml streptomycin. Samples of the culture supernatants were collected at days 0, 1, 2, 3, 5, and 7 post-infection and stored at -70°C for virus titration experiments. SARS-CoV was titrated and the TCID50 determined essentially as described supra.<br>
To detect virus inside the macrophages (due to abortive infection) and to detect potential transcription of SARS-CoV RNA inside the macrophages RNA was isolated from infected macrophages at 3, 6, and 24 hours post-infection using the RNeasy Mini kit (Qiagen) according to the manufacturer's instructions. Reverse transcription with oligo-dT primers was performed by using Superscript II reverse transcriptase (Invitrogen) according to the manufacturer's instructions. Complementary DNA was generated with 10 pi of RNA, and reverse-transcribed by 200 U of Superscript II reverse transcriptase (Invitrogen) in a 20 pi reaction containing 25 ng oligo-dT12-i8 primer, 10 mM dithiothreitol, and 0.5 mM deoxynucleotide triphosphates. Reactions were incubated at 42°C for 50 minutes, followed by a heat inactivation step (72°C for 15 minutes). The reaction mix was diluted 10 times by the addition of 180 pi buffer AE (Qiagen) and stored at -20°C.<br>
Real time quantitative PCR was performed using FastStart DNA Master SYBR Green I fluorescence reaction (Roche). 5 pi of diluted complementary DNA was amplified in a 20 pi reaction containing 4 mM of MgCl2/ 0.5 mM of forward primer (Actin-LF: CCCAAGGCCAACCGCGAGAAGAT (SEQ ID NO:122)), and 0.5 mM of reverse primer (Actin-LR: GTCCCGGCCAGCCAGGTCCAG (SEQ ID NO:123)). Reactions were performed in a LightCycler (Roche) with the following conditions: 10 minutes at 95CC, followed by 40 cycles of 95°C for 0 seconds, 66°C for 5 seconds, and 72°C<br><br>
for 9 seconds. Plasmids containing the target sequence were used as positive controls. Fluorescence signals from these reactions were captured at the end of the extension step in each cycle (87°C). To determine the specificity of the assay, PCR products were subjected to melting curve analysis at the end of the assay (65 to 95°C; 2°C/second). Reverse transcription with sense (negative strand detection) or anti-sense (positive strand detection) primers to the polymerase gene of SARS-CoV was achieved by using Superscript II reverse transcriptase (Invitrogen) according to manufacturer's instructions. Complementary DNA was generated with 5 jiL of RNA, and reverse-transcribed by 200 U of Superscript II reverse transcriptase (Invitrogen) in a 20 pi reaction containing 0.1 jiM gene specific primer, 10 mM dithiothreitol, and 0.5 mM deoxynucleotide triphosphates. Reactions were incubated at 42 °C for 50 minutes, followed by a heat inactivation step (72°C for 15 minutes). The reaction was diluted 10 times by the addition of 180 |iL buffer AE (Qiagen) and stored at -20°C. 2 jil of diluted complementary DNA was amplified in 20 ML   containing 3.5 mM of MgCl2, 0.25 ]iM  of forward primer (coro3: 5'-TACACACCTCAGCGTTG-3f (SEQ ID NO:124)), and 0.25 ]M  of reverse primer (coro4: 5'-CACGAACGTGACGAAT-31 (SEQ ID NO:125)). Reactions were performed in a LightCycler (Roche) with the following conditions: 10 min at 95°C, followed by 50 cycles of 95°C for 10 seconds, 60°C for 5 seconds, and 72°C for 9 seconds. Plasmids containing the target sequence were used as positive controls. Fluorescence signals from these reactions were captured at the end of the extension step in each cycle. To determine the specificity of the assay, PCR products were subjected to melting curve analysis at the end of the assay (65 to 95°C; 0.l°C/seconds).<br><br>
Data for viral RNA were normalised for RNA levels of p-actin housekeeping gene.<br>
After infection of macrophages with SARS-CoV, the cells were monitored daily under the microscope. Ten days after infection with SARS-CoV no detectable cytopathic effect was detected, nor was the abortive infection of macrophages converted into a productive infection. Pre-incubating of SARS-CoV with different concentrations of the monoclonal antibodies CR03-014, CR03-015, CR-JA, convalescent serum or serum from a healthy individual, prior to infection of macrophages did not change this out-come.<br>
Titration of aliquots of supernatant taken from infected macrophage culture at days 0, 1, 2, 3, 5, and 7 days postinfection on FRhk-4 cells revealed no evidence of significant enhancement of SARS-CoV replication in macrophages by the monoclonal antibodies and sera tested (data not shown).<br>
In addition, the effect of antibodies on the (abortive) infection of macrophages by SARS-CoV and the transcription of SARS-CoV genes within the macrophages was measured on the molecular level. To this end, total RNA was extracted from the infected macrophages at various time points post-infection as described supra-   Subsequently, total RNA was analyzed for SARS-CoV viral positive strand RNA and viral negative strand RNA transcripts using real-time RT-PCR. The SARS-CoV RNA<br>
levels were normalized for the levels of p-actin mRNA. The results show that positive strand SARS-CoV RNA was detected in all macrophage cultures that were incubated with SARS-CoV, which confirms the abortive infection of macrophages by SARS-CoV. The levels of positive strand RNA observed in macrophage cultures infected with SARS-CoV in the presence of anti-SARS-CoV monoclonal antibodies CR03-014 or CR03-015 or convalescent serum were not significantly hiqher than in macrophage<br><br>
cultures infected with SARS-CoV in the presence of control monoclonal antibody CR-JA or serum from a healthy individual or in the absence of monoclonal antibody or serum (data not shown).<br>
Furthermore, the presence of negative strand RNA which is indicative for SARS-CoV gene transcriptional activity after infection of macrophages by SARS-CoV was measured. Again no correlation between the levels of negative strand RNA and the presence or absence of anti-SARS-CoV antibodies was observed (data not shown).<br>
Together, the experiments assaying infectious virus yield and virus related RNA levels inside macrophages showed that there was no antibody dependent enhancement of SARS-CoV replication in human macrophages by anti-SARS-CoV monoclonal antibodies CR03-014 and CR03-015 and anti-SARS-CoV antibodies present in convalescent serum from a SARS patient,<br>
Example  11<br>
Identification  of epitopes   recognized by recombinant  human<br>
anti-SARS-CoV antibodies by PEPSCAN-ELISA<br>
15-mer linear and looped/cyclic peptides were synthesized from proteins of SARS-CoV and screened using credit-card format mini-PEPSCAN cards (455 peptide formats/card) as described previously (see inter alia  WO 84/03564, WO 93/09872, Slootstra et   al.   1996). All peptides were acetylated at the amino terminus. In short, series of overlapping peptides, which were either in linear form or in looped/cyclic form, of the spike protein of SARS-CoV Urbani (the protein-id of the surface spike glycoprotein in the EMBL-database is AAP13441), was produced and tested for binding to the recombinant human anti-SARS-CoV antibodies of the invention by means of PEPSCAN analysis.<br><br>
Because the Urbani spike protein indicated above was also found in identical or highly homologous form in other SARS-CoV strains, the antigenic peptides found in the analysis method may not only be used for detection of the SARS-CoV strain Urbani and the prevention and/or treatment of a condition resulting from the SARS-CoV strain Urbani, but may also be useful in detecting SARS-CoV in general and preventing and/or treating a condition resulting from SARS-CoV in general. The protein-id of the surface spike glycoprotein of for instance the SARS-CoV strains T0R2, Frankfurt 1 and HSR 1 in the EMBL-database is AAP41037, AAP33697 and AAP72986. The accession number in the EMBL-database of the complete genome of the strains T0R2, Frankfurt 1 and HSR 1 is AY274119, AY291315 and AY323977, respectively. Under these accession numbers the amino acid sequence of the other (potential) proteins of these strains can be found.<br>
In all looped peptides position-2 and position-14 were replaced by a cysteine (acetyl-XCXXXXXXXXXXXCX-minicard). If other cysteines besides the cysteines at position-2 and position-14 were present in a prepared peptide, the other cysteines were replaced by an alanine. The looped peptides were synthesized using standard Fmoc-chemistry and deprotected using trifluoric acid with scavengers. Subsequently, the deprotected peptides were reacted on the cards with an 0.5 mM solution of 1,3-bis(bromomethyl)benzene in ammonium bicarbonate (20 mM, pH 7.9/acetonitril (1:1 (v/v)). The cards were gently shaken in the solution for 30-60 minutes, while completely covered in the solution. Finally, the cards were washed extensively with excess of H20 and sonicated in disrupt-buffer containing 1% SDS/0.1% beta-mercaptoethanol in<br>
PBS (pH 7.2) at 70°C for 30 minutes, followed by sonication in H20 for another 45 minutes.<br><br>
Recombinant human anti-SARS-CoV antibodies CR03-014 and CR03-022 were tested for binding to each linear and looped peptide in a PEPSCAN-based enzyme-linked immuno assay (ELISA). The 455-well creditcard-format polypropylene cards, containing the covalently linked peptides, were incubated with the antibodies (1-10 jag/ml; diluted in blocking solution which contains 5% horse-serum (v/v) and 5% ovalbumin (w/v)) (4°C, overnight). After washing, the peptides were incubated with anti-human antibody peroxidase (dilution 1/1000) (1 hour, 25°C), and subsequently, after washing the peroxidase substrate 2,2 f-azino-di-3-ethylbenzthiazoline sulfonate (ABTS) and 2 ML/ml   3% H202 were added. Controls (for linear and looped) were incubated with anti-human antibody peroxidase only. After 1 hour the colour development was measured. The colour development of the ELISA was quantified with a CCD-camera and an image processing system. The set-up consisted of a CCD-camera and a 55 mm lens (Sony CCD Video Camera XC-77RR, Nikon micro-nikkor 55 mm f/2.8 lens), a camera adaptor (Sony Camera adaptor DC-77RR) and the Image Processing Software package Optimas, version 6.5 (Media Cybernetics, Silver Spring, MD 20910, U.S.A.). Optimas runs on a pentium II computer system.<br>
The recombinant human anti-SARS-CoV-antibodies CR03-014 and CR03-022 were tested for binding to the 15-mer linear and looped/cyclic peptides synthesized as described supra. Relevant binding of a peptide to a recombinant human anti-SARS-CoV antibody was calculated as follows. The average OD-value for each antibody was calculated for the respective proteins (sum of OD-values of all peptides/total number of peptides). Next, the standard deviation of this average was calculated. The standard deviation was multiplied by 2 and the obtained value was added to the average value to obtain the<br><br>
correction factor. The OD-value of each peptide was then divided by this correction factor. If a value of 0.9 or higher was found, then relevant binding was considered to be present between the specific peptide and the respective antibody. Particularly interesting appear to be domains comprising several reactive peptides, i.e. domains comprising consecutive peptides, wherein at least most of the peptides in the domains are reactive with the antibody.<br>
Monoclonal antibody CR03-014 did not appear to react specifically with a peptide or domains comprising several peptides within the SARS-CoV spike protein indicating that CR03-014 may recognize a discontinuous non-linear epitope. Monoclonal antibody CR03-022 reacted with a series of looped peptides in two domains (data not shown). The domains are comprised of amino acid residues 430-449 and 484-503 of the SARS-CoV spike protein and have the amino acid sequences ATSTGNYNYKYRYLRHGKLR (SEQ ID NO: 12 6) and YTTTGIGYQPYRWVLSFEL (SEQ ID NO;127), respectively. Strikingly, both domains have the motif TXTGXXXXXYR (SEQ ID NO:128) in common, indicating that this motif may be crucial for the binding of antibody CR03-022 to the SARS-CoV spike protein.<br><br>
Table 1. List of currently known SARS-CoV genome sequence and<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
REFERENCES<br>
Boel E, Verlaan S, Poppelier MJ, Westerdaal NA, Van Strijp JA and Logtenberg T (2000), Functional human monoclonal antibodies of all isotypes constructed from phage display library-derived single-chain Fv antibody fragments. J. Immunol. Methods 239:153-166.<br>
Burton DR and Barbas CF (1994), Human antibodies from combinatorial libraries. Adv. Immunol. 57:191-280.<br>
Chou, TC and P Talalay (1984), Quantitative analysis of dose-effect relationships: the combined effects of multiple drugs or enzyme inhibitors. Adv Enzyme Regul. 22:27-55.<br>
De Kruif J, Terstappen L, Boel E and Logtenberg T (1995a), Rapid selection of cell subpopulation-specific human monoclonal antibodies from a synthetic phage antibody library. Proc. Natl. Acad. Sci. USA 92:3938.<br>
De Kruif J, Boel E and Logtenberg T (1995b), Selection and application of human single-chain Fv antibody fragments from a semi-synthetic phage antibody display library with designed CDR3 regions. J. Mol. Biol. 248:97-105.<br>
Emini EA, Nara PL, Schleif WA, Lewis JA, Davide JP, Lee DR, Kessler J, Conley S, Matsushita S, Putney SDJ, Gerety RJ and Eichberg JW (1990), Antibody-mediated in  vitro neutralization of human immunodeficiency virus type 1 abolishes infectivity for chimpanzees. J. Virol. 64:3674-3678.<br>
Fouchi'er RA, Kuiken T, Schutten M, van Amerongen G, van Doornum GJ, van den Hoogen BG, Peiris M, Lim W, Stohr K, Osterhaus AD (2003), Koch's postulates fulfilled for SARS virus. Nature 243:240.<br>
Guan Y, Zheng BJ, He YQ, Liu XL, Zhuang ZX, Cheung CL, Luo SW, Li PH, Zhang LJ, Guan YJ, Butt KM, Wong KL, Chan KW, Lim W, Shortridge KF, Yuen KY, Peiris JS, and Poon LL (2003),<br><br>
Isolation and characterization of viruses related to the SARS coronavirus from animals in southern China, Science 302:27 6-278.<br>
Havenga MJ, Lemckert AA, Grimbergen JM, Vogels R, Huisman LG, Valerio D, Bout A and Quax PH (2001), Improved adenovirus vectors for infection of cardiovascular tissues. J. Virol. 75:3335-3342.<br>
Holmes KV (2003), SARS coronavirus: a new challenge for prevention and therapy. J. Clin. Invest. 111:1605-1609.<br>
Huls G, Heijnen IJ, Cuomo E, van der Linden J, Boel E, van de Winkel J and Logtenberg T (1999), Antitumor immune effector mechanisms recruited by phage display-derived fully human IgGl and IgAl monoclonal antibodies. Cancer Res. 59:5778-5784.<br>
Ksiazek TG, Erdman D, Goldsmith CS, Zaki SR, Peret T, Emery S, Tong S, Urbani C, Comer JA, Lim W, Rollin PE, Dowell SF, Ling AE, Humphrey CD, Shieh WJ, Guarner J, Paddock CD, Rota P, Fields B, DeRisi J, Yang JY, Cox N, Hughes JM, LeDuc JW, Bellini WJ, Anderson LJ (2003), A novel coronavirus associated with severe acute respiratory syndrome. N. Eng. J. Med. 348:1953-1966.<br>
Kuiken T, Fouchier RA, Schutten M, Rimmelzwaan GF, van Amerongen G, van Riel D, Laman JD, de Jong T, van Doornum G, Lim W, Ling AE, Chan PK, Tarn JS, Zambon MC, Gopal R, Drosten C, van der Werf S, Escriou N, Manuguerra JC, Stohr K, Peiris JS and Osterhaus AD (2003), Newly discovered coronavirus as the primary cause of severe acute respiratory syndrome. Lancet 362:263-270.<br>
Li W, Moore MJ, Vasilieva N, Sui J, Wong SK, Berne MA, Somasundaran M, Sullivan JL, Luzuriaga K, Greenough TC, Choe H and Farzan M (2003), Angiotensin-converting enzyme 2 is a<br><br>
functional receptor for the SARS coronavirus. Nature 2003 426:450-454.<br>
Marra MA, Jones SJ, Astell CR, Holt RA, Brooks-Wilson A, Butterfield YS, Khattra J, Asano JK, Barber SA, Chan SY, Cloutier A, Coughlin SM, Freeman D, Girn N, Griffith OL, Leach SR, Mayo M, McDonald H, Montgomery SB, Pandoh PK, Petrescu AS, Robertson AG, Schein JE, Siddiqui A, Smailus DE, Stott JM, Yang GS, Plummer F, Andonov A, Artsob H, Bastien N, Bernard K, Booth TF, Bowness D, Czub M, Drebot M, Fernando L, Flick R, Garbutt M, Gray M, Grolla A, Jones S, Feldmann H, Meyers A, Kabani A, Li Y, Normand S, Stroher U, Tipples GA, Tyler S, Vogrig R, Ward D, Watson B, Brunham RC, Krajden M, Petric M, Skowronski DM, Upton C, Roper RL (2003), The genome sequence, of the SARS-associated coronavirus. Science 300, 1399-1404.<br>
Nicholls JM, Poon LL, Lee KC, Ng WF, Lai ST, Leung CY, Chu CM, Hui PK, Mak KL, Lim W, Yan KW, Chan KH, Tsang NC, Guan Y, Yuen KY, Peiris JS (2003) , Lung pathology of fatal severe acute respiratory syndrome. Lancet 361:1773-1778.<br>
Poon LLM, Leung CSW, Yuen KH, Guan Y, and Peiris JSM (2005),  Recurrent mutations associated with isolation and passage of SARS Coronavirus in cells from non-human primates. J. Med. Virol. 76:435-440.<br>
Rickerts V, Wolf T, Rottmann C, Preiser W, Drosten C, Jakobi V, Leong HN, Brodt HR (2003), Klinik und Behandlung des schweren akuten respiratorischen Syndroms. Dtsch. Med. Wochenschrift 128:1109-1114.<br>
Rota PA, Oberste MS, Monroe SS, Nix WA, Campagnoli R, Icenogle JP, Penaranda S, Bankamp B, Maher K, Chen MH, Tong S, Tamin A, Lowe L, Frace M, DeRisi JL, Chen Q, Wang D, Erdman DD, Peret TC, Burns C, Ksiazek TG, Rollin PE, Sanchez A, Liffick S, Holloway B, Limor J, McCaustland K, Olsen-Rasmussen M, Fouchier R, Gunther S, Osterhaus AD, Drosten C, Pallansch<br><br>
MA, Anderson LJ, Bellini WJ. (2003), Characterization of a novel coronavirus associated with severe acute respiratory syndrome. Science 300:1394-1399.<br>
Slootstra JW, Puijk WC, Ligtvoet GJ, Langeveld JP, Meloen RH (1996), Structural aspects of antibody-antigen interaction revealed through small random peptide libraries. Mol. Divers. 1:87-96.<br>
Ter Meulen J, Bakker ABH, van den Brink EN, Weverling GJ, Martina BEE, Haagmans BL, Kuiken T, de Kruif J, Preiser W, Spaan W, Gelderblom HR, Goudsmit J, Osterhaus ADME (2004), Human monoclonal antibody as prophylaxis for SARS coronavirus infection in ferrets. The Lancet 363:2139-2141.<br><br><br><br><br><br><br><br>
CLAIMS<br>
1.	A composition comprising at least two immunoglobulins that are capable of specifically binding to a SARS-CoV and have SARS-CoV neutralizing activity, characterized in that the relative amounts of the immunoglobulins are such that they exhibit a synergistic neutralizing activity.<br>
2.	A composition according to claim 1, characterized in that the immunoglobulins are capable of reacting with different, non-competing epitopes of the SARS-CoV.<br>
3.	A composition according to claims 1 or 2, characterized in that the immunoglobulins are capable of reacting with different, non-competing epitopes of the S protein of SARS-CoV.<br>
4.	A composition according to any of the claims 1-3, characterized in that the immunoglobulins are capable of reacting with different, non-competing epitopes of amino acids 318-510 of the S protein of SARS-CoV.<br>
5.	A composition according to any of the claims 1-4, characterized in that at least one of the immunoglobulins reacts with an epitope comprising the amino acid sequence of SEQ ID NO:128.<br>
6.	A composition according to any of the claims 1-5, characterized in that at least one of the immunoglobulins is capable of reacting with amino acids 318-510 of the S protein of a human and an animal SARS-CoV.<br>
7.	A composition according to any of the claims 1-6, characterized in that at least one of the immunoglobulins is capable of reacting with amino acids 318-510 of the S protein of a SARS-CoV, wherein amino acid 479 is an amino acid other than asparagine, to a similar extent as with amino acids 318-510 of the S protein of a SARS-CoV, wherein amino acid 479 is an asparagine.<br>
8.	A composition according to any of the claims 1-7, characterized in that each of the immunoglobulins is capable of neutralizing a plurality of SARS-CoV strains.<br>
9.	A composition according to any of the claims 1-8, characterized in that the molar ratio of the two immunoglobulins is from 1:100 to 100:1.<br>
10.	A composition according to any of the claims 1-9, characterized in that a first immunoglobulin comprises at least a heavy chain CDR3 region comprising the amino acid<br><br>
sequence of SEQ ID NO: 1 and a second immunoglobulin comprises at least a heavy chain CDR3 region comprising the amino acid sequence of SEQ ID NO:2.<br>
11.	A pharmaceutical composition comprising a composition according to any of the claims<br>
1-10, the pharmaceutical composition further comprising a pharmaceutically acceptable<br>
excipient.<br>
12.	A pharmaceutical composition according to claim 11 for use as a medicament.<br>
13.	A pharmaceutical composition according to claim 11 for use in the diagnosis, prophylaxis, treatment, or combination thereof of a condition resulting from a SARS-CoV.<br>
14.	Use of a pharmaceutical composition according to claim 11 in the preparation of a medicament for the diagnosis, prophylaxis, treatment, or combination thereof of a condition resulting from a SARS-CoV.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgIDIxLTEwLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   AMENDED  PAGES OF SPECIFICATION   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgQU1FTkRFRCBDTEFJTVMgICAyMS0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   AMENDED CLAIMS   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgQ09SUkVTUE9OREVOQ0UgIE9USEVSUyAgICAwNi0wMy0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   CORRESPONDENCE  OTHERS    06-03-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyNC0xMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   CORRESPONDENCE OTHERS  24-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAgMjEtMTAtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   EXAMINATION REPORT REPLY RECEIVED   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgRk9STS0zICAgMjEtMTAtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   FORM-3   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAgMjEtMTAtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   OTHER PATENT DOCUMENT   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UIDEgICAyMS0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   OTHER PATENT DOCUMENT 1   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgT1RIRVJTICAgMjEtMTAtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   OTHERS   21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICAgUE9XRVIgT0YgQVRUT1JORVkgIDIxLTEwLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007   POWER OF ATTORNEY  21-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICBBTUVOREVEIENMQUlNUyAgMjAtMDMtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007  AMENDED CLAIMS  20-03-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1DSEVOUC0yMDA3ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDIwLTAzLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">2003-CHENP-2007  EXAMINATION REPORT REPLY RECEIVED  20-03-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1jaGVucC0yMDA3LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2003-chenp-2007-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMy1QLTIwMDcgKFBldGl0aW9uIGZvciBTZXEuTGlzdCkucGRm" target="_blank" style="word-wrap:break-word;">2003-P-2007 (Petition for Seq.List).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260328-a-core-wireless-ingine-and-a-method-of-producing-a-wireless-modem-unit.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260330-dual-one-way-clutch-assembly.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260329</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2003/CHENP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>17/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-May-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CRUCELL HOLLAND B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ARCHIMEDESWEG 4, NL-2333 CN LEIDEN, THE NETHERLANDS</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TER MEULEN, JAN, HENRIK</td>
											<td>LEIDSE KADE 59 NL-1016 CX AMSTERDAM THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN DEN BRINK, EDWARD, NORBERT</td>
											<td>BURGEMEESTER VAN BRUGGENSTRAAT 21 NL-1165 NS HALFWEG THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DE KRUIF, CORNELIS, ADRIAAN</td>
											<td>WESTERLAAN 40 NL-3731 EL DE BILT THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GOUDSMIT, JAAP</td>
											<td>KONINGINNEWEG 4 NL-1075 CX AMSTERDAM THE NETHERLANDS</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/42</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/55876</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-11-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>05102117.8</td>
									<td>2005-03-17</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/627,773</td>
									<td>2004-11-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>3</td>
									<td>04106192.0</td>
									<td>2004-11-30</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>4</td>
									<td>04105684.7</td>
									<td>2004-11-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>5</td>
									<td>05107288.2</td>
									<td>2005-08-08</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260329-compositions-against-sars-coronavirus-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:38:53 GMT -->
</html>
