<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260317-method-for-amidating-of-polypeptides-with-basic-aminoacid-c-terminals-by-means-of-specific-endoproteases by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:35:04 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260317:METHOD FOR AMIDATING OF POLYPEPTIDES WITH BASIC AMINOACID C-TERMINALS BY MEANS OF SPECIFIC ENDOPROTEASES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR AMIDATING OF POLYPEPTIDES WITH BASIC AMINOACID C-TERMINALS BY MEANS OF SPECIFIC ENDOPROTEASES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a method for producing C-terminal amidated dibasic or polybasic peptides, consisting in reacting two peptides in the presence of trypsin biologically active enzymes and, if necessary, in purifying the thus obtainable compounds of formula (I) by means of protein chemistry.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Method for-amidating polypeptides with basic aminoacid C-terminals by means of specific endoproteases<br>
The present invention relates to a process for preparing C-terminally amidated di- or polybasic peptides, in particular those having the biological activity of GLP-1 or analogs or derivatives thereof.<br>
The stability, biomedical availability and duration of action of pharmaceutically relevant peptides and proteins depends to a large extent on the nature of the N- or C-terminal end of the molecule. The half-life of biomolecules is influenced markedly by C-terminal extension with basic amino acids. Particularly good pharmaceutical activities can be achieved if on C-terminal extension with more than one basic amino acid the amino acid at the C-terminal end is an amino amide.<br>
Such peptide-based active substances can be prepared, if the peptides are sufficiently small, directly by a complete chemical synthesis according to a modified Merrifield synthesis protocol. However, limitations emerge when such a peptide is required in large quantities. Thus, the amino acids to be employed in the synthesis must initially be prepared and purified in order subsequently to be suitable after chemical modification as reactants in the peptide synthesis. After removal of the protective groups at the end of the synthesis, the target peptide or product can then be purified and formulated as pharmaceutical Depending on the composition of the peptide, moreover, neighborhood effects may result during the synthesis in restricted yields, racemization or by-product formation through faulty coupling in individual coupling steps, thus possibly having an adverse effect on the overall yield or the purity of the product. Total synthesis is too complicated for preparing larger quantities of required product. Preparation via alternative, especially biotechnological, processes would therefore be desirable.<br>
Enzymes capable of C-terminal amidation of peptides have been known for a long time. These enzymes are named (Eipper et al. Mol. Endocrinol. 1987 Nov; 1 (11): 777) as peptidylglycine alpha-amidating enzymes (PAM). The preparation and purification of such PAM enzymes is familiar to the skilled worker and has been described in detail (M. Nogudi et aL Prot. Expr. Purif. 2003, 28: 293). However, the<br><br>
preparation* is costly in relation to the preparation of other industrial enzymes such as, for example, trypsin or carboxypeptidase.<br>
An alternative to the "in vitro" amidation by means of PAM emerges when the enzyme is coexpressed in the same host cell with the precursor protein to be amidated. This is achieved by introducing a gene sequence which codes for a PAM activity into the host cell under the control of a host-specific regulatory sequence. This expression sequence can either be incorporated stably into the respective chromosomal DNA sequence, or be present on a second plasmid parallel to the expression plasmid for the target protein, or be integrated as second expression cassette on the same vector, or be cloned in a polycistronic expression approach in phase with the gene sequence which encodes the target protein under the control of the same promoter sequence. However, the described yields are low, so that the preparation of large quantities can be achieved only via corresponding fermentation volumes. This leads to a more costly complexity of purification. In addition, the amidation does not take place quantitatively, so that it is necessary to separate amidated from unamidated required protein. A decision in favor of a secretory preparation process {e.g. Hong et al., AppI Biochem Biotechnol. 2003;110, p.113-23) must take account of the fact that proteins with polybasic C terminus are secreted to only a very small extent or not at all<br>
A further method for amidation is based on the use of protein-specific self-cleavage mechanisms (Cottingham et al. Nature Biotech. Vol. 19, 974-977, 2001). However, it is not easy to control this reaction, and transthioesterifjcation and thus the formation of unwanted products may occur. The relatively large contribution of the fusion protein may have adverse effects on the yield.<br>
The amidation processes described above start from a C terminus of the target peptide which is extended by at least one amino acid glycine or alternatively intein peptide. Peptides with C-terminal lysine can, however, if they comprise no additional lysine or arginine subsequently in the sequence be prepared as a multimeric structure which can subsequently be converted by digestion with trypsin or trypsin-like enzymes into the monomeric unit. High yields can be achieved in this way. This is not possible in the case of the processes described above.<br><br>
The known biotechnological preparation processes are thus associated with disadvantages, and the object of preparing peptides or proteins which have more than one basic amino acid lysine or arginine at the C terminus and are terminally amidated in large quantities at reasonable cost cannot yet be regarded as satisfactorily solved.<br>
An alternative preparation process would emerge if it were possible to prepare in large quantities a precursor of the target peptide which is truncated by at least one basic amino acid, and subsequently to extend this precursor in an enzyme-catalyzed semisynthesis with lysinamide or argininamide.<br>
Levin et al. (Biochemical Journal 63: 308-16; 1956) describe the effect of trypsin on lysinamide and various polylysinamide peptides. The authors' results show that lysinamide derivatives cannot be converted under the influence of trypsin into high molecular weight lysinamide derivatives because hydrolysis of the intermediately formed amides to the free acid takes place rapidly by comparison with the coupling reaction. It must be concluded therefrom that trypsin-catalyzed semisynthetic processes providing the preparation of peptides with C-linked polylysinamide or poly-argininamide or poly-Lys/Arg mixed sequences are unsuccessful or successful only with low yields.<br>
A peptide chemical process which surprisingly permits trypsin-catalyzed ligation of amidated basic amino acids, analogs or derivatives thereof, to peptides which have a C-terminal basic amino acid with high yields (i.e. &gt;30%) has now been found.<br>
It has additionally been possible to observe for the process of the invention, surprisingly, that introduction of protective groups (ct Pitraschke et al.. Tetrahedron: Asymmetry 9, p 1505-1518, 1998) such as, for example, -Boc (t-butyloxycarbonyl), -Z (benzyloxycarbonyl) or -DDZ (dimethylphenylpropyloxycarbonyl) into the amidated basic amino acids (lysinamide, argininamide) does not lead to an improvement in respect of efficiency and selectivity of the ligation reaction. The process of the invention thus has the advantage that it is possible to dispense with masking with protective groups, thus avoiding losses of yield through removal of the protective<br><br>
group and the disposal of toxic reagents. This results in the process having enormous cost advantages over total chemical synthesis.<br>
One aspect of the invention is thus a process for preparing C-terminally amidated di-or polybasic peptides of the general formula I<br><br>
where<br>
(AA)n is a peptide consisting of n amino acids of the same or different type, in which<br>
AA      is an amino acid or analogs or derivatives thereof;<br>
n        is an integer between 3 and 2000; and<br>
X        is a basic amino acid or analogs or derivatives thereof;<br>
m       is an integer between 2 and 15,<br>
in which<br>
a compound of the general formula II<br><br>
is reacted with a compound of the general formula III<br><br>
in which AA, n and X are as defined above, and<br>
p and q	are integers, and<br>
p + q = m,<br>
in the presence of an enzyme having the biological activity of trypsin, and<br>
where appropriate the resulting compound of the formula I is subjected to protein<br>
chemical purification;<br>
in particular in which m is an integer between 2 and 10, or in which m is an integer<br>
between 2 and 6.<br>
A further aspect of the invention is a process described above in which n is an integer between 10 and 1000, or between 15 and 500, or between 20 and 400,<br>
A further aspect of the invention is a process described above in which<br>
a)	a fusion peptide which comprises one or more compounds of the formula II as part of the fusion peptide is expressed;<br>
b)	the compounds of the formula II are liberated from said fusion peptide by chemical or enzymatic cleavage;<br><br>
c)	' the in'termediate from step b) is, where appropriate after protein chemical<br>
purification, reacted in the presence of an enzyme having the biological activity of trypsin with a compound of the formula 111; and<br>
d)	where appropriate the resulting compound of the formula I is subjected to<br>
protein chemical purification and isolation;<br>
in which in particular the elimination of the compound of the formula II from the fusion protein takes place by means of cyanogen halide, enterokinase, factor Xa, Genenase, thrombin or trypsin; and further preferred the fusion protein is expressed in an expression system selected from the group comprising E, coli, S. carnosus, Salmonella, Bacillus subtilis, Pseudomonas fluorescens, K. lactis, P. pastoris, Schizosaccharomyces pombe and S. cerevisiae,<br>
A further aspect of the invention is a process described above in which said peptides of the general formula I have the biological activity of GLP-1 or derivatives or analogs thereof,<br>
A further aspect of the invention is a process described above in which the<br>
compound of the formula I is characterized by the formula IV:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPS(X)m-NH2       (IV);<br>
which in turn can in particular be characterized by the sequences:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKK-NH2 (SEQ ID<br>
No.1);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK^NH2 (SEQ ID No. 2);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKR^NH2 (SEQ ID No. 3);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKR-NH2 (SEQ ID<br>
No. 4) or<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSRK-NH2 (SEQ ID No. 5).<br>
The invention further relates to compounds of the formula I characterized by the<br>
sequences:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK-NH2 (SEQ ID No. 2);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKR-NH2 (SEQ ID No. 3);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKR-NH2 (SEQ ID<br>
No. 4) or<br><br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSRK-NH2 (SEQ ID No. 5); their use, especially in depot formulations.<br>
A further aspect of the invention is a medicament comprising one or more of the<br>
compounds of the formula I characterized by the sequences:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK-NH2 (SEQ ID No. 2);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKR-NH2 (SEQ ID No. 3);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKR-NH2 (SEQ ID<br>
No. 4) or<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSRK-NH2 (SEQ ID No. 5).<br>
The meanings for the purposes of the present invention are:<br>
The term "analog" of an amino acid means a naturally occurring amino acid which is not encoded in the genetic code but is suitable for incorporation into a peptide chain. Examples of analogs of an amino acid are ornithine and citrulline, and also all further non-naturally occurring amino acids which have any basic function in the side chain, such as, for example, 2,4-diaminobutanoic acid; 3-methylornithine; 4-methylornithine, 5-aminoleucine; 4-aminoleucine; 3-aminoleucine; 5-aminonorleucine; 4-amino-norleucine; 3-aminonorleucine; 4-aminonorvaline; 3-aminonorvaline; 6-methyllysine; 5-methyllysine; 4-methyllysine; 3-methyllysine.<br>
The term "derivative" of an amino acid means an amino acid or an analog of an amino acid which is substituted by one or more chemical groups. Examples of such chemical groups are protective groups customary in peptide chemistry, such as -Boc (t-butyloxycarbonyl), -Z (benzyloxycarbonyl), -DDZ (dimethylphenylpropyloxy-carbonyl), -Fmoc (N-alpha-(9-fluorenyimethyloxycarbonyl), -2-bromo-Z, -2-chloro-Z, -Tfa (trifluoroacetyl), -nicotinoyi, -4-nitro-Z, -2-picolinoyl, -Tos (4-toluenesulfonyl), -For(formyl), -biotinyl, -dansyl, -Dnp (dinitrophenyl), -Mca (monochloracetyl), -Mtt (N-methyltrityl), -Nde (N-1-(4-nitro-1,3-dioxoindan-2-ylidene)ethyl), -acetimidoyi, -acetyl, -myristoyi, -palmitoyi, N-lithocholyl-y-glutamyl or-ω-carboxyheptadecanoyl.<br>
Such a group is furthermore a group C(0)-(C6-C24)alkyL a group C(0HC6-C24)alkenyl, a group C(0)-(C6-C24)alkanedienyl, or a group C(0)-(C6-<br><br>
C24)aikanetnenyl, where alkyl, alkenyl, alkanedienyl and alkanetrienyl groups where present may be branched or straight-chain. (CrC6)Aikyl means a hydrocarbon radical having 1, 2, 3, 4, 5 or 6 C atoms. Examples of (C1-C6)alkyl radicals are methyl, ethyl, n-propyl, isopropyl, (1-methylethyl), n-butyl, isobutyl (2-methylpropyl), sec-butyl (1-methylpropyl), tert-butyl (1,1-dimethylethyl), n-pentyl, isopentyl, tert-pentyl, neopentyl, hexyl. (C6-C24)Alkyl correspondingly means a hydrocarbon radical having 6 to 24 C atoms. Alkyl radicals may be straight-chain or branched. Preferred (C6-C24)alkyl radicals are fatty acid residues, for example hexyl, octyl, decanyl, undecanyl, dodecanyl, tridecanyl, tetradecanyl (myristyl), pentadecanyl, hexadecanyl, heptadecanyl, octadecanyl (stearyl), nonadecanyl, eicosanyl, dicosanyl, 9,11-dimethyltridecanyl, 11-methyltridecanyl.<br>
Such a group is furthermore a group C(0)-pheny-"(C5-C8)heteroaryl-phenyl, 0(0)-biphenyl or C(0)-terphenyl, where the phenyl, biphenyl, terphenyl or heteroaryl groups are unsubstituted or are substituted by one or two groups selected from the group of (CrCio)alkyl or 0(CrCio)alkyl. In monosubstituted phenyl radicals, the substituent may be present in the 2 position, the 3 position or the 4 position. Disubstituted phenyl may be substituted in the 2,3 position, 2,4 position, 2,5 position, 2,6 position, 3,4 position or 3,5 position. In trisubstituted phenyl radicals, the substituents may be present in the 2,3,4 position, 2,3,5 position, 2,4,5 position, 2,4,6 position, 2,3,6 position or 3,4,5 position. Heteroaryl means for example furanyl, thienyl, pyrrolyl, imidazolyl, pyrazolyl, triazolyl, tetrazolyl, oxazolyl, isoxazolyl, thiazolyl, isothiazolyl, pyridyl, pyrazinyl, pyrimidinyl, pyridazinyl, indolyl, indazolyl, quinolyl, isoquinolyl, phthalazinyl, quinoxalinyl, quinazolinyl and cinnolinyl<br>
The term "basic amino acid" means lysine, arginine or derivatives or analogs thereof, such as, for example, ornithine or citrulline, preferably lysine or arginine, especially lysine. In this connection, lysine or arginine may, instead of the carboxamide group, be derivatized by one or more of another reactive group, in particular selected from the group of amino ester, peptide ester, anhydride and halide, and be correspondingly employed in the process of the invention.<br>
The term "enzyme having the biological activity of trypsin" means, besides the known and commercially available trypsins from the conventional sources such as rat,<br><br>
cattle, pig, human, dog, mouse, the isoenzymes, derivatives or variants thereof, also enzymes having highly related biochemical properties such as, for example, cathepsin, trypsin from Fusarium oxysporum and from Streptomyces (S. griseus, S. exfoliatus, S. erythraeus, S. fradiae and S. albidoflavus), tryptase, mastin, acrosin, kaliikrein, hepsin, prostasin I, lysyl endopeptidase (Lys-C) and endoproteinase-Arg-C (clostripain).<br>
It is clear to the skilled person in this connection that this list is not definitive, and further enzymes, isoenzymes, derivatives or variants able specifically to cleave C-terminally to basic amino acids are continually being discovered during biotechnological research. Alternatively, the specificity of enzymes can be altered by peptide chemical modification or mutation at the DNA level (muteins). In addition, the specificity and activity of enzymes can be markedly modified by suitable choice of the reaction conditions.<br>
In the preferred embodiment of the process of the invention, the ability of microorganisms to prepare heterologous peptides is utilized. For this purpose, the desired peptide sequence is translated into the corresponding DNA sequence, which is coupled to a host-specific promoter sequence. It is possible in this case, depending on the expression strategy, to express the target peptide so that it is formed by the cell directly or indirectly as fusion peptide remaining inside the cells.<br>
If a fusion strategy using a suitable fusion peptide is chosen, it is then clear to the skilled worker that the fusion partners must be connected together by a linker, that the specific splitting up of the partners in a way allows so that the desired N terminus of the target peptide is available after processing. For the purposes of the present invention, the target peptide is a compound of the formula IK A large number of possibilities, which are known to the skilled worker and are continually expanded, is available for the design of suitable linkers. If, for example, the amino acid methionine is chosen, a chemical cleavage with cyanogen halide is possible. If, for example, a pentapeptide of the sequence DDDDK is chosen as linker, a cleavage with enterokinase is possible. If, for example, the tetrapeptide sequence lEGR is chosen, the cleavage can take place with factor Xa. With an appropriate design, Genenase® can be used as processing enzyme for peptides whose N terminus start with histjdine. If the N terminus is characterized by the dipeptide Gly-Ser, it is possible to<br><br>
choose the tetrapeptide LVPR as linker, so that a recognition and cleavage site for thrombin results. Although an enzyme having the biological activity of trypsin is employed according to the invention for the peptide coupling, the use of such an enzyme is possible in principle even at this point in the process of the invention. Thus, trypsin can initially be used in aqueous medium for eliminating the fusion part of the fusion peptide as long as lysine or arginine is inserted as linker between fusion part and target peptide.<br>
However, it is also possible alternatively for the target peptide, if it is export-compatible, to be secreted into the cell culture medium either in the form of a fusion peptide or in its native form. It is possible to use for this purpose recombinant host cells, especially those of microorganisms, preferably of bacteria or yeasts. If bacterial cells are chosen as expression system, there is the additional option of the target peptide or a corresponding fusion peptide which includes the target peptide, i.e. a compound of the formula II for the purposes of the invention, being directly secreted into the periplasm or into the culture medium. It is clear to the skilled person in this connection that the exportation is frequently restricted if the C-terminal end consists of more than one basic amino acid.<br>
The host organisms and methods available in principle for this purpose are known to the skilled worker (cf., for example, Gellissen, Gerd (ed.) Production of Recombinant Proteins, ISBN 3-527-31036-3), They are also to a large extent commercially available from a large number of suppliers. Representatives which may be mentioned are the companies New England Biolabs, Invitrogen and Roche. The catalog descriptions of such companies include references to literature providing an overview of the technology. It is also clear to the skilled worker in this connection that the range of microorganisms to be used is continually expanding, as is the repertoire of the biotechnological methods. Embodiments which are more specific in this respect are also included in the subject matter of the present invention. Examples of host/vector systems mentioned as representative are the following; bacteria of the type of E. coli, S.carnosus, Salmonella, Bacillus subtilis or Pseudomonas, especially Pseudomonas fluorescens, and yeasts of the type of K. lactis, P. pastohs, Schizosaccharomyces pombe and S. cerevisiae.<br><br>
However, the skilled worker is aware that these systems mentioned as examples offer a large number of possible variations which emerge for example from the choice of suitable promoters or other regulatory nucleic acid sequences, the genetic properties of the host cell and of the vectors used (e.g. in relation to the DNA copy number, the choice of the selection means etc.).<br>
The skilled worker is likewise aware that a specific purification process must be adapted for each target peptide, because of its physicochemical properties, when it is intended to be provided in isolated form. This is achieved in principle through suitable combination of known biochemical and biophysical separation methods. It is likewise clear in this connection that new possibilities are continually being opened up for achieving or optimizing the desired successful purification on the basis of novel materials (e.g. for the chromatography).<br>
It may be advantageous for the purposes of the present invention to fuse the precursor peptide with a peptide sequence which, for example, permits purification by affinity chromatography.<br>
For further development of the invention, exendin derivatives as disclosed in US 2004/0106547 have been prepared by way of example. Such peptides derived from exendin may, because of their blood glucose-lowering effect, play an important part in medicament development in the treatment of diabetes or other metabolic disorders which lead for example to obesity. It is therefore worthwhile for pharmaceutical purposes to have such peptides available in appropriate form.<br>
The international patent application WO 02/066628 describes a hirudin derivative which has the amino acids Lys-Arg at the C-terminal end. The antithrombotic effect profile can be modified by amidating the C-terminal arginine. For this purpose, the precursor with C-terminal lysine is prepared and subsequently a coupling reaction with argininamide is carried out according to the invention to result in the amidated hirudin derivative. The precursors can be prepared as described in the application via yeast secretion. For this purpose, for example, the gene sequence for Leu-hirudin described in EP-A 0 324 712 is extended by a codon for lysine, and the procedure described by way of example in the patent is continued to prepare the hirudin<br><br>
extended with lysine. Alternatively, it is also possible to follow the route of secretion of the precursor by bacteria. The technology described in the patent application EP-A 1 216 259 can be used for this purpose, for example.<br>
The following examples serve to illustrate the present invention and are by no means to be interpreted as limiting in relation to the subject matters of the present invention.<br>
Example 1: Synthesis of an E. coli-specific DNA sequence coding for AVE (1-43)<br>
Firstly the gene sequence SEQ ID No. 6 coding for the peptide AVE (1-43) (SEQ ID No. 7) was prepared:<br>
SEQ ID No. 6:<br>
TTTTTTAAGCTTGCACGGTGAAGGTACCTTCACCTCCGACCTGTCCAAACAGAT GGAAGAAGAAGCTGTTCGTCTGTTCATCGAATGGCTGAAAAACGGTGGTCCGT CCTCCGGTGCTCCGCCTTCGAAAAAGAAGAAAAAGTGATAATAGCATGCACGTG CGGCCGGACCTGGTCGACGAATTCAAA AAAA<br>
SEQ ID No. 7:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKK<br>
The gene sequence was synthesized by means of PCR technology. For this purpose, the following 5 primers were prepared by chemical DNA synthesis. This synthesis took place using the Expedite™ DNA synthesis system (from Applied Biosystems).<br>
a)	Primer zp5u has the sequence (SEQ ID No. 8):<br>
5'-TTTTTTAAGCTTGCACGGTGAAG -3'<br>
SEQ ID No. 8 includes the 1-23 region of the sense strand ("sense'*). The CAC triplet encodes histidine as first amino acid of the target peptide.<br>
b)	Primer zp3a has the sequence (SEQ ID No. 9):<br>
5'-CTTCCATCTGTTTGGACAGGTCGGAGGTGAAGGTACCTTCACCGTG<br>
CAAG CTTAAAAAA-3'<br><br>
SEQ ID No. 9 includes the 1-59 region of the complementary strain ("antisense").<br>
c)	Primer zp3b has the sequence (SEQ ID No. 10):<br>
5'-GGACGGACCACCGTTTTTCAGCCATTCGATGAACAGACGAACAGCT<br>
TCTTCTTCCATCTGTTTGGACAG-3'<br>
SEQ ID No. 10 includes the 40-108 region of the complementary strand ("antisense").<br>
d)	Primer zp3c has the sequence (SEQ ID No. 11);<br>
5-CGTGGATGCTATTATCACTTTTTCTTCTTTTTCGAAGGCGGAGCACC<br>
GGAGGAGGGACCACCGTTTTTC--3'<br>
SEQ ID No. 11 includes the 91-159 region of the complementary strand ("antisense").<br>
The antisense triplet CTT encodes the last amino acid (AA43) of the target peptide.<br>
e)	Primer zp3d has the sequence (SEQ ID No. 12):<br>
5'-TTTTTTGAATTCGTCGACCAGGTGCGGCCGCACGTGCATGCTATTA<br>
TCACTT-3'<br>
SEQ ID No. 12 includes the remaining region of the complementary strand ("antisense").<br>
Using the primers, 4 PCR reactions were subsequently carried out under standard conditions at 54°C. In reaction 1,100 ng of each of the primers zp3a and zp5u were employed. The number of cycles in the PCR was 5. In the second reaction, 1/40 of the reaction was reacted with 100 ng of each of the primers zp5u and zp3b in 10 cycles. In reaction 3,1/40 of the product of reaction 2 were reacted with 100 ng of      each of the primers zp5u and zp3c in a further 10 cycles. Finally, the desired DNA fragment was synthesized in 25 PCR cycles with 1/40 of the yield from reaction 3 and the primers zp5u and zp3d, and its length was checked by gel electrophoresis. The desired DNA fragment was purified and reacted with the restriction enzymes<br><br>
EcoR1 and subsequently with Hind3 in accordance with the manufacturer's (New England Biolabs) information.<br>
In parallel, DNA of the plasmid pUC19 (New England Biolabs) was reacted with the enzymes EcoR1 and Hind3. The fragments of the cleavage mixtures were separated on a 1.2% agarose gel and then the remaining vector fragment from pUC19 and the desired product from reaction 4 were isolated. The purified fragments were ligated together in a T4 ligase reaction at 16°C overnight. Subsequently, competent E. coli cells (Stratagene, strain E. coli XL10 Gold) were transformed with the ligation mixture and plated out on agar plates comprising 25 mg/l ampicillin. Plasmid DNA was isolated from the individual clones and characterized by DNA sequence analysis.<br>
The plasmid DNA of the desired fragment received the designation pSCHPUCZPI-43 and served as starting material for preparing expression vectors for synthesizing the compounds of the formula I in E.coli K12 cells.<br>
Example 2: Construction of an expression vector for AVE (1-43)<br>
US 5496924, the contents of which are hereby expressly included in the present application by reference, proposes an expression system which in principle permits the preparation of tailored fusion proteins. The advantage of the system is that fusion proteins with a small ballast portion can be prepared. The expression system is used by way of example in the application. Fusion of the sequence segments A-B via the enterokinase recognition sequence DDDDK with AVE (1-43) results in a fusion protein having the following gene sequence and amino acid sequence (SEQ ID No. 13 and No. 14):<br>
SEQ ID No. 13:<br>
5'-GGAAACAGAATTCATGGCGCCGACCTCTTCTTCTACCAAAAAGCTCAACTG<br>
CAACTGGAACACCTGCTGCTGGACCTGCAGATGATCCTGAACGGTATCAACAAC<br>
TACAAAAACGCGAAACTGACGCGTATCGACGATGACGATAAACACGGTGAAGGT<br>
ACCTTCACCTCCGACCTGTCCAAACAGATGGAAGAAGAAGCTGTTCGTCTGTTC<br>
ATCGAATGGCTGAAAAACGGTGGTCCGTCCTCCGGTGCTCCGCCTTCGAAAAA<br>
GAAGAAAAAGTGATAATAGCATGCACGTGCGGCCGCAAGCTTAAAAAA-3^<br><br>
The ATG and AAG codons mark the first and last amino acid of the fusion peptide.<br>
SEQIDNo. 14:<br>
MAPTSSSTKK    TQLQLEHLLL     DLQMILNGIN     NYKNPKLTRI     DDDDKHGEGT<br>
FTSDLSKQME EEAVRLFIEW LKNGGPSSGA PPSKKKKK<br>
The encoded gene sequence was prepared by means of PCR technology. For this purpose, the following primers were synthesized:<br>
1)	Primer psw3_zpcolf (SEQ ID No. 15);<br>
5'-CGTATCGACGATGACGATAAACACGGTGAAGGTACCTTC-3'<br>
The sequence of the primer in this case covers the enterokinase recognition site and the start of the AVE1-43 -encoding sequence.<br>
2)	Primer psw3_zpcolrev (SEQ ID No. 16):<br>
5'-GTGTTTATCGTCATCGTCGATACGCGTCAGTTTCGG-3'<br>
The sequence in this case corresponds to a synthesic sequence derived from interleukin2 which, as shown in Table I of US 5496924, covers amino acids 34-38 and 2/3 of the codon for the amino acid methionine. The remainder of the primer sequence overiaps with primer psw3_zpcolf.<br>
3)pBprimef1 (SEQ ID No. 17): 5'-TGAGCGGATAACAATTTCACAC-3'<br>
The primer hybridizes upstream with the EcoRI cleavage site which is present in the plasmid pK50 (cf. Fig. 33 in US 5496924).<br>
4) psw3_ave_1"43_rev with HindS cleavage site (SEQ ID No. 18): 5'"TTTTTTAAGCTTGCGGCCGCACGTGCATGCTATTATCACTT<br>
Two PCRs were carried out in parallel. One was carried out on DNA of the plasmid pK50 with the primer pair pBprimefl and psw3_zpcolrev at 50°C and the other<br><br>
reaction was carried out with the primer pair psw3_zpcolf and psw3_ave_1-43_rev at 54'C on DNA of the plasmid pSCHPUCZP1-43, The PCR yields were purified after fractionation by gel electrophoresis, and an aliquot of each were mixed in the ratio 1:1 and then reacted in a third PCR with the primer pair pBprimefl and psw3_ave_1-43_rev. The PCR yield was reacted with the enzymes EcoRI and Hind3 and employed in a T4 ligase reaction into the plasmid pK50 which was opened in parallel with these enzymes. Competent E. coli BL21 cells are transformed with the ligation mixture and plated out on selective agar comprising 25 mg/1 ampicillin. Plasmid DNA was reisolated from some clones and analyzed by PCR and subsequent DNA sequence analysis. Correct plasmids receive the name pBZP43. E. coli BL21:pBZP43 clones are checked for expression of the fusion protein. This takes place in a manner analogous to example 14 of US patent 5496924. The expresssion products were analyzed by mass spectrometry and by SDS-PAGE, and the N terminus was determined by protein sequence analysis. A suitable clone for fermentation of larger quantities of material was selected.<br>
Example 3: Construction of an expression vector for AVE (1-39)<br>
Plasmid pBZP43 serves as template for the PCR reaction carried out with the primers pBprimefl (Ex. 2) and psw3_ave_39rev. The PCR product is reacted with restriction enzymes EcoRl and NotI in accordance with the information from the enzyme manufacturer, and inserted in a T4 ligase reaction into the plasmid pBZP43 opened with Ecori/Notl The result is the plasmid pBZP39, with which the procedure described in Example 2 is continued.<br>
psw3_ave_39rev (SEQ ID No. 19): 5'^TTTTTTGCGGCCGCACGTGCATGCTATTATCATTTCGAAGGCGGAGCACC-3'<br>
The TTT triplet encodes lysine in position 39.<br>
Example 4: Construction of an expression vector for AVE (1-38-Arg)<br>
Plasmid pBZP43 serves as template for the PCR reaction carried out with the primers pBprimefl (Ex. 2) and psw3_ave38_argrev. The PCR product is reacted with<br><br>
restriction enzymes EcoRI and NotI in accordance with the information from the enzyme manufacturer, and inserted in a T4 ligase reaction into the plasmid pBZP43 opened with EcoRI/Notl. The result is the plasmid pBZP38arg, with which the procedure described in Example 2 is continued.<br>
Primer: psw3_ave38_argrev (SEQ ID No. 20);<br>
5'-TTTTTTGCGGCCGCACGTGCATGCTATTATCATACGCGAAGGCGGAGC<br>
ACCG-3'<br>
The AGG tnplet encodes arginine in position 39.<br>
Example 5: Fermentation of the strains constructed in Example 2-4<br>
The fermentation took place as described in the German patent application DE 10 2004 058306.4, Ex. 3, with slight differences. E. coli BL21 cells transformed with various plasmid vectors coding for target peptide derivatives (fusion protein) were cultivated in mineral salt medium or complex medium (see Example 1) at 30°C or 37°C and pH of 7.0 in a fermentor. The pH was adjusted with an NH/ solution (26% in water). The aeration of the culture was ensured by a control strategy which kept the dissolved oxygen in the culture broth constant at 30%. For fed batch processes in mineral salt medium, a glucose solution (60% w/v) was fed (8 g/L/h to 26g/L/h). Protein expression was induced by adding IPTG (1-4 mM final conc-. (f.c.)). The duration of induction was 6-8 h. Expression of the target proteins was detected by SDS polyacrylamide gel electrophoresis (SDS-PAGE).<br>
Expression of AVE precursor fusion protein in E. coli BL21/pBZP43 was carried out as described below:<br>
100µL of cell suspension were removed from the continuous culture of E. coli BL21 cells stored at-80°C and incubated in 0.5 L of preculture medium with shaking at 37°C for 10-16 h. The main culture in the fermentor was inoculated with an appropriate quantity of preculture for an inoculation density of from 0.01 to 0.05 OD600.<br><br>
Preculture medium: 5 g/L Bacto tryptone 10 g/L yeast extract 5 g/L NaCI<br>
Main culture medium:<br>
Defined mineral salt medium (minimal medium) based of glucose as carbon source<br>
(Jeffrey H. Miller: Experiments in Molecular Genetics, Cold Spring Harbor Laboratory<br>
(1972)).<br>
After the glucose initially present in the main culture medium had been consumed, a glucose solution was fed in. Protein expression was induced by adding IPTG (1 mM f.c), and the maximum expression of the fusion protein after induction was observed. The SDS-PAGE analytical system from Novex (NuPage® Novex 12% gel system, Invitrogen™) was used for example in accordance with manufacturer's information to analyze in the fermentation in each case 0.02 ODeoonm of cell suspension removed from the fermentor at various cultivation times.<br>
Example 6: Purification of the fusion proteins prepared in Example 5<br>
1000 g of biomass of recombinant E. coli strain were resuspended in 1000 ml of Tris buffer (50 mM Tris/HCI, pH 7.4). The cells were disrupted by high-pressure homogenization (Rannie high-pressure homogenizer, 1000 bar) twice. The genomic DNA was digested by adding Benzonase (1000 U/L) and magnesium chloride (10 mM) for 1.5 hours. The fusion protein was purified by expanded bed chromatography. For this purpose, the cell homogenate was diluted to 10 liters with buffer (50 mM Tris/HCI, pH 7.4) and directly loaded onto a chromatography column (Streamline SP XL, GE Healthcare) previously equilibrated with buffer (50 mM Tris/HCI pH 7.4). Sample loading was followed by a washing step with equilibration buffer (6 column volumes), followed by a further washing step with 7% high-salt buffer (50 mM Tris/HCI pH 7.4; 1 M NaCI). Elution took place by washing with 10 column volumes of 20% high-salt buffer. The elution pool was checked by SDS gel electrophoresis (NuPage® Novex 12% gel system, Invitrogen) and HPLC. The fusion protein pool was employed for the protease cleavage reaction after diafiltration into<br><br>
enterokinase buffer (50 mM Tris/HCI pH 7.4; 50 mM NaCI, 2mM CaCl2). The fusion proteins were cleaved by enterokinase (Invitrogen) in enterokinase buffer (20 mM Tris/HCI, 50 mM NaCI, 2 mM CaCl2 pH 7.4) in accordance with the manufacturer's information.<br>
Example 7: Separation of the cleavage products from the enterokinase cleavage reaction<br>
Separation of the cleavage products takes place in accordance with Example 6 of German patent application DE 102004058306.4, Cleavage of the fusion proteins by enterokinase was followed by separation of the cleavage products from one another by ion exchange chromatography (Source 303, Amersham Biosciences), The ionic strength of the solution was brought to about 7 mS/cm by dilution with H2O. After the protein solution had been loaded onto the previously equilibrated column (20 mM Tris/HCI, pH 7.4; adjusted to a conductivity of about 7 mS/cm with NaCI), unbound material was washed out with 15% buffer B (20 mM Tris/HCI, pH 7.4; 500 mM NaCI), Elution of the AVE peptide precursors took place by applying a gradient over 10 column volumes to 100% buffer B. AVE precursor-containing fractions were identified by SDS gel electrophoresis, HPLC and mass spectrometry. The corresponding fractions were combined, desalted and, after removal of organic solvent, lyophilized.<br>
Example 8: Peptide coupling of AVE (1-39) with H-Lys(Boc)-NH2<br>
0.2 mg of AVE (1-39) (MW 4218; 0.047 /ymol; 1 g/L final concentration) are weighed into a 1.5 mL polypropylene reaction vessel. 11µL of 0.1 M pyridine-acetate buffer of pH 5.6, 60 µL of a 129 g/L solution of H-Lys(Boc)-NH2.HCI in 0.1 M pyridine-acetate buffer of pH 5.6 (contains 7.75 mg of H-Lys(Boc)-NH2-HCi = 27.5µmol = 585 mol of H-Lys(Boc)-NH2.HCl per mole of AVE (1-39)) and 119 µL of DMF are added. The dear solution is equilibrated at 12°C. The reaction is started by adding 10 µL of a 2 g/L trypsin solution in water (contains 0.02 mg of trypsin = 0.002 g of trypsin per g of AVE (1-39)). The reaction solution is incubated at 12°C while shaking at 1000 min-1 Samples for process control are taken regularly and stopped by diluting with 9 vol of a solution of 17% water, 17% acetonitrile and 64% trifluoroacetic acid. The progress of the reaction is followed by LC-MS. After the maximum yield is<br><br>
reached, the reaction is acidified to a pH below 2.5 by adding trifluoroacetic acid and is punfied by chromatography.<br>
Example 9; Peptide coupling of AVE (1-43) with H-LyS-NH2<br>
3.85 mg of AVE (1-43) (MW 4731; 0.814µmol; 20 g/L final concentration) are weighed into a 1.5 ml polypropylene reaction vessel. 41µL of a 620 g/L solution of H-LyS-NH2.2HCI in 0.1 M sodium acetate buffer of pH 5.8 (contains 25.6 mg of H-Lys-NH2.2HCI = 117.5µmol = 144 mol of H-Lys-NH2.2HCI per mole of AVE (1-43)), 116µL of DMF and 32 /vL of 0.1 M sodium acetate buffer of pH 5.8 are added. The clear solution is equilibrated at 12°C. The reaction is started by adding 2.9µL of a 20 g/L trypsin solution in water (contains 0.06 mg of trypsin = 0.015 g of trypsin per g of AVE (1-43)). The reaction solution is incubated at 12°C while shaking at 900 min"\ Samples for process control are taken regularly and stopped by dilution with 9 vol of a solution of 17% water, 17% acetonitrile and 64% trifluoroacetic acid. The progress of the reaction is followed by LC-MS. After the maximum yield is reached, the reaction is acidified to a pH below 2.5 by adding trifluoroacetic acid, and is purified by chromatography.<br>
Example 10: Peptide coupling of AVE (1-39) with H-Lys-NH2<br>
0.2 mg of AVE (1-39) (MW 4218; 0.047µmol; 1 g/L final concentration) are weighed into a 1.5 mL polypropylene reaction vessel, 11 µL of 0.1 M pyridine-acetate buffer of pH 5.6, 60 µL of a 100 g/L solution of H-Lys-NH2.2HCI in 0.1 M pyridine-acetate buffer of pH 5.6 (contains 6.0 mg of H-Lys-NH2.2HCI = 27.5 µmo\ = 585 mol of H-Lys-NH2.2HCI per mole of AVE (1-39)) and 119µL of DMF are added. The clear solution is equilibrated at 12°C. The reaction is started by adding 10µL of a 2 g/L trypsin solution in water (contains 0.02 mg of trypsin = 0.002 g of trypsin per g of AVE (1-39)). The reaction solution is incubated at 12°C while shaking at 1000 min-1' Samples for process control are taken regularly and stopped by diluting with 9 vol of a solution of 17% water, 17% acetonitrile and 64% trifluoroacetic acid. The progress of the reaction is followed by LC-MS. After the maximum yield is reached, the reaction is acidified to a pH below 2.5 by adding trifluoroacetic acid and is purified by chromatography.<br><br>
Example 11: Peptide coupling of AVE (1-39) with H-Arg-NH2<br>
0.2 mg of AVE (1-39) (MW 4218; 0.047µmol; 1 g/L final concentration) are weighed into a 1.5 mL polypropylene reaction vessel. 11 µL of 0.1 M pyridine-acetate buffer of pH 5.6, 60 µL of 113 g/L solution of H-Arg-NH2.2HCI in 0.1 M pyridine-acetate buffer of pH 5.6 (contains 6.77 mg of H-Arg-NH2.2HCI = 27.5 µmol = 585 mol of H-Arg-NH2.2HCI per mole of AVE (1-39)) and 119µL of DMF are added. The clear solution is equilibrated at 12°C. The reaction is started by adding 10µ/L of a 2 g/L trypsin solution in water (contains 0.02 mg of trypsin = 0.002 g of trypsin per g of AVE (1-39)). The reaction solution is incubated at 12°C while shaking at 1000 min"\ Samples for process control are taken regularly and stopped by diluting with 9 vol of a solution of 17% water, 17% acetonitrile and 64% trifluoroacetic acid. The progress of the reaction is followed by LC-MS. After the maximum yield is reached, the reaction is acidified to a pH below 2.5 by adding trifluoroacetic acid and is purified by chromatography.<br>
Example 12: Peptide coupling of AVE (1-43) with H-Arg-NH2<br>
0,25 mg of AVE (1-43) (MW 4731; 0.047 /vmol; 1.1 g/L final concentration) are weighed into a 1.5 mL polypropylene reaction vessel, 11 µL of 0.1 M pyridine-acetate buffer of pH 5.6, 60 µL of a 113 g/L solution of H-Arg-NH2.2HCI in 0.1 M pyridine-acetate buffer of pH 5.6 (contains 6.77 mg of H-Arg-NH2.2HCI = 27.5µmol = 585 mol of H-Arg-NH2.2HCI per mole of AVE (1-43)) and 119 µL of DMF are added. The clear solution is equilibrated at 12°C. The reaction is started by adding 10 µL of a 2 g/L trypsin solution in water (contains 0.02 mg of trypsin = 0.002 g of trypsin per g of AVE (1-43)). The reaction solution is incubated at 12°C while shaking at 1000 min"\ Samples for process control are taken regularly and stopped by diluting with 9 vol of a solution of 17% water, 17% acetonitrile and 64% trifluoroacetic acid. The progress of the reaction is followed by LC-MS. After the maximum yield is reached, the reaction is acidified to a pH below 2.5 by adding trifluoroacetic acid and is purified by chromatography.<br><br>
Example 13': Peptide coupling of AVE (1-38|-Arg with H-Lys-NH2<br>
0.2 mg of AVE (1-38)-Arg (MW 4246; 0.047 µmol; 1 g/L final concentration) are weighed into a 1.5 mL polypropylene reaction vessel. 11 µL of 0.1 M pyridine-acetate buffer of pH 5.6, 60 µL of a 100 g/L solution of H-Lys-NH2-2HCI in 0.1 M pyridine-acetate buffer of pH 5.6 (contains 6,0 mg of H-Lys-NH2.2HCI = 27.5/ymol = 585 mol of H-Lys.NH2.2HCI per mole of AVE (1 -38)-Arg) and 119 /JL of DMF are added. The clear solution is equilibrated at 12°C. The reaction is started by adding 10 µL of a 2 g/L trypsin solution in water (contains 0.02 mg of trypsin = 0.002 g of trypsin per g of AVE (1-38)-Arg). The reaction solution is incubated at 12X while shaking at 1000 min-1 Samples for process control are taken regularly and stopped by diluting with 9 vol of a solution of 17% water, 17% acetonitrile and 64% trifluoroacetic acid. The progress of the reaction is followed by LC-MS. After the maximum yield is reached, the reaction is acidified to a pH below 2.5 by adding trifluoroacetic acid and is purified by chromatography.<br>
Example 14: Peptide coupling of AVE (1-43) with H-Lys(Boc)-NH2<br>
20 mg of AVE (1-43) (MW 4731; 1.058 /vmol; 20 g/L final concentration) are weighed into a 2 mL polypropylene reaction vessel. 370 µL of a 482 g/L solution of H-Lys(Boc)-NH2.HCl in 0.1 M sodium citrate buffer of pH 5.5 (contains 155 mg of H-Lys(Boc)"NH2.HCI = 550µmol = 130 mol H-Lys(Boc)-NH2.HCI per mole of AVE (1-43)) and 600 µL of DMF are added. The clear solution is equilibrated at 12'^C. The reaction is started by adding 30 µL of a 3.3 g/L trypsin solution in water (contains 0.1 mg of trypsin = 5 mg of trypsin per g of AVE (1-43)). The reaction solution is incubated at 12°C while shaking at 1000 min-1 Samples for process control are taken regularly and stopped by dilution with 9 vol of a solution of 17% water, 17%, acetonitrile and 64%) trifluoroacetic acid. The progress of the reaction is followed by LC-MS, After the maximum yield is reached, the reaction is acidified by adding trifluoroacetic acid to a final concentration of 50% (v/v). This stops the reaction and at the same time quantitatively removes the Boc protective group. The reaction product is purified by chromatography.<br><br>
Example 15: Purification of the amidated AVE derivatives<br>
The reaction mixture from the coupling reactions is subsequently separated by RP chromatography using Amberchrom CG300 XT 20 as support material, and the amidated target peptide is subsequently isolated from the appropriate eluate fraction by an ion exchange step with Source SOS as support. After desalting on Amberchrom columns, the product is available for formulation as medicament. The identity of the structure of the product was demonstrated by MALDI-MS and NMR analysis.<br>
Example 16: Preparation of Leu-hirudini.65 Lys-Arg-NH2<br>
Example 1 of the patent application EP-A 1 216 259 describes the preparation of an expression plasmid which permits the secretion of Leu-hirudin into bacterial supernatants. DNA of the plasmid is employed as template in a standard PCR. The forward pnmer used in the reaction is the sequence smompaf2 described in the example. The reverse primer used is an oligonucleotide hirjys66_rev (SEQ ID NO.: 21) which has the following sequence:<br>
5'- TTTTTIAAGC TTCTATTATT TCTGAAGGTA TTCCTCAGGG - 3'<br>
Hinds<br>
The codon underlined therein codes for lysine. The sequence from position 22 to position 40 (end) is complementary to the sequence segment 178-195 of the sequence depicted in Table 1 of the application EP-A 1 216 259. As described in the example, the PCR is carried out and the product is digested with the restriction enzymes EcoRI and Hind3 and inserted into the correspondingly opened vector pJF118. After characterization, DNA is transformed into E. coli K12 as in Example 11 of the patent application EP-A 1 216 259, and the intermediate product is expressed and purified. Then, corresponding to Example IS of the present application, corresponding to the molarities the reaction takes place with argininamide to give Leu-hirudini.65 LyS-Arg-NH2. If the expression is carried out directly in the strain MCI 061 used as intermediate host strain, product is found also in the periplasmic space. This makes necessary a cell disruption in accordance with known methods as additional workup step for isolating the intermediate product.<br><br>
Example 17; Analysis of peptide coupling reactions on AVE derivatives<br>
The coupling reactions according to Examples 8-14 are followed analytically by RP-HPLC on a Symmetry-300 150 x4.6 mm, 5µm. column from Waters. 0.1% (v/v) formic acid (eluent A) and acetonitrile with 0.1 % formic acid (eluent B) serves as eluents. A linear gradient from 20 to 50% B over 15 min at a column temperature of 60°C and a flow rate of 1 mL/min is used for elution. Detection takes place at 215 nm. Normally 5µL of a 1:25-diluted reaction sample are injected. The deprotected AVE derivatives are detected at retention times between 7 and 10 min.<br><br><br><br><br><br><br><br><br><br><br><br>
Claims:<br>
1.	A process for preparing C-terminally amidated di- or polybasic peptides of the<br>
general formula I<br><br>
where<br>
(AA)n is a peptide consisting of n amino acids of the same or different type, in which<br>
AA      is an amino acid or analogs or derivatives thereof;<br>
n        is an integer between 3 and 2000; and<br>
X        is a basic amino acid or analogs or derivatives thereof;<br>
m       is an integer between 2 and 15,<br>
in which<br>
a compound of the general formula II<br><br>
is reacted with a compound of the general formula III (X)q-NH2       (III)<br>
in which AA, n and X are as defined above, and<br>
p and q	are integers, and<br>
p + q = m,<br>
in the presence of an enzyme having the biological activity of trypsin, and<br>
where appropriate the resulting compound of the formula I is subjected to protein<br>
chemical purification.<br>
2.	The process as claimed in claim 1, in which m is an integer between 2 and 10.<br>
3.	The process as claimed in claim 2, in which m is an integer between 2 and 6.<br>
4.	The process as claimed in any of claims 1 to 3, in which n is an integer between 10 and 1000.<br>
5.	The process as claimed in any of claims 1 to 4, in which n is an integer between 15 and 500.<br><br>
6.	The process as claimed in any of claims 1 to 5, in which n is an integer between<br>
20 and 400.<br>
7.	The process as claimed in any of claims 1 to 6, in which<br>
a)	a fusion peptide which comprises one or more compounds of the formula II as part of the fusion peptide is expressed;<br>
b)	the compounds of the formula II are liberated from said fusion peptide by chemical or enzymatic cleavage;<br>
c)	the intermediate from step b) is, where appropriate after protein chemical purification, reacted in the presence of an enzyme having the biological activity of trypsin with a compound of the formula III; and<br>
d)	where appropriate the resulting compound of the formula I is subjected to protein chemical purification and isolation.<br><br>
8.	The process as claimed in claim 7, in which the elimination of the compound of the formula II from the fusion protein takes place by means of cyanogen halide, enterokinase, factor Xa, Genenase, thrombin or trypsin.<br>
9.	The process as claimed in claim 7, in which the fusion protein is expressed in an expression system selected from the group comprising E. coll, S. carnosus, Salmonella Bacillus subtilis, Pseudomonas fluorescens, K. lactis, P. pastoris, Schizosaccharomyces pombe and S. cerevisiae,<br>
10.	A process for preparing C-terminally amidated di- or polybasic peptides of the general formula I as claimed in one or more of claims 1 to 2, in which said peptides of the general formula I have the biological activity of GLP-1 or derivatives or analogs thereof.<br>
11.	The process as claimed in any of claims 1 to 10, in which the compound of the formula I is characterized by the formula IV:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPS(X)m-NH2       (IV).<br>
12       The process as claimed in any of claims 1 to 11, in which the compound of the formula I is characterized by the sequence:<br><br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKK-NH2 (SEQ ID<br>
N0.I);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK-NH2 (SEQ ID No. 2);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKR-NH2 (SEQ ID No. 3);<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKR-NH2 (SEQ ID<br>
No. 4) or<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSRK-NH2 (SEQ ID No. 5).<br>
13.	A compound of the formula I characterized by the sequence:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK-NH2 (SEQ ID No. 2); HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKR-NH2 (SEQ ID No. 3); HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKR-NH2 (SEQ ID No. 4) or HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSRK-NH2 (SEQ ID No. 5).<br>
14.	The use of a compound specified in claim 12 as medicament, especially in depot formulations.<br>
15.	A medicament comprising one or more of the compounds of the formula 1 characterized by the sequences:<br>
HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK-NH2 (SEQ ID No. 2); HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKR-NH2 (SEQ ID No. 3); HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSKK KKKR-NH2 (SEQ ID No. 4) or HGEGTFTSDL SKQMEEEAVR LFIEWLKNGG PSSGAPPSRK-NH2 (SEQ ID No. 5).<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMjQtMTItMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   AMENDED  PAGES OF SPECIFICATION  24-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCBDTEFJTVMgIDI0LTEyLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   AMENDED CLAIMS  24-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDEwLTA0LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   CORRESPONDENCE OTHERS 10-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAyNC0xMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED  24-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgRk9STS0zICAyNC0xMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   FORM-3  24-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyNC0xMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   OTHER PATENT DOCUMENT  24-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICAgT1RIRVJTICAyNC0xMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008   OTHERS  24-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICBBTUVOREVEIENMQUlNUyAgMTYtMDQtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008  AMENDED CLAIMS  16-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0UgT0YgU1BFQ0lGSUNBVElPTiAgMTYtMDQtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008  AMENDED PAGE OF SPECIFICATION  16-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSAgT1RIRVJTICAxNi0wNC0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008  CORRESPONDENCE  OTHERS  16-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1DSEVOUC0yMDA4IEZPUk0tMTggMjQtMDktMjAwOS5wZGY=" target="_blank" style="word-wrap:break-word;">1478-CHENP-2008 FORM-18 24-09-2009.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQ3OC1jaGVucC0yMDA4LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1478-chenp-2008-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260316-extramedullary-jig-for-derotation-screw-fixation-in-trochanteric-fracture.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260318-method-for-printing-a-bookmark-separator-page-between-different-print-jobs.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260317</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1478/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>17/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Apr-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Mar-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANOFI-AVENTIS DEUTSCHLAND GMBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BRUNINGSTRASSE 50 D-65929 FRANKFURT AM MAIN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SALAGNAD CHRISTOPHE</td>
											<td>27, RUE JEAN VILAR 86100 CHATELLERAULT</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ZOCHER, FRANK</td>
											<td>C/O SANOFI-AVENTIS DEUTSCHLAND GMBH 65926 FRANKFURT AM MAIN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LANDRIC-BURTIN, LAURE</td>
											<td>C/O SANOFI-AVENTIS DEPARTEMENT BREVETS 174, AVENUE DE FRANCE 75013 PARIS</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RISSOM, SEBASTIAN</td>
											<td>C/O SANOFI-AVENTIS DEUTSCHLAND GMBH 65926 FRANKFURT AM MAIN</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HABERMANN, PAUL</td>
											<td>C/O SANOFI-AVENTIS DEUTSCHLAND GMBH 65926 FRANKFURT AM MAIN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/575</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP06/08903</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-09-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>102005046113.1</td>
									<td>2005-09-27</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260317-method-for-amidating-of-polypeptides-with-basic-aminoacid-c-terminals-by-means-of-specific-endoproteases by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:35:05 GMT -->
</html>
