<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260382-modified-expandase-hydroxylase-and-its-applications by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:55:37 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260382:MODIFIED EXPANDASE-HYDROXYLASE AND ITS APPLICATIONS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MODIFIED EXPANDASE-HYDROXYLASE AND ITS APPLICATIONS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a mutant expandase-hydroxylase with increased activity and greater substrate specificity for Penicillin G and Phenyl acetyl 7-ADCA for the production of Phenyl acetyl - 7-ADCA and Deacetyl phenyl acetyl 7-ACA respectively; which carries one or more amino acid modification at residue positions when compared with the wild type expandase-hydroxylase from the following group of residues, Lysine at position 14, Serine at position 15, Threonine at position 20, Threonine at position 45, Glutamic acid at position 49, Lysine at position 56, Aspartic acid at position 70, Asparagine at position 72, Alanine at position 73, Valine at position 87, Lysine at position 93, Lysine at position 131, Tyrosine at position 185, Isoleucine at position 190, Tyrosine at position 203, Glutamic acid at position 212, Phenylalanine at position 226, Threonine at position 232, Lysine at position 247, Threonine at position 260, Lysine at position 269, Asparagine at position 275, Asparagine at position 285, Tryptophan at position 282, Isoleucine at position 288, Threonine at position 293, Arginine at position 296, Lysine at position 310 and Threonine at position 332.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Field of invention:<br>
The present invention provides modified expandase-hydroxylase from Cephalosporium acremonium (Acremonium chrysogenum) having better ring-expansion and hydroxylation activity and increased specificity towards substrates such as Penicillin G, Penicillin N, Phenyl acetyl 7-ADCA and deacetoxy cephalosporin C (DAOC). The modified expandase-hydroxylase from Cephalosporium acremonium is found useful in the bioprocess for the preparation of 7-ACA.<br>
Background of the invention:<br>
p-lactam antibiotics such as Penicillins and Cephalosporins are widely used for the treatment of a variety of infectious diseases. As Cephalosporins offer better protection than Penicillins particularly against resistant organisms, significant thrust has been given to derivatizations of Cephalosporins to broaden their spectrum and enhance their efficacy. 7-amino deacetoxy cephalosporanic acid (7-ADCA) and 7-amino cephalosporanic acid (7-ACA) serve as core intermediates for the synthesis of numerous semisynthetic cephalosporins such as Cephradine, Cephalexin, Cephadroxyl, Cefazolin, Cefotaxime, Ceftriaxone, Cefepime, Cefoperazone, and the like. Currently, 7- ACA is derived from Cephalosporin C by the cleavage of the 7-aminoadipyl side chain by either a chemical process or by a two stage enzymatic process (US 5,424,196) and 7-ADCA is derived from the hydrolysis of Phenyl acetyl 7-ADCA by Penicillin G amidase, wherein the Phenyl acetyl 7-ADCA is manufactured by conventional chemistry of ring expansion of Penicillin G. The industrial process of manufacture of these intermediates has become non-competitive, hazardous and generates considerable effluent treatment issues. Alternate technologies and processes, which offer cost-effective and environment friendly manufacture of these compounds, are needed.<br>
Native Penicillins and Cephalosporins are produced by a variety of bacteria and fungal  organisms  and  significant progress  has been  made  on understanding  their<br><br>
regulatory architecture (Aharonowitz, Y., et aL, Annu. Rev. Microbiol. 46: 461-495, 1992; Axel A. Brakhage, Microbiol. Mol. Biol. Rev. 62: 547-585, 1998). Three amino acids, L - lysine, L-ct-amino adipate and L-cysteine condense to form the tripeptide 8-(L-a-aminoadipoyl)-L-cysteinyl-D-Valine (ACV) which, then, gets cyclized to form isoPenicillin N followed by epimerization to form Penicillin N. The rate limiting or rather, the committed step in the biosynthesis of Cephalosporins is the expansion of the five-membered thiazolidine ring of Penicillin N to a six-membered dihydrothiazine ring of deacetoxy  Cephalosporin  C   (DAOC).      In  prokaryotes   such  as  Streptomyces<br>
clavuligerus, the ring-expansion step of Penicillin N is catalyzed by an enzyme called<br>
*<br>
deacetoxy cephalosporin C synthase (DAOCS or cefE) and hydroxylation of DAOC is carried out by deacetyl cephalosporin C synthase (DACS or cefF) or hydroxylase. In eukaryotic organisms such as Cephalosporium acremonium, both of these reactions are catalysed by a single bifunctional enzyme DAOCS-DACS (Scheidegger, A., et. al.9 Journal of Antibiotics, 37, 522-531, 1984) and also known as expandase-hydroxylase (ce/EF). Subsequently, DAC (Deacetyl cephalosporin C) gets acetylated to Cephalosporin C in Cephalosporium acremonium, while further deviation occurs resulting in Cephamycin C in bacteria such as Streptomyces clavuligerus.<br>
The deacetoxy cephalosporin C synthase also known as expandase is encoded by a gene cefE; wherein ceJE gene from S. clavuligerus contains 936 nucleotides and 311 amino acids. The deacetyl cephalosporin C synthase from S. clavuligerus has 318 amino acids and is encoded by ce/F.<br>
The ceJEF gene that codes for 332 amino acids encodes the bifunctional expandase-hydroxylase in C. acremonium. Dotzlaf. J. E. et al.9 (Journal of Bacteriology, 169, 1611-1618, 1987) purified, characterized and studied the kinetics and stochiometry of this enzyme. Sequence similarity reveals that DAOCS-DACS nucleotide sequence of C. acremonium is 67% identical to that of DAOCS of S. clavuligerus and the amino acid<br><br>
sequences are 56.7% identical. Further, the DAOCS-DACS enzyme has 54% amino acid sequence identity with that of the hydroxylase enzyme of S. clavuligerus (Figure 1). Expandase, Expandase-hydroxylase and hydroxylase are iron (II) and a- ketoglutarate dependent oxygenases and they are part of a subfamily of the mononuclear ferrous enzymes.<br>
Penicillin N is the natural substrate for expandase and expandase-hydroxylase (DAOCS-DACS) and DAOC is the substrate for hydroxylase. However, altered substrate specificity has been detected for different substrates such as Penicillin G, Penicillin V, 6-ct- MethylPenicillin N and adipoyl-6-APA (Lloyd, M. D., et. al., Journal of Biological Chemistry, 279, 15420-15426, 2004). As a result, development of green process technologies for the manufacture of Cephalosporin intermediates, thus, narrowed down to these enzymes. On the other hand, these enzymes show poor capability to convert readily available substrates such as Penicillin G and Phenyl acetyl 7-ADCA and hence, engineering them for commercial applications is required. Publications such as US20030186354, WOOl/85951 and US 6,699,699B2 describe such manipulations for expandase and the bioprocess for 7-ADCA manufacture.<br>
An expandase-hydroxylase with enhanced ring expansion and hydroxylation can lead to deacetyl Phenyl acetyl 7-ADCA, which upon transfer of acetyl group by acetyl transferase followed by hydrolysis by Penicillin G amidase would offer cheaper alternative than the current process for the manufacture of 7-ACA. As described and demonstrated in the Journal of Biological Chemistry, 279, 15420-15426, 2004, a single amino acid substitution can significantly influence the activity of the bifunctional DAOC-DACS.<br>
Alternately, deacetyl phenyl acetyl 7-ACA can be modified by conventional chemistry to develop desirable intermediates readily.<br><br>
Taking into account the commercial importance of producing Cephalosporin intermediates, it is imperative to identify a modified mutant expandase-hydroxylase from C.acremonium having increased substrate specificity for substrates such as Penicillin G and Phenyl acetyl 7-ADCA, when compared with the wild-type expandase-hydroxylase. Hence we have focused on this and have succeeded in identifying a modified expandase-hydroxylase from C.acremonium.<br>
Description of the Figure<br>
Figure 1: Alignment of ceffi, cefF and cefEF<br>
Figure 2: Wild-type nucleotide and amino acid sequence of expandase-hydroxylase from C.acremonium is given in SEQ ID No: 1.<br>
Table 1: Activity data for enhanced ring-expansion<br>
Table 2: Activity data for enhanced hydroxylation<br>
Table 3: Activity data for enhanced ring-expansion and hydroxylation<br>
Objective of the invention:<br>
An object of this invention is to provide a modified mutant expandase-hydroxylase from C.acremonium with improved capability for ring expansion and hydroxylation than that occurs in the natural wild type expandase-hydroxylase.<br>
Another object of this invention is to provide a modified expandase-hydroxylase having increased ring-expansion on substrates like Penicillin G, increased hydroxylation activity on substrates like Phenyl acetyl 7-ADCA and improved expansion and hydroxylation activity on substrates such as Penicillin G and Penicillin N.<br><br>
Summary of the invention:<br>
Accordingly, the present invention provides a mutated expandase-hydroxylase from C.acremonium, which consists of amino acid substitution at one or more amino acid residues corresponding to the wild type expandase-hydroxylase for the following group of residues consisting of Lysine at position 14, Serine at position 15, Threonine at position 20, Threonine at position 45, Glutamic acid at position 49, Lysine at position 56, Aspartic acid at position 70, Asparagine at position 72, Alanine at position 73, Valine at position 87, Lysine at position 93, Lysine at position 131, Tyrosine at position 185, Isoleucine at position 190, Tyrosine at position 203, Glutamic acid at position 212, Phenylalanine at position 226, Threonine at position 232, Lysine at position 247, Threonine at position 260, Lysine at position 269, Asparagine at position 275, Asparagine at position 285, Tryptophan at position 282, Isoleucine at position 288, Threonine at position 293, Arginine at position 296, Lysine at position 310 and Threonine at position 332.<br>
Specifically, the invention provides mutants with amino acid substitutions at one or more amino acid residues of S15G, T20A, K56E, D70A, N72S, N72D, N72T, V87A, K93E, K131R, Y185C, I190V, Y203C, K247R, T260A, K269R, N275D, W282R, I288T, I288C, T293M, T293A, R296G, N305T, N305I, A310, T20A: T293A, K14R: T293A, S15G: R296G, N72S: F226L, N72S: E212G: K269R, E49G: I288T: R296G, K56E: N72S: T232A, N72S: F226L: E49G, N72S: F226L: W282R, N285D: T293A, A73T: T332I, K131R: Y185C, N72S: F226L: E212G, N72S: F226L: Y185C, N72S: F226L: M306L wherein M at 306 position is substituted by L and not I, N72S: F226L: R296G, N72S: F226L: N305I wherein N at 305 position is substituted by I and not K, N72S: F226L: K93E, N72S: F226L: K269R, N72S: F226L: I288T and N72S: F226L: K56E, wherein the residue positions of the amino acid substitutions corresponds to those of a wild-type expandase-hydroxylase.<br><br>
The process of invention is to provide a modified cefEF gene encoding the mutated Penicillin expandase-hydroxylase.<br>
Another embodiment of the invention is to provide an expandase-hydroxylase protein with modified expansion and hydroxylation activity.<br>
Another embodiment of the invention is to provide an expandase-hydroxylase protein with modified expansion activity.<br>
Another embodiment of the invention is to provide an expandase-hydroxylase protein with modified hydroxylation activity.<br>
In another aspect, the invention provides a recombinant vector specifically an expression vector, which comprises the modified expandase-hydroxylase gene.<br>
The present invention further relates to a host strain that contains the expression vector with the modified expandase-hydroxylase (cefEF) gene.<br>
Another embodiment of the invention is to provide a method of expression of expandase-hydroxylase in a host strain that contains the expression vector with the modified expandase-hydroxylase (cefEF) gene.<br>
Detailed Description of the invention:<br>
The primary embodiment of the present invention is to provide a mutant expandase-hydroxylase having increased activity and greater specificity for substrates such as Penicillin G and Phenyl acetyl 7-ADCA than the wild type expandase-hydroxylase from the filamentous fungi C acremonium. This enzyme and the corresponding gene (cefEF gene) has been isolated and characterized (Dotzlaf J. E. et aL, Journal of Bacteriology, 169, 1611-1618, 1987). The wild-type nucleotide and amino acid sequence of expandase-hydroxylase from C.acremonium is given in SEQ ID No: 1 (Figure 2).<br><br>
According to the present invention, a mutated Penicillin expandase-hydroxylase which comprises amino acid substitution at one or more amino acid residues corresponding to the wild type expandase-hydroxylase for the group of residues consisting of Lysine at position 14, Serine at position 15, Threonine at position 20, Threonine at position 45, Glutamic acid at position 49, Lysine at position 56, Aspartic acid at position 70, Asparagine at position 72, Alanine at position 73, Valine at position 87, Lysine at position 93, Lysine at position 131, Tyrosine at position 185, Isoleucine at position 190, Tyrosine at position 203, Glutamic acid at position 212, Phenylalanine at position 226, Threonine at position 232, Lysine at position 247, Threonine at positon 260, Lysine at position 269, Asparagine at position 275, Tryptophan at position 282, Asparagine at position 285, Isoleucine at position 288, Threonine at position 293, Arginine at position 296, Lysine at position 310 and Threonine at position 332. Specifically, the invention provides mutants with amino acid substitutions at one or more amino acid residues of S15G, T20A, K56E, D70A, N72S, N72D, V87A, K93E, K131R, Y185C, I190V, Y203C, K247R, T260A, K269R, N275D, W282R, I288T, I288C, T293M, T293A, R296G, N305T, N305I, A310, T20A: T293A, K14R: T293A, S15G: R296G, N72S: F226L, A73T: T332I, K131R: Y185C, N285D: T293A, N72S: E212G: K269R, E49G: I288T: R296G, K56E: N72S: T232A, N72S: F226L: E49G, N72S: F226L: W282R, N72S: F226L: E212G, N72S: F226L: Y185C, N72S: F226L: M306L wherein M at 306 position is substituted by L and not I, N72S: F226L: R296G, N72S: F226L: N305I wherein N at 305 position is substituted by I and not K, N72S: F226L: K93E, N72S: F226L: K269R, N72S: F226L: I288T and N72S: F226L: K56E,wherein the residue positions of the amino acid substitutions corresponds to those of a wild type expandase-hydroxylase from C.acremonium.<br><br>
The following are the variations in the amino acid residues from the sequence in Seq. ID. NO. 1:<br>
-	Serine at position 15 is substituted by Glycine<br>
-	Threonine at position 20 is substituted by Alanine<br>
-	Lysine at position 56 is substituted by Glutamic acid<br>
-	Aspartic acid at position 70 is substituted by Alanine<br>
-	Asparagine at position 72 is substituted by Serine<br>
-	Asparagine at position 72 is substituted by Threonine<br>
-	Asparagine at position 72 is substituted by Aspartic acid<br>
-	Valine at position 87 is substituted by Alanine<br>
-	Lysine at position 93 is substituted by Glutamic acid<br>
-	Lysine at position 131 is substituted by Arginine<br>
-	Tyrosine at position 185 is substituted by Cysteine<br>
-	Isoleucine at position 190 is substituted by Valine<br>
-	Tyrosine at position 203 is substituted by Cysteine<br>
-	Lysine at position 247 is substituted by Arginine<br>
-	Threonine at position 260 is substituted by Alanine<br>
-	Lysine at position 269 is substituted by Arginine<br>
-	Asparagine at position 275 is substituted by Aspartic acid<br>
-	Tryptophan at position 282 is substituted by Arginine<br>
-	Isoleucine at position 288 is substituted by Threonine<br>
-	Isoleucine at position 288 is substituted by Cysteine<br>
-	Threonine at position 293 is substituted by Methionine<br>
-	Threonine at position 293 is substituted by Alanine<br>
-	Arginine at position 296 is substituted by Glycine<br>
-	Asparagine at position 305 is substituted by Threonine<br>
-	Asparagine at position 305 is substituted by Isoleucine<br><br>
-	Deletion of lysine at position 310<br>
-	Lysine at position 14 is substituted by Arginine and Threonine at position 293 is substituted by Alanine<br><br>
-	Serine at position  15 is substituted by Glycine and Arginine at position 296 is substituted by Glycine<br>
-	Threonine at position 20 is substituted by Alanine and Threonine at position 293 is substituted by Alanine<br>
-	Asparagine at position 72 is substituted by Serine and Phenylalanine at position 226 is substituted by Leucine<br>
-	Alanine at position 73 is substituted by Threonine and Threonine at position 332 is substituted by Isoleucine<br>
-	Lysine at position 131 is substituted by Arginine and Tyrosine at position 185 is substituted by Cysteine<br>
-	Asparagine at position 285 is substituted by Aspartic acid and Threonine at position 293 is substituted by Alanine<br>
-	Asparagine at position 72 is substituted by Serine, Glutamic acid at position 212 is substituted by Glycine and Lysine at position 269 is substituted by Arginine<br>
-	Glutamic acid at position 49 is substituted by Glycine, Isoleucine at position 288 is substituted by Threonine and Arginine at position 296 is substituted by Glycine<br>
-	Lysine at position 56 is substituted by Glutamic acid, Asparagine at position 72 is substituted by Serine and Threonine at position 232 is substituted by Alanine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Glutamic acid at position 49 is substituted by Glycine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Tryptophan at position 282 is substituted by Arginine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Glutamic acid at position 212 is substituted by Glycine<br><br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Tyrosine at position 185 is substituted by Cysteine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Methionine at position 306 is substituted by Leucine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Arginine at position 296 is substituted by Glycine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Lysine at position 93 is substituted by Glutamic acid<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Lysine at position 269 is substituted by Arginine<br>
-	Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Isoleucine at position 288 is substituted by Threonine<br>
- Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Lysine at position 56 is substituted by Glutamic acid and Asparagine at position 72 is substituted by Serine, Phenylalanine at position 226 is substituted by Leucine and Asparagine at position 305 is substituted by Isoleucine.<br>
The present invention provides a mutated Penicillin expandase-hydroxylase having enhanced ring-expansion activity for substrates such as Penicillin G containing the following mutations at one or more amino acid positions when compared with the wild-type: S15G, K56E, V87A, K131R, T260A, K247R, K269R, W282R, N305T, N72S: F226L: E49G, N72S: F226L: W282R, N72S: E212G: K269R and E49G: I288T: R296G.<br>
The present invention also provides a mutated Penicillin expandase-hydroxylase having enhanced hydroxylation activity for substrates such a Phenyl acetyl 7-ADCA<br><br>
containing the following amino acid substitutions at one or more amino acid residues when compared with the wild- type: K93E, Y203C, N275D, A310 and K14R: T293A.<br>
The present invention provides a modified Penicillin expandase-hydroxylase having enhanced catalytic activity for both ring expansion and hydroxylation on substrates like Penicillin G and Phenyl acetyl 7-ADCA when compared with the wild-type expandase-hydroxylase. The modified Penicillin expandase-hydroxylase carries the following amino acid substitutions at one or more amino acid residue positions: T20A, D70A, N72S, N72T, N72D, Y185C, II90V, I288T, I288C, T293M, T293A, R296G, N305I, T20A: T293A, S15G: R296G, N72S: F226L, A73T: T332I, K131R: Y185C, N285D: T293A, K56E: N72S: T232A, N72S: F226L: E212G, N72S: F226L: Y185C, N72S: F226L: M306L wherein the substitution at 306 position is not I, N72S: F226L: R296G, N72S: F226L: K93E, N72S: F226L: K269R, N72S: F226L: I288T, N72S: F226L: K56E and N72S: F226L: N305I wherein the substitution at 305 position is not K.<br>
Another embodiment of the invention is to provide an isolated nucleic acid molecule that codes for the mutated Penicillin expandase-hydroxylase. According to the invention, this isolated nucleic acid molecule is obtained by mutating the wild type expandase-hydroxylase. The mutagenesis technique could be by chemical, error- prone PCR or site- directed approach. The suitable mutagenesis technique can be selected and used for introducing mutations and the mutated nucleic acid molecule can be cloned and expressed and the property of the polypeptide can be studied.<br>
In another aspect of the invention, the mutated nucleic acid molecule may be incorporated into a recombinant vector, which is capable of expression or replication when transferred into a host cell. Expression of the polypeptide can be controlled by a regulatory sequence probably a promoter.<br>
The recombinant vector can be introduced into a host strain to produce the mutated Penicillin expandase-hydroxylase.<br><br>
The mutated expandase-hydroxylase when expressed in the host strain is capable of converting the substrate Penicillin G into Phenyl acetyl 7-ADCA by ring expansion and Phenyl acetyl 7-ADCA to deacetyl phenyl acetyl 7-ACA by hydroxylation. Phenyl acetyl 7-ADCA, when treated with a suitable enzyme such as PenG Amidase (EP0453047), the Phenyl acetyl side chain can be removed efficiently and 7- ADCA can be produced. Deacetyl phenyl acetyl 7-ACA can lead to 7-ACA after acetylation by acetyl transferase or suitable chemical conversion followed by enzymatic hydrolysis using Pen G amidase. It has been found that the mutated expandase-hydroxylase when expressed in the host strain according to this invention has greater activity against the conversion of penicillin N into deacetoxy cephalosporanic acid or deacetylcephalsoporanic acid, followed by converting the ensuing product into 7-ACA via cephalosporin C.<br>
According to the present invention, the modified peptide has amino acid sequence different from that of SEQ ID NO: 1. This polypeptide is one, which has ring expansion activity i.e. catalyze the ring expansion of the five membered thiazolidine ring of Penicillin to six membered ring of a Cephalosporin and hydroxylation activity i.e catalyze the hydroxylation of 3-methyl to 3- hydroxymethyl side chain of deacetyl phenyl acetyl 7-ACA. The ring expansion and hydroxylation activity of the polypeptide are modified or increased and also the catalyzing activity for a substrate other than its natural substrate is increased. The invention provides a modified expandase-hydroxylase, which has an enhanced catalytic activity or increased specificity for other substrates such as Penicillin G, Phenyl acetyl 7-ADCA when compared with the wild-type expandase-hydroxylase.<br>
The polypeptides thus produced from the mutated nucleotide sequence can be used to produce chimeras from portions of other expandase-hydroxylase, expandase and hydroxylase polypeptides.<br><br>
Polypeptides from the present invention can be purified with varying level of homogeneity and can be used for other purposes.<br>
The modified expandase-hydroxylase produced according to the invention is found useful in the bioprocess of preparing Cephalosporin C or Deacetyl Cephalosporanic acid by utilizing the process known in the prior art. The invention can be used for the manufacture of modified cephalosporins either as enzymatic or in vivo fermentation based technologies. The detailed procedures, such as transformation and fermentation of such cells, purification and isolation can be found in the literature.<br>
Escherichia coli BL21 (DE3) strains with modified expandase-hydroxylase genes deposited in Microbial Type Culture Collection center Chandigarh, India under Budapest treaty were designated with the following accession numbers; MTCC5167 deposited on 20.07.2004; MTCC 5222, MTCC 5223, MTCC 5224, MTCC 5225, MTCC 5228, MTCC 5229, MTCC 5230, MTCC 5232, MTCC 5233, MTCC 5234, MTCC 5235, MTCC 5236, MTCC 5237, MTCC 5239, MTCC 5240, and MTCC 5241 deposited on 11.07.2005; MTCC 5280, MTCC 5281, MTCC 5282, MTCC 5283, MTCC 5284, MTCC 5285, MTCC 5286, MTCC 5287, MTCC 5288, MTCC 5289, MTCC 5290, MTCC 5291, MTCC 5292, MTCC 5293, MTCC 5294, MTCC 5295, MTCC 5296, MTCC 5297, MTCC 5298, MTCC 5299, MTCC 5300, MTCC 5301 and MTCC 5302 deposited on 07.08.2006.<br>
Materials:<br>
All the chemicals and reagents were purchased either from Sigma-Aldrich Chemicals Pvt. Ltd or USB, USA. Oligonucleotides were synthesized and supplied by Sigma-Aldrich Chemicals Pvt. Ltd, India. Restriction enzymes, pUC19 vector for cloning and strains were obtained from New England Biolabs Inc, USA. pET24a for expression, Escherichia coli BL21 (DE3) the expression host strain and Bugbuster reagent were from Novagen, USA. Cephalosporium acremonium was obtained from the American Type<br><br>
Tissue Culture center (ATCC). Bradford reagent was purchased from Biorad, USA. Cig columns (250x4.6mm, 5u, symmetry) were obtained from Waters, USA. DNeasy Plant mini gDNA isolation Kit was supplied by Qiagen, Germany and growth media components were obtained from Becton Dickinson, USA.<br>
Cephalosporium acremonium growth conditions:<br>
The C. acremonium strain (ATCC No. 11550) was grown under the following medium composition: Conidia was grown in plates containing complete media (sucrose 20 gm, peptone 4 gm, yeast extract 4 gm, NaN03 3 gm, KH2P04 0.5 gm, K2HP04 0.5 gm, KC1 0.5 gm, MgS04.7H20 0.5 gm, FeS04.7H20 0.01 gm and agar 20 gm made up to 1 litre of distilled water at a pH of 6.6) at 25°C. For the isolation of genomic DNA, the spores were seeded on complete liquid media and grown at 25°C at 220 rpm. The mycelia were harvested and cell pellets were collected after centrifugation at 16,000 rpm for 15 minutes.<br>
Polymerase chain reaction:<br>
Genomic DNA was isolated from the mycelia using the DNeasy Plant mini gDNA isolation kit (Qiagen) as recommended by the supplier. The gene coding for the expandase-hydroxylase (ce/EF) (Accession number: AJ404737) was amplified using 20 pmole of primers 5' TCCATATGACTTCC AAGGTCCCCGT 3' and 5' GAAACTCTTGACACTCATGCTTTGTGT 3\ 200 uM dNTPs, 5% DMSO, lOx deep vent DNA polymerase buffer, 2.5U deep vent DNA polymerase enzyme and water in a final reaction volume of 100 |il. PCR condition consists of an initial denaturation for 5 min at 95°C followed by 24 cycles consisting of denaturation at 95°C for 40 sec, annealing at 60°C for 1 min, extension at 72°C for 5 min with a final extension at 72°C<br><br>
for 15 min. An amplified product of length of approximately 1100 bp of the ceflEF gene fragment was seen by agarose gel electrophoresis.<br>
Cloning in pUC19 and pET24a:<br>
The expandase-hydroxylase gene fragment observed after amplification was purified by Qiaquick PCR purification kit supplied by Qiagen and cloned into pUC19 vector through blunt-end ligation using Smal restriction site. Subsequently, the expandase-hydroxylase gene fragment was released by digestion with Ndel/Hindlll and ligated into similarly digested pET24a (+) to give pOCPLEF vector and transformed into competent Escherichia coli BL21 (DE3) strain for further expression.<br>
Random Mutagenesis:<br>
The native EF gene fragment cloned into pUC19 served as template for error-prone PCR<br>
mutagenesis. The amplification was carried out with 20 pmole of 5'<br>
ATCGGTGCGGGCCTCTTCGCTATT	3'	and	5 *<br>
CTCACTCATTAGGCACCCCAGGCT 35 primers in a reaction mix containing 10% DMSO, lOx Taq DNA polymerase buffer, 2.5U Taq DNA polymerase enzyme and water in a final volume of 100 \x\ and amplified as described earlier.<br>
Restriction enzymes Ndel and Hindlll were used to digest the mutated gene fragment and cloned into the expression vector pET24a. E. coli BL21 (DE3) recombinants were screened for inserts by colony PCR using 20 pmoles of 5' TCCATATGACTTCCAAGGTCCCCGT 3', 5' GCTAGTTATTGCTCAGCGG 3' primers in a reaction mix containing 10% DMSO, lOx Taq DNA polymerase buffer, 1U Taq DNA polymerase enzyme and 80 (iM dNTPs in a total reaction volume of 50 (il.<br><br>
Expression:<br>
Glycerol stocks containing the putative mutant EF genes in pET24a (+) vector in E.coli BL21 (DE3) strain was inoculated in 96-well plate containing LB medium with Kanamycin (75 fig/ml) for overnight growth at 37°C at 220 rpm. Overnight culture was subcultured again in 96-well deep well plates and grown till OD60o reached to 0.6 to 0.8 and the induction was carried out with 0.1 mM isopropyl-p-D-thiogalactopyranoside (IPTG). After induction, the culture was allowed to grow for 3 hours at 25°C and pellet was harvested by centrifugation in a micro plate centrifuge at 4,000 rpm for 10 min at 4°C. The pellets were resuspended in a buffer containing 50 mM Tris. HC1 (pH 7.5), 0.1 mM DTT, 0.01 mM EDTA, 10% Glycerol, 50 mM Glucose and stored at ~80°C.<br>
Assay for activity screening:<br>
Enzyme was released using bugbuster reagent and the ring expansion reaction was<br>
assayed with 45.7 mM Ammonium Sulfate, 0.9 mM oc-keto glutarate, 0.9 mM Ascorbate, 0.9 mM DTT, 0.9 mM Ferrous sulfate, 12.8 mM Penicillin G and 16 mM Tris.HCl in a reaction volume of 350 (il at pH 7.5 followed by quenching with methanol. Hydroxylation reaction was carried out using 35.5 mM Ammonium Sulfate, 0.7 mM a-keto glutarate, 0.7 mM DTT, 0.7 mM Ascorbate, 0.07 mM Ferrous Sulfate, 17.33 mM Tris.HCl and saturated Phenyl acetyl 7-ADCA in a reaction volume of 450 }il at 25°C for 30 minutes with shaking at 220 rpm. The reaction was quenched with 3 |il of 10% H3PO4 and analyzed by HPLC.<br>
Bioassay:<br>
Production of the ring-expanded product was detected using the paper disc diffusion<br>
method. 25 (il of the assay mix was spotted on to 6 mm discs and allowed to dry. The discs were subsequently transferred to LB plates containing Penase concentrate (Difco)<br><br>
(54,000 IU/ml) spread with E.coli ESS strain (kindly provided by Dr. S. E. Jensen, University of Alberta, Canada) and incubated overnight at 37°C. Recombinant clones that had zones of size greater than the wild type DAOCS-DACS were selected for further reconfirmation by HPLC analysis. All the mutants were screened for hydroxylation activity by HPLC analysis.<br>
Reconfirmation of the recombinant clones:<br>
The short-listed positive isolates were re-expressed in 50 ml conical flasks containing 10 ml LB with 75 jig /ml Kanamycin in which over night grown culture was inoculated. After OD6oo °f 0.6 to 0.8 was reached, induction was initiated by adding 0.1 mM IPTG. Pellets were harvested from 1.5 ml culture after allowing the culture to grow for 3 hours<br>
at 25°C. The pellets were resuspended in 50 mM Tris. HC1 (pH 7.5), 0.1 mM DTT, 0.01 mM EDTA, 10% Glycerol, 50 mM Glucose and stored at »80°C.<br>
Lysis of the resuspended pellet was carried out at 25°C for 25 min at 220 rpm using Bugbuster reagent. The ring expansion reaction was assayed using 35.6 mM Ammonium Sulfate, 0.7 mM Ascorbate, 0.7 mM DTT, 0.7 mM ct-keto glutarate, 0.07 mM Ferrous sulfate, 10 mM Penicillin G and 12 mM Tris-HCl at pH 7.5 in a reaction volume of 450 |al at 25°C for 30 minutes. The reaction was quenched with 75 \i\ methanol, centrifuged and the supernatant was analyzed. Reconfirmation reaction for hydroxylation activity was performed using the same reagent concentrations as that used for screening.<br>
HPLC analysis:<br>
Cig symmetry column was equilibrated with buffer A containing 3.12g of Sodium phosphate in water with pH adjusted to 2.4 using ortho phosphoric acid. The assay components were eluted using buffer B containing 100% Acetonitrile: buffer A at a ratio<br><br>
of 80:20 using a flow rate of 1.5 ml/min. The components were detected at 215nm and 260 nm and 40 ^il of sample was injected. The retention times of the compounds under these conditions are 2.1 min, 4.3 min and 5.2 min for deacetyl phenyl acetyl 7-ADCA, Phenyl acetyl 7-ADCA and Penicillin G respectively.<br>
Site-directed mutagenesis:<br>
Single-stranded templates of either native or mutant gene in pET24a vector were generated from E. coli CJ236 using M13K07 helper phage by standard procedure as described in "Molecular Cloning, A laboratory Manual, 2nd Edition by Sambrook et al, Cold Spring Harbor Laboratory Press, 1989 and the mutagenesis was based on Kunkel mutagenesis principle (Kunkel, T. A. Proc. Natl. Acad. Sci. USA, 82, 488-492 1985) as follows. Briefly, oligonucleotides to induce desired mutations were annealed either alone or in multiple combinations to the single-strand templates and the in vitro second-strand synthesis was carried out in presence of 200 uM of each dNTPs, 0.2 mg/ml of BSA, 0.5 mM of ATP, 2 units of T4 DNA ligase, 3 units of T4 DNA polymerase and 10 mM MgCl2 at 42°C for 20 minutes in a reaction volume of 20 uL. After terminating the reaction with 2 ul of 0.5 M EDTA, 1 ul was used for transformation of E.coli DH5a. Mutants were confirmed with restriction analysis followed by DNA sequencing.<br><br><br><br><br><br><br><br><br>
 We claim:<br>
1.	A mutant expandase-hydroxylase with increased activity and greater substrate specificity for Penicillin G and Phenyl acetyl 7-ADCA for the production of Phenyl acetyl - 7-ADCA and Deacetyl phenyl acetyl 7-ACA respectively; which carries one or more amino acid modification at residue positions when compared with the wild type expandase-hydroxylase from the following group of residues, Lysine at position 14, Serine at position 15, Threonine at position 20, Threonine at position 45, Glutamic acid at position 49, Lysine at position 56, Aspartic acid at position 70, Asparagine at position 72, Alanine at position 73, Valine at position 87, Lysine at position 93, Lysine at position 131, Tyrosine at position 185, Isoleucine at position 190, Tyrosine at position 203, Glutamic acid at position 212, Phenylalanine at position 226, Threonine at position 232, Lysine at position 247, Threonine at position 260, Lysine at position 269, Asparagine at position 275, Asparagine at position 285, Tryptophan at position 282, Isoleucine at position 288, Threonine at position 293, Arginine at position 296, Lysine at position 310 and Threonine at position 332.<br>
2.	The mutant expandase-hydroxylase and its corresponding gene of claim 1, wherein the wild type expandase-hydroxylase and its corresponding gene are obtained from Cephalosporium acremonium.<br>
3.	The mutant expandase-hydroxylase of claim 1 with variation in one or more amino acid residue positions at S15G, T20A, K56E, D70A, N72S, N72D, V87A, K93E, K131R, Y185C, 1190V, Y203C, K247R, T260A, K269R, N275D, W282R, I288T, I288C, T293M, T293A, R296G, N305T, N305I and A310.<br>
4.	The expandase-hydroxylase of claim 1 with following variation in the amino acid residue positions at T20A: T293A, K14R: T293A, S15G: R296G, N72S: F226L, A73T: T332I, K131R: Y185C, N285D: T293A, N72S: E212G: K269R, E49G: I288T: R296G,<br><br>
K56E: N72S: T232A, N72S: F226L: E49G, N72S: F226L: W282R, N72S: F226L: E212G, N72S: F226L: Y185C, N72S: F226L: M306L wherein M306 position is not substituted by I, N72S: F226L: R296G, N72S: F226L: K93E, N72S: F226L: K269R, N72S: F226L: I288T, N72S: F226L: K56E and N72S: F226L: N305I wherein the substitution at N305 position is not K.<br>
5.	The vector containing the expandase-hydroxylase gene of Claim 1.<br>
6.	The microorganism bearing the recombinant vector of Claim 5, capable of expressing the gene.<br>
7.	A method of expressing the enzyme in the organism described in claim 6.<br>
8.	Use of the expressed enzyme described in claim 7 for the synthesis of modified cephalosporins<br>
9.	Use of the microorganism in claim 6 for the synthesis of modified cephalosporins<br>
10.	An enzyme having significant sequence identity to claim 2, wherein one or more amino acid positions of claim 1 is modified at analogous positions in the second enzyme.<br>
Dated this twenty fourth (24th) day of August 2006<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgIEFNRU5ERUQgQ0xBSU1TICAyMy0xMi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1177-CHE-2005   AMENDED CLAIMS  23-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgIEFNRU5ERUQgQ0xBSU1TICAyNy0wOC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1177-CHE-2005   AMENDED CLAIMS  27-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgIENPUlJFU1BPTkRFTkNFIE9USEVSUyAgMjctMDgtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1177-CHE-2005   CORRESPONDENCE OTHERS  27-08-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgIEVYQU1JTkFUSU9OIFJFUE9SVCBSRVBMWSBSRUNFSVZFRCAgMjMtMTItMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1177-CHE-2005   EXAMINATION REPORT REPLY RECEIVED  23-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgIEVYQU1JTkFUSU9OIFJFUE9SVCBSRVBMWSBSRUNFSVZFRC4gMDUtMDYtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1177-CHE-2005   EXAMINATION REPORT REPLY RECEIVED. 05-06-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgQU1FTkRFRCBDTEFJTVMgIDE3LTA0LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">1177-CHE-2005  AMENDED CLAIMS  17-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAwNS0wNi0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1177-CHE-2005  CORRESPONDENCE OTHERS  05-06-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1DSEUtMjAwNSAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAxNy0wNC0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1177-CHE-2005  EXAMINATION REPORT REPLY RECEIVED  17-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1177-che-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1kZXNjcmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1kZXNjcmlwdGlvbihwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-description(provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1177-che-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE3Ny1jaGUtMjAwNS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">1177-che-2005-form 5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260381-nanoparticulate-ultrasound-contrast-agents-for-medical-diagnostics-and-molecular-imaging.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260383-reforming-process-for-synthesis-gas-production-and-related-plant.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260382</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1177/CHE/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-May-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ORCHID CHEMICALS &amp; PHARMACEUTICALS LTD</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ORCHID TOWERS, 313, VALLUVAR KOTTAM HIGH ROAD, NUNGAMBAKKAM, CHENNAI.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MICHEAL DURAIRAJ</td>
											<td>ORCHID CHEMICALS&amp;PHARMACEUTICALS LTD, 476/14,OLD MAHABALIPURAM ROAD, SHOLINGANALLUR, CHENNAI.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TWINKLE JASMINE MASILAMANI</td>
											<td>ORCHID CHEMICALS&amp;PHARMACEUTICALS LTD, 476/14,OLD MAHABALIPURAM ROAD, SHOLINGANALLUR, CHENNAI-600 119 TAMILNADU</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MEGHANA RAVINDRANATHAN</td>
											<td>ORCHID CHEMICALS&amp;PHARMACEUTICALS LTD, 476/14,OLD MAHABALIPURAM ROAD, SHOLINGANALLUR, CHENNAI-600 119 TAMILNADU</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RAMAKRISHNA PADHARTHI</td>
											<td>ORCHID CHEMICALS&amp;PHARMACEUTICALS LTD, 476/14,OLD MAHABALIPURAM ROAD, SHOLINGANALLUR, CHENNAI-600 119 TAMILNADU</td>
										</tr>
										<tr>
											<td>5</td>
											<td>VASU VINAYAGAM</td>
											<td>ORCHID CHEMICALS&amp;PHARMACEUTICALS LTD, 476/14,OLD MAHABALIPURAM ROAD, SHOLINGANALLUR, CHENNAI-600 119 TAMILNADU</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P35/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260382-modified-expandase-hydroxylase-and-its-applications by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:55:38 GMT -->
</html>
