<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260473-an-in-vitro-method-of-identifying-compounds-modifying-cellular-responses by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:22:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260473:AN IN-VITRO METHOD OF IDENTIFYING COMPOUNDS MODIFYING CELLULAR RESPONSES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN IN-VITRO METHOD OF IDENTIFYING COMPOUNDS MODIFYING CELLULAR RESPONSES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for identifying compounds capable of modulating a cellular response. The methods involve attaching living cells to solid supports comprising a library of test compounds. The test compounds are linked to the solid support via cleavable linkers and may thus be released from the solid supports. Solid supports comprising cells, wherein the cellular response of interest has been modulated are selected and the test compound of the solid support can then be identified. The cellular response may for example be changes in complex formation between proteins.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>All patent and non-patent references cited in the application are hereby incorporated by reference in their entirety.<br>
Field of invention<br>
The present invention relates to a method and tools for extracting information relating to an influence, for example on intracellular molecule(s), in particular an influence caused by contacting a cellular molecule with a substance, which has been released from a solid support to which a cell expressing the cellular component is attached. In particular the method related to a solid support that allow chemical synthesis of individual substances on beads of the solid support<br>
The method of the invention may be used as a very efficient procedure for testing or discovering the influence of a library of substances on a physiological process, for example in connection with screening for new drugs, testing of substances for toxicity, identifying drug targets for known or novel drugs. Other valuable uses of the method and technology of the invention will be apparent to the skilled person on the basis of the following disclosure<br>
Background of invention<br>
Combinatorial synthesis of peptide as well as small-molecule libraries has proven very useful as a method for generating vast numbers of highly diverse compounds (see for example Comprehensive Survey of Combinatorial Library Synthesis: 2002 Roland E. Dolle J. Comb. Chem.,2003, pp 693 - 753). To fully exploit this high caMcity of combinatorial chemistry to produce huge numbers of compounds several tecrmologies have been developed that allow screening directly on the solid support (M.Meldal, 1994, METHODS: A companion to methods of enzymology 6:417-424). In the field of drug discovery such methods have successfully been applied for example for the identification of enzyme modulators. The library can be synthesized on resin beads that each carry one specific compound, and these "one-bead-one<br><br>
compound" libraries are then screened against the purified biological component of interest (e.g. cellular proteins or peptides),<br>
Before progressing active compounds, identified though such procedure, further in the drug discovery process, the compound will have to be re-synthesized and tested for efficacy in a cell-based or in-vivo test system.<br>
Novel ways to screen combinatorial libraries in a physiological more correct way are assumed to greatly accelerate the drug discovery process, and show importance in areas like chemo-genomics and chemo-proteomics.<br>
Screening of combinatorial libraries in intact cells have been done by capturing mammalian or yeast cells together with a limited number of resin-beads in a "nanodroplet" (Borchart et al. Chem Biol 1997 4:961). Compounds immobilized on the resin are released through disruption of a photo-cleavable linker and the compound-associated effects on the intact cells are monitored. In an alternative method the compounds are released through acidolysis resin-beads carrying the library members area are spread out on a lawn of mammalian cells, and the spatial localization of a cellular response is monitored and beads in that region is isolated, and the remaining compound is structure elucidated Jayawickreme et al, 1998, Combinatorial peptide Library Protocols, Ed. Shmuel Cabilly, Humana Press, p. 107-128).<br>
WO03/038431 describes methods for screening combinatorial bead libraries by capturing cells from body fluids. Beads comprising a compound enabling cells to adhere to said bead may be selected.<br>
US2003/0059764 describes multiplexed cell analysis systems using non-positional or positional arrays of coded carriers.<br>
Summary of invention<br>
It is of great importance to provide new and efficient methods for identification of compounds influencing specific cellular processes. In particular, such methods wherein a very large quantity of candidate compounds may be tested for a specific effect on a cell within a relatively short period of time.<br>
It is therefore an object of the present invention to provide very efficient procedures for testing or discovering the influence of compounds of a library on a physiological<br><br>
process in a cell. In particular, the methods provides means for testing very large numbers of different compounds for one or more physiological effects within a rather short time period. This may be obtained by attaching living cells to resin beads coupled to a test compound. The test compounds may be released from the resin beads and thus influence physiological processes in said cells. Said influence(s) may be detected and beads containing cells displaying the desired influence(s) may be selected. Once selected the compounds coupled to the selected beads may be identified. These methods may for example be very useful in connection with screening for new drugs, testing of substances for toxicity, identifying drug targets for known or novel drugs.<br>
Accordingly, it is a first objective of the invention to provide methods of identifying a compound modifying at least one cellular response, wherein each cellular response is linked to different reporter systems generating detectable outputs, said method comprising the steps of:<br>
(a)	Providing multiple resin beads capable of supporting growth of cells,<br>
wherein each resin bead is linked to multiple copies of a member of a li<br>
brary of test compounds via a cleavable linker and wherein at least two<br>
beads comprise different library members; and<br>
(b)	Attaching cells comprising said reporter system(s) onto said resin beads;<br>
and<br>
(c)	Releasing a proportion of said library member from the resin bead; and<br>
(d)	Screening said resin beads for beads comprising cells meeting at least<br>
one predetermined selection criterion, wherein said selection criterion is<br>
linked directly or indirectly to said detectable output; and<br>
(e)	Selecting beads comprising cells meeting said at least one selection cri<br>
terion; and<br>
(f)	Identifying the library member remaining linked to the selected resin<br>
bead, thereby identifying a compound modifying said at least one cellular<br>
response.<br>
The method involves release of a proportion of library member. The released library member may enter into cells in the immediate surroundings and thus influence cellular responses within said cells. In practical terms, the cells present in the immediate surroundings, will be the cells attached to the resin bead, from which the<br><br>
library member is released. Thus resin beads comprising cells, wherein the particular cellular response has been modified may be selected and the library member remaining bound to said resin beads can be identified.<br>
The invention furthermore relates to methods of manufacturing a compound modifying at least one cellular response, wherein said method comprises the steps of:<br>
a)	Identifying said a compound modifying a cellular response according to the<br>
methods described herein<br>
b)	Preparing said compound by chemical synthesis<br>
c)	Thereby manufacturing said compound<br>
The invention also relates to methods of modulating the interaction between two cellular molecules comprising the steps of<br>
a)	Identifying a. modulating interaction between two cellular molecules accord<br>
ing to the methods described herein<br>
b)	Incubating said compound together with cells expressing said two cellular<br>
molecules<br>
c)	Thereby modulating the interaction between the two cellular molecules<br>
Depending on the nature of the two cellular molecules, specific cellular responses may be inhibited/activated. This may in particular be interesting for use of the compounds in therapy.<br>
The invention furthermore relates to compounds identified by the methods disclosed herein.<br>
Description of Drawings<br>
Figure 1A illustrates a method of identifying a resin bead comprising a compound influencing a cellular response linked to a reporter system generating a fluorescent output. The method involves cultivating cells on resin beads, fixing cells, FABS calibration using a positive and a negative control, identification and isolation of positive hits.<br><br>
Figure 1B illustrates a method of identifying a resin bead comprising a compound influencing a cellular response linked to a reporter system generating a fluorescent output detectably using a plate reader or image acquisition analysis. The method involves 1) Grow cells on beads for 24 hrs and Fix cells in EtOH, 2) Add app. 20 beads to each well and Identify hit wells using plate reader or image acquisition/analysis and 3) Transfer beads from hit wells to a new 384 well plate - one bead/well and identify hit wells using plate reader or image acquisition. If for example 500.000 beads are screened with 20 beads/well,<br>
approx. 25.000 wells, i.e. approx. 68 plates must be screened. With a 0.1% hit rate, there will be approx. 500 hit wells comprising approx. 10.000 beads, which amounts to analysis of approx. 27 plates in the second round. Alternatively, positive beads may be picked directly (preferably without fixation) after the first identification using image acquisition analysis. The method may for example be used for analysis of expression of a Cre-YFP construct.<br>
Figure 2A illustrates a multiplexed screen involving FABS and microscopy. The screen involves I) identification of positive hits by FABS as displayed in figure 1, followed by II) a step of microscopy identifying resin beads comprising cells with an internal fluorescent signal.<br>
Figure 2B illustrates a multiplexed screen involving two FABS analysis. The screen involves I) identification of positive hits by FABS as displayed in figure 1, followed by II) a second FABS analysis.<br>
Figure 3 illustrates a plasmid map of pCRE-d2EGFP<br>
Figure 4 illustrates examples of cleavable linkers useful with the present invention.<br>
Figure 5 illustrates spectra and structure determination by accurate mass differences from single beads<br>
Figure 6 illustrates structure determination by accurate mass differences from single beads<br>
Figure 7 illustrates a fragmentation pathway<br><br>
Figure 8 illustrates examples of an adhesion peptide displaying bead covered with cells (U2OS).<br>
Definitions<br>
Naturally occurring amino acids are named herein using either their 1-letter or 3-letter code. If nothing else is specified amino acids may be of D or L-form. In the description (but not in the sequence listing) 3-letter codes starting with a capital letter indicate amino acids of L-form, whereas 3-letter codes in small letters indicate amino acids of D-form. Three- and one-letter abbreviations for amino acids are used according to the recommendations from IUPAC, see for example http://www. chem.qmw.ac.uk/iupac.<br>
The term "a" as used herein, can mean one or more, depending on the context in which it is used.<br>
In the present context, the term "green fluorescent protein" or (GFP) is intended to indicate a protein which, when expressed by a cell, emits fluorescence upon exposure to light of the correct excitation wavelength (cf. [(Chalfie et al.1994}]). "GFP" as used herein means any protein or fragment thereof capable of fluorescing when excited with appropriate radiation. This includes fluorescent proteins that are either naturally occuring or engineered and proteins that have been modified to be fluorescent. Naturally occuring fluorescent proteins have been isolated from the jellyfish, Aequorea vistoria, the sea pansy, Renilla reniformis, Phialidium gregarium and Discosoma coral (W.W. Ward et al. (1982) Photochem. Photobiol, 35:803-808; Levine et al. (1982) Biochem. Physiol., 728:77-85; Fradkov et al. (2000), FEES Lett. 479:127-130). GFPs have also been engineered to emit different colors and to fluoresce more intensely in mammalian organisms (U.S. Pat. Np. 5,625,048; WO 97/28261; WO 96/23810; EP0851874; US6,172,188; W001/98338). A variety of Aequorea-related fluorescent proteins have been engineered to have different excitation and emission spectra by modifying the naturallt occuring amino acid sequence (O.C. Prasher et al. (1992) Gene 111:229-233; Heim et al. (1994)<br><br>
Proc. Natl. Acad. Sci. USA 91: 12501-12504; US. Pat. No. 5,625,048; WO 96/23810 and PCT/US97/14593).<br>
The term "living cell" is used to indicate a cell which is considered living according to standard criteria for that particular type of cell such as maintenance of normal membrane potential, cell membrane integrity and energy metabolism<br>
The terms "image processing" and "image analysis" are used to describe a large family of digital data analysis techniques or combination of such techniques which reduce ordered arrays of numbers (images) to quantitative information describing those ordered arrays of numbers. When said ordered arrays of numbers represent measured values from a physical process, the quantitative information derived is therefore a measure of the physical process.<br>
The term "fluorescent probe" is used to indicate a fluorescent fusion polypeptide comprising a GFP or any functional part thereof which is N- or C-terminally fused to a biologically active polypeptide as defined herein, optionally via a peptide linker consisting of one or more amino acid residues, where the size of the linker peptide in itself is not critical as long as the desired functionality of the fluorescent probe is maintained. A fluorescent probe according to the invention is expressed in a cell and basically mimics the physiological behaviour of the biologically active polypeptide moiety of the fusion polypeptide.<br>
The term "determining the fluorescence" is used to describe the process used to monitor a change in fluorescence properties.<br>
The term "bioluminescence" is used to describe a process where light is produced through a chemical reaction that natively is occuring in a biological system. For the reaction to occur at least two chemicals are required: the one that produces the light (called "luciferin") and the other (called "luciferase") that catalyzes the reaction. Sometimes the luciferin and luciferase are brought together in one single unit (called "photoprotein" an example of the last group is aequorin.<br>
The term "FRET" is used to describe the occurrence of Fluorescence resonance energy transfer between a fluorophore donor and an acceptor fluorophore. It is a<br><br>
distance-dependent interaction between the electronic excited states of two fluoro-phores in which excitation is transferred from a donor fluorophore to an acceptor fluorophore without emission of a photon. The efficiency of FRET is dependent on the inverse sixth power of the intermolecular separation, making it useful over distances comparable with the dimensions of biological macromolecules. Thus, FRET is an important technique for investigating interactions between cellular molecules for example complex formation.<br>
The term "BRET" is used to describe a process that is related to FRET, but differs from FRET in that donor is a bioluminescent protein like luciferase that generates its own luminescence emission in the presence of a substrate, and that can pass the energy to an acceptor fluorophore. For either BRET or FRET to work, the donor's emission spectrum must overlap the acceptor's absorption spectrum, their transition dipoles must be in an appropriate orientation, and the donor and acceptor must be in close proximity (usually within '30-80 A of each other, depending on the degree of spectral overlap).<br>
The term "Scintillation Proximity Assay' is used to describe an assay determining the distance between two compounds, wherein one compound (bound to a bead) will emit light when radiation from an isotope occurs in close proximity and the other compound is containing a radioactive isotope.<br>
The term "Proximity ligation" is used to describe an assay determining the presence of a target molecule through the convergence of two different protein-binding reagents that specifically recognise said target molecule. Attached to each protein-binding reagent are nucleic acid sequences that when broad into close proximity will create a DNA reporter sequence through a ligation reaction (see Gullberg et al. Curr Opinion Biotechnology 2003,14:82)<br>
The term "mammalian cell" is intended to indicate any cell of mammalian origin. The cell may be an established cell line, many of which are available from The American Type Culture Collection (ATCC, Virginia, USA) or a primary cell with a limited life span derived from a mammalian tissue, including tissues derived from a transgenic animal, or a newly established immortal cell line derived from a mammalian tissue including transgenic tissues, or a hybrid cell or cell line derived by fusing different<br><br>
celltypes of mammalian origin e.g. hybridoma cell lines. The cells may optionally express one or more non-native gene products, e.g. receptors.<br>
The phrase "fluorescence properties" means absorption properties, such as wavelength and extension, and spectral properties of the emitted light, such as wavelength, fluorescence lifetime, intensity or polarisation, or the intracellular localisation of the fluorophore. It may thus be localised to a specific cellular component (e.g. organelle, membrane, cytoskeleton, molecular structure) or it may be evenly distributed throughout the cell or parts of the cell.<br>
The term "fixed cells" is meant to cover cells treated with a cytological fixative such as glutaraldehyde, methanol, acetone or formaldehyde, treatments which serve to chemically cross-link and/or stabilize soluble and insoluble proteins within the structure of the cell or to dehydrate cells. Once in this state, such proteins cannot be lost from the structure of the now-dead cell.<br>
The term "cell line" is meant to cover a group of cells, wherein the cells of that group are essentially genetically indistinguishable from each other. The cells of a cell line are thus all progeny of the same cell.<br>
The term "comprising" should be understood in an inclusive manner. Hence, by way of example, a composition comprising compound X, may comprise compound X and optionally additional compounds.<br>
The term "multiple" should be understood as "at least two".<br>
The term "library of test compounds" should be understood as a collection of test compounds comprising at least 2 different test compounds.<br>
The term "small organic molecules or compounds" refers herein to non-oligomeric, carbon containing compounds producible by chemical synthesis and generally having a size of less than 600 mass units.<br>
The term split/mix refer herein to the process of i) dividing a bead assembly into m portions and reacting each portions with a different building block followed by mixing<br><br>
the resin into one portion providing an even distribution throughout the assembly of beads containing said building blocks, ii) preparing (activating) the resin for attachment of the next building block and repeating the process n times of dividing, reacting, mixing and activating, thus providing an exponential growth of the number (mn) of distinct molecular entities of complexity n each attached to separate beads.<br>
The term "one bead-one compound library" refers to libraries immobilised on resin beads, wherein each individual resin bead does not comprise more than one library member in one or multiple copies. In a particular form of such libraries each member is represented by multiple fragments of said member obtained by ladder synthesis encoding.<br>
The term "one bead-two compound library" refers to libraries immobilised on resin beads, wherein each individual resin bead does not comprise more than one library member in one or multiple copies and wherein each individual resin bead in addition to said library member also comprises an adhesion compound. All beads may comprise identical adhesion compounds.<br>
The term "cleavable linker" is used to describe any chemical moiety which may be used to attach any molecule to a solid support either covalently or via complex formation and thereafter release said molecule by the action of either acid, base, elec-trophiles, nucleophiles, oxidative agents, reductive agents, metals, heat or light.<br>
Detailed description of the invention<br>
Library of test compounds<br>
In the present invention, libraries of compounds are used to screen for compounds having a desired physiological influence on a living cell. As used herein, the term "library" means a collection of molecular entities or test compounds, herein also designated "library members" obtained after a series of chemical transformation.<br><br>
In preferred embodiments of the present invention the library is a combinatorial library. Non-limiting examples of combinatorial libraries that may be used with the present invention and methods of producing such libraries are given in: Comprehensive Survey of Combinatorial Library Synthesis: 1998 Roland E. Dolle and Kingsley H. Nelson, Jr. J. Comb. Chem., 1999, pp 235 - 282; Comprehensive Survey of Combinatorial Library Synthesis: 1999 Roland E. Dolle J. Comb. Chem., 2000, pp 383 - 433; Comprehensive Survey of Combinatorial Library Synthesis: 2000 Roland E. Dolle J. Comb. Chem.,2001, pp 477 - 517; Comprehensive Survey of Combinatorial Library Synthesis: 2001 Roland E. Dolle J. Comb. Chem.,2002, pp 369 - 418 and Comprehensive Survey of Combinatorial Library Synthesis: 2002 Roland E. Dolle J. Comb. Chem.,2003, pp 693 - 753. The skilled person will appreciate that these protocols may be easily be adapted to specific need of a particular embodiment of the present invention.<br>
In one embodiment, these molecular entities can be natural oligomers (oligomers of building blocks occurring in Nature) such as peptides, glycopeptides, lipopeptides, nucleic acids (DNA or RNA), or oligosaccharides. By way of example, a natural oli-gomer may be any peptide consisting of naturally occurring amino acid, even if said peptide comprises a sequence not present in nature. The libraries may comprise different natural oligomers or the libraries may comprise only one kind of natural oligomer, for example the library may be a peptide library. In another embodiment, they can be unnatural oligomers (oligomers comprising one or more building blocks not occurring in Nature) such as chemically modified peptides, glycopeptides, nucleic acids (DNA or RNA), or, oligosaccharides, and the like. Said chemical modification may for example be the use of unnatural building blocks connected by the natural bond linking the units (for example, a peptide amide linkage), the use of natural building blocks with modified linking units (for example, oligoureas as discussed in Boeijen et al, 2001, J. Org. Chem., 66: 8454-8462; oligosulfonamides as discussed in Monnee et al, 2000, Tetrahedron Lett., 41: 7991-95), or combinations of these (for example, statine amides as discussed in Dolle et al, 2000, J. Comb. Chem., 2: 716-31.). Preferred unnatural oligomers include oligomers comprising unnatural building blocks connected to each other by a naturally occurring bond linking. Said oligomers may thus comprise a mixture of naturally occurring and unnatural building blocks linked to each other by naturally occurring bonds. By way of example, the oligomer may comprise naturally occurring amino acids and unnatural<br><br>
building blocks linked by peptide bonds f.x. PNA or LNA.Thus, in one embodiment of the invention preferred oligomers comprise modified amino acids or amino acid (mimics). Other preferred unnatural oligomers include, for example oligoureas, poly azatides, aromatic C-C linked oligomers and aromatic C-N linked oligomers. Still other preferred oligomers comprise a mixture of natural and unnatural building blocks and natural and unnatural linking bonds. For example, the unnatural oli-gomer may be any of the oligomers mentioned in recent reviews see: Graven et al., 2001, J. Comb. Chem., 3: 441-52; St. Hilaire etal., 2000, Angew. Chem. Int. Ed. Engl., 39: 1162-79; James, 2001, Curr. Opin. Pharmacol., 1: 540-6; Marcaurelle et al., 2002, Curr. Opin. Chem. Biol., 6: 289-96; Breinbaueret al., 2002, Angew. Chem. Int. Ed. Engl., 41: 2879-90. The libraries of the invention may also comprise cyclic oligomers, for example cyclic natural oligomers, such as cyclic peptides or cyclic unnatural oligomers. In certain embodiments of the invention, libraries of cyclic oligomers may be advantegous to use due to the rigid structure. This may result in higher selectively and affinity.<br>
In yet another embodiment, the molecular entities may comprise non-oligomeric molecules such as peptidomimetics or other small organic molecules. Peptidomi-metics are compounds that mimic the action of a peptidic messenger, such as bi-cyclic thiazolidine lactam peptidomimetics of L-proplyl-L-leucyl-glycinamide (Khalil et al, 1999, J. Med. Chem., 42: 2977-87). In a preferred embodiment of the invention, the library comprises or even more preferably consists of small organic molecules. Small organic molecules are non-oligomeric compounds of less than about 600 mass units containing any of a variety of possible functional groups and are the product of chemical synthesis, or isolated from nature, or isolated from nature and then chemically modified, and include, for example, Bayer's urea-based kinase inhibitors (Smith et al., 2001, Bioorg. Med. Chem. Lett., 11: 2775-78). Small organic compounds may for example be selected from the group consisting of alcohols, ethers, carboxylic acids, aryloxy, acyloxy, thiol, alkylthio, arylthio, heteroarylthio, sulphonyl, sulphoxy, amino, alkylamino, dialkylamino, acylamino, diacylamino, alkoxycarbonylamino, amides, alkyl, branched alkyl, aryl, heteroaryl, nitro, cyano, halogeno, silyloxy, keto, heterocycles, fused ring systems, fused heterocycles and mixtures thereof, wherein each of the aforementioned may be substituted independently on each position with one or more groups selected from the group consisting of -H, -OH, -SH, halogen, carboxyl, carbonyl, alkoxy, aryloxy, acyloxy, alkylthio, aryl-<br><br>
thio, heteroarylthio, sulphonyl, sulphoxy, amino, alkylamino, dialkylamino, acylamino, diacylamino, alkoxycarbonylamino, amides, alkyl, aryl, heteroaryl, nitro, cyano, halogeno, silyloxy, keto, heterocycles, fused ring systems, and fused het-erocycles.<br>
Non-limiting examples of small organic molecule libraries that may be used with the present invention and methods of producing them may for example be found in the reviews Thompson et al., 1996, Chem. Rev., 96: 555-600; Al-Obeidi et al., 1998, Mol. Biotechnol., 9: 205-23; Nefzi etal., 2001, Biopolymers, 60: 212-9; Dolle, 2002, J. Comb. Chem., 4: 369-418.<br>
The libraries according to the invention may comprise at least 20, such as at least 100, for example at least 1000, such as at least 10,000, for example at least 100,000, such as at least 1,000,000 different test compounds. Preferably, the libraries comprises in the range of 20 to 107, more preferably 50 to 7,000,000, even more preferably 100 to 5,000,000, yet more preferably 250 to 2,000,000 different compounds, in a very preferred embodiment of the present invention the libraries comprises in the range of 1000 to 20,000 or for example in the range of 20,000 to 200,000 different test compounds.<br>
In preferred embodiments of the invention the library comprises in the range of 10,000 to 1,000,000 different test compounds.<br>
Preferably, the libraries to be used with the present invention are immobilised on resin beads. Said resin beads may be any of the beads described herein below. At least 2, preferably at least 20, more preferably at least 100, even more preferably at least 1000, yet more preferably at least 10,000, for example at least 100,000, such as at least 1,000,000 resin beads comprising different library members, i.e. different test compounds may be used with the methods according to the invention. Preferably, the in the range of 20 to 107, more preferably 100 to 7,000,000, even more preferably 1000 to 5,000,000, yet more preferably 5000 to 2,000,000, even more preferably 10,000 to 1,000,000 resin beads comprising different library members, are used with the methods according to the invention.<br>
In one very preferred embodiment of the invention, each resin bead does not comprise more than one library member in one or more copies, i.e. each resin bead only comprises one kind of test compound, however said test compound may be present<br><br>
on the resin bead in multiple copies. Such libraries may also be designated one-bead-one-compound libraries. Preferably, each resin beads comprises sufficient copies of said library member in order to exert the desired influence of cells attached to said resin bead and in order to analyse the chemical structure of the compound. Such libraries may be prepared by different methods, for example by a split/mix method or by coupling individually a specific compound to a bead. One-bead-one compound libraries offer the advantage that once a resin bead has been selected according to the methods described herein, the desired compound may easily be identified (see useful methods herein below).<br>
The libraries may in one preferred embodiment be synthesized directly on resin beads using a split/mix method (vide infra) which gives rise to one-bead-one-compound libraries. Split/mix methods in general comprise the steps of:<br>
1.	Providing several pools of resin beads<br>
2.	Performing one or more different chemical synthesis steps on each pool of resin<br>
beads<br>
3.	Mixing pools of resin beads, thereby obtaining a mixed pool.<br>
4.	Splitting the mixed pool of resin beads thereby obtaining new pools.<br>
5.	Optionally repeating step 1 and 4<br>
Alternatively steps 3 and 4 may be as follows:<br>
3.	Splitting said pools to obtain fractions<br>
4.	Mixing fractions from different pools, thereby obtaining new pools<br>
One-bead-one-compound libraries may for example be prepared as described in M. Meldal, Multiple column synthesis of quenched solid-phase bound fluorogenic substrates for characterization of endoprotease specificity in Methods: A Companion to Methods in Enzymology 6:417-424, 1994 or in M. Meldal, The One-bead Two-Compound Assay for Solid Phase Screening of Combinatorial Libraries in Biopoly-mers, Peptide Science 66:93-100, 2002; or in Combinatorial peptide library protocols, Ed. by Shmuel Cabilly, Humana Press, 1998, p. 1-24 and 51 to 82.<br>
In one embodiment of the invention the library may be a library of oligomers of amino acids immobilised on resin beads, wherein said amino acids may be naturally occurring amino acids or any other kind of amino acid or a mixture of both. In gen-<br><br>
eral, oligomers of amino acids are referred to as peptides, regardless whether they comprise naturally occuring or not naturally occurring amino acids or a mixture, unless it is clear from the context that the term only cover oligomers of naturally occurring amino acids. For example, the oligomers may comprise or consist of at least 3. such as at least 4, for example at least 5, such as at least 6, for example at least 7, such as at least 10, for example at least 20, such as in the range of 3 to 5, for example in the range of 4 to 10, such as in the range of 5 to 15, for example in the range of 10 to 20 amino acids, such as in the range of 15 to 30, for example in the range of 3 to 50, such as in the range of 3 to 25, for example in the range of 4 to 15 amino acids. Such libraries may for example be prepared as described in example 1 herein below.<br>
The library of oligomers of amino acids may in one embodiment comprise an appended sequence. Thus, every library member may share a common sequence, i.e. the appended sequence. In addition every library member comprises an individual sequence. The appended sequence may be a sequence of amino acids, for example in the range of 3 to 5, for example in the range of 4 to 10, such as in the range of 5 to 15, for example in the range of 10 to 20 amino acids, such as in the range of 15 to 30, for example in the range of 3 to 50, such as in the range of 3 to 25, for example in the range of 4 to 15 amino acids.<br>
In another embodiment of the invention the library may be a one-bead-two-compounds library. Each individual resin bead of such a library comprises only one library member in one or more copies. In addition each individual resin bead comprises a second compound, such as a cell adhesion compound. The cell adhesion compound could for example be any of the cell adhesion compounds mentioned herein below. It is comprised within the invention that several library resin beads, such as all library resin beads comprises identical adhesion compound(s) in one or more copies.   One-bead-two-compound libraries may for example be prepared by a method involving the steps of:<br>
1.	Providing resin beads comprising a plurality of reactive groups<br>
2.	Reacting said reactive groups with two chemical moeities comprising different<br>
and preferably orthogonal protective groups<br>
3.	Deprotecting a subset of the reactive groups by removal of one kind of protective<br>
groups, preferably selective removal of one kind of protective group,<br><br>
4.	Attaching or synthezising a split/mix library of test compounds to the deprotected<br>
reactive group<br>
5.	Deprotecting the remaining reactive groups by removal the other kind of protec<br>
tive group<br>
6.	Attaching the second compound to the deprotected reactive groups<br>
The method may also be performed by first attaching the second compound and then synthezising the library. Accordingly, the steps of the method may be performed in the following order: 1, 2, 3, 6, 5 and 4. The library of test compounds may<br>
s<br>
be first synthesized and then attached to the resin beads or it may be synthesized directly into the resin bead. Similarly, the second compound may be first synthesized and then attached to the resin beads or it may be synthesized directly into the resin bead.<br>
Preferred resin beads are described in the section "resin beads" herein below. The reactive group may be any suitable reactive group, preferably however, the reactive group is either a hydroxyl group, a thiol or a primary amino group. The reactive group may also preferably be an azido or a secondary amino group. The protective group may be any suitable protective group known to the person skilled in the art, such as acid labile, alkaline labile, fluoride labile, oxidation labile, reduction labile or photolabile protective groups, preferably the protective group is selected from the group consisting of Fmoc, Boc, Alloc and N3. It is preferred that the different protective groups may be removed by different treatment, for example that if one protective group is acid labile, then the other is not acid labile, but instead for example alkaline labile or photo labile. In an preferred embodiment one protective group is Fmoc and the other protective group is Alloc or N3. Step 3 may for example be performed by a split/mix method as described herein above, thereby generating a one-bead-one-compound library. The second compound is preferably a cell adhesion compound.<br>
In yet another embodiment of the invention the library may be a mixed compound library, wherein each individual resin bead comprises a plurality of library members.<br>
Selection of an appropriate library is dependent upon the specific embodiment of the invention.   For example, a totally random library designed to contain interesting and<br><br>
greatly diverse compounds may be used with the invention.   An advantage of this approach is that the outcome of the screening is not prejudiced in any specific manner or at least less prejudiced. Since the invention permits screening of millions of diverse compounds, for example, immobilized on resin beads, a large number, for example in the range of 3 to 5 million, of random molecules can be used in the li-gand library.<br>
Alternatively, a smaller, targeted library (hundreds to thousands of compounds) can be used, for example, starting with a known compound or compounds, and providing numerous variations of these known compounds for targeted screening. For example, in embodiments of the invention wherein compounds modulating the activity of a specific cell surface molecule, a compound known to modulate said specific cell surface molecule may be used as starting compound for the preparation of a targeted library. Alternatively, a smaller targeted library of compounds mimicking a compound known to modulate the activity of said cell surface molecule may be prepared, for example using computer aided modelling followed by chemical synthesis. The smaller, targeted library can also comprise random molecules. Examples of libraries and methods of preparing such libraries, which may useful in embodiments of the invention, wherein the cellular response is mediated through an intracellular signalling pathway are known to the skilled person. The library may contain a parallel array of random modifications of one or more test compounds. In one embodiment, the library may be formed as a parallel array of random modifications to a known compound or compounds. The term "parallel array" is meant to cover synthesis of a library by subjecting a given compound to a known set of reactions in an isolated vessel or well. Thus, the nature of a compound in a given container or well is known. The array of test compounds is preferably prepared directly on resin beads using techniques known by those skilled in the art. Briefly, the resin may be portioned into a number of vessels or wells, usually less than 500 and the reagents added. There is in general no mixing step and after the appropriate washing steps, subsequent reactions are carried out by addition of additional reagents to the wells. There is no exponential increase in the number of compounds generated and that is equal to the number of vessels used. The compound can be easily identified by keeping track of the reagent added to each well.<br><br>
The library may also have been prepared by parallel synthesis using a tag to enable identification of, what chemical synthesis steps the individual resin bead has been submitted to. This may for example be done by IRORI or radiofrequency tag. Alternatively, chemical synthesis steps may be performed in parallel to preparing a polymeric tag. Identification of the tag will thus provide knowledge of the compound.<br>
Attachment of a label to a compound may alter the properties of said compound. Hence, in one embodiment of the present invention, the compounds of the library are not labelled, i.e. the compounds are not connected to a detectable label, such as a fluorescent component, a nucleic acid or a nucleic acid homologue such as PNA, a dye, a probe comprising a reactive moiety or the like. In particular it is preferred that all compounds are not connected to the same detectable label.<br>
The peptides used for preparation of any of the libraries mentioned above may be oligomers of naturally occurring or not naturally occurring amino acids or a mixture of both, preferably they are oligomers of the 20 amino acids naturally present in proteins, wherein said amino acids may be in either D- or L-form. It is preferred that each peptide (or peptide mimetic) is immobilised on a solid support, such as any of the solid supports mentioned herein below. More preferably the solid support is resin beads and it is preferred that each resin bead comprises only one library member in one or more copies.<br>
Preferably at least 2, such as at least 10, for example at least 100, such as at least 1000, for example at least 10.000 different peptides and/or peptide mimetics are provided. Each peptide may comprise in the range of 2 to 100 amino acids, such as in the range of 2 to 50 amino acids, for example 2 to 25 amino acids, such as in the range of 2 to 15 amino acids, for example 2 to 10 amino acids, such as in the range of 3 to 8 amino acids, for example 4 to 6 amino acids,<br>
The invention also relates to libraries prepared by any of the methods described above.<br>
Libraries of heterocyclic compounds obtained by cyclisation of a peptide aldehyde through an intramolecular Pictet-Spengler reaction may also be used with the present invention. Such libraries may for example be any of the libraries described in<br><br>
Danish patent application PA 2003 00967, which is hereby incoporated by reference.<br>
In one embodiment of the invention the library comprises oligomers of amino acids, such as oligomers of 3 to 10 amino acids, such as oligomers of 3 to 6 amino acids, for example oligomers of 3 to 4 amino acids, wherein the oligomer optionally may comprise additional moieties, such as 1 to 4, for example 1 to 3, such as 1to 2, for example 1 additional moiety, which is not an amino acid. The additional moiety may be an acyl moiety, such as an acyl halide. The additional moiety is preferably situated at one end of the oligomer, preferably at the C-termlnus. The amino acids may be either naturally occurring or not naturally occurring amino acids or a mixture of both. They are preferably linked via peptide bonds. In one embodiment the library comprising compounds consisting of 3 amino acids linked via peptide bonds and an additional moeity which may be an amino acid or for example an acyl halid. Such libraries may for example be useful for identifying compounds capable of interacting with inhibitors of apotosis, such as ML-IAP or XIAP. Examples of such libraries and methods of preparing such libraries are described herein below in Examples 4, 5a, 5b and 5c. In one embodiment libraries prepared essentially as described in examples 5b or 5c are preferred.<br>
In one embodiment the library may comprise or essentially consist of oligomers of amino acids, such as oligomers of in the range of 3 to 6 amino acids, for example oligomers of 4 amino acids, wherein the amino acids are selected from the group of amino acids mentioned in table 3, table 4, table 5 and table 6. A preferred example of such a library is given in example 4.<br>
Resin beads<br>
The library members of this invention are preferably bound to a solid support. Preferred solid supports to be used with the present invention are resin beads (see herein below).<br>
The solid support may however also be a spot or region on a surface or a plated gel or a membrane. A spot or a region is a defined area on said surface, to which the<br><br>
library member is bound via a cleavable linker. One can therefore envisage one surface comprising a plurality of spots or regions, wherein each such spot or region is covalently attached to only one library member in one or more copies. Said surface could for example be a silicium wafer, a glass surface, a plastic surface or a gel. Plastic surface may for example be prepared from polystyrene, polycarbonate polypropylene, ethylene and/or teflon. Gels could be prepared from for example poly acrylamid or PEGA.<br>
In this invention however, the compounds of the library are preferably bound to a resin bead, conferring the advantage of compartmentalized "mini-reaction vessels" for attachment of cells.<br>
In general more compounds may be screened and several of the steps in the procedure may be performed on one bead with sufficient material. Hence, preferably, the library is bound to resin beads. Each member of the library is a unique compound and is physically separated in space from the other compounds in the library, preferably, by immobilizing the library on resin beads, wherein each bead at the most comprises one member of the library. Depending on the mode of library synthesis, each library member may contain, in addition, fragments of the library member. Since ease and speed are important features of this process invention, it is preferred that the screening step take place on the same solid support used for synthesis of the library, and also that identification of the members of the binding pair can take place on the same support, such as on a single resin bead. Thus, preferred solid supports useful in the process invention satisfy the criteria of not only being suitable for organic synthesis, but are also suitable for screening procedures, such as "on-bead" screening as well as suitable for attachment of cells. It is furthermore preferred that the resin bead is suitable for "on-bead" identification of library members as described herein below. The resin bead may be prepared from any suitable material such as polystyrene, polyethylene, polyacrylamide, controlled pore glass or PEG. The resin bead could thus for example be selected from the group consisting of Toyopearl, sepharose, sephadex, CPG, silica, POPOP, PEGA, SPOCC, Expan-sin, Tentagel, Argogel, Polystyrene, Jandagel, polydimethylacrylamide resin, Polyacrylamide resin, kieselghur supported resins and polystyrene supported resins. Hydrophilic supports are preferred. Examples of preferred hydrophilic resin beads includes TentaGel (commercially available from Rapp polymere, Tubingen, Ger-<br><br>
many), ArgoGel (commercially available from Argonaut Technologies Inc., San Carlos, CA), PEGA (commercially available from VersaMatrix, Copenhagen), POEPOP (Renil et al., 1996, Tetrahedron Lett., 37: 6185-88; available from Versamatrix, Copenhagen, Denmark) and SPOCC (Rademann et al, 1999, J. Am. Chem. Soc., 121: 5459-66; available from Versamatrix, Copenhagen, Denmark). Examples of on-bead screening attempts are described in the following references: Chen et al., 1996, Methods Enzymol., 267: 211-19; Leon et al., 1998, Bioorg. Med. Chem. Lett., 8: 2997-3002; St. Hilaire et al., 1999, J. Comb. Chem., 1: 509-23; Smith et al., 1999, J. Comb. Chem., 1: 326-32; Graven et al., 2001, J. Comb. Chem. 3: 441-52; Park et al., 2002, Lett. Peptide Sci., 8: 171-78). TentaGel and ArgoGel are made up of polyethylene glycol chains grafted on to a polystyrene core. However, use of these supports in biological screening is limited by a size restriction, and by denaturation of certain proteins, particularly enzymes.<br>
Preferred resin beads according to the present invention are resin beads, useful for on-bead library synthesis, screening and identification of ligand/protein. Hence, preferred resins according to the present invention are resin comprising polyethylene glycol. More preferably, the resin is PolyEthyleneGlycol Acrylamide copolymer (PEGA), Super Permeable Organic Combinatorial Chemistry (SPOCC) resin or PolyOxyEthylene-PolyOxyPropylene (POEPOP) resin. Another preferred resin comprises a crosslinked polyacrylamide resin.<br>
PEGA (PolyEthyleneGlycol Acrylamide copolymer; Meldal M., 1992, Tetrahedron Lett., 33: 3077-80), POEPOP (PolyOxyEthylene-PolyOxyPropylene resin; Renil et al., 1996, Tetrahedron Lett., 37: 6185-88) and SPOCC (Super Permeable Organic Combinatorial Chemistry resin; Rademann et al, 1999, J. Am. Chem. Soc., 121: 5459-66) resins are made primarily of polyethylene glycol and swell well in organic as well as aqueous solvents. Because they have very reduced or no non-specific binding, PEGA and SPOCC resins have been effectively used in the screening of myriad proteins including enzymes of different classes. Furthermore, these resins are available in different pore sizes and can allow large proteins to enter while retaining activity. For example, PEGA6000 resins allow proteins up to 600 kDa to enter. In the Examples below, PEGA4000 and PEGA1900 resin with a molecular weight cut off of 200 and 90 kDa, respectively, were used for screening. In principle, any hydrophilic support that is useful for compartmentalized synthesis, retains the<br><br>
activity of the proteins, and has minimal non-specific binding, may be used in this process invention.<br>
One aspect of the invention relates to a method comprising the step of providing multiple resin beads capable of supporting growth of cells. Preferably, all resin beads provided are capable of supporting growth of cells. In one preferred embodiment all resin beads are similar and each is capable of supporting growth of cells, wherein the resin beads only differs by comprising different library members. In embodiments of the invention wherein the resin beads comprise a cell adhesion molecule, it is preferred that at least 10%, more preferably at least 20%, even more preferably at least 30%, yet more preferably at least 40%, even more preferably at least 50%, yet more preferably at least 60%, %, even more preferably at least 70%, yet more preferably at least 90%, even more preferably essentially all, yet more preferably all resin beads comprise the cell adhesion molecule as well as a library member.<br>
Release of library compounds or of adhesion compound<br>
The library of test compounds is preferably linked to the resin beads or solid supports by a cleavable linker. Hence in one embodiment of the invention, a proportion of the library members may be released from the resin beads, preferably by cleaving the cleavable linker. The thus released library members may then interact with cells in the immediate proximity, i.e. normally with cells attached to the same bead, and it is even possible that the library member may enter the cells and interact with intra-cellular compounds. Later selection of a single bead allows elucidation of the structure of the specific library member remaining attached to said bead.<br>
Preferably, "releasing a proportion of a library member" means releasing one or more copies of the library member attached to a solid support or resin bead. Preferably, said copies of the library member are released by cleaving the cleavable linker. Preferably, in the range of 5 to 95% of all copies of a library member attached to a resin bead are released, more preferably in the range of 10 to 90%, even more preferably in the range of 20 to 80%, such as in the range of 30 to 70%, for example in the range of 40 to 60%, such as at least 5%, for example at least 10%, such as at<br><br>
least 20%, such as at least 30%, for example at least 40%, such as at least 50%, for example at least 60%, such as at least 70%, for example at least 80%, such as at least 90%, for example at least 95%, such as at the most 5%, for example at the most 10%, such as at the most 20%, such as at the most 30%, for example at the most 40%, such as at the most 50%, for example at the most 60%, such as at the most 70%, for example at the most 80%, such as at the most 90%, for example at the most 95% of all copies of a library member attached to a resin bead are released.<br>
It is also comprised within the invention that the adhesion compound may be attached to the resin bead via a cleavable linker. Cleavage of said cleavable linker may release the adhesion compound as well as cells attached to said adhesion compound. When the cleavable linkers linking the library compound and the adhesion compound, respectively, are differentially cleavable, then selective release of either library compound or adhesion compound may be achieved.<br>
The cleavable linker may be any chemical moiety which may be used to attach any molecule to a solid support either covalently or via complex formation, and thereafter release said molecule by the action of either acid, base, electrophiles, nucleo-philes, oxidative agents, reductive agents, metals or light. Preferably, the cleavable linker attaches the library member to the solid support covalently. Depending on the nature of the cleavable linker, a person skilled in the art will be capable of controlling cleavage of the cleavable linker, so only a proportion of the copies of a library member are released. A comprehensive review describing state of the art for "cleavable linkers" is "Linkers and Cleavage Strategies in Solid-Phase Organic Synthesis and Combinatorial Chemistry", F. Guiltier, D. Grain, and M. Bradley, Chem. Rev. 2000, 100, 2091-2157. Any of the cleavable linkers described therein may be used with the present invention.<br>
Examples of useful acid labile linkers include the most commonly used linkers for acidic detachment from a solid support, the Wang and Rink linkers (Fig. 4A and B). Detachment of peptide esters from Wang linkers require up to 95% TFA in DCM whereas detachment of Rink esters (Fig. 4B, X=OH) may be cleaved under milder conditions (AcOH - DCM 1:9) which does not cleave the normal protection groups on the peptides. The Rink amides (Fig. 4A, X=NH2) require 95% TFA (aq). Partial<br><br>
detachment of the compounds attached to the resin may also be achieved using gaseous acids such as HCL or TFA vapour in a sealed container. The use of gases allow rigorous control of the degree of cleavage obtained with concentration of acid and time of exposure. The skilled person may readily establish a suitable concentration of acid and time of exposure to obtain a desried degree of cleavage.<br>
Examples of useful base-labile linkers includesWang and HMBA linkers, which may be cleaved under alkaline conditions. Saponification with 0.1 M NaOH may be applied but even milder conditions such as potassium carbonate in MeOH are applicable. The HMBA linker is stable to TFA under normal conditions.<br>
In a preferred embodiment the cleavable linker is a light sensitive cleavable linker which, upon the action of light with a given wave length and intensity, may release any active compound from the solid support.<br>
Photo-labile linkers provide a tool for solid phase synthesis which enables the detachment of the synthesized molecules in the prescence of acid or base-sensitive functionalities within the molecules. In 1973 Rich proposed the use of o-nitrobenzyl type of linkers (nitrated analogs of the Wang linker). Irradiation with UV-light gave detachment of the free acids or amides although only in moderate yields. Detachment yields could be improved by applying the NBA type linkers (see Fig. 4E). Even better result have been obtained with the Holmes-type linkers (Fig. 4F). Detachment from photolabile linkers is performed by illuminating the resins with ultraviolet light, preferable at 365 nm. The wave length and intensity of the light and the time of exposure might need optimization for the individual case. A person skilled in the art can readily establish conditions wherein a desired proportion of copies of a library member are released. Detachment yields may be over 90 % under ideal conditions.<br>
Example of a preferred photo sensitive cleavable linker:<br>
(Figure Remove) <br>
X = NH, O ; R * H, alkyl ; R1 = linked library member<br>
Depending on the nature of the cleavable linker, the library member may be released using different methods. For example, if the linker is photo labile, the library member may be released by illumination. The release should preferably be partial, so that only a proportion of the library member is released. The person skilled in the art will readily be able to establish the conditions required for partial release using a specific cleavable linker. An example of how to acchieve partial release is given in example 6 herein below.<br>
It is also comprised within the present invention that a library member may be linked to a resin bead via different cleavable linker. For example some copies of a library member may be linked to a resin bead via a first kind of cleavable linker, whereas other copies of the same library member may be linked to the resin bead via a second kind of cleavable linker. Preferably, the first kind of cleavable linker is cleavable by another method than the second kind of cleavable linker. By way of example, the first cleavable linker could be acid or base labile, whereas the second kind of cleavable linker could be photolabile. Thus, some copies of the library member could be released during the screening procedure of the invention, for example during step c) in the method outlined in the "Summary" herein above, whereas other copies could be retained on the resin beads and released during the identification, for example during step f) in the method outlined in the "Summary" herein above. Thus, releasing a proportion of a library member could be controlled by using different cleavable linkers.<br>
Cells<br>
The cells to be used with the present invention may be any useful cells available or prepared for the purpose. Preferably, the cells are selected from the group consist-<br>
ing of mammalian cells. For example the cells may be human cells. The cells may be cells capable of growing in suspension or they may be adherent cells. Adherent cells may preferably be cultivated directly on the resin beads used with the invention (see also herein below). It is preferred that the cells are adherent cells. Cells with a better adherence are preferred over cells with a poorer adherence. Cells which adhere well to resin beads comprising an adhesion compound as described herein above are very preferred.<br>
Cells could for example be primary cells or established cell lines. Preferred cell lines include but are not limited to those mentioned in table 1.<br>
(Table Remove)<br>
In one embodiment of the invention the cells have been genetically or otherwise modified in order to enhance their usability with the present invention. The modification may be stable or only transient or a mixture of both. For example, the cells may have been modified to contain one or more of the reporter systems described herein below. Depending on the nature of the reporter system this may be achieved by a number of different methods. For example, if the reporter system comprises a nucleic acid, said nucleic acid may be inserted into said cell by conventional recombi-nant techniques (see below).<br>
In another preferred example the cell comprises nucleic acid comprising first nucleo-tide sequences encoding cellular proteins or polypeptides being part of an intracellu-lar signal transduction pathway operationally linked to a reporter system detecting the enzymatic activity or subcellular localization of said first sequences, or detecting direct interactions between these first sequences.<br>
Useful second sequences includes for example promoters active in the particular cells, for example mammalian promoters, viral promoters or synthetic promoters. A large number of useful eukaryotic promoters are known to the person skilled in the art, useful promoters are for example described in"Mechanism of Transcription" (1998) Cold Spring Harbor Symposia on Quantitative Biology Vol. LXIII; Cold Spring Harbor Laboratory Press<br>
Such promoters may be constitutively active or they may be active only temporarily. In one example the promoter may be regulated by an external signal, for example the promoter may be inducible or repressable.<br>
The nucleic acid may be inserted into the cells by any useful method, for example by conventional recombinant techniques, such as any of the techniques described in Sambrook et al., Molecular Cloning: A Laboratory Manual, 1989, Cold Spring Harbor Laboratory, New York, USA<br>
In another embodiment the cells are primary cells. Primary cells are cells with a limited life span that preferably are derived from a mammalian tissue. Preferred primary cells are cells which are adherent. The mammalian tissue may for example be a human tissue, such as healthy or diseased tissue. In one embodiment the tissue is or comprises a neoplastic tissue, for example tissue removed from a cancer patient by surgery, for example from a patient suffering from melanoma, breast cancer or colon cancer. The tissue may also be hypertrophic cells, such as cardiac myocytes. Preferably said cancer patient has not been subjected to radiotherapy prior to surgery. In embodiments of the invention wherein the cells are primary cells it is preferred that the reporter system is endogenous to said primary cells.<br>
Cell attachment to resin beads and cell cultivation<br>
The present invention relates to methods comprising the step of attaching cells comprising a reporter system(s) to resin beads. The cells may for example attach to said resin beads directly or by attaching a second compound conferring adhesion to the resin bead.<br>
The resin beads useful for the present invention should preferably be able to support cell growth. The resin beads may per se be able to support cell growth, however frequently the resin beads will comprise a cell adhesion compound that enables the resin beads to support growth of cells. Said cell adhesion compound may be coupled to said resin beads by any useful means known to the person skilled in the art depending on the nature of the cell adhesion compound.<br>
Any cell adhesion compound known to the person skilled in the art may be used with the present invention. It is frequently an advantage if the cell adhesion compound comprises at least one positively charged moiety at neutral pH, more preferably the cell adhesion compound has a positive overall netcharge at neutral pH.<br>
In one preferred embodiment of the invention the cell adhesion compound comprises a peptide or a polypeptide, more preferably the cell adhesion compound consists of a peptide. Such peptides are herein also designated "adhesion peptides".<br>
Said peptide preferably consists of in the range of 3 to 100, preferably in the range of 3 to 75, more preferably in the range of 3 to 50, even more preferably in the range of 3 to 30, yet more preferably in the range of 3 to 25, even more preferably in the range of 3 to 20, yet more preferably in the range of 3 to 15, such as in the range of 3 to 10, for example in the range of 3 to 8, for example in the range of 6 to 7 amino acids. In general, it is sufficient if the peptide comprises at least 3 amino acids.<br>
It is preferred that the peptide comprises at least one amino acid selected from the group consisting of arginine and lysine, more preferably the peptide comprises at least 2 basic amino acids, such as 3 basic amino acids selected from the group consisting of Arg and Lys, even more preferably the peptide has an overall positive netcharge. In one preferred embodiment the peptide comprises the following sequence of 4 amino acids: basic-basic-lipophilic-basic. Basic amino acids may for example be selected from the group consisting of arginine and lysine, whereas the lipophilic amino acid may be selected from the group consisting of Gly, Ala, Val, Leu, lie, Phe, Trp, Pro and Met of either D or L-form. Preferably, the peptide comprise at least 1, preferably at least 2, more preferably at least 3, even more preferably at least 4 amino acid on the D-form, yet more preferably all amino acids are on the D-form. Preferably D-amino acids are used to enhance the metabolic stability but also L-amino acids may be used.<br>
Preferred examples of peptides useful as cell adhesion compounds are given in table 2 herein below:<br>
 (Table Remove)<br>
Furthermore, preferred peptide may be any of the peptides identified by any of SEQ ID: 1 to 70, preferably any of SEQ ID: 1 to 23 and 26 to 35, such as SEQ ID: 1 to 23, for example SEQ ID: 25 to 35, wherein 3 amino acids, preferably 2 amino acids,<br>
more preferably 1 amino acid have been substituted for another amino acid. Preferably, said substition is a conservative substitution, i.e. substition for an amino acid with similar characteristics. Said characteristic could for example be acidic/basic properties, polarity or lipophilicity. It is also comprised within the invention that the peptide may be a peptide of above mentioned size comprising any of the peptides identified by SEQ ID: 1 to 70. In particular, in order to immoblised the peptide on a resin bead it may be useful to synthesise the adhesion peptide on an amino acid immobilized on the resin bead, for example a Gly.<br>
In one embodiment the peptide is preferably selected from the group consisting of peptides identified by SEQ ID: 21 to 23 and 36 to 35, more preferably from the group consisting of 26 to 35, even more preferably SEQ ID:35. In another embodiment the peptide defined by SEQ ID:21 is preferred.<br>
In one embodiment of the invention it is preferred that the peptide has low or essentially no fluorescent properties. It is particularly preferred that the peptide has low or essentially no fluorescent properties when attached to a solid support, such as a resin bead. By "essentially no fluorescent properties" is meant that the peptide does not emit any detectable fluorescence. This is in particularly relevant for embodiments of the invention wherein the detectable output is fluorescence (see herein below). Preferred peptides to use with this embodiment of the invention may be selected from the group consisting of SEQ ID:26 to 35.<br>
Peptides useful as cell adhesion compounds may be identified using any suitable method. Said method may for example include the steps of<br>
i)   synthesizing or coupling a test peptide on to a resin bead;<br>
ii)   incubating said resin bead with cells under cell cultivation conditions;<br>
iii) testing whether said cells attach to said resin bead<br>
iv) identification of the peptide sequence<br>
wherein the test peptide is useful as cell adhesion compound if more cells attach to said resin bead in the presence, than in the absence of said test peptide. Preferably, the test peptide is useful as cell adhesion compound if at least 200, more preferably at least 500, even more preferably at least 1000 cells attach to said resin bead after incubation. This is in particular the case in embodiments of the invention, wherein<br>
the resin beads are PEGA beads. For example useful test peptides may be identified as described in example 1 herein below.<br>
In embodiments of the invention wherein it is preferred that the peptide has no or low fluorescence it is preferred that the method comprises an additional step performed at any point subsequent to step i), such as immediately subsequent to step i) prior to step ii). Said additional step comprises testing whether said peptide has fluorescent properties. This may for example be performed by sorting resin beads in a FABS or manually with the aid of a fluorescence microscope. If this is done prior to step ii) then only resin beads with no or low fluorescence properties are incubeted with cells, A non-limiting example of a useful method is described in example 2b.<br>
The peptide may be coupled to the resin bead by any useful method, for example by synthesising the peptide directly onto an amino functionalised resin bead using a standard Fmoc-protocot for peptide synthesis. Other protective groups may be used instead of Fmoc, for example Boc, N3 or Alloc. In one embodiment Alloc is the preferred protective group. It is preferred that different protecting groups are used for synthesis of the adhesion peptide or for library synthesis. The peptide may also be synthesised by anchoring an Fmoc amino acid to a hydroxyl functionalised resin bead, such as a hydroxy methyl benzole acid (HMBA) derivatised PEGA resin followed by peptide assembly using standard Fmoc technology as described in B. Blankemeyer-Menge, M. Nimtz, and R. Frank, An Efficient method for ancoring Fmoc-amino acids to hydroxyl- functionalised solid supports. Tetrahedron Lett. 31:1701-1704,1990 and A. Dryland and R. C. Sheppard. Peptide synthesis. Part 11. A system for continuous flow solid phase peptide synthesis using fluorenylmethoxycarbonyl-amino acid pentafluorophenyl esters. Tetrahedron 44(3):859-876, 1988 Sidechains may be protected with acid labile protecting groups such as t-Bu, Trt, Pmc, Boc etc. The protected peptide may for example be cleaved off the resin using alkaline conditions or hydrazine and the structure may be determined e.g. by on bead Edman Degradtion. The HMBA-linked peptide may also be cleaved under mild alkaline condition.<br>
In one embodiment the peptide may be linked to the resin bead via a linker, which may be a cleavable linker. This may for example be achieved by synthesizing the<br>
linker directly on resin beads or coupling the linker to the resin beads and subsequently coupling or synthesizing the library onto the resin beads. Thus, before coupling of the library the linker preferably comprises a protective group as described herein above. The cleavable linker may be any of the cleavable linkers described herein above. If the resin beads are coupled to the library via a cieavable linker it is preferred that the cleavable linker linking the adhesion compound is differentially cleavavble.<br>
In embodiments wherein cells adhere to the resin bead via the adhesion compound and the adhesion compound is attached to the resin bead via a cleavable linker, cells may be at least partially or even essentially fully released from the resin bead by cleavage of the cleavable linker.<br>
Testing whether cells attach to resin beads may be done by any conventional methods, such as by manual inspection with the aid of a light microscope. If the cells have fluorescent properties, for example if the cells express a fluorescent protein, then resin beads with attached cells may be identified using a fluorescent microscope or a FABS, preferably a fluorescent microscope.<br>
In one preferred embodiment of the invention, the cells may be cultivated directly on the resin beads. In general, a method of cultivating cells on resin beads may comprise the steps of<br>
•	Providing resin beads capable of supporting growth of cells<br>
•	Seeding cells onto said resin bead<br>
•	Incubating said resin beads comprising said cells in a cell culture medium under<br>
cell cultivation conditions<br>
•	Optionally allowing said cells to divide on said resin bead<br>
•	Thereby cultivating cells on resin beads<br>
The cells may adhere actively to the resin beads and will then generally be referred to as adherent cells.<br>
Cells cultivation conditions depends on the specific cells. For a large number of mammalian cells, such conditions comprise high humidity, preferably close to 100%, approximately 5% CO2 and around 37°C. It is often desirable to keep the resin beads immersed in a suitable cultivation medium and frequently it is also desirable<br>
that the resin beads can be circulated within said medium, for example by stirring or rotation. Said stirring or rotation may be continuous or in intervals. It is also possible that the container comprising the resin beads is simply rocked gently a few times every now and then.<br>
In one embodiment of the invention more than one cell line or type of primary cell is attached to or cultivated on the beads. Hence for example 2, such as 3, for example 4, such as 5, for example 6, such as 7, for example 8, such as 9, for example 10, such as in the range of 10 to 20, for example in the range of 20 to 50, such as more than 50 different cell lines may be attached to or cultivated on said beads. Also different specific primary cells may be attached to the cultivated beads.<br>
It is possible that a subgroup of resin beads only comprise one cell line or a specific kind of primary cells and another subgroup of resin beads comprises another cell line or another specific kind of primary cell and so forth. However, it is also possible that in principle every resin beads comprises all the different cell lines and/or different primary cells.<br>
Intermediates between these two extremes may also be envisaged. Preferably, said different cell lines and/or primary cells comprise different reporter systems, hence it is possible that the different cell lines are derived from the same parent cell lined by insertion of different reporter systems. However, the different cell lines and/or primary cells may also be unrelated.<br>
Cellular molecules<br>
In one particularly preferred embodiment of the invention the methods of the invention involve identification of compounds modulating a cellular response, which is mediated through an interaction between cellular molecules, more preferably through an interaction between intracellular molecules. Cellular molecules may be any cellular molecule, such as proteins, polypeptides, DNA, RNA, molecules of non-protein nature or metal-ions. In preferred embodiment, the cellular molecule is a protein or polypeptide. Intracellular molecules are molecules that are not accessible from the extracellular surface of intact cells. Intracellular molecules may for example<br>
be proteins, polypeptides, DNA, RNA, molecules of non-protein nature or metal-ions. In one preferred embodiment, intracellullar molecules are proteins or polypeptides not accessible from the extracellular surface of intact cells.<br>
In one embodiment, the cellular response is mediated through an interaction between cellular molecules of a cellular signal transduction pathway. Hence, the invention, for example may be useful for identifying compounds modulating the activity of a signal transduction pathway. Such compounds could for example activate or repress a signal transduction pathways by<br>
•	modulating the interaction between different or the same cellular molecules,<br>
•	modulating the catalytic activity of enzymes,<br>
•	modulating the synthesis or degradation of cellular molecules,<br>
•	modulating transcriptional activity,<br>
•	modulating the localization or movement of cellular molecules.<br>
•	modulating the level of cellular molecules (i.e. in a specific cellular<br>
compartment or on average throughout the whole cell)<br>
Within the context of the present invention the term "signal transduction pathway" should be understood in its common cell biological meaning, i.e. modulation of an intracellular event triggered by a cell surface receptor.<br>
Signal transduction pathways may for example involve steps of changed catalytic activity of enzymes, phosphorylation, cleavage of proteins, synthesis of cAMP, activation of transcription, inhibition of transcription, change i intracellular Ca2* concentration, change in membrane potential, subcellular relocalisation of cellular components, complex formation of cellular components, degradation of cellular components and/or change in energy metabolism<br>
The signal transduction pathway could for example be a pathway activated/repressed by a cell surface receptor selected from the group consisting of G-protein coupled receptors (GPCR), protein kinase coupled receptors, receptor ki-nases with intrinsic kinase activity, orphan receptors or transmembrane channels. The signal transduction pathway may also be a pathway resulting in modulation of transcription, for example modulation of transcription regulated by a response element, for example a response element selected from the group consisting of of<br>
CRE, SRE, TRE and AP-1 In one embodiment of the invention the signal transduc-tion pathway is a pathway resulting in apoptosis.<br>
Preferably the cellular molecules are proteins or parts thereof or derivatives thereof, more preferably the cellular molecules are proteins. Even more preferably the cellular molecules belong to the classes of: serine/threonine protein kinases; tyrosine protein kinases, protein phosphatases, phospholipid dependent serine/threonine protein kinases, calmodulin dependent serine/threonine protein kinases, mitogenac-tivated serine/threonine protein kinases, cycline dependent serine/threonine protein kinases, transcription factors, structural proteins, protein scaffolds, proteases, such as caspases, metallo-matrix-proteases, rennin, cathepsins, viral proteases, secreta-ses or ADAM family proteases, or hydrolases, nucleases, synthases, isomerases, polymerises, oxido-reductases, ATPases or GTPases.<br>
The cellular molecules are more preferably proteins that are known to participate in protein-protein interactions or complex formations. Such proteins can be selected from proteins listed in databases like BIND (Biomolecular interaction jNetwork Database) http:// bind.ca.<br>
In one embodiment of the invention the cellular molecules are involved in regulation of apoptosis. Thus the cellular molecules may be proteins or functional fragments thereof involved in regulation of apoptosis. Proteins involved in apoptosis includes for example caspases, inhibitors of apoptosis (lAPs) or Smac. Inhibitors of apoptosis may for example beXIAP. CIAP1/BIRC2. ML-IAP/BIRC7, DIAP1, DIAP2, OPIAP3, c!AP2, NAIP, Apollon or Survivin (see also Vaux and Silke, 2005, Nature Reviews, 6:287-297). Thus, in one example proteins involved in protein-protein interaction may be Smac and XIAP or ML-IAP or a Smac binding fragment thereof. Smac binding fragments of XIAP preferably comprises the BIR3 domain of XIAP, whereas Smac binding fragments of ML-IAP preferably comprises the BIR domain. The do-maine structure of lAPs is well described, see for example Wu, G., Chai, J., Suber, T. L, Wu, J.-W., Du, C., Wang, X., and Shi, Y. (2000) Nature 408,1008-1012; Matthew C. Franklin et a/., Biochemistry 2003, 42, 8223-8231; and Listen et al. Onco-gene. 2003 Nov 24; 22(53): 8568-80<br>
A cell surface receptor within the meaning of the present invention is preferably a protein, more preferably a protein that is accessible from the extracellular surface. Yet more preferably, the cell surface molecule is a cell surface protein receptor (herein also merely designated "receptor"). A "receptor" within the meaning of the present invention, is a molecule, which at least sometimes is localised at the cell surface and which is capable or associating with at least one ligand. The ligand binding site is accessible from the extracellular surface. Frequently, association with said ligand may alter the activity of the receptor.<br>
Cellular response<br>
The invention relates to methods of identifying compounds modulating, such as activating or inhibiting, a cellular response linked to a reporter system. The reporter system may be any of the reporter systems described herein below. The methods disclosed by the present invention may be used to identify compounds modifying any cellular response, which is or may be linked to a reporter system generating a detectable output. The person skilled in the art will appreciate that the specific methods disclosed herein may be adapted to any such cellular response. Below, non-limiting examples of cellular responses are described.<br>
In a particularly preferred embodiment of the invention, the cellular response is mediated through interaction between cellular molecules, such as intracellular molecules. The cellular molecules may for example be components of asignal transduction pathway, and thus the cellular response may be activation or repression of a signal transduction pathway. Hence, the cellular response may for example be modulation of a signal transduction pathway within a cell, such as modulation of a signal transduction pathway mediated by a cell surface molecule. By "activation of a receptor" is meant that the receptor is influenced in a manner that it activates downstream signalling events. Accordingly, the methods according to the present invention may be employed to identify activators or inhibitors of signal transduction pathways.<br>
Examples of modulations of signal transduction pathway includes:<br>
•	Upregulation or downregulation of the level of a member of the pathway<br>
•	Relocalisation of a member of the pathway<br>
•	Complex formation between members of the pathway or between members of<br>
the pathway with other cellular compounds<br>
•	Enhanced or reduced transcription from genes regulated by the pathway<br>
•	Modification by for example phosphorylation of a member of the pathway<br>
•	Activation or inhibition of an enzyme of the pathway<br>
•	Degradation of a cellular compounds due to upregulation or downregulation of<br>
the pathway<br>
•	Altered secretion of a compound<br>
•	Change in ion-flux<br>
•	Morphological changes<br>
•	Change in viability<br>
In a preferred embodiment the modulation of a signal transduction pathway can for example be monitored by measuring:<br>
•	the enzymatic activity of an enzyme being part of said signal transduction<br>
pathway<br>
•	the level of cyclic nucleotides, i.e. cAMP or cGMP<br>
•	the activity of transcription factors<br>
•	the level of specific proteins as quantified through standard proteomics<br>
techniques<br>
•	the level of inositol or lipid phosphates<br>
•	the level of phosphorylation of specific proteins as quantified through standard<br>
proteomics techniques<br>
•	the binding between two or more proteins or polypeptides<br>
•	the cellular localization of proteins or polypeptides<br>
The enzymatic activity could for example be the enzymatic activity of serine/threonine protein kinases or of tyrosine protein kinases or of protein phosphatases or of phospholipid dependent serine/threonine protein kinases or of calmodulin dependent serine/threonine protein kinases or of mitogenactivated serine/threonine protein kinases or of cycline dependent serine/threonine protein kinases or of proteases or of hydrolases or of nucleases or of synthases or of isomerases or of polymerises or of oxido-reductases or of ATPases or of GTPases.<br>
The cellular response may in one embodiment be modulation of transcriptional activity, such as activation or reduction of transcription of one or more genes. In particular, activation or reduction of transcription of genes regulated by a response element. Said response element could for example be selected from the group consisting of CRE. SRE, TRE and AP-1.<br>
Hence, the cellular response may also be an increased or decreased level of a particular mRNA within a cell.<br>
By the term "regulated by a response element" is meant that transcription is modulated by said response element, however other elements may also modulate transcription of said gene. By the term "activation of response element" is meant increased transcription of genes regulated by said response element.<br>
In another embodiment of the invention the cellular response is:<br>
•	change in the intracellular level of a compound; or<br>
•	change in the level of a compound within a specific cellular compartment, for<br>
example within the cytoplasm, in the golgi, in the endoplasmatic reticulumjn ly-<br>
sosomes, in endosomes or in the nucleus<br>
The compound may be any compound, preferably a naturally occurring compound. Frequently, the compound is a compound endogenous to the cell. The compound may thus for example be a salt, an ion, a nucleotide or a derivative thereof, a pep-tide, a saccharide, a lipid or a biomacromolecule. Biomacromolecules includes for example RNA such as mRNA, polypeptides and proteins. An example of an ion is Ca2+ and an example of a nucleotide derivative is cAMP.<br>
In yet another embodiment of the invention the cellular response is relocalisation of a compound. Relocalisation may for example be<br>
•	concentration of a compound otherwise dispersed in one or more specific loca<br>
tions<br>
•	relocalisation from one cellular compartment to another, for example relocalisa<br>
tion from the cellular membrane to the cytoplasma.<br>
•	relocalisation from one location within a compartment to another location within<br>
the same compartment<br>
•	internalisation of an extracellular compound<br>
The compound may be any compound, such as any of the compounds mentioned in the section above. In one preferred embodiment the compound, which is relocalised is a biomacromolecule, such as RNA, polypeptides or proteins. For example, the compound may be a cell surface receptor (receptor). The cellular response may thus be internalisation of said receptor or relocalisation of said receptor from the cellular membrane to the cytoplasma.<br>
In one embodiment of the invention the cellular response is change in the activity of a compound, such as an increase or a decrease in the activity of a compound. Said compound may for example be an enzyme. The cellular response may for example be induction of the activity of a caspase. Preferred caspases are Caspase 3 or 7.<br>
In another embodiment of the invention the cellular response is change in phos-phorylation of a compound.<br>
In another embodiment of the invention the cellular response is formation or disruption of a complex between compounds.<br>
In another embodiment of the invention the cellular response is change in the concentration of a compound.<br>
The cellular response may also be altered secretion of a compound, such as increased or decreased secretion of a compound. Said compound could for example be a biomacromolecule, such as a protein, a polypeptide, a peptide, a hormone, a cytokine, or the like.<br>
In another embodiment of the invention the cellular response is change in pH in an intracellular compartment, for example in the cytoplasm.<br>
In yet another embodiment the the cellular response is a change in a membrane potential, for example a change in membrane potential over the cell membrane or over the mitochondria membrane.<br>
In an even further embodiment of the invention the cellular response is change in morphology, such as change in size or shape. The cellular response may also be change in viability (e.g. apoptosis or necrosis), such as change in viability under specific conditions.<br>
In a preferred embodiment of the present invention the cellular response is change in interaction between two or more cellular molecules, preferably between two cellular molecules, such as establishment of an interaction between two or more cellular molecules or disruption of an interaction between two or more cellular molecules. Thus the cellular response may be formation of a complex or disruption of a complex. The cellular molecules may be any of the cellular molecules mentioned above, however, preferably the cellular molecules are proteins or fragments thereof.<br>
In one embodiment of the present invention the cellular response is induction or facilitation of apoptosis in living cells, such as induction or facilitation of apoptosis in tumour cells, preferably induction of apoptosis. The cellular response may also be induction or facilitation of apoptosis in cells that have undergone an apoptosis promoting treatment. The cellular response may also be induction or facilitation of apoptosis in cells that have undergone an apoptosis inhibiting treatment, In another embodiment of the invention the cellular response is inhibition or reduction of a pop-<br>
tosis, for example reduction of apoptosis in cells prone to undergo apoptosis or reduction in apoptosis in cells that have undergone an apoptosis promoting treatment.<br>
The apoptosis promoting treatment may be contacting cells with an inducer of apoptosis. The inducer of apopotosis may be any compound known to be capable of inducing apoptosis, for example the compound may be staurosporine (STS). Alternatively, the apoptosis promoting treatment may be illumination with radiation, such as with UV-light with a predetermined wave length and intensity.<br>
The methods according to the invention may also include identification of compounds modulating more than one cellular response, such as 2, for example 3, such as 4, for example 5, such as more than 5 different cellular responses. Said cellular responses may be any of the responses discussed above.<br>
Reporter system<br>
The reporter system to be used with the present invention should be selected according to the particular cellular response. The reporter system should be capable of generating a detectable output.<br>
In some embodiments of the invention the reporter system may be identical to the cellular response. This is in particular true when the cellular response may be detected without the aid of an additional reporter system, for example when the cellular response is an increase/decrease in the level of a compound, relocalisation of a compound, change in membrane potential, change in pH, change in morphology or the like.<br>
Hence, the reporter system may be a system endogenous to said cells. For example, the reporter system may comprise the endogenous system regulating the intracellular level of an endogenous compound. By way of example, the reporter system may be the endogenous system of a cell regulating the intracellular Ca2+ level.<br>
In another example, the reporter system comprises the intracellular localisation of an endogenous compound.<br>
In yet another example, the reporter system may comprise the activity of an enzyme. This may in particular be relevant when the cellular response is modulation of an enzymatic activity or modulation of a signal transduction pathway which modulates an enzymatic activity. Then the reporter system could be direct detection of for example the enzymatic activity of serine/threonine protein kinases or of tyrosine protein kinases or of protein phosphatases or of phospholipid dependent serine/threonine protein kinases or of calmodulin dependent serine/threonine protein kinases or of mitogenactivated serine/threonine protein kinases or of cycline dependent serine/threonine protein kinases or of proteases, such as caspases, metallo-matrix-proteases, rennin, cathepsins, viral proteases, secretases or ADAM family proteases or of hydrolases or of nucleases or of syntheses or of isomerases or of polymerises or of oxido-reductases or of ATPases or of GTPases.<br>
It is preferred that said enzymatic activity may be detected for example because the enzymatic activity leads to formation of a coloured compound, a fluorescent compound, a radioactive compound or the like. This may be acchieve by the use of appropriate substrates. If the enzyme is a protease, the enzymatic activity may for example be detected by use of a substrate, which generates a detectable output when cleaved by said protease. For example the substrate could be a peptide or a polypeptide comprising a fluorescent moeity and a quencher, wherein cleavage would lead to formation of two peptides/polypeptides, wherein one comprises the fluorescent moeity and the other comprises the quencher. Fluorescence would thus be detectable only in the presence of an active protease.<br>
In embodiments wherein the enzyme is a caspase, the reporter system may be a substrate for said caspase. Useful caspase substrates are known to the skilled person and several caspase substrates are commercially available, for example from Beckman Coulter Inc. Examples of Caspase substrates are Caspase 3 and/or 7 substrates. It is preferred that cleavage of the substrates is readily detectable. Thus fluorogenic substrates comprising a quenching group which may be cleaved of by caspases may be useful. Cleavage of such substrate can simply be detected by determining fluorescence. Non-limiting examples of fluorogenic caspase substrates are the CellProbe HT Caspase 3/7 Whole Cell Assay, Beckman Coulter, Inc. or any of the substrates described in US6,342,611. It is not required that such a reporter<br>
system is introduced permanently into living cells. Thus the substrate may be added directly to the cell culture medium or to an assay buffer comprising resin beads with cells. Thus such a reporter system may also be useful in embodiments of the invention wherein primary cells are employed. A non-limiting example of a useful Cas-pase assay is given in example 12 herein below.<br>
However, the reporter system may also be heterologous to the cell, i.e. a reporter system which has been inserted into the cell for example by recombinant techniques.<br>
In several embodiments of the invention the reporter system comprises a fusion protein comprising a first protein and detectable polypeptide. The detectable poly-peptide may for example be an enzyme or part thereof, a transcription factor or part thereof or a bioluminiscent protein, such as a fluorescent protein. Preferred detectable polypeptides are luciferase or fluorescent proteins. The first protein may be selected according to the cellular response. If the cellular response is change in level or location of a given protein, the first protein could be that particular protein. If the cellular response is change in interaction between two or more proteins, the first protein could be one of the proteins taking part in that interaction.<br>
In embodiments of the invention, wherein the cellular response is modulation of transcription from gene(s) regulated by a response element, it is preferred that the report system comprises a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to a response element, the activity of which is modulated by the cellular response.<br>
In embodiments of the invention, wherein the cellular response is modulation of a signal tranduction pathway, the reporter system may comprises a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to a response element, the activity of which is modulated by said signal transduction pathway.<br>
For example, if the cellular response is modulation of a signal transduction pathway influencing the activity of CRE and/or SRE, then the reporter system may comprise a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide<br>
operably linked to a response element selected from the group consisting of cAMP response element (CRE) and serum response element (SRE). Examples of such signal transduction pathways include the signal transduction pathways modulated by GPCR of the rhodopsin family or secretin family and by protein kinase receptors and receptors belonging to the family of receptor kinases.<br>
By way of example: 1) If the cellular response is activation of a signal transduction pathway activated by a GPCR coupled to a Gs (see herein above) that stimulates adenylate cyclase, then the reporter system may be a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to CRE. Activation of said GPCR may then be detected by detection of increased levels of said detectable polypeptide. 2) If the cellular response is activation of signal transduction pathway activated by a GPCR coupled to a G, (see herein above) that inhibits adenylate cyclase, then the reporter system may be a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to CRE. Activation of said GPCR may then be detected by detection of decreased levels of said detectable polypeptide.<br>
Similarly, if the cellular response is modulation of a signal transduction pathway that influences the activity of TRE, then the reporter system may comprise a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to TPA response element (TRE). Examples are GPCRs that are linked to activation of Protein Kinase C such as Gq coupled receptors (see herein above).<br>
Similarly, if the cellular response is modulation of a signal transduction pathway that influences the activity of SRE, then the reporter system may comprise a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to SRE. Examples of such signal transduction pathways include the signal tranduction pathways modulated by growth hormones or cytokines through protein kinase receptors and receptors belonging to the family of receptor kinases.<br>
Similarly, if the cellular response is modulation of a signal transduction pathway that influences the activity of AP-1, then the reporter system may comprise a nucleic acid comprising a nucleotide sequence encoding a detectable polypeptide operably linked to AP-1. Examples of such signal transduction pathways include the signal<br>
tranduction pathways modulated by cytokines or growth factors cytokines through protein kinase receptors and receptors belonging to the family of receptor kinases<br>
The detectable polypeptide may be any detectable polypeptide, however preferably the detectable polypeptide is selected from the group consisting of fluorescent proteins and enzymes.<br>
Fluorescent proteins may for example be green fluorescent protein (GFP) and fluorescent mutants thereof, such as yellow fluorescent protein (YFP) or cyan fluorescent protein (CFP). The fluorescent protein can also be a protein complex, e.g. a di-or tetramer of a fluorescent protein, such as dsRed. Enzymes may for example be selected from the group consisting of luciferase, CAT, galactosidase, alkaline phos-phatase and beta-lactamase.<br>
In one embodiment of the invention the reporter system may comprise a biolumi-nescent moiety. For example, if the cellular response is relocalisation of a compound, then the reporter system may for example be said compound linked to a lu-miniscent moiety, such as a fluorescent moeity. Hence, for example if the cellular response is relocalisation of a polypeptide the reporter system may be a chimeric protein made up of said polypeptide and a fluorescent protein, such as GFP, YFP or CFP. In one preferred embodiment said polypeptide may be receptor.<br>
In one embodiment of the invention the reporter system may detect the level of a cellular molecule, such as a protein. This may for example be achieved by quantifying the amount of a compound i.e. an antibody that specifically binds to the cellular molecule. The quantification can for example be achieved by covalently coupling a fluorescent, bioluminescent or coloured moiety to said compound. The quantification could be confined to a specific cellular compartment.<br>
In one embodiment of the invention the reporter system may detect the level of modification of a cellular molecule for example but not limited to phosphorylation, glycosylation or ubiguitination. This may for example be achieved by quantifying the amount of a compound i.e. an antibody that specifically binds to the modified cellular molecule. The quantification can for example be achieved by covalently coupling a<br>
fluorescent, bioluminescent or coloured moiety to said compound. The quantification could be confined to a specific cellular compartment.<br>
In one embodiment of the invention the reporter system may detect complex formation or disruption between two cellular proteins (designated first protein and second protein in this paragraph). This is in particular relevant when the cellular response is change in interaction between two cellular proteins. Several differerent reporter systems may be used to detect interaction between a first and a second protein. Below preferred reporter systems are described, however, the invention is not limited to these specific reporter systems.<br>
The reporter system may for example comprise the first protein linked to a bioluminescent moiety, such as luciferase and the other protein linked to a fluorescent moiety, such as a fluorescent protein. Such reporter systems are referred to as "BRET reporter systems" herein. The bioluminiscent moeity should preferably be able to directly or indirectly generate light of a wavelength capable of exciting the fluorescent moiety. The skilled person will readily be able to select useful bioluminiscent moeities and fluorescent moeities. Preferably, the BRET reporter system comprises a first chimeric protein comprising the first protein linked to a bioluminescent protein, preferably luciferase and a second chimeric protein comprising the second protein linked to a fluorescent protein. Such a reporter system may be introduced into a cell by introducing nucleic acids encoding the first and the second chimeric proteins under control of suitable promoters into said cell. Direct interaction between the proteins can after expression of the two chimeric proteins be detected through occurrence of BRET (Bioluminescence Resonance Energy Transfer). In one embodiment, BRET2 technology may be used which is based on energy transfer between a bioluminescent donor (a Renilla luciferase (Rluc) fusion protein) and a fluorescent acceptor (a Green Fluorescent Protein (GFP2) fusion protein). In presence of its substrate DeepBlueC™ (a coelenterazine derivative), Rluc emits blue light (~395 nm). Thus the reporter system may comprise a first chimeric protein comprising the first protein and Rluc and a second chimeric protein comprising the second protein and GFP2. A protein-protein interaction between Rluc and GFP2 chimeric proteins allows energy transfer to GFP2, which reemits green light (510 nm). Expression of Rluc alone, in the presence of the substrate DeepBlueC™, gives an emission spectrum with a peak at ~395 nm, whereas when the Rluc and GFP2 chimeric proteins<br>
interact, there is efficient energy transfer between Rluc and GFP2 and the 510 nm signal represents a major peak.<br>
In another similar reporter system the first protein and the second protein are linked to different fluorescent moieties, preferably a fluorescent proteins. Such reporter systems are referred to as "FRET reporter systems" herein. Preferably, one fluorescent moiety is capable of emitting light of a wavelength capable of exciting the other fluorescent moeity. FRET reporter systems preferably comprise a first chimeric protein comprising the first protein and a fluorescent protein and a second chimeric protein comprising the second protein and another flourescent protein. It is then possible to detect the complex formation through the occurrence of FRET (Fluorescence Resonance Energy Transfer). BRET or FRET according to the present invention may for example be performed as described in (Nicolas B, R Jockers, and T Issad Trends in Pharmacological Sciences 23 (8):351-354, 2002; and/or A. Roda, M. Guardigli, P. Pasini, and M. Mirasoli. Anal.Bioanal.Chem 377 (5):826-833, 2003)<br>
Complex formation may also be detected by proximity ligation. In such an embodiment the reporter system comprises two affinity probes raised against the first and the second protein. Such reporter systems are designated "proximity ligation reporter systems" herein. When the two proteins come in close proximity a ligation reaction creates a DNA reporter sequence that can be amplified. The amplified sequence can be detected by any useful method, for example it may be detected through photolabelling. Preferably, the DNA reporter sequence is amplified by PCR, rolling circle replication or ligation chain reaction. In order to detect the amplified sequence, the sequence may for example be amplified using primers labelled with a detecable label, such as a fluorescent label or the sequence may be detected using a detectably labelled probe, such as a fluorescently labelled probe. The affinity probes in general comprise or consist of a binding moeity and a nucleic acid moeity. The binding moiety of the affinity probes can be any molecule that binds either the first or the second protein with high affinity. Preferably, the binding moeity is capable of specifically recognising and binding either the first or the second protein. Examples of usefuil binding moieties of affinity probes are monoclonal- or polyclonal antibodies or antigen binding fragments thereof, chimeric antibodies, recombinant antibodies, single chain antibodies or aptamers. Antibodies may be prepared using any conventional method known to the person skilled in the art. Aptamers may be prepared by any method known to the skilled person, for example by iterative cycles of<br>
screening nucleic acid libraries for compounds capable of binding a tare nucleic acid molecules selected for their ability to specifically bind a target. Aptamers may for example be produced using a SELEX process (Sun S Curr Opin Mol Ther 2000 Feb 2:100-5; Javasena SD Clin Chem 1999 Sep 45:1628-50). The nucleic acid moeity may comprise or consist of any nucleic acid sequence, preferably a sequence, which when ligated to another nucleic acid moeity creates a DNA repoter sequence, which can be amplified by PCR, rolling cycle amplification or ligase chain reaction using appropriate primers. A person skilled in the art can design useful nucleic acid moeity sequences and corresponding primers. The affinity probes can be introduced into cells by a number of different methods, for example they may be introduced into the cells after said cells have been fixed and permeabilized or they can be introduced by using traditional cDNA transfection methods, for example by using standard procedure for FugeneG transfection. Proximity ligation may for example be carried out as described in Frederiksson et al. Nature Biotechnology 2002, 20: 473; Gullberg et al. Curr Opinion Biotechnology 2003,14: 82. The reporter system may also be a "two-hybrid reporter system". Two-hybrid reporter systems comprises two chimeric proteins, wherein the first chimeric protein comprises the first protein fused to a DNA binding domain and the second chimeric protein comprises the second protein fused to a transactivating domain. Furthermore, the two hybrid reporter system comprises a reporter construct comprising a nucleic acid sequence encoding a detectable polypeptide the expression of which is controlled by the transactivat-ing/DNA binding domain. Thus if the first protein interacts with the second protein the DNA binding domain and the transactivating domain are brought into close proximity and may activate transcription from the reporter construct. Interaction can then be determined by detection of the detectable polypeptide. The detectable polypeptide may be any of the detectable polypeptides mentioned herein above. Two-hybrid reporter systems are well described in the art, see for example US 5,283,173.<br>
The reporter system may also be an enzyme complementation reporter system. Enzyme complementation reporter systems comprises two chimeric proteins, wherein the first chimeric protein comprises the first protein fused to a first part of an enzyme and the second chimeric protein comprises the second protein fused to a second part of an enzyme. The first and the second part of an enzyme should together constitute a functional enzyme. Thus, when the first protein interacts with the second protein the first part and the second part of an enzyme will form a functional<br>
enzyme, the activity of which may be determined. One example of an enzyme useful for enzyme complementation system is DHFR (dihydrofolate reductase), where the activity of the reconstituted enzyme is monitored as a fluorescent read-out based on stoichiometric binding of fluorescein-methotrexate to reconstituted DHFR (Remv I. Michnick SW Proc Natl Acad Sci U S A 1999 May 96:5394-9).<br>
Hence, if the cellular response is relocalisation of a cell surface molecule, then the reporter system may comprise a fluorescent moiety covalently coupled to said cell surface molecule.<br>
In some embodiments of the invention the cellular response is modulation of a signal transduction pathway involving activation of phospholipase C. Phospholipase C may for example be activated by GPCRs coupled to GQ (see herein above). Activation of phospholipase C in general leads to increase in the intracellular level of Ca2+ and thus in such embodiments the reporter system may be the intracellular Ca2* level. This reporter system may thus be endogenous to the cell.<br>
When the cellular response is induction/facilitation of apoptosis a number of reporter systems may be employed. Induction/facilitation of apoptosis may be determined by determing caspase activity as described herein above. Induction/facilitation of apoptosis may also be determined by determining cell growth/number of cells, for example cell growth/number of cells after cultivation of for example normal cells or tumour cells, such as cells expressing high levels of XIAP or ML-IAP. Other methods of determining apoptosis are well known to the skilled person.<br>
Detectable output<br>
The detectable output may be any output, which is detectable directly or indirectly. For example the detectable output may be the concentration of a compound within a cell, localisation of a compound within a cell, luminiscense, activity of an enzyme or the like.<br>
In preferred embodiments of the invention the detectable output is luminiscense, fluorescence, bioluminescence , FRET or BRET. Bioluminiscence may be detected<br>
by any conventional methods, for example with the aid of a Plate reader. BRET or FRET may for example be detected using FABS, a plate reader, a fluorescence microscope or the like.<br>
Alternatively, the detectable output may preferably be linked (directly or indirectly) to a bioluminiscent signal.<br>
However, the detectable output could also be radioactivity, a coloured compound or a colour signal, a heavy metal, an electrical potential, a redox potential, a temperature or the detectable output may be linked to a radioactive signal, a coloured compound or a colour signal or a heavy metal or an electrical potential, or a redox potential or a temperature. Said radioactive signal could for example be ^S, 32P, 3H. The coloured compound could for example be the product of any of the enzymatic reaction described herein elsewhere. The heavy metal could for example be gold.<br>
In embodiments of the invention, wherein the cellular response is change in the in-tracellular level of a compound or change in the level of a compound within a specific cellular compartment, then the detectable output may be said level of said compound. Depending on the nature of the compound, said level may be detected directly or indirectly.<br>
If the compound for example is a fluorescent compound, the level of said compound may be determined by determining the fluorescence properties. This may be done by any suitable means, for example by the aid of a fluorescence microscope, a FACS (Fluorescence Activated Cell Sorter), a FABS (Fluorescence Activated Bead Sorter), fluorescence plate-reader or a fluorescence spectrometer,<br>
If the compound for example is an enzyme then the level of said compound may be determined by determining the activity of said enzyme. By way of example, if the enzyme catalyses a reaction leading to a product, which is directly detectable, for example by colorimetric or chemiluminescent detection techniques, the activity of said enzyme may be detected by detecting said compound. For example, if the enzyme is luciferase, the activity of said enzyme may be detected by detecting em-mision of light upon oxidation of the added substrate, luciferin.<br>
Several other enzymes such as CAT, |3-galactosidase, alkaline phosphatase, horseradish peroxidase and beta-lactamase are, when provided with suitable substrates, capable of catalysing reactions leading to coloured or chemiluminescent products, which may be detected using any colorimetric or chemiluminescent detection technique.<br>
If the compound for example is Ca2+, then the intracellular concentration of said ion can be measured by using any suitable method, for example by inserting into the cells Ca2+ binding fluorescent compounds like Fura-2, Fluo-3 or Fluo-4 (Molecular Probes), which change fluorescent properties according to a changed Ca2+ concentration. A non-limiting example of a method of determining cytosolic free Ca2+ is given in example 13 herein below. Other ion concentrations can be monitored using suitable fluorescent compounds, which for example are available from Molecular Probes Inc.<br>
If the compound for example is a protein, then it may for example be detected using a first specific binding partner. Said first specific binding partner could be a second protein capable of specifically interacting with said protein, such as a specific antibody or said first specific binding partner could be an aptamer. Said first specific binding partner could be conjugated to a directly detectable compound, such as a fluorescent compound, a radioactive compound or a heavy metal or to an indirectly detectable compound, such as an enzyme, which for example could be any of the enzymes mentioned herein above. It is also possible that the first specific binding partner may be detected with a second specific binding partner, capable of interacting specifically with the first specific binding partner. Said second specific binding partner may be conjugated to a directly or indirectly detectable compound similarly to the first specific binding partner. Additional specific binding partners may be used.<br>
In embodiments of the invention wherein the cellular response is relocalisation of a compound the detectable output could be a detectable label conjugated to said compound. In particular, the compound may be conjugated to a directly detectable label, such as a fluorescent label or a heavy metal. Thus the localisation of the compound may be directly detected, for example using a fluorescence microscope. Fluorescent plate-reader, fluorescence spectrometer, a FACS or a FABS instrument In<br>
one preferred embodiment the compound is a fusion protein comprising a protein of interest and a fluorescent protein, such as GFP. The compound may thus be a fluorescent probe. Thus the detectable output may be localisation of a fluorescent signal. Alternatively, the compound is a fusion protein comprising the protein of interest and a tag. Said tag could be a tag specifically interacting with a specific binding partner, for example the tag could be an HA-tag or a flash domain. Alternatively, localisation of a compound may be determined with the aid of a specific binding partner as outlined above. Intracellular localisation may also be detected using methods capable of detecting distance between two compounds, for example BRET or FRET.<br>
In embodiments of the invention wherein the cellular response is change of activity of a compound, the detectable output may be a product of said activity. I.e. when said compound is an enzyme the detectable output could be a product of a reaction catalysed by said enzyme. Said product could thus be a coloured product or a chemiluminiscent product as discussed herein above.<br>
In embodiments of the invention wherein the cellular response is enhanced or reduced transcription from one or more genes, then the cellular response could be mRNA transcribed from said gene, a protein encoded by said gene or in case the protein is an enzyme, the detectable output could be a product of a reaction catalysed by said enzyme. The enzyme and the products could be any of the enzymes or products discussed herein above.<br>
mRNA may be detected by any useful means, for example with the aid of a probe capable of hybridising specifically with said mRNA. Said probe could be labelled with a directly detectable label, for example a radioactive compound, a fluorescent compound or a heavy metal or an indirectly detectable label such as an enzyme or a specific binding partner.<br>
Said protein may be detected with the aid of specific binding partners as outlined herein above. However, in a preferred embodiment the protein is a fluorescent protein and may thus be detected directly. Hence, the detectable output could be bioluminescence, such as fluorescence.<br>
In embodiments of the invention wherein the cellular response is modification by for example phosphorylation of a compound this can be detected through binding of an antibody that specifically bind the phosphorylated protein said antibody can then be quantified by specific fluorescence labelling.<br>
In embodiments of the invention wherein the cellular response is change in pH in an intracellular compartment, the detectable output will in general be said pH. The pH may be determined using any suitable method, for example using a pH indicator or a pH-meter. For example the pH may be determined using a fluorescent indicator for intracellular pH. Suitable compounds are compounds with a fluorescent excitation profile which is pH-dependent, such as BCECF (available from Molecular Probes). In embodiments of the invention wherein the cellular response is a change in a membrane potential, the detectable output will in general be said membrane potential. The membrane potential may be determined using any suitable method such as applying a fluorescent molecule to cells that distribute over the membrane dependent upon the membrane potential. Examples of such compounds are DiBAC, various ANEP dyes, JC-1 and JC-9 (Molecular Probes). For example, JC-1 and JC-9 are cationic dyes that exhibit potential-dependent accumulation in mitochondria leading to a shift in fluorescence emmision from green to red. Thus mitochondria! depolarization may for example be determined by decrease in red/green fluorescence intensity ratio (see also product information from Molecular Probes). ANEP dyes are in particularly useful for detection of changes in membrane potential. The fluorescence can be read for instance by a fluorescence microscope, a fluorescence plate-reader, a FACS, or a FABS instrument.<br>
In embodiment of the invention wherein the cellular response is change in morphology, the detectable output will in general be the morphology of the cell. The morphology may be observed using any suitable method for example by the aid of a microscope, using a FACS or FABS.<br>
In embodiments of the invention where the cellular response is change in an interaction between two cellular proteins, the detectable outout may for example be BRET or FRET, which is detectable by determining the occurrence of fluorescence of a given wavelength. BRET or FRET may for example be detected using a FABS, FACS, fluorescent microscope or any other equipment useful for detection of fluorescence.<br><br>
In embodiments of the invention wherein the reporter system is proximity ligation the detectable output is dependent on the detectable label used to label the amplified DNA reporter sequence. In embodiments, wherein the DNA reporter sequence is amplified using fluorescently labelled primers, the detectable output will be said fluorescent label, which may be detected using a FABS, FACS, fluorescent microscope or any other equipment useful for detection of fluorescence.<br>
Depending on the detectable output, it will frequently be an advantage to fix cells prior to detecting said detectable output. However, in some embodiments of the invention it is preferred that the cells are not fixed. Cells may be fixed according to any useful protocol (see also definitions herein above).<br>
Selection<br>
The methods according to the invention involves screening resin beads for beads comprising cells meeting at least one predetermined selection criterion, wherein said selection criterion is linked directly or indirectly to said detectable output. Hence, the selection criterion will be dependent on the detectable output.<br>
For example the predetermined selection criterium may be a quantitative criterium, such as a quantitative level of bioluminiscence above or below a specific threshold value.<br>
In embodiments of the invention, wherein the detectable output is fluorescence or the detectable output may be linked to a fluorescent signal, then the predetermined selection criterion could be any fluorescence property. For example, the selection criterion could be intensity of said fluorescence above or below a predetermined threshold value or emission of light of a specific wavelength or absorption of light of a specific wavelength or intensity of emitted light of a specific wavelength above or below a predetermined threshold value. The selection criterion could also be based on Fluorescence lifetime and/or fluorescence polarization The selection criterion could also be a specific localisation of the fluorescent signal, such as intensity of a fluorescent signal in a specific cellular compartment above or below a predeter-<br><br>
mined threshold value. The selection criterion could also be a predetermined change in fluorescence lifetime or in fluorescence polarization. Fluorescence intensity and/or localisation may for example be determined using image processing and/or image analysis, a fluorescence microscope, FACS, FABS or fluorescence plate reader.<br>
In one embodiment of the invention the selection criterion is high fluorescence intensity. This may for example be the case, when the cellular response is activation of a signal transduction pathway and the reporter system comprises a gene encoding a fluorescent protein, where activation of the signal transduction pathway leads to incresed expression of said gene. This may also be the case when the cellular response is establishment of interaction between two cellular proteins, wherein the reporter system is a FRET or BRET reporter system. After release of a proportion of the library members to be tested, resin beads may be selected using a method comprising the steps of:<br>
1.	Determining the fluorescence intensity of positive control resin beads and setting<br>
this fluorescence intensity to 100%<br>
2.	Determining the fluorescence intensity of negative control resin beads and<br>
setting this fluorescence intensity to 0%<br>
3.	Selecting resin beads having a fluorescence intensity corresponding to at least<br>
5%, preferably at least 10%, more preferably at least 20%, even more preferably<br>
at least 30%, such as at least 40%, for example at least 50%, such as at least<br>
60%, for example at leasat 70&amp;, such as at least 80%, for example at least 90%,<br>
such as in the range of 5 to 100%, for example in the range of 10 to 100%, such<br>
as in the range of 20 to 100%, for example in the range of 30 to 100%, such as<br>
in the range of 40 to 100%, for example in the range of 50 to 100%.<br>
The positive control may for example be a resin bead (or optionally several resin beads kept in a separate container or well) comprising a compound known to influence the cellular response. By way of example, if the cellular response is activation of an intracellular signal transduction pathway, then the positive control may be a resin bead comprising a known compound that stimulate one or multiple components of said intracellular signal transduction pathway. The positive control signal is then obtained after release from the resin of the compound. Alternatively the positive control may be a resin bead comprising a known ligand of a receptor<br><br>
activating the signal transduction pathway, for example a naturally occurring ligand. The negative control may be a resin bead (or optionally several resin beads kept in a separate container or well) optionally comprising a cell adhesion compound, but otherwise comprising no library member or other test compound.<br>
In another embodiment of the selection criterion is low fluorescence. This may for example be the case, when the cellular response is inhibition of a signal tranduction pathway and the reporter system comprises a gene encoding a fluorescent protein, where an active signal transduction pathway leads to expression of said gene. This may also be the case when the cellular response is disruption of interaction between<br>
two cellular proteins, wherein the reporter system is a FRET or BRET reporter<br>
system. After release of a proportion of the library members to be tested, resin beads may be selected using a method comprising the steps of:<br>
1.	Determining the fluorescence intensity of positive control resin beads and setting<br>
this fluorescence intensity to 0%<br>
2.	Determining the fluorescence intensity of negative control resin beads and<br>
setting this fluorescence intensity to 100%<br>
3.	Selecting resin beads having a fluorescence intensity corresponding to at least<br>
5%, preferably at least 10%, more preferably at least 20%, even more preferably<br>
at least 30%, such as at least 40%, for example at least 50%, such as at least<br>
60%, for example at leasat 70&amp;, such as at least 80%, for example at least 90%,<br>
such as in the range of 5 to 100%, for example in the range of 10 to 100%, such<br>
as in the range of 20 to 100%, for example in the range of 30 to 100%, such as<br>
in the range of 40 to 100%, for example in the range of 50 to 100%.<br>
The positive control may for example be a resin bead(s) comprising a compound known to influence the cellular response. By way of example, if the cellular response is inhibition of an intracellular signal transduction pathway, then the positive control may be a resin bead(s) comprising a compound known to inhibit one or multiple components of signal transduction pathway. The control signal is then obtained after release from the resin of the compound. Alternatively the positive control may be a resin bead comprising a known antagonist of a receptor known to activate said signal transduction pathway. If the cellular response is induction of apoptosis, then the positive control may be a resin bead comprising a compound known to induce apoptosis The negative control may be a resin bead optionally comprising a cell<br><br>
adhesion compound, but otherwise comprising no library member or other test compound.<br>
One method of selecting resin beads using FABS is illustrated in figure 1 A.<br>
In one preferred embodiment selection is performed manually with the aid of a fluorescence microscope. In this embodiment the fluorescence intensity or other fluorescence properties are judged manually.<br>
When the selection criterion is fluorescence intensity of localisation, the resin beads may also be analysed using a plate reader or image acquisition. An example of such an analysis is given in figure 1B.<br>
If the selection criterion is localisation, then resin beads are generally analysed by a fluorescence or imaging microscope. Said microscope may optionally be equipped with a micromanipulator capable of picking out single beads. Resin beads are scanned for cells where the fluorescence signal is located at the desired intracellular location and these resin beads are selected. The selection may be manually or it may be automated.<br>
In embodiments of the invention, wherein the detectable output is light emission or the detectable output may be linked to a light signal, then the predetermined selection criterion could be any property of the light. For example the selection criteria could be light intensity above or below a predetermined threshold value. Light can be detected for example by the eye, in a microscope, and if the light is emitted via bioluminescence it can be measured by a luminometer<br>
In embodiments of the invention, wherein the detectable output is a radioactive signal or the detectable output may be linked to a radioactive signal, then the selection criterion could be any property of said radioactive signal, such as intensity above or below a predetermined threshold value or localisation of the radioactive signal.<br>
In embodiments of the invention, wherein the detectable output is a colour signal or the detectable output may be linked to a colour signal, then the selection criterion could be any property or said colour signal. For example the predetermined selec-<br><br>
tion criterion could be a colour intensity above or below a specific threshold value or it could be a specific colour. The colour signal could be detected using any suitable colorimetric method, such as a spectrophotometer or a microscope.<br>
Resin beads comprising cells meeting at least one selection criterion, such as any of the selection criteria mentioned herein above are selected. In certain embodiments of the invention resin beads compn'sing cells meeting at least two, for example 2, such as 3, for example 4, such as in the range of 5 to 10, for example of in the range of 10 to 25 selection criteria are selected.<br>
It is also possible within the present invention to select resin beads comprising cells meeting one or more predetermined selection criteria and subsequently to subject said beads to one or more additional selection rounds, wherein resin beads comprising cells meeting one or more additional selection criteria are selected.<br>
Resin beads meeting said at least one predetermined selection criteria may be selected by manually sorting for example with the aid of a microscope, or for example by sorting by fluorescence or by colour or by morphology depending on the detectable output and the selection criterion. Positive beads may be picked directly under the microscope, such as under a fluorescence microscope for example manually or with the aid of a micromanipulator. Frequently, in the range of 100 to 1,000,000, for example in the range of 1000 to 100,000, such as in the range of 5000 to 50,000 resin beads may be placed on a suitable surface, such as in a dish or on a coverglass and subsequently examined by microscopy. Alternatively, the sorting process may be automated with the use of specially designed, commercially available bead sorters (Union Biometrica, Sommerville, Mass.) and detecting for example fluorescence intensity (Meldal, 2002, Biopolymers, 66: 93-100). In general, resin beads can be sorted at a rate of up to 100 beads per second, or even faster depending on the equipment used and its reading capacity. A range of about 5-30 beads per second is generally used with known instruments. Slower rates may be used to increase accuracy, .however any suitable rate may be used with the present invention, such as much higher rates. Preferred, is a rate where only one resin bead passes through the detector at a time. It is also comprised within the present invention to select resin beads using a plate reader. In general in the range of 1 to 1000, such as 10 to 500, for example 50 to 100 resin beads are placed in each well<br><br>
of a multiter plate and analysed. Beads from positive wells may then be further examined.<br>
In one embodiment of the invention resin beads may be selected by comparing the detectable output, with the detectable output generated by control resin beads, for example positive and/or negative control resin beads. Positive control resin beads are beads comprising a compound capable of inducing the desired cellular response, whereas negative control resin beads comprises no such compound. By way of example, if the cellular response is activation of an intracellular signal trans-duction pathway with a known natural ligand, the positive control resin bead may comprise said ligand, whereas the negative control resin bead comprises no compound except optionally a cell adhesion compound.<br>
If the detectable output is a quantifiable signal, then resin beads may be selected, comprising cells where the detectable output is higher or lower than the detectable output from cells attached to the positive or negative control resin bead. By way of example, if the detectable output is fluorescence intensity, then resin beads comprising cells displaying a fluorescence intensity which is higher than the negative control and lower than the positive control could for example be selected.<br>
Non-limiting examples of methods of selecting resin beads are illustrates in figures 1 and 2.<br>
Identification of compound<br>
Once a resin bead has been selected, the compound that is remaining after partial release on said bead may be identified. Preferably, only one resin bead is used at a time. Thus if said resin bead only comprises one library member in one or more copies, then only one compound is identified at a time.<br>
The process for identification of the library member depends on the type of library used. For a library of primarily oligomeric compounds, the library member can be analysed by Mass Spectroscopy (MS), particularly if the library was synthesized in such a way that the synthetic history of the compound is captured, for example,<br><br>
using a capping procedure to generate fragments of the compound that differ in mass by one building block (see, for example, Youngquist et al., 1995, J. Am Chem. Soc., 117: 3900-06). This capping procedure is most efficient when the cap and the building block are reacted at the same time. The capping agent can be any class of compound that has at least one functional group in common with the building block used to generate the oligomer, so that both the capping agent and the building block can react when added to the resin in an appropriate ratio. Alternatively, the capping agent can have two functional groups in common with the building block where one of the groups in common, such as the group in the building block that is used for the elongation of the oligomer, is orthogonally protected. For example, in a synthesis of a peptide using the Fmoc strategy, the capping agent could be the same as the building block but with a Boc group protecting the reactive amine instead of the Fmoc group (see St. Hilaire et al., 1998, J. Am. Chem. Soc., 120: 13312-13320). In another example, if the building block is a protected haloamine, the capping agent could be the corresponding alkylhalide.<br>
Where the library is synthesized by parallel synthesis (a parallel array), the compound can be identified simply by the knowledge of what specific reaction components were reacted in a particular compartment. The structure can be confirmed by cleavage of a small portion of compound from the solid support and analyzed using routine analytical chemistry methods such as infrared (IR), nuclear magnetic resonance (NMR), mass spectroscopy (MS), and elemental analysis. For a description of various analytical methods useful in combinatorial chemistry, see: Fitch, 1998-99, Mol. Divers., 4: 39-45; and Analytical Techniques in Combinatorial Chemistry, M.E. Swartz (Ed), 2000, Marcel Dekker: New York.<br>
In a preferred embodiment however the library has been synthesised by a split-mix approach where the precise structure of the compound of a specific bead is unknown. In this embodiment, the library member can be identified using a variety of methods. The compound may be cleaved off the resin bead, for example by cleaving the cleavable linker and then analyzed using IR, MS, or NMR. For NMR analysis, larger beads containing approximately 5 nmoles of material are preferably used for the acquisition of 1-dimensional (1-D) and 2-dimensional (2-D) NMR spectra. Furthermore, these spectra can be attained using high-resolution MAS NMR (magic angle spinning nuclear magnetic resonance) techniques. Alternatively, high resolu-<br><br>
tion-MAS NMR spectra can be acquired while the ligand is still bound to the solid support, as described for example, in Gotfredsen et al., 2000, J. Chem. Soc., Perkin Trans., 1: 1167-71. The compound may also be identified by release of the compound and fragmentation by MS-MS in MALDI or electrospray mode.<br>
Frequently, resin beads used for library synthesis contain about 100 to 500 pmoles of material, which is generally insufficient for direct analysis using NMR techniques. In such situations, the libraries can be synthesised with special encoding to facilitate identification of the library member. For a review of encoding strategies employed in combinatorial chemistry see: Barnes et al., 2000, Curr. Opin. Chem. Biol., 4: 346-50. Most coding strategies include the parallel synthesis of the encoding molecule (for example, DNA, PNA, or peptide) along with the library compounds. This strategy requires a well-planned, time consuming, orthogonal protecting group scheme. Furthermore, the encoding molecule itself can sometimes influence the cell leading to false positives. Alternatively, the library members can be encoded using radiofre-quency tags or using optical encoding, such as quantum dot encoding, spherical encoding or distance encoding. These methods alleviates the problem of false positives stemming from the coding tags, but is generally only useful for small libraries in a one-bead-one-compound system due to the sheer bulk of the radiofrequency tag. Alternatively, single beads can be analyzed in a non-destructive manner using infrared imaging. This method gives limited information and while useful for pre-screening, is not recommended for conclusive structural determination.<br>
In a preferred embodiment of the invention the library member(s) comprised within selected resin beads are identified using mass spectrometry (MS). MS can be used alone to identify the library member. The library member can be cleaved from the resin bead, the molecular mass determined, and subsequently fragmented into subspecies to conclusively determine the structure by combination with knowledge of contained structures in the library. MS-based methods of compound identification are useful in this invention, as they require very little material, and can utilise pico- to femtomole amounts of compound.<br>
After identification of the compound it may be desirable to confirm the activity of said compounds by further in vitro and/or in vivo assays. For example, resin beads comprising the identified compound and optionally an adhesion compound may be<br><br>
synthesized and the cellular response confirmed. It is also possible to test identified compounds in in vitro assays in the absence of beads. Cells may for example be grown directly in a tissue culture dish, flask or coverglass and the identified compound can be added directly to the medium of said cells. If several reporter systems are available for the particular cellular response then preferably several different reporter assays may be tested in vitro, in order to identify very useful compounds. By way of example, if the cellular response is induction of apoptosis, then for example activity of caspases, cell growth/cell death and/or interaction between Smac and inhibitors of apoptosis may be tested.<br>
Multiplexing<br>
The methods disclosed by the present invention may also be used in multiplexing methods.<br>
For example, the methods may be used to identify compounds modifying at least two cellular responses, such as 2, for example 3, such as 4. for example in the range of 5 to 10, such as in the range of 10 to 25 cellular responses.<br>
In such methods step c) of the method outlined above (see the section "Summary of the invention") preferably involves screening resin beads for beads comprising cells meeting at least two, such as 2, for example 3, such as 4, for example in the range of 5 to 10, such as in the range of 10 to 25 predetermined selection criteria, wherein each selection criterion is preferably related to a different detectable output.<br>
In such a method more than one kind of cell may be attached to each resin bead and the different cellular responses may be detected in different kinds of cells. For example, a first cell line comprising a first reporter system linked to a first cellular response and a second cell line comprising a second reporter system linked to a second cellular response and optionally additional cell line(s) comprising additional reporter system(s) linked to additional cellular response(s) may all be attached to a single bead. Resin beads comprising cells meeting selection criteria linked to all the different reporter systems may then be selected.<br><br>
Depending on the detectable outputs, said detectable output may be determined using any of the methods described herein above. In one preferred embodiment at least two detectable outputs are fluorescent outputs, preferably of different excitation and/or emmision. Thus resin beads meeting said at least two selection criteria may be selected in one step using a FABS with at least 2 channels in both excitation and emmision. Similarly, more than two different fluorescent properties may be selected for in an suitable FABS. The at least two detectable outputs may be in the same cell line or they may be in different cell lines.<br>
Examples of multiplexing methods are illustrated in figure 2A and 2B.<br>
Examples<br>
Example 1<br>
General methods for solid phase peptide synthesis (SPPS)<br>
General for chemical synthesis:<br>
All chemicals described, commercially available and used without further purification. All solvents were HPLC-grade. PEGA - resins were purchased from VersaMa-trix A/S, Copenhagen. Each washing step lasted 2 min unless otherwise stated. Purifications were performed on a standard reverse phase HPLC using gradients of acetonitrile-Water with various amounts of TFA.<br>
Coupling ofHMBA linker to PEGA - resin:<br>
Dry PEGA - resin was swelled in DCM and washed with DMF (3x). 3.0 eq. HMBA, 2.9 eq. TBTU and 3.0 eq. NEM were mixed in appropriate DMF and allowed to react for 10 min. The mixture was added to resin and after 2h the resin was washed with DMF (6x), DCM (6x) and lyophilised.<br>
General Procedure for Coupling ofAmino Acid to HMBA-linker: Dry PEGA - resin with HMBA-linker was swelled in DCM. 3.0 eq. Fmoc-protected amino acid, 2.25 eq. Melm and 3.0 eq. MSNT were mixed in appropriate amount of DCM and added to resin. After 1h the resin was washed with DCM (3x) and the coupling was repeated as above once. After coupling for 1h the resin was washed with DCM (6x), DMF (6x), DCM (6x) and lyophilised.<br><br>
General SPPS Coupling Procedure:<br>
The terminal amino acid on the resin was Fmoc-deprotected by treatment with 20% piperidine in DMF (1x2 min + 1x18 min) followed by washing with DMF (6x). 3.0 eq. Fmoc-protected amino acid, 2.9 eq. TBTU and 3.0 eq. NEM were mixed in appropriate amount of DMF and allowed to react for 10 min. The mixture was added to the resin and after 2h the resin was washed with DMF (6x). /.        General Side Chain Deprotection Procedure:<br>
Dry PEGA - resin with linker, usually acid stable linker(s) and peptide or optionally compound was swelled in H2O and the side chains was deprotected with 95% TFA (aq) (2x15 min). (If Pmc groups were present cleavage time was 6h). The resin was washed with H2O until washing water had pH = 5-7. The resin was then washed with DMF (6x), DCM (6x) and lyophilised.<br>
General HMBA Cleavage Procedure:<br>
Dry PEGA - resin with HMBA linker and attached compound was swelled in water and NaOH (aq.) 0.1 M was added. After 2h HCI (aq.) 0.1 M was used for neutralisation and then AcN was added until the H2O/AcN ratio was 1:1 by volume. The resin was filtered off and the liquid was used direct for RP-HPLC or/and Q-TOF MS analysis.<br>
The above general procedures are used for solid phase peptide synthesis in the following examples unless otherwise specified.<br>
Example 2a<br>
Screening of adhesion peptide library<br>
Approx. 100 adhesion peptide library beads were mixed with 1x1 OE6 cells (BHK, CHO, U2OS, Hek) in each well of a Falcon 12 well plate using 2ml growth medium. The adhesion peptide library was prepared according to the general methods for solid phase peptide synthesis outlined above. The library consisted of heptamers of D-amino acids. The peptide library beads were PEGA beads each coupled to a potential adhesion peptide. The cells and beads were mixed gently every 15 min for 2 hrs. Supernatant with non-<br><br>
attached cells were removed and new growth medium added. Cells/beads were incubating for another 16 hrs. (37°C, 5% CO2). Cell adhesive beads were identified using a microscope with 10 x objective and positive beads were transferred to a filter paper (to suck off medium). Peptides were identified by amino acid sequencing. Examples of useful pep-tides are given in table 2.<br>
Example 2b<br>
Identification of an adhesion peptide with low absorption of fluorescent components from growth medium and high adhesion properties:<br>
An adhesion D-amino peptide library was synthesized (500.000 members) as described above in Example 2a and screened for low fluorescence/high adherence properties. This was done in 4 steps:<br>
1)	Selection of low fluorescent beads by Fluorescence Activated Bead Sorting<br>
(FABS).<br>
The 500.000 member adhesion peptide library was FABSorted and 150.000 low fluorescent beads were isolated.<br>
2)	Selection of beads with good cell adhesion properties.<br>
The 150.000 low fluorescent beads were incubated with GFP expressing U2OS cells followed by FABS sorting for high fluorescence (high cell adhesion). 536 beads were isolated.<br>
3)	Identification and isolation of beads with high Hek293 cell adherence properties.<br>
The 536 beads were cleared for U2OS cells and incubated with GFP expressing<br>
Hek293 cells. 47 beads with high cell adhesion properties were isolated using a fluo<br>
rescence microscope.<br>
4)	Sequence elucidation and re-synthesis of selected peptides.<br>
22 peptides were sequenced and six of them were re-synthesized. Based on Structure-Activity of the six peptides, four additional ones were synthesized. The peptide defined by SEQ ID 35 showed the best overall performance.<br><br>
Example 3:<br>
Synthesis of ueptldes binding to the ML-IAP BIR domain<br>
This example describes the preparation of 6 peptides known to interact with the BIR domain of ML-IAP (livin). The syntheses were performed according to the "general SPPS coupling Procedure0 above.<br>
Synthesis of H-Ala-Val-Pro-lll-Ala-Gln-Lys-Ser-Glu-OH (X), H-Ala-Val-Pro-Phe-Ala-Val-Lys-Ser-Glu-OH (X), and H-Ala-Glu-Ala-Val-Pro-Trp-Lys-Ser-Glu-OH (X)<br>
Coupling 1-3: HMBA modified PEGA 800 beads (600 mg.Versabeads A-800, 315-500 ^m, 0.32 mmol NH2/g) were coupled under standard MSNT coupling conditions (Example 1) with Fmoc-Glu(OtBu)OH followed by standard TBTU coupling of Fmoc-Ser(O'Bu)OH and FmocLys(Boc)OH.<br>
Coupling 4-8: The resin was separated into three 200 mg portions and three parallel couplings were performed with the respective amino acids according to the "general SPPS coupling procedure in Example 1 (Trp, Glu and Gin were side chain protected as the N-Qoc, O-'Bu and /V-trityl derivatives respectively). After the last coupling side chain deprotection was performed according to the general procedure (Example 1). The peptides were detached according to the general HMBA cleavage procedure (Example 1) and the peptides purified with preparative HPLC.<br>
H-Ala-Val-Pro-lll-AJa-Gln-Lys-Ser-Glu-OH (X): Yield after purification : 30 mg (&gt;98% pure according to HPLC). MALDI-MS Calcd. (M+H)* 942.5; Found 942.7. NMR data were in accordance with the predicted structure.<br>
H-Ala-Val-Pro-Phe-Ala-Val-Lys-Ser-Glu-OH (X): Yield after purification : 32 mg (&gt;98% pure according to HPLC). MALDI-MS. Calcd. (M+H)+ 947.5; Found 948.0. NMR data were in accordance with the predicted structure.<br>
H-Ala-Glu-Ala-Val-Pro-Trp-Lys-Ser-Glu-OH (X): Yield after purification : 38 mg (&gt;98% pure according to HPLC). MALDI-MS. Calcd. (M+H)+1016.5; Found 1016.5. NMR data were in accordance with the predicted structure.<br><br>
Synthesis of H-Ala-Val-Pro-lle-OH (X)<br>
Synthesis performed according to the general SPPS procedure on 200 mg of HMBA modified PEGA 800 beads. Yield after HMBA cleavage and purification : 17 mg. (&gt;98% pure according to HPLC). ES-MS. Calcd. (M+H)* 399.2. Found 399.2. NMR data were in accordance with the predicted structure.<br>
Synthesis of H-Ala-Glu-(1-amino-cyclopentanecarboxyl)-Phe-OH (X)<br>
Synthesis performed according to the general SPPS procedure on 200 mg of HMBA modified PEGA 800 beads. Yield after HMBA cleavage and purification : 22 mg. (&gt;98% pure according to HPLC). ES-MS. Calcd. (M+H)+ 477.2. Found 477.2. NMR data were in accordance with the predicted structure.<br>
Synthesis of H-Ala-Glu-(azetidine-1-carboxyl)-(homo-Phe)-OH (X)<br>
Synthesis performed according to the general SPPS procedure on 200 mg of HMBA modified PEGA 800 beads. Yield after HMBA cleavage and purification : 21 mg. (&gt;98% pure according to HPLC). ES-MS. Calcd. (M+H)+ 463.2. Found 463.2. NMR data were in accordance with the predicted structure.<br>
The structures of the 6 synthesized peptides are given below.<br><br><br>
(Figure Remove)<br>
Example 4<br>
Synthesis of a 44.000 member tetrapeotlde library on beads with fixed<br>
adhesion peptide<br>
This example describes the preparation of a library of tetrapepides (44.000 members) on resin beads containing a "cell adhesive peptide". This "two compound one bead" (TCO6) library may be used for in vivo screening of the tetrapeptides for e.g.<br>
antagonists action on the ML-IAP receptor.<br>
FmocLys(Fmoc) / BocVal (-1:1) modification of Pega 1900 beads<br>
Dry Versabeads A-1900 (315-500 urn, 0.2 mmol NH2/g)(1.20 g) were dissolved in dry DMF (10 ml) and coupled with FmocLys(Fmoc)OH (0.24 mmol, 1 eq) and Boc-ValOH (0.48 mmol, 2 eq) using the general SPPS coupling procedure.<br>
Fmoc to Alloc transformation of Lys protecting groups<br>
The beads were Fmoc deprotected using the standard procedure. The beads were washed several times with DCM (-10 °C) and fmaly added DCM (10 ml) with the same temperature. DIPEA (0.9 ml) was added followed by dropwise addition of AllocCI (allyloxycarbonylchloride) (0.26 ml). The reaction mixture was kept under N2 and stirred at RT for 2 h (Kaiser test was negative). The resin was washed with DCM (6x), DMF (6x) and DCM (6x) and lyophilised overnight.<br>
Attachment of photolabile linker (PLL) to Val<br>
The dried beads (~1.2 g) were Boc deprotected with 30% TFA in DCM (2+ 45 min). The resin was washed with DCM (3x), 20% pip in DMF (2x) and DMF (6x). The photo cleavable linker (4-[(1-Fmoc-aminoethyl)-2-methoxy-5-nitrophenoxy]butanoic acid) (187 mg, 0.36 mmol, 3 eq) was dissolved in DMF followed by NEM (100 nL, 0.72 mmol, 6 eq) and TBTU (111 mg, 0.35 mmol, 2.9 eq). After stirring for 5 minutes the mixture was added to the beads and left with occational stirring for 3.5h. (Kaiser test neg). Wash with DMF (10x), DCM (6x) and lyophilised overnight in the dark.<br>
Preparation of tetrapeptide library with 44.000 members<br>
The beads containing the Fmoc protected photo linker (1.00 g) were placed in a custom made 20 well synthesizer. Four couplings using the the general SPPS TBTU coupling procedure were performed using a split and mix protocol. Amino acids (20x20x10x10) used for couplings are given in Tables 3 to 6. For the 3* and 4th coupling each amino acid is added to 2 wells. Protection groups are left on the pep-tides after end couplings. <br>
(Table Remove)<br>
Table 6: Amino acids used for the 4th coupling.<br>
Attachment of "adhesion peptide":<br>
Adhesion peptides were synthesized on the a Hoc-protected lysine. One batch of the library was attached adhesion peptide A, and a second batch with adhesion peptide B.<br>
Adhesion peptide A: (Boc-D-Ala-D-Arg(Pmc)-D-Lys(Boc)-D-Arg(Pmc)-D-lle-D-Arg(Pmc)-D-Gln(Trt)-Gly-)<br>
Alloc deprotection of lysine residues: The library beads were treated under N2 with 3 eq Pd(PPh3)4 in 10 ml degassed CHCI3 containing 5% HOAc and 2.5% NEM for 2h at RT. The resin was washed with CHCI3 (6x), DMF containing 0.5% sodium di-ethyldithiocarbamat and 0.5% DIPEA (2x), and DMF (10x).<br>
Coupling: The adhesive peptide was synthesized directly (stepwise) on the library beads using the general SPPS coupling procedure.<br>
Alternative method: The  purified  peptide  (Boc-D-Ala-D-Arg(Pmc)-D-Lys(Boc)-D-<br>
Arg(Pmc)-D-lle-D-Arg(Pmc)-D-Gln(Trt)-L-Gly-OH) (3 eq) was coupled to the lysine NH2 groups using the general SPPS coupling procedure.<br>
Adhesion peptide B: (Fmoc-D-Arg(Pmc)-D-lle-D-Arg(Pmc)-D-Gln(Trt)-D-Arg(Pmc)-: Analogous to synthesis of adhesion peptide A.<br>
Adhesion     peptide     B:     (Fmoc-D-Arg(Pmc)-D-Gln(Trt)-D-Arg(Pmc)-D-lle-D-Arg(Pmc)-<br>
Analogous to synthesis of adhesion peptide A.<br>
Deprotection of adhesion and library peptides<br>
Final deprotection of protecting groups was performed according to the general procedure described above yielding the TCOB library ready for testing<br>
Example Sa<br>
Preparation of TAT appended "tetrapeotide" library<br>
The peptide library of example 4 is modified to include a TAT sequence appended to the C-terminal end of the peptide library in order to enhance the cell permeation properties of the tetrapeptides.<br>
Synthesisis of TAT sequence (GGYGRKKRRQRRR) on Val residues.<br>
Method A: Beads (1.00 g) containing FmocPLL-Val and AllocLys(Alloc) (as prepared in example 3) are subjected to stepwise SPPS using the general TBTU protocol giving the sequence FmocGGYGRKKRRQRRR attached to the photo labile linker (Arg is side chain protected as Pmc, Gin protected as Trt, Lys protected as Boc, and Tyr as 'Bu).<br>
Method B: Beads (1.00 g) containing FmocPLL-Val and AllocLys(Alloc) (as prepared in example 3) are coupled to the side chain protected and purified TAT sequence FmocGGYGRKKRRQRRR using the general SPPS coupling procedure. Protection of side chains as in method A.<br>
Formation of TAT appended tetrapeptide library<br>
Using the FmocTAT-Val derivatized beads a tetrapetide library is build according to the description in Example 3. The resulting library is derivatized with the cell adhesive peptide and followingly all protection groups are removed as described in example 4.<br>
Example 5b<br>
Preparation of tetrapeptide library with 44.000 members with base labile adhesion peptide<br>
In certain preferred embodiments it might be advantageous to be able to clear selected beads from both cells and adhesion peptide before either analysis of active compound or further processing. This example describes the synthesis of the above described library on beads where the adhesion peptide is linked to the bead via an HMBA linker.<br>
Synthesis of FmocGly/AllocGly (ratio -1:1) modified PEGA1900 beads<br>
PEGA 1900 beads (300-500 ^m, 0.24 mmol NH2/g) (4 g, 0.96 mmol ~ 1 eq)) were treated with a 1:1 TBTU coupling mixture of FmocGlyOH (2 eq) and AllocGlyOH (2 eq), NEM (17 eq) and TBTU (3.8 eq). Coupling time 3.5 h. Beads washed 10 x with DMF.<br>
Coupling of Holmes photolinker to the "core" Fmoc-Glycine<br>
FmocGly/AllocGly beads were standard Fmoc deprotected with 20% piperidine in DMF leaving the Alloc glycine untouched. Then standard TBTU coupling of the Holmes photolinker (4-(1-aminoethyl-2-methoxy-5-nitrophenoxy)butanoic acid) to the deprotected glycine (1 eq = 0.48 mmol). After coupling, the beads were washed with DMF and DCM and lyophilized.<br>
Tetrapeptide library synthesis on the photo linker<br>
Performed analogous to example 4<br>
Alloc deprotection of second "core" olvcine;<br>
The beads from the 20 wells were all combined in a 50 ml syringe and washed with CHCI3 (5x) and with Ar-degassed CHCI3 containing 5% HOAc and 2.5% NEM (5x).<br><br>
A solution of Pd(PPh3)4 (3 eq, 0.72 mmol) in Ar-degassed CHCI3 containing 5% HOAc and 2.5% NEM (10 ml) was added to the beads and after a few minutes bob-ling with Ar the syringe was sealed with parafilm and left for 2 h. The beads were washed with CHCI3 (10x), DMF (10x), MeOH (10x), DMF (20 % pip) (2x), DMF(10x). Kaiser test was positive.<br>
Attachment of HMBA linker<br>
According to standard procedure above.<br>
MSNT coupling of FmocGlvOH<br>
According to standard procedure above.<br>
TBTU coupling of FmocLvsf FmoclOH<br>
According to standard procedure above.<br>
Adhesion peptide synthesis<br>
After deprotection of the lysine Fmoc groups performed analogous to example 4.<br>
Example 5c<br>
Formation of a "tetraoeptlde" library linked via an internal amide nitrogen<br>
Synthesis of FmocGly/AllocGly (ratio -1:1) modified PEGA1900 beads<br>
PEGA 1900 beads (300-500 Dm, 0.24 mmol NH2/g) (4 g, 0.96 mmol - 1 eq)) were treated with a 1:1 TBTU coupling mixture of FmocGlyOH (2 eq) and AllocGlyOH (2 eq), NEM (17 eq) and TBTU (3.8 eq). Coupling time was 3.5 h. Beads were washed 10 x with DMF.<br>
Coupling of aldehyde photolinker to the "core" Fmoc-Glycine.<br>
FmocGly/AllocGly beads were standard Fmoc deprotected with 20% piperidine in DMF leaving the Alloc glycine untouched. Then standard TBTU coupling of the aldehyde photolinker (4-(4-formyl-2-methoxy-5-nitrophenoxy)butanoic acid) to the deprotected glycine (1 eq = 0.48 mmol). After coupling, the beads were washed with DMF and DCM and lyophilized.<br>
(Table Remove)<br>
 Table 11:. Compounds used for TBTU coupling of amino acids and acylation with acyl chlorides<br>
Library synthesis:<br>
Step 1. Reductive amination of photo linker aldehyde:<br>
5 portions of the above beads (0.8 g dry beads) were placed in 5 syringes of 5 mL.<br>
The beads were swollen in DMF and phenylethylamines (see Table 7) were coupled<br>
to the free aldehyde on the photolinker by reductive amination as follows: 0.8 g dry beads is 96 Dmols ~ 1 eq were pre-treated with the reaction solvent (DMF/HOAc/TEOF/EtOH, 1:1:1:1). Then to each syringe was added one of the five phenylethylamines (20 eq) dissolved in the reaction solvent (400 DL), and the the same solvent was added so the beads were covered. After 0.5 h NaBH3CN (20 eq) was added and the beads stirred cautiously until all dissolved. After 1 h another portion of NaBH3CN (20 eq) was added and the mixture left for additional 2 h. The beads were washed with DMF (10x), DCM (10x). MeOH(10%HOAc) (2x), MeOH (10x), DMF(20% pip) (2x), and DMF (10x)( DCM (10x). The 5 samples were lyophi-lized over night.<br>
Step 2. Mix and split of phenvlethvl amines:<br>
0.4 g of each of the 5 bead samples from above were swelled in DMF, mixed and transferred to a custom made 20-well library synthesizer such that each well contained approximate equal amounts of beads.<br>
Step 3. BTQ coupling of first amino acid:<br>
Total amount of beads = 2.0 g ~ 0.24 mmol gives 12 Dmol NH/well ~ 1 eq.<br>
20 BTC-couplings from 11 different amino acids as shown in Table 8 was made as<br>
follows: Beads were pre-treated with a 1:1 vol% THF/DIPEA solution for 5 minutes<br>
and drained. Of each amino acid in Table 2, 3 eq (36 Dmol) was dissolved in dry<br>
THF (200 DL) and BTC (1.67 eq) added as 200 \aL of a freshly made stock solution<br>
in dry THF. Then 2,4,6-collidine (14 eq), as 200 \iL of a freshly made stock solution<br>
in dry THF, was added and the resulting 20 suspensions left for 5 minutes. Each<br>
suspension was added to the respective well and after short mixing the synthesizer<br>
was sealed and left over night with gentle shaking. Next morning the beads were<br>
washed with THF (10x) and DMF (10x) without mixing the wells.<br>
Step 4. Acvlatipn of well 1-10:<br>
Beads in well 1-10 were washed with DCM (10x) and Boc deprotected with 30% TFA in DCM followed by wash with DCM (10x), DCM (5% DIPEA), DMF, and DCM (10x). From each of the 10 acyl chlorides in Table 9 was made a solution of 10 eq acyl chloride (120 Dmol) and DIPEA (20 eq) in dry DCM (400 DL) containing catalytic amounts of DMAP. The resulting solutions were added to well 1-10 and left with gentle shaking for 1 h. The reaction was repeated. After end reactions the beads<br>
were washed with DCM (10x), and DMF (10x).<br>
Step 5. Removal of Fmoc protection groups:<br>
Well 1-20 were standard Fmoc deprotected, followed by wash with DMF (1 Ox).<br>
Step 6. Mix and split of the 20 wells<br>
The content of the 20 wells was thoroughly mixed and re-distributed equally into the<br>
wells.<br>
Step 7. Coupling of second amino acid (20 amino acidsV<br>
To each well was coupled an amino acid according to Table 10 by a standard TBTU<br>
coupling. Coupling time 5 h. After end reaction the beads were washed with DMF<br>
Step 8. Coupling of third amino acid/acvl chloride (15 amino acids - 5 acvl chlo-<br>
rides):<br>
The beads in all wells were standard Fmoc deprotected and for well 1-15 standard<br>
TBTU coupled with the Boc- protected amino acids in Table 1 1. For wells 16-20 the<br>
beads were first washed with DCM (10x) and the N-terminal amines acylated with<br>
the five acyl chlorides listed in Table 1 1 (entry 16-20) analogous to step 4. After cou-<br>
plings all wells were washed with DMF (10x).<br>
Step 9. Alloc deorotection of second "core" alvcine:<br>
The beads from the 20 wells were all combined in a 50 ml syringe and washed with CHCI3 (5x) and with Ar-degassed CHCI3 containing 5% HOAc and 2.5% NEM (5x). A solution of Pd(PPh3)4 (3 eq, 0.72 mmol) in Ar-degassed CHCI3 containing 5% HOAc and 2.5% NEM (10 ml) was added to the beads and after bobling a few minutes with Ar the syringe was sealed with parafilm and left for 2 h. The beads were washed with CHCI3 (10x), DMF (10x), DMF (5% DIPEA, 5% sodium diethyldithio-carbamate)(5x), MeOH (10x), DMF DMF (20x). Kaiser test was positive.<br>
Step 10. Attachment of HMBA linker: According to the standard procedure above.<br>
Step 1 1 . MSNT coupling of FmocGlvOH:<br><br>
According to the standard procedure above.<br>
Step 12. TBTU coupling of FmocLvs(Fmoc)OH According to the standard procedure above.<br>
Step 13. Attachment of adhesion peptides<br>
Adhesion peptides were synthesized on the Fmoc-protected lysine such that the final beads had two adhesion peptides pr. library molecule. One batch of the library was attached adhesion peptide A, and a second batch with adhesion peptide B.<br>
Adhesion peptide A: (Boc-D-Ala-D-Arg(Pmc)-D-Lys(Boc)-D-Arg(Pmc)-D-lle-D-Arg(Pmc)-D-Gln(Trt)-Gly-:<br>
Fmoc deprotection of lysine residues: According to the standard procedure.<br>
Coupling: The adhesive peptide was synthesized directly (stepwise) on the library beads using the general SPPS coupling procedure.<br>
Alternative method: The purified peptide (Boc-D-Ala-D-Arg(Pmc)-D-Lys(Boc)-D-Arg(Pmc)-D-lle-D-Arg(Pmc)-D-Gln(Trt)-L-Gly-OH) (3 eq) was coupled to the lysine NH2 groups using the general SPPS coupling procedure.<br>
Adhesion peptide B: (Fmoc-D-Arg(Pmc&gt;D-Gln(Trt)-D-Arg(Pmc)-D-lle-D-Arg(Pmc)-: (SEQ ID: 35)<br>
Analogous to synthesis of adhesion peptide A.<br>
Deprotection of adhesion and library peptides<br>
Final deprotection of protecting groups was performed according to the general procedure described above yielding the TCOB libraries ready for testing.<br>
Example 6<br>
Partial release of library compounds from beads<br>
The library beads of Examples 4 and 5 are swelled in the assay medium and illuminated with an OMNILUX E-40 (400W UV lamp, 365 nm, # 89514005, Steinigke<br><br>
Showtechnic GmbH, Germany) for 15 min. The beads could be placed in any environment (for example plates, wells etc. used for the assays). This treatment releases appropriate amounts of active peptide for assay use and the beads withholds enough active peptide for subsequent identification.<br>
Example 7<br>
Clearance of cells and adhesion peptide<br>
A portion of the library beads with cells attached were drained from medium and-treated with a 0.1 M NaOH solution for 1h. Beads were gently spinned down and the supernatant removed. The beads were washed with water (I0x) to ensure complete removal of cells and proteins.<br>
Example 8:<br>
Functional assay for identification of compounds that stimulate a Gs coupled signal transduction pathway (Cre-GFP reporter assay detected with a Fluorescence Activated Bead Sorter)<br>
Cre-GFP:<br>
Cre-GFP is commercially available from clontech (pCre-d2eGFP) The vector contains three copies of Cre-binding sequence fused to a TATA-like promoter. The vector is holding a neomycin resistance gene. A map of the vector is shown in figure 3. MC4R:<br>
PCR amplified MC4R encoding DNA is introduced into the gateway Entry Vector (pENTR) by topoisomarase-mediated ligation. The DNA is subsequently recombined into Destination Vector pDEST12.2. (pDEST12.2MC4R)<br>
Cell line establishment:<br>
HEK cells are transfected with pDEST1.2.2MC4R using standard procedure for FugeneG transfection. Cells are put under G418 selection for 4 weeks to obtain a cell line stably expressing MC4R.<br>
The U2OS cell line stably expressing the human MC4R (melanocortin4 receptor) is further transfected with Cre-GFP the day before culturing them on PEGA beads displaying adhesion peptide and respectively 1)Negative control (PEGA beads with adhesion peptide, but no library compound), 2)Positive control (PEGA beads with<br><br>
adhesion peptide but no library compound) and 3)Library compounds. The three cultures are handled separately in each their culture flask.<br>
Bead/cell preparation:<br>
Cells are trypsinized and mixed with the PEGA beads in growth medium (DMEM containing 10% PCS, in the proportion 4000 cells/bead and app. 50ml growth medium/5000 beads.<br>
1)	Positive control: 50ml Growth medium + 5000 positive control beads + 2x1 OE7<br>
cells* 10 pM Forskolin.<br>
2)	Negative control: 50ml Growth medium + 5000 negative control beads +  2x1 OE7<br>
cells.<br>
3)	Screening library (eg. 100.000 compounds): 1000ml Growth medium + 100.000<br>
library beads + 4x1 OE8 cells was illuminated with an OMNILUX E-40 UV light (365<br>
nm) for 30 min for obtaining a partial release of the library compounds (as described<br>
in example 6)<br>
The three culture flasks are placed on a Magnetic stirring platform (Techne) designed for cell culture in suspension and incubated at 37°, 5% CO2 for 16 - 24 hrs using spinning interval 30 rpm, 3 min stirring, 10min pause. Beads, now covered with cells, are allowed to sediment for 10 min (no centrifugation needed) and the growth medium is removed using a 50ml pipette. 10ml 99% EtOH per 5000 beads is added, mixed gently and left for 15 min. Beads are washed w. 10ml PBS/5000 beads x3 by allowing sedimentation for 10 min between each wash. Cells are now preserved and fixed to the beads<br>
Bead sorting:<br>
A Fluorescence Activated Bead Sorter (FABS) equipped with a multiline Argon laser<br>
488 nm excitation line and 500-650 nm emission filter and sorting capability into 96<br>
well plate is used to identify and isolate positive hit beads.<br>
The FABS is calibrated to identify and isolate positive hit beads (increased GFP<br>
fluorescence) by determining the dynamic range of the assay using positive control<br>
beads prepared as described above as Smax (maximum response) and negative<br>
control beads comprising only cell adhesion peptide as Smin (minimum response).<br>
A cut off at 30% response compared to negative control beads is set as threshold<br>
for a positive hit bead.<br><br>
Positive hits are separated into each their well of a 96 well plate and are hereafter ready for compound elucidation, re-synthesis and re-test as well as test for effects in other assays.<br>
Example 9:<br>
Functional assay for identification of compounds that inhibit a Gq coupled<br>
receptor (Muscarinic M1) signal transduction pathway (Ca++ mobilization using<br>
Fluo-4)<br>
Ca++ antagonist assay:<br>
This assay is designed to identify muscarinic M1 antagonist compounds. The read out is changes in intracellular Ca++ cone, detected using the Fluo-4 probe from Molecular probes (see description elsewhere). Positive hits are compounds that inhibit Carbacol (muscarinic M1 agonist) induced increase in intracellular Ca++.<br>
U20S cells are transfected with Muscarinic M1 receptor using standard procedure for Fugene6 transfection. Cells are put under zeocin selection for 4 weeks to obtain a cell line stably expressing the Muscarinic M1 receptor. U2OS cells expressing the Muscarinic M1 receptor are cultured on PEGA beads displaying adhesion peptide and respectively 1)Negative control (Beads comprising only cell adhesion compound ), 2)Positive control (beads comprising cell adhesion compound and Atropine) and 3)library compounds. The three cultures are handled separately in each their culture flask.<br>
Bead/cell preparation:<br>
Cells are trypsinized and mixed with beads in DMEM containing 10% FCS in the<br>
proportion 4000 cells/bead and app. 50ml growth medium/5000 beads.<br>
1)	Positive control: 50ml Growth medium + 5000 positive control beads + 2x1 OE7<br>
cells.<br>
2)	Negative control: 50ml Growth medium + 5000 negative control beads + 2x1 OE7<br>
cells.<br>
3)	Screening library (eg. 100.000 compounds): 1000ml Growth medium + 100.000<br>
library beads + 4x1 OE8 cells was illuminated with an OMNILUX E-40 UV light (365<br>
nm) for 30 min for obtaining a partial release of the library compounds (as described<br>
in example 6)<br><br>
The three culture flasks are placed on a Magnetic stirring platform (Techne) designed for cell culture in suspension and incubated at 37°, 5% CO2 for 16 - 24 hrs using spinning interval 30 rpm, 3 min stirring, 10min pause.<br>
Measurement of changes in the cytoplasmic free calcium concentration [Ca2*]j Beads, now covered with cells, are allowed to sediment for 10 min (no centrifugation needed) and the growth medium is removed using a 50ml pipette. 10ml Krebs Ringer buffer (KRW :KrebsRinaerWollheim. oH 7.4: NaCI 0.14 M, KCL 3.6 mM, NaH2PO4,H2O 0.5 mM, MgSO4, 7H2O O.S.mM, NaHCO3,2H20 1.5 mM, D-Glucose 6 mM, CaCI21.5 mM, HEPES 10 mM) added 1uM Fluo-4 (Molecular Probes F-14201) + 0.02% Pluronic (Molecular Probes F-127) per 5000 beads is added, mixed gently and cells/beads are incubated at 37°c for 30 min. Beads are hereafter washed w. 10ml KRW/5000 beads x3 by allowing sedimentation for 10 min between each wash. The Fluo-4 loaded cells are now ready for detection of changes in [Ca2*]j.<br>
The fluorescence is monitored in either a Fluorescence Activated Bead Sorter (FABS) (COPAS from Union Biometrica, US) that is equipped with multiple laser excitation lines (476 nm, 483 nm, 488 nm, 496 nm, 514 nm, 520 nm, 568 nm, 647 nm, 676 nm) or a fluorescence plate-reader (Polarstar Optima from BMG Labtech, Germany) that is equipped with a flash Xenon blitz lamp. Fluo4 fluorescence is detected in FABS by exciting with 488 nm and collecting the emitted light on to a PMT through a 530±30 nm emission filter, and on the plate-reader the cells were excited through a 490±5 nm excitation filter and the emission collected through a 510±5 nm emission filter. For calculation of the exact [Ca2*]i the fluorescence intensity was converted to [Ca2+]i by using the equation [Ca2*]i=KD[(F-Fmjn)/(Fm8x-F)] where the dissociation constant KD is 345 nM, F is fluorescence intensity, Fmin is total fluorescence in the absence of Ca2* and Fmax is total fluorescence when Fluo3 is saturated with Ca2*. To obtain Fmin the cells were pre-incubated in a calcium low buffer (pJH L4:_NaCI 0.14 M, KCL 3.6 mM, NaH2PO4,H2O 0.5 mM, MgSO4, 7H2O O.S.mM, Na-HC03,2H2O 1.5 mM, D-Glucose 6 mM, EGTA 1.5 mM, HEPES 10 mM) and was challenged with 1 uM ionomycin immediately before the fluorescence detection. Similarly Fmax was obtained by suspending the cells in a calcium saturated buffer (pH 7.4: NaCI 0.14 M, KCL 3.6 mM, NaH2PO4,H2O 0.5 mM, MgSO4, 7H2O O.S.mM,<br><br>
NaHCO3,2H2O 1.5 mM, D-Glucose 6 mM, CaCI21.5 mM, HEPES 10 mM) and challenged with 1 uM ionomycin immediately before detection. In several of our screening assay we do not use exact ion [Ca2+]i, but express the response of screening compounds as relative to control compounds (see below).<br>
Bead sorting for Fluo-4 signal:<br>
A Fluorescence Activated Bead Sorter (FABS) equipped with 488 nm laser excitation line and 528-572 nm emission filter and injection capability is used to identify and isolate positive hit beads (=inhibition of Carbachol induced Ca++ response = decreased fluorescence compared to negative control).<br>
The FABS is calibrated to identify and isolate positive hit beads by determining the dynamic range of the assay using positive control beads as Smax (maximum inhibition = minimal fluorescence) and negative control beads as Smin (minimum inhibition = maximal fluorescence). Carbacol 1 uM is injected into the flow stream resulting in an increase in fluorescence for negative control beads and an unchanged or minor increase in fluorescence for positive control beads. A cut off at 30% inhibition compared to negative control beads is set as threshold for a positive hit bead. Positive hits are separated into each their well of a 96 well plate and are hereafter ready for compound elucidation, re-synthesis and re-test as well as test for effects in other assays.<br>
Example 10<br>
Functional assay for identification of compounds that stimulate a Gs coupled signal transduction pathway (Cre-GFP reporter assay detected with fluorescence plate reader or fluorescence imaging equipment)<br>
Cre-GFP:<br>
Cre-GFP is commercially available from clontech (pCre-d2eGFP) The vector contains three copies of Cre-binding sequence fused to a TATA-like promoter. The vector is holding a neomycin resistance gene. A map of the vector is shown in figure 3.<br>
MC4R:<br><br>
PCR amplified MC4R encoding DMA is introduced into the gateway Entry Vector (pENTR) by topoisomarase-mediated ligation. The DNA was subsequently recom-bined into Destination Vector pDEST12.2. (pDESTl2.2MC4R)<br>
Cell line establishment:<br>
U2OS cells are transfected with pDESTI .2.2MC4R using standard procedure for FugeneG transfection. Cells were put under G418 selection for 4 weeks to obtain a cell line stably expressing the MC4R.<br>
The U2OS cell line stably expressing the human MC4R (melanocortin4 receptor) is further transfected with Cre-GFP the day before culturing them on PEGA beads displaying adhesion peptide and respectively 1)Negative control (PEGA beads with adhesion peptide, but no library compound), 2)Positive control (PEGA beads of example 2) and 3)Library compounds. The three cultures are handled separately in each their culture flask.<br>
Bead/cell preparation:<br>
Cells are trypsinized and mixed with the PEGA beads in growth medium (DMEM containing 10% PCS, in the proportion 4000 cells/bead and app. 50ml growth medium/5000 beads.<br>
1)	Positive control: 50ml Growth medium + 5000 positive control beads + 2x1 OE7<br>
cells.<br>
2)	Negative control: 50ml Growth medium + 5000 negative control beads +  2x1 OE7<br>
cells.<br>
3)	Screening library (eg. 100.000 compounds): 1000ml Growth medium + 100.000<br>
library beads + 4x1 OE8 cells was illuminated with an OMNILUX E-40 UV light (365<br>
nm) for 30 min for obtaining a partial release of the library compounds (as described<br>
in example 6)<br>
The three culture flasks are placed on a Magnetic stirring platform (Techne) designed for cell culture in suspension and incubated at 37°, 5% CO2 for 16 - 24 hrs using spinning interval 30 rpm, 3 min stirring, 10min pause. Beads, now covered with cells, are allowed to sediment for 10 min (no centrifugation needed) and the growth medium is removed using a 50ml pipette. 10ml 99% EtOH per 5000 beads is added, mixed gently and left for 15 min. Beads are washed w.<br><br>
10ml PBS/5000 beads x3 by allowing sedimentation for 10 min between each wash. Cells are now preserved and fixed to the beads.<br>
Plate reader assay:<br>
Control beads as well as library beads are seeded in 384 well black plates (eg. Nunc) with clear bottom app. 20 beads per well. Positive and negative controls are placed in dedicated wells in 2 times 4 replicates in each end of the plate. Negative control = 20 negative control beads, positive control = one positive control bead + 19 negative control beads. The plates are measured in a fluorescence plate reader (PolarStar Optima from BMG) using 490 +-6nm excitation filter and 510 +-5nm emission filter. Positive control wells are used to determine Smax (maximum response) = 100% activity and negative control wells to determine Smin (minimum response) = 0% activity. Beads from wells showing activity &gt; 30% are collected in a tube for re-seeding in a new 384 well plate, this time having one bead per well. Smin = one negative control bead and Smax = one positive control bead. Read plates in plate reader and identify hits beads using same procedure as described above.<br>
Image acquisition and analysis :<br>
Control beads as well as library beads are seeded in 384 well black plates (eg. Nunc) with clear bottom app. 20 beads per well. Positive and negative control beads are placed in dedicated wells in 2 times 4 replicates in each end of the plate. Negative control = 20 negative control beads, positive control = one positive control bead + 19 negative control beads. Plates are placed on a microscope (Zeiss Axiovert 200M) equipped with filters allowing fluorescence imaging of eGFP (excitation: 490nm, emission: 510nm), 10x objective and motorized stage. One image is acquired for each well followed by image analysis (Metamorph) for identification of hit beads (green). Beads from hit wells are seeded in a new 384 well plate this time having one bead per well. Smin = one negative control bead and Smax = one positive control bead. Image acquisition and analysis described above is repeated and final hit beads are identified.<br>
Alternatively, approximately 5000 beads are seeded into Lab-Tech Chambered Coverglass System (#155361; Nalge Nunc INternational), imaging acquisition analysis is performed using the fluorescence equipment described above, and indi-<br><br>
vidual beads that display the required fluorescence properties are isolated using a micromanipulator system (Eppendorf Injectman NK).<br>
Example 11<br>
Assay for identification of protein-protein interaction (ML-IAP:Smac) modulators.<br>
The present assay that is based on the principle: Bioluminescence Resonance Energy Transfer (BRET2), commercially available from Perkin Elmer. One of two interacting proteins is fused to a bioluminescent donor (luciferase, Rluc) and the other protein is fused to a fluorescent acceptor (GFP). Melanoma Inihitor of Apoptosis Protein (ML-IAP) probe:<br>
The probe is constructed by PCR amplification of the ML-IAP (GenBank Accession number. NM_139317 (alpha form); NM_0221.61 (beta form) (the accesion number refer to the sequence available on 25 May 2005) from human cDNA libraries (Marathon-Ready cDNA Library of human kidney cell line and a human fetal brain cell line, Clontech Laboratories, Inc.) using the primers Iivin-F1 (5'-CCA GTG TTC CCT CCA TGG GAC CTA A-3')(SEQ ID 71) and Iivin-R1 (5'-TAA GCC ATC CCC CAC GCC AAG-3')(SEQ ID 72). The ML-IAP cDNA gene fragments are further modified to contain restriction sites by PCR amplification using the primers Livin-F3 (5'-GAT AAG CTT CCA GTG TTC CCT CCA TGG GA-3')(SEQ ID 73), Livin-R3 (5'-TAT GGA TCC AAG GTG CGC ACG CGG CT-3')(SEQ ID 74), and Livin-F4 (5'-GAG AAT TCT CCT AAA GAC AGT GCC AAG TG-3')(SEQ ID 75) for amplification of full-length ML-IAP. The primers Livin-BIR-F1 (5'-CAT GGTACC ATG ACA GAG GAG GAA GAG GAG-3') (SEQ ID 76) and Livin-BIR-R1 (51- GC TGG ATC CGG GTC CCA GGA GCC CAG-3') (SEQ ID 77) are used for amplification of the BIR domain of ML-IAP.. The restriction enzyme-treated amplificons are ligated in the BRET2 vectors (obtained from Perkin Elmer, Inc.) to produce N- and C-terminal in-frame fusions to GFP and Rluc.<br>
Smac probe:<br>
The probe is constructed by ligating restriction enzyme treated PCR amplification products of the cDNA for human Smac (GenBank Accession NM_019887 (variant 1); NM_138929 (variant 3)) in to the BRET2 vectors (obtained from Perkin Elmer,<br><br>
Inc.) to produce C-terminal in-frame fusions to GFP and Rluc, respectively. The following primers are used for PCR: Smac-F1 (5'-GCG CTG CAC AAT GGC GGC TCT-3')(SEQ ID 78), Smac-R1 (5'-GCA CTC ACA GCT CAC AAA GGC GTC T-3')(SEQ ID 79), Smac-F3 (S'-GAT GGT ACC CGC TGC ACA ATG GCG GCT CT-3')(SEQ ID 80), and Smac-R3 (5'-CGT GGA TCC TCA CGC AGG TAG GCC TCC-3')(SEQ ID 81). Cytosolic expression of biologically Smac may be achieved by constructing an ubiquitin-smac fusion as described by Allison M. Hunter, Dan Kottach-chi, Jennifer Lewis, Colin S. Duckett, Robert G. Korneluk, and P. Listen. A Novel Ubiquitin Fusion System Bypasses the Mitochondria and Generates Biologically Active Smac/DIABLO. J.Biol.Chem. 278 (9):7494-7499, 2003.<br>
Cell line establishment:<br>
Cells are co-transfected with the BRET fusions of ML-IAP and Smac using standard procedure for Fugene6 transfection. Cells are put under G418 and zeocin selection for 4 weeks to obtain a cell line stably expressing the two genes.<br>
Selection of stable cells:<br>
The HeLa cell line stably expressing both probes are sorted into a microtitre plate (one cell per well) for high GFP fluorescence using a Fluorescence Activated Cell Sorter. The sorted cells are grown for a week, and split into a plate that is obtimized for bio-luminescence. Final cell-clones are selected, after addition of the luciferase substrate, DeepBlueC (Perkin Elmer), for high lumiscence (detected with a Polar-Star, BMG).<br>
Bead/cell preparation:<br>
HeLa/mammalian cells are cultured on PEGA beads displaying adhesion peptide and respectively 1) Negative control (PEGA beads with adhesion peptide, and a compound with no activity as the library component), 2) Positive control (PEGA beads with adhesion peptide, and a control compound, which disrupts the ML-IAP: Smac interaction e.g. the compound of Example 20 of WO2004/005248, and 3) Library compounds (PEGA beads with adhesion peptide, and screening compounds prepared as described in example 5b or 5c). The three cultures are handled separately in each their culture flask.<br><br>
Cells are trypsinized and mixed with the PEGA beads in growth medium (DMEM containing 10% PCS, in the proportion 4000 cells/bead and app. 50ml growth medium/5000 beads. Culture flasks are placed on a Magnetic stirring platform (Techne) designed for cell culture in suspension and incubated at 37°, 5% CO2 for 16 - 24 hrs using spinning interval 30 rpm, 3 min stirring, 10min pause. Alternatively, the culture flasks are gently rocked a few times. Cell-coated beads are treated as follows:<br>
1)	Positive control (low BRET signal): 50ml Growth medium + approx. 5000 cell-<br>
coated control beads.<br>
2)	Negative control (high BRET signal): 50ml Growth medium + approx. 5000 cell-<br>
coated control beads 3) Screening library (eg. 100.000 compounds): 1000ml Growth<br>
medium + approx. 100.000 cell-coated library beads.<br>
The cell-coated beads are transferred to assay buffer (either KRW (Krebs Ringer Wollheim) or PBS (phosphate buffered saline) and illuminated with UV light (365 nm) for 2-20 min for obtaining a partial release of the library compounds (as described in example 6). The illuminated beads are further incubated at 37°, 5% C02 for 15 min. Beads, now covered with cells, are allowed to sediment for 10 min (no centrifugation needed) and the growth medium is removed using a 50ml pipette.<br>
Positive beads may be indentified by analysing individual beads in a BRET-compatible microplate reader using 410-80 (370-450 nm) and 515-30 (500-530 nm) bandpass filters for dual emission measurements of Rluc-luminescence and GFP-fluorescence, respectively. The luciferase substrate, DeepBlueC, is added by an on-board injector in the microplate reader! The BRET-signal for the protein-protein interation is calculated as the ratio of emission at 515 nm and emission at 410 nm.<br>
Alternatively postivive beads may be identified by Bead sorting: A Fluorescence Activated Bead Sorter (FABS) equipped with filters for BRET detection and sorting capability into 96 well plate is used to identify and isolate positive hit beads.<br>
The luciferase substrate, OeepBlueC is injected into the flow stream to emit lucifer-ase-mediated luminescence.<br><br>
The FABS is calibrated to identify and isolate positive hit beads (decreased BRET signal) by determining the dynamic range of the assay using positive control beads prepared as described in above as Smax (maximum response) and negative control beads comprising only cell adhesion peptide as Smin (minimum response). Positive hits are separated into each their well of a 96 well plate and are hereafter ready for compound elucidation, re-synthesis and re-test as well as test for effects in other assays.<br>
Example 12 - Caspase activity assay<br>
Inhibitor of apoptosis proteins (lAPs) interact with caspases to inhibit their activation and thereby cause a repression of apoptosis. We seek to identify compounds, which can bind to IAP with a higher affinity than caspase and, in turn, relieve lAP-mediated caspase-inhibition. A screening assay is established that measures caspase activation in response to an apoptosis inducing signal (e.g. staurosporine, STS) and in the presence of library compounds with putative lAP-binding activity. The assay is based on measurements of caspase activity in whole mammalian cells (HeLa/MCF-7/MeWo), preferably HeLa/SK-Mel28 cultured on PEGA beads displaying adhesion peptide and respectively 1) Negative control (PEGA beads with adhesion peptide, and a compound with no activity as the library component), 2) Positive control (PEGA beads with adhesion peptide, and a control peptide that disrupts the ML-IAP:Smac interaction prepared as described in example 3 and 3) Library compounds (PEGA beads with adhesion peptide, and screening compounds prepared as described in example 5b or 5c). The three cultures are handled separately in each their culture flask.<br>
Caspase activity is measured in bead-attached whole cells by incubating the cells with a fluorogenic caspase substrate with a quenching leaving group, e.g. DEVD-Rh1 10-C8 or the CellProbe HT Caspase 3/7 Whole Cell Assay, Beckman Coulter, Inc.). A fluorogenic counter stain with different emission properties is applied to correct for cellmass content on the individual beads.<br>
Caspase substrate (DEVD-R-C8) is described in the following publication: Sui Xiong Cai, Han Zhong Zhang, John Guastella, John Orewe, Wu Yang, and Eckard Weber. Design and synthesis of Rhodamine 110 derivative and Caspase-3 substrate for enzyme and cell-based fluorescent assay. Bioorganic &amp; Medicinal Chemistry Letters<br><br>
11 (1):39-42, 2001 and in the patent: US 6,342,611 B1 (Jan 29, 2002, Cytovia, Inc. San Diego, CA)<br>
Bead/cell preparation:<br>
Cells are trypsinized and mixed with the PEGA beads in growth medium (DMEM containing 10% PCS, in the proportion 4000 cells/bead and app. 50ml growth medium/5000 beads.<br>
The beads/cells are placed in three culture flasks and placed on a Magnetic stirring platform (Techne) designed for cell culture in suspension incubated at 37°, 5% CO2 for 16-24 hours using spinning interval 30 rpm, 3 min stirring, 10min pause. Cell-coated beads are treated as follows:<br>
1) Positive control: 50ml Growth medium + approx. 5000 Positive control beads. 2) Negative control: 50ml Growth medium + approx. 5000 Negative control beads. 3) Screening library (eg. 100.000 compounds): 1000ml Growth medium + approx. 100.000 cell-coated Library compound beads.<br>
Screening procedure:<br>
The cell-coated beads are transferred to assay buffer (KRW or PBS and illuminated with UV light (365 nm) for 30 min for obtaining a partial release of the library compounds (as described in example 6). The three culture flasks are placed on a Magnetic stirring platform (Techne) designed for cell culture in suspension and incubated at 37°, 5% C02 for 3 to 6 hrs.<br>
The beads are then allowed to sediment for 10 min (no centrifugation needed) and the growth medium is removed using a 50ml pipette. 1 ml medium containing the fluorogenic caspase substrate and reagent buffer is added to the beads. The beads are incubated without stirring at 37°, 5% CO2 for 0.5-5 hours. The beads are then fixed by treatment with 10ml 99% EtOH per 5000 beads, gentle mixing whereafter they are left for 15 min. Beads are washed w. 10ml PBS/5000 beads x3 by allowing sedimentation for 10 min between each wash. Alternatively, beads may be fixed with paraformaldehyde, acetone or zinc-based fixatives. Cells are now preserved and fixed to the beads.<br><br>
Bead sorting:<br>
The beads are analysed in a fashion similar to the procedure described in example 11. The Fluorescence Activated Bead Sorter (FABS) is equipped with 488 nm excitation filters and 500-550 nm emission filter for rhodamine-110 measurments and sorting capability into 96 well plate is used to identify and isolate positive hit beads. The FABS is calibrated to identify and isolate positive hit beads (increased caspase activity) by determining the dynamic range of the assay using positive control beads with apoptosis induced cells to determine the Smax (maximum response) and negative control beads with uninduced cells as Smin (minimum response). Positive hits are separated into each their well of a 96 well plate and are hereafter ready for compound elucidation, re-synthesis and re-test as well as test for effects in other assays. Microscopy:<br>
An alternative to to FABS-based selection of positive hit beads is epifluorescence microscopy using standard FITC filters (excitation 480±15 nm, emission 535±20 nm) for visualization of rhodamine 110 fluorescence. The microscope is equipped with micromanipulators for extraction of positive hit beads.<br>
Example 12b<br>
Detection of caspase activity using proximity ligation-based assay<br>
Caspase activity is detected with an assay based on the proximity ligation technology described by S. Fredriksson, M. Gullberg, J. Jarvius, C. Olsson, K. Pietras, S. M. Gustafsdottir, A. Ostman, and U. Landegren. Protein detection using proximity-dependent DMA ligation assays. Nat.Biotechnol. 20 (5):473-477, 2002. This assay allows direct detection of both active caspases and cleaved caspase substrates for example PAPR. Furthermore, the assay can be applied to fixed samples of primary tissue.<br>
We have developed the proximity ligationtechnique to emcompass a high volume discovery platform that allow identification of primary hit compounds in a screening system that because of the ability to conduct the screening in primary cells closely resembles the conditions observed directly in the patient.<br><br>
The assay employs two antibodies, an antibody specific for cleaved caspase-3 and and an antibody for recognizing both the pro-enzmye and the cleaved form of caspase-3. Both antibodies are conjugated with an oligo nucleotide. Coordinated and proximal binding of the two antibodies to an activated caspase allows hybridisation and ligation of DNA probes which can be amplicifed by rolling circle amplification using flourescently labelled oligonucleotides or molecular beacons. The quantitative incorpoartion of a fluorescence label may thus be done like in quantitative real-time PCR. Detection and selection of beads with a positive fluorescence signal is conducted either with the Fluorescence-Activated-Bead-Sorter or a fluorescence microscope or a plate reader.<br>
Example 13<br>
Identification of compound<br>
Once a resin bead is selected, the cells may be cleared off the beads either by extensive washing or in case of HMBA -linked adhesion peptides by treatment with 0.1 M NaOH followed by washing (example 7). The library compound comprised within the bead may then be identified. Selected bead(s) are washed and swelled in a small drop of pure water and irradiated for 30 min. with an OMNILUX E-40 (400W UV lamp, 365 nm, # 89514005, Steinigke Showtechnic GmbH, Germany). The compound is identified with advanced mass spectrometry combined with single bead and/or nano-scale NMR techniques. For example advanced MS may be ES MS-MS analysis on a MicroMass QTOF Global Ultima mass spectrometer (mobile phase 50% CH3CN (aq), 0.1 nL/min) employing a linear ramping of the collision energy. The spectra are analyzed by generating the exact mass differences between fragment ions and tabulated to provide the fragmentation pathway and from that the structure of the compound released from the selected bead is elucidated. Examples of spectra, mass differences and fragmentation pathways are given in figures 5 to 7.<br><br>
Abreviations:<br>
HGF: Hepatocyte Growth Factor<br>
NGF: Nerve Growth Factor<br>
PDGF: Platelet Derived Growth Factor<br>
FGF: Fibroblast Growth Factor<br>
EGF: epidermal Growth Factor<br>
GH: Growth hormone<br>
TRE: TPA Response Element<br>
SRE: serum response element<br>
CRE: cAMP response element<br>
AcN: acetonitril;<br>
Boc: tert-butoxycarbonyl; 'Bu: tert-butyl;<br>
DCM: dichloromethane;<br>
DMF: dimethylformamide;<br>
Fmoc: 9-fluorenylmethoxycarbonyl;<br>
HMBA: 4-hydroxymethylbenzoic acid;<br>
Q-TOF MS: quadrupole time-of-flight mass spectrometry;<br>
Melm: /V-methyl imidazole;<br>
MSNT: 1-(mesitylene-2-sulphonyl)-3-nitro-1 H-1,2,4-triazole;<br>
NEM: W-ethyl morpholine;<br>
PEGA: polyethylene glycol-polydimethyl acrylamide resin;<br>
Pfp: pentafluorophenyl;<br>
Pmc: 2,2,5,7,8-pentamethylchroman-6-sulfonyl;<br>
RP-HPLC: reversed phase high pressure liquid chromatography;<br>
SPPS: solid phase peptide synthesis;<br>
TBTU: O-(benzotriazol-1 -yl-A/,N,N',N -tetramethyluronium tetrafluoroborate;<br>
TCOB: Two-compound-one-bead<br>
TFA: trifluoro acetic acid;<br>
Trt: Trityl.<br><br>
1.	A method of Identifying a compound modifying at least one cellular response, wherein each<br>
cellular response is linked to different reporter systems generating detectable outputs, said<br>
method comprising the steps of:<br>
(a)	Providing multiple solid supports capable of supporting adherence and growth of cells,<br>
wherein each solid support is linked to multiple copies of a member of a library of test<br>
compounds via deavabte linkers and wherein at least two solid supports comprise<br>
different library members; and<br>
(b)	Attaching cells comprising said reporter system(s) onto said solid support; and<br>
(<: releasing a proportion of said library member from the solid support and></:>
(d) Screening said solid supports for solid supports comprising cells meeting at least one<br>
predetermined selection criterion, wherein said selection criterion is linked directly or<br>
indirectly to said detectable output; and (o) Selecting solid supports comprising celts meeting said at least one selection criterion;<br>
and (f) Identifying said library member, thereby identifying a compound modifying said at least<br>
one cellular response.<br>
2.	The method according to dalm 1, wherein said adherence is mediated through a cell adhesion<br>
compound coupled to said solid supports, wherein said cell adhesion compound enables said<br>
solid support to support growth of cells.<br>
3.	The method according to any of claims 1 and 2, wherein the solid supports are resfn beads.<br><br>
4.	The method according to daim 3, wherein the resin bead comprises or consists of polyethylene<br>
glyco<br>
5.	The method according to daim 2, wherein said ceil adhesion compound is a peptide with an<br>
overall positive netcharge.<br>
6.	The method according to claim 2, wherein said ceil adhesion compound is selected from the<br>
group consisting of SEQ ID 1, SEQ ID 2, SEQ ID 3, SEQ ID 4, SEQ ID 5, SEQ ID 6, SEQ ID 7,<br>
SEQ ID 8, SEQ ID 9, SEQ ID 10, SEQ ID 11, SEQ ID 12, SEQ ID 13, SEQ ID 14, SEQ ID 15,<br>
SEQ ID 16, SEQ ID 17, SEQ ID 18, SEQ ID 19, SEQ ID 20, SEQ ID 21, SEQ ID 22, SEQ ID<br><br>
23, SEQ 10 24, SEQ ID 25, SEQ ID 26, SEQ ID 27, SEQ ID 28, SEQ ID 28, SEQ ID 30, SEQ ID 31, SEQ ID 32, SEQ ID 33, SEQ ID 34, SEQ ID 35, SEQ ID 46, SEQ ID 47, SEQ ID 48, SEQ ID 49, SEQ ID 50, SEQ ID 51, SEQ ID 52, SEQ ID 53, SEQ ID 54, SEQ ID 55, SEQ ID 56, SEQ ID 57, SEQ ID 58, SEQ ID 59, SEQ ID 60, SEQ ID 61, SEQ ID 62, SEQ ID 63, SEQ ID 64, SEQ ID 65, SEQ ID 66, SEQ ID 67, SEQ ID 68, SEQ ID 69 and SEQ ID 70.<br>
7.	The method according to any of daims 2 to 6, wherein the adhesion compound is linked to the<br>
resin bead via a cleavable linker.<br>
8.	The method according to any of claims 1 to 7, wherein the cellular response is change In<br>
Interaction between two or more cellular molecules.<br>
9.	The method according to claim 8, wherein the cellular response is change in interaction<br>
between proteins involved in regulation of apoptosis.<br>
10.	The method according to any of claims 1 to 7, wherein said cellular response is modulation of a<br>
signal transduction pathway.<br>
11.	The method according to any of claims 1 to 7 and 10, wherein said cellular response is<br>
modulations of signal transduction pathway, wherein said modulation is selected from the<br>
group consisting of<br><br>
•	Upregulation or downregulation of the level of a member of the pathway;<br>
•	Relocalisation of a member of the pathway;<br>
•	Complex formation between members of the pathway or between members of the pathway<br>
with other cellular compounds;<br>
•	Enhanced or reduced transcription from genes regulated by the pathway;<br>
•	Modification by for example phosphorylation of a member of the pathway;<br>
•	Activation or inhibition of an enzyme of the pathway;<br>
•	Degradation of a cellular compounds due to upregulation or downregulation of the pathway;<br>
•	Altered secretion of a compound;<br>
•	Change in ion-flux;<br>
•	Morphological changes; and<br>
•	Change in viability<br><br>
12.	The method according to any of claims 1 to 9, wherein the cellular response is apoptosis.<br>
13.	The method according to any of claims 1 to 12, wherein the reporter system is a system<br>
endogenous to said cells.<br>
14.	The method according to any of daims 1 to 12, wherein the reporter system comprises a<br>
biolumlnescent moiety.<br>
15.	The method according to any of claims 1 to 14, wherein the reporter system is a proximity<br>
llgation reporter system.<br>
16.	The method according to any of claims 1 to 15, wherein one predetermined selection criterium<br>
is a quantitative level of said bioluminiscence above or below a specific threshold.<br>
17.	The method according to any of claims 1 to 16, wherein said ceils are selected from the group<br>
consisting of mammalian cells.<br>
18.	The method according to any of claims 3 to 17, wherein at least 100 resin beads comprising<br>
different library members are provided.<br>
10. The method according to any of claims 1 to 18, wherein the library is selected from the group consisting of libraries of natural oligomers, such as peptldes, grycopeptktes, llpopeptides, nucle-c acids (DNA or RNA), or oHgosaccharides; unnatural oligomers, such as chemically modified peptides, oligomers of naturally occurring amino adds and/or any other kinds of amino acids, gfycopaptides, nucleic acids (DNA or RNA) or oligosaccharides; and small organic molecules.<br>
20.	The method according to any of claims 1 to 20, wherein the library comprises polymers of in<br>
the range of 3 to 6 amino adds selected from the group of amino adds mentioned in table 3,<br>
table 4, table 5 and table 6.<br>
21.	The method according to any of claims 1 to 21, wherein compounds modifying at least two<br>
cellular responses are identified, wherein step c) involves screening said resin beads for beads<br><br>
comprising cells meeting at least two predetermined selection criterion, wherein each selection criterion is related to a different detectable output.<br>
22.	The method according to any of claims 1 to 22, wherein at least one cleavable linker Is<br>
selected from the group consisting of acid-labile, base-labile, fluoride-labile and photo-labile<br>
linkers.<br>
23.	The method according to any of claims 1 to 57, wherein step c) comprises releasing in the<br>
range of B to 95% of the copies of the library member.<br>
24.	A library useful in the method according to any of the preceding claims, comprising multiple<br>
solid supports linked to multiple copies of a member of a library of test compounds via<br>
cleavable linkers and wherein at least two solid supports comprise different library members;<br>
and furthermore wherein the solid supports are coupled to a cell adhesion compound, wherein<br>
said cell adhesion compound is selected from either:<br>
i) the group consisting of peptides of: SEQ ID 1, SEQ ID 2, SEQ ID 3, SEQ ID 4, SEQ (D 5, SEQ ID 6, SEQ ID 7, SEQ ID 8, SEQ ID 9, SEQ ID 10, SEQ ID 11, SEQ ID 12. SEQ ID 13, SEQ ID 14, SEQ ID 15, SEQ ID 16, SEQ ID 17, SEQ ID 18, SEQ ID 19, SEQ ID 20, SEQ ID 21, SEQ ID 22, SEQ ID 23, SEQ ID 26, SEQ ID 27, SEQ ID 28, SEQ ID 29, SEQ ID 30, SEQ ID 31, SEQ ID 32, SEQ ID 33, SEQ ID 34, SEQ ID 35, SEQ ID 46, SEQ ID 47, SEQ ID 48, SEQ ID 49, SEQ ID 50, SEQ ID 51, SEQ ID 52, SEQ ID 53, SEQ ID 54, SEQ ID 55, SEQ ID 56, SEQ ID 57, SEQ ID 58, SEQ ID 59, SEQ ID 60, SEQ ID 61, SEQ ID 62, SEQ ID 63, SEQ ID 64, SEQ ID 65, SEQ ID 66, SEQ ID 67, SEQ ID 68, SEQ ID 69 and SEQ ID 70<br>
or:<br>
il) a peptide comprising at least one D-form amino acid, said peptkle being selected from the group consisting of:<br>
SEQ ID 1, SEQ ID 2, SEQ ID 3, SEQ ID 4, SEQ ID 5, SEQ ID 6, SEQ ID 7, SEQ ID 8, SEQ ID 9, SEQ ID 10, SEQ ID 11, SEQ ID 12, SEQ ID 13, SEQ ID 14, SEQ ID 15, SEQ ID 16, SEQ ID 17, SEQ ID 18, SEQ ID 19, SEQ ID 20, SEQ ID 21, SEQ ID 22, SEQ ID 23, SEQ ID 24, SEQ ID 25, SEQ ID 26, SEQ ID 27. SEQ ID 28, SEQ ID 29, SEQ ID 30, SEQ ID 31, SEQ ID 32, SEQ ID 33, SEQ ID 34, SEQ ID 35, SEQ ID 46, SEQ ID 47, SEQ ID 48, SEQ ID 49, SEQ ID 50, SEQ ID 51, SEQ ID 52, SEQ ID 53, SEQ ID 54, SEQ ID 55, SEQ ID 56, SEQ ID 57, SEQ ID 58, SEQ ID 59,<br><br>
SEQ ID 60, SEQ ID 61, SEQ ID 62, SEQ ID 63, SEQ ID 64, SEQ ID 65, SEQ ID 66, 8EQ ID 67, SEQ ID 68, SEQ ID 69 and SEQ ID 70.<br>
25.	A method of manufacturing a compound modifying at least one cellular response, wherein said<br>
method comprises the steps of:<br>
a)	Identifying said compound by the method according to any of claims 1 to 23<br>
b)	Preparing said compound by chemical synthesis<br>
c)	Thereby manufacturing said compound<br>
26.	A meltrod of modulating the interaction between two or more cellular molecules comprising the<br>
steps of<br>
a)	identifying a compound by the method according to any of claims 8 to 9<br>
b)	Incubating said compound together with cells expressing said two cellular molecules<br>
c)	Thereby modulating the interaction between the two cellular molecules<br><br>
27.	Compound identified by the method according to any of claims 1 to 23.<br>
28.	The compound according to claim 27, wherein the compound is an oligomer of in the range of<br>
3 to 6 amino acids selected from the group of amino acids mentioned in table 3, table 4, table 5<br>
and table 6.<br>
29.	A library comprising at least two dlgomers of in the range of 3 to 6 amino acids selected from<br>
the group of amino acids mentioned in table 3, table 4, table 5 and table 6.<br><br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUFic3RyYWN0LSgyNy0wMi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Abstract-(27-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUNsYWltcy0oMTItMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Claims-(12-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUNsYWltcy0oMjctMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Claims-(27-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTItMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Correspondence Others-(12-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMTgtMTItMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Correspondence-others (18-12-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTQtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Correspondence-Others-(14-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Correspondence-Others-(27-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1ERUxOUC0yMDA2LUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7681-DELNP-2006-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUZvcm0tMTMgKDE4LTEyLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Form-13 (18-12-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUZvcm0tMi0oMTItMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Form-2-(12-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUZvcm0tMi0oMjctMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Form-2-(27-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LUZvcm0tMy0oMTItMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-Form-3-(12-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1ERUxOUC0yMDA2LUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7681-DELNP-2006-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY4MS1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">7681-delnp-2006-pct-search report.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260472-parking-assist-method-and-apparatus-for-estimating-the-parking-space-orientation.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260474-a-control-system-and-mehtod-for-processing-a-sensor-signal-for-a-vehicle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260473</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7681/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-May-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Apr-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Dec-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>2cureX</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BIRKEVEJ 37, 3460 BIRKEROD, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HAGEL, GRITH</td>
											<td>HAREVAENGET 109, DK-2791 DRAGOR, (DK)</td>
										</tr>
										<tr>
											<td>2</td>
											<td>THASTRUP, OLE</td>
											<td>BIRKEVEJ 37, DK-3460 BIRKEROD, (DK)</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MELDAL, MORTEN</td>
											<td>MOSESVINGET 78, DK-2400 COPENHAGEN (DK)</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NORRILD, JENS, CHR.</td>
											<td>CARL PLOUGS VEJ 26, DK-3460 BIRKEROD, (DK)</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HENTZER, MORTEN</td>
											<td>SOLBAKKEN 32, DK-4300 HOLBAEK (DK)</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK2005/000347</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-05-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2004 00821</td>
									<td>2004-05-25</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 2004 00822</td>
									<td>2004-05-25</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260473-an-in-vitro-method-of-identifying-compounds-modifying-cellular-responses by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:22:37 GMT -->
</html>
