<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260549-oxidase-polypeptide-and-nucleic-acid-encoding-the-same-and-vector-and-transformed-host-cell-comprising-the-nucleic-acid-and-uses-of-the-oxidase-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:45:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260549:&quot;OXIDASE POLYPEPTIDE AND NUCLEIC ACID ENCODING THE SAME, AND VECTOR AND TRANSFORMED HOST CELL COMPRISING THE NUCLEIC ACID, AND USES OF THE OXIDASE POLYPEPTIDE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;OXIDASE POLYPEPTIDE AND NUCLEIC ACID ENCODING THE SAME, AND VECTOR AND TRANSFORMED HOST CELL COMPRISING THE NUCLEIC ACID, AND USES OF THE OXIDASE POLYPEPTIDE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a new isolated nucleic acid sequence comprising a gene that encodes a new fungal oxidase enzyme, and nucleic acid fragments thereof. It also relates to said new fungal oxidase isolated and purified from Aspergillus ficuum, its amino acid sequence as shown in SEQ ID NO 40 or 41, and functional equivalents or derivatives thereof. The present invention also relates to constructs, vectors and hosts cells comprising a nucleic acid molecule of the invention, as well as methods for producing an oxidase of the invention. The present invention also relates to the use of an oxidase of the invention in industrial processes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention<br>
The present invention relates to a new isolated nucleic acid sequence comprising a gene that encodes a new fungal oxidase enzyme and nucleic acid fragments thereof.<br>
It also relates to said new fungal oxidase isolated and purified from Aspergillus ficuum its amino acid sequence as shown in SEQ ID NO 40 or 41 and functional equivalents or derivatives thereof.<br>
The present invention also relates to constructs vectors and hosts cells comprising a nucleic acid molecule of the invention as well as methods for producing an oxidase of the invention.<br>
The present invention also relates to the use of an oxidase of the invention in industrial processes.<br>
Background of the invention<br>
Oxidases are enzymes that catalyze many kinds of biological oxidations.<br>
Among  these  oxidases  for  example  laccases<br>
(also referred to as polyphenol oxidases ; EC 1.10.3.1.  ;<br>
benzenediol:oxygen    oxidoreductases)    are    multi-copper<br>
containing enzymes that catalyze the oxidation of a variety of<br>
phenolic compounds with concomitant reduction of 02 to<br>
These polyphenol oxidases are widely spread and produced by a wide variety of (1) fungi including (a) ascomycetes such as Aspergillus Neurospora or Podospora (b) the deuteromycete Botrytisf (c) basidiomycetes such as Collybia Fomes Lentinus Pleurotus Trametes Phlebia or Pycnoporus (d) perfect forms of Rhizoctonia but also by (2) plants such as Rhus vernicifera Liriodendron tulipifera Nicotiana tabacum or Acer pseudoplatanus and by (3) bacteria such as Azospirillum lipoferum. They are also widespread in bacteria (Alexandre and Zhulin 2000 Tibtech'18: 41).<br>
Taking into account the biodiversity of their producers oxidases and laccases exhibit a wide range of substrate specificities with different abilities to oxidize phenolic substrates. Thus laccases are involved in pigmentation fruiting body formation pathogenicity and lignin degradation and biosynthesis.<br>
Because of this substrate specificity oxidases and laccases show potential in industrial applications (pulp and paper processing dye transfer inhibition in detergents or phenol polymerization) in environmental applications (environmental pollutants detoxification or waste water treatment) in food application (baking brewing prevention of wine discoloration color enhancement of tea based foodstuff deoxygenation of food items or juice manufacture) and in pharmaceutical applications (transformations of steroid and antibiotics) (see for examples: Sariaslani 1989 Critic. Rev. Biotechnol. 9:171; Potus et al. 1999 Industries des cereales 115:3; Lopez et al. 2002 J. Biotechnol. 99:249; Duran et al. 2002 Enz. Microbial Technol. 31:907; Minussi et al. 2002 Trends Food Sci. Technol. 13:205; Biotechnology in the pulp and paper industry 2002 Viikari et Lantto eds Elsevier).<br>
Depending on their origins fungal laccases have different temperature and pH optima different redox potential<br>
and substrate specificities (Xu F. &amp; al. 1996 Biochim Biophys Acta. 1292 p.303). A large number of fungal laccases have been isolated and most of their corresponding genes have been cloned. Similarities and strong identities values found between their amino acid sequences show that closely related sequences belongs to organisms which are members of the same phylogenetic group. These values are sometimes higher between enzymes from different species of the same genus than between different laccases produced by the same species (Eggert et al. 1998 Appl Environ Microbiol. 64: 1766).<br>
Depending on their wide range of substrate specificity oxidases and laccases have great commercial potential and the ability to express these enzymes at very high level is critical for commercial purposes. Attempts to express laccase genes in heterologous fungal systems frequently gave very low yields. Thus the expression of Phlebia radiata laccase in Trichoderma reesei gave only 20 mg per liter of active enzyme (Saloheimo et al. Bio/Technology 1991 9: 987) while expression of Coprinus cinereus Lccl laccase in Aspergillus oryzae gave 8 to 135 mg per liter (Yaver et al. 1999 led. Appl Environ Microbiol. 65 (11) :4943-8.) and that of Myceliophtora thermophila in Aspergillus oryzae gave 11 to 19 mg per liter (Berka et al. 1997 Appl Environ Microbiol. 63: 3151) . The laccase of Pycnoporus cinnabarinus has been expressed at a level of about 80 mg per liter in Aspergillus (Sigoillot C. et al. 2004<br>
*<br>
Appl. Microbiol. Biotechnol. 64: 346).<br>
At the present time there is still a need for new oxidases that can be used in the different applications described herein.<br>
Furthermore the expression of their corresponding genes at high level in industrial hosts such as Aspergillus is of great interest.<br>
Summary of the invention<br>
The present invention relates to the isolation and characterization of a new gene encoding an Aspergillus ficuum oxidase.<br>
A new gene encoding an oxidizing enzyme according to the invention has been isolated from A. ficuum and is 1923 base pairs long with two introns and an open reading frame corresponding to 596 amino acids.<br>
An aspect of the invention relates to a nucleic acid molecule comprising or consisting of SEQ ID NO 1 2 3 4 5 6 7 8 9 10 or 11 encoding an A. ficuum oxidase.<br>
Another aspect of the invention relates to a nucleic acid molecule encoding a polypeptide of the present invention.<br>
Another aspect of the invention relates to a polynucleotide selected from the group consisting of:<br>
-	a nucleic acid molecule comprising a nucleotide sequence<br>
having at least 60% advantageously at least 70% more<br>
.advantageously at least 80% preferably at least 85% more preferably at least 90% and even more preferably at least 95% identity with any of SEQ ID NO 1 to 39 the complementary form and RNA form thereof<br>
-	any  fragments  thereof  encoding  a  protein  having  an<br>
oxidase activity the complementary form and RNA form<br>
thereof and<br>
-	any  fragments  thereof  of  at  least  15  nucleotides<br>
preferably of at least 20 nucleotides or more preferably<br>
of at least 25 nucleotides the complementary form and<br>
RNA form thereof.<br>
Another aspect of the invention relates to a protein having an oxidase activity of about 85 kDa as shown by polyacrylamide gel electrophoresis and Coomassie blue staining.<br>
Another aspect of the invention relates to a protein having an oxidase activity with a molecular weight of about 70 kDa after deglycosylation with PNGaseF.<br>
The invention also relates to an isolated oxidase polypeptide the amino acid sequence of which comprises or consists of SEQ ID NO 40 or 41 and any fragments thereof having an oxidase activity.<br>
Another aspect of the invention relates to an isolated polypeptide having an oxidase activity selected from the group consisting of an amino acid sequence having at least 70% advantageously at least 80% preferably at least 85% more preferably at least 90% and even more preferably at least 95% identity with SEQ ID NO 40 or 41.<br>
Another aspect of the invention relates to transformed cells e.g. A. nidulans 2024 comprising a gene according to the invention expressed under the control of its own promoter.<br>
Another aspect of the invention relates to transformed cells e.g. A. nidulans or A. ficuum comprising a gene according to the invention expressed under the control of the gpdA promoter of A. nidulans (glyceraldehyde-phosphate-dehydrogenase promoter) resulting in a several-fold increase of the expression of said gene.<br>
Another object of the invention relates to the use of an oxidase according to the present invention in food and non-food industrial applications where oxidation of phenolics is required for example its use in a baking process or its use as colour enhancer e.g. in tea.<br>
Another object of the invention relates to a bread improving composition comprising an oxidase polypeptide of the invention.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 represents a SDS-PAGE and zymogram analysis of the purified enzyme with oxidase activity from Aspergillus ficuum (maintained as a deposit with DSMZ under the accession number DSM932).<br>
Figure 2 represents the enzyme activity in function of the pH.<br>
Figure 3 represents the nucleotide sequence of the gene coding for the enzyme with oxidase activity from Aspergillus ficuum (DSM932) and the corresponding amino acid sequence.<br>
Figure 4 represents the absorbance spectra of tea solutions treated or not with an oxidase according to the invention.<br>
Figure 5 represents the effect of increasing amounts of an oxidase isolated from Aspergillus ficuum (DSM932) according to the invention on the volume of bread.<br>
Figure 6 represents the effect of increasing amounts of an oxidase isolated from Aspergillus ficuum (DSM932) according to the invention on stickyness of bread.<br>
Figure 7 represents the effect of increasing amounts of an oxidase isolated from Aspergillus ficuum (DSM932) according to the invention on dough consistency.<br>
Figures 8 to 12 represent the nucleotide and amino acid sequences of the invention.<br>
DETAILED DESCRIPTION OF THE INVENTION Oxidase encoding gene.<br>
The present invention provides an isolated nucleic acid molecule encoding an oxidase.<br>
The nucleic acid molecule consisting of SEQ ID NO 1 has been isolated from A. ficuum deposited with DSMZ<br>
under the accession number DSM932 (referred to as A. ficuum DSM932) .<br>
Allelic and species variants are also contemplated and are referred to as homologues or homologous sequences.<br>
In the context of the present invention a homologue or homologous sequence is a nucleotide or an amino acid sequence having at least 60% advantageously at least 70% more advantageously at least 80% preferably at least 90% and more preferably at least 95% 96% 97% 98% or 99% homology (or identity) with any of SEQ ID NO 1 to 50.<br>
It is meant by "homology" or "identity" the value in percentage given when comparing two sequences. Said comparison can be performed using any available software for example the Clustalw program and the default parameters values (e.g. available at http://www.ebi.ac.uk/clustalw).<br>
In the context of the present invention the terms "polynucleotide" or "nucleic acid molecule" refer to single-stranded or double stranded molecules and include DNA molecules (e.g. cDNA or genomic DNA) RNA molecules (e.g. mRNA) and analogs wherein nucleotides have been replaced by nucleotide analogs or derivatives.<br>
A nucleic acid consisting of or comprising any of SEQ ID NO 1 to 39 or any homologue its complementary form (or complementary strand)  or its RNA form can be isolated from different micro-organisms producing an oxidase according to the invention.<br>
Said micro-organisms can be bacteria or fungi including yeasts and can be more specifically other Aspergillus species.<br>
Established standards methods can be used to isolate a homologous sequence of the invention.<br>
Examples of such methods include the construction of a gene library from the genomic DNA of the<br>
micro-organisms in a suitable vector followed by screening of this library by direct expression of said homologous sequence.<br>
Another method comprises a hybridization step with e.g. a fragment of at least 15 nucleotides preferably at least 20 nucleotides and more preferably at least 50 nucleotides of a nucleic acid molecule of the invention e.g. a fragment of SEQ ID NO 37 38 or 39.<br>
Other methods include the amplification of said homologous sequence by molecular techniques for instance PCR techniques using oligonucleotide primers that are designed from a nucleic acid sequence of the invention.<br>
Furthermore there are established methods for obtaining in a relatively short period of time thousands of mutated sequences together with assessed enzymatic activities of their corresponding polypeptide sequences. These methods include e.g. random mutagenesis high-throughput screening etc. and are frequently used to demonstrate the eventual effects of single or multiple mutations.<br>
Such mutations (addition(s) deletion(s) and/or substitution (s) ) can be silent or not can be made inside or outside the regions critical to the function of the molecule and still result in an active protein having an oxidase activity more or less similar to the oxidase activity of a polypeptide of SEQ ID NO 40 or 41 i.e. of at least 70% advantageously of at least 80% preferably of at least 90% or more preferably of at least 95% (also referred to as functional equivalents).<br>
Alternatively a nucleic acid molecule of the invention may be prepared synthetically by methods known in the art. A nucleic acid molecule of the invention may include oligonucleotide analogs or derivatives (e.g. inosine or phosphorothioate nucleotides etc.) so it has for example altered base-pairing abilities or increased resistance to nucleases.<br>
A nucleic acid molecule of the invention may or may not include introns interrupting the coding sequence.<br>
A nucleic acid molecule of the invention may be of mixed genomic synthetic and/or cDNA origin prepared by methods known in the art comprising the step of ligating fragments from different origins.<br>
A preferred isolated and purified nucleotide sequence of the invention corresponds to SEQ ID NO 1 2 3 4 5 6 7 8 9 10 or 11 or a fragment thereof that encodes a peptide having an oxidase activity.<br>
A fragment of said sequence SEQ ID NO 1 2 3 4 5 6 7 8 9 10 or 11 has preferably more than 600 nucleotides more preferably more than 900 nucleotides or even more preferably more than 1200 nucleotides and encodes a protein characterized by a oxidase activity similar to the oxidase activity of the complete amino acid sequence of SEQ ID NO 40 or 41 (also referred to as a functional equivalent).<br>
Preferably said functional equivalent has an oxidase enzymatic activity of more than 70% or more than 80% of the initial oxidase activity of the complete enzyme defined by its amino acid sequence of SEQ ID NO 40 or 41 and preferably has an oxidase activity of at least 90% compared to the one having the amino acid sequence of SEQ ID NO 40 or 41.<br>
A polynucleotide of the invention is selected from the group consisting of:<br>
(a) a nucleic acid molecule comprising a nucleotide sequence having at least 60% advantageously at least 70% more advantageously at least 80% preferably at least 85% more preferably at least 90% and even more preferably at least 95% 96% 97% 98% or 99% identity with any of SEQ ID NO 1 to 39 the complementary form and RNA form thereof<br>
(b) any fragments of a nucleic acid molecule of group (a) encoding a protein having an oxidase activity the complementary form and RNA form thereof and<br>
-	any fragments of a nucleic acid molecule of group (a) or<br>
(b) of at least 15 nucleotides preferably of at least<br>
20 25 or 30 nucleotides or more preferably of at least<br>
50 nucleotides  the complementary  form and RNA form<br>
thereof.<br>
In  other  words  a  nucleic  acid  molecule according to the invention may comprise or consist of:<br>
-	a nucleotide sequence of any of SEQ ID NO 1 to 39 its<br>
complementary form or RNA form<br>
-	a nucleotide sequence having at least 60% advantageously<br>
at  least  70%  more  advantageously  at  least  80%<br>
preferably at least 85% more preferably at least 90% and<br>
even more preferably at least 95% 96% 97% 98% or 99%<br>
identity with any of SEQ ID NO 1 to 39 or with the<br>
complementary form or RNA form thereof<br>
-	a fragment of any of SEQ ID NO 1 to 11 or of any of their<br>
homologues or of any of their complementary form or RNA<br>
form wherein said fragment encodes a protein having an<br>
oxidase activity or<br>
-	a fragment of at least 15 nucleotides preferably at<br>
least 20 nucleotides more preferably of at least 25<br>
nucleotides of any of SEQ ID NO 1 to 39 or of any of<br>
their homologues or of any of their complementary form<br>
or RNA form.<br>
Said fragments of any of SEQ ID NO I to 11 or of any of their homologues or of any of their complementary form or RNA form encoding a protein having an oxidase activity consist preferably of at least 600 nucleotides preferably of at least 900 nucleotides or more preferably of at least 1200 nucleotides.<br>
A nucleic acid molecule according to the invention comprising or consisting of a fragment of at least L5 nucleotides preferably of at least 20 25 or 30 lucleotides more preferably of at least 50 nucleotides of any of SEQ ID NO 1 to 39 or of any of their homologues or of any of their complementary form or RNA form can be used for example for detection or identification purposes as a primer or a probe.<br>
Oxidase Protein.<br>
Also provided is a protein having an oxidase activity of about 85 kDa as shown by polyacrylamide gel electrophoresis and Coomassie blue staining.<br>
A protein of the invention having an oxidase activity has a molecular weight of about 70 kDa after deglycosylation with PNGaseF.<br>
Advantageously the unglycosylated form of the isolated and purified amino acid sequence according to the invention has a molecular weight comprised between about 60 and about 70 kDa preferably about 63 kDa or about 65.5 kDa.<br>
An isolated oxidase of the invention consists of a polypeptide encoded by a nucleic acid of the invention.<br>
An isolated oxidase polypeptide of the invention may comprise or consist of:<br>
-	an amino acid sequence of any of SEQ ID NO 40 to 50<br>
-	a fragment of at least 100 amino acids of SEQ ID NO 40<br>
41 or 45 or<br>
-	an amino acid sequence presenting at least 60% identity<br>
with the amino acid sequence of any of SEQ ID 40 to 50<br>
or with any fragments of at least 100 amino acids of said<br>
SEQ ID NO 40 41 or 45.<br>
More specifically an isolated oxidase of the invention consists of a polypeptide encoded by a nucleic acid molecule comprising or consisting of:<br>
-	Any of SEQ ID NO 1 to 39<br>
-	a nucleotide sequence having at least 60% advantageously<br>
at  least  70%  more  advantageously  at  least  80%<br>
preferably at least 85% more preferably at least 90% and<br>
even more preferably at least 95% 96% 97% 98% or 99%<br>
identity with any of SEQ ID NO 1 to 39 (also referred to<br>
as homologues) or<br>
-	any fragments of any of SEQ ID NO 1 to 11 or of their<br>
homologues  encoding  a  protein  having  an  oxidase<br>
activity.<br>
An isolated oxidase polypeptide of the invention comprises or consists of the amino acid sequence of any of SEQ ID NO 40 to 50 or any fragments thereof that have retained an oxidase activity.<br>
An isolated oxidase polypeptide of the invention comprises or consists of an amino acid sequence presenting at least 60% preferably at least 70% 80% or 85% more preferably at least 90% or even more preferably at least 95% 96% 97% 98% or 99% homology (or sequence identity) with the amino acid sequence of SEQ ID 40 or 41 or with any fragments of said SEQ ID NO 40 or 41 having an oxidase activity.<br>
A preferred fragment of an oxidase polypeptide of the invention in particular a preferred fragment of SEQ ID NO 40 or 41 consists or comprises an amino acid sequence of at least 100 amino acids preferably of at least 200 more preferably of at least 300 amino acids and even more preferably of at least 400 amino acids.<br>
A preferred fragment of an oxidase polypeptide of the invention has at least 70% advantageously at least 80% more advantageously at least 85% preferably at least 90% or more preferably at least 95% 96% 97% 98% or 99% of the oxidase activity of an oxidase polypeptide defined by an amino acid sequence of SEQ ID NO 40 or 41.<br>
A preferred fragment of an oxidase polypeptide of the invention in particular a preferred fragment of SEQ ID NO 40 or 41 consists or comprises an amino acid sequence of at least 100 amino acids preferably of at least 200 more preferably of at least 300 amino acids and even more preferably of at least 400 amino acids and has at least 70% advantageously at least 80% more advantageously at least 85% preferably at least 90% or more preferably at least 95% 96% 97% 98% or 99% of the oxidase activity of an oxidase polypeptide defined by an amino acid sequence of SEQ ID NO 40 or 41.<br>
Indeed an isolated oxidase polypeptide of the invention consisting of or comprising an amino acid sequence of any of SEQ ID NO 40 to 50 can be deleted partially while maintaining its enzymatic activity. Said enzymatic activity can be measured by methods well known in the art.<br>
A protein fragment according to the invention can also be prepared by recombinant techniques.<br>
An isolated oxidase polypeptide according to the invention may also result from the substitution deletion and/or insertion of one or more amino acids in an amino acid sequence of any of SEQ ID NO 40 to 50.<br>
Said substitution can be conservative which means that an amino acid is replaced with an amino acid having a similar side chain without affecting significantly the enzymatic activity of said polypeptide compared to an oxidase polypeptide consisting of the amino acid sequence of SEQ ID NO 40 or 41. These families are known in the art and include amino acids with basic side chains (e.g. lysine arginine and histidine) acidic side chains (e.g. aspartic acid glutamic acid) uncharged polar side chains (e.g. glycine asparagine glutamine serine threonine tyrosine cysteine) non-polar side chains (e.g. alanine valine leucine isoleucine proline phenylalanine methionine tryptophan) beta-branched<br>
side chains (threonine valine isoleucine) and aromatic side chains (e.g. tyrosine phenylalanine tryptophan histidine).<br>
Moreover said substitution deletion or insertion may concern non-essential amino acid also resulting in no significant alteration of the oxidase activity of said polypeptide in comparison with an oxidase polypeptide consisting of the amino acid sequence of SEQ ID NO 40 or 41.<br>
An isolated and purified oxidase enzyme according to the invention is also characterized by an optimum pH around 5.5. More generally the maximum activity is comprised between a pH of about 5 and a pH of about 6.5 (see figure 2) .<br>
An isolated oxidase polypeptide according to the invention can also be characterized by the fact that it can use among others NN-dimethyl-p-phenylenediamine and 22'-azinobis(3-ethylbenzthiazoline-6-sulfonic acid) as substrates.<br>
Therefore an isolated oxidase polypeptide according to the invention can also be referred to as a laccase.<br>
Expression and/or purification of the enzyme<br>
Another aspect of the present invention is related to a recombinant nucleotide sequence comprising operably linked to a nucleotide sequence according to the invention one or more adjacent regulatory sequence (s) . Said adjacent regulatory sequence(s) is/are preferably originating from homologous micro-organisms.<br>
However said adjacent regulatory sequences may also be originating from heterologous micro-organisms.<br>
Said adjacent regulatory sequences are specific sequences such as promoters enhancers secretion signal sequences and/or terminators.<br>
Preferred adjacent regulatory sequences of the invention are capable of directing the overexpression of a<br>
nucleotide sequence of the invention in a recombinant host cell. A preferred adjacent regulatory sequence according to the invention is the constitutive gpdA (glyceraldehyde -3-phosphate dehydrogenase) promoter of Aspergillus nidulans.<br>
Another aspect of the invention is related to a vector comprising a nucleic acid molecule of the invention possibly operably linked to one or more adjacent regulatory sequence(s) originating from homologous or from heterologous micro-organisms.<br>
In the present context "vector" is defined as any biochemical construct which may be used for the introduction of a nucleotide sequence (by transduction tranfection transformation infection conjugation etc.) into a cell.<br>
Advantageously a vector according to the invention is selected from the group consisting of plasmids (including replicative and integrative plasmids) viruses phagemids chromosomes transposons liposomes cationic vesicles or a mixture thereof. Said vector may already comprise one or more adjacent regulatory sequence(s) allowing the expression of said nucleic acid molecule and its transcription into a polypeptide of the invention. Preferably said vector is a plasmid.<br>
The present invention is also related to a transformed host cell or recombinant host cell containing (or having incorporated) one or more of the nucleotide sequences and/or vectors according to the invention.<br>
In the present context a "transformed host cell" or "recombinant cell" also referred to as "transformant" is a cell having incorporated one or more of the nucleotide sequences and/or vectors according to the invention. The transformed host cell may be a cell in which said vector(s) and/or said nucleotide sequence(s) is/are introduced by means of genetic transformation preferably by<br>
means of homologous recombination or by any other well known methods used for obtaining a recombinant organism.<br>
Said host cell used for the transformation may or may not already have (a) nucleotide sequence(s) and/or vector(s) of the invention. Both prokaryotic and eukaryotic cells are included e.g. bacteria fungi yeast etc.<br>
Preferred host cells are A. nidulans in particular A. nidulans 2024 A. niger more specifically A. niger N402 or A. ficuum more particularly A. ficuum DSM 932.<br>
Said host cell may also be the original cell e.g. A. ficuum containing already nucleotide sequences of the present invention and genetically modified to over-express or express more efficiently an oxidase polypeptide of the invention (better pH or temperature profile higher extracellular expression etc.).<br>
A transformed host cell of the invention may<br>
have integrated into its genome an isolated nucleic acid<br>
molecule according to the present invention and/or may contain<br>
(an) episomal vector (s) comprising an isolated nucleic acid<br>
molecule of the invention.<br>
A preferred transformed host cell of the invention is capable of over-expressing (i.e. higher expression than the expression observed in the original or wild-type microorganism) (a) nucleotide sequence(s) and/or vector(s) of the invention advantageously allowing a high production of polypeptide encoded by said nucleotide sequence(s) and/or said vector(s).<br>
Preferably said recombinant host cell contains regulatory sequences adjacent to a nucleic acid molecule according to the invention that are capable of directing the overexpression of said nucleic acid molecule.<br>
An overexpression of an oxidase of the invention may also result from an increasing number of copies of nucleic<br>
acid sequences according to the invention in said recombinant host cell.<br>
For an optimal expression of an oxidase according to the invention the original production species e.g. A. ficuum and/or a suitable transformed host cell e.g. transformed A. niger or transformed A. nidulans are in a suitable growth medium and/or expression medium. Some examples of the optimal conditions such as culture media temperature and pH conditions etc. are described in the examples section.<br>
According to the present invention a polypeptide of the invention with an oxidase activity may be obtained by first culturing the strain in/on a medium suitable for expressing said oxidase and then may be recovered from the medium by conventional methods including but not limited to centrifugation microfiltration ultrafiltration spray-drying evaporation or precipitation. Said oxidase according to the invention may be further purified using for example electrophoretic procedures extraction or a variety of chromatographic procedures such as ion exchange chromatography gel filtration chromatography affinity chromatography etc. All these techniques are described in the scientific literature and are well known techniques.<br>
Said oxidase can be extra-cellular or intra-cellular expressed and/or secreted by a micro-organism producing said oxidase or by a recombinant host according to the invention.<br>
Said polypeptide of the invention may be expressed in a modified form such as a fusion protein and may include one or more secretion signals and/or one or more additional heterologous functional regions. Said regions may consist of particularly charged amino acids added to the Nt of said polypeptide to improve stability and persistence in the host cell during purification of during subsequent handling<br>
and storage. They may also consist of short peptides added to said polypeptide of the invention to facilitate purification.<br>
Different applications of an oxidase of the invention<br>
The oxidase enzymes according to the invention may be used in different kinds of industries.<br>
A polypeptide with oxidase activity according to the present invention further purified or not purified is particularly suited as a bread improving agent. Bread improving agents or bread improving compositions are products that are able to improve and/or increase texture flavour anti-staling effects softness crumb softness upon storage freshness dough machinability and/or volume of a dough and/or of a final baked product.<br>
A polypeptide with oxidase activity according to the invention is preferably used to improve the dough handling and/or increase the specific volume of the final baked product. A polypeptide with oxidase activity according to the invention is advantageously used in a bread improver formula or bread improving composition.<br>
The present invention also relates to a bread improving composition comprising an oxidase polypeptide of the invention.<br>
The term "baked product" includes any product prepared from a dough and obtained after baking of the dough and includes in particular yeast raised baked products.<br>
Dough is obtained from any type of flour or meal (e.g. based on wheat rye barley oat or maize). Preferably dough is prepared with wheat and/or with mixes including wheat.<br>
A bread improving composition according to the invention may also comprised other bread-improving agents such as but not limited to enzymes emulsifiers oxidants milk<br>
powder  fats  sugars amino acids and/or proteins  (gluten cellulose binding site).<br>
Examples of such enzymes include but are not restricted to alpha-amylases beta-amylases maltogenic amylases xylanases proteases glucose oxidases oxido-reductases glucanases cellulases transglutaminases isomerases lipases phospholipases pectinases etc.<br>
A preferred bread improving composition according to the invention comprises an oxidase polypeptide of the invention and an alpha-amylase preferably an alpha-amylase from Aspergillus oryzae.<br>
In another aspect of the present invention an oxidase polypeptide of the invention is particularly suited for the improvement or enhancement of the color of tea based foodstuffs.<br>
An oxidase polypeptide of the invention may be used in food applications such as baking pastry cakes brewing prevention of wine discoloration deoxygenation of food items juice manufacturing etc. or in feed applications.<br>
In another aspect of the present invention an oxidase polypeptide of the invention may be used in industrial applications such as pulp and paper processing dye transfer inhibition in detergents or phenol polymerization etc. in environmental applications such as environmental pollutants detoxification waste water treatment etc. or in Pharmaceuticals applications (transformations of steroids and antibiotics etc.).<br>
The effect of an oxidase polypeptide of the invention may be further improved by adding other enzymes. Such enzymes may belong but are not restricted to hydrolytic enzymes families such as glucanase proteases cellulases hemicellulases and pectinases. Other enzymes are transglutaminases oxido-reductases isomerases etc.<br>
Depending on the application an oxidase polypeptide of the invention may be used under several forms. Micro-organisms (recombinant or not) expressing an oxidase polypeptide of the invention such as yeasts fungi archea bacteria or bacteria may be used directly in the process.<br>
An oxidase polypeptide of the invention may be used as a cell extract a cell-free extract (i.e. portions of the host cell that has been submitted to one or more disruption centrifugation and/or extraction steps) and/or as a purified protein.<br>
One or more of said forms may be used in combination with one or more enzymes under any of the above-described forms.<br>
Said whole cells cell extracts cell-free extracts or said purified oxidase polypeptide of the invention may be immobilized by any conventional means on a solid support for instance to allow protection of the oxidase<br>
s<br>
polypeptide of the invention or to allow continuous hydrolysis of a substrate and/or to allow recycling of the enzymatic preparation.<br>
Said cells cell extracts (including crude and partially purified extracts) cell-free extracts and/or said purified oxidase polypeptide of the invention may be mixed with different ingredients e.g. in the form of a dry powder or a granulate in particular a non-dusting granulate or in a form of a liquid for example with stabilizers such as polyols sugars organic acids sugar alcohols according to well-established methods.<br>
The invention is described in further details in the following examples which are intended for illustration purposes only and should not be construed as limiting the scope of the invention in any way.<br>
EXAMPLES<br>
MATERIALS AND METHODS EXAMPLE 1; Strains and media.<br>
The bacterial strains used were the E. coli strains RR1AM15 (F'lacIQ lacZAMlS hsdS20 supE44 ara-14 proA2 rspL20(strR) lacYl galK2 xyl-5 mtl-1) and MC1061 (hsdR mcrB araD!39 A (araABC-leu) 7697 A lacX74 galU galK rpsL thi) . Fungal strains were Aspergillus ficuum DSM932 Aspergillus nidulans 2024 (biAl argB3) and Aspergillus niger N402 (cspAl derivate of the ATCC strain 9029).<br>
Bacteria were grown at 37°C in LB (0.5% yeast extract 1% Bacto peptone 1% NaCl) or TB medium (Terrific Broth Gibco BRL Life Technologies Inc. Gaithersburg MD) and fungi in Aspergillus minimal medium (Ponteverco &amp; al 1953 Adv. Genet.5:.141) at 28°C and 250 rpm.<br>
EXAMPLE 2; Fermentations<br>
Aspergillus strains were cultivated in 151 Biostat E fermentors (B. Braun Biotech - working volume 101). The culture medium composition was the following: Maldex 15: 40 g/1; Salt solution: 50ml/l; Trace elements solution: lml/1; CuS04 solution (1.6g/l): lml/1. The salt solution contained 120g/l NaN03 10.40g/l KC1 10.40 g/1 MgS04.7H20 and 30.40g/l KH2P04. The Trace elements solution (pH 6.5) contained 22g/l ZnS04.7H20 llg/1 H3B03 4.1g/l MnC12.2H20 5g/l FeS04.7H20 1.7g/l CoC12.6H20 1.6g/l CuS04.5H20 1.5g/l Na2Mo04.2H20 and 50g/l ethylenedinitrilotetraacetic acid disodium salt dihydrate. 1 mg/1 biotine was added after sterilization.<br>
The initial pH of the fermentation was 6.0 and the temperature was fixed at 30°C. The duration of the fermentation was around 60 to 70 hours.<br>
EXAMPLE 3: Isolation of genomic DNA.<br>
The preparation of genomic DNA from mycelium of A. ficuum was based on the protocol of Blin and Stafford (Nucl. Ac. Res.1976 3:.2303).<br>
The mycelium was washed with 100% ethanol dried in a vacuum desiccator and ground to powder under liquid nitrogen. The powder was resuspended in extraction buffer (50 mM Tris-HCl pH 7.4 10 mM MgC12 50 mM NaCl 1% SDS) and incubated at 55 °C for 15 min.<br>
Phenol/chloroform (1/1 v/v) was added and the solution was placed on a rocking platform at room temperature for 30 min. The mixture was centrifuged 15 min at 3000 rpm. The DNA phase was extracted again with phenol/chloroform and then with chloroform.<br>
The DNA was precipitated with the addition of 0.1 volume of 3 M NaAc and 0.5 volume of isopropanol at room temperature.<br>
After centrifugation the pellet was washed with 70% ethanol briefly dried and dissolved in TE-buffer (10 mM Tris-HCl pH 8.0 1 mM EDTA).<br>
To remove residual RNA 100 ug DNAse-free RNAse A was added and the solution was incubated for 30 min at 37°C. SDS to a final concentration of 0.5% and predigested proteinase K to a final concentration of 50 ug/ml were added to the DNA solution for 1 hour at 50°C.<br>
Extraction was performed once with an equal volume of phenol/chloroform and once with chloroform. The DNA was precipitated again centrifuged washed with 70% ethanol and redissolved in TE-buffer.<br>
EXAMPLE 4; PCR amplification on genomic DNA.<br>
Based on the amino-terminal sequence (SEQ ID NO 46) and on the sequence of internal peptide (1) (SEQ ID NO<br>
47)  a  set  of  degenerated  oligonucleotides  primers  was synthesized.<br>
NH2-terminal sequence (SEQ ID NO 46):<br>
AVVQFQLDLT	i<br>
forward PCR-primer (SEQ ID NO 37): 5'  GTI GTI GAG TTT CAG YTI GAT YTI AC 3'<br>
internal sequence 1 (SEQ ID NO 47):<br>
SEDQAGDYTI R reverse PCR-primer (SEQ ID NO 38) 3' CTY CTR GTY CGI CCI CTG ATG TGI TA 5'<br>
These two primers were used for PCR amplification. PCR was performed on 100 ng genomic DNA of A. ficuum DSM932.<br>
The DNA was suspended in 100 ul of thermophilic buffer (50 mM KC1 10 mM Tris-HCl pH 9.0 0.1% Triton X-100; Promega Corporation Madison WI)  supplemented with 0.2 mM dNTP 3 mM MgC12 50 pmoles of each primer and 1.5 units of Taq-DNA-polymerase (Promega Corporation).<br>
The temperature scheme for the amplification was as follows: 10 min denaturation at 95°C hot start at 80°C for the Taq-DNA-polymerase touch-down for two cycles starting at 95°C for 30 sec and 67°C for 2 min. This was repeated with a gradual decline to 65 63 and 61°C instead of 67°C. The final cycling parameters were: 95°C 30 sec 60°C 30 sec and 72°C 1 min (35 cycles) . The reactions were carried out in a Biometra "Trio-Thermoblock" thermocycler (Biometra Gottingen Germany).<br>
EXAMPLE 5; Sequence analysis of the PCR products.<br>
The PCR products were cloned in the pUC18 vector (Yanisch-Perron &amp; al 1985 Gene 33: 103). This plasmid<br><br>
(purified on a Qiagen column (Qiagen Inc. Chatsworth CA) ) was digested with Smal (New England Biolabs Inc. Beverly MA) generating blunt ends.<br>
After extraction with phenol/chloroform the blunt ends were dephosphorylated with calf intestine alkaline phosphatase (Boehringer) in 50 mM Tris-HCl - 0.1 mM EDTA-buffer (pH 8.5) at 37°C for 30 minutes. Electrophoresis through a 1 % agarose gel was performed and the dephosphorylated vector band was eluted and purified by the Geneclean method (Bio 101 La Jolla CA).<br>
The PCR reaction mixtures were initially purified on a Qiagen column. The ends of the PCR DNA fragments were filled in with 0.2 mM dNTP using Pf u-DNA-polymerase (Stratagene La Jolla CA) and T4-DNA-polymerase (Boehringer) in Pfu-buffer (20 mM Tris-HCl pH 8.75 10 mM KC1 10 mM (NH4)2S04 2 mM MgS04 0.1% Triton X-100 0.1 mg/ml bovine serum albumin; Stratagene) at 37°C for 45 min.<br>
After a new Qiagen purification the blunt ends of the PCR fragments were phosphorylated at 37°C for 30 min with 0.2 mM rATP using T4-polynucleotide kinase (Amersham Buckinghamshire England) in kinase buffer (20 mM Tris-HCl pH 7.5 10 mM MgC12 10 mM ft-mercaptoethanol) supplemented with 6% polyethylene glycol 8000 and 8 mM MgC12.<br>
After electrophoresis through a 0.8% agarose gel the 1150 bp DNA fragments was eluted and purified by the Geneclean method.<br>
The eluted DNA fragments were ligated in the pUC18 vector using T4-DNA-ligase (Boehringer) in ligase buffer (66 mM Tris-HCl pH 7.5 5 mM MgC12 1 mM DTE 1 mM ATP; Boehringer) at 18°C for 16 hours.<br>
The ligation mixtures were transformed in the E.<br>
coli strain RR1AM15. The colonies obtained were submitted to a DNA extraction following the Birnboim procedure (Birnboim &amp; al 1979 Nucl. Ac. Res 7: 1513). The DNA was analyzed by<br>
EcoRI - HindiII digestion and electrophoresis through a 1.2% agarose gel.<br>
The plasmid DNA (Qiagen purified) of a positive clone was sequenced following the ABI Taq DyeDeoxy Terminator Cycle Sequencing protocol (ABI Foster City CA) . The samples were run on an automated ABI373A sequencing system (ABI). The DNA sequences obtained were converted to ASCII format and transferred to a HIBIO DNASIS software package (Pharmacia LKB Biotechnology Uppsala Sweden) for assembly.<br>
EXAMPLE 6: Construction of a genomic DNA library of A. ficuum DSM932.<br>
Genomic DNA (5 jag) of A. ficuum DSM932 was partially digested with the restriction enzyme Alul (7.5 units) for 15 min. The DNA ends obtained were further polished with T4-DNA-polymerase (1 unit) and dNTP nucleotides (final concentration of 100 uM of each) for 5 min at 37°C. The partial digest was then extracted with phenol/chloroform and size-fractionated on a 0.7% agarose gel.<br>
The genomic DNA fragments with a length of 9000 to 23000 bp were eluted from the gel by centrifugal filtration (Zhu &amp; al 1985 Bio/Technology 9: 1014; membrane type GV 0.22 urn Millipore Intertech Bedford MA) extracted with phenol/chloroform and concentrated by ethanol precipitation.<br>
Sfi I adaptors (5'-GTTGGCCTTTT) (SEQ ID NO 52) were ligated to the DNA ends. The ligation reaction was performed in PEG buffer (25 mM Tris-HCl pH 7.5 5 mM MgC12 2.5 % (w/v) PEG 8000 0.5 mM DTT 0.4 mM ATP) with 24 units of T4 DNA ligase and 250 pmoles of phosphorylated Sfi I adaptors in a volume of 50 ul overnight at 12°C.<br>
After extraction with phenol/chloroform the Sfil - Sfil genomic DNA fragments were purified and separated again on a 0.7% agarose gel. Portions of the gel containing<br>
fragments from 10 to 20 kb length were cut out membrane-eluted and concentrated by ethanol precipitation. These fragments (100 ng) were finally cloned into the Sfil-digested YCpSOSfil-Sfil vector (an E. coli/S. cerevisiae shuttle vector derived from the plasmid YCpSO (Trash &amp; al 1985 Proc. Nat. Acad. Sci. USA 82: 4374) bearing an EcoRI - HindiII fragment that contains the hlFNft gene flanked by two Sfil sites in inverse direction).<br>
The ligations were performed in a volume of 20 ul at 12 °C for 4 hours with 8 units T4 DNA ligase (Pharmacia LKB Biotechnology Uppsala Sweden) further extracted with phenol/chloroform and finally electroporated in freshly prepared MC1061 cells.<br>
Bacteria were plated on LB agarose and the resulting colonies were scraped off from the plates in groups of 1000 clones. The library size was approximately 100000 clones and the average insert size was ± 16 kb.<br>
EXAMPLE 7;  Screening of the genomic DNA library by colony hybridization.<br>
The 1150 bp fragment was isolated from the pUClS vector by EcoRI-Hindlll digestion agarose gel electrophoresis elution and purification by the Geneclean method.<br>
The fragment was randomly labeled with a32P-dCTP using the procedure of Feinberg and Vogelstein (Anal. Biochem 1984 137:.266). 75 ng of the 1150 bp fragment were boiled for 10 minutes and immediately chilled on ice. The labeling reaction was performed at 37°C for 30 minutes in a total volume of 60 ul using 6 units of Klenow polymerase dGTP dATP and dTTP (25 uM of each)  a hexanucleotide mixture (Random Primed DNA Labeling Kit from Boehringer) and 45 pmoles a32P-dCTP (Amersham Buckinghamshire England). The reaction<br>
mixture was then dialyzed for 90 minutes against bidistilled water using a membrane filter (type VS 0.025 mm; Millipore Intertech Bedford MA).<br>
Groups of clones of the genomic DNA library of Aspergillus ficuum DSM932 were spread out onto LB plates and overnight incubated at 37°C resulting in 2000 to 6000 colonies per plate. These colonies were submitted to colony lifting on nylon membranes (Hybond N Amersham) and UV-cross-linking (UV StratalinkerTM 1800 from Stratagene) followed by hybridization with the radioactively labeled probe. The prehybridization and hybridization steps were performed at 62°C in 7% SDS-phosphate buffer (1 mM EDTA 0.5 M NaHP04 pH 7.2 and 7 % SDS)  the washing steps at 62°C in 5% SDS-phosphate buffer (1 mM EDTA 40 mM NaHP04 pH 7.2 5 % SDS). Filters were finally exposed to X-ray films.<br>
EXAMPLE 8; Subcloning of the genomic DNA.<br>
The plasmid DNA (± 10 ug; Qiagen purified) of positive clones obtained after screening of the genomic DNA library was sonicated (Vibra Cell VC500; Sonics &amp; Materials Danbury CT) on ice in sonication buffer (1 M tetramethyl ammonium chloride 2 mM EDTA 50 mM Tris-HCl pH 7.6) for 30 seconds (50% duty cycle).<br>
After precipitation with isopropanol the DNA was blunted using 5 units each of T4- and Klenow-DNA polymerase (Boehringer) in T4 DNA polymerase buffer (50 mM Tris-HCl pH 8.5 15 mM (NH4)2S04 7 mM MgC12 0.1 mM EDTA 10 mM fi-mercaptoethanol; Boehringer)  supplemented with 1 mM dNTP at 37°C for 30 min. The fragments were size-fractionated by electrophoresis on a 1% agarose gel and the DNA of the portion of the gel comprising fragments from 800 to 1200 bp was eluted and purified using the Geneclean method.<br>
The blunted fragments were introduced into the dephosphorylated Smal site of pUC18. Ligation was performed<br>
(molar ratio of vector/insert 1/3) using T4 DNA ligase (Pharmacia LKB Biotechnology) in blunt-end buffer (25 mM Tris-HC1 pH 7.5 5 mM MgC12 2.5% (w/v) PEG 8000 0.5 mM DTT 0.4 mM ATP) at 23°C for 4 hours. The ligation mixes were transformed into E. coli MC1061 to generate two libraries of subclones.<br>
EXAMPLE 9; DNA preparation for Aspergillus transformation.<br>
Plasmids were isolated by a modified protocol based on the 'cleared lysate' procedure of Kahn &amp; al. (Meth. Enzymol. 1979 68: 268) and purified by equilibrium centrifugation in a cesium chloride - ethidium bromide gradient.<br>
A one liter culture (TB medium) was centrifuged in a GSA Sorvall rotor for 10 min at 5000 rpm. The pellet was resuspended in 15 ml of lysing buffer (25% sucrose 50 mM Tris-HCl pH 8 20 mM EDTA) and 0.6 ml of a lysozyme suspension (10 mg lysozyme/ml H20; Boehringer) was added.<br>
After 30 min incubation on ice 15 ml of 2% Triton X-100 were mixed with the suspension and again incubated on ice for 30 min.<br>
Then the suspension was centrifuged in a SS34 Sorvall rotor for 30 min at 20000 rpm. Cesium chloride (1.1 g cesium chloride/ml supernatant) as well as 1500 ul of ethidium bromide (5 mg/ml) were added. This suspension was ultracentrifuged at 25°C and 55000 rpm for 16 hours.<br>
After isolation of the plasmid DNA band the remaining ethidium bromide was removed by several extractions with an 1/1 volume of isobutanol (centrifugation at 10000 rpm for 10 min) and the DNA was further precipitated (several times) with 0.6 volume of isopropanol washed with 70 % ethanol and finally resuspended in 200 ul of H20.<br>
EXAMPLE 10: Transformation of Aspergillus nidulans.<br>
The transformation of A. nidulans was based on the procedure of Yelton et al. (Proc. Nat. Acad. Sci. USA 1984 81:.1470) and included firstly the protoplasting of the mycelium by lytic enzymes (20 mg/g mycelium; Sigma Chemical St. Louis MO) and secondly the transformation itself using a polyethylene glycol treatment.<br>
107	protoplasts were transformed with 1 ]ag DNA<br>
of pSa!23 selection plasmid and 19 ug DNA of the plasmid of<br>
interest. The protoplasts were spread onto Aspergillus minimal<br>
medium plates  [containing 1.2 M sorbitol  1.5% Noble agar<br>
(Difco Laboratories Detroit MI) and biotin (1 ug/ml) but no arginin] included in a top agar (same composition as the plates but with 0.8% agar). The plates were incubated at 37°C until sporulating colonies appeared (± three days). Spores of these transformants were restreaked several times onto the same selection medium to obtain single colonies.<br>
EXAMPLE 11; Small scale genomic DNA preparation.<br>
108	spores  were  inoculated  into  20  ml  of<br>
Aspergillus minimal medium supplemented with biotin (1 ug/ml)<br>
and incubated overnight at 37°C and 240 rpm.<br>
Genomic DNA was isolated using a modified protocol based on the procedure of Raeder and Broda (Lett. Appl. Microbiol. 1985 1:.17). The mycelium was harvested by filtration using a folded filter (Schleicher &amp; Schuell Dassel Germany) and washed with H20 with 0.5 M EDTA pH 8 and finally with 100% ethanol. The mycelium pellet was dried under vacuum for several hours and then ground to powder under liquid nitrogen. 100 mg of mycelium were resuspended in 500 ul of lysing buffer (200 mM Tris-HCl pH 8.5 250 mM NaCl 25 mM EDTA 0.5 % SDS) and extracted twice with one volume of phenol/ chloroform/isoamylalcohol (25/24/1 - v/v/v) and once<br>
with chloroform/ isoamylalcohol (24/1 - v/v) . The DNA was precipitated with 0.6 volume of isopropanol (after addition of 0.1 volume of 3 M potassium acetate pH 4.8) washed with 70% ethanol and resuspended in H20 with RNAse A (50 ug/ml; Sigma Chemical). After incubation at 37°C for 30 min the DNA is stored at -20°C.<br>
EXAMPLE 12: Southern blot analysis.<br>
10 ug of Aspergillus genomic DNA of were digested with 120 units of Eco RI (New England Biolabs Inc. Beverly MA) at 37°C overnight in the appropriate EcoRI buffer (total volume of 200 ul) . A 0.8 % agarose gel was run and stained with ethidium bromide.<br>
After electrophoresis the gel was incubated in 0.25 M HC1 for 15 min. Transfer of the DNA fragments onto a Hybond N+ filter was performed using the 'alkali blotting' protocol of Amersham.<br>
The filter was hybridized with the 1150 bp DNA fragment described above which was labeled with oc32P-dCTP. The prehybridization and hybridization steps were performed at 68°C in 7% SDS-phosphate buffer (1 mM ^EDTA 0.5 M NaHP04 pH 7.2 and 7% SDS) . The filter was washed twice with 5% SDS-phosphate buffer (1 mM EDTA 40 mM NaHP04 pH 7.2 5% SDS) and once with 1% SDS-phosphate buffer (1 mM EDTA 40 mM NaHP04 pH 7.2 1% SDS) at 68°C and finally exposed to X-ray film.<br>
EXAMPLE 13: Enzymatic assay<br>
Enzymatic assay was performed in microtiter plates using NN-dimethyl-p-phenylenediamine as substrate. 100 ul of substrate solution (0.4 mg of NN-dimethyl-p-phenylenediamine per ml of 40 mM sodium acetate pH 5.3) were added to 25 ul sample at the appropriate dilution and incubated at room temperature protected from light. The optical density was measured at 550 nm.<br>
The  activity  was  expressed  arbitrarily  as<br>
AAbs/sec.<br>
For tea color modification studies the laccase activity was determined with 22'-azinobis(3-ethylbenzthiazoline-6-sulfonic acid (ABTS - Sigma-Aldrich Chemie Germany) as substrate. After oxidation its greenish-blue color was measured photometrically at 415 nm. Enzymatic assay were performed in microtiter plates using 75 ul of 1.66 mM ABTS in 100 mM Citrate buffer pH 4 which were mixed with 25 ul of supernatant and incubated during 10 min at 50°C.<br>
1 laccase unit (LACU) is the amount of enzyme that catalyzes the conversion of 1 umole ABTS per minute in these conditions.<br>
EXAMPLE 14: Deglycosylation with PNGase F.<br>
To a concentrated protein suspension (final volume of 90 ul) 10 jal of denaturation buffer (5% SDS 10% fi-mercaptoethanol) was added and the mix was boiled for 10 min at 100°C.<br>
After addition of 12 ul of NP40 (10 %) and 12 ul of 0.5 M sodium phosphate pH 7.5 the sample was divided into two aliquots. 3000 units of PNGase F (peptide N-glycosidase F; New England Biolabs Inc.) were added to the first sample. The other one was used as negative control.<br>
Incubation was performed overnight at 37 °C. After addition of 2 x Laemmli buffer (Nature 227:680(1970)) and boiling for 5 min the samples were run on a 12.5% SDS-polyacrylamide gel.<br>
The protein bands were visualized by Coomassie blue staining.<br>
EXAMPLE 15: Exchange of the Aspergillus ficuum DSM932 oxidase gene promoter by the gpd&amp; promoter of A. nidulans.<br>
The gene encoding the oxidase protein was isolated from the YCpSO Sfil/Sfil vector by a BamHI-SspI digestion and ligated to the plasmid pMa58 (Stanssens &amp; al 1989 Nucl. Ac. Res. 17: 4411) prepared in the following way: the pMa58 plasmid was digested with Hind III and the sticky ends were filled in with dNTP by T4 DNA polymerase and further digested with BamHI and Pvul. The resulting plasmid was termed pMa58Afic. A BspHI site was then created at the position of the ATG codon of the oxidase gene via site-specific mutagenesis based on the method of Deng and Nickoloff (Anal. Biochem 200: 81 (1992); TransformerTM Site-Directed Mutagenesis Kit of Clontech Inc. CA USA) resulting in plasmid pMac58Aficm.<br>
Via an Xbal digest on pMac58Aficm followed by blunting the sticky ends with dNTP by T4 DNA polymerase and a BspHI digest the oxidase gene was isolated and ligated into the pFGPDGLAT2 vector. The pFGPDLAT2 plasmid contains the glyceraldehyde -3-P dehydrogenase promoter of Aspergillus nidulans that allows a strong constitutive transcription of the genes located downstream of it (Punt &amp; al 1990 Gene 93:.101; Punt &amp; al 1991 J. Biotechnol. 17:.19).<br>
pFGPDGLAT2 was digested first with Hindlll treated with T4 DNA polymerase and finally digested with Ncol. The BspHI - Xbal fragment of pMac58Aficm was then ligated to this vector to generate the plasmid pFGPDAfic.<br>
EXAMPLE 16: PCR analysis of Aspergillus transformants.<br>
Genomic DNA was isolated from the transformants as described. The oligonucleotides used for the PCR reaction were the following : 5' GAA GTG GAA AGG CTG GTG TGC (SEQ ID NO 51) corresponding to a partial sequence of the gpdA promoter and 5' CAA CCC AGG TAG CGT ACT CC (SEQ ID NO 39) corresponding to a partial sequence of the oxidase gene.<br>
50 ng of genomic DNA was suspended in 50 jal of thermophilic buffer (50 mM KC1 10 mM Tris-HCl pH 9.0 0.1% Triton X-100; Promega Corporation) supplemented with 0.2 mM dNTP 3 mM MgCla 25 pmole of each primer and 1.5 units of Taq-DNA-polymerase (Promega Corporation). The temperature scheme for the amplification was as follows: After a 10-min denaturation at 95°C a hot start at 80°C was used for the Taq-DNA-polymerase. 30 cycles of [95°C 30 sec 65°C 30 sec 72°C 2 min] were performed. The reactions were carried out in a Biometra "Trio-Thermoblock" thermocycler (Biometra). The PCR products were analyzed by electrophoresis on a 0.8% agarose gel.<br>
EXAMPLE 17; Purification of an enzyme with oxidase activity from Aspergillus ficuum DSM932.<br>
1. Obtention of the desalted cell-free supernatant<br>
A 10 1 fermentation of the strain Aspergillus ficuura DSM932 was performed for 68 hours as described in example 2.<br>
The supernatant of the culture was separated from the mycelium by centrifugation. It was desalted by passing through a 6 liters Sephadex G-25 column (Amersham Biosciences) equilibrated in 10 mM Tris-HCl pH 7.0 buffer (buffer A) . The same buffer was used to elute the proteins from the column. The final volume of the oxidase containing sample was 25 liters.<br>
2.	Chromatography on DEAE Macro Prep.<br>
The sample of step 1 was applied on a 11 DEAE Macro Prep (Bio-Rad) column equilibrated with buffer A. Flow rate was 145 ml/min. After washing with the same buffer the bound proteins were eluted by increasing stepwise the NaCl concentration in buffer A to give the following fractions: fraction 1 eluted with 1.5 1 buffer A + 50 mM NaCl fraction 2 eluted with 1.5 1 buffer + 60 mM NaCl fraction 3 eluted with 1.5 1 buffer A + 110 mM NaCl fraction 4 eluted with 1.5 1 buffer 1 + 200 mM NaCl and fraction 5 eluted 1.8 1 buffer A + 500 mM NaCl. Oxidase activity was detected in fraction 5.<br>
3.	Chromatography on S-Sepharose.<br>
The buffer of 200 ml of fraction 5 from step 2 was exchanged to buffer B (15 mM ammonium acetate pH 4.0) by passing through a Sepharose G-25 (Amersham Biosciences) column equilibrated in the same buffer.<br>
The 260 ml eluted from G-25 column were applied on a 20 ml Hiload S-Sepharose HP 16/10 column (Amersham Biosciences) equilibrated in buffer B.<br>
After washing with buffer B proteins were eluted with a linear gradient of NaCl in buffer B (250 ml from 0 to 250 mM NaCl at 5 ml/min) . 5 ml fractions were collected and tested for oxidase activity.<br>
Active fractions (10 ml) were pooled and concentrated with Centriprep YM-30 centrifugal filter unit (Millipore) . Buffer was exchanged to buffer C (50 mM MES - pH 5.5 with NaOH). The final volume was 400 pi.<br>
4.	Chromatography on Resource Q.<br>
200 ul from step 3 were applied on a 1 ml Resource Q column (Amersham Biosciences) equilibrated in buffer C.<br>
After washing with buffer C proteins were eluted with a linear gradient of NaCl in buffer C (15 ml from 0 to 175 mM NaCl at 1 ml/min).<br>
5.	Chromatography on TSKG3000SW and TSKG2000SW.<br>
Fractions with oxidase activity from step 4 (3.5ml) were concentrated by centrifugation to 400 ul on a Centriprep YM-30 centrifugal filter unit (Millipore). Thereafter 100 ul were applied on TSKG3000SW and TSKG2000SW columns (Tosohaas) mounted in series and equilibrated in buffer D (20 sodium phosphate 200 mM NaCl pH 6.5). The proteins were eluted with buffer D at 0.5 ml/min. Active fractions were pooled (1.5 ml). 1 ml was desalted using a Microcon YM-10 centrifugal filter unit and concentrated to a volume of 200 ul.<br>
6.	SDS-PAGE electrophoresis<br>
The purified sample from step 5 was loaded on a 10-15% SDS polyacrylamide gel (PhastGel Gradient - 10-15; Amersham Biosciences) and run according to the recommendations of the manufacturer. One half of the gel was stained using the PhastGel Silver staining kit (Amersham Biosciences) and the other half was subjected to a zymogam analysis.<br>
For this purpose the gel is incubated in a NN-dimethyl-p-phenylenediamine solution (Sigma Chemicals; 0.4 mg/ml of 40mM sodium acetate pH 5.3) at room temperature. Proteins with oxidase activity appear black.<br>
The results of this analysis are presented on figure 1. It shows that the enzyme with oxidase activity is pure and that it exhibits oxidase activity. The apparent molecular weight of the purified oxidase is about 90 kDa.<br>
EXAMPLE 18: Determination of the amino acid sequence of the enzyme with laccase activity<br>
General procedures were followed to perform the N-terminal sequencing of the protein after electrophoresis on a 12% SDS-polyacrylamide gel and electroblotting on a PVDF Immobilon-P membrane (Millipore). An automatic 477A Protein Sequencer coupled to a HPLC 120A Analyser (Applied Biosystem) was used.<br>
For the determination of the sequence of internal fragments the protein was first digested on the membrane with trypsine. The resulting peptides were separated by reverse phase chromatography on HPLC and subjected to N-terminal sequencing as above.<br>
The following sequences have been obtained:<br>
-	N-terminal: AVVQFQLDLTYEDVSVAGXVXKA<br>
I V L N G X I (SEQ ID NO 48)<br>
-	Internal  peptide  1:   SEDQAGDYTIR   (SEQ  ID NO  47)<br>
-	Internal   peptide   2:ASQYXSYIYHSHTR    (SEQ   ID<br>
NO  50)<br>
EXAMPLE  19:  cloning  of  an  oxidase  encoding  gene  from Aspergillus ficuuzn DSM932<br>
Partial cloning of an oxidase gene from A. ficuum DSM932.<br>
Based on known partial protein sequences encoded by the oxidase gene the polymerase chain reaction (PCR) was used to isolate a DNA fragment which has then be used as a probe to screen a genomic DNA library of A. ficuum DSM932. A set of degenerated primers (oligonucleotides) were synthesized based on the amino-terminal sequence and on the sequence of an internal peptide of the purified protein.<br>
These nucleotide primers were:<br>
-	forward PCR-primer   5'  GTI GTI CAG TTT CAG YTI GAT YTI<br>
AC 3' (SEQ ID NO 37);<br>
-	reverse PCR-primer    3' CTY CTR GTY CGI CCI CTG ATG TGI<br>
TA 5' (SEQ ID NO 38) .<br>
These two primers were used for PCR amplification on the genomic DNA of A. ficuum DSM932. The PCR reaction mixtures were analyzed directly by electrophoresis through a 1.3% agarose gel followed by ethidium bromide staining.<br>
Two weak DNA bands of ± 550 and 700 bp and one strong DNA band of 1150 bp were generated.<br>
Nucleotide sequence analysis revealed the relation between the 1150 bp PCR product and the purified enzyme with oxidase activity.<br>
For this the PCR products were cloned in the pUC18 vector as described in example 5. Sequencing with the "universal" forward and reverse oligonucleotides primers (compatible with the insert-flanking regions of the vector) resulted in sequence determination of about 400 nucleotides on both sides of the insert. These nucleotide sequences were translated to peptide sequences. These peptides sequences included the sequence YEDVSVAGKVXXAIVLNG (SEQ ID NO 49) corresponding to a part of the amino terminal sequence determined in example 18 (from amino acid 11 to 28 SEQ ID NO 48) and the sequence ASQYXSYIYHSHTR (SEQ ID NO 50) corresponding to the sequence of the internal peptide 2 described in example 18.<br>
Cloning of the entire oxidase encoding gene of A.  ficuum DSM932.<br>
Using PCR amplification groups of clones of the genomic DNA library containing plasmids with the oxidase gene were found.<br>
The PCR reaction was performed on plasmid DNA (80 ng)  extracted by the Birnboim procedure from 20 pools of the library each of about 1000 clones. The resulting PCR products were analyzed by electrophoresis through a 1.2% agarose gel.<br>
Six groups of clones generated a PCR product of the expected size 1150 bp the same length as obtained from the original PCR amplification on the genomic DNA of Aspergillus ficuum DSM932. Three of these positive pools were further screened by colony hybridization using the radioactively labeled 1150 bp DNA fragment as probe. Five positive signals were obtained after autoradiography detection.<br>
The positive clones were further purified by a new cycle of colony hybridization and by PCR amplification of the 1150 bp fragment from the purified plasmids. This finally resulted in the isolation of two single positive clones.<br>
The DNA of the positive clones was analyzed by Sfil digestion resulting in the release of the genomic insert from the YCp50 vector.<br>
One clone showed after electrophoresis on a 0.8 % agarose gel a genomic insert of a length of 11000 bp and the other one an insert of more than 12000 bp. Both inserts contained an internal Sfil cleavage site. The plasmid with the 12000 bp insert was termed pAFLAC and has been deposited as an E. coli MC1061 transformant at the LMBP collection with reference number LMBP4366.<br>
BELDEM S.A. who has registered office at B-5300 Andenne (Belgium) Rue Bourrie 12 has made on 11 May 2001 (under the expert solution) of the microorganism Escherichia coli MC1061(pAFLAC) according to the invention at the BCCM/LMBP Culture Collection (Laboratorium voor Moleculaire Biologie Universiteit Gent xFiers-Schell-Van Montagu' building Technologiepark 927 B-9052 Gent-Zwijnaarde<br><br>
Belgium). This deposit has received the accession number LMBP 4366.<br>
Sequence analysis of the A. ficuum DSM932 oxidase gene.<br>
Genomic DNA inserts from pAFLAC were sheared and subcloned in the pUC18 plasmid vector. The ligation mixes were transformed into E. coli MC1061 to obtain two libraries of subclones. Subclones containing part of the desired gene were selected after colony hybridization using the radioactively labeled 1150 bp PCR fragment as probe.<br>
Purified (Qiagen) plasmid DNA of 12 positive subclones was further sequenced and the DNA sequences obtained were analyzed.<br>
This random sequencing gave the nucleotide sequence of the first 1100 bp of the gene as well as 500 bp of the promoter region. To obtain the entire gene sequence primer walking was performed to sequence the 3' end of the gene. Remaining ambiguities were resolved using custom primers. Only one of the two positive genomic clones (the one with the genomic DNA insert of more than 12000 bp) appeared to contain the entire gene.<br>
The nucleotide sequence and the corresponding amino-acid sequence (SEQ ID NO 1 and SEQ ID NO 40) are shown in figure 3. The gene is 1923 base pairs. It contains two introns of respectively 85 (starting at base pair 245) and 49 (starting at base pair 808) base pairs and encodes an open reading frame of 596 amino acids with a calculated molecular weight of 65476 Da.<br>
A search for similarities between the sequence obtained and sequences in databases was performed using the BLAST program (Basic Local Alignment Search Tool; Altschul &amp; al 1990 J. Mol. Biol. 215 p.403). The closest homologous gene product was a hypothetical protein from Aspergillus nidulans FGSC A4 (AN 0878.2 - accession number EAA65907). The<br><br>
overall homology was 56.7% as found by analysis with the Clustalw program using the default parameters (http://www.ebi.ac.uk.clustalw).<br>
EXAMPLE 20; Heterologous expression of the oxidase gene.<br>
The fungal strain Aspergillus nidulans 2024 which is auxotrophic for biotin and arginin was chosen as host strain for the expression of the isolated Aspergillus ficuum oxidase gene.<br>
Cotransformation of the positive clone (YCpSO vector with the genomic DNA insert of more than 12000 bp) with the selection plasmid pSa!23 (John &amp; al 1984 Enzyme Microbiol. Technol. 6:.386) containing the argB-gene was performed.<br>
Whether the resulting transformants (obtained on plates without arginin) had also integrated the positive clone was checked using PCR amplification and Southern blot analysis.<br>
Genomic DNA was isolated from cultures of the transformants. This genomic DNA was submitted to a PCR amplification reaction. DNA of A. ficuum DSM932 and of the untransformed A. nidulans 2024 were used as positive and negative controls respectively.<br>
The PCR products were analyzed by electrophoresis on a 1.2% agarose gel with ethidium bromide staining.<br>
The PCR reaction with the DNA of A. ficuum DSM932 or the DNA from some transformants generated as expected a 1150 bp DNA fragment indicating the A. ficuum gene integration in the genome of these transformants. The PCR reaction on the untransformed A. nidulans gave no amplification signal. Southern blot analysis confirmed these results.<br>
The cotransformants showed one or more signals on the blot indicating one or more integrated A. ficuum gene copies.<br>
A cotransformation efficiency of 60 % was obtained.<br>
Another phenomenon observed on the positive transformants was the change of the spore color. Non-transformed and pSa!23-transformed A. nidulans colonies showed dark green spores while transformants with the oxidase gene integrated gave yellow spores.<br>
Analysis of Aspergillus nidulans transformants.<br>
The transformants were checked whether expression of the integrated A. ficuum gene was established and whether the synthesized protein was enzymatically active.<br>
Transformants were grown in Aspergillus minimal medium. A. ficuum was used as positive control and the untransformed and pSa!23-transformed A. nidulans as negative controls.<br>
The proteins secreted in the medium were analyzed by electrophoresis on a 12.5% SDS-polyacrylamide gel. The culture medium samples were previously concentrated by deoxycholate-trichloroacetic acid precipitation. Proteins were stained on the gel with Coomassie blue. A very large protein band of 85 kDa with a size close to the size of the A. ficuum oxidase appeared only in the samples of the transformants of A. nidulans. The protein secreted by A. nidulans is about 5 kDa smaller than the one secreted by A. ficuum. This is due to differences in the glycosylation patterns as shown below.<br>
These results show that the isolated A. ficuum oxidase gene was expressed in A. nidulans under the control of its own promoter. More surprinsingly the expression level was even higher in A. nidulans transformants than in the wild-type A. ficuum DSM932 strain.<br>
Zymogram was performed to study the enzymatic activity of the expressed protein. Proteins in the culture medium were concentrated and loaded on a SDS-polyacrylamide gel under semi-denaturating conditions (absence of 15-mercaptoethanol; no boiling of the samples before loading). After electrophoresis the gel was incubated in a NN-dimethyl-p-phenylenediamine substrate solution. The observed protein bands of A. ficuum DSM932 and of the A. nidulans transformants were firstly pink-colored and became black after a longer incubation time confirming that the protein could oxidize the substrate and was enzymatically active. The same results were also obtained in solution in an oxidative assay with the same substrate NN-dimethyl-p-phenylenediamine. The medium of the untransformed A nidulans showed no activity at all.<br>
The results obtained above demonstrated that the integrated A. ficuum gene in the A. nidulans transformants code for an enzymatically active oxidase. Some transformants showed more activity than the original A. ficuum strain due to a higher gene expression.<br>
EXAMPLE 21; Glycosylation of the enzyme with oxidase activity.<br>
A. ficuum DSM932 and one A. nidulans transformants were grown in Aspergillus minimal medium. The secreted proteins in the medium were concentrated and further submitted to a PNGase F treatment. Then the deglycosylated samples were loaded on a polyacrylamide gel and a Coomassie blue staining was performed.<br>
A clear shift of the protein pattern was visible. Deglycosylation with PNGase F decreased the molecular weight of the two oxidases to about 70 kDa. This experiment showed that the observed 5 kDa difference in size of the non-deglycosylated oxidases of A ficuum and A. nidulans<br>
transformant was indeed due to differences in glycosylation since deglycosylation resulted into proteins of the same size.<br>
EXAMPLE 22: Expression of the enzyme with oxidase activity in various Aspergillus species under the control of the gpdA promoter.<br>
The promoter of the isolated A. ficuum oxidase gene was replaced by the constitutive gpdA (glyceraldehyde-3-phosphate dehydrogenase A) promoter of A. nidulans known as a strong fungal promoter (Deng &amp; al 1992 Anal. Biochem. 200: p-81) .<br>
This plasmid was transformed into A. nidulans 2024 A. niger N402 and A. ficuum DSM932.<br>
The transformation procedure was based on the cotransformation with the pSa!23 (arginine selection).<br>
To transform A. niger and A. ficuum another selection procedure was used based on the acetamidase encoding gene (Kelly &amp; al 1985 EMBO J. 4: 475). In these cases the plasmid p3SR2 containing the amdS gene (encoding the acetamidase) of A. nidulans was used for the cotransformation resulting in colonies which could grow on acetamide as sole nitrogen source.<br>
The transformants obtained (146 for A. nidulans 89 for A. niger and 326 for A. ficuum) were analyzed by PCR amplification with a set of two primers. The upstream primer hybridized with a sequence in the gpdA promoter and the downstream primer with a sequence in the oxidase gene. A DNA band of 2281 bp (corresponding to the expected size between the sequences of the oligonucleotide primers in the sequences of the gpdA promoter and the oxidase. gene) was detected in the PCR mix of some transformants. This demonstrated that these transformants were cotransformed and thus had integrated not only the selection plasmid but also the plasmid pFGPDAfic.<br>
These cotransformants of A. niger and A. ficuum exhibited the same spores color change as mentioned above for the A. nidulans transformants (grey to light brown spores instead of black - example 4).<br>
The transformants were further analyzed by SDS-polyacrylamide gel electrophoresis and by the enzymatic assay with the substrate NN-dimethyl-p-phenylenediamine. The transformants were grown in Aspergillus minimal medium (5 ml in a 50 ml plastic tube) for three days and the culture supernatant was submitted to the enzymatic assay and to electrophoresis.<br>
The best A. nidulans transformants showed an expression level 5.5 times higher than the expression obtained with the original oxidase promoter in A. nidulans.<br>
This observation showed that promoter exchange resulted in a higher expression level of the oxidase in A. nidulans.<br>
The best transformants of A. ficuum exhibited also a 4-fold increase in comparison with A. nidulans which was transformed with the oxidase controlled by its own promoter.<br>
The A. niger transformants secreted about the same amount.<br>
EXAMPLE 23: characterization of the oxidase<br>
1. Obtention of an oxidase sample<br>
One transformant of Aspergillus ficuum obtained in the example 22 was selected for further experiments. A 10 1 fermentation was performed as described in example 2.<br>
The supernatant of the fermentation was desalted in the same conditions as described in example 17 and concentrated by ultrafiltration.<br>
2. Determination of the optimum pH<br>
The optimum pH of the oxidase was determined by assaying its activity at various pHs using a sodium phosphate-citrate buffer. The results of this experiment are shown in figure 3.<br>
The optimum pH lies around pH 5.5.<br>
EXAMPLE 24; Tea color modification by laccase<br>
30 ml of a green tea solution made by pre-<br>
incubation of commercial tea (Pickwick Green Tea Douwe Egberts N.V. Belgium) in warm water (Ig per 100 ml) during 15 min were stirred with various amounts of Aspergillus ficuum oxidase (01 and 10 U per ml of green tea) at 40°C.<br>
After 1 hour tea solutions incubated with oxidase became red-brown and their absorbance spectra were recorded between 350 and 680 nm and compared with that of the control without enzyme.<br>
The results on Figure 4 clearly show that the use of the oxidase of Aspergillus ficuum enhance the color of the tea solution as compared with an untreated sample.<br>
EXAMPLE 25: Effect of the oxidase in baking<br>
Baking trials were performed to demonstrate the positive effect of the oxidase of the present invention in baking. The positive effect was evaluated by the increase in bread volume as compared to a reference which does not contain this enzyme.<br>
The oxidase from example 23 was evaluated in mini baking tests consisting of preparing dough with lOOg of flour.<br>
The procedure described is well established and it will be readily apparent to a person skilled in the art that the same results may be obtained by using other protocols or equipments from other suppliers.<br>
The  ingredients  used are  listed  in table  1<br>
below:<br>
The ingredients were mixed for 3.5 min in a National mixer. 150 g dough pieces were weighed and rested for 20 min at 25°C in plastic boxes.<br>
The doughs were reworked and rested for a further 20 min. The final proofing time was 60 min at 36°C. The dough pieces were then baked at 220°C for 24 min.<br>
The volume of the bread was measured using the commonly used rapeseed displacement method.<br>
The results of the baking trials with the oxidase are presented in table 2 below and on figure 5:<br>
Table 2 (*)   See example  13  for the  enzymatic assay<br>
The above results show that the oxidase of the present invention has a positive effect on the volume of bread.<br>
EXAMPLE 26: Effect of the oxidase on rye flour dough stickyness<br>
A series of doughs was prepared by mixing the following ingredients in a Farinograph mixer at 30°C for 2min.<br>
     The     basic     bread     improver     RB14120     contains     alpha-amylase     xylanase ascorbic acid and citric  acid.<br>
The dough stickyness was measured 5 min after mixing using a Stable Micro Systems TA-XT21 texture analyzer equipped with a Chen-Hoseney dough stickyness eel.<br>
The    results    are    presented    on    table    4    and    on<br>
figure  6.<br>
From these data it can be seen that the dough stickyness is significantly reduced by the addition of the oxidase according to the present invention.<br>
Example  27;  effect  of  the  oxidase  on  rye  flour  dough consistency.<br>
The same doughs as described in example 10 were prepared.<br>
The dough consistency was measured using a Physica UDS 200 Rheometer with the following parameters:<br>
-	oscillating mode<br>
-	temperature = 30°C<br>
-	plate-plate system 0=30mm d=2mm<br>
-	frequency sweep: y=0.02% F increases linearly from 1 to<br>
50Hz.<br>
The results obtained at 10 Hz are presented on Table 5 and figure 7.<br>
These data show that the dough consistency is significantly increased by the addition of the oxidase of the present invention. Moreover the dough becomes stiffer and permits a better handling and shaping of the dough. <br><br><br><br><br><br>
WE CLAIM:<br>
1.	An isolated oxidase polypeptide comprising:<br>
-	an amino acid sequence of any of SEQ ID NO 40 to 50, or<br>
-	a fragment of at least 100 amino acids of SEQ ID NO 40,<br>
41 or 45, or<br>
-	an amino acid sequence presenting at least 60% identity<br>
with the amino acid sequence of any of SEQ ID 40, 41 or<br>
45, or<br>
-	an amino acid sequence presenting at least 70% identity<br>
with any fragments of at least 100 amino acids of SEQ ID<br>
NO 40, 41 or 45.<br><br>
2.	An  isolated  nucleic  acid  molecule  encoding  a<br>
polypeptide as claimed in claim 1.<br>
3.	An isolated nucleic acid molecule comprising:<br><br>
-	a nucleotide sequence of any of SEQ ID NO 1 to 36, its<br>
complementary form or RNA form, or<br>
-	a nucleotide sequence having at least 70% identity with<br>
any of SEQ ID NO 1 to 36, or with the complementary form<br>
or RNA form thereof, or<br>
-	a fragment of any of SEQ ID NO 1 to 11 of at least 600<br>
nucleotides, or of any of their complementary form or RNA<br>
form, wherein said fragment encodes a protein having an<br>
oxidase activity, or<br>
-	a fragment of at least 20 nucleotides of any of SEQ ID NO<br>
1 to 36, or of any of their complementary form or RNA<br>
form.<br>
4.	An isolated nucleic acid molecule as claimed in<br>
claim 3 wherein said fragments of any of SEQ ID NO 1 to 11, or<br>
of any of their complementary form or RNA form, encoding a<br>
protein having an oxidase activity, consist of at least 900 nucleotides.<br>
5.	A vector  comprising a nucleic acid molecule as<br>
claimed in any of claims 2 to 4.<br>
6.	A vector as claimed in claim 5 wherein said nucleic<br>
acid molecule  as  claimed  in  any  of  claims  2  to  4  is<br>
operatively linked to one or more regulatory sequences.<br>
7.	A vector as claimed in claim 5 or 6 wherein said<br>
regulatory  sequence  is  the  gpdA promoter  of  Aspergillus<br>
nidulans.<br>
8.	A  transformed  host  cell  having  incorporated  a<br>
nucleic acid molecule as claimed in any of claims 2 to 4.<br>
9.	A transformed host cell  containing  a vector as<br>
claimed in any of claims 5 to 7.<br>
10.	A fusion protein comprising an oxidase polypeptide<br>
as claimed in claim 1.<br>
11.	A bread improving composition comprising an oxidase<br>
polypeptide as claimed in claim 1.<br>
12.	Use of an oxidase polypeptide as claimed in claim 1<br>
in a baking process or as color enhancer.<br>
13.	An isolated oxidase polypeptide substantially as<br>
herein described with reference to foregoing examoples and as<br>
illustrated in the accompanying drawings.<br>
14. An isolated nucleic acid molecule substantially as herein described with reference to foregoing examoples and as illustrated in the accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUFic3RyYWN0LSgyOS0wNC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Abstract-(29-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUFzc2lnbm1lbnQgKDA5LTA0LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Assignment (09-04-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNsYWltcy0oMTAtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Claims-(10-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNsYWltcy0oMTQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Claims-(14-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNsYWltcy0oMjktMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Claims-(29-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRhbmNlIE90aGVycy0oMjktMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondance Others-(29-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondence Others-(10-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondence Others-(14-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondence Others-(14-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMDctMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondence-others (07-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMDktMDQtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondence-others (09-04-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMTYtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Correspondence-others (16-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUZvcm0tMTggKDE2LTA2LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Form-18 (16-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUZvcm0tMi0oMjktMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Form-2-(29-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUZvcm0tMyAoMDctMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Form-3 (07-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUZvcm0tMy0oMTQtMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-Form-3-(14-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LUdQQSAoMDktMDQtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-GPA (09-04-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0xMDUucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-105.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0yMDIucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-202.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjAwMS1kZWxucC0yMDA3LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">2001-delnp-2007-pct-308.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260548-a-plasma-process-for-surface-modification-of-brass-for-brass-to-rubber-bonding.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260550-method-for-producing-extensible-paper.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260549</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2001/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>19/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>09-May-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-May-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Mar-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PURATOS NV</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>INDUSTRIALAAN 25, B-1702, GROOT-BIJGAARDEN, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VANNESTE, GUY</td>
											<td>AVENUE DU GRAND CORTIL, 56,B-1348 LOUVAIN-LA-NEUVE, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GEORIS, JACQUES CLAUDE ELOI</td>
											<td>RUE SCHMERLING 16, B-4000 LIEGE, BELGIUM</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ARNAUT, FILIP</td>
											<td>WEVERSTRAAT 58, B-1761 ROOSDAAL, BELGIUM</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CONTRERAS, ROLAND</td>
											<td>MOLENSTRAAT 53, B-9820 MERELBEKE, BELGIUM</td>
										</tr>
										<tr>
											<td>5</td>
											<td>DAUVRIN, THIERRY</td>
											<td>RUE BORDIA, 15, B-4218 COUTHUIN, BELGIUM</td>
										</tr>
										<tr>
											<td>6</td>
											<td>VIAENE, JASMINE</td>
											<td>OUDENBURGWEG 9, B-8490 VARSENARE, BELGIUM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/BE2005/000138</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-09-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04447202.5</td>
									<td>2004-09-16</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260549-oxidase-polypeptide-and-nucleic-acid-encoding-the-same-and-vector-and-transformed-host-cell-comprising-the-nucleic-acid-and-uses-of-the-oxidase-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:45:20 GMT -->
</html>
