<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260592-a-method-of-obtaining-soybean-germplasm by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:58:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260592:&quot;A METHOD OF OBTAINING SOYBEAN GERMPLASM.&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD OF OBTAINING SOYBEAN GERMPLASM.&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention overcomes the deficiencies of the prior art by providing methods for marker assisted selection to create plants of a soybean variety that exhibit a mid/low linolenic acid content with a commercially significant yield and an agronomically elite phenotype. The invention also provides derivatives and plant parts of these plants. Further provided by the invention are methods for the use of these plants. The invention is significant in that oil with decreased linolenic acid exhibits numerous beneficial characteristics yet prior art varieties with decreased linolenic acid also exhibited decreased yield and poor agronomic quality.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION<br>
HIGH YIELDING SOYBEAN PLANTS WITH LOW LINOLENIC ACID BACKGROUND OF THE INVENTION<br>
This application claims the priority of U.S. Provisional Patent Appl. Ser. No. 60/614,331, filed September 29, 2004, the entire disclosure of which is specifically incorporated herein by reference.<br>
1.	Field of the Invention<br>
The present invention relates generally to the field of plant breeding. In particular, the invention relates to agroriomically elite soybean varieties with commercially significant yield and a mid/low-lmolenic acid content.<br>
2.	Description of Related Art<br>
Soybean seeds are an important source of vegetable oil, which is used in food products throughout the world. The relatively high level (usually about 8%) of linolenic acid (18:3) in soybean oil reduces its stability and flavor.<br>
Hydrogenation of soybean oil is used to lower the level of linolenic acid (18:3) and improve both stability and flavor of soybean oils (Dutton et al., 1951; Lui and White, 1992). However, hydrogenation results in the production of trans fatty acids, which increases the risk for coronary heart disease when consumed (Hu et al., 1997).<br>
Varieties of low linolenic acid soybean have been produced through mutation, screening and breeding (Fehr et al., 1992; Rahman and Takagi, 1997; Ross et al., 2000; Byrum et al., 1997; Stoisin et al., 1998). Varieties with a linolenic acid content on the order of 1% or lower in particular have been produced (U.S. Patent Nos. 5,534,425 and 5,714,670). However, the low linolenic acid lines produced to date have been plagued poor seed yield and other agronomic characteristics desired for commercial production. The problem has been difficult to solve and is complicated by the quantitative nature of agronomic traits such as linolenic acid content and yield. The usefulness of low linolenic acid content soybean has therefore been limited in most commercial settings.<br>
Developing a product with commercially significance seed yield is a high priority in most soybean cultivar development programs. Yield is controlled by many genes and strongly influenced by the environment. It is a characteristic of central importance to the<br><br>
Commercial value of a variety and breeders continually attempt to improve yield beyond that presently available. It is a difficult challenge to incorporate low linolenic acid content into high yielding cultivars.<br>
Likely because of the difficulty, the prior art has failed to provide high yielding soybean varieties that also posses low linolenic acid and agronomically elite characteristics. However, there is a great need in the art for such soybean plants. The Food and Drug Administration (FDA) has proposed regulations on nutrition labeling to require that the amount of trans fatty acids in a food be included in the Nutrition Facts panel. In addition to the health benefits of reducing our reliance on hydrogenation of soybean oils, the aforementioned proposal by FD A has sparked great interest in the production of low linolenic acid (less than 3%) soybean that does not require, or requires less hydrogenation. Decreased linolenic acid can significantly improve the value of a soybean harvest. For the decreased linolenic acid to have commercial significance, yield and/or elite agronomic traits must not be substantially impacted. Therefore, providing soybean plants that are agronomically elite while both high yielding and possessing decreased linolenic acid would represent a substantial advance in the art and benefit farmers and consumers alike.<br>
SUMMARY OF THE INVENTION<br>
In one aspect, the invention provides a soybean plant of an agronomically elite variety having a mid/low-linolenic acid content and a commercially significant yield. Also provided are (he parts of this plant, including, but not limited to, pollen, an ovule, a cell and a seed. Further provided is a tissue culture of regenerable cells of the plant, wherein the tissue culture regenerates soybean plants capable of expressing all the physiological and morphological characteristics of the plant. In one embodiment of the invention, the regenerable cells are embryos, meristematic cells, pollen, leaves, roots, root tips or flowers or are protoplasts or callus derived therefrom. Further provided by the invention is a soybean plant regenerated from the tissue culture capable of expressing all the physiological and morphological characteristics of the starting plant.<br>
In certain embodiments of the invention, a mid/low-linolenic acid content is defined as a linolenic acid content of from about 1.0% to about 3.0% by weight of total seed fatty acids, including from about 1.5% to about 3.0%, about 2% to about 2.6%, about 2% to about 3%, about 1% to about 2.6%, about 1% to about 2.2%, about 1.6% to about 2.6% and about 2% to about 2.4% by weight of total seed fatty acids. Such plants may further be defined as<br><br>
â€¢having a grain yield of, for example, at least about 90%, 94%, 98%, 100%, 103% 105% or about 110% of the check lines AG2703 and DKB23-51. Line AG2703, which also has the designations SN79553 and 9323265446452, was patented in U.S. Patent No. 6,184,442, the disclosure of which is incorporated herein by reference in its entirety. The line designated DKB23-51, which also has the designations 02122920 and 958361722350, was patented in U.S. Patent No. 6,369,302, the disclosure of which is incorporated herein by reference in its entirety. Seeds of ACJ2703 and DKB23-51 have been deposited with the ATCC under ATCC accession numbers PTA-2577 and PTA-3933, respectively.<br>
In yet another aspect, the invention provides plant parts of a plant of the invention. Examples of such parts include pollen, an ovule, a meristem or a cell. The invention also provides seeds of a plant described herein, as well as tissue cultures comprising cells of such a plant, wherein the tissue culture regenerates soybean plants expressing the physiological and morphological characteristics of the plant. The tissue culture may be comprised of regenerable cells such as embryos, meristematic cells, pollen, leaves, roots, root tips or flowers.<br>
In still another aspect, the invention provides a soybean plant of the invention comprising a transgene. The transgene may in one embodiment be defined as conferring a trait selected from the group consisting of herbicide tolerance; disease resistance; insect or pest resistance; altered fatty acid, protein or carbohydrate metabolism; increased grain yield; altered plant maturity, and altered morphological characteristics. One example of herbicide resistance is glyphosate resistance.<br>
In particular embodiments, a plant of the invention may be further defined as produced by a method comprising the steps of: a) crossing first and second soybean plants, wherein the plants comprise Fad3-lb and Fad3-lc alleles conferring decreased linolenic acid content, wherein the first plant has a mid/low-linolenic acid content, and wherein the second plant comprises a commercially significant yield; b) assaying progeny soybean plants resulting from the crossing for yield and for the presence of polymorphisms located in a soybean plant gcnomic region within 50 cM of said Fad3-lb and Fad3-lc alleles; and c) selecting at least a first agrononncally elite progeny plant comprising said polymorphisms and a commercially significant yield to obtain the plant of claim 1.<br>
In still yet another aspect, the invention provides a method of obtaining soybean gennplasm, comprising the steps of: a) identifying at least a first polymorphism in a soybean plant genomic region within 50 cM of a Fad3-lb or Fad3-lc allele conferring decreased<br><br>
Jinolcnic acid content; h) assaying soybean plants for the presence of the polymorphism; and c) selecting at least a first soybean plant comprising the polymorphism. The method may comprise identifying polymorphisms in a soybean plant genomic region within 50 cM of both of said Fad3-!b and Fad3-lc alleles and assaying for the presence of said polymorphisms. In one embodiment, the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3~lh gene sequence corresponding to nucleotide 2021 of SEQ ID NO: 1. In another embodiment, the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lc gene sequence corresponding to nucleotide 687, 1129, 1203, 2316, 3292, 3360 or 3743 of SEQ ID NO:2. The first polymorphism may also comprise a deletion in the Fad3-lc gene sequence, and may comprise a polymorphism in the Fad3-lc promoter, such as a single nucleotide polymorphism at a position corresponding to nucleotide 334, 364, 385, 387, 393, 729 or 747 of SEQ ID NO:3. Detecting the polymorphism may be carried out by any method, including PCR, single strand conformational polymorphism analysis, denaturing gradient gel electrophoresis, cleavage fragment length polymorphism analysis and/or DNA sequencing.<br>
In still yet another aspect, the invention provides a method of plant breeding<br>
comprising the steps of: a) assaying soybean plants for the presence of at least a first<br>
polymorphism in a soybean plant genomic region within 50 cM of a Fad3-lb or Fad3-Jc<br>
allele conferring decreased linolenic acid content; b) selecting at least a first soybean plant<br>
comprising the polymorphism; and c)	crossing the first soybean plant to a second<br>
soybean plant to produce progeny plants comprising the polymorphism. The method may further comprise the step of: d) selecting a progeny plant comprising the polymorphism and crossing the progeny plant with a third soybean plant to produce additional progeny plants. In the method the second and third plants may be of the same variety. In certain embodiments, the method further comprises repeating step d) about 2-10 times. The method may still further comprise assaying soybean plants for the presence of polymorphisms in soybean plant genomic regions within 50 cM of said Fad3-Ib and Fad3-lc alleles and selecting said first soybean plant may be based on the presence of the polymorphisms. In certain embodiments, markers linked to Fad3-lb and Fad3-lc may be assayed without assaying for markers tightly linked to Fad3~la, as the inventors have shown that it is the Fad3-lb and Fad3-lc alleles that contribute a low linolenic acid content.<br>
In certain embodiments of the method, the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lb gene corresponding to nucleotide 2021<br><br>
of SHQ ID NO:1. 1 he first polymorphism may also comprise a single nucleotide polymorphism at a position in the Fad3-Jc gene corresponding to nucleotide 687, 1129, 1203, 2316, 3292, 3360 or 3743 of SEQ ID NO:2. In still other embodiments the first polymorphism comprises a deletion in the Fad3-lc gene sequence and/or a single nucleotide polymorphism at a position in the Fad3-lc promoter corresponding to nucleotide 334, 364, 385, 387, 393, 729 or 747 of SEQ ID NO:3. Selecting at least a first soybean plant comprising the polymorphism may be carried out by any method, such as, for example, PCR, single strand conformational polymorphism analysis, denaturing gradient gel electrophoresis, cleavage fragment length polymorphism analysis and/or DNA sequencing.<br>
In still yet another aspect, the invention provides a probe or primer that hybridizes under stringent conditions to a soybean plant genomic region within 50 cM of a Fad3-lb or Fad3-lc allele, wherein the probe or primer is a nucleic acid sequence selected from the group consisting SEQ ID NOs:4-98.<br>
Still yet another aspect of the invention is a method of producing a food product for human or animal consumption comprising: (a) obtaining a plant of the invention; (b) cultivating the plant to maturity; and (c) preparing a food product from the plant. In certain embodiments of the invention, the food product may be protein concentrate, protein isolate, meal, oil, flour or soybean hulls.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention overcomes the deficiencies of the prior art by providing agronomically elite soybean varieties with a mid/low-Iinolenic acid content and commercially acceptable gram yield. The invention is significant in that, although the benefits of these characteristics have been realized individually, they have not previously been combined in a single variety. The provision of a mid/low linolenic acid content in combination with other desirable characteristics provides many benefits. For example, current soybean oils must typically be at least partially hydrogenated and/or mixed with other oils due to problems with oil stability. Reduced linolenic acid content reduces the need for either solution by improving stability and, depending upon use, can eliminate the need for hydrogenation. The cost and quality of soybean oil can therefore be markedly improved by decreasing linolenic acid content, making the oil increasingly competitive relative to other seed oils. Low linolenic acid content also reduces off-flavors and thus lines with this characteristic have higher commercial value (Liu<br><br>
'and White, 1992).  However, widespread adoption of low-linolenic varieties has to date been hampered by poor yields or agronomic quality.<br>
The invention provides genetic markers and methods for their use for the creation of such improved plants. This is significant because of the complex inheritance of quantitative traits, such as yield and linolenic acid content, which is made exponential when attempting to combine the traits. The markers were identified using the candidate gene approach. Locus-specific nested primers were designed to cover the entire FADS gene family consisting of three independent loci. Amplicons were generated from 25 different genotypes comprising 9 mutants and 16 wild types. SNPs and Indels were identified through sequence alignment. Genetic segregation analysis confirmed that the markers identified linked to the alleles Fad3-Ih and Fad3~lc, which were associated with mutations in the corresponding wildtype sequences yielding low linolenic phenotypes. Analysis of the segregating plants demonstrated that Fad3-lh and Fad3-lc additively control linolenic content in soybean. Therefore, using a combination of markers for Fad3-Ib and Fad3-lc, the invention allows accurate prediction of linolenic acid content in plants without the need for expensive biochemical analyses. These markers were successfully demonstrated for use in low linolenic soybean breeding programs and allowed, for the first time, the development of soybean varieties combining a low linolenic acid phenotype with commercially significant yield and agronomically elite characteristics.<br>
The prior art has failed to provide plants of such a variety, presumably because of the difficulty in combining different traits with complex inheritance and lack of means for overcoming these difficulties. By describing methods for the production of such plants and providing examples of these plants, the invention now allows the preparation of a potentially unlimited number of novel soybean varieties exhibiting a commercially significant yield with combined with low linolenic acid content and specifically a mid/low linolenic acid content. Once such an elite variety is produced the combined yield and low linolenic acid content can be transferred to other varieties with appropriate backcross and selection to maintain the desirable traits as described herein below.<br>
I.	Plants of the Invention<br>
The invention provides for the first time plants and derivatives thereof of soybean varieties that combine commercially significant yield and mid/low-linolenic acid content with an agronomically elite phenotype. Such plants may be defined as having a commercially<br><br>
significant yield, for example, that is defined as a yield of at least 103% of the check lines AG2703 and DKB23-51. In certain further embodiments, plants are provided having a mid/low-linolenic acid content and a grain yield of at least about 90%, 94%, 98%, 100%, 105% or about 110% of these lines. Such plants may be defined, in certain embodiments of the invention, as having a yield a yield in excess of about 35, 37, 39, 41, 43 or 45 bushels per acre over at least 10 environments. In particular embodiments of the invention, the mid/low-linolenic acid content may be defined as from about 1% to about 3% of seed fatty acid content, including from about 1.3% to about 3%, from about 1.5% to about 3%, from about 1.8% to about 3%, from about 2.1% to about 3%, from about 2.4% to about 3%, from about 2.6% to about 3%, from about 1% to about 2.6%, from about 1.3% to about 2.6%, from about 1,8% to about 2.0%, from about 2% to about 2.6%, from about 2% to about 2.4% and from about 1.6% to about 2.4% of seed fatty acid content.<br>
One aspect of the current invention is therefore directed to the aforementioned plants and parts thereof and methods for using these plants and plant parts. Plant parts include, but are not limited to, pollen, an ovule and a cell. The invention further provides tissue cultures of regenerable cells of these plants, which cultures regenerate soybean plants capable of expressing all the physiological and morphological characteristics of the starting variety. Such regenerable cells may include embryos, meristematic cells, pollen, leaves, roots, root tips or flowers, or protoplasts or callus derived therefrom. Also provided by the invention are soybean plants regenerated from such a tissue culture, wherein the plants are capable of expressing all the physiological and morphological characteristics of the starting plant variety from which the regenerable cells were obtained.<br>
II.        Marker Assisted Selection for Production of Soybean Varieties with Mid/Low-Linolenic Acid Content<br>
The current invention provides genetic markers and methods for the introduction of loci conferring a mid/low-linolenic acid content in soybean plants. The invention therefore allows for the first time the creation of plants that combine this linolenic acid content with a commercially significant yield and an agronomically elite genetic background. Using the methods of the invention, loci conferring decreased linolenic acid content may be introduced into potentially any desired soybean genetic background, for example, in the production of new varieties with commercially significant yield and a mid/low-linolenic acid content.<br><br>
Marker assisted introgression involves the transfer of a chromosome region defined by one or more markers from one germplasm to a second germplasm. The initial step in that process is the localization of the trait by gene mapping, which is the process of determining the position of a gene relative to other genes and genetic markers through linkage analysis. The basic principle for linkage mapping is that the closer together two genes are on the chromosome, the more likely they are to be inherited together. Briefly, a cross is generally made between two genetically compatible but divergent parents relative to traits under study. Genetic markers can then be used to follow the segregation of traits under study in the progeny from the cross, often a backcross (BC1), F'2, or recombinant inbred population.<br>
The term quantitative trait loci, or QTL, is used to describe regions of a genome showing qualitative or additive effects upon a phenotype. The current inventors have identified genetic markers tor two such QTLs, Fad3-]b and Fad3-lc. The invention therefore allows the use of molecular tools to combine these QTLs with desired characteristics.<br>
A.        Development of Linked Genetic Markers<br>
A sample first plant population may be genotyped for an inherited genetic marker to form a genotypic database. As used herein, an "inherited genetic marker" is an allele at a single locus. A locus is a position on a chromosome, and allele refers to conditions of genes; that is, different nucleotule sequences, at those loci. The marker allelic composition of each locus can be either homozygous or heterozygous. In order for information to be gained from a genetic marker in a cross, the marker must by polymorphic; that is, it must exist in different forms so that the chromosome carrying the mutant gene can be distinguished from the chromosome with the normal gene by the form of the marker it also carries.<br>
Formation of a phenotypic database can be accomplished by making direct observations of one or more traits on progeny derived from artificial or natural self-pollination of a sample plant or by quantitatively assessing the combining ability of a sample plant. By way of example, a plant line may be crossed to, or by, one or more testers. Testers can be inbred lines, single, double, or multiple cross hybrids, or any other assemblage of plants produced or maintained by controlled or free mating, or any combination thereof For some self-pollinating plants, direct evaluation without progeny testing is preferred.<br>
The marker genotypes may be determined in the testcross generation and the marker loci mapped. To map a particular trait by the linkage approach, it is necessary to establish a positive correlation in inheritance of a specific chromosomal locus with the inheritance of the<br>
strait In the case of complex inheritance, such as with as quantitative traits, including specifically linolenic acid content and yield, linkage will generally be much more difficult to discern. In this case, statistical procedures may be needed to establish the correlation between phenotype and genotype This may further necessitate examination of many offspring from a particular cross, as individual loci may have small contributions to an overall phenotype.<br>
Coinheritance, or genetic linkage, of a particular trait and a marker suggests that they are physically close together on the chromosome. Linkage is determined by analyzing the pattern of inheritance of a gene and a marker in a cross. The unit of recombination is the cenlimorgaii (cM). Two markers are one centimorgan apart if they recombine in meiosis once in every 100 opportunities that they have to do so. The centimorgan is a genetic measure, not a physical one. Those markers located less then 50cM from a second locus are said to be genetically linked, because they are not inherited independently of one another. Thus, the percent of recombination observed between the loci per generation will be less than 50%. In particular embodiments of the invention, markers may be used located less than about 45, 35, 25, 15, 10, 5, 4, 3, 2, or 1 or less cM apart on a chromosome. In certain embodiments of the invention markers may be used detecting polymorphisms within the contributing loci themselves and thus located at 0 cM respective to the loci, for example, comprising a mutation within a h\id3-1 h or Fad3-lc coding sequence or regulatory element.<br>
During meiosis, pairs of homologous chromosomes come together and exchange segments in a process called recombination. The further a marker is from a gene, the more chance there is that there will be recombination between the gene and the marker. In a linkage analysis, the coinhentance of marker and gene or trait are followed in a particular cross. The probability that their observed inheritance pattern could occur by chance alone, i.e., that they are completely unlinked, is calculated. The calculation is then repeated assuming a particular degree of linkage, and the ratio of the two probabilities (no linkage versus a specified degree of linkage) is determined. This ratio expresses the odds for (and against) that degree of linkage, and because the logarithm of the ratio is used, it is known as the logarithm of the odds, eg. an lod score. A lod score equal to or greater than 3, for example, is taken to confirm that gene and marker are linked. This represents 1000:1 odds that the two loci are linked. Calculations of linkage is greatly facilitated by use of statistical analysis employing programs.<br>
The genetic linkage of marker molecules can be established by a gene mapping model such as, without limitation, the flanking marker model reported by Lander and Botstein<br>
(1989), and the interval mapping, based on maximum likelihood methods described by Lander and Botstein (1989), and implemented in the software package MAPMAKER/QTL (Lincoln and Lander, 1990). Additional software includes Qgene, Version 2.23 (1996) (Department of Plant Breeding and Biometry, 266 Emerson Hall, Cornell University, Ithaca, NY).<br>
B.        Inherited markers<br>
Genetic markers comprise detected differences (polymorphisms) in the genetic information carried by two or more plants. Genetic mapping of a locus with genetic markers typically requires two fundamental components: detectably polymorphic alleles and recombination or segregation of those alleles. In plants, the recombination measured is virtually always meiotic, and therefore, the two inherent requirements of animal gene mapping are polymorphic genetic markers and one or more plants in which those alleles are segregating.<br>
Markers are preferably inherited in codominant fashion so that the presence of both alleles at a diploid locus is readily detectable, and they are free of environmental variation, i.e.. their heritability is 1. A marker genotype typically comprises two marker alleles at each locus in a diploid organism such as soybeans. The marker allelic composition of each locus can be either homo/ygous or heterozygous. Homozygosity is a condition where both alleles at a locus are characterized by the same nucleotide sequence. Heterozygosity refers to different conditions of the gene at a locus.<br>
A number of different markers types are available for use in genetic mapping. Exemplary genetic marker types for use with the invention include, but are not limited to, restriction fragment length polymorphisms (RFLPs), simple sequence length polymorphisms (SSLPs), amplified fragment length polymorphisms (AFLPs), single nucleotide polymorphisms (SNPs), and isozymes. Polymorphisms comprising as little as a single nucleotide change can be assayed in a number of ways. For example, detection can be made by dectrophoretic techniques including a single strand conformational polymorphism (Orita et ni, 1989), denaturing gradient gel electrophoresis (Myers et al., 1985), or cleavage fragment length polymorphisms (Life Technologies, Inc., Gathersberg, MD 20877), but the widespread availability of DNA sequencing machines often makes it easier to just sequence amplified products directly. Once the polymorphic sequence difference is known, rapid assays can be designed for progeny testing, typically involving some version of PCR<br>
amplification of specific alleles (PASA, Sommer, et ai,    1992), or PCR amplification of multiple specific alleles (PAMSA, Dutton and Sommer, 1991).<br>
Restriction fragment length polymorphisms (RFLPs) are genetic differences detectable by DNA fragment lengths, typically revealed by agarose gel electrophoresis, after restriction endonuclease digestion of DNA. There are large numbers of restriction endonucleases available, characterized by their nucleotide cleavage sites and their source, e.g., EcoRl. RFLPs result from both single-bp polymorphisms within restriction site sequences and measurable insertions or deletions within a given restriction fragment RFLP are easy and relatively inexpensive to generate (require a cloned DNA, but no sequence) and are co-dominant. RFLPs have the disadvantage of being labor-intensive in the typing stage, although this can be alleviated to some extent by multiplexing many of the tasks and reutilization of blots. Most RFLP are biallelic and of lesser polymorphic content than microsatellites. For these reasons, the use of RFLP in animal gene maps has waned.<br>
One of skill in the art would recognize that many types of molecular markers are useful as tools to monitor genetic inheritance and are not limited to RFLPs, SSRs and SNPs, anil one of skill would also understand that a variety of detection methods may be employed to track the various molecular markers. One skilled in the art would also recognize that markers of different types may be used for mapping, especially as technology evolves and new types of markers and means for identification are identified.<br>
For purposes of convenience, inherited marker genotypes maybe converted to numerical scores, e.g., if there are 2 forms of an SNP, or other marker, designated A and B, at a particular locus using a particular enzyme, then diploid complements may be converted to a numerical score, for example, are AA=2, AB=1, and BB=0; or AA=1, AB=0 and BB=1. The absolute values of the scores are not important. What is important is the additive nature of the numeric designations. The above scores relate to codominant markers. A similar scoring system can be given that is consistent with dominant markers.<br>
C.        Marker Assisted Selection<br>
The invention provides soybean plants with a mid/low-linolenic acid content in combination with a commercially significant yield and agronomically elite characteristics. Such plants may be produced in accordance with the invention by marker assisted selection methods comprising assaying genomic DNA for the presence of markers that are genetically linked to a Fad3-lh and/or Irad3-lc allele conferring decreased linolenic acid.<br>
In certain embodiments of the invention it maybe desired to obtain additional markers linked to Fad3 -Ib and/or Fad3-lc alleles. This may be carried out, for example, by first preparing an F? population by selling an Fl hybrid produced by crossing inbred varieties only one of which comprises a Fad'3-lb and/or Fad3-lc allele conferring decreased linolenic acid content. Recombinant inbred lines (RIL) (genetically related lines; usually &gt;F5, developed from continuously selfing F? lines towards homozygosity) can then be prepared and used as a mapping population Information obtained from dominant markers can be maximized by using RIL because all loci are homozygous or nearly so.<br>
Backcross populations (e.g., generated from a cross between a desirable variety (recurrent parent) and another variety (donor parent) carrying a trait not present in the former can also be utilized as a mapping population. A series of backcrosses to the recurrent parent can be made to recover most of its desirable traits. Thus a population is created consisting of individuals similar to the recurrent parent but each individual carries varying amounts of genomic regions from the donor parent. Backcross populations can be useful for mapping dominant markers if all loci in the recurrent parent are homozygous and the donor and recurrent parent have contrasting polymorphic marker alleles (Reiter et al., 1992).<br>
Useful populations for mapping purposes are near-isogenic lines (NIL). NIJLs are created by many backcrosses to produce an array of individuals that are nearly identical in genetic composition except for the desired trait or genomic region can be used as a mapping population. In mapping with NILs, only a portion of the polymorphic loci are expected to map to a selected region. Mapping may also be carried out on transformed plant lines.<br>
D.        Plant Breeding Methods<br>
Breeding techniques take advantage of a plant's method of pollination. There are two general methods of pollination: self-pollination which occurs if pollen from one flower is transferred to the same or another flower of the same plant, and cross-pollination which occurs if pollen comes to it from a flower on a different plant. Plants that have been self-pollinated and selected for type over many generations become homozygous at almost all gene loci and produce a uniform population of true breeding progeny, homozygous plants.<br>
In development of suitable varieties, pedigree breeding may be used. The pedigree breeding method for specific traits involves crossing two genotypes. Each genotype can have one or more desirable characteristics lacking in the other; or, each genotype can complement the other. If the two original parental genotypes do not provide all of the desired characteristics, other genotypes can be included in the breeding population. Superior plants<br>
*thai are the products of these crosses are selfed and are again advanced in each successive generation. Each succeeding generation becomes more homogeneous as a result of sell-pollination and selection. Typically, this method of breeding involves five or more generations of selfing and selection: S|â€”^82; 82â€”^S^; 83-^84; 84â€”Â»Ss, etc. A selfed generation (S) may be considered to be a type of filial generation (F) and may be named F as such. After at least five generations, the inbred plant is considered genetically pure.<br>
Each breeding program should include a periodic, objective evaluation of the efficiency of the breeding procedure. Evaluation criteria vary depending on the goal and objectives. Promising advanced breeding lines are thoroughly tested and compared to appropriate standards in environments representative of the commercial target area(s) for generally three or more years. Identification of individuals that are genetically superior because genotypic value can be masked by confounding plant traits or environmental factors. One method of identifying a superior plant is to observe its performance relative to other experimental plants and to one or more widely grown standard varieties. Single observations can be inconclusive, while replicated observations provide a better estimate of genetic worth.<br>
Mass and recurrent selections can be used to improve populations of either self-or cross-pollinating crops. A genetically variable population of heterozygous individuals is either identified or created by intercrossing several different parents. The best plants are selected based on individual superiority, outstanding progeny, or excellent combining ability. The selected plants are intercrossed to produce a new population in which further cycles of selection are continued. Descriptions of other breeding methods that are commonly used for different traits and crops can be found in one of several reference books (e.g., Allard, 1960; Simmonds, 1979; 8neep vial., 1979; Fehr, 1987a,b).<br>
The effectiveness of selecting for genotypes with traits of interest (e.g., high yield, disease resistance, fatty acid profile) in a breeding program will depend upon: 1) the extent to which the variability in the traits of interest of individual plants in a population is the result of genetic factors and is thus transmitted to the progenies of the selected genotypes; and 2) how much the variability in the traits of interest among the plants is due to the environment in which the different genotypes are growing. The inheritance of traits ranges from control by one major gene whose expression is not influenced by the environment (i.e., qualitative characters) to control by many genes whose effects are greatly influenced by the environment (i.e., quantitative characters). Breeding for quantitative traits such as yield is further characterized by the fact that: 1) the differences resulting from the effect of each gene are<br>
small, making it difficult or impossible to identify them individually; 2) the number of genes contributing to a character is large, so that distinct segregation ratios are seldom if ever obtained; and 3) the effects of the genes may be expressed in different ways based on environmental variation. Therefore, the accurate identification of transgressive segregates or superior genotypes with the traits of interest is extremely difficult and its success is dependent on the plant breeder's ability to minimize the environmental variation affecting the expression of the quantitative character in the population.<br>
The likelihood of identifying a transgressive segregant is greatly reduced as the number of traits combined into one genotype is increased. For example, if a cross is made between cultivars differing in three complex characters, such as yield, linolenic acid content and at least a first agronomic trait, it is extremely difficult without molecular tools to recover simultaneously by recombination the maximum number of favorable genes for each of the three characters into one genotype. Consequently, all the breeder can generally hope for is to obtain a favorable assortment of genes for the first complex character combined with a favorable assortment of genes for the second character into one genotype in addition to a selected gene.<br>
Backcrossing is an efficient method for transferring specific desirable traits. This can be accomplished, for example, by first crossing a superior variety inbred (A) (recurrent parent) to a donor inbred (non-recurrent parent), which carries the appropriate gene(s) for the trait in question (Fehr, 1987). The progeny of this cross are then mated back to the superior recurrent parent (A) followed by selection in the resultant progeny for the desired trait to be transferred from the non-recurrent parent. Such selection can be based on genetic assays, as mentioned below, or alternatively, can be based on the phenotype of the progeny plant. After five or more backcross generations with selection for the desired trait, the progeny are heterozygous for loci controlling the characteristic being transferred, but are like the superior parent for most or almost all other genes. The last generation of the backcross is selfed, or sibbed, to give pure breeding progeny for the gene(s) being transferred, for example, loci providing the plant with decreased linolenic acid content.<br>
In one embodiment of the invention, the process of backcross conversion may be defined as a process including the steps of:<br>
(a)        crossing a plant of a first genotype containing one or more desired<br>
gene, DNA sequence or element, such as Fad3-lb and/or Fad3-lc alleles<br>
associated  with decreased  linolenic acid content, to a plant of a second genotype lacking said desired gene, DNA sequence or element; (h)        selecting one or more progeny plant containing the desired gene, DNA sequence or element;<br>
crossing the progeny plant to a plant of the second genotype; and<br>
repeating steps (b) and (c) for the purpose of transferring said desired<br>
gene, DNA sequence or element from a plant of a first genotype to a plant of a<br>
second genotype.<br>
Introgression of a particular DNA element or set of elements into a plant genotype is defined as the result of the process of backcross conversion. A plant genotype into which a DNA sequence has been introgressed may be referred to as a backcross converted genotype, lint;, inbred, or hybrid. Similarly a plant genotype lacking the desired DNA sequence may be referred to as an unconverted genotype, line, inbred, or hybrid. During breeding, the genetic markers linked to decreased linolenic acid content may be used to assist in breeding for the purpose of producing soybean plants with decreased linolenic acid content and preferably a mid/low-linolenic acid content. Backcrossing and marker assisted selection in particular can be used with the present invention to introduce the decreased linolenic acid content trait in accordance with the current invention into any variety by conversion of that variety with fr'ad.l- Ih and/or Fad3-lc alleles associated with the trait, with both loci being preferred.<br>
The selection ol a suitable recurrent parent is an important step for a successful backcrossing procedure The goal of a backcross protocol is to alter or substitute a trait or characteristic in the original inbred. To accomplish this, one or more loci of the recurrent inbred is modified or substituted with the desired gene from the nonrecurrent parent, while retaining essentially all of the rest of the desired genetic, and therefore the desired physiological and morphological, constitution of the original inbred. The choice of the particular nonrecurrent parent will depend on the purpose of the backcross, which in the case of the present invention may be add one or more allele conferring decreased linolenic acid content. The exact backcrossing protocol will depend on the characteristic or trait being altered to determine an appropriate testing protocol. Although backcrossing methods are simplified when the characteristic being transferred is a dominant allele, a recessive allele may also be transferred. In this instance it may be necessary to introduce a test of the progeny to determine if the desired characteristic has been successfully transferred. In the case of the present invention, one may test the linolenic acid content of progeny lines generated during<br>
the backcrossing program as well as using the marker system described herein to select lines based upon markers rather than visual traits.<br>
Soybean plants (Clycinc max I,.) can be crossed by either natural or mechanical techniques (see, c(<.f. fehr natural pollination occurs in soybeans either by self or cross which typically is aided pollinating organisms. artificial crosses flowering and time are an important consideration. soybean a short-day plant but there considerable genetic variation for sensitivity to pholoperiod criswell hume the critical day length ranges from about h genotypes adapted tropical latitudes photoperiod-insensitive grown at higher et al. seem be insensitive days after emergence. photoperiods shorter than required complete flower induction parker shanmugasundaram tsou></.f.>
Soybean flowers typically are self-pollinated on the day the corolla opens. The stigma is receptive to pollen about 1 day before anthesis and remains receptive for 2 clays after anlhesis, if the flower petals are not removed. Filaments of nine stamens are fused, and the one nearest the standard is free. The stamens form a ring below the stigma until about 1 day belore anthesis, then their filaments begin to elongate rapidly and elevate the anthers around the stigma. The anthers dehisce on the day of anthesis, pollen grains fall on the stigma, and within 10 h the pollen lubes reach the ovary and fertilization is completed (Johnson and Bernard, 1963). Self-pollination occurs naturally in soybean with no manipulation of the flowers. For the crossing of two soybean plants, it is typically preferable, although not required, to utilize artificial hybridization. In artificial hybridization, the flower used as a female in a cross is manually cross pollinated prior to maturation of pollen from the flower, thereby preventing sell fertilization, or alternatively, the male parts of the flower are emasculated using a technique known in the art. Techniques for emasculating the male parts of a soybean flower include, for example, physical removal of the male parts, use of a genetic factor conferring male sterility, and application of a chemical gametocide to the male parts.<br>
Either with or without emasculation of the female flower, hand pollination can be earned out by removing the stamens and pistil with a forceps from a flower of the male parent and gently brushing the anthers against the stigma of the female flower. Access to the stamens can be achieved by removing the front sepal and keel petals, or piercing the keel with closed forceps and allowing them to open to push the petals away. Brushing the anthers on<br>
1he stignia causes them to rupture, and the highest percentage of successful crosses is obtained when pollen is clearly visible on the stigma. Pollen shed can be checked by tapping the anthers before brushing the stigma. Several male flowers may have to be used to obtain suitable pollen shed when conditions are unfavorable, or the same male may be used to pollinate several llowers with good pollen shed.<br>
Genetic male sterility is available in soybeans and may be useful to facilitate hybridization in the context of the current invention, particularly for recurrent selection programs (Brim and Stuber, 1973). The distance required for complete isolation of a crossing block is not clear; however, outcrossing is less than 0.5% when male-sterile plants are 12 m or more from a foreign pollen source (Boerma and Moradshahi, 1975). Plants on the boundaries of a crossing block probably sustain the most outcrossing with foreign pollen and can be eliminated at harvest to minimize contamination.<br>
Once harvested, pods are typically air-dried at not more than 38Â°C until the seeds contain 13% moisture or less, then the seeds are removed by hand. Seed can be stored satisfactorily at about 25Â°C for up to a year if relative humidity is 50% or less. In humid climates, germination percentage declines rapidly unless the seed is dried to 7% moisture and stored in an air-tighl container at room temperature. Long-term storage in any climate is best accomplished by drying seed to 7% moisture and storing it at 1()Â°C or less in a room maintained at 50% relative humidity or in an air-tight container.<br>
III.      Traits for Modification and Improvement of Soybean Varieties<br>
In certain embodiments, a soybean plant provided by the invention may comprise one or more transgene(s). One example of such a transgene confers herbicide resistance. Common herbicide resistance genes include an EPSPS gene conferring glyphosate resistance, a neomycin phosphotransferase II (nptll) gene conferring resistance to kanamycin (Fraley et al., 1983), a hygromycm phosphotransferase gene conferring resistance to the antibiotic hygromycin (Vanden Elzen et al., 1985), genes conferring resistance to glufosinate or broxynil (Comai et Â«/., 1985; Gordon-Kamm et al, 1990; Stalker et al., 1988) such as dihydrofolate reductase and acetolactate synthase (Eichholtz et al., 1987, Shah et al., 1986, Charest et al., 1990). Further examples include mutant ALS and AHAS enzymes conferring resistance to imidazalinonc or a sulfonylurea (Lee et al., 1988; Miki et ai, 1990), a phosphinothricin-acetyl-lransferase gene conferring phosphinothricin resistance (European Appln. 0 242 246), genes conferring resistance to phenoxy proprionic acids and<br>
â€¢cycloshexones, such as sethoxydim and haloxyfop (Marshall et al., 1992); and genes conferring resistance to triazine (psbA and gs+ genes) and benzonitrile (nitrilase gene) (Pr/ibilae/Â«/., 1991).<br>
A plant of the invention may also comprise a gene that confers resistance to insect, pest, viral or bacterial attack. For example, a gene conferring resistance to a pest, such as soybean cyst nematode was described in PCT Application WO96/30517 and PCT Application W( )93/19181. Jones et al., (1994) describe cloning of the tomato Cf-9 gene for resistance to (.ladosponum fulvum); Martin et al., (1993) describe a tomato Pto gene for resistance to Pscudomonas syringae pv. and Mindrinos et al., (1994) describe an Arabidopsis RSP2 gene for resistance to Pscudomonas syringae. Bacillus thuringiensis endotoxins may also be used for insect resistance. (See, for example, Geiser et al., (1986). A vitamin-binding protein such as avidin may also be used as a larvicide (PCT application US93/06487).<br>
The use of use of viral coat proteins in transformed plant cells is known to impart resistance to viral infection and/or disease development affected by the virus from which the coat protein gene is derived, as well as by related viruses. (See Beachy et al., 1990). Coat protein-mediated resistance has been conferred upon transformed plants against alfalfa mosaic virus, cucumber mosaic virus, tobacco streak virus, potato virus X, potato virus Y, tobacco etch vims, tobacco rattle virus and tobacco mosaic virus. Id. Developmental-arreslive proteins produced in nature by a pathogen or a parasite may also be used. For example, Logemann et al., (1992), have shown that transgenic plants expressing the barley ribosome-inactivating gene have an increased resistance to fungal disease.<br>
Transgencs may also be used conferring increased nutritional value or another value-added trait. One example is modified fatty acid metabolism, for example, by transforming a plant with an antisense gene of stearoyl-ACP desaturase to increase stearic acid content of the plant. (See Knut/on et al., 1992). A sense desaturase gene may also be introduced to alter fatty acid content. Phytate content may be modified by introduction of a phytase-encoding gene to enhance breakdown of phytate, adding more free phosphate to the transformed plant. Modified carbohydrate composition may also be affected, for example, by transforming plants with a gene coding for an enzyme that alters the branching pattern of starch. (See Shiroza et al., 1988) (nucleotide sequence of Streptococcus mutants fructosyltransferase gene); Steinmet/, ct al., (1985) (nucleotide sequence of Bacillus subtilis levansucrase gene); Pen et at., (1992) (production of transgenic plants that express Bacillus lichenifonnis a-amylase); Elliot et al., (199?) (nucleotide sequences of tomato invertase genes); S0gaard et al., (1993)<br>
(site-directed  mutagenesis of barley u-amylase gene); and Fisher et al., (1993)  (maize endosperm starch branching enzyme II)).<br>
Transgenes may also be used to alter protein metabolism. For example, U.S. Patent No. 5,545,545 describes lysine-insensitive maize dihydrodipicolinic acid synthase (DHPS), which is substantially resistant to concentrations of L-lysine which otherwise inhibit the activity of native DHPS. Similarly, EP 0640141 describes sequences encoding lysine-insensitive aspartokinase (AK) capable of causing a higher than normal production of threonme, as well as a subfragment encoding antisense lysine ketoglutarate reductase for increasing lysine.<br>
In another embodiment, a transgene may be employed that alters plant carbohydrate metabolism. For example, fructokinase genes are known for use in metabolic engineering of fruotokinase gene expression in transgenic plants and their fruit (see U.S. Patent No. 6,03 1,154). A further example of transgenes that may be used are genes that alter grain yield. For example, U.S. Patent No. 6,486,383 describes modification of starch content in plants with subunit proteins of adenosine diphosphoglucose pyrophosphorylase ("ADPG PPase"). In BP0797673, transgenic plants are discussed in which the introduction and expression of particular DNA molecules results in the formation of easily mobilised phosphate pools outside the vacuole and an enhanced biomass production and/or altered flowering behavior. Still further known are genes for altering plant maturity. U.S. Patent No. 6,774,284 describes DNA encoding a plant lipa.se and methods of use thereof for controlling senescence in plants. U.S. Patent No. 6,140,085 discusses FCA genes for altering flowering characteristics, particularly timing of flowering. U.S. Patent No. 5,637,785 discusses genetically modified plants having modulated flower development such as having early floral meristem development and comprising a structural gene encoding the LEAFY protein in its genome.<br>
Genes for altering plant morphological characteristics are also known and may be used in accordance with the invention. U.S. Patent No. 6,184,440 discusses genetically engineered plants which display altered structure or morphology as a result of expressing a cell wall modulation transgene. Examples of cell wall modulation transgenes include a cellulose binding domain, a cellulose binding protein, or a cell wall modifying protein or en/.ynie such as endoxyloglucan transferase, xyloglucan endo-transglycosylase, an expansin, cellulose synthase, or a novel isolated endo-l,4-B-glucanase.<br>
Methods for introduction of a transgene are well known in the art and include biological and physical, plant transformation protocols. See, for example, Miki et al. (1993).<br>
Once a transgene is introduced into a variety it may readily be transferred by crossing. By using backcrossing, essentially all of the desired morphological and physiological characteristics of a variety are recovered in addition to the locus transferred into the variety via the backcrossing technique. Backcrossing methods can be used with the present invention to improve or introduce a characteristic into a plant (Poehlman et al., 1995; Fehr, 1987a,b).<br>
IV,      Tissue Cultures and in vitro Regeneration of Soybean Plants<br>
A further aspecf of the invention relates to tissue cultures of a soybean variety of the invention. As used herein, the term "tissue culture" indicates a composition comprising isolated cells of the same or a different type or a collection of such cells organized into parts of a plant. Exemplary types of tissue cultures are protoplasts, calli and plant cells that are intact in plants or parts of plants, such as embryos, pollen, flowers, leaves, roots, root tips, anthers, and the like. In a preferred embodiment, the tissue culture comprises embryos, protoplasts, meristematic cells, pollen, leaves or anthers.<br>
Exemplary procedures for preparing tissue cultures of regenerable soybean cells and regenerating soybean plants therefrom, are disclosed in U.S. Pat. No. 4,992,375; U.S. Pat. No. 5,015,580; U.S. Pat. No. 5,024,944, and U.S. Pat. No. 5,416,011, each of the disclosures of which is specifically incorporated herein by reference in its entirety.<br>
An important ability of a tissue culture is the capability to regenerate fertile plants. This allows, for example, transformation of the tissue culture cells followed by regeneration of transgenic plants. For transformation to be efficient and successful, DNA must be introduced into cells that give rise to plants or germ-line tissue.<br>
Soybeans typically are regenerated via two distinct processes; shoot morphogenesis and somatic embryogenesis (Finer, 1996). Shoot morphogenesis is the process of shoot meristem organization and development. Shoots grow out from a source tissue and are excised and rooted to obtain an intact plant. During somatic embryogenesis, an embryo (similar to the zygotic embryo), containing both shoot and root axes, is formed from somatic plant tissue. An intact plant rather than a rooted shoot results from the germination of the somatic embryo.<br>
Shoot morphogenesis and somatic embryogenesis are different processes and the specific route of regeneration is primarily dependent on the explant source and media used for tissue culture manipulations. While the systems are different, both systems show variety-specific responses where some lines are more responsive to tissue culture manipulations than<br>
others. A line that is highly responsive in shoot morphogenesis may not generate many somatic embryos Lines that produce large numbers of embryos during an 'induction' step may not give rise to rapidly-growing proliferative cultures. Therefore, it may be desired to optimize tissue culture conditions for each soybean line. These optimizations may readily be carried out by one of skill in the art of tissue culture through small-scale culture studies. In addition to line specific responses, proliferative cultures can be observed with both shoot morphogenesis and somatic embryogenesis. Proliferation is beneficial for both systems, as it allows a single, transformed cell to multiply to the point that it will contribute to germ-line tissue.<br>
Shoot morphogenesis was first reported by Wright et al. (1986) as a system whereby shoots were obtained dc novo from cotyledonary nodes of soybean seedlings. The shoot meristems were formed subepidermally and morphogenic tissue could proliferate on a medium containing benzyl adenine (BA). This system can be used for transformation if the subepidermal, multicellular origin of the shoots is recognized and proliferative cultures are ulilt/.ed. The idea is to target tissue that will give rise to new shoots and proliferate those cells within the meristeniatic tissue to lessen problems associated with chimerism. Formation of chimeras, resulting from transformation of only a single cell in a meristem, are problematic if the transformed cell is not adequately proliferated and does not does not give rise to germ-line tissue. Once the system is well understood and reproduced satisfactorily, it can be used as one target tissue for soybean transformation.<br>
Somatic embryogenesis in soybean was first reported by Christiansen et al. (1983) as a system in which embryogenic tissue was initially obtained from the zygotic embryo axis. These embryogenic cultures were proliferative but the repeatability of the system was low and the origin of the embryos was not reported. Later histological studies of a different proliferative embryogenic soybean culture showed that proliferative embryos were of apical or surface origin with a small number of cells contributing to embryo formation. The origin of primary embryos (the first embryos derived from the initial explant) is dependent on the explant tissue and the auxin levels in the induction medium (Hartweck et al., 1988). With proliferative embryonic cultures, single cells or small groups of surface cells of the 'older' somatic embryos form the 'newer' embryos.<br>
Embryogenic cultures can also be used successfully for regeneration, including regeneration of transgenic plants, if the origin of the embryos is recognized and the biological limitations of proliferative embryogenic cultures are understood. Biological limitations<br>
include the difficulty in developing prolifcrative embryogenic cultures and reduced fertility problems (culture-induced variation) associated with plants regenerated from long-term proliferative embryogenic cultures. Some of these problems are accentuated in prolonged cultures. The use of more recently cultured cells may decrease or eliminate such problems.<br>
V.        Utilization of Soybean Plants<br>
A soybean plant provided by the invention may be used for any purpose deemed of value. Common uses include the preparation of food for human consumption, feed for non-human animal consumption and industrial uses. As used herein, "industrial use" or "industrial usage'1 refers to non-food and non-feed uses for soybeans or soy-based products.<br>
Soybeans are commonly processed into two primary products, soybean protein (meal) and crude soybean oil. Both of these products are commonly further refined for particular uses. Refined oil products can be broken down into glycerol, fatty acids and sterols. These can be for food, feed or industrial usage. Edible food product use examples include coffee creamers, margarine, mayonnaise, pharmaceuticals, salad dressings, shortenings, bakery products, and chocolate coatings.<br>
Soy protein products (e.g., meal), can be divided into soy flour concentrates and isolates which have both food/feed and industrial use. Soy flour and grits are often used in the manufacturing of meat extenders and analogs, pet foods, baking ingredients and other food products Food products made from soy flour and isolate include baby food, candy products, cereals, food drinks, noodles, yeast, beer, ale, etc. Soybean meal in particular is commonly used as a source of protein in livestock feeding, primarily swine and poultry. Feed uses thus include, but arc not limited to, aquaculture feeds, bee feeds, calf feed replacers, fish feed, livestock feeds, poultry feeds and pet feeds, etc.<br>
Whole soybean products can also be used as food or feed. Common food usage includes products such as the seed, bean sprouts, baked soybean, full fat soy flour used in various products of baking, roasted soybean used as confectioneries, soy nut butter, soy coffee, and other soy derivatives of oriental foods. For feed usage, hulls are commonly removed from the soybean and used as feed.<br>
Soybeans additionally have many industrial uses. One common industrial usage for soybeans is the preparation of binders that can be used to manufacture composites. For example, wood composites may be produced using modified soy protein, a mixture of hydroly/ed soy protein and FF resins, soy flour containing powder resins, and soy protein<br>
containing foamed glues. Soy-based binders have been used to manufacture common wood products such as plywood for over 70 years. Although the introduction of urea-formaldehyde and phenol-formaldehyde resins has decreased the usage of soy-based adhesives in wood products, environmental concerns and consumer preferences for adhesives made from a renewable feedstock have caused a resurgence of interest in developing new soy-based products for the wood composite industry.<br>
Preparation of adhesives represents another common industrial usage for soybeans. Examples of soy adhesives include soy hydrolyzate adhesives and soy flour adhesives. Soy hydrolyzate is a colorless, aqueous solution made by reacting soy protein isolate in a 5 percent sodium hydroxide solution under heat (120Â° C) and pressure (30 psig). The resulting degraded soy protein solution is basic (pH 11) and flowable (approximately 500 cps) at room temperature. Soy Hour is a finely ground, defatted meal made from soybeans. Various adhesive formulations can be made from soy flour, with the first step commonly requiring dissolving the flour in a sodium hydroxide solution. The strength and other properties of the resulting formulation will vary depending on the additives in the formulation. Soy flour adhesives may also potentially be combined with other commercially available resins.<br>
Soybean oil may find application in a number of industrial uses. Soybean oil is the most readily available and one of the lowest-cost vegetable oils in the world. Common industrial uses for soybean oil include use as components of anti-static agents, caulking compounds, disinfectants, fungicides, inks, paints, protective coatings, wallboard, anti-foam agents, alcohol, margarine, paint, ink, rubber, shortening, cosmetics, etc. Soybean oils have also for many years been a major ingredient in alkyd resins, which are dissolved in carrier solvents to make oil-based paints. The basic chemistry for converting vegetable oils into an alkyd resin under heat and pressure is well understood to those of skill in the art.<br>
Soybean oil in its commercially available unrefined or refined, edible-grade state, is a fairly stable and slow-drying oil. Soybean oil can also be modified to enhance its reactivity under ambient conditions or, with the input of energy in various forms, to cause the oil to copolymerize or cure to a dry film. Some of these forms of modification have included epoxidation, alcoholysis or tranesterification, direct esterification, metathesis, isomerization, monomer modification, and various forms of polymerization, including heat bodying. The reactive linoleic-acid component of soybean oil with its double bonds may be more useful than the predominant oleic- and linolcic-acid components for many industrial uses.<br><br>
Solvents can also he prepared using soy-based ingredients. For example, methyl soyate, a soybean-oil based methyl ester, is gaining market acceptance as an excellent solvent replacement alternative in applications such as parts cleaning and degreasing, paint and ink removal, and oil spill remediation. It is also being marketed in numerous formulated consumer products including hand cleaners, car waxes and graffiti removers. Methyl soyate is produced by the transesterification of soybean oil with methanol. It is commercially available from numerous manufacturers and suppliers. As a solvent, methyl soyate has important environmental- and safety-related properties that make it attractive for industrial applications. It is lower in toxicity than most other solvents, is readily biodegradable, and has a very high flash point and a low level of volatile organic compounds (VOCs). The compatibility of methyl soyate is excellent with metals, plastics, most elastomers and other organic solvents. Current uses of methyl soyate include cleaners, paint strippers, oil spill cleanup and bioremediation, pesticide adjuvants, corrosion preventives and biodiesel fuels additives.<br>
VI.       Definitions<br>
In the description and tables which follow, a number of terms are used. In order to provide a clear and consistent understanding of the specification and claims, the following definitions are provided:<br>
A: When used in conjunction with the word "comprising" or other open language in the claims, the words "a1 and "an" denote "one or more."<br>
Agronomically Elite: As used herein, means a genotype that has a culmination of many distinguishable traits such as emergence, vigor, vegetative vigor, disease resistance, seed set, standability and threshability which allows a producer to harvest a product of commercial significance.<br>
Allele: Any of one or more alternative forms of a gene locus, all of which alleles relate to a trait or characteristic. In a diploid cell or organism, the two alleles of a given gene occupy corresponding loci on a pair of homologous chromosomes.<br>
Backcrossing: A process in which a breeder repeatedly crosses hybrid progeny, for example a first generation hybrid (F\), back to one of the parents of the hybrid progeny. Backcrossing can he used to introduce one or more single locus conversions from one genetic background into another.<br><br>
Commercially Significant Yield: A yield of grain having commercial significance to the grower represented by an actual grain yield of 103% of the check lines AG2703 and DKB23-51 when grown under the same conditions.<br>
Crossing: The mating of two parent plants.<br>
Cross-pollination: Fertilization by the union of two gametes from different plants.<br>
F] Hybrid: The first generation progeny of the cross of two nonisogenic plants.<br>
Genotype:   The genetic constitution of a cell or organism.<br>
Industrial use: A non-food and non-feed use for a soybean plant. The term "soybean plant" includes plant parts and derivatives of a soybean plant.<br>
Linkage: A phenomenon wherein alleles on the same chromosome tend to segregate together more often than expected by chance if their transmission was independent.<br>
Marker: A readily detectable phenotype, preferably inherited in codominant fashion (both alleles at a locus in a diploid heterozygote are readily detectable), with no environmental variance component, i.e., heritability of 1.<br>
Phenotype: The detectable characteristics of a cell or organism, which characteristics are the manifestation of gene expression.<br>
Quantitative Trait Loci (QTL): Quantitative trait loci (QTL) refer to genetic loci that control to some degree numerically representable traits that are usually continuously distributed.<br>
Stringent Conditions: Refers to nucleic acid hybridization conditions of 5X SSC, 50% lormamide and 42"('.<br>
Substantially Equivalent: A characteristic that, when compared, does not show a statistically significant difference (e.g., p = 0.05) from the mean.<br>
Tissue Culture: A composition comprising isolated cells of the same or a different type or a collection of such cells organized into parts of a plant.<br>
Transgene: A genetic locus comprising a sequence which has been introduced into the genome of a soybean plant by transformation.<br>
VII.     Examples<br>
The following examples are included to demonstrate preferred embodiments of the invention. It should be appreciated by those of skill in the art that the techniques disclosed in the examples which follow represent techniques discovered by the inventor to function well in the practice of the invention, and thus can be considered to constitute preferred modes for<br><br>
its practice. However, those of skill in the art should, in light of the present disclosure, appreciate that many changes can he made in the specific embodiments which are disclosed and still obtain a like or similar result without departing from the concept, spirit and scope of the invention. More specifically, it will be apparent that certain agents which are both chemically and physiologically related may be substituted for the agents described herein while the same or similar results would be achieved. All such similar substitutes and modifications apparent to those skilled in the art are deemed to be within the spirit, scope and concept of the invention as defined by the appended claims.<br>
EXAMPLE 1 Mapping Populations and Identification of Polymorphisms<br>
Low linolenic lines 6P248, T27111, T27190, T26767 and T26830 were selected for use in identifying low-lin associated polymorphisms in maturity groups 1-4. The linolenic acid content of these lines was around 3%. The low-lin lines Soyola and N98-44 will be used in maturity groups 4-5. Other low-lin lines include A5 and C1640. Sixteen wild type lines with the following commercial designations were also used: A2247, AG1902, AG2402, AG2703, AG3201, AG3302, AG3702, AJB2102JOC, AJB2302KOC, CSR2533, CSR2622N, CSR3922N, DKB19-51, DK.B23-95 and WP25920. To facilitate mapping efforts, lines from previously used mapping populations, HS-1 and PI507354(PIC), were used in some sequencing regions. DNA was isolated from each genotype using standard DNA extraction protocols.<br>
Nested primers were designed to completely cover the loci Fad3-Ia, Fad3-lb, and Fad3-lc. The amplicons generated were from different lines. Sequences of these amplicons were aligned to identify SNPs and Indels associated with the low linolenic phenotypes. Initially, 13 pairs of primers were designed from Fad3-la, 6 pairs from FadB-lb, and 12 pairs from Fad3-lc. An additional 14 pairs of primers were designed for Fad3-lc from introns once their sequences were determined from this study. Table 1 lists the primers used in this analysis. Sequence alignment was done with Seqman program of DNA Star package.<br>
Taqman assays were designed and manufactured by Applied Biosystems based on the SNP sequences. SNP detection was done according to the instruction of Applied Biosystems.<br>
Table 1. Primers Designed for Resequencing on Fad3-la, Fad3-lh and Fad3-lc genes<br>
(SKQ ID NOs:4-82, respectively)	<br>
(Table Removed)<br>
Example 2 Identification of Polymorphisms at Fad3-la<br>
The sequence coverage of this locus was good. Table 2 shows the polymorphism identified at this locus. Ten SNPs and three indels were detected. However, no SNP or other marker haplotype was found to be associated with the low-linolenic phenotype. Overall, sequence variation at this locus was found to be significantly higher than that at Fad3-lb and Fad3 lc, indicating that it was not under selection pressure for this trait.<br>
Table 2. Polymorphisms at the Fad3-la locus<br><br>
(Table Removed)<br>
Note: The number in the parenthesis indicates number of sequence reads in a given line<br>
Example 3 Identification of Polymorphisms atFad3-lb<br>
The intron sequences of Fad3-lb were quite different from Fad3-la, which allowed the efficient generation of locus specific amplicons. The quality of sequences from most of the lines was generally high except for a small portion of the 5'UTR region. A single nucleotide polymorphism at position 2021 was detected among all the lines in an entire sequence length of 2683hp (Table 3). Interestingly, the SNP was found to associate with a low-lin phenolype. Low-linolenic lines 6P248, T27111, T27190, T26767 and T26830 were<br>
rfbund to carry a "C" allele at this position while all other lines carried a "T". The five lines with a "C" allele were tested tor linolenic acid content and all found to have less than 4% linolenic acid. Other low-lin lines such as A5, Soyola, and N98-4445 carried a wild type allele at this locus, indicating that one or more other loci contribute to the low-lin phenotype in these lines.<br>
To determine whether the SNP at 2021 position was a sense mutation, the ORF was translated into protein. This showed that the mutation in the low linolenic line changed an ammo acid residue from Histidine to Tyrosine. The histidine residue has been found to be critical in a number of genes involved with desaturation. The SNP found caused a mutation in the motif His-Val-Ile-His-His to His-Val-Ile-His-Tyr in the low linolenic lines. The motif has been associated with a low-lin phenotype and is a likely cause for the reduced linolenic acid<br>
Table 3. Polymorphism at the Fad3-lb locus<br>
(Table Removed)<br>
Note: The number in the parenthesis indicates number of sequence reads in a given line<br>
Example 4 Identification of polymorphisms at Fad3-lc<br>
No genomic DNA was initially available for Fad3-lc. However, cDNA was highly conserved between I'ad3-la and Fad3-lc across the entire gene with a sequence identity of higher than 90% To amplify introns using primers from exons, a number of primers targeting Fad3-lc specific regions were manually picked. Once the new sequences were known, new primers were designed from introns. Using this approach, partial sequences were successfully obtained covering all of the introns. Sequence analysis indicated that the Fad3-Ic allele was very similar to the Fad3-la locus even in the introns. A very high sequence similarity was observed near exon/intron junctions between the two loci, but decreased as sequences extended further.<br>
From the sequences obtained, four SNPs and one indel were identified at Fad3-lc (Table 4). The SNPs at positions 687, 2316, 3743, as well as the indel at 1129, appeared to be in linkage equilibrium and associated with the low-linolenic phenotype. Low-linolenic lines, Soyola and N98-4445 both carried a different allele at positions 687 and 1129 from all the other lines. Although sequences were not obtained from all the lines at positions 3360 and 3743, it was indicated that these loci are in linkage equilibrium with 687 and 1129. All four positive SNPs/lndel were located in introns.<br>
It was interesting to note that the iow-linolenic lines Soyola and N98-4445 were derived from gerrnplasm belonging to maturity group 4 to 5, while the other lines belong to maturity group 1 to 4. The mechanism for these mutations to cause the low-lin phenotype is unclear. One explanation was that there is another mutation located outside the coding region, probably in the promoter region, which caused the phenotype and is in linkage equilibrium with the markers detected within intron2. An analysis of the promoter sequence was therefore undertaken as described below.<br>
It was important to note from Table 4 that mutant lines 6P248, T27111, T27190, T26767, T26830 and A5 failed to amplify with nearly all of the Fad3-lc locus-specific primers. This indicated that there was a large deletion at the Fad3-lc locus in these lines. The length of deletion varied slightly in different mutant lines. This result was consistent with an earlier study showing that A5 did indeed carry a deletion at Fad3-lc. Since a large portion of the entire gene was deleted, the enzyme catalyzing the conversion of linoleic acid to linolenic acid could be predicted to not function properly. As a result, plants carrying this mutant would produce less linolenic acid.<br><br>
Table 4. Polymorphisms at the Fad3-lc locus<br>
Sequence position<br><br>
(Table Removed)<br>
Note: I. NA means no amplification detected<br>
2. The number in the parenthesis indicates number of sequence reads in a given line<br>
Example 6 Isolation of the promoter region of Fad3-lc<br>
To determine the factors that contribute to the low linolenic content in Soyola and N98-4445, efforts were made to clone the upstream region of the Fad3-lc gene. Three primers facing toward the upstream region were designed and the primers were used to amplify unknown promoter sequences according to the Genome Walking kit, purchased from CloueTech. An approximate 1 kb fragment was obtained from the A3244 line. The PCR product was directly sequenced after being treated with Exonuclease and Shrimp Alkaline<br>
Phosphatase. To identify polymorphisms associated with low-linolenic acid content, three new primer pairs were designed to cover the entire promoter and 5' end of the coding region, which were then used k&gt; amplify 24 different lines. Sequences from these amplicons were aligned to identify SNRs. Seven SNP were found in the promoter region (Table 5). Soyola carried a different allelc at all seven positions from the other wild-type lines. These SNPs could be the determining factor for the low-linolenic phenotypes. To confirm the association, these SNPs will be tested on a population segregating for linolenic acid content.<br>
Table 5. Polymorphisms at Fad3-lc Promoter Region<br>
Position<br>
(Table Removed)<br>
Example 7 Low-lin Marker Assay Validation<br>
Taqman end point assays were designed from four SNPs identified above (Table 6). Assays were named after the SNP positions in the eonsensus sequences at a given locus. For example, FD3A842 meant SNP from Fad3-la, at position 842 on the consensus sequence. A new marker name NS0193117 was later assigned to FD3A842 and used in production. The assays were validated on the same panel used in re-sequencing. FIG. 1A shows the allelogram of NS019311 7, The allelic patterns from the Taqman assay were consistent with the sequences.<br>
Two assays, FD3B2021A and FD3B2021B (NS0193115), were designed from Fad3-Ih. FIG. IB show the allelogram of NS0193115 on the sequencing panel. All four lines with low linolenic content had a different allele from the wild-type, corresponding well with the sequences. FIG. 1C shows the allelogram of NS0193116, derived from Fad3~lc position 687 Lines Soyola and N98-4445 showed a different allele from the others. All of the assay results corresponded to the sequence data.<br>
The SNP markers were further tested on eight populations segregating for linolenic acid content (Table 7). Six SNPs that were known to be linked to thefanfan locus were also<br>
Â»genolyped on the same populations. Since the populations were F4, they were treated as RI populations when used in the Mapmaker program. Heterozygous alleles were masked out.<br>
NS0131053 is a marker located on linkage group B2/U26, where the fanfan locus resides. Table 7 shows that NS0193116 (Fad3-lc) is linked to NS0131053 by approximately 10 cM distance. Therefore, fad3c corresponds to fanfan, as we expected. Since NS0131053 was found to be linked to NS0193116 (fad3c), NS0131053 could also serve as a reference marker for low-linolenic line selection. Selection could be made based on both the null allele at NS01931 16 and the low-linolenic allele at NS0131053. The data from these populations indicated that the Fcu/3-Ju, -Ib and Ic alleles were all independently inherited.<br>
A combination of the NS0193115 and NS0193116 markers was found to provide accurate diagnosis lor Hnolenic content. From the sequence results, it was clear that both F(nl3-lh and Ic play a role in controlling the linolenic level in soybean. Mutant lines with the I'\id3-lb (NS01931 1 5) mutation alone contained about 4.2% linolenic acid while Fad3-lc (NSOI93116) deletion alone contain about 3.4% (Table 8 ). Therefore, by combining selection for plants comprising markers for a double mutant at the Fad3~lb and Fad3-lc loci, even lower levels of linolenic acid could be obtained. FIG. 2 shows the phenotypic values of I'"ud3-1b and I
Table 6. Primers and Probes for Tag man Assays (SEQ ID NOs:83-97, respectively)<br><br>
(Table Removed)<br>
Fable 7. Genetic linkage between NS0193116 and NS0131053<br>
(Table Removed)<br>
Table 8. Percentage of Linolenic Content in Different Genotypes<br>
 (Table Removed)<br>
REFERENCES<br>
The following references, to the extent that they provide exemplary procedural or other details supplementary to those set forth herein, are specifically incorporated herein by reference.<br>
U.S. Patent 4,992,375 U.S. Patent 5,015,580 U.S. Patent 5,024,944 U.S. Patent 5,416,011 U.S. Patent 5,545,545 U.S. Patent 5,637,785 U.S. Patent 6,031,154 U.S. Patent 6,140,085 U.S. Patent 6,184,440 U.S. Patent 6,184,442 U.S. Patent 6,369,302 U.S. Patent 6,486,383 U.S. Patent 6,774,284<br>
Allard, In: Principle's of Hunt Breeding, John Wiley &amp; Sons, NY, 50-98, I960.<br>
Beachyf/ al., Ann. rev. 1'hytopathol. 28:451 (1990<br>
Boemiaand Moradshahi, Crop ScL, 15:858-861, 1975.<br>
Borthwick and Parker, Bnt. Gaz., 100:374-387, 1938.<br>
Brim and Stuber, Crop AW., 13:528-530, 1973.<br>
Byruin, etal.. Theor. Appl. (it-net., 94:356-359, 1997.<br>
C'harcst et al., Plant Cell Rep 8:643 (1990<br>
Christiansen et al., Science, 222:632-634, 1983.<br>
Comai et al, Nature  17:741 -744 (1985)<br>
Criswcll and Hume, Crop AW.. 12:657-660, 1972.<br>
Dhir etal.. Plant Cell Rep., 10(2):97-101, 1991.<br>
Dutton and Sonimer, Hiotechniques, 11(6):700-7002., 1991.<br>
Dutton et al., ./. Am. Oil ( 'hem. Soc., 28:115-118, 1951.<br>
Eichholtz et al., Somatic ( ell Mol. Genet. 13:67 (1987)<br>
â€¢Elliot et al., Plant Mold. Biol. 21:515 (1993<br>
European Appln. 0 242 246<br>
Fehr et ai, Crop Sci., 32:903-906, 1992.<br>
Fehr, In:   Theory and Technique, and Crop Species Soybean, Iowa State Univ., Macmillian<br>
Pub. Co., NY, (1 )(2):360-376, 1987b. Fehr, In: Soybeans: Improvement, Production and Uses, 2nd Edition, Manograph., 16:249,<br>
1987a. Fehr, In: Hybridization of Crop Plants, Fehr and Hadley (Eds.), Am. Sac. Agron. and Crop<br>
Sci. Soc. Am., Madison, WI, 90-599, 1980. Finer et al.,   In: Soybean: Genetics,   Molecular Biology and Biotechnology, CAB  Intl.,<br>
Venna and Shoemaker (ed), Wallingford, Oxon, UK, 250-251, 1996. Fraleyefrt/., Proc. Natl. Acad. Set. USA, 80:4803, 1983. Geiser et al. Gene, 48:109, 1986. (ionhm-Kamm el al. Plant ('ell, 2:603-618, 1990. Hamner, In:  The Induction of Flowering: Some Case Histories, Evans (ed), Cornell Univ.<br>
Press, Ithaca, NY, 62-89, 1969.<br>
Hartweck et al., In Vitro Cell. Develop. Bio., 24:821-828, 1988. Hu etai, N. Engl J MY/., 337:1491-1499, 1997.<br>
Johnson and Bernard, ;   The Soybean, Norman (ed), Academic Press, NY, 1-73, 1963. ionesetal.. Science. 266:789, 1994.<br>
Knut/on et al., Proc Natl. Acad. Sci. USA, 89:2624, 1992. Lander and Botstein, Genetics, 121(1): 185-199, 1989. Lee el al., EMBO./., 7:1241, 1988. et alcmAnnetal.Jiio/Technology, 10:305, 1992. Lui and White, .et al. Am Oil Chem. Soc., 69:528-532, 1992. Marshall etai, Theor Appl. Genet., 83:435, 1992. Martin et al., Science, 262:1432, 1993. Miki etal., Theor Appl. Genet., 80:449, 1990. Miiidrinost et al., Cell. 78:1089, 1994. Myers, EPO 027308S<br>
Onta et al., Genomics, 8(2):271-278, 1990. PCT Appln. US93/00487 PC "i Appln. WO93/1918I<br>
PCTAppln. WO96/30517<br>
Penetal.,Bio/Technolo.gy, 10:292, 1992.<br>
Poehlman and Slepcr, In: Breeding Field Crops, Iowa State University Press, Ames, 1995.<br>
Przibila Â£/Â«/., Plant Cell. 3:169, 1991.<br>
Rahman and Takagi, Theor. Appl. Genet., 94:299-302, 1997.<br>
Rahman et al., Crop Sci., 38:702-706, 1998<br>
Reitere/a/., Proc. Natl Acad. Sei. USA, 89(4):1477-1481, 1992.<br>
Ross et al, Crop Sci., 40:383-386, 2000.<br>
Shah et al., Science, 233:478, 1986.<br>
Shanmugasundanim and Tsou, Crop Sci., 18:598-601, 1978.<br>
Shibles et al., In: Crop Physiology, Some Case Histories, Evans (ed), Cambridge Univ.<br>
Press, Cambridge, England, 51-189, 1975. Shiroza et al., J. BacteoL, 170:810, 1988.<br>
Simmonds, In: Principles of crop improvement, Longman, Inc., NY, 369-399, 1979. Sneep  and   Hendriksen,  In:  Plant  breeding perspectives,  Wageningen  (ed),  Center for<br>
Agricultural Publishing and Documentation, 1979. Sommer et all., Biotechmques, 12(l):82-87, 1992. Stalker etai, Science, 242:419-423, 1988. Stcmmclzetal.,Mol. Gen. Genet., 20:220, 1985. Stoisinetal., Crop Sci., 38:1441-1444, 1998. Vanden Elzen et al., Plant Mol. BioL, 5:299, 1985. Wright et al., Plant Cell Reports, 5:150-154, 1986.<br><br><br><br><br>
We claim:<br>
1.	A method for obtaining a plant of agronomically elite soybean variety having a commercially<br>
significant yield and a mid/low-linolenic comprising the steps of:<br>
a)	crossing first and second soybean plants, wherein the first and second plants collectively comprise a Fad3-lb allele and a Fad3-lc allele each of which confer decreased linolenic acid content, wherein the first plant comprises at least one of the Fad3-lb or Fad3-lc alleles, and wherein the second plant I comprises a commercially significant yield,<br>
b)	assaying progeny soybean plants resulting from the crossing for yield and for the presence of polymorphisms located in a soybean plant genomic region within 50 cM of said Fad3-lb and Fad3-lc alleles; and<br>
c)	selecting at least a first progeny plant comprising said polymorphisms and a commercially significant yield to obtain the plant of claim 1.<br>
2.	The method as claimed in claim 1, wherein the linolenic acid content is further defined as<br>
selected from the group consisting of from about 1.2% to about 3.0% by weight of total seed fatty<br>
acids, from about 1.5% to about 3.0% by weight of total seed fatty acids, from about 2% to about<br>
2.	6% by weight of total seed fatty acids, from about 2% to about 3% by weight of total seed fatty acids, from about 1% to about 2.6% by weight of total seed fatty acids, from about 1% to about 2.2% by weight of total seed fatty acids, from about 1.6% to about 2.6% by weight of total seed fatty acids, and from about 2% to about 2.4% by weight of total seed fatty acids.<br>
3.	A method of producing soybean seed, comprising crossing the plant as claimed in claim 1 with itself or a second soybean plant.<br>
4.	The method as claimed in claim 3 , further defined as a method of preparing hybrid soybean seed,   comprising   crossing   the   plant   of  claim   1   to   a   second,   distinct   soybean   plant.<br>
5.	A method of producing food or feed comprising:<br><br>
(a)	obtaining the plant of claim 1;<br>
(b)	cultivating said plant to maturity; and<br>
(c)	preparing food or feed from said plant.<br>
6.	The method as claimed in claim 5, wherein the food is protein concentrate, protein isolate,<br>
soybean hulls, meal or flour.<br>
7.The method as claimed in claim 5, wherein the food is oil.<br>
8.	The method as claimed in claim 7, wherein the food comprises beverages, infused foods, sauces, condiments, salad dressings, fruit juices, syrups, desserts, icings and fillings, soft frozen products, confections or intermediate food.<br>
9.	A method of obtaining soybean germplasm, comprising the steps of:<br><br>
a)	identifying at least a first polymorphism in a soybean plant genomic region within 50 cM of a Fad3-lb or Fad3-lc allele conferring decreased linolenic acid content;<br>
b)	assaying soybean plants for the presence of the polymorphism; and<br>
c)	selecting at least a first soybean plant comprising the polymorphism.<br><br>
10.	The method as claimed in claim 9, comprising identifying polymorphisms in a soybean plant genomic region within 50 cM of both of said Fad3-lb and Fad3-lc alleles and assaying for the presence of said polymorphisms.<br>
11.	The method as claimed in claim 9, wherein the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lb gene sequence corresponding to nucleotide 2021 of SEQ ID NO: 1.<br>
12.	The method as claimed in claim 9, wherein the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lc gene sequence corresponding to nucleotide 687, 1129, 1203, 2316, 3292, 3360 or 3743 of SEQ ID NO:2.<br>
13.	The method as claimed in claim 9, wherein the first polymorphism comprises a deletion in the Fad3-lc gene sequence.<br>
14.	The method as claimed in claim 9, wherein the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lc promoter corresponding to nucleotide 334,364, 385, 387,393, 729 or 747 of SEQ ID NO:3.<br>
15.	The method as claimed in claim 9, wherein assaying soybean plants for the presence of the first polymorphism comprises PCR, single strand conformational polymorphism analysis, denaturing gradient gel electrophoresis, cleavage fragment length polymorphism analysis and/or DNA sequencing.<br>
16.	A method of plant breeding comprising the steps of:<br><br>
a)	assaying soybean plants for the presence of at least a first polymorphism in a soybean plant genomic region within 50 cM of a Fad3-lb or Fad3-lc allele conferring decreased linolenic acid content,<br>
b)	selecting at least a first soybean plant comprising the polymorphism; and<br>
c)	crossing the first soybean plant to a second soybean plant to produce progeny plants comprising the polymorphism.<br>
17.	The method as claimed in claim 16, further comprising the step of:<br>
d)	selecting a progeny plant comprising the polymorphism and crossing the I progeny<br>
plant with a third soybean plant to produce additional progeny plants.<br>
18.	The method as claimed in claim 17, wherein the second and third plants are of the same<br>
variety.<br>
19.	The method as claimed in claim 17, further comprising repeating step d) about 2-10 times.<br>
20.	The method as claimed in claim 16, comprising assaying soybean plants for the presence of polymorphisms in soybean plant genomic regions within 50 cM of said Fad3-lb and Fad3-lc alleles.<br>
21.	The method as claimed in claim 20, comprising selecting said first soybean plant based on the presence of the polymorphisms.<br>
22.	The method as claimed in claim 16, wherein the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lb gene sequence corresponding to nucleotide 2021 of SEQ NO:1.<br><br>
23.	The method as claimed in claim 16, wherein the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-lc gene sequence corresponding to nucleotide 687, 1129, 1203, 2316,3292, 3360 or 3743 of SEQ ID NO:2. .<br>
24.	The method as claimed in claim 16, wherein the first polymorphism comprises a deletion in the Fad3-lc gene sequence.<br>
25.	The method as claimed in claim 16, wherein the first polymorphism comprises a single nucleotide polymorphism at a position in the Fad3-I c promoter corresponding to nucleotide 334,364,385, 387, 393, 729 or 747 of SEQ ID NO:3.<br>
26.	The method as claimed in claim 16, wherein selecting at least a first soybean plant comprising the polymorphism comprises PCR, single strand conformational polymorphism analysis, denaturing gradient gel electrophoresis, cleavage fragment length polymorphism analysis and/or DNA sequencing.<br>
27.	A probe or primer that hybridizes under stringent conditions to a soybean plant genomic region within 50 cM of a Fad3-lb or Fad3- 7c allele, wherein the probe or primer comprises a nucleic acid sequence selected from SEQ ID NO:4 through SEQ ID NO:98.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LTMyMjctZGVsbnAtMjAwNy1BYnN0cmFjdC0oMDQtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-3227-delnp-2007-Abstract-(04-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LTMyMjctZGVsbnAtMjAwNy1DbGFpbXMtKDA0LTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-3227-delnp-2007-Claims-(04-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LTMyMjctZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDA0LTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-3227-delnp-2007-Correspondence Others-(04-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LTMyMjctZGVsbnAtMjAwNy1Gb3JtLTItKDA0LTAxLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-3227-delnp-2007-Form-2-(04-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LUNsYWltcy0oMjgtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-Claims-(28-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUNsYWltcy0oMzAtMDQtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Claims-(30-04-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Correspondence Others-(09-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Correspondence Others-(15-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Correspondence Others-(16-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-Correspondence Others-(28-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjktMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-Correspondence Others-(29-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzAtMDQtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Correspondence Others-(30-04-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDMtMDctMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-Correspondence-Others-(03-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUZvcm0tMTMtKDMwLTA0LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Form-13-(30-04-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUZvcm0tMTgtKDE1LTA3LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Form-18-(15-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1ERUxOUC0yMDA3LUZvcm0tMy0oMTYtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3227-DELNP-2007-Form-3-(16-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LUZvcm0tMy0oMjktMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-Form-3-(29-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzIyNy1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3227-delnp-2007-pct-220.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260591-isomerization-catalyst-and-process.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260593-formulation-and-supplemented-matrix-for-the-repair-of-bone-fractures.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260592</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3227/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>20/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>16-May-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-May-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Apr-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY LLC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 N.LINDBERGH BLVD,ST.LOUIS, MO 63167 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WU KUNSHENG , MCLAIRD PAUL, BYRUM JOSEPH R., REITER ROBERT AND ERICKSON MARK</td>
											<td>800 NORTH LINDBERG BLVD., ST. LUIS, MO 63167 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/035106</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-09-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/614,331</td>
									<td>2004-09-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260592-a-method-of-obtaining-soybean-germplasm by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:58:13 GMT -->
</html>
