<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260864-a-method-for-determining-an-immortalized-cancer-cell-in-cancer-cells-in-vitro by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:10:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260864:&quot;A METHOD FOR DETERMINING AN IMMORTALIZED CANCER CELL IN CANCER CELLS IN VITRO&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR DETERMINING AN IMMORTALIZED CANCER CELL IN CANCER CELLS IN VITRO&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a gene related to cancer cell immortalization (an immortalization determining gene) and a process that is useful for selective cancer treatment targeting a cancer cell having the gene. The present invention determines an immortalized cancer cell using a polynucleotide having a base sequence of at least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases within any one of abase sequences represented by SEQ ID Nos. 1 to 13. In the foregoing process, the polynucleotide is used as a primer or probe for detecting an immortalization determining gene that exhibits high expression specifically in an immortalized cancer cell.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>GENE INVOLVED IN IMMORTALIZATION OF HUMAN CANCER CELL AND USE THEREOF<br>
TECHNICAL FIELD<br>
[0001]<br>
The present invention relates to a gene (immortalization-determining gene) involved in the immortalization of human cancer cells, and a method useful for a selective cancer therapy targeting an immortalized cancer cell having the gene. More specifically, the present invention relates to a method for discriminating an immortalized cancer cell using an immortalization-determining gene as an index, and a reagent for the discrimination. The present invention further relates to a method for screening an active ingredient of a selective anticancer drug (immortalized cancer cell growth inhibitor) targeting immortalized cancer cells, and the anticancer drug<br>
(immortalized cancer cell growth inhibitor).<br>
BACKGROUND ART [0002]<br>
Patients are given a poor prognosis for human solid cancer unless radical surgery is performed. This mainly derives from the insufficient effect of the present chemotherapy. Since existing chemotherapy drugs basically influence the nucleic acid biosynthesis, DNA, microtubule, etc., as they inhibit the synthesis process from nucleic acid into protein, they inevitably damage, in addition to cancer cells, healthy cells that are in the process of cell growth. Recent research has identified a number of molecules that are specifically expressed in cancer cells, and inhibitors that target these specific molecules can have a minimal adverse reaction to healthy cells. This advantage allows for a greater dosage of the drug. However, in spite of its active development, such a molecule-targeted drug has the following major defect. The molecule-targeted drug does not necessarily promise a certain effect on a majority of cancer cases; the effect can greatly vary for each case. Moreover, it is<br><br>
usually difficult to precisely estimate the effect before the administration. Such a great variation in a molecule-targeted therapeutic drug in each case can derive from the fact that the drug is directed to the molecules of genes involved in carcinogenesis or metastasis. Carcinogenesis varies among different types of cancer and even within the same cancer; therefore, it is assumed that the effect of the targeted molecules on the cancer growth differs in each case. Conversely, the development of a molecule-targeted therapeutic drug that can exhibit a universal effect on many cancers should become possible by identifying a molecule that has a universal and primary effect on various types of cancers. [0003]<br>
In all vertebrates, including humans, each chromosome end contains a simple iteration sequence: "TTAGGG" (about lOkb in humans and several times greater in mice). This terminal sequence cannot be thoroughly reproduced at every DNA duplication upon cell division, and is gradually shortened (on the basis of about 200 bases in humans). This terminal portion is called a "telomere". When a telomere is shortened to a certain length (about 5kb in human noncancerous cells) by repeated cell divisions, the cell division eventually ceases. Cell division again becomes possible by cell canceration, but then, when the curtailment finally comes to the ultimate level (about 2kb), the cell dies. As disclosed in Non-Patent Document 1, reverse transcriptase "telomerase" serves to extend the shortened telomere to stabilize the telomere length, thereby prolonging the cell division limit. Because the telomerase is not expressed in a healthy cell, the cell division in a healthy cell is limited to several tens of times. However, the telomerase is highly expressed in both a productive cell and an immortalized cancer cell. All cancer cells used to be regarded as immortal; however, reports have revealed that some clinically evident cancers did not exhibit telomerase activities (see Non-Patent Documents 2, 3, etc.).   Other reports have also revealed that the compulsory<br>
expression of telomerase enzyme component TERT in a noncancerous cell did not produce a cancer cell (see Non-Patent Document 4) . Accordingly, at present, canceration and immortalization are recognized as different phenomena.<br>
[Non-Patent Document 1] Harley CB, Mutation Res, 256: 271-82, 1991<br>
[Non-Patent Document 2] Kim NW et al., Science 266: 2011-5, 1994<br>
[Non-Patent Document 3] Hiyama K et al., J Natl Cancer Inst 87: 895-902, 1995<br>
[Non-Patent Document 4] Morales CP et al., Nat Genet 21: 115-8, 1999<br>
[Non-Patent Document 5] Shay JW, Bacchetti S, Eur J Cancer 33: 787-91, 1997<br>
[Non-Patent Document 6] Hiyama K et al., J Immunol 155: 3711-5, 1995<br>
[Non-Patent Document 7] Hiyama E et al., Int J Oncol 9: 453-8, 1996<br>
[Non-Patent Document 8] Forsyth NR et al., Aging Cell 2: 235-43, 2003<br>
[Non-Patent Document 9] Bodnar AG et al., Exp Cell Res 228: 58-64, 1996<br>
[Non-Patent Document 10] Hiyama K et al., Int J Oncol 27: 87-95, 2005<br>
DISCLOSURE OF INVENTION Technical Problem [0004]<br>
The present invention relates to a method for discriminating an immortalized cancer cell, and a reagent used for the discrimination. The present invention further relates to a method for screening an active ingredient for an immortalized cancer cell growth inhibitor that is useful as a selective anticancer drug due to its effect of specifically suppressing immortalized cancer cell growth; and the immortalized cancer cell growth inhibitor.<br>
Technical Solution [0005]<br>
In order to develop a molecule-targeted therapeutic drug that is effective for cancer treatment, especially a drug that is expected to have a universal effect on many types of cancers, the inventors of the present invention focus their attention on another character of cancer, "immortalization", that is different from "canceration (malignant alteration)". Specifically, even if cancer cells are found, they will not spread or metastasize to kill the host if their limitless growth can be stopped. The suppression of the limitless growth also increases the possibility of the natural death of cancer cells given that the number of residual cancer cells is reduced by chemotherapy or nonradical surgery to some extent, because it is expected that the cancer cell division term would run out before the next cell growth begins. It has been reported that telomerase expression was seen in 80% or more of several thousands of clinical specimens (see Non-Patent Document 5) ; telomerase expression was confirmed in various types of cancers examined to date; and telomerase expression was seen in most of the cell strains that derive from human cancers (see Non-Patent Document 2). Based on these reports and the known fact that cancer cells containing telomerase expression will never naturally turn negative again, the immortalization phenomenon seems to exist universally and commonly in many cancer cells, in contrast to the wide variety of carcinogenesis. Accordingly, by targeting the molecule that regulates immortalization, it becomes possible to suppress infinite cancer growth with a universal effect on many types of cancers. Based on this principle, an anti-cancer strategy that directly focuses on telomerase has been tested. [0006]<br>
However, the inventors of the present invention had already found that a precursor cell, lymphocyte, etc., of a regenerating organ exhibits telomerase activation even in a normal healthy cell (see Non-Patent Documents 6 and 7).  It is also known that<br>
cell immortalization is not always induced by compulsive expression of TERT (telomerase enzyme component) in noncancerous cells (see Non-Patent Document 8); and that the healthy precursor cell or lymphocyte that expresses telomerase is not immortalized but has a prolonged life (see Non-Patent Documents 6 and 9) . Accordingly, such exhibition of telomerase activation in healthy cells is obviously merely telomerase expression rather than cell immortalization. More specifically, though human cell immortalization substantially requires the activation of a telomerase enzyme that serves to extend the telomere in the chromosome terminus, the immortalization is not induced only by the telomerase activation, but also by the presence of specific molecules for immortalization. The molecules seem to function particularly in cancer cells. In fact, the inventors of the present invention confirmed that a gene that specifically undergoes expression and change in a clone, which has been immortalized by compulsive telomerase expression through the introduction of TERT into healthy cells, completely differs from the reported expression and change seen in cancer cells (see Non-Patent Document 10) . This is another fact indicating that the gene involved in the immortalization of healthy cells differs from the gene involved in the immortalization of cancer cells. If the drug is directed to telomerase, it suppresses the division ability of the lymphocyte, blood precursor cell, gut mucosa crypt cell, etc., and thereby tends to damage immunity, hematogenous ability, and gastrointestinal function, which is known to be a fatal harm in cancer treatment. However, if the drug is directed to a immortalization-determining factor that works only in cancer cells, it becomes possible to block the specific infinite proliferation of cancer cells without damaging the healthy trunk/precursor cells having telomerase activities. [0007]<br>
Based on this theory, the inventors of the present invention conducted an intensive study to discover genes that are universally involved in the immortalization of human cancer cells<br>
As a result of the research, the inventors of the present invention eventually realized that those genes are not involved in the lifetime extension of cells resulting from telomerase activation in healthy cells, but is specifically involved in the constant proliferation of cancer cells. The inventors then identified thirteen immortalization-determining genes (Table 1) in the following manner. First, the genes that are universally expressed and strengthened in various types of cancers were extracted by microarray assay. Among the obtained genes, thirteen kinds of genes were selected on the basis of their expression in a clinical cancer organ consisting of immortalized cancer cells being stronger than the expression of a cancerous tissue comprised of non-immortalized cancer cells and that undergoing no expression or augmentation in healthy cells expressed by the introduction of telomerase. Then, the inventors confirmed that suppressing the expression of the thirteen genes using siRNA suppresses the proliferation of immortalized cancer cells. [0008]<br>
The inventors of the present invention further conducted more detailed research based on the foregoing theory, and eventually completed the present invention. Specifically, the present invention includes the following forms.<br>
Item 1. A marker gene for determining cancer cell immortalization, the marker gene comprising a polynucleotide having a base sequence of at least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases within any one of base sequences represented by SEQ ID Nos. 1 to 13.<br>
[0009]<br>
Item 2.     The marker gene according to Item 1, wherein the marker gene is a probe or primer. [0010]<br>
Item 3. A method for determining an immortalized cancer cell, comprising the steps of:<br>
(1) bonding the marker gene according to claim 1 or 2 with<br>
RNA prepared from a biological sample that is extracted from a test subject and that may contain a cancer cell, or a derivative of the RNA;<br>
(2)	measuring an amount of the RNA or the derivative bonded with the marker gene, using the marker gene as an index; and<br>
(3)	comparing the amount of RNA or the derivative thereof found in Step (2) (generically referred to as "an RNA amount", hereinafter) with an amount (generically referred to as "a comparative RNA amount", hereinafter) of a corresponding RNA or a derivative thereof in a non-immortalized healthy or cancer cell. [0011]<br>
Item 4. The method for determining an immortalized cancer cell according to Item 3, further comprising the step of:<br>
(4)	determining that the cancer cell of the test subject is<br>
immortalized when the RNA amount found in Step (2) is higher than<br>
the comparative RNA amount, and determining that the cancer cell<br>
of the test subject is not immortalized when the RNA amount found<br>
in Step (2) is not higher than the comparative RNA amount.<br>
[0012]<br>
Item 5.     An antibody for recognizing a polypeptide having any<br>
one of amino acid sequences represented by SEQ ID Nos. 14 to 26.<br>
The antibody is suitable for use in determining an immortalized<br>
cancer cell.<br>
[0013]<br>
Item 6.    A method for determining an immortalized cancer cell,<br>
comprising the step of:<br>
(1' ) bonding a protein-containing fraction containing a polypeptide prepared from a biological sample that is extracted from a test subject and that may contain a cancer cell with the antibody according to Item 5;<br>
(2') measuring the amount of the polypeptide that is bonded with the antibody, using the antibody as an index; and<br>
(3' ) comparing the amount (generically referred to as "a polypeptide amount", hereinafter) of the polypeptide found in Step  (2')  with the  amount  (generically referred to as  "a<br>
comparative polypeptide amount", hereinafter) of a corresponding polypeptide in a non-immortalized healthy or cancer cell. [0014]<br>
Item 7. The method for determining an immortalized cancer cell according to Item 6, further comprising the step of:<br>
(4' ) determining that the cancer cell of the test subject is immortalized when the polypeptide amount found in Step (2') is higher than the comparative polypeptide amount, and determining that the cancer cell of the test subject is not immortalized when the polypeptide amount found in Step (2' ) is not higher than the comparative polypeptide amount. [0015]<br>
Item 8.     An immortalized cancer cell determining reagent kit including at least one member selected from the group consisting of the marker gene according to Item 1 and the antibody according to Item 5. [0016]<br>
Item 9. A method for screening a substance for suppressing proliferation of immortalized cancer cells, comprising the steps of:<br>
(A)	bringing a test material into contact with a cell that can express one of the immortalization-determining genes represented by SEQ ID Nos. 1 to 13;<br>
(B)	measuring an expression amount of the immortalization-determining gene in the cell brought into contact with the test material;<br>
(C)	comparing the expression amount of the immortalization-determining gene found in Step (B) with an expression amount<br>
(generically referred to as "a comparative expression amount", hereinafter) of an immortalization-determining gene in a cell not in contact with the test material; and<br>
(D)	selecting the test material as a candidate substance<br>
for suppressing proliferation of immortalized cancer cells when<br>
the expression amount found in Step (B) is lower than the<br>
comparative expression amount.<br>
 [0017]<br>
Item 10.    A method for screening a substance for suppressing<br>
proliferation of immortalized cancer cells, comprising the steps<br>
of:<br>
(A' ) bringing a test material into contact with a polypeptide having one of the amino acid sequences represented by SEQ ID Nos. 14 to 26;<br>
(B' ) measuring activity of the polypeptide having one of the amino acid sequences represented by SEQ ID Nos. 14 to 26 brought into contact with the test material;<br>
(C ) determining the degree of the activity of the polypeptide found in Step (B' ) by comparing the activity of the polypeptide found in Step (B' ) with activity (generically referred to as "a comparative activity", hereinafter) of a polypeptide not in contact with the test material; and<br>
(D' ) selecting the test material as a candidate substance for suppressing proliferation of immortalized cancer cells when the activity of the polypeptide found in Step (B' ) is lower than the comparative activity.<br>
[0018]<br>
Item 11. An immortalized cancer cell growth inhibitor containing, as an active ingredient, a polynucleotide having a base sequence of at least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases within any one of base sequences represented by SEQ ID Nos. 1 to 13.<br>
[0019]<br>
Item 12. The immortalized cancer cell growth inhibitor according to Item 11, wherein the polynucleotide has one of base sequences represented by SEQ ID No. 27 to<br>
[0020]<br>
Item 13. A method of anticancer treatment comprising the step of administering the immortalized cancer cell growth inhibitor according to Item 11 or 12 to cancer cells of a patient.<br>
[0021]<br>
Item 14.    Use of a polynucleotide having a base sequence of at<br>
least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases within one of base sequences represented by SEQ ID Nos. 1 to 13, for producing an immortalized cancer cell growth inhibitor. [0022]<br>
Item 15. The use according to Item 14, wherein the polynucleotide has one of base sequences represented by SEQ ID Nos. 27 toMol.<br>
EFFECT OF THE INVENTION [0023]<br>
The present invention discovered genes that universally determine the immortalization of many different cancer cells. The genes specifically exhibit greater expression in immortalized cancer cells in a plurality of cancers. The present invention provides a method for discriminating immortalized cancer cells, and a reagent that is used for the discrimination method, which allows for measuring the expression level of the immortalization-determining genes. Referring to the measured level, it becomes possible to distinguish the immortalized cancer cells from finite cells. This differentiation was never possible in conventional general pathological diagnoses. The differentiation thus has great potential as a clinical application that is useful for early diagnosis, new treatment options, prognostification, and the like of various cancers. [0024]<br>
Further, with a screening process targeting an immortalization-determining gene of the present invention, an active ingredient of an anticancer drug that specifically suppresses the proliferation of immortalized cancer cells is obtained.<br>
BEST MODE FOR CARRYING OUT THE INVENTION [0025] (I) Immortalization-determining gene<br>
In the present invention, the term "non-immortalized cancer cells" refers to, among general cancer cells, a finite cell having (1) negative telomerase-activation, (2) a reduced telomere length, and (3) undetectable expression of a telomerase protein. Further, in the present invention, the term "immortalized cancer cell" refers to a cell that lacks at least one of the aforementioned conditions; more specifically, a cell having telomerase activation, a cell having an extended telomere length, and/or a cell detectable expression of a telomerase protein. Such an "immortalized cancer cell" is capable of limitless cell proliferation. [0026]<br>
In  the  present  invention,  the  term  "immortalization-determining  gene"  refers  to  a  gene  that  determines  the immortalization of cancer cells. [0027]<br>
The present invention does not particularly limit the type of cancer cell as a target, and includes cells that derive from, for example, lung cancer, breast cancer, esophagus cancer, head and neck cancer, gastric cancer, colon cancer, rectal cancer, liver cancer, gallbladder and bile duct cancer, pancreatic cancer, bladder cancer, prostatic cancer, or uterine cervix carcinoma. [0028]<br>
The  immortalization-determining genes  are,  namely,  (1) UBE2S gene,  (2) RFC4 gene,  (3) PTGES2 gene,  (4) MAF1 gene,  (5) ACVR2B gene,  (6) FAM119A gene,  (7) LTB4DH gene,  (8) DPM2 gene, (9) SEPX1 gene, (10) PSMA3 gene, (11) CHCHD3 gene, (12) LSM3 gene, and (13) GTSE1 gene.  Table 1 shows the GenBank accession number, formal gene name, and gene symbol of the foregoing genes, and the SEQ ID No.s indicating the base sequences of the genes.  In Table 1, the gene name, gene symbol, etc., are based on NCBI data disclosed on the Internet (http://www.ncbi.nlm.nih.gov/).<br>
[0029]<br>
(Table Removed)<br>
 [0030]<br>
As shown in Example 1, the above-listed immortalization-determining genes are all highly expressed in immortalized cancer cells derived from a plurality of cancers, including lung cancer, esophagus cancer, and breast cancer. As shown in Example 2, when the expression of these thirteen genes was suppressed by siRNA in the immortalized cancer cells, in every case, the proliferation of immortalized cancer cells was suppressed. This shows that the thirteen immortalization-determining genes determine the immortalization of cancer cells (immortalization-determining gene).<br>
[0031]<br>
(II) Reagent for judging the immortalization of cancer cells<br>
(marker gene)<br>
As described above, the immortalization-determining genes<br>
((1) to (13)) having base sequences represented by SEQ ID No.s 1 to 13 are all highly expressed specifically in immortalized cancer cells. Therefore, using the thirteen genes as marker genes, the immortality/mortality of cancer cells can be judged. [0032]<br>
The present invention provides an immortalization-determining marker gene (abbreviated as "marker gene" in the present invention) as a tool (reagent) that is suitably used to judge the immortalization of cancer cells. The marker gene is designed as a polynucleotide of at least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases among one of the base sequences of the immortalization-determining genes represented by the SEQ ID No.s 1 to 13. The marker gene of the present invention may have an arbitrary form according to the object, and may be any of single-strand DNA, single-strand RNA, double-strand DNA, double-strand RNA, and DNA:RNA hybrid. [0033]<br>
The marker gene contains a probe or primer useful for detecting the occurrence of expression of an immortalization-determining gene and/or the level (expression level) in cancer cells to allow for the discrimination of immortalized cancer cells.  Further, the probe or primer is also useful as a tool (detection reagent) for detecting the expression variation of an immortalization-determining gene in the screening of a material that suppresses the proliferation of the immortalized cancer cells. [0034] (II-l) Probe<br>
The immortalization of cancer cells is discriminated by detecting at least one of the above-mentioned immortalization-determining genes: (1) UBE2S gene, (2) RFC4 gene, (3) PTGES2 gene, and (4) MAFl gene, (5) ACVR2B gene, (6) FAM119A gene, (7) LTB4DH gene, (8) DPM2 gene, (9) SEPX1 gene, (10) PSMA3 gene, (11) CHCHD3 gene, (12) LSM3 gene, and (13) GTSE1 gene, which are highly expressed specifically in the immortalized cancer cells.<br>
[0035]<br>
The detection of the immortalization-determining genes uses, as probes, polynucleotides, each of which specifically hybridizes with the base sequence of one of the aforementioned immortalization-determining genes. In the present invention, an oligonucleotide having plural base sequences is included in the range of "polynucleotide". [0036]<br>
Each polynucleotide is designed to specifically hybridize with a continuous base sequence of one of the immortalization-determining genes. More specifically, for each of the aforementioned immortalization-determining genes, a corresponding polynucleotide specifically hybridizes with a continuous base of at least 15 bases, preferably 20 bases, and more preferably 30 bases to the total base length of the immortalization-determining gene. Accordingly, each polynucleotide has a base length corresponding to the defined base length. [0037]<br>
Throughout the present specification and claims, "specific hybridization" refers to a hybridization that forms only a specific hybrid and does not form a nonspecific hybrid under a stringent hybridization condition. The stringent hybridization condition is determined in the usual manner, for example, based on the melting temperature (Tm) of the nucleic acid at which the hybrid is formed. A typical cleaning condition to ensure the hybridization condition is approximately "lxSSC, 0.1%SDS, 37°C", more strictly "0.5xSSC, 0.1%SDS, 42°C", and yet more strictly "O.lxSSC, 0.1%SDS, 65°C". [0038]<br>
The polynucleotide (probe) preferably has a base sequence that is complementary to a continuous base sequence of at least 15 bases of the immortalization-determining gene; however, the probe is not always required to be fully complementary to said continuous base sequence as long as the specific hybridization is possible.  The polynucleotide has an identity of at least 70%,<br>
preferably not less than 80%, more preferably not less than 90%, yet more preferably not less than 95%, and still more preferably not less than 98%, with respect to either the polynucleotide having a continuous base of at least 15 bases within a base sequence of the immortalization-determining gene or the complementary polynucleotide thereof. The identity of each base sequence can be calculated by way of an identity search, alignment program, BLAST, FASTA, ClustalW, or the like. [0039]<br>
Examples of probes suitable for the present invention include oligonucleotides and polynucleotides having a continuous base sequence of 15 bases, preferably 20 bases, and more preferably 30 bases to the total base length of the immortalization-determining gene that hybridizes with at least one polynucleotide (however, when the polynucleotide is RNA, the base "t" in the sequence is replaced with "u") selected from the group consisting of the following (1) to (13). [0040]<br>
(1)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.l) of the UBE2S gene.<br>
(2)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.2) of the RFC4 gene.<br>
(3)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.3) of the PTGES2 gene.<br>
(4)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 4) of the MAF1 gene.<br>
(5)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.5) of the ACVR2B gene.<br>
(6)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence  (SEQ ID No. 6)  of the<br>
F7AM119A gene.<br>
(7)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.7) of the LTB4DH gene.<br>
(8)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 8) of the DPM2 gene.<br>
(9)	A polynucleotide with a continuous base sequence having at<br>
least 15 bases from the base sequence (SEQ ID No.9) of the SEPX1<br>
gene.<br>
(10)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.10) of the PSMA3 gene.<br>
(11)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 11) of the CHCHD3 gene.<br>
(12)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 12) of the LSM3 gene.<br>
(13)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.13) of the GTSE1 gene.<br>
[0041]<br>
These polynucleotides (probes) can be produced by a known method, for example, using a commercially available nucleotide synthesizer, based on the target base sequence of the immortalization-determining gene. The polynucleotides (probes) can also be produced by the PCR method using the target base sequence of the immortalization-determining gene as a template. [0042]<br>
More preferably, to simplify detection of the immortalization-determining genes, each probe may be labeled with a radioactive material, a fluorescent material, a chemical luminescent substance or an enzyme (described later in greater detail).<br>
 [0043]<br>
Further, each probe may be immobilized in an arbitrary solid phase before use.<br>
Accordingly, the probes suitable for the present invention include a probe in which an aforementioned polynucleotide is immobilized in a solid phase (for example, an immobilized probe, including a gene chip, a cDNA microarray, an oligo DNA array, or a membrane filter.). This probe is suitable for a DNA chip for use in immortalization-determining gene detection, or more specifically, for use in immortalized cancer cell detection. [0044]<br>
The solid phase of the immobilized probe (polynucleotide) is not particularly limited as long as it immobilizes the polynucleotide. Examples of solid phases include a glass plate, a nylon membrane, micro beads, a silicon chip, a capillary, and other substrates. The polynucleotide can be immobilized in the solid phase by disposing a synthetic polynucleotide in a solid phase, or by synthesizing a target polynucleotide in a solid phase. The immobilization is a well known technology in the present field, and any suitable method can be used according to the type of immobilized probe. For example, a commercially available spotter (manufactured by Cosmobio Co., Ltd., etc.) is often used to form a DNA microarray. (For example, polynucleotide in situ synthesis using photolithographic technology (AFFYMETRIX CO.), or inkjet technology (ROSETTA INPHARMATICS CO.) [0045] (II-2) Primer<br>
The present invention provides a polynucleotide as a primer that serves as a marker gene to specifically amplify a base sequence region of each immortalization-determining gene. [0046]<br>
The polynucleotide is typically a polynucleotide that has a continuous base sequence of at least 15 bases, preferably 15 to 100 bases, more preferably 15 to 50 bases, yet more preferably 15 to 35 bases, and that specifically hybridizes with a part of at<br>
least one  polynucleotide (however, when the polynucleotide is RNA, the base "t" in the sequence is replaced with "u") selected from the group consisting of the following (1) to (13), so as to amplify part or all of the polynucleotide. [0047]<br>
(1)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.l) of the UBE2S gene.<br>
(2)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 2) of the RFC 4 gene.<br>
(3)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.3) of the PTGES2 gene.<br>
(4)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 4) of the MAF1 gene.<br>
(5)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.5) of the ACVR2B gene.<br>
(6)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 6) of the FAM119A gene.<br>
(7)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.7) of the LTB4DH gene.<br>
(8)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 8) of the DPM2 gene.<br>
(9)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 9) of the SEPX1 gene.<br>
(10)	A polynucleotide with a continuous base sequence having at<br>
least 15 bases from the base sequence (SEQ ID No.10) of the PSMA3<br>
gene.<br>
(11)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 11) of the CHCHD3 gene.<br>
(12)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No. 12) of the LSM3 gene.<br>
(13)	A polynucleotide with a continuous base sequence having at least 15 bases from the base sequence (SEQ ID No.13) of the GTSE1 gene.<br>
[0048]<br>
As with the probe, each of the defined polynucleotides (primers) preferably has a base sequence complementary to the continuous base sequence of at least 15 bases in one of the immortalization-determining genes; however, as long as a specific hybridization is possible, it is not required to be totally complementary. The polynucleotide preferably has an identity of at least 70%, preferably not less than 80%, more preferably not less than 90%, yet more preferably not less than 95%, and still more preferably not less than 98%, with respect to either the polynucleotide having a continuous base of at least 15 bases within a sequence of the immortalization-determining gene or the complementary polynucleotide thereof. [0049]<br>
As with the probes, these polynucleotides (primers) can be produced by a known method, for example, using a commercially available nucleotide  synthesizer,  based on the target base sequence of the immortalization-determining gene. [0050] (II-3) Marker<br>
The marker gene (probe or primer) according to the present invention may be a product obtained by adding a marker suitable for detecting an immortalization-determining gene, such as a fluorescent coloring material, an enzyme, a protein, a radioisotope, a chemical luminescent substance, or a biotin, to the aforementioned polynucleotide.<br>
 [0051]<br>
For example, a suitable example of the marker of the present invention made of a fluorescent coloring material, a radioisotope, or a chemical luminescent substance is a general marker capable of labeling a nucleotide to detect or determine the quantity of nucleic acids. Examples of fluorescent coloring materials include, but are not limited to, HEX (4, 7, 2', 4', 5', 7'-hexachloro-6-carboxylfluorescein, green fluorescent coloring material), fluoresceine, NED (product name; Applied Biosystems Inc., yellow fluorescent coloring material), 6-FAM (product name; Applied Biosystems Inc, yellowish green fluorescent coloring material), and rhodamine or its derivative (e.g., tetra methyl rhodamine (TMR)). Fluorescent color labeling on a nucleotide is performed by a suitable method among the known labeling methods (see Nature Biotechnology, 14, 303-308 (1996)). A commercially available fluorescent label kit (for example, oligonucleotide ECL 3'-oligo labeling system produced by Amersham Pharmacia Biotech) can also be used. [0052]<br>
The above-described marker gene (an unlabeled or labeled probe or primer) may be used as an immortalization-determining gene detection reagent, in other words, an immortalized cancer cell detection reagent. [0053] (II-4) Immortalized cancer cell detecting reagent kit<br>
The present invention further provides an immortalized cancer cell detecting reagent (marker gene (an unlabeled or labeled probe or primer)) kit. The kit includes at least one of the above mentioned unlabeled or labeled polynucleotides used as probes or primers (the polynucleotide may be immobilized in a solid phase). In addition to the marker gene (probe or primer), the reagent kit of the present invention may include other reagents or tools as required during the actual performance (described later) of the method of the present invention, such as a hybridization reagent,  a probe labeling material,  a label<br>
detecting agent, a buffer solution etc. [0054]<br>
(III) Determination method for immortalized cancer cells (III-l) The marker gene (probe or primer) of the present invention enables measurement of the expression amount of the immortalization-determining gene (marker gene) m the cancer cells obtained from a test subject; further, the measured expression amount enables judgment as to whether the cancer cells are "immortalized cancer cells" or "non-immortalized cancer cells". Existing pathological diagnoses are incapable of determining immortalized cancer cells; however, the immortalized cancer cell determination method of the present invention is capable of detecting the early-stage of incipient immortalized cancer cells even before they develop, thereby allowing for rapid provision of an appropriate cancer treatment for each individual. [0055]<br>
In this case, the primer and the probe of the present invention are used as a primer for specifically recognizing and amplifying an RNA resulting from the expression of an immortalization-determining gene or a polynucleotide produced from the RNA (e.g., cDNA), and as a probe for specifically detecting an RNA, or a polynucleotide (e.g., cDNA, DNA amplified from RNA or cDNA) that derives from an RNA, respectively. [0056]<br>
More specifically, the marker gene (primer or probe) of the present invention may be used as a primer and/or a probe for specifically detecting an immortalization-determining gene, using a known method for specifically detecting a specific gene, such as the northern blotting method, an RT-PCR method, an in situ hybridization method, or the like. Further, detection and/or quantity determination of the expression amount of the immortalization-determining gene in the cancer cell may be performed using a DNA chip. In this case, the probe of the present invention can be used as a probe for the DNA chip. [0057]<br>
For example, the measurement of the expression amount of the immortalization-determining gene by way of the northern blotting method may be performed with the probe of the present invention, or more preferably, by the probe labeled with a radioisotope, a fluorescent material, or a chemical luminescent substance. More specifically, RNA extracted from a cancer cell of a test subject is transferred to a nylon membrane or the like, and the labeled probe is hybridized with the RNA. Then, the amount of the resulting double-strand of the probe and the RNA is measured based on a signal given by the marker of the probe. The detection of the signal may be carried out by an appropriate known tool according to the marker of the probe, such as a radiation detector, or a fluorescence detector. Detection may also be performed with a commercially available northern blotting reagent kit in accordance with the protocol of the kit. [0058]<br>
Measurement of the immortalization-determining gene expression amount using the RT-PCR method may also be performed using a primer of the present invention, preferably, a primer labeled with a radioisotope, a fluorescent material, a chemical luminescent substance, or the like. More specifically, a pair of marked primers is hybridized with a template cDNA prepared from RNA deriving from a cancer cell of a test subject, and PCR is performed in the general manner. Then, the amount of the resulting amplified double-strand DNA is measured based on the signal given by the marker of the primer. The detection of the signal may be carried out by an appropriate known tool, such as a radiation detector, or a fluorescence detector. The detection may also be performed with a commercially available RT-PCR reagent kit in accordance with the protocol of the kit. [0059]<br>
Measurement of the expression amount of an immortalization-determining gene using a DNA chip assay may also be carried out using a probe of the present invention, preferably, a probe labeled with a radioisotope, a fluorescent material, etc.  More<br>
specifically, a DNA chip, in which a probe labeled with a radioisotope, a fluorescent material, a chemical luminescent substance, or the like, is fixed to an appropriate carrier. Then, the DNA chip is hybridized with a marked DNA or RNA that is prepared from RNA deriving from a cancer cell of a test subject. Then, the amount of the resulting double-strand of the probe and marked DNA or RNA is measured based on the signal given by the marker of the probe. Detection of the signal may be carried out by an appropriate known tool such as a radiation detector or a fluorescence detector. This method may be performed with a commercially available DNA chip in which DNA corresponding to a probe of the present invention is fixed. [0060]<br>
Measurement of the expression amount of immortalization-determining genes includes the following two steps (1) and (2).<br>
(1)	a step of bonding a probe or primer of the present invention<br>
to RNA prepared from a biological sample that may contain a<br>
cancer cell obtained from a test subject, or a derivative of the<br>
RNA; and<br>
(2)	a step of measuring the amount of the RNA or derivative<br>
thereof bonded with the probe or primer, using the probe or<br>
primer as an index.<br>
[0061]<br>
In Step (1) , the biological sample can be any sample extracted from a test subject, as long as it has the potential to contain a cancer cell. Examples of the biological sample include body fluid (blood, urine, etc.), tissues, or the extracts thereof, and cultures of the obtained tissues. Extraction of the biological sample from the test object is performed by an appropriate method according to the type of biological sample or type of cancer. Preparation of the RNA from the biological sample is performed by a known method. [0062]<br>
Throughout the specification and claims, the term "an RNA derivative" refers to a sample produced by RNA extracted from a<br>
biological sample. Examples of an RNA derivative include a complementary polynucleotide (cDNA) prepared by RNA transcription, and DNA amplified from the RNA or cDNA using PCR. The cDNA may be prepared using a known method, and the above-mentioned DNA is prepared by PCR using the primer of the present invention for the immortalization-determining gene. In addition, the probe or primer of the present invention used in Step (1) is preferably labeled with a marker, such as a radioisotope, a fluorescent material, or a chemical luminescent substance. [0063]<br>
The amount of the RNA or the RNA derivative found in Step<br>
(2)	depends on the expression amount of the immortalization-<br>
determining gene in the test subject. Accordingly, this amount of<br>
RNA or RNA derivative (hereinafter comprehensively referred to as<br>
"RNA amount") found m Step (2) is compared with the amount of<br>
the corresponding RNA or RNA derivative (hereinafter<br>
comprehensively referred to as "comparative RNA amount") in non-<br>
immortalized healthy or cancer cells to judge the immortalization<br>
of the cancer cells of the test subject. More specifically, if<br>
the RNA amount of the test subject found in Step (2) is greater<br>
than the comparative RNA amount, it is judged that the cancer<br>
cells of the test subject are immortalized. In the reverse case,<br>
it is judged that the cancer cells of the test subject are not<br>
immortalized.<br>
[0064]<br>
Therefore,  in  addition  to  Steps  (1)  and  (2) ,  the immortalized cancer cell determination method of the present invention further includes the following Step (3), and preferably, also Step (4). [0065]<br>
(3)	a step of comparing the amount of RNA or the RNA derivative<br>
(RNA  amount)  found  in  Step  (2)  with  the  amount  of  the<br>
corresponding RNA or RNA derivative (comparative RNA amount) in a<br>
non-immortalized healthy or cancer cell; and<br>
(4)	a step of judging that the extracted cancer cell of the test<br>
subject is immortalized when the RNA amount in Step (2) is higher than the comparative RNA amount, and judging that the extracted cancer cell of the test subject is not immortalized when the RNA amount in Step (2) is not higher than the comparative RNA amount. [0066]<br>
The RNA amount (comparative RNA amount) of the immortalization-determining gene in a non-immortalized healthy or cancer cell is calculated by first measuring the RNA amount of the immortalization-determining gene in a plurality of non-immortalized healthy or cancer cells under the same condition, and then finding a mean or intermediate value of the calculated values. [0067]<br>
(III-2) Further, the immortalization of cancer cells can be judged based on the quantity of the expression product of the immortalization-determining gene of the present invention, in other words, based on the production quantity of a polypeptide (immortalization-determining polypeptide, hereinafter) encoded by the aforementioned gene. The production quantity of the immortalization-determining polypeptide can be measured using an antibody that recognizes the polypeptide. [0068]<br>
The following are immortalization-determining polypeptides that can be used for the present invention that correspond to the aforementioned immortalization-determining genes (1) to (13). [0069]<br>
(14)	UBE2S polypeptide: a polypeptide encoded by a UBE2S gene (1) having a base sequence represented by the SEQ ID No.l, namely, a polypeptide having an amino acid sequence represented by the SEQ ID No.14.<br>
(15)	RFC4 polypeptide: a polypeptide encoded by a RFC4 gene (2) having a base sequence represented by the SEQ ID No.2, namely, a polypeptide having an amino acid sequence represented by the SEQ ID No.15.<br>
(16)	PTGES2 polypeptide: a polypeptide encoded by a PTGES2 gene<br>
(3) having a base sequence represented by the SEQ ID No.3, namely, a polypeptide having an amino acid sequence represented by the SEQ ID No.16.<br>
(17)	MAF1 polypeptide: a polypeptide encoded by a MAF1 gene (4)<br>
having a base sequence represented by the SEQ ID No.4, namely, a<br>
polypeptide having an ammo acid sequence represented by the SEQ<br>
ID No.17.<br>
(18)	ACVR2B polypeptide: a polypeptide encoded by an ACVR2B gene<br>
(5)	having a base sequence represented by the SEQ ID No.5, namely,<br>
a polypeptide having an ammo acid sequence represented by the<br>
SEQ ID No.18.<br>
(19)	FAM119A polypeptide: a polypeptide encoded by a FAM119A gene<br>
(6)	having a base sequence represented by the SEQ ID No.6, namely,<br>
a polypeptide having an amino acid sequence represented by the<br>
SEQ ID No.19.<br>
(20)	LTB4DH polypeptide: a polypeptide encoded by a LTB4DH gene<br>
(7)	having a base sequence represented by the SEQ ID No.7, namely,<br>
a polypeptide having an amino acid sequence represented by the<br>
SEQ ID No.20.<br>
(21)	DPM2 polypeptide: a polypeptide encoded by a DPM2 gene (8) having a base sequence represented by the SEQ ID No. 8, namely, a polypeptide having an amino acid sequence represented by the SEQ ID No.21.<br>
(22)	SEPX1 polypeptide: a polypeptide encoded by a SEPX1 gene (9) having a base sequence represented by the SEQ ID No.9, namely, a polypeptide having an amino acid sequence represented by the SEQ ID No.22.<br>
(23)	PSMA3 polypeptide: a polypeptide encoded by a PSMA3 gene<br>
(10)	having a base sequence represented by the SEQ ID No. 10,<br>
namely, a polypeptide having an amino acid sequence represented<br>
by the SEQ ID No.23.<br>
(24)	CHCHD3 polypeptide: a polypeptide encoded by a CHCHD3 gene<br>
(11)	having a base sequence represented by the SEQ ID No. 11,<br>
namely, a polypeptide having an amino acid sequence represented<br>
by the SEQ ID No.24.<br>
(25)	LSM3 polypeptide: a polypeptide encoded by a LSM3 gene (12) having a base sequence represented by the SEQ ID No.12, namely, a polypeptide having an amino acid sequence represented by the SEQ ID No.25.<br>
(26)	GTSEl polypeptide: a polypeptide encoded by a GTSE1 gene (13) having a base sequence represented by the SEQ ID No. 13,<br>
namely, a polypeptide having an amino acid sequence represented<br>
by the SEQ ID No.26.<br>
[0070]<br>
The (14) UBE2S polypeptide is a publicly known polypeptide, and<br>
its production is also known, as disclosed in J. Biol. Chem. 267<br>
(22),  15829-15835  (1992).   The  (15)  RFC4 polypeptide is also<br>
publicly known as disclosed in Proc. Natl. Acad. Sci. U.S.A. 89<br>
(12), 5211-5215 (1992).  Similarly, the polypeptides are publicly<br>
known polypeptides, as the (16) PTGES2 polypeptide is disclosed<br>
in Biochem. Biophys. Res. Commun. 291 (4), 884-889 (2002); the<br>
(18) ACVR2B polypeptide is disclosed in Mol. Cell Biol. 16 (3),<br>
1066-1073 (1996); the (20) LTB4DH polypeptide is disclosed in J.<br>
Biol. Chem. 271 (5), 2844-2850 (1996); the (21) DPM2 polypeptide<br>
is disclosed in EMBO J. 19 (11), 2475-2482 (2000); the (22) SEPX1<br>
polypeptide is disclosed in J. Biol. Chem. 274 (48), 33888-33897<br>
(1999); the  (23)  PSMA3 polypeptide is disclosed in Biochem.<br>
Biophys. Res. Commun. 207  (1), 318-323 (1995); the (25) LSM3<br>
polypeptide is disclosed m EMBO J. 18 (12), 3451-3462 (1999);<br>
and the (26) GTSEl polypeptide is disclosed in Gene 254 (1-2),<br>
229-236 (2000).<br>
[0071]<br>
The immortalization-determining polypeptides (14) to (26) may also be prepared by a process comprising cloning the corresponding immortalization-determining genes (1) to (13), ligating the genes into a vector plasmid, transforming the genes into host cells, such as E.coli, culturing the transformed cells, and isolating the polypeptides from the culture. [0072]<br>
The antibody used in the present invention is not limited<br>
as long as it recognizes the immortalization-determining polypeptide, and may be a monoclonal antibody or a polyclonal antibody. The antibody may be an antibody prepared as an antibody against the immortalization-determining polypeptide as an immunizing antigen, or an antigen binding antibody with respect to a polypeptide having at least continuous 8 amino acids, preferably 15 amino acids, and more preferably 20 amino acids out of the amino acid sequences forming the immortalization-determining polypeptide. The polypeptide can be generally synthesized by a publicly known method based on the amino acid sequence of the immortalization-determining polypeptide of the present invention or the base sequence that encodes the amino acid. For example, chemical synthesis using an amino acid synthesizer or gene industrial synthesis can be employed. [0073]<br>
The antibody of the present invention can be produced by a general method (see, for example, Current protocols in Molecular Biology, edit. Ausubel et al. (1987), Publish. John Wiley and Sons. Section 11.12-11.13). For example, a polyclonal antibody can be produced by using a polypeptide obtained by expressing a polypeptide in E.coli or a synthetic polypeptide that is given the amino acid sequence portions by a general method, immunizing a test animal with the aforementioned polypeptid, and extracting the antibody from a serum obtained from an immune animal. A monoclonal antibody can also be produced from the polypeptide expressed by E.coli or a synthetic polypeptide that is given the amino acid sequence portions by a general method, immunizing a test animal with the aforementioned polypeptide, fusing the spleen cells and myeloma cells extracted from the test animal to synthesize a hybridoma cell, and extracting the antibody from the hybridoma cell. [0074]<br>
These antibodies are also included as a component of the aforementioned cancer cell immortalization determining reagent kit for use in judging the immortalization of cancer cells.<br>
 [0075]<br>
Specifically, judgment of immortalized cancer cells is carried out with the following Step (1') and Step (2'). (1') a step of mixing a protam-containing fraction containing a polypeptide prepared from a biological sample that has the potential to contain cancer cells obtained from a test subject, with an antibody that recognizes the immortalization-determining polypeptide of the present invention; and<br>
(2' ) a step of measuring the amount of polypeptide bonded with the antibody, using the antibody as an index.<br>
The amount of polypeptide found in Step (2' ) depends on the production   quantity   of   the   immortalization-determining polypeptide. [0076]<br>
In Step (2' ) , the biological sample can be any sample extracted from the test subject, as long as it has the potential to contain cancer cells. Examples of biological samples include body fluid (blood, urine, etc.), tissues, or extracts thereof, and cultures of the obtained tissues. Extraction of the biological sample from the test subject is performed by an appropriate method according to the type of biological sample or type of cancer. Preparation of the protein-containing fraction (including polypeptide) from the biological sample is performed by an appropriate combination of known differentiation and purification methods. The immortalization-determining polypeptide can be detected by way of various immunological detection methods, such as western blotting, enzyme immunoassay (EIA), radioisotope immunoassay (RIA), luminescence immunoassay, or the like, using the antibody as a probe. [0077]<br>
The following explains Steps (1') and (2') for measuring the amount of polypeptide (production quantity of the immortalization-determining polypeptide) by way of western blotting. First, an antibody against the immortalization-determining polypeptide is mixed as a primary antibody with a<br>
protein-containing fraction prepared from a biological sample of the test subject to be bonded with the immortalization-determining polypeptide contained in the protein-containing fraction. Next, a secondary antibody labeled with a radioisotope, a fluorescent material, or a chemical luminescent substance is bonded with the primary antibody. Then, the amount of immortalization-determining polypeptide is measured based on the signal that derives from the marker of the secondary antibody. Detection of the signal can be carried out by an appropriate known method using, for example, a radiation detector, or a fluorescence detector. [0078]<br>
The immortalization of the cancer cells of the test object<br>
can be judged by comparing the obtained amount of polypeptide<br>
(production   quantity   of   the   immortalization-determining<br>
polypeptide) in the test subject in Step (2') with the amount of<br>
comparative immortalization-determining polypeptide (comparative<br>
production quantity) in a non-immortalized healthy or cancer cell,<br>
and finding the difference level.  More specifically, if the<br>
obtained amount of polypeptide  (production quantity of the<br>
immortalization-determining polypeptide) in the test subject in<br>
Step (2' ) is greater than the amount of comparative polypeptide<br>
(comparative production quantity), it is judged that the cancer<br>
cells of the test subject are immortalized.  In the reverse case,<br>
it is judged that the cancer cells of the test subject are not<br>
immortalized.<br>
[0079]<br>
Accordingly, in addition to the aforementioned Steps (1') and (2'), the immortalized cancer cell determination method of the present invention includes the following Step  (3' ) ,  and preferably, also Step (4').<br>
(3' ) a step of comparing the amount of polypeptide found in Step (2') with the amount (comparative polypeptide amount) of the corresponding polypeptide in a non-immortalized healthy or cancer cell; and<br>
 (4') a step of judging that the extracted cancer cell of the test subject is immortalized when the amount of polypeptide found in Step (2' ) is higher than the comparative polypeptide amount, and judging that the extracted cancer cell of the test subject is not immortalized when the polypeptide amount is not higher than the comparative polypeptide amount. [0080]<br>
The production quantity of the immortalization-determining polypeptide in a non-immortalized healthy or cancer cell is calculated by first measuring the production quantity of the immortalization-determining polypeptide in a plurality of non-immortalized healthy or cancer cells under the same condition, and then finding a mean or intermediate value of the calculated values. [0081]<br>
(IV) A method for screening immortalized cancer cell growth inhibitor<br>
(IV-1) As shown in the Examples, the immortalization-determining genes ( (1) to (13)) according to the present invention exhibit high expression specifically in immortalized cancer cells. Further, by suppressing the expression using siRNA of the immortalization-determining genes, proliferation of the immortalized cancer cells is suppressed. This indicates that the substance that suppresses the expression of the immortalization-determining genes ((1) to (13)) can serve as an immortalized cancer cell growth inhibitor (anticancer drug). Therefore, by using the expression suppression of the immortalization-determining genes ( (1) to (13) ) as an index, the substance that suppresses proliferation of the immortalized cancer cells can be screened. [0082]<br>
Accordingly, the present invention provides a method for screening immortalized cancer cell growth inhibitors using the expression suppression of the immortalization-determining genes ( (1) to (13)) as an index.<br>
[0083]<br>
The present method may include the following Steps (A) to (D) .<br>
(A)	a step of bringing a test material into contact with a cell<br>
that can express one of the immortalization-determining genes (1)<br>
to (13);<br>
(B)	a step of measuring the expression amount of the immortalization-determining gene in the cell brought into contact with the test material;<br>
(C)	a step of comparing the expression amount of the immortalization-determining gene found in Step (B) with the expression amount (comparative expression amount) of the immortalization-determining gene in a cell not in contact with the test material; and<br>
(D)	a step of selecting the test material as a substance for<br>
inhibiting the proliferation of the immortalized cancer cells<br>
when the expression amount found in Step (B) is lower than the<br>
comparative expression amount.<br>
[0084]<br>
The cell used in the present invention can derive from any kind of cell as long as the cell can express at least one of the immortalization-determining genes (1) to (13). Examples of organs from which the cell used may derive include lung, stomach, colon, rectum, liver, gallbladder, bile duct, pancreas, kidney, bladder, prostate gland, womb, marrow, lymph gland and blood. The cell may derive from mammals or birds, preferably mammals, and more preferably humans. The expression of the immortalization-determining gene may be endogenous or exogenous. [0085]<br>
The test material is not limited and may be nucleic acid (including polynucleotides), a peptide (including polypeptides), an organic compound, or an inorganic compound. The screening is carried out by bringing a test material or a sample containing the test material (test sample) into contact with a cell that can express the immortalization-determining gene.  Examples of the<br>
test sample include cell extracts, expression products from a gene library, and extracts of natural vegetable or animal-origin products. [0086]<br>
In the screening, the process of bringing the test material into contact with the cell is not limited as long as it does not kill the cell and does not  inhibit the expression of the immortalization-determining gene in the cell. [0087]<br>
In Steps (B) and (C), the expression amount of the immortalization-determining gene can be found by measuring the amount of mRNA of the immortalization-determining gene or its derivative cDNA or double-strand DNA by northern blotting or RT-PCR, using the aforementioned probe or primer of the present invention. Further, the amount of the immortalization-determining polypeptide as an expression product of the immortalization-determining gene may be used to find the expression amount of the immortalization-determining gene in Steps (B) and (C). The amount of the immortalization-determining polypeptide can be found through various immunological detection methods, such as western blotting, enzyme immunoassay (EIA), radioisotope immunoassay (RIA), luminescence immunoassay, or the like, using an antibody against the immortalization-determining polypeptide. [0088]<br>
In Step (C) , the substance that inhibits proliferation of the immortalized cancer cells can be screened by comparing the expression amount of the immortalization-determining gene measured in Step (B) with the expression amount of the immortalization-determining gene (comparative expression amount) in a cell not in contact with the test material. More specifically, when the expression amount of the immortalization-determining gene found in Step (B) is lower than the comparative expression amount, the test material can be selected as a substance for inhibiting the proliferation of the immortalized cancer cell.<br>
[0089]<br>
(IV-2) Activity inhibition of the immortalization-determining polypeptides ((14) to (26)), which are the expression products of the immortalization-determining genes ((1) to (13)) of the present invention also constitutes an index to screen the immortalized cancer cell growth inhibitor. For example, the (14) UBE2S polypeptide is known to have a ubiquitm activator (E2) (J. Biol. Chem. 267 (22), 15829-15835 (1992)); The (16) PTGES2 polypeptide is known to have an activity to convert prostaglandin H2 to prostaglandin E2 (Biochem. Biophys. Res. Commun. 291 (4), 884-889 (2002)); the (18) ACVR2B polypeptide is a transmembrane receptor, and has a Ser/Thr kinase activity in the intercellular domain (Mol. Cell Biol. 16 (3), 1066-1073 (1996)); the (20) LTB4DH polypeptide has an activity to convert leukotriene B4 to 12-oxo-leukotriene B4 (J. Biol. Chem. 271 (5), 2844-2850 (1996)); and the (22) SEPX1 polypeptide has a methionine sulfoxide reducing activity (Mol. Biol. Cell 15 (3), 1055-1064 (2004)); therefore, inhibitions of these activities of the immortalization-determining polypeptides is particularly useful as indices to screen the immortalized cancer cell growth inhibitor. [0090]<br>
More specifically, by carrying out the following Steps (A') to (D'), a substance for suppressing the proliferation of the immortalized cancer cells can be screened.<br>
(A' ) a step of bringing a test material into contact with one of the immortalization-determining polypeptides (14) to (26); (B' ) a step of measuring the activity of the immortalization-determining polypeptide in contact with the test material; (C ) a step of comparing the activity of the immortalization-determining polypeptide measured in Step (B' ) with the activity (comparative  activity)   of  the   immortalization-determining polypeptide in an aqueous solution, a cell, or a cell fraction that is not in contact with the test material; and (D' ) a step of selecting the test material as a substance for<br>
inhibiting the proliferation of the immortalized cancer cells when  the  activity  found  in  Step  (B' )  is  lower  than  the comparative activity. [0091]<br>
The screening uses the activity of the immortalization-determining polypeptide as an index, and may be carried out with respect to any object that contains or may contain one of the immortalization-determining polypeptides (14) to (26) (preferably (14), (16), (18), (20) or (22)). An appropriate object is selected according to the desired function (activity) of the immortalization-determining polypeptide. Examples of objects subjected to screening include an aqueous solution containing one of the immortalization-determining polypeptides (14) to (26) (preferably (14), (16), (18), (20) or (22)); a cell that can express one of the immortalization-determining genes (1) to (13) (preferably (1), (3), (5), (7) or (9)); or a cell fraction prepared from the aforementioned cell. The aqueous solution is not particularly limited as long as it contains an immortalization-determining polypeptide. In addition to a general aqueous solution, a cell solution, a nuclear extraction, or a liquid culture supernatant may be used. The cell may be endogenous or exogenous, and may be any cell that can express one of the immortalization-determining genes. The cell fraction designates a fraction that derives from the aforementioned cell, such as a cell membrane fraction, a cytoplasmic fraction, a cell nucleus fraction, or the like. [0092]<br>
The same cell and test material as those for the previously described screening can be used for this screening process. [0093]<br>
Measurement of the ubiquitin activation capability of the<br>
UBE2S polypeptide (14) in Step (B' ) can be carried out by a known<br>
method.   For example,  "J.  Biol.  Chem.  267  (22),  15829-15835<br>
(1992)" discloses a method of performing ubiquitin activation<br>
reaction  of  the  UBE2S  polypeptide  in  a  system  containing<br>
ubiquitin, a ubiquitin activating enzyme (El), and UBE2S polypeptide, and measuring the amount of ubiquitin bonded to the UBE2S polypeptide. Measurement of the ubiquitin amount may be performed by a known method. For example, the reaction is performed using ubiquitin labeled with a radioactive material or enzyme, and the amount of the marker is measured (J. Biol. Chem. 267 (22), 15829-15835 (1992), J. Biol. Chem. 279 (51), 52970-52977 (2004)). Alternatively, measurement can be performed by detecting ubiquitin using an anti-ubiquitin antibody (J. Biol. Chem. 279 (51), 52970-52977 (2004)). The ubiquitin and ubiquitin activating enzyme (El) may be a marketed commodity, or may be obtained by a known protein synthesis process. [0094]<br>
Measurement of the prostaglandin E2 synthesis activity of the PTGES2 polypeptide (16) in Step (B') may be performed using a known method. For example, as disclosed in "Biochem. Biophys. Res. Commun. 291 (4), 884-889 (2002)", the production amount of the prostaglandin E2 can be measured by performing a prostaglandin E2 conversion reaction using PTGES2 polypeptide in a system containing PTGES2 polypeptide and prostaglandin H2. The amount of the prostaglandin E2 can be measured using a known method. For example, according to "Proc. Natl. Acad. Sci. U.S.A. 96 (13), 7220-7225 (1999))", the reaction is performed using prostaglandin H2 labeled with a radioactive material or enzyme, and then the amount of the marker is measured to find the production quantity of the prostaglandin E2. Alternatively, the measurement can be performed by using an anti-prostaglandin E2 antibody (J. Dairy. Sci. 87 (7), 2197-2210 (2004)). The prostaglandin H2 may be a marketed commodity, or may be obtained by a known synthesis process. [0095]<br>
Measurement of the Ser/Thr kinase activity of the ACVR2B polypeptide (18) in Step (B') may be performed using a known method. For example, as disclosed in "Mol. Cell Biol. 16 (3), 1066-1073 (1996)", the Ser/Thr kinase activity can be measured by<br>
performing a kinase reaction in a system containing ACVR2B polypeptide in the presence of a substrate and ATP, and then measuring the phosphorylation quantity of the substrate. The phosphorylation quantity of the substrate may be measured by a known method. For example, the phosphorylation quantity may be measured by measuring radioactivity using Y32p_ATP or Y33P-ATP AS A tracer (Mol. Cell Biol. 16 (3), 1066-1073 (1996)). Alternatively, the phosphorylation quantity may be measured by performing a phosphorylation reaction using a substrate labeled with a fluorescent material or biotin, and measuring the quantity of the phosphorylation marker. Examples of substrates include ACVR2B polypeptide, and specific substrates and peptides. [0096]<br>
Measurement of the leukotriene B4 conversion activity of the LTB4DH polypeptide (20) in Step (B') may be performed using a known method. For example, as disclosed in "J. Biol. Chem. 271 (5), 2844-2850 (1996)", the leukotriene B4 conversion activity can be measured by performing a leukotriene B4 conversion reaction using LTB4DH polypeptide in a system containing LTB4DH polypeptide and leukotriene B4, and then measuring the production quantity of the 12-oxo-leukotriene B4. The production quantity of the 12-oxo-leukotriene B4 may be measured by a known method, such as HPLC. The leukotriene B4 may be a marketed commodity, or may be obtained by a known chemical synthesis process. [0097]<br>
Measurement of the methionine sulfoxide reducing activity of the SEPX1 polypeptide (22) in Step (B') may be performed using a known method. For example, as disclosed in "Mol. Biol. Cell 15 (3), 1055-1064 (2004)", the methionine sulfoxide reducing activity can be measured by performing a methionine sulfoxide reducing reaction using SEPX1 polypeptide in a system containing SEPX1 polypeptide and methionine sulfoxide, and then measuring the production quantity of the methionine. The production quantity of the methionine may be measured by a known method, such as HPLC.   The methionine sulfoxide may be a marketed<br>
commodity,  or may be obtained by a known chemical synthesis process. [0098]<br>
In Steps (C ) and (D' ) , the substance that inhibits the proliferation of the immortalized cancer cells can be screened by comparing the measured activity of the immortalization-determining polypeptide in Step (B' ) with the comparative activity. More specifically, when the measured activity of the immortalization-determining polypeptide is lower than the comparative activity, the test material is determined to be a substance for inhibiting the proliferation of the immortalized cancer cell. [0099]<br>
As explained in the Examples, each of the antisence polynucleotides (siRNA) (SEQ ID No.s 27 to 76 shown in Tables 2 as hybridizes with one of the immortalization-determining genes of the present invention in a cancer cell to suppress the expression of the immortalization-determining gene, thereby suppressing the proliferation of the immortalized cancer cells. Therefore, the antisence polynucleotide (siRNA) can be used as an immortalized cancer cell growth inhibitor. Accordingly, the present invention provides an immortalized cancer cell growth inhibitor containing the antisence polynucleotide as an active ingredient. [0100]<br>
The antisence polynucleotide is formed of a polynucleotide of at least 15 bases that specifically hybridizes with a polynucleotide having a continuous base sequence of at least 15 bases within a base sequence of the immortalization-determining genes (1) to (13) . The polynucleotide preferably has a base sequence complementary to the continuous base sequence of at least 15 bases in one of the immortalization-determining genes (1) to (13) ; however, as long as a specific hybridization is possible and thereby the expression of the immortalization-determining gene is suppressed, it is not required to be totally complementary.    For  example,  the  base  sequence  of  the<br>
polynucleotide may have an identity of 70%, preferably not less than 80%, more preferably not less than 90%, yet more preferably not less than 95%, and still more preferably not less than 98%, with respect to the complementary sequence of the immortalization-determining gene; and the polynucleotide may hybridize with RNA of the immortalization-determining gene to suppress the expression of the immortalization-determining gene. The antisence polynucleotide of the present invention preferably has 15 to 1,000 bases, more preferably 15 to 500 bases, and yet more preferably 16 to 30 bases. [0101]<br>
The antisence polynucleotide of the present  invention preferably has a base sequence represented by one of SEQ ID No.s 27 to     and more preferably a double-strand RNA having one of the base sequences. [0102]<br>
The antisence polynucleotide of the present invention may be modified with a known technique to improve stability and cell permeability. For example, a phosphoric residue of each nucleotide may be substituted with a chemically modified phosphoric residue, such as phosphorothioate, methyl phosphonate, or phosphorodithioate, to prevent decomposition by a hydrolase such as nuclease. [0103]<br>
The form of the antisence polynucleotide of the present invention is not limited and may be single-strand DNA, single-strand RNA,  double-strand DNA,  double-strand RNA or DNA:RNA hybrid. [0104]<br>
Generally,  the antisence polynucleotide of the present invention may be synthesized using a known method,  such as chemical synthesis using a synthesizer. [0105]<br>
The antisence polynucleotide of the present invention is introduceable into cancer cells of a patient using a virus vector,<br>
such as a retrovirus vector, adenovirus vector, or adeno-associated virus vector, or a non-virus vector such as liposome, through an ex vivo process or an in vivo process, thereby suitably serving as an anticancer treatment. Accordingly, the present invention includes an anticancer treatment method comprising the step of administering an immortalized cancer cell growth inhibitor containing the antisence polynucleotide as an active ingredient to a patient. [0106]<br>
The immortalized cancer cell growth inhibitor of the present invention contains the aforementioned antisence polynucleotide as an active ingredient, and may also contain a stabilizer, a suspension, a freezing mixture, a buffer solution, a solvent, or the like insofar as the cell proliferation suppression effect of the inhibitor is not impaired. The dosage and administration schedule of the immortalized cancer cell growth inhibitor of the present invention to a cancer patient can be suitably determined by a person having skill in the art according to the type of disease, patient's age, weight, etc. Examples of administration schedules include a 3-week cycle of intravenous administration in which a consecutive 2 to 3-week administration of the antisence polynucleotide is carried out with a dosage of 2 to 10 mg/kg per day; a 1-week cycle of intravenous administration in which consecutive 5-day administration of the antisence polynucleotide is carried out with a dosage of 80 to 120 mg/m2 per day; and a 4-week cycle of intravenous administration in which a consecutive 3-week administration of the antisence polynucleotide is carried out with a dosage of 80 to 120 mg/m2 per day.<br>
EXAMPLE [0107]<br>
The present invention is more specifically explained below in reference to examples. The present invention is, however, not limited to those examples.<br><br>
 [0108] Example 1<br>
Identification of immortalization-determining gene<br>
1.	Preparation of total RNA from human tissue sample, human<br>
cancer cell strain, and human nonneoplastic culture cell<br>
Using an RNeasy™ Mini kit (Qiagen) , total RNAs were extracted from nine kinds of lung cancer cell strains, 21 kinds of esophagus cancer cell strains, nine kinds of digestive system cancers, various other cancer cell strains (gastric cancer, colon cancer, head neck cancer, leukemia), two kinds of nonneoplastic esophagus epithelial cells, and two kinds of normal respiratory epithelia, according to the attached protocol. The extracted total RNAs were preserved at -80°C.<br>
[0109]<br>
Further, total RNAs were also extracted from eleven kinds of breast cancer cell strains, ten kinds of ovary cancer cell strains, ten kinds of pancreas cancer cell strains, one kind of nonneoplastic immortalization mammary gland epithelia, one kind of nonneoplastic mammary gland epithelia, ten sets of pancreatic cancer tissue and non-cancer pancreas tissue of the same pancreatic cancer patient, the primary focus and three metastasis portions of lung cancer tissue of the same non-small cell lung cancer patient. The obtained total RNAs were preserved. The total RNAs were examined to ensure high quality, with clear 18S and 28S rRNA peaks using a 2100 Bioanalyzer (Agilent Technologies) and RNA LabChip (Agilent Technologies) , before they were subjected to a microarray assay.<br>
[0110]<br>
2.	Microarray assay<br>
Using the extracted total RNAs, an exhaustive gene expression assay was carried out by way of a microarray. The microarray used a CodeLink UniSet Human 20K I Bioarray (19881 probe), and comprised (1) a synthesis of cDNA from total RNA, (2) a synthesis of labeled cRNA from the cDNA, (3) a fragmentation process of the labeled cRNA, (4) a hybridization of fragment cRNA<br><br>
and microarray, (5) staining of a microarray, (6) scanning of a microarray, and (7) a gene expression assay. [0111] (1) Synthesis of cDNA from total RNA<br>
Using a CodeLink Expression Assay Reagent Kit (GE Healthcare Bio Science), a first-strand cDNA was synthesized according to the protocol. More specifically, 1 pg each of the total RNAs obtained in section 1 were adjusted to 10 ul with nuclease-free water, mixed with 1 ul of a bacterial control mRNA diluent solution and 1 pi of T7 oligo (dT) primer, both contained in the kit. The total 12 pi of liguid was heated at 70°C for 10 minutes, followed by 3 minutes rapid cooling on ice. On the ice, the liquid was mixed with 2 pi of 10 * first-strand buffer, 4 pi of 5-mM dNTP Mix, 1 pi of RNase inhibitor, and 1 pi of reverse transcriptase (200 U/pl), all contained in the kit. The total 20 pi of liquid was heated at 42°C for two hours. [0112]<br>
Subsequently, according to the protocol, the RNA in the RNA-DNA hybrid was decomposed, and the RNA strand was substituted with a DNA strand, thereby synthesizing a second-strand cDNA (double-strand cDNA). More specifically, 63 pi of nuclease-free water, 10 * second-strand buffer (10 pi), 5-mM dNTP mix (4 pi), DNA polymerase mix (2 pi:10 U/pl), 1 pi of RNase H, all contained in the kit, were added to the 20 pi of first-strand cDNA reaction solution. The total 100 pi of liquid was heated at 16°C for two hours. [0113]<br>
After the reaction, the synthesized double-strand cDNA was purified using a QIA quick PCR purification Kit (QIAGEN) according to the attached protocol. More specifically, 100 pi of the cDNA solution was mixed with 500 µl of buffer PB contained in the kit, and the mixed liquid was placed in a QIA quick spin column set in a 2-ml centrifugal tube and subjected to centrifugation for 50 seconds at 13,000 rpm. After the elution liquid was removed, the QIA quick spin column was set again, 700<br>
µl of buffer PE contained in the kit was added thereto, and 13,000 rpm centrifugation was performed for 1 minute. The elution liquid was removed, the QIA quick spin column was set again, and still another centrifugation was performed at 13,000 rpm for 1 minute so as to completely remove the buffer PE. The QIA quick spin column was set in a new 1.5-ml tube, and 30 ul of nuclease-free water was directly added onto the membrane in the QIA quick spin column, allowed to stand for 1 minute, and followed by 13,000-rpm centrifugation for 1 minute. This process was performed twice (60 µl was used in total). Then, the cDNA solution was concentrated to less than 9.5 µl using a vacuum dryer, and adjusted by adding nuclease-free water to 9.5 µl. [0114] (2) Synthesis of labeled-cRNA from cDNA<br>
Subsequently, according to the protocol of a CodeLmk Expression Assay Reagent Kit (GE Healthcare Bio Science), an in vitro transcription (IVT) reaction was performed in the presence of a biotin-labeled nucleotide to synthesize a cRNA. More specifically, 4.0 µl of a 10 x T7 reaction buffer, 4.0 µl of a T7 ATP solution, 4.0 µl of a T7 GTP solution, 4.0 pi of a T7 CTP solution, and 3.0 µl of a T7 UTP solution, all contained in the kit, were mixed. After further adding 7.5 µl of 10-mM biotin-11-UTP (Perkin Elmer Corporation), the total 26.5 µl of liquid was added to the 9.5 µl of cDNA solution prepared in the foregoing process (1). Further, 4.0 µl of a 10 x T7 Enzyme Mix contained in the kit was added, and the total 40.0 µl of liquid was mixed well and heated in a vapor phase at 37°C for 14 hours. After the reaction, the synthesized cRNA was purified using an RNeasy Mini Kit(QIAGEN) according to the attached protocol. More specifically, 60 µl of nuclease-free water was added to the IVT reaction solution (40 µl) , and 350 µl of buffer RLT contained in the kit and 250 µl of 100% ethanol was added and mixed well, and the total 700 µl of mixed liquid was placed in an RNeasy mini spin column to be subjected to centrifugation for 15 seconds at 12,000 rpm.   The RNeasy mini spin column was set in a new 2-ml<br>
centrifugal tube, 500 ul of buffer RPE contained in the kit was added to the column, and another 12,000-rpm centrifugation was performed for 15 seconds. This process was performed twice. After the elution liquid was removed, the RNeasy mini spin column was set in the 2-ml centrifugal tube, and 12,000 rpm centrifugation was performed for 2 minutes to dry the membrane in the column. Thereafter, the RNeasy mini spin column was set in a new 1.5-ml tube, and 50 ul of nuclease-free water was directly added onto the membrane, allowed to stand for 10 minute at room temperature, and followed by 12,000-rpm centrifugation for 1 minute. This process was performed twice (100 ul was used in total). [0115]<br>
The sample was mixed well, and dispensed into a 2-ul portion for cRNA quality check using an Agilent 2100 Bioanalyzer and a 2-ul portion diluted 50-fold with sterile distilled water for cRNA quantitative determination; the rest was preserved at -80°C. For the quantitative determination, the absorbency of the 50-fold diluted solution was measured with respect to 260 nm and 280 nm, and the cRNA concentration was determined. The absorbency ratio of 260 nm to 280 nm was confirmed as 1.8 or greater. When the RNA concentration was 0.5 µg/µl or less, the sample was concentrated using a vacuum dryer. The cRNA quality check was performed by electrophoresis using an Agilent 2100 Bioanalyzer and RNA LabChip according to the attached protocol, confirming that the smear peak had at least 500 bases. [0116] (3) Fragmentation of labeled cRNA<br>
Subsequently, according to the protocol of the CodeLink Expression Assay Reagent Kit (GE Healthcare Bio Science), the cRNA was fragmented into about 100 to 200 bases. More specifically, nuclease-free water was added so that the 10 µg of cRNA became 20 ul; then, 5 ul of 5 * fragmentation buffer contained in the kit was added. After heating at 94°C for 20 minutes,  the liquid was rapidly cooled on ice.   78 ul of<br>
hybridization buffer A, 130 µl of hybridization buffer B, 27 ul of nuclease-free water, all contained in the kit, were added to the 25 ul of solution containing 10 ug of cRNA fragments, preparing a hybridization solution of 260 ul in total. After vortexing at maximum speed for 5 seconds and spinning-down, the sample was heated at 90°C for 5 minutes to denaturate the cRNA by heat, followed by cooling on ice. [0117]<br>
(4)	Hybridization of cRNA fragment and microarray<br>
Using a CodeLink Shaker Kit (GE Healthcare Bio Science) and a CodeLink Innova shaker, hybridization was performed according to the protocol. More specifically, a CodeLink uniSet human 20K I bioarray was set in the 12-slide shaker tray contained in the kit. The hybridization solution prepared in the foregoing process (3) was vortexed at maximum speed for 5 seconds and spinned down. The 250 ul of hybridization solution was injected to a right-bottom hole of the sealing chamber of the array,  and the hole was covered with a sealing strip (sticker) attached to the array. The shaker tray having an array thereon was placed in the CodeLink Innova shaker,  and heated at 37 °C for 18 hours while being rotated at 300 rpm. [0118]<br>
(5)	Staining of microarray<br>
Using a CodeLink Parallel Processing Kit (GE Healthcare Bio Science), the array was stained and washed according to the protocol. More specifically, the sealing chamber was removed from the hybridized array, 13 ml of 0.75 x TNT buffer (0.1M Tris-HCl(pH 7.6), 0.15M NaCl, 0.05% Tween 20) was added at room temperature, and the resulting liquid was divided into the slots of a medium reagent reservoir with a bioarray rack contained in the kit. The rack was moved up and down several times to shake off extra hybridization solution. The bioarray rack was then shifted to a large reagent reservoir filled with 240 ml of 0.75 * TNT buffer heated to 46°C. The bioarray rack was heated to 46°C for an hour.<br>
[0119]<br>
As a staining fluid, for each array, 6.8 ul of Streptavidin-Cy5 liquid dissolved by nuclease-free water to 1 mg/ml was diluted 500-fold using 3393.2 ul of TNB buffer (0.1 M Tris-HCl(pH 7.6), 0.15 M NaCl, 0.5 % TSA Blocking Reagent (Perkm Elmer Corporation)) at room temperature. The total 3.4 ml was supplied to fill up the slots of a small reagent reservoir contained in the kit. The bioarray rack holding the array was shifted to the small reagent reservoir, blocked from light with aluminum or the like, and subjected to staining for 30 minutes at room temperature (23°C ±2°C).<br>
[0120]<br>
After staining, the bioarray rack was shifted to a large reagent reservoir filled with 240 ml of TNT buffer (room temperature) , moved up and down several times and allowed to stand for 5 minutes at room temperature. The rack was then shifted to a large reagent reservoir filled with new TNT buffer<br>
(room temperature), moved up and down several times and allowed to stand for 5 minutes at room temperature. This washing process was repeated four times. Finally, the rack was washed with 240 ml of 0.05% Tween 20 solution by being moved up and down for five seconds. This process was repeated twice. The bioarray rack was set in a dried medium reagent reservoir, followed by 3 minutes of centrifugation at room temperature, 600 * g.<br>
[0121]<br>
(6)	Scanning of microarray<br>
Each of the stained arrays was scanned by an Agilent G2565 scanner (Agilent), to be saved a staining pattern as TIFF image. Each TIFF image was processed by CodeLink expression analysis software, thereby digitizing a signal strength for each gene spot on the array. [0122]<br>
(7)	Gene expression assay<br>
The signal strength data thus obtained was normalized using GeneSpring GX (Agilent Technologies) microarray gene expression<br>
assay software, and the data was then assayed. Specifically, the background signal was subtracted from the spot signal. Values less than 0.01 were equalized to 0.01, and each value was divided by the intermediate value of all spot signals so as to find a standardized relative expression amount of the gene. Genes whose relative expression amount was higher universally in the immortalized cancer cell strains (FIG. 1, lower right) than a healthy cell (FIG. 1, upper left) were discriminated in the following manner. Genes that exhibited enhanced expression universally in various organs appeared to be genes involved in cancer cell immortalization (FIG. 1, large arrow pointing to the right), rather than genes (FIG. 1, downward arrow) involved in canceration of healthy cells (FIG. 1, upper left). [0123]<br>
Screened as the first group was any gene in which at least 8 out of 9 kinds of lung cancer cell strains exhibited expression in a quantity of two times or higher than the intermediate value of two normal respiratory epithelia, any gene in which at least 19 out of 21 kinds of esophagus cancer cell strains exhibited expression in a quantity of two times or higher than the intermediate value of two nonneoplastic esophagus epithelia, and any gene in which at least 8 out of 9 digestive cancers, or various other cancer cell strains (gastric cancer, colorectal cancer, head and neck cancer, leukemia) exhibited expression in a quantity of two times or higher than the intermediate value of the four kinds of nonneoplastic epithelia (two kinds of respiratory epithelia and two kinds of esophagus epithelia). As a result, fifty-one genes were screened (FIG. 2, upper left). As the second group, screened was any gene in which the mean value of the expression levels of the three groups of cancer cell strains of different organs was two times or higher than the mean value of the corresponding epithelia (note that, the esophagus cancer cell strains used for this screening based on the mean value correspond to the seven kinds asseyed first) , and in which the expression  level  of  the metastasis  lung cancer tissue<br>
consisting of immortalization lung cancer cells was two times or higher than the mean value of the primary focus tissue and metastasis tissue consisting of non-immortalization lung cancer cells of the same patient. As a result, 80 genes were extracted (FIG. 2, upper right). Among the two screened groups, seven genes were common. [0124]<br>
It is a known fact that cancer cell strains are all immortalized. For each of the immortalization/non-immortalization lung cancer tissues used in the experiments, it was confirmed that the primary focus (FIG. 3, "D2Pr") and hepatic metastasis focus (FIG. 3, "D5M3") of the same patient were not immortalized (telomerase activitation: negative; telomere length: reduced; hTERT protein expression: not observed) , and that one of the lymph node metastasis focuses (FIG. 3, "D4M2") substantially consisted only of immortalized cells (high telomerase activation, extended telomere length, hTERT protein expression was observed in almost all cells) and the other metastasis focus (FIG. 3, "D3M1") consisted of both immortalized cells and non-immortalization cells (low telomerase activation; hTERT protein expression cells were observed among the non-expression cells). These characteristics were confirmed by assaying telomerase activation (Hiyama K et al., J Natl Cancer Inst 87: 895-902, 1995, Case G) , telomere length (Hiyama K et al., Oncogene 10: 937-44, 1995, Case 92-D), and hTERT protein in situ expression (Hiyama E et al., Neoplasia 3: 17-26, 2001, Fig 6A, B). [0125]<br>
For further refinement, a gene that exhibits high expression universally in 11 cell strains of breast cancer, 10 cell strains of ovarian cancer, and 10 cell strains of pancreatic cancer was screened. Finally, in total, the thirteen genes: (1) ubiquitin-conjugating enzyme E2S (UBE2S), (2) replication factor C (activator 1) 4, 37kDa(RFC4), (3) prostaglandin E synthase 2, (PTGES2), (4) MAF1 homolog (S. cerevisiae) (MAF1), (5) activin A receptor, type IIB(ACVR2B), (6) family with sequence similarity<br>
119, member A (FAM119A) , (7) leukotriene B4 12-hydroxydehydrogenase (LTB4DH), (8) dolichyl-phosphate mannosyltransferase polypeptide 2, regulatory subunit (DPM2), (9) selenoprotein X, l(SEPXl), (10) proteasome (prosome, macropain) subunit, alpha type, 3 (PSMA3), (11) coiled-coil-helix-coiled-coil-helix domain containing 3 (CHCHD3), (12) LSM3 homolog, U6 small nuclear RNA associated (S. cerevisiae) (LSM3), and (13) G-2 and S-phase expressed l(GTSEl), were found as genes that exhibit higher expression universally in most lung cancer cell strains, esophagus cancer cell strains, digestive cancer cell strains, breast cancer cell strains, ovarian cancer cell strains, pancreatic cancer cell strains and various other cancer cell strains than mortal nonneoplastic cells; exhibit higher expression in a telomerase-positive lung cancer metastasis tissue<br>
(immortalized cancer cells) than in a telomerase-negative lung cancer metastasis tissue and primary tissue (non-immortalized cancer cells) of the same patient; and exhibit higher expression in pancreatic cancer cell strains consists only of immortalized cells than in a pancreatic cancer tissue containing both immortalized cells and non-immortalized cells (FIG. 3) . The pancreatic cancer tissue used in this experiment showed a higher expression of full length mRNA of the telomerase-coding gene TERT than in the healthy pancreatic tissue of the same patient. However, since the expression was evidently lower compared with the immortalized pancreatic cancer cell strains, it is assumed that the tissue contained non-immortalized cells.<br>
[0126]<br>
As shown in FIGS. 3 to 15, the foregoing thirteen genes have the following characteristics.<br>
(1)	each gene exhibits more enhanced expression in<br>
immortalization cells (solid bar in FIGS. 3 to 15, human cancer<br>
derived cell strains except for MCF-12A, SVlle, SVll-106) than in<br>
nonneoplastic mortal cells (open bar in FIGS. 3 to 15);<br>
(2)	each gene exhibits more enhanced expression in immortalized<br>
cancer cells derived human cancer (solid bar in FIGS. 3 to 15,<br>
except for MCF-12A, SVlle, SV11-106) than finite life cancer cells (diagonal lined bar in FIGS. 3 to 15, except for SV12e, SV12-72); and<br>
(3) its expression is less enhanced in nonneoplastic telomerase expression life-extension cells obtained by introducing a telomerase-coding gene TERT into healthy lung fibroblasts (diamond-shaped dotted bars m FIGS. 3 to 15; TERT6e, TERT6-85) than in the parent cells (TIG-1). These genes are highly expressed specifically in telomerase-positive immortalized cancer cells. This shows that these genes serve to determine the immortalization of cancer cells (immortalization-determining gene). [0127]<br>
Moreover, in the immortalization transformed cells obtained by introducing genes encoding a telomerase-coding gene TERT and SV40 early antigen into healthy lung fibroblasts (FIGS. 3 to 15, SVlle, SV11-106: colony formation and 300 PDL or greater passage capability of passage were confirmed using colony formation assay with soft agar), the expression was enhanced except for MAF1, LTB4DH, than in the parent cells (TIG-1) or nonneoplastic life-extension cells (TERT6e, TERT6-85: absence of colony formation and cessation of cell proliferation at 100PDL or less were confirmed using colony formation assay with soft agar). This is presumably because the cells transformed in vitro are not always equal to human cancer cells. [0128] Example 2<br>
Suppression of cancer cell proliferation by siRNA (small interfering RNA) of immortalization-determining gene To analyze how the thirteen kinds of immortalization-determining genes identified in Example 1 relate to cancer cell proliferation, siRNAs specified according to the respective gene sequences were designed and created. They were introduced into three kinds of cancer cell strains (HeLa: uterine cervix cancer, KYSE150: esophagus cancer, and HCC50: colon cancer).  With these samples,<br>
the  suppression  of mRNA  expression  of  an  immortalization-determining gene, and the effect on cell proliferation at 96 hours after introduction were observed. [0129]<br>
(1) Designing and creation of immortalization-determining gene sequence specific siRNA (small interfering RNA)<br>
In each of the thirteen immortalization-determining genes, two or four kinds of specific siRNA sequences were determined, and Double-strand siRNAs each formed of a sense strand and an antisense strand was purchased from Qiagen and Japan Bio Service. Among the created siRNAs sequences, only the sense strands are shown in Tables 2 and 3. The siRNA-1 of each gene was an equimolar mixture of two kinds. [0130] Table 2<br><br>
(Table Removed) <br>
[0131]<br><br>
Table 3<br><br>
(Table Removed) <br>
[0132]<br>
(2) Culture of human cancer cell strains<br>
The three kinds of cancer cell strains (HeLa: uterine cervix cancer, KYSE150: esophagus cancer, and HCC50: colon cancer) were cultured under the following conditions. HeLa was cultured in a DMEM culture medium (Nacalai Tesque Inc.) containing 10% fetal bovine serum (Sigma Corporation) and gentamycin (Sigma Corporation). KYSE150 and HCC50 were cultured in RPMI 1640 culture media (Nacalai Tesque Inc.) containing 10% Fetal Bovine Serum and gentamycin. [0133]<br>
(3)	Gene  introduction  of  immortalization-determining  gene<br>
sequence specific siRNA into cancer cell strain<br>
On the day before the transfection, cancer cells previously cultured were isolated using trypsin. After being suspended in the culture medium detailed in (2) above, HeLa =1.2 x 104, KYSE150 = 2 x 104, and HCC50 = 1.5 x 104 were seeded for each well of a 24-well plate. On the day of transfection, the media were replaced with serum-free media Opti-MEM (Invitrogen), and the siRNA shown in Table 2 and 3 were subjected to transfection using Oligofectamine (Invitrogen) or Lipofectamine 2000 (Invitrogen). The concentration of siRNA upon transfection was 40 nM per well. A control (non-silencing) siRNA (Qiagen) was used as a negative control. [0134]<br>
(4)	Assay of the suppression of relative mRNA expression of<br>
immortalization-determining gene<br>
One day after the transfection detailed in (3), total RNA was extracted from each cancer cell strain using RNeasy Mini (Qiagen) according to the instruction book.  Using a QuantiTect probe RT-PCR  (Qiagen)  and an immortalization-determining gene sequence specific TaqMan probe (ABI), quantitative RT-PCR (ABI PRISM 7000 sequence detection system (produced by ABI) was used) was performed under the following conditions, i) 50°C, 30 minutes, ii) 95°C, 15 minutes, iii) 94°C, 15 seconds →  60°C, 1 minutes x 35 to 45 cycles.<br><br>
 [0135]<br>
Using ß-actin as an internal standard, the expression amount was digitalized based on each amplification curve. As a control, the value of a cell into which NS-sequence siRNA was introduced was used. Measurement of the relative mRNA expression of the immortalization-determining gene was performed twice with the same target gene and the same cancer cell strains under the same conditions. [0136]<br>
FIGS. 16 to 28 show the percentages of relative mRNA amounts of immortalization-determining genes in the cells to which each of siRNAs was introduced, when the immortalization-determining gene mRNA percentage in the control is assumed to be 100%. The measurements are shown as mRNA-1 and mRNA-2, respectively, in FIGS. 16 to 28. The suppression of mRNA expression by any one of siRNAs (siRNA-1 to siRNA-3) was observed in all target genes. [0137] (5) Measurement of the proliferation of cancer cell strains<br>
The proliferation of cancer cell strains was measured using viable cell count measurement reagent SF (Nacalai Tesque Inc.). At 96 hours after the transfection detailed in (3) , 10 ul of WST reagent was added to each well. After 3-hour incubation at 37°C, absorbencies at 450 nm and 595 nm (reference wavelengths) were measured using a plate reader (Wallac ARVO MX1420 Multilabel Counter: produced by Perkin Elmer).  According to: absorbency at 450 nm - absorbency at 595 nm - absorbency at BG (only culture medium) , the data was digitalized, and a mean value of three wells was found.  Then, the value was shown as a percentage when the percentage of NS sequence siRNA introduction cell is assumed to be 100%.  Measurement of proliferation of cancer cell strains was performed twice with the same target gene and same cancer cell strains under the same conditions. [0138]<br>
The respective cell counts upon siRNA introduction are<br>
shown as MTT-1 and MTT-2, respectively, in FIGS. 16 to 28, when the percentage of cell count upon the NS sequence siRNA introduction is assumed to be 100%. In all target genes, the suppression of proliferation by any one of the siRNAs (siRNA-1 to siRNA-3) was seen in three kinds of cancer cells, indicating that those thirteen kinds of immortalization-determining genes can be regarded as being target molecules to attain universal suppression of immortalized cancer cell proliferation. [0139]<br>
The discrimination of immortalized cancer cells from mortal cancer cells has significance in functioning as a malum marker, as well as in contributing to remedy development. General cancer diagnosis is based on pathological findings, with which the diagnosis of differentiation, atypism, etc., is also possible. However, general pathologic assays do not allow judgment as to whether the cells are immortalized, in other words, whether the cells have infinite lives. Human telomerase activation has attracted attention to date as an immortalization marker. However, as shown above, telomerase activation has also been seen in some healthy cells; therefore, the new idea that telomerase activation does not always immediately immortalize cells has become knowledge. Further, since the telomerase expression amount is very low, even though immune system staining was established, only a few laboratories have reported their achievements (Hiyama E et al., Neoplasia 3: 17-26, 2001). The use of the marker of the present invention to detect the expression of an immortalization-determining gene allows immortalized cancer cells to be discriminated from finite life cells. Such discrimination was not possible by general conventional pathological diagnosis. The present invention thus has great potential as a clinical application useful for early diagnosis, new treatment options, prognostification, and the like.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
[0140]<br>
FIG. 1 is a schematic view illustrating a mechanism of human cell<br>
canceration and immortalization.<br>
FIG. 2 is a schematic view illustrating a target gene extraction<br>
process according to the present invention.<br>
FIG. 3 is a graph showing expression levels of UBE2S gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 4 is a graph showing expression levels of RFC4 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 5 is a graph showing expression 1 evels of PTGES2 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 6 is a graph showing expression levels of MAF1 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 7 is a graph showing expression levels of ACVR2B gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 8 is a graph showing expression levels of FAM119A gene in<br>
the various immortalized cancer cells, using an oligo array.<br>
FIG. 9 is a graph showing expression levels of LTB4DH gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 10 is a graph showing expression levels of DPM2 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 11 is a graph showing expression levels of SEPX1 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 12 is a graph showing expression levels of PSMA3 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 13 is a graph showing expression levels of CHCHD3 gene in<br>
the various immortalized cancer cells, using an oligo array.<br>
FIG. 14 is a graph showing expression levels of LSM3 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 15 is a graph showing expression levels of GTSE1 gene in the<br>
various immortalized cancer cells, using an oligo array.<br>
FIG. 16 is a graph showing an effect of expression suppression of<br>
UBE2S gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 17 is a graph showing an effect of expression suppression of<br>
RFC4 gene and suppression of cell proliferation given by siRNA in<br>
cancer cell strains (Example 2).<br>
FIG. 18 is a graph showing an effect of expression suppression of<br>
PTGES2 gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 19 is a graph showing an effect of expression suppression of<br>
MAF1 gene and suppression of cell proliferation given by siRNA in<br>
cancer cell strains (Example 2).<br>
FIG. 20 is a graph showing an effect of expression suppression of<br>
ACVR2B gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 21 is a graph showing an effect of expression suppression of<br>
FAM119A gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 22 is a graph showing an effect of expression suppression of<br>
LTB4DH gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 23 is a graph showing an effect of expression suppression of<br>
DPM2 gene and suppression of cell proliferation given by siRNA in<br>
cancer cell strains (Example 2).<br>
FIG. 24 is a graph showing an effect of expression suppression of<br>
SEPX1 gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 25 is a graph showing an effect of expression suppression of<br>
PSMA3 gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 26 is a graph showing an effect of expression suppression of<br>
CHCHD3 gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
FIG. 27 is a graph showing an effect of expression suppression of<br>
LSM3 gene and suppression of cell proliferation given by siRNA in<br>
cancer cell strains (Example 2) .<br>
FIG. 28 is a graph showing an effect of expression suppression of<br>
GTSE1 gene and suppression of cell proliferation given by siRNA<br>
in cancer cell strains (Example 2).<br>
SEQUENCE LISTING FREE TEXT [0141]<br>
SEQ ID No.27 represents the base sequence of the sense strand of siRNA-1 (44-64), of the UBE2S gene.<br>
SEQ ID No.28 represents the base sequence of the sense strand of siRNA-1 (146-166), of the UBE2S gene.<br>
SEQ ID No.29 represents the base sequence of the sense strand of siRNA-2, of the UBE2S gene.<br>
SEQ ID No.30 represents the base sequence of the sense strand of siRNA-3, of the UBE2S gene.<br>
SEQ ID No.31 represents the base sequence of the sense strand of siRNA-1 (897-917), of the RFC4 gene.<br>
SEQ ID No.32 represents the base sequence of the sense strand of siRNA-1 (189-209), of the RFC4 gene.<br>
SEQ ID No.33 represents the base sequence of the sense strand of siRNA-2, of the the RFC4 gene.<br>
SEQ ID No.34 represents the base sequence of the sense strand of siRNA-3, of the RFC4 gene.<br>
SEQ ID No.35 represents the base sequence of the sense strand of siRNA-1 (2003-2023), of the PTGES2 gene.<br>
SEQ ID No.36 represents the base sequence of the sense strand of siRNA-1 (1105-1125), of the PTGES2 gene.<br>
SEQ ID No.37 represents the base sequence of the sense strand of siRNA-2, of the PTGES2 gene.<br>
SEQ ID No.38 represents the base sequence of the sense strand of siRNA-3, of the PTGES2 gene.<br>
SEQ ID No.39 represents the base sequence of the sense strand of siRNA-1 (1538-1558), of the MAF1 gene.<br>
SEQ ID No.40 represents the base sequence of the sense strand of siRNA-1 (1031-1051), of the MAF1 gene.<br>
SEQ ID No.41 represents the base sequence of the sense strand of siRNA-2, of MAF1 gene.<br>
SEQ ID No.42 represents the base sequence of the sense strand of siRNA-3, of the MAF1 gene.<br>
SEQ ID No.43 represents the base sequence of the sense strand of<br>
siRNA-1 (626-646), of the ACVR2B gene.<br>
SEQ ID No.44 represents the base sequence of the sense strand of<br>
siRNA-1 (208-228), of the ACVR2B gene.<br>
SEQ ID No.45 represents the base sequence of the sense strand of<br>
siRNA-2, of the ACVR2B gene.<br>
SEQ ID No.46 represents the base sequence of the sense strand of<br>
siRNA-3, of the ACVR2B gene.<br>
SEQ ID No.47 represents the base sequence of the sense strana of<br>
siRNA-1(754-774), of the FAM119A gene.<br>
SEQ ID No.48 represents the base sequence of the sense strand of<br>
siRNA-1(1068-1088), of the FAM119A gene.<br>
SEQ ID No.49 represents the base sequence of the sense strand of<br>
siRNA-2, of the FAM119A gene.<br>
SEQ ID No.50 represents the base sequence of the sense strand of<br>
siRNA-3, of the FAM119A gene.<br>
SEQ ID No.51 represents the base sequence of the sense strand of<br>
siRNA-1(775-795) , of the LTB4DH gene.<br>
SEQ ID No.52 represents the base sequence of the sense strand of<br>
siRNA-1(658-678), of the LTB4DH gene.<br>
SEQ ID No.53 represents the base sequence of the sense strand of<br>
siRNA-2, of the LTB4DH gene.<br>
SEQ ID No.54 represents the base sequence of the sense strand of<br>
siRNA-3, of the LTB4DH gene.<br>
SEQ ID No.55 represents the base sequence of the sense strand of<br>
siRNA-1(145-165), of the DPM2 gene.<br>
SEQ ID No.56 represents the base sequence of the sense strand of<br>
siRNA-1(84-104), of the DPM2 gene.<br>
SEQ ID No.57 represents the base sequence of the sense strand of<br>
siRNA-2, of the DPM2 gene.<br>
SEQ ID No.58 represents the base sequence of the sense strand of<br>
siRNA-3, of the DPM2 gene.<br>
SEQ ID No.59 represents the base sequence of the sense strand of<br>
siRNA-1(870-890), of the SEPX1 gene.<br>
SEQ ID No.60 represents the base sequence of the sense strand of<br>
siRNA-1(794-814), of the SEPX1 gene.<br>
SEQ ID No.61 represents the base sequence of the sense strand of<br>
siRNA-2, of the SEPX1 gene.<br>
SEQ ID No.62 represents the base sequence of the sense strand of<br>
siRNA-3, of the SEPX1 gene.<br>
SEQ ID No.63 represents the base sequence of the sense strand of<br>
siRNA-1(853-873), of the PSMA3 gene.<br>
SEQ ID No.64 represents the base sequence of the sense strand of<br>
siRNA-1(686-706), of the PSMA3 gene.<br>
SEQ ID No.65 represents the base sequence of the sense strand of<br>
siRNA-2, of the PSMA3 gene.<br>
SEQ ID No.66 represents the base sequence of the sense strand of<br>
siRNA-3, of the PSMA3 gene.<br>
SEQ ID No.67 represents the base sequence of the sense strand of<br>
siRNA-1(546-566), of the CHCHD3 gene.<br>
SEQ ID No.68 represents the base sequence of the sense strand of<br>
siRNA-1(1450-1470), of the CHCHD3 gene.<br>
SEQ ID No.69 represents the base sequence of the sense strand of<br>
siRNA-2, of the CHCHD3 gene.<br>
SEQ ID No.70 represents the base sequence of the sense strand of<br>
siRNA-3, of the CHCHD3 gene.<br>
SEQ ID No.71 represents the base sequence of the sense strand of<br>
siRNA-1(540-560), of the LSM3 gene.<br>
SEQ ID No.72 represents the base sequence of the sense strand of<br>
siRNA-1(37-57), of the LSM3 gene.<br>
SEQ ID No.73 represents the base sequence of the sense strand of<br>
siRNA-2, of the LSM3 gene.<br>
SEQ ID No.74 represents the base sequence of the sense strand of<br>
siRNA-3, of the LSM3 gene.<br>
SEQ ID No.75 represents the base sequence of the sense strand of<br>
siRNA-2, of the GTSE1 gene.<br>
SEQ ID No.76 represents the base sequence of the sense strand of<br>
siRNA-3, of the GTSE1 gene.<br><br><br><br><br><br><br><br>
WE CLAIMS:-<br>
1.	A marker gene for determining cancer cell immortalization, the marker gene comprising a polynucleotide having a base sequence of at least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases within any one of base sequences represented by SEQ ID Nos.l to 13.<br>
2.	The marker gene according to claim 1, wherein the marker gene is a probe or primer.<br>
3.	A method for determining an immortalized cancer cell,<br>
comprising the steps of:<br>
(1)	bonding the marker gene according to claim 1 or 2 with RNA prepared from a biological sample that is extracted from a test subject and that may contain a cancer cell, or a derivative of the RNA;<br>
(2)	measuring an amount of the RNA or the derivative thereof bonded with the marker gene, using the marker gene as an index; and<br>
(3)	comparing the amount of RNA or the derivative<br>
thereof found in Step (2) (generically referred to as "an<br>
RNA amount", hereinafter) with an amount (generically<br>
referred to as "a comparative RNA amount", hereinafter) of<br>
a corresponding RNA or a derivative thereof in a non-<br>
immortalized healthy or cancer cell.<br>
The method for determining an immortalized cancer cell according to claim 3, further comprising the step of:<br>
(4)	determining that the cancer cell of the test<br>
subject is immortalized when the RNA amount found in Step<br>
(2) is higher than the comparative RNA amount, and<br>
determining that the cancer cell of the test subject is not<br>
immortalized when the RNA amount found in Step (2) is not<br><br>
higher than the comparative RNA amount.<br>
5. An antibody for recognizing a polypeptide having any one of amino acid sequences represented by SEQ ID Nos.14 to 26.<br>
6.   A method for determining an immortalized cancer cell, comprising the step of:<br>
(1') bonding a protein-containing fraction containing a polypeptide prepared from a biological sample that is extracted from a test subject and that may contain a cancer cell with the antibody according to claim 5;<br>
(2' ) measuring the amount of the polypeptide that is bonded with the antibody, using the antibody as an index; and<br>
(3') comparing the amount (generically referred to as "a polypeptide amount", hereinafter) of the polypeptide found in Step (2') with the amount (generically referred to as "a comparative polypeptide amount", hereinafter) of a corresponding polypeptide in a non-immortalized healthy or cancer cell.<br>
7. The method for determining an immortalized cancer cell according to claim 6, further comprising the step of:<br>
(4') determining that the cancer cell of the test subject is immortalized when the polypeptide amount found in Step (2' ) is higher than the comparative polypeptide amount, and determining that the cancer cell of the test subject is not immortalized when the polypeptide amount found in Step (2' ) is not higher than the comparative polypeptide amount.<br>
8. An immortalized cancer cell determining reagent kit including at least one member selected from the group consisting of the marker gene according to claim 1 and the<br>
antibody according to claim 5.<br>
9.	A method for screening a substance for suppressing<br>
proliferation of immortalized cancer cells, comprising the<br>
steps of:<br>
(A)	bringing a test material into contact with a cell<br>
that can express one of the immortalization-determining<br>
genes represented by SEQ ID Nos. 1 to 13;<br>
(B)	measuring an expression amount of one of the<br>
immortalization-determining genes represented by SEQ ID Nos.<br>
1 to 13 in the cell brought into contact with the test<br>
material;<br>
(C)	comparing the expression amount of the<br>
immortalization-determining gene found in Step (B) with an<br>
expression amount (generically referred to as "a<br>
comparative expression amount", hereinafter) of an<br>
immortalization-determining gene in a cell not in contact<br>
with the test material; and<br>
(D)	selecting  the  test  material  as  a  candidate<br>
substance  for suppressing proliferation of immortalized<br>
cancer cells when the expression amount found in Step (B)<br>
is lower than the comparative expression amount.<br>
10.	A method for screening a substance for suppressing<br>
proliferation of immortalized cancer cells, comprising the<br>
steps of:<br>
(A' ) bringing a test material into contact with a polypeptide having one of the amino acid sequences represented by SEQ ID Nos. 14 to 26;<br>
(B') measuring activity of the polypeptide having one of the amino acid sequences represented by SEQ ID Nos. 14 to 26 brought into contact with the test material;<br>
(C ) determining the degree of the activity of the polypeptide found in Step (B') by comparing the activity of the  polypeptide  found  in  Step   (B' )  with  activity<br>
 (generically referred to as "a comparative activity", hereinafter) of a polypeptide not in contact with the test material; and<br>
(D' ) selecting the test material as a candidate substance for suppressing proliferation of immortalized cancer cells when the activity of the polypeptide found in Step (B' ) is lower than the comparative activity.<br>
11.	An immortalized cancer cell growth inhibitor containing, as an active ingredient, a polynucleotide having a base sequence of • at least 15 bases that specifically hybridizes with a continuous base sequence of at least 15 bases within any one of base sequences represented by SEQ ID Nos. 1 to 13.<br>
12.	The immortalized cancer cell growth inhibitor according to claim 11, wherein the polynucleotide has any one of base sequences represented by SEQ ID Nos. 27 to<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1BYnN0cmFjdC0oMTMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Abstract-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1BYnN0cmFjdC0oMTUtMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Abstract-(15-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1DbGFpbXMtKDEzLTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Claims-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1DbGFpbXMtKDE1LTA1LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Claims-(15-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDEzLTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Correspondence Others-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE1LTA1LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Correspondence Others-(15-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1Gb3JtLTE4LSgxOC0wMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Form-18-(18-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1Gb3JtLTItKDEzLTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Form-2-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1Gb3JtLTItKDE1LTA1LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Form-2-(15-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1Gb3JtLTMtKDEzLTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Form-3-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1HUEEtKDEzLTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-GPA-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1wY3QtMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1wY3QtMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1wY3QtMzA4LnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0OTctZGVsbnAtMjAwOC1QZXRpdGlvbi0xMzctKDEzLTA5LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">10497-delnp-2008-Petition-137-(13-09-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260863-anaerobic-digester-system-for-animal-waste-stabilization-and-biogas-recovery.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260865-laminate-of-thermoplastic-film-materials-exhibiting-throughgoing-porosity.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260864</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>10497/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-May-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-May-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Dec-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>KABUSHIKI KAISHA YAKULT HONSHA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1-91, HIGASHI-SHINBASHI 1-CHOME, MINATO-KU, TOKYO 1058660, JAPAN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MASAHIKO NISHIYAMA</td>
											<td>C/O RESEARCH INSTITUTE FOR RADIATION BIOLOGY AND MEDICINE, HIROSHIMA,1-2-3, KASUMI, MINAMI-KU, HIROSHIMA-SHI, HIROSHIMA 7348553, JAPAN.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KEIKO HIYAMA</td>
											<td>C/O RESEARCH INSTITUTE FOR RADIATION BIOLOGY AND MEDICINE, HIROSHIMA,1-2-3, KASUMI, MINAMI-KU, HIROSHIMA-SHI, HIROSHIMA 7348553, JAPAN.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KEIJI TANIMOTO</td>
											<td>C/O RESEARCH INSTITUTE FOR RADIATION BIOLOGY AND MEDICINE, HIROSHIMA,1-2-3, KASUMI, MINAMI-KU, HIROSHIMA-SHI, HIROSHIMA 7348553, JAPAN.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NORIO MASUKO</td>
											<td>C/O TAIHO PHARMACEUTICAL CO., LTD., 1-27, MISUGIDAI, HANNO-SHI, SAITAMA 3578527, JAPAN.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2007/061652</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-06-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2006-161350</td>
									<td>2006-06-09</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260864-a-method-for-determining-an-immortalized-cancer-cell-in-cancer-cells-in-vitro by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:10:52 GMT -->
</html>
