<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/260892-transgenic-plants-with-enhanced-agronomic-traits by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:17:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 260892:&quot;TRANSGENIC PLANTS WITH ENHANCED AGRONOMIC TRAITS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;TRANSGENIC PLANTS WITH ENHANCED AGRONOMIC TRAITS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention provides transgenic plant cells with recombinant DNA for expression of proteins that are useful for imparting enhanced agronomic trait(s) to transgenic crop plants. This invention also provides transgenic plants and progeny seed comprising the transgenic plant cells where the plants are selected for having an enhanced trait selected from the group of traits consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil. Also disclosed are methods for manufacturing transgenic seed and plants with enhanced traits.91</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Transgenic Plants With Enhanced Agronomic Traits<br>
Cross Reference To Related Applications<br>
This application claims benefit under 35USC ยง 119(e) of United States provisional application Serial No. 60/638,099, filed 12/21/2004, and United States provisional application Serial No. 60/660,320, filed 3/10/2005, both of which herein incorporated by reference.<br>
Incorporation Of Sequence Listing<br>
Two copies of the sequence listing (Copy 1 and Copy 2) and a computer readable form (CRF) of the sequence listing, all on CD-ROMs, each containing the text file named "38-21_53720B_red_seqListing.txt", which is about 515 kilobytes (measured in MS-WINDOWS) and was redacted from the sequence listing that was originally filed with the International Application PCT/US2005/046031 and created on December 13, 2005, are herein incorporated by reference.<br>
Field Of The Invention<br>
Disclosed herein are inventions in the field of plant genetics and developmental biology. More specifically, the present inventions provide plant cells with recombinant DNA for providing an enhanced trait in a transgenic plant, plants comprising such cells, seed and pollen derived from such plants, methods of making and using such cells, plants, seeds and pollen.<br>
Background Of The Invention<br>
Transgenic plants with improved agronomic traits such as yield, environmental stress tolerance, pest resistance, herbicide tolerance, improved seed compositions, and the like are desired by both farmers and consumers. Although considerable efforts in plant breeding have provided significant gains in desired traits, the ability to introduce specific DNA into plant genomes provides further opportunities for generation of plants with improved and/or unique traits. Merely introducing recombinant DNA into a plant genome doesn't always produce a transgenic plant with an enhanced agronomic trait. Methods to select individual transgenic events from a population are required to identify those transgenic events that are characterized by the enhanced agronomic trait.<br>
Summary Of The Invention<br>
This invention employs recombinant DNA for expression of proteins that are useful for imparting enhanced agronomic traits to the transgenic plants. Recombinant DNA in this invention is provided in a construct comprising a promoter that is functional in plant cells and that is operably linked to DNA that encodes a protein having at least one amino acid domain in a sequence that exceeds the Pfam gathering cutoff for amino acid sequence alignment with a protein domain family identified by a Pfam name in the group of Pfam names as identified in Table 28.   hi more specific embodiments of the invention the protein expressed in plant cells has an amino acid sequence with at least 90% identity to a consensus amino acid sequence in the group of consensus amino acid sequences consisting of the consensus amino acid sequence constructed for SEQ ID NO: 84 and homologs thereof through the consensus amino acid sequence constructed for SEQ ID NO: 166 and homologs thereof. In even more specific embodiments of the invention the protein expressed in plant cells is a protein selected from the group of proteins identified in Table 2.<br>
Other aspects of the invention are specifically directed to transgenic plant cells comprising the recombinant DNA of the invention, transgenic plants comprising a plurality of such plant cells, progeny transgenic seed and transgenic pollen from such plants.   Such plant cells are selected from a population of transgenic plants regenerated from plant cells transformed with recombinant DNA and that express the protein by screening transgenic plants in the population for an enhanced trait as compared to control plants that do not have said recombinant DNA, where the enhanced trait is selected from group of enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil.<br>
In yet another aspect of the invention the plant cells, plants, seeds and pollen further    . comprise DNA expressing a protein that provides tolerance from exposure to an herbicide applied at levels that are lethal to a wild type of said plant cell. Such tolerance is especially useful not only as a advantageous trait in such plants but is also useful in a selection step in the methods of the invention. In aspects of the invention the agent of such herbicide is a glyphosate, dicamba, or glufosinate compound.<br>
Yet other aspects of the invention provide transgenic plants which are homozygous for the recombinant DNA and transgenic seed of the invention from corn, soybean, cotton, canola, alfalfa, wheat or rice plants. In other important embodiments for practice of various aspects of the invention in Argentina the recombinant DNA is provided in plant cells derived<br>
from corn lines that that are and maintain resistance to the Mai de Rio Cuarto virus or the Puccina sorghi fungus or both.<br>
This invention also provides methods for manufacturing non-natural, transgenic seed that can be used to produce a crop of transgenic plants with an enhanced trait resulting from expression of stably-integrated, recombinant DNA for expressing a protein having at least one domain of amino acids in a sequence that exceeds the Pfam gathering cutoff for amino acid sequence alignment with a protein domain family identified by a Pfam name in the group of Pfam names identified in Table 28. More specifically the method comprises (a) screening a population of plants for an enhanced trait and a recombinant DNA, where individual plants in the population can exhibit the trait at a level less than, essentially the same as or greater than the level that the trait is exhibited in control plants which do not express the recombinant DNA, (b) selecting from the population one or more plants that exhibit the trait at a level greater than the level that said trait is exhibited in control plants, (c) verifying that the recombinant DNA is stably integrated in said selected plants, (d) analyzing tissue of a selected plant to determine the production of a protein having the function of a protein encoded by nucleotides in a sequence of one of SEQ ID NO: 1-83; and (e) collecting seed from a selected plant, hi one aspect of the invention the plants in the population further comprise DNA expressing a protein that provides tolerance to exposure to an herbicide applied at levels that are lethal to wild type plant cells and the selecting is effected by treating the population with the herbicide, e.g. a glyphosate, dicamba, or glufosinate compound, hi another aspect of the invention the plants are selected by identifying plants with the enhanced trait. Ihe methods are especially useful for manufacturing corn, soybean, cotton, alfalfa, wheat or rice seed.<br>
Another aspect of the invention provides a method of producing hybrid corn seed comprising acquiring hybrid corn seed from a herbicide tolerant corn plant which also has stably-integrated, recombinant DNA comprising a promoter that is (a) functional in plant cells and (b) is operably linked to DNA that encodes a protein having at least one domain of amino acids in a sequence that exceeds the Pfam gathering cutoff for amino acid sequence alignment with a protein domain family identified by a Pfam name in the group of Pfam names identified in Table 28.   The methods further comprise producing corn plants from said hybrid corn seed, wherein a fraction of the plants produced from said hybrid corn seed is homozygous for said recombinant DNA, a fraction of the plants produced from said hybrid corn seed is hemizygous for said recombinant DNA, and a fraction of the plants produced from said hybrid corn seed has none of said recombinant DNA; selecting com plants which<br>
are homozygous and hemizygous for said recombinant DNA by treating with an herbicide; collecting seed from herbicide-treated-surviving com plants and planting said seed to produce further progeny corn plants; repeating the selecting and collecting steps at least once to produce an inbred corn line; and crossing the inbred corn line with a second corn line to produce hybrid seed.<br>
Another aspect of the invention provides a method of selecting a plant comprising plant cells of the invention by using an immunoreactive antibody to detect the presence of protein expressed by recombinant DNA in seed or plant tissue. Yet another aspect of the invention provides anti-counterfeit milled seed having, as an indication of origin, a plant cells of this invention.<br>
Still other aspects of this invention relate to transgenic plants with enhanced water use . efficiency or enhanced nitrogen use efficiency. For instance, this invention provides methods of growing a corn, cotton or soybean crop without irrigation water comprising planting seed having plant cells of the invention which are selected for enhanced water use efficiency. Alternatively methods comprise applying reduced irrigation water, e.g. providing up to 300 millimeters of ground water during the production of a corn crop. This invention also provides methods of growing a corn, cotton or soybean crop without added nitrogen fertilizer comprising planting seed having plant cells of the invention which are selected for enhanced nitrogen use efficiency. Brief Description of the Drawings<br>
Figure 1 is a DNA sequence. Detailed Description Of The Invention Table 1 is a brief description of the sequence listing.<br>
(Table 1 Removed)<br>
As used herein a "plant cell" means a plant cell that is transformed with stably-integrated, non-natural, recombinant DNA, e.g. by Agrobacterium-mediated transformation or by baombardment using microparticles coated with recombinant DNA or other means. A plant cell of this invention can be an originally-transformed plant cell that exists as a microorganism or as a progeny plant cell that is regenerated into differentiated tissue, e.g. into a transgenic plant with stably-integrated, non-natural recombinant DNA, or seed or pollen derived from a progeny transgenic plant.<br>
As used herein a "transgenic plant" means a plant whose genome has been altered by the stable integration of recombinant DNA. A transgenic plant includes a plant regenerated from an originally-transformed plant cell and progeny transgenic plants from later generations or crosses of a transformed plant.<br>
As used herein "recombinant DNA" means DNA which has been a genetically engineered and constructed outside of a cell including DNA containing naturally occurring DNA or cDNA or synthetic DNA.<br>
As used herein "consensus sequence" means an artificial sequence of amino acids in a conserved region of an alignment of amino acid sequences of homologous proteins, e.g. as determined by a CLUSTALW alignment of amino acid sequence of homolog proteins.<br>
As used herein "homolog" means a protein in a group of proteins that perform the same biological function, e.g. proteins that belong to the same Pfam protein family and that provide a common enhanced trait in transgenic plants of this invention.   Homologs are expressed by homologous genes. Homologous genes include naturally occurring alleles and artificially-created variants. Degeneracy of the genetic code provides the possibility to substitute at least one base of the protein encoding sequence of a gene with a different base without causing the amino acid sequence of the polypeptide produced from the gene to be changed. Hence, a polynucleotide useful in the present invention may have any base sequence that has been changed from SEQ ID NO:1 through SEQ ID NO: 83 by substitution in accordance with degeneracy of the genetic code. Homologs are proteins that, when optimally aligned, have at least 60% identity, more preferably about 70% or higher, more preferably at least 80% and even more preferably at least 90% identity over the full length of<br>
a protein identified as being associated with imparting an enhanced trait when expressed in plant cells.<br>
Homologs are be identified by comparison of amino acid sequence, e.g. manually or by use of a computer-based tool using known homology-based search algorithms such as those commonly known and referred to as BLAST, FASTA, and Smith-Waterman. A local sequence alignment program, e.g. BLAST, can be used to search a database of sequences to find similar sequences, and the summary Expectation value (E-value) used to measure the sequence base similarity. As a protein hit with the best E-value for a particular organism may not necessarily be an ortholog or the only ortholog, a reciprocal query is used in the present invention to filter hit sequences with significant E-values for ortholog identification. The reciprocal query entails search of the significant hits against a database of amino acid sequences from the base organism that are similar to the sequence of the query protein. A hit is a likely ortholog, when the reciprocal query's best hit is the query protein itself or a protein encoded by a duplicated gene after speciation. A further aspect of the invention comprises functional homolog proteins that differ in one or more amino acids from those of disclosed protein as the result of conservative amino acid substitutions, for example substitutions are among: acidic (negatively charged) amino acids such as aspartic acid and glutamic acid; basic (positively charged) amino acids such as arginine, histidine, and lysine; neutral polar amino acids such as glycine, serine, threonine, cysteine, tyrosine, asparagine, and glutamine; neutral nonpolar (hydrophobic) amino acids such as alanine, leucine, isoleucine, valine, proline, phenylalanine, tryptophan, and methionine; amino acids having aliphatic side chains such as glycine, alanine, valine, leucine, and isoleucine; amino acids having aliphatic-hydroxyl side chains such as serine and threonine; amino acids having amide-containing side chains such as asparagine and glutamine; amino acids having aromatic side chains such as phenylalanine, tyrosine, and tryptophan; amino acids having basic side chains such as lysine, arginine, and histidine; amino acids having sulfur-containing side chains such as cysteine and methionine; naturally conservative amino acids such as valine-leucine, valine-isoleucine, phenylalanine-tyrosine, lysine-arginine, alanine-valine, aspartic acid-glutamic acid, and asparagine-glutamine. A further aspect of the homologs encoded by DNA useful in the transgenic plants of the invention are those proteins that differ from a disclosed protein as the result of deletion or insertion of one or more amino acids in a native sequence.<br>
As used herein, "percent identity" means the extent to which two optimally aligned DNA or protein segments are invariant throughout a window of alignment of components, for example nucleotide sequence or amino acid sequence. An "identity fraction" for aligned<br>
segments of a test sequence and a reference sequence is the number of identical components that are shared by sequences of the two aligned segments divided by the total number of sequence components in the reference segment over a window of alignment which is the smaller of the full test sequence or the full reference sequence. "Percent identity" ("% identity") is the identity fraction times 100.<br>
As used herein "Pfam" refers to a large collection of multiple sequence alignments and hidden Markov models covering many common protein families, e.g. Pfam version 18.0 (August 2005) contains alignments and models for 7973 protein families and is based on the Swissprot 47.0 and SP-TrEMBL 30.0 protein sequence databases. See S.R. Eddy, "Profile Hidden Markov Models", Bioinformatics 14:755-763,1998. Pfam is currently maintained and updated by a Pfam Consortium. The alignments represent some evolutionary conserved structure that has implications for the protein's function. Profile hidden Markov models (profile HMMs) built from the Pfam alignments are useful for automatically recognizing that a new protein belongs to an existing protein family even if the homology by alignment appears to be low. Once one DNA is identified as encoding a protein which imparts an enhanced trait when expressed in transgenic plants, other DNA encoding proteins in the same protein family are identified by querying the amino acid sequence of protein encoded by candidate DNA against the Hidden Markov Model which characterizes the Pfam domain using HMMER software publically available through Pfam Consortium. Candidate proteins meeting the gathering cutoff for the alignment of a particular Pfam are in the protein family and have cognate DNA that is useful in constructing recombinant DNA for the use in the plant cells of this invention. Hidden Markov Model databases for use with HMMER software in identifying DNA expressing protein in a common Pfam for recombinant DNA in the plant cells of this invention are included in United States Application Serial No. 11/311,940. The HMMER software and Pfam databases are version 18.0 and were used to identify known domains in the proteins corresponding to amino acid sequence of SEQ ID NO:84 through SEQ ID NO:166. All DNA encoding proteins that have scores higher than the gathering cutoff disclosed in Table 27 by Pfam analysis disclosed herein can be used in recombinant DNA of the plant cells of this invention, e.g. for selecting transgenic plants having enhanced agronomic traits. The relevant Pfams for use in this invention, as more specifically disclosed below, are AAA, AP2, Aldo ket red, Alpha-amylase, Aminotran 1 2, Ank, ArfGap, Asn synthase, BRO1, CBFD NFYB HMF, Catalase, CorA, Cpn60 TCP1, Cystatin, DNA photolyase, DSPc, DUF1685, DUF296, Dil9, E2F TDP, FAD binding 7, FA desaturase, FBPase, GAF, GATA, GATase 2, Glyco hydro 1, Glyoxalase, Gotl, HATPase c, HSF DNA-bind, HSP20, HisKA, Homeobox, Hpt, Isoamylase N, K-box, Lactamase B, Metallophos, MtN3 slv, NAF, NAM, NIF, Oxidored FMN, PAS, PDZ, PRA1, Peptidase CIS, Peptidase S10, Peptidase S41, Phytochrome, Peinase, Pkinase Tyr, Pyridoxal deC, RIO1, RRM 1, RTC, RTC insert, Ras, Response reg, SPC25, SPX, SRF-TF, Synaptobrevin, UPF0057, zf-C2H2, and zf-C3HC4.<br>
As used herein "promoter" means regulatory DNA for initializing transcription. A "plant promoter" is a promoter capable of initiating transcription in plant cells whether or not its origin is a plant cell, e.g. is it well known that Agrobacterium promoters are functional in plant cells. Thus, plant promoters include promoter DNA obtained from plants, plant viruses and bacteria such as Agrobacterium and Bradyrhizobium bacteria. Examples of promoters under developmental control include promoters that preferentially initiate transcription in certain tissues, such as leaves, roots, or seeds. Such promoters are referred to as "tissue preferred". Promoters that initiate transcription only in certain tissues are referred to as "tissue specific". A "cell type" specific promoter primarily drives expression in certain cell types in one or more organs, for example, vascular cells in roots or leaves. An "inducible" or "repressible" promoter is a promoter which is under environmental control. Examples of environmental conditions that may effect transcription by inducible promoters include anaerobic conditions, or certain chemicals, or the presence of light. Tissue specific, tissue preferred, cell type specific, and inducible promoters constitute the class of "non-constitutive" promoters. A "constitutive" promoter is a promoter which is active under most conditions.<br>
As used herein "operably linked" means the association of two or more DNA fragments in a DNA construct so that the function of one, e.g. protein-encoding DNA, is controlled by the other, e.g. a promoter.<br>
As used herein "expressed" means produced, e.g. a protein is expressed in a plant cell when its cognate DNA is transcribed to mRNA that is translated to the protein.<br>
As used herein a "control plant" means a plant that does not contain the recombinant DNA that expressed a protein that impart an enhanced trait. A control plant is to identify and select a transgenic plant that has an enhance trait. A suitable control plant can be a non-transgenic plant of the parental line used to generate a transgenic plant, i.e. devoid of recombinant DNA. A suitable control plant may in some cases be a progeny of a hemizygous transgenic plant line that is does not contain the recombinant DNA, known as a negative segregant.<br>
As used herein an "enhanced trait" means a characteristic of a transgenic plant that includes, but is not limited to, an enhance agronomic trait characterized by enhanced plant<br>
morphology, physiology, growth and development, yield, nutritional enhancement, disease or pest resistance, or environmental or chemical tolerance. In more specific aspects of this invention enhanced trait is selected from group of enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil. In an important aspect of the invention the enhanced trait is enhanced yield including increased yield under non-stress conditions and increased yield under environmental stress conditions. Stress conditions may include, for example, drought, shade, fungal disease, viral disease, bacterial disease, insect infestation, nematode infestation, cold temperature exposure, heat exposure, osmotic stress, reduced nitrogen nutrient availability, reduced phosphorus nutrient availability and high plant density.   "Yield" can be affected by many properties including without limitation, plant height, pod number, pod position on the plant, number of internodes, incidence of pod shatter, grain size, efficiency of nodulation and nitrogen fixation, efficiency of nutrient assimilation, resistance to biotic and abiotic stress, carbon assimilation, plant architecture, resistance to lodging, percent seed germination, seedling vigor, and juvenile traits. Yield can also affected by efficiency of germination (including germination in stressed conditions), growth rate (including growth rate in stressed conditions), ear number, seed number per ear, seed size, composition of seed (starch, oil, protein) and characteristics of seed fill.<br>
Increased yield of a transgenic plant of the present invention can be measured in a number of ways, including test weight, seed number per plant, seed weight, seed number per unit area (i.e. seeds, or weight of seeds, per acre), bushels per acre, tonnes per acre, tons per acre, kilo per hectare. For example, maize yield may be measured as production of shelled corn kernels per unit of production area, for example in bushels per acre or metric tons per hectare, often reported on a moisture adjusted basis, for example at 15.5 percent moisture. Increased yield may result from improved utilization of key biochemical compounds, such as nitrogen, phosphorous and carbohydrate, or from improved responses to environmental stresses, such as cold, heat, drought, salt, and attack by pests or pathogens. Recombinant DNA used in this invention can also be used to provide plants having improved growth and development, and ultimately increased yield, as the result of modified expression of plant growth regulators or modification of cell cycle or photosynthesis pathways. Also of interest is the generation of transgenic plants that demonstrate enhanced yield with respect to a seed component that may or may not correspond to an increase in overall plant yield. Such properties include enhancements in seed oil, seed molecules such as tocopherol, protein and<br>
starch, or oil particular oil components as may be manifest by an alterations in the ratios of seed components.<br>
A subset of the nucleic molecules of this invention includes fragments of the disclosed recombinant DNA consisting of oligonucleotides of at least 15, preferably at least 16 or 17, more preferably at least 18 or 19, and even more preferably at least 20 or more, consecutive nucleotides. Such oligonucleotides are fragments of the larger molecules having a sequence selected from the group consisting of SEQ ID NO:1 through SEQ ID NO:83, and find use, for example as probes and primers for detection of the polynucleotides of the present invention.<br>
In some embodiments of the present invention, a dominant negative mutant of a native gene is generated to achieve the desired effect. As used herein, "dominant negative mutant" means a mutant gene whose gene product adversely affects the normal, wild-type gene product within the same cell, usually by dimerizing (combining) with it. In cases of polymeric molecules, such as collagen, dominant negative mutations are often more deleterious than mutations causing the production of no gene product (null mutations or null alleles). SEQ ID NO: 6 and SEQ ID NO: 7 are constructed to encode agll 1 protein with K-box deleted and MADs 3 protein with MAD box deleted, respectively. MADS box proteins similar to AGL11 can be considered as having three functional domains. There is an N-terminal DNA-binding domain (the MADS box), a more distal dimerization domain (the K-box) and a C-terminal domain that is usually involved in interactions with other proteins. In plants the region between the MADS box and the K-box has been shown to be important for DNA binding in some proteins and is often referred to as the I-box (Fan et al., 1997).Several different classes of dominant negative constructs are considered. Deletion or inactivation of the DNA-binding domain can create proteins that are able to dimerize with their native full length counterparts as well as other natural dimerization partners. Likewise, removal of the C-terminal domain can allow dimerization with both the native protein and it's natural dimerization partners. In both cases these types of constructs disable both the target protein and any other protein capable of interacting with the K-box.<br>
hi other embodiments of the invention a constitutively active mutant is constructed to achieve the desired effect. SEQ ID NO:3 encodes only the kinase domain from a calcium-dependent protein kinase (CDPK). CDPK1 has a domain structure similar to other calcium-dependant protein kinases in which the protein kinase domain is separated from four efhand domains by 42 amino acid "spacer" region. Calcium-dependant protein kinases are thought to be activated by a calcium-induced conformational change that results in movement of anautoinhibitory domain away from the protein kinase active site (Yokokura et al., 1995). Thus, constitutively active proteins can be made by over expressing the protein kinase domain alone.<br>
DNA constructs are assembled using methods well known to persons of ordinary skill in the art and typically comprise a promoter operably linked to DNA, the expression of which provides the enhanced agronomic trait. Other construct components may include additional regulatory elements, such as 5' leasders and introns for enhancing transcription, 3' untranslated regions (such as polyadenylation signals and sites), DNA for transit or signal peptides.<br>
Numerous promoters that are active in plant cells have been described in the literature. These include promoters present in plant genomes as well as promoters from other sources, including nopaline synthase (NOS) promoter and octopine synthase (OCS) promoters carried on tumor-inducing plasmids of Agrobacterium tumefaciens, caulimovirus promoters such as the cauliflower mosaic virus. For instance, see U.S. Patents No. 5,858,742 and 5,322,938, which disclose versions of the constitutive promoter derived from cauliflower mosaic virus (CaMV35S), U.S. Patent 5,641,876, which discloses a rice actin promoter, U.S. Patent Application Publication 2002/0192813A1, which discloses 5', 3' and intron elements useful in the design of effective plant expression vectors, U.S. patent application Serial No. 09/757,089, which discloses a maize chloroplast aldolase promoter, U.S. patent application Serial No. 08/706,946, which discloses a rice glutelin promoter, U.S. patent application Serial No.09/757,089, which discloses a maize aldolase (FDA) promoter, and U.S. patent application Serial No.60/310, 370, which discloses a maize nicotianamine synthase promoter, all of which are incorporated herein by reference. These and numerous other promoters that function in plant cells are known to those skilled in the art and available for use in recombinant polynucleotides of the present invention to provide for expression of desired genes in transgenic plant cells.<br>
In some aspects of the invention, sufficient expression in plant seed tissues is desired to effect improvements in seed composition. Exemplary promoters for use for seed composition modification include promoters from seed genes such as napin (U.S. Patent 5,420,034), zein Z27 and glutelinl (Russell et al. (1997) Transgenic Res. 6(2): 157-166), and peroxiredoxin antioxidant (Perl) (Stacy et al. (1996) Plant Mol Biol. 31(6):1205-1216).,maize L3 oleosin (U.S. 6,433,252), globulin 1 (Belanger et al (1991) Genetics 129:863-872).<br>
In other aspects of the invention, preferential expression in plant green tissues is<br><br>
desired. Promoters of interest for such uses include those from genes such as Arabidopsis thaliana ribulose-l,5-bisphosphate carboxylase (Rubisco) small subunit (Fischhoff et al (1992) Plant Mol Biol. 20:81-93), aldolase and pyruvate orthophosphate dikinase (PPDK) (Taniguchi et al. (2000) Plant Cell Physiol. 41(l):42-48).<br>
Furthermore, the promoters may be altered to contain multiple "enhancer sequences" to assist in elevating gene expression. Such enhancers are known in the art. By including an enhancer sequence with such constructs, the expression of the selected protein may be enhanced. These enhancers often are found 5' to the start of transcription in a promoter that functions in eukaryotic cells, but can often be inserted upstream (5') or downstream (3') to the coding sequence. In some instances, these 5' enhancing elements are introns. Particularly useful as enhancers are the 5' introns of the rice actin 1 (see US Patent 5,641,876)and rice actin 2 genes, the maize alcohol dehydrogenase gene intron, the maize heat shock protein 70 gene intron (U.S. Patent 5,593,874) and the maize shrunken 1 gene.<br>
In other aspects of the invention, sufficient expression in plant seed tissues is desired to effect improvements in seed composition. Exemplary promoters for use for seed composition modification include promoters from seed genes such as napin (U.S. 5,420,034), maize L3 oleosin (U.S. 6,433,252), zein Z27 (Russell et al. (1997) Transgenic Res. 6(2): 157-166), globulin 1 (Belanger etal (1991) Genetics 129:863-872), glutelin 1 (Russell (1997) supra), and peroxiredoxin antioxidant (Perl) (Stacy et al. (1996) Plant Mol Biol. 31(6): 1205-1216).<br>
Recombinant DNA constructs prepared in accordance with the invention will also generally include a 3' element that typically contains a polyadenylation signal and site. Well-known 3' elements include those from Agrobacterium tumefaciens genes such as nos 3', tml 3', tmr 3', tms 3', ocs 3', tr7 3', for example disclosed in U.S. 6,090,627, incorporated herein by reference; 3' elements from plant genes such as wheat (Triticum aesevitum) heat shock protein 17 (Hsp17 3'), a wheat ubiquitin gene, a wheat fructose-1,6-biphosphatase gene, a rice glutelin gene a rice lactate dehydrogenase gene and a rice beta-tubulin gene, all of which are disclosed in U.S. published patent application 2002/0192813 Al, incorporated herein by reference; and the pea (Pisum sativum) ribulose biphosphate carboxylase gene (rbs 3'), and 3' elements from the genes within the host plant.<br>
Constructs and vectors may also include a transit peptide for targeting of a gene target to a plant organelle, particularly to a chloroplast, leucoplast or other plastid organelle. For descriptions of the use of chloroplast transit peptides see U.S. Patent 5, 188,642 and U.S. Patent No. 5,728,925, incorporated herein by reference. For description of the transit peptide<br><br>
region of an Arabidopsis EPSPS gene useful in the present invention, see Klee, H.J. et al (MGG (1987) 210:437-442).<br>
Transgenic plants comprising or derived from plant cells of this invention transformed with recombinant DNA can be further enhanced with stacked traits, e.g. a crop plant having an enhanced trait resulting from expression of DNA disclosed herein in combination with herbicide and/or pest resistance traits. For example, genes of the current invention can be stacked with other traits of agronomic interest, such as a trait providing herbicide resistance, or insect resistance, such as using a gene from Bacillus thuringensis to provide resistance against lepidopteran, coliopteran, homopteran, hemiopteran, and other insects. Herbicides for which transgenic plant tolerance has been demonstrated and the method of the present invention can be applied include, but are not limited to, glyphosate, dicamba, glufosinate, sulfonylurea, bromoxynil and norflurazon herbicides. Polynucleotide molecules encoding proteins involved in herbicide tolerance are well-known in the art and include, but are not limited to, a polynucleotide molecule encoding 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) disclosed in U.S. Patent 5,094,945; 5,627,061; 5,633,435 and 6,040,497 for imparting glyphosate tolerance; polynucleotide molecules encoding a glyphosate oxidoreductase (GOX) disclosed in U.S. Patent 5,463,175 and a glyphosate-N-acetyl transferase (GAT) disclosed in U.S. Patent Application publication 2003/0083480 Al also for imparting glyphosate tolerance; dicamba monooxygenase disclosed in U.S. Patent Application publication 2003/0135879 Al for imparting dicamba tolerance; a polynucleotide molecule encoding bromoxynil nitrilase (Bxn) disclosed in U.S. Patent 4,810,648 for imparting bromoxynil tolerance; a polynucleotide molecule encoding phytoene desaturase (crtl) described in Misawa et al, (1993) Plant J. 4:833-840 and Misawa et al, (1994) Plant J. 6:481-489 for norflurazon tolerance; a polynucleotide molecule encoding acetohydroxyacid synthase (AHAS, aka ALS) described in Sathasiivan et al. (1990) Nucl. Acids Res. 18:2188-2193 for imparting tolerance to sulfonylurea herbicides; polynucleotide molecules known as bar genes disclosed in DeBlock, et al (1987) EMBO J. 6:2513-2519 for imparting glufosinate and bialaphos tolerance; polynucleotide molecules disclosed in U.S. Patent Application Publication 2003/010609 Al for imparting N-amino methyl phosphonic acid tolerance; polynucleotide molecules disclosed in U.S. Patent 6,107,549 for impartinig pyridine herbicide resistance; molecules and methods for imparting tolerance to multiple herbicides such as glyphosate, atrazine, ALS inhibitors, isoxoflutole and glufosinate herbicides are disclosed in U.S. Patent 6,376,754 and U.S. Patent Application Publication 2002/0112260, all of said U.S. Patents and Patent Application Publications are incorporated<br><br>
herein by reference.   Molecules and methods for imparting insect/nematode/virus resistance is disclosed in U.S. Patents 5,250,515; 5,880,275; 6,506,599; 5,986,175 and U.S. Patent Application Publication 2003/0150017 Al, all of which are incorporated herein by reference.<br>
In particular embodiments, the inventors contemplate the use of antibodies, either monoclonal or polyclonal which bind to the proteins disclosed herein. Means for preparing and characterizing antibodies are well known in the art (See, e.g., Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988; incorporated herein by reference). The methods for generating monoclonal antibodies (mAbs) generally begin along the same lines as those for preparing polyclonal antibodies. Briefly, a polyclonal antibody is prepared by immunizing an animal with an immunogenic composition in accordance with the present invention and collecting antisera from that immunized animal. A wide range of animal species can be used for the production of antisera. Typically the animal used for production of anti-antisera is a rabbit, a mouse, a rat, a hamster, a guinea pig or a goat. Because of the relatively large blood volume of rabbits, a rabbit is a preferred choice for production of polyclonal antibodies.<br>
As is well known in the art, a given composition may vary in its immunogenicity. It is often necessary therefore to boost the host immune system, as may be achieved by coupling a peptide or polypeptide immunogen to a carrier. Exemplary and preferred carriers are keyhole limpet hemocyanin (KLH) and bovine serum albumin (BSA). Other albumins such as ovalbumin, mouse serum albumin or rabbit serum albumin can also be used as carriers. Means for conjugating a polypeptide to a carrier protein are well known in the art and include using glutaraldehyde, m-maleimidobencoyl-N-hydroxysuccinimide ester, carbodiimide and bis-biazotized benzidine.<br>
As is also well known in the art, the immunogenicity of a particular immunogen composition can be enhanced by the use of non-specific stimulators of the immune response, known as adjuvants. Exemplary and preferred adjuvants include complete Freund's adjuvant (a non-specific stimulator of the immune response containing killed Mycobacterium tuberculosis), incomplete Freund's adjuvants and aluminum hydroxide adjuvant.<br>
The amount of immunogen composition used in the production of polyclonal antibodies varies upon the nature of the immunogen as well as the animal used for immunization. A variety of routes can be used to administer the immunogen (subcutaneous, intramuscular, intradermal, intravenous and intraperitoneal). The production of polyclonal antibodies may be monitored by sampling blood of the immunized animal at various points following immunization. A second, booster, injection may also be given. The process of<br><br>
boosting and titering is repeated until a suitable liter is achieved. When a desired level of immunogenicity is obtained, the immunized animal can be bled and the serum isolated and stored, and/or the animal can be used to generate mAbs.<br>
mAbs may be readily prepared through use of well-known techniques, such as those exemplified in U.S. Pat. No. 4,196,265, incorporated herein by reference. Typically, this technique involves immunizing a suitable animal with a selected immunogen composition, e.g., a purified or partially purified antifungal protein, polypeptide or peptide. The immunizing composition is administered in a manner effective to stimulate antibody producing cells. Rodents such as mice and rats are preferred animals, however, the use of rabbit, sheep, or frog cells is also possible. The use of rats may provide certain advantages (Coding, 1986, pp. 60-61), but mice are preferred, with the BALB/c mouse being most preferred as this is most routinely used and generally gives a higher percentage of stable fusions.<br>
Following immunization, somatic cells with the potential for producing antibodies, specifically B lymphocytes (B cells), are selected for use in the mAb generating protocol. These cells may be obtained from biopsied spleens, tonsils or lymph nodes, or from a peripheral blood sample. Spleen cells and peripheral blood cells are preferred, the former because they are a rich source of antibody-producing cells that are in the dividing plasmablast stage, and the latter because peripheral blood is easily accessible. Often, a panel of animals will have been immunized and the spleen of animal with the highest antibody titer will be removed and the spleen lymphocytes obtained by homogenizing the spleen with a syringe. Typically, a spleen from an immunized mouse contains approximately 5xl07 to 2xl08 lymphocytes.<br>
The antibody-producing B lymphocytes from the immunized animal are then fused with cells of an immortal myeloma cell, generally one of the same species as the animal that was immunized. Myeloma cell lines suited for use in hybridoma-producing fusion procedures preferably are non-antibody-producing, have high fusion efficiency, and enzyme deficiencies that render them incapable of growing in certain selective media which support the growth of only the desired fused cells (hybridomas).<br>
Any one of a number of myeloma cells may be used, as are known to those of skill in the art (Coding, 1986, pp. 65-66; Campbell, 1984, pp. 75-83). For example, where the immunized animal is a mouse, one may use P3-X63/Ag8, X63-Ag8.653, NSl/l.Ag 4 1, Sp210-Agl4, FO, NSO/U, MPC-11, MPC11-X45-GTG 1.7 and S194/5XXO Bui; for rats, one may use R210.RCY3, Y3-Ag 1.2.3, IR983F and 4B210; and U-266, GM1500-GRG2,<br><br>
LICR-LON- HMy2 and UC729-6 are all useful in connection with human cell fusions.<br>
One preferred murine myeloma cell is the NS-1 myeloma cell line (also termed P3-NS-l-Ag4-l), which is readily available from the NIGMS Human Genetic Mutant Cell Repository by requesting cell line repository number GM3573. Another mouse myeloma cell line that may be used is the 8-azaguanine-resistant mouse murine myeloma SP2/0 non-producer cell line.<br>
Methods for generating hybrids of antibody-producing spleen or lymph node cells and myeloma cells usually comprise mixing somatic cells with myeloma cells in a 2:1 ratio, though the ratio may vary from about 20:1 to about 1:1, respectively, in the presence of an agent or agents (chemical or electrical) that promote the fusion of cell membranes. Fusion methods using Spend virus have been described (Kohler and Milstein, 1975; 1976), and those using polyethylene glycol (PEG), such as 37% (v/v) PEG, (Gefter et al., 1977). The use of electrically induced fusion methods is also appropriate (Coding, 1986, pp. 71-74).<br>
Fusion procedures usually produce viable hybrids at low frequencies, about IxlO"6 to<br>
n<br>
1x10" . However, this does not pose a problem, as the viable, fused hybrids are differentiated from the parental, unfused cells (particularly the unfused myeloma cells that would normally continue to divide indefinitely) by culturing in a selective medium. The selective medium is generally one that contains an agent that blocks the de novo synthesis of nucleotides in the tissue culture media. Exemplary and preferred agents are aminopterin, methotrexate, and azaserine. Aminopterin and methotrexate block de novo synthesis of both purines and pyrimidines, whereas azasenne blocks only purine synthesis. Where aminopterin or methotrexate is used, the media is supplemented with hypoxanthine and thymidine as a source of nucleotides (HAT medium). Where azaserine is used, the media is supplemented with hypoxanthine.<br>
The preferred selection medium is HAT. Only cells capable of operating nucleotide salvage pathways are able to survive in HAT medium. The myeloma cells are defective in key enzymes of the salvage pathway, e.g., hypoxanthine phosphoribosyl transferase (HPRT), and they cannot survive. The B-cells can operate this pathway, but they have a limited life span in culture and generally die within about two weeks. Therefore, the only cells that can survive in the selective media are those hybrids formed from myeloma and B-cells.<br>
This culturing provides a population of hybridomas from which specific hybridomas are selected. Typically, selection of hybridomas is performed by culturing the cells by single-clone dilution in microtiter plates, followed by testing the individual clonal supernatants (after about two to three weeks) for the desired reactivity. The assay should be sensitive,<br><br>
simple and rapid, such as radioimmunoassays, enzyme immunoassays, cytotoxicity assays, plaque assays, dot immunobinding assays, and the like.<br>
The selected hybridomas would then be serially diluted and cloned into individual antibody-producing cell lines, which clones can then be propagated indefinitely to provide mAbs. The cell lines may be exploited for mAb production in two basic ways. A sample of the hybridoma can be injected (often into the peritoneal cavity) into a histocompatible animal of the type that was used to provide the somatic and myeloma cells for the original fusion. The injected animal develops tumors secreting the specific monoclonal antibody produced by the fused cell hybrid. The body fluids of the animal, such as serum or ascites fluid, can then be tapped to provide mAbs in high concentration. The individual cell lines could also be cultured in vitro, where the mAbs are naturally secreted into the culture medium from which they can be readily obtained in high concentrations. mAbs produced by either means may be further purified, if desired, using filtration, centrifugation and various chromatographic methods such as HPLC or affinity chromatography.<br>
Plant Cell Transformation Methods<br>
Numerous methods for transforming plant cells with recombinant DNA are known in the art and may be used in the present invention. Two commonly used methods for plant transformation are Agrobacterium-mediated transformation and microprojectile bombardment. Microprojectile bombardment methods are illustrated in U.S. Patents 5,015,580 (soybean); 5,550,318 (corn); 5,538,880 (corn); 5,914,451 (soybean); 6,160,208 (corn); 6,399,861 (corn) and 6,153,812 (wheat) and Agrobacterium-mediated transformation is described in U.S. Patents 5,159,135 (cotton); 5,824,877 (soybean); 5,591,616 (corn); and 6,384,301 (soybean), all of which are incorporated herein by reference.    For Agrobacterium tumefaciens based plant transformation system, additional elements present on transformation constructs will include T-DNA left and right border sequences to facilitate incorporation of the recombinant polynucleotide into the plant genome.<br>
In general it is useful to introduce recombinant DNA randomly, i.e. at a non-specific location, in the genome of a target plant line. In special cases it may be useful to target recombinant DNA insertion in order to achieve site-specific integration, for example to replace an existing gene in the genome, to use an existing promoter in the plant genome, or to insert a recombinant polynucleotide at a predetermined site known to be active for gene expression. Several site specific recombination systems exist which are known to function<br><br>
implants include cre-lox as disclosed in U.S. Patent 4,959,317 and FLP-FRT as disclosed in U.S. Patent 5,527,695, both incorporated herein by reference.<br>
Transformation methods of this invention are preferably practiced in tissue culture on media and in a controlled environment. "Media" refers to the numerous nutrient mixtures that are used to grow cells in vitro, that is, outside of the intact living organism. Recipient cell targets include, but are not limited to, meristem cells, callus, immature embryos and gametic cells such as microspores, pollen, sperm and egg cells. It is contemplated that any cell from which a fertile plant may be regenerated is useful as a recipient cell. Callus may be initiated from tissue sources including, but not limited to, immature embryos, seedling apical meristems, microspores and the like. Cells capable of proliferating as callus are also recipient cells for genetic transformation. Practical transformation methods and materials for making transgenic plants of this invention, for example various media and recipient target cells, transformation of immature embryo cells and subsequent regeneration of fertile transgenic plants are disclosed in U.S. Patents 6,194,636 and 6,232,526, which are incorporated herein by reference.<br>
The seeds of transgenic plants can be harvested from fertile transgenic plants and be used to grow progeny generations of transformed plants of this invention including hybrid plants line for selection of plants having an enhanced trait. In addition to direct transformation of a plant with a recombinant DNA, transgenic plants can be prepared by crossing a first plant having a recombinant DNA with a second plant lacking the DNA. For example, recombinant DNA can be introduced into first plant line that is amenable to transformation to produce a transgenic plant which can be crossed with a second plant line to introgress the recombinant DNA into the second plant line. A transgenic plant with recombinant DNA providing an enhanced trait, e.g. enhanced yield, can be crossed with transgenic plant line having other recombinant DNA that confers another trait, for example herbicide resistance or pest resistance, to produce progeny plants having recombinant DNA that confers both traits. Typically, in such breeding for combining traits the transgenic plant donating the additional trait is a male line and the transgenic plant carrying the base traits is the female line. The progeny of this cross will segregate such that some of the plants will carry the DNA for both parental traits and some will carry DNA for one parental trait; such plants can be identified by markers associated with parental recombinant DNA, e.g. marker identification by analysis for recombinant DNA or, hi the case where a selectable marker is linked to the recombinant, by application of the selecting agent such as a herbicide for use with a herbicide tolerance marker, or by selection for the enhanced trait. Progeny plants<br><br>
carrying DNA for both parental traits can be crossed back into the female parent line multiple times, for example usually 6 to 8 generations, to produce a progeny plant with substantially the same genotype as one original transgenic parental line but for the recombinant DNA of the other transgenic parental line<br>
In the practice of transformation DNA is typically introduced into only a small percentage of target plant cells in any one transformation experiment. Marker genes are used to provide an efficient system for identification of those cells that are stably transformed by receiving and integrating a transgenic DNA construct into their genomes. Preferred marker genes provide selective markers which confer resistance to a selective agent, such as an antibiotic or herbicide. Any of the herbicides to which plants of this invention may be resistant are useful agents for selective markers. Potentially transformed cells are exposed to the selective agent. In the population of surviving cells will be those cells where, generally, the resistance-conferring gene is integrated and expressed at sufficient levels to permit cell survival. Cells may be tested further to confirm stable integration of the exogenous DNA. Commonly used selective marker genes include those conferring resistance to antibiotics such as kanamycin and paromomyein (nptll), hygromycin B (aph IV) and gentamycin (aac3 and oacC4) or resistance to herbicides such as glufosinate (bar or pat) and glyphosate (aroA or EPSPS).   Examples of such selectable are illustrated in U.S. Patents 5,550,318; 5,633,435; 5,780,708 and 6,118,047, all of which are incorporated herein by reference. Selectable markers which provide an ability to visually identify transformants can also be employed, for example, a gene expressing a colored or fluorescent protein such as a luciferase or green fluorescent protein (GFP) or a gene expressing a beta-glucuronidase or uidA gene (GUS) for which various chromogenic substrates are known.<br>
Plant cells that survive exposure to the selective agent, or plant cells that have been scored positive in a screening assay, may be cultured in regeneration media and allowed to mature into plants. Developing plantlets regenerated from transformed plant cells can be transferred to plant growth mix, and hardened off, for example, in an environmentally controlled chamber at about 85% relative humidity, 600 ppm COa, and 25-250 microeinsteins m2 s-1 of light, prior to transfer to a greenhouse or growth chamber for maturation. Plants are regenerated from about 6 weeks to 10 months after a transformant is identified, depending on the initial tissue. Plants may be pollinated using conventional plant breeding methods known to those of skill in the art and seed produced, for example self-pollination is commonly used with transgenic corn. The regenerated transformed plant or its progeny seed or plants can be<br>
ested for expression of the recombinant DNA and selected for the presence of enhanced<br>
agronomic trait.<br>
Transgenic Plants and Seeds<br>
Transgenic plants derived from the plant cells of this invention are grown to generate transgenic plants having an enhanced trait as compared to a control plant and produce transgenic seed and haploid pollen of this invention. Such plants with enhanced traits are identified by selection of transformed plants or progeny seed for the enhanced trait. For efficiency a selection method is designed to evaluate multiple transgenic plants (events) comprising the recombinant DNA , for example multiple plants from 2 to 20 or more transgenic events. Transgenic plants grown from transgenic seed provided herein demonstrate improved agronomic traits that contribute to increased yield or other trait that provides increased plant value, including, for example, improved seed quality. Of particular interest are plants having enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil.<br>
Table 2 provides a list of protein encoding DNA ("genes") that are useful as recombinant DNA for production of transgenic plants with enhanced agronomic trait, the elements of Table 2 are described by reference to:<br>
"PEP SEQ" which identifies an amino acid sequence from SEQ ID NO:84 to 166. "NUC SEQ" which identifies a DNA sequence from SEQ ID NO:1 to 83. "Base Vector" which identifies a base plasmid used for transformation of the recombinant DNA.<br>
"PROTEIN NAME" which is a common name for protein encoded by the recombinant DNA. "Enhanced trait" which identifies an enhanced trait which is imparted by the expression of the protein in a transgenic crop plant.<br>
"Plasmid ID" which identifies an arbitrary name for the plant transformation plasmid comprising recombinant DNA for expressing the recombinant DNA hi plant cells.<br>
Table 2<br>
(Table 2  Removed)<br>
Selection methods for transgenic plants with enhanced agronomic trait<br>
Within a population of transgenic plants regenerated from plant cells transformed with the recombinant DNA many plants that survive to fertile transgenic plants that produce seeds and progeny plants will not exhibit an enhanced agronomic trait. Selection from the population is necessary to identify one or more transgenic plant cells that can provide plants with the enhanced trait. Transgenic plants having enhanced traits are selected from populations of plants regenerated or derived from plant cells transformed as described herein by evaluating the plants in a variety of assays to detect an enhanced trait, e.g. enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil. These assays also may take many forms including, but not limited to, direct screening for the trait in a greenhouse or field trial or by screening for a surrogate trait. Such analyses can be directed to detecting changes in the chemical composition, biomass, physiological properties, morphology of the plant. Changes in chemical compositions such as nutritional composition of grain can be detected by analysis of the seed composition and content of protein, free amino acids, oil, free fatty acids, starch or tocopherols. Changes in biomass characteristics can be made on greenhouse or field grown plants and can include plant height, stem diameter, root and shoot dry weights; and, for corn plants, ear length and diameter. Changes in physiological properties can be identified by evaluating responses to stress conditions, for example assays using imposed stress conditions such as water deficit, nitrogen deficiency, cold growing conditions, pathogen or insect attack or light deficiency, or increased plant density. Changes in morphology can be measured by visual observation of tendency of a transformed plant with an enhanced agronomic trait to also appear to be a normal plant as compared to changes toward bushy, taller, thicker, narrower leaves, striped leaves, knotted trait, chlorosis, albino, anthocyanin production, or altered tassels, ears or roots. Other selection properties include days to pollen shed, days to silking, leaf extension rate, chlorophyll content, leaf temperature, stand, seedling vigor, internode length, plant height, leaf number, leaf area, tillering, brace roots, stay green, stalk lodging, root lodging, plant health, barreness/prolificacy, green snap, and pest resistance. In addition, phenotypic characteristics of harvested grain may be evaluated, including number of kernels per row on the ear, number of rows of kernels on the ear, kernel abortion, kernel weight, kernel size, kernel density and physical grain quality. Although the plant cells and methods of this invention can be applied to any plant cell, plant, seed or pollen, e.g. any fruit, vegetable, grass, tree or ornamental plant, the various aspects of<br>
the invention are preferably applied to corn, soybean, cotton, canola, alfalfa, wheat and rice plants. In many cases the invention is applied to corn plants that are inherently resistant to disease from the Mai de Rio Cuarto virus or the Puccina sorghi fungus or both.<br>
The following examples are included to demonstrate aspects of the invention, those of skill in the art should, in light of the present disclosure, appreciate that many changes can be made in the specific aspects which are disclosed and still obtain a like or similar results without departing from the spirit and scope of the invention.<br>
Example 1. Plant Expression Constructs<br>
A. Plant expression constructs for corn transformation<br>
This example illustrates the construction of plasmids for transferring recombinant DNA into plant cells which can be regenerated into transgenic plants of this invention. Primers for PCR amplification of protein coding nucleotides of recombinant DNA were designed at or near the start and stop codons of the coding sequence, in order to eliminate most of the 5' and 3' untranslated regions. Each recombinant DNA coding for a protein identified in Table 2 was amplified by PCR prior to insertion into the insertion site of one of the base vectors as referenced in Table 2.<br>
A base plant transformation vector pMON65154 was fabricated for use in preparing recombinant DNA for transformation into corn tissue using GATEWAYโข Destination plant expression vector systems (available from Invitrogen Life Technologies, Carlsbad, CA). With reference to the elements described in Table 3 below and SEQ ED NO: 167, pMON65154 comprises a selectable marker expression cassette and a template recombinant DNA expression cassette. The marker expression cassette comprises a CaMV 35S promoter operably linked to a gene encoding neomycin phosphotransferase II (nptll) followed by a 3' region of an Agrobacterium tumefaciens nopaline synthase gene (nos). The template recombinant DNA expression cassette is positioned tail to tail with the marker expression cassette. The template recombinant DNA expression cassette comprises 5' regulatory DNA including a rice actin 1 promoter, exon and intron, followed by a GATEWAYโข insertion site for recombinant DNA, followed by a 3' region of a potato proteinase inhibitor II (pinll) gene. Once recombinant DNA has been inserted into the insertion site, the plasmid is useful for plant transformation, for example by microprojectile bombardment.Table 3<br><br>
(Table3 Removed)<br>
A similar base vector plasmid pMON72472 (SEQ ID NO: 168) was constructed for use in Agrobacterium-mediated methods of plant transformation similar to pMON65154 except (a) the 5' regulatory DNA in the template recombinant DNA expression cassette was a rice actin promoter and a rice actin intron, (b) left and right T-DNA border sequences from Agrobacterium are added with the right border sequence is located 5' to the rice actin 1 promoter and the left border sequence is located 3' to the 35S promoter and (c) DNA is added to facilitate replication of the plasmid in both E. coli and Agrobacterium tumefaciens. The DNA added to the plasmid outside of the T-DNA border sequences includes an oriV wide host range origin of DNA replication functional in Agrobacterium, a pBR322 origin of replication functional in E.coli, and a spectinomycin/stretptomycin resistance gene for selection in both E. coli and Agrobacterium.<br>
Another base vector pMON82060 (SEQ ID NO: 169), illustrated in Table 4, was assembled using the technology known in the art.Table 4<br>
(Table 4 Removed)<br>
B. Plant expression vector for soybean transformation<br>
Plasmids for use in transformation of soybean were also prepared. Elements of an exemplary common expression vector plasmid pMON74532 (SEQ ID NO: 170) are shown in Table 5 below.<br>
Table 5<br>
(Table5 Removed)<br><br>
Another base vector pMON82053 (SEQ ID NO: 171), illustrated in Table 6, was assembled using the technology known in the art.<br><br>
Table 6<br>
(Table 6 Removed)<br>
Protein coding segments of recombinant DNA are amplified by PCR prior to insertion into vectors at the insertion site. Primers for PCR amplification are designed at or near the start and stop codons of the coding sequence, in order to eliminate most of the 5' and 3' untranslated regions.<br>
Example 2. Corn Transformation<br>
This example illustrates plant cell transformation methods useful in producing transgenic corn plant cells, plants, seeds and pollen of this invention and the production and identification of transgenic corn plants and seed with an enhanced trait, i.e. enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil. Plasmid vectors were prepared by cloning DNA identified in Table 2 in the identified base vectors for use in corn transformation of corn plant cells to produce transgenic corn plants and progeny plants, seed and pollen.<br>
For Agrobacterium-mediated transformation of corn embryo cells corn plants of a readily transformable line (designated LH59) is grown in the greenhouse and ears harvested when the embryos are 1.5 to 2.0 mm in length. Ears are surface sterilized by spraying or soaking the ears in 80% ethanol, followed by air drying. Immature embryos are isolated from individual kernels on surface sterilized ears. Prior to inoculation of maize cells, Agrobacterium cells are grown overnight at room temperature. Immature maize embryo cells are inoculated with Agrobacterium shortly after excision, and incubated at room temperature with Agrobacterium for 5-20 minutes. Immature embryo plant cells are then co-cultured with Agrobacterium for 1 to 3 days at 23ยฐC in the dark. Co-cultured embryos are transferred to selection media and cultured for approximately two weeks to allow embryogenic callus to<br>
develop. Embryogenic callus is transferred to culture medium containing 100 mg/L paromomycin and subcultured at about two week intervals. Transformed plant cells are recovered 6 to 8 weeks after initiation of selection.<br>
For Agrobacterium-mediated transformation of maize callus immature embryos are cultured for approximately 8-21 days after excision to allow callus to develop. Callus is then incubated for about 30 minutes at room temperature with the Agrobacterium suspension, followed by removal of the liquid by aspiration. The callus and Agrobacterium are co-cultured without selection for 3-6 days followed by selection on paromomycin for approximately 6 weeks, with biweekly transfers to fresh media, and paromomycin resistant callus identified as containing the recombinant DNA in an expression cassette.<br>
For transformation by microprojectile bombardment immature maize embryos are isolated and cultured 3-4 days prior to bombardment. Prior to microprojectile bombardment, a suspension of gold particles is prepared onto which the desired recombinant DNA expression cassettes are precipitated. DNA is introduced into maize cells as described in U.S. Patents 5,550,318 and 6,399,861 using the electric discharge particle acceleration gene delivery device. Following microprojectile bombardment, tissue is cultured in the dark at 27 degrees C. Additional transformation methods and materials for making transgenic plants of this invention, for example, various media and recipient target cells, transformation of immature embryos and subsequence regeneration of fertile transgenic plants are disclosed in U.S. Patents 6,194,636 and 6,232,526 and U.S. Patent application Serial No. 09/757,089, which are incorporated herein by reference.<br>
To regenerate transgenic corn plants a callus of transgenic plant cells resulting from transformation is placed on media to initiate shoot development in plantlets which are transferred to potting soil for initial growth in a growth chamber at 26 degrees C followed by a mist bench before transplanting to 5 inch pots where plants are grown to maturity. The regenerated plants are self fertilized and seed is harvested for use in one or more methods to select seed, seedlings or progeny second generation transgenic plants (R2 plants) or hybrids, e.g. by selecting transgenic plants exhibiting an enhanced trait as compared to a control plant.<br>
Transgenic corn plant cells were transformed with recombinant DNA from each of the genes identified in Table 2. Progeny transgenic plants and seed of the transformed plant cells were screened for enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil as reported in Example 5.<br>
Example 3. Soybean transformation<br>
This example illustrates plant transformation useful in producing the transgenic soybean plants of this invention and the production and identification of transgenic seed for transgenic soybean having enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil.<br>
For Agrobacterium mediated transformation, soybean seeds are germinated overnight and the meristem explants excised. The meristems and the explants are placed in a wounding vessel. Soybean explants and induced Agrobacterium cells from a strain containing plasmid DNA with the gene of interest cassette and a plant selectable marker cassette are mixed no later than 14 hours from the time of initiation of seed germination and wounded using sonication. Following wounding, explants are placed in co-culture for 2-5 days at which point they are transferred to selection media for 6-8 weeks to allow selection and growth of transgenic shoots. Trait positive shoots are harvested approximately 6-8 weeks and placed into selective rooting media for 2-3 weeks. Shoots producing roots are transferred to the greenhouse and potted in soil. Shoots that remain healthy on selection, but do not produce roots are transferred to non-selective rooting media for an additional two weeks. Roots from any shoots that produce roots off selection are tested for expression of the plant selectable marker before they are transferred to the greenhouse and potted in soil. Additionally, a DNA construct can be transferred into the genome of a soybean cell by particle bombardment and the cell regenerated into a fertile soybean plant as described in U.S. Patent 5,015,580, herein incorporated by reference.<br>
Transgenic soybean plant cells were transformed with recombinant DNA from each of the genes identified in Table 2. Progeny transgenic plants and seed of the transformed plant cells were screened for enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil as reported in Example 5.Example 4. Homolog Identification<br>
This example illustrates the identification of homologs of proteins encoded by the DNA identified in Table 2 which is used to provide transgenic seed and plants having enhanced agronomic traits. From the sequence of the homologs, homologous DNA sequence can be identified for preparing additional transgenic seeds and plants of this invention with enhanced agronomic traits.<br>
An "All Protein Database" was constructed of known protein sequences using a proprietary sequence database and the National Center for Biotechnology Information (NCBI) non-redundant amino acid database (nr.aa).  For each organism from which a polynucleotide sequence provided herein was obtained, an "Organism Protein Database" was constructed of known protein sequences of the organism; it is a subset of the All Protein Database based on the NCBI taxonomy ID for the organism.<br>
The All Protein Database was queried using amino acid sequences provided herein as SEQ ID NO:84 through SEQ ED NO: 166 using NCBI "blastp" program with E-value cutoff of le-8. Up to 1000 top hits were kept, and separated by organism names. For each organism other than that of the query sequence, a list was kept for hits from the query organism itself with a more significant E-value than the best hit of the organism. The list contains likely duplicated genes of the polynucleotides provided herein, and is referred to as the Core List. Another list was kept for all the hits from each organism, sorted by E-value, and referred to as the Hit List.<br>
The Organism Protein Database was queried using polypeptide sequences provided herein as SEQ ID NO:84 through SEQ ID NO: 166 using NCBI "blastp" program with E-value cutoff of le-4. Up to 1000 top hits were kept. A BLAST searchable database was constructed based on these hits, and is referred to as "SubDB". SubDB was queried with each sequence in the Hit List using NCBI "blastp" program with E-value cutoff of le-8. The hit with the best E-value was compared with the Core List from the corresponding organism. The hit is deemed a likely ortholog if it belongs to the Core List, otherwise it is deemed not a likely ortholog and there is no further search of sequences in the Hit List for the same organism.<br>
Table 2.<br>
Example 5. Selection of transgenic plants with enhanced agronomic trait(s)<br>
This example illustrates identification of plant cells of the invention by screening derived plants and seeds for enhanced trait. Transgenic com seed and plants with recombinant DNA identified in Table 2 were prepared by plant cells transformed with DNA that was stably integrated into the genome of the corn cell. The transgenic seed, plantlets and progeny plants were selected using the methods that measure Transgenic corn plant cells were transformed with recombinant DNA from each of the genes identified in Table 2. Progeny transgenic plants and seed of the transformed plant cells were screened for enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil as compared to control plants. A. Selection for enhanced Nitrogen Use Efficiency<br>
The physiological efficacy of transgenic corn plants (tested as hybrids) can be tested for nitrogen use efficiency (NUE) traits in a high-throughput nitrogen (N) selection method. The collected data are compared to the measurements from wildtype controls using a statistical model to determine if the changes are due to the transgene. Raw data were analyzed by SAS software. Results shown herein are the comparison of transgenic plants relative to the wildtype controls. (J) Media Preparation for Planting a NUE Protocol<br>
Planting materials used: Metro Mix 200 (vendor: Hummert) Cat. # 10-0325, Scotts Micro Max Nutrients (vendor: Hummert) Cat. # 07-6330, OS 4 1/3" x 3 7/8" pots (vendor: Hummert) Cat. # 16-1415, OS trays (vendor: Hummert) Cat. # 16-1515, Hoagland's macronutrients solution, Plastic 5" stakes (vendor: Hummert) yellow Cat. # 49-1569, white Cat. # 49-1505, Labels with numbers indicating material contained in pots. Fill 500 pots to rim with Metro Mix 200 to a weight of ~140g/pot. Pots are filled uniformly by using a balancer. Add 0.4g of Micro Max nutrients to each pot. Stir ingredients with spatula to a depth of 3 inches while preventing material loss. (2) Planting a NUE selection in the Greenhouse<br>
(a) Seed Germination - Each pot is lightly atered twice using reverse osmosis purified' water. The first watering is scheduled to occur just before planting; and the second watering, after the seed has been planted in the pot.   Ten Seeds of each entry (1 seed per pot) are planted to select eight healthy uniform seedlings. Additional wild type controls are planted<br>
for use as border rows. Alternatively, 15 seeds of each entry (1 seed per pot) are planted to select 12 healthy uniform seedlings (this larger number of plantings is used for the second, or confirmation, planting). Place pots on each of the 12 shelves in the Conviron growth chamber for seven days. This is done to allow more uniform germination and early seedling growth. The following growth chamber settings are 25ยฐ C/day and 22ยฐ C/night, 14 hours light and ten hours dark, humidity ~ 80%, and light intensity -350 ยตmol/m2/s (at pot level). Watering is done via capillary matting similar to greenhouse benches with duration of ten minutes three times a day.<br>
(b) Seedling transfer - After seven days, the best eight or 12 seedlings for the first or confirmation pass runs, respectively, are chosen and transferred to greenhouse benches. The pots are spaced eight inches apart (center to center) and are positioned on the benches using the spacing patterns printed on the capillary matting. The Vattex matting creates a 384-position grid, randomizing all range, row combinations. Additional pots of controls are placed along the outside of the experimental block to reduce border effects.<br>
Plants are allowed to grow for 28 days under the low N run or for 23 days under the high N run. The macronutrients are dispensed in the form of a macronutrient solution (see composition below) containing precise amounts of N added (2mM NRjNOs for limiting N. selection and 20mM NRjNOs for high N selection runs). Each pot is manually dispensed 100ml of nutrient solution three times a week on alternate days starting at eight and ten days after planting for high N and low N runs, respectively. On the day of nutrient application, two 20 min waterings at 05:00 and 13:00 are skipped. The vattex matting should be changed every third run to avoid N accumulation and buildup of root matter. Table 7 shows the amount of nutrients in the nutrient solution for either the low or high nitrogen selection.<br>
Table 7<br><br>
(Table 7 Removed)<br>
Note: Adjust pH to 5.6 with HCI or KOH<br>
(c) Harvest Measurements and Data Collection - After 28 days of plant growth for low N runs and 23 days of plant growth for high N runs, the following measurements are taken (phenocodes in parentheses): total shoot fresh mass (g) (SFM) measured by Sartorius electronic balance, V6 leaf chlorophyll measured by Minolta SPAD meter (relative units) (LC), V6 leaf area (cm2) (LA) measured by a Li-Cor leaf area meter, V6 leaf fresh mass (g) (LFM) measured by Sartorius electronic balance, and V6 leaf dry mass (g) (LDM) measured by Sartorius electronic balance. Raw data were analyzed by SAS software. Results shown are the comparison of transgenic plants relative to the wildtype controls.<br>
To take a leaf reading, samples were excised from the V6 leaf. Since chlorophyll meter readings of com leaves are affected by the part of the leaf and the position of the leaf on the plant that is sampled, SPAD meter readings were done on leaf six of the plants. Three measurements per leaf were taken, of which the first reading was taken from a point one-half the distance between the leaf tip and the collar and halfway from the leaf margin to the midrib while two were taken toward the leaf tip. The measurements were restricted in the area from 1/2 to 3/4 of the total length of the leaf (from the base) with approximately equal spacing between them. The average of the three measurements was taken from the SPAD machine.<br>
Leaf fresh mass is recorded for an excised V6 leaf, the leaf is placed into a paper bag. The paper bags containing the leaves are then placed into a forced air oven at 80ยฐ C for 3 days. After 3 days, the paper bags are removed from the oven and the leaf dry mass measurements are taken.<br>
From the collected data, two derived measurements are made: (l)Leaf chlorophyll area (LCA), which is a product of V6 relative chlorophyll content and its leaf area (relative<br>
units). Leaf chlorophyll area= leaf chlorophyll X leaf area. This parameter gives an indication of the spread of chlorophyll over the entire leaf area; (2)specific leaf area (LSA) is calculated as the ratio of V6 leaf area to its dry mass (cm2/g dry mass), a parameter also recognized as a measure of NUE. The data are shown in Table 8.<br>
Table 8.<br>
(Table 8 Removed)<br>
Nitrogen use field efficacy assay<br>
Level I. Transgenic plants provided by the present invention are planted in field without any nitrogen source being applied. Transgenic plants and control plants are grouped by genotype and construct with controls arranged randomly within genotype blocks. Each type of transgenic plants are tested by 3 replications and across 5 locations. Nitrogen levels in the fields are analyzed in early April pre-planting by collecting 30 sample soil cores from 0-24" and 24 to 48" soil layer. Soil samples are analyzed for nitrate-nitrogen, phosphorus(P), Potassium(K), organic matter and pH to provide baseline values. P, K and micronutrients are applied based upon soil test recommendations.<br>
Level II. Transgenic plants provided by the present invention are planted in field with three levels of nitrogen (N) fertilizer being applied, i.e. low level (0 N), medium level (80 Ib/ac) and high level (180 Ib/ac). Liquid 28% or 32% UAN (Urea, Ammonium Nitrogen) are used as the N source and apply by broadcast boom and incorporate with a field cultivator with rear rolling basket in the same direction as intended crop rows. Although there is no N applied to the 0 N treatment the soil should still be disturbed in the same fashion as the treated area. Transgenic plants and control plants are grouped by genotype and construct with controls arranged randomly within genotype blocks. Each type of transgenic plants is tested by 3 replications and across 4 locations. Nitrogen levels in the fields are analyzed in early April pre-planting by collecting 30 sample soil cores from 0-24" and 24 to 48" soil layer. Soil samples are analyzed for nitrate-nitrogen, phosphorus(P), Potassium(K), organic matter and pH to provide baseline values. P, K and micronutrients are applied based upon soil test reco<br>
Table 9. Genes increase seed yield in transgenic plants at different nitrogen levels.<br><br>
(Table 9 Removed)<br>
B. Selection for increased yield<br>
Many transgenic plants of this invention exhibit improved yield as compared to a control plant. Improved yield can result from enhanced seed sink potential, i.e. the number and size of endosperm cells or kernels and/or enhanced sink strength, i.e. the rate of starch biosynthesis. Sink potential can be established very early during kernel development, as endosperm cell number and size are determined within the first few days after pollination.<br>
Much of the increase in corn yield of the past several decades has resulted from an increase in planting density. During that period, corn yield has been increasing at a rate of 2.1 bushels/acre/year, but the planting density has increased at a rate of 250 plants/acre/year. A characteristic of modern hybrid corn is the ability of these varieties to be planted at high density. Many studies have shown that a higher than current planting density should result in more biornass production, but current germplasm does not perform, well at these higher densities. One approach to increasing yield is to increase harvest index (HI), the proportion of biomass that is allocated to the kernel compared to total biomass, in high density plantings.<br>
Effective yield selection of enhanced yielding transgenic corn events uses hybrid progeny of the transgenic event over multiple locations with plants grown under optimal production management practices, and maximum pest control. A useful target for improved yield is a 5% to 10% increase in yield as compared to yield produced by plants grown from seed for a control plant. Selection methods may be applied in multiple and diverse geographic locations, for example up to 16 or more locations, over one or more plating seasons, for example at least two planting seasons to statistically distinguish yield improvement from natural environmental effects. It is to plant multiple transgenic plants, positive and negative control plants, and pollinator plants in standard plots, for example 2 row plots, 20 feet long by 5 feet wide with 30 inches distance between rows and a 3 foot alley between ranges. Transgenic events can be grouped by recombinant DNA constructs with groups randomly placed in the field. A pollinator plot of a high quality corn line is planted for every two plots to allow open pollination when using male sterile transgenic events. A useful planting density is about 30,000 plants/acre. High planting density is greater than 30,000 plants/acre, preferably about 40,000 plants/acre, more preferably about 42,000 plants/acre, most preferably about 45,000 plants/acre. Surrogate indicators for yield improvement include source capacity (biomass), source output (sucrose and photosynthesis), sink components (kernel size, ear size, starch in the seed), development (light response, height, density tolerance), maturity, early flowering trait and physiological responses to high density planting, for example at 45,000 plants per acre, for example as illustrated in Table 10 and 11.<br>
Table 10<br><br>
(Table 10 Removed)<br>
Electron transport rates (ETR) and CO2 exchange rates (CER): ETR and CER were measured with U6400LCF (Licor, Lincoln, NE) around V9-R1 stages. Leaf chlorophyll fluorescence is a quick way to monitor the source activity and was reported to be highly correlated with CO2 assimilation under varies conditions (Photosyn Research, 37: 89-102). The youngest fully expanded leaf or 2 leaves above the ear leaf was measured with actinic light 1500 (with 10% blue light) micromol m-2 s-1, 28oC, CO2 levels 450ppm. Ten plants were measured in each event. There were 2 readings for each plant.<br>
A hand-held chlorophyll meter SPAD-502 (Minolta - Japan) was used to measure the total chlorophyll level on live transgenic plants and the wild type counterparts a. Three trifoliates from each plant were analyzed, and each trifoliate were analyzed three times. Then 9 data points were averaged to obtain the chlorophyll level. The number of analyzed plants of each genotype ranged from 5 to 8.<br>
Table 12<br><br>
(Table 12 Removed)<br>
When selecting for yield improvement a useful statistical measurement approach comprises three components, i.e. modeling spatial autocorrelation of the test field separately for each location, adjusting traits of recombinant DNA events for spatial dependence for each<br>
location, and conducting an across location analysis.  The first step in modeling spatial autocorrelation is estimating the covariance parameters of the semivariogram. A spherical covariance model is assumed to model the spatial autocorrelation. Because of the size and<br>
nature of the trial, it is likely that the spatial autocorrelation may change. Therefore,<br>
anisotropy is also assumed along with spherical covariance structure. The following set of<br>
equations describes the statistical form of the anisotropic spherical covariance model.<br>
(Formula Removed)<br>
where Sj = (x\, y\) are the spatial coordinates of one location and 82 = fe, ya) are the spatial coordinates of the second location. There are 5 covariance parameters, 6 = (v,a2,p,(On,coj) ,<br>
where v is the nugget effect, o2 is the partial sill, p is a rotation in degrees clockwise from north, a^ is a scaling parameter for the minor axis and cq is a scaling parameter for the major axis of an anisotropical ellipse of equal covariance. The five covariance parameters that defines the spatial trend will then be estimated by using data from heavily replicated pollinator plots via restricted maximum likelihood approach. In a multi-location field trial, spatial trend are modeled separately for each location.<br>
After obtaining the variance parameters of the model, a variance-covariance structure is generated for the data set to be analyzed. This variance-covariance structure contains spatial information required to adjust yield data for spatial dependence. In this case, a nested model that best represents the treatment and experimental design of the study is used along with the variance-covariance structure to adjust the yield data. During this process the nursery or the seed batch effects can also be modeled and estimated to adjust the yields for any yield parity caused by seed batch differences.   After spatially adjusted data from different locations are generated, all adjusted data is combined and analyzed assuming locations as replications. In this analysis, intra and inter-location variances are combined to estimate the standard error of yield from transgenic plants and control plants. Relative mean<br>
comparisons are used to indicate statistically significant yield improvements.<br>
Table 14.<br>
(Table 14 Removed)<br>
C. Selection for enhanced water use efficiency (WUE)<br>
Described in this example is a high-throughput method for greenhouse selection of transgenic corn plants to wild type corn plants (tested as inbreds or hybrids) for water use efficiency. This selection process imposes 3 drought/re-water cycles on plants over a total period of 15 days after an initial stress free growth period of 11 days. Each cycle consists of 5 days, with no water being applied for the first four days and a water quenching on the 5th day of the cycle. The primary phenotypes analyzed by the selection method are the changes in plant growth rate as determined by height and biomass during a vegetative drought treatment. The hydration status of the shoot tissues following the drought is also measured. The plant height are measured at three time points. The first is taken just prior to the onset drought when the plant is 11 days old, which is the shoot initial height (SIH). The plant height is also measured halfway throughout the drought/re-water regimen, on day 18 after planting, to give rise to the shoot mid-drought height (SMH). Upon the completion of the final drought cycle on day 26 after planting, the shoot portion of the plant is harvested and measured for a final height, which is the shoot wilt height (SWH) and also measured for shoot wilted biomass (SWM). The shoot is placed in water at 40 degree Celsius in the dark. Three days later, the shoot is weighted to give rise to the shoot turgid weight (STM). After drying in an oven for four days, the shoots are weighted for shoot dry biomass (SDM). The shoot average height (SAH) is the mean plant height across the 3 height measurements. The procedure described above may be adjusted for +/- ~ one day for each step given the situation.<br>
To correct for slight differences between plants, a size corrected growth value is derived from SIH and SWH. This is the Relative Growth Rate (RGR). Relative Growth Rate (RGR) is calculated for each shoot using the formula   [RGR% = (SWH-SIH)/((SWH+SIH)/2)* 100].   Relative water content (RWC) is a measurement of how much (%) of the plant was water at harvest. Water Content (RWC) is calculated for each shoot using the formula [RWC% = (SWM-SDM)/(STM-SDM)*100]. Fully watered com plants of this age run around 98% RWC.<br>
The transgenic plants provided by this invention were selected through the selection process according to the standard procedure described above and the performance of these transgenic plants are shown in Table 16 below.<br>
Table 16.<br><br>
(Table 16 Removed)<br>
Transgenic plants transformed with pMON67754 comprising the recombinant DNA as set forth in SEQ ID NO: 3 were tested in field with moderate drought conditions in Satanta, IL and Dixon CA. SPAD readings on leaves under a moderate drought stress showed a significant increase in chlorophyll level in the transgenic plants as compared to the control plants. Two events showed a significant increase in SPAD reading for chlorophyll level, indicating an improvement in drought tolerance. In replicated field trials, 2 events (ZM_M16396 and ZM_M16401) out of 6 tested, showed significantly (p
P. Selection for Growth Under Cold Stress<br>
(1) Cold germination assay - Three sets of seeds are used for the assay. The first set consists of positive transgenic events (Fl hybrid) where the genes of the present invention are expressed in the seed. The second seed set is nontransgenic, wild-type negative control made from the same genotype as the transgenic events. The third set consisted of two cold tolerant and one cold sensitive commercial check lines of corn. All seeds are treated with a fungicide " Captan" (MAESTROยฎ 80DF Fungicide, Arvesta Corporation, San Francisco, CA, USA). 0.43 mL Captan is applied per 45 g of corn seeds by mixing it well and drying the fungicide prior to the experiment.<br>
Corn kernels are placed embryo side down on blotter paper within an individual cell (8.9 x 8.9 cm) of a germination tray (54 x 36 cm). Ten seeds from an event are placed into one cell of the germination tray. Each tray can hold 21 transgenic events and 3 replicates of wildtype (LH244SDms+LH59), which is randomized in a complete block design. For every event there are five replications (five trays). The trays are placed at 9.7C for 24 days (no light) in a Convrion growth chamber (Conviron Model PGV36, Controlled Environments, Winnipeg, Canada). Two hundred and fifty millilters of deionized water are added to each germination tray.   Germination counts are taken 10th, llth, 12th, 13th, 14th, 17th, 19th, 21st, arid 24th day after start date of the experiment. Seeds are considered germinated if the emerged radicle size isl cm. From the germination counts germination index is calculated.<br>
The germination index is calculated as per:<br>
Germination index = (ฮฃ ([T+l-n] *[Pi-Pi.,]))/T<br>
Where T is the total number of days for which the germination assay is performed. The number of days after planting is defined by n. "i" indicated the number of times the germination had been counted, including the current day. P is the percentage of seeds germinated during any given rating. Statistical differences are calculated between transgenic events and wild type control. After statistical analysis, the events that show a statistical significance at the p level of less than 0.1 relative to wild-type controls will advance to a secondary cold selection. The secondary cold screen is conducted in the same manner of the primary selection only increasing the number of repetitions to ten. Statistical analysis of the data from the secondary selection is conducted to identify the events that show a statistical significance at the p level of less than 0.05 relative to wild-type controls.<br>
Table 17.<br>
(Table 17 Removed)<br><br>
(2) Cold Shock assay - The experimental set-up for the cold shock assay was the same as described in the above cold germination assay except seeds were grown in potted media for the cold shock assay.<br>
The desired numbers of 2.5" square plastic pots were placed on flats (n=32,4x8). Pots were filled with Metro Mix 200 soil-less media containing 19:6:12 fertilizer (6 Ibs/cubic yard) (Metro Mix, Pots and Flat are obtained from Hummert International, Earth City, MO). After planting seeds, pots were placed in a growth chamber set at 23ยฐ C, relative humidity of 65% with 12 hour day and night photoperiod (300 uE/m2-min). Planted seeds were watered for 20 minute every other day by sub-irrigation and flats were rotated every third day in a growth chamber for growing corn seedlings.<br>
On the 10th day after planting the transgenic positive and wild-type negative (WT) plants were positioned in flats in an alternating pattern. Chlorophyll fluorescence of plants was measured on the 10th day during the dark period of growth by using a PAM-2000 portable fluororneter as per the manufacturer's instructions (Walz, Germany). After<br>
chlorophyll measurements, leaf samples from each event were collected for confirming the expression of genes of the present invention. For expression analysis six VI leaf tips from each selection were randomly harvested. The flats were moved to a growth chamber set at 5ยฐ C   All other conditions such as humidity, day/night cycle and light intensity were held constant in the growth chamber. The flats were sub-irrigated every day after transfer to the cold temperature. On the 4th day chlorophyll fluorescence was measured. Plants were transferred to normal growth conditions after six days of cold shock treatment and allowed to recover for the next three days. During this recovery period the length of the V3 leaf was measured on the 1SI and 3rd days. After two days of recovery V2 leaf damage was determined visually by estimating percent of green V2 leaf.<br>
Statistical differences in V3 leaf growth, V2 leaf necrosis and fluorescence during pre- shock and cold shock can be used for estimation of cold shock damage on corn plants. (3) Early seedling growth assay - Three sets of seeds were used for the experiment. The first set consists of positive transgenic events (Fl hybrid) where the genes of the present invention were expressed in the seed. The second seed set was nontransgenic, wild-type negative control made from the same genotype as the transgenic events. The third seed set consisted of two cold tolerant and two cold sensitive commercial check lines of corn. All seeds were treated with a fungicide " Captan", (3a,4,7,a-tetrahydro-2-[(trichloromethly)thio]-lH-isoindole-l,3(2H)-dione, Drex Chemical Co. Memphis, TN). Captan (0.43 mL) was applied per 45 g of com seeds by mixing it well and drying the fungicide prior to the experiment.<br>
Seeds were grown in germination paper for the early seedling growth assay. Three 12"xl8" pieces of germination paper (Anchor Paper #SD7606) were used for each entry in the test (three repetitions per transgenic event). The papers were wetted in a solution of 0.5% KNO3 and 0.1% Thyram.<br>
For each paper fifteen seeds were placed on the line evenly spaced down the length of the paper. The fifteen seeds were positioned on the paper such that the radical would grow downward, for example longer distance to the paper's edge. The wet paper was rolled up starting from one of the short ends. The paper was rolled evenly and tight enough to hold the seeds in place. The roll was secured into place with two large paper clips, one at the top and one at the bottom. The rolls were incubated in a growth chamber at 23ยฐ C for three days in a randomized complete block design within an appropriate container. The chamber was set for 65% humidity with no light cycle. For the cold stress treatment the rolls were then incubated in a growth chamber at 12ยฐ C for twelve days. The chamber was set for 65% humidity with no light cycle.<br><br>
After the cold treatment the germination papers were unrolled and the seeds that did not germinate were discarded. The lengths of the radicle and coleoptile for each seed were measured through an automated imaging program that automatically collects and processes the images . The imaging program automatically measures the shoot length, root length, and whole seedling length of every individual seedling and then calculates the average of each roll.<br>
After statistical analysis, the events that show a statistical significance at the p level of less than 0.1 relative to wild-type controls will advance to a secondary cold selection. The secondary cold selection is conducted in the same manner of the primary selection only increasing the number of repetitions to five. Statistical analysis of the data from the secondary selection is conducted to identify the events that show a statistical significance at the p level of less than 0.05 relative to wild-type controls.<br>
Table 18.<br><br>
(Table 18  Removed)<br>
4. Cold field efficacy trial<br>
This example sets forth a cold field efficacy trial to identify gene constructs that confer enhanced cold vigor at germination and early seedling growth under early spring planting field conditions in conventional-till and simulated no-till environments. Seeds are planted into the ground around two weeks before local farmers are beginning to plant com so that a significant cold stress is exerted onto the crop, named as cold treatment. Seeds also are planted under local optimal planting conditions such that the crop has little or no exposure to cold condition, named as normal treatment. The cold field efficacy trials are carried out in five locations, including Glyndon MN, Mason MI, Monmouth IL, Dayton IA, Mystic CT. At each location, seeds are planted under both cold and normal conditions with 3 repetitions per treatment, 20 kernels per row and single row per plot. Seeds are planted 1.5 to 2 inch deep into soil to avoid muddy conditions. Two temperature monitors are set up at each location to monitor both air and soil temperature daily.<br>
Seed emergence is defined as the point when the growing shoot breaks the soil surface. The number of emerged seedling in each plot is counted everyday from the day the earliest plot begins to emerge until no significant changes in emergence occur. In addition, for each planting date, the latest date when emergence is 0 in all plots is also recorded. Seedling vigor is also rated at V3-V4 stage before the average of corn plant height reaches 10 inches, with l=excellent early growth, 5=Average growth and 9=poor growth. Days to 50% emergence, maximum percent emergence and seedling vigor are calculated using SAS software for the data within each location or across all locations.<br>
The following table lists the data that were collected and analyzed based on the procedure illustrated above. The analyzed data across all locations only include those from Glyndon MN, Mason MI, and Mystic CT.<br><br>
Table 19.<br>
(Table 19  Removed)<br>
E. Screens for transgenic plant seeds with increased protein and/or oil levels<br>
This example sets forth a high-throughput selection for identifying plant seeds with improvement in seed composition using the Infratec 1200 series Grain Analyzer, which is a near-infrared transmittance spectrometer used to determine the composition of a bulk seed sample. Near infrared analysis is a non-destructive, high-throughput method that can analyze multiple traits in a single sample scan.. An NIR calibration for the analytes of interest is used to predict the values of an unknown sample. The NIR spectrum is obtained for the sample and compared to the calibration using a complex chemometric software package that provides a predicted values as well as information on how well the sample fits in the calibration.<br>
Infratec Model 1221,1225, or 1227 with transport module by Foss North America is used with cuvette, item # 1000-4033, Foss North America or for small samples with small cell cuvette, Foss standard cuvette modified by Leon Girard Co. Com and soy check samples of varying composition maintained in check cell cuvettes are supplied by Leon Girard Co. NIT collection software is provided by Maximum Consulting Inc.Software. Calculations are performed automatically by the software. Seed samples are received in packets or containers with barcode labels from the customer. The seed is poured into the cuvettes and analyzed as received.<br>
(Table 20-22 Removed)<br>
Exam-le 6.--<br>
This example illustrates the preparation of transgenic plant cells containing recombinant DNA (SEQ ID NO:82) expressing a maize phytochrome A protein (PHYA). A full-length cDNA encoding a corn PHYA protein was cloned from corn. The cDNA clone contained 3396 bp of nucleotides encoding a 1131 amino acid PHYA protein with molecular weight at 125.2 kD. Based on the cDNA sequences, primers were designed to clone a genomic DNA, illustrated in Figure 1, from a maize inbred LH172 genomic library. Recombinant DNA comprising a rice actin promoter operably linked to the genomic DNA encoding the corn PHYA protein followed by a Hspl7 terminator was inserted into transformation vector of pMON74916 as set forth in SEQ ID NO: 173. Corn plant cells were transformed with recombinant DNA expressing PHA using pMON74916 and used to regenerate a population of transgenic plants. Transgenic plants were regenerated from about 100 events of transformed plant cells; plants from 90 of the events with various expression levels were selected for pollination to produce Rl and Fl seeds; and plants from 31 events were selected for screening for an enhanced trait.. Seed germination and seedling development<br>
Five events were selected to be analyzed phenotypic effect on seed germination and seedling development in the dark condition along with other transgenic material. 12 inbred seeds of each wild-type and transgenic maize events were germinated in a wetted and rolled germination paper in a complete dark growth chamber for 10 days. The length of mesocotyl, coleoptiles and root were measured for every seedling. The transgenic maize seedlings were identified showing great elongation growth of both mesocotyl and expanded coleoptiles imparted from recombinant DNA expressing PHYA protein as compared to non-transgenic controls. Density Study<br>
Transgenic plants were grown in fields at three densities: high density at 42,000 plants per acre; medium density at 35,000 plants per acre; and low density at 28,000 plants per acre. Plnats from three plant cell events expressing PHYA were selected for studying physiological and yield responses to different densities. The physiological data from the density trial YI130 is summarized in the Table 23 shown below. Event ZM_S 83483 under high planting density showed significant decrease in plant height, ear height, and internode length and had a significant increase in chlorophyll content.<br>
Table 23.<br>
(Table 23 Removed)<br>
Kernel trait analysis<br>
As shown in Table 24, events ZM_S83444, ZM_S83446, ZM_S83473, ZM_S83480, ZM_S83483, and ZM__S83907 show significant increases in single kernel weight. Event ZM_S83452 shows significant increases in single kernel weight and total kernel weight. The screening data show that plant cells with stably-integrated, non-natural, recombinant DNA expressing a phytochrome A protein can be regenerated into plants exhibiting increased yield as compared to control plants.<br>
Table 24.<br>
(Table 24 Removed)<br>
Example 7.<br>
This example illlustrates the preparation of transgenic plant cells containing recombinant DNA (SEQ ID NO:77) expressing a soybean MADS box transcription factor protein and identified as G1760.<br>
The DNA encoding the soybean MADS box transcription factor was cloned from a soybean library and inserted into a recombinant DNA construct comprising a CaMV 35S promoter operably linked to the DNA encoding the transcription factor followed by a terminator. The recombinant DNA construct was inserted into a transformation vector plasmid to produce plasmid pMON74470, as set forth in SEQ ID NO: 172 which was used for Agrobacteriu-mediated transformation of soybean plant cells.<br>
Soybean plant cells were transformed with recombinant DNA expressing the MADS box transcription factor using MON74470 and used to regenerate a population of transgenic plants. Transgenic soybean plants were regenerated and selected for screening for an enhanced trait.<br>
Transgenic soybean plants exhibited flowers with highly enlarged sepals and a winding stem. The main stem exhibited reduced lateral branching and increased raceme formation. Flowering time was decreased by about 2 to 4 days as compared to control plants under short day (10 hr) and long day (14 hr) conditions. Transgenic plants also flowered by 5 weeks when placed under non-inductive 20hr light; wild-type control plants did not flower under such conditions. Floral and pod abscission was greatly reduced in the transgenic plants resulting in an increase in the number of pods per plant. Wild type control plants produced on the order of 100 pods, specific transgenic plants produced at least 125 pods per plant and plants regenerated from plant cells of one transgenic event produced greater than 200 pods per plant. There was also a delay in maturity ranging from one week exhibited by plants from single copy event A29204 to a month exhibited by plants from a multi-copy event A28877. Over 95% of the pods on transgenic plants from event A29204 mature in a time period; but only 50% of the pods on transgenic plants from event A28877 mature in the same time period. Seeds from transgenic plants were smaller than seed from control plants and greater in number than seeds from control plants, e.g. about 1800 more seed per pound. Transgenic plants were also shown to be have enhanced water use efficiency.<br>
In testing soybeans for drought tolerance, 4.5" pots were prepared with Metromix 200 and the pots were adjusted to the same weight. Pots were saturated with water. R2 or R3 homozygous seeds were placed in the soil in the pots, 15 pots per event, 3 to 6 events per construct. Plants were grown with a light intensity of 600 ยตEM-2S-1; Temperature: 28ยฐC; Relative humidity (RH): 60%. A gene check with gene check strip (Trait RUR Lateral Flow 50 tests, from Strategic Diagnostics, Inc.) for the presence of the CP4 gene was done on selected plants. Unwanted negative plants were discarded. When plants reached the VI stage. Pots were saturated with water by thorough irrigation. A picture was taken of the plant in the water saturated pot. Excess water was drained and further water was withheld until the pot water content of 50% and 10% of the water capacity for well watered controls and drought treated plants, respectively (monitor the water content by measuring soil moisture or pot weight every 3-5 days). At approximately 10% of the saturated water weight, the plants began to show the onset of the wilting phenotype. Limited-watering was continued every 1-<br>
2 days to maintain pot water content at 50 or 10%. The drought injury phenotype was determined for next 14 days (see the table of measurements). Photograph of plants and physiological assays were run on each at 14 days after the onset of drought treatment. Theses included, but were not limited to, plant height, leaf relative water content, leaf water potential, chlorophyll content and chlorophyll fluorescence. Pot were saturated with nutrient solution and resume regular watering schedule after 14 days.<br>
(Table 25 Removed)<br>
Drought assay measurements as described in Table 25 taken on transgenic soybean plants showed that transgenic soybean plants from transgenic plant cells of event GM 29204 exhibited enhanced water use efficiency.<br>
RO plants regenerated from one transgenic plant cell event (28877) of 41 transgenic plant cells events produced a large number of pods per node and seeds/plant - 531 Rl seeds per plant compared to an average of 150 seeds per plant, i.e. increased yield.<br>
Example 6. Consensus sequence<br>
'This example illustrates the identification of consensus amino acid sequence for the proteins and homologs encoded by DNA that is used to prepare the transgenic seed and plants of this invention having enhanced agronomic traits.<br>
ClustalW program was selected for multiple sequence alignments of the amino acid<br>
sequence of SEQ ID NO: 136 and its nine homologs, and SEQ ID NO: 151 and its 11 homologs. Three major factors affecting the sequence alignments dramatically are (1) protein weight matrices; (2) gap open penalty; (3) gap extension penalty. Protein weight matrices available for ClustalW program include Blosum, Pam and Gonnet series. Those parameters with gap open penalty and gap extension penalty were extensively tested. On the basis of the test results, Blosum weight matrix, gap open penalty of 10 and gap extension penalty of 1 were chosen for multiple sequence alignment. SEQ ID NO: 174 is the consensus sequence of SEQ ID NO: 136 and its homologs, and SEQ ID NO: 175 is the consensus sequence of SEQ ID NO: 151 and its homologs. The symbols for consensus sequence are (1) uppercase letters for 100% identity in all positions of multiple sequence alignment output; (2) lowercase letters for &gt;=70% identity; symbol; (3) "X" indicated =70% sequences.<br>
The consensus amino acid sequence can be used to identify DNA corresponding to the full scope of this invention that is useful in providing transgenic plants, for example corn and soybean plants with enhanced agronomic traits, for example improved nitrogen use efficiency, improved yield, improved water use efficiency and/or improved growth under cold stress, due to the expression in the plants of DNA encoding a protein with amino acid sequence identical to the consensus amino acid sequence.<br>
Example 7. Identification of amino acid domain by Pfam analysis<br>
The amino acid sequence of the expressed proteins that were shown to be associated with an enhanced trait were analyzed for Pfam protein family against the current Pfam collection of multiple sequence alignments and hidden Markov models using the HMMER software. The Pfam protein families for the proteins of SEQ ID NO: 84 through 166 are shown in Table 26. The Hidden Markov model databases for the identified protein families allow identification of other homologous proteins and their cognate encoding DNA to enable the full breadth of the invention for a person of ordinary skill in the art. Certain proteins are identified by a single Pfam domain and others by multiple Pfam domains. For instance, the protein with amino acids of SEQ ID NO: 91 is characterized by two Pfam domains, i.e. SRF-TF and K-box; and, the protein with amino acids of SEQ ID NO: 165 is characterized by six Pfam domains, i.e. GAF, Phytochrome, PAS, a repeated PAS, HisKA, and HATPase.<br>
Table 26.<br>
(Table 26-27 Removed)<br>
Example 8. Selection of transgenic plants with enhanced agronomic trait(s)<br>
This example illustrates the preparation and identification by selection of transgenic seeds and plants derived from transgenic plant cells of this invention where the plants and seed are identified by screening a having an enhanced agronomic trait imparted by expression of a protein selected from the group including the homologous proteins, SEQ ID NO: 121, 128,152-160, 162 and 164. Transgenic plant cells of corn, soybean, cotton, canola, wheat and rice are transformed with recombinant DNA for expressing each of the homologs identified in Example 4. Plants are regenerated from the transformed plant cells and used to produce progeny plants and seed that are screened for enhanced water use efficiency,<br>
enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil. Plants are identified exhibiting enhanced traits imparted by expression of the homologous proteins.<br><br><br><br><br>
What is claimed is:<br>
1.	A plant cell with stably integrated, recombinant DNA comprising a promoter that is<br>
functional in plant cells and that is operably linked to DNA from a plant, bacteria or yeast<br>
that encodes a protein having at least one domain of amino acids in a sequence that exceeds<br>
the Pfam gathering cutoff for amino acid sequence alignment with a protein domain family<br>
identified by a Pfam name in the group of Pfam names consisting of AAA, AP2, Aldo ket<br>
red, Alpha-amylase, Aminotran 1 2, Ank, ArfGap, Asn synthase, BRO1, CBFD NFYB HMF,<br>
Catalase, CorA, Cpn60 TCP1, Cystatin, DNA photolyase, DSPc, DUF1685, DUF296, Dil9,<br>
E2F TDP, FAD binding 7, FA desaturase, FBPase, GAP, GATA, GATase 2, Glyco hydro 1,<br>
Glyoxalase, Gotl, HATPase c, HSF DNA-bind, HSP20, HisKA, Homeobox, Hpt, Isoamylase<br>
N, K-box, Lactamase B, Metallophos, MtN3 slv, NAF, NAM, NIF, Oxidored FMN, PAS,<br>
PDZ, PRA1, Peptidase C15, Peptidase S10, Peptidase S41, Phytochrome, Peinase, Pkinase<br>
Tyr, Pyridoxal deC, RIO1, RRM 1, RTC, RTC insert, Ras, Response reg, SPC25, SPX, SRF-<br>
TF, Synaptobrevin, UPF0057, zf-C2H2, and zf-C3HC4; wherein the Pfam gathering cuttoff<br>
for said protein domain families is stated in Table 28; wherein said plant cell is selected from<br>
a population of plant cells with said recombinant DNA by screening plants that are<br>
regenerated from plant cells in said population and that express said protein for an enhanced<br>
trait as compared to control plants that do not have said recombinant DNA; and wherein said<br>
enhanced trait is selected from group of enhanced traits consisting of enhanced water use<br>
efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency,<br>
enhanced seed protein and enhanced seed oil.<br>
2.	A plant cell of claim 1 wherein said protein has an amino acid sequence with at least<br>
90% identity to a consensus amino acid sequence in the group of consensus amino acid<br>
sequences consisting of the consensus amino acid sequence constructed for SEQ ID NO: 84<br>
and homologs thereof through the consensus amino acid sequence constructed for SEQ ID<br>
NO: 166 and homologs thereof.<br>
3.	A plant cell of claim 1 wherein said protein is selected from the group of proteins<br>
identified in Table 2.<br>
4.	A plant cell of claim 1 further comprising DNA expressing a protein that provides<br>
tolerance from exposure to an herbicide applied at levels that are lethal to a wild type of said<br>
plant cell.<br>
5.	A plant cell of claim 4 wherein the agent of said herbicide is a glyphosate, dicamba,<br>
or glufosinate compound.  <br>
6.	A transgenic plant comprising a plurality of the plant cell of claim 1<br>
7.	A transgenic plant of claim 6 which is homozygous for said recombinant DNA.<br>
8.	A transgenic seed comprising a pluarility of the plant cell of claim 1.<br>
9.	A transgenic seed of claim 8 from a corn, soybean, cotton, canola, alfalfa, wheat or<br>
rice plant.<br>
10.	Non-natural, transgenic corn seed of claim 9 wherein said seed can produce corn<br>
plants that are resistant to disease from the Mai de Rio Cuarto virus or the Puccina sorghi<br>
fungus or both.<br>
11.	A transgenic pollen grain comprising a haploid derivative of a plant cell of claim 1.<br>
12.	A method for manufacturing non-natural, transgenic seed that can be used to produce<br>
a crop of transgenic plants with an enhanced trait resulting from expression of stably-<br>
integrated, recombinant DNA comprising a promoter that is (a) functional in plant cells and<br>
(b) is operably linked to DNA from a plant, bacteria or yeast that encodes a protein having at<br>
least one domain of amino acids in a sequence that exceeds the Pfam gathering cutoff for<br>
amino acid sequence alignment with a protein domain family identified by a Pfam name in<br>
the group of Pfam names consisting of AAA, AP2, Aldo ket red, Alpha-amylase, Aminotran<br>
1 2, Ank, ArfGap, Asn synthase, BRO1, CBFD NFYB HMF, Catalase, CorA, Cpn60 TCP1,<br>
Cystatin, DNA photolyase, DSPc, DUF1685, DUF296, Dil9, E2F TDP, FAD binding 7, FA<br>
desaturase, FBPase, GAF, GATA, GATase 2, Glyco hydro 1, Glyoxalase, Gotl, HATPase c,<br>
HSF DNA-bind, HSP20, HisKA, Homeobox, Hpt, Isoamylase N, K-box, Lactamase B,<br>
Metallophos, MtN3 slv, NAF, NAM, NIF, Oxidored FMN, PAS, PDZ, PRAl, Peptidase C15,<br>
Peptidase S10, Peptidase S41, Phytochrome, Peinase, Pkinase Tyr, Pyridoxal deC, RIO1,<br>
RRM 1, RTC, RTC insert, Ras, Response reg, SPC25, SPX, SRF-TF, Synaptobrevin,<br>
UPF0057, zf-C2+H2, and zf-C3HC4; wherein the gathering cutoff for said protein domain<br>
families is stated in Table 28; and wherein said enhanced trait is selected from the group of<br>
enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance,<br>
increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed<br>
oil, said method for manufacturing said seed comprising:<br><br>
(a)	screening a population of plants for said enhanced trait and said recombinant DNA,<br>
wherein individual plants in said population can exhibit said trait at a level less than,<br>
essentially the same as or greater than the level that said trait is exhibited in control plants<br>
which do not express the recombinant DNA,<br>
(b)	selecting from said population one or more plants that exhibit the trait at a level greater<br>
than the level that said trait is exhibited in control plants verifying that said recombinant DNA is stably integrated in said selected plants,<br>
(c)	analyzing tissue of a selected plant to determine the production of a protein having the<br>
function of a protein encoded by nucleotides in a sequence of one of SEQ ID NO: 1-83; and<br>
(d)	collecting seed from a selected plant.<br>
13.	A method of claim 12 wherein plants in said population further comprise DNA<br>
expressing a protein that provides tolerance to exposure to an herbicide applied at levels that<br>
are lethal to wild type plant cells, and wherein said selecting is effected by treating said<br>
population with said herbicide.<br>
14.	A method of claim 13 wherein said herbicide comprises a glyphosate, dicamba, or<br>
glufosinate compound.<br>
15.	A method of claim 12 wherein said selecting is effected by identifying plants with<br>
said enhanced trait.<br>
16.	A method of claim 12 wherein said seed is corn, soybean, cotton, alfalfa, wheat or<br>
rice seed.<br>
17.	A method of producing hybrid corn seed comprising:<br><br>
(a)	acquiring hybrid corn seed from a herbicide tolerant corn plant which also has stably-<br>
integrated, recombinant DNA comprising a promoter that is (a) functional in plant<br>
cells and (b) is operably linked to DNA that encodes a protein having at least one<br>
domain of amino acids in a sequence that exceeds the Pfam gathering cutoff for amino<br>
acid sequence alignment with a protein domain family identified by a Pfam name in<br>
the group of Pfam names consisting of AAA, AP2, Aldo ket red, Alpha-amylase,<br>
Aminotran 1 2, Ank, ArfGap, Asn synthase, BRO1, CBFD NFYB HMF, Catalase,<br>
CorA, Cpn60 TCP1, Cystatin, DNA photolyase, DSPc, DUF1685, DUF296, Dil9,<br>
E2F TOP, FAD binding 7, FA desaturase, FBPase, GAP, GATA, GATase 2, Glyco<br>
hydro 1, Glyoxalase, Gotl, HATPase c, HSF DNA-bind, HSP20, HisKA, Homeobox,<br>
Hpt, Isoamylase N, K-box, Lactamase B, Metallophos, MtN3 slv, NAF, NAM, NIF,<br>
Oxidored FMN, PAS, PDZ, PRA1, Peptidase C15, Peptidase S10, Peptidase S41,<br>
Phytochrome, Peinase, Pkinase Tyr, Pyridoxal deC, RIO1, RRM 1, RTC, RTC insert,<br>
Ras, Response reg, SPC25, SPX, SRF-TF, Synaptobrevin, UPF0057, zf-C2H2, and<br>
zf-C3HC4; wherein the gathering cuttoff for said protein domain families is stated in<br>
Table 28;<br>
(b)	producing corn plants from said hybrid corn seed, wherein a fraction of the plants<br>
produced from said hybrid corn seed is homozygous for said recombinant DNA, a<br>
fraction of the plants produced from said hybrid corn seed is hemizygous for said<br>
recombinant DNA, and a fraction of the plants produced from said hybrid corn seed has none of said recombinant DNA;<br>
(c)	selecting corn plants which are homozygous and hemizygous for said recombinant<br>
DNA by treating with an herbicide;<br>
(d)	collecting seed from herbicide-treated-surviving corn plants and planting said seed to<br>
produce further progeny corn plants;<br>
(e)	repeating steps (c) and (d) at least once to produce an inbred corn line;<br>
(f)	crossing said inbred corn line with a second corn line to produce hybrid seed.<br><br>
18.	The method of selecting a plant comprising cells of claim 1 wherein an<br>
immunoreactive antibody is used to detect the presence of said protein in seed or plant tissue.<br>
19.	Anti-counterfeit milled seed having, as an indication of origin, a plant cell of claim 1.<br>
20.	A method of growing a corn, cotton or soybean crop without irrigation water<br>
comprising planting seed having plant cells of claim 1 which are selected for enhanced water<br>
use efficiency.<br>
21.	A method of claim 20 comprising providing up to 300 millimeters of ground water<br>
during the production of said crop.<br>
22.	A method of growing a corn, cotton or soybean crop without added nitrogen fertilizer<br>
comprising planting seed having plant cells of claim 1 which are selected for enhanced<br>
nitrogen use efficiency.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LTEtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMi0wOC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-1-Correspondence Others-(12-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LTEtRm9ybS0zLSgxMi0wOC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-1-Form-3-(12-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNsYWltcy0oMTgtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Claims-(18-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNsYWltcy0oMjItMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Claims-(22-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Correspondence Others-(04-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTItMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Correspondence Others-(12-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Correspondence Others-(16-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Correspondence Others-(18-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Correspondence Others-(22-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Correspondence-Others-(18-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUZvcm0tMTgtKDE4LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Form-18-(18-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUZvcm0tMi0oMTgtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Form-2-(18-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUZvcm0tMy0oMTgtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Form-3-(18-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LUdQQS0oMTYtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-GPA-(16-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTcwMS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTgtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5701-delnp-2007-Petition-137-(18-02-2014).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="260891-methods-of-reparing-polymer-organoclaycomposites-and-articles-derived-therefrom.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="260893-power-consumption-controlled-transmitter.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>260892</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5701/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-May-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-May-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY,LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH, LINDBERGH BOULEVARD, ST.LOUIS, MISSOURI 63167 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ABAD MARK</td>
											<td>800 NORTH, LINDBERGH BOULEVARD, ST.LOUIS, MISSOURI 63167 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/046031</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/638,099</td>
									<td>2004-12-21</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/660,320</td>
									<td>2005-03-10</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/260892-transgenic-plants-with-enhanced-agronomic-traits by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:17:23 GMT -->
</html>
