<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/261111-a-nucleic-acid-molecule-dbait by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:08:04 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 261111:“A NUCLEIC ACID MOLECULE, DBAIT”</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">“A NUCLEIC ACID MOLECULE, DBAIT”</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to compositions and methods for interfering with the DNA repair of double strand breaks (DSBs). The invention discloses novel double-stranded nucleic acid molecules that act as baits and hijack the holocomplex of enzymes responsible of DNA DSB sensing, signaling and/or repair pathways, in particular the non homologous end joining (NHEJ) pathway of DSB repair. The invention discloses the use of these molecules as adjuvant compositions to be used in association with a DNA breaking treatment, particularly radiotherapy or chemotherapy, in combination with a pharmaceutically acceptable carrier, in an efficient amount to be introduced in the tumor cell nuclei in order to neutralize transiently their DNA repair capacity and trigger their death.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Dbait and uses thereof<br><br>
This invention relates to compositions and methods of interfering with DNA repair pathways in mammalian cells. The invention particularly relates to nucleic acid molecules that interfere with DNA damage sensing, signaling and/or repair pathways, in particular the non homologous end joining (NHEJ) pathway of double-stranded break (DSB) repair, as well as to the uses thereof, particularly for triggering cell lethality of tumors submitted to anticancer therapies. The invention discloses a novel class of nucleic acid molecules, termed "DSB bait" or "Dbait", which may be used in a variety of therapeutic conditions in mammalian subjects, to interfere with DNA DSB repair pathways.<br><br>
Background<br><br>
Radiotherapy and chemotherapy, alone or combined together with surgery, are essential therapeutic arsenals against human cancer.<br><br>
Ionizing radiations cause directly or indirectly double-stranded breaks (DSBs) and trigger cell/tissue death (necrosis or apoptosis). The cytotoxic effect of ionizing radiation forms the basis for radiotherapy, which is widely used in the treatment of human cancer. The efficacy of radiotherapy is currently limited by the radio-resistance of certain tumors (for example, glioblastoma, head and neck squamous cell carcinoma) and by the side effects caused by irradiation of nearby normal tissues (for example, in the treatment of breast and cervical cancer).<br><br>
In the past years, many studies have focused on biological mechanisms related to the ionizing radiation response, in order to gain insights into the complexity of phenomena underlying radio-sensitivity or radio-resistance of tumor cells. The understanding of the different pathways which finely regulate the response to ionizing radiation is an important step towards the identification of molecular targets for new drugs and therapies that, in association with radiotherapy, can improve the chance of recovery from tumors highly resistant to radiation, such as brain or head and neck tumors.<br><br>
The use of chemotherapeutic agents can cause DNA damages, including direct or indirect DSBs. Examples of mostly used families of chemotherapeutic agents (chemical cytotoxics) are: inhibitors of topoisomerases I or Il (camptothecin/topotecan, epirubicin/etoposide), DNA crosslinkers (cisplatin/carboplatin/oxaliplatin), DNA alkylating agents (carmustine/dacarbazine) or anti-metabolic agents (5- fluorouracil/gemcitabine/capecitabine), as well as inhibitors of the mitotic spindles <br><br>
(paclitaxel/docetaxel/vinorelbine).<br><br>
Recent progress in developing biological drugs (monoclonal antibodies, cytokines/kinase inhibitors, immunotherapies/vaccines) has proven their efficiency and specificity towards a subset of tumors. But they are often used in combination with chemical cytotoxics. Despite of many progresses in the development of new cytotoxic drugs, the drug resistance to chemotherapy is still a major clinical concern in the treatment of cancer. The understanding of the mechanism of drug resistance related to drug uptake/efflux, metabolic degradation, mutagenesis of target, enhanced repair, signaling of cell death (apoptosis and necrosis) is essential for insuring efficiency of chemotherapy and improve therapeutic index, especially, in some treatment-resistant tumors.<br><br>
The association between chemotherapy and radiotherapy was widely used in cancer treatment. Although still not completely elucidated, the biological basis of action of the cytotoxics relies on cellular mechanisms, such as cell cycle or DNA damage, which is also important for the radio-induced cell death, leading to the additive or even better synergistic benefits by combining different treatments in cancer therapies.<br><br>
In the last decade, many investigations were carried out in this field, and the complexity of signal transduction in response to radiation began to be delineated. In this respect, genes of particular interest to be targeted with ionizing radiations are those involved in the regulation of radiation-induced lethality mechanisms, such as apoptosis or DNA repair. As DSBs are the most lethal DNA damages, the efficacy of ionizing radiation decreases as that of DSB repair increases.<br><br>
Two mechanisms are involved in the repair of DSBs: non homologous end-joining (NHEJ, sequence-independent pathway) and homologous recombination (HR, sequence- dependent pathway) (reviewed by Jackson, 2002). Targeting genes involved in these two main DSB repair pathways has so far led to little or moderate radio-sensitivity, depending on the used approaches and cancer cell lines (Belenkov et al.,2002 ; Marangoni et al. 2000; Ohnishi et al, 1998).<br><br>
Ku (e.g., Ku70 and Ku80) and DNA-PKcs proteins are important in the repair of radiation- or chemo-induced DNA DSBs. If damage cannot be repaired on time, cells die. Therefore, they represent potentially interesting molecular targets for sensitizing target cells and tissues to radiotherapy and chemotherapy. Many approaches have thus been conceived <br><br>
and carried out to try to inhibit these key proteins (Ku70/Ku80, DNA-PKcs, etc. ) involved in the NHEJ pathway, which is predominant in mammalian cells:<br><br>
1 ) Inhibitors of PI3K (phosphatidylinositol-3-kinase) (i.e., DNA-PKcs, ATM, ATR) (Boulton et al., 2000 ; Durant &amp; Karran, 2003; Willmore et al., 2004 ; Vauger et al., 2004) ; 2) Negative dominant &amp; peptides (C-terminal of KU80) (Marangoni et al. , 2000;<br><br>
Kim et al., 2002) ;<br><br>
3) Single chain antibody variable fragment (scFv) (DNA-PKcs) (Li et al. 2003) ;<br><br>
4) RNA Aptamer (SELEX: RNA binding Ku) (Yoo &amp; Dynan, 1998) ;<br><br>
5) Antisense (Ku70,Ku80, DNA-PKcs) (Li et al., 2003b; Marangoni et al., 2000; Sak et al., 2002) ;<br><br>
6) siRNA (DNA-PKcs) (Peng et al. 2000).<br><br>
Despite these tremendous efforts, the combination of the targeting of genes involved in DNA repair pathways and cancer therapies is still in early experimental stage and no clinical study has shown any proven benefits so far. It is worth to note that the above described approaches share a common feature: they target a single effector (protein) involved in a complex cascade pathway (such as NHEJ) with possible bypass or compensation.<br><br>
Summary of the Invention<br><br>
The present invention relates to novel compositions and methods of interfering with DNA repair pathways in mammalian cells. The invention particularly relates to nucleic acid molecules that interfere, in a non gene-specific manner, with DNA damage sensing, signaling and/or repair pathways, as well as to the uses thereof, particularly for triggering cell lethality of tumors submitted to anticancer therapies.<br><br>
The inventors have found that the sensitivity of cells to direct or indirect DNA damaging therapies can be enhanced by using (chemically modified or not) short dsDNA molecules which act as mimics of broken DNA fragments and are recognized as DSB sites induced by the DNA damaging treatments (i.e. the substrate mimics of DSB).<br><br>
As shown in the examples, the molecules of this invention are effective in vitro as well as in vivo, and may be used to confer or increase sensitivity of any tumor cell to DNA damaging cancer therapy.<br><br>
An object of the invention thus relates to such dsDNA molecules, also designated by the name of "DSB bait" molecules (Dbait in short), which are capable of enhancing the <br><br>
response of treatment-resistant tumors to radiotherapy and chemotherapy. As will be disclosed further below, Dbait molecules act by baiting and hijacking the holocomplex of DNA repair enzymes, and thereby interfere with DNA lesion sensing, signaling and/or repair processes. This novel approach is named "DNA bait".<br><br>
In a preferred embodiment, the invention relates to a Dbait molecule which is a nucleic acid molecule, wherein said molecule comprises a double stranded portion of at least 24 bp, has at least one free end, is devoid of CpG, has less than 70 % sequence identity to any gene in a human genome, comprises one or several phosphorothioates or nucleotides with methylphosphonate backbone at the end of each strand or at least at the 3' end strand, and wherein said molecule is substrate for binding by at least a Ku protein involved in the NHEJ pathway of double strand breaks (DSB) repair. Preferably, the Dbait molecule is selected from the group consisting of Dbait32Ha, Dbait32Hb, Dbait32Hc, Dbait32Hd, Dbait32Hc-3'mp, Dbait32Hc-5'3'mp, Dbait32Hc-Cy3, Dbait32Hc-Cy5, and Dbait32Hd-FITC. Still more preferably, the Dbait molecule is Dbait32Hc.<br><br>
A further object of this invention resides in a composition comprising a Dbait molecule and a pharmaceutically acceptable carrier or excipient. In a particular embodiment, the composition is suitable for oral route or for intravenous, intra-tumoral or sub-cutaneous injection, or for intracranial or intra artery injection or infusion or for topic administration.<br><br>
A further object of this invention resides in a Dbait molecule in combination with (a) physical and/or chemical agent(s) which can directly or indirectly cause DSBs of DNA. In particular, the invention concerns a pharmaceutical product comprising a Dbait molecule of the invention and a chemical agent which can directly or indirectly cause double-strand breaks of DNA as a combined preparation for use for treating cancer. Preferably, the Dbait molecule is to be administered prior to or along with the chemical agent.<br><br>
A further object of this invention is a method for treating a proliferative disorder (e.g., a cancer) using a combination of a Dbait molecule and a therapy which directly or indirectly causes DNA damage(s). Accordingly, the present invention concerns the use of a Dbait molecule of the invention for the manufacture of a medicament for treating cancer to be used in combination with a DNA damaging anticancer therapy. In particular, the DNA damaging anticancer therapy is selected from radiotherapy and chemotherapy. Preferably, the Dbait molecule is to be administered prior to radiotherapy. Alternatively, the Dbait molecule is administered prior to or along with chemotherapy. In a particular embodiment, the molecule is to be administered by oral route or by intravenous, intra- <br><br>
tumoral or sub-cutaneous injection, or by intracranial or intra artery injection or infusion or for topic administration. Preferably, the cancer is selected from CNS, head and neck, colorectal, liver, gastrointestinal tract, genitor-urinary tract, lung, skin, breast cancer and cervical cancer.<br><br>
Another object of the invention relates to the use of Dbait molecules for making anticancer therapeutic adjuvant for enhancing efficiency of cancer treatment, particularly for the tumors poorly responding to radio- and/or chemotherapies.<br><br>
A further object of this invention is a method of enhancing tumor sensitivity to DNA damaging anticancer therapy, the method comprising administering to a subject a Dbait molecule as defined above.<br><br>
Accordingly, the present invention concerns the use of a Dbait molecule of the invention for the manufacture of a medicament for enhancing tumor sensitivity to DNA damaging anticancer therapy. In particular, the DNA damaging anticancer therapy is selected from radiotherapy and chemotherapy. Preferably, the Dbait molecule is to be administered prior to radiotherapy. Alternatively, the Dbait molecule is administered prior to or along with chemotherapy. In a particular embodiment, the molecule is to be administered by oral route or by intravenous, intra-tumoral or sub-cutaneous injection, or by intracranial or intra artery injection or infusion or for topic administration. Preferably, the cancer is selected from CNS, head and neck, colorectal, liver, gastrointestinal tract, genitor-urinary tract, lung, skin, breast cancer and cervical cancer.<br><br>
A further object of this invention is a method of treating cancer, the method comprising administering to a subject, a Dbait molecule in combination with a DNA damaging anticancer therapy.<br><br>
A further object of this invention is a composition for use in association with a DNA breaking treatment, particularly radiotherapy or chemotherapy, said composition comprising at least one Dbait molecule in combination with a pharmaceutically acceptable carrier, in an efficient amount to be introduced in the nucleus of tumor cells.<br><br>
The invention may be used to confer sensitivity to cancer therapy in various types of cancers in mammalian subjects, particularly in human subjects, such as solid cancers and leukemia, particularly radio- or chemo-resistant cancers. <br><br>
Legend to the Figures<br><br>
Figure 1.1 : Band-shift assays performed on different 32P radio-labelled Dbait molecules in the presence of various amounts of nuclear extract (0, 10, 20, 40, 80, 160, 320 ng/µl) from Hep2 cells. The shifted bands were numbered 1 and 2. The band 3 is loading well.<br><br>
Figure 1.2: Identification of the presence of Ku proteins in the retarded bands of different 32P radio-labelled Dbait molecules involving proteins in Hep2 cell nuclear extract (0, 20, 80, 320 ng/µl). On line a-Ku, it is indicated when anti-Ku antibodies were added (+) or not (-) to the binding reaction before loading of sample on the gel. The shifted bands were numbered 1 , 2 and 3 and a star was added to number for bands showing a shifted migration after anti-Ku binding.<br><br>
Figure 1.3: DNA end-joining assay with 20 µg Hep2 nuclear protein extract. Upper panel: the ligation of 0.2 µM 32P-labeled DNA fragments in the absence and in the presence of 20 µM Dbait in 20 µl assay buffer over various times. The bands 1-4 indicate the initial 605-bp DNA fragments (monomers [1]), the ligation products which migrate as dimers [2], trimers [3] or tetramers [4]. Lower panel: The percentage of ligation products was quantified and shown as a function of time (diamonds, without Dbait; circles, with 200 nM Dbait32H), and as a function of the chemical structure of Dbait molecules (after 2 hour incubation with 200 nM various Dbait).<br><br>
Figure 1.4: DNA-dependent protein kinase (DNA-PK) activity assay in 1.5µg of Hep2 nuclear protein extract of a number of 2µg Dbait molecules with various lengths, sequences and chemical structures including modified backbones: Dbait32ss and Dbait32css are two 32-nt single strand DNA; Dbait32C is a dumbbell 32-bp double strand DNA (without free blunt end); DbaitdH, Dbait16H, Dbait24H and Dbait32H are hairpin double strand DNA with a stem of 8, 16, 24 and 32-bp, respectively. Dbait32H, Dbait32Hb, Dbait32Hc and Dbait32Hd are 32-bp hairpin DNA with different sequences but the same base composition; Dbait32d-F and DbaitHc-cy3 are fluorescein and cyanine 3 tagged Dbait32Hd and Dbat32Hc, respectively; Dbait32H-po is a full length phosphodiester Dbait32H, as compared to other Dbait molecules which have 3-bp phosphorothioate at the free blunt end, except DbaitHc-3'mp and Dbait32Hc-5'3'mp which have 3-nt methylphosphonate at either the 3'-end or the both 5' and 3' end, respectively. Dbait32Hc5'5' has a 3'-3' linkage, thus exhibiting a 575' blunt end; Cf. table 1.1 and table 1.2 for the details of these Dbait molecules. Data represent the mean value and standard deviation of at least 3 independent experiments. <br><br>
Figure 2.1 : Clonogenic survival assay of HeIa cells after 4x0.5Gy (spaced 2 hours) irradiation performed with ?-rays from a 137Cesium source in the presence of various Dbait molecules. Panel A: Dose-dependence of normalized survival clone number in the presence of Dbait32 and Dbait32H. Panel B: Normalized survival clone number in the presence of different Dbait molecules at 83 nM (concentration in culture medium).<br><br>
Figure 2.2: Additional clonogenic survival assay of HeIa cells after 4x0.5Gy (spaced 2 hours) irradiation performed with ?-rays from a 137Cesium source in the presence of various Dbait molecules. Upper panel: Dose-dependence of normalized survival clone number in the presence of Dbait32H and DbaitdH. Lower panel: Normalized survival clone number in the presence of different 2µg Dbait molecules.<br><br>
Figure 2.3: Inhibition of radiation-enhanced illegitimate integration of a linear plasmid fragment (2µg) carrying the gene coding for neomycin resistance by 2µg Dbait32H molecules.<br><br>
Figure 2.4: Additional assays of the inhibition of radiation-enhanced illegitimate integration of a linear plasmid fragment (2µg) carrying the gene coding for puromycin resistance by various Dbait32 molecules (2µg). Upper panel: Dose-dependence of Dait32H molecule in the presence of fractionated (4x0.5Gy) irradiation (filled circles) or in the absence of irradiation (filled triangles); Lower panel: The efficiency of plasmid integration in the presence 2µg various Dbait molecules or 200µM DNA-PK inhibitors (wortmannin or NU7026), and with (black) or without irradiation (grey).<br><br>
Figure 2.5: Immunodetection of double strand break (DSB) sites as revealed by ?-H2AX foci in HeIa cells transfected by fluorescent Dbait32H-FITC molecules at 2 hours after 2Gy irradiation. Left panel: fluorescence of Dbait32H-FITC (bright dots and patches) and DSB sties detected by immunofluorecsence of ?-H2AX antibody in nuclei; Right panel: the same image of nuclei with DSB sites detected by immunofluorescence of ?-H2AX antibody and DAPI counterstaining. The arrows at the lower left corner show the absence of Dbait32H-FITC and ?-H2AX signal in nucleus. The arrows at the upper right corner show the co-localized Dbait32H-FITC and ?-H2AX signals.<br><br>
Figure 2.6: Upper panel: Histone H2AX phosphorylation by PIKKs. Total cell extracts were analyzed by western blotting for the level of phosphorylated form of histone H2AX (?-<br><br>
H2AX) as compared to the total H2AX protein. Hep2 cells were transfected 5 hours with various Dbait molecules 32Hc, 24H, 16H and 8H for 5 h, or not transfected. They were <br><br>
irradiated at the end of transfection, incubated for one hour and then analyzed. The resultants are presented as histogram of normalized ratio of ?-H2AX/H2AX. Lower panel: Kinetics of the persistence of DSB sites revealed by ?-H2AX foci by FACS in the irradiated (IR) cells: Dbait32Hc+IR (solid line), IR alone (dashed line) and untreated (dotted line).<br><br>
Figure 3.1 : FACS analyses of the untreated GMA32 cells, the cells transfected alone, or transfected with different Dbait molecules by lipofectamine, but without further irradiation or mitotic inhibitor treatment. The M1 phase represents the percentage of cells in sub-G1 stage indicative of cell death.<br><br>
Figure 3.2: Immunodetection of DNA repair foci by ?-H2AX labeling (bright dots or patches in nuclei) in the untreated GMA32 cells, the cells transfected alone, or transfected with different Dbait molecules by lipofectamine. The counterstaining of cell membranes and nuclei were achieved by FITC-DiOC6 and DAPI.<br><br>
Figure 3.3: Western blot analysis of the phosphorylation status of p53 serine 15 residue of the untreated GMA32 cells, the cells transfected alone, or transfected with different Dbait molecules by lipofectamine, but without further irradiation or mitotic inhibitor treatment.<br><br>
Figure 3.4: Clonogenical survival of untreated and treated GMA32 cells by 4Gy irradiation or by different mitotic inhibitors (20OnM nocodazole, 10OnM navelbine (vinorelbine) or 20OnM taxol (paclitaxel)) in the presence of different Dbait molecules.<br><br>
Figure 4.1 : Growth of the xenografted human larynx tumor on mice monitored as the ratio of the tumor volume at time t over the initial volume (VtA/,) with or without treatments. Panel A: Untreated arm (n=38); Panel B: control arm with 20µl culture medium (MEM) + 3x2Gy/week irradiation (n=30); Panel C: the arm with 1 nmole (20µg) Dbait32H + 3x2Gy/week irradiation (n=35). MEM or Dbait32H was delivered by intratumoral injection 5 hours prior irradiation. The split irradiation dose (2Gy) was given one of every two days, three times a week. The treatment lasted 5 weeks totalling 30Gy irradiation. The dots represent the time course of tumor volume of each mouse. The solid lines are the best polynomial fitting. Panel D shows a Kaplan-Meier plot of all mice of which the increase in tumor volume (VtA/,) 
<br>
Figure 4.2: Distribution of cyanine C3 labelled Dbait32H in Hep2 xenograft tumor in nude mice. 20µg Dbait32H-Cy3 formulated with Superfect (transfection agent) was injected into <br><br>
1.5cm3 Hep2 tumor. The mice were sacrificed 6 hours after the injection. The tumors were taken out and cryo-sliced for analysis without fixing. DAPI was used for nuclei staining.<br><br>
Figure 4.3: Radiosensitization of Hep2 xenografted tumors in nude mice. Tumor growth was monitored during treatment (15 sessions within 35 days; grey background) and after<br><br>
(white background) in four groups of 10 animals with different treatments. Individual tumor growth is indicated for each animal. The treatment protocol was indicated at the top. For each treatment session, 2µg Dbait32H formulated with transfection agent (PEI) was injected into tumor 5 hours prior 2Gy irradiation. The average times for 5 fold increase of tumor volume of each group are also indicated. The p-value was calculated for the group received combined treatment as compared to the group received the irradiation alone.<br><br>
Figure 4.4: Upper panel: Kaplan-Meier representation of survival of nude mice subcutaneously xenografted by Hep2 tumor. The treatment protocol was described in figure 4.3. The five groups were included: untreated, mock-tranfected and irradiated, treated by combined irradiation and increasing amount of Dbait32H (20, 60 and 120µg/session). Number of animals for each group is indicated in table 3.2. Grey background indicates the period of treatment. Lower panel: Pictures of tumors representative of the three groups (untreated, treated with 20 and 60µg Dbait32H/session associated with 2Gy irradiation) which were taken 15 days after beginning of treatment, at the end of treatment (35 days) and 13 days after the end of treatment (35+13 days).<br><br>
Figure 4.5: Histological analysis of xenograted Hep2 tumors at mid-course treatment (7 sessions). Tumors were taken out 20 days after beginning of various treatment protocols as indicated. They were fixed in formalin and tissue sections were stained with hematoxylin, eosin and safran. Two tumors for each treatment protocol were analyzed by microscopy. Panel a: Pictures of representative fields of each protocol. Scale bars indicate 400mm (panels 1-4) and 100mm (panels 5-8). Color pictures are available upon request. Panel b: The extent of necrosis is expressed as the proportion of the surface area of the tissue section analyzed that was necrotic. The number of mitotic cells and apoptotic cells were estimated from representative non-necrotic fields of about 1 ,000 cells analyzed at high power.<br><br>
Figure 4.6: NMR imaging of xenografted Hep2 tumors at the mid-course treatment (7 sessions). Three representative cross-section images were shown with untreated tumor, tumor treated by irradiation (2Gy/session), and by the combined Dbait32H (20µg/session) <br><br>
and irradiation (2Gy/session). The tumors are outlined by white circle. The grey mass inside tumor indicated the necrotic area.<br><br>
Figure 4.7: Kaplan-Meier representation of survival of nude mice subcutaneously xenografted by various tumors (Hep2: squamous cell carcinoma; U87: glioblastoma; LU 1205 and SK28: two types of melanoma). The period of treatment is indicated by grey area.<br><br>
Figure 5.1 : Panel A: Treatment protocol for three groups/arms of the K-Rasv12G x Apc1638N transgenic mice at the mean age of 12 weeks: the control group (untreated), the group treated by 5FU+CPT1 1 , the group treated by 5FU+CPT11 and Dbait32H. It was performed by three cycles of treatment. Each cycle consists of intraperitoneal injection of<br><br>
0.6mg 5FU and 0.6mg CPT1 1 , along with 0.1 mg Dbait32H by oral administration, three times a week, followed by one week rest. The number of mice involved in each groups is indicated in parenthesis. The end point is the time of survival; Panel B: Kaplan-Meyer plot of survival curves of the three groups; Panel C: The median survival time of three groups as shown in panel B.<br><br>
Figure 5.2: Mean number of tumors in digestive tract per animal by macroscopy or histology examination of the groupes treated by 5FU+CPT11 and by 5FU+CPT1 1 and Dbait32H. The number of animals in each group was indicated in parenthesis. All mice were sacrificed two weeks (week 18) after the protocol shown in the upper panel. The mean number of the control arm (untreated group, n=101 ) is 30.8/animal (data not shown).<br><br>
Figure 5.3: Fluorescence microscopy analysis of digestive tract tissue. Panel A: Protocol schema (i.p.: intraperitoneal injection; o.: oral administration). Panel B: Fluorescence of Dbait32H-FITC molecules (left) and of immonufluorescence labelled ?-H2AX (right) on the 5µm section from the tumor tissue of the treated animal according to the protocol given in panel A. Lower parts show the details (using 63x lens) of the indicated zone in the upper parts (using 10x lens, white box). Co-localization of fluorescent Dbait32H-FITC and labelled ?-H2AX appears as bright dots over DAPI counterstained nuclei.<br><br>
Detailed Description of the Invention As discussed above, the invention discloses a novel class of therapeutic molecules which can interfere, in a non gene-specific manner, with DNA repair systems in mammalian cells. These new molecules, termed Dbait molecules, are substrates for the holocomplex <br><br>
of proteins involved in the NHEJ pathway (sequence-independent pathway), particularly Ku and/or DNA-PKcs proteins, and can neutralize the DNA repair capacity of cells, thereby increasing their sensitivity to DNA damaging treatments.<br><br>
The invention thus relates to such molecules, their manufacture and their therapeutic use, particularly for treating proliferative diseases in combination with a DNA damaging treatment.<br><br>
Dbait molecules of the present invention may be defined by a number of characteristics, such as their minimal length, the presence of a free end at least, and the presence of a double stranded portion. As will be discussed below, an important feature of Dbait molecules is that their precise nucleotide sequence does not impact substantially on their activity. Furthermore, Dbait molecules may contain a modified and/or non-natural backbone.<br><br>
Accordingly, a first object of this invention resides in a nucleic acid molecule, wherein said molecule comprises a double stranded portion of at least about 16 bp, has at least one free end, and binds at least a Ku complex involved in the NHEJ pathway.<br><br>
The molecule is preferably of non-human origin (i.e., its nucleotide sequence and/or conformation (e.g., hairpin) do not exist as such in a human cell), most preferably of recombinant and/or synthetic origin.<br><br>
According to the mechanism of action of Dbait molecules, the sequence of the Dbait molecules plays little, if any role. Accordingly, in contrast with molecules used in the prior art for gene/protein-specific targeting (e.g., antisense, antigene, siRNA, aptamer, ribozyme, etc.), Dbait molecules may not have any significant degree of sequence homology or identity to known genes, promoters, enhancers, 5'- or 3'- upstream sequences, exons, introns, etc.. In other words, the action of Dbait molecules to interfere with NHEJ pathway is sequence-independent, and Dbait molecules can have less than<br><br>
70%, even less than 50% sequence identity to any gene in a human genome.<br><br>
This sequence independent mechanism of action is a hallmark of Dbait molecules, which clearly distinguishes them from other gene-specific or protein-specific (sequence dependent) therapeutic agents such as antisense oligonulceotides, small interference RNA (siRNA, shRNA and miRNA), and immunostimulating CpG oligonucleotides, as well as aptamers designed to trap a specific proteins. <br><br>
In a preferred embodiment, the sequence of the Dbait molecules has an overall degree of identity to human nucleic acid sequences which is less than about 70%, 60%, 55% or 50%. Methods of determining sequence identity are well known in the art and include, e.g., Blast.<br><br>
In a particular embodiment, the Dbait molecule does not hybridize, under stringent conditions, with human genomic DNA. Typical stringent conditions are such that they allow to discriminate fully complementary nucleic acids from partially complementary nucleic acids (Cf e.g. Sambrook et al).<br><br>
In a preferred embodiment, the sequence of the Dait molecules is devoid of CpG in order to avoid the well known toll-like receptor-mediated immunological reactions, if such effect is undesirable. CpG refers to a dinucleotide consisting of a cytosine following by a guanine.<br><br>
Considering their mechanism of action, the length of Dbait molecules may be variable, as long as it is sufficient to allow appropriate binding of Ku protein complex. The experimental section shows that the minimal length of Dbait molecules is about 16 bp, to ensure binding to a Ku complex. Preferably, Dbait molecules comprise between 16-200 bp, and most preferably between 24-100 bp. Specific examples of Dbait molecules contain 24 bp, most preferably 32 bp. As shown in the examples, such a length is sufficient to allow binding of a Ku complex comprising Ku and DNA-PKc proteins.<br><br>
Particularly preferred Dbait molecules comprise 24-100 bp, and more advantageously 32- 100 bp.<br><br>
The Dbait molecules according to the invention must have at least one free end, as a mimic of DSB. Said free end may be either a free blunt end or a 5'-/3'- protruding end. In a particular embodiment, they contain only one free end. In another particular embodiment, they contain two free ends.<br><br>
Dbait molecules can be linear or, preferably, made of hairpin double-stranded nucleic acids. In such a case, the loop can be nucleic acids, or other chemical groups known by skilled person, preferably a linker such as hexaethyleneglycol or tetradeoxythymidylate (T4). <br><br>
In a preferred embodiment, the Dbait molecules are such that:<br><br>
1 ) the double-stranded Dbait molecules are capable of being uptaken by cells/tissue body into the cell nucleus when used with pharmaceutically acceptable carriers/excipients ; 2) the at least one free end of the Dbait molecules is recognizable by the holocomplex of enzymes involved in DSB damage sensing, signaling and/or repair processes; 3) the at least one free end of the Dbait molecules is amenable by said complex to be incorporated in the tumor cell genomic DNA.<br><br>
In a particular embodiment, the Dbait molecules have a non replicative structure, due their structure and/or backbone.<br><br>
In this respect, the Dbait molecules according to the invention may have exclusively or mainly (above 50%) a native phosphodiester backbone or a chemically modified phosphodiester backbone, or another backbone with chemical groups or mixtures of chemical groups, provided the modified dsDNA remain substrates for the holocomplex involved in the NHEJ pathway, particularly Ku and DA-PKcs proteins, as well as DSB damage sensing or signaling pathway. Advantageously, the chemical modifications are intended to confer chemical stability to Dbait molecules and/or to prevent them for further replication (potential cause of mutagenic effect) upon their genomic integration if it occurs.<br><br>
They can also have sugar mimics such as 2'-O-alkylribose, 2'-O-alkyl-C4' branched ribose, cyclobutyls or other carbocyclics or hexitol in place of the pentofuranosyl group.<br><br>
Preferred Dbait comprise one or several chemical groups at the end of one or of each strand. Preferred chemical groups comprise phosphorothioates. Alternatively, preferred Dbait have nucleotides with methylphosphonate backbone.<br><br>
Other modified backbones of the invention comprise phosphoramidates, morpholino nucleic acid, 2'-0,4'-C methylene/ethylene bridged locked nucleic acid, peptide nucleic acid (PNA), and short chain alkyl, or cycloalkyl intersugar linkages or short chain heteroatomic or heterocyclic intrasugar linkages of variable length, or any modified nucleotides known by skilled person.<br><br>
US patent No. 5,677, 437 describes heteroaromatic oligonucleoside linkages. Nitrogen linkers or groups containing nitrogen can also be used to prepare oligonucleotide mimics <br><br>
(U. S. PatentsNo. 5,792, 844 and No. 5, 783,682). U. S. Patent No. 5,637, 684 describes phosphoramidate and phosphorothioamidate oligomeric compounds. Also envisioned are oligonucleotides having morpholino backbone structures (U. S. Patent No. 5,034, 506). In other embodiments, such as the peptide-nucleic acid (PNA) backbone, the phophodiester backbone of the oligonucleotide may be replaced with a polyamide backbone, the bases being bound directly or indirectly to the aza nitrogen atoms of the polyamide backbone. Other synthetic oligonucleotides may contain substituted sugar moieties comprising one of the following at the 2'position: OH, SH,OCH3,SCH3, F, OCN, OCH2CH2OCH3, O(CH2)nNH2 or O (CH2) nCH3 where n is from 1 to about 10 ; Cl to C10 lower alkyl, substituted lower alkyl, alkaryl or aralkyl ; Cl ; Br; CN ; CF3 ; OCF3. ;0-S- ; or N-alkyl ; 0-, S-, or N-alkenyl; SOCH3 ; S02CH3 ; ONO ; NO ; N3.<br><br>
Said non-replicable element(s) can be incorporated at the internal position or at the end of the double-stranded fragment. It (they) may comprise: a) a unit which cannot be used as a template for DNA replication, such as a polyethyleneglycol chain, preferably a hexaethyleneglycol chain, or any hydrocarbon chain, eventually interrupted and/or substituted by one or more heteroatoms e. g. oxygen, sulfur, nitrogen, or heteroatomic or heterocyclic groups comprising one or more of heteroatoms; b) a unit which is a blocking element as it is not amenable by DNA polymerases or exonucleases, such as any 3'- modified nucleotides, or other ones known by skilled person; c) a native oligonucleotide, such as Tn, when used in the loop of an hairpin fragment, such as a tetradeoxythymidylate (T4).<br><br>
Said strands are made by chemical synthesis, semi-biosynthesis or biosynthesis, any method of amplification, followed by any extraction and preparation methods and any chemical modification.<br><br>
The bioactivity of Dbait molecules can be assessed by in vitro and cultured cell based assays, as described e.g., in examples 2 and 3, and/or also by in vivo assays, as described e.g., in examples 4 and 5. The most easy and relevant assay is the DNA- dependent protein kinase activity assay (cf. example 2, figure 1.4). This simple assay has been so far predictive of in vivo activity of Dbait molecules. However, other cultured cell based assays, such as the assay of the inhibition of radiation-enhanced illegitimate integration is also relevant (cf. example 3, figure 2.3 &amp; figure 2.4).<br><br>
In a particular embodiment, the Dbait molecules of this invention are capable of activating DNA-PK. <br><br>
In a particular embodiment, the Dbait molecules of this invention are capable of inhibiting radiation-enhanced illegitimate DNA integration.<br><br>
In another particular embodiment, the Dbait molecules of this invention bind a Ku complex in vitro, e.g., as determined by gel shift assay. Such a Ku complex may comprise one or several Ku proteins only, such as Ku70 and/or Ku80, or a combination of one or several Ku proteins and at least a DNA-PKc protein.<br><br>
In another particular embodiment, the Dbait molecules of this invention penetrate the nucleus.<br><br>
Most preferred Dbait molecules of this invention combine several or all of the above characteristics.<br><br>
The experiments carried out in cultured cells and in xenografted tumors on nude mice and genetically modified mice have shown that Dbait molecules of this invention trigger cell/tissue lethality of tumors submitted to a radio- and/or chemotherapy.<br><br>
The invention thus also relates to adjuvant compositions to be used in association with a DNA breaking treatment, said compositions comprising a Dbait molecule such as above defined, in combination with a pharmaceutically acceptable carrier/excipient, in an efficient amount to be introduced in the nucleus of tumor cells.<br><br>
The invention also relates to a method for promoting tumor sensibility to anticancer therapies which comprises, in association,<br><br>
- introducing into cancer cell/tissue Dbait molecules such as above defined, and<br><br>
- inducing in cells, DNA breakage by a DNA damaging method.<br><br>
According to an embodiment of the invention, a transfection agent is used in said introduction step.<br><br>
Based on the protocol used in in vivo studies, the invention provides rational to establish clinical protocol of the use of Dbait molecules in combination with radiotherapy or chemotherapy. The rational underlying any protocol is that the Dbait molecules should be delivered in the nucleus of cells when DNA damaging event occurs. Therefore, Dbait molecules shall preferably be administred prior to radiotherapy, whereas they can be <br><br>
given along with chemotherapeutic agent(s) depending on the administration mode and the pharmacokinetics of each drug.<br><br>
A typical protocol comprises administration of Dbait molecules before irradiation, for example 5 hours. The use of a fractionated irradiation is particularly efficient, for example 15x2Gy in six weeks, or 6x5Gy in two weeks.<br><br>
Advantageously, said method comprises coupling the treatment with Dbait molecules with a double chemotherapy. For example 5FU and CPT 11 are injected together 3 times, 3 consecutive days, spaced by a full week of rest. Alternatively the treatment with Dbait molecules is coupled with radiotherapy.<br><br>
It will be easily adapted for humans by the one skilled in the art, particularly depending on the weight/body surface of the patient.<br><br>
In a preferred embodiment, the Dbait molecules are chemically modified Dbait molecules such as above defined and other practice in human therapy.<br><br>
In another embodiment, the Dbait molecules are not chemically modified and correspond to native nucleic acid fragments, but exhibit the characteristics of chemically modified fragments, particularly have the number of base pairs and properties defined with respect to said chemically modified Dbait molecules.<br><br>
More particularly DNA strand breakage is achieved by ionized radiation (radiotherapy) or chemical reaction (chemotherapy).<br><br>
Such a method is a new therapeutic adjuvant in conjunction with DNA damaging therapies to treatment the diseases resulting from uncontrolled cell proliferation, in particular cancer. In other words, Dbait is mainly intended to be used in anticancer therapeupies, but it may also be used in many antiproliferation treatments, such as for treating psoriasis.<br><br>
This method is addressable to treat proliferative disorders:<br><br>
They can be non malignant, such as psoriasis and vascular proliferative stenosis/restenosis.<br><br>
They can be malignant. <br><br>
The concerned organ or region can be: lung and bronchi, head and neck, gastro-intestinal tract, colorectal cancer, genito urinary tract, gynecologic organs, breast, endocrines, skin, retina, CNS, hematological organs, metastasis of known or unknown primary site, remnants (thymus for instance).<br><br>
Histological nature can be epithelial, squamous cell carcinoma, adenocarcinoma, transitional carcinoma, fibroblast/angioblast derived (sarcomas), neuronal, glial derived, endocrine, carcinoid, gastrointestinal stroma, endothelial, hematopoietic, embryonic.<br><br>
The invention also relates to the use of said non-chemically modified Dbait molecules for making anticancer drugs for treating tumors, particularly highly resistant tumors to radio- and/or chemotherapies, said drugs being used in association with a DNA breaking (e.g. damaging) treatment, particularly radiotherapy or chemotherapy.<br><br>
In vivo, the chemically modified or non-modified Dbait molecules are administrated by any appropriate route, with appropriate acceptable carrier/excipient, such as oral, or intravenous, or intratumoral administration, or sub-cutaneous injections, or topic administration, or others.<br><br>
A further object of this invention is a composition for use in association with a DNA breaking treatment, particularly radiotherapy or chemotherapy, said composition comprising at least one Dbait molecule in combination with a pharmaceutically acceptable carrier, in an efficient amount to be introduced in the nucleus of tumor cells. For instance, when the intratumoral administration is used, the said efficient amount is at least 0.01 mg per 1 cm3 of tumor, preferably 0.1 mg per 1 cm3 of tumor, most preferably 0.5 mg per 1 cm3 of tumor. The efficient amount can be administered in a daily treatment protocol (e.g., 5 days per week for 3 to 6 consecutive weeks or 3 times a week for 3 to 6 consecutive weeks). Alternatively, an efficient amount of at least 0.1 mg per 1 cm3 of tumor, preferably 0.5 mg per 1 cm3 of tumor, most preferably 1 mg per 1 cm3 of tumor, can be administered in a weekly treatment protocol for 3-6 consecutive weeks, for instance. When other administration routes are used, the one skilled in the art can adapt the amount in order to obtain an efficient amount of the Dbait molecules in the tumor of at least 0.01 mg per 1 cm3 of tumor, preferably 0.1 mg per 1 cm3 of tumor, most preferably 0.5 mg per 1 cm3 of tumor, in particular in a daily treatment protocol, or to obtain an efficient amount of the Dbait molecules in tumor of at least 0.1 mg per 1 cm3 of tumor, preferably 0.5 mg per 1 cm3 of tumor, most preferably 1 mg per 1 cm3 of tumor, in particular in a weekly treatment <br><br>
protocol. Of course, the dosage and the regimen can be adapted by the one skilled in art in consideration of the chemotherapy and/or radiotherapy regimen.<br><br>
A further object of this invention resides in the use of a Dbait molecule as defined above for the manufacture of a medicine to enhance cell (e.g., tumor) sensitivity to DNA damaging therapy.<br><br>
A further object of this invention resides in the use of a Dbait molecule as defined above for the manufacture of medicine for treating cancer in combination with a DNA damaging anticancer therapy.<br><br>
Preferably, the DNA damaging anticancer therapy is selected from radiotherapy and chemotherapy. Further preferably, the molecule is administered prior to radiotherapy and/or prior to and/or along with chemotherapy.<br><br>
Others characteristics and advantages of the invention will be given in the following examples, with reference to the attached figures and Tables.<br><br>
Examples Molecular and cellular studies as well as assays in xenografted human radio-resistant tumor (head &amp; neck squamous cell carcinoma, glioblastoma, melanoma) on nude mice and in Rasv12G x Apc1638N double mutation induced tumor in digestive track on transgenic mice were performed in order to: i) assess the biological activities of Dbait molecules; ii) validate DNA bait approach by using Dbait molecules in sensitizing anticancer therapies; iii) elucidate molecular and cellular mechanisms underlying the observed Dbait-mediated sensitization. The outcomes of these investigations are outlined and summarized in the examples.<br><br>
Example 1 : Design, synthesis and preparation of Dbait molecules.<br><br>
Two types of Dbait molecules were designed: linear or hairpin dsDNA fragments. For hairpin Dbait molecules, a hexaethyleneglycol linker or a tetradeoxythymydylate was used as loop.<br><br>
The end(s) of dsDNA stem can be protected against chemical degradation by 3'- <br><br>
exonucleases by the incorporation of phosphorothioates, methylphosphonates or 3'-3' nucleotide linkage. In principle, other chemical modifications can be used provided that they are compatible with Ku70/Ku80 binding and DNA-PKcs activation (Martensson &amp; Hammarten, 2002). Different Dbait molecules with various stem length 8 bp (DbaitdH), 16 bp (Dbait16H), 24 bp (Dbait24H) and 32 bp (Dbait32H), as well as different stem sequences were assayed. A dumbell dsDNA fragment (Dbait32C) where both ends were sealed by two hexaethylene loops was also designed, as control. Some Dbait molecules were labelled via a T tagged with fluorescein (Dbait32H-FITC), cyanine 3(Dbait32H-Cy3), cyanine 5 (Dbait32Hc-Cy5), or biotin (Dbait32H-Biot). Table 1.1 , 1.2 and 1.3 summarized the sequences and chemical structures of Dbait molecules used in this work. <br><br><br>
Dbait molecules Sequences and chemical structures<br><br>
5 ?CGCACGGGTGTTGGGTCGTTTGTTCGGATCT3 '<br><br>
Dbait32 3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGA5 '<br><br>
5 ' ACGCACGGGTGTTGGGTCGTTTGTTCGGATCT<br><br>
Dbait32-T4 ^i T<br><br>
3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGA -^ 4<br><br>
5 ?CGCACGGGTGTTGGGTCGTTTGTTCGGATCT3<br><br>
Dbait32H-po 3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGA5<br><br>
5 ?CGCACGGGTGTTGGGTCGTTTGTTCGGATCT3 ".z&gt;<br><br>
Dbait32H 3 3 '' TTGGCCGGTTGGCCCCCCAACCAAAACCCCCCAAGGCCAAAAAACCAAAAGCCTAGA5<br><br>
5 5 ??CCGGCCAACCGGGGGGTTGGTTTTGGGGGGTTCCGGTTTTTTGGTT33<br><br>
Dbait24H 3 ' TGCGTGCCCACAACCCAGCAAACA5<br><br>
5 ?CGCACGGGTGTTGGG3<br><br>
Dbait16H 3 ' TGCGTGCCCACAACCC5 :z&gt;<br><br>
5'ACGCACGG3<br><br>
DbaitdH D? 3'TGCGTGCC5'. Dbait32Ha 5 ' GCTAGGTCTGTTTGGTGGCTTTGCAGTGGCAC3 ' 3 ' CGATCCAGACAAACCACCGAAACGTCACCGTG5 ¦ Z)<br><br>
Dbait32Hb 5 ' GCTAGGCTTGTTTGCTGGGTTGTAGGCACAGC3 3 ' CGATCCGAACAAACGACCCAACATCCGTGTCG5 '.Z) Dbait32Hc 5 ' GCTGTGCCCACAACCCAGCAAACAAGCCTAGA3 .' 3 ' CGACACGGGTGTTGGGTCGTTTGTTCGGATCT5Z) Dbait32Hd 5 ' GCTAGGTCTGTTTGGTGGCTTTGCAGTGGCAC3 3 ' CGATCCAGACAAACCACCGAAACGTCACCGTG5',Z)<br><br>
5 ' GCTGTGCCCACAACCCAGCAAACAAGCCTAGA3 Dbait32Hc-3'mp 3 ' cgaCACGGGTGTTGGGTCGTTTGTTCGGATCT5 :z&gt;<br><br>
5 ' gctGTGCCCACAACCCAGCAAACAAGCCTAGA3 ' -^<br><br>
Dbait32Hc-5'3'mp 3 ' cgaCACGGGTGTTGGGTCGTTTGTTCGGATCT5 ' J) Dbait32Hc-5'5' 5 ' GCTAGGCTTGTTTGCTGGGTTGTAGGCACAGC3 ' -3'GATCCGAACAAACGACCCAACATCCGTGTCGS '.Z)<br><br>
5'ACGCACGGGTGTTGGGTCGTTTGTTCGGATCTS' -NH<br><br>
Dbait32-NH2 3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGA5 ' -NH<br><br>
5'ACGCACGGGTGTTGGGTCGTTTGTTCGGATCTS'<br><br>
Dbait32C ^33'' TGCGTGCCCACAACCCAGCAAACAAGCCTAGA5 . Z) Dbait32ss 5'ACGCACGGGTGTTGGGTCGTTTGTTCGGATCT-S' Dbait32Hcss-po 5 ' GCTGTGCCCACAACCCAGCAAACAAGCCTAGA3 '<br><br>
Table 1.1 : Sequences and chemical structures of Dbait molecules. The uppercase letters are nucleotides with phosphodiester backbone. The bold uppercase letters are nucleotides with phosphorothioate backbone. Half circle solid line symbolizes hexaethyleneglycol linker. Dbait32-T4 contains four thymines (T4) as a linker instead of a hexaethyleneglycol linker. Dbait32C is a dumbbell (closed) molecule. Dbait32Hc-5'5' has a shuffled sequence (same base composition but in different order, cf. Dbait32Hb in table 1.1 ) and a 3'-3' linkage. Dbait32Ha, Dbait32Hb, Dbait32Hc and Dbait32Hd have the same base composition but in different order as compared to the sequence of Dbait32H. <br><br>
Dbait molecules Sequences and chemical structures<br><br>
Dbait32 H-FlTC<br><br>
Dbait32 H-Cy3 5 ' ACGCACGGGTGTTGGGTCGTTTGTTCGGATC tS ' -N.<br><br>
Dbait32H-Biot 3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGAS ' -^ t = fluorescein (FITC), cyanine 3 (Cy3) or biotin (Biot)-tagged T<br><br>
Dbait8Hc-Cy3 5' GCTGTGCA3 --^<br><br>
3 ' CGACACGt 5 '-^ t = cyanine 3 (Cy3)-tagged T Dbait32Hc-Cy3 S ' GCTGTGCCCACAACCCAGCAAACAAGCCTAGAS ' ~^<br><br>
Dbait32 Hc-Cy5 3 ' CGACACGGGTGTTGGGTCGTTTGTTCGGATCt S ' — ^ t = cyanine 3 (Cy3) or Cyanine 5 (Cy5)-tagged T<br><br>
Dbait32Hd-FITC 5 ' GCTAGGTCTGTTTGGTGGCTTTGCAGTGGCAC 3<br><br>
3 ' CGATCCAGACAAACCACCGAAACGTCACCG tG5 : :&gt; t = fluorescein (FITC)-tagged T<br><br>
Table 1.2: Sequences and chemical structures of various labelled Dbait molecules as indicated. The uppercase letters are nucleotides with phosphodiester backbone. The bold uppercase letters are nucleotides with phosphorothioate backbone. The bold lowercase letters are nucleotides with methylphosphonate backbone. Half circle solid line symbolizes hexaethyleneglycol linker. Various labelled (cyanine 3 or 5, FITC) DbatdHc, Dbait32H, Dbait32Hc and Dbait32Hd molecules are indicated.<br><br>
Dbait molecules Sequences and chemical structures<br><br>
Dbait64 5 ?CGCACGGGTGTTGGGTCGTTTGTTCGGATCTACGCACGGTCGTTTGTTCGGTGTTGGCGATCTS '<br><br>
3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGATGCGTGCCAGCAAACAAGCCACAACCGCTAGAS '<br><br>
Dbait64L<br><br>
5 'ACGCACGGGTGTTGGGTCGTTTGTTCGGATCT ACGCACGGTCGTTTGTTCGGTGTTGGCGATCTS'<br><br>
3 ' TGCGTGCCCACAACCCAGCAAACAAGCCTAGA TGCGTGCCAGCAAACAAGCCACAACCGCTAGA5 '<br><br>
Table 1.3: Sequences and chemical structures of 64-bp Dbait64 and Dbait64L molecules. The uppercase letters are nucleotides with phosphodiester backbone. The bold uppercase letters are nucleotides with phosphorothioate backbone. Solid line symbolizes a hexaethyleneglycol linker.<br><br>
All Dbait molecules were made by automated solid phase oligonucleotide synthesis <br><br>
(Eurogentec, Belgium). They were purified by denaturing reverse phase HPLC. Denaturing capillary gel electrophoresis. MALDI-TOF/LC-MS were used for quality control. More than 90% of oligonucleotides are full length. All samples were lyophilized before shipping.<br><br>
Upon reception, all samples were dissolved in bi-distilled water. The concentrations of Dbait molecules were calculated from absorbance at 260nm (Cantor &amp; Warshaw, 1970) under denaturing condition (60°C-90°C depending on the thermal stability of Dbait molecules). The concentrations of fluorescent dye tagged Dbait molecules were calculated from absorbance at the appropriate wavelength of the particular dye (FITC: e=80000 M"1.cm"1 at 490nm; Cy3: e=150000 M"1.cm"1 at 550nm; Cy5: e=250000 M"1.cm"1 at 650nm). The dumbell dsDNA fragment (Dbait32C) was prepared by annealing and ligation by DNA T4 ligase (BioLabs) of two semi hairpins carrying hexaethyleneglycol linker and with 3'-protruding and complementary ends.<br><br>
Based on the thermodynamic and kinetic considerations, the following protocols were used for preparing the samples of Dbait molecules, according to their molecularity:<br><br>
- For bi-molecular Dbait molecules (Dbait32, Dbait32-NH2, Dbait64 and Dbait64L): The mixture of 1 :1 stock solution (highest concentration possible) of each strand in bi- distilled water has to be heated at 900C for 5 minutes for complete denaturation of each strand. The annealing was carried out by smooth return to room temperature (the samples are typically left in water bath) and the resulting duplex molecules were stored in aliquot at -200C.<br><br>
- For mono-molecular Dbait molecules (hairpin):<br><br>
The solution containing 200 µM of hairpin Dbait molecules in bi-distilled water has to be heated at 900C for 5 minutes for complete denaturation. The annealing has to be carried out by chilling the samples into ice-water (0°C). Storage of aliquots was at -20°C.<br><br>
Example 2: Biochemical analysis of Dbait molecules.<br><br>
As the first step to dissect the mechanism of action of Dbait molecules, a series of band- shift assays were carried out with different 32P radio-labelled Dbait molecules in the presence of nuclear protein extracts from Hep2 cells according to standard protocol. Typically, 10 nM 32P radio-labelled Dbait molecules were incubated in the presence of various concentrations of nuclear proteins (0, 10, 20, 40, 80,160, and 320ng/µl_) at 30°C for 10 minutes in TBE buffer. Then the samples were loaded on a 5% acrylamide native <br><br>
gel. The electrophoresis was run at 95V for 2 hours at 4°C. The gel was dried and scanned by phosphorimager (Molecular Dynamics).<br><br>
Figure 1.1 shows the retarded band pattern of the titration of Hep2 nuclear protein extracts with various Dbait molecules of different lengths. Except the the shortest 8 bp long Dbait molecule (DbaitdH), up to 2 retarded bands were observed for longer Dbait molecules:. One retarded band was observed for the 16-, 24-bp long Dbait molecules<br><br>
(Dbait16H and Dbait24H), whereas two retarded band were observed for 32-bp long Dbait molecules (Dbait32H, Dbait32H-po and Dbait32). For 32-bp Dbait molecules, the intensity of the retarded band 1 increases and then decreases as the concentration of protein increases, while the intensity of the retarded band 2 increases as a function of the concentration of nuclear protein extracts.<br><br>
The combination of immunobinding and band-shift assays with mouse monoclonal anti- Ku70 antibody (Santa Cruz Biotechnology) revealed that retarded bands 1 and 2 contain the Ku complex Band 1 and 2 were further shifted into bandi * and 2* upon addition of anti-Ku70 antibody (figure 1.2). It is likely that band 1 has one Ku70/80 complex bound to the 16 to 32-bp Dbait, whereas the band 2 has two Ku70/80 complexes bound to the 32- bp Dbait molecules. Control experiments performed with purified Ku proteins confirmed this interpretation.<br><br>
The identification of Ku proteins clearly indicates that Dbait molecules interact with NHEJ machinery in a length-dependent manner.<br><br>
DNA end-joining was monitored by incubating 32P-labeled 605-bp linear DNA fragments with Hep2 nuclear extract in the presence of various Dbait molecules. Ligation products migrate as dimers, trimers or tetramers, starting 605-bp monomer.<br><br>
Figure 1.3 shows the effect of various Dbait molecules on DNA end-joining reaction in the nuclear extracts of Hep2 cells. Approximately 16% of the 32P-labeled blunt linear duplex<br><br>
DNA molecules were ligated into dimers and trimers during the first 2 h of incubation. The amount of high molecular weight ligation products increased up to 30% of the total input<br><br>
DNA after 16 h. When Dbait32H was added to the reaction in a 100-fold molar excess compared to the linear 32P-labeled fragment the reaction was strongly inhibited. Similar inhibition of end-joining activity was also observed with extracts prepared from HeLa cells<br><br>
(data not shown). In most experiments, Dbait32H was added simultaneously with the labeled DNA fragments to the nuclear extract. When Dbait32H was incubated with the <br><br>
extract for 30 min before addition of fragments, the extent of inhibition was similar. By contrast, when Dbait32H was added 30 min after the nuclear extract there was no inhibition of the ligation (data not shown). These data indicate that Dbait molecules are competitors of DNA end-joining reaction, but do not displace the bound complex.<br><br>
Various Dbait molecules were tested for their effect on the cell-free assay by adding the molecules to nuclear extract and incubating for 2 h with the DNA fragments. Ligation was largely unaffected by the short molecule (DbaitdH), the single strand 32-nt long molecule (Dbait32ss), or the dumbell molecule (Dbait32C). Dbait24H and Dbait16H, which bind only one Ku heterodimer, inhibited the ligation as efficiently as Dbait32H. These data indicate that DNA fragment religation is strongly inhibited by Dbait molecules that are able to recruit Ku.<br><br>
DNA-PK activity was monitored using the kit SignaTECT DNA-dependent Protein Kinase Assay System (Promega, Madison, USA). Increasing amounts of Hep2 nuclear extract (cleaned of endogenous DNA by DEAE-Sepharose filtration) were assayed in the presence of 250 nM Dbait. Extract, biotinylated peptide substrate and various amount of nuclear extract were incubated 5 minutes at 300C with (?-32P)ATP according to manufacturer indications. The biotinylated substrate was capture on streptavidine membrane, washed and counted in a scintillation counter. Percentage of phosphorylation is calculated by dividing the bound radioactivity by the total count of (?-32P)ATP per sample. Reactions (10 µl) were carried out in 60 mM KOAc, 100 µg/ml BSA, 0.5 mM Mg(CI)2, 1 µl T4 DNA ligase 10X buffer (Promega, Madison, USA). Nuclear extract and Dbait were incubated 2 minutes before addition of 32P-labeled DNA (10 ng). Samples were incubated various times at 37°C before ligation being stopped by 20 mM EDTA and 1 mg/ml Proteinase K addition. Ligation products were analyzed by electrophoresis through 0.7% agarose gels followed by autoradiography and quantification by phosphorimaging.<br><br>
Figure 1.4 shows DNA-dependent protein kinase (DNA-PK) activity of a number of 2µg Dbait molecules with various lengths, sequences and chemical structures including modified backbones, in 1.5µg of Hep2 nuclear protein extract. The kinase activity depended directly on the length and structure of the double-stranded 'stem' of the Dbait molecule. High DNA-PK activation was observed with the 32-bp long Dbait molecules which were bound by two Ku dimmer complexes. The Dbait molecules that bound only one Ku dimer (Dbait16H and Dbait24H) were as inefficient as the short DbaitdH that did not bind Ku. Similarly, the single strand Dbait32ss/Dbait32css and dumbell Dbait32C, <br><br>
which have no free double-stranded ends, did not activate DNA-PK. In addition, various backbone modifications (phosphothioate, methyphosphonate, 3'-3' linkage) at the free blunt end (up to 3-bp), as well as the internally tagged ligands (e.g. fluorescent dyes), are capable of activating DNA-PK activity. It is worthy noticing that the DNA-PK activity does not significantly, if any, depend on the sequence of Dbait molecules, as shown by Dbait32H, Dbait32Hb, Dbait32Hc and Dbait32Hd.<br><br>
This simple cell free DNA-PK activity assay points out that only the length (at least about 32-bp) and the double stranded DNA with a free end of Dbait molecules are required for the kinase activation, regardless their sequence and chemical modifications to some extent. This is consistent with the implication of DNA-PK in the NHEJ pathway, a sequence independent DNA end joining mechanism.<br><br>
Example 3 : In vitro activity of Dbait molecules The activity of Dbait molecules in cultured cells was studied by clonogenic survival assay in two radio-resistant human cancer cell lines derived from a female cervix carcinoma (HeLa) and from HNSCC (Hep2) in association with ionizing radiation, by the inhibition of illegitimate integration of exogenous DNA fragment and by detecting the persisting DSB foci after irradiation in the cells transfected by Dbait molecules.<br><br>
Established human cell lines Hep2 (head and neck squamous cell carcinoma, HNSCC), LU1205 and SK28 (melanomas) were used for animal studies. Studies of cells in culture were performed using Hep2, HeLa S3 (epithelia cervical carcinoma), MO59K and MO59J (glioblastoma). Cells were grown at 37°C in monolayer cultures in complete DMEM containing 10% heat-inactivated fetal bovine serum (FBS; Invitrogen, Cergy Pontoise, France) and antibiotics (100 µg/ml streptomycin and 100 µg/ml penicillin) under conditions of 100% humidity, 95% air and 5% CO2. LU 1205 were grown in MCDB containing 4% heat-inactivated FBS, 1 % glutamine and antibiotics (100 µg/ml streptomycin and 100 µg/ml penicillin).<br><br>
Exponentially growing cells in six-well plates were harvested and incubated with 700 ml of complete DMEM containing a mixture of Dbait molecules and Superfect reagent (Qiagen, Courtaboeuf, France) in a ratio of 10 µl Superfect per µg DNA. After 5 h at 37°C under standard conditions, the cells were washed with PBS and complete DMEM was added. Where indicated, cells were exposed to irradiation either in one session (10 Gy) 5 h after the beginning of transfection or in four sessions of 0.5 Gy administered with a 137Cs unit (1 Gy/min) 3, 4, 5 and 6 h after the beginning of transfection and allowed to grow for two <br><br>
weeks. When plasmid integration was measured, 2 µg of plasmid was added to Dbait transfection and puromycin (1.7 µg/ml) was added to the growth medium 48 h after transfection. Plasmid transfection efficiency was estimated by analyzing 10,000 cells for GFP expression on a FACScan flow cytometer (FACSalibur, Beckton-Dickinson, USA).<br><br>
3.1) Induced cell lethality<br><br>
Upon 8 hours transfection of Dbait molecules in HeIa cells and four irradiation with 0.5 Gy fractionated irradiation spaced 2 hours (4x0.5 Gy), performed with ?-rays from a 137Cs source, a significant reduction of clonogenic survival was observed as compared to untransfected cells. The results are given on figure 2.1 wherein Panel A gives the dose- dependence of normalized survival clone number in the presence of Dbait32 and Dbait32H, and Panel B, the normalized survival clone number in the presence of different Dbait molecules at 83 nM (concentration in culture medium). Cell culture was in MEM supplemented with 10% serum. Superfect (Qiagene) was used as transfection agent according to the manufacturer's instruction. Clonogenic survival was estimated as the number of treated cells forming colonies on the number of untreated cells.<br><br>
The effect depends on the length and the chemical nature of Dbait molecules in a dose- dependent manner. In this assay, the hairpin Dbait molecules (Dbait32H, Dbait32-T4 and<br><br>
Dbait24H) and the linear double-stranded Dbait molecules (Dbait64 and Dbait64L) significantly reduced clonogenic survival. It is worth to note that the dumbell Dbait32C molecule which lacks free dsDNA ends (capped by hexaethyleneglycol linker at both ends) did not exhibit any effect. The chemical nature of loop did not matter (Dbait32H versus Dbait32-T4). These observations indicate that some of the Dbait molecules can effectively sensitize cells to ionizing radiation in cultured cells.<br><br>
Figure 2.2 confirms the previous observations, and provides additional data that either single strand or short 8-bp Dbait molecules did not have any effect. In addition, it shows that the effect was not sequence-dendent (Dbait32H versus Dbait32Hc).<br><br>
treatment survival<br><br>
Cell line Irradiation Transfection %<br><br>
MO59K no no 100 (4.41 ) 10 Gy no 9.83 (0.61 ) 10 Gy Dbait32Hc 3.47 (1.01 )<br><br>
MO59J no no 100 (2.97) DNA-PK-null 10 Gy no 3.97 (0.38) 10 Gy Dbait32Hc 3.98 (0.35) <br><br>
Table 2: Cell survival in DNA-PK competent cell (MO59K) and DNA-PK deficient cell<br><br>
(MO59J) after treatment with radiation and Dbait. Cells were diluted and plated to form colonies on flasks that were then either irradiated with 10 Gy and/or transfected with 2µg<br><br>
Dbait32Hc. When both treatments were conjugated the irradiation was performed 5 h after transfection. Survival is calculated as the number of cells forming clones after treatment divided by the number of cells forming clones in the non-treated sample. The mean value and the standard deviation (in brackets) were calculated from three independent experiments.<br><br>
The effect of Dbait32Hc transfection on DNA-PK wild-type and mutant cell survival after ?- irradiation was estimated by colony formation. Survival after irradiation decreased from 9.83% in untransfected MO59K cells to 3.47% in Dbait32Hc-transfected cells. In the corresponding DNA-PK-null mutant cell line, MO59J, by contrast, survival was unaffected by transfection with Dbait32Hc. The level of survival was similar in the irradiated and Dbait32Hc-transfected wild-type cell line to that observed in the DNA-PK-null mutant cells after only irradiation (3.47% compared to 3.97%). This suggests that Dbait32Hc inhibits DNA-PK-dependent repair in the wild-type transfected cells.<br><br>
3.2) Inhibition of illegitimate integration of exogenous DNA by Dbait molecules<br><br>
Ionizing radiation is known to improve illegitimate integration of exogenous DNA, a process termed radiation-enhanced integration. HeIa cell culture was used for this assay. Cells were transfected during 8 hours by 2µg of a linear plasmid carrying the gene coding for neomycin resistance, and three different ratio of DNA/superfect (1 :2, 1 :5, 1 :10). During transfection time, the cells were exposed to different irradiation protocols: no irradiation, one single irradiation of 1 Gy and 2Gy, as well as a 2Gy irradiation delivered by fractionated doses of 0.5 Gy every 2 hours (4x0.5Gy). The integration of the plasmid was monitored by selection of NeoR cells growing in a medium containing 0.6 mg/ml of G418. Plasmid integration was significantly enhanced by the fractionated irradiation protocol. When 2µg of Dbait32H molecules were added to the transfection mix, the radiation- enhanced integration was abolished (figure 2.3).<br><br>
Figure 2.4 provides additional data which shows that only the 32-bp Dbait molecules (Dbait32H and Dbait32Hc) were capable of inhibiting radiation-enhanced illegitimate integration of a circular plasmid carrying the gene conding for puromycin resistance, <br><br>
whereas shorter Dbait molecules (Dbait16H, DbaitdH), single strand or dumble Dbait molecules (Dbait32ss and Dbait32C) were not effective. Wortmann (Wort) and NU7026 (NU) which are known inhibitors of PI3K (including DNA-PK) were used as control.<br><br>
These experiments showed that the radiation-enhanced illegitimate integration of exogeneous DNA which required Ku, DNA-PK and ATM proteins (Nimura et al., 2002) is inhibited by 32-p Dbait molecules in a sequence-independent way, as expected, as the mechanism of action of these Dbait molecules act through the kidnapping the proteins involved in a NHEJ pathway.<br><br>
3.3) Persistence of DSB sites after irradiation in the cells transfected by Dbait molecules<br><br>
DSB sites in nuclei can be monitored by immunofluorescence of ?-H2AX antibody which binds on the phosphorylated H2AX (a variant of the histone H2A). Most of the ?-H2AX foci appear rapidly after irradiation and disappeared as DSB repair process progressed.<br><br>
Transfection and irradiation protocoles were similar to those described above. For immunodetection, the cells were grown on surface coverslip in 5 cm diameter Petri dishes, transfected with 2 µg Dbait32H-FITC molecules labeled with FITC with Superfect (Qiagene) according to the manufacturer's instruction. Four hours after the beginning of the transfection, cells were irradiated (2Gy), then rest for 2 hours in the medium at 37°C. After 3 washing cycles, the cells were fixated with 2% PFA for 10 minutes. After one additional washings, the presence of ?-H2AX was detected with rabbit anti- ?-H2AX antibody (441 1 -PC, Trevigen) diluted 1/100 in 1x PBS, 1 % BSA. Cells were washed three times with 1x PBS, 0.5% TritonX-100, then incubated for 1 hour at room temperature with rhodamine-conjugated goat anti-rabbit antibodies diluted 1/100 in 1x PBS, 1% BSA. Cells were visualized by epifluorescence microscopy.<br><br>
Figure 2.5 shows the results obtained with HeIa cells transfected by fluorescent Dbait32H- FITC molecules at 2 hours after 2Gy irradiation. Left panel: fluorescence of Dbait32H-<br><br>
FITC (bright dots and patches) and DBS foci detected by immunofluorescence of ?-H2AX antibody in nuclei; Right panel: the same image of nuclei with DSB foci detected by immunofluorescence of ?-H2AX antibody and DAPI counterstaining. The arrows at the lower left corner show the absence of Dbait32H-FITC and ?-H2AX signal in nucleus. The arrows at the upper right corner show the co-localized Dbait32H-FITC and ?-H2AX signals. As shown on figure 2.5, DSB sites persisted in HeIa cells transfected by<br><br>
Dbait32H-FITC molecules two hours after irradiation (2Gy), as shown by double <br><br>
fluorescent labeling withDbait32H-FITC and that of ?-H2AX antibody. It is worth to note that the DSB foci were almost undetectable in the cells that were not efficiently transfected by Dbait32H-FITC. These data suggest that DSB repair was impaired in the cells efficiently transfected by Dbait32H, while the DNA repair was complete in the cells less well transfected .<br><br>
Western blotting was performed using rabbit monoclonal anti-phosphoThr68-Chk2 (Cell Signaling Technology, Denver, USA), monoclonal anti-ß-actin clone AC-15 (Sigma, MS, USA), anti-H2AX (Cell Signaling Technology, Denver, USA) and a mouse anti-phospho- histone H2AX (SeM 39) (Upstate, Tempcula, CA, USA).<br><br>
Figure 2.6 upper panel shows the western blot analysis of the phosphorylated form of histone H2AX (?-H2AX) as compared to the total H2AX protein phosphorylation of the histone H2AX by PIKKs. Hep2 cells were transfected 5 hours with various Dbait molecules 32Hc, 24H, 16H and 8H for 5 h, or not transfected. They were irradiated at the end of transfection, incubated for one hour and then analyzed. It shows that Dbait32Hc greatly enhanced the phosphorylated form of H2AX (?-H2AX) as compared to the control (both irradiated cells, or none). Other shorter Dbait molecules had much less or none effect.<br><br>
To investigate the effect of Dbait on ionizing radiation-induced ?-H2AX foci formation and loss, Dbait32H-Cy3 were transfected in the Hep2 cells by superfect (Qiagen). Transfected or untransfected Hep2 cells were irradiated at 10 Gy. Cells were fixed at different times (0 min, 30 min, 1 hour, 5 hours, 24 hours, 48 hours, 72 hours and 7 days). Primary mouse monoclonal antibody for g-H2AX (ser139) (Upstate, Tempcula, CA, USA) was used in 1/500 dilution and incubated for 2 hours at 00C, then washed by PBS buffer and incubated with Alexa 488-conjugated second antibody anti mouse IgG (Molecular Probe, Eugene, OR, USA) diluted in 1/200 for 1 hour in dark room.<br><br>
The kinetics of the persistence of DSB sites was revealed by ?-H2AX in the irradiated cells by FACScan flow cytometer (FACSalibur, Beckton-Dickinson, USA). Figure 2.6 lower panel shows that the level of ?-H2AX remained high and long in the Dbait32H transfected cells as compared to the controls (either irradiated but untransfected or untreated cells). This experiment indicated that Dbait32H substantially retarded the repair of ionizing radiation-induced DSBs. <br><br>
Example 4: Effects of Dbait molecules in GMA32 cell line and their association with irradiation or mitotic inhibitors.<br><br>
The GMA32 Chinese hamster fibroblast cells permissive of DNA breaks were maintained in MEM medium (Gibco) supplemented with 1 mM sodium pyruvate, 2mM glutamine, 1x MEM non essential amino acids, 1x penicillin/streptomycin and 10% horse serum.<br><br>
Typically, 2x105 to 4x105 cells were seeded in medium without antibiotics, in 5cm diameter<br><br>
Petri dishes 24 hours before the transfection of different Dbait molecules (4.5µg) with lipofectamine 2000 (Life Technologies) as transfection agent (in a 1 :3 ratio), according to the manufacturer's instructions. At the end of the transfection the cells were either irradiated (4Gy) or treated with mitotic inhibitors: nocodazole (20OnM), navelbine (10OnM) or taxol (20OnM).<br><br>
About 16 hour later the drug was removed and the cells were allowed to recover. Cell irradiation was performed with ?-rays from a 137Cs source. After a 24 hours recovery, the cells were collected and used either for FACS, western blot analyses or to determine the clonogenicity (survival) and the effect of each treatment.<br><br>
Figure 3.1 shows FACS analyses of the untreated GMA32 cells, the cells transfected alone, or transfected with different Dbait molecules by lipofectamine, but without further irradiation or mitotic inhibitor treatment. The M1 phase represents the percentage of cells in sub-G1 stage indicative of cell death. Significant cell death was observed only in the presence of double-stranded Dbait32 and hairpin Dbait32H molecules, whereas hairpin Dbait16H and single-strand Dbait32ss induced intermediate and moderate cell death, respectively. The shortest hairpin DaitdH failed to trigger cell death as compared to the control (cells transfected by lipofectamine alone).<br><br>
The experiments were performed with a FACS calibur flow cytometer (Becton Dickinson). Cells were collected, suspended in 1 ml of cold GM buffer (6.5mM glucose, 137mM NaCI, 5.4mM KCI, 2mM Na2HP04,1 mM KHPO4, 0.5mM EDTA), and stored at 4°C for at least 2 hours after addition of 3ml of cold 100% ethanol.<br><br>
At that stage, cells were eventually washed with 1x PBS, then stained for 30 minutes at room temperature in Pl solution (50µg/ml propidium iodide, 25µg/ml RNase A in 1x PBS buffer). 10,000 events were analyzed with Cellquest software, and cell aggregates were gated out. The percentage of cells with a sub-G1 content of DNA was scored.<br><br>
Under the same conditions, the immunodetection of DSB foci of ?-H2AX phosphorylated <br><br>
on serine 139 by ?-H2AX labeling (bright dots or patches in nuclei) was performed in the untreated GMA32 cells, the cells transfected alone, or transfected with different Dbait molecules by lipofectamine. The counterstaining of cell membranes and nuclei were achieved byFITC-DiOC6 and DAPI. Similar effects of Dbait molecules were observed (Figure 3.2). This experiment shows that both double-stranded Dbait32 and hairpin Dbait32H can effectively trigger similar cell response as if DNA damages were occurred in nuclei. This provides visual evidence that these Dbait molecules can be used for trapping proteins involved in DSB repair via NHEJ pathway.<br><br>
For immunodetection, the cells were grown on coverslip in 5cm diameter Petri dishes 24 hours before the transfection with different Dbait molecules. One day after the transfection, FITC-DiOC6 (Molecular probes) was added in the medium 5 minutes at 37°C (to counterstain the membranes). After 3 washing cycles, the cells were fixated with 4% PFA for 20 minutes.<br><br>
After additionnal washing, ?-H2AX phosphorylated on serine 139 (?-H2AX) was detected with rabbit anti- ?-H2AX antibody (441 1-PC, Trevigen) diluted 1/100 in 1x PBS, 1 % BSA. Cells were washed three times with 1x PBS, 0.5%, TritonX-100, then incubated for 1 hour at room temperature with goat anti-rabbit antibodies Alexa 594 (Molecular Probes) diluted 1/100 in 1x PBS, 1% BSA. Cells were visualized by epifluorescence microscopy.<br><br>
Further experiment was carried out in order to look for evidence of DNA damage signaling. The protein p53 is a well known major protein in mediating DNA damage signaling and in coordinating appropriate responses (DNA repair, apoptosis, etc.) by changing its phosphorylation status. In particular,, the phosphorylation of serine 15 residue is involved in the interaction with MDM2 protein which acts as a feed back control Thus, the phosphorylation status of the serine 15 of p53 was assessed by Western blot. Figure 3.3 shows that the p53 serine15 was highly phosphorylated when cells were transfected by either double-stranded Dbait32 or hairpin Dbait32H molecules, whereas the shorter hairpin Dbait16H induced moderate phosphorylation. Neither the shortest DbaitdH nor single strand Dbaut32ss molecules were able to induced significant phosphorylation on the serine 15 of p53 protein.<br><br>
This experiment provides additional evidence that the presence of both double-stranded Dbait32 and hairpin Dbait32H in GMA32 cells was detected as DNA damage and induced the signal to transducer responses such as p53 protein phosphorylation, likely through ATM activation pathway. <br><br>
For Western blot analysis, cells were lysed in Laemmli buffer. Equal amounts of lysates were resolved in 12% polyacrylamide gels. Proteins were transferred to nitrocellulose membranes, which were blocked with 5% nonfat milk (1 hour) before overnight incubation with anti-p53Ser15 antibody (9284, Cell Signaling) diluted 500 folds in TBST buffer (1OmM Tris-HCI pH7.5,150mM NaCI, 0.1 % Tween 20) containing5% nonfat milk. Blots were then incubated with horseradish peroxidase-conjugated goat anti-rabbit IgG secondary antibodies (P0448, Dako) diluted 1/5000 in TBST. Protein-antibody complexes were detected by enhanced chemiluminescence (RPN2106 ECL, Amersham).<br><br>
The effect of radiosensitization and chemosensitization of Dbait molecules in GMA32 cells was evaluated by clonogenicity (clonal survival) assay. For the clonogenicity assay, serial dilutions were made after counting the cells to seed 5cm Petri dishes with different amounts of cells. The number of cells range from 100-200 (control cells) to 3000 (transfected or/and treated cells). Ten days after, the cells (forming clone) were fixed with 4% paraformaldehyde (20 minutes), then colored with methylene blue (15 minutes), and the number of clone in each plate (in triplicates) was scored.<br><br>
Figure 3.4 shows that the radiosensitization to 4Gy irradiation was observed in GMA32 cells transfected with either double-stranded Dbait32 or hairpin Dbait32H molecules. In addition, the chemosensitization was also observed for GMA32 cells transfected with either double-stranded Dbait32 or hairpin Dbait32H molecules when they were treated by mitotic inhibitors (20OnM nocodazole, 10OnM navelbine (vinorelbine) or 20OnM taxol<br><br>
(paclitaxel)). These drugs are known as potent inhibitors of either polymerization or depolymerization of microtubules. Dbait32 and Dait32H were able to enhance to cytotoxic activity of these mitotic inhibitors..<br><br>
Example 5: Radiosensitization of the treatment of xenografted human tumors on nude mice In vivo activity of Dbait molecules in association with radiotherapy was assessed by using nude mice xenografted with human tumors by subcutaneous injection of radio-resistant cell lines (Hep2 derived from head and neck squmous cell carcinoma, HNSCC) or tumor fragments (previously obtained by subcutaneous injection of the U87 cell lines derived of glioblastoma).<br><br>
Investigations were mainly carried out on the mice xenografted with radio-resistant human HNSCC tumors in order to establish proof of concept in vivo. Irradiation was performed <br><br>
with ?-rays from a 137Cs source with appropriate protection of mice in order to perform localized irradiation of tumors. Typical assay condition consists of intratumoral injection of an appropriate preparation of 1 nmole Dbait molecules with transfecting agents (cationic dendrimer (Superefct, Qiagene), dioctadecylamidoglycyl-spermine (DOGS, Polyplus transfection), polyethyleneimine (PEI, Polyplus Transfection) according to manufacturer's instruction, 5 hours prior irradiation. A total dose of 30 Gy was delivered in 5 weeks: i) 3x2Gy/week (about one every two days); ii) 5Gy/week; iii) 15Gy/2 weeks.<br><br>
The size of tumor was measured 2-3 times a week. Treatment by irradiation and intratumoral injection of MEM medium (the Dbait dilution buffer), was used as a control of irradiation treatment without Dbait. The volume of tumor was calculated (V=2xaxb2, where a=length, b=width). The ratio of volume measured at t time over the initial volume (VtA/,) was used as indicator of tumor progression. The mice were followed up to 100 days. At least 4 independent series of six animals were tested.<br><br>
The results are illustrated by figure 4.1 (Panel A: Untreated arm (n=38; Panel B: control arm with 20µl culture medium (MEM) + 3x2Gy/week irradiation (n=30); Panel C: the arm with 1 nmole (20µg) Dbait32H + 3x2Gy/week irradiation (n=35).<br><br>
MEM or Dbait32H was delivered by intratumoral injection 5 hours prior irradiation. The fractionated irradiation dose (2 Gy) was given one of every two days, three times a week. The treatment lasted 5 weeks totaling 30 Gy irradiation. The dots represent the time course of tumor volume of each mouse. The solid lines are the best polynomial fitting. Panel D shows a Kaplan-Meyer plot of all mice of which the increase in tumor volume (V1A/,) 
<br>
A significant amount of data has been accumulated on the arm of Dbait32H with 3x2Gy/week irradiation (panel C, n=35) which clearly showed radiosensitization as compared to the control arms: untreated (panel A, n=38), MEM + 3x2Gy (panel B, n=30). The Man and Whitney statistical test gave p-value = 0.00067 for the arm of Dbait32H + 3x2Gy versus MEM + 3x2Gy. The same trend was observed in a Kaplan-Meyer plot of mice with a tumor volume (VtA/, 
<br>
Further investigations were subsequently carried out on mice with xenografted human HNSCC, U87, LU1205 and SK28 tumors in order to define molecular features of Dait molecules and optimal protocol for in vivo activity. The data obtained from the studied cohort were consistent with molecular features of Dbait molecules observed in <br><br>
biochemical and in vitro studies (cf. examples 2, 3 and 4). In addition, it was shown that the radiosensitization is dependent on the dwell time between the intratumoral injection of Dbait32H and the ionizing radiation: 5 hours &gt;&gt; 1 hour.<br><br>
Radio-sensitization was also observed in mice xenografted with human glioblastoma tumors. The glioblastoma is the highest grade of brain-tumor, and is characterized by its extraordinary aggressive progression with fast fatal outcome and resistance to radio-and chemotherapies. 2-3 millions of U87 cells derived from human glioblastoma was first injected subcutaneously in nude mouse. The grafted tumor was then took out and used to seed subsequently other nude mice by subcutaneous transplant of about 8mm3 glioblastoma tumor.<br><br>
Table 3.1 shows data of a pilot series of xenografted human glioblastome tumors on nude mice. 50% mice in the arm which received Dbait32H (I nmole) by intratumoral injection and irradiation (1x15 Gy/week or 3x5Gy/week, followed by one week rest, then second treatment cycle, the total dose of ionizing radiation was 30Gy) had tumor volume 
<br>
Assay groups Number of mice where<br><br>
(xenografted Tumor volume 
<br>
Glioblastome) At the day 25 (6 mice per group)<br><br>
Untreated 0 / 6<br><br>
PBS + 1x15 Gy /week 0 / 6 Dbait32H+- 1x15 Gy /week 3 / 6 PBS + 3x5 Gy /week 0 / 6 Dbait32hH- 3x5 Gy/week 3 / 6<br><br>
Table 3.1 : Assay of radiosensitization of xenografted human glioblastoma on nude mice by Dbait32H (1 nmole/intratumoral injection). Two protocols of irradiation (5 hours after intratumoral injection) were used: 1x15Gy/week, or 3x5Gy/week, followed by one week <br><br>
rest, the second treatment cycle. The total irradiation dose was 30Gy. Control groups were the untreated or the groups received saline solution (PBS) injection.<br><br>
Based on these encouraging in vivo data which provided evidence that the Dbait molecules can efficiently enhance the efficacy of radiotherapy, further experiments were designed and carried out to provide additional data related to the use of the Dbait molecules as an adjuvant agent to sensitize radiotherapy, and thus to strengthen the proof of principle of DNA bait approach in anticancer therapy.<br><br>
Figure 4.2 shows the distribution of cyanine 3 labeled Dbait32H in Hep2 (HNSCC cell line) xenograft tumor in nude mice. 20µg Dbait32H-Cy3 formulated with Superfect (transfection agent) was injected into 1.5cm3 Hep2 tumor. The mice were sacrificed 6 hours after the injection. The tumors were taken out and cryo-sliced for analysis without fixing. DAPI was used for nuclei staining. The fluorescence of cyanine 3 shows that the Dbait32H-Cy3 molecules were distributed in the tumor tissue from blood capillary vessel, and were localized in the cell nucleus.<br><br>
Figure 4.3 shows another experiment on Hep2 xenografted tumors as described in figure 4.1. Tumor growth was monitored during treatment and after in four groups of 10 animals with different treatments (untreated, treated with Dbait32H alone, treated by irradiation alone, and the combined Dbait32H and irradiation). Individual tumor growth is indicated for each animal. The treatment protocol was the same as described in figure 4.1. The experiment started when the volume of Hep2 tumors reached 150-200mm3. For each treatment session, 20µg Dbait32H formulated with polyethyleneimine (PEI, Polyplus Transfection, Strasbourg, France) according to the manufacturer's instruction was injected into tumor 5 hours prior 2Gy irradiation. Tumor growth was greatly reduced in group treated by the combined Dbait32H and irradiation as compared to the groups treated by irradiation or Dbait32H alone.<br><br>
Figure 4.4 shows Kaplan-Meier representation of survival of nude mice subcutaneously xenografted by Hep2 tumor. For ethical reason, the animals were sacrificed when their tumors reached 2cm3. This end-point was used as death in survival analysis. The treatment protocol was described in figure 4.3. The five groups were included: untreated, mock-tranfected and irradiated, treated by combined irradiation and increasing amount of Dbait32H (20, 60 and 120µg/session). Number of animals for each group is indicated in table 3.2. A clear dose-dependent effect was observed in the groups treated by Dbait32H and 2Gy irradiation. Pictures of tumors representative of the groups (untreated, treated <br><br>
with 20 and 60µg Dbait32H/session associated with 2Gy irradiation) which were taken 15 days after beginning of treatment, at the end of treatment (35 days) and 13 days after the end of treatment (35+13 days). They provided clear visual comparison of the benefit of the combined Dbait32H and irradiation treatment.<br><br>
Figure 4.5 shows the histological analysis of xenograted Hep2 tumors at mid-course treatment (7 sessions). Tumors were taken out 20 days after beginning of various treatment protocols as indicated in figure 4.3. They were fixed in formalin and tissue sections were stained with hematoxylin, eosin and safran. Two tumors for each treatment protocol were analyzed by microscopy. The enhancement of necrosis and apoptosis was observed in the tumors treated by combined Dbait32H and irradiation as compared to the tumors treated by irradiation alone.<br><br>
Figure 4.6 shows NMR imaging of xenograftd Hep2 tumors at the mid-course treatment (7 sessions). Three representative crosssection images were shown with untreated tumor, tumor treated by irradiation, and by the combined Dbait32H (20µg/session) and irradiation (2Gy/session). The necrotic area was more important in the tumor treated by Dbait32H and irradiation than that treated by irradiation alone. This is consistent with the cytological analysis of tumors (cf. figure 4.5).<br><br>
Figure 4.7 shows Kaplan-Meier representation of survival of nude mice subcutaneously xenografted by Hep2, U87, LU 1205 and SK28 tumors, and their controls groups (untreated, treated by irradiation alone). The protocol of Hep2 was described in figure 4.3. Other tumors were treated by a modified protocol where a 5Gy fractionated irradiation was applied in three consecutive days, and followed four day rest, and the treatment was repeated once. The total irradiation dose (6x5Gy) is equal to that of the protocol used to treat Hep2 tumor (15x2Gy).<br><br>
The benefit outcome of the combined Dbaut32H and irradiation was observed in all four xenografted human tumors. As the underlying mechanism of action of Dbait molecules and the ubiquitous NHEJ pathway in all cells, it is anticipated that this holds true for other tumors with different histology.<br><br>
Descriptive analyses of the tumor response were performed for each treatment and each tumor type. Day 1 was the day of the first treatment session. All the animals were followed for at least 150 days or until their ethical sacrifice. Median lifetime was estimated according to the Kaplan-Meier method. TGD was calculated by subtracting the mean <br><br>
tumor volume quadrupling time of the control group from tumor volume quadrupling times of individual mice in each treated group. The mean TGD was calculated for each treated group using the individual measurements.<br><br>
Overall survival curves were assessed by Kaplan-Meier estimates and compared using the non-parametric Log Rank test since the data do not follow a normal distribution. The analysis used S-Plus 6.2 version software (MathSoft Inc., Seattle, WA) and statEL (ad Science, Paris, France). A global Log Rank was first performed for each group with a same tumor type. Then treatments with Dbait were compared to the mock-treated control. The number of animals (n), the relative risk (RR) and the P value are reported in Table 3.2. All tests were considered significant at the 0.05 significance level.<br><br>
Cell Irradiation Dbait Dbait Number Number Median "Relative **Mean **STD "Range **Mean line concentration of mice of cured survival risk TGD TGD TGD % TGD mice*** time (P value)<br><br>
(days)<br><br>
Hep2 - - - 21 49 C 0 3.8 -5 ; 12 100<br><br>
Hep2 - 32H 15 x 20 µg 10 55 0.62 7 6.7 -3 ; 17 168<br><br>
(1 nmole) (p
<br>
Hep2 - 32ss 15 x 120 µg 12 46 0.71 6 4.9 0 ; 19 156<br><br>
(12 nmole) (p
<br>
Hep2 15 x 2Gy Mock - 17 61 C* 12 14.2 -3 ; 45 217<br><br>
Hep2 15 x 2Gy 32H 15 x 20 µg 15 93 0.42 49 26.6 8 ; 134 561<br><br>
(1 nmole) (p?.55.10"3)<br><br>
Hep2 15 x 2Gy 32Hc 15 x 20 µg 11 94 0.55 &gt;56 41.1 6 ; 139 629<br><br>
(1 nmole) (p
<br>
Hep2 15 x 2Gy 32H 15 x 60 µg 20 3 129 0.28 &gt;83 50.7 -3 ; 139 880<br><br>
(3 nmole) (p
<br>
Hep2 15 x 2Gy 32Hc 15 x 60 µg 23 3 123 0.36 &gt;59 40.7 0 ; 139 652<br><br>
(3 nmole) (p
<br>
Hep2 15 x 2Gy 32H 15 x 120 µg 20 4 150 0.18 (p91 44.4 1 ; 139 952<br><br>
(6 nmole)<br><br>
Hep2 15 x 2Gy 32ss 15 x 120 µg 12 80 0.57 29 19.8 3 ; 59 374<br><br>
(12 nmole) (p
<br>
LU - - - 21 24 C 0 2.8 -4 ; 4 100<br><br>
LU 6 x 5Gy Mock - 30 61 C* &gt;22 29.2 -7 ; 140 348<br><br>
LU 6 x 5Gy 32Hc 6 x 60 µg 17 1 80 0.45 27 13.8 -7 ; 58 400<br><br>
(3 nmole) (p
<br>
SK 21 54 C 0 8.2 -19 ; 14 100 SK 6 x 5Gy Mock 17 86 C* 17 19.5 -3 ; 83 187 SK 6 x 5Gy 32H 6 x 60 µg 10 176 0.22 64 44.9 -3 ; 156 422 (3 nmole) (p
<br>
SK 6 x 5Gy 32Hc 6 x 60 µg 22 135 0.29 32 26.9 -8 ; 100 261 (3 nmole) (p
<br>
Table 3.2 summarizes part of the above described data and provides the comparison of survival of xenog rafted animals treated by different protocols (various Dbait32 molecules + irradiation versus irradiation + Mock injection) in three human tumor cell lines subcutaneously xenografted in nude mice. In addition, it shows that the sequence of Dbait molecules does not matter as evidenced by the similar outcome of Dbait32Hc as <br><br>
compared to Dbait32H. It shows also that the single strand Dbaut32ss was inactive even at high dose.<br><br>
It should be pointed out that the Dbait32Hc is devoid of CpG in its sequence in order to avoid the effect of immunological reaction due to well known Toll-like receptor-mediated immunostimulations.<br><br>
In conclusion, the significant reduction in tumor growth of human radio-resistant tumors (IHep2, U87 and SK28) and radio-sensitive tumor (LU1205) xenografted on nude mice provides evidence that the Dbait molecules can efficiently radiosensitize the effect of radiotherapy on these aggressive tumors. Thus, proof of concept of DNA bait approach has been achieved in vivo.<br><br>
Example 6: Chemosensitization of the treatment of digestive tumors induced in K- Rasv12G x Apc1638N transgenic mice<br><br>
An endogenous mouse tumor model was chosen to assess the ability of Dbait molecules to sensitize anticancer chemotherapy. To this end, transgenic mice carrying K-Rasv12G and Apc1638N mutations were used. They were obtained by breeding two transgenic mice: one carries K-Rasv12G mutant under the control of the mouse villin promoter(pVill/K- Rasv12G) (Janssen et al.,2002), the other contains Apc1638N mutation in one allele (Fodde et al., 1994). Transgenic mice with pVill/K-Rasv12G x Apc1638N mutations developed spontaneous tumors in the digestive tract at the age of about 5 months and died rapidly.<br><br>
They were treated at the mean age of 12 weeks by a combination of chemotherapy (5FU+CPT1 1 ) and of Dbait32H versus chemotherapy alone, according to the protocol shown in figure 5.1 panel A. The protocol includes three treatment cycles.<br><br>
Each cycle consists of intraperitoneal injection of 0.6mg 5FU and 0.6mg CPT1 1 , along with 0.1 mg Dbait32H by oral administration, three times a week, followed by one week rest.<br><br>
5FU (5 fluorouracile, Teva) was prepared in 0.9% NaCI solution at the concentration of 50mg/ml. CPT1 1/lrinotecan (Campto, Aventis) was prepared in 0.9% NaCI solution at the concentration of 20mg/ml. The health status and survival of the mice were monitored till the death. No clinical indication of additional toxic effect due to Dbait molecules was observed. <br><br>
The results are shown on figure 5.1. Panel A: Treatment protocol for three groups/arms of the K-Rasv12G x Apc1638N transgenic mice at the mean age of 12 weeks: the control group<br><br>
(untreated), the group treated by 5FU+CPT1 1 , the group treated by 5FU+CPT11 and Dbait32H. It was performed by three cycles of treatment. Each cycle consists of intraperitoneal injection of 0.6mg 5FU and 0.6mg CPT1 1 , along with 0.1 mg Dbait32H by oral administration, three times a week, followed by one week rest. The number of mice involved in each groups is indicated in parenthesis. The end point is the time of survival;<br><br>
Panel B: Kaplan-Meier plot of survival curves of the three groups; Panel C: The median survival time of three groups as shown in panel B.<br><br>
Despite reduced cohort, an improvement of survival time was observed in the arm which received the combination of chemotherapy (5FU+11 CPT) and Dbait32H (median survival=226 days, p-value=0.2), as compared to that of chemotherapy alone (173 days) and control arm (175 days) (panel B and C).<br><br>
Additional assays are currently underway to increase the cohort of 5FU+CPT11 +Dbait32H and 5FU+CPT1 1 arms, in order to enhance statistical significance.<br><br>
A series of mice was sacrificed two weeks after the end of treatment (at the mean age of 18 weeks) in order to evaluate the mean number of tumors per animal. The intestine was examined by macroscopy and histology examination (standard staining by Hematoxyline- Eosine-Safran).<br><br>
The results are given on figure 5.2. The number of animals in each group was indicated in parenthesis. All mice were sacrificed two weeks (week 18) after the protocol shown in figure 5.1 panel A. The mean number of the control arm (untreated group, n=101 ) is 30.8/animal.<br><br>
Both examinations consistently showed a significant reduction of tumor numbers( &gt; 30%) in the arm which received the combination of 5FU+CPT11 and Dbait32H (n=8) as compared to the arm which received chemotherapy alone (n=7) (figure 5.2). It is worthy to note that the mean number of the control arm (untreated group, n=101 ) is 30.8/animal.<br><br>
Tumor samples prepared from animals treated with Dbait molecules tagged by fluorescein (Dbait32H-FITC) and 5FU+CPT1 1 were analysed using immunofluorescence staining methods. ?-H2AX labeled foci were co-stained with fluorescent Dbait molecules, in <br><br>
reminiscence of the in vitro finding (cf. example 3.3 and 4). Figure 5.3 shows an additional assay where a 18 week-old K-Rasv12G x Apc1638N transgenic mouse was consecutively treated by chemotherapy(5FU+CPT1 1 ) and Dbait32H-FITC for three days and sacrificed two hours later after the last treatment as indicated in the panel A. The intestine was taken out and washed by PBS. Then the tumor tissues were sampled and frozen at -800C. For the analysis, 5µm histological samples were made from the frozen tumor tissues by cryostat. DNA repair foci were detected by immunofluorescence with polyclonal rabbit anti-?-H2AX antibody (Trevigen) diluted 1/500 in PBS, then with goat anti-rabbit antibody tagged by cyanine 3 (Jackson) diluted1/200 in PBS. The samples were also counterstained by DAPI Samples were visualized by epifluorescence microscopy. It was found that the fluorescence of Dbait32H-FITC was heterogeneously disseminated in tumor tissues (epithelium and stroma between, glandular structures) and had preferential nucleus localization (figure 5.3, panel B, left). Similar pattern was found for ?-H2AX sites (figure 5.3, panel B, right). The co-localized Dbait32H-FITC and ?-H2AX signals were almost observed.<br><br>
In conclusion, the improvement of survival and the reduction of tumor number per animal consistently show the evidence of chemosensitization of the treatment of digestive tumors in the transgenic mice carrying K-Rasv12G x Apc1638N mutations by Dbait molecules (Dbait32H). In-depth analysis of tumor tissues in treated animals provides evidence that Dbait molecules interfere with DNA repair process.<br><br>
It should be pointed out that the oral administration of Dbait32H molecules did not include any transfection agent in this study.<br><br>
To sum up, biochemical and in vitro data are clearly consistent with a mechanism of action of Dbait molecules through interference with DSB repair by NHEJ pathway, and the repair signal transduction pathway caused by direct or indirect DNA damage (ionizing radiation or chemotherapeutic agents). Due to the nature of the sequence-independent, NHEJ pathway (Jackson, 2002; Barnes, 2001 , Downs &amp; Jackon,2004), there is no limitation on the sequences and the length of Dbait molecules beyond a minimal length (about 32-bp). In vivo studies have confirmed efficient radio- and chemo-sensitization of tumors in mice by Dbait molecules. Taken together, all data have consistently provided with proofs of concept of the DNA Bait approach, characterized the molecular features of Dbait molecules. <br><br>
References<br><br>
Barnes, D. E. Non-homologous end joining as a mechanism of DNA repair. Curr. Biol. (2001 )1 1 , R455-7.<br><br>
Belenkov Al, Paiement JP, Panasci LC, Monia BP, Chow TY. An antisense oligonucleotide targeted to human Ku80 messenger RNA sensitizes M059K malignant glioma cells to ionizing radiation, bleomycin, and etoposide but not DNA cross-linking agents. Cancer Res. (2002), 62,5888-96.<br><br>
Bouton., S. : Kyle. S. : Durkacz. B. W. Mechanisms of enhancement of cytotoxicity in etoposide and ionising radiation-treated cells by the protein kinase inhibitor wortmannin. Eur. J Cancer (2000),36,535-41.<br><br>
Cantor, C. R.; Warshaw, M. M.;Shapiro, H. Oligonucleotide interactions. III. Conformational differences between deoxy-and ribodinucleoside phosphates Biopolymers (1970), 9,1059- 77.<br><br>
Cary, R. B.; Peterson, S. R.; Wang, J. T.; Bear, D. G.; Bradbury, E. M. &amp; Chen, D. J. DNA looping by Ku and the DNA-dependent protein kinase. Proc. Natl. Acad. Sci. USA (1997) 94, 4267-4272.<br><br>
Downs, J. A.; Jackson, S. P A means to a DNA end: The many roles of Ku. Nat. Rev. MoI. Cell. Biol. (2004), 5,367-78.<br><br>
Durant, S.; Karran, P. Vanillins-a novel family of DNA-PK inhibitors. Nucleic Acids Res. (2003), 31 ,5501-12.<br><br>
Fodde, R.; Edelmann, W.; Yang, K.; van Leeuwen, C; Carlson, C; Renault, B.; Breukel, C; Alt, E.; Lipkin, M.; Khan, P. M. A targeted chain-termination mutation in the mouse Ape gene results in multiple intestinal tumors. Proc Natl Acad Sci U S A. (1994), 91 ,8969-73.<br><br>
Jackson, S. P. Sensing and repairing DNA double-strand breaks. Carcinogenesis. (2002), 23, 687-96.<br><br>
Janssen, K. P.; el-Marjou, F.; Pinto, D.; Sastre, X.; Rouillard, D.; Fouquet, C; Soussi,T. ; Louvard, D.; Robine, S. Targeted expression of oncogenic K-ras in intestinal epithelium causes spontaneoustumorigenesis in mice. Gastroenterology (2002), 123,492-504. <br><br>
Kim, C. H.; Park, S. J.; Lee, S. H.; A targeted inhibition of DNA-dependent protein kinase sensitizes breast cancer cells following ionizing radiation. J ; Pharmacol. Exp. Ther. (2002), 303,753-9.<br><br>
Lee, H.; Sun, D.; Larner, J. M.; Wu, F. S. The tumor suppressor p53 can reduce stable transfection in the presence of irradiation. JBiomed. Sci. (1999), 6,285-92.<br><br>
Li, S.; Takeda, Y.; Wragg, S.; Barrett, J.; Phillips, A.; Dynan, W. S. Modification of the ionizing radiation response in living cells byan scFv against the DNA-dependent protein kinase. Nucleic Acid Res. (2003) 31 ,5848-57.<br><br>
Li, G. C; He, F.; Shao, X.; Urano, M.; Shen, L.; Kim, D.; Borrelli, M.; Leibel, S. A.; Gutin, P. H.; Ling, C. C. Adenovirus-mediated heat-activated antisense Ku70 expression radiosensitizes tumor cells in vitro and in vivo. Cancer Res. (2003b), 63,3268-74.<br><br>
Maacke, H.; Jost, K.; Opitz, S.; Miska, S.; Yuan, Y.; Hasselbach, L.; Luttges, J.;Kalthoff, H.; Sturzbecher, H. W. DNA repair and recombination factorRad51 is over-expressed in human pancreaticadenocarcinoma. Oncogene (2000). 19,2791-5.<br><br>
Mallya, S. M.; Sikpi, M. O. Evidence of the involvement of p53 in gamma-radiation- induced DNA repair in human lymphoblasts.vint.J Radiat. Biol. (1998), 74,231-8.<br><br>
Marangoni, E.; Bay, J. O.; Verrelle, P.; Bourhis, J. Tranfert de gene pour modifier Ia reponse a Ia radiotherapie Cancer Radiother. (2000), 4,175-80.<br><br>
Marangoni, E.; Foray, N.; O'Driscoll, M.; Douc-Rasy, S.; Bernier, J.; Bourhis, J.; Jeggo, P.<br><br>
A Ku80 fragment with dominant negative activity imparts a radiosensitive phenotype to<br><br>
CHO-K1 cells. Nucleic Acid Res. (2000a), 28,4778-82.<br><br>
Marangoni, E.; Le Romancer, M.; Foray, N.; Muller, C; Douc-Rasy, S.; Vaganay, S.;<br><br>
Abdulkarim, B.; Barrois, M.; Calsou, P.; Bernier, J.; Salles, B.; Bourhis J. Transfer of Ku80<br><br>
RNA antisense decreases the radioresistance of human fibroblasts. Cancer GeneTher.<br><br>
(2000b), 7,339-46.<br><br>
Martensson, S.;Hammarsten,O. DNA-dependent protein kinase catalytic subunit: structural requirements for kinase activity by DNA ends.J Biol. Che77i. (2002), 277,3020- <br><br>
29.<br><br>
Mimori, T.; Hardin, J. A. Mechanism of interaction between Ku protein and DNA. J Biol. Chem. (1986), 261 ,10375-10379.<br><br>
Nimura, Y.; Ismail, S. M.; Kurimas, A.; Chen, D. J.; Stevens, C. W. DNA-PK and ATM are required for radiation-enhanced integration. Radiat Res. (2002), 157,562-7.<br><br>
O'Driscoll, M.; Jeggo, P. Immunological disorders and DNA repair. Mutat Res. (2002), 509, 109-26.<br><br>
Ohnishi, T.; Taki, T. ; Hiraga, S.; Arita, N.; Morita, T. In vitro and in vivo potentiation of radiosensitivity of malignant gliomas by antisense inhibition of the RAD51 gene. Biochem Biophys Res Commun. (1998), 245,319-24.<br><br>
Pang, D. L.; Yoo, S. H. ; Dynan, W. S. ; Jung, M.; Dritschilo, A. Ku proteins join DNA fragments as shown by atomic force microscopy. Cancer Res. (1997), 57, 1412-5.<br><br>
Peng, Y.; Zhang, Q.; Nagasawa, H.; Okayasu, R.; Liber, H. L.; Bedford, J. S. Silencing expression of the catalytic subunit of DNA-dependent protein kinase by small interfering RNA sensitizes human cells for radiation-induced chromosome damage, cell killing, and mutation. Cancer Res. (2002), 62,6400-4.<br><br>
Rassool, F. V. DNA double strand breaks (DSB) and non-homologous end joining (NHEJ) pathways in human leukemia. Cancer Lett. (2003), 193,1-9.<br><br>
Roddam, P. L.; Rollinson, S.; O'Driscoll, M.; Jeggo, P. A.; Jack, A.; Morgan, G. J. Genetic variants of NHEJ DNA ligase IV can affect the risk of developing multiple myeloma, a tumour characterised by aberrant class switch recombination. J : Med. Genet. (2002) 39,900- 5.<br><br>
Sak, A.; Stuschke, M.; Wurm, R.; Schroeder, G.; Sinn, B.; Wolf, G.; Budach, V. Selective inactivation of DNA-dependent protein kinase with antisense oligodeoxynucleotides: consequences for the rejoining of radiation-induced DNA double-strand breaks and radiosensitivity of human cancer cell lines. Cancer Res. (2002), 62,6621-24.<br><br>
Stevens, C. W.; Zeng, M.; Cemiglia, G. J. Ionizing radiation greatly improves gene transfer <br><br>
efficiency in mammalian cells. Hure Gene Ther. (1996), 7,1727-34.<br><br>
Stevens, C. W.; Stamato, T. D.,Mauldin, S. K. ; Getts, R. C; Zeng, M.;Cerniglia, G. J. Radiation-induced recombination is dependent on KUdO.Radiat Res. (1999), 151 ,408-13.<br><br>
Stevens, C. W.;Cerniglia, G. J.; Giandomenico, A. R.; Koch, C. J. DNA damaging agents improve stable gene transfer efficiency in mammalian cells. Radiat Oncollnvestig. (1998), 6, 1-9.<br><br>
Verrelle, P.; Bourhis, J. Modulation de Ia reponse cellulaire aux radiations ionisantes: vers de nouvelles cibles moleculaires. Cancer Radiother. (1997), 1 , 484-93.<br><br>
Walker, J. R.; Corpina, R. A.; Goldberg, J. Structure of the Ku heterodimer bound to DNA and its implications for double-strand break repair. Nature (2001 ), 412,607-14.<br><br>
Yoo, S. H.; Dynan, W. S. Characterization of the RNA binding properties of Ku protein. Biochemsitry (1998), 37,1336-43.<br><br>
Yoo, S. H.; Kimzey, A.; Dynan, W. S. Photocross-linking of an oriented DNA repair complex. Ku bound at a single DNA end. J. Biol. Chem. (1999), 274,20034-9.<br><br>
Yoo, S. H.; Dynan, W. S. Geometry of a complex formed by double strand break repair proteins at a single DNA end: recruitment of DNA-PKcs induces inward translocation of Ku protein. Nucleic Acid Res. (1999), 27,4679-86.<br><br>
Zhao, J. K. ; Wang, J. T.;Chen, D. J.; Peterson, S. R.; Trewhella, J. The solution structure of the DNA double-stranded break repair protein Ku and its complex with DNA: a neutron contrast variation study. Biochemisty (1999), 3, 2152-9. <br><br>
CLAIMS<br><br>
1. A nucleic acid molecule, wherein said molecule comprises a double stranded portion of at least 24 bp, has at least one free end, is devoid of CpG, has less than 70 % sequence identity to any gene in a human genome, comprises one or several phosphorothioates or nucleotides with methylphosphonate backbone at the end of each strand or at least at the 3' end strand, and wherein said molecule is substrate for binding by at least a Ku protein involved in the NHEJ pathway of double strand breaks DSB repair.<br><br>
2. The molecule of claim 1 , wherein said molecule comprises between 24 and 100 bp.<br><br>
3. The molecule of claim 1 or 2, wherein said molecule is a linear or a hairpin nucleic acid molecule.<br><br>
4. The molecule of claim 3, wherein said molecule is a hairpin nucleic acid molecule and wherein the loop comprises nucleic acid or chemical groups.<br><br>
5. The molecule of any one of claims 1 to 4, wherein the free end is blunt or 5'- or 3'- protruding.<br><br>
6. The molecule of any one of claims 1 to 5, wherein said molecule inhibits in vitro radiation-enhanced illegitimate exogenous DNA integration.<br><br>
7. The molecule of any one of claims 1 to 6, wherein said molecule binds in vitro a Ku complex comprising at least Ku70 or Ku80.<br><br>
8. The molecule of any one of claims 1 to 7, wherein said molecule binds in vitro a Ku complex comprising at least a DNA-PKc protein.<br><br>
9. The molecule of any one of claims 1 to 8, wherein said molecule is capable of being up- taken by cell into the cell nucleus.<br><br>
10. The molecule of any one of claims 1 to 9, wherein said molecule comprises a 2'- deoxynucleotide backbone, and optionally comprises one or several modified nucleotides and/or nucleobases other than adenine, cytosine, guanine and thymine. <br><br><br>
1 1. The molecule of claim 10, further comprising sugar mimetics such as 2'-O-alkylribose, 2'-0-alkyl-C4' branched ribose, cyclobutyls or other carbocyclics or hexitol in place of the pentofuranosyl group.<br><br>
12. The molecule of any one of claims 1 to 11 , further comprising at least one embedded element, which hampers DNA replication, DNA repair, or damage signalling process, said at least one element being incorporated in the centre or at the end of the double-stranded molecule.<br><br>
13. The molecule of claim 12, comprising a) a polyethyleneglycol chain, preferably a hexaethyleneglycol chain, or any hydrocarbon chain, optionally interrupted and/or substituted by one or more heteroatoms e.g., oxygen, sulfur, nitrogen, or heteroatomic or heterocyclic groups, comprising one or several heteroatoms; b) a unit is a blocking element as it is not amenable by DNA polymerases or exonucleases, such as any 3'-modified nucleotides, c) a native oligonucleotide, such as Tn, when used in the loop of an hairpin fragment, preferably a tetradeoxythymidylate (T4).<br><br>
14. The molecule of any one of claims 1 to 13 selected from Dbait32Ha, Dbait32Hb, Dbait32Hc, Dbait32Hd, Dbait32Hc-3'mp, Dbait32Hc-5'3'mp, Dbait32Hc-Cy3, Dbait32Hc- Cy5, and Dbait32Hd-FITC.<br><br>
15. The molecule of claim 14, being Dbait32Hc.<br><br>
16. A pharmaceutical composition comprising a molecule of any one of claims 1-15 and a pharmaceutically acceptable carrier or excipient.<br><br>
17. Pharmaceutical composition of claim 16, suitable for oral route or for intravenous, intra-tumoral or sub-cutaneous injection, or for intracranial or intra artery injection or infusion or for topic administration.<br><br>
18. Pharmaceutical product comprising a molecule of any one of claims 1-15 and a chemical agent which can directly or indirectly cause double-strand breaks of DNA as a combined preparation for use for treating cancer. <br><br><br>
19. Pharmaceutical product of claim 18, wherein the molecule is to be administered prior to or along with the chemical agent.<br><br>
20. Use of a molecule of any one of claims 1-15 for the manufacture of a medicament for enhancing tumor sensitivity to DNA damaging anticancer therapy.<br><br>
21. Use of a molecule of any one of claims 1-15 for the manufacture of a medicament for treating cancer to be used in combination with a DNA damaging anticancer therapy.<br><br>
22. The use of claim 20 or 21 , wherein the DNA damaging anticancer therapy is selected from radiotherapy and chemotherapy.<br><br>
23. The use of any one of claim 20 to 22, wherein the molecule is to be administered prior to radiotherapy.<br><br>
24. The use of any one of claims 20 to 22, wherein the molecule is administered prior to or along with chemotherapy.<br><br>
25. The use of any one of claims 20 to 24, wherein the cancer is selected from CNS, head and neck, colorectal, liver, gastrointestinal tract, genitor-urinary tract, lung, skin, breast cancer and cervical cancer.<br><br>
26. The use of any one of claims 20 to 25, wherein the molecule is to be administered by oral route or by intravenous, intra-tumoral or sub-cutaneous injection, or by intracranial or intra artery injection or infusion or for topic administration.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQUJTVFJBQ1QoMjAtMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-ABSTRACT(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQU5ORVhVUkUgMSgxMC0xLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-ANNEXURE 1(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQVVTVFJBTElBTiBET0NVTUVOVCgxMC0xLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-AUSTRALIAN DOCUMENT(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ0xBSU1TKDIwLTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CLAIMS(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ0xBSU1TKEFNRU5ERUQpLSgxMC0xLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CLAIMS(AMENDED)-(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ0xBSU1TKEFNRU5ERUQpLSgyOC0yLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CLAIMS(AMENDED)-(28-2-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ0xBSU1TKE1BUktFRCBDT1BZKS0oMTAtMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CLAIMS(MARKED COPY)-(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ0xBSU1TKE1BUktFRCBDT1BZKS0oMjgtMi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CLAIMS(MARKED COPY)-(28-2-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ09SUkVTUE9OREVOQ0UoMTYtNy0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CORRESPONDENCE(16-7-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ09SUkVTUE9OREVOQ0UoMTctMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CORRESPONDENCE(17-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ09SUkVTUE9OREVOQ0UoMjktNC0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CORRESPONDENCE(29-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktQ09SUkVTUE9OREVOQ0UoOC05LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-CORRESPONDENCE(8-9-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktREVTQ1JJUFRJT04oQ09NUExFVEUpLSgyMC0yLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-DESCRIPTION(COMPLETE)-(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRFJBV0lORygyMC0yLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-DRAWING(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRFJBV0lORygyOC0yLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-DRAWING(28-2-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRVVSQVNJQU4gRE9DVU1FTlQoMTAtMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-EURASIAN DOCUMENT(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAxKDE5LTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 1(19-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAxKDI5LTQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 1(29-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAxOCgxNi03LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 18(16-7-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAyKENPTVBMRVRFKS0oMjAtMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 2(COMPLETE)-(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAyKFRJVExFIFBBR0UpLSgyMC0yLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 2(TITLE PAGE)-(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAyNigyOS00LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 26(29-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAzKDEwLTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 3(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAzKDI5LTQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 3(29-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSAzKDgtOS0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 3(8-9-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktRk9STSA1KDE5LTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-FORM 5(19-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMTAtMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-PETITION UNDER RULE 137(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDEwLTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-REPLY TO EXAMINATION REPORT(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktUkVQTFkgVE8gSEVBUklORygyOC0yLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-REPLY TO HEARING(28-2-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktU0VRVUVOQ0UgTElTVElORygxNy0zLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-SEQUENCE LISTING(17-3-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktVVMgRE9DVU1FTlQoMTAtMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-US DOCUMENT(10-1-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyLU1VTU5QLTIwMDktV08gSU5URVJOQVRJT05BTCBQVUJMSUNBVElPTiBSRVBPUlQoMjAtMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">392-MUMNP-2009-WO INTERNATIONAL PUBLICATION REPORT(20-2-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=RHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">Drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">Form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="261110-steam-turbine-with-a-plurality-of-stage-sections-and-a-device-for-output-control-of-the-steam-turbine.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="261112-alpha-beta-unsaturated-sulfoxides-for-treating-proliferative-disorders.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>261111</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>392/MUMNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>23/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Jun-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Jun-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Feb-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MUSEUM NATIONAL D&#x27;HISTOIRE NATURELLE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>43&amp;nbsp; rue Cuvier&amp;nbsp; F-75231 PARIS CEDEX 05&amp;nbsp; FRANCE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DUTREIX  Marie</td>
											<td>6 rue de Chalais  F-94240 L’HAY-LES ROSES  FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SUN  Jian-Sheng</td>
											<td>24 place du President J. F. Kennedy  F-94100 SAINT MAUR DES FOSSES  FRANCE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/11</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2007/059948</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-09-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>11/524,528</td>
									<td>2006-09-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/261111-a-nucleic-acid-molecule-dbait by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:08:05 GMT -->
</html>
