<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/261201-double-stranded-rna-molecule-and-method-of-preparing-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:28:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 261201:DOUBLE- STRANDED RNA MOLECULE AND METHOD OF PREPARING THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">DOUBLE- STRANDED RNA MOLECULE AND METHOD OF PREPARING THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Isolated double-stranded RNA molecule, wherein each RNA strand has a length from 19-23 nucleotides, wherein said RNA molecule is capable of target-specific nucleic acid modifications and wherein at least one strand has a 3&amp;quot;-overhang from 1-5 nucleotides.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Description<br>
The present invention relates to sequence and structural features of<br>
double-stranded (ds)RNA molecules required to mediate target-specific<br>
nucleic acid modifications such as RNA-interference and/or DNA methyla-<br>
tion.<br>
The term "RNA interference" (RNAi) was coined after the discovery that<br>
injection of dsRNA into the nematode C. elegans leads to specific silencing<br>
of genes highly homologous in sequence to the delivered dsRNA (Fire et<br>
al., 1998). RNAi was subsequently also observed in insects, frogs (Oelge-<br>
schlager et al., 2000), and other animals including mice (Svoboda et al.,<br>
2000; Wianny and Zernicka-Goetz, 2000) and is likely to also exist in<br>
human. RNAi is closely linked to the post-transcriptional gene-silencing<br>
(PTGS) mechanism of co-suppression in plants and quelling in fungi (Cata-<br>
lanotto et al., 2000; Cogoni and Macino, 1999; Dalmay et al., 2000;<br>
Ketting and Plasterk, 2000; Mourrain et al., 2000; Smardon et al., 2000)<br>
and some components of the RNAi machinery are also necessary for post-<br>
transcriptional silencing by co-suppression (Catalanotto et al., 2000; Dern-<br>
burg et al., 2000; Ketting and Plasterk, 2000). The topic has also been<br>
reviewed recently (Bass, 2000; Bosher and Labouesse, 2000; Fire, 1999;<br>
Plasterk and Ketting, 2000; Sharp, 1 999; Sijen and Kooter, 2000), see also<br>
the entire issue of Plant Molecular Biology, vol. 43, issue 2/3, (2000).<br>
In plants, in addition to PTGS, introduced transgenes can also lead to<br>
transcriptional gene silencing via RNA-directed DNA methylation of cytosi-<br>
nes (see references in Wassenegger, 2000). Genomic targets as short as<br>
30 bp are methylated in plants in an RNA-directed manner (Pelissier,<br>
2000). DNA methylation is also present in mammals.<br>
The natural function of RNAi and co-suppression appears to be protection<br>
of the genome against invasion by mobile genetic elements such as retro-<br>
transposons and viruses which produce aberrant RNA or dsRNA in the host<br>
cell when they become active (Jensen et al, 1999; Ketting et al., 1999;<br>
Ratcliff et al., 1999; Tabara et al., 1999). Specific mRNA degradation<br>
prevents transposon and virus replication although some viruses are able to<br>
overcome or prevent this process by expressing proteins that suppress<br>
PTGS (Lucy et al., 2000; Voinnet et al., 2000).<br>
DsRNA triggers the specific degradation of homologous RNAs only within<br>
the region of identity with the dsRNA (Zamore et al., 2000). The dsRNA is<br>
processed to 21-23 nt RNA fragments and the target RNA cleavage sites<br>
are regularly spaced 21-23 nt apart. It has therefore been suggested that<br>
the 21-23 nt fragments are the guide RNAs for target recognition (Zamore<br>
et al., 2000). These short RNAs were also detected in extracts prepared<br>
from D. melanogaster Schneider 2 cells which were transfected with<br>
dsRNA prior to cell lysis (Hammond et al., 2000), however, the fractions<br>
that displayed sequence-specific nuclease activity also contained a large<br>
fraction of residual dsRNA. The role of the 21-23 nt fragments in guiding<br>
mRNA cleavage is further supported by the observation that 21-23 nt<br>
fragments isolated from processed dsRNA are able, to some extent, to<br>
mediate specific mRNA degradation (Zamore et al., 2000). RNA molecules<br>
of similar size also accumulate in plant tissue that exhibits PTGS (Hamilton<br>
and Baulcombe, 1999).<br>
Here, we use the established Drosophila in vitro system (Tuschl et al.,<br>
1999; Zamore et al., 2000) to further explore the mechanism of RNAi. We<br>
demonstrate that short 21 and 22 nt RNAs, when base-paired with 3'<br>
overhanging ends, act as the guide RNAs for sequence-specific mRNA<br>
"degradation. Short 30 bp dsRNAs are unable to mediate RNAi in this sys:<br>
tern because they are no longer processed to 21 and 22 nt RNAs. Fur-<br>
thermore, we defined the target RNA cleavage sites relative to the 21 and<br>
22 nt short interfering RNAs (siRNAs) and provide evidence that the direc-<br>
tion of dsRNA processing determines whether a sense or an antisense<br>
target RNA can be cleaved by the produced siRNP endonuciease complex.<br>
Further, the siRNAs may also be important tools for transcriptional modula-<br>
ting, e.g. silencing of mammalian genes by guiding DNA methylation.<br>
Further experiments in human in vivo cell culture systems (HeLa cells)<br>
show that double-stranded RNA molecules having a length of preferably<br>
from 19-25 nucleotides have RNAi activity. Thus, in contrast to the results<br>
from Drosophila also 24 and 25 nt long double-stranded RNA molecules are<br>
efficient for RNAi.<br>
The object underlying the present invention is to provide novel agents<br>
capable of mediating target-specific RNA interference or other target-speci-<br>
fic nucleic acid modifications such as DNA methylation, said agents having<br>
an improved efficacy and safety compared to prior art agents.<br>
The solution of this problem is provided by an isolated double-stranded<br>
RNA molecule, wherein each RNA strand has a length from 19-25, particu-<br>
larly from 19-23 nucleotides, wherein said RNA molecule is capable of<br>
mediating target-specific nucleic acid modifications, particularly RNA inter-<br>
ference and/or DNA methylation, Preferably at least one strand has a 3'-<br>
overhang from 1-5 nucleotides, more preferably from 1-3 nucleotides and<br>
most preferably 2 nucleotides. The other strand may be blunt-ended or has<br>
up to 6 nucleotides 3' overhang. Also, if both strands of the dsRNA are<br>
exactly 21 or 22 nt, it is possible to observe some RNA interference when<br>
both ends are blunt (0 nt overhang). The RNA molecule is preferably a<br>
synthetic RNA molecule which is substantially free from contaminants<br>
occurring in cell extracts, e.g. from Drosophila embryos. Further, the RNA<br>
molecule is preferably substantially free from any non-target-specific conta-<br>
minants, particularly non-target-specific RNA molecules e.g. from contami-<br>
nants occuring in cell extracts.<br>
Further, the invention relates to the use of isolated double-stranded RNA<br>
molecules, wherein each RNA strand has a length from 19-25 nucleotides,<br>
for mediating, target-specific nucleic acid modifications, particularly RNAi,<br>
in mammalian cells, particularly in human cells.<br>
Surprisingly, it was found that synthetic short double-stranded RNA mole-<br>
cules particularly with overhanging 3'-ends are sequence-specific mediators<br>
of RNAi and mediate efficient target-RNA cleavage, wherein the cleavage<br>
site is located near the center of the region spanned by the guiding short<br>
RNA.<br>
Preferably, each strand of the RNA molecule has a length from 20-22<br>
nucleotides (or 20-25 nucleotides in mammalian cells), wherein the length<br>
of each strand may be the same or different. Preferably, the length of the<br>
3'-overhang reaches from 1-3 nucleotides, wherein the length of the over-<br>
hang may be the same or different for each strand. The RNA-strands<br>
preferably have 3'-hydroxyl groups. The 5'-terminus preferably comprises<br>
a phosphate, diphosphate, triphosphate or hydroxyl group. The most<br>
effective dsRNAs are composed of two 21 nt strands which are paired<br>
such that 1-3, particularly 2 nt 3' overhangs are present on both ends of<br>
the dsRNA.<br>
The target RNA cleavage reaction guided by siRNAs is highly sequence-<br>
specific. However, not all positions of a siRNA contribute equally to target<br>
recognition. Mismatches in the center of the siRNA duplex are most critical<br>
and essentially abolish target RNA cleavage. In contrast, the 3' nucleotide<br>
of the siRNA strand (e.g. position 21) that is complementary to the single-<br>
stranded target RNA, does not contribute to specificity of the target reco-<br>
gnition. Further, the sequence of the unpaired 2-nt 3' overhang of the<br>
siRNA strand with the same polarity as the target RNA is not critical for<br>
target RNA cleavage as only the antisense siRNA strand guides target reco-<br>
gnition. Thus, from the single-stranded overhanging nucleotides only the<br>
penultimate position of the antisense siRNA (e.g. position 20) needs to<br>
match the targeted sense mRNA.<br>
Surprisingly, the double-stranded RNA molecules of the present invention<br>
exhibit a high in vivo stability in serum or in growth medium for cell cul-<br>
tures. In order to further enhance the stability, the 3'-overhangs may be<br>
stablized against degradation, e.g. they may be selected such that they<br>
consist of purine nucleotides, particularly adenosine or guanosine nucleoti-<br>
des. Alternatively, substitution of pyrimidine nucleotides by modified ana-<br>
logues, e.g. substitution of uridine 2 nt 3' overhangs by 2'-deoxythymidine<br>
is tolerated and does not affect the efficiency of RNA interference. The<br>
absence of a 2' hydroxyl significantly enhances the nuclease resistance of<br>
the overhang in tissue culture medium.<br>
In an especially preferred embodiment of the present invention the RNA<br>
molecule may contain at least one modified nucleotide analogue. The<br>
nucleotide analogues may be located at positions where the target-specific<br>
activity, e.g. the RNAi mediating activity is not substantially effected, e.g.<br>
in a region at the 5'-end and/or the 3'-end of the double-stranded RNA<br>
molecule. Particularly, the overhangs may be stabilized by incorporating<br>
modified nucleotide analogues.<br>
Preferred nucleotide analogues are selected from sugar- or backbone-modi-<br>
fied ribonucleotides. It should be noted, however, that also nucleobase-<br>
modified ribonucleotides, i.e. ribonucleotides, containing a non-naturally<br>
occurring nucleobase instead of a naturally occurring nucleobase such as<br>
uridines or cytidines modified at the 5-position, e.g. 5-(2-amino)propyl<br>
uridine, 5-bromo uridine; adenosines and guanosines modified at the 8-<br>
position, e.g. 8-bromo guanosine; deaza nucleotides, e.g. 7-deaza-adeno-<br>
sine; O- and N-alkylated nucleotides, e.g. N6-methyl adenosine are suit-<br>
able. In preferred sugar-modified ribonucleotides the 2 ' OH-group is repla-<br>
ced by a group selected from H, OR, R, halo, SH, SR, NH2, NHR, NR2 or<br>
CN, wherein R is C1-C6 alkyl, alkenyl or alkynyl and halo is F, CI, Br or I.<br>
In preferred backbone-modified ribonucleotides the phosphoester group<br>
connecting to adjacent ribonucleotides is replaced by a modified group,<br>
e.g. of phosphothioate group. It should be noted that the above modifi-<br>
cations may be combined.<br>
The sequence of the double-stranded RNA molecule of the present inven-<br>
tion has to have a sufficient identity to a nucleic acid target molecule in<br>
order to mediate target-specific RNAi and/or DNA methylation. Preferably,<br>
the sequence has an identity of at least 50%, particularly of at least 70%<br>
to the desired target molecule in the double-stranded portion of the RNA<br>
molecule. More preferably, the identity is at least 85% and most preferably<br>
100% in the double-stranded portion of the RNA molecule. The identity of<br>
a double-stranded RNA molecule to a predetermined nucleic acid target<br>
molecule, e.g. an mRNA target molecule may be determined as follows:<br><br>
wherein I is the identity in percent, n is the number of identical nucleotides<br>
in the double-stranded portion of the ds RNA and the target and L is the<br>
length of the sequence overlap of the double-stranded portion of the<br>
dsRNA and the target.<br>
Alternatively, the identity of the double-stranded RNA molecule to the<br>
target sequence may also be defined including the 3' overhang, particularly<br>
an overhang having a length from 1-3 nucleotides. In this case the se-<br>
quence identity is preferably at least 50%, more preferably at least 70%<br>
and most preferably at least 85% to the target sequence. For example, the<br>
nucleotides from the 3' overhang and up to 2 nucleotides from the 5'<br>
and/or 3' terminus of the double strand may be modified without signifi-<br>
cant loss of activity.<br>
The double-stranded RNA molecule of the invention may be prepared by a<br>
method comprising the steps:<br>
(a) synthesizing two RNA strands each having a length from 19-25, e.g.<br>
from 19-23 nucleotides, wherein said RNA strands are capable of<br>
forming a double-stranded RNA molecule, wherein preferably at least<br>
one strand has a 3'-overhang from 1-5 nucleotides,<br>
(b) combining the synthesized RNA strands under conditions, wherein a<br>
double-stranded RNA molecule is formed, which is capable of media-<br>
ting target-specific nucleic acid modifications, particularly RNA<br>
interference and/or DNA methylation.<br>
Methods of synthesizing RNA molecules are known in the art. In this<br>
context, it is particularly referred to chemical synthesis methods as de-<br>
scribed in Verma and Eckstein (1998).<br>
The single-stranded RNAs can also be prepared by enzymatic transcription<br>
from synthetic DNA templates or from DNA plasmids isolated from recom-<br>
binant bacteria. Typically, phage RNA polymerases are used such as T7,<br>
T3 or SP6 RNA polymerase (Milligan and Uhlenbeck (1989)).<br>
A further aspect of the present invention relates to a method of mediating<br>
target-specific nucleic acid modifications, particularly RNA interference<br>
and/or DNA methylation in a cell or an organism comprising the steps:<br>
(a) contacting the cell or organism with the double-stranded RNA mole-<br>
cule of the invention under conditions wherein target-specific nucleic<br>
acid modifications may occur and<br>
(b) mediating a target-specific nucleic acid modificiation effected by the<br>
double-stranded RNA towards a target nucleic acid having a<br>
sequence portion substantially corresponding to the double-stranded<br>
RNA.<br>
Preferably the contacting step (a) comprises introducing the double-stran-<br>
ded RNA molecule into a target cell, e.g. an isolated target cell, e.g. in cell<br>
culture, a unicellular microorganism or a target cell or a plurality of target<br>
cells within a multicellular organism. More preferably, the introducing step<br>
comprises a carrier-mediated delivery, e.g. by liposomal carriers or by<br>
injection.<br>
The method of the invention may be used for determining the function of<br>
a gene in a cell or an organism or even for modulating the function of a<br>
gene in a cell or an organism, being capable of mediating RNA interference.<br>
The cell is preferably a eukaryotic cell or a cell line, e.g. a plant cell or an<br>
animal cell, such as a mammalian cell, e.g. an embryonic cell, a pluripotent<br>
stem cell, a tumor cell, e.g. a teratocarcinoma cell or a virus-infected cell.<br>
The organism is preferably a eukaryotic organism, e.g. a plant or an animal,<br>
such as a mammal, particularly a human.<br>
The target gene to which the RNA molecule of the invention is directed<br>
may be associated with a pathological condition. For example, the gene<br>
may be a pathogen-associated gene, e.g. a viral gene, a tumor-associated<br>
gene or an autoimmune disease-associated gene. The target gene may also<br>
be a heterologous gene expressed in a recombinant cell or a genetically<br>
altered organism. By determinating or modulating, particularly, inhibiting<br>
the function of such a gene valuable information and therapeutic benefits<br>
in the agricultural field or in the medicine or veterinary medicine field may<br>
be obtained.<br>
The dsRNA is usually administered as a pharmaceutical composition. The<br>
administration may be carried out oy known methods, wherein a nucleic<br>
acid is introduced into a desired target cell in vitro or in vivo. Commonly<br>
used gene transfer techniques include calcium phosphate, DEAE-dextran,<br>
electroporation and microinjection and viral methods (Graham, F.L. and van<br>
der Eb, A.J. (1973) Virol. 52, 456; McCutchan, J.H. and Pagano, J.S.<br>
(1968), J. Natl. Cancer Inst. 41, 351; Chu, G. et al (1987), Nucl. Acids<br>
Res. 15, 1311; Fraley, R. etal. (1980), J. Biol. Chem. 255, 10431; Capec-<br>
chi, M.R. (1980), Cell 22, 479). A recent addition to this arsenal of techni-<br>
ques for the introduction of DNA into cells is the use of cationic liposomes<br>
(Feigner, P.L. etal. (1987), Proc. Natl. Acad. Sci USA 84, 7413). Commer-<br>
cially available cationic lipid formulations are e.g. Tfx 50 (Promega) or<br>
Lipofectamin2000 (Life Technologies).<br>
Thus, the invention also relates to a pharmaceutical composition containing<br>
as an active agent at least one double-stranded RNA molecule as described<br>
above and a pharmaceutical carrier. The composition may be used for<br>
diagnostic and for therapeutic applications in human medicine or in veteri-<br>
nary medicine.<br>
For diagnostic or therapeutic applications, the composition may be in form<br>
of a solution, e.g. an injectable solution, a cream, ointment, tablet, suspen-<br>
sion or the like. The composition may be administered in any suitable way,<br>
e.g. by injection, by oral, topical, nasal, rectal application etc. The carrier<br>
may be any suitable pharmaceutical carrier. Preferably, a carrier is used,<br>
which is capable of increasing the efficacy of the RNA molecules to enter<br>
the target-cells. Suitable examples of such carriers are liposomes, particu-<br>
larly cationic liposomes. A further preferred administration method is injec-<br>
tion.<br>
A further preferred application of the RNAi method is a functional analysis<br>
of eukaryotic cells, or eukaryotic non-human organisms, preferably mam-<br>
malian cells or organisms and most preferably human cells, e.g. cell lines<br>
such as HeLa or 293 or rodents, e.g. rats and mice. By transfection with<br>
suitable double-stranded RNA molecules which are homologous to a prede-<br>
termined target gene or DNA molecules encoding a suitable double-stran-<br>
ded RNA molecule a specific knockout phenotype can be obtained in a<br>
target cell, e.g. in cell culture or in a target organism. Surprisingly it was<br>
found that the presence of short double-stranded RNA molecules does not<br>
result in an interferon response from the host cell or host organism.<br>
Thus, a further subject matter of the invention is a eukaryotic cell or a<br>
eukaryotic non-human organism exhibiting a target gene-specific knockout<br>
phenotype comprising an at least partially deficient expression of at least<br>
one endogeneous target gene wherein said cell or organism is transfected<br>
with at least one double-stranded RNA molecule capable of inhibiting the<br>
expression of at least one endogeneous target gene or with a DNA enco-<br>
ding at least one double stranded RNA molecule capable of inhibiting the<br>
expression of at least one endogeneous target gene. It should be noted<br>
that the present invention allows a target-specific knockout of several<br>
different endogeneous genes due to the specificity of RNAi.<br>
Gene-specific knockout phenotypes of cells or non-human organisms,<br>
particularly of human cells or non-human mammals may be used in analytic<br>
procedures, e.g. in the functional and/or phenotypical analysis of complex<br>
physiological processes such as analysis of gene expression profiles and/or<br>
proteomes. For example, one may prepare the knock-out phenotypes of<br>
human genes in cultured cells which are assumed to be regulators of<br>
alternative splicing processes. Among these genes are particularly the<br>
members of the SR splicing factor family, e.g. ASF/SF2, SC35, SRp20,<br>
SRp40 or SRp55. Further, the effect of SR proteins on the mRNA profiles<br>
of predetermined alternatively spliced genes such as CD44 may be analy-<br>
sed. Preferably the analysis is carried out by high-throughput methods<br>
using oligonucleotide based chips.<br>
Using RNAi based knockout technologies, the expression of an endoge-<br>
neous target gene may be inhibited in a target cell or a target organism.<br>
The endogeneous gene may be complemented by an exogeneous target<br>
nucleic acid coding for the target protein or a variant or mutated form of<br>
the target protein, e.g. a gene or a cDNA, which may optionally be fused<br>
to a further nucleic acid sequence encoding a detectable peptide or poly-<br>
peptide, e.g. an affinity tag, particularly a multiple affinity tag. Variants or<br>
mutated forms of the target gene differ from the endogeneous target gene<br>
in that they encode a gene product which differs from the endogeneous<br>
gene product on the amino acid level by substitutions, insertions and/or<br>
deletions of single or multiple amino acids. The variants or mutated forms<br>
may have the same biological activity as the endogeneous target gene. On<br>
the other hand, the variant or mutated target gene may also have a biologi-<br>
cal activity, which differs from the biological activity of the endogeneous<br>
target gene, e.g. a partially deleted activity, a completely deleted activity,<br>
an enhanced activity etc.<br>
The complementation may be accomplished by coexpressing the polypep-<br>
tide encoded by the exogeneous nucleic acid, e.g. a fusion protein com-<br>
prising the target protein and the affinity tag and the double stranded RNA<br>
molecule for knocking out the endogeneous gene in the target cell. This<br>
coexpression may be accomplished by using a suitable expression vector<br>
expressing both the polypeptide encoded by the exogeneous nucleic acid,<br>
e.g. the tag-modified target protein and the double stranded RNA molecule<br>
or alternatively by using a combination of expression vectors. Proteins and<br>
protein complexes which are synthesized de novo in the target cell will<br>
contain the exogeneous gene product, e.g. the modified fusion protein. In<br>
order to avoid suppression of the exogeneous gene product expression by<br>
the RNAi duplex molecule, the nucleotide sequence encoding the exoge-<br>
neous nucleic acid may be altered on the DNA level (with or without cau-<br>
sing mutations on the amino acid level) in the part of the sequence which<br>
is homologous to the double stranded RNA molecule. Alternatively, the<br>
endogeneous target gene may be complemented by corresponding nucleo-<br>
tide sequences from other species, e.g. from mouse.<br>
Preferred applications for the cell or organism of the invention is the analy-<br>
sis of gene expression profiles and/or proteomes. In an especially preferred<br>
embodiment an analysis of a variant or mutant form of one or several<br>
target proteins is carried out, wherein said variant or mutant forms are<br>
reintroduced into the cell or organism by an exogeneous target nucleic acid<br>
as described above. The combination of knockout of an endogeneous gene<br>
and rescue by using mutated, e.g. partially deleted exogeneous target has<br>
advantages compared to the use of a knockout cell. Further, this method<br>
is particularly suitable for identifying functional domains of the target<br>
protein. In a further preferred embodiment a comparison, e.g. of gene<br>
expression profiles and/or proteomes and/or phenotypic characteristics of<br>
at least two cells or organisms is carried out. These organisms are selected<br>
from:<br>
(i) a control cell or control organism without target gene inhibition,<br>
(ii) a cell or organism with target gene inhibition and<br>
(iii) a cell or organism with target gene inhibition plus target gene com-<br>
plementation by an exogeneous target nucleic acid.<br>
The method and cell of the invention are also suitable in a procedure for<br>
identifying and/or characterizing pharmacological agents, e.g. identifying<br>
new pharmacological agents from a collection of test substances and/or<br>
characterizing mechanisms of action and/or side effects of known pharma-<br>
cological agents.<br>
Thus, the present invention also relates to a system for identifying and/or<br>
characterizing pharmacological agents acting on at least one target protein<br>
comprising:<br>
(a) a eukaryotic cell or a eukaryotic non-human organism capable of<br>
expressing at least one endogeneous target gene coding for said<br>
target protein,<br>
(b) at least one double-stranded RNA molecule capable of inhibiting the<br>
expression of said at least one endogeneous target gene, and<br>
(c) a test substance or a collection of test substances wherein pharma-<br>
cological properties of said test substance or said collection are to<br>
be identified and/or characterized.<br>
Further, the system as described above preferably comprises:<br>
(d) at least one exogeneous target nucleic acid coding for the target<br>
protein or a variant or mutated form of the target protein wherein<br>
said exogeneous target nucleic acid differs from the endogeneous<br>
target gene on the nucleic acid level such that the expression of the<br>
exogeneous target nucleic acid is substantially less inhibited by the<br>
double stranded RNA molecule than the expression of the endoge-<br>
neous target gene.<br>
Furthermore, the RNA knockout complementation method may be used for<br>
preparative purposes, e.g. for the affinity purification of proteins or protein<br>
complexes from eukaryotic cells, particularly mammalian cells and more<br>
particularly human cells. In this embodiment of the invention, the exoge-<br>
neous target nucleic acid preferably codes for a target protein which is<br>
fused to an affinity tag.<br>
The preparative method may be employed for the purification of high<br>
molecular weight protein complexes which preferably have a mass of &gt;<br>
150 kD and more preferably of &gt; 500 kD and which optionally may con-<br>
tain nucleic acids such as RNA. Specific examples are the heterotrimeric<br>
protein complex consisting of the 20 kD, 60 kD and 90 kD proteins of the<br>
U4/U6 snRNP particle, the splicing factor SF3b from the 17S U2 snRNP<br>
consisting of 5 proteins having molecular weights of 14, 49, 120, 145 and<br>
155 kD and the 25S U4/U6/U5 tri-snRNP particle containing the U4, U5<br>
and U6 snRNA molecules and about 30 proteins, which has a molecular<br>
weight of about 1.7 MD.<br>
This method is suitable for functional proteome analysis in mammalian<br>
cells, particularly human cells.<br>
Further, the present invention is explained in more detail in the following<br>
figures and examples.<br>
Figure Legends<br>
Figure 1: Double-stranded RNA as short as 38 bp can mediate RNAi.<br>
(A) Graphic representation of dsRNAs used for targeting Pp-luc mRNA.<br>
Three series of blunt-ended dsRNAs covering a range of 29 to 504 bp were<br>
prepared. The position of the first nucleotide of the sense strand of the<br>
dsRNA is indicated relative to the start codon of Pp-luc mRNA (p1). (B)<br>
RNA interference assay (Tuschl et al., 1999). Ratios of target Pp-luc to<br>
control Rr-luc activity were normalized to a buffer control (black bar).<br>
DsRNAs (5 nM) were preincubated in Drosophila lysate for 15 min at 25°C<br>
prior to the addition of 7-methyl-guanosine-capped Pp-luc and Rr-luc<br>
mRNAs ( ~ 50 pM). The incubation was continued for another hour and<br>
then analyzed by the dual luciferase assay (Promega). The data are the<br>
average from at least four independent experiments ± standard deviation.<br>
Figure 2: A 29 bp dsRNA is no longer processed to 21-23 nt fragments.<br>
Time course of 21-23 mer formation from processing of internally 32P-<br>
labeled dsRNAs (5 nM) in the Drosophila lysate. The length and source of<br>
the dsRNA are indicated. An RNA size marker (M) has been loaded in the<br>
left lane and the fragment sizes are indicated. Double bands at time zero<br>
are due to incompletely denatured dsRNA.<br>
Figure 3: Short dsRNAs cleave the mRNA target only once.<br>
(A) Denaturing gel electrophoreses of the stable 5' cleavage products<br>
produced by 1 h incubation of 10 nM sense or antisense RNA 32P-labeled<br>
at the cap with 10 nM dsRNAs of the p133 series in Drosophila lysate.<br>
Length markers were generated by partial nuclease T1 digestion and partial<br>
alkaline hydrolysis (OH) of the cap-labeled target RNA. The regions<br>
targeted by the dsRNAs are indicated as black bars on both sides. The 20-<br>
23 nt spacing between the predominant cleavage sites for the 111 bp long<br>
dsRNA is shown. The horizontal arrow indicates unspecific cleavage not<br>
due to RNAi. (B) Position of the cleavage sites on sense and antisense<br>
target RNAs. The sequences of the capped 177 nt sense and 180 nt<br>
antisense target RNAs are represented in antiparallel orientation such that<br>
complementary sequence are opposing each other. The region targeted by<br>
the different dsRNAs are indicated by differently colored bars positioned<br>
between sense and antisense target sequences. Cleavage sites are<br>
indicated by circles: large circle for strong cleavage, small circle for weak<br>
cleavage. The 32P-radiolabeled phosphate group is marked by an asterisk.<br>
Figure 4: 21 and 22 nt RNA fragments are generated by an RNase Ill-like<br>
mechanism.<br>
(A) Sequences of ~ 21 nt RNAs after dsRNA processing. The ~21 nt RNA<br>
fragments generated by dsRNA processing were directionally cloned and<br>
sequenced. Oligoribonucleotides originating from the sense strand of the<br>
dsRNA are indicated as blue lines, those originating from the antisense<br>
strand as red lines. Thick bars are used if the same sequence was present<br>
in multiple clones, the number at the right indicating the frequency. The<br>
target RNA cleavage sites mediated by the dsRNA are indicated as orange<br>
circles, large circle for strong cleavage, small circle for weak cleavage (see<br>
Figure 3B). Circles on top of the sense strand indicated cleavage sites<br>
within the sense target and circles at the bottom of the dsRNA indicate<br>
cleavage site in the antisense target. Up to five additional nucleotides were<br>
identified in ~21 nt fragments derived from the 3' ends of the dsRNA.<br>
These nucleotides are random combinations of predominantly C, G, or A<br>
residues and were most likely added in an untemplated fashion during T7<br>
transcription of the dsRNA-constituting strands. (B) Two-dimensional TLC<br>
analysis of the nucleotide composition of ~21 nt RNAs. The -21 nt RNAs<br>
were generated by incubation of nternally radiolabeled 504 bp Pp-luc<br>
dsRNA in Drosophila lysate, gel-purified, and then digested to mononucleo-<br>
tides with nuclease P1 (top row) or ribonuclease T2 (bottom row). The<br>
dsRNA was internally radiolabeled by transcription in the presence of one<br>
of the indicated a-32P nucleoside triphosphates. Radioactivity was detected<br>
by phosphorimaging. Nucleoside 5'-monophosphates, nucleoside 3'-mono-<br>
phosphates, nucleoside 5',3'-diphosphates, and inorganic phosphate are<br>
indicated as pN, Np, pNp, and Pi, respectively. Black circles indicate UV-<br>
absorbing spots from non-radioactive carrier nucleotides. The 3',5'-bis-<br>
phosphates (red circles) were identified by co-migration with radiolabeled<br>
standards prepared by 5'-phosphorylation of nucleoside 3'-mono-<br>
phosphates with T4 polynucleotide; kinase and ?-32P-ATP.<br>
Figure 5: Synthetic 21 and 22 nt RNAs Mediate Target RNA Cleavage.<br>
(A) Graphic representation of control 52 bp dsRNA and synthetic 21 and<br>
22 nt dsRNAs. The sense strand of 21 and 22 nt short interfering RNAs<br>
(siRNAs) is shown blue, the antisense strand in red. The sequences of the<br>
siRNAs were derived from the cloned fragments of 52 and 111 bp dsRNAs<br>
(Figure 4A), except for the 22 nt antisense strand of duplex 5. The siRNAs<br>
in duplex 6 and 7 were unique to the 111 bp dsRNA processing reaction.<br>
The two 3' overhanging nucleotides indicated in green are present in the<br>
sequence of the synthetic antisense strand of duplexes 1 and 3. Both<br>
strands of the control 52 bp dsRNA were prepared by in vitro transcription<br>
and a fraction of transcripts may contain untemplated 3' nucleotide<br>
addition. The target RNA cleavage sites directed by the siRNA duplexes are<br>
indicated as orange circles (see legend to Figure 4A) and were determined<br>
as shown in Figure 5B. (B) Position of the cleavage sites on sense and<br>
antisense target RNAs. The target RNA sequences are as described in<br>
Figure 3B. Control 52 bp dsRNA (10 nM) or 21 and 22 nt RNA duplexes 1-<br>
7 (100 nM) were incubated with target RNA for 2.5 h at 25°C in Droso-<br>
phila lysate. The stable 5' cleavage products were resolved on the gel. The<br>
cleavage sites are indicated in Figure 5A. The region targeted by the 52 bp<br>
dsRNA or the sense (s) or antisense (as) strands are indicated by the black<br>
bars to the side of the gel. The cleavage sites are all located within the<br>
region of identity of the dsRNAs. For precise determination of the cleavage<br>
sites of the antisense strand, a lower percentage gel was used.<br>
Figure 6: Long 3' overhangs on short dsRNAs inhibit RNAi.<br>
(A) Graphic representation of 52 bp dsRNA constructs. The 3' extensions<br>
of sense and antisense strand are indicated in blue and red, respectively.<br>
The observed cleavage sites on the target RNAs are represented as orange<br>
circles analogous to Figure 4A and were determined as shown in Figure<br>
6B. (B) Position of the cleavage sites on sense and antisense target RNAs.<br>
The target RNA sequences are as described in Figure 3B. DsRNA (10 nM)<br>
was incubated with target RNA for 2.5 h at 25°C in Drosophila lysate. The<br>
stable 5' cleavage products were resolved on the gel. The major cleavage<br>
sites are indicated with a horizontal arrow and also represented in Figure<br>
6A. The region targeted by the 52 bp dsRNA is represented as black bar at<br>
both sides of the gel.<br>
Figure 7: Proposed Model for RNAi.<br>
RNAi is predicted to begin with processing of dsRNA (sense strand in<br>
black, antisense strand in red) to predominantly 21 and 22 nt short inter-<br>
fering RNAs (siRNAs). Short overhanging 3' nucleotides, if present on the<br>
dsRNA, may be beneficial for processing of short dsRNAs. The dsRNA-<br>
processing proteins, which remain to be characterized, are represented as<br>
green and blue ovals, and assembled on the dsRNA in asymmetric fashion.<br>
In our model, this is illustrated by binding of a hypothetical blue protein or<br>
protein domain with the siRNA strand in 3' to 5' direction while the hypo-<br>
thetical green protein or protein domain is always bound to the opposing<br>
siRNA strand. These proteins or a subset remain associated with the siRNA<br>
duplex and preserve its orientation as determined by the direction of the<br>
dsRNA processing reaction. Only the siRNA sequence associated with the<br>
blue protein is able to guide target RNA cleavage. The endonuclease com-<br>
plex is referred to as small interfering ribonucleoprotein complex or siRNP.<br>
It is presumed here, that the endonuclease that cleaves the dsRNA may<br>
also cleave the target RNA, probably by temporarily displacing the passive<br>
siRNA strand not used for target recognition. The target RNA is then<br>
cleaved in the center of the region recognized by the sequence-complemen-<br>
tary guide siRNA.<br>
Figure 8: Reporter constructs and siRNA duplexes.<br>
(a) The firefly (Pp-luc) and sea pansy (Rr-luc) luciferase reporter gene re-<br>
gions from plasmids pGL2-Control, pGL-3-Control and pRL-TK (Promega)<br>
are illustrated. SV40 regulatory elements, the HSV thymidine kinase pro-<br>
moter and two introns (lines) are indicated. The sequence of GL3 luciferase<br>
is 95% identical to GL2, but RL is completely unrelated to both. Luciferase<br>
expression from pGL2 is approx. 10-fold lower than from pGL3 in trans-<br>
fected mammalian cells. The region targeted by the siRNA duplexes is<br>
indicated as black bar below the coding region of the luciferase genes, (b)<br>
The sense (top) and antisense (bottom) sequences of the siRNA duplexes<br>
targeting GL2, GL3 and RL luciferase are shown. The GL2 and GL3 siRNA<br>
duplexes differ by only 3 single nucleotide substitutions (boxed in gray). As<br>
unspecific control, a duplex with the inverted GL2 sequence, invGL2, was<br>
synthesized. The 2 nt 3' overhang of 2'-deoxythymidine is indicated as TT;<br>
uGL2 is similar to GL2 siRNA but contains ribo-uridine 3' overhangs.<br>
Figure 9: RNA interference by siRNA duplexes.<br>
Ratios of target control luciferase were normalized to a buffer control (bu,<br>
black bars); gray bars indicate ratios of Photinus pyralis (Pp-luc) GL2 or<br>
GL3 luciferase to Renilla reniformis (Rr-luc) RL luciferase (left axis), white<br>
bars indicate RL to GL2 or GL3 ratios (right axis). Panels a, c, e, g and i<br>
describe experiments performed with the combination of pGL2-Control and<br>
pRL-TK reporter plasmids, panels b, d, f, h and j with pGL3-Control and<br>
pRL-TK reporter plasmids. The cell line used for the interference experiment<br>
is indicated at the top of each plot. The ratios of Pp-luc/Rr-luc for the<br>
buffer control (bu) varied between 0.5 and 10 for pGL2/pRL and between<br>
0.03 and 1 for pGL3/pRL, respectively, before normalization and between<br>
the various cell lines tested. The plotted data were averaged from three<br>
independent experiments ± S.D.<br>
Figure 10: Effects of 21 nt siRNA, 50 bp and 500 bp dsRNAs on luciferase<br>
expression in HeLa cells.<br>
The exact length of the long dsRNAs is indicated below the bars. Panels a,<br>
c and e describe experiments performed with pGL2-Control and pRL-TK<br>
reporter plasmids, panels b, d and f with pGL3-Control and pRL-TK reporter<br>
plasmids. The data were averaged from two independent experiments ±<br>
S.D. (a), (b) Absolute Pp-luc expression, plotted in arbitrary luminescence<br>
units, (c), (d) Rr-luc expression, plotted in arbitrary luminescence units, (e),<br>
(f) Ratios of normalized target to control luciferase. The ratios of luciferase<br>
activity for siRNA duplexes were normalized to a buffer control (bu, black<br>
bars); the luminescence ratios for 50 or 500 bp dsRNAs were normalized<br>
to the respective ratios observed for 50 and 500 bp dsRNA from humani-<br>
zed GFP (hG, black bars). It should be noted that the overall differences in<br>
sequences between the 49 and 484 bp dsRNAs targeting GL2 and GL3 are<br>
not sufficient to confer specificity between GL2 and GL3 targets (43 nt<br>
uninterrupted identity in 49 bp segment, 239 nt longest uninterrupted<br>
identity in 484 bp segment).<br>
Figure 11: Variation of the 3' overhang of duplexes of 21-nt siRNAs.<br>
(A) Outline of the experimental strategy. The capped and polyadenylated<br>
sense target mRNA is depicted and the relative positions of sense and<br>
antisense siRNAs are shown. Eight series of duplexes, according to the<br>
eight different antisense strands were prepared. The siRNA sequences and<br>
the number of overhanging nucleotides were changed in 1-nt steps. (B)<br>
Normalized relative luminescence of target luciferase (Photinus pyralis, Pp-<br>
luc) to control luciferase (Renilla reniformis, Rr-luc) in D. melanogaster<br>
embryo lysate in the presence of 5 nM blunt-ended dsRNAs. The lumi-<br>
nescence ratios determined in the presence of dsRNA were normalized to<br>
the ratio obtained for a buffer control (bu, black bar). Normalized ratios less<br>
than 1 indicate specific interference. (C-J) Normalized interference ratios<br>
for eight series of 21-nt siRNA duplexes. The sequences of siRNA duplexes<br>
are depicted above the bar graphs. Each panel shows the interference ratio<br>
for a set of duplexes formed with a given antisense guide siRNA and 5<br>
different sense siRNAs. The number of overhanging nucleotides (3' over-<br>
hang, positive numbers; 5' overhangs, negative numbers) is indicated on<br>
the x-axis. Data points were averaged from at least 3 independent experi-<br>
ments, error bars represent standard deviations.<br>
Figure 12: Variation of the length of the sense strand of siRNA duplexes.<br>
(A) Graphic representation of the experiment. Three 21-nt antisense<br>
strands were paired with eight sense siRNAs. The siRNAs were changed in<br>
length at their 3' end. The 3' overhang of the antisense siRNA was 1-nt<br>
(B), 2-nt (C), or 3-nt (D) while the sense siRNA overhang was varied for<br>
each series. The sequences of the siRNA duplexes and the corresponding<br>
interference ratios are indicated.<br>
Figure 13: Variation of the length of siRNA duplexes with preserved 2-nt 3'<br>
overhangs.<br>
(A) Graphic representation of the experiment. The 21-nt siRNA duplex is<br>
identical in sequence to the one shown in Figure 11H or 12C. The siRNA<br>
duplexes were extended to the 3' side of the sense siRNA (B) or the 5'<br>
side of the sense siRNA (C). The siRNA duplex sequences and the respec-<br>
tive interference ratios are indicated.<br>
Figure 14: Substitution of the 2'-hydroxyl groups of the siRNA ribose<br>
residues.<br>
The 2'-hydroxyl groups (OH) in the strands of siRNA duplexes were repla-<br>
ced by 2'-deoxy (d) or 2'-0-methyl (Me). 2-nt and 4-nt 2'-deoxy substitu-<br>
tions at the 3'-ends are indicated as 2-nt d and 4-nt d, respectively. Uridine<br>
residues were replaced by 2'-deoxy thymidine.<br>
Figure 15: Mapping of sense and antisense target RNA cleavage by 21-nt<br>
siRNA duplexes with 2-nt 3' overhangs.<br>
(A) Graphic representation of 32P (asterisk) cap-labelled sense and anti-<br>
sense target RNAs and siRNA duplexes. The position of sense and anti-<br>
sense target RNA cleavage is indicated by triangles on top and below the<br>
siRNA duplexes, respectively. (B) Mapping of target RNA cleavage sites.<br>
After 2 h incubation of 10 nM target with 100 nM siRNA duplex in D.<br>
melanogaster embryo lysate, the 5' cap-labelled substrate and the 5'<br>
cleavage products were resolved on sequencing gels. Length markers were<br>
generated by partial RNase T1 digestion (T1) and partial alkaline hydrolysis<br>
(OH-) of the target RNAs. The bold lines to the left of the images indicate<br>
the region covered by the siRNA strands 1 and 5 of the same orientation<br>
as the target.<br>
Figure 16: The 5' end of a guide siRNA defines the position of target RNA<br>
cleavage.<br>
(A, B) Graphic representation of the experimental strategy. The antisense<br>
siRNA was the same in all siRNA duplexes, but the sense strand was<br>
varied between 18 to 25 nt by changing the 3' end (A) or 18 to 23 nt by<br>
changing the 5' end (B). The position of sense and antisense target RNA<br>
cleavage is indicated by triangles on top and below the siRNA duplexes,<br>
respectively. (C, D) Analysis of target RNA cleavage using cap-labelled<br>
sense (top panel) or antisense (bottom panel) target RNAs. Only the cap-<br>
labelled 5' cleavage products are shown. The sequences of the siRNA<br>
duplexes are indicated, and the length of the sense siRNA strands is mar-<br>
ked on top of the panel. The control lane marked with a dash in panel (C)<br>
shows target RNA incubated in absence of siRNAs. Markers were as<br>
described in Figure 15. The arrows in (D), bottom panel, indicate the target<br>
RNA cleavage sites that differ by 1 nt.<br>
Figure 17: Sequence variation of the 3' overhang of siRNA duplexes.<br>
The 2-nt 3' overhang (NN, in gray) was changed in sequence and composi-<br>
tion as indicated (T, 2'-deoxythymidine, dG, 2'-deoxyguanosine; asterisk,<br>
wild-type siRNA duplex). Normalized interference ratios were determined as<br>
described in Figure 11. The wild-type sequence is the same as depicted in<br>
Figure 14.<br>
Figure 18: Sequence specificity of target recognition.<br>
The sequences of the mismatched siRNA duplexes are shown, modified<br>
sequence segments or single nucleotides are underlayed in gray. The<br>
reference duplex (ref) and the siRNA duplexes 1 to 7 contain 2'-deoxythy-<br>
midine 2-nt overhangs. The silencing efficiency of the thymidine-modified<br>
reference duplex was comparable to the wild-type sequence (Figure 17).<br>
Normalized interference ratios were determined as described in Figure 11.<br>
Figure 19: Variation of the length of siRNA duplexes with preserved 2-nt 3'<br>
overhangs.<br>
The siRNA duplexes were extended to the 3' side of the sense siRNA (A)<br>
or the 5' side of the sense siRNA (B). The siRNA duplex sequences and the<br>
respective interference ratios are indicated. For HeLa SS6 cells, siRNA<br>
duplexes (0.84µg) targeting GL2 luciferase were transfected together with<br>
pGL2-Control and pRL-TK plasmids. For comparison, the in vitro RNAi<br>
activities of siRNA duplexes tested in D. melanogaster lysate are indicated.<br>
Example 1<br>
RNA Interference Mediated by Small Synthetic RNAs<br>
1.1. Experimental Procedures<br>
1.1.1 In Vitro RNAi<br>
In vitro RNAi and lysate preparations were performed as described<br>
previously (Tuschl et al., 1999; Zamore et al., 2000). It is critical to use<br>
freshly dissolved creatine kinase (Roche) for optimal ATP regeneration. The<br>
RNAi translation assays (Fig. 1) were performed with dsRNA concentra-<br>
tions of 5 nM and an extended preincubation period of 15 min at 25°C<br>
prior to the addition of in vitro transcribed, capped and polyadenylated Pp-<br>
luc and Rr-luc reporter mRNAs. The incubation was continued for 1 h and<br>
the relative amount of Pp-luc and Rr-luc protein was analyzed using the<br>
dual luciferase assay (Promega) and a Monolight 3010C luminometer<br>
(PharMingen).<br>
1.1.2 RNA Synthesis<br>
Standard procedures were used for in vitro transcription of RNA from PCR<br>
templates carrying T7 or SP6 promoter sequences, see for example (Tuschl<br>
et al., 1998). Synthetic RNA was prepared using Expedite RNA phosphor-<br>
amidites (Proligo). The 3' adapter oligonucleotide was synthesized using<br>
dimethoxytrityl-1,4-benzenedimethanol-succinyl-aminopropyl-CPG. The<br>
oligoribonucleotides were deprotected in 3 ml of 32% ammonia/ethanol<br>
(3/1) for 4 h at 55°C (Expedite RNA) or 16 h at 55°C (3' and 5' adapter<br>
DNA/RNA chimeric oligonucleotides) and then desilylated and gel-purified<br>
as described previously (Tuschl et al., 1993). RNA transcripts for dsRNA<br>
preparation including long 3' overhangs were generated from PCR tem-<br>
plates that contained a T7 promoter in sense and an SP6 promoter in<br>
antisense direction. The transcription template for sense and antisense<br>
target RNA was PCR-amplified with<br>
GCGTAATACGACTCACTATAGAACAATTGCTTTTACAG (underlined, T7<br>
promoter) as 5' primer and<br>
ATTTAGGTGACACTATAGGCATAAAGAATTGAAGA (underlined, SP6<br>
promoter) as 3' primer and the linearized Pp-luc plasmid (pGEM-luc<br>
sequence) (Tuschl et al., 1999) as template; the T7-transcribed sense RNA<br>
was 177 nt long with the Pp-luc sequence between pos. 113-273 relative<br>
to the start codon and followed by 17 nt of the complement of the SP6<br>
promoter sequence at the 3' end. Transcripts for blunt-ended dsRNA<br>
formation were prepared by transcription from two different PCR products<br>
which only contained a single promoter sequence.<br>
DsRNA annealing was carried out using a phenol/chloroform extraction.<br>
Equimolar concentration of sense and antisense RNA (50 nM to 10 µM,<br>
depending on the length and amount available) in 0.3 M NaOAc (pH 6)<br>
were incubated for 30 s at 90°C and then extracted at room temperature<br>
with an equal volume of phenol/chloroform, and followed by a chloroform<br>
extraction to remove residual phenol. The resulting dsRNA was precipitated<br>
by addition of 2.5-3 volumes of ethanol. The pellet was dissolved in lysis<br>
buffer (100 mM KCI, 30 mM HEPES-KOH, pH 7.4, 2 mM Mg(OAc)2) and<br>
the quality of the dsRNA was verified by standard agarose gel electro-<br>
phoreses in 1 x TAE-buffer. The 52 bp dsRNAs with the 17 nt and 20 nt 3'<br>
overhangs (Figure 6) were annealed by incubating for 1 min at 95 °C, then<br>
rapidly cooled to 70°C and followed by slow cooling to room temperature<br>
over a 3 h period (50 µl annealing reaction, 1 µM strand concentration,<br>
300 mM NaCI, 10 mM Tris-HCI, pH 7.5). The dsRNAs were then phenol/<br>
chloroform extracted, ethanol-precipitated and dissolved in lysis buffer.<br>
Transcription of internally 32P-radiolabeled RNA used for dsRNA preparation<br>
(Figures 2 and 4) was performed using 1 mM ATP, CTP, GTP, 0.1 or 0.2<br>
mM UTP, and 0.2-0.3 µM -32P-UTP (3000 Ci/mmol), or the respective ratio<br>
for radiolabeled nucleoside triphosphates other than UTP. Labeling of the<br>
cap of the target RNAs was performed as described previously. The target<br>
RNAs were gel-purified after cap-labeling.<br>
1.1.3 Cleavage Site Mapping<br>
Standard RNAi reactions were performed by pre-incubating 10 nM dsRNA<br>
for 15 min followed by addition of 10 nM cap-labeled target RNA. The<br>
reaction was stopped after a further 2 h (Figure 2A) or 2.5 h incubation<br>
(Figure 5B and 6B) by proteinase K treatment (Tuschl et al., 1999). The<br>
samples were then analyzed on 8 or 10% sequencing gels. The 21 and 22<br>
nt synthetic RNA duplexes were used at 100 nM final concentration (Fig<br>
5B).<br>
1.1.4 Cloning of -21 nt RNAs<br>
The 21 nt RNAs were produced by incubation of radiolabeled dsRNA in<br>
Drosophila lysate in absence of target RNA (200 µl reaction, 1 h incuba-<br>
tion, 50 nM dsP111, or 100 nM dsP52 or dsP39). The reaction mixture<br>
was subsequently treated with proteinase K (Tuschl et al., 1999) and the<br>
dsRNA-processing products were separated on a denaturing 15% poly-<br>
acrylamide gel. A band, including a size range of at least 18 to 24 nt, was<br>
excised, eluted into 0.3 M NaCI overnight at 4°C and in siliconized tubes.<br>
The RNA was recovered by ethanol-precipitation and dephosphorylated (30<br>
µl reaction, 30 min, 50°C, 10 U alkaline phosphatase, Roche). The reaction<br>
was stopped by phenol/chloroform extraction and the RNA was ethanol-<br>
precipitated. The 3' adapter oligonucleotide (pUUUaaccgcatccttctcx: upper-<br>
case, RNA; lowercase, DNA; p, phosphate; x, 4-hydroxymethylbenzyl) was<br>
then ligated to the dephosphorylated -21 nt RNA (20µl reaction, 30 min,<br>
37°C, 5 µM 3' adapter, 50 mM Tris-HCI, pH 7.6, 10 mM MgCI2, 0.2 mM<br>
ATP, 0.1 mg/ml acetylated BSA, 1 5% DMSO, 25 U T4 RNA ligase, Amers-<br>
ham-Pharmacia) (Pan and Uhlenbeck, 1992). The ligation reaction was<br>
stopped by the addition of an equal volume of 8 M urea/50 mM EDTA<br>
stopmix and directly loaded on a 15% gel. Ligation yields were greater<br>
50%. The ligation product was recovered from the gel and 5'-phosphory-<br>
lated (20 µl reaction, 30 min, 37°C, 2 mM ATP, 5 U T4 polynucleotide<br>
kinase, NEB). The phosphorylation reaction was stopped by phenol/chloro-<br>
form extraction and RNA was recovered by ethanol-precipitation. Next, the<br>
5' adapter (tactaatacgactcactAAA: uppercase, RNA; lowercase, DNA) was<br>
ligated to the phosphorylated ligation product as described above. The new<br>
ligation product was gel-purified and eluted from the gel slice in the<br>
presence of reverse transcription primer<br>
(GACTAGCTGGAATTCAAGGATGCGGTTAAA: bold, Eco Rl site) used as<br>
carrier. Reverse transcription (15 µl reaction, 30 min, 42°C, 150 U Super-<br>
script II reverse transcriptase, Life Technologies) was followed by PCR<br>
using as 5' primer CAGCCAACGGAATTCATACGACTCACTAAA (bold, Eco<br>
R1 site) and the 3' RT primer. The PCR product was purified by phenol/<br>
chloroform extraction and ethanol-precipitated. The PCR product was then<br>
digested with Eco Rl (NEB) and concatamerized using T4 DNA ligase (high<br>
cone, NEB). Concatamers of a size range of 200 to 800 bp were<br>
separated on a low-melt agarose gel, recovered from the gel by a standard<br>
melting and phenol extraction procedure, and ethanol-precipitated. The<br>
unpaired ends were filled in by incubation with Taq polymerase under<br>
standard conditions for 15 min at 72°C and the DNA product was directly<br>
ligated into the pCR2.1-T0P0 vector using the TOPO TA cloning kit (Invi-<br>
trogen). Colonies were screened using PCR and M13-20 and M13 Reverse<br>
sequencing primers. PCR products were directly submitted for custom<br>
sequencing (Sequence Laboratories Gottingen GmbH, Germany). On aver-<br>
age, four to five 21mer sequences were obtained per clone.<br>
1.1.5 2D-TLC Analysis<br>
Nuclease P1 digestion of radiolabeled, gel-purified siRNAs and 2D-TLC was<br>
carried out as described (Zamore et al., 2000). Nuclease T2 digestion was<br>
performed in 10 µl reactions for 3 h at 50°C in 10 mM ammonium acetate<br>
(pH 4.5) using 2µg/µl carrier tRNA and 30 U ribonuclease T2 (Life Techno-<br>
logies). The migration of non-radioactive standards was determined by UV<br>
shadowing. The identity of nucleoside-3',5'-disphosphates was confirmed<br>
by co-migration of the T2 digestion products with standards prepared by<br>
5'-32P-phosphorylation of commercial nucleoside 3'-monophosphates using<br>
?-32P-ATP and T4 polynucleotide kinase (data not shown).<br>
1.2 Results and Discussion<br>
1.2.1 Length Requirements for Processing of dsRNA to 21 and 22 nt RNA<br>
Fragments<br>
Lysate prepared from D. melanogaster syncytial embryos recapitulates<br>
RNAi in vitro providing a novel tool for biochemical analysis of the<br>
mechanism of RNAi (Tuschl et al., 1 999; Zamore et al., 2000). In vitro and<br>
in vivo analysis of the length requirements of dsRNA for RNAi has revealed<br>
that short dsRNA (
degrading target mRNA (Caplen et al., 2000; Hammond et al., 2000; Ngo<br>
et al., 1998); Tuschl et al., 1999). The reasons for reduction in mRNA<br>
degrading efficiency are not understood. We therefore examined the pre-<br>
cise length requirement of dsRNA for target RNA degradation under opti-<br>
mized conditions in the Drosophila lysate (Zamore et al., 2000). Several<br>
series of dsRNAs were synthesized and directed against firefly luciferase<br>
(Pp-luc) reporter RNA. The specific suppression of target RNA expression<br>
was monitored by the dual luciferase assay (Tuschl et al., 1999) (Figures<br>
1A and 1B). We detected specific inhibition of target RNA expression for<br>
dsRNAs as short as 38 bp, but dsRNAs of 29 to 36 bp were not effective<br>
in this process. The effect was independent of the target position and the<br>
degree of inhibition of Pp-luc mRNA expression correlated with the length<br>
of the dsRNA, i.e. long dsRNAs were more effective than short dsRNAs.<br>
It has been suggested that the 21-23 nt RNA fragments generated by<br>
processing of dsRNAs are the mediators of RNA interference and co-<br>
suppression (Hamilton and Baulcombe, 1999; Hammond et al., 2000;<br>
Zamore et al., 2000). We therefore analyzed the rate of 21-23 nt fragment<br>
formation for a subset of dsRNAs ranging in size between 501 to 29 bp.<br>
Formation of 21-23 nt fragments in Drosophila lysate (Figure 2) was readily<br>
detectable for 39 to 501 bp long dsRNAs but was significantly delayed for<br>
the 29 bp dsRNA. This observation is consistent with a role of 21-23 nt<br>
fragments in guiding mRNA cleavage and provides an explanation for the<br>
lack of RNAi by 30 bp dsRNAs. The length dependence of 21-23 mer<br>
formation is likely to reflect a biologically relevant control mechanism to<br>
prevent the undesired activation of RNAi by short intramolecular base-<br>
paired structures of regular cellular RNAs.<br>
1.2.2 39 bp dsRNA Mediates Target RIMA Cleavage at a Single Site<br>
Addition of dsRNA and 5'-capped target RNA to the Drosophila lysate<br>
results in sequence-specific degradation of the target RNA (Tuschl et al.,<br>
1999). The target mRNA is only cleaved within the region of identity with<br>
the dsRNA and many of the target cleavage sites were separated by 21-23<br>
nt (Zamore et al., 2000). Thus, the number of cleavage sites for a given<br>
dsRNA was expected to roughly correspond to the length of the dsRNA<br>
divided by 21. We mapped the target cleavage sites on a sense and an<br>
antisense target RNA which was 5' radiolabeled at the cap (Zamore et al.,<br>
2000) (Figures 3A and 3B). Stable 5' cleavage products were separated on<br>
a sequencing gel and the position of cleavage was determined by<br>
comparison with a partial RNase T1 and an alkaline hydrolysis ladder from<br>
the target RNA.<br>
Consistent with the previous observation (Zamore et al., 2000), all target<br>
RNA cleavage sites were located within the region of identity to the<br>
dsRNA. The sense or the antisense traget was only cleaved once by 39 bp<br>
dsRNA. Each cleavage site was located 10 nt from the 5' end of the region<br>
covered by the dsRNA (Figure 3B). The 52 bp dsRNA, which shares the<br>
same 5' end with the 39 bp dsRNA, produces the same cleavage site on<br>
the sense target, located 10 nt from the 5' end of the region of identity<br>
with the dsRNA, in addition to two weaker cleavage sites 23 and 24 nt<br>
downstream of the first site. The antisense target was only cleaved once,<br>
again 10 nt from the 5' end of the region covered by its respective dsRNA.<br>
Mapping of the cleavage sites for the 38 to 49 bp dsRNAs shown in Figure<br>
1 showed that the first and predominant cleavage site was always located<br>
7 to 10 nt downstream of the region covered by the dsRNA (data not<br>
shown). This suggests that the point of target RNA cleavage is determined<br>
by the end of the dsRNA and could imply that processing to 21-23 mers<br>
starts from the ends of the duplex.<br>
Cleavage sites on sense and antisense target for the longer 111 bp dsRNA<br>
were much more frequent than anticipated and most of them appear in<br>
clusters separated by 20 to 23 nt (Figures 3A and 3B). As for the shorter<br>
dsRNAs, the first cleavage site on the sense target is 10 nt from the 5' end<br>
of the region spanned by the dsRNA, and the first cleavage site on the<br>
antisense target is located 9 nt from the 5' end of region covered by the<br>
dsRNA. It is unclear what causes this disordered cleavage, but one possi-<br>
bility could be that longer dsRNAs may not only get processed from the<br>
ends but also internally, or there are some specificity determinants for<br>
dsRNA processing which we do not yet understand. Some irregularities to<br>
the 21-23 nt spacing were also previously noted (Zamore et al., 2000). To<br>
better understand the molecular basis of dsRNA processing and target RNA<br>
recognition, we decided to analyze the sequences of the 21-23 nt frag-<br>
ments generated by processing of 39, 52, and 111 bp dsRNAs in the<br>
Drosophila lysate.<br>
1.2.3 dsRNA is Processed to 21 and 22 nt RNAs by an RNase Ill-Like<br>
Mechanism<br>
In order to characterize the 21-23 nt RNA fragments we examined the 5'<br>
and 3' termini of the RNA fragments. Periodate oxidation of gel-purified 21 -<br>
23 nt RNAs followed by ß-elimination indicated the presence of a terminal<br>
2' and 3' hydroxyl groups. The 21-23 mers were also responsive to alka-<br>
line phosphatase treatment indicating the presence of a 5' terminal phos-<br>
phate group. The presence of 5' phosphate and 3' hydroxyl termini<br>
suggests that the dsRNA could be processed by an enzymatic activity<br>
similar to E. coli RNase III (for reviews, see (Dunn, 1982; Nicholson, 1999;<br>
Robertson, 1990; Robertson, 1982)).<br>
Directional cloning of 21-23 nt RNA fragments was performed by ligation<br>
of a 3' and 5' adapter oligonucleotide to the purified 21-23 mers using T4<br>
RNA ligase. The ligation products were reverse transcribed, PCR-amplified,<br>
concatamerized, cloned, and sequenced. Over 220 short RNAs were<br>
sequenced from dsRNA processing reactions of the 39, 52 and 111 bp<br>
dsRNAs (Figure 4A). We found the following length distribution: 1 % 18 nt,<br>
5% 19 nt, 12% 20 nt, 45% 21 nt, 28% 22 nt, 6% 23 nt, and 2% 24 nt.<br>
Sequence analysis of the 5' terminal nucleotide of the processed fragments<br>
indicated that oligonucleotides with a 5' guanosine were underrepresented.<br>
This bias was most likely introduced by T4 RNA ligase which discriminates<br>
against 5' phosphorylated guanosine as donor oligonucleotide; no signifi-<br>
cant sequence bias was seen at the 3' end. Many of the -21 nt fragments<br>
derived from the 3' ends of the sense or antisense strand of the duplexes<br>
include 3' nucleotides that are derived from untemplated addition of nu-<br>
cleotides during RNA synthesis using T7 RNA polymerase. Interestingly, a<br>
significant number of endogenous Drosophila ~21 nt RNAs were also<br>
cloned, some of them from LTR and non-LTR retrotransposons (data not<br>
shown). This is consistent with a possible role for RNAi in transposon<br>
silencing.<br>
The ~21 nt RNAs appear in clustered groups (Figure 4A) which cover the<br>
entire dsRNA sequences. Apparently, the processing reaction cuts the<br>
dsRNA by leaving staggered 3' ends, another characteristic of RNase III<br>
cleavage. For the 39 bp dsRNA, two clusters of ~21 nt RNAs were found<br>
from each dsRNA-constituting strand including overhanging 3' ends, yet<br>
only one cleavage site was detected on the sense and antisense target<br>
(Figures 3A and 3B). If the ~21 nt fragments were present as single-<br>
stranded guide RNAs in a complex that mediates mRNA degradation, it<br>
could be assumed that at least two target cleavage sites exist, but this<br>
was not the case. This suggests that the ~21 nt RNAs may be present in<br>
double-stranded form in the endonuclease complex but that only one of the<br>
strands can be used for target RNA recognition and cleavage. The use of<br>
only one of the ~ 21 nt strands for target cleavage may simply be deter-<br>
mined by the orientation in which the ~ 21 nt duplex is bound to the nucle-<br>
ase complex. This orientation is defined by the direction in which the<br>
original dsRNA was processed.<br>
The ~ 21mer clusters for the 52 bp and 111 bp dsRNA are less well de-<br>
fined when compared to the 39 bp dsRNA. The clusters are spread over<br>
regions of 25 to 30 nt most likely representing several distinct subpopula-<br>
tions of ~21 nt duplexes and therefore guiding target cleavage at several<br>
nearby sites. These cleavage regions are still predominantly separated by<br>
20 to 23 nt intervals. The rules determining how regular dsRNA can be<br>
processed to ~21 nt fragments are not yet understood, but it was previ-<br>
ously observed that the approx. 21-23 nt spacing of cleavage sites could<br>
be altered by a run of uridines (Zamore et al., 2000). The specificity of<br>
dsRNA cleavage by E. coli RNase III appears to be mainly controlled by<br>
antideterminants, i.e. excluding some specific base-pairs at given positions<br>
relative to the cleavage site (Zhang and Nicholson, 1997).<br>
To test whether sugar-, base- or cap-modification were present in<br>
processed ~21 nt RNA fragments, we incubated radiolabeled 505 bp Pp-<br>
luc dsRNA in lysate for 1 h, isolated the -21 nt products, and digested it<br>
with P1 or T2 nuclease to mononucleotides. The nucleotide mixture was<br>
then analyzed by 2D thin-layer chromatography (Figure 4B). None of the<br>
four natural ribonucleotides were modified as indicated by P1 or T2 di-<br>
gestion. We have previously analyzed adenosine to inosine conversion in<br>
the ~21 nt fragments (after a 2 h incubation) and detected a small extent<br>
(
(1 h) reduced this inosine fraction to barely detectable levels. RNase T2,<br>
which cleaves 3' of the phosphodiester linkage, produced nucleoside 3'-<br>
phosphate and nucleoside 3',5'-diphosphate, thereby indicating the pre-<br>
sence of a 5'-terminal monophosphate. All four nucleoside 3',5'-diphos-<br>
phates were detected and suggest that the internucleotidic linkage was<br>
cleaved with little or no sequence-specificity. In summary, the ~ 21 nt<br>
fragments are unmodified and were generated from dsRNA such that 5'-<br>
monophosphates and 3'-hydroxyls were present at the 5'-end.<br>
1.2.4 Synthetic 21 and 22 nt RNAs Mediate Target RNA Cleavage<br>
Analysis of the products of dsRNA processing indicated that the ~21 nt<br>
fragments are generated by a reaction with all the characteristics of an<br>
RNase III cleavage reaction (Dunn, 1982; Nicholson, 1999; Robertson,<br>
1990; Robertson, 1982). RNase III makes two staggered cuts in both<br>
strands of the dsRNA, leaving a 3' overhang of about 2 nt. We chemically<br>
synthesized 21 and 22 nt RNAs, identical in sequence to some of the<br>
cloned ~21 nt fragments, and tested them for their ability to mediate<br>
target RNA degradation (Figures 5A and 5B). The 21 and 22 nt RNA du-<br>
plexes were incubated at 100 nM concentrations in the lysate, a 10-fold<br>
higher concentrations than the 52 bp control dsRNA. Under these condi-<br>
tions, target RNA cleavage is readily detectable. Reducing the concen-<br>
tration of 21 and 22 nt duplexes from 100 to 10 nM does still cause target<br>
RNA cleavage. Increasing the duplex concentration from 100 nM to 1000<br>
nM however does not further increase target cleavage, probably due to a<br>
limiting protein factor within the lysate.<br>
In contrast to 29 or 30 bp dsRNAs that did not mediate RNAi, the 21 and<br>
22 nt dsRNAs with overhanging 3' ends of 2 to 4 nt mediated efficient<br>
degradation of target RNA (duplexes 1, 3, 4, 6, Figures 5A and 5B). Blunt-<br>
ended 21 or 22 nt dsRNAs (duplexes 2, 5, and 7, Figures 5A and 5B) were<br>
reduced in their ability to degrade the target and indicate that overhanging<br>
3' ends are critical for reconstitution of the RNA-protein nuclease complex.<br>
The single-stranded overhangs may be required for high affinity binding of<br>
the ~ 21 nt duplex to the protein components. A 5' terminal phosphate,<br>
although present after dsRNA processing, was not required to mediate<br>
target RNA cleavage and was absent from the short synthetic RNAs.<br>
The synthetic 21 and 22 nt duplexes guided cleavage of sense as well as<br>
antisense targets within the region covered by the short duplex. This is an<br>
important result considering that a 39 bp dsRNA, which forms two pairs of<br>
clusters of ~21 nt fragments (Fig. 2), cleaved sense or antisense target<br>
only once and not twice. We interpret this result by suggesting that only<br>
one of two strands present in the ~21 nt duplex is able to guide target<br>
RNA cleavage and that the orientation of the ~21 nt duplex in the nu-<br>
clease complex is determined by the initial direction of dsRNA processing.<br>
The presentation of an already perfectly processed ~21 nt duplex to the in<br>
vitro system however does allow formation of the active sequence-specific<br>
nuclease complex with two possible orientations of the symmetric RNA<br>
duplex. This results in cleavage of sense as well as antisense target within<br>
the region of identity with the 21 nt RNA duplex.<br>
The target cleavage site is located 11 or 12 nt downstream of the first<br>
nucleotide that is complementary to the 21 or 22 nt guide sequence, i.e.<br>
the cleavage site is near center of the region covered by the 21 or 22 nt<br>
RNAs (Figures 4A and 4B). Displacing the sense strand of a 22 nt duplex<br>
by two nucleotides (compare duplexes 1 and 3 in Figure 5A) displaced the<br>
cleavage site of only the antisense target by two nucleotides. Displacing<br>
both sense and antisense strand by two nucleotides shifted both cleavage<br>
sites by two nucleotides (compare duplexes 1 and 4). We predict that it<br>
will be possible to design a pair oi 21 or 22 nt RNAs to cleave a target<br>
RNA at almost any given position.<br>
The specificity of target RNA cleavage guided by 21 and 22 nt RNAs<br>
appears exquisite as no aberrant cleavage sites are detected (Figure 5B). It<br>
should however be noted, that the nucleotides present in the 3' overhang<br>
of the 21 and 22 nt RNA duplex may contribute less to substrate recog-<br>
nition than the nucleotides near the cleavage site. This is based on the<br>
observation that the 3' most nucleotide in the 3' overhang of the active<br>
duplexes 1 or 3 (Figure 5A) is not complementary to the target. A detailed<br>
analysis of the specificity of RNAi can now be readily undertaken using<br>
synthetic 21 and 22 nt RNAs.<br>
Based on the evidence that synthetic: 21 and 22 nt RNAs with overhanging<br>
3' ends mediate RNA interference, we propose to name the ~21 nt RNAs<br>
"short interfering RNAs" or siRNAs and the respective RNA-protein com-<br>
plex a "small interfering ribonucleoprotein particle" or siRNP.<br>
1.2.5 3' Overhangs of 20 nt on short dsRNAs inhibit RNAi<br>
We have shown that short blunt-ended dsRNAs appear to be processed<br>
from the ends of the dsRNA. During our study of the length dependence of<br>
dsRNA in RNAi, we have also analyzed dsRNAs with 17 to 20 nt overhang-<br>
ing 3' ends and found to our surprise that they were less potent than<br>
blunt-ended dsRNAs. The inhibitory effect of long 3' ends was particularly<br>
pronounced for dsRNAs up to 100 bp but was less dramatic for longer<br>
dsRNAs. The effect was not due to imperfect dsRNA formation based on<br>
native gel analysis (data not shown). We tested if the inhibitory effect of<br>
long overhanging 3' ends could be used as a tool to direct dsRNA process-<br>
ing to only one of the two ends of a short RNA duplex.<br>
We synthesized four combinations of the 52 bp model dsRNA, blunt-<br>
ended, 3' extension on only the sense strand, 3'-extension on only the<br>
antisense strand, and double 3' extension on both strands, and mapped<br>
the target RNA cleavage sites after incubation in lysate (Figures 6A and<br>
6B). The first and predominant cleavage site of the sense target was lost<br>
when the 3' end of the antisense strand of the duplex was extended, and<br>
vice versa, the strong cleavage site of the antisense target was lost when<br>
the 3' end of sense strand of the duplex was extended. 3' Extensions on<br>
both strands rendered the 52 bp dsRNA virtually inactive. One explanation<br>
for the dsRNA inactivation by ~20 nt 3' extensions could be the associa-<br>
tion of single-stranded RNA-binding proteins which could interfere with the<br>
association of one of the dsRNA-processing factors at this end. This result<br>
is also consistent with our model where only one of the strands of the<br>
siRNA duplex in the assembled siRNP is able to guide target RNA cleavage.<br>
The orientation of the strand that guides RNA cleavage is defined by the<br>
direction of the dsRNA processing reaction. It is likely that the presence of<br>
3' staggered ends may facilitate the assembly of the processing complex.<br>
A block at the 3' end of the sense strand will only permit dsRNA process-<br>
ing from the opposing 3' end of the antisense strand. This in turn<br>
generates siRNP complexes in which only the antisense strand of the<br>
siRNA duplex is able to guide sense target RNA cleavage. The same is true<br>
for the reciprocal situation.<br>
The less pronounced inhibitory effect of long 3' extensions in the case of<br>
longer dsRNAs (&gt;500 bp, data not shown) suggests to us that long<br>
dsRNAs may also contain internal dsRNA-processing signals or may get<br>
processed cooperatively due to the association of multiple cleavage fac-<br>
tors.<br>
1.2.6 A Model for dsRNA-Directed mRNA Cleavage<br>
The new biochemical data update the model for how dsRNA targets mRNA<br>
for destruction (Figure 7). Double-stranded RNA is first processed to short<br>
RNA duplexes of predominantly 21 and 22 nt in length and with staggered<br>
3' ends similar to an RNase Ill-like reaction (Dunn, 1982; Nicholson, 1999;<br>
Robertson, 1982). Based on the 21-23 nt length of the processed RNA<br>
fragments it has already been speculated that an RNase Ill-like activity may<br>
be involved in RNAi (Bass, 2000). This hypothesis is further supported by<br>
the presence of 5' phosphates and 3' hydroxyls at the termini of the<br>
siRNAs as observed in RNase III reaction products (Dunn, 1 982; Nicholson,<br>
1999). Bacterial RNase III and the eukaryotic homologs Rntlp in S. cerevi-<br>
siae and Padp in S. pombe have been shown to function in processing of<br>
ribosomal RNA as well as snRNA and snoRNAs (see for example Chanfreau<br>
et al., 2000).<br>
Little is known about the biochemistry of RNase III homologs from plants,<br>
animals or human. Two families of RNase III enzymes have been identified<br>
predominantly by database-guided sequence analysis or cloning of cDNAs.<br>
The first RNase III family is represented by the 1327 amino acid long D.<br>
melanogaster protein drosha (Ace. AF116572). The C-terminus is com-<br>
posed of two RNase III and one dsRNA-binding domain and the N-terminus<br>
is of unknown function. Close homologs are also found in C. elegans (Ace.<br>
AF160248) and human (Ace. AF189011) (Filippov et al., 2000; Wu et al.,<br>
2000). The drosha-like human RNase III was recently cloned and charac-<br>
terized (Wu et al., 2000). The gene is ubiquitously expressed in human<br>
tissues and cell lines, and the protein is localized in the nucleus and the<br>
nucleolus of the cell. Based on results inferred from antisense inhibition<br>
studies, a role of this protein for rRNA processing was suggested. The<br>
second class is represented by the C. elegans gene K12H4.8 (Ace.<br>
S44849) coding for a 1822 amino acid long protein. This protein has an N-<br>
terminal RNA helicase motif which is followed by 2 RNase III catalytic<br>
domains and a dsRNA-binding motif, similar to the drosha RNase III family.<br>
There are close homologs in S. pombe (Ace. Q09884), A. thaliana (Ace.<br>
AF187317), D. melanogaster (Ace. AE003740), and human (Ace.<br>
AB028449) (Filippov et al., 2000; Jacobsen et al., 1999; Matsuda et al.,<br>
2000). Possibly the K12H4.8 RNase lll/helicase is the likely candidate to be<br>
involved in RNAi.<br>
Genetic screens in C. elegans identified rde-1 and rde-4 as essential for<br>
activation of RNAi without an effect on transposon mobilization or co-<br>
suppression (Dernburg et al., 2000; Grishok et al., 2000; Ketting and<br>
Plasterk, 2000; Tabara et al., 1999). This led to the hypothesis that these<br>
genes are important for dsRNA processing but are not involved in mRNA<br>
target degradation. The function of both genes is as yet unknown, the rde-<br>
1 gene product is a member of a family of proteins similar to the rabbit<br>
protein elF2C (Tabara et al., 1999), and the sequence of rde-4 has not yet<br>
been described. Future biochemical characterization of these proteins<br>
should reveal their molecular function.<br>
Processing to the siRNA duplexes appears to start from the ends of both<br>
blunt-ended dsRNAs or dsRNAs with short (1-5 nt) 3' overhangs, and<br>
proceeds in approximately 21-23 nt steps. Long ( ~ 20 nt) 3' staggered<br>
ends on short dsRNAs suppress RNAi, possibly through interaction with<br>
single-stranded RNA-binding proteins. The suppression of RNAi by single-<br>
stranded regions flanking short dsRNA and the lack of siRNA formation<br>
from short 30 bp dsRNAs may explain why structured regions frequently<br>
encountered in mRNAs do not lead to activation of RNAi.<br>
Without wishing to be bound by theory, we presume that the dsRNA-<br>
processing proteins or a subset of these remain associated with the siRNA<br>
duplex after the processing reaction. The orientation of the siRNA duplex<br>
relative to these proteins determines which of the two complementary<br>
strands functions in guiding target RNA degradation. Chemically syn-<br>
thesized siRNA duplexes guide cleavage of sense as well as antisense<br>
target RNA as they are able to associate with the protein components in<br>
either of the two possible orientation.<br>
The remarkable finding that synthetic 21 and 22 nt siRNA duplexes can be<br>
used for efficient mRNA degradation provides new tools for sequence-<br>
specific regulation of gene expression in functional genomics as well as<br>
biomedical studies. The siRNAs may be effective in mammalian systems<br>
where long dsRNAs cannot be used due to the activation of the PKR<br>
response (Clemens, 1997). As such, the siRNA duplexes represent a new<br>
alternative to antisense or ribozyme therapeutics.<br>
Example 2<br>
RNA Interference in Human Tissue Cultures<br>
2.1 Methods<br>
2.1.1 RNA preparation<br>
21 nt RNAs were chemically synthesized using Expedite RNA phosphorami-<br>
dites and thymidine phosphoramidite (Proligo, Germany). Synthetic oligonu-<br>
cleotides were deprotected and gel-purified (Example 1), followed by Sep-<br>
Pak C18 cartridge (Waters, Milford, MA, USA) purification (Tuschl, 1993).<br>
The siRNA sequences targeting GL2 (Ace. X65324) and GL3 luciferase<br>
(Ace. U47296) corresponded to the coding regions 153-173 relative to the<br>
first nucleotide of the start codon, siRNAs targeting RL (Ace. AF025846)<br>
corresponded to region 119-129 after the start codon. Longer RNAs were<br>
transcribed with T7 RNA polymerase from PCR products, followed by gel<br>
and Sep-Pak purification. The 49 and 484 bp GL2 or GL3 dsRNAs corre-<br>
sponded to position 113-161 and 113-596, respectively, relative to the<br>
start of translation; the 50 and 501 bp RL dsRNAs corresponded to posi-<br>
tion 118-167 and 118-618, respectively. PCR templates for dsRNA syn-<br>
thesis targeting humanized GFP (hG) were amplified from pAD3 (Kehlen-<br>
bach, 1998), whereby 50 and 501 bp hG dsRNA corresponded to position<br>
118-167 and 118-618, respectively, to the start codon.<br>
For annealing of siRNAs, 20µM single strands were incubated in annealing<br>
buffer (100 mM potassium acetate, 30 mM HEPES-KOH at pH 7.4, 2 mM<br>
magnesium acetate) for 1 min at 90°C followed by 1 h at 37°C. The 37°C<br>
incubation step was extended overnight for the 50 and 500 bp dsRNAs<br>
and these annealing reactions were performed at 8.4 µM and 0.84 µM<br>
strand concentrations, respectively.<br>
2.1.2 Cell Culture<br>
S2 cells were propagated in Schneider's Drosophila medium (Life Technolo-<br>
gies) supplemented with 10% FBS, 100 units/ml penicillin and 100µg/ml<br>
streptomycin at 25°C. 293, NIH/3T3, HeLa S3, COS-7 cells were grown at<br>
37°C in Dulbecco's modified Eagle's medium supplemented with 10%<br>
FBS, 100 units/ml penicillin and 100µg/ml streptomycin. Cells were regu-<br>
larly passaged to maintain exponential growth. 24 h before transfection at<br>
approx. 80% confluency, mammalian cells weretrypsinized and diluted 1:5<br>
with fresh medium without antibiotics (1-3 x 105 cells/ml) and transferred<br>
to 24-well plates (500 µl/well). S2 cells were not trypsinized before split-<br>
ting. Transfection was carried out with Lipofectamine 2000 reagent (Life<br>
Technologies) as described by the manufacturer for adherent cell lines. Per<br>
well, 1.0 µg pGL2-Control (Promega) or pGL3-Control (Promega), 0.1 µg<br>
pRL-TK (Promega) and 0.28 µg siRNA duplex or dsRNA, formulated into<br>
liposomes, were applied; the final \/olume was 600 µl per well. Cells were<br>
incubated 20 h after transfection and appeared healthy thereafter. Lucife-<br>
rase expression was subsequently monitored with the Dual luciferase assay<br>
(Promega). Transfection efficiencies were determined by fluorescence<br>
microscopy for mammalian cell lines after co-transfection of 1.1 fjg hGFP-<br>
encoding pAD3 and 0.28 µg invGL2 inGL2 siRNA and were 70-90%.<br>
Reporter plasmids were amplified in XL-1 Blue (Stratagene) and purified<br>
using the Qiagen EndoFree Maxi Plasmid Kit.<br>
2.2 Results and Discussion<br>
To test whether siRNAs are also capable of mediating RNAi in tissue cultu-<br>
re, we synthesized 21 nt siRNA duplexes with symmetric 2 nt 3' over-<br>
hangs directed against reporter genes coding for sea pansy (Renilla renifor-<br>
mis) and two sequence variants of firefly (Photinus pyralis, GL2 and GL3)<br>
luciferases (Fig. 8a, b). The siRNA duplexes were co-transfected with the<br>
reporter plasmid combinations pGL2/pRL or pGL3/pRL into D. melanogaster<br>
Schneider S2 cells or mammalian cells using cationic liposomes. Luciferase<br>
activities were determined 20 h after transfection. In all cell lines tested,<br>
we observed specific reduction of the expression of the reporter genes in<br>
the presence of cognate siRNA duplexes (Fig. 9a-j). Remarkably, the ab-<br>
solute luciferase expression levels were unaffected by non-cognate<br>
siRNAs, indicating the absence of harmful side effects by 21 nt RNA<br>
duplexes (e.g. Fig. 10a-d for HeLa cells). In D. melanogaster S2 cells (Fig.<br>
9a, b), the specific inhibition of luciferases was complete. In mammalian<br>
cells, where the reporter genes were 50- to 100-fold stronger expressed,<br>
the specific suppression was less complete (Fig. 9c-j). GL2 expression was<br>
reduced 3- to 12-fold, GL3 expression 9- to 25-fold and RL expression 1-<br>
to 3-fold, in response to the cognate siRNAs. For 293 cells, targeting of RL<br>
luciferase by RL siRNAs was ineffective, although GL2 and GL3 targets<br>
responded specifically (Fig. 9i, j). The lack of reduction of RL expression in<br>
293 cells may be due to its 5- to 20-fold higher expression compared to<br>
any other mammalian cell line tested and/or to limited accessibility of the<br>
target sequence due to RNA secondary structure or associated proteins.<br>
Nevertheless, specific targeting of GL2 and GL3 luciferase by the cognate<br>
siRNA duplexes indicated that RNAi is also functioning in 293 cells.<br>
The 2 nt 3' overhang in all siRNA duplexes, except for uGL2, was compo-<br>
sed of (2'-deoxy) thymidine. Substituion of uridine by thymidine in the 3'<br>
overhang was well tolerated in the D. melanogaster in vitro sytem and the<br>
sequence of the overhang was uncritical for target recognition. The thymi-<br>
dine overhang was chosen, because it is supposed to enhance nuclease<br>
resistance of siRNAs in the tissue culture medium and within transfected<br>
cells. Indeed, the thymidine-modified GL2 siRNA was slightly more potent<br>
than the unmodified uGL2 siRNA in all cell lines tested (Fig. 9a, c, e, g, i).<br>
It is conceivable that further modifications of the 3' overhanging nucleoti-<br>
des may provide additional benefits to the delivery and stability of siRNA<br>
duplexes.<br>
In co-transfection experiments, 25 nM siRNA duplexes with respect to the<br>
final volume of tissue culture medium were used (Fig. 9, 10). Increasing<br>
the siRNA concentration to 100 nM did not enhance the specific silencing<br>
effects, but started to affect transfection efficiencies due to competition<br>
for liposome encapsulation between plasmid DNA and siRNA (data not<br>
shown). Decreasing the siRNA concentration to 1.5 nM did not reduce the<br>
specific silencing effect (data not shown), even though the siRNAs were<br>
now only 2- to 20-fold more concentrated than the DNA plasmids. This<br>
indicates that siRNAs are extraordinarily powerful reagents for mediating<br>
gene silencing and that siRNAs are effective at concentrations that are<br>
several orders of magnitude below the concentrations applied in conventio-<br>
nal antisense or ribozyme gene targeting experiments.<br>
In order to monitor the effect of longer dsRNAs on mammalian cells, 50<br>
and 500 bp dsRNAs cognate to the reporter genes were prepared. As non-<br>
specific control, dsRNAs from humanized GFP (hG) (Kehlenbach, 1998)<br>
was used. When dsRNAs were co-transfected, in identical amounts (not<br>
concentrations) to the siRNA duplexes, the reporter gene expression was<br>
strongly and unspecifically reduced. This effect is illustrated for HeLa cells<br>
as a representative example (Fig. 10a-d). The absolute luciferase activities<br>
were decreased unspecifically 10- to 20-fold by 50 bp dsRNA and 20- to<br>
200-fold by 500 bp dsRNA co-transfection, respectively. Similar unspecific<br>
effects were observed for COS-7 and NIH/3T3 cells. For 293 cells, a 10- to<br>
20-fold unspecific reduction was observed only for 500 bp dsRNAs. Un-<br>
specific reduction in reporter gene expression by dsRNA &gt; 30 bp was<br>
expected as part of the interferon response.<br>
Surprisingly, despite the strong unspecific decrease in reporter gene ex-<br>
pression, we reproducibly detected additional sequence-specific, dsRNA-<br>
mediated silencing. The specific silencing effects, however, were only<br>
apparent when the relative reporter gene activities were normalized to the<br>
hG dsRNA controls (Fig. 10e, f). A 2- to 10-fold specific reduction in<br>
response to cognate dsRNA was observed, also in the other three mamma-<br>
lian cell lines tested (data not shown). Specific silencing effects with<br>
dsRNAs (356-1662 bp) were previously reported in CHO-K1 cells, but the<br>
amounts of dsRNA required to detect a 2- to 4-fold specific reduction were<br>
about 20-fold higher than in our experiments (Ui-Tei, 2000). Also CHO-K1<br>
cells appear to be deficient in the interferon response. In another report,<br>
293, NIH/3T3 and BHK-21 cells were tested for RNAi using luciferase/lacZ<br>
reporter combinations and 829 bp specific lacZ or 717 bp unspecific GFP<br>
dsRNA (Caplen, 2000). The failure of detecting RNAi in this case may be<br>
due to the less sensitive luciferase/lacZ reporter assay and the length<br>
differences of target and control dsRNA. Taken together, our results indi-<br>
cate that RNAi is active in mammalian cells, but that the silencing effect is<br>
difficult to detect, if the interferon system is activated by dsRNA &gt; 30 bp.<br>
In summary, we have demonstrated for the first time siRNA-mediated gene<br>
silencing in mammalian cells. The use of short siRNAs holds great promise<br>
for inactivation of gene function in human tissue culture and the develop-<br>
ment of gene-specific therapeutics.<br>
Example 3<br>
Specific Inhibition of Gene Expression by RNA Interference<br>
3.1 Materials and Methods<br>
3.1.1 RNA preparation and RNAi assay<br>
Chemical RNA synthesis, annealing, and luciferase-based RNAi assays<br>
were performed as described in Examples 1 or 2 or in previous publications<br>
(Tuschl et al., 1999; Zamore et al., 2000). All siRNA duplexes were direc-<br>
ted against firefly luciferase, and the luciferase mRNA sequence was<br>
derived from pGEM-luc (GenBank ace. X65316) as described (Tuschl et al.,<br>
1999). The siRNA duplexes were incubated in D. melanogaster RNAi/trans-<br>
lation reaction for 15 min prior to addition of mRNAs. Translation-based<br>
RNAi assays were performed at least in triplicates.<br>
For mapping of sense target RNA cleavage, a 177-nt transcript was<br>
generated, corresponding to the firefly luciferase sequence between posi-<br>
tions 113-273 relative to the start codon, followed by the 17-nt comple-<br>
ment of the SP6 promoter sequence. For mapping of antisense target RNA<br>
cleavage, a 166-nt transcript was produced from a template, which was<br>
amplified from plasmid sequence by PCR using 5' primer<br>
TAATACGACTCACTATAGAGCCCATATCGTTTCATA (T7 promoter<br>
underlined) and 3' primer AGAGGATGGAACCGCTGG. The target sequence<br>
corresponds to the complement of the firefly luciferase sequence between<br>
positions 50-215 relative to the start codon. Guanylyl transferase labelling<br>
was performed as previously described (Zamore et al., 2000). For mapping<br>
of target RNA cleavage, 100 nM siRNA duplex was incubated with 5 to 10<br>
nM target RNA in D. melanogaster embryo lysate under standard condi-<br>
tions (Zamore et al., 2000) for 2 h at 25°C. The reaction was stopped by<br>
the addition of 8 volumes of proteinase K buffer (200 mM Tris-HCI pH 7.5,<br>
25 mM EDTA, 300 mM NaCI, 2% w/v sodium dodecyl sulfate). Proteinase<br>
K (E.M. Merck, dissolved in water) was added to a final concentration of<br>
0.6 mg/ml. The reactions were then incubated for 15 min at 65°C, ex-<br>
tracted with phenol/chloroform/isoamyl alcohol (25:24:1) and precipitated<br>
with 3 volumes of ethanol. Samples were located on 6% sequencing gels.<br>
Length standards were generated by partial RNase T1 digestion and partial<br>
base hydrolysis of the cap-labelled sense or antisense target RNAs.<br>
3.2 Results<br>
3.2.1 Variation of the 3' overhang in duplexes of 21 -nt siRNAs<br>
As described above, 2 or 3 unpaired nucleotides at the 3' end of siRNA<br>
duplexes were more efficient in target RNA degradation than the respective<br>
blunt-ended duplexes. To perform a more comprehensive analysis of the<br>
function of the terminal nucleotides, we synthesized five 21-nt sense<br>
siRNAs, each displayed by one nucleotide relative to the target RNA, and<br>
eight 21-nt antisense siRNAs, each displaced by one nucleotide relative to<br>
the target (Figure 11 A). By combining sense and antisense siRNAs, eight<br>
series of siRNA duplexes with synthetic overhanging ends were generated<br>
covering a range of 7-nt 3' overhang to 4-nt 5' overhang. The interference<br>
of siRNA duplexes was measured using the dual luciferase assay system<br>
(Tuschl et al., 1999; Zamore et al., 2000). siRNA duplexes were directed<br>
against firefly luciferase mRNA, and sea pansy luciferase mRNA was used<br>
as internal control. The luminescence ratio of target to control luciferase<br>
activity was determined in the presence of siRNA duplex and was normali-<br>
zed to the ratio observed in the absence of dsRNA. For comparison, the<br>
interference ratios of long dsRNAs (39 to 504 pb) are shown in Figure 11B.<br>
The interference ratios were determined at concentrations of 5 nM for long<br>
dsRNAs (Figure 11 A) and at 100 nM for siRNA duplexes (Figure 11C-J).<br>
The 100 nM concentrations of siRNAs was chosen, because complete<br>
processing of 5 nM 504 bp dsRNA would result in 120 nM total siRNA<br>
duplexes.<br>
The ability of 21-nt siRNA duplexes to mediate RNAi is dependent on the<br>
number of overhanging nucleotides or base pairs formed. Duplexes with<br>
four to six 3' overhanging nucleotides were unable to mediate RNAi (Figure<br>
11C-F), as were duplexes with two or more 5' overhanging nucleotides<br>
(Figure 11G-J). The duplexes with 2-nt 3' overhangs were most efficient in<br>
mediating RNA interference, though the efficiency of silencing was also<br>
sequence-dependent, and up to 12-fold differences were observed for<br>
different siRNA duplexes with 2-nt 3' overhangs (compare Figure 11D-H).<br>
Duplexes with blunted ends, 1-nt 5' overhang or 1- to 3-nt 3' overhangs<br>
were sometimes functional. The small silencing effect observed for the<br>
siRNA duplex with 7-nt 3' overhang (Figure 11C) may be due to an<br>
antisense effect of the long 3' overhang rather than due to RNAi.<br>
Comparison of the efficiency of RNAi between long dsRNAs (Fig. 11B) and<br>
the most effective 21-nt siRNA duplexes (Fig. 11E, G, H) indicates that a<br>
single siRNA duplex at 100 nM concentration can be as effective as 5 nM<br>
504 bp dsRNA.<br>
3.2.2 Length variation of the sense siRIMA paired to an invariant 21 -nt<br>
antisense siRNA<br>
In order to investigate the effect of length of siRNA on RNAi, we prepared<br>
3 series of siRNA duplexes, combining three 21-nt antisense strands with<br>
eight, 18- to 25-nt sense strands. The 3' overhang of the antisense siRNA<br>
was fixed to 1, 2, or 3 nt in each siRNA duplex series, while the sense<br>
siRNA was varied at its 3' end (Figure 1 2A). Independent of the lenght of<br>
the sense siRNA, we found that duplexes with 2-nt 3' overhang of anti-<br>
sense siRNA (Figure 12C) were more active than those with 1- or 3-nt 3'<br>
overhang (Figure 12B, D). In the first series, with 1-nt 3' overhang of<br>
antisense siRNA, duplexes with a 21- and 22-nt sense siRNAs, carrying a<br>
1- and 2-nt 3' overhang of sense siRNA, respectively, were most active.<br>
Duplexes with 19- to 25-nt sense siRNAs were also able to mediate RNA,<br>
but to a lesser extent. Similarly, in the second series, with 2-nt overhang<br>
of antisense siRNA, the 21-nt siRNA duplex with 2-nt 3' overhang was<br>
most active, and any other combination with the 18- to 25-nt sense<br>
siRNAs was active to a significant degree. In the last series, with 3-nt anti-<br>
sense siRNA 3' overhang, only the duplex with a 20-nt sense siRNA and<br>
the 2-nt sense 3' overhang was able to reduce target RNA expression.<br>
Together, these results indicate that the length of the siRNA as well as the<br>
length of the 3' overhang are important, and that duplexes of 21 -nt siRNAs<br>
with 2-nt 3' overhang are optimal for RNAi.<br>
3.2.3 Length variation of siRNA duplexes with a constant 2-nt 3' overhang<br>
We then examined the effect of simultaneously changing the length of both<br>
siRNA strands by maintaining symmetric 2-nt 3' overhangs (Figure 13A).<br>
Two series of siRNA duplexes were prepared including the 21-nt siRNA<br>
duplex of Figure 11H as reference. The length of the duplexes was varied<br>
between 20 to 25 bp by extending the base-paired segment at the 3' end<br>
of the sense siRNA (Figure 13B) or at the 3' end of the antisense siRNA<br>
(Figure 13C). Duplexes of 20 to 23 bp caused specific repression of target<br>
luciferase activity, but the 21-nt siRNA duplex was at least 8-fold more<br>
efficient than any of the other duplexes. 24- and 25-nt siRNA duplexes did<br>
not result in any detectable interference. Sequence-specific effects were<br>
minor as variations on both ends of the duplex produced similar effects.<br>
3.2.4 2'-Deoxy and 2'-0-methyl-modified siRNA duplexes<br>
To assess the importance of the siRNA ribose residues for RNAi, duplexes<br>
with 21-nt siRNAs and 2-nt 3' overhangs with 2'-deoxy- or 2'-0-methyl-<br>
modified strands were examined (Figure 14). Substitution of the 2-nt 3'<br>
overhangs by 2'-deoxy nucleotides had no effect, and even the replace-<br>
ment of two additional riboncleotides adjacent to the overhangs in the<br>
paired region, produced significantly active siRNAs. Thus, 8 out of 42 nt of<br>
a siRNA duplex were replaced by DNA residues without loss of activity.<br>
Complete substitution of one or both siRNA strands by 2'-deoxy residues,<br>
however, abolished RNAi, as did substitution by 2'-0-methyl residues.<br>
3.2.5 Definition of target RNA cleavage sites<br>
Target RNA cleavage positions were previously determined for 22-nt siRNA<br>
duplexes and for a 21-nt/22-nt duplex. It was found that the position of<br>
the target RNA cleavage was located in the centre of the region covered by<br>
the siRNA duplex, 11 or 12 nt downstream of the first nucleotide that was<br>
complementary to the 21 - or 22-nt siRNA guide sequence. Five distinct 21 -<br>
nt siRNA duplexes with 2-nt 3' overhang (Figure 15A) were incubated with<br>
5' cap-labelled sense or antisense target RNA in D. melanogaster lysate<br>
(Tuschl et al., 1999; Zamore et al., 2000). The 5' cleavage products were<br>
resolved on sequencing gels (Figure 1 5B). The amount of sense target RNA<br>
cleaved correlates with the efficiency of siRNA duplexes determined in the<br>
translation-based assay, and siRNA duplexes 1, 2 and 4 (Figure 15B and<br>
11H, G, E) cleave target RNA faster than duplexes 3 and 5 (Figure 1 5B and<br>
11F, D). Notably, the sum of radioactivity of the 5' cleavage product and<br>
the input target RNA were not constant over time, and the 5' cleavage<br>
products did not accumulate. Presumably, the cleavage products, once<br>
released from the siRNA-endonuclease complex, are rapidly degraded due<br>
to the lack of either of the poly(A) tail of the 5'-cap.<br>
The cleavage sites for both, sense and antisense target RNAs were located<br>
in the middle of the region spanned by the siRNA duplexes. The cleavage<br>
sites for each target produced by the 5 different duplexes varied by 1-nt<br>
according to the 1-nt displacement of the duplexes along the target se-<br>
quences. The targets were cleaved precisely 11 nt downstream of the<br>
target position complementary to the 3'-most nucleotide of the sequence-<br>
complementary guide siRNA (Figure 15A, B).<br>
In order to determine, whether the 5' or the 3' end of the guide siRNA sets<br>
the ruler for target RNA cleavage, we devised the experimental strategy<br>
outlined in Figure 16A and B. A 21-nt antisense siRNA, which was kept<br>
invariant for this study, was paired with sense siRNAs that were modified<br>
at either of their 5' or 3' ends. The position of sense and antisense target<br>
RNA cleavage was determined as described above. Changes in the 3' end<br>
of the sense siRNA, monitored for 1-nt 5' overhang to 6-nt 3' overhang,<br>
did neither effect the position of sense nor antisense target RNA cleavage<br>
(Figure 16C). Changes in the 5' end of the sense siRNA did no affect the<br>
sense target RNA cleavage (Figure 16D, top panel), which was expected<br>
because the antisense siRNA was unchanged. However, the antisense<br>
target RNA cleavage was affected and strongly dependent on the 5' end of<br>
the sense siRNA (Figure 16D, bottom panel). The antisense target was only<br>
cleaved, when the sense siRNA was 20 or 21 nt in size, and the position<br>
of cleavage different by 1-nt, suggesting that the 5' end of the target-<br>
recognizing siRNA sets the ruler for target RNA cleavage. The position is<br>
located between nucleotide 10 and 11 when counting in upstream direc-<br>
tion from the target nucleotide paired to the 5'-most nucleotide of the<br>
guide siRNA (see also Figure 15A).<br>
3.2.6 Sequence effects and 2'-deoxy substitutions in the 3' overhang<br>
A 2-nt 3'overhang is preferred for siRNA function. We wanted to know, if<br>
the sequence of the overhanging nucleotides contributes to target reco-<br>
gnition, or if it is only a feature required for reconstitution of the endonu-<br>
clease complex (RISC or siRNP). We synthesized sense and antisense<br>
siRNAs with AA, CC, GG, UU, and UG 3' overhangs and included the 2'-<br>
deoxy modifications TdG and TT. The wild-type siRNAs contained AA in<br>
the sense 3' overhang and UG in the antisense 3' overhang (AA/UG). All<br>
siRNA duplexes were functional in the interference assay and reduced<br>
target expression at least 5-fold (Figure 17). The most efficient siRNA<br>
duplexes that reduced target expression more than 10-fold, were of the<br>
sequence type NN/UG, NN/UU, NN/TdG, and NN/TT (N, any nucleotide).<br>
siRNA duplexes with an antisense siRNA 3' overhang of AA, CC or GG<br>
were less active by a factor 2 to 4 when compared to the wild-type se-<br>
quence UG or the mutant UU. This reduction in RNAi efficiency is likely<br>
due to the contribution of the penultimate 3' nucleotide to sequence-speci-<br>
fic target recognition, as the 3' terminal nucleotide was changed from G to<br>
U without effect.<br>
Changes in the sequence of the 3' overhang of the sense siRNA did not<br>
reveal any sequence-dependent effects, which was expected, because the<br>
sense siRNA must not contribute to sense target mRNA recognition.<br>
3.2.7 Sequence specifity of target recognition<br>
In order to examine the sequence-specifity of target recognition, we in-<br>
troduced sequence changes into the paired segments of siRNA duplexes<br>
and determined the efficiency of silencing. Sequence changes were in-<br>
troduced by inverting short segments of 3- or 4-nt length or as point muta-<br>
tions (Figure 18). The sequence changes in one siRNA strand were com-<br>
pensated in the complementary siRNA strand to avoid pertubing the base-<br>
paired siRNA duplex structure. The sequence of all 2-nt 3' overhangs was<br>
TT (T, 2'-deoxythymidine) to reduce costs of synthesis. The TT/TT refe-<br>
rence siRNA duplex was comparable in RNAi to the wild-type siRNA duplex<br>
AA/UG (Figure 17). The ability to mediate reporter mRNA destruction was<br>
quantified using the translation-based luminescence assay. Duplexes of<br>
siRNAs with inverted sequence segments showed dramatically reduced<br>
ability for targeting the firefly luciferase reporter (Figure 18). The sequence<br>
changes located between the 3' end and the middle of the antisense siRNA<br>
completely abolished target RNA recognition, but mutations near the 5' end<br>
of the antisense siRNA exhibit a small degree of silencing. Transversion of<br>
the A/U base pair located directly opposite of the predicted target RNA<br>
cleavage site, or one nucleotide further away from the predicted site,<br>
prevented target RNA cleavage, therefore indicating that single mutation<br>
within the centre of a siRNA duplex discriminate between mismatched<br>
targets.<br>
3.3 Discussion<br>
siRNAs are valuable reagents for inactivation of gene expression, not only<br>
in insect cells, but also in mammalian cells, with a great potential for<br>
therapeutic application. We have systematically analysed the structural<br>
determinants of siRNA duplexes required to promote efficient target RNA<br>
degradation in D. melanogaster embryo lysate, thus providing rules for the<br>
design of most potent siRNA duplexes. A perfect siRNA duplex is able to<br>
silence gene expression with an efficiency comparable to a 500 bp dsRNA,<br>
given that comparable quantities of total RNA are used.<br>
3.4 The siRNA user guide<br>
Efficiently silencing siRNA duplexes are preferably composed of 21-nt<br>
antisense siRNAs, and should be selected to form a 19 bp double helix<br>
with 2-nt 3' overhanging ends. 2-deoxy substitutions of the 2-nt 3' over-<br>
hanging ribonucleotides do not affect RNAi, but help to reduce the costs of<br>
RNA synthesis and may enhance RNAse resistance of siRNA duplexes.<br>
More extensive 2'-deoxy or 2'-0-methyl modifications, however, reduce<br>
the ability of siRNAs to mediate RNAi, probably by interfering with protein<br>
association for siRNAP assembly.<br>
Target recognition is a highly sequence-specific process, mediated by the<br>
siRNA complementary to the target. The 3'-most nucleotide of the guide<br>
siRNA does not contribute to specificity of target recognition, while the<br>
penultimate nucleotide of the 3' overhang affects target RNA cleavage, and<br>
a mismatch reduces RNAi 2- to 4-fold. The 5' end of a guide siRNA also<br>
appears more permissive for mismatched target RNA recognition when<br>
compared to the 3' end. Nucleotides in the centre of the siRNA, located<br>
opposite the target RNA cleavage site, are important specificity determi-<br>
nants and even single nucleotide changes reduce RNAi to undetectable<br>
level. This suggests that siRNA duplexes may be able to discriminate<br>
mutant or polymorphic alleles in gene targeting experiments, which may<br>
become an important feature for future therapeutic developments.<br>
Sense and antisense siRNAs, when associated with the protein compo-<br>
nents of the endonclease complex or its commitment complex, were sug-<br>
gested to play distinct roles; the relative orientation of the siRNA duplex in<br>
this complex defines which strand can be used for target recognition.<br>
Synthetic siRNA duplexes have dyad symmetry with respect to the double-<br>
helical structure, but not with respect to sequence. The association of<br>
siRNA duplexes with the RNAi proteins in the D. melanogaster lysate will<br>
lead to formation of two asymmetric complexes. In such hypothetical<br>
complexes, the chiral environment is distinct for sense and antisense<br>
siRNA, hence their function. The prediction obviously does not apply to<br>
palindromic siRNA sequences, or to RNAi proteins that could associate as<br>
homodimers. To minimize sequence effects, which may affect the ratio of<br>
sense and antisense-targeting siRNPs, we suggest to use siRNA sequences<br>
with identical 3' overhanging sequences. We recommend to adjust the<br>
sequence of the overhang of the sense siRNA to that of the antisense 3'<br>
overhang, because the sense siRNA does not have a target in typical<br>
knock-down experiments. Asymmetry in reconstitution of sense and anti-<br>
sense-cleaving siRNPs could be (partially) responsible for the variation in<br>
RNAi efficiency observed for various 21-nt siRNA duplexes with 2-nt 3'<br>
overhangs used in this study (Figure 14). Alternatively, the nucleotide<br>
sequence at the target site and/or the accessibility of the target RNA<br>
structure may be responsible for the variation in efficiency for these siRNA<br>
duplexes.<br>
References<br>
Bass, B. L. (2000). Double-stranded RNA as a template for gene silencing.<br>
Cell 101, 235-238.<br>
Bosher, J. M., and Labouesse, M. (2000). RNA interference: genetic wand<br>
and genetic watchdog. Nat. Cell Biol. 2, E31-36.<br>
Caplen, N. J., Fleenor, J., Fire, A., and Morgan, R. A. (2000). dsRNA-<br>
mediated gene silencing in cultured Drosophila cells: a tissue culture model<br>
for the analysis of RNA interference. Gene 252, 95-105.<br>
Catalanotto, C, Azzalin, G., Macino, G., and Cogoni, C. (2000). Gene<br>
silencing in worms and fungi. Nature 404, 245.<br>
Chanfreau, G., Buckle, M., and Jacquier, A. (2000). Recognition of a<br>
conserved class of RNA tetraloops by Saccharomyces cerevisiae RNase III.<br>
Proc. Natl. Acad. Sci. USA 97, 3142-3147.<br>
Clemens, M. J. (1997). PKR--a protein kinase regulated by double-stranded<br>
RNA. Int. J. Biochem. Cell Biol. 29, 945-949.<br>
Cogoni, C, and Macino, G. (1999). Homology-dependent gene silencing in<br>
plants and fungi: a number of variations on the same theme. Curr. Opin.<br>
Microbiol. 2, 657-662.<br>
Dalmay, T., Hamilton, A., Rudd, S., Angell, S., and Baulcombe, D. C.<br>
(2000). An RNA-dependent RNA polymerase gene in Arabidopsis is<br>
required for posttranscriptional gene silencing mediated by a transgene but<br>
not by a virus. Cell 101, 543-553.<br>
Dernburg, A. F., Zalevsky, J., Colaiacovo, M. P., and Villeneuve, A. M.<br>
(2000). Transgene-mediated cosuppression in the C. elegans germ line.<br>
Genes &amp; Dev. 14, 1578-1583.<br>
Dunn, J. J. (1982). Ribonuclease III. In The enzymes, vol 15, part B, P. D.<br>
Boyer, ed. (New York: Academic Press), pp. 485-499.<br>
Filippov, V., Solovyev, V., Filippova, M., and Gill, S. S. (2000). A novel<br>
type of RNase III family proteins in eukaryotes. Gene 245, 213-221.<br>
Fire, A. (1999). RNA-triggered gene silencing. Trends Genet. 15, 358-363.<br>
Fire, A., Xu, S., Montgomery, M. K., Kostas, S. A., Driver, S. E., and<br>
Mello, C. C. (1998). Potent and specific genetic interference by double-<br>
stranded RNA in Caenorhabditis elegans. Nature 391, 806-811.<br>
Grishok, A., Tabara, H., and Mello, C. C. (2000). Genetic requirements for<br>
inheritance of RNAi in C. elegans. Science 287, 2494-2497.<br>
Hamilton, A. J., and Baulcombe, D. C. (1999). A species of small anti-<br>
sense RNA in posttranscriptional gene silencing in plants. Science 286,<br>
950-952.<br>
Hammond, S. M., Bernstein, E., Beach, D., and Hannon, G. J. (2000). An<br>
RNA-directed nuclease mediates post-transcriptional gene silencing in<br>
Drosophila cells. Nature 404, 293-296.<br>
Jacobsen, S. E., Running, M. P., and M., M. E. (1999). Disruption of an<br>
RNA helicase/RNase III gene in Arabidopsis causes unregulated cell division<br>
in floral meristems. Development 126, 5231-5243.<br>
Jensen, S., Gassama, M. P., and Heidmann, T. (1999). Taming of trans-<br>
posable elements by homology-dependent gene silencing. Nat. Genet. 21,<br>
209-212.<br>
Kehlenbach, R. H., Dickmanns, A. &amp; Gerace, L. (1998). Nucleocytoplasmic<br>
shuttling factors including Ran and CRM1 mediate nuclear export of NFAT<br>
In vitro. J. Cell Biol. 141, 863-874<br>
Kennerdell, J. R., and Carthew, R. W. (1998). Use of dsRNA-mediated<br>
genetic interference to demonstrate that frizzled and frizzled 2 act in the<br>
wingless pathway. Cell 95, 1017-1026.<br>
Ketting, R. F., Haverkamp, T. H., van Luenen, H. G., and Plasterk, R. H.<br>
(1999). Mut-7 of C. elegans, required for transposon silencing and RNA<br>
interference, is a homolog of Werner syndrome helicase and RNaseD. Cell<br>
99, 133-141.<br>
Ketting, R. F., and Plasterk, R. H. (2000). A genetic link between co-<br>
suppression and RNA interference in C. elegans. Nature 404, 296-298.<br>
Lucy, A. P., Guo, H. S., Li, W. X., and Ding, S. W. (2000). Suppression of<br>
post-transcriptional gene silencing by a plant viral protein localized in the<br>
nucleus. EM BO J. 19, 1672-1680.<br>
Matsuda, S., Ichigotani, Y., Okuda, T., Irimura, T., Nakatsugawa, S., and<br>
Hamaguchi, M. (2000). Molecular cloning and characterization of a novel<br>
human gene (HERNA) which encodes a putative RNA-helicase. Biochim.<br>
Biophys. Acta 31, 1-2.<br>
Milligan, J.F., and Uhlenbeck, O.C. (1989). Synthesis of small RNAs using<br>
T7 RNA polymerase. Methods Enzymol. 180, 51-62.<br>
Mourrain, P., Beclin, C, Elmayan, T., Feuerbach, F., Godon, C, Morel, J.<br>
B., Jouette, D., Lacombe, A. M., Nikic, S., Picault, N., Remoue, K.f Sanial,<br>
M., Vo, T. A., and Vaucheret, H. 12000). Arabidopsis SGS2 and SGS3<br>
genes are required for posttranscriptional gene silencing and natural virus<br>
resistance. Cell 101, 533-542.<br>
Ngo, H., Tschudi, C, Gull, K., and Ullu, E. (1998). Double-stranded RNA<br>
induces mRNA degradation in Trypanosoma brucei. Proc. Natl. Acad. Sci.<br>
USA 95, 14687-14692.<br>
Nicholson, A. W. (1999). Function, mechanism and regulation of bacterial<br>
ribonucleases. FEMS Microbiol. Rev. 23, 371-390.<br>
Oelgeschlager, M., Larrain, J., Geissert, D., and De Robertis, E. M. (2000).<br>
The evolutionary conserved BMP-binding protein Twisted gastrulation<br>
promotes BMP signalling. Nature 405, 757-763.<br>
Pan, T., and Uhlenbeck, 0. C. (1992). In vitro selection of RNAs that<br>
undergo autolytic cleavage with Pb2~. Biochemistry 31, 3887-3895.<br>
Pelissier, T., and Wassenegger, M. (2000). A DNA target of 30 bp is<br>
sufficient for RNA-directed methylation. RNA 6, 55-65.<br>
Plasterk, R. H., and Ketting, R. F. (2000). The silence of the genes. Curr.<br>
Opin. Genet. Dev. 10, 562-567.<br>
Ratcliff, F. G., MacFarlane, S. A., and Baulcombe, D. C. (1999). Gene<br>
Silencing without DNA. RNA-mediated cross-protection between viruses.<br>
Plant Cell 11, 1207-1216.<br>
Robertson, H. D. (1990). Escherichia coli ribonuclease III. Methods Enzy-<br>
mol. 181, 189-202.<br>
Robertson, H. D. (1982). Escherichia coli ribonuclease III cleavage sites.<br>
Cell 30, 669-672.<br>
Romaniuk, E., McLaughlin, L. W., Neilson, T., and Romaniuk, P. J. (1982).<br>
The effect of acceptor oligoribonucleotide sequence on the T4 RNA ligase<br>
reaction. Eur J Biochem 125, 639-643.<br>
Sharp, P. A. (1999). RNAi and double-strand RNA. Genes &amp; Dev. 13, 139-<br>
141.<br>
Sijen, T., and Kooter, J. M. (2000). Post-transcriptional gene-silencing:<br>
RNAs on the attack or on the defense? Bioessays 22, 520-531.<br>
Smardon, A., Spoerke, J., Stacey, S., Klein, M., Mackin, N., and Maine, E.<br>
(2000). EGO-1 is related to RNA-directed RNA polymerase and functions in<br>
germ-line development and RNA interference in C. elegans. Curr. Biol. 10,<br>
169-178.<br>
Svoboda, P., Stein, P., Hayashi, H., and Schultz, R. M. (2000). Selective<br>
reduction of dormant maternal mRNAs in mouse oocytes by RNA inter-<br>
ference. Development 127, 4147-4156.<br>
Tabara, H., Sarkissian, M., Kelly, V\/. G., Fleenor, J., Grishok, A., Timmons,<br>
L., Fire, A., and Mello, C. C. (1999). The rde-1 gene, RNA interference,<br>
and transposon silencing in C. elegans. Cell 99, 123-132.<br>
Tuschl, T., Ng, M. M., Pieken, W., Benseler, F., and Eckstein, F. (1993).<br>
Importance of exocyclic base functional groups of central core guanosines<br>
for hammerhead ribozyme activity. Biochemistry 32, 11658-11668.<br>
Tuschl, T., Sharp, P. A., and Bartel, D. P. (1998). Selection in vitro of<br>
novel ribozymes from a partially randomized U2 and U6 snRNA library.<br>
EMBO J. 17, 2637-2650.<br>
Tuschl, T., Zamore, P. D., Lehmann, R., Bartel, D. P., and Sharp, P. A.<br>
(1999). Targeted mRNA degradation by double-stranded RNA in vitro.<br>
Genes &amp; Dev. 13, 3191-3197.<br>
Ui-Tei, K., Zenno, S., Miyata, Y. &amp; Saigo, K. (2000). Sensitive assay of<br>
RNA interference in Drosophila and Chinese hamster cultured cells using<br>
firefly luciferase gene as target. FEBS Letters 479, 79-82.<br>
Verma, S., and Eckstein, F. (1999). Modified oligonucleotides: Synthesis<br>
and strategy for users. Annu. Rev. Biochem. 67, 99-134.<br>
Voinnet, O., Lederer, C, and Baulcombe, D. C. (2000). A viral movement<br>
protein prevents spread of the gene silencing signal in Nicotiana ben-<br>
thamiana. Cell 103, 157-167.<br>
Wassenegger, M. (2000). RNA-directed DNA methylation. Plant Mol. Biol.<br>
43, 203-220.<br>
Wianny, F., and Zernicka-Goetz, M. (2000). Specific interference with gene<br>
function by double-stranded RNA in early mouse development. Nat. Cell<br>
Biol. 2, 70-75.<br>
Wu, H., Xu, H., Miraglia, L. J., and Crooke, S. T. (2000). Human RNase III<br>
is a 160 kDa Protein Involved in Preribosomal RNA Processing. J. Biol.<br>
Chem. 17, 17.<br>
Yang, D., Lu, H. and Erickson, J.W. (2000) Evidence that processed small<br>
dsRNAs may mediate sequence-specific mRNA degradation during RNAi in<br>
drosophilia embryos. Curr. Biol., 10, 1191-1200.<br>
Zamore, P. D., Tuschl, T., Sharp, P. A., and Bartel, D. P. (2000). RNAi:<br>
Double-stranded RNA directs the ATP-dependent cleavage of mRNA at 21<br>
to 23 nucleotide intervals. Cell 101, 25-33.<br>
Zhang, K., and Nicholson, A. W. (1997). Regulation of ribonuclease III<br>
processing by double-helical sequence antideterminants. Proc. Natl. Acad.<br>
Sci. USA 94, 13437-13441.<br>
WE CLAIM:<br>
1. Isolated double-stranded RNA molecule, wherein each RNA strand<br>
has a length from 19-23 nucleotides, wherein said RNA molecule is<br>
capable of target-specific nucleic acid modifications and wherein at<br>
least one strand has a 3-overtiang from 1-5 nucleotides.<br>
2. The RNA molecule as claimed in claim 1 wherein at least one strand<br>
has a 3-overhang from 1-5 nucleotides.<br>
3. The RNA molecule as claimed in claim 1 or 2 capable of target-specific<br>
RNA interference and/or DNA methylation.<br>
4. The RNA molecule as claimed in any one of claims 1-3, wherein each<br>
strand has a length from 19-23, particularly from 20-22 nucleotides.<br>
5. The RNA molecule as claimed in any one of claim 2-4, wherein the 3-<br>
over-hang is from 1-3 nucleotides.<br>
6. The RNA molecule as claimed in any one of claims 2-5, wherein the 3*-<br>
over-hand is stabilized against degradation.<br>
7. The RNA molecule as claimed in any one of claims 1-6, which contains<br>
at least one modified nucleotide analogue.<br>
8. The RNA molecule as claimed in claim 7, wherein the modified<br>
nucleotide analogue is selected from sugar-or backbone modified<br>
ribonucleotides.<br>
9. The RNA molecule as claimed in claim 7 or 8, wherein the nucleotide<br>
analogue is a sugar-modified ribonucleotide, wherein the 2-OH group<br>
is replaced by a group selected from H, OR, R, halo, SH, SR1,NH2,<br>
NHR, NH2 or CN, wherein R is C1-C6 alkyl, alkenyl or alkynyl and<br>
halo is F, CI, Br or I.<br>
10. The RNA molecule as claimed in claim 7 or 8, wherein the nucleotide<br>
analogue is a backbone-modified ribonucleotide containing a<br>
phosphothioate group.<br>
11. The RNA molecule as claimed in any one of claims 1-10, which has a<br>
sequence having an identity of at least 50 percent to a predetermined<br>
mRNA target molecule.<br>
12. The RNA molecule as claimed in claim 11, wherein the identity is at<br>
least 70 percent.<br>
13. A method of preparing a double- stranded RNA molecule as claimed in<br>
any one of claims 1-12 comprising the steps<br>
(a) synthesizing two RNA strands each having a length from 19-25<br>
nucleotides, wherein said RNA strands are capable of forming a<br>
double-stranded RNA molecule,<br>
(b) combining the synthesized RNA strands under conditions, wherein<br>
a double-stranded RNA molecule is formed, which is capable of<br>
target-specific nucleic acid modifications.<br>
14. The method as claimed n claim 13, wherein the RNA strands are<br>
chemically synthesized.<br>
15. The method as claimed in claim 13, wherein the RNA strands are<br>
enzymatically synthesized<br>
16. A method of mediating target-specific nucleic acid modifications Hi a<br>
cell or an organism comprising the steps:<br>
(a) contacting said cell or organism with the double-stranded RNA<br>
molecule as claimed in any one of claims 1-12 under conditions<br>
wherein target-specific nucleic acid modifications can occur, and<br>
(b) mediating a target-specific nucleic acid modification effected by the<br>
double-stranded RNA towards a target nucleic acid having a<br>
sequence portion substantially corresponding to the double-<br>
stranded RNA.<br>
17. The method as claimed in claim 16, wherein the nucleic acid<br>
modification is RNA interference and/or DNA methylation.<br>
18. The method as claimed in claims 16 and 17 wherein said contacting<br>
comprises introducing said double-stranded RNA molecule into a<br>
target cell in which the target-specific nucleic acid modification can<br>
occur.<br>
19. The method as claimed in cairn 18 wherein the introducing comprises<br>
a carrier-mediated delivery or injection.<br>
20. The method as claimed in any one of claims 16-19 for determining the<br>
function of a gene in a cell o art organism in vitro<br>
21. The method as claimed in any one of claims 16 to 19 for modulating<br>
the function of a gene in a call or an organism.<br>
22. The method as claimed in claim 20 or 21, wherein the gene is<br>
associated with pathological condition.<br>
23. The method as claimed in claim 22, wherein the gene is a pathogen-<br>
associated gene.<br>
24. The method as claimed in caim 23, wherein the gene is a viral gene.<br>
25. The method as claimed in claim 22, wherein the gene is a tumor-<br>
associated gene.<br>
26. The method as claimed in claim 22, wherein the gene Is an<br>
autoimmune disease-associated gene.<br>
27. Pharmaceutical composition containing as any active agent at least<br>
one double-stranded RNA molecule as claimed in any one of claims 1<br>
-12 and a pharmaceutical carrier<br>
28. The composition as claimed n claim 27 for diagnostic applications.<br>
29. The composition as claimed in claim 27 for therapeutic applications.<br>
30. A genetically engineered isolated eukaryotic cell or an eukaryotic non-<br>
human organism exhibiting a target gene-specific knockout phenotype<br>
wherein said cell or organism is transfected with at least one double-<br>
stranded RNA molecule capable of inhibiting the expression of an<br>
endogeneous target gene or with a DNA encoding at least one double-<br>
stranded RNA molecule capable of inhibiting the expression of at least<br>
one endogeneous target gene.<br>
31. The cell or organism as claimed in claim 30 which is mammalian cell.<br>
32. The cell or organism as claimed in claim 31 which is a human cell.<br>
33. The ceil or organism as claimed in any one claims 30-32 which is<br>
transfected with at least one exogeneous target nucleic acid coding for<br>
the target protein or a va iant or mutated form of the target protein,<br>
wherein said exogeneous target nucleic acid differs from the<br>
endogeneous target gen 3 on the nucleic acid level such that<br>
expression of the exogencous target nucleic acid is substantially less<br>
inhibited by the double stranded RNA molecule than the expression of<br>
the endogeneous target gene.<br>
34. The cell or organism as claimed in claim 33 wherein the exogeneous<br>
target nucleic acid is fused to a nucleic acid sequence encoding a<br>
detectable peptide or polypeptide.<br>
35. A method for carrying out at analytic procedure comprising the use of<br>
a call or organism as claimed in any of claims 30-34 in vitro.<br>
36. The method as claimed in claim 35 for carrying out the analysis of<br>
gene expression profiles.<br>
37. The method as claimed in claim 35 for carrying out a proteome<br>
analysis.<br>
38. The method as claimed in any one of claims 35-37 wherein an analysis<br>
of a variant or mutant form of the target protein encoded by an<br>
exogeneous target nucleic acid is carried out.<br>
39. The method as claimed in claim 38 for Identifying functional domains of<br>
the target protein.<br>
40. The method as claimed in any one of claims 35-39 wherein a<br>
comparison of at least two cells or organisms is carried out selected<br>
from:<br>
(i) a control cell or control organism without target gene inhibition,<br>
(ii) a cell or organism with target gene inhibition and<br>
(j) a cell or organism with target gene inhibition plus target gene<br>
complementation by an exogeneous target nucleic acid.<br>
41. The method as claimed in any one of claims 35-40 wherein the<br>
analysis comprises a functional and/or phenotypic analysis.<br>
42. A method for carrying out preparative procedures comprising the use<br>
of a cell as claimed in any one clams 30-34.<br>
43. The method as claimed in claim 41 for the isolation of proteins or<br>
protein complexes from eukeryotic cells.<br>
44. The method as claimed in claim 43 for the isolation of high molecular<br>
weight protein complexes which may optionally contain nucleic acid.<br>
45. The method as claimed in any one of claims 35-44 in a procedure for<br>
identifying and/or characterizing pharmacological agents.<br>
46. The system for identifying and/or characterizing a pharmacological<br>
agent acting on at least one target protein comprising:<br>
(a) a genetically engineered isolated eukaryotic cell or an eukaryotic<br>
non-human organism capable of expressing at least one target<br>
gene coding for said at least one target protein.<br>
(b) at least one double-stranded RNA molecule capable of inhibiting<br>
the expression of said at least one endogeneous target gene, and<br>
(c)a test substance or a collection of test substances wherein<br>
pharmacological properties of said test substance or said collection<br>
are to be identified and/or characterized.<br>
47. The system as claimed in claim 46 comprising:<br>
(d) at least one exogeneous target nucleic sod coding for the target<br>
protein or a variant or mutated from of the target protein wherein said<br>
exogeneous target nucleic acid differs from the endogeneout target<br>
gene on the nucteic acid level such that the expression of the<br>
exogoneous target nucleic acid is substantially lose inhibited by the<br>
double stranded RNA molecule than the expression of the<br>
endogeneous target gene.<br><br>
Isolated double-stranded RNA molecule, wherein each RNA strand has a<br>
length from 19-23 nucleotides, wherein said RNA molecule is capable of<br>
target-specific nucleic acid modifications and wherein at least one strand has<br>
a 3'-overhang from 1-5 nucleotides.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLUtPTE5QLTIwMDMtKDIyLTA0LTIwMTMpLUNPUlJFU1BPTkRFTkNFLnBkZg==" target="_blank" style="word-wrap:break-word;">612-KOLNP-2003-(22-04-2013)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLUtPTE5QLTIwMDMtQU1BTkRFRCBDTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">612-KOLNP-2003-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLUtPTE5QLTIwMDMtQ09SUkVTUE9OREVOQ0UgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">612-KOLNP-2003-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtcGN0IHJlcXVlc3QgZm9ybS5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-pct request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjEyLWtvbG5wLTIwMDMtdHJhbnNsYXRlZCBjb3B5IG9mIHByaW9yaXR5IGRvY3VtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">612-kolnp-2003-translated copy of priority document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="261200-process-for-conversion-of-a-dimethyl-2h-cyclopenta-alpha-phenanthren-17-14h-into-a-10-13-dimethyl-1h-cyclopenta-alpha-phenanthren-17-yl-triflate.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="261202-impact-modified-thermoplastic-resin-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>261201</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>612/KOLNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>24/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jun-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Jun-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-May-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MAX-PLANCK-GESELLSHAFT ZUR FORDERUNG DER WISSENSCHAFTEN E.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HOFGARTENSTRASSE 8, 80539 MUNCHEN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TUSCHL THOMAS</td>
											<td>KEPLERSTRASSE 9, 37085 GOTTINGEN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LENDECKEL WINFRIED</td>
											<td>BLINDE GASSE 52, 37318</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MATTHIAS WILM</td>
											<td>LANDFRIEDSTR. 1, 69117 HEIDELBERG</td>
										</tr>
										<tr>
											<td>4</td>
											<td>REINHARD LUHRMANN</td>
											<td>EWIGES TAL 16B 35041 MARBURG-MICHEL-BACH</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ELBASHIR SAYDA</td>
											<td>SULTEBECKSBREITE 2, 37075, GOTTINGEN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2001/13968</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>00126325.0</td>
									<td>2000-12-01</td>
								    <td>EPO</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/279,661</td>
									<td>2001-03-30</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/261201-double-stranded-rna-molecule-and-method-of-preparing-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:28:46 GMT -->
</html>
