<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/262602-an-isolated-peptide-useful-for-diagnosing-preventing-or-treating-coeliac-disease by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:45:01 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 262602:AN ISOLATED PEPTIDE USEFUL FOR DIAGNOSING, PREVENTING OR TREATING COELIAC DISEASE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED PEPTIDE USEFUL FOR DIAGNOSING, PREVENTING OR TREATING COELIAC DISEASE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention herein disclosed is related to epitopes useful in methods of diagnosing, treating, and preventing coeliac disease. Therapeutic compositions which comprise at least one epitope are provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>EPITOPES RELATED TO COELIAC DISEASE<br>
The invention relates to epitopes useful in the diagnosis and therapy of coeliac disease, including diagnostics, therapeutics, kits, and methods of using the foregoing.<br>
Coeliac disease is caused by an immune mediated hypersensitivity to dietary gluten. Gluten proteins in wheat, rye, barley and in some cases oats are toxic in coeliac disease. Gluten is composed of alpha/beta, gamma and omega gliadins, and low and high molecular weight (LMW and HMW) glutenins in wheat, hordeins in barley, secalins in rye and avenins in oats. Hordeins and secalins are homologous to gamma and omega gliadins and low and high molecular weight glutenins in wheat. Avenins are phylogenetically more distant than hordeins and secalins fro.ni wheat gluten.<br>
The goal of research in coeliac disease has been to define the toxic components of gluten by defining the peptides that stimulate gluten-specific T-cells. Precise definition of gluten epitopes permits development of new diagnostics, therapeutics, tests for gluten contamination in food and non-toxic grains that retain the cooking/baking qualities of traditional gluten. Many of these applications require a comprehensive understanding of all rather man the most common toxic peptides in gluten.<br>
Genes encoding HLArDQ2 and/or HLA-DQ8 are present in over 99% of individuals with coeliac disease compared to approximately 35%, of the general Caucasian population. Gluten-derived peptides (epitopes) bound to HLA-DQ2 or .HLA-DQ8 stimulate specific T-cells. HLA-DQ2 and DQS-restricted epitopes include a,"core" 9.amino acid sequence that directly interacts with the peptide binding groove of HLA-DQ2 or DQ8 and with cognate T-cell receptors. In general, libraries of overlapping peptides (usually 15 to 20mers) containing all unique 10 or 12mer peptides in an antigen have been used to map HLA class H-restricted T-cells epitopes.<br>
A series of gluten peptides are known to activate gluten specific T-cells in coeliac disease.<br>
Previous studies have identified'gluten peptides from selected gluten proteins or gluten digests. T-<br>
cell clones and lines isolated from intestinal biopsies have been used to screen these gluten<br>
components.	.<br>
Modification of gluten by the enzyme, tissue transglutaminase (tTG) present in intestinal tissue, substantially increases gluten's stimulatory capacity on gluten specific T-cells. Most of the known epitopes for gluten-specific T-cells correspond to tTG-deamidated gluten peptides. Transglutaminase mediates deamidation of specific glutarriine residues (to glutamate) in gluten. Glutamine-containing sequences susceptible to •deamidation by tTG generally conform to a motif: QXPX or QXX(FYMILVW) (see Vader W.'et al 2002 J. JExp. Med. 195:643-649, PCT WO 03/066079, and Fleckenstein B. 2002. J Bio] Chem 277:34109-16). The motif for peptides that bind to HLA-DQ2 and that are susceptible to deamidation by tTG has been used to predict certain gluten epitopes (Vader et al J Exp Med 2002 J, Exp. Med. 195:643-649, PCT WO 03/066079).<br>
However, other groups have identified epitopes for gluten-specific intestinal T-cell clones and lines using panels of eleven recombinant alpha/beta (11) and five gamma gliadins (Arentz-<br><br>
HansenH. 2000. J. Exp. Med. 191:603-612, Arentz-Hansen H. 2002. Gastroenterology 123:803-809, PCT WO 02/083722), and lysates of purified gluten proteins (Sjostrom H. et al 1998. Scand. J. Immunol 48,111-115; van de Wai, Y. et al 1998. J. Immmol. 161(4):1585-1588; van de Wai, Y. et al 1999. Ew: J. Immunol. 29:3133-3139; Vader W. et al 2002. Gastroenterology 122:1729-1737.).<br>
Our work has exploited the observation that gluten challenge in vivo induces HLA-DQ2 restricted CD44- gluten-specific T-cells in peripheral blood expressing a gut-homing integrin (alpha4beta7). This technique allowed the mapping of the dominant epitope in A-gliadin (57-73 QE65) (Anderson, RP et al 2000. Nat. Med 6:337-342., WO 01/25793). A-gliadin 57-73 QE65 corresponds to two overlapping epitopes identified using intestinal T-cell clones (Arentz-Hansen H. et al 2000. J. Exp. Med. 191:603-612, Arentz-Hansen H. et al 2002. Gastroenterology 123:803-809). The advantage of in vivo gluten challenge to induce gluten specific T-cells is that any food can be consumed and the resulting T-cells induced in blood (quantified in peripheral blood using a simple overnight interferon gamma ELISPOT assay) will have been stimulated in vivo by endogenously presented epitopes, rather than primed in vitro by a synthetic or purified antigen. Overnight assays of fresh polyclonal peripheral blood T-cells also avoid the potential for artefacts associated with the lengthy purification of T-cell clones.<br>
Interestingly, T-cell clones and lines specific for several gamma-gliadin epitopes (Arentz-Hansen H. 2002. Gastroenterology 123:803-809, PCT WO 02/083722) cross-react with the originally defined A-gliadin epitope 57-73 .QE65. -<br>
Although there is substantial homology within the alpha/beta gliadins, earlier work (see<br>
WO 03/104273) has shown that the. dominant epitope recognized in HLA-DQ2-associated coeliac<br>
disease, "A-gliadin 57-73 QE65", is encoded by a minority of the alpha/beta gliadins present in<br>
Genbank.	'<br>
SUMMARY OF THE INVENTION<br>
The current study set out to develop a method that would allow mapping of all T-cell epitopes in gluten. Consumption of wheat bread (200g daily for 3 days) or oats (lOOg daily for 3 days) was used to induce gluten or avenin-specific T-cells in peripheral blood (collected 6 days after beginning the challenge). Peripheral blood mononuclear cells (PBMC) were assessed hi overnight interferon gamma ELISPOT assays'using a library of gluten and avenin peptides including all unique 12mer sequences included in every Genbank entry for wheat gluten and/or oat avenins. This goal was achieved by establishing an algorithm to design peptides spanning all potential epitopes in gluten proteins in Genbank (2922 20mers included all 14 '964 unique 9mers - potential T-cell epitopes), adapting the interferon-gamma ELISPOT assay to a high throughput assay capable of screening over 1000 peptides with a single individual's blood and developing bioinformatics tools to analyse and interpret the data generated.<br>
A series of 41 "superfamilies" of wheat gluten peptides were identified as putative T-cell epitopes. Superfamilies shared motifs in which a limited level of redundancy was allowed. Many of<br><br>
the most potent families include known T-cell epitopes including the previously described dominant epitope, A-gliadin 57-73.<br>
Through comprehensive mapping of gluten epitopes using PBMC after gluten challenge, the inventors have found a series of novel 'gliadin, LMW and HMW glutenin, and avenin epitopes for coeliac disease associated with HLA-DQ2 and HLA-DQ8. Novel epitopes were identified for HLA-DQ2 and HLA-DQ8-associated coeliac disease. HLA-DQ2 and HLA-DQ8 associated coeliac disease are genetically and functionally distinct in terms of the range of T-cell epitopes that are recognized. In addition, three peptides present in avenin proteins of oats also activated peripheral blood mononuclear cells (PBMC) following oats challenge in HLA-DQ2+ coeliac subjects, the first time oats epitopes have been defined. Identification of avenin peptides recognized by T-cells following oats challenge in vivo provides a molecular basis for the observed occasional relapse of coeliacs following oat exposure (Lundin KEA et al. 2003 Gut 52:1649-52) and may provide a basis for a predictive diagnostic or genetic de-toxification of oats.<br>
The data presented here will provide a comprehensive basis for definition of both common "dominant" and occasional "weak" T-cell epitopes in coeliac disease. This information is the platform for functional applications such as. diagnostics, food tests, irnmunotherapeutics and prophylactics, and for design of non-toxic gluten proteins useful in modified grains.<br>
In particular, through comprehensive mapping, of gluten T cell epitopeSj the inventors have found epitopes bioactive in coeliac disease in HLA-DQ2+ patients in wheat gliadins and glutenins, having similar core sequences (e.g., SEQ ID NOS: 1-199) and similar extended sequences (e.g., SEQ ID NOS:200-15543 1555-1655, 1656-1671,,and 1830-1903). The inventors have also found epitopes bioactive in coeliac disease in HLA-J)Q2+ patients in: oat avenins having similar core sequences (e.g., SEQ ID NOS: 1684-1695) and similar extended .sequences (e.g., SEQ ID NO: 1672-1683, 1696-1698, and 1764-1768); rye secalins (SEQ ID NOS: 1769-1786); and barley hordeins (SEQ ID NOS: 1787-1829). Additionally, epitopes bioactive in coeliac disease in HLA-DQ8+ patients have been identified in wheat gliadins having similar .core sequences (e.g., SEQ ID NOS: 1699-1721) and similar extended sequences (e.g., SEQ ID NOS: 1722-1763 and 1908-1927). This comprehensive mapping thus provides dominant epitopes recognized by T cells in coeliac patients. Thus, the methods of the invention described herein may be performed using any of these identified epitopes, and analogues and equivalents thereof. That is,' the agents of the invention include these epitopes. Additionally, combinations of epitopes, i^e., "combitopes" or single peptides comprising two or more epitiopes, have been shown to induce equivalent responses as the individual epitopes, indicating that several epitopes may be utilized for therapeutic, 'diagnostic, and other uses of the invention. Such combitopes may be in the form of, e;g., SEQ ID NO: 1906. Preferably, the agents of the invention include one or more of the epitopes having the sequences listed recited in SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655,: 1656-1671, 1672-1698, 1699-1763, 1764-1768,<br><br>
1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927 and analogues and equivalents thereof as defined herein.<br>
Preferred agents that are bioactive in .coeliac disease in HLA-DQ8+ patients possess a glutamine in a sequence that suggests susceptibility to deamidation separated by seven residues from a second glutamine also susceptible to deamidation (e.g., as found in QGSFQPSQQ) wherein the deamidated sequences are high affinity binders for HLA-DQ8 following deamidation by tTG (The binding motif for HLArDQ8 favours glutamate at positions 1 and 9.) In a less preferred embodiment, the agent possesses glutamine residues .susceptible to deamidation but not separated by seven residues from a second glutamine susceptible to.tTG-mediated .deamidation.<br>
The invention thus provides a method of diagnosing coeliac disease, or susceptibility to coeliac disease, in an individual, comprising the steps of: (a) contacting a sample from the host with an agent selected from (i) the epitope comprising an amino acid sequence selected from SEQ ID NOS: 1-1927, preferably selected from SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927, or an equivalent sequence from a naturally occurring gluten protein, (ii) an analogue of (i) which is capable of being recognised by a T cell receptor that recognises (i), which in the case of a peptide analogue is not more than 50 amino acids in length, or (iii) a product comprising two or more agents as defined in (i) or (ii); and (b) determining in vitro "whether T cells in the sample recognise the agent, with recognition by the T cells indicating that the individual has, or is susceptible to, coeliac disease.<br>
The term "gluten protein" encompasses alpha/beta, gamma and omega gliadins, and low and high molecular weight (LMW and HMW) gmtenins in wheat, hordeins in barley, secalins in rye, and avenins in oats. The invention is particularly concerned with gliadins and avenins.<br>
The invention also provides use of the agent for the preparation of a diagnostic means for use in a method of diagnosing coeliac disease, or susceptibility to coeliac disease, hi an individual, said method comprising determining whether T-cells of the individual recognise the agent, recognition by the T cells indicating that the individual has, or is susceptible to, coeliac disease.<br>
The finding of epitopes which are modified by transglutaminase also allows diagnosis of coeliac disease based on determining whether other types of immune response to these epitopes are present. Thus the invention also provides a method of diagnosing coeliac disease, or susceptibility to coeliac disease, in an individual comprising .determining the presence of an antibody that binds to the epitope in a sample from the individual, the presence of the,antibody indicating that the individual has, or is susceptible to, coeliac disease: ....,,'•<br>
The invention provides a method of determining whether a composition is capable of causing coeliac disease comprising determining whether a protein capable of being modified by a transglutaminase to an oligopeptide sequence as defined above is present in the composition, the presence of the protein indicating that the composition is capable of causing coeliac disease.<br><br>
The invention also provides a mutant gluten protein whose wild-type sequence can be modified by a transglutaminase to a sequence that comprises an epitope comprising sequence as defined above, but which mutant gluten protein has been modified in such a way that it does not contain sequence which can be modified by a transglutaminase to a sequence that comprises such an epitope comprising sequence; or a fragment of such a mutant gluten protein which is at least 7 amino acids long (e.g., at least 7, 8, 9, 10, 11, 12, 13, 14 or 15 amino acids long) and which comprises sequence which has been modified in said way.<br>
The invention also provides a protein that comprises a sequence which is able to bind to a T cell receptor, which T cell receptor recognises the agent, and which sequence is able to cause antagonism of a T cell that carries such a T cell receptor.<br>
Additionally the invention provides a food that comprises the proteins defined above.<br>
The invention additionally provides the agent, optionally in association with a carrier, for use in a method of treating or preventing coeliac disease by tolerising T cells which recognise the agent. Also provided is an antagonist of a T cell .which has a T cell receptor that recognises (i), optionally in association with a carrier, for use in a .method of treating or preventing coeliac disease by antagonising such T cells. Additionally provided is the agent or an analogue that binds an antibody (that binds the agent) for use in a method of treating or preventing coeliac disease in an individual by tolerising the individual to prevent the production of such an antibody.<br>
The invention also provides methods of preventing or treating coeliac disease comprising administering to an individual at least one agent selected from: a) a peptide comprising at least one epitope comprising a sequence selected from the group consisting of SEQ ID NOs: 1-1927, preferably from the group consisting of SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927, and equivalents thereof,.and b) an analogue of a) which is capable of being recognised by a T cell receptor that recognises the peptide of a) and which is not more than 50 amino acids in length. In some embodiments, the agent is HLA-DQ2-restricted;, HLA-DQ8-restricted or one agent is HLA-DQ2-restricted and a second agent is HLA-DQ8-restricted. In some embodiments, the agent comprises.a wheat epitope, an oat epitope, a rye epitope, a barley epitope or any combination thereof either as a single agent or as multiple agents.<br>
The present invention also provides methods .of preventing or treating coeliac disease comprising administering to an individual a pharmaceutical composition comprising an agent as described above and pharmaceutically acceptable carrier or diluent.<br>
The present invention also provides methods of preventing or treating coeliac disease comprising administering to an individual a pharmaceutical composition comprising an antagonist of a T cell which has a T cell receptor as defined above, and a pharmaceutically acceptable carrier or diluent.<br><br>
The present invention also provides methods of preventing or treating coeliac disease comprising administering to an individual a composition for tolerising an individual to a gluten protein to suppress the production of a T cell or antibody response to an agent as defined above, which composition comprises an agent as defined above.<br>
The present invention also provides methods of preventing or treating coeliac disease by 1) diagnosing coeliac disease in an individual by either: a) contacting a sample from the host with at least one agent selected from: i) a peptide comprising at least one epitope comprising a sequence selected from the group consisting of: SEQ ID NOS: 1-1927, preferably selected from the group consisting of SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927, and equivalents thereof; and ii) an analogue of i) which is capable of being recognised by a T cell receptor that recognises i) and which is not more than 50 amino acids in length; and determining in vitro whether T cells in the sample recognise the agent; recognition by the T cells indicating that the individual has, or is susceptible to, coeliac disease; or b) administering an agent as defined above and determining in vivo whether T cells in the individual recognise the agent, recognition of the agent indicating that the individual has or is -susceptible to coeliac disease; and 2) administering to an individual diagnosed as having, or being susceptible to, coeliac disease a therapeutic agent for preventing or treating coeliac disease.<br>
The present invention also provides agents as defined above, optionally in association with a<br>
carrier, for use hi a method of treating or preventing coeliac disease by tolerising T cells which<br>
recognise the agent.	. ^<br>
The present invention also provides antagonists of a T cell which has a T cell receptor as<br>
defined above, optionally in association with a carrier, for use in a method of treating or preventing<br>
coeliac disease by antagonising such T cells. •-, •	.<br>
The present invention also provides proteins that comprises a sequence which is able to bind to a T cell receptor, which T cell receptor recognises an agent as defined above, and which sequence is able to cause antagonism of a T cell that carries such a T cell receptor.<br>
The present invention also provides pharmaceutical compositions comprising an agent or antagonist as defined and a pharmaceutically acceptable carrier or diluent.<br>
The present invention also provides compositions for tolerising an individual to a gluten protein to suppress the production of a T cell or antibody response to an agent as defined above, which composition comprises an agent as defined above.<br>
The present invention also provides compositions for antagonising a T cell response to an agent as defined above, which composition comprises an antagonist as defined above.<br>
The present invention also provides mutant gluten proteins whose wild-type sequence can be modified by a transglutaminase to a sequence which is an agent as defined above, which mutant gluten protein comprises a mutation'which prevents its modification by a transglutaminase to a<br><br>
sequence which is an agent as defined above; or a fragment of such a mutant gluten protein which is at least 7 amino acids long (e.g., at least 7, 8, 9,10,11,12,13,14 or 15 amino acids long) and which comprises the mutation.<br>
The present invention also provides polynucleotides that comprise a coding sequence that encodes a protein or fragment as defined above.<br>
The present invention also provides cells comprising a polynucleotide as defined above or which have been transformed with such a polynucleotide.<br>
The present invention also provides mammals that expresses a T cell receptor as defined above.<br>
The present invention also provides methods of diagnosing coeliac disease, or susceptibility to coeliac disease, in an individual comprising: a) contacting a sample from the host with at least one agent selected from i) apeptide comprising at least one epitope comprising a sequence selected from the group consisting of: SEQ ID NOS: 1-1927, preferably selected from the group consisting of SEQ ID NOS: SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927, and equivalents thereof; and ii) an analogue of i) which is capable of being recognised by a T cell receptor that recognises i) and which is not more than 50 amino acids in length; and b) determining in vitro whether T cells in the sample recognise the agent; recognition by the T cells indicating that the individual has, or is susceptible to, coeliac disease.<br>
The present invention also provides methods of determining whether a composition is capable of causing coeliac disease comprising determining whether a protein capable of being modified by a transglutatninase to an, oligopeptide sequence is present in the composition, the presence of the protein indicating that the composition is capable of causing coeliac disease.<br>
The present invention also provides methods of identifying an antagonist of a T cell, which T<br>
cell recognises an agent as defined above, comprising contacting a candidate substance with the T<br>
cell and detecting whether the substance causes a decrease in the ability of the T cell to undergo an<br>
antigen specific response, the detecting ,of! any such decrease in said ability indicating that the<br>
substance is an antagonist. ;	•<br>
The present invention also provides kits for carrying out any of the methods described above comprising an agent as defined above and a means to detect the recognition of the peptide by the T cell.<br>
The present invention also provides methods of identifying a product which is therapeutic for coeliac disease comprising.administering a candidate substance to a mammal as defined above which has, or which is susceptible to, coeliac disease and determining whether substance prevents or treats coeliac disease in the mammal, the prevention or treatment, of coeliac disease indicating that the substance is a therapeutic product.<br><br>
The present invention also provides .processes for the production of a protein encoded by a<br>
coding sequence as defined above which process comprises: a) cultivating a cell described above<br>
under conditions that allow the expression of the protein; and optionally b) recovering the expressed<br>
protein.	.	,<br>
The present invention also provides methods of obtaining a transgenic plant cell comprising transforming a plant cell with'a vector as described above to give a transgenic plant cell.<br>
The present invention also provides methods of obtaining a first-generation transgenic plant comprising regenerating a transgenic plant cell transformed with a vector as described above to give a transgenic plant.<br>
The present invention also provides methods of obtaining a transgenic plant seed comprising obtaining a transgenic seed from a transgenic plant obtainable as described above.<br>
The present invention also provides methods of obtaining a transgenic progeny plant comprising obtaining a second-generation transgenic progeny plant from a first-generation transgenic plant obtainable by a method as described above, and optionally obtaining transgenic plants of one or more further generations from the second-generation progeny plant thus obtained.<br>
The present invention also ^provides transgenic plant cells, plants, plant seeds or progeny plants obtainable by any of the methods described above.<br>
The present invention also provides transgenic plants or plant seeds comprising plant cells as described above.<br>
The present invention also provides transgenic plant cell calluses comprising plant cells as<br>
described above obtainable from a transgenic plant cell, first-generation plant, plant seed or progeny<br>
as defined above.	. •-.'. •:.,••'	r<br>
The present invention also provides methods of obtaining a crop product comprising harvesting a crop product from a plant according to any method described above and optionally further processing the harvested product.<br>
The present invention also provides food that comprises a protein as defined above.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows a method to generate all possible peptide epitopes from a group of proteins.<br>
Figure 2 shows Genbank accession numbers for gluten gene products present hi the Genbank<br>
database on 16th June 2003.<br>
Figure 3A shows an expectation maximization^OBM) algorithm to analyze data from ELISpot.<br>
Figure 3B shows a test on a dataset of patients with coeliac disease.<br>
Figure 4 shows an iterative procedure to find minimal set of responsive epitopes.<br>
Figure 5 shows gliadin and g]utenin sequences (SEQ ID NOS: 1-1554). In the "consensus" column, letters in lower case use the standard one letter amino acid code, but letters in upper case have a<br><br>
different meaning: E=[e or q\, F=[f or y or w],.I=|a or 1 or v]3 S=[s or t], R=[r or k or h]. The "sequence" column uses the standard one letter amino acid code.<br>
Figure 6 shows gluten peptides that stimulate gamma interferon in PBMC collected 6 days after gluten challenge in HLA-DQ2+ coeliac disease volunteers (SEQ ID NOS: 1555-1655). The indicated 9mers are common to 200 groups of bioactive "structurally" related 20mer peptides. The gluten sequences are ranked according to the bioactivity X proportion of subjects responding.<br>
Figure 7 shows the results of a wheat challenge experiment (SEQ ID NOS: 1656-1671). These peptides gave high quality responses (indicated ' Y') in ten subjects (A-J) after wheat challenge.<br>
Figure 8 shows Avenin peptides (+/-deamidation by tTG) that stimulate interferon-y in PBMC collected 6 days after gluten challenge in HLA-DQ2+ coeliac disease volunteers (SEQ ID NOS: 1672-1698). Those marked with a * are optimal unique 20mers inducing IFN-^ after oats challenge. Figure 9 shows the most potent 40 20mers (SEQ ID NOS: 1699-1763) in two HLA-DQ8 (not HLA-DQ2) subjects grouped according to shared core sequences. The core sequence of group 6 (QGSFQPSQQ) corresponds to the alpha-gliadin epitope described by van de Wai et al (J. Immunol. 1998, 161(4):1585-1588). The maximum response in Subject A was 271 SFC (medium alone, no peptide response: 4 SFC), and in B it was 26 SFC (medium alone, no peptide response: 1 SFC).<br>
Figure 10 shows the amino acid sequence .of A-gliadin (SEQ ID NO: 1928) based on ammo acid<br>
sequencing.	:	.	.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The term "coeliac disease" encompasses a spectrum of conditions caused by varying degrees of gluten sensitivity, including a severe form characterised by a flat small intestinal mucosa (hyperplastic villous atrophy) and other forms characterised by milder symptoms.<br>
The individual mentioned above (in the context of diagnosis or therapy) is human. They may have coeliac disease (symptomatic or asymptomatic) or be suspected of having it. They may be on a gluten free diet. They may be hi an acute phase response (for example they may have coeliac disease, but have only ingested gluten in the last 24 hours before which they had been on a gluten free diet for 14 to 28 days).<br>
The individual may be susceptible to coeliac disease, such as a genetic susceptibility (determined for example by the individual having relatives with coeliac disease or possessing genes which cause predisposition to coeliac disease).<br>
The agent<br>
The agent is typically a peptide, for example of length 7 to 50 amino acids, such as 10 to 40, 12 to 35 or 15 to 30 amino acids in length.<br>
The agent may be the peptide represented by any of SEQ ID NOS: 1-1927 or an epitope comprising sequence that comprises any of SEQ ID NOS: 1-1927 which is an isolated oligopeptide<br><br>
derived from a gluten protein; or an equivalent of these sequences from a naturally occurring gluten protein. In a further set of embodiments of the invention, the agent is any peptide'epitope of an oat gluten (e.g. any T eel] epitope of an avenin).<br>
Preferably, the agent is the peptide represented by any of SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927 or an epitope comprising sequence that comprises any of SEQ ID NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927 which is an isolated oligopeptide derived from a gluten protein; or an equivalent of these sequences from a naturally occurring gluten protein.<br>
Thus the epitope may be a derivative of a naturally occurring gluten protein, particularly from a wheat or oat gluten. Such a derivative is typically a fragment of the gluten protein, or a mutated derivative of the whole protein or fragment. Therefore the epitope of the invention does not include the naturally occurring whole gluten protein, and does not include other whole naturally occurring gluten proteins.<br>
Typically such fragments will be at least 7 amino acids in length (e,g., at least 7, 8, 9, 10, 11, 12,13,14 or 15 amino acids .in length).<br>
Typically such fragments will be recognised by T cells to at least the same extent that the<br>
agents from which they .are derived are.recognised hi any of the assays described herein using<br>
samples from coeliac disease patients.	'	•' • •<br>
The agent may be the peptide represented by any of SEQ ED NOS: 1-1927, preferably the peptide represented by any of SEQ ID NOS: '15784579, .1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927 or a protein comprising a sequence corresponding to any of SEQ ED NOS: 1-1927, preferably comprising a sequence corresponding to any of from SEQ ED NOS: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927 (such as fragments of a gluten protein comprising any of SEQ ID NOS: 1-1927. and preferably any of from SEQ ID NOS: 1578-1579, 15824583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927, for example after the gluten protein has been treated with transglutaminase). Bioactive, fragments of such sequences are also agents of the invention. Typically such fragments will be at least 7 amino acids in length (e.g., at least 7, 8, 9, 10, 11, 12, 13, 14 or 15 amino acids in length). Sequences equivalent to any of SEQ ID NOS: 1-1927 or analogues of these sequences are also agents.of the invention.<br>
In the case where the'epitope comprises a sequence equivalent to the above epitopes (including fragments) from another gluten protein (e.g. any of the gluten proteins mentioned herein or any gluten proteins which cause coeliac disease), such equivalent sequences will correspond to a<br><br>
fragment of a gluten protein typically treated (partially or fully) with transglutaminase. Such equivalent peptides can be determined by aligning the sequences of other gluten proteins with the gluten protein from which the original epitope derives (for example using any of the programs mentioned herein). Transglutaminase is commercially available (e.g. Sigma T-5398).<br>
The agent which is an analogue is capable of being recognised by a TCR which recognises (i). Therefore generally when the analogue is added to T cells in the presence of (i), typically also in the presence of an antigen presenting cell (APC) (suc'h as any of the APCs mentioned herein), the analogue inhibits the recognition of (i), i.e. the analogue is able to compete with (i) in such a system.<br>
The analogue may be one which is capable of binding the TCR which recognises (i). Such binding can be tested by standard techniques. Such TCRs can be isolated from T cells which have been shown to recognise (i) (e.g. using the method of the invention). Demonstration of the binding of the analogue to the TCRs can then shown by determining whether the TCRs inhibit the binding of the analogue to a substance that binds the analogue, e.g. an antibody to the analogue. Typically the analogue is bound to a class It MHC molecule (e.g. HLA-DQ2) in such an inhibition of binding assay.<br>
Typically the analogue inhibits the binding of (i) to a TCR. In this case the amount of (i) which can bind the TCR in the presence of the analogue is decreased. This is because the analogue is able to bind the TCR and therefore competes with (i) for binding to the TCR.<br>
T cells for use in the above binding experiments can be isolated from patients with coeliac disease, for example with the aid of the method of the" invention. Other binding characteristics of the analogue may also be the same as (i), and thus typically the analogue binds to the same MHC class n molecule to which the peptide binds (HLA-DQ2 :'or -DQ8). The analogue typically binds to antibodies specific for (i), and thus inhibits binding.of (i) to such antibodies.<br>
The analogue is typically a peptide. It may have homology with (i), typically at least 70% homology, preferably at least 80, 90%, 95%, 97% or 99% homology with (i), for example over a region of at least 7, 8, 9, 10, 11, 12, 13, 14, 15 or more (such as the entire length of the analogue and/or (i), or across the region which contacts the TCR or binds the MHC molecule) contiguous amino acids. Methods of measuring1 protein homology are well known in the art and it will be understood by those of skill in the,art that in the present context, homology is calculated on the basis of amino acid identity (sometimes referred to as "hard homology").<br>
For example the UWGCG Tackage provides the BESTFTT program which can be used to calculate homology (for example used on its'default .settings) (Devereux et al (1984) Nucleic Acids Research 12, p387-395). The PILEUP and BLAST algorithms can be used to calculate homology or align sequences (typically on their default settings), for example as described in Altschul S. F. (1993) J Mol Evol 36:290-300; Altschul, S, F et al (1990) J Mol Biol 215:403-10.<br>
Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information on the world wide web through the Internet at, for example,<br><br>
"www.ncfai.nlmaiih.gov/". This algorithm involves first identifying high scoring sequence pair (HSPs) by identifying short words of length W in the query sequence that either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighbourhood word score threshold (Altschul et al, supra). These initial neighbourhood word hits act as seeds for initiating searches to find HSPs containing them. The word hits are extended in both directions along «ach sequence for as far as the cumulative alignment score can be increased. Extensions for the word hits hi each direction are halted when: the cumulative alignment score falls off by the quantity X from its maximum achieved value; the cumulative score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments; or the end of either sequence is reached. The BLAST algorithm parameters W, T and X determine the sensitivity and speed of the alignment. The BLAST program uses as defaults a word length (W) of 11, the BLOSUM62 scoring matrix (see Henikoff and Henikoff (1992) Proc. Natl. Acad. Sci. USA 89: 10915-10919) alignments (B) of 50, expectation (E) of 10, M=5, N=4, and a comparison of both strands.<br>
The BLAST algorithm performs a statistical analysis of the similarity between two sequences; see e.g., Karlin and Altschul (1993) Proc. Natl. Acad. Sci. USA 90: 5873-5787. One measure of similarity provided by the BLAST:algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example/a'sequence is considered similar to another sequence if the smallest sum probability in comparison of the first sequence to the second sequence is less than about 1, preferably Jess than'.about 0.1, more preferably less than about 0,01, and most preferably less than about 0.001.<br>
The homologous peptide analogues typically differ from (i) by 1,2, 3, 4, 5, 6, 7, 8 or more mutations (which may be substitutions, deletions or insertions). These mutations may be measured across any of the regions mentioned above in relation to calculating homology. The substitutions are preferably 'conservative'. These are defined according to the following Table. Amino acids in the same block hi the second column and preferably in the same line in the third column may be substituted for each other:<br><br>
ALIPHATIC 	Non-polar 	GAP <br><br><br>
	I  L V <br><br>
		C   S   T  M <br><br>
	.   •     Poiar — uncharged 	N  Q <br><br>
	• -'     • • - , 	D E <br><br>
	Polar   charged 	K  R <br>
AROMATIC 		H  F W Y <br><br>
Typically the amino acids in the analogue at the equivalent positions to amino acids in (i) that contribute to binding the MHC molecule or are responsible for the recognition by the TCR, are the same or are conserved.<br>
Typically the analogue peptide comprises one or more modifications, which may be natural<br>
post-translation modifications or artificial modifications. The modification may provide a chemical<br>
moiety (typically by substitution of a hydrogen, e.g. of a C-H bond), such as an amino, acetyl,<br>
hydroxy or halogen (e.g. fluorine) group or carbohydrate group. Typically the modification is present<br>
on the N or C terminus.	:<br>
The analogue may comprise one or more non-natural amino acids, for example amino acids with a side chain different from natural amino acids. Generally, the non-natural ammo acid will have an N terminus and/or a C terminus. The non-natural amino acid may be an L- or a D- amino acid.<br>
The analogue typically has a shape, size, flexibility or electronic configuration that is substantially similar to (i). It is typically a derivative of (i). In one embodiment the analogue is a fusion protein comprising the sequence of any of SEQ ID NOS: 1-1927 and non-gluten sequence. Preferably, the analogue according to this embodiment is a fusion protein comprising the sequence of any of SEQ ID NOs: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, 1764-1768, 1769-1786, 1787-1829, 1895-1903, 1906, and 1908-1927 and non-gluten sequence<br>
In one embodiment the analogue is or mimics (i) bound to a MHC class II molecule. 2, 3, 4 or more of such complexes .may be associated or bound to each other, for example using a biotin/streptavidin based system, in which typically 2, 3 or 4 biotin labelled MHC molecules bind to a streptavidin moiety. This analogue typically inhibits the binding of the (i)/MHC Class D complex to a TCR or antibody which is specific for the complex.<br>
The analogue is typically, an antibody or a fragment of an antibody, such as a Fab or F(ab')2 fragment. The analogue may be immobilised on a solid support, particularly an analogue that mimics peptide bound to a MHC molecule.<br>
The analogue is typically designed by computational means and then synthesised using methods known in the art. Alternatively the analogue can be selected from a library of compounds. The library may be a combinatorial library or a display library, such as a phage display library. The library of compounds may be expressed in the display-library in the form of being bound to a MHC class U molecule, such as HLA-DQ2 or -DQ8. Analogues are generally selected from the library based on then- ability to mimic the binding characteristics (i)- Thus they may be selected based on ability to bind a TCR or antibody which recognises (i).<br>
Typically analogues will be recognised by T cells to at least the same extent as any of the agents (i), for example at least to the same extent .as the equivalent epitope is recognised in any of the assays described herein, typically using T .cells from coeliac disease patients. Analogues may be recognised to these extents in vivo and thus may be able to induce coeliac disease symptoms to at<br><br>
least the same extent as any of the agents mentioned herein (e.g. in a human patient or animal model).<br>
Analogues may be identified in a method comprising determining whether a candidate substance is recognised by a T cell receptor mat recognises an epitope of the invention, recognition of the substance indicating that the substance is an analogue. Such TCRs may be any of the TCRs mentioned herein, and may be present on T cells. Any suitable assay mentioned herein can be used to identify the analogue. In one embodiment this method is carried out in vivo. As mentioned above preferred analogues are recognised to at least the same extent as the equivalent epitope, and so the method may be used to identify analogues which are recognised to this extent.<br>
In one embodiment the method comprises determining whether a candidate substance is able to inhibit the recognition of an epitope of the invention, inhibition of recognition indicating that the substance is an analogue.<br>
The agent may be a product comprising at least 2,5, 10 or 20 agents as defined by (i) or (ii). Typically the composition comprises epitopes of the invention (or equivalent analogues) from different gluten proteins, such as any of the species or variety of or types of gluten protein mentioned herein. Preferred compositions comprise at least one epitope of the invention, or equivalent analogue, from all of the glutens present in any of the species or variety mentioned herein, or from 2, 3, 4 or more of the species mentioned herein (such as from the panel of species consisting of wheat, rye, barley, oats and triticale). Thus, the agent may be monovalent or multivalent.<br>
According to certain embodiments -of the .invention, the agent does not have or is not based on a sequence disclosed in WO 02/083722 and/or WO ,01/25793 and/or WO03/104273 and/or recited in any of SEQ ID NOS: 1555-1577, .1580-1581, 1584-1586, 1594-1599, 1621-1622 and/or is not an agent derived from A-gliadin, the sequence of which is -given in Figure 10.<br>
Within SEQ ID NOs: 1-1927, a preferred subset is SEQ ID NOs: 1-1763. Within SEQ ID NOs: 1764-1927, preferred subsets are: (a),oat sequences 1764-1768, (b) rye sequences 1769-1786, (c) barley sequences 1787-1829, (d) wheat ^sequences 1895-1903, (e) wheat DQ8 sequences 1908-1927, and (f) combitope sequence 1906. Other preferred subsets are: (a) 1764-1768, (b) 1769-1773, (c) 1774-1786, (d) 1787-1792, (e) 1793-1829, (f&gt; 1830-1894, (g) 1895-1903, (h) 1830-1903, (i) 1904-1906, (j) 1908-1916, (k) 1917-1927,, and (1) 1908-1913,1915-1923 and 1925-1927.<br>
Within SEQ ID NOs: 1578-1579, 1582-1583, 1587-1593, 1600-1620, 1623-1655, 1656-1671, 1672-1698, 1699-1763, &amp; 1764-1927, particularly preferred wheat epitopes are SEQ ID NOs: 1656-1671,1830-1903, and 1907-1927.<br>
Within SEQ ID NOs: 1830-1894 (Table 1), some sequences have N-terminal and C-terminal glycines. The invention extends to these sequences omitting the C-terminal glycine and/or the N-terminal glycine. Preferably, both the C-terminal glycine and the N-terminal glycine are omitted.<br><br>
Diagtiosis<br>
As mentioned above the method of diagnosis of the invention may be based on the detection of T cells that bind the agent or on the detection of antibodies that recognise the agent.<br>
The T cells that recognise the agent in the method (which includes the use mentioned above) are generally T cells that have been pre-sensitised in vivo to one or more gluten proteins. As mentioned above such antigen-experienced T cells have been found to be present in the peripheral blood.<br>
In the method the T cells can be contacted with the agent in vitro or in vivo, and determining whether the T cells recognise the agent can be performed in vitro or in vivo. Thus the invention provides the agent for use in a method of diagnosis practiced on the human body. Different agents are provided for simultaneous, separate or sequential use in such a method.<br>
The in vitro method is typically carried out in aqueous solution into which the agent is added. The solution will also comprise the T cells (and in certain embodiments the APCs discussed below). The term 'contacting' as used herein includes adding the particular substance to the solution.<br>
Determination of whether the T cells recognise the agent is generally accomplished by detecting a change in the state of the T cells in the presence of the agent or determining whether the T cells bind the agent. The change in state is generally caused by antigen specific functional activity of the T cell after the TCR binds the agent. The change of state may be measured inside (e.g. change in intracellular expression of proteins) or outside (e;g. detection of secreted substances) the T cells.<br>
The change hi state of the T cell may be the start of or increase in secretion of a substance<br>
from the T cell, such as a cytokine, especially IFN-y, IL-2 or TNF-a. Determination of IFN-y<br>
secretion is particularly preferred. The substance can typically be detected by allowing it to bind to a<br>
specific binding agent and then measuring the presence of the specific binding agent/substance<br>
complex. The specific binding agent is typically an antibody, such as polyclonal or monoclonal<br>
antibodies. Antibodies to cytokines are commercially available, or can be made using standard<br>
techniques.	•<br>
Typically the specific binding agent is immobilised on a solid support. After the siibstance is allowed to bind the solid support can optionally be washed to remove material which is not specifically bound to the agent The agent/substance .complex may be detected by using a second binding agent that will bind the complex. Typically 'the second agent binds the substance at a site which is different from the site which binds the first agent. The second agent is preferably an antibody and is labelled directly or indirectly by a detectable label.<br>
Thus the second agent may be detected by a third agent that is typically labelled directly or indirectly by a detectable label. 'For example the second agent may comprise a biotin moiety, allowing detection by a third agent which comprises a streptavidin moiety and typically alkaline phosphatase as a detectable label.<br><br>
In one embodiment the detection system which is used is the ex-vivo ELISPOT assay described in WO 98/23960. In that assay IFN-y secreted from the T cell is bound by a first EFN-y specific antibody that is immobilised on a solid support. The bound IFN-y is then detected using a second IFN-y specific antibody which is labelled with a detectable label. Such a labelled antibody can be obtained from MABTECH (Stockholm, Sweden). Other detectable labels which can be used are discussed below.<br>
The change in state of the T cell that can be measured may be the increase in the uptake of substances by the T cell, such as the uptake of thymidine. The change in state may be an increase in the size of the T cells, or proliferation of the T cells, or a change in cell surface markers on the T cell.<br>
In one embodiment the change of state is detected by measuring the change in the intracellular expression of proteins, for example the increase in intracellular expression of any of the cytokmes mentioned above. Such intracellular changes may be detected by contacting the inside of the T cell with a moiety that binds the expressed proteins in a specific manner and which allows sorting of the T cells by flow cytometry.<br>
In one embodiment when binding the TCR the agent is bound to an MHC class If molecule<br>
(typically HLA-DQ2 or -DQ8), which is typically present on the surface of an antigen presenting cell<br>
(APC). However as mentioned herein other agents can bind a TCR without the need to also bind an<br>
MHC molecule.	,	.<br>
Generally the T cells which are contacted in.the method are taken from the individual in a<br>
blood sample, although other types of samples which contain T cells can be used. The sample may be<br>
added directly to the assay or may be processed first. Typically the processing may comprise diluting<br>
of the sample, for example with water or buffer. Typically the sample is diluted from 1.5 to 100 fold,<br>
for example 2 to 50 or 5 to 10 fold.	.<br>
The processing may comprise separation of components of the sample. Typically mononuclear cells (MCs) are separated,from the samples. The MCs will comprise the T cells and APCs. Thus in the method the APCs present in the separated MCs can present the peptide to the T cells. In another embodiment only T cells, such as only CD4 T cells, can be purified from the sample. PBMCs, MCs and T cells can ,be separated from the sample using techniques known in the art, such as those described in Lalvani et al (1997) J. Exp. Med. 186, p859-865.<br>
In one embodiment, the T cells used in the assay are in the form of unprocessed or diluted samples, or are freshly isolated T cells (such as;,in the form of freshly isolated MCs or PBMCs) which are used directly ex vivo, i.e. they are not cultured before being used in the method. Thus the T cells have not been restimulated in an antigen specific manner in vitro. However the T cells can be cultured before use, for example in the presence of one or more of the agents, and generally also exogenous growth promoting cytokines. During culturing the agent(s) are typically present on the surface of APCs, such as the APC used in the method. Pre-culturing of the T cells may lead to an<br><br>
increase in the sensitivity of the method. Thus the T cells can be converted into cell lines, such as short term cell lines (for example as described in Ota et al (1990) Nature 346, pi 83-187).<br>
The APC that is typically present in the method may be from the same individual as the T cell or from a different host. The APC may be a naturally occurring APC or an artificial APC. The APC is a cell that is capable of presenting the peptide to a T cell. It is typically a B cell, dendritic cell or macrophage. It is typically separated from the same sample as the T cell and is typically co-purified with the T cell. Thus the APC may be present in MCs or PBMCs. The APC is typically a freshly isolated ex vivo cell or a cultured cell. It may be in the form of a cell line, such as a short term or immortalised cell line. The APC may express empty MHC class II molecules on its surface.<br>
In the method one or more (different) agents may be used. Typically the T cells derived from the sample can be placed into an assay with all the agents which it is intended to test or the T cells can be divided and placed into separate assays each of which contain one or more of the agents.<br>
The invention also provides the agents such as two or more of any of the agents mentioned herein (e.g. the combinations of agents which are present in the composition agent discussed above) for simultaneous separate or sequential use (eg. for in vivo use).<br>
In one embodiment agent per se is added,directly to an assay comprising T cells and APCs. As discussed above the T cells and APCs in such an assay could be hi the form of MCs. When agents that can be recognised fay the T cell without the need for presentation by APCs are used then APCs are not required. Analogues which mimic the original (i) bound to a MHC molecule are an example of such an agent.<br>
In one embodiment the agent is provided to the APC in the absence of the T cell. The APC is then provided to the T cell, typically after being allowed to present the agent on its surface. The peptide may have been taken up inside the APC and presented, or simply be taken up onto the surface without entering inside the APC.<br>
The duration for which the agent is contacted with the T cells will vary depending on the method used for determining recognition of the peptide. Typically 105 to 107, preferably 5xl05 to 106 PBMCs are added to each assay. In the case where agent is added directly to the assay its concentration is from 10"1 to ] 03ug/ml, preferably 0.5 to 50ug/ml or 1 to lOug/ml.<br>
Typically the length of time for which the T-cells are incubated with the agent is from 4 to 24 hours, preferably 6 to 16 hours. When using ex vivo PBMCs it has been found that O.SxlO6 PBMCs can be incubated in 10|ig/ml of peptide for 12 hours at 37°C.<br>
The determination of the recognition of the agent by the T cells may be done by measuring<br>
the binding of the agent to theT cells (this can be carried out using any suitable binding assay format<br>
discussed herein). Typically T cells which bind the agent can be sorted based on this binding, for<br>
example using a FACS machine. The presence of T cells that recognise the agent will be deemed to<br>
. occur if the frequency of cells sorted using--the agent is above a "control" value. The frequency of<br><br>
antigen-experienced T cells is generally 1 in 106 to 1 in 103, and therefore whether or not the sorted cells are antigen-experienced T cells can be determined.<br>
The determination of the recognition of the agent by the T cells may be measured in vivo. Typically the agent is administered to the host and then a response which indicates recognition of the agent may be measured. The agent is typically administered intradermally or epidermally. The agent is typically administered by contacting with the outside of the skin, and may be retained at the she with the aid of a plaster or dressing. Alternatively the agent may be administered by needle, such as by injection, but can also be administered by other methods such as ballistics (e.g. the ballistics techniques which have been used to deliver nucleic acids). EP-A-0693119 describes techniques that can typically be used to administer the agent. Typically from 0.001 to 1000 ug, for example from 0.01 to 100 u.g or 0.1 to 10 u,g of agent is administered.<br>
In one embodiment a product can be administered which is capable of providing the agent in vivo. Thus a polynucleotide capable of expressing the agent can be administered, typically in any of the ways described above for the administration of the agent. The polymicleotide typically has any of the characteristics of the polynucleotide provided by the invention which is discussed below. The agent is expressed from the polynucleotide in vivo. Typically from 0.001 to 1000 u,g, for example from 0.01 to 100 ngorO.l to 10.fig of polynucleotide is administered.<br>
Recognition of the agent administered to the skin is typically indicated by the occurrence of inflammation (e.g. induration, erythema or oedema) at the site of administration. This is generally measured by visual examination of the site.<br>
The method of diagnosis based on the detection of an antibody that binds the agent is typically carried out by contacting a sample from the individual (such as any of the samples mentioned here, optionally processed in any manner mentioned herein) with the agent and determining whether an antibody in the sample binds the agent, such a binding indicating that the individual has, or is susceptible to coeliac disease. Any suitable format of binding assay may be used, such as any such format mentioned herein. , .<br>
Therapy<br>
The identification of the immunodominant epitope and other epitopes described herein allows therapeutic products to be made which target the T cells which recognise this epitope (such T cells being ones which participate hi the immune response against gluten proteins). These findings also allow the prevention or treatment of coeliac disease by suppressing (by tolerisation) an antibody or T cell response to the epitope(s).<br>
Certain agents of the invention bind the TCR that recognises the epitope of the invention (as measured using any of the binding assays discussed above) and cause tolerisation of the T cell that carries the TCR. Such agents, optionally in association with a carrier, can therefore be used to prevent or treat coeliac disease.<br><br>
Generally tolerisation can be caused by the same peptides which can (after being recognised by the TCR) cause antigen specific functional activity of the T cell (such as any such activity mentioned herein, e.g. secretion of cytokines). Such agents cause tolerisation when they are presented to the immune system in a 'tolerising' context.<br>
Tolerisation leads to a decrease in the recognition of a T cell or antibody epitope by the immune system. In the case of a T cell epitope this can be caused by the deletion or anergising of T cells that recognise the epitope. Thus T cell activity (for example as measured in suitable assays mentioned herein) hi response to the epitope is decreased. Tolerisation of an antibody response means that a decreased amount of specific antibody to the epitope is produced when the epitope is administered.<br>
Methods of presenting antigens to the immune system in such a context are known and are described for example in Yoshida et al. Clin. Immunol. Imrminopathol. 82, 207-215 (1997), Thurau et al. Clin. Exp. Immunol. 109, 370-6 (1997), and Weiner et al. Res. Imrmraol. 148, 528-33 (1997). In particular certain routes of administration can cause tolerisation, such as oral, nasal or intraperitoneal. Tolerisation may also be accomplished via dendritic cells and tetramers presenting peptide. Particular products which cause tolerisation may be administered (e.g. in a composition that also comprises the agent) to the individual. Such products include cytokines, such as cytokines that favour a Th2 response (e.g. TL-4, TGF-fl-or TL-10). Products or agent may be administered at a dose that causes tolerisation.<br>
The invention provides a protein that comprises a sequence able to act as an antagonist of the T cell (which T cell recognises the .agent). Such proteins and such antagonists can also be used to prevent or treat coeliac disease. The antagonist will cause a decrease in the T cell response. In one embodiment, the antagonist .binds the TCR of the. T cell (generally in the form of a complex with HLA-DQ2 or -DQ8) but instead of causing normal 'functional activation causing an abnormal signal to be passed through the TCR intraoellular signalling cascade, which causes the T cell to have decreased function activity (e.g. in response to recognition of an epitope, typically as measured by any suitable assay mentioned herein)..<br>
In one embodiment the antagonist competes with epitope to bind a component of MHC processing and presentation pathway, such as an MHC molecule (typically HLA-DQ2 or -DQ8). Thus the antagonist may bind HLA-DQ2 or -DQ8 (and thus be a peptide presented by this MHC molecule) or a homologue thereof. ,<br>
Methods of causing antagonism are known in the art. In one embodiment the antagonist is a homologue of the epitopes mentioned above and may have any of the sequence, binding or other properties of the agent (particularly analogues). The antagonists typically differ from any of the above epitopes (which are capable of causing a normal antigen specific function in the T cell) by 1, 2, 3, 4 or more mutations (each of which may be a substitution,, insertion or deletion). Such<br><br>
antagonists are termed "altered peptide ligands" or "APL" in the art. The mutations are typically at the ammo acid positions that contact the TCR.<br>
For example, the antagonist may differ from the epitope by a substitution within the sequence that is equivalent to the sequence represented by amino acids 64 to 67 of A-gliadin (the sequence of A-gliadin is given in Figure 10). Thus preferably the antagonist has a substitution at the equivalent of position 64, 65 or 67. Preferably the substitution is 64W, 67W, 67M or 65T.<br>
Since the T cell immune -response to the epitope of the invention in an individual is polyclonal, more than one antagonist may need to be administered to cause antagonism of T cells of the response which have different TCRs. Therefore the .antagonists may be administered in a composition which comprises at least 2, 4, 6 or more different antagonists, which each antagonise different T cells.<br>
The invention also provides a method of identifying an antagonist of a T cell (which recognises the agent), comprising contacting a candidate substance with the T cell and detecting whether the substance causes a decrease in the ability of the T cell to undergo an antigen specific response (e.g. using any suitable assay mentioned herein), the detecting of any such decrease in said ability indicating that the substance is an antagonist.<br>
In one embodiment, the antagonists (including combinations of antagonists to a particular epitope) ortolerising (T cell and antibody tolerising) agents are present hi a composition comprising at least 2, 4, 6 or more antagonists or agents which antagonise or tolerise to different epitopes of the invention, for example to the combinations of epitopes discussed above in relation to the agents which are a product comprising more than one substance.<br>
Testing whether a composition is capable of causing coeliac disease<br>
As mentioned above the invention provides a method of determining whether a composition is capable of causing coeliac disease comprising detecting the presence of a protein sequence which is capable of being modified by a transghrtaminase to as sequence comprising the agent or epitope of the invention (such transglutaminase activity may be a human intestinal transglutaminase activity). Typically this is performed by using a binding assay in which a moiety which binds to the sequence in a specific manner is contacted with the composition and the formation of sequence/moiety complex is detected and used to ascertain the presence of the agent. Such a moiety may be any suitable substance (or type of substance) mentioned herein, and is typically a specific antibody. Any ' suitable format of binding assay can be used (such as those mentioned herein).<br>
In one embodiment, the composition is contacted with at least 2, 5, 10 or more antibodies which are specific for epitopes of the invention from different gluten proteins, for example a panel of antibodies capable of recognising the combinations of epitopes discussed above in relation to agents of the invention which are a product comprising more than one substance.<br>
The composition typically comprises material from a plant that expresses a gluten protein which is capable of causing coeliac disease [for example any of the gluten proteins or plants<br><br>
mentioned herein). Such material may be a plant part, such as a harvested product (e.g. seed). The<br>
material may be processed products of the plant material (e.g. any such product mentioned herein),<br>
such as a flour or food that comprises the gluten protein. The processing of food material and testing<br>
in suitable binding assays is routine, for example as mentioned in Kricka LJ, J. Biolumin.<br>
Chemilumin. 13,189-93 (1998).	'<br>
Binding assays<br>
The determination of binding between any two substances mentioned herein may be done by measuring a characteristic of either or both substances that changes upon binding, such as a spectroscopic change.<br>
The binding assay format may be a 'band shift' system. This involves determining whether the presence of one substance (such as a candidate substance) advances or retards the progress of the other substance during gel electrophoresis.<br>
The format may be a competitive binding method which determines whether the one substance is able to inhibit the binding of the other substance to an agent which is known to bind the other substance, such as a specific antibody.<br>
Mutant gluten proteins<br>
The invention provides a gluten protein in which an epitope sequence of the invention, or sequence which can be modified by a transglutaminase to provide such a sequence has been mutated so that it no longer causes, or is recognised by, a T cell response that recognises the epitope. In this context the term recognition refers to the TCR binding the epitope hi such a way that normal (not antagonistic) antigen-specific functional activity of the T cell occurs.<br>
Methods of identifying equivalent epitopes in other gluten proteins are discussed above. The wild type of the mutated gluten protein is one which causes coeliac disease. Such a mutated gluten protein may have homology with the wild type of the mutated gluten protein, for example to the degree mentioned above (in relation to the analogue) across all of its sequence or across 15, 30, 60, 100 or 200 contiguous amhio acids of its sequence. The sequences of other natural gluten proteins are known in the art.<br>
The mutated gluten protein will not cause coeliac disease or will cause decreased symptoms of coeliac disease. Typically the mutation decreases the ability of the epitope to induce a T cell response. The mutated epitope may have a decreased binding to HLA-DQ2 or -DQ8, a decreased ability to be presented by an APC or a decreased ability to bind to or to be recognised (i.e. cause antigen-specific functional activity) by T cells thatrecognise the agent. The mutated gluten protein or epitope will therefore show no or reduced recognition in any of the assays mentioned herein in relation to the diagnostic aspects of the invention..<br>
The mutation may be.one or more deletions, additions or substitutions of length 1 to 3,4 to 6, 6 to 10,11 to 15 or more in the epitope, for example across any of SEQ ID NOS: 1-1927; or across<br><br>
equivalents thereof. Preferably the mutant gluten protein has at least one mutation in the sequence of any of SEQ ID NO: 1-1927. A preferred mutation is at the position equivalent to position 65 in A-gliadin (see Figure 10). Preferably, a naturally occurring glutamine is substituted to histidine, tyrosine, tryptophan, lysine, proline, or arginine.<br>
The invention thus also provides use of a mutation (such any of the mutations in any of the sequences discussed herein) in an epitope of a gluten protein, which epitope is an epitope of the invention, to decrease the ability of the gluten protein to cause coeliac disease.<br>
In one embodiment the mutated sequence is able to act as an antagonist. Thus the invention provides a protein that comprises a sequence which is able to bind to a T cell receptor, which T cell receptor recognises an agent of the invention, and which sequence is able to cause antagonism of a T cell that carries such a T cell receptor.<br>
The invention also provides proteins which are fragments of the above mutant gluten proteins, which are at least 7 amino acids long (e.g. at least 8= 9, 10, 11, 12, 13, 14, 15, 30, 60, 100, 150, 200, or 250 amino acids long) and which comprise the mutations discussed above which decrease the ability of the gluten protein to be recognised. Any of the mutant proteins (including fragments) mentioned herein may also be present in the form of fusion proteins, for example with other gluten proteins or with non-gluten proteins.<br>
The equivalent wild type protein to the mutated gluten protein is typically from a graminaceous monocotyledon, :such as a plant of a genus selected from Triticum, Secale, Hordeian, Triticale or Avena, (e.g. wheat, rye, barley, oats or triticale). For example, the protein may be an a, a(5, p, 7 or co gliadin or an avenin, , .<br>
Sits	/_'••'.',;' _    .	'<br>
The invention also provides a kit for carrying out the method comprising one or more agents and optionally a means to detect the recognition of the agent by the T cell. Typically the different agents are provided for simultaneous, separate or sequential use. Typically the means to detect recognition allows or aids detection based on the techniques discussed above.<br>
Thus the means may allow detection'of a substance secreted by the T cells after recognition. The kit may thus additionally include a specific binding moiety for the substance, such as an antibody. The moiety is typically specific for IFN-y. The moiety is typically immobilised on a solid support. This means that after binding the moiety the substance will remain in the vicinity of the T cell which secreted it. Thus "spots" of substance/moiety complex are formed on the support, each spot representing a T cell which is secreting the substance. Quantifying the spots, and typically comparing against a control, allows determination of recognition of the agent.<br>
The kit may also comprise a means to detect the substance/moiety complex. A detectable change may occur in the moiety itself after binding the substance, such as a colour change. Alternatively a second moiety directly or indirectly labelled for detection may be allowed to bind 1he substance/moiety complex to allow the determination of the spots. As discussed above the second<br><br>
moiety may be specific for the substance, but binds a different site on the substance than the first moiety.<br>
The immobilised support may be a plate with wells, such as a microtitre plate. Each assay can therefore be carried out in a separate well in the plate.<br>
The kit may additionally comprise medium for the T cells, detection moieties or washing buffers to be used in the detection steps. The kit may additionally comprise reagents suitable for the separation from the sample, such as the separation of PBMCs or T cells from the sample. The kit may be designed to allow detection of the T cells directly in the sample without requiring any separation of the components of the sample.<br>
The kit may comprise an instrument which allows administration of the agent, such as intradermal or epidermal administration. Typically such an instrument comprises plaster, dressing or one or more needles. The instrument may allow ballistic delivery of the agent. The agent in the kit maybe in the form of a pharmaceutical composition.<br>
The'kit may also comprise controls, such as positive or negative controls. The positive control may allow the detection system to be tested. Thus the positive control typically mimics recognition of the agent in any of the above methods. Typically in the kits designed to determine recognition in vitro the positive control is a cytokine. In the kit designed to detect in vivo recognition of the agent the positive control may be antigen to which most individuals should response.<br>
The kit may also comprise a means to take a sample containing T cells .from the host, such as<br>
a blood sample. The kit may comprise a means to separate mononuclear cells or T cells from a<br>
sample from the host.	.<br>
Polynucleotides, cells, transgenic mammals and antibodies<br>
The invention also provides a polynucleotide which is capable of expression to provide the agent or mutant gluten proteins. Typically the polynucleotide is DNA or KNA, and is single or double stranded. The polynucleotide will preferably comprise at least 50 bases or base pairs, for example 50 to 100, 100 to 500, 500 to 1000 or 1000 to 2000 or more bases or base pairs. The polynucleotide therefore comprises a sequence which encodes the sequence of any of SEQ ID NO: 1-1927 or any of the other agents mentioned herein. To the 5' and 3' of this coding sequence the polynucleotide of the invention has sequence or codons which are different from the sequence or codons 5' and 3' to these sequences in the corresponding gluten protein gene.<br>
5' and/or 3' to the sequence encoding the peptide the polynucleotide has coding or non-coding sequence. Sequence 5' and/or 3" to the coding sequence may comprise sequences which aid expression, such as transcription and/or translation, of the sequence encoding the agent. The polynucleotide may be capable of expressing the agent prokaryotic or eukaryotic cell. In one embodiment the polynucleotide is capable of expressing the agent in a mammalian cell, such as a human, primate or rodent (e.g. mouse or rat) cell.<br><br>
A polynucieotide of the invention may hybridise selectively to a polynucleotide that encodes a gluten protein from which the agent is derived at a level significantly above background. Selective hybridisation is typically achieved using conditions of medium to high stringency (for example 0.03M sodium chloride and 0.03M sodium citrate at from about'50°C to about 60°C). However, such hybridisation may be carried out under any suitable conditions known in the art (see Sambrook et al (1989), Molecular Cloning: A Laboratory Manual). For example, if high stringency is required, suitable conditions include 0.2 x SSC at 60°C. If lower stringency is required, suitable conditions include 2 x SSC at 60°C.<br>
Agents or proteins of the invention may be encoded by the polynucleotides described herein.<br>
The polynucleotide may form or be incorporated into a replicable vector. Such a vector is able to replicate in a suitable cell. The vector may be an expression vector. In such a vector the polynucleotide of the invention is operably linked to a control sequence which is capable of providing for the expression of the polynucleotide. The vector may contain a selectable marker, such as the ampicillin resistance gene.<br>
The polynucleotide or vector may be present in a cell. Such a cell may have been transformed by the polynucleotide or vector. The cell may express the agent. The cell will be chosen to be compatible with the said vector' and may for example be a prokaryotic (bacterial), yeast, insect or mammalian cell. The polynucleotide or vector may be introduced into host cells using conventional techniques including calcium phosphate precipitation, DEAE-dextran transfection, or electroporation.<br>
The invention provides processes for the production of the proteins of the invention by<br>
recombinant means. This may comprise (a) cultivating a transformed cell as defined above under<br>
conditions that allow the expression of the protein; and preferably (b) recovering the expressed<br>
polypeptide. Optionally, the polypeptide may be isolated and/or purified, by techniques known in the<br>
art.	••."' :.-;....<br>
The invention also provides TCRs which recognise (or bind) the agent, or fragments thereof<br>
which are capable of such recognition (or binding). These can be present in the any form mentioned<br>
herein (e.g. purity) discussed herein in relation to the protein of the invention. The invention also<br>
provides T cells which express such TCRs which can be present in any form (e.g. purity) discussed<br>
herein for the cells of the invention.	,<br>
The invention also provides monoclonal or polyclonal antibodies which specifically recognise the agents (such as any of the epitopes of the invention) and which recognise the mutant gluten proteins (and typically which do not recognise the equivalent wild-type gluten proteins) of the invention, and methods of making such antibodies. Antibodies of the invention bind specifically to these substances of the invention.<br>
For the purposes of this invention, the term "antibody" includes antibody fragments such as Fvs F(ab) and F(ab')2 fragments, as well as single-chain antibodies.<br><br>
A method for producing a polyclonal antibody comprises immunising a suitable host animal, for example an experimental animal, with the immunogen and isolating immunoglobulins from the serum. The animal may therefore be inoculated with the immunogen, blood subsequently removed from the animal and the IgG fraction purified. A method for producing a monoclonal antibody comprises immortalising cells which produce the desired antibody. Hybridoma cells may be produced by fusing spleen cells from an inoculated experimental animal with tumour cells (Kohler and Milstein (1975) Nature 256, 495-497).<br>
An immortalized cell producing the desired antibody may be selected by a conventional procedure. The hybridomas may be grown in culture or injected intraperitoneally for formation of ascites fluid or into the blood stream of an allogenic host or immunocompromised host. Human antibody may be prepared by in vitro immunisation of human lymphocytes, followed by transformation of the lymphocytes with Epstein-Barr virus.<br>
For the production of both monoclonal and potyclonal antibodies, the experimental animal is suitably a goat, rabbit, rat or mouse. If desired, the immunogen may be administered as a conjugate in which the immunogen is coupled, for example via a side chain of one of the amino acid residues, to a suitable carrier. The carrier molecule is typically a physiologically acceptable carrier. The antibody obtained may be isolated and, if desired, 'purified.<br>
The polynucleotide, agent, protein or antibody of the invention, may carry a detectable label. Detectable labels which allow detection of,the secreted substance by visual inspection, optionally with the aid of an optical magnifying means,.are preferred. Such a system is typically based on an enzyme label which causes colour change in a substrate, for example alkaline phosphatase causing a colour change in a substrate. Such substrates are commercially available, e.g. from BioRad. Other suitable labels include other enzymes such as peroxidase, or protein labels, such as biotin; or radioisotopes, such as 32P or 35S. The above labels may be detected using known techniques.<br>
Polynucleotides, agents, proteins, antibodies or cells of the invention may be in substantially purified form. They may be in substantially isolated form, in which case they will generally comprise at least 80% e.g. at least 90, 95, 97 or 99% of the polynucleotide, peptide, antibody, cells or dry mass in the preparation. The polynucleotide, 'agent, protein or antibody is typically substantially free of other cellular components. The polynucleotide, agent, protein or antibody may be used in such a substantially isolated, purified or free form in the method or be present in such forms in the kit<br>
The invention also provides-a transgenic non-human mammal which expresses a TCR of the invention. This may be any ofthe mammals discussed herein (e.g. in relation to the production of the antibody). Preferably the mammal has, or is susceptible, to coeliac disease. The mammal may also express HLA-DQ2 or -DQ8 or HLA-DE3-DQ2 and/or may be given a diet comprising a gluten protein which causes coeliac disease (e.g. any ofthe gluten proteins mentioned herein). Thus the mammal may act as an animal model for coeliac disease.<br><br>
The invention also provides a method of identifying a product which is therapeutic for coeliac disease comprising administering a candidate substance to a mammal of the invention which has, or which is susceptible to, coeliac disease and determining whether substance prevents or treats coeliac disease in the mammal, the prevention or treatment of coeliac disease indicating that the substance is a therapeutic product. Such a product may be used to treat or prevent coeliac disease.<br>
The invention provides therapeutic (including prophylactic) agents or diagnostic substances (the agents, proteins and porynucleotides of the invention). These substances are formulated for clinical administration by mixing them with a pharmaceutically acceptable carrier or diluent. For example they can be formulated for topical, parenteraL, intravenous, intramuscular, subcutaneous, intraocular, intradermal, epidermal or transdermal administration. The substances may be mixed with any vehicle which is pharmaceutically acceptable and appropriate for the desired route of administration. The pharmaceutically carrier or diluent for injection may be, for example, a sterile or isotonic solution such as Water for Injection or physiological saline, or a carrier particle for ballistic delivery.<br>
The dose of the substances may be adjusted according to various parameters, especially according to the agent used; the age, weight and condition of the patient to be treated; the mode of administration used; the severity of the condition to be treated; and the required clinical regimen. As a guide, the amount of substance administered by injection is suitably from 0.01 rag/kg to 30 mg/kg, preferably from 0.1 mg/kg to 10 mg/kg.<br>
The routes of administration and dosages described are intended only as a guide since a skilled practitioner will be able to determine readily the optimum route of administration and dosage for any particular patient and. condition.<br>
The substances of the invention may thus be used in a method of treatment of the human or animal body, or in a diagnostic method practised on,the human body. In particular they may be used in a method of treating or preventing coeliac disease. The invention also provide the agents for use in a method of manufacture of a medicament for treating or preventing coeliac disease. Thus the invention provides a method of preventing or treating coeliac disease comprising administering to a human in need thereof a substance of the invention (typically a non-toxic effective amount thereof).<br>
The agent of the invention can be made using standard synthetic chemistry techniques, such as by use of an automated synthesizer. The agent may be made from a longer polypeptide e.g. a fusion protein, which polypeptide typically comprises the sequence of the peptide. The peptide may be derived from the polypeptide by for example hydrolysing the polypeptide, such as using a protease; or by physically breaking the polypeptide. The polynucleoti.de of the invention can be made using standard techniques, such as by using a synthesiser.<br><br>
Plant cells and plants that express mutant gluten proteins or express proteins comprising sequences which can act as antagonists<br>
The cell of the invention may be a plant cell, such as a cell of a graminaceous monocotyledonous species. The species may be one whose wild-type form expresses gluten proteins, such as any of the gluten proteins mentioned herein. Such a gluten protein may cause coeliac disease in humans. The cell may be of wheat, maize, oats, rye, rice, barley, triticale, sorghum, or sugar cane. Typically the cell is of the Triticum genus, such as aestivum, spelta, polonicum or monococcum.<br>
The plant ceil of the invention is typically one which does not express one or more wild-type gluten proteins (such as any of the gluten proteins mentioned herein which may cause coeliac disease), or one which does not express one or more gluten proteins comprising a sequence that can be recognised by a T cell that recognises the agent. Thus if the wild-type plant cell did express such a gluten protein then it may be engineered to prevent or reduce the expression of such a gluten protein or to change the amino acid sequence of the gluten protein so that it no longer causes coeliac disease (typically by no longer expressing the epitope of the invention).<br>
This can be done for example by introducing mutations into 1, 2, 3 or more or all of such gluten protein genes in the cell, for example into coding or non-coding (e.g. promoter regions). Such mutations can be any of the type or length of mutations discussed herein (e.g., in relation to homologous proteins). The mutations can be introduced in a directed manner (e.g., using site directed mutagenesis or homologous recombination techniques) or in a random manner (e.g. using a imitagen, and then typically selecting for mutagenised cells which no longer express the gluten protein (or a gluten protein sequence which causes coeliac .disease)).<br>
In the case of plants or plant cells that express a protein that comprises a sequence able to act as an antagonist such a plant or plant cell may express a wild-type gluten protein (e.g. one which causes coeliac disease). Preferably though .the presence of the, antagonist sequence will cause reduced coeliac disease symptoms (such as no symptoms) in an individual who ingests a food comprising protein from the plant or plant cell.<br>
The polynucleotide which is present in (or which was transformed into) the plant cell will generally comprise promoter capable of expressing, the mutant gluten protein the plant cell. Depending on the pattern of expression desired, the promoter may be constitutive, tissue- or stage-specific; and/or inducible. For example, strong constitutive expression in plants can be obtained with the CAMV 35S, Rubisco ssu, or histone promoters: Also, tissue-specific or stage-specific promoters may be used to target expression of protein of the invention to particular tissues in a transgenic plant or to particular stages in its development. Thus, for example seed-specific, root-specific, leaf-specific, flower-specific etc promoters may be used. Seed-specific promoters include those described byDalta et al (Biotechnology Ann. Rev. (1997), 3, pp:269-296). Particular examples of seed-specific promoters are napin promoters (EP-A-0 255, 378), phaseolin promoters, glutenine promoters.<br><br>
helianthenine promoters (W092/17580), albumin promoters (WO98/45460), oleosin promoters (WO98/45461) and ATS1 and ATS3 promoters (WO99/20775).<br>
The cell may be in any form. For example, it may be an isolated cell, e.g. a protoplast, or it may be part of a plant tissue, e.g. a callus, or a tissue excised from a plant, or it may be part of a whole plant. The cell may be of any type (e.g. of any type of plant part). For example, an undifferentiated cell, such as a callus cell; or a differentiated cell, such as a cell of a type found in embryos, pollen, roots, shoots or.leaves. Plant parts include roots; shoots; leaves; and parts involved in reproduction, such as pollen, ova, stamens, anthers, petals, sepals and other flower parts.<br>
The invention provides a method of obtaining a transgenic plant cell comprising transforming a plant cell with a polynucleotide or vector of the invention to give a transgenic plant cell. Any suitable transformation method may be used (in the case of wheat the techniques disclosed in Vasil V et al, Biotechnology 10, 667-674 (1992) may be used). Preferred transformation techniques include electroporation of plant protoplasts and particle bombardment. Transformation may thus give rise to a chimeric tissue or plant in which some cells are transgenic and some are not.<br>
The cell of the invention or thus obtained cell may be regenerated into a transgenic plant by<br>
techniques known in the art. "These may involve the use of plant growth substances such as auxins,<br>
giberellins and/or cytokinins to stimulate the growth and/or division of the transgenic cell. Similarly,<br>
techniques such as somatic embryogenesis .and meristem culture may be used. Regeneration<br>
techniques are well known in the art and examples can be found in, e.g. US 4,459,355, US 4,536,475,<br>
US 5,464,763, US.5, 177,010, US 5,187,073, EP.267,159, EP 604, 662, EP 672, 752, US 4,945,050,<br>
US 5,036,006, US 5,100,792, US 5,371,014, US 5,478,744, US 5,179,022, US 5,565,346, US<br>
5,484,956, US 5,508,468, US 5,53.8,877, US 5,554,798,^8 5,489,520, US 5,510,318, US 5,204,253,<br>
US 5,405,765, EP 442,174, EP 486,233, EP 486,234,.EP 539,563, EP 674,725, WO91/02071 and<br>
WO95/06128.	.	..'••-'...,...<br>
In many such techniques, one step is the formation of a callus, i.e. a plant tissue comprising expanding and/or dividing cells. Such calliare a further aspect'of the invention as are other types of plant cell cultures and plant parts. Thus, for example, the invention provides transgenic plant tissues and parts, including embryos, meristems, seeds, shoots, roots, stems, leaves and flower parts. These may be chimeric in the sense that some of their, cells are cells of the invention and some are not. Transgenic plant parts and tissues, plants and seeds of the invention may be of any of the plant species mentioned herein.<br>
Regeneration procedures will typically involve the selection of transformed cells by means of marker genes.<br>
The regeneration step gives rise to ia first generation transgenic plant. The invention also provides methods of obtaining transgenic plants of further generations from this first generation plant. These are known as progeny transgenic plants. Progeny plants of second, third, fourth, fifth,<br><br>
sixth and further generations may be obtained from the first generation transgenic plant by any means known in the art.<br>
Thus, the invention provides a method of obtaining a transgenic progeny plant comprising obtaining a second-generation transgenic progeny plant from a first-generation transgenic plant of the invention, and optionally obtaining transgenic plants of one or more further generations from the second-generation progeny plant thus obtained.<br>
Progeny plants may be produced from their predecessors of earlier generations by any known technique. In particular, progeny plants may be produced by: (a) obtaining a transgenic seed from a transgenic plant of the invention belonging to a previous generation, then obtaining a transgenic progeny plant of the invention belonging to a new generation by growing up the transgenic seed; and/or (b) propagating clonally a transgenic pknt of the invention belonging to a previous generation to give a transgenic progeny plant of the invention belonging to a new generation; and/or (c) crossing a first-generation transgenic plant of the invention belonging to a previous generation with another compatible plant to give a transgenic progeny plant of the invention belonging to a new generation; and optionally (d) obtaining transgenic progeny plants of one or more further generations from the progeny plant thus obtained.<br>
These techniques may be used in any combination. For example, clonal propagation and sexual propagation may be used at different points in a process that gives rise to a transgenic plant suitable for cultivation. In particular, repetitive back-crossing with a plant taxon with agronomically desirable characteristics may be undertaken. Further steps of removing cells from a plant and regenerating new plants therefrom may also be carried out<br>
Also, further desirable characteristics may be introduced by transforming the cells, plant tissues, plants or seeds, at any suitable stage in the above process, to introduce desirable coding sequences other than the polynucleotides of the invention. This may be carried out by the techniques described herein for the introduction of polynucleotides of the invention.<br>
For example, further transgenes may be selected from those coding for other herbicide resistance traits, e.g. tolerance to: Glyphosate (e.g. using an EPSP synthase gene (e.g. EP-A-0,293,358) or a glyphosate oxidoreductase (WO 92/00377) gene); or tolerance to fosametin; a dihalobenzonitrile; glufosinate, e.g. using a phosphinothrycin acetyl transferase (PAT) or glutamine synthase gene (cf. EP-A-0,242,236); asulam, e.g. using a dihydropteroate synthase gene (EP-A-0,369,367); or a sulphonylurea, e.g. using an ALS gene); diphenyl ethers such as acifluorfen or oxyfluorfen, e.g. using a protoporphyrogen oxidase gene); an oxadiazole such as oxadiazon; a cyclic imide such as chlorophthalim; a phenyl pyrazole such as TNP, or a phenopylate or carbamate analogue thereof.<br>
Similarly, genes for beneficial properties other than herbicide tolerance may be introduced. For example, genes for insect resistance may be introduced, notably genes encoding Bacillus<br><br>
thuringiensis (Bf) toxins. Likewise, genes for disease resistance may be introduced, e.g. as in WO91/02701 or W095/06128.<br>
Typically, a protein of the invention is expressed in a plant of the invention. Depending on the promoter used, this expression may be constitutive or inducible. Similarly, it may be tissue- or stage-specific, i.e. directed towards a particular plant tissue (such as any of the tissues mentioned herein) or stage in plant development<br>
The invention also provides methods of obtaining crop products by harvesting., and optionally processing further, transgenic plants of the invention. By crop product is meant any useful product obtainable from a crop plant.<br>
Products that contain mutant gluten proteins or proteins that comprise sequence capable of acting as an antagonist<br>
The invention provides a product that comprises the mutant gluten proteins or protein that comprises sequence capable of acting as an antagonist. This is typically derived from or comprise plant parts from plants mentioned herein which express such proteins. Such a product may be obtainable directly by harvesting or indirectly, by harvesting and further processing the plant of the invention. Directly obtainable products include grains. Alternatively, such a product may be obtainable indirectly, by harvesting and further processing. Examples of products obtainable by further processing are flour or distilled alcoholic beverages; food products made from directly obtained or further processed material, e.g. baked products (e.g. bread) made from flour. Typically such food products, which are ingestible and digestible (i.e. non-toxic and of nutrient value) by human individuals.<br>
In the case of food products that comprise the protein which comprises an antagonist<br>
sequence the food product may also comprise the wild-type gluten protein, but preferably the<br>
antagonist is able to cause a reduction (e.g. completely) in the coeliac disease symptoms after such<br>
food is ingested.	•,.'.••••<br>
Deamidation<br>
Where a sequence described herein includes a Gin residue, the invention also provides that sequence where the Gin residue has been deamidated to a Glu residue. One or more (e.g., 1, 2, 3, 4, 5, etc.) Gin residue(s) per sequence may be deamidated, but when there is more than one Gin residue, not all of them must be deamidated. Preferably, the Gin residues that are deamidated are those susceptible to deamidation by transglutaminase.<br>
Examples where Gin may be deamidated are given in the sequence listing. For example., residue 4 of SEQ ID KO:1 can be a Gin residue or a Glu residue, residue 6 of SEQ ID NO:2 can be a Gin residue or a Glu residue, residues 4 and 7 of SEQ ID NO:6 can each independently be Gin or Glu residues, etc. The Gin residues that are susceptible to deamidation, and their deamidated Glu counterparts, are referred to as "Glx" residues. ;<br><br>
Where the agent includes more than one Glx residue, these may-be arranged in any configuration. For example, the Glx residues may be consecutive residues, and/or may be separated by one or more (e.g., 1, 2, 3, 4. 5, 6, 7, 8, etc.} other residues. As mentioned above, for HLA-DQ8 epitopes, the agent preferably comprises a Glx residue that is separated by seven residues from another Glx residue.<br>
Preferred agents of the invention are deamidated agents, i.e., the agent comprises the one or more Glx residues in the Glu form. This can be achieved hi various ways, e.g., by including Glu residues during production, or by converting Gin residues to Glu by dearnidation. Conversion of Gin to Glu can be achieved by treating an agent that contains Gin residues that are susceptible to dearnidation with a deamidating agent The one or more Gin residues are preferably deamidated to Glu by transglutaminase, for example as described in the examples.<br>
The skilled person will be able to determine which particular Gin residues in the agent are susceptible to deamidation and thus which residues should be Glu residues arising from dearnidation of a Gin residue. For example, Gln-contauirng sequences susceptible to deamidation by transglutaminase generally conform to a motif: e.g., QXPX, QXPF (Y), QXX(FYMILW), QXPF, QXX (FY), PQ(QL)P(FY)P. For example, the sequence PQ(QL)P(FY)P facilitates deamidation of the underlined Q at position 2 by transglutaminase.<br>
In particular, agents comprising the deamidated versions of SBQ ID NOs: 1-1927 are preferred (where such sequences are not already deamidated). Most preferably, the agents of the invention comprise the transglutaminase-deamidated versions of SEQ ID NOs: 1-1927 (again, where not already deamidated). Analogues and equivalents of these agents, as defined herein, are also encompassed within the scope of the invention.<br>
EXAMPLES<br>
The invention is illustrated by the following nonlimrting Examples:<br>
Initial gliadin epitope screening library<br>
In initial experiments involving 29 HLA-DQ2+ individuals with coeliac disease on long-term gluten free diet, interferon-gammaELISPOT assays were used to screen a previous Pepset (described in WO 03/104273, which is incorporated herein by reference) initially as pools of peptides and then ha 15 subjects as individual peptides with and without deamidation by tTG. This Pepset library consisted of 652 20mer gliadin peptides spanning all unique 12mers contained within all Genbank entries described as wheat gliadins found in September 2001. This Pepset library was designed "manually" from gene-derived protein sequences aligned using ClustalW software (MegAHgn) arranged into phylogenetic groupings .<br><br>
Approximately 0.6 micromole of each of 652 of the 20mers was provided. Two marker 20mer peptides were included in each set of 96 (VLQQHNIAHGSSQVLQESTY - peptide 161, and IKDFKVYFRESRDALWKGPG) and were characterized by reverse phase-HPLC and amino acid sequence analysis. Average purities of these marker peptides were 19% and 50%, respectively. Peptides were initially dissolved in acetonitrile (10%) and Hepes lOOmM to lOmg/rnl. The final concentration of individual peptides incubated with PBMC for the IFNy ELISpot assays was 20 meg/ml. These peptides were deamidated by incubation with guinea pig tissue tTG (Sigma T5398) in the ratio 100-.32 meg/ml for two hours at 37°C. Peptides solutions were stored at-20°C and freshly thawed prior to use. These studies were conducted in Oxford, UK. ELIspot assays were performed as described for those conducted in Melbourne, Australia (all other studies described herein). "Oxford" data regarding subject responses to Individual peptides was pooled with "Melbourne" data for subsequent "minimal" epitope analysis in the "EM algorithm" (see below).<br>
Second round gliadin epitope screening library<br>
A second round gliadin epitope library was designed according the bioactive sequences identified from the initial gliadin epitope screening library of 652 20mers. Gliadin 20mers with mean bioactivity equivalent to &gt;5% of the most potent gliadin 20mer (91: PQPFPPQLPYPQPQLPYPQP) in 15 HLA-DQ2+ subjects assessed with all 652 deamidated 20mers were defined. Since earlier studies (see WO 03/104273) indicated that deamidated pools of this Pepset were more potent than without deamidation, glutamine residues within bioactive 20mers potentially deamidated by tTG were identified according to the motif QXPX, QXZ (FYWILVM) where X is any amino acid except pioline, and P is praline, Z is any amino acid, and FYWILVM represent hydrophobic amino acids (consistent with the motifs for tTG-mediated deamidation published by Vader W. et al J Exp Med 2002 J. Exp. Med. 195:643-649, PCT WO 03/066079, and Fleckenstein B. 2002. J Biol Chem 277:34109-16).<br>
12mer peptides were then identified in which each potential deamidation site could be in position 4, 6 or 7 hi the 9mer located within HLA-DQ2 binding groove (HLA-DQ2 anchors at these positions show a preference for glutamate). Candidate 12mer core epitope sequences were then flanked with glycine followed by the N-terminal residue present in the parent gliadin polypeptide and at the C-termhial by the C-terminal residue present in the parent gliadin polypeptide followed by glycine (i.e. GXXXXXXXQXXXXXXG).<br>
Peptides were synthesised with glutamine or glutamate in position 9. Peptides (1 OOmcg/ml) (+/- deamidation by tTG) were then assessed in interferon gamma ELISPOT assays using PBMC from 15 HLA-DQ2+ coeliac volunteers after gluten challenge. Results of these assays were analysed according to the EM algorithm (see below). In addition, the most potent distinct peptides were synthesised and purified to &gt;80% (Mimotopes) and assessed in interferon gamma ELISPOT assays using PBMC from 15 HLA-DQ2+ coeliac volunteers after wheat gluten challenge.<br><br>
Complete gluten epitope screening libraty<br>
To make practical the design of a substantially larger peptide library spanning all wheat gliadin and glutenin, rye, barley, and oat gluten-like proteins (prolamines), and to confirm data from the previous gliadin peptide library,, an iterative algorithm was developed to automate design of a minimal set of 20mers including all unique 12mers (excluding signal peptide sequences) in gluten proteins. The ScanSet algorithm is shown in Figure 1.<br>
The method tests for all possible peptide epitopes from a group of proteins whether they are potential antigens in a range of patients. T-cell epitopes range in size between 9 and 15 AA. To test all possible 12mers in a set of proteins, becomes quickly unfeasible because of the high numbers.<br>
Here we use the fact that, for example, a 20mer peptide can cover up to 9 different 12mers. We therefore developed a combinatorial approach to .cover all possible 12mers represented in a family of proteins.<br>
20 ammo acid (20mer) long peptides are generated that are tested as antigens, and that cover all 12mer peptide sequences that exist in the group of proteins. We define the length of peptides to generate as L (e.g. 20) and the length of the epitopes we want to cover as S. We developed a computer program that generates all uniquely occurring Zmers from a set of proteins. Further, we generate all uniquely occurring iSmers from this set of proteins. Next we select a set of JVZrners that contains all sequences of imers. Figure 1 outlines,how this algorithm works.<br>
On 16 June 2003, Genbank contained accession numbers for 53 alpha/beta, 53 gamma and 2<br>
omega gliadins, and 77 LMW and 55 HMW glutenins from 1. aestivum, 59 hordeins, 14 secalins,<br>
and 20 avenins (see Figure 2). In total, ScanSet identified 18117 unique 12mers contained in the 225<br>
gluten gene products.	"': , •<br>
All unique gluten 12mers could be subsumed hi 2922 20mers. These 20mers were<br>
synthesised in a Pepset peptide library (Mimotopes Inc., Melbourne, Australia). Pepset peptides were<br>
synthesized in batches of 96 (Mimotopes Inc., Melbourne Australia). Approximately 0.7 to 1.3<br>
micromole of each of 2922 20mers was provided. Two marker 20mer peptides were included in each<br>
set of 96 (one representative peptide from the 94 other peptides on each particular plate, and<br>
IKDFHVYFRESRDALWKGPG) and were characterized by reverse phase-HPLC and mass<br>
spectroscopy. Average purities of these marker peptides were 36% (range: 5-68%) and 64%<br>
(range:55-71%), respectively.	'<br>
Peptides were initially dissolved in aqueous acetonitrile (50%). Peptides in aqueous acetontrile were transferred to sterile 96-well plates and diluted in sterile PBS with ImM calcium (250 meg/ml) and then incubated with tTG.(25 meg/ml) (Sigma T5398) for 6h 37°C and then stored frozen (-20°C) until use. • .<br>
Subjects all had biopsy-proven coeliac disease and had followed a strict gluten free diet for at least 6 months. All subjects possessed HLA-DQB01*02 (HLA-DQ2) alone (n=100) or HLA-DQA1*03 and HLA-DQB1*0302 (HLA-DQ8) alone (n=5). In all cases, tTG-IgA was assessed before<br><br>
gluten challenge and was in the normal range (30% of initial volunteers were found to have elevated tTG-IgA and were excluded since chronic gluten exposure is associated with failure to induce peripheral blood gluten-specific T-cells by short-term gluten challenge). Volunteers consumed Baker's Delight "white bread block loaf (200g daily for three days) or Uncle Toby's oats (lOOg daily for three days). All but three subjects completed the three day challenge (one withdrew after first mouthful of bread, and the other two vomited after initial two slices of bread. Data from the latter two were included in subsequent analysis). Blood (300ml) was drawn six days after commencing gluten challenge. Gluten peptide-specific IFNy ELISpot responses have not been found in our previous studies, and so "pre-challenge" blood was not assessed in this set of experiments (Anderson, KP et al 2000. Nat Med. 6:337-342., WO 01/25793, WO 03/104273).<br>
IFNy ELISpot assays (Mabtech, Sweden) were performed in 96-well plates (MAP S-45, Millipore) in which each well contained 25mcl of peptide solution and lOOmcl of PBMC (2-8xl05/well) in RPMI containing 10% heat inactivated human AB serum. After development and drying, IFNy ELISpot plates were assessed using the MAIP automated ELISpot plate counter. Data was then analysed according to a novel algorithm (Expectation Maximization: EM) to define and quantify interferon-garama responses to 9mer sequences contained within the peptide library (see Figure 3 and below). 9mer peptides were then rationalised according to an algorithm that assumes redundancy in T-cell recognition, the "IterativeCluster" algorithm (see Figure 4 and below), by allowing groups of amino acids with similar chemical properties at any one position in the 9mer, or for glutamate to replace glutamine at any position .(assuming deamidation may have occurred).<br>
Since there were data sets from only two HLA-DQ8+ individuals who were not also HLA-DQ2+, and these were utilizing only the 721 wheat gliadin 20mers from the "Complete gluten epitope screening library", bioacfve peptides were identified by taking the average rank of peptide-specific IFNy ELISPOT responses in the,two subjects. For prediction of likely HLA-DQ8-restricted gliadin epitopes, it was assumed that a glutamine residue susceptible to tTG-mediated deamidation occupied either position 1 or 9 in potential 9mer core regions of epitopes, consistent with the HLA-DQ8 binding motif and the findings of van de Wai et al (van de Wai, Y. et al 1998. J. Immunol. 161(4):1585-1588).<br>
Expectation Maximization (EM) algorithm to analyze data from ELISpot:<br>
Figure 3 shows an algorithm to analyze data coming from an assay using the ELISpot. T-cell responses to different peptides are measured in 96 well plates using T-cell assays. Assays are performed on many patients using many different peptide antigens. The result of the T-cell assays can be summarized in a table where the rows represent peptides and the columns patients and the individual measurements (counts) are in the table (e.g., see Figure 3B). The purpose of the EM algorithm is to differentiate between response and non response of a patient to a peptide and to estimate a mean rate of response and a proportion of people responding for each peptide.<br><br>
Responses are measured for a number of different patients (i will be used to indicate the patient) and for many different peptides (/ will be used to indicate the peptide). Each, measurement (yij) represents a count of T-cells from, patient i responding to peptide j. In order to estimate, whether a measurement for a certain peptide in a patient can be called a response or whether it is more likely to be coming from a background distribution, we propose a model for an incomplete data 'problem, wtihyij being the observed count of spots and zij an unobserved indicator, whether person / responds to peptide/.<br>
The observed number of counts yij are modelled to come from independent Poisson distributions: poisson(cu, X/), if patient i is responding to peptide j, i.e. zij = I, and poisson(az, X0), if patient / is not responding to peptide/, i.e. zij — 0.<br>
•	Complete data: yij (observed counts), zij (response indicator, not observed).<br>
•	Parameters: 6 = (ou, 'kj, W,pf)<br><br>
•	ai'. Patients overall responsiveness.<br>
•	Ij: Peptide induced rate of response.<br>
•	W: Background rate of response.<br>
•	pj: Proportion of people responding to peptide j.<br>
EM algorithm:<br><br>
•	Set variables initially to random values        '<br>
•	E-step: compute likelihood	•<br>
•	M-step: maximize likelihood function<br>
•	Iterate E- and M-step	'<br>
Iterative procedure to find minimal set of responsive epitopes<br>
A program to compute a minimal set of peptides for use in a vaccine based on the T-cell responses estimated in the EM algorithm was developed. We measured T-Cell responses to imers from a group of proteins. The peptides were generated to cover all possible Smers. We estimated the following parameters for the response by an EM algorithm: rate of response, number of people responding, proportion of people responding. The proportion of people responding multiplied with the estimated rate of response is used as a criterion to define epitopes which are good antigens. Many of the measured Imers contain the same \Smer epitopes. In order to find the epitopes (iStners) which can explain all the responses in imers we select the jSmer which is contained in imers that hi the mean have the highest responses. Then we remove all imers that contain this Smer from our measurements. Next we select the finer with the highest responses in the remaining imers. We iterate this procedure until no iSmers with responses higher than a specified cutoff exist. We use several iterations with different cutoffs. This process is sketched in Figure 4. The such defined list of clustered imers can be used as a basis to define the optimal epitopes and select peptides that function as good antigens.<br><br>
HLA-DQ2 epitopes in wheat gluten<br>
HLA-DQ2 epitopes in wheat gliadins and glutenins were identified using PBMC collected on day 6 after commencing gluten challenge in a total of 76 HLA-DQ2+ individuals in gamma-interferon ELISPOT assays (Initial gliadin epitope library: n=15, Second round gliadin epitope screening library: n=15, Complete gluten epitope screening library: n= 46). All data relating to individual peptide responses in coeliac subjects was pooled and analysed by the EM algorithm.<br>
A series of 9mer sequences were identified and ordered according to the intensity of gamma-interferon responses and the proportion of individuals responding (see Pigure 5). Many of the sequences identified could be grouped in "superfamilies" allowing for several different amino acids with similar chemical properties to be present at any one position in the putative epitope (see Figure 6). For example, in "Sequence 1" of Figure 6 (SEQ ID NO: 1555) P (QR) P (QE)LP (FY) PQ, glutamine (Q) or arginine (R) are both accepted at position 2 except that Q generates a substantially more bioactive epitope.<br>
By reviewing the 110 most "active" 9mer sequences identified by the EM algorithm, the "list" of 9mer motifs could be condensed to 41 9mers, many of which overlapped (for example "Sequence 1" and "2" (SEQ.ID NOS: 1555 and 1558 respectively) overlap by 7 residues and are both present in A-gliadin 57-73 QE65). In selected cases, high-grade peptides were synthesised and confirmed the bioactivity of peptides identified by the EM algorithm (see Figure 7).<br>
HLA-DQ2 epitopes in oats avenins	&gt;<br>
Avenin peptides were assessed after challenge with oats (n=30 subjects) or after wheat bread (n=8) in HLA-DQ2+ coeliac subjects. ELISPOT responses were found for the peptides found in Figure 8. One of the reactive avenin peptides was homologous to a sequence in wheat gluten (SEQ ID NO: 1590).<br>
Oats (avenin) high quality peptide studies	,<br>
High grade avenin peptides were assessed 3 days after completing oats challenge with pure wheat-free oats, lOOg/d for 3 days ("day 6" PBMC interferon gamma ELISPOT responses). These peptides were designed upon peptides previously defined using the screening grade ("first round") avenin peptide library and on potential deamidation sites. There were 25 peptides (as 16mers) with purity verified by HPLC as &gt;80%, and sequences confirmed by mass spectroscopy.<br>
Interferon gamma ELISPOT responses to the high grade avenin peptides following deamidation by tTG were compared in 18 subjects with DQ2+ coeliac disease .<br>
The dominant (&gt;70% maximal response) peptides after oats challenge included: EQQFGQNIFSGFSVQL (SEQ ID NO: 1764) (11/18 subjects), QLRCPAIHSWQAIIL (SEQ ID NO: 1765) (4/18 subjects), and QYQPYPEQEQPILQQQ (SEQ ID NO: 1766) (3/18 subjects). 2/18 subjects did not have avenin specific responses (defined by SFU (spot forming units) &gt; 3 X blank) and 6/18 subjects mean maximal SFU were less than 10. Two additional peptides elicited positive responses:<br><br>
QIPEQLRCPAIHSVVQ (SEQ ID NO: 1767) (3/18 subjects) and EQYQPEQQPFMQPL (SEQ ID NO: 1768) (&gt;40% maximal peptide response in 5/18 subjects). The panel of 25 peptides included several peptides similar to peptide 1490 (SEQYQPYPEQQEPFVQ) reported in Arentz-Hansen, PLoS Medicine (Oct. 2004, vol. 15 issue 1 (84-92), however, that peptide induced a strong positive response in only one subject, and far weaker response in 5 subjects.<br>
Interferon gamma ELISPOT responses to high grade avenin peptides "were absent prior to gluten challenge, and were blocked by pre-treatment of PBMG with anti-HLA DQ but not anti-HLA DR antibody.<br>
Rye and barley screening peptide libraries<br>
Secalin and hordein 20mer first round peptide libraries were assessed 3 days after completing rye (bread, lOOg/d for 3 days) or barley (boiled, lOOg/d for 3 days) challenge ("day 6" PBMC interferon gamma ELISPOT responses). Although iterative analysis using 2nd and 3rd round peptide libraries to define epitopes has not yet been performed, the 20mers pre-treated with tTG found to induce "potent" responses shared substantial structural similarity to the bioactive peptides identified after wheat challenge. However, the dominant peptide sequences after rye or barley challenge did not include peptides with the PQPQLPY sequence found to be dominant after wheat challenge. The dominant (&gt;70% maximal response) 20mer after rye challenge was usually PQQLFPLPQQPFPQPQQPFP (SEQ ID NO: 1769) (8/14 subjects), or occasionally QPFPQPQQPTPIQPQQPFPQ (SEQ ID NO: 1770) (4/14), QQPQQLFPQTQQSSPQQPQQ (SEQ ID NO: 1771) (1/14), PQTQQPQQPFPQPQQPQQLF (SEQ ID NO: 1772) (1/14) and/or QEQREGVQILLPQSHQQLVG (SEQ ID NO: 1773) (1/14). Additional peptides noted for greater than 40% maximal response in at least 1 subject include:<br>
FPQQPQQPFPQPQQQLSLQP (SEQ ID NO: 1774) (3/14, 2 &gt; 70%)<br>
PQQPFPQQPEQIIPQQPQdP (SEQ ID NO: 1775) (5/14, 3 &gt; 70%)<br>
QQLPLQPQQPFPQPQQPIPQ (SEQ ID NO: 1776) (6/14, 2 &gt; 70%)<br>
QQPQQPFPLQPQQPVPQQPQ (SEQ ID NO: 1777) (3/14,1 &gt; 70%)<br>
SIPQPQQPFPQPQQPFPQSQ (SEQ ID NO: 1778) (4/14, 1 &gt; 70%)<br>
QTQQSIPQPQQPFPQPQQPF (SEQ ID NO: 1779) (3/14, 1 &gt; 70%)<br>
NMQVGPSGQVEWPQQQPLPQ (SEQ ID NO: 1780) (2/14,1 &gt; 70%)<br>
VGPSGQVSWPQQQPLPQPQQ (SEQ ID NO: 1781) (2/14, 2 &gt; 70%)<br>
QQPFLLQPQQPFSQPQQPFL(SEQIDNO: 1782) (1/14, 1 &gt; 70%)<br>
FPLQPQQPFPQQPEQIISQQ (SEQ ID NO: 1783) (5/14, 1 &gt; 70%)<br>
PQQPQRPFAQQPEQIISQQP (SEQ ID NO: 1784) (3/14, 1 &gt; 70%)<br>
SPQQPQLPFPQPQQPFVVVV (SEQ ID NO: 1785) (4/14, 1 &gt; 70%)<br>
QQPSIQLSLQQQLNPCKNVL,(SEQ ID NO: 1786) (1/14, 1 &gt; 70%)<br><br>
Typically, the dominant peptides after barley challenge included one of six peptide motifs, or were one of eight other individual 20mers "dominant" in only one of 17 subjects after barley challenge. The six motife identified:<br>
QQPIPQQPQPY (SEQ ID NO: 1787)<br>
PFPQPQQPFPW (SEQ ID NO: 1788)<br>
LQPQQPFPQ (SEQ ED NO: 1789)<br>
PQPQQASPL (SEQ ID NO: 1790)<br>
IIPQQPQQPF (SEQ ID NO: 1791)<br>
YPEQPQQPF (SEQ ID NO: 1792)<br>
The barley hordein peptides showing at least 40% maximal peptide response in at least one subject include the following, wherein an asterisk indicates the eight individual peptides showing maximal response in a single individual:<br>
QQQPFPQQPIPQQPQPYPQQ (SEQ ID NO: 1793) (8/17,2 &gt; 70%) QQPQPFSQQPIPQQPQPYPQ (SEQ ID NO: 1794) (9/17, 8 &gt; 70%) PQQPVPQQPQPYPQQPQPFP (SEQ ID NO: 1795) (5/17, 1 &gt; 70%) PQPFPQQPIPQQPQPYPQQP (SEQ ID NO: 1796) (6/17,2 &gt; 70%) YPQQPQPFPQQPIPQQPQPY (SEQ ID NO: 1797) (6/17, 2 &gt; 70%) QPQPYPQQPQPYPQQPFQPQ (SEQ ID NO: 1798) (7/17, 2 &gt; 70%) QPQQPQPFPQQPVPQQPQPY (SEQ ID NO: 1799) (5/17, 2 &gt; 70%) PQPYPQQPQPFPQQPPFCQQ (SEQ ID NO: 1800) (1/17, 1 &gt; 70%)* QPFPQPQQPFPWQPQQPFPQ (SEQ ID NO: 1801) (10/17,2 &gt; 70%) PFPQQPQQPFPQPQQPFRQQ (SEQ ID NO: 1802) (6/17, 3 &gt; 70%) WQPQQPFPQPQQPFPLQPQQ (SEQ ID NO: 1803) (9/17, 5 &gt; 70%)* PWQPQQPFPQPQEPIPQQPQ (SEQ ID NO: 1804) (1/17, 1 &gt; 70%) QQPFPQPQQPIPYQPQQPFN (SEQ ID NO: 1805) (5/17,1 &gt; 70%) PQQPQQPFPQPQQPFSWQEQ (SEQ ID NO: 1806) (6/17,2 &gt; 70%)* QPQQPFPQPQQPIPYQPQQP (SEQ ID NO: 1807),(4/17,1 &gt;70%)* QSQQQFPQPQQPFPQQPQQP (SEQ ID NO: 1808) (1/17, 0 &gt; 70%) PFPQPQQPFSWQPQQPFLQP (SEQ ID NO: 1809) (1/17, 0 &gt; 70%) FPQPQEPFPQQPQQPFPLQP (SEQ ID NO: 1810) (1/17, 0 &gt; 70%) PFPQPQQPFPWQPQQPFPQP (SEQ ID NO: 1811) (6/17, 0 &gt; 70%) FPQYQIPTPLQPQQPFPQQP (SEQ ID,NO: 1812) (2/173 1 &gt; 70%) FPLQPQQPFPQQPQQPFPQQ (SEQ ED NO: 1813) (1/17, 0 &gt; 70%) QQPFPLQPQQPFPQPQPFPQ (SEQ ID NO: 1814) (1/17, 0 &gt; 70%) SPLQPQQPFPQGSEQIIiPQQ(SEQIDNO: 1815) (1/17, 0&gt; 70%)<br><br>
PQQASPLQPQPQQASPLQPQ (SEQ ID NO: 1816) (1/17, 1 &gt; 70%) PQQPPFWPQQPFPQQPPFGL (SEQ ID NO: 1817) (1/17, 1 &gt; 70%)* PVLSQQQPCTQDQTPLLQEQ(SEQIDNO: 1818) (1/17, 1 &gt; 70%) RQLPKYIIPQQPQQPFLLQP (SEQ ID NO: 1819) (1/17, 1 &gt; 70%) QGSEQIIPQQPQQPFPLQPH (SEQ ID NO: 1820) (7/17, 3 &gt; 70%)* PQGSEQIIPQQPFPLQP.QPF (SEQ ID NO: 1821) (2/17, 1 &gt; 70%) QPFPTPQQFFPYLPQQTFPP (SEQ ID NO: 1822) (4/17, 1 &gt; 70%) PFPQPPQQKYPEQPQQPFPW (SEQ ID NO: 1823) (1/17,1 &gt; 70%) QKYPEQPQQPFPWQQPTIQL (SEQ ID NO: 1824) (1/17, 1 &gt; 70%) FQQPQQSYPVQPQQPFPQPQ (SEQ ID NO: 1825) (3/17, 1 &gt; 70%) QIPYVHPSILQQLNPCKVFL (SEQ ID NO: 1826) (1/17, 1 &gt; 70%) LAAQLPAMCRLEGGGGLLAS (SEQ ID NO: 1827) (1/17, 1 &gt; 70%) PYLPEELSPQYQIPTPLQPQ (SEQ ID NO: 1828) (1/17, 1 &gt; 70%)* VSPHPGQQTTVSPHQGQQTT (SEQ ID NO: 1829) (1/17, 1 &gt; 70%)*<br>
Second and third round wheat glutenin and gliadin peptide libraries<br>
The second round wheat gliadin and glutenin library was designed upon the sequences of<br>
20mer wheat gliadin and glutenin peptides 'that induced at least 5% of the response (interferon<br>
gamma ELISPOT) stimulated by the most active transglutaminase (tTG) pre-treated (enzymatically<br>
deamidated) 20mer peptide in any subject. All 2nd round 16meT peptides were assessed in at least 18<br>
subjects. The 2nd round library generated from the "Oxford" gliadin 20mer library had been assessed<br>
in ten subjects - this data was merged with data generated from the 18 subjects used to assess the new<br>
2nd round (expanded) gliadin/glutenin.library. Hence, individual 16mer peptides pre-treated with<br>
transglutaminase were assessed in either 18 (novel gliadin/glutenin sequences based on "Melbourne"<br>
20mer library) or 28 subjects (gliadin sequences based on "Oxford" 20mer library). All 16mers<br>
identified for the second round Oxford library also fulfilled the selection criteria for the Melbourne<br>
second round library.	,<br>
The second round peptide library data was analysed according to the "dominance" of peptide responses in the interferon gamma ELISPOT in individual subjects i.e. the percent response of an individual's PBMC to a specific peptide normalized against that individual's maximal peptide-induced response. Sequences of peptides that stimulated at least 40% of the maximal peptide-specific response in at least one subject are shown in Table 1 below. The dataset supports the consistency and "dominance" of peptides conforming to the sequences identified using the first round 20mer peptide library using the Expectation Maximization (EM) algorithm described above. Table 1: Peptides confirmedin Second Round Library as at least 40% as active as the peptide -with maximal activity in any one subject: Ranked according to potency of peptide family<br><br>
Peptide 	SEQIDNO: 	&gt;70% 	40-70%      10-40% 	
G -QLPYPQPQLPYPQP - G 	1830 	18/28 	4/28 	3/28 	3/28 <br>
G-LQPFPQPQLPYPQP-G 	1831 	14/28 	8/28 	Nil 	6/28 <br>
G-LQPFPQPQLPFPQP-G 	1832 	4/28 	8/28 	5/28 	10/28 <br>
G-LQPFPQPQLPYLQP-G 	1833 	1/28 	1/28 	12/28 	14/28 <br>
G-LQPFPQPQLPYSQP-G 	1834 	2/28 	3/28 	12/28 	1 1/28 <br>
G-LQPFPQPQLSYSQP-G 	1835 	Nil 	1/28 	2/28 	25/28 <br>
G-QQPFPQPQQPFPWQ-G 	1837 	9/28 	8/28 	5/28 	6/28 <br>
G-QQPFPQPQQPIPVQ-G 	1838 	8/28 	5/28 	7/28 	8/28 <br>
G-QQPFPQPQQPFSQQ-G 	1839 	4/28 	7/28 	8/28 	9/28 <br>
G-QQPFPQPQQPFCQQ-G 	1840 	2/28 	2/28 	13/28 	11/28 <br>
G-GLERPWQQQPLPPQ-G 	1841 	2/18 	1/18 	Nil 	15/18 <br>
G-QTFPHQPQQAFPQP-G 	1842 	1/28 	2/28 	Nil 	25/28 <br>
LQQQCSPVAMPQRLAR 	1S43 	1/28 	1/28 	11/28 	15/28 <br>
QGQQGYYP ISPQQSGQ 	1844 	1/18 	1/18 	1/18 	15/18 <br>
PGQGQSGYYPTSPQQS 	1845 	1/18 	1/18 	Nil 	16/18 <br>
QGQPGYYPTSPQQIGQ 	1846 	1/18 	1/18 	1/18 	15/18 <br>
GQGQSGYYPTSPQQSG 	1847 	1/18 	Nil 	2/18 	15/18 <br>
QQGYYPTSPQQSGQGQ 	1848 	Nil 	1/18 	Nil 	17/ <br>
QGQQGYYPTSPQQPPQ 	1849 	Nil 	1/18 	Nil 	Nil <br>
QQGYYPISPQQLGQGQ 	1850 	Nil 	1/18 	Nil 	Nil <br>
YVPPDCSTINVPY2NI 	1851 	-    1/18 	1/18 	1/18 	15/18 <br>
IIMQQEQQEQRQGVQI 	1852 	1/28 	Nil 	8/28 	19/28 <br>
VAHAIIMHQQQQQQQE 	1853 	Nil 	1/28 	2/28 	25/28 <br>
G-QPIPgQPQQPFPLQ-G 	1854 	.1/28 	Nil 	5/28 	<br>
G-FPQLQQPQQPFPQQ-G 	1855 	1/28 	Nil 	1/28 	26/28 <br>
G-FPQTQQPQQPFPQQ-G 	1856 .. 	Nil 	1/28 	2/28 	25/28 <br>
G-QPLSQQPQQTFPQQ-G 	1857 	Nil 	1/28 	Nil 	27/28 <br>
G-QQPQQQPQQPFPQQ-G 	1858 	Nil      , 	1/28 	5/28 	22/28 <br>
G-FPQPQQPQQPFPQQ-G 	1859 	.    Nil 	1/28 	3/28 	25/28 <br>
G-FPQPQQPQQSFPQQ-G 	1860          ; 	Nil 	1/28 	.    1/28 	26/28 <br>
G- QPQQTFPgQPQLPF-G 	1861 	1/18 	Nil 	2/18 	15/18 <br>
G-MQVDPSGgVQWPQQ-G 	1862 	1/18 	Nil 	Nil 	17/18 <br>
G-IQVDPSGQVQWPQQ-G 	1863 	1/18 	Nil 	Nil 	17/18 <br>
G-MQADP SGQVQWPQQ-G 	1864 	1/18 	Nil 	Nil 	17/18 <br>
G-MQVDP S S QVQWP QQ- G 	1865 	1/18 	Nil 	Nil 	17/18 <br>
G-QQEQQILQQILQQQ-G 	1866 	1/18 	Nil 	Nil 	17/18 <br>
VPLYRTTTSVPFGVGT 	1867 	1/18 	Nil 	Nil 	17/18 <br>
LQTLPSMCNVYIPPYC 	1868 	1/18 	Nil 	Nil 	17/18 <br>
LALQTLPAMCNVYIPP 	1869 	1/18 	Nil 	Nil 	17/18 <br>
DAIRAIir S I VLQEQQ 	1S70 	1/18 	NH 	Nil 	17/18 <br>
G-QQQFSQPQQQFPQP-G 	1871      ' 	Nil 	5/28 	7/28 	16/28 <br>
G-FFPQPQQQFPQPQQ-G 	1872 	Nil 	1/28 	10/28 	17/28 <br><br>
G-FPQQPQQQFPQPQQ-G 	1873 	Nil 	1/28 	Nil 	I 	 27/28 <br>
G-QQPFPQPQQQFPQP-G 	1874 	Nil 	1/28 	12/28 	15/28 <br>
G-QPQPFiPQiPrPBP- G 	1875 	Nil 	4/28 	9/28 	15/28 <br>
G-QQPFPQPQQQLPQP-G 	1876 	Nil 	3/28 	6/28 	19/28 <br>
G-LPFPQQPQQPLPQP-G 	1877 	•      Nil 	2/18 	4/18 	12/18 <br>
G- QQAFPQPQQTFFHQ- G 	1878 	Nil 	3/28 	8/28 	17/28 <br>
G-QQPFTQPQQPTPIQ-G 	1879 	Nil 	1/28 	4/28 	23/28 <br>
G-QQIFPQPQQTFPHQ-G 	1880 	Nil 	1/28 	10/28 	17/28 <br>
G- QQQFIQPQQPFPQQ- G 	1881 	Nil 	2/28 	11/28 	15/28 <br>
G-QPFPLQPQQPFPQQ-G 	1882 	Nil 	2/28 	7/28 	19/28 <br>
G-QPFPWQPQQPFPQQ-G 	1883 	Nil 	2/28 	8/28 	18/28 <br>
G-QPTPIQPQQPFPQQ-G 	1884 	Nil 	2/28 	5/28 	21/28 <br>
G-QVSFQQPQQQYPSP-G 	1885 	Nil 	2/28 	4/28 	22/28 <br>
G-FFQQPQQQYPSSQQ-G 	1886 	Nil 	1/28 	1/28 	26/28 <br>
G-GKSQVLQQSTYQLL-G 	1887 	Nil 	2/18 	1/18 	15/18 <br>
GQWNNHGQTVFNDIG 	1888 	Nil 	1/18 	4/18 	<br>
G-QPQLPFPQQPQQQF-G 	1889 	Nil 	1/28 	2/28 	25/28 <br>
G-QPFPQPQQAQLPFP-G 	1890 	Nil 	1/28 	2/28 	25/28 <br>
G-HQQPGQRQQGYYPT-G 	1891 	Nil 	1/18 	1/18 	16/18 <br>
G-HQQFPQQQIPWQP-G 	1892 	Nil 	1/18 	1/18 	16/18 <br>
LEAVTSIALRT1PTMC 	1893     . 	Nil 	1/18 	1/18 	<br>
G-QQPQFSQQQQIPVI-G 	1894 	. Nil 	1/18 	Nil 	17/18 <br>
The third round peptide library consisted of 74 peptides based upon structurally distinct sequences in the second round library found to induce at least 10% of the maximal response to any peptide in any subject. These peptides corresponded to wild-type (non-deamidated) sequences virtually identical to those used in the second round library. The distinct feature of this library was that it consisted of peptides with purity verified by HPLC as &gt;80%, and with sequences confirmed by mass spectroscopy.<br>
Interferon gamma ELSIPOT responses to the 3rd round library peptides following deamidation by tTG were compared in 14 subjects. Once again, sequences including the PQPQLPY motif were "dominant" in 9/14 subjects. However PFPQPQQPFPW (SEQ ID NO: 1895) stimulated &gt;70% of maximal response in 1/14 subjects, PFPQQPQQPFPQ (SEQ ED NO: 1896) in 1/14, PQPFLPQLPYPQP (SEQ ID NO: 1897) in 1/14, QPFPQPQQPQQP (SEQ ID NO: 1898) in 4/14 (including 3 subjects in whom PQPQLPY peptides were not potent epitopes) SGQGVSQSQQQSQQQ (SEQ ID NO: 1899) in 2/14 (including one in which PQPQLPY peptides were not potent), QYEVIRSLVLRTLPNM (SEQ ID NO: 1900) and GLARSQMLQQSICHVG (SEQ ED NO: 1901) each in one (the same) subject in whom PQPQLPY peptides were not potent epitopes, RTTTSVPFGVGTGVGA (SEQ ID NO: 1902) in 1/14 subjects and AIHTVIHSIIMQQEQQ (SEQ ID NO: 1903) in 1/14 subjects.<br><br>
Many of the sequences tested in third round were structurally related and individual subject's responses were present or absent according to the "relatedness" of certain sequences, suggesting redundancy of peptides recognized by gluten specific T cells induced by in vivo gluten challenge.<br>
Interferon gamma ELISPOT responses to 3rd round peptides were absent before gluten challenge, and were blocked by pre-treatment of PBMC with anti-HLA DQ but not HLA DR antibody.<br>
Combitopes<br>
The issue of epitope redundancy and the potential utility in diagnostics and therapeutics of peptides designed to combine "unique" dominant epitopes was addressed by comparing interferon gamma ELISPOT responses after wheat (n=16 HLA DQ2 coeliac disease subjects), rye (n=17) or barley (n=13) challenge to the sequences: QLQPFPQPELPYPQPQL (SEQ ID NO: 1904) ("P04724E"), QPEQPFPQPEQPFPWQP (SEQ ID NO: 1905) ("626fEE")5 and' QLQPFPQPELPYPQPFPQQPEQPFPQPEQPFPWQP (SEQ ID NO: 1906) ("Combitope"). After rye and barley challenge the sum of the median ELISPOT responses (spot forming units) to P04724E and 626fEE were almost identical (99%, and 102%, respectively) to the response to a similar (optimal) concentration of the Combitope. However, after wheat challenge (n=16 subjects), median P04724E response was 89% of that to Combitope, and median 626fEE responses was 70% of the response to Combitope. These findings would be consistent with substantial redundancy of these related epitope sequences, P04724E and 626£EE, after wheat challenge but not after rye or barley, and that combining dominant epitope sequences within longer peptides does not reduce their biological availability. Hence, combitopes derived from selected potent epitopes may be efficient delivery devices for T cell epitope-based therapeutics and diagnostics in coeliac disease.<br>
Epitopes in wheat gluten associated with HLA-DQ8+ coeliac disease<br>
Epitopes in wheat gliadins were identified using PBMC after gluten challenge in two individuals, one HLA-DQ8 homozygous, and one HLA-DQ8 heterozygote. Induced T-cell responses in other HLA-DQ8 (not DQ2) coeliac inidividuals responded weakly to gluten challenge and their data did not allow detailed analysis'.<br>
Deamidated 20mers including the core sequence: QGSFQPSQQ (SEQ ID NO: 1907), corresponding to the known HLA-DQ8-restricted alpha-gliadin epitope (in which Ql and Q9 are deamidated by tTG for optimal activity), induced moderately strong peptide responses. However, a series of "core" peptides were associated with more potent responses in 20mers derived from gamma and omega gliadins (see Figure 9). The most potent peptides^iossessed glutamine in a sequence that would suggest susceptibility to deamidation separated by seven residues from a second glutamine also susceptible to deamidation (as found in QGSFQPSQQ (SEQ ID NO: 1907)) suggesting that these deamidated sequences would become high affinity binders for HLA-DQ8 following deamidation by tTG. (The binding motif for HLA-DQ8 favours glutamate at positions 1 and 9.) A further group of<br><br>
20mers possessed glutamine residues susceptible to deamidation but not separated by seven residues from a second glutamine susceptible to tTG-mediated deamidation.<br>
HLA DQ8 coeliac disease gliadin and glutenin epitopes<br>
Five subjects with coeliac disease that possess HLA DQ2 and HLA. DQ8 alleles underwent wheat gluten challenge. PBMC from two subjects initially challenged were used to screen the first round "Melbourne" wheat gliadin 20mer library. The 20mer sequences identified using PBMC from these two HLA DQ8 CD subjects were dissected further by screening, in five HLA DQ8+ DQ2- CD subjects including the two original subjects, a second round library based on reactive 20mers in the 1st round library. The 2nd round library consisted of screening grade overlapping 16mers, and ISmers predicted to correspond to tTG-mediated deamidation products of epitopes with the potential for deamidation of glutamine at position 1 and/or position 9 (consistent with the HLA DQ8 peptide binding motif). In addition, the 1400 glutenin (HMW and LMW) tTG-pretreated 20mers in the "Melbourne" wheat gluten library were also screened in these five subjects.<br>
The most potent and consistently dominant gliadin 16mers were the related sequences<br>
VYIPPYCTIAPFGIFG (SEQIDNO: 1908) (3/5 subjects &gt;70% response to maximal gliadin 16mer)<br>
and AMCNVYIPPYCAMAPF (SEQ ID NO: 1908) also dominant in 3/5 subjects (4/5 subjects<br>
produced dominant responses to one or both of these peptides). In addition, a series of peptides<br>
derived from previously identified bioactive 20mers whose responses in the ELISPOT were<br>
enhanced  or permissive  to  specific   glutamine  residues  being  deamidated  were   identified:<br>
(QE)QPTPIQP(QE)    (SEQ   ID   NO:   1909),    (QE)QPFPLQP(QE)    (SEQ   ID   NO:   1910),<br>
(QE)QPIPVQPJQE)    (SEQ   ID   NO:   1911),    (QE) QPQQPFP (QE)    (SEQ   ID   NO:   1912),<br>
(QE)QP(QE)LPFP(QE)   (SEQ ID NO:   1913),   (QE) GSFQPSQ (QE)   (SEQ ID NO:   1914)<br>
(previously published HLA DQ8 epitope, van der Wai 1998), (QE) LPFP (QE) QP (QE) (SEQ ID<br>
NO: 1915), and (QE) QPFP (QE') QP (QE) (SEQ DDNO: 1916).<br>
Screening the glutenin 20mer library identified a further series of sequences that were dominant in at least one of the five subjects. Dominant 20mer peptides shared the motifs or had the sequences: PQQQQQQLVQQQ (SEQ ID NO: 1917), QGIFLQPH (LQ) I (AS) QLEV (SEQ ID NO: 1918), QPGQGQQG(HY)Y (SEQ ID NO: 1919), QSRYEAIRAII (FY) S (SEQ ID NO: 1920), RTTTSVPFD (SEQ ID NO: 1921), QPPFWRQQP (SEQ ID NO: 1922), Q(PS) (PS) (FI) (PS)QQQQ (SEQ ID NO: 1923), (QPLR)GYYPTSPQ (SEQ ID' NO: 1924) (previously identified HLA DQ8 epitope, van der Wai 2001), QGSYYPGQASPQ (SEQ ID NO: 1925), GYYPTSSLQPEQGQQGYYPT (SEQ ID NO: 1926), and QGQQLAQGQQGQQPAQVQQG (SEQ ID NO: 1927). Glutenin peptides were assessed after pre-treatment with transglutaminase. Hence, the requirement for deamidation for these epitopes is not known.<br>
A comprehensive library of "uncharacterised" screening grade peptides including all unique 12mer sequences encoded by genes present in Genbank defined as (bread making) wheat (Triticum<br><br>
aestivum), rye, barley,, or oats gluten, gliadin, glutenin, secalin, hoidein, or avenin have been assessed using T cells from HLA DQ2+ (and in some cases HLA DQ8+) coeliac disease volunteers six days after commencing in vivo gluten challenge. A relatively consistent pattern of epitope hierarchy has been identified in HLA DQ2 coeliac disease that is similar to but not identical after consumption of other grains toxic in coeliac disease. Peptides with the sequence PQPQLPY are dominant after wheat challenge in at least two thirds of HLA DQ2+ coeliac disease, but other epitopes are occasionally dominant while PQLPY peptides are essentially inactive in fewer than one in six HLA DQ2+ subjects with coeliac disease. The contribution of rare dominant epitopes will be better assessed after screening large numbers (e.g. &gt;30) subjects (in progress). Epitope hierarchy after rye and barley consumption is similar to that after wheat with the exception that deamidated peptides similar to the gliadin/hordein/secalin sequences PQPQQPFP or PFPQQPQQP are usually dominant rather than PQPQLPY (a sequence unique to wheat alpha-gliadins). Combitopes that comprise serial and partially overlapping gluten epitopes are as active or more active than single epitopes alone and offer a means of efficiently delivering multiple gluten epitopes for T cell recognition. Such combitopes are therefore useful in design arid delivery of peptide therapies in coeliac disease that target multiple unique T cell .epitbpes.<br>
REFERENCES<br>
1.	Molberg 0, et al. Nature Med. 4, 713-717 (1998).<br>
2.	Quarsten H, et al. Eur. J. Iramunol. 29,2506-2514 (1999).<br>
3.	GreenbergCS etal. FASEB 5, 3071-3077 (1991).<br>
4.	Mantearis G, Jewell D. Scand. J. Gastroenterol. 26, 392-398 (1991).<br>
5.	Mauri L, et ah Scand. 1 Gastroenterol. 31, 247-253 (1996).<br>
6.	Bunce M, et al. Tissue Antigens 46, 355-367 (1995).<br>
7.	Olenip O, et al. Tissue antigens 41, 119-134 (1993).<br>
8.	Mullighan CG, et al. Tissue-Antigens. 50, 688-92 (1997).<br>
9.	Plebanski M et al. Eur. J. Immunol. 28,4345-4355 (1998).<br>
10.	Anderson DO, Greene PC. The alpha-gliadin gene family. IL DNA and protein sequence<br>
variation, subfamily structure, and origins of pseudogenes. Theor Appl Genet (1997) 95:59-65.<br>
11.	Arente-Hansen H, Korner R, Molberg 0, Quarsten H, Van der Wai Y, Kooy YMC, Lundin KEA,<br>
Koning F, Roepstorff P, Sollid LM, McAdam SN. The intestinal T cell response to alpha-gliadin in<br>
adult celiac disease is focused on a single deamidated glutamine targeted by tissue transglutaminase.<br>
J Exp Med. 2000; 191:603-12.<br>
12.	Vader LW, de Ru A, van der Wai, Kooy YMC, Benckhuijsen W, Mearin ML, Drijfhout JW, van<br>
Veelen P, Koning F. Specificity of tissue transglutaminase explains cereal toxicity in celiac disease. J<br>
Exp Med 2002; 195:643-649.<br>
13.	van der Wai Y, Kooy Y, van Veelan P, Pena S, Mearin L, Papadopoulos G, Koning F. Selective<br>
deamidation by tissue transglutaminase strongly enhances  gliadin-specific T cell reactivity. J<br>
Immunol. 1998; 161:1585-8.<br>
14.	van der Wai Y, Kooy Y, van Veelan P, Pena S, Mearin. L, Molberg O, Lundin KEA, Sollid L,<br>
Mutis T, Benckhuijsen WE, Drijfhout JW, Koning F. Proc Nati Acad Sci USA 1998; 95:10050-<br>
10054,	.<br>
15.	Vader W, Kooy Y, Van Veelen P et al. The gluten response in children with celiac disease is<br>
directed toward multiple gliadin and glutenin peptides. Gastroenterology 2002,122:1729-37<br>
16.	Arentz-Hansen H, McAdam SN, Molberg O, et al. Celiac lesion T cells recognize epitopes that<br>
cluster in regions of gliadin rich in proline residues. Gastroenterology 2002,123:803-809.<br>
Each of the PCT publications, U.S. patents, other patents, journal references, and any other publications cited or referred to herein is incorporated.herein by reference in their entirety.<br><br>
WE CLAIM:<br>
1.	A method of preventing or treating coeliac disease comprising administering to an<br>
individual at least one agent selected from:<br>
(a)	a peptide comprising at least one epitope comprising a sequence selected from the group<br>
consisting of SEQ ID NOs: 1-1927, and equivalents thereof; and<br>
(b)	an analogue of (a) which is capable of being recognised by a T cell receptor that<br>
recognises the peptide of (a) and which is not more than 50 amino acids in length.<br><br>
2.	A method of claim 1 wherein the agent is HLA-DQ2-restricted.<br>
3.	A method of claim 1 wherein the agent is HLA-DQS-restricted.<br>
4.	A method of claim 1 wherein one agent is HLA-DQ2-restricted and a second agent is<br>
HLA-DQS-restricted.<br>
5.	A method of claim 1 wherein the agent comprises a wheat epitope,<br>
6.	A method of claim 1 wherein the agent comprises an oat epitope.<br>
7.	A method of claim 1 wherein one agent comprises a wheat epitope and one agent<br>
comprises an oat epitope.<br>
8.	A method of preventing or treating coeliac disease comprising administering to an<br>
individual a pharmaceutical composition comprising an agent as defined in claim 1  and a<br>
pharmaceutically acceptable carrier or diluent.      :<br>
9.	A method of preventing or treating coeliac disease comprising administering to an<br>
individual a pharmaceutical composition comprising an antagonist of a T cell which has a T cell<br>
receptor as defined in claim 1, and. a pharmaceutically acceptable carrier or diluent.<br>
10.	A method of preventing or treating coeliac disease comprising administering to an<br>
individual a composition for tolerising an individual to a gluten protein to suppress the production of<br>
a T cell or antibody response to an agent as defined in claim 1, which composition comprises an<br>
agent as defined in claim 1.<br>
11.	A method of preventing or treating coeliac disease comprising:<br>
diagnosing coeliac disease in an individual by either:<br>
a)   contacting a sample from the host with at least one agent selected from:<br>
i) a peptide comprising at least one epitope comprising a sequence selected from the<br>
group consisting of: SEQ ID NOS: 1-1927, and equivalents thereof; and ii) an analogue of i) which is capable of being recognised by a T cell receptor that<br>
recognises i) and which is not more than 50 amino acids in length; and<br><br>
determining in vitro whether T cells in the sample recognise the agent; recognition by<br>
the T cells indicating that the individual has, or is susceptible to, coeliac disease; or b)   administering an agent as defined in claim I and determining in vivo whether T cells in<br>
the individual recognise the agent, recognition of the agent indicating that the individual<br>
has or is susceptible to coeliac disease; and<br>
administering to an individual diagnosed as having, or being susceptible to, coeliac disease a therapeutic agent for preventing or treating coeliac disease.<br>
12.	Use of an agent for the preparation of a medicament for treating or preventing coeliac<br>
disease, wherein the agent comprises:<br>
(a)	a peptide comprising at least one epitope comprising a sequence selected from the group<br>
consisting of SEQ ID NOs: 1-1927 and equivalents thereof; and<br>
(b)	an analogue of (a) which is capable of being recognised by a T cell receptor that<br>
recognises the peptide of (a) and which is not more than 50 amino acids in length.<br><br>
13.	A use of claim 12 wherein the agent is HLA-DQ2-restricted.<br>
14.	A use of claim 12 wherein the agent is HLA-DQ8-restricted.<br>
15.	A use of claim 12 wherein one agent is HLA-DQ2-restricted and a second agent is BLA-<br>
DQ8-restricted.<br>
16.	A use of claim 12 wherein the agent comprises a wheat epitope.<br>
17.	A use of claim 12 wherein the agent comprises an oat epitope.<br>
18.	A use of claim 1'2 wherein one .agent comprises a wheat epitope and one agent comprises<br>
an oat epitope.<br>
19.	A use of claim 12 wherein the agent is present within a pharmaceutical composition<br>
comprising a pharmaceutically acceptable'carrier or diluent.<br>
20.	A use of claim 12 wherein the agent is present within a pharmaceutical composition<br>
comprising an antagonist of a T cell which has a T cell receptor as defined in claim 12, and a<br>
pharmaceutically acceptable carrier or diluent.<br>
21.	A use of claim 12 wherein the agent is present within a composition for tolerising an<br>
individual to a gluten protein to suppress the production of a T cell or antibody response to an agent<br>
as defined in claim 1,<br>
22.	An agent as defined in claim 1, optionally in association with a carrier, for use in a<br>
method of treating or preventing coeliac disease by tolerising T cells which recognise the agent.<br><br>
23.	An antagonist of a T cell which has a T cell receptor as defined in claim 1, optionally in<br>
association with a carrier, for use in a method of treating or preventing coeliac disease by<br>
antagonising such T cells.<br>
24.	An agent as defined in claim 1 or an analogue that binds an antibody that binds to an<br>
epitope of an agent as defined in claim 1 for use in a method of treating or preventing coeliac disease<br>
in an individual by tolerising the individual to prevent the production of such an antibody.<br>
25.	A protein that comprises a sequence which is able to bind to a T cell receptor, which T<br>
cell receptor recognises an agent as defined in claim 1. and which sequence is able to cause<br>
antagonism of a T cell that carries such a T cell receptor.<br>
26.	An agent as defined in claim 1 or an antagonist as defined in claim 9.<br>
27.	A pharmaceutical composition comprising an agent as defined hi claim  1  or an<br>
antagonist as defined in claim 9 and a pharmaceutically acceptable carrier or diluent.<br>
28.	A composition for tolerising an individual to a gluten protein to suppress the production<br>
of a T cell or antibody response to an agent as defined in claim 15 which composition comprises an<br>
agent as defined in claim 1.<br>
29.	A composition for antagonising a T cell response to an agent as defined in claim 1,<br>
which composition comprises an antagonist as defined in claim 9.<br>
30.	A  mutant   gluten  .protein  whose   wild-type   sequence   can   be   modified   by   a<br>
transglutaminase to a sequence which is an agent as defined in claim 1, which mutant gluten protein<br>
comprises a mutation which prevents its modification by a transglutaminase to a sequence which is<br>
an agent as defined in claim 1; or a fragment of such a mutant gluten protein which is at least 7<br>
amino acids long and which comprises the mutation.<br>
31.	A polynucleotide that comprises a coding sequence that encodes a protein or fragment as<br>
defined in claim 25 or 30.	.	-   ,<br>
32.	A polynucleotide according to claim 31 'that additionally comprises one or more<br>
regulatory sequences operably linked to the coding sequence, which regulatory sequences are<br>
capable of securing the expression of the coding sequence in a cell.<br>
33.	A polynucleotide according to claim 32 wherein the regulatory sequence(s) allow<br>
expression of the coding sequence in a prokaryotic or mammalian cell.<br>
34.	A polynucleotide according to any one of claims 31 to 33 which is a vector or which is<br>
in the form of a vector.<br><br>
35.	A cell comprising apolynucleotide as defined in any one of claims 31 to 34 or which has<br>
been transformed with such a polynucleotide.<br>
36.	A cell according to claim 35 which is a prokaryotic cell or a mammalian cell.<br>
37.	A mammal that expresses a T cell receptor as defined in claim 1.<br>
38.	A method of diagnosing coeliac disease, or susceptibility to coeliac disease, in an<br>
individual comprising:<br>
(a)	contacting a sample from the host with at least one agent selected from:<br>
i) a peptide comprising at least one epitope comprising a sequence selected from the group consisting of: SEQ ID NO.S: 1-1927, and equivalents thereof; and<br>
ii) an analogue of (i) which is capable of being recognised by a T cell receptor that recognises (i) and which is not more than 50 amino acids in length; and<br>
(b)	determining in vitro whether T cells in the sample recognise the agent; recognition by<br>
the T cells indicating that the individual has, or is susceptible to, coeliac disease.<br>
39.	Use of an agent as defined in claim 38 for the preparation of a diagnostic means for use<br>
in a method of diagnosing coeliac disease, or susceptibility to coeliac disease, in an individual, said<br>
method comprising determining whether T cells of the individual recognise the agent, recognition by<br>
the T cells indicating that the individual has, or is susceptible to, coeliac disease.<br>
40.	A method or use according to claim 38 or 39 wherein the agent is an analogue (iii) which<br>
comprises (i) or (ii) bound to (a) an HLA molecule, or (b) a fragment of an HLA molecule capable of<br>
binding (i) or (ii).<br>
41.	A method or use according to claim 40 wherein the HLA molecule or fragment is in a<br>
complex comprising four HLA molecules or fragments of HLA molecules.<br>
42.	Use according to any one of claims 39-41 wherein the method comprises administering<br>
the agent to the skin of an individual and detecting the presence of inflammation at the site of<br>
administration, the .detection of inflammation indicating that the T cells of the individual recognise<br>
the agent.	.    .        . •<br>
43.	A.method according to claim 39, 41 or 42 wherein the sample is blood sample.<br>
44.	A method according to claim 38, 40, 41 or 43 wherein the T cells are not restimulated in<br>
antigen specific manner in vitro before the said determining.<br>
45.	A method or use according to any one claims 3 8-44 in which the recognition of the agent<br>
by the T cells is determined by detecting the secretion of a cytokine from the T cells.<br>
46.	A method or use according to claim 45 hi which the cytokine is EFN-y.<br><br>
47.	A method or use according to claim 45 or claim 46 in which the cytokine is detected by<br>
allowing the cytokine to bind to an-immobilised antibody specific to the cytokine and then detecting<br>
the presence of the antibody/cytokine complex.<br>
48.	A method or use according to any one of claims 38 to 44 wherein said determining is<br>
done by measuring whether the agent binds the .Tcell receptor.<br>
49.	A method for identifying an analogue as defined in a claim 38, 40 or 41 comprising<br>
determining whether a candidate substance is recognised by a T eel] receptor that recognises an<br>
epitope comprising sequence as defined in claim 38, recognition of the substance indicating that the<br>
substance is an analogue.<br>
50.	A method of diagnosing coeliac disease, or susceptibility to coeliac disease, in an<br>
individual comprising determining the presence of an antibody that binds to an epitope of an epitope<br>
comprising sequence as defined in claim 38 in a sample from the individual, the presence of the<br>
antibody indicating that the individual has, or is susceptible to, coeliac disease.<br>
51.	A method of determining whether a composition is capable of causing coeliac disease<br>
comprising determining whether a protein capable of being modified by a transglutaminase to an<br>
oligopeptide sequence as defined in claim 38 is present in the composition,, the presence of the<br>
protein indicating that the composition is capable of causing coeliac disease.<br>
52.	A method according to claim 51 wherein the said determining is done by contacting the<br>
composition with an antibody specific for the sequence which is capable of being modified to the<br>
oligopeptide sequence, binding of the antibody to a protein hi the composition indicating the<br>
composition is capable of causing coeliac disease. :<br>
53.	A method of identifying an antagonist of a T cell, which T cell recognises an agent as<br>
defined in claim 1, comprising contacting a candidate substance with the T cell and detecting<br>
whether the substance causes a decrease in the ability of the T cell to undergo an antigen specific<br>
response, the detecting of any such decrease in said ability indicating that the substance is an<br>
antagonist.	•        •<br>
54.	A kit for carrying out a method of use 'according to any one of claims 38 to 48<br>
comprising an agent as defined in claim 38, 40 or 41 and a means to detect the recognition of the<br>
peptide by the T cell.<br>
55.	A kit according to claim 54 wherein the means to detect recognition comprises an<br>
antibody to IFN-y.<br>
56.	A Idt according to claim 55 wherein the antibody is immobilised on a solid support and<br>
optionally the kit also comprises a means to detect the antibody/IFN-y complex.<br><br>
57.	Use of an agent or antagonist as defined in claim 56 or a wild type sequence as defined<br>
in claim 30 to produce an antibody specific to the agent, antagonist or wild type sequence.<br>
58.	Use of a mutation in an epitope of a gluten protein, which epitope is as defined in claim<br>
38, to decrease the ability of the gluten protein to cause coeliac disease.<br>
59.	Method of identifying a product which is therapeutic for coeliac disease comprising<br>
administering a candidate substance to a mammal as defined in claim 37 which has, or which is<br>
susceptible to, coeliac disease and determining whether substance prevents or treats coeliac disease<br>
in the mammal, the prevention or treatment of coeliac disease indicating that the substance is a<br>
therapeutic product.<br>
60.	A therapeutic product as identified in the method of claim 59 for use in a method of<br>
preventing or treating coeliac disease.<br>
61.	A method of diagnosing coeliac disease, or susceptibility to coeliac disease in an<br>
individual comprising administering an agent as defined in claim 38 and determining in vivo whether<br>
T cells in the individual recognise the agent, recognition of the agent indicating that the individual<br>
has or is susceptible to coeliac disease.<br>
62.	A cell according to claim 35 which is. a cell of a graminaceous monocotyledonous<br>
species.<br>
63.	A cell according to claim 62 which is a cell of wheat, maize, oats, rye, rice, barley,<br>
triticale, sorghum, or sugar cane.<br>
64.	A process for the production of a protein encoded by a coding sequence as defined in<br>
claim 31 which process comprises:<br><br>
(a)	cultivating a cell according to any one of claims 35S 36, 62 or 63 under conditions<br>
that allow the expression of the protein; .and optionally<br>
(b)	recovering the expressed protein.<br>
65.	A method of obtaining a trans genie plant cell comprising:<br>
(a)	transforming a plant cell with a vector according to claim 34 to give a transgenic<br>
plant cell.<br>
66.	A method of obtaining a first-generation transgenic plant comprising:<br>
(b)	regenerating a transgenic plant cell transformed with a vector according to claim 34<br>
to give a transgenic plant.<br>
67.	A method of obtaining a transgenic plant seed comprising:<br>
(c)	obtaining a transgenic seed from a transgenic plant obtainable by step (b) of claim 66.<br><br>
68.	A method of obtaining &amp; transgenic progeny plant comprising obtaining a second-<br>
generation transgenic progeny plant from a first-generation transgenic plant obtainable by a method<br>
according to claim 66, and optionally obtaining transgenic plants of one or more further generations<br>
from the second-generation progeny plant thus obtained,<br>
69.	A method according to claim 68 comprising:<br>
(d)	obtaining a transgenic seed from a first-generation transgenic plant obtainable by the<br>
method according to claim 67, then obtaining a second-generation transgenic progeny plant from the<br>
transgenic seed;<br>
and/or<br>
(e)	propagating clonally a first-generation transgenic plant obtainable by the method<br>
according to claim 66 to give a second-generation progeny plant;<br>
and/or<br>
(f)	crossing a first-generation transgenic plant obtainable by a method according to claim<br>
66 with another plant to give a second-generation progeny plant;<br>
and optionally<br>
(g)	obtaining transgenic progeny plants of one or more further generations from the<br>
second-generation progeny plant thus obtained.<br>
70.	A transgenic plant cell, plant plant seed or progeny plant obtainable by a method<br>
according to snj one of claims 65 to 69.<br>
11.    A transgenic plant or plant seed comprising plant cells according to claim 62 or 63.<br>
72.	A transgenic plant cell callus comprising plant cells according to claim 62 or 63<br>
obtainable from a transgenic plant cell, first-generation plant, plant seed or progeny as defined in any<br>
one of claims 62, 63, or 65 to 69.       „	,<br>
73.	A plant or callus according to any one of claims claim 70 to 72 which is of a species as<br>
defined in claim 62 or 63.<br>
74.	A method of obtaining a crop product comprising harvesting a crop product from a plant<br>
according to any one of claims 70 to 73 and optionally further processing the harvested product<br>
75.	A method according to claim 74 wherein the plant is a wheat plant and the harvested<br>
crop product is grain; optionally further processed into flour or another grain product<br>
76.	A crop product obtainable by a method according to claim 74 or 75.<br>
77.	A food that comprises a protein as defined in any claim 25 or 30.<br>
78.	A food according to claim 77 in which a protein as defined in claim 25 or 30 is used<br>
instead of wild-type gluten.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUFic3RyYWN0LSgxNC0wOC0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Abstract-(14-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUFic3RyYWN0LSgyMS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Abstract-(21-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNsYWltcy0oMTQtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Claims-(14-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence Others-(07-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence Others-(14-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTktMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence Others-(19-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Correspondence Others-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence Others-(21-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence Others-(26-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence-Others-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDgtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Correspondence-Others-(08-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDgtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Correspondence-Others-(08-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Correspondence-Others-(11-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDIxLTAyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Description (Complete)-(21-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUZvcm0tMi0oMTQtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Form-2-(14-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUZvcm0tMy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Form-3-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUZvcm0tMy0oMDgtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Form-3-(08-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUZvcm0tMy0oMTEtMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Form-3-(11-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUZvcm0tMy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Form-3-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUZvcm0tMy0oMjEtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Form-3-(21-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LUZvcm0tMy0oMjYtMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Form-3-(26-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1ERUxOUC0yMDA2LUZPUk0xMy0oMTQtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7114-DELNP-2006-FORM13-(14-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMjEtMDItMjAxMyktMS5wZGY=" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Petition-137-(21-02-2013)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzExNC1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMjEtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7114-delnp-2006-Petition-137-(21-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Q2xhaW1zLSgyMS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">Claims-(21-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS0xMy0oMjEtMDItMjAxMykudGlm" target="_blank" style="word-wrap:break-word;">Form-13-(21-02-2013).tif</a></p>
		<br>
		<div class="pull-left">
			<a href="262601-a-method-for-realization-of-multifunction-numerical-line-protection-relay-adopting-a-real-time-operating-system-and-a-single-processor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="262603-intravenous-cardiac-pacing-system-with-wireless-power-supply.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>262602</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7114/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>36/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Sep-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Aug-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Nov-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BTG INTERNATIONAL LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>10 FLEET PLACE LIMEBURNER LANE LONDON EC4M 7SB UNITED KINGDOM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANDERSON ROBERT</td>
											<td>THE WALTER AND ELIZA HALL INSTITUTE OF MEDICAL RESEARCH 1G ROYAL PARADE PARKVILLE VICTORIA 3050 AUSTRALIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BEISSBATH TIM</td>
											<td>DKFZ (BO50)IM NEUENHEIMER FELD 580 69120 HEIDELBERG GERMANY</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DIN JASON TYE</td>
											<td>THE WALTER AND ELIZA HALL INSTITUTE OF MEDICAL RESEARCH 1G ROYAL PARADE PARKVILLE VICTORIA 3050 AUSTRALIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 38/16,A61P 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2005/001621</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-04-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2005900650</td>
									<td>2005-02-11</td>
								    <td>Australia</td>
								</tr>
								<tr>
									<td>2</td>
									<td>20040201774</td>
									<td>2004-04-28</td>
								    <td>Australia</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/262602-an-isolated-peptide-useful-for-diagnosing-preventing-or-treating-coeliac-disease by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:45:02 GMT -->
</html>
