<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/262907-methods-and-vectors-for-transforming-plant-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:04:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 262907:METHODS AND VECTORS FOR TRANSFORMING PLANT CELLS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS AND VECTORS FOR TRANSFORMING PLANT CELLS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods and compositions for improved bacterial-mediated plant transformation are provided. The methods generally allow plant transformation with reduced vector backbone integration and a high frequency of low-copy transformation events. Vectors for achieving these results are described, as are methods for their use.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>VECTORS AND METHODS FOR IMPROVED PLANT TRANSFORMATION<br>
EFFICIENCY<br>
BACKGROUND OF THE INVENTION<br>
This application claims the priority of U.S. provisional application Ser. No. 60/714,501, filed September 6, 2005, the entire contents of which are incorporated by reference herein.<br>
1.	Field of the Invention<br>
The invention relates generally to the field of molecular biology. More specifically, the invention relates to improved methods for plant genetic transformation and compositions for achieving the same.<br>
2.	Description of Related Art<br>
Transformation of plant cells by an Agrobacterium-mediaied method involves exposing plant cells and tissues to a suspension of Agrobacterium cells that contain certain DNA plasmids. These plasmids have been specifically constructed to contain transgenes that will express in plant cells (see, for example, U.S. Patent No. 5,034,322). Most often, one or more of the transgenes is a positive selectable marker transgene that permits plant cells to grow in the presence of a positive selection compound, such as an antibiotic or herbicide. These cells can be further manipulated to regenerate into whole fertile plants.<br>
The methods for introducing transgenes into plants by an Agrobacterium-mediated transformation method generally involve a T-DNA (transfer DNA) that incorporates the genetic elements of at least one transgene and transfers those genetic elements into the genome of a plant. The transgene(s) are typically constructed in a DNA plasmid vector and are usually flanked by an Agrobacterium Ti plasmid right border DNA region (RE!) and a left border DNA region (LB). During the process of Agrobacterium-mediated transformation, the DNA plasmid is nicked by an endonuclease, VirD2, at the right and left border regions. A single strand of DNA from between the nicks, called the T-strand, is transferred from the "Agrobacterium cell to the plant cell. The sequence corresponding to the T-DNA region is inserted into the plant genome.<br>
The integration of the T-DNA into the plant genome generally begins at the RB and continues to the end of the T-DNA, at the LB. However, the endonucleases sometimes do not nick equally at both borders. When this happens, the T-DNA that is inserted into the plant genome often contains some or all of the plasmid vector DNA. This phenomenon is referred to as "border read-through." It is usually preferred that only the transgene(s) located between the right and left border regions (the T-DNA) is transferred into the plant genome without any of the adjacent plasmid vector DNA (the vector backbone). The vector backbone DNA contains various plasmid maintenance elements, including for example, origin of replications, bacterial selectable marker genes, and other DNA fragments that are not required to express the desired trait(s) in commercial crop products.<br>
Considerable resources are directed at screening the genome of transgenic crop plants for the presence the vector backbone DNA. Methods such as polymerase chain reaction (PCR) and Southern blot analysis are most often employed to identify the extraneous vector backbone DNA. These methods are time consuming and expensive for large-scale screening work. The transgenic plants that are found to contain vector backbone DNA are generally not preferred for commercialization. Further, transgenic plants containing more than two transgencs are usually of little value for commercial development. Substantial efforts are expended regenerating plants from plant cell culture that have no commercial potential.<br>
Thus, it would be of great benefit if methods and compositions could be developed that would greatly reduce the occurrence of vector backbone DNA in the genome of transgenic plants and/or increase the frequency of low copy transformation events. Fewer transgenic plants would have to be produced if a greater number were free from vector backbone DNA and most plants have one or two copies of the transgenes, greatly increasing the efficiency of transgenic plant production.<br>
SUMMARY OF THE INVENTION<br>
In accordance with one aspect of the invention, constructs and methods are provided for improving the quality of events in bacterially-mediated plant cell transformation, such as Agrobacterium-mediated plant transformation. The invention is advantageous in providing a reduced frequency of vector backbone DNA, i.e., non-T-<br>
DNA  region,  transformation  events  and  of increasing  the   frequency of T-DNA transformation events with one or two copies of the T-DNA.<br>
The invention also provides DNA constructs for transforming plants comprising: i) at least one T-DNA border region; ii) at least one heterologous transgene adjacent to the border region; iii) a coding region for a bacterial selectable marker; and iv) at least one segment of DNA, comprising a cis and/or trans element or elements of a replication origin for maintaining a low copy number of the DNA construct in a plant cell transforming bacterium.<br>
In certain embodiments of the invention, the elements of the replication origin for maintaining low copy number of a DNA construct in a plant cell transforming (plant cell transformation competent) bacterial cell comprise one or more of repA (e.g. SEQ ID NO:32 or SEQ ID' NO:38); repB (e.g. SEQ ID NO:33 or SEQ ID NO:39); repC (e.g. SEQ ID NO:34 or SEQ ID NO:40); igsl (e.g. SliQ ID NO:35 or SEQ ID NO:41); igs2 (e.g. SEQ ID NO:36 or SEQ ID NO:42). A construct may also optionally include a palindromic sequence, for example, the 16 bp palindromic sequence of SEQ ID NO:37. These elements may be arranged in cis or in trans with respect to each other.<br>
In some embodiments, the a T-DNA border region is defined as a right border region (KB) or a left border (LB) region. Further, in certain embodiments the RB and/or LB sequences comprise SEQ ID NO:43 or SEQ ID NO:44. In particular embodiments, a heterologous transgene when expressed in a transformed plant cell provides an agronomic phenotype to the cell or transformed plant derived from the cell. In further embodiments, a coding region for a bacterial selectable marker is an antibiotic resistance gene selected from kanamycin resistance gene, gentamycin resistance gene, chloramphenicol resistance gene, spectinomycin resistance gene, streptomycin resistance gene, tetracycline resistance gene, ampicillin resistance gene, blasticidin resistance gene, hygromycin resistance gene, puromycin resistance gene, or Zeocin resistance gene.<br>
In some embodiments, a replication origin comprises a repABC sequence selected from SEQ ID NOs: 1, 2, 3, or 4. In one embodiment, the at least one segment of DNA comprises replication genes repA, repB, and repC. In some embodiments, the at least one segment of DNA comprises intergenic sequence 1 (igsl) and intergenic sequence 2 (igs2). In some embodiments, the at least one segment of DNA comprises a 16 base pair palindromic sequence. In other embodiments the segment of DNA comprises all<br>
sequences ofrepABC necessary for maintaining low copy number (1-3 copies per cell) in Agrobacterium or other plant cell transforming competent bacteria. Constructs containing repABC and methods or their use are described in more detail below.<br>
The DNA constructs of the present invention may be transferred to any cell, for example, such as a plant cell transformation competent bacterium. Such bacteria are known in the art and may, for instance, belong to the following species: Agrobacterium spp., Rhizobium spp., Sinorhizobium spp., Mesorhizobium spp., Phyllobacterium spp. Ochrobactrum spp. and Bradyrhizobium spp. Preferably, such bacteria may belong to Agrobacterium spp. The DNA constructs of the present invention may further comprise one or more replication origins for maintaining copies of the construct in E. coli. In some embodiments, origin of replication for maintaining copies of the construct in E. coli is derived from at least one of pBR322 and pUC.<br>
The present invention also relates to a plant cell transforming bacterium comprising the DNA construct of the present invention, and which may be used for transforming a plant cell. In some embodiments, the plant transforming bacteria is selected from Agrobacterium spp., Rhizobium spp., Sinorhizobium spp., Mesorhizobium spp., Phyllobacterium spp. Ochrobactrum spp. or Bradyrhizobium spp.<br>
The present invention also relates to a method for transforming a plant cell comprising: contacting at least a first plant cell with a plant cell transforming bacteria of the present invention; and selecting at least a plant cell transformed with at least one heterologous transgene. In some embodiments, the plant cell is a soybean, canola, corn, or cotton plant cell. In one embodiment, a method of the invention further comprises regenerating a plant from the plant cell.<br>
The present invention also relates to a method of producing food, feed or an industrial product comprising: obtaining the plant of the present invention or a part thereof; and preparing the food, feed or industrial product from the plant or part thereof.<br>
The invention also includes methods of genetically transforming plants with the DNA constructs of the present invention and reducing the frequency of plants transformed with non-T DNA vector region In some embodiments, the frequency of plants transformed with non-T-DNA region may be defined as less than or equal to about 20%. hi some embodiments, the frequency is less than or equal to about 15%, in some embodiments less than about 10%, and in some embodiments less than or equal to about<br>
8% or 5%. In some embodiments of the methods of the invention, the frequency of one-or two-copy T-DNA transformation events obtained is greater than or equal to about 70% or 75%. In some instances, that frequency can be raised to greater than or equal to about 80% or 85%, and in some embodiments, to greater than about 90% or 95%.<br>
In addition to a full length oriRi, the present invention encompasses plant transformation vectors containing mutants and variants of oriRi that retain substantially similar function to the full length sequence when used in the present invention. For example, applicants have identified the truncated sequence disclosed in SEQ ID NO: 3 that is suitable for use in the present invention. One of skill in the art can readily make additional changes, deletions, substitutions, etc in the oriRi sequence and screen for functional activity using the methods of the present invention. Thus, the present invention encompasses such variants and mutants.<br>
Additional features and advantages of the invention will be set forth in part in the description which follows, and in part will be obvious from the description, or may be learned by practice of the invention. The features and advantages of the invention will be realized and attained by means of the elements and combinations particularly pointed out in the appended claims.<br>
It is to be understood that both the foregoing general description and the following detailed description are exemplary and explanatory only and are not restrictive of the invention, as claimed.<br>
The accompanying drawings, which are incorporated in and constitute a part of this specification, illustrate different embodiments of the invention and together with the description, serve to explain the principles of the invention.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. The terminology used in the description of the invention herein is for describing particular embodiments only and is not intended to be limiting of the invention. As used in the description of the invention and the appended claims, the singular forms "a," "an," and "the" are intended to include the plural forms as well, unless the context clearly indicates otherwise.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 shows a plasmid map of pMON83856, the base vector with a 5.6 kb oriRi fragment used for construction of pM()N83882 for soybean transformation and pMON97352 for corn transformation.<br>
FIG. 2 shows a plasmid map of pMON83882, the testing vector with a 5.6 kb oriRi fragment and 5-enolpyruvylshikimate-3-phosphate synthase (CP4-EPSPS) selectable marker gene and the uidA (gus) gene of interest, as an example, for soy transformation.<br>
FIG. 3 shows a plasmid map of pMON83934, a base vector with a 4.2 kb oriRi fragment used for construction of pMON83937.<br>
FIG. 4 shows a plasmid map of pM()N83937, a testing vector with a 4.2 kb oriRi fragment and CP4 selectable marker gene and GOI (gus), as an example, for soy transformation.<br>
FIG. 5 shows a plasmid map of pMON97352, a test vector with a 5.6 kb oriRi fragment and CP4 selectable marker gene and GOI (gus), as an example, for corn transformation.<br>
FIG. 6 shows a plasmid map of pMON92726 which was used as a control for corn transformation. It contains the same gene structure as pMON97352 except for the replication origin. It contains ori V instead of oriRi.<br>
FIG. 7 shows a plasmid map of pMON87488, a 2 T-DNA transformation vector, which was used as a control for soybean transformation. It contains oriV instead of oriRi.<br>
FIG. 8 shows a plasmid map of pMON96001, a 2 T-DNA transformation test vector with a 4.3 kb oriRi fragment and CP4 selectable marker gene and GOI (gus), as an example, for soybean transformation.<br>
FIG. 9 shows a plasmid map of pMON96010, a 2 T-DNA transformation test vector with a 5.6 kb oriRi fragment and CP4 selectable marker gene and GOI (gus), as an example, for soybean transformation.<br>
FIG. 10 shows the effect of type of replication origin on transformation frequency in corn. Error bars represent the 95% confidence interval; * indicates a significant difference between oriRi (pMON97352) and or/T (pMON92726).<br>
FIG. 11 shows the effect of type of replication origin on backbone sequence transfer in corn transformation.    Error bars represent the 95% confidence interval; indicates a significant difference between oriRi (pMON97352) and oriV (pMON92726). oriRiloriV, RB, and LB indicates the type of backbone probe used.<br>
FIG. 12 shows the effect of type of replication origin on transgene quality in corn transformation. Error bars represent the 95% confidence interval. 0, 1,2, 3+ indicates copy number.<br>
FIG. 13 shows a plasmid map of pMON96951 containing rep ABC replication origin from Rhizobium etli CFN42 p42b plasmid and CP4 EPSPS selectable marker gene for soybean.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention will now be described by reference to more detailed embodiments, with occasional reference to the accompanying drawings. This invention may, however, be embodied in different forms and should not be construed as limited to the embodiments set forth herein. Rather, these embodiments are illustrative and provided so that this disclosure will be thorough and complete, and will fully convey the scope of the invention to those skilled in the art.<br>
The present invention relates, in part, to the discovery that the repABC origin of replication, such as the one from Agrobacterium rhizogenes pRi ("oriRi" or "oriPRi"), which maintains 1-3 copies in bacteria such as Agrobacterium, can be used in plant transformation vectors and can impart highly desirable transformation events. The vectors of the invention can, for example, significantly reduce the frequency of transformation events that transfer non-T-DNA region, i.e., vector backbone DNA, and can, for example, significantly increase the number of one- or two-copy T-DNA, i.e., the gene of interest, transformation events. The prior art does not teach or suggest the use of oriRi to achieve these unexpected transformation results. The vectors and methods of the invention improve transformation events in the preparation of transgenic crop plants.<br>
As used herein, a transgenic crop plant contains an exogenous polynucleotide molecule or a heterologous transgene inserted into the genome of a crop plant cell. A crop plant cell, includes without limitation, a plant cell, and further comprises suspension cultures, embryos, meristematic regions, callus tissue, leaves, roots, shoots,<br>
25693539.1<br>
gametophytes, sporophytes, ovules, pollen and microspores, and seeds, and fruit. By "exogenous" or "heterologous" it is meant that a polynucleotide molecule which originates from outside the plant cell into which the polynucleotide molecule is introduced. An exogenous polynucleotide molecule can have a naturally occurring or non-naturally occurring nucleotide sequence. One skilled in the art understands that an exogenous polynucleotide molecule can he a heterologous molecule derived from a different species of any other organism or the plant species than the plant into which the polynucleotide molecule is introduced or can be a polynucleotide molecule derived from the same plant species as the plant into which it is introduced.<br>
The exogenous polynucleotide (heterologous transgene) when expressed in a transformed plant cell provides an agronomic trait to the cell or to a transformed plant derived from the cell. These genes of interest (GOI) provide beneficial agronomic traits to crop plants, for example, including, but not limited to genetic elements comprising herbicide resistance (U.S. Pat. No. 5,633,435; U.S. Pat. No. 5,463,175), increased yield (U.S. Pat. No. 5,716,837), insect control (U.S. Pat. No. 6,063,597; U.S. Pat. No. 6,063,756; U.S. Pat. No. 6,093,695; U.S. Pat. No. 5,942,664; U.S. Pat. No. 6,110,464), fungal disease resistance (U.S. Pat. No. 5,516,671; U.S. Pat. No. 5,773,696; U.S. Pat. No. 6,121,436; and U.S. Pat. No. 6,316,407; and U.S. Pat. No. 6,506,962), virus resistance (U.S. Pat. No. 5,304,730 and U.S. Pat. No. 6,013,864), nematode resistance (U.S. Pat. No. 6,228,992), bacterial disease resistance (U.S. Pat. No. 5,516,671), starch production (U.S. Pat. No. 5,750,876 and U.S. Pat. No. 6,476,295), modified oils production (U.S. Pat. No. 6,444,876), high oil production (U.S. Pat. No. 5,608,149 and U.S. Pat. No. 6,476,295), modified fatty acid content (U.S. Pat. No. 6,537,750), high protein production (U.S. Pat. No. 6,380,466), fruit ripening (U.S. Pat. Mo. 5,512,466), enhanced animal and human nutrition (U.S. Pat. No. 5,985,605 and U.S. Pat. No. 6,171,640), biopolymers (U.S. Pat. No. 5,958,745 and U.S. Patent Pub 2003/0028917), environmental stress resistance (U.S. Pat. No. 6,072,103), pharmaceutical peplides (U.S. Pat. No. 6,080,560), improved processing traits (U.S. Pat. No. 6,476,295), improved digestibility (U.S. Pat. No. 6,531,648) low raffmose (U.S. Pat. No. 6,166,292), industrial enzyme production (U.S. Pat. No. 5,543,576), improved flavor (U.S. Pat. No. 6,011,199), nitrogen fixation (U.S. Pat. No. 5,229,114), hybrid seed production (U.S. Pat. No. 5,689,041), and biofuel production (U.S. Pat. No. 5,998,700), the genetic elements and transgenes described in the patents listed above are herein incorporated by reference.<br>
The present invention provides plant recombinant DNA constructs for producing transgenic crop plants. Methods that are well known to those skilled in the art may be used to prepare the crop plant recombinant DNA construct of the present invention. These methods include in vitro recombinant DNA techniques, synthetic techniques, and in vivo genetic recombination. Such techniques are described in Sambrook et al. (1989). Exogenous polynucleotide molecules created by the methods may be transferred into a crop plant cell by a plant transforming bacterium such as Agrobacterium or by other methods known to those skilled in the art of plant transformation. Gene transfer to plants has been shown using diverse species of bacteria as described in Broothaerts et al. (2005) and the US provisional application 60800872, herein incorporated by reference. The recombinant DNA constructs of the present invention may be transferred into a crop plant cell by transformation using Sinorhizobium meliloti, Rhizobium sp., Mesorhizobium loti, or any other bacterium capable of transforming a plant cell.<br>
The invention includes plant transformation vectors for use in Agrobacterium-mediated transformation of plants. The vectors of the present invention are generally plasmids, but the oriRi replication origin may be replaced by other replication origins from the rep ABC family for maintaining very low copy number (1-3 copies per cell) in Agrobacterium. Also, vectors without an Agrobacterium-maintaining replication origin are capable of maintaining themselves in Agrobacterium with sequences in the T-DNA that can be integrated into either the Ti plasmid in A. tumefaciens, Ri plasmid in A. rhizogenes or Agrobacterium chromosome through homologous recombination. This results in the same effect of a reduction in non T-DNA sequence insertion and an increase in low copy number transformation events, since the chromosome, Ti or Ri plasmid is present in a single copy. When Rhizobia or non-Agrobacterium species are used for plant transformation, T-DNA containing GOI can be integrated into the chromosome or into repABC plasmids in the host bacteria, which can produce the same effect as Agrobacterium does.<br>
A vector of the present invention may comprise at least one of Agrobacterium Ti plasmid right border or left border region. The vectors may comprise at least one pair of borders. The vectors can include four pairs of borders, but the vector often comprise one or two pairs.<br>
The vectors of the present invention may further comprise a coding region for a selectable marker for the maintenance in bacterial hosts. Coding regions for selectable<br>
markers include Spec/Strp that encodes for Tn7 aminoglycoside adenyltransferase (aadA) conferring resistance to spectinomycin or streptomycin, or a gentamycin (Gm, Gent) selectable marker gene. Other resistance genes include carbenecillin, ampicillin, and kanamycin resistance genes. Others are known and may be readily used in the present invention by those of skill in the art.<br>
The vectors of the present invention may also comprise a coding region for a plant selectable marker gene, which is typically located in T-DNA, to select transformed plant cells with the corresponding reagent. The plant selectable marker may provide resistance to a positive selection compound, for example, antibiotic resistance (e.g., kanamycin, G418, bleomyciri, hygromycin, etc.), or herbicide resistance (e.g., including but not limited to: glyphosate, Dicamba, glufosinatc, sulfonylureas, imidazolinones, bromoxynil, dalapon, cyclohexanedione, protoporphyrinogen oxidase inhibitors, and isoxaflutole herbicides). Polynucleotide molecules encoding proteins involved in herbicide tolerance are known in the art, and include, but are not limited to a polynucleotide molecule encoding a) tolerance to a glyphosate include 5-enolpyruvylshikimate-3-phosphate synthases (EPSPS; U.S. Patent 5,627,061, U.S. Patent 5,633,435, U.S. Patent 6,040,497, U.S. Patent 5,094,945, WO04074443, and WO04009761), glyphosate oxidoreductase (GOX; U.S. Patent 5,463,175), glyphosate decarboxylase (WO05003362 and U.S. Patent Application 20040177399), and glyphosate-N-acetyl transferase (GAT; U.S. Patent publication 20030083480) conferring tolerance to glyphosate; b) dicamba monooxygenase (DMO, encoded by ddmC) conferring tolerance to auxin-like herbicides such as dicamba (U.S. Patent Applications 20030115626, 20030135879; Herman et ai, 2005); c) phosphinothricin acetyltransferase (bar) conferring tolerance to phosphinothricin or glufosinate (U.S. 5,646,024, U.S. 5,561,236, EP 275,957; U.S. 5,276,268; U.S. 5,637, 489; U.S. 5,273, 894); d) 2,2- dichloropropionic acid dehalogenase conferring tolerance to 2,2-dichloropropionic acid (Dalapon) (WO9927116); e) acetohydroxyacid synihase or acetolactate synthase conferring tolerance to acetolactate synthase inhibitors such as sulfonylurea, imidazolinone, triazolopyrimidine, pyrimidyloxybenzoates and phthalide (U.S. 6,225,105; U.S. 5,767,366, U.S. 4,761,373; U.S. 5,633,437; U.S. 6,613,963; U.S. 5,013,659; U.S. 5,141,870; U.S. 5,378,824; U.S. 5,605,011); 0 haloarylnitrilase (Bxn) for conferring tolerance to bromoxynil (WO8704181A1; U.S. 4,810,648; WO8900193A); g) modified acetyl-coenzyme A carboxylase for conferring tolerance to cyclohexanedione<br>
(sethoxydim) and aryloxyphenoxypropionate (haloxyfop) (U.S. 6,414,222); h) dihydropteroate synthase (sulI) for conferring tolerance to sulfonamide herbicides (U.S. 5,597,717; U.S. 5,633,444; U.S. 5,719,046); i) 32 kD photosystem II polypeptide (psbA) for conferring tolerance to triazine herbicides (Hirschberg et al., 1983); j) anthranilate synthase for conferring tolerance to 5-methyl tryptophan (U.S. 4,581,847); k) dihydrodipicolinic acid synthase (dapA) for conferring to tolerance to aminoethyl cysteine (WO8911789); 1) phytoene desaturase (crtl) for conferring tolerance to pyridazinone herbicides such as norflurazon (JP06343473); m) hydroxy-phenyl pyruvate dioxygenase for conferring tolerance to cyclopropyhsoxazole herbicides such as isoxaflutole (WO 9638567; U.S. 6,268,549); n) modified protoporphyrinogen oxidase I (protox) for conferring tolerance to protoporphyrinogen oxidase inhibitors (U.S. 5,939,602); and o) aryloxyalkanoate dioxygenase (AAD-1) for conferring tolerance to an herbicide containing an aryloxyalkanoate moiety (WO05107437). Examples of such herbicides include phenoxy auxins (such as 2,4-D and dichlorprop), pyridyloxy auxins (such as fluroxypyr and triclopyr), aryloxyphenoxypropionates (AOPP) acetyl- coenzyme A carboxylase (ACCase) inhibitors (such as haloxyfop, quizalofop, and diclofop), and 5-substituted phenoxyacetate protoporphyrinogen oxidase IX inhibitors (such as pyraflufen and flumiclorac).<br>
In addition to a plant selectable marker, in some embodiments it may be desirable to use a reporter gene. In some instances, a reporter gene may be used with or without a selectable marker. Reporter genes are genes that are typically not present in the recipient organism or tissue and typically encode for proteins resulting in some phenotypic change or enzymatic property. Examples of such genes are provided in Wising et al. (1988), that is incorporated herein by reference. Preferred reporter genes include the beta-glucuronidase (GUS) of the uidA locus of E. coli, the chloramphenicol acetyl transferase gene from Tn9 of E. coli, the green fluorescent protein from the bioluminescent jellyfish Aequorea victoria, and the luciferase genes from firefly Photinus pyralis. An assay for detecting reporter gene expression may then be performed at a suitable time after said gene has been introduced into recipient cells, A preferred such assay entails the use of the gene encoding beta-glucuronidase (GUS) of the uidA locus of E. coli as described by Jefferson et al. (1987) to identify transformed cells, referred to herein as GUS.<br>
In some embodiments, the vectors of the present invention comprise one replication origin for maintenance in E. coli. These origins of replication may be derived,<br>
for example, from pBR322 or from pUC. One example of such an origin of replication is Co/El. The vectors of the present invention can include any origin of replication which maintains at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, or 100 or more plasmid copies in E. coli. In some embodiments, the ori maintains a higher copy number in E. coli (e.g., greater than 5, 10, 15, 20, 25, or 30, and in some further embodiments, greater than 10, 15, 20, 25 or 30). Such high copy number vectors make it easier to amplify the amount of DNA available for transformation. Thus, in some embodiments, the present invention includes a vector containing an ori that provides high copy numbers in E. coli cells, and a second ori, such as an ori belonging to the repABC family, for example, on'/?/, that maintains extremely low copy number, for example 1-3, in Agrobacterium.<br>
The vectors of the present invention also comprise at least one segment of DNA, comprising cis and/or trans elements, which is necessary and sufficient for low-copy plasmid maintenance (1-3 copies) in a plant transforming bacterium such as Agrobacterium. The at least one segment of DNA preferably comprises an origin of replication from the repABC family. Further, the at least one segment of DNA may include a Ri plasmid replication origin of Agrobacterium rhizogenes (oriRi) and/or repABC origin from plasmid p42b of Rhizobium etli. Still further, the repABC family preferably comprises replication genes rep A, repB, and repC. The at least one segment also may comprise intergenic region sequences, igsl and igs2. Still further, the at least one segment of DNA may include a 16-bp palindrome after repC.<br>
As used herein, the "origin of replication" of a "repABC" plasmid, i.e. a plasmid that utilizes such sequences for replication and partitioning in a cell, may be defined as comprising repA (e.g. SEQ ID NO:32 from pRi of A. rhizogenes; SEQ ID NO:38 from p42b of R. etli), repB (e.g. SEQ ID NO: SEQ ID NO:33 from A. rhizogenes; SEQ ID NO:39 from p42b of R. etli), repC (e.g. SEQ ID NO: SEQ ID NO:34 from A. rhizogenes; SEQ ID NO:40 from p42b of R. etli),igs] (e.g. SEQ ID NO:35 from A. rhizogenes; SEQ ID NO:41 from p42b of R. etli)), igs2 (e.g. SEQ ID NO:36 from A. rhizogenes; SEQ ID NO:42 from p42b of R. etli), and may optionally comprise a palindromic sequence (e.g. SEQ ID NO:37). In other embodiments, the origin of replication may comprise those sequences required for maintaining low copy number (for example, from 1 to about 3 copies) within a plant-cell-transforming bacterial cell. Examples of such repABC origins<br><br>
of replication are found in SEQ ID NOs:l-4, in the oriRi of the Ri plasmid of A. rhizogenes, and in plasmid p42b of R, etli, among others.<br>
In one embodiment, the at least one segment is as identified in SEQ ID NO:I. In other embodiments, the at least one segment consists essentially of the following sequence: 5618 bp oriRi of pMON83882 bases (633-6250) identified as SEQ ID NO:2. In another embodiment, the at least one segment comprises and/or consists essentially of the sequence identified in SEQ ID NO:3. In yet another embodiment, the at least one segment may be repABC from p42b from R. etli as identified in SEQ ED NO:4.<br>
Encompassed within the definition of any of the nucleic acid or polypeptide sequence defined herein are sequences which exhibit a specified degree of identity to such sequences. For example, it should be noted that while specific nucleic acid sequences are set forth herein, modifications may be made to the described sequences without departing from the scope of the invention. In particular, nucleic acid sequences which are substantially identical to those set forth herein may be used with the present invention. A first nucleic acid sequence displays "substantial identity" to a reference nucleic acid sequence if, when optimally aligned (with appropriate nucleotide insertions or deletions totaling less than 20 percent o f the reference sequence over the window of comparison) with the other nucleic acid (or its complementary strand), there is at least about 75% nucleotide sequence identity, preferably at least about 80% identity, more preferably at least about 85%, 86%, 87%, 88%, or 89% identity, and most preferably at least about 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99% identity over a comparison window of at least 200 nucleotide positions, preferably at least 300 nucleotide positions, more preferably at least 400 nucleotide positions, and most preferably over the entire length of the first nucleic acid. Optimal alignment of sequences for aligning a comparison window may be conducted by the similarity method of Pearson and Lipman (1988); preferably by computerized implementations of these algorithms in the Wisconsin Genetics Software Package Release 7.0, Genetics Computer Group, Madison, WI. The reference nucleic acid may be a full-length molecule or a portion of a longer molecule.<br>
Alternatively, two nucleic acids have substantial identity if one hybridizes to the other under stringent hybridization conditions. The term "stringent hybridization conditions" is defined as conditions under which a test sequence hybridizes specifically with a target sequence(s) but not with non-target sequences, as can be determined<br>
empirically. The term "stringent conditions" is functionally defined with regard to the hybridization of a nucleic-acid probe to a target nucleic acid (i.e., to a particular nucleic acid sequence of interest) by the specific hybridization procedure (see for example Sambrook et al., 1989, at 9.52-9.55, Sambrook et al, 1989 at 9.47-9.52, 9.56-9.58; Kanehisa, 1984; and Wetmur and Davidson, 1968). Appropriate stringent conditions for DNA hybridization are, for example, 6.0x sodium chloride/sodium citrate (SSC) at about 45°C, followed by a wash of 0.2x SSC at 50°C. Details of such methods are known to those skilled in the art or can be found in laboratory manuals including but not limited to Current Protocols in Molecular Biology (1989). In some embodiments, lower stringency conditions may be used by changing the wash to about 2.0x SSC at 50°C. In addition, the temperature in the wash step can be increased from low stringency conditions at room temperature, about 22°C, to higher stringency conditions at about 65°C. Both temperature and salt may be varied, or either the temperature or the salt concentration may be held constant while the other variable is changed. For example, hybridization using DNA or RJSfA probes or primers can be performed at 65°C in 6x SSC, 0.5% SDS, 5x Denhardt's, 100 µg/mL nonspecific DNA (e.g., sonicated salmon sperm DNA) with washing at 0.5x SSC, 0.5% SDS at 65°C, for high stringency.<br>
A nucleic acid molecule is said to be the "complement" of another nucleic acid molecule if they exhibit complete complementarity. As used herein, molecules are said to exhibit "complete complementarity" when every nucleotide of one of the molecules is complementary to a nucleotide of the other. Two molecules are said to be "minimally complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under at least conventional "low stringency" conditions. Similarly, the molecules are said to be "complementary" is they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under conventional "high stringency" conditions. It is contemplated that lower stringency hybridization conditions such as lower hybridization and/or washing temperatures can be used to identify related sequences having a lower degree of sequence similarity if specificity of binding of the probe or primer to target sequence(s) is preserved. Accordingly, the nucleotide sequences of the present invention can be used for their ability to selectively form duplex molecules with complementary stretches of DNA fragments. Detection of DNA segments via hybridization is well-known to those of skill in the art, and thus depending on the application envisioned, one will desire to<br>
employ varying hybridization conditions lo achieve varying degrees of selectivity of probe towards target sequence and the method of choice will depend on the desired results. Conventional stringency conditions are described in Sambrook, et al. (1989) and byUaymesetal. (1985).<br>
The DNA construct of the present invention may be introduced into the genome of a desired plant host by a suitable Agrobacterium-mediated plant transformation method. Methods for transforming plants by Agrobacterium tumefaciens-medialed transformation include: Fraley et al., (1985), and Rogers et al., (1987). Agrobacterium-mediaied transformation is achieved through the use of a genetically engineered soil bacterium belonging to the genus Agrobacterium. Several Agrobacterium species mediate the transfer of a specific DNA known as "T-DNA," that can be genetically engineered to carry any desired piece of DNA into many plant species. The major events marking the process of T-DNA mediated pathogenesis are induction of virulence genes, and processing and transfer of the T-strand. This process is the subject of many reviews (Ream, 1989; Howard and Citovsky, 1990; Kado, 1991; Winans, 1992; Zambryski, 1992; Gelvin, 1993; Binns and Howitz, 1994; Hooykaas and Beijersbergen, 1994; Lessl and Lanka, 1994; Zupan and Zambryski, 1995). Non-Agrobacterium species such as Sinorhizobium meliloti, Rhizobium sp., Mesorhizobium loti, may also be used for transferring genes using the DNA constructs of the present invention (Broothaerts et al., 2005 and the US provisional application 60/800,872, herein incorporated by reference). Other methods for introducing DNA into cells may also be used. These methods are well known to those of skill in the art and can include to chemical methods and physical methods such as microinjection, electroporation, and micro-projectile bombardment.<br>
Plant cell regeneration techniques rely on manipulation of certain phytohormones in a tissue culture growth medium, also typically relying on a biocide and/or herbicide marker that has been introduced together with the desired nucleotide sequences. Choice of methodology with suitable protocols being available for hosts from Leguminosae (alfalfa, soybean, clover, etc.), Umbelliferae (carrot, celery, parsnip), Cruciferae (cabbage, radish, canola/rapeseed, etc.), Cucurbitaceae (melons and cucumber), Gramineae (wheat, barley, rice, maize, etc.), Solanaceae (potato, tobacco, tomato, peppers), various floral crops, such as sunflower, and nut-bearing trees, such as almonds, cashews, walnuts, and pecans. See, for example, Ammirato et al. (1984); Shimamoto et al. (1989); Fromm (1990); Vasil et al., (1990); Vasil et al. (1992); Hayashimoto (1990);<br>
and Datta et al. (1990). Such regeneration techniques are described generally in Klee et al. (1987). Methods and compositions for transforming plants by introducing a transgenic DNA construct into a plant genome in the practice of this invention can include any of the well-known and demonstrated methods. For example, Agrobacterium-mediated transformation as illustrated in U.S. Pat. Nos. 5,824,877; 5,591,616; and 6,384,301, all of which are incorporated herein by reference.<br>
Plants that can be made by practice of the present invention include any plants that are subject to transformation and regeneration and include, but are not limited to, Acacia, alfalfa, aneth, apple, apricot, artichoke, arugula, asparagus, avocado, banana, barley, beans, beet, blackberry, blueberry, broccoli, brussels sprouts, cabbage, canola, cantaloupe, carrot, cassava, cauliflower, celery, Chinese cabbage, cherry, cilantro, citrus, Clementines, coffee, corn, cotton, cucumber, Douglas fir, eggplant, endive, escarole, eucalyptus, fennel, figs, forest trees, gourd, grape, grapefruit, honey dew, jicama, kiwifruit, lettuce, leeks, lemon, lime, Loblolly pine, mango, melon, mushroom, nut, oat, okra, onion, orange, an ornamental plant, papaya, parsley, pea, peach, peanut, pear, pepper, persimmon, pine, pineapple, plantain, plum, pomegranate, poplar, potato, pumpkin, quince, radiata pine, radicchio, radish, raspberry, rice, rye, sorghum, Southern pine, soybean, spinach, squash, strawberry, sugarbeet, sugarcane, sunflower, sweet potato, sweetgum, tangerine, tea, tobacco, tomato, turf, a vine, watermelon, wheat, yams, and zucchini. In. preferred embodiments, the plant is a soybean, com, canola, or cotton plant. In particular embodiments, the plant is a corn plant. In particular embodiments, the plant is a soybean plant. In other embodiments, the plant is a cotton plant. And in still further embodiments, the plant is a canola plant.<br>
The present methods yield improved transformation efficiencies, significantly reducing the frequency of plants transformed with vector backbone DNA, or "non-T-DNA." In some embodiments, the frequency of plants transformed with non-T DNA is less than or equal to about 20%. In some embodiments, the frequency of plants transformed with non-T DNA is less than or equal to about 15%, or less than equal to about 10%, or less than or equal to about 8% or 5%. Additionally, the methods of the invention yield very high one- or two-copy T-DNA transformation events. For example, in some embodiments, the frequency of one- or two-copy T-DNA transformation events is greater than or equal to about 70% or 75%, as measured using Southern blotting. In<br>
some embodiments, that frequency is greater than or equal to about 80% or 85%, and in some cases, 90% or 95%, as measured with Southern blotting.<br>
In some embodiments, the frequency of plants transformed with non-T DNA vector region is aibout 50%, 40%, 20%, 15%, or 10% lower than the frequency of T-DNA found in the same variety of plant that has been transformed with a plant transformation vector that does not contain the repABC element such as oriRi element of the present invention. In other embodiments, the frequency of one- or two-copy T-DNA transformation events is about 50%, 40%, 20%, 15%, or 10% higher than the frequency of one- or two-copy T-DNA transformation events relative to the same variety of plant transformed with plant transformation vector that does not contain the repABC element such as oriRi element of the present invention.<br>
Thus, certain embodiments of the present invention provide methods involving use of repABC element such as oriRi from Agrobacterium rhizogenes and repABC origin from plasmid p42b of Rhizobium etli, including the sequences set forth herein, to reduce the frequency of plants transformed with non-T-DNA vector region. The present invention is also directed to the use of repABC element such as oriRi from Agrobacterium rhizogenes and repABC origin from plasmid p42b of Rhizobium etli, including the sequences set forth herein, to increase the frequency of one- or two-copy T-DNA transformation events. In some embodiments, both methods are employed in combination to achieve transformation events having lower frequencies of backbone incorporation and higher frequencies of one- or two-copy T-DNA transformation events.<br>
EXAMPLES<br>
The following examples are provided to better elucidate the practice of the present invention and should not be interpreted in any way to limit the scope of the present invention. Those skilled in the art will recognize that various modifications, additions, substitutions, truncations, etc., can be made to the methods and genes described herein while not departing from the spirit and scope of the present invention.<br>
Example 1<br>
Preparation of Vectors<br>
Cloning steps followed standard protocols described by Sambrook et al. (1989). The 5.6 kb oriRi fragment, excised from pCGN1589 with Oral digestion, was used to replace the oriV fragment of pMON67438 that was digested with PshAI and BstXI and<br>
blunted with T4 DNA polymerase. This resulted in an oriRi base vector pMON83856<br>
(FIG. 1). The 5.6 kb oriRi testing vector pMON83882 (FIG. 2) was made by insertion of<br>
the gus and CP4 fragments from the oriV control vector pMON67438, sequentially<br>
digested with AccI (blunted)/BamHI, into pMON83856 that had been opened with<br>
Pmel/BamHI.	To truncate the oriRi fragment, two primers, 5'<br>
CACGTGTACAAGGTAGAATCCGCCTGAG 3' (oriRi 5' promoter upstream; SEQ ID NO:5) and 5' GTATACAGGCTCTCCTTCACGATCAAC 3' (oriRi 3' after repC: SEQ ID NO:6 ), were synthesized and PCR was performed with high fidelity pfu polymerase and pMON83856 as a template. The PCR product was purified and inserted into pMON83930, which was digested with Afel and Xhol and blunted with T4 DNA polymerase, to replace oriV fragment. The resulting vector pMON83934 (FIG. 3), with the truncated oriRi, was then confirmed by DNA sequencing. In order to make the testing vector pMON83937 (FIG. 4) with a truncated oriRi, pMON83934 was opened with Kpnl, blunted with T4 DNA polymerase, and followed by BamHI digestion; the gus and Cp4 fragment from oriV control vector pMON67438 was sequentially digested with Accl, filled in with T4 DNA polymerase, and then further digested with BamHI. The two fragments were ligated with T4 DNA ligase and resulted in pMON83937.<br>
To construct the corn transformation vector pMON97352 (FIG. 5), a linker sequence containing Spel and PspOMI sites were synthesized by annealing two oligos 5'-AGCTTGGGCCCCTCGAGGCTAGCACTAGTG-31 (SEQ ID NO:7 and 5'-GATCCACTAGTGCTAGCCTCGAGGGGCCCA-31 (SEQ ID NO: 8), and inserted into pMON83856 with the BamHI and Hindlll digestion. The resulting intermediate oriRi vector was opened with Spel and PspOMI and ligated with the CP4 and gus expression cassettes excised from pMON92726 (Fig. 6) with NotI and Spel digestion (see FIG. 5, pMON97352). The oriV parental vector pMON992726 was used as a control vector to compare with com oriRi test vector pMON97352.<br>
To construct oriV 21 control vector pMON87488 (FIG. 7), the octopine right border was excised from pMON51676, blunted with T4 DNA polymerase and inserted into pMON87485 which was opened with Sall/Spel, filled-in with T4 DNA polymerase and CIP treated. To make the oriRi 2T vectors, pMON96001 (FIG 8) and pMON96010 (FIG. 9), the oriV in pMON87488 was removed with Afel and Xhol, the vector was filled-in with T4 DNA polymerase and inserted with the 4.2 kb oriRi replicon from pMON83934 obtained by digesting pMON83934 with Pmll and BstZ17I, resulting into<br>
pMON96001.   The 5.6 kb oriRi replicon from pCGNl589 obtained by digesting it with Dral digestion was inserted to yield pMON96010.<br>
EXAMPLE 2 Soybean Crop Transformation<br>
Transformation of soybean cells and regeneration of the cells into intact fertile plants by Agrobacterium-mediated transformation can be conducted using various methods known in the art. A method for soybean transformation (e.g. U.S. Patent Nos. 6,384,301 and 7,002,058) was utilized with an organogenesis process.<br>
The DNA constructs described in the present invention (e.g., plasmid pMON83882, substantially as shown in FIG. 2, and plasmid pMON83937, substantially as shown in FIG. 4) are transformed into a disarmed Agrobacterium strain ABI. The two oriV control vectors, pMON67438 and pMON83898 (same GOI but in different orientations), were also transferred into Agrobacterium cells. Two T DNA constructs and respective controls as described above were also used for transforming soybean. The DNA construct was transferred into Agrobacterium by electroporation. Single colonies were recovered on LB medium with spectinomycin 50 mg/1 (for oriRi vectors) or 75 mg/1 (for oriV vectors) and kanamycin 50 mg/1 and inoculated in 20-50 ml liquid LB medium with same selection in a shaker with 200 ipm. The plasmid in the Agrobacterium was verified by restriction enzyme digestion of mini-prepared plasmid from 10 ml culture. The remainder of the liquid culture was mixed with glycerol to a final concentration of 20%, aliquoted and stored at -80 C as seed cultures.<br>
To prepare the Agrobacterium inoculum for transformation, 0.25-1 ml frozen seed culture was inoculated into 250 or 500 ml LB medium with same antibiotic selection as above-mentioned and grown overnight at 26°C-28°C with shaking at 200 rpm to mid-log growth phase. The culture was spun down and directly suspended in an inoculation medium (INO medium) at the concentration of OD600 about 0.3, as measured on a spectrophotometer.<br>
Soybean cultivar A3525 was used for Agrobacterium-mediated transformation as described (e.g. U.S. Patent 7,002,058). Soybean seeds were germinated for less than 14 hours at room temperature in BGM medium and the meristem explants from soy mature seeds were excised by machine as described in the US Patent Application 2005000532!. For batch sonication, about 100 explants in PLANTCON lid were mixed with 20 ml of<br>
the Agrobacterium suspension in INO medium and sonicated in W-l 13 Sonicator for 20 seconds. For bulk sonication, about 1000 explants in plantcon lid were mixed with 100 ml of the Agrobacterium suspension in INO medium and sonicated for 20 seconds. After sonication, the explants were co-cultured in the same PLANTCON for 2-4 days at 23°C with 16/8 hour light/dark period. Then the explants were transferred onto the surface of the WPM selection medium with 75 ^iM glyphosate. Each PLANTCON contained about 50 explants. After 2 weeks, explants were transferred again to 75 uM glyphosate solid WPM medium with primary radicle inserted in medium, each PLANTCON containing about 25 explants. Shoots with fully expanded trifolia recovered after 6-10 weeks post-inoculation were rooted in BRM medium with IAA 0.1 mg/1 and 25 |uiM glyphosate selection. The rooted plantlets were transferred to greenhouse for maturity. Various media used for soybean transformation is detailed below in Table 1.<br>
The above transformation and regeneration methods provides for plants that are greatly reduced in the occurrence of vector backbone DNA. Additionally, the plants have an added benefit of having reduced copy number of the insert T-DNA. The plants produced by the method are an aspect of the invention. <br>
TABLE 1 Media components for soybean transformation  (Table Removed)<br>
Other dicot plant cells can be transformed and regenerated into intact plants by methods known in the art of plant transformation and tissue culture. The use of Agrobacterium-mediated methods to transfer the T-DNA of the plasmids of the present<br>
invention are well known in the art. For example cotton (U.S. Pat. Nos. 5,004,863; 5,159,135; 5,518,908, 5,846,797, and 6,624,344, herein incorporated by reference) and Brassica (US Pat. No. 5,750,871).<br>
Example 3 Corn Crop Transformation<br>
Two vectors, pMON97352 (oriRi, single copy in Agrobacterium) and pMON92726 (oriV, multiple copies in Agrobacterium as a control) containing same plant selectable marker gene CP4 and gus gene cassettes, both driven by rice actin promoters (FIG. 5), were electroporated into Agrobacterium tumefaciens strain ABI for corn transformation. Agrobacterium containing the vector in glycerol stock was streaked out on solid LB medium supplemented with antibiotics (all in active ingredient) kanamycin (40 mg/L), spectinomycin (31 mg/L), streptomycin (38 mg/L) and chloramphenicol (25 mg/L) and incubated at 28°C for 2 days. Two days before Agrobacterium inoculation of the maize immature embryos, one colony or a small loop of Agrobacterium from the Agrobacterium plate was picked up and inoculated into 25 mL of liquid LB medium supplemented with 62 mg/L of spectinomycin and 40 mg/L of kanamycin in a 250 mL flask. The flask was placed on a shaker at approximately 150-200 rpm and 27-28°C overnight. The Agrobacterium culture was then diluted (1 to 5) in the same liquid medium and put back on the shaker. Several hours later, one day before inoculation, the Agrobacterium cells were spun down at 3500 rpm for 15 min. The bacterium cell pellet was re-suspended in induction broth with 200 µM of acetosyringone and 50 mg/L spectinomycin and 25 mg/L kanamycin and the cell density was adjusted to 0.2 at OD660-The bacterium cell culture (50 mL in each 250 mL flask) was then put back on the shaker and grown overnight. On the morning of inoculation day, the bacterium cells were spun down and washed with liquid 1/2 MS VI medium (Table 2) supplemented with 200 µM of acetosyringone. After one more centrifugation, the bacterium cell pellet is re-suspended in 1/2 MS PL medium (Table 2) with 200 µM of acetosyringone (Table 2) and the cell density was adjusted to 1.0 at OD660 for inoculation.<br>
Reagents are commercially available and can be purchased from a number of suppliers (see, for example Sigma Chemical Co., St. Louis, Mo).  <br>
Table 2 Media used in Example 3   (Table Removed)<br>
Corn line LH244 (U.S. Patent No. 6,252,148) was used in this study. Ears containing immature embryos were harvested and kept refrigerated at 4°C until use. Immature embryos were isolated from surface sterilized ears and directly dropped into the prepared Agrobacterium cell suspension in microcentrifuge tube and inoculated for 5 to 20 min. After Agrobacterium cell suspension was removed using a fine tipped sterile transfer pipette, the immature embryos were transferred onto the co-culture medium (Table 2). The embryos were cultured in a dark incubator (23°C) for approximately 24 h.<br>
After the co-cultivation, the embryos were transferred onto a modified MS medium (Induction MS, Table 2) supplemented with 500 mg/L carbenicillin and 0.1 mM glyphosate at 30°C for 2 weeks followed by an additional week in a dark culture room at 27-28°C. All the callus pieces were then transferred individually onto the first regeneration medium, the same medium mentioned above except 2,4-D and picloram were replaced by 3.5 mg/L BAP (MS/6BA, Table 2) and the carbenicillin level was dropped to 250 mg/L. The cultures were moved to a culture room with 16-h light/8-h dark photoperiod and 27°C. After 5-7 days, the callus pieces were transferred onto the second regeneration medium (MSOD, Table 2). hi another 2 weeks, the callus pieces that had shoots regenerated were transferred onto the same hormone-free medium in<br>
Phytatrays™ for further growth. Regenerated plants (RO) with one or more healthy roots were moved to soil in peat pots in a growth chamber. In 7 to 10 days, they were transplanted into 12-inch pots and moved to a greenhouse with conditions for normal corn plant growth. The plants were either self-pollinated or crossed with wild-type plants.<br>
EXAMPLE 4 Molecular Analysis for Backbone DNA and Copy Number<br>
A.        Soybean transformed with One T-DNA Constructs<br>
DNA was extracted from tissue samples collected from greenhouse grown plants transformed with the DNA plasmids of the present invention. A PCR-based method was used to assay the DNA for the presence of the oriRi sequence using repA forward primer 5'-ACAAGGTAGAATCCGCCTGAG-3' (Xd487b; SEQ ID NO: 11) and repA reverse primer 5'-TTCAACTCTGGCATCTCAGAC-3' (Xd488; SEQ ID NO: 12) as an indicator of vector backbone. This DNA is adjacent to the LB and its presence in the DNA extracted from the regenerated plants indicates that transfer of vector sequences beyond the LB has occurred. DNA can be isolated from plant tissues by any number of methods for example, the CTAB procedure (Rogers et al., 1985) or DNAeasy™ 96 Plant Kit (Cat. # 69181, Qiagen Inc., Valencia, Calif.) following the manufacturers instructions. Taqman® (PE Applied Biosystems, Foster City, Calif.) is described as a method of detecting and quantifying the presence of a DNA sequence and is fully understood in the instructions provided by the manufacturer. DNA primer molecules (SEQ ID NO: 11 and 12) were used in the described method to identify the oriRi DNA from plant extracts. The conditions and apparatus used can be modified by those skilled in the art to provide the same results.<br>
Soybean meristem axis excised from mature seeds were transformed with a control DNA plasmid (pMON67438 or pMON83898) and a plasmid of the present invention (pMON83882 or pMON83937), then regenerated into intact plants. The intact plants were analyzed for the presence of the gene of interest (CP4) and non T-DNA oriRi and oriV sequences. Table 3 shows the results of this analysis.<br>
Out of 39 plants tested, only 3 had vector backbone sequences, a frequency of 7.7%. hi comparison, the control plasmid (an oriV plasmid) exhibited a vector backbone frequency of from 21% to 25% (depending on which samples were considered). This difference was quite dramatic and entirely unexpected.<br>
Another important component of a commercially viable transgenic plant is the occurrence of low insert complexity. This is often referred to as "low copy number." It is difficult to select progeny and to successfully breed the transgenic trait if the copy number of the insert is too high. Ideally, only a single copy of the transgene would be present in a transgenic event.<br>
Copy number can be determined by several methods known in the art of molecular biology. Southern blot analysis is the most commonly used method. Methods using the Invader® technology (Third Wave Inc. Madison, WI) are also used for determining copy number of T-DNA inserts in transgenic plants.<br>
The soybean plant cells were transformed with a control DNA plasmid (pMON67438) and a plasmid of the present invention (pMON83882), then regenerated into intact plants, as described above. The resulting transgenic plants were analyzed for copy number using two different methods 1) "Invader CP4" method and 2) Southern blotting. The results are shown in Table 3.<br>
Table 3. pRi ori vector backbone effects on transformation frequency, copy number, and the insert in soybean transformation." Pooled data from three side by side experiments. b Plant number from side by side comparison. c Pooled plant number of pMON67438 plants produced.<br>
(Table Removed)<br>
Using the "Invader CP4" method, the inventive plasmid, pMON83882, resulted in single-copy transformation events 41% of the time, as compared to the control plasmid pMON67438 which resulted in a single-copy transformation event only 25% of the time using pooled samples of other experiments or only 37% of the time using side by side experiments. Considering one- and two-copy events together, pMON83882 resulted in said events 79.5% of the time, as compared to 60% from side by side experiments and 63% from pooled samples employing the control plasmid. Using the Southern blotting comparison, pMON83882 produced one-copy events 56.4% of the time and 33.3% of the time two-copy events, for a total of nearly 90% one- and two-copy events combined. Again, these results were both dramatic and unexpected.<br>
B.        Molecular Analysis of Transgene Copy Number in Soybean Two T-DNA Transformants.<br>
In order to analyze the oriRi impact on 2T-DNA transformation, the 4.2 kb oriRi 2T vector pMON96001 was side by side compared with the oriV 2T vector pMON87488 for six times. The 5.6 kb oriRi 2T vector pMON96010 was also included in the last two comparison experiments in these preliminary experiments (Table 4, Group A ). Total DNA was extracted from seed segments of mature seeds harvested from greenhouse and used to determine transgene copy number by Invader technology using CP4 probe and NOS probes. CP 4 Invader® detection assay used the following sequence as a targeting site: 5' TCGCTTTCCTTGACGCGGAGTTCTTCCA GACCGTTCAT CACGG 3' (SEQ ID NO: 13) and GTAGGTGATTGGCGTTG (SEQ ID NO: 14) as a probe sequence. NOS Invader® detection assay used the following nos 3' sequence as a targeting site TGAATTACGTTAAGCATGTAATAATTAACATGTAATGCATGACT (SEQ ID NO: 15 and GTTATTTATGAGATGGGTTTTTATGA (SEQ ID NO: 16) as NOS Invader® probe sequence.<br>
The 1 and/or 2 copy transformants were increased from 35% in the control oriV vector to over 40% in both oriRi vectors. The frequency of marker-free plants in RO plants (marker free events/total RO plants) was increased from 4.76% in the oriV control vector to 16.4% and 18.4%, respectively, in the two oriRi vector. Overall, the marker-free transformation frequency (marker free events/initial explant number) more than doubled in the two oriRi 2T vectors compared to the oriV 2T vector.<br>
Additional in-parallel experiments with larger initial explant number were further used to compare the oriV control 2T vector with the 4.2 kb oriRi 2T vector (Table 4, Group B). The oriRi 2T vector reproduced the results: the 1+2 copy event frequency and the marker free frequency (MF%) are both increased and the marker-free transformation frequency is more than doubled in the oriRi vector. The results indicated that the oriRi 2T vectors are significantly more efficient to produce marker free plants than the oriV2T vector by increasing low copy events and improving marker free transformation frequency.<br>
Table 4 . Effect of type of origin of replication on 2T-DNA vector transformation of soybean. Note: 1) TF: transformation frequency=RO plants/total explants; 2) 1+2 copy Co-T: 1 and/or 2 copy gus and CP4 marker 2 T-DNA co-transformants. Only 1-2 copy events were further advanced to analyze the marker segregation in seeds; 3) MF events: gus gene positive, marker-free events from the<br>
1+2 copy co-transformants analyzed by seed segregation; 4) MF % plants: gus positive, marker free plants/RO plant #; 5) Marker-free transformation frequency (MF TF) =unlinked event # / initial explant #.<br>
(Table Removed)<br>
C.       Molecular Analysis of Vector Backbone DNA and Transgene Copy Number in Corn transformed with One T-DNA constructs<br>
To investigate the oriRi replication origin effect on corn transformation, the 5.6 kb oriRi vector pMON97352 containing gus and CP4 genes, both driven by rice actin promoter, was compared to the oriV vector pMON92726 with the same T-DNA structure containing gus and CP4 genes. Corn immature embryos from Cultivar LH244 were inoculated with Agrobacterium containing either the oriRi or the ori V vector in parallel. Each treatment consisted of about 110 embryos. In total 16 side by side comparison experiments were initiated. The oriRi vector showed significantly lower transformation frequency than the oriV control vector (Fig. 10) with average TF 9.5% and 16.3% for oriRi and oriV vectors, respectively. However, the overall transformation efficiency with oriRi and oriV vectors was same.<br>
The presence of backbone sequence in transgenic corn plants was determined by Endpoint TaqMaa® assay (Applied Biosystems, Foster City, CA).for the oriRi or oriV sequence, LB sequences that are 3' flanking for short read-through close to the LB nicking site and RB sequences that are 5' flanking before the RB nicking site for intact backbone readthrough. The oriV primers 5' AACGCCTGATTTTACGCGAG 3' (forward; SEQ ID NO: 17) and 5' CAATACCGCAGGGCACTTATC 3' (reverse; SEQ ID NO: 18), with probe CCCACAGATGATGTGGAC (SEQ ID NO: 19) labeled with fluorescent dye 6-FAM at the 5' end. The oriRi primers are 5'<br>
TGGCAAGGAATGGGTTTGAG 3' (forward; SEQ ID NO:20) and 5' CTACAACTACAGGCGCTGCTTTT 3' (reverse; SEQ ID NO:21) with the probe 6-FAM-TGGCGAAGTCTGTCC (SEQ ID NO:22) to detect oriRi sequence 621 bp downstream of LB nicking site. The RB .5' flanking primers are 5' GCCAAGGGATCTTTTTGGAAT 3' (forward; SEQ ID NO:23) and 5' CCACCCAAACGTCGGAAA 3' (reverse; SEQ ID NO:24) with the probe 6FAM-TGCTCCGTCGTCAGG (SEQ ID NO:25) to detect 5' RB flanking 186 bp before the RB nicking site. The primers for detecting LB 35 bp downstream of the LB nicking site were 5' GCACCCGGTGGAGCTT 3' (forward; SEQ ID NO:26) and 5' TCTGCCTAACCGGCTCAGT 3' (reverse; SEQ ID NO:27) with the probe CATGTTGGTTTCTACGCAG (SEQ ID NO:28) labeled with fluorescent dye 6-FAM at the 5' end. Approximately 10 ng DNA was used for Endpoint TaqMan® reaction according to the manufacturer's instruction (Applied Biosystems, Foster City, CA).<br>
As shown in Figure 11, about 95% of the transgenic plants derived from oriRi vector were backbone-free, which is significantly better than ori V vector control showing about 78% backbone free frequency. Since the vector backbone starts at LB after nicking site and ends before RB nicking site, use of three different backbone probes (LB downstream, oriRilonV and RB upstream) in both oriRi and oriV vector derived plants revealed that most vector backbone transferred contained entire vector backbone sequence.<br>
The CP4 transgene copy number was determined by Invader® assay according to the manufacturer's instruction (Third Wave Technologies Inc. Madison, WI) using the a CP4 sequence. The gus transgene copy number was measured by determining the gus cassette transcriptional terminator pinll (shown as Pis 4 in FIG. 5 and 6) using TaqMan® technology (Applied Biosystems, Foster City, CA). The pinll TaqMan® primers were 5' ATGAAATAAAAGGATGCACACAT 3' (forward; SEQ ID NO:29) and 5' ACAACTTTGATGCCCACATT 3' (reverse; SEQ ID NO:30) with the probe TGACATGCTAATCAC- (SEQ ID NO:31) labeled with fluorescent dyes 6-FAM at the 5' end and MGBNFQ at the 3' end.<br>
FIG. 12 shows the transgene copy number assay results. The oriRi vector significantly increased single copy plant frequency compared to the control oriV vector, while the frequency of 2 or more copy events in oriRi vector was decreased. Both TaqMan® and Invader® assays with two different probes showed similar results. Since<br>
the single copy backbone free plants are the most valuable for both research and development, the oriRi vector increasing the single copy frequency and decreasing backing containing events in corn represents a significant improvement for quality in transgenic event production.<br>
Example 5<br>
Construction of plant transformation vector containing repABC replication origin<br>
from Rhizobium<br>
To clone repABC replication origin from Rhizobium etli CFN42 strain, the USDA strain was obtained from USDA Rhizobium Collection Center. A 4.3 kb repABC fragment was amplified from R. etli plasmid p42b found in strain CFN42 by PCR with primers 5' CCACGTGAGTTACGGCTGATCGACCAGAC 3' (Xd745; SEQ ID NO:9) and 5' GCCTAGGACGTCAACTCCAACC'GCACCGT 3' (Xd746; SEQ ID NO:10) and ligated to TOPO blunt cloning vector (Invitrogen, Carlsbad, CA, USA), which resulted in pMON96941 and was confirmed by sequencing. The repABC fragment was digested with Pmll and Avrll (shown by underlined sequence in primers) and ligated to pMON96948 opened with Smal and Avrll. The ligate was directly transferred into Agrobacterium tumefaciens AB2 (a kanamycin sensitive strain) competent cells, and plated onto LB plate with kanamycin 50 mg/1. The kan resistant colonies were inoculated into liquid LB medium with 50 mg/1 kanamycin, and a plasmid DNA was prepared by miniprep. The construct was confirmed by restriction digestion. Since the vector pMON96951 does not contains an E. coli replication origin, it can not be maintained in E. coli.<br>
pMON96951 (FIG. 13) is used for transforming plant species as described above. The transgenic events are analyzed for reduced vector backbone integration and high frequency of low-copy transformation events by methods described in Example 4.<br>
Other embodiments of the invention will be apparent to those skilled in the art from consideration of the specification and practice of the invention disclosed herein. It is intended that the specification and examples be considered as exemplary only, with a true scope and spirit of the invention being indicated by the following claims.<br>
REFERENCES<br>
The following are herein incorporated by reference:<br>
U.S. Patent 4,581,847; U.S. Patent 4,761,373; U.S. Patent 4,810,648; U.S. Patent 5,004,863; U.S. Patent 5,013,659;U.S. Patent 5,034,322; U.S. Patent 5,094,945; U.S. Patent 5,141,870; U.S. Patent 5,159,135; U.S. Patent 5,229,114; U.S. Patent 5,273, 894; U.S. Patent 5,276,268; U.S. Patent 5,304,730; U.S. Patent 5,378,824; U.S. Patent 5,463,175; U.S. Patent 5,463,175; U.S. Patent 5,512,466; U.S. Patent 5,516,671; U.S. Patent 5,518,908; U.S. Patent 5,543,576; U.S. Patent 5,561,236; U.S. Patent 5,591,616; U.S. Patent 5,597,717; U.S. Patent 5,605,011; U.S. Patent 5,608,149; U.S. Patent 5,627,061; U.S. Patent 5,633,435; U.S. Patent 5,633,437; U.S. Patent 5,633,444; U.S. Patent 5,637, 489; U.S. Patent 5,646,024; U.S. Patent 5,689,041; U.S. Patent 5,716,837; U.S. Patent 5,719,046; U.S. Patent 5,750,871; U.S. Patent 5,750,876; U.S. Patent 5,767,366; U.S. Patent 5,773,696; U.S. Patent 5,824,877; U.S. Patent 5,846,797; U.S. Patent 5,939,602; U.S. Patent 5,942,664; U.S. Patent 5,958,745; U.S. Patent 5,985,605; U.S. Patent 5,998,700; U.S. Patent 6,011,199; U.S. Patent 6,013,864; U.S. Patent 6,040,497; U.S. Patent 6,063,597; U.S. Patent 6,063,756; U.S. Patent 6,072,103; U.S. Patent 6,080,560; U.S. Patent 6,093,695; U.S. Patent 6,110,464; U.S. Patent 6,121,436; U.S. Patent 6,166,292; U.S. Patent 6,171,640; U.S. Patent 6,225,105; U.S. Patent 6,228,992; U.S. Patent 6,252,148; U.S. Patent 6,268,549; U.S. Patent 6,316,407; U.S. Patent 6,380,466; U.S. Patent 6,384,301; U.S. Patent 6,414,222; U.S. Patent 6,444,876; U.S. Patent 6,476,295; U.S. Patent 6,506,962; U.S. Patent 6,531,648; U.S. Patent 6,537,750; U.S. Patent 6,613,963; U.S. Patent 6,624,344; U.S. Patent 7,002,058; U.S. Patent Pub. 2003/0028917; U.S. Patent Pub. 20040177399; U.S. Patent Pub. U.S. Patent Pub. 20030083480; U.S. Patent Pub. 20030115626; U.S. Patent Pub. 20050005321; U.S. Patent Pub. 20030135879; U.S. Prov. Appln. 60/800,872;<br>
PCT Appln. WO04074443; PCT Appln. WO04009761; PCT Appln. WO9927116; PCT Appln. WO8704181A1; PCT Appln. WO8900193A; PCT Appln. WO8911789; PCT Appln. WO 9638567; PCT Appln. WO05107437; PCT Appln. WO05003362<br>
European Appln. 275,957<br>
Japanese Appln. 06343473<br>
Ammirato et al, In:   Handbook of Plant Cell Culture—Crop Species, Macmillan Publ.<br>
Co., 1984. Ausubel et al., In: Current Protocols in Molecular Biology, John, Wiley &amp; Sons, Inc, NY,<br>
6.3.1-6.3.6, 1989. Binns and Howitz, In: Bacterial Pathogenesis of plants and Animals, Dang (Ed.), Berlin:<br>
Stringer Verlag, 119-138, 1994. Broothaerts et al., Nature, 433:629-633, 2005. Dattaetal, Bio/Technology, 8:736-740, 1990. Fraleyetal., Bio/Technology, 3:629-635, 1985. Fromm, UCLA Symposium on Molecular Strategies for Crop Improvement, Apr. 16-22,<br>
CO, 1990. Gelvin, In:   Transgenic Plants, Kung and Wu (Eds.), Academic Press, San Diego, 49-87,<br>
1993.<br>
Hayashimoto, Plant Physiol, 93:857-863, 1990. Haymes et al., In:    Nucleic Acid Hybridization, A Practical Approach, IRL Press,<br>
Washington, D.C., 1985<br>
Herman et al, J. Biol Chem., 280(26):24759-24767, 2005. Hooykaas and Beijersbergen, Ann. Rev. Phytopathol, 32:157-179, 1994. Howard and Citovsky, Bioassays, 12:103-108, 1990. Jefferson etal, Biochem. Soc. Trans., 15:7-19, 1987. Kado, Crit. Rev. Plant ScL, 10:1-32, 1991. Kanehisa, Nucl. Acids Res., 12:203-213, 1984. Klee etal, Ann. Rev. PlantPhys., 38:467-486, 1987. Lessl and Lanka, Cell, 77:321-324, 1994.<br>
Pearson and Lipman, Proc. Natl Acad. Sci. USA, 85(8):2444-2448, 1988. Ream; Ann. Rev. Phytopathol., 27: 583-618, 1989. Rogers et al, Methods Enzymol, 153:253-277, 1987. Rogers et al, Plant Mol Biol, 5:69-76, 1985. Sambrook et al, In: Molecular cloning: a laboratory manual, 2nd Ed., Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, NY, 1989. Sambrook et al, In: Molecular cloning: a laboratory manual, 2nd Ed., Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, NY, 9.52-9.55, 1989. Sambrook et al, In: Molecular cloning: a laboratory manual, 2nd Ed., Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, NY, 9.47-9.52, 9.56-9.58, 1989.<br>
Shimamoto et al., Nature, 338:274-276, 1989.<br>
Vasil et al, Bio/Technology, 10:667-674, 1992.<br>
Vasil et al, Bio/Technology, 8:429434, 1990.<br>
Wetmur and Davidson, J. Mol. BioL, 31:349-370, 1968.<br>
Winans, Microbiol. Rev., 56:12-31, 1992.<br>
Wising et al. Ann. Rev. Genetics, 22, 421, 1988.<br>
Zambryski, Ann. Rev. Plant Physiol. Plant Mol. BioL, 43: 465-490, 1992.<br>
Zupan and Zambryski, Ann. Rev. PhytopathoL, 27:583-618, 1995.<br><br><br><br><br><br><br>
WHAT IS CLAIMED IS:<br>
1.	A DNA construct for transforming plants comprising:<br>
i)         at least one T-DNA border region;<br>
ii)        at least one heterologous transgene adjacent to the border region; iii)       a coding region for a bacterial selectable marker; and<br>
iv) at least one segment of DNA, comprising a cis and/or trans element of a replication origin for maintaining a low copy number of the DNA construct in a bacterium competent for the transformation of at least a first plant cell.<br>
2.	The DNA construct according to claim 1, wherein the at least one T-DNA border<br>
region is either a right border region or a left border region.<br>
3.	The DNA construct according to claim 1, wherein the heterologous transgene<br>
when expressed in a transformed plant cell provides an agronomic phenotype to<br>
the cell or a transformed plant derived from the cell.<br>
4.	The DNA construct according to claim 1, wherein the coding region for the<br>
bacterial selectable marker comprises an antibiotic resistance gene selected from<br>
the group consisting of a kanamycin resistance gene, gentamycin resistance gene,<br>
chloramphenicol resistance gene,   spectinomycin resistance gene, streptomycin<br>
resistance   gene,   tetracycline   resistance   gene,   ampicillin   resistance   gene,<br>
blasticidin resistance gene, hygromycin resistance gene, puromycin resistance<br>
gene, and Zeocin resistance gene.<br>
5.	The DNA construct according to claim 1, wherein the replication origin comprises<br>
a repABC origin.<br>
6.	The DNA construct according to claim 5, wherein the repABC origin is selected<br>
from the group consisting of SEQ ID NO:1, SEQ ID NO:2, SEQ ID NO:3 and<br>
SEQ ID NO:4.<br>
7.	The DNA construct according to claim 1, wherein the at least one segment of<br>
DNA comprises replication origin genes repA, repB, and repC.<br>
8.	The DNA construct according to claim 1, wherein the at least one segment of<br>
DNA comprises intergenic sequence 1 (igsl) and intergenic sequence 2 (igs2).<br>
9.	The DNA construct according to claim 1, wherein the at least one segment of<br>
DNA comprises a 16 base pair palindromic sequence.<br>
10.	The DNA construct according to claim 1, wherein the bacterium is selected from<br>
the group consisting of Agrobacterium spp., Rhizobium spp., Sinorhizobium spp.,<br>
Mesorhizobium     spp.,     Phyllobactcrium     spp.     Ochrobactrum     spp.     and<br>
Bradyrhizobium spp.<br>
11.	The DNA. construct according to claim 1, further comprising at least a first<br>
replication origin for maintaining copies of the construct in E. coli.<br>
12.	The DNA. construct according to claim 11, wherein the replication origin is<br>
derived from pBR322 and/or pUC.<br>
13.	A cell transformed with the DNA construct of claim 1.<br>
14.	The cell of claim 13, defined as a plant cell.<br>
15.	The cell of claim 13, defined as a bacterial cell<br>
16.	The bacterial cell of claim 15, wherein the cell is competent for the transformation<br>
of at least a first plant cell.<br>
17.	The bacerial cell of claim 16, wherein the bacteria is selected from the group<br>
consisting   of   Agrobacterium   spp.,   Rhizobium    spp.,    Sinorhizobium    spp.,<br>
Mesorhizobium     spp.,     Phyllobacterium     spp..     Ochrobactrum     spp.     and<br>
Bradyrhizobium spp.<br>
18.	A method for transforming a plant cell comprising contacting at least a first plant<br>
cell with the bacterial cell of claim 16, wherein the bacterial cell is competent for<br>
the transformation of said first plant cell; and selecting said first cell based on the<br>
presence of at least one heterologous transgene introduced into the plant cell by<br>
the bacterial cell.<br>
19.	The method of claim 18, wherein the plant cell is a soybean, canola, corn, or<br>
cotton plant cell.<br>
20.	The method of claim 18, further comprising regenerating a plant from the plant<br>
cell.<br>
21,            A method of producing food, feed or an industrial product comprising: obtaining the plant of claim 20 or a part thereof; and preparing the food, feed or industrial product from the plain or part thereof.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUFic3RyYWN0LSgwMi0wOS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Abstract-(02-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUFic3RyYWN0LSgwMy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Abstract-(03-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1ERUxOUC0yMDA4LUFzc2lnbm1lbnQtKDIzLTA2LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2653-DELNP-2008-Assignment-(23-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUNsYWltcy0oMDItMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Claims-(02-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUNsYWltcy0oMDMtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Claims-(03-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Correspondence Others-(02-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Correspondence Others-(03-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDctMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Correspondence Others-(04-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1ERUxOUC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjMtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2653-DELNP-2008-Correspondence-Others-(23-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1ERUxOUC0yMDA4LUZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2653-DELNP-2008-Form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUZvcm0tMi0oMDItMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Form-2-(02-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUZvcm0tMy0oMDMtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Form-3-(03-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1ERUxOUC0yMDA4LUZvcm0tMy0oMjMtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2653-DELNP-2008-Form-3-(23-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUdQQS0oMDItMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-GPA-(02-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LUdQQS0oMDMtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-GPA-(03-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1My1kZWxucC0yMDA4LVBldGl0aW9uLTEzNy0oMDMtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2653-delnp-2008-Petition-137-(03-12-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="262906-method-for-allocating-gap-patterns-in-a-wireless-communication-environment.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="262908-bisphosphines-as-bidentate-ligands.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>262907</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2653/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>39/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Sep-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Sep-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH LINDBERGH BLVD ST.LOUIS, MO 63167, U.S</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YE, XUDONG</td>
											<td>800 NORTH LINDBERGH BLVD ST.LOUIS, MO 63167, U.S</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GILBERTSON, LARRY, A</td>
											<td>800 NORTH LINDBERGH BLVD ST.LOUIS, MO 63167, U.S</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PETERSON, MICHAEL, W</td>
											<td>800 NORTH LINDBERGH BLVD ST.LOUIS, MO 63167, U.S</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/034503</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-09-06</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/714,501</td>
									<td>2005-09-06</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/262907-methods-and-vectors-for-transforming-plant-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:04:23 GMT -->
</html>
