<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263049-an-isolated-primate-t-lymphotropic-virus-and-vector-comprising-the-said-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:31:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263049:&quot;AN ISOLATED PRIMATE T-LYMPHOTROPIC VIRUS AND VECTOR COMPRISING THE SAID VIRUS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED PRIMATE T-LYMPHOTROPIC VIRUS AND VECTOR COMPRISING THE SAID VIRUS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed are compositions and methods related to the isolation and identification of the novel primate T-lym-photropic viruses, HTLV-3 and HTLV-4. The diversity of HTLVs was investigated among central Africans reporting contact with NHP blood and body fluids through hunting, butchering, and keeping primate pets. Herein it is shown that this population is infected with a variety of HTLVs, including two novel retroviruses; HTLV-4 is the first member of a novel phylogenetic lineage that is distinct from all known HTLVs and STLVs; HTLV-3 falls within the genetic diversity of STLV-3, a group that has not previously been seen in humans. The present invention also relates to vectors and vaccines for use in humans against infection and disease. The invention further relates to a variety of bioassays and kits for the detection and diagnosis of infection with and diseases caused by HTLV-3 and HTLV-4 and related viruses.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL PRIMATE T-LYMPHOTROPIC VIRUSES<br>
1.	This application claims the benefit of U.S. Provisional Application No.<br>
60/654,484, filed on February 21,2005, which is incorporated by reference herein in its<br>
entirety. This invention was made with government support. The government has certain<br>
rights in the invention.<br>
I.   BACKGROUND<br>
2.	Primate T-lymphotropic viruses (PTLVs) are diverse deltaretroviruses,<br>
composed of 3 distinct species (PTLV-1, -2, -3) which by conventional nomenclature are<br>
named 'STLV (simian T-lymphotropic virus) when found in non-human primates (NHPs)<br>
and 'HTLV (human T-lymphotropic virus) when found in humans, regardless of suspected<br>
zoonotic origin (Mahieux et al. 1998; Salemi et al. 1999; Slattery et al. 1999; Courgnaud et<br>
al. 2004). Like HIV, HTLV has the potential to cause disease and circulate globally in<br>
humans sexually, from mother-to-child, and by exposure to contaminated blood from<br>
transfusions and intravenous drug use. HTLV-1 causes adult T-cell leukaemia and HTLV-1-<br>
associated myelopathy/tropical spastic paraperesis (HAM/TSP) and other inflammatory<br>
diseases (Gessain &amp; Mahieux 2000) and HTLV-2 has been associated with a neurologic<br>
disease similar to HAM/TSP (Araujo &amp; Hall 2004). There has been no evidence to date of<br>
STLVs crossing into people occupationally exposed to NHPs in laboratories and primate<br>
centers, as has been documented with other primate retroviruses, including simian<br>
immunodeficiency virus (SIV) (Khabbaz et al. 1994), simian foamy virus (SFV) (Switzer et<br>
al. 2004, Heneine et al. 1998), and simian type D retrovirus (Lerche et al. 2001).<br>
Nevertheless, ongoing zoonotic transmission of STLV to widespread human populations<br>
naturally exposed to NHPs through hunting or butchering, similar to that recently reported<br>
for SFV in African hunters (Wolfe et al. 2004b), would be of particular public health<br>
significance due to the transmissible and pathogenic nature of this group of viruses among<br>
humans. Previous studies have not documented evidence of HTLV outside of the PTLV-1<br>
and PTLV-2 groups (Busch et al. 2000; VanDamme et al. 1997; Salemi et al. 1999; Slattery<br>
etal. 1999).<br>
II.       SUMMARY<br>
3.  Disclosed are methods and compositions related to the isolation of the novel primate T-lymphotropic viruses, HTLV-3 and HTLV-4.<br>
4.  The present invention is directed to compositions and methods comprising a novel primate T-lymphotropic virus or primate T-lymphotropic virus, known as HTLV-3 or HTLV-4. The novel primate T-lymphotropic virus of the present invention has utility as a reagent for the immunological screening of the human population for the prevalence of such viruses in the population. The novel primate T-lymphotropic virus of the present invention can also serve as a vector in gene therapy because the virus appears to cause no disease in humans and may not transmitted to other humans. Additionally, the novel primate T-lymphotropic virus of the present invention can be used as a reagent in pathogenicity studies of these and related viruses. Moreover, the sequences of the novel primate T-lymphotropic virus of the present invention can be used as probes to detect virus in biological samples. Vectors include, but are not limited to, prokaryotic, eukaryotic and viral vectors. The primate T-lymphotropic virus of the present invention can also be used as a live recombinant virus vaccine. Additionally, the primate T-lymphotropic virus of the present invention can be used as a replicating viral system to kill live dividing cells, either in vitro or in vivo.<br>
5.	The present invention includes the isolation and characterization of primate T-<br>
lymphotropic viruses, HTLV-3 or HTLV-4, that are believed to have been transmitted from<br>
non-human primates to humans at some point in the past. The primate T-lymphotropic virus<br>
of the present invention does not appear to be readily transmitted from human to human.<br>
The primate T-lymphotropic virus of the present invention can be used in constructing<br>
protocols for diagnosing primate T-lymphotropic virus infections and may be used as a<br>
vector in gene therapy procedures.<br>
6.	The present invention provides compositions comprising live replicating<br>
retroviral vectors, wherein the vector is derived from a primate T-lymphotropic virus, and<br>
wherein the vector further comprises a nucleic acid that encodes a non-primate T-<br>
lymphotropic virus peptide, polypeptide, or protein. For example, the vector can be derived<br>
from an HTLV-3 or HTLV-4 virus. Thus, in one aspect, the present invention provides<br>
compositions comprising live replicating primate T-lymphotropic virus vectors.<br>
7.	Also provided by the present invention is a method of treating a subject with a<br>
condition, wherein the condition can be a viral infection, bacterial infection, parasitic<br>
infection, proliferative disorder (e.g., cancer), or a condition associated with a genetic or<br>
autoimmune disorder; comprising administering to the subject a live replicating viral vector,<br>
wherein the immunizing construct is specific for the condition.<br>
8.	Also provided by the present invention is a method of preventing a condition in a<br>
subject, wherein the condition can be a viral infection, bacterial infection, parasitic<br>
infection, proliferative disorder, or a condition associated with a genetic or autoimmune<br>
disorder; comprising administering to the subject a live replicating viral vector, wherein the<br>
antigen-encoding nucleic acid is specific for the condition.<br>
9.	Also provided are methods of using the present vector, isolated viruses, or<br>
infectious clones for making viral infection models and using models to study diseases and<br>
potential treatments, as well as the models themselves.<br>
10.	The present invention also includes methods and compositions for detecting<br>
primate T-lymphotropic virus in biological fluids. The methods and compositions,<br>
including kits, can be in any configuration well known to those of ordinary skill in the art.<br>
The present invention also includes antibodies specific for the primate T-lymphotropic virus<br>
and antibodies that inhibit the binding of antibodies specific for the primate T-lymphotropic<br>
virus. These antibodies can be polyclonal antibodies or monoclonal antibodies, which also<br>
includes fragments of any type of antibody. Thus, specifically disclosed and herein<br>
contemplated are antibodies to HTLV-3 and HTLV-4. The antibodies specific for the<br>
primate T-lymphotropic virus can be used in diagnostic kits to detect the presence and<br>
quantity of primate T-lymphotropic virus in biological fluids or in organs from nonhuman<br>
primates for xenotransplantation. For example, the HTLV-3 antibody can be used in a<br>
diagnostic kit to detect HTLV-3. Antibodies specific for primate T-lymphotropic virus may<br>
also be administered to a human or animal to passively immunize the human or animal<br>
against primate T-lymphotropic virus, thereby reducing infection after accidental exposure<br>
to nonhuman primate bodily fluids.<br>
11.	The present invention also includes compositions and methods, including kits,<br>
for detecting the presence and quantity of antibodies that bind primate T-lymphotropic virus<br>
in body fluids. The methods, including kits, can be in any configuration well known to<br>
those of ordinary skill in the art. Such kits for detection of primate T-lymphotropic virus<br>
itself or detection of antibodies to the primate T-lymphotropic virus can be used to monitor<br>
the blood supply for the presence of primate T-lymphotropic virus in the blood supply. It is<br>
understood and herein contemplated that the disclosed kits can include, for example, a kit<br>
for the detection of antibodies to HTLV-3 or HTLV-4.<br>
12.	The present invention also includes methods and compositions comprising<br>
recombinant live virus vaccines. The virus of the present invention has areas of its genome<br>
that make it ideal for the insertion of exogenous genes. The genes can code for any protein for which vaccination or gene therapy is desired. A novel aspect of such recombinant live viruses is that the recombinant HTLV-3 or HTLV-4 does not cause disease in the host organism. The recombinant live virus vaccines of the present invention are a safe way to provide antigen in a most optimum method to the immune system.<br>
13.	Accordingly, it is an object of the present invention to provide a composition<br>
comprising a novel primate T-lymphotropic virus. It is understood and herein contemplated<br>
that an example of the disclosed primate T-lymphotropic virus includes, but is not limited to<br>
HTLV-3 and HTLV-4.<br>
14.	It is another object of the present invention to provide a method of detecting a<br>
primate T-lymphotropic virus, such as HTLV-3 or HTLV-4.<br>
15.	It is yet another object of the present invention to provide methods and<br>
compositions for detecting the presence and amount of primate T-lymphotropic virus hi a<br>
body fluid or organ.<br>
16.	A further object of the present invention is to provide compositions and methods<br>
for treating genetic and physiologic disorders using gene therapy techniques comprising the<br>
novel primate T-lymphotropic virus of the present invention as a vector for nucleic acid<br>
sequences and antisense sequences.<br>
17.	Another object of the present invention is to provide compositions and methods<br>
useful for manipulating the expression of genes.<br>
18.	Yet another object of the invention is to provide vaccines.<br>
19.	Yet another object of the present invention is to provide compositions and<br>
methods for treating viral infections in humans or animals.<br>
20.	Another object of the present invention is to provide compositions and methods<br>
that are effective in treating genetic diseases.<br>
21.	Yet another object of the present invention is to provide a method of treating<br>
microbial infections in humans or animals.<br>
22.	It is another object of the present invention to provide for treatments of<br>
conditions that are caused in part by rapidly dividing cellular growth.<br>
23.	Another object of the present invention is to provide live recombinant virus<br>
vaccines.<br>
24.	An object of the present invention is to provide diagnostic tools such as<br>
antibodies or antigens for the monitoring of the blood supply or organ and tissue donation<br>
for the presence of primate T-lymphotropic virus.<br>
25.	These and other features and advantages of the present invention will become<br>
apparent after a review of the following detailed description of the disclosed embodiments<br>
and the appended claims.<br>
ID.      BRIEF DESCRIPTION OF THE DRAWINGS<br>
26.	The accompanying drawings, which are incorporated in and constitute a part of<br>
this specification, illustrate several embodiments and together with the description illustrate<br>
the disclosed compositions and methods.<br>
27.	Figure 1 shows the Western blot serological pattern of Human T-cell<br>
lymphotropic virus (HTLV) infected African hunters. HTLV classification based on<br>
phylogenetic analyses is provided above specimen names. Reactivity to HTLV-specific<br>
proteins is indicated on left.<br>
28.	Figure 2 shows the phylogenetic relationships of (a) primate T-cell lymphotropic<br>
virus polymerase (a) PTLV pol (662-bp), (b) PTLV env (297-bp), (c) PTLV tax (730-bp),<br>
(d) PTLV-3 long terminal repeat (LTR) (398-bp), and (e) PTLV pol-env-tax region (5258-<br>
bp) sequences by neighbour joining analysis. Sequences generated in the current study are<br>
noted with boxes. Nonhuman primate taxon codes are provided in Methods. Support for the<br>
branching order was determined by 1,000 bootstrap replicates, and only values 60% or<br>
greater are shown. Branch lengths are proportional to the evolutionary distance (scale bar)<br>
between the taxa.<br>
29.	Figure 3 shows the Phylogenetic relationships of PTLV type 1 LTR (377-bp)<br>
sequences by neighbour joining analysis. Sequences generated in the current study are noted<br>
with boxes. Nonhuman primate taxon codes are provided in Methods. Support for the<br>
branching order was determined by 1,000 bootstrap replicates and only values 60% or<br>
greater are shown. Branch lengths are proportional to the evolutionary distance (scale bar)<br>
between the taxa.<br>
30.	Figure 4 shows the strategy for PCR-amplifying the entire HTLV-3(2026ND)<br>
genome. Small proviral sequences were first amplified in each major gene region and the<br>
long terminal repeat (stippled bars) using generic primers as described in methods, The<br>
complete proviral sequence was then obtained by using PCR primers located within each'<br>
major gene region by genome walKing as indicated with arrows and orange bars. The typical HTLV-1 genomic organization is provided for reference.<br>
31.	Figure 5(a) shows the nucleotide sequence of the HTLV-3(2026ND) LTR and<br>
pre-gog region. The U3-R-U5 locations (vertical lines), the approximate cap site (cap), the<br>
polyadenylation signal, TATA box, the predicted splice donor site (sd-LTR), and two 21-bp<br>
repeats are indicated. In the R and U5 regions, the predicted Rex core elements and nuclear<br>
riboprotein Al binding sites are underlined. The pie-gag region and primer binding site<br>
(PBS, underlined) are in italics. Figure 5(b) shows the plot of predicted RNA stem loop<br>
secondary structure of HTLV-3(2026ND) LTR region. Position of the Rex responsive<br>
element (RexRE) core is indicated.<br>
32.	Figure 6 shows a comparison of predicted Tax amino acid sequences of primate<br>
T-cell lymphotropic viruses (PTLV). Shown in boxes are known functional motifs: NLS,<br>
nuclear localization signal; (CBP)/P300, cAMP response element (CREB) binding protein;<br>
NES, nuclear export signal; CR2, C-terminal transcriptional activating domain binding;<br>
PDZ.<br>
33.	Figure 7 shows a comparison of predicted amino acid sequences of primate T-<br>
cell lymphotropic viruses (PTLV) and cellular basic leucine zipper (bZIP) transcription<br>
factors. Conserved arginine rich and leucine zipper regions of the bZIP proteins are boxed.<br>
Frameshift mutation of HTLV-2(MoT) leucine zipper region is depicted with italics.<br>
34.	Figure 8 shows the phylogenetic relationship of HTLV-3(2026ND) to other<br>
PTLVs (a) entire genome sans long terminal repeat (LTR), (b) gag, (c), polymerase (pol),<br>
and (d) envelope (env). Sequences generated in the current study are shown in boxes.<br>
Support for the branching order was determined by 1,000 bootstrap replicates; only values<br>
of 60% or more are shown. Branch lengths are proportional to the evolutionary distance<br>
(scale bar) between the taxa.<br>
35.	Figure 9 shows the estimated divergence dates for the most recent common<br>
ancestor of HTLV-3(2026ND) and other PTLVs. Divergence dates are provided for each<br>
major node of a neighbor-joining tree rooted with PTLV-1 as the outgroup; estimates are<br>
provided as ranges using as calibration points 40,000 and 60,000 years ago (YA) as the<br>
separation of the Melanesisan HTLV-1 (MEL5) sequence from other PTLV-1 strains.<br>
Bootstrap analysis of 1000 replicates is shown on the tree branches; only values &gt; 60% are<br>
shown.<br>
IV.      DETAILED DESCRIPTION<br>
36.	Before the present compounds, compositions, articles, devices, and/or methods<br>
are disclosed and described, it is to be understood that they are not limited to specific<br>
synthetic methods or specific recombinant biotechnology methods unless otherwise<br>
specified, or to particular reagents unless otherwise specified, as such may, of course, vary.<br>
It is also to be understood that the terminology used herein is for the purpose of describing<br>
particular embodiments only and is not intended to be limiting.<br>
A. Definitions<br>
37.	As used in the specification and the appended claims, the singular forms "a,"<br>
"an" and "the" include plural referents unless the context clearly dictates otherwise. Thus,<br>
for example, reference to "a pharmaceutical carrier" includes mixtures of two or more such<br>
carriers, and the like.<br>
38.	Ranges can be expressed herein as from "about" one particular value, and/or to<br>
"about" another particular value. When such a range is expressed, another embodiment<br>
includes from the one particular value and/or to the other particular value. Similarly, when<br>
values are expressed as approximations, by use of the antecedent "about," it will be<br>
understood that the particular value forms another embodiment. It will be further<br>
understood that the endpoints of each of the ranges are significant both in relation to the<br>
other endpoint, and independently of the other endpoint. It is also understood that there are<br>
a number of values disclosed herein, and that each value is also herein disclosed as "about"<br>
that particular value in addition to the value itself. For example, if the value "10" is<br>
disclosed, then "about 10" is also disclosed. It is also understood that when a value is<br>
disclosed that "less than or equal to" the value, "greater than or equal to the value" and<br>
possible ranges between values are also disclosed, as appropriately understood by the skilled<br>
artisan. For example, if the value "10" is disclosed the "less than or equal to 10"as well as<br>
"greater than or equal to 10" is also disclosed. It is also understood that the throughout the<br>
application, data is provided in a number of different formats, and that this data, represents<br>
endpoints and starting points, and ranges for any combination of the data points. For<br>
example, if a particular data point "10" and a particular data point 15 are disclosed, it is<br>
understood that greater than, greater than or equal to, less than, less than or equal to, and<br>
equal to 10 and 15 are considered disclosed as well as between 10 and 15.<br>
39. In this specification and in the claims which follow, reference will be made to a number of terms which shall be defined to have the following meanings:<br>
40.	"Optional" or "optionally" means that the subsequently described event or<br>
circumstance may or may not occur, and that the description includes instances where said<br>
event or circumstance occurs and instances where it does not.<br>
41.	"Primers" are a subset of probes which are capable of supporting some type of<br>
enzymatic manipulation and which can hybridize with a target nucleic acid such that the<br>
enzymatic manipulation can occur. A primer can be made from any combination of<br>
nucleotides or nucleotide derivatives or analogs available ha the art which do not interfere<br>
with the enzymatic manipulation.<br>
42.	"Probes" are molecules capable of interacting with a target nucleic acid, typically<br>
in a sequence specific manner, for example through hybridization. The hybridization of<br>
nucleic acids is well understood in the art and discussed herein. Typically a probe can be<br>
made from any combination of nucleotides or nucleotide derivatives or analogs available in<br>
the art.<br>
43.	Throughout this application, various publications are referenced. The<br>
disclosures of these publications in their entireties are hereby incorporated by reference into<br>
this application in order to more fully describe the state of the art to which this pertains.<br>
The references disclosed are also individually and specifically incorporated by reference<br>
herein for the material contained in them that is discussed hi the sentence in which the<br>
reference is relied upon.<br>
B. Compositions<br>
44.	Disclosed are the components to be used to prepare the disclosed compositions<br>
as well as the compositions themselves to be used within the methods disclosed herein.<br>
These and other materials are disclosed herein, and it is understood that when combinations,<br>
subsets, interactions, groups, etc. of these materials are disclosed that while specific<br>
reference of each various individual and collective combinations and permutation of these<br>
compounds may not be explicitly disclosed, each is specifically contemplated and described<br>
herein. For example, if a particular HTLV-3 or HTLV-4 or gene of the HTLV-3 or HTLV-4<br>
such as gag, pol, env, LTR, rex, and tax is disclosed and discussed and a number of<br>
modifications that can be made are discussed, specifically contemplated is each and every<br>
combination and permutation of HTLV-3 or HTLV-4 or genes of the HTLV-3 or HTLV-4<br>
such as gag, pol, env, LTR, rex, and tax and the modifications that are possible unless<br>
specifically indicated to the contrary. Thus, if a class of molecules A, B, and C are<br>
disclosed as well as a class of molecules D, E, and F and an example of a combination<br>
molecule, A-D is disclosed, then even if each is not individually recited each is individually and collectively contemplated meaning combinations, A-E, A-F, B-D, B-E, B-F, C-D, C-E, and C-F are considered disclosed. Likewise, any subset or combination of these is also disclosed. Thus, for example, the sub-group of A-E, B-F, and C-E would be considered disclosed. This concept applies to all aspects of this application including, but not limited to, steps in methods of making and using the disclosed compositions. Thus, if there are a variety of additional steps that can be performed it is understood that each of these additional steps can be performed with any specific embodiment or combination of embodiments of the disclosed methods.<br>
45. Disclosed herein are compositions relating to novel primate T-lymphotropic viruses HTLV-3 (SEQ ID NO: 36) and HTLV-4 (SEQ ID NO: 53). It is understood and herein contemplated that the compositions of the invention can comprise the entire HTLV-3 or HTLV-4 virus. It is also understood that the disclosed compositions of the invention can comprise proteins of the disclosed primate T-lymphocyte viruses or fragments of the disclosed proteins. For example, specifically disclosed and herein contemplated are compositions comprising SEQ ID NOs: 1, 3,5, 35,45,47,49, 51, and 52, or any combination thereof. Also disclosed are compositions comprising SEQ ID NOs: 2,4, 6,59, 61, and 63 or any combination thereof. Also disclosed are compositions comprising SEQ ID NOs: 37,40,44,46,48, and 50 or any combination thereof. Also dis'closed are compositions comprising SEQ ID NOs: 54, 57,58, 60, and 62 or any combination thereof. Also disclosed are compositions comprising fragments of the disclosed proteins. Thus, for example are compositions comprising SEQ ID NOs: 38,39,41,42, and 43 or any combination thereof. Also disclosed are compositions comprising SEQ ID NOs: 55 and 56. It is understood and herein contemplated that any of the disclosed proteins can be used in combination with any of the protein fragments in the compositions disclosed herein. Thus, for example, disclosed herein are compositions comprising SEQ ID NOs: 37, 38,39,40,41, 42,43, 44,46, 48, and 50 or any combination thereof. Also disclosed are SEQ ID NOs: 54, 55, 56, 57, 58, 60, and 62 or any combination thereof. SEQ ID NOs 1-6,35, and 45 can be used for all the molecular biological techniques known to those skilled in the art. Such uses include, but are not limited to, generation of probes and vectors containing the sequences, antisense sequences derived from such sequences, and proteins synthesized using the sequences. RNA and other nucleic acid derivatives are contemplated by the present invention.<br>
46. It is understood that there are known viruses in the art that based on certain genomic or sequence similarity or taxonomically related to the novel viruses disclosed herein. It is also understood that the known viruses in the art thought related taxonomically do not encode the specific viruses disclosed herein. Thus specifically disclosed and herein contemplated are isolated primate T-lymphotropic viruses having apol gene that has less than 63.5% identity to thepol gene of HTLV-1, HTLV-2, STLV-2, and STLV-3, for example, HTLV-4. Also disclosed are isolated primate T-lymphotropic viruses having a gag gene that has less than 69% identity to the gag gene of HTLV-1, HTLV-2, STLV-2, and STLV-3, for example, HTLV-3. Also disclosed are isolated primate T-lymphotropic viruses having apol gene that has less than 62% identity to ihepol gene of HTLV-1, HTLV-2, STLV-2, and less than 86% identity to thepol gene of STLV-3, for example, HTLV-3. Similarly, the disclosed viruses can be distinguished based on the genes encoded by the viruses of the invention and specifically the identity of said genes to the corresponding genes of known viruses. Thus, specifically disclosed are isolated primate T-lymphotropic viruses having a LTR that has less than 41% identity to the LTR of HTLV-1, HTLV-2 and STLV-3. Also disclosed are isolated primate T-lymphotropic viruses having at least 92.8% identity to the nucleic acid SEQ ID NO: 1.<br>
47.	Also disclosed are isolated primate T-lymphotropic virus having at least 92.5%<br>
identity to the nucleic acid SEQ ID NO: 3. Also disclosed are primate T-lymphotropic<br>
viruses having at least 94.2% identity to the nucleic acid SEQ ID NO: 5. Also disclosed are<br>
primate T-lymphotropic viruses having at least 91.5% identity to the nucleic acid SEQ ID<br>
NO: 35. Also disclosed are isolated primate T-lymphotropic viruses having at least 92.8%<br>
identity to the nucleic acid SEQ ID NO: 1, at least 92.5% identity to the nucleic acid SEQ<br>
E&gt; NO: 3, and at least 94.2% identity to the nucleic acid SEQ ID NO: 5.<br>
48.	HTLV-4 represents a unique delta primate T-lymphotropic virus. The most<br>
notable rinding was the discovery of a novel human virus that is distinct from all known<br>
PTLV lineages with 29-34.4% and 18.3-25% nucleotide divergence in the conservedpol<br>
and tax genes, respectively, a range of divergence similar to that between PTLV-1, PTLV-2,<br>
and PTLV-3. This virus formed a separate phylogenetic lineage with a long branch length<br>
and significant bootstrap support in both the pol (Figure 2a) and tax (Figure 2c) trees.<br>
Identical topologies were obtained by using maximum likelihood analysis. Phylogenetic<br>
analyses combined with GenBank blast searches show that this is the only known virus in<br>
this novel group. For these reasons, this virus, which was designated HTLV-4, qualifies as<br><br>
the first member of a novel group in the deitaretroviras genus. following the guidelines oi the International Committee on Taxonomy of Viruses and pending formal classification, primate T-lymphotropic virus 4 (PTLV-4) was proposed as the name for this species, and PTLV-4(1863LE) as the prototype strain. Due to the classification of the virus within the family retroviridae, certain sequence similarity is expected to exist with known retroviruses. It is understood that the known viruses in the art thought to be related taxonomically do not encode the specific viruses disclosed herein. Thus specifically disclosed and herein contemplated are isolated primate T-lymphotropic viruses having at least 71.5% identity to the nucleic acid SEQ ID NO: 2. Also disclosed are isolated primate T-lymphotropic viruses having at least 73.5% identity to the nucleic acid SEQ ID NO: 4. Also disclosed are isolated primate T-lymphotropic viruses having at least 82% identity to the nucleic acid SEQ ID NO: 6. Also disclosed are isolated primate T-lymphotropic viruses having at least 71.5% identity to the nucleic acid SEQ ID NO: 2, at least 73.5% identity to the nucleic acid SEQ ID NO: 4, and at least 82% identity to the nucleic acid SEQ ID NO: 6.<br>
49.	Knowing the sequence for HTLV-3 or HTLV-4, allows for various uses of the<br>
virus and viral sequences. The env gene of HTLV-3 or HTLV-4 is necessary for primate T-<br>
lymphotropic virus entry into animal cells. The gene of the present invention is effective in<br>
permitting infection of cells in a human host. Thus, for example, the env gene is used for<br>
uptake of foreign DNA by a wide range of human cells. There has long been a need for<br>
vectors for getting foreign nucleic acids into cells, both in vivo and in vitro. The<br>
introduction of foreign or exogenous nucleic acids into cells has been a technological hurdle<br>
for many gene therapy applications and has now been solved by the virus and sequences of<br>
the present invention. The env sequences can be used with any vector known to those<br>
skilled in the art, and with any other genetic sequences of choice, to allow for entry of the<br>
nucleic acids into the cells.<br>
50.	The recent advent of technology, and advances in the understanding of the<br>
structure and function of many genes makes it possible to selectively turn off or modify the<br>
activity of a given gene. Alteration of gene activity can be accomplished many ways. For<br>
example, oligonucleorides that are complementary to certain gene messages or viral<br>
sequences, known as "antisense" compounds, have been shown to have an inhibitory effect<br>
against viruses. By creating an antisense compound that hybridizes with the targeted RNA<br>
message of cells or viruses the translation of the message into protein can be interrupted or<br>
prevented. In this fashion gene activity can be modulated.<br>
51.	The ability to deactivate specific genes provides great therapeutic benefits. For<br>
example, it is theoretically possible to fight viral diseases with antisense molecules that seek<br>
out and destroy viral gene products. In tissue culture, antisense oligonucleotides have<br>
inhibited infections by herpes-viruses, influenza viruses and the human immunodeficiency<br>
virus that causes ADOS. It may also be possible to target antisense oligonucleotides against<br>
mutated oncogenes. Antisense technology also holds the potential for regulating growth and<br>
development. However, in order for the gene therapy to work, antisense sequences must be<br>
delivered across cellular plasma membranes to the cytosol.<br>
52.	Gene activity is also modified using sense DNA in a technique known as gene<br>
therapy. Defective genes are replaced or supplemented by the administration of "good" or<br>
normal genes that are not subject to the defect. Instead of being defective, the gene may<br>
have been deleted, thus replacement therapy would provide a copy of the gene for use by the<br>
cell. The administered normal genes can either insert into a chromosome or may be present<br>
as extracellular DNA and can be used to produce normal RNA, leading to production of the<br>
normal gene product. In this fashion gene defects and deficiencies in the production of a<br>
gene product may be corrected.<br>
53.	Still further gene therapy has the potential to augment the normal genetic<br>
complement of a cell. For example, it has been proposed that one way to combat HIV is to<br>
introduce into an infected person's T cells a gene that makes the cells resistant to HIV<br>
infection. This form of gene therapy is sometimes called "intracellular immunization."<br>
Genetic material such as a polynucleotide sequence maybe administered to a mammal in a<br>
viral vector to elicit an immune response against the gene product of the administered<br>
nucleic acid sequence. Such gene vaccines elicit an immune response in the following<br>
manner. First, the viral vector containing the nucleic acid sequence is administered to a<br>
human or animal. Next, the administered sequence is expressed to form a gene product<br>
within the human or animal. The gene product inside the human or animal is recognized as<br>
foreign material and the immune system of the human or animal mounts an immunological<br>
response against the gene product. The virus of the present invention may be used as a viral<br>
vector to provide the foreign nucleic acid sequences to the intracellular metabolic processes.<br>
54.	Additionally, gene therapy may be used as a method of delivering drugs in vivo.<br>
For example, if genes that code for therapeutic compounds can be delivered to endothelial<br>
cells, the gene products would have facilitated access to the blood stream. Additionally,<br>
cells could be infected with a retroviral vector such as the present invention carrying nucleic<br>
acid sequences coding for pharmaceutical agents that prevent infection from occurring in the retrovirally infected cells.<br>
55.	The novel primate T-lymphotropic viruses of the present invention can also be<br>
used a safe and effective vaccine agent. Genetic sequences for immunogenic proteins from<br>
a variety of infectious agents can be incorporated into the primate T-lymphotropic virus<br>
RNA. Once inside a cell, the gene product is expressed and releases the immunizing<br>
peptide to the body's immune system. In another method, the virus of the present invention<br>
can be used to immunize the body against cell markers found on cancer or tumor ceLls. The<br>
genetic sequence of the cancer cell marker is incorporated into the primate T-lympho1ropic<br>
virus RNA and after infection with the virus, the expressed gene product stimulates the<br>
immune system. The patient's immune system is used to remove the cancerous cells,<br>
obviating the need for chemotherapeutic methods.<br>
56.	The antibodies of the present invention can be used to detect the presence of the<br>
virus or viral particles of the present invention. These antibodies can be used in diagnostic<br>
or screening kits to assess the presence of the virus. Additionally, the antibodies can be<br>
used to screen organs from nonhuman primates that may be used in humans. Detection of<br>
the presence of a virus that is transmitted from nonhuman primates to humans would be<br>
crucial in providing virus-free organs for transplantation.<br>
57.	Such treatment with HTLV-3 or HTLV-4 can be used for any condition in which<br>
rapidly dividing cells provide an aspect of the pathology of the condition. One such<br>
condition is the presence of uncontrolled angiogenesis within the body. Angiogenesis<br>
dependent diseases are well known in the art and are caused in part by the rapid growth of<br>
blood vessels. Another such condition is cancer or tumor growth. Cancer or tumors include<br>
both solid tumors and other types. Infection with the virus of the present invention, which<br>
can cause no disease and does not effect the host systemically, is an improvement over<br>
currently known treatments that involved systemically administered agents. Such<br>
chemotherapeutic agents kill rapidly dividing cells but also cause trauma to the entire<br>
person. The dosages of such chemotherapeutic agents must be titered between killing the<br>
cancer and killing the patient.<br>
58.	hi contrast, treatments of cancer with the present invention are not as harmful to<br>
the patient. The virus can either be administered systemically or injected in situ into the<br>
tumor. The infected cells are killed and tumor growth is stopped. The virus may be<br>
administered in one treatment or in a series of treatments.<br>
59.	The HTLV-3 or HTLV-4 of the present invention can be recombinantly modified<br>
to be selective for cellular receptors on the tumor to make the virus even more specifically<br>
targeted to just those cells. Additionally, the virus may have altered promoter regions that<br>
can be selectively activated to cause a productive infection. The combination of different<br>
levels of control of the virus, both natural and recombinantly- produced, are contemplated in<br>
the present invention. A virus could be made specific for attachment to only certain types of<br>
cellular receptors, for those cells that are dividing, and will only undergo replication if<br>
another exogenous promoter factor is present. Viral infection by two or more individually<br>
defective viruses, that require factors or promoters supplied by other primate T-<br>
lymphotropic viruses or any type of virus, could provide for many levels of control of<br>
infection or treatment of specific conditions.<br>
60.	The virus may be administered to the host, for cancer treatment, gene therapy or<br>
vaccination by any methods known to those skilled in the art. Such methods include but are<br>
not limited to injection, inhalation, ingestion, topical administration and implantation. The<br>
virus may be killed or live, depending on the treatment considered.<br>
61.	The inventors of the present invention believe that the virus of the present<br>
invention, comprising the isolates from HTLV-3, is the first definitive isolation of an<br>
STLV-3-like primate T-lymphotropic virus from persons exposed to nonhuman primates.<br>
This belief is supported by the epidemiology data, the PCR and sequencing data and the<br>
serology data and the absence of such reports in the literature. It is understood that HTV-1<br>
and HTV-2 used to be called HTLV-HI and HTLV-IV before it was known they were<br>
different types of viruses. Additionally, the virus of the present invention comprising the<br>
isolates from HTLV-4, are a new species in the delta primate T-lymphotropic viruses.<br>
1.   Vectors<br>
62.	Disclosed are live replicating human primate T-lymphotropic virus vectors<br>
suitable for human use comprising an immunizing construct, wherein the immunizing<br>
construct is inserted in nontranslated region between env and tax/rex. The disclosed<br>
immunizing construct can be an antigen-encoding nucleic acid.<br>
63.	Where reference is made to "antigen"-encoding nucleic acid, it is understood that<br>
in the context of the invention antigens encoded by the antigen-encoding nucleic acid can<br>
include but are not limited to immunogenic or non-immunogenic peptides, polypeptides,<br>
proteins, enzymes, cytokines. These antigens can be non-human exogenous antigenic<br>
sequences from viruses, bacteria, or parasites. The antigens can also be antigenic<br>
endogenous human or human derived sequences from a condition such as a cancer. Also, peptides encoded by the antigen-encoding nucleic acid can include non-antigenic sequences for the purposes of gene therapy,<br>
64.	In another embodiment of the present invention, sequences of the primate T-<br>
lymphotropic viruses of the present invention can be used for other molecular biological<br>
applications. Regions of the gag gene are important in packaging genetic material. For<br>
example, the gag sequence or regions of the sequence are incorporated into other vectors<br>
and direct the packaging of the resultant genetic material for the particular application<br>
desired, such as packaging recomhinant sequences to make altered infectious virions.<br>
Regions of the^o/ gene are known to be critical for the stable integration of foreign/viral<br>
DNA into the host genome. Vectors comprising thepol gene sequences can be used to<br>
integrate any DNA into a genome. The primate T-lymphotropic virus and sequences of the<br>
present invention infect human cells, and thus, these sequences are used with other foreign<br>
or exogenous sequences in humans in methods, including, but not limited to, entry into<br>
cells, packaging, and insertion into the genome. Additionally, methods of using the primate<br>
T-lymphotropic virus and sequences of the present invention are not limited to human cells,<br>
but all cells that allow for infection or entry of the nucleic acids.<br>
65.	The present invention is directed to compositions and methods comprising a new<br>
primate T-lymphotropic virus, HTLV-3 or HTLV-4, particularly compositions and methods<br>
for the sequences of the viral genome. The virus was obtained from humans. The new<br>
virus of the present invention can be an excellent vector for gene therapy and for<br>
vaccination purposes. Additionally, the antibodies or other detection methods for detecting<br>
the new virus may be important in detecting the presence of this and related viruses for<br>
xenotransplantation. In addition, the novel primate T-lymphotropic virus of the present<br>
invention can be used as a reagent in pathogenicity studies of these and related viruses.<br>
Moreover, the sequences of the novel primate T-lymphotropic virus of the present invention<br>
can be used as probes to detect virus in biological samples. Vectors include but are not<br>
limited to prokaryotic, eukaryotic and viral vectors.<br>
66.	Many new and potentially useful technologies are being developed which use<br>
viral vectors and may form the basis of future medical cures and therapies. Examples of<br>
such technologies include, but are not limited to, gene replacement, antisense gene therapy,<br>
in situ drug delivery, treatment of cancer or infectious agents, and vaccine therapy.<br>
However, to be successful, these technologies require an effective means for the delivery of the genetic information across cellular membranes.<br>
67. It is well-known in the art that vaccinations can be used prophylacticly for the prevention of infections as well as therapeutically for the treatment of ongoing conditions. Such infections or conditions can be but are not limited to viral infections. Thus, also disclosed are vectors of the invention, wherein the antigen-encoding nucleic acid is an antigen from a virus. The viral antigen can be selected from the group of viruses consisting of Herpes simplex virus type-1, Herpes simplex virus type-2, Cytomegalovirus, Epstein-Barr virus, Varicella-zoster virus, Human herpesvirus 6, Human herpesvirus 7, Human herpesvims 8, Variola virus, Vesicular stomatitis virus, Hepatitis A virus, Hepatitis B virus, Hepatitis C virus, Hepatitis D virus, Hepatitis E virus, Rhinovirus, Coronavirus, Influenza virus A, Influenza virus B, Measles virus, Polyomavirus, Human Papilomavirus, Respiratory syncytial virus, Adenovirus, Coxsackie virus, Dengue virus, Mumps virus, Poliovirus, Rabies virus, Rous sarcoma vims, Yellow fever virus, Ebola virus, Marburg virus, Lassa fever virus, Eastern Equine Encephalitis virus, Japanese Encephalitis virus, St. Louis Encephalitis virus, Murray Valley fever virus, West Nile virus, Rift Valley fever virus, Rotavirus A, Rotavirus B, Rotavirus C, Sindbis virus, Simian Immunodeficiency virus, Human T-lymphotropic virus type-1, Human T-lymphotropic virus type-2, Primate T-lymphotropic virus, Hantavirus, Rubella virus, Simian Immunodeficiency virus, Human Immunodeficiency virus type-1, and Human Immunodeficiency virus type-2, and Simian Immunodeficiency virus (SIV). Also disclosed are vectors of the invention, wherein the antigen-encoding nucleic acid is SW-GAG. The art is replete with examples of viral antigens whose sequences and methods of obtaining them are well known.<br>
68. Vaccinations are also known for the prevention of bacterial infections. Additionally, antibiotics are well-known in the art for the treatment of various bacterial infections. Herein contemplated and disclosed are vectors of the invention, wherein the antigen-encoding nucleic acid is an antigen from a bacterium. The bacterial antigen can be selected from the group consisting of M. tuberculosis, M. bovis, M. bavis strain BCG, BCG substrains, M. avium, M. intracellulare, M. qfricanum, M. kansasii, M. marinum, M. ulcerans, M. avium subspecies paratuberculosis, Nocardia asteroides, other Nocardia species, Legionella pneumophila, other Legionella species, Salmonella typhi, other Salmonella species, Shigella species, Yersiniapestis, Pasteurella haemofytica, Pasteuretta multocida, other Pasteurella species, Actinobacillus pleuropneumoniae, Listeria<br>
monocyiogenes, Listena ivanovii, tiruceua aoortus, ow&amp;caruceiia species, ruminantium, Chlamydia pneumoniae, CMamydia trachomatis, Chlamydia psittaci, Coxiella burnetti, other Rickettsial species, Ehrlichia species, Staphylococcus aureus, Staphylococcus epidermidis, Streptococcus pyogenes, Streptococcus agalactiae, Bacillus anthracis, Escherichia coli, Vibrio cholerae, Campylobacter species, Neiserria meningitidis, Neiserria gonorrhea, Pseudomonas aeruginosa, other Pseudomonas species, Haemophilus influenzae, Haemophilus ducreyi, other Hemophilus species, Clostridium tetani, other Clostridium species, Yersinia enterolitica, and other Yersinia species. The art is replete with examples of bacterial antigens whose sequences and methods of obtaining them are well known.<br>
69.	Vaccinations are also known for the prevention of fungal infections.<br>
Additionally, antibiotics are well-known in the art for the treatment of various fungal<br>
infections. Herein contemplated and disclosed are vectors of the invention, wherein the<br>
antigen-encoding nucleic acid is an antigen from a fungus. The fungal antigen can be<br>
selected from the group consisting of Candida albicans, Cryptococcus neoformans,<br>
Histoplama capsulation, Aspergillus fumigatus, Coccidiodes immitis, Paracoccidiodes<br>
brasiliensis, Blastomyces dermitidis, Pneomocystis carnii, Penicillium marneffi, and<br>
Alternaria alternata<br>
70.	The vectors of the invention are not limited to fungi, bacteria, and viruses. Also<br>
disclosed are vectors of the invention, wherein the antigen-encoding nucleic acid is an<br>
antigen from a parasite. The parasitic antigen can be selected from the group consisting of<br>
Toxoplasma gondii, Plasmodium falciparum, Plasmodium vivax, Plasmodium malariae,<br>
other Plasmodium species., Trypanosoma brucei, Trypanosoma cruzi, Leishmania major,<br>
other Leishmania species., Schistosoma mansoni, other Schistosoma species., and<br>
Entamoeba histofytica. The art is replete with examples of parasitic antigens whose<br>
sequences and methods of obtaining them are well known.<br>
71.	There are instances wherein it is advantageous to administer the vector of the<br>
invention in a pharmaceutical composition that comprises other vaccines. Pharmaceutical<br>
compositions comprising multiple vaccines can be for therapeutic or prophylactic purposes.<br>
An example Examples of such a composition is other vaccines include the mumps,<br>
measles, rubella (MMR) vaccine, and vaccines against M. tuberculosis, M. bovis, M. bovis strain BCG, BCG substrains, M. avium, M. intracellulare, M. africanum, M. kansasii, M. marinum, M. ulcerans, M. avium subspecies paratubercuhsis, Nocardia asteroides, other<br>
Nocardia species, Legionella pneumophila, other Legionella species, Salmonella typhi, other Salmonella species, Shigella species, Yersinia pestis, Pasteurella haemolytica, Pasteurella multocida, other Pasteurella species, Actinobacillus pleuropneumoniae, Listeria monocytogenes, Listeria ivanovii, Brucella abortus, other Brucella species, Cowdria ruminantium, Chlamydiapneumoniae, Chlamydia trachomatis, Chlamydia psittaci, Coxiella bumetti, other Rickettsial species, Ehrlichia species, Staphylococcus aureus, Staphylococcus epidermidis, Streptococcus pyogenes, Streptococcus agalactiae, Bacillus anthracis, Escherichia coli, Vibrio cholerae, Campylobacter species, Neiserria meningitidis, Neiserria gonorrhea, Pseudomonas aeruginosa, other Pseudomonas species, Haemophilus influenzae, Haemophilus ducreyi, other Hemophilus species, Clostridium tetani, other Clostridium species,Yersinia enterolitica, and other Yersinia species. Specifically contemplated and disclosed are pharmaceutical compositions comprising the vector of the invention and one or more additional vaccines. Also disclosed and herein contemplated are instances in which the vector of the invention comprises more than one antigen-encoding nucleic acid. In such a situation the vector of the invention will produce each antigen encoded in the vector as a separate antigen.<br>
72.	There are instances in which a vector of the invention alone may not be suitable<br>
for a given purpose (e.g., A kit designed to screen potential drugs for the treatment of a<br>
condition such kit, intended for use in laboratories without the capabilities to transfect a<br>
cell-line with the vector). In such cases, cells previously transfected with the vector of the<br>
invention are needed. Thus, also disclosed are cells comprising the vector of the invention.<br>
73.	La an embodiment of the invention the antigen-encoding nucleic acid can encode<br>
a non-antigenic sequence of DNA. This sequence can provide a functional copy of a<br>
disrupted, mutated, disregulated or deleted gene. Examples of nucleic acids encoding<br>
proteins that play a role in genetic disorders are known in the literature relating to genetic<br>
disorders which is incorporated herein by reference. Methods of making these cells are<br>
described and exemplified herein and in the art.<br>
74.	The ability to detect the presence of a construct can be a desirable feature of any<br>
vector. As such, vectors are often contain a marker to show that the construct of interest has<br>
been delivered to the subject (e.g., a cell), and once delivered, is being expressed. A marker<br>
can take the form of a gene that is detectable when expressed. Thus, also disclosed are<br>
vectors of the invention further comprising a reporter gene. One example of a reporter gene<br>
is green fluorescence protein (GFP).<br>
2. Delivery of the compositions to cells<br>
75.	There are a number of compositions and methods which can be used to deliver<br>
nucleic acids to cells, either in vitro or in vivo. These methods and compositions can<br>
largely be broken down into two classes: viral based delivery systems and non-viral based<br>
delivery systems.  For example, the nucleic acids can be delivered through a number of<br>
direct delivery systems such as, electroporation, lipofection, calcium phosphate<br>
precipitation, plasmids, viral vectors, viral nucleic acids, phage nucleic acids, phages,<br>
cosmids, or via transfer of genetic material in cells or carriers such as cationic liposomes.<br>
Appropriate means for transfection, including viral vectors, chemical transfectants, or<br>
physico-mechanical methods such as electroporation and direct diffusion of DNA, are<br>
described by, for example, Wolff, J. A., et al., Science, 247,1465-1468, (1990); and Wolff,<br>
J. A. Nature, 352, 815-818, (1991)Such methods are well known in the art and readily<br>
adaptable for use with the compositions and methods described herein. In certain cases, the<br>
methods will be modified to specifically function with large DNA molecules. Further, these<br>
methods can be used to target certain diseases and cell populations by using the targeting<br>
characteristics of the carrier.<br>
a) Nucleic acid based delivery systems<br>
76.	Transfer vectors can be anynucleotide construction used to deliver genes into<br>
cells (e.g., a plasmid), or as part of a general strategy to deliver genes, e.g., as part of<br>
recombinant retrovirus or adenovirus (Ram et al. Cancer Res. 53:83-88, (1993)).<br>
77.	As used herein, plasmid or viral vectors are agents that transport nucleic acids<br>
into the cell without degradation and include a promoter yielding expression of the gene in<br>
the cells into which it is delivered. In some embodiments the vectors are derived from<br>
either a virus or specifically a retrovirus. Viral vectors can include for example, for<br>
example, HTLV-1, HTLV-2, HTLV-3, HTLV-4, Adenovirus, Adeno-associated virus,<br>
Herpes virus, Vaccinia virus, Polio virus, ADDS virus, neuronal trophic virus, Sindbis and<br>
other RNA viruses, including these viruses with the HTV backbone. Also preferred are any<br>
viral families which share the properties of these viruses which make them suitable for use<br>
as vectors. Retroviruses include Murine Maloney Leukemia virus, MMLV, and retroviruses<br>
that express the desirable properties of MMLV as a vector. Retroviral vectors are able to<br>
carry a larger genetic payload, i.e., a transgene or marker gene, than other viral vectors, and<br>
for this reason are a commonly used vector. However, they are not as useful in non-<br>
proliferating cells. Adenovirus vectors are relatively stable and easy to work with, have<br>
high titers, and can be delivered in aerosol formulation, and can transfect non-dividing cells. Pox viral vectors are large and have several sites for inserting genes, they are thermostable and can be stored at room temperature. A preferred embodiment is a viral vector which has been engineered so as to suppress the immune response of the host organism, elicited by the viral antigens. Preferred vectors of this type will carry coding regions for Interleukin 8 or 10.<br>
78.	Viral vectors can have higher transaction (ability to introduce genes) abilities<br>
than chemical or physical methods to introduce genes into cells. Typically, viral vectors<br>
contain, nonstructural early genes, structural late genes, an RNA polymerase HI transcript,<br>
inverted terminal repeats necessary for replication and encapsidation, and promoters to<br>
control the transcription and replication of the viral genome. When engineered as vectors,<br>
viruses typically have one or more of the early genes removed and a gene or gene/promoter<br>
cassette is inserted into the viral genome in place of the removed viral DNA. Constructs of<br>
this type can carry up to about 8 kb of foreign genetic material. The necessary functions of<br>
the removed early genes are typically supplied by cell lines which have been engineered to<br>
express the gene products of the early genes in trans.<br>
3.  Retroviral Vectors<br>
79.	Primate T-lymphotropic viruses are retroviruses. A retrovirus is an animal virus<br>
belonging to the virus family of Retro viridae, including any types, subfamilies, genus, or<br>
tropisms. Retroviral vectors, in general, are described by Verma, I.M., Retroviral vectors<br>
for gene transfer. In Microbiology-1985, American Society for Microbiology, pp. 229-232,<br>
Washington, (1985), which is incorporated by reference herein. Examples of methods for<br>
using retroviral vectors for gene therapy are described in U.S. Patent Nos. 4,868,116 and<br>
4,980,286; PCT applications WO 90/02806 and WO 89/07136; and Mulligan, (Science<br>
260:926-932 (1993)); the teachings of which are incorporated herein by reference.<br>
Although the present primate T-lymphotropic virus vector is unique, the methods described<br>
for using other types of viral vectors can be useful in certain contexts. See for example U.S.<br>
Patent No. 5,646,032, which is incorporated herein for its teaching of those methods.<br>
80.	A retrovirus is essentially a package which has packed into it nucleic acid cargo.<br>
The nucleic acid cargo carries with it a packaging signal, which ensures that the replicated<br>
daughter molecules will be efficiently packaged within the package coat. In addition to the<br>
package signal, there are a number of molecules which are needed in cis, for the replication,<br>
and packaging of the replicated virus. Typically a retroviral genome, contains the gag.pol,<br>
and env genes which are involved in the making of the protein coat. It is the gag, pol, and env genes which are typically replaced by the foreign DNA that is to be transferred to the target cell. Retrovirus vectors typically contain a packaging signal for incorporation into the package coat, a sequence which signals the start of the gag transcription unit, elements necessary for reverse transcription, including a primer binding site to bind the tRNA primer of reverse transcription, terminal repeat sequences that guide the switch of RNA strands during DNA synthesis, a purine rich sequence 5' to the 3' LTR that serves as the priming site for the synthesis of the second strand of DNA synthesis, and specific sequences near the ends of the LTRs that enable the insertion of the DNA state of the retrovirus to insert into the host genome. The removal of the gag, pol, and env genes allows for large fragments of foreign sequence to be inserted into the viral genome, become reverse transcribed, and upon replication be packaged into a new retroviral particle. This amount of nucleic acid is sufficient for the delivery of a one to many genes depending on the size of each transcript. It is preferable to include either positive or negative selectable markers along with other genes in the insert.<br>
81.	Since the replication machinery and packaging proteins in most retroviral vectors<br>
have been removed (gag, pol, and env), the vectors are typically generated by placing them<br>
into a packaging cell line. A packaging cell line is a cell line which has been transfected or<br>
transformed with a retrovirus that contains the replication and packaging machinery, but<br>
lacks any packaging signal. When the vector carrying the DNA of choice is transfected into<br>
these cell lines, the vector containing the gene of interest is replicated and packaged into<br>
new retroviral particles, by the machinery provided in cis by the helper cell. The genomes<br>
for the machinery are not packaged because they lack the necessary signals.<br>
82.	A packaging cell line is a cell line which has been transfected or transformed<br>
with a retrovirus that contains the replication and packaging machinery, but lacks any<br>
packaging signal. When the vector carrying the DNA of choice is transfected into these cell<br>
lines, the vector containing the gene of interest is replicated and packaged into new<br>
retroviral particles, by the machinery provided in cis by the helper cell. The genomes for<br>
the machinery are not packaged because they lack the necessary signals.<br>
83.	It is also understood and herein contemplated that the pX region can be used to<br>
construct a vector. The pX region is located between the end of env and the beginning of<br>
Tax and contains small ORFs hence this can be another good region for insertion of foreign<br>
DNA in an HTLV genome based vector.<br>
84.	Disclosed are methods of detecting the expression of the vector of the invention<br>
comprising using a first antibody to the antigen to measure protein expression in a<br>
quantitative or qualitative way, further comprising detecting the first antibody directly via a<br>
colorimetric measurement produced through the use of a substrate and a conjugated<br>
antibody or indirectly via a first antibody to the antigen which in turn is bound by a second<br>
antibody which is conjugated and will result in a colorimetric measurement when combined<br>
with a substrate.<br>
85.	Also disclosed are methods of the invention, wherein the antigen is detected by<br>
placing an aliquot of the vector of the invention in a lane on a gel and probing the gel for the<br>
antigen.<br>
86.	Disclosed are methods of detecting the expression of the vector of the invention<br>
comprising using a fluorescently labeled first antibody specific for the antigen and<br>
visualizing the antigen using a flow cytometer, fluorescence microscope, or<br>
chemiluminescence.<br>
87.	Also disclosed are methods of the invention, wherein the first antibody is not<br>
fluorescently labeled, but a target for a second antibody with a fluorescent label.<br>
88.	Disclosed are methods of detecting the expression of a disclosed vector<br>
comprising using cytolytic killing assay to assess activity.<br>
89.	Disclosed are methods of detecting the vector of the invention further<br>
comprising obtaining a sample from a subject comprising a tissue biopsy or removal of<br>
blood or bone marrow.<br>
a) Non-nucleic acid based systems<br>
90.	The disclosed compositions can be delivered to the target cells in a variety of<br>
ways. For example, the compositions can be delivered through electroporation, or through<br>
lipofection, or through calcium phosphate precipitation. The delivery mechanism chosen<br>
will depend in part on the type of cell targeted and whether the delivery is occurring for<br>
example in vivo or in vitro.<br>
91.	In the methods described above which include the administration and uptake of<br>
exogenous DNA into the cells of a subject (i.e., gene transduction or transfection), delivery<br>
of the compositions to cells can be via a variety of mechanisms. As one example, delivery<br>
can be via a b'posome, using commercially available liposome preparations such as<br>
LIPOFECTIN, LIPOFECTAMINE (GBCO-BRL, Inc., Gaithersburg, MD), SUPEKFECT<br>
(Qiagen, Inc. Hilden, Germany) and TRANSFECTAM (Promega Biotec, Inc., Madison,<br><br>
WT), as well as other liposomes developed according to procedures standard in the art. In addition, the disclosed nucleic acid or vector can be delivered in vivo by electroporation, the technology for which is available from Genetronics, Inc. (San Diego, CA) as well as by means of a SONOPORATION machine (ImaRx Pharmaceutical Corp., Tucson, AZ).<br>
92. The materials may be in solution, suspension (for example, incorporated into microparticles, liposomes, or cells). These may be targeted to a particular cell type via antibodies, receptors, or receptor ligands. The following references are examples of the use of this technology to target specific proteins to tumor tissue (Senter, et al., Bioconjugate Chem., 2:447-451, (1991); Bagshawe, K.D., Br. J. Cancer, 60:275-281, (1989); Bagshawe, et al., Br. J. Cancer, 58:700-703, (1988); Senter, et al., Bioconjugate Chem., 4:3-9, (1993); Battelli, et al., Cancer Immunol. Immunother., 35:421-425, (1992); Pietersz and McKenzie, Immunolog. Reviews, 129:57-80, (1992); and Roffler, et al., Biochem. Pharmacol, 42:2062-2065, (1991)). These techniques can be used for a variety of other specific cell types. Vehicles such as "stealth" and other antibody conjugated liposomes (including lipid mediated drug targeting to colonic carcinoma), receptor mediated targeting of DNA through cell specific ligands, lymphocyte directed tumor targeting, and highly specific therapeutic retroviral targeting of murine glioma cells in vivo. The following references are examples of the use of this technology to target specific proteins to tumor tissue (Hughes et al., Cancer Research, 49:6214-6220, (1989); and Litzinger and Huang, Biochimica et Biophysica Acta, 1104:179-187, (1992)). In general, receptors are involved in pathways of endocytosis, either constitutive or ligand induced. These receptors cluster in clathrin-coated pits, enter the cell via clathrin-coated vesicles, pass through an acidified endosome in which the receptors are sorted, and then either recycle to the cell surface, become stored intracellularly, or are degraded in lysosomes. The internalization pathways serve a variety of functions, such as nutrient uptake, removal of activated proteins, clearance of macromolecules, opportunistic entry of viruses and toxins, dissociation and degradation of ligand, and receptor-level regulation. Many receptors follow more than one intracellular pathway, depending on the cell type, receptor concentration, type of ligand, ligand valency, and ligand concentration. Molecular and cellular mechanisms of receptor-mediated endocytosis has been reviewed (Brown and Greene, DNA and Cell Biology 10:6,399-409 (1991)).<br>
93. Nucleic acids that are delivered to cells which are to be integrated into the host cell genome, typically contain integration sequences. These sequences are often viral<br>
related sequences, particularly when viral based systems are used. These viral integration systems can also be incorporated into nucleic acids which are to be delivered using a non-nucleic acid based system of deliver, such as a liposome, so that the nucleic acid contained in the delivery system can be come integrated into the host genome.<br>
94.	Other general techniques for integration into the host genome include, for<br>
example, systems designed to promote homologous recombination with the host genome.<br>
These systems typically rely on sequence flanking the nucleic acid to be expressed that has<br>
enough homology with a target sequence within the host cell genome that recombination<br>
between the vector nucleic acid and the target nucleic acid takes place, causing the delivered<br>
nucleic acid to be integrated into the host genome. These systems and the methods<br>
necessary to promote homologous recombination are known to those of skill in the art.<br>
b) In vivo/ex vivo<br>
95.	As described above, the compositions can be administered in a pharmaceutically<br>
acceptable carrier and can be delivered to the subject's cells in vivo and/or ex vivo by a<br>
variety of mechanisms well known in the art (e.g., uptake of naked DNA, liposome fusion,<br>
intramuscular injection of DNA via a gene gun, endocytosis and the like).<br>
96.	If ex vivo methods are employed, cells or tissues can be removed and maintained<br>
outside the body according to standard protocols well known in the art. The compositions<br>
can be introduced into the cells via any gene transfer mechanism, such as, for example,<br>
calcium phosphate mediated gene delivery, electroporation, microinjection or<br>
proteoliposomes. The transduced cells can then be infused (e.g., in a pharmaceutically<br>
acceptable carrier) or homo topically transplanted back into the subject per standard methods<br>
for the cell or tissue type. Standard methods are known for transplantation or infusion of<br>
various cells into a subject.<br>
4.  Expression systems<br>
97.	The nucleic acids that are delivered to cells typically contain expression<br>
controlling systems. For example, the inserted genes in viral and retroviral systems usually<br>
contain promoters, and/or enhancers to help control the expression of the desired gene<br>
product. A promoter is generally a sequence or sequences of DNA that function when in a<br>
relatively fixed location in regard to the transcription start site. A promoter contains core<br>
elements required for basic interaction of RNA polymerase and transcription factors, and<br>
may contain upstream elements and response elements.<br>
a) Viral Promoters and Enhancers<br>
98.	Preferred promoters controlling transcription from vectors in mammalian host<br>
cells may be obtained from various sources, for example, the genomes of viruses such as:<br>
polyoma, Simian Virus 40 (SV40), adenovirus, retroviruses, hepatitis-B virus and most<br>
preferably cytomegalovirus, or from heterologous mammalian promoters, e.g. beta actin<br>
promoter. The early and late promoters of the SV40 virus are conveniently obtained as an<br>
SV40 restriction fragment which also contains the SV40 viral origin of replication (Fiers et<br>
al., Nature, 273: 113 (1978)). The immediate early promoter of the human<br>
cytomegalovirus is conveniently obtained as a HindIII E restriction fragment (Greenway,<br>
PJ. et al., Gene 18: 355-360 (1982)). Of course, promoters from the host cell or related<br>
species also are useful herein. Such preferred promoters are in the LTRs of HTLV.<br>
99.	Enhancer generally refers to a sequence of DNA that functions at no fixed<br>
distance from the transcription start site and can be either 5' (Laimins, L. et al., Proc. Natl.<br>
Acad. Sci. 78: 993 (1981)) or 3' (Lusky, M.L., et al., Mol. Cell Bio. 3: 1108 (1983)) to the<br>
transcription unit. Furthermore, enhancers can be within an intron (Banerji, J.L. et al., Cell<br>
33: 729 (1983)) as well as within the coding sequence itself (Osbome, T.F., et al., Mol. Cell<br>
Bio. 4: 1293 (1984)). They are usually between 10 and 300 bp in length, and they function<br>
in cis. Enhancers function to increase transcription from nearby promoters. Enhancers<br>
also often contain response elements that mediate the regulation of transcription. Promoters<br>
can also contain response elements that mediate the regulation of transcription. Enhancers<br>
often determine the regulation of expression of a gene. While many enhancer sequences are<br>
now known from mammalian genes (globin, elastase, albumin, fetoprotein and insulin),<br>
typically one will use an enhancer from a eukaryotic cell virus for general expression.<br>
Preferred examples are the SV40 enhancer on the late side of the replication origin (bp<br>
100-270), the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late<br>
side of the replication origin, and adenovirus enhancers.<br><br>
100.	The promoter and/or enhancer may be specifically activated either by light or<br>
specific chemical events which trigger their function. Systems can be regulated by reagents<br>
such as tetracycline and dexamethasone. There are also ways to enhance viral vector gene<br>
expression by exposure to irradiation, such as gamma irradiation, or alkylating<br>
chemotherapy drugs.<br>
101.	In certain embodiments the promoter and/or enhancer region can act as a<br>
constitutive promoter and/or enhancer to maximize expression of the region of the<br>
transcription unit to be transcribed. In certain constructs the promoter and/or enhancer<br>
region be active in all eukaryotic cell types, even if it is only expressed in a particular type of cell at a particular time. A preferred promoter of this type is the CMV promoter (650 bases). Other preferred promoters are SV40 promoters, cytomegalovirus (full length promoter), and retrovtral vector LTR.<br>
102.	It has been shown that all specific regulatory elements can be cloned and<br>
used to construct expression vectors that are selectively expressed hi specific cell types such<br>
as melanoma cells.  The glial fibrillary acetic protein (GFAP) promoter has been used to<br>
selectively express genes in cells of glial origin.<br>
103.	Expression vectors used in eukaryotic host cells (yeast, fungi, insect, plant,<br>
animal, human or nucleated cells) may also contain sequences necessary for the termination<br>
of transcription which may affect mRNA expression. These regions are transcribed as<br>
polyadenylated segments in the untranslated portion of the mRNA encoding tissue factor<br>
protein. The 3' untranslated regions also include transcription termination sites. It is<br>
preferred that the transcription unit also contain a polyadenylation region. One benefit of<br>
this region is that it increases the likelihood that the transcribed unit will be processed and<br>
transported like mRNA.  The identification and use of polyadenylation signals hi<br>
expression constructs is well established. It is preferred that homologous polyadenylation<br>
signals be used in the transgenc constructs. In certain transcription units, the<br>
polyadenylation region is derived from the SV40 early polyadenylation signal and consists<br>
of about 400 bases. It is also preferred that the transcribed units contain other standard<br>
sequences alone or hi combination with the above sequences improve expression from, or<br>
stability of, the construct.<br>
b) Markers<br>
104.	The viral vectors can include nucleic acid sequence encoding a marker<br>
product. This marker product is used to determine if the gene has been delivered to the cell<br>
and once delivered is being expressed. Preferred marker genes are the E. Coli lacZ gene,<br>
which encodes B-galactosidase, and green fluorescent protein.<br>
105.	In some embodiments the marker may be a selectable marker.  Examples of<br>
suitable selectable markers for mammalian cells are dihydrofolate reductase (DHFR),<br>
thymidine Idnase, neomycin, neomycin analog G418, hydromycin, and puromycin. When<br>
such selectable markers are successfully transferred into a mammalian host cell, the<br>
transformed mammalian host cell can survive if placed under selective pressure. There are<br>
two widely used distinct categories of selective regimes. The first category is based on a<br>
cell's metabolism and the use of a mutant cell line which lacks the ability to grow independent of a supplemented media. Two examples are: CHO DHFR- cells and mouse LTK- cells. These cells lack the ability to grow without the addition of such nutrients as thymidine or hypoxanthine. Because these cells lack certain genes necessary for a complete nucleotide synthesis pathway, they cannot survive unless the missing nucleotides are provided in a supplemented media. An alternative to supplementing the media is to introduce an intact DHFR or TK gene into cells lacking the respective genes, thus altering their growth requirements. Individual cells which were not transformed with the DHFR or TK gene will not be capable of survival in non-supplemented media.<br>
106.	The second category is dominant selection which refers to a selection scheme<br>
used in any cell type and does not require the use of a mutant cell line. These schemes<br>
typically use a drug to arrest growth of a host cell. Those cells which have a novel gene<br>
would express a protein conveying drag resistance and would survive the selection.<br>
Examples of such dominant selection use the drugs neomycin, (Southern P. and Berg, P.,_J.<br>
Molec. Appl. Genet. 1: 327 (1982)), mycophenolic acid, (Mulligan, R.C. and Berg, P.<br>
Science 209: 1422 (1980)) or hygromycin, (Sugden, B. et al., Mol. Cell. Biol. 5: 410-413<br>
(1985)). The three examples employ bacterial genes under eukaryotic control to convey<br>
resistance to the appropriate drug G418 or neomycin (geneticin), xgpt (mycophenolic acid)<br>
or hygromycin, respectively. Others include the neomycin analog G418 and puramycin.<br>
5.  Sequence similarities<br>
107.	It is understood that as discussed herein the use of the terms homo logy and<br>
identity mean the same thing as similarity. Thus, for example, if the use of the word<br>
homology is used between two non-natural sequences it is understood that this is not<br>
necessarily indicating an evolutionary relationship between these two sequences, but rather<br>
is looking at the similarity or relatedness between their nucleic acid sequences. Many of the<br>
methods for determining homology between two evolutionarily related molecules are<br>
routinely applied to any two or more nucleic acids or proteins for the purpose of measuring<br>
sequence similarity regardless of whether they are evolutionarily related or not.<br>
108.	In general, it is understood that one way to define any known variants and<br>
derivatives or those that might arise, of the disclosed genes and proteins herein, is through<br>
defining the variants and derivatives in terms of homology to specific known sequences.<br>
This identity of particular sequences disclosed herein is also discussed elsewhere herein. In<br>
general, variants of genes and proteins herein disclosed typically have at least, about 70,71,<br>
72, 73, 74,75, 76, 77, 78, 79, 80, 81,82, 83, 84, 85, 86, 87, 88, 89,90,91, 92, 93,94,95, 96,97,98, or 99 percent homology to the stated sequence or the native sequence. Those of skill in the art readily understand how to determine the homology of two proteins or nucleic acids, such as genes. For example, the homology can be calculated after aligning the two sequences so that the homology is at its highest level.<br>
109.	Another way of calculating homology can be performed by published<br>
algorithms. Optimal alignment of sequences for comparison may be conducted by the local<br>
homology algorithm of Smith and Waterman Adv. Appl. Math. 2: 482 (1981), by the<br>
homology alignment algorithm of Needleman and Wunsch, J. MoL Biol. 48:443 (1970), by<br>
the search for similarity method of Pearson and Lipman, Proc. Natl. Acad. Sci. U.S.A. 85:<br>
2444 (1988), by computerized implementations of these algorithms (GAP, BESTFJT,<br>
FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer<br>
Group, 575 Science Dr., Madison, WT), or by inspection.<br>
110.	The same types of homology can be obtained for nucleic acids by for<br>
example the algorithms disclosed in Zuker, M. Science 244:48-52,1989, Jaeger et al. Proc.<br>
Natl. Acad. Sci. USA 86:7706-7710,1989, Jaeger et al. Methods Enzymol. 183:281-306,<br>
1989 which are herein incorporated by reference for at least material related to nucleic acid<br>
alignment. It is understood that any of the methods typically can be used and that in certain<br>
instances the results of these various methods may differ, but the skilled artisan understands<br>
if identity is found with at least one of these methods, the sequences would be said to have<br>
the stated identity, and be disclosed herein.<br>
111.	For example, as used herein, a sequence recited as having a particular percent<br>
homology to another sequence refers to sequences that have the recited homology as<br>
calculated by any one or more of the calculation methods described above. For example, a<br>
first sequence has 80 percent homology, as defined herein, to a second sequence if the first<br>
sequence is calculated to have 80 percent homology to the second sequence using the Zuker<br>
calculation method even if the first sequence does not have 80 percent homology to the<br>
second sequence as calculated by any of the other calculation methods. As another<br>
example, a first sequence has 80 percent homology, as defined herein, to a second sequence<br>
if the first sequence is calculated to have 80 percent homology to the second sequence using<br>
both the Zuker calculation method and the Pearson and Lipman calculation method even if<br>
the first sequence does not have 80 percent homology to the second sequence as calculated<br>
by the Smith and Waterman calculation method, the Needleman and Wunsch calculation<br>
method, the Jaeger calculation methods, or any of the other calculation methods. As yet another example, a first sequence has 80 percent homology, as defined herein, to a second sequence if the first sequence is calculated to have 80 percent homology to the second sequence using each of calculation methods (although, in practice, the different calculation methods will often result in different calculated homology percentages). 6.  Nucleic acids<br>
112.	There are a variety of molecules disclosed herein that are nucleic acid based,<br>
including for example the nucleic acids that encode, for example HTLV-3 or HTLV-4. The<br>
disclosed nucleic acids are made up of, for example, nucleotides, nucleotide analogs, or<br>
nucleotide substitutes. Non-limiting examples of these and other molecules are discussed<br>
herein. It is understood that for example, when a vector is expressed in a cell, that the<br>
expressed mRNA will typically be made up of A, C, G, and U. Likewise, it is understood<br>
that if, for example, an antisense molecule is introduced into a cell or cell environment<br>
through for example exogenous delivery, it is advantageous that the antisense molecule be<br>
made up of nucleotide analogs that reduce the degradation of the antisense molecule in the<br>
cellular environment.<br>
a) Nucleotides and related molecules<br>
113.	A nucleotide is a molecule that contains a. base moiety, a sugar moiety and a<br>
phosphate moiety. Nucleotides can be linked together through their phosphate moieties and<br>
sugar moieties creating an internucleoside linkage. The base moiety of a nucleotide can be<br>
adenin-9-yl (A), cytosin-1-yl (C), guanin-9-yl (G), uracil-1-yl (U), and thymin-1-yl (T). The<br>
sugar moiety of a nucleotide is a ribose or a deoxyribose. The phosphate moiety of a<br>
nucleotide is pentavalent phosphate. An non-Limiting example of a nucleotide would be 3'-<br>
AMP (3'-adenosine monophosphate) or 5'-GMP (S'-guanosine monophosphate). There are<br>
many varieties of these types of molecules available in the art and available herein.<br>
114.	A nucleotide analog is a nucleotide which contains some type of<br>
modification to either the base, sugar, or phosphate moieties. Modifications to nucleotides<br>
are well known in the art and would include for example, 5-methylcytosine (5-me-C),<br>
5-hydroxymethyl cytosine, xanthine, hypoxanthine, and 2-aminoadenine as well as<br>
modifications at the sugar or phosphate moieties. There are many varieties of these types of<br>
molecules available in the art and available herein.<br>
115.	Nucleotide substitutes are molecules having similar functional properties to<br>
nucleotides, but which do not contain a phosphate moiety, such as peptide nucleic acid<br>
(PNA). Nucleotide substitutes are molecules that will recognize nucleic acids in a Watson-Crick or Hoogsteen manner, but which are linked together through a moiety other than a phosphate moiety. Nucleotide substitutes are able to conform to a double helix type structure when interacting with the appropriate target nucleic acid. There are many varieties of these types of molecules available in the art and available herein.<br>
116.	It is also possible to link other types of molecules (conjugates) to nucleotides<br>
or nucleotide analogs to enhance for example, cellular uptake. Conjugates can be<br>
chemically linked to the nucleotide or nucleotide analogs. Such conjugates include but are<br>
not limited to lipid moieties such as a cholesterol moiety. (Letsinger et al., Proc. Natl. Acad.<br>
Sci. USA, 1989,86, 6553-6556). There are many varieties of these types of molecules<br>
available in the art and available herein.<br>
117.	A Watson-Crick interaction is at least one interaction with the Watson-Crick<br>
face of a nucleotide, nucleotide analog, or nucleotide substitute. The Watson-Crick face of<br>
a nucleotide, nucleotide analog, or nucleotide substitute includes the C2, Nl, and C6<br>
positions of a purine based nucleotide, nucleotide analog, or nucleotide substitute and the<br>
C2, N3, C4 positions of a pyrimidine based nucleotide, micleotide analog, or nucleotide<br>
substitute.<br>
118.	A Hoogsteen interaction is the interaction that takes place on the Hoogsteen<br>
face of a nucleotide or nucleotide analog, which is exposed in the major groove of duplex<br>
DNA. The Hoogsteen face includes the N7 position and reactive groups (NH2 or O) at the<br>
C6 position of purine nucleotides.<br>
b) Sequences<br>
119.	There are a variety of sequences related to the protein molecules, for example<br>
the protein coding regions gag, pol, env, tax, rex, and protease (pro) genes and noncoding<br>
regions such as the LTR of HTLV-3 and HTLV-4, or any of the nucleic acids disclosed<br>
herein for making HTLV-3 or HTLV-4, all of which are encoded by nucleic acids or are<br>
nucleic acids. The sequences for the human analogs of these genes, as well as other<br>
analogs, and alleles of these genes, and splice variants and other types of variants, are<br>
available in a variety of protein and gene databases, including GenBank. Those sequences<br>
available at the time of filing this application at GenBank are herein incorporated by<br>
reference in their entireties as well as for individual subsequences contained therein.<br>
GenBank can be accessed at http://www.ncbi.nili.gov/entrez/query.fcgi. Those of skill in<br>
the art understand how to resolve sequence discrepancies and differences and to adjust the<br>
compositions and methods relating to a particular sequence to other related sequences. Primers and/or probes can be designed for any given sequence given the information disclosed herein and known in the art.<br>
c) Primers and probes<br>
120.    Disclosed are compositions including primers and probes, which are capable of interacting with the disclosed nucleic acids, such as the HTLV-3 or HTLV-4 as disclosed herein. In certain embodiments the primers are used to support nucleic acid (DNA, RNA, etc.) amplification reactions. Thus, for example, disclosed herein are primers wherein the primer comprises SEQ ID NOs: 7 and 8, SEQ ID NOs: 11 and 12, SEQ ID NOs: 15 and 16, SEQ ID NOs: 23 and 24, SEQ ID NOs: 27 and 28, SEQ ID NOs: 31 and 32, SEQ ID NOs: 69 and 70, SEQ ID NOs: 73 and 74, SEQ ID NOs: 77 and 78, SEQ ID NOs: 9 and 10, SEQ ID NOs: 13 and 14, SEQ ID NOs: 17 and 18, SEQ ID NOs: 25 and 26, SEQ ID NOs: 29 and 30, SEQ ID NOs: 33 and 34, SEQ ID NOs: 64 and 65, SEQ ID NOs: 71 and 72, SEQ ID NOs: 75 and 76, and SEQ ID NOs: 79 and 80. Typically the primers will be capable of being extended in a sequence specific manner. Extension of a primer in a sequence specific manner includes any methods wherein the sequence and/or composition of the nucleic acid molecule to which the primer is hybridized or otherwise associated directs or influences the composition or sequence of the product produced by the extension of the primer. Extension of the primer in a sequence specific manner therefore includes, but is not limited to, PCR, DNA sequencing, DNA extension, DNA polymerization, RNA transcription, or reverse transcription. Techniques and conditions that amplify the primer in a sequence specific manner are preferred. In certain embodiments the primers are used for the DNA amplification reactions, such as PCR or direct sequencing. Thus, herein are disclosed primer pairs used in conjunction with a second nested set of primers pairs. For example, disclosed herein are PCR amplification methods comprising a first primer pair and a second primer pair, wherein the second primer pair is internal to the first primer pair and wherein the first primer pair is selected from the group consisting of SEQ ID NOs: 7 and 8, SEQ ID NOs: 11 and 12, SEQ ID NOs: 15 and 16, SEQ ID NOs: 23 and 24, SEQ ID NOs: 27 and 28, SEQ ID NOs: 31 and 32, SEQ ID NOs: 69 and 70, SEQ ID NOs: 73 and 74, and SEQ ID NOs: 77 and 78, wherein the second set of primers is selected from the group consisting of SEQ ID NOs: 9 and 10, SEQ ID NOs: 13 and 14, SEQ ID NOs: 17 and 18, SEQ ID NOs: 25 and 26, SEQ ID NOs: 29 and 30, SEQ ID NOs: 33 and 34, SEQ ID NOs: 71 and 72, SEQ ID NOs: 75 and 76, and SEQ ID NOs: 79 and 80It is understood that in certain<br>
embodiments the primers can also be extended using non-enzymatic techniques, where for example, the nucleotides or oligonucleotides used to extend the primer are modified such<br>
that they will chemically react to extend the primer in a sequence specific manner.<br>
j<br>
Typically the disclosed primers hybridize with the disclosed nucleic acids or region of the nucleic acids or they hybridize with the complement of the nucleic acids or complement of a region of the nucleic acids.<br>
d) Functional Nucleic Acids<br>
121.	Functional nucleic acids are nucleic acid molecules that have a specific<br>
function, such as binding a target molecule or catalyzing a specific reaction. Functional<br>
nucleic acid molecules can be divided into the following categories, which are not meant to<br>
be limiting. For example, functional nucleic acids include antisense molecules, aptamers,<br>
ribozymes, triplex forming molecules, and external guide sequences. The functional nucleic<br>
acid molecules can act as affectors, inhibitors, modulators, and stimulators of a specific<br>
activity possessed by a target molecule, or the functional nucleic acid molecules can possess<br>
a de novo activity independent of any other molecules.<br>
122.	Functional nucleic acid molecules can interact with any macromolecule, such<br>
as DNA, RNA, polypeptides, or carbohydrate chains. Thus, functional nucleic acids can<br>
interact with the mRNA of any of the disclosed nucleic acids, such as the/&gt;o/, tax, env, gag,<br>
rex andpro genes and noncoding regions such as the LTR of HTLV-3 and HTLV-4, or the<br>
nucleic acids used for the generation of HTLV-3 and HTLV-4, or the genomic DNA of any<br>
of the disclosed viruses, such as HTLV-3 and HTLV-4, or they can interact with the<br>
polypeptide encoded by any of the disclosed nucleic acids, such z&amp;pol, tax, rex, env, gag, or<br>
pro genes of HTLV-3 and HTLV-4, or the nucleic acids used for the generation ofpol, tax,<br>
rex, env, gag, or LTR proteins of HTLV-3 and HTLV-4. Often functional nucleic acids are<br>
designed to interact with other nucleic acids based on sequence homo logy between the<br>
target molecule and the functional nucleic acid molecule. In other situations, the specific<br>
recognition between the functional nucleic acid molecule and the target molecule is not<br>
based on sequence homology between the functional nucleic acid molecule and the target<br>
molecule, but rather is based on the formation of tertiary structure that allows specific<br>
recognition to take place.<br>
7.   Peptides<br>
a) Protein variants<br>
123.    As discussed herein there are numerous variants of the HTLV-3 genes encoded herein such as gag (SEQ ID NO: 40),pol (SEQ ID NO: 44), env (SEQ ID NO: 37), tax (SEQ ID NO: 50), rex (SEQ ID NO: 48), protease (SEQ ID NO: 46) and noncoding regions such as the LTR and HTLV-4 genes encoded herein such as gag, pol (SEQ ID NO: 57), env (SEQ ID NO: 54), tax (SEQ ID NO: 62), rex (SEQ ID NO: 60), protease (SEQ ID NO: 58) and noncoding regions such as the LTR that are known and herein contemplated. In addition, to the known functional HTLV-3 and HTLV-4 strain variants there are derivatives of the HTLV-3 and HTLV-4 gag, pol, tax, rex, and env, LTR proteins which also function in the disclosed methods and compositions. Protein variants and derivatives are well understood to those of skill in the art and in can involve amino acid sequence modifications. For example, amino acid sequence modifications typically fall into one or more of three classes: substitutional, insertional or deletional variants. Insertions include amino and/or carboxyl terminal fusions as well as intrasequence insertions of single or multiple amino acid residues. Insertions ordinarily will be smaller insertions than those of amino or carboxyl terminal fusions, for example, on the order of one to four residues. Immunogenic fusion protein derivatives, such as those described in the examples, are made by fusing a polypeptide sufficiently large to confer immunogenicity to the target sequence by cross-linking in vitro or by recombinant cell culture transformed with DNA encoding the fusion. Deletions are characterized by the removal of one or more amino acid residues from the protein sequence. Typically, no more than about from 2 to 6 residues are deleted at any one site within the protein molecule. These variants ordinarily are prepared by site specific mutagenesis of nucleotides in the DNA encoding the protein, thereby producing DNA encoding the variant, and thereafter expressing the DNA in recombinant cell culture. Techniques for making substitution mutations at predetermined sites in DNA having a known sequence are well known, for example Ml 3 primer mutagenesis and PCR mutagenesis. Amino acid substitutions are typically of single residues, but can occur at a number of different locations at once; insertions usually will be on the order of about from 1 to 10 amino acid residues; and deletions will range about from 1 to 30 residues. Deletions or insertions preferably are made in adjacent pairs, i.e. a deletion of 2 residues or insertion of 2 residues. Substitutions, deletions, insertions or any combination thereof may be combined to arrive at a final construct. The mutations must not place the sequence out of<br><br>
reading frame and preferably will not create complementary regions that could produce secondary mKNA structure. Substitutional variants are those in which at least one residue has been removed and a different residue inserted in its place. Such substitutions generally are made in accordance with the following Tables 1 and 2 and are referred to as conservative substitutions.<br>
124.	TABLE 1: Amino Acid Abbreviations <br>
 (Table Removed)<br>
TABLE 2:Amino Acid Substitutions<br>
Original Residue Exemplary Conservative Substitutions, others are known in the art  <br>
(Table Removed)<br>
125.	Substantial changes in function or immunological identity are made by<br>
selecting substitutions that are less conservative than those in Table 2, i.e., selecting<br>
residues that differ more significantly in their effect on maintaining (a) the structure of the<br>
polypeptide backbone in the area of the substitution, for example as a sheet or helical<br>
conformation, (b) the charge or hydrophobicity of the molecule at the target site or (c) the<br>
bulk of the side chain. The substitutions which in general are expected to produce the<br>
greatest changes in the protein properties will be those in which (a) a hydrophilic residue,<br>
e.g. seryl or threonyl, is substituted for (or by) ahydrophobic residue, e.g. leucyl, isoleucyl,<br>
phenylalanyl, valyl or alanyl; (b) a cysteine or proline is substituted for (or by) any other<br>
residue; (c) a residue having an electropositive side chain, e.g., lysyl, arguiyl, or histidyl, is<br>
substituted for (or by) an electronegative residue, e.g., glutamyl or aspartyl; or (d) a residue<br>
having a bulky side chain, e.g., phenylalanine, is substituted for (or by) one not having a<br>
side chain, e.g., glycine, in this case, (e) by increasing the number of sites for sulfation<br>
and/or glycosylation.<br>
126.	For example, the replacement of one ammo acid residue with another that is<br>
biologically and/or chemically similar is known to those skilled in the art as a conservative<br>
substitution. For example, a conservative substitution would be replacing one hydrophobic<br>
residue for another, or one polar residue for another. The substitutions include<br>
combinations such as, for example, Gly, Ala; Val, Ile, Leu; Asp, Glu; Asn, Gin; Ser, Thr;<br>
Lys, Arg; and Phe, Tyr. Such conservatively substituted variations of each explicitly<br>
disclosed sequence are included within the mosaic polypeptides provided herein.<br>
127.	Substitutional or deletional mutagenesis can be employed to insert sites for<br>
N-glycosylation (Asn-X-Thr/Ser) or O-glycosylation (Ser or Thr). Deletions of cysteine or<br>
other labile residues also may be desirable. Deletions or substitutions of potential<br>
proteolysis sites, e.g. Arg, is accomplished for example by deleting one of the basic residues<br>
or substituting one by glutaminyl or histidyl residues.<br>
128.	Certain post-translational derivatizatiqns are the result of the action of<br>
recombinant host cells on the expressed polypeptide. Glutaminyl and asparaginyl residues<br>
are frequently post-translationally deamidated to the corresponding glutamyl and asparyl<br>
residues. Alternatively, these residues are deamidated under mildly acidic conditions.<br>
Other post-translational modifications include hydroxylation of proline and lysine,<br>
phosphorylation of hydroxyl groups of seryl or threonyl residues, methylation of the o-<br>
amino groups of lysine, arginine, andhistidine side chains (T.E. Creighton, Proteins:<br>
Structure and Molecular Properties, W. H. Freeman &amp; Co., San Francisco pp 79-86 [1983]),<br>
acetylation of the N-terminal amine and, in some instances, amidation of the C-terminal<br>
carhoxyl.<br>
129.	It is understood that one way to define the variants and derivatives of the<br>
disclosed proteins herein is through defining the variants and derivatives in terms of<br>
homology/identity to specific known sequences. For example, SEQ ID NO: 1 sets forth a<br>
particular sequence of HTLV-3 pol protein and SEQ ID NO: 2 sets forth a particular<br>
sequence of a HTLV-4-pol protein. Specifically disclosed are variants of these and other<br>
proteins herein disclosed which have at least, 70% or 75% or 80% or 85% or 90% or 95%<br>
homology or any amount of homology in between to the stated sequence. Those of skill in<br>
the art readily understand how to determine the homology of two proteins. For example, the<br>
homology can be calculated after aligning the two sequences so that the homology is at its<br>
highest level.<br>
130.	Another way of calculating homology can be performed by published<br>
algorithms. Optimal alignment of sequences for comparison may be conducted by the local<br>
homology algorithm of Smith and Waterman Adv. Appl. Math, 2: 482 (1981), by the<br>
homology alignment algorithm of Needleman and Wunsch, J. MoL Biol. 48: 443 (1970), by<br>
the search for similarity method of Pearson and Lipman, Proc. Natl. Acad. Sci. U.S.A. 85:<br>
2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT,<br>
FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer<br>
Group, 575 Science Dr., Madison, WT), or by inspection.<br>
131.	The same types of homology can be obtained for nucleic acids by for<br>
example the algorithms disclosed in Zuker, M. Science 244:48-52,1989, Jaeger et al. Proc.<br>
Natl. Acad. Sci. USA 86:7706-7710,1989, Jaeger et al. Methods Enzymol. 183:281-306,<br>
1989 which are herein incorporated by reference for at least material related to nucleic acid<br>
alignment.<br>
132.	It is understood that the description of conservative mutations and homology<br>
can be combined together in any combination, such as embodiments that have at least 70%<br>
homology to a particular sequence wherein the variants are conservative mutations.<br>
133.	As this specification discusses various proteins and protein sequences it is<br>
understood that the nucleic acids that can encode those protein sequences are also disclosed.<br>
This would include all degenerate sequences related to a specific protein sequence, i.e. all<br>
nucleic acids having a sequence that encodes one particular protein sequence as well as all nucleic acids, including degenerate nucleic acids, encoding the disclosed variants and derivatives of the protein sequences. Thus, while each particular nucleic acid sequence may not be written out herein, it is understood that each and every sequence is in fact disclosed and described herein through the disclosed protein sequence. For example, one of the many nucleic acid sequences that can encode the protein sequence set forth in SEQ ID NO: 44 is set forth in SEQ ID NO: 1. In addition, for example, disclosed are conservative derivatives of SEQ ID NO: 44.<br>
134.	It is understood that there are numerous amino acid and peptide analogs<br>
which can be incorporated into the disclosed compositions. For example, there are<br>
numerous D amino acids or amino acids which have a different functional substituent then<br>
the amino acids shown in Table 1 and Table 2. The opposite stereo isomers of naturally<br>
occurring peptides are disclosed, as well as the stereo isomers of peptide analogs. These<br>
amino acids can readily be incorporated into polypeptide chains by charging tRNA<br>
molecules with the amino acid of choice and engineering genetic constructs that utilize, for<br>
example, amber codons, to insert the analog amino acid into a peptide chain in a site<br>
specific way (Thorson et al., Methods in Molec. Biol. 77:43-73 (1991), Zoller, Current<br>
Opinion in Biotechnology, 3:348-354 (1992); Ibba, Biotechnology &amp; Genetic Engineering<br>
Reviews 13:197-216 (1995), Cahill et al., TIBS, 14(10):400-403 (1989); Benner, TIB Tech,<br>
12:158-163 (1994); Ibba and Hennecke, Bio/technology, 12:678-682 (1994) all of which are<br>
herein incorporated by reference at least for material related to amino acid analogs).<br>
135.	Molecules can be produced that resemble peptides, but which are not<br>
connected via a natural peptide linkage. For example, linkages for amino acids or amino<br>
acid analogs can include CH2NH~, -CH2S«, -CH2~CH2 -, -CH=CH-- (cis and trans), -<br>
COCH2 --, -CH(OH)CH2-, and -CHH2SO—(These and others can be found in Spatola, A.<br>
F. in Chemistry and Biochemistry of Amino Acids, Peptides, and Proteins, B. Weinstein,<br>
eds., Marcel Dekker, New York, p. 267 (1983); Spatola, A. F., Vega Data (March 1983),<br>
Vol. 1, Issue 3, Peptide Backbone Modifications (general review); Morley, Trends Pharm Sci (1980) pp. 463-468; Hudson, D. et al., lot J Pept Prot Res 14:177-185 (1979) (-CH2NH-, CH2CH2-); Spatola et al. Life Sci 38:1243-1249 (1986) (-CH H2-S); Harm J. Chem. Soc Perkin Trans. 1307-314 (1982) (-CH-CH--, cis and trans); Almquist et al. J. Med. Chem. 23:1392-1398 (1980) (-COCH2-); Jennings-White et al. Tetrahedron Lett 23:2533 (1982) (-COCH2-); Szelke et al. European Appln, EP 45665 CA (1982): 97:39405 (1982) (~CH(OH)CH2--); Holladay et al. Tetrahedron. Lett 24:4401-4404 (1983) (»C(OH)CH2--); and Hruby Life Sci 31:189-199 (1982) (--CH2--S~); each of which is incorporated herein by reference. A particularly preferred non-peptide linkage is —CH2NH~. It is understood that peptide analogs can have more than one atom between the bond atoms, such as b-alanine, g-aminobutyric acid, and the like.<br>
136.	Amino acid analogs and analogs and peptide analogs often have enhanced or<br>
desirable properties, such as, more economical production, greater chemical stability,<br>
enhanced pharmacological properties (half-life, absorption, potency, efficacy, etc.), altered<br>
specificity (e.g., a broad-spectrum of biological activities), reduced antigenicity, and others.<br>
137.	D-amino acids can be used to generate more stable peptides, because D<br>
amino acids are not recognized by peptidases and such. Systematic substitution of one or<br>
more amino acids of a consensus sequence with a D-amino acid of the same type (e.g., D-<br>
lysine in place of L-lysine) can be used to generate more stable peptides. Cysteine residues<br>
can be used to cyclize or attach two or more peptides together. This can be beneficial to<br>
constrain peptides into particular conformations. (Rizo and Gierasch Ann. Rev. Biochem.<br>
61:387 (1992), incorporated herein by reference).<br>
8.   Pharmaceutical carriers/Delivery of pharamceutical products<br>
138.	As described above, the compositions can also be administered in vivo in a<br>
pharmaceutically acceptable carrier. By "pharmaceutically acceptable" is meant a material<br>
that is not biologically or otherwise undesirable, i.e., the material may be administered to a<br>
subject, along with the nucleic acid or vector, without causing any undesirable biological<br>
effects or interacting in a deleterious manner with any of the other components of the<br>
pharmaceutical composition in which it is contained. The carrier would naturally be<br>
selected to minimize any degradation of the active ingredient and to minimize any adverse<br>
side effects in the subject, as would be well known to one of skill in the art.<br>
139.	The compositions may be administered orally, parenterally (e.g.,<br>
intravenously), by intramuscular injection, by intraperitoneal injection, transdermally,<br>
extracorporeally, topically or the like, including topical intranasal administration or administration by inhalant. As used herein, "topical intranasal administration" means delivery of the compositions into the nose and nasal passages through one or both of the nares and can comprise delivery by a spraying mechanism or droplet mechanism, or through aerosolization of the nucleic acid or vector. Administration of the compositions by inhalant can be through the nose or mouth via delivery by a spraying or droplet mechanism. Delivery can also be directly to any area of the respiratory system (e.g., lungs) via intubation. The exact amount of the compositions required will vary from subject to subject, depending on the species, age, weight and general condition of the subject, the severity of the allergic disorder being treated, the particular nucleic acid or vector used, its mode of admirdstration and the like. Thus, it is not possible to specify an exact amount for every composition. However, an appropriate amount can be determined by one of ordinary skill in the art using only routine experimentation given the teachings herein.<br>
140.	Parenteral administration of the composition, if used, is generally<br>
characterized by injection. Injectables can be prepared in conventional forms, either as<br>
liquid solutions or suspensions, solid forms suitable for solution of suspension hi liquid<br>
prior to injection, or as emulsions. A more recently revised approach for parenteral<br>
administration involves use of a slow release or sustained release system such that a<br>
constant dosage is maintained. See, e.g., U.S. Patent No. 3,610,795, which is incorporated<br>
by reference herein.<br>
141.	The materials may be in solution, suspension (for example, incorporated into<br>
microparticles, liposomes, or cells). These may be targeted to a particular cell type via<br>
antibodies, receptors, or receptor ligands. The following references are examples of the use<br>
of this technology to target specific proteins to tumor tissue (Senter, et al., Bioconjugate<br>
Ghent., 2:447-451, (1991); Bagshawe, K.D., Br. J. Cancer, 60:275-281, (1989); Bagshawe,<br>
et al., Br. J. Cancer, 58:700-703, (1988); Senter, et al., Bioconjugate Chem., 4:3-9, (1993);<br>
Battelli, et al., Cancer Immunol. Immunother., 35:421-425, (1992); Pietersz and McKenzie,<br>
Imrmmolog. Reviews, 129:57-80, (1992); and Roffler, et al., Biochem. Pharmacol, 42:2062-<br>
2065, (1991)). Vehicles such as "stealth" and other antibody conjugated liposomes<br>
(including lipid mediated drug targeting to colonic carcinoma), receptor mediated targeting<br>
of DNA through cell specific ligands, lymphocyte directed tumor targeting, and highly<br>
specific therapeutic retroviral targeting of murine glioma cells in vivo. The following<br>
references are examples of the use of this technology to target specific proteins to tumor<br>
tissue (Hughes et al., Cancer Research, 49:6214-6220, (1989); and Litzinger and Huang, Biochimica et Biophysica Acta, 1104:179-187, (1992)). In general, receptors are involved in pathways of endocytosis, either constitutive or ligand induced. These receptors cluster in clathrin-coated pits, enter the cell via clathrin-coated vesicles, pass through an acidified endosome in which the receptors are sorted, and then either recycle to the cell surface, become stored intracellularly, or are degraded in lysosomes. The internalization pathways serve a variety of functions, such as nutrient uptake, removal of activated proteins, clearance of macromolecules, opportunistic entry of viruses and toxins, dissociation and degradation of ligand, and receptor-level regulation. Many receptors follow more than one intracellular pathway, depending on the cell type, receptor concentration, type of ligand, ligand valency, and ligand concentration. Molecular and cellular mechanisms of receptor-mediated endocytosis has been reviewed (Brown and Greene, DNA and Cell Biology 10:6,399-409 (1991)).<br>
a) Pharmaceutically Acceptable Carriers<br>
142.	The compositions, including antibodies, can be used therapeutically in<br>
combination with a pharmaceutically acceptable carrier.<br>
143.	Suitable carriers and their formulations are described in Remington: The<br>
Science and Practice of Pharmacy (19th ed.) ed. A.R. Gennaro, Mack Publishing Company,<br>
Easton, PA 1995. Typically, an appropriate amount of a pharmaceutically-acceptable salt is<br>
used in the formulation to render the formulation isotonic. Examples of the<br>
pharmaceutically-acceptable carrier include, but are not limited to, saline, Ringer's solution<br>
and dextrose solution. The pH of the solution is preferably from about 5 to about 8, and<br>
more preferably from about 7 to about 7.5. Further carriers include sustained release<br>
preparations such as semipermeable matrices of solid hydrophobic polymers containing the<br>
antibody, which matrices are in the form of shaped articles, e.g., films, liposomes or<br>
microparticles. It will be apparent to those persons skilled in the art that certain carriers<br>
may be more preferable depending upon, for instance, the route of administration and<br>
concentration of composition being administered.<br>
144.	Pharmaceutical carriers are known to those skilled in the art. These most<br>
typically would be standard carriers for administration of drugs to humans, including<br>
solutions such as sterile water, saline, and buffered solutions at physiological pH. The<br>
compositions can be administered intramuscularly or subcutaneously. Other compounds<br>
will be administered according to standard procedures used by those skilled in the art.<br>
145.	Pharmaceutical compositions may include carriers, thickeners, diluents,<br>
buffers, preservatives, surface active agents and the like in addition to the molecule of choice.<br>
Pharmaceutical compositions may also include one or more active ingredients such as<br>
antimicrobial agents, antiinflammatory agents, anesthetics, and the like.<br>
146.	The pharmaceutical composition may be administered in a number of ways<br>
depending on whether local or systemic treatment is desired, and on the area to be treated.<br>
Administration maybe topically (including ophthalmically, vaginally, rectally, intranasally),<br>
orally, by inhalation, or parenterally, for example by intravenous drip, subcutaneous,<br>
intraperitoneal or intramuscular injection. The disclosed antibodies can be administered<br>
intravenously, intraperitoneally, intramuscularly, subcutaneously, intracavity, or<br>
transdermally.<br>
147.	Preparations for parenteral administration include sterile aqueous or non-<br>
aqueous solutions, suspensions, and emulsions. Examples of non-aqueous solvents are<br>
propylene glycol, polyethylene glycol, vegetable oils such as olive oil, and injectable<br>
organic esters such as ethyl oleate. Aqueous carriers include water, alcoholic/aqueous<br>
solutions, emulsions or suspensions, including saline and buffered media. Parenteral<br>
vehicles include sodium chloride solution, Ringer's dextrose, dextrose and sodium chloride,<br>
lactated Ringer's, or fixed oils. Intravenous vehicles include fluid and nutrient replenishers,<br>
electrolyte replenishers (such as those based on Ringer's dextrose), and the like.<br>
Preservatives and other additives may also be present such as, for example, antimicrobials,<br>
anti-oxidants, cbelating agents, and inert gases and the like.<br>
148.	Formulations for topical administration may include ointments, lotions,<br>
creams, gels, drops, suppositories, sprays, liquids and powders. Conventional pharmaceutical<br>
carriers, aqueous, powder or oily bases, thickeners and the like may be necessary or desirable.<br>
149.	Compositions for oral administration include powders or granules, suspensions<br>
or solutions in water or non-aqueous media, capsules, sachets, or tablets. Thickeners,<br>
flavorings, diluents, emulsifiers, dispersing aids or binders maybe desirable..<br>
150.	Some of the compositions may potentially be administered as a<br>
pharmaceutically acceptable acid- or base- addition salt, formed by reaction with inorganic<br>
acids such as hydrochloric acid, hydrobromic acid, perchloric acid, nitric acid, thiocyanic<br>
acid, sulfuric acid, and phosphoric acid, and organic acids such as formic acid, acetic acid,<br>
propionic acid, glycolic acid, lactic acid, pyruvic acid, oxalic acid, malonic acid, succinic<br>
acid, maleic acid, and fumaric acid, or by reaction with an inorganic base such as sodium<br>
hydroxide, ammonium hydroxide, potassium hydroxide, and organic bases such as mono-, di-, trialkyl and aryl amines and substituted ethanolamines.<br>
b) Therapeutic Uses<br>
151.    Effective dosages and schedules for administering the compositions may be determined empirically, and making such determinations is within the skill in the art. The dosage ranges for the administration of the compositions are those large enough to produce the desired effect in which the symptoms disorder are effected. The dosage should not be so large as to cause adverse side effects, such as unwanted cross-reactions, anaphylactic reactions, and the like. Generally, the dosage will vary with the age, condition, sex and extent of the disease in the patient, route of administration, or whether other drugs are included in the regimen, and can be determined by one of skill in the art. The dosage can be adjusted by the individual physician in the event of any counterindications. Dosage can vary, and can be administered in one or more dose administrations daily, for one or several days. Guidance can be found in the literature for appropriate dosages for given classes of pharmaceutical products. For example, guidance in selecting appropriate doses for antibodies can be found in the literature on therapeutic uses of antibodies, e.g., Handbook of Monoclonal Antibodies, Ferrone et al., eds., Noges Publications, Park Ridge, N J., (1985) ch. 22 and pp. 303-357; Smith et al., Antibodies in Human Diagnosis and Therapy, Haber et al., eds., Raven Press, New York (1977) pp. 365-389. A typical daily dosage of the antibody used alone might range from about 1 ug/kg to up to 100 mg/kg of body weight or more per day, depending on the factors mentioned above.<br>
152.	Following administration of a disclosed composition, such as an antibody,<br>
for treating, inhibiting, or preventing an HTLV-3 or HTLV-4 infection, the efficacy of the<br>
therapeutic antibody can be assessed in various ways well known to the skilled practitioner.<br>
For instance, one of ordinary skill in the art will understand that a composition, such as an<br>
antibody, disclosed herein is efficacious in treating or inhibiting an HTLV-3 or HTLV-4<br>
infection in a subject by observing that the composition reduces viral load or prevents a<br>
further increase hi HTLV-3 or HTLV-4 viral load. Techniques used to measure viral<br>
HTLV-3 or HTLV-4 treatment with an antibody partially or completely inhibits the<br>
appearance of the virus in the blood or other body fluid.<br>
153.	Other molecules that interact with HTLV-3 or HLV-4 or the genes encoded<br>
by those viruses to inhibit HTLV-3 or HTLV-4 and cell interactions which do not have a<br>
specific pharmacuetical function, but which may be used for tracking changes within<br>
cellular chromosomes or for the delivery of diagnostic tools for example can be delivered in ways similar to those described for the pharmaceutical products.<br>
154.	The disclosed compositions and methods can also be used for example as<br>
tools to isolate and test new drug candidates for a variety of primate T-lymphotropic virus<br>
related diseases.<br>
C. Treatment and Prevention Methods<br>
155.	By "treating" is meant an improvement in the disease state (i.e., viral<br>
infection, bacterial infection, parasitic infection, cancer, genetic disorder, or autoimmune<br>
disease) is observed and/or detected upon administration of a substance of the present<br>
invention to a subject. Treatment can range from a positive change in a symptom or<br>
symptoms of the disease to complete amelioration of the viral infection, bacterial infection,<br>
parasitic infection, or cancer (e.g., reduction in severity or intensity of disease, alteration of<br>
clinical parameters indicative of the subject's condition, relief of discomfort or increased or<br>
enhanced function), as detected by art-known techniques. The methods of the present<br>
invention can be utilized to treat an established viral infection, bacterial infection, parasitic<br>
infection, or cancer. One of skill in the art would recognize that viral infection, bacterial<br>
infection, parasitic infection, or cancer refer to conditions characterized by the presence of a<br>
foreign pathogen or abnormal cell growth. Clinical symptoms will depend on the particular<br>
condition and are easily recognizable by those skilled in the art of treating the specific<br>
condition. Treatment methods can include, but are not limited to therapeutic vaccinations.<br>
Thus, disclosed are methods of treating a subject with a condition comprising administering<br>
to the vector of the invention.<br>
156.	Also disclosed are methods of the invention, wherein the condition being<br>
treated is a viral infection. The viral infection can be selected from the list of viruses<br>
consisting of Herpes simplex virus rype-1, Herpes simplex virus type-2, Cytomegalovirus,<br>
Epstein-Barr virus, Varicella-zoster virus, Human herpesvirus 6, Human herpesvirus 7,<br>
Human herpesvirus 8, Variola virus, Vesicular stomatitis virus, Hepatitis A virus, Hepatitis<br>
B virus, Hepatitis C virus, Hepatitis D virus, Hepatitis E virus, Rhinovirus, Coronavirus,<br>
Influenza virus A, Influenza virus B, Measles virus, Polyomavirus, Human Papilomavirus,<br>
Respiratory syncytial virus, Adenovirus, Coxsackie virus, Dengue virus, Mumps virus,<br>
Poliovirus, Rabies virus, Rous sarcoma virus, Yellow fever virus, Ebola virus, Marburg<br>
virus, Lassa fever virus, Eastern Equine Encephalitis virus, Japanese Encephalitis virus, St.<br>
Louis Encephalitis virus, Murray Valley fever virus, West Nile virus, Rift Valley fever<br>
virus, Rotavirus A, Rotavirus B, Rotavirus C, Sindbis virus, Simian Immunodeficiency virus, Human T- lymphotropic virus type-1, Human T-lymphotropic virus type-2, Primate T-lymphotropic virus, Hantavirus, Rubella virus, Simian Immunodeficiency virus, Human Immunodeficiency virus type-1, and Hunian Immunodeficiency virus type-2.<br>
157.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a virus. The viral antigen can be selected from the group of<br>
viruses consisting of Herpes simplex virus type-1, Herpes simplex virus type-2,<br>
Cytomegalovirus, Epstein-Barr virus, Varicella-zoster virus, Human herpesvirus 6, Human<br>
herpesvirus 7, Hunian herpesvirus 8, Variola virus, Vesicular stomatitis virus, Hepatitis A<br>
virus, Hepatitis B virus, Hepatitis C virus, Hepatitis D virus, Hepatitis E virus, Rhinovirus,<br>
Coronavirus, Influenza virus A, Influenza virus B, Measles virus, Polyomaviras, Human<br>
Papilomavirus, Respiratory syncytial virus, Adenovirus, Coxsackie virus, Dengue virus,<br>
Mumps virus, Poliovirus, Rabies virus, Rous sarcoma virus, Yellow fever virus, Ebola<br>
virus, Marburg virus, Lassa fever virus, Eastern Equine Encephalitis virus, Japanese<br>
Encephalitis virus, St. Louis Encephalitis virus, Murray Valley fever virus, West Nile virus,<br>
Rift Valley fever virus, Rotavirus A, Rotavirus B, Rotavirus C, Sindbis virus, Simian<br>
Immunodeficiency virus, Human T-lymphotropic virus type-1, Human T-lymphotropic<br>
virus type-2, Primate T-lymphotropic virus, Hantavirus, Rubella virus, Simian<br>
Immunodeficiency virus, Human Immunodeficiency virus type-1, and Human<br>
Immunodeficiency virus type-2.<br>
158.	Also disclosed are methods of the invention; wherein the condition being<br>
treated is a bacterial infection. The bacterial infection can be selected from the list of<br>
bacterium consisting of M. tuberculosis, M. bovis, M. bovis strain BCG, BCG substrains, M.<br>
avium, M. intracellulare, M. africanum, M. kansasii, M. marintim, M. ulcerans, M. avium<br>
subspecies paratuberculosis, Nocardia asteroides, other Nocardia species, Legionella<br>
pneumophila, other Legionella species, Salmonella typhi, other Salmonella species, Shigella<br>
species, Yersinia pestis, Pasteurella haemofytica, Pasteurella multocida, other Pasteurella<br>
species, Actinobacillus pleuropneumoniae, Listeria monocytogenes, Listeria ivanovii,<br>
Brucetta abortus, other Brucetta species, Cowdria ruminantium, Chlamydia pneumoniae,<br>
Chlamydia trachomatis, Chlamydia psittaci, Coxiella bwnetti, other Rickettsial species,<br>
Ehrlichia species, Staphylococcus aureus, Staphyhcoccus epidermidis, Streptococcus<br>
pyogenes, Streptococcus agalactiae, Bacillus anthracis, Escherichia coli, Vibrio cholerae,<br>
Campylobacter species, Neiserria meningitidis, Neiserria gonorrhea, Pseudomonas<br>
aeruginosa, other Pseudomonas species, Haemophilus influenzae, Haemopkilus ducreyi, other Hemophilus species, Clostridium tetani, other Clostridium species, Yersinia enterolitica, and other Yersinia. species.<br>
159.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a bacterium. The bacterial antigen can be selected from the<br>
group consisting of M. tuberculosis, M. bovis, M. bavis strain BCG, BCG substrains, Af.<br>
avium, M. intracellulare, M. africanum, M. kansasii, M. marinum, M. ulcerans, M. avium<br>
subspecies paratuberculosis, Nocardia asteroides, other Nocardia species, Legionella<br>
pneumophila, other Legionella species, Salmonella typhi, other Salmonella species, Shigella<br>
species, Yersinia pestis, Pasteuretta haemolytica, Pasteurella multocida, other Pasteurella<br>
species, Actinobacillus pleuropneumoniae, Listeria monocytogenes, Listeria ivanovii,<br>
Brucella abortus, other Brucella species, Cowdria ruminantium, Chlamydia pneumoniae,<br>
Chlamydia trachomatis, Chlamydia psittaci, Coxiella burnettt, other Rickettsial species,<br>
Ehrlichia species, Staphylococcus aureus, Staphylococcus epidermidis, Streptococcus<br>
pyogenes, Streptococcus agalactiae, Bacillus anthracis, Escherichia coli, Vibrio cholerae,<br>
Campylobacter species, Neiserria meningitidis, Neiserria gonorrhea. Pseudomonas<br>
aeruginosa, other Pseudomonas species, Haemophilus influenzae, Haemophilus ducreyi,<br>
other Hemophilus species, Clostridium tetani, other Clostridium specics,Yersinia<br>
enterolitica, and other Yersinia species.<br>
160.	Also disclosed are methods of the invention, wherein the condition being<br>
treated is a fungal infection. The fungal infection can he selected from the list of fungus<br>
consisting of Candida albicans, Cryptococcus neoformans, Histoplama capsulatum,<br>
Aspergillusjumigatus, Coccidiodes immitis, Paracoccidiodes brasiliensis, Blastomyces<br>
dermitidis, Pneomocystis carnii, Penicillium marneffi, sndAlternaria alternatas.<br>
161.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a fungus. The fungal antigen can be selected from the group<br>
consisting of Candida albicans, Cryptococcus neoformans, Histoplama capsulatum,<br>
Aspergillusjumigatus, Coccidiodes immitis, Paracoccidiodes brasiliensis, Blastomyces<br>
dermitidis, Pneomocystis carnii, Penicillium marneffi, and Altemaria alternata.<br>
162.	Also disclosed are methods of the invention, wherein the condition being<br>
treated is a parasitic infection. The parasitic infection can be selected from the list of<br>
parasites consisting ofToxoplasma gondii, Plasmodiumfalciparum, Plasmodium vivax,<br>
Plasmodium malariae, other Plasmodium species., Trypanosoma brucei, Trypanosoma<br>
cruzi, Leishmania major, other Leishmania species., Schistosoma mansoni, other Schistosoma species., and Entamoeba histolytica.<br>
163.    Also disclosed are methods of the invention, wherein the antigen-encoding nucleic acid is an antigen from a parasite. The parasitic antigen can be selected from the group consisting of Toxoplasma gondii, Plasmodiumfalciparum, Plasmodium vivax, Plasmodium malariae, other Plasmodium species., Trypanosoma brucei, Trypanosoma cruzi, Leishmania major, other Leishmania species., Schistosoma mansoni, other Schistosoma species., and Entamoeba histolytica.<br>
\ 64.    Also disclosed are methods of the invention, wherein the condition being treated is cancer,<br>
165.	The disclosed vectors and vector containing compositions can be used to<br>
treat any disease where uncontrolled cellular proliferation occurs such as cancers. A non-<br>
limiting list of different types of cancers is as follows: lymphomas (Hodgkins and non-<br>
Hodgkins), leukemias, carcinomas, carcinomas of solid tissues, squamous cell carcinomas,<br>
adenocarcinomas, sarcomas, gliomas, high grade gliomas, blastomas, neuroblastomas,<br>
plasmacytomas, histiocytomas, melanomas, adenomas, hypoxic tumors, myelomas, AIDS-<br>
related lymphomas or sarcomas, metastatic cancers, or cancers in general.<br>
166.	A representative but non-limiting list of cancers that the disclosed<br>
compositions can be used to treat is the following: lymphoma, B cell lymphoma, T cell<br>
lymphoma, mycosis fungoides, Hodgkin's Disease, myeloid leukemia, bladder cancer, brain<br>
cancer, nervous system cancer, head and neck cancer, squamous cell carcinoma of head and<br>
neck, kidney cancer, lung cancers such as small cell lung cancer and non-small cell lung<br>
cancer, neuroblastoma/glioblastoma, ovarian cancer, pancreatic cancer, prostate cancer, skin<br>
cancer, liver cancer, melanoma, squamous cell carcinomas of the mouth, throat, larynx, and<br>
lung, colon cancer, cervical cancer, cervical carcinoma, breast cancer, and epithelial cancer,<br>
renal cancer, genitourinary cancer, pulmonary cancer, esophageal carcinoma, head and neck<br>
carcinoma, large bowel cancer, hematopoietic cancers; testicular cancer; colon and rectal<br>
cancers, prostatic cancer, or pancreatic cancer.<br>
167.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is a tumor antigen. The tumor antigen can be selected from the list consisting<br>
of human epithelial cell mucin (Muc-1; a 20 amino acid core repeat for Muc-1 glycoprotein,<br>
present on breast cancer cells and pancreatic cancer cells), the Ha-ras oncogene product,<br>
p53, carcmo-embryonic antigen (CEA), the raf oncogene product, gpl00/pmell7 GD2,<br>
GD3, GM2, TF, sTn, MAGE-1, MAGE-3, BAGE, GAGE, tyrosinase, gp75, Melan-A/Mart-1, gpl00, HER2/neu, EBV-LMP 1 &amp; 2, HPV-F4,6, 7, prostate-specific antigen (PSA), HPV-16, MUM, alpha-fetoprotein (AFP), CO17-1A, GA733, gp72, p53, the ras oncogene product, HPV E7, Wilm's tumor antigen-1, telomerase, and melanoma gangliosides.<br>
168.	Disclosed are methods of treating a condition in a subject comprising<br>
administering to the subject the vector of the invention, wherein the condition is due to a<br>
mutated, disregulated, disrupted, or deleted gene; autoimmunity; or inflammatory diseases.<br>
169.	Disclosed are methods of treating a condition in a subject, wherein the<br>
condition can be selected from list consisting of cystic fibrosis, asthma, multiple sclerosis,<br>
muscular dystrophy, diabetes, tay-sachs, spinobifida, cerebral palsy, parkinson's disease, lou<br>
gehrigg disease, alzheimer's, systemic lupus erythamatosis, hemophelia, Addsion's disease,<br>
Cushing's disease.<br>
170.	By "preventing" is meant that after administration of a substance of the<br>
present invention to a subject, the subject does not develop the symptoms of the viral,<br>
bacterial, or parasitic infection, and/or does not develop the viral, bacterial, or parasitic<br>
infection. "Preventing" or "prevention" can also refer to the ultimate reduction of an<br>
infection, condition, or symptoms of an infection, or condition relative to infections or<br>
conditions in subjects that do not receive the substance. Methods of prevention can include,<br>
but are not limited to profilactic vaccination. As such, disclosed are methods of preventing<br>
an infection in a subject comprising administering to the subject the vector of the invention.<br>
171.	Also disclosed are methods of the invention, wherein the infection prevented<br>
is a viral infection. The viral infection can be selected from the list of viruses consisting of<br>
Herpes simplex virus type-1, Herpes simplex virus type-2, Cytomegalovirus, Epstein-Barr<br>
virus, Varicella-zoster virus, Human herpesvirus 6, Human herpesvirus 7, Human<br>
herpesvirus 8, Variola virus, Vesicular stomatitis virus, Hepatitis A virus, Hepatitis B virus,<br>
Hepatitis C virus, Hepatitis D virus, Hepatitis E virus, Rhinovirus, Coronavirus, Influenza<br>
virus A, Influenza virus B, Measles virus, Polyomavirus, Human Papilomavirus,<br>
Respiratory syncytial virus, Adenovirus, Coxsackie virus, Dengue virus, Mumps virus,<br>
Poliovirus, Rabies virus, Rous sarcoma virus, Yellow fever virus, Ebola virus, Marburg<br>
virus, Lassa fever virus, Eastern Equine Encephalitis virus, Japanese Encephalitis virus, St.<br>
Louis Encephalitis virus, Murray Valley fever virus, West Nile virus, Rift Valley fever<br>
virus, Rotavirus A, Rotavirus B, Rotavirus C, Sindbis virus, Simian Immunodeficiency<br>
virus, Human T-lymphotropic virus type-1, Human T-lymphotropic virus type-2, Primate T-<br>
lymphotropic virus, Hantavirus, Rubella virus, Simian Immunodeficiency virus, Human Immunodeficiency virus type-1, and Human Immunodeficiency virus type-2.<br>
172.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a virus. The viral antigen can be selected from the group<br>
consisting of Herpes simplex virus type-1, Herpes simplex VUTIS type-2, Cytomegalovirus,<br>
Epstein-Barr virus, Varicella-zoster virus, Human herpesvirus 6, Human herpesvirus 7,<br>
Human herpesvirus 8, Variola virus, Vesicular stomatitis virus, Hepatitis A virus, Hepatitis<br>
B virus, Hepatitis C virus, Hepatitis D virus, Hepatitis E virus, Rhinovirus, Coronavirus,<br>
Influenza virus A, Influenza virus B, Measles virus, Polyomavirus, Human Papilomavirus,<br>
Respiratory syncytial virus, Adenovirus, Coxsackie virus, Dengue virus, Mumps virus,<br>
Poliovirus, Rabies virus, Rous sarcoma virus, Yellow fever virus, Ebola virus, Marburg<br>
virus, Lassa fever virus, Eastern Equine Encephalitis virus, Japanese Encephalitis virus, St.<br>
Louis Encephalitis virus, Murray Valley fever virus, West Nile virus, Rift Valley fever<br>
virus, Rotavirus A, Rotavirus B, Rotavirus C, Sindbis virus, Simian Immunodeficiency<br>
virus, Human T-lymphotropic virus type-1, Human T-lymphotropic virus type-2, Primate T-<br>
lymphotropic virus, Hantavirus, Rubella virus, Simian Immunodeficiency virus, Human<br>
Immunodeficiency virus type-1, and Human Immunodeficiency virus type-2.<br>
173.	Also disclosed are methods of the invention, wherein the infection prevented<br>
is a bacterial infection. The bacterial infection can be selected from the list of bacterium<br>
consisting of M tuberculosis, M. bovis, M. bovis strain BCG, BCG substrains, M. avium, M.<br>
intracellulare, M. africanum, M, kansasti, M. marinum, M. ulcerans, M. avium subspecies<br>
paratuberculosis, Nocardia asteroides, other Nocardia species, Legionella pneumophila,<br>
other Legionella species, Salmonella typhi, other Salmonella species, Shigella species,<br>
Yersiniapestis, Pasteurella haemolytica, Pasteurella multocida, other Pasteurella species,<br>
Actinobacillus pleuropneumoniae, Listeria monocytogenes, Listeria ivanovii, Brucella<br>
abortus, other Brucella species, Cowdria ruminantium, Chlamydia pneumoniae, Chlamydia<br>
trachomatis, Chlamydia psittaci, Coxietta burnetti, other Rickettsial species, Ehrlichia<br>
species, Staphylococcus aureus, Staphylococcus epidermidis, Streptococcus pyogenes,<br>
Streptococcus agalactiae, Bacillus anthracis, Escherichia coli, Vibrio cholerae,<br>
Campylobacter species, Neiserria meningitidis, Neiserria gonorrhea, Pseudomonas<br>
aeruginosa, other Pseudomonas species, Haemophilus influenzae, Haemophilus ducreyi,<br>
other Hemophilus species, Clostridium tetani, other Clostridium species, Yersinia<br>
enterolitica, and other Yersinia species.<br>
174.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a bacterium. The bacterial antigen can be selected from the<br>
group consisting of M. tuberculosis, M. bovis, M. bovis strain BCG, BCG substrains, M.<br>
aviian, M. intracellulare, M. africanum, M. kansasii, M. marinum, M. ulcerans, M. avium<br>
subspecies paratuberculosis, Nocardia asteroides, other Nocardia species, Legionella<br>
pneumophila, other Legionella species, Salmonella typhi, other Salmonella species, ShigeUa<br>
species, Yersinia pestis, Pasteurella haemolytica, Pasteurellamultocida, other Pasteurella<br>
species, Actinobacittus pleuropneumoniae, Listeria monocytogenes, Listeria ivanovii,<br>
Brucella abortus, other Brucella species, Cowdria naninantium, Chlamydia pneumoniae,<br>
Chlamydia trachomatis, Chlamydia psittaci, Coxiella burnetti, other Rickettsial species,<br>
Ehrlichia species, Staphylococcus aureus, Staphylococcus epidermidis, Streptococcus<br>
pyogenes, Streptococcus agalactiae, Bacillus anthracis, Escherichia coli, Vibrio cholerae,<br>
Campylobacter species, Neiserria meningitidis, Neiserria gonorrhea, Pseudomonas<br>
aeruginosa, other Pseudomonas species, Haemophilus influenzae, Haemophilus ducreyi,<br>
other Hemophilus species, Clostridiwn tetani, other Clostridium species, Yersinia<br>
enterolitica, and other Yersinia species.<br>
175.	Also disclosed are methods of the invention, wherein the infection prevented<br>
is a fungal infection. The fungal infection can be selected from the list of Candida albicans,<br>
Cryptococcus neoformans, Histoplama capsulation, Aspergillusfianigaius, Coccidiodes<br>
immitts, Paracoccidiodes brasiliensis, Blastomyces dermitidis, Pneomocystis carnii,<br>
Penicillium marneffi, and Alternaria alternata.<br>
176.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a fungus. The fungal antigen can be selected from the group<br>
consisting of Candida albicans, Cryptococcus neoformans, Histoplama capsulatum,<br>
Aspergillus fumigatus, Coccidiodes immitis, Paracoccidiodes brasiliensis, Blastomyces<br>
dermitidis, Pneomocystis carnii, Penicillium marneffi, and Alternaria alternata.<br>
177.	Also disclosed are methods of the invention, wherein the infection prevented<br>
is a parasitic infection. The parasitic infection can be selected from the list of parasites<br>
consisting of Toxoplasma gondii, Plasmodium falciparum, Plasmodium vivax, Plasmodium<br>
malariae, other Plasmodium species., Trypanosoma brucei, Trypanosoma cruzi,<br>
Leishmania major, other Leishmania species., Schistosoma mansoni, other Schistosoma<br>
species., and Entamoeba histolytica.<br>
178.	Also disclosed are methods of the invention, wherein the antigen-encoding<br>
nucleic acid is an antigen from a parasite. The parasitic antigen can be selected from the<br>
group consisting of Toxoplasma gondii, Plasmodium falciparum, Plasmodiurn vivax,<br>
Plasmodium malariae, other Plasmodium species., Trypanosoma brucei, Trypanosoma<br>
cruzi, Leishmania major, other Leishmania species., Schistosoma mansoni, other<br>
Schistosoma species., and Entamoeba histolytica.<br>
179.	Also disclosed are methods of the invention, wherein the subject is a horse,<br>
cow, pig, dog, car, mouse, monkey, human, or a cell isolated from such an animal.<br>
D. Screening Methods<br>
180.	Disclosed herein are methods of identifying new primate T-lymphotropic<br>
viruses comprising: a) contacting a nucleic acid using a first set of primers and a second set<br>
of primers internal to the first set of primers, wherein the first set of primers is SEQ ID<br>
NOs: 19 and 20, and wherein the second set of primers is SEQ ID NOs: 21 and 22 under<br>
conditions that permit primer extension; b) identifying any amplified nucleic acid; and c)<br>
comparing the sequence to known primate T-lymphotropic viral sequences, wherein a<br>
sequence divergence greater than 5% indicates a new virus.<br>
181.	Also disclosed are methods of identifying new primate T-lymphotropic<br>
viruses comprising: a) contacting a nucleic acid using a first set of primers and a second set<br>
of primers internal to the first set of primers, wherein the first set of primers is selected from<br>
the group of primers pairs consisting of SEQ ID NOs: 7 and 8, SEQ ID NOs: 11 and 12,<br>
SEQ ID NOs: 15 and 16, SEQ ID NOs: 23 and 24, SEQ ID NOs: 27 and 28, SEQ ID NOs:<br>
31 and 32, SEQ ID NOs: 69 and 70, SEQ ID NOs: 73 and 74, and SEQ ID NOs: 77 and 78,<br>
wherein the second set of primers is selected from the group consisting of SEQ ID NOs: 9<br>
and 10, SEQ ID NOs: 13 and 14, SEQ ID NOs: 17 and 18, SEQ ID NOs: 25 and 26, SEQ<br>
ID NOs: 29 and 30, SEQ ED NOs: 33 and 34, SEQ ID NOs: 71 and 72, SEQ ID NOs: 75<br>
and 76, and SEQ ID NOs: 79 and 80; b) identifying any amplified nucleic acid; and c)<br>
comparing the sequence to known primate T-lymphotropic viral sequences, wherein<br>
sequence divergence greater than 5% indicates a new virus.<br>
182.	It is also understood that the disclosed methods of identifying a new primate<br>
T-lyrnphotrophic virus can be achieved using non-nested PCR techniques such as real-time<br>
PCR. Thus, for example, specifically disclosed are methods of identifying new primate T-<br>
lymphotropic viruses comprising a) contacting a nucleic acid using a set of primers, wherein<br>
the set of primers is selected from the set of primers consisting of SEQ ID NOs: 19 and 20,<br>
SEQ ID NOs: 21 and 22, SEQ ID NOs: 7 and 8, SEQ ID NOs: 11 and 12, SEQ ID NOs: 15 and 16, SEQ ID NOs: 23 and 24, SEQ ID NOs: 27 and 28, SEQ ID NOs: 31 and 32, SEQ ID NOs: 69 and 70, SEQ ID NOs: 73 and 74, SEQ ID NOs: 77 and 78, SEQ ID NOs: 9 and 10, SEQ ID NOs: 13 and 14, SEQ ID NOs: 17 and 18, SEQ ID NOs: 25 and 26, SEQ ID NOs: 29 and 30, SEQ ID NOs: 33 and 34, SEQ ID NOs: 71 and 72, SEQ ID NOs: 75 and 76, SEQ ID NOs: 79 and 80, and SEQ ID NOs: 64 and 65; b) identifying any amplified nucleic acid; and c) comparing the sequence to known primate T-lymphotropic viral sequences, wherein sequence divergence greater than 5% indicates a new virus. Also disclosed are identification methods wherein the method is a real-time PCR method.<br>
183.	It is also contemplated that the disclosed methods can be used in conjunction<br>
with probes to detect the presence of amplification product. Specifically disclosed are<br>
fluorescently labeled probes that can be used to detect the amplification product of the<br>
disclosed methods. For example, a fluorescent probe, can comprise<br>
TTCCCCAAGGCTTCAAAAACAGCCCCACGC (SEQ ID NO: 66).<br>
184.	It is understood and herein contemplated that the surface antigen (SU) and<br>
transmembrane are regions ofenv can be used serologically for the identification and<br>
differentiation of PTLVs (the type specific peptides MTA-1 and K55 are in SU; likewise the<br>
p24 region of gag can be used for the serological identification of PTLV). Thus, disclosed<br>
herein are methods of identifying a PTLV comprising contacting a nucleic acid with a set of<br>
primers specific for the surface antigen or transmembrane regions of env and indentifying<br>
any amplified nucleic acid.<br>
185.	It is also understood that the disclosed peptides, polypeptides, proteins, and<br>
protein fragments can be used to generate antibodies that can be used to identify new and<br>
known primate T-lymphotropic viruses. Specifically disclosed are methods of identifying<br>
the presence of a primate T-lymphotropic virus in a subject comprising taking a tissue<br>
sample from the subject and contacting the sample with an antibody directed to an HTLV-3<br>
or HTLV-4 peptide, polypeptide, protein, or protein fragment, wherein the peptide,<br>
polypeptide, protein, or protein fragment can be selected from the group consisting of SEQ<br>
ID NOs. 1,2, 3, 4, 5, 6,35, 36, 37, 38, 39,40, 41,42, 43, 44,45,46,47, 48, 49, 50, 51, 52,<br>
53,54, 55, 56, 57, 58, 59, 60, 61,62, 63,67, and 68, and wherein biding of the antibody to<br>
the sample indicates the presence of a new or known primate T-lymphotropic virus. It is<br>
understood and herein contemplated that the disclosed methods can be used to identify new<br>
primate T-lymphotropic viruses as well as detect all primate T-lymphotropic viruses or a<br>
group of particular primate T-lymphotropic viruses. Those of skill in the art will know which antibodies to use to accomplish their detection goal. For example, to detect more than one of the known HTLV viruses (HTLV-1,2, and 3, or HTLV-1,2, and 4) one can use type specific peptide of HTLV-1 and HTLV-2 such as SEQ ID NO: 67 and 68.<br>
186.	Also provided by the present invention is a method of screening a substance<br>
for effectiveness in treating or reducing the severity of the condition (e.g., HTLV-3 or<br>
HTLV-4 infection) comprising: a) obtaining an animal having the condition or characteristic<br>
(e.g., symptom) of the condition; b) administering the substance to an animal having one or<br>
more characteristics of the condition; and assaying the animal for an effect on the condition,<br>
thereby identifying a substance effective in reducing the condition.<br>
187.	The ability of a substance to reduce the severity of a condition can be<br>
determined by evaluating the histological and clinical manifestations, of the animal with<br>
condition before and after administration of the substance of interest and quantitating the<br>
amount of reduction of the condition.<br>
188.	The animal in which the condition or characteristic (e.g., symptom) of the<br>
condition is produced can be any mammal and can include but is not limited to mouse, rat,<br>
guinea pig, hamster, rabbit, cat, dog, goat, monkey, and chimpanzee. The condition or<br>
characteristic (e.g., symptom) of the condition can be produced in the animal by any method<br>
known in the art. For example, HTLV-3 or HTLV-4 can be produced by introducing into<br>
the animal (eg., chimpanzee infected with HTLV-3 or HTLV-4 or rhesus macaques or<br>
nemestrina macaques infected with an HTLV-3 or HTLV-4 env on an SIV backbone.<br>
Pullium, JK, et. al., J. Infectious Dis. 183:1023, 2001) an infectious amount of HTLV-3 or<br>
HTLV-4.<br>
189.	The present invention also provides a method of screening for a substance<br>
effective in preventing the condition (e.g., HTLV-3 or HTLV-4 infection) comprising: a)<br>
administering the substance to an animal susceptible to the condition; b) subjecting the<br>
animal to treatment that will induce the condition or characteristic (e.g., symptom) of the<br>
condition; and c) assaying cells from the animal for an change in immune responses as<br>
compared to an the immune responses in a control animal having the condition in the<br>
absence of the substance identifies a substance that is effective in preventing the condition.<br>
190.	A model for use in screening for substances effective in treating or<br>
preventing a disease comprising an animal capable of manifesting a characteristic of the<br>
disease is provided, wherein the animal has been administered the vector of the invention.<br>
191.	A method of making the model of the invention comprising obtaining<br>
administering to an animal capable of manifesting a characteristic of the disease and<br>
administering to said animal the vector of the invention which encodes an antigen associate<br>
with the disease.<br>
192.	A method of screening for a substance effective in treating a disease<br>
associated with an immunizing construct comprising: a) administering the substance to the<br>
model of the invention; and b) assaying for an change in the course of the disease as<br>
compared to an the course of the disease in a control subject; an improvement in the course<br>
of the disease hi the presence of the substance identifies a substance that is effective in<br>
treating the disease.<br>
193.	A method of screening for a substance effective in preventing a disease<br>
associated with an immunizing construct comprising: a) administering a the vector of the<br>
invention to a subject; b) subjecting the subject to treatment that will induce the disease or<br>
characteristic (e.g., symptom) of the disease; and c) assaying for an change in the course of<br>
the disease as compared to an the course of the disease in a control subject; an improvement<br>
in the course of the disease in the presence of the substance identifies a substance that is<br>
effective in preventing the disease.<br>
194.	A method of screening for a substance effective hi treating a disease<br>
associated with an immunizing construct comprising: a) subjecting the subject to treatment<br>
that will induce the disease or characteristic (e.g., symptom) of the disease; b) administering<br>
a the vector of the invention to a subject; and c) assaying for an change in the course of the<br>
disease as compared to an the course of the disease in a control subject; an improvement in<br>
the course of the disease in the presence of the substance identifies a substance that is<br>
effective in treating the disease.<br>
1.  Methods of using the compositions as research tools<br>
195.	The disclosed compositions can also be used diagnostic tools related to<br>
primate T-lymphotropic diseases such as HTLV-3 and HTLV-4.<br>
£. Methods of making the compositions<br>
196.	The compositions disclosed herein and the compositions necessary to<br>
perform the disclosed methods can be made using any method known to those of skill in the<br>
art for that particular reagent or compound unless otherwise specifically noted.<br>
1.  Process claims for making the compositions<br>
197.	Disclosed are processes for making the compositions as well as making the<br>
intermediates leading to the compositions. For example, disclosed are nucleic acids in SEQ<br>
ID NOs: 1-6, 35, and 45. There are a variety of methods that can be used for making these<br>
compositions, such as synthetic chemical methods and standard molecular biology methods.<br>
It is understood that the methods of making these and the other disclosed compositions are<br>
specifically disclosed.<br>
198.	Disclosed are nucleic acid molecules produced by the process comprising<br>
linking in an operative way a nucleic acid comprising the sequence set forth in SEQ ID<br>
NOs: 1-6, 35, and 45 and a sequence controlling the expression of the nucleic acid.<br>
199.	Also disclosed are nucleic acid molecules produced by the process<br>
comprising linking in an operative way a nucleic acid molecule comprising a sequence<br>
having 80% identity to a sequence set forth in SEQ ID NOs: 1-6, 35, and 45, and a sequence<br>
controlling the expression of the nucleic acid.<br>
200.	Disclosed are nucleic acid molecules produced by the process comprising<br>
linking in an operative way a nucleic acid molecule comprising a sequence that hybridizes<br>
under stringent hybridization conditions to a sequence set forth SEQ ID NOs: 1-6,35, and<br>
45 and a sequence controlling the expression of the nucleic acid.<br>
201.	Disclosed are nucleic acid molecules produced by the process comprising<br>
Unking in an operative way a nucleic acid molecule comprising a sequence encoding a<br>
peptide set forth in SEQ ID NOs: 1-6, 35, and 45 and a sequence controlling an expression<br>
of the nucleic acid molecule.<br>
202.	Disclosed are nucleic acid molecules produced by the process comprising<br>
Unking in an operative way a nucleic acid molecule comprising a sequence encoding a<br>
peptide having 80% identity to a peptide set forth in SEQ ID NOs: 1-6,35, and 45 and a<br>
sequence controlling an expression of the nucleic acid molecule.<br>
203.	Disclosed are cells produced by the process of transforming the cell with any<br>
of the disclosed nucleic acids. Disclosed are cells produced by the process of transforming<br>
the cell with any of the non-naturally occurring disclosed nucleic acids.<br>
204.	Disclosed are any of the disclosed peptides produced by the process of<br>
expressing any of the disclosed nucleic acids. Disclosed are any of the non-naturally<br>
occurring disclosed peptides produced by the process of expressing any of the disclosed<br>
nucleic acids. Disclosed are any of the disclosed peptides produced by the process of expressing any of the non-naturally disclosed nucleic acids. F. Examples<br>
205.	The following examples are put forth so as to provide those of ordinary skill<br>
La the art with a complete disclosure and description of how the compounds, compositions,<br>
articles, devices and/or methods claimed herein are made and evaluated, and are intended to<br>
be purely exemplary and are not intended to limit the disclosure. Efforts have been made to<br>
ensure accuracy with respect to numbers (e.g., amounts, temperature, etc.), but some errors<br>
and deviations should be accounted for. Unless indicated otherwise, parts are parts by<br>
weight, temperature is in °C or is at ambient temperature, and pressure is at or near<br>
atmospheric.<br>
1.  Example 1<br>
206.	To determine whether novel HTLVs are present among individuals exposed<br>
to the blood and body fluids of wild primate populations (Wolfe et al. 2004a) known to be<br>
infected with STLV (Courgnaud et al. 2004), individuals were examined from twelve<br>
villages in southern Cameroon proximal to both forested and nonforested NHP habitats.<br>
Individuals were asked to identify and quantify their exposure to NHPs, which were<br>
organized according to three categories reliably distinguished by this population:<br>
chimpanzee, gorilla and monkey (Wolfe et al. 2004a). A total of 930 who reported exposure<br>
to NHP blood and body fluids, mainly through hunting and butchering were selected for<br>
further analysis. Plasma specimens from exposed people were screened for PTLVs using an<br>
HTLV-1/2 EIA capable of detecting antibodies to a broad range of PTLVs, followed by<br>
confirmation with an HTLV Western blot (WB) assay that can distinguish HTLV-1 and<br>
HTLV-2 (van Dooren et al. 2004). A total of 97 (10.4%) persons were EIA reactive of<br>
which 90 (9.7%) were also reactive in the WB assay. A broad range of WB profiles were<br>
seen, including HTLV-1-like (1.1 %), HTLV-2-like (0.5 %), HTLV-positive but untypeable<br>
(1.4%), and HTLV indeterminate (6.7%).<br>
207.	DNA from peripheral blood mononuclear cells (PBMCs) available from 86<br>
of the90 WB reactive samples were then subjected to PCR amplification of several viral<br>
regions. Viral sequences from 13 persons were obtained using this strategy. The WB<br>
reactivities of these 13 persons is shown in Fig. 1 and included HTLV-1-like (n=9), HTLV-<br>
2-like (n=l), and HTLV indeterminate (n=3) profiles. All 13 HTLV-infected persons were<br>
exclusively from lowland forest sites, including both men and women who often reported<br>
multiple opportunities for contact with the blood and body fluids of NHPs (Table 3). Since PTLV diversity is influenced more by geography than by primate species (Salerai et al 1999, Slattery et aJ. 1999, Gessain &amp; Mabieux 2000), viral sequences were analyzed phylogenetically along with African and global representatives of HTLV and STLV.<br>
Table 3. Nonhuman primate exposures for human T-lymphotropic virus (HTLV)-infected central African hunters   <br>
(Table Removed)<br>
1	Table 4: Nucleotide and Arnino Acid Percent Identities'  <br>
(Table Removed)<br>
208.    Most notable of the findings was the discovery of a novel human virus that is distinct from all known PTLV lineages with 26-34% and 18-25% nucleotide divergence in the conserved pol and tax genes, respectively, a range of nucleotide divergence similar to that seen between HTLV-1, HTLV-2, and STLV-3 (Meertens at al 2002)(Table 4). This virus formed a separate phylogenetic lineage with a long branch length and significant bootstrap support in both Has pol (Figure 2a: pol tree) and tax trees. Phylogenetic analyses combined with GenBank blast searches show that this is the only known virus in this novel group. For these reasons this virus, which is designate HTLV-4, qualifies as the first member of a novel species in the deltaretrovirus genus. Following ICTV guidelines (van Regenmortel 2000) and pending formal classification, it is proposed that primate T-lymphotropic virus 4 (PTLV-4) be the name for this species, with PTLV-4(1863LE) as the prototype strain. HTLV-4 was found in a 48 year old male hunter (1863LE) from the southern forests of Cameroon who had an HTLV-2-like WB result and reported hunting monkeys, chimpanzees, and gorillas, and also being bitten and scratched by a wild animal, although the animal causing the injury was not specified.<br><br>
209.	Also documented, with significant phylogenetic bootstrap support, is the first<br>
evidence of human infection within the PTLV-3 group (Figure 2a: pol tree). This virus,<br>
which is designated HTLV-3, clusters with STLV-3 viruses present in west African MHPs<br>
as expected (Figure 2d: LTR tree). HTLV-3 was found in a 63 year old male (2026ND)<br>
from the southern forests of Cameroon who had an HTLV-1-like WB result and who<br>
reported hunting and butchering of monkeys. The fact that this virus falls within the<br>
diversity of a group of STLVs first identified in 1994 (Goubau et al 1994) without evidence<br>
of a human counterpart to date, indicates that this infection was most likely acquired<br>
zoonotically through exposure to the blood or body fluids of a hunted NHP from this region<br>
(Courgnaud et al. 2004).<br>
210.	In addition, broad diversity of HTLV-1 viruses was also found in this<br>
collection. Of the 11 HTLV-1 sequences, two did not fall within any of the known HTLV-1<br>
subtypes but clustered clearly within a clade that included only STLV-1 from central and<br>
west Africa (Fig 2d: LTR tree). One of these viruses clustered with STLV-1 from monkeys<br>
in Cameroon and was from a 65 year-old male (2656ND) from the southern forest zone of<br>
Cameroon. He reported hunting and butchering of monkeys and kept a gorilla as a pet<br>
(Table 3). The second virus clustered with STLV-1 recently identified in chimpanzees and<br>
red colobus monkeys (Leendertz et al. 2004) and was from a 55 year old male (281 OYT) who<br>
reported hunting and butchering of monkeys (Table 3). The presence of these viruses in<br>
hunters, seen previously only among NHPs, indicates that these persons were infected<br>
zoonotically. This distinct clade is referred to as HTLV-1 subtype G. Three subjects<br>
(1259NG, 1127MO, 1842LE) from different villages were found to have HTLV-1 subtype<br>
D, viruses known to infect geographically overlapping populations of humans and mandrills<br>
in central Africa (Fig 3 LTR tree (Mahieux et al. 1998). Two of the three viruses were<br>
found in hunters (Table 3), providing indirect evidence of cross-species transmission<br>
between humans and mandrills within subtype D and supporting further the claims of cross-<br>
species transmission of this subtype (Mahieux et al. 1998). These results are consistent with<br>
SFV infection from mandrills that was documented previously in this population (Wolfe et<br>
al 2004) and indicate that the frequent hunting of mandrills may explain the widespread<br>
transmission of mandrill retroviruses. Five persons (979MO, 1380MV, 1443MV, 1503MV,<br>
1537MV) were infected with HTLV-1 subtype B viruses, which are known to be endemic among humans in central Africa and which are believed to have originated from STLV-1 in this region (Mahieux, R. et al 1997, Gessain, A. &amp; Mahieux, R 2000) (Fig 3: LTR tree).<br>
Thus, these five new subtype B viruses may have been acquired either zoonotically from STLV-1-infected primates or from human-to-human transmission, or both. Notably, a 71 year old female (1443MV) who reported butchering gorillas was found to be infected witib a virus most closely related to STLV-1 found in two gorillas s from Cameroon (Nemenet 2004, Courgnaud et al 2004), although without significant bootstrap support (Fig. 3 LTR). Interestingly, person 1503MV is also WB positive for SFV (Wolfe et al 2004), indicating that zoonotic transmission in an individual is not limited to a single retrovirus and providing a biological setting for viral recombination and altered pathogenicity and transmissibility of these viruses. One person (2472LE) was infected with the HTLV-1 subtype A virus, a clade consisting of sequences from only globally disseminated HTLV-1 and thus this infection was most likely acquired through human-to-human transmission. DNA samples from the remaining 73 persons with reactive WB results were all negative by the generic PCR assay for tax sequences and four other sequences specific for each PTLV clade, including HTLV-4. The results presented in this study demonstrate that HTLV diversity is far greater than previously understood. The data indicates that contact with the blood and body fluids of NHPs is a major factor in the emergence of novel HTLVs, which are known to be transmissible among humans and have the potential to cause disease. Because the hunting and butchering of wild NHPs is widespread throughout central Africa (Bowen-Jones &amp; Pendry 1999) and STLVs are known to be highly prevalent among hunted NHPs (Courgnaud et al. 2004), it is suspected that zoonotic transmission of STLV is not a restricted risk. Since blood banks in central Africa do not generally screen for HTLV, further spread of these viruses among central Africans maybe facilitated by blood donations from infected persons. While a recent simian origin could be recognized for at least 6 of the 13 viruses identified, the origin of HTLV-4 is presently unclear since this virus does not have a known primate counterpart. The possibility that HTLV-4 represents a previously unrecognized virus being transmitted between humans indicates that more substantial screening for this virus in central African populations is needed. The finding that both HTLV-4 and HTLV-3 are serologically indistinguishable from HTLV-1 and HTLV-2 in current assays can explain why these viruses have not been previously identified, and highlight the importance of improved diagnostic assays. The increasing evidence that primate hunting is associated with the emergence of a range of simian retroviruses (Wolfe et al. 2004b) calls for increased surveillance and follow-up of individuals exposed to the blood and body fluids of wild NHPs, and for effective strategies to control the hunting of NHPs.<br>
a) Methods<br>
(1)	Ethical approvals;<br>
211.	Studies were conducted in the context of a community-based HTV prevention<br>
campaign designed to provide information using Cameroonian educators and counselors and<br>
therefore to decrease transmission. Participation in the study was completely voluntary. The<br>
study protocol was approved by the Johns Hopkins Committee for Human Research, the<br>
Cameroon National Ethical Review Board, and the HIV Tri-Services Secondary Review<br>
Board. Questionnaires and matching samples were anonymized by removing all personal<br>
identifiers to provide an unlinked study population.<br>
(2)	Sample preparation and serology.<br>
212.	Blood was collected from participants, transported to a central laboratory,<br>
processed into plasma and PBMC aliquots and stored at -80°C. Initial screening for HTLV<br>
antibodies in serum and plasma samples was performed by using the Vironostika HTLV-1/2<br>
microelisa system (Organon-Teknika, Durham, N.C.) following the manufacturer's<br>
instructions. Reactive samples were then tested in a WB test (HTLV Blot 2.4, Genelabs<br>
Diagnostics, Singapore) that contains disrupted HTLV-1 virions, a gp21 recombinant<br>
protein (GD21) common to both HTLV-1 and HTLV-2, and two HTLV-type specific<br>
recombinant envelope (Env) peptides, MTA-1 and K55, which allow serological<br>
differentiation of HTLV-1 and HTLV-2, respectively. Samples with reactivity to the Gag<br>
(p24) and Env (GD21) proteins were considered seropositive. Seropositive samples with<br>
reactivity to MTA-1 or K55 were considered HTLV- l-like or HTLV-2-like, respectively.<br>
Samples with reactivity to either p24 or GD21 alone or in combination with other HTLV<br>
proteins (Fig. 1) were considered indeterminate.<br>
(3)	PCR and sequence analysis.<br>
213.	DNA was prepared from uncultured PBMCs and its integrity was confirmed<br>
by JJ-actin PCR as previously described. All DNA preparation and PCR assays were<br>
performed in a laboratory where only human samples are processed and tested following<br>
recommended precautions to prevent contamination. DNA samples were first screened with<br>
a generic PTLV tax PCR assay capable of detecting 222-bp sequences from each of the<br>
three major PTLV groups (Busch et al. 2000, van Dooren et al. 2004). Sequence analysis of<br>
this tax sequence provided broad genetic classification into each PTLV group. Phylogenetic<br>
resolution within the PTLV-1 and PTLV-3 groups was done using LTR sequences as<br>
described previously (van Dooren et al. 2004, Meertens et al. 2001). A portion of the 3'<br>
HTLV-1 LTR from selected samples (1259NG, 1127MO, 1842LE, and 2810YI) was amplified by nested PCR using external primers SVLTRext 5' AACCACCCATTTCCTCCCCATG 3' (SEQ ID NO: 19)(Meertens et al. 2001) and IMNDRl 5'GTCGTGAATGAAAGGGAAAGGGGT 3' (SEQ ID NO: 20) (Meertens et al. 2001) and the internal primers Enh280 5' TGACGACAACCCCTCACCTCAA 3' (SEQ ID NO: 21) (Meertens et al. 2001) and 1MNDR2 5' AGGGGTGGAACTTTCGATCTGTAA 3'(SEQ ID NO: 22) (Meertens et al. 2001). The tax (577-bp) and polymerase (pol) (709-bp) sequences of HTLV-3 and HTLV-4 were amplified by nested PCR using primers designed from conserved PTLV regions. The external and internal tax primers are PTLVTPG 5'T(C/T)ACCT(G/A)GGACCCCATCGATGGACG 3' (SEQ ID NO: 7) and PGTAXR1 5' GAIGA(T/C)TGI A(C/G)TAC(T/C)AAAGATGGCTG 3' (SEQ ID NO: 8) and PH2Rrev 5' CCTTATCCCTCGICTCCCCTC CTT 3' (SEQ ID NO: 9) and PGTAXR2 5' TTIGGG(T/C)AIGGICCGG AAATCAT 3'(SEQ ID NO: 10), respectively. The external and internal pol primers are PGPOLF1 5' C(T/G)TTAAACCIGA(A/G)CGCCT CCAGGC 3' (SEQ ID NO: 11) and PGPOLR1 GG(T/C)(A/G)TGIA<br>
(A/G)CCA(A/G)(A/G)CIAG(T/G)GG CCA 3' (SEQ ID NO: 12) and PGPOLF2 5' AC(T/C)TGGT(C/T)(C/T) (G/C)(G/C)A(A/G)GGCCCTGGAGG 3' (SEQ ED NO: 13) and PGPOLR2 5' G(A/G)(T/C)(A/G)GGIGTIC CTTTIGAGACCCA 3'(SEQ ID NO: 14), respectively. Ihosines (I) and wobble bases (N/N) were used to accommodate areas of heterogeneity (Table 5).<br>
TABLE 5: Sequences of primers used for amplifying partial tea, envelope (env), polymerase (pot) and LTR regions of primate T-cell lymphotropic viruses    <br>
(Table Removed)<br>
Additional diagnostic PCR with PTLV-specific primers was done on samples with negative results for the generic 222-bp tax fragments. Assays described previously were used for PTLV-1 env and STLV-3 LTR (van Dooren et al. 2004) and HTLV-2 env (Switzer et al.1995). For HTLV-4, a new nested PCR assay was developed based on the HTLV-4 tax sequence using the external primers 1863TF1 5' CTCCTTCTTTCAGTCCGTGCGGAG 3' (SEQ ID NO: 15) and 1863TR1 5' GGGGTAGTCAGGTTTGGCTGGTAT 3' (SEQ ID NO: 16) and the internal primers 1863TF2 5' CCTACCGCAACGGATGTCTTGAAA 3' (SEQ ID NO: 17) and 1863TR2 5' TATGGCGCC GGTGTGATGATAAAG 3' (SEQ ID NO: 18) and standard conditions to generate a 275-bp fragment. Percent nucleotide divergence was calculated using the Gap program in the Genetic Computer Group's Wisconsin package. Sequences were aligned using the Clustal W program, gaps were removed, and distance-based trees were generated by using the Kimura two-parameter model in conjunction with the NJ method in the MEGA program (version 2.1) as described elsewhere (van Dooren et al 2004). 1000 bootstrap replicates were used to test the reliability of the final topology of the trees.<br>
(4)	Primate taxonomic nomenclature.<br>
214.	Nomenclature used herein was as described. NHPs were coded using the first<br>
letter of the genus and the first two letters of the species names with their house names or<br>
codes within parentheses. Cmo=Cercqpithecus mono (Mona monkey), Cne=C. neglectus<br>
(De Brazza's guenon), Cmi=C. mitis (Sykes's monkey), Cni=C. nictitans (greater spot-<br>
nosed guenon), Cae=Chlorocebus species (African green monkey), Cpo=C. pogonias<br>
(crowned monkey), Cto=Cercocebus torquatus (red-capped mangabey), Cag=Cercocebus<br>
agilis (agile mangabey), Mog =Miopithecus ogouensis (talapoin monkey),<br>
Aia=Anenopitheciu/ nigrovridis (Allen's swamp monkey), Msp=MandrilIus sphinx<br>
(mandrill (mnd)), Paa-=Papio anubis (olive baboon (bab)), Pcy=P. cynocephalus (yellow<br>
baboon), Pha=P. hamadryas (sacred baboon), Ppu=P. ursinus (chacma baboon), Ppa=P.<br>
papio (Guinea baboon), Pba -Piliocolobus badius (red colobus monkey), Mto=Macaca<br>
tonkeana (Celebes macaque), Ptr= Pan troglodytes (chimpanzee), Ppn=Pan paniscus<br>
(bonobo), Ggo=Gorilla gorilla (western lowland gorilla).<br>
(5)	Nucleotide sequence accession numbers.<br>
215,	The GenBank accession numbers for the 28 new HTLV sequences include<br>
AY818406 and AY818433.<br>
2. Example 2: Ancient Origin and Molecular Features of the Novel Human T-lymphotropic virus type 3 Revealed by Complete Genome Analysis<br>
a) Results<br>
(1) Comparison of the HTLV-3(2026ND) proviral genome with prototypical PTLVs.<br>
216.    Using a combination of primers designed from small sequences obtained in each of the three major genes of PTLV and the LTR region the complete genome of HTLV-3(2026ND) was successfully generated as depicted in Fig. 4. Sequence analysis of the overlapping regions, followed by comparison with the genetic structure of other PTLVs, demonstrated that the complete proviral genome of HTLV-3(2026ND) is 8917-bp. Despite being genetically equidistant from HTLV-1 and HTLV-2, the genomic structure of HTLV-3(2026ND) was similar to that of other PTLVs and included the structural, enzymatic, and regulatory proteins all flanked by long terminal repeats (LTRs). Comparison of HTLV-3(2026ND) with prototypical PTLV genomes demonstrates that this new human virus is equidistant from the PTLV-1 (62% identity) and PTLV-2 (63% identity) groups across the genome. The results also confirm that HTLV-3 has the closet nucleotide and protein sequence identity to STLV-3 (87-92% identity) (Table 6).<br>
Table 6. Percent Nucleotide and Ammo Acid Identity of HTLV-3(2026ND) with other PTLV Prototypes      (Table Removed)   <br>
The most genetic divergence between the PTLV groups was seen in the LTR region (52-59%) while the highest inter-group identity was observed in the highly conserved regulatory genes, tax and rex (72-77%). Interestingly, within the PTLV-3 group, HTLV-3(2026ND), which was identified in a hunter from Cameroon, was unique but shared the most overall sequence identity to STLV-3(PPAF3) (92%) from a Senegalese baboon instead of STLV-3(CTO604) (88.4%) identified in red-capped mangabeys, also from Cameroon. This relationship is highlighted further by comparison of HTLV-3(2026ND) with all available full-length STLV-3 genomes in similarity plot analysis where the highest identity was seen in the highly conserved tax gene. As seen within other PTLV groups, there was no clear evidence of genetic recombination of HTLV-3(2026ND) with STLV-3 or PTLV-1 and PTLV-2 proviral sequences by using bootscanning analysis. HTLV-3(2026ND) was not compared to the recently reported second strain of HTLV-3 because only two short sequences were available at GenBank and hi these region this virus has been shown to be nearly identical to STLV-3(CTO604) (Callatini, S., et al. (2005) Retrovirology. 2:30).<br>
(2)	Organization of the LTR and pre-gag region. 217.	As with STLV-3, the HTLV-3(2026ND) LTR (697-bp) was smaller than<br>
that of HTLV-1 (756-bp) and HTLV-2 (764-bp), by having two and not three of the 21-bp<br>
transcription regulatory repeat sequences in the U3 region (Fig. 5a) (Meertens, L., and A.<br>
Gessain. (2003) J. Virol. 77:782-789; Meertens, L., et al. (2002) J. Virol 76:259-268; Van<br>
Brussel, M., et al. (1997) J. Virol. 7:5464-5472; Van Dooren, S., et al. (2004) J. Gen. Virol.<br>
85:507-519). Other regulatory motifs such as the polyadenylation signal, TATA box, and<br>
cap site were all conserved in the HTLV-3 (2026ND) LTR (Fig. 5a). By secondary structure<br>
analysis of the LTR RNA sequence, a stable stem loop structure from nucleotides 421- 464<br>
(Fig. 5b) was also observed similar to that shown to be essential for Rex-responsiveness<br>
control of viral expression in both HTLV-1 and HTLV-2.<br>
(3)	Analysis of the genomic structure of HTLV-<br>
3(2026ND).<br>
218.	Translation of predicted protein open reading frames (ORFs) across the viral<br>
genome identified all major Gag, Pol, Pro (protease), and Env proteins, as well as the<br>
regulatory proteins, Tax and Rex. Translation of the overlapping gag and pro andpro and<br>
pol ORFs occurs by one or more successive -1 ribosomal frameshifts that align the different<br>
ORFs. The conserved slippage nucleotide sequence 6(A)-8nt-6(G)-l lnt-6(C) is present in<br>
the Gag-Pro overlap, while a point mutation in the Pro-Pol overlap slippage sequence<br>
(GTTAAAC compared to TTTAAAC in HTLV-1 and HTLV-2) was observed in HTLV-3(2026ND) but the asparagine codon (AAC) crucial for the slippage mechanism was unaffected.<br>
219.	The structural and group specific precursor Gag protein consisted of 422<br>
amino acids (aa) that is predicted to be cleaved into the three core proteins p!9 (matrix), p24<br>
(capsid), and p!5 (nucleocapsid) similar to HTLV-1, HTLV-2, and STLV-3. Across PTLVs,<br>
Gag was one of the most conserved proteins with identities ranging from 81% and 83% for<br>
HTLV-1 and PTLV-2, to 95% for STLV-3 supporting the observed cross-reactivity seen<br>
with PTLV-3 antisera in Western blot assays using HTLV-1 antigens. Within Gag, the<br>
capsid protein showed greater than 90% identity to HTLV-1, while the matrix and<br>
nucleocapsid proteins were more divergent sharing less than 78% identity to PTLV-1 and<br>
PTLV-2 indicating their potential use in serologic assays for discriminating the three major<br>
PTLV groups.<br>
220.	The predicted size of the Env polyprotein is 491 aa, which is slightly shorter<br>
than that found in STLV-3s (313 aa versus 314 and 315 for STLV-3(PH969) and STLV-<br>
3(CTO-604) due to sequence variation at the carboxy terminus of the surface (SU) protein.<br>
.In contrast, the transmembrane (TM) protein (178 aa) was highly conserved across all<br>
PTLVs supporting further the use of the recombinant HTLV-1 GD21 protein spiked onto<br>
WB strips for the identification of divergent PTLVs. Despite the weak reactivity of anti-<br>
HTLV-3(2026ND) antibodies to the HTLV-1 type specific SU peptide (MTA-1) Wolfe, N.<br>
D., et al. (2005) Proc. Natl. Acad. Sci. U SA. 102:7994-7999 spiked onto WB strips, there<br>
was only 70.8% identity of MTA-1 to HTLV-3(2026ND), which is similar to the 68.8%<br>
identity of MTA-1 to HTLV-2, demonstrating no clear correlation of WB profile and<br>
predicted SU sequence.<br>
221.	The HTLV-1 and HTLV-2 Tax proteins (Taxi and Tax2, respectively)<br>
transactivate initiation of viral replication from the promoter in the 5* LTR and are thus<br>
essential for viral expression (Feuer, G., and P. L. Green. (2005) Oncogene. 24:5996-6004).<br>
Taxi and Tax2 have also been shown to be important for T-cell immortalization, while the<br>
HTLV-3 Tax (Tax3) has not yet been characterized (Feuer, G., and P. L. Green. (2005)<br>
Oncogene. 24:5996-6004). Hence, the Tax3 sequences were compared with those of<br>
prototypic HTLV-1, PTLV-2, and STLV-3s to determine if motifs associated with these<br>
functional characteristics are preserved. Alignment of predicted Tax3 sequences shows<br>
excellent conservation of the critical functional regions, including the nuclear localization<br>
signal (NLS), cAMP response element (CREB) binding protein (CBP)/P300 binding motifs, and nuclear export signal (NES) (Fig. 6). The C-terminal transcriptional activating domain (CR2), essential for CBP/p300 binding, was also very conserved except for a single I/V to F mutation at position five of the motif compared to HTLV-1 and PTLV-2, respectively (Fig. 6). However, this single amino acid change in the STLV-3 Tax has recently been shown hi transient transfection assays to have no deleterious effect on viral transactivation (Chevalier, S., et al. (2005) AIDS Res. Hum. Retrovir. 21:513 (Abs. P174)). Since the predicted CR2 domain is conserved in Tax3, similar transactivation activity can be seen with HTLV-3.<br>
222.	Interestingly, although these important functional motifs are highly<br>
conserved in PTLV, phenotypic differences of HTLV-1 and HTLV-2 Tax proteins have<br>
been observed leading to speculation that these differences account for the different<br>
pathologies associated with both HTLVs (Feuer, G., and P. L. Green. (2005) Oncogene.<br>
24:5996-6004). Recently, the C-terminus of Taxi, and not Tax2, has been shown to contain<br>
a conserved PDZ domain present in cellular proteins involved in signal transduction and<br>
induction of the IL-2-independent growth required for T-cell transformation (Rousset, R., et<br>
al. (1998) Oncogene. 6:643-654; Tsubata, C., et al. (2005) Retrovirol 2:46). The presence<br>
of a PDZ domain in PTLV-1 and its absence in PTLV-2 indicates a potential role of this<br>
motif in the phenotypic differences of the two viral groups. The consensus PDZ domain has<br>
been defined as S/TXV-COOH, where the first amino acid is serine or threonine, X is any<br>
amino acid, followed by valine and the carboxy terminus. Examination of the PTLV-3 Tax<br>
sequences showed that both HTLV-3 and STLV-3 have predicted PDZ domains with the<br>
consensus sequence S(P/S)V compared to T(E/D)V in PTLV-1 (Fig. 6).<br>
223.	Besides Tax and Rex, two additional ORFs coding for four proteins (p27£,<br>
pl2J, p30n, and p!3n where I and n denote ORFI and ORFU, respectively) have been<br>
identified in the pX region of HTLV-1 (Fig. 4) and are important in viral infectivity and<br>
replication, T-cell activation, and cellular gene expression (Bindhu, M, et al. (2004) Front.<br>
Biosc. 9:2556-2576). Analysis of the pX region of HTLV-3(2026ND) revealed a total of<br>
four putative ORFs (named I-IV, respectively) coding for 96,122, 72, and 118 aa in length.<br>
While both ORFIH (72 aa) and ORFIV (118 aa) shared identity to the ORFE of STLV-3 and<br>
HTLV-1 and STLV-2/HTLV-2, respectively, and each contained two PXXP motifs, only<br>
ORF in was leucine rich like that seen in the leucine zipper motifs of ORFI p12I (Bindhu,<br>
M., et al. (2004) Front, Biosc. 9:2556-2576). However, ORFHI did not share any sequence<br>
homology with p12I and both ORFI and ORFn shared only weak sequence identity to<br>
miscellaneous cellular proteins available at GenBank. Interestingly, 22 of 28 (79%) ammo acids in ORFIV (pos 64-91) were identical among the ORFIIs of all PTLVs indicating a conserved functionality of this motif. Analysis of alternatively spliced messenger RNA expression in viable cells or tissue culture, and/or in vitro characterization, will be required to investigate the functionality of these putative accessory proteins.<br>
224.	A novel protein termed the HTLV-1 basic leucine zipper ZIP (bZEP) factor<br>
(HBZ) was recently identified in translation of the complementary strand of the viral RNA<br>
genome between the env and tax/rex genes (Gaudray, G., et al. (2002) J Virol. 76:12813-<br>
12822). Although originally reported to be exclusive to PTLV-1 (Gaudray, G., et al. (2002)<br>
J Virol. 76:12813-12822), HBZ is conserved among PTLVs (Fig. 7), including HTLV-<br>
3(2026ND), demonstrating further the potential importance of this protein in viral<br>
replication and oncogenesis. The carboxy terminus of the HBZ ORF contains a 21 aa<br>
arginine rich region that is relatively conserved in PTLV and known cellular bZIP<br>
transcription factors, followed by a leucine zipper region possessing five or four conserved<br>
leucine heptads in HTLV-1 and all other PTLVs, respectively (Fig. 7). PTLV-1 has 5<br>
leucine heptads similar to that found in mammalian bZIP proteins, while PTLV-1 and<br>
PTLV-2 have four leucine heptads followed by leucine octet (Fig. 7). Of all PTLVs with full<br>
length genomes available at GenBank, only HTLV-2(MoT) did not have the full<br>
complement of leucine heptads but was limited to the initial three leucine motifs due to a<br>
one nucleotide deletion at position 6823 causing a frameshift in the predicted HBZ<br>
sequence.<br>
(4) Phylogenetic analysis.<br>
225.	The genetic relationship of HTLV-3(2026ND) to PTLV-3 was confirmed by<br>
using aligned full-length prototype sequences excluding the LTR region (Fig. 8a).<br>
Phylogenetic analysis inferred three major PTLV groups with very high bootstrap support<br>
(100%) with HTLV-1, HTLV-2 and HTLV-3 each clustering in separate clades (Fig. 8a).<br>
Within the PTLV-3 phylogroup, HTLV-3(2026ND) formed a separate lineage but clustered<br>
with high bootstrap support with STLV-3s from west central Africa (strains CTO604, CTO-<br>
NG409, and PPA-F3) indicating a possible primate origin for this human infection in this<br>
geographic region. The relationship of HTLV-3 to STLV-3 was supported further by<br>
phylogenetic inference of identical tree topologies using an alignment of each major gene<br>
region (Fig. 8b-8d). The phylogenetic stability seen across the PTLV genome also<br>
demonstrates further the absence of major recombination events occurring in PTLV despite<br>
evidence of dual infections in humans and primates (Courgnaud, V., S. Et al. (2004) J. Virol. 78 :4700-4709), compared to other retroviruses such as HTV which undergo frequent recombination.<br>
(5) Dating the origin of HTLV~3(2026ND) and other PTLVs.<br>
226.    The finding of HTLVs in three distinct clades indicates an ancient, independent evolution of these viruses. Hence, additional molecular analyses was undertaken in order estimate the divergence times of the PTLV lineages. Although others have reported finding a clock-like behavior of STLV-3 sequences (Meertens, L., and A. Gessain. (2003)/. Virol 77:782-789; Meertens, L., et al. (2002)7. Virol. 76:259-268; Meertens, L., et al. (2003) J. Gen. Virol. 84:2723-2727; 22)), these results were not confirmed and instead found that PTLVs evolved at different rates by using an alignment of full-length PTLV genomes sans LTR sequences. However, reliable retrovirus divergence times can be obtained by using nonparametric rate smoothing of the sequences to relax the stringency of a clock assumption followed by time calibration of the tree using a value of 40,000 - 60,000 YA for the origin of the Melanesian HTLV-1 (Sanderson, M. J. (2003) Bioinformatics. 19:301-2; Switzer, W. M., et al. (2005) Nature. 434:376-380; Van Dooren, S., et al. (2004) /. Gen. Virol. 85:507-519). By using these dates and methods, the mean evolutionary rate for PTLV was estimated to be 1.12x 10"6 (confidence interval 6.82 x 10"7 to 1.56 x 10"6) substitutions/site/ year, respectively, which is consistent with rates determined previously both with and without enforcing a molecular clock (Lemey, P., et al. (2005) Infect. Gen. Evol. 5:291-298; Meertens, L., and A. Gessain. (2003) J. Virol. 77:782-789; Meertens, L., et al. (2002) J. Virol 76:259-268; Meertens, L., et al. (2003) J. Gen. Virol. 84:2723-2727; Salemi, M., et al. (2000) Mol. Biol Evol. 17:374-386; Van Dooren, S., et al. (2004) J. Gen. Virol. 85:507-519). The mean evolutionary rate for HTLV-3(2026ND) is estimated to be 9.94 x 10'7 (confidence interval 6.04 x 10'7 to 1.38 x Iff6). The PTLV ancestor was estimated to have originated about 630,000 - 947,000 YA confirming an archaic evolution of the primate deltaretroviruses (Fig. 9) (Salemi, M., et al. (2000) Mol. Biol. Evol. 17:374-386). The separation of PTLV-1 and PTLV-2 occurred about 579,077 - 867,458 YA, while HTLV-2 and STLV-2 diverged around 191,621 -286,730 YA (Fig. 9). The origin of all PTLV-3s was estimated to be between 63,294 - 94,700 YA with the ancestor of HTLV-3(2026ND) occurring about 36,087 - 54,067 YA (Fig. 9) indicating an ancient origin of this virus in humans. Alternatively, HTLV-3 may represent a<br>
recent zoonoses from a primate infected with a very old, divergent STLV-3. However, if HTLV-3 is an old human infection, then it appeared during the same period as the ancestor of both HTLV-1 and HTLV-2 (40,000 - 60,000 and 28,800 - 43,392 YA, respectively) and may have also spread to become endemic in specific populations yet to be identified. b) Discussion<br>
227.	The complete nucleotide sequence and genomic characterization of the first<br>
HTLV-3 that is clearly distinct from all STLV-3s and is genetically equidistant to HTLV-1<br>
and HTLV-2 is described herein. HTLV-3(2026ND) is also unique from the second HTLV-<br>
3(Pyl43) reported recently in a Bakola pygmy from Cameroon since the latter strain is<br>
nearly identical to STLV-3 found in a red-capped mangabey, based on the limited sequence<br>
data available for this virus (Callatini, S., et al. (2005) Retrovirology. 2:30). Although<br>
HTLV-1 and HTLV-2 are pathogenic and have spread globally to become endemic in<br>
different human populations, little is known about the epidemiology of HTLV-3 infection.<br>
However, detailed, comparative sequence analyses of viral genomes can help provide<br>
important molecular clues to the origin, evolution, and public health importance of novel<br>
human infections.<br>
228.	Like other PTLVs, HTLV-3(2026ND) is genetically stable and its slow<br>
evolutionary rate, combined with estimates of known human migrations, can then be used to<br>
infer divergence times for HTLV. The finding that the predecessor of HTLV-3(2026ND)<br>
originated over 30 millenia ago, an age which is estimated that the ancestors of both HTLV-<br>
1 and -2 to have-appeared, combined with the wide geographic distribution of STLVs and<br>
the recent finding of another HTLV-3 in an African pygmy (Callatini, S., et al. (2005)<br>
Retrovirology. 2:30; Gessain, A., andR. Mahieux. (2000) Bull Soc. Pathol. Exot. 93:163-<br>
171; Meertens, L., and A. Gessain. (2003) J. Virol 77:782-789; Meertens, L., et al. (2002)<br>
J. Virol. 76:259-268; Meertens, L., et al. (2003) J. Gen. Virol. 84:2723-2727; Takemura,<br>
T., et al. (2002) J. Virol. 76:1642-1648; Van Dooren, S., et al. (2004) J. Gen. Virol. 85:507-<br>
519), collectively indicate that HTLV-3 infection be more frequent than previously<br>
understood. In addition, the archaic age of the ancestral HTLVs and the recent finding of<br>
STLV-like infections in African hunters collectively imply that cross-species transmission<br>
of STLVs to humans is both an ancient and contemporary phenomenon coupled to behavior<br>
that exposes humans to nonhuman primates. The ancient origin of HTLV contrasts with that<br>
reported for HTV, which is believed to have only crossed over into humans from SIV-<br>
infected NHPs within the last century, and indicates a long period of viral evolution and<br>
adaptation in humans possibly resulting in the observed lower pathogenicity for HTLV compared to HTV (Hahn, B. H., et al. (2000) Science 287:607-614; Sharp, P. M., et al. (2000) Biochem Soc Trans. 28:275-282).<br>
229.	Screening for HTLV-3 can be facilitated by the application of diagnostic<br>
serologic and molecular assays based on the sequences reported here. For example, the data<br>
show that the Gag matrix and nucleocapsid regions and the envelope surface protein are<br>
relatively conserved within PTLV-3 but are divergent from PTLV-1 and PTLV-2 and can<br>
thus be used to differentiate the three PTLV groups with serological methods.<br>
230.	At the molecular level, examination of the genomic structure showed that the<br>
enzymatic, regulatory, and structural proteins are well preserved in HTLV-3(2026ND),<br>
including conserved functional motifs in Tax important for viral expression and T-cell<br>
proliferation. The finding of a PDZ domain in the Tax protein of HTLV-3 (2026ND), like<br>
that seen in HTLV-1 but not HTLV-2 (Feuer, G., and P. L. Green. (2005) Oncogene.<br>
24:5996-6004), which has been shown to be important in cellular signal transduction and T-<br>
cell transformation (Rousset, R., et al. (1998) Oncogene. 6:643-654; Tsubata, C., et al.<br>
(2005) Retrovirol. 2:46), indicates that the HTLV-3 Tax is more phenotypically similar to<br>
HTLV-1 than HTLV-2. The high amino acid identity of the PTLV-3 Tax proteins combined<br>
with the ability of STLV-3 to transform human cells in vitro indicates that the HTLV-3 Tax<br>
functions similarly (Goubau, P., et al. (1994) Proc. Natl. Acad. Sci. USA 91:2848-2852).<br>
231.	In contrast to the tax gene, the HTLV-3 (2026ND) LTR has only two of the<br>
three conserved promoters identified in HTLV-1 and HTLV-2 that are responsible for basal<br>
viral transcription levels and like STLV-3 is missing the TATA-distal 21-bp repeat element<br>
(Meertens, L., and A. Gessain. (2003) J. Virol. 77:782-789; Meertens, L., et al. (2002) J.<br>
Virol. 76:259-268; Meertens, L., et al. (2003) J. Gen. Virol. 84:2723-2727; Van Brussel,<br>
M., et al. (1997) J. Virol. 7:5464-5472; Van Dooren, S., et al. (2004) J. Gen. Virol 85:507-<br>
519). All of the remaining functional elements in the LTR were conserved, including the<br>
stem loop structure necessary for Rex responsive control of viral expression in HTLV-1 and<br>
-2.<br>
232.	Recently, a novel HBZ protein was identified in translation of the<br>
complementary strand of the viral RNA genome between the env and tax/rex genes<br>
(Gaudray, G., et al. (2002) J Virol. 76:12813-12822). Protein translation on the minus-<br>
strand RNA is a unique feature of HTLV-1 not previously seen in retroviruses. HBZ was<br>
shown to be involved hi the negative regulation of viral replication (Gaudray, G., et al.<br>
(2002) J Viroi. 70:l2813-12822).The more recem naming 01 ntsz,mMRNA expression in ATL patients indicates a role of HBZ mRNA in the survival of leukemic cells in vivo and in HTLV-1-associated oncogenesis (Satou, Y., et al. (2006) Proc. Natl. Acad. Sci. USA. 103:720-725). Although originally reported to be exclusive to PTLV-1 (Gaudray, G., et al. (2002) J Virol. 76:12813-12822), HBZ is conserved among PTLVs, including HTLV-3(2026ND), demonstrating further the potential importance of this protein in viral replication and oncogenesis. Of all PTLVs with full length genomes available at GenBank, only HTLV-2(MoT) did not have the foil complement of leucine heptads in the leucine zipper due to a frameshift mutation in the predicted HBZ sequence.<br>
233.	In summary, disclosed herein, the novel and unique HTLV-3(2026ND) is<br>
genetically stable and has an ancient origin. HTLV-3(2026ND) genomic structure is<br>
relatively conserved and contains many of the functional motifs important for the viral<br>
expression and pathology associated with HTLVs.<br>
c) Materials and Methods<br>
(1) DNA Preparation and PCR-based Genome Walking.<br>
234.	DNA was prepared from uncultured PBMCs available from person 2026ND<br>
identified in the original PTLV surveillance study in Cameroon reported in detail elsewhere<br>
(Wolfe, N. D., et al. (2005) Proc. Natl. Acad. Sci. USA. 102:7994-7999). DNA integrity<br>
was confirmed by 6-actin polymerase chain reaction (PCR) as previously described (Wolfe,<br>
N. D., et al. (2005) Proc. Natl. Acad. Sci. USA. 102:7994-7999). All DNA preparation and<br>
PCR assays were performed in a laboratory where only human specimens are processed and<br>
tested according to recommended precautions to prevent contamination. To obtain the full-<br>
length genomic sequence of HTLV-3 small regions of each major coding region were PCR-<br>
amplified by using nested PCR and degenerate PTLV primers. The tax (577-bp) and<br>
polymerase (pot) (709-bp) sequences were amplified by using primers and conditions<br>
provided elsewhere (Wolfe, N. D., et al. (2005) Proc. Natl. Acad. Sci. US,4.102:7994-<br>
7999). Envelope (env) (371-bp) sequences were amplified by using standard PCR<br>
conditions with a 45°C annealing temperature and the external and internal primers<br>
PGENVF1 5' TGGATCCCGTGG(A/C)GI(C/T)TCCTIAA 3' (SEQ ID NO: 27) and<br>
PGENVR1 5' GT(A/G)TAIG(C/G)(A/G)(C/G)AIGTCCAIG(A/C)(T/C)TGG 3' (SEQ ID<br>
NO: 28) and PFENVF2 5' AIAGACC(T/A)(C/T)CAAC(A/T)CCATGGGTAA 3' (SEQ ID<br>
NO: 29) and PGENVR2 5' G(A/C)(T/C)TGGCAICCIA(A/G)GTAIGGGCA 3' (SEQ ID<br>
NO: 30), respectively. A 398-bp fragment of the long terminal repeat (LTR) was obtained<br>
by using conserved STLV-3 primers as previously reported (Wolfe, N. D., et al. (2005) Proc.Natl.Acad. Sci. USA. 102:7994-7999).<br>
235.	HTLV-3(2026ND)-specific primers were then designed from sequences<br>
obtained in each of the four viral regions described above and were used in nested, long-<br>
template PCRs to fill in the gaps in the genome as depicted in Fig. 4 by using an expand<br>
high fidelity kit containing both Taq and Tgo DNA polymerases (Roche). The external and<br>
internal primer sequences for the LTR-pol and pol-env fragments are 2026LF1 5'<br>
GGTAAGATCCCACTGGGTCGAGC 3'(SEQ ID NO: 69) and 2026PR1 5'<br>
GAAGCCAGGTCTCGGGTGACG 3' (SEQ ID NO: 70) and 2026LF2 5'<br>
CGCTCCCCTGGAGCTCTCTCG 3'(SEQ ED NO: 71) and 2026PR2 5'<br>
GCCACTTCCCATTGGGCTTTTTGACGG 3' (SEQ ID NO: 72) and 2026PF3 5'<br>
GCTCTCACCGATAAAGTAACAAACG 3' (SEQ ID NO: 73) and 2026ER1 5'<br>
GGTAGGAAGAGGCTCCTATGAACAG 3' (SEQ ID NO: 74) and 2026PF2 5'<br>
CAGGACTGCATAACATACGAGACCCTCC 3' (SEQ ID NO: 75) and 2026ER3 5'<br>
CCTATGAACAGGGTGCATCGACTGGG 3' (SEQ ID NO: 76), respectively. The<br>
external and internal primer sequences used to obtain about 3 kb of the 3' end of the<br>
genome (env-tax-LTR) are 2026EF1 5' CCTAAGCCCCCCATGTCCAGAC 3' (SEQ ID<br>
NO: 77) and 2026LR1 5' CGAGAGAGCTCCAGGGGAGCG 3' (SEQ ID NO: 78) and<br>
2026EF3 5' CCTACTCCCTGTATGTATTCCCCCATTGG 3' (SEQ ID NO: 79) and<br>
2026LR2 5' GCTCGACCCAGTGGGATCTTACCGAGTGG 3' (SEQ ID NO: 80),<br>
respectively.<br>
236.	PCR products were revealed on 1.5% agarose gels stained with ethidimn<br>
bromide, purified with a Qiaquick PCR purification kit (Qiagen) and sequenced in both<br>
directions with a BigDye terminator cycle kit and automated sequencers (Applied<br>
Biosystems). Selected PCR products were also cloned into the pCR4-TOPO vector using<br>
the TOPO TA Cloning kit (Invitrogen) and recombinant plasmid DNA was prepared using<br>
the Qiagen plasmid purification kit prior to automated sequencing.<br>
(2) Sequence and phylogenetic analysis.<br>
237.	Percent nucleotide divergence was calculated by using the GAP program in<br>
the Genetic Computer Group's (GCG) Wisconsin package (Thompson, J. D., et al. (1994)<br>
Nucleic Acids Res. 22:4673-4680). LTR RNA secondary structure was determined using the<br>
program RNAstructure v4.2 (Mathews, D. et al. (1999) J. Mol. Biol. 288:911-940).<br>
Sequences were aligned by using the Clustal W program (Womble, D. D. (2000) Methods<br>
Mol. Biol. 132:3-22), gaps were removed, and distance-based trees were generated by using the Kimura two-parameter model in conjunction with the neighbor-joining (NJ) method in the MEGA program (version 2.1) and maximum likelihood (ML) analysis in the PAUP* program as described in detail elsewhere (Switzer, W. M., et al. (2005) Nature. 434:376-380; Wolfe, N. D., et al. (2005) Proc. Natl. Acad, Sci. USA. 102:7994-7999). The reliability of the final topology of the trees was tested with 1,000 bootstrap replicates. Comparison of full-length PTLV genomes available at GenBank was done using HTLV-3(2026ND) as the query sequence and the F84 (ML) model and a transition/transversion ratio of 2.0 implemented in the program SimPlot (Lole, K. S., et al. (1999) J. Virol. 73:152-160).<br>
238.	For dating of HTLV-3(2026ND), full-length genomes from prototypical<br>
PTLVs available at GenBank were aligned with HTLV-3(2026ND) by using Clustal W,<br>
gaps were removed, and minor adjustments in the alignment were made manually. LTR<br>
sequences were excluded from the analysis since this region does not align accurately in<br>
PTLVs. The best fitting evolutionary model for the aligned sequences was determined with<br>
Modeltest v3.6 (Posada, D., and K. A. Crandall. (1998) Bioinformatics. 14:817-818). The<br>
general time-reversible model, allowing six different substitution rate categories, with<br>
gamma-distributed rate heterogeneity (1.9724) and an estimated proportion of invariable<br>
sites (0.3687), was determined to be the best fit to the data. Little substitution saturation was<br>
observed in the 7213-bp alignment (P 
and was therefore satisfactory for use in phylogenetic analyses. Likewise, using the best-<br>
fitting evolutionary model defined above, good phylogenetic signal in the alignment was<br>
also found with likelihood mapping analysis using the program Tree-Puzzle v5.2.<br>
239.	The molecular clock hypothesis, or constant rate of evolution, was tested by<br>
using the likelihood ratio test with the likelihoods for the ML and clock-like ML trees<br>
obtained in PAUP*. The clock was tested with the best-fitting evolutionary model estimated<br>
in Modeltest, and ML trees were constructed in PAUP* starting from the NJ tree that is<br>
iterative!/ optimized using two consecutive heuristic searches with nearest neighbor<br>
interchange followed by a final heuristic search with the tree-bisection-reconnection<br>
algorithm. To adjust for rate heterogeneity among different PTLV taxa, clock-like ML trees<br>
were then transformed into ultrametric trees using the nonparametric rate smoothing<br>
(NPRS) algorithm in the program TreeEdit (vl.OalO carbon) (Sanderson, M. J. (2003)<br>
Bioinformatics. 19:301-2). The branches of the NPRS tree were then scaled by using a<br>
divergence time of 40,000 - 60,000 years ago (ya) for the Melanesian HTLV-lmel lineage based on genetic and archaelogical evidence of when the ancestors of indigenous Melanesians and Australians migrated from Southeast Asia (Lemey, P., et al. (2005) Infect. Gen. Evol. 5:291-298; Salemi, M., et al. (2000) Mol. Biol. Evol 17:374-386; Salemi, M., et al. (1999) AIDS Rev. 1:131-139). Variance in age estimates (branch lengths) was determined in PAUP* with 100 bootstrap repetitions by enforcing topological constraints and using a heuristic search without branch swapping on the clock-like ML tree. Branch lengths in all 100 trees were calibrated as before and average divergence times and confidence intervals (a = 0.05) were calculated in Excel. The evolutionary rate was estimated based on a known divergence time point of 40,000 - 60,000 ya and on the branch length of the ML clock-like tree according to the formula: evolutionary rate (r)= branch length (bl)/divergence time (t) (VanDooren, S., et al. (2004) J. Gen. Virol. 85:507-519).<br>
(3) Nucleotide sequence accession number.<br>
240.    The HTLV-3(2026ND) proviral sequence has the GenBank accession number DQ093792.<br>
G. References<br>
Araujo, A. &amp; Hall, W.W. Human T-lymphotropic virus type n and neurological disease. Ann. Neurol 56,10-19 (2004).<br>
Barnhart, M. K., L. M. Connor, S. J. Marriott. 1997. Function of the human T-cell leukemia virus type 1 21-base-pair repeats in basal transcription. J. Virol. 71:337-344.<br>
Bindhu, M., A. Nair, and M. D. Lairmore. 2004. Role of accessory proteins of HTLV-1 in viral replication, T cell activation, and cellular gene expression. Front. Biosc. 9:2556-2576.<br>
Bowen-Jones, E. &amp; Pendry, S. The threat to primates and other mammals from the bushmeat trade in Africa, and how this threat could be diminished. Oryx 33,233-246 (1999).<br>
Busch, M.P., Switzer, W.M., Murphy E., L. &amp; Heneine, W. and the Retrovirus Epidemiology donor study (REDS). Absence of evidence of infections with divergent simian T-lymphotropic viruses in US blood donors with seroindeterminate human T-lymhotropic results. Transfusion 40,443-449 (2000).<br>
Callatini, S., S. A. Chevalier, R. Duprez, S. Bassot, A. Froment, R. Mahieux, and A.Gessain. 2005. Discovery of a new human T-cell lymphotropic virus (HTLV-3) in Central Africa. Retrovirology. 2:30.<br>
Chevalier, S., L. Meertens, C. Pise-Masison, J. Brady, A. Gesain, and R. Mahieux. 2005. Molecular characterization of the Tax protein from the highly divergent simian T-cell lymphotropic virus type 3 strain. AIDS Res. Hum. Retrovir. 21:513 (Abs. P174).<br>
Courgnaud, V., Van Dooren, S., Liegeois, R, Pourrut, X., Abela, B., Loul, S., Mpoudi-Ngole, E., Vandamme, A., Delaporte, E. &amp; Peeters, M. Simian T-cell leukemia virus (STLV) infection in wild primate populations in Cameroon: evidence for dual STLV type 1 and type 3 infection in agile mangabeys (Cercocebus agilis). J. Virol 78,4700-4709 (2004).<br>
Digilio, L., A. Giri, N. Cho, J. Slattery, P. Markham, G. Franchini.1997. The simian T-lymphotropic/leukemia virus from Pan paniscus belongs to the type 2 family and infects Asian macaques. J Virol. 71:3684-3692.<br>
Feuer, G., and P. L. Green. 2005. Comparative biology of human T-cell lymphotropic virus type 1 (HTLV-1) and HTLV-2. Oncogene. 24:5996-6004.<br>
Gaudray, G., F. Gachon, J. Basbous, M. Biard-Piechaczyk, C. Devaux, and J. M. Mesnard. 2002. The complementary strand of the human T-cell leukemia virus type 1 RNA genome encodes a bZEP transcription factor that down-regulates viral transcription. J Virol. 76:12813-12822.<br>
Gessain, A. &amp; Mahieux, R. Epidemiology, origin and genetic diversity of HTLV-1 retrovirus and STLV-1 simian affiliated retrovnus. Bull. Soc. Pathol. Exot. 93,163-171 (2000).<br>
Goubau, P., Van Brussel, M., Vandamme, A.M., Liu, H.F. &amp; Desmyter, J. A primate T-lymphotropic virus, PTLV-L, different from human T-lymphotropic viruses types I and n, in a wild-caught baboon (Papio hamadryas). Proc. Natl. Acad. Sci. USA 91,2848-2852 (1994).<br>
Groves, C. Primate Taxonomy (Smithsonian Institution Press, Washington, D.C., 2001).<br>
Hahn, B. H., G. M. Shaw, K. M. De Cock, and P. M. Sharp. 2000. AIDS as a zoonosis: scientific and public health implications. Science 287:607-614.<br>
Heneine, W., Switzer, W.M., Sandstrom, P., Brown, J., Vedapuri, S., Schable, C.A., Khan, A.S., Lerche, N.W., Schweizer, M., Neumann-Haefelin, D., Chapman, L.E. &amp; Folks, T.M. Identification of a human population infected with simian foamy viruses. Nat. Med. 4,403-407 (1998).<br>
Khabbaz, R.F., Heneine, W., George, J.R., Parekh, B., Rowe, T., Woods, T., Switzer, W.M., McClure, H.M., Murphey-Corb, M., &amp; Folks, T.M. Brief report: infection of a laboratory worker with simian immunodeficiency virus. N. Engl. J. Med. 330,172-177 (1994).<br>
Leendertz, F.H., Junglen, S., Boesch, C., Formenty, P., Couacy-Hymann, E., Courgnaud, V., Pauli, G. &amp; Ellerbrok, H. High variety of different simian T-cell leukemia virus type 1 strains in chimpanzees (Pan troglodytes vents) of Tai National Park, Cote d'lvoire. J. Virol. 78,4352-4356 (2004).<br>
Lemey, P., O. G. Pybus, S. Van Dooren, and A. M.Vandamme. 2005. A Bayesian statistical analysis of human T-cell lymphotropic virus evolutionary rates. Infect. Gen. Evol. 5:291-298.<br>
Lerche, N.W., Switzer, W.M., Yee, J.L., Shanmugam, V., Rosenthal, A.N., Chapman, L.E.,<br>
Folks, T.M. &amp; Heneine W. Evidence of infection with simian type D retrovirus in persons occupationally exposed to nonhuman primates../ Virol. 75,1783-1789(2001).<br>
Lole, K. S., R. C. Bellinger R. S. Paranjape, D. Gadkari, S. S. Kulkarni, N. G. Novak, R. Ingersoll, H. W. Sheppard, and S. C. Ray. 1999. Full-length human immunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination. J. Virol. 73:152-160.<br>
Mahieux, R., Chappey, C., Georges-Courbot, M.C., Dubreuil, G., Mauclere, P., Georges, A. &amp; Gessain, A. Simian T-cell lymphotropic virus type 1 from Mandrillus sphinx as a simian counterpart of human T-cell lymphotropic virus type 1 sybtype D. J. Virol. 72,10316-10322 (1998).<br>
Mahieux, R., Ibrahim, F., Mauclere, P., Herve, V., Michel, P., Tekaia, F., Chappey, C., Garin, B., Van Der Ryst, E., Guillemain, B., Ledru, E., Delaporte, E., de The, G. &amp; Gessain, A. Molecular epidemiology of 58 new African human T-cell leukemia virus type 1 (HTLV-1) strains: identification of a new and distinct molecular subtypes in central Africa and in pygmies. J. Virol. 71,1317-1333 (1997).<br>
Mahieux, R., P. Horal, P. Mauclere, O. Mercereau-Puijalon, M. Guillotte, L. Meertens, E. Murphy, and A. Gessain. 2000. Human T-cell lymphotropic virus type 1 gag indeterminate western blot patterns in Central Africa: relationship to Plasmodium falciparum infection. /. Clin. Microbiol. 38 :4049-4057.<br>
Mathews, D. H., J. Sabina, M. Zuker, and D. H. Turner. 1999. Expanded sequence dependence of thennodynamic parameters improves prediction of RNA recondary structure. J.Mol.Biol. 288:911-940.<br>
Meertens, L., and A. Gessain. 2003. Divergent simian T-cell lymphotropic virus type 3 (STLV-3) in wild-caught Papio hamadryas papio from Senegal: widespread distribution of STLV-3 in Africa. J. Virol. 77:782-789.<br>
Meertens, L., Mahieux, R., Mauclere, P., Lewis, J. &amp; Gessain, A. Complete sequence of a novel highly divergent simian T-cell lymphotropic virus from wild-caught red-capped mangabeys (Cercocebus torquatus) from Cameroon: a new primate T-lymphotropic virus type 3 subtype. J. Virol. 76, 259-268 (2002).<br>
Meertens, L., Rigoulet, J., Mauclere, P., Van Beveren, M., Chen, G.M., Diop, O., Dubreuil, G., Georges-Goubot, M.C., Berthier, J.L., Lewis, J. &amp; Gessain, A. Molecular and Phylogenetic Analyses of 16 Novel Simian T Cell Leukemia Virus Type 1 from Africa: Close Relationship of STLV-1 finmAHenopithecus nigroviridis to HTLV-1 Subtype B Strains. Virology 287,275-285 (2001).<br>
Meertens, L., V. Shanmugam, A. Gessain, B. E. Beer, Z. Tooze, W. Heneine, and W. M. Switzer. 2003. A novel, divergent simian T-cell lymphotropic virus type 3 in a wild-caught red-capped mangabey (Cercocebus torquatus torquatus) from Nigeria. J. Gen. Virol. 84:2723-2727.<br>
Milner-Gulland, E. J., Bennett, E.L. &amp; the SCB 2002 Annual Meeting Wild Meat Group Wild meat: the bigger picture. TREE 18,351-357 (2003).<br>
Nerrieuet E, Meertens L, Kfutwah A, Foupouapouognigni Y, Ayouba A, Gessain A. Simian T cell leukaemia virus type I subtype B in a wild-caught gorilla (Gorilla gorilla gorilla) and chimpanzee (Pan troglodytes vellerosus) from Cameroon. J Gen Virol. 85: 25-9 (2004).<br>
Posada, D., andK. A. Crandall. 1998. MODELTEST: testing the model ofDNA substitution. Bioinformatics. 14:817-818.<br>
Rousset, R., S. Fabre, C. Desbios, F. Bantignies, and P. Jalinot. 1998. The C-terminus of the HTLV-1 Tax oncoprotein mediates interaction with the PDZ domain of cellular proteins. Oncogene. 6:643-654.<br>
Salemi, M., Desmyter, J. &amp; Vandamme, A.M. Tempo and mode of human and simian T-lyphotropic virus (HTLV/STLV) evolution revealed by analyses of full-genome sequences. Mol Biol. Evol. 17,374-386 (2000).<br>
Salemi, M., Van Dooren, S. &amp; Vandamme, A.M. Origin and evolution of human and simian T-cell lymphotropic viruses. AIDS Rev. 1,131-139 (1999).<br>
Sanderson, Mj J. 2003. r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock. Bioinformatics. 19:301-2.<br>
Satou, Y., J.-I. Yasunaga, M. Yoshida, and M. Matsuoka. 2006 HTLV-1 basic leucine zipper factor gene mRNA supports proliferation of adult T cell leukemia cells. Proc. Nail. Acad. Sci. USA. 103:720-725..<br>
Sharp, P. M., E. Bailes, F. Gao, B. E. Beer, V. M. Hirsch, B. H. Hahn. 2000. Origins and evolution of AIDS viruses: estimating the time-scale. Biochem Soc Trans. 28:275-282.<br>
Slattery, J.P., Franchini, G. &amp; Gessain, A. Genomic evolution, patterns of global dissemination, and interspecies transmission of human and simian T-cell leukemia/lymphotropic viruses. Genome Res. 9, 525-549 (1999).<br>
Switzer, W. M., M. Salemi, V. Shanmugam, F. Gao, M. E. Cong, C. Kuiken, V. Bhullar, B. E. Beer, D. Vallet, A. Gautier-Hion, Z. Tooze, F. Villinger, E. C. Holmes, and W. Heneine. 2005. Ancient co-speciation of simian foamy viruses and primates. Nature. 434:376-380.<br>
Switzer, W.M., Pieniazek, D., Swanson, P., Samdal, H.H., Soriano, V., Khabbaz, R.F., Kaplan, I.E., Lai, R.B. &amp; Heneine, W. Phylogenetic relationship and geographic distribution of multiple human T-cell lymphotropic virus type n subtypes. J. Virol. 69,621-632 (1995).<br>
Takemura, T., M. Yamashita, M. K. Shimada, S. Ohkura, T. Shotake, M. Ikeda, T. Miura and M. Hayami. 2002. High prevalence of simian T-lymphotropic virus type L in wild ethiopian baboons. J. Virol. 76:1642-1648.<br>
Thompson, J. D., D. G. Higgins, and T. J. Gibson. 1994. CLUSTAL W: improving the sensitivity of progtessive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22:4673-4680.<br>
Tsubata, C., M. Higuchi, M. Takahashi, M. Oie, Y. Tanaka, F. Geyjo, and M. Fujii. 2005. PDZ domain-binding motif of human T-cell leukemia virus type 1 Tax oncoprotein is essential for the interleukin 2 independent growth induction of a T-cell line. Retrovirol.<br>
2:46.<br>
Van Brussel, M., M. Salemi, H. F. Liu, J. Gabriels, P. Goubau, J. Desmyter, and A. M. Vandamme. 1998. The simian T-lymphotropic virus STLV-PP1664 from Pan paniscus is distinctly related to HTLV-2 but differs in genomic organization. Virology. 243:366-379.<br>
Van Brussel, M., P. Goubau, R. Rousseau, J. Desmyter, and A. M. Vandamme. 1997. Complete nucleotide sequence of the new simian T-lymphotropic virus, STLV-PH969 from a Hamadryas baboon, and unusual features of its long terminal repeat. J. Virol. 7:5464-5472.<br>
Van Dooren S, Salemi M, Pourrut X, Peeters M, Delaporte E, Van Ranst M, Vandamme AM. Evidence for a second simian T-cell lymphotropic virus type 3 in Cercopithecus nictitans from Cameroon. J Virol. 2001 Dec;75(23): 11939-41.<br>
Van Dooren, S.3 Shanmugam, V., Bhullar, V., Parekh, B., Vandamme, A.M., Heneine, W. &amp; Switzer, W.M. Identification in gelada baboons (Theropithecus gelada) of a distinct simian T-cell lymphotropic virus type 3 with a broad range of Western blot reactivity. J. Gen. Virol. 85, 507-519 (2004).<br>
van Regenmortel, M.H.V., Fauquet, C.M., Bishop, D.H.L., Carstens, E.B., Estes, M.K., Lemon, S.M., Maniloff, J., Mayo, M.A., McGeoch, D.J., Pringle, C.R. &amp; Wickner. R.B. Seventh Report of the International Committee on Taxonomy of Viruses. (Academic Press, San Diego, Wien, New York, 2000). (online version http://www.virustaxonomyonUne.com/)<br>
Vandamme, A.M., Van Laethem, K., Liu, H.F., Van Brussel, M., Delaporte, E., de Castro Costa, C.M., Fleischer, C., Taylor, G., Bertazzoni, U., Desmyter, J. &amp; Goubau, P. Use of a generic polymerase chain reaction assay detecting human T-lymphotropic virus (HTLV) types I, n and divergent simian strains in the evaluation of individuals with indeterminate HTLV serology. J. Med. Virol. 52, 1-7 (1997).<br>
Wolfe, N. D., W. Heneine, J. K. Carr, A. D. Garcia, V. Shanmugam, U. Tamoufe, J. N. Torimiro, A. T. Prosser, M. Lebreton, E. Mpoudi-Ngole, F. E. McCutchan, D. L. Birx, T. M. Folks, D. S. Burke, and W. M. Switzer. 2005. Emergence of unique primate T-lymphotropic viruses among central African bushmeat hunters. Proc. Natl. Acad. Sci. US A 102:7994-7999.<br>
Wolfe, N.D., Prosser, A.T., Carr, J.K., Tamoufe, U., Mpoudi-Ngole, E., Torimiro, J.N., LeBreton, M., McCutchan, F.E., Birx, D.L. &amp; Burke D.S. 2004. Exposure to nonhuman primates in rural Cameroon. Emerg. Infect. Dis. 10, 2094 (2004).<br>
Wolfe, N.D., Switzer, W.M., Carr, J.K., Bhullar, V.B., Shanmugam, V., Tamoufe, U., Prosser, A.T., Torimiro, J.N., Wright, A., Mpoudi-Ngole, E., McCutchan, F.E., Birx, D.L., Folks, T.M., Burke, D.S. &amp; Heneine, W. Naturally acquked simian retrovirus infections in central African hunters. Lancet 363, 932-937 (2004b).<br>
Wolfe, N.D., Switzer, W.M., Folks, T.M., Burke, D.S. &amp; Heneine, W. Simian retroviral infections in human beings. Lancet 364,139-140 (2004c).<br>
Womble, D. D. 2000. GCG: The Wisconsin Package of sequence analysis programs.<br>
Methods Mol. Biol. 132:3-22.<br>
Yamashita, M.5 Ido, E., Miura, T.,Hayami, M. Molecular Epidemiology of HTLV-I in the world. JAcquirImmuneDeficSyndrHum Retrovirol. 13: S124-31 (1996).   H. Sequences<br>
SEQ ID NO: 1 (HTLV-3pol) (pos 2407-5076) SEQ ID NO: 2 (HTLV-4pol) (pos 3-2549) SEQ ID NO: 3 (HTLV-3 env) (pos 5069-6544) SEQ ID NO: 4 (HTLV-4 env) (pos 2542-3999) SEQ ID NO: 5 (HTLV-3 tax) SEQ ID NO: 6 (HTLV-4 tax) SEQ ID NO: 7 (PTLVTPG) SEQ ID NO: 8 (PGTAXR1) SEQ ID NO: 9 (PH2Rrev) SEQ ID NO: 10 (PGTAXR2) SEQ ID NO: 11 (PGPOLF1) SEQ ID NO: 12 (PGPOLR1) SEQ ID NO: 13 (PGPOLF2) SEQ ID NO: 14 (PGPOLR2) SEQ ID NO: 15 (1863TF1) SEQ ID NO: 16 (1863TR1) SEQ ID NO: 17 (1863TF2) SEQ ID NO: 18 (1863TR2) SEQ ID NO: 19 (SVLTRext) SEQ ID NO: 20 (1MNDR1) SEQ ID NO: 21 (Eoh280) SEQ ID NO: 22 (1MNDR2) SEQ ID NO: 23 (PH1F) SEQ ID NO:24(PH2R) SEQ ID NO:25(PH2F) SEQ ID NO: 26 (PH2R) SEQ ID NO: 27 (PGENVF1) SEQ ID NO: 28 (PGENVR1) SEQ ID NO: 29 (PGENVF2) SEQ ID NO: 30 (PGENVR2) SEQ ID NO: 31 (GPLTRF1) SEQ ID NO: 32 (GPLTRR1)<br>
SEQ D3 NO: 33 (GPLTRF2)<br>
SEQ ID NO: 34 (GPLTRR2)<br>
SEQ ID NO: 35 (HTLV-3 gag) (pos 756-2023)<br>
SEQ ID NO: 36 (HTLV-3 Complete genome: 2026ND.seq (8917 bp)<br>
SEQ ID NO: 37 (HTLV-3 env amino acid)<br>
SEQ ID NO: 38 (HTLV-3 env surface antigen (SU)=aa 1-315)<br>
SEQ ID NO: 39 (HTLV-3 env transmembrane=aa 316-491)<br>
SEQ ID NO: 40 (HTLV-3 gag amino acid)<br>
SEQ ID NO: 41 (HTLV-3 gag p15=aa 337-422)<br>
SEQ ID NO: 42 (HTLV-3 gag p19=aa 1-123)<br>
SEQ ID NO: 43 (HTLV-3 gag p24=aa 124-336)<br>
SEQ ID NO: 44 (HTLV-3 pal) amino acid<br>
SEQ ED NO: 45 (HTLV-3 LTR) (pos 1-697 &amp; 8221-8917)<br>
SEQ ID NO: 46 (HTLV-3 pro) amino acid<br>
SEQ ID NO: 47 (HTLV-3 pro) (pos 1976-2509)<br>
SEQ ID) NO: 48 (HTLV-3 rex) amino acid<br>
SEQ ID NO: 49 (HTLV-3 rex)(pos 5010-5071..7245-7730)<br>
SEQ ID NO: 50 (HTLV-3 tax) amino acid<br>
SEQ ID NO: 51 (HTLV-3 tax) (pos5069-5071..7244-8293)<br>
SEQ ID NO: 52 (HTLV-3 pX (pos 6545-7243)<br>
SEQ ID NO: 53 (HTLV-4 (pol-env-tax region)<br>
SEQ ID NO: 54 (HTLV-4 env) amino acid<br>
SEQ ID NO: 55 (HTLV-4 env surface antigen (SU)=aa 1-307)<br>
SEQ ID NO: 56 (HTLV-4 env transmembrane=aa 308-485)<br>
SEQ ID NO: 57 (HTLV-4pot) amino acid<br>
SEQ ID NO: 58 (HTLV-4 pro) amino acid<br>
SEQ ID NO: 59 (HTLV-4 pro) (pos 1-273)<br>
SEQ ID NO: 60 (HTLV-4 rex) amino acid<br>
SEQ ID NO: 61 (HTLV-4 rex) (pos 2483-2545..4560-5009)<br>
SEQ ID NO: 62 (HTLV-4 tax) amino acid<br>
SEQ ID NO: 63 (HTLV-4 pX) (pos.4000-4558)<br>
SEQ ID NO: 641863PF1<br>
SEQ ID NO: 651863PR2<br>
SEQ ID NO: 661863PP2 FAM (flourescent labeled probe)<br>
SEQ ID NO: 67 region of HTLV-3 where type specific peptides of HTLV-1 and<br>
HTLV-2 are Located SEQ ED NO: 68 region of HTLV-4 where type specific peptides of HTLV-1 and<br>
HTLV-2 are located SEQ ID NO:692026LF1 SEQ ID NO: 70 2026PR1 SEQ ED NO: 71 2026LF2 SEQ ID NO: 72 2026PR2 SEQ ID NO: 73 2026PF3 SEQ ID NO: 74 2026ER1 SEQ ID NO: 75 2026PF2 SEQ ED NO: 76 2026ER3 SEQ ID NO: 77 2026EF1 SEQ ID NO: 78 2026LR1 SEQ ED NO: 79 2026EF3 SEQ ID NO: 80 2026LR2<br><br><br><br><br><br>
V.       CLAIMS<br>
What is claimed is:<br>
1.	An isolated primate T-lymphotropic virus comprising one or more of SEQ ID NOs:<br>
1,3,5, 35, and 45.<br>
2.	The primate T-lymphotropic virus of claim 1, wherein the isolated primate T-<br>
lymphotropic virus comprises SEQ ID NO: 1.<br>
3.	The primate T-lymphotropic virus of claim 2, further comprising a nucleic acid<br>
selected from the group consisting of SEQ ID NO: 3, 5,35, and 45.<br>
4.	The primate T-lymphotropic virus of claim 3, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 1 and 3.<br>
5.	The primate T-lymphotropic virus of claim 3, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 1 and 5.<br>
6.	The primate T-lymphotropic virus of claim 3, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 1,3, and 5.<br>
7.	The primate T-lymphotropic virus of claim 3, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 1,3,35, and 45.<br>
8.	The primate T-lymphotropic virus of claim 1 comprising SEQ ID NO: 3.<br>
9.	The primate T-lymphotropic virus of claim 1 comprising SEQ ID NO: 5.<br>
10.	An isolated primate T-lymphotropic virus comprising one or more of SEQ ID NOs:<br>
2,4, and 6.<br>
11.	The primate T-lymphotropic virus of claim 10, wherein the isolated primate T-<br>
lymphotropic virus comprises SEQ ID NO: 2.<br>
12.	The primate T-lymphotropic virus of claim 11, further comprising a nucleic acid<br>
selected from the group consisting of SEQ ID NO: 4 and 6.<br>
13.	The primate T-lymphotropic virus of claim 12, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 2 and 4.<br>
14.	The primate T-lymphotropic virus of claim 12, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 2 and 6.<br>
15.	The primate T-lymphotropic virus of claim 12, wherein the primate T-lymphotropic<br>
virus comprises SEQ ID NOs: 2,4, and 6.<br>
16.	The primate T-lymphotropic virus of claim 10 comprising SEQ ID NO: 4.<br>
17.	The primate T-lymphotropic virus of claim 10 comprising SEQ ID NO: 6.<br>
18.	An isolated primate T-lymphotropic virus having a gag gene that has less than 69%<br>
identity to the gag gene of HTLV-1, HTLV-2, STLV-2, and STLV-3.<br>
19.	An isolated primate T-lymphotropic virus having a pol gene that has less than 63.5%<br>
identity to the pol gene of HTLV-1, HTLV-2, STLV-2, and STLV-3.<br>
20.	An isolated primate T-lymphotropic virus having a pol gene that has less than 62%<br>
identity to the pol gene of HTLV-1, HTLV-2,STLV-2, and less than 86% identity to<br>
the pol gene of STLV-3.<br>
21.	An isolated primate T-lymphotropic virus having a LTR that has less 41% identity to<br>
the LTR of HTLV-1, HTLV-2 and STLV-3.<br>
22.	An isolated primate T-lymphotropic virus having at least 92.8% identity to the<br>
nucleic acid SEQ ID NO: 1.<br>
23.	An isolated primate T-lymphotropic virus having at least 92.5% identity to the<br>
nucleic acid SEQ ID NO: 3.<br>
24.	An isolated primate T-lymphotropic virus having at least 94.2% identity to the<br>
nucleic acid SEQ ID NO: 5.<br>
25.	An isolated primate T-lymphotropic virus having at least 92.8% identity to the<br>
nucleic acid SEQ ID NO: 1, at least 92.5% identity to the nucleic acid SEQ ID NO:<br>
3, and at least 94.2% identity to the nucleic acid SEQ ID NO: 5.<br>
26.	An isolated primate T-lympnotropic virus having at least 71.5% identity to the<br>
nucleic acid SEQ ID NO: 2.<br>
27.	An isolated primate T-lymphotropic virus having at least 73.5% identity to the<br>
nucleic acid SEQ ID NO: 4.<br>
28.	An isolated primate T-lymphotropic virus having at least 82% identity to the nucleic<br>
acid SEQ ID NO: 6.<br>
29.	An isolated primate T-lymphotropic virus having at least 71.5% identity to the<br>
nucleic acid SEQ ID NO: 2, at least 73.5% identity to the nucleic acid SEQ ID NO:<br>
4, and at least 82% identity to the nucleic acid SEQ ID NO: 6.<br>
30.	A vector comprising the primate T-lymphotropic virus of any of claims 1-29.<br>
31.	The vector of claim 30, further comprising a heterologous nucleic acid.<br>
32.	The vector of claim 31, wherein the heterologous nucleic acid is a nucleic acid<br>
encoding an antigen from a known pathogen under the control of a promoter<br>
element.<br>
33.	The vector of claim 31, wherein the pathogen is a virus selected from the group of<br>
viruses consisting of Herpes simplex virus type-1, Herpes simplex virus type-2,<br>
Cytomegalovirus, Epstein-Barr virus, Varicella-zoster virus, Human herpesvirus 6,<br>
Human herpesvirus 7, Human herpesvirus 8, Variola virus, Vesicular stomatitis<br>
virus, Hepatitis A virus, Hepatitis B virus, Hepatitis C virus, Hepatitis D virus,<br>
Hepatitis E virus, Rhinovirus, Coronavirus, Influenza virus A, Influenza virus B,<br>
Measles virus, Polyomavirus, Human Papilomavirus, Respiratory syncytial virus,<br>
Adenovirus, Coxsackie virus, Dengue virus, Mumps virus, Poliovirus, Rabies virus,<br>
Rous sarcoma virus, Yellow fever virus, Ebola virus, Marburg virus, Lassa fever<br>
virus, Eastern Equine Encephalitis virus, Japanese Encephalitis virus, St. Louis<br>
Encephalitis virus, Murray Valley fever virus, West Nile virus, Rift Valley fever<br>
virus, Rotavirus A, Rotavirus B, Rotavirus C, Sindbis virus, Simian<br>
Immunodeficiency virus, Human T-lymphotropic virus type-1, Human T-<br>
lymphotropic virus type-2, Primate T-lymphotropic virus, Hantavirus, Rubella virus,<br>
Human Immunodeficiency virus type-1, and Human Immunodeficiency virus type-2, and Simian Immunodeficiency virus (SIV).<br>
34.	The vector of claim 31, wherein the pathogen is a bacterium selected from the group<br>
of bacteria consisting of M tuberculosis, M. bovis, M. bovis strain BCG, BCG<br>
substrains, M. avium, M. intracellulare, M. africanum, M. kansasii, M. marinum, M.<br>
ulcerans, M. avium subspecies paratubercuhsis, Nocardia asteroides, other<br>
Nocardia species, Legionellapneumophila, other Legionella species, Salmonella<br>
typhi, other Salmonella species, Shigetta species, Yersiniapestis, Pastewrella<br>
haemolytica, Pastewrella multocida, other Pasteurella species, Actinobacillus<br>
pleuropneumoniae, Listeria monocytogenes, Listerla ivanovii, Brucella abortus,<br>
other Brucella species, Cowdria ruminantium, Chlamydia pneumoniae, Chlamydia<br>
trachomatis, Chlamydia psittaci, Coxiella burnettt, other Rickettsial species,<br>
Ehrlichla species, Staphylococcus aureus, Staphylococcus epidermidis,<br>
Streptococcus pyogenes, Streptococcus agalactiae, Bacillus anthracis, Escherichia<br>
coli, Vibrio choleras, Campylobacter species, Neiserria meningitidis, Neiserria<br>
gonorrhea, Pseudomonas aeruginosa, other Pseudomonas species, Haemophilus<br>
influenzae, Haemophilus ducreyi, other Hemophilus species, Clostridium tetani,<br>
other Clostridium species, Yersinia enterolitica, and other Yersinia species.<br>
35.	The vector of claim 31, wherein the pathogen is a parasite selected from the group of<br>
parasites consisting of Toxoplasma gondii, Plasmodiumfalciparwn, Plasmodium<br>
vivax, Plasmodium malariae, other Plasmodium species., Trypanosoma brucei,<br>
Trypanosoma cruzi, Leishmania major, other Leishmania species., Schistosoma<br>
mansoni, other Schistosoma species., and Entamoeba histolytica.<br>
36.	The vector of claim 31, wherein the pathogen is a fungi selected from the group of<br>
fungi consisting of Candida albicans, Cryptococcus neoformans, Histoplama<br>
capsulatum, Aspergillusfianigatus, Coccidiodes immitis, Paracoccidiodes<br>
brasiliensis, Blastomyces dennitidis, Pneomocystis carnii, Penicillium marneffi, and<br>
Alternaria alternata.<br>
37.	The vector of claim 31, wherein the antigen is a cancer related antigen selected from<br>
the group of cancers consisting of lymphoma, B cell lymphoma, T cell lymphoma,<br>
mycosis fungoides, Hodgkin's Disease, myeloid leukemia, bladder cancer, brain cancer, nervous system cancer, head and neck cancer, squamous cell carcinoma of head and neck, kidney cancer, lung cancers such as small cell lung cancer and non-small cell lung cancer, neuroblastoma/glioblastoma, ovarian cancer, pancreatic cancer, prostate cancer, skin cancer, liver cancer, melanoma, squamous cell carcinomas of the mouth, throat, larynx, and lung, colon cancer, cervical cancer, cervical carcinoma, breast cancer, and epithelial cancer, renal cancer, genitourinary cancer, pulmonary cancer, esophageal carcinoma, head and neck carcinoma, large bowel cancer, hematopoietic cancers; testicular cancer; colon and rectal cancers, prostatic cancer, and pancreatic cancer.<br>
38.	The vector of claim 37, wherein the cancer antigen is selected from the group of<br>
cancer antigens consisting of human epithelial cell mucin (Muc-1; a 20 amino acid<br>
core repeat for Muc-1 glycoprotein, present on breast cancer cells and pancreatic<br>
cancer cells), the Ha-ras oncogene product, p53, carcino-embryonic antigen (CEA),<br>
the raf oncogene product, gpl00/pmell7, GD2, GD3, GM2, TF, sTn, MAGE-1,<br>
MAGE-3, BAGE, GAGE, tyrosinase, gp7.5, Melan-A/Mart-1, gpl00, HER2/neu,<br>
EBV-LMP 1 &amp; 2, HPV-F4, 6, 7, prostate-specific antigen (PSA), HPV-16, MUM,<br>
alpha-fetoprotein (AFP), CO17-1A, GA733, gp72, p53, the ras oncogene product,<br>
HPV E7, Wilm's tumor antigen-1, telomerase, and melanoma gangliosides.<br>
39.	A method of vaccinating a subject against an illness associated with a pathogen<br>
comprising administering the vector of claim 30.<br>
40.	A method of identifying new primate T-lymphotropic viruses comprising:<br><br>
a)	contacting a nucleic acid using a first set of primers and a second set of<br>
primers internal to the first set of primers, wherein the first set of primers is<br>
SEQ ID NOs: 19 and 20, wherein the second set of primers is SEQ ID NOs:<br>
21 and 22;<br>
b)	identifying any amplified nucleic acid; and<br>
c)	comparing the sequence to known primate T-lymphotropic viral sequences,<br>
wherein sequence divergence greater than 5% indicates a new virus.<br>
41.	A method of identifying new pnmate T-lymphotropic viruses comprising:<br>
a)	contacting a nucleic acid using a first set of primers and a second set of<br>
primers internal to the first set of primers, wherein the first set of primers is<br>
selected from the group of primers pairs consisting of SEQ ID NOs: 7 and 8,<br>
SEQ ID NOs: 11 and 12, SEQ ID NOs: 15 and 16, SEQ ID NOs: 23 and 24,<br>
SEQ ID NOs: 27 and 28, SEQ ID NOs: 31 and 32, SEQ ID NOs: 69 and 70,<br>
SEQ ID NOs: 73 and 74, and SEQ ID NOs: 77 and 78, wherein the second<br>
set of primers is selected from the group consisting of SEQ ID NOs: 9 and<br>
10, SEQ ID NOs: 13 and 14, SEQ ID NOs: 17 and 18, SEQ ED NOs: 25 and<br>
26, SEQ ID NOs: 29 and 30, SEQ ID NOs: 33 and 34, SEQ ID NOs: 71 and<br>
72, SEQ ID NOs: 75 and 76, and SEQ ID NOs: 79 and 80;<br>
b)	identifying any amplified nucleic acid; and<br>
c)	comparing the sequence to known primate T-lymphotropic viral sequences,<br>
wherein sequence divergence greater than 5% indicates a new virus.<br><br>
42.	A method of identifying a PTLV comprising contacting a nucleic acid with a set of<br>
primers specific for the surface antigen or transmembrane regions of env and<br>
indentifying any amplified nucleic acid.<br>
43.	A method of identifying new primate T-lymphotropic viruses comprising:<br>
a)	contacting a nucleic acid using a set of primers, wherein the set of primers is<br>
selected from the set of primers consisting of SEQ ED NOs: 19 and 20, SEQ<br>
ID NOs: 21 and 22, SEQ ID NOs: 7 and 8, SEQ ID NOs: 11 and 12, SEQ ID NOs: 15 and 16, SEQ ID NOs: 23 and 24, SEQ ID NOs: 27 and 28, SEQ ID NOs: 31 and 32, SEQ ID NOs: 69 and 70, SEQ ID NOs: 73 and 74, SEQ ID NOs: 77 and 78, SEQ ID NOs: 9 and 10, SEQ ID NOs: 13 and 14, SEQ ID NOs: 17 and 18, SEQ ID NOs: 25 and 26, SEQ ID NOs: 29 and 30, SEQ ID NOs: 33 and 34, SEQ ID NOs: 71 and 72, SEQ ID NOs: 75 and 76, and SEQ ID NOs: 79 and 80;<br>
b)	identifying any amplified nucleic acid; and<br>
c)  comparing the sequence to known primate T-lymphotropic viral sequences, wherein sequence divergence greater than 5% indicates a new virus.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUFic3RyYWN0LSgyMi0xMC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Abstract-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUNsYWltcy0oMDgtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Claims-(08-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUNsYWltcy0oMjItMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Claims-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDgtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Correspondence Others-(08-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Correspondence Others-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6679-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LURyYXdpbmdzLSgyMi0xMC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Drawings-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUZvcm0tMy0oMjItMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Form-3-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LUdQQS0oMjItMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-GPA-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMjItMTAtMjAxMyktMS5wZGY=" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Petition-137-(22-10-2013)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjY3OS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMjItMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">6679-delnp-2007-Petition-137-(22-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="263048-method-and-system-for-controlling-a-power-inverter-in-electric-drives-of-vehicles-with-two-mode-transmissions.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263050-service-assembly.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263049</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6679/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>41/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Sep-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Aug-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>(1) THE GOVERNMENT OF THE UNITED STATES OF AMERICA AS REPRESENTED BY THE SECRETARY OF THE DEPARTMENT OF HEALTH AND HUMAN SERVICES,CENTERS FOR DISEASE CONTROL AND PREVENTION (2) JOHN HOPKINS UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGY TRANSFER OFFICE, 4770 BUFORD HIGHWAY (K79), ATLANTA, GA 30341, USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DONALD BURKE</td>
											<td>3241 LIVINGSTON ST., WASHINGTON, DISTRICT OF COLUMBIA 20015,USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WILLIAM SWITZER</td>
											<td>5745 REDCOATRUN, STONE MOUNTAIN,GEORGIA 30087, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>WALID HENEINE</td>
											<td>2830 WOODLAND PARK DRIVE, ATLANTA, GEORGIA 30345, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>THOMAS FOLKS</td>
											<td>3815 BELE GLADE TRAIL, SNELLVILLE, GEORGIA 30039, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NATHAN WOLFE</td>
											<td>1101 ST. PAUL, #604, BALTIMORE, MARYLAND 21202, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MPOUDI EITEL</td>
											<td>P.O. BOX 906, YAOUNDE, CAMEROON</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N7/00; C12N15/48</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/005869</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-02-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/654,484</td>
									<td>2005-02-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263049-an-isolated-primate-t-lymphotropic-virus-and-vector-comprising-the-said-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:31:23 GMT -->
</html>
