<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263062-oligonucleotides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:28:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263062:&quot;OLIGONUCLEOTIDES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;OLIGONUCLEOTIDES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A single stranded oligonucleotide having substantially no secondary structure and formed of nucleotide residues wherein two or more internal nucleotide residues are labelled with a fluorophore without an associated quencher. Typically at least two of the nucleotide residues labelled with a fluorophore are separated by at least two unlabelled nucleotide residues. The oligonucleotides are useful in the investigation of target polynucleotide sequences.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>OLIGONUCLEOTTOES<br>
The present invention relates to oligonucleotides, in particular to fluorescently labelled oligonucleotides for use as hybridisation probes.<br>
A multitude of technologies for detecting specific DNA sequences ond scoring known single nucleotide polymorphisms are known. Several of these detection methods utilise hybridisation of fluorescent probes and rely on the transfer of energy between donor and acceptor moieties.<br>
Homogenous Polymerase Chain Reaction (PCR) is a method that couples amplification and detection processes. The benefits of homogeneous PCR over traditional methods, such as target amplification followed by gel electrophoresis, are a reduction in test duration, hands-on-time and the potential for cross-contamination. Fluorescent probes are employed with real-time PCR instruments, such as the LightCycler (Roche), ABI PRISM 7700 &amp; 7900 (Applied Biosystems), Smartcycler (Cepheid), Rotogene (Corbett Research), MX4000 (Stratagene), SynChron (Biogene) and iCycler (BioRad), to detect and identify amplified products during (real-time) or post (end-point) amplification. Examples of probe based technologies include, molecular beacons (Tyagi et al, 1996), the 5'-exonuclease assay (US 5,691,146), hybridisation probes (US 6,174,670), Scorpion primers (Whitcombe et al, 1999), and ResonSense (Lee et al, 2002). Fluorescent probes are typically in a quenched or unexcited state until hybridisation to a target sequence results in dequenching, such as by conformational changes or probe degradation, or excitation as a result of p^obe interactions. These probe systems rely on the transfer of energy between donor (e.g. fluorophore) and acceptor (e.g. quencher or secondary fluorophore) moieties. The energy absorbed by a fluorophore may be transferred to a quencher or an acceptor fluorophore and released as heat or emitted as light of a different wavelength. Quenching of fluorescent signal may occur by Fluorescence Resonance Energy Transfer (FRET) or non-FRET mechanisms. FRET quenching requires a spectral overlap between the donor and acceptor, where the efficiency<br><br>
of quenching is related to the distance between the two moieties. Non-FRET quenching occurs through short-range 'contacts' between fluorophore and quencher, requiring no spectral overlap between moieties.<br>
The 5'-exonuclease (TaqMan™) assay uses FRET quenching to detect the PCR amplified target DNA. TaqMan probes are oligonucleotides that contain fluorophore and quencher moieties preferably located on 5' and 3' termini. Very little fluorescence is emitted from intact probe due to efficient intra-molecular quenching. However, during PCR amplification, the probe specifically hybridises to its target sequence and the 5'-3'-exonuclease activity of Taq polymerase cleaves the probe between fluorophore and quencher moieties. Enzymatic cleavage of TaqMan Probes spatially separates fluorophore and quencher components, causing significant increases in fluorescence emission correlated with target amplification. Careful design of TaqMan Probes allows discrimination of polymorphic targets, where only perfectly matched probes are degraded to generate increases in fluorescence signal. Since TaqMan Probes are digested during real-time PCR amplification, probes are not available for post-amplification sequence analysis.<br>
Eclipse Probes are short linear oligonucleotides that have a minor groove binder (MGB) and a quencher moiety at the 5' end and a fluorophore attached to the 3' end. It is thought that the MGB prevents the exonuclease activity of Taq from cleaving the probe. Random coiling of the probe brings the fluorophore and quencher components into close proximity causing fluorescence quenching. Hybridisation to target sequences causes the probe to straighten out such that fluorophore and quencher moieties become spatially separated.<br>
Molecular Beacons are single-stranded oligonucleotide probes that are non-fluorescent in isolation, but become fluorescent upon hybridisation to target sequences. Non-hybridised Molecular Beacons form stem-loop structures, possessing a fluorophore covalently linked to one end of the molecule and a quencher linked to the other, such that the hairpin of the beacon places the<br><br>
fluorophore moiety in close proximity with the quencher. When Molecular Beacons hybridise to target sequences, fluorophore and quencher moieties become spatially separated, such that the fluorophore is no longer quenched and the Molecular Beacon fluoresces. Molecular Beacons may be employed in real-time and end-point assays for sequence detection and SNP discrimination. The secondary structure of the Molecular Beacon conveys high specificity to the probe, allowing the identification of targets that differ by a single nucleotide. However, the Molecular Beacon's intra-molecular interaction potentially produces a source of competition for inter-molecular target hybridisation and, because the Molecular Beacon is an internal probe, it must compete with the amplicon's opposite strand for binding to the target sequence. The combination of both forms of competition may reduce Molecular Beacon hybridisation efficiency to some target molecules.<br>
Scorpions are PCR primers with a stem-loop tail containing a fluorophore and a quencher (Whitcombe et al, 1999). The stem-loop structure brings fluorophore and quencher components into close proximity. The loop component of the Scorpion contains the probing sequence and target detection occurs by a highly efficient intra-molecular mechanism. During PCR, Scorpions are extended to form products that include the probe target sequence. The tail of the Scorpion folds over and enables hybridisation of probe and target sequences. Hybridisation of the probing sequence to the amplified target is thermodynamically favoured over the stem-loop structure and careful probe design enables discrimination of targets differing by as little as a single nucleotide. Probe hybridisation causes the stem-loop structure to dissociate such that fluorophore and quencher components become spatially separated and large increases in fluorescence emission are generated. The stem-loop tail is separated from the PCR primer sequence by a PCR stopper (e.g. HEG) that prevents the DNA polymerase from copying the stem-loop sequence of the Scorpion. Duplex Scorpions (Solinas et al, 2001) utilise two labelled oligonucleotides, one comprising the primer, fluorophore and probing sequence, and the other containing the quencher moiety. Again, extension of the duplex Scorpion enables the probing sequence to fold over and hybridise to<br><br>
the amplified target, thereby dissociating fluorophore and quencher oligonucleotides and increasing the amount of fluorescent signal emitted.<br>
Hybridisation Probes are oligonucleotides that are singly labelled with fluorophore moieties. Two such oligonucleotides are required for each Hybridisation Probe assay, one labelled with a donor fluorophore and the other with an acceptor fluorophore. Fluorescein is commonly employed as the donoi and Cy5, LC-RED 640 and LC-RED 705 are commonly used as acceptors. Excitation of the donor fluorophore produces an emission spectrum that overlaps with the absorption spectrum of the acceptor fluorophore. Hybridisation Probe pairs are designed to recognise adjacent nucleotide sequences within target molecules. In isolation, the acceptor oligonucleotide is not excited and does not generate a fluorescent signal. However, during hybridisation to polynucleotide target sequences, the donor and acceptor probes are brought into close proximity, allowing fluorescence resonance energy transfer from the donor to the acceptor. Fluorescent signal from the acceptor fluorophore is only emitted when both probes are hybridised to the target molecule. When incorporated into PCR reactions, fluorescence from the acceptor probe is monitored once per cycle of amplification, to facilitate real-time measurement of product accumulation, where the amount of fluorescence emitted by the acceptor is proportional to the quantity of target synthesised. Careful design of probes and assay conditions permits discrimination of closely related targets by real-time PCR. Furthermore, pairs of Hybridisation Probes may be employed to discriminate alleles by melt peak analysis and Tm determination. Homozygous samples may be identified by the generation of specific peaks during melting curve analysis and heterozygous samples may be identified by the presence of two peaks within a single melting trace.<br>
ResonSense Probes are single-labelled oligonucleotides that bind to target sequences between the positions of primer pairs. ResonSense Probes do nut yield detectable signal in isolation or the absence of target sequence. Instead, DNA intercalaters, such as SYBR Gold, are included to excite the ResonSense Probes. The DNA intercalater binds to duplex DNA and, when excited by the real-time<br><br>
instrument, transfers the energy to the fluorophore of the ResonSense Probe (Lee et al, 2002).<br>
The above probe technologies utilise FRET and quencher labels to detect and discriminate polynucleotide sequences. However, the alternative Light-up Probe (Svanvik et al, 2000) system does not require quencher labels or FRET transfer between donor and acceptor moieties to detect and discriminate DNA sequences. These Light-up Probes comprise a sequence recognising oligonucleotide, composed of peptide nucleic acid (PNA), and a single fluorescent reporter group, where the reporter is typically a derivative of the asymmetric cyanine dye thiazole orange and carries a positive charge. The fluorescent dye component of Light-up Probes is attached to the terminus of the oligonucleotide molecule. When single-stranded, probes emit significantly lower levels of fluorescence than when hybridised to complementary nucleic acid sequences. In single-stranded probes, fluorophores are free and excited energy may be released in the form of movement or heat, such that probes do not fluoresce significantly. Upon hybridisation, the positively charged fluorophore inserts between the bases of DNA and becomes 'locked' into position as a result of the charge interaction, such that excited energy may only be released as light. By measuring the quantity of fluorescence emission, Light-up Probes may be employed to detect nucleic acid sequences and differentiate between targets differing by a single position.<br>
Several other probe technologies have also been described that do not require the attachment of quencher moieties to enable target detection and identification. The emission of many fluorophores has been reported to change through their interaction with nucleobases (Siedel et al, 1996, Knemeyer et al, 2000, Lee et al, 1994, Crockett &amp; Wittwer 2001, Nazerenko et al, 2002). Interaction of fluorophores with DNA has also been reported to affect the wavelength of fluorescence emission. The greatest effect on fluorescence has been attributed to guanine, which is an efficient electron donor. Quenching is thought to arise from electron transfer of the guanine nucleobase to the fluorescent dye. The degree of quenching is reported to depend on the proximity of fluorophores to guanines in<br><br>
probe and target strands (Xu et al, 1994, Hawkins et al, 1997, Norlund et al, 1989). The magnitude and direction of emission change in fluorescent labelled probes is reported to be dependent upon the location of guanine residues in probe and target sequences. LUX Primers (Invitrogen) employ hairpin structures to quench fluorophores attached close to the 3' end of the hairpin (Nazerenko et al, 2002). The hairpin structure of the LUX Primers frequently brings fluorophore moieties into close proximity with a region of G's, thereby quenching fluorescence. When the primer becomes incorporated into double-stranded PCR product, the hairpin structure dissociates causing the fluorophore to become dequenched. The presence of guanine residues in proximity to the label is reported to affect the functionality of LUX Primers. LUX Primers yield fluorescent signal when incorporated into any double-stranded molecule and will, therefore, detect non-specific amplification products and primer-dimers.<br>
Gene Pin Probes (Atto-tec GmbH) or Smart Probes (Knemeyer et al, 2000, Heinlein et al, 2003)) are stem-loop structures similar to Molecular Beacons. The loop is the probing component and the stem is employed to quench fluorescence in the single-stranded state (WO 01/36668). The fluorophore ct GenePin Probes is attached to the poly(C) component of the stem and fluorescence is quenched efficiently by the complementary poly(G) stem component. Upon target hybridisation, the poly(C) and poly(G) components become separated, the affect of guanine quenching is removed and the level of fluorescence emission increases.<br>
Simple Probes are single-labelled oligonucleotides that exhibit changes in fluorescence emission upon target hybridisation (US 6,635,427). Fluorophores are typically terminally attached, where 5' labelled probes also preferably include a 3' phosphate. The magnitude and direction of fluorescence change in Simple Probes is dependent upon the probe DNA sequence flanking the position of fluorophore attachment and the target DNA sequence that the fluorophore interawts with vpon hybridisation. Fluorescence quenching of Simple Probes has been reported upon hybridisation when fluorophores are brought into proximity with deoxyguanosine nucleotides in the target strand (Crockett &amp; Wittwer 2001). In contrast<br><br>
fluorescence enhancement has been reported when terminal fluorophores, attached to or near G residues, are hybridised to complementary target sequences (US 6,635,427). In a specific example, where the fluorophore replaced an internal base ("virtual nucleotide"), the fluorescence of a Simple Probe has been demonstrated to be both enhanced and quenched upon target hybridisation, when the fluorophore was brought into opposition with A and G nucleotides respectively in polymorphic target alleles. Heterozygous samples did not exhibit a significant increase or decrease in fluorescence upon probe/target hybridisation.<br>
Hybridisation beacons (also called HyBeacons®) are internally labelled oligonucleotides that emit greater amounts of fluorescence when hybridised to target sequences than when in the single-stranded state (WO 01/73118). HyBeacons do not require a quencher moiety for fluorescence signal generation, and do not rely on probe secondary structures, enzymatic cleavage or FRET to another fluorophore. In the absence of a quencher label, the reported quenching properties of DNA are responsible for the alteration in fluorescence emission upon hybridisation. Unlike the other single-labelled probe technologies described above, the direction of fluorescence change is not affected by the nucleotide sequence of probe and target strands.<br>
The listing or discussion of a prior-published document in this specification should not necessarily be taken as an acknowledgement that the document is part of the state of the art or is common general knowledge.<br>
The inventors have now found that, surprisingly, oligonucleotides with multiple internal fluorophores, preferably at defined spacings from each other, give unexpectedly large fluorescent signals on binding a target polynucieotide sequence when compared to otherwise equivalent single-labelled versions.<br>
A first aspect of the invention provides a single stranded oligonucleotide having substantially no secondary structure and formed of nucleotide residues wherein<br><br>
two or more internal nucleotide residues are labelled with a fluorophore without an associated quencher.<br>
As is discussed in detail below, the oligonucleotides of the invention are particularly useful as hybridization probes.<br>
The oligonucleotides of the invention, like the HyBeacons® described in WO 01/73118, do not rely on oligonucleotide secondary structures, enzyme action or quencher moieties attached to oligonucleotide in order to be effective as readily detectable hybridisation probes. Interaction between these oligonucleotides and their target sequences (as discussed below) generate significant alterations in fluorescence emission that enable reliable target detection. Variations in probe stability enable discrimination of targets differing by as little as a single nucleotide through the measurement of duplex melting temperature as discussed in more detail below.<br>
Contrary to the sequence dependent data presented in US 6,635,427, which suggests that fluorophores attached to oligonucleotides would be expected to be quenched upon probe/target hybridisation due to fluorophores being flanked by C residues and target strands having an abundance of G residues in the region of the fluorophore, the oligonucleotides of the invention exhibit enhanced levels of fluorescence when hybridised to full complementary and (partially) mismatched (ie partially complementary) target sequences, irrespective of the probe and target sequences.<br>
Data demonstrates that the fluorophores of the oligonucleotides of the invention interact with the single-stranded oligonucleotide itself, but not with duplex DNA in the hybrid formed upon hybridisation. Therefore, while the sequence of the probe may affect the magnitude of fluorescence change, probes do not become quenched upon hybridisation. Even when fluorophore-labelled bases of the oligonucleotides of the invention are brought into proximity with a G rich target region upon hybridisation, dequenching and positive melt peaks (-dF/dT yields<br><br>
positive value) are observed as demonstrated herein, particularly with oligonucleotide probes where the directly fluorophore labelled nucleotide residues are separated by two or more unlabelled nucleotide residues. The sequence independence of the oligonucleotides of the invention is in contrast to that reported by Nazerenko et al, (2002b), where internal fluorescein dT emission was enhanced or quenched upon hybridisation dependent upon the location of the fluorophore within the probe and the proximity of guanines to the fluorophore It was reported that at least one guanine base was required within four nucleotides of the label to enable fluorescence enhancement upon hybridisation.<br>
As with the HyBeacons® of WO 01/73118, the multiple-labelled oligonucleoiides of the invention described herein yield increased levels of fluorescence enhancement irrespective of probe and target sequences and do not require guanines at defined locations to facilitate fluorescence enhancement upon target hybridisation. Although not being bound by theory, it is believed that the fluorophores attached to the oligonucleotides of the invention interact with probe (oligonucleotide) DNA when single-stranded (for example, by being buried in a single-stranded hydrophobic pocket) thereby causing fluorescence quenching (eg by a base stack causing collisional quenching). However, it is believed that fluorophores do not interact with duplex DNA when probes are hybridised to a target, such that the fluorophores attached to the oligonucleotides of the invention project into solution and exhibit elevated levels of emission, where such increases are independent of the target sequence. Molecular and computational modelling described in Marks et al (2005), incorporated herein by reference, is relevant to this point.<br>
By "single stranded oligonucleotide having substantially no secondary structure" we include the meaning that the oligonucleotide does not have a sequence whereby there are substantial parts which are the inverse complement of each other which allow for intramolecular base pairing. By "substantial parts" we mean four or more consecutive nucleotide residues. In particular, unlike prior art oligonucleotide probes such as Molecular Beacons, Scorpions, LUX and Smart<br><br>
probes which are designed to contain substantial regions of inverse complementarity, the probes of the invention are typically designed to avoid such inverse complementarity. The oligonucleotide probes of the invention only employ the inverse complement of the target and do not include additional sequences to generate secondary structure.<br>
Potential secondary structure of an oligonucleotide may be predicted using nucleic acid folding software such as Mfold (Zucker (2003)). The stability of secondary structure is frequently represented by its free energy (-G) ie the energy required to break the secondary structure, where large negative -G values indicate stable structures. The probes of the invention typically possess positive or small negative -G values, preferably less negative than -2kcal/mol and more preferably less negative than -1 kcal/mol. Probe technologies that rely on secondary structure, such as Scorpions, typically exhibit much greater negative -G values. The Scorpion probes described by Thelwell et al (2000) possess -G values ranging from -7.7 kcal/mol to -12.2 kcal/mol.<br>
Secondary structure arises when one region of an oligonucleotide hybridises with another e.g. forming a loop (as in conventional Molecular Beacons) which decreases the efficiency of the oligonucleotide hybridising with its target. In the oligonucleotides of the invention there is substantially no tendency for one region of the oligonucleotide to hybridise with another, and this can be assessed, for example, using the Mfold software mentioned above.<br>
Fluorescent groups (fluorophores) are attached to internal bases in the oligonucleotide of the invention, as opposed to 5' or 3' termini, and are believed to be quenched to a greater extent when single-stranded than when in the duplex (ie upon hybridisation to the target), as the DNA bases are proposed to be able to stack on the fluorophore when they are not forming base pairs. In the single-stranded state, the fluorophore may be sandwiched between two DNA bases forming a stable hydrophobic structure. This structure will open up upon duplex formation as the nucleotides participate in base pairing.<br><br>
With terminal labels, base stacking may still only occur on one side of the fluorophore on hybridisation. When the oligonucleotides of the invention hybridise, the fluorophore will not be positioned between DNA bases, but is able to extend unhindered from the base. Terminal labelled probes such as Simple Probes will become quenched upon duplex formation if the fluorophore is in closer proximity to more guanines in the duplex than in the single-stranded state. However, if terminal labels are attached to the probe in a region of high guunine abundance, fluorescence will be enhanced upon duplex formation (US 6,635,427). This sequence dependence of fluorescence quenching and enhancement, according to the location and abundance of guanines in the probe and target strand, is not observed in the oligonucleotides of the invention.<br>
All DNA bases are able to quench fluorescence to some extent, where G has the greatest such ability. For the avoidance of doubt, the term "associated quencher" does not include DNA bases which form part of the oligonucleotide. Fluorophores on the oligonucleotides of the invention will interact with the bases of single-stranded probes such that fluorescence is quenched. Since fluorophores are expected to project into solution upon target hybridisation and not interact with duplex DNA, the sequence of the target may not affect the direction of fluorescence change, i.e. guanines in the target may not quench fluorescence upon hybridisation. Therefore, the oligonucleotides of the invention exhibit increased levels of fluorescence when hybridised to target sequences. Gs may modulate fluorescence strength but all dequench significantly on hybridisation. Fluorescence from the internally attached fluorophores of the oligonucleotides of the invention is always enhanced upon duplex formation irrespective of the location and abundance of guanines in the probe and target strand.<br>
The dual and multiple-labelled oligonucleotides of the invention emit significantly greater amounts of fluorescence when hybridised to complementary nucleic acid sequences compared with the single-stranded (non-hybridised) conformation despite the absence of a quencher component although simple spacing constraints<br><br>
as disclosed herein are typically followed in order to avoid adverse interactions leading to reduced peak heights or negative peaks as shown in Figure 11.<br>
The signal-to-noise ratio is the ratio of the signal intensity of a double-stranded hybrid comprising an oligonucleotide of the invention to the signal intensity of the single-stranded probe and is preferably as large as possible, e.g. 2 or more. Signal-to-noise ratios in the oligonucleotides of the invention are significant and usefully greater than 1 despite the absence of associated quencher inoieties. It is believed that fluorophore moieties emit significantly more fluorescent signal when probes are hybridised to target molecules than when in the single-stranded state due to some form of interaction with DNA sequences. The dual or multiple-labelled oligonucleotide probes of the invention possess ratios that are greater than expected from single-labelled oligonucleotides of identical sequence.<br>
Fluorescence enhancement occurs upon target hybridisation when fluorophore-labelled residues are placed in all sequence environments. Placing fluorophore-labelled residues adjacent to G's may result in the highest levels of fluorescence enhancement in the duplex state. However, fluorescence enhancement upon target hybridisation will also occur when fluorophore-labelled residues are located within regions of high C abundance. In one embodiment of this invention, the residues in the target strand are not responsible for either quenching or enhancing probe fluorescence upon hybridisation. The nature of the interaction of the fluorophore with the oligonucleotide differs between free and hybridised oligonucleotide so that the oligonucleotides of the invention exhibit higher levels of fluorescence emission when in duplexes than when single-stranded demonstrating that there are no apparent sequence constraints for functionality, although the sequence can affect the magnitude of fluorescence enhancement.<br>
Typically, the oligonucleotide has 2 or 3 or 4 or 5 or 6 or 7 or 8 or 9 or 10 internal residues labelled with a fluorophore. The number may depend on the length of the oligonucleotide. Typically up to about one-third of the internal residues are labelled with a fluorophore, but it may be fewer.<br>
The length of the oligonucleotide of the invention is preferably such that it is suitable for hybridising with a complementary polynucleotide target, to provide a stable hybrid whose melting temperature depends on the exact sequence of the target. Oligonucleotides containing less than 15 nucleotide residues in many cases do not form sufficiently stable hybrids, particularly where the two hybridising sequences are not fully complementary, although they can be used in some circumstances. Oligonucleotides, which are longer than about 30 nucleotide residues may form hybrids whose melting temperature is relatively insensitive to the possible presence of a single nucleotide mismatch, although they can be used in some circumstances.<br>
Typically, the oligonucleotide is from 10 to 50 nucleotide residues in length, preferably from 15 to 30 nucleotide residues in length. Thus, typically, the oligonucleotide is from 10 or 11 or 12 or 13 or 14 or 15 nucleotide residues in length up to (and including) 25 or 26 or 27 or 28 or 29 or 30. Thus, the invention includes Oligonucleotides within any of the size ranges mentioned.<br>
An oligonucleotide within the size range from 15 to 30 nucleotides may have up to around 10 of its internal nucleotide residues labelled with a fluorophore, but conveniently an oligonucleotide in this size range has 2 or 3 or 4 or 5 of its internal residues labelled with a fluorophore.<br>
It is preferred if at least two (or the two as the case may be) of the nucleotide residues labelled with a fluorophore are separated by at least two unlabelled nucleotide residues. Typically, there are 2 or 3 or 4 or 5 or 6 unlabelled nucleotide residues that separate the labelled nucleotide residues. It is particularly preferred if two unlabelled residues separate the labelled residues. It is also particularly preferred if all of the labelled nucleotide residues are separated by at least two unlabelled nucleotide residues. Preferably, the fluorophores are spaced such that direct 'contact' quenching between the fluorophores is avoided. Contact quenching arises from physical contact of fluorophores and is typically avoided by<br>
physical separation (ie at least 2 nucleotide residues between fluorophorc-labelled bases).<br>
Nucleotide residues are usually derived from the naturally occurring nuoleosides A, C, G, T and U. However, nucleotide analogues may be used at one or more locations of the oligonucleotide of the invention, such nucleotide analogues being modified e.g. in the base portion and/or the sugar portion and/or the phosphate link. Base modifications, such as propynyl dU (dT-analogue) and 2-amino dA (dA analogue), generally alter the hybridisation properties and may make the use of oligonucleotides having less than 15 nucleotide residues attractive. In the case of propynyl dU-containing oligonucleotides, they be around 10 residues in length depending on the melting temperature with the target sequence required.<br>
Alternatively, oligonucleotides composed of or comprising peptide nucleic acid (PNA), locked nucleic acid (LNA), 2'-O-methyl RNA, phosphoramidite DNA, phosphorothioate DNA, methyl phosphonate DNA or phosphotriester DNA may be employed to form chemically or enzymatically more stable interactions with target sequences.<br>
It is preferred if the same fluorophore is used throughout the oligonucleotide of the invention. Any fluorophore that can be attached to a nucleotide residue may be used, provided that it does not prevent the oligonucleotide from hybridising to its target sequence.<br>
Suitable fluorophores include fluorescein-based fluorophores such as FAM (6-<br>
Carboxyfluorescein),	TET	(Tetrachlorofluorescein),	HEX<br>
(hexachlorofluorescein); rhodamine-based fluorophores such as ROX (6-Carboxy-X-Rhodamine) and TAMRA (6-Carboxytetramethylrhodamine); the Cy family of dyes, especially Cy3 and Cy5, all available from Glen Research, 22825 Davis Drive, Sterling, VA 20164, USA.<br>
Other fluorescein dyes, for example those with different emission spectra may be used, such as NED and JOE. Other fluorophores may also be used, such as those in the Alexa, Atto, Dyomics, Dyomics Megastokes and Thilyte dye families as detailed in Tables 9 to 15 below.<br>
In a preferred embodiment of the invention, the oligonucleotides are labelled at the 5-position of internal uracil/thymine bases using C6 FAM dU (available from University of Southampton, UK) or Fluorescein dT (available from Glen Research, Sterling, VA) respectively (in this context, the structure of dT and dU are identical and the terms therefore interchangeable). FMOC-protected phosphoramidites may be incorporated at internal T positions within oligonucleotides and can be used as a point of attachment for a variety of fluorescent dyes, including but not limited to FAM, TET, HEX, ROX, TAMRA, Cy3 and Cy5, all available from Glen Research After oligonucleotide synthesis, the FMOC group may be removed from the 2'-protected uridine and a fluorophore phosphoramidite, such as a suitably protected 6-carboxyfluorescein phosphoramidite, may be coupled to the free 2'-hydroxy group. In yet another embodiment, oligonucleotides may be labelled at internal A, C or G positions, where labelled nucleotides are either incorporated as phosphoramidites during solid phase oligonucleotide synthesis or fluorophores attached post oligonucleotide synthesis using protected phosphoramidites (eg 8-aminoalky! -dA, 7-aminoalkyl 7-deaza-dA, N(4)-aminoalkyl dC and 5-aminoalkyl-dC).<br>
Although typically the same fluorophore is used throughout the oligonucleotide of the invention, it may be advantageous to use different fluorophores in the same oligonucleotide. For example, as discussed in more detail in the examples, the presence of both ROX and FAM in the same oligonucleotide is advantageous. It is particularly preferred if the oligonucleotide contains two fluorophores, one of which is ROX and the other FAM. The use of two or more different fluorophores in the same oligonucleotide may be particularly advantageous when the oligonucleotide is used in multiplexing.<br>
It is preferred that the fluorophore does not intercalate into double-standard DNA. It is preferred that the fluorophore is not thiazole orange(TO).<br>
Dual-labelled oligonucleotides of the invention may be expected to exhibit fluorescent signals and background noise that are double that observed with single-labelled probes, such that the signal-to-noise ratio is maintained (ie an additive effect would be expected). However, surprisingly, the signal-to-noise ratios of dual-labelled probes are larger than those observed with single-labelled probes of identical sequence. A similar unexpected increase in the signal to-noise ratio is also observed with other multiple-labelled oligonucleotides of the invention.<br>
Furthermore, as described in more detail in the Examples, dual, triple, quadruple and further multiple-labelled probes have been demonstrated to generate considerably larger melt peaks than would be expected from the cumulative effect of the individual single-labelled probes of identical nucleotide sequence if the effect was simply additive.<br>
Typically, the oligonucleotide of the invention has a sequence complementary to a target polynucleotide sequence. Thus, the oligonucleotide is able to hybridise to the target polynucleotide sequence under appropriate conditions. Thus, unless the context indicates otherwise, by "complementary" we include the meaning that the oligonucleotide is able to hybridise to a target polynucleotide sequence. The oligonucleotide may be fully complementary to the target polynucleotide sequence (ie there is a perfect match in terms of base pairing between the oligonucleotide), or the oligonucleotide may be partially complementary to the target polynucleotide sequence (ie there are one or more mismatches between the oligonucleotide and the target polynucleotide sequence, but the oligonucleotide is still able to hybridise). Typically, when the oligonucleotide is partially complementary with the target polynucleotide, there are fewer than 5 mismatches, preferably 1 or 2 or 3 or 4 mismatches, more preferably one mismatch. Conveniently, the oligonucleotide has at least 70% sequence identity to the<br>
complement of its target, more preferably at least 80% or at least 85% or at least 90% or at least 95%. For example, for an oligonucleotide of 20 residues, there may be 6 or 4 or 3 or 2 or 1 mismatch with the target.<br>
Typically, the target polynucleotide sequence is selected from the group consisting of the human genome including any gene therein or allele Thereof, the mouse genome including any gene therein or allele thereof, and the genome of any infectious agent including any gene therein or allele thereof. Preferably, the oligonucleotide is one which has a sequence complementary to any one of the following human genes or an allele thereof:<br>
\The sheep PrP gene (NM_001009481) is also a preferred gene for analysis.<br>
Also preferably the oligonucleotide is one which has a sequence complementary to the genome or any gene therein or allele thereof of the following infectious agents:<br>
As is described further in the Examples, the oligonucleotide mwy be one which has a sequence complementary to one allele of a gene. For example, it may complementary to a mutant allele of a gene which, in humans, may be associated with a particular genetic disease (eg cystic fibrosis), or it may be complementary to the wild-type allele. It is preferable that the sequence complementary to the<br>
nucleotide sequence difference of the given allele compared to another allele of the gene is located towards the middle of the oligonucleotide (eg within the middle half to one-third of nucleotide residues). This allows for a good discrimination in melting temperatures for a given such oligonucleotide (which may hybridise to both alleles but with different Tms).<br>
In a preferred embodiment, the oligonucleotide according to the invention preferably has a sequence fully complementary to one allele of a known polynucleotide target having a known polymorphism, e.g. a point mutation or a single base insertion or deletion (SNP). The site of polymorphism is preferably, though not essentially, located centrally within the oligonucleotide probe. Alternatively, the oligonucleotide may be complementary to a known non-polymorphic polynucleotide sequence and may simply be used to detect that target, e.g. for rapid pathogen detection. Also, the oligonucleotide may be used to study potentially polymorphic targets with unknown and uncharacterised polymorphisms. The possibility is envisaged of mapping the position and/or nature of unknown polymorphisms by differential oligonucleotide hybridisation and differences in melt peak Tm. When the oligonucleotide of the invention is caused to hybridise with a polynucleotide target, all the above features contribute to the formation of a stable hybrid with an optimal melting temperature and a substantial difference in melting temperatures (Tm) between strands that are perfectly matched and strands which have a single or multiple positions of mismatch.<br>
As discussed in more detail below, particularly when the oliponueleotide of the invention is used as probe of the production of a target polynucleotide sequence in a nucleic acid amplification reaction, the oligonucleotide is one in which the 3' nucleotide residue does not contain a 3' hydroxyl group. It is preferred that the oligonucleotide is one which is not able to act as a primer for a DNA polymerase, and is not capable of ligation by a DNA ligase. Conveniently, the 3' nucleotide residue is a 3'deoxy residue. Also conveniently, the 3' nucleotide residue may have a 3' phosphate, or other 3' groups which would prevent chain extension.<br>
A further aspect of the invention provides an oligonucleotide of the invention immobilised on a solid support. The solid support may be any suitable solid support. Techniques and linkers for immobilising oligonucleotides using supports in forms that leave them free to hybridise with complementary targets in solution are well described in the literature.<br>
A still further aspect of the invention provides an array of oligonucleotides of the invention. The array may comprise two or more individually addressable sites of oligonucleotide immobilised on a solid support. Typically, the array will contain at least 10, or at least 100, or at least 1000 such sites. Typically, the array comprises oligonucleotides with different sequences, typically each different site is associated with an oligonucleotide with a difference sequence. In a preferred embodiment the array is an array of oligonucleotide probes immobilised at spaced locations using a support, wherein the different oligonucleotide probes are different oligonucleotides according to this invention. Furthermore, the oligonucleotides of the invention may be employed to analyse DNA targets immobilised on or within a support, where distinct targets may be positioned at spaced locations on array type formats.<br>
Methods of immobilising oligonucleotides and making arrays of oligonucleotides are well known in the art.<br>
A yet still further aspect of the invention provides a method of investigating a target polynucleotide sequence in a sample containing polynucleotide sequences, the method comprising contacting the sample with an oligonucleotide of the invention which is able to hybridise to the target polynucleotide sequence, and determining whether hybridisation occurs.<br>
The sample may be any suitable sample, and may be one which is known to contain the target polynucleotide sequence, or it may be one where it is not known<br>
that it contains the target polynucleotide sequence, and the purpose of the investigation is to determine whether it does or not.<br>
Since the oligonucleotide is fluorescently labelled, and because there is an increase in fluorescence upon binding of the oligonucleotide to the target, conveniently an increase in fluorescence is used to determine whether hybridisation has occurred between the oligonucleotide and the target polynucleotide sequence.<br>
It will be appreciated that, under given conditions, the oligonucleotide is able to hybridise to its target at certain temperatures. As is discussed in more detail below, it may be desirable to vary the temperature of the hybridisation and detect fluorescence at different temperatures in order to investigate the target polynucleotide sequence. Thus, in a preferred embodiment of the invention the method is performed at a predetermined temperature, or over a range of temperatures, near the melting point (Tm) of the hybrid formed between the target polynucleotide sequence and the said oligonucleotide. Tms are typically determined by melt curve analysis where 50% of the probe is hybridised to target sequence and 50% is dissociated and free in solution.<br>
The oligonucleotide may be fully complementary to the target polynucleotide sequence, and that oligonucleotide will have a characteristic Tm under given hybridisation conditions. Alternatively, the oligonucleotide may not fully complementary to the target polynucleotide sequence, but nevertheless is able to hybridise to it (it is partially complementary), and that oligonucleotide will have a characteristic (but almost certainly different) Tm.<br>
The method of the invention is particularly suited to the analysis of polymorphisms of a gene and so in one embodiment the target polynucleotide sequence investigated is one or more alleles of a gene. An (or the) oligonucleotide used in the method is conveniently one which is able to hybridise to more than one<br>
allele of the gene, and typically has different Tms under given hybridisation condition with respect to the different alleles.<br>
The method may used to detect the presence of the target sequence, for example in a sample in which it is not known whether it is present. The method may also be used to identify the target sequence or it may be used to quantitate the amount of the target polynucleotide sequence in the sample. Typically, the increase in fluorescence upon binding of the oligonucleotide to the target polynucleotide is used for this.<br>
It is particularly convenient if the sample contains polynucleotide sequences produced by amplification since often an initial source of biological material (.•.uch as saliva, a mouth scrape, small quantities of dried blood etc) contains only a small amount of nucleic acid that may contain the target polynucleotide.<br>
Suitably, the polynucleotide sequences are produced by any one or more of polymerase chain reaction (PCR, including asymmetric PCR amplification), reverse transcription PCR (RT-PCR), ligase chain reaction (LCR), strand displacement amplification (SDA), transcription mediated amplification (TMA), rolling circle amplification (RCA) or nucleic acid sequence based amplification (NASBA).<br>
In a preferred embodiment, the amplification of the polynucleotide sequences in the sample is performed in the presence of the oligonucleotide which is able to hybridise to the target polynucleotide sequence. In this embodiment, it is particularly preferred if the 3' nucleotide residue of the oligonucleotide does not contain a 3' hydroxyl. It is particularly preferred if the oligonucleotide of the invention present in the reaction is not capable of chain extension by a DNA polymerase and not capable of ligation by DNA ligase. In this embodiment it is also preferred if observation of fluorescence is made during the amplification of the polynucleotide sequences in the sample.<br>
There may be no need to extract nucleic acid from the biological source prior to amplification, since the process of amplification may be capable of such extraction. Thus, for example, the polynucleotide sequences in the sample may be produced by amplification from a biological source such as saliva without prior nucleic acid extraction.<br>
The invention provides a method of investigating a polynucleotide target having a known polymorphism, which method comprises providing an oligonucleotide probe comprising fluorophore-labelled nucleotide residues. The polynucleotide target is incubated with the oligonucleotide probe to form a hybrid, the oligonucleotide probe exhibiting a higher level of fluorescence when in the form of the hybrid than when in single-stranded form. The level of fluorescence emitted by the oligonucleotide probe is observed at a predetermined temperature, or monitored over a range of temperatures.<br>
Conveniently, if a single oligonucleotide probe is used in an investigation in which the oligonucleotide may hybridise to two different alleles with different Tms, a temperature intermediate to the two Tms may be used, or the assay may be performed over a range of temperatures (ie by melt analysis).<br>
It will be appreciated that the method may include two or more oligonucleotide probes of the invention which hybridise to different target sequences (eg different alleles of the same gene). Conveniently, these may be distinguished by using fluorophores with different fluorescent properties, or the two or more oligonucleotides have different melting characteristics (eg Tms) when hybridised to (or hybridising to) their respective target polynucleotide sequences.<br>
Typically, when a range of temperatures is used, the range encompasses the Tm for the hybrid formed between the oligonucleotide and its polynucleotide target. Conveniently, the temperature range is ± 10°C, typically ± "°C compared to the<br>
Tm.<br>
Conveniently, if two different oligonucleotide probes are included in an investigation the assay is performed over a range of temperatures (ie by melt analysis).<br>
It is particularly useful to carry out the method of the invention such that an observation of the rate of change of the fluorescent signal with temperature is made.<br>
Typically, the oligonucleotides of the invention are ones which form a hybrid with their target polynucleotide sequence wherein the hybrid has a Tm ot from 20°C to 90°C, preferably from 40°C to 70°C.<br>
In a situation where an oligonucleotide probe of the invention is used to detect a fully complementary target sequence and a partially complementary target sequence, preferential hybridisation to the fully complementary sequence may occur if the target concentration exceeds the concentration of probe since mismatched sequences are less stable. In extreme cases, preferential hybridisation may detect only the fully complementary form of the target. Increasing the concentration of the probe may, therefore, be employed to encourage hybridisation to mismatched targets and improve the balance of peak heights.<br>
In one embodiment of this aspect of the invention one or more oligonucleotides of the invention are included in PCR assays and emit greater amounts of fluorescence when hybridised to complementary target sequences than when single-stranded. Following amplification, the presence and identity of target sequences is determined by melting curve analysis (French et al, 2001, French et al, 2002). The stability and melting temperature of hybridised probes depends on the degree of homology between probe and target sequence. Increasing the reaction temperature above the Tm of the oligonucleotide of the invention causes probe/target duplexes to dissociate and the amount of fluorescence emission to decrease Sequences differing by as little as a single nucleotide may be detected and differentiated on the basis of melt peak Tm. Fifty cycles of amplification and melt analysis may be<br>
completed in as little as 16 minutes. One advantage of using the oligonucleotides of the invention for homogeneous sequence analysis derives from their ability to reliably identify homozygous and heterozygous samples using a single oligonucleotide probe.<br>
The polynucleotide target may be DNA, RNA or cDNA, and is used in single-stranded form, or a DNA duplex may be probed to form a triplex. In one embodiment, the polynucleotide target has a known polymorphism, preferably a single nucleotide polymorphism (SNP). The target is incubated under hybridising conditions with an oligonucleotide probe of the invention. It is necessary that the hybrid generate a stronger fluorescence signal than the single-stranded oligonucleotide probe. The melting temperature of the hybrid will depend, amongst other things, on whether the polynucleotide target and oligonucleotide probe are fully complementary or whether there is a single or even a double mismatch arising at or close to the location of the SNP. The method involves observing the level of fluorescence signal, emitted by the oligonuc'sotide probe, at a predetermined temperature near the melting temperature of the hybrid, 01 over a range of temperatures. Two alternatives are described, although others are possible:-<br>
a)	The temperature of a solution containing the hybrid is slowly<br>
raised, while continuously observing a fluorescence signal, in order to construct a<br>
graph of the negative derivative of fluorescence signal intensity with respect to<br>
temperature (-dF/dT) against temperature.   The melting temperature (Tm) of the<br>
hybrid appears as a peak, and provides information about the sequence of the<br>
polynucleotide target.    The Tms generated through melting analysis of the<br>
oligonucleotide of the invention may be used to distinguish polymorphic targets.<br>
Similarly, the analysis may be performed by cooling the solution slowly from high<br>
temperature and determining the annealing temperature.<br>
b)	A solution of the hybrid is held at a predetermined temperature and<br>
the level of fluorescence observed.   Generally, the predetermined temperature is<br>
chosen to be intermediate between the melting temperature of a perfectly matched<br>
hybrid and the melting temperature of a hybrid with one or two mismatches. The<br>
level of fluorescence signal observed provides an indication of whether or not the hybrid has dissociated, and so provides information about the sequence of the polynucleotide target. Polymorphic targets may be distinguished in end-point and real-time formats.<br>
Conveniently, the polynucleotide target may be a PCR amplimer, formed by using suitable primers to amplify a desired portion of genomic DNA. It may be convenient to perform the amplification and target investigation in a homogenous mode, e.g. in a single reaction vessel by adding the oligonucleotide probe before, during or after the amplification cycling procedure. It may also be convenient to perform target amplification and sequence investigation directly from samples, such as saliva, without prior extraction of DNA, RNA or cDNA. Preferably the oligonucleotide probe is modified at its 3'-end so as to prevent chain extension during PCR amplification. In another aspect, target amplification mav be performed using asymmetric PCR to generate an abundance of the single-stranded target. Preferential probe hybridisation to fully complementary target sequences over mismatched targets may be avoided by increasing probe concentration. Target amplification for assays using the oligonucleotide of the invention may also be performed using alternate techniques such as ligation chain reaction (LCR) rolling circle amplification (RCA), strand displacement amplification (SDA), transcription mediated amplification (TMA) and nucleic acid sequence based amplification (NASBA). Typically, probe concentrations higher than 150nM are used in asymmetric assays.<br>
It may be convenient to provide two or more of the oligonucieotides of the invention, one fully complementary to each allele of the SNP under investigation. Where each oligonucleotide of the invention carries a different fluoiophore, it may be convenient to mix the probes in solution for analysis of homozygous and heterozygous targets. In the same way, a mixture of oligonucieotides of the invention complementary to the various alleles of several different SNPs may be used together in solution for multiplex analysis, provided that each is labelled with a spectrally distinct fluorophore. Such multiplex analysis may also be performed<br>
using a single type of reporter dye (fluorophore), employing two or more oligonucleotides of the invention that exhibit different matched and mismatched Tms, such that multiple targets and alleles may be detected and identified simultaneously on the basis of melting temperature.<br>
The method of the invention may be carried out wherein the cligonucleotide which is able to hybridise to the target polynucleotide sequence, the sample which may contain the polynucleotide target sequence or both are immobilised on or within a support.<br>
The target polynucleotide sequence is preferably selected from the group consisting of the human genome including any gene therein or allele thereof, the mouse genome or any gene therein or allele thereof, and the genome of any infectious agent including any gene therein or allele thereof. Preferred target sequences are as described above.<br>
A further aspect of the invention provides a process for making an oligonucleotide of the invention, the process comprising selecting a target polynucleotide sequence and synthesising an oligonucleotide which has a sequence complementary to the target polynucleotide and which has the properties as set out in the first aspect of the invention. Typically, the oligonucleotide is one which is able to hybridise to the target sequence. The oligonucleotide may be one which is fully complementary to the target sequence or it may be one which is mismatched as described above.<br>
Preferably, the target polynucleotide sequence is selected from the group consisting of the human genome including any gene therein or allele thereof, the mouse genome including any gene therein or allele thereof, and the genome of any infectious agent including any gene therein or allele thereof as descried in more detail above.<br>
The oligonucleotides of the invention produced by this method may be placed on an array, thus the invention also includes a process of producing an array of oligonucleotides by placing two or more oligonucleotides of the invention on a solid support. Typically, the sites where the oligonucleotides have been placed (attached) are uniformly distributed on the support.<br>
All documents herein referred to are incorporated into the description.<br>
The invention will now be described in more detail by reference to the following non-limiting Examples and Figures.<br>
Figure 1 shows the chemical structures of the C6 FAM dU and Glen Fluorescein dT labels employed in the oligonucleotides of the invention.<br>
Figure 2 shows that a single-labelled oligonucleotide (HYBA1928C) was employed to directly analyse saliva samples with respect to the 5,10-methylenetetrafolate reductase (MTHFR) A1928C polymorphism. The negative derivative of fluorescence with respect to temperature (-dF/dT on y-axis) was plotted against sample temperature (x-axis) to generate melt peaks. The oligonucleotide yields melt peaks with Tms of approximately 51.5°C and 61°C in the presence of A and C alleles respectively. Heterozygous samples generate both 51.5°C and 6 PC melt peaks.<br>
Figure 3 shows melt peaks derived from single-labelled (FVG1; and dual-labelled (FVG11) factor V oligonucleotides. The additional fluorophore ca;;ser. melt peak height to increase by 4-5 fold and reduces the probe Tm by approximately 5°C.<br>
Figure 4. A) HYBCH2 probe melt peaks obtained with purified DNA from Chlamydia trachomatis (LGV-II, strain 434, ATCC VR-920B, Advanced Biotechnologies, Maryland, U.S.) and processed clinical swab samples. A single melt peak, with a Tm of approximately 60°C, is generated in the presence of Chlamydia target DNA. B) Real-time PCR and melt peak data obtained with the<br>
HYBCH6 Chlamydia probe. The triple-labelled Chlamydia probe yields melt peaks that are approximately 6 times the magnitude of the single-labelled HYBCH2 oligonucleotide.<br>
Figure 5 shows HYBNG (a dual-labelled oligonucleotide of the invention) melt peaks obtained with purified Neisseria gonorrhoeae DNA. A single melt peak is generated with a Tm of approximately 59°C.<br>
Figure 6. A) Melt peaks obtained from a series of purified human genomic DNAs, using the dual-labelled FVG11 probe to type samples with respect to the factor V Leiden polymorphism. Homozygous wild-type samples generate a single melt peak with a Tm of approximately 55°C. Heterozygous samples simultaneously yield matched and mismatched melt peaks possessing Tms of approximately 55°C and 45°C respectively. Homozygous 'mutant' samples would yield only the 45°C melt peaks. Negative control reactions generate neither melt peak. B) The LightCycler software may be employed to type samples with respect to the Factor V polymorphism based on the number and Tm of melt peaks.<br>
Figure 7 shows a method to calculate probe signal-to-noise ratio. Fluorescence readings are measured at Tm plus and minus 10°C for dissociated and hybridised probe states respectively. The signal-to-noise ratio may be calculated as the hybridised signal divided by the dissociated signal.<br>
Figure 8. A) The dual-labelled FVG11 probe exhibits approximately twice the amount of background fluorescence (above 70°C) compared with the single-labelled FVG1 probe. However, B) the dual-labelled factor V probe yields melt peaks that are approximately 5-6 times the size of the peaks generated with the single-labelled probe. This shows that the signal-to-noise ratio is ~3 times better with FVG11 compared to FVG1 compared to the expected result which is that twice the fluorophore would produce twice the background and twice the signal, therefore an unchanged signal-to-noise ratio. The FVG1 and FVG11 probes exhibit 33% and 82% increases in fluorescence upon hybridisation, respectively.<br>
Figure 9 shows a comparison of single (FVG1) and dual-labelled (FVG11) factor V probes using purified genomic samples. Melt peaks were analysed using 10°C to average smoothing of data and the polynomial with background correction LightCycler function. Negative control reactions (NC) do not produce melt peaks with either FVG1 or FVG11 probes. Ing/µl of homozygous 'wild-type' DNA yields single melt peaks with Tms of approximately 59°C and 54.6°C for single and dual-labelled probes respectively. Heterozygous (HET) genomic samples yield both matched and mismatched melt peaks. Xhe heights of dual-labelled melt peaks are approximately 3-4 times greater than those generated with the single-labelled probe.<br>
Figure 10 shows a comparison of 3', 5' and internally labelled probes of identical sequence. Xhe 5'-labelled probe exhibited fluorescence quenching upon target hybridisation and generated inverted peaks in -dF/dX traces. Xhe 3'-labelled probe displayed neither fluorescence enhancement nor quenching upon hybridisation. Xhe internally labelled oligonucleotide displayed fluorescence enhancement upon target hybridisation and positive melt peaks in -dF/dX traces. Both the single and dual-labelled probes are independent of the target sequence and always exhibit enhanced levels of fluorescence when hybridised.<br>
Figure 11. A) Comparison of HYBCH (1), HYBCH2 (2), HYBCH3 (3) and HYBCH4 (4) Chlamydia probes. Xhe probes labelled with 3 and 4 FAM dyes emit considerably lower levels of fluorescence compared with the single and dual-labelled probes due to the close proximity of the labels. Xhe triple-labelled probe (3) exhibits increased levels of emission upon duplex dissociation. B) Melt peaks derived from the four Chlamydia probes. Xhe triple-labelled probe yields small inverted melt peaks at a Tm higher than the dual-labelled variant Xhe probe labelled with 4 FAM dyes yields a small positive melt peak with a Xm approximately equal to that generated with the dual-labelled probe. C) Comparison of dual-labelled HYBCH (1), single-labelled HYBCH2 (2), and triple-labelled HYBCH5 (5) and HYBCH6 (6) oligonucleotides.<br>
Figure 12 shows melt peaks derived from single-labelled (1), dual-labelled (2) and triple-labelled (3) factor V oligonucleotide probes.<br>
Figure 13 shows single-labelled (1) and dual-labelled (2) The oligonucleotides of the invention containing A) FAM dA and B) FAM dC.<br>
Figure 14. A) Comparison of symmetric and asymmetric PCR protocols for the detection of Chlamydia trachomatis using the HYBCH2 probe. B) Comparison of symmetric PCR and single-labelled probe (1) with asymmetric amplification and single-labelled probe (2) and asymmetric PCR with the dual-labelled probe variant (3). The height of Chlamydia melt peaks is enhanced by more than 7-fold using asymmetric PCR and the dual-labelled HYBCH probe.<br>
Figure 15. 150nM of FVG11 was hybridised to 75nM of FVG and 75nM of FVA oligonucleotide targets (1). 300nM of the probe was hybridised to 150nM of FVG and 150nM of FVA (2). 150nM of probe was also hybridised to 500nM of FVG and 5OOnM of FVA. At high target concentration, preferential hybridisation of the probe to the fully complementary target sequence completely prevents detection of the mismatched target variant (3).<br>
Figure 16. A) Following 60 cycles of asymmetric target amplification, the heights of matched and mismatched factor V melt peaks are unbalanced when 150nM of FVG 11 probe is employed. Increasing probe concentration to 5OOnM improves the balance of melt peak heights. B) The heights of FVG11 melt peaks were measured after 50, 60 and 70 cycles of symmetric and asymmetric target amplification using 150nM, 300nM and 5OOnM of the FVG11 probe.<br>
Figure 17 shows multiplex detection of target sequences using PolyT (1), HYBINF (2), HYBCH2 (3) and HyBAdC (4) oligonucleotide probes. Multiplex detection was on the basis of melt peak Tm and employed FAM on all probes.<br>
Figure 18 shows melt peaks obtained with the triple-labelled HYBCH6 probe detecting matched and mismatched target sequences amplified, from purified Chlamydia DNA and mimic plasmid respectively. The probe displays Tms of approximately 60.3°C and 51.8°C with Chlamydial DNA and mimic targets respectively. No peaks are generated in the absence of target.<br>
Figure 19 shows the dual-labelled FVG11 probe was employed to generate a realtime standard curve using dilutions of human genomic DNA. The dilution series comprised lOOng/ul, lOng/µl, Ing/µl, O.lng/µl and O.Olng/µl of genomic DNA.<br>
Figure 20 shows melting curves for dual-labelled FVG11 derivative probes labelled with A. TAMRA, B. ROX, C. Cy5, D. ROX and FAM and inflections therein indicating changes in fluorescence as a result of hybridisation state.<br>
Materials and methods pertaining to the Examples<br>
Oligomtcleotide probe design and synthesis<br>
The oligonucleotide probes of the invention are typically designed to be approximately 18-25 nucleotides in length or possess Tms of approximately 55-60°C when hybridised to fully complementary target sequences. Fluorophores are attached to internal residues in the probe sequence using either C6 FAM dU (University of Southampton, UK) or Fluorescein dT (Glen Research, Sterling, VA). In the case of C6 FAM dU (Figure 1), 6-Carboxyfluorescein (FAM) is attached to the 5-position of uracil bases through methods of DNA synthesis which are well known to those in the field. Fluorophores may be incorporated into probes as phosphoramidites during solid-phase oligonucleotide synthesis (Brown et al, 2001, Brown et al, 2003). Other fluorescent modifications can also be made post synthesis using monomers such as 8-aminoalkyl-dA and 5-aminoalkyl-dC from Glen Research. The oligonucleotides of the invention may possess a 3'-phosphate component or other blocking agent to prevent Tag mediated extension when the probes are incorporated into real-time PCR assays.<br>
The quantity of probe obtained from the synthesis is determined by dissolving an aliquot of the oligonucleotide probe in a specific volume of water and measuring the UV absorbance at 260nm. The concentration of the probe is calculated from the UV absorbance of the oligonucleotide and its extinction coefficient at 260nm. The extinction coefficient of the oligonucleotide is calculated from the sum of the individual extinction coefficients of the unmodified and fluorescently labelled nucleosides of which it is composed.<br>
Polymorphic sites are conveniently positioned towards the centre of the oligonucleotides to maximise the ATm. The nucleotide at the site of polymorphism is typically selected such that highly destabilising interactions occur when probes are hybridised to mismatched targets. Probes exhibiting C/A, C/T and C/C mismatches are employed preferentially whilst G/T, G/A and G/G mismatches are avoided. As discussed above, the oligonucleotide probe may have 70% or 80% or 85% or 90% or 95% sequence identity to the complement of its target.<br>
Polymerase chain reaction<br>
PCR volumes are typically 20µl, comprising 2µl of sample, Ix PCR buffer, 0.5 µM primers, 1 unit Taq polymerase (Amersham Pharmacia Biotech), 3mM total MgCl2, 5ng/µl BSA (Roche Diagnostics), ImM dNTPs (Amersham Pharmacia Biotech) and 150nM of probe. The buffers employed in assays using the oligonucleotides are l0x PCR buffer (TaKaRa, contains 1.5mM MgC'l2), l0x HEPES#7 (l00mM HEPES pH8.3, 250mM KC1) for direct urine analysis and lOx HEPES#8 (00mM HEPES pH8.3, 500mM KC1) for direct saliva analysis and purified/processed samples. Homogeneous amplification and detection of targets may be performed using a LightCycler instrument (Roche Diagnostics). Realtime PCR detection and target quantification may be achieved with a 3-stage thermal protocol where, following an initial denaturation reaction step (95°C 1 min), targets are amplified using 50 cycles comprising denaturation (95°C 5s), primer annealing (55°C 10 s) and extension of products (72°C 10s). Fluorescence<br>
acquisition is performed once per cycle, in 3-stage assays, at the end of each primer-annealing step. Alternatively, targets may be rapidly amplified using a 2-stage thermal protocol where, following an initial denaturation step (95°C 1 min), targets are amplified using 50 cycles comprising denaturation (95°C 5s) and a combined annealing/extension phase (65°C 10s). Fluorescence is not acquired during 2-stage amplification.<br>
Target detection and identification<br>
Following 3-stage and 2-stage amplification, reactions are immediately denatured (95°C Os) and cooled (35°C 30s) prior to melt curve analysis (35-95°C with a 0.2°C/s transition rate) where fluorescence is acquired continuously. Melt peaks are constructed using the LightCycler software (version 3.5) by plotting the negative derivative of fluorescence with respect to temperature (-dF/dT on the y-axis) against temperature (x-axis). Targets may be reliably detected and identified using the melting temperatures (Tm) of the oligonucleotide probe peaks.<br>
Example 1: Exemplary single-labelled oligonucleotide probes<br>
The following are examples of probes that have been used for the detection of target sequences and identification of polymorphic targets. The primer sequences used to amplify corresponding targets are provided. This example contains single labelled oligonucleotide probes for comparison with the dual and multiple labelled probes of the invention.<br>
Analysis of the NAT2*5A (C481T) polymorphism may be performed by melt analysis using oligonucleotide probes such as 2303002 (5'GAGAGGAATCFGGTACCTGGACC (SEQ ID NO:1», where F is a fluorescent labelled nucleotide and underlined bases represent the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The target sequence containing the NAT2*5A SNP may be amplified using<br>
primers such as 195993 (5'CCTCTAGAATTAATTTCTGGG (SEQ ID NO:2)) and 195991 (5'CTGCTCTCTCCTGATTTGGTCC (SEQ ID NO:3)). The fully complementary *4 and mismatched (C:A) *5A alleles may be reliably detected and identified on the basis of probe Tm (table 1). The 2303002 probe is designed to hybridise to a section of the NAT2 gene (Genbank Accession X14672) having a sequence of either:<br>
NAT2*4         5'GGTCCAGGTACCAGATTCCTCTC (SEQ ID NO:4) NAT2*5A      5'GGTCCAAGTACCAGATTCCTCTC (SEQ ID NO:5)<br>
Analysis of the NAT2*5C (A803G) polymorphism may be performed by melt analysis using an oligonucleotide probe such as DdeFLl*4 (5'GAAGTGCFGAAAAATATATTTAAG (SEQ ID NO:6)), where F is a fluorescent labelled nucleotide and underlined bases represent the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The target sequence containing the NAT2*5C SNP may L'e amplified using primers such as DdeF2 (5'CCTATAGAAAATTCAATTATAAAG (SEQ ID NO:7)) and DdeR (5'CACGAGATTTCTCCCCAAGG (SEQ ID NO:8)). The fully complementary *4 and mismatched (A:C) *5C alleles. may be reliably detected and identified on the basis of probe Tm (table 1). The DdeFLl*4 probe is designed to hybridise to a section of the NAT2 gene (Genbank Accession XI4672) having a sequence of either:<br>
NAT2*4   5 'CTTAAATATATTTTTCAGCACTTC (SEQ ID NO:9) NAT2*5C  5'CTTAAATATATTTCTCAGCACTTC (SEQ ID NO: 10)<br>
Analysis of the NAT2*7 (G857A) polymorphism may be performed by melt analysis using an oligonucleotide probe such as BamFLl*7 (5'CCTGGTGAFGAATCCCTTAC (SEQ ID NO: 11)), where F is a fluorescent labelled nucleotide and underlined bases represent the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The target sequence containing the NAT2*7 SNP may be amplified using primers such as BamF2<br>
(5'CCTATAGAAAATTCAATTATAAAG (SEQ ID NO: 12)) and BamR (5'CACGAGATTTCTCCCCAAGG (SEQ ID NO: 13)). The fully complementary *7 and mismatched (A:C) *4 alleles may be reliably detected and identified on the basis of probe Tm (see table 1). The DdeFLl*7 probe is designed to hybridise to a section of the NAT2 gene (Genbank Accession XI4672) having a sequence of either:<br>
NAT2*4   5'GTAAGGGATTCATCACCAGG (SEQ ID NO: 14) NAT2*7   5'GTAAGGGATCCATCACCAGG (SEQ ID NO: 15)<br>
Analysis of two NAT1*10 SNPs may be performed by melt analysis using an<br>
oligonucleotide	probe	such	as	HYBNAT3S<br>
(5'CTTTAAAATACAFTTTTTATTATTA (SEQ ID NO: 16)), where F is a fluorescent labelled nucleotide and underlined nucleotides represent positions of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. Fully complementary, mismatched and double mismatched alleles may be reliably detected and identified on the basis of probe Tm (see table 1). Functionality of this probe has been demonstrated using complementary oligonucleotides possessing all possible base combinations at the positions of polymorphism. The HYBNAT3S probe is designed to hybridise to a section of the NAT! gene (Genbank Accession AY376850) having a sequence of either:<br>
(Sequence Removed)<br>
 (5'CTGTAFTCCTCGCCTGTCC (SEQ ID NO:21)), where F is a fluorescent labelled nucleotide and the underlined nucleotide is the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The target sequence containing the factor V SNP may be amplified using primers such as FVF1.3 (5'GGACTACTTCTAATCTGTAAGAGCAGATC (SEQ ID NO:22)) and FVR3.5 (5'GCCCCATTATTTAGCCAGGAGACCTAACATG (SEQ ID NO:23)). The fully complementary 'wild-type' and mismatched (C:A) 'mutant' alleles may be reliably detected and identified on the basis of probe Tm (see table 1). The FVG1 probe is designed to hybridise to a section of the Factor V gene (Genbank Accession AY364535) having a sequence of either:<br>
Simultaneous analysis of the sickle cell anaemia HbS and HbC polymorphisms may be performed by melt analysis using a single labelled oligonucleotide probe such as SCT1 (5'GTGCACCTGACFCCTGTGG (SEQ ID NO:26)). where F is a fluorescent labelled nucleotide and underlined nucleotides are the positions of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The target sequence containing the sickle cell polymorphisms may be amplified using primers such as SCF1 (5'AGGGCAGAGCCATCTATTGCT (SEQ ID NO:27)) and SCR2 (5'CATCCACGTTCACCTTGCC (SEQ ID NO:28)). The fully complementary (GT:CA) HbS, mismatched (GT:CT) wild-type and double mismatched (GT:TT) HbC alleles may be reliably detected and identified on the basis of probe Tm (table 1). The SCT1 probe is designed to hybridise to a section of the p-globin gene (Genbank Accession AY356351) having a sequence of either:<br>
be performed by melt analysis using an oligonucleotide probe such as HYBCH2 (5'CAAGCCTGCAAAFGTATACCAAG (SEQ ID NO:32)), where F is a fluorescent labelled nucleotide. PCR extension from the probe is blocked with a 3' phosphate. The cryptic plasmid target sequence may be amplified using primers such as CHF3-1 (5'GGGTTCGTTGTAGAGCCATGTCCTATCTTG (SEQ ID NO:33)) and CHR4-1 (5'CGCAGCTGCTGTAATCACCCAGTCGATAAA (SEQ ID NO:34)). The Chlamydia cryptic plasmid and a mismatched (C:A) positive amplification control (see below) may be reliably detected and identified on the basis of probe Tm (see table 1). The HYBCH2 probe is designed to hybridise to a section of the cryptic plasmid (Genbank Accession X07547) having a sequence of either:<br>
Further assays employing single-labelled oligonucleotide probes to analyse targets such as CYP2D6*3, CYP2D6*4, CYP2C19ml, CYP2C19m2, CYP2C9*2 and CYP2C9*3 SNPs are described in WO 01/73118 A2.<br>
TABLE 1 - Oligonucleotide probe Tms, at 3mM MgCl2, with fully complementary and mismatched targets. Melt peaks and Tms were obtained from complementary oligonucleotides and PCR amplified target sequences. Sequence identification numbers for probes and targets are included.<br>
(Sequence Removed)<br>
Example 2: Exemplary dual-labelled oligonucleotide probes of the invention<br>
Analysis of the Factor V Leiden (G1691 A) polymorphism may be performed by melt analysis using an oligonucleotide probe such as FVG11 (5'CTGTAFTCCTCGCCFGTCC (SEQ ID NO:60)), where F are fluorescent labelled nucleotides and the underlined nucleotide is the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The target sequence containing the factor V SNP may be amplified using primers such as FVF1.3 (5'GGACTACTTCTAATCTGTAAGAGCAGATC (SEQ ID NO:22)) and FVR3.5 (5'GCCCCATTATTTAGCCAGGAGACCTAACATG (SEQ ID NO:23)). The fully complementary 'wild-type' and mismatched (C:A) 'mutant' alleles may be reliably detected and identified on the basis of probe Tm (see table 1). The FVG1 probe is designed to hybridise to a section of the factor V gene (Genbank Accession AY364535) having a sequence of either:<br>
Detection of the Chlamydia trachomatis cryptic plasmid may be performed by melt analysis using an oligonucleotide probe such as HYBCH (5'CAAGCCFGCAAAFGTATACCAAG (SEQ ID NO:61)), where F are fluorescent labelled nucleotides. PCR extension from the probe is blocked with a 3' phosphate. The cryptic plasmid target sequence may be amplified using primers such as CHF3-1 (5'GGGTTCGTTGTAGAGCCATGTCCTATCTTG (SEQ ID NO:33)) and CHR4-1 (5'CGCAGCTGCTGTAATCACCCAGTCGATAAA (SEQ<br>
ID NO:34)). The Chlamydia cryptic plasmid and a mismatched (C:A) positive amplification control may be reliably detected and identified on the basis of probe Tm (see table 1). The HYBCH2 probe is designed to hybridise to a section of the cryptic plasmid (Genbank Accession X07547) having a sequence of either:<br>
HYBCHRC 5 'CTTGGTATACATTTGCAGGCTTG (SEQ ID NO:35) Mimic     5 'CTTGGTATACATTTACAGGCTTG (SEQ ID NO:36)<br>
Detection of the Neisseria gonorrhoeae cryptic plasmid may be performed by melt analysis using an oligonucleotide probe such as HYBNG (5TCTGCFTCCGCFACGGCTTC (SEQ ID NO:58)), where F are fluorescent labelled nucleotides. PCR extension from the probe is blocked with a 3' phosphate. The cryptic plasmid target sequence may be amplified using primers such as NGF1 (5'ACTTTGGCGATATTGCTCGG (SEQ ID NO:74)) and NGR1 (5TACCGAGAACGAACGCGACA (SEQ IDNO:75)). The gonorrhoeae cryptic plasmid may be reliably detected and identified on the basis of probe Tm (table 1). The HYBNG probe is designed to hybridise to a section of the cryptic plasmid (Genbank Accession Ml0316) having a sequence of:<br>
HYBNGRC    5'GAAGCCGTAGCGGAAGCAGA (SEQ ID NO:59)<br>
Detection of the hexon gene in Adenovirus C and Adenovirus D strains may be<br>
performed by melt analysis using The oligonucleotides of the invention such as<br>
HyBAdC (5'GACGTGGFCCGTGFGCACCAGCCT (SEQ ID NO:68)) and<br>
HyBAdD (5'GACGTGGFCAGAGFGCACCAGCCT (SEQ ID NO:71)), where F<br>
are fluorescent labelled nucleotides. PCR extension from these probes is blocked<br>
with 3' phosphates. The hexon target sequence may be amplified using primers<br>
such as ADRJC1 (5'GACATGACTTTCGAGGTCGATCCCATGGA (SEQ ID<br>
NO:76))	(Elnifro	et	al,	2000)	and	PB00432<br>
(5'GCCGAGAAGGGCGTGCGCAGGTA (SEQ ID NO:77)) obtained from the VirOligo database. The Adenovirus strains may be reliably detected and identified on the basis of probe Tm (see table 1). The HyBAdC probe is fully<br>
complementary to Adenovirus C sequence and exhibits both C:T and T:T mismatches when hybridised to Adenovirus D sequence. The HyBAdD probe is fully complementary to Adenovirus D sequence and exhibits both A:G and A:A mismatches when hybridised to Adenovirus C sequence. Larger variations in probe melting temperature are obtained with the HyBAdC probe (table 1). The Adenovirus probes are designed to hybridise to a section of the hexon gene (Genbank Accession AJ293905) having a sequence of either:<br>
HyBAdCRC   5'AGGCTGGTGCACACGGACCACGTC (SEQ ID NO:69) HyBAdDRC   5 'AGGCTGGTGCACTCTGACCACGTC (SEQ ID NO:70)<br>
Detection of the Influenza A matrix gene may be performed by melt analysis using an oligonucleotide probe such as HYBINF (5'GGGAFCCAAAFAACATGGACAGAGCT (SEQ ID NO:66)), where F are fluorescent labelled nucleotides. PCR extension from the probe is blocked with a 3' phosphate. The matrix target sequence may be amplified using primers such as INFA-1 (5'GGACTGCAGCGTAGACGCTT (SEQ ID NO:78)) and FLU-4 (5'ATTTCTTTGGCCCCATGGAATGT (SEQ ID NO:79)). Since Influenza is an RNA virus, a reverse transcription (RT) step is required prior to target amplification. cDNA production, PCR amplification and target detection may all be performed in a single reaction vessel using a Roche one-step LightCycler RT-PCR kit. Influenza A may be reliably detected and identified on the basis of probe Tm (table 1). The HYBINF probe is designed to hybridise to a section of the matrix gene (Genbank Accession AY 13 0766) having a sequence of:<br>
INFRC	5 'AGCTCTGTCCATGTTATTTGGATCCC (SEQ ID NO:67)<br>
Detection of the Streptococcus pneumoniae pneumolysin gene may be performed by melt analysis using an oligonucleotide probe such as SP1 (5'GGGGTCTFCCACTFGGAGAAAGCTATC (SEQ ID NO:64)), where F are fluorescent labelled nucleotides. A single-labelled version of the Streptococcus probe was also investigated (5'GGGGTCTFCCACTTGGAGAAAGCTATC<br>
(SEQ ID NO:80)). PCR extension from the probe is blocked with a 3' phosphate. The pneumolysin target sequence may be amplified using primers such as SPf 2 (5'CTTGCGGTTGATCGTGCTCCGATGAC (SEQ ID NO:81)) and SPR2 (5'CATTATTGACCTGACCATAATCTTGATGCC (SEQ ID NO:82)). Streptococcus pneumoniae may be reliably detected and identified on the basis of probe Tm (table 1). The SP1 probe is designed to hybridise to a section of the pneumolysin gene (Genbank Accession X52474) having a sequence of:<br>
SP1RC	5'GATAGCTTTCTCCAAGTGGAAGACCCC (SEQ ID NO:65)<br>
Detection of the Factor II (G20210A) SNP may be performed by melt analysis using an oligonucleotide probe such as HYBFII (5'GCATFGAGGCTCGCFGAGAGTC (SEQ ID NO: 115)), where F are fluorescent labelled nucleotides and the underlined nucleotide is the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The Factor II target sequence may be amplified using primers such as FIIF2 (5'CTGGGCTCCTGGAACCAATC (SEQ ID NO:118)) and FIIR1 (5'GCTGCCCATGAATAGCACTGG (SEQ ID NO: 119)). The fully complementary and mismatched (C:A) alleles may be reliably detected and identified on the basis of probe Tm (see table 1). The HYBFII probe is designed to hybridise to a section of the factor II gene (Genbank Accession AF493953) having a sequence of either:<br>
Detection of the MTHFR (A1928C) SNP may be performed by melt analysis using an oligonucleotide probe such as HYBA1928C (5'GACCAGFGAAGCAAGFGTCTTTG (SEQ ID NO:112)), where F are fluorescent labelled nucleotides and the underlined nucleotide is the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The A1928C target sequence may be amplified using primers such as 1928F<br>
(5'CCCAAGGAGGAGCTGCTGAA (SEQ ID NO: 120)) and 1928R (5' CCATTCCGGTTTGGTTCTCC (SEQ ID NO: 121)). The fully complementary and mismatched (C:T) alleles may be reliably detected and identified on the basis of probe Tm (see table 1). The HYBA1928C probe is designed to hybridise to a section of the 5,10-methylenetetrahydrofolate reductase gene (Genbank Accession NM_005957) having a sequence of either:<br>
Detection of the MTHFR (C677T) SNP may be performed by melt analysis using<br>
an	oligonucleotide	probe	such	as	HYBMTH<br>
(5'GTCFGCGGGAGCCGAFTTCATC (SEQ ID NO:112)), where F are fluorescent labelled nucleotides and the underlined nucleotide is the position of polymorphism. PCR extension from the probe is blocked with a 3' phosphate. The C677T target sequence may be amplified using primers such as MTHF2 (5'CTGACCTGAAGCACTTGAAGGAG (SEQ ID NO:122)) and MTHR2 (5'GCGGAAGAATGTGTCAGCCTCAAAG (SEQ ID NO: 123)). The fully complementary and mismatched (C:A) alleles may be reliably detected and identified on the basis of probe Tm (see table 1). The HYBMTH probe is designed to hybridise to a section of the 5,10-methylenetetrahvdiofolate reductase gene (Genbank Accession NM_005957) having a sequence of either:<br>
Example 3: Direct analysis of saliva samples<br>
A series of saliva samples were analysed with the MTHFR (A1928C) primers and probe, as described above, without DNA purification. Mouthwashes, which comprise approximately 50% saliva in water, were included d.:rectly in the assays.<br>
Buccal swabs expressed in water may also analysed directly. The HYBA1928C probe generates single melt peaks with Tms of approximately 51.5°C and 61°C with samples homozygous for the A and C alleles respectively (Figure 2). Saliva samples that are heterozygous for the A and C alleles are clearly identified by the generation of both 51.5°C and 61°C peaks. Detection and identification of polynucleotide targets have also been achieved directly from blood without the requirement for DNA purification.<br>
Example 4: Rapid target detection using dual and triple-labelled probes<br>
Chlamydia cryptic plasmid targets may be amplified using primers CHF3-1 (SEQ ID NO:33) and CHR4-1 (SEQ ID NO:34). Assays employ 3-stage LightCycler protocols consisting of an initial denaturation (95°C 1 min) followed by 50 cycles of PCR comprising denaturation (95°C 5s), primer annealing (55°C 10s) and extension of products (72°C 10s). Amplified target is detected and characterised through the inclusion of HYBCH (SEQ ID NO:61) or HYBCH6 probes (SEQ ID NO: 104) and melt curve analysis comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat towards 95°C using a 0.2°C/s transition rate. In the presence of Chlamydia DNA, the HYBCH and HYBCH6 probes generate clear melt peaks with Tms of approximately 56°C and 60.5°C respectively. The assays using the oligonucleotides of the invention have detected the presence of Chlamydia in purified DNAs, neat urines, processed (Becton Dickinson, Oxford, U.K.) urines &amp; swabs and liquid samples pipetted directly from swabs (Figure 4).<br>
Gonorrhoeae plasmid targets are amplified using primers NGF1 (SEQ ID NO:74) and NGR1 (SEQ ID NO:75). Assays employ 3-stage LightCycler protocols consisting of an initial denaturation (95°C 1 min) followed by 50 cycles of PCR comprising denaturation (95°C 5s) primer annealing (55°C 10s) and extension of products (72°C 10s). Amplified target is detected and characterised through the inclusion of HYBNG probe (SEQ ID NO:58) and melt curve analysis comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat towards 95°C using a<br>
0.2°C/s transition rate. In the presence of Gonorrhoeae DNA, the HYBNG probe generates clear melt peaks with Tms of approximately 59°C (Figure 5).<br>
Example 5: Comparison of single-labelled and dual-labelled probes<br>
Factor V targets are amplified directly from saliva samples, without DNA purification, using primers FVF1.3 (SEQ ID NO:22) and FVR3.5 (SEQ ID NO:23). Assays employ 2-stage LightCycler protocols consisting of an initial denaturation (95°C 1 min) followed by 50 cycles of PCR comprising denaturation (95°C Is) and combined annealing/extension (65°C 5s) phases. Amplified target is detected and characterised through the inclusion of FVG1 (SEQ ID NO:21) or FVGll (SEQ ID NO:60) oligonucleotide probes and melt curve analysis comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat towards 95°C using a 0.2°C/s transition rate. Factor V assays may be completed in as little as 16 minutes. Homozygous wild type samples yield single melt peaks possessing Tms of approximately 59°C and 55°C when FVG1 and FVGll probes are employed respectively. Homozygous mutant samples generate single melt peaks possessing Tms of approximately 49.5°C and 45°C when FVG1 and FVGll probes are employed respectively. Heterozygous Factor V samples yield both 59°C and 49.5°C melt peaks when the single-labelled FVG1 probe is used and generate both 55°C and 45°C peaks when the dual-labelled FVGll probe is employed (Figure 6). The assays function efficiently with purified DNAs, swab samples and buccal washes. Over 450 genomic samples have been reliably typed, with respect to the Factor V polymorphism, using the FVG1 and FVG11 probes.<br>
The dual-labelled probe does exhibit higher levels of background signal, compared with the single-labelled probe, when single-stranded. However, the dual-labelled FVGll probe yields melt peaks that are approximately four times the size of those generated with the single-labelled FVG1 probe which is significantly greater than would be expected if the affect of the additional fluorophore was simply additive. The additional fluorophore in FVGll reduces the Tm of the probe by approximately 4.5°C compared with the single-labelled<br>
FVG1 probe (Figure 3). A similar reduction in Tm is observed with the single-labelled HYBCH2 (SEQ ID NO:32) and dual-labelled HYBCH (SEQ ID NO:61) Chlamydia probes (Figure 11).<br>
Example 6: Probe signal-to-noise<br>
The amount of fluorescence emission from The oligonucleotides of the invention was measured in single-stranded and double-stranded states to investigate the affect of probe sequence, fluorophore placement and dual/triple-labelling. 150nM of probe was typically hybridised to 150nM of fully complementary oligonucleotide, in TaKaRa buffer, to determine the signal-to-noise ratio of The oligonucleotides of the invention. Fluorescence measurements were made using a LightCycler melting curve protocol comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat to 95°C using a 0.2°C/s transition rate. Fluorescence values were obtained from melt curves at temperatures of probe Tm plus and minus 10°C for dissociated and hybridised states respectively (Figure 7). Fluorescence measurements were made over this defined temperature range in an attempt to remove some of the affects that temperature has on the amount of emission. The hybridised and dissociated fluorescence values provide a measure of signal-to-noise ratio and an indication of the affect of probe hybridisation on emission (table 2), where signal-to noise ratio is calculated as hybridised signal divided by dissociated signal. Probe signal-to-noise ratios have been found to be highly reproducible and exhibit low run-to-run and instrument-to-instrument variation.<br>
The single-labelled FVG1 probe (SEQ ID NO:21) possesses a Tm of approximately 59°C when hybridised to fully complementary target sequence (SEQ ID NO:24) and exhibits fluorescence levels of 8 and 6 (arbitrary LightCycler fluorescence units) at 49°C and 69°C respectively. The dual-labelled FVG11 probe (SEQ ID NO:60)) possesses a Tm of approximately 55°C when hybridised to the fully complementary target sequence and exhibits fluorescence levels of 21.8 and 12 at 45°C and 65°C respectively. The signal-to-noise ratios of<br>
single and dual-labelled factor V probes are calculated as 1.33 and 1.82 respectively.<br>
Dual-labelled probes may exhibit higher levels of background fluorescence noise compared with equivalent single-labelled probes. However, despite the elevated background noise, dual-labelled probes consistently yield considerably greater signal-to-noise ratios and melt peak heights. With oligonucleotide targets, single-labelled probes have yielded emission increases between 20% and 92%. Whereas, dual-labelled probes have demonstrated fluorescence increases between 71% and 199%. Hybridisation of 150nM FVG11 probe to the complementary oligonucleotide FVG (SEQ ID NO:24) results in approximately two times more background fluorescence (at 90°C) than hybridisation of 150nM FVG1 probe (Figure 8). However, the background correction function of the LightCycler software demonstrates that the dual-labelled peak is approximately five times greater in height than the equivalent single-labelled probe peak with oligonucleotide targets (Figures 3 &amp; 8). Background corrected data derived from heterozygous Factor V carrier DNA samples demonstrates that the dual-labelled probe yields two melt peaks that are each approximately three times the height of peaks generated with single-labelled probe in target amplification assays (Figure 9).<br>
Fluorophores flanked by guanine nucleotides<br>
Given that guanine bases are efficient quenchers of fluorescence, fluorophores flanked by G's may be expected to be efficiently quenched when probes are in the dissociated state and fluorescence emission may be expected to increase considerably upon probe hybridisation. All probes containing internal fluorophores adjacent to G's (5', 3' or both positions) have demonstrated signal enhancement upon target hybridisation (table 2). The C9*2C (SEQ ID NO:55), 136A (SEQ ID NO:83) and HYBA1928C (SEQ ID NO:112) oligonucleotide probes are good examples of probes with fluorescent labels positioned between two G nucleotides. The signal-to-noise ratio of C9*2C is comparable to the<br>
2D64C* probe (SEQ ID NO:51), whose fluorophore is adjacent to only one G but is very G rich in general. The A/T rich DdeFLl*4 probe (SEQ ID NO:6) also exhibits a similar signal-to-noise ratio and possesses a single G nucleotide adjacent to the fluorophore. The placement of fluorophores adjacent to G residues are more likely, although not necessary, to generate large peak heights and signal-to-noise ratios. The absence of G residues at suitable locations within the probe is not problematic as probes yield efficient hybridisation signals when fluorophores are placed in all sequence environments. Although this example shows single-labelled oligonucleotides, as well as oligonucleotides of the invention, it nevertheless demonstrates that it is the independence of target sequence, and not an interaction between fluorophores, that causes the sequence independent enhancement of fluorescence upon target hybridisation.<br>
Fluorophores flanked by cytosine nucleotides<br>
If guanine residues in target sequences have the ability to cause fluorescence quenching, emission from fluorophores flanked by C residues may be expected to decrease upon target hybridisation. However, all oligonucleotide probes with fluorophores adjacent to C residues, such as HYBNG (SEQ ID NO:58), 171R (SEQ ID NO:85), C 19mlA (SEQ ID NO:39), SCT1 (SEQ ID NO:26) and SP2 (SEQ ID NO:80), have demonstrated fluorescence enhancement upon probe hybridisation. Even highly C rich oligonucleotide probes, such as HEPB (SEQ ID NO:45) and FVG2 (5'GTATFCCTCGCCTGTCCAG (SEQ ID NO:87)), emit higher levels of fluorescence when hybridised to targets than when single-stranded. Fluorescence enhancement upon probe hybridisation appears to be independent of the presence of guanine nucleotides in the target sequence, as predicted by molecular modelling, which indicates that fluorophores do not interact with duplex DNA.<br>
50 Fluorophores flanked by adenine and thymine nucleotides<br>
Oligonucleotide probes with fluorophores placed between A and.T residues also exhibit enhancement of fluorescence emission upon target hybridisation. The NAT2*6 (SEQ ID NO:88), HYBNAT3S (SEQ ID NO: 16) and FVG1 (SEQ ID NO:21) probes are good examples of probes whose fluorophore-labelled bases are neighboured by A and T nucleotides. An artificially constructed PolyT probe (5TTTTTTTTTTTFTTTTTTTTTTT (SEQ ID NO:37)) displays increased levels of fluorescence emission when hybridised to target sequences, compared with the single-stranded state, despite the complete absence of G nucleotides in both probe and target strands. These oligonucleotides contain a single fluorophore and so are not oligonucleotides of the invention. Nevertheless, they demonstrate the sequence independence of the fluorescent nature of the probes.<br>
Signal strength and sequence context of fluorophore attachment<br>
The data in table 2 demonstrates that probe functionality is not reliant on the presence of guanine residues within the probe or target sequences. There is, however, a possible association between the higher signal-to-noise ratios and the presence of Gs at probe positions immediately neighbouring the fluorophore. For single-labelled probes, many of the large signal-to-noise ratios are derived from probes with fluorophores flanked by guanine bases. However, the largest single-labelled signal-to-noise ratio was obtained from an oligonucleotide possessing a fluorophore flanked by A and C nucleotides (SEQ ID NO:85).<br>
Probes with fluorophores flanked by A, C and T bases have yielded larger signals than several probes with G nucleotides directly adjacent to the fluorophore-labelled base. Furthermore, all single, dual, and multiple-labelled oligonucleotide probes have exhibited enhanced levels of fluorescence emission upon target hybridisation irrespective of the sequence flanking the internal fluorophores. Therefore, whilst larger melt peaks may be obtained by placing fluorophore-labelled nucleotides adjacent to guanine residues, high quality melt peaks may be<br>
obtained without the requirement for G residues at specific sites in the probe sequence. Although these oligonucleotides contain a single fluorophore, they demonstrate the sequence independence and also will assist in optimal spacing in certain sequence contexts.<br>
Independence of target sequence<br>
LightCycler melt curve analysis strongly suggests that the sequence of the target has little or no affect on the functionality of the oligonucleotide probes. If the fluorophores of the oligonucleotide probes project into solution upon hybridisation, such that they do not interact with duplex DNA, fluorophores may be positioned within highly C rich regions of the probe without being quenched by the G nucleotides of the target. All single and dual-labelled oligonucleotide probes designed to date exhibit enhanced levels of fluorescence upon hybridisation relative to the single-stranded state irrespective of probe sequence and the nucleotides flanking the fluorophore-labelled bases.<br>
TABLE 2 - Signal-to-noise ratios obtained from hybridising 150nM oligonucleotide probes to fully complementary oligonucleotides. Sequence identification numbers for probes and targets are provided. Signal-to-noise ratios are ranked from lowest to highest. Melt curves were all generated using a single LightCycler instrument (LC2).<br><br>
(Table Removed).<br><br>
(Formula Removed)<br>
Measurement of UV-visible absorption and fluorescence spectra (Marks et al (2005)) demonstrates that there is a blue shift of approximately 4nm and an increase in intensity on hybridisation of the oligonucleotide probes to complementary target sequences. The fluorescence properties of free FAMCAP dye are more like those of the probe/target duplex. Incorporation of FAMCAP into<br><br>
DNA probe constructs causes a red shift in the max of excitation and emission. The red shift suggests a possible interaction of the fluorophore with ssDNA, which may include π-π stacking interactions between the dye and the DNA bases . The results of Marks et al, also suggest that fluorophores do not interact with the double-stranded DNA by either intercalation or groove binding since these interactions would cause a strong red shift upon hybridisation. Therefore, there may be an interaction between the fluorophore and the bases of the single-stranded probe DNA that causes fluorescence quenching. This interaction between fluorophore and probe DNA is removed upon target hybridisation such that the dye projects into solution and fluorescence is enhanced. This mechanism of fluorescence enhancement is not expected to operate with terminally labelled probes since base stacking can occur on only one side of fluorophore. The data indicates that oligonucleotide probe fluorescence increases are caused by the removal of quenching rather than the enhancement of fluorescence. This model of the probe functionality also supports our proposition that the probe signals are independent of the target sequence, such that fluorophores may be positioned even within C rich regions and still exhibit enhanced levels of fluorescence upon target hybridisation.<br>
Example 7: Peak heights<br>
The peak heights of single, dual and triple-labelled probes were also measured to investigate the affect of probe sequence, fluorophore placement and multiple fluorophores. Peak height measurements are slightly less reliable than signal-to-noise ratios for probe comparisons due to run-to-run and instrument-to-instrument variations. However, melt peaks generated in a single LightCycler experiment may be employed to reliably compare probe sequences. 150nM of probe was typically hybridised to 150nM of fully complementary oligonucleotide in TaKaRa buffer. Fluorescence measurements were made using a LightCycler melting curve protocol comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat to 95°C using a 0.2°C/s transition rate. Melt peak data obtained only with LightCycler software version 3.5 have been included in table 3. Version 3.01 of<br>
the LightCycler software required the user to set the fluorescence gain manually. Whereas software version 3.5 automatically determines fluorescence levels relative to the sample that exhibits the highest level of emission. All The oligonucleotides of the invention exhibit positive melt peaks when -dF/dT is plotted on the y-axis against temperature on the x-axis, irrespective of the probe and target sequences and the position of G's within them. The placement of single fluorophores, in the Factor V probing sequence, between A and T bases (FVG1) and between C and G bases (FVG1ALT) yields melt peaks of approximately equal height. The dual-labelled Factor V probe (FVG11) yields melt peaks that are approximately four times the height of those generated with the two single-labelled probes FVG1 and FVG1ALT (table 3). The sum of the single-labelled peak heights is less than that exhibited by the dual-labelled probe. Furthermore, employing 75nM or 150nM of both single-labelled probes in a single reaction vessel also results in melt peaks that are smaller than those generated by the dual-labelled oligonucleotide probes. An interaction between the fluorophore-labelled bases may be responsible for the enhanced signal-to-noise ratio and melt peak heights, making the dual-labelled probe greater than the sum of the two single-labelled probes. Further enhancement of signal is derived from triple-labelled probes (see below). The location and degree of separation of fluorophores within probes may affect the magnitude of fluorescence change upon target hybridisation and dissociation. Single-labelled probes possess reduced Tms compared with unlabelled oligonucleotides and dual-labelled probes consistently exhibit reduced Tms compared with equivalent single-labelled probes (table 1). Since the fluorophores are believed to project into solution upon target hybridisation, thereby not interacting with the duplex DNA, it is thought that fluorophores stabilise the single-stranded probe species rather than destabilise the duplex. Multiple-labelled probes are thought to stabilise single-stranded probe structures more than equivalent single-labelled oligonucleotide probes. Dual and multiple-labelled probes frequently exhibit lower levels of fluorescence noise than expected (table 5) due to the stabilised and efficiently quenched single-stranded structures. The reduced backgrounds and increased signals of dual and multiple-labelled<br>
probes give rise to larger peak heights and signal-to-noise ratios compared with equivalent single-labelled oligonucleotides.<br>
Example 8: Comparison of internal and terminally labelled probes<br>
Factor V target sequences were amplified using primers FVF1 (5'GACTACTTCTAATCTGTAAGAGCAG SEQ ID NO:92)) and FVR3 (5'CCATTATTTAGCCAGGAGAC (SEQ ID NO:93)). Assays employed 3-stage LightCycler protocols consisting of an initial denaturation (95°C 1 min) followed by 50 cycles of PCR comprising denaturation (95°C Os), primer annealing (55°C 10s) and extension of products (72°C 10s). Amplified target was detected and characterised through the inclusion of FV3 (5'CTGTATTCCTCGCCTGTCCF (SEQ ID N0:94)) or FV5 (5'FCTGTATTCCTCGCCTGTCC (SEQ ID NO:95)) terminally labelled probes and melt curve analysis comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat towards 95°C using a 0.1°C/s transition rate. The 3'-labelled FV3 probe was completely non-functional, yielding no peaks in melt curve analyses. Measuring probe emission spectra verified efficient oligonucleotide synthesis and fluorophore attachment. The FV5 probe, possessing a 5'-FAM attachment, exhibited a reduction in fluorescence emission upon target hybridisation. The FV5 probe generated inverted melt peaks in -dF/dT plots (Figure 10) that allowed identification of homozygous and heterozygous samples. The inverted melt peaks derived from the 5'-labelled probe were of similar magnitude to the heights of peaks obtained from the single-labelled FVG1 Factor<br>
 (Sequence Removed)<br>
The 5'-labelled probe places the fluorophore in close proximity to G residues, at positions 0 and +1, upon hybridisation to the target sequence, explaining the<br>
observed reduction in fluorescence emission. The 3'-labelled probe also places the fluorophore in proximity with G residues, at positions 0 and -1, but fluorescence was neither enhanced nor quenched upon hybridisation.<br>
TABLE 3 - Melt peak heights obtained from hybridising 150nM oligonucleotide probes to fully complementary oligonucleotides. Sequence identification numbers for probes and targets are provided. Peak heights are ranked from lowest to highest. All peaks were positive when the negative derivative of fluorescence with respect to temperature (-dF/dT) was plotted against temperature on the y and x axis respectively. All melt peaks were derived using a single LightCycler (LC2) to avoid instrument-to-instrument variability.<br><br>
(Table Removed)<br>
Fluorophores attached to internal residues of The oligonucleotides of the invention are quenched in the single-stranded state and hybridisation relieves a proportion of this quenching. Quenching in the single-stranded probe may arise as the DNA bases can stack on the fluorescent group when they are not forming base pairs. In the single stranded form the fluorophore will be sandwiched between two DNA bases forming a fairly stable hydrophobic structure. This will break down on duplex formation as the bases have to participate in base pairing. The phenomenon may not merely be "guanine quenching", although guanines would possibly have a greater influence than other bases.<br>
The situation with terminal labels is different, as base stacking can only occur on one side of the fluorophore. In the single strand (and in the duplex) the fluorophore will not be sandwiched between DNA bases, but just stacked on a base. Quenching upon hybridisation may occur if the fluorophore is brought into close proximity to guanine bases in the target strand. Conversely, fluorescence enhancement may occur on hybridisation if fluorophores are attached to terminal G residues of the probe. The internal labels of the oligonucleotide probes are affected considerably less by the probe and target sequences and always exhibit fluorescence increases upon hybridisation regardless of the position of guanines in the duplex. The terminal labelled probes are influenced by the oligonucleotides in the target sequence and can exhibit increased or decreased levels of fluorescence<br>
upon target hybridisation. The dual-labelled oligonucleotides of the invention exhibit increased levels of fluorescence on hybridisation due to independence of target sequence and not an interaction between fluorophores.<br>
Example 9: Signal strength andfluorophore spacing in multi-labelled probes<br>
Incorporating additional fluorophores into oligonucleotide probe sequences may be expected to increase probe emission in both hybridised (signal) and single-stranded (noise) states, thereby maintaining the signal-to-noise ratio. Alternatively, since FAM dyes can actually quench each other when in proximity, labelling probes with multiple fluorophores may cause the signal-to-noise ratio to deteriorate. However, probe signal-to-noise ratios and melt peak heights have been improved considerably through the inclusion of two or more FAM dyes within the oligonucleotide probe sequences. All probes described above utilise either C6 FAM dU or Glen fluorescein dT to replace thymine nucleotides and fluorophores in these probes are separated by at least three nucleotides. The inclusion of additional fluorophores in The oligonucleotides of the invention (e.g. 3 or 4 FAM labels) may further enhance signal-to-noise ratios and melt peak heights. However, fluorophore-labelled bases should be separated by a minimum number of nucleotides to avoid direct quenching between fluorophores by non-radiative energy transfer. This non-FRET quenching can occur through short-range 'contacts' between dyes and does not require overlap of excitation and emission spectra. FRET quenching may also occur between FAM labels due to the overlap between excitation and emission spectra. The preferred spacing when designing the oligonucleotide probes of the invention is 3-6. However, all probes with fluorophores separated by at least 2 unlabelled nucleotides exhibit considerable increases in fluorescence on target hybridisation. Spacings greater than 11 nucleotides have not been tested but may yield efficient signals depending on their relative angular disposition in the duplex (see table 7).<br>
150nM of HYBCHRC (SEQ ID NO:35) reverse homologue oligonucleotide was hybridised to 150nM of single-labelled (HYBCH2), dual-labelled (HYBCH),<br>
triple-labelled (HYBCH3) and quadruple (HYBCH4) Chlamydia probes in TaKaRa PCR buffer and a total of 3mM MgCl2.<br>
(Sequence Removed)<br>
Melt peaks were generated using a LightCycler protocol comprising denaturation (95°C 5s), cooling (35°C 30s) and a slow heat to 95°C at 0.2°C/s. Compared with the single-labelled probe, the dual-labelled HYBCH oligonucleotide probe displayed an elevated signal-to-noise ratio and melt peak height along with a reduced Tm as described above. The fluorophore-labelled bases of HYBCH are separated by 5 nucleotides. The triple-labelled HYBCH3 probe is quenched slightly upon target hybridisation and exhibits a small negative/inverted peak in -dF/dT traces (Figure 11). The Tm of HYBCH3 is reduced compared with the single-labelled probe, but is higher than that of the dual-labelled oligonucleotide probe. The quadruple-labelled HYBCH4 probe exhibits a small positive peak in -dF/dT traces and a Tm similar to that of HYBCH. Both HYBCH3 and HYBCH4 probes contain fluorophore-labelled bases that are separated by only a single nucleotide and display considerably reduced levels of fluorescence in both hybridised and dissociated states (Figure 11). Fluorescence data indicates that the proximal FAM dyes are quenching each other in the single-stranded state. The HYBCH3 probes is quenched to a greater extent upon hybridisation to complementary target sequences, suggesting that when the fluorophores project into solution they may be brought into even closer contact than when in the single-stranded conformation.<br>
Additional triple-labelled Chlamydia probes were designed with a greater level of separation between fluorophore-labelled bases. 150nM of HYBCH6RC (SEQ ID NO: 103) reverse homologue oligonucleotide was hybridised to 150nM of single-labelled (HYBCH2), dual-labelled (HYBCH) and triple-labelled (HYBCH5 &amp;<br>
HYBCH6) probes in TaKaRa PCR buffer and a total of 3mM MgCl2. Melt peaks were generated using a LightCycler protocol comprising denaturation (95°C 5s), cooling (35°C 30s) and a slow heat to 95°C at 0.2°C/s.<br>
(Sequence Removed)<br>
The triple-labelled HYBCH5 probe exhibits a Tm very similar to that of the HYBCH dual-labelled probe (table 1). Adding more than two fluorescent labels may not provide further stabilisation of the single-stranded structure. The triple-labelled HYBCH6 probe is 5 nucleotides longer than the previously mentioned Chlamydia probes and displays a Tm similar to the single-labelled HYCH2 probe. The triple-labelled probes both yield melt peaks that are 6.5 and 3.25 times the height of the single-labelled HYBCH2 and dual-labelled HYBCH probes respectively (Figures 4 &amp; 11C).<br>
To investigate the contributions that each fluorophore has on the total signal strength, the triple-labelled HYBCH5 probe was compared with the three possible single-labelled Chlamydia probes and three dual-labelled oligonucleotide probes. Each modified position within the triple-labelled oligonucleotide probes was represented in the single and dual-labelled probes. These probes were also compared with a quadruple-labelled oligonucleotide probes (SEQ ID NO: 126). As described previously, 150nM of HYBCH6RC (SEQ ID NO: 103) reverse homologue oligonucleotide was hybridised to 150nM of probe in TaKaRa PCR<br>
 (Sequence Removed)<br>
The heights of melt peaks generated in two LightCycler instruments (LCI and LC2) exhibited greater levels of variation than the calculated signal-to-noise ratios (table 4). However, with both LightCycler instruments, the peak heights of the triple-labelled probe were approximately twice the height of the most efficient dual-labelled probe and more than four times greater than the most efficient single-labelled oligonucleotide probe (table 4). The height of the triple-labelled melt peaks is greater than the sum of the three single-labelled oligonucleotide probes and is also greater than the 'sum of any dual-labelled probe plus any equivalent single-labelled oligonucleotide probe. The quadruple-labelled oligonucleotide probe also yielded high quality melt data, exhibiting peak heights that were approximately 25% larger than the triple-labelled probes (table 4).<br>
TABLE 4: Peak heights and signal-to noise ratios generated with single, double and triple-labelled Chlamydia probes using two LightCycler instruments (LC1 &amp; LC2). Each melt peak and curve was analysed in duplicate and the mean value is presented.<br><br>
(Table Removed)<br><br>
The single, dual and triple-labelled Chlamydia probes were also investigated at 35°C in the presence and absence of target to compare the levels of fluorescence emission in hybridised and single-stranded states. 150nM of HYBCH6RC (SEQ ID NO: 103) reverse homologue oligonucleotide was hybridised to 150nM of probe in TaKaRa PCR buffer and a total of 3mM MgCl2 Samples were denatured in LightCycler capillaries (95°C 5s) prior to cooling to 35°C with continuous fluorescence acquisition. Probes were analysed in duplicate in the presence and absence of target and the mean emission values for hybridised and single-stranded states are detailed in table 5. In the absence of target, the triple-labelled oligonucleotide probe emits less fluorescence than any of the single and dual-labelled probes. Whereas, in the presence of target, the level of fluorescence from the triple-labelled probe was very similar to that emitted from the dual-labelled HYBCH10 probe. The triple-labelled probe exhibits the biggest difference between duplex and single-stranded states and thus yields the largest melt peaks. An interaction between fluorophores, such as quenching in the single-stranded probe structure, may be responsible for the elevated peak heights and signal-to-noise ratios exhibited by dual and triple-labelled oligonucleotide probes. In the single-strand (random coil) the fluorophores can contact each other and stack giving rise to collisional quenching whereas this is not possible in the duplex as the fluorophores are kept apart by the rigid B-DNA structure when separated by at least two nucleotides. If they are closer together the flexible linker on the 5-position of the thymine base can allow the fluorophores to approach each other.<br>
TABLE 5: Levels of emission (LightCycler arbitrary fluorescence units) at 35°C in the presence and absence of a complementary oligonucleotide. The difference between hybridised and single-stranded states in also included.<br><br><br>
(Table Removed)<br>
Example 10: Further comparison of single, dual and triple-labelled oligonucleotides<br>
Three single-labelled Factor V oligonucleotide probe were compared with dual and triple-labelled probes of identical sequence. All fluorophores present in the dual and triple-labelled oligonucleotide probes were represented in the single-labelled probes. 150nM of FVG (SEQ ID NO:24) reverse homologue oligonucleotide was hybridised to 150nM of oligonucleotide probe in TaKaRa PCR buffer and a total of 3mM MgCl2. Melt peaks were generated using a LightCycler protocol comprising denaturation (95°C 5s), cooling (35°C 30s) and a slow heat to 95°C at 0.2°C/s. The peak heights and signal-to-noise ratios of single, dual and triple-labelled probes are presented in table 6.<br>
(Sequence Removed)<br>
The dual-labelled FVG11 probe yields melt peaks that are approximately 3-4 times larger than the single-labelled oligonucleotide probes that possess fluorophores at identical positions within the sequence. As such, dual-labelled melt peaks are greater than the sum of the single-labelled peaks. Furthermore, the triple-labelled oligonucleotide probe generates melt peaks that are at least 7 times larger than the single-labelled oligonucleotide probes and 2.6 times larger than the dual-labelled probe (Figure 12). Melt peaks from the triple-labelled probes are considerably larger than the sum of the peaks derived from single-labelled oligonucleotides. As with the Chlamydia oligonucleotides, the addition of multiple fluorophores to probes causes the generation of melt peaks whose heights are greater than would be expected from the single-labelled oligonucleotide probes. The data derived from Chlamydia and Factor V probes demonstrates that incorporating two or three fluorophore-labelled bases does not necessarily double or triple the height of oligonucleotide probe melt peaks.<br>
TABLE 6: Peak heights and signal-to noise ratios generated with single, double and triple-labelled Factor V probes in a single LightCycler experiment (LC2). Each melt peak and curve was analysed in duplicate and the mean value is presented.<br><br>
(Table Removed)<br>
67 Example 11: Minimum spacing between fluorophores<br>
A series of oligonucleotide probes was synthesised to determine the minimum spacing required between fluorophore-labelled bases in dual-labelled oligonucleotide probes. The probes possessed between 0 and 9 nucleotides between the fluorophore-labelled bases. 150nM of MODRC (5'CCCCCCTTTTTTTTTTTTTCCCCCC (SEQ ID NO: 139)) reverse homologue oligonucleotide was hybridised to 150nM of the oligonucleotide probe in TaKaRa PCR buffer and a total of 3mM MgC^. Melt peaks were generated using a LightCycler protocol comprising denaturation (95°C 5s), cooling (35°C 30s) and a slow heat to 95°C at 0.2°C/s. The signal-to-noise ratios and heights of melt peaks are presented in table 7. The probes possessing 0 and 1 nucleotides between labelled bases demonstrated considerably reduced levels of fluorescence due to contact quenching caused by the close proximity of fluorophores. Both of these probes exhibited negative (inverted) melt peaks, where the level of fluorescence decreased when hybridised to target sequences relative to the single-stranded state. All probes possessing between 2 and 9 nucleotides separating the fluorophore-labelled bases exhibited positive melt peaks and enhancement of fluorescence upon target hybridisation. The probes possessing fluorophores separated by 7 and 8 nucleotides exhibited slightly reduced levels of fluorescence and smaller melt peaks compared with the 2-6 and 9 spaced probes. Probes that possess multiple fluorophores on the same surface of duplex DNA may encounter a degree of contact quenching, thereby displaying reduced peak heights and signal-to-noise ratios. These angular disposition affects may be sequence specific since 8 nucleotide spacings have been employed in FVG11 (SEQ ID NO:60), HYBMRSA (SEQ ID NO:99) and HYBA1928C (SEQ ID NO:112) probes, generating large melt peaks and signal-to-noise ratios. At least two nucleotides separating fluorophore-labelled bases are required for the oligonucleotide probes to yield higher levels of fluorescence emission upon target hybridisation relative to the free probe.<br>
TABLE 7: Melt peaks generated in a single LightCycler experiment (LC2) to determine the minimum spacing required between fluorophore-labelled bases. Each melt peak was analysed in duplicate and the mean value is presented.<br><br>
(Table Removed)<br>
All dual, triple and quadruple-labelled oligonucleotide probes designed to date, which possess between two and eleven nucleotides separating fluorophore-labelled bases, demonstrate considerable enhancement of fluorescence upon hybridisation to complementary target sequences. Only those probes with zero or one nucleotide separating fluorophore-labelled bases have been found to exhibit<br>
small levels of fluorescence quenching upon target hybridisation. A maximum separation of fluorophore-labelled nucleotides has not yet been identified.<br>
Example 12: Attachment of fluorophores to dA and dC<br>
FAM dA and FAM dC oligonucleotide probes were synthesised using amino-dA and amino-dC monomers (Glen Research, Sterling, VA) and post-synthetic labelling with FAM. Single and dual-labelled probes containing FAM dC and FAM dA were synthesised and compared with single and dual-labelled C6 FAM dU oligonucleotide probes (table 8). 150nM of the appropriate reverse homologue oligonucleotide (FVG or GLENARC) was hybridised to 150nM the oligonucleotide probe in TaKaRa PCR buffer and a total of 3mM MgCl2. Melt peaks were generated using a LightCycler protocol comprising denaturation (95°C 5s), cooling (35°C 30s) and a slow heat to 95°C at 0.2°C/s.<br>
(Sequence Removed)<br>
The FAM dC and FAM dA probes all exhibited enhanced levels of fluorescence upon target hybridisation relative to the single-stranded state (Figure 13). Enhancement of fluorescence emission upon FAM dC hybridisation supports the model that the oligonucleotide probe fluorophores do not interact with the DNA of the target strand since interaction with Gs in the target would be expected to cause fluorescence quenching. As with C6 FAM dU and fluorescein dT, the dual-labelled FAM dC and FAM dA probes both yield melt peaks that are greater than twice the height of the equivalent single-labelled oligonucleotide probe.<br>
Dual-labelled C6 FAM dU and fluorescein dT probes exhibit Tms that are approximately 4-5°C lower than single-labelled probes of identical sequence. The probes labelled with FAM dC and FAM dA exhibited 1°C and 8.3°C Tm differences respectively (table 8).<br>
TABLE 8: Peak heights and signal-to-noise ratios derived from oligonucleotide probes containing C6 FAM dU, FAM dC and FAM dA labels.<br><br>
(Table Removed)<br>
For A, T, C and U, it does not matter whether fluorophores are attached to nucleotides pre, during or post-synthesis. Attachment to these nucleotides is possible whilst maintaining the 'natural' base. Attachment of the fluorophore directly to G produces a base analogue.<br>
Example 13: Asymmetric PCR to enhance melt peak height<br>
Following the PCR stage of assays using the oligonucleotides of the invention, probes compete with amplified products to hybridise to the target sequence. This competition reduces the potential for probe/target interactions and limits the height of melt peaks. If target hybridisation is shifted in favour of the probe, the height of melt peaks can be increased. This may be achieved by either increasing the concentration of the probe or decreasing the abundance of the PCR competitor strand.<br>
The optimal concentration of the oligonucleotide probes has been found to be 150nM. Increasing the probe concentration above 200nM has not been found to improve signal-to-noise ratios. In fact, increased probe concentration has an adverse affect on Chlamydia melt traces, generating a second peak at approximately 80°C that increases in height along with probe concentration. Peak qualities have also been found to be reduced at elevated probe concentrations due to additional background fluorescence arising from single-stranded probes that are unable to hybridise to target sequences.<br>
The abundance of the competing DNA strand may be reduced, relative to the target sequence, by using an asymmetric amplification method. The concentration of Chlamydia forward and reverse primers was optimised to maximise the height of melt peaks. Chlamydia cryptic plasmid targets were amplified using primers CHF3-1 (SEQ ID NO:33) and CHR4-1 (SEQ ID NO:34). Assays employed 3-stage LightCycler protocols consisting of an initial denaturation (95°C 1 min) followed by 50 cycles of PCR comprising denaturation (95°C 5s), primer annealing (55°C 10s) and extension of products (72°C 10s). Amplified target was detected and characterised through the inclusion of HYBCH probe (SEQ ID NO:61) and melt curve analysis comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat towards 95°C using a 0.2°C/s transition rate. In the presence of Chlamydia DNA, the HYBCH probe generates clear melt peaks with Tms of approximately 56°C (Figure 14). It was found that a ten-fold dilution of the forward primer (O.OSµM) relative to the reverse primer (O.SµM) increased the height of Chlamydia peaks by three to five-fold compared with standard<br>
equimolar assays (Figure 14). This increased peak height was observed with purified genomic DNA and processed clinical samples. The combination of asymmetric PCR with a triple-labelled labelled probe (HYBCH6) generated melt peaks that were approximately 20 times the size of those generated in symmetric assays employing the single-labelled oligonucleotide probe.<br>
Example 14: Asymmetric PCR, probe concentration and peak heights<br>
Asymmetric PCR has yielded increased peak heights, relative to symmetric amplification, with Chlamydia, Gonorrhoea, Factor II, A1928C, and C677T assays. However, asymmetric amplification failed to enhance peak heights in the NAT2*5C test. Since asymmetric PCR generates single-stranded targets, probes do not have to compete for hybridisation with complementary PCR strands and melt peaks have the potential to be of similar magnitude to those derived with oligonucleotides. If symmetric amplification yields melt peaks that are already of similar magnitude to oligonucleotide peaks (e.g. NAT2*5C assay), no benefit may be derived from asymmetric amplification. It appears that only those probes that display reduced peak heights, compared with oligonucleotides, exhibit enhanced peak heights when assays are converted to asymmetric tests. Asymmetric PCR may be beneficial if assays utilise long target amplicons or if primer-dimers are generated.<br>
The Factor V probes (FVG1 and FVG11) are a more complicated situation. Melt peaks generated in symmetric assays are of reasonable height, allowing reliable identification of samples, but are reduced compared with peaks derived from oligonucleotide targets. Fifty cycles of asymmetric amplification, performed with a heterozygous sample, has been found not to affect the height of either matched or mismatched peaks relative to symmetric assays. However, 60 and 70 cycles of asymmetric amplification were found to increase the height of the matched melt peak whilst reducing/not affecting the height of the mismatched peak (Figure 14). After 70 cycles, the probe becomes saturated with target such that preferential hybridisation to the fully complementary allele results in an imbalance of peak<br>
heights. The increased stability of fully complementary probe/target duplexes is responsible for this preferential hybridisation and increased peak height relative to the mismatched duplex.<br>
The FVGl 1 probe (SEQ ID NO:60) was hybridised to fully complementary (FVG (SEQ ID NO:24)) and mismatched (FVA (SEQ ID NO:25)) oligonucleotides to investigate the affect of target concentration on the heights of melt peaks. Melt analysis was performed in a LightCycler instrument employing a thermal protocol comprising denaturation (95°C Os), cooling (35°C 30s) and a slow heat to 95°C using a 0.2°C/s transition rate. Matched and mismatched targets were employed at equimolar concentrations. 150nM of probe hybridised to 150nM of target yielded matched and mismatched melt peaks of identical height (Figure 15). 300nM of probe hybridised to 300nM of target also yielded matched and mismatched peaks of comparable height. However, 150nM of probe hybridised to lµM of target yielded only the matched melt peak (Figure 15). The fully complementary target sequesters the probe and competitive hybridisation prevents efficient hybridisation to the mismatched target. The factor V SNP affects the stability of probe/target duplexes considerably, such that at high target concentrations the probe binds preferentially to the fully complementary sequence. This observation also explains the imbalance of peak heights between fully complementary and double mismatched targets, such as in the sickle cell and NAT1 * 10 assays.<br>
To ascertain whether higher concentrations of probe could re-establish the balance of matched and mismatched peak heights at elevated target concentrations, symmetric and asymmetric assays were performed using 150nM, 300nM and 500nM of FVG 11 probe and a heterozygous Factor V genomic DNA sample. Fifty, sixty and seventy cycles of 2-stage amplification were performed. After 50 cycles, the matched and mismatched peak heights were balanced and there was not a considerable difference between symmetric and asymmetric reactions (Figure 16). The 150nM probe reactions yielded the highest quality data. The heights of matched and mismatched melt peaks generated, with 150nM of probe, after 60 and 70 asymmetric cycles were unequal as demonstrated in figure 14.<br>
The height of the mismatched melt peak was considerably improved at the higher probe concentrations. After 60 asymmetric cycles, 150nM FVG11 yielded matched and mismatched melt peaks with heights of 0.83 and 0.35 respectively (2.3 times difference). Whereas, 500nM of probe yielded matched and mismatched melt peaks with heights of 0.63 and 0.50 respectively (1.13 times difference). The combined height of heterozygous peaks in 150nM and 500nM probe reactions was 1.19 and 1.13 respectively, similar to that observed with homozygous oligonucleotide target (Figure 3). The increased concentration of probe balanced the height of matched and mismatched melt peaks generated with asymmetrically amplified products. The oligonucleotide and PCR data does confirm the competitive hybridisation of probes to matched and mismatched target molecules. At high target concentrations, this competitive hybridisation and the reduced stability of mismatched duplexes results in the reduction or loss of peaks in a multi-target melt profile. Probe concentrations higher than 150nM may be required in certain asymmetric assays.<br>
Example 15: Multiplex analysis by melting temperature<br>
Four oligonucleotide probes were simultaneously hybridised to their complementary oligonucleotide targets in a single LightCycler capillary to demonstrate the multiplex potential of melt peak analysis. The probes and targets were employed at various concentrations to equalise the heights of peaks derived from single-labelled and dual-labelled probes. The polyT, HYBINF, HYBCH2 and HYBAdC probes and target oligonucleotides were employed at 1.2µM, 233nM, 183nM and 27nM respectively. Melt peaks were generated in TaKaRa PCR buffer, and a total of 3mM MgCl2 using a LightCycler protocol comprising denaturation (95°C 5s), cooling (35°C 30s) and a heat towards 95°C at 0.4°C/s. The polyT, HYBINF, HYBCH2 and HYBAdC probes generated clear melt peaks with Tms of approximately 49°C, 54°C, 62°C and 68°C respectively (Figure 17). At least four target sequences may be detected and identified on the basis of Tm using the oligonucleotide probes permitting simultaneous analysis of two biallelic<br>
SNPs. Multiplex potential may be enhanced further using spectrally distinct fluorescent dyes.<br>
Example 16: Positive amplification control<br>
Oligonucleotides of the invention may be employed to determine the presence or absence of infectious pathogens in clinical samples. The generation of specific melt peaks indicates the presence of particular infectious agents within the sample. However, the total absence of melt peaks does not reliably signify the lack of pathogen. Clinical samples, such as urine, may contain high levels of PCR inhibitors that may prevent pathogen detection in positive samples. An amplification control is required to distinguish reactions that are negative due to pathogen absence from those that failed to generate melt peaks due to PCR inhibition.<br>
The Chlamydia amplification control (mimic) was constructed using a 142 base oligonucleotide (SEQ ID NO:98). The long oligonucleotide was employed as PCR template for cloning purposes and contained a C to T base substitution within the probing region to enable differentiation of mimic from Chlamydial target on the basis of Tm. Amplification from the long oligonucleotide was performed using CHF3-1 (SEQ ID NO:33) and CHR4-1 (SEQ ID NO:34) primers. The amplified product was ligated into the pDrive vector using a PCR cloningplus Kit (Qiagen Ltd, Crawley, UK) and transformed into Qiagen EZ competent cells. Colonies containing the mimic sequence were identified directly using the Chlamydia assay using the oligonucleotide probes of the invention. Plasmids were purified from transformant cultures using a QIAprep Spin Miniprep kit (Qiagen Ltd, Crawley, UK) using the manufacturer's recommended procedures. Hybridisation of Chlamydia Oligonucleotides of the invention to the mimic target sequence results in a C:A mismatch that reduces probe Tms by approximately 8°C (Figure 18).<br>
Example 17: DNA quantification using the oligonuclotide probes of the invention<br>
Factor V targets were amplified from a series of DNA standards using primers FVF1.3 (SEQ ID NO:22) and FVR3.5 (SEQ ID NO:23). The DNA standards contained 100ng/µl, 10 ng/ul, 1 ng/ul, 100pg/µl and 10pg/(µl of human genomic DNA. Assays employed 3-stage LightCycler protocols consisting of an initial denaturation (95°C 1 min) followed by 50 cycles of PCR comprising denaturation (95°C Os) and annealing (50°C 10s) and extension (72°C 10s) phases. Amplified target was detected and characterised through the inclusion of FVGl (SEQ ID NO:21) or FVG11 (SEQ ID NO:60) and fluorescence acquisition during the annealing phase of PCR. Measuring the cycle number at which fluorescence emission increases above a threshold level (CT) enables quantification of DNA targets (Figure 19). The CT values derived during real-time PCR assays are directly proportional to the starting copy number of the target sequence and can be used to construct standard curves, from which 'unknown' samples may be quantified. Saliva samples have been typically found to contain approximately 1 ng/ul of human genomic DNA.<br>
Example 18: Analysis using fluorophores other than fluorescein<br>
Four Factor V oligonucleotide probes were synthesised each labelled with fluorophores at two internal positions. These probes were:<br>
FVGl ITAM 5'CTGTAXTCCTCGCCXGTCC (SEQ ID NO:140) where X is a dT residue labelled with TAMRA.<br>
FVGl 1ROX 5'CTGTAXTCCTCGCCXGTCC (SEQ ID NO:141) where X is a dT residue labelled with ROX.<br>
FVGllCy5 5'CTGTAXTCCTCGCCXGTCC (SEQ ID NO:142) where X is a dT residue labelled with Cy5.<br>
FVGllMix 5'CTGTAFTCCTCGCCXGTCC (SEQ ID NO: 143) where F is a dT residue labelled with FAM and X is a dT residue labelled with ROX.<br>
The probes were hybridised to their complementary oligonucleotide targets (SEQ ID NO:24) in 96 well optical plates in TaKaRa PCR buffer and total of 3 mM MgCl2 and melt curves generated using an ABI 7700 thermocycler. The melt protocol comprised denaturation (95°C 30s), cooling (35°C 30s) and a melting profile comprising a stepwise progression from 35°C to 95°C in 1°C increments with a 15s hold at each temperature. The probes and complements were used at concentrations of 150 nM. Raw data was exported from the ABI 7700 thermocycler and processed using the Excel spreadsheet programme from Microsoft. Figures 20A, 20B, 20C and 20D show the melt curves of these probes from their complements and in all cases an inflection in the curve is seen and demonstrates a decrease in fluorescent as a direct result of the dissociation of the probe from its target in comparison to the linear decrease of fluorescence as a function of temperature despite the optics system being sub-optimal for the detection of Cy5. This data confirms previous data and conclusions that this phenomenon is a general feature of fluorophores and is not restricted to fluorescein and other fluorescein based derivatives.<br>
Conclusion<br>
Oligonucleotide probes possessing single and multiple fluorophore-labelled internal bases all exhibit increased levels of fluorescence emission when hybridised compared with the single-stranded state. Since probe fluorescence always increases upon hybridisation irrespective of oligonucleotide probe and target sequences, it is envisaged that the fluorophores interact with probe DNA when single-stranded, resulting in quenching, but do not interact with probe or target sequences when in the duplex state such that fluorophores are dequenched and fluorescence increases. The increased levels of fluorescence emitted by<br>
hybridised probes is thought to arise from the fluorophores projecting into solution away from the quenching influence of DNA.<br>
An unexpected finding from comparing single-labelled oligonucleotide probes with dual and triple-labelled probes of identical sequence was that fluorescence was not simply doubled or tripled with the inclusion of additional fluorophores. Double and triple-labelled probes exhibited melt peaks that were larger than expected from each of the single-labelled oligonucleotide probes suggesting an interaction between fluorophores. Fluorophores in oligonucleotide probes are preferably separated by at least two nucleotides to prevent contact quenching and ensure efficient signal generation upon target detection.<br>
TABLE 9;<br>
General fluorescent dyes for oligonucleotide labelling<br>
 (Table Removed)<br>
TABLE 10; <br>
Alexa dyes (Invitrogen)<br>
 (Table Removed)<br>
TABLE 11;<br>
ATTO dyes (ATTO-TEC GmbH)<br>
 (Table Removed)<br>
TABLE 12; <br>
Dyomics dyes (Dyomics GmbH)<br>
 (Table Removed)<br>
TABLE 13; Dyomics Metastokes dyes (Dyomics GmbH). All can be excited at 488nm<br>
 (Table Removed)<br>
TABLE 14;<br>
Hilyte dyes (Cambridge Biosciedve)<br>
 (Table Removed)<br>
TABLE 15;<br>
Low excitation wavelength (UV) fluorophores <br>
 (Table Removed)<br>
sequence listing<br>
(Sequence Removed) <br>
Refernces cited<br>
patent documents<br>
US 5,691,146    11/1197 Maryland <br>
US 6,174,670 B1 1/2001 Rasmussen et al<br>
WO 01/36668 A1  5/2001 Sauer et al<br>
US 6, 635,427 B2 10/2001 Witter et al <br>
WO 01/73118 a2 10/2001  French et al<br>
Other publicatios<br>
Brown et al (2001) Synthesis of a modified thymidine monomer for site-<br>
specific incorporation of reporter groups into oligonucleotides. Tetrahedron<br>
Lett. 42, 2587-2591.<br>
1.	Brown et al (2003) Synthesis of fluorophore and quencher monomers for use<br>
in   Scorpion   primers   and   nucleic   acid   structure   probes.   Organic  and<br>
Biomolecular Chem. 1, 2267-2275.<br>
2.	Crockett &amp; Wittwer (2001) Fluorescein-Labeled Oligonucleotides for Real-<br>
Time PCR: Using the Inherent Quenching of Deoxyguanosine Nucleotides.<br>
Anal. Biochem., 290, 89-97.<br>
3.	Dobson et al (2003)  Synthesis of HyBeacons and dual-labelled probes<br>
containing 2'-fluorescent groups for use in genetic analysis   Chem. Comm.<br>
1234-1235<br>
4.	Elnifro et al (2000) PCR and restriction endonuclease analysis for rapid<br>
identification of Human Adenovirus subgenera. J. Clin. Microbiol. 38: 2055-<br>
2061<br>
5.	French et al (2001) HyBeacon probes: a new tool for DNA sequence detection<br>
and allele discrimination. Molecular and Cellular Probes. 15, 363-74<br>
6.	French et al (2002) Ultra-rapid DNA analysis using HyBeacon probes and<br>
d'rect PCR amplification direct from saliva. Molecular and Cellular Probes.<br>
16,319-26<br>
8.	Hawkins et al (1997) Fluorescence properties of pteridine nucleoside analogs<br>
as monomers and incorporated into oligonucleotides. Anal. Biochem. 244, 86-<br>
95<br>
9.	Heinlein et al (2003) Photoinduced electron transfer between fluorescent dyes<br>
and guanosine residues DNA-hairpins. J. Phys. Chem. 107, 7957-7964<br>
10.	Knemeyer et al (2000). Probes for detection of specific DNA sequences at the<br>
single-molecule level. Anal. Chem. 72, 3717-24<br>
11.	Lee   et  al  (1994)   A   fluorogenic   assay   for   DNA   cleavage   reactions<br>
characterized with BamHI restriction endonucleoase.   Anal. Biochem. 220,<br>
377-83<br>
12.	Lee et al (2002) ResonSense: simple linear fluorescent probes for quantitative<br>
homogeneous rapid polymerase chain reaction. Analytica Chimica Ada 457,<br>
61-70<br>
13.	Marks et al (2005)    Molecular basis of action of HyBeacon fluorogenic<br>
probes:  A  spectroscopic  and  molecular  dynamics  study.     Journal of<br>
Biomolecular Structure &amp; Dynamics 23, 49-62<br>
14.	Nazerenko et al (2002) Multiplex quantitative PCR using self-quenched<br>
primers labeled with a single fluorophore. Nucleic Acids Res. 30, e37<br>
15.	Nazerenko et al (2002b) Effect of primary and  secondary structure  of<br>
oligodeoxyribonucleotides on the fluorescent properties of conjugated dyes.<br>
Nucleic Acids Res. 30, 2089-2195<br>
16.	Norlund et al (1989) Structure and dynamics of a fluorescent DNA oligomer<br>
containing the EcoRI recognition sequence: fluorescence, molecular dynamics<br>
and NMR studies. Biochemistry 28, 9095-9103<br>
17.	Seidel et al (1996) Nucleobase-specific quenching of fluorescent dyes.  1.<br>
Nucleobase one-electron redox potentials and their correlation with static and<br>
dynamic quenching efficiencies. J. Phys. Chem., 100, 5541-5553.<br>
18.	Solinas et al (2001) Duplex Scorpion primers in SNP assays and FRET<br>
applications. Nucleic Acids Res. 29, e96<br>
19.	Svanvik et al (2000) Detection of PCR products in real time using light-up<br>
probes. Analytical Biochemistry 287', 179-182<br>
117 20. Thelwell et al (2000) Mode of action and application of Scorpion Primers to<br>
mutation detection. Nucleic Acids Res. 28, 3752-3761 21.Tyagi &amp; Kramer (1996) Molecular Beacons: probes that fluoresce upon<br>
hybridisation. Nature Biotechnology 14, 303-308<br>
22.	Whitcombe et al (1999) Detection of PCR products using self-probing<br>
amplicons and fluorescence. Nature Biotechnology 17, 804-807.<br>
23.	Xu et al (1994) Melting and premelting transitions of an oligomer measured<br>
by DNA base fluorescence and absorption. Biochemistry 33, 9592-9599<br>
24.	Zucker (2003) Mfold web server for nucleic acid folding and hybridisation<br>
prediction. Nucleic Acids Res. 31,3406-3415<br><br><br><br><br><br>
 CLAIMS<br>
1.	A  single  stranded oligonucleotide having  substantially no  secondary<br>
structure and formed of nucleotide residues wherein two or more internal<br>
nucleotide residues are labelled with a fluorophore without an associated<br>
quencher.<br>
2.	An oligonucleotide according to Claim 1 wherein up to about one-third of<br>
the internal residues are labelled with a fluorophore.<br>
3.	An oligonucleotide according to Claim 1 or 2 wherein at least two of the<br>
nucleotide residues labelled with a fluorophore are separated by at least<br>
two unlabelled nucleotide residues.<br>
4.	An oligonucleotide according to any one of the preceding claims wherein<br>
all of the nucleotide residues labelled with a fluorophore are separated by<br>
at least two unlabelled nucleotide residues.<br>
5.	An oligonucleotide according to any one of the preceding claims and<br>
which contains at least one G residue, wherein at least one of the<br>
fluorophore-labelled nucleotide residues is positioned adjacent to at least<br>
one of the G residues.<br>
6.	An oligonucleotide according to any one of the preceding claims wherein<br>
the 3' nucleotide residue does not contain a 3' hydroxyl.<br>
7.	An oligonucleotide according to Claim 6 wherein the 3' nucleotide residue<br>
contains a 3' phosphate or is a 3' deoxy residue.<br>
8.	An oligonucleotide according to any one of the preceding claims which<br>
contains from 10 to 50 nucleotide residues.<br><br>
9.	An oligonucleotide according to Claim 8 which contains from 15 to 30<br>
nucleotide residues.<br>
10.	An oligonucleotide according to Claim 8 which contains from 2 to 10<br>
fluorophore-labelled internal nucleotide residues.<br>
11.	An oligonucleotide according to Claim 9 which contains from 2 to 5<br>
fluorophore-labelled internal residues.<br>
12.	An oligonucleotide according to any one of the preceding claims wherein<br>
each fluorophore is the same fluorophore.<br>
13.	An oligonucleotide according to any one of Claims 1 to 11 contains two or<br>
more different fluorophores.<br>
14.	An oligonucleotide according to any one of the preceding claims wherein<br>
the fluorophore is selected from the group consisting of fluorescein-based<br>
dyes such as 6-carboxyfluorescein (FAM), tetrachlorofluorescein (TET),<br>
hexachlorofluorescein (HEX), rhodamine-based dyes such as 6-carboxy-<br>
X-rhodamine (ROX), 6-carboxytetramethylrhodamine (TAMRA); a Cy<br>
family dye such as Cy3 or Cy5; other fluorophores such as NED or JOE,<br>
and Alexa dyes, Atto dyes, Dyomic dyes, Dyomic megastoke dyes and<br>
Thilyte dyes.<br>
15.	An oligonucleotide according to Claim 13 which contains two different<br>
fluorophores, one being FAM and the other ROX.<br>
16.	An oligonucleotide according to any one of Claims 1 to 14 wherein the<br>
fluorophore does not intercalate into double standard DNA,<br>
17.	An oligonucleotide according to any one of the preceding claims which<br>
has a sequence complementary to a target polynucleotide sequence.<br><br>
18.	An   oligonucleotide   according   to   Claim    17   wherein   the   target<br>
polynucleotide sequence is selected from the group consisting of the<br>
human genome including any gene therein or allele thereof, the mouse<br>
genome including any gene therein or allele thereof, and the genome of<br>
any infectious agent including any gene therein or allele thereof.<br>
19.	An   oligonucleotide  according  to   Claim   18  which  has  a  sequence<br>
complementary to any one of the following human genes or an allele<br>
thereof:<br>
N-acetyltransferase 2	XI4672<br>
Factor V Leiden	AY364535<br>
Factor II	AF493953<br>
MTHFR	NM_005957<br>
Sickle cell anaemia	AY356351;<br>
and sheep PrP gene (NM_001009481).<br>
20.	An  oligonucleotide  according to  Claim   18  which  has  a  sequence<br>
complementary to the genome or any gene therein or allele thereof of the<br>
following infectious agents:<br>
Chlamydia trachomatis	X07547<br>
Adenovirus	AJ293905<br>
Influenza A	AY 130766<br>
Streptococcus pneumoniae	X52474<br>
MRSA	AJ810121<br>
21.	An oligonucleotide according to any one of Claims 18 to 20 which has a<br>
sequence fully complementary to one allele of a gene.<br>
22.	An oligonucleotide according to any one of Claims 17 to 20 which has at<br>
least 70% sequence identity with the target polynucleotide sequence.<br><br>
23.	has at least 80%<br>
sequence identity with the target polynucleotide sequence.<br>
24.	An   oligonucleotide   according   to   Claim   21   wherein   the   sequence<br>
complementary to the nucleotide sequence difference of the given allele<br>
compared to another allele of the gene is located towards the middle of the<br>
oligonucleotide.<br>
25.	An  oligonucleotide  according  to  any  one  of the  preceding  claims<br>
immobilised on or within a support.<br>
26.	An array of oligonucleotides immobilised at spaced locations on or within<br>
a support wherein at least one of the oligonucleotides is an oligonucleotide<br>
according to any one of Claims 1 to 24.<br>
27.	An  array  of oligonucleotides  according  to  Claim  26  wherein  each<br>
oligonucleotide is an oligonucleotide according to any one of Claims 1 to<br>
24.<br>
28.	A method of investigating a target polynucleotide sequence in a sample<br>
containing polynucleotide sequences, the method comprising contacting<br>
the sample with an oligonucleotide according to any one of Claims 1 to 25<br>
which is able to hybridise to the target polynucleotide sequence, and<br>
determining whether hybridisation occurs.<br>
29.	A method according to Claim 28 wherein detection of an increase in<br>
fluorescence is used to determine whether hybridisation has occurred<br>
between the oligonucleotide and the target polynucleotide sequence.<br>
30.	A method according to Claim 28 or Claim 29 wherein the method is<br>
performed at a predetermined temperature near,  or over a range of<br>
temperatures encompassing, the melting point of the hybrid or hybrids<br><br>
formed   between   the   target   polynucleotide   sequence   and   the   said oligonucleotide.<br>
31.	A method according to any one of Claims  28  to  30  wherein  the<br>
oligonucleotide  is  fully  complementary  to  the  target  polynucleotide<br>
sequence.<br>
32.	A method according to any one of Claims 28 to  30  wherein  the<br>
oligonucleotide is not fully complementary to the target polynucleotide<br>
sequence.<br>
33.	A method according to Claim 32 wherein the oligonucleotide has at least<br>
70% sequence identity with the target polynucleotide sequence.<br>
34.	A method according to Claim 33 wherein the oligonucleotide has at least<br>
80% sequence identity with the target polynucleotide sequence.<br>
35.	A method according to any one of Claims 28 to 33 wherein the target<br>
polynucleotide sequence investigated is one or more alleles of a gene.<br>
36.	A method according to Claim 35 wherein the oligonucleotide is able to<br>
hybridise to more than one allele of the gene.<br>
37.	A method according to any one of Claims 28 to 34 wherein the method is<br>
used to detect, identify or quantitate the presence or amount of the target<br>
polynucleotide sequence in the sample.<br>
38.	A method according to any one of Claims 28 to 35 wherein the sample<br>
contains polynucleotide sequences produced by amplification.<br>
39.	A method according to Claim 38 wherein the polynucleotide sequences are<br>
produced by any one or more of polymerase chain reaction (PCR), reverse<br><br>
transcription PCR (RT-PCR), ligase chain reaction (LCR), strand displacement amplification (SDA), transcription mediated amplification (TMA), rolling circle amplification (RCA) or nucleic acid sequence based amplification (NASBA).<br>
40.	A method according to Claim 38 or 39 wherein the amplification of the<br>
polynucleotide sequences in the sample is performed in the presence of the<br>
oligonucleotide which is able to hybridise to the target polynucleotide<br>
sequence.<br>
41.	A method according to Claim 40 wherein the 3' nucleotide residue of the<br>
oligonucleotide does not contain a 3' hydroxyl.<br>
42.	A  method  according to  Claim  40  or  41   wherein  observation  of<br>
fluorescence is made during the amplification of the polynucleotide<br>
sequences in the sample.<br>
43.	A method according to any  one of Claims 38 to 42 wherein the<br>
polynucleotide sequences in the sample are produced by amplification<br>
from a biological source without prior nucleic acid extraction.<br>
44.	A method according to Claim 39 wherein the PCR is an asymmetric<br>
amplification.<br>
45.	A method according to any one of Claims 30 to 44 wherein the<br>
oligonucleotide which is able to hybridise to the target polynucleotide<br>
sequence,  the  sample  which  may  contain  the  poiynucleotide  target<br>
sequence or both are immobilised on or within a support.<br>
46.	A method according to any one of Claims 30 to 45 wherein the target<br>
polynucleotide sequence is selected from the group consisting of the<br>
human genome including any gene therein or allele thereof, the mouse<br><br>
genome or any gene therein or allele thereof, and the genome of any infectious agent including any gene therein or allele thereof.<br>
47.	A process for making an  oligonucleotide  according to  any one  of<br>
Claims 17 to 24 the process comprising selecting a target polynucleotide<br>
sequence and synthesising an oligonucleotide which has a sequence<br>
complementary to the target polynucleotide and which has the properties<br>
as set out in any of Claims 1 to 25.<br>
48.	A process according to Claim 47 wherein the target polynucleotide<br>
sequence is selected from the group consisting of the human genome<br>
including any gene therein or allele thereof, the mouse genome including<br>
any gene therein or allele thereof, and the genome of any infectious agent<br>
including any gene therein or allele thereof.<br>
49.	Any novel single stranded oligonucleotide wherein two or more internal<br>
nucleotide residues are labelled with a fluorophore without an associated<br>
quenches as herein described, with particular reference to the tables and<br>
sequence listing in the description.<br>
50.	Any novel method of investigating a target polynucleotide sequence as<br>
herein described.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDYtQ2xhaW1zLSgxMi0wOS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2006-Claims-(12-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDYtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMi0wOS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2006-Correspondence Others-(12-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDYtRGVzY3JpcHRpb24gKENvbXBsZXRlKS0oMTItMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">485-delnp-2006-Description (Complete)-(12-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDYtRm9ybS0yLSgxMi0wOS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2006-Form-2-(12-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQWJzdHJhY3QtKDIyLTAzLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Abstract-(22-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">485-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ2xhaW1zLSgyMi0wMy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Claims-(22-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">485-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMy0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence Others-(13-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNy0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence Others-(17-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwMS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence-Others-(01-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwMy0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence-Others-(03-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LURFTE5QLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgxMi0wOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-DELNP-2008-Correspondence-Others-(12-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgxNi0wMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence-Others-(16-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgxNi0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence-Others-(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyMi0wMy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Correspondence-Others-(22-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LURFTE5QLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">485-DELNP-2008-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">485-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtRm9ybS0xLSgyMi0wMy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Form-1-(22-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">485-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">485-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtRm9ybS0zLSgwMS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Form-3-(01-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtRm9ybS0zLSgwMy0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Form-3-(03-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LURFTE5QLTIwMDgtRm9ybS0zLSgxMi0wOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-DELNP-2008-Form-3-(12-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtRm9ybS0zLSgxMy0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Form-3-(13-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">485-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">485-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtR1BBLSgwMS0wMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-GPA-(01-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTIwNi5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-206.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTIyMC5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtcGN0LTMwOC5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg1LWRlbG5wLTIwMDgtUGV0aXRpb24tMTM3LSgyMi0wMy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">485-delnp-2008-Petition-137-(22-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS0xMy0oMjItMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">Form-13-(22-03-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263061-methods-for-managing-data-transmissions-between-a-mobile-station-and-a-serving-station.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263063-mold-for-molding-and-method-for-molding-product.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263062</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>485/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>41/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Sep-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jan-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LGC LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>QUEENS ROAD, TEDDINGTON, MIDDLESEX TW11 0LY U.K.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MCDOWELL DAVID GORDON</td>
											<td>C/O LGC LIMITED, OF QUEENS ROAD, TEDDINGTON, MIDDLESEX TW11 0LY U.K.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FRENCH DAVID JOHN</td>
											<td>C/O LGC LIMITED, OF QUEENS ROAD, TEDDINGTON, MIDDLESEX TW11 0LY U.K.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2006/002719</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-07-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0514889.5</td>
									<td>2005-07-20</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263062-oligonucleotides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:28:21 GMT -->
</html>
