<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263177-genetic-analysis-and-authentication by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:02:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263177:&quot;GENETIC ANALYSIS AND AUTHENTICATION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;GENETIC ANALYSIS AND AUTHENTICATION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides compositions for genetic testng of an organism and for correlating the results of the genetic testing with a unique marker that unambiguously identifies the organism. The markers may be internal markers, such as for example eingle nucleotide polymorphism (SNPs), short tandem repeats ((STRs), or the other sites within a genomic locus Alternatively, the markers may be external, such that they are separately added to the genetic sample before testing.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>GENETIC ANALYSIS AND AUTHENTICATION<br>
BACKGROUND OF THE INVENTION<br>
Genetic analysis is widely used in basic and applied lesearch as well as m diagnostics to screen, to profile and to genotype patients Clinical laboratories currently offer genetic tests for more than 300 diseases or conditions including the analysis of mutations in the BRCA1 and BRCA2 genes, as well as in the p53, N-, C- and K-RAS, cytochrome P450, CFTR, HLA class 1 and II, Duchenne Muscular Dystrophy and beta-globm genes The test menu continues to grow as advances in the Human Genome Project lead to the identification of genetic determinants that play a role in causing disease<br>
Genetic testing involves the analysis of genes and/or chromosomes to detect inheritable or other mutations as well as chromosome aberrations in order to provide a diagnosis for disease susceptibility In addition, protein levels are monitored to obtain an indication of disease progression or response to treatment Genetic testing has been used to diagnose and to monitor cancer, as well as to assess the pre-symptomatic risk of individuals to develop the disease At present, for example, members of families diagnosed   for   several   diseases   such   as   Atexia-telangiectasia,   Bloom's   syndrome,<br>
Fancom's anemia or Xeioderma Pigmentosum can be tested for the occurrence of mutations in the respective genes  In addition, several mutations in the regulatory gene p53 also have been correlated with the risk of developing different types of cancers Those who inherit p53 mutations are at high risk or developing sarcoma, brain tumors or leukemia<br>
The standards analysis methods used in genetic analysis, DNA typing and DNA fingerprinting include (1) analysis or variable Number of Tandem Repeats (VNTR) (e g. Nakamura et al, Science, Vol 235, pp 1616-1622 (1987), (2) analysis of Short Tandem Repeats (STR) (eg, Edwards et al, Am J Hum genet Vol 49, pp 746-756 (1991), Ricciardone et a], Bioteehmques, Vol 23, pp 742-747 (1997), (3) analysis of Single Nucleotide Polymorphisms (SNP) (e g, Nickerson et al, Proc Natl Acad Sci USA, Vol 87, pp 8923-8927 (1990), Nikiforov et al Nucleic Acids Res Vol 22, pp 4167-4175 (1994), Ross et al Anal Chem Vol 69 pp 4197-4202)), (d) analysis of Restriction Fragment Length Polymorphisms (RFLPs) (e g , Botstem et al Am J Hum Genet Vol 32, pp 314-331 (1980)), and (4) analysis of mitochondrial DNA sequences VNTR and STR analyses utilize simple or multiplex Polymerase Cham Reaction (PCR) technology (e g , Mullis et al, Cold Spring Harbor Symp Quant Biol, Vol 51, pp 263-273 (1986), Mullis et al, Science, Vol 239, Vol 487-491 (1988)) RFLP analysis utilizes restriction enzyme digestion of DNA followed by DNA hybridization techniques with labeled probes, and mitochondrial DNA sequence analysis utilizes a combination of<br>
PCR technology and conventional dideoxy sequencing in a process known as cycle sequencing<br>
Variations among individuals in the number of STRs in specific genetic locations have been shown to be associated with several common genetic diseases For example, unstable doublet repeats are known to be associated with disease states such as cystic fibrosis and colorectal cancer Certain unstable triplet repeats are known to be associated with several genetic diseases, including Kennedy's disease, fragile-X syndrome and Myotonic dystrophy Huntington'&amp; disease in particular has been investigated extensively and STRs have been mapped across a section of the gene to identify 51 triplet repeats spanning a 1 86 Mbp DNA segment Higher-order repeats, such as tetramers have also been associated with particular disease states including Huntington's disease and spinocerebellar ataxia type 1<br>
DNA typing based on the standard laboratory methods requires extensive sample preparation and significant post-PCR processing The latter includes the steps of restriction enzyme digestion, agarose/acrylamide gel electrophoresis, sequence analysts or a combination of these methods These multi-step protocols introduce considerable bias in the data and are labor intensive and time consuming<br>
DNA fingerprinting, also referred to as identity testing, relies on the analysis of highly polymorphic genetic loci    to provide unambiguous molecular identification of<br>
individuals A variety of polymorphic markers are available for this purpose including restriction fragment length polymorphisms (RFLPs), single nucleotide polymorphisms (SNPs), STRs/microsatelhtes dnd variable number of tandem repeats (VNTRs)/mmisatellites RFLP analysis requ.res enzyme digestion of genomic DNA followed by gel electrophoresis and hybridization of radiolabeled probes to the gel The complexity of this procedure has prevented RFLP analysis from being widely adopted foi identity testing SNPs, wherein one allele differs from another allele at a single position, occur with an average frequency of 1 in 1,000 bases in both coding and non-coding regions and constitute 90% of all polymorphisms within the human genome (Brooker, Gene, 234 177-186 (1999)) They have be^n used for the mapping of genes associated with diseases such as cancer, for the typing of donors for bone manow engraftment and for studying inheritance within the context of population genetics (Kwok at a! , Mol Med Today, 538-543, (1999)) However, while suitable sets of SNPs are being developed to provide unambiguous DNA fingerprints, those new markers will require careful validation In addition, in comparison to the STR markers commonly used at present, the set of SNP markers required to ensure a given probability of exclusion of ambiguity will be large Both SNPs and STR polymorphisms can be used as markers, however about 7 to 12 SNPs per STR polymorphism are required to get a pov*er of exclusion of 99 73%<br>
STRs and VNTRs are highly informative polymorphic markers Many genetic loci contain a polymorphic STR region consisting of short, repetitive sequence elements, typically 3 to 7 bases in length Tnnienc and tetramenc STRs occur as frequently as once per 15,000 bases of a given sequence and are widely used for identity typing in parentage and forensic analysis In contrast to the case for SNPs, where a large number of loci are needed for exclusion, only nine specific STR loci are required to provide a combined average power of exclusion of 99 73% (Alford et al Current Opinion in Biotechnology, 29-33 (1994), Latour et al, 829-37 (2001) STRs may be amplified via the polymerase chain reaction (PCR) by employing specific primer sequences directed to the regions flanking the tandem repeat<br>
Other polymorphisms arising from differences in the number of repeated elements in an allele include variable number of tandem repeats (VNTRs)/mimsatelhtes, which are tandem repeats of a short sequence containing from 9 to 60 bases, and microsatellites which contain from one to five bases Mmisatelhtes and microsateilites are generally considered to be a subclass of VNTRs Since it is estimated that about 500,000 microsateilite repeats are distributed throughouf the human genome at an average spacing of 7,000 bases, VNTR regions *750 can be used in identity testing<br>
In conventional laboratory practice, STRs and VNTRs are amplified by PCR using radio-labeled 01 fluorescence-labeled primers The PCR products are separated by gel electrophoresis or capillary electrophoresis for identification<br>
In conventional implementations of genetic testing, information relating to sample and patient identification is recorded manually, typically involving the completion of bar coded labels which are affixed to sample collection containers Such labeling procedures represent a potentially significant source of error involving mishandling, mislabeling and switching of samples<br>
Thus a need exists for a mechanism whereby collected known biological samples would be unambiguously marked and identified at the time of collection   This would safeguard against the mishandling, mislabeling and switching of samples during analysis SUMMARY OF THE INVENTION<br>
This invention provides methods of analyzing STRs and related repeated sequence elements in parallel, in order to unambiguously link samples with genetic test results and patient identity Specifically, the present invention provides methods for recording a molecular identification (ID) concurrently with the completion of a genetic analysis, by linking a patient's genetic profile to a patient's molecular fingerprint, thereby  minimizing the incidence of inadvertent mishandling of samples and permitting<br>
unambiguous authentication by comparison against previously recorded, or subsequently recorded molecular identification<br>
One aspect of this invention is to provide a composition for analyzing a target nucleic acid sequence obtained from a patient sample while concurrently providing the genetic fingerprint of the patient This composition compnses a first set of probes and a second set of probes The first set of probes comprises oligonucleotide probes that hybridize to a target nucleic acid sequence obtained from a patient sample for genetic testing, while the second set compnses oligonucleotide probes for hybridizing to a plurality of polymorphic markers The hybridization to these markers provides a genetic fingerprint that identifies the patient The probes of these two sets are attached to beads that are associated with a chemically or physically distinguishable characteristic that can be used to uniquely identify the probes that are attached to the beads<br>
Another aspect of this invention is to provide a method for analyzing a target nucleic acid sequence obtained from a patient sample while concurrently providing the genetic fingerprint of the patient This method compnses providing a first set of probes and a second set of probes The first set of probes compnses oligonucleotide probes that hybridize to a target nucleic acid sequence obtained from a patient sample for genetic testing, while the second set of probes compnses oligonucleotide probes for hybridizing to a plurality of polymorphic markers    The hybridization to these markers provides a<br>
genetic fingerprint that allows the identification of the patient The probes of the first and the second set are attached to beads that are associated with a chemically or physically distinguishable characteristic that uniquely identifies the probes that are attached to said beads This method further comprises contacting a target sequence and a plurality of polymorphic markers to the first and second set of probes, and then detecting the hybridization between the probes of the first set to the target sequence and detecting the hybridization between the probes of the second set to the polymorphic markers<br>
Another aspect of this invention is to provide a method of analysing a target nucleic acid sequence obtained from a patient sample   This method involves providing a means for uniquely linking the sequence analysis  with  the sample and comprises providing a set of probes comprising oligonucleotide probes that hybridize to a target nucleic acid sequence obtained from a patient sample   The probes are attached to beads that are associated with a chemically or physically distinguishable characteristic that uniquely identifies the probes attached to- the beads     The method further comprises contacting the oligonucleotide probes with a solution containing the target nucleic acid sequence to allow the target sequence to hybridize with the corresponding probe and detecting the hybridization of the piohes with the target sequence   The solution is labeled with a molecular label that uniquely identifies the target solution, such that the patient identity is determined by interrogating the label    The label may be added to the sample before or after the solution is introduced to the oligonucleotides, or at the same time<br>
Another aspect of this invention is to provide a method of determining the number of tandem nucleotide repeats in a target nucleic acid sequence, where the tandem repeats are flanked at each side by a non-repeat flanking sequence The method comprises providing a set of oligonucleotide probes attached to beads, wherein each bead is associated with a chemically Or physically distinguishable characteristic that uniquely identifies the probe attached to the bead Each probe is capable of annealing to the target sequence and contains an interrogation site The set of probes is designed such that the probes differ in the number of repeated nucleotides When the probes are annealed to the target sequence to form hybridization complexes, the interrogation site of each probe is aligned with a target site located either within the tandem repeats or outside the tandem repeats The method further comprses contacting a target sequence to the oligonucleotide probes, so that the target sequence forms hybridization complexes with the probes The hybridization complexes between the target sequence and probes in the set are interrogated in parallel to determine whether the interrogation site of the probes end outside the repeats of the target or mside the repeats of the target The number of repeats in the target sequence is also determined<br>
Yet another aspect of this invention is to provide a method of sequence-specific amplification of assay signals produced in the analysis of a target nucleic acid sequence This     method permits real-time monitoring of the amphfied signal  and comprises<br>
providing a temperature-controlled sample containment device that pemits real-time recording of optical assay signals produced withm the device The method further comprises providing a temperature control means for controlling the temperature of the device and providing, within the sample containment device, a set of interrogation oligonucleotide probes These probes are capable of forming a hybridization complex with the target nucleic acid and are attached to beads Each bead is associated with a chemically or physically distinguisahble characteristic that identifies the probe attached to the bead The ohgonucleotide probes are contacted with the target sequence to form a hybridization complex between the probes and the target sequence This hybridization complex is contacted with a second oligonucleotide probe that comprises a label and is capable of being hgated to the interrogation probes contained withm the hybridization complex This method also comprises providing means to ligate the second labeled oligonucleotide probe to the interrogation probe withm the hybridization complex and then detecting the optical signals from the tÂ»et of immobilized probes in real-time One or more anneahng-hgating-detecting-denatunng cycles are performed, with each cycle increasing the number of extended probes in anthmetic progression and involving the following steps<br>
(1)        providing a first temperature for the formation of the hybridization complex,<br>
(ii)      providing a second temperature for ligase-catalyzed ligation of<br>
interrogation probe and the second labeled probe to occur, wherein<br>
ligation is associated with a change in optical signature of beads<br>
associated with the hgated probe, (in)      imaging and/or recording optical signals from the probes, and (iv)      providing a third temperature for denaturing all hybridization complexes<br>
Objects, features and advantages of the invention will be more clearly understood when taken together with the following detailed description which will be understood as being illustrative only, and the accompanying Figures<br>
DESCRIPTION OF THE FIGURES<br>
Fig 1 is an illustration showing a protocol for creating an embedded genetic ID<br>
Fig 2 is an illustration showing a restriction map for the CFTR region (X is an SNP marker, D7S122 and D7S8 are STRs, MET is methionine, NOT1 is restriction site and IRP is a gene)<br>
Fig 3 is an illustration showing mutations within exons of the CFTR gene<br>
Fig 4 is a DNA sequence showing polymorphic markers within exon 7 of the CFTR gene (capital letters indicate known polymorphisms)<br>
Fig  5a is a DNA sequence showing polymorphic markers within exon 10 of CFTR gene<br>
Fig 5b is an illustration of probes designed for SNP identification within exon 10<br>
of the CFTR gene Fig 6 is an illustration showing a protocol for PCR analysis with phosphorylated<br>
primers<br>
Fig 7 is an illustration showing a protocol of analyzing dystrophin gene deletions Most polymorphic marker-i are deletions in this gene   Primers can be designed to amplify flanking sequence of exons where deletions occur The   deleted   sequences   and   polymorphic   markers   can   be   identified simultaneously<br>
Fig 8 is an illustration showing a mitochondrial genome with genes on outside of the circle and various disease causing mutations and polymorphisms inside of the circle<br>
Fig 9a  is an illustration of probe design for STR length polymorphism with an anchor sequence using labeled ddNTPs<br>
Fig   9b  is  an  illustration  of results  with the THOl   locus  using  an  STR polymorphism<br>
Fig 30a is an illustration of probe design for an STR length polymorphism without an anchor sequence<br>
Fig 10b is an illustration ot results wan THOi locus using probes without an anchor<br>
Fig 11a is an illustration of on-chip STR length polymorphisms with two differentially labeled ddNTPs In this case, the ddNTPs are differentially labeled with different colors<br>
Fig 1 lb is an illustration of the results obtained with two differentially labeled ddNTPs where the green- colored ddNTP is incorporated with the correct match and the orange colored ddNIP is incorporated when the probe terminates w.thin the repeat<br>
Fig   12 is an  illustration of the effect of anchor length on SBE for STR polymorphism analysis using two anchors of six or eleven bases<br>
Fig 13 is an illustration of hybrid primers for STR polymorphism analysis<br>
Fig  14 is an illustration of the effect of annealing temperature on SBE for STR polymorphism analysis.<br>
Fig    15a is  an  illustration  of fluorescence energy  transfer with  an interior interrogation probe<br>
Fig    15b  is  an   illustration of fluorescence energy  transfer with  an exterior interrogation probe<br>
Fig 16 is an illustration of probe sequence design for the identification of poly-T variants in mtron 8 of the CFTR gene<br>
Fig   17 is an illustration of the results of identification of vanous targets using poly-T variants in mtron 8 of the CFTR gene<br>
Fig 18 is an illustration of the probe sequence design for the identification of longer repeats, such as those that are ten (10) to se\eral hundreds of bases long<br>
Fig 19a is an illustration of identification of repeat sequence by ligation of interrogation probes (Ti is the initial assay temperature and T^ is the final assay temperature, wherein T2 is greater than Ti)<br>
Fig 19b is an illustration of the identification of repeat sequence by ligation and on-chip cycling (Tt, T2 and T3 indicate three assay different temperatures, wherein Tj &gt; T2 &gt; Ti)<br>
DETAILED DESCRIPTION<br>
The genetic profiling of patients plays an increasingly important role, not only in basic ard applied clmical re&amp;earch, but also m the diagnose of disease and m the assessment ot predisposition to disease A safe, reliable genetic testing protocol preferably will incorporate all rele\ant information relating to patient identification within individual tests The present invention provides methods and compositions for linking the genetic profile obtained from the analysis of a patient's sample to a patient's identity   This correlation between a patient's genetic profile and identity is established<br>
concurrentJy with the genetic test or any diagnostic or prognostic test, on the basis of recording a genetic fingerprint or molecular identifier (ID)<br>
The methods of the present invention are useful m the prevention of mishandling, mislabeling and switching of samples in the course of genetic testing These methods are useful m paternity and maternity testing, immigration and inheritance disputes, zygosity testing in twins, tests for inbreeding, evaluation of the success of bone marrow transplants, identification of human remains, and forensic testing such as those involving semen or blood This invention prevents or corrects identification errors associated with mishandling, mislabeling and switching of samples by incorporating a genetic fingerprint or molecular identifier into the record of the genetic or other test obtained, for example m the form of an image as elaborated herein In this, way, an unambiguous link between that record and the patient's identity n established The molecular identifier may serve to track and to confirm the identity of the sample, thereby providing a means for authentication The methods ot the present invention provide compositions and methods to create a genetic ID, also referred to herein as an ID, concurrently with the completion of a genetic or other diagnostic or prognostic test In cases of analyzing genetic loci such as CFTR that contain multiple mutations and widely dispersed markers, the present invention provides the means of recording ID markers located within the targets already amplified for the purpose of genetic analysis     In analyses wherem the genetic loci<br>
contain only few  mutations,  ID markers located in other genomic regions can be amplified b\ providing additional primers<br>
One widely used method of genetic fingerprinting involves the analysis of polymorphisms in a number of repeated sequence elements within certain loci To facilitate the integration of repeated sequence polymorphisms, methods are provided herein for an array format of identity testing by STR/VNTR analysis The methods of this mvention minimize the number of steps required for sample handling and processing, thereby minimizing the possibility that the measurement process influences the results a potential concern in the development of databases for patient identification Although the genetic fingerprinting methods disclosed herein are particularly useful in providing patient identity in the context of genetic profiling, they also may be used in connection with other genetic analysis, such as genotypmg, haplotyping or HLA molecular typing. The methods of the present invention also find utility in the context of genetic authentication performed independently of genetic analysis<br>
Furthermore, by assigning an ID to a particular sample container or earner, the methods of the present invention create an unambiguous link between rhe carrier ID and the genetic ID, thereby not only minimizing the possibility of error in sample handling but also enabhng verification of assay results Carrier ID and genetic ID can be hnked to a database for data corroboration and authentication of patient identity<br>
The use of a set of DNA fragments of known sequence as external labels as described in this invention is advantageous over the pnor art in at least two respects First, the determination of the tag, and hence the identification of the tagged sample, can be performed concurrently with the genetic analysis of interest using methods such as hybridization, elongation and ligation Second, the resulting sample ID generally will be embedded in the image or data record produced by the genetic analysis That is, the sample ID and the results of the genetic analysis remain linked In contrast, tag identification by methods of the prior art generally require completion of a separate analytical procedure such as electrophoresis for the determination of DNA fragment lengths of the external labels, in addition to the genetic test itself<br>
As used herein, the term "polymorphism" refers to a sequence variation in a gene, and "mutation" refers to a sequence variation in a gene that is associated or believed to be associated with a phenotype The term "gene" refers to a segment of the genome that codes for a functional product protein control region Polymorphic markers used in accordance with the present invention for patiem identification may be located in coding or non-coding regions of the genome The term "patient," as used herein refers to an individual providing a test sample from which target nucleic acids are obtained for the purpose of genetic testing<br>
The terms "ID", "ID marker', and 'marker ' are used herein interchangeably to refer to internal markers and external markers whose high variability renders them suitable as a molecular identifier of a particular individual For the purposes of this invention, internal markers, specifically including genetic ID markers for DNA fingerprinting, can be "intrinsic" markers which are located within the gene of interest or can be non-intrinsic, or "extrinsic" markers<br>
Internal markers include SNPs, ST&amp;s, VNTRs and other polymorphic sites within a genomic locus Externa) markers include chemical, fluorescent, magnetic, molecular and other tags that generally can be incorporated info the sample at the time of sample collection They are useful tn uniquely associating the results obtained in a genetic or other test such as prognostic and diagnostic tests with the identity of the patient Examples of external markers include biological labels such as tags composed of oligonucleotides, peptide nucleic acids, DNA, RNA, proteins, ABO blood type and the like, and chemical labels such as optically active particles, dyes and the like<br>
A collection of random DNA sequence tags, such as those constructed for use in certain hybridization assays represent a class of external markers providing a very large encoding capacity For example, a recently published set of 164 sequence tags (http //waldo wi nut edu/pubhcations/SBE-TAGS), derived from the genome of bacteriophage X, would provide 2164 distinct combinations   In a preferred embodiment,<br>
the presence or absence in a given sample of each of the DNA sequence tags selected from a library is determined by provid-ng an identification array of color-encoded beads wherein beads of each type display an oligonucleotide probe that uniquely matches one of the sequence tags The set of signals from beads withm the identification array, averaged over beads of the same type, constitutes a molecular barcode of the patient sample<br>
Inorganic fluorescent nanoparticles synthesized by methods known to the art to emit light m response to single wavelength excitation at wavelengths determined by the particle size represent another class of exte *nal markers The presence in the sample of a specific subset of nanoparticle tags is determined by spectral analysis of the sample concurrently with the completion of the DNA analysis of interest Two types of inorganic nanoparticle labels have been described m prior art, namely semiconductor Q-dot (Quantum Dot) particles and RLS (Resonant Light Scattering) metal nanoparticles Quantum dots are nanometer (10" meter) scale particles made from semiconductor materials such as cadmium selenide (CdSe), cadmium tellunde (CdTe), or indium arsenide (InAs) Their composition and small size (a few hundred to a few thousand atoms) give these dots extraordinary optical properties that can be readily customized by changing the size or composition of the dots Quantum dots absorb light, then quickly re* emit the light at a different wavelength Although other organic dyes and inorganic materials exhibit this phenomenon, quantum dots have the advantage of being bright and<br>
non-bleachable with narrow, symmetry emission spectra, and have multiple resolvable colors that can be excited simultaneously using a single excitation wavelength (Bruchez et al, "Semiconductor Nanocrystals as Fluorescent Biological Labels, Science, 281, 2013 (1998), Ahvisatos, "Semiconductor Clusters, Nanocrystals, and Quantum Dots Science, 271, 933 (1996))<br>
Resonance Light Scattering Technology (RLS Technology) is based on "nano-sized" metal (for example gold or silver) colloidal particles that radiate energy in the form of scattered light when illuminated by a simple white light source The monochromatic light signal generated by a single RLS Particle is 104 to 106 times greater than the signal obtained from the most sensitive fluorophore and hence these nanoparticles can act as ultra-sensitive biological labels in a wide vanety of analytical bioassay and test formats (Yguerabide, J Analytical Biochemistry, 262, 137-156(1998), Yguerabide, J Analytical Biochemistry. 262, 157-176(1998))<br>
To use an external marker according to this invention, a sample such as a blood, sputum, hair or bone marrow sample is collected An external marker is incorporated as part of the sample and remains incorpoiated or otherwise associated with the sample while the sample undergoes processing and analysis Such markers can be incorporated m the content of assays involving population carrier screening, genotypmg, haplotyping or protein analysis such as profiling of Lytokmes, antigens, or antibodies in serum<br>
In certain embodiments, one or more external markers can be used in lieu of one or more internal markers In other embodiments, one or more external markers can be used in combination with one or more internal markers, thus providing an additional means for sample identification and authentication External markers also can be incorporated in an assay contained in a cartridge designed and provided for collection and/or storage of a patient sample, thereby creating a physical linkage between the sample, the assay container or earner and the genetic test<br>
The target sequence or target nucleic acid for genetic testing and for genetic fingerprinting may be a portion of a gene, a regulatory sequence, genomic DNA, cDNA, and RNA (including mRNA and rRNA) Genomic DNA samples are usually amplified before being brought into contact with a probe Genomic DNA can be obtained from any tissue source or circulating cells (other than pure red blood cells) For example, convenient sources of genomic DNA include whole blood, semen saliva, tears, urine, fecal material, sweat, buccal cells, skin and hair Amplification of genomic DNA containing a polymorphic site generates a single species of target nucleic acid if the individual from which the sample was obtained is homozygous at the polymorphic site, or two species of target molecules rf the individual is heterozygous RNA samples also are often subject to amplification In this case, amplification is typically preceded by reverse  transcription   Amplification of all  expressed  mRNA  can  be  performed  as<br>
probes in a pair are complementary to the target nucleic acid sequence except at the mutation site where at least one probe in ths pair is complementary to the target Probes are of sufficient length to allow hybridization to the target and are preferably 10 to 50 bases long, more preferably 10 to 20 bases long The probes also may be attached to a solid support by a linker Mutation analysis involves interrogation of one or more target sequences and may be performed in a multiplexed format that facilitates high-throughput screening For example, probes directed to different target sequences may be arranged m a planar, random array by attaching probes to encoded beads, as described herein In another example, different sub-arrays on the same substrate (e g , a silicon electrode) may be formed, with both bead encoding and location of bead subarray providing information about the identity of the probes located on individual beads As with the analysis of repeated sequences, parallel interrogation involves the formation of hybridization complexes between target and sequence-specific probes In a preferred embodiment, probe elongation provides the means for the direct labeling of elongation products by incorporation of fluorescently labeled dNTPs including, but not limited to methods that combine elongation and on-chip temperature cycling Labeled elongation products are detected by imaging as described herein<br>
In a preferred embodiment of this invention, sets of elongation probes capable of "pnming" the elongation reaction are immobilized on solid phase carriers in a way to preserve their identity and  to reduce ambiguities in  the identification of elongated<br>
described in, for example, WO 96/14839 and WO 97/01603 which are hereby incorporated by reference in their entirety Amplification of an RNA sample from a diploid sample can generate two species of target molecules if the individual providing the sample is heterozygous at a polymorphic site occurring within the expressed RNA, or possibly more if the species of the RNA is subjected to alternative splicing Amplification generally can be performed using the PCR methods known in the art Nucleic acids in a target sample can be labeled in the course of amplification by inclusion of one or more labeled nucleotides in the amplification mixture Labels also can be attached to amplification products after amplification (eg, by end-labeling) The amplification product can be RNA or DNA, depending on the enzyme and substrates used m the amplification reaction<br>
Tandem repeats, such as shorf tandem repeats (STRs) for example, can be used as genetic ID markers Frequently, genetic loci of interest to genotypmg and related genetic testing contain a repeated sequence composed of a number of repeated sequence elements that is highly variable between individuals In one aspect of the invention wherein the marker is a tandem repeat, the number of tandem repeats in a target nucleic acid sequence may be determined by parallel interrogation as elaborated herein<br>
In one preferred embodiment, in order to score mutations in the course of genetic testing probe pairs generally are designed for each variable target sue of interest     Both<br>
products   For example, this can be achieved by spatially separating different probes and/or by chemically encoding probe identities<br>
When the probes are contacted with target strands under conditions permitting formation of hybridization complexes, the interrogation site located at or near the 3' terminus of each probe will align with the target generally in one of two configurations, namely either in a "rep eat-in tenor' configuration or in a "repeat-ex ten or" configuration In the former, the intenogation sue is juxtaposed to a site within the target's repeated sequence, m the latter, it is juxtaposed to a site within the target's leading sequence<br>
Probes are usually designed with complementary trailing and leading sequences to the desired genomic regions To make universal trailing and leading sequences, hybnd pnmers can be designed with sequences that will be part of the amplified product For example, forward and reverse PCR pnmers can be designed to contain at their respective 5' ends a GC rich tag PCR amplification of genomic DNA introduces this tag at the 51 end oi the product (Fig 13) Oligonucleotide probes are then designed to contain a 5' anchor sequence that is complementary to the pnmer tag This design flexibility facilitates correct probe alignment to the amplified target to minimize slippage (i e the target hybridizing at several different places) and enhances discnmination in assays For example, the trailing sequence is designed in such a way to start with a nucleotide that is<br>
not present in the repeat sequence (for di- and tn-nucleotide repeats) and can be detected by single base extension<br>
The step of interrogating the repeated sequence of a target by a set of interrogation probes containing variable numbers of repeats is designed to assign to each probe one of two values corresponding to one of these two configurations, namely matched, (numerically represented by 1), or non-matched, (numerically represented by 0) The binary sequence of interrogation results produced from a set of probes directed against a repeated sequence polymorphism determines the number of target repeats within that polymorphism As elaborated herein in numerous examples many variations of probe construction and detection methods are possible including, but not limited to, direct hybridization with detection by way of fluorescence energy transfer, and template mediated probe elongation including single base extension or ligation<br>
In certain embodiments, the probe sequence may be constructed to contain an offset sequence, located on the 5' side of the first probe repeat, with the offset sequence matching the first one or more (but not all) nucleotides of the target repeat Annealing of an offset-containing probe to the target in perfect alignment again produces a hybridization complex in one of two possible configurations In the first of these, probe repeats are displaced relative to target repeats by an amount equal to the size of the offset in the probe's 3' direction, so that the 3' terminus of the probe sequence is aligned with an<br>
interior position of one of the target repeats In the second configuration, the 3' terminus of the probe sequence is aligned with a position within the leading sequence, that position being determined by the size of the offset<br>
For each repeated sequence polymorphism of interest, a set of probes can be constructed to contain the same (optional) anchor sequence and the same (optional) offset sequence, but successively higher numbers of repeats, such that the set of probes spans the range of possible target repeats as elaborated herein In one embodiment, probes can be designed to contain 5'- and 3'-anchor sequences that are complementary to the target's trailing and leading sequences so as to stabilize desirable alignments of probe and target Leading and trailing sequences of desired composition may be introduced at the point of amplification of a patient's DNA by PCR methods known in the art (Innis et al, Academic Press, San Diego, CA (1990), Mattila et al , Nucleic Acid Research, 4967 (1991)) Leading and trailing sequences can be switched to determine the repeats on the complementary DNA strand<br>
In other embodiments, oligonucleotide probes containing neither 5'- nor 3'-anchor sequences can be used in competitive hybndization In one such embodiment, a labeled or unlabeled target is first permitted to form a hybndization complex with a "blocking" DNA strand designed to contain a sufficiently large number of repeats so as to exceed the largest expected number of target repeats In this hybridization complex, all<br>
target repeats are therefore in a duplex configuration A solution of this "blocked" target is now placed in contact with an array of encoded beads displaying probes with varying numbers of repeats The reaction mixture is incubated to permit probes to compete with the blocking strand for binding to the target repeats In this design, only those probes that contain a number of repeats matching or exceeding the number of target repeats will displace (a portion oO the blocking strand from the target and thereb) acquire assay signal Washing at increasing stringency, as well as adjustment of annealing temperature as described herein, enhances the level of discrimination<br>
In another preferred embodiment, assay images are recorded at increasing temperatures and signals are recorded as a function of increasing temperature Successive peaks in the intensity-versus-temperature plots for all probe-target hybridization complexes indicate the respective melting temperatures, with probes containing a number of repeats that match or exceed the number of target repeats having the highest melting temperature Differently colored labeled targets can be analyzed within the same reaction<br>
A target that forms a hybridization complex with immobilized probes can be visualized by using detection methods previously described herein For example, probes annealed to target strands can be elongated with labeled dNTPs, such that extension occurs when the probe perfectly matches the number of repeats in the target Several other configurations for generating positive assay signals may be readily constructed<br>
As described for sequence-specific probes in general, probes for parallel interrogation repeated sequences may be immobilized on solid supports via a linker moiety, use of which is well known in the art As a general rule, probes should be sufficiently long to avoid annealing to unrelated DNA target sequences The length of the probe may be about 10 to 50 bases more preferably about 15 to 25 bases, and even more preferably 18 to 20 bases In a multiplexed assay, one or more solution-borne targets are then allowed to contact a multiplicity of immobilized probes under conditions permitting annealing and elongation reactions Thus, the present invention offers advantages over the existing methods of analyzing repeated sequence polymorphisms by gel electrophoresis, a methodology which is not adaptable to high throughput operation<br>
The present invention also includes methods for the parallel interrogation of single nucleotide polymorphisms and single site mutations as well as for detecting other types of mutations and polymorphisms such as multiple nucleotide polymorphisms (for example double, triple and the like), as well as small insertions and deletions commonly observed and useful for genetic testing<br>
To minimize labor, matenals, and time to completion required for analysis, it is desirable to amplify and to analyze multiple loci simultaneously in a single reaction Multiplexed amplification methods aie particularly useful in the analysis of genetic<br>
diseases including, among others, Duchenne Muscular Dystrophy, Lesch-Nyhan Syndrome, and Cystic Fibrosis In addition, several autoimmune diseases (such as Diabetes melhtus) have been linked to polymorphisms m the Human Leukocyte Antigen (HLA) system The polymorphic loci of the HLA complex exhibit strong linkage disequtlibnum, such that particular haplotypes occur together on the chromosome more often than would be predicted Susceptibility to insulin-dependent diabetes melhtus (IDDM) has been found to be associated with particular class II alleles encoded by DQ loci The loci DQ alpha are used for molecular typing and can be used for simultaneous analysis involving disease sequencing and molecular typing<br>
Genetically defined mitochondrial diseases, most of which are caused by mutations m mitochondrial (mt) DNA, provide another example The average human celt contains thousands of mtDNA molecules, which are closed circular molecules of 16,586 nucleotide pairs that code for 37 genes, 13 oxidative phosphorylation (OXPHOS) polypeptides, rRNAs and 22 tRNAs The mtDNA molecules have a much higher mutation rate than does the nuclear genome<br>
Several of mtDNA mutations have been linked to degenerative diseases of brain, heart, skeletal muscle and kidney (Fig 8) Mitochondrial encephalomyopathies form a genetically heterogeneous group of disorders associated with impaired oxidative phosphorylation  Patients may exhibit a wide range of clinical symptoms from muscle<br>
weakness to vision loss and brain degenerative disorders, for which there currently is no curative treatment Base substitutions and rearrangement mutations occurring in the mitochondnal genome are correlated with ocular myopathies, Kearn-Sayre or Pearson syndromes and adult-onset diabetes melhtus Mitochondrial diseases result from mutations m the female germhne or acquired mutations Base substitution mutations in ATP synthase have been associated with muscle weakness, ataxia, retimtus pigmentosa (NARP), Leigh syndrome, central vision loss (LHON) dystonia, and MELAS Three mtDNA mutations have been identified to be associated with Alzheimer's and Parkinson's diseases<br>
Several polymorphic markers have been identified in mtDNA and have been used extensively in studies of population genetics Most of these markers are located within, and can be co-amphfied with, genes containing disease-causing mutations Specifically, 32 sequence polymorphisms, located within the tRNA gene, are suitable as individual ID market's which can be embedded withm the mutation profile (Garboczi, et a!, Mol Cell BioChem 107 21-29 0991))<br>
The methods of the present invention involve the concurrent interrogation of target nucleic acid sequences as well as genetic ID markers This can be accomplished by providing two or more sets of nucleic acid probes, such as DNA or RNA in single-stranded     or  double-stranded  form,  or nucleic-acid  like   structures   with   synthetic<br>
backbones such as peptide nucleic ac ds According to the invention, the first set of probes is designed to be complementary to a target nucleic acid sequence of interest and the second set of probes is designed to be complementary to one or more designated ID markers These first and second sets of probes can be attached to solid phase earners such as, for example, a chip, wafer, slide, membrane, particle, bead, or any surface which would be compatible with the assay considered<br>
As used herein, the terms ' bead," "microsphere," "microparticle," and "particle" are used interchangeably Bead composition may include, but is not limited to, plastics, ceramics, glass, polystyrene, methylstyrene, acrylic polymers, paramagnetic materials, carbon graphite, titanium dioxide, latex or cross-linked dextrans such as sepharose, cellulose, nylon, cross-linked micelles and polytetrafluoroethylene<br>
Beads may be associated with a physically or chemically distinguishable characteristic For example, beads may be stained with sets of optically distinguishable tags, such as those containing one or more fiuorophorc or chromophore dyes distinguishable by excitation wavelength, emission wavelength, excited-state lifetime or emission intensity Optically distingu shabie dyes combined in certain molar ratios may be used to stain beads in accordance with methods known in the art Combinatorial color codes for exterior and interior surfaces are disclosed in International Application No PCT/US98/10719, incorporated herein by reference    Beads capable of being identified<br>
on the basis of a physically or chemically distinguishable characteristic are said to be "encoded "<br>
The detection of the chemically or physically distinguishable characteristic of each set of beads and the identification of optical signatures on such beads generated in the course of a genetic or other test (such as diagnostic or prognostic test) using such beads may be performed by respectively recording a decoding image and an assay image of a set or array of such beads and comparing the two images For example, m certain embodiments, a system with an imaging detector and computerized image capture and analysis apparatus may be used The decoding image is obtained to determine the chemical and/nr physical distinguishable characteristic that uniquely identifies the probe displayed on the bead surface In this way, the identity of the probe on each particle in the array is provided by the distinguishable characteristic The assay image of the array is obtained to detect an optical signature produced in the assay as elaborated herein below<br>
In addition to being encoded, beads having specific oligonucleotide probes or pnmers may be spatially separated in a manner such that the bead location provides information about bead and hence about probe or primer identity In one example, spatial encoding may be provided by placing beads in two or more spatially separate subarrays<br>
In a preferred embodiment, beads can be arranged in a planar array on a substrate<br>
before decoding and analysis Bead arrays may be prepared by the methods disclosed in<br>
PCT/USO1/20179, incorporated herein by reference in its entirety Bead arrays also may<br>
be formed using the methods described in US Patent No 6,251,691, incorporated herein<br>
by reference in its entirety For example, light-controlled electrokmetic forces may be<br>
used to assemble an array of beads in a process known as "LEAPS", as described m US<br>
Patent No 6,251,691 Alternatively, if paramagnetic beads are used, arrays may be<br>
formed on a substrate surface by applying a magnetic field perpendicular to the surface<br>
Bead arrays also may be formed by mechanically depositing the beads into an array of<br>
restraining structures (e g, recesses) at the surface of the substrate	In certain<br>
embodiments, the bead arrays may be immobilized after they are formed by using physical means, such as, for example, by embedding the beads&gt; in a gel to form a gel-particle film<br>
An example of multiplexed molecular analysis using random encoded bead arrays is provided by genetic analysis and testing of ABO (and RH) blood type In this embodiment, encoded bead arrays are assembled in separate locations on a given chip to permit concurrent genetic analysis and testing This analysis is performed by displaying, on encoded beads, individual antigens corresponding to ABO-type and RH-factor and assembling these beads into an array which is then used to determine the antibody profile in the patient's serum in a multiplexed immunoassay<br>
ABO blood typing is based on the fact that humans and most vertebrates carry complex oligosaccharides attached to serine side chains of certain membrane proteins Oxygen-linked polysacchandes complexes frequently are exposed on the outer surface of human cells and elicit a specific immune response when the cell carrying them is injected into individuals that do not contain the same cell surface antigens This pattern of adverse immune response in mismatched individuals forms the basis of ABO-blood group classification, wherein individuals of the same blood type can accept blood transfusions from one another, whereas individuals with different blood types cannot Additional blood types are defined on the basis of compatibility or incompatibility with rhesus factors and serves to correlate blood type with distinct individual genetic fingerprint<br>
Blood type reflects the expression of two genes that determine A and B blood type The A-gene encodes a glycosyItransferase that catalyzes the addition of a terminal N-acetyl-galactosamine (Gal Nac) residue to a core polysaccharide, while the B gene encodes a similar enzyme that adds a galactose (Gal) residue to the same site When A and B genes are present, both structures are found, but when only O genes are present, the site on the oligosaccharide is left exposed Cells of an AA or AO individual carry A-antigen, cells of a BB or BO individual carry B antigen, while cells of an AB individual carry both A and B antigens and cells of an OO individual carry neither antigen<br>
Thus, biochemical markers which form part of the patient's medical record, such as the set of cell surface antigens that define the blood type of an individual, can be used to link a genetic profile to a patient's identity This information can be obtained by on-chip genetic testing and can be linked to a concurrently recorded biochemical CD marker which in turn can be cross-referenced v. ith existing patient records to ensure authenticity<br>
According to the methods of this invention, the concurrent interrogation of a target nucleic acid sequence and a genetic fingerprint can be performed by first identifying a genetic ID marker or plurality of markers of interest, as elaborated in Examples herein, and then designing a plurality of oligonucleotide probes a first set directed to target nucleic acid sequences of interest, and a second set directed to the marker or markers of interest That is, the first set of probes is used in an assay designed for genetic testing or profiling and the second set of probes is used in the determination of a molecular fingerprint, concurrently with the genetic testing When intrinsic markers are used, the intrinsic markers are coamplified with other designated mutations of polymorphisms in a target sequence When extrinsic markers are used, separate primer sets for separate but contempoianeous amplification may be required<br>
Interrogation of an amplified target nucleic acid sequence containing polymorphic sites or plurality of such sequences for genetic profiling as well as genetic ID markers<br>
involves forming a hybridization complex by annealing targets to encoded, sequence-specific oligonucleotide probes to determine the degree of sequence complementarity between probes and target fragments<br>
Interrogation of target genes and markers may be accomplished in various ways within the scope of the invention For example, m one embodiment (referred to herein as "direct hybridization"), fluorescently labeled target fragments may be used to impart a detectable optical signature, such as fluorescence, to the hybndized probe-target complex A labeled target is readily produced by prior target amplification with labeled pnmers In another embodiment, direct hybridization can employ a labeled detection probe For example, fluorescence energy transfer may be used to create a detectable optical signature via the formation of three-member hybridization complexes, as elaborated in the Examples provided herein In a further embodiment, the target sequence composition can be determined from the signal pattern of enzyme-mediated ligation and elongation reactions applied to the probe sets provided In elongation-mediated detection, a polymerase-mediated elongation reaction produces a signal pattern reflecting the elongation or extension of sequence-specific probes that are designed to function as pnmers Signal patterns generated by any of these methods of the present invention contain a unique individual genotype along with a "fingerprint "<br>
As used herein, "hybridization" refers to the binding, annealing, duplexing, or hybridizing of a first nucleic acid molecule preferentially to a particular second nucleotide molecule The stability of a hybridization complex vanes with sequence composition, length and external conditions Hybridization methods include those that rely on the control of stnngency in reaction conditions to destabilize some but not all hybridization complexes formed in a mixture Using these methods, it is possible to distinguish complete complementarity from partial complementarity between probe and target sequences that form a hybridization complex<br>
To facilitate detection, hybridization complexes can be modified to contain one or more labels These labels can be incorporated by any of a number of means well known to those skilled in the art Detectable labels suitable for use in the present invention include any composition detectable by spectroscopic, photochemical, biochemical, immunochemical, electrical, optical, or chemical means Useful labels m the present invention include high affinity binding labels such as biotirt for staining with labeled streptavidm or its conjugate, magnetic beads, fluorescent dyes (for example, fluorescein, Texas red, rhodamine, green fluorescent protein, and the like), radiolabels (for example 3H, 125I, 35S, 14C, or P), enzymes (for example horseradish peroxidase, alkaline phosphatase and others commonly used in an ELISA), epitope labels, and colonmetnc labels such as colloidal gold, colored glass or plastic beads (for example polystyrene, polypropylene, latex, and the like)   Means of detecting such labels are well known to<br>
those of skill in the art Thus, for example, radiolabels can be detected using photographic film or scintillation counters, and fluorescent markers can be detected using a photodetector to detect emitted light Enzymatic labels are typically detected by providing the enzyme with a substrate and detecting the reaction product produced by the action of the enzyme on the substrate, and colonmetnc labels are detected by simply visualizing the colored label One method uses colloidal gold as a label that can be detected by measuring light scattered from the gold The label can be added to the amplification products prior to or after the hybridization<br>
"Direct labels" are detectable labels that are directly attached to, or incorporated into, the nucleic acids prior to hybridization In contrast, "indirect labels" are affixed to, or incorporated into the hybridization complex following hybridization Often, the indirect label is attached to a binding moiety that has been attached to the amplified nucleic acid prior to hybridization Thus, tor example, the amplified nucleic acid can be biotmylated before hybridization After hybridization, an avidin or streptavidin conjugated fluorophore will bind the biotin-bearing hybnd duplexes, providing a label that is easily detected<br>
Means for detecting labeled nucleic acids hybridized to probes in an array are known to those skilled in the art For example, when a colonmetnc label is used, simple visualization of the label is sufficient When tadiolabeled probes are used, detection of<br>
the radiation (for example, with photographic film or a solid state detector) is sufficient Detection of fluorescently labeled target nucleic acids can be accomplished by means of fluorescence microscopy An array of hybridization complexes can be excited with a light source at the excitation wavelength of the particular fluorescent label of choice and the resulting fluorescence at the emission wavelength detected The excitation light source can be, for example, a laser appropriate for the excitation of the fluorescent label<br>
In a preferred embodiment, the interrogation step involves the elongation of target-annealed probes This reaction, catalyzed by a polymerase, produces an elongated hybridization complex by appending to the probe sequence one or more nucleoside triphosphates m an order reflecting die composition of the target sequence in the existing hybridization complex In order for this elongation reaction to proceed, the probe length must contain a terminal elongation initiation ("TEI") sequence The TEI sequence in turn contains an interrogation site which preferably coincides with the 3' terminus but also may be displaced from the 3' terminus by 3-4 bases within the pnmer sequence Elongation proceeds if the composition, of the interrogation site matches that of the designated target site<br>
A method of extending the recessed 3' end of a double-stranded DNA by addition of selected deoxynucleoside triphosphates (dNTPs) in order to copy the protruding single complementary strand and to determine the specificity of the reaction has been reported<br>
in the art (Wu, Proc Natl Acad Sci, 57(1) 170-171 (1967), Wu, J  Mol Biol, 14 35(3) 523-37 (1968))    They incorporated one dNTP at a time, trying up to four dNTPs per position and using radioactively labeled dNTP's to detect successful incorporation<br>
In contrast to the pnor art, the present invention provides a parallel format of repeated sequence analysis in which all extension reactions occur simultaneously on multiple copies of double stranded DNA formed by site-specific interrogation probes and a target In some embodiments of this invention, to facilitate the parallel detection of successful extensions, optical signatures produced by the extension reaction are imaged<br>
In one embodiment of the invention, two or more probes may be provided for interrogation of a specific designated site, these probes being constructed to anticipate polymorphisms or mutations at the interrogation site and non-designated polymorphic sites within a certain range of proximity of the designated polymorphic site In a preferred embodiment, this multiplicity of probe sequences contains at least one probe that matches the specific target sequence m all positions within the range of proximity to ensure elongation<br>
Furthermore, in some embodiments of this invention, a covenng probe set is used. A covering probe set, (described in US Provisional Patent Application 60/364,416 which is hereby incorporated by reference m its entirety) contains probes permitting the<br>
concurrent interrogation of a given multiplicity of designated polymorphic sites within a nucleic acid sequence and comprises, for each site, at least one probe capable of annealing to the target so as to permit, on the basis of a subsequent elongation reaction, assignment of one of two possible values, "matched" (elongation) or "unmatched" (no elongation} to that site<br>
The covenng probe set associated with each designated site may contain two or more probes differing m one or more positions In certain embodiments, the probe sequence may contain universal nucleotides capable of forming a base-pan with any of the nucleotides encountered in DNA In certain embodiments, probes may be attached to encoded microparticles, and specifically, two or more different types of probes in a covenng set may be attached to the same type of mi crop article The process of attaching two or more different types of probes to a bead is referred to as "probe pooling"<br>
A mismatch in a single position within the TEI region, or a mismatch m three or more positions within the duplex anchoring ("DA") region (1 e , annealing subsequence) suffices to preclude elongation Accordingly, the elongation of two probes displaying such differences in composition generally will produce distinct elongation patterns All such probes can be multiplexed m a parallel elongation reaction as long as they are segregated, that is, individually encoded<br>
Probes displaying identical TE1 subsequences and displaying DA subsequences differing in not more than two positions generally will produce elongation reactions at a yield (and hence signal intensity) either comparable to, or lower than that of a perfect match In the first case, indicating tolerance of the mismatch, the set of alleles matched by the probe in question will be expanded to include those alleles displaying the tolerated mismatched sequence configurations within the DA region In the second case, indicating only partial tolerance, three approaches are descnbed herein to further elucidate the allele matching pattern In the first approach, those probes displaying one or two nucleotide polymorphisms m their respective DA. regions are included in the covering set, and information regarding the target sequence is obtained by quantitatively comparing the signal intensities produced by the different probes within the covering set In the second approach, probes comprising separate PEI and DA regions joined by a tether are used to place the DA region farther away from the TE1 region in order to avoid target polymorphisms In the third approach, probes aie optionally pooled in such cases offering only a modest expansion of the set of matched alleles<br>
While this method of accommodating or identifying non-designated polymorphic sites is especially useful in connection with the multiplexed elongation of sequence specific probes, it also may be used in mini-sequencing reactions (see e g , Pastinen, et al Genome Res 7 606-614 (1997), incorporated herein by reference)<br>
In certain embodiments, the polymerase catalyzing primer elongation is a DNA polymerase lacking 3' to 5' exonuclease activity Examples of such polymerases include T7 DNA polymerase, T4 DNA polymerase, ThermoSequenase and Taq polymerase A reverse transcriptase may be used when the target comprises an RNA sequence In addition to polymerase, nucleoside triphosphates can be added, preferably all four bases For example dNTPs or analogues may be added In certain other embodiments, ddNTPs may be added<br>
Successful probe extension may be indicated by a change in the optical signature of the solid phase earners associated with the extended primers This is accomplished by direct or indirect labeling methods well known in the art (For a detailed review of methods of labeling nucleic acids and detecting labeled hybridized nucleic acids, see Laboratory Techniques in Biochemistry and Molecular Biology, Vol 24, P Tigson Ed , Elsevier, NY (1993))<br>
In direct labeling, the extension reaction produces a product with a corresponding optical signature In certain embod'mems, fluorophore or chromophore dyes may be attached to one or more of the nucleotides added during extension so that the elongated pnmei acquires a charactenstic optical signature Successful extension has previously been described which involves the use of labeled deoxynucleoside triphosphates (dNTPs)<br>
such as Cye3-dUTP or dideoxynucleoside triphosphates (ddNTPs)   (Wu,   1968, see above)<br>
In indirect labeling, an optical signature is produced in additional steps performed subsequent to the elongation reaction This invention also provides novel methods of providing optical signatures for detecting successful extension reactions, thus eliminating the need for labeled dNTPs or ddNTPs, which is advantageous because the efficiency of available polymerases in accommodating dNTPs or ddNTPs is reduced if the dNTPs or ddNTPs are labeled<br>
The methods of the present invention further include the formation of three-member hybridization complexes and the r application to the parallel interrogation of tandem repeats, including, but not limited to methods that combine ligation and on-chip temperature cycling<br>
In one embodiment of the invention, the interrogation step utilizes the formation of a three-member hybridization complex In addition to the target and the interrogating probe, this complex contains a modifying probe that is annealed to the target in a position immediately adjacent to the annealed interrogation probe Optionally, the modifying probe may be ligated to the mterrogat-ng probe<br>
In one embodiment, a three-member hybridization complex is formed that includes an interrogating probe and a read-out probe that are designed to be a fluorescence energy transfer pair The read-out probe sequence matches a selected subsequence of the target, covering one or more repeats adjacent to the 3' terminus of the target-annealed interrogation probe In other words, the three-member hybridization complex will form only when the 3' terminus of the interrogation probe is aligned with a target site interior to the repeated sequence polymorphism To permit the interrogation probe and the read-out probe within the three-member hybridization complex to form a fluorescence energy transfer pair, the interrogation probe is constructed to contain at or near its 3' terminus, a first fluorophore, referred to as the donor, and the read-out probe is constructed to contain, at or near its 5' terminus, a second fluorophore, referred to as the acceptor When the donor and acceptor are annealed to the target, a gap of generally not more than 3-4 nucleotides separates the donor from the acceptor Under these conditions, fluorescence energy transfer from donor to acceptor occurs when the three-member hybridization complex forms but does not occur when the three-member hybridization complex is not formed<br>
An example of fluorescence energy transfer is shown in Figs 15a and 15b Two sets of oligonucleotide probes were designed to be complementary to a sequence in the amplified products just before the repeated sequence In this design, a three-member hybridization complex is formed in such as way as to place only the interrogation probe<br>
containing the correct number of repeats into immediate proximity of the read-out probe, thereby permitting fluorescence energy transfer from donor to acceptor<br>
In this way, the "repeat-interior termination" and "repeat-ex tenor termination" configurations of the hybridization complex formed by interrogation probe and target are readily distinguished m a parallel interrogation assay using dual color detection The former permits formation of a three-member hybridization complex and thus permits fluorescence energy transfer so that illumination at the excitation wavelength of the donor produces fluorescence at the emission wavelength of the acceptor In contrast, the latter does not permit formation of the three-member hybridization complex so that illumination at the excitation wavelength of the donor produces fluorescence at the emission wavelength of the donor<br>
The present method of interrogation, when using read-out probes covering multiple target sequence repeats, is particularly useful in the analysis of long repeated sequence polymorphisms which do not require single-repeat resolution An example is provided by Huntington's gene, which contains a polymorphic stretch of CAG trinucleotide repeats at the start of the protem-coding sequence The disease is caused by an abnormally large expansion of this repeat m one copy of the gene The present invention can be used to identify the repeat expansion m patient samples<br>
Another embodiment involves the formation, at a first temperature, Ti, of a three-member hybridization complex with no gap between the interrogation probe and read-out probe, the latter of which is designed to contain, at or near its 3' terminus, a fluorescent dye Subsequent to forming the three-member complex, the read-out probe is hgated to the interrogation probe at a second temperature, T2, and the hybridization complex is denatured at a third temperature, T3 &gt; Tm (where Tm is the melting temperature), to leave a fluorescent strand indicating that the read-out probe terminated in the interior of the repeat secuon of the original hybridization complex This method can be generalized to a dual-color detection format in which a second read-out probe with a different fluorescent dye anneals to the sequence in a region exterior to the repeated sequence pol&gt;morphism (Figure 19)<br>
This method of interrogation permits linear on-chip amplification by repeatedly cycling between temperatures T|, T2 and T3, each target sequence thereby serving as a template in the formation of a ligation product<br>
To the extent that the method for STR analysis, as disclosed in the present invention, is a general method for the analysis of repeated sequence elements and other polymorphisms in nucleic acids, it can be applied to the analysis of DNA from other than human origin<br>
For example, microbial DNA (such as bacterial DNA) can be analyzed to perform strain typing including the identification of drug resistant strains and to guide the selection of antibiotic treatments DNA markers such as restriction length polymorphisms (RFLPs) and polymorphic tandem repeats (STRs, VNTRs) have been used for bacterial and yeast strain identification This is particularly helpful when strains cannot be discriminated or distinguished by morphological and biochemical markers alone A specific application is the strain typing of Bacillus anthracis (Anthrax) which is one of the most monomorphic bacterial species known Five known strains have been identified on the basis of variable number of tandem repeats in the variable region of the vrrk gene<br>
Another application of the STR analysis method of the present invention is tn the context of the identification and selection of specific genetic varieties displaying desirable traits For example, these genetic markers are used to tag interesting traits determined by uncharactenzed genetic factors with a closely linked well-defined polymorphic locus Several classes of DNA markers that have been used for purposes of marker assisted selection (MAS), identification and plant variety protection, plant breeding, and fingerprinting include smgle site alterations (such as SNPs), as well as smgle and multi-locus repeat markers such as VNTRs, STRs, and simple sequence repeats (SSRs)<br>
The present invention will be better understood from the following Examples However, one skilled in the art wilt readily appreciate that the specific methods and results discussed herein are merely illustrative of the invention described in the claims which follow thereafter<br>
EXAMPLES<br>
Example 1: Cystic Fibrosis Mutation Profile with Embedded Genetic Fingerprint<br>
In this Example, the analysis of mutations in the CFTR gene was performed so as to produce a genetic profile with an embedded panel of genetic identifiers as internal markers (Fig 1) Most CF mutations of interest are located on exons 3 to 21 (Fig 2 and Table 1) Given that the frequency of polymorphic markers within the CFTR gene does not differ significantly for CF patients as compared to the total population, the intenogation of these markers will not produce bias in the analysis of mutant versus normal chromosomes<br>
The choice of suitable primers ensures that sequence polymorphisms located withm the CFTR gene are simultaneously amplified along with designated mutations to be probed by subsequent hybridization  In a preferred embodiment involving an array<br>
composed of encoded beads, one set of beads is modified with probes that are complementary to the CFTR mutations of interest and a second set of beads is modified with piobes that are complementary to the selected intrinsic polymorphic ID markers An aliquot of (amplified) patient sample is placed on the assembled bead array to generate and to record, in a single step, a genetic profile with embedded genetic ID In a preferred embodiment involving random encoded bead array, the following are typical assay conditions<br>
Beads stained with different combinations of two fluorescent dyes were functionahzed with neutravidin and biotmylated oligonucleotide probes, the latter step performed in TBS (10 mM Tris HC1, 05 M NaCl w/v) for 45 minutes at room temperature The target was amplified using a forward primer 5'-labeled with Cy5 and a reverse pnmer modified with a phosphate group Pnmer sequences were designed to be homologous either with mtron sequences immediately flanking specific exons or with the sequences marking the beginning or end of certain exons The use of such primers permits PCR amplification of the target gene concurrent with that of polymorphic markers located within the loci delineated by the chosen primers<br>
EXON 7 polymorphisms When exon 7 containing three CF mutations (334W, 347P and 1078T), is amplified with a set of primers flanking its ends, sequence polymorphisms at positions dbSNP I00083 C/T, dbSNP 100084 C/G, dbSNP 100085<br>
A/G, dbSNP 1799834 C/G, dbSNP 1800086 C/G, dbSNP 1800087 A/G are amplified as well (Figs 3 and 4) These polymorphisms represent intrinsic ID markers that are interrogated by hybridization to a corresponding set of oligonucleotide probes on encoded beads within a random array<br>
EXQN 10 polymorphisms The most common CF mutation, delta 508, is located on Exon 10 SNPs located on exon 10 including dbSNP 100089 C/T, dbSNP 100090 C/T, dbSNP 213950 A/G, dbSNP 100092 C/G, dbSNP 1801178 A7G dbSNP 180093 GAT, dbSNP 180094 A/G and dbSNP 1900095 G/T (Fig 5) can be amplified along with the mutation and interrogated by hybridization to a corresponding set of oligonucleotides on encoded beads within random array<br>
Genomic DNA extracted from patient samples was amplified using a set of pnmers in a multiplex PCR (mPCR) reaction A preferred embodiment of a mPCR protocol for cystic fibrosis analysis (L McCurdy, Thesis, Mount Sinai School of Medicine, 2000, which is hereby incorporated by reference) is as follows Multiplex PCR was performed using chimeric primers tagged, at their respective 5' ends, with a universal sequence to narrow the range of respective melting temperatures PCR pnmers were synthesized to contain a Cy5 or Cy55 (Amersham) label on the 51 end Using a Perkm Elmer 9600 thermal cycler, 28 cycles of amplification were performed, each cycle consisting of a 10 second denaturation step at 94 Â°C with a 48 second ramp, a 10 second<br>
annealing step at 60 Â°C with a 36 second ramp and a 40 second extension step at 72 Â°C with a 38 second ramp Each reaction mixture of 50 Âµl contained 500ng of genomic DNA, IX PCR buffer (10 mM Tns HC1, 50 mM KC1, 0 1% Tnton X-100), 1 5 mM MgCl2, 200 uM each of PCR grade dKIPs and 5 units Taq DNA polymerase Optimal primer concentrations were determined for each pnmer pair<br>
Following amplification, products were purified to remove all reagents and the DNA concentration was determined by spectrophotometnc analysis To generate a single-stranded target, PCR products (200 ng) were incubated with 2 5 units of (lambda) exonuclease in IX buffer ax 37Â°C for 20 mm and inactivated at 75Â°C for 10 mm Under these conditions, the enzyme digests one strand of duplex DNA from the 5'-phosphorylated end and releases 5' phosphomononucleotides (Fig 6) The products were used directly in a hybridisation rmx'ure (2 2 M TMAC, 0 15%SDS, 3 mM EDTA;<br>
For each hybridization reaction, 5 Âµl of hybridization mixture were placed on the surface of silicon chips, each chip cairymg an array of encoded beads displaying oligonucleotide probes Chips were placed in a covered dish and placed on a shaking surface (-200 rpm) in an incubator at 55Â°C for 15 minutes Chips were washed by flushing the array three times with IX TMAC buffer The Cy5 fluorescence signal from each bead within the array was recorded and analyzed using a fluorescence microscope, CCD camera and image analysis software to determine the mutation profile<br>
Example 2- Duchenne Muscular Dystrophy    Mutation Profile    with Embedded Genetic Fingerprint<br>
This Example illustrates the design of a genetic ID for Duchenne muscular dystrophy (DMD), an X-hnked recessive trait mostly occurring in males that is characterized by progressive loss of muscle strength Although DMD protein (dystrophin) analysis of muscle provides an accurate diagnostic test, it is invasive and cames high cost and risk Further, because the protein is not expressed in amniotic fluid or in chorionic villus tissue, the protein test is not suitable for prenatal diagnosis On the other hand, the DMD gene has been cloned and several deletions have been identified (Fig 7), with Southern blots requiring hybridization with ten cDNA probes (Kunkel, et al, Nature, 322 73-77 (1986))<br>
A multiplex PCR protocol has been described for the simultaneous analysis of these gene deletions (Monaco et al. Nature, 323 646-650 (1986)) \n intrinsic genetic ID can be derived from a set of several dmucleotide repeat polymorphisms located at many sites within the DMD gene Fig 7 illustrates a protocol for the analysis of gene deletions and polymorphic ID markers within the DMD gene PCR pnmers are designed to flank exon sequences such that the presence of a deletion within the flanked sequence blocks target amplification, leading to a null result in subsequent interrogation<br>
Example .3 - Matching Genetic Profiles to Genetic Fingerprint Records<br>
Given a genetic fingerprint recorded by known methods of DNA fingerprinting, such as, for example, the methods used as part of neonatal testing or those applied to entire selected populations (eg, members of the defense forces or prison inmates), the methods of the present invention provide a means for authenticating the genetic fingerprint concurrently with genetic testing For example, markers, such as STRs, that are recorded for paternity and forensic analysis could be used In addition, markers derived from SNP genotypmg or haplotypmg could be used Consequently, a genetic profile with embedded genetic ID would be unambiguously linked to a specific patient record by way of matching genetic fDs For example, a person's genetic fingerprint may be recorded in a database along with other genetic data according to the methods of this invention If an individual is then subjected to a subsequent genetic test (for example, for any genetic disease or haplotypmg), the results of the second test may be verified unambiguously by comparing the genetic fingerprints associated with the first and second tests Example 4: On-chip Identification of STR Length Polymorphism in TH01<br>
The HUMTH01 locus contains tetranucleotide repeats (CATT) in the Tyrosine Hydroxylasegene To determine the length of these STR polymorphisms, oligonucleotides probes were synthesized    to contain a 5' anchor sequence of six-<br>
nucleotides designed to be complementary to the target's trading sequence as well as a 3* anchor sequence of six nucleotides designed to be complementary to the target's leading sequence, as follows<br>
5' ttc cct	cac cat 3'<br>
To determine the length of the repeated sequence in the target, a set of oligonucleotides probes is provided to contain one probe matching in length, and thus matching in the number of repeated CATT sequence elements, any anticipated number of complementary target repeats In this design, only the probe containing the correct number of tetramer repeats will form a hybridization complex with the target in which both 5' anchor and 3' anchor are properly aligned with the target's trailing and leading sequence, respectively, and only correctly aligned probes are extended to produce a positive assay signal<br>
Oligonucleotide probes were synthesized to contain 17 nucleotides and a biotin label attached to the 5'end by way ot a 12-C spacer (Biotin-TEG) (Fig 9a) and were purified using reverse phase HPLC by a commercial vendor (Synthegen TX) Using the biotin moiety, probes were attached to streptavidin-coated encoded beads which were assembled into a planar array on a silicon chip One micromoie of a single-stranded target containing six CATT repeats in   a hybridization   mixture containing 10 mM Tns-HCI<br>
(pH 7 4) ImM EDTA, 0 2 M NaCI and 0 1% Triton X-100 was placed m contact with the random encoded bead array and incubated at 50Â°C for 20 minutes The hybridization mixture was then replaced by a mixture containing 3 U of Thermo Sequenase (Amersham Pharmacia Biotech NJ), and IX enzyme buffer with TAMRA-Iabeled dideoxynucleotide (ddNTP) analogs (NEN Life Sciences) and probe extension was allowed to proceed for 3 minutes at o0Â°C The annealing and elongation reactions can be performed in a single step by adding ddNTPs or dNTPs in the mixture and running the reaction at two temperatures The bead array was then washed with distilled H2O for 15 minutes and an image containing the fluorescence signal from each bead within the array was recorded using a fluorescence microscope and a CCD camera Images were analysed to determine the identity of each of the extended (and hence fluorescent) probes As shown m Fig 9b, the largest signal was recorded from beads displaying the six-tetramer probe ST-6, indicating that this probe contains the correct number of lepeats to match the number of target repeats<br>
Example 5 On chip identification of STR length polymorphism using probes containing a 3'anchor ("hook") sequence<br>
Oligonucleotides designed to contain a short 3' anchor ("hook") sequence (but not<br>
a 5' anchor sequence) were attached to encoded beads  (Fig  10a)   Specifically, probes<br>
respectively containing 3, 4, and 6 tetramer repeats were designed to interrogate a target<br>
fragment containing six tetramer repeats flanked by 5' leading sequence and a 3' trailing<br>
sequence<br>
In this design, all probes will form a hybridization complex with the target in which the probe's 3' anchor sequence is properly aligned with the target's leading sequence However, by setting the assay temperature at a value exceeding the melting temperatures of all but the longest probe, only hybridization complexes containing that probe remain, and only that probe is extended to produce a positive assay signal<br>
Single base extension was performed in the presence of DNA polymerase and ddGTP as previously described herein As expected, the largest signal was recorded from beads displaying the six-tetramer probe (Fig 10b) These results demonstrate the high level of discnmination produced by the compositions and methods of the present invention<br>
Example 6    On-chip identification of STR length polymorphism using dual color detection<br>
Oligonucleotides were designed as in Example 4, but the single base extension reaction was performed in the presence of two ddNTPs respectively labeled with TAMRA (green) and fluorescein (orange) (NEN Life Sciences) (Fig 11a) The green ddNTP is incorporated if the annealed probe terminates exterior to the repeated sequence (external marker) while the orange ddNTP is incorporated if the annealed probe terminates interior to the repeated sequence (internal marker) A target containing six tetramer repeats was added to the solution, and the assay and integrated analysis were performed as described earlier As before, high signal recorded in the green channel produced by extension of the six tetramer probe ST-6 indicated that this probe contained the correct number of repeats In addition, however, high signal in the orange channel produced by extension of the three-tetramer probe ST3 and the four-telramer probe ST4 indicated that these probes terminated interior to the repeated sequence and contained an incorrect number of repeats (Fig lib) This dual color fonnat affords additional confirmation of assay results by producing a positive signal for all probes in the set Example 7: Effect of anchor length on probe slippage<br>
Two sets of oligonucleotide probes were synthesized to contain, in addition to three, four and six tetramer repeats, 5' anchor sequences of respectively six and eleven nucleotides for the two probe sets, as well as identical 3' anchor sequences, as follows<br>
5' ttc cct		cac cat 3'<br>
5' ctt att tec ct	-   â â cac cat 3'<br>
As before, the 5'-anchor sequence was designed to be complementary to the target's trailing sequence flanking the repeats To determine the length of the repeated sequence in the target, a set of oligonucleotides probes was provided to contain one probe matching in length, and thus matching m the number of repeated CATT sequence elements, any anticipated number of complementary target repeats<br>
Using reaction conditions as described earlier (Example 4), probes containing the longer 5' anchor sequence produced a higher signal level than did probes containing the shorter 5' anchor sequence, indicating higher stability of the hybridization complexes formed by the longer probes (Figure 12) The anchor sequence can be varied to fit the experimental requirements<br>
Example 8 Effect of annealing temperature on the specificity ot STR polymorphism analysis<br>
Oligonucleotide probes were designed, and single base extension was performed<br>
as in Example 5, but at two different annealing temperatures, namely at 37 CC and at 50<br>
Â°C	The image analysis of results obtained at 50 Â°C showed high discrimination<br>
between the ST-6 probe containing the correct number of repeats and the other probes<br>
containing an incorrect number of repeats, while image analysis of results obtained at 37 Â°C showed essentially no discrimination (Fig 14) Selection of the correct assay temperature substantially enhanced the specificity of detection<br>
Example 9   Sense and Anti-sense Probes<br>
This Example illustrates the use of oligonucleotides probes containing neither 5'-nor 3'-anchor sequences for the analysis of both sense and anti-sense DNA strands Oligonucleotide probes containing such anchor sequences complementary to trailing and leading sequences in both sense and anti-sense DNA also can be used in other embodiments Both sets of probes are attached to encoded beads which are separately assembled on the surface of two silicon chips that are placed on the same chip earner The target is amplified with two sets ot primers, each target containing six repeats as well as a 3' trailing sequence and a 5' leading sequence Single base extension is performed m the presence of DNA polymerase and ddNTPs as described in Example 5 In this design, extension occurs only for the probe containing the correct number of repeats to match the number of target repeats<br>
Example 10' Poly T identification<br>
Oligonucleotide probes were designed for the identification of intron 8 poly-thymidine (T) tract variants (5T, 7T, 9T) within the CFTR gene   Each probe was<br>
synthesized and purified using reverse phase HPLC by a commercial vendor (IDT) to contain a biotin label attached to the 5' end by way of a 12-C spacer (Biotin-TEG) Probes were designed to contain both a 5' anchor sequence matching the 3' trailing sequence of the target and a 3' anchor ("hook") sequence matching the 5' leading sequence of the target The anchor sequence length was varied between four and ten bases, longer anchor sequence lengths also are possible (Fig 16) Probes containing variable numbers of repeats were immobilized on encoded beads as described earlier<br>
In this design, only the probe that aligns with the target sequence and matches the number of target repeats will be elongated in the presence of DNA polymerase In the elongation step, dNTPs are provided, either at least one fluorescently labeled dNTP to produce a fluorescently labeled elongation product, or one or more labeled ddNTPs for single base extension as described in previous examples The signal is recorded by instant imaging of the array as described in this application The results of experiments using targets containing poly-T vanants of different lengths demonstrate that identification of respective poly-T vanants is achieved<br>
Example 11 Identification of Poly T variant of CFTR gene<br>
The melting temperature, Tn(n) of a given hybridization complex containing an nT<br>
variant reflects the length of the variant as well as the degree of mismatch between probe and target That is, discrimination is optimized by setting the assay temperature so as to destabilize all hybridization complexes except those containing probes either matching or exceeding the length of the poly-T sequence in the target For example, a target containing a 7T variant is readily identified by setting the assay temperature T, so as to satisfy the condition Tm(S) 
To perform the assay, 1 Âµmole of target was added to an annealing and elongation mixture containing 10 mM Tns-HCl (pH 7 4) ImM EDTA, 0 2 M NaCl, 0 \% Triton X-100, 3 U of Thermo Sequenase (Amersham Pharmacia Biotech NJ), along with TAMRA-labeled dideoxynucleotide (ddNTP) analogs (NEN Life Sciences)<br>
Following annealing and elongation, bead arrays were washed with distilled H2O for 15 minutes An image containing the fluorescence signal from each bead within the array was recorded using a fluorescence microscope and a CCD camera, and images were analyzed to determine the identity of each of the elongated primers The results in Figure 37 demonstrate that polyT variants in the target are properly identified by the compositions and methods of the present invention<br>
Example 12. Hierarchical probe design for the analysis of long repeats<br>
This example illustrates an approach to the design ot probes for the analysis of long target repeats that are common in many genetic diseases   This approach involves a<br>
"base-offset" construction wherein a first set of "base" probes is constructed to contain IN, 2N, 3N, (N &gt; 1) repeats, with these "base" probes being attached to separate encoded beads A second set of "offset-probes" is constructed to contain all N+n repeats, with n<n importantly to minimize the number of codes required all probes containing same offset are attached type encoded bead fig depicts two separate probe designs with common factors such as anchor sequence></n>
This approach is particularly useful when single-repeat resolution is not required for diagnosis For example, in the diagnosis of predisposition to certain disease states (such as Huntington's disease), it is critical to determine with single repeat resolution only the number of repeats between 35 and 41, the critical range determining the likelihood of pathology That is, patients with more than 41 repeats will develop the disease while patients with fewer than 35 repeats will not, with the probability of pathology increasing with repeat number between 35 and 41<br>
For example, to determine the number of target repeats up to 42 to within a resolution of seven, (i e , N=7), six "base-probes" are constructed to respectively contain 1, 14, 21, , 42 repeats These probes are attached to encoded beads Under assay conditions descnbed in previous examples, all but those hybridization complexes containing probes with a number of repeats matching or exceeding the number of target repeats will be destabilized   For example, by setting the assay temperature, T, to an<br>
appropriately high value to exceed Tml rcpea si, a number that is readily determined with great precision prior to the assay, a sample containing 32 target repeats produces a signal only for the base probes containing 35 and 42 repeats while a sample containing 39 target repeats produces a signal only for the base probes containing 42 repeats In the first case, no further analysis is required In the second case, the target is determined to contain at least 35 repeats, and a set of "offset probes" is invoked to determine the exact number Here, offsets, n
A related design invokes two sets of "base probes", the second set shifted with respect to the first In the example descnbed above, a second set of base probes shifted by AN=3 would contain probes With 0+3, 7+3 14+3, 35-+3 repeats, each attached to encoded beads   For the patient sample with 39 repeats, the first set of base-probes<br>
produced a signal only for the base probe with 42 repeats thereby placing the number of target repeats to &gt;35, the second set of base-probes produces no signal at all, placing the number of target repeats to &gt;38 This alternative design using multiple shifted sets of offset probes is particularly useful to bracket the number of target repeats<br>
Example 13    Identification of polymorphic repeats by ligation and on chip cycling<br>
In this Example, an experiment designed to identify the number of repeats in the TH01 locus uses ligation to attach a labeled detection probe to an adjacent immobilized capture probe within a three-member hybridization complex formed by the two probes and the target<br>
The detection probes are designed to be complementary to the leading sequence of the target (as in Example 1) and to a portion of the repeat sequence At a first temperature, T1, the target will anneal to an immobilized capture probes A detection probe is added, at either the first, or preferably at a second, higher temperature, T2 &gt; T1 and is permitted to anneal to the leading sequence of the target but is ligated only if it contains the correct number of target repeats At a third temperature, T3 &gt; Tlt T2, chosen so as to destabilize non-hgated three-member hybridization complexes, signal remains only on those beads retaining ligated detection probe (Fig 19a) These beads identify the probes with the correct number of repeats<br>
In a variation of the above design, the assay is cycled multiple times through the sequence Ti 
<br><br><br><br><br>
We Claim:<br>
1.	A composition for analyzing a target nucleic acid sequence in a patient genomic sample to determine if said sequence contains a mutation or polymorphism and for identifying by DNA fingerprinting said patient providing the sample, said composition characterized in that there is a mixture of: a first set of oligonucleotides, wherein members of the first set have different sequences and said members are attached to beads which are associated with an optically distinguishable characteristic that identifies the sequences of the oligonucleotides attached thereto, the first set of oligonucleotides for identifying target nucleic acid sequences in the patient sample, the first set of oligonucleotides comprising at least one oligonucleotide for each anticipated polymorphism; a second set of oligonucleotides, wherein members of the second set have different sequence and said members are attached to beads which are associated with an optically distinguishable characteristic that identifies the sequences of the oligonucleotides attached thereto, the second set of oligonucleotides for identifying a number of marker sequences in the patient genomic sample, said marker sequence are derived from subsequences in the patient genomic sample, said subsequence each including at least one known polymorphism site, which is a single nucleotide polymorphism or a short term tandem repeat site) that can be used as a polymorphic marker due to variations in polymorphisms between different individual, the identification of said marker sequences providing for identification of said patient, the second set of oligonucleotides comprising at least one oligonucleotide for each of said marker sequences.<br>
2.	The composition as claimed in claim 1, wherein mixture of beads is arranged in a planar array and the position of individual beads in the array is not pre-determined.<br>
3.	The composition as claimed in claim 2, wherein the bead array comprises subarrays, with the oligonucleotides of the second set and oligonucleotides of the first set being located in different subarrays.<br>
4., The composition as claimed in claim 1, wherein the optically distinguishable characteristic is a chemical label.<br>
5. The composition as claimed in claim 1, wherein the target nucleic acid sequence is analyzed to determine if it contains a mutation and wherein the target nucleic acid and the polymorphic markers are located in the same gene.<br>
6.	The composition as claimed in claim 1, wherein the target nucleic<br>
acid sequence is analyzed to determine if it contains a mutation and<br>
wherein the target nucleic acid and the polymorphic markers are<br>
located in the different genes.<br>
7.	The composition as claimed in claim 1, wherein the target nucleic acid sequence and the marker sequences are derived from the patient genomic sample by amplification of one or more nucleic acid subsequences within the patient genomic sample.<br>
8.	The composition as claimed in claim 9, wherein the amplification is performed using the polymerase chain reaction.<br>
9.	The composition as claimed in claim 1, wherein members of the first set of oligonucleotides are capable of annealing to the target nucleic acid sequence and members of the second set of oligonucleotides are capable of annealing to the marker sequences.<br>
10.	The composition as claimed in claim 1 wherein members of the<br>
first set of oligonucleotides or members of the second set of<br>
oligonucleotides, or members of both the first and second sets, are<br>
capable of being elongated following annealing by incorporation of one<br>
or more types of deoxyribonucleotide triphosphates or di-<br>
deoxyribonucleotide triphosphates.<br>
11.	The composition as claimed in claim 12, wherein the<br>
deoxyribonucleotide triphosphates or di-deoxyribonucleotide<br>
triphosphates are labeled, and can be optically detected in the elonged<br>
oligonucleotides.<br>
12.	The composition as claimed in claim 13 wherein the lebels are optically detectable.<br>
13.	A composition for analyzing a target nucleic acid sequence obtained or derived from a patient genomic sample and for identifying said patient providing the sample substantially as herein described with reference to the foregoing description, examples and the accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjQtMDktMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1324-DELNP-2005-Correspondence-Others-(24-09-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tMTMtKDI0LTA5LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-13-(24-09-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMyNC1kZWxucC0yMDA1LXBldGl0aW9uLTEzNy5wZGY=" target="_blank" style="word-wrap:break-word;">1324-delnp-2005-petition-137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263176-a-compund-of-formmula-i.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263178-improved-brake-system-for-automobiles.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263177</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1324/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HASHMI, GHAZALA</td>
											<td>56 TELEGRAPH HILL ROAD, HOLMDEL, NJ 07733, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SEUL, MICHAEL</td>
											<td>84 PLEASANT AVENUE, FANWOOD, NJ 07023, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MESSING, JOACHIM</td>
											<td>17 NEUVILLE DRIVE, SOMERSET, NJ 08873, USA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NA</td>
											<td>NA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NA</td>
											<td>NA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>NA</td>
											<td>NA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/027456</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-09-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/238,439</td>
									<td>2002-09-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263177-genetic-analysis-and-authentication by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:02:04 GMT -->
</html>
