<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263198-dna-coding-for-polypeptide-participating-in-biosynthesis-of-pladienolide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:57:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263198:&quot;DNA CODING FOR POLYPEPTIDE PARTICIPATING IN BIOSYNTHESIS OF PLADIENOLIDE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;DNA CODING FOR POLYPEPTIDE PARTICIPATING IN BIOSYNTHESIS OF PLADIENOLIDE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides polypeptides that participate in the biosynthesis of the pladienolide macrolide compounds, DNA that encodes these polypeptides and variants of this DNA, transformants that maintain all or a portion of this DNA or variant thereof, and a method of producing the pladienolide macrolide compounds using these transformants. More particularly, it provides an isolated pure DNA that contains at least one region encoding a polypeptide that participates in pladienolide biosynthesis; polypeptide encoded by this DNA; a self-replicating or integrated-replicating recombinant plasmid carrying this DNA; a transformant maintaining this DNA; and a method of producing a pladienolide, characterized by culturing this transformant on culture medium and collecting pladienolide from this culture medium.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Description<br>
DNA coding for polypeptide participating in biosynthesis of pladienolide<br>
Technical Field<br>
The present invention relates to polypeptides that participate in the biosynthesis of the pladienolide macrolide compounds, to DNA that encodes these polypeptides, and to variants of this DNA.  The present invention also relates to transformants that maintain all or a portion of this DNA or variants thereof and to a method of producing the pladienolide macrolide compounds using these transformants.<br>
Prior Art<br>
Important bioactive substances have been discovered among the various metabolites produced by the actinomycetes. In particular, many compounds have been found having a polyketide in the basic structure (hereunder called polyketide compounds).  Compounds having various biological functions are known including for example the antibacterial agents erythromycin, josamycin, tylosin, midecamycin and mycinamicin, the antifungal agents nystatin and amphotericin, the insecticidal agents milbemycin and avermectin, the immune suppressors tacrolimus and rapamycin<br><br>
and the anti-tumor agents daunomycin, adriamycin, aclacinomycin and the like.<br>
The compounds include a group of macrolide compounds with excellent anti-tumor activity called pladienolides. "Pladienolide" is a general term for a group of compounds discovered in culture of the Streptomyces sp. Mer-11107 strain, and more than 50 relatives are known, beginning with 11107B (also called pladienolide B), which is represented by the formula (I) below (See WO 2002/060890),<br>
(Figure Remove)<br>
11107B (Pladienolide B)<br>
Much is also known with regard to the mechanism of polyketide compound biosynthesis.  It has been reported that the diverse polyketide compounds cited above share a common biosynthetic mechanism that very closely resembles fatty acid biosynthesis.  Thus, polyketide compound biosynthesis proceeds by a process in which lower fatty acid, such as acetic acid or propionic acid, is successively condensed and the p-carbonyl group in the elongated acyl group is then variously subjected to ketone reduction, dehydration, or enoyl reduction by the same<br>
processes as in fatty acid synthesis.  It has been reported that for many of these polyketide compounds the various repetitive synthesis steps are mediated by a high molecular weight multifunctional enzyme complex that has separate active sites (domains) required for the respective catalytic reaction activities.  The general reaction scheme of polyketide biosynthesis is outlined in, for example, Ann. Rev. Gen., 24 (1990) 37-66 and Ann. Rev. Microbiol., 47 (1993) 874-912.<br>
It has been demonstrated that the DNA sequence encoding a polyketide synthase generally encodes all the activity required for synthesis of the polyketide skeleton and is organized in repeat units, i.e., modules, that include the condensation step and post-condensation modification steps. Different domains participate with regard to the individual catalytic activities: these domains participate in the specificity for the specific carboxylic acid structural units present in the individual condensation steps or prescribe the particular post-condensation modification function that will be implemented.  For example, the gene encoding the polyketide synthase for pikromycin biosynthesis by Streptomyces venezuelae ATCC15439 is described in Proc. Natl. Acad. Sci. USA, 95 (1998) 12111-12116.  In addition, WO 93/13663 describes the structure of a gene that encodes the erythromycin polyketide synthase of Saccharopolyspora erythraea.  This gene is constituted of 6<br>
modules wherein each module carries out one condensation step.  In other words, the exact sequence of acyl chain elongation and modification of the elongated chain are governed by the genetic information present in each module.<br>
For a wide variety of polyketide compounds, synthesis of the polyketide skeleton by the polyketide synthase is frequently followed by modification by enzymes (hereafter also referred to as modification enzymes) that catalyze modification reactions such as hydroxylation, epoxidation, and methylation to give the final metabolite.  It has been shown that the gene group that participates in the production of these compounds, that is, the enzymes necessary for the biosynthesis of the final metabolite, and genes that encode, for example, regulatory factors required to regulate production, are generally arranged in the form of a cluster in a DNA region on the genome or a plasmid of the producing microorganism (the gene group that participates in the biosynthesis of these compounds is also generally referred to hereafter simply as the "biosynthetic gene") .<br>
Determination of the nucleotide sequence information of the gene encoding a polyketide synthase raises the possibility, through domain modification based on this information, of changing the size of the carbon chain and the functional group on the p-carbon from the condensation step.  For example, Proc. Natl. head. Sci. USA, 90 (1993)<br>
7119-7123 reports that novel derivatives of erythromycin can be produced by the selective deactivation of particular domains within the polyketide synthase gene for erythrmoycin.  Furthermore, the predictable production of novel compounds is made possible by the recombination of the domains of individual modules with other domains.  For example, Proc. Natl. head. Sci. USA, 96 (1999) 1846-1851 reports that a variety of novel compounds can be become accessible through the recombination of some of the domains within the polyketide synthase gene for erythromycin.<br>
In addition, determination of the nucleotide sequence of a biosynthetic gene cluster that includes the genes that encode modification enzymes (hereafter also referred to as modification enzyme genes) raises the possibility of the predictable production of novel compounds through the selective alteration of modification enzyme genes based on this information.  For example, Science 252 (1991) 114-116 reports that the novel derivative, 6-deoxyerythronolide B, can be produced by disruption of the hydroxylase gene eryF present in the region of the polyketide synthase gene for erythromycin.<br>
It may also be possible, by activation of the expression of a modification enzyme gene, to reduced unwanted by-products and produce solely a desired component. Methods generally known for the activation of gene expression include transcriptional activation based on<br>
promoter replacement, increasing the gene copy number using a multicopy vector, and increasing the gene production capacity by mutagenesis.  It may also be possible to raise the productivity by activating a regulatory gene by the same methods, or conversely by deactivating a regulatory gene.<br>
A desired polyketide compound can also be produced by a heterologous strain by acquiring the genes encoding the biosynthetic gene cluster and introducing them into a heterologous strain by an appropriate method. The heterologous strain used for this purpose is advantageously a microorganism and particularly E. coli with its capacity for rapid culture.  For example, it is reported in Science, 291 (2001) 1790-1792 that, by incorporating a polyketide synthase gene into E. coli, the target 6-deoxyerythronolide B, a precursor for erythromycin, can be produced at good efficiencies.<br>
Disclosure of the Invention<br>
An object of the present invention is to provide polypeptides that participate in the biosynthesis of pladienolide macrolide compounds, DNA that encodes these polypeptides, and variants of this DNA.  An additional object of the present invention is to provide transformants that maintain all or a portion of this DNA or variants thereof and a method of producing pladienolide macrolide<br>
compounds using these transformants.<br>
In order to achieve these objects, the present inventors attempted to obtain the target DMA from the strain Streptomyces sp. Mer-11107 (hereafter also referred to as strain Mer-11107), which is a strain that produces pladienolide macrolide compounds, by the colony hybridization procedure using a probe produced based on a sequence reported to be generally conserved in the ketosynthase domains of polyketide synthases; however, a large number of cosmids were selected and the target DNA could not be directly identified.<br>
The present inventors therefore focused on the strong possibility that a modification enzyme gene would be present in the region of the polyketide synthase gene, and, using PCR, obtained gene fragments of a hydroxylase (cytochrome P450 enzyme), which is one of the modification enzymes, from publicly known actinomycetes.  Using these as probes, several cosmids containing the target DNA were selected from the large number of cosmids obtained based on the sequence of the polyketide synthase region.<br>
The Mer-11107 strain presumably contains a large number of modification enzymes based on the fact that it has the capacity to produce a variety of pladienolide analogues. The present inventors discovered that, among these numerous modification enzymes, the hydroxylase enzyme present in the selected cosmids was a 6-hydroxylase.  Furthermore, the<br>
present inventors succeeded in obtaining and identifying the target DNA for the first time by overcoming characteristics inherent to the Mer-11107 strain that are unfavorable to the application of genetic engineering technology, such as resistance to conversion to the protoplast and resistance to the generally used drug markers.<br>
That is, the present invention relates to the following (1) to (20) .<br>
(1)	A DNA that is isolated and pure, and that contains at<br>
least one region encoding a polypeptide that participates<br>
in pladienolide biosynthesis.<br>
(2)	The DNA described in (1), characterized by containing<br>
the complete region encoding the polypeptide that<br>
participates in pladienolide biosynthesis.<br><br>
(3)	The DNA described in (1) or (2), characterized in that<br>
the polypeptide participating in pladienolide biosynthesis<br>
is at least one type selected from polyketide synthases, 6-<br>
hydroxylases, 7-acylation enzymes, 18,19-epoxidases and<br>
transcription regulator factors.<br>
(4)	The DNA described in any of (1) to (3), characterized<br>
by originating in a microorganism belonging to the genus<br>
Streptomyces.<br>
(5)	The DNA described in (1), comprising at least one<br>
nucleotide sequence selected from the nucleotide sequences<br>
defined in any of the following 1) to 5):<br>
1)	nucleotide sequences defined in any of the following (a)<br>
to (i) :<br>
(a)	the continuous nucleotide sequence from the base 8340<br>
to base 27935 of the Sequence No. 1<br>
(b)	the continuous nucleotide sequence from the base 28021<br>
to base 49098 of the Sequence No. 1<br>
(c)	the continuous nucleotide sequence from the base 49134<br>
to base 60269 of the Sequence No. 1<br>
(d)	the continuous nucleotide sequence from the base 60269<br>
to base 65692 of the Sequence No. 1<br>
(e)	the continuous nucleotide sequence from the base 65707<br>
to base 66903 of the Sequence No. 1<br>
(f)	the continuous nucleotide sequence from the base 68160<br>
to base 66970 of the Sequence No. 1<br>
(g)	the continuous nucleotide sequence from the base 69568<br>
to base 68270 of the Sequence No. 1<br>
(h) the continuous nucleotide sequence from the base 72725 to base 70020 of the Sequence No. 1<br>
(i) the continuous nucleotide sequence from the base 1 to base 74342 of the Sequence No. 1<br>
2)	a nucleotide sequence of a DNA that hybridizes under<br>
stringent conditions with a DNA comprising any of the<br>
nucleotide sequences defined in 1)<br>
3)	a nucleotide sequence having at least 70% homology with<br>
any of the nucleotide sequences defined in 1)<br>
4)	a nucleotide sequence complementary to any of the<br>
nucleotide sequences defined in any of 1) to 3) 5) a nucleotide sequence that, due to the degeneracy of the genetic code, does not hybridize under stringent conditions with a DNA comprising a nucleotide sequence defined in 1), but which codes for the same amino acid sequence as a nucleotide sequence defined in any of 1) to 3). (6) The DNA described in (1), comprising at least one nucleotide sequence selected from the nucleotide sequences defined in any of the following (a) to (i):<br>
(a)	the continuous nucleotide sequence from the base 8340<br>
to base 27935 of the Sequence No. 1<br>
(b)	the continuous nucleotide sequence from the base 28021<br>
to base 49098 of the Sequence No. 1<br>
(c)	the continuous nucleotide sequence from the base 49134<br>
to base 60269 of the Sequence No. 1<br>
(d)	the continuous nucleotide sequence from the base 60269<br>
to base 65692 of the Sequence No. 1<br>
(e)	the continuous nucleotide sequence from the base 65707<br>
to base 66903 of the Sequence No. 1<br>
(f)	the continuous nucleotide sequence from the base 68160<br>
to base 66970 of the Sequence No. 1<br>
(g)	the continuous nucleotide sequence from the base 69568<br>
to base 68270 of the Sequence No. 1<br>
(h) the continuous nucleotide sequence from the base 72725<br>
to base 70020 of the Sequence No. 1<br>
(i) the continuous nucleotide sequence from the base 1 to<br>
base 74342 of the Sequence No. 1.<br>
(7)	A polypeptide encoded by the DNA described in any of<br>
(1) to (6) .<br>
(8)	The polypeptide described in (7), characterized by<br>
having a polyketide synthase activity.<br>
(9)	The polypeptide described in (8) , characterized by<br>
having the amino acid sequence described by Sequence No. 2,<br>
3, 4 or 5, or having a partial sequence thereof.<br><br>
(10)	The polypeptide described in (7), characterized by<br>
having a 6-hydroxylase activity.<br>
(11)	The polypeptide described in (10), characterized by<br>
having the amino acid sequence described by Sequence No. 6<br>
or having a partial sequence thereof.<br>
(12)	The polypeptide described in (7), characterized by<br>
having an 18,19-epoxidase activity.<br>
(13)	The polypeptide described in (12) , characterized by<br>
having the amino acid sequence described by Sequence No. 8<br>
or having a partial sequence thereof.<br>
(14)	The polypeptide described in (7), characterized by<br>
having a transcription regulator factor activity.<br>
(15)	The polypeptide described in (14), characterized by<br>
having the amino acid sequence described by Sequence No. 9<br>
or having a partial sequence thereof.<br>
(16)	The polypeptide described in (7), characterized by<br>
having a 7-acylation enzyme activity.<br>
(17)	The polypeptide described in (16), characterized by<br>
having the amino acid sequence described by Sequence No. 7 or having a partial sequence thereof.<br>
(18)	A self-replicating or integrated-replicating<br>
recombinant plasmid carrying the DNA described in any of<br>
(I) to (6) .<br>
(19)	A transformant maintaining the DNA described in any of<br>
(1) to (6) .<br>
(20)	A method of producing a pladienolide, characterized by<br>
culturing the transformant described in (19) on culture<br>
medium; and collecting pladienolide from the culture broth.<br>
(21)	The method of production described in (20), wherein<br>
the pladienolide is pladienolide B.<br>
(22)	A method of producing a pladienolide D derivative<br>
represented by the formula (VI):<br>
(Figure Remove)<br>
 (wherein RN represents a lower alkyl group or a cyclic<br>
lower alkyl group; and n represents 1 or 2), comprising the steps of:<br>
1) introducing a hydroxyl group at position 16 of the compound of the formula (I):<br>
(Figure Remove)<br>
 (pladienolide B) obtained by the method of production described in (20) or (22), thereby converting the compound of the formula (I) into a compound of the formula (II) (pladienolide D):<br>
(Figure Remove)]<br>
2) introducing a suitable protective group onto the hydroxyl groups at position 3, 6, 16 and/or 21 of the compound of the formula (II), thereby converting the compound of the formula (II) into a compound of the formula<br>
(III) : <br>
(Figure Remove)<br>
(wherein R3A, R6A, R16A and R21A represent a hydrogen atom or<br>
a protecting group of the hydroxyl group, provided that R3A, R6A, R16A and R21A do not all represent the hydrogen atom simultaneously) ;<br>
3) eliminating the acetyl group at position 7 of the compound of the formula (III), thereby converting the compound of the formula (III) into a compound of the formula (IV) :<br>
(Figure Remove) <br>
(wherein R3A, R6A, R16A and 4) introducing a substituent at position 7 of the compound of the formula (IV), thereby converting the compound of the formula (IV) into a compound of the formula (V):<br><br>
(Figure Remove)<br>
 (wherein RN,   R3A,   R6A,   R16A and R21A are  defined  as   above) ;<br>
and<br>
5) eliminating the protective group from the compound of<br>
the formula (V).<br>
Detailed Description of the Invention<br><br>
Embodiments of the present invention are described in detail in the following.<br>
In the present specification, "lower alkyl group" denotes an alkyl group having 1 to 6 carbons and can be specifically exemplified by methyl, ethyl, propyl, isopropyl, butyl and so forth, wherein methyl, ethyl and isopropyl are particularly preferred.<br>
"Cyclic lower alkyl group" denotes an alkyl group having 3 to 6 carbons and can be specifically exemplified by cyclopropyl, cyclobutyl, cyclohexyl and so forth, wherein cyclopropyl and cyclobutyl are particularly preferred.<br>
"DNA that hybridizes under stringent conditions" denotes, for example, DNA obtained using a colony hybridization procedure, plaque hybridization procedure, Southern hybridization procedure, or the like, employing DNA having a nucleotide sequence as defined in any of the aforementioned (a) to (i) as probe, and can be specifically exemplified by DNA that can be identified by carrying out hybridization at 65°C in the presence of 0.7-1.0 M sodium chloride using a filter on which DNA of colony or plaque origin has been immobilized, followed by washing the filter at 65°C using 0.1X-2X SSC solution (the composition of IX SSC solution comprises 150 mM sodium chloride and 15 mM sodium citrate).<br>
A "DNA variant" denotes DNA that has been modified by,<br><br>
for example, the deletion, exchange, addition or insertion of a constituent nucleotide or derivatives thereof.<br>
"Homology" refers to the percentage of nucleotides that are identical between two sequences that have been optimally aligned.  In specific terms, the homology can be calculated from homology=(number of identical<br>
positions/total number of positions)xlOO, and can be calculated using commercially available algorithms.  In addition, algorithms of this nature are incorporated in the NBLAST and XBLAST programs described in Altschul et al., J. Mol. Biol., 215 (1990) 403-410.<br>
"Analogue" refers to a compound that has the same basic skeleton characteristic of a chemical structure, but which differs with respect to, for example, the type of modification or the form of a side chain.<br>
In the present invention, the DNA encoding all or a portion of a polypeptide that participates in pladienolide biosynthesis can be isolated from the cultured mycelia of a microorganism that has the capacity to produce a pladienolide macrolide compound and the nucleotide sequence of this DNA can be determined.  Any microorganism that has the capacity to produce pladienolide can be used as this microorganism regardless of the strain or species, but a preferred microorganism is the strain Streptomyces sp. Mer-11107, which was isolated from soil.  The strain was deposited as FERM P-18144 at the National Institute of<br><br>
Bioscience and Human-Technology Agency of Industrial Science and Technology (1-3, Higashi 1-chome Tsukuba-shi, Ibaraki-ken 305-8566 Japan) , which was subsequently reorganized into the International Patent Organism Depository (IPOD), National Institute of Advanced Industrial Science and Technology (Tsukuba Central 6, 1-1, Higashi 1-chome, Tsukuba, Ibaraki-ken, 305-8566 Japan), as of December 19, 2000, and then transferred to International Deposit FERM BP-7812 at International Patent Organism Depositary (IPOD) National Institute of Advanced Industrial Science and Technology (Tsukuba Central 6, 1-1, Higashi 1-Chome, Tsukuba-shi, Ibaraki-ken 305-8566 Japan) as of November 27, 2001.  The taxonomical properties of the strains are as follows.<br>
(1)	Morphology<br>
Spiriles type aerial hyphae were extended from the vegetative hyphae.  Spore chains consisting of about 10 to 20 cylindrical spores were formed at the end of the matured aerial hyphae.  The size of the spores was about 0.7 x 1.0 (am, the surface of the spores was smooth, and specific organs such as sporangium, sclerotium and flagellum were not observed.<br>
(2)	Cultural characteristics on various media<br>
Cultural characteristics of the strain after incubation<br>
at 28°C for two weeks on various media are shown as follows. The color tone is described by the color names and codes<br><br>
which are shown in the parentheses of Tresner's Color wheels.<br>
1)	Yeast extract-malt extract agar medium<br>
The strain grew well, the aerial hyphae grew up on the surface, and light gray spores (Light gray; d) were observed.  The reverse side of colony was Light melon yellow (Sea).  Soluble pigment was not produced.<br>
2)	Oatmeal agar medium<br>
The strain grew moderately, the aerial hyphae grew slightly on the surface, and gray spores (Gray; g) were observed.  The reverse side of colony was Nude tan (4gc) or Putty (1 l/2ec).  Soluble pigment was not produced.<br>
3)	Inorganic salts-starch agar medium<br>
The strain grew well, the aerial hyphae grew up on the surface, and gray spores (Gray; e) were observed.  The reverse side of colony was Fawn (4ig) or Gray (g).  Soluble pigment was not produced.<br>
4)	Glycerol-asparagine agar medium<br>
The strain grew well, the aerial hyphae grew up on the surface, and white spores (White; a) were observed.  The reverse side of colony was Pearl pink (3ca).  Soluble pigment was not produced.<br>
5)	Peptone-yeast extract-iron agar medium<br>
The strain growth was bad, and the aerial hyphae did not grow on the surface. The reverse side of colony was Light melon yellow (3ea). Soluble pigment was not produced.<br><br>
6) Tyrosine agar medium<br>
The strain grew well, the aerial hyphae grew up on the surface, and white spores (White; a) were observed.  The reverse side of colony was Pearl pink (3ca).  Soluble pigment was not produced.<br>
(3)	Utilization of various carbon sources<br>
Various carbon sources were added to Pridham-Gottlieb<br>
agar and incubated 28°C for 2 weeks.  The growth of the strain is shown below.<br>
1)	L-arabinose <br>
2)	D-xylose    <br>
3)	D-glucose     <br>
4)	D-fructose      Sucrose    Inositol    L-rhamnose<br>
5)	D-mannitol     <br>
6)	Raffinose        positive, slightly positive, negative)<br>
(4)	Various physiological properties<br>
Various physiological properties of the present strain are as follows.<br>
1)	Range of growth temperature (yeast extract-malt extract<br>
agar, incubation for 2 weeks): 12°C to 37°C<br>
2)	Range of optimum growth temperature (yeast extract-malt<br>
extract agar, incubation for 2 weeks): 21°C to 33°C<br><br>
3)	Liquefaction of gelatin (glucose-peptone-gelatin<br>
medium): negative<br>
4)	Coagulation of milk (skim milk medium): negative<br>
5)	Peptonization of milk (skim milk medium): negative<br>
6)	Hydrolysis of starch (inorganic salts-starch agar):<br>
positive<br>
7)	Formation of melanoid pigment (peptone-yeast extract-<br>
iron agar): negative<br>
(tyrosine agar): negative<br>
8)	Production of hydrogen sulfide (peptone-yeast extract-<br>
iron agar): negative<br>
9)	Reduction of nitrate (broth containing 0.1% potassium<br>
nitrate): negative<br>
10)	Sodium chloride tolerance (yeast extract-malt extract<br>
agar, incubation for 2 weeks): grown at a salt content of<br>
4% or less<br>
(5) Chemotaxonomy<br>
LL-diaminopimelic acid and glycin were detected from the cell wall of the present strain.<br>
The present inventors attempted to obtain DNA according to the present invention from this microorganism using the colony hybridization procedure described in Molecular Cloning/ 2nd Edition.  First, genomic DNA from the Mer-11107 strain was partially digested with a suitable restriction enzyme (for example, SauSAI); this partial digest was ligated with a restriction enzyme digest (for<br><br>
example, BamHI) of a cosmid vector capable of replicating in E. coli to give recombinant DNA; and this recombinant DMA was incorporated into E. coli to give transductants. On the other hand, using DNA recovered from the Mer-11107 strain as template, amplified DNA was obtained by PCR using primers designed with reference to sequence information reported to be generally conserved in the ketosynthase domain of polyketide synthase and sequence information for a ketosynthase region in a pikromycin-producing organism (Proc. Natl. Acad. Sci. USA, 95 (1998) 12111-12116).  The initially prepared transductants were screened using the obtained DNA as probe; however, a large number of positive clones (cosmids) was obtained and it was not possible to directly identify transductants having the target DNA.<br>
Attention was then turned to the strong possibility that a modification enzyme gene would be present in the region of the polyketide synthase gene, and fragments of two types of hydroxylase (cytochrome P450 enzyme) genes were obtained by PCR from two publicly known actinomycetes. Using these as probes, the large number of already obtained transductants was screened and a single type of transductant binding to the probes was selected.  The hydroxylase gene-binding DNA present in the selected cosmid was recovered and its sequence was determined.  It was transduced into E. coli, and it was discovered that the<br><br>
transformed E. coli had the capacity to convert ME-265, whose formula is given below and which is the 6-deoxy form of pladienolide B, into pladienolide B.  This DNA was therefore confirmed to be DNA coding for a 6-hydroxylase.<br><br><br><br>
(Figure Remove)<br>
Since this had resulted in the confirmation of a portion of the DNA participating in pladienolide biosynthesis, cosmids containing the pladienolide biosynthetic gene cluster adjacent to the cytochrome P450 gene were selected and aligned, using Southern hybridization and using the cytochrome P450 gene encoding this 6-hydroxylase in the probe, from the large number of positive clones (cosmids) that had already been obtained.<br>
Among the several cosmids thus obtained, disrupted-gene strains were then prepared using cosmids thought to contain the polyketide synthesis region; it was confirmed that these disrupted strains had in fact lost the capacity to produce pladienolide, which thereby confirmed the functionality of the recovered DNA.  The attempt was first made to obtain the disrupted-gene strain by constructing a cosmid having a partial deletion in the region thought to<br><br>
be the polyketide synthesis region and carrying out homologous recombination with the Mer-11107 strain using procedures in general use.  However, several problems were encountered at this time.  Thus, the Mer-11107 strain was not converted to the protoplast by the standardly used lysozyme treatment, which made it impossible to use the protoplast-PEG method in general use for the transformation of plasmids into actinomycetes.<br>
The present inventors therefore attempted to replace the protoplast-PEG transformation procedure with a fusion method in which the DNA was delivered by mixing E. coli in the early logarithmic growth phase with a suitable amount of the actinomycetes spores.  However, since a characteristic of the Mer-11107 strain is its refractoriness to spore formation, additional investigations were carried out, and transformation was finally accomplished through the use of mycelia cultured up to the early logarithmic growth phase instead of the actinomycetes in spore form.<br>
Another problem stemmed from the fact that the Mer-11107 strain has a certain degree of natural resistance to thiostrepton, which as a consequence made it impossible to employ a thiostrepton resistance gene as a marker, although this is standardly used in the transformation of actinomycetes.  The transformation procedure was therefore subjected to additional investigations, whereupon it was<br><br>
discovered that transformants from the Mer-11107 strain could be efficiently selected using an aminoglycoside phosphotransferase gene (aminoglycoside resistance gene) as the marker and using a ribostamycin-containing medium as the culture medium.  Using this method, disrupted-gene strains were constructed in which the DMA thought to be the polyketide synthesis region was disrupted, and it was confirmed that these disrupted strains had in fact lost the capacity to produce pladienolide.<br>
Since it had been confirmed that genes present in the previously obtained cosmids were related to pladienolide biosynthesis, the nucleotide sequences of the DNA fragments inserted in the individual cosmids were determined.  First, after isolation of the individual cosmids by the cesium chloride method, shearing to about 1 kb and subcloning were carried out and the nucleotide sequences of the individual fragments were then determined for the obtained subclones, which resulted in the determination of an approximately 75 kb nucleotide sequence containing DNA related to pladienolide synthesis (refer to Sequence No. 1).<br>
The DNA shown by this Sequence No. 1 contained 8 open reading frames (ORF): pldA I (bases 8340 to 27935), pldA II (bases 28021 to 49098), pldA III (bases 49134 to 60269), pldA IV (bases 60269 to 65692), pldB (bases 65707 to 66903), pldC (bases 68160 to 66970), pldD (bases 69568 to 68270), and pldR (bases 72725 to 70020).  The amino acid sequences<br><br>
of the polypeptides encoded by these sequences are shown in Sequence Nos. 2 to 9, respectively.<br>
In the thusly obtained DMA related to pladienolide biosynthesis by the Mer-11107 strain, pldA I, pldA II, pldA III and pldA IV had several transcription reading frames, each containing one or more repeat units, known as modules, in the same manner as other already elucidated polyketide biosynthetic genes.  As described below, each of the modules coded for some or all of the following domains:<br>
acyl carrier protein (ACP), p-ketoacyl ACP synthase (KS) and acyl transferase (AT), which participate in the condensation reaction in polyketide synthesis, and ketoacyl reductase (KR), dehydrogenase (DH), and enoyl reductase (ER), which participate in the p-carbonyl group modification reactions.  The final module contained a thioesterase (TE) domain, which releases the polyketide chain from the polyketide synthase.<br>
Fig. 1 shows the biosynthesis pathway of a pladienolide in the Mer-11107 strain.  Unlike the other modules the loading module has the central cysteine replaced by glutamine, indicating that pldA I participates in the initial reaction.  Module 10 includes a thioesterase (TE) domain, indicating that pldA IV participates in the final reaction of synthesizing the basic polyketide skeleton. After the basic skeleton of the polyketide has been formed in this way, it is thought that pladienolide biosynthesis<br><br>
proceeds through modifications by the enzyme group (PldB, PldC and PldD) coded by pldB, pldC and pldD.  pldR, by virtue of its high homology with the aveR gene that encodes a transcription regulator factor in avermectin biosynthesis, is believed to code for a transcription regulator factor for the DNA participating in pladienolide biosynthesis.<br>
The thusly elucidated modules and corresponding domains of the DNA participating in pladienolide biosynthesis are given below.<br>
ORF pldA I (bases 8340 to 27935 of Sequence No. 1) encodes for the loading module, module 1, module 2 and module 3, and the corresponding polypeptide is shown by the amino acid sequence of Sequence No. 2.<br>
Loading module (bases 8340 to 11384) KSs: bases 8358 to 9620 ATs: bases 9702 to 10781 ACPs: bases 11148 to 11327<br>
Module 1 (bases 11385 to 16070) KSl:s bases 11385 to 12650 ATI: bases 12747 to 13829 KR1: bases 14940 to 15803 ACPI: bases 15825 to 16007<br>
Module 2 (bases 16071 to 21431) KS2: bases 16071 to 17336 AT2: bases 17445 to 18536 DH2: bases 18717 to 19418<br><br>
KR2: bases 20298 to 21167 ACP2: bases 21189 to 21371<br>
Module 3 (bases 21432 to 27935) KS3: bases 21432 to 22695 ATS: bases 22800 to 23880 DH3: bases 24066 to 24779 ER3: bases 25659 to 26588 KR3: bases 26610 to 27476 ACP3: bases 27498 to 27680<br>
The amino acid sequence of the corresponding polypeptide is shown below. KSs: amino acids 7 to 427 ATs: amino acids 455 to 814 ACPs: amino acids 937 to 996 KS1: amino acids 1016 to 1437 ATI: amino acids 1470 to 1830 KR1: amino acids 2201 to 2488 ACPI: amino acids 2496 to 2556 KS2: amino acids 2578 to 2999 AT2: amino acids 3036 to 3399 DH2: amino acids 3460 to 3693 KR2: amino acids 3987 to 4276 ACP2: amino acids 4284 to 4344 KSS: amino acids 4365 to 4786 ATS: amino acids 4821 to 5181 DH3: amino acids 5243 to 5480<br>
ER3: amino acids 5774 to 6083 KR3: amino acids 6091 to 6379 ACP3: amino acids 6387 to 6447<br>
ORF pldA II (bases 28021 to 49098 of Sequence No. 1) encodes for module 4, module 5, module 6 and module 7, and the corresponding polypeptide is shown by the amino acid sequence of Sequence No. 3.<br>
Module 4 (bases 28021 to 33540) KS4: bases 28132 to 29397 AT4: bases 29530 to 30627 DH4: bases 30865 to 31566 KR4: bases 32413 to 33276 ACP4: bases 33298 to 33480<br>
Module 5 (bases 33541 to 39003) KS5: bases 33541 to 34806 ATS: bases 34912 to 35994 DH5: bases 36175 to 36876 KR5: bases 37755 to 38625 ACP5: bases 38647 to 38829<br>
Module 6 (bases 39004 to 43686) KS6: bases 39004 to 40269 AT6: bases 40372 to 41454 KR6: bases 42550 to 43407 ACP6: bases 43429 to 43611<br>
Module 7 (bases 43687 to 49098) KS7: bases 43687 to 44952<br>
AT7: bases 45031 to 46128 DH7: bases 46303 to 47022 KR7: bases 47881 to 48744 ACP7: bases 48766 to 48948<br>
The amino acid sequence of the corresponding polypeptide is shown below. KS4: amino acids 38 to 459 AT4: amino acids 504 to 869 DH4: amino acids 949 to 1182 KR4: amino acids 1465 to 1752 ACP4: amino acids 1760 to 1820 KS5: amino acids 1841 to 2262 ATS: amino acids 2298 to 2658 DH5: amino acids 2719 to 2952 KR5: amino acids 3246 to 3535 ACP5: amino acids 3543 to 3603 KS6: amino acids 3662 to 4083 AT6: amino acids 4118 to 4478 KR6: amino acids 4844 to 5129 ACP6: amino acids 5137 to 5197 KS7: amino acids 5223 to 5644 AT7: amino acids 5671 to 6036 DH7: amino acids 6095 to 6334 KR7: amino acids 6621 to 6908 ACPI: amino acids 6916 to 6976<br>
ORF pldA III (bases 49134 to 60269 of Sequence No. i;<br>
encodes for module 8 and module 9, and the corresponding polypeptide is shown by the amino acid sequence of Sequence No. 4.<br>
Module 8 (bases 49134 to 53885) KS8: bases 49235 to 50501 ATS: bases 50580 to 51656 KR8: bases 52752 to 53621 ACP8: bases 53642 to 53825<br>
Module 9 (bases 53886 to 60269) KS9: bases 53886 to 55151 AT9: bases 55245 to 56342 DH9: bases 56514 to 57230 ER9: bases 58029 to 58925 KR9: bases 58947 to 59804 ACP9: bases 59826 to 60008<br>
The amino acid sequence of the corresponding polypeptide is shown below. KS8: amino acids 35 to 456 ATS: amino acids 483 to 841 KR8: amino acids 1207 to 1496 ACP8: amino acids 1504 to 1564 KS9: amino acids 1585 to 2006 AT9: amino acids 2038 to 2403 DH9: amino acids 2461 to 2699 ER9: amino acids 2966 to 3264 KR9: amino acids 3272 to 3557<br><br>
ACP9: amino acids 3565 to 3625<br>
ORF pldA IV (bases 60269 to 65692 of Sequence No. 1) encodes for module 10, and the corresponding nucleotide is shown by the amino acid sequence of Sequence No. 5.<br>
Module 10 (bases 60269 to 65692) KS10: bases 60431 to 61696 AT10: bases 61781 to 62869 KR10: bases 63752 to 64609 ACP10: bases 64631 to 64813 TE10: bases 64832 to 65692<br>
The amino acid sequence of the corresponding polypeptide is shown below. KS10: amino acids 55 to 476 AT10: amino acids 505 to 867 KR10: amino acids 1162 to 1447 ACP10: amino acids 1455 to 1515 TE10: amino acids 1522 to 1808<br>
ORF pldB (bases 65707 to 66903 of Sequence No. 1) encodes for a pladienolide 6-hydroxylase, and the corresponding polypeptide is shown by the amino acid sequence in Sequence No. 6.  ORF pldC (bases 68160 to 66970 of Sequence No. 1) encodes for a pladienolide 7-acylation enzyme, and the corresponding polypeptide is shown by the amino acid sequence in Sequence No. 7.  ORF pldD (bases 69568 to 68270 in Sequence No. 1) encodes for a pladienolide 18,19-epoxidase, and the corresponding<br>
polypeptide is shown by the amino acid sequence in Sequence No. 8. ORF pldR (bases 72725 to 70020 in Sequence No. 1) encodes for a transcription regulator factor in pladienolide biosynthesis, and the corresponding polypeptide is shown by the amino acid sequence in Sequence No. 9.<br>
Furthermore, the DNA according to the present invention encompasses not only the aforementioned DNA, but also variants thereof as well as DNA that hybridizes with the aforementioned DNA under stringent conditions and participates in pladienolide biosynthesis.  Such variants can be more specifically illustrated by sequences that exhibit at least 70% homology and preferably at least 80% homology and more preferably at least 90% homology with any of the following sequences: the nucleotide sequence continuously running from base 8340 to base 27935 of Sequence No. 1; the nucleotide sequence continuously running from base 28021 to base 49098 of Sequence No. 1; the nucleotide sequence continuously running from base 49134 to base 60269 of Sequence No. 1; the nucleotide sequence continuously running from base 60269 to base 65692 of Sequence No. 1; the nucleotide sequence continuously running from base 65707 to base 66903 of Sequence No. 1; the nucleotide sequence continuously running from base 68160 to base 66970 of Sequence No. 1; the nucleotide sequence continuously running from base 69568 to 68270 of<br>
Sequence No. 1; and the nucleotide sequence continuously running from base 72725 to base 70020 of Sequence No. 1.<br>
Thus, once it has been possible to establish a nucleotide sequence, DNA participating in pladienolide biosynthesis in accordance with the present invention can also be obtained by publicly known methods based on this information.<br>
For example, DNA with the nucleotide sequence shown in Sequence No. 1 is digested with a suitable restriction enzyme and the digested DNA is separated and recovered by a method described in Molecular Cloning, 2nd Edition to generate oligonucleotide for use as a probe or primer.  In the case of use as a probe, the obtained DNA fragment is preferably labeled with, for example, digoxygenin.  For example, a DIG Labeling &amp; Detection Kit (Roche Diagnostics) is preferably used for digoxygenin labeling.<br>
A library is then constructed from a microorganism that exhibits the capacity to produce pladienolide, using a cDNA cloning procedure or a genomic cloning procedure as described in Molecular Cloning, 2nd Edition.  Clones (colonies) that hybridize with the already prepared probe are selected from the resulting library; plasmid extraction is carried out on the selected clones according to the procedures described in Molecular Cloning, 2nd Edition; and target DNA that participates in pladienolide biosynthesis can be recovered from the plasmids thereby obtained.<br>
When, in this case, only partial fragments of the DNA participating in pladienolide biosynthesis are present in the extracted plasmids, a restriction enzyme map of the plasmids is constructed by a standard method based on digestion of the extracted plasmids with suitable restriction enzymes/ for example, BamHI.  Restriction enzyme fragments present in common in several clones are then elucidated from this restriction enzyme map, and DNA containing the total DNA that participates in pladienolide biosynthesis can be obtained by stringing together the cloned fragments at their regions of overlap.<br>
Or, the DNA participating in pladienolide biosynthesis can also be obtained using the aforementioned library and primers, by direct amplification of the target DNA by the direct PCR reaction.<br>
The nucleotide sequence of the DNA encoding polypeptide that participates in pladienolide biosynthesis can be identified by analysis using the nucleotide sequence analysis procedures in general use, for example, using the dideoxy method (Proc. Natl. Acad. Sci. USA, 74, 5463 (1977)) or a nucleotide sequence analyzer, for example the 373A-DNA Sequencer (Perkin-Elmer).  In specific terms, double-stranded plasmid DNA is used directly as the template in a cycle sequence reaction using various sequence-specific oligonucleotide primers.  Or, the DNA fragments can be subdivided and randomly inserted into the<br>
bacteriophage M13, and, using a plasmid vector or library in which the individual fragments are partially overlapped, an overlap library can be constructed in which progressive deletion is introduced from the terminal region of the DNA fragments; the DNA sequence of the various recombinant DNA fragments can then be determined using a vector sequence-specific oligonucleotide primer.<br>
In addition, based on the nucleotide sequence determined for the DNA, a target DNA can also be prepared by chemical synthesis using a DNA synthesizer, for example, a Model 8905 DNA Synthesizer (PerSeptive Biosystems).  The processing, compilation, editing, and analysis of the obtained nucleotide sequence data can be carried out using existing software, for example, Genetyx™ from Software Development.<br>
Polypeptide according to the present invention can be produced by inducing the expression of DNA according to the present invention in a host cell using, for example, the procedures described in Molecular Cloning, 2nd Edition, or Current Protocols in Molecular Biology.  The site of incorporation of the DNA or variant thereof according to the present invention may be on either a plasmid or chromosome of the host microorganism.  In addition to the subject DNA or variant thereof, such a plasmid may also contain, for example, a self-replicating sequence, promoter sequence, terminator sequence, and drug-resistance gene.In addition, the plasmid may be an integration plasmid that has a sequence homologous with a particular region of the genome of the anticipated host.<br>
The host or plasmid-vector system for expression of polypeptide encoded by DMA according to the present invention may be any system in which this DNA can be stably maintained and expressed.  However, when the host is an actinomycetes or related strain that has a native capacity to produce pladienolide, this enables the use of, for example, the self-replicating vector pIJ6021 (Gene, 166, 133-137 (1995)) or the chromosome-integrating vector KC515 (The Bacteria, Vol. 9, Antibiotic-Producing Streptomyces (ed: Queener, S.E. and Day, L.E.), pp. 119-158, Academic<br>
Press, Orlando, Fgla.).<br>
The procedures for isolating and purifying transformant-produced polypeptide according to the present invention can be those procedures in general use for the isolation and purification of enzymes.  For example, when polypeptide according to the present invention is expressed in a soluble state within the cell, the cell is recovered by centrifugal separation after cultivation has been completed, and suspended in an aqueous buffer, and after disruption of the cell by, for example, an ultrasonic homogenizer, French press, Manton-Gaulin homogenizer, or Dynomill, a noncellular extract is obtained.  A supernatant is prepared by centrifugal separation of the noncellular<br>
extract thus obtained, and a purified target product can be obtained from this supernatant by procedures in general use for the isolation and purification of enzymes.<br>
Moreover, polypeptide according to the present invention can also be produced by chemical synthesis methods, such as the fluorenylmethyloxycarbonyl method (Fmoc method) or t-butoxycarbonyl method (t-Boc method), based on the data for the amino acid sequence of the previously obtained polypeptide.<br>
In addition, pladienolide can be obtained by culturing, on a medium, a transformant containing a previously obtained pladienolide biosynthetic gene; allowing the pladienolide product to accumulate in the culture; and recovering the pladienolide from the culture.  The culture conditions are not particularly limited, but are based on the general culture conditions for the host.<br>
Based on the nucleotide sequence information of the DNA that participates in pladienolide biosynthesis, the size of the carbon chain in the basic polyketide skeleton and the functional group at the p-carbon from the condensation step can also be altered by modification of the modules. Moreover, by selectively inactivating a modification enzyme that acts after polyketide formation, it may be possible to preferentially produce a specific component of a predictable pladienolide.  For example, it is possible to convert the Mer-11107 strain, which produces mainly<br>
pladienolide B, into a strain that produces mainly ME-265, the 6-deoxy form of pladienolide B, by deletion mutation of pldB.  The procedure for effecting deletion mutation of pldB can be exemplified by conversion or substitution by homologous recombination by the general methods described in Molecular Cloning,. 2nd Edition.<br>
Using a thusly obtained strain endowed with the capacity to preferentially produce a specific pladienolide, it becomes possible to produce a specific pladienolide patterned on the method of producing pladienolide B.<br>
The present invention enables the isolation of DNA that encodes polypeptide that participates in the biosynthesis of a pladienolide macrolide compound and enables the determination of its nucleotide sequence.  In addition, a plasmid containing this DNA can be constructed; a transformant transformed by such a plasmid can be constructed; and pladienolide can be produced at good efficiencies using such a transformant.  Moreover, by modifying or altering the sequence of the obtained DNA, it becomes possible to produce novel or specific pladienolides by altering the type of carboxylic acid that is incorporated, the post-condensation modification reactions, the modification reactions that occur after skeleton formation, and their numerous combinations.<br>
Brief Description of the Drawings<br>
Fig. 1 shows the biosynthesis pathway of pladienolides in Mer-11107 strain.<br>
Fig. 2 shows the correspondence between cosmids and each of ORFs of DMA participating in biosynthesis of pladienolides in Mer-11107 strain.<br>
Fig. 3 shows the structure of plasmid pKU253.<br>
Examples<br>
The present invention is explained in detail below using examples, but the present invention is not limited by these examples.  In the explanations below, concentrations are expressed as weight percentages unless otherwise specified.<br>
Example 1: Cultivation of Mer-11107 and isolation of genomic DNA<br>
Hyphae of Streptomyces sp. Mer-11107 were inoculated into 25 mL of Tryptic Soy Broth, and cultured with shaking at 28°C for 3 days.  Genomic DNA was prepared from the resulting culture broth according to the methods described under "Isolation genomic DNA" (pp. 162-170) in D.A. Hopwood et al's Practical Streptomyces Genetics (The John Innes Foundation, Norwich, England, 2000).<br>
Example 2: Preparation of Mer-11107 genomic library<br>
160 yL of sterile purified water, 200 uL of Mer-11107 genome DNA solution (1 mg/mL), 40 yL of lOx concentration M buffer solution (100 mM Tris-HCl (pH 7.5), 100 mM MgCl2, 10mM dithiothreitol, 500 mM NaCl) and 1 uL of restriction enzyme Sau3AI (1 unit/uL) were mixed and incubated at 37°C for 3 minutes.  50 uL was then taken out and extracted with 50 uL of phenol-chloroform mixture<br>
(phenol:chloroform:isoamyl alcohol=25:24:1, volume ratio), the aqueous layer was collected and extracted again with 50 uL of chloroform , and the aqueous layer was again collected.  5 uL of 3 M sodium acetate (pH 6.0) and 150 uL of ethanol were added to the liquid, which was then left at -80°C for 30 minutes and centrifuged to collect the precipitated DNA.  After being washed in 70% ethanol, this DMA was dissolved in 90 uL of sterile purified water, and incubated at 37°C for 3 hours after addition of 10 uL of 10 times concentration BAP buffer solution (500 mM Tris-HCl (pH 9.0), 10 mM MgCl2) and 5 uL of bacterial alkaline phosphatase (0.5 unit/uL, Takara Shuzo Co., Ltd.).  The reaction liquid was extracted with 100 uL of phenol-chloroform mixture (phenol:chloroform:isoamyl alcohol=25:24:1, volume ratio), the aqueous layer was collected and extracted again with 100 uL of chloroform, and the aqueous layer was again collected.  10 uL of 3 M sodium acetate (pH 6.0) and 300 uL of ethanol were added to this liquid, which was then left at -80°C for 30 minutes and centrifuged to collect the precipitated DNA.  After being washed in 70% ethanol, this DNA was dissolved in 20 uL of TE buffer solution (10 mM Tris-HCl (pH 8.0), 1 mM<br>
EDTA).<br>
Meanwhile, 10 ug of SuperCos cosmid vector (Stratagene Co.) was digested with restriction enzyme Xbal in accordance with the Stratagene manual, the DNA terminals were de-phosphorylated with calf intestinal alkaline phosphatase (Takara Shuzo Co., Ltd.), and after being digested with restriction enzyme BamHI and purified, this was dissolved in 10 uL of TE buffer solution.<br>
2.5 uL of the Sau3Al partial digest solution of Mer-11107 DNA described above was added to luL of this cosmid DNA solution, and 1.5 uL of sterile purified water, 5 yL of DNA Ligation Kit (Takara Shuzo Co., Ltd.) Solution II and 10 \iL of Solution I were added in that order and incubated at 23°C for 10 minutes.  4 uL of the reaction liquid was packaged into a lambda-phage using Gigapack III XL Kit (Stratagene Co.).  When the resulting packaged liquid (total 500 uL) was subjected to a transduction test, colony formation ability was tested at 380 cfu (colony forming units)/uL.<br>
Example 3: Preparation of various probes<br>
(1) Preparation of probes comprising keto synthetase coding regions<br>
The following two primers, KS-3F and KS-4R, respectively comprising the nucleotide sequences in Sequence Nos. 10 and 11 below, were synthesized based on sequences that are generally conserved in the ketosynthase<br>
domains of polyketide synthases.<br>
KS-3F: 5'-GACCGCGGCTGGGACGTGGAGGG-3' (Sequence No. 10) KS-4R: 5'-GTGCCCGATGTTGGACTTCAACGA-3' (Sequence No. 11) These primers were used to carry out PCR under the following conditions.<br>
(PCR reaction solution composition)<br>
Sterile purified water	31 uL<br>
2x GC buffer	50 pL<br>
dNTP mixed solution	16 uL<br>
(2.5 mM each dATP, dGTP, dTTP and dCTP)<br>
KS-3F (100 pmol/uL)	0.5 uL<br>
KS-4R (100 pmol/uL)	0.5 uL<br>
Mer-11107 total DNA (100 ng/uL)	1 uL<br>
LA Taq polymerase (5 U/pL, Takara Shuzo Co., Ltd.)     1 uL<br>
(Reaction temperature conditions) 95°C 3 minutes<br>
(98°C 20 sec, 63°C 30 sec, 68°C 2 minutes)        30 cycles 72°C 5 minutes<br>
The 930 b DNA fragments amplified as a result of this reaction were electrophoresed on 0.8% agarose gel, and the isolated 930 b DNA fragments were excised and collected and purified using SUPREC-01 (Takara Shuzo Co., Ltd.).  Using 10 ng of the resulting DNA fragments as the template, 930 b DNA fragments comprising the keto synthetase coding region were amplified again under the same PCR conditions as above except that the number of reaction cycles was changed to 20.<br><br>
These DNA fragments were concentrated and purified using SUPREC-02 (Takara Shuzo Co., Ltd.), and 50yL of the resulting TE solution was taken as the probe solution.<br>
(2) Preparation of probe comprising cytochrome P450 gene region<br>
Two known cytochrome P450 genes were amplified from actinomycetes for purposes of preparing a cytochrome P450 gene probe.  That is, the two primers CB-1F and CB-2R comprising the sequences shown in the following Sequence Nos. 12 and 13 were synthesized for purposes of amplifying the ORF-A gene derived from Streptomyces thermotolerans ATCC11416 (Biosci. Biotechnol. Biochem. 59: 582-588, 1995). CB-1F: 5'-ATGACAGCTTTGAATCTGATGGATCCC-3' (Sequence No. 12) CB-2R: 5'-TCAGAGACGGACCGGCAGACTCTTCAGACG-3' (Sequence No. 13)<br>
Meanwhile, the two primers PKC-1F and PKC-2R comprising the sequences shown in the following Sequence Nos. 14 and 15 were synthesized for purposes of amplifying the pik-C gene derived from Streptomyces venezuelae ATCC15439 (Chem. Biol. 5: 661-667, 1998).<br>
PKC-1F: 5'-GTGCGCCGTACCCAGCAGGGAACGACC-3' (Sequence No. 14) PKC-2R: 5'-TCACGCGCTCTCCGCCCGCCCCCTGCC-3' (Sequence No. 15)<br>
These primers were used to carry out PCR under the following conditions.<br>
(PCR reaction solution composition)<br>
Sterile purified water	31 uL<br><br>
2x GC buffer	50 yL<br>
dNTP mixed solution	16 pL<br>
(2.5 mM each dATP, dGTP, dTTP, and dCTP)<br>
primer-F (100 pmol/uL)	0. 5 yL<br>
primer-R (100 pmol/yL)	0.5 pL<br>
ATCC11416 or ATCC15439 genome DNA (100 ng/yL)	1 yL<br>
LA Taq polymerase (5 U/yL, Takara Shuzo Co., Ltd.)     1 pL<br>
(Reaction temperature conditions) 95°C 3 minutes<br>
(98°C 20 sec, 63°C 30 sec, 68°C 2 minutes)        30 cycles 72°C 5 minutes<br>
The two 1.2 kb DNA fragments amplified as a result of this reaction were purified by QIAGEN PCR Purification Kit<br>
(QIAGEN Co.), and a mixed solution comprising 10 ng/yL of each DNA fragment was prepared and used as the probe.<br>
Example 4: Screening using probe comprising keto synthetase coding region<br>
An E. coli XL-lBlue MR host (Stratagene Co.) was transduced with the Mer-11107 genome DNA library packaged solution prepared in the above (2) in accordance with the Stratagene manual.  After transduction the bacterial suspension was dispensed and spread onto ten LB-50 pg/mL ampicillin-1.5% agar medium plates (inner diameter 90 mm, height 15 mm), and cultured for 18 hours at 37°C.  The colonies growing on each plate were transferred to HybondoN+ filters (Amersham Biosciences), alkali and<br><br>
neutral treated under the conditions described in the manual for the HybondoN+ filters, and dried for 2 hours at 80°C to fix DNA derived from the colonies onto the filters.<br>
The genome DNA library was screened by colony hybridization using an AlkPhos Direct System (Amersham Biosciences) with 100 ng of the 930 bp DNA fragment comprising the keto synthetase region prepared in Example 3 (1) as the probe.  Hybridization was performed for 2 hours at 65°C at a salt concentration of 0.5 M NaCl.  The conditions for hybridization and detection were those described in the manual attached to the AlkPhos Direct System.  Of the roughly 7,600 colonies tested, 59 colonies which hybridized strongly with the alkali phosphatase-labeled probe were isolated.  Cosmids were extracted and purified from E. coli clones derived from these colonies.<br>
Example 5: Selection and verification of cosmid clones having pladienolide biosynthesis gene region using probe comprising cytochrome P450 gene region<br>
Two yL of each of the cosmid DNA solutions obtained in Example 4 was spotted onto a HybondoN+ filter, alkali and neutral treated under the conditions described in the attached manual, and dried for 2 hours at 80°C to fix the DNA on the filter.  Hybridization was performed with these filters under the same conditions as in Example 4 using the cytochrome P450 gene fragment described in Example 3 as the probe.  One cosmid that hybridized strongly with the probe<br><br>
was selected as a result and named pKS58.<br>
The pKS58 DNA was partially digested with Sau3AI restriction enzyme, ligated with the BamHI-CIAP treated phage vector Zap Express (Stratagene Co.)/ and packaged into a lambda phage using a Gigapack III XL Kit (Stratagene Co.).  E. coli XLl-Blue MRF' was infected with this phage solution, and made to form a plaque.  Plaque hybridization was performed using the cytochrome P450 gene probe prepared in Example 3 (2) to subclone an approximately 2 kb lengh DNA fragment containing cytochrome P450 gene.<br>
This cytochrome P450 gene DNA fragment was sequenced, and two primers PDL58-1F and PDL58-2R having the sequences shown in the following Sequence Nos. 16 and 17 were synthesized from the N- and C-terminals, which are considered to be the cytochrome P450 coding regions. PDL58-1F: 5'-GCCCCGCATATGGATCTGGAAACCCAACTTCTC-3' (Sequence No. 16)<br>
PDL58-2R: 5'-GCACTAGTCAGCCGCGCTCGACGAGGAGGTG-3' (Sequence No. 17)<br>
These primers were used to carry out PCR under the following conditions.<br>
(PCR reaction solution composition)<br>
Sterile purified water	31 uL<br>
2x GC buffer	50 uL<br>
dNTP mixed solution	16 uL<br>
(2.5 mM each dATP, dGTP, dTTP and dCTP)<br><br>
PDL58-1F (100 pmol/uL)	0.5 uL<br>
PDL58-2R (100 pmol/uL)	0.5 pL<br>
pKS58 DNA (100 ng/pL)	1 uL<br>
LA Taq polymerase (5 U/uL, Takara Shuzo Co., Ltd.)     1 uL<br>
(Reaction temperature conditions) 95°C 3 minutes<br>
(98°C 20 sec, 63°C 30 sec, 68°C 2 minutes)        20 cycles 72°C 5 minutes<br>
The 1.2 kb DNA fragment amplified as a result of this reaction was purified with QIAGEN PCR Purification Kit<br>
(QIAGEN Co.), and digested with Ndel and Spel restriction enzymes.  After the reaction the DNA was electrophoresed on 0.8% agarose gel, the isolated 1.2 kb DNA fragment was excised and DNA was collected and purified using QIAGEN GelExtraction Kit (QIAGEN Co.).  This DNA fragment was inserted into the Ndel and Spel sites of the cytochrome P450 gene expression plasmid pT7NS-camAB (Japanese Patent Application No. 2002-110311) to construct pPDL96.<br>
E. coli BL21 (DE3) was transformed using this plasmid and cultured in M9CG medium (1.28% Na2HP04 • 7H2O, 0.3% KH2P04, 0.05% NaCl, 0.1% NH4C1, 1% casamino acid, 0.4% glucose, 1 mM MgCl2, 100 uM CaCl2, 50 ug/mL ampicillin) to a density of 0.8 ODeoo (optical density at 600 nm) .  5-Aminolevulinic acid was added to 80 ug/mL and IPTG to 0.1 mM, and cultivation was continued at 22°C for 25 hours to induce the cytochrome P450 protein.  After induction, the mycelia<br><br>
were collected and suspended in 5 mL of CV buffer solution (50 mM NaP04 (pH 7.3), 1 mM EDTA, 10% glycerol, 1 mM glucose).  1 mL of this suspension was taken in a test tube and 5 uL of a DMSO solution (50 mg/mL) of ME-265 (the 6-position deoxide of pladienolide B) was added and incubated at 28°C for 15 hours.  1 mL of acetonitrile was added and mixed with this reaction solution, which was then centrifuged and supernatant analyzed by HPLC under the following conditions to confirm conversion to pladienolide B.  These results lead the pladienolide biosynthesis gene region is involved in pKS58.<br>
(HPLC analysis conditions) Analyzer: Shimadzu HPLC 10 Avp<br>
Column: Develosil ODS UG-3 (cp 4.6 mm * 50 mm 3 urn) Mobile phase:        45% to 55% methanol (0 to 5 minutes)<br>
55% methanol (5 to 13 minutes) 55% to 70% methanol (13 to 21 minutes) 45% methanol (21 to 25 minutes) Flow rate: 1.2 mL/min Detection: UV 240 nm Injection volume: b uL Column temperature: 40°C Analysis time: 25 minutes Retention time: ME-265: 20 minutes,<br>
Pladienolide B: 13 minutes Example 6: Selection of cosmid comprising biosynthesis<br>
gene cluster neighboring cytochrome P450 gene<br>
A cosmid comprising the biosynthesis gene cluster neighboring the cytochrome P450 gene obtained in Example 5 was selected from the 59 cosmid DNA samples obtained in Example 4.<br>
The 59 cosmid DNA samples were digested with restriction enzymes EcoRI and BamHI, and the DNA obtained in each case was electrophoresed on agarose gel and subjected to Southern hybridization using as probes the KS domain DNA (aveA2 KS6 domain) and the AT domain DNA (aveAl AT2 domain) of the avermectin aglycone biosynthesis gene (see Proc. Natl. Acad. Sci. USA 96 (1999) 9509-9514; JP-A 2000-245457; or WO 00/50605) and the cytochrome P450 gene obtained in Example 5.<br>
Those cosmids having DNA fragments that hybridized at the same length were grouped on the basis of the electrophoresis patterns of DNA digested with restriction enzymes EcoRI and BamHI, and the hybridization band patterns using the various probes..  Of these, all but one of the cosmids exhibiting similar patterns were deleted, and the remaining cosmids were organized according to partially matching band patterns.  Beginning with the pKS58 cosmid comprising the cytochrome P450 gene obtained in Example 5, pKS56 and pKS54 were selected as cosmids neighboring the side comprising the polyketide synthetase gene from the cytochrome P450 gene side, and pKS35 was<br>
selected as a cosmid neighboring pKS54.  pKS23 was also selected as a cosmid neighboring the cosmid pKS58 from cytochrome P450 gene side to the side not comprising the polyketide synthetase gene.  As a result, as shown in Figure 2, pKS23, pKS58, pKS56, pKS54 and pKS35 were selected as cosmid clones encompassing the pladienolide biosynthesis gene cluster.<br>
Example 7: Production of pladienolide biosynthetic gene cluster-deficient strain<br>
From among the cosmids selected in Example 6, a disrupted biosynthetic gene strain was produced using the DMA of pKS56, which was thought to contain the polyketide synthesis region.<br>
The cosmid DNA from pKS56 was digested with the BamHI restriction enzyme and a 2 kb spectinomycin resistance gene<br>
(aminoglycoside 3"-adenyltransferase, abbreviated hereafter as aadA) was ligated with the BamHI digestion fragments using an NEB Quick Ligation Kit (New England Biolabs Inc.). This resulted in the BamHI-mediated deletion of 30 kb of the cosmid DNA from pKS56 (region A in Fig. 2: nucleotides 31194 to 61374 in Sequence No. 1), and cosmid p56aadA, which was recombined with the 2 kb spectinomycin resistance gene, was obtained.  The aadA was prepared by digesting the plasmid pHP45omega (Gene 190, 315-317 (1997)) with the BamHI restriction enzyme.<br>
The shuttle vector pKU253 was used to incorporate<br>
cosmid p56aadA into the Mer-11107 strain.  p56aadA was digested with the EcoRI restriction enzyme, and 14 kb lacking any cosmid vector regions was separated by agarose gel electrophoresis and was purified using a Gene Clean II Kit (BiolOl Co.).  The obtained 14 kb EcoRI fragment was ligated with the EcoRI digest of the shuttle vector pKU253 using an NEB Quick Ligation Kit, yielding pKU253-56aadA. As shown in Fig. 3, pKU253 was constructed by joining the E. coli plasmid pUC19 (Gene, 33(1), 103-119, 1985) to the base of the SCP2 plasmid originating from the actinomycetes Streptomyces coelicolor A3 (2) (<j. gen. microbiol. and introducing the aminoglycoside resistance gene aphll conjugation orit bacterial.></j.>
The resulting pKU253-56aadA was transformed into conjugated E. coli S17-1 (ATCC47055) by electroporation to obtain S17-l/pKU253-56aadA.  The resulting Sl7-l/pKU253-56aadA was inoculated into 10 mL of LB medium (1% bacto tryptone, 0.5% yeast extract, 0.5% NaCl) comprising 25 ug/mL of kanamycin and 200 iag/mL of spectinomycin and shaking cultured at 30°C for 2 hours, and the mycelia were collected, washed twice with 10 mL of LB medium and suspended in 5 mL of LB medium.  This was the donor suspension.<br>
While the donor suspension was being prepared, Mer-11107 was inoculated into 10 mL of TSB medium (Trypto-Soya<br>
broth: Nissui Pharmaceutical Co., Ltd.) and shaking cultured at 30°C for 5 hours, and the mycelia were collected, washed twice with 10 mL of sterile water and suspended in 1 mL of sterile water.  This was the recipient suspension.<br>
500 |j,L of the S17-l/pKU253-56aadA donor suspension was mixed with 10 |iL of the Mer-11107 recipient suspension followed by plating on Actino Medium No. 4 agar medium (Nihon Pharmaceutical Co., Ltd.).  After culture at 30°C for 18 hours, 2.5 mL SNA (0.8% nutrient medium: Difco, 0.4% agar) containing 2 mg/mL ribostamycin was layered on. Incubation for at 30°C 7 days then gave a ribostamycin-resistant pKU253-56aadA transformant strain.<br>
The resulting pKU253-56aadA transformant was seeded to 10 mL TSB medium that did not contain ribostamycin and shaking cultured at 30°C for 24 hours.  The plasmid vector pKU253 has a poor replication efficiency in the Mer-11107 strain, and the Mer-11107 strain is unable to maintain pKU253 when cultured on medium lacking a drug resistance marker (ribostamycin).<br>
The cells were collected from the pKU253-56aadA transformant culture medium and were washed twice with 10 mL sterile water and suspended in 10 mL sterile water.  The suitably diluted suspension was plated on YMS agar medium (0.4% yeast extract, 1% malt extract, 0.4% soluble starch, 2% agar, 10 mM calcium chloride) containing 200<br>
spectinomycin and was cultured at 30°C for 4 days.  Single colonies that grew on the spectinomycin-containing YMS agar medium were reseeded to YMS agar medium containing 200 pig/mL spectinomycin and YMS agar medium containing 200 (j,g/mL ribostamycin followed by culture at 30°C for 2 days.<br>
After culture, the spectinomycin-resistant, ribostamycin-sensitive strain was selected, and it was confirmed by Southern hybridization that the spectinomycin resistance gene had been inserted in the region regarded as the targeted biosynthetic gene on the genomic DNA.  The resulting strain was named Mer-11107-56::aadA.<br>
Example 8: Pladienolide productivity test of pladienolide biosynthetic gene cluster-deficient strain<br>
The productivity test of pladienolide B was conducted in a total of three strains: the Mer-11107-56::aadA strain obtained in Example 7, the original Mer-11107 strain and its transformant, the Mer-11107/pKU253 strain as control.<br>
200 uL each of frozen seed of Mer-11107-56::aadA strain prepared in Example 7, Mer-11107 strain and Mer-11107/pKU253 strain was inoculated into 20 mL of seed medium (soluble starch 2%, ESUSAN-MEAT 2%, yeast extract 0.5%, K2HP04 0.1%, MgS04-7H20 0.25%, CaC03 0.3%, pH not adjusted) and incubated at 25°C for 2 days.<br>
300 uL of the resulting seed culture broth was inoculated into 30 mL of seed culture medium (5% Stabilose, 1% glucose, 3% Pharmamedia, 2% (3-cyclodextrin, 0.1% CaC03,<br>
pH 7.5) and cultured at 25°C for 4 and 5 days. After the completion of the cultivation, the resulting culture liquid was extracted by addition of 9 times the amount of acetonitrile.  The amounts of pladienolide B in the resulting extract was measured by HPLC.  The measurement results are shown in Table I.<br>
The HPLC measurement conditions are shown below. Analyzer: Shimadzu HPLC lOAvp<br>
Column: Develosil ODS UG-3 (4.6 mm x 50 mm 3 jam)<br>
Mobile phase (volume %): 45% to 55% methanol (0 to 5 min)<br>
55% methanol (5 to 13 min) 55% to 70% methanol (13 to 21 min) 45% methanol (21 to 25 min)<br>
Flow rate:  1.2 mL/minute<br>
Detection: UV 240 nm<br>
Injection capacity: 5 uL<br>
Column temperature: 40°C<br>
Analyzing time: 25 minutes<br>
Retention time: pladienolide B: 13 min<br>
(Table Remove)<br>
These results confirmed that the Mer-11107-56::aadA strain, which had been subjected to deletion of the A region shown in Fig. 2, was completely unable to produce pladienolide B.  This demonstrated that the gene at the A region is related to pladienolide biosynthesis.<br>
Example 9: Determination of the nucleotide sequence of the pladienolide biosynthetic gene cluster<br>
The nucleotide sequence of a DNA group coding for the pladienolide biosynthesis gene was determined.  The gene at the A region shown in Fig. 2 was confirmed to be related to pladienolide biosynthesis by the fact that the A site-deficient strain in Example 8 was unable to produce pladienolide B.  The nucleotide sequence of the DNA fragment inserted into each of the 4 cosmids selected in Example 6, pKS35, pKS54, pKS58 and pKS23, was therefore determined.<br>
Each cosmid, after isolation by the cesium chloride method, was then sheared to approximately 1 kb using a HydroShear (Genomic Solutions Inc.) and subcloned using a BKL Kit (Takara Shuzo Co., Ltd.).<br>
The resulting subclones were subjected to a cycle sequence reaction (Amersham Biosciences Co.) using fluorescent-labeled primers and the nucleotide sequences of respective fragments were determined (MegaBACE 1000: Amersham Biosciences Co.), thus, an approximately 75 kb nucleotide sequence comprising DNA associated with<br>
pladienolide biosynthesis (see Sequence No. 1) was determined.<br>
A search of the open reading frames (ORF) in this DNA showed it to contain the following 8 ORFs. pldA I: bases 8340 to 27935 pldA II: bases 28021 to 49098 pldA III: bases 49134 to 60269 pldA IV: bases 60269 to 65692 pldB: bases 65707 to 66903 pldC: bases 68160 to 66970 pldD: bases 69568 to 68270 pldR: bases 72725 to 70020<br>
The correlation between the ORFs and the cosmids is shown in Fig. 2.<br>
Example 10: Preparation of pladienolide 6-hydroxylase gene (pldB)-deficient strain<br>
It has been demonstrated that pladienolide is biosynthesized by the biosynthetic pathway shown in Fig. 1 from the approximately 75 kb nucleotide sequence (see Sequence No. 1) comprising DNA associated with pladienolide biosynthesis which was sequenced in Example 9.  A pldB-deficient strain was therefore prepared as described hereinbelow based on the idea that it would be possible to obtain a strain that produces only ME-265, the 6-deoxy form of pladienolide B, by disrupting only the cytochrome P450 gene pldB.<br><br>
Four primers, pldB-L-Bg!2F, pldB-L-Hind3R, pldB-R-Hind3F and pldB-R-Bg!2R, comprising the nucleotide sequences shown in the following Sequence Nos. 18, 19, 20 and 21, were synthesized based on the nucleotide sequence of Sequence No. 1. pldB-L-Bg!2F: 5'-GGGAGATCTAGAGGCCGGTTACCTCTACGAGTA-3'<br>
(Sequence No. 18) pldB-L-Hind3R: 5'-GGGAAGCTTGCGATGAGCTGTGCCAGATAG-3'<br>
(Sequence No. 19) pldB-R-Hind3F: 5'-GGGAAGCTTGAACTGGCGCGACAGTGTCTT-3'<br>
(Sequence No. 20) pldB-R-Bg!2R: 5'-GGGAGATCTGCAGCGGATCGTCTTCGAGACCCTT-3'<br>
(Sequence No. 21)<br>
PCR was performed under the following conditions using these primers.<br>
(PCR reaction solution composition)<br>
Sterile purified water	30 uL<br>
2x GC buffer	50 uL<br>
dNTP mixed solution	16 uL<br>
(2.5 mM each dATP, dGTP, dTTP and dCTP)<br>
pldB-L-Bg!2F or pldB-R-Hind3F (50 pmol/uL)	1 uL<br>
pldB-L-Hind3R or pldB-R-Bg!2R (50 pmol/uL)	1 uL<br>
Mer-11107 total DNA (100 ng/]iL)	I uL<br>
LA Taq polymerase (5 U/uL, Takara Shuzo Co., Ltd.)     1 uL<br>
(Reaction temperature conditions) 95°C 3 minutes<br><br>
(98°C 20 sec, 63°C 30 sec, 68°C 2 minutes)       30 cycles 72°C 5 minutes<br>
As a result, a 1.57 kb DNA fragment (DMA fragment LI) comprising nucleotides 64756 to 66302 in the Sequence No. 1 was amplified by the reaction using pldB-L-Bg!2F and pldB-L-Hind3R, while a 1.54 kb DNA fragment (DNA fragment Rl) comprising nucleotides 66849 to 68368 in the Sequence No. 1 was amplified from the reaction using pldB-R-Hind3F and pldB-R-Bg!2R.  DNA fragments LI and Rl were purified with a QIAGEN PCR purification Kit (QIAGEN Co.), and digested with restriction enzymes Bglll and Hindlll.<br>
The DNA fragments LI and Rl which had been digested with restriction enzymes Bglll and Hindlll, a 2.3 kb hygromycin B resistance gene (derived from pHP45omegahyg: Gene 190, 315-317, 1997, sometimes abbreviated hereunder as "hyg") which had been digested with restriction enzyme Hindlll and the shuttle vector pKU253 (see Fig. 3) which had been digested with restriction enzyme BamHI were all four connected to DNA ligation kit Ver. 2.1 (Takara Shuzo Co., Ltd.).  A roughly 5.4 kb DNA fragment having the hygromycin B resistance gene inserted between DNA fragments LI and Rl was thus inserted into pKU253 to construct a roughly 21.4 kb plasmid called pKU253-Ll-hyg-Rl.<br>
The resulting pKU253-Ll-hyg-Rl was transformed into conjugative E. coli S17-1 by electroporation to obtain S17-l/pKU253-Ll-hyg-Rl.  The resulting S17-l/pKU253-Ll-hyg-Rl<br><br>
was inoculated into 10 mL of LB medium (1% bacto tryptone, 0.5% yeast extract, 0.5% NaCl) comprising 25 ug/mL of kanamycin and 100 ug/mL of hygromycin B and shaking cultured at 30°C for 2 hours, and the mycelia were collected, washed twice with 10 mL of LB medium and suspended in 5 mL of LB medium.  This was the donor suspension.<br>
While the donor suspension was being prepared, Mer-11107 was inoculated into 10 mL of TSB medium (Trypto-Soya broth: Nissui Pharmaceutical Co., Ltd.) and shaking cultured at 30°C for 5 hours, and the mycelia were collected, washed twice with 10 mL of sterile water and suspended in 1 mL of sterile water.  This was the recipient suspension.<br>
500 uL of the S17-l/pKU253-Ll-hyg-Rl donor suspension was mixed with 10 ]iL of the Mer-11107 recipient suspension, and plated to Actino Medium No. 4 agar medium (Nihon Pharmaceutical Co., Ltd.).  After incubated at 30°C for 18 hours, this was covered with 2.5 mL of SNA (0.8% nutrient medium: Difco, 0.4% agar) comprising 2 mg/mL ribostamycin, and incubated at 30°C for 7 days to obtain a ribostamycin-resistant pKU253-Ll-hyg-Rl transformant strain.<br>
The resulting pKU253-Ll-hyg-Rl transformant strain was inoculated into 10 mL of TSB medium containing no ribostamycin, and shaking cultured at 30°C for 24 hours. Mycelia were collected from the pKU253-Ll-hyg-Rl<br><br>
transformant culture broth, washed twice with 10 mL of sterilized water and suspended in 10 mL of sterilized water. After being diluted appropriately, the suspension was plated to YMS agar medium (0.4% yeast extract, 1% wheat germ extract, 0.4% soluble starch, 2% agar, 10 mM calcium chloride) comprising 200 pg/mL hygromycin B, and incubated at 30°C for 4 days.  Single colonies growing on the YMS agar medium comprising hygromycin B were transplanted to YMS agar medium comprising 200 ug/mL of hygromycin B and YMS agar medium comprising 200 pg/mL of ribostamycin, and incubated at 30°C for 2 days.<br>
After incubated, a hygromycin B -resistant, ribostamycin-sensitive strain was selected.  The resulting strain, called Mer-11107 pldB::hyg, was a pldB-deficient strain lacking 546 bp (nucleotides 66303 to 66848 in the Sequence No. 1) of the pldB gene from the genome, with the hygromycin B resistance gene inserted in its place.<br>
Example 11: Pladienolide productivity test of pladienolide 6-position hydroxylase gene (pldB)-deficient strain<br>
200 pL of frozen seed of the Mer-11107 pldB::hyg strain obtained in Example 10 was inoculated into 20 mL of seed medium (soluble starch 2%, ESUSAN-MEAT 2%, yeast extract 0.5%, K2HP04 0.1%, MgS04-7H20 0.25%, CaC03 0.3%, pH not adjusted) and incubated at 25°C for 2 days.<br>
300 uL of the resulting seed culture broth was<br><br>
inoculated into 30 mL of seed culture medium (5% Stabilose, 1% glucose, 3% Pharmamedia, 2% |3-cyclodextrin, 0.1% CaC03, pH 7.5) and cultured at 25°C for 4 and 5 days.<br>
After the completion of the cultivation, 20 mL of the resulting culture liquid was extracted by adding an equal amount of acetonitrile thereto.  Part of this extract was taken and diluted with 5 times the amount of acetonitrile, and levels of pladienolide B and ME-265 were measured by HPLC under the following conditions.  The measurement results are shown in Table 2.<br>
(HPLC analysis conditions) Analyzer: Shimadzu HPLC lOAvp<br>
Column: Develosil ODS UG-3 (4.6 mm x 50 mm 3 urn)<br>
Mobile phase (vol%): 45% to 55% methanol (0 to 5 minutes)<br>
55% methanol (5 to 13 minutes) 55% to 70% methanol (13 to 17 minutes) 70% methanol (17 to 35 minutes) 45% methanol (35 to 40 minutes)<br>
Flow rate: 1.2 mL/min<br>
Detection: UV 240 nm<br>
Injection volume: 10 uL<br>
Column temperature: 40°C<br>
Analyzing time: 35 minutes<br>
Retention time: ME-265: 22 minutes, pladienolide B: 16<br>
minutes<br>
(Table Remove)<br>
Example 12: Isolation and purification of ME-265 and its structure confirmation<br>
The acetonitrile extraction solution obtained in Example 11 was filtered and the mycelia were washed with 10 mL water and 40 mL water.  The filtrate and the washed solution were then combined and extracted with 100 mL ethyl acetate.  To the aqueous layer was added 50 mL brine, which was then re-extracted with 50 mL ethyl acetate.  The ethyl acetate layers were then combined and washed with 50 mL brine, dried over anhydrous sodium sulfate and then the solvent was removed.  Then, the residue was purified by thin-layer chromatography (TLC, Merck Art. 5744, developing solvent: toluene:acetone=2:1), to give 20.3 mg ME-265.<br>
1H-NMR spectrum (CD3OD, 500 MHz) : 5 ppm (integration, multiplicity, coupling constant J (Hz)):<br>
0.87(3H,d,J=7.OHz), 0.90(3H,d,J=7.OHz), 0.94(3H,d,J=7.3Hz), 0.97(3H,d, J=7.0Hz), 1.08(3H,d,J=7.OHz) , 1.17-1.21 (1H,m), 1.24-1.36(2H,m), 1.42-1.52(3H,m), 1.61-1.66(3H,m), 1.74(3H,d,J=l.lHz), 1.89-1.96(lH,m), 2.00(3H,s), 2.41-2.47(lH,m), 2.43(lH,dd,J=5.5,13.9Hz), 2.51-2.58(1H,m),<br>
2.56(lH,dd,J=3.7,13.9Hz), 2.65(1H,dd,J=2.2,8.IHz), 2.72(lH/dt,J=2.2,5.9Hz), 3.51(1H, dt,J=4.4,8.4Hz), 3.75-3.80(lH,m), 4.91 (lH,dd,J=8.8,10.6Hz) , 5.00(IE,d,J=10.6Hz), 5.42 (lH,dd,J=9.2Hz/15.0Hz), 5.49(1H,dd,J=9.2,15.OHz), 5.65(lH,dd,J=8.4,IS.OHz), 6.08(lH,d,J=10.6Hz), 6.32(lH,dd,J=l0.6,IS.OHz)<br>
These results demonstrated that the pldB-deficient strain Mer-11107 pldB::hyg does not produce pladienolide B and does produce ME-265.  That is, ME-265 could be produced and obtained by the method described above.<br>
Example 13: Production of pladienolide 7-acylation enzyme gene (pldC)-deficient strain<br>
It has been shown that a pladienolide is biosynthesized by the biosynthesis pathway shown in Fig. 1 from the roughly 75 kb nucleotide sequence comprising DNA participating in pladienolide biosynthesis which was sequenced Example 9 (see Sequence No. 1).  A pldC-deficient strain was thus prepared by the following methods with the idea that a strain producing the 7-deacyl form of pladienolide (pladienolide B12) could be obtained by disrupting only the 7-acylation enzyme gene, pldC.<br>
Four primers, pldB-L-Bg!2F, pldC-L-Hind3R, pldC-R-Hind3F and pldC-R-Bg!2R, having the nucleotide sequences shown in the following Sequence Nos. 18, 22, 23 and 24, were synthesized based on the nucleotide sequence of Sequence No. 1.<br>
pldB-L-Bg!2F: 5'-GGGAGATCTAGAGGCCGGTTACCTCTACGAGTA-3'<br>
(Sequence No. 18) pldC-L-Hind3R: 5'-GGGAAGCTTCCAGTCTCGTGCTCACCAA-3'<br>
(Sequence No. 22) pldC-R-Hind3F: 5'-GGGAAGCTTAGGCCCGTTGGAGAAGCTGTT-3'<br>
(Sequence No. 23) pldC-R-Bg!2R: 5'-GGGAGATCTGCAGCCTCATCCTCACCGAGCTGAA-3'<br>
(Sequence No. 24)<br>
PCR was performed under the following conditions using these primers.<br>
(PCR reaction solution composition)<br>
Sterile purified water	30 uL<br>
2x GC buffer	50 uL<br>
dNTP mixed solution	16 uL<br>
(2.5 mM each dATP, dGTP, dTTP and dCTP)<br>
pldB-L-Bg!2F or pldC-R-Hind3F (50 pmol/uL)	1 uL<br>
pldC-L-Hind3R or pldC-R-Bg!2R (50 pmol/yL)	1 uL<br>
Mer-11107 strain total DNA (100 ng/uL)	1 uL<br>
LA Taq polymerase (5 U/uL, TAKARA HOLDINGS INC.)    1 uL<br>
(Reaction temperature conditions) 95°C 3 minutes<br>
(98°C 20 sec, 63°C 4 sec)	30 cycles<br>
68°C 5 minutes<br>
As a result of this reaction, an approximately 2.5 kb DNA fragment (DNA fragment L2) comprising bases 64756 to 67220 in the Sequence No. 1 was amplified by the reaction<br>
using pldB-L-Bg!2F and pldC-L-Hind3R, while an approximately 3.0 kb DNA fragment (DMA fragment R2) comprising bases 68106 to 71112 in the Sequence No. 1 was amplified by the reaction using pldC-R-Hind3F and pldC-R-Bgl2R.  DNA fragments L2 and R2 were purified with a QIAGEN PCR Purification Kit (QIAGEN Co.) and were digested with restriction enzymes Bglll and Hindlll.<br>
The DNA fragments L2 and R2 which had been digested with restriction enzymes Bglll and Hindlll, a 2.3 kb hygromycin B resistance gene (derived from pHP45omegahyg: Gene 190, 315-317, 1997, sometimes abbreviated hereunder as "hyg") which had been digested with restriction enzyme Hindlll and the shuttle vector pKU253 (see Fig. 3) which had been digested with restriction enzyme BamHI were all four connected to DNA ligation kit Ver. 2.1 (Takara Shuzo Co., Ltd.).  A roughly 7.8 kb DNA fragment having the hygromycin B resistance gene inserted between DNA fragments L2 and R2 was thus inserted into pKU253 to construct an approximately 23.8 kb plasmid called pKU253-L2-hyg-R2.<br>
The resulting pKU253-L2-hyg-R2 was transformed into conjugative E. coli S17-1 by electroporation to obtain S17-l/pKU253-L2-hyg-R2.  The resulting S17-l/pKU253-L2-hyg-R2 was inoculated into 10 mL of LB medium (1% bacto tryptone, 0.5% yeast extract, 0.5% NaCl) comprising 25 ug/mL of kanamycin and 100 ug/mL of hygromycin B and shaking cultured at 30°C for 2 hours, and the mycelia were<br>
collected, washed twice with 10 mL of LB medium and suspended in 5 mL of LB medium. This was the donor suspension.<br>
While the donor suspension was being prepared, Mer-11107 was inoculated into 10 mL of TSB medium (Trypto-Soya broth: Nissui Pharmaceutical Co., Ltd.) and shaking cultured at 30°C for 5 hours, and the mycelia were collected, washed twice with 10 mL of sterile water and suspended in 1 mL of sterile water.  This was the recipient suspension.<br>
500 yL of the S17-l/pKU253-L2-hyg-R2 donor suspension was mixed with 10 yiL of the Mer-11107 recipient suspension, and plated to Actino Medium No. 4 agar medium (Nihon Pharmaceutical Co., Ltd.).  After incubated at 30°C for 18 hours, this was covered with 2.5 mL of SNA (0.8% nutrient medium: Difco, 0.4% agar) comprising 2 mg/mL ribostamycin, and incubated at 30°C for 7 days to obtain a ribostamycin-resistant pKU253-L2-hyg-R2 transformant strain.<br>
The resulting pKU253-L2-hyg-R2 transformant strain was inoculated intp 10 mL of TSB medium containing no ribostamycin, and shaking cultured at 30°C for 24 hours. Mycelia were collected from the pKU253-L2-hyg-R2 transformant culture broth, washed twice with 10 mL of sterilized water and suspended in 10 mL of sterilized water. After being diluted appropriately, the suspension was plated to YMS agar medium (0.4% yeast extract, 1% wheat<br>
germ extract, 0.4% soluble starch, 2% agar, 10 mM calcium chloride) comprising 200 ug/mL hygromycin B, and incubated at 30°C for 4 days.  Single colonies growing on the YMS agar medium comprising hygromycin B were transplanted to YMS agar medium comprising 200 ug/mL of hygromycin B and YMS agar medium comprising 200 ug/mL of ribostamycin, and incubated at 30°C for 2 days.<br>
After incubated, a hygromycin B-resistant, ribostamycin-sensitive strain was selected.  The resulting strain, called Mer-11107 pldC::hyg, was a pldC-deficient strain lacking 886 bp (bases 67221 to 68105 in Sequence No. 1) of the pldC gene from the genome, with the hygromycin B resistance gene inserted in its place.<br>
Example 14: Pladienolide production test of pladienolide 7-acylation enzyme gene (pldC)-deficient strain<br>
200 uL of frozen seed of the Mer-11107 pldC::hyg strain obtained in Example 13 was inoculated into 20 mL of seed medium (soluble starch 2%, ESUSAN-MEAT 2%, yeast extract 0.5%, K2HP04 0.1%, MgS04-7H20 0.25%, CaC03 0.3%, pH not adjusted) and incubated at 25°C for 2 days.<br>
300 uL of the resulting seed culture broth was inoculated into 30 mL of seed culture medium (5% Stabilose, 1% glucose, 3% Pharmamedia, 2% p-cyclodextrin, 0.1% CaC03, pH 7.5) and cultured at 25°C for 4 and 5 days.<br>
After the completion of the cultivation, 25 mL of the<br>
resulting culture liquid was extracted by adding an equal<br>
amount of acetonitrile thereto.  Part of this extract was<br>
taken and diluted with 5 times the amount of acetonitrile,<br>
and levels of pladienolide B and pladienolide Bi2 were<br>
measured by HPLC under the following conditions.  The<br>
measurement results are shown in Table 3. (HPLC analysis conditions)<br>
Analyzer: Shimadzu HPLC lOAvp<br>
Column: Develosil CDS UG-3 (4.6 mm x 50 mm 3 urn)<br>
Mobile phase (vol%): 45% to 55% methanol (0 to 5 minutes)<br>
55% methanol (5 to 13 minutes) 55% to 70% methanol (13 to 17 minutes; 70% methanol (17 to 35 minutes) 45% methanol (35 to 40 minutes)<br>
Flow rate: 1.2 mL/min<br>
Detection: UV 240 nm<br>
Injection volume: 10 uL<br>
Column temperature: 40°C<br>
Analysis time: 35 minutes<br>
Retention time: pladienolide Bi2: 16 minutes, pladienolide<br>
B: 12 minutes<br>
(Figure Remove)<br>
Example 15: Isolation and purification of pladienolide 612 and its structure confirmation<br>
The acetonitrile extraction solution obtained in Example 14 was filtered and further washed with 10 mL water and 10 mL acetonitrile.  The filtrate was extracted with 40 mL ethyl acetate, and the organic layer was dried over sodium sulfate, filtered and evaporated.  The resulting residue 91.4 mg was purified by thin-layer chromatography (TLC, Merck Art. 5744, developing solvent: hexane:ethyl acetate=10:50), to give pladienolide Bi2 (Rf=0.46, 3.1 mg).<br>
1.	Molecular weight: 478, ESI-MS m/z 501(M+Na)+, 477(M-H)~<br>
2.	^-NMR spectrum (CD3OD, 500 MHz): 5 ppm (integration,<br>
multiplicity, coupling constant J (Hz)):<br>
0.89(3H,d,J=6.7Hz), 0.90(3H,d,J=7.IHz), 0.94(3H,t,J=7.5Hz),<br>
1.07 (3H,d, J=6.8 Hz) , 1.08(3H,d,J=6.8Hz), 1.16-1.26(2H,m),<br>
1.27-1.36(lH,m), 1.41-1.67(7H,m), 1.74(3H,d,J=l.IHz),<br>
2.42(lH,dd,J=5.4,14.2Hz), 2.44-2.58(2H,m),<br>
2.56(lH,dd,J=3.5,14.IHz), 2.65(1H,dd,J=2.3,8.2Hz),<br>
2.72(lH,dt,J=2.3,6.0HZ), 3.51(1H,dt,J=4.4,8.6Hz),<br>
3.57(lH,dd,J=9.6,9.6HZ), 3.72-3.79(lH,m),<br>
5.00(lH,d,J=10.7Hz),5.30(lH,dd, J=9.7,15.IHz) ,<br>
5.46(lH,dd,J=9.5,15.0Hz), 5.65(1H,dd,J=8.4, 15.IHz),<br>
6.07(lH,d,J=10.9Hz), 6.32(1H,dd,J=10.9,15.IHz)<br>
(Figure Remove)<br>
Pladienolide Bi2<br>
These results confirmed that the Mer-11107 pldC::hyg strain, which is a pldC-deficient strain, did not produce pladienolide B and did produce pladienolide Bi2.  In other words, pladienolide 612 could be produced and obtained by the method described above.<br>
Example 16: Production of pladienolide 18,19-epoxidase gene (pldD)-deficient strain<br>
It has been demonstrated that pladienolide is biosynthesized by the biosynthetic pathway shown in Fig. 1 from the approximately 75 kb nucleotide sequence (see Sequence No. 1) comprising DMA associated with pladienolide biosynthesis which was sequenced in Example 9.  A pldD-deficient strain was therefore prepared as described hereinbelow based on the idea that it would be possible to obtain a strain that produces the 7-deacyl, 18,19-olefin form of pladienolide (pladienolide Z) by disrupting the 18,19-epoxidase gene (pldD) and inhibiting the expression of the downstream 7-acylation enzyme gene (pldC).<br>
Four primers, pldD-L-Bg!2F, pldD-L-Hind3R, pldD-R-Hind3F and pldD-R-Bg!2R, comprising the nucleotide<br>
sequences shown in the following Sequence Nos. 25, 26, 27 and 28, were synthesized based on the nucleotide sequence of Sequence No. 1. pldD-L-Bg!2F: 5'-GGGAGATCTAGACCTGTCCATGGATCTGGAAAC-3'<br>
(Sequence No. 25} pldD-L-Hind3R: 5'-GGGAAGCTTCGGATCGTCTTCGAGACCCTT-3'<br>
(Sequence No. 26) pldD-R-Hind3F: 5'-GGGAAGCTTGTGGGGTGCCCTTTCTGACTT-3'<br>
(Sequence No. 27) pldD-R-Bg!2R: 5'-GGGAGATCTGCAGGAGGAGCTGCTCGGGCTGAA-3'<br>
(Sequence No. 28)<br>
PCR was performed under the following conditions using these primers.<br>
(PCR reaction solution composition)<br>
Sterile purified water	30 pL<br>
2x GC buffer	50 uL<br>
dNTP mixed solution	16 uL<br>
(2.5 mM each dATP, dGTP, dTTP, and dCTP)<br>
pldD-L-Bg!2F or pldD-R-Hind3F (50 pmol/uL)	1 uL<br>
pldD-L-Hind3R or pldD-R-Bg!2R (50 pmol/uL)	1 uL<br>
Mer-11107 total DNA (100 ng/uL)	1 pL<br>
LA Taq polymerase (5 U/pL, Takara Shuzo Co., Ltd.)     1 uL<br>
(Reaction temperature conditions) 95°C 3 minutes<br>
(98°C 20 sec, 63°C 4 minutes)	30 cycles<br>
68°C 5 minutes<br>
As a result, a 2.7 kb DNA fragment (DNA fragment L3) comprising bases 65700 to 68368 in Sequence No. I was amplified by the reaction using pldD-L-Bg!2F and pldD-L-Hind3R, while a 2.4 kb DNA fragment (DNA fragment R3) comprising bases 69514 to 71951 in Sequence No.1 was amplified from the reaction using pldD-R-Hind3F and pldD-R-Bgl2R.  DNA fragments L3 and R3 were purified with a QIAGEN PCR purification Kit (QIAGEN Co.), and digested with restriction enzymes Bglll and Hindlll.<br>
The DNA fragments L3 and R3 which had been digested with restriction enzymes Bglll and Hindlll, a 2.3 kb hygromycin B resistance gene (derived from pHP45omegahyg: Gene 190, 315-317, 1997, sometimes abbreviated hereunder as "hyg") which had been digested with restriction enzyme Hindlll and the shuttle vector pKU253 (see Fig. 3) which had been digested with restriction enzyme BamHI were all four connected to DNA ligation kit Ver. 2.1 (Takara Shuzo Co., Ltd.).  A roughly 7.4 kb DNA fragment having the hygromycin B resistance gene inserted between DNA fragments L3 and R3 was thus inserted into pKU253 to construct an approximately 22.4 kb plasmid called pKU253-L3-hyg-R3.<br>
The resulting pKU253-L3-hyg-R3 was transformed into conjugative E. coli S17-1 by electroporation to obtain S17-l/pKU253-L3-hyg-R3.  The resulting S17-l/pKU253-L3-hyg-R3 was inoculated into 10 mL of LB medium (1% bacto tryptone, 0.5% yeast extract, 0.5% NaCl) comprising 25 ug/mL of<br>
kanamycin and 100 ug/mL of hygromycin B and shaking cultured at 30°C for 2 hours, and the mycelia were collected, washed twice with 10 mL of LB medium and suspended in 5 mL of LB medium.  This was the donor suspension.<br>
While the donor suspension was being prepared, Mer-11107 was inoculated into 10 mL of TSB medium (Trypto-Soya broth: Nissui Pharmaceutical Co., Ltd.) and shaking cultured at 30°C for 5 hours, and the mycelia were collected, washed twice with 10 mL of sterile water and suspended in 1 mL of sterile water.  This was the recipient suspension.<br>
500 uL of the S17-l/pKU253-L3-hyg-R3 donor suspension was mixed with 10 uL of the Mer-11107 recipient suspension, and plated to Actino Medium No. 4 agar medium (Nihon Pharmaceutical Co., Ltd.).  After incubated at 30°C for 18 hours, this was covered with 2.5 mL of SNA (0.8% nutrient medium: Difco, 0.4% agar) comprising 2 mg/mL ribostamycin, and incubated at 30°C for 7 days to obtain a ribostamycin-resistant pKU253-L3-hyg-R3 transformant strain.<br>
The resulting pKU253-L3-hyg-R3 transformant strain was inoculated intp 10 mL of TSB medium containing no ribostamycin, and shaking cultured at 30°C for 24 hours. Mycelia were collected from the pKU253-L3-hyg-R3 transformant culture broth, washed twice with 10 mL of sterilized water and suspended in 10 mL of sterilized water.<br>
After being diluted appropriately, the suspension was plated to YMS agar medium (0.4% yeast extract, 1% wheat germ extract, 0.4% soluble starch, 2% agar, 10 mM calcium chloride) comprising 200 ug/mL hygromycin B, and incubated at 30°C for 4 days.  Single colonies growing on the YMS agar medium comprising hygromycin B were transplanted to YMS agar medium comprising 200 ug/mL of hygromycin B and YMS agar medium comprising 200 ug/mL of ribostamycin, and incubated at 30°C for 2 days.<br>
After incubated, a hygromycin B-resistant,<br>
ribostamycin-sensitive strain was selected.  The resulting strain, called Mer-11107 pldD::hyg, was a pldD-deficient strain lacking 1146 bp (nucleotides 68369 to 69513 in Sequence No.1) of the pldD gene from the genome, with the hygromycin B resistance gene inserted in its place.<br>
Example 17: Pladienolide production test of pladienolide 18,19-epoxidase gene (pldD)-deficient strain<br>
200 uL frozen seed of the Mer-11107 pldDC::hyg strain obtained in Example 16 was inoculated into 20 mL of seed medium (soluble starch 2%, ESUSAN-MEAT 2%, yeast extract 0.5%, K2HP04 0.1%, MgS04-7H20 0.25%, CaC03 0.3%, pH not adjusted) and incubated at 25°C for 2 days.<br>
300 uL of the resulting seed culture broth was inoculated into 30 mL of seed culture medium (5% Stabilose, 1% glucose, 3% Pharmamedia, 2% (3-cyclodextrin, 0.1% CaC03, pH 7.5) and cultured at 25°C for 4 and 5 days.<br>
After the completion of the cultivation, 20 mL of the resulting culture liquid was extracted by adding an equal amount of acetonitrile thereto.  Part of this extract was taken and diluted with 5 times the amount of acetonitrile, and levels of pladienolide B and pladienolide Z were measured by HPLC under the following conditions.  The measurement results are shown in Table 4.<br>
(HPLC analysis conditions) Analyzer: Shimadzu HPLC lOAvp<br>
Column: Develosil ODS UG-3 (4.6 mm x 50 mm 3 urn)<br>
Mobile phase (vol%): 45% to 55% methanol (0 to 5 minutes)<br>
55% methanol (5 to 13 minutes) 55% to 70% methanol (13 to 17 minutes) 70% methanol (17 to 35 minutes) 45% methanol (35 to 40 minutes)<br>
Flow rate: 1.2 mL/min<br>
Detection: UV 240 nm<br>
Injection volume: 10 uL<br>
Column temperature: 40°C<br>
Analysis time: 35 minutes<br>
Retention time: pladienolide Z: 20 minutes, pladienolide B:<br>
12 minutes<br>
(Table Remove)<br>
Example 18: Isolation and purification of pladienolide Z and its structure confirmation<br>
The acetonitrile extraction solution obtained in Example 17 was filtered and washed with 10 mL water and 10 mL ethyl acetate.  To the filtrate were added 40 mL brine and 90 mL ethyl acetate, and the mixture was extracted and the extract was washed with 50 mL brine.  The organic layer was dried over sodium sulfate, filtered, and evaporated. The resulting residue was purified by thin-layer chromatography (TLC, Merck Art. 5744, developing solvent: hexane:ethyl acetate=10:50), to give pladienolide Z (Rf=0.59, 22.8 mg).<br>
1.	Molecular weight: 462, ESI-MS m/z 485(M+Na)+, 461(M-H)"<br>
2.	1H-NMR spectrum (CD3OD, 500 MHz): 8 ppm (integration,<br>
multiplicity, coupling constant J (Hz)):<br>
0.89(3H,d,J=6.8Hz), 0.92(3H,t,J=7.5Hz), 0.98(3H, d, J=6.8Hz) ,<br>
1.01(3H,d, J=6.8Hz) , 1.07(3H,d,J=6.8Hz), 1.17-1.37(3H,m),<br>
1.49-1.67(4H,m), 1.73(3H,d,J=l.OHz),<br>
2.04(2H,dd,J=6.8, 6.8Hz) , 2.07-2.15(1H,m), 2.23-2.31(1H,m),<br>
2.42(lH,dd,J=5.3,14.1Hz), 2.50-2.59(1H,m),<br>
2.55(lH,dd,J-3.4,14.1Hz), 3.16-3.22(lH,m),<br>
3.57(lH,dd,J=9.6,9.6Hz), 3.72-3.79(1H,m), 5.00(lH,d,J=10.7Hz), 5.17-5.43(3H,m), 5.46(lH,dd,J=9.5/IS.OHz), 5.64(1H,dd,J=7.8,IS.lHz), 6.05(lH,d,J=10.8Hz), 6.21(1H, dd,J=l0.8,15.IHz)<br>
(Figure Remove)<br>
Pladienolide Z<br>
These results confirmed that the Mer-11107 pldDC::hyg strain, which is a pldD-deficient strain, did not produce pladienolide B and did produce pladienolide Z.  In other words, pladienolide Z could be produced and obtained by the method described above.<br><br>
Claims<br>
1.	A DNA that is isolated and pure, and that contains<br>
at least one region encoding a polypeptide that<br>
participates in pladienolide biosynthesis.<br>
2.	The DNA according to claim 1, characterized by<br>
containing the complete region encoding the polypeptide<br>
that participates in pladienolide biosynthesis.<br>
3.	The DNA according to claim 1 or claim 2,<br>
characterized in that the polypeptide participating in<br>
pladienolide biosynthesis is at least one type selected<br>
from polyketide synthases, 6-hydroxylases, 7-acylation<br>
enzymes, 18,19-epoxidases and transcription regulator<br>
factors.<br><br>
4.	The DNA according to any of claims 1 to 3,<br>
characterized by originating in a microorganism belonging<br>
to the genus Streptomyces.<br>
5.	The DNA according to claim 1, comprising at least<br>
one nucleotide sequence selected from the nucleotide<br>
sequences defined in any of the following (1) to (5):<br>
(1) nucleotide sequences defined in any of the following (a) to (i) :<br>
(a)	the continuous nucleotide sequence from the base 8340<br>
to base 27935 of the Sequence No. 1<br>
(b)	the continuous nucleotide sequence from the base 28021<br>
to base 49098 of the Sequence No. 1<br><br>
(c)	the continuous nucleotide sequence from the base 49134<br>
to base 60269 of the Sequence No. 1<br>
(d)	the continuous nucleotide sequence from the base 60269<br>
to base 65692 of the Sequence No. 1<br>
(e)	the continuous nucleotide sequence from the base 65707<br>
to base 66903 of the Sequence No. 1<br>
(f)	the continuous nucleotide sequence from the base 68160<br>
to base 66970 of the Sequence No. 1<br>
(g)	the continuous nucleotide sequence from the base 69568<br>
to base 68270 of the Sequence No. 1<br>
(h) the continuous nucleotide sequence from the base 72725 to base 70020 of the Sequence No. 1<br>
(1)	the continuous nucleotide sequence from the base 1 to<br>
base 74342 of the Sequence No. 1<br>
(2)	a nucleotide sequence of a DNA that hybridizes under<br>
stringent conditions with a DNA comprising any of the<br>
nucleotide sequences defined in (1)<br>
(3)	a nucleotide sequence having at least 70% homology with<br>
any of the nucleotide sequences defined in (1)<br><br>
(4)	a nucleotide sequence complementary to any of the<br>
nucleotide sequences defined in any of (1) to (3)<br>
(5)	a nucleotide sequence that, due to the degeneracy of<br>
the genetic code, does not hybridize under stringent<br>
conditions with a DNA comprising a nucleotide sequence<br>
defined in (1), but which codes for the same amino acid<br>
sequence as a nucleotide sequence defined in any of (1) to<br><br>
(3) .<br>
6.	The DNA according to claim 1, comprising at least<br>
one nucleotide sequence selected from the nucleotide<br>
sequences defined in any of the following (a) to (i):<br>
(a)	the continuous nucleotide sequence from the base 8340<br>
to base 27935 of the Sequence No. 1<br>
(b)	the continuous nucleotide sequence from the base 28021<br>
to base 49098 of the Sequence No. I<br>
(c)	the continuous nucleotide sequence from the base 49134<br>
to base 60269 of the Sequence No. 1<br>
(d)	the continuous nucleotide sequence from the base 60269<br>
to base 65692 of the Sequence No. 1<br>
(e)	the continuous nucleotide sequence from the base 65707<br>
to base 66903 of the Sequence No. 1<br>
(f)	the continuous nucleotide sequence from the base 68160<br>
to base 66970 of the Sequence No. 1<br>
(g)	the continuous nucleotide sequence from the base 69568<br>
to base 68270 of the Sequence No. 1<br>
(h) the continuous nucleotide sequence from the base 72725 to base 70020 of the Sequence No. 1<br>
(i) the continuous nucleotide sequence from the base 1 to base 74342 of the Sequence No. 1.<br>
7.	A polypeptide encoded by the DNA according to any<br>
of claims 1 to 6.<br>
8.	The polypeptide according to claim 7, characterized<br>
by having a polyketide synthase activity.<br><br>
9.	The polypeptide according to claim 8, characterized<br>
by having the amino acid sequence described by Sequence No.<br>
2, 3, 4 or 5, or having a partial sequence thereof.<br>
10.	The polypeptide according to claim 7,<br>
characterized by having a 6-hydroxylase activity.<br>
11.	The polypeptide according to claim 10,<br>
characterized by having the amino acid sequence described<br>
by Sequence No. 6 or having a partial sequence thereof.<br>
12.	The polypeptide according to claim 7,<br>
characterized by having an 18,19-epoxidase activity.<br>
13.	The polypeptide according to claim 12,<br>
characterized by having the amino acid sequence described<br>
by Sequence No. 8 or having a partial sequence thereof.<br>
14.	The polypeptide according to claim 7,<br>
characterized by having a transcription regulator factor<br>
activity.<br>
15.	The polypeptide according to claim 14,<br>
characterized by having the amino acid sequence described<br>
by Sequence No. 9 or having a partial sequence thereof.<br>
16.	The polypeptide according to claim 7,<br>
characterized by having a 7-acylation enzyme activity.<br>
17.	The polypeptide according to claim 16,<br>
characterized by having the amino acid sequence described<br>
by Sequence No. 7 or having a partial sequence thereof.<br>
18.	A self-replicating or integrated-replicating<br>
recombinant plasmid carrying the DNA according to any of<br><br>
claims 1 to 6.<br>
19.	A transformant maintaining the DNA according to<br>
any of claims 1 to 6.<br>
20.	A method of producing a pladienolide,<br>
characterized by culturing the transformant according to<br>
claim 19 on culture medium; and collecting pladienolide<br>
from the culture broth.<br>
21.	The method of production according to claim 20,<br>
wherein the pladienolide is pladienolide B.<br>
22.	A method of producing a pladienolide D derivative<br>
represented by the formula (VI):<br>
(Figure Remove)<br>
(wherein RN represents a lower alkyl group or a cyclic lower alkyl group; and n represents 1 or 2), comprising the steps of:<br>
(1) introducing a hydroxyl group at position 16 of the compound of the formula (I):<br>
(Figure Remove)<br>
 (pladienolide B) obtained by the method of production according to claim 20 or claim 21, thereby converting the compound of the formula (I) into a compound of the formula (II) (pladienolide D):<br><br>
(Figure Remove)<br>
 (2) introducing a suitable protective group onto the hydroxyl groups at position 3, 6, 16 and/or 21 of the compound of the formula (II), thereby converting the compound of the formula (II) into a compound of the formula (III) :<br><br>
(Figure Remove)<br>
 (wherein R3A, R6A, R1SA and R21A represent a hydrogen atom or a protecting group of the hydroxyl group, provided that R3A,<br><br><br><br>
simultaneously);<br>
(3) eliminating the acetyl group at position 7 of the compound of the formula (III), thereby converting the<br>
compound of the formula (III) into a compound of the formula (IV):<br>
(Figure Remove)<br><br><br>
(wherein R3Ar R6A, R16A and R21A are defined as abqve) ;<br>
(4) introducing a substituent at position 7 of the compound of the formula (IV) , thereby converting the compound of the formula (IV) into a compound of the formula (V) :<br><br>
(Figure Remove)<br><br>
 (wherein RN, R3A, R6A, R16A and R21A are defined as above) ; and<br>
(5) eliminating the protective group from the compound of the formula (V).</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUFic3RyYWN0LSgxOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Abstract-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUFic3RyYWN0LSgzMS0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Abstract-(31-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUFzc2lnbm1lbnQtKDE4LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Assignment-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUNsYWltcy0oMTgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Claims-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUNsYWltcy0oMzEtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Claims-(31-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Correspondence Others-(10-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Correspondence Others-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzEtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Correspondence Others-(31-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Correspondence-Others-(27-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7814-DELNP-2006-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUZvcm0tMTMtKDIzLTA1LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Form-13-(23-05-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUZvcm0tMy0oMTgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Form-3-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUZvcm0tMy0oMzEtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Form-3-(31-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1ERUxOUC0yMDA2LUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7814-DELNP-2006-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LUdQQS0oMTgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-GPA-(18-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMTgtMTItMjAxMiktMS5wZGY=" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Petition-137-(18-12-2012)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzgxNC1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMTgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7814-delnp-2006-Petition-137-(18-12-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263197-a-tyre-for-vehicles-carrying-heavy-loads.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263199-process-for-the-preparation-of-chiral-propionic-acid-derivatives.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263198</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7814/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Dec-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EISAI R&amp;D MANAGEMENT CO., LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>6-10, KOISHIKAWA 4-CHOME, BUNKYO-KU, TOKYO, JAPAN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KAZUHIRO MACHIDA</td>
											<td>1797-336, NAKAIZUMI, IWATA-SHI,SHIZUOKA, JAPAN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>AKIRA ARISAWA</td>
											<td>1797-313, NAKAIZUMI, IWATA-SHI, SHIZUOKA, JAPAN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SUSUMU TAKEDA</td>
											<td>2-19(7), SANRAKUCHO, YATSUSHIRO-SHI, KUMAMOTO, JAPAN</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MASASHI YOSHIDA</td>
											<td>1797-326, NAKAIZUMI, IWATA-SHI, SHIZUOKA, JAPAN</td>
										</tr>
										<tr>
											<td>5</td>
											<td>TOSHIO TSUCHIDA</td>
											<td>118-1-102, KONODAI, IWATA-SHI, SHIZUOKA, JAPAN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/52</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2005/013541</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-07-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2004-211279</td>
									<td>2004-07-20</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263198-dna-coding-for-polypeptide-participating-in-biosynthesis-of-pladienolide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:57:10 GMT -->
</html>
