<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263223-an-isolated-influenza-virus-capable-of-infecting-canid-animal by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:51:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263223:&quot;AN ISOLATED INFLUENZA VIRUS CAPABLE OF INFECTING CANID ANIMAL&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED INFLUENZA VIRUS CAPABLE OF INFECTING CANID ANIMAL&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The subject invention pertains to isolated influenza virus that is capable of infecting canids and causing respipatory disease in the canid. The subject invention also pertains to compositions and methods for inducing an immune response against an influenza virus of the present invention. The subject invention also pertains to compositions and methods for identifying a virus of the invention and diagnosing infection of an animal with a virus of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION<br>
MATERIALS AND METHODS FOR RESPIRATORY DISEASE CONTROL IN CANINES<br>
CROSS-REFERENCE TO RELATED APPLICATION<br>
This application claims the benefit of U.S. Provisional Application Serial No. 60/673,443, filed April 21, 2005, which is hereby incorporated by reference herein in its entirety, including any figures, tables, nucleic acid sequences, amino acid sequences, and drawings.<br>
BACKGROUND OF THE INVENTION<br>
"Kennel cough" or infectious tracheobronchitis (ITB) is an acute, contagious respiratory infection in dogs characterized mainly by coughing (Ford et al, 1998). Canine ITB is considered one of the most prevalent infectious respiratory diseases of dogs worldwide, and outbreaks can reach epidemic proportions when dogs are housed in high-density population environments such as kennels. Most outbreaks are due to direct dog-to-dog contact or aerosolization of respiratory secretions (Ford et al, 1998). The clinical signs are caused by infection with one or a combination of bacterial and viral agents that colonize the epithelium of the upper and lower respiratory tract. Canine parainfluenza virus (CPiV) and Bordetella bronchiseptica bacteria are the most common organisms isolated from affected dogs, but several other viruses such as canine distemper virus (CDV) and canine adenoviruses-1 and -2 (CAV-1, CAV-2), along with bacteria such as Streptococcus sp., Pasteurella multicoda and Escherichia coli, can influence the clinical course and outcome (Ford et al, 1998). While outbreaks occur most efficiently and rapidly in high-density populations with high morbidity, complicated respiratory infections and death are uncommon. Although life-threatening secondary bacterial pneumonia can develop, the majority of ITB cases are self-limiting and resolve without any treatment (Ford et al, 1998).<br>
In July 1992, a respiratory infection presumed to be "kennel cough" became epidemic at several greyhound tracks in New England, Florida, West Virginia, Wisconsin, Kansas, Colorado, Oklahoma and Arizona. According to veterinarians, most of the affected dogs had a mild cough that resolved, but more than a dozen greyhounds developed an acute hemorrhagic pneumonia followed by rapid death (Greyhound Daily News, 1999).<br>
In late 1998 to early 1999, several outbreaks of "kennel cough" occurred in racing greyhound kennels across the country, resulting in mandatory closure of tracks and quarantine of all racing greyhounds in the U.S. for several weeks (Greyhound Daily News, 1999). At one track in Florida (Palm Beach Kennel Club), coughing was recorded in nearly 40% of the dog population on a single day (Personal communication from Dr. William Duggar). Similar to the outbreak in 1992, the coughing resolved in most greyhounds, but 10 dogs in Florida died from a hemorrhagic pneumonia syndrome uncharacteristic of "kennel cough" (Putnam, 1999).<br>
In March-April 2003, another outbreak of "kennel cough" occurred at greyhound tracks in the eastern U.S. The outbreak is believed to have originated in kennels at four tracks in Florida and caused the suspension of racing and quarantine of dogs for almost three weeks. Nearly 25% of the dogs at the track in West Palm Beach were<br><br>
affected, while almost 50% of the 1400 dogs at Derby Lane in St. Petersburg developed coughing. Again, most dogs^*overed, but several dogs have died from the respiratory infection. The estimated economic impact of the respiratory outbreak at the Derby Lane track alone was $2 million.<br>
There are no published reports documenting the etiology or clinicopathology of the "kennel cough" epidemics in racing greyhound kennels in 1992, 1998-1999, or 2003. The assumption has been that the infections were due to CPiV and/or B. bronchiseptica, the two most common causes of kennel cough. Unsubstantiated communications such as web sites have attributed the fatal hemorrhagic pneumonias reported in some of the coughing dogs to infection with p-hemolytic Streptococcus equi subspecies zooepidemicus, and refer to the syndrome as "canine streptococcal toxic shock."<br>
Transmission of virus from one host species to another is a crucial feature of the ecology and epidemiology of influenza virus (Webster, 1998). Two basic mechanisms of interspecies transmission of influenza virus are possible (Webster et al., 1992; Lipatov et al, 2004). One is the direct transfer of an essentially unaltered virus from one species to another. Examples of this mechanism include the recent human infections with the H5N1 subtype of avian influenza virus (Subbarao et al, 1998; Peiris et al, 2004; Guan et al, 2004) and possibly the pandemic of 1918, known as Spanish flu (Reid et al, 2004). The second mechanism is a consequence of the segmented nature of the influenza genome. Co-infection of a host with viruses from different species can result in reassortment of the segmented viral genes and the generation of a recombinant with the ability to infect other species. For example, novel viruses generated by gene reassortment between avian and human influenza viruses resulted in human influenza pandemics in 1957 and 1968 (Webster et al, 1992; Lipatov et al, 2004; Kawaoka et al, 1989).<br>
Most direct transmissions of unaltered influenza viruses from the natural host species to a different species are terminal events because sustained transmission between individuals of the new species fails to occur. Multiple virus-host interactions are necessary for replication and horizontal transmission and provide a formidable barrier to perpetuation of influenza viruses in the new host (Webby et al, 2004). Therefore, establishment of new host-specific lineages of influenza virus is uncommon and has only occurred in domestic poultry, pigs, horses, and humans (Webster et al, 1992; Lipatov etal, 2004).<br>
Because of the serious nature of influenza virus infection, there remains a need for methods for diagnosing, preventing, and treating infection by influenza virus.<br>
BRIEF SUMMARY OF THE INVENTION<br>
The subject invention pertains to isolated influenza virus that is capable of infecting canids and causing respiratory disease in the canid. The subject invention also pertains to compositions and methods for inducing an immune response against an influenza virus of the present invention. The subject invention also pertains to compositions and methods for identifying a virus of the invention and diagnosing infection of an animal with a virus of the invention.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br><br>
The patent or application file contains at least one drawing executed in color. Copies of this patent or pateq|-«pplication publication with color drawings will be provided by the Patent Office upon request and payment of the necessary fee.<br>
Figures 1A-1B show phylogenetic relationships among the hemagglutinin genes. Figure 1A shows a tree of HA genes from representative canine, human, avian, swine, and equine isolates, including A/Budgerigar/Hokkaido/1/77 (H4) as outgroup. Figure IB shows a tree of the canine influenza virus HA genes with contemporary and older equine HA genes, using A/Duck/Ukraine/63 (H3) as outgroup. Phylogenetic trees were inferred from nucleotide sequences by the neighbor joining method and bootstrap analysis values &gt;90% are shown. The bar denotes the number of nucleotide changes per unit length of the horizontal tree branches.<br>
Figures 2A-2B show immunohistochemical detection of influenza H3 antigen in the lungs. Lung tissue sections were probed with a mouse monoclonal antibody to H3 hemagglutinin and binding was detected by immunoperoxidase reaction (brown precipitate). Figure 2A shows bronchial epithelium from a greyhound with spontaneous disease. Viral H3 antigen was detected in bronchial epithelial cell cytoplasm and in macrophages in airway lumens and in alveolar spaces. Figure 2B shows bronchial epithelium from a dog 5 days after inoculation with A/canine/Florida/43/04 (H3N8). Viral H3 antigen was detected in bronchial epithelial cell cytoplasm. Scale bar, 66 urn.<br>
Figure 3 shows the characteristic histological changes in the bronchi of greyhounds that died from hemorrhagic pneumonia associated with influenza virus infection. The tissues are stained with H&amp;E. Upper panel: Normal bronchus with ciliated epithelial cells, mucous cells, and basal cells. Lower panel: Bronchus from a greyhound with spontaneous influenza. There is necrosis and erosion of the bronchial ciliated epithelial cells. Scale bar, 100 urn.<br>
Figures 4A-4B shows phylogenetic relationships among the H3 hemagglutinin genes. Figure 4A shows a phylogenetic tree of the canine influenza virus HA genes with contemporary and older equine HA genes. Figure 4B shows a phylogenetic tree of the canine influenza virus HA protein with contemporary and older equine HA. Phylogenetic trees were inferred from genetic or amino acid sequences by the neighbor joining method and bootstrap analysis values &gt;80% are shown. The bar denotes the number of amino acid changes per unit length of the horizontal tree branches.<br>
Figure 5 shows Influenza virus H3 protein in epithelial cells of bronchi and bronchial glands in lungs of dogs that died of pneumonia associated with influenza virus infection. Upper panels: Erosion of ciliated bronchial epithelial cells in bronchi. Tissues were stained with H&amp;E. Lower panels: Influenza virus H3 protein in the cytoplasm of bronchial (left) and bronchial gland (right) epithelial cells. Tissues were stained with a monoclonal antibody to influenza H3 detected by immunoperoxidase reaction (brown precipitate) and counterstained with hematoxylin.<br>
Figures 6A-6D show amplification plots of H3 and Matrix genes (Figure 6A and Figure 6B) obtained from the amplification of 10-fold serially diluted in vitro transcribed RNA standards. Standard curves of H3 and Matrix genes (Figure 6C and Figure 6D) constructed by plotting the log of starting RNA concentrations against the threshold cycle (Ct) obtained from each dilution.<br>
Figure 7 shows sensitivity of Directigen Flu A was tested with 10-fold serially diluted virus stocks inching A/Wyoming/3/2003 and A/canine/FL/242/2003. The purple triangle indicates positive result.<br>
BRIEF DESCRIPTION OF THE SEQUENCES<br>
SEQ ID NO: 1 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding a PB2 protein that can be used according to the present invention.<br>
SEQ ID NO: 2 is the amino acid sequence encoded by SEQ ID NO: 1.<br>
SEQ ID NO: 3 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding a PB1 protein that can be used according to the present invention.<br>
SEQ ID NO: 4 is the amino acid sequence encoded by SEQ ID NO: 3.<br>
SEQ ID NO: 5 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding a PA protein that can be used according to the present invention.<br>
SEQ ID NO: 6 is the amino acid sequence encoded by SEQ ID NO: 5.<br>
SEQ ID NO: 7 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding an NS protein that can be used according to the present invention.<br>
SEQ ID NO: 8 is the amino acid sequence encoded by SEQ ID NO: 7.<br>
SEQ ID NO: 9 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding an NP protein that can be used according to the present invention.<br>
SEQ ID NO: 10 is the amino acid sequence encoded by SEQ ID NO: 9.<br>
SEQ ID NO: 11 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding an NA protein that can be used according to the present invention.<br>
SEQ ID NO: 12 is the amino acid sequence encoded by SEQ ID NO: 11.<br>
SEQ ID NO: 13 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding an MA protein that can be used according to the present invention.<br>
SEQ ID NO: 14 is the amino acid sequence encoded by SEQ ID NO: 13.<br>
SEQ ID NO: 15 is a nucleotide sequence of a canine influenza virus (Florida/43/04) encoding an HA protein that can be used according to the present invention.<br>
SEQ ID NO: 16 is the amino acid sequence encoded by SEQ ID NO: 15.<br>
SEQ ID NO: 17 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding a PB2 protein that can be used according to the present invention.<br>
SEQ ID NO: 18 is the amino acid sequence encoded by SEQ ID NO: 17.<br>
SEQ ID NO: 19 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding a PB1 protein that can be used according to the present invention.<br>
SEQ ID NO: 20 is the amino acid sequence encoded by SEQ ID NO: 19.<br>
SEQ ID NO: 21 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding a PA protein that can be used according to the present invention.<br>
SEQ ID NO: 22 is the amino acid sequence encoded by SEQ ID NO: 21.<br>
SEQ ID NO: 23 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding an NS protein that «^ be used according to the present invention.<br>
SEQ ID NO: 24 is the amino acid sequence encoded by SEQ ID NO: 23.<br>
SEQ ID NO: 25 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding an NP protein that can be used according to the present invention.<br>
SEQ ID NO: 26 is the amino acid sequence encoded by SEQ ID NO: 25.<br>
SEQ ID NO: 27 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding an NA protein that can be used according to the present invention.<br>
SEQ ID NO: 28 is the amino acid sequence encoded by SEQ ID NO: 27.<br>
SEQ ID NO: 29 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding an MA protein that can be used according to the present invention.<br>
SEQ ID NO: 30 is the amino acid sequence encoded by SEQ ID NO: 29.<br>
SEQ ID NO: 31 is a nucleotide sequence of a canine influenza virus (FL/242/03) encoding an HA protein that can be used according to the present invention.<br>
SEQ ID NO: 32 is the amino acid sequence encoded by SEQ ID NO: 31.<br>
SEQ ID NO: 33 is the mature form of the HA protein shown in SEQ ID NO: 16 wherein the N-terminal 16 amino acid signal sequence has been removed.<br>
SEQ ID NO: 34 is the mature form of the HA protein shown in SEQ ID NO: 32 wherein the N-terminal 16 amino acid signal sequence has been removed.<br>
SEQ ID NO: 35 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 36 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 37 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 38 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 39 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 41 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 42 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 43 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 44 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 45 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 46 is an oligonucleotide that can be used according to the present invention.<br>
SEQ ID NO: 47 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding a PB2 protein that can be used according to the present invention.<br>
SEQ ID NO: 48 is the amino acid sequence encoded by SEQ ID NO: 47.<br>
SEQ ID NO: 49 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding a FBI protein that can be used according to the present invention.<br>
SEQ ID NO: 50 is the amino acid sequence encoded by SEQ ID NO: 49.<br>
SEQ ID NO: 51 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding a PA protein that G^fTbe used according to the present invention.<br>
SEQ ID NO: 52 is the amino acid sequence encoded by SEQ ID NO: 51.<br>
SEQ ID NO: 53 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding an NS protein that can be used according to the present invention.<br>
SEQ ID NO: 54 is the amino acid sequence encoded by SEQ ID NO: 53.<br>
SEQ ID NO: 55 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding an NP protein that can be used according to the present invention.<br>
SEQ ID NO: 56 is the amino acid sequence encoded by SEQ ID NO: 55.<br>
SEQ ID NO: 57 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding an NA protein that can be used according to the present invention.<br>
SEQ ID NO: 58 is the amino acid sequence encoded by SEQ ID NO: 57.<br>
SEQ ID NO: 59 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding an MA protein that can be used according to the present invention.<br>
SEQ ID NO: 60 is the amino acid sequence encoded by SEQ ID NO: 59.<br>
SEQ ID NO: 61 is a nucleotide sequence of a canine influenza virus (Miami/2005) encoding an HA protein that can be used according to the present invention.<br>
SEQ ID NO: 62 is the amino acid sequence encoded by SEQ ID NO: 61.<br>
SEQ ID NO: 63 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding a PB2 protein that can be used according to the present invention.<br>
SEQ ID NO: 64 is the amino acid sequence encoded by SEQ ID NO: 63.<br>
SEQ ID NO: 65 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding a PB1 protein that can be used according to the present invention.<br>
SEQ ID NO: 66 is the amino acid sequence encoded by SEQ ID NO: 65.<br>
SEQ ID NO: 67 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding a PA protein that can be used according to the present invention.<br>
SEQ ID NO: 68 is the amino acid sequence encoded by SEQ ID NO: 67.<br>
SEQ ID NO: 69 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding an NS protein that can be used according to the present invention.<br>
SEQ ID NO: 70 is the amino acid sequence encoded by SEQ ID NO: 69.<br>
SEQ ID NO: 71 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding an NP protein that can be used according to the present invention.<br>
SEQ ID NO: 72 is the amino acid sequence encoded by SEQ ID NO: 71.<br>
SEQ ID NO: 73 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding an NA protein that can be used according to the present invention.<br>
SEQ ID NO: 74 is the amino acid sequence encoded by SEQ ID NO: 73.<br>
SEQ ID NO: 75 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding an MA<br>
can be used according to the present invention. SEQ ID NO: 76 is the amino acid sequence encoded by SEQ ID NO: 75.<br>
SEQ ID NO: 77 is a nucleotide sequence of a canine influenza virus (Jacksonville/2005) encoding an HA protein that can be used according to the present invention.<br>
SEQ ID NO: 78 is the amino acid sequence encoded by SEQ ID NO: 77. SEQ ID NO: 79 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 80 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 81 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 82 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 83 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 84 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 85 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 86 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 87 is an oligonucleotide that can be used according to the present invention. SEQ ID NO: 88 is an oligonucleotide that can be used according to the present invention.<br>
DETAILED DISCLOSURE OF THE INVENTION<br>
The subject invention concerns isolated influenza virus that is capable of infecting canids and causing respiratory disease. In one embodiment, an influenza virus of the invention comprises a polynucleotide which encodes a protein having an amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14 , 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic fragment or variant thereof. In a specific embodiment, the polynucleotide comprises the nucleotide sequence shown in any of SEQ ID Nos: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19,21,23,25,27,29,31,47,49,51,53,55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, or a fragment or variant thereof. In a specific embodiment, an influenza virus of the present invention is a subtype H3. Virus can be isolated from infected dogs and cultured in cells or eggs according to methods described herein. In an exemplified embodiment, the influenza virus is an influenza A virus.<br>
The subject invention also concerns polynucleotides that comprise all or part of a gene or genes or a genomic segment of an influenza virus of the present invention. In one embodiment, a polynucleotide of the invention comprises an influenza hemagglutinin (HA) gene, neuraminidase (NA) gene, nucleoprotein (NP) gene, matrix protein (MA or M) gene, polymerase basic (PB) protein gene, polymerase acidic (PA) protein gene, non-structural (NS) protein gene, or a functional fragment or variant of any of these genes. In a specific embodiment, a polynucleotide of the invention comprises the hemagglutinin (HA) gene, or a functional fragment or variant thereof. In a further embodiment, the HA gene encodes a hemagglutinin protein having one or more of the following: a serine at position 83; a leucine at position 222; a threonine at position 328; and/or a threonine at position 483, versus the amino acid sequence of equine H3 consensus sequence. In one embodiment, the HA gene<br>
encodes a polypeptide having an amino acid sequence shown in SEQ ID NOs: 16, 32, 62, or 78, or a functional and/q^f immunogenic fragment or variant thereof. In a specific embodiment, the HA gene comprises a nucleotide sequence shown in SEQ ID NOs: 15, 31, 61, or 77.<br>
In one embodiment, a polynucleotide of the invention encodes a polypeptide having the amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic fragment or variant thereof. In a specific embodiment, the polynucleotide encoding the amino acid sequence shown in SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, comprises the nucleotide sequence shown in SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, respectively, or a sequence encoding a functional and/or immunogenic fragment or variant of any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28,<br>
30,	32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78. Thus, the subject invention concerns<br>
polynucleotide sequences comprising the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13,<br>
15, 17, 19, 21, 23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, or a fragment or<br>
variant, including a degenerate variant, of any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29,<br>
31,	47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77.    In a further specific embodiment, a<br>
polynucleotide of the invention can comprise: Nucleotides 1-2271 of SEQ ID NO: 3; Nucleotides 1-2148 of SEQ<br>
ID NO: 5; Nucleotides 1-657 of SEQ ID NO: 7; Nucleotides 1-1494 of SEQ ID NO: 9; Nucleotides 1-1410 of SEQ<br>
ID NO: 11; Nucleotides 1-756 of SEQ ID NO: 13; Nucleotides 1-1695 of SEQ ID NO: 15; Nucleotides 1-2271 of<br>
SEQ ID NO: 19; Nucleotides 1-2148 of SEQ ID NO: 21; Nucleotides 1-657 of SEQ ID NO: 23; Nucleotides 1-<br>
1494 of SEQ ID NO: 25; Nucleotides 1-756 of SEQ ID NO: 29; Nucleotides 1-1695 of SEQ ID NO: 31;<br>
Nucleotides 1-2277 of SEQ ID NO: 47; Nucleotides 1-2271 of SEQ ID NO: 49; Nucleotides 1-2148 of SEQ ID<br>
NO: 51; Nucleotides 1-690 of SEQ ID NO: 53; Nucleotides 1-1494 of SEQ ID NO: 55; Nucleotides 1-1410 of<br>
SEQ ID NO: 57; Nucleotides 1-756 of SEQ ID NO: 59; Nucleotides 1-1695 of SEQ ID NO: 61; Nucleotides 1-<br>
2277 of SEQ ID NO: 63; Nucleotides 1-2271 of SEQ ID NO: 65; Nucleotides 1-2148 of SEQ ID NO: 67;<br>
Nucleotides 1-690 of SEQ ID NO: 69; Nucleotides 1-1494 of SEQ ID NO: 71; Nucleotides 1-1410 of SEQ ID NO:<br>
73; Nucleotides 1-756 of SEQ ID NO: 75; and Nucleotides 1-1695 of SEQ ID NO: 77. Nucleotide and amino acid<br>
sequences of viral polynucleotide and polypeptide sequences contemplated within the scope of the present<br>
invention have also been deposited with GenBank at accession Nos.  DQ124147 through DQ124161  and<br>
DQ124190, the disclosure of which is incorporated herein by reference.<br>
The subject invention also concerns polypeptides encoded by polynucleotides of an influenza virus of the present invention. The subject invention also concerns functional and/or immunogenic fragments and variants of the subject polypeptides. Polypeptides contemplated include HA protein, NA protein, NS protein, nucleoprotein, polymerase basic protein, polymerase acidic protein, and matrix protein of an influenza virus of the invention. In an exemplified embodiment, a polypeptide of the invention has an amino acid sequence shown in any of SEQ ID<br>
NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74^ 76. or 78. or a functional and/or immunogenic fragment or variant thereof.<br>
The subject invention also concerns polynucleotide expression constructs comprising a polynucleotide sequence of the present invention. In one embodiment, an expression construct of the invention comprises a polynucleotide sequence encoding a polypeptide comprising an amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic fragment or variant thereof. In a specific embodiment, the polynucleotide encoding the amino acid sequence shown in SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78 comprises the nucleotide sequence shown in SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, respectively, or a sequence encoding a functional and/or immunogenic fragment or variant of any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78. Thus, the subject invention concerns expression constructs comprising a polynucleotide sequence comprising the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7,9, 11, 13, 15, 17, 19, 21,23, 25, 27, 29, 31,47, 49, 51,53, 55, 57, 59, 61,63, 65, 67, 69, 71,73, 75, or 77, or a fragment or variant, including a degenerate variant, of any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77. In a preferred embodiment, an expression construct of the present invention provides for overexpression of an operably linked polynucleotide of the invention.<br>
Expression constructs of the invention generally include regulatory elements that are functional in the intended host cell in which the expression construct is to be expressed. Thus, a person of ordinary skill in the art can select regulatory elements for use in, for example, human host cells, mammalian host cells, insect host cells, yeast host cells, bacterial host cells, and plant host cells. In one embodiment, the regulatory elements are ones that are functional in canine cells. Regulatory elements include promoters, transcription termination sequences, translation termination sequences, enhancers, and polyadenylation elements. As used herein, the term "expression construct" refers to a combination of nucleic acid sequences that provides for transcription of an operably linked nucleic acid sequence. As used herein, the term "operably linked" refers to a juxtaposition of the components described wherein the components are in a relationship that permits them to function in their intended manner. In general, operably linked components are in contiguous relation.<br>
An expression construct of the invention can comprise a promoter sequence operably linked to a polynucleotide sequence encoding a polypeptide of the invention. Promoters can be incorporated into a polynucleotide using standard techniques known in the art. Multiple copies of promoters or multiple promoters can be used in an expression construct of the invention. In a preferred embodiment, a promoter can be positioned about the same distance from the transcription start site in the expression construct as it is from the transcription start site in its natural genetic environment. Some variation in this distance is permitted without substantial decrease in promoter activity. A transcription start site is typically included in the expression construct.<br>
Preferably, the promoter associated with an expression construct of the invention provides for overexpression of an operaW^ linked polynucleotide of the invention.<br>
Promoters for use with an expression construct of the invention in eukaryotic cells can be of viral or cellular origin. Viral promoters include, but are not limited to, cytomegalovirus (CMV) gene promoters, SV40 early or late promoters, or Rous sarcoma virus (RSV) gene promoters. Promoters of cellular origin include, but are not limited to, desmin gene promoter and actin gene promoter Promoters suitable for use with an expression construct of the invention in yeast cells include, but are not limited to, 3-phosphoglycerate kinase promoter, glyceraldehyde-3-phosphate dehydrogenase promoter, metallothionein promoter, alcohol dehydrogenase-2 promoter, and hexokinase promoter.<br>
If the expression construct is to be provided in or introduced into a plant cell, then plant viral promoters, such as, for example, a cauliflower mosaic virus (CaMV) 35S (including the enhanced CaMV 35S promoter (see, for example U.S. Patent No. 5,106,739 and An, 1997)) or a CaMV 19S promoter can be used. Other promoters that can be used for expression constructs in plants include, for example, prolifera promoter, Ap3 promoter, heat shock promoters, T-DNA 1'- or 2'-promoter of A. tumefaciens, polygalacturonase promoter, chalcone synthase A (CHS-A) promoter from petunia, tobacco PR-la promoter, ubiquitin promoter, actin promoter, ale A gene promoter, pin2 promoter (Xu et al., 1993), maize Wipl promoter, maize trpA gene promoter (U.S. Patent No. 5,625,136), maize CDPK gene promoter, and RUBISCO SSU promoter (U.S. Patent No. 5,034,322) can also be used. Root-specific promoters, such as any of the promoter sequences described in U.S. Patent No. 6,455,760 or U.S. Patent No. 6,696,623, or in published U.S. patent application Nos. 20040078841; 20040067506; 20040019934; 20030177536; 20030084486; or 20040123349, can be used with an expression construct of the invention. Constitutive promoters (such as the CaMV, ubiquitin, actin, or NOS promoter), developmentally-regulated promoters, and inducible promoters (such as those promoters than can be induced by heat, light, hormones, or chemicals) are also contemplated for use with polynucleotide expression constructs of the invention. Tissue-specific promoters, for example fruit-specific promoters, such as the E8 promoter of tomato (accession number: AF515784; Good et al. (1994)) can also be used. Seed-specific promoters such as the promoter from a P-phaseolin gene (for example, of kidney bean) or a glycinin gene (for example, of soybean), and others, can also be used.<br>
For expression in prokaryotic systems, an expression construct of the invention can comprise promoters such as, for example, alkaline phosphatase promoter, tryptophan (tip) promoter, lambda PL promoter, p-lactamase promoter, lactose promoter, phoA promoter, T3 promoter, T7 promoter, or tac promoter (de Boer et al., 1983).<br>
Expression constructs of the invention may optionally contain a transcription termination sequence, a translation termination sequence, a sequence encoding a signal peptide, and/or enhancer elements. Transcription termination regions can typically be obtained from the 3' untranslated region of a eukaryotic or viral gene sequence. Transcription termination sequences can be positioned downstream of a coding sequence to provide for efficient termination. A signal peptide sequence is a short amino acid sequence typically present at the amino terminus of a protein that is responsible for the relocation of an operably linked mature polypeptide to a wide range of post-translational cellular destinations, ranging from a specific organelle compartment to sites of protein action<br>
and the extracellular environment. Targeting gene products to an intended cellular and/or extracellular destination througM the use of an operably linked signal peptide sequence is contemplated for use with the polypeptides of the invention. Classical enhancers are cis-acting elements that increase gene transcription and can also be included in the expression construct. Classical enhancer elements are known in the art, and include, but are not limited to, the CaMV 35S enhancer element, cytomegalovirus (CMV) early promoter enhancer element, and the SV40 enhancer element. Intron-mediated enhancer elements that enhance gene expression are also known in the art. These elements must be present within the transcribed region and are orientation dependent.<br>
DNA sequences which direct polyadenylation of mRNA transcribed from the expression construct can also be included in the expression construct, and include, but are not limited to, an octopine synthase or nopaline synthase signal.<br>
Expression constructs can also include one or more dominant selectable marker genes, including, for example, genes encoding antibiotic resistance and/or herbicide-resistance for selecting transformed cells. Antibiotic-resistance genes can provide for resistance to one or more of the following antibiotics: hygromycin, kanamycin, bleomycin, G418, streptomycin, paromomycin, neomycin, and spectinomycin. Kanamycin resistance can be provided by neomycin phosphotransferase (NPT II). Herbicide-resistance genes can provide for resistance to phosphinothricin acetyltransferase or glyphosate. Other markers used for cell transformation screening include, but are not limited to, genes encoding p-glucuronidase (GUS), p-galactosidase, luciferase, nopaline synthase, chloramphenicol acetyltransferase (CAT), green fluorescence protein (GFP), or enhanced GFP (Yang et al., 1996).<br>
The subject invention also concerns polynucleotide vectors comprising a polynucleotide sequence of the invention that encodes a polypeptide of the invention. Unique restriction enzyme sites can be included at the 5' and 3' ends of an expression construct or polynucleotide of the invention to allow for insertion into a polynucleotide vector. As used herein, the term "vector" refers to any genetic element, including for example, plasmids, cosmids, chromosomes, phage, virus, and the like, which is capable of replication when associated with proper control elements and which can transfer polynucleotide sequences between cells. Vectors contain a nucleotide sequence that permits the vector to replicate in a selected host cell. A number of vectors are available for expression and/or cloning, and include, but are not limited to, pBR322, pUC series, Ml3 series, pGEM series, and pBLUESCRIPT vectors (Stratagene, La Jolla, CA and Promega, Madison, WI).<br>
The subject invention also concerns oligonucleotide probes and primers, such as polymerase chain reaction (PCR) primers, that can hybridize to a coding or non-coding sequence of a polynucleotide of the present invention. Oligonucleotide probes of the invention can be used in methods for detecting influenza virus nucleic acid sequences. Oligonucleotide primers of the invention can be used in PCR methods and other methods involving nucleic acid amplification. In a preferred embodiment, a probe or primer of the invention can hybridize to a polynucleotide of the invention under stringent conditions. Probes and primers of the invention can optionally comprise a detectable label or reporter molecule, such as fluorescent molecules, enzymes, radioactive moiety, and the like. Probes and primers of the invention can be of any suitable length for the method or assay in which they are being employed. Typically, probes and primers of the invention will be 10 to 500 or more nucleotides in length. Probes and primers that are 10 to 20, 21 to 30, 31 to 40, 41 to 50, 51 to 60, 61 to 70, 71 to 80, 81 to 90, 91<br>
to 100, or 101 or more nucleotides in length are contemplated within the scope of the invention. In one embodiment, probes and primers are any of 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30 nucleotides in length. Probes and primers of the invention can have complete (100%) nucleotide sequence identity with the polynucleotide sequence, or the sequence identity can be less than 100%. For example, sequence identity between a probe or primer and a sequence can be 99%, 98%, 97%, 96%, 95%, 90%, 85%, 80%, 75%, 70% or any other percentage sequence identity so long as the probe or primer can hybridize under stringent conditions to a nucleotide sequence of a polynucleotide of the invention. Exemplified probes and primers of the invention include those having the nucleotide sequence shown in any of SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, and SEQ ID NO: 46, or a functional fragment or variant of any of the SEQ ID NOs: 35-46.<br>
As used herein, the terms "nucleic acid," "polynucleotide," and "oligonucleotide" refer to a deoxyribonucleotide, ribonucleotide, or a mixed deoxyribonucleotide and ribonucleotide polymer in either single-or double-stranded form, and unless otherwise limited, would encompass known analogs of natural nucleotides that can function in a similar manner as naturally-occurring nucleotides. Polynucleotide sequences include the DNA strand sequence that can be transcribed into RNA and the RNA strand that can be translated into protein. The complementary sequence of any nucleic acid, polynucleotide, or oligonucleotide of the present invention is also contemplated within the scope of the invention. Polynucleotide sequences also include both full-length sequences as well as shorter sequences derived from the full-length sequences. The subject invention also encompasses those polynucleotides that are complementary in sequence to the polynucleotides disclosed herein. Polynucleotides and polypeptides of the invention can be provided in purified or isolated form.<br>
Because of the degeneracy of the genetic code, a variety of different polynucleotide sequences can encode a polypeptide of the present invention. A table showing all possible triplet codons (and where U also stands for T) and the amino acid encoded by each codon is described in Lewin (1985). In addition, it is well within the skill of a person trained in the art to create alternative polynucleotide sequences encoding the same, or essentially the same, polypeptides of the subject invention. These degenerate variant and alternative polynucleotide sequences are within the scope of the subject invention. As used herein, references to "essentially the same" sequence refers to sequences which encode amino acid substitutions, deletions, additions, or insertions which do not materially alter the functional and/or immunogenic activity of the polypeptide encoded by the polynucleotides of the present invention.<br>
The subject invention also concerns variants of the polynucleotides of the present invention that encode polypeptides of the invention. Variant sequences include those sequences wherein one or more nucleotides of the sequence have been substituted, deleted, and/or inserted. The nucleotides that can be substituted for natural nucleotides of DNA have a base moiety that can include, but is not limited to, inosine, 5-fluorouracil, 5-bromouracil, hypoxanthine, 1-methylguanine, 5-methylcytosine, and tritylated bases. The sugar moiety of the nucleotide in a sequence can also be modified and includes, but is not limited to, arabinose, xylulose, and hexose. In addition, the adenine, cytosine, guanine, thymine, and uracil bases of the nucleotides can be modified with<br>
acetyl, methyl, and/or thio groups.  Sequences containing nucleotide substitutions, deletions, and/or insertions can be prepared and tested using standard techniques known in the art.<br>
Substitution of amino acids other than those specifically exemplified or naturally present in a polypeptide of the invention are also contemplated within the scope of the present invention. For example, non-natural amino acids can be substituted for the amino acids of a polypeptide, so long as the polypeptide having the substituted amino acids retains substantially the same functional activity as the polypeptide in which amino acids have not been substituted. Examples of non-natural amino acids include, but are not limited to, ornithine, citrulline, hydroxyproline, homoserine, phenylglycine, taurine, iodotyrosine, 2,4-diaminobutyric acid, a-amino isobutyric acid, 4-aminobutyric acid, 2-amino butyric acid, y-amino butyric acid, e-amino hexanoic acid, 6-amino hexanoic acid, 2-amino isobutyric acid, 3-amino propionic acid, norleucine, norvaline, sarcosine, homocitrulline, cysteic acid, i-butylglycine, T-butylalanine, phenylglycine, cyclohexylalanine, p-alanine, fluoro-amino acids, designer amino acids such as |3-methyl amino acids, C-methyl amino acids, N-methyl amino acids, and amino acid analogues in general. Non-natural amino acids also include amino acids having derivatized side groups. Furthermore, any of the amino acids in the protein can be of the D (dextrorotary) form or L (levorotary) form. Allelic variants of a protein sequence of a polypeptide of the present invention are also encompassed within the scope of the invention.<br>
Amino acids can be generally categorized in the following classes: non-polar, uncharged polar, basic, and acidic. Conservative substitutions whereby a polypeptide of the present invention having an amino acid of one class is replaced with another amino acid of the same class fall within the scope of the subject invention so long as the polypeptide having the substitution still retains substantially the same functional activity as the polypeptide that does not have the substitution. Polynucleotides encoding a polypeptide having one or more amino acid substitutions in the sequence are contemplated within the scope of the present invention. Table 11 below provides a listing of examples of amino acids belonging to each class. Single letter amino acid abbreviations are defined in Table 12.<br>
Fragments and variants of polypeptides of influenza virus of the present invention can be generated using standard methods known in the art and tested for the presence of function or immunogenecity using standard techniques known in the art. For example, for testing fragments and/or variants of a neuraminidase polypeptide of the invention, enzymatic activity can be assayed. Thus, an ordinarily skilled artisan can readily prepare and test fragments and variants of a polypeptide of the invention and determine whether the fragment or variant retains activity relative to full-length or a non-variant polypeptide.<br>
Polynucleotides and polypeptides contemplated within the scope of the subject invention can also be defined in terms of more particular identity and/or similarity ranges with those sequences of the invention specifically exemplified herein. The sequence identity will typically be greater than 60%, preferably greater than 75%, more preferably greater than 80%, even more preferably greater than 90%, and can be greater than 95%. The identity and/or similarity of a sequence can be 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, or 99% as compared to a sequence exemplified herein. Unless otherwise specified, as used herein percent<br>
sequence identity and/or similarity of two sequences can be determined usina the aleorithm of Karlin and Altschiil (199tf&gt;, modified as in Karlin and Altschul (1993). Such an algorithm is incorporated into the NBLAST and XBLAST programs of Altschul et al. (1990). BLAST searches can be performed with the NBLAST program, score = 100, wordlength = 12, to obtain sequences with the desired percent sequence identity. To obtain gapped alignments for comparison purposes, Gapped BLAST can be used as described in Altschul et al. (1997). When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (NBLAST and XBLAST) can be used. See NCBI/NIH website.<br>
The subject invention also contemplates those polynucleotide molecules having sequences which are sufficiently homologous with the polynucleotide sequences exemplified herein so as to permit hybridization with that sequence under standard stringent conditions and standard methods (Maniatis et al., 1982). As used herein, "stringent" conditions for hybridization refers to conditions wherein hybridization is typically carried out overnight at 20-25 C below the melting temperature (Tm) of the DNA hybrid in 6x SSPE, 5x Denhardt's solution, 0.1% SDS, 0.1 mg/ml denatured DNA. The melting temperature, Tm, is described by the following formula (Beltz et al., 1983):<br>
Tm=81.5 C+16.6 Log[Na+]+0.41(%G+C)-0.61(% formamide)-600/length of duplex in base pairs.<br>
Washes are typically carried out as follows:<br>
(1)	Twice at room temperature for 15 minutes in Ix SSPE, 0.1% SDS (low stringency wash).<br>
(2)	Once at Tm-20 C for 15 minutes in 0.2x SSPE, 0.1% SDS (moderate stringency wash).<br>
The subject invention also concerns viral proteins and peptides encoded by the genes of an influenza virus of the present invention. In one embodiment, the viral protein is a mature HA protein. In a specific embodiment, the mature HA protein comprises one or more of the following: a serine at position 82; a leucine at position 221; a threonine at position 327; and/or a threonine at position 482. In an exemplified embodiment, the mature HA protein has an amino acid sequence shown in SEQ ID NO: 33 or SEQ ID NO: 34, or a functional and/or immunogenic fragment or variant of SEQ ID NO: 33 or SEQ ID NO: 34. In another embodiment, the viral protein is an NA protein, NS protein, PB protein, PA protein, or MA protein. Viral proteins and peptides of the invention can be used to generate antibodies that bind specifically to the protein or peptide. Viral proteins and peptides of the present invention can also be used as immunogens and in vaccine compositions.<br>
The subject invention also concerns compositions and methods for inducing an immune response against an influenza virus that is capable of infecting a susceptible host animal and causing respiratory disease. The invention can be used to induce an immune response against an influenza virus of any subtype in a susceptible host animal. For example, the influenza virus can be an HA subtype of HI, H2, H3, H4, H5, H6, H7, H8, H9, H10, Hll, H12, H13, H14, H15, or H16, and an NA subtype of Nl, N2, N3, N4, N5, N6, N7, N8, or N9. In one embodiment, the HA subtype is H3 or H5. In a further embodiment, the NA subtype is N7 or N8. In a specific embodiment, an immune response is induced against an influenza virus of subtype H3N8. In one embodiment, the host animal is a canid. Canines include wild, zoo, and domestic canines, such as wolves, coyotes, and foxes. Canines also include dogs, particularly domestic dogs, such as, for example, pure-bred and/or mongrel companion dogs, show dogs, working dogs, herding dogs, hunting dogs, guard dogs, police dogs, racing dogs, and/or<br>
laboratory dogs. In a specific embodiment, the host animal is a domesticated dog, such as a greyhound. In one embodiment, an animal is administered an effective amount of an immunogenic composition of the present invention sufficient to induce an immune response against an influenza virus of the invention. The immune response can be a humoral and/or cellular immune response. In a specific embodiment, the immune response is a protective immune response that is capable of preventing or minimizing viral infection in the immunized host animal for some period of time subsequent to the immunization. Thus, the subject invention also concerns vaccine compositions and methods that can provide a vaccinated animal with a protective immune response to a virus of the present invention.<br>
As described herein, the vaccine or immunogenic compositions of the subject invention may comprise cell-free whole virus, including attenuated or inactivated virus, or portions of the virus, including subvirion particles (including "split vaccine" wherein a virion is treated to remove some or all viral lipids), viral proteins (including individual proteins and macromolecular complexes of multiple proteins), polypeptides, and peptides, as well as virus-infected cell lines, or a combination of any of these. Vaccine or immunogenic compositions comprising virus-infected cell lines may comprise multiple cell lines, each infected with a different viral strain.<br>
The vaccine or immunogenic compositions of the subject invention also encompass recombinant viral vector-based constructs that may comprise, for example, genes encoding HA protein, NA protein, nucleoprotein, polymerase basic protein, polymerase acidic protein, and/or matrix protein of an influenza virus of the present invention. Any suitable viral vector that can be used to prepare a recombinant vector/virus construct is contemplated for use with the subject invention. For example, viral vectors derived from adenovirus, avipox, herpesvirus, vaccinia, canarypox, entomopox, swinepox, West Nile virus and others known in the art can be used with the compositions and methods of the present invention. Recombinant polynucleotide vectors that encode and express components can be constructed using standard genetic engineering techniques known in the art. In addition, the various vaccine compositions described herein can be used separately and in combination with each other. For example, primary immunizations of an animal may use recombinant vector-based constructs, having single or multiple strain components, followed by secondary boosts with vaccine compositions comprising inactivated virus or inactivated virus-infected cell lines. Other immunization protocols with the vaccine compositions of the invention are apparent to persons skilled in the art and are contemplated within the scope of the present invention.<br>
The subject invention also concerns reassortant virus comprising at least one gene or genomic segment of an influenza virus of the present invention and the remainder of viral genes or genomic segments from a different influenza virus of the invention or from an influenza virus other than a virus of the present invention. Reassortant virus can be produced by genetic reassortant of nucleic acid of a donor influenza virus of the present invention with nucleic acid of a recipient influenza virus and then selecting for reassortant virus that comprises the nucleic acid of the donor virus. Methods to produce and isolate reassortant virus are well known in the art (Fields et al., 1996). In one embodiment, a reassortant virus of the invention comprises genes or genomic segments of a human, avian, swine, or equine influenza virus. A reassortant virus of the present invention can include any combination of nucleic acid from donor and recipient influenza virus so long as the reassortant virus comprises at least one gene or<br>
genomic segment from a donor influenza virus of the present invention. In one embodiment, a recipient influenza virusj^n be an equine influenza virus.<br>
Natural, recombinant or synthetic polypeptides of viral proteins, and peptide fragments thereof, can also be used as vaccine compositions according to the subject methods. In one embodiment, viral polypeptides derived from multiple strains can be combined in a vaccine composition and are used to vaccinate a host animal. For example, polypeptides based on the viral HA protein from at least two different strains of influenza virus of the invention can be combined in the vaccine. The polypeptides may be homologous to one strain or may comprise "hybrid" or "chimeric" polypeptides whose amino acid sequence is derived from joining or linking polypeptides from at least two distinct strains. Procedures for preparing viral polypeptides are well known in the art. For example, viral polypeptides and peptides can be synthesized using solid-phase synthesis methods (Merrifield, 1963). Viral polypeptides and peptides can also be produced using recombinant DNA techniques wherein a polynucleotide molecule encoding an viral protein or peptide is expressed in a host cell, such as bacteria, yeast, or mammalian cell lines, and the expressed protein purified using standard techniques of the art.<br>
Vaccine compositions of the present invention also include naked nucleic acid compositions. In one embodiment, a nucleic acid may comprise a nucleotide sequence encoding an HA and/or an NA protein of an influenza virus of the present invention. Methods for nucleic acid vaccination are known in the art and are described, for example, in U.S. Patent Nos. 6,063,385 and 6,472,375. The nucleic acid can be in the form of a plasmid or a gene expression cassette. In one embodiment, the nucleic acid is provided encapsulated in a liposome which is administered to an animal.<br>
Vaccine compositions and immunogens, such as polypeptides and nucleic acids, that can be used in accordance with the present invention can be provided with a pharmaceutically-acceptable carrier or diluent. Compounds and compositions useful in the subject invention can be formulated according to known methods for preparing pharmaceutically useful compositions. Formulations are described in detail in a number of sources which are well known and readily available to those skilled in the art. For example, Remington's Pharmaceutical Science by E.W. Martin, Easton Pennsylvania, Mack Publishing Company, 19th ed., 1995, describes formulations which can be used in connection with the subject invention. In general, the compositions of the subject invention will be formulated such that an effective amount of an immunogen is combined with a suitable carrier in order to facilitate effective administration of the composition. The compositions used in the present methods can also be in a variety of forms. These include, for example, solid, semi-solid, and liquid dosage forms, such as tablets, pills, powders, liquid solutions or suspension, suppositories, injectable and infusible solutions, and sprays. The preferred form depends on the intended mode of administration and therapeutic application. The compositions also preferably include conventional pharmaceutically acceptable carriers and diluents which are known to those skilled in the art. Examples of carriers or diluents for use with the subject peptidomimetics include, but are not limited to, water, saline, oils including mineral oil, ethanol, dimethyl sulfoxide, gelatin, cyclodextrans, magnesium stearate, dextrose, cellulose, sugars, calcium carbonate, glycerol, alumina, starch, and equivalent carriers and diluents, or mixtures of any of these. Formulations of an immunogen of the invention can also comprise suspension agents, protcctants, lubricants, buffers, preservatives, and stabilizers. To provide for the administration of such dosages for<br>
the desired therapeutic treatment, pharmaceutical compositions of the invention will advantageously comprise betwejjm^ about 0.1% and 45%, and especially, 1 and 15% by weight of the immunogen or immunogens based on the weight of the total composition including carrier or diluent.<br>
The vaccine and immunogenic compositions of the subject invention can be prepared by procedures well known in the art. For example, the vaccine or immunogens are typically prepared as injectables, e.g., liquid solutions or suspensions. The vaccine or immunogens are administered in a manner that is compatible with dosage formulation, and in such amount as will be therapeutically effective and immunogenic in the recipient. The optimal dosages and administration patterns for a particular vaccine or immunogens formulation can be readily determined by a person skilled in the art.<br>
Peptides and/or polypeptides of the present invention can also be provided in the form of a multiple antigenic peptide (MAP) construct. The preparation of MAP constructs has been described in Tarn (1988). MAP constructs utilize a core matrix of lysine residues onto which multiple copies of an immunogen are synthesized (Posnett et al., 1988). Multiple MAP constructs, each containing the same or different immunogens, can be prepared and administered in a vaccine composition in accordance with methods of the present invention. In one embodiment, a MAP construct is provided with and/or administered with one or more adjuvants. Influenza polypeptides of the invention can also be produced and administered as macromolecular protein structures comprising one or more polypeptides. Published U.S. Patent Application US2005/0009008 describes methods for producing virus-like particles as a vaccine for influenza virus.<br>
According to the methods of the subject invention, the vaccine and immunogenic compositions described herein are administered to susceptible hosts, typically canids, and more typically domesticated dogs, in an effective amount and manner to induce protective immunity against subsequent challenge or infection of the host by virus. In one embodiment, the host animal is a canid. Canines include wild, zoo, and domestic canines, such as wolves, coyotes, and foxes. Canines also include dogs, particularly domestic dogs, such as, for example, pure-bred and/or mongrel companion dogs, show dogs, working dogs, herding dogs, hunting dogs, guard dogs, police dogs, racing dogs, and/or laboratory dogs. In a specific embodiment, the host animal is a domesticated dog, such as a greyhound. The vaccines or immunogens are typically administered parenterally, by injection, for example, either subcutaneously, intraperitoneally, or intramuscularly. Other suitable modes of administration include oral or nasal administration. Usually, the vaccines or immunogens are administered to an animal at least two times, with an interval of one or more weeks between each administration. However, other regimens for the initial and booster administrations of the vaccine or immunogens are contemplated, and may depend on the judgment of the practitioner and the particular host animal being treated.<br>
Virus and virus-infected cells in a vaccine formulation may be inactivated or attenuated using methods known in the art. For example, whole virus and infected cells can be inactivated or attenuated by exposure to paraformaldehyde, formalin, phenol, UV light, elevated temperature and the like. The amount of cell-free whole virus in a vaccine dose will usually be in the range from about 0.1 mg to about 5 mg, and more usually being from about 0.2 mg to about 2 mg. The dosage for vaccine formulations comprising virus-infected cell lines will usually contain from about 106 to about 108 cells per dose, and more usually from about 5 x 106 to about 7.5 x 107 cells per<br>
dose. The amount of protein or peptide immunogen in a dose for an animal can vary from about 0.1 ug to 10000 µg, or about 1 µg to 5000 µg, or about 10 µg to 1000 µg, or about 25 µg to 750 µg, or about 50 µg to 500 µg, or 100 µg to 250 µg, depending upon the size, age, etc., of the animal receiving the dose.<br>
An immunogenic or vaccine composition of the invention, such as virus or virus-infected cells or viral proteins or peptides, can be combined with an adjuvant, typically just prior to administration. Adjuvants contemplated for use in the vaccine formulations include threonyl muramyl dipeptide (MDP) (Byars et al, 1987), saponin, Cornebacteriumparvum, Freund's complete and Fruend's incomplete adjuvants, aluminum, or a mixture of any of these. A variety of other adjuvants suitable for use with the methods and vaccines of the subject invention, such as alum, are well known in the art and are contemplated for use with the subject invention.<br>
The subject invention also concerns antibodies that bind specifically to a protein or a peptide of the present invention. Antibodies of the subject invention include monoclonal and polyclonal antibody compositions. Preferably, the antibodies of the subject invention are monoclonal antibodies. Whole antibodies and antigen binding fragments thereof are contemplated in the present invention. Thus, for example, suitable antigen binding fragments include Fab2, Fab and Fv antibody fragments. Antibodies of the invention can be labeled with a detectable moiety, such as a fluorescent molecule (e.g., fluorescein or an enzyme).<br>
The subject invention also concerns methods and compositions for detection and identification of an influenza virus of the invention and for diagnosis of infection of an animal with an influenza virus of the present invention. The methods of the invention include detection of the presence of canine influenza, in a biological sample from an animal. The detection of canine influenza in a sample, is useful to diagnose canine influenza in an animal. In turn, this information can provide the ability to determine the prognosis of an animal based on distinguishing levels of canine influenza present over time, and can assist in selection of therapeutic agents and treatments for the animal, and assist in monitoring therapy. The method also provides the ability to establish the absence of canine influenza in an animal tested.<br>
The ability to detect canine influenza in an animal permits assessment of outbreaks of canine influenza in different geographical locations. This information also permits early detection so that infected animals can be isolated, to limit the spread of disease, and allows early intervention for treatment options. In addition, having this information available can provide direction to medical personnel for preparing to treat large numbers of ill animals, including assembling medical supplies, and, if available, vaccines.<br>
In one embodiment, a method of the present invention involves the collection of a biological sample from a test animal, such as a canine. The biological sample may be any biological material, including, cells, tissue, hair, whole blood, serum, plasma, nipple aspirate, lung lavage, cerebrospinal fluid, saliva, sweat and tears.<br>
The animal test sample may come from an animal suspected of having canine influenza virus, whether or not the animal exhibits symptoms of the disease. Control samples can also be provided or collected from animals known to be free of canine influenza. Additional controls may be provided, e.g., to reduce false positive and false negative results, and verify that the reagents in the assay are actively detecting canine influenza A virus.<br>
In addition to detecting the presence or absence of canine influenza in a biological sample, the methods of detection used in the invention can detect mutations in canine influenza virus, such as changes in nucleic acid<br>
sequence, that may result from the environment, drug treatment, genetic manipulations or mutations, injury, change in diet^ aging, or any other characteristic(s) of an animal. Mutations may also cause canine influenza A to become resistant to a drug that was formerly effective, or to enable the virus to infect and propagate in a different species of animal, or human. For example, avian influenza A virus has been shown to infect other animals and humans.<br>
In one embodiment for detecting an influenza virus in an animal, diagnosis is facilitated by the collection of high-quality specimens, their rapid transport to a testing facility, and appropriate storage, before laboratory testing. Virus is best detected in specimens containing infected cells and secretions. In one embodiment, specimens for the direct detection of viral antigens and/or for nucleic acids and/or virus isolation in cell cultures are taken during the first 3 days after onset of clinical symptoms. A number of types of specimens are suitable to diagnose virus infections of the upper respiratory tract, including, but not limited to, nasal swab, nasopharyngeal swab, nasopharyngeal aspirate, nasal wash and throat swabs. In addition to swabs, samples of tissue or serum may be taken, and invasive procedures can also be performed.<br>
In one embodiment, respiratory specimens are collected and transported in 1-5 ml of virus transport media. A number of media that are satisfactory for the recovery of a wide variety of viruses are commercially available. Clinical specimens are added to transport medium. Nasal or nasopharyngeal swabs can also be transported in the virus transport medium. One example of a transport medium is 10 gm of veal infusion broth and 2 gm of bovine albumin fraction V, added to sterile distilled water to 400 m. Antibiotics such as 0.8 ml gentamicin sulfate solution (50 mg/ml) and 3.2 ml amphotericin B (250 ug/ml) can also be added. The medium is preferably sterilized by filtration. Nasal washes, such as sterile saline (0.85% NaCl), can also be used to collect specimens of respiratory viruses.<br>
In one embodiment, sera is collected in an amount of from 1-5 ml of whole blood from an acute-phase animal, soon after the onset of clinical symptoms, and preferably not later than 7 days. A convalescent-phase serum specimen can also be collected, for example at about 14 days after onset of symptoms. Serum specimens can be useful for detecting antibodies against respiratory viruses in a neutralization test.<br>
In some instances, samples may be collected from individual animals over a period of time (e.g., once a day, once a week, once a month, biannually or annually). Obtaining numerous samples from an individual animal, over a period of time, can be used to verify results from earlier detections, and/or to identify response or resistance to a specific treatment, e.g., a selected therapeutic drug.<br>
The methods of the present invention can be used to detect the presence of one or more pathological agents in a test sample from an animal, and the level of each pathological agent. Any method for detecting the pathological agent can be used, including, but not limited to, antibody assays including enzyme-linked immunosorbent assays (ELISAs), indirect fluorescent antibody (IFA) tests, hemagglutinating, and inhibition of hemagglutination (HI) assays, and Western Blot. Known cell-culture methods can also be used. Positive cultures can be further identified using immunofluorescence of cell cultures or HI assay of the cell culture medium (supernatant).<br>
In addition, methods for detecting nucleic acid (DNA or RNA) or protein can be used. Such methods include, but are not limited to, polymerase chain reaction (PCR), and reverse transcriptase (RT) PCR tests and real<br>
time tests, and quantitative nuclease protection assays. There are commercially available test kits available to perfon^i these assays. For example, QIAGEN (Valencia, CA) sells a one step RT-PCR kit, and viral RNA extraction kit.<br>
In one embodiment, the method utilizes an antibody specific for a virus or viral protein of the invention. In a specific embodiment, an antibody specific for an HA protein of a virus of the invention is utilized. In another embodiment, an antibody specific for an NP protein of a virus of the invention is used. A suitable sample, such as from the nasal or nasopharyngeal region, is obtained from an animal and virus or viral protein is isolated therefrom. The viral components are then screened for binding of an antibody specific to a protein, such as HA or NP, of a virus of the invention. In another embodiment, a serum sample (or other antibody containing sample) is obtained from an animal and the serum screened for the presence of antibody that binds to a protein of a virus of the invention. For example, an ELISA assay can be performed where the plate walls have HA and/or NP protein, or a peptide fragment thereof, bound to the wall. The plate wall is then contacted with serum or antibody from a test animal. The presence of antibody in the animal that binds specifically to the HA and/or NP protein is indicative that the test animal is infected or has been infected with an influenza virus of the present invention.<br>
In one embodiment, the presence of a pathological agent is detected by determining the presence or absence of antibodies against the agent, in a biological sample. It can take some time (e.g. months) after an animal is infected before antibodies can be detected in a blood test. Once formed, antibodies usually persist for many years, even after successful treatment of the disease. Finding antibodies to canine influenza A may not indicate whether the infection was recent, or sometime in the past.<br>
Antibody testing can also be done on fluid(s). Antibody assays include enzyme-linked immunosorbent assays (ELISAs), indirect fluorescent antibody (IFA) assays, and Western Blot. Preferably, antibody testing is done using multiple assays, for example ELISA or IFA followed by Western blot. Antibody assays can be done in a two-step process, using either an ELISA or IFA assay, followed by a Western blot assay. ELISA is considered a more reliable and accurate assay than IFA, but IFA may be used if ELISA is not available. The Western blot test (which is a more specific test) can also be done in all animals, particularly those that have tested positive or borderline positive (equivocal) in an ELISA or IFA assay.<br>
Other antibody-based tests that can be used for detection of influenza virus include hemagglutination inhibition assays. Hemagglutination activity can be detected in a biological sample from an animal, using chicken or turkey red blood cells as described (Burleson et al., 1992) and Kendal et al., 1982). In one embodiment, an influenza or an HA protein or peptide of the invention is contacted with a test sample containing serum or antibody. Red blood cells (RBC) from an animal, such as a bird, are then added. If antibody to HA is present, then the RBC will not agglutinate. If antibody to HA is not present, the RBC will agglutinate in the presence of HA. Variations and modifications to standard hemagglutination inhibition assays are known in the art and contemplated within the scope of the present invention.<br>
Infection of an animal can also be determined by isolation of the virus from a sample, such as a nasal or nasopharyngeal swab. Viral isolation can be performed using standard methods, including cell culture and egg inoculation.<br>
In a further embodiment, a nucleic acid-based assay can be used for detection of a virus of the present invention. In one embodiment, a nucleic acid sample is obtained from an animal and the nucleic acid subjected to PCR using primers that will generate an amplification product if the nucleic acid contains a sequence specific to an influenza virus of the present invention. In a specific embodiment, RT-PCR is used in an assay for the subject virus. In an exemplified embodiment, real-time RT-PCR is used to assay for an influenza virus of the invention. PCR, RT-PCR and real-time PCR methods are known in the art and have been described in U.S. Patent Nos. 4,683,202; 4,683,195; 4,800,159; 4,965,188; 5,994,056; 6,814,934; and in Saiki et at (1985); Sambrook et al. (1989); Lee et al. (1993); and Livak et al. (1995). In one embodiment, the PCR assay uses oligonucleotides specific for an influenza matrix (MA) gene and/or HA gene. The amplification product can also be sequenced to determine if the product has a sequence of an influenza virus of the present invention. Other nucleic acid-based assays can be used for detection and diagnosis of viral infection by a virus of the invention and such assays are contemplated within the scope of the present invention. In one embodiment, a sample containing a nucleic acid is subjected to a PCR-based amplification using forward and reverse primers where the primers are specific for a viral polynucleotide or gene sequence. If the nucleic acid in the sample is RNA, then RT-PCR can be performed. For real-time PCR, a detectable probe is utilized with the primers.<br>
Primer sets specific for the hemagglutinin (HA) gene of many of the circulating influenza viruses are known, and are continually being developed. The influenza virus genome is single-stranded RNA, and a DNA copy (cDNA) must be made using a reverse transcriptase (RT) polymerase. The amplification of the RNA genome, for example using RT-PCR, requires a pair of oligonucleotide primers, typically designed on the basis of the known HA sequence of influenza A subtypes and of neuraminadase (NM)-l. The primers can be selected such that they will specifically amplify RNA of only one virus subtype. DNAs generated by using subtype-specific primers can be further analyzed by molecular genetic techniques such as sequencing. The test is preferably run with a positive control, or products are confirmed by sequencing and comparison with known sequences. The absence of the target PCR products (i.e, a "negative" result) may not rule out the presence of the virus. Results can then be made available within a few hours from either clinical swabs or infected cell cultures. PCR and RT-PCR tests for influenza A virus are described by Fouchier et al., 2000 and Maertzdorf et al., 2004.<br>
The subject invention also concerns methods for screening for compounds or drugs that have antiviral activity against a virus of the present invention. In one embodiment, cells infected with a virus of the invention are contacted with a test compound or drug. The amount of virus or viral activity following contact is then determined. Those compounds or drugs that exhibit antiviral activity can be selected for further evaluation.<br>
The subject invention also concerns isolated cells infected with an influenza virus of the present invention. In one embodiment, the cell is a canine cell, such as canine kidney epithelial cells.<br>
The subject invention also concerns cells transformed with a polynucleotide of the present invention encoding a polypeptide of the invention. Preferably, the polynucleotide sequence is provided in an expression construct of the invention. More preferably, the expression construct provides for overexpression in the cell of an operably linked polynucleotide of the invention. In one embodiment, the cell is transformed with a polynucleotide sequence comprising a sequence encoding the amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10,<br>
12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional fragment or variant thereof. In a specific embodiment, the cell is transformed with a polynucleotide encoding the amino acid sequence shown in SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78 comprises the nucleotide sequence shown in SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, respectively, or a sequence encoding a functional fragment or variant of any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78. Thus, the subject invention concerns cells transformed with a polynucleotide sequence comprising the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7,9, 11, 13, 15, 17, 19,21, 23, 25,27,29, 31,47,49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, or a fragment or variant, including a degenerate variant, of any of SEQ ID NOs: 1,3,5,7,9, 11, 13, 15, 17, 19,21,23,25,27,29,31,47,49,51,53,55,57,59,61,63,65, 67, 69,71,73, 75, or 77.<br>
The transformed cell can be a eukaryotic cell, for example, a plant cell, including protoplasts, or the transformed cell can be a prokaryotic cell, for example, a bacterial cell such as E. coll or B. subtilis. Animal cells include human cells, mammalian cells, partially canine cells, avian cells, and insect cells. Plant cells include, but are not limited to, dicotyledonous, monocotyledonous, and conifer cells.<br>
The subject invention also concerns plants, including transgenic plants that express and produce a viral protein or polypeptide of the present invention. Plants, plant tissues, and plant cells transformed with or bred to contain a poiynucleotide of the invention are contemplated by the present invention. Preferably, the polynucleotide of the invention is overexpressed in the plant, plant tissue, or plant cell. Plants can be used to produce influenza vaccine compositions of the present invention and the vaccines can be administered through consumption of the plant (see, for example, U.S. Patent Nos. 5,484,719 and 6,136,320).<br>
The subject invention also concerns kits for detecting a virus or diagnosing an infection by a virus of the present invention. In one embodiment, a kit comprises an antibody of the invention that specifically binds to an influenza virus of the present invention, or an antigenic portion thereof. In another embodiment, a kit comprises one or more polypeptides or peptides of the present invention. In a specific embodiment, the polypeptides have an amino acid sequence shown in any of SEQ ID NOs. 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52. 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic fragment or variant thereof. In a further embodiment, a kit comprises one or more polynucleotides or oligonucleotides of the present invention. In a specific embodiment, the polynucleotides have a nucleotide sequence shown in any of SEQ ID NOs. 1,3,5,7,9, 11, 13, 15, 17, 19,21,23,25,27,29,31,47,49,51,53,55,57,59,61,63,65,67,69,71,73, 75, or 77, or a fragment or variant thereof. A kit may optionally comprise one or more control antibody, control polypeptide or peptide, and/or control polynucleotide or oligonucleotide. The antibody, polypeptides, peptides, po\ynuc\eotides, and/or oligonucleotides of the kit can be provided in a suitable container or package.<br>
The subject application also concerns the use of mongrel dogs as a model for infection and pathogenesis of influenza virus. In one embodiment, a mongrel dog is inoculated with an influenza virus, such as a canine influenza virus of the present invention. Optionally, the dog can be administered therapeutic agents subsequent to<br>
inoculation. The dog can also have been administered a composition for generating an immune response against an influcn/za virus prior to inoculation with virus. Tissue, blood, serum, and other biological samples can be obtained before and/or after inoculation and examined for the presence of virus and pathogenesis of tissue using methods known in the art including, but not limited to, PCR, RT-PCR, nucleic acid sequencing, and immunohistochemistry.<br>
Any element of any embodiment disclosed herein can be combined with any other element or embodiment disclosed herein and such combinations are specifically contemplated within the scope of the present invention.<br>
All patents, patent applications, provisional applications, and publications referred to or cited herein are incorporated by reference in their entirety, including all figures and tables, to the extent they are not inconsistent with the explicit teachings of this specification.<br>
MATERIALS AND METHODS FOR EXAMPLES 1-6<br>
Blood and Nasal Swab Collection from Greyhounds.<br>
Acute and convalescent blood samples were collected by jugular venipuncture from clinically diseased or normal greyhounds in racing kennels experiencing outbreaks of respiratory disease. Convalescent samples were collected 4 to 12 weeks after the acute sample. Serum was harvested and stored at -80°C. Nasal swabs were collected and placed in Amies transport medium with charcoal (Becton Dickinson Biosciences) pending submission for bacterial isolation.<br>
Postmortem examination of greyhounds.<br>
Complete postmortem examinations were performed by the Anatomic Pathology Service at the University of Florida College of Veterinary Medicine (UF CVM) on 5 of the 8 greyhounds that died in the January 2004 outbreak at a Florida track. Postmortem examination of another dog was performed at a private veterinary clinic with submission of tissues to the UF CVM for histopathologic diagnosis. Tissues were fixed in 10% neutral buffered formalin, embedded in paraffin, and 5-u.m sections were either stained with hematoxylin and eosin for histopathologic diagnosis or processed for immunohistochemistry as described below. Unfixed tissues were submitted for bacterial culture and also stored at -80°C.<br>
Serological tests for canine viral respiratory pathogens.<br>
Paired acute and convalescent serum samples were submitted to the Animal Health Diagnostic Laboratory (AHDL) at the Cornell University College of Veterinary Medicine for serum neutralization assays against canine distemper virus, adenovirus type 2, and parainfluenza virus. Antibody liters were expressed as the last dilution of serum that inhibited viral infection of cell cultures. Seroconversion, defined as a ≥ 4-fold increase in antibody titer between the acute and convalescent sample, indicated viral infection. No seroconversions to these viral pathogens were detected.<br>
Microbial tests for canine bacterial respiratory pathogens.<br>
/Paired    nasal    swabs    and    postmortem    tissues    were    submitted    to    the    Diagnostic    Clinical ^ Microbiology/Parasitology/Serology Service at the UF CVM for bacterial isolation and identification. The samples<br>
were cultured on nonselective media as well as media selective for Bordetella species (Regan-Lowe; Remel) and Mycoplasma species (Remel). All cultures were held for 21 days before reporting no growth. Nasal swabs from some of the greyhounds were also submitted to the Department of Diagnostic Medicine/Pathobiology at the Kansas State University College of Veterinary Medicine for bacterial culture. Of 70 clinically diseased dogs tested, Bordetella bronchiseptica was isolated from the nasal cavity of 1 dog, while Mycoplasma spp. were recovered from the nasal cavity of 33 dogs. Pasteurella multocida was commonly recovered from the nasal cavity of dogs with purulent nasal discharges. Two of the dogs that died in the January 2004 outbreak had scant growth of Escherichia coli in the lungs postmortem, one dog had scant growth of E. coli and Streptococcus canis, and another had scant growth of Pseudomonas aeruginosa and a yeast. Neither Bordetella bronchiseptica nor Mycoplasma was isolated from the trachea or lungs of dogs that died.<br>
Virus isolation from postmortem tissues.<br>
Frozen tissues were thawed and homogenized in 10 volumes of minimum essential medium (MEM) supplemented with 0.5% bovine serum albumin (BSA) and antibiotics. Solid debris was removed by centrifugation and supernatants were inoculated onto cultured cells or into 10-day old embryonated chicken eggs. Tissue homogenates from greyhounds that died were inoculated into diverse cell cultures that supported the replication of a broad range of viral pathogens. The cell cultures included Vero (African green monkey kidney epithelial cells, ATCC No. CCL-81), A-72 (canine tumor fibroblasts, CRL-1542), HRT-18 (human rectal epithelial cells, CRL-11663), MDCK (canine kidney epithelial cells, CCL-34), primary canine kidney epithelial cells (AHDL, Cornell University), primary canine lung epithelial cells (AHDL), and primary bovine testicular cells (AHDL). MDCK and HRT cells were cultured in MEM supplemented with 2.5 ug/mL TPCK-treated trypsin (Sigma); the remaining cell lines were cultured in MEM supplemented with 10% fetal calf serum and antibiotics. Cells were grown in 25 cm2 flasks at 37°C in a humidified atmosphere containing 5% CO2 A control culture was inoculated with the supplemented MEM. The cultures were observed daily for morphologic changes and harvested at 5 days post inoculation. The harvested fluids and cells were clarified by centrifugation and inoculated onto fresh cells as described for the initial inoculation; two blind passages were performed. Hemagglutination activity in the clarified supernatants was determined using chicken or turkey red blood cells as described (Burleson et al, 1992; Kendal et al, 1982). For virus isolation in chicken embryos, 0.1 mL of tissue homogenate was inoculated into the allantoic sac and incubated for 48 hours at 35°C. After two blind passages, the hemagglutination activity in the allantoic fluids was determined as described (Burleson et al., 1992; Kendal etal, 1982).<br>
RT-PCR, nucleotide sequencing, and phylogenetic analyses.<br>
Total RNA was extracted from tissue culture supernatant or allantoic fluid using the RNeasy kit (Qiagen, Valencia, CA) according to manufacturer's instructions. The total RNA (10 ng) was reverse transcribed to cDNA<br>
using a one-step RT-PCR Kit (Qiagen, Valencia, CA) according to manufacturer's instructions. PCR amplification of the/coding region of the 8 influenza viral genes in the cDNA was performed as previously described (Klimov et al., 1992a), using universal gene-specific primer sets. The resulting DNA amplicons were used as templates for automated sequencing on an Applied Biosystems 3100 automated DNA sequencer using cycle sequencing dye terminator chemistry (ABI). Nucleotide sequences were analyzed using the GCG Package0, Version 10.0 (Accelyrs) (Womble, 2000). The Phylogeny Inference Package0 Version 3.5 was used to estimate phylogenies and calculate bootstrap values from the nucleotide sequences (Felsenstein, 1989). Phylogenetic trees were compared to those generated by neighbor-joining analysis with the Tamura-Nei gamma model implemented in the MEGA0 program (Kumar et al., 2004) and confirmed by the PALJP0 4.0 Beta program (Sinauer Associates).<br>
Experimental inoculation of dogs.<br>
Four 6-month old specific pathogen-free beagles [(2 males and 2 females (Liberty Research)] were used. Physical examination and baseline blood tests including complete blood cell count/differential, serum chemistry panel, and urinalysis determined that the animals were healthy. They were housed together in a BSL 2-enhanced facility accredited by the Association for Assessment and Accreditation of Laboratory Animal Care. Baseline rectal temperatures were recorded twice daily for 7 days. The dogs were anesthetized by intravenous injection of propofol (Diprivan®, Zeneca Pharmaceuticals, 0.4 mg/kg body weight to effect) for intubation with endotracheal tubes. Each dog was inoculated with a total dose of 1066 median tissue culture infectious doses (TCID50) of A/Canine/Florida/43/2004 (Canine/FL/04) (H3N8) virus with half the dose administered into the distal trachea through the endotracheal tube and the other half administered into the deep nasal passage through a catheter. Physical examinations and rectal temperature recordings were performed twice daily for 14 days post inoculation (p.i.). Blood samples (4 mL) were collected by jugular venipuncture on days 0, 3, 5, 7, 10, and 14 p.i. Nasal and oropharyngeal specimens were collected with polyester swabs (Fisher Scientific) from each dog on days 0 to 5, 7, 10, and 14 p.i. The swabs were placed in viral transport medium (Remel) and stored at -80°C. Two dogs (1 male and 1 female) were euthanatized by intravenous inoculation of Beuthanasia-D® solution (1 mL/5 kg body weight; Schering-Plough Animal Health Corp) on day 5 p.i. and the remaining 2 dogs on day 14 for postmortem examination. Tissues for histological analysis were processed as described. Tissues for virus culture were stored at -80°C. This study was approved by the University of Florida Institutional Animal Care and Use Committee.<br>
Virus shedding from experimentally inoculated dogs.<br>
Serial dilutions of lung homogenates and swab extracts, prepared by clarification of the swab transport media by centrifugation, were set up in MEM supplemented with 0.5% BSA and antibiotics. Plaque assays were performed as described (Burleson et al, 1992) using monolayers of MDCK cells in 6-well tissue culture plates. Inoculated cell monolayers were overlaid with supplemented MEM containing 0.8% agarose and 1.5 ug/mL of TPCK-trypsin. Cells were cultured for 72 hours at 37°C in a humidified atmosphere containing 5% CO2 prior to fixation and staining with crystal violet. Virus concentration was expressed as plaque forming units (PFU) per gram of tissue or per swab.<br>
Immunohistochemistry.<br>
Deparaffinized and rehydrated 5-µm lung tissue sections from the greyhounds and beagles were mounted on Bond-Rite™ slides (Richard-Allan Scientific, Kalamazoo, MI) and subsequently treated with proteinase K (DakoCytomation, Carpenteria, CA) followed by peroxidase blocking reagent (Dako® EnVision™ Peroxidase Kit, Dako Corp.). The sections were incubated with 1:500 dilutions of monoclonal antibodies to canine distemper virus (VMRD, Inc.), canine adenovirus type 2 (VMRD, Inc.), canine parainfluenza virus (VMRD, Inc.), or influenza A H3 (Chemicon International, Inc.) for 2 hours at room temperature. Controls included incubation of the same sections with mouse IgG (1 mg/mL, Serotec, Inc.), and incubation of the monoclonal antibodies with normal canine lung sections. Following treatment with the primary antibodies, the sections were incubated with secondary immunoperoxidase and peroxidase substrate reagents (Dako® EnVision™ Peroxidase Kit, Dako Corp.) according to the manufacturer's instructions. The sections were counterstained with hematoxylin, treated with Clarifier #2 and Bluing Reagent (Richard-Allan Scientific, Kalamazoo, MI), dehydrated, and coverslips applied with Permount (ProSciTech).<br>
Hemagglutination inhibition (HI) assay.<br>
Serum samples were incubated with receptor destroying enzyme (RDE, Denka) (1 part serum: 3 parts RDE) for 16 hours at 37°C prior to heat inactivation for 60 minutes at 56°C. Influenza A/Canine/FL/04 (H3N8) virus was grown in MDCK cells for 36-48 hr at 37°C. Virus culture supernatants were harvested, clarified by centrifugation, and stored at -80°C. The HI assay was performed as described previously (Kendal et a/., 1982). Briefly, 4 hemagglutinating units of virus in 25ul were added to an equal volume of serially diluted serum in microtiter wells and incubated at room temperature for 30 minutes. An equal volume of 0.5% v/v turkey erythrocytes was added and the hemagglutination liters were estimated visually after 30 minutes. The endpoint HI liter was defined as the last dilution of serum that completely inhibited hemagglutination. Seroconversion was defined as ≥ 4-fold increase in HI titer between paired acute and convalescent samples. Seropositivity of a single sample was defined as a HI antibody titer ≥1:32.<br>
Microneutralization (MN) assay.<br>
Neutralizing serum antibody responses to A/Canine/FL/04 (H3N8) were detected by a MN assay as described previously (Rowe et al., 1999) except that canine sera were RDE-treated as described above prior to the assay. The endpoint titer was defined as the highest dilution of serum that gave 50% neutralization of 100 TCID50 of virus. Seroconversion was defined as ≥ 4-fold increase in MN titer between paired acute and convalescent samples. Seropositivity of a single sample was defined as a MN titer ≥ 1:80.<br>
Following are examples which illustrate procedures for practicing the invention. These examples should not be construed as limiting. All percentages are by weight and all solvent mixture proportions are by volume unless otherwise noted.<br>
EXAMPLE<br>
^ In January 2004, an outbreak of respiratory disease occurred in 22 racing greyhounds housed in 2 kennels at a Florida track and the local farm that supplied dogs to these kennels. There were approximately 60 dogs in each kennel building and 300 dogs at the farm. The outbreak occurred over a 6-day period after which no new cases were identified. Fourteen of the 22 dogs had fevers of 39.5 to 41.5°C, a soft, gagging cough for 10 to 14 days, and eventual recovery. Of the remaining 8 dogs, 6 apparently healthy dogs died unexpectedly with hemorrhage from the mouth and nose. Two other dogs were euthanatized within 24 hours of onset of hemorrhage from the mouth and nose due to rapid deterioration. Both of these dogs had fevers of 41°C. Four of the 8 deaths occurred in the kennel buildings and 4 occurred at the farm. Fifty percent of the deaths occurred on day 3 of the outbreak. The 22 dogs ranged in age from 17 months to 4 years, but 73% were 17 to 33 months old.<br>
Two clinical syndromes were evident: a milder illness characterized by initial fever and then cough for 10-14 days (14 dogs) with subsequent recovery, or a peracute death associated with hemorrhage in the respiratory tract (8 dogs for a mortality rate of 36%). Postmortem examinations were performed on 6 of the 8 fatal cases. All dogs had extensive hemorrhage in the lungs, mediastinum, and pleural cavity. Histological examination of the respiratory tract revealed that in addition to pulmonary hemorrhage, all dogs had tracheitis, bronchitis, bronchiolitis, and suppurative bronchopneumonia (Figure 3). The epithelial lining and airway lumens in these tissues were infiltrated by neutrophils and macrophages. Lung homogenates prepared from these dogs were inoculated into a variety of monkey, human, bovine, and canine cell lines for virus culture. The lung homogenate from one dog caused cytopathic effects in Madin-Darby canine kidney epithelial cells (MDCK) cultured in the presence of trypsin, and the cell culture supernatant agglutinated chicken red blood cells. Preliminary evidence of an influenza type A virus was provided by a commercial ELISA for detection of the nucleoprotein of influenza A and B viruses, and by PCR analysis using primers specific for the matrix gene of influenza A viruses. In addition, the hemaggtutinating activity was inhibited by reference antisera to the equine influenza A H3 subtype, but not by antisera specific for human influenza A subtypes H1-HI1 and H13 (Table 3). To characterize the molecular properties of the virus, we determined the nucleotide sequences of the 8 RNA segments of the viral genome. Sequence comparisons with known influenza virus genes and phylogenetic analyses indicated that the 8 genes of the canine isolate were most similar to those from contemporary equine influenza A (H3N8) viruses, with which they shared 2:96-97% sequence identity (Figure 1A, Table 4). In contrast, representative genes from avian, swine, and human influenza A isolates had 
EXAMPLE 2<br>
To investigate the role of the Canine/FL/04 virus in the clinical and pathological observations in the greyhounds, we performed immunohistochemical staining (IHC) on lung tissues using a monoclonal antibody to influenza A H3. Viral H3 antigen was consistently detected in the cytoplasm of bronchial and bronchiolar<br>
epithelial cells, bronchial gland epithelial cells, and macrophages in airway lumens and alveolar spaces (Figure 2A). /These data support a diagnosis of pulmonary infection with influenza virus of the H3 subtype in multiple dogs.<br>
EXAMPLE 3<br>
To determine involvement of a Canine/FL/04-like virus in the etiology of the respiratory disease outbreak, we analyzed paired acute and convalescent sera from 11 sick dogs and 16 asymptomatic contacts by hemagglutination inhibition (HI) and microneutralization (MM). Seroconversion, defined as a &gt; 4-fold rise in antibody liter to Canine/FL/04 from the acute to convalescent phase, occurred in 8 of 11 (73%) sick dogs in both assays (Table 1). Seroconversion occurred in 6 of 16 (38%) asymptomatic contacts in the HI assay, while 8 of 16 (50%) seroconverted in the MN assay (Table 1). The Seroconversion data demonstrated infection of the dogs with a Canine/FL/04-like virus which coincided temporally with the onset of respiratory disease in most animals.<br>
Single serum samples were collected 3 months after the outbreak from an additional 46 asymptomatic dogs housed with the sick dogs. Of these, 43 (93%) were seropositive in both assays. For the total population of 73 dogs tested, 93% were seropositive in both assays, including 82% (9/11) of the sick dogs and 95% (59/62) of the healthy contacts. The high seroprevalence in dogs with no history of respiratory disease indicates that most infections with canine influenza virus are subclinical and suggest efficient spread of the virus among dogs. It is not known if subclinical infections contribute to the spread of the virus.<br>
EXAMPLE 4<br>
To better understand the capacity of the Canine/FL/04 virus to infect dogs, four 6-month old purpose-bred beagles were each inoculated with 1066 median tissue culture infectious doses (TCIDso) by the intratracheal and intranasal routes. All dogs developed a fever (rectal temperature S39°C) for the first 2 days postinoculation (p.i.), but none exhibited respiratory symptoms such as cough or nasal discharge over a 14 day observation period. Virus shedding was examined by quantification of virus in nasal and oropharyngeal swabs. Only 2 of the 4 dogs shed detectable amounts of virus. One dog shed virus on days 1 and 2 p.i. (1.0-2.5 logic PFU per swab), whereas the other dog shed virus for 4 consecutive days after inoculation (1.4-4.5 logio PFU per swab). Postmortem examination of 2 dogs on day 5 p.i. revealed necrotizing and hyperplastic tracheitis, bronchitis, and bronchiolitis similar to that found in the spontaneous disease in greyhounds, but there was no pulmonary hemorrhage or bronchopneumonia. Viral H3 antigen was detected in the cytoplasm of epithelial cells of bronchi, bronchioles, and bronchial glands by IHC (Figure 2B). Infectious virus was recovered from the lung tissue of one of the dogs. Postmortem examination of the remaining 2 dogs on day 14 p.i. showed minimal histological changes in respiratory tissues, no viral H3 antigen by IHC, and no recovery of virus from lung homogenates. Seroconversion in these latter 2 dogs was detected in MN assays by day 7 p.i., with a further 2-to 3-fold increase in antibody titers by day 14. These results established the susceptibility of dogs to infection with Canine/FL/04, as evidenced by the febrile response, presence of viral antigen and infectious virus in the lung parenchyma, histopathological findings typical for influenza, and Seroconversion. The failure to reproduce severe disease and death in the experimentally<br>
inoculated  beagles  is  not surprising since  a large proportion  of the naturally  infected greyhounds were asymptomatic.<br>
EXAMPLE 5<br>
To investigate whether a Canine/FL/04-like influenza virus had circulated among greyhound populations in Florida prior to the January 2004 outbreak, archival sera from 65 racing greyhounds were tested for the presence of antibodies to Canine/FL/04 using the HI and MN assays. There were no detectable antibodies in 33 dogs sampled from 1996 to 1999. Of 32 dogs sampled between 2000 and 2003, 9 were seropositive in both assays - 1 in 2000, 2 in 2002, and 6 in 2003 (Table 5). The seropositive dogs were located at Florida tracks involved in outbreaks of respiratory disease of unknown etiology from 1999 to 2003, suggesting that a Canine/FL/04-like virus may have been the causative agent of those outbreaks. To investigate this possibility further, we examined archival tissues from greyhounds that died from hemorrhagic bronchopneumonia in March 2003. Lung homogenates inoculated into MDCK cells and chicken embryos from one dog yielded H3N8 influenza virus, termed A/Canine/Florida/242/2003 (Canine/FL/03). Sequence analysis of the complete genome of Canine/FL/03 revealed &gt;99% identity to Canine/FL/04 (Table 4), indicating that Canine/FL/04-like viruses had infected greyhounds prior to 2004.<br>
EXAMPLE 6<br>
From June to August 2004, respiratory disease outbreaks occurred in thousands of racing greyhounds at 14 tracks in Florida, Texas, Alabama, Arkansas, West Virginia, and Kansas.<br>
Officials at some of these tracks estimated that at least 80% of their dog population had clinical disease. Most of the dogs had clinical signs of fever (&gt; 39°C) and cough similar to the dogs in the January 2004 outbreak, but many dogs also had a mucopurulent nasal discharge. Multiple deaths were reported but an accurate mortality rate could not be determined.<br>
We collected paired acute and convalescent sera from 94 dogs located at 4 Florida tracks: 56% of these dogs had S4-fold rises in antibody liters to Canine/FL/04, and 100% were seropositive (Table 6). Convalescent sera from 29 dogs in West Virginia and Kansas also had antibodies to Canine/FL/04. We isolated influenza A (H3N8) virus from the lungs of a greyhound that died of hemorrhagic bronchopneumonia at a track in Texas. Sequence analysis of the entire genome of this isolate, named A/Canine/Texas/1/2004 (Canine/TX/04), revealed &gt;99% identity to Canine/FL/04 (Table 4). The isolation of three closely related influenza viruses from fatal canine cases over a 13-month period and from different geographic locations, together with the substantial serological evidence of widespread infection among racing greyhounds, suggested sustained circulation of a Canine/FL/04-like virus in the dog population.<br>
Phylogenetic analysis of the HA genes of Canine/FL/03, Canine/FL/04, and Canine/TX/04 showed that they constitute a monophyletic group with robust bootstrap support that was clearly distinct from contemporary H3 genes of equine viruses isolated in 2002 and 2003 (Figure IB). Phylogentic analysis and pairwise nucleotide sequence comparisons of the other 7 genomic segments supported the segregation of the canine genes as a distinct<br>
sub-lineage most closely related to the equine virus lineage (data not shown, and Table 4). The clustering of the canin/b influenza genes as a monophyletic group separate from equine influenza is also supported by the presence of 4 signature amino acid changes in the HA (Table 2). Together with the serological results from 2003 and 2004, these data are consistent with a single virus transmission event from horses to dogs with subsequent horizontal spread of the virus in the greyhound population. However, repeated introductions of this unique lineage of influenza virus from an unidentified reservoir species can not be formally excluded, unlikely as it may be.<br>
The viral HA is a critical determinant of host species specificity of influenza virus (Suzuki et al, 2000). To identify residues within HA that may be associated with adaptation to the canine host, we compared the deduced amino acid sequence of canine HAs to those of contemporary equine viruses. Four amino acid changes differentiate the equine and canine mature HA consensus amino acid sequences: N83S, W222L, I328T, and N483T (see Table 2). The canine viruses have an amino acid deletion when compared to the consensus equine sequences. Therefore, amino acid position 7 in the HA equine sequence is position 6 in the HA canine sequence, amino acid position 29 in the HA equine sequence is position 28 in the HA canine sequence, amino acid position 83 in the HA equine sequence is position 82 in the HA canine sequence, etc. Thus, the four substituted amino acids are at position 82, 221, 327, and 482 of the amino acid sequence shown in SEQ ID NO: 33 and SEQ ID NO: 34. The substitution of serine for asparagine at consensus sequence position 83 is a change of unknown functional significance since various polar residues are found in H3 molecules from other species. The strictly conserved isoleucine at consensus sequence position 328 near the cleavage site of the H3 HA has been replaced by threonine. The pivotal role of HA cleavage by host proteases in pathogenesis suggests that this change merits further study. The substitution of leucine for tryptophan at consensus sequence position 222 is quite remarkable because it represents a non-conservative change adjacent to the sialic acid binding pocket which could modulate receptor function (Weis et al, 1988). Interestingly, leucine at position 222 is not unique to canine H3 HA since it is typically found in the H4, H8, H9, and H12 HA subtypes (Nobusawa et al., 1991; Kovacova et al., 2002). The leucine substitution may be more compatible with virus specificity for mammalian hosts since infections of swine with subtype H4 (Karasin et al., 2000) and humans and swine with subtype H9 (Peiris et al., 1999) viruses have been reported. The substitution of asparagine with threonine at consensus sequence position 483 resulted in the loss of a glycosylation site in the HA2 subunit that is conserved in all HA subtypes (Wagner et al, 2002). Although the importance of these amino acid changes in the HA for adaptation of an equine virus to dogs remains to be determined, similar amino acid changes have been observed previously in association with interspecies transfer (Vines et al, 1998; Matrosovich et al, 2000). Amino acid differences between other influenza viral proteins of the invention and equine consensus sequence are shown in Tables 19 to 25.<br>
The source of the equine influenza virus that initially infected racing greyhounds remains speculative. Kennels at greyhound racetracks are not located near horses or horse racetracks, suggesting that contact between greyhounds and shedding horses is not a sufficient explanation for the multiple outbreaks in different states in 2004. A potential source of exposure to the equine virus is the feeding of horsemeat to greyhounds, whose diet is supplemented with raw meat supplied by packing houses that render carcasses, including horses which could carry influenza. Precedents for this mode of infection include reports of interspecies transmission of H5N1 avian<br>
influenza virus to pigs and zoo felids fed infected chicken carcasses (Webster, 1998; Keawcharoen et al, 2004; Kuik^n el al., 2004). Although this is a plausible route for the initial introduction of equine influenza into dogs, it does not explain the recent multiple influenza outbreaks in thousands of dogs in different states. Our experimental inoculation study demonstrated the presence of virus in the nasal passages and oropharynx of dogs, albeit at modest tilers. Nevertheless, these results indicate that shedding is possible, and that dog-to-dog transmission of virus by large droplet aerosols, fomites, or direct mucosal contact could play a role in the epizootiology of the disease.<br>
(Table Removed)<br>
The interspecies transfer of a whole mammalian influenza virus to an unrelated mammal species is a rare event. Previous studies have provided limited serological or virological evidence, but not both, of transient infection of dogs with human influenza A (H3N2) viruses (Nikitin et al., 1972, Kilbourne, et al, 1975; Chang et al, 1976; Houser et al, 1980). However, there was no evidence of sustained circulation in the canine host. Although direct transfer of swine influenza viruses from pigs to people is well-documented (Dacso et al, 1984; Kimura et al, 1998; Patriarca et al, 1984; Top et al, 1977), there is no evidence for adaptation of the swine viruses in human hosts. In this report, we provide virological, serological, and molecular evidence for interspecies transmission of an entire equine influenza A (H3N8) virus to another mammalian species, the dog. Unique amino acid substitutions in the canine virus HA, coupled with serological confirmation of infection of dogs in multiple states in the U.S., suggest adaptation of the virus to the canine host. Since dogs are a primary companion animal for humans, these findings have implications for public health; dogs may provide a new source for transmission of novel influenza A viruses to humans.<br>
(Table Removed)<br><br>
a Number of dogs with clinical signs of disease.<br>
b Number of asymptomatic dogs housed in contact with clinically diseased dogs.<br>
c Hemagglutination-inhibition (HI) assay using A/Canine/Florida/43/04 virus.<br>
d Microneutralization (MN) assay using A/Canine/Florida/43/04 virus.<br>
e Percentage of dogs with at least a 4-fold increase in antibody titer in paired acute and       convalescent sera.<br>
' Percentage of dogs with a positive antibody titer (HI titer £32: MN titer £80) in the convalescent sera.<br>
11 Geometric mean antibody titer for the convalescent sera.<br>
(Table Removed)<br>
* Amino acid residue (single letter code) and position in the mature H3 HA. The amino acid code is: A=alanine, D=aspartic  acid,  G=glycine,   l=isoleucine,  K=Iysine,  L=leucine,  M=methionine,  N=asparagine,   R=arginine,<br>
S=serine, T=threonine, V=valine, W=tryptophan .<br>
f Denotes no change from the consensus equine H3 HAs.<br><br>
(Table Removed)<br>
a Hemagglutination inhibition titer to virus isolate from dog # 43.<br>
b Polyclonal antisera were produced in ferrets, whereas all other antisera were produced in sheep or goats.<br><br><br>
(Table Removed)<br>
a Percent nucleotide and amino acid (in parentheses) sequence identity of A/Canine/Florida/43/04 (H3N8) genes to the most homologous gene of influenza virus virus isolates from the species, followed by their Genbank sequence database accession numbers. h  Not applicable: N8 neuraminidase was never reported in human or swine viruses.<br><br><br>
(Table Removed)<br>
"The year of serum sample collection from racing greyhounds in Florida.<br>
b Microneutralization assay antibody liters for seropositive dogs, including the range for the six 2003 seropositive dogs.<br><br>
(Table Removed)<br>
' Number of clinically diseased dogs tested by HI using A/canine/Florida/43/04 (H3N8).<br>
b Percentage of dogs with &gt;4-fold increase in antibody titer between acute and convalescent sera.<br>
c Percentage of dogs with a positive antibody titer (HI titer&gt;16) in the convalescent sera.<br>
d Geometric mean antibody titer for the convalescent sera.<br>
MATERIALS AND METHODS FOR EXAMPLES 7-11 Canine tissues<br>
Postmortem examinations were performed by the Anatomic Pathology Service at the University of Florida College of Veterinary Medicine on 6 mixed breed dogs that died in April/May 2005 during an influenza outbreak in a shelter facility in northeast Florida, and on a pet Yorkshire Terrier dog that died in May 2005 during an influenza outbreak in a veterinary clinic in southeast Florida. Tissues were fixed in 10% neutral buffered formalin, embedded in paraffin, and 5-um sections were stained with hematoxylin and eosin for histopathologic diagnosis. Unfixed tissues were stored at -80°C pending virological analyses.<br>
RNA extraction from canine tissue samples<br>
Frozen lung tissues from each of the 7 dogs were thawed and homogenized in minimum essential medium (MBM) supplemented with 0.5% bovine serum albumin (BSA) and antibiotics (gentamycin and ciprofloxacin) using a disposable tissue grinder (Kendall, Lifeline Medical Inc., Danbury, CT). Total RNA was extracted using a commercial kit (RNeasy® Mini Kit, QIAGEN Inc., Valencia, CA) according to manufacturer's instructions and eluted in a final volume of 60 (xL of buffer. Total RNA was also extracted from lung tissue collected from dogs without respiratory disease.<br>
Real-time RT-PCR<br>
A single-step quantitative real-time RT-PCR was performed on total RNA extracted from the canine tissue samples using the QuantiTect® Probe RT-PCR Kit containing ROX as a passive reference dye (QIAGEN Inc., Valencia, CA). Briefly, 2 primer-probe sets were used for detection of influenza A sequences in each sample (Table 7). One primer-probe set was selective for canine hemagglutinin (H3) gene sequences. The other primer-probe set targeted a highly conserved region of the matrix (M) gene of type A influenza virus. For each real-time RT-PCR reaction, 5 jiL of extracted total RNA were added to a reaction mixture containing 12.5 jaL of 2X QuantiTech® Probe RT-PCR Master Mix, 0.25 ^L of QuantiTech® RT Mix, forward and reverse primers (0.4 uM final concentration for each), probe (0.1 uM final concentration) and RNase-free water in a final volume of 25 ^L. The TaqMan® Ribosomal RNA Control Reagents (Applied Biosystems, Foster City, CA) were used according to manufacturer's instructions for detection of 18S rRNA as an endogenous internal control for the presence of RNA extracted from the canine tissue samples.<br>
Quantitative one-step real-time RT-PCR was performed on the reaction mixtures in a Mx3000P® QPCR System (Stratagene, La Jolla, CA). Cycling conditions included a reverse transcription step at 50°C for 30 minutes, an initial denaturation step at 95°C for 15 minutes to activate the HotStarTaq® DNA polymerase, and amplification for 40 cycles. Each amplification cycle included denaturation at 94°C for 15 seconds followed by annealing/extension at 60°C for 1 minute. The FAM (emission wavelength 518 nm) and VIC (emission wavelength 554 nm) fluorescent signals were recorded at the end of each cycle. The threshold cycle (Ct) was determined by setting the threshold fluorescence (dR) at 1000 in each individual experiment. The Mx3000P® version 2.0 software program (Stratagene, La Jolla, CA) was used for data acquisition and analysis. Samples were considered positive for influenza A virus when the threshold cycle (Ct) for the H3 or M gene was 3 units smaller than the Ct for lung tissues from dogs without respiratory disease. The positive control consisted of amplification of RNA extracted from A/canine/FL/242/03 (H3N8) virus.<br>
Virus isolation in MDCK cells<br>
Frozen lung tissues from each of the 7 dogs were thawed and homogenized in 10 volumes of Dulbecco's Modified Eagle Medium (DMEM) supplemented with 0.5% (BSA) and antibiotics (gentamycin and ciprofloxacin). Solid debris was removed by centrifugation and supernatants were inoculated onto Madin-Darby canine kidney (MDCK) cells cultured in DMEM supplemented with 1 u,g/mL TPCK-treated trypsin (Sigma-Aldrich Corp., St. Louis, MO) and antibiotics (gentamycin and ciprofloxacin). Cells were grown in 25 cm2 flasks at 37°C in a humidified atmosphere containing 5% CO2. The cultures were observed daily for morphologic changes and harvested at 5 days post inoculation. The harvested cultures were clarified by centrifugation and the supernatants inoculated onto fresh MDCK cells as described for the initial inoculation; two additional passages were performed for samples that did not show evidence of influenza virus by hemagglutination or RT-PCR. Hemagglutination activity in the clarified supernatants was determined using 0.5% turkey red blood cells as previously described (Burleson, F. et«/., 1992; Kendal, P. et al., 1982). RT-PCR was performed as described below.<br>
Virus isolation in embryonated chicken eggs<br>
Homogenates were prepared from frozen lung tissues as described above for inoculation of MDCK cells. The Hbmogenates (0.2 mL) were inoculated into the allantoic sac of 10-day old embryonated chicken eggs. After 48 hours of incubation at 35°C, the eggs were chilled at 4°C overnight before harvesting the allantoic fluid. Hemagglutination activity in the clarified supernatants was determined using 0.5% turkey red blood cells as previously described (Burleson, F. et al., 1992; Kendal, P. et al, 1982). RT-PCR was performed as described below. Two additional passages in embryonated eggs were performed for samples that did not show evidence of influenza virus after the initial inoculation.<br>
RT-PCR. nucleotide sequencing, and phylogenetic analyses<br>
Viral RNA was extracted from MDCK supernatant or allantoic fluid using the QIAamp® Viral RNA Mini Kit (Q1AGEN Inc., Valencia, CA) according to manufacturer's instructions. The viral RNA was reverse transcribed to cDNA using the QIAGEN® OneStep RT-PCR Kit (QIAGEN Inc., Valencia, CA) according to manufacturer's instructions. PCR amplification of the coding region of the 8 influenza viral genes in the cDNA was performed as previously described (Klimov, A. et al, 1992b), using universal gene-specific primer sets (primer sequences available on request). The resulting DNA amplicons were used as templates for automated sequencing in the ABI PRISM® 3100 automated DNA sequencer using cycle sequencing dye terminator chemistry (Applied Biosystems, Foster City, CA). Nucleotide sequences were analyzed using the Lasergene 6 Package® (DNASTAR, Inc., Madison, WI). The PHYLIP Version 3.5° software program was used to estimate phylogenies and calculate bootstrap values from the nucleotide sequences (Felsenstein, J., 1989). Phylogenetic trees were compared to those generated by neighbor-joining analysis with the Tamura-Nei model implemented in the MEGA0 program (Kumar, S. et al., 2004) and confirmed by the PAUP° 4.0 Beta program (Sinauer Associates, Inc., Sunderland, MA).<br>
Hemagglutination inhibition (HI) assay<br>
Serum samples were incubated with receptor destroying enzyme (RDE, DENKA SEIKEN Co., Ltd., Tokyo, Japan) (1 part serum: 3 parts RDE) for 16 hours at 37°C prior to heat inactivation for 30 minutes at 56°C. Influenza A/Canine/Jacksonville/05 (H3N8) virus was grown in MDCK cells for 72 hrs at 37°C in 5% CO2. Virus culture supernatants were harvested, clarified by centrifugation, and stored at -80°C. All other viruses used in the HI assay were grown in 10-day old embryonated chicken eggs from which allantoic fluid was collected and stored at -80°C. The HI assay was performed as described previously (Kendal, P. et al., 1982). Briefly, 4 hemagglutinating units of virus in 25ul were added to an equal volume of serially diluted serum in 96-well plastic plates and incubated at room temperature for 30 minutes. An equal volume of 0.5% turkey erythrocytes was added and the hemagglutination liters were estimated visually after 30 minutes. The endpoint HI titer was defined as the last dilution of serum that completely inhibited hemagglutination.<br>
EXAMPLE 7-CLINICAL CASES<br>
In April and May 2005, a previously described (Crawford, P.C. et al., 2005) respiratory disease outbreak occurred in dogs housed in a shelter facility in northeast Florida. The outbreak involved at least 58 dogs ranging in age from 3 months to 9 years, and included purebred dogs as well as mixed breeds. The most common clinical signs were purulent nasal discharge and a cough for 7 to 21 days. Of the 43 dogs that had clinical disease for &gt;7 days, 41 had HI antibody liters to canine/FL/04 (H3N8) ranging from 32 to &gt;1024. At least 10 dogs progressed to pneumonia, of which 6 were euthanized. These 6 mixed breed dogs included 3 males and 3 females ranging in age from 4 months to 3 years. The duration of clinical signs ranged from 2 to 10 days at the time of euthanasia. On postmortem examination, these dogs had pulmonary congestion and edema. Histological examination of the respiratory tract revealed rhinitis, tracheitis, bronchitis, bronchiolitis, and suppurative bronchopneumonia. There was epithelial cell necrosis and erosion in the trachea, bronchi, bronchioles, and bronchial glands. The respiratory tissues were infiltrated by neutrophils and macrophages.<br>
In May 2005, a respiratory disease outbreak occurred in 40 pet dogs at a veterinary clinic in southeast Florida. The most common clinical signs were purulent nasal discharge and a cough for 10 to 30 days. Of the 40 dogs, 17 were seropositive for canine/FL/04 (H3N8) with HI antibody tiers ranging from 32 to &gt;1024. Seroconversion occurred in 10 dogs for which paired acute and convalescent sera were available. Three dogs progressed to pneumonia. One of these dogs, a 9-year old male Yorkshire Terrier, died 3 days after onset of clinical signs. This dog had tracheobronchitis, pulmonary edema and congestion, and severe bronchopneumonia. Similar to the 6 shelter dogs, there was epithelial cell necrosis and erosion of the airways and neutrophilic infiltrates in the tissues.<br>
EXAMPLE 8-REAL-T1ME RT-PCR AND VIRAL ISOLATION<br>
Lung tissues from the 7 dogs were analyzed by quantitative real-time RT-PCR assays that detect the M gene of influenza type A and the H3 gene of canine H3N8 influenza A virus. The lungs from all 7 dogs were positive for both the influenza A M gene and the canine influenza H3 gene (Table 8). After 3 passages in MDCK cells, influenza A subtype H3N8 virus was isolated from the lungs of a shelter dog that died after 3 days of pneumonia. This virus was named A/canine/Jacksonville/05 (H3N8) (canine/Jax/05). After 2 passages in embryonated chicken eggs, influenza A subtype H3N8 virus was recovered from the lungs of the pet dog that also died after 3 days of pneumonia. This virus was named A/canine/Miami//05 (H3N8) (canine/Miami/05).<br>
EXAMPLE 9-GENETIC ANALYSES OF THE CANINE INFLUENZA A H3N8 ISOLATES<br>
Sequence analyses of canine/Jax/05 and canine/Miami/05 revealed that their hemagglutinin (HA) genes were 98% identical to the canine/FL/04, canine/TX/04, and canine/Iowa/05 isolates recovered from the lungs of racing greyhounds that died of pneumonia during influenza outbreaks at tracks in 2004 and 2005 (Crawford, P.C. et a/., 2005; Yoon K-Y. et al, 2005). In addition, the HA genes of canine/Jax/05 and canine/Miami/05 were 98% identical to contemporary equine influenza viruses isolated after the year 2000. Phylogenetic comparisons of the I1A genes showed that the canine/Jax/05 and canine/Miami/05 viruses were clustered with the canine/FL/04, canine/TX/04, and canine/Iowa/05 greyhound isolates and contemporary equine isolates, forming a distinct group from the older equine viruses isolated in the early 1990's (Figure 4). Furthermore, the canine/Jax/05,<br>
canine/Miami/05, and canine/Iowa/05 isolates were more closely related to canine/Tx/04 than to either caniri£/FL/04 or canine/FL/03. The 2005 isolates formed a subgroup that appears to branch off from the earlier 2003\nd 2004 canine viruses with differences at approximately 10 parsimony-informative sites. These differences support the hypothesis that canine influenza virus is being transmitted horizontally from dog-to-dog as opposed to being reintroduced periodically from an outside source. The accumulation of mutations from 2003 to 2005 illustrates the ongoing process of adaptation that the virus must undergo after being transmitted to a new host, as is expected to have happened for the canine influenza viruses.<br>
EXAMPLE 10-AMINO ACID ANALYSES OF THE CANINE INFLUENZA A H3N8 ISOLATES.<br>
There were conserved amino acid substitutions in all 6 canine isolates that differentiated them from contemporary equine influenza viruses (Table 9). These conserved substitutions were II5M, N83S, W222L, 13281, and N483T. Phylogenetic comparisons of the mature HA protein showed that the canine/Jax/05, canine/Miami/05, and canine/Iowa/05 viruses formed a subgroup with the canine/TX/04 isolate (Figure 4). There were 3 amino acid changes (LI 18V, K.261N, and G479E) that differentiated this subgroup from the other canine viruses (Table 9). There were 2 amino acid changes (F79L and G218E) that differentiated the 2005 isolates from their canine/TX/04 root. Furthermore, the 2005 isolates from non-greyhound dogs, canine/Jax/05 and canine/Miami/05, differed from the canine/Iowa/05 greyhound isolate by one amino acid change, R492K. Finally, canine/Jax/05 differed from canine/Miami/05 at a single amino acid, S107P. In all other H3N8 equine and canine viruses, S is conserved at position 107 except for A/Equine/Jilin/1/89 which has a T (Guo Y. et al, 1992).<br>
EXAMPLE 11-ANT1GEN1C ANALYSES OF THE CANINE INFLUENZA A H3N8 ISOLATES<br>
Hemagglutination inhibition (HI) tests were performed using an antigen panel of older and contemporary equine influenza viruses and the canine influenza viruses, and serum collected in 2005 from horses and dogs that had been infected with influenza virus (Table 10). Serum from ferrets immunized against canine/FL/04 was also included in the analyses. The HI antibody liters in equine serum were 8 to 16-fold higher when tested with contemporary equine viruses compared to older isolates, but decreased by at least 4-fold when tested with the canine viruses. The canine serum was nonreactive with the older equine viruses, but the antibody liters increased 4-fold when tested with contemporary equine isolates and canine isolates. This was also observed for Ihe serum from ferrets immunized against canine influenza virus. These seroreactivily patterns demonstraled the antigenic similarity belween Ihe canine influenza viruses and conlemporary equine influenza viruses and were consislenl wilh Ihe phylogenelic analyses. The anlibody tilers in equine, canine, and ferret sera to the canine/Miami/05 isolate were similar to those for the 2003 and 2004 canine isolates. However, the tilers were 2 lo 4-fold lower for the canine/Jax/05 isolate. This suggests that canine/Jax/05 is antigenically distinct from the other canine isolates, which may in part be related to the single amino acid change at position 107 in the malure HA.<br>
(Table Removed)<br>
Underlined letter r represents nucleotide a or g and underlined letter k represents nucleotide g or t. Uppercase letters represent locked nucleic acid residues.<br><br>
(Table Removed)<br>
a Antibody liters were determined in a hemagglutination inhibition assay performed with serial<br>
dilutions of equine, canine, or ferret serum and the<br>
viruses listed in the antigen column.<br>
b Serum from ferrets immunized with canine/FL/04 virus.<br>
MATERIALS AND EXAMPLES METHODS FOR EXAMPLES 12-15<br>
Canine influenza virus inoculum.<br>
The virus inoculum was prepared by inoculation of Madin-Darby canine kidney (MDCK) epithelial cells with a stock of A/canine/FL/43/04 (H3N8) representing passage 3 of the original isolate previously described (Crawford et al, 2005). The inoculated MDCK cells in Dulbecco's Minimal Essential Media (DMEM) supplemented with 1 ug/mL TPCK-treated trypsin (Sigma-Aldrich Corp., St. Louis, MO) and antibiotics (gentamycin and ciprofloxacin) were grown in 250 cm2 flasks at 37°C in a humidified atmosphere containing 5% CO2. The cultures were observed daily for morphologic changes and harvested at 5 days post inoculation. The harvested cultures were clarified by centrifugation and the supernatants were stored at -80°C pending inoculation of<br>
dogs.  An aliquot of supernatant was used for determination of virus titer by the Reed and Muench method. The titer ^as 107 median tissue culture infectious doses (TCID50)of A/canine/Florida/43/04 (canine/FL/04) per mL.<br>
Experimental inoculation.<br>
Eight 4-month old colony bred mongrel dogs (Marshall BioResources, North Rose, NY) (4 males and 4 females) were used for the experimental inoculation study approved by the University of Florida Institutional Animal Care and Use Committee. The dogs' body weights ranged from 13 to 17 kg. The dogs were healthy based on physical examinations, baseline blood tests, and recording of body temperatures for 2 weeks prior to inoculation. All dogs were free from prior exposure to canine influenza virus based on serology tests performed on paired serum samples collected at the time of arrival into the facility and 2 weeks later. The dogs were anestheti/ed by intravenous injection of propofol (Diprivan®, Zeneca Pharmaceuticals, 0.4 mg/kg body weight to effect) for intubation with endotracheal tubes. Six dogs (3 males and 3 females) were each inoculated with 107 TCID5o of canine/FL/04 virus in 5 mL of sterile saline administered into the distal trachea through a small diameter rubber catheter inserted into the endotracheal rube. Two dogs (1 male and 1 female) were sham-inoculated with an equal volume of sterile saline. The sham-inoculated control dogs were housed in a different room from the virus-inoculated dogs and cared for by different personnel. Physical examinations and rectal temperature recordings were performed twice daily for 6 days post inoculation (p.i.).<br>
Pharyngeal and rectal swab collection.<br>
To monitor for virus shedding, oropharyngeal specimens were collected twice daily from each dog on days 0 to 6 p.i. using polyester swabs (Fisher Scientific International Inc., Pittsburgh, PA). The swabs were placed in 1 mL of sterile phosphate-buffered saline (PBS) containing 0.5% bovine serum albumin (BSA). Rectal swabs were collected from each dog daily from days 0 to 6. Swab extracts were prepared by clarification of the swab transport media by centrifugation. An aliquot of swab extract was tested immediately for influenza A virus nucleoprotein using the Directigen™ commercial immunoassay kit (BD, Franklin Lakes, NJ) according to the manufacturer's instructions. The remaining extract was stored at -80°C pending other virological assays.<br>
Postmortem examinations.<br>
On day 1 p.i., one sham-inoculated dog and one virus-inoculated dog were euthanatized by intravenous inoculation of Beuthanasia-D® solution (1 mL/5 kg body weight; Schering-Plough Animal Health Corp). One virus-inoculated dog was similarly euthanatized each day from days 2 to 5 p.i. On day 6 p.i., the remaining sham-inoculated and virus-inoculated dog were euthanatized. Complete postmortem examinations were performed by one of the investigators (WLC). Tissues were fixed in 10% neutral buffered formalin, embedded in paraffin, and 5-um sections were either stained with hematoxylin and eosin for histopathologic diagnosis or processed for immunohistochemistry as described below. Unfixed lung tissues were submitted to the Diagnostic Clinical Microbiology/Parasitology/ Serology Service at the University of Florida College of Veterinary Medicine for bacterial isolation and identification. The samples were cultured on nonselective media as well as media selective<br>
for Bordetella species (Regan-Lowe; Remel, Lenexa, KS) and Mycoplasma species (Remel). All cultures were held/for 21 days before reporting no growth. Unfixed tissues were also stored at -80°C pending virological<br>
Immunohistochemistry.<br>
Deparaffmized and rehydrated 5-um trachea and lung tissue sections were mounted on Bond-Rite™ slides (Richard-Allan Scientific, Kalamazoo, MI) and subsequently treated with proteinase K (DAKOCytomation Inc., Carpenteria, CA) followed by peroxidase blocking reagent (DAKO® EnVision™ Peroxidase Kit, DAKO Corp., Carpenteria, CA ). The sections were incubated with a 1:500 dilution of monoclonal antibody to influenza A H3 (Chemicon International, Inc., Ternecula, CA) for 2 hours at room temperature. Controls included incubation of the same sections with mouse IgG (1 mg/mL, Serotec, Inc. Raleigh, NC), and incubation of the monoclonal antibody with normal canine lung sections. Following treatment with the primary antibody, the sections were incubated with secondary immunoperoxidase and peroxidase substrate reagents (Dako® EnVision™ Peroxidase Kit, Dako Corp.) according to the manufacturer's instructions. The sections were counterstained with hematoxylin, treated with Clarifier #2 and Bluing Reagent (Richard-Allan Scientific, Kalamazoo, MI), dehydrated, and coverslips applied with Permount (ProSciTech, Queensland, Australia).<br>
RNA extraction from swabs and tissues.<br>
Lung and tracheal tissues from each dog were thawed and homogenized in minimum essential medium (MEM) supplemented with 0.5% bovine serum albumin (BSA) and antibiotics (gentamycin and ciprofloxacin) using a disposable tissue grinder (Kendall, Lifeline Medical Inc., Danbury, CT). Total RNA was extracted from the tissue homogenates as well as orpharyngeal and rectal swab extracts using a commercial kit (RNeasy® Mini Kit, Q1AGEN Inc., Valencia, CA) according to manufacturer's instructions and eluted in a final volume of 60 u,L of buffer.<br>
Real-time RT-PCR.<br>
A single-step quantitative real-time RT-PCR was performed on the total RNA using the QuantiTect* Probe RT-PCR Kit containing ROX as a passive reference dye (QIAGEN Inc., Valencia, CA) and a primer-probe set that targeted a highly conserved region of the matrix (M) gene of type A influenza virus (Payungporn S. et al, 2006a; Payungporn S. et al, 2006b). For each real-time RT-PCR reaction, 5 \\L of extracted total RNA were added to a reaction mixture containing 12.5 jaL of 2X QuantiTech® Probe RT-PCR Master Mix, 0.25 uL of QuantiTech® RT Mix, forward and reverse primers (0.4 jaM final concentration for each), probe (0.1 jaM final concentration) and RNase-free water in a final volume of 25 jiL. The TaqMan® GAPDH Control Reagents (Applied Biosystems, Foster City, CA) were used according to manufacturer's instructions for detection of GAPDH as an endogenous internal control for the presence of RNA extracted from the swab and tissue samples and as a normalization control,<br>
Quantitative one-step real-time RT-PCR was performed on the reaction mixtures in a MxSOOOP® QPCR Syste/m (Stratagene, La Jolla, CA). Cycling conditions included a reverse transcription step at 50°C for 30 minutes, an initial denaturation step at 95°C for 15 minutes to activate the HotStarTaq® DNA polymerase, and amplification for 40 cycles. Each amplification cycle included denaturation at 94°C for 15 seconds followed by annealing/extension at 60°C for 1 minute. The FAM (emission wavelength 518 nm) and VIC (emission wavelength 554 nm) fluorescent signals were recorded at the end of each cycle. The threshold cycle (Ct) was determined by setting the threshold fluorescence (dR) at 1000 in each individual experiment. The Mx3000P® version 2.0 software program (Stratagene, La Jolla, CA) was used for data acquisition and analysis. The positive control consisted of amplification of RNA extracted from A/canine/FL/242/03 (H3N8) virus. The results were normalized by dividing the M Ct value by the corresponding GAPDH Ct value for each sample.<br>
Virus re-isolation from tissues.<br>
Frozen lung and trachea tissues from virus-inoculated dogs were thawed and homogenized in 10 volumes of DMEM supplemented with 0.5% BSA and antibiotics. Solid debris was removed by centrifugation and supernatants were inoculated onto MDCK cells cultured in DMEM supplemented with 1 ug/mL TPCK-treated trypsin (Sigma-Aldrich Corp., St. Louis, MO) and antibiotics as described above. Cells were grown in 25 cm2 flasks at 37°C in a humidified atmosphere containing 5% CO2. The cultures were observed daily for morphologic changes and harvested at 5 days post inoculation. The harvested cultures were clarified by centrifugation and the supernatants inoculated onto fresh MDCK cells as described for the initial inoculation; two additional passages were performed for samples that did not show evidence of influenza virus by hemagglutination or RT-PCR. Hemagglutination activity in the clarified supernatants was determined using 0.5% turkey red blood cells as previously described (Crawford et al., 2005). RT-PCR was performed as described below.<br>
RT-PCR, nucleotide sequencing, and phylogenetic analyses.<br>
Viral RNA was extracted from MDCK supernatant using the QIAamp® Viral RNA Mini Kit (QIAGEN Inc., Valencia, CA) according to manufacturer's instructions. The viral RNA was reverse transcribed to cDNA using the QIAGEN® OneStep RT-PCR Kit (QIAGEN Inc., Valencia, CA) according to manufacturer's instructions. PCR amplification of the coding region of the 8 influenza viral genes in the cDNA was performed as previously described (Crawford et al., 2005), using universal gene-specific primer sets (primer sequences available on request). The resulting DNA amplicons were used as templates for automated sequencing in the ABI PRISM® 3100 automated DNA sequencer using cycle sequencing dye terminator chemistry (Applied Biosystems, Foster City, CA). Nucleotide sequences were analyzed using the Lasergene 6 Package® (DNASTAR, Inc., Madison, WI). The nucleotide sequences for viruses recovered from infected dogs were compared to the sequences of the virus in the inoculum to determine if any changes had occurred during replication in the respiratory tract.<br>
EXAMPLE 12-CLINICAL DISEASE<br>
All 6 virus-inoculated dogs developed a transient fever (rectal temperature &gt;39°C) for the first 2 days p.i., but n/bne exhibited respiratory signs such as cough or nasal discharge over the 6-day observation period. The sharrPinoculated dogs remained clinically healthy.<br>
EXAMPLE 13-VIRUS SHEDDING<br>
Influenza A nucleoprotein was detected in the oropharyngeal swab collected from one of the virus-inoculated dogs at 24 hours p.i. The oropharyngeal swabs collected from one dog at 72, 84, and 120 hours p.i., and another dog at 108, 120, and 132 hours p.i., were positive for virus by quantitative real-time RT-PCR (Table 11). The absolute number of influenza M gene copies per uL of swab extract increased with time from 3 to 6 days p.i. No virus was detected in the rectal swabs.<br>
EXAMPLE 14-POSTMORTEM EXAMINATIONS<br>
In contrast to the previous experimental infection using specific pathogen-free Beagles (Crawford et al, 2005), the virus-inoculated mongrel dogs had pneumonia as evidenced by gross and histological analyses of the lungs from days 1 to 6 p.i. In addition to pneumonia, the dogs had rhinitis, tracheitis, bronchitis, and bronchiolitis similar to that described in naturally infected dogs (Crawford et al., 2005). There was epithelial necrosis and erosion of the lining of the airways and bronchial glands with neutrophil and macrophage infiltration of the submucosal tissues (Figure 5, upper panels). Immunohistochemistry detected viral H3 antigen in the epithelial cells of bronchi, bronchioles, and bronchial glands (Figure 5, lower panels). No bacterial superinfection was present. The respiratory tissues from the 2 sham-inoculated dogs were normal.<br>
EXAMPLE 15-VIRUS REPLICATION IN TRACHEA AND LUNGS<br>
The trachea and lungs were positive for virus by quantitative real-time RT-PCR in all dogs from 1 to 6 days p.i. (Table 12). The absolute number of influenza M gene copies per µL of trachea homogenate increased from 1 to 5 days p.i., then decreased on day 6. The absolute number of M gene copies per uL of lung homogenate decreased from 1 to 6 days p.i. In general, the trachea contained ≥ one logio more virus than the lung on each of the 6 days p.i.<br><br><br>
(Table Removed)<br>
"Time that oropharyngeal swabs were collected from the dogs following inoculation<br>
with A/canine/FL/43/04 (H3N8) virus.<br>
b Normalization ratios were calculated by dividing the M (Ct) by the GAPDH (Ct)<br>
for each swab extract.<br>
cThe absolute number of matrix gene copies per uL of swab extract.<br><br>
(Table Removed)<br>
'Time that tissues were collected from the dogs following inoculation with A/canine/FL/43/04 (H3N8)<br>
'Normalization ratios were calculated by dividing the M (Ct) by the GAPDH (Ct) for each tissue homogenate. The absolute number of matrix gene copies per uL of tissue homogenate.<br>
MATERIALS AND EXAMPLES METHODS FOR EXAMPLE 16<br>
Virus strains<br>
Canine influenza virus strains as well as those of avian, equine and human origin (listed in Table 15) were propagated in embryonated eggs or MDCK cells and their infectivity was titrated by endpoint dilution in chicken embryos, or plaque assay. Rapid virus quantification was performed by hemagglutination assay using turkey red blood cell erythrocytes.<br>
Diagnostic specimens<br>
A Total of 60 canine's lung tissues collected from suspect cases of viral respiratory disease during the year of 2005<br>
were tested for the presence of canine influenza virus.<br>
RNA extraction from canine tissue samples<br>
Blocks of lung tissue weighing between 20 and 30 mg were homogenized in a disposable tissue grinder (Keridal). Total RNA was extracted using a commercial kit (RNeasy Mini Kit, Qiagen, Valencia, CA) and eluted in a final volume of 60 jo.L, following the manufacturer's recommendations.<br>
Primers and probes design<br>
Multiple sequence alignments of the H3 and M genes from various subtypes and from diverse species were performed using the CLUSTAL X program (Version 1.8). Matrix (M) primers and probe were selected from the conserved regions of over the known sequences corresponding to different subtypes of influenza A virus, whereas the H3 hemagglutinin gene-specific primers and probe set were selected to specifically match equine and canine influenza A virus genes and mismatch the homologous avian and human genes (Table 13). Primer design software (OLIGOS Version 9.1) and the web based analysis tools provided by EXIQON (http://lnatools.com) was used for Tin calculation and prediction of secondary structure as well as self hybridization. A conserved region of an 18S rRNA gene was used as endogenous internal control for the presence of RNA extracted from canine tissue sample. The Pre-Developed TaqMan® Assay Reagents for Eukaryotic 18S rRNA (VIC/TAMRA) (Applied Biosystems) was used for the real-time detection of 18S rRNA in tissue samples.<br>
Real-time RT-PCR condition<br>
A single-step real-time RT-PCR was performed by using the Quantitect Probe RT-PCR Kit containing ROX as a passive reference dye (Qiagen, Valencia, CA). In each real-time RT-PCR reaction, 5 µL of RNA sample were used as a template to combine with a reaction mixture containing 10 µL of 2X QuantiTech Probe RT-PCR Master Mix, 0.2 µL of QuantiTech RT Mix, primers (0.4 µM final cone, for H3 gene or 0.6 (J.M final cone, for M gene), probe (0.1 µM final cone, for H3 gene or 0.2 µM final cone, for M gene) and RNase-free water in a final volume of 20 µL. One-step real-time RT-PCR was performed in the Mx3005P Real-Time QPCR System (Stratagene). Cycling conditions included a reverse transcription step at 50°C for 30 minutes. After an initial denaturation step at 95°C for 15 minutes in order to activate the HotStarTaq DNA polymerase, amplification was performed during 40 cycles including denaturation (94°C for 15 seconds) and annealing/extension (60°C for 30 seconds). The FAM (emission wavelength 516 nm for H3 and M detection) and VIC (emission wavelength 555 nm for 18S rRNA detection) fluorescent signals were obtained once per cycle at the end of the extension step. Data acquisition and analysis of the real-time PCR assay were performed using the Mx3005P software version 2.02 (Stratagene).<br>
Specificity of H3 primers/ probe set for canine influenza (H3N8) and universality of M primers/probe set for type A influenza virus<br>
In order to test the specificity of each primers/probe set, RNA extracted from several known subtypes of influenza A viruses were used as a template in the real-time RT-PCR assay (Table 15).<br>
RNA standard for determination of the real-timer RT-PCR performance<br>
The genes of canine influenza A virus (A/canine/Florida/242/2003(H3N8)) were used to generate the PCR amplffcons for H3 (nt 1-487) and M (nt 1-276) by using primers linked with T7 promoter (Table 13). Then the purified PCR amplicons of H3 and M genes were used as templates for in vitro transcription by using Riboprobe in vitro Transcription System-T7 (Promega) following the manufacturer's recommendations. The concentration of the transcribed RNAs was calculated by measuring absorbance at 260 nm. The RNAs were then serially diluted 10-fold, ranging from 108 to 10 copies/ ^L to perform sensitivity tests. Moreover, a standard curve was generated by plotting the log of initial RNA template concentrations (copies/ (JL) against the threshold cycle (Ct) obtained from each dilution in order to determine the overall performance of real-time RT-PCR.<br>
Comparative sensitivity tests between real-time RT-PCR and Directigen Flu A test kit<br>
Stock viruses of two viral strains including A/Wyoming/3/2003 (H3N2) at 10667 EID50/mL (HA=64) and A/canine/Florida/242/2003(H3N8) at 10717 EID50/mL (HA=16) were used for the detection threshold assay. Logarithmic dilution of specimens in phosphate-buffered saline (PBS) (125 jaL) were used in a rapid influenza A antigen detection kit, Directigen Flu A, (Becton, Dickinson and Company) following the manufacturer's instructions, Each Directigen Flu A test device has an H1N1 influenza antigen spot in the center of the membrane which develops as a purple dot and indicates the integrity of the test, which is based on a monoclonal antibody to the nucleoprotein (NP). The development of a purple triangle surrounding the dot is indicative of the presence of influenza NP in the tested specimen. The intensity of the purple signal from the triangle was scored as + (outline of triangle), ++ (lightly colored triangle), +++ (dark-purple triangle) and ++++ (very dark-purple triangle). Viral RNA was extracted 125 JJ.L aliquots of each virus dilution by using QIAamp Viral RNA Mini Kit (Qiagen, Valencia, CA) and eluting in a final volume of 50 uL. A volume of 5 uL of the extracted viral RNAs were tested by real-time RT-PCR for comparative sensitivity test with Directigen Flu A kit.<br>
EXAMPLE 16<br>
The real-time RT-PCR assay for canine influenza relies on information from three molecular probes which target 18S rRNA from host cell was well as M and H3 from the influenza A virus genome (Table 14). Amplification of the host gene is a reporter of specimen quality and integrity. Clinical, necropsy or laboratory samples containing canine influenza (H3N8) virus are expected to yield amplification signal with the three probes. Specimens yielding amplification signal with M and 18S rRNA probes but negative for H3 would be indicative of an influenza virus subtype H3 from human, swine or avian origin or from non-H3 subtypes. These rare cases could be resolved by RT-PCR using HA universal primers to generate amplicon cDNA that can be analyzed by sequencing. Properly collected and handled specimens lacking influenza A virus yield 18S rRNA amplification signal only. Situations in which only the 18S rRNA probe and the H3 probes yield amplification signal are indicative of faulty technique, unless proven otherwise; either a false negative with M probes or false positive for 113 need to be demostrated. Finally, specimens failing to yield amplification signals with the three probes are<br>
indicative of defective sample collection, degradation, faulty RNA extraction or the presence of inhibitors the polymjerases used in PCR.<br>
•-. in order to test the specificity of the H3 primers/probe set for canine influenza A virus (H3N8) and the universality of M primers/probe set for type A influenza, several subtypes of influenza A viruses were tested by real-time RT-PCR. The results show that H3 primers/probe set yielded a positive amplification signal only with canine influenza (H3N8). No significant false positive or non-specific amplification signals were observed in other subtypes or human H3 strains. The M primers/probe set yielded positive amplification signal with all of the strains tested (Table 15). These results indicated that H3 primers/probe specifically detects canine influenza A virus (H3N8) whereas M primers/probe detect multiple subtypes of type A influenza viruses.<br>
The performance of real-time RT-PCR assays was evaluated by endpoint dilution of M and H3 in vitro transcribed RNAs. As expected, the threshold cycle (Ct) increased in direct correlation with the dilution of the RNA standards. The fluorescent signals can be detected at RNA standard dilutions of M and H3 as low as 103 and I02 copies/^L, respectively (Figure 6A and 6B). The standard curves of M and H3 genes were constructed by plotting the log of starting RNA concentrations against the threshold cycle (Ct) obtained from each dilution (Figure 6C and 6D). The slope of the standard curve is used to determine the PCR reaction efficiency, which is theoretically exponential; 100% amplification efficiency would imply doubling of amplicon cocentration each cycle. The standard curves with a slope between approximately -3.1 and -3.6 are typically acceptable for most applications requiring accurate quantification (90-110 % reaction efficiency). An Rsq value is the fit of all data to the standard curve plot. If all the data lie perfectly on the line, the Rsq will be 1.00. As the data fall further from the line, the Rsq decreases. An Rsq value &gt; 0.985 is acceptable for most assays. The M standard curve revealed a slope of -3.576 (efficiency= 90.4 %) and Rsq= 1.00 whereas H3 standard curve yielded a slope of -3.423 (efficiency= 95.9%) and Rsq= 0.999. These values indicate satisfactory amplification efficiency and overall performance of the real-time RT-PCR assays. We attribute the lower efficiency and sensitivity of M primers/probe set as compared to H3 primers/probe set to the N-fold degeneracy of M primer sequences required to ensure broad coverage of M gene sequences variability across viruses of multiple subtypes, hosts and lineages.<br>
The sensitivity of real-time RT-PCR assay was also compared with the commercial rapid antigen detection assay (Directigen Flu A). Logarithmic dilutions of A/Wyoming/3/2003 (H3N2) and A/canine/Florida/242/2003(H3N8) were analyzed with Directigen Flu A and by real-time RT-PCR. The results of Directigen Flu A showed that the sensitivities against both viral strains are approximately 100-fold dilution from the stock viruses used in these experiments (Figure 7). The signals (purple color) generated by the canine virus (A/canine/Florida/242/2003: 106x PFU/ml) samples were much weaker than those found in human virus (A/Wyoming/3/2003: 107x PFU/ml), in agreement with the lower virus concentration in these samples. Alternatively, lower signal for canine influenza could be attributed to the molecular specificity of monoclonal antibodies against the NP; i.e. poor conservation of the amino acids within the NP epitope of canine influenza A viruses.<br>
Real-time RT-PCR of the M gene yielded Ct values above threshold with virus 10 and 30 PFU equivalents per reaction of A/canine/Florida/242/2003 and A/Wyoming/3/2003, respectively (Table 16). The differences<br>
between the sensitivity value of 2 viral strains because the differences in the original viral liters. The H3 gene detection comparison between canine and human influenza viruses was not performed because the H3 primers/probe in our realtime RT-PCR assay amplifies exclusively canine influenza A virus. RT-PCR was 105 times more sensitive than the rapid antigen detection kit.<br>
To evaluate the performance of the RT-PCR test in necropsy specimens from dogs with acute respiratory disease, sixty canine lung tissue samples submitted during the year of 2005 were tested for the presence of canine influenza A virus by real-time RT-PCR. A total of 12 out of 60 samples (20%) were positive with both M and H3 genes whereas the remaining 48 samples yielded negative result for both M and H3 gene. Virus isolation attempts were conducted by egg and MDCK. cell inoculation, to evaluate the specificity of the realtime assay; 2 out 12 samples that were positive for canine influenza by RT-PCR yielded canine influenza virus (data not shown, manuscript in preparation). Although all of the tissues were collected from dogs with a history of severe respiratory disease, most of the samples yielded no canine influenza virus by either realtime PCR or conventional isolation, suggesting a high incidence of other respiratory pathogens such as Bordetella bronchiseptica, canine distemper or parainfluenza virus. The single step real-time RT-PCR assay herein provides a rapid, sensitive and cost-effective approach for canine influenza A virus (H3N8) detection. Rapid laboratory diagnosis of canine influenza A virus (H3N8) infections in the early stage of the disease can yield information relevant to clinical patient and facility management.<br><br>
(Table Removed)<br>
)te: Uppercases = LNA (Locked Nucleic Acid) residues, r = a or g, k= g or t, underline= T7 promoter sequence<br>
(Table Removed)<br>
* Note that subtypes of clinical samples were confirmed by nucleotide sequencing.<br>
(Table Removed)<br>
Table 19. Amino acid differences between PB2 proteins of H3N8 equine and canine influenza<br>
viruses<br>
(Table Removed)<br>
* Based on available genes of viruses isolated between 1963 and 1998.<br>
(Table Removed)<br>
* Based on available genes of viruses isolated between 1963 and 1998.<br>
It should be understood that the examples and embodiments described herein are for illustrative purposes only ijlid that various modifications or changes in light thereof will be suggested to persons skilled in the art and arc t0N)e included within the spirit and purview of this application.<br>
REFERENCES<br>
U.S. Patent No. 5,106,739<br>
U.S. Patent No. 5,034,322<br>
U.S. Patent No. 6,455,760<br>
U.S. Patent No. 6,696,623<br>
U.S. Patent No. 4,683,202<br>
U.S. Patent No. 4,683,195<br>
U.S. Patent No. 4,800,159<br>
U.S. Patent No. 4,965,188<br>
U.S. Patent No. 5,994,056<br>
U.S. Patent No. 6,814,934<br>
Published U.S. Application No. 20040078841<br>
Published U.S. Application No. 20040067506<br>
Published U.S. Application No. 20040019934<br>
Published U.S. Application No. 20030177536<br>
Published U.S. Application No. 20030084486<br>
Published U.S. Application No. 20040123349<br>
Greyhound Daily News, 1/28/99. National Greyhound Association (NGA), Abilene, Kansas. http://www.NGAgreyhounds.com.<br>
Personal communication. Dr. William Duggar, veterinarian at Palm Beach Kennel Club, West Palm Beach, Florida.<br>
Altschul, S. F. et al. (1990) "Basic Local Alignment Search Tool" J. Mol. Biol. 215:402-410.<br>
Altschul, S. F. et al. (1997) "Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs" Nucl. Acids Res. 25:3389-3402.<br>
An, G. (1987) "Binary Ti vectors for plant transformation and promoter analysis" Methods Enzymol. 153:292-305.<br>
Beltz, G. A., Jacobs, K. A., Eickbush, T. H., Cherbas, P. T., Kafatos, F. C. (1983) "Isolation of multigene families and determination of homologies by filter hybridization methods" Methods of Enzymology, R. Wu, L. Grossman and K. Moldave [eds.] Academic Press, New York 100:266-285.<br>
Burleson, F. et al. (1992) Virology: A Laboratory Manual (Academic Press).<br>
Byarsy,N.E., A.C. Allison (1987) "Adjuvant formulation for use in vaccines to elicit both cell-mediated and humoral immunity" Vaccine 5:223-228.<br>
Crawford, P.C. et al. (2005) "Transmission of equine influenza virus to dogs" Science 310:482-485.<br>
Chang, C.P. et al. (1976) "Influenza virus isolations from dogs during a human epidemic in Taiwan" Int J Zoonoses 3:61-64.<br>
Dacso, C.C. el al. (1984) "Sporadic occurrence of zoonotic swine influenza virus infections" J Clin Microbiol 20:833-835.<br>
de Boer, H. A., Comstock, L. J., Yasser, M. (1983) "The tac promoter: a functional hybrid derived from the trp and lac promoters" Proc. Nad. Acad. Sci. USA 80(l):21-25.<br>
Felsenstein, J. (1989) Cladistics 5:164.<br>
Fields el al. (1946) Fields Virology, 3rd ed., Lippincott-Raven publishers.<br>
Ford, R.B., Vaden, S.L. (1998) "Canine infectious tracheobronchitis" In Infectious Diseases of the Dog and Cat, 2nd edition, C.E. Greene, editor, W.B. Saunders Co., Philadelphia, PA, pp. 33-38.<br>
Fouchier et al, (2000) Journal of Clinical Microbiology 38 (11):4096-4101.<br>
Good, X. et al. (1994) "Reduced ethylene synthesis by transgenic tomatoes expressing S-adenosylmethionine hydrolase" Plant Molec. Biol. 26:781-790.<br>
Guan, Y. el al. (2004) "H5N1 influenza: a protean pandemic threat" Proc Natl Acad Sci USA 101:8156-8161.<br>
Guo, Y. et al. (1992) "Characterization of a new avian-like influenza A virus from horses in China" Virology<br>
188:245-255.<br>
Houser, R.E. et al. (1980) "Evidence of prior infection with influenza A/Texas/77 (H3N2) virus in dogs with clinical parainfluenza" CanJComp Med44:396-402.<br>
Karasin, A.I. et al. (2000) "Isolation and characterization of H4N6 avian influenza viruses from pigs with pneumonia in Canada" J Virol 74:9322-9327.<br>
Karlin S. and Altschul, S. F. (1990) "Methods for Assessing the Statistical Significance of Molecular Sequence Features by Using General Scoring Schemes" Proc. Natl. Acad. Sci. USA 87:2264-2268.<br>
Karlin S. and Altschul, S. F. (1993) "Applications and Statistics for Multiple High-Scoring Segments in Molecular Sequences"/Voc. Natl. Acad. Sci. USA 90:5873-5877.<br>
Kawaoka, Y. et al., (1989) "Avian-to-human transmission of the PB1 gene of influenza A viruses in the 1957 and 1968 pandemics" J Virol 63:4603-4608.<br>
Keawcharoen, J. et al. (2004) "Avian influenza H5N1 in tigers and leopards" Emerg Infect Dis 10:2189-2191.<br>
Kendal, A. P. et al. (1982) In Concepts and Procedures for Laboratory-based Influenza Surveillance. A. P. Kendal, M. S. Pereira, J. J. Skehel, Eds. (U.S. Department of Health and Human Services, Centers for Disease Control and Prevention and Pan-American Health Organization, Atlanta, GA, United States) pp. B17-B35.<br>
Kilbourne, E.D. et al. (1975) "Demonstration of antibodies to both hemagglutinin and neuraminidase antigens of H3N2 influenza A virus in domestic dogs" Intervirology 6:315-3 18.<br>
Kimura, K. et al. (1998) "Fatal case of swine influenza virus in an immunocompetent host" Mayo Clin Proc<br>
. 73:243-245.<br>
Klimoy, A. I. et al. (1992a) "Sequence changes in the live attenuated, cold-adapted variants of influenza A/Leningrad/I34/57 (H2N2) virus" Virology 186:795-797.<br>
Klimov A. el al. (1992b) "Subtype H7 influenza viruses: comparative antigenic and molecular analysis of the HA-, M-, and NS-genes" Arch Virol. 122:143-161.<br>
Kovacova, A. et al. (2002) "Sequence similarities and evolutionary relationships of influenza virus A hemagglutinins" Virus Genes 24:57-63.<br>
Kuiken, T. et al. (2004) "Avian H5N1 influenza in cats" Science 306:241.<br>
Kumar, S. et al. (2004) "MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment" Brief Bioinform 5:150-163.<br>
Lee, L.G. et al. (1993) "Allelic discrimination by nick-translation PCR with fluorogenic probes" Nucleic Acids Res. 21(16):3761-3766.<br>
Lewin. B. (1985) Genes 11. John Wiley &amp; Sons, Inc., p. 96.<br>
Lipakn, A.S. el al. (2004) "Influenza: emergence and control" J Virol 78:8951-8959.<br>
Livak, K. J. et al. (1995) "Oligonucleotides with fluorescent dyes at opposite ends provide a quenched probe system useful for detecting PCR product and nucleic acid hybridization" PCR Methods Appl. 4(6):357-362.<br>
Maniatis, T., E.F. Fritsch, J. Sambrook (1982) "Nuclease 5o/31" Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.<br>
Matrosovich, M. et al. (2000) "Early alterations of the receptor-binding properties of HI, H2, and H3 avian influenza virus hemagglutinins after their introduction into mammals" J Virol 74:8502-8512.<br>
Maertzdorf et al., (2004) Clin Microbiol. 42(3):981-986.<br>
Merrifield, R.B. (1963) "Solid Phase Peptide Synthesis. I. The Synthesis of a Tetrapeptide" J. Amer. Chem. Soc. 85:2149-2154.<br>
Nikitin, A. et al. (1972) "Epidemiological studies of A-Hong Kong-68 virus infection in dogs" Bull World Health Organ 47:471 -479.<br>
Nobusawa, E. et al. (1991) "Comparison of complete amino acid sequences and receptor-binding properties among 13 serotypes of hemagglutinins of influenza A viruses" Virology 182:475-485.<br>
Patriarca, P. A. et al. (1984) "Lack of significant person-to person spread of swine influenza-like virus following fatal infection in an immunocompromised child"/1m JEpidemiot 119:152-158.<br>
Payungporn S. et al. (2006a) "Detection of canine influenza A virus (H3N8) RNA by real-time RT-PCR" (in preparation for Journal of Clinical Microbiology).<br>
Payungporn S, et al. (2006b) "Isolation and characterization of influenza A subtype H3N8 viruses from dogs with respiratory disease in a shelter and veterinary clinic in Florida" (in preparation for Emerging Infectious<br>
Diseases).<br>
Peiris, M. et al. (1999) "Human infection with influenza H9N2" Lancet 354:916-917.<br>
Peiris, J.S. et al. (2004) "Re-emergence of fatal human influenza A subtype H5N1 disease" Lancet 363:617-619.<br>
Posneitt, D. N. et al. (1988) "A Novel Method for Producing Anti-peptide Antibodies" J. Biol. Chem. 263(4): 1719-<br>
1725.<br>
Putnam, Bob (February 10, 1999) "Two illnesses seen in death of dogs" St. Petersburg Times.<br>
Reid, A.H. et al. (2004) "Evidence of an absence: the genetic origins of the 1918 pandemic influenza virus" Nat Rev Microbiol 2:909-914.<br>
Rowe, T. et al. (1999) "Detection of antibody to avian influenza A (H5N1) virus in human serum by using a combination of serologic assays" JClin Microbiol 37: 937-943.<br>
Saiki, R. (1985) "Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia" Science 230:1350-1354.<br>
Sambrook, J. et al. (1989) "Plasmid Vectors" In: Molecular Cloning: A Laboratory Manual, 2d Edition, pp. 1.82-1.104. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York.<br>
Subbarao, K. et al. (1998) "Characterization of an avian influenza A (H5N1) virus isolated from a child with a fatal respiratory illness" Science 279:393-396.<br>
Suzuki, Y. et al. (2000) "Sialic acid species as a determinant of the host range of influenza A viruses" J Virol 74:11825-11831.<br>
Tam, J. P. (1988) "Synthetic Peptide Vaccine Design: Synthesis and Properties of a High-Density Multiple Antigenic Peptide System" PNAS USA 85(15):5409-5413.<br>
Top, Jr., F.H. et al. (1977) "Swine influenza A at Fort Dix, New Jersey (January-February 1976). IV. Summary and speculation" J Infect Dis 136 Suppl:S376-S380.<br>
Vines. A. et al. (1998) "The role of influenza A virus hemagglutinin residues 226 and 228 in receptor specificity and host range restriction" J Virol 72:7626-7631.<br>
Wagner, R. et al. (2002) "N-Glycans attached to the stem domain of haemagglutinin efficiently regulate influenza A virus replication" J Gen Virol 83:601-609.<br>
Webby, R. et al. (2004) "Molecular constraints to interspecies transmission of viral pathogens" Nat Med 10:S77-S81.<br>
Webster, R.G. (1998) "Influenza: an emerging disease" Emerg Infect Dis. 4:436-441.<br>
Webster, R.G. et al. (1992) "Evolution and ecology of influenza A viruses" Microbiol Rev. 56 :152-179.<br>
Weis, W. et al. (1988) "Structure of the influenza virus haemagglutinin complexed with its receptor, sialic acid"<br>
Nature 333:426-431.<br>
Womble, D.D. (2000) "GCG: The Wisconsin Package of sequence analysis programs" Methods Mol Biol 132:3-22.<br>
Xu, D., McElroy, D., Thornburg, R. W., Wu, R. et al. (1993) "Systemic induction of a potato pin2 promoter by wounding, methyl jasmonate, and abscisic acid in transgenic rice plants" Plant Molecular Biology 22:573-588.<br>
Yang. T. T. et al. (1996) "Optimized Codon Usage and Chromophore Mutations Provide Enhanced Sensitivity<br>
with the Green Fluorescent Protein" Nucleic Acid Research 24(22):4592-4593. Yoon K-Y. et al. (2005) "Influenza virus infection in racing greyhounds" Emerg Infect Dis. 11:1974-1975.<br><br><br>
CLAIMS<br>
We claim:<br>
1.	An isolated influenza virus that is capable of infecting a canid animal.<br>
2.	The influenza virus according to claim 1, wherein said influenza virus has an HA serotype of H3 or H5.<br>
3.	The influenza virus according to claim 1, wherein said influenza virus has the serotype H3N8.<br><br>
4.	The influenza virus according to claim 1, wherein said influenza virus comprises a polynucleotide<br>
which encodes a polypeptide having the amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14,<br>
16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a<br>
functional and/or immunogenic fragment thereof, or said polynucleotide encodes a polypeptide having 99% or<br>
greater sequence identity with the amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18,<br>
20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78.<br>
5.	The influenza virus according to claim 1, wherein said influenza virus comprises a polynucleotide<br>
having the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21,23,25,27,29,<br>
31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, or said polynucleotide has 98% or greater<br>
sequence identity with the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21,<br>
23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77.<br>
6.	The influenza virus according to claim 1, wherein said influenza virus is the viral isolate designated as<br>
A/canine/Florida/43/04 or the viral isolate designated as A/canine/Florida/242/2003.<br>
7.	The influenza virus according to claim 1, wherein said influenza virus is the viral isolate designated as<br>
canine/Jax/05 or the viral isolate designated as canine/Miami/05.<br>
8.	An isolated polynucleotide that comprises all or part of a genomic segment or gene of an influenza virus<br>
of any of claims 1 to 7.<br>
9.	The polynucleotide according to claim 8, wherein said polynucleotide comprises all or part of an<br>
influenza gene selected from the group consisting of a neuraminidase gene, a nucleoprotein gene, a matrix protein<br>
gene, a polymerase basic protein gene, a polymerase acidic protein gene, a non-structural protein gene, and a<br>
hemagglutinin gene.<br><br>
10. The polynucleotide according to claim 8, wherein said influenza virus comprises a polynucleotide whicfy encodes a polypeptide having the amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16. 1^.20, 22. 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic fragment thereof, or said polynucleotide encodes a polypeptide having 99% or greater sequence identity with the amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78.<br>
11. The polynucleotide according to claim 8, wherein said influenza virus comprises a polynucleotide having the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31. 47, 49. 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77, or said polynucleotide has 98% or greater sequence identity with the nucleotide sequence shown in any of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21,<br>
23, 25, 27, 29, 31, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, or 77.<br>
12.	A polynucleotide expression construct comprising a polynucleotide of any of claims 8 to 11.<br>
13.	An oligonucleotide that can hybridize to a coding or a non-coding sequence of a polynucleotide of any<br>
of claims 8 to 11.<br><br>
14.	The oligonucleotide according to claim 13, wherein said oligonucleotide hybridizes to said sequence<br>
under high stringency conditions.<br>
15.	The oligonucleotide according to claim 13, wherein said oligonucleotide is 10 to 500 nucleotides, 10<br>
to 20 nucleotides, 21 to 30 nucleotides, 31 to 40 nucleotides, 41 to 50 nucleotides, 51 to 60 nucleotides, 61 to 70<br>
nucleotides, 71 to 80 nucleotides, 81 to 90 nucleotides, or 91 to 100 nucleotides in length.<br>
16.	The oligonucleotide according to claim 13, wherein said oligonucleotide comprises a detectable label<br>
or a reporter molecule.<br>
17.	The oligonucleotide according to claim 13, wherein said oligonucleotide has from about 90 percent to<br>
100 percent nucleotide sequence identity with a polynucleotide of claim 8.<br>
18.	The oligonucleotide according to claim 13, wherein said oligonucleotide comprises a nucleotide<br>
sequence shown in any of SEQ ID NOs: 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 79, 80, 81, 82, 83, 84, 85, 86,<br>
87, or 88.<br>
19.	An isolated polypeptide encoded by a polynucleotide of any of claims 8 to 11.<br>
20.	The polypeptide according to claim 19, wherein said polypeptide is selected from the group consisting<br>
of a^hpmagglutinin protein, a neuraminidase protein, a nucleoprotein, a matrix protein, a polymerase basic protein,<br>
a poljtijierase acidic protein, and a non-structural protein.<br>
21.	A composition comprising an immunogen of an influenza virus of any of claims 1 to 7, wherein said<br>
immunogen is capable of inducing an immune response against an influenza virus that is capable of infecting a<br>
canid animal.<br>
22.	The composition according to claim 21, wherein said immunogen comprises cell-free whole virus, or a<br>
portion thereof; a viral polynucleotide; a viral protein; a viral polypeptide or peptide; a virus infected cell; a<br>
recombinant viral vector based construct; a reassortant virus; or naked nucleic acid of said virus.<br><br>
23.	The composition according to claim 22, wherein said viral protein, polypeptide, or peptide comprises<br>
an ammo acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33,<br>
34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic fragment<br>
thereof, or said polynucleotide encodes a polypeptide having 99% or greater sequence identity with the amino acid<br>
sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 33, 34, 48, 50, 52,<br>
54. 56. 58. 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78.<br>
24.	The composition according to claim 22, wherein said viral polynucleotide encodes a polypeptide<br>
comprising an amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28,<br>
30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or immunogenic<br>
fragment thereof.<br>
25.	A method for inducing an immune response in an animal against an influenza virus of any of claims 1<br>
to 7, said method comprising administering to said animal an effective amount of a composition according to any<br>
of claims 21, 22, 23, or 24.<br>
26.	The method according to claim 25, wherein said composition comprises a viral protein, polypeptide, or<br>
peptide comprising an amino acid sequence shown in any of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24,<br>
26, 28, 30, 32, 33, 34, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, or 78, or a functional and/or<br>
immunogenic fragment thereof.<br>
27.	The method according to claim 25, wherein said immune response is a protective immune response<br>
that prevents or inhibits infection of said animal by an influenza virus.<br>
28.	The method according to claim 25, wherein said influenza virus has an HA serotype selected from H3<br>
and Hj&gt;.<br>
29.	The method according to claim 25, wherein said influenza virus has the serotype H3N8.<br>
30.	The method according to claim 25, wherein said composition further comprises an adjuvant.<br>
31.	The method according to claim 25, wherein said animal is a canid.<br>
32.	The method according to claim 31, wherein said canid animal is a domesticated dog.<br>
33.	The method according to claim 25, wherein said composition is administered parenterally.<br>
34.	The method according to claim 33, wherein said composition is administered subcutaneously,<br>
intraperitoneally, or intramuscularly.<br>
35.	The method according to claim 25, wherein said composition is administered nasally or orally.<br>
36.	An isolated antibody that binds specifically to an epitope of a polypeptide of any of claims 19 or 20.<br>
37.	The antibody according to claim 36, wherein said antibody is a monoclonal antibody.<br>
38.	The antibody according to claim 36, wherein said antibody binds to a hemagglutinin protein.<br>
39.	The antibody according to claim 38, wherein said hemagglutinin protein is an H3 or an H5<br>
hemagglutinin protein.<br>
40.	The antibody according to claim 38, wherein said hemagglutinin protein comprises an amino acid<br>
sequence shown in SEQ ID NO: 16, SEQ ID NO: 32, SEQ ID NO: 62, or SEQ ID NO: 78.<br>
41.	A cell comprising an influenza virus of any of claims 1 to 7 or a polynucleotide of claim 8.<br>
42.	The cell according to claim 41, wherein said cell is a canid animal cell.<br>
43.	A method for detection of an influenza virus of that is capable of infecting a canid animal, said method<br>
comprising obtaining a biological sample from an animal and screening said sample for the presence of an<br>
antibody or a ligand that binds to a protein, polypeptide, or peptide of an influenza virus of any of claims 1 to 7.<br>
44.	A method for detection of an influenza virus that is capable of infecting a canid animal, said method<br>
comprising obtaining a nucleic acid sample from an animal, and contacting said nucleic acid sample with an<br>
oligonucleotide according to any of claims 13 to 18, and determining if said oligonucleotide hybridizes under<br>
stringent conditions to said nucleic acid.<br>
45.	A method for screening for a compound that possesses antiviral activity against an influenza virus that<br>
is capable of infecting a canid animal, said method comprising contacting a cell infected with or comprising a virus<br>
of any of claims 1 to 7 with a test compound and determining whether said test compound inhibits activity of said<br>
virus in said cell.<br>
46.	A reassortant virus comprising at least one gene, genomic segment, or a polynucleotide of an influenza<br>
virus of claim 1.<br>
47.	An   isolated  virus that  comprises  one  or  more  polynucleotides  wherein   said  one  or  more<br>
polynucleotides encode a canine influenza virus protein selected from the group consisting of HA, NA, NP, MA,<br>
FBI, PB2, PA, and NS, or a functional or immunogenic fragment thereof.<br>
48.	The virus according to claim 47, wherein said one or more polynucleotides comprise a nucleotide<br>
sequence selected from the group consisting of SEQ ID NOs: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31,<br>
47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, and 77.<br>
49.	The virus according to claim 47, wherein said HA protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 16, 32, 62, and 78.<br>
50.	The virus according to claim 47, wherein said NA protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 12, 28, 58, and 74.<br>
51.	The virus according to claim 47, wherein said NP protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 10, 26, 56, and 72.<br>
52.	The virus according to claim 47, wherein said MA protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 14, 30, 60, and 76.<br>
53.	The virus according to claim 47, wherein said PB1 protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 4, 20, 50, and 66.<br>
54.	The virus according to claim 47, wherein said PB2 protein has an amino acid sequence selected from<br>
the g?
55.	The virus according to claim 47, wherein said PA protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 6, 22, 52, and 68.<br>
56.	The virus according to claim 47, wherein said NS protein has an amino acid sequence selected from<br>
the group consisting of SEQ ID NOs: 8, 24, 54, and 70.<br>
57.	A method for detection of an influenza virus that is capable of infecting a canid animal, said method<br>
comprising obtaining a sample containing nucleic acid from an animal, and contacting said nucleic acid sample<br>
with a forward and reverse oligonucleotide primer that binds specifically to a polynucleotide or gene sequence of<br>
said virus under conditions sufficient for generation of an amplification product if the nucleic acid contains a<br>
sequence to which said primers bind; and detecting said amplification product.<br>
58.	The method according to claim 57, wherein said oligonucleotide primer is an oligonucleotide<br>
according to any of claims 13 to 18.<br>
59.	An isolated influenza virus substantially as herein described with<br>
reference to the foregoing description, examples, tables and the<br>
accompanying drawings.<br>
60.	The   polynucleotide   substantially   as   herein   described   with<br>
reference to the foregoing description, examples, tables and the<br>
accompanying drawings.<br>
61.	An   oligonucleotide   substantially   as   herein   described   with<br>
reference to the foregoing description, examples, tables and the<br>
accompanying drawings.<br>
62.	A composition substantially as herein described with reference to<br>
the    foregoing    description,     examples,     tables    and    the<br>
accompanying drawings.<br>
63.	A method substantially as herein described with reference to the<br>
foregoing description, examples, tables and the accompanying<br>
drawings.<br>
64.	An   isolated   antibody  substantially  as   herein  described  with<br>
reference to the foregoing description, examples, tables and the<br>
accompanying drawings.<br>
65.	A cell substantially as herein described with reference to the<br>
foregoing description, examples, tables and the accompanying<br>
drawings.<br>
66.	A   reassortant   virus   substantially   as   herein   described   with<br>
reference to the foregoing description, examples, tables and the<br>
accompanying drawings.<br>
67. An isolated virus substantially as herein described with reference to the foregoing description, examples, tables and the accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="263222-method-and-radio-communication-device-for-the-transmission-efficient-editing-of-multimedia-news.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263224-inline-intrusion-detection-using-a-single-physical-port.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263223</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>8115/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>15-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Oct-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CORNELL RESEARCH FOUNDATION, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>20 THORNWOOD DRIVE, SUITE 105, ITHACA, NEW YORK 14850, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CRAWFORD, PATTI, C.</td>
											<td>210 NW 79TH DRIVE, GAINESVILLE, FLORIDA 32607, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GIBBS, PAUL J.</td>
											<td>3650 NW 30TH PLACE, GAINESVILLE, FLORIDA 32605, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DUBOVI, EDWARD J.</td>
											<td>33 CHELSEAS WALK, ITHACA, NEW YORK 14850-1087, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DONIS, RUBEN, OMAR</td>
											<td>429 EMORY DRIVE NE, ATLANTA, GEORGIA 30307, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KATZ, JACQUELINE</td>
											<td>447 HARDENDORF AVE., N.E. ATLANTA, GEORGIA 30307, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>KLIMOV, ALEXANDER I.</td>
											<td>3197 AMBLEWOOD CT., ATLANTA, GEORGIA 30345, U.S.A.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>COX, NANCY, J</td>
											<td>857 OAKDALE ROAD, ATLANTA, GA 30307, U.S.A.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>CASTLEMAN, WILLIAM,L.</td>
											<td>4330 SW 7TH STREET, GAINESVILLE, FL 32608, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/44</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/015090</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-04-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/673,443</td>
									<td>2005-04-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263223-an-isolated-influenza-virus-capable-of-infecting-canid-animal by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:51:25 GMT -->
</html>
