<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263207-an-isolated-mutant-atpe-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:55:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263207:&quot;AN ISOLATED MUTANT ATPE PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED MUTANT ATPE PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention provides an isolated mutant atpE protein and departing from said mutant atpE protein the identification of an ATPase binding domain. This invention also provides related nucleic acids, vectors, host cells, pharmaceutical compositions and articles of manufacture. This invention further provides methods for determining whether a test compound interacts with an atpE protein, i.e. with the ATPase binding domain of the present invention, as well as Pharmaceuticals compositions comprising said test compound, in particular as antimicrobials, more particular as antimycobacterial agent, even more particular for treating tuberculosis in a subject.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A BACTERIAL ATP SYNTHASE BINDING DOMAIN<br>
This invention provides an isolated mutant atpE protein and departing from said mutant atpE protein the identification of an ATPase binding domain. This invention also provides related nucleic acids, vectors, host cells, pharmaceutical compositions and articles of manufacture. This invention further provides methods for determining whether a test compound interacts with an atpE protein, i.e. with the ATPase binding domain of the present invention, as well as pharmaceuticals compositions comprising said test compound, in particular as antimicrobials, more particular as antimycobacterial agent, even more particular for treating tuberculosis in a subject.<br>
BACKGROUND OF THE INVENTION<br>
After AIDS, tuberculosis (TB) is the leading cause of adult mortality (2-3 million deaths per year) in the world and is a critical impediment to alleviating global poverty and suffering (/). Factors contributing to the resurgence of the disease include difficulties in implementing anti-TB programs in many countries, the dramatic increase in the number of immunosuppressed individuals—due mainly to HIV infection—and the movement of people through and from areas where TB is endemic. The TB and HTV epidemics fuel one another in co-infected people—currently 11 million adults— increasing both morbidity and mortality (2, 3). In addition, TB is the leading cause of death in HTV-infected people (4).<br>
Although first-line anti-TB drug regimens can achieve more than 90% efficacy rates, their complexity can lead to poor compliance when adequate medical support and TB treatment programs are not available and, in turn, to the emergence of resistance (5). Multidrug-resistant (MDR) strains of TB complicate treatment substantially (6). The Global Alliance for TB Drug Development has recommended that any new treatment should offer at least one of the following three advantages over existing therapies: shortening or simplifying effective treatment of TB; increasing efficacy against MDR-TB; and improving treatment of the latent form of TB infection. Such a new drug would greatly improve patient compliance, thereby reducing the cost of TB treatment programs like the World Health Organization (WHO)'s Directly Observed Treatment Short-course (DOTs) strategy (7).<br>
either from existing families of drugs (such as moxifloxacin), or analogs of first-line drugs such as MJH-98-1-81 (from isoniazid), the oxazolidinones, and rifapentine (a<br>
close analog of rifampin) (8). Although these new drugs may be potent, analog compounds provide only temporary solutions to resistance (9), as they rely on the same mechanism of action as the existing families of drugs. Antibiotics in general usually inhibit bacterial replication by inhibiting bacterial metabolism though a specific mechanism. For example, isoniazid interferes with the enzymatic machinery that synthesizes mycolic acids, necessary components of the cell wall, while rifampicin interferes with the bacterial machinery for transcribing RNA from DNA. It is accordingly of interest to explore novel methods to identify anti-TB compounds that target different mycobacterial specific aspects of cell growth and replication compared to the known agents.<br>
SUMMARY OF THE INVENTION<br>
This invention provides for isolated mutant atpE proteins, in particular encoded by the amino acid sequences selected from (SEQ ID No.l), (SEQ ID No.2), (SEQ ID No.3), (SEQ ID No.4) and (SEQ ID No.5), an isolated nucleic acid encoding said mutant atpE proteins, in particular selected from the group consisting of (SEQ ID No.6), (SEQ ID No.7), (SEQ ID No.8), (SEQ ID No.9) and (SEQ ID No. 10) and a vector comprising the instant nucleic acid. In a particular embodiment the mutant atpE protein is encoded by SEQ ID No.2 and the isolated nucleic acid sequence encoding said protein consists of SEQ ID No.7.<br>
This invention further provides a host-vector system comprising a host cell having therein the instant expression vector.<br>
This invention further provides an isolated cell comprising a mutant atpE protein, wherein said protein induces anti-microbial resistance in the cell.<br>
This invention further provides a method for identifying an anti-microbial compound said method comprising the steps of<br>
(a)	contacting a cell expressing an atpE protein with a test compound under<br>
physiological conditions;<br>
(b)	determining whether the test compound interacts with the atpE protein.<br>
This invention further provides a method for evaluating the potential of a test compound to interact with an atpE protein said method comprising; (a) using molecular modeling techniques to generate a three-dimensional structure ot the atpE protein;<br>
(b)	employing computational means to perform a fitting operation between the test<br>
compound and the three-dimensional structure of the atpE protein; and<br>
(c)	analyzing the results of said fitting operation to quantify the association of the test<br>
compound with the three dimensional structure of the atpE protein.<br>
It is also an object of the present invention to provide a binding site of an FO part of an ATPase comprising at least the amino acids Ala24, Gly27, Phe53, Val", Gly58, Glu61, Tyr64 and Phe65 of one C subunit; the amino acids Ser182, Leu183, Ser184, Leu185, and Arg186 of one A subunit and said amino acids having the atomic coordinates of any of Tables 3, 4 or 5.<br>
In a further object, the present invention provides the use of the aforementioned binding domain in a method to identify compounds that interact with the F0 part of an ATPase and to their potential as anti-microbial compounds, in particular in a method to identify anti-mycobacterial compounds.<br>
It is accordingly, an object of the present invention to provide a method of treating a subject with a microbially-based infection, comprising administering to the subject a compound that interacts with the F0 part of an ATPase, in particular with an atpE protein at the resistance-conferring mutation sites or with the binding site of the present invention. This invention further provides a method for treating a subject afflicted with tuberculosis comprising administering to the subject an agent that interacts with an atpE protein using any of the aforementioned screening methods. In the methods of treatment comprising the use of a compound that interacts with the FO part of an ATPase, in particular with an atpE protein, compounds previously known to interact with the FO part of an ATPase, and in particular with an atpE protein are to be excluded. More in particular the use of the DARQ J compounds described in (11) in any of the disclosed method of treatments is to be excluded.<br>
This invention further provides for a pharmaceutical composition comprising an agent that interacts with an atpE protein in a cell, and a pharmaceutically acceptable carrier. Finally, this invention provides for an article of manufacture comprising a packaging and a pharmaceutical agent, wherein (a) the pharmaceutical agent interacts with an atpE protein in a cell, and (b) the packaging comprises a label indicating the use of the agent for treating a bacterial infection in a subject. In a particular embodiment the present invention provides the use of DARQ J in the manufacture of an anti-microbial medicine.<br>
This and further aspects of the present invention will be discussed in more detail hereinafter.<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
Table. 1   The minimum inhibitory concentrations (MICs) of the lead DARQ compound (J), that inhibited 90% of the growth of different mycobacterial species, the number of strains tested were n=l, unless otherwise indicated.<br>
Table. 2   The amino acids surrounding the binding site for the DARQ J compound.<br>
Table. 3   The atomic coordinates for the amino acids surrounding the binding site for the DARQ J compound derived from both the wild type and DARQ J mutant M. tuberculosis strain.<br>
Table 4.   The atomic coordinates for the binding site for the DARQ J compound in the wild type M. tuberculosis.<br>
Table 5.   The atomic coordinates for the binding site for the DARQ J compound in the DARQ J mutant M. tuberculosis strain.<br>
Table 6.   The atomic coordinates for the mutant atpE protein (SEQ ID No.2) of M. tuberculosis.<br>
Table 7.   The atomic coordinates for the wild type atpE protein (SEQ ID No.l) of M. tuberculosis.<br>
Fig. 1       Absolute configuration of R207910, hereinafter also refered to as J or DARQ J.<br>
Fig. 2      atpE protein sequence alignments for M. tuberculosis and M. smegmatis mutants. Mtb_S: drug-sensitive strain of M. tuberculosis H37RV, atpE<br>
(1-81). Accession number: Swiss-Prot Q10598 (SEQ ID No.l). Mtb_R: drug-resistant strain of M. tuberculosis BK12, atpE (1-81) (SEQ ID<br>
No.2). Msm_S: drug-sensitive strain of M. smegmatis, atpE (1-86). Sequence obtained by the Institute for Genome Research (SEQ ID No.3). Msm_R09 (SEQ ID No.4) and RIO (SEQ ID No.5): drug-resistant strains of M. smegmatis atpE (1-86). Sequences obtained in-house. Human: Homo sapiens, ATP5G3 (66-142). Accession number: Ensembl ENSP00000284727. Top numbering: M. tuberculosis and M. smegmatis atpE. Bottom numbering: H sapiens ATP5G3 (66-142) Shading indicates amino acid similarity using BLOSUM62 matrix (black = high, grey = medium). Arrows indicate the positions of the point mutations observed in the resistant strains.<br>
Fig. 3      Total cellular ATP-measurement of M. tuberculosis in presence of DARQ J, Isoniazid &amp; DCCD. Relative Luminescence Units of Oxyluciferin measured at 526 nm, both in wild type M. tuberculosis and DARQ J mutant M. tuberculosis.<br>
Fig. 4. Ribbon representation of the three C-subunits (A Chain, K Chain and L Chain) and the A subunit (M chain) which form together the binding site for the DARQ J compound.<br>
DETAILED DESCRIPTION<br>
Definitions<br>
As used in this application, except as otherwise expressly provided herein, each of the<br>
following terms shall have the meaning set forth below.<br>
"atpE protein" shall mean the C chain of the FO subunit of ATPase complex as represented by SwissProt entry Q10598 for M. tuberculosis, or a protein having at least 70, 80,90, 95,97 or 99% sequence identity to said M. tuberculosis sequence.<br>
"F]Fo ATPase" also referred to as ATPase, ATP synthase or FoF1 ATP ase shall mean a large-multisubunit complex that catalyses the synthesis or hydrolysis of ATP. FoF1 ATPases are composed of two domains: an F] part, which is extrinsic to the membrane and contains the catalytic sites and an F0 part which spans the bilayer and contains a proton pore. The A11J ascs arc luuiid in the plasma membrane or bucttrm, the thylakoid membrane of chloroplasts, and the inner membrane of mitochondria where they use the energy of a prolon electrochemical gradient to drive ATP synthesis.<br>
"Administering" shall mean delivering in a manner, which is effected or performed using any of the various methods and delivery systems known to those skilled in the art. Administering can be performed, for example, topically, intravenously, pericardially, orally, via implant, transmucosally, transdermally, intramuscularly, subcutaneously, intraperitoneally, intrathecally, intralymphatically, intralesionally, or epidurally. Administering can also be performed, for example, once, a plurality of times, and/or over one or more extended periods.<br>
"Host cells" include, but are not limited to, bacterial cells, yeast cells, fungal cells, insect cells, and mammalian cells. Bacterial cells can be transfected by methods well-known in the art such as calcium phosphate precipitation, eleclroporation and microinjection.<br>
"Isolated", with respect to atpE protein, shall mean an atpE protein- containing membrane fragment preparation or other suitable preparation wherein atpE retains its natural function and is free from some or all of the other proteins in its native milieu. It is meant to include membrane preparations comprising the F0 part of an FoFi ATPase, in particular an Fo part comprising the mutant atpE proteins of the present invention.<br>
"Bacteria] cell" shall mean any bacterial cell. Bacterial cells include, without limitation, cells which are normal, abnormal and transformed, and are exemplified by mycobacteria, in particular Mycobacterium tuberculosis and Mycobacterium smegmatis, corynebacteria, nocardia, gram-positive bacteria such as for example streptococcus, staphylococcus and enterococcus or gram negative bacteria such as for example Escherichia coli, Heamophilus influenzae and Helicobacterpylori.<br>
The terms "nucleic acid" and "polynucleotide" are used interchangeably herein, and each refers to a polymer of deoxyribonucleotides and/or ribonucleotides. The deoxyribonucleotides and ribonucleotides can be naturally occurring or synthetic analogues thereof.<br>
The term "physiological conditions" shall mean, with respect to a given cell, such conditions, which would normally constitute the cell's biochemical milieu. The cell's biochemical milieu includes, without limitation some or all the proteases to which the cell is normally exposed. Such conditions include, but are not limited, to in vivo conditions.<br>
The terms "polypeptide," "peptide" and "protein" are used interchangeably herein, and each means a polymer of amino acid residues. The arnino acid residues can be naturally occurring or chemical analogues thereof. Polypeptides, peptides and proteins can also<br>
include modifications such as glycosylation, lipid attachment, sulfation, hydroxylation, and ADP- ribosylation.<br>
"Subject" shall mean any animal, such as a mammal or a bird, including, without limitation, a cow, a horse, a sheep, a pig, a dog, a cat, a rodent such as a mouse or rat, a turkey, a chicken and a primate. In the preferred embodiment, the subject is a human being.<br>
"Treating" shall include, without limitation, eliminating, reversing the course of, slowing the progression of, reducing the symptoms of, or otherwise ameliorating, a disease in a subject.<br>
"Vector" shall mean any nucleic acid vector known in the art. Such vectors include, but are not limited to, plasmid vectors, cosmid vectors, and bacteriophage vectors.<br>
The terms "Candidate substance" and "Test compound" are used interchangeably and refer to a substance that is believed to interact with another moiety, i.e. the atpE protein, as a biological response modifier. For example a representative candidate substance is believed to interact with an atpE protein and is believed to modify the ATPase activity. Exemplary candidate substances that can be investigated using the methods of the present invention include, but are not restricted to peptides, enzymes, enzyme substrates, co-factors, sugars, oligonucleotides, chemical compounds small molecules and monoclonal antibodies.<br>
"Modulate" shall mean an increase, decrease or other alteration of any or all chemical and biological activities or properties of a wild type or mutant atpE protein.<br>
"Interact" shall mean detectable interactions between molecules, including "binding" interactions beween molecules. Interactions can, for example, be protein-protein or protein-nucleic acid in nature. Such interactions can be detected using art know procedures, for example, yeast two-hybrid assay, immunoprecipitation, SPA-assay or filter binding assays.<br>
As used herein, the term "atomic coordinates" or "structure coordinates" refers to mathematical coordinates that describe the positions of atoms in Protein Data Bank (PDB) format, including X, Y, Z and B for each atom. Those of skilled in the art understand that a set of structure coordinates determined by X-ray crystallography is not without standard error. For the purpose of this invention, any set of structure coordinates for ATPsynthase from any source having a root mean square deviation of position of the corresponding atomic coordinates of Tables 3,4,5, 6 or 7 are considered substantially identical or homologous. In a more preferred embodiment, any set of<br><br>
structure coordinates for ATPsynthase from any source having a root mean square deviation of non-hydrogen atoms of less than 0.75A when superimposed on the non-hydrogen atom position of the corresponding atomic coordinates of Tables 3,4,5, 6 or 7 are considered substantially identical.<br>
Embodiments of the Invention<br>
Mutant atpE proteins<br>
This invention provides for isolated mutant atpE proteins, in particular bacterial atpE proteins, more particular mycobacterial atpE-proteins, even more particular M. tuberculosis or M. smegmatis atpE proteins. The mutation is selected from single point mutations, insertions or deletions. In one embodiment of the invention the mutation consist of at least one point mutation located in any one of amino acids 20 to 40, in particular 30 to 40, preferable in amino acid 34 or of amino acids 60 to 75, in particular 62 to 73, preferably in amino acid 69 as shown in the sequence alignment of Fig.2. In a further embodiment the isolated mutant atpE proteins are selected from Mtb_R (SEQ ID No.2), Msm_R09 (SEQ ED No.4) and Msm_R10 (SEQ ID No.5) as shown in Fig.2 or of an amino acid sequence having at least 70, 80,90,95,97 or 98% sequence identity to any of the aforementioned amino acid sequences.<br>
This invention further provides an isolated nucleic acid encoding said mutant atpE proteins. In one embodiment said nucleic acid sequence consists of all genes that encode an FO part, such as for example described in J.Biol.Chem, 1994, Vol.269(10),p.7285-7289, wherein said genes are transcribed from a single promoter and comprise the nucleic acid sequence encoding the mutant atpE protein of the present invention. The nucleic acid can be DNA or RNA, and preferably DNA and are in a further embodiment selected from the nucleic acid sequences encoding Mtb_R (SEQ ID No. 7) Msm_R09 (SEQ ID No.9), MsmJUO (SEQ ID No. 10) or a nucleic acid sequence having at least 70, 80,90,95, 97 or 98% sequence identity to any of the aforementioned nucleic acid sequences.<br>
The percentage identity of nucleic acid and polypeptide sequences can be calculated using commercially available algorithms which compare a reference sequence with a query sequence. The following programs (provided by the National Center for Biotechnology Information) may be used to determine homologies/identities: BLAST, gapped BLAST, BLASTN and PSI-BLAST, which may be used with default parameters.<br>
The algorithm GAP (Genetics Computer Group, Madison, WI) uses the Needleman and Wunsch algorithm to align two complete sequences that maximizes the number of matches and minimizes the number of gaps. Generally, the default parameters are used, with a gap creation penalty = 12 and gap extension penalty = 4.<br>
Another method for determining the best overall match between a nucleic acid sequence or a portion thereof, and a query sequence is the use of the FASTDB computer program based on the algorithm of Brutlag et al (Comp. App. Biosci., 6; 237-245 (1990)). The program provides a global sequence alignment. The result of said global sequence alignment is in percent identity. Suitable parameters used in a FASTDB search of a DNA sequence to calculate percent identity are: Matrix=Unitary, k-tuple=4, Mismatch penalty=l, Joining Penalty=30, Randomization Group Length=0, Cutoff Score=l, Gap Penalty=5, Gap Size Penalty=0.05, and Window Size=500 or query sequence length in nucleotide bases, whichever is shorter. Suitable parameters to calculate percent identity and similarity of an amino acid alignment are: Matrix=PAM 150, k-tuple=2, Mismatch Penalty=l, Joining Pena!ty=20, Randomization Group Length=0, Cutoff Score=l, Gap Penalty=5, Gap Size Penalty=0.05, and Window Size=500 or query sequence length in nucleotide bases, whichever is shorter.<br>
This invention further provides a vector comprising the instant nucleic acid. In one embodiment the vector is a plasmid vector.<br>
This invention further provides a host-vector system comprising a host cell having therein the instant plasmid vector. The cell can be prokaryotic or eukaryotic, in one embodiment the host cell is a bacterial cell, in particular a mycobacterial cell such as for example M.tuberculosis or M. smegmatis.<br>
This invention further provides an isolated cell comprising a mutant atpE protein, which protein induces anti-microbial resistance in the cell. In one embodiment the isolated cell consists of a M. smegmatis cell transformed with a mutant mycobacterial atpE protein, in particular transformed with a mutant mycobacterial atpE protein wherein the mutation consist of at least one point mutation located in any one of amino acids 20 to 40, in particular 30 to 40, preferable in amino acid 34 or of amino acids 60 to 75. in particular 62 to 73, preferably in amino acid 69 as shown in the sequence alignment of Fig.2.<br>
Screening methods<br>
This invention further provides a method for identifying an anti-microbial compound said method comprising the steps of<br>
(a)	contacting a cell expressing an atpE protein with a test compound under<br>
physiological conditions;<br>
(b)	determining whether the test compound interacts with the atpE protein.<br>
In one embodiment the atpE protein used in the aforementioned method, consist of a<br>
bacterial atpE protein, in particular a mycobacterial protein and is meant to include both<br>
the wild type atpE proteins as well as the mutant atpE proteins as decribed<br>
hereinbefore. In a further emlbodiment of the present invention, the mycobacterial atpE<br>
protein used in the aforementioned method consists of a mutant mycobacterial atpE<br>
protein according to the invention. In a particular embodiment of the aforemeantioned<br>
assay a host cell transformed with a mutant atpE protein of the invention is used and the<br>
interaction of the test compound with said atpE protein is asses.sed by determining the<br>
possible inhibition of the enz;miatic activity of. the FiFo-ATPase comprising said<br>
mutant atpE protein. Inhibition of the FiFo- ATPase activity is determined using art<br>
known procedures, such as for example by adding the substance to a system comprising<br>
the FiFo-ATPase and ATP as a substrate, with detection of the enzymatic activity by<br>
coupling the production of ADP to the oxidation of NADH via pyruvate kinase and<br>
lactate hydrogenase reactions	,<br>
In one embodiment of the assay, the atpE protein may be employed in a binding assay. Binding assays may be competitive or non-competitive. Such an assay can accommodate the rapid screening of a large number of compounds to determine which compounds, if any, are capable of binding to the polypcptidcs.<br>
Within this context, the present invention provides a method to identify whether a test compound binds to an i isolated atpE protein of the present invention, and is thus a potential anti-microbial compound, said method comprising;<br>
a)	contacting cells expressing the atpE protein wherein such cells do not<br>
normally express said atpE protein, with the test compound in the presence and absence<br>
of a compound known to bind the atpE protein,<br>
b)	determine the binding of the test compound to the atpE protein using the<br>
compound known to bind to the atpE protein as a reference.<br>
Binding of the test compound or of the compound known to bind to the atpE protein, hereinafter also referred to as reference compound, is assessed using art-known methods for the study of protein-ligand interactions. For example, such binding can be measured by employing a labeled substance or reference compound. The test compound or reference compound, in particular compound J (Fig.l) can be labeled in any convenient manner known in the art, e.g. radioactively, fluorescently or enzymatically. In a particular embodiment of the aforementioned method, the compound known to bind to the atpE protein, also known as the reference compound is detectably labeled, and . Said label is used to determine the binding of the test compound to the atpE protein. Said reference compound being labeled using a radiolabel, a fluorescent label or an enzymatic label, more preferably a radiolabel.<br>
In an alternative embodiment of the present invention, the aforementioned binding assays are performed on a cellular composition, i.e a cellular extract, a cell fraction or cell organels comprising an atpE protein as defined hereinbefore. More in particular, the aforementioned binding assays are performed on a cellular composition, i.e. a membrane preparation comprising an atpE protein as defined hereinbefore, wherein said cellular composition, i.e. membrane preparation is obtained from a M. smegmatis cell transformed with a mutant mycobacterial atpE protein, in particular transformed with a mutant mycobacterial atpE protein wherein the mutation consist of at least one point mutation located in any one of amino acids 20 to 40, in particular 30 to 40, preferable in amino acid 34 or of amino acids 60 to 75, in particular 62 to 73, preferably in amino acid 69 as shown in the sequence alignment of Fig.2. Taking the numbering of Mtb_S (SEQ ID No.l) or of Mtb_.R (SEQ ID No.2) as a reference, the aforementioned regions correspond to amino acids 14 to 34, in particular 24 to 34, preferably in amino acid 28 or to amino acids 54 to 69, in particular 56 to 67, preferably to amino acid 63.<br>
In one embodiment the binding assays are performed using membrane preparations. These membrane preparations can be used in conventional filter-binding assays (eg. Using Brandel filter assay equipment) or in high throughput Scintillation Proximity type binding assays (SPA and Cytostar-T flashplate technology; Amersham Pharmacia Biotech) to detect binding of radio-labelled atpE ligands (including 3H labelled DARQs) and displacement of such radio-ligands by competitors for the binding site.<br>
Radioactivity can be measured with Packard Topcount, or similar instrumentation.<br>
capable of making rapid measurements from 96-, 384-, 1536- microtitre well formats. SPA/Cytostar-T technology is particularly amenable to high throughput screening and<br>
therefore this technology is suitable to use as a screen for compounds able to displace standard ligands.<br>
Another approach to study binding of ligands to atpE protein in an environment approximating the native situation makes use of a surface plasmon resonance effect exploited by the Biacorc instrument (Biacore). atpE protein in membrane preparations or whole cells could be attached to the biosensor chip of a Biacore and binding of ligands examined in the presence and absence of compounds to identify competitors of the binding site.<br>
Molecular Modeling<br>
This invention further provides a method for evaluating the potential of a test compound to interact with an atpE protein said method comprising;<br>
(a)	using molecular modeling techniques to formulate a three dimensional structure of<br>
the atpE protein;<br>
(b)	employing computational means to perform a fitting operation between the test<br>
compound and the three-dimensional structure of the atpE protein; and<br>
(c)	analyzing the results of said fitting operation to quantify the association of the test<br>
compound with the three dimensional structure of the atpE protein.<br>
Molecular modeling techniques are known in the art, including both hardware and software appropriate for creating and utilizing models of receptors and enzyme conformations.<br>
Numerous computer programs are available and suitable for the processes of computer modeling, model building and computationally identifying, selecting and evaluating potential atpE interacting compounds in the methods described herein. These include for example, GRID (available from Oxford University, UK), MCSS (available from Accelrys, Inc., San Diego, CA), AUTODOCK (available from Oxford Molecular Group), FLEX X (available form Tripos, St. Louis. MO), DOCK (available from University of California, San Francisco, CA), CAVEAT (available from University of California, Berkeley), HOOK (available from Accelrys, Inc., San Diego, CA) and 3D database systems such as MACCS-3D (available from MDL Information Systems, San Leandro, CA), UNITY (available from Tripos, St. Louis.MO) and CATALYST (available from Accelrysjnc., San Diego, CA). Potential candidate substances may also be computationally designed "de novo'using software packages as LUDI (available from Biosym Technologies, San Diego, CA), LEGEND (available from<br>
Accelrys, Inc. San Diego, CA) and LEAPFROG (available from Tripos, St. Louis.MO). Compound deformation energy and electrostatic repulsion, may be analysed using programs such as GAUSSIAN 92, AMBER, QUANTA/CHARMM and INSIGHT n/DISCOVER. These computer evaluation and modeling techniques may be performed on any suitable hardware including for example, workstations available from Silicon Graphics, Sun Microsystems and others. These modeling techniques, methods, hardware and software packages are representative and are not intended to be a comprehensive listing. Other modeling techniques known in the art may also be employed in accordance with this invention. See for example, N.C. Cohen, Molecular Modeling in Drug Design, Academic Press (1996).<br>
In one embodiment of the present invention, the three-dimensiona] structure of the atpE protein is generated using the atomic coordinates of the De28, Glu61 and Ile63 of E. coli (Protein Database 1Q01) +/- a root mean square deviation of the backbone atoms of said amino acids of not more that 10A, preferably not more that 5 A.<br>
As provided in the examples hereinafter, it has been an object of the present invention to provide the three-dimensional structure of the atpE protein. Tables 6 and 7 provide the atomic coordinates for the mutant and wild type atpE protein with SEQ ID No.2 and SEQ ID No.l. Thus, in one embodiment the three-dimensional structure of the atpE protein is generated using the atomic coordinates of Tables 6 or 7. In a particular embodiment the three-dimensional structure of the atpE protein is generated using the atomic coordinates of Table 7. The DARQ J compound inhibits the interaction of Arg18fi of the A subunit with Gl u61 of the C-subunit in its deprotonated from. It is accordingly an object of the present invention to provide the use of the atomic coordinates of Tables 6 or 7 in a method to evaluate the potential of a test compound to interact with an atpE protein.<br>
Binding Site<br>
In another embodiment the present invention provides the characterization of a binding site in the F0 part of an ATPase. This binding site, identified as being capable of binding the DARQ J compound, was found to coincide with the regions identified hereinbefore, as the resistance-conferring mutation sites in the atpE proteins of<br>
M.tuberculosis and M. smegmatis (17). Hence, the present invention provides a<br>
binding site in the F0 part of an ATPase characterised in that it comprises the resistance-conferring mutation sites of an atpE protein. The resistance-conferring mutation sites<br>
as used herein refer to ammo acids 14 to 34, in particular 24 to 34 and to amino acids 53 to 69, in particular 56 to 67 of an atpE protein, taking the numbering of Mtb_S (SEQ ID No.l) or of MtbJR (SEQ ID No.2) as a reference.<br>
In a further embodiment the binding site comprises at least the amino acids Ala24, Gly27, Phe33, Val57, Gly58, Glu61, Tyr64 and Phe63 of one C subunit and the amino acids Ser182, Leu183, Leu185, and Arg186 of one A subunit (having for the A subunit, the codes Ser 206 - Leu 207 - Leu 209 and Arg 210 in Tables 3,4 and 5), wherein said amino acids have the atomic coordinates of any of Tables 3,4 or 5 or homologous structure coordinates comprising a root mean square deviation of non-hydrogen atoms of less than about 1.5 A, preferably not more that 0.75 A, when superimposed on the non-hydrogen atom positions of the corresponding atomic coordinates of Tables 3,4 or 5. In a particular embodiment the binding site comprises amino acids Ala21, Gly25 of a first C subunit; amino acids Ala24, Gly27, Phe53, Phe54, Val57, Gly58, Glu61, Tyr64, Phe65 of a second C subunit; amino acids Met17, Gly19, Gly20, Ala21, He22, Gly23, Ala24, Gly25, He26, Gly27, Asp28, Gly29, Ala31, Phe53, Thr56, Val57, Gly58, Leu59, Val60, Glu61, AJa62, Ala63/Pro63, Tyr64, Phe65 of a third C subunit and amino acids Leu183, Leu185 and Arg186 of an A subunit; wherein said amino acids have the atomic coordinates of any of Tables 3,4 or 5 or homologous structure coordinates comprising a root mean square deviation of non-hydrogen atoms of less than about 1.5 A, preferably not more that 0.75 A, when superimposed on the non-hydrogen atom positions of the corresponding atomic coordinates of Tables 3,.4 or 5. In an even more particular embodiment the binding site consists of the amino acids Ala21, Gly25 of a first C subunit; amino acids Ala24, Gly27, Phe53, Phe54, Val57, Gly58, Glu61, Tyr64, Phe65 of a second C subunit; amino acids Met17, Gly19, Gly20, Ala21, He22, Gly23, Ala24, Gly25, He26, Gly27, Asp28, Gly29, Ala31, Phe53, Thr56, Val57, Gly58, Leu59, Val*0, Glu61, Ala62, Ala63/Pro63, Tyr64, Phe65 of a third C subunit and amino acids Leu183, Leu185, and Arg186 of an A subunit; wherein said amino acids have the atomic coordinates of any of Tables 3,4 or 5. In a most particular embodiment the binding site consists of the amino acids Ala21, Gly25 of a first C subunit; amino acids Ala24, Gly27, Phe53, Phe54, Val57, Gly58, Glu61, Tyr64, Phe65 of a second C subunit; amino acids Mel17, Gly19, Gly20, Ala21, He22, Gly23, Ala24, Gly25, He26, Gly27, Asp28, Gly29, Ala31, Phe53, Thr56, Val57, Gly58, Leu59, Val60, Glu61, Ala62, Ala63/Pro63, Tyr64, Phe65 of a third C subunit and amino acids Leu183, Leu185 and Arg186 of an A subunit; wherein said amino acids have the atomic coordinates of Table 3.<br>
It is accordingly an object of the present invention to evaluate the potential of a test compound to interact with an atpE protein using the atomic coordinates as outlined<br>
above, in computational screening programs. In one embodiment, the present invention provides a method to evaluate the potential of a test compound to interact with an atpE protein, said method comprising; - molecular modeling techniques to generate the three-dimensional structure of a binding site of an FQ part of an ATPase; - employing computational means to perform a fitting operation between the test compound and the three-dimensional structure of the binding site; and - analyzing the results of said fitting operation to quantify the association of the test compound with the three-dimensional structure of the binding site. In an further embodiment of the present invention, the three-dimensional structure of the binding site is generated using the atomic coordinates of Tables 3,4 or 5 or homologous structure coordinates comprising a root mean square deviation of non-hydrogen atoms of less than about 1.5A, preferably not more that 0.75 A, when superimposed on the non-hydrogen atom positions of the corresponding atomic coordinates of Tables 3, 4 or 5. In a particular embodiment the three-dimensional structure is generated using the atomic coordinates of the amino acids Ala21, Gly25 of the A Chain of any of Tables 3,4 or 5; the amino acids Ala24, Gly27, Phe53, Phe54, Val57, Gly58, Glu61, Tyr64, Phe65 of the K Chain of any of Tables 3, 4 or 5; the amino acids Met17, Gly19, Gly20, Ala21, De22, Gly23, Ala24, Gly25, He26, Gly27, Asp28, Gly29, Ala31, Phe53, Thr56, Val57, Gly58, Leu59, Val60, Glu61, Ala62, Ala63/Pro63, Tyr64, Phe65 of the L Chain of any of Tables 3, 4 or 5; and the amino acids Set206, Leu207, Leu207, and Arg210 of the M Chain of any of Tables 3,4 or 5.<br>
In this screening, the quality of fit of such compounds to the binding site may be judged either by shape complementarity or by estimated interaction energy (Meng, E.G. et al., J. Coma.Chem 13:505-524 (1992)).<br>
Use of Binding Site<br>
The design of compounds that bind to, promote or inhibit the functional activity of atpE<br>
according to this invention generally involves consideration of two factors. First, the<br>
compound must be capable of physically and structurally associating with atpE. Non-<br>
covalent molecular interactions important in the association of atpE with the<br>
compound, include hydrogen bonding, van der Waals and hydrophobic interactions.<br>
Second, the compound must be able to assume a conformation that allows it to<br>
associate with atpE. Although certain portions of the compound may not directly<br>
conformation of the molecule. This, in turn, may have a significant impact on binding affinities, therapeutic efficacy, drug-like qualities and potency. Such conformation<br>
requirements include the overall three-dimensional structure and orientation of the chemical entity or compound in relation to all or a portion of the active site or other region of atpE or the spacing between functional groups of a compound comprising several chemical entities that directly interact with atpE.<br>
The potential, predicted, inhibitory agonist, antagonist or binding effect of a ligand or other compound on atpE may be analyzed prior to its actual synthesis and testing by the use of computer modeling techniques. If the theoretical structure of the given compound suggests insufficient interaction and association between it and atpE, synthesis and testing of the compound may be obviated. However, if computer modeling indicates a strong interaction, the molecule may then be synthesized and tested for its ability to interact with atpE. In this manner, synthesis of inoperative compounds may be avoided. In some cases, inactive compounds are synthesized predicted on modeling and then tested to develop a SAR (structure-activity relationship) for compounds interacting with a specific region of atpE. One skilled in the art may use one of several methods to screen chemical entities fragments, compounds, or agents for their ability to associate with atpE and more particularly with the individual binding pockets or active sites of atpE. This process may begin by visual inspection of, for example, the active site on the computer screen based on the atomic coordinates of atpE or atpE complexed with a ligand. Selected chemical entities, compounds, or agents may then be positioned in a variety of orientations, or docked within an individual binding pocket of atpE. Docking may be accomplished using software such as Quanta and Sybyl, followed by energy minimization and molecular dynamics with standard molecular mechanics forcefields, such as CHARMM and AMBER.<br>
Specialized computer programs may also assist in the process of selecting chemical entities. These include but are not limited to: GRID (Goodford, P. J., "A Computational Procedure for Determining Energetically Favorable Binding Sites on Biologically Important Macromolecules," J. Med. Chem. 28:849-857 (1985), available from Oxford University, Oxford, UK); MCSS (Miranker, A. and M. Karplus, "Functionality Maps of Binding Sites: A Multiple Copy Simultaneous Search Method." Proteins: Structure, Function and Genetics 11: 29-34 (1991), available from Molecular Simulations, Burlington, Mass.); AUTODOCK (Goodsell, D. S. and A. J. Olsen, "Automated<br>
Docking of Substrates to Proteins by Simulated Annealing" Proteins: Structure.<br>
Function, and Genetics 8:195-202 (1990), available from Scripps Research Institute, La Jolla, Calif.); and DOCK (Kuntz, I. D. et al., "A Geometric Approach to<br>
Macromolecule-Ligand Interactions," J.-Mol. Biol. 161:269-288 (1982), available from University of California, San Francisco, Calif.).<br>
The use of software such as GRID, a program that determines probable interaction sites between probes with various functional group characteristics and the macromolecular surface, is used to analyze the surface sites to determine structures of similar inhibiting proteins or compounds. The GRID calculations, with suitable inhibiting groups on molecules (e.g., protonated primary amines) as the probe, are used to identify potential hotspots around accessible positions at suitable energy contour levels. The program DOCK may be used to analyze an active site or ligand binding site and suggest ligands with complementary steric properties.<br>
Once suitable chemical entities, compounds, or agents have been selected, they can be assembled into a single ligand or compound or inhibitor or activator. Assembly may proceed by visual inspection of the relationship of the fragments to each other on the three-dimensional image. This may be followed by manual model building using software such as Quanta or Sybyl.<br>
Useful programs to aid in connecting the individual chemical entities, compounds, or agents include but are not limited to: CAVEAT (Bartlett, P. A. et al., "CAVEAT: A Program to Facilitate the Structure-Derived Design of Biologically Active Molecules." In Molecular Recognition in Chemical and Biological Problems, Special Pub., Royal Chem. Soc., 78, pp. 82-196 (1989)); 3D Database systems such as MACCS-3D (MDL Information Systems, San Leandro, Calif, and Martin, Y.C., "3D Database Searching in Drug Design", J. Med. Chem. 35: 2145-2154 (1992); and HOOK (available from Molecular Simulations, Burlington, Mass.).<br>
Several methodologies for searching three-dimensional databases to test pharmacophore hypotheses and select compounds for screening are available. These include the program CAVEAT (Bacon et al., J. Mol. Biol. 225:849-858 (1992)). For instance, CAVEAT uses databases of cyclic compounds which can act as "spacers" to connect any number of chemical fragments already positioned in the active site. This allows one skilled in the art to quickly generate hundreds of possible ways to connect the fragments already known or suspected to be necessary for tight binding. Instead of proceeding to build an inhibitor activator, agonist or antagonist of atpE in a step-wise fashion one chemical entity at a time as described above, suchcompounds may be designed as a whole or "de novo" using either an empty active site or optionally including some portion(s) of a known molecules. These methods include: LUDI<br>
 (Bohm, H.-J., "The Computer Program LIJDI: A New Method for the De Novo Design of Enzyme Inhibitors", J, ComR. Aid. Molec. Design, 6, pp. 61-78 (1992), available from Biosym Technologies, San Diego, Calif.); LEGEND (Nishibata, Y. and A. Itai, Tetrahedron 47:8985 (1991), available from Molecular Simulations, Burlington, Mass.); and Leapfrog (available from Tripos Associates, St. Louis, Mo.). For instance, the program LUDI can determine a list of interaction sites into which to place both hydrogen bonding and hydrophobic fragments. LUDI then uses a library of linkers to connect up to four different interaction sites into fragments. Then smaller "bridging" groups such as - -CH2- and -COO- are used to connect these fragments. For example, for the enzyme DHFR, the placements of key functional groups in the well- known inhibitor methotrexate were reproduced by LUDI. See also, Rotstein and Murcko, / Med. Chem. 36: 1700-1710 (1992).<br>
Other molecular modeling techniques may also be employed in accordance with this invention. See, e.g., Cohen, N. C. et al., "Molecular Modeling Software and Methods for Medicinal Chemistry, J. Med. Chem. 33:883-894 (1990). See also, Navia, M. A. and M. A. Murcko, "The Use of Structural Information in Drug Design," Current Opinions in Structural Biology, 2, pp. 202-210 (1992).<br>
Once a compound has been designed or selected by the above methods, the affinity with which that compound may bind or associate with atpE may be tested and optimized by computational evaluation and/or by testing biological activity after synthesizing the compound. Inhibitors or compounds may interact with the atpE in more than one conformation that is similar in overall binding energy. In those cases, the deformation energy of binding is taken to be the difference between the energy of the free compound and the average energy of the conformations observed when the compound binds to atpE.<br>
A compound designed or selected as binding or associating with atpE may be further computationally optimized so that in its bound state it would preferably lack repulsive electrostatic interaction with atpE. Such non- complementary (e.g., electrostatic) interactions include repulsive charge- charge, dipole-dipole and charge- dipole interactions. Specifically, the sum of all electrostatic interactions between the inhibitor and atpE when the inhibitor is bound, preferably make a neutral or favorable contribution to the enthalpy of binding. Weak binding compounds will also be designed by these methods so as to determine SAR. See, for example, U. S. Appl. Nos. 60/275,629; 60/331,235; 60/379,617; and, 10/097,249.<br>
Specific computer software is available in the art to evaluate compound deformation energy and electrostatic interaction. Examples of programs designed for such uses include: Gaussian 92, revision C (M. J. Frisch, Gaussian, Inc., Pittsburgh, Pa., COPYRGT 1992); AMBER, version 4.0 (P. A. Kollman, University of California at San Francisco, COPYRGT 1994); QUANT A/CHARMM (Molecular Simulations, Inc., Burlington, Mass. COPYRGT 1994); and Insight E/Discover (Biosysm Technologies Inc., San Diego, Calif. COPYRGT 1994). Other hardware systems and software packages will be known to those skilled in the art.<br>
Once a compound that associates with atpE has been optimally selected or designed, as described above, substitutions may then be made in some of its atoms or side groups in order to improve or modify its binding properties. Generally, initial substitutions are conservative, i.e., the replacement group will have approximately the same size, shape, hydrophobicity and charge as the original group. It should, of course, be understood that components known in the art to alter conformation may be avoided. Such substituted chemical compounds may then be analyzed for efficiency of fit to atpE by the same computer methods described in detail, above.<br>
The present invention further provides systems, particularly computer-based systems, which contain the sequence and/or structure coordinates described herein. Such systems are designed to do structure determination and rational drug design for atpE or for the binding site in the F0 part of the ATPase. The computer-based systems refer to the hardware means, software means and data storage means used to analyze the sequence and/or structure coordinates of the present invention in any of the computer methods described in detail, above. The minimum hardware means of the computer-based system of the present invention comprises a central processing unit (CPU), input means, output means and data storage means. A skilled person can readily appreciate which of the currently available computer-based systems are suitable for use in the present invention.<br>
It is accordingly an object of the present invention to provide computer readable data storage medium containing the structure coordinates described herein. As used herein, "computer readable data storage medium" refers to any medium which can be read or accessed directly by a computer. Such media include, but are not limited to: magnetic storage media, such as floppy disks, hard disc storage media and magnetic tape; optical<br>
storage media such as optical discs or CD-ROM; electrical storage media such as RAM and ROM; and hybrids of these categories such as magnetical/optical storage media.<br>
Methods of Treatment<br>
As already mentioned hereinbefore, it is also an object of the present invention to provide the use of compounds identified using any of the aforementioned screening methods in a method of treating a subject with a microbially-based infection. In general, bacterial pathogens may be classified as either gram-positive or gram-negative pathogens. Antimicrobial compounds with activity against both gram-positive and gram-negative pathogens are generally regarded as having a broad spectrum of activity. The compounds of the present invention are regarded as active against gram-positive and/or gram-negative bacterial pathogens. In particular, the present compounds are active against at least one gram-positive bacterium, preferably against several gram-positive bacteria, more preferably against one or more gram-positive bacteria and/or one or more gram-negative bacteria.<br>
Examples of gram-positive and gram-negative aerobic and anaerobic bacteria, include Staphylococci, for example 5. aureus; Enterococci, for example E. faecalis; Streptococci, for example 5. pneumonias, S. mutans, S. pyogens', Bacilli, for example Bacillus subtilis; Listeria, for example Listeria monocytogenes; Haemophilus, for example H. influenza; Moraxella, for example M. catarrhalis; Pseudomonas, for example Pseudomonas aeruginosa; and Escherichia, for example £. coli. Gram-positive pathogens, for example Staphylococci, Enterococci and Streptococci are particularly important because of the development of resistant strains which are both difficult to treat and difficult to eradicate from for example a hospital environment once established. Examples of such strains are methicillin resistant Staphylococcus aureus (MRSA), methicillin resistant coagulase negative Staphylococci (MRCNS), penicillin resistant Streptococcus pneumoniae and multiple resistant Entewcoccus faecium.<br>
The compounds of the present invention also show activity against resistant bacterial strains.<br>
The compounds of the present invention are especially active against those bacteria of which the viability depends on proper functioning of FrFnATP synthase. Without being bound to any theory, it is taught that the activity of the present compounds lies in inhibition of the FtF0 ATP synthase, in particular the inhibition of the F0 complex of the<br>
FjFo ATP synthase, more in particular the inhibition of the proton transfer from Argl86 of the A subunit to Glu61 of the C subunit of the FO complex of the FjFo ATP synthase, leading to killing of the bacteria by depletion of the cellular ATP levels of the bacteria. The compounds identified using any of the aforementioned screening methods are particularly active against Grarh-positive bacteria, more particular mycobacteria, and most particular against infections caused by M. africanum, M. avium, M. bovis, M. bovis-BCG, M. chelonae, M. fortuitum, M. gordonae, M. intmcellulare, M, kansasii, M. microti, M. scrofulaceum, M. paratuberculosis, M. leprea, M. tuberculosis, M. ulcerans and M. ranae.<br>
Whenever used hereinbefore or hereinafter, that the compounds can treat a bacterial infection it is meant that the compounds can treat an infection with one or more bacterial strains. When used however, in relation to the use of DARQ J as an antimicrobial compound, anti-microbial is meant to be a compound that can treat an infection with one or more bacterial strains, provided lhat said bacterial strains are other than mycobacteria.<br>
Bacterial infections which may be treated by the present compounds include, for example, central nervous system infections, external ear infections, infections of the middle ear, such as acute otitis media, infections of the cranial sinuses, eye infections, infections of the oral cavity, such as infections of the teeth, gums and mucosa, upper respiratory tract infections, lower respiratory tract infections, genitourinary infections, gastrointestinal infections, gynecological infections, septicemia, bone and joint infections, skin and skin structure infections, bacterial endocarditis, burns, antibacterial prophylaxis of surgery, and antibacterial prophylaxis in immunosuppressed patients, such as patients receiving cancer chemotherapy, or organ transplant patients.<br>
This invention further provides a method for treating a subject afflicted with tuberculosis comprising administering to the subject an agent that interacts with an atpE protein.<br>
Pharmaceutical Compositions<br>
This invention further provides for a pharmaceutical composition comprising an agent Jaat interacts with an atpE protein in a cell, and a pharrnaceutically acceptable carrier.<br>
Such agents may be formulated into compositions comprising an agent together with a pharmaceutically acceptable carrier or diluent. The agent may in the form of a physiologically functional derivative, such as an ester or a salt, such as an acid addition salt or basic metal salt, or an N or S oxide. Compositions may be formulated for any suitable route and means of administration. Pharmaceutically acceptable carriers or diluents include those used in formulations suitable for oral, rectal, nasal, inhalable, topical (including buccal and sublingual), vaginal or parenteral (including subcutaneous, intramuscular, intravenous, intradermal, intrathecal and epidural) administration. The choice of carrier or diluent will of course depend on the proposed route of administration, which, may depend on the agent and its therapeutic purpose. The formulations may conveniently be presented in unit dosage form and may be prepared by any of the methods well known in the art of pharmacy. Such methods include the step of bringing into association the active ingredient with the carrier which constitutes one or more accessory ingredients. In general the formulations are prepared by uniformly and intimately bringing into association the active ingredient with liquid carriers or finely divided solid carriers or both, and then, if necessary, shaping the product.<br>
For solid compositions, conventional non-toxic solid carriers include, for example, pharmaceutical grades of mannitol, lactose, cellulose, cellulose derivatives, starch, magnesium stearate, sodium saccharin, talcum, glucose, sucrose, magnesium carbonate, and the like may be used. The active compound as defined above may be formulated as suppositories using, for example, polyalkylene glycols, acetylated triglycerides and the like, as the carrier. Liquid pharmaceutically administrable compositions can, for example, be prepared by dissolving, dispersing, etc, an active compound as defined above and optional pharmaceutical adjuvants in a carrier, such as, for example, water, saline aqueous dextrose, glycerol, ethanol, and the like, to thereby form a solution or suspension. If desired, the pharmaceutical composition to be administered may also contain minor amounts of non-toxic auxiliary substances such as wetting or emulsifying agents, pH buffering agents and the like, for example, sodium acetate, sorbitan monolaurate, triethanolamine sodium acetate, sorbitan monolaurate, triethanolamine oleate, etc. Actual methods of preparing such dosage forms are known, or will be apparent, to those skilled in this art; for example, see Oennaro et al., Remington's Pharmaceutical Sciences, Mack Publishing Company, Easton, Pennsylvania, 18th Edition. 1990.<br>
The composition or formulation to be administered will, in any event, contain a quantity of the active compound(s) in an amount effective to alleviate the symptoms of the subject being treated.<br>
The exact dosage and frequency of administration of the present compounds depends on the particular compound used, the particular condition being treated, the severity of the condition being treated, the age, weight, gender, diet, time of administration and general physical condition of the particular patient, the mode of administration as well as other medication the individual may be taking, as is well known to those skilled in the art. Furthermore, it is evident that the effective daily amount may be lowered or increased depending on the response of the treated subject and/or depending on the evaluation of the physician prescribing the compounds of the instant invention.<br>
Dosage forms or compositions containing active ingredient in the range of 0.25 to 95% with the balance made up from non-toxic carrier may be prepared. Depending on the mode of administration, the pharmaceutical composition will preferably comprise from 0.05 to 99 % by weight, more preferably from 0.1 to 70 % by weight of the active ingredients, and, from 1 to 99.95 % by weight, more preferably from 30 to 99.9 weight % of a pharmaceutically acceptable carrier, all percentages being based on the total composition.<br>
For oral administration, a pharmaceutically acceptable non-toxic composition is formed by the incorporation of any of the normally employed excipients, such as, for example, pharmaceutical grades of mannitol, lactose, cellulose, cellulose derivatives, sodium crosscarmellose, starch, magnesium stearate, sodium saccharin, talcum, glucose, sucrose, magnesium, carbonate, and the like. Such compositions take the form of solutions, suspensions, tablets, pills, capsules, powders, sustained release formulations and the like. Such compositions may contain l%-95% active ingredient, more preferably 2-50%, most preferably 5-8%.<br>
Parenteral administration is generally characterized by injection, either subcutaneously, intramuscularly or intravenously. Injectables can be prepared in conventional forms, either as liquid solutions or suspensions, solid forms suitable for solution or suspension in liquid prior to injection, or as emulsions. Suitable excipients are, for example, water, saline, dextrose., glycerol. ethanolorthe jikeJn addition, if desired,the pharmaceutical compositions to be administered may also contain minor amounts of non-toxic auxiliary substances such as wetting or emulsifying agents, pH buffering agents and the like,<br>
such as for example, sodium acetate, sorbitan monolaurate, triethanolamine oleate, triethanolamine sodium acetate, etc.<br>
The percentage of active compound contained in such parental compositions is highly dependent on the specific nature thereof, as well as the activity of the compound and the needs of the subject. However, percentages of active ingredient of 0.1 % to 10% in solution are employable, and will be higher if the composition is a solid which will be subsequently diluted to the above percentages. Preferably, the composition will comprise 0.2-2% of the active agent in solution.<br>
Finally, this invention provides for an article of manufacture comprising a packaging and a pharmaceutical agent, wherein (a) the pharmaceutical agent interacts with an atpE protein in a cell, and (b) the packaging comprises a label indicating the use of the agent for treating a bacterial infection in a subject. In particular as an anti-mycobacterial medicine.<br>
Throughout this description the terms "standard methods", "standard protocols" and "standard procedures", when used in the context of molecular biology techniques, are to be understood as protocols and procedures found in an ordinary laboratory manual such as: Current Protocols in Molecular Biology, editors F. Ausubel et al., John Wiley and Sons, Inc. 1994, or Sambrook, J., Fritsch, E.F. and Maniatis, T., Molecular Cloning: A laboratory manual, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY 1989.<br>
This invention will be better understood by reference to the Experimental Details that follow, but those skilled in the art will readily appreciate that these are only illustrative of the invention as described more fully in the claims that follow thereafter. Additionally, throughout this application, various publications are cited. The disclosure of these publications is hereby incorporated by reference into this application to describe more fully the state of the art to which this invention pertains.<br>
EXPERMENTAL.<br>
Using Mycobacterium smegmatis as a surrogate, we discovered a series of DARQs with potent in vitro activity against several mycobacteria (77). To date, 20 molecules of the DARQ series have a minimal inhibitory concentration (MIC) below 0.5 fig/ml against Mycobacterium tuberculosis H37Rv, and for three of these the anti-mycobacterial activity was confirmed in the in vivo mouse model.<br>
Structurally and mechanistically, DARQs are very different from both fluoroquinolones (including methoxyquinolones) and other quinoline classes, including mefloquine and its analogs, 4-methylquinolines and 4-quinolylhydrazones (12-16). One of the major structural differences between DARQs and other quinolone or quinoline classes is the specificity of the functionalized lateral (31) chain borne by the DARQ class. In addition, the lack of mycobacterial cross-resistance with existing chemical classes points to a different mechanism of action.<br>
The lead compound of the DARQs, hereinafter referred to as J or DARQ J(Fig.l), was found to have an unique spectrum of potent and selective anti-mycobacterial activity in vitro (Table 1). The median MIC, obtained for the laboratory strain H37Rv and six fully susceptible isolates was 0.060 jig/ml, versus 1.00 fig/ml for rifampin. J demonstrated similar in vitro efficacy against M. tuberculosis clinical isolates resistant to the first-line TB agents rifampin, streptomycin, ethambutol and pyrazinamide; and the second-line TB agent moxifloxacin. For eight clinical isolates resistant to isoniazid, the median MIC was 0.010 pig/ml. The lack of cross-resistance with currently used anti-TB agents suggested that J may retain activity against MDR-TB strains. Indeed, using the BACTEC™ culture system, a clear concentration-dependent inhibition of bacterial growth was seen when MDR-TB strains were exposed to fixed concentrations of J. Out of the 30 isolates of MDR-TB, 13 (43%) were found to be susceptible to 0.100 ug/ml of J and 17 (57%) were susceptible to 0.010 Jig/ml of i. A similar high degree of susceptibility (MIC below 0.010 jig/ml) was seen for only one of 10 additional fully drug-susceptible strains, when tested using the BACTEC™ system, while all strains were susceptible to 0.100 ng/ml of J.<br>
Potent activity was also demonstrated against other mycobacterial species including Mycobacterium bovis and Mycobacterium kansasii, as well as species naturally resistant to many other anti-TB jigents and involved in opportunistic infections, suchas Mycobacterium avium complex (MAC), Mycobacterium abcessus, Mycobacterium fortuitum and Mycobacterium marinum (Table 1).<br>
Surprisingly, the activity of J appeared to be specific for mycobacteria. J was barely active against species close to mycobacteria such as Corynebacterium (MIC 4.00 jig/ml) and Nocardia (MIC &gt; 4.00 u,g/ml) and not active against other organisms including Gram positive Streptococcus pneumoniae, Staphylococcus aureus—including methicillin-resistant strains (MIC &gt; 32 ng/ml)—and Enterococcus faecalis, or Gram negative Escherichia coli, Haemophilus influenzae, and Helicobacterpylori. Exposure of M. tuberculosis in log-phase growth to concentrations of J at 100 x MIC resulted in a 103 log reduction in bacterial counts after 12 days, indicating that J has bactericidal activity in vitro. The effect of J on stationary phase tubercle bacilli has not yet been studied.<br>
Isolation of mutants, cross-resistance and postulated drug target<br>
By investigating mycobacteria] resistance, we aimed to identify the molecular drug target and propose a mechanism of action. Resistant mutants of M. tuberculosis and M. smegmatis were derived by in vitro selection at inhibitory concentrations of J, in order<br>
to:<br>
quantify the proportion of resistant mutants in mycobacteria (with rifampin<br>
as a control)<br>
assess the resistance pattern of resistant mutants (including cross-/non-cross-<br>
resistance to quinolones)<br>
investigate the mechanism of action.<br>
From selection experiments, the proportion of mutants with reduced sensitivity to J was 5 x 107 and 2 x 10'8 at MIC x 4, and 5 x 10'8 and 1 x lO'8 at MIC x 8 for M. tuberculosis and M. smegmatis, respectively (supporting online text). In the case of M. tuberculosis, these proportions were comparable to those of mutants resistant to rifampin (10'7 to 10'8) and indicates that naturally-occurring resistance to J is rare. In addition, the sensitivity of the M. tuberculosis strains resistant to J remained unchanged to the anti-TB agents isoniazid, rifampin, streptomycin, amikacin, ethambutol and moxifloxacin. Further analysis of M. tuberculosis and M. smegmatis mutants with a reduced susceptibility to J showed that there were no mutations in the DN A gyrase regions gyrA and gyrB, sequences in which quinolone resistance typically develops. This confirms that the molecular target for J is different from that of the fluoroquinolones.<br>
One approach to determining the molecular target for J and inferring a mechanism of action is to identify and compare resistance-conferring mutations in sensitive and<br>
resistant strains of M tuberculosis and M. smegmatis. The genomes of the resistant M. tuberculosis strain BK12 and the two resistant M. smegmatis strains R09 and RIO, as well as the parental M. smegmatis, were sequenced to near completion. We identified resistance-conferring mutations by comparative analysis of the genome sequences of sensitive and resistant strains of M. tuberculosis andM. smegmatis (Fig. 2). We showed that the only gene affected in all three independent mutants versus corresponding parental wild-type encodes for atpE, a part of the FO subunit of ATP synthase. This suggests that atpE is responsible for resistance to J in the mutant strains, indicating that J inhibits a new M. tuberculosis target, the proton pump of ATP synthase.<br>
Complementation studies were performed to show that the mutant atpE gene is responsible for resistance to J and by direct inference that the atpE gene product is the target of J in mycobacteria. Given the fact that it is known that all genes of the ATP synthase operon have to be expressed in a coordinated way, i.e. all genes that encode the FO part have to expressed from the same location, we amplified the FO part of the operon from the resistant M. smegmatis strain (D32V) and selected clones that did not acquire additional mutations through the PCR process. Wild-type M. smegmatis was transformed with a plasmid containing the thus selected mutant FO fragment. This rendered the cells resistant to J with a MIC practically identical to that of the resistant strain M. smegmatis R09 (D32V),. In addition, when the plasmid was re-isolated from these transformants and the atpE gene, was sequenced, it was shown to have remained the mutant allele (D32V).<br>
The actual effect of DARQ J on ATP production in M. tuberculosis was further demonstrated by measuring the effect of J on the total cellular ATP present in the mycobacteria using the ATP Bioluminescence Luciferase Assay Kit HS n of Roche. This assay is based on the ATP driven conversion of D-Luciferin to Oxyluciferin that can be measured at 526 nm.<br>
Briefly, the effect of DARQ J on total ATP was tested in both wild type M, tuberculosis<br>
used as a positive control and isoniazid which is an inhibitor for biosynthesis of certain<br>
cell wall components, but has no effect on ATP production, was used as a negative<br>
control.<br>
As can be seen in Fig.3, treatment of the wild type M.tuberculosis with DARQ J, leads to dose dependent decrease in ATP production in these bacteria. In contrast isoniazid has no effect on ATP production. As already described hereinbefore, exposing these bacteria to high concentration of the DARQ J raised the diarylquinolines resistant mutants of M.tuberculosis. When these resistant M.tuberculosis were treated with DARQ J these bacteria did not show any decrease in the ATP production even at 100 times the Minimal Inhibitory Concentration (MIC) of this compound. In contrast DCCD was able to block the ATP production in these bacilli suggesting that DARQ J and DCCD have different binding pockets in ATP synthases.<br>
Computer modeling and identification of the DARQ J binding region<br>
To further investigate the different mode of action of DCCD and DARQ J, a computer generated 3D model of the ATP synthase for both the wild type and DARQ J mutant M. tuberculosis were generated. The atomic coordinates provided in Tables 4 and 5 were computed by preparing a model of the 3D structure of the published amino acid sequences P63691 and AJ865377. The actual DARQ J binding site was found to be located at the contact area of the A and C subunits, more in particular around Amino Acid 'Arg 210' of the A subunit and 'Glu 61' of the C subunit, as referred to in tables 3, 4 or 5. This fits nicely with the results seen for the screening of resistance-conferring mutations in sensitive and resistant strains of M. tuberculosis and M. smegrmtis, described above.<br>
The model is based on optimization of the relative placement of the A- and C helices of the ATPase structure and the orientation of the amino acids back-bone and side-chain towards minimal computed internal strain. The geometry was obtained by means of a number of molecular dynamics simulation cycles and molecular mechanics relaxations starting from previously published general helix geometry of a different organism [E-Coli PDB entry code 1C17 - V.K. Rastogi and M.E. Girvin, Nature, 402, 263-268 (1999)]. Molecular dynamics and geometry relaxation were both performed with a forcefield parameterization based on MMFF94s [Halgren, T.A. (1996), J. Comput. Chem., 17,490-519], but any state of the art molecular dynamics software could be employed[[Berendsen. HJ.C., van der Sgoel,D. andvanDrunenjl., Comp. Phys. Comm. 91 (1995), 43-56; Lindahl, E,, Hess, B. and van der Spoel, D., J. Mol. Mod. <br>
 (2001) 306-317.] followed by a suitable geometry optimization [J. W. Ponder and F. M. Richards, J. Comput. Chem., 8,1016-1024 (1987).].<br>
The computed coordinates in the tables 3,4 and 5 comprise the part (with 30 Angstrom radius volume) of the predicted structure of the region considered relevant for inhibiton of the MTB ATPase activity in these enzymes, based on the proposed mode of inhibition and on the occurrance of resistance inducing point mutations in biological assays.<br>
Discussion<br>
The DARQ J is a member of a new chemical class of anti-TB agents with an MIC equal to or lower than that of reference compounds. Its spectrum is unique in its specificity to mycobacteria, including atypical species important in humans; MAC, M. kansasii and the fast growers M.fortuitum and M. abscessus. This anti-mycobacteria]-specific spectrum differs from that of isoniazid, which has no activity against MAC. The clinical use of J will be highly targeted to the treatment of TB and mycobacterial infections. The inability of J to inhibit non-mycobacteria should translate into less selective pressure and a lower risk of resistance developing in other bacterial species, when compared with antibiotics with broader spectra (9).<br>
The target and mechanism of action of J is different from that of other anti-TB agents. A comparison of the sequences of ATP synthases of different bacteria and of eukaryotic ATP synthase, and in particular of the C chain of the FO subunit of the ATPase complex, together with the 3D modelling of the ATP synthases of the wild type and mutant M. tuberculosis, provides a rationale for the specificity of the antibacterial spectrum and, to a lesser extent, the safety profile.<br>
The dynamics study performed on the constructed Mycobacterium ATPase models show that in these structures a cavity (the binding site according to the atomic coordinates of Table 3) exists on the contact area of the A and C subunits (around Amrno Acid 'Arg 210' of the A subunit and 'Glu.611 of the C subunit). The tables 4 and 5 provide the coordinates of two studied variants of the atoms surrounding this site, and their average position, The DARQ J inhibitor is able to interfere with the normal proton transfer step involving these two amino acids by prohibiting these two amino-acids to interact. The stereospecifity of DARQ J can be understood from the asymmetry of the _predlcted_binding site: theactiye chiral enantiomer accommodates this; cavity optimally other forms of the compound and variants of ATPase are less well matched.<br>
The binding site we have derived here is on an entirely different part of the ATPase system than DCCD (DARQs are in the membrane part of the enzyme, DCCD binding occurs approximately 90 Angstrom away inside the cell; based on the PDB structure of a Bovine ATPase crystal "1E79" - published in C.Gibbons, M.G.Montgomery, A.G.W. Leslie, J.E. Walker, Nat.Struct.Biol.,7,1055 (2000). Therefore the atoms that coujd potentially be involved with DCCD type inhibition of MTB ATPAse are not in the region listed in the coordinate tables of the binding site (which only just spans the membrane portion of the enzyme) and, reciprocally, this part of the enzyme involved in the DARQ J mode of inhibition is not present in the published "1E79" structure (which only shows the intracellular part). This difference in binding site may explain the different response of M. tuberculosis observed in Cthe in vitro ATP production assay.<br>
Notwithstanding the above, older studies with DCCD on mitochondrial ATPases suggest another binding site located around an acidic amino-acid in a lipophyllic environment of the FO region of the enzyme e.g. by Sebald W, Machleidt W, Wachter E., Proc Natl Acad Sci USA. .1980 Feb;77(2):785-789. This binding position on mitochondrial ATPases can be considered analogous to the one described for the Mycobacterium species here.<br>
At the same time, targeting a new mechanism ensures that currently circulating TB strains with resistance mutations to available treatments are not cross-resistant to J. It is obvious from our in vitro studies that J has at least as high anti-bacterial effect against MDR-TB isolates, even against those with a broad four-drug resistance, as against normal wild type pan-susceptible strains of M. tuberculosis. This observation is important since it clearly demonstrates that there is no cross-resistance with existing anti-TB drugs. Given the further identification of the binding pocket in the membrane part of the ATPase, the results of this study will allow further development of new antibacterial compounds, in particular anti-mycobacterial compounds targeting ATP synthesis in these organisms.<br>
References<br>
1.	Global Alliance for TB Drug Development, Developing a faster TB cure (2004;<br>
http://www.tballiance.org').<br>
2.	E. L. Corbett el al., Arch. Intern. Med. 163, 1009 (2003).<br>
3.	UN AIDS, AIDS epidemic update 2005(2003;<br>
www.unaids.org/Unaids/EN/Resources).<br>
4.	World Health Organization, Tuberculosis (2004;<br>
http://www.who.int/health topics/tuberculosis/en/).<br>
5.	R. J. O'Brien, P. P. Nunn, Am. J. Respir. Crit. Care Med. 163,1055 (2001).<br>
6.	World Health Organization, Tuberculosis Fact Sheet No 104 (2004;<br>
http://www.who.int/rncdiacentre/factsheets/fsl04/en/).<br>
7.	A. J. Claxton, J. Cramer, C. Pierce, Clin. Ther. 23, 1296 (2001).<br>
8.	N. Lounis et al., Antimicrob. Agents Chemother. 45, 3482 (2001).<br>
9.	A. S. Ginsburg, J. H. Grosset, W. R. Bishai, Lancet Infect. Dis. 3,432 (2003).<br><br>
10.	C. K. Stover et al, Nature 405, 962 (2000).<br>
11.	Guillemont J, Emile G, Patent (International Publication Number. WO<br>
2004/011436. International Publication Date 5 February, 2004).<br>
12.	Barbachyn MR, Brickner SJ, Patent (Internationa] Publication Number:<br>
WO 93/09103, International Publication Date: 13 May 1993, International<br>
Application Number: PCT/US92/08267, International Filing Date: 1992).<br>
13.	R. Jain, B. Vaitilingam, A. Nayyar, P. B. Palde, Bioorg. Med. Chem.<br>
Lett. 13, 1051 (2003).<br>
14.	C. M. Kunin, W. Y. Ellis, Antimicrob. Agents Chemother. 44, 848<br>
(2000).<br>
15.	L. Savini, L. Chiasserini, A. Gaeta, C. Pellerano, Bioorg. Med Chem 10,<br>
2193(2002).<br>
16.	S. Vangapandu, M. Jain, R. Jain, S. Kaur, P. P. Singh, Bioorg. Med.<br>
Chem. 12, 2501 (2004).<br>
17.	K. Andries et al, Science 307, 223 (2005).<br>
(Table Removed)<br><br><br><br><br><br>
WHAT IS CLAIMED IS:<br>
1.	A binding site in the FO part of an ATPase characterised in that it comprises the<br>
resistance-conferring mutation sites of an atpE protein.<br>
2.	A binding site according to claim 1 wherein the resistance-conferring mutation<br>
sites refer to amino acids 14 to 34 and to amino acids 53 to 69 of an atpE<br>
protein, taking the numbering of Mtb_S (SEQ ID No.l) or of Mtb_R. (SEQ ED<br>
No.2) as a reference.<br>
3.	A binding site according to claim 1 comprising at least the amino acids Ala24,<br>
Gly27, Phe53, Val57, Gly58, Glu61, Tyr64 and Phe65 of one C subunit and the<br>
amino acids Ser182, Leu183, Leu185, Arg186 of one A subunit.<br>
4. A binding site according to claim 4 wherein said amino acids have the atomic coordinates of any of Tables 3, 4 or 5 or homologous structure coordinates comprising a root mean square deviation of non-hydrogen atoms of less than about 1.5A.<br>
5.        A binding site according to any one claims 1 to 4 identifying compounds that interact with the F0 part of an ATpase and to their potential as anti microbial compounds.<br>
6.        A method to evaluate the potential of a test compound to interact with an atpE protein, said method comprising;<br>
-	molecular modeling techniques to generate the three-dimensional structure of a<br>
binding site according to any of claims 1 to 4;<br>
-	employing computational means to perform a fitting operation between the test<br>
compound and the three-dimensional structure of the binding site; and<br>
-	analyzing the results of said fitting operation to quantify the association of the<br>
test compound with the three-dimensional structure of the binding site.<br>
7.	A method according to claim 6 wherein the three dimensional structure is<br>
generated with at least the amino acids Ala24, Gly27, Phe53, Val", Gly58, Glu61,<br>
Tyr64 and Phe55 of one C subunit and the amino acids Ser182, Leu183, Leu185,<br>
and Arg186 of one A subunit, using the atomic coordinates of any of Tables 3,4<br>
or 5 or homologous structure coordinates comprising a root mean square<br>
deviation of non-hydrogen atoms of less than about 1.5A.<br>
8.	A method according to claim 6 wherein the three-dimensional structure is<br>
generated using the atomic coordinates of the amino acids Ala21, Gly25 of the A<br>
Chain of any of Tables 3, 4 or 5; the amino acids Ala24, Gly27, Phe53, Phe54,<br>
Val57, Gly58, Glu61, Tyr64, Phe65 of the K Chain of any of Tables 3, 4 or 5; the<br>
amino acids Met17, Gly19, Gly20, Ala21, De22, Gly23, Ala24, Gly25, He26, Gly27,<br>
Asp28, Gly29, Ala31, Phe53, Thr56, Val57, Gly58, Leu59, Val60, Glu", Ala62,<br>
Ala63/Profi3, Tyr64, Phe65 of the L Chain of any of Tables 3, 4 or 5; and of the<br>
amino acids Ser'82, Leu183, Leu185, and Arg186 of the M Chain of any of Tables<br>
3, 4 or 5.<br>
9.  A method according to claim ^wherein the resistance-conferring mutation sites refer to amino acids 14 to 34 and to amino acids 53 to 69 of an atpE protein, taking the numbering of Mtb_S (SEQ ID No.l) or of Mtb_R (SEQ ID No.2) as a reference.<br>
10.A medicament comprising DARQJ for treating a bacterial infection provided that said infection is other than a mycobacterial infection.<br>
11. An isolated mutant atpE protein wherein the mutation consists of at least one point mutation located in any one of amino acids 20 to 40, in particular 30 to 40 or of amino acids 60 to 75, in particular 62 to 73 as shown in the sequence alignment of Fig.2.<br>
12.   An isolated mutant atpE protein according to claim 11 wherein the mutation consists of at least one point mutation, in amino acid 34 or amino acid 69 as shown in the sequence alignment of Fig.2<br>
13.An isolated mutant atpE protein according to claim 12. selected from Mtb_R (SEQ ID No. 2), Msm_R09 (SEQ ID No.4), Msm_R10 (SEQ ID No.5) and an amino acid sequence having at least 70, 80, 90, 95, 97 or 98% sequence identity to Mtb_R, Msm_R09 or Msm_R10.<br>
14.      An isolated nucleic acid sequence encoding an isolated mutant atpE protein according to any one of claims 11 to 14.<br>
15.      A vector comprising the nucleic acid sequence according to claim 14. <br>
16.      A host cell carrying the vector according to claim 15<br>
17.      A method for identifying an anti-microbial compound, said method comprising<br>
a)	contacting cells expressing an atpE protein with a test compound under<br>
physiological conditions, and<br>
b)	determining whether the test compound interacts with the atpE protein<br>
18.A method according to claim 17, wherein the atpE protein is selected from the group consisting of;<br>
SwissProt entry Q10598 or a protein having at least 70, 80, 90, 95, 97 or 98% sequence identity to SwissProt entry Q 10598; and<br>
a mutant atpE protein according to any one of claims 1 to 3.<br>
19.A method to identify whether a test compound binds to an isolated atpE protein, said method comprising;<br>
a)	contacting cells expressing an atpE protein wherein such cells do not<br>
normally express said atpE protein, with the test compound in the presence and<br>
absence of a compound known to bind the atpE protein,<br>
b)	determine the binding of the test compound to the atpE protein using the<br>
compound known to bind to the atpE protein as a reference.<br>
20.     A method according to claim 19 wherein the atpE protein is selected from the group consisting of;<br>
SwissProt entry Q10598 or a protein having at least 70, 80, 90, 95, 97 or 98% sequence identity to SwissProt entry Q10598; and a mutant atpE protein according to any one of claims 16 to 18.<br>
21.      A method according to claim 19, wherein the compound known to bind to the atpE protein is detectably labeled, and consists of (J).<br>
22 A method according to claim 19 wherein step a) consists of contacting a cellular composition comprising an atpE protein, with the test compound in the presence and absence of a compound known to bind the atpE protein<br>
23.A method according to claim 22 wherein the cellular composition consists of a membrane preparation obtained from the cell according to claim 21.<br>
24.      A method for evaluating the potential of a test compound to interact with an atpE protein, said method comprising;<br>
a)	using molecular modeling techniques to formulate a three-dimensional<br>
structure of the atpE protein,<br>
b)	employing computational means to perform a fitting operation between the<br>
test compound and the three-dimensional structure of the atpE protein, and<br>
c)	analyzing the results of said fitting operation to quantify the association of the<br>
test compound with the three-dimensional structure of the atpE protein.<br>
    25.      A method according to claim 24 wherein the three-dimensional structure of the atpE protein is generated using the atomic coordinates of the He28, Glu61 and Ile63 of E. coli (Protein Database 1Q01) +/- a root mean square deviation of the backbone atoms of said amino acids of not more that 10A.<br>
26.      A method according to claim 24" wherein the three-dimensional structure of the atpE protein is generated using the atomic coordinates according to tables 6 or 7 +/- a root mean square deviation of the backbone atoms of said amino acids of not more that 1.5 A.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUFic3RyYWN0LSgwNC0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Abstract-(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNsYWltcy0oMDQtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Claims-(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNsYWltcy0oMjEtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Claims-(21-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Correspondence Others-(05-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Correspondence Others-(14-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Correspondence Others-(16-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Correspondence Others-(21-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Correspondence-Others-(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LURyYXdpbmdzLSgyMS0wOC0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Drawings-(21-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUZvcm0tMS0oMDQtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Form-1-(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUZvcm0tMi0oMDQtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Form-2-(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUZvcm0tMy0oMTQtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Form-3-(14-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LUdQQS0oMDQtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-GPA-(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LVBldGlvbi0xMzctLSgwNC0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Petion-137--(04-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI3Mi1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMDQtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2272-delnp-2007-Petition-137-(04-09-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263206-a-bimorph-mirror-with-two-piezoelectric-layers-separated-by-a-central-core-of-semirigid-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263208-process-for-producing-imidazo-1-2-b-pyridazine-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263207</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2272/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>14-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Mar-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>JANSSEN PHARMACEUTICA N.V.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TURNHOUTSEWEG 30, 2340 BEERSE, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JEAN-MARC EDMOND FERNAND MARIE NEEFS</td>
											<td>C/O JANSSEN PHARMACEUTICA N.V.,TURNHOUTSEWEG 30, 2340 BEERSE, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JOHANN WINKLER</td>
											<td>C/O JANSSEN PHARMACEUTICA N.V.,TURNHOUTSEWEG 30, 2340 BEERSE, BELGIUM</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARC RENE DE JONGE</td>
											<td>C/O MOLMO SERVICES BVBA, CAMPUS BLAIRON 424, 2300 TURNHOUT, BELGIUM</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LUCIEN MARIA HENRICUS KOYMANS</td>
											<td>C/O MOLMO SERVICES BVBA, CAMPUS BLAIRON 424, 2300 TURNHOUT, BELGIUM</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KOENRAAD JOZEF LODEWIJK ANDRIES</td>
											<td>C/O JANSSEN PHARMACEUTICA N.V.,TURNHOUTSEWEG 30, 2340 BEERSE, BELGIUM</td>
										</tr>
										<tr>
											<td>6</td>
											<td>HINRICH WILHELM HELMUT GOHLMANN</td>
											<td>C/O JANSSEN PHARMACEUTICA N.V.,TURNHOUTSEWEG 30, 2340 BEERSE, BELGIUM</td>
										</tr>
										<tr>
											<td>7</td>
											<td>PETER KAREL MARIA VERHASSELT</td>
											<td>C/O JANSSEN PHARMACEUTICA N.V.,TURNHOUTSEWEG 30, 2340 BEERSE, BELGIUM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/35</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/054893</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-09-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04104720.0</td>
									<td>2004-09-28</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/620,500</td>
									<td>2004-10-20</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263207-an-isolated-mutant-atpe-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:55:19 GMT -->
</html>
