<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263308-method-of-mutagenesis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:29:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263308:&quot;METHOD OF MUTAGENESIS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD OF MUTAGENESIS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method of mutagenesis for introducing mutations into a molecule and, in particular, to methods that may be applied to populations of molecules for the generation or screening of libraries involving the mutation of multiple molecules.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHOD OF MUTAGENESIS<br>
The present invention relates to a method of producing mutagenised nucleic acid molecules. In particular, the present invention relates to a method in which the primer used in the mutagenesis reaction (the mutagenic primer) is, itself, synthesised from a template nucleic acid molecule. The output of the mutagenesis reaction is a mutagenised circular nucleic acid molecule which can be e.g., transformed into bacteria without the need for additional modification. In preferred embodiments, the method is applied to populations of molecules, for example in the creation of or screening of populations of molecules. The method may also be used in a combinatorial manner, in which multiple mutagenic primers are used to mutate multiple parent molecules.<br>
One of the steps involved in molecular biological techniques is the introduction of nucleic acid into a vector. For example, a DNA fragment or population of DNA fragments may be digested and introduced by ligation into a vector. For example, a DNA fragment (or population of fragments) may be digested and removed from a first vector (e.g, a plasmid) into a second vector in a process known as sub-cloning. In this way, a first vector is digested with the same restriction enzymes as the second vector so as to produce complementary overhanging ends for ligation. The DNA fragment and the second vector are then ligated by virtue of their complementary overhanging ends and the result is a completed vector, containing the DNA fragment, that can replicate in a host cell (e.g in bacteria).<br>
Hence large quantities of DNA may be produced for use in various molecular biological applications. Or, the vector into which the DNA has been introduced may have a particular purpose (e.g. may be an expression vector). Once introduced into such an expression vector, protein encoded by the restriction fragment may be expressed from the vector in the host cell. Once expressed, the resultant protein could be purified for its end use. Or, particularly where the method is used to incorporate a population of DNA fragments into an expression vector, the expressed protein may be used in a screen (e.g., by binding affinity to a target molecule).<br>
Various methods of introducing mutations into DNA are also known. It is known from the work of Barik (Bank, Methods in Molecular Biology Vol. 192: PCR Cloning Protocols, 2nd edition p189 - 196, Eds B-Y Chen and H.W Janes @ Humana Press Inc, Totowa, NJ; Burke and Barik, Methods in Molecular Biology Vo. 226: PCR Protocols, 2nd Edition p 525-531, Eds JMS Bartless and D Stirling @ Humana Press Inc, Totowa, NJ) to use a PCR-based approach to introduce mutations into linear DNA. According to these methods, so-called megaprimers have been used to introduce mutations into a linear template in a PCR reaction. However, such methods result in a linear, double-stranded DNA product that must itself be sub-cloned into an appropriate vector in order to produce a replicative form for e.g., scale up, or for protein expression.<br>
Miyazaki proposed a method involving PCR of a whole plasmid (Miyazakl, Methods in Molecular Biology: Directed Evolution Library Creation: Methods and Protocols. Eds Arnold and Georgiou, Humana Press Inc. Totowa, NJ; BioTechniques 33: 1033-1038). In this method, a double-stranded plasmid is used as the template in a PCR reaction and to which a double-stranded PCR product, containing the mutation to be introduced, is added for priming. However, the double-strandedness of the primer and template can cause the primer strands and the template strands to self-anneal rather than to cross-anneal and therefore a specific ratio of primer to template is used to minimise the self-annealing.<br>
In addition to the problems of self-annealing that result from a double-stranded template, there are further problems when a double stranded template is used in a PCR reaction. In an exponential reaction the capacity for errors being introduced is high as an error introduced in one round is carried forward into each of the further rounds on an exponential basis. Furthermore where this PCR template is a large template such as a vector, e.g., a plasmid, then this further increases the likelihood of errors being introduced.<br>
It is also known to use synthetic oligonucleotides to mutate DNA (Kunkel et al.,(1985) PNAS USA, Vol 82, pp 488-492; Sidhu et at, Phage Display for the Selection of Novel Binding Peptides). However, there are limitations as to the<br>
length of the oligonucleotides that can be chemically synthesised. In addition, to program the synthesiser, the sequence must be known and inputted in advance into the synthesiser for its production.<br>
There is therefore a need for alternative methods of mutagenesis. In particular, for methods that produce an output that can be directly transformed into bacteria, ready for downstream use, such as in expression screening (avoiding the need for additional digestion and ligation steps to introduce nucleic acid into an appropriate vector). There is also a need for fast, efficient mutagenesis methods and for methods that are amenable to scale-up, e.g., for use in the production of populations for screening or production of libraries.<br>
The present invention, in one aspect, provides a method of introducing one or more mutations into a nucleic acid molecule, comprising: annealing a single-stranded form of a mutagenic primer to a parent molecule, said mutagenic primer having been synthesised from a template nucleic acid molecule and containing one or more mutations relative to said parent molecule; and synthesising a complementary strand from said mutagenic primer so as to produce a circular-form nucleic acid molecule containing said mutations.<br>
In one aspect, the present invention provides a method of introducing one or more mutations Into a nucleic acid molecule, comprising the following steps:<br>
(a)	synthesising a mutagenic primer from a template nucleic acid molecule,<br>
wherein said mutagenic primer contains one or more mutations relative to<br>
a parent molecule;<br>
(b)	separating a single-stranded form of said mutagenic primer;<br>
(c)	annealing said single-stranded mutagenic primer to a single-stranded<br>
form of said parent molecule;<br>
and<br>
(d)	synthesising a complementary strand from said mutagenic primer so as<br>
to produce a circular-form nucleic acid molecule containing said mutations.<br>
Thus, the mutagenic primer is synthesised from a template nucleic acid molecule using a suitable enzyme, using methods known in the art. It is preferred that said mutagenic primer is synthesised using a polymerase chain reaction (PCR)-based method.<br>
The template molecule and the parent molecule may be the same, or may contain the same sequence, which forms the basis of the synthesis of the mutagenic primer. Thus, in some embodiments, the sequence of the template for the mutagenic primer synthesis is the same as that found in the parent molecule. In these embodiments, the mutations may be introduced by virtue of the introduction of errors during the synthesis of the mutagenic primer from the template molecule, e.g., by means of an error-prone polymerase. Thus, In some embodiments, the mutagenic primer Is synthesised by a synthesis method in which errors are introduced when synthesisihg a second strand from a first strand, preferably by error-prone PCR (suitable methods are known to the person of skill in the art, e.g, Molecular Cloning: a Laboratory Manual: 2* edition , Sambrook el al, 1989 Cold Spring Harbour Laboratory Press).  Various kits are available  in  the art for carrying out error-prone  PCR,  such  as  Diversify (BDBiosciences 63070).   In another approach, the template molecule may be transformed into an E.coli strain (suitable strains are available in the art, e.g., XL1-RED (Stratagene)) which introduces errors into the DNA sequence.   Other methods for introducing errors into the mutagenic primer when produced from the template will be known to the person of skill in the art.<br>
The template and the parent molecule may be different molecules, i.e., may contain regions in which there is a difference in sequence between the parent molecule and the template molecule respectively. Thus, when the mutagenic primer is synthesised from the template, it contains mutations in comparison with the parent molecule (in this situation, a reaction which introduces errors, e.g., error-prone PCR, is not needed).<br>
The person of skill in the art knows of suitable reagents and reaction conditions for the synthesis of the mutagenic primer from the template (see, for example, Molecular Cloning: a Laboratory Manual: 2nd edition, Sambrook et al., 1989 Cold<br>
Spring Harbour Laboratory Press). Where the mutagenic primer is synthesised using the polymerase chain reaction, suitable primers are chosen by the person of skill in the art, having regard to the template being used. The primers used to produce the mutagenic primer are referred to herein as the initial primers. These initial primers comprise a forward and a reverse primer, which are generally based on the ends of the sequence to be amplified, i.e., at the ends of the sequence that will be the mutagenic primer. The initial primers are generally synthetic oligonucleotides, produced by any standard methods using an oligonucleotide synthesiser. These initial primers may be of any suitable length, for example around 15 to 100 nucleotides in length, e.g., around 15 to 75 nucleotides in length, around 15 to 50 nucleotides in length, generally 18 to 25 nucleotides in length. The skilled person can choose or design suitable initial primers, depending on, e.g., the sequence of template molecule, the size of the resultant mutagenic primer and/or the application of the method.<br>
A further advantage of the present invention arises from the fact that the mutagenic primer is synthesised from a template nucleic acid molecule, so that the sequence of the mutagenic primer can be unknown to the user. For example, where the mutagenic primer is produced by PCR using initial primers directed to a known part of the template, the region amplified by these initial primers may be unknown. Thus in some embodiments, the sequence of the mutagenic primer is unknown prior to said annealing. This has particular application where the method of the invention is applied to libraries (discussed further below), but is not limited thereto. There are many other applications where a particular sequence, in sub-cloning is unknown, but the boundaries of the vector in which it is contained are known (one example would be in "shotgun" cloning). The skilled person, can select suitable initial primers based on his knowledge of the parent molecule.<br>
The molecule defined in step (b) is thus a single-stranded mutagenic primer that has been isolated from its complementary strand and the terms separated and isolated are used interchangeably with respect to the single-stranded mutagenic primer. It is preferred that this separation or isolation of the single-stranded mutagenic primer from its complementary strand is carried out using a separating<br>
medium (such as beads.or a separating column) as; described below. In this manner a binding moiety may be introduced in to one of the strands of the mutagenic primer when it is synthesised, thus allowing one of the strands of the mutagenic primer to be isolated from the other e.g., in a capture step as described below.<br>
It is preferred that the single-stranded primer defined in step (b) is separated by use of a separating medium. For example, the step of synthesising the mutagenic primer may introduce a binding moiety in either the positive strand or negative strand of the mutagenic primer and the separation of the mutagenic primer into its positive and negative strands may then be carried out via said binding moiety binding to its binding partner on a separating medium. The separating medium may be any suitable solid phase medium, such as beads or a column (e.g., beads or column to which is attached the binding partner).<br>
More particularly, the binding moiety may be introduced into the positive strand of the mutagenic primer (e.g., by using an initial forward primer in the mutagenesis reaction that incorporates a binding moiety) and the negative strand of the mutagenie primer may then be eluted after the positive stranded form binds to the separating medium. It is preferred that the binding moiety is biotin and said binding partner is streptavidin. Other suitable binding moieties and binding partners will be known to persons of skill in the art and include fluoroscein and digoxin, and antibodies (including fragments and derivatives) and antigens.<br>
The skilled person is well aware of other binding moieties and binding partners that could be used. For example, in addition to streptavidin, other binding partners that bind to biotin are available such as avidin and neutravidin. The skilled person can select suitable binding moieties and binding partners to use in the capture step involved in strand separation, in addition biotin and/or streptavdin analogues may be used (see e.g., J Mol Biol (1994) Sep 30: 242 (4):559-65)). Biotin and streptavidin are most preferred.<br>
For example a particular antigenic moiety could be incorporated into one of the strands of the mutagenic primer during its synthesis, in a similar manner as that<br>
described for the biotin example mentioned above. Then one strand could be separated from the other using a column containing antibodies to the antigen.<br>
Another approach is to use a separating system that contains proteins that bind to specific DNA sequences. For example it is known in the art that Haelll methylase binds to a specific 5' DNA sequence, another example of protein that binds to a specific DNA sequence is P2A. For a discussion of such an approach see, Bertschinger and Neri "Protein, Engineering, Design and Selection (2004), Volume 17, page 699.<br>
The binding partner and binding moiety should be of sufficient affinity to withstand the treatments that is used for separation of the DNA strands. For strand separation high or low pH may be used. Where low pH is used this is preferably £ pH 3, more preferably £ pH 2, even more preferably  pH 11, more preferably 2 pH 12, even more preferably &gt; pH 13. Preferred alkaline agents include sodium hydroxide, and other Group I hydroxides such as LiOH or KOH. Group II hydroxides apart from Be(OH)2 and Ba(OH)2 could also be used. It is most preferred that the strands are separated using sodium hydroxide (e.g., 0.1 N NaOH, which has a pH of 13). Appropriate agents and conditions can be selected by the person skilled in the art. Most preferred is the use of biotin and streptavidin and/or their analogues and high pH strand separation treatment as defined above.<br>
Chemical groups could also be used to chemically attach one of the strands to a support. For example it is possible for oligonucleotides to be synthesised so as to incorporate a particular chemical group such as an amine or a thiol group. Indeed such oligonucleotides can be purchased from suppliers. Where such oligonucleotides are used as one of the initial primers in the reaction of the invention then the relevant chemical group will be incorporated into the relevant strand of the resultant mutagenic primer. The separation of the strands can then be brought about by coupling of these chemical agents (that are now incorporated into the DNA strand) to amine-linking agents or thiol linking agents,<br>
by way of example, and suitable agents are available to the person of skill in the art.<br>
After the single stranded form of the mutagenic primer has been generated as<br><br>
described above, it is then introduced to the parent molecule to carry out the annealing step (c). In this way, the single-stranded form of the mutagenic primer is then added to the parent molecule for use in the annealing reaction.<br>
In step (c), the annealing of the mutagenic primer to the parent molecule can use any suitable conditions and reagents, depending on for example the size of the mutagenic primer. The selection of appropriate conditions is understood by the person of skill in the art. By way of example, a molar ratio of 3:1 mutagenic primer to template may be used. Suitable reaction conditions are incubation at 90°C for 2mins, 50°C for 3mins and 20°C for 5mins. The conditions can be varied if necessary by the person of skill in the art.<br>
The circular-form nucleic acid molecule resulting from step (d) Is also referred to herein as the "output of the mutagenesis reaction" (or, in short-form, as the "output"). Preferably, the output of the reaction is replicative, meaning that on transformation into a host cell it can replicate to produce copies of itself. This has the advantage that the output of the mutagenesis reaction can be directly transformed into a host cell (such as a host cell for the purpose of expression of protein from the nucleic acid) without the need for further manipulations, e.g., without needing to excise the relevant portion of the nucleic acid with restriction enzymes and ligate into a further plasmid.<br>
Alternatively, or additionally, It Is preferred that the output of the mutagenesis reaction is double-stranded, to allow for digestion with restriction enzymes.<br>
It is most preferred that the circular-form molecule (output of the mutagenesis reaction) is covalently closed circular DNA, preferably double-stranded.<br>
As noted, step (d) produces a circular reaction product from the parent molecule. Such a reaction involves the priming of complementary strand synthesis from the<br>
annealed mutagenic primer, such that resultant complementary strand contains the mutations from the mutagenic primer. In general, the reaction mixture will contain suitable amounts of the nucleotide bases, DNA polymerase, DNA ligase and an appropriate buffer (the examples give suitable amounts). This is incubated under appropriate conditions for second-strand synthesis to be complete. Such methods are known to the person of skill in the art. It is preferred that this reaction be of a non-amplification type (e.g., a non-PCR-type reaction) so that it does not involve successive cycles of amplification and it could therefore be described as a linear reaction (i.e., one complementary strand is produced per parent molecule single-strand). For example, in the method of the invention, the mutagenesis reaction on the single-stranded parent molecule produces a second single strand which is opposite in sense to the single-strand of the parent molecule. Thus, where the single stranded parent molecule is a positive strand, the second strand synthesised in the mutagenesis reaction will a negative strand.<br>
By using a single-stranded template and a single-stranded primer in the mutagenesis reaction, problems associated with self-annealing are avoided. Moreover, the output of the mutagenesis reaction is double-stranded and as such transforms bacteria more efficiently than a single-stranded molecule. Therefore, when transformed into a bacterial host, the mutated double-stranded form will transform bacteria in preference to the single-stranded template (effectively a selection of the mutated form over the non-mutated template).<br>
The method of the invention comprises the steps listed above (herein, the term comprise is used in the sense of include, i.e., allowing the presence of one or more further features, e.g., reaction steps). Thus the method of the invention may include further steps, including but not limited to those steps mentioned elsewhere herein. The method of the invention may, alternatively, consist of the steps listed above, or mentioned herein. The steps of the reaction may be combined into single reaction vessels, and it is not necessary for each step to be carried out in a separate vessel. For example, the step of annealing the mutagenic primer to the parent molecule and the step of synthesising a complementary strand may be carried out in one vessel.<br>
Preferably, the single-stranded parent molecule is single-stranded circular DNA. For example, the single-stranded parent molecule may be single-stranded circular DNA that is extracted from phage particles.<br>
Preferably in the method of the invention, prior to said annealing step (c), there is the separate step of adding the single-stranded parent molecule to the single-stranded primer. More preferably, the method further comprises the step of isolating said single-stranded parent molecule prior to step (c).<br>
The order of the steps relating to the mutagenic primer and the parent molecule, respectively need not be consecutive. Thus, the single-stranded form of the mutagenic primer may be separated before the single-stranded form of the parent molecule is isolated. Or, the single-stranded form of the parent molecule may be isolated before the single-stranded form of the mutagenic primer is separated. Or, the separation of the single-stranded mutagenic primer and the isolation of the single-stranded parent molecule, respectively, may be carried out side-by-side, or simultaneously.<br>
It is preferred that the method of the invention further comprises a size exclusion step performed on said single-stranded parent molecule prior to step (c). For example, the single-stranded parent molecule is passed through a porous resin, any small fragments are trapped in the resin whereas the larger parent molecule passes through. This step removes short linear fragments of DNA (thought to be degradation products that co-purify with the template). Removal of these fragments improves the mutagenesis efficiency by avoiding priming events from the short fragments that do not lead to mutagenesis.<br>
The parent molecule will contain the necessary sequence elements for the downstream use of the output of the method. Suitable vectors, for use as the parent molecule, are available in the art or may be constructed by the skilled person, using conventional techniques. Such vectors comprise appropriate regulatory sequences, including promoter sequences, termination sequences, enhancer sequences, marker genes etc. The vectors may be of any suitable<br>
type, e.g., plasmid, phage, phagemid etc. For example, where the output of the method is to be used in protein expression (for example, for screening of the expressed product) then the parent molecule will contain the necessary sequences for protein expression in a host cell.<br>
Suitable expression systems are well known in the art, and a variety of host cells may be used. These include, bacteria, yeast, insect and mammalian cells (e.g., HeLa, CHO, or BHK cells). Bacterial cells are preferred, most preferably E.coli. Suitable vectors for each of these hosts are available in the art, or may be constructed by routine methods. Viral expression systems may also be used, such as baculovirus systems or plant viral expression systems. Techniques for the manipulation of DNA are well known in the art (e.g., cloning, sequencing, expression and analysis etc). Further details of known methods can be found in. e.g., Molecular Cloning: a laboratory manual: 2nd edition, Sambrook et al., 1989. Cold Spring Harbor Laboratory Press; or Current Protocols in Molecular Biology, second edition, Ausubel et al eds. John Wiley &amp; Sons, 1992.<br>
Preferably, the method of the invention further comprises the step of transforming the mutagenised_nuclec acid molecule into host cells,. Transformation may be by any suitable method and these are known to the skilled person. Examples include the heat shock method. It is preferred that said transforming is performed using electroporation. The step of transforming the mutagenised product may be carried out directly after step (d), or there may be intermediate steps between step (d) and the transformation into a host cell. In a preferred embodiment discussed below, transformation into a host cell is used to preferentially select the mutated complementary strand over the non-mutated parental molecule.<br>
The method of the invention may comprise a step carried out under conditions such that the complementary strand containing the mutations is preferentially selected. Such preferential selection of the complementary strand may result from the preferential digestion of the parent molecule, or from the preferential survival of the complementary strand in a host cell.<br>
The parent molecule may contain modifications such that the complementary strand is preferentially selected. The method of the invention may further comprise the step of introducing said modifications. In preferred embodiments, the modifications in the parent molecule include the introduction of dU in place of<br>
i<br>
dT. Where the method further comprises the step of introducing such mutations, it is understood that this includes embodiments in which the parent molecule contains such modifications initially and further modifications are introduced in this step and embodiments in which there are no modifications initially and any such modifications are introduced by this step.<br>
The situation in which dU replaces dT may be explained as follows. The parent molecule is grown in an E.coli strain that supports the introduction of uracil, e.g. a duf ung strain such as CJ236 (Kunkel et al.,(1985) PNAS USA, Vol 82, pp 488-492;   Sidhu et al., Phage Display for the Selection of Novel Binding Peptides, Methods in Enzymology 2000, Volume 238, page 333-363). According to these protocols the parent molecule is multiplied in E.coli in phagemid form.   Single-stranded uracil-containing DNA is extracted via standard methods (e.g., the Qiagen M13 Spin Prep Kit).   Following the annealing of the mutagenic primer to the   single-stranded   uracil-containing   DNA,   the   complementary   strand   is synthesised (dT is used in the synthesis reaction rather than dU). This results in a heteroduplex DNA molecule, in which one strand (the parent strand) is dU-containing and does not contain the desired mutations and the second strand (the complementary strand synthesised in step (d)) is dT- containing and contains the mutations.   The resultant molecule is then transformed into an E.coli host that does not support the presence of uracil, e.g., a duf ung* host such as JM101. Thus  the  complementary  strand  containing  the   mutation  is  replicated  in preference to the parent strand not containing the mutations (the parent, dU containing strand, is hydrolysed by uracil-glycosylase in the duf ung+ host).<br>
Other selection methods include the use of methylation. The parent molecule will be methylated in almost all strains of E.coli. In step (d), a complementary strand can be synthesised in vitro that is not methylated, resulting in a heteroduplex comprising a methylated parent strand that does not contain the mutations and a non-methylated complementary strand that does contain the mutations. The<br>
heteroduplex is then digested with the Dpn1 restriction enzyme, which cuts double-stranded DNA at the site 5'-Gm6ATC-3' and is specific for methylated or hemimethylated DNA. Thus the parent strand is digested and the complementary strand containing the mutations is not. Alternative enzymes, e.g., other than Dpn1 may be used.<br>
As a further possibility for the selection method, a conditionally lethal gene may be used. One example is ccdB, also known as the "control of cell death" gene. This gene could be inserted into the parent molecule into the region that would be replaced following the mutagenesis reaction. The parent molecule would be prepared from an E.coli strain that is not susceptible to this gene. The mutagenesis reaction would then replace the ccdB gene with the gene of interest Those unmutated parent molecules, which still express ccdB will cause their E.coli hosts to die. Thus, the mutated molecules resulting from the mutagenesis reaction are selected.<br>
Other suitable systems and selective enzymes may be used provided that they involve the selective breakdown, of the parent strand of the heteroduplex. Examples include endonucleases and endogiycosylases.<br>
Thus in preferred embodiments, the method involves the use of a single-stranded parent molecule comprising modification to the nucleotides (e.g., dU replaces dT). Synthesis of the complementary strand primed from the mutagenic primer produces a newly synthesised mutated strand which does not include such modifications (e.g., by using dTTP rather than dUTP in the synthesis reaction). The non-modified, newly synthesised, mutated strand is preferentially selected (e.g., by transforming the heteroduplex molecule into a host cell that supports the replication of the non-modified, newly synthesised, mutated strand only, or by digesting the parent strand with a selective enzyme).<br>
In a preferred embodiment, the invention provides a method wherein said preferential selection is carried out by transforming said circular-form molecule from step (d) into a host cell that is selective for the complementary, non-modified, strand (the newly synthesised, non-modifed, mutated strand).<br>
Thus, in a preferred embodiment, the invention provides a method of introducing one or more mutations into a nucleic acid molecule comprising the following steps:<br>
(a)	synthesising (preferably by PCR) a mutagenic primer from a template<br>
nucleic acid molecule wherein said mutagenic primer contains one or<br>
more mutations relative to a parent molecule;<br>
(b)	separating a single stranded form of said mutagenic primer;<br>
(c)	annealing said single-stranded mutagenic primer to a single stranded<br>
form of said parent molecule, wherein said parent molecule contains<br>
one or more selectable modifications;<br>
(d)	synthesising a complementary strand of said mutagenic primer so as to<br>
produce a circular-form nucleic acid molecule that does not contain<br>
said modifications;<br>
(e)	undertaking a selection step such that the complementary strand is<br>
selected in preference to the parent molecule.<br>
In preferred embodiments, the method of the invention may be applied to populations of molecules: Thus, a population of molecules may be mutated using the above method, or a population of sequences may be inserted into an appropriate vector for screening (e.g., expression screening).<br>
For example, the method may be used for creating a library of molecules. In this way, multiple mutagenic primers can be used to mutate a suitable vector to produce a library of products. Combinatorial mutagenesis using multiple mutagenic primers and multiple parent molecules may also be used to produce a library with extensive permutations. For example, multiple parent molecules with a variation at a first position in that molecule may be used in the mutagenesis reaction, together with multiple mutagenic primers that introduce mutations at a second position, thus generating an extensive library as a result of the combination or permutation of the variations at the first and second positions. This is exemplified by example 1 and illustrated schematically in Figure 6.<br>
The method may also be used to facilitate the screening of a population of molecules. For example a population of mutagenic primers may be derived from a population of template molecules. This population may then be introduced into e.g., a common vector, such as an expression vector for screening. This is exemplified in example 2.<br>
Thus, in preferred embodiments the method is applied to populations of molecules, e.g. in the screening of a population of molecules or in the sub-cloning of a population of molecules to create a library. Thus, there may be a plurality of mutagenic primer, template and/or parent molecules. Such populations may be at least 1000 molecules in size, more preferably at least 104, more preferably at least 105, more preferably at least 106, more preferably at least 107, more preferably at least 108, more preferably at least 109, more preferably at least 1010&gt; more preferably at least 1011, more preferably at least 1012, more preferably at least 1013.<br>
Advantages associated with the application of the method of the invention to populations, especially libraries, are based in part, on the fact that the mutagenic primer sequences do not need to be known at the start of the protocol. Knowledge of a short region or regions of commonality in the template molecules (e.g., a known sequence that abuts a region of variability, or a known sequence in a vector that contains the template molecule(s)) allows the design of the initial primers.    These initial primers may therefore be used to amplify a whole population of molecules that have variable regions as well as the common regions to which the initial primers are directed.    Likewise a population of molecules may be inserted Into a common parent molecule (e.g, a vector) for screening, such as expression screening, provided that there is region of complementarity between the mutagenic primer and the parent molecule.  This region of complementarity may be the same as, or partially the same as, the region to which the initial primers were directed In the template molecule.<br>
Thus, in the method of the invention, there may be multiple templates, or multiple forms of the template.  For example, the template may be a library of different<br>
molecules contained within a vector, e.g., a plasmid. Thus, multiple mutageriic primers produced from such a library can then be used to produce a number of circular-form mutagenised outputs, because each of the multiple mutagenic primers introduces different mutations into the parent molecule when the complementary strand is synthesised in step (d). Where these multiple outputs are transformed into a suitable host, e.g., a bacterial host, such as E.coli, each resultant colony contains the output resulting from one of the multiple mutagenic primers.<br>
Where multiple different mutagenic primers are to be produced from a library, e.g., a plasmid library, then the initial primers used to sythesise the mutagenic primer are chosen so as to be complementary to a region of the vector, which is common to the library. Thus, the multiple templates may comprise a region of variability and a region that is in common between the multiple templates. For example, the region of variability may be a randomised region of nucleic acid which is adjacent to regions that are non-randomised. In another example, the region of variability may be produced via multiple inserts having been introduced into multiple molecules of the same base vector. The initial primers are therefore chosen to be complementary to a region in the template that lies outside the region of variability, e.g., complementary to all or part of the non-randomised region of the template, or to part of the base vector. In this way, multiple mutagenic primers may be produced in one synthesis reaction using the same initial primers.<br>
Alternatively or additionally, there may be multiple parent molecules. Thus, in step (c) the mutagenic primer may be annealed to multiple different parent molecules. Where there are multiple mutagenic primers, as previously described, these may also be annealed to multiple parent molecules. Thus a combinatorial effect can result, in which multiple mutagenic primers are annealed to multiple parent molecules with the production of combinations of mutagenic primers and parent molecules.<br>
Thus the method of the invention may include multiple parent molecules and/or multiple mutagenic primer molecules.<br>
This has the advantage that it can be used for large repertoires of parent molecules and/or templates, e.g. to combine large repertoires of parent molecules and/or templates. Preferred repertoire sizes are at least 100 parent molecule and/or templates, more preferably at least 1000, even more preferably at least 2500.<br>
Using the combinational approach large repertoires can be produced, e.g. after combination of the various parent molecules with the various mutagenic primers there may be at least 1000 resultant outputs, more preferably at least 104, more preferably at least 105, more preferably at least 106, more preferably at least 107, more preferably at least 108, more preferably at least 109, more preferably at least 1010&gt; more preferably at least 1011 , more preferably at least 1012 , more preferably at least 1013.<br>
The mutations that may be introduced by this method include substitutions, deletions and additions. The mutations may be in a plurality of nucleotides (but may also be only in a single nucleotide). For example, a plurality of substitutions may be made in the parent molecule by the mutagenic primer, or a number of nucleotides may be deleted, or added. Combinations of the different types of mutations are also possible. Thus, there may be a combination of substitutions and deletions, a combination of substitutions and additions, a combination of additions and deletions, or a combination of additions, deletions and substitutions.<br>
There may also be multiple regions of mutations, e.g., two regions in which one or more nucleotides are substituted, separated by a region in which no change is made by the mutagenic primer (because in this interstitial region, there is complementarity between the mutagenic primer and the parent molecule).<br>
Further possibilities include: multiple regions in which one or more nucleotides are substituted, separated by regions of complementarity; multiple regions in which one or more nucleotides are added, separated by regions of complementarity; multiple regions In which one or more nucleotides are deleted,<br>
separated by regions of complementarity. Combinations are also possible. Thus there may be: one or more regions in which nucleotides are substituted and one or more regions in which nucleotides are added, separated by regions of complementarity; one or more regions in which nucleotides are substituted and one or more regions in which nucleotides are deleted, separated by regions of complementarity; one or more regions in which nucleotides are deleted and one or more regions in which nucleotides are added, separated by regions of complementarity; and one or more regions in which nucleotides are substituted, one or more regions in which nucleotides are added, and one or more regions in which nucleotides are deleted, separated by regions of complementarity.<br>
The nature and number of the mutations is determined by the number and nature of the mismatches, as between the parent molecule and the mutagenic primer. Thus, where there is an addition of a single nucleotide in the mutagenic primer compared with the parent molecule then a single nucleotide will be added by virtue of the mutagenesis reaction. Where there are dispersed regions of mismatch in the mutagenic primer compared with the parent molecule (e.g., dispersed regions of additions, deletions and substitutions as set out above), this will result dispersed regions of corresponding mutations as a result of the mutagenesis reaction. The mutagenic primers may replace stretches of nucleotides in the parent molecule. For example, the mutagenic primer may contain a region of say 70% complementarity to the parent molecule and in this way, after the mutagenesis reaction, this region in the parent molecule Is replaced by the 70% complementary region from the mutagenic primer.<br>
In order for the mutagenic primer to anneal, there must be a degree of complementarity between the mutagenic primer and the parent molecule. There would typically be minimum regions of complementarity at the ends of the mutagenic primer. Where the mutagenic primer has been produced by PCR and the initial primers (defined above) are complementary to the parent molecule, then these provide a minimum region of 18 to 25 base pairs complementarity at the ends of the mutagenic primer.<br>
The degree of complementarity over the whole of the mutagenic primer is preferably at least 60%, more preferably at least 70% more preferably at least 80%, more preferably at least 90%, more preferably at least 95%, more preferably at least 97%, more preferably at least 98%, more preferably at least 99 %. By way of examples, at least 97% complementarity corresponds to 20 mismatches in a mutagenic primer that is 830 nucleotides long (810 complementary). At least 60% complementarity corresponds to a mismatch region in the mutagenic primer that is 730 nucleotides long and is 70% identical to the parent molecule in a mutagenic primer that is 830 nucleotides long (the two ends either side of the "70% region" being perfectly complementary to the parent molecule). Non-limiting examples of the mutagenic primers are shown in the figures.<br>
The mutagenic primer may be from 100 nucleotides to 4000 nucleotides in length, preferably 200 to 3000 nucleotides, more preferably 300 to 2000 nucleotides, more preferably 400 to 1500 nucleotides, more preferably 400 to 1000 nucleotides, even more preferably 400 to 850 nucleotides in length or 800 to 1500 nucleotides in length.<br>
The size of the mutagenic primer may depend upon the size of the parent molecule. For example, where the parent molecule is 5.5kb the mutagenic primer is preferably upto 1kb. Thus, the mutagenic primer is preferably around 1/50 to 1/5 the size of the parent molecule, more preferably 1/25, more preferably 3/10 the size of the parent molecule, most preferably around 1/5 the size of the parent molecule in nucleotides length.<br>
It is preferred that the molar ratio of mutagenic primer to parent molecule In step (c) is around 2:1 to 5:1, most preferably around 3:1.<br>
Preferred applications of the method of the invention include In the screening or creation of libraries, in particular antibody libraries. For example, the parent molecule may be a vector for the expression of antibodies or antibody domains. Having introduced mutations into the parent molecule (for example by making the mutagenic primers from a template comprising a library of antibody domains), the<br>
resultant mutated parent molecules can be expressed in a suitable host and screened against an appropriate antigen (for further information, see Vaughan et al, (1996) Nature Biotechnology, Volume 14, pp 309 to 314; and Edwards et a/., (2003;. J. Mol. Biol, 334, pp 103-118). Other antibody-related example's include the sub-cloning of antibody domains into an immunoglobulin acceptor plasmid, e.g., an IgG acceptor plasmid (such as a pEU IgG expression vector see Persic et al (1997) Gene 187:9-18. The term antibody describes an immunoglobulin and includes natural and synthetic versions.   The term encompasses polypeptides having an antibody binding domain, which is or is substantially homologous to, an antibody binding domain. Examples include, Fab (VL, VH, CL and CH1 domains), scFv (VH domain and VL domain linked by a linker such as (Gly-Gly-Gly-Gly-Ser)3, e.g., Bird et al (1988), Science, 242, 423-426, Huston et al. (1988), PNAS USA, 85 pp 5879-5883 Fv (VL and VH domains of a single antibody), Fd (VH and CH1 domains); dAb (Holt etal (2003), Trends in Biotech. 21 pp484-490) and diabodies (WO94/13804).<br>
Thus in preferred embodiments, the parent molecule, template or mutagenic primer comprises sequence encoding an antibody variable domain. The antibody variable domain may be a VH or VL domain. Preferably, the parent molecule comprises an scFv.<br>
Where an antibody or antibody domain is referred to herein in the context of nucleic acid such as vectors, it is understood to refer to the nucleic acid sequence encoding that antibody or domain.<br>
The method may also be used to shuffle antibody domains to create antibody repertoires. Thus, the parent molecule may contain an antibody VH domain and the mutagenic primer molecule may contain an antibody VL domain, or the parent molecule may contain an antibody VL domain and the mutagenic primer molecule may contain an antibody VH domain. A library of antibody VH domains may be combined (shuffled) with a library of antibody VL domains, or vice versa. Thus one of the two libraries may be used as the template to form multiple mutagenic primers and the other library may be present as a library in multiple parent molecules.<br>
Thus the invention provides a method in which the parent molecule contains an antibody VH domain and an antibody VL domain and the mutagenic primer molecule contains an antibody VL domain to mutate the VL domain of the parent molecule. The invention also provides a method in which the parent molecule contains an antibody VL domain and an antibody VH domain and the mutagenic primer molecule contains an antibody VH domain to mutate the antibody VH domain on the parent molecule. In this way, a library of antibody VH domains may be combined (shuffled) with a library of antibody VL domains, or vice versa.<br>
It may be desired that only one of the VL and VH domains of the parent molecule is to be mutated, or it may be desirable that both the VL and VH domains of the parent molecule be mutated. Single domain systems are also envisaged, e.g., in which the constructs contain a VL domain (but no VH domain) and that VL domain is mutated by a VL mutagenic primer, or e.g. in which the construct contains a VH domain (but no VL domain) and that VH domain is mutated by a VH mutagenic primer. The invention provides for all of these possibilities.<br>
Thus, the invention encompasses the situation in which both the VH and the VL domains on the parent molecule will be mutated by mutagenic primers containing each of these domains. Thus a VH mutagenic primer would mutate the VH domain on the parent molecule and a VL mutagenic primer would mutate the VL domain on the parent molecule. A single mutagenic primer containing both of these domains could also be used.<br>
The invention therefore provides a method, wherein first and second mutagenic prime.rs are used to mutate the parent molecule, the first mutagenic primer containing an antibody VH domain to mutate the VH domain on the parent molecule and the second mutagenic primer containing an antibody VL domain to mutate the VL domain on the parent molecule. The first and second mutagenic primers may be used in separate mutagenesis reactions steps, in which case the first and second mutagenic primers may take part in separate mutagenesis reactions in any order (i.e. the first mutagenic primer could be used in a mutagenesis reaction preceding or succeeding the mutagenesis reaction<br>
involving the second mutagenic primer). Another possibility is that said first and second mutagenic primers are used in the same mutagenesis reaction step.<br>
The aspects described above in which the method of the invention is used to combine (or shuffle) libraries of VH with libraries of VL domains in the antibody context could be applied to other systems in which the shuffling of combinations of domains is desired, such as other molecules related to the immunoglobulins and to multi-domain proteins in general. Thus the invention provides a method of combining a library of first domains with a library of second domains, as described above for the antibody aspects (in which the first domain is the VH domain and the second domain is the VL domain, or vice versa).<br>
In some preferred embodiments, the parent molecule comprises nucleic acid encoding an scFv.<br>
The present invention also provides kits for carrying out the above-mentioned methods. For example, such a kit may comprise initial primers suitable for synthesising a single-stranded primer molecule from a template molecule, a single-stranded parent molecule and instructions for use. The parent molecule may be modified as described herein. Suitable reagents and buffers may also be included, such as those referred to herein. Further kit components as defined in any of the embodiments of the method of the invention may also be included.<br>
As noted, the methods and kits of the invention have a number of applications, including the following. The methods can be used in sub-cloning, for example a region of nucleic acid can be generated by step (a) and introduced into an appropriate vector by the mutagenesis reaction, avoiding the need for digestion and ligation of the nucleic acid. For example, a target antigen could be cloned into an expression vector. The methods may be applied to the creation and screening of libraries as mentioned above. More specific utilities are outlined below.<br>
The methods described herein can be used to convert single chain Fv (scFv) fragments (VH and VL domains linked as a single peptide, via a linker) into IgG molecules. Thus, the VH and VL domains may be introduced into lgG_VH and lgG_V|_ acceptor plasmids. Using initial primers that are so-called generic oligonucleotides, directed to non-variable parts of the vector that contains the scFv, or to the linker between the two domains (e.g., one initial primer that is directed to the sequence upstream or downstream of each of the domains and a second initial primer that is directed to the linker between the two domains). For example, if using the scFvs described in Vaughan et al, one of the initial primers could be directed to sequence upstream of the VH domain in pCantabG and the other to the linker (i.e. directed to part of the sequence encoding (Gly-Gly-Gly-Gly-Ser)3).<br>
The methods of the invention can also be used to generate naive libraries (i.e., libraries from non-immunised human donors). Currently these are generated by carrying out PCR of naive antibody VH and VL regions from non-immunised human donors, followed by restriction digest and ligation into acceptor plasmids. Using the method of the invention, mutagenic primers may be separated from these PCR products derived from the non-immunised donors. Again, the use of the above-mentioned methods removes the need for restriction and ligation. Suitable primers may be based on sequences at each end of the heavy chain and each end of the light chain (see, for example, Vaughan et al., supra).<br>
Aspects and embodiments of the invention will now be illustrated by way of examples and with reference to the following figures, in which:<br>
List of Figures<br>
Figure 1 shows the mutagenic primers for use in the method of the invention. In each case, the numbers are the approximate number of bases and the mismatch region is in grey. Figure 1a shows mutagenic primer 1, used in Example 1 (VH / VL recombination). Figure 1b shows mutagenic primer 2a, used in Example 2 (Ribosome Display subcloning) for VnCDRS variants. Figure 1c shows mutagenic primer 2b, used in Example 2 for V1CDR3 variants. Figure 1d shows<br>
a mutagenic primer for producing VH or VL variants. The region in grey is about 70% identical to the template of Example 2. Figure 1e shows mutagenic primers for cloning into an IgG acceptor plasmid. Figure 1f shows mutagenic primers for cloning into VH and VL acceptor plasmids for the creation of naive libraries.<br>
Figure 2 shows the position of the initial primers for the synthesis of the mutagenic primer for the use in the VH / VL recombination embodiments (e.g, Example 1). A and B show the forward and reverse primers, respectively.<br>
Figure 3 shows the position of the initial primers for the synthesis of the mutagenic primer for the use in the ribosome display aspects (e.g., Example 2). A and B show the forward and reverse primers, respectively.<br>
Figure 4 shows results of the mutagenesis procedure of Example 1.<br>
Figure 5 is a schematic showing the mutagenesis procedure of the invention. In the schematic, the VHCDR3 from an scFv-containing plasmid containing a particular mutation in the VHCDR3 (shown by a star) is amplified by PCR and separated into single-stranded form by virtue of biotin being introduced by the primers in the PCR reaction. Single stranded DNA is purified from an scFV-containing plasmid, which contains a particular mutation in the VLCDR3 (also shown by a star). In the mutagenesis reaction between the single stranded PCR product containing the VHCDR3 mutation and the single-stranded plasmid containing the VLCDR3 mutation the two mutations become combined to produce an scFv-containing plasmid that contains both the mutations.<br>
Figure. 6 shows a combinatorial embodiment of the invention. At the top of the figure are shown multiple template molecules that contain mutations a, b and c. The regions of these molecules containing the mutations are amplified by PCR and separated into multiple single-stranded mutagenesis primers using biotin. At the bottom of the figure is shown multiple parent molecules which contain mutations x, y and z. Single-stranded DNA is produced from these molecules, so as to result in multiple parent molecules each containing mutations x, y or z. In<br>
the mutagenesis reaction, the mutagenic primers containing mutations a, b and c are used to mutate the multiple parent molecules. This results in the combination of outputs shown in the right of the figure.<br>
EXAMPLES:<br>
EXAMPLE 1; Mutaqenesis Method to Recombine Optimised VH and Vt Regions<br>
1.       Overview of method<br>
The recombination of optimised VH and VL regions has been shown to lead to increases in potency during antibody affinity maturation in vitro (Osbourn et a/., (1996), Immunotechnology, Vol 2, pp 181-196). In this example, the mutagenesis method of the invention is used to recombine one pool of variants in which the VHGDR3 has been -randomised with another pool of variants in which the VLCDR3 has been randomised. Both pools of variants contain the samexiriginal antibody sequence, except for the short regions of randomisation in the VH and VL CDR3. Following randomisation, the VH and VL pools have each undergone two rounds of phage display selection for improved affinity. The VH and VL outputs of the round two selection, each containing an estimated 10s to 106 different variant sequences, form the starting point for the recombination method described in this example.<br>
In the method of this example, the VH pool is amplified by PCR using the biotinylated PAMA-IN oligonucleotide at the 5' end and the H-LINK oligonucleotide at the 3' end (see Figure 2). Following capture on streptavidin beads, the non-biotinylated strand is eluted and this forms the mutagenic primer.<br>
The VL pool is prepared as single stranded plasmids, following rescue of bacteriophage particles from the dut- ung- Eco//strain CJ236. By bringing the VH<br>
encoding mutagenic primer together with the single stranded plasmids representing the VL pool, the recombination of VH and VL regions can occur.<br>
2.	Oligo Design<br>
The dU-ss DNA template isolated from M13 is plus-strand so the mutagenic primer is complementary to the plus-strand (i.e. it is minus-strand). By biotinylating the forward primer, the minus strand can be eluted following capture on streptavidin. The oligonucleotides used are as follows (51 to 3'):-<br>
Bio-PAMA-IN:biotin-GCGGCCCAGCCGGCCATGG H-LINK: ACCGCCAGAGCCACCTCCGCC<br>
3.	Preparation of VL Outputs as dsPlasmid DNA<br>
1.	The glycerol stock of the VL repertoire in pCantabS (Vaughan et al) was<br>
inoculated 100ul into 50ml 2xTYA in a small conical flask.<br>
2.	The culture was grown for 2 hours at 37°C and 300rpm.<br>
3.	Cells were pelleted by centrifugation in 50ml Falcon tubes at 3200rpm for 10<br>
minutes.<br>
4.	Qiagen HiSpeed Plasm id Midi Kit protocol was carried out to isolate ds plasmid<br>
DNA in 1ml water.<br>
5.	The concentration of DNA was checked by UV spectrometry or agarose gel.<br>
4. dU-ss DNA Template Preparation<br>
The dsPlasmid DNA from part 3 was then used in the following protocol:<br>
Reagents:<br>
Plasmid Midiprep of VL repertoire in pCantab6 (Vaughan et al)<br><br>
Chloramphenicol (CAM) at 10mg/ml in ethanol and store at -20°C<br>
Uridine (Sigma U-6381) at 0.25mg/ml in water and store at 4°C<br>
2xTYAG (4-10ug/ml CAM) plates:  (TY is as described in Sambrook et al, supra<br>
plus 100ng/ml amplicillin (A) and 2 % glucose (G) Chloramphenicol (CAM) is<br>
added at 20jil stock to 80 \i\ ethanol and spread onto plate).<br>
Preparation of CaCl2 competent CJ236<br>
1.	5 ml of 2xTY media (plus 10ug/ml CAM) is inoculated with a single colony of E.<br>
coll CJ236 cells (e.g. supplied by Biorad). Cells were grown overnight at 37°C<br>
300rpm. CAM selects for F episome in CJ236.<br>
2.	500ml of 2xTY was inoculated with 5 ml of the overnight culture and incubated<br>
with shaking at 37°C.<br>
3.	When  the  OD  at 600nm   reached  0.9  the  culture  was  harvested  by<br>
centrifugation at 4°C (5 min, 5,000 rpm). The supernatant was poured off and cell<br>
pellet drained. The cells were resuspended gently in 100ml of ice cold 100mM<br>
MgCI2.<br>
4.	Cells were again harvested by centrifugation (centrifuge at 4°C) and pellets<br>
were drained. Cells were gently resuspended in 20ml of cold 100mM CaCI2 until<br>
a smooth suspension was obtained. 200ml of cold 100mM CaCI2 was added and<br>
mixed. The cells were placed on ice for 30-90 mins.<br>
5.	The cells were harvested by centrifugation in a cold centrifuge and the pellets<br>
drained. Cells were resuspended in 20ml of cold 85mM CaCIa and 15% glycerol.<br>
6.	The cell suspension was aliquoted immediately (the aliquots can be frozen in<br>
dry ice/ethanol if need be).<br>
Transformation of template into CJ236<br>
1. The CJ236 CaCl2 competent cells from above were thawed on ice (and were kept on ice throughout the protocol).<br>
2.	5μl of pCantab6 plasmid containing the VL repertoire from section 3 was added<br>
to 100μl of competent cells in a PCR tube.<br>
3.	The following programme was run on the PCR machine block:<br>
0.1 °C for 30 min 42°C for 45s 0.1°Cfor2min<br>
4.	The cells were transferred 15ml Falcon tube in 0.5 ml of 2xTY.<br>
5.	The cells were incubate at 37°C for 45-60 min with shaking at 200rpm.<br>
6.	0.5ml of cells were plated onto 2xTYAG (+10ug/ml CAM) plates and the plates<br>
incubated o/n at 37°C.<br>
Rescue ofphage from CJ236 and DNA preparation<br>
1.	A lawn arising from step 6 above of CJ236 transformants was scraped into<br>
1ml 2xTY.  SOOu.1 was inoculated into 50ml 2xTYGAC (ie 2% glucose, 100μg/ml<br>
AMP, 10ng/ml CAM) and incubated at 37°C and 300rpm until OD600= 0.5 to 1.<br>
2.	The cell density was calculated from the OD reading (OD600 of 1=5x10s<br>
cells/ml) and wild-type KO7 helper phage were added (Amersham Blosciences<br>
M13 KO7 Helper phage) to ensure a multiplicity of infection (MOI) of 10 phage: 1<br>
cell (wt KO7 stock usually contains 1010 phage in 0.33μl). The cells were then<br>
incubated at 37°C (no shaking) for 10 minutes to allow infection.<br>
3.	1ml culture was transferred to 30ml 2xTYAC supplemented with 25ug/ml KAN<br>
(to select KO7) and 0.25μg/ml urldine (to allow synthesis of uracil-contalning<br>
template). This was incubated overnight at 37°C 300rpm.<br>
4.	Cells were centrifuged for 10 mins at 15000rpm 2°C In a SS-34 rotor.<br>
Supernatant was transferred to a fresh tube and 1/5 volume of PEG-NaCI was<br>
added (20% PEG8000, 2.5M NaCI). This was incubated for 5 mins at room temperature and then centrifuged for 10 mins at 10000rpm 2°C in SS-34.<br>
5.	A small white phage pellet was produced. Supernatant was decanted and tube<br>
inverted on tissue to drain.  The phage pellet was resuspended in 0.5ml of PBS<br>
(the walls of the tube were rinsed to capture as many phage as possible). The<br>
phage were transferred to eppendorfs and centrifuged for 5 mins at 15000rpm in<br>
a microfuge to pellet remaining insoluble matter. The supernatant was transferred<br>
to a new eppendorf.<br>
6.	The dU-ss DNA template was purified using a Qiagen Qiaprep Spin M13 kit,<br>
beginning with the addition of the Qiagen MP buffer (Sidhu etal, supra). The DNA<br>
was eluted in 100μl 10mM Tris-HCI, pH8.O and examined on a 1% agarose gel<br>
The ss DNA preparation ran as a distinct band at 2.5kb and was used in the<br>
mutagenesis reaction in section 6.<br>
5. Preparation of ssDNA Mutagenic Primer from VH Repertoire<br>
The region representing the repertoire of different VH sequences was amplified from pCantab 6 by PCR using the oligonucleotides bio-PAMA-IN andH-LINK. Standard PCR conditions can be used on a dilution series of the midiprep template prepared in section 3.<br>
Abgene (2x) Mastermix	25μl<br>
Template midiprep	1μl [neat, 10'1, 10"e, 1CT3,10'4, 10"5,10'e, none]<br>
Bio-PAMA-IN	1μl<br>
H-LINK	1μl<br>
Water	22μl'<br>
5μl of each PCR was run on a 1 % agarose gel (this confirmed the product size, showed the yield etc).<br>
1. A microMACS column (Miltenyi Biotec: 130-042-701) was placed in a magnetic rack.<br>
2. 250μl of 1x binding buffer (2x binding buffer: 10mM Tris pH7.5, 2M NaCI, 1mM EDTA, 0.1% Tween 20) was allowed to flow through.<br>
3.	45M-I PCR reaction was mixed with 105μl microMACS streptavidin beads<br>
(Miltepyi Biotec: 130-074-101) and 150μl 2 x binding buffer and incubated for two<br>
minutes.<br>
4.	DNA+beads was applied to a microMACS column and allowed to flow through.<br>
5.	The column was washed with 2x1 ml 1 x binding buffer.<br>
6.	DNA was eluted into eppendorf with 200μl 0.1N NaOH (made fresh).<br>
7.	To the eluted DNA was added 20μl sodium acetate pH5.2, 550μl ethanol, 1μl<br>
glycpgen,<br>
8.	This was incubated at -70°C for 30 minutes.<br>
9.	This was centrifuged at 13000 rpm for 10 minutes to pellet DNA.    The<br>
supernatant was removed and replaced with 500μl 70% ethanol.<br>
10.	This was centrifuged at 13000 rpm for a further 5 minutes, the supernatant<br>
removed, air dried in 37°C heat block for 2 minutes. DNA was resuspended in<br>
50μl water.<br>
6. Mutagenesis Reaction<br>
Reagents:<br>
TM buffer (10x): 500mM Tris-HCI, 100mM MgCI2, pH7.5<br>
10mM ATP: (1M stock made by dissolving 0.6052g in 1ml water and adjusting pH<br>
to 7.0 with 0.1M NaOH. Stock diluted 1:100 to make 10mM ATP - can be and<br>
stored at -70°C)<br>
100mM DTT: {1M stock made by dissolving 0.1542g in 1ml 0.01 M sodium<br>
acetate (pH5.2). Stock diluted 1:10 to make 100mM DTT - can be stored at -<br>
20°C).<br>
Enzymes: T4 polynucleotide kinase, T4 DNA ligase and T7 DNA polymerase<br>
from New England Biolabs (NEB).<br>
25mM dNTPs<br>
Qiagen Qiaquick PCR purification kit (Qiagen)<br>
Phosphorylation of mutagenic primer The following were added to an Eppendorf:<br>
ssDNA mutagenic primer   1 .Oμg (1.0 μg /132000 daltons = 7.6 pmoles)<br>
TM buffer (1 Ox)       5.5μl10mMATP	5μl<br>
100mM DTT	2.5μl<br>
Water was added to a total volume of 53μl. 2μl (20U) of T4 polynucleotide kinase (10Uμl) were added and incubated for 1 hour at 37°C.<br>
Annealing of mutagenic primer to template at a ratio of 3:1 The following were added to an eppendorf:<br>
dU-ssDNA template (from Section 4)      4.4ng   (4.4μg  /   1,750,000  daltons  =<br>
2.5pmoles)<br>
phosphorylated mutagenic primer 1.0μg (1.0μg /132000 daltons = 7.6pmoles)<br>
TM buffer (10x)       25μl<br>
Water was added to a volume of 250μl.<br>
The 250μl was incubated at 90°C for 2mins, 50°C for 3mins and 20°C for 5mins to anneal the mutagenic primer to the template.<br>
Mutagenesis reaction<br>
To the annealed oligo/template mix, the following were added:<br>
10mMATP<br>
25mMdNTP's<br>
100mMDTT<br>
30 Weiss units T4 DNA ligase (6U/ul)*   5μl<br>
30 units T7 DNA polymerase (1 OU/ul)    3μl<br>
The units marked on the enzyme tube are not Weiss units - 6U/ul is the correct concentration of enzyme activity<br>
The reactions were incubated at 20°C for 3 hours.<br>
Each reaction was affinity purified to 35μl water using the Qiagen Qiaquick PCR purification kit. 1μl of each reaction was run on a 1 % agarose gel and compared with the output for the reactions with and without mutagenic primers. An intense ssDNA band (2.5kb for pCantab6) was observed in the 'without primer" lane. In the 'with primer lane this ssDNA band was reduced in intensity and replaced by one or more larger bands representing cccDNA (covalently closed circular DNA)(&gt;3kb for pCantab6).<br>
7. Electroporation into E.coli TG1's<br>
Fresh electrocompetent TG1 cells (Stratagene) were prepared according to the following protocol:<br>
1.	An appropriate volume of 2TY media was inoculated with a colony from a TG1<br>
minimal media plate. Cells were grown overnight at 25°C and 300rpm.<br>
2.	5 flasks containing 500ml each of 2TY were inoculated with 10ml (per flask) of<br>
the overnight culture.   This was grown at 25°C 300 - 350rpm until OD6oo -0.5-<br>
0.6) [this usually takes around 2-3hours]<br>
3.	6 x 500ml centrifuge bottles and the Sorval SLA 3000 rotor were pre-chilled to<br>
2°C.<br>
4.	Once optimum OD is reached, cells were chilled for SOmins on ice in the<br>
centrifuge bottles and spun at 4,000rpm for 15mins at 2°C.<br>
5.	The supernatant was poured off and the pellet resuspended in a small volume<br>
of ice cold autoclaved MilliQ water.  This was made upto ~300ml with water<br>
and spun at 4,000rpm for 15mins at 2°C.<br>
6.	The supernatant was poured off and resuspended in small volume of the<br>
remaining water and then 300ml water added and spun as above.<br>
7.	8 x 50ml Falcon tubes were prechllled at -20°C. A Sorvall bench top<br>
centrifuge was prechilled to 2°C.<br>
8.	The supernatant was poured off and the pellet resuspended in the remaining<br>
water.<br>
9.	Once resuspended, this was tipped into the Falcon tubes and water added to<br>
50ml. This was spun at 3,200 rpms for 10mins.<br>
10.	The supernatant was removed and each pellet was resuspended in a small<br>
amount of water, the cells were combined into one 50ml Falcon tube, made<br>
up to 50ml with water and spun as above.<br>
11. The supernatant was removed and the cells resuspended in remaining volume of water.<br>
For the electroporation a dilution series of cells (cells only) was used to determine a suitable concentration (e.g., dilutions of cells at 100%, 80%, 60% and 50%). The cells only control is plated out on 2TYAG as confirmation that there are no ampicillin-resistant cells in the competent cell preparation.<br>
The total 35μl completed mutagenesis reaction was electropo rated directly into 400μl electrocompetent TG1 cells in a single cuvette (Bio-Rad E.coli Pulser cuvette 0.2cm electrode gap; Cat No 165-2086), using the following settings:<br>
2.5KV field strength<br>
200Ω resistance<br>
25uF capacitance<br>
Time constant ~ 4.2 milliseconds<br>
Immediately after electroporating, 1ml of 2TYG was added to the cuvette and the cells were transferred to a 50ml Falcon tube. The cuvette was rinsed with a further 1ml of 2TYG to transfer the cells from the bottom of the cuvette (with an elongated P200 pipette tip) to the Falcon tube. This was transferred to an incubator at 37°C and ~150-250rpm to allow the cells to recover for 1 hour.<br>
A dilution series of each library (each Falcon tube) was plated onto small 2xTYAG plates to estimate library size and the remaining cells on a large 2xTYAG plate. The remaining cells were spun (3200rpm 10 mins) and resuspended in 1ml 2TYG before plating on the large plate. The plates were incubated overnight at 30°C.<br>
The library was then scraped from the large plate into 5ml 2xTY and then supplemented with 2.5ml 50% glyceroi. The cell density was calculated by testing the OD600 (OD600 of 1 = 5x108 cells/ml) of a 1:100 dilution. For storage, multiple aliquots of each library were stored at -70°C, each aliquot containing a ten-fold<br>
excess of cells over library size. A single aliquot can then be thawed and added to a 500ml culture for rescue of the library as phage, e.g, for selection of phage by affinity binding (Hawkins et al (1992) J.Mol.Biol, 226, pp889-896).<br>
The results of this method are as follows, and are also shown in Figure 4<br>
(Table Removed) <br>
EXAMPLE 2:Subcloning of ribosome display outputs<br>
This example relates to the sub-cloning of outputs from ribosome display. In this subcloning method, the template was a scFv, described as the parent scFv. The protocols used are as for Example 1.<br>
The initial PCR step that produces the mutagenic primer by using the initial primers is carried out on the outputs of a ribosome display library (i.e., RT-PCR products), see Jermutus et al (2001) PNAS, vol 98, no 1, 75-80. The oligonucleotides used as the initial PCR primers are shown in Figure 3.<br>
Thus a repertoire of different scFv sequences from ribosome display outputs (Jermutus et al P.N.A.S 2001, Volume 98 pp 75 to 80) were PCR amplified using a biotinylated forward primer annealing at the beginning of the VH and the reverse primer mycRestore which primes in the myc tag, just downstream of the end of the VL (see Figure 3).<br>
Depending of the amino acid sequence at the beginning of the heavy chain, either Bio-EVQ or Bio-QVQ were used. Standard PCR conditions were used (see Example 1 for details).<br>
The resultant PCR products were separated into single-stranded form to'form the mutagenic primers as described in Example 1 and used in the mutagenesis reaction.<br>
The parent molecule for the mutagenesis reaction was single-stranded scFv in pCantab6 (Vaughan et al, (1996) Nature Biotechnology, Volume 14 pp 309 to 314). In this example, only a single sequence scFv parent pCantab 6 was used. Thus, the single parent molecule is mutagenised by multiple mutagenic primers thereby to sub-clone the multiple outputs from the ribosome display into the parent plasmid.<br>
The preparation of the single-stranded parent molecule, the protocols for the mutagenesis reaction and for transformation of cells are as for Example 1.<br>
The outputs of the mutagenesis reactions resulted in &gt; 10000 transformants after electroporation. In contrast, the control with no mutagenic primer and no enzyme produced -10 transformants.<br>
The results of the above example were as follows:<br><br>
(Table Removed)<br>
The mutagenic primers referred to above are shown in the Figures.    By comparison, when synthetic oligonucleotides are used in mutagenesis (typically,<br><br>
18 complementary bases either side of an 18 base mismatch) the average mutagenesis efficiency is 64% (n=350 sequences).<br>
Example3: Size Exclusion procedure<br>
Mutagenesis efficiency was further improved by carrying out a size exclusion step on the single-stranded parent molecule of Example 2, prior to the mutagenesis reaction. This resulted in a 1.5x improvement in mutagenesis efficiency (measured by sequence analysis, counting the number of successfully mutated sequences).<br><br><br><br>
CLAIMS:<br>
1.       A method of introducing one or more mutations into a nucleic acid molecule, comprising the following steps:<br>
(a) synthesising a mutagenic primer from a template nucleic acid molecule, wherein said mutagenic primer contains one or more mutations relative to a parent molecule; ; (b) isolating a single-stranded form of said mutagenic primer<br>
(c)	annealing said single-stranded mutagenic primer to a single-stranded<br>
form of said parent molecule<br>
and<br>
(d)	synthesising a complementary strand from said mutagenic primer so as<br>
to produce a circular-form nucleic acid molecule containing said mutations.<br>
2.	The method according to claim 1, wherein prior to said annealing step (c),<br>
the single-stranded primer is admixed with the single-stranded parent molecule.<br>
3.	The method of claim 1, wherein the single-stranded primer of step (b) is<br>
isolated by use of a separating medium.<br>
4.	The method according to claim 2, further comprising the step of isolating<br>
said single-stranded parent molecule prior to step (c).<br>
5.	The method according to claim 4, further comprising a size exclusion step<br>
performed on said single-stranded parent molecule prior to step (c).<br>
6.	The method according to any one of the preceding claims, wherein said<br>
single-stranded parent molecule is single-stranded circular DNA.<br>
7.	The method according to any one of the preceding claims, wherein the<br>
parent molecule is an expression vector or phage display vector.<br>
8.	The method according to any one of the preceding claims, wherein said<br>
circular-form molecule is covalently closed circular DNA.<br>
9.	The method according to any one of preceding claims, further comprising<br>
the step of transforming the mutagenised nucleic acid molecule into host cells.<br>
10.	The method according to claim 9, wherein the said transforming is<br>
performed using electroporation.<br>
11.	The method according to any one of the preceding claims, comprising a<br>
step carried out under conditions such that the complementary strand containing<br>
the mutations is preferentially selected.<br>
12.	The method of claim  11, wherein said preferential selection of the<br>
complementary strand results from the preferential digestion of the parent<br>
molecule.<br>
13.	The   method   of  claim   11,   wherein   the   parent   molecule   contains<br>
modifications such that the complementary strand is preferentially selected.<br>
14.	The method of claim 13, further comprising the step of introducing said<br>
modifications.<br>
15.	The method of claim 13 or claim 14, wherein the modification is the<br>
introduction of dU in place of dT.<br>
16.	The method of claim 13 or claim 14, wherein in modification is methylation<br>
or the introduction of a conditionally lethal gene<br>
17.	The method of any one of claims 11 to 16 wherein said preferential<br>
selection is carried out by transforming said circular-form molecule from step (d)<br>
into a host cell that is selective for the complementary, non-modified, strand.<br>
18,   The method according to any one of the preceding claims, wherein said mutagenic primer is synthesised by a PCR-based method.<br>
19.     The method according to any one of the preceding claims, wherein said<br>
mutagenic primer is synthesised by a synthesis method in which errors are<br>
introduced when synthesising a second strand from a first strand.<br>
20.	The method according to claim 19, wherein said method is error-prone<br>
PCR.<br>
21.	The method according to claim 19 or claim 20, wherein the sequence of<br>
the template for the mutagenic primer synthesis is the same as that found in the<br>
parent molecule.<br>
22.	The method according to claim 3, wherein the step of synthesising said<br>
mutagenic primer introduces a binding moiety in either the positive strand or<br>
negative strand of the mutagenic primer and said separation is carried out via<br>
said binding moiety binding to its binding partner on said separating medium.<br>
23.	The method according to claim 22, wherein said binding moiety is<br>
introduced into the positive strand of the mutagenic primer and the negative<br>
strand of the mutagenic primer is eluted from said separating medium.<br>
24.	The method according to claim 22 or claim 23, wherein said binding moiety<br>
is biotin and said binding partner is streptavidin<br>
25.	The method according to any one of the preceding claims, wherein said<br>
mutation(s) include substitutions.<br>
26.	The method according to any one of the preceding claims, wherein said<br>
mutation(s) include deletions.<br>
27.	The method according to any one of the preceding claims, wherein said<br>
mutation(s) include additions.<br>
28.	The method according to any one of the preceding claims, wherein said<br>
mutations are mutations in a plurality of nucleotides.<br>
29.	The method according to any one of the preceding claims, wherein said<br>
mutagenic primer has at least 80% complementarity with the parent molecule.<br>
30.	The method according to claim 29, wherein said mutagenic primer has at<br>
least 90% complementarity with the parent molecule.<br>
31.	The method according to claim 30, wherein said mutagenic primer has at<br>
least 95% complementarity with the parent molecule.<br>
32.	The method according to any one of the preceding claims, wherein said<br>
mutagenic primer is from 100 nucleotides to 4000 nucleotides in size.<br>
33.	The method according to any one of the preceding claims, wherein the<br>
molar ratio of mutagenic primer to parent molecule in step (c) is around 3:1.<br>
34.	The method according to any one of the preceding claims, wherein there<br>
are multiple parent molecules and/or multiple mutagenic primer molecules.<br>
35.	The method according to any one of the preceding claims, wherein the<br>
parent molecule, template or mutagenic primer contains an antibody variable<br>
domain.<br>
36.	The method according to claim 35, wherein said antibody variable domain<br>
is an antibody VL domain.<br>
37.	The method according to claim 35 wherein said antibody variable domain<br>
is an antibody VH domain.<br>
38.	The method according to claim 35, wherein the parent molecule contains<br>
an antibody VH domain and an antibody VL domain and the mutagenic /primer<br>
molecule contains an antibody VL domain to mutate the VL domain of the parent<br>
molecule.<br>
39.	The method of claim 35, wherein the parent molecule contains an antibody<br>
VL domain and an antibody VH domain and the mutagenic primer molecule<br>
contains an antibody VH domain to mutate the antibody VH domain on the parent<br>
molecule.<br>
40.	The method of claim 38 or claim 39, wherein first and second mutagenic<br>
primers are used to mutate the parent molecule, the first mutagenic primer<br>
containing an antibody VH domain to mutate the VH domain on the parent<br>
molecule and the second mutagenic primer containing an antibody VL domain to<br>
mutate the VL domain on the parent molecule.<br>
41.	The method of claim 40, wherein said first and second mutagenic primers are<br>
used in separate mutagenesis reactions steps.<br>
42.	The method of claim 41, wherein said first and second mutagenic primers<br>
are used in the same mutagenesis reaction step.<br>
43.	The method of claim 38 or claim 39, wherein a single mutagenic primer<br>
containing both the VH and the VL domains is used to mutate the VH and the VL<br>
domains on the parent molecule.<br>
44.    The method according to any of claims 34 to 43, wherein the parent molecule comprises nucleic acid encoding an scFv.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUFic3RyYWN0LSgxMS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Abstract-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNsYWltcy0oMDEtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Claims-(01-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNsYWltcy0oMTEtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Claims-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNsYWltcy0oMTMtMDctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Claims-(13-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNsYWltcy0oMjYtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Claims-(26-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDEtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence Others-(01-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence Others-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence Others-(13-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence Others-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence Others-(14-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence Others-(14-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence Others-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence Others-(26-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjktMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence Others-(29-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMDItMDMtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence-Others (02-03-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTEtMDItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence-Others (11-02-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence-Others-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence-Others-(07-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence-Others-(07-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence-Others-(07-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTMtMDYtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence-Others-(13-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTMtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Correspondence-Others-(13-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTUtMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Correspondence-Others-(15-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LURyYXdpbmdzLSgxMS0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Drawings-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LURyYXdpbmdzLSgxMy0wNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Drawings-(13-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMSAoMDItMDMtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-1 (02-03-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMS0oMTMtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-1-(13-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMTMtKDEzLTA3LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-13-(13-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMiAoMDItMDMtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-2 (02-03-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMi0oMjYtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-2-(26-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMjYtKDA3LTA0LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-26-(07-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-3-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMy0oMDctMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-3-(07-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMy0oMDctMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-3-(07-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMy0oMTEtMDItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-3-(11-02-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMy0oMTMtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-3-(13-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMy0oMTUtMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-3-(15-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-3-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUZvcm0tMy0oMjYtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Form-3-(26-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1ERUxOUC0yMDA3LUZvcm0tMy0oMjktMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">5234-DELNP-2007-Form-3-(29-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LUdQQS0oMTEtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-GPA-(11-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIzNC1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5234-delnp-2007-Petition-137-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=TWFya2VkLXVwIGNvcGllcyBvZiByZXZpc2VkIEZvcm0gMyBhbmQgUGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">Marked-up copies of revised Form 3 and Petition.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UmV2aXNlZCBGb3JtcyAzIGFuZCBQZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">Revised Forms 3 and Petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263307-a-recombinant-negative-strand-rna-virus-containing-a-viral-genome-with-a-mutation-in-the-p-gene.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263309-an-isolated-dna-molecule-an-expression-cassette-and-vector-comprising-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263308</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5234/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MEDIMMUNE LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>MILSTEIN BUILDING, GRANTA PARK, CAMBRIDGE CB1 6GH, GREAT BRITAIN.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MINTER, RALPH, RAYMOND</td>
											<td>CAMBRIDGE ANTIBODY TECHNOLOGY LIMITED, MILSTEIN BUILDING, GRANTA PARK, CAMBRIDGE CB1 6GH, GREAT BRITAIN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LANE, STEVEN, GODFREY</td>
											<td>CAMBRIDGE ANTIBODY TECHNOLOGY LIMITED, MILSTEIN BUILDING, GRANTA PARK, CAMBRIDGE CB1 6GH, GREAT BRITAIN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HOLGATE, ROBERT, GEORGE, EDWARD</td>
											<td>CAMBRIDGE ANTIBODY TECHNOLOGY LIMITED, MILSTEIN BUILDING, GRANTA PARK, CAMBRIDGE CB1 6GH, GREAT BRITAIN</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GROVES, MARIA, ANASTASIA, THERESA</td>
											<td>CAMBRIDGE ANTIBODY TECHNOLOGY LIMITED, MILSTEIN BUILDING, GRANTA PARK, CAMBRIDGE CB1 6GH, GREAT BRITAIN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2006/000059</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-01-09</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/642,729</td>
									<td>2005-01-19</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0500417.1</td>
									<td>2005-01-10</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263308-method-of-mutagenesis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:29:04 GMT -->
</html>
