<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263366-a-recombinant-vector-construct-comprising-a-first-nucleic-acid-and-a-second-nucleic-acid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:11:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263366:&quot;A RECOMBINANT VECTOR CONSTRUCT COMPRISING A FIRST NUCLEIC ACID AND A SECOND NUCLEIC ACID&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A RECOMBINANT VECTOR CONSTRUCT COMPRISING A FIRST NUCLEIC ACID AND A SECOND NUCLEIC ACID&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Isolated polynucleotides and polypeptides encoded thereby are described, together with the use of those products for making transgenic plants with increased tolerance to abiotic stress (e.g. high or low temperatures, drought, flood).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>- -THEREBY USEFUL FOR MODIFYING PfcANX-<br>
This Nonprovisional application claims priority under 35 U.S.C. § 119(e) on U.S. Provisional Application No(s). 60/575,253 filed on May 27, 2005, the entire contents of which are hereby incorporated by reference.<br>
FIELD OF THE INVENTION<br>
The present invention relates to isolated polynucleotides, polypeptides encoded thereby, and the use of those sequences for making transgenic plants with modulated water use efficiency.<br>
BACKGROUND OF THE INVENTION<br>
Plants are constantly exposed to a variety of biotic (e.g., pathogen infection and insect herbivory) and abiotic (e.g., high or low temperature, drought, flood, anaerobic conditions and salinity) stresses. To survive these challenges, plants have developed elaborate mechanisms to perceive external signals and to manifest adaptive responses with proper physiological and morphological changes (Bohnert et al., 1995). Plants exposed to heat and/or low water or drought conditions typically have low yields of plant material, seeds, fruit and other edible products. Some countries of the world consistently have very low rainfall and therefore have problems growing sufficient food crops for their population. Yet it has been observed that some plants survive and thrive in low water environments. It would, therefore, be of great interest and importance to be able to identify genes that confer improved water efficiency characteristics to thereby enable one to create transformed plants (such as crop plants) with modulated water efficiency characteristics to, thereby, better survive high and/or low heat, high and/or low water, and drought or flood conditions. Exogenous application to plants of high concentrations of PEG and mannitol are known to produce osmotic stress resulting in the retardation of growth and vigor and are used to<br>
assess drought responses. Exogenous application of ABA stimulates drought-responses in plants and can, therefore, be an important screen to identify genes that confer improved water efficiency.<br>
In the field of agriculture and forestry efforts are constantly being made to produce plants with an increased growth potential in order to feed the ever-increasing world population and to guarantee the supply of reproducible raw materials. This is done conventionally through plant breeding. The breeding process is, however, both time-consuming and labor-intensive. Furthermore, appropriate breeding programs must be performed for each relevant plant species.<br>
Progress has been made in part by the genetic manipulation of plants; that is by introducing and expressing recombinant nucleic acid molecules in plants. Such approaches have the advantage of not usually being limited to one plant species, but instead being transferable among plant species. (Zhang et al. (2004) Plant Physiol. 135:615). There is a need for generally applicable processes that improve forest or agricultural plant growth potential. Therefore, the present invention relates to a process for increasing the abiotic stress tolerance and consequently the growth potential in plants, characterized by,expression of recombinant DNA molecules stably integrated into the plant genome.<br>
SUMMARY OF THE INVENTION<br>
The present invention, therefore, relates to isolated polynucleotides, polypeptides encoded thereby, and the use of those sequences for making transgenic plants with modulated water use efficiency.<br>
The present invention also relates to processes for increasing the growth potential in plants under abnormal water conditions, recombinant nucleic acid molecules and polypeptides used for these processes and their uses, as well as to plants themselves.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
1.        DEFINITIONS<br>
The following terms are utilized throughout this application:<br>
Constitutive Promoter:         Promoters referred to herein as "constitutive promoters" actively promote transcription under most, but not necessarily all, environmental conditions and states of development or cell differentiation. Examples of constitutive promoters include the cauliflower mosaic virus (CaMV) 35S transcript initiation region and the 1' or 2' promoter derived from T-DNA ofAgrobacterium tumeface ubiqiens, and other transcription initiation regions from various plant genes, such as the maizuitin-1 promoter, known to those of skill.<br>
Domain:        Domains are fingerprints or signatures that can be used to characterize protein families and/or parts of proteins. Such fingerprints or signatures can comprise conserved (1) primary sequence, (2) secondary structure, and/or (3) three-dimensional conformation. Generally, each domain has been associated with either a family of proteins or motifs. Typically, these families and/or motifs have been correlated with specific in-vitro and/or in-vivo activities. A domain can be any length, including the entirety of the sequence of a protein. Detailed descriptions of the domains, associated families and motifs, and correlated activities of the polypeptides of the instant invention are described below. Usually, the polypeptides with designated domain(s) can exhibit at least one activity that is exhibited by any polypeptide that comprises the same domain(s).<br>
Drought: Plant species vary in their capacity to tolerate drought conditions. For each species, optimal growth can be achieved if a certain level of water is always available. Other factors such as temperature and soil conditions have a significant impact on the availability of water to the plant. "Drought" can be defined as the set of environmental conditions under which a plant will begin to suffer the effects of water deprivation, such as decreased photosynthesis, loss of turgor (wilting) and decreased<br>
stomatal conductance. This drought condition results in a significant reduction in yield. Water deprivation may be caused by lack of rainfall or limited irrigation. Alternatively, water deficit may also be caused by high temperatures, low humidity, saline soils, freezing temperatures or water-logged soils that damage roots and limit water uptake to the shoot. Since plant species vary in their capacity to tolerate water deficit, the precise environmental conditions that cause drought stress can not be generalized. However, drought tolerant plants produce higher biomass and yield than plants that are not drought tolerant. Differences in physical appearance, recovery and yield can be quantified and statistically analyzed using well known measurement and analysis methods.<br>
Endogenous: The term "endogenous," within the context of the current invention refers to any polynucleotide, polypeptide or protein sequence which is a natural part of a cell or organisms regenerated from said cell.<br>
Exogenous:    "Exogenous," as referred to within, is any polynucleotide, polypeptide or protein sequence, whether chimeric or not, that is initially or subsequently introduced into the genome of an individual host cell or the organism regenerated from said host cell by any means other than by a sexual cross. Examples of means by which this can be accomplished are described below, and include Agrobacterium-mediated transformation (of dicots - e.g. Salomon et al. EMBO J. 3:141 (1984); Herrera-Estrella et al. EMBOJ. 2:987 (1983); of monocots, representative papers are those by Escudero et al., Plant J. 10:355 (1996), Ishida et al., Nature Biotechnology 14:745 (1996), May et al., Bio/Technology 13:486 (1995)), biolistic methods (Armaleo et al., Current Genetics 17:97 1990)), electroporation, inplanta techniques, and the like. Such a plant containing the exogenous nucleic acid is referred to here as a TO for the primary transgenic plant and T1 for the first generation. The term "exogenous" as used herein is also intended to encompass inserting a naturally found element into a non-naturally found location.<br>
Flood: Plant species vary in their capacity to tolerate flooding. Some plants, such as rice, are cultivated in water while plants such as com do not tolerate flooding. 'Flood," as referred to within, is the state of water saturation at which soils become hypoxic or anoxic, thus limiting respiration in the root. Reduced respiration damages roots and can limit the permeability of roots to water, resulting in decreased leaf water potential and wilting. Since plant species vary in their capacity to tolerate flooding, the precise environmental conditions that cause flood stress can not be generalized. However, flood tolerant plants are characterized by their ability to retain their normal appearance or recover quickly from flood. Such flood tolerant plants produce higher biomass and yield than plants that are not flood tolerant. Differences in physical appearance, recovery and yield can be quantified and statistically analyzed using well known measurement and analysis methods.<br>
Functionally Comparable Proteins: This phrase describes those protejns that have at least one characteristic in common. Such characteristics include sequence similarity, biochemical activity, transcriptional pattern similarity and phenotypic activity. Typically, the functionally comparable proteins share some sequence similarity or at least one biochemical and within this definition, homologs, orthologs and analogs are considered to be functionally comparable. In addition, functionally comparable proteins generally share at least one biochemical and/or phenotypic activity.<br>
Functionally comparable proteins will give rise to the same characteristic to a similar, but not necessarily to the same degree. Typically, comparable proteins give the same characteristics where the quantitative measurement due to one of the comparables is at lest 20% of the other; more typically, between 30 to 40%; even more typically, between 50-60%; even more typically, 70 to 80%; even more typically between 90 to 100%.<br>
Heterologous sequences:       "Heterologous sequences" are those that are not operatively linked or are not contiguous to each other in nature. For example, a promoter from corn is considered heterplogous to saArabidopsis coding region sequence. Also, a promoter from a gene encoding a growth factor from corn is considered heterologous to a sequence encoding the corn receptor for the growth factor. Regulatory element sequences, such as UTRs or 3' end termination sequences that do not originate in nature<br>
from the same gene as the coding sequence originates from, are considered heterologous to said coding sequence. Elements operatively linked in nature and contiguous to each other are not heterologous to each other. On the other hand, these same elements remain operatively linked but become heterologous if other filler sequence is placed between them. Thus, the promoter and coding sequences of a corn gene expressing an amino acid transporter are not heterologous to each other, but the promoter and coding sequence of a corn gene operatively linked in a novel manner are heterologous.<br>
High Temperature: Plant species vary in their capacity to tolerate high temperatures. Very few plant species can survive temperatures- higher than 45°C. The effects of high temperatures on plants, however, can begin at lower temperatures depending on the species and other environmental conditions such as humidity and soil moisture. "High temperature" can be defined as the temperature at which a given plant species will be adversely affected as evidenced by symptoms such as decreased photosynthesis. Since plant species vary in their capacity to tolerate higyh temperature, the precise environmental conditions that cause high temperature stress can not be generalized. However, high temperature tolerant plants are characterized by their ability to retain their normal appearance or recover quickly from high temperature conditions. Such high temperature tolerant plants produce higher biomass and yield than plants that are not high temperature tolerant. Differences in physical appearance, recovery and yield can be quantified and statistically analyzed using well know measurement and analysis methods.<br>
Inducible Promoter: An "inducible promoter" in the context of the current invention refers to a promoter which is regulated under certain conditions, such as light, chemical concentration, protein concentration, conditions in an organism, cell, or organelle, etc. A typical example of an inducible promoter, which can be utilized with the polynucleotides of the present invention, is PARSK1, the promoter from the Arabidopsis gene encoding a serine-threonine kinase enzyme, and which promoter is induced by dehydration, abscissic acid and sodium chloride (Wang and Goodman, Plant J. 8:37 (1995)).<br>
Examples of environmental conditions that may affect transcription by inducible promoters include anaerobic conditions, elevated temperature, or the presence of light. Low Temperature: Plant species vary in their capacity to tolerate low temperatures. Chilling-sensitive plant species, including may agronomically important species, can be injured by cold, above-freezing temperatures. At temperatures below the freezing-point of water most plant species will be damaged. Thus, "low temperature" can be defined as the temperature at which a given plant species will be adversely affected as evidenced by symptoms such as decreased photosynthesis and membrane damage (measured by electrolyte leakage). Since plant species vary in their capacity to tolerate low temperature, the precise environmental conditions that cause low temperature stress can not be generalized. However, low temperature tolerant plants are characterized by their ability to retain their normal appearance or recover quickly from low temperature conditions. Such low temperature tolerant plants produce higher biomass and yield than plants that are not low temperature tolerant. Differences in physical appearance, recovery and yield can be quantified and statistically analyzed using well known measurement and analysis methods.<br>
r<br>
Plant seeds vary considerably in their ability to germinate under low temperature conditions. Seeds of most plant species will not germinate at temperatures less than 10°C. Once seeds have imbibed water they become very susceptible to disease, water and chemical damage. Seeds that are tolerant to low temperature stress during germination can survive for relatively long periods under which the temperature is too low to germinate. Since plant species vary in their capacity to tolerate low temperature during germination, the precise environmental conditions that cause low temperature stress during germination can not be generalized. However, plants that tolerate low temperature during germination are characterized by their ability to remain viable or recover quickly from low temperature conditions. Such low temperature tolerant plants produce, germinate, become established, grow more quickly and ultimately produce more biomass and yield than plants that are not low temperature tolerant. Differences in germination rate, appearance, recovery and yield can be quantified and statistically analyzed using well known measurement and analysis methods.<br>
Masterpool:   The "master pools" discussed in these experiments are a pool of seeds from five different transgenic plants transformed with the same exogenous gene.<br>
Misexpression: The term "misexpression" refers to an increase or a decrease in the transcription of a coding region into a complementary RNA sequence as compared to the wild-type. This term also encompasses expression of a gene or co,ding region for a different time period as compared to the wild-type and/or from a non-natural location within the plant genome.<br>
Percentage of sequence identity:     "Percentage of sequence identity," as used herein, is determined by comparing two optimally aligned sequences over a comparison window, where the fragment of the polynucleotide or amino acid sequence in the comparison window may comprise additions or deletions (e.g., gaps or overhangs) as compared to the reference sequence (which does not comprise additions or deletions) for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical nucleic acid base or amino acid residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison and multiplying the result by 100 to yield the percentage of sequence identity. Optimal alignment of sequences for comparison may be conducted by the local hornology algorithm of Smith and Waterman Add. API. Math. 2:482 (1981), by the homology alignment algorithm of Needleman and Wunsch.7. Mol. Biol. 48:443 (1970), by the search for similarity method of Pearson and Lipman Proc. Natl. Acad. Sci. (USA) 85: 2444 (1988), by computerized implementations of these algorithms (GAP, BESTFTT, BLAST, PASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, WI), or by inspection. Given that two sequences have been identified for comparison, GAP and BESTFTT are preferably employed to determine their optimal alignment. Typically, the default values of 5.00 for gap weight and 0.30 for gap weight length are used. The term "substantial sequence identity" between polynucleotide or polypeptide sequences refers to polynucleotide or polypeptide comprising a sequence that has at least 80% sequence identity, preferably at .east 85%, more preferably at least 90% and most preferably at least 95%, even more<br>
preferably, at least 96%, 97%, 98% or 99% sequence identity compared to a reference sequence using the programs.<br>
Query nucleic acid and amino acid sequences were searched against subject nucleic acid or amino acid sequences residing in public or proprietary databases. Such searches were done using the Washington University Basic Local Alignment Search Tool Version 1.83 (WU-Blast2) program. The WU-Blast2 program is available on the internet from Washington University. A WU-Blast2 service for Arabidopsis can also be found on the internet. Typically the following parameters of WU-Blast2 were used: Filter options were set to "default," Output format was set to "gapped alignments," the Comparison Matrix was set to "BLOSUM62," Cutoff Score (S value) was set to "default," the Expect (E threshold) was set to "default," the Number of best alignments to show was set to "100," and the "Sort output" option was set to sort the output by "pvalue."<br>
Plant Promoter:        A "plant promoter" is a promoter capable of initiating transcription in plant cells and can drive or facilitate transcription of a nucleotide sequence or fragment thereof of the instant invention. Such promoters need not be of plant origin. For example, promoters derived from plant viruses, such as the CaMV35S promoter or from Agrobacterium tumefaciens such as the T-DNA promoters, can be plant promoters. A typical example of a plant promoter of plant origin is the maize ubiquitin-1 (ubi-1) promoter known to those of skill.<br>
Specific Promoter:   In the context of the current invention, "specific promoters" refers to promoters that have a high preference for being active in a specific tissue or cell and/or at a specific time during development of an organism. By "high preference" is meant at least 3-fold, preferably 5-fold, more preferably at least 10-fold still more preferably at least 20-fold, 50-fold or 100-fold increase in transcription in the desired tissue over the transcription in any other tissue. Typical examples of temporal and/or tissue specific promoters of plant origin that can be used with the polynucleotides of the present invention, are: SH-EP from Vigna mungo and EP-C1 from Phaseolus vulgaris (Yamauchi et al. (1996) Plant Mol Biol. 30(2):321-9.); RCc2 and RCc3, promoters that direct root-specific gene transcription in rice (Xu et al., Plant<br>
Mol. Biol. 27:237 (1 995) and TobRB27, a root-specific promoter from tobacco (Yamamoto et al. (1991) Plant Cell 3:371).<br>
Stringency:    "Stringency" as used herein is a function of probe length, probe composition (G + C content), and salt concentration, organic solvent concentration, and temperature of hybridization or wash conditions. Stringency is typically compared by the parameter Tm, which is the temperature at which 50% of the complementary molecules in the hybridization are hybridized, in terms of a temperature differential from Tm. High stringency conditions are those providing a condition of Tm - 5°C to Tm - 10°C. Medium or moderate stringency conditions are those providing Tm - 20°C to Tm - 29°C. Low stringency conditions are those providing a condition of Tm - 40°C to Tm - 48°C. The relationship of hybridization conditions to Tm (in °C) is expressed in the mathematical equation<br>
Tm = 81.5 -le.eOogiotNa*]) + 0.41(%G+C) - (600/N) (1)<br>
where N is the length of the probe. This equation works well for probes 14 to 70 nucleotides in length that are identical to the target sequence. The equation below for Tn of DNA-DNA hybrids is useful for probes in the range of 50 to greater than 500 nucleotides, and for conditions that include an organic solvent (formamide).<br>
Tm = 81.5+16.6 log {[Na+]/(l+0.7[Na*])}+ 0.41(%G+C)-500/L 0.63(%formamide) (2)<br>
where L is the length of the probe in the hybrid. (P. Tijessen, "Hybridization with Nucleic Acid Probes" in Laboratory Techniques in Biochemistry and Molecular Biology, P.C. vand der Vliet, ed., c. 1993 by Elsevier, Amsterdam.) The Tm of equation (2) is affected by the nature of the hybrid; for DNA-KNA hybrids Tm is 10-15°C higher than calculated, for RNA-RNA hybrids Tm is 20-25°C higher. Because the Tm decreases about 1 °C for each 1% decrease in homology when a long probe is used (Bonner et al., J. Mol. Biol. 81:123 (1973)), stringency conditions can be adjusted to favor detection of identical genes or related family members.<br>
Equation (2) is derived assuming equilibrium and therefore, hybridizations according to the present invention are most preferably performed under conditions of probe excess and for sufficient time to achieve equilibrium. The time required to reach equilibrium can be shortened by inclusion of a hybridization accelerator such as dextran sulfate or another high volume polymer in the hybridization buffer.<br>
Stringency can be controlled during the hybridization reaction or after hybridization has occurred by altering the salt and temperature conditions of the wash solutions used. The formulas shown above are equally valid when used to compute the stringency of a wash solution. Preferred wash solution stringencies lie within the ranges stated above; high stringency is 5-8°C below Tm, medium or moderate stringency is 26-29°C below Tm and low stringency is 45-48°C below Tm.<br>
Superpool: As used in the context of the current invention, a "superpool" refers to a mixture of seed from 100 different "master pools". Thus, the superpool contains an equal amount of seed from 500 different events, but only represents 100 transgenic plants with a distinct exogenous nucleotide sequence transformed into them, because the master pools are of 5 different events with the same exogenous nucleotide sequence transformed into them.<br>
T0: As used in the current application, the term "To" refers to the whole plant, explant or callus tissue inoculated with the transformation medium.<br>
TJ: As used in the current application, the term TI refers to either the progeny of the TO plant, in the case of wholerplant transformation, or the regenerated seedling in the case of explant or callous tissue transformation.<br>
T2: As used in the current application, the term T2 refers to the progeny of the T1 plant. T2 progeny are the result of self-fertilization or cross pollination of a T1 plant.<br>
T3: As used in the current application, the term T3 refers to second generation progeny of the plant that is the direct result of a transformation experiment. T3 progeny are the result of self-fertilization or cross pollination of a T2 plant.-<br>
2.         IMPORTANT CHARACTERISTICS OF THE<br>
POLYNUCLEOT1DESAND POLYPEPTBDES OF THE INVENTION<br>
The polynucleotides and polypeptides of the present invention are of interest because when they are misexpressed (i.e. when expressed at a non-natural location or in an increased or decreased amount) they produce plants with modified water use efficiency. "Water use efficiency" is a term that includes various responses to environmental conditions that affect the amount of water available to the plant. For example, under high heat conditions water is rapidly evaporated from both the soil and from the plant itself, resulting in a decrease of available water for maintaining or initiating physiological processes. Likewise, water availability is limited during cold or drought conditions or when there is low water content in the soil. Interestingly, flood conditions also affect the amount of water available to the plant because it damages the roots and thus limits the plant's ability to transport water to the shoot. As used herein, modulating water use efficiency is intended to encompass all of these situations as well as other environmental situations that affect the plant's ability to use and/or maintain water effectively (e.g. osmotic stress, salinity, etc.).<br>
The polynucleotides and polypeptides of the invention, as discussed below and as evidenced by the results of various experiments, are useful for modulating water use efficiency. These traits can be used to exploit or maximize plant products for agricultural, ornamental or forestry purposes in different environment conditions of water supply. Modulating the expression of the nucleotides and polypeptides of the present invention leads to transgenic plants that will require less water and result in better yield in high heat and/or drought conditions, or that have increased tolerance levels for an excess of water and result in better yield in wet conditions. Both categories of transgenic plants lead to reduced costs for the farmer and better yield in their respective environmental conditions.<br>
3.        THE POLYNUCLEOTIDES AND POLYPEPTIDES OF THE<br>
INVENTION<br>
The polynucleotid.es of the invention, and the proteins expressed thereby, are set forth in the Sequence Listing. Some of these sequences are functionally comparable proteins.<br>
Functionally comparable proteins are those proteins that have at least one characteristic in common. Such characteristics can include sequence similarity, biochemical activity and phenotypic activity. Typically, the functionajly comparable proteins share some sequence similarity and generally share at least one biochemical and/or phenotypic activity. For example, biochemical functionally comparable proteins are proteins that act on the same reactant to give the same product.<br>
Another class of functionally comparable proteins is phenotypic functionally comparable proteins. The members of this class regulate the same physical characteristic, such as increased drought tolerance. Proteins can be considered phenotypic functionally comparable proteins even if the proteins give rise to the same physical characteristic, but to a different degree.<br>
The polypeptides of the invention also include those comprising the consensus sequences described in Tables 1-4,2- 8,3-9,5-8, 6-8,7-6, 8-7,10-6 and 11-6. A consensus sequence defines the important conserved amino acids and/or domains within a polypeptide. Thus, all those sequences that conform to the consensus sequence are suitable for the same purpose. Polypeptides comprised of a sequence within and defined by one of the consensus sequences can be utilized for the purposes of the invention namely to make transgenic plants with improved water use efficiency, including improved tolerance to heat or high or low water conditions.<br>
4.   USE OF THE POLYNUCLEOTIDES AND POLYPEPTIDES TO MAKE TRANSGENIC PLANTS<br>
To use the sequences of the present invention or a combination of them or parts and/or mutants and/or fusions and/or variants of them, recombinant DNA constructs are prepared which comprise the polynucleotide sequences of the invention inserted into a vector, and which are suitable for transformation of plant cells. The construct can be made using standard recombinant DNA techniques (Sambrook et al. 1989) and can be introduced to the species of interest by Agrobacterium-mediated transformation or by other means of transformation as referenced below.<br>
The vector backbone can be any of those typical in the art such as plasmids, viruses, artificial chromosomes, BACs, YACs and PACs and vectors of the sort described by<br>
(a)	BAC: Shizuya et al., Proc. Nati. Acad. Sci. USA 89: 8794-8797 (1992);<br>
Hamilton et al., Proc. Natl. Acad. Sci. USA 93: 9975-9979 (1996);<br>
(b)	YAC: Burke et al., Science 236:806-812 (1987);<br>
(c)	PAC:   SternbergN. etal., Proc Nati Acad Sci USA. Jan;87(l):103-7 (1990);<br>
(d)	Bacteria-Yeast Shuttle Vectors: Bradshaw et al., Nucl Acids Res 23:4850-<br>
4856(1995);<br>
(e)	Lambda Phage Vectors: Replacement Vector, e.g., Frischauf etal., J. Mol<br>
Biol 170: 827-842 (1983); or Insertion vector, e.g., Huynh et al., In: Glover<br>
NM (ed) DNA Cloning: A practical Approach, Vol.1 Oxford: IRL Press<br>
(1985); T-DNA gene fusion vectors :Walden et al., Mol Cell Biol 1: 175-194<br>
(1990); and<br>
(g)      Plasmid vectors: Sambrook et al., infra.<br>
Typically, the construct comprises a vector containing a sequence of the present invention with any desired transcriptional and/or translational regulatory sequences, such as promoters, UTRs, and 3' end termination sequences. Vectors can also include origins of replication, scaffold attachment regions (SARs), markers, homologous sequences, introns, etc. The vector may also comprise a marker gene that -confers a selectable phenotype on plant cells. The marker typically encodes biocide resistance, particularly antibiotic resistance, such as resistance to kanamycin, bleomycin, hygromycin, or herbicide resistance, such as resistance to glyphosate, chlorosulfuron or phosphinotricin.<br>
A plant promoter is used that directs transcription of the gene in all tissues of a regenerated plant and may be a constitutive promoter, such as p326 or CaMV35S. Alternatively, the plant promoter directs transcription of a sequence of the invention in a specific tissue manner (tissue-specific promoter) or is otherwise under more precise environmental control (inducible promoter). Various plant promoters, including constitutive, tissue-specific and inducible, are known to those skilled in the art and can be utilized in the present invention. Typically, preferred promoters to use in the present invention are those that are induced by heat or low water conditions Such as the RD29a<br>
promoter (Kasuga et al., Plant Cell Physiol. 45:346 (2004) and Yamaguchi-Shinozaki and Shinozaki, Mol Gen Genet. 236:331 (1993)) or other DRE-containing (dehydration-responsive elements) promoters (Liu et al, Cell 10: 1391 (1998)). Another preferred embodiment of the present invention is the use of root specific promoters such as those present in the AtXTH17, AtXTH18, AtXTH19 and AtXTH20 genes ofArabidopsis (Vissenberg et al. (2005) Plant Cell Physiol 46:192) or guard cell specific promoters such as TGG1 or KST1 (Husebye et al. (2002) Plant Physiol 128:1180; Plesch et al. (2001) Plant 728:455).<br>
Alternatively, misexpression can be accomplished using a two component system, whereby the first component comprises a transgenic plant comprising a transcriptional activator operatively linked to a promoter and the second component comprises a transgenic plant comprising a sequence of the invention operatively linked to the target binding sequence/region of the transcriptional activator. The two transgenic plants are crossed and the sequence of the invention is expressed in their progeny. In another alternative, the misexpression can be accomplished by transforming the sequences of the two component system into one transgenic plant line.<br>
Any promoter that functions in plants can be used in the first component, such as those discussed above. Suitable transcriptional activator polypeptides include, but are not limited to, those encoding HAP1 and.GAIA The binding sequence recognized and targeted by the selected transcriptional activator protein (e.g. a UAS element) is used in the second component.<br>
Nucleotide sequences of the invention are introduced into the genome or the cell of the appropriate host plant by a variety of techniques. These techniques for transforming a wide variety of higher plant species are well known and described in the technical and scientific literature. See, e.g. Weising et al., Ann. Rev. Genet. 22:421 (1988); andChristou, Euphytica, v. 85,n.l-3:13-27, (1995).<br>
Processes for the transformation and regeneration of monocotyledonous and dicotyledonous plants are known to the person skilled in the art. For the introduction of DNA into a plant host cell a variety of techniques is available. These techniques include transformation of plant cells by injection (e.g. Newell, 2000), microinjection<br>
(e.g. Griesbach (1987) Plant Sci. 50 69-77), electroporation of DNA (e.g. Fromm et al. (1985) Proc. Natl Acad. Sci. USA 82:5824 and Wan and Lemaux, Plant Physiol. 104 (1994), 37-48), PEG (e.g. PaszkowsM et al. (1984) EMBOJ. 3:2717), use of biolistics (e.g. Klein et al. (1987) Nature 327:773), fusion of cells or protoplasts (Willmitzer, L., 1993 Transgenic plants. In: Biotechnology, A Multi-Volume Comprehensive Treatise (HJ. Rehm, G. Reed, A. Piihler, P. Stadler, eds., Vol. 2, 627-659, VCH Weinheim-New York-Basel-Cambridge), via T-DNA using Agrobacterium tumefaciens (e.g. Fraley et al. (Crit Rev. Plant. Sci. 4, 1-46 and Fromm et al., Biotechnology 8 (1990), 833-844) or Agrobacterium rhizogenes (e.g. Cho et al. (2000) Planta 210: 195-204) or other bacterial hosts (e.g. Brootghaerts et al. (2005) Nature 433:629-633), as well as further possibilities.<br>
In addition, .a number of non-stable transformation methods well known to those skilled in the art may be desirable for the present invention. Such methods include, but are not limited to, transient expression (e.g. Lincoln et al. (1998) Plant Mol. Biol. Rep. 16:1-4) and viral transfection (e.g. Lacomme et al. (2001) In "Genetically Engineered Viruses" (C.J.A. Ring and E.D. Blair, Eds). Pp. 59-99, BIOS Scientific Publishers, Ltd. Oxford, UK).<br>
Seeds are obtained from the transformed plants and used for testing stability and inheritance. Generally, two or more generations are cultivated to ensure that the phenotypic feature is stably maintained and transmitted.<br>
One of skill will recognize mat after the expression cassette is stably incorporated in transgenic plants and confirmed to be operable, it can be introduced into other plants by sexual crossing. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed.<br>
The nucleic acids of the invention can be used to confer the trait of increased tolerance to heat and/or low water conditions, without reduction in fertility, on essentially any plant.<br>
The nucleotide sequences according to the invention encode appropriate proteins from any organism, in particular from plants, fungi, bacteria or animals.<br>
The process according to the invention can be applied to any plant, preferably higher plants, pertaining to the classes ofAngiospermae and Gymnospermae. Plants of the subclasses of the Dicotylodenae and the Monocotyledonae are particularly<br>
suitable. Dicotyledonous plants belong to the orders of the Magniolales, Uliciales, Laurales, Piperales Aristochiales, Nymphaeales, Ranunculales, Papeverales, Sarraceniaceae, Trochodendrales, Hamamelidales, Eucomiales, Leitneriales, Myricales, Fagales, Casuarinales, Caryophyllales, Batales, Polygonales, Plumbaginales, Dilleniales, Theales, Malvales, Urticales, Lecythidales, Violales, Salicales, Capparales, Ericales, Diapensales, Ebenales, Primulales, Resales, Fabales, Podostemales, Haloragales, Myrtales, Cornales, Proteales, Santales, Rafflesiales, Celastrales, Euphorbiales, Rhamnales, Sapindales, Juglandales, Geraniales, Polygalales, Umbellales, Gentianales, Polemoniales, Lamiales, Plantaginales, Scropkulariales, Campanulales, Rubiales, Dipsacales, and Asterales. Monocoryledonous plants belong to the orders of the Alismatales, Hydrocharitales, Najadales, Triuridales, Commelinales, Eriocaulales, Restionales, Poales, Juncales, Cyperales, Typhales, Bromeliales, Zingiberales, Arecales, Cyclanthales, Pandanales, Arales, Lilliales, and Orchidales. Plants belonging to the class of the Gymnospermae are finales, Ginkgoales, Cycadales and Gnetales.<br>
The method of the invention is preferably used with plants that are interesting for agriculture, horticulture, biomass for byconversion and/or forestry. Examples are tobacco, oilseed rape, sugar beet, potato, tomato, cucumber, pepper, bean, pea, citrus fruit, apple, pear, berries, plum, melon, eggplant, cotton, soybean, sunflower, rose, poinsettia, petunia, guayule, cabbage, spinach, alfalfa, artichoke, com, wheat, rye, barley, grasses such as switch grass or turf grass, millet, hemp, banana, poplar, eucalyptus trees, conifers.<br>
Homologs Encompassed by the Invention<br>
Sequences of the invention include proteins comprising at least about a contiguous 10 amino acid region preferably comprising at least about a contiguous 20 amino acid region, even more preferably comprising at least about a contiguous 25, 35, 50, 75 or 100 amino acid region of a protein of the present invention. In another preferred embodiment, the proteins of the present invention include between about 10 and about 25 contiguous amino acid region, more preferably between about 20 and about 50 contiguous amino acid region, and even more preferably between about 40<br>
and about 80 contiguous amino acid region.<br>
Due to the degeneracy of the genetic code, different nucleotide codons may be used to code for a particular amino acid. A host cell often displays a preferred pattern of codon usage. Nucleic acid sequences are preferably constructed to utilize the codon usage pattern of the particular host cell. This generally enhances the expression of the nucleic acid sequence in a transformed host cell. Any of the above described nucleic acid and amino acid sequences may be modified to reflect the preferred codon usage of a host cell or organism in which they are contained. Modification of a nucleic acid sequence for optimal codon usage in plants is described in U.S. Pat. No. 5,689,052. Additional variations in the nucleic acid sequences may encode proteins having equivalent or superior characteristics when compared to the proteins from which they are engineered.<br>
It is understood that certain amino acids may be substituted for other amino acids in a protein or peptide structure (and the nucleic acid sequence that codes for it) without appreciable change or loss of its biological utility or activity. The amino acid changes may be achieved by changing the codons of the nucleic acid sequence.<br>
It is well known in the art that one or more amino acids in a native sequence can be substituted with other amino acid(s), the charge and polarity of which are similar to that of the native amino acid, i.e., a conservative amino acid substitution, resulting in a silent change. Conservative substitutes for an amino acid within the native polypeptide sequence can be selected from other members of the class to which the amino acid belongs (see below). Amino acids can be divided into the following four groups: (1) acidic (negatively charged) amino acids, such as aspartic acid and glutamic acid; (2) basic (positively charged) amino acids, such as arginine, histidine, and lysine; (3) neutral polar amino acids, such as glycine, serine, threonine, cysteine, cystine, tyrosine, asparagine, and glutamine; and (4) neutral nonpolar (hydrophobia) amino acids such as alanine, leucine, isoleucine, valine, proline, phenylalam'ne, tryptophan, and methionine.<br>
In a further aspect of the present invention, nucleic acid molecules of the present invention can comprise sequences that differ from those encoding a protein or fragment thereof selected from the group consisting of the sequences present in the Sequence Listing due to the fact that the different nucleic acid sequence encodes a protein having one or more conservative amino acid changes.<br>
In another aspect, biologically functional equivalents of the proteins or fragments thereof of the present invention can have about 10 or fewer conservative amino acid changes, more preferably about 7 or fewer conservative amino acid changes, and most preferably about 5 or fewer conservative amino acid changes. In a preferred embodiment, the protein has between about 5 and about 500 conservative changes, more preferably between about 10 and about 300 conservative changes, even more preferably between about 25 and about 150 conservative changes, and most preferably between about 5 and about 25 conservative changes or between 1 and<br>
about 5 conservative changes.<br>
i<br>
5.        EXPERIMENTS CONFIRMING THE USEFULNESS OF THE POLYNUCLEOTIDES AND POLYPEPTIDES OF THE INVENTION<br>
5.1 Procedures<br>
The nucleotide sequences of the invention were identified by use of a variety of screens for modified water conditions, including heat and/or low water conditions. These screens are recognized by those skilled in the art to be predictive of nucleotide sequences that provide plants with improved water use efficiency including improved tolerance to heat and/or low water conditions because they emulate the different environmental conditions that can result from increased heat and/or low water conditions. These screens generally fall into two categories (1) soil screens and (2) in vitro screens.<br>
Soil screens have the advantage of assaying the response of the entire plant to particular conditions, such as drought or high heat. On the other hand, in vitro screens have the advantage of relying on defined media and so allow more defined manipulation of growth conditions. "Surrogate" in vitro screens use particular<br>
chemicals to alter the water available to the plant by manipulating the concentrations and/or components of the growth media. For example, the ability of the plant to maintain the water concentration within its cells, which can occur during times of low water in the soil, can be tested by growing plants on high sucrose media. Such a screen thus allows one to separate the effects of water loss from roots from, for example, the water loss from leaves during high heat conditions. Each of the screens used is described in more detail below.<br>
In general, the screens used to identify the polynucleotides and polypeptides of the invention were conducted using superpools of Arabidopsis T2 transformed plants. The T1 plants were transformed with a Ti plasmid containing a particular SEQ ID NO in the sense orientation relative to a constitutive promoter and harboring the plant-selectable marker gene phosphinothricin acetyltransferase (PAT), which confers herbicide resistance to transformed plants. For surrogate screens, seed from multiple superpools (1,200 T2 seeds from each superpool) were tested. T3 seed were collected from the resistant plants and retested on all other surrogate screens. The results of the screens conducted for each SEQ ID NO can be found in the Examples below.<br>
5.1.1.  MANNTTOL<br>
Screens for mannitol resistant seedlings are surrogate screens for drought (see Quesada et al., Genetic analysis of salt-tolerant mutants in Arabidopsis thaliana. Genetics. 2000 154:421-36).<br>
Seeds are sterilized in 30% household bleach for 5 minutes and then washed with double distilled deionized water three times. Sterilized seed is stored in the dark at 4°C for a miniumum of 3 days before use.<br>
Manitol media is prepared by mixing 375 ml sterile ImM mannitol with 375 ml sterile IX MS. Approximately 1200 seeds are evenly spaced per PEG plate before incubating at 22°C for 14 days.<br>
Putative mannitol-resistant seedlings are transferred to MS with 3% sucrose for recovery. Approximately one week later, resistant seedlings are transferred to soil and sprayed with Finale. Finale resistant plants are genotyped as described below.<br>
DNA is isolated from each plant and used in PCR reactions using the following cycling conditions 95°C for 30 sec; five cycles of 51°C for 30 sec, 72°C for 1.15 min,<br>
95°C for 30 sec, 25 cycles of 48°C for 30 sec, 72°C for 1,15 min, 72°C for 7 min and 4°C hold. Aliquots of the reaction product are analyzed on a 1.2% agarose gel stained with ethidium bromide.<br>
T3 Seed from those plants containing the expected PCR product are collected and retested on 375 mM mannitol media.<br>
5.1.2.   POLYETHYLENE GLYCOL (PEG)<br>
Screens for PEG resistant seedlings are surrogate screens for drought (see van der Weele et al., Growth of Arabidopsis thaliana seedlings under water deficit studied by control of water potential in nutrient-agar media. JExp Bot. 2000 51(350): 1555-62).<br>
Seeds are sterilized in 30% household bleach for 5 minutes and then washed with double distilled deionized water three times. Sterilized seed is stored in the dark at 4°C for a miniumum of 3 days before use.<br>
18% PEG media is prepared by mixing 360 ml of hot sterile 50% PEG with 400 ml of hot sterile 0.5X MS media. Approximately 1200 seeds are evenly spaced per PEG plate before incubating at 22°C for 14 days.<br>
Putative PEG-resistant seedlings are transferred to MS with 0.01% Finale. One week later, resistant seedlings are transferred to soil. Three days later the seedlings are genotyped as described below.<br>
DNA is isolated from each plant and used in PCR reactions using the following cycling conditions 95°C for 30 sec, five cycles of 51°C for 30 sec, 72°C for 1.15 min, 95°C for 30 sec, 25 cycles of 48°C for 30 sec, 72°C for 1,15 min, 72°C for 7 min and 4°C hold. Aliquots of the reaction product are analyzed on a 1.2% agarose gel stained with ethidium bromide.<br>
Ta Seed from those plants containing the expected PCR product are collected and retested using 20% PEG media.<br>
5.1.3.       SOIL DROUGHT<br>
Soil drought screens identify plants with enhanced tolerance to drought and enhanced recovery after drought.<br>
Seeds are planted in holed flats containing Zonolite vermiculite that are placed in no-holed flats. Flats are watered with 3 L of Hoagland's solution and covered with a<br>
plastic dome before being placed at at 4°C. After 4 days, the flats are moved to the Greenhouse (22°C) and grown 2 weeks, with 1.5 L Hoagland's solution being added every 4 days, or when top of vermiculite is dry. The final application of Hoagland's solution is 4 days prior to the end of the 2 weeks. After 2 weeks, 1L Hoagland's solution is added to the no-holed flat. After 10 days plants are wilted but still green.<br>
Green, turgid plants are transplanted to 4" square pots containing 60% sunshine mix #5 and 40% thermo-o-rock vermiculite, with Osmocote (1 tbsp / 8L) and Marathon (1 tbsp / 8L). The soil is moistened and the pots sub-irrigated with water. They are grown under a plastic dome for one day, then the plastic dome is removed for the remaining growth period.<br>
To assess plants for enhanced recovery after drought the green wilted plants remaining in the flats are sub-irrigated with 2.5 L Hoagland's solution and cover with cleared with a plastic dome. The following day, the dome is removed and green survivors transplanted to 4" square pots containing 60% sunshine mix #5,40% Therm-o-rock vermiculite, Osmocote (1 tbsp/8 L) and marathon (1 tbsp/8 L). The soil is moistened and the pots sub-irrigated with water. They are grown under a plastic dome for one day, then the plastic dome is removed for the remaining growth period.<br>
DNA from a leaf from each plant is transferred to FTA paper via pressure and an aliquot of the DNA containing paper used in PCR reactions using the following cycling conditions 94°C for 10 min, five cycles of 94°C for 30 sec, 60°C for 30 sec, 72°C for 3 min, five cycles of 94°C for 30 sec, 60°C for 30 sec, 72°C for 3 min, 30 cycles of 94°C for 30 sec, 53°C for 30 sec, 72°C for 3 min, 72°C for 7 min and 4°C hold. Aliquots of the reaction product are analyzed on a 1% agarose gel stained with ethidium bromide.<br>
T3 Seed from those plants containing the expected PCR product are collected and retested using 50 seeds from each line.<br>
5.1.4.   HEAT<br>
High heat screens identify plants .with enhanced tolerance to heat and enhanced recovery after heat.<br><br>
Seeds are sterilized in 30% household bleach for 5 minutes and then washed with double distilled deionized water three times. Sterilized seed is stored in the dark at 4°C for a miniumum of 3 days before use.<br>
MS media, pH 5.7 is prepared. Approximately 12 seeds are evenly spaced per MS plate before incubating in the vertical position at 22°C for 14 days. Under these conditions, the plates are exposed to 12,030 LUX from above and 3,190 LUX from the bottom.<br>
On day 15 the plates are transferred to a 22°C oven, which increased temperature in 5°C increments to 45°C. The duration of treatment at 45°C was based on complete and homogenous wilting of 100 wild-type seedlings (~10 plates). After exposure to 45°C, seedlings were placed in the horizontal position at 23°C for recovery, where they remained for 4-11 days. Heat recovery was assessed based on vigor and greenness and continued growth after treatment<br>
Leaves of control and non-resistant plants become wilted and yellowed after only 2 days, completely bleaching after an additional 4 days. All wild type (WS) and non-heat resistant plants die by day 6.<br>
DNA is isolated from each plant and used in PCR reactions using the following cycling conditions 95°C for 30 sec, five cycles of 51°C for 30 sec, 72°C for 1.15 min, 95°C for 30 sec, 25 cycles of 48°C for 30 sec, 72°C for 1,15 min, 72°C for 7 min and 4°C hold. Aliquots of the reaction product are analyzed on a 1.2% agarose gel stained with ethidium bromide.<br>
Ta Seed from those plants containing the expected PCR product are collected and retested.<br>
To differentiate between natural acquired thermo tolerance of recovered T3 events, seeds were sterilized and stratified hi parallel to wildtype seed that was never heat treated and wild-type controls previously heat treated with the T2 events. 15 day old seedlings were heat shocked in the dark at 45°C for 5 hours, as described above. Duration of treatment at 45°C was based on complete and homogenous wilting of preheat treated wild-type seed and un-pretreated wild-type controls. After exposure to 45°C, seedlings were returned to the permissive temperature of 23 °C for recovery where they remained for another 7 days. Thermo tolerance was assessed based on prolonged greenness and continued growth after treatment.<br>
5.1.5	HEAT (SOIL)<br>
Seeds are sown in pots containing soil of the following composition: 60% autoclaved Sunshine Mix #5,40% vermiculite with 2.5 Tbsp Osmocote and 2.5 Tbsp 1% granular Marathon per 25 L of soil. After sowing, pots are covered with plastic propagation domes and seed is placed at 4°C in the dark for at least 3 days. Pots are then returned to the greenhouse (long day light conditions of 16 hours), covered with 55% shade cloth and provided a normal watering regime.<br>
After 7 days, seedlings were transferred to a 36°C growth chamber under continuous light and allowed to grow until harvest. Plants were watered minimally so as to allow for some drying of the top soil similar to that in heat-induced drought conditions in die field.<br>
Plants are sprayed with a mixture of 3 ml Finale in 48 oz of water. Spraying is repeated every 3-4 days until only transformants remain. The remaining transfonnants were weeded to a maximum of 5 evenly spaced transformants per pot.<br>
T3 seed was recovered and tested for thermotolerance and recovery as described above.<br>
5.1.6	HIGH SUCROSE<br>
Screens for germination and growth on limited nutrients and 9% sucrose are surrogate screens for the altered carbon/nitrogen balance frequently associated with drought (see Laby et al., The Ardbidopsis sugar-insensitive mutants sis4 and sisS are defective in abscisic acid synthesis and response. Plant Journal 23: 587-596).<br>
Seeds are sterilized in 30% household bleach for 5 minutes and then washed with double distilled deionized water three times. Sterilized seed is stored in the dark at 4°C for a miniumum of 3 days before use.<br>
MS media containing 9% sucrose is prepared. Approximately 1200 seeds are evenly spaced per MS-sucrose plate before incubating at 22°C for 9 days.<br>
Putative sucrose-resistant green seedlings are transferred to MS plates. After one week of recovery, resistant the seedlings are genotyped as described below.<br>
DNA from a leaf from each plant is transferred to FTA paper via pressure and an aliquot of the DNA containing paper used in PCR reactions using the following cycling conditions 94°C for 10 min, five cycles of 94°C for 30 sec, 60°C for 30 sec, 72°C for 3 min, five cycles of 94°C for 30 sec, 60°C for 30 sec, 72°C for 3 min, 30 cycles of 94°C for 30 sec, 53°C for 30 sec, 72°C for 3 min, 72°C for 7 min and 4°C hold. Aliquots of the reaction product are analyzed on a 1% agarose gel stained with ethidium bromide.<br>
T3 Seed from those plants containing the expected PCR product are collected and retested using 9% sucrose MS media.<br>
5.1.7.   ABA<br>
Screens for ABA resistant seedlings are surrogate screens for drought.<br>
Seeds are sterilized in 30% household bleach for 5 minutes and then washed with double distilled deionized water three times. Sterilized seed is stored in the dark at 4°C for a miniumum of 3 days before use.<br>
MS media containing 1.5 µM ABA is prepared. Approximately 1200 seeds are evenly spaced per PEG plate before incubating at 22°C for 14 days.<br>
Putative ABA-resistant seedlings are transferred to MS with 0.01% Finale. One week later, resistant seedlings are transferred to soil. Three days later the seedlings are genotyped as described below.<br>
DNA is isolated from each plant and used in PCR reactions using the following cycling conditions 95°C for 30 sec, five cycles of 51DC for 30 sec, 72°C for 1.15 min, )5°C for 30 sec, 25 cycles of 48°C for 30 sec, 72°C for 1.15 min, 72°C for 7 min and \°C hold. Aliquots of the reaction product are analyzed on a 1.2% agarose gel stained vith ethidium bromide.<br>
Ts Seed from those plants containing the expected PCR product are collected and stested using 1.5 µM ABA MS media.<br>
5.1.8.        PROCEDURE FOR IDENTIFYING FUNCTIONAL HOMOLOGS AND CONSENSUS SEQUENCES<br>
The isolated sequence of the invention was compared to the sequences present in the various gene banks. Pairwise comparisons were conducted and those sequences • having the highest percent identity to the query sequence identified as functional homologs.<br>
A multi-pairwise alignment was generated using the amino acid query sequence and the amino acid sequence of the functional homologs. This allowed identification of the conserved regions or domains of the polypeptide. Using the conserved regions as a guide, a consensus sequence was generated. This consensus sequence indicates the critical amino acid residues and those can be either substituted and/or deleted without impacting the biological function of the protein.<br>
5.2 Results<br>
The results of the above experiments are set forth below wherein each individual example relates to all of the experimental results for a particular polynucleotide/polypeptide of the invention.<br>
Example 1 - Ceres cDNA 12331850<br>
Clone 11830, Ceres cDNA 12331850, encodes a full-length glycosyl hydrolase family 17 protein, which has similarity to elicitor inducible chitinase Nt-SubE76 GI: 11071974 from (Nicotiana tabacum (C-terminal homology only) and |3 -1,3-glucanase.<br>
Ectopic expression of Ceres cDNA 12331850 under the control of the CaMV35S or 32449 promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG),<br>
mannitol, and abscissic acid (ABA). o   Continued growth on high concentration of PEG, mannitol, and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 12331850.<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 12331850 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338,<br>
contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the T1 generation. No positive or negative phenotypes were observed in the T1 plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. T3 seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 12331850 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing mis cDNA (ME01297) were screened on high PEG, mannitol, and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M=mannitol, and A = ABA) and the number following the letter refers to a number assigned to each plant obtained from that screen on mat supeipool. For example, SP1-M18 is the 18* plant isolated from a mannitol screen of Superpool 1.<br>
RESULTS:<br>
Qualitative analysis of 13 Superpools on PEG, mannitol, and ABA<br>
Resistant candidates were selected based on increased size compared to the largest wild-type control. All three screens resulted in a decrease in germination for both wildtype and superpools as compared to seeds on control media. Wild-type seeds that germinated on any of the screens were small and never developed any first leaves even after 40 days. To ensure that even slightly tolerant individuals were not omitted, seedlings that showed any growth and greening whatsoever were recovered and transferred to soil for assessment in the T3 generation. All recovered candidates<br>
showed signs of vigorous re-growth on soil, although the development was slightly delayed as compared to unstressed plants, presumably because of the transient exposure to stress. The plants transferred to soil were sprayed with BastaR to eliminate any false-positives, or any lines where the BastaR marker was suppressed. All of the BastaR -resistant candidates flowered and set seed. Resistant seedlings were recovered from Superpools 1,7 and 11 on all screens (Table 1-1).(Table Removed) <br>
Qualitative and quantitative analysis of 5 independent events representing 35S::cDNA 12331850 on PEG, mannitol, and ABA screens.<br>
Seedlings that survived transfer to soil and which were BastaR -resistant were subjected to PCR and sequencing. At least one resistant plant in each of the three screens contained 35S::cDNA 12331850 (ME01297) making this a good candidate for further testing. T2 seeds from the 5 independent transformants that contain this clone and that were used in the pooling process were tested for BastaR resistance and for stress tolerance in the 3 surrogate drought screens.<br>
To identify two independent events of 35S::cDNA 12331850 showing PEG, mannitol and ABA resistance, 36 seedlings from each of five events, ME01297-01, 02,   03, 04, and 05 were screened as previously described. Simultaneously, BastaR segregation was assessed to identify events containing a single insert segregating in a<br>
3:1 (R:S) ratio as calculated by a phi-square test (Table 1-2). All of the events segregated for a single functional insert.<br>
Table 1-2. Basta segregation for ME01297 individual events.<br>
(Table Removed)<br>
Events MEO 1297-02 and 03 were chosen as the two events because they had the strongest and most consistent resistance to PEG, mannitol and ABA. Resistance was observed for MEO 1297-01,04, and 05 although not in expected ratios in all three screens (data not shown). The controls were sown the same day and on the same plate as the individual events. The transgenic control in each of these plates is a segregant, from this ME line or another ME line being tested, that failed to show resistance to the particular stress. The PEG, mannitol and ABA (Table 1-3) segregation ratios observed for ME01297-02 and 03 are consistent with the presence of a single insert, similar to what we observed for Basta resistance (Table 1-1).<br>
On 18% PEG, the resistant seedlings from these two events show some root growth and they are green with new leaves emerging. The mannitol resistant seedlings also showed more root and shoot growth than the sensitive seedlings. The ABA resistant seedlings showed a slight increase in growth. The phenotype of the resistant seedlings is unique to each of the screens.<br>
Table 1-3. Segregation of Resistance to PEG, mannitol and ABA in ME01297-02 and ME01297-03 Progeny. .(Table Removed)<br><br>
Qualitative and quantitative analysis of progeny of T2 plants isolated on high concentrations of PEG, mannitol, and ABA screens.<br>
Progeny from Ta plants that were recovered from the three screens and which contained cDNA 12331850 (SP1-A1, SP1-P1, and SP1-M18) were analyzed and also found to be resistant to PEG, mannitol and ABA indicating that resistance is transmitted to the next generation. Taken together, 1) the isolation of resistant seedlings containing cDNA 12331850 from all three screens, 2) the inheritance of this resistance in a subsequent generation, and 3) the fact that the progeny from two or more events from the original transformation also segregated for resistance to the stresses, provide strong evidence that cDNA 12331850 when over-expressed can provide tolerance to osmotic stress.<br>
This gene is annotated as a glycosyl hydrolase family 17, which has similarity to elicitor inducible chitinase Nt-SubE76 GI: 11071974 from Nicotiana tabacum (C-terminal homology only). TIGR also notes that the protein contains similarity to beta-l,3-glucanase.<br>
Table 1-4 provides the results of the consensus sequence analysis based on Ceres cDNA 12331850.<br>
.(Table Removed)<br>
Example 2 - Ceres cDNA 12334963<br>
Clone 35743, Ceres cDNA 12334963, encodes a full-length putative hypothetical protein. Ectopic expression of Ceres cDNA 12334963 under the control of the 35S promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG), mannitol,<br>
and abscissic acid (ABA), o   Continued growth on high PEG, mannitol, and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 12334963.<br>
Wild-type Arabidopsis Wassilewsldja (WS) plants were transformed with a Ti plasmid containing cDNA 12334963 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the T1 generation. No positive or negative phenotypes were observed in me T1 plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol and ABA) as described above. T3 seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 12334963 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (ME01467) were screened on high PEG, mannitol and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M=mannitol, and A=ABA) and the number<br>
following the letter refers to a number assigned to each plant obtained from that screen on that superpool. For example, SP1-M18 is the 18th plant isolated from a mannitol screen of Superpool 1.<br>
RESULTS:<br>
Qualitative analysis of 13 Superpools on PEG, mannitol, and ABA,<br>
Resistant candidates were selected based on increased size when compared to the largest wild-type control seedlings. All three screens resulted in a decrease in germination for both wildtype and superpools compared to seeds on control media. Wild-type seeds that germinated on any of the screens were small and never developed any first leaves even after 40 days. To ensure that even slightly tolerant individuals were not omitted, seedlings that showed any growth and greening whatsoever were recovered and transferred to soil for assessment in the T3 generation. All recovered candidates showed signs of vigorous re-growth on soil, although the development was slightly delayed as compared to unstressed plants, presumably because of the transient exposure to stress. The plants transferred to soil were sprayed • with BastaR to eliminate any false-positives, or any lines where the BastaR marker was suppressed. All of the BastaR -resistant candidates flowered and set seed. Resistant seedlings were recovered from Superpools 1, 7 and 11 on all screens (Table 2-1).<br>
Table 2-1. Number of stress-tolerant and BastaR seedlings identified on drought surrogate screens.(Table Removed)<br>
Qualitative and quantitative analysis of 5 independent events representing 35S::cDNA 12334963 on PEG, mannitol, and ABA<br>
Seedlings that survived transfer to soil and which were BastaR -resistant were subjected to PCR and sequencing. At least one resistant plant in each of the three osmotic screens contained 35S::cDNA 12334963 (ME01467) making this a good candidate for further testing. Ta seeds from the 5 independent transformants that contain this clone and that were used in the pooling process were tested for BastaR resistance and for stress tolerance in the 3 surrogate drought screens.<br>
To identify two independent events of 35S::cDNA 12334963 showing PEG, mannitol and ABA resistance, 36 seedlings from each of five events, ME01467-01, 02,03, 04, and 05 were screened as previously described. Simultaneously, Basta segregation was assessed to identify events containing a single insert segregating in a 3:1 (R:S) ratio as calculated by a Chi-square test (Table 2-2). All of the lines segregated for. a single functional insert<br>
.(Table Removed)<br>
Events MEO1467-03 and 05 were chosen as the two events because had the strongest and most consistent resistance to PEG, mannitol and ABA. Resistance was observed for MEO 1467-01, 02, and 04, although not in expected ratios in all three screens (data not shown). The controls were sown the same day and in the same plate as the individual events. The transgenic control in each of these plates is a segregant, from this ME line or another ME line being tested, that failed to show resistance to the particular osmotic stress. The PEG (Tables 2-3 and 2-4), mannitol (Tables 2-5 and 2-6) and ABA (Tables 2-7 and 2-8) segregation ratios observed for MEO 1467-03 and 05 are consistent with the presence of a single insert as demonstrated by Chi-Square. This result is similar to the observation for BastaR resistance (Table 2-2). .(Table Removed)<br>
 of the resistant seedlings is unique on each of the screens.<br>
Table 2-4. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01467-05 containing 35S::cPNA 12334963 on PEG. .(Table Removed)<br>
Table 2-5. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01467-03 containing 35S::cDNA 12334963 on mannitol.<br>
.(Table Removed)<br>
Table 2-6.  Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01467-05 containing 35S::cDNA 12334963 on mannitol. .<br>
(Table Removed)   Table2-7. Chi-square analysis assmning a 3:1 (RjS) ratio for progeny of ME01467-03 Table 2-8. Chi-square analysis assuminga 3:1 (R: S) ratio for progeny of ME01467-05<br>
.(Table Removed)<br>
containing 35S::cDNA 12334963 on ABA. containing J5S::cDNA 12334963 on ABA. Qualitative and quantitative analysis of progeny of Ta plants isolated on PEG, mannitol and ABA<br>
Progeny of the T2 plants that were recovered from the three screens and which contained cDNA 12334963 (SP1-A12, SP1-P9, and SP1-M4) were analyzed and found to be resistant to PEG, mannitol and ABA, indicating that resistance is transmitted to the next generation.<br>
Taken together, 1) the isolation of resistant seedlings containing cDNA 12334963 from all three surrogate screens for drought, 2) the inheritance of this resistance in the next generation, and 3) the fact mat the progeny from two or more events from the original transformation also segregated for resistance to these osmotic stresses, provides strong evidence that cDNA 12334963, when over-expressed, can provide tolerance to drought,-freezing and other osmotic stresses.<br>
Table 2-8 provides the results of the consensus sequence analysis based on Ceres cDNA 1233                                                         .(Table Removed)<br>
  Example 3 - Ceres cDNA 12333678<br>
Clone 26006, Ceres cDNA 12333678, encodes a full-length glycosyl hydrolase. Ectopic expression of Ceres cDNA 12333678 under the control of the CaMV35S promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG), mannitol<br>
and abscissic acid (ABA). o   Continued growth on high PEG, mannitol and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 12333678.<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 12333678 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the T1 generation. No positive or negative phenotypes were observed in the Ti plants.<br>
Screens of superpools on high PEG, mannitol and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol and ABA) as described above. T3 seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 12333678 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (ME01334) were screened on high PEG, mannitol and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M = mannitol, and A = ABA) and the number<br>
following the letter refers to a number assigned to each plant obtained from that screen on that superpool. For example, SP1-M18 is the 18* plant isolated from a mannitol screen of Superpool 1.<br>
RESULTS:<br>
Qualitative analysis of 13 Superpools on PEG, mannitol and ABA.<br>
Resistant candidates were selected based on increased size when compared to the largest wild-type control seedlings. All three screens resulted in a decrease in germination for both wildtype and superpools as compared to seeds on control media. Wild-type seeds that germinated on any of the screens were small and never developed any first leaves even after 40 days. To ensure that even slightly tolerant individuals were not omitted, seedlings that showed any growth and greening whatsoever were recovered and transferred to soil for assessment in the T3 generation. All recovered candidates showed signs of vigorous re-growth on soil, although the development was slightly delayed compared to unstressed plants, presumably because of the transient exposure to stress. The plants transferred to soil were sprayed with B BastaR to eliminate any false-positives, or any lines where the BastaR marker was suppressed. All of the BastaR -resistant candidates flowered and set seed. Resistant seedlings were recovered from Superpools 1,7 and 11 on all screens (Table 3-1).<br>
Table 3-1. Number of stress-tolerant and BastaR seedlings identified on drought surrogate screens. .(Table Removed)<br>
Qualitative and quantitative analysis of 5 independent events representing 35S::cDNA 12333678 on PEG, mannitol and ABA<br>
Seedlings that survived transfer to soil and which were BastaR-resistant were subjected to PCR and sequencing. At least one resistant plant in each of the three osmotic screens contained 35S::cDNA 12333678 (ME01334). T2 seeds from the 5 independent transformants that contain this clone and that were used in the pooling process were tested for BastaR resistance and for stress tolerance in the 3 surrogate drought screens.<br>
To identify two independent events of 35S::cDNA 12333678 showing PEG, mannitol, and ABA resistance, 36 seedlings from each of four events, MEO1334-01, 02,03, and 04 were screened as previously described. Simultaneously, BastaR segregation was assessed to identify events containing a single insert segregating in a 3:1 (R:S) ratio as calculated by a Chi-square test (Table 3-2). All of the events tested segregated for a single functional insert.Table 3-2. Basta* segregation for MEO 1334 individual events<br>
(Table Removed) Events MEO 1334-01 and 04 were chosen as the two events because they had the strongest and most consistent resistance to PEG, mannitol and ABA. Resistance was observed for ME01334-02 and 03 although not in expected ratios in all three screens (data not shown). The controls were sown the same day and in the same plate as the individual events. The transgenic control in each of these plates is a segregant, from this ME line or another ME line being tested, that failed to show resistance to the particular stress. The PEG (Tables 3-3 and 3-4), mannitol (Tables 3-5 and 3-6) and ABA (Tables 3-7 and 3-8) segregation ratios observed for ME01334-01 and 01 are consistent with the presence of a<br>
single insert as demonstrated by Chi-Square, This is similar to that observed for BastaR resistance (Table 3-2).<br>
On 18% PEG, the resistant seedlings from these two events show some root growth but they are also green with new leaves emerging at day 14. The mannitol-resistant seedlings showed more root and shoot growth than the PEG resistant seedlings. The resistant seedlings on ABA show the least amount of growth, and very little root growth relative to the mannitol and PEG screens. The phcnotype of the resistant seedlings is unique on each of the screens.<br>
Table 3-3. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01334-01 containing 35S::cDNA 12333678 on PEG. .(Table Removed)<br>
Table 3-4. Chi-square analysis assuming a 3:1 (RS) ratio for progeny of ME01334-04 containing 35S::cDNA 12333678 on PEG. .(Table Removed)<br>
Table 3-5. Chi-square analysis assuming a 3:1 (R:S) ratio forprogeny of ME01334-01 containing 35S::cDNA 12333678 on mannitol. .(Table Removed)<br>
Table 3-6. Chi-square analysis assuming a 3:1 (R:S) ratio forprogeny of ME01334-04 containing 35S::cDNA 12333678 on mannitol. .(Table Removed)<br>
Table 3-7. Chi-square analysis assuming a 3:1 (R:S) ratio forprogeny of ME01334-01 containing 35S::cDNA 12333678 on ABA. .(Table Removed)<br>
Table 3-8. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01334-04 containing 35S::cDNA 12333678 on ABA. .(Table Removed)<br>
Qualitative and quantitative analysis of progeny of T2 plants isolated on PEG, mannitol and ABA<br>
Progeny from T2 plants that were recovered from the three screens and contained cDNA 12333678 (SP1-A2, SP1-P3, and SP1-M19) were analyzed and also found to be resistant to PEG, mannitol and ABA indicating that resistance is transmitted to the next generation. Taken together, 1) the isolation of resistant seedlings containing cDNA 12333678 from all three surrogate screens for drought, 2) the inheritance of this resistance in the next generation, and 3) the feet that the progeny from two or more events from the original transformation also segregated for resistance to these osmotic stresses, provides strong evidence that cDNA 12333678 when over-expressed provides tolerance to osmotic stress.<br>
This gene is annotated as an alpha/beta hydrolase, a probable acetone-cyanohydrin lyase. Acetone-cyanohydrin lyase is involved in the catabolism of cyanogenic glycosides.<br>
Table 3-9 provides the results of the consensus sequence analysis based on Ceres cDNA 12333678. <br>
.(Table Removed)<br><br>
Example 4 - Ceres cDNA 12384873<br>
Clone 34419, Ceres cDNA 12384873, encodes a full-length strictosidine synthase. Ectopic expression of Ceres cDNA 12384873 under the control of the CaMV35S promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene^glycol (PEG), mannitol,<br>
and abscissic acid (ABA), o   Continued growth on high PEG, mannitol, and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 12384873<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 12384873in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phehotype in the T1 generation. No positive or negative phenotypes were observed in the T1 plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. T3 seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 12384873 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (ME01490) were screened on high PEG, mannitol and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P=PEG, M= mannitol and A=ABA) and the number following the letter refers to a number assigned to each plant obtained from that screen on<br>
that superpool. For example, SP1-M18 is the 18th plant isolated from a mannitol screen of Superpool 1.<br>
Qualitative analysis of 13 Superpools on PEG, maunitol,and ABA.<br>
Resistant candidates were selected based on increased size when compared to the largest wild-type control seedlings. All three screens resulted in a decrease in germination for both wildtype and superpools as compared to seeds on control media. Wild-type seeds that germinated on any of thr, screens were small and never developed any first leaves even after 40 days. To ensure that even slightly tolerant individuals were not omitted, seedlings that showed any growth and greening whatsoever were recovered and transferred to soil for assessment in the Ta generation. All recovered candidates showed signs of vigorous re-growth on soil, although the development was slightly delayed as compared to unstressed plants, presumably because of the transient exposure to stress. The plants transferred to soil were sprayed with BastaR to eliminate any false-positives, or any lines where the BastaR marker was suppressed. All of the BastaR -resistant candidates flowered and set seed. Resistant seedlings were recovered from Superpools 1,  and 11 on all screens (Table 4-1).<br>
Table 4-1. Number of stress-tolerant and BastaR seedlings identified on drought surrogate screens. .(Table Removed)<br>
Qualitative and quantitative analysis of 5 independent events representing 35S::cDNA 12384873 on PEG, mannitol and ABA<br>
Seedlings that survived transfer to soil and which were BastaR -resistant were subjected to PCR and sequencing. At least one resistant plant in each of the three osmotic screens contained 35S::cDNA 12384873 (ME01490g. T2 seeds from the 5 independent transformants that contain this clone and that were used in the pooling process were tested for BastaR resistance and for stress tolerance in the 3 surrogate drought screens.<br>
To identify two independent events of ?5S::cDNA 12384873 showing PEG, mannitol and ABA resistance, 36 seedlings from each of five events, MEO1490-01, 02,03,04, and 05 were screened as previously described. Simultaneously, BastaR segregation was assessed to identify events containing a single insert segregating in a 3:1 (R:S) ratio as calculated by a Chi-square test (Table 4-2). Three of the events segregated for a single functional insert (-01,-02 and -04). For the other two events one segregated for two independent inserts (-03) and one segregated for a deficiency of BastaR seedlings (-05).<br>
. Table 4-2. BastaR segregation for ME01490 individual events.(Table Removed)<br>
Events MEO 1490-01 and -02 were chosen as the two events because they had the strongest and most consistent resistance to PEG, mannitol and ABA. Resistance was observed for ME01490-03, -04, and -05 although not in expected ratios in all three screens (data not shown). The controls were sown the same day and in the same plate as the individual lines. The transgenic control in each of these plates is a segregant, from this ME line or another ME line being tested, that failed to show resistance to the particular stress. The PEG (Tables 4-3 and 4-4), mannitol (Tables 4-5 and 4-6) and ABA (Tables 4-7 and 4-8) segregation ratios observed for ME01490-01 and -02 are consistent with the presence of single insert, as demonstrated by Chi-square. This result is similar to that observed for BastaR resistance (Table 4-2).<br>
Table 4-3. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01490-01 containing 3SS::cDNA 12384873 on PEG. .(Table Removed)<br>
Table 4-4. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01490-02 containing 35S::cDNA 12384873 on PEG. .(Table Removed)<br>
Table 4-5. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01490-01 containing 35S::cDNA 12384873 on mannitol. .(Table Removed)<br>
Table 4-6.  Chi-square analysis assuming a 3:1 (R: S) ratio for progeny of ME01490-02 containing 35S::cDNA 12384873 on mannitol. .(Table Removed)<br>
Table 4-7. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01490-01 containing 35S::cDNA 12384873 on ABA. .(Table Removed)TabIe4-8. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME01490-02 containing 35S::cDNA 12384873 on ABA. .(Table Removed)<br>
Qualitative and quantitative analysis of progeny of TI plants isolated on PEG, mannitol and ABA<br>
Progeny of the T2 plants that were recovered from the three screens and contained cDNA 12384873 (SP1-A18, SP1-P14, SP1-M5, SP1-M6 and SP1-M7) were analyzed and also found to be resistant to PEG, mannitol and ABA indicating that resistance is transmitted to the next generation. Taken togeihei, 1) the isolation of resistant seedlings containing cDNA 12384873 from all three surrogate screens for drought, 2) the inheritance of this resistance in the next generation, and 3) the fact that the progeny from two or more events from the original transformation also segregated for resistance to these osmotic stresses, provides strong evidence that cDNA 12384873 when over-expressed can provide tolerance to osmotic stress.<br>
Clone 34419 encodes the first 29 amino acids of a strictosidine synthase protein, and then a frame shift results in a novel stretch of 63 amino acids on the 3' end of the protein.<br>
Example 5 - Ceres cDNA 12659859<br>
Ceres cDNA 12659859 encodes aFAD-linked oxidoreductase family, a probable berberine bridge enzyme from Arabidopsis thaliana. Ectopic expression of Ceres cDNA 12659859 under the control of the CaMV35S promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG),<br>
mannitol and abscissic acid (ABA), o   Continued growth on high concentration of PEG, mannitol and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 12659859.<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 12659859 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the TI generation. No positive or. negative phenotypes were observed in the T! plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. T3 seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 12659859 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (SR01010) were screened on high PEG, mannitol, and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M — mannitol, and A = ABA) and the number following the letter refers to a number assigned to each plant obtained from that screen on that superpool. For example, SP 1-M18 is the 18th plant isolated from a mannitol screen of Superpool 1.<br>
Qualitative and quantitative analysis of 2 independent events representing 35S::cDNA 12659859 (SR01010) on PEG, mannitol and ABA<br>
To identify two independent events of 35S::cDNA 12659859 showing PEG, mannitol, and ABA resistance, 36 seedlings from each of three events, SR01010-01, 02, and 03 were screened as previously described. BastaR segregation was assessed to identify lines containing a single insert segregating in a 3:1 (R:S) ratio as calculated<br><br>
by a Chi-square test (Table 5-1). Two of three TI generation events (01 and 03) segregated for a single insert although the segregation ratio for -01 is also not different than al5:l(R:S)ratio.<br>
Table 5-1. BastaR segregation for SR01010 individual events.(Table Removed)<br>
Lines SR0101-01 and -03 were chosen as the two events because they had a strong and consistent resistance to PEG, mannitol and ABA. Resistance was observed for SRO1010-02 although not in expected ratios in all three screens (data not shown). The controls were sown the same day and in the same plate as the individual lines. The PEG (Tables 5-2 and 5-3), mannitol (Tables 5-4 and 5-5) and ABA (Tables 5-6 and 5-7) segregation ratios observed for SR01010-01 and -03 are consistent with the presence of single insert as demonstrated by chi-square, similar to what we observed for BastaR resistance (Table 5-1).<br>
The progeny from one resistant T2 plant from each of these two events was tested in the T3 generation in the same manner. Resistance to PEG, mannitol and ABA was also observed in the T3 generation. Taken together, the segregation of resistant seedlings containing cDNA 12659859 from two events on all three drought surrogate screens and the inheritance of this resistance in a subsequent generation, provide strong evidence that cDNA 12659859 when over-expressed can provide tolerance to drought<br>
Table 5-2. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01010-01T2 containing 35S: :cDNA 12659859 on PEG. .(Table Removed)<br>
Table 5-3. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01010-03 T2 containing 35S::cDNA 12659859 on PEG. .(Table Removed)<br>
Table 5-4.  Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01010-01T2 containing 35S::cDNA 12659859 onmannitol. .(Table Removed)<br>
Table 5-5. Chi-square analysis assuming a 3:1 (R-'S) ratio for progeny of SR01010-3 T2 containing 35S::cDNA 12659859 on mannitol. .(Table Removed)<br>
Table 5-6.  Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01010-01T2 containing 35S::cDNA 12659859 on ABA. .(Table Removed)<br>
Table 5-7. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01010-03 T2 containing 35S::cDNA 12659859 on ABA. .(Table Removed)<br>
Table 5-8 provides the results of the consensus sequence analysis based on Ceres cDNA 12659859.<br>
.(Table Removed)<br>
Example 6 - Ceres cDNA 12723147<br>
Ceres cDNA 12723147 encodes an Arabidopsis putative aldo/keto reductase. Ectopic expression of Ceres cDNA 12723147 under the control of the CaMV35S promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG),<br>
mannitol and abscissic acid (ABA). o   Continued growth on high concentration of PEG, mannitol and ABA.<br>
Generation and phenotypic evaluation of Tj lines containing 35S::cDNA 12723147.<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 12723147 in the sense orientation relative to the CaMV35S constitutive promoter. The Tj plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the T1 generation. No positive or negative phenotypes were observed in the TI plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. Ta seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 12723147 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (SRO1013) were screened on high PEG, mannitol, and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M = mannitol, and A = ABA) and the number following the letter refers to a number assigned to each plant obtained from that screen on<br>
that superpool. For example, SP1-M18 is the 18th plant isolated from a mannitol screen of Superpool 1.<br>
Qualitative and quantitative analysis of 2 independent events representing 35S::cDNA 12659859 (SR01010) on PEG, mannitol and ABA<br>
To identify two independent events of 35S::cDNA 12659859 showing PEG, mannitol, and ABA resistance, 36 seedlings from each of two events, SR01013-01 and -02 were screened as previously described. BastaR segregation was assessed to verify that the lines contained a single insert segregating in a 3:1 (R:S) ratio as calculated by a chi-square test (Table 6-1). Both lines (01 and 02) segregated for a single insert in the T2generation (Table 1) .(Table Removed)<br>
Lines SR01013-01 and -02 were chosen as the two events because they had a strong and consistent resistance to PEG, mannitol and ABA. The controls were sown the same day and in the same plate as the individual lines. The PEG (Tables 6-2 and 6-3), mannitol (Tables 6-4 and 6-5) and ABA (Tables 6-6 and 6-7) segregation ratios observed for SRO1013-01 and -02 are consistent with the presence of single insert as demonstrated by chi-square, similar to what we observed for BastaR resistance (Table 6-1).<br>
The progeny from one resistant T2 plant from each of these two events were tested in the same manner as the T2. Resistance to PEG, mannitol and ABA was also observed in the T3 generation. Taken together, the segregation of resistant seedlings containing cDNA 12723147 from two events on all three drought surrogate screens and the inheritance of this resistance in a subsequent generation, provide strong evidence that cDNA 12723147 when over-expressed can provide tolerance to drought.<br>
Table 6-2. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01013-01T2 containing 35S::cDNA 12723147 on PEG. .(Table Removed)<br>
Table 6-3. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01013-02 T2 containing 35S::cDNA 12723147 on PEG. .(Table Removed)<br>
Table 6-4. Chi-square analysis assuming a 3:1 (R:S) ratio forprogeny of SR01013-01 containing 35S::cDNA 12723147 on mannitol. .(Table Removed)<br><br>
Table 6-5. Chi-square analysis assuming a 3:1 (P.:S) ratio forprogeny of SR01013-02 Ti containing 35S::cDNA 12723147 on mannitol. .(Table Removed)<br>
Table 6-6. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01013-02 T2 containing 35S::cDNA 12723147 on ABA. .(Table Removed)<br>
Table 6-7.. Ghi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01013-02 T2 containing 35S::cDNA 12723147 on ABA. .(Table Removed)<br>
Table 6-8 provides the results of the consensus sequence analysis based on Ceres cDNA 12723147. .(Table Removed)<br>
Example 7 - Ceres cDNA 13488750<br>
Clone 125039, Ceres cDNA 13488750, encodes a full-length putative adenylylsulfate (APS) kinase ftomArabidopsis thaliana... Ectopic expression of Ceres cDNA 13488750 under the control of the CaMV35S promoter induces the following phenotypes:<br>
o   Continued growth under high heat conditions.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 13488750.<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 13488750 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and. confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative pheriotype in the TI generation. No negative phenotypes were observed in the T] plants although three of the TI lines produced a small rosette (ME02526-01, 02 and 05). T2 and T3 lines of events ME02526-01 and ME02526-05 did not show the small rosette phenotype.<br>
Screens of masterpools for heat tolerance via heat shock in vitro.<br>
Seeds from 100 masterpools from the CaMV35S or 32449 over-expression<br>
lines were tested for heat tolerance in vitro as described above-Once cDNA 13488750 was identified in tolerant plants from the screen, five<br>
individual T2 events containing this cDNA (ME02526) were screened on soil as described<br>
above to identify events with the resistance phentrype.<br>
Qualitative analysis of the T2 masterpool of cDNA 13488750 plants heat shocked on plates<br>
Visual phenotyping of the masterpool containing cDNA 13488750 (ME02526) on agar or soil showed no visible alterations in phenotype (data not shown). After heat-shock at 15 days of age, the ME02526 masterpool showed greater heat recovery as compared to the wild-type control and other transgenic masterpools. Assessment was a measure of "greenness" as well as continued growth at 23°C after heat shock at 45°C for between 5 and<br>
8 hours. Immediately after the heat shock stress, the extent of stress-induced damage in the control and ME02526 masterpool appeared comparable. The leaves and cotyledons were wilted and droopy although still green. However, after 4 days of recovery, 2 of 10 plants in the ME02526 masterpool had completely recovered and were growing again.  Other seedlings showed some recovery as measured by greenness compared to the wild-type control.<br>
Qualitative and Quantitative analysis of individual T2 events of cDNA 1348^750 under continual heat treatment on soil<br>
Five independent events of ME02526 were tested on soil as described above. Two of the events (ME02526-04 and -05) showed heat resistance after continual<br>
growth at 36°C. Heat resistance was noted as decreased chlorosis compared to wild-type. Segregation frequencies of the transgene under test suggest that these two events contain a single insert, as calculated by a chi~square test (Tables 7-1, 7-2 and 7-3). Ten and 11 plants from events 04 and 05, respectively, showed continued vigor and decreased chlorosis after continuous heat treatment compared to (wild-type controls.<br>
Table 7-1. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME02526-04 and -05<br>
containing 35S::cDNA 13488750 on Finale. .(Table Removed)<br>
Table 7-2. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME02526-04 containing 35S::cDNA 13488750 for thermo tolerance (continual growth at 36°C on soil). .(Table Removed)<br>
Table 7-3. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME02526-05 containing 35S::cDNA 13488750 for thermo tolerance (continual growth at 36°C on soil).<br>
.(Table Removed)<br>
The plants that survive the heat treatment show premature bolting and reduced fecundity but much less so than the control plants. Control and ME02526 plants bolt after only 4 days of exposure to 36°C (11 -day old plants), but were more vigorous and less chlorotic than the controls and the height and branch number was comparable to those of wild-type (data not shown). Heat-tolerant lines all showed seed abortion and reduced fecundity (data not shown). Seed abortion and reduced silique size was also prevalent in all wild-type controls (data not shown). Events 04 and 05, which had a thermo tolerant phenotype in the T2 generation, were evaluated in greater detail in the T3 generation for heat resistance and fecundity after prolonged heat stress on MS plates.<br>
Qualitative and Quantitative analysis of individual T3 events under 36°C heat treatment on plates<br>
Seeds from five individuals of the T3 generation for ME02526-04 and -05 lines and controls were sterilized, stratified and germinated for 7 days at 23°C prior to exposure to 36°C heat stress. The events were evaluated for heat resistance to prolonged heat stress.<br>
The thermo tolerant phenotype became apparent after 15 days of 36°C treatment. T3 progeny from ME02526-04 (Table 7-4) and ME02526-05 (Table 7-5) were found to segregate in the expected 3:1 ratio for the thermo tolerant phenotype.<br>
Table 7-4. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of ME02526-04 containing 35S::cDNA 13488750 for tomo tolerance (continual growth at 36°C.(Table Removed)<br>
Table 7-5. Chi-square analysis assuming a 3:1 (R:S) ratio forprogeny of ME02526-05 containing 35S::cDNA 13488750 for thermo tolerance (continual growth at 36°C.<br>
(Table Removed)<br><br>
Qualitative analysis of individual T3 events heat snocked on plates for differentiation of natural acquired thermotolerance<br>
.Plants acquire thermotolerance to lethal high temperatures such as 45°C if previously exposed to moderately high temperature or if the temperature is raised gradually to an otherwise lethal temperature (Vierling, 1991). To ascertain whether the thermotolerance observed in the TS generation is due to some naturally acquired thermotolerance imparted by heat exposure of the T2 parent plant (ME2526-04 and 05), thermotolerance was assessed by comparing pre-heat treated wild-type and transgenic controls and unheated wild-type controls. Tj events of ME02526-04 and 05 were heat shocked for 4 hours as described above. ME02526-04 and -05 were able to stay greener longer than both pre-heat treated controls and un-heat treated controls. However, both ME02526 lines and all controls (wild-type and transgenic) failed to elongate and thrive after heat treatment at 45°C and eventually all died. Un-heat treated wild-type control became chlorotic faster (1-day after treatment) than pre-heat treated wildtype control and pre-heat treated transgenic control suggesting that there is some natural acquired thermotolerance that occurs that is not correlated with the over-expression of 35S::cDNA 13488750. Even with exposure to this lethal temperature, ME02526 was greener after 7 days than controls, indicating that ME02526 has a thermotolerance phenotype that is unrelated to the natural mechanisms of acquired thermotolerance.<br>
Table 7-6 provides the results of the consensus sequence analysis based on Ceres cDNA 13488750.<br>
.(Table Removed)<br>
Example 8 - Ceres cDNA 13489782<br>
Clone 10044, Ceres cDNA 13489782, encodes a full-length putative 114-amino acid hypothetical protein from Arabidopsis thaliana. Ectopic expression of Ceres cDNA 13489782 under the control of the 32449 promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG) and abscissic<br>
acid (ABA) and o   Continued growth on high concentrations of PEG and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 32449: rcDNA 13489782<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 13489782 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the TI generation. No positive or negative phenotypes were observed in the TI plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. TB seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 13489782 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (ME00446) were screened on high PEG, mannitol, and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M = mannitol, and A = ABA) and the number following the letter refers to a number assigned to each plant obtained from that screen on<br>
that superpool. For example, SPl-M18isthe 18th plant isolated from a mannitol screen of<br>
Superpool 1.<br>
Qualitative analysis of superpools on PEG, mannitol, and ABA.<br>
Resistant candidates were selected based on increased size when compared to the largest wild-type control seedlings. All three screens resulted in a decrease in germination for both wildtype and superpools as compared to seeds on control media. Wild-type seeds that germinated on any of the screens were small and never developed any first leaves even after 40 days. To ensure that even slightly tolerant individuals were not omitted, seedlings that showed any growth and greening whatsoever were recovered and transferred to soil for assessment in the T3 generation. All recovered candidates showed signs of vigorous re-growth on soil, although the development was slightly delayed as compared to unstressed plants, presumably because of the transient exposure to stress. The plants transferred to soil were sprayed with Basta to eliminate any false-positives, or any lines where the BastaR marker was suppressed. All of the Basta-resistant candidates flowered and set seed. Resistant seedlings were recovered from Superpools 1, 7 and 11 on all screens (Table 8-1).<br>
Table 8-1. Number of stress-tolerant and BastaR seedlings identified on drought surrogate screens.<br>
.(Table Removed)<br>
We obtained sequence from 3, 3, and 13 plants that were both BastaR and resistant to PEG, mannitol or ABA, respectively. For each of the three surrogate drought screens, one or more plants contained the 32449:: clone 10044 (ME00446). The probability of finding a plant containing this cDNA at random in all three screens is 0.03 x 0.03 x 0.03.<br>
Qualitative and quantitative analysis of 6 independent events representing 32449: rcDNA 13489782 on PEG, mannitol and ABA<br>
To identify independent events of 32449:: cDNA 13489782 showing PEG, mannitol and ABA resistance, 36 seedlings from each of six events, ME00446-01,02, 03,04,05 and 06 were screened as previously described. Simultaneously, BastaR segregation was assessed to identify lines containing a single insert segregating in a 3:1 (R:S) ratio as calculated by a Chi-square test (Table 8-2). All of the lines segregated for a single functional insert. .<br>
(Table Removed)<br>
Events ME00446-02 and 04 were chosen as the two events for further analysis because they had the strongest and most consistent resistance to PEG and ABA. None of the lines showed mannitol resistance at 375mM concentration. The controls were sown the same day and in the same plate as the individual lines. The PEG (Tables 8-3 and 8-4) and ABA (Tables 8-5 and 8-6) segregation ratios observed for ME00446-02 and-04 are consistent with the presence of single insert as demonstrated by the Chi-Square test. This is similar to that observed for Basta resistance (Table 8-2).<br>
Despite the fact that this line was isolated from all three screens, it was subsequently concluded that it could not be considered mannitol resistant. This is likely due to an overly stringent mannitol concentration. In a superpool screen setting, the<br>
seedlings are more densely grown than in an individual line setting. This means that in a superpool screen, there is a lower effective concentration of mannitol. When putative tolerant plant from a superpool is tested as an individual line, the effective concentration it is grown on is actually higher. In the case of ME00446, this difference was enough to invalidate it as mannitol tolerant. In fact, a resistant plant to mannitol was isolated from superpool 11 that corresponds to clone 10044. (Table Removed)<br>
 (Table Removed)<br>
 (Table Removed)<br>
 (Table Removed)<br>
 Qualitative and quantitative analysis of progeny of Ta plants isolated on PEG, mannitol and ABA<br>
Progeny from T2 plants that were recovered from the three screens and containing clone 10044 (SP11-M13 and SP11-P5) were found to be resistant to PEG and ABA. Taken together, 1) the isolation of resistant seedlings containing clone 10044 from all three surrogate screens for drought, 2) the inheritance of this resistance in the next generation, and 3) the fact that the progeny from two or more events from the original transformation also segregated for resistance to these osmotic stresses, provide strong evidence that clone 10044 when over-expressed provides resistance to osmotic and dehydration stress.<br>
Table 8-7 provides the results of the consensus sequence analysis based on Ceres cDNA 13489782.<br>
(Table Removed)<br>
Example 9 - Ceres cDNA 13486759:<br>
Clone 10987, corresponding Ceres cDNA 13486759, encodes anArabidopsis 251-amino acid expressed protein. Ectopic expression of clone 10987 under the control of the CaMV35S promoter induces the following phenotypes:<br>
•	Germination on high concentrations of polyethylene glycol (PEG), mannitol, and<br>
abscissic acid (ABA), and<br>
•	Continued growth on high concentrations of PEG, mannitol, and ABA.<br>
Generation and phenotypic evaluation of TI lines containing 35S::cDNA 13486759<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 13486759 in the sense orientation relative to the CaMV35S constitutive promoter. The Ti plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the TI generation. No positive or negative phenotypes were observed in the Ti plants.<br>
Screens of superpools on high PEG, mannitol and ABA as surrogate screens for drought tolerance<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. TS seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
Once cDNA 13486759 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (ME03316) were screened on high PEG, mannitol, and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) are referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M = mannitol, and A = ABA) and the number<br>
following the letter refers to a number assigned to each plant obtained from that screen on that superpool. For example, SP1-M18 is the 18th plant isolated from a mannitpl screen of Superpool 1.<br>
Qualitative analysis of 13 Superpools on PEG, mannitol and ABA<br>
Resistant candidates were selected based on increased size when compared to the largest wild-type control seedlings. All three screens resulted in a decrease in germination for both wildtype and superpools as compared to seeds on control media. Wild-type seeds that germinated on any of the screens were small and never developed any first leaves even after 40 days. To ensure that even slightly tolerant individuals were not omitted, seedlings that showed any growth and greening whatsoever were recovered and transferred to soil for assessment in the T3 generation. All recovered candidates showed signs of vigorous re-growth on soil, although the development was slightly delayed as compared to unstressed plants, presumably because of the transient exposure to stress. The plants transferred to soil were sprayed with Basta to eliminate any false-positives, or any lines where the BastaR marker was suppressed. All of the Basta-resistant candidates flowered and set seed. Resistant seedlings were.recovered from Superpools 1, 7 and 11 on all screens (Table 9-1).<br>
Table 9-1. Number of stress-tolerant and BastaR seedlings identified on drought surrogate screens. (Table Removed)<br>
We obtained sequence from 3, 3, and 13 plants from Superpool 1 1 that were both BastaR - resistant and resistant to PEG, mannitol or ABA, respectively. For each<br>
of the three osmotic screens, one or more plants contained the 35S::clone 10987 (ME03316), which made it a good candidate for farther testing. The probability of finding a plant containing this clone 10987 at random in all three screens is 0.03 x 0.03.<br>
Qualitative and quantitative analysis of 5 independent events representing 35S::cDNA 13486759 on PEG, mannitol and ABA<br>
To identify independent events of 35S::cDNA 13486759 showing PEG, mannitol and ABA resistance, 36 seedlings from each of five events, ME03316-01,-02,-03,-04, and -05 were screened as previously described. Simultaneously, BastaR segregation was assessed to identify events containing a single insert segregating in a 3:1 (R:S) ratio as calculated by a chi-square test (Table 9-2). All of the events segregated for a single functional insert. ME03316-02 could be segregating for two linked or unlinked inserts but the ratios on the surrogate drought screens indicate it is likely to be a single insert. <br>
(Table Removed)<br>
Lines ME03316-01 and 02 were chosen as the two events for further analysis because they had the strongest and most consistent resistance to PEG, mannitol and ABA. Resistance was observed for ME03316-03, 04, and 05 although not in expected ratios in all three screens (data not shown). The controls were sown the same day and on the same plate as the individual lines. The PEG (Tables 9-3 and 9-4), mannitol (Tables 9-5 and 9-6) and ABA (Table 9-7) segregation ratios observed are consistent with the presence of a single insert as demonstrated by chi-square. ME03316-02 seedlings on ABA (Table 9-8) appear to be segregating for two inserts which is still consistent with the ratio observed on BastaR. Both events segregate for a deficiency of resistant seedlings on mannitol. (Table Removed) (Table Removed) (Table Removed) (Table Removed) (Table Removed) <br><br>
Qualitative and quantitative analysis of progeny of T2 plants isolated on PEG, mannitol and ABA<br>
Progeny from T2 plants that were recovered from the three screens and containing clone 10987 (SP11-A15, SP11-A16, SP11-P2, and SP11-M10) were found to be resistant to PEG, mannitol and ABA.<br>
On PEG, the progeny of SP11 -M10 segregated for a deficiency of resistant seedlings similar to the deficiency that noted for the T2 seedlings in Tables 9-5 and 9-6. A deficiency of resistant seedlings is also noted for the progeny of SP11P2 on PEG.<br>
Taken together, 1) the isolation of resistant seedlings containing clone 10987 from airthree surrogate screens for drought, 2) the inheritance of this resistance in me next generation, and 3) the fact that the progeny from two or more events from the original transformation also segregated for resistance to these osmotic stresses, these findings provide strong evidence that clone 10987 when over-expressed can provide tolerance to osmotic stresses.<br>
Example 10 - Ceres cDNA 13500101<br>
Clone 17206, corresponding to Ceres cDNA 13500101, encodes a putative strictosidine synthase. Ectopic expression of cDNA 13500101 under the control of the CaMV35S promoter induces the following phenotypes:<br>
o   Germination on high concentrations of polyethylene glycol (PEG) and<br>
mannitol. o   Continued growth on high concentrations of PEG and mannitol.<br>
Generation and phenotypic evaluation of Tj lines containing 35S::cDNA 13500101.<br>
Wild-type Arabidopsis Wassilewskija (WS) plants were transformed with a Ti plasmid containing cDNA 13500101 in the sense orientation relative to the CaMV35Sconstitutive promoter. The Tj plasmid vector used for this construct, CRS338, contains PAT and confers herbicide resistance to transformed plants. Ten independently transformed events were selected and evaluated for their qualitative phenotype in the T1 generation. No positive or negative phenotypes were observed in the TI plants.<br>
Screens of superpools on high PEG, mannitol, and ABA as surrogate screens for drought tolerance.<br>
Seeds from 13 superpools (1,200 T2 seeds from each superpool) from the CaMV35S or 32449 over-expression lines were tested on 3 drought surrogate screens (high concentrations of PEG, mannitol, and ABA) as described above. T3 seeds were collected from the resistant plants and analyzed for resistance on all three surrogate drought screens.<br>
OncecDNA 13500101 was identified in resistant plants from each of the three surrogate drought screens, the five individual T2 events containing this cDNA (SR01000) were screened on high PEG, mannitol, and ABA to identify events with the resistance phenotype.<br>
Superpools (SP) axe referred to as SP1, SP2 and so on. The letter following the hyphen refers to the screen (P = PEG, M=mannitol, and A = ABA) and the number following the letter refers to a number assigned to each plant obtained from that screen on that superpool. For example, SP1-M18 is the 18th plant isolated from a mannitol screen of Superpool 1.<br>
Qualitative and quantitative analysis of 2 independent events representing 35S::cDNA 13500101 (SR01000) on PEG, mannitol and ABA<br>
To identify two independent events of 35S::clone 17206 showing PEG, mannitol, and ABA resistance, 36 seedlings from each of three events, SR01000-01, 02 and 03 were screened as previously described. Basta segregation was assessed to verify that the lines contained a single insert segregating in a 3:1 (R:S) ratio as calculated by a chi-square test (Table 1). Two lines (-01 and -02) segregated for a single insert (Table 1).<br>
Table 10-1.  BastaR segregation for SRQ1000 individual events (Table Removed) <br>
Testing of the progeny from the T2 Resistant plants on the 3 surrogate drought screens showed that lines SR01000-01 and 02 had a strong and consistent resistance to PEG, mannitol, but not to ABA. These were chosen as the two events for further analysis. The controls were sown the same day and in the same plate as the individual lines. The PEG (Tables 10-2 and 10-3), and mannitol (Tables 10-4 and 10-5) segregation ratios observed for SRO1000-01 and 02 are consistent with the presence a of single insert as demonstrated by chi-square (Table 10-1).<br>
 Table 10-2. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01000-01T2 containing 35S::cDNA 135Q0101onPEG. (Table Removed) <br>
Table 10-3.  Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01000-02 T2 containing 35S::cDNA 13500101onPEG. (Table Removed)Table 10-4. Chi-square analysis assuming a 3:1 (R: S) ratio for progeny of SR01000-01 T2 containing 35S::cDNA 13500101on mannitol. (Table Removed) <br>
Table 10-5. Chi-square analysis assuming a 3:1 (R:S) ratio for progeny of SR01000-02 T2 containing 35S::cDNA 13500 lOlon mannitol. (Table Removed) <br><br>
Qualitative and quantitative analysis of progeny of T2 plants isolated on PEG, mannitol and ABA<br>
The progeny from one resistant Tj plant from each of these two events were tested in the same manner as the 12- Resistance to PEG and mannitol persisted in the second generation. Taken together, 1) the isolation of resistant seedlings containing clone 17026 from two of the surrogate screens for drought (PEG and mannitol), 2) the inheritance of this resistance in the next generation, and 3) the fact that the progeny from two or more events from the original transformation also segregated for resistance to these osmotic stresses, these findings provide strong evidence that clone 17026 when over-expressed can provide tolerance to osmotic stresses.Table 10-6 provides the results of the consensus sequence analysis based on Ceres cDNA 13500101.(Table Removed) <br>
(Table Removed) <br>
The invention being thus described, it will be apparent to one of ordinary skill in the art that various modifications of the materials and methods for practicing the invention can be made. Such modifications are to be considered within the scope of the invention as defined by the following claims.<br>
Each of the references from the patent and periodical literature cited herein is<br>
f<br>
hereby expressly incorporated in its entirety by such citation.<br><br><br><br><br><br><br><br>
CLAIMS<br>
What is claimed is:<br>
I.      An isolated nucleic acid molecule comprising:<br>
a nucleic acid having a nucleotide sequence which encodes an amino acid<br>
sequence exhibiting at least 85% sequence identity to any one of the<br>
sequences present in the Sequence Listing;<br>
a nucleic acid which is a complement of a nucleotide sequence according<br>
to paragraph (a);<br><br>
a nucleic acid which is the reverse of the nucleotide sequence according to<br>
subparagraph (a), such that the reverse nucleotide sequence has a sequence<br>
order which is the reverse of the sequence order of the nucleotide<br>
sequence according to subparagraph (a); or<br>
a nucleic acid capable of hybridizing to a nucleic acid according to any<br>
one of paragraphs (a) - (c), under conditions that permit formation of a<br>
nucleic acid duplex at a temperature from about 40°C and 48°C below the<br>
melting temperature of the nucleic acid duplex.<br><br>
The isolated nucleic acid molecule according to claim 1, which has the<br>
nucleotide sequence according to any one of the sequences present in the<br>
Sequence Listing.<br>
The isolated nucleic acid molecule according to claim 1, wherein said amino<br>
acid sequence comprises a polypeptide according to any one of the consensus<br>
sequences set forth in Tables 1-4, 2- 8, 3-9, 5-8, 6-8, 7-6, 8-7, 10-6 and 11-6 .<br>
The isolated nucleic acid molecule according to claim 1, wherein said amino<br>
acid sequence has a sequence according to any one of the sequences present in<br>
the Sequence Listing.<br>
A vector construct comprising:<br>
a first nucleic acid having a regulatory sequence capable of causing<br>
transcription and/or translation in a plant; and<br>
a second nucleic acid having the sequence of the isolated nucleic acid<br>
molecule according to any one of claims 1-4;<br>
wherein said first and second nucleic acids are operably linked and wherein said second nucleic acid is heterologous to any element in said vector construct.<br>
The vector construct according to claim 5, wherein said first nucleic acid is<br>
native to said second nucleic acid.<br>
The vector construct according to claim 5, wherein said first nucleic acid is<br>
heterologous to said second nucleic acid.<br>
A host cell comprising an isolated nucleic acid molecule according to any one<br>
of claims 1-4 wherein said nucleic acid molecule is flanked by exogenous<br>
sequence.<br>
A host cell comprising a vector construct according to claim 5.<br>
An isolated polypeptide comprising an amino acid sequence exhibiting at least<br>
85% sequence identity to any sequence present in the Sequence Listing.<br>
A method of introducing an isolated nucleic acid into a host cell comprising:<br><br>
providing an isolated nucleic acid molecule according to claim 1; and<br>
contacting said isolated nucleic acid with said host cell under conditions<br>
that permit insertion of said nucleic acid into said host cell.<br><br>
A method of transforming a host cell trut comprises contacting a host cell with<br>
a vector construct according to claim 5.<br>
A method for detecting a nucleic acid in a sample which comprises:<br>
providing an isolated nucleic acid molecule according to any one of<br>
claims 1 -4;<br>
contacting said isolated nucleic acid molecule with a sample under<br>
conditions which permit a comparison of the sequence of said isolated<br>
nucleic acid molecule with the sequence of DNA in said sample; and<br>
analyzing the result of said comparison.<br><br>
A plant, plant cell, plant material or seed of a plant which comprises a nucleic<br>
acid molecule according to claim 1 which is exogenous or heterologous to said<br>
plant or plant cell.<br>
A plant, plant cell, plant material or seed of a plant which comprises a vector<br>
construct according to claim 5.<br>
A plant that has heen regenerated from a plant cell or seed according to claim<br>
14.<br>
A plant, plant cell, plant material or seed of a plant which comprises a nucleic<br>
acid molecule according to claim 1, wherein said plant has improved water use<br>
efficiency characteristics as compared to a wild-type plant cultivated under the<br>
same conditions.<br>
A method for increasing water use efficiency in a plant comprising<br>
transforming a plant with a nucleic acid sequence according to claim 1.<br>
A transgenic plant having a gene construct comprising a nucleic acid encoding<br>
a water use efficiency component operably linked to a plant promoter so that<br>
the water use efficiency component is ectopically overexpressed in the<br>
transgenic plant, and the transgenic plant exhibits:<br>
i) faster rate of growth.<br>
ii) greater fresh or dry weight at maturation,<br>
iii) greater fruit or seed yield,<br>
iv) higher tolerance to heat,<br>
v) higher tolerance to drought,<br>
vi) higher tolerance to cold,,<br>
vii) greater genrmination rate,<br>
viii) reduced water consumption, or<br>
ix), greater tolerance to flood<br>
than a progenitor plant which does not contain the polynucleotide construct, when the transgenic plant and the progenitor plant are cultivated under identical environmental conditions, wherein the water use efficiency component is any one of the polypeptides set forth in the Sequence Listing, or any one of the consensus sequences in claim 3.<br>
20. A method for increasing water use efficiency in a plant which comprises<br>
transforming a plant with a nucleic acid sequence that encodes a polypeptide that comprises at least one of the following:<br>
an amino acid sequence that comprises the residues at positions 10-276 of<br>
the consensus sequence of Table 3-9,<br>
an amino acid sequence that comprises the residues at positions 4-563 of<br>
the consensus sequence of Table 5-8,<br>
) an amino acid sequence that comprises the residues at positions 143-336<br>
of the consensus sequence of Table 7-6,<br>
an amino acid sequence that comprises the residues at positions 351-369<br>
of the consensus sequence of Table 7-6 ,and<br>
an amino acid sequence that comprises the residues at positions 51-370 of<br>
the consensus sequence of Table 10-6.<br>
2 1 . A plant, plant cell, plant material of a plant with improved water use efficiency characteristics as compared to a wild-type plant cultivated under the same conditions which comprises a nucleic acid sequence that encodes a polypeptide that comprises at least one of the following:<br>
an amino acid sequence that comprises the residues at positions 10-276 of<br>
the consensus sequence of Table 3-9,<br>
an amino acid sequence that comprises the residues at positions 4-563 of<br>
the consensus sequence of Table 5-8,<br>
) an amino acid sequence that comprises the residues at positions 143-336<br>
of the consensus sequence of Table 7-6,<br>
an amino acid sequence that comprises the residues at positions 351-369<br>
of the consensus sequence of Table 7-6 ,and<br>
an amino acid sequence that comprises the residues at positions 51-370 of<br>
the consensus sequence of Table 10-6.<br>
22. An isolated nucleic acid molecule, a vector construct, an isolated polypeptide, a method of introducing an isolated nucleic acid into a host cell, a transgenic plant having a gene construct, a method of increasing water use efficiency in a plant and a plant, plant cell, plant material of a plant substantially as herein described as illustrated in the foregoing examples.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUFic3RyYWN0LSgzMC0wNy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Abstract-(30-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUNsYWltcy0oMTQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Claims-(14-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1ERUxOUC0yMDA2LUNsYWltcy0oMTUtMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7327-DELNP-2006-Claims-(15-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Correspondence Others-(02-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7327-DELNP-2006-Correspondence Others-(15-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzAtMDctMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Correspondence Others-(30-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Correspondence-Others-(14-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7327-DELNP-2006-Description (Complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1ERUxOUC0yMDA2LUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7327-DELNP-2006-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUZvcm0tMi0oMzAtMDctMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Form-2-(30-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUZvcm0tMy0oMTQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Form-3-(14-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LUdQQS0oMDItMDUtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-GPA-(02-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzMyNy1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMTQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7327-delnp-2006-Petition-137-(14-05-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263365-epigallocatechin-gallate-derivatives-for-inhibiting-proteasome.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263367-fastening-means-for-rail.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263366</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7327/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Dec-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CERES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1535 RANCHO CONEJO BOULEVARD, THOUSAND OAKS, CA 91320-1440, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHRISTENSEN, CORY, R</td>
											<td>3658 VALENCIA AVE., SIMI VALLEY, CA 93063, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>APUYA, NESTOR</td>
											<td>5396 DOBSON WAY, CULVER CITY, CA 90230, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FELDMANN, KENNETH,A</td>
											<td>3806 CALLE MAZATLAN, NEWBURY PARK, CA 91320, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82,A01H 5/00,C07K 14/415,C12N 5/</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/018950</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-05-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/575,253</td>
									<td>2004-05-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263366-a-recombinant-vector-construct-comprising-a-first-nucleic-acid-and-a-second-nucleic-acid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:11:45 GMT -->
</html>
