<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263309-an-isolated-dna-molecule-an-expression-cassette-and-vector-comprising-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:28:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263309:&quot;AN ISOLATED DNA MOLECULE, AN EXPRESSION CASSETTE AND VECTOR COMPRISING THE SAME&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED DNA MOLECULE, AN EXPRESSION CASSETTE AND VECTOR COMPRISING THE SAME&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An isolated DNA molecule for enhancing gene expression of a coding sequence, fragment, genetic variant, cassette, vector, cell, plant and seed containing said molecule, wherein the molecule comprises introns for induce an increase in the expression level of a transgene, useful in DNA constructions for transforming plant cells, wherein the cell or plant has a construction containing one of these introns under control of a promoter and upstream of a coding sequence stably integrated in its genome displaying higher expression levels as compared to non-transformed cells or plants, or cells or plants transformed with constructions that lack the inventive intron, wherein the Sequences of 5&quot;-non-coding sequences of the genes mentioned above also comprise promoters and exonic sequences in DNA constructions displaying synergism with the effect produced by the intron, and wherein the 5&quot;-non-coding sequences of COX5-C genes lacking said intron promote tissue-specific expression in pollen when they are comprised in DNA constructions for plant transformation.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>AN ISOLATED DNA MOLECULE FOR ENHANCING GENE EXPRESSION OF A CODING SEQUENCE, FRAGMENT, GENETIC VARIANT, CASSETTE, VECTOR, CELL, PLANT AND SEED CONTAINING SAID MOLECULE<br>
BACKGROUND OF THE INVENTION<br>
Field of the Invention<br>
The present invention relates to biotechnology field, more specifically to genetic engineering in plants. The invention provides useful DNA sequences and constructions to regulate recombinant gene expression in plants. More specifically, the invention provides new regulatory sequences derived from Arabidopsis thaliana COX5c-1, COX5c-2 and COX5c-3 genes.<br>
Description of the Prior Art<br>
Genetic engineering in plant biotechnology made an amazing advance in the fields of research and production of new products.<br>
Research in genetic engineering requires access to a wide variety of sequences that are useful for the regulation of transgenes. There are two examples of such elements: introns and promoters.<br>
Selection of promoters controlling or directing transcription levels of one or more genes is one of the<br>
greatest challenges that molecular biology must achieve for the success of plant biotechnology.<br>
Therefore, many efforts have been made in the last two decades to find promoters and other sequences capable of guaranteeing the expression according to the needs of each transgene. Several promoters of different species have been studied for production of transgenic plants; including vegetal, viral, from the Ti and Ri plasmids of Agrobacterium tumefaciens, as well as natural exons and introns, namely the first intron of maize alcohol dehydrogenase-1 (Callis et al., 1987) or the first exon/intron of rice actin-1 gene (McElroy et al., 1991).<br>
Gene transcription is regulated by a promoter region (cis element) and multiple regulatory proteins (trans elements). A genetic engineering project requires the simultaneous use of several promoters. A first promoter can be used upstream of the gene of interest, while a second promoter can be used to express a selection marker.<br>
Eukaryotic genes are usually interrupted by non-coding sequences, the introns. Eukaryotic genes are first transcribed as pre-RNA, which contain introns within their sequences. Introns are then removed during the splicing, the final product, the mRNA, can be translated to a protein.<br>
It has been shown that introns are involved in the regulation of gene expression in plants. The first intron of maize enzyme alcohol dehydrogenase-1 (Adh-1) gene is capable of increasing transcription in anaerobiosis (Callis et al. , 1987) . Although in a smaller degree, the intron also stimulates transcription in aerobic conditions. It has been proposed that the enhancement of gene expression is due to pre-RNA stabilization produced by the introns. Remarked increases have also been reported in the expression of gene CAT (from 12 to 20 times) when the first intron of the maize dehidrogenase alcohol (Mascarenhas et al, 1990). It has also been reported that these effects are produced at the transcription level.<br>
Naturally occurring introns with their adjacent sequences have been employed to increase transcription, especially when intron is located near the 5" end of the gene. It has also been reported that potentiation of intron-mediated mRNA accumulation (1MB) depends on intron origin, exonic regions flanking said intron and cell type. Molecular mechanisms underlying IME have not been completely elucidated (Simpson et al., 1996; Lorkovic et al., 2000).<br>
Enhancement of expression by introns has been reported for several genes from maize and other monocots (see W098/5921, and also see Callis et al. , 1987; McElroy<br>
et al., 1990; Christensen et al. , 1992; Xu et al. , 1994; Jeon et al. , 2000; Morello et al. , 2002) and dicot plants (Norris et al. , 1993; Gidekel et al. , 1996; Rose and Last, 1997; Plesse et al. , 2001; Mun et al . , 2002). Introns influencing expression are more frequently located near the translation start site within non-coding regions, as is the case for COX5c genes. The exact role of introns in promoting an increase in expression levels is not clear. Some introns seem to contain transcriptionally active regulatory elements (Gidekel et al. , 1996), while others seem to act post-transcriptionally (Rose and Last, 1997), suggesting the existence of different mechanisms of action. It has recently been proposed that many introns would act by increasing the processivity of the transcription machinery (Rose, 2004). Besides the quantitative enhancement of expression, some introns direct tissue-specific patterns of expression (Bolle et al. , 1996; Jeon et al. , 2000) . In some cases, like those of the Petunia actin-depolymerizing factor (Mun et al., 2002), the rice a-tubulin OstubAl (Jeon et al., 2000) and the Arabidopsis polyubiquitin Ubil y Ubi4 genes (Plesse et al. , 2001), expression is specifically observed in vascular tissues and/or metabolically active dividing cells. These expression patterns are similar to those observed here for the COXBc genes, probably indicating that these introns<br>
operate with similar mechanisms or respond to similar factors.<br>
Although the involvement of introns in translation seems an unexpected consequence, similar observations have been made in animal and plant systems (Le Hir et al., 2003; Rose, 2004). It has been proposed that increment in translational efficiency by introns is related to the location of proteins near the exon-exon junctions during splicing that subsequently increase the interaction of ribosomes with the mRNA (Wiegand et al., 2003; Nott et al., 2004) .<br>
Cytochrome c oxidase (COX) is a multimeric complex composed of several different subunits, two or three of them encoded by the mitochondrial genome and the rest encoded in the nucleus (Grossman et al. , 1997; Jansch et al., 1996) .<br>
Three different nuclear-encoded subunits, COXBb, COX6a, and COX6b, have been identified in plants through sequence comparisons with yeast and animal counterparts (Kadowaki et al. , 1996; Ohtsu et al. , 2001; Curi et al. , 2003) .<br>
A fourth subunit, COX5c, is the smallest COX plant subunit and has been discovered by protein purification studies (Nakagawa et al., 1987, 1990). Recent studies using 2D gel electrophoresis combined with mass spectrometry<br>
indicated  the  presence  of  additional  plant-specific subunits (Millar et al., 2004).<br>
COXBc is a polypeptide of about 63 amino acids with sequence similarity to yeast COX Vila and mammalian COX VIII (Nakagawa et al., 1990). COX5c cDNAs have been isolated from sweet potato, rice, and sunflower (Nakagawa et al. , 1990; Hamanaka et al. , 1999; Curi et al. , 2002), and ESTs from several species are available. The first COXBc gene was also isolated from sweet potato (Nakagawa et al. , 1993) , and related sequences could be detected in the totally or partially sequenced genomes from Arabidopsis, rice, and Lotus corniculatus. Expression studies in rice and sunflower indicated that COX5c genes are expressed at different levels throughout the plant (Hamanaka et al., 1999; Curi et al., 2002). However, no detailed analysis on tissue specificity of expression or on the gene sequences involved in directing this expression have been performed by any plant COX5c gene.<br>
It is generally assumed that the expression of components of the plant mitochondrial respiratory chain must somehow be co-ordinated. It is now well established that most mitochondrial components show enhanced expression in flowers (Huang et al. , 1994; Landschutze et al. , 1995; Felitti et al., 1997; Heiser et al., 1997; Zabaleta et al., 1998). Expression in flowers is mainly localized in anthers<br>
as indicated by in situ hybridization experiments (Smart et al., 1994; Ribichich et al. , 2001; Elorza et al. , 2004). Expression studies in Arabidopsis thaliana have shown similar responses for the nuclear genes encoding cytochrome c and COX subunits 5b, 6a, and 6b (Welchen et al. , 2002; Curi et al., 2003). Notably, a different behaviour has been observed for genes encoding COX subunit 5c (COX5c), at least in sunflower (Curi et al., 2002). However, no functional studies have been performed on the cis-acting sequences required for the expression of COX5c genes from any species.<br>
SUMMARY OF THE INVENTION<br>
It is therefore an object of the invention to provide an isolated DNA molecule comprising at least one of the following introns: COX5c-I gene intron, as shown in SEQ N° 3, preferably from nucleotides 22 to 698 from the start codon of COX5c-1; COX5c-2 gene intron, as shown in SEQ N° 7, preferably from nucleotides 12 to 591 from the start codon of COX5c-2; COX5c-3 gene intron, as shown in SEQ N° 11, preferably nucleotides 25 to 1058 from the start codon of COX5c-3; or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences that conserve the capability of enhancing gene<br>
expression in plant cells when it is located in a construction under control of a non-related promoter and upstream of at least one coding sequence.<br>
It is still another object of the present invention to provide an isolated DNA molecule comprising at least one of the following sequences: COX5C-1 promoter-exon 1-intron-exon 2, SEQ N° 1-2-3-4, preferably from nucleotides 1 to 1211 of Cox5C-I; COX5C-2 promoter-exon 1-intron-exon 2, SEQ N° 5-6-7-8, preferably from nucleotides 1 to 1153 from Start Codon upstream to non coding region of Cox5C-2; COX5C-3 promoter-exon 1-intron-exon 2, SEQ N° 9-10-11-12, preferably from nucleotides 1 to 2205 of Cox5C-3; or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences that conserve the capability of enhancing gene expression of at least one coding sequence of a gene of interest.<br>
It is a further object of the present invention to provide an isolated DNA molecule comprising at least one of the following sequences: COX5c-l gene exon 1-intron-exon 2, SEQ N° 2-3-4, preferably from nucleotides 1 to 741 from the start codon of Cox5C-1; COX5c-2 gene exon 1-intron-exon 2, SEQ N° 6-7-8, preferably from nucleotides 1 to 640 from the start codon of Cox5C-2; COX5c-3 gene exon 1-intron-exon 2, SEQ N° 10-11-12 preferably from nucleotides 1 to 1123 from the start codon of Cox5C-3, or a portion thereof, namely a<br>
fragment, genetic variant or deletion of at least one of said sequences keeping the capability of enhancing gene expression in plant cells when it is located in a construction under control of a non-related promoter and upstream of at least one coding sequence of interest.<br>
It is a further object of the present invention to provide an isolated DNA molecule comprising at least one of the following sequences: COX5c-l promoter-exon 1-exon 2, SEQ N°:l-2-4, preferably from base 1 to 21 upstream of the start codon of COX5c-l linked to the sequence from base 699 to 1211 downstream of the start codon of COX5c-l; COX5c-2 promoter-exori 1-exon 2, SEQ N°:5-6-8, preferably from base 1 to 11 upstream of the start codon of COX5c-2 linked to the sequence from base 592 to 1153 of the start codon of COX5c-2; COX5c-3 promoter-exon 1-exon 2, SEQ N°:9-10-12, preferably from base 1 to 24 upstream of the start codon of COX5c-3 linked to the sequence from base 1059 to 2205 upstream of the start codon of COX5c-3; or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences keeping the capability of enhancing tissue-specific gene expression in plant cells when some of these sequences is located upstream of at least one heterologous coding sequence of a gene of interest,<br>
It is therefore an object of the invention to provide an isolated DNA molecule comprising at least one of the following introns: COX5c-I gene intron, as shown in SEQ N° 3, preferably from nucleotides 22 to 698 from the start codon of COX5c-I; COX5c-2 gene intron, as shown in SEQ N° 7, preferably from nucleotides 12 to 591 from the start codon of COX5c-2; COX5c-3 gene intron, as shown in SEQ N° 11, preferably nucleotides 25 to 1058 from the start codon of COX5c-3; or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences that conserve the capability of enhancing gene expression in plant cells when it is located in a construction under control of a non-related promoter and upstream of at least one coding sequence, and wherein the molecule includes at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 22 and 698 when the sequence is SEQ ID N° 3; at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 12 and 591 when the sequence is SEQ ID N° 7 ; and at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 25 and 1058 when the sequence is SEQ ID N° 11.<br>
It is still another object of the present invention to provide an isolated DNA molecule comprising at least one of the following sequences: COX5C-1 promoter-exon 1-intron-exon 2, SEQ N° 1-2-3-4, preferably from nucleotides 1 to 1211 of Cox5C-l; COX5C-2 promoter-exon 1-intron-exon 2, SEQ N° 5-6-7-8, preferably from nucleotides 1 to 1153 from Start Codon upstream to non coding region of Cox5C-2; COX5C-3 promoter-exon 1-intron-exon 2, SEQ N° 9-10-11-12, preferably from nucleotides 1 to 2205 of Cox5C-3; or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences that conserve the capability of enhancing gene expression of at least one coding sequence of a gene of interest, and wherein the molecule includes at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 1211 when the sequence is SEQ ID N° 1-2-3-4; at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 1153 when the sequence is SEQ ID N° 5-6-7-8; and at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 2205 when the sequence is SEQ ID N° 9-10-11-12.<br>
It is a further object of the present invention to provide an isolated DNA molecule comprising at least one of the following sequences: COX5c-l gene exon 1-intron-exon 2, SEQ N° 2-3-4, preferably from nucleotides 1 to 741 from the start codon of Cox5C-l; COX5c-2 gene exon 1-intron-exon 2, SEQ N° 6-7-8, preferably from nucleotides 1 to 640 from the start codon of Cox5C-2; COX5C-3 gene exon 1-intron-exon 2, SEQ N° 10-11-12 preferably from nucleotides 1 to 1123 from the start codon of Cox5C-3, or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences keeping the capability of enhancing gene expression in plant cells when it is located in a construction under control of a non-related promoter and upstream of at least one coding sequence of interest, and wherein the molecule includes at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 741 when the sequence is SEQ ID N° 2-3-4; at least 100 consecutive bases with a hornology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 640 when the sequence is SEQ ID N° 6-7-8; and at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 1123 when the sequence is SEQ ID N° 10-11-12.<br>
It is a further object of the present invention to provide an isolated DNA molecule comprising at least one of the following sequences: COX5c-l promoter-exon 1-exon 2,SEQ N°:l-2-4, preferably from base 1 to 21 upstream of the start codon of COX5c-1 linked to the sequence from base 699 to 1211 downstream of the start codon of COX5c-1; COX5c-2 promoter-exon 1-exon 2, SEQ N°:5-6-8, preferably from base 1 to 11 upstream of the start codon of COX5c-2 linked to the sequence from base 592 to 1153 of the start codon of COX5c-2; COX5c-3 promoter-exon 1-exon 2, SEQ N°:9-10-12, preferably from base 1 to 24 upstream of the start codon of COX5c-3 linked to the sequence from base 1059 to 2205 upstream of the start codon of COX5c-3; or a portion thereof, namely a fragment, genetic variant or deletion of at least one of said sequences keeping the capability of enhancing tissue-specific gene expression in plant cells when some of these sequences is located upstream of at least one heterologous coding sequence of a gene of interest, and wherein the molecule includes at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence SEQ ID N° 1-2-4; at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence SEQ ID N° 5-6-8; and at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence SEQ ID N° 9-10-12.<br>
It is another object of the present invention to provide a construction comprising from 5' to 3': a) a plant functional  promoter,  b)  a  COX5c  intron,  c)  a  coding sequence, d) a 3'UTR.<br>
It is another object of the present invention to provide a construction comprising from 5' to 3': a) a COX5c promoter-exon 1-intron-exon 2, b) a coding sequence, c) a 3'UTR.<br>
It is another object of the present invention to provide a construction comprising from 5' to 3': a) a plant functional promoter, b) a COX5c gene exon 1-intron-exon 2, c) a coding sequence, d) a 3'UTR.<br>
It is another object of the present invention to provide a construction comprising from 5' to 3': a) a COXBc promoter-exon 1-exon 2, b) a coding sequence, c) a 3'UTR.<br>
It is a further object of the present invention to provide a plant and its descendant that contain in at least one of its cells the sequences named above, an active fragment, deletion or genetic variant thereof, under control of a plant functional promoter, wherein the plant is a monocot or a dicot plant, and it belongs to a species of a group consisting of rice, maize, wheat, alfalfa, soy, tobacco and cotton.<br>
It. is a still another object of the present invention to provide a DNA molecule capable of hibridizing in astringent conditions with at least one of the sequences mentioned above. <br>
The above and other objects, features and advantages of this invention will be better understood when taken in connection with the accompanying drawings and description.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The present invention is illustrated by way of example in the following drawings wherein:<br>
Figure 1 shows COX5c genes schemes (A, upper panel) and (B, lower panel), which contain an intron within the 5'-non-coding region.<br>
(A)	Scheme  of  the  six  sequenced  COX5c  genes<br>
available  in data banks:  three genes  from Arabidopsis<br>
thaliana  (AtCOX5c)  and one  each  from Ipomoea batatas<br>
(IbCOXBc) , Oryza sativa (OsCOX5c) , and Lotus corniculatus<br>
(LcCOXBc) . All genes contain a single intron (white boxes)<br>
located in the 5'-non-coding region. The OsCOX5c intron is<br>
not drawn to scale due to its length (2.4 kbp). Non-coding<br>
and coding regions of exons are shown in light and dark<br>
grey, respectively.<br>
(B)	Scheme of the different constructs used to<br>
analyse COX5c regions required for expression. Different<br>
regions of the COX5c-1 and COX5c-2 genes, containing 5'non coding region (black boxes), non coding regions of exons 1and 2 located upstream of the start codon (light grey boxes) , and the leader introii (white box) were fused to the gus coding region and introduced into plants. In a similar way, COX5c-2 untranslated leader sequences were fused to the COX5b-l promoter (striped boxes) in both orientations.<br>
Figure 2 shows histochemical localization of GUS activity in Arabidopsis plants transformed with constructs carrying different portions of the COXBc-2 gene fused to gus. Plants transformed with fragments containing non-transcribed upstream regions plus either the complete transcribed 5"-non-coding region (A-J), only exon 1 sequences (K-0), or the 5'-non-coding region without the intron (P-T) were analysed. Two- (A), 3- (B, K, P), 15- (D, F), and 20-d-old plants (C, E) are shown. Leaves (G, L, Q, R), flowers (H, I, M, N, S), anthers (T), and siliques (J, O) from adult plants are also shown.<br>
Figure 3 shows bar-graphic illustrating that removal of the COXSc intron produces a pronounced decrease in expression of the reporter gene. GUS-specific activity of extracts from plants transformed with construct pBI5c2 (5c2+I) or the respective constructs in which either the intron plus a portion of exon 2 (5c2-I/E2) or only the  intron (5c2-I) were removed was measured using the fluorogenic substrate MUG and protein extracts prepared from rosettes of 20-d-old plants. GUS activity was also measured in extracts from non-transformed plants (wt) or from plants carrying the promoterless gus gene (pBHOl) or the gus gene under the control of the CaMV 35S promoter (pBI121). Error bars represent SE of three independent measurements with one representative line from each construct. Similar results were obtained with different independent lines. The inset shows an enlargement of the bars corresponding to plants that display low GUS activity values.<br>
Figure 4 shows histochemical localization of GUS activity in Arabidopsis plants transformed with the COX5b-l promoter fused to the COX5c-2 leader intron. (A, B) Three-and 15-d-old plants transformed with a 609 bp COX5b-l promoter fragment. (C-F) Similar plants transformed with the same promoter fragment fused to the COX5c-2 leader intron in the sense (D, E) or antisense (C, F) orientation. (G-L) Leaves from adult plants carrying the promoter alone (G, J) or with the intron in the sense (H, K) or the antisense (I, L) orientation. (M-P) Flowers and siliques from plants transformed with the COX5b-l promoter fragment without (M, N) or with the COX5c-2 intron (O, P).<br>
Figure 5 shows a graph illustrating that the COX5c-2 leader intron increases expression from the unrelated COX5b-l promoter. Specific GUS activity in extracts from plants transformed with a 609 bp COX5b-l promoter fragment fused to gus (5b) or with a similar construct in which the COX5c-2 leader intron was inserted between the promoter and the gus coding region either in the sense (5b+Is) or antisense (5b+Ias) orientation was measured using the fluorogenic substrate MUG. GUS activity was also measured in extracts from non-transformed plants (wt) or from plants carrying the promoterless gus gene (pBHOl) or the gus gene under the control of the CaMV 35S promoter (pBI121). Error bars represent SE of three independent measurements with one representative line from each construct. Similar results were obtained with different independent lines.<br>
Figure 6 shows a northern blot analysis of gus steady-state transcript levels using total RNA from plants transformed with constructs containing different portions of the COX5c-2 gene fused to gus. Total RNA (20 μg) from plants transformed with fragments containing COX5c-2 non-transcribed upstream regions and either the complete transcribed 5'-non-coding region (lane I), only exon 1 sequences (lane 2), or the 5'-non-coding region without the intron (lane 3). Lanes 4, 5, and 6 contain RNA from plants transformed with a 609 bp COX5b-l promoter fragment or the same fragment fused to the COX5c-2 leader intron in the antisense or sense orientation, respectively. Plants transformed with the promoterless gus gene (lane 7), non-transformed plants (lane 8), and plants transformed with the gus gene under the control of the CaMV 35S promoter (lane 9) were also analysed.<br>
Figure  7  shows  DNA  sequences  isolated  in  the present invention:<br>
SEQ N° 1: COX5C-I promoter; SEQ N° 2: COX5C-I exon 1; SEQ N° 3 COX5c-I intron; SEQ N° 4: COXSc-I exon 2<br>
Figure  8  shows	DNA  sequences  isolated  in  the<br>
present invention:<br>
SEQ N° 5: COX5c-2	promoter;<br>
SEQ N° 6: COX5c-2	exon 1;<br>
SEQ N° 7: COX5c-2	intron;<br>
SEQ N° 8: COX5c-2	exon 2<br>
Figure  9  shows  DNA  sequences  isolated  in  the present invention: <br>
SEQ N°	9: COX5c-3 promoter;<br>
SEQ N°	10: COX5C-3 exon 2;<br>
SEQ N°	11: COXBc-3 intron;<br>
SEQ N°	12: COX5c-3 exon 2<br>
DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
Now referring in detail to the invention, the same refers to DNA molecules isolated from Arabidopsis thaliana, useful for the construction of expression cassettes and vectors in plants, their descendant, seeds, cells, wherein some of the sequences provided by the inventors enhance gene expression (introns, exons, promoters) in the whole plant, while the others produce such effect in a tissue-specific manner in the pollen.<br>
The isolated DNA molecule can comprise SEQ N° 3, SEQ N° 7 or SEQ N° 11, corresponding to COX5c-l, -2 and -3 introns respectively, or fragments, genetic variants or deletions thereof that conserve the capability of enhancing gene expression in plant cells.<br>
The isolated DNA molecule can comprise SEQ N° 1-2-3-4, SEQ N° 5-6-7-8 or SEQ N° 9-10-11-12, corresponding to COX5c-I, -2 and -3 gene promoter-exon 1-intron-exon 2, respectively, or fragments, genetic variants or deletions thereof that conserve the capability of enhancing gene expression in plant cells.<br>
The isolated DNA molecule can comprise SEQ N° 2-3-4, SEQ N° 6-7-8 or SEQ N° 10-11-12, corresponding to COX5c-1, -2 and -3 gene exon 1-intron-exon 2, respectively, or fragments, genetic variants or deletions thereof that conserve the capability of enhancing gene expression in plant cells.<br>
The invention discloses the sequences located upstream of the start codon of the three Arabidopsis thaliana COX5c genes that drive tissue-specific expression in pollen, as well as the introns located in the 5'-non-coding region of all COX5c genes, and also that they are directly responsible for high gene expression levels observed in the tissues.<br>
The isolated DNA molecule can comprise SEQ N° 1-2-4, SEQ N° 5-6-8 or SEQ N° 9-10-12,- corresponding to COXSc-1, -2 and -3 gene promoter-exon 1-exon 2, respectively, or fragments, genetic variants or deletions thereof that conserve the capability of enhancing tissue-specific gene expression in pollen.<br>
Expression is observed throughout development, specially in vascular and meristematic tissues, and in pollen grains and siliques. The tissue-specific patterns of COXSc expression may be the consequence of responses to cell-specifie factors or to the metabolic status of these tissues, which undergo constant cell proliferation.<br>
It is noteworthy that removal of the leader intron produces a pronounced decrease in expression levels for both genes, making reporter gene activity barely detectable, except in pollen grains.<br>
Arabidopsis thaliana Heyhn. ecotype Columbia (Col-0) was purchased from Lehle Seeds (Tucson, AZ). Plants were grown on soil in a growth chamber at 22-24 °C under long-day photoperiods (16 h of illumination by a mixture of cool-white and GroLux fluorescent lamps) at an intensity of approximately 200 μE m-2 s"1. Plants used for the different treatments were grown in Petri dishes containing 0.5X Murashige and Skoog medium and 0.8% agar. The dishes were kept at 4 °C for 2 d and then transferred to growth chamber conditions and kept in complete darkness for 7 d.<br>
Isolation of genomic clones was performed as follows: Arabidopsis EST clones encoding COX5c-l and COXSc-2 (clones 245H10T7, accession no. N97140, and 248L23T7, accession no. AA713295) were obtained from ABRC. For the isolation of genomic clones, a mixture of these cDNAs was used to screen IxlO5 pfu from an Arabidopsis genomic library (Voytas et al., 1990). Phage DNA was transferred to Hybond-N and, after overnight hybridization; filters were washed and exposed to X-ray films. Positive clones were purified through successive rounds of plating and hybridization. Isolated phage DNA from purified clones was characterized by restriction analysis and hybridization. A 1.8 kbp EcoRI fragment from COX5c-l and a 3.2 kbp EcoRI/Nhel fragment from COX5c-2, comprising the entire transcribed region and upstream sequences, were subcloned into pBluescript SKT digested with EcoRI or EcoRI and Xbal, respectively. Positive clones were checked by partial sequencing and named VCAT1 and VCAT2.<br>
In order to analyse transgenic plants, total RNA was isolated as described by Carpenter and Simon (1998). For northern blot analysis, specific amounts of RNA were electrophoresed through 1.5% (w/v) agarose/6% formaldehyde gels. The integrity of the RNA and equality of RNA loading were verified by ethidium bromide staining. RNA was transferred to Hybond-N nylon membranes (Amersham Corporation) and hybridized overnight at 68 °C to a 32P-labelled cDNA probe, comprising the entire GUS coding region isolated from vector pBI101.3, in buffer containing 6X SSC, 0.1% (w/v) polyvinylpyrrolidone, 0.1% (w/v) BSA, 0.1% (w/v) Ficoll, 0.2% (w/v) SDS, and 10% (w/v) polyethylene glycol 8000. Filters were washed with 2X SSC plus 0.1% (w/v) SDS at 68 °C (4 times, 15 min each), 0.IX SSC plus 0.1% (w/v) SDS at 37 °C for 15 min, dried, and exposed to Kodak BioMax MS films. To check the amount of total RNA loaded in each lane, filters were then re-probed with a 25S rRNA from Vicia faba under similar conditions as those described above, except that hybridization was performed at 62 °C and the wash with 0.1X SSC was omitted.<br>
Reporter gene constructs and plant transformation were performed as follows: a 1.3 kbp BglII/SalI fragment, comprising sequences upstream of the ATG initiation codon (i.e. exon 1, the intron and part of exon 2, plus non-transcribed upstream sequences) from COX5c-l, was amplified from clone VCAT1 using oligonucleotide COXC32: 5'-GGCAGATCTCTCTTCTTTCTTCTTCTC-3" (BglII site underlined) and universal primer -40 and cloned in plasmid pBI101.3 digested with BamHI and Sail. A similar construct for COX5c-2 was made by amplifying a 2.2 kbp fragment from VCAT2 with primers COXC41: 5"-GCGTCTAGATTCTTCTCAACCTAGCAC-3' (Xbal site underlined) and -40 and cloning in pBI101.3 digested with Xbal and Hindlll. Constructs containing exon 1 and upstream sequences were obtained in a similar way by amplification with either COXC33: 5'-GGCGGATCCCAAGTCGGAGTGTGGAGG- 3' (COX5c-1) or COXC4 2: 5'-GGCGGATCCCGAGTCAGATTGTGTAGA-3' (COX5c-2), followed by cloning in the Sail/BamHI or Hindi 11/BamHI sites of pBI101.3, respectively. A construct containing the entire COX5c-2 5'-non-coding region without the intron was obtained by amplification with primers COXC45: 5'- GCGTCTAGATTCTTCTCAACCGAGTCAGATTGTGTAGA-3" and -40 followed by cloning into the Xbal and HindiII sites of pBI101.3. To test the effect of the intron on an exogenous promoter, the COX5c-2 intron and transcribed 5'-non-coding sequences were amplified with primers COXC44: 5'-GGCTCTAGAGTTTTCGTCGTGAGCTTC-3" and COXC41 and cloned in both orientations into the Xbal site of a construct containing a 609 bp promoter fragment from the COX5b-l gene fused to gus (Welchen et al. , 2004). In this way, the intron was placed between the COX5b-l promoter and the gus coding region. The different constructs were introduced into Agrobacterium tumefaciens strain GV2260, and transformed bacteria were used to obtain transgenic Arabidopsis plants by the floral dip procedure (Clough and Bent, 1998). Transformed plants were selected on the basis of kanamycin resistance and positive PCR carried out on genomic DNA with primers specific for COX5c-l or -2 and the gus-specific primer 5'-TTGGGGTTTCTACAGGAC-3'. Five to ten independent lines (depending on the construct) were further reproduced and homozygous T3 and T4 plants were used to analyse gus expression. Plants transformed with pBI101.3 or pBI121 were obtained in a similar way and used as negative or positive controls of expression, respectively.<br>
ß-glucuronidase (GUS) activity of transgenic plants was  analysed  by  histochemical   staining  using  the chromogenic substrate 5-bromo-4-chloro-3-indolyl-ß-D-glucuronic acid (X-gluc) as described by Hull and Devic (1995). Whole plants or separated organs were immersed in a 1 mM X-gluc solution in 100 mM sodium phosphate, pH 7.0, and 0.1% Triton X-100 and, after applying vacuum for 5 min, they were incubated at 37 °C until satisfactory staining was observed. Tissues were cleared by immersing them in 70% ethanol.<br>
Specific GUS activity in protein extracts was measured using the fluorogenic substrate 4-methylumbelliferyl ß-D-glucuronide (MUG) essentially as described by Jefferson et al. (1987). Total protein extracts were prepared by grinding the tissues in extraction buffer (50 mM sodium phosphate, pH 7.0, 10 mM EDTA, 10 mM ß-mercaptoethanol) containing 0.1% (w/v) SDS and 1% Triton X-100, followed by centrifugation at 13 000 g for 10 min. GUS activity in supernatants was measured in extraction buffer containing 1 mM MUG and 20% methanol. Reactions were stopped with 0.2 M Na2C03 and the amount of 4-methylumbelliferone was calculated by relating relative fluorescence units with those of a standard of known concentration. The protein concentration of extracts was determined as described by Kruger (1996) .<br>
The involvement of regions located upstream of the translation start site in the expression of two Arabidopsis thaliana nuclear COX5c genes encoding subunit 5c of mitochondrial cytochrome c oxidase has been analysed. It was observed that these regions, which include a leader intron, direct the tissue-specific expression of the gus reporter gene, mainly in root and shoot meristems, actively growing tissues and vascular strands. It is important to remark that based in the similarity of sequences and origin of COXBc genes, it is expected that the results obtained for COX5C-1 and COX5c-2 would be the same to be obtained for COXBc-3 or any fragment thereof.<br>
Expression was also observed in flowers, specifically localized in anthers, stigma, and the receptacle, and in developing seeds.<br>
GUS activity measurements in protein extracts from transformed plants shows that expression levels are higher than those observed with the constitutive CaMV 35S promoter.<br>
Removal of the leader intron produced a significant decrease in expression to values only slightly higher than those observed with a promoterless gus gene.<br>
Histochemical staining of plants transformed with the intronless construct revealed expression only in pollen, suggesting that regulatory elements capable of directing pollen-specific expression are present upstream of the intron. The COX5c-2 intron also increased GUS expression levels when fused in the correct orientation with the promoter of the unrelated COX5b-l gene. It is noteworthy that the invention can contain one of the COX5c introns fused with a variety of promoters, such as Hahb-4 promoter sequences of Patent No. PCT/US 03/13770 which is incorporated as a reference. The effects obtained by the expression of heterologous genes in plants depends on the kinetic of their expression and the reached final levels of the expression. Thus the sequences of the present invention may be applied in the transformation of plants. Particularly in connection to a transformation to improve tolerance to salinity and drought induced by Hahb-4, the expression of Hahb-4, enhanced by the inventive sequences, results into higher survival levels of the transformed plant as well as a faster response of the plant to resist the environmental stresses.<br>
Comparison of GUS activity values with the transcript levels suggests that the intron also increases translation efficiency of the corresponding mRNA. The results obtained point to an essential role of the intron present in the 5'-non-coding region of all known COX5c genes in directing the expression of these genes in plants.<br>
The invention may be better understood with reference to the following examples which are not limitative or restrictive of the scope of protection. On the contrary, it must be clearly understood that many other embodiments, modifications and alterations equivalent to the elements of the invention may be suggested by persons skilled in the art after reading the present description, without departing from the spirit of the present invention and/or the scope of the appended claims.<br>
EXAMPLES<br>
Example  1:   COX5c  related  sequences  in  the Arabidopsis genome<br>
A search for COX5c coding regions in the Arabidopsis genome using sunflower COX5c protein sequences (Curi et al. , 2002) and the program TBLASTN revealed the existence of four genomic regions from which proteins with COX5c-related sequences could be deduced. For three of them (At2g47380, At3g62400, and At5g61310) mRNA sequences are also deposited in data banks, indicating that they are expressed. The fourth region, located in chromosome 5, encodes a protein more distantly related for which no transcripts have been detected, suggesting that it may be a pseudogene. Accordingly, the genes present in chromosomes 2, 3 and 5 have arbitrarily been named COX5c-1, COX5c-2,<br><br>
and COX5c-3, respectively. Upon comparing the corresponding genomic and cDNA sequences, it becomes evident that the three Arabidopsis COX5c genes contain a single intron located within the 5'-non-coding region, at variable distances with respect to the ATG start codon (Figure 1A) . An intron in the same location is also present in the other COX5c genes for which sequences are available, the single genes from rice (Oryza sativa; BAC clone accession number AB027123), sweet potato (Ipomoea batatas; Nakagawa et al. , 1993), and Lotus corniculatus (BAG clone accession number AP006137).<br>
Example 2; Sequences upstream of the translation start site of COX5c-1 and COX5c~2 promote high-level expression of a reporter gene in specific cell-types<br>
Two Arabidopsis thaliana clones containing the COX5c-l and -2 genes and additional genomic regions were isolated by direct screening of a library using a mixture of ESTs derived from these genes. Subclones of these lambda clones were used to amplify fragments containing sequences located upstream of the translation start site which were introduced in vector pBI101.3 in front of the gus coding region (Figure IB). These constructs (pBI5cl and pBI5c2 for COX5c-1 and -2, respectively) were introduced into Arabidopsis by Agrobacterium-mediated transformation. An initial screening of kanamycin-resistant lines was carried out by histochemistry to define expression characteristics common to most of them. Plants from at least five independent, representative transgenic lines carrying each construct were then analysed in detail. The results were essentially the same for all plants analysed, carrying either pBI5cl or pBI5c2.<br>
Seedlings grown on Petri dishes on MS medium showed strong staining along roots and hypocotyls, while activity in cotyledons was detected only in plants older than 3 d post-germination (Figure 2A, B) . Activity in roots was progressively localized to the vascular cylinder and the root meristem upon growth (Figure 2B) . After 15 d, strong expression was also detected in developing secondary roots (Figure 2C). In hypocotyls, activity was also progressively localized to vascular tissues upon development (Figure 2D). Cotyledons displayed GUS activity in the lamina and especially in vascular tissues (Figure 2D, E) . A similar expression pattern was evident in developing leaves (Figure 2D, E) . Strong activity was also observed in the shoot apical meristem (Figure 2F).<br>
Adult (45-d-old) plants grown on soil displayed activity in roots, leaves, and flowers. Expression in roots was similar to that described for younger plants. In leaves, vascular tissues and, to a lesser extent, mesophyll tissues were stained (Figure 2G) . In flowers, strong expression was detected in anthers, especially in reproductive tissues and pollen grains when these were formed (Figure 2H, I) . Activity was also detected in the stigma, receptacle, and petal and sepal veins, and in siliques and developing seeds (Figure 21, J) .<br>
To estimate the relative expression levels produced by both constructs, fluorometric assays of GUS activity in protein extracts from transformed 20-d-old plants were performed. Plants transformed with pBI5cl and pBI5c2 displayed activities of 35 000 pmol min"1 mg'1 and 66 000 pmol min'1 mg'1, respectively. These values are even higher than those observed with plants transformed with pBI121 (i.e. the gus gene under the control of the strong constitutive CaMV 35S promoter) , 18 000 pmol min"1 mg"1, indicating that the sequences contained within the constructs direct high-level expression. Activity measurements using extracts from different organs indicated that highest expression was attained in leaves followed by flowers, siliques, and roots (not shown) .<br>
Example 3; Removal of the leader intron originates<br>
plants with pollen specific expression  <br>
The presence of a conserved intron in the leader region led to the investigation of its role in expression of COX5c genes. Seedlings of plants transformed with constructs in which the respective leader introns from COX5c-l or COX5c-2 and downstream sequences were removed (5cl-I/E2 and 5c2-I/E2 in Figure 1B) showed no GUS activity when analysed by histochemical staining (Figure 2K) . Analysis of adult plants revealed the presence of GUS activity only in pollen grains (Figure 2M, N), while no staining was evident in leaves, siliques, or flower organs other than anthers (Figure 2L-O). Similar results were observed for both genes under study, indicating that the leader intron is essential to direct high-level expression throughout the plant. Indeed, activity in pollen was also reduced in these plants, since longer incubation times were required to reach similar staining: 18 h with plants transformed with the intronless constructs versus 3-5 h with plants transformed with the entire fragments. The levels of GUS activity present in protein extracts from plants transformed with the intronless constructs were extremely low: 800 and 1400 pmol min-1 mg-1, respectively, for COX5c-1 and COX5c-2, that is 40-50 times lower than those observed when the intron was present.<br>
Zabaleta et al.  (1998) have studied the promoter regions involved in pollen/anther expression of three genes that encode components of the NADH dehydrogenase (Complex I) . Within these regions, they have identified conserved GT-rich elements similar to those found in other genes expressed in pollen. They have postulated that these motifs are involved in the co-ordinated expression of the three genes. Analysis of the promoter regions of both COX5c genes showed the presence of GT-rich elements (TGTGGTT and TGTGTTG for COX5c-l and -2) , located at -208 and -210, respectively, from the putative transcription start site. The first of these elements is identical to those observed in one of the Complex I genes and in the tomato LAT52 and LAT56 genes, specifically expressed in pollen (Twell et al. , 1991) . Both genes also possess two close copies of site II elements (TGGGCC/T), located at -109/-90 and -84/-71, respectively, known to be present in genes preferentially expressed in cycling cells (Kosugi et al. , 1995; Tremousaygue et al., 2003). The functional significance of these sequences must be assessed by mutagenesis experiments.<br>
Example 4; Untranslated exon sequences influence gus expression<br>
Comparison of GUS activities with the respective transcript  levels  indicate  that  COX5c  5'-non-coding sequences also increase translation efficiency. This observation could be made with constructs that possess the COX5c leader introns, but not with those that only carry non-coding exon sequences, due to the low expression levels produced by the latter. It should be emphasized that the context of the start codon, which could affect translation efficiency, is the same in all constructs analysed, since they use the ATG provided by the pBHOl.3 vector which is placed several nucleotides downstream of the cloning sites. The differences in translation efficiency are then produced by the intron itself or by the presence of translational enhancers in the 5'-non-coding region. So far, similarities with other known translational enhancers (Yamamoto et al. , 1995; Dickey et al., 1998) within COX5c untranslated regions have not been observed.<br>
The effect of 5' untranslated exon regions on expression was also analysed. For this purpose, plants transformed with a construct in which the COX5c-2 leader intron was removed, but the promoter and 5'-non-coding sequences from exons 1 and 2 were conserved (5c2-I in Figure IB), were used. These plants showed considerably lower GUS activity levels than plants carrying the construct with the intron (Figure 3). They consistently showed, however, slightly higher activities than plants bearing only the promoter and exon 1 fused to gus (Figure 3, inset). The different expression produced by the inclusion of exon 2 non-coding sequences was also observed by histochemical staining. Indeed, activity, although low, was detected in cotyledon veins and tips and also in leaf veins, trichomes, and hydathodes (Figure 2P-R). In reproductive tissues, expression was only detected in pollen, as with the construct in which the intron and exon 2 were removed (Figure 2S, T) . These results, on one side, confirm the importance of the leader intron in determining high level expression and, on the other side, indicate that non-coding exon sequences, though slightly, also influence gene expression.<br>
Example 5; The COX5c-2 leader intron and adjacent regions increase the expression of an unrelated promoter<br>
The effect of the inclusion of the COX5c-2 intron between the promoter of an unrelated gene and the gus coding region was also tested. For this purpose, a region covering the entire intron and surrounding untranslated transcribed sequences from COX5c-2 was inserted between the promoter of the Arabidopsis gene COX5b-l and the gus coding region, either in the sense or antisense orientation (5b+Is and 5b+Ias, respectively, in Figure 1B). The portion of the COX5b-l promoter used (-1 to -609 respective to the translation start site) directs relatively low expression localized in meristems, root, and cotyledon vascular tisssues, the leaf central vein, and in anthers (Welchen et al., 2004) . Inclusion of the COX5c-2 intron in the correct orientation resulted in considerably higher expression levels throughout the plant, in a pattern similar to those observed with the COX5c-l and COX5c-2 promoter plus intron fragments (Figure 4) . As a consequence, GUS activity was extended from vascular tissues and anthers, as observed in plants bearing only the COX5b-l promoter fragment without the intron (Figure 4A, B, G, J, M, N) , to the lamina of cotyledons and leaves (Figure 4D, E, H, K) and to petals and siliques (Figure 40, P) . GUS activity measurements in protein extracts from transformed plants indicated that the intron produces a 6-fold increase in expression levels from the COX5b-l promoter (Figure 5).<br>
When the intron was placed in the same location, but in reverse orientation, GUS activity was not detected in any tissue or developmental stage by either the histochemical (Figure 4C, F, I, L) or the fluorometric method (Figure 5) . It is logical to assume that intron splicing does not take place when the intron is present in the reverse orientation. The presence of a large non-coding sequence in the 5' region of the transcript may either affect its stability or its translation efficiency. Six spurious ATG codons are introduced when the intron is placed in the reverse orientation, thus probably affecting the recognition of the correct start codon by the translation machinery. In addition, it is well-known that transcripts with premature termination stop codons in phase with a start codon are degraded by a process termed nonsense-mediated mRNA decay (Baker and Parker, 2004).<br>
Example  6;  COX5c  leader  introns  and  adjacent regions increase translation efficiency<br>
The increase in expression promoted by the presence of the intron was also observed when analysing transcript levels of the gus gene under the control of the respective fragments from COX5c-l and COX5c-2. Figure 6 shows a northern hybridization using a gus probe and total RNA from plants carrying different portions of the COX5c-2 gene. While no or extremely faint signals were obtained in northern blots with total RNA from plants carrying the intronless constructs (Figure 6, lanes 2, 3), a distinct band was visible in the sample from plants bearing the construct with the COX5c-2 intron (Figure 6, lane 1). Transcript levels in these plants, however, were considerably lower than those observed in plants transformed with the T-DNA region of plasmid pBI121, which contains the gus gene under the control of the CaMV 35S promoter (Figure 6, lane 9). Densitometric analysis of the respective bands, compared with the signals obtained with the rRNA probe, indicated that gus transcript levels are seven times higher in plants transformed with the CaMV 35S promoter fusion. Since plants transformed with the entire COX5c fragments showed 3.5 times higher GUS activities than the latter (see above), this suggests that transcripts containing COX5c 5"-non-coding sequences are more efficiently translated, producing a 25-fold increase in the protein/transcript ratio. Although protein levels were not directly quantified, it can be assumed that enzyme activity measurements constitute an appropriate estimation, since all constructs must produce proteins with the same amino acid sequence. An effect of the intron on translation could also be observed when analysing transcript levels of plants transformed with the COX5b-l gene promoter with or without the COX5c-2 intron (Figure 6, lanes 4, 6) . In this case, the increase in transcript levels promoted by the presence of the intron was very low compared with the 6-fold increase in GUS activity observed in plants with the intron in the correct orientation. On the other hand, plants with the intron in the antisense orientation showed undetectable levels of gus transcripts (Figure 6, lane 5). Although this may be due to a lower transcription efficiency, a more likely explanation is that the unspliced RNA is rapidly degraded.<br>
While preferred embodiments of the present invention have been illustrated and described, it will be obvious to those skilled in the art that various changes and modifications may be made therein without departing from the scope of the invention as defined in the appended claims.<br>
References:<br>
Baker KE, Parker R. 2004. Nonsense-mediated mRNA decay: terminating erroneous gene expression. Current Opinion in Cell Biology 16, 293-299.<br>
Bolle C, Hermann RG, Oelmuller R. 1996. Intron sequences are involved in the plastid and light-dependent expression of the spinach PsaD gene. The Plant Journal 10, 919-924 .<br>
Callis J, Fromm M, Walbot V. 1987. Introns increase gene expression in cultured maize cells. Genes and Development 1, 1183-1200.<br>
Grossman, L, Lomax M. 1997. Nuclear genes for cytochrome c oxidase. Biochimica et Biophysica acta 1352, 174-192.<br>
Carpenter CD, Simon AE. 1998. Preparation of RNA. In: Martinez-Zapater J, Salinas J, eds. Methods in molecular biology, Vol. 82. Arabidopsis protocols. Totowa, NJ: Humana Press Inc, 85-89.<br>
Christensen AH, Sharrock RA, Quail PH. 1992. Maize polyubiquitin genes: structure, thermal perturbation of expression and transcript splicing, and promoter activity following transfer to protoplasts by electroporation. Plant Molecular Biology 18, 675-689.<br>
Clough SJ, Bent AF. 1998. Floral dip: a simplified method for Agrobacteri urn- media ted transformation of Arabidopsis thaliana. The Plant Journal 16, 735-743.<br>
Curi GC, Chan RL, Gonzalez DH. 2002. Genes encoding cytochrome c oxidase subunit 5c from sunflower (Helianthus annuus L.) are regulated by nitrate and oxygen availability. Plant Science 163, 897-905.<br>
Curi GC, Welchen E, Chan RL, Gonzalez DH. 2003. Nuclear and mitochondrial genes encoding cytochrome c oxidase subunits respond differently to the same metabolic factors. Plant Physiology and Biochemistry 41, 689-693.<br>
Dickey LF, Petracek ME, Nguyen TT, Hansen ER, Thompson WF. 1998. Light regulation of Fed-1 mRNA requires an element in the 5' untranslated region and correlates with differential polyribosome association. The Plant Cell 10, 475-484.<br>
Elorza A, Leon G, Gomez I, Mouras A, Holuigue L, Araya A, Jordana X. 2004. Nuclear SDH2-1 and SDH2-2 genes, encoding the iron-sulfur subunit of mitochondrial complex II in Arabidopsis, have distinct cell-specific expression patterns and promoter activities. Plant Physiology 136, 4072-4087.<br>
Felitti SA, Chan RL, Gago G, Valle EM, Gonzalez DH. 1997. Expression of sunflower cytochrome c mRNA is tissue-specific and controlled by nitrate and light. Physiologia Plantarum 99, 342-347. <br>
Gidekel M, Jimenez B,  Herrera-Estrella L. 1996. The first intron of the Arabidopsis thaliana gene coding for elomgation factor 1 beta contains an enhancer-like element. Gene 170, 201-206.<br>
Hamanaka S, Ohtsu K, Kadowaki K, Nakazono M, Hirai A. 1999. Identification of cDNA encoding cytochrome c oxidase subunit 5c (COX5c) from rice: comparison of its expression with nuclear-encoded and mitochondrial-encoded COX genes. Genes and Genetic Systems 74, 71-75.<br>
Reiser V, Brennicke A, Grohmann L. 1996. The plant mitochondrial 22 kDa (PSST) subunit of respiratory chain complex I is encoded by a nuclear gene with enhanced transcript levels in flowers. Plant Molecular Biology 31, 1195-1204 .<br>
Huang J, Struck F, Matzinger DF, Levings III CS. 1994. Flower-enhanced expression of a nuclear-encoded mitochondrial respiratory protein is associated with changes in mitochondrion number. The Plant Cell 6, 439-448.<br>
Hull GA, Devic M. 1995. The beta-glucuronidase (gus) reporter gene system. Gene fusions; spectrophotometric, fluorometric, and histochemical detection. In: Jones H, ed. Methods in plant molecular biology, Vol. 49. Plant gene transfer and expression protocols. Totowa, NJ: Humana Press Inc, 125-141.<br>
Jansch L, Kruft V, Schmitz UK, Braun H-P. 1996. New insights into the composition, molecular mass and stoichiometry of the protein complexes of plant mitochondria. The Plant Journal 9, 357-368.<br>
Jefferson RA, Kavanagh TA, Sevan MW. 1987. GUS fusions: ß-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO Journal 20, 3901-3907. Jeon J-S, Lee S, Jung K-H, Jun S-H, Kim C, An G. 2000. Tissue preferential expression of a rice α-tubulin gene, OstubAl,  mediated by the first intron.  Plant Physiology 123, 1005-1014.<br>
Kadowaki K, Kubo N, Ozawa K, Hirai A. 1996. Targeting presequence acquisition after mitochondrial gene transfer to the nucleus occurs by duplication of existing targeting signal. EMBO Journal 15, 6652-6661.<br>
Kosugi S, Suzuka I, Ohashi Y. 1995. Two of three promoter elements identified in a rice gene for proliferating cell nuclear antigen are essential for meristematic tissue-specific expression. The Plant Journal 7, 877-886.<br>
Landschutze V, Miiller-Rober B, Willmitzer L. 1995. Mitochondrial citrate synthase from potato: predominant expression in mature leaves and young flower buds. Planta 196, 756-764.<br>
Le Hir H, Nott A, Moore MJ. 2003. How introns influence and enhance eukaryotic gene expression. Trends in Biochemical Sciences 28, 215-220.<br>
Zdravko J. Lorkovi'c, Dominika A. Wieczorek Kirk, Mark H.L. Lambermon and Witold Filipowicz. 2000. Pre-mRNA splicing in higher plants. Trends in plant science Reviews. 5 (4) , 160-167.<br>
Mascarenhas. D. ; Mettler, I. J.; Pierce, D. A. and Lowe, H. W. (1990). Intron-mediated enhancement of heterologous gene expression in maize. Plant Mol. Biol. 15:913-920.<br>
McElroy D, Zhang J, Cao J, Wu R. 1991. Isolation of an efficient actm promoter for use in rice transformation. The Plant Cell 2, 163-171. <br>
Millar AH, Eubel H, Jansch L, Kruft V, Heazlewood JL, Braun H-P. 2004. Mitochondrial cytochrome c oxidase and succinate dehydrogenase complexes contain plant specific subunits. Plant Molecular Biology 56, 77-90.<br>
Morello L, Bardini M, Sala F, Breviario D. 2002. A long leader intron of the Ostubl6 rice ß-tubulin gene is required for high-level gene expression and can autonomously promote transcription both in vivo and in vitro. The Plant Journal 29, 33-44.<br>
Mun JH, Lee SY, Yu HJ, Jeong YM, Shin MY, Kim H, Lee I, Kim SG. 2002. Petunia actin-depolymerizing factor is mainly accumulated in vascular tissue and its gene expression is enhanced by the first intron. Gene 292, 233-243 .<br>
Nakagawa T, Maeshima M, Muto H, Kajiura H, Hattori H, Asahi T. 1987. Separation, amino-terminal sequence and cell-free synthesis of the smallest subunit of sweet potato cytochrome c oxidase. European Journal of Biochemistry 165, 303-307.<br>
Nakagawa T, Maeshima M, Nakamura K, Asahi T. 1990. Molecular cloning of a cDNA for the smallest nuclear-encoded subunit of sweet potato cytochrome c oxidase. Analysis with the cDNA of the structure and import into mitochondria of the subunit. European Journal of Biochemistry 191, 557-561.<br>
Nakagawa T, Maeshima M, Nakamura K, Asahi T. 1993. The nuclear gene for subunit Vc of sweet potato cytochrome c oxidase. Plant and Cell Physiology 34, 621-626.<br>
Norris SR, Meyer SE, Callis J. 1993. The intron of Arabidopsis thaliana polyubiquitin genes is conserved in location and is a quantitative determinant of chimeric gene expression. Plant Molecular Biology 21, 895-906.<br>
Nott A, Le Hir H, Moore MJ. 2004. Splicing enhances translation in mammalian cells: an additional function of the exon junction complex. Genes and Development 18, 210-222 .<br>
Ohtsu K, Nakazono M, Tsutsumi N, Hirai A. 2001. Characterization and expression of the genes for cytochrome c oxidase subunit VIb (COX6b) from rice and Arabldopsis thaliana. Gene 264, 233-239.<br>
Plesse B, Criqui MC, Durr A, Parmentier Y, Fleck J, Genschik P. 2001. Effects of the polyubiquitin gene Ubi.U4 leader intron and first ubiquitin monomer on reporter gene expression in Nicotlana tabacum. Plant Molecular Biology 45, 655-667.<br>
Ribichich K, Tioni MF, Chan RL, Gonzalez DH. 2001. Cell-type specific expression of plant cytochrome c mRNA in developing flowers and roots. Plant Physiology 125, 1603-1610.<br>
Rose AB. 2004. The effect of intron location on intron-mediated enhancement of gene expression in Arabidopsis. The Plant Journal 40, 744-751.<br>
Rose AB, Last RL. 1997. Introns act post-transcriptionally to increase expression of the Arabidopsis thaliana tryptophan pathway gene PAT1. The Plant Journal 11, 455-464.<br>
Kruger N. 1996. The Bradford Method for Protein Quantitation. In: John Walker editor. The Protein Protocols. Humana Press 4, 15-20.<br>
Simpson, G.G. and Filipowicz, W. 1996. Splicing of precursors to mRNA in higher plants: mechanism, regulation and sub-nuclear organisation of the spliceosomal machinery. Plant Molecular Biology. 32, 1-41.<br>
Smart C, Moneger F, Leaver CJ. 1994. Cell-specific regulation of gene expression in mitochondria during anther development in sunflower. The Plant Cell 6, 811-825.<br>
Tremousaygue D, Garnier L, Bardet C, Dabos P, Herve C, Lescure B. 2003. Internal telomeric repeats and "TCP-domain" protein binding sites co-operate to regulate gene expression in Arabidopsis thaliana cycling cells. The Plant Journal 33, 957-966.<br>
Twell D, Yamaguchi J, Wing R, Ushiba J, McCormick S. 1991. Promoter analysis of genes that are coordinately expressed during pollen development reveals pollen-specific enhancer sequences and shared regulatory elements. Genes and Development 5, 496-507.<br>
Voytas DF, Konieczny A, Cummings MP, Ausubel FM. 1990. The structure, distribution and evolution of the Tal retrotransposable element family of Arabidopsis thaliana. Genetics 126, 713-721.<br>
Welchen E, Chan RL, Gonzalez DH. 2002. Metabolic regulation of genes encoding cytochrome c and cytochrome c oxidase subunit Vb in Arabidopsis. Plant Cell and Environment 25, 1605-1615.<br>
Welchen E, Chan RL, Gonzalez DH. 2004. The promoter of the Arabidopsis nuclear gene COX5b-l, encoding subunit 5b of the mitochondrial cytochrome c oxidase, directs tissue-specific expression by a combination of positive and negative regulatory elements. Journal of Experimental Botany 55, 1997-2004.<br>
Wiegand HL, Lu S, Cullen BR. 2003. Exon junction complexes mediate the enhancing effect of splicing on mRNA expression. Proceedings of the National Academy of Sciences of the USA 100, 11327-11332.<br>
Xu Y, Yu H, Hall TC. 1994. Rice triosephosphate isomerase gene 5' sequence directs ß-glucuronidase activity in transgenic tobacco but requires an intron for expression in rice. Plant Physiology 106, 459-467.<br>
Yamamoto YY, Tsuji H, Obokata J. 1995. 5' Leader of a photosystem I gene in Nicotiana sylvestris, psaDb, contains a translational enhancer. The Journal of Biological Chemistry 270, 12466-12470.<br>
Zabaleta E, Reiser V, Grohmann L, Brennicke A. 1998. Promoters of nuclear-encoded respiratory chain complex I genes from Arabidopsis thaliana contain a region essential for anther/pollen-specific expression. The Plant Journal 15, 49-59. <br><br><br><br><br>
We claim;<br>
1.	An isolated DNA molecule comprising at least one<br>
of the following introns:<br>
-COX5c-l gene intron,	SEQ N°	3;<br>
-COX5c-2 gene intron,	SEQ N°	7;<br>
-COX5c-3 gene intron,	SEQ N°	11,<br>
or a portion therof.<br>
2.	The molecule of claim 1, wherein<br>
intron sequence SEQ ID N° 3 goes from nucleotides 22 to 698 from the start codon of COX5c-l;<br>
intron sequence SEQ ID N° 7 goes from nucleotides 12 to 591 from the start codon of COX5c-2;<br>
intron sequence SEQ ID N° 11 goes from nucleotides 25 to 1058 from the start codon of COX5c-3;<br>
and wherein said portion comprises a fragment, genetic variant or deletion of at least one of said sequences that conserve the capability of enhancing gene expression in plant cells when it is located in a construction under control of a non-related promoter and upstream of at least one coding sequence. <br>
3.	The fragment, genetic variant or deletion of the<br>
DNA molecule of claim 2, wherein the molecule includes at<br>
least 100 consecutive bases with a homology of 80% with 100<br>
consecutive bases of the sequence between nucleotides 22<br>
and 698 when the sequence is SEQ ID N° 3; at least 100<br>
consecutive  bases  with  a  homology  of  80%  with  100<br>
consecutive bases of the sequence between nucleotides 12<br>
and 591 when the sequence is SEQ ID N° 7; and at least 100<br>
consecutive  bases  with  a  homology  of  80%  with  100<br>
consecutive bases of the sequence between nucleotides 25<br>
and 1058 when the sequence is SEQ ID N° 11.<br>
4.	An expression cassette  for  promoting  high<br>
expression levels of at least one coding sequence, wherein<br>
the cassette comprises at least one of the introns of claim<br>
1 associated with a coding sequence and regulatory elements<br>
of gene expression.<br>
5.	A  plant  expression  vector  including  the<br>
expression cassette of claim 4.<br>
6.	A plant expression vector of claim 5, wherein<br>
the effect of said expression cassette is produced in the<br>
whole plant. <br>
7.	A cell comprising the vector of claim 5 and any<br>
descendant of the cell.<br>
8.	The cell of claim 7, wherein the cell is a<br>
bacterium.<br>
9.	The cell of claim 7, wherein the cell is a plant<br>
cell.<br>
10.	The  cell  of  claim  9  expressing  a  coding<br>
sequence under control of a promoter contained in the<br>
expression cassette of claim 4.<br>
11.	The cell of claim 9 comprising the expression<br>
cassette of claim 4, stably integrated in its genome.<br>
12 . The cell of claim 9 comprising the vector of claim 5, stably integrated in its genome.<br>
13. A transgenic plant and any seed and descendant thereof, wherein the plant is regenerated from the plant cell of claim 11. <br>
14.	The transgenic plant of claim 13 expressing in<br>
at least one of its cells a coding sequence under control<br>
of the expression cassette of claim 4.<br>
15.	The plant of claim 13, wherein the plant is a<br>
monocot plant.<br>
16.	The plant of claim 13, wherein the plant is a<br>
dicot plant.<br>
17.	An isolated DNA molecule comprising at least<br>
one of the sequences:<br><br>
-	COX5C-1 promoter-exon 1-intron-exon 2, SEQ N° 1-<br>
2 - 3 - 4 ;<br>
-	COX5C-2 promoter-exon 1-intron-exon 2, SEQ N° 5-<br>
6-7-8;<br>
-	COX5C-3 promoter-exon 1-intron-exon 2, SEQ N° 9-<br>
10-11-12,<br>
or a portion thereof.<br>
18.	The DNA molecule of claim 17, wherein:<br>
the COX5C-1 promoter-exon 1-intron-exon 2, SEQ N° 1-2-3-4; is from nucleotides 1 to 1211 of Cox5C-l, <br>
the COX5C-2 promoter-exon 1-intron-exon 2, SEQ N° 5-6-7-8 is from nucleotides 1 to 1153 fron Start Codon upstream to non coding region of Cox5C-2, and<br>
the COX5C-3 promoter-exon 1-intron-exon 2, SEQ N° 9-10-11-12 is from nucleotides 1 to 2205 of Cox5C-3,<br>
and wherein said portion comprises a fragment, genetic variant or deletion of at least one of said sequences that conserve the capability of enhancing gene expression of at least one coding sequence of a gene of interest.<br>
19.	The fragment, genetic, variant or deletion of<br>
the DNA molecule of claim 18, wherein the molecule includes<br>
at least 100 consecutive bases with a homology of 80% with<br>
100 consecutive bases of the sequence between nucleotides 1<br>
and 1211 when the sequence is SEQ ID N° 1-2-3-4; at least<br>
100 consecutive bases with a homology of 80% with 100<br>
consecutive bases of the sequence between nucleotides 1 and<br>
1153 when the sequence is SEQ ID N° 5-6-7-8; and at least<br>
100 consecutive bases with a homology of 80% with 100<br>
consecutive bases of the sequence between nucleotides 1 and<br>
2205 when the sequence is SEQ ID N° 9-10-11-12.<br>
20.	An expression cassette comprising the DNA<br>
molecule of claim 17 associated with a coding sequence. <br>
A  plant  expression  vector  comprising  the<br>
cassette of claim 20.<br>
21.	The  plant  expression vector  of  claim  21,<br>
wherein the effect of said expression cassette is produced<br>
in the whole plant.<br><br>
23.	A cell and any descendant thereof, transformed<br>
with the vector of claim 21.<br>
24.	The cell of claim 23, wherein the cell is a<br>
bacterium.<br>
25.	The cell of claim 23, wherein the cell is a<br>
plant cell.<br>
26.	The plant cell of claim 25 expressing a coding<br>
sequence under control of a promoter comprised in the<br>
expression cassette of claim 20.<br>
27.	The plant cell of claim 25, wherein the cell<br>
has the cassette of claim 20 stably integrated in its<br>
genome. <br>
28.	A transgenic plant and any seed and descendant<br>
thereof, wherein the plant is regenerated from the plant<br>
cell of claim 27.<br>
28.	The transgenic plant of claim 28 expressing in<br>
at least one of its cells a coding sequence under control<br>
of the expression cassette of claim 20.<br>
29.	The plant of claim 29, wherein the plant is a<br>
monocot plant.<br>
30.	The plant of claim 29, wherein the plant is a<br>
dicot plant.<br>
31.	An isolated DNA molecule comprising the introns<br>
of claim 1 in at least one of the following sequences:<br><br>
-	COX5c-1 gene exon 1-intron-exon 2, SEQ N° 2-3-4;<br>
-	COX5c-2 gene exon 1-intron-exon 2, SEQ N° 6-7-8;<br>
-	COX5c-3 gene exon 1-intron-exon 2, SEQ N° 10-11-<br>
12,<br>
or a portion thereof.<br>
32.	The isolated DNA molecule of claim 32, wherein: <br>
33.	the COX5c-l gene exon 1-intron-exon 2, SEQ N° 2-3-4 is from nucleotides 1 to 741 from the start codon of Cox5C-1,<br>
the COX5c-2 gene exon 1-intron-exon 2, SEQ N° 6-7-8 is from nucleotides 1 to 640 from the start codon of Cox5C-2,<br>
the COX5c-3 gene exon 1-intron-exon 2, SEQ N° 10-11-12 is from nucleotides 1 to 1123 from the start codon of Cox5C-3,<br>
and wherein said portion comprises, a fragment, genetic variant or deletion of at least one of said sequences keeping the capability of enhancing gene expression in plant cells when it is located in a construction under control of a non-related promoter and upstream of at least one coding sequence of interest.<br>
34.	A fragment, genetic variant or deletion of the DNA molecule of claim 33, wherein the molecule includes at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 741 when the sequence is SEQ ID N° 2-3-4; at least 100 consecutive bases with a homology of 80% with 100 consecutive bases of the sequence between nucleotides 1 and 640 when the sequence is SEQ ID N° 6-7-8; and at least 100 consecutive bases with a homology of 80% with 100 <br>
consecutive bases of the sequence between nucleotides 1 and 1123 when the sequence is SEQ ID N° 10-11-12.<br>
35.	An  expression  cassette  for  enhancing  the<br>
expression levels of at least one coding sequence of a gene<br>
of interest, wherein the cassette comprises at least one of<br>
the sequences exon l~intron-exon 2 of claim 32 associated<br>
to a coding sequence and to non-related regulatory elements<br>
of gene expression.<br>
36.	A  plant  expression  vector  comprising  the<br>
cassette of claim 35.<br>
37.	The  plant  expression vector  of  claim  36,<br>
wherein the effect of said expression cassette is produced<br>
in the whole plant.<br><br>
38.	A cell transformed with the cassette of claim<br>
35, and any descendant thereof.<br>
39.	The cell of claim 38, wherein the cell is a<br>
bacterium.<br>
40.	The cell of claim 38, wherein the cell is a<br>
plant cell. <br>
41.	The plant cell of claim 40 expressing a coding<br>
sequence under control of a non-related promoter comprised<br>
in the expression cassette of claim 35.<br>
42.	The cell of claim 38 comprising the expression<br>
cassette of claim 35 stably integrated in its genome.<br>
43 . The cell of claim 38 comprising the vector of claim 36 stably integrated in its genome.<br>
44.	A transgenic plant and any seed and descendant<br>
thereof, regenerated from the plant cell of claim 40.<br>
45.	The transgenic plant of claim 44 expressing in<br>
at least one of its cells a coding sequence under control<br>
of the expression cassette of claim 35.<br>
46.	The plant of claim 45, wherein the plant is a<br>
monocot plant.<br>
47.	The plant of claim 45, wherein the plant is a<br>
dicot plant. <br>
         48.	An isolated DNA molecule comprising at least<br>
one of the following sequences:<br>
-	COX5c-1 promoter-exon 1-exon 2, SEQ N°:1-2-4;<br>
-	COX5c-2 promoter-exon 1-exon 2, SEQ N°:5-6-8; and<br>
-	COX5c-3 promoter-exon 1-exon 2, SEQ N°:9-10-12,<br>
or a portion thereof.<br>
49.	The molecule of claim 48, wherein:<br>
the COX5c-1 promoter-exon 1-exon 2 sequence, SEQ N°:l-2-4, is from base 1 to 21 upstream of the start codon of COX5c-l linked to the sequence from base 699 to 1211 downstream of the start codon of COX5c-l;<br>
the COX5c-2 promoter-exon 1-exon 2 sequence, SEQ N°:5-6-8, is from base 1 to 11 upstream of the start codon of COX5c-2 linked to the sequence from base 592 to 1153 of the start codon of COX5c-2;<br>
the COX5c-3 promoter-exon 1-exon 2 sequence, SEQ N°: 9-10-12, is from base 1 to 24 upstream of the start codon of COX5c-3 linked to the sequence from base 1059 to 2205 upstream of the start codon of COX5c-3;<br>
and wherein said portion comprises a fragment, genetic variant or deletion of at least one of said sequences keeping the capability of enhancing tissue-specific gene expression in plant cells when some of these sequences is located upstream of at least one heterologous coding sequence of a gene of interest.<br>
50.	A fragment, genetic variant or deletion of the<br>
DNA molecule of claim 49, wherein the molecule includes at<br>
least 100 consecutive bases with a homology of 80% with 100<br>
consecutive bases of the sequence SEQ ID N° 1-2-4; at least<br>
100 consecutive bases with a homology of 80% with 100<br>
consecutive bases of the sequence SEQ ID N° 5-6-8; and at<br>
least 100 consecutive bases with a homology of 80% with 100<br>
consecutive bases of the sequence SEQ ID N° 9-10-12.<br>
51.	An  expression  cassette  for  enhancing  the<br>
expression levels of at least one coding sequence of a gene<br>
of interest, wherein the cassette comprises at least one of<br>
the sequences promoter-exon 1-exon 2 of claim 48 associated<br>
to a coding sequence.<br>
52.	A  plant  expression  vector  comprising  the<br>
cassette of claim 51.<br>
53.	The  plant  expression vector  of  claim  52,<br>
wherein the effect produced by said expression cassette is<br>
tissue-specific„ <br>
54.	The vector of claim 53, wherein the specific<br>
tissue is pollen.<br>
55.	A cell transformed with the cassette of claim<br>
51, and any descendant thereof.<br>
56.	The cell of claim 55, wherein the cell is a<br>
bacterium.<br>
57.	The cell of claim 55, wherein the cell is a<br>
plant cell.<br>
58.	The plant cell of claim 57 expressing a coding<br>
sequence under control of at least one of the sequences<br>
prornoter-exon  1-exon  2  of  claim 48  comprised  in  the<br>
expression cassette of claim 51.<br>
59.	The cell of claim 55, comprising the expression<br>
cassette of claim 51 stably integrated in its genome.<br>
60.	The cell of claim 55, comprising the vector of<br>
claim 52 stably integrated in its genome.<br>
61.	A transgenic plant and any seed and descendant<br>
thereof, regenerated from the plant cell of claim 57. <br>
62.	The transgenic plant of claim 61 expressing in<br>
at least one of its cells a coding sequence under control<br>
of at least one of the sequences promoter-exon 1-exon 2 of<br>
claim 48 and comprising the expression cassette of claim<br>
51.<br>
63.	The plant of claim 62, wherein the plant is a<br>
monocot plant.<br>
64.	The plant of claim 62, wherein the plant is a<br>
dicot plant.<br>
65.	The isolated DNA molecule of claim 1, wherein<br>
it is from Arabidopsis thaliana.<br>
66.	The isolated DNA molecule of claim 17, wherein<br>
it is from Arabidopsis thaliana.<br>
67.	The isolated DNA molecule of claim 33, wherein<br>
it is from Arabidopsis thaliana.<br>
68.	The isolated DNA molecule of claim 49, wherein<br>
it is from Arabidopsis thaliana.<br>
69.	A DNA molecule  capable  of  hibridizing  in<br>
astringent conditions with at least one of the sequences of<br>
claim 2.<br>
70.	A DNA molecule  capable  of  hibridizing  in<br>
astringent conditions with at least one of the sequences of<br>
claim 18.<br>
71.	A DNA molecule  capable  of  hibridizing  in<br>
astringent conditions with at least one of the sequences of<br>
claim 33.<br>
72.	A DNA molecule  capable  of  hibridizing  in<br>
astringent conditions with at least one of the sequences of<br>
claim 49.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1BYnN0cmFjdC0oMTYtMTAtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-Abstract-(16-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1BYnN0cmFjdC0oMjItMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-Abstract-(22-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1758-del-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1DbGFpbXMtKDE2LTEwLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Claims-(16-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1DbGFpbXMtKDIyLTA4LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Claims-(22-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE2LTEwLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Correspondence Others-(16-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDIxLTAyLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Correspondence Others-(21-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDIyLTA4LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Correspondence Others-(22-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1ERUwtMjAwNi1Db3JyZXNwb25kZW5jZS1PdGhlcnMgKDIzLTExLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-DEL-2006-Correspondence-Others (23-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1758-del-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1ERUwtMjAwNi1Gb3JtLTE4LSgyMy0xMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1758-DEL-2006-Form-18-(23-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1Gb3JtLTItKDIyLTA4LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Form-2-(22-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1Gb3JtLTMtKDE2LTEwLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Form-3-(16-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1Gb3JtLTMtKDIxLTAyLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Form-3-(21-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1HUEEtKDIyLTA4LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-GPA-(22-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1ncGEucGRm" target="_blank" style="word-wrap:break-word;">1758-del-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc1OC1kZWwtMjAwNi1QZXRpdGlvbi0xMzctKDIyLTA4LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1758-del-2006-Petition-137-(22-08-2014).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263308-method-of-mutagenesis.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263310-method-and-apparatus-for-performing-uplink-timing-synchronization-procedure-upon-handover-in-a-mobile-communication-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263309</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1758/DEL/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Aug-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CONSEJO NACIONAL DE INVESTIGACIONES CIENTIFICAS Y TECNICAS (CONICET)</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>AV. RIVADAVIA 1917-(C1033AAJ) CAPITAL FEDERAL -ABS. AS. - ARGENTINA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GRACIELA C. CURI</td>
											<td>CC 242 &quot;EL POZO&quot; S/N, 3000 PROVINCIA DE SANTA FE, ARGENTINA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DANIEL HECTOR GONZALEZ</td>
											<td>CC 242 &quot;EL POZO&quot; S/N, 3000 PROVINCIA DE SANTA FE, ARGENTINA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JULIETA CABELLO</td>
											<td>CC 242 &quot;EL POZO&quot; S/N, 3000 PROVINCIA DE SANTA FE, ARGENTINA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LIA RAQUEL CHAN</td>
											<td>CC 242 &quot;EL POZO&quot; S/N, 3000 PROVINCIA DE SANTA FE, ARGENTINA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0105268</td>
									<td>2006-12-15</td>
								    <td>Argentina</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263309-an-isolated-dna-molecule-an-expression-cassette-and-vector-comprising-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:28:48 GMT -->
</html>
