<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263381-a-method-to-increase-the-production-of-a-desired-chemical-compound-in-a-microorganism by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:07:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263381:A METHOD TO INCREASE THE PRODUCTION OF A DESIRED CHEMICAL COMPOUND IN A MICROORGANISM</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD TO INCREASE THE PRODUCTION OF A DESIRED CHEMICAL COMPOUND IN A MICROORGANISM</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method to increase the production of a desired chemical compound in a microorganism by introduction of a DNA sequence at the 5&#x27; end of the encoding DNA gene sequence capable of forming a stem loop and capable of increasing the stability of mRNA transcripts from one or more genes, thus stabilized mRNAs, corresponding DNA sequences and microorganisms.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Method for mRNA stabilization<br>
Pantothenate is a member of the B complex of vitamins and is a nutritional requirement for mammals including humans and livestock. In cells, pantothenate is converted to coenzyme A (CoA) and acyl carrier protein, the biologically active forms of the cofactor. These two coenzymes participate in over 100 different enzymatic reactions in the cell.<br>
Published PTC patent applications WO 01/21772, WO 02/057474, WO 02/061108, and WO 04/005527 (all filed by Omnigene Bioproduct s Inc., USA) describe methods to produce pantothenate using strains of Bacillus subtilis 168 that have higher expression levels of biosynthetic genes involved in pantothenate production. These genes include panB, panC, panD, panE, ilvB, ilvN, ilvC, ilvD, glyA, and serA. To achieve higher expression levels of these genes, the native promoters controlling transcription of said genes were removed and replaced by stronger constitutive promoters derived from an endogenous bacteriophage, SPOl, using standard genetic recombinant methods known in the art. Increasing the levels of transcription of a gene is well known in the art to lead to higher levels of the protein encoded by the overexpressed gene.<br>
It is also well known in the art that overproduction of proteins by means of transcription overexpression may lead to undesirable effects on cellular metabolism (WO 98/07846). Furthermore, it has also been described that protein overproduction may lead to deleterious effects in the translational machinery of the host cell (Hengjiang et al., 1995, J. Bacteriol. 177:1497-1504) and/or induction of proteolytic activities mediated by stress responses<br>
BESTATIGUNGSKOPIE<br>
(Ramirez D. M„ and W. E. Bentley, 1995, Biotechnol. Bioeng. 47:596-608) which could be the consequence of lower production titers. Therefore, devising methods for protein overproduction alternative to the use of constitutive strong promoters could be advantageous for the production at industrial scale of fine chemicals, like such as pantothenate.<br>
Transcript degradation is utilized by microorganisms as a means to control cellular protein content. On the other hand, microorganisms have developed mechanisms by which the stability of a given transcript is enhanced. To achieve this, transcripts are provided with nucleotide sequences capable of forming secondary structures which impose an impediment for mRNA degrading enzymes to exert their action. Despite substantial knowledge about mRNA degradation and stability, there are only a few examples where this knowledge has been applied for the expressed purpose of redirecting bacterial carbon flow for the production of fine chemicals.<br>
Smolke et al. (2001, Metabolic Engineering. 3: 313-321) describe the use of artificially generated sequences capable of stem-loop structure formation as mRNA stability elements to increase the steady-state level of transcripts encoded by two plasmid-borne crt genes in order to increase phytoene production in Escherichia coli. For this method to be useful, the above-mentioned mRNA stability elements must be precisely placed no more than one nucleotide away from a promoter transcriptional start site (Carrier and Keasling 1999, Biotechnol. Prob. 15: 58-64). Alternatively, if cleavage is desired at a site within the native mRNA molecules, the mRNA stabilizing element is required to be co-introduced with an RNase E cleavage site so that RNase E - specific cleavage results in a new mRNA molecule of similar structure, i.e. placement of the RNA stability element one (1) nucleotide from the 5' end. Either example requires laborious experimental work, limiting the usefulness of the method. Thus the development of stabilizing mRNA independent of promoters transcriptional start sites or independent of RNase E cleavage could offer a better alternative to engineer microorganisms interesting for the manufacture of fine chemicals and/or proteins at the industrial level.<br>
Gene orthologs to E. coli RNase E have not been found in microorganisms interesting for the manufacture of metabolites and/or proteins at industrial scale (Condon, 2003, Microbiol. Mol. Biol. Rev. 67:157-174). For Bacillus subtilis, evidence suggests that this<br>
bacterium contains two genes (ykqC [RNAseJl] and ymfA [RNAseJ2]) which could be functionally homologous to E. coli RNase E, yet do not show any significant nucleotide or amino acid sequence similarity (Even et al., 2005, Nucleic Acids Res. 31:2141-2152). Accordingly, the transcript degradation machinery of B. subtilis is quite different: only 6 out of 17 characterized E. coli-like enzymes involved in RNA degradation activities have been identified in B. subtilis.<br>
The above cited mRNA stabilization methods were first applied to plasmid-based replicons containing only one gene. Widner et al. (1999, W099/43835) discloses a method for producing a polypeptide in B. subtilis by addition of a stabilizing element from the B. thuringiensis crylllA gene inserted between tandem promoters and the structural gene encoding the polypeptide. However the presence of two promoters was necessary to achieve "saturating levels of mRNA" (W099/43835). Hue et al. (1995, J. Bacterid. 177:3465-3471) discloses a 5' mRNA stabilizer which stabilizes mRNA sequences by homology to the 3' end of 16S RNA. Also, Daguer et al. (2005, Lett. Appl. Microbiol. 41:221-226) discloses a plasmid-based mRNA stabilization method in which ribosomes bind to ribosome binding sites to generate RNA stability. This method achieved only a rather insignificant increase in product formation (i.e. 1.5 fold increase in levansucrase production), and in some cases actually decrease formation of a protein product.<br>
Moreover, biosynthesis, of fine chemicals, proteins, and other chemical compounds often utilize complex multi-gene clusters (i.e. operons) located at different sites on the chromosome and which generate multiple mRNA transcripts.'Consequently, such previous cited mRNA stabilization methods could be ill-suited to increase the protein synthesis since expression of cloned multi-gene clusters from plasmids can sometimes be unstable (Kim et al., 1982, Han'guk Saenghwa Hakhoechi 15:305-314; Gryczan, 1982, The Molecular Biology of the Bacilli [Dubnau, ed.], Academic Press, New York, New York, pp 307-329; Piece and Gutteridge, 1985, App. Environ. Microbiol. 49:1094-1100; Newell et al., 1987, Biochem. Soc. Trans. 11:281-282; Haeseleer, 1994, Res. Microbiol. 145:683-387; Al-Allaf et al., 2005, J. Biochem. Biophys. Methods 64:142-146). Notwithstanding, patent application WO 02/055711 discloses the possibility of improving the expression of genes involved in pantothenate production of Corynebacterium glutamicum (viz. at least one of ilvBN, ilvC, ilvD, panB, panC, and panD) by prolonging the lifetime of their mRNA transcripts, however, without disclosing how this can be achieved.<br>
Consequently, it is an object of the present invention to provide a method to increase the production of a desired chemical compound in a microorganism without strengthening native promoter signals controlling transcription of said structural gene sequences, viz. by i the introduction of a DNA sequence at the 5' end of the encoding DNA gene sequence capable of forming a stem loop and capable of increasing the stability of mRNA transcripts from one or more genes. This method is characterized in that the loop-forming DNA sequence is introduced seven or more DNA unpaired nucleotides down-stream of the start site of transcription of the relevant gene(s) of the microorganism.<br>
It is a further object of the present invention to provide stabilized mRNA sequences which contain a stabilizing element at their 5'end. The stabilizing element is transcribed from a DNA sequence introduced seven or more DNA unpaired nucleotides down-stream of the start of transcription of the relevant gene of the microorganism. Addition of this stabilizing element has the effect that the mRNA is no longer or less accessible to enzymatic degradation and thus a higher production of a desired chemical compound in a microorganism is the result.<br>
In a further embodiment the present invention relates to corresponding DNA sequences containing these mRNA stabilizing sequences and which upon transcription by a microorganism result stabilized mRNA transcripts, as well as to transformed microorganisms comprising such DNA sequences.<br>
Finally, the use of such DNAs or stabilized mRNA transcripts in a method to increase the stability of mRNA transcripts of one or more genes that generate multiple mRNA transcripts and that are located on a chromosome, plasmid or any other self-replicating DNA molecule, or a method to increase the production of a desired chemical compound by a transformed microorganism, respectively, are objects of the present invention.<br>
The term "chemical compound" means any carbon-based substance originating from cellular metabolism, i.e. the breakage and/or formation of one or more chemical bonds in a reaction facilitated by one or more enzymes in a cell that has biological activity. Examples of such compounds are proteins, enzymes, nucleotides, ribonucleotides, amino acids,<br>
vitamins (e.g. ascorbic acid, pantothenic acid), vitamin-like substances {e.g. coenzyme Q10), carotenoids, lipids and fatty acids.<br>
The term "microorganism" means a microscopic, self-reproducing, respiring organism including, but not limited to, bacteria, fungi (including yeast) and algae. The term bacteria includes both Gram-negative and Gram-positive microorganisms. Examples of Gram negative bacteria are any from the genera Escherichia, Gluconobacter, Rhodobacter, Pseudomonas, and Paracoccus. Gram-positive bacteria are selected from, but not limited to any of the families Bacillaceae, Brevibacteriaceae, Corynebacteriaceae, Lactobacillaceae, and Streptococaceae and belong especially to the genera Bacillus, Brevibacterium, Corynebacterium, Lactobacillus, Lactococcus and Streptomyces. Among the genus Bacillus, B. subtilis, B. amyloliquefaciens, B. licheniformis and B. pumilus are preferred microorganisms in the context of the present invention. Among Gluconobacter, Rhodobacter and Paracoccus, C. oxydans, R. sphaeroides and P. zeaxanthinifaciens are preferred, respectively. Examples of yeasts are Saccharomyces, particularly S. cerevisiae. Examples of preferred other fungi are Aspergillus niger and Pencillium chrysogenum.<br>
The phrase "strengthening of native promoter signals" refers to any genetic change of one or more nucleotides within a contiguous DNA sequence that interacts with RNA polymerase (for example, but not limited to, the "-35" and "-10" binding sites), which ftsults in the synthesis of a greater number of messenger RNA molecules (i.e. RNA transcripts) compared to the unmodified DNA sequence. The phrase also comprises replacement of a promoter by a stronger one. Methods of strengthening native promoter signals are well-known and generally used in the art. The phrase "without strengthening native promoter signals" is used to indicate that the method of increasing the production of a desired chemical compound in a microorganism in accordance with the present invention is different from these well-known methods. This phrase, however, does not mean that methods which combine the present new method and the well-known methods are not encompassed by the present invention.<br>
The term "promoter signals" means a contiguous DNA nucleotide sequence that specifically interacts with RNA polymerase (for example, but not limited to, the "-35" and "-10" binding sites), and allows for initiation of messenger RNA synthesis (i.e. synthesis of RNA transcripts).<br>
The term "native promoter signal" means a naturally occurring promoter signal found in a microorganism.<br>
The phrase "introduction of a DNA sequence" refers to any addition or insertion of a DNA sequence by DNA transformation, conjugation or transduction into the chromosome of a microorganism. Said addition or insertion occurs by DNA recombination that may or may not also result in a removal or deletion of chromosomal DNA nucleotides. Methods by which introduction of DNA sequences into microorganisms are achieved, especially by site-specific introduction, are well-known in the art and described in text books and scientific literature. They are standard procedures practiced by persons skilled in the art. The DNA sequence capable of forming one or more stem loops is introduced seven or more unpaired DNA nucleotides, i.e. at least 7, e.g. 8, 9, 10 or 11 down-stream of the start site of transcription of the relevant gene(s) of the microorganism. The number and kind of nucleotide changes are limited by the fact that no sequence is formed representing an RNase E-specific nuclease cleaving site.<br>
The term "increasing the stability of mRNA" means extending the half-life of mRNA sequences or blocking/delaying their degradation.<br>
The DNA sequence to be introduced (i.e. mRNA stabilizing element) can be any sequence capable of forming a double-stranded stem loop, viz. a sequence which is naturally occurring or derived from a naturally occurring sequence or a sequence which is completely or partly synthesized using methods well-known in the art. The sequence can be of any length but preferably consists of a minimum of 15 nucleotides, more preferably 23-100 nucleotides. The stem should consist of at least 6 base pairs, preferably at least 10 base pairs (with mismatch nucleotides or bulge loops possibly being present) and the loops may consist of 3-30 nucleotides. The calculated thermodynamic stability (AG) of the stem loop should be -2,8 kcal/mol or lower, preferably - 5kcal/mol or lower, according to algorithms developed by Zuker (2003, Nucleic Acids Res. 31:3406-3415), preferably lower, i.e. -3, -4, -5 or -6, preferably lower than -7, e.g. -8, -9, -10, -11 or -12 kcal/mol. In a preferred embodiment of the invention the DNA sequence to be introduced is a genome sequence or sequence derived from the genome of a microorganism, i.e. the same or a different microorganism to be used for the production of the desired chemical compound.<br>
In a particular/preferred embodiment of the present invention the DNA sequence is aenvea from the genome of a Bacillus, especially from B. subtilis. Examples of such sequences are contiguous sequences occurring in sequences from gene cggR to gene gapA, from gene hrcA to gene grpE, from gene ilvN to gene ilvC, from gene aprE to gene yhfO, from gene ybdA to gene gsiB and from gene ytxC to gene thrS of B. subtilis and particularly those represented by SEQ ID Nos. 1 - 6. The term "sequence occurring in a sequence from gene ... to gene ..." means a sequence between these two genes, i.e. the whole sequence or part thereof, including sequences extending to the genes themselves.<br>
Generally, a nucleic acid is considered to be within the scope of this invention if it is at least 70%, preferably at least 80%, or most preferably at least 90%, homologous to a naturally occurring nucleic acid sequence that can generate a m.RNA stabilizing element. Such homology can be determined experimentally by Southern hybridization analysis under the following conditions: hybridization in 5X SSC, 10-50% formamide, IX Denhardts solution, 100 µg/ml denatured salmon sperm DNA at 37-42 °C.<br>
SEQ ID No. 1 represents the cggR-gapA gene sequence.<br>
SEQ ID No. 2 represents a specific sequence between the genes cggR and gapA.<br>
SEQ ID No. 3 represents the hrcA-grpE gene sequence.<br>
SEQ ID No. 4 represents a specific sequence between the genes hrcA and grpE<br>
SEQ ID No. 5 represents a chromosomal DNA sequence in which a cggR-gapA stabilizing<br>
element is inserted within the 5' leader sequence of the ilvB gene in the<br>
operon ilvBNC. SEQ ID No. 6 represents a chromosomal DNA sequence in which a hrcA-grpE stabilizing<br>
element is inserted within the 5' leader sequence of the ilvD gene.<br>
While the method of the present invention will be described in detail with respect to the expression of pantothenic acid one skilled in the art will recognize that this method can be applied universally to increase the production of any microbial metabolite or of any chemical compound and/or protein to be synthesized by microbial cells that utilize gene-encoded biosynthetic enzymes that convert a substrate (e.g. glucose) or precursor (e.g. pyruvate) to such a chemical compound or to increase the production of any protein.<br>
EXAMPLES<br>
General Methodology<br>
Strains and plasmids. Bacillus subtilis strains of the present invention are derived from strains CU550 (trpCl ilvC leuC and 1A747 (SPßc, prototroph), which are derivatives of B. subtilis 168 (trpC2). Both strains were obtained from the Bacillus Genetic Stock Center, The Ohio State University, Columbus, Ohio 43210 USA. E. coli strain Top10 (Invitrogen) was utilized for regular cloning purposes. Plasmids pUCl8, pUC19, and pBR322 (New England Biolabs) were used as general purpose cloning vectors. Antibiotic resistance genes that confer resistance to chloramphenicol (cat), tetracycline (tet), erythromycin (erm), and spectinomycin (spec) were obtained from plasmid pC194 (GeneBank Ml9465, Cat# IE 17 Bacillus Genetic Stock Center, The Ohio State University, Columbus, Ohio 43210 USA), pBC16 (GeneBank X51366, Cat# 1E9 Bacillus Genetic Stock Center), pDG646 and pDG1726 (Guerot-Fleury et. al., 1995, Gene 167:335-336). The P26 and P,s promoters of the B. subtilis bacteriophage SPOl (Lee et al., 1980, Mol. Gen. Genet. 180:57-65) was obtained from plasmids pUC18SPO1-26 and pXI23roDTD-SPO1-15, a derivative of plasmid pX12 (Humbelin et al, 1999, J. Ind. Microbiol. Biotech. 22:1 -7), respectively.<br>
Media. Standard minimal medium (MM) for B. subtilis contains IX Spizizen salts, 0.04% sodium glutamate, and 0.5% glucose. Standard solid complete medium is Tryptose Blood Agar Broth (TBAB, Difco). Standard liquid complete medium is Veal Infusion-Yeast Extract broth (VY). The compositions of these media are described below:<br>
TBAB medium: 33g Difco Tryptose Blood Agar Base (Catalog # 0232), 1 L water. Autoclave.<br>
VY medium: 25g Difco Veal Infusion Broth (Catalog # 0344), 5g Difco Yeast Extract (Catalog #0127), 1L water. Autoclave.<br>
Minimal Medium (MM-): 100ml 10X Spizizen salts; 10 ml 50% glucose; 1 ml 40% sodium glutamate, qsp 1L water.<br>
IPX Spizizen salts: 140g K2HPO4; 20g (NH4)SO4; 60g KH2PO4; l0g Na3 citrate.2H2O; 2g MgSO4.7H2O; qsp 1L with water.<br>
VFB MMGT medium: 100 mi 10X VFB MM; 100 ml 0.5 M Tris (pH 6.8); 44 ml 50% glucose; 2 ml Trace elements solution; 2 ml Fe solution; 2 ml CaCl2 solution; 2 ml Mg/Zn solution; 748 ml sterile distilled water.<br>
IPX VFB minimal medium (10X VFB MMV. 2.5g Na-glutamate; 15.7g KH2PO4; 15.7g K2HPO4; 27.4 g Na2HPO4.12H2O; 40g NH4C1; 1 g citric acid; 68 g (NH4)SC4; qsp 1 L water.<br>
Trace elements solution: 1.4g MnSO4-H2O; 0.4g CoCl2-6H2O; 0.15g (NH4)6Mo70244H2O; O.lg A1C13-6H2O; 0.075g CuCl2-2H2O; qsp 200 ml water.<br>
Fe solution: 0.21g FeSO4.7H2O; qsp 10 ml water.<br>
CaCb solution: 15.6g CaCl2.2H2O; qsp 500 ml water.<br>
Mg/Zn solution: lOOg MgSO4.7H2O; 0.4g ZnSO4.7H2O; qsp 200 ml water.<br>
SMG medium: 62.78g MOPS, 20 g Cargill soy four (200/20), 1 ml PSTE-1000X solution, 5 g Na-glutamate and 8 g (NH4)2SO4, water up to 735 ml (pH 7.2); autoclaved (30 min. at 121°C). After autoclaving, 100 ml 1 M K-phosphate buffer (pH 7.2), 120 ml 50% glucose, 10 ml 1 M MgSO4.7H2O, 1.4 ml 1 M CaCl2.2H2O and 35 ml sterile distilled water was added.<br>
PSTE-1000X solution: 0.2g MnCI.4H2O; 0.15g ZnSO4.7H2O; 0.2g CoCl2.6H2O; 0.025g CuSO4.5H2O; Na2Mo04.2H2O; qsp 100 ml water.<br>
Antibiotics: Ampicilin (Amp) or kanamycin (Km) were utilized at concentrations of 100 µg/rnl and 50 µg/ml, respectively, to transform and propagate plasmids in E. coli cells grown in LB complex medium. To transform antibiotic gene-containing DNA fragments into B. subtilis, 5 µg/ml chloramphenicol (Cm), 15 µg/ml tetracycline (Tc) and 50 µg/ml spectinomycin (Spec) was added to the media. For erythromycin (Erm).gene selection, a mixture of 1 µg/ml erythromycin/ 25 µg/ml lincomycin was used.<br>
Pantothenate assays in shake flasks:<br>
Shake flask culture conditions: Cell cultures grown overnight in VY rich medium were used to inoculate VFB MMGT medium (1:100 dilution). Growth was monitored until cells reached an OD600 of-0.6-0.8 at which time they were diluted once more in same medium to an OD600 of 0.03. Growth was resumed for an additional 18 hours after which samples were collected, cells removed and supernatant analyzed by FTPLC. Alternatively cell<br>
cultures grown overnight can be used to inoculate SMG medium ana supernaxams analyzed by HPLC after 24 hours growth.<br>
HPLC assay: Chromatography of samples was performed on a Phenomenex LUNA C8 column, using an Agilent 1100 HPLC system equipped with a thermostat-maintained autosampler and a diode array detector. The column dimensions are 150 x 4.6 mm, particle size 5 micron. The column temperature was kept constant at 20°C. The mobile phase is a mixture of 0.1% acetic acid (A) and methanol (B). Gradient elution is applied, ranging from 1% B to 45% B in 15 minutes. The flow rate is 1 ml/min. Pantothenate was monitored using UV absorption at 220 run, and is eluted at approximately 9.6 min. The calibration range of the method is from 1 to 100 mg/1 pantothenate.<br>
Molecular and genetic techniques. Standard genetic and molecular biology techniques are generally know in the art and have been previously described. DNA transformation, PBS1 generalized transduction, and other standard B. subtilis genetic techniques are also generally know in the art and have been described previously (Harwood and Cutting (eds), 1992, Molecular biological methods for Bacillus. New York: John Wiley and Sons).<br>
Northern blot analysis. Cells grown in VFB MMGT medium on the logarithmic phase (OD60o=~0.6) were harvested at 4 °C and immediately frozen in liquid nitrogen after decanting of supernatant. Total RNA was extracted as follows. The pellet was resuspended in ice-cold TE buffer (10 mM Tris, 1 mM EDTA, pH 8.0). The cells were lysed in a mixture containing macaloid, phenol/chloroform, SDS and acid-washed glass beads by shaking in a bead beater (BioSpec) for 2 min. After centrifugation the supernatant was subjected to phenol/chloroform extraction three times. Total RNA was resuspended in diethylpyrocarbonate (DEPC)-treated H2O after two steps of precipitation and washing. After DNase I treatment total RNA was purified using the RNeasy Midi Kit (Qiagen). In this step smaller RNA molecules like tRNAs were eliminated. For quality control an aliquot of total RNA was subjected to 1.2% agarose gel analysis and/or was analyzed on a RNA6000 NanoChip (Agilent BioAnalyzer). Equal amounts of total RNA were loaded on a 1.2% agarose gel and transcripts separated by electrophoresis. After transferring the RNAs from the agarose gels to nylon membranes, these were probed against DIG-labeled anti-sense mRNA probes. The probes were generated from PCR fragments including appropriate T7 polymerase binding sites by the use of primer pairs. Probe generation from<br>
these PCR fragments as well as blotted membrane testing was developed by the use ot tne DIG Northern Starter Kit (Roche Diagnostics) following manufacturer's instructions.<br>
Example I<br>
Introduction of mRNA stabilizing elements downstream of the native promoter of B. subtilis ilvD resulting in increased protein synthesis.<br>
To analyze the possibility of protein overproduction mediated by an unmodified promoter expressing a gene to be translated from a stabilized transcript, an mRNA stabilizing element was inserted as 5' untranslated leader region between the native ilvD promoter and the ilvD gene. PCR was used to generate a DNA fragment, which included the intergenic region existing between the B. subtilis genes ypgR and ilvD. This fragment, of 519 bp, was engineered to include BgIII and HindEl contiguous restriction sites 21 bp upstream of ilvD translational start codon. Primers PilvD+7 and PilvD-2 (Table 1) were utilized to amplify a fragment from B. subtilis DNA, which spans the entire ypgR-ilvD intergenic region starting 250 bp from the ilvD translational start codon. The DNA fragment was cloned in pCRXLTOPO (Invitrogen), its identity confirmed by sequencing analysis, and isolated from the vector backbone as EcoKUBamHI cassette. After ligation with likewise digested plasmid pDG1728 (Guerout-Fleury et al., 1996, Gene 180:57-61), the resulting pPA475 plasmid was transformed in B. subtilis 1A747 resulting in strain PA494, which contained a single copy of the ilvD promoter region transcriptionally fused to a lacZ gene integrated within the amyE gene, including BglR and Hindis, sites upstream of the putative RBS of ilvD. A second promoter probe was generated to further analyze the effect on protein production by the introduction of an mRNA stabilizing element as untranslated leader sequence. Thus, a DNA fragment containing the hrcA-grpE intergenic region was amplified from B. subtilis chromosomal DNA by PCR using primers 2HrcLoop+ and 2HrcLoop- (Table 1). The synthesized fragment 127 bp was digested with BglU and Hindlll and ligated to BgnUHindIII digested pPA475 DNA. The resulting plasmid pPA477 was then transformed in 1A747, selecting for spectinomycin resistance. This yielded strain PA517. Strains PA494 and PA517 are isogenic strains except PA494 contains an ilvD-lacZ fusion with the wild-type 5' untranslated leader region and PA517 contains an ilvD-lacZ fusion a 5' untranslated leader region with the hrcA-grpE RNA stabilizing element. In standard ONPG assays well known to those skilled in the art, strain PA517 produced fourfold more P-galactosidase activity than strain PA494 after 48 hours of growth in minimal medium  shake flask cultures.  This increase in P-galactosidase activity can only be<br>
attributed to the presence  of the hrcA-grpE stabilizing element preceding me  liv structural gene.<br>
Table ] - List of primers used to engineer microorganisms overproducing the LacZ protein under control of naturally occurring and not modified promoters.<br>
Primer      ID No.       Sequence<br>
PilvD+7	7	5'-GCGACTCCAGCAAGCTTGTTCGC-3'<br>
PilvD-2	g	5'-<br>
TTCTGGATCCATGGTGATCCTCCTAAGATCTAAGCrrCAATTGTTTGATTGGATTTTATTTTG-3'<br>
2HrcLoop+	9	5'-AAAAGCATTAAGCTTTCTGTATGATGAATAAGGG-3'<br>
2HrcLoop-	10	5'-CAACGGATCCTTTTTCTrCTGACATTGTGTTC-3'<br>
Example II<br>
High titers of pantothenate production can be achieved when IlvD protein is overproduced using either a constitutively-expressed exogenous promoter or a native promoter containing an mRNA stabilizing element.<br>
To compare the effect on metabolite production of enzyme overexpression mediated by stabilization of mRNA expressed under the control of a native promoter with the method well described in prior art of substituting native promoters for constitutively expressed strong ones, two strains were obtained by engineering the chromosomal DNA regions upstream of the B. subtilis ilvD gene. A first strain was constructed to overexpress the IlvD protein under the control of the strong constitutive SPO1-26 promoter. To do so, B. subtilis PA49 (P15 panBCD P,5 panE) a derivative of strain CU550 that contains SPO1-15 promoter modifications of pantothenate biosynthetic genes panBCD and panE (WO 2004/113510), was further modified to contain an ilvD gene the expression of which is controlled by the SPO1-26 promoter. To achieve this, the promoter region of ilvD (ilvDp~) was first deleted from strain B. subtilis 1A747 by the use of Long Flanking Homology PCR (LFH-PCR). PCR-generated fragments Fl and F2 were obtained using primer pairs Pl/ilvD/for, P2/ilvD/r/sp and P3/ilvD/f/sp, P4/ilvD/rev (Table 2), and B. subtilis 1A747 chromosomal DNA as template. These fragments were then used as primers in a second PCR reaction using as template the spectinomycin-resistance gene cassette from plasmid pSPEC12flip (Wade et al.,1999,  J Bacteriol. !£l_:4365-4373). This final  PCR DNA<br>
product was transformed into B. subtilis 1A747 selecting for spectinomycm-resistance (Spec1). Many Specr colonies were recovered and several were confirmed to have a deletion of the ilvD promoter region by PCR analysis using primers PlilvD/for and P4ilvD/rev. A 4000 bp PCR fragment (indication deletion of the ilvD promoter region) was detected using DNA isolated from Specr colonies whereas DNA from non-transformed cells (no deletion) generated a 3000 bp fragment. Colonies were also tested for the expected auxotrophy to isoleucine (He), leucine (Leu), and valine (Val) amino acids: all Specr, PCR positive colonies failed to grow on minimal medium lacking He, Leu, and Val amino acids. One Specr, PCR positive, Ilv' auxotroph was named PA24 (P/s panBCD P15 panE AihDp::spec) and saved for further use. PA49 was next transduced with a PBS1 lysate of PA24, selecting for spectinomycin resistance (Spec). Spec' colonies were recovered and one was confirmed by PCR and Ilv auxotroph to contain the ∆ilvDp::spec mutation. This colony was renamed PA60 (P15panBCD P15panE AilvDp::spec).<br>
Strain PA24 was then used to generate an ilvD gene under the expression of the SPO1-26 promoter. Two DNA fragments, Fl and F2, were generated by LFH-PCR using primer pairs Pl/ilvD/for and P2/ilvD/f7263 and P3/ilvD/r/26 and P4/ilvD/rev, respectively (Table 2), and B. subtilis 1A747 chromosomal DNA as template. These fragments were then used in a second PCR reaction using as template the SPO1-26-containing plasmid pUC18SPO1-26. This final PCR DNA product was transformed into PA24 selecting for Ilv+ prototrophy. Many Ilv+ colonies were recovered and shown to have lost resistance of spectinomycin resistance (i.e., Specs) as expected by replacement of the spec gene with the P26 promoter fragment. Moreover, diagnostic PCR analysis of several Ilv+ Spec5 colonies using primers P26-seq and P4ilvD/rev confirmed the presence of the P26 promoter adjacent to the ilvD structural gene: a 2000 bp PCR fragment (indication of the presence of P26) was detected using DNA isolated from Ilv+ Spec5 colonies whereas DNA from non-transformed cells (no SPO1-26 promoter) generated no PCR fragment with the same primers. DNA from non-transformed cells generated a 2000 bp PCR fragment (indication of the presence of Pwl) with primers ilvDwt-prom and P4ilvD/rev, whereas DNA from Uv+ Specs colonies generated no PCR products. One I!v+ Specs PCR positive colony was renamed PA27 (P26 ilvD). The P26 ilvD modified gene was then transferred to B. subtilis PA60 (P15 panBCD P15panE ∆ilvDp::spec) by PBS1 transduction using methods know by those skilled in the art. By selecting for llv+ prototrophy and screening form Spec5 colonies, the AilvDp::spec was replaced by P26 livD resulting in the strain PA62 (P15 panBCD P15 panE P26 ilvD). DNA sequencing of the chromosomal P26 ilvD region of PA62 detected a single point mutation within the ilvD coding region, which caused a Gly-to-Asp amino acid change in residue 320. The ilvD coding sequence was then restored to wild-type by first removing an internal segment of the ilvD gene encompassing this mutation, creating an auxotrophic IlvD" mutant of PA62 (renamed PA64), then by converting PA64 to Ilv+ prototrophy using<br>
wild type chromosomal DNA., using methods well-known to the skilled worker in the nem. This generated strain PA73 (P15panBCD P15panE P26 UvD).<br>
A second strain isogenic to PA73 but containing an unmodified livD gene promoter region and an mRNA stabilizing element between the UvD gene and its native promoter instead of a strong constitutive one was constructed as follows. Three overlapping DNA fragments of 272 bp, fragment Fl, 1683 bp, fragment F2 and 1102 bp, fragment F3 respectively were generated by PCR. Fragment Fl was obtained by using plasmid pPA477 as template and the synthetic oligonucleotides PilvDHrcLoop- and PilvDUP+ (Table 2) as primers. Fragment F2 was obtained by using B. subtilis 168 chromosomal DNA as template and synthetic oligonucleotides P4/ilvD/rev and HrcALoopPilvD+ as primers (Table 2). Fragment F3 was obtained by the use of B. subtilis 168 as template and synthetic oligonucleotides PilvDUP- and Pl/i!vD/for as primers (Table 2). Fragments Fl and F3 were purified by agarose gel, mixed, and used as template in yet a fourth PCR reaction which included oligonucleotides PilvDHrcLoop- and P'lilvD as primers. This generated fragment F13 of 1354 bp. Fragment 2 was gel purified, mixed with fragment Fl and the mixture utilized as template in a fifth reaction which included oligonucleotides PilvDUP+ and P'4ilvD as primers. This generated fragment F12 of 1935 bp. Fragments F12 and F13 were cloned in pCRXLTOPO (invitrogen) following manufacturer instructions. This generated plasmids pF12 (Fragment F12 cloned in pCRXLTOPO) and pF13 (fragment F13 cloned in pCRXLTOPO). Fragment F13 was amplified again using pF13 as template and oligonucleotides PilvDHcrl3+ and PilvDHcrl3- as primers. This generated fragment F13_2. Fragment Fl3_2 was digested with Pstl and Xbal and ligated to a likewise digested pUC19 plasmid (New England Biolabs). This yielded plasmid pUCPilvDHcrl3. After digestion with Kpnl and Xbal, fragment F12 was ligated to likewise digested plasmid pUCPilvDHcrB. The ligation mixture was directly transformed into PA60 competent cells obtaining this way strain PA590. Shake flask analysis in SMG medium revealed production titers of 1.7 g/1 for PA49, 2.5 g/1 for PA73 and 2.6 g/1 for PA590, proving the ability of the engineered strain with the stabilized transcript to yield similar titers to that engineered with a constitutively expressed strong promoter.<br>
Table 2. List of primers used to engineer the livD gene in pantothenate overproducer strains.<br>
Primer	ID No.      <br>
 (Sequence Removed)<br>
Example III<br>
Introduction of mRNA stabilizing elements downstream of the native promoter of the Uv operon increases synthesis of proteins.<br>
The ability of an mRNA stabilizing element to induce protein overproduction was tested under the environment of another native promoter expressing genes involved in pantothenate synthesis. To do so, the promoter region of ilvB (UvBp) including the untranslated leader region was engineered to include two restriction sites to allow subsequent modification. The two restriction sites were located 474 bp (PshAl) and 7 bp (Nhel) upstream of the ilvB structural gene translational start codon. Two overlapping PCR<br>
fragments were generated from B. subtilis 168 chromosomal DNA witn primer pairs PilvUP2+, PilvUP- and Pilv+, Pilv- (Table 3). After assembling by a third PCR reaction to generate a single DNA fragment using the two shorter overlapping fragments as primers, the resulting fragment was TA-cloned in pCRXLTOPO (Invitrogen) following manufacturer instructions, and its sequence confirmed. The cloned fragment was then removed by EcoRIIBamHI digestion and subcloned into the lacZ promoter probe vector pDG1728 (Guerout-Fleury et al., 1996, Gene 180:57-61) following procedures known to those skilled in the art, yielding plasmid pPA415. This plasmid was next transformed in B. subtilis 1A747 selecting for spectinomycin-resistance to integrate the PlivB-IacZ fusion into the amyE chromosomal locus, generating strain PA431.<br>
Plasmid pPA415 was further modified to contain mRNA stabilizing elements. To achieve this, this plasmid was digested with PshAI/Nhel to release the ilv-leu leader region from the remainder of the vector sequences, ilvB native promoter region and upstream regions, and the lacZ structural gene segment (i.e. backbone fragment). This backbone DNA fragment was purified and ligated to sequences containing the B. subtilis cggR-gapA region that harbors an mRNA stabilizing DNA sequence (i.e. stabilizing element, Meinken et al., 2003, Microbiology 149: 751-76). This fragment was PCR amplified from B. subtilis 168 by using primer pair CggRLoop+/CggRLoop- (Table 3), purified and digested with PshAl and Nhel. The ligated DNA was transformed into E. coli competent cells selecting for ampicillin-resistance. This resulted in plasmid pPA422, in which the untranslated leader region between the ilvB promoter and the lacZ reporter gene was replaced by the cggR-gapA RNA stabilizing element (i.e. PilvΩcggR-gapA-lacz). To insert the PilΩcggR-gapA-locZ cassette into the amyE chromosomal locus of B. subtilis, pPA422 plasmid DNA was then transformed into 1A747, selecting for spectinomycin-resistance. This yielded strain PA432. In standard ONPG assays developed with exponentially grown shake flask cultures containing minimal medium, strain PA432 produced 7-fold higher p-galactosidase levels than the isogenic strain PA431 containing the wild type promoter fusion (i.e. PilvB-lacZ) without the cggR-gapA intergenic region.<br>
Table 3. List of primers used to engineer microorganisms overproducing the LacZ protein under control of naturally occurring and not modified promoters.<br>
Primer       ID No,    Sequence	<br>
(Sequence Removed)<br>
Example IV<br>
Strains expressing the ilvBNC-leuABCD operon using its native promoter and an exogenous mRNA stabilizing element with the 5' leader region can achieve similar protein levels as strains containing the UvBNC-leuABCD operon under transcription control of a constitutively strong exogenous promoter.<br>
To replace the native UvB promoter with strong constitutive promoters or to introduce an mRNA stabilizing element between the native UvB promoter region and the UvB structural gene, a deletion of the native UvB promoter region (PilvB) was first constructed as follows, A plasmid containing a chloramphenicol (cat)-resistance gene cassette flanked by B. subtilis chromosomal sections upstream and downstream of PilvB was first constructed. To achieve this, a DNA fragment containing the 5' end of the ysnD gene (located upstream of the PilvB promoter) was synthesized by PCR using from B. subtilis 168 chromosomal DNA as template and primers ysnD3- and ysnD+ (Table 4). A second PCR-generated fragment including a cat resistance-encoding gene was prepared using primers ilvBCat+ and ilvBCat- (Table 4) and plasmid pDG1661 (Guerout-Fleury et al. 1996, Gene 57-61) as template. This fragment overlaps the 3' end of the ysnD-containing PCR fragment. After purification, both PCR fragments were used as template in a third PCR reaction with primers ilvBCat+ and ysnD3-, yielding an ysdD'-cat fragment. A fourth PCR reaction was used to amplify the 5' end of the ilvB structural gene, which is downstream of PilvB. Thus, a PCR product was obtained from B. subtilis 168 chromosomal DNA by the use of primers ilvB+ and ilvB- (Table 4). This ilvB' fragment was cloned by the pCXLTOPO kit (Invitrogen) following manufacturer's instructions. The cloned fragment was subcloned<br>
into pUC19 (New England Biolabs) as a Kpnl/Sacl fragment. After its integrity was confirmed by restriction analysis, the ysnD'-cat fragment described above was inserted (using BamHI/Kpnl sites) upstream of the ilvB' segment yielding the final plasmid pPA40l. This plasmid was engineered in such a way that the cat gene sequences could be removed from the ysnD'-cal-ilvB' cassette by KpnVMlul digestion and replaced by any other DNA element. Plasmid pPA401 was then transformed into a B. subtilis prototrophic wild-type strain (1A747) and into a pantothenate producer (PA73), selecting for chloramphenicol resistance. Cmr colonies were found to be an Ilv" auxotroph (i.e. bacteria that fail to grow on minimal medium without the addition of valine, leucine, and isoleucine amino acids). The resulting strains were named PA401 (&amp;ilvp::cat) and PA441 (P/s panBCD P15panE P26ilvD ∆ilvp::cat).<br>
To generate a strain containing an ilv-leu operon under the control of its native promoter but including a modified leader region, a DNA fragment containing the PilvΩcggR-gapA cassette was amplified by PCR from plasmid pPA422 (see above) with primers PilvUP3+ and CggRLoop2- and pPA422 plasmid DNA as template (see above). The thus generated PCR fragment was digested with KpnVMlul and ligated to a purified KpnVMlul cat-free fragment of pPA401 (i.e. a DNA fragment with the structure, 5'-Kpnl-ysnD-vector replication sequences-ilvB'-MluI-3'). The ligation mixture was directly transformed into PA441 and colonies selected for growth on minimal medium without supplementation with He, Val, or Leu amino acids. Many Ilv+ colonies were obtained and the genetic background of several was confirmed by PCR. In addition these Ilv+ PCR+ colonies were also sensitive to chloramphenicol (Cms) as expected of replacement of the cat gene with the PuvB promoter-mRNA stabilizing element region. One IIv+ PCR+Cms colony was saved (PA445) for further study.<br>
To generate strains containing an ilv-leu operon under the control of SPO1-26 if 26), a DNA fragment which contained a P26 promoter was first PCR amplified using primers P26+1 and P26- and pUC19SPOl-26 plasmid DNA as template. The resulting DNA product was digested with KpnVMlul and ligated to a purified KpnVMlul cat-bee fragment of pPA40l (i.e. a DNA fragment with the strict, 5'-Kpnl-ysnD-vecior replication sequences-ilvB'-Mlul-3'). The ligation mixture was directly transformed into PA401 (Ailvp:;cat) and colonies selected for growth on minimal medium without supplementation with He, Val, or Leu. Many Ilv+ colonies were obtained and the genetic background of several was confirmed by PCR. In addition these Ilv+ PCR+ colonies were also sensitive to chloramphenicol (Cms) as expected of replacement of the cat gene with the cggR-modified PUVB promoter region. One<br>
IIv+ PCR+ Cms colonies was saved and named PA402. DNA sequencing contirmea tne presence of a modified ilv-leu operon in which the cggR-gapA mRNA stabilizing element was inserted downstream of the ilvB promoter region (i.e. PilvΩcggR-gaPA ilvBNC-leuABCD). The PilvΩcicggR-gapA ilvBNC-leuABCD operon was next transferred to the pantothenate-production strain PA73 using PBS1 generalized transduction in the following way: A PBS1 phage lysate was prepared using PA402 and a method known to those skilled in the art. This lysate was used to infect PA441 and colonies selected for growth on minimal medium without supplementation with He, Val, or Leu. Many Ilv+ colonies were obtained and the genetic background of several was confirmed by PCR. In addition these Ilv+ PCR+ colonies were also sensitive to chloramphenicol (Cms) as expected of replacement of the cat gene with the PilvB promoter-mRNA stabilizing element region. One Ilv+ PCR+ Cms colony was saved (PA444) for further study.<br>
Two dimensional protein gels were used to compare the level of protein synthesis of ilv-and /eu-encoded genes between strain PA444 containing the strong constitutive SPO1-26 promoter and wild-type 5' untranslated leader region, and strain PA445 containing the wild-type promoter and a 5' untranslated leader region modified with the cggR-gopA RNA stabilizing element, using methods known by those skilled in the art. Significant increases in protein levels were observed in both PA444 and PA445 with respect to the parental PA73 strain:<br>
PA444 - IlvB 56%, IIvC 473%, LeuA 271%, LeuB 579%, LeuC 153%, and LeuD 306%;<br>
PA445, IlvB 60%, IlvC 648%, LeuA 274 %, LeuB 499%, LeuC 195%, and LeuD 523%.<br>
It can be concluded from these results that the increase in protein levels encoded by genes from the ilv-leu operon was within the same range for both PA444 and PA445 strains.<br>
Table 4. List of primers used to engineer the ilvBNC-leuABCD operon in pantothenate overproducer strains.<br>
Primer     ID No.      Sequence	<br>
(Sequence Removed)<br>
Example V<br>
Introduction   of endogenous   mRNA   stabilizing   elements   are   able   to   increase transcript abundance of mRNA encoded by native promoters.<br>
PA444 and PA455 are isogenic strains except for the presence of different DNA elements controlling transcription of the ilv-leu operon: PA444 contains the strong constitutive SPO1-26 promoter and wild-type 5' untranslated leader region, and PA445 contains the wild-type promoter and a modified 5' untranslated leader region with the cggR-gapA RNA stabilizing element. To analyze the effect of the two different DNA elements on transcription of the ilv-leu operon, standard Northern blots were used to analyze the mRNA transcript profiles of both strains. Labeled antisense mRNA probes were generated to the ilvB, ilvC, and leuD genes from PCR fragments obtained with primers IlvBFor/IlvBRev (IlvB probe), IlvCFor/IlvCRev (IlvC probe), and LeuDFor/LeuDRev (LeuD probe) (Table 5), and hybridized separately to total RNA separated on denaturing agarose gels under standard conditions. Results showed that that the two strains generated different transcript profiles. A high abundance 3.5 kb mRNA species that hybridized to both the ilvB and ilvC probes, but not to the leuD probe, was detected in PA445 total RNA, but not in PA444 total RNA. According to Mader et al, this 3.5 kb transcript encompasses the ilvB, ilvN, and ilvC genes and is not detected from total RNA from wild type bacteria (Mader et al.3 2004, J. Bacterid. 186:2240-2252). Moreover since this RNA species is not detected in PA445, anyone skilled in the art will recognize that the increase abundance of this mRNA species was caused by increasing the stability of the mRNA message by the cggR-gapA DNA element and not by increasing the level of transcription.<br>
Likewise, the abundance of two additional transcripts of 8.5 kb and 2.5 kb was greater in strain PA445 than PA444. Since the 2.5 kb transcript hybridized to the ilvB probe but not to the ilvC and leuD probes, it can be concluded that this mRNA encompasses the ilvB and ilvN genes. Since the 8.5 kb transcript hybridized to the leuD probes, it must encompass the entire ilv-leu operon. Using the same rationale above, the greater abundance of these two RNA species in PA445 than in PA444 can only be attributable to their higher stability, which in turn must come as a consequence of the introduction of the cggR-gapA element into 5' leader sequence of the ilv-leu operon.<br>
Table 5. List of primers used to obtain DNA fragments encoding KIN A anusense iv the ilvB, UvC or leuC genes.<br>
Primer        ID No,	Sequence	<br>
(Sequence Removed)<br>
Example VI<br>
Increasing the level of proteins encoded by the ilvBNC-leuABCD operon by use of mRNA stabilizing elements, increases the production of pantothenate in B. subtilis cells.<br>
 Strains PA73, PA444, and PA445 were evaluated for pantothenate production in shake flask cultures containing VFB MMGT minimal medium and grown for 48 hours. HPLC analysis of cell-free supernatants prepared from these shake flask cultures revealed the presence of pantothenate at the following levels:<br>
PA73, 550 mg/1;<br>
;	PA444, 150 mg/1;<br>
PA445, 750 mg/1.<br>
Results showed that increasing the transcription level of the UvBNC-leuABCD operon by a constitutive strong promoter yielded a production of pantothenate lower than the parental strain. Conversely, increasing the stability of the UvBNC-leuABCD mRNA transcripts by the cggR-gapA stabilizing element led to pantothenate titers higher than that produced by the PA73 parental strain. Since the protein content was shown to be almost identical in<br>
both PA444 and PA445, increasing the stability of the ilvBNC-leuABL.V transcripts using the cggR-gapA stabilizing was more effective in enhancing pantothenate production than increasing the level of transcription by use of the constitutive SPOl promoter. In good agreement with this conclusion, strain PA444 grew slower than PA445 and had a higher rate of cell lysis during prolonged growth.<br>
Example VII<br>
Stabilization of mRNA is independent of RNase E activity in B. sub tills.<br>
Strain B. subtilis 168 has been described to contain two genes (ymfA and ykqC) encoding supposedly functional homologues of E. coli RNase E. Strain SSB348 is a derivative of wild type B. subtilis 168 in which gene ymfA is deleted and ykqC expression is placed under the control of an IPTG (isopropylgalactopiranoside)-inducible promoter (Pspac) (Even et al„ 2005, Nucleic Acids Res 33:2141). Strain SSB348 was transformed with DNA extracted from strains PA431, PA432, PA494, and PAS 17 to yield strains PA602 (UvB-lacZ fusion expressed under control of native UvB promoter signals), PA603 (ilvB-lacZ fusion expressed as stabilized transcript by addition of the cggR-gapA mRNA stabilizing element at the 5' end expressed under control of native UvB promoter signals), PA604 (UvB-lacZ fusion expressed as stabilized transcript by addition of the cggR-gapA mRNA stabilizing element at the 5' end expressed under control of native UvB promoter signals) and PA605 (UvD-lacZ fusion transcribed under control of native promoter signals ilvD gene as stabilized transcript by incorporating the hrcA-grpE stabilizing element at the 5' end), respectively. Each strain was grown in 25 ml shake flask cultures for 18 hours at 37°C, and P-galactosidase levels were measured using standard (ONPG assay) methods; these results are summarized in Table 6. In each strain, no difference in p-galactosidase levels was observed when cells were grown in the presence or absence of IPTG induction. These results demonstrate that the activities encoded by the genes ymfA and ykqC are not required for the function of mRNA stabilizing elements in B. subtilis.<br>
Table 6. ONPG assay results obtained irom experiments ueveiopeo  with ams PA602, PA603, PA604 and PA605.<br>
(Table Removed)<br>
Example YTII<br>
The effects of the secondary structure of the stabilizing elements and the length of unpaired 5' sequence on the enhanced protein synthesis mediated by mRNA stabilization.<br>
Sharp and Bechhofer (2005, Molecular Microbiology 57:484-495) demonstrated that for the efficient mRNA stabilization in B. subtilis (i) the 5'-terminal secondary structure has to have a free energy (AG) under a minimal value (-2.8 and -4.7 kcal/mol) and (ii) located Jess than 4 and 7 unpaired nucleotides (nts) from the 5'-end of the transcript. The presence of four unpaired nucleotides (at the ermC stabilizing element) and seven unpaired nucleotides (at the SP82 stabilizing element) resulted in a complete loss of stabilization.<br>
In Example III it was shown that PA432 {PilvΩcsgR-gaPA-locZ) containing the cggR-gapA mRNA stabilizing element upstream of the lacZ reporter gene produced 7-fold more P-galactosidase than the isogenic strain PA431 containing only the wild type ilvB promoter lacZ fusion (PilvB-tacZ). In PA432 the two stabilizing stem-loop structure of the cggR-gapA mRNA stabilizing element was located 93 nucleotides downstream of the +1<br>
transcriptional start site of the IacZ transcript. lo predict trie secondary siruciuie ui mc cggR-gapA stabilizing element and to determine the length of unpaired sequence at the 5'-end of the transcript, the leader region of the IacZ mRNA in PA432 was analyzed using the mfold structure prediction program of Zuker (2003, Nucleic Acids Research 31:3406-3415). The 260-nt leader region (from +1 transcriptional start until, but not including, the spoVC RBS sequence for the translation of IacZ) contained 8 unpaired nucleotides at the 5'-end of the transcript followed by a very weak stem-loop (AG= -0.5 kcal/mol). The first stabilizing stem-loop (SL1) of the cggR-gapA stabilizing element (predicted structure in Ludwig et al., 2001, Molecular Microbiology 41:409-422) was truncated in PA432. The truncated SL1 (SLIT) had features of AG= -4.8 kcal/mol; 8 nts in the double-stranded stem; 1 nt bulge loop; and 3 nts hairpin loop.<br>
Using the mfold program it was predicted that the SL1 stabilizing stem-loop (AG= -11.6 kcal/mol; 11 nts in the double-stranded stem; 2 nts bulge loop; and 3 nts hairpin loop) could be restored by removing 75 nts (+19 nt to +93 nt) from the 277-nt leader region of the IacZ mRNA in PA432 (SEQ ID No. 47). To test whether SL1 confers higher levels of mRNA stabilization than SLIT, a leader deletion derivative, called cggR-gapA§10 was constructed upstream of IacZ. First, 130-bp PCR product containing the cggR-gapA stabilizing element was amplified with primers GAP#10-FOR and GAP#10-REV (Table 7) using pPA422 as a template DNA. The PCR product was digested with PshAl and Nhel enzymes and cloned into the PshAl and Nhel sites in pPA415 plasmid. After transformation of E. coli TOP 10, Ampr and Specr transformants were selected. The resulting pBestlO was next transformed into B. subtilis 1A747 selecting for Specr to integrate the PilvΩcggR-gapA#10-lacZ fusion into the amyE chromosomal locus, generating strain BE 10. In shake flask assays cultures were grown in both LB medium and minimal medium, and the BE10 (PnvcicggR-gapA#}0-lacZ) strain produced similar levels of p1-galactosidase as the isogenic strain PA432 (PilvΩcggR-gapA-locZ). The data indicated that efficient mRNA stabilization mediated by the cggR-gapA element was independent on (i) the distance from 5'-terminus to the stabilizing sequence (19 nts vs. 93 nts); and (ii) the strength of the secondary structure (SLIT vs. SL1 stabilizing stem-loop).<br>
To study the effect of unpaired 5'-terminal nucleotides 39 nts (+180 nt to +218 nt) was removed from the 277-nt leader region of the IacZ mRNA in PA432 (SEQ ID No. 47) and the structure of the 238-nt leader deletion derivative, called cggR-gapA#\4, was predicted using the mfold program. The 277-nt original leader sequence contained 8 unpaired nucleotides, and the cggR-gapAU\A deletion leader contained 16 unpaired nucleotides at the 5'-end of the transcript. To construct a B. subtilis mutant carrying the cggR-gapA#\4 deletion leader, first a 39-bp fragment was removed with BamHl digestion between the stabilizing element and the IacZ gene in the pPA422 plasmid. After self-ligation of the<br>
digested plasmid and transformation of E. coli TOP 10, Amp" and Spec transrormanxs were selected. The resulting pBestl4 was next transformed into B. subtilis 1A747 selecting for Specr to integrate the PilvΩcggR-gapA#14-locZ fusion into the amyE chromosomal locus, generating strain BE] 4. Bacterial cultures were grown in both LB medium and minimal medium in shake flask assays and the BE 14 (PthncggR-gapAni-t-iacZ) strain produced similar levels of (3-galactosidase as the isogenic strain PA432 {PilvtΩcggR-gapA-lacZ). The data indicated that efficient mRNA stabilization was achieved when 16 unpaired nucleotides were at the 5'-end of the lacZ transcript in the BE 14 strain.<br>
Table 7. List of primers used to study the effect of the secondary structures of the cggR-gapA stabilizing element and the length of unpaired 51 sequence.<br>
Primer	ID No.     Sequence	<br>
(Sequence Removed)<br><br><br><br><br><br><br>
WE CLAIM :	<br>
1.	A method to increase the production of a desired chemical compound in a microorganism without strengthening native promoter signals controlling transcription of said structural gene sequences by the mtroduction of a DNA sequence at the 5' end of the encoding DNA gene sequence capable of forming a stem loop and capable of increasing the stability of mRNA transcripts from one or more genes, a method which is characterized in that the loop-forming DNA sequence is introduced seven or more DNA nucleotides down-stream of the start site of transcription of the relevant gene(s) of the microorganism.<br>
2.	A method according to claim 1 wherein the stem loop-forming DNA sequence is naturally occurring.<br>
3.	A method of claim 1 or claim 2 wherein said DNA sequence consists of a minimum of 15 nucleotides forming one or more stem loop secondary structures with the following properties:<br>
a double-stranded stem structure of at least 6 base pairs and a hairpin loop structure of 3 - 30 nucleotides and -    a calculated thermodynamic stability (AG) of said structure of-2.8 kcl/mol or lower.<br>
4.	A method of claim 2 or claim 3, wherein the loop-forming DNA sequence is derived from the chromosome of a microorganism.<br>
5.	A method of any one of claims 1-4, wherein the microorganism is of the genus Bacillus.<br>
6.	A. method of any one of claims 1-5, wherein the microorganism is of the species Bacillus subtilis.<br>
7.	The method of any one of claims 1-6, wherein the DNA sequence is defined to be a chromosomal nucleotide region of Bacillus subtilis consisting of a contiguous sequence occurring in a member of the group consisting of sequences from gene cggR to gene gapA, from gene hrcA to gene grpE, from gene ilvN to gene ilvC, from gene aprE to gene yhfO, from gene ybdA to gene gsiB and from gene ytxC to gene thrS .<br>
8.	The method of any one of claims 1 - 7, wherein the DNA sequence is a sequence occurring naturally from gene cggR to gene gapA, preferably the 3'-end of gene cggR and 10 more nucleotides downstream of the stop codon of the cggR gene, of Bacillus subtilis, represented by SEQ ID No. 1 or a part thereof.<br>
9.	The method of claim 8, wherein the DNA sequence is SEQ ID No. 2.<br>
10.	The method of any one of claims 1 - 7, wherein the DNA sequence is a sequence occurring naturally from gene hrcA to gene grpE, preferably between those genes, of Bacillus subtilis, represented by SEQ ID No. 3.<br>
11.	The method of claim 10, wherein the DNA sequence is ID No. 4.<br>
12.	The method of any one of claims 1-7 wherein the DNA sequence is SEQ ED No. 5 or SEQ ID No. 6.<br>
13.	The use of stabilized mRNA sequences to increase the production of a desired chemical compound in a microorganism, said stabilized mRNA sequences containing a stabilizing element at their 5' end, which stabilizing element is transcribed from a DNA sequence introduced seven or more DNA nucleotides down-stream of the start of transcription of the relevant gene of the microorganism.<br>
14.	The use of a DNA sequence which upon transcription by a microorganism form the stabilized mRNA sequences of claim 13, said DNA sequence being used for increasing the stability of mRNA transcripts from one or more genes in a method as defined in any one of claims 1 - 12.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUFic3RyYWN0LSgwNS0wOS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Abstract-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUFzc2lnbm1lbnQtKDIxLTA4LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Assignment-(21-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUNsYWltcy0oMDUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Claims-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Correspondence Others-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTktMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Correspondence Others-(19-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDgtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Correspondence Others-(21-08-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Correspondence Others-(25-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1ERUxOUC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMjQtMTEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4908-DELNP-2008-Correspondence-Others (24-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDA1LTA5LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Description (Complete)-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWZvcm0tMTMtZS0xOC0oaSktODI0LTIwMDgtZGVsLnBkZg==" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-form-13-e-18-(i)-824-2008-del.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1ERUxOUC0yMDA4LUZvcm0tMTggKDI0LTExLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4908-DELNP-2008-Form-18 (24-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUZvcm0tMi0oMDUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Form-2-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUZvcm0tMy0oMDUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Form-3-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUZvcm0tMy0oMTktMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Form-3-(19-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUZvcm0tMy0oMjUtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Form-3-(25-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LUdQQS0oMDUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-GPA-(05-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDkwOC1kZWxucC0yMDA4LVBldGl0aW9uLTEzNy0oMDUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4908-delnp-2008-Petition-137-(05-09-2014).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263380-non-nucleoside-reverse-transcriptase-inhibitors.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263382-a-machine-for-continuous-deskinning-of-sugarcanes.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263381</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4908/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Jun-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, NL-6411 TE HEERLEN, THE NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FERRANDEZ, ABEL</td>
											<td>RIXHEIMERSTRASSE 17, CH-4055 BASEL, SWITZERLAND.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PERKINS, JOHN B.</td>
											<td>UAN BOETZELAERLAAN 4, NL-2242 SW WASSENAAR, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SCHABER, MICHELE</td>
											<td>SCHAUENBURGERSTRASSE 21, CH-4052 BASEL, SWITZERLAND</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/67</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/011547</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-12-01</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>05026475.3</td>
									<td>2005-12-05</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263381-a-method-to-increase-the-production-of-a-desired-chemical-compound-in-a-microorganism by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:07:15 GMT -->
</html>
