<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263307-a-recombinant-negative-strand-rna-virus-containing-a-viral-genome-with-a-mutation-in-the-p-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:29:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263307:A RECOMBINANT NEGATIVE-STRAND RNA VIRUS CONTAINING A VIRAL GENOME WITH A MUTATION IN THE P-GENE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT NEGATIVE-STRAND RNA VIRUS CONTAINING A VIRAL GENOME WITH A MUTATION IN THE P-GENE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a genome-replication- deficient and transcription-competent negative-strand RNA virus, which can be used for the expression of transgenes and in particular for the area of vaccine development.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Replication-deficient RNA viruses as vaccines<br>
Description<br>
The present invention relates to a replication-<br>
defective and transcription-competent negative-strand<br>
RNA virus, which can be used for the expression of<br>
transgenes and in particular for the area of vaccine<br>
development.<br>
Immunizations with live vaccines imitate natural<br>
infection and produce a comprehensive immune response.<br>
Attenuated, but still viable, viruses are used for<br>
vaccination. Multiplication of the vaccine viruses must<br>
take place so slowly that an immunological response and<br>
therefore control of multiplication and/or elimination<br>
of the virus is ensured. The live vaccine concept has<br>
frequently proved itself in various age groups. There<br>
are, however, important target groups for whom<br>
immunization with live vaccines is problematic and<br>
extra safety measures are required: maternal antibodies<br>
protect infants in the first few months of life. At the<br>
same time they represent a barrier that must be<br>
overcome in immunization with live vaccine, though<br>
without leading to excessive multiplication of the<br>
vaccine virus and associated vaccination lesions.<br>
Another target group are the elderly, whose immune<br>
system is no longer so efficient, so that it can be<br>
overloaded by vaccination, and increased multiplication<br>
of the vaccine virus may lead to vaccination lesions.<br>
There is therefore the problem of making the<br>
immunologically excellent live vaccination even safer<br>
for application in certain target groups, as well as<br>
increasing the safety profile for general use.<br>
For some years it has been possible to alter negative-<br>
strand RNA viruses, such as the rabies virus or the<br>
Sendai virus (SeV) for example, purposefully by reverse<br>
genetic engineering. EP-A-0 702 085 describes the<br><br>
- 2 -<br>
production of recombinant, infectious, replicating<br>
unsegmented negative-strand RNA viruses from cloned<br>
cDNA. EP-A-0 863 202 describes a recombinant Sendai<br>
virus, in whose genome a heterologous gene is inserted<br>
or a gene is deleted or inactivated, but whose genome<br>
replication is still intact.<br>
Negative-strand RNA viruses are especially suitable as<br>
the backbone of vaccines, as their multiplication in<br>
the cytoplasm takes place at the RNA level and genes<br>
within the viral genome can simply be exchanged. Thus,<br>
there has already been success in producing recombinant<br>
viruses with surface proteins of various virus types<br>
and using them as vaccines in animal experiments<br>
(Schmidt et al., J. Virol. 75 (2001), 4594-4603 and WO<br>
01/42445). By recombinant insertion of F- and HN-<br>
proteins of human parainfluenza virus type 3 (hPIV 3)<br>
and of the G- or F-protein of Respiratory Syncytial<br>
Virus (RSV) in a vector based on bovine parainfluenza<br>
virus type 3 (bPIV 3) , a mucosal immune response to<br>
hPIV 3 and RSV was detected after application in<br>
hamsters. A bivalent antigenicity of this live vaccine,<br>
which has been tested in animal experiments, has thus<br>
already been achieved.<br>
Owing to the involvement of the species barrier, this<br>
bovine parainfluenza virus with human PIV 3 and RSV<br>
surface antigens should already be sufficiently<br>
attenuated for application in humans. Reversions to the<br>
wild type should not be expected, as complete genes<br>
were exchanged for viral surface proteins. Clinical<br>
testing of the vaccine has already begun.<br>
As the virus mutants described are, however,<br>
replication-competent, virus multiplication will<br>
undoubtedly occur in the vaccinee, the intensity of<br>
which is attenuated by modification of the virus, but<br>
is not excluded completely. The intensity of the<br>
viremia that is to be expected and therefore the side-<br><br>
- 3 -<br>
effects suffered by the vaccinee then depend on<br>
individual factors.<br>
Within the scope of the present invention, an attempt<br>
is to be made to substantially reduce the risks of live<br>
vaccination, and especially the risks for vaccinees in<br>
certain target groups.<br>
One approach is the suppression of viral genome<br>
replication after application of the vaccine. As a<br>
result, multiplication of the virus and corresponding<br>
vaccination lesions will not occur, regardless of the<br>
vaccinee's state of immunity.<br>
A fundamental difficulty in this approach is that the<br>
viral RNA polymerase performs two functions: synthesis<br>
of viral mRNA and multiplication of the viral genomes.<br>
This coupling must be removed in the new vaccine, as<br>
the vaccine must now only be capable of synthesis of<br>
viral mRNA.<br>
Another problem is that the recombinant virus must<br>
perform efficient synthesis of viral mRNA, if it is to<br>
be suitable at all as live vaccine. There are thus two<br>
basically contradictory requirements, which mean that<br>
considerable difficulties are to be expected in the<br>
production of safe, but efficient live vaccines based<br>
on negative-strand RNA viruses.<br>
Shoji et al. (Virology 318 (2004), 295-305) describe<br>
the production and characterization of a P gene-<br>
deficient rabies virus. The virus was produced by means<br>
of P-protein-expressing helper cells. Without de novo<br>
synthesis of P-protein, the viruses are only capable of<br>
primary transcription. The slight viral gene expression<br>
is manifested in a very weak signal for N-protein in<br>
immunofluorescence and only in very few cells, and<br>
convincing proof of this slight viral gene expression<br>
will only be provided by PCR analysis. Use of this<br><br>
- 4 -<br>
mutant virus in a challenge test in the mouse model<br>
should show protection, but there is no control<br>
experiment with transcription-inactive virus and the<br>
time interval for the viral challenge is too short. The<br>
duration of supposed protection is not being<br>
investigated. The use of such mutant viruses for the<br>
development of an attenuated rabies vaccine therefore<br>
seems not to be very promising.<br>
Within the scope of the investigations that led to the<br>
present invention, it was found that in<br>
paramyxoviruses, decoupling of the replication and<br>
transcription functions can be achieved by partially<br>
removing the constituents of polymerase that are<br>
essential for the genome replication function. This may<br>
involve one of the viral proteins N, P and L, or a<br>
special functional domain of such a protein.<br>
Surprisingly it	was found that by mutations in which<br>
the function of	the proteins encoded by the viral genes<br>
N, L and/or P is not deleted completely, but partially,<br>
it is possible	to produce replication-defective RNA<br>
viruses which	possess an adequate transcription<br>
function to be	suitable for the production of live<br>
vaccines.<br>
One object of the present invention is thus a<br>
recombinant negative-strand RNA virus, which is<br>
replication-deficient and transcription-competent. The<br>
virus according to the invention contains a viral<br>
genome with a mutation in at least one of the genes N,<br>
L and P, with the mutation leading to loss of genome<br>
replication without loss of secondary transcription<br>
capacity.<br>
The virus according to the invention is a prerequisite<br>
for the production of live vaccines, especially for the<br>
production of live vaccines with an enhanced safety<br>
profile, which is especially suitable for use in high-<br><br>
- 5 -<br>
risk patients with a weak or damaged immune system.<br>
The invention also relates to a nucleocapsid of the<br>
recombinant virus, comprising the viral negative-strand<br>
RNA, complexed with the proteins N, L and P plus the<br>
negative-strand RNA of the recombinant virus in<br>
isolated form.<br>
The invention also relates to a cDNA, which codes for a<br>
negative-strand RNA according to the invention, in<br>
particular a viral RNA and/or an RNA complementary to<br>
it.<br>
The invention further relates to a cell line for<br>
multiplication of the recombinant negative-strand RNA<br>
virus according to the invention.<br>
The recombinant negative-strand RNA virus according to<br>
the invention can be obtained by mutation of a starting<br>
virus in at least one of the genes N, L and P. The<br>
starting virus can be a natural negative-strand RNA<br>
virus, especially from the families Paramyxoviridae or<br>
Rhabdoviridae or recombinant variants thereof.<br>
Especially preferred representatives are<br>
paramyxoviruses, e.g. Sendai virus, human or bovine<br>
parainfluenza virus, e.g. human parainfluenza virus<br>
(hPIV) type 1, 2, 3, 4a or 4b, Newcastle disease virus,<br>
mumps virus, measles virus or human respiratory<br>
syncytial virus (hRSV) or rhabdoviruses, e.g. vesicular<br>
stomatitis virus (VSV). Especially preferably, the<br>
virus is a Sendai virus, e.g. of the Fushimi strain<br>
(ATCC VR105). Recombinant variants of the<br>
aforementioned viruses, as described for example in EP-<br>
A-702 085, EP-A-0 863 202 or WO 01/42445, are also<br>
covered by the invention.<br>
Further preferred negative-strand RNA viruses are<br>
representatives of the Rhabdoviridae, Filoviridae,<br>
Bornaviridae, Arenaviridae or Bunyaviridae, e.g. VSV.<br><br>
- 6 -<br>
Like other paramyxoviruses, the Sendai virus is an<br>
enveloped virus with a helical nucleocapsid (Fig. 1) .<br>
The envelope consists of a lipid membrane, which is<br>
derived from the plasma membrane of the host cell from<br>
which the virus was released. Transmembrane<br>
glycoproteins, namely the fusion protein (F) and<br>
hemagglutinin-neuramidase (HN), are anchored in the<br>
viral envelope. The matrix protein (M) lines the inside<br>
of the membrane. The nucleocapsid contained in the<br>
envelope consists of single-stranded RNA complexed with<br>
nucleoprotein (N) , with in each case 6 nucleotides of<br>
the RNA bound by one N protein, an RNA-dependent RNA<br>
polymerase (L) and the co-factor phosphoprotein (P).<br>
The negative-strand RNA genome of the Sendai virus<br>
contains the genes of the 6 structural proteins in the<br>
order: 3'-N-P/C-M-F-HN-L-5' (Fig. 2). The P/C gene<br>
codes for a total of 8 proteins, the structural<br>
phosphoprotein and all non-structural proteins known to<br>
date.<br>
The proteins P, N and L are important for functional<br>
transcription and replication (Lamb et al. ,<br>
Paramyxoviridae: The Viruses and their Replication.<br>
Fields Virology, 4th edition (2001) , Lippincott,<br>
Williams &amp; Wilkins, Philadelphia, 1305-1340).<br>
The recombinant negative-strand RNA virus according to<br>
the invention contains a mutation in at least one of<br>
the genes N, L and P. The mutation can be a deletion,<br>
substitution and/or insertion in one of the genes N, L<br>
or P, which gives rise to a replication deficiency of<br>
the virus, but does not disturb the capacity for<br>
transcription. The mutation preferably affects a<br>
partial sequence of the proteins encoded by the genes<br>
N, L and/or P, which is necessary for replication,<br>
whereas other partial sequences necessary for<br>
transcription remain functional.<br><br>
- 7 -<br>
In a preferred embodiment of the invention, the<br>
recombinant virus has a mutation in gene P, namely in<br>
an N-terminal partial sequence of gene P. The mutation<br>
preferably affects at least the region of amino acids<br>
33-41 of the protein P, which are important for the<br>
capacity for replication. It is further preferred that<br>
the C-terminal region (starting from amino acid 320)<br>
does not have any mutations impairing the transcription<br>
function. Especially preferably, the mutation is a<br>
mutation in the region of amino acids 2-77 leading to<br>
loss of capacity for replication, for example a<br>
deletion of (a) the amino acids 2-77 of the protein<br>
encoded by gene P or (b) a partial sequence of (a)<br>
sufficient for loss of the capacity for replication.<br>
Corresponding mutations can also take place in P<br>
proteins of other negative-strand RNA viruses, e.g. of<br>
other paramyxoviruses, e.g. hPIV3.<br>
The recombinant virus according to the invention is<br>
replication-deficient and transcription-competent. Loss<br>
of the capacity for replication means that in a target<br>
cell (a cell which does not produce in trans any of the<br>
functions deleted by mutation) no detectable virus<br>
genome multiplication is found, and in contrast to a<br>
reduced or conditional replication deficiency, also no<br>
permissive conditions exist, in which replication can<br>
occur. The loss of the capacity for replication can be<br>
determined as described in Example 8. However, the<br>
virus according to the invention is capable of<br>
transcribing the gene products encoded by it after<br>
infection in a target cell, so that expression of the<br>
viral proteins including one or more heterologous gene<br>
products can take place in the target cell. It is<br>
important that the recombinant virus according to the<br>
invention should possess the capacity for secondary<br>
transcription, i.e. the viral gene products that arise<br>
through primary transcription with the protein<br>
components originally contained in the nucleocapsid are<br>
capable of bringing about and/or supporting a secondary<br><br>
- 8 -<br>
transcription themselves. The extent of the secondary<br>
transcription then leads to protein synthesis of<br>
preferably at least 1%, at least 2%, at least 3%, at<br>
least 4% or at least 5% relative to a corresponding<br>
wild-type virus, i.e. a virus without the mutation in<br>
at least one of the genes N, L and P. The capacity for<br>
secondary transcription can be reduced relative to the<br>
corresponding wild-type virus, though preferably at<br>
most by a factor of 20, especially preferably at most<br>
by a factor of 10. The capacity for secondary<br>
transcription can be determined as in Example 7.1<br>
and/or 7.3 by quantitative determination of the<br>
expression of a heterologous gene product, e.g. a<br>
reporter protein.<br>
Besides the mutation, the recombinant virus according<br>
to the invention preferably contains at least one<br>
transgene, i.e. at least one sequence coding for a<br>
heterologous gene product. The heterologous gene<br>
product can be a protein, for example a reporter<br>
protein, e.g. a fluorescence protein such as GFP or a<br>
derivative thereof, or an antigen, against which an<br>
immune response is to be produced, or a therapeutic<br>
protein, e.g. a protein for virotherapy or a functional<br>
RNA molecule, e.g. an antisense RNA, a ribozyme or an<br>
siRNA molecule capable of RNA interference. Preferably<br>
the heterologous gene product is an antigen,<br>
originating from a pathogen, such as a virus, a<br>
bacterium, a fungus or a protozoon, a tumor antigen or<br>
an autoantigen. Especially preferably, the antigen is a<br>
viral antigen, derived from a heterologous negative-<br>
strand RNA virus, such as a human parainfluenza virus<br>
or RSV, e.g. hPIV3 F and HN or hRSV F and G. The virus<br>
according to the invention can contain one or more,<br>
e.g. two or three, sequences coding for a heterologous<br>
gene product.<br>
Sequences coding for heterologous gene products can be<br>
inserted in the genome of the recombinant virus. On the<br><br>
- 9 -<br>
other hand, sequences coding for homologous gene<br>
products, e.g. genes F and/or HN, can also be<br>
substituted with sequences that code for heterologous<br>
gene products, e.g. chimeric gene products.<br>
Combinations of inserted and substituted transgenes are<br>
also possible.<br>
For example, sequences of a Sendai virus can be<br>
replaced completely or partially with heterologous<br>
sequences of other negative-strand RNA viruses, e.g.<br>
with sequences of parainfluenza viruses, e.g. hPIV3,<br>
and/or with sequences of RSV. Use of chimeric sequences<br>
is especially preferred, i.e. sequences comprising<br>
segments of the base virus genome and segments of a<br>
heterologous virus genome. For example, chimeric genes<br>
F and/or HN can be inserted in the virus genome, which<br>
comprise sequences of the base virus genome, e.g.<br>
Sendai virus and heterologous sequences, e.g. from<br>
human parainfluenza viruses such as hPIV3, and/or RSV.<br>
The recombinant virus can contain one or more different<br>
transgenes. If several transgenes are present, these<br>
can be of the same or of different origin, which can be<br>
derived for example from a single or from several<br>
different pathogens, e.g. viruses. Thus, transgenes<br>
from several, e.g. 2 or 3 different pathogens,<br>
preferably viruses, and especially negative-strand RNA<br>
viruses, can be present.<br>
The incorporation of transgenes in paramyxoviruses is<br>
described for example in Hasan et al. (J. Gen. Virol.<br>
78 (1997), 2813-2820), Bukreyev et al. , (J. Virol. 70<br>
(1996), 6634-6641), Yu et al. (Genes Cells 2 (1997),<br>
457-466), Masaki et al. (FASEBVB J. 15 (2001), 1294-<br>
1296) , Shiotani et al. (Gene Therapy 8 (2001) , 1043-<br>
1050) and Bitzer et al. (Mol. Therapy 7 (2003), 210-<br>
217). Preferably the transgenes are inserted in the 3'<br>
'region of the viral genome. Insertion is effected for<br>
example in the form of transcription cassettes, with<br><br>
- 10 -<br>
one or more transcription cassettes with singular<br>
restriction sites for integration of the respective<br>
reading frames inserted directly after the leader<br>
region at the vector level (i.e. at the level of the<br>
vector, e.g. of a plasmid vector, which codes for the<br>
negative-strand RNA). Integration of several transgenes<br>
is preferably effected in independent transcription<br>
cassettes in each case. A transcription cassette<br>
preferably contains the sequence coding for the<br>
heterologous gene product in operational linkage with a<br>
transcription start sequence and a transcription<br>
termination sequence and preferably translation<br>
signals.<br>
A. further object of the present invention is a single-<br>
stranded or double-stranded DNA molecule, e.g. a cDNA<br>
molecule, which codes for a recombinant negative-strand<br>
RNA virus genome according to the invention or a<br>
precursor thereof or the virus-antigenome or a<br>
precursor thereof. The term "precursor" means in this<br>
context that the DNA molecule does not yet contain a<br>
sequence coding for a heterologous gene product, but<br>
only a cloning site for insertion of such a sequence.<br>
The cloning site can be a restriction site, for example<br>
a singular or non-singular restriction site in the DNA<br>
or a multiple cloning site, containing several<br>
consecutive restriction sites, preferably singular<br>
restriction sites. The DNA molecule coding for the<br>
virus genome and/or the complementary sequence is<br>
preferably in operational linkage with suitable<br>
sxpression control sequences.<br>
rhe DNA molecule is preferably a vector, for example a<br>
plasmid vector, which is suitable for propagation in a<br>
suitable host cell, i.e. in a vector or plasmid<br>
amplification cell, preferably in a prokaryotic cell,<br>
out also in a eukaryotic cell, especially in a<br>
nammalian cell, and has the necessary genetic elements<br>
for this, such as replication origin, integration<br><br>
- 11 -<br>
sequences and/or selectable marker sequences.<br>
The DNA molecule contains the sequence coding for the<br>
recombinant virus or the complementary sequence,<br>
preferably under the control of a transcription signal,<br>
so that during transcription with a DNA-dependent RNA<br>
polymerase in a host cell suitable for the initial<br>
production of the virus, i.e. in a virus production<br>
cell, the viral negative-strand RNA can be formed. The<br>
transcription signal is selected to permit efficient<br>
transcription of the DNA in the host cell used in each<br>
case. It is also possible to use a heterologous<br>
transcription signal for the particular cell, e.g. a<br>
bacteriophage promoter, such as the T7 or SP6 promoter,<br>
and then the virus production cell must also contain a<br>
corresponding heterologous DNA-dependent RNA<br>
polymerase, e.g. T7 or SP6 RNA polymerase, which<br>
effects the transcription. In addition to the<br>
transcription signal, the DNA molecule further<br>
contains, preferably at the 3' end of the sequence<br>
coding for the recombinant virus, a transcription<br>
terminator and a ribozyme sequence, which permits<br>
cleavage of the transcript at the end of the viral<br>
sequence. The virus production cell is preferably a<br>
eukaryotic cell and especially a mammalian cell.<br>
In addition to the DNA coding for the replication-<br>
deficient paramyxovirus, the virus production cell<br>
according to the invention also contains helper<br>
sequences, whose gene products permit assembly of the<br>
recombinant virus RNA according to the invention in<br>
trans. For this, the cell can for example additionally<br>
contain one or more vectors which produce the N<br>
protein, the P protein and/or the L protein in trans.<br>
This makes assembly of nucleocapsids of the recombinant<br>
virus according to the invention possible in the<br>
production cell.<br>
Multiplication of the recombinant virus initially<br><br>
- 12 -<br>
assembled in the virus production cell takes place in a<br>
virus multiplication cell, which is infected with the<br>
virus according to the invention. In addition the virus<br>
multiplication cell contains helper sequences as<br>
mentioned above, for production of the N protein, the P<br>
protein and/or the L protein in trans. Preferably a<br>
virus multiplication cell is used in which there is<br>
stable expression of the helper sequences, e.g. by<br>
genomic integration. The virus multiplication cell is<br>
preferably a mammalian cell. An especially preferred<br>
multiplication cell is cell H29, derived from a 293<br>
cell, of a human embryonic renal fibroblast cell line,<br>
or a cell derived from that. Cell H29 was deposited on<br>
11.05.2004 (DSM ACC 2702) in accordance with the<br>
provisions of the Budapest Treaty with the Deutsche<br>
Sammlung fur Mikroorganismen und Zellkulturen GmbH,<br>
Braunschweig, Mascheroder Weg. Vero cells, of a renal<br>
cell line from the African green monkey, or cells<br>
derived from LLCMK2 cells, of a renal cell line from<br>
the rhesus monkey, which have been stably transfected<br>
with corresponding helper sequences, e.g. SeV N and P<br>
genes, are also suitable.<br>
The invention therefore further relates to a cell,<br>
preferably a eukaryotic cell and especially preferably<br>
a mammalian cell, which contains (i) a DNA molecule,<br>
which codes for the genome of the recombinant virus<br>
according to the invention and/or the complementary<br>
sequence thereof or a precursor thereof, and/or (ii) an<br>
RNA genome of the virus according to the invention. The<br>
cell can be a vector multiplication cell, a virus<br>
production cell or a virus multiplication cell, as<br>
explained previously.<br>
If the cell is a vector multiplication cell, e.g. a<br>
plasmid multiplication cell, any cell that is suitable<br>
for multiplication of the corresponding vector can be<br>
used, e.g. also a prokaryotic cell such as a<br>
transformed E. coli cell.<br><br>
- 13 -<br>
If the cell is a virus production or multiplication<br>
cell, it contains helper sequences for production of<br>
the virus proteins N, P and/or L in trans. The DNA<br>
inserted in a virus production cell is preferably under<br>
the control of a heterologous transcription signal, and<br>
advantageously the cell further contains a DNA that<br>
codes for a heterologous DNA-dependent RNA polymerase,<br>
which recognizes the heterologous transcription signal<br>
and effects transcription of the DNA coding for the<br>
recombinant negative-strand RNA virus.<br>
If the cell is a virus multiplication cell,	it is<br>
infected with a genomic viral RNA molecule, e.g.	in the<br>
form of a nucleocapsid, and contains the	helper<br>
sequences in stably expressible form.<br>
The present invention further relates to a method of<br>
production of a recombinant negative-strand RNA virus<br>
according to the invention comprising the steps: (a)<br>
preparation of a virus production cell, which is<br>
transfected with a DNA molecule that codes for the<br>
genome of a negative-strand RNA virus, containing a<br>
mutation in at least one of the genes N, L and P, which<br>
leads to loss of the capacity for genome replication<br>
without loss of the capacity for transcription, and<br>
optionally at least one sequence coding for a<br>
heterologous gene product, and (b) cultivation of the<br>
host cell in conditions such that transcription of the<br>
DNA molecule according to (a) takes place and the<br>
recombinant negative-strand RNA virus is formed<br>
initially. The host cell is preferably capable of<br>
producing the N protein, the P protein and/or the L<br>
protein in trans, e.g. by transfection with the<br>
corresponding helper plasmids which contain sequences<br>
coding for the proteins N, P and/or L.<br>
For the production of large quantities of the<br>
nucleocapsids or of the virus particles, preferably a<br>
cell is used which stably expresses, constitutively or<br><br>
- 13 -<br>
If the cell is a virus production, or multiplication<br>
cell, it contains helper sequences for production of<br>
the virus proteins N, P and/or L in trans. The DNA<br>
inserted in a virus production cell is preferably under<br>
the control of a heterologous transcription signal, and<br>
advantageously the cell further contains a DNA that<br>
codes for a heterologous DNA-dependent RNA polymerase,<br>
which recognizes the heterologous transcription signal<br>
and effects transcription of the DNA coding for the<br>
recombinant negative-strand RNA virus.<br>
If the cell is a virus multiplication cell,	it is<br>
infected with a genomic viral RNA molecule, e.g.	in the<br>
form of a nucleocapsid, and contains the	helper<br>
sequences in stably expressible form.<br>
The present invention further relates to a method of<br>
production of a recombinant negative-strand RNA virus<br>
according to the invention comprising the steps: (a)<br>
preparation of a virus production cell, which is<br>
transfected with a DNA molecule that codes for the<br>
genome of a negative-strand RNA virus, containing a<br>
mutation in at least one of the genes N, L and P, which<br>
leads to loss of the capacity for genome replication<br>
without loss of the capacity for transcription, and<br>
optionally at least one sequence coding for a<br>
heterologous gene product, and (b) cultivation of the<br>
host cell in conditions such that transcription of the<br>
DNA molecule according to (a) takes place and the<br>
recombinant negative-strand RNA virus is formed<br>
initially. The host cell is preferably capable of<br>
producing the N protein, the P protein and/or the L<br>
protein in trans, e.g. by transfection with the<br>
corresponding helper plasmids which contain sequences<br>
coding for the proteins N, P and/or L.<br>
For the production of large quantities of the<br>
nucleocapsids or of the virus particles, preferably a<br>
cell is used which stably expresses, constitutively or<br><br>
- 14 -<br>
inducibly, the proteins N, L and/or P, preferably at<br>
least protein P of a negative-strand RNA virus. The<br>
invention thus also relates to a method of<br>
multiplication of a recombinant negative-strand RNA<br>
virus according to the invention, comprising the steps:<br>
(a) preparation of a virus multiplication cell, which<br>
is infected with the genome of a negative-strand RNA<br>
virus, containing a mutation in at least one of the<br>
genes N, L and P, which leads to loss of the capacity<br>
for genome replication without loss of the capacity for<br>
transcription, and optionally at least one sequence<br>
coding for a heterologous gene product, and (b)<br>
cultivation of the host cell in conditions such that<br>
multiplication of the virus takes place.<br>
The present invention further relates to a<br>
pharmaceutical composition, which contains a<br>
recombinant replication-deficient and transcription-<br>
competent negative-strand RNA virus, as stated<br>
previously, or its nucleocapsid as active substance and<br>
optionally as pharmaceutically usual vehicles and/or<br>
excipients. The pharmaceutical composition is suitable<br>
for applications in human and veterinary medicine. It<br>
can be used in particular as vaccine or for antitumor<br>
therapy, in particular for application in high-risk<br>
patients, such as children	, the elderly and/or persons<br>
with a damaged or weak immune system. The<br>
pharmaceutical composition can contain the negative-<br>
strand RNA virus in its native viral envelope.<br>
Application as vaccine is especially preferred, e.g. as<br>
vaccine against pathogens such as viruses, bacteria or<br>
protozoa. When the recombinant virus contains a<br>
transgene or several transgenes of the same origin,<br>
e.g. from a single pathogen, it is a monovalent<br>
vaccine. When the recombinant virus contains transgenes<br>
of various origins, it can be used as a polyvalent<br>
vaccine, e.g. as bivalent or trivalent vaccine. For<br>
example, it is possible to produce a polyvalent vaccine<br><br>
	- 15 -<br>
against several pathogenic viruses, e.g. against<br>
several pathogenic negative-strand RNA viruses, such as<br>
human parainfluenza virus and RSV.<br>
A vaccine according to the invention is capable of<br>
triggering a humoral immune response, preferably the<br>
formation of neutralizing antibodies, and/or a T-cell<br>
immune response. Especially preferably, a humoral<br>
immune response and a T-cell immune response are<br>
triggered.<br>
The pharmaceutical composition can be in the form of a<br>
solution, a suspension, a lyophilizate or in any other<br>
suitable form. In addition to the active substance, the<br>
composition can contain agents for adjusting the pH<br>
value, buffers, agents for adjusting tonicity, wetting<br>
agents and the like, and adjuvants. It can be<br>
administered by the usual routes, e.g. oral, topical,<br>
nasal, pulmonary etc., in the form of aerosols,<br>
liquids, powders etc. The therapeutically effective<br>
dose of the virus is administered to the patient, and<br>
this dose depends on the particular application (e.g.<br>
virotherapy or vaccine), on the type of disease, the<br>
patient's weight and state of health, the manner of<br>
administration and the formulation etc. Usually 103 to<br>
107 virus particles, especially preferably about 104 to<br>
106 virus particles are administered per application.<br>
Optionally, several different virus particles can be<br>
administered together, e.g. in the case of combination<br>
vaccinations. Administration can be single or multiple,<br>
as required.<br>
Preferred fields of application are for example the<br>
prevention or treatment of respiratory viral diseases.<br>
The invention will be further explained with the<br>
following drawings and examples.<br>
Examples<br><br>
- 15 -<br>
1. General<br>
Fig. 1 shows the morphology of a Sendai virus (SeV)<br>
according to Fields (Virology. Lippincott, Williams and<br>
Wilkins (2001), 4th edition; modified). The genome<br>
comprises a single-stranded RNA, which has the proteins<br>
N, P and L in the form of a nucleocapsid. The<br>
nucleocapsid is surrounded by a membrane envelope, in<br>
which the proteins HN and F (each consisting of one F1<br>
and F2 subunit) are incorporated. Protein M is<br>
associated with the inside of the membrane, and is also<br>
bound to the nucleocapsid components at the same time.<br>
The single-stranded negatively oriented RNA genome of<br>
the wild-type Sendai virus comprises 15384 nucleotides.<br>
The genes of the 6 structural proteins are located<br>
thereon in the order 3'-N-P/C-M-F-HN-L-5' (Fig. 2).<br>
Between the genes there are transitions of 50-80<br>
nucleotides, each containing a highly conserved region<br>
of 22 nucleotides: the termination signal of the<br>
preceding gene, an intergenic sequence and the start<br>
signal for the next gene. A unit comprising start<br>
signal, open reading frame (ORF), optionally<br>
untranslated regions and termination signal is called a<br>
transcription cassette. Before the N gene there is a<br>
leader sequence (Id) 55 nucleotides long, which is<br>
transcribed, but not translated. The L gene is followed<br>
by a trailer sequence (tr) 54 nucleotides long. The Id<br>
and tr regions function as genomic and antigenomic<br>
promoters for the replication of genome or antigenome.<br>
With the exception of P/C-RNA, the mRNA molecules<br>
formed by transcription are monocistronic.<br>
The multiplication cycle of the Sendai virus comprises<br>
entry into the host cell, transcription and translation<br>
plus replication and virus maturation followed by<br>
release of newly produced viruses. In particular the<br>
proteins N, P and L are involved in the transcription<br>
process, with L representing the viral RNA-dependent<br><br>
- 17 -<br>
RNA polymerase with all catalytic activities. As the<br>
genome of the Sendai virus is in negative-strand<br>
orientation, the viral RNA cannot be converted to<br>
proteins directly. First there is primary transcription<br>
to mRNAs by RNA polymerase, which is brought into the<br>
cell associated with the nucleocapsid.<br>
Fig. 3 is a schematic representation of the<br>
transcription mode of the Sendai virus. The polymerase<br>
complex, comprising an L protein and a homotetramer of<br>
P proteins, migrates along the RNA packed with N<br>
proteins toward the 5' end. The genetic information on<br>
the genomic negative-strand RNA is read off and<br>
transcribed into positive-strand mRNA.<br>
Replication of the genome comprises the production of<br>
new virus genomes with negative polarity. For this,<br>
first antigenomes are formed, which then serve as<br>
matrixes for the formation of the genomes. As<br>
transcription begins at the 3' end of the genome (Id),<br>
switching from transcription to replication mode is<br>
required. This switching is determined by the amount of<br>
free N protein in the cell. Replication cannot take<br>
place until sufficient N protein has been formed after<br>
translation of mRNA molecules. Once an antigenome,<br>
which is complexed with N proteins over its entire<br>
length, is present, this can serve as a matrix for the<br>
production of further genomes. These are also packed<br>
directly with N proteins. Once again the proteins N, P<br>
and L are responsible for the process of replication<br>
(Fig. 4).<br>
During virus replication, owing to the increasing<br>
number of mRNA molecules there is also increasing<br>
synthesis of viral proteins. Then complexes of viral<br>
RNA and viral proteins (nucleocapsids) are transported<br>
in the form of secretory vesicles to the cytoplasmic<br>
membrane, where enveloping with viral surface proteins<br>
and budding of virus particles occur.<br><br>
- 18 -<br>
Within the scope of the present invention, recombinant<br>
virus mutants are prepared, in which the functions of<br>
transcription and replication are decoupled, i.e. the<br>
viruses are transcription-competent, but replication-<br>
deficient. The missing genome replication function<br>
must, for the production of virus particles and/or<br>
their nucleocapsids, be compensated by helper cells<br>
which complement the missing or functionally deficient<br>
viral protein in trans. A preferred helper cell of this<br>
kind is the cell line H29 (Willenbrink and Neubert, J.<br>
Virol. 68 (1994), 8413-8417). Within the scope of the<br>
present application, this cell line was deposited under<br>
reference DSM ACC2702 on 11.05.2004 in accordance with<br>
the provisions of the Budapest Treaty. For the<br>
production of replication-deficient, but transcription-<br>
competent viruses, the gene coding for the P protein<br>
was not removed completely, but only a domain essential<br>
for genome replication. Thus, it was known from earlier<br>
works (Curran, Virology 221 (1996), 130-140; Curran et<br>
al., J. Virol. 69 (1995), 849-855) that in an in vitro<br>
system, on deleting the amino acids 2-77 of protein P,<br>
genome replication is inhibited, whereas viral<br>
transcription remains active.<br>
Fig. 5 is a schematic representation of the P protein<br>
with its N-terminal and C-terminal domains (PNT, PCT) ,<br>
the tetramerization domain (amino acids 320-446), the<br>
P:L domain (ainino acids 411-445) and the P:RNP binding<br>
domain (amino acids 479-568). For switching off the<br>
viral genome replication function while simultaneously<br>
retaining the capacity for viral mRNA synthesis, a<br>
deletion of the first 77 amino acids of the protein P<br>
was selected. Firstly a corresponding Sendai virus<br>
mutant (SeV-PA2-77) was produced, in which the<br>
5'-terminal region of the P-0RF was deleted. Only N-<br>
terminal-shortened P proteins can be encoded by these<br>
viruses. Infection studies showed that the virus mutant<br>
is not capable of multiplying in cell culture. By means<br>
of helper cell line H29 (DSM ACC2702) , which among<br><br>
- 19 -<br>
other things provides the required wild-type P protein,<br>
multiplication of the virus mutant can be achieved. The<br>
efficiency of virus multiplication is approx. 45%<br>
compared with wild-type Sendai viruses.<br>
After infection, the virus mutant according to the<br>
invention is able to express virus-encoded transgenes<br>
in infected cells. The shortened protein P produced by<br>
the virus mutant gives sufficient support for secondary<br>
viral mRNA synthesis. Synthesis of virus-encoded<br>
proteins continues over several days in the infected<br>
cells and is only reduced by a factor of approx. 10<br>
relative to the wild-type virus, so that a sufficient<br>
immune response can reliably be expected on using the<br>
mutant as vaccine.<br>
2. Production of basic constructs for replication-<br>
deficient Sendai virus vectors (Sevv)<br>
2.1 Production of a cDNA construct pSeV-X<br>
An encoding transcription cassette was inserted in the<br>
3' region of SeV, Fushimi strain (ATCC VR105). In all<br>
manipulations of the genome it is essential to ensure<br>
that the total number of nucleotides of the recombinant<br>
SeV genome is divisible by six ("rule of six").<br>
Starting from the cDNA pSeV used as matrix, two PCR<br>
fragments PCR X I and PCR X II were prepared for the<br>
production of pSeV-X (Fig. 6) .<br>
PCR. X I (370 bp) comprises the sequence of the T7<br>
promoter (T7-prom.), the leader (Id) sequence, the N-<br>
gene start with 5'-NTR up to before the start codon of<br>
the N-ORF (open reading frame). Via the reverse primer<br>
IX I (+) (Table 3), a singular NotI restriction site and<br>
24 nucleotides of the N-gene stop sequence were<br>
attached. The 24 nucleotides of the N-gene stop<br>
sequence of PCR X I, inserted by the mutagenic primer X<br><br>
	- 20 -<br>
I ( + ) , serve in the subsequent fusion step as the<br>
region overlapping PCR X II.<br>
PCR X II (970 bp) comprises the sequence of the N-gene<br>
start and the first third of the N-ORF. Via the forward<br>
primer X II, the sequence of the 3'-NTR N and the gene-<br>
stop sequence of N, as well as the intergenic region<br>
(IR) were attached. The reverse primer XII (+) binds in<br>
the first third of the N-ORF just behind the singular<br>
SphI site in the SeV genome. The amplicon PCR X I was<br>
complementary, in the 3' region, to the 5" region of<br>
PCR X II. Through this overlapping region, the two PCR<br>
fragments X I and X II could be fused. After completion<br>
of PCR, the fusion product (1310 bp) could be inserted,<br>
by restriction cleavage with the enzymes Mlul and SphI,<br>
in the vector pSeV, also treated with Mlul and SphI.<br>
From the clones obtained, plasmid-DNA was isolated by<br>
plasmid preparation, and verified by restriction<br>
analysis and sequencing for correct insertion of the<br>
transcription cassette. The cDNA construct pSeV-X was<br>
thus made available.<br>
So that the production of recombinant viruses can be<br>
monitored easily, the gene for the enhanced green<br>
fluorescent protein (eGFP) was now inserted in the<br>
empty cassette of pSeV-X. The eGFP-ORF was amplified by<br>
iPCR from the expression plasmid pEGFP-Nl (from<br>
Clontech), maintaining the "rule of six" and achieving<br>
attachment of two flanking NotI sites by means of<br>
mutagenic primers. The resultant 771-bp PCR fragment<br>
was cleaved with the restriction enzyme NotI and a 738-<br>
bp fragment was isolated by gel elution, and was<br>
inserted via the NotI site of pSeV-X in its "empty"<br>
transcription cassette pSeV-X. After transformation of<br>
E. coll, plasmid preparation and subsequent sequencing<br>
of the eGFP reading frame inserted via PCR, the cDNA<br>
construct pSeV-eGFP was made available.<br>
2.2 Production of the cDNA construct pSeV-X-X<br><br>
- 21 -<br>
With the construct pSeV-X-X, two additional<br>
transcription cassettes were to be made available, in<br>
which two transgenes can be incorporated. The use of<br>
pSeV-X-X as base vector for the production of the<br>
replication-deficient vectors should make it possible<br>
to equip the vector with multivalent, e.g. trivalent,<br>
properties.<br>
pSeV-X-X was produced via a PCR reaction, in which<br>
pSeV-X served as template (Fig. 7) . The primer XX-<br>
forward hybridizes with pSeV-X in the region of the<br>
NotI site and the 3' -NTR of the second transcription<br>
cassette that is to be integrated. A singular SgrAI<br>
restriction site was introduced by means of the XX-<br>
forward primer between the NotI site and the 3'-NTR. It<br>
serves as singular restriction site for the later<br>
insertion of the ORF of a transgene. Gene stop,<br>
intergenic region (IR), gene start and 5'-NTR follow in<br>
the PCR product XX. The singular restriction sites Fsel<br>
and Nru I were inserted by the primer XX ( + ) , which<br>
hybridizes with the 5'-NTR. The Fsel site serves for<br>
incorporation of the ORF of a second transgene. The<br>
singular Nru I site was cloned-in prospectively, so as<br>
to be able to integrate a third transcription cassette<br>
if necessary. Primer XX (+) hybridizes in the 3' region<br>
with the sequence of the NotI site of pSeV-X. The PCR<br>
product XX (220 bp) was treated with the restriction<br>
enzyme NotI and a fragment of 144 bp was isolated by<br>
gel extraction. This fragment, designed maintaining the<br>
"rule of six", could then be incorporated in the<br>
plasmid pSeV-X, which was also treated with NotI. After<br>
checking for correct orientation of the NotI PCR<br>
fragment XX and verification of the sequence, the<br>
plasmid pSeV-X-X was ready. Any desired transgenes can<br>
be integrated in the singular sites SgrAI and Fsel.<br>
For the investigations in this work, the two<br>
transcription cassettes (X) of pSeV-X-X were provided<br>
with reading frames of two different fluorescent<br><br>
- 22 -<br>
proteins. On the one hand, the reading frame for the<br>
fluorescent protein eGFP from the expression plasmid<br>
pEGFP-Nl was amplified by PCR while observing the "rule<br>
of six", attaching two flanking SgrAI sites by means of<br>
mutagenic primers. After restriction cleavage with<br>
SgrAI and gel elution, the approx. 738-bp fragment<br>
could be incorporated in the first transcription<br>
cassette of pSeV-X-X (pSeV-eGFP-X). On the other hand,<br>
in the same way the ORF of the fluorescent protein<br>
DsRed (from the plasmid "pDsRed", from Clontech) was<br>
provided by PCR, observing the "rule of six", with the<br>
restriction sites of Fsel, the DNA was cleaved, gel-<br>
eluted and this fragment (702 bp) was then cloned into<br>
the second transcription cassette in the 3' region of<br>
pSeV-eGFP-X. The result was the genomic SeV cDNA<br>
construct pSeV-eGFP-DsRed.<br>
3. Production of replication-deficient Sendai virus<br>
vectors (Sevv)<br>
cDNA constructs pSevv-eGFP-AN, -AP and -AL were<br>
produced, which code for replication-deficient Sendai<br>
viruses, in each of which the gene for the protein N, P<br>
and L has been deleted. For this, in each case a<br>
reading frame of the genes N, P or L had to be deleted<br>
while observing the rule of six, and a non-coding<br>
transcription cassette was to be retained at the<br>
corresponding position (Fig. 8A) .<br>
By incorporating a restriction site instead of the<br>
deleted ORF, an additional functional transcription<br>
cassette, into which a further transgene can be<br>
inserted if required, was to be made available, for<br>
later applications, in each cDNA construct pSevv-<br>
eGFP-AN, -AP and -AL.<br>
As a further variant of a replication-deficient Sevv,<br>
the deletion mutant pSevv-eGFP-PA2-77 was produced,<br>
which codes for an N-terminal-shortened P protein<br><br>
- 23 -<br>
lacking amino acids 2 to 77 (Fig. SB).<br>
The clonings pSevv-eGFP-AN, -AP and -AL were all<br>
carried out according to the same principle. As an<br>
example, the cloning of pSevv-eGFP-AP will be described<br>
in detail in the next section. Then just the<br>
differences in the clonings of pSevv-eGFP-AN, and -AL<br>
will be presented in a table.<br>
3.1 Cloning of the cDNA constructs pSevv-eGFP-AP and<br>
pSevv-eGFP-PA2-77<br>
The ORF of the P protein was removed from the cDNA<br>
construct of the replication-competent virus pSeV-eGFP,<br>
to produce the new cDNA pSevv-eGFP-AP, coding for the<br>
replication-deficient vector. An Xhol restriction site<br>
was used instead of the P-ORF.<br>
For the cloning of pSevv-eGFP-DP, two PCR fragments<br>
named PCR DP I and PCR DP II were produced and then<br>
fused. pSeV-eGFP served as template for both PCR<br>
reactions. In the case of fragment PCR DP I (1272 bp) ,<br>
by means of the forward primer DP I ( = N-578; Table 3)<br>
hybridization with the template in the region of the N<br>
ORF was achieved before a singular SphI site. The<br>
reverse primer AP I (+) hybridizes with the template in<br>
the 5' -NTR region of the P-gene up to before the ATG<br>
codon of P and inserts the restriction site Xhol there.<br>
The fragment PCR AP II comprises 1938 bp, and pSeV also<br>
serves as template here. The forward primer DP II<br>
hybridizes with a portion of the 5'-NTR P sequence and<br>
attaches an Xhol site. The reverse primer of PCR DP II<br>
( + ) binds in the ORF of the F gene after a singular<br>
Eco47III site and additionally has an artificial Mlul<br>
site.<br>
The two PCR fragments DP I and DP II were combined via<br>
the Xhol site. The fusion product - comprising a<br><br>
- 24 -<br>
partial sequence of the N ORF, the non-coding P-<br>
transcription cassette with inserted Xhol restriction<br>
site, the M plus a quarter of the F ORF - was cleaved<br>
with the restriction enzymes SphI and Miul, intercloned<br>
and sequence-verified. An Sphl-Eco47III fragment with a<br>
size of 3006 bp was cut out of a subclone with correct<br>
sequence and was ligated in the identically treated<br>
vector pSeV-eGFP. A corresponding pSevv-eGFP-AP clone<br>
(genomic viral cDNA) was now ready, after sequence<br>
verification, for the production of the replication-<br>
deficient Sevv-eGFP-AP (Fig. 9) .<br>
A PCR with two mutagenic primers was employed for<br>
constructing the deletion mutant pSevv-eGFP-PD2-77. The<br>
forward primer "Xhol PD2-77" contains an Xhol site,<br>
followed by an ATG start codon plus codons for the<br>
amino acids 78 to 86 of the P protein. The reverse<br>
primer "PD2-77 (+) Xhol" contains the last 10 codons of<br>
the P protein and an Xhol site. The reading frame of<br>
the P protein shortened by 76 amino acids at the N-<br>
terminal was produced by PCR, starting from the<br>
template pSeV, observing the rule of six. The Xhol-<br>
cleaved, 1488-bp fragment was inserted via two cloning<br>
steps into the non-coding transcription cassette of<br>
pSevv-eGFP-AP at the position of the original P-ORF.<br>
After sequence verification, a genomic cDNA clone was<br>
now also ready for production of the replication-<br>
deficient Sevv-eGFP-PD2-77.<br>
Deletion of the codons 2 to 77 in the P ORF has the<br>
result that, in the case of the non-structural<br>
proteins, the V and W proteins are also shortened at<br>
the N-terminal end and, of the C family, only C' - also<br>
truncated - is still encoded; because the start codons<br>
are missing, the proteins C, Yl and Y2 can no longer be<br>
translated by the shortened mRNA.<br>
3.2. Cloning of the cDNA constructs pSevv-eGFP-DN and<br>
-DL<br><br>
- 25 -<br>
Production of pSevv-eGFP-DN and pSevv-eGFP-DL was<br>
carried out by a similar strategy to the cloning of<br>
pSevv-eGFP-DP. In order to summarize the cloning<br>
procedures, all the decisive parameters are presented<br>
in Table 1.<br>
Through the production in each case of two fusible PCR<br>
products PCR I and PCR II, the ORF of the genes N or L<br>
were removed from pSevv-eGFP, observing the rule of six,<br>
and were replaced with a singular Apal restriction site<br>
both in pSevv-eGFP-DN and in Sevv-eGFP-DL. The<br>
sequences of the primers for cloning pSevv-eGFP-DN, -AP<br>
and -DL are listed together with the DNA<br>
oligonucleotides used (Table 3). The PCR products<br>
obtained, PCR I and PCR II, were fused and amplified<br>
using the forward primer of PCR I and the reverse<br>
primer of PCR II. Then the fusion PCR products were<br>
cleaved with restriction enzymes which occur singly in<br>
pSeV-eGFP and allow the insertion of the corresponding<br>
fusion product in pSeV-eGFP (e.g.: Narl when cloning<br>
pSevv-eGFP-DN, see Table 1) . The purified cleavage<br>
product was inserted by ligation in the vector pSeV-<br>
eGFP, which was also digested with the corresponding<br>
enzymes.<br>
Table 1. Review of the primers and restriction sites<br>
used in the cloning of pSevv-eGFP-DX<br><br>
	pSevv-eGFP-DN	pSevv-eGFP-DP	pSevv-eGFP-DL<br>
Primer pair PCR 1	DN 1, DN 1 (+)	DP 1, DP 1 (+)	DL 1, DL 1 (+)<br>
Primer pair PCR II	DN II, DN II (+)	DP II, DP II (+)	DL II, DL II (+)<br>
RS§ of thetranscriptioncassette	Apal	Xhol	Apal<br>
RS§ of cloning	Narl	Sphl + Eco47lll	Eco47lll + AscI<br>
§ RS = restriction site<br>
E. coli cells were transformed with a portion of the<br>
ligation preparations and plasmid DNA of the clones<br>
obtained was isolated by plasmid mini-preparation.<br>
After verification of the correct sequence by<br><br>
- 26 -<br>
restriction analysis and sequencing, a plasmid<br>
preparation was prepared from one positive clone in<br>
each case (DNA Maxi Prep-Kit, Qiagen) , and the various<br>
pSevv-eGFP-DX were thus ready for the production of<br>
recombinant deletion mutants .<br>
4. Production of replication-deficient virus mutants<br>
Replication-deficient SeV vectors (Sevv-eGFP-DX) were<br>
produced in cell culture from the cDNA constructs<br>
pSevv-eGFP-DN, -DP and -DL.<br>
4.1 Initial production of Sevv-eGFP-DX<br>
For the production of reactive SeV with a complete<br>
genome:<br>
-	either the cell line "BSR-T7", which stably expresses<br>
the T7 RNA polymerase (Buchholz et al. (1999) J.<br>
Virol. 73, 251-259)<br>
-	or cell cultures are infected with the recombinant<br>
vaccinia virus MVA-T7, which expresses T7 RNA<br>
polymerase (Sutter et al. (1995) FEBS 371, 9-12), and<br>
-	transfected with the cDNA of the viral genome (pSeV)<br>
and the plasmid-encoded genes N, P and L (pTM-N, -P,<br>
-L; Elroy-Stein et al. (1989) PNAS 86, 6126-6130).<br>
The T7 polymerase now transcribes the viral<br>
antigenome and/or the complementary sequence and the<br>
genes N, P and L. The N protein expressed via pTM-N<br>
packages the synthesized viral antigenomic and/or<br>
complementary RNA, and this nucleocapsid core (RNP)<br>
forms, together with the proteins P and L, the<br>
replication complex, via which genomic RNA can in<br>
turn be produced and packaged as nucleocapsids<br>
(Leyrer et al., J. Virol. Meth. 75 (1998), 47-55).<br>
This is followed by transcription of all virus-<br>
encoded proteins and replication of further viral<br>
genomes (Fig. 10) .<br>
In contrast to the production of recombinant SeV wt+<br>
- 27 -<br>
with complete genome described, the plasmid-encoded<br>
cDNA of pSeVV-eGFP-N, -P and -L lacks the genomic<br>
information of one of the genes N, P or L in each case.<br>
Accordingly the nucleocapsids initially produced are<br>
only able to express two of the genes N, P or L in each<br>
case. The amount of the missing protein required for<br>
multiplication of SeVV-eGFP-N, -P and -L<br>
nucleocapsids must therefore be provided exclusively<br>
via the T7-promoter-controlled expression of the<br>
plasmid-encoded genes N, P and L.<br>
Production of the replication-deficient SeVV-eGFP-N,<br>
-P and -L was similar to the production of<br>
replication-competent SeV variants in HeLa cells (ATCC<br>
CCL2) or BSR-T7 cells. After incubation of the HeLa<br>
cells for 48 hours, we investigated whether viral<br>
particles of SeVV-eGFP-N, -P or -L had been released<br>
into the culture supernatants, and how many initial<br>
deletion mutants had been produced.<br>
4.2. Detection of initially produced SeVV-eGFP-N, -P<br>
 or -L<br>
The supernatants of HeLa cells or BSR-T7 cells, in<br>
which SeVV-eGFP-N, -P or -L should have been<br>
produced initially, were investigated for the presence<br>
of these viral vectors. SeVV-eGFP-N, -P or -L have,<br>
in contrast to the recombinant SeV wt, the reporter<br>
gene for eGFP integrated in the 3' region. This<br>
detection marker was now used for analyzing how many<br>
SeVV-eGFP-X had been formed.<br>
In parallel assays, 5 x 105 Vero cells (ATCC CCL18) were<br>
in each case co-infected with 1 ml of cell culture<br>
supernatant of the HeLa cells transfected during the<br>
initial production of SeVV-eGFP-N, -P and -L, and<br>
simultaneously with SeV wt (MOI = 3) for<br>
transcomplementation of the missing protein. As<br>
control, Vero cells were infected either only with 1 ml<br><br>
- 28 -<br>
of the initially produced SeV-eGFP or alternatively<br>
with initially produced SeV-eGFP and simultaneously<br>
with SeV wt (MOI = 3).<br>
The result of co-infection of cells with culture<br>
supernatants from production of SeVV-eGFP-ΔN, -ΔP or<br>
-ΔL and SeV wt shows that all three virus mutants SeVV-eGFP-ΔX can be produced initially and after initial<br>
production are also capable of infecting cells, which<br>
can be detected by a detectable eGFP transgene<br>
expression. Approximately 13 x 102 SeV-eGFP, 6.7 x 102<br>
SeVV-eGFP-ΔN, 3.2 x 102 SeVV-eGFP-ΔP and 0.55 x 102<br>
SeVV-eGFP-ΔL virus mutants can be produced initially.<br>
5. Multiplication of SeVV-eGFP-X<br>
Investigations were conducted to determine whether the<br>
SeVV-eGFP-ΔX vectors are multipliable, i.e. are<br>
biologically functional. Capacity for replication was<br>
first to be investigated by preparing the proteins of<br>
the missing genes N, P or L by viral<br>
transcomplementation with SeV.<br>
5.1 Demonstration of the ability of SeVV-eGFP-ΔX to<br>
multiply<br>
It was first necessary to demonstrate that the mutants,<br>
on corresponding transcomplementation by the SeV wt<br>
virus, are able to multiply and infect the surrounding<br>
cells. For investigating the ability of SeVV-eGFP-ΔX to<br>
multiply, it is again possible to use co-infection of<br>
cells with SeVV-eGFP-ΔX and SeV wt. In this test the<br>
cells were infected with SeV wt at low MOI, co-infected<br>
with SeVV-eGFP-ΔX and incubated for several days, so as<br>
to be able to detect the spread of the SeVV-eGFP-ΔX<br>
vectors from the increasing number of fluorescing<br>
cells.<br>
5 x 105 Vero cells were infected simultaneously with on<br><br>
- 29 -<br>
average 100 initially produced SeVV-eGFP-ΔN, -ΔP or -ΔL<br>
and SeV wt (MOI = 0.2). Co-infection of Vero cells with<br>
about 100 particles of the replication-competent virus<br>
SeV-eGFP and SeV wt was used as positive control.<br>
During a 48-hour incubation phase, multiplication of<br>
all three deletion mutants was observed: in each case,<br>
at first there was only fluorescence of individual Vero<br>
cells, which had been co-infected with SeVV-eGFP-ΔX and<br>
SeV wt. After about 24 h, newly synthesized virus<br>
particles were released from these cells, and were<br>
capable of penetrating nearby cells. If these cells<br>
were also infected simultaneously with SeVV-eGFP-ΔX and<br>
SeV wt, there was also detectable fluorescence in those<br>
cells. The multiplication of SeVV-eGFP-ΔN, -P and -ΔL<br>
in cells co-infected with wt could be detected from<br>
this "tailing" of fluorescing cells 48 h p.i. and<br>
beyond.<br>
This test established that the genomic cDNA constructs,<br>
from which SeVV-eGFP-ΔN, -P and -L were derived, are<br>
functional in all regions. It could also be shown that<br>
multiplication of virus mutants is possible on adding<br>
the missing viral protein. The protein (e.g. P)<br>
produced exclusively by wt virus is sufficient for<br>
multiplying both the deletion mutant and the wt, i.e.<br>
even a possibly suboptimal amount of P protein leads to<br>
the formation of functional nucleocapsids of SeV wt and<br>
SeVV-eGFP-ΔP. Supply of the missing protein by the<br>
transcomplementation partner SeV wt led to visible<br>
multiplication of all SeVV-eGFP-ΔX, measured from the<br>
increase in fluorescing cells in the cultures.<br>
5.2 Determination of the proteins required for<br>
multiplication of SeVV-eGFP-ΔX<br>
Next we investigated which proteins must be made<br>
available by a helper cell in each individual case of<br>
multiplication of SeVV-eGFP-ΔN, -ΔP and -ΔL, if it is<br>
to be possible for the SeV proteins N, P and L to be<br><br>
	- 30 -<br>
synthesized independently of one another. The three<br>
recombinant vaccinia viruses (VV) VV-N, VV-P and VV-L,<br>
which provide recombinant expression of the SeV<br>
proteins N, P or L, were available for independent<br>
synthesis of the SeV proteins N, P and L (Graef, H.<br>
(1994) Functional characterization of the recombinant<br>
Sendai virus large (L) protein. Thesis, Eberhard-Karls<br>
University Tübingen).<br>
1 x 106 Vero cells were co-infected simultaneously with<br>
SeVV-eGFP-X or SeV-eGFP (MOI = 0.01) and VV. VV-N, -P<br>
and -L were used individually or in combinations (MOI =<br>
0.5). After an adsorption time of one hour, the medium<br>
was replaced with DMEM + 10% fetal calf serum (FCS) +<br>
cytosine-arabinoside (AraC) (100 Δg/ml) and the cells<br>
were incubated for 72 h at 33°C, changing the medium<br>
daily, to add fresh AraC.<br>
The propagation of SeVV-eGFP-ΔX was analyzed via eGFP<br>
expression after 72 h. In this time, in the positive<br>
assays there was multiplication of green-fluorescing<br>
cells from one initial cell to 10-30 adjacent<br>
fluorescing cells.<br>
The results of the investigation of which SeV proteins<br>
are necessary for the multiplication of SeVV-eGFP-ΔN,<br>
-ΔP or -ΔL in Vero cells are presented in Table 2. Just<br>
expression of SeV N by VV-N does not lead to<br>
multiplication of SeVV-eGFP-ΔN. SeVV-eGFP-ΔN only<br>
multiplies if the SeV proteins N and P are expressed<br>
simultaneously in the infected cell by means of<br>
recombinant vaccinia viruses VV-N and VV-P. SeVV-<br>
eGFP-ΔP can be multiplied just by the expression of SeV<br>
P proteins by means of VV-P in Vero cells.<br>
Multiplication of SeVV-eGFP-ΔL required simultaneous<br>
synthesis of proteins P and L by VV. No multiplication<br>
of SeVV-eGFP-ΔL occurs in an SeVV-eGFP-ΔL and only VV-L<br>
infected Vero cell.<br><br>
- 31 -<br>
Table 2. SeV proteins required for multiplication of<br>
SeVV-eGFP-ΔX<br><br>
	in	vitro multiplication by	means of VV<br>
SeVV-eGFP-N	N		(N	+ P)+<br>
SeVV-eGFP-P	P	+	(N	+ P)+<br>
SeVV-eGFP-L	L		(L	+ P) +<br>
For the multiplication of SeVV-eGFP-ΔN, a helper cell<br>
must express the SeV proteins N and P simultaneously,<br>
expression of SeV P protein in the helper cell is<br>
sufficient for multiplication of SeVV-eGFP-ΔP, and the<br>
amplification of SeVV-eGFP-ΔL should be possible by<br>
cellular expression of the SeV proteins P and L.<br>
5.3 Supporting of amplification of SeVV-eGFP-ΔX by H29<br>
helper cells<br>
For the investigations of the multiplication of SeVV-<br>
eGFP-ΔX with the aid of the H29 cell line, 1 x 106 H29<br>
cells in four different assays were each infected with<br>
approx. 100 SeVV-eGFP-ΔN, -ΔP and -ΔL virus particles<br>
initially produced in HeLa cells or, as control for the<br>
successful multiplication of replicable SeV in H29<br>
cells, with about 100 SeV-eGFP virus particles. In the<br>
period from 1 to 10 d p.i., investigation of the<br>
multiplication of SeVV-eGFP-ΔX was based on detection<br>
of a tail-like spread of the fluorescing cells (spot<br>
formation), starting from an initially infected H29<br>
cell.<br>
Multiplication of SeV-eGFP was observed in the control<br>
assay, starting from singly fluorescing cells 1 d p.i.<br>
to spots with up to 50 fluorescing cells 3 d p.i. It<br>
was thus established that the selected test setup leads<br>
to multiplication of SeV.<br>
Virus multiplication also occurred in H29 cells<br><br>
- 32 -<br>
infected with SeVV-eGFP-ΔN. In addition to H29 (a<br>
derivative of human 293 renal cells) , derivatives of<br>
Vero cells (renal cells of the African green monkey)<br>
and derivatives of LLCMK2 cells (renal cells of the<br>
rhesus monkey), stably transfected with SeV P and N<br>
genes, are also suitable for virus multiplication.<br>
In the assay with SeVV-eGFP-ΔP, about a hundred<br>
initially infected individual cells could be detected 1<br>
d p.i. About 70% of the fluorescing individual cells<br>
had developed to spots, with up to 30 fluorescing<br>
cells, 3 d p.i. Therefore propagation of SeVV-eGFP-ΔP<br>
to surrounding H29 cells could definitely be observed.<br>
Thus, it is possible for the first time to multiply a<br>
viral SeV vector whose P-ORF has been deleted.<br>
Characterization of the multiplication of SeVV-eGFP-P<br>
will be discussed in the next subsection.<br>
5.4 Multiplication of SeVV-eGFP-ΔP on H29 cells<br>
SeVV-eGFP-ΔP can be amplified by the SeV P proteins<br>
produced by H29 helper cells. The P-deletion mutants<br>
released are able to infect surrounding H29 cells. It<br>
was now necessary to analyze the propagation of SeVV-<br>
eGFP-ΔP in comparison with the propagation of the<br>
replication-competent SeV-eGFP.<br>
For this purpose, 1 x 106 H29 cells were infected with<br>
on average 100 SeVV-eGFP-ΔP or SeV-eGFP. 3, 5 and 10<br>
days p.i., green-fluorescing cells were detected using<br>
the fluorescence microscope.<br>
SeVV-eGFP-P could be multiplied successfully by<br>
cellular supply of SeV P proteins. It was found, at all<br>
times of investigation, that SeVV-eGFP-ΔP and SeV-eGFP<br>
multiply efficiently on H29 cells.<br>
In contrast, propagation of SeVV-eGFP-P to cells that<br>
do not supply the missing P protein ("target cells",<br><br>
- 33 -<br>
e.g. Vero cells) was not observed, which confirms that<br>
SeVV-eGFP-ΔP is replication-deficient (see Section 8) .<br>
5.5 Gene expression of SeVV-eGFP-P in infected target<br>
cells<br>
Absence of multiplication of SeVV-eGFP-P on cell types<br>
which do not supply the P protein in trans was<br>
verified. At the same time, capacity for expression was<br>
way below expectations.<br>
As in the case of the rabies virus P mutant (Shoji et<br>
al. (2004) Virology 318, 295-305) , very few infected<br>
cells displayed a weak eGFP fluorescence (less than 5%;<br>
see Fig. 11) , although statistically at an MOI = 1 in<br>
fact approx. 70% of the cells are each infected with<br>
one virus particle. Even at a higher MOI = 5, only<br>
isolated green-fluorescing Vero cells are observed due<br>
to SeVV-eGFP-ΔP (see Fig. 12, top left) .<br>
This confirms the assumption that after a cell is<br>
infected with a P gene-deficient virus, only a primary<br>
transcription is possible via the polymerase complex<br>
that is also supplied from the virus particle. In the<br>
case of the SeV ΔP mutant, furthermore, apparently only<br>
a small percentage of the infecting particles are<br>
capable of that, or gene expression is only observed if<br>
several transcribable nucleocapsids are present<br>
simultaneously in a cell.<br>
For a therapeutic application of this replication-<br>
deficient SeVV, the capacity for expression seems too<br>
weak, or disproportionately many particles of SeVV ΔP<br>
would have to be applied per patient. Therefore it is<br>
desirable to use a replication-deficient SeV variant<br>
that also performs a secondary transcription. This<br>
leads to the development of additional modified<br>
polymerase complexes, which cannot replicate the viral<br>
genome, but are capable of increased expression of the<br><br>
- 34 -<br>
therapeutic gene or antigen.<br>
6. Production of a modified SeVV-eGFP-ΔP cDNA<br>
construct<br>
For possible improvement of the transcription capacity<br>
of P gene-deficient SeVV in the target cell, another<br>
recombinant construct was produced, which codes for a<br>
form of the P protein shortened by 76 amino acids at<br>
the N-terminal end, at the position of the original P<br>
reading frame ("pSeVV-eGFP-P2-77"; see Section 3.1 and<br>
Fig. 8B).<br>
SeVV-eGFP-P2-77 particles were generated and<br>
multiplied as in Section 4.1 and 5.4.<br>
6.1 Growth behavior of SeVV-eGFP-P2-77 in H29 helper<br>
cells<br>
In SeVV-eGFP-P2-77 infected H29 helper cells, the<br>
viral-encoded P2-77 protein, shortened by 76 amino<br>
acids at the N-terminal end, is synthesized together<br>
with the cellular-encoded P protein.<br>
In order to investigate the effect of expression of the<br>
shortened P protein P2-77 on viral replication, H29<br>
cells were infected (MOI = 3) with SeVV-eGFP-P2-77,<br>
SeV-eGFP-P or the control virus SeV-eGFP as in the<br>
method described with reference to the growth kinetics<br>
of SeVV-eGFP-P. The supernatants of the individual<br>
assays were determined over a period of 120 h by a cell<br>
infection dose test of the titers of progeny, viruses<br>
from the number of eGFP-expressing cells.<br>
From one SeV-eGFP infected H29 cell (positive control),<br>
on average 80 virus particles are released in a period<br>
of 120 h, and in this case transcomplementation of the<br>
P protein by H29 cells was not required. In the H29<br>
transcomplementation system, SeVV-eGFP-P2-77 could be<br><br>
35	- 35 -<br>
multiplied with about equal efficiency as SeVV-eGFP-ΔP:<br>
From infected H29 cells, after 120 h about 20 x 106<br>
virus particles of SeVV-eGFP-ΔP or SeVV-eGFP-PΔ2-77 are<br>
released, which corresponds to a number of about 40<br>
released virus particles of the P mutants per H29 cell.<br>
7. Comparison of gene expression of SeVV-eGFP-ΔP and<br>
SeVV-eGFP-P2-77 and quantification of protein<br>
synthesis in infected target cells<br>
To investigate whether the vector SeVV-eGFP-PΔ2-77<br>
displays increased transgene expression - compared with<br>
SeVV-eGFP-ΔP - in infected target cells, the virus-<br>
encoded expression of the reporter gene eGFP and of the<br>
HN protein was characterized in detail.<br>
5 x 105 Vero cells were infected with SeVV-eGFP-P2-77<br>
15(M0I = 1) , and on day 2 p.i. approx. 70% fluorescing<br>
Vero cells were observed (not shown). This means that<br>
almost every RNP complex of this viral vector variant<br>
is capable of inducing a measurable transcription in<br>
the target cell.<br>
207.1 Quantification of eGFP expression by FACS analysis<br>
In subsequent applications in the medical area, the<br>
transcription cassette, into which the reporter gene<br>
eGFP in SeVV-eGFP-P2-77 was inserted, is to encode the<br>
antigen of a pathogen, e.g. of a desired virus. This<br>
antigen expression must be sufficient to elicit a<br>
protective immune response in the patient.<br>
In order to establish that each SeVV-eGFP-P2-77<br>
nucleocapsid that infects a target cell is able to<br>
perform a detectable transgene expression, the same<br>
30number of H29 and Vero cells were infected with the<br>
same quantity of virus particles and the number of<br>
eGFP-expressing H29 or Vero cells were compared by FACS<br>
(fluorescence-activated cell sorting) analysis using a<br><br>
- 36 -<br>
FACS-Calibur flow cytometer. The data were evaluated<br>
from a computer-generated histogram, plotting the<br>
fluorescence signals of infected cells against the<br>
total cell count.<br>
2.5 x 105 Vero cells or H29 cells were infected with<br>
SeVV-eGFP-PΔ2-77 or with the P gene-deficient virus<br>
SeVV-eGFP-ΔP at MOI = 1. FACS analysis of the samples<br>
was carried out 24 h after infection. The infected<br>
cells were taken up in PBS and the number of<br>
fluorescing cells was determined by flow cytometry. The<br>
result is shown in Fig. 11 as the proportion [%] of<br>
fluorescing Vero and H29 cells.<br>
24 h after infection of H29 helper cells with SeVV-<br>
eGFP-ΔP, 86% eGFP-expressing H29 cells were detected by<br>
FACS analysis. Cellular synthesis of the P protein<br>
supports the formation of new P:L complexes and<br>
therefore production of new mRNA, which leads to<br>
synthesis of eGFP proteins in the infected cell.<br>
However, when Vero cells were infected with SeVV-<br>
eGFP-ΔP, expression of the eGFP protein was not<br>
detected 24 h p.i. nor at a later time in additional<br>
assays. The P:L complexes transferred by SeVV-eGFP-ΔP<br>
nucleocapsids are not able, on infection at MOI = 1, to<br>
bring about detectable expression in infected Vero<br>
cells.<br>
In contrast, 24 h after SeVV-eGFP-P2-77 infection,<br>
almost 80% eGFP-expressing H29 cells and likewise 75%<br>
eGFP-expressing Vero cells were identified by FACS<br>
analysis. When H29 cells are infected, transcription.of<br>
the eGFP mRNA can be supported by the cellular<br>
synthesis of the P protein and therefore by new P:L<br>
complexes. When Vero cells are infected with SeVV-eGFP-<br>
P2-77, transcription is intensified by new synthesis<br>
of the P protein P2-77 shortened at the N-terminal<br>
end.<br><br>
- 37 -<br>
Based on these results, an important statement can be<br>
made concerning the number of transcribable SeVV-eGFP-<br>
P2-77: H29 cells and Vero cells were infected with an<br>
MOI of 1. Theoretically occurring multiple infections<br>
of a cell are statistically of equally low probability<br>
in both assays. 24 h p.i., almost 80% of eGFP-<br>
expressing H29 cells and about 75% eGFP-expressing Vero<br>
cells were identified. It can therefore be concluded<br>
that each RNP complex with the shortened P-variant P2-<br>
1077, which is capable of transgene expression in a P<br>
helper cell, likewise brings about transgene expression<br>
in the infected target cell. In contrast, the variant<br>
SeVV-eGFP-ΔP with a complete deletion of the P ORF does<br>
not have this property.<br>
7.2 Function test of the HN protein expressed in<br>
infected target cells<br>
The efficiency of binding of human erythrocytes and<br>
therefore the efficiency of exposure of viral HN<br>
proteins on individual infected cells were detected<br>
using a heme adsorption (HAD) test (Fig. 12).<br>
5 x 105 Vero cells were infected with SeVV-eGFP-P2-77<br>
at a low MOI = 0.5. In contrast, in the case of SeVV-<br>
eGFP-P, Vero cells had to be infected with a 10-times<br>
higher MOI = 5, to be able to observe even occasional<br>
fluorescing cells. After 1 h adsorption, the medium was<br>
replaced with DMEM + 10% FCS. Then the cells were<br>
incubated at 33°C for several days. At first, the<br>
transgene expression of both vector variants was<br>
monitored from the eGFP fluorescence. On day 5 and 9<br>
p.i., HAD test series were performed, for analyzing the<br>
exposure of the viral HN proteins on the basis of<br>
binding of human erythrocytes.<br>
Although in the case of SeVV-eGFP-ΔP, at MOI = 5,<br>
statistically 99.3% of the Vero cells ought to be<br>
infected, only 0.01% eGFP-positive cells could be<br><br>
- 38 -<br>
observed under the microscope (Fig. 12 top left). In<br>
contrast, with SeVV-eGFP-P2-77 a green fluorescence is<br>
seen at MOI - 0.5 in 40% of the cells, as was expected<br>
(Fig. 12 bottom left).<br>
Two days after infection with SeVV-eGFP-ΔP, only half<br>
of the eGFP-positive cells (25 of 5x 105 infected) were<br>
capable of binding erythrocytes on their surface (Fig.<br>
12 top centre). After further incubation and HAD<br>
testing again, there was no longer adsorption of<br>
erythrocytes on infected cells. With the second<br>
replication-deficient SeVV variant SeVV-eGFP-P2-77,<br>
much better results were achieved: 5 days after<br>
infection, approx. 40% of the infected cells (MOI =<br>
0.5) were capable of binding erythrocytes on their<br>
surface (Fig. 12 bottom center). A variation in binding<br>
activity from 10 to 70 complexed red blood cells was<br>
observed for the individual cells. It had thus been<br>
shown that cells infected with SeVV-eGFP-P2-77 are<br>
capable, 5 d p.i., of exposing HN proteins on the cell<br>
surface, and the functional HAD test can be assessed as<br>
positive. At the same time, efficient expression of the<br>
HN protein in the infected cells was found, based on<br>
the difference in quantity of bound erythrocytes.<br>
The infected cells were further incubated at 33°C,<br>
during which the neuraminidase activity of the HN<br>
protein causes the erythrocytes bound to infected cells<br>
to be released. The cells were washed to remove the<br>
released erythrocytes, and incubated for a further 4 d<br>
at 33°C. A HAD test was performed again on day 9 p.i.<br>
About 30% HAD-posit.ive Vero cells were now detected.<br>
The number of bound red blood cells dropped to 5-20<br>
erythrocytes per cell.<br>
Thus, 9 days i.p., sufficient functional HN was still<br>
synthesized by the replication-deficient variant SeVV-<br>
eGFP-P2-77. In contrast, the variant SeVV-eGFP-ΔP with<br>
complete deletion of the P ORF does not have these<br><br>
- 39 -<br>
properties.<br>
7.3	Quantification of eGFP expression by Western blot<br>
analysis<br>
A semi-quantitative assessment of eGFP expression of<br>
the replication-deficient SeV vector in comparison with<br>
replication-competent SeV-eGFP was carried out by<br>
Western blot analysis using serial dilution of total<br>
cellular protein.<br>
5 x 105 Vero cells were infected with SeV-eGFP or SeVV-<br>
eGFP-P2-77 (MOI = 3). Cell disruption was performed 24<br>
h p.i. The cellular extracts were separated in SDS-PAGE<br>
in serial dilutions (1:2) from 20 Δg to 2,5 Δg total<br>
quantity. The proteins were transferred to a PVDF<br>
membrane and the viral-encoded eGFP protein (26 kDa)<br>
was detected first by Western blot analysis (Fig. 13).<br>
The fluorescence protein eGFP was detected using<br>
Western blot analysis both in SeV-eGFP and in SeVV-<br>
eGFP-P2-77 infected Vero cells. It can be concluded,<br>
from comparison of the intensities of the eGFP signals,<br>
that expression - mediated by the replication-deficient<br>
SeVV-eGFP-P2-77 - is reduced by about a factor of 16<br>
compared with the replication-competent SeV-eGFP.<br>
A reduction by a factor of 16 is slight, and it can<br>
therefore be concluded that despite the modified P<br>
protein, SeVV-eGFP-P2-77 can produce very efficient<br>
secondary transcription.<br>
7.4	Estimation of HN expression by Western blot<br>
analysis<br>
In contrast to the eGFP protein, the SeV HN protein is<br>
a membrane-bound surface protein and is an important<br>
antigenic determinant. By relative quantification of<br>
the level of expression of the HN protein in SeVV-eGFP-<br><br>
	- 40 -<br>
PΔ2-77 infected cells, a conclusion could be drawn<br>
concerning the intensity of expression of the SeV HN<br>
antigen.<br>
A semi-quantitative assessment of HN expression of the<br>
replication-deficient SeV vector in comparison with the<br>
replication-competent SeV-eGFP was carried out by<br>
Western blot analysis using serial dilutions of total<br>
cellular protein. In each case, 5 x 105 Vero cells were<br>
infected with SeV-eGFP or SeVV-eGFP-P2-77 in 2<br>
parallel assays (MOI = 1) and incubated for 24 h or 48<br>
h. Then the cells were disrupted. The cellular extracts<br>
were separated in SDS-PAGE in serial dilutions (1:2)<br>
from 16 g to 2 g total quantity. The proteins were<br>
transferred to a PVDF membrane and the viral-encoded HN<br>
protein (60 kDa) was detected by means of a monoclonal<br>
HN-antibody (Fig. 14).<br>
The HN protein was detected efficiently in the case of<br>
SeV-eGFP infected Vero cells after both incubation<br>
times in all traces (16 to 2  total protein; traces<br>
202-5 left and right). In the case of SeVV-eGFP-P2-77,<br>
the band of the HN protein is still visible in the<br>
traces with 16 and 8 g total protein (trace 7, 8),<br>
though at lower intensity. Relative quantification of<br>
HN expression in SeVV-eGFP-P2-77 infected Vero cells<br>
relative to SeV-eGFP was carried out by comparing the<br>
traces with 16 and 8  vs. 2 g total protein (traces<br>
7 and 8 vs. 5, left and right) and permits an estimated<br>
reduction of HN expression by a factor of 8-16 of the<br>
P-deletion variant, regardless of the incubation time.<br>
It can be concluded that the transcription rate in<br>
SeVV-eGFP-P2-77 infected cells is relatively high, and<br>
therefore transgene expression of the viral<br>
replication-deficient vector is generally high.<br>
Taking both measurements (eGFP protein and HN protein)<br>
into account, it can be assumed that there is an<br>
average reduction of expression by a factor of 10.<br><br>
- 41 -<br>
8. Replication deficiency of SeVV-eGFP-P2-77 in the<br>
target cell<br>
If Vero cells are infected with the replication-<br>
competent SeV-eGFP, in the next two days a spot<br>
comprising up to a thousand additional fluorescing<br>
cells forms around the initially infected, strongly<br>
fluorescing cell. To prove the replication deficiency<br>
of SeVV-eGFP-P2-77 in Vero cells, it was necessary to<br>
confirm the absence of this increase in green-<br>
fluorescing cells around the initially infected target<br>
cell, taking into account the natural rate of division<br>
of Vero cells. Vero cells divide on average every 24 h.<br>
If Vero cells are infected with SeVV-eGFP-P2-77, a<br>
detectable eGFP expression can be seen after about 24<br>
h. It was observed that after a further 24 h incubation<br>
phase, in some cases two (more weakly) fluorescing<br>
daughter cells are produced from this initially<br>
infected, fluorescing Vero cell on account of natural<br>
division. This observation has nothing to do with virus<br>
multiplication, in which between 101 to 104 virus<br>
particles are released from an infected target cell,<br>
and can then infect nearby cells. This natural rate of<br>
division of infected cells does not, however, affect<br>
the level of eGFP expression, which is reduced with<br>
increasing number of cell divisions. This observation<br>
shows that the viral vector SeVV-eGFP-P2-77 is genome-<br>
replication-deficient, so no new genomes are<br>
synthesized. If several successive cell divisions of an<br>
infected cell lead to a continuous decrease in<br>
fluorescence intensity, until it finally stops, virus<br>
multiplication can be ruled out.<br>
For final confirmation of the replication deficiency of<br>
SeVV-eGFP-P2-77 in target cells, one last study was<br>
conducted:<br>
~ 20 x 106 Vero cells were placed in a T75 flask. The<br>
cells had been seeded at high density at the beginning<br>
of the incubation phase and accordingly were no longer<br><br>
- 42 -<br>
dividing actively. These Vero cells were infected with<br>
SeVV-eGFP-PΔ2-77 at an MOI of 0.001. The medium was<br>
changed to DMEM with 5% FCS (for reduced activity of<br>
division) after incubation for 1 hour, and the Vero<br>
cells were incubated at 33°C (P1). Two days p.i.,<br>
according to the selected MOI, initially several<br>
thousand separate infected, fluorescing Vero cells were<br>
observed. Owing to the high cell density, over the next<br>
4 days of incubation there was hardly any cell<br>
division, i.e. the number of initially infected cells,<br>
detected by fluorescence, remained constant. If virus<br>
particles had been formed in this period, they would<br>
have been able to infect nearby cells, and this would<br>
have been reflected in increased fluorescence. Even<br>
after 8 days, propagation of the viral vector could be<br>
ruled out, owing to absence of new infections of<br>
surrounding cells. To supply the cells with fresh<br>
medium, the supernatant was removed and the Vero cells<br>
were covered with fresh medium. 12 days after the start<br>
of incubation, the Vero cells became detached from the<br>
culture medium. For the whole test period, no<br>
replication of the viral vector was observed in the<br>
form of an increase in fluorescing cells. Propagation<br>
of SeVV-eGFP-P2-77 from the originally infected cells<br>
to surrounding Vero cells by production of new virus<br>
genomes and particles could thus be ruled out.<br>
Therefore SeVV-eGFP-P2-77 can be described as a<br>
replication-deficient viral vector.<br>
Summary:<br>
The above results show that specific manipulations of<br>
genes for components of the polymerase complex can lead<br>
to the production of replication-deficient negative-<br>
strand RNA viruses, which are still able to transcribe<br>
the virus-encoded genes, but are no longer able to<br>
replicate the viral genome.<br>
In the case of the Sendai virus, two particular<br>
variants were investigated more closely, in which the<br><br>
- 43 -<br>
gene for the polymerase cofactor phosphoprotein was<br>
deleted completely ("SeVV-eGFP-ΔP") or the codons for<br>
amino acids 2 to 77 were removed ("SeVV-eGFP-PΔ2-77") .<br>
Both SeV vectors are replication-deficient in cells<br>
which do not supply the P protein in trans (so-called<br>
target cells) , but they differ considerably in their<br>
capacity for gene expression.<br>
Although in the case of SeVV-eGFP-ΔP at an MOI = 5,<br>
statistically only 0.7% of the Vero cells remain<br>
uninfected - 99.3% should contain at least one RNP<br>
complex - only 0.01% eGFP-positive cells were observed<br>
under the microscope. It can be concluded from this<br>
mathematically that visible transgene expression only<br>
occurs if 15 or more RNPs of SeVV-eGFP-ΔP are present<br>
simultaneously in an infected target cell.<br>
This P gene-deficient SeVV displays similar weak<br>
expression as the analogous rabies P variant (Shoji et<br>
al., supra). Both vectors are only capable of primary<br>
transcription in the infected target cell, via the<br>
polymerase complex that is supplied from the virus<br>
particle. However, stronger expression of the encoded<br>
transgene or antigen is desired for therapeutic<br>
application of the vector. This condition can be<br>
fulfilled with the aid of the replication-deficient<br>
variant SeVV-eGFP-P2-77, which only gives a capacity<br>
for expression in the target cells that is reduced on<br>
average by a factor of 10 in comparison with<br>
replication-competent SeV. Owing to the presence of the<br>
gene for a P protein shortened at the N-terminal end in<br>
the vector genome, not only primary, but also secondary<br>
transcription is possible. This is realized with newly<br>
formed, modified polymerase complexes, which contain<br>
the vector-encoded P2-77 protein; this does not,<br>
however, support the replication mode of polymerase.<br>
Quantification of protein synthesis in infected target<br>
cells has demonstrated that the replication-deficient<br><br>
- 44 -<br>
viral vector SeVV-eGFP-PΔ2-77 is capable of performing<br>
efficient transcription and expression of viral-encoded<br>
genes. Not only is the 3' -proximal transgene (eGFP)<br>
effectively synthesized; the HN gene located at genome<br>
position 6 is transcribed for at least 9 days after<br>
infection and the protein is exposed functionally on<br>
infected target cells.<br>
9. Determination of the immune response induced in a<br>
mouse model by a replication-deficient RNA vaccine<br>
It was shown that preferably by a deletion in the P<br>
gene ("P2-77") an altered viral polymerase complex is<br>
produced, which no longer allows synthesis of new<br>
genomes. At the same time, after infection with these<br>
replication-deficient viruses, the viral gene<br>
expression mediated in the target cell is only approx.<br>
l0x lower in comparison with infections with<br>
replication-competent virus.<br>
In order to demonstrate a sufficiently immunogenic<br>
property of the replication-deficient negative-strand<br>
RNA virus as vaccination vector, antigens or antigenic<br>
determinants of two heterologous viruses (human<br>
parainfluenza virus type 3, hPIV3, and respiratory<br>
syncytial virus, RSV) were inserted in the virus<br>
genome: for this, a replication-deficient SeV P2-77<br>
was constructed, in which the genes of the original<br>
surface proteins F and HN were replaced with genes<br>
coding for chimeric F and HN proteins SeV/hPIV3. The<br>
chimeric F protein contains 558 amino acids and<br>
comprises the extracellular domain of hPIV3 (493 amino<br>
acids), the transmembrane domain of SeV (23 amino<br>
acids) and the cytoplasmic domain of SeV (42 amino<br>
acids). The chimeric HN protein has 579 amino acids and<br>
comprises the cytoplasmic domain of SeV (35 amino<br>
acids), the transmembrane domain of SeV (25 amino<br>
acids) and the extracellular domain of hPIV3 (519 amino<br>
acids). The amino acid sequences of the chimeric F<br><br>
	- 45 -<br>
protein and of the chimeric HN protein are shown in the<br>
sequence listing as SEQ ID No. 27 and 28.<br>
Inserting chimeric genes in the virus genome produces a<br>
novel antigenicity and in addition ensures efficient<br>
assembly of vaccine particles during their production.<br>
The surface proteinF of RSV was encoded in an<br>
additional expressioncassette interposed between two<br>
viral genes, so thatthe construct was extended to a<br>
bivalent vaccine.<br>
This new vaccine was tested in an animal model. Groups<br>
of Balb/C mice were immunized intranasally three times<br>
with two different virus preparations (group A or C, 104<br>
infectious units each) at intervals of three weeks, and<br>
a control group (B) received PBS instead of the<br>
vaccine. After the third immunization, nasal wash fluid<br>
(NW) was obtained for analysis of the mucosal immune<br>
response, and broncho-alveolar (BAL) flushing was<br>
carried out, and the serum was isolated for analysis of<br>
the humoral immune response. Using ELISA, we determined<br>
the quantity of induced immunoglobulins IgA and IgG<br>
specifically against hPIV3 and RSV. The replication-<br>
deficient vaccine prototype produced a definite<br>
induction of IgA antibodies specifically against hPIV3<br>
(Fig. 15A), but there was less induction of anti-RSV<br>
IgA antibodies (not shown) . The induction of a humoral<br>
immune response to the surface antigens of both viruses<br>
produced comparable titers, and the amount of specific<br>
IgG differs by a factor of 2 (Fig. 15B) . Further<br>
analysis of the anti-hPIV3-IgG showed that the induced<br>
antibodies have neutralizing properties (titer 1/64).<br>
In contrast, as expected, no specific IgA or IgG<br>
induction was found in the control group.<br>
The vaccine according to the invention was able to<br>
induce a specific mucosal and humoral immune response<br>
to heterologous viral antigens. Additional experiments<br><br>
- 46 -<br>
showed that lymphocytes of immunized mice produced<br>
interferon-y whereas IL-5 could not be detected. This<br>
finding indicates that the bivalent, replication-<br>
deficient RNA vaccine is able to trigger a T-cell<br>
immune response, which is a prerequisite for long-<br>
lasting immunity.<br>
Summary<br>
Following infection of experimental animals with a<br>
modified vector, in which coding sequences for antigens<br>
of two heterologous viruses were inserted, the<br>
induction of neutralizing antibodies was detected. This<br>
shows the potential of replication-deficient negative-<br>
strand RNA viruses for the development of novel<br>
vaccines.<br>
10. List of DNA oligonucleotides used<br>
The DNA oligonucleotides used in the above examples are<br>
shown below in Table 3.<br><br>
- 47 -<br>
Table 3:<br><br>
SEQIDNo.	Designation	Length[nt]	Sequence 5 ' ® 3 '	Tm[°C]<br>
1	X I = M13	19	GGAAACAGCTATGACCATG	54<br>
2	X I ( + )	59	GGATCATTAGTACCTTGAAGCCTCGTAGATCGCGGCCGCGTGAACTTTGGCAGCAAAG	56<br>
3	X II	57	GGCTTCAAGGTACTAATGATCCGTAGTAAGAAAAACTTAGGGTGAAAGTATTCCACC	64<br>
4	X II (+) =N-1029 (+)	18	GGTAGGTGTCTATGAGGC	56<br>
5	XX	44	GGAAGGAAAAGCGGCCGCCGGCGGGATCATACGAGGCTTCAAGG	61<br>
6	XX ( + )	57	CCTGTGTTTCTGCGGCCGCCGTTCGCGAGGCCGGCCCGTGAACTTTGGCAGCAAAGC	61<br>
7	NotI eGFP	44	CGCGGGCCCGGGGCGGCCGCGTCGCCACCATGGTGAGCAAGGGC	60<br>
8	eGFP NotI( + )	26	GATGCATGCTCGAGCGGCCGCTTTAC	58<br>
9	SgrAI eGFP	36	GGATTACTATCGCCGGCGGTCGCCACCATGGTGAGC	61<br>
10	eGFP SgrAI( + )	37	CGCTAACTGTCGCCGGCGTTTACTTGTACAGCTCGTCC	63<br>
11	FseI DsRed	36	CGGATCAAGTGGCCGGCCGTCGCCACCATGGTGCGC	59<br>
12	DsRed FseI( + )	42	CGCGAATATCGGCCGGCCAAGTCTACAGGAACAGGTGGTGGC	63<br>
13	DN I = M13	19	GGAAACAGCTATGACCATG	54<br>
14	DN I (+)	35	CGGTGCGGGCCCGCACGTGAACTTTGGCAGCAAAGC	50<br>
15	DN II	28	GTTCACGTGCGGGCCCGATCATACGAGG	44<br>
16	DN II (+)=P-2892 (+)	19	CGCGTCTCGGGATGATTCG	62<br>
17	DP I = N-578	17	CCCTGACACACTCCTTC	54<br><br>
- 48-<br><br>
SEQIDNo.	Designation	Length[nt]	Sequence 5 ' ® 3 '	T[°C]<br>
18	DP I (+)	31	GCGCCGCTCGAGGCGGTAAGTGTAGCCGAAG	64<br>
19	DP II	34	CCTGCGCTCGAGCTCATCCCGGGTGAGGCATCCC	64<br>
20	DP II(+)	34	GGCGACGCGTCAGTCTCACAGCCTAATTCG	64<br>
21	XhoI P2-77	47	CCCCCTTTTTCTCGAGATGTCGACCCAAGATAATCGATCAGGTGAGG	84<br>
22	P2-77 (+)XhoI	46	TTTTTCCCCCCTCGAGTTACTAGTTGGTCAGTGACTCTATGTCCTC	80<br>
23	DL I = F-4871	20	AGCATATATCCAGAGGTCAC	58<br>
24	DL I (+)	38	GGGACTAATTAGTCGGGCCCGACC	58<br>
25	DL II	31	GCACTTGGGCCCGACTAATTAGTCCCTC	60<br>
26	DL II ( + )	21	CGAATGGCGCGCCTGATGCGG	64<br><br>
- 49 -<br>
Patent claims<br>
1.	A recombinant negative-strand RNA virus,<br>
containing a viral genome with a mutation in at<br>
least one of the genes N, L and P, which leads to<br>
 loss of capacity for replication without loss of<br>
capacity for secondary transcription.<br>
2.	The virus as claimed in claim 1, characterized in<br>
that it is a paramyxovirus.<br>
3.	The virus as claimed in claim 1 or 2,<br>
 characterized in that it is a Sendai virus.<br>
4.	The virus as claimed in one of claims 1 to 3,<br>
characterized in that it has a mutation in gene P.<br>
5.	The virus as claimed in claim 4, characterized in<br>
that the mutation relates to the N-terminal<br>
 partial sequence of the protein encoded by gene P.<br>
6.	The virus as claimed in claim 5, characterized in<br>
that the mutation comprises: a deletion of<br>
(a) amino acids 2-77 of the protein encoded by<br>
gene P or<br>
 (b) a partial sequence of (a) sufficient for loss<br>
of the capacity for replication.<br>
7.	The virus as claimed in one of claims 1 to 6,<br>
characterized in that the viral genome contains at<br>
least one sequence coding for a heterologous gene<br>
 product.<br>
8.	The virus as claimed in claim 7, characterized in<br>
that the heterologous gene product is a protein, a<br>
ribozyme, an antisense molecule or an siRNA<br>
molecule.<br>
9. The virus as claimed in claim 7 or 8,<br><br>
50	- 50 -<br>
characterized in that the heterologous gene<br>
product is a reporter protein, an antigen or a<br>
therapeutic protein.<br>
10.	The virus as claimed in one of claims 7 to 9,<br>
 characterized in that the heterologous gene<br>
product is an antigen of a heterologous pathogen,<br>
selected from viruses, bacteria and protozoa.<br>
11.	The virus as claimed in one of claims 7 to 10,<br>
characterized in that the heterologous gene<br>
 product is a viral antigen.<br>
12.	The virus as claimed in claim 11, characterized in<br>
that the viral genome codes for several<br>
heterologous antigens from the same or different<br>
viruses.<br>
13. The virus as claimed in one of claims 7 to 12,<br>
characterized in that the sequence coding for at<br>
least one heterologous gene product is inserted in<br>
the viral genome and/or replaces sequences coding<br>
for a homologous gene product.<br>
14. The virus as claimed in one of claims 1 to 13,<br>
characterized in that the virus has a capacity for<br>
transcription that is reduced by at most a factor<br>
of 20 relative to the wild-type virus.<br>
15.	A nucleocapsid of a negative-strand RNA virus as<br>
 claimed in one of claims 1 to 14.<br>
16.	A genome of a negative-strand RNA virus as claimed<br>
in one of claims 1 to 14.<br>
17.	A DNA molecule that codes for the genome and/or<br>
antigenome of a recombinant negative-strand RNA<br>
 virus as claimed in one of claims 1 to 14.<br><br>
- 51 -<br>
18.	The DNA molecule as claimed in claim 17,<br>
characterized in that it is linked operationally<br>
with a transcription signal.<br>
19.	The DNA molecule as claimed in claim 18,<br>
 characterized in that the transcription signal is<br>
a bacteriophage promoter, e.g. a T7 or SP6<br>
promoter.<br>
20.	A cell that contains a virus as claimed in one of<br>
claims 1 to 14, a nucleocapsid as claimed in claim<br>
 15, a genome as claimed in claim 16 or a DNA<br>
molecule as claimed in one of claims 17 to 19.<br>
21.	The cell as claimed in claim 20, characterized in<br>
that it is a vector multiplying cell.<br>
22.	The cell as claimed in claim 20, characterized in<br>
 that it is a virus producing cell.<br>
23.	The cell as claimed in claim 22, characterized in<br>
that it further contains a DNA molecule coding for<br>
a heterologous DNA-dependent RNA polymerase, which<br>
effects the transcription of the DNA molecule<br>
 coding for the recombinant negative-strand RNA<br>
virus.<br>
24.	The cell as claimed in claim 20, characterized in<br>
that it is a virus multiplying cell.<br>
25.	The cell as claimed in one of claims 20 to 24,<br>
 characterized • in that it further contains DNA<br>
molecules coding for the viral L, N and/or P<br>
protein.<br>
26.	A method of production of a negative-strand RNA<br>
virus as claimed in one of claims 1 to 14,<br>
 comprising the steps:<br>
(a) preparation of a cell that is transfected<br><br>
- 52 -<br>
with a DNA molecule that codes for the genome<br>
of a negative-strand RNA virus, containing a<br>
mutation in at least one of the genes N, L<br>
and P, which leads to loss of the capacity<br>
 for viral genome replication without loss of<br>
the capacity for secondary transcription, and<br>
optionally at least one sequence coding for a<br>
heterologous gene product, and<br>
(b) cultivation of the cell under conditions such<br>
 that a transcription of the DNA according to<br>
(a) takes place and the recombinant negative-<br>
strand RNA virus is formed.<br>
27.	The method as claimed in claim 26, further<br>
comprising the obtaining of the nucleocapsid or of<br>
 the viral genome from the negative-strand RNA<br>
virus.<br>
28.	A method of multiplying a negative-strand RNA<br>
virus as claimed in one of claims 1 to 14,<br>
comprising the steps:<br>
 (a) preparation of a cell that is infected with a<br>
negative-strand RNA virus, containing a<br>
mutation in at least one of the genes N, L<br>
and P, which leads to loss of the capacity<br>
for viral genome replication without loss of<br>
 the capacity for secondary transcription, and<br>
optionally at least one sequence coding for a<br>
heterologous gene product, and<br>
(b) cultivation of the cell under conditions such<br>
that multiplication of the virus takes place.<br>
29. A pharmaceutical composition, characterized in<br>
that it contains a recombinant negative-strand RNA<br>
virus as claimed in one of claims 1 to 14, a<br>
nucleocapsid as claimed in claim 15 or a viral<br>
genome as claimed in claim 16 as active substance<br>
 and optionally pharmaceutically acceptable<br>
vehicles and/or excipients.<br><br>
- 53 -<br>
30.	The pharmaceutical composition as claimed in claim<br>
29	for use as vaccine.<br>
31.	The pharmaceutical composition as claimed in claim<br>
30	as monovalent or polyvalent vaccine.<br>
32. The pharmaceutical composition as claimed in claim<br>
30 or 31 as vaccine against viral infections, for<br>
example as vaccine against infections with<br>
pathogenic negative-strand RNA viruses.<br>
33.	The pharmaceutical composition as claimed in claim<br>
 29 for use for antitumor therapy.<br>
34.	The pharmaceutical composition as claimed in one<br>
of claims 29 to 33 for use in high-risk patients.<br>
35.	The pharmaceutical composition as claimed in one<br>
of claims 29 to 34, comprising the nucleocapsid in<br>
 the native viral envelope.<br>
36.	The use of a cell which stably expresses,<br>
constitutively or inducibly, the proteins N, L<br>
and/or P of a negative-strand RNA virus, for the<br>
production or multiplication of recombinant<br>
 negative-strand RNA viruses as claimed in one of<br>
claims 1 to 14, of nucleocapsids as claimed in<br>
claim 15 or of viral genomes as claimed in claim<br>
16.<br>
37.	The use as claimed in claim 36, characterized in<br>
 that the cell is a mammalian cell.<br>
38.	The use as claimed in claim 36 or 37,<br>
characterized in that the cell is selected from<br>
the H29 cell (DSM ACC2702) or a cell derived<br>
therefrom.<br><br>
The present invention relates to a genome-replication-<br>
deficient and transcription-competent negative-strand<br>
RNA virus, which can be used for the expression of<br>
transgenes and in particular for the area of vaccine<br>
development.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-correspondence others 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1wY3QgcmVxdWVzdCBmb3JtLnBkZg==" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-pct request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy1wcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDMwMjUta29sbnAtMjAwNy10cmFuc2xhdGVkIGNvcHkgb2YgcHJpb3JpdHkgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">03025-kolnp-2007-translated copy of priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwMi0wOS0yMDE0KS1BTUVOREVEIFBBR0VTLnBkZg==" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(02-09-2014)-AMENDED PAGES.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwMi0wOS0yMDE0KS1DTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(02-09-2014)-CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwMi0wOS0yMDE0KS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(02-09-2014)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwMy0wNy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(03-07-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwMy0wNy0yMDEyKS1TRVFVRU5DRSBMSVNUSU5HLnBkZg==" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(03-07-2012)-SEQUENCE LISTING.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwNi0wNy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(06-07-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgwNi0wNy0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(06-07-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgxOC0wNS0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(18-05-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgxOC0wNS0yMDEyKS1GT1JNLTMucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(18-05-2012)-FORM-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgyMC0wNy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(20-07-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgyMC0wNy0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(20-07-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgyNy0wNi0yMDEyKS1QRVRJVElPTiBVTkRFUiBSVUxFIDEzNy5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(27-06-2012)-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgyOC0xMC0yMDExKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(28-10-2011)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgyOC0xMC0yMDExKS1FTkdMSVNIIFRSQU5TTEFUSU9OLnBkZg==" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(28-10-2011)-ENGLISH TRANSLATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgzMC0wOS0yMDExKS1FWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDSUVWRUQucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(30-09-2011)-EXAMINATION REPORT REPLY RECIEVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LSgzMC0wOS0yMDExKS1QQ1QgSVBFUi5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-(30-09-2011)-PCT IPER.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUFTU0lHTk1FTlQucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-ASSIGNMENT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFIDEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFIE9USEVSUyAxLjIucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-CORRESPONDENCE OTHERS 1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFIE9USEVSUyAxLjMucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-CORRESPONDENCE OTHERS 1.3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFLnBkZg==" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1rb2xucC0yMDA3LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">3025-kolnp-2007-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUlOVEVSTkFUSU9OQUwgUFJFTElNSU5BUlkgUkVQT1JULnBkZg==" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-INTERNATIONAL PRELIMINARY REPORT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LUlOVEVSTkFUSU9OQUwgU0VBUkNIIFJFUE9SVCAxLjEucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-INTERNATIONAL SEARCH REPORT 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LU9USEVSUyAxLjEucGRm" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-OTHERS 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzAyNS1LT0xOUC0yMDA3LU9USEVSUy5wZGY=" target="_blank" style="word-wrap:break-word;">3025-KOLNP-2007-OTHERS.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263306-process-for-preparation-of-water-soluble-azole-prodrugs.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263308-method-of-mutagenesis.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263307</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3025/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Aug-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MAX-PLANCK GESELLSCHAFT ZUR FORDERUNG DER WISSENSCHAFTEN E.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HOFGARTENSTRASSE 8 80539 MUNCHEN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NEUBERT, WOLFGANG, J</td>
											<td>BAHNHOFSTRASSE 7B, 86926 GREIFENBERG</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SCHLECHT, SABINE</td>
											<td>PILARSTR.8, 80638 MUNCHEN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BOSSOW, SASCHA</td>
											<td>OBSTMARK 11, 86152 AUGSBURG</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/115</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/001251</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-02-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10 2005 006 388.8</td>
									<td>2005-02-11</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263307-a-recombinant-negative-strand-rna-virus-containing-a-viral-genome-with-a-mutation-in-the-p-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:29:20 GMT -->
</html>
