<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263435-fusion-proteins-comprising-flagellin-and-dengue-virus-envelope-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:51:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263435:&quot;FUSION PROTEINS COMPRISING FLAGELLIN AND DENGUE VIRUS ENVELOPE PROTEINS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;FUSION PROTEINS COMPRISING FLAGELLIN AND DENGUE VIRUS ENVELOPE PROTEINS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A fusion protein comprising at least a portion of at least one flagellin and at least a portion of at least one Den2 viral envelope protein, wherein the Den2 viral envelope protein is at least one member selected from the group consisting of SEQ ID NO: 22, SEQ ID NO: 40 and SEQ ID NO: 97, wherein the fusion protein elicits an immune response that neutralizes a Dengue 2 virus</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>COMPOSITIONS OF PATHOGEN-ASSOCIATED MOLECULAR PATTERNS AND METHODS OF USE<br>
RELATED APPLICATIONS<br>
This application claims the benefit of U.S. Provisional Application Nos. 60/645,170, filed January 19, 2005; 60/653,405, filed February 15, 2005; 60/704,160, filed July 29,2005; 60/723,409, filed October 4,2005; and 60/725,919, filed October 11. 2005. The teachings of the above applications are incorporated herein by reference.<br>
BACKGROUND OF THE INVENTION<br>
Infections with viruses, including fiaviviruses, such as West Nile flavivirus. Dengue flavivirus, Japanese encephalitis flavivirus. Langat flavivirus, Kunjin flavivirus, Murray Valley encephalitis flavivirus, Tick-borne flavivirus and Yellow fever flavivirus, can result in serious disease and, possibly death. Mosquitos and ticks transmit many of the flaviviruses. For example, severe symptoms of West Nile virus infection include high fever, headache, neck stiffness, stupor, disorientation, coma, tremors, convulsions, muscle weakness, vision loss, numbness, meningoencephalitis and paralysis. These symptoms may last several weeks, and neurological effects may be permanent. In cases with milder symptoms (e.g., fever, headache, and body aches, nausea, vomiting, and sometimes swollen lymph glands or a skin rash on the chest, stomach and back), certain symptoms, such as fever and aches, can pass on their own. In more severe cases, people usually require hospitalization for treatment, such as administration of intravenous fluids and assistance with breathing.<br>
Methods to prevent flavivirus infection include compositions of live attenuated and inactivated virus. However, such compositions may be less than optimally immunogenic,.may result in unknown hazards if improperly prepared and may have adverse side effects, There is a need to develop new compositions and methods to prevent flavivirus infection.<br>
SUMMARY OF THE, INVENTION<br>
The present invention relates to compositions, fusion proteins and polypeptides of at least a portion of an antigen and a flagellin that lacks a hinge region; and at least a portion of at least one pathogen-associated molecular pattern (PAMP) and at least a portion of at least one flavivirus. The compositions., fusion protein and polypeptides of the invention can be emploj'ed in methods to stimulate an immune response in a subject.<br>
In one embodiment, the invention is a composition comprising at least a portion of at least one antigen and at least a portion of at least one flagellin, wherein at least one of the flagellin lacks at least a portion of a hinge region.<br>
In another embodiment, the invention is a fusion protein comprising at least a portion of at least one antigen and at least a portion of at least one flageEin, wherein at least one of the flagellin lacks at least a portion of a hinge region.<br>
In an additional embodiment, the invention is a composition comprising at<br>
/ least a portion of at least one pathogen-associated molecular pattern and at least a<br>
portion of at least one viral protein selected from the group consisting of a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a tickbome encephalitis viral protein, and a Yellow fever viral protein.<br>
In yet another embodiment, the invention is a composition comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one Den2 viral envelope protein, wherein the Den2 viral envelope protein is at least one. member selected from the group consisting of SEQ ID NO: 22 and SEQ ID NO: 40.<br>
hi another embodiment, the invention is a composition comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In still another embodiment, the invention is a fusion protein comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one viral protein selected from the group consisting of a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a tickbome encephalitis viral protein, and a Yellow fever viral protein.<br>
An additional embodiment of the invention is a fusion protein comprising at least a portion of at least one member selected from the group consisting of a Salmonella typhimurium flagellin type 2 (fljB/STF2), an E. coli fliC, and a 5. muenchen fliC and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In another embodiment, the invention is a fusion protein comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
Another embodiment of the invention is a polypeptide encoded by SEQ ID NO; 29.<br>
In yet another embodiment, the invention is a polypeptide that includes SEQ ED NO: 30.<br>
In a further embodiment, the invention is a polypeptide having at least about 85% identity to SEQ ID NO: 30.<br>
In still another embodiment, the invention is a polypeptide encoded by SEQ ID NO: 31.<br>
In another embodiment, the invention is a polypeptide that includes SEQ ID NO: 32.<br>
In an additional embodiment, the invention is a polypeptide having at least about 70% identity to SEQ ID NO: 32.<br>
In yet another embodiment, the invention is a polypeptide encoded by SEQ ID NO: 33.<br>
In another embodiment, the invention is a polypeptide tliat includes SEQ ID NO: 34.<br>
hi still another embodiment, the invention is a polypeptide having at least about 70% identity to SEQ ED NO: 34.<br>
In an additional embodiment, the invention is a polypeptide encoded by SEQ ID NO: 35.<br>
In a further embodiment, the invention is a polypeptide that includes SEQ ID NO: 36.<br>
In yet another embodiment, the invention is a polypeptide having at least 80% identity to SEQ ID NO: 36.<br>
In another embodiment, the invention is a polypeptide encoded by SEQ ID NO: 37.<br>
In still another embodiment, the invention is a polypeptide that includes SEQ ID NO: 38.<br>
In another embodiment, the invention is a polypeptide having at least 70% identity to SEQ ID NO: 38.<br>
In an additional embodiment, the invention is a polypeptide encoded by SEQ ID NO: 54.<br>
In another embodiment, the invention is a polypeptide that includes SEQ ID NO: 55.<br>
Another embodiment of the invention is a polypeptide having at least about 70% identity to SEQ ID NO: 55.<br>
In still another embodiment, the invention is a polypeptide that includes at least one member selected from the group consisting of SEQ ID NO: 71 and SEQ ED NO: 72.<br>
In another embodiment, the invention is a polypeptide encoded by at least one member selected from the group consisting of SEQ ED NO: 70 and,SEQ ID NO: 73.<br>
In yet another embodiment, the invention is a polypeptide having at least about 70% identity to at least one member selected from the group consisting of SEQ ID NO: 71 and SEQ ID NO: 72. .<br>
In stilJ another embodiment, the invention .is a polypeptide that includes at least one member selected from the group consisting of SEQ ID NO: 76 and SEQ ID NO: 6.<br>
ID a further embodiment, the invention is a polypeptide encoded by at least<br>
one member selected from the group consisting of SEQ ID NO: 77 and SEQ ID NO:<br>
1<br>
In another embodiment, the invention is a polypeptide having at least about 70% identity to at least one member selected from the group consisting of SEQ ID NO: 76 and SEQ ID NO: 6.<br>
In an additional embodiment, the invention is a polypeptide that includes at least one member selected from the group consisting of SEQ ID NO: 80, SEQ ID NO: 82, SEQ ID NO: 84 and SEQ ID NO: 86.<br>
In still another embodiment, the invention is a polypeptide encoded by at least one .member selected from the group consisting of SEQ ID NO: 81, SEQ ID NO: 83, SEQ ID NO: 85 and SEQ ID NO: 87.<br>
In a further embodiment, the invention is a polypeptide having at least about 70% identity to at least one member selected from the group consisting of SEQ ID NO: 80, SEQ ID NO: 82, SEQ ID NO: 84 and SEQ ID NO: 86.<br>
In an additional embodiment, the invention is a polypeptide that includes SEQ ID NO: 159.<br>
In yet another embodiment, the invention is a polypeptide encoded by SEQ ID NO: 158.<br>
In another embodiment, the invention is a polypeptide having at least about 70% identity to SEQ E) NO: 159.<br>
In yet another embodiment, the invention is a composition comprising at least one PamSCys and at least a portion of at least one fiavivirus protein.<br>
In an additional embodiment, the invention is a composition comprising at least one Pam2Cys and at least a portion of at least one fiavivirus protein.<br>
In still another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a composition that includes at least a portion of at least one pathogen-associated<br>
molecular pattern and at least a portion of at least one viral protein selected from the group consisting of a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a tickborne encephalitis viral protein, and a Yellow fever virus protein.<br>
In a further embodiment, the invention is a method of stimulating an immune response in a subject comprising the step of administering to the subject the composition that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one Den2 envelope protein, wherein the Den2 envelope protein is selected from the group consisting of SEQ ID NO: 20 and SEQ ID NO: 40.<br>
In yet another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one viral protein selected from the group consisting of a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a tickborne encephalitis viral protein and a Yellow fever viral protein.<br>
hi another embodiment, the invention is a method stimulating an immune response in a subject, comprising'the step of administering to the subject a composition that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In still another embodiment, the invention is a method stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein that includes at least a portion of at least one member selected from the group consisting of &amp; Salmonella rypliimurium flagellin type 2 (3.JB/STF2), an E. colt fiiC. and a S. muenchen fliC and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In an additional embodiment, the invention is a method stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In a further embodiment, the invention is a method of stimulating an immune response hi a subject, comprising the step of administering to the subject a composition comprising at least a portion of at least one antigen and at least a portion of at least one flagellin, wherein at least one of the flagellins lack at least a portion of a hinge region.<br>
In another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein comprising at least a portion of at least one antigen and at least a portion of at least one flagellin., wherein at least one of the flaggelins lack at least a protion of a hinge region.<br>
The compositions, fusions proteins and polypeptides of the invention can be employed to stimulate an immune response in a subject. Advantages of the claimed invention can include, for example, prevention of flaviviras infection in a subject in a manner specific for a particular antigen or virus, such as a flavivirus protein, that has effective immunogencity and reduced side effects. The claimed compositions, fusion proteins, polypeptides and methods can be employed to prevent or treat infection and, therefore, avoid serious diseases consequent to antigen or viral infection.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 depicts the amirio acid sequence (SEQ ID NO: 1) ofSalmojiella typhimwiwn flagellin type 2 (fljB/STF2, also referred to herein as "STF2"). The hinge region (also referred to herein as "the hypervariable" or the "hvpervariable hinge region") is underlined.<br>
Figure 2 depicts the nucleic acid sequence (SEQ ED NO: 2) encoding SEQ ED NO: 1, The nucleic acid sequence encoding the hinge region is underlined.<br>
Figure 3 depicts the ammo acid sequence (SEQ ID NO: 3) of a fljB/STF2A (also referred to herein as "fljB/STF2A" or "STF2A"). STF2A is a STF2 lacking at least a portion of the hinge region. The artificial hinge region is underlined.<br>
Figure 4,depicts the nucleic acid sequence (SEQ ED NO: 4) encoding SEQ ED NO: 3. The nucleic acid sequence encoding the artificial hinge region is underlined.<br>
Figure 5 depicts the nucleic acid sequence (SEQ ID NO: 5) encoding a pET/STF2A.JEEI+ fusion protein. The nucleic acid sequence encoding the artificial hinge region is double underlined. The nucleic acid sequence encoding the linker between STF2A and JEHI+ is underlined. The nucleic acid sequence encoding JEIII- is bolded.<br>
Figure 6 depicts the amino acid sequence (SEQ ID NO: 6) encoded by SEQ ED NO: 5. The artificial hinge is double underlined. The linker between STF2A and JEII!-r is underlined. The amino acid sequence of JEIH+ is bolded.<br>
Figure 7 depicts the nucleic acid sequence (SEQ ID NO: 29) encoding a STF2.EIIR fusion protein. The nucleic acid sequence encoding the hinge region of STF2 is underlined.<br>
Figure 8 depicts the amino acid sequence (SEQ 3D NO: 30) encoded by SEQ ID NO: 29. The hinge region of STF2 is underlined.<br>
Figure 9 depicts the nucleic acid sequence (SEQ ID NO: 31) encoding a STF2A.EII1+ fusion protein. The naturally occurring hinge region of STF2 has been removed and replaced with an artificial hinge region. The nucleic acid sequence encoding the artificial hinge region is underlined. The nucleic acid sequence encoding EHI-t- is bolded.<br>
Figure 10 depicts the amino acid sequence (SEQ ID NO: 32) encoded by SEQ ID NO: 31. The artificial hinge region is underlined. The EEQ+ amino acid sequence is bolded.<br>
Figure 11 depicts the nucleic acid sequence (SEQ ED NO: 33) of a STF2A.EEQ+ fusion protein. The nucleic acid sequence encoding the artificial hinge<br>
region is double underlined. The nucleic acid sequence encoding a linker between STF2A and EIII4- is underlined. The nucleic acid sequence encoding EHI+ is bolded. Vector sequence is unbolded at the 3' end of the nucleic acid sequence. Figure 12 depicts the amino acid sequence (SEQ ID NO: 34) encoded by SEQ ID NO; 33. The artificial binge region is double underlined. The linker between STF2A and EIII+ is underlined. The amino acid sequence of the EEQ+ is bolded. Domain I of the West Nile virus protein is bolded and italicized (MEKLQ, SEQ ID NO: 172). The remainder of the bolded sequence<br>
(LKGTTYGVCSKAFKFLGTPADTGHGTWLELQYTGTDGPCKVPISSVASLN DLTPVGRLVTVNPFVSVATANAKVLIELEPPFGDSYTVVGRGEQQINHHWH KSGSSIGK, SEQ ID NO: 176) is domain III of the envelope protein of the West Nile virus. Vector sequence at the carboxy-terminus is not bolded at the carboxy-terminus.<br>
Figure 13 depicts the nucleic acid sequence (SEQ ID NO: 35) of a STF2.EIE+ fusion protein. The nucleic acid sequence encoding the hinge region of STF2 is underlined. The nucleic acid sequence encoding a linker between STF2 and EIIH- is bolded and underlined. The nucleic acid sequence encoding EIII+ is bolded.<br>
Figure 14 depicts the amino acid sequence (SEQ ID NO: 36) encoded by SEQ ID NO: 35. The hinge region is underlined. The linker between STF2 and Ein+ is bolded and underlined. The amino acid sequence of EIE+ is bolded.<br>
Figure 15 depicts the nucleic acid sequence (SEQ ID NO: 37) encoding a fljB/STF2A.EIII+fusion protein. There is no linker between STFA and EHI-K<br>
Figure 16 depicts the amino acid sequence (SEQ ID NO: 38) encoded by SEQ ID NO: 37. The amino acid sequence of EIH+ is bolded.<br>
Figure 17 depicts the nucleic acid sequence (SEQ ID NO: 54) of a fljB/STF2.EIII+ fusion protein. The nucleic acid sequence encoding the hinge region of STF2 is underlined. The nucleic acid sequence encoding a linker between STF2 and EIII+ is bolded and underlined. The nucleic acill sequence encoding EIII+ is bolded.<br>
Figure 18 depicts the amino acid sequence (SBQ ID NO: 55) encoded by SEQ ED NO: 54. The amino acid sequence of the hinge region of STF2 is underlined. The amino acid sequence of the linker between STF2 and EIII+ is bolded and underlined. The amino acid sequence of EUR is bolded.<br>
Figure 19 depicts the amino acid sequence (SEQ ID NO: 58) of Salmonella muenchen flagellin fliC. The amino acid sequence of the hinge region is underlined. Figure 20 depicts the nucleic acid sequence (SEQ ID NO: 59) encoding SEQ ID NO: 58. The nucleic acid sequence encoding the binge region is underlined. Figure 21 depicts the nucleic acid sequence (SEQ ID NO: 63) of a linker. Figure 22 depicts the amino acid sequence (SEQ ED NO: 64) of Hepatitis C El.<br>
'Figure 23 depicts the amino acid sequence (SEQ ED NO: 65) of Hepatitis C ,E2.<br>
Figure 24 depicts the nucleic acid sequence (SEQ ID NO: 66) encoding SEQ ED NO: 64.<br>
Figure 25 depicts the nucleic acid sequence (SEQ ID NO: 67) encoding SEQ ED NO: 65,<br>
Figure 26 depicts the amino acid sequence (SEQ ED NO: 68) ofE.Coli fliC. The amino acid sequence of the hinge region is underlined.<br>
Figure 27 depicts the nucleic acid sequence (SEQ ED NO: 69) encoding SEQ ED NO: 68. The nucleic acid sequence encoding the hinge region is underlined. Figure 28 depicts the nucleic acid sequence (SEQ ED NO: 70) encoding a fljB/STF2A.EnR fusion protein. The nucleic acid sequence encoding the artificial binge region is double underlined. The nucleic acid sequence encoding a linker between STF2A and EUR is underlined. The nucleic acid sequence encoding the EIII+ is bolded. Vector sequence is not bolded at the 3' end of the sequence.<br>
Figure 29 depicts the amino acid sequence (SEQ ID NO: 71) encoded by SEQ ID NO: 70. The artificial hinge region is double underlined. The amino acid sequence of the linker between STF2A and EIER is underlined. The amino acid sequence of the EIER is bolded. Vector sequence at the carboxy-terminus is not bolded.<br>
Figure 30 depicts the amino acid sequence (SEQ .ID NO: 72) of a fljB/ STF2A.ElGs-r fusion protein. The artificial hinge region is double underlined. The amino acid sequence encoding the linker between STF2A and EHI+ is underlined. Domain 1 of the West Nile virus protein is bolded and italicized (SEQ ID NO: 172). The remainder of the bolded sequence is domain HI of the envelope protein (SEQ ED NO: 176) of the West Nile virus. Portions of domains I and in are referred to as EIII+. Vector sequence at the carboxy-terminus of the protein is unbolded. The serine residue of the linker region is bolded and is a substitution of the cysteine residue in the same region of the linker of SEQ ID NO: 71 of Figure 29.<br>
Figure 31 depicts the nucleic acid sequence (SEQ ID NO: 73) encoding SEQ ID NO: 72. The nucleic acid sequence encoding the artificial hinge region is double underlined. The nucleic acid sequence encoding the linker between STF2A and Elll-f is underlined with the codon encoding the serine residue bolded. The nucleic acid sequence encoding EIH+ is indicated by bolded text. Linker sequence is unbolded text at the 3s end.<br>
Figure 32 depicts the amino acid sequence (SEQ ID NO: 76) of a pET/STF2A.JEIII+ fusion protein. The artificial hinge region is double underlined. The amino acid sequence of the linker between STF2A and JEHI+ is underlined. The amino acid sequence of a portion of domain I of the Japanese encephalitis virus is bolded and italicized (MDKLAL, SEQ ID NO: 173). The amino acid sequence of a portion of the domain III of the Japanese encephalitis virus is bolded (KGTTYGMCTEKTSFAJCNPVDTGHGTA^VIELSySOSDGPCKIPrVSVASLND MTPVGRL\T\'M)FVATSSANSKVLMEPPFGDSYIVVGRGDKQ^NHHWH KAGSTLGKA, SEQ ED NO: 177). Portions of domains I and III are referred to as "JEII1+."<br>
Figure 33 depicts the nucleic acid sequence (SEQ ID NO: 77) encoding SEQ ID NO: 76. The nucleic acid sequence encoding the artificial hinge region is double underlined. The nucleic acid sequence encoding a linker between STF2A and JEIII+ is underlined. The nucleic acid sequence encoding a portion of domain I of the Japanese encephalitis virus is bolded and italicized. The nucleic acid sequence<br>
encoding a portion of domain III of the Japanese encephalitis virus is bolded. P onions of domains I and IE are referred to as "JEIII+."<br>
Figure 34 depicts the nucleic acid sequence (SEQ ID NO: 78) encoding JEER. The nucleic acid sequence encoding at least a portion of domain I of the envelope protein is underlined. The remaining nucleic acid sequence encodes at least a portion of domain III of the -envelope protein.<br>
Figure 35 depicts the amino acid sequence (SEQ ED NO: 79) encoded by SEQ ED NO: 78. At least a portion of domain I of the envelope protein is bolded and italicized. The remaining sequence is at least a portion of domain IH of the envelope protein.<br>
Figure 36 depicts the amino acid sequence (SEQ ED NO: 80) of a pET/STF2A.Denl EIII fusion protein. The artificial hinge region is double underlined. A linker between STF2A and Denl EEI is underlined.<br>
Figure 37 depicts the nucleic acid sequence (SEQ ID NO: 81) encoding SEQ ED NO: 80, The nucleic acid sequence encoding the artificial hinge region is double underlined. The nucleic acid sequence encoding the linker between STF2A and Denl EIIl is underlined.<br>
Figure 38 depicts the amino acid sequence (SEQ ED NO: 82) of a pET/STF2A.Den2 EIII fusion protein. The artificial Mnge region is double underlined. The amino acid sequence of the linker between STF2A and Den2 EIII is underlined.<br>
Figure 39 depicts the nucleic acid sequence (SEQ ID NO: 83) encoded by SEQ ED NO: 82. The nucleic acid sequence encoding the artificial hinge region is double underlined. The nucleic acid sequence encoding the linker between STF2A and Den2 EIII is underlined.<br>
Figure 40 depicts the amino acid sequence (SEQ ID NO: 84) of a pET/STF2A.Den3 EIII fusion protein. The artificial hinge region is double underlined. The amino acid sequence of the linker between STF2A and.Den3 EIII is underlined.<br>
Figure 41 depicts the nucleic acid sequence (SEQ ID NO: 85) encoding SEQ ID NO: 84. The nucleic acid sequence encoding the artificial hinge region is double<br>
underlined. The nucleic acid sequence encoding the linker between STF2A and Den3 EEI is underlined.<br>
Figure 42 depicts the amino acid sequence (SEQ ID NO: 86) of a pET/STF2A.Den4 EIII fusion protein. The artificial hinge region is double underlined. The amino acid sequence of the linker between STF2A and Den4 EIII is underlined.<br>
Figure 43 depicts the nucleic acid sequence (SEQ ID NO: 87) encoding SEQ ED NO: 86. The nucleic acid sequence encoding the artificial hinge region is double underlined. The nucleic acid sequence encoding the linker between STF2A and Den4 EIII is underlined.<br>
Figure 44 depicts the amino acid sequence (SEQ ID NO: 174) of the envelope protein of the Tick-borne encephalitis envelope protein.<br>
Figure 45 depicts the amino acid sequence (SEQ ID NO: 39) of a West Nile virus envelope protein (WNE) (amino acids 1-406). The amino acid sequence incorporated into EIII+ constructs is underlined (amino acids 292-406). Amino acids 292-297 correspond to a portion of domain I; amino acids 298-406 correspond to domain HI. SEQ ID NO: 39 is encoded by SEQ ID NO: 57 (Figure 67).<br>
Figure 46 depicts fusion contracts in a pET24 vector. T7:T7 promoter; lacO: lac operator; STF2: Salmonella tjphimurium fiagellin; STF2A= STF2 with the hinge region deleted; Enf' is domain El of a West Nile envelope protein with 6 ammo acids of domain I amino acid.<br>
Figures 47A and 47B depict TLR-5 bioactivity of STF2.EHI+ (SEQ ED NOS: 54, 55) and STF2AEffl+ (SEQ ID NOS: 70, 71) fusion proteins. Serial dilutions of purified proteins were added to HEK293 (TLR5-K) cells overnight and IL-8 content of the supematants measured by ELISA. Purified STF2.0VA was used as a positive control (Figure 47A). • The TLR-2 agonist Pam3CSK4 was used as a negative control (Figure 47B).<br>
Figure 48 depicts STF2A.EH3+ antigenic epitopes assessed by ELISA. Plates were coated with full-length WNE (open bars) (SEQ ID NO: 39) or STF2A.EID+ (SEQ ID NOS: 70, 71) and probed with the indicated antibodies (mAb). Poly = polyclonal antiserum to WNE; 3D9 through 7H2 = neutralizing<br>
monoclonal antibodies to WNE epitopes; anti-flagellin = monoclonal antibody to fkgellin.<br>
Figures 49A, 49B, 49C and 49D depict reactivity of STF2.E (SEQ E) NOS: 158, 159); STF2.EEI+ (SEQ ID NOS: 54,. 55) and STF2A.EEI+ (SEQ ID NOS: 70, 71) fusion proteins with antibodies to WNE and flagellia. Plates were coated with fusion proteins, blocked and incubated with antibodies to WNE or flagellin. Antibody reactivity was detected following incubation with HRP -labeled species specific IgG. Plates were developed in the presence of TMB substrate and O.D.450/650 using a TECAN plate reader and Magellian software.<br><br>
Figure 50 depicts IgG serum following injection with fusion proteins. Mice were immunized with either PBS, Drosophila conditioned medium containing STF2.E (CM., positive control), 25 ug of STF2A.EIII+ (SEQ E) NOS: 70, 71) i.p., 5 ng STF2A.EHI+ s.c., 25 u.g STF2.EIE&gt; (SEQ E) NO: 54, 55) i.p., 25 ug STF2.EIII+ (SEQ ID NOS: 54, 55) or 25ug STF2.E (SEQ E) NOS: 158, 159). On day 35, immunized animals were challenged with WNV. Sera from individual mice (day 35) were characterized by direct ELISA to determine IgG levels. Purified WNV-E protein (SEQ ID NO: 39) was used as the antigen in this assay. This antigen (60) was produced in Drosophila as a bis-tagged protein.<br>
Figure 51 depicts STF2A.EIII+ (SEQ E) NOS: 70, 71) and STF2.Ein+ (SEQ ID NOS : 54, 55) protective immunity to WNV viral challenge. Mice were immunized and challenged with a lethal dose of WNV strain 2741 on day 35. Survival was monitored for 21 days.<br>
Figure 52 depicts IgG seratiters following immunization with fusion proteins. STF2A.EHI+ proteins induce WNV-specific IgG antibodies. Mice were immunized s.c. on days 0, 14 and 28 with PBS alone or about 25 ug of STF2A.EIIH-(SEQ ID NOS: 70, 71) (045 [positive control]), STF2A.EHI4- (067, trimer), STF2A.EIII+ (070, monomer) or STF2A.EEIs+ (SBQ ID NOS: 72, 73) (069). On day 35 sera from individual mice were characterized by direct ELISA to determine IgG levels. Purified WNV-E protein (060, produced in Drosophila as a bis-tagged protein) was used as the antigen in this assay.<br>
Figure 53 depicts STF2A.EIH+ (SEQ IDNOS: 70, 71) and STF2A.EIIIs-(SEQ ID NOS: 72, 73) protective immunity in mice from WNV lethal challenge. On day 38 following immunization with fusion proteins, all groups were challenged with a lethal dose of WNV strain 2741 and survival was monitored for 21 days. Survival for each group (10 mice/group) is indicated as a percentage.<br>
Figure 54 depicts competition assays. Serial dilutions (five fold starting at 1:25) of antisera from immunized animals were incubated with biotinylated WNE protein (SEQ ID NO: 39) and then added to the wells of ELISA plates coated with mAb 7H2 at about 2mg/ml. Wells were developed using avidin-HRP to determine inhibition of West Nile protein binding as a results of competition with mAb 7H2.<br>
Figure 55 depicts epitope mapping of the antibody response induced by STF2A.Eni-f (SEQ ID NOS: 72, 73) fusion proteins. -Immune sera from animals immunized with indicated STF2A-fusion proteins (E2-21, E27-E52, Figure 60) were examined for the ability to recognize overlapping peptides corresponding to the junction of domains I and HE of the WNV envelope protein.<br>
Figure 56 depicts epitope mapping of the antibody response induced by STFA.EIIIs+ (SEQ ID NOS: 72, 73) E-21 (envelope protein) epitope fusion proteins, hnmune sera from animals immunized with the indicated STF2A-fusion proteins (E2-21, E2-21-1(S,C), E2-21-2(C,S), E2-21-2(C,S) and E2-21-4 through E2-21-24, see Figure 57) were evaluated to identify the residues defining the E-21 epitope of West Nile envelope protein. Data reflects the response of sera to E-21 following the substitution of cysteine with serine (indicated by C.S); and the sequential replacement of amino acids with alanine. The peptides tested are listed in Figure 57.<br>
Figure 57 depicts EHI-f peptide arrays. The sequences include domains land III of the West Nile virus envelope protein. Amino acids that correspond to domain 'HI are underlined. Amino acids that are not underlined correspond to domain I. Figure 58 depicts PamSCys.WNVOOl (SEQ ID NO: 168) inducing EHI specific IgG antibodies. Mice were immunized s.c. on days 0, 14 and 28 with PBS alone, 22mg of unmodified WNV001 (SEQ ID NO: 168) or 30 ug of<br>
PamSCys.WNVOOl. On day 35 sera from individual mice were .characterized by direct ELISA to determine IgG levels to synthetic WNV001 peptide.<br>
Figure 59 depicts the amino acid sequences (SEQ ID NOS: 88-95) of the EI/EIII junction for West Nile, Japanese encephalitis and Dengue (serotypes 1 through 4) viruses. The West Nile epitope identified using antisera from STF2A.Enis+ immunized animals is underlined. This sequence corresponds to peptide E2-21 (SEQ ID NO: 125).<br>
Figure 60 depicts E2-21 peptide (SEQ ED NOS: 125-151) alanine scan array. Amino acids that correspond to domain III of the West Nile virus envelope protein are underlined. Amino acids that are not underlined correspond to domain I of the West Nile virus.<br>
Figure 61 depicts a STF2.0VA nucleic acid sequence (SEQ ID NO: 152). The nucleic acid sequence encoding the linker between STF2 and ovalbumin (OVA) is underlined. Vector sequence at the 3' end is bolded and underlined.<br>
Figure 62 depicts an amino acid sequence (SEQ ED NO: 153) encoded by SEQ ED NO: 152. The linker sequence between STF2 and OVA is underlined. Vector sequence is underlined and bolded.<br>
Figure 63 depicts the amino acid sequence (SEQ ID NO: .154) of ovalbumin. Figure 64 depicts the nucleic acid sequence (SEQ ED NO: 155) of ovalbumin.<br>
Figure 65 depicts the nucleic acid sequence (SEQ ED NO: 158) encoding a STF2.E fusion protein. The nucleic acid sequence encoding the full-length West Nile virus envelope protein (E) is underlined.<br>
Figure 66 depicts the amino acid sequence (SEQ ID NO: 159) encoded by SEQ ED NO: 158. The amino acid sequence of the West Nile virus envelope protein is underlined.<br>
Figure 67 depicts the nucleic acid sequence (SEQ ED NO: 57) encpding SEQ ID NO: 39 (Figure 45). The full length sequence of the West Nile virus envelope protein is depicted.<br>
Figure 68 depicts the amino acid sequence (SEQ ED NO: 160) of the Dengue 1 virus (also referred to herein as "Den-1," "Den 1" or "Deal").<br>
Figure 69 depicts the nucleic acid sequence (SEQ ID NO: 161) encoding SEQIDNO: 160.<br>
Figure 70 depcits the amino acid sequence (SEQ ID NO: 162) of the Dengue<br>
2	virus (also referred to herein as "Den-2," "Den 2" or "Den2").<br>
Figure 71 depicts the nucleic acid sequence (SEQ ID NO: 163) encoding SEQ ID NO: 162.<br>
Figure 72 depicts the amino acid sequence (SEQ ID NO: 164) of the Dengue<br>
3	virus (also referred to herein as "Den-3," "Den 3" or "Den3"),<br>
Figure 73 depicts the nucleic acid sequence (SEQ ID NO: 165) encoding SEQIDNO: 164).<br>
Figure 74 depicts the^amino acid sequence (SEQ ID NO: 166) of the Dengue<br>
4	virus (also referred to here in as "Den-4," "Den 4" or "Den4").<br>
Figure 75 depicts the nucleic acid sequence (SEQ ID NO: 167) encoding SEQ ID NO: 166.<br>
Figure 76 depicts the nucleic acid sequence (SEQ ID NO: 170) encoding a Japanese encephalitis virus.<br>
Figure 77 depicts the amino acid sequence (SEQ ED NO: 171) encoded hy SEQ ID NO: 170.<br>
Figure 78 depicts the nucleic acid sequence (SEQ ID NO: 175) encoding SEQ ID NO: 174, depicted in Figure 44.<br>
Figure 79 depicts the nucleic acid sequence (SEQ ED NO: 178) encoding E1I1+ (amino acids of 292-406 of SEQ ID NO: 39, depicted in Figure 45 and SEQ ID NO: 7).<br>
Figure 80 depicts a tripalrnitoylatedpeptide.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The features and other details of the invention, either as steps of the<br>
invention or as a combination of parts of the invention, will now be more particularly described and pointed out in the claims. It will be understood that the particular embodiments of the invention are shown by way of illustration and not as limitations of the invention. The principle features of this invention can be<br><br>
employed in various embodiments without departing from the scope of the invention.<br>
The present invention relates to compositions, fusion proteins and poiypeptides of at least a portion of at least one antigen and at least a portion of a flagellin that lacks a hinge region; and at least a portion of at least one pathogen-associated molecular pattern (PAMP) and at least a portion of at least one flavivirus. The compositions, fusion proteins and poiypeptides of the invention can be employed in methods to stimulate an immune response in a subject.<br>
In one embodiment, the invention is a composition comprising at least a portion of at least one antigen and at least a portion of at least one flagellin, wherein at least one of the flagellins lacks at least a portion of a binge region.<br>
Pathogen-associated molecular pattern (PAMP), such as a flagellin or a<br>
bacterial lipoprotein, refers to a class of molecules (e.g., protein, peptide,<br>
. carbohydrate, lipid, lipopeptide, nucleic acid) found in microorganisms that, when<br>
bound to a pattern recognition receptor (PRR), can trigger an innate immune<br>
response. The PRR can be a Toll-like receptor (TLR). Toll-like receptors refer to a<br>
family of receptor proteins that are homologous to the Drosophila melangogaster<br>
Toll protein. Toll-like receptors are type I transmembrane signaling receptor<br>
proteins characterized by an extracellular leucine-rich repeat domain and an<br>
intracellular domain homologous to an interleukin 1 receptor. Toll-like receptors<br>
include TLR1, TLR2, TLR3, TLR4, TLR5, TLR6, TLR7, TLR 8, TLR9, TLR10,<br>
TLRllandTLR12.<br>
The pathogen-associated molecular pattern can be an agonist of a toll-like receptor, for example, a TLR2 agonist (i.e., Pam2Cys, Pam3Cys, a bacterial lipoprotein) or a TLR5 agonist, such as a flagellin. "Agonist," as used herein hi referring to a TLR, means a molecule that activates a TLR signaling pathway. A TLR signaling pathway is an intracellular signal transduction pathway employed by a particular TLR that can be activated by a TLR ligand or a TLR agonist. Common intracellular pathways are employed by TLRs and include, for example, NF-xfi, Jun N-terminal kinase and mitogen-activated protein kinase. The pathogen-associated molecular pattern can include at least one member selected from the group<br>
consisting of a TLR1 agonist, a TLR2 agonist (e.g., PamSCys, Pam2Cys, bacterial lipoprotein), a TLR3 agonist (e.g., dsRNA). a TLR4 agonist (e.g., bacterial lipopolysaccharide). a TLR5 agonist (e.g., a flagellin). a TLR6 agonist, a TLR7 agonist, a TLR8 agonist, a TLR9 agonist (e.g., unmethylated DNA motifs), TLR10 agonist, a TLR11 agonist and a TLR12 agonist.<br>
In a particular embodiment, the TLR2 agonist is'a bacterial lipoprotein, such as Pam2Cys, Pam3Cys or Pseudomonas aeiiiginosa OprI lipoprotein (OprI). Exemplar)' OprI lipoproteins include MNNVLKFSALALAAVLATGCSSH (SEQ ID NO: 179), encoded by<br>
ATGAAAGCTACTAAACTGGTACTGGGCGCOGTAATCCTGOGTTCTACTCT GCTGGCAGGTTGCTCCAGCAAC (SEQ ID NO: 180). An exemplar)' E.coli bacteria] lipoprotein for use in the invention described herein is MKATKLVLGAVILGSTLLAGCSSN (SEQ ID NO: 181) encoded by ATGAAAGCTACTAAACTGGTACTGGGCGCGGTAATCCTGGGTTCTACTCT GCTGGCAGGTTGCTCCAGCAAC (SEQ ID NO: 182). A bacterial lipoprotein that activates a TLR2 signaling pathway (a TLR2 agonist) is a bacterial protein that includes apalmitoleic acid (Omueti, K.O., etal., J. Biol. Chem. 280: 36616-36625 (2005)). For example, expression of SEQ ID NOS: 180 and 182 in bacterial expression systems (e.g., E.coli) results in the addition of a palmitoleic acid moiety to a cysteine residue of the resulting protein (e.g., SEQ ID NOS: 179, 181) thereby generating a TLR2 agonist for use in the compositions, fusion proteins and polypeptides of the invention. Production of tripahnitoylated-lipoproteins (also referred to as triacyl-lipoproteins) in bacteria occurs through the addition of a diacylglycerol group to the sulfhydryl group of a cysteine (Cysteine 21 of SEQ ID NO: 181) followed by cleavage of the signal sequence and addition of a third acyl chain to the free N-terminal group of the same cysteine (Cysteine 21 of SEQ ID NO: 181) (Sankaran, K., et al, J. Biol. Chem. 26P/19706 (1994)), to generate a tnpalmitylated peptide (a TLR2 agonist) as shown,.for example, in Figure 80.<br>
An antigen is any molecule (e.g., protein, peptide, glycoprotein, glycopeptide, carbohydrate, lipid, lipopeptide, polysaccharide) that generates an immune response in a subject either when employed in combination with a PAMP<br>
 (e.g., a flagellin, Pam2Cys. Pam3Cys) or in the absence of a PAMP. The antigen can be a fragment or portion of a naturally occurring antigen or a synthetic molecule that mimics the naturally occurring antigen or a portion of the naturally occurring antigen.<br>
The antigen can be a viral antigen. A "viral antigen," as used herein, refers to any portion of a virus (e.g., fiavivirus) that generates an immune response in a. subject either when employed in combination with a PAMP (e.g., a flagellin, Pam2Cys, PamSCys) or in the absence of a PAMP, The viral antigen can be a portion or a fragment of a naturally occurring virus or a synthetic molecule that mimics a naturally occurring virus, such as a recombinant or synthetic protein (e.g., a fiavivirus), peptide, lipid, carbohydrate, that generates an immune response in the subject. "At least a portion," as used herein in reference to at least a portion of an antigen (e.g., a viral antigen), means any part of the antigen or the entirety of the antigen. For example, at least a portion of a flaviviral antigen can be an envelope protein, or a domain (e.g., domain 1, II, HI) of an envelope protein of a flavivirus antigen.<br>
The flagellin employed in the compositions, fusion proteins and polypeptides of the invention can lack at least a portion of a hinge region. Hinge regions are the hypervariable regions of a flagellin that link the ammo-terminus and carbo\7-terminus of the flagellin. Hinge regions-of a fiagellin are also referred to herein as "hypervariable regions" or "hypervariable hinge regions." "Lack," as used herein in reference to a hinge region of a flagellin. means that at least one ammo acid or at least one nucleic acid codon encoding at least one amino acid that comprises the hinge region of a flagellin is absent in the flagellin. Example of hinge regions include amino acids 176-415 of SEQ ID NO: 1, which are encoded by nucleic acids 528-1245 of SEQ ID NO: 2; amino acids 174-422 of SEQ ID NO: 68, which are encoded by nucleic acids 522-1266 of SEQ ID NO: 69; or amino acids 173-464 of SEQ ED NO: 58, which are encoded by nucleic acids 519-1392 of SEQ ID NO: 59. Thus, if amino acids 176-415 were absent from the flagellin of SEQ ID NO: 1, the flagellin would lack a hinge region. A flagellin lacking at least a portion of a hinge region is also referred to herein as a "truncated version" of a flagellin.<br>
"At least a portion of a hinge region," as used herein, refers to any part of the hinge region of the PAMP (e.g., flagellin). or the entirety of the hinge region. "At least a portion of a hinge region" is also referred to herein as a "fragment of a hinge region." For example, the hinge region ofS, typhimurium flagellin B (fljB, also referred to herein as "fljB/STF2" or "STF2") is amino acids 176-416 of SEQ ID NO: 1, which is encoded by nucleic acids at position 528-1245 of SEQ ID NO; 2. At least a portion of the hinge region of fijB/STF2 can be. for example, amino acids 200-300 of SEQ ID NO: 1. Thus, if amino acids 200-300 were absent from SEQ ID NO: 1, the resulting amino acid sequence of STF2 would lack at least a portion of a hinge region.<br>
At least a portion of a naturally occurring a flagellin can be replaced with at least a portion of an artificial hinge region. "Naturally -occurring." as used herein in reference to a hinge region of a flagellin, means the hinge region that is present ifi the native flagellin. For example, amino acids 176-415 of SEQ ID NO: 1. amino acids 174-422 of SEQ ED NO: 68 and ammo acids 173-464 of SEQ ID NO: 58, are the ammo acids corresponding to the natural hinge region of STF2, E. coli fliC and S. muenchen flagellins, fliC, respectively. "Artificial," as used herein in reference to a hinge region of a flagellin, means a hinge region that is inserted in the native flagellin in any region of the flagellin that contains or contained the native hinge region. For example, SEQ ID NO: 32 lacks the natural!)'- occurring hinge region, which has been replaced by amino acids 176-186, the artificial hinge region..<br>
An artificial hinge region may be employed in a flagellin that lacks at least a portion of a hinge region to facilitate interaction of the carboxy-and amino-terminus of the flagellin for binding to TLR5 and, thus, activation of the TLR5 innate signal transduction pathway. A flagellin lacking at least a portion of a hinge region is designated by the name of the flagellin followed by a "A." For example, an STF2 (e.g., SEQ ID NO: 1) that lacks at least a portion of a hinge region is referenced to as "STF2A" or "fljB/ STF2A" (e.g., SEQ ED NO: 3).<br>
•The flagellin'employed in the compositions, fusion proteins andpolypeptides of the invention can be at least one member selected from the group consisting of fljB/STF2 (S. typhimurium flagellin B, Genbanlc Accession. Number AF045151). a<br>
fragment of fijB/STF2, E. coli flagellin.fliC (also referred to herein as E. coli fliC) (Genbank Accession Number AB028476), a fragment of E. coli fiagellin fliC, S. muencheii flagellin fiiC (also referred to herein as S. muenchen fliC), and a fragment of S. muenchen flagellin fliC.<br>
The flagellin employed in the compositions, fusion proteins and polypeptides of the invention include the polypeptides of SEQ ID NO: 1, SEQ ID NO: 3, SEQ ID NO: 58 and SEQ ID NO: 68; at least a portion of SEQ ID NO: 1, at least a portion of SEQ ID NO: 3, at least a portion of SEQ ID NO: 58 and at least a portion of SEQ ID NO: 68; and a polypeptide encoded by SEQ ID NO: 2, SEQ ID NO: 4, SEQ ID NO: 59 and SEQ ID NO: 69; or at least a portion of a polypeptide encoded by SEQ ID NO: 2, SEQ ID NO: 4, SEQ ID NO: 59 and SEQ ID NO: 69.<br>
In another embodiment, the invention is a fusion protein comprising at least a portion of at least one antigen and at least a portion of at least one flagellin, wherein at least one of the flagellins lack at least a portion of a hinge region,<br>
"Fusion protein." as used herein, refers to a protein generated from at least two similar or distinct components (e.g., Pam2Cys, Pam3Cys, PAMP, at least a portion of an antigen, at least a portion of a viral protein) that are linked covalentiy or noncovalently. The components of the fusion protein can be made, for example, synthetically (e.g., PamSCys, Pam2Cys) or by recombinant nucleic acid techniques (e.g., transfection of a host cell with a nucleic acid sequence encoding a component of the fusion protein, such as at least a portion of a PAMP, or at least a. portion of an; • antigen or a viral protein). One component of the fusion protein (e.g., Pam2Cys, PamSCys, PAMP, at least a portion of an antigen or at least a portion of a viral protein) can be linked to another component of the fusion protein (e.g., Parn2Cys, PamSCys, PAMP, at least a portion of an antigen or at least a portion of a viral protein) using chemical conjugation techniques, including peptide conjugation, or using molepular biological techniques; including recombinant technology, such as the generation of a fusion protein construct. Chemical conjugation (also referred to herein as "chemical coupling") can include conjugation by a reactive group, such as a thiol group (e.g., a cysteine residue) or by derivatization of a primary (e.g., a amino-terminal) or secondary (e.g., lysine) group. Exemplary fusion proteins of the<br>
inventiori include SEQ ID NOS: 6, 71, 72, 76: 80.. 82,. 84, 86 and 159 (Figures 6,29, 30, 32r 36, 38,40 and 42), encoded by SEQ ID NOS: 5: 70, 73, 77, 81, 83, 85, 87 and 158 (Figures 5, 28, 31,33, 37, 39, 41 and 43)<br>
Fusion proteins of the invention can be designated by components of the fusion proteins separated by a "." or "-." For example, "STF2.EIII" refers to a fusion protein comprising one fijB/STF2 protein and at least a portion of domain III (see, infra) of at least one West Nile virus envelope protein; and "STF2A.EDT refers to a fusion protein comprising one fljB/STF2 protein lacking at least a portion of its hinge region and having at least a portion of domain HI of at least one West Nile virus envelope protein.<br>
In yet another embodiment, the invention is a composition comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one viral protein selected from the group consisting of &amp; West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-borne encephalitis viral protein, and a Yellow fever viral protein, which are flaviviral proteins. The pathogen-associated molecular pattern and viral protein can be components of a fusion protein.<br>
The genus flavivirus is in the virus family Flaviviridae and consists of about 70 viruses. Mosquito or ticks transmit most of these viruses. Several flaviviruses are significant human pathogens, including the four dengue viruses (Denl, Den2. Den3 and Den4), yellow fever (YF), Japanese encephalitis (JE), West Nile (WN, also referred to herein as "WNV") and Tick-boms encephalitis (TBE) (Weaver S.C.. et al, Nat Rev Microbiol 10: 789-801 (2004)). The flavivirus genus is divided into a number of serogroups based on cross-neutralization tests, including the dengue serogroup that contains four serologically and genetically distinct viruses termed DEN-1, DEN-2, DEN-3 and DEN-4.<br>
Flaviviruses are small, enveloped viruses with icosahedral capsids. The flavivirus genome is a single-stranded positive-sense RNA (about 11 kb) that is directly translated by the host cell machinery following infection. The viral genome is Translated as a single polypeptide that undergoes co- and post-translational<br>
cleavage by viral and cellular enzymes to generate three structural proteins of the flavivirus (the capsid (C), the membrane (M) and the envelope (E) proteins); and sevennonstructuralproteins (NS1, NS2A: NS2B, NS3, NS4A, NS4B.. and NS5) (Weaver, et al., Annu Rev Microbiol 1990:44-649 (2004)). The flavivirus genome organization is depicted in Figure 44. The viral capsid is composed of the C-protein, while both the M- and envelope proteins are located on the envelope surface of the virion (Weaver, S.C., et al., Nat. Rev. Microbiol 70:789-801 (2004);, Chambers et al, Annu Rev. Microbiol. 44: 649-688 (1990)). A major immunogen for flaviviruses is the membrane envelope protein.<br>
The flavivirus envelope protein plays a role in virus assembly. These proteins form a protective shell around the virus, which serves as a cage for the genetic material inside, sheltering the virus until it is released inside a host cell. While simple viruses consist of only a protein shell and genetic information, more complex viruses, such as flaviviruses, also contain a lipid bilayer between the protein shell and viral genome. A flavivirus can enter a host cell when the viral envelope protein binds to a receptor and responds by conformational rearrangement to the reduced pH of an endosome. The conformational change induces fusion of viral and host-cell membranes.<br>
The envelope of a flavivirus may function as a receptor binding protein and to facilitate fusion of the virus and host cell membrane. As a receptor binding protein, the envelope protein is a determinant of host range, cell tropism, virulence and elicits neutralizing antibodies during the immune response (Roehrig, Adv Virus Res 55:141-175 (2003)). The envelope protein is responsible for fusing the virus and host membranes (Chu, et al., J. Virol 78:10543-10555 (2004); Heinz, et al., Adv Virus Res 59:63-91 (2003); Chu: et al, J. Gen Virol 86:405-412 (2005)). Crystallograpbic structures of the Tick-borne encephalitis virus envelope protein and the Dengue-2 (Den 2) virus envelppe protein have been determined (Rey, et al, Nature 575:291-298(1995.); Modis, etal, Proc Natl AcadSci USA 100:6986-6991(2003)). Envelope proteins of flaviviruses have common structural (domains I, II and IE) and functional features (receptor binding of virus and host cell and fusion<br>
functions) and are class II fusion glycoproteins (Lescar etal, Cell 105.'137-148 (2001)).<br>
In the pre-fusion conformation, envelope proteins form homodimers on the outer surface of the virus particles (Rey, et al, Nature 575:291-298); Kuiui, et al, Cell 108:111-125 (2002); Mukhopadhyay, et al, Science 502:248 (2003)). Each envelope protein monomer folds into three structural domains (domains I. H and IE) predominantly composed of (3-strands. Domain I (also referred to herein as "P or "DI") is centrally located in the structure and has an N-glycos3'lation site in glycosylated envelope proteins. Domain II (also referred to herein as "IP"' or "DH") of the envelope protein promotes dimerization and has a fusion loop that inserts into the target host membrane during the pH-dependent fusion of the virus (Modis, et al, Nature ¥27:313-319 (2004); Bressanelli, et al, EMBOJ23:728-738 (2004)). Domain IE (also referred to herein as "ID" or "DIE") is at the carboxy-terminus of the envelope protein. Domain III is also referred to as "domain B" in earlier antigenic mapping studies. Domain III has several epitopes that can elicit virus-neutralizing antibodies (Roehrig, Adv Virus Res 59:141-175 (2003)). In addition, studies with several fiaviviruses, including Tick-borne encephalitis (Mandle, et al, J. Virol 75:5627-5637 (2001)), indicate that domain El, which has a fold typical of an immunoglobulin constant domain, may mediate flavivirus attachment to host cells (Anderson, Adv Vims Res 59:229-274 (2003)) and, thus, be a receptor-binding domain.<br>
The crystal structure of domains I. II and El of the envelope protein from the Tick-borne encephalitis flavivirus and the Dengue 2 flavivirus has been determined (Rey, FA., etal.,Nature 375:291-298 (1995); Modis, Y., etal, Nature 427:313-319 (2004), respectively). Domain I of the Tick-borne encephalitis envelope protein corresponds to amino acids 1-51, 137-189 and 285-302 of SEQ ID NO: 174; domain II of the Tick-borne encephalitis envelope, protein of SEQ ID NO: 174 corresponds to amino acids 52-136 and 190-284; and domain El corresponds to amino acids 303-395 of SEQ ED NO: 174. (Key, FA., etal., Nature 375:291-298 (1995)). SEQ ID NO: 174 (Figure 44) is encoded by SEQ ED NO: 175 (Figure 78). Domain I of the Dengue 2 flavivirus envelope protein corresponds to amino acids 1-52,132-193 and<br>
280-296 of SEQ ID NO: 160 (Figure 70); domain II corresponds to amino acids 53-131 and 194-279 of SEQ ID NO: 160; and domain III corresponds to amino acids 297-495 of SEQ ID NO: 160 (Modis, Y., et d.t Nature 427:313-319 (2004)). The location of domains I, II and IE of other flavivirus (e.g., West Nile virus, Japanese encephalitis, Dengue 1 virus, Dengue 3 virus and Dengue 4 virus) is based on homology of the Tick-borne encephalitis envelope protein domains and the Dengue 2 envelope protein domains. Thus, reference herein to domains of flavivirus proteins, in particular, flaviviruses other than Tick-borne encephalitis flavivirus envelope proteins and Dengue 2 flavivirus envelope proteins, are based on homology to domains in the Tick-borne encephalitis flavivirus envelope protein and the Dengue 2 flavivirus envelope protein.<br>
The domain El of the envelope protein of the DEN flavivirus encodes the majority of the flavivirus type-specific contiguous critical/dominant neutralizing epitopes (Roehring, IT., Adv. Virus Res. 59:141 (2003)), including the four DEN (DEN1, DEN2, DEN3, DEN4) viruses. Flavivirus envelope proteins axe highly homologous. Exemplary envelope protein sequences are shown in Figures 45, 68, 70, 72, 74 and 77 (SEQ ID NOs: 39, 160, 162, 164, 166 and 171, respectively).<br>
The seven nonstructural proteins of fiavivirus envelope proteins are involved in the replication of the virus. NS3 is a multifunctional enzyme that encodes a serine protease at the aminus-ierminal region; and helicase, RNA triphosphatase and NTPase activities in the carboxy-terminal region. NS5 encodes a methyltransferase and the KNA-dependent-KNA polymerase. NS2A, NS2B, NS4A and NS4B are four poorly characterized proteins. The central domain of NS2B is a co-factor for the NS3 serine protease while NS2A and NS4A are known to be components of the replication complex. NS1 is located at both the plasma membrane and in the lumen of intracelMar vesicles of virus-infected cells. NS1 is a multifunctional protein that is associated with an early step in the replication cycle pither prior to or early in negative-strand RNA synthesis and is also thought to be involved in virus maturation and/or release (Brinton, M.A., Annu Rev Microbiol 56:371 (2002)).<br>
West Nile virus (WNV) is a single-stranded positive sense RNA envelope virus. It was first isolated and identified in the West Nile region of Uganda in 1937<br>
from a febrile female adult (Smithbum. et al., Am J Trap MedHyg 3:9-18 (1954)). West Nile Virus has been classified as a member of the family Flaviviridae using cross-neutralization tests with polyclonal antisera (Doctor, er al., J. Virol Methods 2O05-311 (1989)). West Nile virus is neuroinvasive (George, etal, Bull WHO £2:879-882 (1984)); and severe human meningoencephalitis might occur consequent to infection with West Nile virus, as seen in the outbreaks in North America (CDC, Update: West Nile Virus Encephalitis - New York 1999, MMWR Morbid Mortal WTdy Rep 45:994-946; CDC, Update: West Nile Virus Encephalitis - New York 1999. MMWR Morbid Mortal WklyRep 57:1135-1136). During 1999-2002, WNV extended its range throughout much of the eastern part of the United States, and its range within the western hemisphere is expected to continue to expand. Birds are the natural reservoir hosts, and WNV is maintained in nature in a mosquito-bird-mosquito transmission cycle primarily involving Culex species mosquitoes.<br>
Recently, West Nile virus has emerged in temperate regions of Europe and North America, presenting a threat to public and animal health. The most serious manifestation of WNV infection is fatal encephalitis (inflammation of the brain) in humans and horses, as well as mortality in certain domestic and wild birds. West Nile virus infection has also been a significant cause of human illness in the United States. The envelope glycoprotein of the West Nile virus (WNV-E) and other flaviviruses may be important in formulating compositions to stimulate an immune response to generate neutralizing and protective antibodies. Currently, there are no compositions that prevent West Nile virus infection, for example, b)r stimulating an immune response in a subject.<br>
Japanese encephalitis'(JE) virus is localized in Asia and northern Australia (about 50,000 cases with aboutlO,000 deaths annually). A composition comprising an inactivated virus was recently associated with a case of acute disseminated encephalomyelitis, prompting the Japanese Ministry of Health, Labor and Welfare to recommend the nationwide suspension of compositions comprising inactivated virus.<br>
The Dengue (DEN) disease is caused by four mosquito-borne, serologically related flaviviruses known as DEN-1 (also referred to herein as "Denl" or Den I"),<br>
DEN-2 (also referred to herein as "Den2" or "Den 2"), DEN-3 (also referred to herein as "Den3" or "Den 3"), and DEN-4 (also referred to herein as "Den4" or Den 4"). and is an important arboviral disease of humans. DEN is a major public health problem in all tropical areas of the world. About three billion people are at risk for DEN and about 50 to about 100 million cases of dengue fever (DF) and hundreds of thousands of cases of dengue hemorrhagic fever (DHF) occur in the tropics each year, including Mexico, the Caribbean and parts of Asia and the South Pacific (Gubler, D.J., Ann Acad Med Singapore 27: 227-34 (1998)). Dengue viruses are transmitted by peridomestic Aedes spp. mosquitoes, which inhabit the tropics, allowing endemicrry of DF hi these areas. Infection by one virus causes Dengue Fever (DF), a febrile illness, which is not nonnally life-threatening, and leads to lifelong protective immunity against the infecting DEN serotype/virus. However, individuals that are infected by one serotype/virus remain susceptible to infection by the other three DEN serotypes/viruses. Subsequent infection by one of the other DEN serotypes/viruses can lead to Dengue hemorrhagic fever (DHF) or dengue shock syndrome (DSS), which are life-threatening diseases.<br>
DHF may be the result of an antibody dependent enhancement (ADE) where non-neutralizing antibodies induced by the primary DEN infection form virus-antibody complexes in secondary infections that are taken up into macrophages by Fc receptors and, thus, enhance virus infection. About 500.000 cases of DHF occur each year, mostly in children, with a fatality rate of about 5%. About 600 million children are at risk for DEN infections, about 60 million may get DEN infections each year, and about 60,000 may be hospitalized.   In addition, to the public health problem, military personnel are often sent overseas to tropical areas of the world where DEN viruses are found. Significant numbers of soldiers succumb to DEN while performing overseas, such as in Somalia, Grenada, Viet Nam and the Gulf conflicts. Attempts to develop a DEN vaccine have proved difficult due to ,the need to develop a tetravalent vaccine that protects against all four DEN serotypes/viruses.<br>
Methods to prevent flavivirus disease include vaccines to the flaviviruses (Barrett, A.D., Arm. N.Y. Acad. Sci 951:262 (2001). These compositions can be divided into two categories: live attenuated and inactivated. Compositions<br>
comprising live fiavivirus have been developed for YF and JE based on strains 17D and SA14-14-2, respectively, and were derived by empirical passage in chicken and hamster tissue, respectively. SA14-14-2 is produced in the People's Republic of China, grown in primary hamster kidney cell culture and very recently has been approved for use outside China. Both compositions are efficacious and require one or two doses, respectively, to develop protective immunity. There are inactivated virus compositions for JE and TBE. The inactivated JE compositions are based on strains Nakayama, Beijing-1 or P3. while inactivated TBE compositions are based on Central European TBE strains Neudorfl and K23, and Russian Spring Summer encephalitis strains Sofjin and 205. These killed flavivirus compositions require about two doses (given about 1 week to about 2 months apart), a booster dose at about one year and periodic boosters about every 3 to about 4 years. The antibody-mediated immunity, hi particular neutralizing antibodies, may be important in preventing flavivirus disease. Long-lived neutralizing antibody responses following administration of compositions to treat flavivirus disease or to prevent fiavivirus disease may also be important<br>
Many different approaches have been taken to develop compositions to prevent flavivirus infection, but many have not been successful. With respect to the disease DEN, which is the result of four related viruses (DEN1, DEN2, DEN3, DEN4), a composition may need to be developed to one or more the DEN flaviviruses. For example, a tetravalent (DEN1, DEN2, DEN3 and DEN4) composition may stimulate an immune response simultaneously against all four DEN viruses and thereby eliminate the potential of antibody dependent enhancement.<br>
Currently, there is no effective compositions to prevent infection by many flaviviruses, including West Nile virus, Dengue virus, Tick-borne virus, Kunjun virus, Murra)' Valley encephalitis virus and Yellow fever virus (Chang, G.J.. et al., Expert Rev. Vaccine 5:199 (2004)). Attenuation and immunogenicity may occur in compositions with live attenuated flavivirus. Furthermore, compositions with tetravalent live Dengue flaviviruses may have problems with interference and unbalanced immune response resulting in. many compositions being tested with<br>
variation in the quantity of each of the four DEN viruses. Compositions comprising inactivated flavivirus may have problems with immunogeniciTy and the need for multiple doses. In addition, the production of inactivated flavivirus compositions in infected mouse brains or cell culture can be complex, tedious, may result in<br>
t<br>
unknown hazards if not properly inactivated and may result in adverse effects when administered to subjects. Thus, there is a need to develop new compositions for use in the prevention of flavivirus infection in subjects.<br>
The compositions, fusion proteins and pol}7peptides of the invention employ pathogen-associated molecular patterns that trigger cellular events resulting in the expression of costimulatory molecules, secretion of critical cytokines and chemokines; and efficient processing and presentation of antigens to T-cells. As discussed above, TLRs recognize PAMPs including bacterial cell wall components (e.g., bacterial hpoproteins and lipopolysaccharides), bacterial DNA sequences that contain unmethylated CpG residues and bacterial flagellin. TLRs act as initiators of the innate immune response and gatekeepers of the adaptive immune response (Medzhitov, R., et al., Cold Springs Harb. Symp. Quant. Biol. 64:429 (1999); Pasare, C., et al., Semin, Immunol 16:23 (2004); Medzhitov, R., et al, Nature 388:394 (1997); Barton, G.M., et al, Curr. Opin. Immunol 14:380 (2002): Bendelac, A., et al, J. Exp- Med. ;P5:F19 (2002)).<br>
As discussed above, the binding of PAMPs to TLRs activates immune pathways for use in the compositions, fusion proteins and polypeptides of the invention, which can be employed in stimulating the immune system in a subject The compositions, fusion proteins and polypeptides of the invention can trigger an immune response to an antigen (e.g., a viral protein, such as a flaviviral protein) and trigger signal transduction pathways of the innate and adaptive immune system of the subject to thereby stimulate the immune system of a subject. Stimulation of the immune system of the subject may prevent infection by an antigen or a virus (e.g., a flavivirus) and thereby treat the subject or prevent the subject from disease, illness and, possibly, death.<br>
The compositions, fusion proteins and polypeptides of the invention, can include, for example, one. two, three, four, five, six or more pathogen-associated<br>
molecular patterns (e.g., Pam.2C.ys, PamSCys, flagellin) and one, two, three, four, five, six or more antigens. When two or more PAMPs and/or two or more antigens and/or viral proteins comprise the compositions, fusion proteins and polypeptides of the invention, they are also referred to as "multimers."<br>
The pathogen-associated molecular pattern can be a TLR5 agonist (e.g., at least a portion of at least one flagellin). The flagellin- can be at least one member selected from the group consisting of a fijB/STF2. an E. coli fliC, and a S1. muenchen fliC. The flagellin can include fijB/STF2 (e.g., SEQ ID NO: 1) or a flagellin lacking a hinge region (e.g., SEQ ID NO: 3).<br>
The pathogen-associated molecular pattern can be a TLR2 agonist. The TLR2 agonist includes at least one member selected from the group consisting of a PamlCys and a Pam3Cys. Pam3Cys is ([Palmitoyl]-Cys((RS)-2,3-di(palmitoyloxy)-propyl cysteine). Pam3Cys is also referred to herein as "P2." Pam2Cys is (S-[2,3-bis(palrhitoyloxy) propyl] cysteine).<br>
The viral protein for use in the compositions, fusion proteins and polypeptides of the invention can be an envelope protein of at least one member selected from the group consisting of a West Nile viral envelope protein, a Langat viral envelope protein, a Kunjin viral envelope protein, a Murray Valley encephalitis viral envelope protein, a Japanese encephalitis viral envelope protein, a Tick-borne encephalitis viral envelope protein, a Yellow fever viral envelope protein and a Dengue viral envelope protein.<br>
The compositions, fusion proteins and polypeptides of the invention can employ any portion of the envelope protein of a flavivirus. The compositions, fusion proteins and polypeptides of the invention can include at least a portion of at least one member selected from the group consisting of domain I. domain II and domain IE of an envelope protein of a flavivirus, "At Least a portion," as used herein, in reference to a domain of an envelope protein, means any part of the envelope protein domain or the entirety of the envelope protein. For example, SEQ IDNOS: 88 and 100-151, include at least a portion of domain m of the West Nile virus envelope protein.<br>
"El," "EII," and "Em," as used herein, refer to domains I, II and EL respectively, of the West Nile flavivirus envelope protein. "JEI," "JEE." and "JEIII," as used herein, refer to domains I. II and III, respectively, of the Japanese encephalitis flavivirus envelope protein. "Denl I," "Denl II," and "Denl III,'" as used herein refer to domains I, II and EL respectively, of the Dengue 1 flavivirus envelope protein. Likewise, designations for the domains of envelope proteins of other flaviviruses are referenced by the flavivirus name followed by the domain number (e.g., (Tick-borne) TBI (Tick-borne), TBII, TBIII. Den2 I, Den2 II, Den2 IE).<br>
The portion of an envelope protein of a flavivirus employed in the compositions, fusion proteins and polypeptides of the invention can include at least one member selected from the group consisting of at least a portion of domain I, at least a portion of domain E and at least a portion of domain IE. When a domain is designated with a "+," for example "EEI+" or "JEHI+," the portion of the envelope protein referenced as "El" is one component of the total of that domain plus at least one of at least a portion of either or both of domains I and II. For example, "EEI+," as used herein, means the compositions, fusion proteins and polypeptides of the invention include domain III and at least a portion of domain I. "EEI+" is also referred to as "El/IE." "JEIE+" is also referred to as "JEI/EI." Similarly, when compositions, fusion proteins and polypeptides of the invention include domains'of envelope proteins of flavivirus, the domains can be any combination of domains I, II, and El and can be.designated based on the domain. For example, EI/II includes domain I and II of the West Nile fiavivirus. The absence of a "+" in reference to a domain (e.g., Bill, JEIE, Denl IE) of an envelope protein employed in the compositions, fusion proteins and polypeptides of the invention means that the composition, fusion protein and polypeptide includes the referenced domain. For example, "Denl El" means the compositions, fusion proteins and compositions include domain III. not domains I and II, of the Dengue 1 virus.<br>
The West Nile viral envelope protein for use in the compositions, fusion proteins and polypeptides of the invention can include at least a portion of at least one member selected from the group consisting of<br>
ME£LgLKGTTYGVCSKAFKFLGTPADTGHGTVVLELQYTGTDGPCKVPISS VASLMDLTPVGRLVTWPFVSVATANAKVLffiLEPPFGDSYTVVGRGEQQlN HHWHKSGSSIGK (SEQ ID NO: 7, which is an EIII+ amino acid sequence, the italicized amino acids are domain I of the envelope protein and the remaining sequence is domain III of the envelope protein);<br>
GTTYGVCSKAFKFARTPADTGHGT^^LELQ^TGKDGPCKWISSVASLNDL TPVGRLVTAn^FVSVATANSKVLEELEPPFGDSYTVVGRGEQQINHHWHKSG (SEQ ID NO: 8, West Nile virus, Stanford, CT, also referred to as "West Nile S"); GTTYGVCSKAFKFLGTPADTGHGTVVLELQYTGTDGPCKVPISSVASLNDLT PYGRL\TVNPFVSVATANAKVLIELEPPFGDSYVVGRGEQQINHHW1QCSG (SEQ ID NO: 9, West Nile virus, New York, NY, also referred to- as "West Nile NY"); and ELEPPFGDSYIWGRGEQQINHFIWHKS (SEQ ID NO: 10). SEQ ID NO: 7 is encoded by<br>
ATGGAAAAATTGCAGTTGAAGGGAACAACCTATGGCGTCTGTTCAAAGG CTTTCAAGTTTCTTGGGACTCCCGCAGACACAGGTCACGGCACTGTGGTG TTGGAATTGCAGTACACTGGCACGGATGGACCTTGCAAAGTTCCTATCTC GTCAGTGGCTTCATTGAACGACCTAACGCCAGTGGGCAGATTGGTCACT GTCAACCCTTTTGTTTCAGTGGCCACGGCCAACGCTAAGGTCCTGATTGA ATTGGAACCACCCTTTGGAGACTCATACATAGTGGTGGGCAGAGGAGAA CAACAGATCAATCACCATTGGCACAAGTCTGGAAGCAGCATTGGCAAA (SEQ ID NO: 11). The West Nile viral envelope protein can include a protein that has at least about 70% identity, at least about 75% identity, at least about 80% identity, at least about 85% identity, at least about 90% identity, at least about 95% identity and at least about 99% identity to a polypeptide that includes SEQ ED NO: 7, which include portions of domains I and III (referred to herein as "EER") of the West Nile virus.<br>
The Langat virus envelope protein for use in the compositions, fusion proteins and polypeptides of the invention can include at least a portion of GLTYTYCDKTKTTWI
GTTYGVCSEL^FPJLGTPADTGHGTVVLELQYTGTDGPCKfflSSVASLNDLT PVGRLVTVNPFVSVSTANAKVLIELEPPFGDSYrS^VGRGEQQINHHWHKSG (SEQ ED NO: 13). The Murray Valle}' encephalitis envelope protein can include at leas: a portion of<br>
GTTYGMCTEKFTFSKNPADTGHGTVVLELQYTGSDGPCKIPISSVASLNDMT PVGRMVTAOTyVASSTANAKVLVEmPPFGDSYT^GRGDKQINHHWHKEG (SEQ ED NO: 14), The Japanese encephalitis envelope protein can include at least one member selected from the group consisting of at least a portion of GTTYGMCTEKFSFAKNPADTGHGTWEELSYSGSDGPCKIPrVSVASLNDMT PVGRLVTVNTFVATSSANSKVLVEMEPPFGSDYR^GMGDKQINHHU'HIC^ G (SEQ ED NO: 15) and EMEPPFGDSYIWMGDKQINHHWHKA (SEQ ED NO: 16). The Tick-borne encephalitis envelope protein can include at least a portion of GLTYTMCDKTEETWKRAPTDSGro^<br>
D^/NVAh^ITPNPTTENNGGGFIEMQLPPGDNIIYVGELSHQWFQK (SEQ ID NO: 17). The Yellow fever virus envelope protein can include at least a portion of GLTYTMCDKTFT^/KRAPTDSGHDT\^MEVTFSGTKPCRIPVRAVAHGSPD Vm^AMLITPNPirENNGGGFIEMQLPPGDNIIYVGELSHQWFQK (SEQ ED NO: 18). The envelope protein of a flavivirus can include at least a portion of at least one member selected from the group consisting of<br>
GTTYGMCSKKFTFRPADTGHGTVVLELQYSGDGPCKEPISVASKNDLTPVGPv LVTVNPFVSSTANAKVLIEMEPPFGDSYTVVGGEQINHHWHKG (SEQ ED NO: 19) and<br>
GMSYSMCTGKFKVVKEIAETQHGTWIRVQYEGDGSPCKIPFEIMDLEKMrv' LGRLITVNPIVTEKDSPVNTEAEPPFGDSYniGVEPQQLKLNWFKK (SEQ ID NO: 40). SEQIDNOS: 12, 13, 14, 15, 16, 17,18,19 and 40 are portions of domain III of the viral envelope protein.<br>
In another embodiment, the invention is a composition comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one Den2 viral envelope protein, wherein the Den2 viral envelope protein is at least one member selected from the group consisting of<br>
EAEPPFGDSYII1GVEPQQLKLNWFKK (SEQ ID NO: 22), SEQ ED NO: 40 and SEQ ID NO: 97.<br>
The compositions, fusion proteins and polypeptides of the invention can include Den 1 (EAEPPFGESYIVVGAGEKALKLSWFKK (SEQ ID NO.: 20); Den 1 PR 94 (Puerto Rico, 1994) (ETEPPFGESYTVVGAGEKALKLSWFKK (SEQ ID NO: 21)); Den 3 (EAEPPFGESNIVIGIGDKALKINWYKK (SEQ ED NO: 23)); and Den 4 (ELEPPFGESYIYIGVGNSALTLHWFRK (SEQ ID NO: 24)). SEQ ID NOS: 20, 2L 22, 23 and 24 are portions of domain III of Denl, Den2, Den3 and Den4 fiaviviruses. At least a portion of domain III of the four Dengue viruses can be employed together or separately in the compositions, fusion proteins or polypeptides of the ivnetion. 'For example, domain HI of Denl (strain 16007), Den2 (strain 516803), Dero (strain H53489) and Den4 (strain 703) can be employed separately or in combination. The pathogen-associated molecular pattern and Den2 envelope viral protein can be components of a fusion protein.<br>
In still another embodiment, the invention is a composition comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In an additional embodiment, the invention is a fusion protein comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one viral protein selected from the group consisting of .a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-borne encephalitis viral protein, a Yellow fever viral protein and a hepatitis C viral protein. The hepatitis C viral protein for use in the compositions, fusion proteins and polypeptides of the invention can include a polygeptide of at least one member selected from the group consisting of SEQ ID NO: 64 (Figure 22) and SEQ ED NO: 65 (Figure 23), which are encoded by SEQ ID NOS: 66 (Figure 24) and 67 (Figure 25). respectively.<br>
In another embodiment the invention is a fusion protein comprising at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
Fusion proteins of the inventiorj can further include a linker between the pathogen-associated molecular pattern and the viral protein. The linker can be an am inn acid linker. The amino acid linker can include synthetic or naturally occurring amino acid residues. The amino acid linker employed in the fusion proteins of the invention can include at least one member selected from, the group consisting of a lysine residue, a glutamic acid residue, a serine residue, a cysteine residue and an arginine residue. "Amino acid linker," as used herein, is also referred to as a "peptide linker." The amino acid linker can include at least one member selected from the group consisting of a peptide of KGNSKLEGQLEFPRTS (SEQ ID NO: 26); EFCRYPAQWRPL (SEQ ID NO: 28); EFSRYPAQWRPL (SEQ ID NO: 60): KGNSKLEGQLEFPRTSPVWWNSADIQHSGGRQCDGYLQNSPLRPL (SEQ ID NO: 62); EFSRYPAQWRPL (SEQ ID NO: 75); which are encoded by AAGGGCAATTCGAAGCTTGAAGGTCAATTGGAATTCCCTAGGACTAGT (SEQ ID NO: 25); GAATTCTGCAGATATCCAGCACAGTGGCGGCCGCTC (SEQ ID NO: 27); GAATTCTCTAGATATCCAOCACAGTOOCGGCCGCTC (SEQ ID NO: 61);<br>
AAGGGCAATTCGAAGCTTGAAGGTCAATTGGAATTCCCTAGGACTAGTC CAGTGTGGTGGAATTCTGCAGATATCCAGCACAGTGGCGGCCGCCAGTG TGATGGATATCTGCAGAATTCGCCCTTGCGGCCGCTC (SEQ ID NO: 63); and GAATTCTCTAGATATCCAGCACAGTGGCGQCCGCT ((SEQ ID NO: 74).<br>
The fusion proteins of the invention can further include a linker between at least one component of the fusion protein (e.g., PamBCys, Pam2Cys, flagellin, PAMP) and at least one other component of the fusion protein (e.g.. at least a portion of an antigen, at least a portion of a viral protein) of the composition, a linker between at least two of similar components of the fusion protein (e.g.,<br>
Pam3Cys. Pam2Cys, flagellin. PAMP, at least a portion of an antigen, at least a portion of a viral protein) or any combination thereof.<br>
"Linker," as used herein in reference 10 a fusion protein of the invention, refers to a connector between components of the fusion protein in a manner that the components of the fusion protein are not directly joined. For example, one component of the fusion protein (e.g., Pam3Cys, PamlCys, PAMP) can be linked to a distinct component (e.g.. at least a portion of an antigen, at least a portion of a viral protein) of the fusion protein. Likewise, at least two or more similar or like components of the fusion protein can be linked (e.g., two PAMPs can further include a linker between each PAMP, or two antigens can further include a linker between each antigen, or two viral proteins can further include a linker between each viral protein).<br>
Additionally or alternatively, the fusion proteins of the invention can include a combination of a linker between distinct components of the fusion protein and similar or like components of the fusion protein. For example, a fusion protein can comprise at least two PAMPs. PamSCys and/or Pam2Cys components that further includes a linker between, for example, two or more PAMPs; at least two antigens or at least two viral proteins that further include a linker between them; a linker between one component of the fusion protein (e.g., PAMP) and another distinct component of the fusion protein (e.g., at least a portion of an antigen, at least a portion of a viral protein), or any combination thereof. Thus, the fusion proteins of the invention can further include a linker between at least two pathogen-associated molecular patterns, a linker between at least two antigens, a linker between at least two viral proteins, or any combination thereof.<br>
The pathogen-associated molecular pattern of the fusion proteins of the invention can be fused to a carboxy-terminus, an amino-terminus or both a carboxy-and an amino-terminus of an antigen or at least a portion of a viral protein (e.g., a flavivirus viral protein, such as at least a portion of domain Etl of the West Nile envelope protein, referred to as "EIIL," at least a portion of domain El of Dengue 1 envelope protein) referred to as "Denl III." "Fused to," as used herein, means covalently or noncovalently linked or recombinantly produced together.<br>
The fusion proteins of the invention can include at least one pathogen-associated molecular pattern between at least two antigens or at least two viral proteins, which can, optionally, include a linker between the pathogen-associate molecular pattern and the antigen or the viral protein. The fusion proteins of the invention can include a pathogen-associated molecular pattern fused between at least two viral proteins (e.g., designated as "viral protein.?AMP .viral protein").   The<br>
4<br>
fusion proteins of the invention can include an antigen or a viral protein fused between at least two pathogen-associated molecular patterns (e.g., designated as "P AMP .viral protein.? AMP").<br>
The pathogen-associated molecular pattern of the fusion proteins of the invention can be a TLR5 agonist, such as a flagellin. The antigen or viral protein of the fusion proteins of the invention can be fused to the fiagellin in a region of the flagellin that lacks at least a portion of a hinge region of the flagellin. For example, at least a portion of the hinge region of the fijB/STF2 flagellin of SEQ ID NO: 1 (Figure 1) can be deleted and an antigen or a viral protein can be fused to the flagellin in the region of the deletion.<br>
The antigen or viral protein of the fusion proteins of the invention can be fused to the fiagellin in a region of the flageHin that contains a binge region of the flageliin.  For example, an antigen or viral protein can be fused to the fljB/STF2 , fiagellin of SEQ ID NO: 1 (Figure 1) at any place in the binge region, for example, at any place with amino acids 176-415 of SEQ ID NO: 1.<br>
The antigen or viral protein of the fusion proteins of the invention can be fused to the flagellin in a region of the flagellin that lacks a hinge region of the flagellin, wherein the hinge region has been replaced with an artificial hinge region, such as an amino acid linker.   For example, an antigen or viral protein can be fused to the fijB/STF2A flagellin of SEQ ID NO: 3 (Figure 3) by placing an amino acid linker (also referred to herein as an "artificial hinge" or "an artifical hinge region" or "an artificial hypervariable region"), as depicted, for example, with the placement of an amino acid linker (HGAPVDPASPW, SEQ D NO: 183) between amino acids 175 to 186 of SEQ ID NO: 3.<br>
In another embodiment, the invention is a fusion protein comprising at least a portion of at least one member selected from the group consisting of fljB/STF2. an £. coli fiiC, and a 5. muenchen fliC and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein. The portion of the envelope protein can be at least a portion of at least one member selected from the group consisting of domain I, domain II and domain III of the envelope protein.<br>
In still another embodiment, the invention includes a polypeptide that includes SEQ ID NOS: 71, 72, 30, 32, 34, 36, 38, 55, 76, 6, 80, 82, 84, 86 and 159 and a polypeptide encoded by SEQ ED NOS: 70, 73,29, 31, 33,35, 37, 54, 77, 5, 81, 83, 85, 87 and 158.<br>
In an additional embodiment, the invention includes a polypeptide having at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%. at least about 98% and at least about 99% sequence identity to the polypeptides of SEQ ID NOS: 71, 72, 30, 32, 34, 36, 38, 55, 75, 6, 80, 82, 84, 86 and 159 and the nucleic acids of SEQ ID NOS: 70, 73,29, 31, 33, 35, 37, 54, 77, 5, 81, 83, 85, 87 and 158.<br>
In a further embodiment, the invention includes compositions, fusion proteins and polypeptides that include a polypeptide having a flagellin that includes polypeptides having at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 98% and at least about 99% sequence identity to the polypeptides of SEQ ID NOS: 1, 3, 58 and 68 and the nucleic acids of 2, 4, 59 and 69.<br>
The percent identity of two amino acid sequences (or two nucleic acid sequences) can be determined by aligning the sequences for optimal comparison purposes (e.g., gaps can be introduced in the sequence of a first sequence). The amino acid sequence or nucleic acid sequences at corresponding positions are then compared, and the percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % identity = # of identical positions/total # of positions x 100). The length of the protein or nucleic<br>
acid encoding a PAMP, at least a portion of a fusion protein, a viral protein, or a. polypeptide of the invention aligned for comparison purposes is at least 30%, preferably, at least 40%, more preferably, at least 60%, and even more preferably, at least 70%, 75%, 80%: 85%, 90%, 95%, 98%, 99% or 100%, of the length of the reference sequence, for example, the nucleic acid sequence of a PAMP, at least a portion of a viral protein, or a polypeptide or fusion protein, for example, as depicted in SEQ ID NOS: 71, 72, 30, 32, 34, 36, 38,55, 75, 6, 80, 82, 84, 86 and 159.<br>
The actual comparison of the two sequences can be accomplished by well-known methods, for example, using a mathematical algorithm. A preferred, non-limiting example of such a mathematical algorithm is described in Karlin et al. (Proc. Nad. Acad. Sci. USA, P0:5S73-5877 (1993), the teachings of which are hereby incorporated by reference in its entirety). Such an algorithm is incorporated into the BLASTN and BLASTX programs (version 2.2) as described in Schaffer et al. (Nucleic Acids Res., 29:2994-3005 (2001), the teachings of which are hereby incorporated by reference in its entirety). When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (e.g., BLASTN: available at the Internet site for the National Center for Biotechnology Information) can be used. In one embodiment, the database searched is a non-redundant (NR) database, and parameters for sequence comparison can be set at: no filters; Expect value of 10; Word Size of 3; the Matrix is BLOSUM62; and Gap Costs have an Existence of 11 and an Extension of 1.<br>
Another mathematical algorithm employed for the comparison of sequences is the algorithm of Myers and Miller, CABIOS (1989), the teachings of which are hereby incorporated by reference in its entirety. Such an algorithm is incorporated into the ALIGN program (version 2.0), which is part of the GCG (Accelrys, San Diego, California) sequence alignment software package. When utilizing the ALIGN program for comparing amino acid sequences, a PAM 120 weight residue table, a gap length penalty of 12, and a gap penalty of 4 is used. Additional algorithms for sequence analysis are known in the art and include ADVANCE and ADAM as described in Torellis andRobotti (Comput. Appl. Biosci., 10: 3-5 (1994), the teachings of which are hereby incorporated by reference in its entirety); and<br>
FASTA described in Pearson and Lipman (Proc. Natl. Acad. Sci USA, 85: 2444-2448 (1988). the teachings of which are hereby incorporated by reference in its entirety).<br>
In a further embodiment, the invention is host cells and vectors that include the nucleic acid sequences of the invention. The host cells can be prokaryotic (e.g., E. coif) or eukaryotic (e.g.. insect cells, such as Drosophila Dmel2 cells; Baculovirus; CHO cells; yeast cells, such as Pichia) host cells.<br>
The percent identity between two amino acid sequences can also be accomplished using the GAP program in the GCG software package (Accelrys, San Diego, California) using either a Blossom 63 matrix or aPAM250 matrix, and a gap weight of 12. 10, 8, 6, or 4 and a length weight of 2, 3, or 4. In yet another embodiment, the percent identity between two nucleic acid sequences can be accomplished using the GAP program in the GCG software package (Accelrys. San Diego, California), using a gap weight of 50 and a length weight of 3.<br>
The nucleic acid sequence encoding a PAMP, at least a portion of a viral protein, fusion proteins of the invention and polypeptides of the invention can include nucleic acid sequences that hybridize to, for example, a fljB/STF2 (e.g., SEQ ID NOS: 2, 4), a fliC (e.g., SEQ ID NOs: 59, 69), at least a portion of a viral protein (e.g., SEQ ID NOS: 39, 160,162, 164, 166 and 177 and fusion proteins of The invention (e.g., SEQ ID NOS: 71, 72, 30, 32: 34, 36, 38, 55, 75: 6, 80, 82, 84 and 86) under selective hybridization conditions (e.g., highly stringent hybridization conditions). As used herein, the terms "hybridizes under low stringency," "hybridizes under medium stringency," "hybridizes under high stringency," or "hybridizes under very high stringency conditions," describe conditions for hybridization and washing of the nucleic acid sequences, Guidance for performing hybridization reactions, which can include aqueous and nonaqueous methods, can be found in Aubusel, P.M.. et al., Current Protocols in Molecular Biology. John Wiley &amp; Sons, N.Y. (2001), the teachings of which are hereby incorporated herein in its entirety.<br>
For applications that require high selectivity, relatively high stringency conditions to form hybrids can be employed. In solutions used for some membrane<br>
based hybridizations, addition of an organic solvent, such as formamide. allows the reaction to occur at a lower temperature. High stringency conditions are, for example, relatively low salt and/or high temperature conditions. High stringency are provided by about 0.02 M to about 0.10 M NaCl at temperatures of about 50°C to about 70°C. High stringency conditions allow for limited numbers'of mismatches between the two sequences. In order to achieve less stringent conditions, the salt concentration may be increased and/or the temperature may be decreased Medium stringency conditions are achieved at a salt concentration of about 0.1 to 0.25 M NaCl and a temperature of about 37°C to about 55°C, while low stringency conditions are achieved at a salt concentration of about 0.15 M to about 0.9 M NaCl, and a temperature ranging from about 20°C to about 55°C. Selection of components and conditions for hybridization are well known to those skilled in the art and are reviewed in Ausubel et al. (1997. Short Protocols in Molecular Biology, John Wiley &amp; Sons, New York N.Y., Units 2.8-2.11, 3.18-3.19 and 4-64.9).<br>
In yet another embodiment, the invention is a composition comprising at least one Pam3Cys and at least a portion of at least one fiavivirus protein (e.g., at least one member selected from the group consisting of a West Nile viral protein, a Dengue viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-borne encephalitis viral protein, a Yellow fever viral protein and a hepatitis C viral protein)- The Dengue viral protein can be at least one member selected from the group consisting of a Denl viral protein, a Den2 viral protein, a Den3 viral protein and a Den4 viral protein. The Pam3Cys and the fiavivirus protein can be components of a fusion protein.<br>
An additional embodiment of the invention is a composition comprising at least one Pam2Cys and at least a portion of at least one fiavivirus protein (e.g., at least one member selected from the group consisting of a West Nile viral protein, a Dengue viral protein, a Langat viral protein, a Kunjin viral protein, a Murra}' Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-borne encephalitis viral protein, a Yellow fever viral protein and a hepatitis C viral protein). The Dengue viral protein can be at least one member selected from the<br>
group consisting of a Denl viral protein, a Den2 viral protein, a Den3 viral protein and a Den4 viral protein. The Pam?,Cys and the flavivirus protein can be components of a fusion protein.<br>
In still another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a composition that includes the compositions, fusion proteins and polypeptides of the invention.<br>
"Stimulating an immune response." as used herein, refers to the generation of antibodies to at least a portion of an antigen or a viral protein (e.g., a West Nile viral protein, a Dengue viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-borne encephalitis viral protein, a Yellow fever viral protein and a hepatitis C viral protein). Stimulating an immune response in a subject can include the production of humoral and/or cellular immune responses that are reactive.against the antigen or viral protein. In stimulating an immune response in the subject, the subject ma)' be protected from infection by the antigen or virus or conditions associated with infection by the antigen or virus that may diminish or be halted as a consequence of stimulating an immune response in the subject.<br>
The compositions, fusion proteins and polypeptides of the invention for use in methods to stimulate immune responses in subjects, can be evaluated for the ability to stimulate an immune response in a subject using well-established methods. Exemplar)' methods to determine whether the compositions, fusion proteins and polypeptides of the invention stimulate an immune response in a subject, include measuring the production of antibodies specific to the antigen or viral protein (e.g., IgG antibodies) by a suitable technique such as, ELISA assays; the potential to induce antibody-dependent enhancement (ADE) of a secondary infection; macrophage-like assays; neutralization assessed by using the Plaque Reduction Neutralization Test (PPJSTTgo); the ability to generate serum antibodies in non-human models (e.g., mice, rabbits, monkeys) (Putnak, et al, Vaccine 23:4442-4452 (2005)); the ability to survive a challenge of exposure to an antigen, in particular, a viral antigen employing non-human animals, such as mice and monkeys.<br>
A "subject," as used herein, can be a mammal, such as a primate or rodent (e.g., rat, mouse). In a particular embodiment, the subject is a human.<br>
In a further embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a composition that includes at least a portion of at least one pathogen-associated • molecular pattern and at least a portion of at least one viral protein selected from the group consisting of a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-borne encephalitis viral protein, and a Yellow fever virus protein.<br>
In yet another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject the composition that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one Den2 envelope protein, wherein the Den2 envelope protein is selected from the group consisting of SEQ ID NO: 22.. SEQ ID NO: 40 and SEQ ID NO: 97.<br>
In another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one viral protein selected from the group consisting of a West Nile viral protein, a Langat viral protein, a Kunjin viral protein, a Murray Valley encephalitis viral protein, a Japanese encephalitis viral protein, a Tick-bome encephalitis, viral-protein and a Yellow fever viral protein.<br>
In still another embodiment, the invention is a method stimulating an immune response in a subject, comprising the step of administering to the subject a composition that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of aDenl viral envelope protein. aDen2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
An additional embodiment of the invention is a method stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein that includes at least a portion of at least one member selected from<br>
the group consisting of a fijB/STF2, an £. coll fiiC, and a S. mmnchen fliC and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein (e.g.,<br>
KGMSYVMCTGSFKLEKEVAETQHGTVLVQVKYEGTDAPCICIPFSTQDEKG VTQNGRLITAWIVTDKEIsTVNlEAEPPFGENYIVVGAGEKALKLSWFKJK: (SEQ ID NOS: 21 and 96)), a Den2 viral envelope protein (e.g., SEQ ID NOS: 22, 40 and<br>
KGMSYSMCTGKFKVVKEIAETQHGTIVIRVQYEGDGSPCKTPFEIMDLEKRH VLGRLTTVW^TEKDSPVNIEAEPPFGDSYfflGVEPGQLKLDWTKK (SEQ ID NO: 97)). a Den3 viral envelope protein (e.g., SEQ ID NO: 23 and KGMSY.AMCLNTFVLKKEVSETQHGTILDCVEYICGEDAPCKIPFSTEDGQGK<br>
ID NO: 98)) and a Den4 viral envelope protein (e.g.. SEQ ID NO: 24 and KGMSYTMCSCHCFSmKEMAETQHGTTVVKVKYEQAGAPCKWIEnUDVNKE KVTGRIISPTPFAENTNSVTNTELERPLDSYIVIGVGDSALTLHWFRK (SEQ ID NO: 99)).<br>
In a further embodiment, the invention is a method stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein that includes at least a portion of at least one pathogen-associated molecular pattern and at least a portion of at least one member selected from the group consisting of a Denl viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein.<br>
In yet another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a composition comprising at least a portion of at least one antigen and at least a portion of at least one flagellin, wherein at least one of the flagellins lack at least a portion of a hinge region.<br>
In another embodiment, the invention is a method of stimulating an immune response in a subject, comprising the step of administering to the subject a fusion protein comprising at least a portion of at least one antigen and at least a portion of<br>
at least one flagellin, wherein at least one of the fiagellins lack at least a portion of the hinge region.<br>
An "effective amount," when referring to the amount of a composition, fusion protein or a polypeptide of the invention, refers to that amount or dose of the composition, fusion protein, or a polj^peptide. that, when administered to the subject is an amount sufficient for therapeutic efficacy (e.g., an amount sufficient to stimulate an immune response in the subject). The compositions, fusion proteins, or polypeptides of the invention can be administered in a single dose or in multiple doses.<br>
The methods of the present invention can be accomplished by the administration of the compositions, fusion proteins or polypeptides of the invention by enteral or parenteral means. Specifically, the route of administration is by oral ingestion (e.g., drink, tablet, capsule form) or intramuscular injection of the composition, fusion protein or polypeptide. Other routes of administration as also encompassed by the present invention including intravenous, intradermal, intraarterial, intraperitoneal, or subcutaneous routes, and nasal administration. Suppositories or transdermal patches can also be employed.<br>
The compositions, fusion proteins or polypeptides of the invention can be administered ex vivo to a subject's autologous dendritic cells. Following exposure of the dendritic cells to the composition, fusion protein or polypeptide of the invention, the dendritic cells can be administered to the subject.<br>
The compositions, fusion proteins or polypeptides of the invention can be administered alone or can be coadministered to the patient Coadminstration is meant to include simultaneous or sequential administration of the composition, fusion protein or polypsptide of the invention individually or hi combination. Where the composition, fusion protein or polypeptide are administered individually, the mode of administration can be conducted sufficiently close in time to each other (for example, administration of the composition close in time to administration of the fusion protein) so that the effects on stimulating an immune response in a subject are maximal, It is also envisioned that multiple routes of administration (e.g.,<br>
intramuscuiar, oral, transdermal) can be used to administer the compositions and fusion proteins of the invention.<br>
The compositions, fusion proteins or polypeptide of the invention can be administered alone or as admixtures with conventional excipients, for example, pharmaceutically, or physiologically, acceptable organic, or inorganic carrier substances suitable for enteral or parenteral application which do not deleteriously react with the extract. Suitable pharmaceutically acceptable carriers include water, salt solutions (such as Ringer's solution), alcohols, oils, gelatins and carbohydrates such as lactose, amylose or starch, fatty acid esters, hydroxymethycellulose, and polyvinyl pyrolidine. Such preparations can be sterilized and, if desired, mixed with auxiliary agents such as lubricants, preservatives, stabilizers, wetting agents, emulsifiers, salts for influencing osmotic pressure, buffers, coloring, and/or aromatic substances and the like which do not deleteriousiy react with the compositions, fusion proteins or polypeptides of the invention. The preparations can also be combined, when desired, with other active substances to reduce metabolic degradation. The compositions; fusion proteins or polypeptides of the invention can be administered by is oral administration, such as a drink, intramuscular or intraperitoneal injection. The compositions, fusion proteins , or polypeptides alone, or when combined with an admixture, can be administered in a single or in more than one dose over a period of time to confer the desired effect (e.g.. alleviate prevent flavivirus infection, to alleviate symptoms of fiavivirus infection).<br>
When parenteral application is needed or desired, particularly suitable admixtures for the compositions, fusion proteins or polypeptides are injectable, sterile solutions, preferably oily or aqueous solutions, as well as suspensions, emulsions, or implants, including suppositories. In particular, carriers for parenteral • administration include aqueous solutions of dextrose, saline, pure water, ethanol, glycerol, propylene glycol. peanut oil, sesame oil, polyoxyethylene-block polymers, and the like. Ampules are convenient unit dosages. The compositions, fusion proteins or polypeptides can also be incorporated into liposomes or administered via transdermal pumps or patches. Pharmaceutical admixtures suitable for use in the present invention are well-known to those of skill in the art and are described, for<br>
example, in Pharmaceutical Sciences (17th Ed., Mack Pub. Co.. Easton, PA) and WO 96/05309 the teachings of which are hereby incorporated by reference.<br>
The compositions, fusion proteins and polypeptides of the invention can be administered to a subject on a carrier. "Carrier," as used herein, means any composition that presents the compositions, fusion proteins and polypeptides of the invention to the immune system of the subject to generate an immune response in the subject. The presentation of the compositions, fusion proteins and polypeptides of the invention would preferably include exposure of antigenic portions of the viral protein to generate antibodies. The components (e.g., PAMP and a viraJ protein) of the compositions, fusion proteins and polypeptides of the invention are in close physical proximity to one another on the carrier. The compositions, fusion proteins and polypeptides of the invention can be attached to the carrier by covalent or noncovalent attachment. Preferably, the carrier is biocompatible. "Biocompatible," as used herein, -means that the carrier does not generate an immune response in the subject (e.g., the production of antibodies). The carrier can be a biodegradable substrate carrier, such as a polymer bead or a liposome. The carrier can further include alum or other suitable adjuvants. The carrier can be a virus (e.g., adenovirus, poxvirus, alphavirus), bacteria (e.g., Salmonella) or a nucleic acid (e.g., plasmidDNA).<br>
The dosage and frequency (single or multiple doses) administered to a subject can vary depending upon a variety of factors, including prior exposure to an antigen, a viral protein, the duration of viral infection, prior treatment of the viral infection, the route of administration of the composition, fusion protein or polypeptide; size, age, sex, health, body weight, body mass index, and diet of the subject; nature and extent of symptoms of flavivirus exposure, flavivirus infection and the particular flavivirus responsible for the infection (e.g., a West Nile flavivirus, a Dengue flavivirus, a Langat flavivirus, a Kunjia fiavivirus, a Murray Valley encephalitis fiavivirus, a Japanese encephalitis flavivirus, a Tick-bome encephalitis flavivirus. a Yellow fever flavivirus and a hepatitis C fiavivirus), kind of concurrent treatment, complications from the flavivirus exposure, fiavivirus infection or other health-related problems. Other therapeutic regimens or agents can<br>
be used in conjunction with the methods and compositions, fusion proteins or polypeptides of the present invention. For example, the administration of the compositions, fusion proteins or polypeptides can be accompanied by other viral therapeutics or use of agents to treat the symptoms of the flavivirus infection (e.g.., high fever, numbness, DHF. meningoencephalitis). Adjustment and manipulation of established dosages (e.g., frequency and duration) are well within the ability of those skilled in the art.<br>
The present invention is further illustrated by the following examples, which are not intended to be limited in anjr way.<br>
EXEMPLIFICATION MATERIALS AND METHODS<br>
PCR Amplification and DNA primers<br>
All PCR amplifications were performed using Pfu Ultra Hotstart PCR Master Mix (Catalog number 600630) from Stratagene. (La Jolla, CA) according to the manufacturer's recommendations. DNA primers were purchased from Sigma Genosys and are described below.<br>
STF28BGF-1:<br>
CTCGGGAGATCTGCACAAGTAATCAACACTAACAGTCT (SEQ ID NO; 41)<br>
STF28MCR-1:<br>
CCATGGGCTAGCAGGATCCACCGGCGCTCCCTGCACGTTCA (SEQ ID NO: 42)<br>
STF28MCF-2:<br>
GGAGCGCCGGTGGATCCTGCTAGCCCATGGACCGAAAACCCG (SEQ ID NO: 43)<br>
STF28ECR-2: TCTOCAGAATTCACGTAACAGAGACAGCACGTTCTOCOOGACGTCCCGC<br>
AGAACGTGCTGTCTCTGTTACGTGAATTCTGCAGA (SEQ ID NO: 44)<br>
pET24AR:5   TCCGGCGTAGAGGATCGAGA (SEQ ID NO: 45)<br>
STF2-E3R3:<br>
CAATTGACCTTCAAGCTTCGAATTGCCCTTACGTAACAGAGACAGCACGT<br>
TCTG(SEQIDNO:46)<br>
AX-E3F3:<br>
AAGCTTGAAGGTCAATTGGAATTCCCTAGGACTAGTATGGAAAAATTOC AGTTGAAG (SEQ ID NO: 47)<br>
pET24AF:    GCTTAATGCGCCGCTACAGG (SEQ ID NO: 48)<br>
5'WNE28:    GCGGCCGCTCATGGAAAAATTGCAGTTGAAGGGAACAACC (SEQ ID NO: 49)<br>
3'WNE28:    CCGCGGTTTGCCAATGCTGCTTCCAGACTTGT (SEQ ID NO: 50)<br>
• NdeI-STF2:<br>
CCGGCATOCCATATGGCACAAGTAATCAACACTAACAGTCTGTCGCTGC (SEQ ID NO: 51)<br>
Blpl-Edffl:<br>
GCATGCTCAGCTTATTAAGGGTTTGCCAATGCTGCTTCCCAGACTTGTG (SEQ ID NO: 52)<br>
IE EIH primer: TACGTGAATTCAGCAGATATCCAGCAC (SEQ ID NO: 53)<br>
Cloning of pET/STF2A.EIII<br>
/ Full length flagellin of Salmonella typhimurium fljb (flagellin phase 2 ) (also referred to herein as "STF2") is encoded by a 1.5 kb gene. A truncated version of the STF2 (STF2A, SEQ ID NO: 3, encoded by SEQ ID NO: 4) was generated by deleting the hyper-variable region that spans amino acids 170 to 415 of SEQ ID NO: 1. The deleted region was replaced with a short flexible linker (GAPVDPASPW, SEQ ID NO: 56) designed to facilitate interactions of the NH2 and COOH termini sequences necessary for TLR5 signaling. To generate this construct, a two-step PCR was used. In the first reaction, STF2.0VA ((Figure 61) SEQ ID NO: 152 encoding amino acid sequence SEQ ID NO: 153 of Figure 62) served as the DNA template and STF28BGF-1 and STF28MCR-1 were used as'primei pairs.   In a separate reaction, the same DNA template was combined with, primers STF28MCF-2 and STF28ECR-2.<br>
The PCR amplification reactions generated about 500 bp and about 270 bp fragments, respectively. These PCR products were combined in a final PCR reaction using STF28BGF-1 and STF28ECR-2 as primers. The amplified DNA product from this reaction (about 0.77 kb) was digested with BglU. and EcoRl restriction enzymes and ligated into pMTBiPA;5-His B (Invitrogen, Carlsbad, CA) that had previously been digested with BgKI and EcoPJ and treated with alkaline phosphatase. An aliquot of the ligation mix was used to transform TOP 10 cells (InVitrogen. Carlsbad, CA). PCR screening was performed using vector specific primers, pMTFOR (methionine promoter) (CATCTCAGTGCAACTAAA, SEQ ED NO: 156) and BGHREV (bovine growth hormone poly A) (TAGAAGGCACAGTCGAGG, SEQ ID NO: 157), to identify several positive clones. All positive clones were further anatyzed by restriction mapping analysis and confirmed by DNA sequencing. The resultant construct pMT/STF2A was used to generate pMT/STF2A.EIII+.<br>
The domain in of the West Nile virus envelope protein (Figures 45 and 46) of pET/STF2A.EHI+ (SEQ ID NOS: 70, 71) was derived from the Drosophila expression plasmid pMT/STF2.E. This plasmid contains full-length STF2 (amino acids 1 -506, SEQ ID NO: 1) fused to the West Nile Virus envelope protein (amino acids 1-406 , SEQ ID N0:39, Figure 45). The pMT/STF2:E (SEQ ID NO: 158) clone AX-1 was used as a DNA template and 5'WNE28 (SEQ ID NO: 49) and 3'WNE28 (SEQ ID NO: 50) served as primers for PCR amplification. In order to facilitate restriction analysis and subsequent cloning steps, the 5' primer encoded a novel Notl site (New England Biolabls, Beverly, MA) and the 3' primer contained a unique SacE site, The amplified EHI+ DNA fragment (345bp; SEQ ID NO: 178 that encodes amino acids 292-406 of SEQ ID NO: 39) was subcloned into pCR-Blunt II-TOPO cloning vector (InVitrogen, Carlsbad, CA) to generate plasmid TOPOEHI. A stop codon was subsequently introduced downstream of the EIII+ sequence by blunting the SacE and Spel restriction sites using T4 DNA polymerase.<br>
To generate pMT/STF2A.EHI+ (SEQ ID NOS: 70, 71), the EIII+ fragment was isolated from TOPOEEI+ using Notl and BamHl restriction sites and ligated into the Notl and SacE restriction sites in pMT/STF2A. The BamHl site of the<br>
EIEE+ DNA fragment and the Sacll site of pMTSTF2A were blunted with T4 DNA polymerase prior to ligatioa. The STF2A.EII1+ sequence (SEQ ID NOS: 70, 71) from pMT/STF2A.EIII+ was isolated by PCR amplification using the primers Ndel-STF2 and BlpI-EdHI. To generate pET/STF2A.EIII+ (SEQ ID NO: 71), the PCR product was digested with Ndel and Blpl and ligated into pET24a plasmid thai had been predigested with Ndel and Blpl. The ligation mix was transformed into Mach-1 cells (In Vitro gen. Carlsbad, CA) and the cells were grown on LB supplemented with 50 ug/ml kanamycin. Several colonies were screened by restriction mapping and were verified by DNA sequencing.<br>
Cloning of pET/STF2.Em+<br>
The West Nile virus EIII+ sequence of pET/STF2.EITI+ (SEQ ID NOS: 54, 55) was derived from pETSTF2.E (SEQ ID NOS: 158, 159). This E. coli expression plasmid contains full-length STF2 (ammo acids 1-506) fused to the West Nile Virus envelope protein (amino acids 292-406 of SEQ ID NO: 39, which is SEQ ID NO: 7). In two independent PCR reactions, pET/STF2.E was used as the DNA template. One reaction used the primers pET24AR:5 (SEQ 3D NO: 45) and STF2-E3R3: (SEQ ID NO: 46) and the other used AX-E3F3 (SEQ ID NO: 47) and pET24AF (SEQ ID NO: 48). These PCR reactions generated a 1.5 kb fragment that consisted of full-length STF2 and a 340bp fragment that comprised the'Em domain plus additional amino acids that extended into domain I of the envelope protein. Aliquots of these PCR amplification reactions were combined, and the two products served as templates for a PCR reaction with the external primers pET24AR (SEQ ID NO: 45) and-pET24AF (SEQ ID NO: 48). This resulted hi the generation of about a 1.8kb DNA fragment that fused EIII-f sequence (SEQ ID NO: 178, a nucleic acid sequence encoding ammo acids 292-406 of SEQ ID NO: 39, which is SEQ ID NO: 7) to STF2. The PCR product was digested with Ndel and Blpl and gel purified and ligated by compatible ends to a pET24a vector that had previously been digested with compatible enzymes and de-phosphorylated. The ligation mix was transformed into Mach-1 cells (InVitrogen, Carlsbad, CA) as described for pET/STF2A.EIIR.<br>
Several colonies were screened by restriction mapping and two clones were verified by DNA sequencing.<br>
Cloning of pET/STF2A.JEin-+<br>
A portion of the envelope protein of a. Japanese encephalitis virus (JEV) (strain SA-14-14-2 (Jai, L., et al, Chin MedJ(Eng) 77(5:941-943 (2003)); currently employed in a JEV vaccine encoded by domain IE was custom synthesized by DNA 2. Inc (Menlo Park, CA). The portion of domain III was excised from the pJ2:G01510 using NotI and Blp 1 site that flank the insert The DNA insert was gel isolated and cloned by compatible ends to pET24A/STF2A.EIII+ (SEQ ID NOS: 70; 71) that had previously been digested with the appropriate enzymes to release the West Nile virus EIII+ insert. The deleted vector was then gel purified and ligated to an aliquot of JE EIEI+. The ligation mix was used to transform TOP-10 cells (InVitrogen. Carlsbad, CA) and the cells were grown on LB supplemented with 50 ug/ml kanamycin. Several colonies were screened by restriction mapping and were verified by DNA sequencing.<br>
The resulting construct, pET24A/STF2A.JEin (SEQ ID NOS: 5, 6) was BLR (DE3) strain (Novagen) and expression was monitored in several clones using Commassie Blue staining which was confirmed by Western blot using anti-flagellin antibodies. Using, pET24A/STF2A.JEIH+ as the DNA template and the JE EHI+ oligonucleotide as primer (SEQ ID NO: 53) the cysteine residue in the linker region between STF2A and JEITI+ was changed to a serine residue using QuikChange Site Directed Mutagenesis Kit (Stratagene, LaJolla, CA) according to the manufacturer's instructions. The clone was verified by sequencing and assayed for expression as described for pET24A/STF2A.JEItI+ above.<br>
When a cysteine residue in a linker in change to a serine residue the fusion protein in also referred to herein by includsion of an "s" in the designation of the fusion protein. For example, "STF2A.EIII+" includes a cysteine residue in the linker (Figure 29, SEQ ID NO: 71), whereas "STF2A.EEIS+" include a serine residue substituted for the cysteine residue in the linker (Figure 30, SEQ ID NO: 72).<br>
Cloning the Bill domain of each dengue virus flagellin fSTF2A)<br>
Initially, obtaining biologically active material from the fusion of the entire envelope protein of West Nile virus was difficult, perhaps due to the presence of multiple cysieines residues (12 cysteines) in the envelope protein (see SEQ ID NO: 39, Figure 45). However, when the region encoding domain m (EHT) of the protein was sub-cloned, the fusion protein was abundantly expressed in E. coli and was highly efficacious in mice. Although there is an overall sequence dissimilarity among the 4 distinct DEN viruses (Denl, Den2, Den3, Den4, SEQ ID NOS: 160-167, Figures 67-74) the three-dimensional structures within domain III of the envelope protein are similar among the flaviviruses. This domain in DEN and other flaviviruses encodes the majority of the type-specific contiguous critical/dominant neutralizing epitopes. Domain III of the dengue viruses (Denl, Den2, Den3 and Den4) has been expressed in bacteria and shown to be immunogenic, capable of inducing neutralizing antibodies in experimental animals (Simmons, M., et al, Am. J. Trap. MedHyg 6"5:159 (2001)). Domain IE corresponding to residues about 295 to about 399 (exact numbering depends on the particular DEN virus, for example, of SEQ ID NOS: 160, 162,164,166) of the four differentDEN viruses have been codon-optimized for expression in E. coli. The synthetic gene was amplified by using PCR and sub-cloned into the Not! site of the vector pET/STF2A generating pET/STF2A.DENlEm, pET/STF2A.DEN2EHI, pET/STF2A.DEN3EIII and pET/STF2A.DEN4EIII (SEQ ID NOS: 80, 82, 84 AND 86).<br>
E. coli production of STF2.EIII+. STF2A.EIIH-. STF2A.EIIIs+ and STF2A.JEIU+<br>
Cell cultures (6L) of BLR(DE3) pLysS that harbor pETSTF2.EIII+ (SEQ ID NOS: 54, 55), pETSTF2A.EIII+ (SEQ ID NOS: 70, 71), pETSTF2A.EIIIs+ (SEQ ID NOS: 72, 73) or pETSTF2A.JEHH- SEQ ID NOS: 5, 6) were grown in LB medium containing 15 fig/ml kanamycin, 12.5 ug/ml tetracycline and 24 ug/ml chloramphenicol. At an ODeoo of about 0.6 protein expression was induced with 1 mM EPTG for about 3 h at about 37°C.   Following induction, cells were harvested by centrifugation (7000 rpm x 7 minutes in a Sorvall RC5C centrifuge) and<br>
resuspended in 2x PBS, 1% glycerol, DNAse, 1 mM PMSF, protease inhibitor cocktail and 1 mg/ml lysozyme. The suspension was passed through a microfiuidizer to lyse the cells and the lysate was centrifuged (45,000 g for one hour in a Beckman Optima L ultracentrifuge) to separate the soluble fraction from inclusion bodies. Under these growth and induction conditions, STF2-.EIIH was expressed as a soluble protein and STF2A.EII1- (SEQ ID NOS: 70, 71), STF2A.EIIIS+ (SEQ ID NOS: 72, 73) and STF2A.JEIII+ (SEQ ID NOS: 5, 6) formed inclusion bodies.<br>
Purification of STF2.EIII+<br>
Cell lysate containing soluble STF2.EEI+ (SEQ ID NOS: 54, 55) was , applied to Sepharose Q resin (Amersham Biosciences, Piscataway, NT) in the presence of 0.5 M NaCl to reduce DNA, endotoxin, and other contaminants. The flow-through fraction was collected and the conductivity adjusted by a 10-fold dilution with buffer A (Buffer A: 100 mM Tris-CL pH 8.0). The diluted material was re-loaded onto Q Sepharose and bound protein was eluted with a linear gradient from 20% to 60% Buffer B (Buffer B: 100 mM Tris-Cl, 1 M NaCl, pH 8.0). Fractions containing STF2.EIII+ were pooled and further processed by Superdex-200 gel (SD200) filtration chrbmatography in the presence of-Na-deoxycholate to remove residual endotoxin (running buffer: 1% Na-deoxycholate, 100 mM NaCl, 100 mM Tris-HCl, 1% glycerol, pH 8.0). Following SD200 chromatography, the eluted protein was loaded directly onto Q Sepharose and washed extensively with buffer A to remove detergent. Bound protein was again eluted with a linear gradient from 20% to 60% Buffer B. In one preparation (Batch 057), this step was substituted with a detergent removal procedure using Extract-D detergent removal gel (Pierce Biotechnology, Rockford, IL). The purified protein was dialyzed against buffer containing 50 mM Tris, 100 mM NaCl and 1% glycerol and stored at -80°C.<br>
Purification of STF2A.EIII+<br>
STF2A.EIII+ inclusion bodies were collected by low-speed centrifugation (7000 rpra x 7 minutes in a Sorvall RC5C centrifuge) and solubilized with buffer containing 8 M urea, 1 OOmM Tris-HCl, 5 mM EDTA, pH 8.0. The urea    . concentration of the solubilized protein was adjusted to 1 M and the sample was loaded onto Q Sepharose. The bound protein was eluted using a linear gradient from 0% to 100% Buffer B. (Buffer A: 100 mM Tris-HCl, 5mM EDTA, 1 M urea; pH 8.0. Buffer B: 100 mM Tris-Cl, 5 mM EDTA, 1 M NaCl, 1 M urea, pH 8.0). Due to the formation of protein aggregates following elution, the urea concentration of the Q Sepharose material was adjusted to 8 M. The protein was further purified by gel filtration chromatography using SD200. The column was pre-equilibrated with 100 mM Tris-HCl, pH 8.0, 100 mM NaCl, 1% glycerol, 8 M urea plus 1% Na-deoxycholate. The eluted protein was subjected to a second IEX chromatography step using Source Q to remove 1% Na-deoxycholate. Bound protein was eluted with a linear gradient from 20% to 60% Buffer B. (Buffer A: 100 mM Tris-Cl, pH 8.0, 8 M urea, 5mM EDTA. Buffer B: 100 mM Tris-HCl, pH 8.0, 5 mM EDTA, 8 M urea, 1 M NaCl).   Final polishing of the protein was completed by gel filtration chromatography using SD200 (Running Buffer: 100 mM Tris-HCl, pH 8.0, 8 M urea, 100 mM NaCl and 1% glycerol). Reducing agent was added to the SD200 fraction (2.5 mM DTT) and the protein was refolded by step-wise dialysis against decreasing concentrations of urea. The urea concentration was reduced sequentially against buffers that contained 100 mM Tris-HCl, pH 8.0,100 mM NaCl, 1% glycerol and 6 M, 4 M, 2 M or no urea.<br>
Refolding and purification of STP2A.EITI+ trimer<br>
STF2A.EHR (SEQ ID NOS: 70, 71) from urea-solubilized inclusion bodies was efficiently refolded to form trimer product by simple dialysis as described above the trimer (3 of thes STFA.EHI fusion proteins) was deduced based on molecular weight in SDS-PAGE. Following dialysis, endotoxin was removed by multiple extractions with Triton X-l 14. The trimer was purified and separated from monomer and aggregates by S200 size exclusion chromatography. The final product migrated<br>
as a single band with an apparent molecular weight of about 130 kDa on SDS-PAGE.<br>
Refolding and purification of STF2A.EITK monomer<br>
The monomeric form of STF2A.EIII+ (SEQ ID NOS: 70, 71) was produced consistent!)' and efficiently by refolding using rapid dilution, which prevented<br>
I<br>
individual STF2A.EIII+ fusion proteins from interacting with one another to form men-timers, such as timers (supra). STF2A.EIH+ solubilized from inclusion bodies in 4M urea was raised to 8M urea without reductant. The protein was then rapidly diluted in Tris/NaCl/glycerol buffer, pH 8.0, to about 0.1 mg/ml and a final urea concentration of 0.1M at room temperature. The monomer was further purified and separated from aggregates by S200 size exclusion chromatography. The final product migrated as a single band with an apparent molecular weight of about 43 kDa on SDS-PAGE.<br>
Purification of STF2A.EIIIs+ (serine substitution of the linker between STF2A and EHI-r. SEO ID NO: 721<br>
STF2A.Enis+ (SEQ ID NOS: 72, 73) from solubilized inclusion bodies was refolded using a rapid dilution method similar to that used to refold the STF2A.EIII-i-monomer. The refolded protein was captured on a butyl sepharose column and eluted while removing most of the endotoxin contamination. Eluate from the butyl sepharose purification was concentrated and put through 4 cycles of Triton X-l 14 extractions to reduce endotoxin levels down to about 
Purification of STF2A.JEIII+ TSBO ID NOS: 5. 6)<br>
Protein was isolated from inclusion bodies under denaturing conditions. Inclusion bodies were washed with detergent (0.5% Triton X 100) and solubilized in 8 M urea, resulting in partial purification of the target protein. For endotoxin<br>
removal protein was applied on a Source S cation exchange column at low pH (about 3.5) and eluted with a salt gradient (0 to about IMNaCl). The protein was refolded using rapid dilution as described for STF2A.EIII+ monomer. The protein was then concentrated and further purified using SD200 to separate the monomeric form of the protein from aggregates. The purified material migrated with an apparent molecular weight of about 43 kDa on SDS PAGE and contained acceptable levels of endotoxin (about 0.03 EU/ug).<br>
Fed Batch Production of Fusion Proteins<br>
STF2A.EflIs+ was produced hi an aerobic bioreactor using a fed batch process. Three control loops were placed to control pH by acid (2 N HC1) or base (3 N NtLjOH) addition, temperature by heating (heating blanket) or cooling (time cycled cooling loop), and dissolved oxygen by compressed air flow (manually controlled), agitation (mixing speed) and C&gt;2 flow (tuned cycled) in cascade. Cells [BLR(DE3) pLysS that harbor the STF2A.EIIIs+ were adapted to and banked in MRSF media (see infra}, and frozen in 25% glycerol. Cells were scaled up for the bioreactor by adding 1 mL of banked cells to 1 L of MRSF media and agitating at about 37°C for about 15.5 to about 16.5 hours. Cells from the scale up process were added in a about 1:10 ratio to MRSF or MRBR synthetic media at about 37DC and about 0.5 wm air flow.<br>
The process was run in batch mode at about 37°C until the cells oxygen consumption was such that the compressed air flow is about 1.5 wm and the agitation is at the maximum, about 6 hours, when the temperature is dropped to between about 25°C and about 33°C. The feed can be started before the culture is induced, or up to about 1 and about 1/2 hours after. The feed rate can be kept constant, or adjusted based on process variables (dissolved oxygen, glucose concentration). The culture was induced with IPTG upon batch glucose exhaustion. The culture was maintained for a minimum of about 2 hours and a maximum of about 20 hours.<br><br>
(Table Removed)<br>
STF2A.EIIIs+ was produced as inclusion bodies. Upon harvest, the cells were separated from the conditioned media by centrifugation (Beckman Avanti J-20 XP, JLA 8.1000 rotor, 10 kxg for about 20 minutes at about 4°C) and resuspended in equal volume of 50 mM Tris, 100 mM NaCl, 1 mM EDTA, pH 8.0. The centrifugation was repeated under the same conditions, with the cells resuspended in a minimum volume of the same buffer. The suspension was passed through a homogenizer (APV-1000) at &gt; 10,000 psi for at least two passes.<br>
The solids can be separated and the STF2A.EIIIs solubilized by one of three methods; centrifugation, filtration, or fluidized bed chromatography.<br>
Method 1<br>
Solids are separated by centrifugation (Beckman Avanti J-20 XP, JA 20 rotor. 20 kxg for 20 minutes at 4°C) and resuspended in 50 mM tris. 1 m NaCl, ImM EDTA, 1% glycerol, 0.5% Triton X-100, pH 8.0, This process was repeated up to 6 times (total) at increasing speeds and times (to a maximum of about 40 kxg for about 20 minutes). After the final pellet recover}', the pellet was resuspended in 50 mM Tris: O.IM Nad, 1 mM EDTA, pH 8.0 and clarified by centrifugation (Beckman Avanti J-20 XP, J A 20 rotor, 40 kxg for about 20 minutes at about 4°C) The pellet was resuspended and dissolved in 50 mM Tris, O.IM NaCl, 1 mM EDTA. 4 M urea, pH 8.0. Insolubles were removed by centrifugation (Beckman Avanti J-20 XP, JA 20 rotor, 40 kxg for about 50 minutes at about 4°C). the supernatant retained for further processing.<br>
After the multiple washes described above, STF2A.EIH.S can also be dissolved in 50 mM acetate, 10 mM NaCl, 8M urea, pH about 4.1 to about 5.3 and clarified by centrifugation (Beckman Avanti J-20 XP, JA 20 rotor, 20 kxg for about 20 minutes).<br>
Method 2<br>
After homogenization, the lysate was captured in body feed and STF2A.EIEs+ extracted with urea containing buffer. Body feed is a filter aid designed to trap particles in a cake above a depth filter. The body feed (Advanced Minerals Corporation CelPure 65) is a diatomite (silica powder) with a high surface area and low permeability, retaining 
Method 3<br>
After the cells'were initially rssuspended in buffer, the}' were resuspended in sodium chloride and urea containing buffer at pH about 6 to about 8 and homogenized. The lysate was applied on a Streamline CST fluidized bed column (GE Healthcare) where the STF2A.EIIIs-f binds to the resin and the particulates flow through. STF2A.EIIIs+ may be eluted in low salt conditions at a pH greater than the load pH, in the presence or absence of detergents such as Triton X-100 or polysorbate 80.<br>
SDS-PAGE<br>
Proteins (typically about 5 ug) were diluted in SDS-PAGE sample buffer with and without p-mercaptoethanol. The samples were boiled for 5 minutes and loaded onto a 4-20% SDS polyacrylamide gel. Following electrophoresis, gels were stained with coomassie blue to visualize protein bands.<br>
Endotoxin assay<br>
Endotoxin levels were measured using the QCL-1000 Quantitative Chromogenic LAL test kit (BioWhittaker #50-648U, Walkersville. MD), following the manufacturer's instructions for the microplate method.<br>
Protein Assay<br>
Protein concentrations were determined by the MicroBCA Protein Assay Reagent Kit in a 96-well format using BSA as a standard (Pierce Biotechnology, Rockford, IL).<br>
TLR5 bioactivitv assay<br>
HEK293 cells (ATCC, Catalog No. CRL-1573 Manassas, Virginia) constitutively express TLR5, and secrete several soluble factors, including IL-8, in response to TLR5 signaling. Cells were seeded in 96-well microplates (about 50,000 cells/well), fusion proteins added and incubated overnight. The next day. the conditioned medium was harvested, transferred to a clean 96-well microplate.<br>
and frozen at -20°C. After thawing, the conditioned medium was assayed for the presence of EL-8 in a sandwich ELISA using an anti-human IL-8 matched antibody pair (Pierce, #M801E and #M802B, Rockford, EL) following the manufacturer's instructions. Optical density was measured using a microplate spectrophotometer (FARCyte, Amersham Biosciences, Piscataway, N.T).<br>
Plaque reduction neturalization test fPRNT)<br>
PRNT was performed according to Wang, el al., J. Immunol. 167:5273-5277 (2001). Briefly, serum samples were heat inactivated by incubation in a 56°C water bath for about 30 min and were serially diluted in PBS with 5% gelatin from 1/10 to 1/2560. West Nile virus was diluted in PBS with 5% gelatin so that the final concentration was about 100 PFU/well. Virus was mixed with about 75 ul serum in a 96-well plate at about 37°C for about 1 h. Aiiquots of serum-virus mixture were inoculated onto confluent monolayers of Vero cells in a six-well tissue-culture plate. The cells were incubated at about 37°C for 1 h, and the plates were shaken every 15 min. The agarose overlay was then added. The overlay was prepared by mixing equal volumes of a solution consisting of 100 ml 2x MEM (Life Technologies) with sterile 2% agarose. Both solutions were placed in a 40°C water bath for 1 h before adding the overlay. The cells were incubated for 4 days at 37°C in a humidified 5% CO2-air mixture. A second overlay with an additional 4% neutral red was added on day 5. Virus plaques were counted about 12 h later.<br>
Antigenicity of STF2A-fusion proteins<br>
ELISA plates (96-well) were coated overnight at 4°C with serial dilutions (100 ul/well) of purified STF2A-fusion proteins (SEQ ID NOS: 158, 159, 54, 55, 70, 71) in PBS (about 2ug/ml). Plates were blocked with 200 ul/well of Assay Diluent Buffer (ADB; BD Pharmingen) for one hour at room temperature. The plates were washed 3x in PBS-Tween, and then incubated with antibodies reactive with flagellm or the E domain of the construct. The expression of flagellin was detected using the mAb 6H11 (Intotek), while the antigenicity of WNV-E was monitored using a panel of mAb (5C5, 7H2,5H10,3A3, and 3D9) (Beasley, D.W., et al., J. Virol 76; 13097-<br>
13100 (2002)) were purchased from Bioreliance (Road Rockville, MD). Antibodies diluted in ADB (about 100)il/well) were incubated overnight at 4°C. The plates were washed 3x with PBS-T. HRP-labeled goat anti-mouse IgG antibodies (Jackson Immunochemical, West Grove. PA) diluted in ADB were added (lOOul/well) and the plates were incubated at room temperature for 1 hour. The plates were washed 3x with PBS-T. After adding TMB (3,3',5,5'-tetramentylbenzidine) Ultra substrate (Pierce Biotechnology, Rockford, IL) and monitoring color development, A^so was measured on a Tecan Farcyte microspectrophotometer.<br>
Immunization of mice<br>
C3H/HeN mice (10 per group) were immunized intraperitoneally or subcutaneously with me indicated concentrations effusion proteins or synthetic peptides on days 0. 14 and 28 . On days 21 and 35, immunized animals were bled by retro-orbital puncture. Sera were harvested by clotting and centrifugation of the heparin-free blood samples. On day 35, mice were challenged with a lethal dose of WNV strain 2741 (Wang, T., et al, J, Immunol 7(57:5273-5277 (2001)). Survival was monitored for 21 days post-challenge.<br>
Serum antibody determination<br>
West Nile envelope protein specific IgG levels were determined by ELISA. ELISA plates (96-wells) were coated overnight at about 4°C with 100 ul/well of West Nile E protein mAb 5C5, 7H2, 5H10, 3A3, and 3D9 (Beasley, D.W., et al, J. Viro. 7(5:13097-13100 (2002)) (Bioreliance, Road Rockville, MD) in PBS at a concentration of 2ug/ml. Plates were blocked with 200 ul/well of Assay Diluent Buffer (ADB; BD Phanningen, San Diego CA) for one hour at room temperature. The plates were washed 3x in PBS-T. Dilutions of the sera in ADB were added (lOOul/well) and the plates were incubated overnight at 4°C. The plates were washed'3x with PBS-T, HRP-labeled goat anti-mouse IgG antibodies (Jackson Immunochemical, West Grove. PA) diluted in ADB were added (lOOpl/well) and the plates were incubated at room temperature for 1 hour. The plates were washed 3x with PBS-T. After adding TMB (3:3',5.,5'-tetramentylbenzidine) Ultra substrate<br>
 (Pierce Biotechnology, Rockford, IL) and monitoring color development, A4so was measured on a Tecan Farcyte microspectrophotometer,<br>
Production of PamSCys.WNVOOl peptide synthesis<br>
PamSCys.WNVOOl was synthesized by Bachem Bioscience, Inc. (King of Prussia, PA). WNV001 is a 20 amino acid peptide (SEQ ID NO: 168) of the West Nile virus envelope protein chemically coupled to a tri-pahnitoylcysteine (PamSCys) moiety through the amino terminal serine residue of the peptide. The chemical name for Pam3Cys.WNV001 is [Paknitoyl-Cys((RS)-2,3-di(palmitoyloxy)-propyl)-LTSGHLKCRVKMEKLQLKGT (SEQ ID NO: 168) acetate salt]. The molecular mass of Pam3Cys.WNV001 is 3163.3 daltons. The peptide was synthesized by Bachem using solid phase synthesis methodologies andFMOC chemistry. The amino acid sequence of PamSCys.WNVQOl was assembled on an H-Pro-2-chlorotrityl chloride resin by solid phase peptide synthesis. The peptide chain was elongated by successive coupling of the amino acid derivatives. Each coupling step was preceded by an Fmoc-deprotection step and were accompanied by repeated washing of the resin. After coupling of the last amino acid derivative, the final Fmoc-deprotection step was performed. Finally, the peptide resin was washed and dried under reduced pressure. During solid phase peptide synthesis color indicator tests were performed for each step to monitor the completion of the Fmoc-cleavage and the subsequent coupling of the amino acid derivatives. To couple Pam3Cys-OH to the elongated peptide, the lipid moiety was pre-activated with N,N'-dicyclohexyl-carbodiimide (DCCI) in the presence of 1-hydroxybenzotriazole (HOBt). The resulting solution was filtered and added to the peptide resk. At the end of the reaction time the peptide resin was washed and dried under reduced pressure. Color indicator tests were performed to control the coupling of Pam3Cys-OH. The completed peptide was cleaved from the resin by incubating with trifluroacetic acid (TFA). The liberated product (crude peptide material) was precipitated from the reaction mixture and lyophilized, The crude product was used for initial immunogemcity studies.<br>
Svnthesis of W/NV-E pepti.de arrays<br>
Peptide arrays (Figures 57 and 60) were synthesized by Sigma Genosys (Woodlands, TX).<br>
RESULTS:<br>
West Nile fusion protein	<br>
West Nile virus (WNV) has emerged in recent years hi temperate regions of Europe and North America, presenting a threat to public and animal health. The most serious manifestation of WNV infection is fatal encephalitis (inflammation of the brain) in humans and horses, as well as mortality in certain domestic and wild birds. WNV has also been a significant cause of human illness in the United States. The envelope glycoprotein of West Nile (WNV-E) and other flaviviruses may generate neutralizing and protective antibodies. By linking this antigen to a Toll-like receptor ligand, the compositions, fusion proteins and polypeptides described herein ma3' target appropriate antigen presenting cells without the need for adjuvant or other immune modulator formulations.<br>
As described herein, several strategies have been implemented to facilitate production of West Nile virus envelope (WNV-E) fusion proteins in E. coll. One approach is to engineer a smaller WNV-E antigen by fusing domain El (EHI) and, optionally, with amino acids of domain II of the WNV-E protein to full-length STF2 (e.g.. STF2.E, STF2.EIII+). Domain III is responsible for virus-host interactions and retains many West Nile virus neutralizing antibody epitopes. It also contains only 2 of the 12 cysteine residues present within the full length envelope protein, making expression in E. coli more feasible. A second approach has been to delete the hyper-variable hinge region of flagellin (e.g., STF2A) thereby creating a smaller fusion protein (STF2A.EHI+) The hyper-variable region of flagellin is not required for TLR5 signaling and its removal may also reduce the irnmunogenic potential of flagellin. Both STF2.EIII- and STF2A.EIII+ have been expressed in E. coli and purified. The purified proteins have been characterized for TLR5 signaling activity in bioassays and for E epitope display in ELISA assays using a panel of WNV-E polyclonal and neutralizing monoclonal antibodies. Results from these studies<br>
indicate that STF2AJEID+ has higher PAMP activity and more conformation-sensitive neutralizing WNV-E epitopes than STF2.EDI+.<br>
Puritv of STF2.EIII+ and STF2A.EEI+<br>
Several lots of STF2.EEI+ and STF2A.EIU+ have been produced in E. coli and purified (Table 1). STF2.EIII+ was expressed as a soluble protein and purified under non-denaturing conditions using a 4~step process, as described above, that included anion exchange chromatography and gel filtration. Final yields from 6 L cultures ranged from about 0.9 mg to about 3.8 mg and all preparations contained low levels of endotoxin as measured by standard LAL procedures (about 
Table 1: Endotoxin levels and TLR-5 activity for STF2.EEI+ (SEQ ID NO: 55) and STF2A.EII3+ (SEQ ID NO: 71) fusion proteins.<br><br>
(Table Removed)<br>
TLR5 activity m the HEK293 IL-8 assay<br>
To compare the PAMP activity of-both fusion proteins, a TLR5 bioassay was performed. HEK293 IL-8 cells were treated with serial dilutions of two independent protein batches (Figures 47A and 47B). Cultures were incubated for e. 24 hour period and conditioned media were harvested and assayed for IL-8 production by ELISA. As shown in Figure 47A, STF2A.EIII+ showed potent TLR-5 activity. Regression analysis of the titration curve determined the ECso of batches 2004-044 and 2004-045 to be 1.13 ng/ml and 4.34 ng/ml, respectively (Table 1, supra). In both cases, the TLR5 specific-activity was at least about 10-fold higher than the control protein STF2.OVA. In contrast, 2 preparations of STF2.EIII+ showed significantly weaker TLR5 activity than STF2.0VA. The EC50 of STF2.EHI+ batches 054 and 057 were about 1195.00 ng/ml and about 197.92 ng/ml.<br>
Antigenicity of STF2.EIII- and STF2A.EIII+<br>
The antigenicity of STF2.EHR and STF2A.EIII+ was examined by direct ELISA using a fiagellin monoclonal antibody specific for the N-terminal region of STF2 (6H11. Inotek Pharmaceuticals, Beverly, MA) and a panel of WNV-E-specific antibodies (5C5, 5H10, 3A3, 7H2 and 3D9, Bioreliance, Road Rockville, MD ) previously shown to neutralize West Nile virus in vitro. As shown in Figure 48, a comparison of the reactivity of full length West Nile virus envelope protein with STF2A.EIII+ revealed that West Nile virus monoclonal antibodies 5C5: 5H10, 3A3 and 7H2, but not 3D9 recognize the fusion protein. This pattern of reactivity is<br>
consistent with the proposed location of 5C5, 5H10, 3A3 and 7H2 epitopes within EIII. The epitope for 3D9 lies outside of domain HI of the West Nile virus envelope protein. As expected, all West Nile virus monoclonal antibodies reacted with full length West Nile virus envelope protein and the flagellin monoclonal only reacted with STF2A.EHI-K Both proteins reacted with a polyclonal West Nile virus envelope antiserum, but STF2A.EITI+ reactivity was somewhat reduced, perhaps due to the reduced number of potential epitopes present in the smaller domain.<br>
Using 5C5 and 7H10 WNV monoclonal antibodies, a direct antigenic comparison was made between STF2.EIH+ and STF2A.EHI+ (Figures 49A, 49B, 49C and 49D). In these studies, plates were coated with the indicated proteins and then detected with polyclonal rabbit anti-E, or mouse monoclonal antibodies as described. As shown in Figures 49A, 49B, 49C and 49D, both STF2.EHI+ and STF2A.EIII+ were readily detected with the flagellin monoclonal antibody with no significant differences in reactivity. However, distinct reactivity with the anti-envelope monoclonal antibodies was observed. The reactivity of STF2A.EIH+ with either 5C5 or 7H2 was significantly greater than that observed with STF2.EIII+. Collectively, these results indicate that the flagellin 6H11 epitope of STF2A.EU3-f- is uncompromised and is comparable to the flagellin sequence of STF2.EIH+. They also highlight distinct differences in the antigenicity of the EDI domains of these proteins and indicate that STF2A.EIEB- contains more of the critical conformation dependent neutralizing epitopes than STF2.EIII+.<br>
Efficacy .and immunogenicirv<br>
Several efficacy studies designed to examine the protective efficacy our candidates in C3H/HeN mice following challenge with West Nile virus have been completed. Studies typically consisted of 5 groups of mice (10 mice per group) immunized imraperitoneall}' (i.p.) or subcutaneously (s.c.) on days 0, 14 and 28. On days 21 and 35. sera were harvested and tested for West Nile virus envelope protein - IgG antibody (ELISA) and the ability to neutralize West Nile virus in vitro (PKNT assay). On day 35, mice were challenged with a lethal dose of West Nile virus strain 2741. Survival was monitored for 21 days post-challenge.<br>
Mice were immunized with PBS, Drosophila conditioned medium containing STF2.E (CM, positive control), 25 ug of STF2A.EIII+ i.p., 25 \ig STF2A.EIIR E.G., 25 \ig STF2.EHR i.p. and 25 ug STF2.EIH+ s.c. The West Nile virus envelope protein antibody responses and survival data are shown Figures 50 and 51. By day 35 all groups that received STF2A.EIII+ had significant levels of West Nile virus envelope protein IgG. In contrast, mice that received STF2.EIH+ had no measurable West Nile virus envelope protein antibody response. Administration of STF2A.EIII+ i.p. or s.c led to 100% survival following West Nile virus challenge. Consistent with the poor immunogenicity of STF2.EHI+, little to no protection was provided by this candidate when compared to the PBS control. The poor immunogenicity and efficacy of STF2.EIH+ in this study are attributed to the reduced TLR5 activity and/or the weak EDI epitope reactivity of this protein.<br>
Plaque reduction neutralization liters<br>
To further evaluate the West Nile virus envelope protein antibody response elicited by STF2A.EIII+ and potentially correlate protective efficacy with neutralizing antibody liters, the plaque reduction neutralization test (PRNT) was performed. .Day 35 serum samples from efficacy studies described above were tested for their ability to block West Nile virus infection in cultured Vero cells. Briefly, pooled mouse serum samples were heat-inactivated and serially diluted twofold in PBS with 0.5% gelatin. Dilutions starting with 1:10 were incubated with about 100 pfu of the West Nile virus strain 2741. The virus/serum mixture was incubated at about 37°C for 1 h and then inoculated onto confluent monolayers of Vero cells (ATCC, Catalog Number CCL-81, Manassas, Virginia) in duplicate wells of 6-well tissue culture plates. The virus was allowed to adsorb to the cell monolayer prior to adding a 1% agarose overlay. Infected cell cultures were incubated for 4 days at 37°C followed by a second agarose overlay containing 4%<br>
t<br>
neutral red. Virus plaques were counted 12 h later. Serum titers that led to 80% reduction in viral plaque numbers (PRNTjo) were recorded.<br>
A summary of the PRNTgo data from efficacy studies concerning STF2.EIIH-and STF2A.EIII-T is presented in Table 2 below. In two independent studies<br>
involving STF2.EIQ+ where survival of about 50% or less was reported, pooled sera failed to inhibit plaque formation. This finding is not surprising given the weak antibody response elicited by this construct. In three efficacy studies involving STF2A.EIII+ where survival was about 70% or greater, pooled sera had neutralization liters of 1:40 or better. Neutralization liters of 1:40 or greater typically correlate with protection in vivo.<br>
Table 2: Survivial andPRNTgo Results for STF2.Effl-f (SEQ ID NO: 55), STF2A.EIII+ (SEQ ID NO: 71) and STF2.E (SEQ ID NO: 159) CM (Control Media) Fusion Proteins<br><br>
(Table Removed)<br>
STF2A.EIIIS4- a modified version of STF2A.EIII+<br>
Protein preparations of STF2A.EIII+ tested in the mouse efficacy studies described above were purified by anion-exchange and size-exclusion chromatography steps carried out under denaturing conditions followed by refolding using step-wise dialysis. With this process, two predominant species that correspond to the monomeric and trimeric forms of STF2A.EIII-i- were generated and present as a mixture in the final product. To minimize the heterogeneity of the final product, new refolding and purification methods have been developed that favor the production of either monomer or trimer. Because it is unclear which form of STF2A.EIII+ is the active component or if both are equally potent, both species have been produced in milligram quantities and tested for efficacy in mice.<br><br>
It was initially unclear as to why STF2A.EHI+ refolding resulted in the formation of a trim eric species. However, when the sequence of the STF2A.EEII+ expression construct was re-examineds we identified a cysteine residue within the linker sequence that separates STF2A from EEQ+. The presence of this cysteine would likely interfere with the formation of the appropriate disulfide bond during refolding and might account for the trimeric form of STF2A.EIE-K This unnecessary cysteine was changed to a serine using site-directed mutagenesis and the modified protein (STF2A.EIEs+) was produced and purified. It should be noted that refolding the serine-substituted construct yielded only monomeric protein.<br>
Protective efficacy of STF2A.EHI+ (monomer) and STF2A.EHls+ (trimer) were evaluated in C3H/HeN mice following challenge with West Nile virus. Five groups of mice (10 per group) were immunized with about 25 ug of protein s.c. on days 0, 14 and 28. On days 21 and 35, sera were harvested and tested for WNV-E IgG antibody (ELISA), On day 38, mice were challenged with a lethal dose of WNV strain 2741 and survival was monitored for 21 days. ELISA results from boost 2 (day 35, Figure 52) and survival data (Figure 53) indicate that all constructs elicited significant levels of WNV-E reactive IgG prior to viral challenge and provided about 90% to about 100% protection against the lethal infection. These findings indicate that monomeric or multimeric (e.g., trimers) forms of STFA.EIII+ are efficacious and removal of the additional cysteine from the construct does not appreciably impact potency. Removal of the cysteine within the linker sequence may simplify purification of the protein by reducing heterogeneity following protein refolding.<br>
Conclusion<br>
Two recombinant fusion proteins containing the Salmonella typhimurium flagellin (STF2) fused to EU3+ domain of West Nile virus envelope protein have been generated. One includes the full length STF2 sequence (STF2.EE+) and the other a modified version of STF2 that lacks the internal hypervariable region of STF2 (STF2A.EDI+). Both proteins have been expressed in E. coli and purified by conventional means using anion exchange chromatography and gel filtration.<br>
STF2.EIII+ was produced as a soluble protein and was purified under non-denaturing conditions. In contrast, STF2A.EIII+ was expressed as an insoluble protein and was purified under denaturing conditions and refolded by step-wise dialysis to remove urea. In HEK293 IL8 assays, preparations of STF2A.Effl-i-showed greater TLR-5 activity than STF2.EH.I-K<br>
In envelope protein epitope display analysis using ELISA assays and West Nile virus envelope prtoein antibodies, STF2A.EIE+ displayed more of the critical conformation dependent neutralizing epitopes. Consistent with the potent TLR-5 activity and envelope protein epitope anugenicity observed with STF2A.EIIB-, STF2A.EHI+ was highly immunogenic and efficacious in mice challenged with a lethal dose of West Nile virus. Because monomeric and trimeric species of STF2AJEIIH- were generated during the purification process of this protein, a cysteine within the linker sequence of the expression construct was changed to a serine. Removal of this cysteine eliminated the production of trimeric forms of the protein during refolding and resulted hi the generation of monomeric product that displayed potent efficacy in vivo. .<br>
Japanese encephalitis fusion protein<br>
JE virus is localized in Asia and northern Australia (about 50,000 cases with about 10,000 deaths annually). An approved inactivated virus vaccine was recently associated with a case of acute disseminated encephalomyelitis, prompting the Japanese Ministry of Health, Labor and Welfare to recommend the nationwide suspension of the vaccine. Given the complexities of producing inactivated viruses in infected mouse brains or even in cell culture, and the potential for adverse events associated with inactivated viruses, the opportunity for recombinant-based JE vaccine is appealing.<br>
A STF2A.JEIE+ fusion construct was constructed. The JE EHI+ DNA<br>
V<br>
fragment was generated synthetically and. codon optimized for expression in E. coli. The sequence was ligated into pET24STF2A to generate pETSTF2A.JEHI+. Expression constructs have been screened by restriction analysis and for expression in £. coif BLR(DE3) by FPTG induction. The DNA sequence of each construct has<br>
been confirmed, and production of the protein has been scaled up. A batch of material has been generated. A total of about 24 mg of material was purified. This material has potent TLR5 activity, acceptable levels of endotoxin (about 0.03 EU/ug) and a A280/A260 ratio of about 1.3.<br>
Flavivirus peptides<br>
Identification of WNV-E specific antibody epitopes<br>
To identify linear epitopes •within the West Nile virus envelope protein that are recognized by antisera from STFA.EIIIs+ immunized mice, several synthetic peptide arrays were generated. One array consisted of overlapping peptides of 20 amino acids in length that spanned the entire West Nile virus domain IH and parts of domain II (Figure 60). ELISA results with this array identified a highly reactive 20 amino acid sequence that mapped to the N-terminal region of domain III and included part of the domain I domain CRVKMEKLQLKGTTYGVCSK (SEQ ID NO: 125), To fine map this epitope, additional arrays were generated that focused on the domain I and n junctions (Figures 57 and 60). These arrays included an alanine substitution scan to identify ammo acids critical for antibody binding (Figure 60). As shown in Figures 54 and 55, antisera from STF2A.EHI (monomer and trimer) and STF2A.EIIIs+ immunized mice reacted with peptides that spanned the El/EDI junction (peptides E-30 to E-42) and included the E2-21 peptide CRVKMEKLQLKGTTYGVCSK (SEQ ED NO: 125). This reactivity was severely reduced when specific amino acids (E6, K7, L10 and Kl 1) were changed to alanines (Figure 56). Although it is not known if the antibodies that recognize this epitope are neutralizing, efforts are underway to design and test a peptide vaccine based on this region of WNV-E.<br>
Immunogenicitv of Pam3Cys.,WNV001 peptide vaccine<br>
A lipidated West Nile virus envelope protein fused to PamSCys on the N-termmal end was synthesized using the 20 amino acid sequence LTSGHLKCRVKMEKLQLKGT (SEQ ED NO: 169) (Putnak, R., et al, Vaccine 23:4442-4452 (2005)). The immunogenicity of this peptide was tested in C3H/HeN<br>
mice and compared to peptide without PamSCys (Figure 58). The reactivity of antisera from immunized animals was tested by direct ELIS A as described in the legend and the results indicate that the PamBCys.WNVOOl peptide is significantly more immunogenic than the peptide without the TLR2 modification. The antisera from these studies will be tested in virus neutralization assays (PRNT) to determine if the antibodies elicited will neutralize West Nile virus in vitro. The lipidated peptide will also be tested in the West Nile virus challenge model to assess protective efficacy against a lethal virus challenge.<br>
Assay Development<br>
Competition ELISA assay development<br>
To assess the neutralizing potential of antisera derived from immunized mice, a competition ELISA assay was developed using well-characterized monoclonal antibody (7H2) that neutralizes West Nile virus in culture and reacts with a conformation-sensitive epitope within the EIH domain of the West Nile virus envelope protein antigen. The assay was designed as a capture ELISA that measures the ability of sera from immunized animals to prevent 7H2 from binding West Nile virus envelope protein. S erial dilutions ranging from 1:10 to 1:5 000 of day 3 5 mouse antisera from efficacy study 4 (Figures 50 and 51, Table 2) were incubated with biotinylated West Nile virus envelope protein and then added to ELISA plates pre-coated with 7H2 monoclonal antibody (Bioreliance, Road Rockville, MD). Following several washes to remove unbound material, bound West Nile virus envelope protein was detected using avidin-HRP. Results from a representative experiment are shown in Figure 54, At dilutions of 1:25, a measurable loss of West Nile virus envelope protein binding to 7H2-coated plates was observed when antisera derived from animals immunized with STF2A.EHIs where tested. No<br>
competition was detected with antisera derived from mock immunized animals that<br>
received PBS in place of antigen. These initial results demonstrate that antibodies elicited by STF2A.EHI+ compete with 7H2 for binding Wests Nile virus envelope protein. These findings are consistent with the-protection from WNV infection<br>
observed iii animals immunized with STF2A.EIH+ and help establish a correlation between antibody epitope reactivity in vitro and efficacy in vivo.<br>
EQUIVALENTS<br>
While this invention has been particularly shown and described with   . references to preferred embodiments thereof; it will be understood by those skilled in the art that various changes in form and details may be made therein without departing from the scope of the invention encompassed by the appended claims.<br><br><br><br><br>
WE CLAIM:<br>
1.	A fusion protein comprising at least a portion of at least one flagellin and at least a portion of at least one Den2 viral envelope protein, wherein the Den2 viral envelope protein is at least one member selected from the group consisting of SEQ ID NO: 22, SEQ ID NO: 40 and SEQ ID NO: 97, wherein the fusion protein elicits an immune response that neutralizes a Dengue 2 virus.<br>
2.	A fusion protein comprising at least a portion of at least one flagellin and at least a portion of at least one member selected from the group consisting of a Den 1 viral envelope protein, a Den2 viral envelope protein, a Den3 viral envelope protein and a Den4 viral envelope protein, wherein the fusion protein elicits an immune response that neutralizes at least one of a Dengue 1, 2, 3 and 4 virus.<br>
3.	The fusion protein as claimed in claims 1 or 2, further including a linker between the flagellin and the viral protein.<br>
4.	The fusion protein as claimed in claim 3, wherein the linker is an amino acid linker.<br>
5.	The fusion protein as claimed in claim 4, wherein the amino acid linker includes SEQ ID NO: 26, SEQ ID NO: 28, SEQ ID NO: 60 or SEQ ID NO: 62.<br>
6.	The fusion protein as claimed in claims 1 or 2, further including a linker between at least two flagellin proteins.<br>
7.	The fusion protein as claimed in claims 1 or 2, further including a linker between at least two viral proteins.<br>
8.	The fusion protein as claimed in claims 1 or 2, wherein the flagellin is fused to a carboxy-terminus of the viral protein.<br>
9.	The fusion protein as claimed in claims 1 or 2, wherein the flagellin is fused to an amino-terminus of the viral protein.<br>
10.	The fusion protein as claimed in claims 1 or 2, wherein the flagellin is fused between at<br>
least two viral proteins.<br>
11.	The fusion protein as claimed in claims 1 or 2, wherein the viral protein is fused between at least two flagellin proteins.<br>
12.	The fusion protein as claimed in claims 1 or 2, wherein the viral protein is fused to the flagellin in a region of the flagellin that lacks at least a portion of a hinge region of the flagellin.<br>
13.	The fusion protein as claimed in any of the claims as and when used in the manufacture of a medicament to prevent or treat disease consequent to a Dengue virus.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUFic3RyYWN0LSgxMC0wMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Abstract-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNsYWltcy0oMDktMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Claims-(09-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUNsYWltcy0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Claims-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence Others-(09-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Correspondence Others-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence Others-(15-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence Others-(16-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence Others-(22-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence Others-(24-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDEtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence-Others-(01-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTQtMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence-Others-(14-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Correspondence-Others-(27-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LURyYXdpbmdzLSgxMC0wMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Drawings-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUZvcm0tMS0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Form-1-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUZvcm0tMi0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Form-2-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUZvcm0tMy0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Form-3-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUZvcm0tMy0oMTUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Form-3-(15-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUZvcm0tMy0oMjQtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Form-3-(24-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LUZvcm0tMy0oMjctMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-Form-3-(27-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LUdQQS0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-GPA-(10-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0yMDYucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-206.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0yMTcucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-217.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1kZWxucC0yMDA3LXBjdC0zMTEucGRm" target="_blank" style="word-wrap:break-word;">5594-delnp-2007-pct-311.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU5NC1ERUxOUC0yMDA3LVBldGl0aW9uLTEzNy0oMTAtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">5594-DELNP-2007-Petition-137-(10-02-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263434-improved-ball-penetrometer-for-soft-soils-testing.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263436-apparatus-and-method-of-transmitting-receiving-mbms.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263435</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5594/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>VAXINNATE CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>3 CEDAR BROOK DRIVE,CRANBURY, NEW JERSEY 08512, USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THOMAS J.POWELL</td>
											<td>154 DORSET LANE, MADISON, CONNECTICUT 06443, USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VALERIAN NAKAAR</td>
											<td>338 HUME DRIVE, HAMDEN, CONNECTICUT 06514, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LANGZHOU SONG</td>
											<td>6 PETUNIA PLACE, FREEHOLD, NEW JERSEY 07728, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>JAMES W.HULEATT</td>
											<td>5 EVERGREEN PLACE, BRANDFORD, CONNECTICUT 06405, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>WILLIAM F.MCDONALD</td>
											<td>343 WARPAS ROAD, MADISON, CONNECTICUT 06443, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>DUANE D.HEWITT</td>
											<td>24 QUEENSDALE AVENUE EAST, HAMILTON, ONTARIO L8S 1K1, CANADA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/001623</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-01-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/725,919</td>
									<td>2005-10-11</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/704,160</td>
									<td>2005-07-29</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/645,170</td>
									<td>2005-01-19</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/723,409</td>
									<td>2005-10-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/653,405</td>
									<td>2005-02-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263435-fusion-proteins-comprising-flagellin-and-dengue-virus-envelope-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:51:04 GMT -->
</html>
