<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263425-production-of-recombinant-proteins-by-autoproteolytic-cleavage-of-a-fusion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:54:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263425:&quot;PRODUCTION OF RECOMBINANT PROTEINS BY AUTOPROTEOLYTIC CLEAVAGE OF A FUSION PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PRODUCTION OF RECOMBINANT PROTEINS BY AUTOPROTEOLYTIC CLEAVAGE OF A FUSION PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed is a method for the production of a heterologous polypeptide of interest with a homogenous N-terminus, using a fusion polypeptide comprising the polypeptide of interest and N-terminally thereto a polypeptide exhibiting autoproteolytic .function, said method comprising the steps of a) binding of the fusion polypeptide in a soluble, autoproteolytically inactive form by an affinity chromatography system, b) refolding of the fusion polypeptide, thereby activating the autoproteolytic function of the fusion polypeptide and causing cleavage of the heterologous polypeptide of interest, and c) subsequently eluting the heterologous polypeptide of interest, wherein said steps are conducted on one affinity chromatography system.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Organic Compounds<br>
Field of invention<br>
The present invention relates to a method for the production of a heterologous recombinant polypeptide of interest with a defined homogenous N-terminus. The present invention combines a chromatography system with a fusion polypeptide that comprises the polypeptide of interest and an additional part, a second polypeptide, which exhibits an autoproteolytic function and which is linked to the N-terminus of the polypeptide of interest. The chromatography system which forms part of the present invention allows for the activation of the autoproteolytic function of the N-terminal part of said fusion polypeptide while the fusion polypeptide is bound to the chromatography system. Binding, refolding and cleavage of the fusion polypeptide are conducted in the same chromatography system, from which the polypeptide of interest can then be isolated in its purified form.<br>
Background of invention<br>
While most polypeptides of interest, e.g. pharmaceutically useful proteins, originate from eukaryotes, they are, due to high expression rates and high yields, usually produced in bacterial cells. However, the mechanism of polypeptide synthesis in bacteria differs from that in eukaryotes; polypeptides expressed in bacterial cells usually have either an additional foreign amino acid at the N-terminus or are inhomogenous in respect to their N-terminus, since cleavage of the additional amino apid can occur but remains incomplete most of the time.<br>
Such inhomogeneity is however unacceptable in particular in the pharmaceutical field, because these polypeptides show properties that are different from the properties of the naturally occurring polypeptide, e.g. induction of antibody formation, half-life, pharmacokinetics etc. An N-terminus that deviates from the naturally occurring protein and/or is inhomogenous is an unacceptable feature. For the production of pharmaceutical polypeptides it is in most cases necessary to produce a nature-identical product (homogeneous with the correct N-terminus, which has no additional amino acids. The known methods attempt to reach this goal by incorporating additional steps in the process of<br><br>
polypeptide production, with expenditure of costs and materials, making further work up, the so-called downstream processing of the product, more complex.<br>
Known methods for the production of a polypeptide in bacterial cells with a defined, homogenous N-terminus employ a fusion polypeptide comprising the poiypeptide of interest and, N-terminally linked thereto, a polypeptide with autoproteolytic activity, preferably the autoprotease Npro of pestivirus. The autoproteolytic activity of the fusion partner leads to the cleavage of the polypeptide of interest with a homogenous N-terminus.<br>
If a polypeptide is produced in the cytoplasm of bacterial cells, under certain conditions, the production rate of the polypeptide is faster than the folding kinetics. Therefore high density polypeptide aggregates are formed, which are deposited in the cytoplasm of the cell as inclusion bodies. The production of polypeptides in the form of inclusion bodies is of special interest for production on industrial scale, since the expressed polypeptide is present in the inclusion bodies in high amounts and a high degree of purity. Also, the inclusion bodies of the cell lack proteases, so that the polypeptide is protected when stored in inclusion bodies. In addition, inclusion bodies are easy to isolate. However, major drawbacks of production of polypeptides in the form of cytoplasmatic inclusion bodies are low solubility of the inclusion bodies and the necessity to refold the polypeptide.<br>
Accordingly, processing of inclusion bodies is complex, especially since correct refolding is required in order to gain the biologically active form of the polypeptide of interest. Therefore, although the use of the autoproteolytic activity of a fusion polypeptide as described above consistently leads to the production of a polypeptide with a homogenous N-terminus, the process of purification of the desired product remains tedious, especially if it is expressed in form of cytoplasmatic inclusion bodies. The processing involves numerous steps including washing, refolding, cleavage, purification, and isolation.<br>
Thus, the complex downstream processing poses a big challenge with regard to fast and cost effective production of polypeptides. This is exceedingly the case for production on industrial scale. Accordingly, there is an ongoing need for a simple and feasible process for production and purification of polypeptides.<br>
Summary of the invention<br><br>
Within the scope of the present invention it has now surprisingly been found that a process for obtaining a heterologous polypeptide of interest e.g. from inclusion bodies, can be greatly facilitated by the combination of a specific affinity chromatography method with a fusion polypeptide system, which exerts autoproteolytic activity. Thus the processing can be conducted in a concerted action in one chromatography system.<br>
Initially the fusion polypeptide is provided. This fusion polypeptide comprises a polypeptide having autoproteolytic function, preferably the autoproteolytic function of an autoprotease, more preferably the autoproteolytic function of the autoprotease N^of Pestivirus, as well as derivatives thereof. C-terminally of the polypeptide with autoproteolytic function, said fusion polypeptide comprises the heterologous polypeptide of interest.<br>
The fusion polypeptide is produced in a host cell under conditions that inhibit the autoproteolytic activity of its N-terminal part. In particular the fusion polypeptide is produced in a denatured form in cytoplasmatic inclusion bodies. These inclusion bodies are isolated from the cells and solubilized under conditions that preserve the inactivity. Then the fusion polypeptide is selectively bound to a chromatography system, in particular a column under conditions that keep the N-terminal part of the fusion polypeptide in its inactive, denatured state.<br>
Once the fusion polypeptide is bound, if necessary, unbound contaminating components are washed out.<br>
When only the fusion polypeptide is left bound to the chromatography system, the conditions in the thus purified system are changed from inhibiting to activating the autoproteolytic function. This change of conditions allows the fusion polypeptide to regain its native configuration, whereby the autoproteolytic function of its N-terminal part is activated and the polypeptide of interest is cleaved off, resulting in the elution of an already purified, refolded polypeptide of interest with a homogenous N-terminus, while the N-terminal part remains bound to the chromatography system.<br>
Once the fusion polypeptide is bound, the steps of 1) washing out of unbound contaminating components, 2) refolding and 3) cleaving of the polypeptide of interest as well as 4) purifying<br><br>
the polypeptide of interest take place in the same chromatography system. This greatly facilitates the work up. Unbound contaminating components are easily washed out within this system, while the fusion polypeptide stays selectively bound to the chromatography system.<br>
Detailed description of the invention<br>
A new method for the production of a heterologous polypeptide of interest with a homogenous N-terminus is provided, which greatly reduces the complex processing usually necessary in order to acquire active polypeptide.<br>
Accordingly, the present invention relates to a method for the production of a heterologous polypeptide of interest with a homogenous N-terminus, using a fusion polypeptide comprising the polypeptide of interest and N-terminally thereto a polypeptide exhibiting autoproteolytic function, said method comprising the steps of a) binding of the fusion polypeptide in a soluble, autoproteolytically inactive form by an affinity chromatography system, b) refolding of the fusion polypeptide, thereby activating the autoproteolytic function of the fusion polypeptide and causing cleavage of the heterologous polypeptide of interest, and c) subsequently eluting the heterologous polypeptide of interest, wherein said steps are conducted on one affinity chromatography system.<br>
As used herein the following terms shall have the meanings described below:<br>
The term "heterologous polypeptide of interest" refers to a polypeptide which is not naturally cleaved by a naturally occurring autoprotease from a naturally occurring (fusion) polypeptide. Examples of heterologous polypeptides of interest are industrial enzymes (process enzymes) or polypeptides with therapeutic activity, in particular for the therapy of humans.<br>
The term "fusion polypeptide" refers to a polypeptide consisting of two or more polypeptides. In particular, a fusion polypeptide may comprise an affinity tag, an autoproteolytic part, preferably an autoprotease, and a polypeptide of interest In the meaning of the present invention, the fusion polypeptide comprises the polypeptide of interest and, N-terminally linked thereto, a polypeptide with an autoproteolytic function.<br>
The term "denatured form" in the meaning of the present invention designates the biologically inactive form of the expressed fusion polypeptide, as obtained as a product of<br><br>
the recombinant production process, usually as obtained after solubilising the inclusion bodies.<br>
The term "refolding" refers to the mechanism during which the solubilized polypeptide regains its native conformation and biological activity, i.e. reconstituting a protein from its denatured, inactive state to its active form.<br>
The term "autoproteolytic function" refers to the autoproteolytic activity of one of the fusion partners, which is inhibited while the fusion polypeptide is in its denatured state and which is activated upon refolding of the fusion polypeptide.<br>
The fusion polypeptide is bound to the chromatography system in a state, when its autoproteolytic functioning part is inactive. Binding has to be such that it is maintained during the change of conditions, cleavage of the polypeptide of interest, and thereafter. Within the scope of the present invention the initiation of cleavage is accomplished while the fusion polypeptide is refolded, whereby it is transferred from the Inactive to the active state. The present invention provides a chromatographic affinity system, which establishes binding of the fusion polypeptide at its N-terminus under denaturing conditions, and maintains binding of the fusion partner that exerts the autoproteolytic function through all changes of conditions which follow. Since the refolding takes place while the polypeptide is bound to the chromatography system, an additional requirement is that the affinity system does not interfere with the refolding process. Also this problem is solved by the present invention.<br>
As used herein the term "denaturing" shall mean conditions under which the native three dimensional structure of a polypeptide is disrupted.<br>
In the autoproteolytically active part of the fusion polypeptide, refolding leads to an activation and thus initiation of cleavage. Simultaneously the polypeptide of interest part regains its native conformation, consequently, the cleaved polypeptide of interest is in its native, active form. Since the autoproteolytically active part of the fusion polypeptide remains bound to the column after cleavage, and since unbound contaminating components are washed from the column prior to initiation of cleavage, already purified, refolded polypeptide of interest is eluted from the column. Thus further work up for separation of the two parts of the fusion polypeptide, separation from cleavage agent or refolding is dispensable.<br><br>
The fusion polypeptide within the scope of the present invention is provided by production within a bacterial host cell, in an initially inactive form.<br>
In a preferred embodiment of the present invention the fusion polypeptide is provided by recombinant expression in a bacterial host cell in the form of inclusion bodies, utilizing a host cell that is transformed with an expression vector comprising a nucleic acid molecule which codes for the fusion polypeptide.<br>
As used herein the term "inclusion bodies" shall refer to aggregates containing heterologous polypeptides present in the cytoplasm of transformed host cells. These appear as bright spots under the microscope and can be recovered by separation of the cytoplasm.<br>
As used herein the term "transformed host cell" shall refer to a cell containing a vector coding for a heterologous polypeptide.<br>
In order to initiate cleavage on the column, the autoproteolytic activity of the fusion polypeptide has to be inhibited from the start, already during expression of the polypeptide within the host cell. Expression under conditions that cause the deposition of the expressed polypeptide in the cytoplasm of the host cell, usually in form of inclusion bodies, the prerequisite of inactivity is met.<br>
A bacterial host cell to be employed in accordance with the present invention can be, for example, gram-negative bacteria such as Escherichia species, for example E. coli, or other gram-negative bacteria, for example Pseudomonas sp., such as Pseudomonas aeruginosa, or Caulobacter sp., such as Cautobacter crescentus, or gram-positive bacteria such as Bacillus sp., in particular Bacillus subtilis. E. coli is particularly preferred as host cell.<br>
The expression vector used in the method of the present invention comprises a nucleic acid molecule, which codes for a fusion polypeptide, comprising a polypeptide which exhibits autoproteolytic function and C-terminally thereto the polypeptide of interest The cleavage is exerted at the C-terminal end of the autoproteolyticatly active polypeptide, resulting in a homogenous N-terminus of the desired polypeptide.<br><br>
In a preferred embodiment of the present invention the polypeptide exhibiting autoproteolytic function is an autoprotease.<br>
As used herein the term "autoprotease" shall refer to a polypeptide that possesses autoproteolytic activity and is capable of cleaving itself from a larger polypeptide moiety, preferably a naturally occurring autoprotease. The concept of autoproteases as such is well available to the skilled man in the art; many naturally occurring autoprotease systems are known. Well studied autoprotease systems are e.g. viral proteases, developmental proteins (e.g. HetR, Hedgehog proteins (the carboxyterminal autoprotease thereof), RumA autoprotease domain, UmuD, etc).<br>
Viruses within the Flaviviridae, including the pestiviruses all have the NS3 proteases in common. It has been shown in yellow fever, dengue type 2 and West Nile viruses that the protease domain is located in the N-terminal ~ 180 residues of NS3 and is responsible for cleavage at the NS2A/2B and NS2B/NS3 junctions in an apparent intramolecular fashion. Analysis of hepatitis C and GB virus NS3 sequences showed a close relationship with flavi-and pestivirus NS3 sequences.<br>
An N-terminal autoprotease is also encountered in aphthoviruses [foot-and-mouth disease virus (FMDV)], which are positive-strand RNA viruses belonging to the family Picomaviridae. This proteinase, also referred to as leader protease (Lpro), belongs to the papain family of cysteine-proteases. In addition to cleaving itself off the polyprotein, it causes the proteolytic degradation of the 220-kDa subunit of the eukaryotic initiation factor 4G and thus contributes to the shutoff of cap-dependent host cell protein synthesis. Since picomaviral RNA is not capped, it continues to be translated as the cap-binding protein complex is inactivated. However, the aphthovirus leader proteinase gene is not required for viral replication in cell culture.<br>
Two other autoproteases of the picomavirus family are 2A and 3C, which have significant identity with chymotrypsin-like serine-proteases. Both proteases are contained within the polyprotein precursor. One short example for autoproteolysis in plant viruses is the leader proteinase of the Beet yellows virus, which possesses a non-conserved N-terminal domain (functions in RNA amplification) and a conserved, papain-like C-terminal domain required for autoproteolysis.<br><br>
Autoproteases can also be found in retroviruses, such as in the Gag-Pol polyprotein of human immunodeficiency virus (HIV-1). The polyprotein comprises a 99 amino acid rotease that releases itself after dimerization with a second protease from another polyprotein.<br>
More preferably the term "autoprotease" refers to the autoprotease Npro of a pestivirus, including all derivatives thereof with autoproteolytic activity.<br>
The present invention relates to a further embodiment, wherein the autoprotease is Npro of a pestivirus, or a derivative thereof with autoproteolytic function.<br>
Pestiviruses are small enveloped viruses with a genome which acts directly as mRNA. Two virus-encoded proteases that have been identified in Pestiviruses are the autoprotease Npro and the serine protease NS3. The protease Npro is located at the N-terminus of the polyprotein. Npro constitutes the first protein in the polyprotein of Pestiviruses and undergoes autoproteolytic cleavage from the following nucleocapsid protein. This cleavage takes place after the last amino acid in the sequence of N**0, Cys168.<br>
Pestiviruses form a group of pathogens which include amongst others, the classical swine fewer virus, (CSFV), the border disease virus (BDV) and the bovine viral diarrhoea virus (BVDV).<br>
Accordingly in a more preferred embodiment of the present invention, the pestivirus is selected from the group of CSFV, BDV and BVDV, with CSFV being particularly preferred.<br>
In an even more preferred embodiment of the present invention the autoprotease Npro of CSFV has the following amino acid sequence:<br>
SEQIDNO1:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFCEVTKRIGR VTGSDGKLYHIWCVDGCILLKLAKRGTPRTLKWIRNFTNCPLWVTSC-(168),<br>
or the amino acid sequence of a derivative thereof with autoproteolytic function.<br><br>
See also EMBL database accession number X87939, amino acids 1 to 168, reading from N-terminal to the C-terminal direction.<br>
Derivatives with autoproteolytic function according to the present invention are derived from the autoprotease Npro of a pestivirus by mutagenesis, in particular amino acid substitution, deletion, addition and/or amino acid insertion, as long as the required autoproteolytic activity, in particular for generating a desired heterologous polypeptide of interest with homogeneous N-terminus, is retained. Methods for generating such derivatives by mutagenesis are familiar to the skilled person. It is possible by such mutations to adapt the properties of the autoprotease Npro, in relation to different heterologous polypeptides to be cleaved from the fusion polypeptide. In particular a polypeptide within the scope of the present invention can be designed to have improved properties compared to the originally occurring autoprotease, while still exhibiting the autoproteolytic activity of Npro, of a pestivirus. Particularly preferred in this respect are derivatives, which show improved properties in respect of solubility as well as superior binding to the chromatographic affinity system, which properties are especially useful within the context of the present invention.<br>
The autoproteolytic properties of derivatives obtained by mutagenesis can be tested as described e.g. in WO 01/11056.<br>
Derivatives of the naturally occurring Npro, of a pestivirus as depicted in sequence ID No 1 above are especially preferred, wherein cysteine residues are replaced. More preferred in this respect are derivatives of the naturally occurring Npro, wherein the three cysteine residues C112, C134 and C138 are replaced by other amino acid residues, e.g. glutamic acid. A particularly preferred derivative comprises the following amino acid sequence:<br>
SEQ ID NO 2:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPNATQDYTGPVYHRAPLEFFDEAQFEEVTKRIGR VTGSDGKLYHIWEVDGEILLKLAKRGTPRTLKWIRNFTNCPLWVTSC-(168)<br>
Another preferred derivative of the naturally occurring Npro of a pestivirus, is the one wherein in addition to the cysteine mutations described above, the arginines in positions 53 and 57<br><br>
are exchanged to glutamic acid residues, glycine 54 is exchanged to aspartic acid and leucin 143 to glutamine. This derivative comprises the following amino acid sequence:<br>
SEQ ID NO 3:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFEEVTKRIGR VTGSDGKLYHIWEVDGEILLKQAKRGTPRTLKWIRNFTNCPLWVTSC-(168)<br>
Thus in another aspect the present invention also relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro, of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, at least one hydrophobic amino acid residue is replaced by a hydrophilic residue.<br>
Preferred within the present invention is a derivative of an autoprotease Npro, of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above furthermore at least one of the following amino acids are replaced: V24, A27, L32, G54, L75, A109, V114, V121, L143, 1155 and F158. A preferred example is a derivative wherein the following amino adds are exchanged by threonine (T): alanine (A) 109, valine (V) 114, isoleucine (I) 155 and phenylalanine (F)158.<br>
Thus in another aspect the present invention relates preferably to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro, of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, the following amino acids are replaced by threonine (T): alanine (A) 109, valine (V) 114, isoleucine (1)155 and phenylalanine (F)158. Another, within the present invention more preferred derivative of an autoprotease Npro, of CSFV, comprises the following amino acid sequence:<br>
SEQ ID NO 4:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETTKRIGRV TGSDGKLYHIYVEVDGEILLKLAKRGTPRTLKWTRNTTNCPLVWTSC-(168)<br><br>
Thus in another aspect the present invention more preferably relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro of CSFV having a sequence according to SEQ ID NO 4.<br>
Even more preferred within the present invention is a derivative of an autoprotease Npro of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above the following amino acids have been exchanged: alanine (A) 109, valine (V) 114, isoleucine (I) 155 and phenylalanine (F) 158 by threonine (T), arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), and leucine (L) 143 with glutamine (Q).<br>
Thus in another aspect the present invention relates even more preferably to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro, of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above the following amino acids have been exchanged: alanine (A) 109, valine (V) 114, isoleucine (I) 155 and phenylalanine (F) 158 by threonine (T); arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), and leucine (L) 143 with glutamine (Q).<br>
Most preferably the derivative of an autoprotease Npro of CSFV according to the present invention comprises the following amino acid sequence:<br>
SEQ ID NO 5:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETTKRIGRV TGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWTRNTTNCPLWVTSC-(168).<br>
Thus in another, most preferred aspect the present invention also relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro of CSFV having a sequence according to SEQ ID NO 5.<br>
in another equally preferred aspect the present invention relates to a process for the production of heterologous proteins as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro, of CSFV having a sequence according to<br><br>
SEQ. ID NO. 5, wherein in addition asparagine (N) 35 is replaced with threonine (T), and threonine (T) 158 is replaced with serine (S).<br>
The derivative of an autoprotease Npro of CSFV which is utilized in the process according to the above aspect of the present invention forms also part of the present invention and comprises the following amino acid sequence:<br>
SEQ ID NO 32:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGTPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPWHRAPLEFFDETQFEETTKRIGRV TGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWTRNSTNCPLWVTSC-(168).<br>
In another preferred aspect the present invention relates to a process for the production of heterologous proteins as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease Npro, of CSFV having a sequence according to SEQ. ID NO. 32, wherein in addition alanine (a) 28 is replaced with glutamic acid (E), serine (S) 71 is replaced with phenylalanine (F) and arginine (R) 150 is replaced with histidine (H).<br>
The derivative of an autoprotease Npro, of CSFV which is utilized in the process according the above aspect of the present invention forms also part of the present invention and comprises the following amlno acid sequence:<br>
SEQ ID NO 33:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTEGRPLFGTPSEVHPQSTLKLPHDRGEDDIETT<br>
LRDLPRKGDCRFGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETTKRIGRV<br>
TGSDGKLYHIWEVDGEILLKQAKRGTPHTLKWTRNSTNCPLWVTSC-(168).<br>
Preferably in the process according to the present invention the derivative of an autoprotease Npro, of CSFV with the sequence according to SEQ ID NO 32 is used in fusion with a protein that contains at least the three first amino acids of proinsulin, more preferably with proinsulin, further more preferably with human proinsulin, most preferably with recombinant human proinsulin, for the production of proinsulin.<br><br>
It is preferred according to the present invention if the derivative of an autoprotease Npro of CSFV has in addition to the replacement of at least one cysteine residue as described above at least one of the following amino acids have been exchanged: arginine (R) 53, glycine (G) 54, arginine (R) 57, threonine (T) 109, 114, 155, 158 and leucine (L) 143. Preferred derivatives of the autoprotease Npro of CSFV according to the present invention have in addition to the replacement of at least one cysteine residue as described above, the following amino acids are exchanged: arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), threonine (T) 109, 114, 155, 158 with serine (S) and leucine (L) 143 with glutamine (Q) or asparagine (N) or aspartic acid (D) or serine (S) or histidine:<br>
Such preferred derivatives of an autoprotease Npro of CSFV which are utilized in the process according the above aspect of the present invention forms also part of the present invention and comprise the following amino acid sequences:<br>
SEQ ID 92:<br>
(1 )-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT<br>
LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR<br>
VTGSDGKLYHIWEVDGEILLKSAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 95:<br>
(1J-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIYVEVDGEILLKNAKRGTPRTLKWSRNSTNCPLWVTSC-{168).<br>
SEQ ID 96:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIWEVDGEILLKDAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 97:<br><br>
 (I)-MELNHFELLYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT<br>
LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPWHRAPLEFFDESQFEESTKRIGR<br>
VTGSDGKLYHIWEVDGEILLKHAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 98:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPNATQDYTGPWHRAPLEFFDESQFEESTKRIGC VTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
The expression vector encodes the polypeptide of interest as the part of the fusion polypeptide which is to be cleaved off autoproteolytically. In accordance with the present invention, a variety of polypeptides of interest can be produced by use of such an expression vector. For example, the polypeptide of interest is one, that exerts pharmacological activity and can for example be selected from the group consisting of an interferon, an interieukin, a growth hormone, a growth factor, a cytokine, an enzyme, an enzyme inhibitor, an antibody and an antibody fragment, and the like, for example interferon alpha 2A, interferon alpha 2B, interleukin-3, interieukin-6, human growth hormone, (pro)insulin, insulin like growth factor, granulocyte-colony stimulating factor, granulocyte macrophage-colony stimulating factor, macrophage-colony stimulating factor, interferon beta 1, bovine somatropin, porcine somatropin, interieukin 11, interieukin-2, a Fab-fragment, and small peptides such as calcitonin, parathyroid hormone (PTH), or a glucagon, CD 40 ligand soluble form, plasminogen activator, sex steroid binding protein, epidermal growth factor, tissue factor extra cellular domain.<br>
In addition the polypeptide of interest can be any other kind of polypeptide in particular a polypeptide which is especially suited for analytical methods, e.g. Green Fluorescent Protein.<br>
In the expression vector to be employed in the process according to the present invention, the fusion polypeptide is operably linked to at least one expression control sequence. Expression control sequences are, in particular, promoters (such as the lac, tac, 13, T7, trp, gac, vhb, lambda pL or phoA promoter), ribosome binding sites (for example natural ribosome binding sites which belong to the abovementioned promoters, cro or synthetic ribosome binding sites), or transcription terminators (for example rmB T1T2 or bla).<br><br>
The vector may also contain sequences encoding fusion domains, as described below, that are present at the N-terminal end of the fusion polypeptide and that are required for its binding to the affinity chromatography system, e.g. polyamino acids like polylysine or, for immunoaffinity chromatogography, so-called "epitope tags", which are usually short peptide sequences for which a specific antibody is available. Well known epitope tags for which specific monoclonal antibodies are readily available include FLAG, influenza virus haemagglutinin (HA), and c-myc tags.<br>
In a preferred embodiment of the present invention, the expression vector is a plasmid.<br>
The transformed bacterial host cell, i.e. the expression strain, is cultivated in accordance with microbiological practice known per se.<br>
The host strain is generally brought up starting from a single colony on a nutrient medium, but it is also possible to employ cryo-preserved cell suspensions (cell banks). The strain is generally cultivated in a multistage process in order to obtain sufficient biomass for further use.<br>
On a small scale, this can take place in shaken flasks, it being possible in most cases to employ a complex medium (for example LB broth). However, it is also possible to use defined media (for example citrate medium). Since in the preferred embodiment of the present invention it is intended that the expressed fusion polypeptide is in the form of insoluble inclusion bodies, the culture will in these cases be carried out at relatively high temperature (for example 30°C or 37°C). Inducible systems are particularly suitable for producing inclusion bodies (for example with the trp, lac, tac or phoA promoter). On a larger scale, the multistage system consists of a plurality of bioreactors (fermenters), it being preferred to employ defined nutrient media. In addition, it is possible greatly to increase biomass and product formation by metering in particular nutrients (fed batch). Otherwise, the process is analogous to the shaken flask.<br>
In the process according to the present invention, the inclusion bodies are isolated from the host cell in a manner known per se.<br>
For example, after the fermentation has taken place, the host cells are harvested by centrifugation, micro filtration, flocculation or a combination thereof, preferably by<br><br>
centrifugation. The wet cell mass is disintegrated by mechanical, chemical or physical means such as high pressure homogenizer, beads mills, french press, hughes press, osmotic shock, detergents, enzymatic lysis or a combination thereof. Preferably, disruption of the cells takes place by high pressure homogenization. In the preferred embodiment where the recombinant fusion polypeptide is deposited as inclusion bodies, the inclusion bodies can be obtained for example by means of high-pressure dispersion or, preferably, by a simple centrifugation at low rotor speed. The inclusion bodies are separated by centrifugation or microfiltration or a combination thereof. The purity in relation to the desired polypeptide of interest can then be improved by multiple resuspension of the inclusion bodies in various buffers, for example in the presence of NaCI (for example 0.5-1.0 M) and/or detergent (for example Triton X-100). Preferably the purity of the inclusion body preparation is improved by several washing steps with various buffers (e.g. 0.5 % Deoxycholate followed by two times 1 M NaCI solution — and finally distilled water). This usually results in removal of most of the foreign polypeptides from the inclusion bodies.<br>
In preparation for the affinity chromatography, the isolated inclusion bodies need to be solubilized.<br>
The present invention relates to a process as described above, where prior to application to the chromatography system, the provided fusion polypeptide is solubilized under chaotropic conditions that inhibit its autoproteolytic activity.<br>
As used herein the term "chaotropic" shall refer to conditions where no or little intramolecular interactions can be observed. These conditions can be achieved for example by addition of detergents, alcohol, urea or guanidine HCI. Conditions may differ for different polypeptides. However it is within the competency of the person skilled in the art to adjust conditions accordingly for any respective polypeptide.<br>
The inclusion bodies are solubilized using a chaotropic agent. Upon solubilization inclusion bodies are dissolved and a monomolecular suspension with substantially reduced intra- and inter-molecular interactions is obtained. Preferred solvents are urea, guanidine HCI and strong tonic detergents as N-lauroylsarcosine. In another embodiment of the present invention inclusion bodies are also solubilized using an aqueous alcohol solution at alkaline pH or simply an aqueous solution at alkaline pH.<br><br>
As used herein the term "solubilization" shall refer to the process necessary to dissolve the inclusion bodies. Solubilization results in a monomolecular dispersion of the polypeptides with minimum intra- and inter-molecular interactions.<br>
A preferred way of solubilization of inclusion bodies within the scope of the present invention, is conducted by suspension in 50 mM Tris/HCI, 8 M urea, pH 7.3, adding a reducing agent, e.g. 50 mM DTT, in the case that oxidized cysteine residues are present.<br>
Where necessary it is possible to remove potentially insoluble material, for example by centrifugation.<br>
In the case that the inactive fusion polypeptide is produced soluble within the cell, the clarified cell homogenate is subjected to the further work up described in the following for the solubilized inclusion bodies.<br>
The solubilized polypeptide is further diluted and applied to the chromatography system by loading it onto the affinity chromatography column. Within the scope of the present invention the chromatography system is adjusted such that the part of the fusion polypeptide that exhibits the autoproteolytic function is selectively recognized and bound under denaturizing, chaotropic conditions. Under these conditions the fusion polypeptide is denaturized, and inactive. In the course of the processing of the polypeptide on the column, conditions are changed to renaturazing, cosmotropic, with the result that the fusion polypeptide folds back into its native configuration and the autoproteolytic function is activated. The binding of the part exerting the autoproteolytic function is maintained during the change of conditions.<br>
As used herein the term "cosmotropic" shall refer to conditions that promote molecular interactions and therefore formation of biological structures. Conditions may differ for different molecules. Citrate, and sulfate ions as anions and quartemary amines or ammonium ions as cations exhibit the highest cosmotropic effect. Also other reagents, such as detergents or redox systems, may be introduced to promote refolding. It is within the competency of a person skilled in the art to adjust conditions accordingly for every respective polypeptide.<br><br>
In principle any chromatography system capable of selectively binding fusion polypeptides under chaotropic conditions and maintaining binding under cosmotropic conditions can be used within the framework of the present invention. The matrix of the chromatography system may, in a preferred embodiment, be in the form of a column, however, it may also be in other forms, like beads or organic materials like polyethylene glycol modified with an affinity peptide.<br>
Chromatography systems suitable for use within the present invention may be based on a cellulose binding domain, they may be cation exchange chromatography systems using polycationic tags like e.g. polyarginine or polylysine as well as anion exchange chromatography with polyanionic tags like e.g. polyasparagine.<br>
Accordingly within the present invention the affinity chromatography system is preferably selected from the group consisting of immobilized metal ion chromatography (IMAC), cation exchange chromatography, anion exchange chromatography, cellulose binding domain chromatography and peptide affinity chromatography.<br>
More preferably the affinity chromatography system used is cation exchange chromatography, wherein the fusion polypeptide comprises a polycationic tag. Even more preferred is the use of either a polyarginine or polylysine affinity tag.<br>
For cation exchange chromatography the expressed fusion polypeptide comprises an N-terminal polycationic tag, for example a polyarginine or polylysine tag. The solution containing the expressed fusion polypeptide that was extracted from the host cells is (filtered) and loaded onto a column packed with any medium suitable for cation exchange chromatography such as e.g. SP Sepharose FF, CM Sepharose FF, Fractogel EMD SO3-. Preferably buffers with low conductivity are applied. After loading unbound material may be washed out and refolding may be started by introduction of a buffer with low urea concentration. At a urea concentration lower than 0.5M the target protein is cleaved off and can be eluted from the column.<br>
Another preferred embodiment of the present invention is one, wherein the affinity chromatography system is anion exchange chromatography and wherein the fusion<br><br>
polypeptide comprises a polyanionic tag. More preferably, polyasparagine is used as affinity tag.<br>
A further preferred embodiment to achieve the desired binding properties is immobilized metal ion affinity chromatography (IMAC).<br>
Accordingly, in a preferred embodiment of the present invention the affinity chromatography system is immobilized metal ion affinity chromatography (IMAC), and the fusion polypeptide comprises a metal chelate affinity tag.<br>
In this case the fusion polypeptide is detected and bound by means of a metal chelate affinity tag comprised in it.<br>
In a more preferred embodiment of the present invention, the metal chelate affinity tag is a polyhistidine affinity tag.<br>
IMAC is based on the specific coordinate covalent binding between histidine or other suitable unique amino acids (either naturally present on the surface of the protein or grafted with recombinant DNA techniques) and various immobilized metal ions, such as copper, nickel, zinc, or iron. Chromatographic materials known in the art for the use in IMAC may also be useful within the present invention. In a preferred embodiment of the present invention, Ni2+-Chelating Sepharose Fast flow (GE Healthcare, Uppsala, SE) is used as matrix.<br>
Alternatively, the affinity chromatography may be immunoaffinity chromatography, employing epitope tags as described above which are present at the N-terminus of the fusion polypeptide and are bound to the Chromatographic matrix via an antibody recognizing said tag.<br>
Another, within the scope of the present invention a preferred affinity Chromatographic method, which has the required binding properties, is affinity chromatography using oligopeptide ligands.<br>
As used herein the term "oligopeptides" shall refer to proteinaceous compounds, containing<br><br>
at least three amino acids. Usually such oligopeptides have a length of up to 35 amino acids.<br>
Accordingly, in a preferred embodiment of the present invention the affinity chromatography system utilizes an oligopeptide ligand of five to twelve amino acids length, more preferred of six to eight amino acids length, comprising a tryptophan residue, which ligand selectively binds to the part of the fusion polypeptide exerting autoproteolytic function under chaotropic conditions and maintains binding during change towards as well as under cosmotropic conditions.<br>
This form of affinity chromatography makes use of the specific binding of certain polypeptides to other polypeptides, as for example known from antibodies. Oligopeptides are capable of serving as affinity ligands as well. These molecules offer high chemical stability, efficiency, selectivity, low price and they are usually not toxic. These features are considered as an advantage especially when applied in a biopharmaceutical process. Peptide ligands directed against a target molecule can be identified from combinatorial peptide libraries or biological libraries in a way, known to the person skilled in the art. In the context of the present invention, screening for peptide ligands was performed under chaotropic conditions.<br>
Methods for peptide synthesis known in the art, are suitable for preparation of the oligopeptide ligands which are subject to the present invention. Preferably though, the peptide ligands are generated by SPOT synthesis, PIN synthesis, teabag synthesis, mix and split method, described in Ruiwu Liu, et al. Experimental Hematology 31 (2003) 11-30 or the PELICAN method, described in Joseph A. Buettner et al., Int. J. Peptide Protein Res. 47 (1996), 70-83. Several linker chemistries can be applied for anchoring of the first amino acid. In one preferred embodiment of the present invention, the ligands are generated separately and afterwards immobilized on the chromatographic matrix. In another preferred embodiment of the present invention, the peptide ligands are synthesized directly on the chromatographic matrix.<br>
The oligopeptide ligand exerts a high degree of specificity. The oligopeptides that are synthesized within the scope of the present invention are characterized by their ability to selectively bind Npro, Npro, derivatives and fusion polypeptides thereof under denaturing conditions. Within the scope of the present invention such an oligopeptide ligand is directed<br><br>
against the part of the fusion polypeptide according to the invention that exerts autoproteolytic function.<br>
In a further preferred embodiment of the present invention the oligopeptide ligand has an<br>
amino acid sequence selected from the group consisting of<br>
SEQ ID NO 6:	VSIFEW,<br>
SEQ ID NO 7:	AVSIEWY,<br>
SEQ ID NO 8:	AVSFIWY,<br>
SEQ ID NO 9:	VSFIWYK,<br>
SEQ ID NO 10:	ASRFWYA,<br>
SEQ ID NO 11:	AFYTWYA,<br>
SEQ ID NO 12:	AFYRWYK,<br>
SEQ ID NO 13:	AFYRWY,<br>
SEQ ID NO 14:	AFYRWYA,<br>
SEQ ID NO 15:	AVSIFEWY,<br>
SEQ ID NO 16:	AVSRNWY,<br>
SEQ ID NO 17:	ASRFWY,<br>
SEQ ID NO 18:	AFYRWYM,<br>
SEQ ID NO 19:	AFYRWY,<br>
SEQ ID NO 20:	ASRFWYM,<br>
SEQ ID NO 21:	AFYRWYM,<br>
SEQ ID NO 22:	AFYSWYAA.<br>
Within the scope of the present invention oligopeptide ligands may be used with a free N-terminus or with a blocked N-terminus, blocking being achieved e.g. by acetylation.<br>
Most preferred is an embodiment of the present invention, wherein the derivative of the naturally occurring Npro of CSFV according to SEQ ID NO 5 is used in combination with an oligopeptide ligand selected from the group consisting of SEQ ID NO 10, SEQ ID NO 11, SEQ ID NO 12, SEQ ID NO 13 and SEQ ID N014.<br>
Therefore, the present invention also provides an affinity matrix comprising a solid phase and an affinity ligand comprising peptide bonds coupled to this solid phase, wherein the affinity ligand comprising peptide bond is selected from the following group of ligands:<br><br>
a)	peptides comprising the formula X1X2X3X4l wherein X1 to X4are amino acid residues<br>
and at least two of X, to X4 is W, Y or F;<br>
b)	peptides comprising the formula X5X6X7X8, wherein X5to X6 are amino acid residues, at<br>
least one of X5 to X8 is W, and at least one of X5 to X8 is E or D; and<br>
c)	poly-amino acids consisting of an amino acid monomer of the group consisting of R, K,<br>
E and D and an amino acid monomer of the group consisting of Y, F and W, preferably poly-<br>
KY, poly-KF, poly-KW, poly-RY, poly-RF, poly-RW, poly-EY, poly-DY, poly-EF, poly-EW,<br>
poly-DF and poly-DW,<br>
with the proviso that the peptides according to a) and b) have a maximum length of 35 amino acid residues and that the poly-amino acids according to c) have a minimum length of 20 amino acid residues.<br>
These affinity ligands have a high affinity to the autoprotease molecules described herein, especially to bind Npro, its derivatives and fusion proteins thereof. Specifically, these ligands or affinity matrices bind Npro,, its derivatives and fusion proteins thereof under chaotropic conditions and also under kosmotropic conditions, at least the Npro, -part of e.g. a fusion protein.<br>
Preferably, the peptides according to a) and b) (herein also referred to as "oliogopeptides") have a length of 5 to 12, especially of 6 to 8, amino acid residues. Preferably, at least one positively charged amino acid is present in these oligopeptides. The poly-amino acids according to c) have a preferred length of at least 35 amino acid residues, more preferred at least 50 amino acid residues, especially at least 100 amino acid residues. Specifically preferred poly-amino acids are e.g. commercially available poly-amino acids for culture media, such as poly-KW, 4:1 (MW 20.000 - 50.000 Da; SIGMA product No. P9285), poly-KY, 4:1 (MW 20.000 - 50.000 Da; SIGMA product No. P4695) or poly-KF, 1:1 (MW 20.000 -50.000 Da; SIGMA product No. P3150).<br>
The affinity ligand according to the present invention may be chemically modified, especially acetylated, esterified, amidated, oxidised, reduced or provided with a linker molecule.<br>
The affinity ligand is preferably linked to the solid matrix by covalent bonds.<br><br>
As solid phase material, all materials already applied in the present field are appropriate. Preferably, the solid phase is selected from the group consisting of chromatography material, especially supports based on cellulose, agarose, acrylamide, poly(styrene-divinylbenzene) or ethylene glycol-methacrylate copolymers, microtiter plates, nitrocellulose membranes, microchips, glass plates or metal coated supports.<br>
According to the present invention various types of solid phase supports may be used, such as the supports based on cellulose, agarose (Sepharose or Macro-Prep gels), dextran (Sephadex gels), acrylamide (Sephacryl, Trisacryl gels), silica (TSK, SW gels), poly(styrene-divinylbenzene) (Source or Poros gels), ethylene glycol-methacrylate copolymers (Toyopearl HW, TSK, PW, fractogel EMD gels) or mixtures, in particular of agarose and dextran (Superdex gel). The supports approved for human or veterinary use by the competent American authorities (FDA for food and drug administration) or the European Union agencies will be more particularly selected. In addition, the support selected must be bonded, preferably by covalent bonding, to the affinity ligand according to the present invention (the support is said to be functionalized). The solid phase matrix may comprise, as the matrix backbone, any natural or synthetic and organic or inorganic material known per se to be applicable in solid phase separation of proteins and other biomolecules, e.g. natural or synthetic polysaccharides such as agar-agar and agaroses; celluloses, cellulose ethers such as hydroxypropyl cellulose, carboxymethyl celluose; starches; gums such as guar gum, and gum arabic, gum ghatti, gum tragacanth, locust bean gum, xanthan gum; pectins; mucins; dextrans; chitins; chitosans; alginates; carrageenans; heparins; gelatins; synthetic polymers such as polyamides such as polyacrylamides and polymethacrylamides; polyimides; polyesters; polyethers; polymeric vinyl compounds such as polyvinylalcohols and polystyrenes; polyalkenes; inorganic materials such as silicious materials such as silicon dioxide including amorphous silica and quartz; silicas; metal silicates, controlled pore glasses and ceramics; metal oxides and sulfides, or combinations of these natural or synthetic and organic or inorganic materials.<br>
The matrix backbone is preferably selected from agar-agar, agaroses, celluloses, cellulose ethers such as hydroxypropyl cellulose, carboxymethyl cellulose, polyamides such as poly(meth)acryl-amides, polyvinylalcohols, silicas, and controlled pore glasses.<br><br>
Especially interesting solid phase materials as matrix backbones are e.g. agar or agarose beads such as Sepharose and Superose beads from Pharmacia Biotech, Sweden and Biogel A from Biorad, USA; dextran based beads such as Sephadex, Pharmacia Biotech; cellulose based beads and membranes such as Perloza cellulose from Secheza, Czechoslovakia; composite beads such as Sephacryl and Superdex, Pharmacia Biotech; beads of synthetic organic polymers such as Fractogel from Toso-Haas, USA; POROS media from Perceptive Biosystems, USA, Bio-Rex, Bio-Gel P and Macro Prep from Biorad, HEMA and Separon from TESSEK and Hyper D and Trisacryl media from BioSepra, USA, Enzacryl and Azlactone, 3M, USA; beads of siliceous materials such as controlled pore glass, PROSEP, from Bioprocesing, England and Spherocil, BioSepra; and coated silica composites in the form of beads or membranes such as ACTI-DISK, ACTI-MOD and CycloSep from Arbor Technologies, USA.<br>
Typically, the solid phase matrix backbone, as well as the resulting functionalised solid phase matrix, may, e.g., be in the form of irregular particles or spherical beads, membranes or sheets, moulded surfaces, or sticks. The solid phase material may further be fully or partly permeable or completely impermeable to proteins. In a particularly interesting embodiment of the present invention, the matrix is in the form of irregular or spherical beads with sizes in the range of 1-10000 µm, preferably 10-1000 µm; such as 10-60µm for high performance applications and such as 50-500 µm, preferably 50-300 µm, for preparative purposes.<br>
A particular interesting form of matrix is a density controlled matrix in the form of a conglomerate comprising density controlling particles. These conglomerates, which are especially applicable in large scale operations for fluidised or expanded bed chromatography as well as different batch-wise chromatography techniques in non-packed columns, e.g. simple batch adsorption in stirred tanks.<br>
The affinity ligands according to the present invention may be attached to the solid phase material by any type of covalent bond known per se to be applicable for this purpose, either by a direct chemical reaction between the affinity ligand according to the present invention and the solid phase material or by a preceding activation of the solid phase material or of the ligand with a suitable reagent known per se making it possible to link the matrix backbone and the ligand. Examples of such suitable activating reagents are epichlorohydrin, epibromohydrin, allyl-glycidylether; bis-epoxides such as butanedioldiglycidylether; halogen-<br><br>
substituted aliphatic compounds such as di-chloro-propanol, divinyl sulfone; carbonyldiimidazole; aldehydes such as glutaric dialdehyde; quinones; cyanogen bromide; periodates such as sodium-meta-periodate; carbodiimides; chloro-triazines such as cyanuric chloride; sulfonyl chlorides such as tosyl chlorides and tresyl chlorides; N-hydroxy succinimides; 2-fluoro-1-methylpyridinium toluene-4-sulfonates; oxazolones; maleimides; pyridyl disulfides; and hydrazides. Among these, the activating reagents leaving a spacer group SP1 different from a single bond, e.g. epichlorohydrin, epibromohydrin, allyl-glycidylether; bis-epoxides; halogen-substituted aliphatic compounds; divinyl sulfone; aldehydes; quinones; cyanogen bromide; chloro-triazines; oxazolones; maleimides; pyridyl disulfides; and hydrazides, are preferred.<br>
Especially interesting activating reagents are believed to be epoxy-compounds such as epichlorohydrin, allyl-glycidylether and butanedioldiglycidylether.<br>
For peptide affinity chromatography within the scope of the present invention, any matrix useful for the immobilization of peptide ligands can be used. Preferably Fractogel epoxy (M), from Merck, Darmstadt, Germany) or equally preferred "monolithic chromatography medium" CIM-epoxy is used. The ligands can be immobilized either directly onto the chemically activated backbone of the chromatography matrix, or via a spacer or linker. In the latter case a spacer is coupled to the chromatographic matrix, said spacer is then chemically activated, in order to allow binding of the ligand. Preferably Fractogel epoxy matrices are used in combination with spacers.<br>
In a particularly preferred embodiment of the present invention the spacer is generated by reaction of the chromatographic matrix with diaminodipropylamine (DADPA) and subsequent reaction with succinic anhydride (SA). The resulting terminal carboxy group on the spacer is chemically activated and preferably linked to a terminal amino-group. The ligand is immobilized on the matrix or on the spacer via a reactive group that it comprises. In the case of peptide ligands such reactive groups may be either the amino, carboxy or the sulfhydryl group. Within the present invention anchorage of the peptide on the matrix or the spacer via an amino bond is particularly preferred.<br><br>
Preferably, the affinity matrix according to the present invention, especially provided as affinity chromatography material, exhibits oligopeptide ligands as defined under a) and b) above.<br>
As used herein the term "oligopeptides" shall refer to proteinaceous compounds, containing at least three amino acids. Usually such oligopeptides have a length of up to 35 amino acids, preferably a length of 4 to 20 amino acid residues.<br>
Accordingly, in a preferred embodiment of the present invention the affinity chromatography system utilizes an oligopeptide ligand of five to twelve amino acids length, more preferred of six to eight amino acids length, especially comprising a tryptophan residue, which ligand selectively binds to the part of the fusion polypeptide exerting autoproteolytic function under chaotropic conditions and maintains binding during change towards as well as under cosmotropic conditions.<br>
This form of affinity chromatography makes use of the specific binding of certain polypeptides to other polypeptides, as for example known from antibodies. Oligopeptides are capable of serving as affinity ligands as well. These molecules offer high chemical stability, efficiency, selectivity, low price and they are usually not toxic. These features are considered as an advantage especially when applied in a biopharmaceutical process. Peptide ligands directed against a target molecule can be identified from combinatorial peptide libraries or biological libraries in a way, known to the person skilled in the art. In the context of the present invention, screening for peptide ligands was performed under chaotropic conditions.<br>
These affinity ligands according to the present invention have turned out to be specifically characterized by their ability to bind Npro and Npro-fusion proteins (and proteins being or comprising mutants thereof) under denaturing conditions, e.g. 4 M urea.<br>
Methods for peptide synthesis known in the art, are suitable for preparation of the oligopeptide ligands which are subject to the present invention. Preferably though, the peptide ligands are generated by SPOT synthesis, PIN synthesis, teabag synthesis, mix and split method, described in Ruiwu Liu, et al. Experimental Hematology 31 (2003) 11-30 or the PELICAN method, described in Joseph A. Buettner et al., Int. J. Peptide Protein Res. 47 (1996), 70-83. Several linker chemistries can be applied for anchoring of the first amino acid.<br><br>
In one preferred embodiment of the present invention, the ligands are generated separately and afterwards immobilized on the chromatographic matrix. In another preferred embodiment of the present invention, the peptide ligands are synthesized directly on the chromatographic matrix.<br>
The oligopeptide ligand exerts a high degree of specificity. The oligopeptides that are synthesized within the scope of the present invention are characterized by their ability to selectively bind Npro, Npro derivatives and fusion polypeptides thereof under denaturing conditions. Within the scope of the present invention such an oligopeptide ligand is directed against the part of the fusion polypeptide according to the invention that exerts autoproteolytic function.<br>
In a further preferred embodiment of the present invention the oligopeptide ligand has an amino acid sequence selected from the group consisting of VSIFEW, AVSIEWY, AVSFIWY, VSFIWYK, ASRFWYA, AFYTWYA, AFYRWYK, AFYRWY, AFYRWYA, AVSIFEWY, AVSRNWY, ASRFWY, AFYRWYAA, AFYRWY, ASRFWYAA, AFYRWYM and AFYSWYM.<br>
Within the scope of the present invention oligopeptide ligands may be used with a free N-temninus or with a blocked N-terminus, blocking being achieved e.g. by ac(et)ylation.<br>
Most preferred is an embodiment of the present invention, wherein the derivative of the naturally occurring Npro of CSFV according to SEQ ID NO 5 (since amino acid sequence of this mutant has a sequence motif "EDDIE" from residue 53 to 57 (instead of "RGDIR" in the wild type), this mutant (and other mutants comprising this motif) is termed "EDDIE'-mutant herein) is used in combination with an oligopeptide ligand selected from the group consisting of ASRFWYA, AFYTWYA, AFYRWYK, AFYRWY and AFYRWYA.<br>
Accordingly, preferred affinity ligands are selected from the group consisting of VSDDWY, VSEDWY, VSIDWY, VSYDWY, VSVDWY, VSWDWY, VSYDWY, VSFDWY, VSDEWY, VSEEWY, VSIEWY, VSYEWY, VSVEWY, VSWEWY, VSYEWY, VSFEWY, DDDDWY, DDEDWY, DDIDWY, DDYDWY, DDVDWY, DDWDWY, DDYDWY, DDFDWY, VSIFWE, FSIFEW, WSIFEW, VSLIWY, VSLIDW, VSLIEW, VSLIWE, FSLEEW, VSDLDW, VSDLEW, VSYIDW, VSYIWE (all these peptides are binding Npro at pH 5.5), VSIDWY, VSIEWY,<br><br>
VSIWWY, VSIIWY, VSYIWY, VSVIWY, VSFIWY, VSFIWE, VSIFEW, VSIFWE, FSIFEW, WSIFEW, VSLIWY, VSLIDW, VSLIEW, VSLIWE, FSLIEW, WSLIEW, FSYFEW, FSFYEW, WSFYEW, FSYIEW, WSYIEW (all these peptides are binding N1*0 at pH 7.3), AFYTWYA, AFYRWYK, AFYRWY, AFYRWYA, AFFRWYA, AFGRWYA, AFHRWYA, AFIRWYA, AFLRWYA, AFMRWYA, AFNRWYA, AFPRWYA, AFQRWYA, AFRRWYA, AFSRWYA, AFTRWYA, AFVRWYA, AFYRWYA, AFYFWYA, AFYGWYA, AFYLWYA, AFYMWYA, AFYNWYA, AFYPWYA, AFYTWYA, AFYVWYA, AFYWWYA, AFYYWYA, AKWFRYA, VSRNWY, ASRNWYA, ASRFWYA, FSRNWYA, VFRNWYA, VWRNWYA, VYRNWYA, VSRAWYA, VSRFWYA, VSRWWYA, VSRYWYA, VSRNFYA, VSRNYYA, VSRNWFA, VSRNWWA (all these peptides have a specifically high affinity to N""" mutants with the EDDIE motif in amino acid residues 53 to 57), Ac-AFYTWYAK, Ac-AFYRWYKK, Ac-AFYRWYK, Ac-AFYRWYAK, Ac-AFFRWYAK, Ac-AFGRWYAK, Ac-AFHRWYAK, Ac-AFIRWYAK, Ac-AFLRWYAK, Ac-AFMRWYAK, Ac-AFNRWYAK, Ac-AFPRWYAK, Ac-AFQRWYAK, Ac-AFRRWYAK, Ac-AFSRWYAK, Ac-AFTRWYAK, Ac-AFVRWYAK, Ac-AFYRWYAK, Ac-AFYFWYAK, Ac-AFYGWYAK, Ac-AFYLWYAK, Ac-AFYMWYAK, Ac-AFYNWYAK, Ac-AFYPWYAK, Ac-AFYTWYAK, Ac-AFYVWYAK, Ac-AFYWWYAK, Ac-AFYYWYAK, Ac-AKWFRYAK, Ac-VSRNWYK, Ac-ASRNWYAK, Ac-ASRFWYAK, Ac-FSRNWYAK, Ac-VFRNWYAK, Ac-VWRNWYAK, Ac-VYRNWYAK, Ac-VSRAWYAK, Ac-VSRFWYAK, Ac-VSRWWYAK, Ac-VSRYWYAK, Ac-VSRNFYAK, Ac-VSRNYYAK, Ac-VSRNWFAK, Ac-VSRNWWAK, YWKA, Ac-YWKAK, YKYA, Ac-YKYAK, YWRA, Ac-YWRAK, ARWY, Ac-ARWYK, YWRA, Ac-YWRAK (all these peptides have improved immobilisation capabilities to the substrate due to N-terminal acetylation and C-terminal lysination).<br>
For peptide affinity chromatography within the scope of the present invention, any matrix useful for the immobilization of peptide ligands can be used. Preferably Fractogel epoxy (M), from Merck, Darmstadt, Germany) or equally preferred "monolithic chromatography medium" CIM-epoxy is used. The ligands can be immobilized either directly onto the chemically activated backbone of the chromatography matrix, or via a spacer or linker. In the latter case a spacer is coupled to the chromatographic matrix, said spacer is then chemically activated, in order to allow binding of the ligand. Preferably Fractogel epoxy matrices are used in combination with spacers.<br><br>
In a particularly preferred embodiment of the present invention the spacer is generated by reaction of the chromatographic matrix with diaminodipropylamine (DADPA) and subsequent reaction with succinic anhydride (SA). The resulting terminal carboxy group on the spacer is chemically activated and preferably linked to a terminal amino-group. The ligand is immobilized on the matrix or on the spacer via a reactive group that it comprises. In the case of peptide ligands such reactive groups may be either the amino, carboxy or the sulfhydryl group. Within the present invention anchorage of the peptide on the matrix or the spacer via an amino bond is particularly preferred.<br>
When the binding of the fusion polypeptide to the chromatography system has been accomplished, unbound contaminating components can easily be washed off the column. Such contaminating compounds might for example be host cell polypeptides and nucleic acids, which were occluded into or adsorbed on the inclusion bodies, and remain in the polypeptide solution after solubilization, as well as residual components from an enzymatic cell disruption. After washing only the fusion polypeptide remains bound to the column so that the following steps are conducted in a purified system.<br>
Binding of the fusion polypeptide is established under chaotropic, inactivating conditions. In order to induce refolding, conditions are changed to cosmotropic.<br>
In a preferred embodiment the step of refolding of the fusion polypeptide is performed by the change from chaotropic to cosmotropic conditions via buffer exchange.<br>
Buffers can be alternatively gradually or instantaneously changed to cosmotropic conditions. In one preferred embodiment of the present invention the exchange of chaotropic buffer with cosmotropic buffer is conducted instantaneously, by application of the buffer as a plug. In another equally preferred embodiment of the present invention the exchange of buffers is conducted gradually.<br>
Binding of the fusion polypeptide to the column and/or refolding and cleaving of said fusion polypeptide might be facilitated if the buffer exchange is accompanied by a temperature adjustment. This can, for example, be introduced by a cooling/heating jacket Therefore, in a preferred embodiment, a cooling/heating jacket is applied for temperature adjustment; more<br><br>
preferably, the buffer is brought to the desired temperature prior to its application. In this way such temperature adjustment is achieved.<br>
Upon change of conditions in the packed bed the fusion polypeptide starts to refold and the part exerting the autoproteolytic function becomes active. As a result, the C-terminaliy fused polypeptide of interest is cleaved off at a distinct site defined by the specificity of the autoproteolytic part, thereby producing a homogenous N-terminus of the polypeptide of interest. Depending on the time required for refolding of the fusion polypeptide, the velocity of the mobile phase with the cosmotropic buffer is reduced or stopped when all chaotropic buffer is displaced from the packed bed. After refolding is complete, the liberated polypeptide of interest is washed out from the packed bed by further feeding of cosmotropic buffer. The N-terminal autoproteolytic part of the fusion polypeptide as well as un-cleaved fusion polypeptide is eluted by conventional means, e. g. high salt concentration, a pH-change or NaOH, to regenerate the chromatography material. For regeneration the packed bed is washed with a buffer that strips the autoprotease from the adsorbent These buffers comprise either acidic or alkaline solutions or organic solvents. After re-equilibration with starting buffer/chaotropic buffer the packed bed is ready for the next cycle.<br>
When necessary, because the cleavage rate might not be as high as desired, un-cleaved fusion polypeptide that is washed off the column during the regeneration step can be re-fed into another circle of the chromatography process according to the present invention.<br>
The liberated polypeptide of interest can be obtained optionally via choice of the respective buffers either in a partially or in a completely refolded state. Within the scope of the present invention, the polypeptide of interest in the effluent is either partially or preferably completely refolded. In one embodiment of the present invention, refolding of the autoproteolytic active part of the fusion polypeptide might be complete, while the polypeptide of interest remains partly unfolded. This situation can occur for example when the polypeptide of interest has a very complex conformation, for example a di- or trimerizaton, or comprises a prosthetic group or a cofactor. Such polypeptide of interests might require particular conditions in order to complete refolding. Accordingly in such cases folding may be completed in a separate step, where special conditions, e.g. protonic strength and pH or the complete removal of detergents, which are usually added during refolding, can be generated.<br><br>
Within the scope of the present invention, the conditions may be changed to any state where the fusion polypeptide stays adsorbed to the column.<br>
The present invention also discloses oligopeptide ligands and derivatives of Npra of CSFV as described hereinabove for use according to the present invention. The present invention also relates to the use of an oligopeptide and a derivative of Npro of CSFV as described hereinabove according to the present invention.<br>
Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art.<br>
The present invention is described further with reference to the following examples, which are illustrative only and non-limiting. In particular, the examples relate to preferred embodiments of the present invention.<br>
Examples<br>
Example 1:<br>
Production of a heterologous polypeptide in accordance with the invention, utilizing<br>
peptide affinity chromatography and different fusion polypeptides<br>
1.1. Production of a heterologous polypeptide utilizing Npro autoprotease of pestivirus<br>
This example describes the production of GFPmut3.1 as a fusion polypeptide of pestivirus autoprotease 6xHis-Npro, whereby refolding and cleavage are performed on a peptide affinity matrix.<br>
The GFPmut3.1 used in the fusion construct with Nproin the examples hereafter, is a mutant of GFP which is optimized for production in E. coli. GFPmut3.1 carries the following amino acid replacements: S 2 is replaced with R, S 65 with G and S 72 is replaced with A. Position 178 to 415 of the sequence of the entire fusion construct named 6H-sNp-Gmut3.1-pET30a refer to the sequence of GFPmut3.1.<br><br>
Constmction of the vector 6H-sNp-Gmut3.1-pET30a is described under 1.2.1.1 and 1.2.1.2<br>
below.<br>
Transformation of hostcells is performed as described in 1.2.2 below.<br>
1.1.1	Chromatography equipment<br>
The Chromatography runs in example 1 are performed on an AKTA 100 Explorer Chromatography system (Amersham Biosciences). The prepared peptide affinity sorbents are packed into HR 5 columns (5 mm i.d., Amersham Biosciences). The gel volume is approximately 1 ml.<br>
1.1.2	Preparation of oligopeptide ligands<br>
The oligopeptide ligands used in example 1 are produced in the following way:<br>
Solid Phase Peptide Synthesis is performed on a 433A peptide synthesizer (Applied Biosystems, Vienna, Austria) with 1-hydroxy-1H-benzotriazol/N,N'-dicyclohexylcarbodiimide (HOBt/DCC)-activation of Fmoc-protected amino acids (Bachem, Bubendorf, Switzerland). Peptides are synthesized on a 4-hydroxymethyl-phenoxymethyl-copolystyrene-1% divinylbenzene resin (HMP resin, Wang resin). Protecting groups for side chains are tert-butyl (t-Bu) for tyroslne, serine and threonine, OtBu for glutamic acid and aspartic add, tert-butoxycarbonyl (Boc) for lysine and tryptophane and trityl (Trt) for cystein, histidine, asparagine and glutamine. For the coupling of the first amino acid 4-dimethylaminopyridine (DMAP) is used as a catalyst. After coupling of the first amino add, a capping step is accomplished by using benzole anhydride. Deprotection of the Fmoc group is performed with 20% piperidin. Side chain deprotection and cleavage from the resin are carried out by reaction with a cleavage mixture containing 95% triflouroacetic add (TFA), 2.5 % water and 2.5 % triisopropylsilane (TIS). After washing with dichloromethane (DCM) the crude peptide is purified by repeated ether predpitation followed by lyophilization. The peptides are further purified by RP-HPLC on a Luna 15 µ C18(2) 250 x 21.2 mm column (Phenomenex, Torrence, CA, USA) with P 3500 pumps (Amersham Biosciences, Uppsala, Sweden), using a linear gradient of 5-50% acetonttrile vs. water (0.1% TFA) at 30 ml/min. Purity is confirmed by analytical RP-HPLC with a HP 1090 liquid chromatograph (Hewlett Packard, USA) using a Luna 3µ C18(2) 100 x 4.6 mm column (Phenomenex) with a linear gradient of 1% acetonitrile per minute. Homogeneity and identity are verified by matrix assisted laser<br><br>
desorption ionization - time of flight mass spectrometry (ThermoBioanalysis, Hempstead, UK).<br>
1.1.3	Preparation of affinity matrix<br>
The affinity matrices used in example 1 are prepared in the following way:<br>
10 g of Fractogel epoxy (M) (Merck, Darmstadt, Germany) is reacted with 50 ml 1 M Diaminodipropylamine (DADPA) for 48 hours at room temperature. After the reaction the gel is washed with a 50 ml 10mM HCI and 3 times 50 ml water. The gel is resuspended in water, the pH is adjusted to 7.0 by addition of 0.1 M NaOH and 2 g of succinic anhydrid is added. After 30 minutes gentle stirring the pH is adjusted to 7.0 by addition of 10 M NaOH and another 2 g succinic anhydride are added. After another 30 minutes stirring the gel is washed with 50 ml 0.1 M NaOH, 50 ml phosphate buffered saline (PBS), 3 times with 50 ml water and 20 % ethanol. After suction drying the gel is stored at 4° C.<br>
1.1.4	Activation of the carboxy-group and immobilization ofpeptides:<br>
The affinity matrices according to example 1 are activated in the following way:<br>
1 g of wet Fractogel is modified with a DADPA-SA spacer as described in chapter 1.1.3 and washed 2 times with 5 ml Acetonitrile. Activation is perfomed with 3 ml 0.1 M Succinimidyl-trichloroethylcarbonate and 0.1 M triethylamine dissolved in acetonitrile for 3 hours. The gel is subsequently washed with acetonitrile and 1 mM HCI. The peptide AFYRWYA is dissolved in PBS at a concentration of 3 mg/ml. 5 ml of the peptide solution is rapidly added to the gel and reacted for 24 hours. The peptide VSFIWYK, is dissolved in dimethylformamide (DMF) containing 0.1 M triethylamine. 5 ml of the peptide solution are rapidly added to the gel and reacted for 24 hours. Coupling yield is determined by RP-HPLC of samples before and after coupling.<br>
Immobilization ofpeptides on CIM-epoxy:<br>
Peptides are dissolved in a 100 mM Na2C03 buffer pH 10.0 containing 0.15 M NaCI. The CIM-disks are mounted in a cartridge supplied by the manufacturer and the peptide solution is slowly pumped through the disk using a P1 pump (Amersham Biosciences) in a circulation mode for 48 hours at room temperature. Coupling yield is determined by RP-HPLC of samples before and after coupling. After coupling remaining epoxy groups are blocked with 0.5 M ethanolamine, pH 10.0 for 48 hours.<br><br>
1.1.5 Expression of the fusion polypeptide<br>
Recombinant E. coli HMS 174 (DE3) expressing a fusion polypeptide comprising the N-terminal autoprotease Npro with a 6xHis-tag and a C-terminally fused GFPmut3.1 are cultured in a 101 -fermenter. The fusion polypeptide comprises the following amino acid sequence:<br>
SEQ ID NO 23:<br>
1  MHHHHHHELN HFELLYKTSK QKPVGVEEPV YDTAGRPLFG NPSEVHPQST LKLPHDRGRG	60<br>
61  DIRTTLRDLP RKGDCRSGNH LGPVSGIYIK PGPVYYQDYT GPVYHRAPLE FFDEAQFCEV	120<br>
121 TKRIGRVTGS DGKLYHIYVC VDGCILLKLA KRGTPRTLKW IRNFTNCPLW VTSCSGTMRK	180<br>
181 GEELFTGWP ILVELDGDVN GHKFSVSGEG EGDATYGKLT LKFICTTGKL PVPWPTLVTT	240<br>
241 FGYGVQCFAR YPDHMKQHDF FKSAMPEGYV QERTIFFKDD GNYKTRAEVK FEGDTLVNRI	300<br>
301 ELKGIDFKED GNILGHKLEY NYNSHNVYIM ADKQKNGIKV NFKIRHNIED GSVQLADHYQ	360<br>
361 QNTPIGDGPV LLPDNHYLST QSALSKDPNE KRDHMVLLEF VTAAGITHGM DELYK<br>
The bacterial host cell, i.e. the expression strain, is cultivated in accordance with microbiological practice known per se. The strain is generally brought up starting from a single colony on a nutrient medium, but it is also possible to employ cryopreserved cell suspensions (cell banks). The strain is generally cultivated in a multistage process in order to obtain sufficient biomass for further use.<br>
On a small scale, this can take place in shaken flasks, it being possible in most cases to employ a complex medium (for example LB broth). However, it is also possible to use defined media (for example citrate medium). For the cultivation, a small-volume pre-culture of the host strain (inoculated with a single colony or with cell suspension from a cryo-culture) is grown, the temperature for this cultivation not generally being critical for the later expression result, so that it is possible routinely to operate at relatively high temperatures (for example 30°C or 37°C). The main culture is set up in a larger volume (for example 500 ml), where it is in particular necessary to ensure good aeration (large volume of flask compared with the volume of contents, high speed of rotation). Since it is intended that expression take place in the form of insoluble inclusion bodies, the main culture will in most cases also be carried out at relatively high temperature (for example 30 or 37°C). Inducible systems are particularly suitable for producing inclusbn bodies (for example with trp, lac, tac or phoA promoter). After the late logarithmic phase has been reached (usually at an optical density of 0.5 to 1.0 in shaken flasks), in these cases the inducer substance (for example indoleacrylic acid, isopropyl ß-D-thiogalactopyranoside = IPTG) is added and incubation is continued for 1 to 5 hours. During this time, most of the Npro fusion polypeptide is deposited<br><br>
as inclusion bodies in the bacterial cytoplasm. The resulting cells can be harvested and processed further.<br>
On a larger scale, the multistage system consists of a plurality of bioreactors (fermenters), it being preferred to employ defined nutrient media in this case in order to be able to improve the process engineering control of the process. In addition, it is possible greatly to increase biomass and product formation by metering in particular nutrients (fed batch). Otherwise, the process is analogous to the shaken flask. For example, a preliminary stage fermenter and a main stage fermenter are used, the cultivation temperature being chosen similar to that in the shaken flask. The preliminary stage fermenter is inoculated with a so-called inoculum which is generally grown from a single colony or a cryoculture in a shaken flask. Good aeration and a sufflcient inducer concentration must also be ensured in the fermenter - and especially in the main stage thereof. The induction phase must, however, in some cases be made distinctly longer compared to the shaken flask. The resulting cells are once again delivered for further processing.<br>
1.1.6	Isolation of inclusion bodies<br>
After harvesting, the cells (850 g wet weight) are suspended in 2500 ml of 50 mM Tris/HCI, 5 mM EDTA, 1 %Triton X-100, pH 8.0. The chilled suspension is passed through an APV-2000 high pressure homogenizer (Invensys) for three times at 800 bar to disrupt the cells. Between the passages the suspension is chilled on ice and homogenized using an Ultraturrax. The homogenate is centrifuged at low speed (JLA 10.500, 7500 rpm, 30 min) to obtain the inclusion bodies containing the recombinant fusion polypeptide.<br>
1.1.7	Solubilization of inclusion bodies<br>
The pellet is suspended in 50 mM Tris/HCI, 5 mM EDTA, 1 %Triton X-100, pH 8.0 and centrifuged. This step is repeated. After a H2O-washing step the pellet is suspended in H2O. The obtained inclusion body-suspension is stored at -20 °C for further use. The inclusion body-suspension Is diluted 1 : 5 with 50 mM Tris/HCI, 10 M urea, 50 mM DTT, pH 7.3 at room temperature. Insoluble components are removed by centrifugation. A polypeptide concentration of about 15 mg/ml is obtained. The polypeptide solution is diluted with 50 mM Tris/HCI, 100 mM NaCI, 4 M urea, pH 7.3 to reach a polypeptide concentration of about 2 mg/ml.<br><br>
1. 1.8 Binding of the fusion polypeptide to the chromatographic column 0.5 ml of the polypeptide solution is applied to a Fractogel-DADPA-SA-VSFIWYK (0.5 x 5 cm) matrix, whereby preparation and coupling of the respective peptide is conducted as described above in 1.1.2 and 1.1.3. The column is equilibrated with 50 mM Tris/HCI, 100 mM NaCI, 4 M urea, pH 7.3 with a linear flow rate of 50 cm/h. The flow rate is increased to 150 cm/h after sample injection.<br>
1 . 1.9 Washing out of unbound contaminating material<br>
Unbound components are washed out with 5 column volumes of equilibration buffer. A buffer exchange to refolding buffer, specifically to 0.5 M Tris/HCI, 2 mM EDTA, 3 % glycerol, 5 mM DTT, pH 7.3, is performed with 4.5 column volumes.<br>
1. 1. 10 Refolding, cleavage and elation<br>
After changing the conditions from chaotropic to cosmotropic, the fusion polypeptide is allowed to refold for 25 h on the chromatography resin by stopping the flow. The active autoprotease cleaves off the C-terminally fused GFPmut3.1. The subsequent elution with refolding buffer at a flow rate of 50cm/h results in purified native GFPmut3.1, as is confirmed by fluorescence measurements and SDS-PAGE.<br>
1.1. 11 Regeneration<br>
Regeneration of the chromatography resin is performed with 0.1 M NaOH at a flow rate of<br>
50cm/h.<br>
1.2 Production of a heteroloaous polvpeptide utilizing a derivative of Npro autoprotease of oestivirus<br>
This example describes the production of GFPmut3.1 as a fusion polypeptide of a mutant of pestivirus autoprotease Npro: 6xHis- Npro EDDIE, whereby refolding and cleavage are performed on a peptide affinity matrix.<br>
Preparation of oligopeptide ligands and affinity matrix are performed as described in example 1.1., the same chromatography equipment as described in example 1.1 is used.<br>
1.2.1 Construction of the plasmld:<br><br>
1,2.1.1 Construction of 7H-Np-Gmut3.1-pET30a plasmid:<br>
A DNA fragment containing the gene for an N-terminally truncated Npro including a 7-His tag at the N-terminus is amplified by PCR primer pair:<br>
T7-pET (SEQ ID NO 24):<br>
5'- GAA ATT AAT ACG ACT CAC TAT AGG -3';<br>
Npro R-Kpn (SEQ ID NO 25):<br>
5'- ATA CGG TAC CAG AGC AAC TAG TTA CCC ATA ATG-3'<br>
from NP6-pET (Sandoz) plasmid and inserted via Ndel and Kpn\ (Asp718) restriction sites into pET-30a (# 69909-3, 2002-2003 catalogue, Novagen, CN Biosciences Inc., Merck KgaA, Darmstadt, Germany). Transformation of the ligation reaction into E.coli DH5alpha (# 10643-013, Invitrogen catalogue 2003, Invitrogen Life Technologies Corporation, 1600 Faraday Avenue, PO Box 6482 Carlsbad, California 92008), isolation of plasmid DNA from transformed clones and verification by sequencing results in plasmid 7H-Npro-pET30a plasmid. From the plasmid pGFPmut3.1 (#6039-1, catalogue 1999, BD Biosciences Clonetech, 1020 East Meadow Circle, Palo Alto, CA 94303-4230, USA) the GFPmut3.1 sequence is also amplified by PCR primer pair:<br>
GFP F-Kpn (SEQ ID NO 26):<br>
5'- GAA AGG TAC CAT GCG TAA AGG AGA AG-3'<br>
GFP R-Sal (SEQ ID NO 27):<br>
5'- TAA GTC GAG TTA TTT GTA TAG TTC ATC CAT GCC-3'<br>
isolated by gel extraction and cloned via Kpn\ - Sa/l restriction sites into the 7H-Npro-pET30a construct thus creating the amino acid sequence SGT (serine-glycine-threonine) immediately following the cleavage site. The sequence of construct 7H-Np-Gmut3.1-pET30a is verified as described above.<br><br>
1.2.1.2 Construction of 6H-sNp-Gmut3.1-pET30a plasmid: The DNA sequence for Npre-pro-insulin (SEQ ID NO 28):<br>
ATGGAACTCAATCATTTCGAACTGCTCTACAAAACTAGCAAGCAAAAACCTGTTGGCGT<br>
TGAAGAGCCGGTCTACGATACTGCAGGTCGTCCTCTTTTTGGGAATCCGTCCGAAGTG<br>
CACCCCCAGTCAACCCTCAAGCTTCCCCATGACCGCGGACGCGGTGACATTCGTACAA<br>
CGCTGCGCGATCTGCCTCGTAAAGGCGATTGTCGCTCTGGAAACCACCTAGGTCCGGT<br>
GTCGGGCATTTACATTAAACCAGGTCCCGTCTATTACCAAGACTACACTGGTCCGGnT<br>
ACCATCGTGCACCTCTGGMTTCTTTGATGAAGCTCAArnTGCGAAGTGACTAAACGT<br>
ATTGGCCGTGTAACCGGTTCGGACGGGAAACTGTACCACATCTACGTGTGCGTTGATG<br>
GCTGTATCCTGCTGAAACTCGCGAAGCGCGGAACCCCTCGCACCCTGAAATGGATCCG<br>
TAACTTCACTAACTGTCCACTGTGGGTCACTAGTTGCTTCGTTAACCAACATCTGTGCG<br>
GTTCACACCTTGTGGAAGCCCTGTATCTGGTGTGTGGCGAACGCGGATTCTTTTATACC<br>
CCGAAAACGCGGCGCGAAGCCGAAGATCTTCAGGTTGGTCAAGTGGAACTGGGCGGA<br>
GGTCCGGGAGCCGGGAGCCTGCAACCGCTGGCGCTTGAAGGGTCGCTGCAAAAACGC<br>
GGTATTGTTGAACAGTGCTGTACCTCCATCTGCTCTCTGTATCAGCTGGAAAACTACTG<br>
CAATTAATAA<br>
is custom-synthesized and inserted into pUC119 ( NCBI # U07650: National Center for Biotechnology Information Plasmid Database, National Library of Medicine, Building 38A, Bethesda, MD 20894, USA) by Operon Biotechnologies Inc. (1000 Atlantic Avenue, Suite 108 Alameda , CA 94501, USA). From this plasmid the Npro-pro-insulin sequence, indicated in bold, is amplified by PCR using the following primer pain<br>
6H-Npro-F-Ndel (SEQ ID NO 29):<br>
5'- CTC TCA TAT GCA TCA CCA TCA TCA TCA CGA ACT CAA TCA TTT CGA ACT GCT C-<br>
3*<br>
and Ins-R-Sall (SEQ ID NO 30):<br>
5'- CTT TCG TCG ACT TAT TAA TTG GAG TAG TTT TC-3'<br>
and the resulting fragment is isolated by agarose gel electrophoresis and gel extraction and ligated via the newly created restriction sites for Ndel and Sail (bold letters) into the vector<br><br>
pET30a (# 69909-3, 2002-2003 catalog, Novagen, CN Biosciences Inc., Merck KgaA, Darmstadt, Germany) cut at the same restriction sites creating 6H-sNpro-lns-pET30a. 6H-sNpro-lns-pET30a is cut at the Spel and Sa/l restriction sites, the larger fragment isolated by gel electrophoresis and extraction, thereby removing the sequence for pro-insulin. To prepare the insert the vector 7HNp-Gmut3.1-pET30a (for construction see 1.2.1.1) is digested by the same enzymes and the excised DNA fragment coding for GFPmut3.1 is isolated by gel extraction. By ligation of this DNA fragment into the prepared vector the construct 6H-sNp-Gmut3.1-pET30a is obtained, coding for a fusion of synthetic Npro with GFPmut3.1. The DNA sequences are controlled as described under 1.2.1.1).<br>
12.13 Construction of S-Np-lns-pET30a<br>
From the construct containing the DNA sequence for Npra-pro-insulin custom-synthesized and inserted into pUC119 by Operon Biotechnologies Inc the Npro-pro-insulin sequence required is amplified by PCR using the following primer pain<br>
Npro-F-NdelfSEQIDNO 31): 5'-CGCGACATATGGAACTCMTCATTTCGAAC-3' and Ins-R-Sall (SEQ ID NO 30)<br>
The resulting fragment is isolated by agarose gel electrophoresis and gel extraction and ligated via the newly created restriction sites for Ndel and Sail (bold letters) into the vector pET30a, cut at the same restriction sites. The DNA sequence of plasmid sNpro-lns-pET30a is controlled as described (see 1.2.1.1).<br>
1.2.1.4 Construction of 6H-EDDIE-sGmut3. 1-pET30a plasmid:<br>
From the construct containing the DNA sequence for Npro-pro-insulin custom-synthesized and inserted into pUC119 by Operon Biotechnologies Inc the Npro-sequence required is amplified by PCR using the following primer pair:<br>
Npro-F-Ndel (SEQ ID NO 31)<br>
and<br>
Npro-R-Sall (SEQ ID NO 32):<br><br>
5XX3C AGA GAT GTT GGT CGA CGC TGC AAC TAG TG-3'<br>
and inserted via the newly created restriction sites for Ndel and Sail (bold letters) into the vector pET30a creating S-Np-6H-pET30a. From S-Np-6H-pET30a the Npro sequence is amplified by two standard 50µl PCR reactions: one with 50 pmol Npro-F-Ndel primer (table 1 below) and 50 pmol reverse mutation primer (3'-), 5 units Taq DNA-polymerase (# GC 002004, 2004 catalog, Genecraft, Treskow Straße 10, D-48163 Munster, Germany), 1x PCR buffer (# GC 002006, 2004 catalog, Genecraft) and 20 nmol each dNTP mixture (# GC 013004, 2004 catalog, Genecraft); the second with 50 pmol Npro-R-Sall primer (see table 1 below) and 50 pmol forward mutation primer (5'-), 5 units Taq DNA-polymerase, 1x PCR buffer and 20 nmol each dNTP mixture. PCR reaction takes place in a heated lid thermocycler using the following program: 94°C for 3 min; 25 cycles: 94°C for 30 sec, 54°C for 30 sec, 68°C for 1 min; final incubation at 68°C for 7 min. Free primers are removed by QIAquick® PCR Purification Kit (QIAGEN GmbH, Qiagenstrasse 1, D 40724 Hilden, Germany, Cat. Nr. 28106, Qiagen Product Guide 2004) according to the manufacturers recommendations (QIAquick® Spin Handbook July 2002). One-hundredth of both PCRs is combined and amplified in a standard 50µl PCR reaction with 50 pmol Npro-F-Ndel primer (SEQ ID NO31) and 50 pmol Npro-R-Sall primer (SEQ ID NO32), 5 units Taq DNA-polymerase (Genecraft), 1x PCR buffer (Genecraft) and 20 nmol each dNTP mixture (Genecraft) in a heated lid thermocycler using the following program: 94°C for 3 min; 25 cylces: 94°C for 30 sec, 54°C for 30 sec, 68°C for 1 min; final incubation at 68°C for 7 min. Free primers are removed by QIAquick PCR Purification Kit (QIAGEN) according to the manufacturers recommendations. The PCR fragments are inserted via the Ndel and Sail restriction sites into vector pET30a. The construct is then used for the next mutational step. This is done in six consecutive steps to introduce the amino acid changes one by one, selecting the respective primers as indicated in table 1 below. The outcoming plasmid of each step is controlled by DNA sequence analysis as described (see 1.2.1.1). The last mutational step (I155T and F158T) is done by a single PCR reaction with the primer pair Npro-F-NdelfSEQ ID NO31)<br>
and<br>
3'_I155T,F158T (SEQ ID NO 33):<br><br>
5'-GCA ACT AGT GAC CCA CAG TGG ACA GTT AGT GGT GTT ACG GGT CCA TTT CAG G-3'<br>
and the resulting PCR product is inserted via the Ndel and Spel (bold letters, table 1 below) restriction sites into S-Np-6H-pET30a. DNA sequence analysis as described (see 1.2.1.1) verifies the sequence of construct EDDIE-6H-pET30a. Table 1:<br><br>
(Table Removed)<br>
From EDDIE-6H-pET30a EDDIE is amplified by PCR using the following primer pair:<br>
6H-Npra-F-Ndel, (SEQ ID NO 29) and Npro-R-Sal, (SEQ ID NO 32) and the resulting fragment are used to replace Npro via restriction sites for Ndel and Spel (bold letters) in the construct S-Np-lns-pET30a (see 1.2.1.3) creating 6H-EDDIE-lns-pET30a. The vector 6H-EDDIE-lns is digested by Spel/Sa/l, the larger fragment isolated by gel electrophoresis and extraction, thereby removing the sequence for pro-insulin. The insert is generated by PCR from a construct containing a synthetic sGFPmut3.1 gene<br>
SEQ ID NO 46:<br>
TGCAGCAAAGGCGAAGAACTGTTTACCGGTGTGGTGCCGATTCTGGTGGAACTGGATG<br>
GCGATGTGAACGGTCATAAATTTAGCGTGAGCGGCGAAGGTGAAGGCGATGCGACCTA<br>
TGGTAAACTGACCCTGAAATTTATTTGCACCACCGGCAAACTGCCGGTGCCGTGGCCG<br>
ACCCTGGTGACCACCTTTGGTTATGGCGTGCAGTGCTTTGCGCGCTATCCGGATCACA<br>
TGAAACAGCATGATTTTTTTAAAAGCGCGATGCCGGAAGGTTATGTGCAGGAACGCACC<br>
AI I I I I 1 I IAAAGATGATGGCAACTATAAAACCCGCGCGGAAGTGAAATTTGAAGGTGAT<br>
ACCCTGGTGAACCGCATTGAACTGAAAGGCATTGATTTTAAAGAAGATGGTAACATTCT<br>
GGGCCATAAACTGGAATATAACTATAACAGCCATAACGTGTATATTATGGCGGATAAAC<br>
AGAAAAACGGTATTAAAGTGMCTTTAAAATTCGCCATAACATTGAAGATGGCAGCGTG<br>
CAGCTGGCGGATCATTATCAGCAGAACACCCCGATTGGTGATGGCCCGGTGCTGCTGC<br>
CGGATAACCATTATCTGAGCACCCAGAGCGCGCTGAGCAAAGATCCGAACGAAAAACG<br>
CGATCACATGGTGCTGCTGGAATTTGTGACCGCGGCGGGTATTACGCATGGCATGGAT<br>
GAACTGTATAAA TAATAA<br>
in pUC119 custom-prepared by Operon Biotechnologies Inc with the primer pair<br>
sGFP-F-Spe, (SEQ ID NO 47):<br>
5'-GGA TCC ACT ACT TGC AGC AAA GGC GAA G-3'<br>
and<br>
sGFP-R-Sal (SEQ ID NO 48):<br>
5'-CGA GOT CGA CTT ATT ATT TAT ACA GTT CAT C-3'.<br>
The purified PCR product is then digested with Spel/Safl and ligated into the Spel/Sa/l fragment of 6H-EDDIE-lns, thereby substituting the proinsuline gene by sGmut3.1 to form the construct 6H-EDDIE-sGmut3.1-pET30a. The DNA sequences of each step are controlled as described (see 1.2.1.1).<br>
1.2.2 Transformation:<br>
Electrocompetent cells are prepared from one liter of bacterial culture (grown at 37 °C and 225 rpm to OD600=0, 5). The cell suspension is cooled on ice for 15 min (continuous agitation) pelleted (4 °C, 2500 g, and 10 min) and the supernatant removed completely. The remaining pellet is re-suspend in one liter of de-ionized water at 4 °C, spun down (4 °C,<br>
2500 g ,10 min) again and washed 2 times in 50 ml de-ionized water (4 °C) with intermittent centrifuging steps (4°C, 2500 g , 10 min). The pellet is finally washed with 50 ml 10% sterilized glycerol solution (4 °C) pelleted (4 °C, 2500 g ,10 min) and re-suspended in 2.5 ml 10 % sterilized glycerol solution (4 °C), frozen and stored in 40 ul aliquots at -80 °C. One aliquot of electrocompetent cells is thawed on ice, 1ul of ligation reaction containing 5ng DNA added and transferred without air bubbles to an electroporation cuvette with 1mm electrode gap. Electroporation takes place with a BIO-RAD Gene Pulser™ (Bio-Rad Laboratories Inc., 2000 Alfred Nobel Drive, Hercules, CA 94547, USA; cat. n. 1652077, Life Science Research Products 1998) including BIO-RAD pulse controller (Bio-Rad Laboratories Inc., 2000 Alfred Nobel Drive, Hercules, CA 94547, USA; cat. n. 1652098, Life Science Research Products 1998) set to 1,5 kV, 25 µF, 200 Ohms with a time constant shorter than 4.5 ms. Immediately thereafter 180 ul TY-broth (1.0% w/v Peptone, 0.7% w/v Yeast extract, 0.25% w/v NaCI) is added and the suspension transferred to a sterile 14 ml plastic tube and Incubated for 30 min (37 °C, 225 rpm). The suspension is then plated on selection medium. After incubation over night at 37 °C colonies are picked, transferred to 2 ml TY-broth and incubated over night at 37 °C and 225 rpm. 1 ml of the overnight culture is used for plasmid preparation by standard methods and the plasmid preparation subjected to restriction analysis and DNA sequencing. After verification by sequence analysis the plasmid is used for further transformation in expression strains by the method described herein.<br>
1.2.3 Expression of the fusion polypeptide<br>
Recombinant E. coli HMS 174 (DE3) containing a pET30 plasmid expressing a fusion polypeptide with an N-terminal autoprotease 6H-NproEDDIE and a C-terminal GFPmut3.1 with the following amino acid sequence are cultivated in baffled flasks in LB-broth at a total volume of 1.81.<br>
SEQ ID NO 49:<br>
1  MHHHHHHELN HFELLYKTSK QKPVGVEEPV YDTAGRPLFG NPSEVHPQST LKLPHDRGED	60<br>
61 DIETTLRDLP RKGDCRSGNH LGPVSGIYIK PGPVYYQDYT GPVYHRAPLE FFDETQFEET	120<br>
121 TKRIGRVTGS DGKLYHIYVE VDGEILLKQA KRGTPRTLKW TRNTTNCPLW VTSCSKGEEL	180<br>
181 FTGWPILVE LDGDVNGHKF SVSGEGEGDA TYGKLTLKFI CTTGKLPVPW PTLVTTFGYG	240<br>
241 VQCFARYPDH MKQHDFFKSA MPEGYVQERT IFFKDDGNYK TRAEVKFEGD TLVNRIELKG	300<br>
301 IDFKEDGNIL GHKLEYNYNS HNVYIMADKQ KNGIKVNFKI RHNIEDGSVQ LADHYQQNTP	360<br>
361 IGDGPVLLPD NHYLSTQSAL SKDPNEKRDH MVLLEFVTAA GITHGMDELY K<br>
Cells are cultivated as described under 1.1.5.<br>
12.4	Isolation of inclusion bodies<br>
Disruption of the cells is performed enzymatically. Briefly, the cells are suspended in 40 ml of 20 mM Tris/HCI, 5 mM EDTA, 2 mM MgCI2, pH 8.2. 72 mg of lysozyme and 300 U of Benzonase® are added. After incubation for 45 min at RT.1.3 g NaCI and 0.5 ml Triton X-100 are added. After another 15 min the suspension is centrifuged (Beckman JA 25.50, 10 000 rpm, 15 min, 4 °C) to obtain the inclusion bodies.<br>
12.5	Solubilization of inclusion bodies<br>
The pellet is washed once in 20 ml of 0.5 % deoxycholate, twice in 20 ml of 1 M NaCI, then washed with H2O. The resulting pellet (approximately 2 g wet weight) is suspended in 10 ml of H2O and stored at -20 °C for further use.<br>
An aliquot of the suspension is diluted 1 : 5 with 50 mM Tris/HCI, 10 M urea pH 7.3 to dissolve the inclusion bodies. After centrifugation to remove insoluble components the solution is diluted 1 : 5 with 50 mM Tris/HCI, 100 mM NaCI, 4 M urea, pH 7.3.<br>
12.6	Binding of the fusion polypeptide<br>
2 ml of the solution with an approximate polypeptide concentration of 2 mg/ml are applied onto a peptide affinity matrix as described above. Briefly, a Fractogel-DADPA-SA-AFYRWYA (0.5 inner diameter x 5 cm) is equilibrated with 50 mM Tris/HCI, 100 mM NaCI, 4 M Urea, pH 7.3.2 ml of the sample are injected at a flow rate of 25 cm/h.<br>
12.7	Washing out of unbound contaminating material<br>
Unbound components are washed out with 5 column volumes of equilibration buffer at a flow rate of 150 cm/h. Refolding is induced by a buffer exchange with 4.5 column volumes of 200 mM Tris/HCI, 2 mM EDTA, 10 % Glycerol, pH 7.3.<br>
1.2.8 Refolding, cleavage and elut'ion<br>
The bound fusion polypeptide is allowed to refold for 25 h by stopping the flow. Upon the refolding the bound autoprotease cleaves at its specific site and releases the fusion partner GFPmut3.1. Then the fusion polypeptide is washed out by the a 200 mM Tris/HCI, 2 mM EDTA, 10 % Glycerol, pH 7.3 buffer using a flow rate of 150 cm/h. 1 ml fractions are collected and analyzed for UV absorbance at 280 nm and fluorescence at an excitation of 488 nm and emission of 520 nm. Fractions containing the fusion partner GFPmut3.1 are further analyzed by SDS-PAGE for purified GFPmut3.1.<br>
12.9 Regeneration<br>
After elution of the cleaved off polypeptide of interest regeneration of the column is<br>
performed with 5 column volumes of 0.1 M NaOH at a flow rate of 150 cm/h.<br>
Example 2:<br>
Production of a heterologous polypeptide in accordance with the invention, utilizing<br>
immobilized metal ion affinity chromatography<br>
This example describes the production of GFPmut3.1 as a fusion polypeptide of a mutant of pestivirus autoprotease Npro: 6xHis- NproEDDIE, whereby refolding and cleavage are performed on an immobilized metal ion affinity matrix.<br>
The His-tag is introduced into the fusion polypeptide in order to be able to use the same construct in both (MAC and peptide affinity chromatography so that both methods can be directly compared. The tag is not required for the interaction of the fusion polypeptide with the oligopeptide ligand during affinity chromatography.<br>
Preparation and solubilization of inclusion bodies is performed as described in example 1.2.4 and 1.2.5.<br>
2.1	Preparation of chromatographic column binding of the fusion polypeptide to the<br>
chromatographic column<br>
Chelating Sepharose Fast flow (Amersham Biosciences) is packed into the column to a bed dimension of 0.5 inner diameter x 5 cm and storage solution is washed out with water. In a next step the metal ion Ni2+ is loaded onto the column. About two thirds of the total column volume of 100 mM NiCI2 or NiSO4 are applied. Unbound Ni2+ ions are washed out. After equilibration of the column with 50 mM Tris, 100 mM NaCI, 4 M urea, pH 7.3, 0.5 ml of the polypeptide solution with a concentration of about 2 mg/ml is applied onto the column. The loading flow rate is 50 cm/h.<br>
2.2	Washing out of unbound contaminating material<br>
After washing out unbound sample components with 5 column volumes of equilibration buffer, a buffer exchange to 500 mM Tris/Acetate, 0.25 M sucrose, 1 mM DTT, pH 7.3 is performed.<br>
2.3	Refolding, cleavage and elutlon<br>
After 4.5 column volumes the flow is stopped to allow the fusion polypeptide to refold and upon refolding the autoprotease cleaves off the fusion partner. Elution of the polypeptide of interest is performed by activating the flow again at a velocity of 150 cm/h using 1ml of 500 mM Tris/Acetate, 0.25 M sucrose, 1 mM DTT, phi 7.3buffer. Fractions are collected and analyzed by fluorescence measurements and SDS-PAGE.<br>
2.4	Regeneration<br>
Regeneration of the chromatography resin is carried out with 50 mM acetate, 6 M guanidinium chloride, pH 3.5 at a flow rate of 50cm/h.<br>
Example 3:<br>
On column cleavage of NproEDDIE-sSPA-D<br>
This example describes the production of staphyiococcal Protein A domain D by expression as a fusion polypeptide of a mutant of pestivirus autoprotease Npro: NproEDDIE, whereby refolding and cleavage are performed on a peptide affinity matrix. A fusion protein comprising the autoprotease NproEDDIE and the C-terminally fused Protein A domain D (sSPA-D) referred to as NproEDDIE-sSPA-D are prepared as follows:<br>
Recombinant E. coll HMS 174 (DE3) containing a pET30 plasmid expressing the fusion polypeptide with the following amino acid sequence are cultivated in a 101 fermenter as described in 1.1.5. Amino acids 1 to 168 of the entire fusion construct refer to the sequence of NproEDDIE whereas amino acids 169 to 229 refer to the sequence of sSPA-D.<br>
1        11       21        31        41       51<br>
I         1         I         I         I         I<br>
1  MELNHFELLY KTSKQKPVGV EEPVYDTAGR PLFGNPSEVH PQSTLKLPHD RGEDDIETTL	60<br>
61 RDLPRKGDCR SGNHLGPVSG IYIKPGPVYY QDYTGPVYHR APLEFFDETQ FEETTKRIGR	120<br>
121 VTGSDGKLYH IYVEVDGEIL LKQAKRGTPR TLKWTRNTTN CPLWVTSCAD AQQNKFNKDQ	180<br>
181 QSAFYEILNM PNLNEEQRNG FIQSLKDDPS QSTNVLGEAK KLNESQAPK<br>
Isolation and solubilization of inclusion bodies takes place as described in example 1.1.<br>
Binding of the fusion polypeptide to the chromatographic column Fractogel-DADPA-IT-peptide (0.5 x 5 cm) matrix, whereby selection and coupling of the respective peptide is conducted as described above, is equilibrated with 50 mM Tris/HCI, 100 mM NaCI, 4 M urea, pH 7.3 with a linear flow rate of 150 cm/h. 1 ml of the polypeptide solution is applied at a linear flow velocity of 50 cm/h. The flow rate is increased to 150 cm/h after sample injection.<br>
Washing out of unbound contaminating material<br>
Unbound components are washed out with 5 column volumes of equilibration buffer. A buffer exchange to refolding buffer, specifically to 1 M Tris/HCI, 2 mM EDTA, 0.25 M sucrose, 10 mM D-monothioglycerol, pH 7.3, is performed with 6 column volumes.<br>
Refolding, cleavage and elution<br>
After changing the conditions from chaotropic to cosmotropic, the fusion polypeptide is allowed to refold for 25 h on the chromatography resin by stopping the flow. The active autoprotease cleaves off the C-terminally fused sSPA-D. The subsequent elution with refolding buffer results in native sSPA-D. Regeneration of the matrices is performed with 10 CV of 0.2 M NaOH at 150 cm/h.<br>
Example 4:<br>
On column refolding and cleavage of 6His-Npr°EDDIE-GFPmut3.1<br>
This example describes the production of native GFPmut3.1 by expression of a 6His-NproEDDIE-GFPmut3.I fusion polypeptide, refolding and cleavage on an affinity matrix referred to as Actigel-polyKW. Chromatography conditions are the same as described above.<br>
Example 5:<br>
On column cleavage of NproEDDIE-sSPA-D<br>
This example describes the production of native sSPA-D by expression of a NproEDDIE-sSPA-D fusion polypeptide, refolding and cleavage on an affinity matrix referred to as Actigel-polyKY. Chromatography conditions are the same as described above.<br>
Example 6:<br>
On-column refolding using cation-exchange Chromatography<br>
Crude Npro37-6His<br>
((I)MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETTKRIGRV TGSDGKLYHIYVEVDGEILLKLAKRGTPRTLKWTRNTTNCPLWVTSC-{168)) inclusion body extracts were resuspended in 8 M urea, 50 mM Na-phosphate pH 7.0. The final protein concentration was 0.5 mg/ml. 2 ml were loaded onto a HiTrap SP Sepharose FF column (2.5 x 0.7 cm i.d.; column volume 1 ml; GE Healthcare) at a linear velocity of 50 cm/h previously equilibrated with the same buffer as described above. The column was then buffer-exchanged into a buffer containing 50 mM Na-phosphate pH 7, 2 mM EDTA, 5 % glycerol, 10 mM a-monothioglycerol (MTG). The protein was allowed to refold for 1 hour at room temperature. Elution of refolded and cleaved proteins was carried out by further application of refolding buffer. Regeneration was performed with a buffer containing 2 M NaCI, 50 mM Na-phosphate pH 7. Refolding of the fusion protein (6 His) was monitored by SDS-PAGE analysis.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Claims<br>
1.	A method for the production of a heterologous polypeptide of interest with a homogenous<br>
N-terminus, using a fusion polypeptide comprising the polypeptide of interest and N-<br>
terminally thereto  a  polypeptide  exhibiting  autoproteolytic  function,   said   method<br>
comprising  the  steps  of  a)   binding   of the  fusion   polypeptide  in   a  soluble,<br>
autoproteolytically inactive form by an affinity chromatography system, b) refolding of the<br>
fusion  polypeptide,  thereby activating  the autoproteolytic function of the fusion<br>
polypeptide and causing cleavage of the heterologous polypeptide of interest, and c)<br>
subsequently eluting the heterologous polypeptide of interest, wherein said steps are<br>
conducted on one affinity chromatography system.<br>
2.	The method according to claim 1, wherein the fusion polypeptide is provided by<br>
recombinant expression in a bacterial host cell in the form of inclusion bodies, utilizing a<br>
host cell that is transformed with an expression vector comprising a nucleic acid<br>
molecule which codes for the fusion polypeptide.<br>
3.	The method according to claim 1, wherein the polypeptide exhibiting autoproteolytic<br>
function is an autoprotease.<br>
4.	The method according to claim 3, wherein the autoprotease is Npr0 of a pestivirus, or a<br>
derivative thereof with autoproteolytic function.<br>
5.	The method according to claim 4, wherein the pestivirus is selected from the group<br>
consisting of CSFV, BDV and BVDV.<br>
6.	The method according to claim 5, wherein the autoprotease is Npro of CSFV and has the<br>
following amino acid sequence:<br>
SEQIDN01:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDI RmRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFCEVT KRIGRVTGSDGKLYHIWCVDGCILLKLAKRGTPRTLKWIRNFTNCPLWVTSC-(168),<br><br>
or the amino acid sequence of a derivative thereof with autoproteolytic function.<br>
7.	The method according to claim 5, wherein the autoprotease is a derivative of Npro of<br>
CSFV and has the following amino acid sequence:<br>
SEQ ID NO 2:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDI RTTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFEEV TKRIGRVTGSDGKLYHIYVEVDGEILLKLAKRGTPRTLKWIRNFTNCPLWVTSC-(168)<br>
8.	The method according to claim 5, wherein the autoprotease is a derivative of Npro of<br>
CSFV and has the following amino add sequence:<br>
SEQ ID NO 3:<br>
(1)-MELNHFELLYKTSKQKPVGVEEP\ADTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFEEV TKRIGRWGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWIRNFTNCPLVVVTSC-(168)<br>
9.	The method according to claim 5, wherein the autoprotease is a derivative of Npra of<br>
CSFV and has the following amino acid sequence:<br>
SEQ ID NO 4:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDI RTTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEET TKRIGRWGSDGKLYHIYVE\/DGEILLKLAKRGTPRTLKWTRNTTNCPLVVVTSC-{168)<br>
10.	The method according to claim 5, wherein the autoprotease is a derivative of N*"0 of<br>
CSFV and has the following amino acid sequence:<br>
SEQ ID NO 5:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETT KRIGRWGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKVVTRNTTNCPLVVVTSC-(168).<br><br>
11.	The  method   according   to  any  of  the   preceding   claims  wherein  the  affinity<br>
chromatography system is selected from the group consisting of immobilized metal ion<br>
chromatography   (IMAC),    cation   exchange    chromatography,    anion    exchange<br>
chromatography,   cellulose   binding  domain  chromatography  and  peptide  affinity<br>
chromatography.<br>
12.	The method according to claim 11, wherein the affinity chromatography system is<br>
immobilized metal ion chromatography and wherein the fusion polypeptide comprises a<br>
metal chelate affinity tag.<br>
13 The method according to claim 12, wherein the metal chelate affinity tag is polyhistidine.<br>
14.	The method according to claim 11, wherein the affinity chromatography system is cation<br>
exchange chromatography and wherein the fusion polypeptide comprises a polycationic<br>
affinity tag.<br>
15.	The method according to claim  14, wherein the polycationic affinity tag is selected from<br>
polyarginine and polylysine.<br>
16.	The method according to claim 11, wherein the affinity chromatography system is anion<br>
exchange chromatography and wherein the fusion polypeptide comprises a polyanionic<br>
tag.<br>
17.	The method according to claim 16, wherein the polyanionic tag is polyasparagine.<br>
18.	The method according to claim 11, wherein said peptide affinity chromatography system<br>
utilizes an oligopeptide ligand of five to twelve amino acids length, comprising a<br>
tryptophan residue, which ligand selectively binds to the part of the fusion polypeptide<br>
exerting autoproteolytic function under chaotropic conditions and maintains binding<br>
during change towards as well as under cosmotropic conditions.<br>
19.	The method according to claim 18, wherein said oligopeptide ligand has a length of six<br>
to eight amino acids.<br><br>
20. The method according to claim 19 , wherein the oligopeptide ligand has an amino acid sequence selected from the group consisting of<br>
SEQ ID NO 6:	VSIFEW,<br>
SEQ ID NO 7:	AVSIEWY,<br>
SEQ ID NO 8:	AVSFIWY,<br>
SEQ ID NO 9:	VSFIWYK,<br>
SEQ ID NO 10:	ASRFWYA,<br>
SEQ ID NO 11:	AFYTWYA,<br>
SEQ ID NO 12:	AFYRWYK,<br>
SEQ ID NO 13:	AFYRWY,<br>
SEQ ID NO 14:	AFYRWYA,<br>
SEQ ID NO 15:	AVSIFEWY,<br>
SEQ ID NO 16:	AVSRNWY,<br>
SEQ ID NO 17:	ASRFWY,<br>
SEQ ID NO 18:	AFYRWYAA,<br>
SEQ ID NO 19:	AFYRWY,<br>
SEQ ID NO 20:	ASRFWYAA,<br>
SEQ ID NO 21:	AFYRWYAA and<br>
SEQ ID NO 22:	AFYSWYAA.<br>
21.   The method according to claim 18, wherein the derivative of the naturally occurring N™ of CSFV according to SEQ ID NO 5 is used in combination with an oligopeptide ligand selected from the group consisting of SEQ ID NO 10:     ASRFWYA, SEQ ID NO 11:     AFYTWYA, SEQ ID NO 12:     AFYRWYK, SEQ ID NO 13:     AFYRWY and SEQ ID NO 14:     AFYRWYA.<br>
22.	The method according to any of the preceding claims, wherein the step of refolding the<br>
fusion polypeptide is performed by the change from chaotropic to cosmotropic<br>
conditions via buffer exchange.<br>
23.	An oligopeptide ligand for use in a method according to any of the preceding claims.<br><br>
24.	A derivative of Npro of CSFV for use in a method according to any of the claims 1 to 22.<br>
25.	Use of an oligopeptide ligand in a method according to any of the claims 1 to 22.<br>
26.	Use of a derivative of Npro of CSFV in a method according to any of the claims 1 to 22.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi0tZGVsbnAtMjAwNy1EZXNjcmlwdGlvbiAoQ29tcGxldGUpLSgxMi0wNC0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">7912--delnp-2007-Description (Complete)-(12-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNsYWltcy0oMTItMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Claims-(12-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNsYWltcy0oMjYtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Claims-(26-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMTAtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence Others-(07-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTItMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence Others-(12-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence Others-(14-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence Others-(15-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence Others-(26-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTMtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7912-DELNP-2007-Correspondence-Others (13-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTktMTEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence-Others (19-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence-Others-(04-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTAtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7912-DELNP-2007-Correspondence-Others-(10-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjYtMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Correspondence-Others-(26-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDEyLTA0LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Description (Complete)-(12-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1ERUxOUC0yMDA3LUZvcm0tMS0oMTAtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7912-DELNP-2007-Form-1-(10-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWZvcm0tMTMtKDEwLTA4LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-form-13-(10-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1ERUxOUC0yMDA3LUZvcm0tMi0oMTAtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7912-DELNP-2007-Form-2-(10-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUZvcm0tMyAoMTktMTEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Form-3 (19-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUZvcm0tMy0oMDctMTAtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Form-3-(07-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUZvcm0tMy0oMTQtMDYtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-Form-3-(14-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LUdQQS0oMjYtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-GPA-(26-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzkxMi1kZWxucC0yMDA3LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">7912-delnp-2007-pct-308.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263424-a-drying-apparatus.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263426-distributed-architecture-and-methods-for-broadcast-multicast-service.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263425</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7912/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Oct-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANDOZ AG.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LICHTSTRASSE 35, CH-4056 BASEL SWITZERLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JUNGBAUER ALOIS</td>
											<td>ICHAGASSE 31-1, A-1210 VIENNA AUSTRIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HAHN RAINER</td>
											<td>LEOPOLD ERNST GASSE 4/2/1, A-1180 VIENNA AUSTRIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TSCHELIESSNIG ANNE</td>
											<td>EGGERSTRASSE 6, A-9581 LEDENITZEN AUSTRIA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KAAR WALTRAUD</td>
											<td>15 CARRINGTON ROAD, INDOOROOPILLY, QLD 4068 AUSTRALIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/62</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AT2006/000168</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-04-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0605379.7</td>
									<td>2006-03-16</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0508435.5</td>
									<td>2005-04-26</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>0508434.8</td>
									<td>2005-04-26</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263425-production-of-recombinant-proteins-by-autoproteolytic-cleavage-of-a-fusion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:54:04 GMT -->
</html>
