<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263449-lipase-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:46:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263449:LIPASE VARIANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">LIPASE VARIANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT LIPASE VARIANTS The invention provides variant lipases, preferably, variants with reduced tendency to odor generation obtained by introducing mutations in one or more regions identified in a parent lipase.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
LIPASE VARIANTS<br>
FIELD OF THE INVENTION<br>
The present invention relates to lipase variants.<br>
BACKGROUND OF THE INVENTION<br>
Lipases are useful, e.g., as detergent enzymes to remove lipid or fatty stains from clothes and other textiles, as additives to dough for bread and other baked products. Thus, a lipase derived from Thermomyces lanuginosus (synonym Humicola lanuginosa, EP 258 068 and EP 305 216) is sold for detergent use under the trade name Lipolase ® (product of Novo Nordisk A/S). WO 0060063 describes variants of the T. lanuginosus lipase with a particularly good first-wash performance in a detergent solution. WO 9704079, WO 9707202 and WO 0032758 also disclose variants of the T. lanuginosus lipase.<br>
In some applications, it is of interest to minimize the formation of odor-generating short-chain fatty acids. Thus, it is known that laundry detergents with lipases may sometimes leave residual odors attached to cloth soiled with milk (EP 430315). WO 02062973 discloses lipase variants where the odor generation has been reduced by attaching a C-terminal extension.<br>
SUMMARY OF THE INVENTION<br>
The inventors have found that by introducing mutations in certain regions/positions of a lipase it is possible to improve the properties or characteristics of the lipase.<br>
In a preferred embodiment, the present invention relates to lipases having improved properties for use in detergents. For example, the invention provides variants with reduced ten¬dency to odor generation obtained by introducing mutations in one or more regions identified in the parent lipase. In another preferred embodiment, the present invention provides lipase vari¬ants which, as compared to the parent lipase, have reduced potential for odor generation with¬out the attachment of a C-terminal extension.<br>
In a further aspect the invention relates to a DNA sequence encoding the lipase variant of the invention, an expression vector harbouring said DNA sequence and a transformed host cell containing the DNA sequence or the expression vector.<br>
In another aspect, the invention provides a method of producing the lipase variant of the present invention.<br><br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 shows the alignment of lipases.<br>
SEQUENCE LISTINGS<br>
SEQ ID NO: 1 shows the DMA sequence encoding lipase from Thermomyces lanoginosus. SEQ ID NO: 2 shows the amino acid sequence of a lipase from Thermomyces lanoginosus. SEQ ID NO: 3 shows the amino acid sequence of a lipase from Absidia reflexa. SEQ ID NO: 4 shows the amino acid sequence of a lipase from Absidia corymbifera. SEQ ID NO: 5 shows the amino acid sequence of a lipase from Rhizomucor miehei. SEQ ID NO: 6 shows the amino acid sequence of a lipase from Rhizopus oryzae. SEQ ID NO: 7 shows the amino acid sequence of a lipase from Aspergillus niger. SEQ ID NO: 8 shows the amino acid sequence of a lipase from Aspergillus tubingensis. SEQ ID NO: 9 shows the amino acid sequence of a lipase from Fusarium oxysporrum. SEQ ID NO: 10 shows the amino acid sequence of a lipase from Fusarium heterosporum. SEQ ID NO: 11 shows the amino acid sequence of a lipase from Aspergillus oryzae. SEQ ID NO: 12 shows the amino acid sequence of a lipase from Penicillium camemberti. SEQ ID NO: 13 shows the amino acid sequence of a lipase from Aspergillus foetidus. SEQ ID NO: 14 shows the amino acid sequence of a lipase from Aspergillus niger SEQ ID NO: 15 shows the amino acid sequence of a lipase from Aspergillus oryzae. SEQ ID NO: 16 shows the amino acid sequence of a lipase from Landerina penisapora.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Parent lipases<br>
Any suitable parent lipase may be used. In a preferred embodiment, the parent lipase may be a fungal lipase. In another preferred embodiment, the parent lipase may be a lipase with an amino acid sequence having at least 50%, 60%, 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99% or even 100% homology as defined in the section "Homology and aligment" to the sequence of the T. lanuginosus lipase shown in SEQ ID NO: 2.<br>
The parent lipase may be a yeast polypeptide such as a Candida, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, or Yarrowia polypeptide; or more preferably a filamentous fungal polypeptide such as an Acremonium, Aspergillus, Aureobasidium, Crypto-coccus, Filobasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocalli-<br><br>
mastix, Neurospora, Paecilomyces, Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Trichoderma polypeptide.<br>
In a preferred aspect, the parent lipase is a Saccharomyces carlsbergensis, Sac-charomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis, or Saccharomyces oviformis polypeptide having lipase ac¬tivity.<br>
In another preferred aspect, the parent lipase is an Aspergillus aculeatus, Aspergillus awamori, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidu-lans, Aspergillus niger, Aspergillus oryzae, Aspergillus turbigensis, Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusa¬rium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, Fusa¬rium venenatum, Humicola insolens, Thermomyces lanoginosus (synonym: Humicola lanugi-nose), Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpuro-genum, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Tricho¬derma reesei, or Trichoderma viride polypeptide.<br>
In another preferred aspect, the parent lipase is a Thermomyces lipase.<br>
In a more preferred aspect, the parent lipase is a Thermomyces lanuginosus lipase. In an even more preferred embodiment the parent lipase is the lipase of SEQ ID NO: 2.<br>
Variant lipases<br>
The lipase variants of the present invention comprise, as compared to the parent li¬pase, at least three substitutions selected from the group consisting of:<br>
a)	at least two substitution in Region I, and<br>
b)	at least one substitution in Region II, and<br>
c)	at least one substitution in Region III, and<br>
d)	at least one substitution in Region IV;' and wherein the variant has lipase activity.<br>
In a preferred embodiment, the variant lipase is a variant of a Thermomyces lipase, more preferably, a T. lanuginosus lipase, and even more preferably, the T. lanuginosus lipase shown in SEQ ID NO: 2. In a preferred embodiment, the variant lipase has at least 50%, 60%, 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to SEQ ID N0:2.<br>
The variant lipase may be a variant of a parent lipase encoded by a gene de-<br><br>
rived/obtained from one of the following parent organisms: Candida, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, or Yarrowia, Acremonium, Aspergillus, Aureobasidium, Cryptococcus, Filobasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neo-callimastix, Neurospora, Paecilomyces, Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thieiavia, Tolypocladium, or Trichoderma. In a preferred embodiment, the vari¬ant lipase has at least 50%, 60%, 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to a parent lipase encoded by a gene derived/obtained from one of the following parent organ¬isms: Candida, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, or Yarrowia, Acremonium, Aspergillus, Aureobasidium, Cryptococcus, Filobasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thieiavia, Tolypocladium, or Tricho¬derma.<br>
In a preferred aspect, the variant lipase is a variant Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharo¬myces kluyveri, Saccharomyces norbensis, or Saccharomyces oviformis. In a preferred em¬bodiment, the variant lipase has at least 50%, 60%, 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to a parent lipase encoded by a gene derived/obtained from Saccharomy¬ces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis, or Saccharomyces oviformis.<br>
The variant lipase may be a variant of a parent lipase encoded by a gene de¬rived/obtained from one of the following parent organisms: Aspergillus aculeatus, Aspergillus awamori, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidu-lans, Aspergillus niger, Aspergillus oryzae, Aspergillus turbigensis, Fusarium bactridioides, Fusarium cereaiis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusa¬rium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, Fusa¬rium venenatum, Humicola insolens, Thermomyces lanoginosus (synonym: Humicola lanugi-nose), Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpuro-genum, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Tricho¬derma reesei, or Trichoderma viride. In a preferred embodiment, the variant lipase has at least 50%, 60%, 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to a parent lipase en¬coded by a gene derived/obtained from one of the following parent organisms: Aspergillus acu¬leatus, Aspergillus awamori, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus.<br><br>
Aspergillus nidulans, Aspergillus niger, Aspergillus oryzae, Aspergillus turbigensis, Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium ox-ysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sar-cochroum, Fusarium sporotrichioides, Fusarium sulpfiureum, Fusarium torulosum, Fusarium trichothecioides, Fusarium venenatum, Humicola insolens, Thermomyces lanoginosus (syno¬nym: Humicola lanuginose), Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpurogenum, Triclioderma harzianum, Trichoderma koningii, Trichoderma longi-brachiatum, Trichoderma reesei, or Trichoderma viride.<br>
In another preferred aspect, the variant is a variant of a Thermomyces lipase.<br>
In a more preferred aspect, the parent lipase is a Thermomyces lanuginosus lipase. In an even more preferred embodiment the parent lipase is the lipase of SEQ ID NO: 2.<br>
Identification of regions and substitutions<br>
The positions referred to in Region I through Region IV below are the positions of the amino acid residues in SEQ ID N0:2. To find the corresponding (or homologous) positions in a different lipase, the procedure described in "Homology and alignment" is used.<br>
Substitutions in Region I<br>
Region I consists of amino acid residues surrounding the N-terminal residue E1. In this region, it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid. The lipase variant may comprise at least two substitutions in Region I, such as three, four, five or six substitutions in Region I.<br>
Amino acid residues corresponding to the following positions are comprised by Region I: 1, 2 to 11 and 223-239. The following positions are of particular interest: 1, 4, 8, 11, 223, 227, 229,231,233,234,236.<br>
In particular the following substitutions have been identified: X1N/* X4V, X227G, X231RandX233R.<br>
In a preferred embodiment the variant lipase has at least 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99% identity to SEQ ID N0:2<br>
In a most preferred embodiment the variant lipase is a variant of the lipase having the aminio acid sequence of SEQ ID NO: 2. Substitutions in Region II<br>
Region 11 consists of amino acid residues in contact with substrate on one side of the<br><br>
acyl chain and one side of tlie alcohol part. In this region it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid or with a less hydrophobic amino acid.<br>
The lipase variant may comprise at least one substitution in Region II, such as two, three, four, five or six substitutions in Region II.<br>
Amino acid residues corresponding to the following positions are comprised by Region II: 202 to 211 and 249 to 269. The following positions are of particular interest: 202, 210, 211, 253, 254, 255, 256.<br>
In particular the following substitutions have been identified: X202G, X210K/W/A, X255YA//A and X256K/R and X259G/M/QA/.<br>
In a preferred embodiment the variant lipase has at least 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to SEQ ID N0:2<br>
In a most preferred embodiment the variant lipase is a variant of the lipase having the aminio acid sequence of SEQ ID NO: 2.<br>
Substitutions in Region III<br>
Region III consists of amino acid residues that forms a flexible structure and thus allow¬ing the substrate to get into the active site. In this region it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid or a less hydrophobic amino acid.<br>
The lipase variant may comprise at least one substitution in Region III, such as two, three, four, five or six substitutions in Region III.<br>
Amino acid residues corresponding to the following positions are comprised by Region III: 82 to 102. The following positions are of particular interest: 83, 86, 87, 90, 91, 95, 96, 99.<br>
In particular the following substitutions have been identified: X83T, X86V and X90A/R.<br>
In a preferred embodiment the variant lipase has at least 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to SEQ ID N0:2<br>
In a most preferred embodiment the variant lipase is a variant of the lipase having the aminio acid sequence of SEQ ID NO: 2.<br>
Substitutions in Region IV<br>
Region IV consists of amino acid residues that binds electrostatically to a surface. In this region it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid.<br>
The lipase variant may comprise at least one substitution in Region IV, such as two, three, four, five or six substitutions in Region IV.<br><br>
Amino acid residues corresponding to the following positions are comprised by Region IV: 27 and 54 to 62. The following positions are of particular interest: 27, 56, 57, 58, 60.<br>
In particular the following substitutions have been identified: X27R, X58N/AG/T/P and X60V/S/G/N/R/K/A/L.<br>
In a preferred embodiment the variant lipase has at least 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% identity to SEQ ID N0:2<br>
In a most preferred embodiment the variant lipase is a variant of the lipase having the aminio acid sequence of SEQ ID NO: 2.<br>
Amino acids at other positions<br>
The parent lipase may optionally comprise additional alterations, e.g, substitution of other amino acids, particularly less than 10, less than 9, less than 8, less than 7, less than 6, less than 5 alterations as compared to a parent lipaase. Examples are substitutions correspond¬ing to one or more of the positions 24, 37, 38,46, 74, 81, 83, 115, 127, 131, 137, 143, 147, 150, 199, 200, 203, 206, 211, 263, 264, 265, 267 and 269 of the parent lipase. In a particular em¬bodiment there is a substitution in at least one of the positions corresponding to position 81, 147, 150, 227 and 249. In a preferred embodiment the at least one substitution is selected from the group consisting of X38R, X81Q/E, X143S/C/N/D/A, X147MA', X150G/K, X227G and X249R/I/L.<br>
The variant may comprise substitutions outside the defined Region I to IV; the number of substitutions outside the defined Region I to IV is preferably less than six, such as five, four, three, two or one substitution.<br>
Further substitutions may e.g., be made according to principles known in the art, e.g. substitutions described in WO 92/05249, WO 94/25577, WO 95/22615, WO 97/04079 and WO 97/07202.<br><br><br><br><br><br>
Nomenclature for amino acid modifications<br>
In describing lipase variants according to the invention, the following nomenclature is used for ease of reference:<br>
Original amino acid(s):position(s):substituted amino acid(s)<br>
According to this nomenclature, for instance the substitution of glutamic acid for glycine in position 195 is shown as G195E. A deletion of glycine in the same position is shown as G195*, and insertion of an additional amino acid residue such as lysine is shown as G195GK.<br>
Where a specific lipase contains a "deletion" in comparison with other lipases and an insertion is made in such a position this is indicated as *36D for insertion of an aspartic acid in position 36.<br>
Multiple mutations are separated by pluses, i.e.: R170Y+G195E, representing muta¬tions in positions 170 and 195 substituting tyrosine and glutamic acid for arginine and glycine, respectively.<br>
X231 indicates the amino acid in a parent polypeptide corresponding to position 231, when applying the described alignment procedure. X231R indicates that the amino acid is re¬placed with R. For SEQ ID N0:2 X is T, and X231R thus indicates a substitution of T in position 231 with R. Where the amino acid in a position (e.g. 231) may be substituted by another amino acid selected from a group of amino acids, e.g. the group consisting of R and P and Y, this will be indicated by X231R/PA'.<br>
In all cases, the accepted lUPAC single letter or triple letter amino acid abbreviation is employed.<br>
Amino acid grouping<br>
In this specification, amino acids are classified as negatively charged, positively charged or electrically neutral according to their electric charge at pH 10. Thus, negative amino acids are E, D, C (cysteine) and Y, particularly E and D. Positive amino acids are R, K and H, particularly R and K. Neutral amino acids are G, A, V, L, 1, P, F, W, S, T, M, N, Q and C when forming part of a disulfide bridge. A substitution with another amino acid in the same group (negative, positive or neutral) is termed a conservative substitution.<br>
The neutral amino acids may be divided into hydrophobic or non-polar (G, A, V, L, I, P, F, W and C as part of a disulfide bridge) and hydrophilic or polar (S, T, M, N, Q).<br><br>
Amino acid identity<br>
The relatedness between two amino acid sequences or between two nucleotide se¬quences is described by the parameter "identity".<br>
For purposes of the present invention, the alignment of two amino acid sequences is determined by using the Needle program from the EMBOSS package (http://emboss.org) ver¬sion 2.8.0. The Needle program implements the global alignment algorithm described in Nee-dleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453. The substitution matrix used is BLOSUM62, gap opening penalty is 10, and gap extension penalty is 0.5.<br>
The degree of identity between an amino acid sequence of the present invention ("in¬vention sequence"; e.g. amino acids 1 to 269 of SEQ ID N0:2) and a different amino acid se¬quence ("foreign sequence") is calculated as the number of exact matches in an alignment of the two sequences, divided by the length of the "invention sequence" or the length of the "for¬eign sequence", whichever is the shortest. The result is expressed in percent identity.<br>
An exact match occurs when the "invention sequence" and the "foreign sequence" have identical amino acid residues in the same positions of the overlap. The length of a se¬quence is the number of amino acid residues in the sequence (e.g. the length of SEQ ID N0:2 is 269).<br>
The above procedure may be used for calculation of identity as well as homology and for alignment. In the context of the present invention homology and alignment has been calcu¬lated as described below.<br>
Homology and alignment<br>
For purposes of the present invention, the degree of homology may be suitably deter¬mined by means of computer programs known in the art, such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genet¬ics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology 48, 443-45), using GAP with the follow¬ing settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP exten¬sion penalty of 0.1.<br>
In the present invention, corresponding (or homologous) positions in the lipase se¬quences of Absidia reflexa, Absidia corymbefera, Rhizmucor miehei, Rhizopus delemar, Asper¬gillus niger, Aspergillus tubigensis, Fusarium oxysporum, Fusarium heterosporum, Aspergillus oryzea, Penicilium camembertii, Aspergillus foetidus, Aspergillus niger, Thermomyces lanogi-nosus (synonym: Humicola lanuginose) and Landerina penisapora are defined by the alignment<br><br>
shown in Figure 1.<br>
To find the homologous positions in lipase sequences not shown in the alignment, the sequence of interest is aligned to the sequences shown in Figure 1. The new sequence is aligned to the present alignment in Figure 1 by using the GAP alignment to the most homolo¬gous sequence found by the GAP program. GAP is provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-45). The following settings are used for polypeptide se-quence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1.<br>
Hybridization<br>
The present invention also relates to isolated polypeptides having lipase activity which are encoded by polynucleotides which hybridize under very low stringency conditions, prefera¬bly low stringency conditions, more preferably medium stringency conditions, more preferably medium-high stringency conditions, even more preferably high stringency conditions, and most preferably very high stringency conditions with (i) nucleotides 178 to 660 of SEQ ID NO: 1, (ii) the cDNA sequence contained in nucleotides 178 to 660 of SEQ ID NO: 1, (iii) a subsequence of (i) or (ii), or (iv) a complementary strand of (i), (ii), or (iii) (J. Sambrook, E.F Fritsch, and T. Maniatus, 1989, Molecular Cloning, A Laboratory Manual, 2d edition. Cold Spring Harbor, New York). A subsequence of SEQ ID NO: 1 contains at least 100 contiguous nucleotides or pref¬erably at least 200 contiguous nucleotides. Moreover, the subsequence may encode a polypep¬tide fragment which has lipase activity.<br>
For long probes of at least 100 nucleotides in length, very low to very high stringency conditions are defined as prehybridization and hybridization at 42°C in 5X SSPE, 0.3% SDS, 200 ug/ml sheared and denatured salmon sperm DNA, and either 25% formamide for very low and low stringencies, 35% formamide for medium and medium-high stringencies, or 50% for¬mamide for high and very high stringencies, following standard Southern blotting procedures for 12 to 24 hours optimally.<br>
For long probes of at least 100 nucleotides in length, the carrier material is finally washed three times each for 15 minutes using 2X SSC, 0.2% SDS preferably at least at 45°C (very low stringency), more preferably at least at 50°C (low stringency), more preferably at least at 55°C (medium stringency), more preferably at least at 60°C (medium-high stringency), even more preferably at least at 65°C (high stringency), and most preferably at least at 70°C (very<br><br>
high stringency).<br>
DNA sequence, Expression vector, Host cell, Production of lipase<br>
The invention provides a DNA sequence encoding the lipase of the invention, an ex¬pression vector harboring the DNA sequence, and a transformed host cell containing the DNA sequence or the expression vector. These may be obtained by methods known in the art.<br>
The invention also provides a method of producing the lipase by culturing the trans¬formed host cell under conditions conducive for the production of the lipase and recovering the lipase from the resulting broth. The method may be practiced according to principles known in the art.<br>
Lipase activity<br>
Lipase activity on tributvrin at neutral pH (LU)<br>
A substrate for lipase is prepared by emulsifying tributyrin (glycerin tributyrate) using gum Arabic as emulsifier. The hydrolysis of tributyrin at 30 °C at pH 7 or 9 is followed in a pH-stat titration experiment. One unit of lipase activity (1 LU) equals the amount of enzyme capable of releasing 1 micro mol butyric acid/min at pH 7.<br>
Benefit Risk<br>
The Benefit Risk factor describing the performance compared to the reduced risk for odor smell is defined as: BR = RPavg / R, as described below.<br>
Uses<br>
Enzymes of the present invention may find industrial use, e.g. be included in detergent compositions for removing of fatty matter.<br>
EXAMPLES<br>
Chemicals used as buffers and substrates were commercial products of at least reagent grade.<br>
IVIedia and Solutions<br>
Product	Tradename<br>
LAS :	Surfac PS<br>
Zeolite A	Wessalith P<br><br>
other ingredients used are standard laboratory reagents.<br>
Materials<br>
Product Supplier<br>
EMPA221	EMPA St. Gallen, Lerchfeldstrasse 5, CH-9014 St. Gallen, Switzerland<br>
Example 1 Production of enzyme<br>
A plasmid containing the gene encoding the lipase is constructed and transformed into a suitable host cell using standard methods of the art.<br>
Fermentation is carried out as a fed-batch fermentation using a constant medium tem¬perature of 34°C and a start volume of 1.2 liter. The initial pH of the medium is set to 6.5. Once the pH has increased to 7.0 this value is maintained through addition of 10% H3P04. The level of dissolved oxygen in the medium is controlled by varying the agitation rate and using a fixed aeration rate of 1.0 liter air per liter medium per minute. The feed addition rate is maintained at a constant level during the entire fed-batch phase.<br>
The batch medium contains maltose syrup as carbon source, urea and yeast extract as nitrogen source and a mixture of trace metals and salts. The feed added continuously during the fed-batch phase contains maltose syrup as carbon source whereas yeast extract and urea is added in order to assure a sufficient supply of nitrogen.<br>
Purification of the lipase may be done by use of standard methods known in the art, e.g. by filtering the fermentation supernatant and subsequent hydrophobic chromatography and ion exchange chromatography, e.g. as described in EP 0 851 913 EP, Example 3.<br>
Example 2<br>
AMSA - Automated Mechanical Stress Assay - for calculation of Relative Performance<br>
(RP)<br>
The enzyme variants of the present application are tested using the Automatic Me¬chanical Stress Assay (AMSA). With the AMSA test the wash performance of a large quantity of small volume enzyme-detergent solutions can be examined. The AMSA plate has a number of slots for test solutions and a lid firmly squeezing the textile swatch to be washed against all the slot openings. During the washing time, the plate, test solutions, textile and lid are vigorously shaken to bring the test solution in contact with the textile and apply mechanical stress. For fur¬ther description see WO 02/42740 especially the paragraph "Special method embodiments" at<br><br>
page 23-24. The containers, which contain the detergent test solution, consist of cylindrical holes (6 mm diameter, 10 mm depth) in a metal plate. The stained fabric (test material) lies on the top of the metal plate and is used as a lid and seal on the containers. Another metal plate lies on the top of the stained fabric to avoid any spillage from each container. The two metal plate together with the stained fabric are vibrated up and down at a frequency of 30 Hz with an amplitude of 2 mm.<br><br><br>
Cream-turmeric swatches were prepared by mixing 5 g of turmeric (Santa IVIaria, Den¬mark) with 100 g cream (38% fat, Aria, Denmark) at 50°C, the mixture was left at this tempera¬ture for about 20 minutes and filtered (50°C) to remove any undissolved particles. The mixture is cooled to 20°C) woven cotton swatches, EMPA221, were immersed in the cream-turmeric mix¬ture and afterwards allowed to dry at room temperature over night and frozen until use. The preparation of cream-turmeric swatches is disclosed in the patent application WO 2006/125437.<br>
The performance of the enzyme variant is measured as the brightness of the colour of the textile samples washed with that specific enzyme variant. Brightness can also be expressed as the intensity of the light reflected from the textile sample when luminated with white light. When the textile is stained the intensity of the reflected light is lower, than that of a clean textile. Therefore the intensity of the reflected light can be used to measure wash performance of an enzyme variant.<br>
Color measurements are made with a professional flatbed scanner (PFU DL2400pro), which is used to capture an image of the washed textile samples. The scans are made with a resolution of 200 dpi and with an output color depth of 24 bits. In order to get accurate results, the scanner is frequently calibrated with a Kodak reflective ITS target.<br>
To extract a value for the light intensity from the scanned images, a special designed software application is used (Novozymes Color Vector Analyzer). The program retrieves the 24 bit pixel values from the image and converts them into values for red, green and blue (RGB). The intensity value (Int) is calculated by adding the RGB values together as vectors and then taking the length of the resulting vector:<br>
The wash performance (P) of the variants is calculated in accordance with the below formula:<br>
P	=   Int(v) - Int(r)<br>
where<br>
Int(v) is the light intensity value of textile surface washed with enzyme, and Int(r) is the light intensity value of textile surface washed without enzyme.<br>
A relative performance score is given as the result of the AMSA wash in accordance with the<br>
definition:<br>
Relative Performance scores (RP) are summing up the performances (P) of the tested<br>
enzyme variants against the reference enzyme:<br><br>
RP = P(test enzyme) / P(reference enzyme).<br>
RPavg indicates the average relative performance compared to the reference enzyme at all four enzyme concentrations (0.125, 0.25, 0.5, 1.0 mg ep/l)<br>
RPavg = avg(RP(0.125), RP(0.25) RP(0.5), RP(1.0))<br>
A variant is considered to exhibit improved wash performance, if it performs better than the reference.<br>
In the context of the present invention the reference enzyme is the lipase of SEQ ID N0:2 with the substitutions T231R + N233R.<br>
Example 3<br>
GC - Gas Chromatograph - for calculation of risk factor<br>
The butyric acid release from the lipase washed swatches were measured by Solid Phase Micro Extraction Gas Chromatography (SPME-GC) using the following method. Four textile pieces (5 mm in diameter), washed in the specified solution in Table 3 containing 1 mg/l lipase, were transferred to a Gas Chromatograph (GC) vial. The samples were analysed on a Vanan 3800 GC equipped with a Stabilwax- DA w/lntegra-Guard column (30m, 0.32 mm ID and 0.25 micro-m df) and a Carboxen PDMS SPME fibre (75 micro-m). Each sample was preincubated for 10 min at 40°C followed by 20 min sampling with the SPME fibre in the head-space over the textile pieces. The sample was subsequently injected onto the column (injector tempera-ture=250°C). Column flow = 2 ml Helium/min. Column oven temperature gradient: 0 min = 40°C, 2 min = 40°C, 22 min = 240°C, 32 min = 240°C. The butyric acid was detected by FID detection and the amount of butyric acid was calculated based on a butyric acid standard curve.<br>
The Risk Performance Odour, R, of a lipase variant is the ratio between the amount of re¬leased butyric acid from the lipase variant washed swatch and the amount of released butyric acid from a swatch washed with the lipase of SEQ ID NO: 2 with the substitutions T231R + N233R (reference enzyme), after both values have been corrected for the amount of released butyric acid from a non-lipase washed swatch. The risk (R) of the variants is calculated in ac¬cordance with the below formula:<br>
Odour =measured in micro g buturic acid developed at 1 mg enzyme protein /1 cor¬rected for blank<br><br><br><br>
A variant is considered to exhibit reduced odor compared to the reference, if the R factor is lower than 1.<br>
Example 4<br>
Activity (LU) relative to absorbance at 280nm<br>
The activity of a lipase relative to the absorbance at 280 nm is determined by the fol¬lowing assay: LU/A280:<br>
The activity of the lipase is determined as described above in the section Lipase activity. The absorbance of the lipase at 280 nm is measured (A280) and the ratio LU/A280 is calcu¬lated. The relative LU/A280 is calculated as the LU/A280 of the variant divided by the LU/A280 of a reference enzyme. In the context of the present invention the reference enzyme is the li¬pase of SEQ ID N0:2 with the substitutions T231R + N233R.<br>
Example 5<br>
BR - Benefit Risk<br>
The Benefit Risk factor describing the performance compared to the reduced risk for odour smell is thus defined as:<br>
BR = RPavg ' R<br>
A variant is considered to exhibit improved wash performance and reduced odor, if the BR fac¬tor is higher than 1.<br><br><br><br>
CLAIMS<br>
1.	A variant of a parent lipase, wherein the variant comprises at least three substitutions<br>
selected from the group consisting of (using SEQ ID N0:2 for numbering):<br>
a)	at least two substitutions in Region I, and<br>
b)	at least one substitution in Region II, and<br>
c)	at least one substitution in Region III, and<br>
d)	at least one substitution in Region IV; and wherein the variant has lipase activity.<br><br>
2.	A lipase variant according to claim 1, wherein the at least two substitutions in Region I of the parent lipase comprises substitutions of the amino acids at the positions corresponding to the positions 231 and 233 (using SEQ ID N0:2 for numbering).<br>
3.	A lipase variant according to claim 2, wherein the at least two substitutions in Region I of the parent lipase comprises substitutions of an R of the amino acids at the positions corre¬sponding to the positions 231 and 233 (using SEQ ID N0:2 for numbering).<br>
4.	A lipase variant according to claim 1, wherein the variant lipase is at least 80%. 85%, 90%, 95%, 96%, 97%, 98%, 99% identical to SEQ ID N0:2.<br>
5.	A lipase variant according to claim 1, wherein the parent lipase is the lipase having the aminio acid sequence of SEQ ID N0:2.<br>
6.	A lipase variant according to claim 2, wherein the lipase comprises a further substitu¬tion in the position corresponding to position 4 and/or position 227 of SEQ ID N0:2.<br>
7.	A lipase variant according to claim 6, wherein the lipase has a subsitution correspond¬ing to X4V and X227G (using SEQ ID N0:2 for numbering).<br>
8.	A lipase variant according to claim 1, wherein the at least one substitution in Region II of the parent lipase comprises substitutions selected from the group consisting of substitutions<br><br>
in positions corresponding to the positions 202, 210, 211, 255 and 256 (using SEQ ID N0:2 for numbering).<br>
9.	A lipase variant according to claim 8, wherein the at least one substitution in the parent lipase is selected from the group consisting of X202G, X210K, X211L, X255YA/ and X256K (us¬ing SEQ ID N0:2 for numbering).<br>
10.	A lipase variant according to claim 1, wherein the at least one substitution in Region III of the parent lipase comprises substitutions selected from the group consisting of substitutions in positions corresponding to the positions 83, 86 and 90 (using SEQ ID N0:2 for numbering).<br>
11.	A lipase variant according to claim 10, wherein the at least one substitution in the par¬ent lipase is selected from the group consisting of X83T, X86\/ and X90A/R (using SEQ ID N0:2 for numbering).<br>
12.	A lipase variant according to claim 1, wherein the at least one substitution in Region IV of the parent lipase comprises substitutions selected from the group consisting of substitutions in positions corresponding to the positions 27, 58 and 60 (using SEQ ID N0:2 for numbering).<br>
13.	A lipase variant according to claim 1, wherein the at least one substitution in Region IV of the parent lipase comprises substitutions selected from the group consisting of X27R, X58N/A/G/P/T and X60SA//G/N/R/K/A/L (using SEQ ID N0:2 for numbering).<br>
14.	A lipase variant according to claim 1, wherein the lipase variant further comprises at least one substitution in the parent lipase is selected from the group consisting of substitutions in positions corresponding to position 81, 147, 150, 227 and 249 (using SEQ ID N0:2 for num¬bering).<br>
15.	A lipase variant according to claim 1, wherein the lipase variant further comprises at least one substitution selected from the group consisting of X81Q/E, X147MA', X150G and X249R/I/L (using SEQ ID N0:2 for numbering).<br>
16.	A lipase variant of claim 1, wherein the lipase comprises a substitution selected from<br><br>
the following group of substitutions: a)T231R + N233R + l255Y b)l202G + T231R + N233R<br>
c)	186V + L227G + T231R + N233R + P256K<br>
d)	Q4V + S58N + V60S + T231R + N233R<br>
e)	S58N + V60S + I90R + T231R + N233R<br>
f)	I90A + T231R + N233R + 1255V<br>
g)	S58N + V60S + 186V + A150G + L227G + T231R + N233R + P256K<br>
h) S58N + V60S + L147M + F211L + T231R + N233R<br>
i) Q4V + S58A + V60S + S83T + 186V + A150G + E210K + L227G + T231R + N233R +<br>
P256K<br>
j) S58N + V60S + 186V + A150G + L227G + T231R + N233R + P256K<br>
17.	A DNA sequence encoding the lipase variant of claims 1-16.<br>
18.	An expression vector harbouring the DNA sequence of claim 17.<br>
19.	A transformed host cell containing the DNA sequence of claim 17.<br>
20.	A method of producing a lipase variant, which method comprises culturing the trans¬<br>
formed host cell of claim 19 under conditions conducive for the production of the lipase variant<br>
and recovering the lipase variant from the resulting broth.<br>
21.	A variant of SEQ ID NO: 2 comprising at least one of the mutations Q4V,<br>
S58N/A/G/P/T, I90R or Q249I/L.<br><br><br><br><br><br><br><br><br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMjAtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   AMENDED  PAGES OF SPECIFICATION  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgQU1FTkRFRCBDTEFJTVMgIDIwLTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   AMENDED CLAIMS  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyOS0wNC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   CORRESPONDENCE OTHERS  29-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDEwLTA5LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   CORRESPONDENCE OTHERS 10-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAyMC0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgRk9STS0xICAyMC0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   FORM-1  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgRk9STS0zICAyMC0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   FORM-3  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyMC0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   OTHER PATENT DOCUMENT  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgT1RIRVJTICAyMC0wOS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   OTHERS  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICAgUE9XRVIgT0YgQVRUT1JORVkgIDIwLTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008   POWER OF ATTORNEY  20-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBBTUVOREVEIENMQUlNUyAgMjgtMDItMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  AMENDED CLAIMS  28-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBBTUVOREVEIENMQUlNUy4gMjUtMDktMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  AMENDED CLAIMS. 25-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0UgT0YgU1BFQ0lGSUNBVElPTiAgMjgtMDItMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  AMENDED PAGE OF SPECIFICATION  28-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04uIDI1LTA5LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  AMENDED PAGES OF SPECIFICATION. 25-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMDctMDQtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  CORRESPONDENCE OTHERS 07-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDI4LTAyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  EXAMINATION REPORT REPLY RECEIVED  28-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQuIDI1LTA5LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  EXAMINATION REPORT REPLY RECEIVED. 25-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBGT1JNLTMgIDI4LTAyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  FORM-3  28-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4ICBQT1dFUiBPRiBBVFRPUk5FWSAgMDctMDQtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008  POWER OF ATTORNEY  07-04-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA4IEZPUk0tMTggIDI1LTExLTIwMDkucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2008 FORM-18  25-11-2009.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA4IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2008 pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263448-adhesive-and-marking-compositions-made-from-interpolymers-of-ethylene-alpha-olefins.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263450-polymer-blends-from-interpolymer-of-ethylene-alpha-olefin-with-improved-compatability.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263449</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3843/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jul-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VIND, JESPER</td>
											<td>HEJREBAKKEN 20, DK-3500 VAERLOSE, DENMARK</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KNOTZEL, JURGEN CARSTEN, FRANZ</td>
											<td>OSTERBROGADE 226, 2, TH., DK-2100 COPENHAGEN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BORCH, KIM</td>
											<td>VANDTAARNSVEJ 18, DK-3460 BIRKEROED</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LAMSA, MICHAEL</td>
											<td>2625 DUCHAMP STREET, DAVIS, CA 95618</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SVENDSEN, ALLAN</td>
											<td>OVERDAMSVEJ 13, DK-2970 HORSHOLM</td>
										</tr>
										<tr>
											<td>6</td>
											<td>CALLISEN, THOMAS, HONGER</td>
											<td>FORCHHAMMERSVEJ 13, DK-1920 FREDERIKSBERG C, DENMARK</td>
										</tr>
										<tr>
											<td>7</td>
											<td>YAVER, DEBBIE</td>
											<td>5632 HOAG PLACE, DAVIS, CA 95618</td>
										</tr>
										<tr>
											<td>8</td>
											<td>BJORNVAD, MADS, ESKELUND</td>
											<td>BREDEVEJ 21, DK-2830 VIRUM</td>
										</tr>
										<tr>
											<td>9</td>
											<td>HANSEN, PETER, KAMP</td>
											<td>STORE STENSAGER 22, DK-4320 LEJRE</td>
										</tr>
										<tr>
											<td>10</td>
											<td>GE, HAIYAN</td>
											<td>1801 RAPHAEL PLACE, DAVIS, CA 95618</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/43</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US07/60841</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-01-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/854, 891</td>
									<td>2006-10-27</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/761, 109</td>
									<td>2006-01-23</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263449-lipase-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:46:52 GMT -->
</html>
