<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263563-variant-neutral-metalloproteases-having-improved-thermostability-wash-performance-or-autolytic-stability by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:12:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263563:&quot;VARIANT NEUTRAL METALLOPROTEASES HAVING IMPROVED THERMOSTABILITY, WASH PERFORMANCE OR AUTOLYTIC STABILITY&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;VARIANT NEUTRAL METALLOPROTEASES HAVING IMPROVED THERMOSTABILITY, WASH PERFORMANCE OR AUTOLYTIC STABILITY&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides methods and compositions comprising at least one neutral metalloprotease enzyme that has improved storage stability. In some embodiments, the neutral metalloprotease finds use in cleaning and other applications. In some particularly preferred embodiments, the present invention provides methods and compositions comprising neutral metalloprotease(s) obtained from Bacillus sp. In some more particularly preferred embodiments, the neutral metalloprotease is obtained from B. amyloliquefaciens. In still further preferred embodiments, the neutral metalloprotease is a variant of the B. amyloliquefaciens neutral metalloprotease. In yet additional embodiments, the neutral metalloprotease is a homolog of the the B. amyloliquefaciens neutral metalloprotease. The present invention finds particular use in applications including, but not limited to cleaning, bleaching and disinfecting.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>USE AND PRODUCTION OF STORAGE-STABLE NEUTRAL METALLOPROTEASE<br>
The present application claims priority to pending U.S. Provisional Patent Application Serial No. 60/726,448, filed October 12,2005.<br>
FIELD OF THE INVENTION<br>
The present invention provides methods and compositions comprising at least one neutral metalloprotease enzyme that has improved storage stability. In some embodiments, the neutral metalloprotease finds use in cleaning and other applications. In some particularly preferred embodiments, the present invention provides methods and compositions comprising neutral metalloprotease(s) obtained from Bacillus sp. In some more particularly preferred embodiments, the neutral metalloprotease is obtained from B. amyloliquefaci0ens. In still further preferred embodiments, the neutral metalloprotease is a variant of the B. amyloliquefaaeiisneulral metalloprotease. In yet additional embodiments, the neutral metalloprotease is a homolog of the the B. omyloliquefociens neutral metalloprotease. The present invention finds particular use in applications including, but not limited to cleaning, bleaching and disinfecting.<br>
BACKGROUND OF THE INVENTION<br>
Detergent and other cleaning compositions typically include a complex combination of active ingredients. For example, most cleaning products include a surfactant system, enzymes for cleaning, bleaching agents, builders, suds suppressors, soil-suspending agents, soil-release agents, optical brighteners, softening agents, dispersants, dye transfer inhibition<br>
compounds, abrasives, bactericides, and perfumes. Despite the complexity of current detergents, there are many stains that are difficult to completely remove. Furthermore, there is often residue build-up, which results in discoloration (e.g., yellowing) and diminished aesthetics due to incomplete cleaning. These problems are compounded by the increased use of low (e.g., cold water) wash temperatures and shorter washing cycles. Moreover, many stains are composed of complex mixtures of fibrous material, mainly incorporating carbohydrates and carbohydrate derivatives, fiber, and cell wall components (e.g., plajit material, wood, mud/clay based soil, and fruit). These stains present difficult challenges to the formulation and use of cleaning compositions.<br>
In addition, colored garments tend to wear and show appearance losses. A portion of this color loss is due to abrasion in the laundering process, particularly in automated washing and drying machines. Moreover, tensile strength loss of fabric appears to be an unavoidable result of mechanical and chemical action due to use, wearing, and/or washing and drying. Thus, a means to efficiently and effectively wash colored garments so that these appearance losses are minimized is needed.<br>
In sum, despite improvements in the capabilities of cleaning compositions, there remains a need in the art for detergents that remove stains, maintain fabric color and appearance, and prevent dye transfer. In addition, there remains a need for detergent and/or fabric care compositions that provide and/or restore tensile strength, as well as provide anti-wrinkle, anti-bobbling, and/or anti-shrinkage properties to fabrics, as well as provide static control, fabric softness, maintain the desired color appearance, and fabric anti-wear properties and benefits. In particular, there remains a need for the inclusion of compositions that are capable of removing the colored components of stains, which often remain attached to the fabric being laundered. In addition, there remains a need for improved methods and compositions suitable for textile bleaching.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides methods and compositions comprising at least one neutral metalloprotease enzyme that has improved storage stability. In some embodiments, the neutral metalloprotease finds use in cleaning and other applications. In some particularly preferred embodiments, the present invention provides methods and compositions comprising<br>
neutral metalloprotease(s) obtained from Bacillus sp. In some more particularly preferred embodiments, the neutral metalloprotease is obtained from B. amyloliquefaciens. In still further preferred embodiments, the neutral metalloprotease is a variant of the B. amyloliquefaciens neutral metalloprotease. In yet additional embodiments, the neutral metalloprotease is a homolog of the the B. amyloliquefaciens neutral metalloprotease. The present invention finds particular use in applications including, but not limited to cleaning, bleaching and disinfecting.<br>
The present invention provides novel neutral metalloproteases, novel genetic material encoding the neutral metalloprotease enzymes, and neutral metalloprotease proteins obtained from Bacillus sp., in particular, B. amyloliquefaciens, and variant proteins developed therefrom. In particular, the present invention provides neutral metalloprotease compositions obtained from Bacillus sp., particularly B. amyloliquefaciens, DNA encoding the protease, vectors comprising the DNA encoding the neutral metalloprotease, host cells transformed with the vector DNA, and an enzyme produced by the host cells. The present invention also provides cleaning compositions (e.g., detergent compositions), animal feed compositions, and textile and leather processing compositions comprising neutral metalloprotease(s) obtained from a Bacillus species, in particular, B. amyloliquefaciens. In alternative embodiments, the present invention provides mutant (i.e., variant) neutral metalloproteases derived from the wild-type neutral metalloproteases described herein. These mutant neutral metalloproteases also find use in numerous applications.<br>
The present invention provides isolated neutral metalloproteases obtained from a Bacillus species, in particular, B. amyloliquefaciens. In further embodiments, the neutral metalloprotease comprises the amino acid sequence set forth in SEQ ID NOS:3,4 or 18: In additional embodiments, the present invention provides isolated neutral metalloproteases comprising at least 45% amino acid identity with the neutral metalloprotease comprising SEQ ID NOS:3,4 or 18. In some embodiments, the isolated neutral metalloproteases comprise at least 50% identity, preferably at least 55%, more preferably at least 60%, yet more preferably at least 65%, even more preferably at least 70%, more preferably at least 75%, still more preferably at least 80%, more preferably 85%, yet more preferably 90%, even more preferably at least 95%, and most preferably 99% identity with the neutral metalloprotease comprising SEQ ID NO:3,4 or 18.<br>
The present invention also provides isolated neutral metalloproteases having immunological cross-reactivity with the metalloprotease obtained from B. amyloliquefaciens, as well as compositions comprising these neutral metalloproteases. In alternative embodiments, the neutral metalloproteases have immunological cross-reactivity with neutral metalloproteases comprising the amino acid sequence set forth in SEQ ID NOS:3,4 or 18. In still further embodiments, the neutral metalloproteases have cross-reactivity with fragments. (i.e., portions) of the neutral metalloprotease off. amyloliquefaciens, and/or neutral me'talloprotease comprising the amino acid sequence set forth in SEQ ID NOS:3,4 or 18. Indeed, it is intended that the present invention encompass fragments (e.g., epitopes) of the B. amyloliquefaciens metalloprotease that stimulate an immune response in animals (including, but not limited to humans) and/or are recognized by antibodies of any class. The present invention further encompasses epitopes on metalloproteases that are cross-reactive with B. amyloliquefaciens metalloprotease epitopes. In some embodiments, the metalloprotease epitopes are recognized by antibodies, but do not stimulate an immune response in animals (including, but not limited to humans), while in other embodiments, the metalloprotease epitopes stimulate an immune response in at least one animal species (including, but not limited to humans) and are recognized by antibodies of any class. The present invention also provides means and compositions for identifying and assessing cross-reactive epitopes.<br>
In some embodiments, the present invention provides the amino acid sequences set forth in SEQ ID NOS:3,4 or 18. In alternative embodiments, the sequence comprises substitutions at least one amino acid position in SEQ ID NOS:3, 4 or 18. In some particularly preferred alternative embodiments, the sequence comprises substitutions at least one amino acid position in SEQ ID NO: 18. In some preferred embodiments, the present invention provides neutral metalloprotease variants having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a B. amyloliquefaciens neutral metalloprotease comprising the amino acid sequence set forth in SEQ ID NOS:3,4 or 18. In some additional preferred embodiments, the .present invention provides neutral metalloprotease variants having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a B. amyloliquefaciens neutral metalloprotease comprising the amino acid sequence set forth in SEQ ID NO: 18. In alternative embodiments, the present invention provides neutral metalloprotease variants having an amino acid sequence comprising at least one substitution<br>
of an amino acid made at a position equivalent to a position in a B. amyloliquefaciens neutral metalloprotease comprising at least a portion of SEQ ID NOS:3,4 or 18. In some alternative preferred embodiments, the neutral metalloproteases comprise multiple mutations in at least a portion of SEQ ID NOS:3,4 or 18. In some alternative preferred embodiments, the neutral metalloproteases comprise multiple mutations in at least a portion of SEQ ID NO: 18.<br>
In yet additional embodiments, the present invention provides the amino acid sequence set forth in SEQ ID NO: 18. In alternative embodiments, the sequence comprises substitutions at least one amino acid position in SEQ ID NO: 18.   In some preferred embodiments, the present invention provides neutral metalloprotease variants having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a B. amyloliquefaciens neutral metalloprotease comprising the amino acid sequence set forth in SEQ ID NO: 18. In alternative embodiments, the present invention provides neutral metalloprotease variants having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a B. amyloliquefaciens neutral metalloprotease comprising at least a portion of SEQ ID NO:18. In some alternative preferred embodiments, the neutral metalloproteases comprise multiple mutations in at least a portion of SEQ ID NO:18.<br>
In some particularly preferred embodiments, these variants have improved performance as compared to wild-type B. amyloliquefaciens neutral metalloprotease. The present invention also provides neutral metalloprotease variants having at least one improved property as compared to the wild-type neutral metalloprotease. In some additional particularly preferred embodiments, these variants have improved stability as compared to wild-type B. amyloliquefaciens neutral metalloprotease. In some further preferred embodiments, these variants have improved thermostability as compared to wild-type B. amyloliquefaciens neutral metalloprotease. In yet additional preferred embodiments, these variants have improved performance under lower or higher pH conditions, as compared to wild-type B. amyloliquefaciens neutral metalloprotease.<br>
The present invention also provides neutral metalloproteases comprising at least a portion of the amino acid sequence set forth in SEQ ID NOS:3,4 or 18. In some embodiments, the nucleotide sequences encoding these neutral metalloproteases comprise a nucleotide sequence selected from SEQ ID NOS:1,2,12, and/or 13. In some embodiments, the neutral metalloproteases are variants having amino acid sequences that are similar to that<br>
set forth in SEQ ID NOS:3,4 or 18. In yet additional embodiments, the neutral metalloproteases are variants and/or homologs. In still further embodiments, the neutral metalloproteases are those set forth in any of Figures 3 through 5. In other embodiments, the neutral metalloproteases are variants of those set forth in Figure 3,4 and/or 5.<br>
The present invention also provides expression vectors comprising a polynucleotide sequence encoding at least a portion of the neutral metailoprotease set forth in SEQ ID NOS:3, 4 or-18. The present invention further provides expression vectors comprising a polynucleotide sequences that encode at least one neutral metailoprotease variant having amino acid sequence(s) comprising at least one substitution of an amino acid made at a position equivalent to a position in a Bacillus neutral metailoprotease comprising the amino acid sequence set forth in SEQ ID NOS:3,4 or 18. In further embodiments, the present invention provides host cells comprising these expression vectors. In some particularly preferred embodiments, the host cells are selected from the group consisting of Bacillus sp. The present invention also provides the neutral metalloproteases produced by the host cells.<br>
The present invention also provides compositions comprising at least a portion of an isolated neutral metailoprotease of obtained from a Bacillus sp., particularly, B. amyloliquefaciens, wherein at least a portion of the neutral metailoprotease is encoded by a polynucleotide sequence selected from SEQ ID NOS:1,2,12 and/or 13. In further embodiments, the present invention provides host cells comprising these expression vectors. In some particularly preferred embodiments, the host cells are Bacillus sp. The present invention also provides the neutral metalloproteases produced by the host cells.<br>
The present invention also provides variant neutral metalloproteases, wherein the neutral metalloproteases comprise at least one substitution corresponding to the amino acid positions in SEQ ID NO:3 and/or SEQ ID NO: 18, and wherein variant metalloproteases have better performance in at least one property, as compared to wild-type B. amyloliquefaciens metailoprotease. In some particularly preferred embodiments, the present invention also provides variant neutral metalloproteases, wherein the neutral metalloproteases comprise at least one substitution corresponding to the amino acid positions in SEQ ID NO: 18, and wherein variant metalloproteases have better performance in at least one property, as compared to wild-type B. amyloliquefaciens metailoprotease.<br>
The present invention also provides variant amino acids, wherein the variants comprise at least one substitution of an amino acid made at a position equivalent to a position<br>
in a neutral metalloprotease comprising the amino acid set forth in SEQ ID NO: 18, wherein the position(s) is or are selected from positions 1,3,4, 5,6,11,12,13,14,16,21, 23,24,25, 31,32, 33, 35, 36, 38,44,45,46,47,48,49, 50, 51, 54, 55,58,59, 60,61,62,63, 65, 66, 69, 70,76, 85, 86, 87, 88, 90,91,92, 96, 97, 98,99,100,102, 109,110, 111, 112,113,115,117, 119,127,128,129,130,132,135,136,137,138,139,140,146,148,151,152,153,154, 155,157,158,159,161,162,169,173,178,179,180,181,183,184,186,190,191,192, 196,198,199, 200,202,203,204, 205,210,211,214, 215,216,217, 218,219,220,221, 222,223, 224,228,229,237,239, 240,243,244, 245,248,252,253, 260,261,263,264, 265,267, 269,270,273,277,280, 282,283, 284,285,286,288,289, 290,292,293,296, 297, and 299.<br>
The present invention also provides isolated neutral metalloprotease variants having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a neutral metalloprotease comprising the amino acid sequence set forth in SEQ ID NO: 18. In some embodiments, the isolated neutral metalloprotease variants have substitutions that are made at positions equivalent to positions 1,3,4,5, 6,11,12,13,14,16,21,23,24,25, 31,32, 33,35, 36, 38,44,45,46,47,48,49, 50,51,54, 55,58,59,60, 61, 62,63, 65,66,69,70,76, 85,86, 87, 88,90,91,92, 96,97, 98, 99,100,102,109,110, 111, 112,113,115,117,119,127,128,129,130,132,135,136,137, 138,139,140,146,148,151,152,153,154,155,157,158,159,161,162,169,173,178, 179,180, 181,183,184,186,190,191,192,196,198,199,200,202, 203,204,205, 210, 211,214, 215,216,217,218, 219,220,221,222,223, 224,228,229,237,239,240,243, 244, 245, 248,252,253,260, 261,263,264,265,267, 269,270,273,277,280,282, 283, 284,285, 286,288,289,290,292,293,296,297, and 299 of a neutral metalloprotease comprising an amino acid sequence set forth in SEQ ID NO: 18.<br>
In additional embodiments, the isolated neutral metalloprotease variant comprises at least one mutation selected from T004C, T004E, T004H, T004I, T004K, T004L, T004M, T004N, T004P, T004R, T004S, T004V, T004W, T004Y, G012D, G012E, G012I, G012K, G012L, G012M, G012Q, G012R, G012T, G012V, G012W, K013A, K013C, K013D, K013E, K013F, K013G, K013H, K013I, K013L, K013M, K013N, K013Q, K013S, K013T, K013V, K013Y,T014F, T014G, T014H, T014I, T014K.T014L, T014M, T014P, T014Q, T014R, T014S, T014V, T014W, T014Y, S023A, S023D, S023F, S023G, S023I, S023K, S023L, S023M, S023N, S023P, S023Q, S023R, S023S, S023T, S023V, S023W, S023Y, G024A,<br>
G024D, G024F, G024G, G024H, G024I, G024K, G024L, G024M, G024N, G024P, G024R, G024S, G024T, G024V, G024W, G024Y, K033H, Q045C, Q045D, Q045E, Q045F, Q045H, Q045I, Q045K, Q045L, Q045M, Q045N, Q045P, Q045R, Q045T, Q045W, N046A, N046C, N046E, N046F, N046G, N046H, N046I, N046K, N046L, N046M, N046P, N046Q, N046R, N046S, N046T, N046V, N046W, N046Y, R047E, R047K, R047L, R047M, R47Q, R047S, R047T, Y049A, Y049C, Y049D, Y049E, Y049F, Y049H, Y049I, Y049K, Y049L, Y049N, Y049R, Y049S, Y049T, Y049V, Y049W, N050D, N050F, N050G, N050H, N050I, N050K, N050L, N050M, N050P, N050Q, N050R, N050W, N050Y, T054C, T054D, T054E, T054F, T054G, T054H, T054IT054K, T054L, T054M, T054N, T054P, T054Q, T054R, T054S, T054V, T054W, T054Y, S058D, S058H, S058I, S058L, S058N, S058P, S058Q, T059A, T059C, T059E, T059G, T059H, T059I, T059K, T059L T059M, T059N, T059P, T059Q, T059R, T059S, T059V.T059W, T060D, T060F, T060I, T060K, T060L, T060N, T060Q, T060R, T060V, T060W, T060Y, T065C, T065E.T065F, T065H, T065I, T065K, T065L, T065M, T065P, T065Q, T065R, T065V, T065Y, S066C, S066D, S066E, S066F, S066H, S066I, S066K, S066L,S066N, S066P, S066Q, S066R, S066T, S066V, S066W, S066Y, Q087A, Q087D, Q087E, Q087H..Q087I, Q087K, Q087L, Q087M, Q087N, Q087R, Q087S, Q087T, Q087V, Q087W, N090C, N090D, N090E, N090F, N090G, N090H, N090K, N090L, N090R, N090T, N096G, N096H, N096K, N096R, K097H, K097Q, K097W, KIOOA, KIOOD, K100E, K100F, K100H, K100N, K100P, KlOOQ, K100R, K100S, K100V, K100Y, R110A, R110C,R1]OE,R110H,R110K,R110L,R110M,R110N,R110Q,R110S,R110Y,D119E, D119H, D119I, D119L, D119Q, D119R, D119S, Dl 19T, D119V, Dl 19W, G128C, G128F, G128H, G128K, G128L, G128M, G128N, G128Q, G128R, G128W, G128Y, S129A, S129C, S129D, S129F, S129G, S129H, S129I, S129K, S129L, S129M, S129Q, S129R, S129T, S129V, S129W, S129Y, F130I, F130K, F130L, F130M, F130Q, F130R, F130T, F130V, F130Y, S135P, G136I, G136L, G136P, G136V, G136W, G136Y, S137A, M138I, M138K, M138L, M138Q, M138V, D139A, D139C, D139E, D139G, D139H, D139I, D139K, D139L, D139M, D139P, D139R, D139S, D139V, D139W, D139Y, V140C, Q151I, E152A, E152C, E152D, E152F, E152G, E152H, E152L, E152M, E152N, E152R, E152S, E152W, N155D, N155K, N155Q, N155R, D178A, D178C, D178G, D178H, D178K, D178L, D178M, D178N, D178P, D178Q, D178R, D178S, D178T, D178V, D178W, D178Y, T179A, T179F, T179H, T179I, T179K, T179L, T179M, T179N, T179P, T179Q, T179R, T179S, T179V, T179W, T179Y, E186A, E186C, E186D, E186G, E186H, E186K, E186L, E186M, E186N, E186P,<br>
E186Q, E186R, E186S, E186T, E186V, E186W, E186Y, V190H, V190I, V190K, V190L, V190Q, V190R, S191F, S191G, S191H, S191I, S191K, S191L, S191N, S191Q, S191R, S191W, L198M, L198V, S199C, S199D, S199E, S199F, S199I, S199K, S199L, S199N, S199Q, S199R, S199V, Y204H, Y204T, G205F, G205H, G205L, G205M, G205N, G205R, G205S, G205Y, K211A, K211C, K211D, K211G, K211M, K211N, K211Q, K211R, K211S, K211T, K211V, K214A, K214C, K214E, K214I, K214L, K214M, K214N, K214Q, K214R, K214S, K214V, L216A, L216C, L216F, L216H, L216Q, L216R, L216S, L216Y, N218K, N218P, T219D, D220A.D220E, D220H.D220K, D220N, D220P, A221D, A221E, A221F, A221I, A221K, A221L, A221M, A221N, A221S, A221V, A221Y, G222C, G222H, G222N, G222R, Y224F, Y224H, Y224N, Y224R, T243C, T243G, T243H, T243IJ T243K, T243L, T243Q, T243R, T243W, T243Y, K244A, K244C, K244D, K244E, K244F, K244G, K244L, K244M, K244N, K244Q, K244S, K244T, K244V, K244W, K244Y, V260A, V26QD, V260E, V260G, V260H, V260I, V260K, V260L.V260M, V260P, V260Q, V260R V260S, V260T, V260W, V260Y, Y261C, Y261F, Y2611, Y261L, T263E, T263F, T263H, T263I, T263L, T263M, T263Q, T263V, T263W, T263Y, S265A, S265C, S265D, S265E, S265K, S265N, S265P, S265Q, S265R.S265T, S265V, S265W, K269E, K269F, K269G, K269H, K2691, K269L, K269M, K269N, K269P, K269Q, K269S, K269T, K269V, K269W, K269Y, A273C, A273D, A273H, A273I, A273K, A273L, A273N, A273Q, A273R, A273Y, R280A, R280C, R280D, R280E, R280F, R280G, R280H, R280K, R280L, R280M, R280S, R280T, R28QV, R280W, R280Y, L282F, L282G, L282H, L282I, L282K, L282M, L282N, L282Q, L282R, L282V, L282Y, S285A, S285C, S285D, S285E, S285K, S285P, S285Q, S285R, S285W, Q286A, Q286D, Q286E, Q286K, Q286P, Q286R, A289C, A289D, A289E, A289K, A289L, A289R, A293C, A293R, N296C, N296D, N296E, N296K, N296R, N296V, A297C, A297K, A297N, A297Q, A297R, and G299N.<br>
In still further embodiments the present invention provides isolated variant neutral metalloproteases, wherein the metalloproteases comprise multiple mutations selected from S023W/G024M, T004V/S023W/G024W, S023W/G024Y/A288V, T004L7S023W/G024Y, N046Q/NQ50F/T054L, N050Y/T059RyS129Q, S023W/G024W, A273H/S285P/E292G, S023Y/G024Y, S023Y/G024W, T004S/S023Y/G024W, N046Q/T054K, S023W/G024Y, T004V/S023 W, T059K/S066N, N046Q/N050W/T054H/T153A, T004V/S023W/G024Y, L282M/Q286P/A289R, N046Q/R047KyN050Y/T054K, L044Q/T263W/S285R, T004L/S023W/G024W, R047K/N050F/T054K, A273H/S285R, N050Y/T059K/S066Q,<br>
T054K/Q192K, N046Q/N050W, L282M/Q286K, T059K/S066Q, T004S/S023W, L282WQ286R;A289R/KOi IN, L282M/A289R, N046Q/N050W/T054H, T059K/S129Q, T004S/S023N/G024Y/F21OL, T004V/S023W/G024M/A289V,<br>
L282M/Q286K/A289R/S132T, N050W/T054H, L282M/Q286R, L282F/Q286K/A289R, T059R/S066Q, R047K/N050W/T054H, S265P/L282M/Q286K/A289R, L282M/Q286R/T229S, L282F/Q286K, T263W/S285R, S265P/L282M/Q286K, T263H/A273H/S285R,T059R/S129V, S032TAT263H/A273H/S285R, T059R/S066Q/S129Q, T004S/G024W, T004V/S023W/G024M, T059K/S066Q/S129Q, L282M/Q286K/A289R^253V, T004V/S023Y/G024W, T059R/S066N/S129Q, N050F/T054L, T004S/S023N/G024W, T059R/S066N, T059R/S066N/S129V, Q286R/A289R, M046Q/R047K/N050F/T054K, S265P/L282M;Q286P;A289R, S265P/L282M/Q286R/A289R Q062K/S066Q/S129I, S023N/G024W, N046Q/R047KyN050W/T054H, R047K/T054K, T004L/G024W, T014M/T059R/S129V, T059R/S066Q/N092S/S129I, R047K^050W/T054K, T004V/G024W, N047K/N050F/T054K, S265P/L282F/Q286K/N061Y, L282F/Q286K7E159V, T004V/S023Y/G024M, S265P/L282F/A289Rm)65S, T059K/F063IVS066N/S129V, T004L/S023W, N050F/T054H, T059R/S066Q/S129V, V190I/D220F7S265W/L282F, T004S/S023Y/G024M, T004L/S023N/G024Y, T059K/S066N/S129I, T059R/S066N/S129I, L282MyQ286R/A289R^P162S, N046Q/N050F/T179N, X059K/Y082C/S229V, T059K/S129I.N050Y/T054K, T059K/S066QA^102A/S129Q, T059R/S066Q/S129I, T059W;S066N/SJ29V/S290R, T059R/SJ29I, T059K/S066Q/S129I, T059K/S066Q/S129V, S265PyL282M/Q286R/A289R/T202S/K203N, T004V/S023N/G024W, S265P/Q286K, S265P/L282F/A289R, D220P/S265W, L055F/T059W/S129V, T059R/S129Q/S191R, N050W/T054K, T004S/S023W/G024M, R047K^050F/T054H, T059K/S066N/K088E-T059K/S066Q/S129W291L, L282M/Q286R/A289R, T059R/S066N/F085S/S129I, L282F/Q286P/A289R, L282F/Q286R/A289R, G099D/S265P/L282F/Q286K/A289R, N046Q/N050F, N050Y/T059W/S066N/S129V, T009I/D220P/S265N, V190F/D220P/S265W, N157Y/T263WM273H/S285R, T263W/A273H/S285R, T263W/S285W, T004V/S023Y, N046Q/R047K/N050W, N050W/T054L, N200Y/S265P/L282F/Q2B6P/A289R, T059R/S066Q/P264Q, T004V/G024Y, T004L/G024Y, N050Y/S19II, N050Y/T054L, T004US023W/G024Y/N155K, FJ691/L282F/Q286RyA289R,L282MyQ286K/A289R,F330L/M138IVE152W/D183N,<br>
N046Q/R047K/N050Y/T054H, T004V/G024M, N050Y/T059W/S066Q/S129V, S023N/G024Y, T054H/P162Q, T004S/S023 W/G024Y, N050Y/T054H, L282F/Q286RyA289R/Fl69I, R047K/N050W, V190F/D220P, L282M/F173Y, T004L/S023Y, N050W/A288D, V1901/D220P/S265Q, S265P/L282F/Q286P/A289R, S265P/L282F/Q286R/A2S9R, N046Q/N050Y/T054K, T059W/S066Q, T263W/A273H/S285WT263W/A2-73H/S285P, S023Y/G024M, T0041VS023N/G024W, T004V/S023N/G024Y, T059W/S066N/S129Q, T004S/S023Y, T004S/S023N/G024M, T059W/S066NyA070T, T059W/S066Q/S129Q, T263W/A273H, A273H/285P, N046Q/R047KyN050Y/T054L, N046Q/R047K/N050Y, R047K/N050Y, T263H/S285W, R047K/N050F, N046Q/R047K/N050F/T054H, S023N/G024M, T004S/G024Y, R047K/N050Y/T054H, T059W/S066N/S1291, R047K/T054L, T004S/S023 W/G024W, M138IJE152F/T146S, D220P/S265N, T004S/G024M, T004V/S023N, N046Q/N050F/T054K, N046Q/N050Y/T054H, Q062H/S066Q/S129Q, T059W/S129Q, T059W/S129V, N050F/T054K, R047KW050F/T054L, V1901/D220P/S265W, N112LT263H/A273H/S285R, T059W/S066N/S129V, T059W/S066Q/S129I, T059W/SJ29I, T263W/S285P, V190LD220P, A289V/T263H/A273H, T263H/A273H/S285P, N90S/A273H/S285P, R047K/N050Y/T054L, T004S/S023N, T059R/S129Q, N046Q/R047K/T054H, T059W/S066Q/S129V, E152W/T179P, N050Y/S066Q/S129V, T202S/T263W/A273H, T263W/A273H/S285P, M138L/E152W/T179P, N046Q/R047K, N046Q/T054H/F176L, T004UG024M, T004S/L282M, T263H/A273H, T263H/A273H/S285W, T004L/S023Y/G024M, L282F/Q286P, T004V/S023Y/G024Y, V190F/S265W, MI38L/E152F, V190F/D220E/S265W, N046Q/N050F/T054H, N157Y/S285W, T004F/S023Y/G024M, T004V/S023N/G024M, L1981/D220E/S265Q, N046Q/N050Y/T054K7A154T, S016UD220E/S265W, D220E/S265W, D220E/A237S/S265W, S066Q/S129Q, V190F/D220E/S265Q/T267I, L282M&gt;F173Y/r219S, E152F/T179P, V190I/S265W, M138L/S066Q, M138LE152W,<br>
T059W/S066Q/A070T/S129I, V190F/D220E/S265N, V190F/S265N, N046Q/N050Y, and M138L/E152F/T179P.<br>
In yet further embodiments, the present invention provides isolated variant neutral metalloproteases, wherein the metaHoproteases comprise multiple mutations selected from V190I/D220P, V1901/D220P7S265Q, V190L/D220E, V190I/D220E/S265Q, V190I/D220E/S265W/L282F, V190L/D220E/S265Q, V190I/D220E/S265W,<br>
V190L/D220E/S265N, T059R/S066Q/S129I, V190I/D220E/S265N, V190L/D220E/S265W, V1901/D220E, T059W/S066N/S129V, T059K/S066Q/S129V, T059K/Y082C/S129V, T059R/S066N/S129I, S066Q/S129V, T059R/S066Q/S129V, T059R/S129I, N050Y/T059W/S066N/S129V, D220P/S265N, S066Q/S129I, T059W/S066Q/S129V, T059K/S066Q/S129I, T059R/S129V, N050Y/S066Q/S129V, T059W/S066Q/S129I, N050Y/T059W/S066Q/S129V, T059K/S129I, D220P/S265W, F130L/M138L/T179P, S066N/S129I, T059R/S066N/S129V, F130I/M138L/T179P, T059R/S066Q/N092S/S129I, S066N/S129V, D220E/S265Q, F130L/M138UE152W/T179P, T059W/S129V, S265P/L282M/Q286R/A289R, S265P/L282F/Q286R/A289R, T059W/S066N/S129I, V1901/D220P/S265W, F130L/E152W/T179P, F130L/M138L/E152F/T179P, Q062K/S066Q/S129I, T059K/S066N/S129I, E152H/T179P, S265P/L282M/Q286K/A289R, F130UM138L/E152H/T179P, T263W/A273H/S285R, D220E/S265N, F130I/M138L/E152H/T179P, F130V/M138L/E152W/T179P, F1301/M138L/E152W/T179P, T059W/S129I, D220E/S265W, F130V/M138UT179P, F130L7E152V/T179P, T059R/S129Q, T263W/S285P, F130IM13817E152F/T179P, E152W/T179P, V190L/S265Q, F130L/E152F/T179P, L282M/Q286R/A289R/P162S, D220P/S265Q, M138L/E152F/T179P, F1301/E152H/T179P, M138L/E152W/T179P, F130L/T179P,<br>
F130L/M138iyE152W/T]79P/Q286H, F130LyM138L/E152H, T263W/A273H/S285W, S265P/Q286K, T059W/S066Q/S129Q, T263W/S285R, T059W/S066N/S129Q, T263W/S285W, T059R/S066N/S129Q, S265P/L282M/Q286R/A289R/T202S/K203N, T059W/S129Q, Q062H/S066Q/S129Q, L282M/Q286R/A289R, V190L/D220E/S265N/V291I, V190L/S265N, F130L/M138L/E152W, N050Y/T059R/S129Q, F130FT179P, T059K/S066Q/S129Q, T059K/S129Q, S265P/L282M/Q286P/A289R, S265P/L282F/Q286P/A289R, T263W/A273H/S285P, S265P/L282M/Q286K, S016L/D220E/S265W, S066Q/S129Q, S265P/L282M/Q286P, L282F/Q286R/A289R, F130V/E152W/T179P, L044Q/T263W/S285R L055F/T059W/S129V, V19CJL/S265W, Q286R/A289R,<br>
G99D/S265P/L282F/Q286K;A289R, F130L/M13817E152F, T059R/S066Q/S129Q, F130L/E152H, S066N/S129Q, T004S/S023N/G024M/K269N, S265P/L282M, E152F^179P, T059W/S066N/S129V/S290R, L282F/Q286K/A289R, F130UM138L, F130I/M138L/E352W, S265P/L282F, F130I/M138L/E152H, F130V/M138L/E152H, V190I/S265Q, M138UE152M, S265P/L282F/Q286P, M138L/E152H,<br>
T059K/S066N/K088E, V190I/S265W, F130L/E152W, L282M/Q286K/A289R, L282M/Q286K/A289R/I253V, T263W/A273H, V190yS265N, M138L/E152W, A273H/S285R, F1301/M138L, F130L/E152F, F130V/M138L/E152W, T059KyS066Q/V102A/S129Q, F130V/E152H/T179P, F130I/M138UE152F, F130V/M138L/E152F, M138L/E152F, L282M/Q286R, F130I/E152H, S265P/L282F/A289R/T065S, T263H/A273H/S285R, F130V/M138L, T014M/T059R/S129V, L282M/Q286R/A289R/K1 IN, A273H/S285P, L282M/Q286K/A289R/S132T, T263H/A273H/S285W, F130WE152W, S265P/L282F/Q286K/N061Y, F130I/E152W, L198I/D220E/S265Q, V190I/S265L, T263H/S285W, S265P/L282F/A289R, M138L/S066Q, F130I/E152F, N90S/A273H/S285P, S032T/T263H/A273H/S285R, L282F/Q286P/A289R, N157Y/T263W/A273H/S285R, V105A/S129V, T263H/A273H/S285P, S129Q/L282H, T059W/S066Q, F130V/E152H, S023W/G024Y, T004V/S023N, T059R/S066Q, N050W/T054L, L282M/Q286P/A289R, Al 15VA^190I7S265W, L282MyQ286K, T059R/S066N, L282F/Q286P, T004V/S023W/G024M, S265P/L282F/Q286R/L78H, L282F/Q286K, T004V/S023W/G024Y, S023W/G024M, T059R/R256S, F130V/E152F, T004V/G024W, N050W/T054K, S023Y/G024M, T004V/S023Y, T004V/S023Y/G024M, N050Y/T054H, S023W/G024W, T004V/S023Y/G024Y, T004V/S023N/G024W, F130L/M138L^152F/T179PA7291I, N050Y/T059K/S066Q, T004V/S023Y/G024W, T059KyS066N,T004V/S023N/G024Y, S023Y/G024W, N050F/T054L, R047K/T054K, S023N/G024W, L282M/A289R, S023Y/G024Y, T004V/G024M, R047K/N050F/T054K, N050F/T054K, T059K/S066Q, S023N/G024M, S023N/G024Y, T004US023N, R047K/N050W/T054H, T004I7S023W/G024Y, T004S/S023W, N046Q/N050W/T054H/A142T, T004L/S023Y, T004V/S023W, N050W/T054H, T004S/S023N, T004S/L282M, T004L/S023W, N050F/T054H, N050Y/T054L, and R047K/N050W/T054K.<br>
In yet further embodiments, the present invention provides isolated neutral metalJoproteases comprising multiple mutations selected from S066Q/S129V, S066Q/S129I, N050Y/S066Q/S129V, S066N/S129I, T059K/S066Q/S129V, S066N/S129V, F130L/E152W/T179P, S265P/L282M/Q286R/A289R, F130LVE152V/T179P, T059K/S066Q/S129I, T263W/S285P, T059K/S066N/S129I, T263W/A273H/S285P, S265P/L282F/Q286R/A289R, F130V/E152W/T179P, T263W/A273H/S285R,<br>
V190I/D220P/S265W, F130L/E152H, S066N/S129Q, S265P/L282M/Q286K7A289R, V190I/D220E, T059R/S066N/S129I, V190I/D220E/S265W, T059K/S129I, T059R/S066Q/S129I, F130I/M138UE152H/T179P, F130I/T179P, T263W/A273H/S285W, S01617D220E/S265W, S066Q/S129Q, V190I/D220E/S265Q, T059R/S066Q/S129V, D220E/S265N, V190L/D220E, D220E/S265W, V190I/D220P, V190L/D220E/S265N, L044Q/T263W/S285R, S265P/L282M/Q286P/A289R, F130L/M138L/E152H/T179P, T263W/S285R, L282M/Q286R/A289R, T263W/S285W, F130I/E152H/T179P, V190I/D220E/S265N, V190L/D220E/S265W, V190I/D220P/S265Q, T059R/S066N/S129V, V190L/D220E/S265Q, E152H/T179P, F130L/M138L/E152F/T179P, Q062H/S066Q/S129Q, T059R/S129V, V190I/D220E/S265W/L282F, V190I/S265Q, F130L/E152F/T179P, D220E/S265Q, E152W/T179P, T059K/S066Q/S129Q, F130L/M138L/T179P, F1301/M138L/E152F/T179P, F130L/M138L/E152W/T179P, N050Y/T059W/S066Q/S129V, S265P/L282M/Q286K, T059R/SJ29I, F130V/E152H/T179P, D220P/S265N, S265P/L282M/Q286P, F130I/E152H, T059R/S066Q/N092S/S129I, F130L/T179P, G99D/S265P/L282F/Q286K/A289R, T263W/A273H, V190I/S265N, D220P/S265W, F130L/E152W, F130L/M138L/E152H, S265P/L282M, V190yS265Q, F130L/E152F, T059K/S129Q, Q286R/A289R, M138L/E152W/T179P, F130I/M138L/E152H, D220P/S265Q, V190L/S265N, F130I/M138L/E152W, S265P/Q286K, V190US265Q, V190I/S265W, F130L/M138L/E152F, F130V/E152H, E152F/T179P, N050Y/T059W/S066N/S129V, T059R/S066N/S129Q, F1301/E152W, F130V/E152W, T059R/S066Q/S129Q, T263H/A273H/S285P, N90S/A273H/S285P, V190L/D220E/S265N/V291I, T059R/S129Q, A273H/S285P, F130I/M138L/E152W/T179P, F130V/M138L/E152F, N050Y/T059R/S129Q, T059W/S066Q/S129I, F130V/M138L/T179P, F130V/M138L/E152W/T179P, V190L/S265W, F130V/M138L/E152W, T059W/S066Q/S129V, V1901/S265Q, F130V/M138L/E152H, F130I/E152F, N157Y/T263W/A273H/S285R, T263H/S285W, M138UE152F/T179P, A115V/V190L/S265W, M1381JE152M, T263H/A273H/S285W, F130L/M138L/E152W, T059K/S066N/K088E, F130I/M138L/E152F, F130IM138L/T179P, T004V/S023N, T059K/S066Q/V102A/S129Q, F130IJM138L, N047K/N050F/T054K, T263H/A273H/S285R.F130L/M138L/E152W/T179P/Q286H, M13817E152H, M138L/S066Q, L282M/Q286R/A289R/P162S, L282F/Q286R/A289R, Q062K/S066Q/S129I, A273H/S285R, S265P/L282F/Q286P, S265P/L282F/Q286P/A289R,<br>
S265PyL282MyQ286R/A289R/T202S/K203N, T059W/S066N/S129I, V190I/S265L, T059W/S066N/S129V, F130I/M138L, L282M;Q2S6K/A289R/I253V, R047K/N050F/T054K, M138L/E152F, N050W/T054K, L198I/D220E/S265Q, L282F/Q286K/A289R, N050F/T054K, L282M/Q286R, M138UE152W, S265P/L282F, F130V/E152F, T059W/S066N/S129Q, F130WM138L, T263H/A273H, L282M/Q286K/A289R, N046Q/N050W/T054H/A142T, T059W/S066Q/S129Q, S265P/L282F/A289R/T065S, N050F/T054H, S129Q/L282H, L282M/Q286K/A289R/S132T, L282M/Q286R/A289R;K1 IN, T059K/S066N, R047KyN050W/T054K, T059K/S066Q, T004V/S023Y, T059W/S066N/S129V/S290R, N050Y/T059K/S066Q, and R047K/N050Y.<br>
The present invention also provides isolated polynucleotides comprising a nucleotide sequence (i) having at least 70% identity to SEQID NOS:1,2,12 and/or 13, or (ii) being capable of hybridizing to a probe derived from any of the nucleotide sequence set forth herein, including the primer sequences provided in the Examples, under conditions of intermediate to high stringency, or (Hi) being complementary to the nucleotide sequence set forth in SEQ ID NOS:1,2, 12, and/or 13. In some embodiments, the present invention provides expression vectors encoding at least one such polynucleotide. In further embodiments, the present invention provides host cells comprising these expression vectors. In some particularly preferred embodiments, the host cells are Bacillus sp. The present invention also provides the neutral metalloproteases produced by the host cells. In further embodiments, the present invention provides polynucleotides that are complementary to at least a portion of the sequence set forth in SEQ ID NOS:1, 2, 12, and/or 13.<br>
The present invention also provides methods of producing an enzyme having neutral metalloprotease activity, comprising: transforming a host cell with an expression vector comprising a polynucleotide having at least 70% sequence identity to SEQ ID NO:1,2,12 and/or 13; cultivating the transformed host cell under conditions suitable for the host cell. In some preferred embodiments, the host cell is a Bacillus species.<br>
The present invention also provides probes comprising 4 to 150 nucleotide sequence substantially identical to a corresponding fragment of SEQ ID NOS:1,2,12, and/or 13, wherein the probe is used to detect a nucleic acid sequence coding for an enzyme having metalloproteolytic activity. In some embodiments, the nucleic acid sequence is obtained from a Bacillus sp.<br>
The present invention also provides cleaning compositions comprising at least one neutral metalloprotease obtained from a Bacillus sp. In some embodiments, at least one neutral metalloprotease is obtained from B. amyloliquefaciens. In some particularly preferred embodiments, at least one neutral metalloprotease comprises the amino acid sequence set forth in SEQ ID NOS:3,4, and/or 18. In some further embodiments, the present invention provides isolated neutral metalloproteases comprising at least 45% amino acid identity with neutral metalloprotease comprising SEQ ED NOS:3,4 and/or 18. In some embodiments, the isolated neutral metalloproteases comprise at least 50% identity, preferably at least 55%, more preferably at least 60%, yet more preferably at least 65%, even more preferably at least 70%, more preferably at least 75%, still more preferably at least 80%, more preferably 85%, yet more preferably 90%, even more preferably at least 95%, and most preferably 99% identity with SEQ ID NOS:3,4, and/or 18.<br>
The present invention further provides cleaning compositions comprising at least one neutral metalloprotease, wherein at least one of the neutral metalloproteases has immunological cross-reactivity with the neutral metalloprotease obtained from a Bacillus sp. In some preferred embodiments, the neutral metalloproteases have immunological cross-reactivity with neutral metalloprotease obtained from B. amyloliquefaciens. In alternative embodiments, the neutral metalloproteases have immunological cross-reactivity with neutral metalloprotease comprising the amino acid sequence set forth in SEQ ID NOS:3,4 and/or 18. In still further embodiments, the neutral metalloproteases have cross-reactivity with fragments (i.e., portions) of &amp; Bacillus sp. neutral metalloprotease and/or the neutral metalloprotease comprising the amino acid sequence set forth in SEQ ID NOS:3,4, and/or 18.       The present invention further provides cleaning compositions comprising at least one neutral metalloprotease, wherein the neutral metalloprotease is a variant neutral metalloprotease having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a Bacillus sp. neutral metalloprotease having an amino acid sequence set forth in SEQ ID NOS:3,4 and/or 18, particularly B. amyloliquefaciens neutral metalloprotease. In some particularly preferred embodiments, the present invention also provides cleaning compositions comprising at least one neutral metalloprotease, wherein the neutral metalloprotease is a variant neutral metalloprotease having an amino acid sequence comprising at least one substitution of an amino acid made at a position equivalent to a position in a Bacillus sp. neutral<br>
metalloprotease having an amino acid sequence set forth in SEQ ID NO:18, particularly B, amyloliguefaciens neutral metalloprotease.<br>
In yet additional embodiments, the cleaning compositions contain at least one neutral metalloprotease comprising a set of mutations in SEQ ID NOS:3,4 and/or 18. In some particularly preferred embodiments, the variant neutral metalloproteases comprise at least one substitution corresponding to the amino acid positions in SEQ ID NOS:3,4, and/or 18, and wherein the variant neutral metalloproteases have better performance in at least one property, as compared to wild-type B. amyloliquefaciens neutral metalloprotease.<br>
The present invention also provides cleaning compositions comprising a cleaning effective amount of at least one metalloproteolytic enzyme, the enzyme comprising an amino acid sequence having at least 70 % sequence identity to SEQ ID NOS:3,4, and/or 18, and a suitable cleaning formulation. In some preferred embodiments, the cleaning compositions further comprise one or more additional enzymes or enzyme derivatives selected from the group consisting of proteases, amylases, Upases, mannanases, pectinases, cutinases, oxidoreductases, hemicellulases, and celluloses.<br>
The present invention also provides compositions comprising at least one neutral metalloprotease obtained from a Bacillus sp., in particular B. amyloliquefaciens, wherein the compositions further comprise at least one stabilizer. In some embodiments, the stabilizer is selected from borax, glycerol, zinc ions, calcium ions, and calcium formate. In some embodiments, the present invention provides competitive inhibitors suitable to stabilize the enzyme of the present invention to anionic surfactants. In some embodiments, at least one neutral metalloprotease is obtained from a Bacillus sp. In some particularly preferred embodiments, the at least one neutral metalloprotease is obtained from B. amyloliquefaciens. In some particularly preferred embodiments, the at least one neutral metalloprotease comprises the amino acid sequence set forth in SEQ ID NOS:3,4, and/or 18.<br>
The present invention further provides compositions comprising at least one neutral metalloprotease obtained from a Bacillus sp., wherein the neutral metalloprotease is an autolytically stable variant. In some embodiments, at least one variant neutral metalloprotease is obtained from B. amyloliquefaciens. In some particularly preferred embodiments, the at least one variant neutral metalloprotease comprises the amino acid sequence set forth in SEQ ID NOS:3,4, and/or 18.<br>
The present invention also provides cleaning compositions comprising at least 0.0001 weight percent of the neutral metalloprotease of the present invention, and optionally, an adjunct ingredient. In some embodiments, the composition comprises an adjunct ingredient. In some preferred embodiments, the composition comprises a sufficient amount of a pH modifier to provide the composition with a neat pH of from about 3 to about 5, the composition being essentially free of materials that hydrolyze at a pH of from about 3 to about 5. In some particularly preferred embodiments, the materials that hydrolyze comprise a surfactant material. In additional embodiments, the cleaning composition is a liquid composition, while in other embodiments, the cleaning composition is a solid composition and in still further embodiments, the cleaning composition is a gel. Indeed, it is not intended that the present invention be limited to any particular formulation and/or composition, as various formulations and/or compositions find use in the present invention. In further embodiments, the surfactant material comprises a sodium alkyl sulfate surfactant that comprises an ethylene oxide moiety.<br>
The present invention additionally provides cleaning compositions that in addition to at least one neutral metalloprotease of the present invention, further comprise at least one acid stable enzyme, the cleaning composition comprising a sufficient amount of a pH modifier to provide the composition with a neat pH of from about 3 to about 5, the composition being essentially free of materials that hydrolyze at a pH of from about 3 to about 5. In further embodiments, the materials that hydrolyze comprise a surfactant material. In some preferred embodiments, the cleaning composition being a liquid composition. In yet additional embodiments, the surfactant material comprises a sodium alkyl sulfate surfactant that comprises an ethylene oxide moiety. In some embodiments, the cleaning composition comprises a suitable adjunct ingredient. In some additional embodiments, the composition comprises a suitable adjunct ingredient. In some preferred embodiments, the composition comprises from about 0.001 to about 0.5 weight % of neutral metalloprotease.<br>
In some alternatively preferred embodiments, the composition comprises from about 0.01 to about 0.1 weight percent of neutral metalloprotease.<br>
The present invention also provides methods of cleaning, the comprising the steps of: a) contacting a surface and/or an article comprising a fabric with the cleaning composition comprising the neutral metalloprotease of the present invention at an appropriate concentration; and b) optionally washing and/or rinsing the surface or material. In<br>
alternative embodiments, any suitable composition provided herein finds use in these methods. In some embodiments, the fabric comprises at least one grass stain. In some particularly preferred embodiments, the cleaning compositions of the present invention find use in removing grass and other stains from fabrics.<br>
The present invention also provides animal feed comprising at least one neutral metalloprotease obtained from a Bacillus sp. In some embodiments, at least one neutral metalloprotease is obtained from B. amyloliquefaciens. In some particularly preferred embodiments, at least one neutral metalloprotease comprises the amino acid sequence set forth in SEQ ID NOS-.3,4 or 18. In some alternative particularly preferred embodiments, at least one neutral metalloprotease comprises the amino acid sequence set forth in SEQ ID NO-.18.<br>
The present invention provides an isolated polypeptide having metalloproteolytic activity, (e.g., a neutral metalloprotease) having the amino acid sequence set forth in SEQ ID NO: 18. In some embodiments, the present invention provides isolated polypeptides having approximately 40% to 98% identity with the sequence set forth in SEQ ID NO:18. In some preferred embodiments, the polypeptides have approximately 50% to 95% identity with the sequence set forth in SEQ ID NO: 18. In some additional preferred embodiments, the polypeptides have approximately 60% to 90% identity with the sequence set forth in SEQ ID NO: 18. In yet additional embodiments, the polypeptides have approximately 65% to 85% identity with the sequence set forth in SEQ ID NOS:3,4, or 18. In some particularly preferred embodiments, the polypeptides have approximately 90% to 95% identity with the sequence set forth in SEQ ID NOS:3,4, or 18.<br>
The present invention further provides isolated polynucleptides that encode neutral metalloproteases comprise an amino acid sequence comprising at least 40% amino acid sequence identity to SEQ ID NOS:3, 4, or 18. In some embodiments, the neutral metalloproteases have at least 50% amino acid sequence identity to SEQ ID NOS:3,4 and/or 18.  In some embodiments, the neutral metalloproteases have at least 60% amino acid sequence identity to SEQ ID NOS:3,4 or 18. In some embodiments, the neutral metalloproteases have at least 70% amino acid sequence identity to SEQ ID NOS:3,4 or 18. In some embodiments, the neutral metalloproteases have at least 80% amino acid sequence identity to SEQ ID NOS:3,4 or 18. In some embodiments, the neutral metalloproteases have at least 90% amino acid sequence identity to SEQ ID NOS:3,4 or 18. In some embodiments,<br>
the neutral metalloproteases have at least 95% amino acid sequence identity to SEQ ID NO:3, 4 or 18. The present invention also provides expression vectors comprising any of the polynucleotides provided above.<br>
The present invention further provides host cells transformed with the expression vectors of the present invention, such that at least one neutral metalloprotease is expressed by the host cells. In some embodiments, the host cells are bacteria, while in other embodiments, the host cells are fungi.<br>
The present invention also provides isolated polynucleotides comprising a nucleotide sequence (i) having at least 70% identity to SEQ ID NO: 1,2,12 and/or 13, or (ii) being capable of hybridizing to a probe derived from the nucleotide sequence of SEQ ID NO: 1, 2, 12, and/or 13, under conditions of medium to high stringency, or (iii) being complementary to the nucleotide sequence of SEQ ID NO:1,2,12, and/or 13. In some embodiments, the present invention provides vectors comprising such polynucleotide. In further embodiments, the present invention provides host cells transformed with such vectors.<br>
The present invention further provides methods for producing at least one enzyme having neutral metalloprotease activity, comprising: the steps of transforming a host cell with an expression vector comprising a polynucleotide comprising at least 70% sequence identity to SEQ ID NO:1,2, 12, and/or 13, cultivating the transformed host cell under conditions suitable for the host cell to produce the neutral metalloprotease; and recovering the neutral metalloprotease. In some preferred embodiments, the host cell is a Bacillus sp, while in some alternative embodiments, the host cell is B. amyloliquefaciens.<br>
The present invention also provides fragments (i.e., portions) of the DNA encoding the neutral metalloproteases provided herein. These fragments find use in obtaining partial length DNA fragments capable of being used to isolate or identify polynucleotides encoding mature neutral metalloprotease enzyme described herein from B. amyloliquefaciens , or a segment thereof having proteolytic activity. In some embodiments, portions of the DNA provided in SEQ ID NO:2 find use in obtaining homologous fragments of DNA from other species which encode a neutral metalloprotease or portion thereof having metalloproteolytic activity.<br>
The present invention further provides at least one probe comprising a     polynucleotide substantially identical to a fragment of SEQ ID NOS:1,2,6, 7, 8,9,10,11, 12,13,14,15, and/or any primer sequence set forth herein, wherein the probe is used to<br><br>
detect a nucleic acid sequence coding for an enzyme having metalloproteolytic activity, and wherein the nucleic acid sequence is obtained from a bacterial source. In some embodiments, the bacteria] source is a Bacillus sp. In some preferred embodiments, the bacterial source is B. amyloliquefaciens.<br>
The present invention further provides compositions comprising at least one of the neutral metalloprpteases provided herein. In some preferred embodiments, the compositions are cleaning compositions. In some embodiments, the present invention provides cleaning compositions comprising a cleaning effective amount of at least one neutral metalloprotease comprising an amino acid sequence having at least 40% sequence identity to SEQ ID NO: 18 at least 90% sequence identity to SEQ ID NO.18, and/or having an amino acid sequence of SEQ ID NO:18. In some embodiments, the cleaning compositions further comprise at least one suitable cleaning adjunct. In some embodiments, the neutral metalloprotease is derived from a Bacillus sp. In some preferred embodiments, the Bacillus sp., is B. amyloliquejaciens.<br>
In still further embodiments, the cleaning composition further comprises at least one additional enzymes or enzyme derivatives selected from the group consisting of proteases, amylases, Upases, mannanases, and cellulases.<br>
The present invention also provides isolated naturally occurring neutral metalloproteases comprising an amino acid sequence having at least 45% sequence identity to SEQ ID NO:18, at least 60% sequence identity to SEQ ID NO:18, at least 75% sequence identity to SEQ ID NO: 18, at least 90% sequence identity to SEQ ID NO: 18, at least 95% sequence identity to SEQ ID NO: 18, and/or having the sequence SEQ ID NO:18, the neutral metalloprotease being isolated from a Bacillus sp. In some embodiments, the neutral metalloprotease is isolated from B. amyloliquefaciens.<br>
In additional embodiments, the present invention provides engineered variants of the neutral metal] oproteases of the present invention. In some embodiments, the engineered variants are genetically modified using recombinant DNA technologies, while in other embodiments, the variants are naturally occurring. The present invention further encompasses engineered variants of homologous enzymes, as well as isolated enzyme homologs. In some embodiments, the engineered variant homologous neutral metalloproteases are genetically modified using recombinant DNA technologies, while in other embodiments, the variant homologous neutral metalloproteases are naturally occurring.<br>
The present invention also provides methods for producing neutral metalJoproteases, comprising: (a) transforming a host cell with an expression vector comprising a polynucleotide having at least 70% sequence identity to SEQ ID NO:2, at least 95% sequence identity to SEQ ID NO:2, and/or having a polynucleotide sequence of SEQ ID NO:2; (b) cultivating the transformed host cell under conditions suitable for the host cell to produce the neutral metalloprotease; and (c) recovering the neutral metalloprotease. In some embodiments, the host cell is a Bacillus species (e.g., B. subtilis, B. clausii, or B. licheniformis). In alternative embodiments, the host cell is a B. amyloliquefaciens<br>
In further embodiments, the present invention provides means to produce host cells that are capable of producing the neutral metalloproteases of the present invention in relatively large quantities. In particularly preferred embodiments, the present invention provides means to produce neutral metalloprotease with various commercial applications where degradation or synthesis of polypeptides are desired, including cleaning compositions, as well as food and/or feed components, textile processing, leather finishing, grain processing, meat processing, cleaning, preparation of protein hydrolysates, digestive aids, microbicidal compositions, bacteriostatic compositions, fungistatic compositions, personal care products (e.g., oral care, hair care, and/or skin care).<br>
The present invention also provides variant neutral metalloproteases having improved performance as compared to wild-type B. amyloliquefaciens neutral metalloprotease. In some preferred embodiments, the improved performance comprises improved thermostability, as compared to wild-type B. amyloliquefaciens neutral metalloprotease. In alternative preferred embodiments, the improved performance comprises improved performance under lower or higher pH conditions, as compared to wild-type B. amyloliquefaciens neutral metalloprotease. In additional preferred embodiments, the improved performance comprises improved autolytic stability, as compared to wild-type B. amyloliquefaciens neutral metalloprotease. In some particularly preferred embodiments, the enzyme compositions of the present invention have comparable or improved wash performance, as compared to presently used neutral metalloproteases. Other objects and advantages of the present invention are apparent herein.<br>
DESCRIPTION OF THE FIGURES<br>
Figure 1 provides a graph showing the results from the determination of the affinity constants of purified MULTIFECT neutral binding protein for zinc and calcium cations using the fluorescent dyes Fluo-Zn3 and Fmo-3, respectively.<br>
Figure 2 provides a graph showing inhibition of protease activity of 0.36 mg/ml formulated recombinant B. amyloliquefaciens nprE by Linear Alkylbenaene Sulfonate (LAS) assayed using the QuantiCJeave protease assay.<br>
Figure 3 provides a sequence alignment of various metalloprotease homologues (SEQ ID NOS:173-181) that find use in the present invention-Figure 4 provides a sequence alignment of various metalloprotease homologues (SEQ ID NOS:182-191) that find use in the present invention. In this Figure, the numbering is for thermolysin (B. tliermoproieolyiicus). As in Figure 3, the "*" indicates conserved residues,":" indicates conservatively replaced residues, and "." indicates similar residues.<br>
Figure 5 provides a sequence alignment of various metalloprotease homologues (SEQ ID NOS: 192-195) identified through homoJogy modeling.<br>
Figure 6 provides a map of plasmid pJ4:GO!905.<br>
Figure 7 provides a map of pJasmid pJ4: G01906.<br>
Figure 8 provides a map of plasmid pJ4:G01907.<br>
Figure 9 provides a map of plasmid pJ4:G01908.<br>
Figure I 0 provides a map of plasmid pJ4 :G01909.<br>
Figure 11 provides a map of plasmid pJ4:GO!938.<br>
Figure 12 provides a map of plasmid pJHT.<br>
Figure 13 provides a map of plasmid pAC.<br>
Figure 14 provides a map of pUBnprE.<br>
Figure 15 provides a schematic showing the amplification of the aprE promoter and B. subnlis nprE gene fragments.<br>
Figure 16 provides a map of p] asmid pELSOl.<br>
Figure 17 provides a schematic showing the amplification of ihe aprE promoter and B. subtilis nprB gene fragments.<br>
Figure 18 provides a map of plasmid pEL508.<br>
Figure 19 provides a schematic showing the amplification of the aprE promoter and B. stearotliennophilus nprT gene fragments, used in the production of strain EL560.<br>
Figure 20 provides a diagram showing the construction of strain EL560.<br>
Figure 21 provides a schematic showing the amplification of the aprE promoter and B. caldolyticus npr gene fragments, used in the production of strain EL561.<br>
Figure 22 provides a diagram showing the construction of strain EL561.<br>
Figure 23 provides a schematic showing the amplification of the aprE promoter and B. thuringiensis nprB gene fragments.<br>
Figure 24 provides a map of plasmid pEL568.<br>
Figure 25 provides a graph showing results from experiments designed to determine the long-term storage of 0.36 mg/ml UF concentrate of neutral metalloprotease (nprE) in TIDE 2005 base in the presence of zinc and calcium ions at 32 C. For comparative purposes, results obtained for testing without salt and excess calcium are provided.<br>
Figure 26 provides wash performance test data using Terg-O-Tometer (TOM) and varying soiled substrates. Panel A provides results showing the delta soil removal (%) of subtilisin (BPN' Y217L) and purified MULTIFECT Neutral on EMPA 116 (fixed and unfixed on cotton) after washing at 15C in TIDE-2005 detergent liquid. Panel B provides results showing the delta soil removal (%) of subtilisin (BPN1 Y217L) and purified MULTIFECT Neutral on Equest grass medium soiled on cotton after washing at 15C in TIDE-2005 detergent liquid. Panel C provides results showing the delta soil removal (%) of subtilisin (BPN' Y217L) and purified MULTIFECT Neutral on CFT C-10 (pigment, oil, milk on cotton) after washing at 15C in TIDE-2005 detergent liquid.<br>
Figure 27 provides a graph showing the results of DSC scans for 440 ppm NprE and variants obtained using the VP-Cap DSC (MicroCal).<br>
Figure 28 provides a graph showing the results for DSC scans for 440 ppm NprE and variants in the presence of 130 mM citrate were obtained using the VP-Cap DSC (MicroCal).<br>
Figure 29 provides a graph showing the thermal melting points for 440 ppm NprE in the presence of various additives and obtained using the VP-Cap DSC (MicroCal). In this Figure, the horizontal line represents the Tm for wild-type NprE with no additives.<br>
Figure 30 provides a graph showing the remaining activity of nprE and nprE homologs in 25% TIDE at 25C, after 90 minutes.<br>
Figure 31 provides a graph showing the BMI wash performance of nprE and nprE homologs.<br>
Figure 32 provides a graph showing the results of NprE stability measurements in various formulation mixes.<br>
Figure 33 provides graphs (Panels A, B and C showing the rate of NprE inactivation with different % DTPA concentrations at a fixed calcium formate concentration.<br>
Figure 34 provides graphs (Panels A, B and C) showing the DOE analysis software generated prediction profiles of a DTPA and calcium formate composition based on response goal (decay rate).<br>
Figure 35 provides the amino acid sequences (SEQ ID NOS:222-226) for the citrate-induced autolytic fragments of NprE highlighting the autolysis sites. Fragment 1 and 2 are the first clip, Fragment 3-5 represent the second clip. The italicized letters represent the sequenced N-termini and bold letters highlight the peptides that were identified from the in-gel digestion of the respective fragments.<br>
DESCRIPTION OF THE INVENTION<br>
The present invention provides methods and compositions comprising at least one neutral metalloprotease enzyme that has improved storage stability. In some embodiments, the neutral metalloprotease finds use in cleaning and other applications. In some particularly preferred embodiments, the present invention provides methods and compositions comprising neutral metalloproiease(s) obtained from Bacillus sp. In some more particularly preferred embodiments, the neutral metailoprotease is obtained from B. canyloliquefaciens. In still further preferred embodiments, the neutral metalloprotease is a variant of the B. canyloliquefaciens neutral metalloprotease. In yet additional embodiments, the neutral metalloprotease is a homolog of the the B. canyloliquefaciens neutral metalloprotease. The present invention finds particular use in applications including, but not limited to cleaning, bleaching and disinfecting.<br>
Unless otherwise indicated, the practice of the present invention involves conventional techniques commonly used in molecular biology, microbiology, protein purification, protein engineering, protein and DNA sequencing, and recombinant DNA fields, which are within the skill of the art. Such techniques are known to those of skill in the art and are described in numerous texts and reference works (See e.g., Sambrook et a/., "Molecular Cloning: A Laboratory Manual", Second Edition (Cold Spring Harbor), [1989]);<br>
and Ausubel el al., "Current Protocols in Molecular Biology" [1987]). All patents, patent applications, articles and publications mentioned herein, both supra and infra, are hereby expressly incorporated herein by reference.<br>
Furthermore, the headings provided herein are not limitations of the various aspects or embodiments of the invention which can be had by reference to the specification as a whole. Accordingly, the terms defined immediately below are more fully defined by reference to the specification as a whole. Nonetheless, in order to facilitate understanding of the invention, a number of terms are defined below.<br>
Definitions<br>
Unless defined otherwise herein, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention pertains. For example, Singleton and Sainsbury, Dictionary of Microbiology and Molecular Biology, 2d Ed., John Wiley and Sons, NY (1994); and Hale and Marham, The Harper Collins Dictionary of Biology, Harper Perennial, NY (1991) provide those of skill hi . the art with a general dictionaries of many of the terms used herein. Although any methods and materials similar or equivalent to those described herein find use in the practice of the present invention, the preferred methods and materials are described herein. Accordingly, the terms defined immediately below are more fully described by reference to the Specification as a whole. Also, as used herein, the singular terms "a," "an," and "the" include the plural reference unless the context clearly indicates otherwise. Unless otherwise indicated, nucleic acids are written left to right in 5' to 3' orientation; amino acid sequences are written left to right in amino to carboxy orientation, respectively. It is to be understood that this invention is not limited to the particular methodology, protocols, and reagents described, as these may vary, depending upon the context they are used by those of skill in the an.<br>
It is intended that every maximum numerical limitation given throughout this specification includes every lower numerical limitation, as if such lower numerical limitations were expressly written herein. Every minimum numerical limitation given throughout this specification will include every higher numerical limitation, as if such higher numerical limitations were expressly written herein. Every numerical range given throughout<br>
this specification will include every narrower numerical range that falls within such broader numerical range, as if such narrower numerical ranges were all expressly written herein.<br>
All documents cited are, in relevant part, incorporated herein by reference; the citation of any document is not to be construed as an admission that it is prior art with respect to the present invention.<br>
As used herein, the term "bleaching" refers to the treatment of a material (e.g., fabric, laundry, pulp, etc.) or surface for a sufficient length of time and under appropriate pH and temperature conditions to effect a brightening (Le., whitening) and/or cleaning of the material. Examples of chemicals suitable for bleaching include but are not limited to ClO2, HaO2, peracids, NO2, etc.<br>
As used herein, the term "disinfecting" refers to the removal of contaminants from the surfaces, as well as the inhibition or killing of microbes on the surfaces of items. It is not intended that the present invention be limited to any particular surface, item, or contaminants) or microbes to be removed.<br>
As used herein, the term "multimer" refers to two or more proteins or peptides that are covalently or non-covale'ntly associated and exist as a complex in solution. A "dimer" is a multimer that contains two proteins or peptides; a "trimer" contains three proteins or peptides, etc. As used herein, "octamer" refers to a multimer of eight proteins or peptides.<br>
As used herein, "personal care products" means products used in the cleaning, bleaching and/or disinfecting of hair, skin, scalp, and teeth, including, but not limited to shampoos, body lotions, shower gels, topical moisturizers, toothpaste, and/or other topical cleansers. In some particularly preferred embodiments, these products are utilized on humans, while in other embodiments, these products find use with non-human animals (e.g., in veterinary applications).<br>
As used herein, "cleaning compositions" and "cleaning formulations," unless otherwise indicated, refer to compositions that find use in the removal of undesired compounds from items to be cleaned, such as fabric, dishes, contact lenses, other solid substrates, hair (shampoos), skin (soaps and creams), teeth (mouthwashes, toothpastes) etc. The term encompasses any materials/compounds selected for the particular type of cleaning  . composition desired and the form of the product (e.g., liquid, gel, granule, or spray composition), as long as the composition is compatible with the neutral metalloprotease and other enzyme(s) used in the composition. The specific selection of cleaning composition<br>
materials are readily made by considering the surface, item or fabric to be cleaned, and the<br>
desired form of the composition for the cleaning conditions during use.<br>
The terms further refer to any composition that is suited for cleaning, bleaching,<br>
disinfecting, and/or sterilizing any object and/or surface. It is intended that the terms include, but are not limited to detergent compositions (e.g., liquid and/or solid laundry detergents and fine fabric detergents; hard surface cleaning formulations, such as for glass, wood, ceramic and metal counter tops and windows; carpet cleaners; oven cleaners; fabric fresheners; fabric softeners; and textile and laundry pre-spotters, as well as dish detergents).<br>
Indeed, the term "cleaning composition" as used herein, includes unless otherwise<br>
indicated, granular or powder-form all-purpose or heavy-duty washing agents, especially<br>
cleaning detergents; liquid, gel or paste-form all-purpose washing agents, especially the so-<br>
called heavy-duty liquid (HDL) types; liquid fine-fabric detergents; hand dishwashing agents<br>
or light duty dishwashing agents, especially those of the high-foaming type; machine<br>
dishwashing agents, including the various tablet, granular, liquid and rinse-aid types for<br>
household and institutional use; liquid cleaning and disinfecting agents, including<br>
antibacterial hand-wash types, cleaning bars, mouthwashes, denture cleaners, car or carpet<br>
shampoos, bathroom cleaners; hair .shampoos and hair-rinses; shower gels and foam baths<br>
and metal cleaners; as well as cleaning auxiliaries such as bleach additives and "stain-stick"<br>
or pre-treat types.<br>
As used herein, the terms "detergent composition" and "detergent formulation" are used in reference to mixtures which are intended for use in a wash medium for the cleaning of soiled objects. In some preferred embodiments, the term is used in reference to laundering fabrics and/or garments (e.g., "laundry detergents"). In alternative embodiments, the term refers to other detergents, such as those used to clean dishes, cutlery, etc. (e.g., "dishwashing detergents"). It is not intended that the present invention be limited to any particular detergent formulation or composition. Indeed, it is intended that in addition to neutral metal loprotease, the term encompasses detergents that contain surfactants, transferase(s), hydrolytic enzymes, oxido reductases, builders, bleaching agents, bleach activators, bluing agents and fluorescent dyes, caking inhibitors, masking agents, enzyme activators, antioxidants, and solubilizers.<br>
As used herein, "Applicant Enzyme" refers to the neutral metalloproteases of the present invention.<br>
As used herein, "enhanced performance" in a detergent is defined as increasing cleaning of bleach-sensitive stains (e.g., grass, tea, wine, blood, dingy, etc.), as determined by usual evaluation after a standard wash cycle. In particular embodiments, the neutral metalloprotease of the present invention provides enhanced performance in the removal of colored stains and soils. In further embodiments, the enzyme of the present invention provides enhanced performance in the removal and/or decolorization of stains.<br>
As used herein the term "hard surface cleaning composition," refers to detergent compositions for cleaning hard surfaces such as floors, walls, tile, bath and kitchen fixtures, and the like. Such compositions are provided in any form, including but not limited to solids, liquids, emulsions, etc.<br>
As used herein, "dishwashing composition" refers to all forms for compositions for cleaning dishes, including but not limited to granular and liquid forms.<br>
As used herein, "fabric cleaning composition" refers to all forms of detergent compositions for cleaning fabrics, including but not limited to, granular, liquid and bar forms.<br>
As used herein, "textile" refers to woven fabrics, as well as staple fibers and filaments suitable for conversion to or use as yarns, woven, knit, and non-woven fabrics. The term encompasses yarns made from natural, as well as synthetic (e.g., manufactured) fibers.<br>
As used herein, "textile materials" is a general term for fibers, yam intermediates, yam, fabrics, and products made from fabrics (e.g., garments and other articles).<br>
As used herein, "fabric" encompasses any textile material. Thus, it is intended that the term encompass garments, as well as fabrics, yarns, fibers, non-woven materials, natural materials, synthetic materials, and any other textile material.<br>
As used herein, the term "compatible," means that the cleaning composition materials do not reduce the enzymatic activity of the neutral metalloprotease to such an extent that the neutral metalloprotease is not effective as desired during normal use situations. Specific cleaning composition materials are exemplified in detail hereinafter.<br>
As used herein, "effective amount of enzyme" refers to the Quantity of enzyme necessary to achieve the enzymatic activity required in the specific application (e.g., personal care product, cleaning composition, etc.). Such effective amounts are readily ascertained by one of ordinary skill in the art and are based on many factors, such as the particular enzyme<br>
variant used, the cleaning application, the specific composition of the cleaning composition, and whether a liquid or dry (e.g., granular, bar) composition is required, and the like.<br>
As used herein, "non-fabric cleaning compositions" encompass hard surface cleaning compositions, dishwashing compositions, personal care cleaning compositions (e.g., oral cleaning compositions, denture cleaning compositions, personal cleansing compositions, etc.), and compositions suitable for use in the pulp and paper industry.<br>
As used herein, "oral cleaning compositions" refers to dentifrices, toothpastes, toothgels, toothpowders, mouthwashes, mouth sprays, mouth gels, chewing gums, lozenges, sachets, tablets, biogels, prophylaxis pastes, dental treatment solutions, and the like.<br>
As used herein, the term "transferase" refers to an enzyme that catalyzes the transfer of functional compounds to a range of substrates.<br>
As used herein, "leaving group" refers to the nucleophile which is cleaved from the acyl donor upon substitution by another nucleophile.<br>
As used herein, the term "enzymatic conversion" refers to the modification of a substrate to an intermediate or the modification of an intermediate to an end-product by contacting the substrate or intermediate with an enzyme. In some embodiments, contact is  made by directly exposing the substrate or intermediate to the appropriate enzyme. In other embodiments, contacting comprises exposing the substrate or intermediate to an organism that expresses and/or excretes the enzyme, and/or metabolizes the desired substrate and/or intermediate to the desired intermediate and/or end-product, respectively.<br>
As used herein, the phrase "detergent stability" refers to the stability of a detergent composition. In some embodiments, the stability is assessed during the use of the detergent, while in other embodiments, the term refers to the stability of a detergent composition during storage.<br>
As used herein, the phrase, "stability to proteolysis" refers to the ability of a protein (e.g., an enzyme) to withstand proteolysis. It is not intended that the term be limited to the use of any particular protease to assess the stability of a protein.<br>
As used herein, "oxidative stability" refers to the ability of a protein to function under oxidative conditions. In particular, the term refers to the ability of a protein to function in the presence of various concentrations of H2O2 and/or peracid. Stability under various oxidative conditions can be measured either by standard procedures known to those in the art and/or by the methods described herein. A substantial change in oxidative stability is<br>
evidenced by at least about a 5% or greater increase or decrease (in most embodiments, it is preferably an increase) in the half-life of the enzymatic activity, as compared to the enzymatic activity present in the absence of oxidative compounds.<br>
As vised herein, "pH stability" refers to the ability of a protein to function at a particular pH. In general, most enzymes have a finite pH range at which they will function. In addition to enzymes that function in mid-range pHs (i.e., around pH 7), there are enzymes that are capable of working under conditions with very high or very low pHs. Stability at various pHs can be measured either by standard procedures known to those in the art and/or by the methods described herein. A substantial change in pH stability is evidenced by at least about 5% or greater increase or decrease (in most embodiments, it is preferably an increase) in the half-life of the enzymatic activity, as compared to the enzymatic activity at the enzyme's optimum pH. However, it is not intended that the present invention be limited to any pH stability level nor pH range.<br>
As used herein, "thermal stability" refers to the ability of a protein to function at a particular temperature. In general, most enzymes have a finite range of temperatures at which they will function. In addition to enzymes that work in mid-range temperatures (e.g., room temperature), there are enzymes that are capable of working in very high or very low temperatures. Thermal stability can be measured either by known procedures or by the methods described herein. A substantial change in thermal stability is evidenced by at least about 5% or greater increase or decrease (in most embodiments, it is preferably an increase) in the half-life of the catalytic activity of a mutant when exposed to a different temperature (i.e., higher or lower) than optimum temperature for enzymatic activity.  However, it is not intended that the present invention be limited to any temperature stability level nor     . temperature range.<br>
As used herein, the term "chemical stability" refers to the stability of a protein (e.g., an enzyme) towards chemicals that adversely affect its activity. In some embodiments, such chemicals include, but are not limited to hydrogen peroxide, peracids, anionic detergents, cationic detergents, non-ionic detergents, chelants, etc. However, it is not intended that the present invention be limited to any particular chemical stability level nor range of chemical stability.<br>
As used herein, the phrase "neutral metalloprotease activity improvement" refers to the relative improvement of neutral metalloprotease activity, in comparison with a standard<br>
enzyme. In some embodiments, the term refers to an improved rate of product formation, while in other embodiments, the term encompasses compositions that produce less hydrolysis product. In additional embodiments, the term refers to neutral metallpprotease compositions with altered substrate specificity.<br>
As used herein, the phrase "alteration in substrate specificity" refers to changes in the substrate specificity of an enzyme. In some embodiments, a change in substrate specificity is-defined as a difference between the Kcat/Km ratio observed with an enzyme compared to enzyme variants or other enzyme compositions.  Enzyme substrate specificities vary, depending upon the substrate tested. The substrate specificity of an enzyme is determined by comparing the catalytic efficiencies it exhibits with different substrates. These determinations find particular use in assessing the efficiency of mutant enzymes, as it is generally desired to produce variant enzymes that exhibit greater ratios for particular substrates of interest. However, it is not intended that the present invention be limited to any particular substrate composition nor any specific substrate specificity.<br>
As used herein, "surface property" is used in reference to an electrostatic charge, as well as properties such as the hydrophobicity and/or hydrophiJicity exhibited by the surface of a protein.<br>
As used herein, the phrase "is independently selected from the group consisting of.. .." means that moieties or elements that are selected from the referenced Markush group can be the same, can be different or any mixture of elements as indicated in the following example:<br>
A molecule having 3 R groups wherein each R group is independently selected from the group consisting of A, B and C. Here the three R groups may be: AAA, BBB, CCC, AAB, AAC, BBA, BBC, CCA, CCB, or ABC.<br>
In reference to chemical compositions, the term "substituted" as used herein, means that the organic composition or radical to which the term is applied is:<br>
(a)	made unsaturated by the elimination of at least one element or radical; or<br>
(b)	at least one hydrogen in the compound or radical is replaced with a moiety<br>
containing one or more (i) carbon, (ii) oxygen, (iii) sulfur, (iv) nitrogen or (v)<br>
halogen atoms; or<br>
(c)	both (a) and (b).<br>
Moieties which may replace hydrogen as described in (b) immediately above, that contain only carbon and hydrogen atoms, are hydrocarbon moieties including, but not limited to, alkyl, alkenyl, alkynyl, alkyldienyl, cycloalkyl, phenyl, alkyl phenyl, naphthyl, anthryl, phenanthryl, fluoryl, steroid groups, and combinations of these groups with each other and with polyvalent hydrocarbon groups such as alkylene, alkylidene and alkylidyne groups. Moieties containing oxygen atoms that may replace hydrogen as described in (b) immediately above include, but are not limited to, hydroxy, acyl or keto, ether, epoxy, carboxy, and ester containing groups. Moieties containing sulfur atoms that may replace hydrogen as described in (b) immediately above include, but are not limited to, the sulfur-containing acids and acid ester groups, thioether groups, mercapto groups and thioketo groups. Moieties containing nitrogen atoms that may replace hydrogen as described in (b) immediately above include, but are not limited to, amino groups, the nitro group, azo groups, ammonium groups, amide groups, azido groups, isocyanate groups, cyano groups and nitrile groups. Moieties containing halogen atoms that may replace hydrogen as described in (b) immediately above include chloro, bromo, fluoro, iodo groups and any of the moieties previously described where a hydrogen or a pendant alkyl group is substituted by a halo group to form a stable substituted moiety.<br>
It is understood that any of the above moieties (b)(i) through (b)(v) can be substituted into each other in either a monovalent substitution or by loss of hydrogen in a polyvalent substitution to form another monovalent moiety that can replace hydrogen in the organic compound or radical.<br>
As used herein, the terms "purified" and "isolated" refer to the removal of contaminants from a sample. For example, neutral metalloprotease are purified by removal of contaminating proteins and other compounds within a solution or preparation that are not neutral metalloprotease. In some embodiments, recombinant neutral metalloprotease are expressed in bacterial or fungal host cells and these recombinant neutral metalloproteases are purified by the removal of other host cell constituents; the percent of recombinant neutral metalloprotease polypeptides is thereby increased in the sample. In particularly preferred embodiments, .the metalloprotease of the present invention is substantially purified to a level of at least about 99% of the protein component, as determined by SDS-PAGE or other standard methods known in the art. In alternative preferred embodiments, the metalloprotease of the present invention comprise at least about 99% of the protease<br>
component of the compositions. In yet alternative embodiments, the metalloprotease is present in a range of about at least 90-95% of the total protein and/or protease.<br>
As used herein, "protein of interest," refers to a protein (e.g., an enzyme or "enzyme of interest") which is being analyzed, identified and/or modified. Naturally-occurring, as well as recombinant proteins find use in the present invention.<br>
As used herein, "protein" refers to any composition comprised of amino acids and recognized as a protein by those of skill in the art. The terms "protein," "peptide" and polypeptide are used interchangeably herein. Wherein a peptide is a portion of a protein, those skilled in the art understand the use of the term in context.<br>
As used herein, functionally and/or structurally similar proteins are considered to be "related proteins." In some embodiments, these proteins are derived from a different genus and/or species, including differences between classes of organisms (e.g., &amp; bacterial protein and a fungal protein). In some embodiments, these proteins are derived from a different genus and/or species, including differences between classes of organisms (e.g., a bacterial enzyme and a fungal enzyme). In additional embodiments, related proteins are provided from the same species. Indeed, it is not intended that the present invention be limited to related proteins from any particular source(s). In addition, the term "related proteins" encompasses tertiary structural homologs and primary sequence homologs (e.g., the neutral metalloprotease of the present invention). For example, the present invention encompasses such homologs as those provided in Figures 3-5. Additional homologs are contemplated, including but not limited to metalloprotease enzymes obtained from B. cereus, B. cereus E33L, B. caldolyticus, B.pumulis, B. megaterium, B subtilis amylosacchariticus, Brfvibacillus brevis, Paenibacillus polymyxa (Bacillus polymyxa),B. stearothermophilus, B. thuringiensis, B. subtilis and S. aureus, as well as aureolysin, extracellular elastase, and neutral protease B. In further embodiments, the term encompasses proteins that are immunologically cross-reactive.<br>
As used herein, the term "derivative" refers to a protein which is derived from a protein by addition of one or more amino acids to either or both the C- and N-terminal end(s), substitution of one or more amino acids at one or a number of different sites in the amino acid sequence, and/or deletion of one or more amino acids at either or both ends of the<br>
protein or at one or more sites in the amino acid sequence, and/or insertion of one or more amino acids at one or more sites in the amino acid sequence. The preparation of a protein<br>
derivative is preferably achieved by modifying a DNA sequence which encodes for the native protein, transformation of that DNA sequence into a suitable host, and expression of the modified DNA sequence to form the derivative protein.<br>
Related (and derivative) proteins comprise "variant proteins." In some preferred embodiments, variant proteins differ from a parent protein and one another by a small number of amino acid residues. The number of differing amino acid residues may be one or more, preferably 1,2, 3,4,5,10,15,20,30,40,50, or more amino acid residues. In some preferred embodiments, the number of different amino acids between variants is between 1 and 10. In some particularly preferred embodiments, related proteins and particularly variant proteins comprise at least 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, 97%, 98%, or 99% amino acid sequence identity. Additionally, a related protein or a variant protein as used herein, refers to a protein that differs from another related protein or a parent protein in the number of prominent regions. For example, in some embodiments, variant proteins have 1, 2, 3,4, 5, or 10 corresponding prominent regions that differ from the parent protein.<br>
Several methods are known in the art that are suitable for generating variants of the enzymes of the present invention, including but not limited to site-saturation mutagenesis, scanning mutagenesis, insertional mutagenesis, random mutagenesis, site-directed mutagenesis, and directed-evolution, as well as various other recombinatorial approaches.<br>
Characterization of wild-type and mutant proteins is accomplished via any means suitable and is preferably based on the assessment of properties of interest. For example, pH and/or temperature, as well as detergent and /or oxidative stability is/are determined in some embodiments of the present invention. Indeed, it is contemplated that enzymes having various degrees of stability in one or more of these characteristics (pH, temperature, proteolytic stability, detergent stability, and/or oxidative stability) will find use.<br>
As used herein, "expression vector" refers to a DNA construct containing a DNA sequence that is operably linked to a suitable control sequence capable of effecting the expression of the DNA in a suitable host. Such control sequences include a promoter to effect transcription, an optional operator sequence to control such transcription, a sequence encoding suitable mRNA ribosome binding sites and sequences which control termination of transcription and translation. The vector may be a plasmid, a phage panicle, or simply a potential genomic insert. Once transformed into a suitable host, the vector may replicate and<br>
function independently of the host genome, or may, in some instances, integrate into the genome itself. In the present specification, "plasmid," "expression plasmid," and "vector" are often used interchangeably as the plasmid is the most commonly used form of vector at present. However, the invention is intended to include such other forms of expression vectors that serve equivalent functions and which are, or become, known in the art.<br>
In some preferred embodiments, the neutral metalloprotease gene is ligated into an appropriate expression plasmid. The cloned neutral metalloprotease gene is then used to transform or transfect a host cell in order to express the neutral metalloprotease gene. This plasmid may replicate in hosts in the sense that it contains the well-known elements necessary for plasmid replication or the plasmid may be designed to integrate into the host chromosome. The necessary elements are provided for efficient gene expression (e.g., a promoter operably linked to the gene of interest). In some embodiments, these necessary elements are supplied as the gene's own homologous promoter if it is recognized, (i.e., transcribed, by the host), a transcription terminator (a polyadenylation region for eukaryotic host cells) which is exogenous or is supplied by the endogenous terminator region of the neutral metalloprotease gene. In some embodiments, a selection gene such as an antibiotic resistance gene that enables continuous cultural maintenance of plasmid-infected host cells by growth in antimicrobial-containing media is also included.<br>
The following cassette mutagenesis method may be used to facilitate the construction of the neutral metalloprotease variants of the present invention, although other methods may be used. First, as described herein, a naturally-occurring gene encoding the neutral metalloprotease is obtained and sequenced in whole or in part. Then, the sequence is scanned for a point at which it is desired to make a mutation (deletion, insertion or substitution) of one or more amino acids in the encoded neutral metalloprotease. The sequences flanking this point are evaluated for the presence of restriction sites for replacing a short segment of the gene with an oligonucleotide pool which when expressed will encode various mutants. Such restriction sites are preferably unique sites within the protein gene so as to facilitate the replacement of the gene segment. However, any convenient restriction site which is not overly redundant in the neutral metalloprotease gene may be used, provided the gene fragments generated by restriction digestion can be reassembled in proper sequence. If restriction sites are not present at locations within a convenient distance from the selected point (from 10 to 15 nucleotides), such sites are generated by substituting nucleotides in the<br>
gene in such a fashion that neither the reading frame nor the amino acids encoded are changed in the final construction. Mutation of the gene in order to change its sequence to conform to the desired sequence is accomplished by M13 primer extension in accord with generally known methods. The task of locating suitable flanking regions and evaluating the needed changes to arrive at two convenient restriction site sequences is made routine by the redundancy of the genetic code, a restriction enzyme map of the gene and the large number of different.restriction enzymes. Note that if a convenient flanking restriction site is available, the above method need be used only in connection with the flanking region which does not contain a site.<br>
Once the naturally-occurring DNA and/or synthetic DNA is cloned, the restriction sites flanking the positions to be mutated are digested with the cognate restriction enzymes and a plurality of end termini-complementary oligonucleotide cassettes are ligated into the gene. The mutagenesis is simplified by this method because all of the oligonucleotides can be synthesized so as to have the same restriction sites, and no synthetic linkers are necessary to create the restriction sites.<br>
As used herein, "corresponding to," refers to a residue at the enumerated position in a protein or peptide, or a residue that is analogous, homologous, or equivalent to an enumerated residue in a protein or peptide.<br>
As used herein, "corresponding region," generally refers to an analogous position along related proteins or a parent protein.<br>
The terms "nucleic acid molecule encoding," "nucleic acid sequence encoding," "DNA sequence encoding," and "DNA encoding" refer to the order or sequence of deoxyribonucleotides along a strand of deoxyribonucleic acid. The order of these deoxyribonucleotides determines the order of amino acids along the polypeptide (protein) chain. The DNA sequence thus codes for the amino acid sequence.<br>
As used herein, the term "analogous sequence" refers to a sequence within a protein that provides similar function, tertiary structure, and/or conserved residues as the protein of interest (i,e., typically the original protein of interest). For example, in epitope regions that contain an alpha helix or a beta sheet structure, the replacement amino acids in the analogous sequence preferably maintain the same specific structure. The term also refers to nucleotide sequences, as well as amino acid sequences. In some embodiments, analogous sequences are developed such that the replacement amino acids result in a variant enzyme showing a similar<br>
or improved function. In some preferred embodiments, the tertiary structure and/or conserved residues of the amino acids in the protein of interest are located at or near the segment or fragment of interest. Thus, where the segment or fragment of interest contains, for example, an alpha-helix or a beta-sheet structure, the replacement amino acids preferably maintain that specific structure.<br>
As used herein, "homologous protein" refers to a protein (e.g., neutral metalloprotease) that has similar action and/or structure, as a protein of interest (e.g., an neutral metalloprotease from another source). It is not intended that homologs (also referred to herein as "homologues") be necessarily related evolutionarily. Thus, it is intended that the term encompass the same or similar enzyme(s) (Le., in terms of structure and function) obtained from different species. In some preferred embodiments, it is desirable to identify a homolog that has a quaternary, tertiary and/or primary structure similar to the protein of interest, as replacement for the segment or fragment in the protein of interest with an analogous segment from the homolog will reduce the disruptiveness of the change.<br>
As used herein, "homologous genes" refers to at least a pair of genes from different species, which genes correspond to each other and which are identical or very similar to each other. The term encompasses genes that are separated by speciation (Le., the development of new species) (e.g., orthologous genes), as well as genes that have been separated by genetic duplication (e.g., paralogous genes). These genes encode "homologous proteins."<br>
As used herein, "ortholog" and "orthologous genes" refer to genes in different species that have evolved from a common ancestral gene (Le., a homologous gene) by speciation. Typically, orthologs retain the same function during the course of evolution. Identification of orthologs finds use in the reliable prediction of gene function in newly sequenced genomes.<br>
As used herein, "paralog" and "paralogous genes" refer to genes that are related by duplication within a genome. While orthologs retain the same function through the course of evolution, paralogs evolve new functions, even though some functions are often related to the original one. Examples of paralogous genes include, but are not limited to genes encoding trypsin, chymotrypsin, elastase, and thrombin, which are all serine proteinases and occur together within the same species.<br>
As used herein, "wild-type" and "native" proteins are those found in nature. The terms "wild-type sequence," and "wild-type gene" are used interchangeably herein, to refer to<br>
a sequence that is native or naturally occurring in a host cell. In .some embodiments, the wild-type sequence refers to a sequence of interest that is the starting point of a protein engineering project. The genes encoding the naturally-occurring protein may be obtained in accord with the general methods known to those skilled in the art. The methods generally comprise synthesizing labeled probes having putative sequences encoding regions of the protein of interest, preparing genomic libraries from organisms expressing the protein, and screening the libraries for the gene of interest by hybridization to the probes. Positively hybridizing clones are then mapped and sequenced.<br>
The term "recombinant DNA molecule" as used herein refers to a DNA molecule that is comprised of segments of DNA joined together by means of molecular biological techniques.<br>
The term "recombinant oligonucleotide" refers to an oligonucleotide created using molecular biological manipulations, including but not limited to, the ligation of two or more oligonucleotide sequences generated by restriction enzyme digestion of a polynucleotide sequence, the synthesis of oligonucleotides (e.g., the synthesis of primers or oligonucleotides) and the like.<br>
The degree of homology between sequences may be determined using any suitable method known in the art (See e.g., Smith and Waterman, Adv. Appl. Math., 2:482 [1981]; Needleman and Wunsch, J. Mol. Biol., 48:443 [1970]; Pearson and Lipman, Proc. Natl. Acad. Sci. USA 85:2444 [1988]; programs such as GAP, BESTFTT, FASTA, and TFASTA in the Wisconsin Genetics Software Package (Genetics Computer Group, Madison, WJ); and Devereux et al., Nucl. Acid Res., 12:387-395 [1984]).<br>
For example, PflJEUP is a useful program to determine sequence homology levels. PILEUP creates a multiple sequence alignment from a group of related sequences using progressive, pairwise alignments. It can also plot a tree showing the clustering relationships used to create the alignment. PILEUP uses a simplification of the progressive alignment method of Feng and Doolittle, (Feng and Doolittle, J. Mol. Evol., 35:351-360 [1987]). The method is similar to that described by Higgins and Sharp (Higgins and Sharp, CABIOS 5:151-153 [1989]). Useful PILEUP parameters including a default gap weight of 3.00, a default gap length weight of 0.10, and weighted end gaps. Another example of a useful algorithm is the BLAST algorithm, described by Altschul et al., (Altschul et al., J. Mol. Biol., 215:403-41.0, [1990]; and Karlin et.al* Proc. Natl. Acad. Sci. USA 90:5873-5787 [1993]).<br>
One particularly useful BLAST program is the WU-BLAST-2 program (See, Altschul el al., Meth. Enzymo).,, 266:460-480 [1996]). parameters "W," "T," and "X" determine the sensitivity and speed of the alignment. The BLAST program uses as defaults a wordlength (W) of 11, the BLOSUM62 scoring matrix (See, Henikoff and Henikoff, Proc. Natl. Acad. Sci. USA 89:10915 [1989]) alignments (B) of 50, expectation (E) of 10, M'5, N'-4, and a comparison of both strands.<br>
As used herein, "percent (%) nucleic acid sequence identity" is defined as the percentage of nucleotide residues in a candidate sequence that is identical with the nucleotide residues of the sequence.<br>
As used herein, the term "hybridization" refers to the process by which a strand of nucleic acid joins with a complementary strand through base pairing, as known in the art.<br>
As used herein, the phrase "hybridization conditions" refers to the conditions under which hybridization reactions are conducted. These conditions are typically classified by degree of "stringency" of the conditions under which hybridization is measured. The degree of stringency can be based, for example, on the melting temperature (Tm) of the nucleic acid binding complex or probe. For example, "maximum stringency" typically occurs at about Tm-5C (5 below the Tm of the probe); "high stringency" at about 5-10 below the Tm; "intermediate stringency" at about 10-20 below the Tm of the probe; and "low stringency" at about 20-25 below the Tm. Alternatively, or in addition, hybridization conditions can be based upon the salt or ionic strength conditions of hybridization and/or one or more stringency washes. For example, 6xSSC = very low stringency; 3xSSC = low to medium stringency; IxSSC = medium stringency; and 0.5xSSC = high stringency. Functionally, maximum stringency conditions may be used to identify nucleic acid sequences having strict identity or near-strict identity with the hybridization probe; while high stringency conditions are used to identify nucleic acid sequences having about 80% or more sequence identity with the probe.<br>
For applications requiring high selectivity, it is typically desirable to use relatively stringent conditions to form the hybrids (e.g., relatively low salt and/or high temperature conditions are used).<br>
The phrases "substantially similar and "substantially identical" in the context of at least two nucleic acids or polypeptides typically means that a polynucleotide or polypeptide<br>
comprises a sequence that has at least about 40% identity, more preferable at least about 50% identity, yet more preferably at least about 60% identity, preferably at least about 75% identity, more preferably at least about 80% identity, yet more preferably at least about 90%, still more preferably about 95%, most preferably about 97% identity, sometimes as much as about 98% and about 99% sequence identity, compared to the reference (i.e., wild-type) sequence. Sequence identity may be determined using known programs such as BLAST, ALIGN, and CLUSTAL using standard parameters. (See e.g., Altschul, et al., J. Mol. Biol. 215:403-410 [1990]; Henikoff et al, Proc. Nad. Acad. Sci. USA 89:10915 [1989]; Karin et al., Proc. Natl. Acad. Sci USA 90:5873 [1993]; and Higgins et al, Gene 73:237 - 244 [1988]). Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information. Also, databases may be searched using FASTA (Pearson et al., Proc. Natl. Acad. Sci. USA 85:2444-2448 [1988]). One indication that two polypeptides are substantially identical is that the first polypeptide is immunologically cross-reactive with the second polypeptide. Typically, polypeptides that differ by conservative amino acid substitutions are immunologically cross-reactive. Thus, a polypeptide is substantially identical to a second polypeptide, for example, where the two peptides differ only by a conservative substitution. Another indication that two nucleic acid sequences are substantially identical is that the two molecules hybridize to each other under stringent conditions (e.g., within a range of medium to high stringency).<br>
As used herein, "equivalent residues" refers to proteins that share particular amino acid residues. For example, equivalent resides may be identified by determining homology at the level of tertiary structure for a protein (e.g., neutral metalloprotease) whose tertiary structure has been determined by x-ray crystallography. Equivalent residues are defined as those for which the atomic coordinates of two or more of the main chain atoms of a particular amino acid residue of the protein having putative equivalent residues and the protein of interest (N on N, CA on CA, C on C and O on O) are within 0.13 nm and preferably 0.1 nm after alignment. Alignment is achieved after the best model has been oriented and positioned to give the maximum overlap of atomic coordinates of non-hydrogen protein atoms of the proteins analyzed. The preferred model is the crystallographic model giving the lowest R factor for experimental diffraction data at the highest resolution available, determined using methods known to those skilled in the art of crystallography and protein characterization/analysis.<br>
As used herein, the terms "hybrid neutral metalloproteases" and "fusion neutral metalloproteases" refer to proteins that are engineered from at least two different or "parental" proteins. In preferred embodiments, these parental proteins are homologs of one another. For example, in some embodiments, a preferred hybrid neutral metalloprotease or fusion protein contains the N-terminus of a protein and the C-terminus of a homolog of the protein. In some preferred embodiment, the two terminal ends are combined to correspond to the full-length active protein.<br>
The term "regulatory element" as used herein refers to a genetic element that controls some aspect of the expression of nucleic acid sequences. For example, a promoter is a regulatory element which facilitates the initiation of transcription of an operably linked coding region. Additional regulatory elements include splicing signals, polyadenylation signals and termination signals.<br>
As used herein, "host cells" are generally prokaryotic or eukaryotic hosts which are transformed or transfected with vectors constructed using recombinant DNA techniques known in the art. Transformed host cells are capable of either replicating vectors encoding the protein variants or expressing the desired protein variant. In the case of vectors which encode the pre- or prepro-form of the protein variant, such variants, when expressed, are typically secreted from the host cell into the host cell medium.<br>
The term "introduced" in the context of inserting a nucleic acid sequence into a cell, means transformation, transduction or transfection. Means of transformation include protoplast transformation, calcium chloride precipitation, electro oration, naked DNA and the like as known in the art. (See, Chang and Cohen, Mol. Gen. Genet., 168:111 -115 [1979]; Smith el al., Appl. Env. Microbiol., 51:634 [1986]; and the review article by Ferrari et al., in Harwood, Bacillus. Plenum Publishing Corporation, pp. 57-72 [1989]).<br>
The term "promoter/enhancer" denotes a segment of DNA which contains sequences capable of providing both promoter and enhancer functions (for example, the long terminal repeats of retroviruses contain both promoter and enhancer functions). The enhancer/promoter may be "endogenous" or "exogenous" or "heterologous." An endogenous enhancer/promoter is one which is naturally linked with a given gene in the genome. An exogenous (heterologous) enhancer/promoter is one which is placed in juxtaposition to a gene by means of genetic manipulation (i.e., molecular biological techniques).<br>
The presence of "splicing signals" on an expression vector often results in higher levels of expression of the recombinant transcript. Splicing signals mediate the removal of introns from the primary RNA transcript and consist of a splice donor and acceptor site (Sarnbrook et al, Molecular Cloning: A Laboratory Manual, 2nd ed., Cold Spring Harbor Laboratory Press, New York [1989], pp. 16.7-16.8). A commonly used splice donor and acceptor site is the splice junction from the 16S RNA of SV40.<br>
The term "stable transfection" or "stably transfected" refers to the introduction and integration of foreign DNA into the genome of the transfected cell. The term "stable transfectant" refers to a cell which has stably integrated foreign or exogenous DNA into the genomic DNA of the transfected cell.<br>
The terms "selectable marker" or "selectable gene product" as used herein refer to the use of a gene which encodes an enzymatic activity that confers resistance to an antibiotic or drug upon the cell in which the selectable marker is expressed.<br>
As used herein, the terms "amplification" and "gene amplification" refer to a process by which specific DNA sequences are disproportionately replicated such that the amplified gene becomes present in a higher copy number than was initially present in the genome. In some embodiments, selection of cells by growth in the presence of a drug (e.g., an inhibitor of an inhibitable enzyme) results in the amplification of either the endogenous gene encoding the gene product required for growth in the presence of the drug or by amplification of exogenous (i.e., input) sequences encoding this gene product, or both. Selection of cells by growth in the presence of a drug (e.g., an inhibitor of an inhibitable enzyme) may result in the amplification of either the endogenous gene encoding the gene product required for growth in the presence of the drug or by amplification of exogenous (i.e., input) sequences encoding this gene product, or both.<br>
"Amplification" is a special case of nucleic acid replication involving template specificity. It is to be contrasted with non-specific template replication (i.e., replication that is template-dependent but not dependent on a specific template). Template specificity is here distinguished from fidelity of replication (i.e., synthesis of the proper polynucleotide sequence) and nucleotide (ribo- or deoxyribo-) specificity. Template specificity is frequently described in terms of "target" specificity. Target sequences are "targets" in the sense that they are sought to be sorted out from other nucleic acid. Amplification techniques have been designed primarily for this sorting out.<br>
As used herein, the terra "co-amplification" refers to the introduction into a single cell of an amplifiable marker in conjunction with other gene sequences (i.e., comprising one or more non-selectable genes such as those contained within an expression vector) and the application of appropriate selective pressure such that the cell amplifies both the amplifiable marker and the other, non-selectable gene sequences. The amplifiable marker may be physically linked to the other gene sequences or alternatively two separate pieces of DNA, one containing the amplifiable marker and the other containing the non-selectable marker, may be introduced into the same cell.<br>
As used herein, the terms "amplifiable marker," "amplifiable gene," and "amplification vector" refer to a marker, gene or a vector encoding a gene which permits the amplification of that gene under appropriate growth conditions.<br>
As used herein, the term "amplifiable nucleic acid" refers to nucleic acids which may be amplified by any amplification method. It is contemplated that "amplifiable nucleic acid" will usually comprise "sample template."<br>
As used herein, the term "sample template" refers to nucleic acid originating from a sample which is analyzed for the presence of "target" (defined below). In contrast, "background template" is used in reference to nucleic acid other than sample template which may or may not be present in a sample. Background template is most often inadvertent. It may be the result of carryover, or it may be due to the presence of nucleic acid contaminants sought to be purified away from the sample. For example, nucleic acids from organisms other than those to be detected may be present as background in a test sample.<br>
'Template specificity" is achieved in most amplification techniques by the choice of enzyme. Amplification enzymes are enzymes that, under conditions they are used, will process only specific sequences of nucleic acid in a heterogeneous mixture of nucleic acid. For example, in the case of Qp replicase, MDV-1 RNA is the specific template for the replicase (See e.g., Kacian el al, Proc. Natl. Acad. Sci. USA 69:3038 [1972]). Other nucleic acids are not replicated by this amplification enzyme. Similarly, in the case of T7 RNA polymerase, this amplification enzyme has a stringent specificity for its own promoters (See, Chamberlin et al, Nature 228:227 [1970]). In the case of T4 DNA ligase, the enzyme will not ligate the two oligonucleotides or polynucleotides, where there is a mismatch between the oligonucleotide or polynucleotide substrate and the template at the ligation junction (See, Wu and Wallace, Genomics 4:560 [1989]). Finally, Taq and Pfu polymerases, by virtue of their<br>
ability to function at high temperature, are found to display high specificity for the sequences bounded and thus defined by the primers; the high temperature results in thermodynamic conditions that favor primer hybridization with the target sequences and not hybridization with non-target sequences.<br>
As used herein, the terra "primer" refers to an oligonucleotide, whether occurring naturally as in a purified restriction digest or produced synthetically, which is capable of acting as a point of initiation of synthesis when placed under conditions in which synthesis of a primer extension product which is complementary to a nucleic acid strand is induced, (i.e., in the presence of nudeotides and an inducing agent such as DNA polymerase and at a suitable temperature and pH). The primer is preferably single stranded for maximum efficiency in amplification, but may alternatively be double stranded. If double stranded, the primer is first treated to separate its strands before being used to prepare extension products. Preferably, the primer is an oligodeoxyribonucleotide. The primer must be sufficiently long to prime the synthesis of extension products in the presence of the inducing agent. The exact lengths of the primers will depend on many factors, including temperature, source of primer and the use of the method.<br>
As used herein, the term "probe" refers to an oligonucleotide (i.e., a sequence of nucleotides), whether occurring naturally as in a purified restriction digest or produced synthetically, recombinantly or by PCR amplification, which is capable of hybridizing to another oligonucleotide of interest. A probe may be single-stranded or double-stranded. Probes are useful in the detection, identification and isolation of particular gene sequences. It is contemplated that any probe used in the present invention will be labeled with any "reporter molecule," so that is detectable in any detection system, including, but not limited to enzyme (e.g., EL1SA, as well as enzyme-based histochemical assays), fluorescent, radioactive, and luminescent systems. It is not intended that the present invention be limited to any particular detection system or label.<br>
As used herein, the term "target," when used in reference to amplification methods (e.g., the polymerase chain reaction), refers to the region of nucleic acid bounded by the primers used for polymerase chain reaction. Thus, the "target" is sought to be sorted out from other nucleic acid sequences. A "segment" is defined as a region of nucleic acid within the target sequence.<br>
As used herein, the term "polymerase chain reaction" ("PCR") refers to the methods of U.S. Patent Nos. 4,683,195,4,683,202, and 4,965,188, hereby incorporated by reference, which include methods for increasing the concentration of a segment of a target sequence in a mixture of genomic DNA without cloning or purification. This process for amplifying the target sequence consists of introducing a large excess of two oligonucleotide primers to the DNA mixture containing the desired target sequence, followed by a precise sequence of thermal cycling in the presence of a DNA polymerase. The two primers are complementary to their respective strands of the double stranded target sequence. To effect amplification, the mixture is denatured and the primers then annealed to their complementary sequences within the target molecule. Following annealing, the primers are extended with a polymerase so as to form a new pair of complementary strands. The steps of denaturation, primer annealing and polymerase extension can be repeated many times (i.e., denaturation, annealing and extension constitute one "cycle"; there can be numerous "cycles") to obtain a high concentration of an amplified segment of the desired target sequence. The length of the     amplified segment of the desired target sequence is determined by the relative positions of the primers with respect to each other, and therefore, this length is a controllable parameter. By virtue of the repeating aspect of the process, the method is referred to as the "polymerase chain reaction" (hereinafter "PCR"). Because the desired amplified segments of the target sequence become the predominant sequences (in terms of concentration) in the mixture, they are said to be "PCR amplified".<br>
As used herein, the term "amplification reagents" refers to those reagents (deoxyribonucleotide triphosphates, buffer, etc.), needed for amplification except for primers, nucleic acid template and the amplification enzyme. Typically, amplification reagents along with other reaction components are placed and contained in a reaction vessel (test tube, microwell, etc.).<br>
With PCR, it is possible to amplify a single copy of a specific target sequence in genomic DNA to a level detectable by several different methodologies (e.g., hybridization with a labeled probe; incorporation of biotinylated primers followed by avidin-enzyme conjugate detection; incorporation of 32P-labeled deoxynucleotide triphosphates, such as dCTP or dATP, into the amplified segment). In addition to genomic DNA, any oligonucleotide or polynucleotide sequence can be amplified with the appropriate set of<br>
primer molecules. In particular, the amplified segments created by the PCR process itself are, themselves, efficient templates for subsequent PCR amplifications.<br>
As used herein, the terms "PCR product," "PCR fragment," and "amplification product" refer to the resultant mixture of compounds after two or more cycles of the PCR steps of denaturation, annealing and extension are complete. These terms encompass the case where there has been amplification of one or more segments of one or more target sequences.<br>
As used herein, the terms "restriction endonucleases" and "restriction enzymes" refer to bacterial enzymes, each of which cut double-stranded DNA at or near a specific nucleotide sequence.<br>
DETAILED DESCRIPTION OF THE PRESENT INVENTION<br>
The present invention provides methods and compositions comprising at least one neutral metalloprotease enzyme that has improved storage stability. In some embodiments, the neutral metalloprotease finds use in cleaning and other applications. In some particularly preferred embodiments, the present invention provides methods and compositions comprising neutral metal!oprotease(s) obtained from Bacillus sp. In some more particularly preferred embodiments, the neutral metal!oprotease is obtained from B. amyloliquefaciens. In still further preferred embodiments, the neutral metalloprotease is &amp; variant of the B. amyloliquefaciens neutral metalloprotease. In yet additional embodiments, the neutral metalloprotease is a homoJog of the the 5. amyloliquefaciens neutral metalloprotease. The present invention finds particular use in applications including, but not limited to cleaning, bleaching and disinfecting.<br>
Also as described in more detail in the Examples below, the present invention provides many advantages for cleaning of a wide range of objects, including but not limited to clothing, fabrics, medical devices; etc. In addition, the present invention provides compositions that are effective in.cleaning, bleaching, and disinfecting, over a range of wash temperatures and pHs.<br>
In genera], proteases hydrolyze amide linkages of proteins via addition of a water molecule to the peptide bond(s). Cleavage occurs at the carbonyl-group of the peptide bond.<br>
In bacterial species such as Bacillus, there are two main classes of extracellular proteases namely, alkaline or serine proteases and neutral metalloproteases.<br>
Neutral metalloendopeptidases (i.e., neutral metalloproteases) (EC 3.4.24.4) belong to a protease class that has an absolute requirement for zinc ions for catalytic activity. These enzymes are optimally active at neutral pH and are in the 30 to 40 kDa size range. Neutral metalloproteases bind between two and four calcium ions thai contribute to the structural stability of the protein. The bound metal ion at the active site of metalloproteases is an essential feature that allows the activation of a water molecule. The water molecule then functions as the nucleophile and cleaves the carbonyl group of the peptide bond.<br>
The neutral zinc-binding metalloprotease faraiiy includes the bacterial enzyme thermolysin, and thermolysin-Jike proteases ("TLPs"), as well as carboxypeptidase A (a digestive enzyme), and the matrix metalloproteases that catalyze the reactions in tissue remodeling and degradation. The onjy well characterized of these proteases, with respect to stability and function, is thermolysin and its variants (TLPs). Indeed, much research has been focused on the engineering Bacillus subtilis neutral proteases to increase the thermal stability of the enzyme (See e.g., Vriend et al., In, Tweel et al. (eds), Stability and Stabilization of enzymes. Elsevier, pp. 93-99 [1993]).<br>
Most effort has been focused on increasing the stability of the protease by altering structural determinants identified through the use of molecular modeling suggested to prevent local unfolding processes that would result in autolysis of the protein and cause the neutral protease to denature at high temperatures (See e.g., van den Burg et al., in Hopsu-Havu etal., (eds), Proteolvsis in Cell Functions Manipulating the Autolvtic Pathway of a Bacillus Protease. Biomedical and Health Research Vol. 13, IOS Press [1997] p. 576).<br>
Compositions and methods to engineer neutral metalloproteases with improved characteristics are provided herein. As indicated herein, calcium ions have been reported for other proteases such as thermolysin to prevent autolysis. The B. stearothermophilus neutral protease has been stabilized against autolysis and proteolytic degradation by addition of calcium (See, Durrschmidt et al, FEBS J., 272:1523-1534 [2005]).<br>
Indeed, the present invention provides compositions and methods suitable for the engineering of neutral metalloproteases that are independent of calcium in order to maintain their structural stability. In some embodiments, engineering prevents the local unfolding in a particular secondary structural element that may prevent proteolysis.<br>
Natural and engineered proteases, such as subtilisin are often expressed in Bacillus subtilis and several have been applied in detergent formulations to remove proteinaceous stains. Others have been applied for example in the baking industry (e.g., thermolysin from Bacillus ihennoproteotyiicus; See e.g., Galante and Formantici, Curr. Organic Chem., 7, 1399-1422 [2003]). In general, the serine proteases have been more widely utilized in detergents, at least partially due to the relative ease with which these proteases can be stabilized.<br>
Indeed, metalloproteases are less frequently used in industry, and particularly in the detergent industry for a number of reasons. These enzymes involve more complex protein systems, as the enzymes have the absolute requirement for calcium and zinc ions for stability and function, respectively. Further, the detergent solution as well as the water used in the laundry process often contains components that often interfere with the binding of the ions by the enzyme or chelate these ions, resulting in a decrease or loss of proteolytic function and destabilization of the protease.<br>
In contrast to the currently used metalloprotease enzyme systems, the present invention provides neutral metalloproteases that are sufficiently stabilized to facilitate long-term shelf storage in liquid laundry detergent compositions. In particularly preferred embodiments, the metalloprotease stability and activity are preserved through complexing the enzyme with its obligatory active-site zinc molecule. Importantly, the combination of calcium and zinc ions does not have a deleterious effect on the enzyme's function. In some embodiments, the neutral metalloprotease stabilized is the wild-type metalloprotease from Bacillus amyloHquefaciens (e.g., purified MULTIFECT Neutral; "PMN"). In alternative preferred embodiments, recombinant neutral metalloprotease (e.g., Bacillus amyloHquefaciens neutral metalloprotease cloned into Bacillus subtilis ("nprE")). In additional embodiments, metalloproteases with improved stability encompass enzymes with increased affinity for one or more of the calcium binding sites of the enzyme.. In preferred embodiments, the neutral metalloproteases of the present invention find use in general detergent applications, including but not limited to cold water temperatures, grass stains, and/or low pH conditions.<br>
The present invention provides conditions that stabilize zinc-binding neutral metalloprotease for increased storage stability in detergent bases and/or compositions. In preferred embodiments, the detergent compositions comprise at least one metalloprotease<br>
(e.g., any Bacillus neutral metal loprotease) that is stabilized against autolysis and unfolding, by the inclusion within the detergent formulation of the essential zinc and/or calcium ions. In some particularly preferred embodiments, the neutral metalloprotease from Bacillus amyloliquefaciens (PMN) and the recombinant form expressed in Bacillus subtilis (nprE) that bind zinc ion with 10-fold greater affinity than the calcium ion find use in the present invention. The stabilized protease in the presence of essential zinc ions has improved stability against proteolysis when compared to the same proteases with in the absence of ions.<br>
Although some experimental results indicated that nprE loses some proteolytic activity (~ 20 %) after one hour of adding the detergent base, nprE incubated at 32 C in the presence of zinc ions showed significant stabilization over the test conditions with no additional salts or calcium ions. The presence of both calcium and zinc ions did not show an additive effect. At zinc ion concentrations lower than 15 mM neutral metalloprotease is sufficiently stable over approximately 4 weeks. Thus, the present invention provides compositions comprising the addition of zinc to increase the storage life of neutral metalloprotease in the presence of detergent components.<br>
Furthermore, in alternative embodiments, the zinc cation is replaced with Co2+, Mn2+ or Fe2*, since all of these ions have been shown to bind and restore the protease activity of neutral metalloproteases.' However, it was determined that Mn2+ and Fe2* 'do not restore all of the native activity. While Co2+ restores the highest percentage of the activity, it is apparently less firmly bound than Zn2+. The zinc cation is an essential feature in the active site of all neutral metalloproteases, as it is known to play a role in substrate binding and enzyme catalysis (See e.g., Holmquist and Vallee, J. Biol. Chem., 249:4601-4607 [1974]). The relatively tight affinity of the neutral metalloprotease for the zinc cation (~ |lM range) and the approximately 10-fold greater affinity for this ion relative to calcium, suggest that zinc functions as a stabilizer, thereby preventing autolysis, proteolysis and unfolding. However, it is not intended that the present invention be limited to any particular mechanisms.<br>
The present invention provides extremely beneficial opportunities for application in the production and development of industrial detergents. Many detergents are available with high specificity towards the removal of protein, starch and grease stains. In particular, the better wash performance of PMN or neutral metalloprotease from B. amyloliquefaciens on Equest Grass (Warwick) indicates that the neutral metalloproteases of the present invention in a detergent base that also contains zinc finds use in improved detergent compositions.<br>
Detailed Description of Cleaning and Detergent Formulations of the Present Invention<br>
Unless otherwise noted, all component or composition levels provided herein are made in reference to the active level of that component or composition, and are exclusive of impurities, for example, residual solvents or by-products, which may be present in commercially available sources.<br>
Enzyme components weights are based on total active protein.<br>
All percentages and ratios are calculated by weight unless otherwise indicated. All percentages and ratios are calculated based on the total composition unless otherwise indicated.<br>
In the exemplified detergent compositions, the enzymes levels are expressed by pure enzyme by weight of the total composition and unless otherwise specified, the detergent ingredients are expressed by weight of the total compositions.<br>
Cleaning Compositions Comprising Neutral MetaJloprotease<br>
The stabilized neutral metalloproteases of the present invention are useful in formulating various detergent compositions. The cleaning composition of the present invention may be advantageously employed for example, in laundry applications, hard surface cleaning, automatic dishwashing applications, as well as cosmetic applications such as dentures, teeth, hair and skin. However, due to the unique advantages of increased effectiveness in lower temperature solutions and the superior color-safety profile, the enzymes of the present invention are ideally suited for laundry applications such as the bleaching of fabrics. Furthermore, the enzymes of the present invention find use in both granular and liquid compositions.<br>
The enzymes of the present invention also find use in cleaning additive products.. A cleaning additive product including at least one enzyme of the present invention is ideally suited for inclusion in a wash process when additional bleaching effectiveness is desired. Such instances include, but are not limited to low temperature solution cleaning applications. The additive product may be, in its simplest form, one or more neutral metalloprotease enzyme as provided by the present invention. Li some embodiments, the additive is<br>
packaged in dosage form for addition to a cleaning process where a source of peroxygen is employed and increased bleaching effectiveness is desired. In some embodiments, the single dosage form comprises a pill, tablet, gelcap or other single dosage unit including pre-measured powders and/or liquids. In some embodiments, filler and/or carrier material(s) are included, in order to increase the volume of such composition. Suitable filler or carrier materials include, but are not limited to, various salts of sulfate, carbonate and silicate as well as talc, clay and the like. In some embodiments filler and/or carrier materials for liquid compositions include water and/or low molecular weight primary and secondary alcohols including polyols and diols. Examples of such alcohols include, but are not limited to, methancl, ethanol, propanol and isopropanol. In some embodiments, the compositions comprise from about 5% to about 90% of such materials. In additional embodiments, acidic fillers are used to reduce the pH of the composition. In some alternative embodiments, the cleaning additive includes at least one activated peroxygen source as described below and/or adjunct ingredients as more fully described below.<br>
The cleaning compositions and cleaning additives of the present invention require an effective amount of neutral metalloprotease enzyme as provided in the present invention. In some embodiments, the required level of enzyme is achieved by the addition of one or more species of neutral metalloprotease provided by the present invention.  Typically, the cleaning compositions of the present invention comprise at least 0.0001 weight percent, from about 0.0001 to about 1, from about 0.001 to about 0.5, or even from about 0.01 to about 0.1 weight percent of at least one neutral metalloprotease provided by the present invention.<br>
In some preferred embodiments, the cleaning compositions provided herein are typically formulated such that, during use in aqueous cleaning operations, the wash water has a pH of from about 5.0 to about 11.5, or in alternative embodiments, even from about 6.0 to about 10.5. In some preferred embodiments, liquid product formulations are typically formulated to have a neat pH from about 3;0 to about 9.0, while in some alternative embodiments the formulation has a neat pH from about 3 to about 5. In some preferred embodiments, granular laundry products are typically formulated to have a pH from about 8 to about 11. Techniques for controlling pH at recommended usage levels include the use of buffers, alkalis, acids, etc., and are well known to those skilled in the art.<br>
In some particularly preferred embodiments, when at least one neutral metalloprotease is employed in a granular composition or liquid, the neutral metalloprotease<br>
is in the form of an encapsulated particle to protect the enzyme from other components of the granular composition during storage. In addition, encapsulation also provides a means of controlling the availability of the neutral metal]oprotease(s) during the cleaning process and may enhance performance of the neutral metalloprotease(s). It is contemplated that the encapsulated neutral metalloproteases of the present invention will find use in various settings. It is also intended that the neutral metalloprotease be encapsulated using any suitable encapsulating material(s) and method(s) known in the art.<br>
In some preferred embodiments, the encapsulating material typically encapsulates at least part of the neutral metalloprotease catalyst. In some embodiments, the encapsulating material is water-soluble and/or water-dispersible. In some additional embodiments, the encapsulating material has a glass transition temperature (Tg) of 0C or higher (See e.g., WO 97/11151, particularly from page 6,line 25 to page 7, line 2, for more information regarding glass transition temperatures).<br>
In some embodiments, the encapsulating material is selected from the group consisting of carbohydrates, natural or synthetic gums, chitin and chitosan, cellulose and cellulose derivatives, silicates, phosphates, borates, polyvinyl alcohol, polyethylene glycol, paraffin waxes and combinations thereof. In some embodiments in which the encapsulating material is a carbohydrate, it is selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and combinations thereof. IN some preferred embodiments, the encapsulating material is a starch (See e.g., EP 0 922 499; US 4,977,252. US 5,354,559, and US 5,935,826, for descriptions of some exemplary suitable starches)..<br>
In additional embodiments, the encapsulating material comprises a microsphere made from plastic(e.g., thermoplastics, acrylonitrile, methacrylonitrile, polyacrylonitrile, polymethacrylonitrile and mixtures thereof; commercially available microspheres that find use include, but are not limited to EXPANCEL [Casco Products, Stockholm, Sweden], PM 6545, PM 6550, PM 7220, PM 7228, EXTENDOSPHERES, and Q-CEL [PQ Corp., Valley Forge, PA], LUXSIL and SPHERICELI [Potters Industries, Inc., Carlstadt, NJ and Valley Forge, PA]).<br>
Processes of Making and Using of Applicants' Cleaning Composition<br>
In some preferred embodiments, the cleaning compositions of the present invention are formulated into any suitable form and prepared by any process chosen by the formulator, (See e.g., U.S. 5,879,584, U.S. 5,691,297, U.S. 5,574,005, U.S. 5,569,645, U.S. 5,565,422, U.S. 5,516,448, U.S. 5,489,392, and U.S. 5,486,303, for some non-limiting examples), in some embodiments in which a low pH cleaning composition is desired, the pH of such composition is adjusted via the addition of an acidic material such as HC1.<br>
Adjunct Materials<br>
While not essential for the purposes of the present invention, in some embodiments, the non-limiting list of adjuncts described herein are suitable for use in the cleaning compositions of the present invention. Indeed, in some embodiments, adjuncts are incorporated into the cleaning compositions of the present invention. In some embodiments, adjunct materials assist and/or enhance cleaning performance, treat the substrate to be cleaned, and/or modify the aesthetics of the cleaning composition (e.g., perfumes, colorants, dyes, etc.). It is understood that such adjuncts are in addition to the neutral metalloproteases of the present invention. The precise nature of these additional components, and levels of incorporation thereof, depends on the physical form of the composition and the nature of the cleaning operation for which it is to be used. Suitable adjunct materials include, but are not limited to, surfactants, builders, chelating agents, dye transfer inhibiting agents, deposition aids, dispersants, additional enzymes, and enzyme stabilizers, catalytic materials, bleach activators, bleach boosters, hydrogen peroxide, sources of hydrogen peroxide, preformed peracids, polymeric dispersing agents, clay soil removal/anti-redeposition agents, brighteners, suds suppressors, dyes, perfumes, structure elasticizing agents, fabric softeners, carriers, hydrotropes, processing aids and/or pigments. In addition to those provided explicitly herein, additional examples are known in the art (See e.g., U.S. Patent Nos. 5,576,282,6,306,812 Bl and 6,326,348 Bl). In some embodiments, the aforementioned adjunct ingredients constitute the balance of the cleaning compositions of the present invention.<br>
Surfactants - In some embodiments, the cleaning compositions of the present invention comprise at least one surfactant or surfactant system, wherein the surfactant is selected from nonionic surfactants, anionic surfactants, cation ic surfactants, ampholytic surfactants, zwitterionic surfactants, semi-polar nonionic surfactants, and mixtures thereof.<br>
In some low pH cleaning composition embodiments (e.g., compositions having a neat pH of from about 3 to about 5), the composition typically does not contain alky] ethoxylated sulfate, as it is believed that such surfactant may be hydrolyzed by such compositions the acidic contents.<br>
In some embodiments, the surfactant is present at a level of from about 0.1 % to about 60%, while in alternative embodiments, the level is from about 1 % to about 50%, while in still further embodiments, the level is from about 5% to about 40%, by weight of the cleaning composition.<br>
Builders - In some embodiments, the cleaning compositions of the present invention comprise one or more detergent builders or builder systems. In some embodiments incorporating at least one builder, the cleaning compositions comprise at least about 1%, from about 3% to about 60% or even from about 5% to about 40% builder by weight of the cleaning composition.<br>
Builders include, but are not limited to, the alkali metal, ammonium and alkanolammonium salts of polyphosphaies, alkali metal silicates, alkaline earth and alkali metal carbonates, aluminosilicate builders poJycarboxylate compounds, ether hydroxypolycarboxylates, copolymers of maleic anhydride with ethylene or vinyl methyl ether, 1, 3, 5-trihydroxy benzene-2,4, 6-trisulphonic acid, and carboxymethyloxysuccinic acid, the various alkali metal, ammonium and substituted ammonium salts of polyacetic acids such as ethylenediamine tetraacetic acid and nitrilotriacetic acid, as well as polycarboxylates such as mellitic acid, succinic acid, citric acid, oxydisuccinic acid, polymaleic acid, benzene 1,3,5-tricarboxylic acid, carboxymethyloxysuccinic acid, and soluble salts thereof. Indeed, it is contemplated that any suitable builder will find use in various embodiments of the present invention.<br>
Chela ting Agents - In some embodiments, the cleaning compositions of the present invention contain at least one chelating agent, Suitable chelating agents include, but are not limited to copper, iron and/or manganese chelating agents and mixtures thereof. In embodiments in which at least one chelating agent is used, the cleaning compositions of the present invention comprise from about 0.1% to about 15% or even from about 3.0% to about 10% chelating agent by weight of the subject cleaning composition.<br>
Deposition Aid - In some embodiments, the cleaning compositions of the present invention include at least one deposition aid. Suitable deposition aids include, but are not limited to polyethylene glycol, polypropylene glycpl, polycarboxylate, soil release polymers such as polytelephthalic acid, clays such as kaolinite, montmorillonite, atapulgite, illite, bentonite, halloysite, and mixtures thereof.<br>
Pve Transfer Inhibiting Agents - In some embodiments, the cleaning compositions of the present invention include one or more dye transfer inhibiting agents. .Suitable polymeric dye transfer inhibiting agents include, but are not limited to, polyvinylpyrrolidone polymers, polyamine N-oxide polymers, copolymers of N-vinylpyrrolidone and N-vinylimidazole, polyvinyloxazolidones and polyvinylimidazoles or mixtures thereof.<br>
In embodiments in which at least one dye transfer inhibiting agent is used, the cleaning compositions of the present invention comprise from about 0.0001% to about 10%, from about 0.01% to about 5%, or even from about 0.1 % to about 3% by weight of the cleaning composition.   Dispcrsants - In some embodiments, the cleaning compositions of the present invention contains at least one dispersants. Suitable water-soluble organic materials include, but are not limited to the homo- or co-polymeric acids or their salts, in which the polycarboxylic acid comprises at least two carboxyl radicals separated from each other by not more than two carbon atoms.<br>
Enzymes - In some embodiments, the cleaning compositions of the present invention . comprise one or more detergent enzymes which provide cleaning performance and/or fabric care benefits. Examples of suitable enzymes include, but are not limited to, hemicellulases, peroxidases, proteases, cellulases, xylanases, lipases, phospholipases, esterases, cutinases, pectinases, keratinases, reductases, oxidases, phenoloxidases, lipoxygenases, ligninases, . pullulanases, tannases, pentosanases, malanases, 6-glucanases, arabinosidases, hyaluronidase, chondroitinase, laccase, and amylases, or mixtures thereof. In some embodiments, a combination of enzymes is used (i.e., a "cocktail") comprising conventional applicable enzymes like protease, lipase, cutinase and/or cellulase in conjunction with amylase is used.<br>
Enzyme Stabilizers - In some embodiments of the present invention, the enzymes used in the detergent formulations of the present invention are stabilized. It is contemplated that various techniques for enzyme stabilization will find use in the present invention. For example, in some embodiments, the enzymes employed herein are stabilized by the presence<br>
of water-soluble sources of zinc (II), calcium (II) and/or magnesium (II) ions in the finished compositions that provide such ions to the enzymes, as well as. other metal ions (e.g., barium (II), scandium (II), iron (II), manganese (II), aluminum (IS), Tin (II), cobalt (II), copper (II), Nickel (II), and oxovanadium (IV)).<br>
Catalytic Metal Complexes  In some embodiments, the cleaning compositions of the present invention contain one or more catalytic metal complexes. In some embodiments, a metal-containing bleach catalyst finds use. In some preferred embodiments, the metal bleach catalyst comprises a catalyst system comprising a transition metal cation of defined bleach catalytic activity, (e.g., copper, iron, titanium, ruthenium, tungsten, molybdenum, or manganese cations), an auxiliary metal cation having little or no bleach catalytic activity (f.g., zinc or aluminum cations), and a sequestrate having defined stability constants for the catalytic and auxiliary metal cations, particularly ethylenediaminetetraacetic acid, ethylenediaminetetra (methylenephosphonic acid) and water-soluble salts thereof are used (.Seee.g.,U.S. 4,430,243).<br>
In some embodiments, the cleaning compositions of the present invention are catalyzed by means of a manganese compound. Such compounds and levels of use are well known in the art (See e.g., U.S. 5,576,282).<br>
In additional embodiments, cobalt bleach catalysts find use in the cleaning compositions of the present invention. Various cobalt bleach catalysts are known in the art (See e.g., U.S. 5,597,936, and U.S. 5,595,967). Such cobalt catalysts are readily prepared by known procedures (See e.g., U.S. 5,597,936, and U.S. 5,595,967).<br>
In additional embodiments, the cleaning compositions of the present invention include a transition metal complex of a macropolycyclic rigid ligand ("MKL"). As a practical matter, and not by way of limitation, in some embodiments, the compositions and cleaning processes provided by the present invention are adjusted to provide on the order of at least one part per hundred million of the active MRL species in the aqueous washing medium, and in some preferred embodiments, provide from about 0.005 ppm to about 25 ppm, more preferably from about 0.05 ppm to about 10 ppm, and most preferably from about 0.1 ppm to about 5 ppm, of the MRL in the wash liquor.<br>
Preferred transition-metals in the instant transition-metal bleach catalyst include, but are not limited to manganese, iron and chromium. Preferred MRLs also include, but are not limited to special ultra-rigid ligands that are cross-bridged (e.g., 5,12-diethyl-l,5(8,12-<br>
tetraazabicyclo[6.6.2]hexadecane). Suitable transition metal MRLs are readily prepared by known procedures (See e.g., WO 00/32601, and U.S. 6,225,464).<br>
Processes of Making and Using Cleaning Compositions<br>
The cleaning compositions of the present invention are formulated into any suitable form and prepared by any suitable process chosen by the formulator, (See e.g., U.S. 5,879,584, U.S. 5,691,297, U.S. 5,574,005, U.S. 5,569,645, U.S. 5,565,422, U.S. 5,516,448, U.S. 5,489,392, U.S. 5,486,303, U.S. 4,515,705, U.S. 4,537,706, U.S. 4,515,707, U.S. 4,550,862, U.S. 4,561,998, U.S. 4,597,898, U.S. 4,968,451, U.S. 5,565,145, U.S. 5,929,022, U.S. 6,294,514, and U.S. 6,376,445, all of which are incorporated herein by reference for some non-limiting examples).<br>
Method of Use<br>
In preferred embodiments, the cleaning compositions of the present invention find use in cleaning surfaces and/or fabrics.  In some embodiments, at least a portion of the surface and/or fabric is contacted with at least one embodiment of the cleaning compositions of the present invention, in neat form or diluted in a wash liquor, and then the surface and/or fabric is optionally washed and/or rinsed. For purposes of the present invention, "washing" includes, but is not limited to, scrubbing, and mechanical agitation. In some embodiments, the fabric comprises any fabric capable of being laundered in normal consumer use conditions. In preferred embodiments, the cleaning compositions of the present invention are used at concentrations of from about 500 ppm to about 15,000 ppm in solution. In some embodiments in which the wash solvent is water, the water temperature typically ranges from about 5 C to about 90 C. In some preferred embodiments for fabric cleaning, the water to fabric mass ratio is typically from about 1:1 to about 30:1.<br>
EXPERIMENTAL<br>
The following examples are provided in order to demonstrate and further illustrate certain preferred embodiments and aspects of the present invention and are not to be construed as limiting the scope thereof.<br>
In the experimental disclosure which follows, the following abbreviations apply: C (degrees Centigrade); rpm (revolutions per minute); H2O (water); HC1 (hydrochloric acid); aa<br>
and AA (amino acid); bp (base pair); kb (kilobase pair); kD (kilodaltons); gm (grams); u.g and ug (micrograms); mg (milligrams); ng (nanograms); 1 and ul (microliters); ml (milliliters); mm (millimeters); nm (nanometers); urn and um (micrometer); M (molar); mM (millimolar); M and uM (micromolar); U (units); V (volts); MW (molecular weight); sec (seconds); min(s) (minute/minutes); hr(s) (hour/hours); MgCl2 (magnesium chloride); NaCl (sodium chloride); OD280 (optical density at 280 nm); OD405 (optical density at 405 nm); ODff (optical density at 600 nm); PAGE (polyacrylamide gel electrophoresis); EtOH (ethanol); PBS (phosphate buffered saline [150 mM NaCl, 10 mM sodium phosphate buffer, pH 7.2]); LAS (lauryl sodium sulfonate); SDS (sodium dodecyl sulfate); Tris . (tris(hydroxymethyl)aminomethane); TAED (N.N.N'N'-tetraacetylethylenediamine); BES (polyesstersulfone); MES (2-morpholinoethanesulfonic acid, monohydrate; f.w. 195.24; Sigma # M-3671); CaCl2 (calcium chloride, anhydrous; f.w. 110.99; Sigma # C-4901); DMF (N,N-dimethylformamide, f.w. 73.09, d = 0.95); Abz-AGLA-Nba (2-Aminobenzoyl-L-alanylglycyl-L-leucyl-L-alanino-4-nitrobenzylamide, f.w. 583.65; Bachem # H-6675, VWR catalog # 100040-598); SBG1% ("Super Broth with Glucose"; 6 g Soytone [DifcoJ, 3 g yeast extract, 6 g NaCl, 6 g glucose); the pH was adjusted to 7.1 with NaOH prior to sterilization using methods known in the art; w/v (weight to volume); v/v (volume to volume); Npr and npr (neutral metalloprotease); SEQUEST (SEQUEST database search program, University of Washington); Npr and npr (neutral metalloprotease gene); nprE and NprE (B. amyloliquefaciens neutral metalloprotease); PMN (purified MULTIFECT metalloprotease); MS (mass spectroscopy); SRI (Stain Removal Index); TIGR (The Institute for Genomic Research, Rockville, MD); AATCC (American Association of Textile and Coloring Chemists); Amersham (Amersham Life Science, Inc. Arlington Heights, IL); Corning (Corning Internationa], Corning, NY); ICN (ICN Pharmaceuticals, Inc., Costa Mesa, CA); Pierce (Pierce Biotechnology, Rockford, IL); Equest (Equest, Warwick International Group, Inc., Flintshire, UK); EMPA (Eidgenossische Material Prufungs und Versuch Anstalt, St. Gallen, Switzerland); CFT (Center for Test Materials, Vlaardingen, The Netherlands); Amicon (Amicon, Inc., Beverly, MA); ATCC (American Type Culture Collection, Manassas, VA); Becton Dickinson (Becton Dickinson Labware, Lincoln Park, NJ); Perkin-Elmer (Perkin-Elmer, Wellesley, MA); Rainin (Rainin Instrument, LLC, Wobum, MA); Eppendorf (Eppendorf AG, Hamburg, Germany); Waters (Waters, Inc., Milford, MA); Geneart (Geneart GmbH, Regensburg, Germany); Perseptive Biosystems (Perseptive Biosystems, Ramsey,<br>
MM); Molecular Probes (Molecular Probes, Eugene, OR); BioRad (BioRad, Richmond, CA); Clontech (CLONTECH Laboratories, Palo Alto, CA); Cargill (Cargill, Inc., Minneapolis, MN); Difco (Difco Laboratories, Detroit, MI); GIBCO BRL or Gibco BRL (Life Technologies, Inc., Gaithersburg, MD); New Brunswick (New Brunswick Scientific Company, Inc., Edison, NJ); Thermoelectron (Thermoelectron Corp., Waltham, MA); BMG (BMG Labtech, GmbH, Offenburg, Germany);Greiner (Greiner Bio-One, Kremsmuenster, Austria); Novagen (Novagen, Inc., Madison, WI); Novex (Novex, San Diego, CA); Finnzymes (Finnzymes OY, Finland) Qiagen (Qiagen, Inc., Valencia, CA); Invitrogen (Invitrogeri Corp., Carlsbad, CA); Sigma (Sigma Chemical Co., St. Louis, MO); DuPont Instruments (Asheville, NY); Global Medical Instrumentation or GMI (Global Medical Instrumentation; Ramsey, MN); MJ Research (MJ Research, Waltham, MA); Infers (Infers AG, Bottmingen, Switzerland); Stratagene (Stratagene Cloning Systems, La Jolla, CA); Roche (Hoffmann La Roche, Inc., Nutley, NJ); Agilent (Agilent Technologies, Palo Alto, CA); S-Matrix (S-Matrix Corp., Eureka, CA); US Testing (United States Testing Co., Hoboken, NY); West Coast Analytical Services (West Coast Analytical Services, Inc., Santa Fe Springs, CA); Ion Beam Analysis Laboratory (Ion Bean Analysis Laboratory, The University of Surrey Ion Beam Centre (Guildford, UK); TOM (Terg-o-Meter); BMI (blood, milk, ink); BaChem (BaChem AG, Bubendorf, Switzerland); Molecular Devices (Molecular Devices, Inc., Sunnyvale, CA); Corning (Coming International,.Corning, NY); MicroCal (Microcal, Inc., Northhampton, MA); Chemical Computing (Chemical Computing Corp., Montreal, Canada); NCBI (National Center for Biotechnology Information); Argo Bioanalytica (Argo Bioanalytica. Inc, New Jersey); Vydac (Grace Vydac, Hesperia, CA); Minolta (Konica Minolta, Ramsey, NJ); and Zeiss (Carl Zeiss, Inc., Thomwood, NY).<br>
In these experiments, a spectrophotometer was used to measure the absorbance of the products formed after the completion of the reactions. A reflectometer was used to measure the reflectance of the swatches. Unless otherwise indicated, protein concentrations were estimated by Coomassie Plus (Pierce), using BSA as the standard.<br>
The following assays were used in the Examples described below.<br>
A.        Bradford Assay for Protein Content Determination in 96-well Microtiter Plates (MTPs)<br>
In these assays, the Bradford dye reagent (Quick Start) assay was used to determine the protein concentration in NprE protease samples on MTP scale. In this assay system, the chemical and reagent solutions used were:<br>
Quick Start Bradford Dye Reagent         BIO-RAD, #500-0205<br>
Dilution buffer	l0mM NaCl, 0.lmM CaC12, 0.005% TWEEN-80<br>
The equipment used was a Biomek FX Robot (Beckman) and a SpectraMAX (type 340) MTP reader; the MTPs were from Costar (type 9017).<br>
In the test, 200 l Bradford Dye Reagent was pipetted into each well, followed by 15 l dilution buffer. Finally 10 l of filtered culture broth were added to the wells. After thorough mixing, the MTPs were incubated for at least 10 minutes at room temperature; Possible air bubbles were blown away and the ODs of the wells were read at 595 run.<br>
To determine the protein concentration, the background reading (i.e., from uninoculated wells) was subtracted form the sample readings. The obtained OD39J values provide a relative measure of the protein content in the samples. The linearity of the NprE calibration lines between 0 to 5 ug enabled the use of OD595 nm values as a relative measure for the protein content. As the expected content of NprE in supernatant was 200-300 g/ml, the l0l sample volume used in the test contains less than 5 ug protein, providing values in the linear range.<br>
B,	Microswatch Assay for Testing Protease Performance<br>
The detergents used in this assay did not contain enzymes. The equipment used was a Biomek FX Robot (Beckman) and a SpectraMAX (type 340) MTP reader; the MTPs were from Costar (type 9017).<br>
Detergent Preparation (cold water liquid detergent; US conditions):<br>
Milli-Q water was adjusted to 6 gpg water hardness (Ca/Mg=3/l), and 0.78 g/1 TIDE 2007-2x detergent was added. The detergent solution was vigorously stirred for at least 15 minutes. Then. 5 mM HEPES (free acid) was added and the pH adjusted to 8.2.<br>
Microswatches<br>
Microswatches of W circular diameter were obtained from CFT. Before cutting of the swatches, the fabric (EMPA 116) was washed with water. Two microswatches were placed in each well of a 96-well microtiter plate vertically to expose the whole surface area (/.., not flat on the bottom of the well);<br>
Test Method<br>
The incubator was set to 20"C. The filtered culture broth samples were tested at an appropriate concentration by dilution with a mixture of 10 mM NaCl, 0.1 mM CaC12 and 0.005% TWEEN-80 solution. The desired detergent solution was prepared as described above. Then, 190 l of detergent solution was added to each well of the MTP, containing microswatches. To this mixture, 10 1 of the diluted enzyme solution were added (to provide a total volume of 200 l/well). The MTP was sealed with tape and placed in the incubator for 30 minutes, with agitation at 1400 rpm. Following incubation under the appropriate conditions, 100 l of solution from each well were removed and placed into a fresh MTP. The new MTP containing 100 l of solution/well was read at 405 nm in a MTP reader. Blank controls, as well as a control containing two microswatches and detergent but no enzyme were also included.<br>
Calculation of the BM3 Performance:<br>
The obtained absorbance value was corrected for the blank value (i.e., obtained after incubation of microswatches in the absence of enzyme). The resulting absorbance was a measure for the hydrolytic activity. For each sample (e.g., nprE or variant) the performance    index was calculated. The performance index compared the performance of the variant (actual value) and the standard enzyme (theoretical value) at the same protein concentration. In addition, the theoretical values were calculated, using the parameters of the Langmuir equation of the standard enzyme. A performance index (PI) that is greater than 1 (PI&gt;1) identified a better variant (as compared to the standard [e.g., wild-type]), while a PI of 1<br>
 (PI=1) identified a variant that performed the same as the standard, and a PI that is less than 1 (PI
C.   Citrate Stability Assay for NprE protease.<br>
Citrate stability was measured after incubation of wild-type NprE and variants in the presence of 50 mM citrate. The initial and residua! activity was determined using the DMC hydrolysis assay. In this assay system, the chemical and reagent solutions used were:<br>
Citric acid monohydrate	Merck 1.00244<br>
Pipes (free acid)	Sigma P-1851<br>
Tris (free acid)	Sigma T-1378<br>
HEPES (Ultra&gt;99.5%)	Sigma-H7523<br>
TWEEN-80	Sigma P-8074<br>
Diroethylcasein(DMC)	Sigma C-9801<br>
Tris buffer (free acid)	6.04 g dissolved in 1000 ml water (= 50 mM)<br>
HEPES buffer	11.9 g. dissolved in 1000 ml water (= 50 mM)<br>
Citrate buffer (free acid)	21.0 g. dissolved in 1000 ml water (= 100 mM),<br>
PIPES buffer (free acid):	3.32 g dissolved in about 960 ml water,<br>
DMC solution	1 % w/v in 55 mM PIPES buffer, final pH = 6.0<br>
Dilution buffer 1	0.1 mM CaC12/25 mM Tris; pH 8.2<br>
Dilution buffer 2	0.1 mM CaCJ2/50 mM Citrate/25 mM Tris; pH82<br>
The concentrations of these dilution buffers are indicated as final concentrations. The initial concentration was proportionally higher and dependent on the dilution rate. The initial concentration was proportionally higher and dependent on the dilution rate. In alternative experiments, HEPES finds use in exchange for Tris. The equipment used was a Biomek FX Robot (Beckman), and an incubator/shaker (Innova, type 4230; New Brunswick). The PIPES buffer was adjusted to pH 5.8 with 4 N HC1 (final concentration of 55 mM). The Tris buffer was adjusted to pH 8.2 with 4 N HC1 (final concentration of 25 mM). The 50 mM citrate/25 mM Tris buffer was adjusted to pH 8.2 with 4 N NaOH. The HEPES buffer was<br>
adjusted to pH 8.2 with 4 N NaOH (final concentration of 25 mM). The 50 mM citrate/25 mM HEPES buffer was adjusted to pH 8.2 with 4 N NaOH.<br>
Protein Determination<br>
In order to establish the desired dilution rate in the citrate stability assay the protease concentration of the wild-type NprE controls for each plate were determined with the TCA assay. In this method, 25 ul filtered culture broth were added to 200 ul 16.875% (w/v) TCA. After incubation for 10 to 15 minutes at ambient temperature, the light scattering/absorbance at 405 rim was determined. The protein concentration was determined by using a calibration line, constructed with purified NprE.<br>
Test Method<br>
The dilution rate of the filtered culture broth was determined using the TCA assay, as described above.<br>
Stressed Conditions:<br>
The filtered culture broth was diluted with dilution buffer 2. The MTP was covered with tape, shaken for a few seconds and placed in the incubator at 25C for 60 minutes at 200 rpm. After incubation, 20l of the mixture were taken from each well and transferred into a new MTP, containing 180 l 1% DMC preheated substrate solution (the substrate was preheated at 25C). The MTP was placed directly in the incubator/shaker and incubated at 25C for 30 minutes at 200 rpm agitation. The residual protease activity was determined using the dimethylcasein hydrolysis assay, described below.<br>
Unstressed Conditions<br>
The filtered culture broth was diluted with dilution buffer 1. Immediately, 20l of the mixture were taken from each well and transferred into a new MTP, containing 180 l of preheated 1% DMC substrate solution (the substrate was preheated at 25C). The MTP was placed directly in the incubator/shaker and incubated for 25C for 30 minutes at 200 rpm agitation. The initial protease activity as determined with TNBS, using the dimethylcasein hydrolysis assay, described below.<br>
All residual activity values (determined with the dimethylcasein hydrolysis assay) were calculated using the following equation.<br>
% Residual Activity = OD60 min value * 1 00/ODoo min value<br>
D.    Dimethvlcasein Hydrolysis Assay<br>
In this assay system, the chemicals and reagent solutions used were:<br>
Dimethylcasein (DMC)	Sigma C-9801<br>
TWEEN-80	Sigma P-8074<br>
PIPES buffer (free acid)	Sigma P-l 851; 15.1 g dissolved in about 960 ml<br>
water; pH adjusted to 6.0 with 4N NaOH, 1 ml of 5% TWEEN-80 added and the volume<br>
brought	up to 1000 ml. Final concentration<br>
of PIPES and	TWEEN-80: 50 mM and 0.005%<br>
respectively. PicryJsulfonic acid (TNBS)       Sigma P-2297 (5% solution in water)<br>
Reagent A	45.4 g Na2B4O7.10 H2O (Merck 6308) and<br>
15 ml of 4N NaOH dissolved together to a<br>
final	volume of 1000 ml (by heating if<br>
needed)<br>
Reagent B	35.2 g NaH2PO4.lH2O (Merck 6346) and 0.6g<br>
Na2SOs (Merck 6657) dissolved together to a final volume of 1000 ml.<br>
Method<br>
To prepare the substrate, 4 g dimethylcasein was dissolved in 400 ml PIPES buffer. The filtered culture supernatants were diluted with PIPES buffer. Then, 10 pi of each diluted supernatant were added to 200 pi substrate in the wells of a MTP. The MTP was covered with tape, shaken for a few seconds and placed in an oven at 25C for 30 minutes without agitation. About 15 minutes before removal of the 1" plate from the oven, the TNBS reagent<br>
was prepared by mixing 1 ml TNBS solution per 50 ml of Reagent A. MTPs were filled with 60 ul TNBS Reagent A per well. The incubated plates were shaken for a few seconds, after which 10 pi was transferred to the MTPs with TNBS Reagent A. The plates were covered with tape and shaken for 20 minutes in a bench shaker (BMG Thermostar) at room temperature and 500 rpm. Finally, 200 ul Reagent B was added to the wells, mixed for 1 minute on a shaker, and the absorbance at 405 nm was determined using a MTP reader. The obtained absorbance value was corrected for the blank value (Le., substrate without enzyme). The resulting absorbance was a measure of the hydrblytic activity. The (arbitrary) specific activity of a sample was calculated by dividing the absorbance and the determined protein concentration.<br>
E.    TIDE Stability Assay<br>
The stability of NprE and variants was measured after an incubation step in the presence of 25% TIDE compact detergent. The initial and residual activity was determined using the AGLA-assay described below. The equipment used was a Biomek FX Robot (Beckman), a fluorescence meter (FLUOstar Optima; BMG), an incubator/shaker (iEMS; Thermoelectron) and an incubator/shaker (Innova; New Brunswick (type 4230)); the MTPs were from Costar (type 9017) and from Greiner (black plates, type 655076).<br>
Chemicals and reagents:<br>
In this assay system, the chemical and reagent solutions used were:<br>
TIDE-compact detergent	With and without DTPA<br>
TIDE-compact detergent solution        125 g TIDE-compact dissolved in a mixture of 50<br>
g	of 50 mM HEPES pH 8.2 and 275 ml water;<br>
concentration of TIDE was 27.7%,<br>
after dilution	with supernatant 25 %<br>
MES dilution buffer	52.6 mM MES/NaOH, 2.6 mM CaCI2,0.005%<br>
TWEEN-80,pH6.5<br>
AGLA substrate	BaChem, cat no. H-6675 or American Peptide<br>
Company, cat no. 81-0-31<br>
GC889-2-PCT<br>
AGLA substrate solution	451 mg of AGLA dissolved in 16 ml N,N-<br>
dimethylformamide; this solution was<br>
poured into	304 ml of MES-buffer<br>
(52.6 mM MES/NaOH, 2.6	mM CaCl2,<br>
0.005% TWEEN-80, pH 6.5) with stirring<br>
Test method:<br>
Unstressed conditions:<br>
First, 20 pi filtered culture broth was diluted with 180 pi MES dilution buffer. Then, 20 pi of this diluted broth was diluted with 180 pi MES dilution buffer. Then, 10 pi of this dilution was diluted with 190 l AGLA-substrate solution in a pre-warmed plate at 25C. Any air bubbles present were blown away and the plate was measured according to the AGLA protease assay protocol.<br>
Stressed conditions:<br>
First, 20 pi filtered culture broth was diluted with 180 pi TIDE-compact detergent solution without DTPA and after premising in the iEMS shaker for 5 minutes, were incubated further in the Innova shaker. The plate was incubated for a total of 60 minutes at 32C, at 200 rpm. In addition, 20 ul filtered culture broth were diluted with 180 ul TIDE-compact detergent solution with DTPA and after premising in the iEMS shaker for 5 minutes, were incubated further in the Innova shaker. The plate was incubated for a total of 40 minutes at 20C, at 200 rpm. Then, 20 pi of either of these solutions were diluted with 180 pi MES dilution buffer and 10 pi of this dilution were diluted with 190 pi AGLA-substrate solution in a pre-warmed plate at 25C. Any air bubbles present were blown away and the plate was measured according to the AGLA protease assay protocol. .<br>
Calculations:<br>
Fluorescence measurements were taken at excitation of 350 nm and emission of 415 nm. The spectrofluorometer software calculated the reaction rates of the increase in fluorescence for each well to a linearly regressed line of milli-RFU / min:<br>
Percentage of residual activity:  (Slope of stressed condition) * 100<br>
(Slope of unstressed condition)<br>
F.    2-Aminobcnzoyl-L-alanylglycy]-L-leucyl-L-alanino-4-nitrobenzylamide Protease Assay ( Ab2-AGLA-Nba)<br>
The method provided below provides a degree of technical detail that yields reproducible protease assay data independent of time and place. While the assay can be adapted to a given laboratory condition, any data obtained through a modified procedure must be reconciled with results produced by the original method. Neutral metalloproteases cleave the peptide bond between glycine and leucine of 2-aminobenzoyl-L-alanylglycYl-L-leucyl-L-alanino-4-nitrobenzylamide (Abz-AGLA-Nba). Free 2-aminobenzoyl-L-alanylglycine (Abz-AG) in solution has a fluorescence emission maximum at 415 nm with an excitation maximum of 340 nm. Fluorescence of Abz-AG is quenched by nitrobenzylamide in the intact Abz-AGLA-Nba molecule.<br>
In these experiments, the liberation of Abz-AG by protease cleavage of Abz-AGLA-Nba was monitored by fluorescence spectroscopy (Ex. 340 / Em. 415). The rate of appearance of Abz-AG was a measure of proteolytic activity. Assays were performed under non-substrate limited initial rate conditions.<br>
A microplate mixer with temperature control (e.g., Eppendorf Thermomixer) was required for reproducible assay results. The assay solutions were incubated to desired temperature (e.g., 25C) in the microplate mixer prior to enzyme addition. Enzyme solutions were added to the plate in the mixer, mixed vigorously and rapidly transferred to the plate reader.<br>
A spectrofluorometer with capability of continuous data recording, linear regression analysis, and with temperature control was required (e.g., SpectraMax M5, Gemini EM, Molecular Devices). The reader was always maintained at the desired temperature (e.g., 25C). The reader was set for top-read fluorescence detection and the excitation was set to 350 nm and emission to 415 nm without the use of a cut-off filter. The PMT was set to medium sensitivity and 5 readings per well. Autocalibration was turned on, but only to<br>
calibrate before the first reading. The assay was measured for 3 minutes with the reading interval minimized according to the number of wells selected to be monitored. The reader was set to calculate the rate of milli-RFU/min (thousandths of relative fluorescence units per minute). The number of readings used to calculate the rate (Vmax points) was set to the number equivalent to 2 minutes, as determined by the reading interval (e.g., a reading every 10 seconds would use 12 points to calculate the rate). The max RFU was set to 50,000.<br>
All pipeting of enzyme and substrate stock solutions were done with positive displacement pipets (Rainin Microman). Buffer, assay, and enzyme working solutions were pipetted by single or multi-channel air-displacement pipets (Rainin LTS) from tubes, reagent reservoirs or stock microplates. A repeater pipet (Eppendorf) finds use iri transferring the assay solution to microplate wells when few wells are used, to minimize reagent loss. Automated pipetting instruments such as the Beckman FX or Cybio Cybi-well also find use in transferring enzyme solutions from a working stock microplate to the assay microplate in order to initiate an entire microplate at once.<br>
Reagents and Solutions:<br>
52.6 mMMES/NaOH, 2.6 mMCaCI2,pH 6.5  -  MES Buffer<br>
MES acid (10.28 g) and 292 mg anhydrous CaCl2 were dissolved in approximately 900mL purified water. The solution was titrated with NaOH to pH 6.5 (at 25C or with temperature adjustment pH probe). The pH-adjusted buffer was made up to 1L total volume. The final solution was filtered through a 0.22 Jim sterile filter and kept at room temperature.<br>
48 mM Abz-AGLA-Nba in DMF  -  Abz-AGLA-Nha Stock<br>
Approximately 28 mg of Abz-AGLA-Nba was placed in a small tube. It was dissolved in mL of DMF (volume will vary depending upon Abz-AGLA-Nba massed) and vortexed for several minutes. The solution was stored at room temperature shielded from light<br>
50 mM MES, 23 mM CaCl2, 5% DMF, 2.4 mM Abz-AGLA-Nba pH 6.5  -  Assay Solution<br>
One mL Abz-AGLA-Nba stock was added to 19 mLMES Buffer and vortexed. The solution was stored at room temperature shielded from light.<br>
50 mM MES, 2.5 mM CaCl2, pH 6.5  -  Enzyme Dilution Buffer<br>
This buffer was produced by adding 5 mL purified water to 95 mL MES Buffer.<br>
50 mM MES, 2.5 mM CaCl2,5% DMF, pH 6.5 - Substrate Dilution Buffer<br>
Five mL pure DMF were added to 95 mL MES Buffer. This buffer was used to determine kinetic parameters.<br>
Enzyme solutions<br>
The enzyme stock solutions were diluted with enzyme dilution buffer to a concentration of approximately 1 ppm (1 ug/mL). MULTIEECT neutral protease (wild-type NprE) was diluted to concentrations below 6 ppm (6 ug/mL). Serial dilutions were preferred. Solutions were stable at room temperature for 1 hour, but for longer term storage, the solutions were maintained on ice.<br>
Procedure<br>
Fist all buffers, stock, and working solutions were prepared. Each enzyme dilution was assayed in triplicate, unless otherwise indicated. When not completely full, the enzyme working solution stock microplate was arranged in full vertical columns starting from the left of the plate (to accommodate the plate reader). The corresponding assay plate was similarly set up. The microplate spectrofluorometer was set up as previously described.<br>
First, a 200 uL aliquot of assay solution were placed in the wells of a 96-well microplate. The plate was incubated for 10 min at 25C in a temperature controlled microplate mixer, shielded from light. The assay was initiated by transferring 10 uL of the working enzyme solutions from the stock microplate to the assay microplate in the mixer. Optimally, 96-well pipetting head finds use, or an 8-well multi-channel pipet was used to transfer from the left-most column first. The solutions were vigorously mixed for 15 seconds (900rpm in Eppendorf Thermomixer). Immediately, the assay microplate was transferred to the microplate spectrofluorometer and recording of fluorescence measurements at excitation of 350 nm and emission of 415 nm were begun. The spectrofluorometer software calculated<br>
the reaction rates of the increase in fluorescence for each well to a linearly regressed line of milli-RFU / min. In some experiments, a second plate was placed in the microplate mixer for temperature equilibration while the first plate was being read.<br>
The rate initial velocities were linear with respect to product concentration (i.e., liberated 2-aminobenzoyl fluorescence) up to 0.3 mM product, which corresponded to approximately 50,000 RFU in a solution starting at 2.3mM Abz-AGLA-Nba with background fluorescence of approximately 22,000 RFU. Abz-AGLA-Nba was dissolved in DMF and was been used  the day it was prepared.<br>
DETERGENT COMPOSITIONS:<br>
In the exemplified detergent compositions, the enzymes levels are expressed by pure   . enzyme by weight of the total composition and unless otherwise specified, the detergent ingredients are expressed by weight of the total compositions. The abbreviated component identifications therein have the following meanings:<br>
Abbreviation	Ingredient<br>
LAS	:   Sodium linear C11-13 alkyl benzene sulfonate.<br>
NaC16-17HSAS	:	Sodium C16-17 highly soluble alkyl sulfate<br>
TAS	:	Sodium tallow alkyl sulphate.<br>
CxyAS	:	Sodium C1x - C1y alkyl sulfate.<br>
CxyEz	:	C1x - C1y predominantly linear primary alcohol condensed with an<br>
average of z moles of ethylene oxide.<br>
CxyAEzS	:   C1x - C1y sodium alkyl sulfate condensed with an average of z<br>
moles of ethylene oxide. Added molecule name in the examples.<br>
Nonionic	:   Mixed ethoxylated/propoxylated fatty alcohol e.g. Plurafac LF404<br>
being an alcohol with an average degree of ethoxylation of 3.8 and an average degree of propoxylation of 4.5.<br>
QAS	:	R2N+CH3)2(C2H4OH) with R2 = C12-C14.<br>
Silicate	:	Amorphous Sodium Silicate (SiO2'.Na2O ratio = 1.6-3.2:1).<br>
Metasilicate	:	Sodium metasilicate (SiO2:Na2O ratio = 1.0).<br>
Zeolite A	:	Hydrated Aluminosilicate of formula Naj 2(A102SiO2)j2- 27H2O<br>
SKS-6	Crystalline layered silicate of formula 6-Na2Si2O5-<br>
Sulfate	Anhydrous sodium sulphate.<br>
STPP	Sodium Tripolyphosphate.<br>
MA/AA	Random copolymer of 4:1 acrylate/maleate, average molecular<br>
weight about 70,000-80,000.<br>
AA	Sodium polyacrylate polymer of average molecular weight 4,500.<br>
Polycarboxylate	Copolymer comprising mixture of carboxylated monomers such as<br>
acrylate, maleate and methyacrylate with a MW ranging between 2,000-80,000 such as Sokolan commercially available from BASF, being a copolymer of acrylic acid, MW4.500.<br>
BB1	3-(3,4-Dihydroisoquinolinium)propane sulfonate<br>
BB2	1 -(3,4-dihydroisoquinolinium)-decane-2-sulfate<br>
PB1	Sodium perborate monohydrate.<br>
PB4	Sodium perborate tetrahydrate of nominal formula NaBO3.4H2O. <br>
Percarbonate	Sodium percarbonate of nominal formula 2Na2C(&gt;3.3H2O2 .<br>
TAED	Tetraacetyl ethylene diamine.<br>
NOBS    	Nonanoyloxybenzene sulfonate in the form of the sodium salt.<br>
DTPA	Diethylenetriamine-pentaaceticacid.<br>
HEDP	1,1 -hydroxyethane diphosphonic acid.<br>
DETPMP	Diethyltriamine penta (methylene) phosphonate, marketed by<br>
Monsanto under the Trade name Dequest 2060.<br>
HDDS	Ethylenediamine-N,N'-disuccinic acid, (S,S) isomer in the form of<br>
its sodium salt<br>
Diamine	Dimethyl aminopropyl amine; 1,6-hezane diamine; 1,3-propane<br>
diamine; 2-methyl-l,5-pentane diamine; 1,3-pentanediamine; 1-<br>
methyl-diaminopropane.<br>
. DETBCHD	5,12-diethyl-1,5,8,12-tetraazabicyclo [6,6,2] hexadecane,<br>
dichloride, Mn(II) SALT<br>
PAAC	:   Pentaamine acetate cobaltGH) salt.<br>
Paraffin	:   Paraffin oil sold under the tradename Winog 70 by Wintershall.<br>
Paraffin Sulfonate	:   A Paraffin oil or wax in which some of the hydrogen atoms have<br>
been replaced by sulfonate groups.<br>
Aldose oxidase	:   Oxidase enzyme sold under the tradename Aldose Oxidase by<br>
NovozymesA/S<br>
Galactose oxidase	:   Galactose oxidase from Sigma<br>
nprE	:   The recombinant form of neutral metalloprotease expressed in<br>
Bacillus subtilis.<br>
PMN	:   Purified neutral metalloprotease from Bacillus amyloliquefacients.<br>
Amylase	:   Amylolytic enzyme sold under the tradename PURAEECT  Ox<br>
described in WO 94/18314, WO96/05295 sold by Genencor,<br>
NATALASE, TERMAMYL, FUNGAMY1 and DURAMYL, all available from Novozymes A/S.<br>
Lipase	:   Lipolytic enzyme sold under the tradename LIPOLASE,<br>
LffOLASE Ultra by Novozymes A/S and Lipomax by Gist-<br>
Brocades.<br>
Cellulase	Cellulytic enzyme sold under the tradename Carezyme, Celluzyme<br>
and/or Endolase by Novozymes AJS.<br>
Pectin Lyase	PECTAWAY and PECTAWASH available from Novozymes<br>
A/S.<br>
PVP	Polyvinylpyrrolidone with an average molecular weight of 60,000<br>
PVNO	Polyvinylpyridine-N-Oxide, with an average molecular weight of<br>
50,000.<br>
PVPVI	Copolymer of vinylimidazole and vinylpyrrolidone, with an average<br>
molecular weight of 20,000.<br>
Brightener 1	Disodium 4,4'-bis(2-sulphostyryl)biphenyl.<br>
Silicone antifoam	Polydimethylsiloxane foam controller with siloxane-oxyalkylene<br>
copolymer as dispersing agent with a ratio of said foam controller to<br>
said dispersing agent of 10:1 to 100:1.	.<br>
Suds Suppressor	12% Silicone/silica, 18% stearyl alcohol, 70% starch in granular<br>
form.<br>
SRP 1	Anionically end capped poly esters.<br>
PEG X	Polyethylene glycol, of a molecular weight of x.<br>
PVPK60	Vinylpyrrolidone homopolymer (average MW 160,000)<br>
Jeffamine  ED-2001	Capped polyethylene glycol from Huntsman<br>
Isachem  AS	A branched alcohol alkyl sulphate from Enichem<br>
MME PEG (2000)	Monomethyl ether polyethylene glycol (MW 2000) from Fluka<br>
ChemieAG.	<br>
DC3225C	Silicone suds suppressor, mixture of Silicone oil and Silica from<br>
Dow Corning.<br>
TEPAE	Tetreaethylenepentaamine ethoxylate.<br>
BTA	Benzotriazole.<br>
Betaine	(CH3)3N+CH2COO'<br>
Sugar	Industry grade D-glucose or food grade sugar<br>
CFAA	Ci2~Cu alky* N-methyl glucamide<br>
TPKFA	Cn-Cu topped whole cut fatty acids.<br>
Clay	A hydrated aluminumu silicate in a general formula<br>
AljOaSiCVxHaO. Types: Kaolinite, montmorillonite, atapulgite,<br>
illite, bentonite, halloysite.<br>
pH	:   Measured as a 1% solution in distilled water at 20C.<br>
EXAMPLE 1<br>
Cloning of the Neutral Metalloprotease Gene from B. amyloliquefaciens.<br>
In this Example, methods used to clone the B. amyloliquefaciens neutral metalloprotease gene are described. The gene-encoding neutral metalloprotease was cloned from B. amyloliquefaciens using well-established methods in this art. The non-exempt (le., the strain carries extrageneric DNA (besides the chloramphenicol selectable marker which is allowed in an exempt strain), specifically the plasmid pJM102 sequences) strain BC91504 (aprE/nprE~pJMlQ2 in BG3594::comk) carries the B. subtilis aprE promoter and signal sequence fused to B. amyloliquefaciens nprE propeptide/mature gene in integrating plasmid pJM102.<br>
The following two sequences (SEQ ID NO:1 and SEQ ID NO:2) of B.' subtilis and B. amyloliquefaciens were generated via PCR with the oligonucleotide primers corresponding to the underlined sequences.<br>
B subtilis chromosomal EcoRl restriction site (GAATTC) and aprE start codon (GTG) and B. amyloliquefaciens nprE stop codon are shown in the following sequences in boldface type as well as a synthetically introduced HindlR restriction site (AAGCTT) designed into primer # 4,<br>
The B. amyloliquefaciens aprE 5' upstream sequence, promoter and signal sequence coding region are shown in the following sequence (SEQ ID NO:1). Primer 1 (apr-f; GAGCTGGGTAAAGCCTATGAAT; SEQ ID NO:5) is shown underlined, at the beginning of the sequence, while the aprE portion of primers 2 and 3 (npr-f and npr-r; GTTCAGCAACATGTCTGCGCAGGCT; SEQ ID NO:6) are shown double underlined at the end of the sequence.   (Sequence Removed)<br>
The sequence of the B. amyloliquefaciens propeptide and mature nprE coding sequence and transcription terminator are provided in the sequence below. In this sequence, the nprE portion of primers 2 and 3 is underlined (GCTGAGAATCCTCAGCTTAAAGAAAACCTG; SEQ ID NO:7), while the npr-r portion of primer 4<br>
(GGCTTCACCATGATCATATATGTCAAGCTTGGGGGG; SEQ ID NO:8) is shown double underlined.  (Sequence Removed)<br>
The amino acid sequence of the full-length NprE (pre-, pro- and mature sequence) is provided below:<br>
(Sequence Removed)<br>
In some alternative embodiments, the following NprE sequence finds use in the present invention.<br>
(Sequence Removed)<br>
The primer sequences used in these PCR experiments are provided below:<br>
(Sequence Removed)<br>
Primers 2 and 3 are reverse complements of each other and correspond to either non-coding (#2) or coding (#3) strands of the chromosomal DNAs. For the coding strand, they correspond to the last 25 base pairs of the aprE signal sequence and the first 30 base pairs of the nprE propeptide.  Primer #4 is the reverse complement to the underlined sequence, comprising 24 base pairs 3' of the nprE stop codon and tenninator with an introduced fJindIII site preceded by six dCTP residues, to provide a so-called "clamp," allowing more efficient cleavage with Hindm restriction endonuclease, as some restriction enzymes cleave inefficiently if their recognition sequence is located at the very ends of DNA fragments.<br>
The two PCR fragments were generated with the following protocol and reagents (except DNA template and oligonucleotide primers) from Applied Biosystems' rTTH DNA Polymerase, XL Kit:<br>
40.6l H2O<br>
30 l 3.3x iTth PCR buffer<br>
10  2 mM dNTP mix<br>
4.4 l 25 mM Mg-acetate<br>
5 l 50 M primer # 1 or * 3 (forward primers)<br>
5 l 50 M primer # 2 or # A (reverse primers)<br>
2 l B. sublilis or B. amyloliquefaciens chrcrnosomal DNA<br>
2 l rTth polymerase<br>
1 l Pfu Turbo polvmerase<br>
100 ul total reaction volume<br>
The PCR conditions used in these experiments were (95'C, 30 sec./58'C, 30sec/68C, ] min.) x 30 cycles followed by rapid cooling to 4"C. Reactions were run on 1.2% agarose/TBE preparative gels, the appropriately-sized fragments excised and purified using the QIAGEN Gel Extraction Kit. In a second fusion, PCR reactions were conducted in which chromosomal DNAs were replaced by 1 ul each of the two separate fragments and only outside primers Ws 1 and #2 were used. The same PCR conditions as described above were used. Due to the complementary ends formed on the two fragments from the use of complementary primers 2 and 3 in the first PCRs, the two fragments were precisely fused.<br>
The fusion fragment was digested with EcoRl and Hind/77 and gel purified as described above. The integration plasmid pJM102 was also digested with EcoKl and HincHQ, and the linear plasmid was then gel purified and ligated by standard techniques to the digested apr/npr fusion fragment. This ligation reaction was subsequently used to directly transform a xylose-induced B. subtilis strain.<br>
After purification, the two fragments were generated by PCR with primers 1 and 2 from wild-type B. subtilis chromosomal DNA, and with primers 3 and 4 from chromosomal DNA from a B. amyloliquefaciens strain. This fragment was again purified as descried above, followed by cutting with EcoRI and HindUl as in the same digestion of the integrating<br><br>
plasmid pJM102 and subsequent ligation of the fusion fragment to the plasmid. Several transformants had the fusion sequenced from the chromosome to verify the absence of any PCR-derived mutations. One of these was then amplified stepwise from 5-25 mg/mL chloramphenicol, the selectable marker on pJM102, to co-amplify the linked expression cassette.<br>
The selected sequence verified transformant was obtained by selection for pJM102's chloramphenicol (CMP) resistance marker on LB/agar plates containing 5 mg/ml CMP. This was then inoculated into LB broth at 10 mg/ml CMP overnight at 37C, with shaking at 250 RPM. This culture was then streaked onto LB/agar plates with 10 mg/ml CMP to isolate single colonies. One colony was then inoculated into LB broth at 25 mg/ml CMP overnight, at 37*C, with shaking at 250 RPM. This culture was then streaked to LB/agar plates with 25 mg/ml CMP to isolate single colonies. These colonies were harvested and stored in glycerol at -70C until use, as known in the art.<br>
The deletion of the two non-essential proteases present in J3. subtilis (aprE and nprE), as well as amylase, reduced the total extracellular protease level during the production of metalloprotease. The DNA encoding the neutral metalloprotease was cloned into an amylase-deleted host. The inducible comK for competence development was inserted in the middle of the amylase locus, making the strain "amyV The secretion of the expressed protein was ensured by insertion of the nucleotides encoding the signal sequence prior to the coding sequence of the gene.<br>
EXAMPLE 2<br>
Expression and Fermentation of the Purified MULTIFECT Neutral and Recombinant<br>
Neutral Metalloprotease (nprE).<br>
The recombinant Bacillus subtilis produced as described in Example 1 was cultivated by conventional batch fermentation in a nutrient medium as described below. One glycerol vial (prepared as described in Example 1) of B. subtilis culture containing the B. amyloliquefaciens neutral metalloprotease was used to inoculate 600 ml of SBGl % medium containing 200 mg/L chloramphenicol. The cultures were grown for 48 hours at 37C, after which time, the culture fluid was recovered by centrifugation at 12,000 rpm, as known in the<br>
art. This procedure was done in duplicate. The final enzyme concentrations obtained were in  the range of about 1.4 and 2 g/L.<br>
EXAMPLES Purification and Characterization of Neutral Metalloprotease<br>
This Example describes the methods used to purify the neutral metalloprotease expressed by the organisms described in Example 2. After 36 hours of incubation at 37 C, -the fermentation broth was recovered and centrifuged at 12 000 rpm (SORVALL centrifuge model RC5B). The secreted neutral meialloproteases were isolated from the culture fluid and concentrated approximately 10-fold using an Amicon filter system 8400 with a BES (polyethersulfone) lOkDa cutoff.<br>
The concentrated supernatant was dialyzed overnight at 4 C against 25 mM MES buffer, pH 5.4, containing 10 mM NaCl. The dialysate was then loaded onto a cation-exchange column Porous HS20 (total volume ~ 83 mL; binding capacity - 4.5g protein/mL column; Waters) as described below. The column was pre-equilibrated with 25 mM MES buffer, pH 5.4, containing ] 0 mM NaCl. Then, approximately 200-300 mL of sample was loaded onto the column. The bound protein was eluted using a pH gradient from 5.4 to 6.2 over 10-column volumes of MES buffer. Elution of the protein was between pH 5.82 and 6.0, and was assessed using proteolytic activity as described herein and 10 % (w/v) NUPAGE SDS-PAGE (Novex). The neutral protease containing fractions were then pooled. Calcium and zinc chloride salts in the ratio of 3:1 were added prior to the adjustment of the pH to 5.8. The Perceptive Biosystems BIOCAD Vision (GMT) was used for protein purification.<br>
The purified protein, assessed using a 10 % (w/v) NUPAGE SDS-PAGE, was determined to homogenous, with greater than 95 % purity. Typically, less than 1% of the purified preparations showed serine protease activity when assessed using the standard protease assay with the small substrate, suc-p-AAPF-pNA (N-succinyl-L-Ala-L-Ala-L-Pro-L-Phe-/?-nitroanilide) (Sigma). This assay was performed in microtiter plate format (96 well) using a 100 mM Tris-HCJ buffer, pH 8.5, containing 10 mM CaCl2 and 0.005 % TWEEN-80. The substrate (p-AAPF NA) was prepared by making a 160 mM stock in DMSO (dimethylsulfoxide) (100 mgAnl) and diluting this stock 100-fold with the Tris-HCl buffer containing CaCJ2 and 0.005 % TWEEN-80. Then, 10 uL of diluted protease solution<br>
(dilutions were prepared using 100 mM Tris-HCl buffer, pH 8.5, containing 10 mM CaClj and 0.005 % TWEEN-80) was added to 190 uL Img/ml p-AAPF solution. The assay was mixed for 5 minutes and the kinetic change at 410 nm was read over 2 to 5 minutes. The slope of the response was measured and used as an indication of the amount of serine protease, activity. The protein was formulated for storage using 25 mM MES buffer, pH 5.8, containing 1 mM zinc chloride, 4 mM calcium chloride, and 40 % propylene glycol.<br>
EXAMPLE 4<br>
Affinity of Purified MULTIFECT Neutral Metalloprotease (PMN) for Calcium and Zinc Cations<br>
In this Example, methods to determine the affinity of the neutral metalloprotease (PMN) prepared as described in the above Examples are described. The affinities of PMN for calcium and zinc ions were performed using the fluorescent indicators Fluo-3 and FluoZin-3, respectively obtained from Molecular Probes. All fluorescence measurements were recorded on a LS50B Luminescence spectrophotometer (Perkin-Elmer). The binding of Fluo-3 was monitored by excitation at 500 nm and the emission spectra were recorded from 505 to 550 nm. Similarly, the binding of FluoZin-3 was monitored by excitation at 495 nm and the emission spectra were collected from 500 to 550 nm. The excitation and emission slit width were both set at 2.5 nm.<br>
In these determinations, 100 uM neutral metalloprotease in 50 mM Tris-HCl buffer, pH 8.4, was titrated with increasing amounts of the relevant indicator. The titration curves are shown in Figure 1. In this Figure, the triangles represent the curve binding data obtained for Zn2+, using the Fluo-Zin3 dye monitored at 516 nm, while the circles represent the data obtained for Ca2+ using the Fluo-3 dye monitored at 522 nm. The association constants (Ka's) for zinc and calcium (assuming a single binding site) were determined to be 0.401 nM and 0.037 nM, respectively. These results indicate that purified MULTIFECT neutral metalloprotease bound the zinc ion with approximately 10-fold greater affinity than the calcium ion. Based on the weaker binding of calcium, initial protein engineering experiments are designed to involve either (i) designing tighter calcium binding site(s) and/or (ii) eliminating the structural stability requirement for calcium (e.g., to stabilize the protein to greater than 80 %).<br>
EXAMPLE 5 Storage Stability<br>
In this Example, experiments conducted to assess the storage stability of PMN and recombinant B. amyloliquefaciens neutral metalloprotease expressed in B. subtilis are described. Proteolysis of these neutral metalloprotease preparations was assessed in the presence of increasing LAS (lauryl sodium sulfate; Sigma) solutions (0 % up to an including 10 %). Proteolytic fragments generated from the purified MULTIFECT neutral metalloprotease (PMN) were observed using 10 % (w/v) NUPAGE SDS-PAGE.<br>
The storage stability of the recombinant neutral metalloprotease from B. amyloliquefaciens expressed in B. subtilis produced as described above, was determined in buffer alone (50 mM Tris-HCl buffer, pH 8.4) and in the presence of detergent base obtained from Procter &amp; Gamble. The buffer and/or detergent base contained zinc ions, calcium ions or a combination thereof. The concentration of both the zinc and calcium ions was varied from 0 to 25 mM. These results were always compared with those for the neutral metalloprotease incubated in buffer alone.<br>
Protease Assays<br>
Azo-casein Assay:<br>
The azo-casein endpoint assay was used to assess the amount of proteolysis that occurred under certain conditions. In these assays, 75 uL of enzyme were incubated with excess calcium or zinc or both ions added to 250 l of 1 % (w/v) azo-casein (Sigma). The reaction proceeded at 30 C for 15 minutes, after which 10 % (w/v) trichloroacetic acid was added to stop the reaction. The precipitated protein and the unreacted azo-casein were removed by centrifugation for 10 minutes at 14 000 rpm. The color of the azo-group was developed by addition of 750 L 1 M sodium hydroxide. The development of the color proceeded for 5 minutes, after which the reaction was stopped and the absorbance was measured at 440 nm.<br>
Succinvlated-casein and TNBSA assay:<br>
The activity of the neutra} metalloprotease was determined using the QuantiCeave Protease Assay Kit (Pierce). This assay is based on the digestion of succinylated-casein by the enzyme. The primary amino groups formed are then reacted with trinitrobenzene suJfonic acid (TNBSA) and form a colored complex that has maximum absorbance at 450 nm. The assay is performed in 96-weIl microtiter format. The assay requires a 15-minute incubation with the succinylated casein and a 15-minute reaction with the TNBSA. During both incubations, the samples are placed on a shaker. TPCK-trypsin (Pierce) is the general standard used for overall protease activity determinations. However, optimum conditions for activity for specific proteases require the use of the protease of interest. In the case of the assays performed in these experiments, both trypsin and the protease of interest were used, in order to calibrate the assay. The accuracy of the assay requires that the standard dilutions made of 0.5 mg/mL trypsin always result in absorbance values (at 450 nm) below 0.5.<br>
Every sample was measured relative to a control containing no casein. The reported change in absorbance (AAbs(450 nm)) accounts for the interference from the amino groups of casein. Further, any possible interference from primary amino groups in the buffer and/or; other components of the detergent was/were also corrected for in this manner. The activity of all samples was determined relative to detergent with no added neutral metalloprotease, as well as for enzyme incubated in BupH borate buffer supplied with the kit, for the same length of time and at the same temperature.<br>
This test is an end-point assay, in which 50 mM borale buffer, pH 8.5, was used at 32 C. The protease assays were typically performed in duplicate. In most experiments to determine stability measurements, the protein and detergent were diluted using the above-mentioned buffer by 1:1000, although in some experiments dilutions of were also 1:500 or 1: 200, in order to obtain readings where the absorbance of the blanks was less than 0.5. The microtiter spectrophotometer used in these experiments was a SpectraMax250 (Molecular Devices) and a]] assays were conducted in medium protein-binding 96-well plates (Corning).<br>
The results for the standards protein samples (eg., trypsin and purified metalloprotease) obtained in these assays indicated that there was a non-linear response (a linear scale may be adequate only in a narrow assay range). Hence, the curve was fitted to a quadratic function where f = yc+ax2 +bx; f is fit to y (SigmaPlot* v. 9; SPSS, Inc.). Thus, if a linear equation was used to quantitate the amount of protein, inaccurate data were obtained; the quadratic equation was found to be required in order to obtain accurate results. It is noted<br>
that the manufacturer's (Pierce) kit insert indicates that the results may be fitted with "x" being a log scale.<br>
EXAMPLE6 Effect of pH and LAS on Neutral Metalloprotease Activity<br>
The pH optimum of the activity for 0.36 mg/mL of formulated nprE was also determined. The buffers investigated in this study were 50 mM sodium acetate over the pH range 3.5-5.5 (pKa = 4.76), 50 mM MES buffer over the pH range 5.5 to 7.0 (pKa = 6.10), and 50 mM Tris-HCl buffer at pH 8.4. The pH optimum for formulated nprE was determined to be between 5.5 and 6.0.<br>
The effect of the detergent component LAS on the activity of 0.36 mg/ml of .formulated nprE was investigated by incubation with 0 to 1 % (w/v) LAS. The results are shown in the graph provided at Figure 2. As these results indicate, the protease is significantly inactivated by the detergent component, thereby necessitating a means to stabilize the protease against this deleterious effect.<br>
In some experiments, the high density liquid detergent (HDL) composition designated as 'TIDE 2005," provided by Procter &amp; Gamble was used. As supplied, this detergent contained all necessary components, except for the neutral metalloprotease of the present invention.<br>
Storage Stability in Liquid Detergent Base as a Function of Time<br>
The stability test was performed in a mini-storage manner. The conditions to be varied and the various concentrations of calcium and zinc chloride salts to be added were assessed using a matrix designed using the FusionPro (S-Matrix) software. The following table summarizes the conditions tested to ascertain the long-term storage stability of neutral metalloprotease from B. amyloliquefaciens.  <br>
Table 6. Long-Term Storage Test Conditions <br>
(Table Removed)<br>
The final volume of each tested condition was 1 mL. TIDE 2005 was dosed with 0:36 mg enzyme/mL. Formulated culture fluid and purified recombinant metalloprotease were incubated in the TIDE 2005 base at 32 C over a period of approximately 4 weeks. The storage stability of the metalloprotease in detergent was compared to the stability of the neutral metalloprotease in 50 mM MES buffer, pH 5.8.<br>
Prior to testing, the samples were diluted 5 in 1000 using assay buffer (50 mM borate buffer, pH 8.5). The residual activity was determined and compared relative to the neutral metalloprotease in assay buffer. All measurements were determined in duplicate. Each sample was tested in parallel with appropriate control blanks (i'.e., the detergent, buffer and any necessary additives being tested). The samples were then assayed as described in the instructions provided with the QuantiCleave Protease Assay Xit (Pierce).<br>
The results of these stability tests conducted over a 3-4 week period are shown in Figure 22. In TIDE 2005, the neutral metalloprotease in the absence of ions (i.e., no added salt) rapidly lost all of its proteolytic/hydrolytic activity against casein. Indeed it was determined that less than 20% of the activity remained after less than 1 hour of incubation. In contrast, incubation of nprE in TIDE 2005 containing zinc ions (up to and including 15 . mM) stabilized the protease and prevented proteolysis over a 7-day period. Thus, the presence of zinc ions in this formulation functioned well in maintaining at least 60 % of the protease activity. Likewise, a concentration of 7.5 mM zinc ions resulted in a similar stabilization effect. This concentration of zinc ions is exceeding low and is contemplated to find use in a variety of detergent formulations.   In these experiments, no added effect was<br>
provided by the inclusion of calcium ions. Furthermore, the addition of calcium ions in excess of 15 mM, and up to and including 25 mM, induced precipitation when added to TIDE 2005 base. Although it is not intended that the present invention be limited to any particular mechanism, it was contemplated that the absence of an effect of added calcium ions on protease stabilization in these experiments was the result of the detergent composition.<br>
For thermolysin, which displays 55 % amino acid sequence identity with neutral metalloprotease from B. amyloliquefaciens (sequence alignment performed using CLUSTAL .W, v. 1.82), it has been clearly shown that zinc ions are essential for activity, whereas the calcium ions and engineering of the calcium binding sites have been shown to play a stabilization role (See e.g., Mansfield., et al., J. Biol. Chem., 272:11152-11156 [1997]; and Van den Berg et al, Biotechnol. Appl. Biochem., 30:35-40 [1999]).<br>
In alternative embodiments, other cations (e.g., Co2*, Mn2+ and Fe2*) find use in the present invention for the stabilization of neutral metalloprotease from B. amyloliquefaciens. This is in contrast to prior data that has indicated that none of these ions resulted in 100 % restoration of specific activity (Holmquist. and Vallee, J. Biol. Chem., 249:4601-4607 [1974]). It is contemplated that these ions will affect stability by preventing the unfolding and subsequent proteolytic degradation of the metalloprotease. However, it is not intended that the present invention be limited to any particular mechanism of action.<br>
EXAMPLE 7 <br>
NprE Protease Production in B. sublilis using the nprE Expression Vector pUBnprE<br>
In this Example, experiments conducted to produce NprE protease in B. subtilis, in particular, the methods used in the transformation of plasmid pUBnprE into B. subtilis are described. Transformation was performed as known in the art (See e.g., WO 02/14490, incorporated herein by reference). The DNA sequence (nprE leader, nprE pro and nprE mature DNA sequence from B.amyloliquefaciens) provided below, encodes the NprE precursor protein:<br><br>
(Sequence Removed)<br>
 (SEQ ID NO:12)<br>
In the above sequence, bold indicates the DNA that encodes the mature NprE protease, standard font indicates the leader sequence (nprE leader), and underlined indicates the pro sequences (nprE pro). The amino acid sequence (NprE leader, NprE pro and NprE mature DNA sequence) (SEQ ID NO: 13) provided below, encodes the NprE precursor<br>
protein. In this sequence, underlined indicates the pro sequence and bold indicates the mature NprE protease. SEQ ID NO: 17 provides the NprE pro-sequence separately from the mature NprE sequence and SEQ ID NO: 18 provides the mature NprE sequence. This sequence was used as the basis for making the variant libraries described herein.<br>
(Sequence Removed)<br>
The pUBnprE expression vector was constructed by amplifying the nprE gene from the chromosomal DNA of B. amyloliquefaciens'by PCR using two specific primers: Oligo (Sequence Removed)<br>
 (SEQ ID<br>
NO:20)<br>
PCR was performed on a thermocycler with Phusion High Fidelity DNA polymerase (Finnzymes. The PCR mixture contained 10 l 5x buffer (Finnzymes Phusion), lul l0mM dNTP's, 1.5l DMSO, 1l of each primer, ll Finnzymes Phusion DNA polymerase, ll chromosomal DNA solution 50ng/J, 34.5 pi MilliQ water. The following protocol was used:<br>
-PCR protocol:<br>
1)	30sec 98C;<br>
2)	10 sec 98C;<br>
3)	20 sec 55C;<br>
4)	l min72C;<br>
5)	25 cycles of steps 2 to 4; and<br>
6)	5 min 72C.<br>
This resulted in a 1.9 kb DNA fragment which was digested using BglK and Ben<br>
DNA restriction enzymes. The multicopy Bacillus vector pUBl 10 (See e,g., Gryczan, J.<br>
Bacteriol., 134:318-329 [1978]) was digested with BamHL. The PCR fragment x BgUI x Bell<br>
was then ligated in the pUB 110 x BamHX vector to form pUBnprE expression vector (See,<br>
Figure 14).	.<br>
pUBnprE was transformed to a B. subtilis (AaprE, AnprE, oppA, AspoIJE, degUHy32, &amp;amyE::(xylR,pxylA-comlC) strain: Transformation into B. subtilis was performed as described in WO 02/14490, incorporated herein by reference. Selective growth of B. s ubtilis Iransformants harboring the pUBnprE vector was performed in shake flasks containing 25 ml MBD medium (a MOPS based defined medium), with 20 mg/L neomycin. MBD medium was made essentially as known in the art (See, Neidhardt et al., J. Bacteriol., 119:736-747 [1974]), except that NH4Cl2, FeSO4, and CaCJ2 were left out of the base medium, 3 mM KZHPO4 was used, and the base medium was supplemented with 60 mM urea, 75 g/L glucose, and 1 % soytone. Also, the micronutrients were made up as a 100 X stock containing in one liter, 400 mg FeSO4 .7H20,100 mg MnSO4 .H20,100 mg ZnSO4.7H20,50<br>
rog CuCl2.2B2O,100 mg CoCl2.6H20,100 mg NaMoO4.2H20,100 mg Na2B4O7.10H20,10 ml of 1M CaCl2, and 10 ml of 0.5 M sodium citrate. The culture was incubated for three days at 37C in an incubator/shaker (Infers). This culture resulted in the production of secreted NprE protease with proteolytic activity as demonstrated by protease assays. Gel analysis was performed using NuPage Novex 10% Bis-Tris gels (Invitrogen, Cat.No. NP0301BOX). To prepare samples for analysis, 2 volumes of supernatant were mixed with 1 volume 1M HC1,1 volume 4xLDS sample buffer (Invitrogen, Cat.No. NP0007), and 1% PMSF (20 mg/ml) and subsequently heated for 10 minutes at 70C. Then, 25 L of each sample were loaded onto the gel, together with 10 L of SeeBlue plus 2 pre-stained protein standards (Invitrogen, Cat.No.LC5925). The results clearly demonstrated that the nprE cloning strategy described in this example yield active NprE produced by B. subtilis.<br>
EXAMPLES Generation of nprE Site Evaluation Libraries (SELs)<br>
In this Example, methods used in the construction of nprE SELs are described. The pUBnprE vector, containing the nprE expression cassette described above, served as template  DNA. This vector contains a unique Bgffl restriction site, which was utilized in the site evaluation library construction.<br>
The pUBnprE expression vector, primers, synthesized at Invitrogen (desalted, 50<br>
nmol scale) were used to generate the libraries. The sequences of the primers are provided in<br>
Table 8-1.	.<br>
To construct a nprE site evaluation library, three PCR reactions were performed, including two mutagenesis PCRs to introduce the mutated codon of interest in the mature nprE DNA sequence and a third PCR used to fuse the two mutagenesis PCRs in order to construct the pUBnprE expression vector including the desired mutated codon in the mature nprE sequence..<br>
The method of mutagenesis was based on the codon-specific mutation approach, in which the creation of all possible mutations at a time in a specific DNA triplet was performed using a forward and reverse oligonucleotide primer with a length of 25 to 45 nucleotides enclosing a specific designed triple DNA sequence NNS ((A,C,T or G), (A,C,T or G), (C or G)) that corresponded with the sequence of the codon to be mutated and guaranteed random<br>
incorporation of nudeotides at that specific nprE mature codon. The number listed in the primer names (See, Table 8-1) corresponds with the specific nprE mature codon position.<br>
Two additional primers used to construct the site evaluation libraries contained the BglE restriction site together with a part of the pUBnprE DNA sequence flanking the 8gIU restriction site. These primers were produced by Invitrogen (50nmole scale, desalted) and are listed in Table 8-1.  <br>
Table 8-1. Primer Sequences  (Table Removed)<br>
Construction of each site evaluation library started with two primary PCR amplifications using the pUB-BglD-FW primer and a specific nprE reverse mutagenesis primer. For the second PCR, the pUB-BglD -RV primer and a specific nprE forward mutagenesis primer (equal nprE mature codon positions for the forward and reverse mutagenesis primers) were used.<br>
The introduction of the mutations in the mature nprE sequence was performed using Phusion High-Fidelity DNA Polymerase (Finnzymes; Cat. no. F-530L). All PCRs were performed according to the Finnzymes protocol supplied with the polymerase. The PCR conditions for the primary PCRs were:<br>
For primary PCR 1:<br>
pUB-Bglll-FW primer and a specific NPRE reverse mutagenesis primer - both 1 L (10 M);<br>
For primary PCR 2:<br>
pUB-BglH -RV primer and a specific NPRE forward mutagenesis primer - both 1 L (10 M); together with<br>
5 x Phusion HF buffer	l0L<br>
10 mM dNTP mixture	1 pL	<br>
Phusion DNA polymerase        0.75 L (2 units/ L)<br>
DMSO.100%	IL<br>
pUBnprE template DNA	1 L (0.1 -1 ng/ L)<br>
Distilled, autoclaved water        up to 50 L<br>
The PCR program was: 30 seconds 98C, 30x (10 seconds 98C, 20 seconds 55C, 1,5 minute 72C) and 5 min 72Q, performed in a PTC-200 Peltier thermal cycle (MJ Research). The PCR experiments result in two fragments of approximately 2 to 3 kB, which had about 30 nucleotide base overlap around the NPRE mature codon of interest Fragments were fused in a third PCR reaction using these two aforementioned fragments and the forward and reverse Bglfi primers. The fusion PCR reaction was carried out in the following solution:<br>
pUB-BgiE-FW primer and pUB-BglD-RV primer- both 1 L (10 M)<br>
5 x Phiision HF buffer	10L<br>
l0mMdNTP mixture	1 L<br>
Phusion DNA polymerase         0.75 L (2 units/ L)<br>
DMSO, 100%	1 L<br>
primary PCR 1 reaction mix      1 L<br>
primary PCR 2 reaction mix      1 L<br>
Distilled, autoclaved water        up to 50 L<br>
The PCR fusion program was as follows: 30 seconds 98C, 30x (10 seconds 98C, 20 seconds 55C, 2:40 minute 72C) and 5 min 72C, in a PTC-200 Peltier thermal cycler (MJ Research).<br>
The amplified linear 6.5 Kb fragment was purified using the Qiaquick PCR purification kit (Qiagen, Cat. no. 28106) and digested with BglQ restriction enzyme to create cohesive ends on both sides of the fusion fragment:<br>
-	35 L purified linear DNA fragment<br>
-	4 L REACT * 3 buffer (Invitrogen)<br>
-1 L BglII, 10 units/ml (Invitrogen)<br>
Reaction conditions: 1 hour, 30C.<br>
Ligation of the BglQ digested and purified using Qiaquick PCR purification kit (Qiagen, Cat, no. 28106) fragment results in circular and multimeric DNA containing the desired mutation:<br>
-	30 L of purified Bgffl digested DNA fragment<br>
-	8 L T4 DNA Ligase buffer (Invitrogen* Cat. no. 46300-018)<br>
-1 L T4 DNA Ligase, 1 unit/L (Invitrogen* Cat. no. 15224-017) Reaction conditions: 16-20 hours, 16<br>
Subsequently, the 1 igation mixture was transformed to a B. subtilis (AoprE, AnprE, oppA, hspollE, degUHy32, kamyE::(xylR,pxylA-comK) strain. Transformation to B. subtilis was performed as described in WO 02/14490, incorporated herein by reference. For each library, 96 single colonies were picked and grown in MOPS media with neomycin and 1.25 g/L yeast extract for sequence analysis (BaseClear) and screening purposes. Each library included a maximum of 19 nprE site-specific variants.<br>
The variants were produced by growing the B. subtilis SEL transformants in 96 well MTP at 37C for 68 hours in MBD medium with 20 mg/L neomycin and 1.25 g/L yeast extract (See, above).<br>
EXAMPLE 9 Generation of nprE Combinatorial Libraries (RCLs)<br>
In this Example, methods used to generate nprE combinatorial libraries are described. For this enzyme, one property was chosen as the property that needed to be changed the most. This property is defined herein as the "primary property." All other properties were' "secondary properties" for the purpose of combinatorial library design. The basic strategy for improving a protein as used herein, was to combine mutations that improve the primary property and also maintain or improve the secondary properties. The site evaluation data were used to identify those mutations which improved the primary property while maintaining or improving the secondary properties. Mutations that were to be combined were identified by their Performance Index (PI or Pi) and associated AAGipp values.<br>
The "Apparent Free Energy Change" (AAGipp) as used herein is defined as:<br>
where P variant  the performance value for the variant and Parent is the performance value for the parent enzyme under the same conditions. The ratio Pvariant P parent is defined as the performance index (Pi) for the property. The AAGW values were expected to behave in a similar fashion to actual AAG values for data distributions and additivity. However, since AAG represents the maximum amount of work that can be carried out by the variant compared to the parent enzyme, the quantity AAGm generally underestimates the AAG and may lead to results that appear synergistic in that the properties of two additive positions may be greater than the value predicted by adding their AAGapp values together.<br>
For example, when TIDE stability is the primary property and BMI activity is the secondary property, mutations that have AAGam, values 1) and BMI AAG,pp values 0.9) may be chosen for combination. Indeed, these relationships were explored in these experiments.<br>
To produce the variants used in these experiments, synthetic nprE library fragments, containing multiple mutations at multiple nprE mature DNA positions, were produced by GeneArt (Geneart). These 1.5 kB nprE library fragments were digested with DNA restriction enzymes Pvul and Aval, purified and ligated in the 5 kB pUB vector fragment (also digested with DNA restriction enzymes Pvul and Aval) by a ligase reaction using T4 DNA Ligase (Invitrogen* Cat. no. 15224-017).    .<br>
To transform the ligation reaction mix directly into Bacillus cells, the library DNA (nprE library fragment mix ligated in pUB vector fragment) was amplified using the TempliPhi kit (Amersham cat. #25-6400). For this purpose, 1L of the ligation reaction mix was mixed with 5L of sample buffer from the TempliPhi kit and heated for 3 minutes at 95 *C to denature the DNA. The reaction was placed on ice to cool for 2 minutes and then    . spun down briefly. Next, 5pL of reaction buffer and 0.2}iL of phi29 polymerase from the TempliPhi kit were added, and the reactions were incubated at 30*C in an MJ Research PCR machine for 4 hours. The phi29 enzyme was heat inactivated in the reactions by incubation at 65*C for 10 min in the PCR machine.<br>
For transformation of the libraries into Bacillus, 0.1 yL of the TempliPhi amplification reaction product was mixed with 500 pL of competent B. subtilis cells (AaprE, AnprE, oppA, AspoDE, degUHy32, AamyE::(xylR,pxylA-comK) followed by vigorous<br>
/<br>
shaking at 37*C for 1 hour. Then, 100 and 500 uL were plated on Hl-agar plates containing 20 mg/L neomycin and 0.5% skim milk. In general, transformation to B. subtilis was<br>
performed as described in WO 02/14490, incorporated herein by reference. B. subtilis nprE combinatorial libraries, constructed by this method are contemplated to contain wild type amino acids at one or more of the positions targeted for mutagenesis.    <br>
The variants obtained in these libraries were then tested for their stability in TIDE and their performance in BMI wash performance tests as described herein. Table 9 provides performance indices for the variants tested in the BMI assay. In this Table, "Pos." indicates the position in the NprE amino acid sequence that was changed, and "AA" indicates the amino acid substitution made for each variant.         <br>
Table 9. Results (Performance Indices) for Tested Variants    <br>
(Table Removed)<br>
EXAMPLE 10<br>
Alternative Method Generate nprE Site Evaluation Libraries (SELs) via QuikChange Mutagenesis<br>
In this Example, alternative methods to generate nprE SELs are described. As in Example 8, above, the pUBnprE vector served as the template DNA source for the generation of nprE SELs. The major difference between the two methods is that this method requires amplification of the entire vector using complementary site-directed routagenic primers.<br>
Materials;<br>
Bacillus strain containing the pUBnprE vector<br>
Qiagen Plasmid Midi Kit (Qiagen cat # 12143)<br>
Ready-Lyse Lysozyme (Epicentre cat # R1802M)<br>
dam Methylase Kit (New England Biolabs cat # M0222L)<br>
Zymoclean Gel DNA Recovery Kit (Zyroo Research cat # D4001)<br>
nprE site-directed mutagenic primers, l00nmole scale, 5' Phospborylated, PAGE purified<br>
(Integrated DNA Technologies, Inc.)<br>
QuikChange Multi Site-Directed Mutagenesis Kit (Stratagene cat # 200514)<br>
MJ Research PTC-200 PeJtier Thermal Cycler (Bio-Rad Laboratories)<br>
1.2% agarose E-gels (Invitrogen cat # G5018-01)<br>
TempliPhi Amplification Kit (GE Healthcare cat # 25-6400-10)<br>
Competent B. subnlis cells (AaprE, AnprE, oppA, AspoIlE, degUHy32, AamyE::(xylRj&gt;xylA-<br>
comK)<br>
Methods:<br>
To obtain the pUBnprE vector, a single colony of a Bacillus strain containing the pUBnprE vector was used to inoculate a 5ml LB + l0ppm neomycin tube. This was the starter culture used in these methods. The culture was grown at 37C, with shaking at 225 rpm for 6 hours. Then, 100 ml of fresh LB + l0ppm neomycin were inoculated with 1ml of the starter culture. This culture was grown overnight at 37C, with shaking at .225 rpm. Following this incubation, the cell pellet was harvested by sufficient centrifugation to provide a cell pellet. The cell pellet was resuspended in 10 ml Buffer PI (Qiagen Plasmid Midi Kit). Then, l0ul of Ready-Lyse Lysozyme was added to the resuspended cell pellet and incubated at 37C for 30 min. Then, the Qiagen Plasmid Midi Kit protocol was continued (using 10ml of Buffer P2 and P3 to account for the increased volume of cell culture). After isolation of pUBnprE vector from Bacillus, the concentration of pUBnprE vector was quantitated. The vector was then dam methylated using the dam Methylase Kit (New England Biolabs), using the methods set forth in the kit protocols, to methylate approximately 2 ug of pUBnprE vector per tube. The Zymoclean Gel DNA recovery kit was used to purify and concentrate the dam-methylated pUBnprE vector. The dam-methylated pUBnprE vector was quantitated and then diluted to a working concentration, of 50ng/ul. Complementary site-directed mutagenic primers (1 ul of each primer at lOuM) (See, Table 10-1), were .used in a PCR reaction in the QuikChange Multi Site-Directed Mutagenesis Kit (Stratagene), following the manufacturer's protocol (e.g., lul dam methylated pUBnprE vector (50ng/ul), lul nprE site-directed Forward . mutagenic primer (l0uM), lul nprE site-directed Forward mutagenic primer (l0uM), 2.5ul l0x QuikChange Multi Reaction buffer, lul dNTP Mix, lul QuikChange Multi enzyme blend (2.5U/ul), and 17.5ul distilled, autoclaved water, to provide a 25ul total reaction mix). The nprE site evaluation libraries were amplified using the following conditions: 95C, for 1 min. (ltt cycle only), followed by 95C for 1 min., 55C for 1 min, 65C for 13 Vi min., and repeat cycling 23 times. The reaction product was stored at 4C overnight. Then, the reaction mixture underwent Dpril digest treatment (supplied with QuikChange Multi Site-Directed Mutagenesis Kit) to digest parental pUBnprE vector, using the manufacturer's protocol (i.e.,<br>
1.5ul Dpnl restriction enzyme was added to each tube and incubated at 37C for 3 hours; 2ul of DpnI-digested PCR reaction was then analyzed on a 1.2% E-gel for each nprE SEL to ensure PCR reaction worked and that parental template was degraded). TempliPhi rolling circle amplification was then used to generate large amounts of DNA for increasing library size of each nprE SEL, using the manufacturer's protocol (i.e., lul Dpnl treated QuikChange Multi Site-Directed Mutagenesis PCR, 5ul TempliPhi Sample Buffer, 5ul TempliPhi Reaction Buffer, and 0.2ul TempliPhi Enzume Mix, for an ~1 lul total reaction; incubated at 3QC for 3 hours; the TempliPhi reaction was diluted by adding 200ul distilled, autoclaved water and briefly vortexed. Then, 1.5ul of diluted TempliPhi material was transformed into competent B. subtilis cells, and nprE SELs were selected for using LA + l0ppm Neomycin + 1.6% skim milk plates. Table 10-1 provides the names, sequences and SEQ ID NOS for the primers used in these experiments. All of the primers were synthesized by Integrated DNA Technologies, on l00nmole scale, 5'-phosphorylated, and PAGE purified.  <br><br>
Table 10-1. Primers  (Table Removed)<br>
EXAMPLE 11<br>
Identification of nprE Homologues<br>
In this Example, experiments conducted to identify npr homologues are described. In particular, in this Example, experiments were conducted to'clone neutral protease (npr) homologs from different and closely related Bacillus species. The different species were chosen in order to explore the diversity and properties from which these different species are isolated.<br>
The various npr homologs explored included:<br>
B. caldofyticus npr (P23384)<br>
B. cereus nprC (P05806)<br>
B. cereus E33L npr (AAY60523)<br>
B. stearothermophilus nprT<br>
B. sublilis nprB<br>
B. sublilis nprE<br>
B. thuringiensis nprB (AAK00602)<br>
S. aureusaur(P81177)<br>
Figure 3 provides a sequence alignment of these homologs (SEQ ID NOS:173-181) and Figure 4 (SEQ ID NOS:182-191) provides another sequence alignment of various other homologs.<br>
In these experiments, the materials included:<br>
Chromosomal DNA of B. subtilis strain 1168<br>
The following DNA plasmids were synthesized at DNA2.0 with B. subtilis codon<br>
optimization:<br>
pJ4:G01905 (B. thuringiensis nprB) (See, Figure 6)<br>
pJ4:GO 1906 (B. cereus E33L npr) (See, Figure 7)<br>
pJ4:G01907 (B. cereus nprC) (See, Figure 8)<br>
pJ4:G01908 (B. caldolyticus npr; (See, Figure 9)<br>
pJ4:G01909 (S. aureus aur) (See, Figure 10)<br>
pJ4:G01938 (S. stearothermophilus nprT) (See, Figure 11)<br>
pJHT vector (See, Figure 12)<br>
pAC vector (See, Figure 13)<br>
MJ Research PTC-200 Peltier Thermal Cycler (Bio-Rad Laboratories)<br>
Primers (Operon Inc)<br>
PfuUltra D Fusion HS DNA Polymerase (Stratagene)<br>
Restriction endonucteases (Roche)<br>
TOP10 chemically competent E. coli cells (Invitrogen)<br>
B. subtilis competent cells ((AaprE, bnprE, oppA, AspoIIE, degUHy32, &amp;amyE::(xylRfxylA-<br>
comK)<br>
 Table 11. Primers  (Table Removed)<br>
A.        Cloning of B. subtilis nprE<br>
To construct the B. subtilis nprE plasmid, the amplified aprE promoter fragment (from pJHT vector) and B. subtilis nprE gene with terminator fragment (from B. subtilis strain 1168) were separately prepared. Figure 15 provides a schematic, illustrating the amplification of the individual DNA fragments.<br>
PCR Splice Overlap Extension (SOE) reaction was used to join the 2 separate DNA fragments together. In this reaction, the following reagents were combined: lul aprE promoter DNA fragment, lul B. subtilis nprE gene + Terminator fragment, lul Primer EL-689 (25uM), lul Primer EL-695 (25uM), 5ul l0x PfuUltra II Fusion HS DNA polymerase buffer, lul dNTP (l0mM), lul PfuUltra H Fusion HS DNA polymerase, and 39ul distilled, autoclaved water to provide a total reaction volume of 50 ul. The PCR cycles were: 95C for 2 minutes (1s1 cycle only), followed by 28 cycles of 95C for 30 seconds, 54C for 30 seconds, and 72C for 0:45 seconds.<br>
The PCR fusion fragment of aprE promoter-B. subtilis nprE gene + Terminator was digested with Mfel and BaniHl restriction endonucleases. The pJHT vector was digested with coRI and BaniHl restriction endonucleases. The restriction endonuclease digested aprE promoter-B. subtilis nprE gene + Terminator DNA fragment was then ligated with the restriction endonuclease digested pJHT vector. The ligation mixture was then transformed into TOP10 chemically competent E. coli cells for selection on LA + 50ppm carbenicillin. After identification of plasmids containing the correct DNA construct sequence for plasmid pELSOl (See, Figure 16), transformed into competent B. subtilis cells for integration into aprE promoter locus. Transformants were selected for protease activity (i.e. skim milk clearing) on LA + 5ppm chloramphenicol + 1.6% skim milk plates. Amplified strains were<br>
then transferred to LA + 25ppm chloramphenicol + 1.6% skim milk plates. Strains were then transferred to LA -f 25ppm chloramphenicol + 1.6% skim milk plates for amplification.<br>
B.        Cloning of B. subtilis nprB<br>
To construct the B. subtilis nprB plasmid, amplified the aprE promoter fragment (from pJHT vectqr), B. subtilis nprB gene fragment (from B. subtilis strain 11.68), and Terminator fragment (from pJHT vector) were separately prepared. Figure 17 provides a schematic diagram of the amplification of the individual DNA fragments.<br>
PCR Splice Overlap Extension (SOE) reaction was used to join the three separate DNA fragments together. In this reaction, the following reagents were combined: 1'ul aprE promoter DNA fragment, lul B. subtilis nprB gene + Terminator fragment, lul Terminator DNA fragment, lul Primer EL-689 (25uM), lul Primer EL-700 (25uM), 5ul lOx PfuUltra H Fusion HS DNA polymerase buffer, lul dNTP (lOmM), lul PfuUltra D Fusion HS DNA Polymerase, and 38ul distilled, autoclaved water, for a 50 ul total reaction volume. The PCR cycles were: 95C for 2 minutes (1* cycle only), followed by 28 cycles of 95C for 30 seconds, 54C for 30 seconds, and 72C for 0:45 seconds.<br>
The PCR fusion fragment of aprE promoter-.fi. subtilis nprB gene + Terminator was digested with Mfel and Sphl restriction endonucleases. The pJHT vector was digested with EcoRl and Sphl restriction endonucleases. The restriction endonuclease digested aprE promoter-!?, subiilis nprB gene + Terminator DNA fragment was then ligated with the' restriction endonuclease digested pJHT vector. The ligation mixture was then transformed into TOP10 chemically competent E. coli cells for selection on LA + 50ppm carbenicillin. After identification of plasmids containing the correct DNA construct sequence for plasmid pEL508 (See, Figure 18), transformed into competent B. subtilis cells for integration into aprE promoter locus. Transformants were selected for protease activity (le. skim milk clearing) on LA + 5ppm chloramphenicol + 1.6% skim milk plates. Amplified strains were then transferred to LA + 25ppm chloramphenicol -f 1.6% skim milk plates. Strains were then transferred to LA + 25ppm chloramphenicql + 1.6% skim milk plates for amplification.<br>
C.        Cloning of B. stearotherinophHus nprT<br>
To construct the B. stearothermophilus nprT plasmid, the amplified aprE promoter fragment (from pJHT vector) and B. stearothermophilus nprT fragment (from plasmid pJ4:G01938) were separately prepared. Figure 19 provides a schematic diagram of the amplification of the individual DNA fragments.<br>
PCR Splice Overlap Extension (SOE) reaction was used to join the 2 separate DNA<br>
fragments together. In this reaction, the following reagents were combined: lul aprE<br>
promoter DNA fragment, lul B. stearothermophilus nprT gene fragment, lul Primer EL-755<br>
(25uM), lul Primer EL-735 (25uM), 5ul 1 Ox PfuUltra E Fusion HS DNA Polymerase buffer,<br>
lul dNTP (l0mM), lul PfuUltra H Fusion HS DNA Polymerase, and 39ul distilled,<br>
autoclaved water, to provide a total reaction volume of 50'ul. The PCR cycles were: 95C<br>
for 2 minutes (1st cycle only), followed by 28 cycles of 95C for 30 seconds, 54G for 30<br>
seconds, and 72C for 0:45 seconds.	<br>
The PCR fusion fragment of aprE promoter-5. stearothennophilus nprT gene + Terminator was digested with EcoRI and HindIII restriction endonucleases. The pAC vector was digested with EcoRl and HindIII restriction endonucleases. The restriction endonuclease digested aprE promoter-fi. stearothermophilus nprT DNA fragment was then ligated with the restriction endonuclease digested pAC vector. TempliPhi rolling circle amplification was then used to generate large amounts of the ligated aprE promoter-/?, stearothermophilus nprT pAC DNA molecule, using the manufacturer's protocol (i.e., lul aprE promoter-!?. stearothennophilus nprT pAC ligation reaction, 5ul TempliPhi Sample Buffer, 5ul TempliPhi Reaction Buffer, and 0.2ul TempliPhi Enzume Mix, for an -llul total reaction; incubated at 30C for 3 hours). The TempliPhi reaction was then transformed directly into competent B. subtilis cells for integration into aprE promoter locus, thereby generating Bacillus strain EL560, confirmed by DNA sequencing analysis (See, Figure 20). Transformants were selected for protease activity (Le. skim milk clearing) on LA 15ppm chloramphenicol + 1.6% skim milk plates. Strains were then transferred to LA + 25ppm chloramphenicol + 1.6% skim milk plates for amplification.<br>
D.	Qoning of B. caldolyticus npr<br>
To construct the B. caldolyticus npr plasmid, the amplified aprE promoter fragment (from pJHT vector) and B. caldolyticus npr fragment (from plasmid pJ4:G01908) were separately prepared. Figure 21 provides a schematic diagram of the amplification of the individual DNA fragments.<br>
PCR Splice Overlap Extension (SOE) reaction was used to join the 2 separate DNA fragments together. In this reaction, the following reagents were combined: lul aprE promoter DNA fragment, lul B. caldolyticus npr gene fragment, lul Primer EL-755 (25uM), lul Primer EL-741 (25uM), 5ul l0x PfuUlra n Fusion HS DNA Polymerase buffer, lul dNTP (l0mM), lul PfuUltra n Fusion HS DNA Polymerase, and 39ul distilled, autoclaved water, to provide a total reaction volume of 50 ul. The PCR cycles were: 95C for 2 minutes (I11 cycle only), followed by 28 cycles of 95C for 30 seconds, 54C for 30 seconds, and 72C for 0:45 seconds.<br>
The PCR fusion fragment of aprE promoter-B. caldolyticus npr gene + Terminator was digested with EcdKl and HindIII restriction endonucleases. The pAC vector was digested with EcoRI and HindIII restriction endonucleases. The restriction endonuclease digested aprE promoter-B. caldolyticus npr DNA fragment was then ligated with the restriction endonuclease digested pAC vector. TempliPhi rolling circle amplification was then used to generate large amounts of the ligated aprE promoter-B. caldolyticus npr pAC DNA molecule, using the manufacturer's protocol (le., lul aprE promoter-B. caldolyticus npr pAC ligation reaction, 5ul TempliPhi Sample Buffer, 5ul TempliPhi Reaction Buffer, and 0.2ul TempliPhi Enzume Mix, for an -1 lul total reaction; incubated at 30C for 3 hours). The TempliPhi reaction was then transformed directly into competent B. subtilis cells for integration into aprE promoter locus, thereby generating Bacillus strain EL561 (See, Figure 22), confirmed by DNA sequencing analysis. Transformants were selected for protease activity (i.e. skim milk clearing) on LA + 5ppm chloramphenicol + 1.6% skim milk plates. Strains were then transferred to LA + 25ppm chloramphenicol -f 1.6% skim milk plates for amplification.<br>
E.	Cloning of B. thuringiensis nprB<br>
To construct the B. thuringiensis nprB plasmid, the amplified aprE promoter fragment (from pJHT vector) and B. thuringiensis nprB fragment (from plasmid<br>
pJ4:G01905) were separately prepared. Figure 23 provides a schematic showing the<br>
amplification of the individual DNA fragments.	<br>
PCR Splice Overlap Extension (SOE) reaction was used to join the 2 separate DNA fragments together. In this reaction, the following reagents were combined: lul aprE promoter DNA fragment, lul B. thuringiensis nprB gene fragment, lul Primer EL-755 (25uM), lul Primer EL-744 (25uM), 5ul l0x PfuUltra D Fusion HS DNA Polymerase buffer, lul dNTP (l0mM), 1 ul PfuUltra n Fusion HS DNA Polymerase, and 39ul distilled, autoclaved water, to provide a total reaction volume of 50 ul. The PCR cycles were: 95C for 2 minutes (1* cycle only), followed by 28 cycles of 95C for 30 seconds, 54C for 30 seconds, and 72C for 0:45 seconds.<br>
The PCR fusion fragment of aprE promoter-B. thuringiensis nprB gene + Terminator was digested with EcoRl and HindlR restriction endonucleases. The pAC vector was digested with EcoRl and Hindis, restriction endonucleases. The restriction endonuclease digested aprE promoter-^, thuringiensis nprB DNA fragment was then ligated with the restriction endonuclease digested pAC vector. The ligation mixture was then transformed . into TOP10 chemically competent E. coli cells. After identification of plasmids containing the correct DNA construct sequence for plasmid pEL568, transformed into competent B. subtilis cells. Transformants were then selected for protease activity (i.e. skim milk clearing) on LA + 5ppm chloramphenicol + 1.6% skim milk plates. DNA plasmid preparation shows that plasmid pEL568 is stable in B. subtilis and does not integrate into the aprE promoter locus. Figure 24 provides a map of plasmid pEL568.<br>
E.        Homology Modeling<br>
In yet additional embodiments, a homology model of the mature domain of NprE from Bacillus amyloliqmfadens is provided. In these experiments, the protein sequence of the mature domain of the NprE sequence (SEQ ID NO: 192) was run through the program BlastP (NCBI). This program matches the sequence to other known sequences of varying sequence identity. From the output, sequences for which an X-ray crystal structures are known were identified. These sequences included S. aureus Npr (pdb id 1BQB) P. aeruginosa Npr (pdb idlEZM), B. thermolyticus thermolysin (pdb id 1KEI) and B. cereus Npr (pdb id 1NPC). Figure 5 provides a sequence alignment of the various sequences analyzed. (SEQ ID NOS:192-195)<br>
The sequence of the B. cereus Npr was found to be the most identical to NprE. B. thermolyticus thermolysin (pdb id 1KEI) was excluded from subsequent steps as it is very similar to B. cereus Npr. A homology model was then prepared as detailed below, and all calculation were made using the program MOE (Chemical Computing). First, the S. aureus Npr (pdb id 1BQB; SEQ ID NO:193) P. aeruginosa Npr (pdb idlEZMXSEQ ID NO: 194), and B. cereus Npr (pdb id 1NPC)(SEQ ID NO:195) sequences . were aligned using known structure, in order to obtain the most accurate sequence alignment (i.e., a structure based sequence alignment was produced). Next, the NprE mature sequence was aligned to this structure based sequence alignment Then, an initial homology model of NprE was produced using the JB. cereus Npr (pdb id INPC) structure as a template, and using the sequence alignment of NprE to B. cereus Npr produced above. It was clear from inspection of this alignment that whereas B. cereus Npr contains four Ca2* ion binding sites, while NprE only contains two Ca2+ ion binding sites.<br>
Finally, after the initial homology model was built, the metal ions (i.e., Zn2*, and two Ca2* ) were computationally fixed, as were their respective protein ligands. The rest of the. model structure was computationally minimized using the CHARMM22 parameter set, resulting in the final homology model.<br>
EXAMPLE 12 Wash Performance Tests<br>
In this Example, experiments conducted to determine the wash performance of the metal loprotease of the present invention are described. All wash performance tests were performed under American wash conditions, as indicated below:<br>
Laundry Wash Performance Tests<br>
Equipment:	Terg-O-Tometer (US Testing) 6 pot bench top model<br>
Temperature:	15C / 60F<br>
Wash Time:	12 minutes<br>
Agitation:	l00rpm<br>
Rinse Time:	3 minutes<br>
Water Hardness:	6 grains per gallon /105 ppm as CaCO3 (3/1 Ca*2/ Mg*2)<br>
Sud concentration:	1.6 g/l TIDE 2005 liquid detergent base<br>
Enzyme dosage:	0.00 - 0.55 - 2.75 - 5.50 mg active protein/1 wash solution<br>
Swatches:	EMPA 116 Fixed, fixated at 20C: Blood, milk, ink on cotton (10 x<br>
7.5cm)<br>
EMPA 116 Unfixed: Blood, milk, ink on cotton (10 x 7.5 cm) Equest grass: Grass Medium scrubbed on cotton (10 x 7.5 cm) CFT C-10: Pigment, oil ,milk on cotton (10 x 7.5 cm) EMPA 221: Unsoiled cotton used as ballast (10x10 cm) 6 EMPA 116 fixed + 2 EMPA 221 were put in one vessel 6 EMPA 116 unfixed + 2 EMPA 221 were put in one vessel 6 Equest grass + 2 EMPA 221 were put in one vessel 6 CFT C-10 + 2 EMPA 221 were put in one vessel<br>
Drying conditions:       Spin-drier, Grass stains were dried to the air, covered with dark clothes. The other stains were ironed.<br>
Measuring swatches:    Tristimulus Minolta Meter CR-300 with equation L (L*a*b), D65 Std. Illuminate, on a white background. Expressed on Delta % Soil Removal. 3 readings per swatch (before and after washing)<br>
% Stain Removal = (L value after washing - L value before washing)/(L0mmn- L value<br>
before washing) x 100%	.<br>
All experiments were done in quadruplicate<br>
The proteases were tested in a specially developed washing test, using three different cotton swatches, soiled with:<br>
(a)	milk, blood and ink (10.0 x 7.5 cm; EMPA), designated with the numbers 116<br>
unfixed and fixed (the stains were fixed at 20C);<br>
(b)	grass medium (10.0 x 7.5 cm; Equest); and<br>
(c)	pigment, oil and milk (10.0 x 7.5 cm designated with the numbers C-lO CFT).<br>
These experiments are described in greater detail below. The washing tests were performed in a bench top model Terg-O-Tometer (US Testing), equipped with six stainless steel test vessels. The stainless steel test vessels each contained 1.6 g of TIDE 2005 liquid detergent base, dissolved in 1000 ml water of 105 pprn/6 grains per gallon, and were each loaded with six of the same soiled cotton swatches and two extra ballast cotton swatches (EMPA 221). A selected protease (e.g., neutral metalloprotease or another protease) was added to each vessel in a concentration from 0.00 to 5.50 mg active protein per liter suds.<br>
The tests were carried out for 12 minutes at 15 C/60 F, with an agitation of 100 rpm. After washing, the swatches were rinsed for 3 minutes under cold tap water and placed<br><br>
in a spin-drier. The grass swatches were air-dried and covered with dark clothing to limit the sensitivity of the grass stains to light. All other swatches were ironed. All experiments were performed in quadruplicate.<br>
The reflectance of the tested swatches was measured with a Tristimulus Minolta Meter CR-300 using the equation L (L*a*b). Wash performance values were calculated using the following relationship:<br>
% Stain Removal = (L value after washing - L value before washing)/(Lo white cotter- L value before washing) x 100%<br>
The results of the Terg-O-Tometer (TOM) assay for purified MULTffECT arc shown in Figure 26, and compared with those of subtilisin (BPN1 Y217L), a serine alkaline protease. The TOM provided a fully operational and valid means for discriminating between the different wash performances of various proteases (e.g., serine proteases, neutral metalloprotease, and variants thereof). The TOM tests were performed on BMI and Equest medium-soiled grass surface with TIDE 2005 as the base detergent.<br>
As indicated in Figure 26, it was apparent that the purified neutral metalloprotease clearly performed better in the wash test than the serine protease (BPN' Y217L). In particular, 2.75 ppm of purified neutral metalloprotease was required to show a delta soil removal of - 10 % compared to only 0.55 ppm of the serine protease (BPN' Y217L) on the Equest grass stain. The wash performance of the neutral metalloprotease was also tested at low temperature and found to perform very well medium solid Equest stain fabric. Similar results were obtained with purified commercially available MTLTIFECT neutral protease as with the recombinant nprE produced as described above.<br>
EXAMPLE13<br>
Performance of nprE Variants in BMI-TIDE 2X Performance Assay In this Example, experiments conducted to assess the perfonnance of various nprE variants in the BMI assay outlined above are described. The methods provided prior to Example 1 were used (See, "Microswatch Assay for Testing Protease Performance"). The results for multiply-substituted variants with Perfonnance Indices greater than one (PI &gt; 1) and those with Performance Indices less than one (PI 
In Table 13.1, data obtained for selected single-substitution variants in the BMI-TIDE 2X performance assay are provided. The Table provides performance indices, which where calculated as described above for the variants, which show improved perfonnance .compared to the WT enzyme. Those variants, which have a perfonnance index greater than 1, have an improved performance.   .<br>
In Table 13.2, data obtained for selected multiple-substitution variants in the BMI-TIDE 2X performance assay are provided. The Table provides perfonnance indices, which where calculated as described above for the variants, which show improved performance compared to the WT enzyme. Those variants, which have a performance index greater than 1, have an improved performance.   <br>
Table 13.1 Performance Assay Results for All Variants with Performance Index &gt;1 <br>
 (Table Removed ) <br>
 Table 13 .2 BMI Performance Assay Results for All Variants with Performance Index &gt; 1    <br>
 (Table Removed ) <br>
EXAMPLE14 Stability of nprE Variants<br>
In this Example, experiments conducted to determine the stability of nprE variants are described. In these experiments, the methods describe prior to Example 1 were used to determine the performance indices (.See, "NprE stability assays in the presence of detergent" above). The following tables provide the results for those variants with Performance Indices greater than one (PI&gt;1) tested with and without DTPA.<br>
The stability was measured by determining AGLA activity before and after incubation at elevated temperature. The table contains the relative stability values compared to WT under mese conditions. It is the quotient of (Variant residual activity/WT residual activity). A value greater than one indicates higher stability in the presence of detergent. In Tables 14.1 and 14.2, data are provided showing the relative stability data of single-substitution variants of NprE relative to the stability of the WT NprE stability under these conditions, with and without DTPA.<br>
In Tables 14.3 and 14.4, data are provided showing the relative stability data of multiple-substitution variants of NprE relative to the stability of the WT NprE stability under these conditions, with and without DTPA.<br>
 Table 14.1 Stability Results in the Presence of 25% TIDE 2X with DTPA     <br>
(Table Removed ) <br>
Table 14.2 Stability of Variants in Tide  2X Without DTPA  <br>
(Table Removed ) <br>
Table 14.3 Stability Assay Results in the Presence of 25% TIDE 2X With DTPA<br>
(Table Removed ) <br>
Table 14.4 Stability Assay Results in the Presence of 25% TIDE 2X Without DTPA<br>
(Table Removed ) <br>
Table 14.5. Citrate Stability Assay Results<br>
The data in the following table (Table 14.5) represent the relative stability data of variants of NprE relative to the stability of the WT NprE stability in the citrate stability assay. The stability was measured by determining casein activity by determining AGLA activity before and after incubation at elevated temperature (See, "Citrate Stability Assay" above). The table contains the relative stability values compared to WT under these conditions. It is presented as the quotient of (Variant residua] activity W/<br>
T residual activity). A value greater than one indicates higher stability in the presence of detergent   <br><br>
(Table Removed ) <br>
EXAMPLE 15 pH Performance Profile of nprE Compared to BPN' 217L<br>
In this Example, experiments conducted to evaluate the comparative performance of nprE and BPN' Y217L are described. In these experiments, EMPA 116 (BMI) and Equest grass stains were used.<br>
Materials:<br>
NprE, 8 mg/mL<br>
BPN' Y217L, 25.6 mg/mL<br>
EMPA 116 soil cloth, 3" x 4.5" (Testfabrics)<br>
Equest grass (med.), 3" x 4" (Warwick Equest)<br>
EMPA 221 white cotton swatches, 3" x 4.5"<br>
Minolta Chrbmameter CR200<br>
TIDE 2005 (provided by Procter &amp; Gamble)<br>
Water hardness concentrate: 15,000 grains per gallon (gpg), 3:1 Ca:Mg<br>
1 M Bis-TRIS-propane buffer<br>
Cone, sulfuric acid<br>
50 L mix tank with spigot and agitator<br>
Terg-O-Tometer<br>
DI Water<br>
The swatches were prepared for treatment. Three replicates per treatment were conducted, with 18 swatches used per treatment. The grass swatches were prepared in a dark room to prevent fading. The reflectance values of about 18 soiled swatches were obtained using a Minolta Chromameter. Three readings were obtained per swatch. The L values, average L value and standard deviation were recorded. This is the LinWai value.<br>
The detergent solution was prepared as follows (for 40 L total). It was preferred to prepare this solution the night before testing. The solution was stored in the cold over night The solution was prepared by adding 39.724 Kg of DI water to 50 L mix tank, starting the agitator, mixing in 60 grams of TIDE liquid detergent, .mixing in 16 mL of water hardness solution, and 200 mL of 1 M Bis-TRIS-propane. The pH was adjusted using concentrated sulfuric acid (adjusted to 0.2 pH units below desired pH, if solution was stored overnight), as pH creeps up overnight). The final concentrations were: TIDE = 1.5 g/L; water hardness = 6 gpg; and Bis-TRIS-propane = 5 mM.<br>
For testing in the Terg-O-Tometer, 1 L of detergent solution was added to each Terg pot and allowed to come to temperature. Enzyme was added to the pots at varying concentrations.<br>
For BMI tests, the enzyme concentrations used were 0 mg/L, 0.275 mg/L, 0.55 mg/ml, 1.65 mg/L, 2.65 mg/L, and 5,50 mg/L. For grass stains, the nprE concentrations Used were 0 mg/L, 0.1925 mg/L, 0.385 mg/L, 1.155 mg/L, 1.925 mg/L, and 3.85 mg/L (the concentrations of BPN' Y217L were the same as those used in the BMI tests). Agitation was started and the swatches were added. All replicates were run side-by-side in the same Terg-O-Tometer (e.g., OX &amp; H X in the 1" run, IX &amp; 3X in the 2nd run, and 5X &amp; 10X in the 3rd run). The temperature was 15C, the agitation speed was 100 cpm, and the wash time was 12 minutes. The treated swatches were rinsed three times in 4 L tap water (-6 gpg). The swatches were air-dried overnight on paper towels. The grass swatches were covered with paper towels and allowed to dry in a darkened .    room. The reflectance values of the dried swatches were determined as described above. Three readings were obtained per swatch. The L values, average L value and standard deviation were recorded. This is the Las] value.<br>
The percentage of soil removal (%SR) was determined for each testing condition and both enzymes using the equation below:<br>
%SR =<br>
Where: LO = reflectance of unsoiled swatches<br>
i - reflectance of soiled swatches reflectance of washed swatches<br>
The delta %SR over no enzyme control was determined using the following formula:<br>
A%SR = ^SRtrcjtmea ' %SRno enzyme control<br>
BPN' Y217L was compared to nprE on EMPA 1 16 (BMI), at pH values of 6.7, 7.5, 8.5, and 9.5. The performance of nprE on EMPA 116 appeared to peak at about pH 8, while the performance of BPN' Y217L peaked at about pH 8.8. The results showed that nprE performed better than BPN' Y217L at pH 7.5 and 8.5, although it does not perform as well as BPN' Y217L<br>
at pH 6.7. The performance of these enzymes was equal at pH 9.5. In addition, there was no difference in the performance of these enzymes on Equest grass (med) at pH 7.8-8.4.<br>
EXAMPLE 16 Comparison of PMN and nprE Enzymes in Liquid Detergent<br>
This Example describes cleaning experiments to determine the cleaning performance of PMN and nprE. The cleaning performance of PMN and nprE enzymes were tested in Liquid TIDE detergent in comparison with a benchmark serine'protease (Protease A) on protease sensitive stains.  As shown in the table below, PMN and nprE remove stains much better than protease A, even at low enzyme levels. In the following Tables, the higher SRI values indicate a better cleaning performance.<br>
Table 16.1. Comparison of Cleaning Performance of PMN vs. Protease A in Liquid TIDE (in full size washing machine)<br>
(Table Removed ) <br>
Table 16.2. Comparison of Cleaning Performance of PMP vs. Protease A in Liquid TIDE (in mini size washing machine)<br>
(Table Removed ) <br>
Table 16.2. Comparison of Cleaning Performance of PMP vs. Protease A in Liquid TIDE (in mini size washing machine)<br>
Table 16.3. Comparison of Cleaning Performance of nprE vs. Protease A in Liquid TIDE (in mini-size washing machine)  <br>
(Table Removed ) <br>
EXAMPLE 17 Thermostability of NprE and NprE Variants<br>
In this Example, experiments conducted to determine the thermostability of NprE and NprE variants are described. The enzymes were produced and purified as described above. The purified proteins were judged to be sufficiently homogenous, with greater than 95% purity as determined using 10% SDS-PAGE, as only one major protein was observed in the gel. This protein was approximately 32 kDa, which is the molecular weight of the mature nprE sequence. The protein was formulated for storage using the 25 mM MES buffer, pH 5.8, containing 1 mM zinc chloride, 4 mM calcium chloride, and 40 % propylene glycol. The assays used in these experiments were the protease assay using fluorescence AGLA activity described above and differential scanning calorimetry (DSC), described below.<br>
Differential Scanning Calorimetrv (DSC)<br>
Excessive heat capacity curves were measured using an ultrasensitive scanning high throughput microcalorimeter VP-Cap DSC (Microcal). The standard procedure for DSC measurements and the theory of the technique is well known to those of skill in the art (See e.g.,<br>
Freire, 1995) Meth. Mol. Biol., 41,191-218 [1995]). Briefly, approx. 500 uL of 200-400 ppm pure or ultrafiltrate concentrate (UFC) protein samples were needed. Typically, 400 ppm of NprE and the variant proteins (in the absence and presence 130 mM citrate) were scanned over 20-l00 C temperature range using a scan rate of 200 C/hr in 5 mM HEPES, pH 8.0 buffer. The same sample was then rescanned to check the reversibility of the process. For NprE, the thermal unfolding process was irreversible. Scan rate dependence data of the thermal melting for NprE was assessed over a scan rate of 25 to 200C/hr. The effect of various additives (e.g., primary and secondary alcohols, salts, cyclodextrin, PEG, sorbitol, glycerol) on the thermal melting point of NprE was also assessed.<br>
The thermal stability of wild-type NprE was determined at two different concentrations, in order to show the effect of protein concentration on the thermal melting point. The Tm values for 220 ppm and 440 ppm were determined to be 67.6  0.5 and 69.2  0.5 C, respectively. The protein concentration effect highlights a second-order event It is contemplated that this is either aggregation or autolysis. However, it is not intended that the present invention be limited to any particular mechanism. Nonetheless, these results indicate that for an accurate determination and any comparison of thermal melting points for NprE require that the protein concentrations be well matched. The effect of the scan rate on the thermal melting point also showed a dependence where the Tm was dependent on the scan rate up to 150 0C/hr, and then leveled off between 150-200 C/hr. Based on these results, 200 C/hr was selected as the upper scan rate for all studies to.<br>
\		.<br>
minimize the dependence of the Tm on scan rate.<br>
All data collected for NprE and variants are shown in Table 4. Table 4 also includes the DSC thermal melting points obtained for NprE and variants in the presence of 130 mM citrate. In most cases, two protein concentrations were scanned. As indicated in this Table, in the case of the scans in the-presence of 130 mM citrate not all proteins showed a thermal unfolding profile.<br>
Table 17. DSC Results<br>
(Table Removed ) <br>
A representative Figure of the thermal unfolding profiles (DSC scans) for wild type and various mutants of NprE are shown in Figure 27. The unfolding profiles indicate the wild-type midpoint and show selective mutants that display increased thermal melting points relative to wild-type and those that display decreased melting points relative to wild-type. This Figure clearly highlights that the DSC distinguished between stable and less stable NprE variants, and is useful as a secondary screen. A general trend is observed between the thermal melting points of the variants and their stability in detergent. For example, the variants S66E, S199E, D220P/E, S129I/V are all winners in TIDE and show an approximate 1 C increase in thermal melting point relative to wild type NprE. This 1 C increase in thermal melting point is small yet significant, as thermal stability typically requires multiple amino acid substitutions.<br>
Figure 28 shows the thermal unfolding of NprE variants that display a thermal unfolding profile in the presence of 130 mM citrate. Citrate is a detergent component that rapidly causes the autolysis of NprE, in the absence of calcium. For wild-type NprE, there is no thermal unfolding profile in the presence of citrate, which is consistent with a protein that is already Unfolded or lacks a well-formed hydrophobia core. Mutants that display a thermal unfolding profile in the presence of citrate are included in Table 17. These variants have thermal melting points in the range of 47- 56 C. The DSC scans in the presence of 130 mM citrate indicated variants that are more stable than wild-type NprE to citrate. For example, citrate-stable variants are show to contain either S129I or S129V and combinatorials containing either of these substitutions show a + 5C increase in thermal melting point.<br>
Effect of Additives on the Thermal Melting Points of NprE:<br>
Figure 29 shows the results of experiments including various additives. The buffer was 5 mM HEPES, pH 8.0. The samples were scanned from 20 -100 C using a scan rate of 200C/hr. In this Figure, the horizontal line represents the Tm for wild-type NprE with no additive. In these experiments, the data showed little or no effect on the thermal melting point (Tm) of NprE in the presence of these reagents. The inclusion of an inhibitor of NprE activity, namely phosphoramidon, was shown to increase the Tm by approx. 1C, suggesting that the inhibitor may impart some stabilization to NprE against the thermal unfolding process. None of the<br>
conditions above assisted in making the thermal unfolding process reversible. However, it is not intended that the present invention be limited to any particular mechanism.<br>
EX AMPLE 18 <br>
NprE Homologue Stability in TIDE and Homolog BMI Wash Performance<br>
In this Example, experiments conducted to assess the stability of nprE homologs in TIDE, as well as the wash performance of these homologs are described. Purified NprE ("NprE"), Bacillus subtilis NprE (B.S. NprE), Bacillus thuringiensis NprB <b.t. nprb and bacillus thermoproteolyticus thermolysin were incubated in tide lomm hepes ph8 at loug for mins. the initial activities remaining measured using agla assay as described above. briefly loul of sample added into buffer mes ph6.5 tween cacl2 then diluted was substrate abz-agla-nba excitation emission nm monitored first seconds slope recorded enzyme activity. percent activity calculated by dividing over figure provides a graph showing after minutes.></b.t.>
To determine the wash performance of these homologues in TIDE, one pre-washed BMI microswatch was first added into each well of a 96-well plate. Then, 190ul of Ix compact TIDE (780ug/ml compact TIDE, 5mM HEPES, pH8, 6gpg water hardness) were added. Then, lOul of purified NprE, Bacillus subtilis NprE, Bacillus thuringiensis NprB and Bacillus thermoproteolyticus thermolysin were added to the wells to produce a final enzyme concentration is 0.25ug/ml. The plate was incubated at 25C for 30mins with shaking at 1400rpm on Thermomixer. At the end of incubation, l00ul of supernatant were transferred into a new 96-well plate. The OD at 405nm of supernatant was then measured. The supernatant OD was subtracted with the OD of a blank control without enzyme addition. The performance index was calculated by dividing the OD of each homologue to the OD of NprE. Figure 31 provides a graph showing the BMI was performance of NprE, as well as the nprE homologs described herein.<br>
EXAMPLE 19 Metal Analysis of Wild-Type nprE and Variants<br>
In this Example, experiments conducted to determine the zinc and calcium content of nprE and nprE variants are described. In these experiments, total trace metal analysis by inductively coupled plasma - mass spectrometry (ICPMS) and particle induced X-ray emission with a microfocused beam (micro-PIXE) were performed to confirm the zinc and calcium content of NprE. Overall, one zinc and two calcium ions are tightly bound.<br>
All ICPMS and micro-PIXE samples were prepared in metal free buffer to remove any exogenous metal contaminants. Typically, 250 uL of 40 mg/mL NprE samples were buffer exchanged three times with 20 mM HEPES, pH 8.2 using YM-10 microdialysis apparatus. Metal free buffer was generated by passing the buffer through a column packed with Chelax 100 resin. The final protein concentration was determined using Bicinchoninic acid protein determination assay kit (BCA assay) from Sigma. ICPMS samples were analyzed at the West Coast Analytical Services, Inc. Micro-PIXE samples were analyzed at the Ion Beam Analysis Laboratory.<br>
Table 19-1 shows ICPMS metal analysis results for calcium and zinc ions from NprE wild type. Relative to protein concentration, two calcium ions and two zinc ions were found to be present in the sample.     (Table Removed ) <br>
The MicroPKE elemental composition analysis plot measured the metal contents relative to a protein internal standard. All peaks detected using Micro-PEXE were calculated<br>
relative to the sulfur peak arising from three methionines in the case of NprE. An observed large chloride ion peak was due to the presence of salt in buffer.<br>
Table 18-2 shows the metal content determined by Micro-PIXE, which indicates that in general, NprE contains two tightly bound calcium and one zinc ion per protein molecule. Wild type NprE showed 1 zinc ion with 2 calcium ions. It is contemplated that calcium ions may have shown a low occupancy rate due to preparation of the sample. S58D and T60D showed close to two zinc ions per protein indicating a possible extra zinc ion binding to the site. The double variant has two added cysteines adding the accuracy of the technique. However, it is not intended that the present invention be limited to any particular embodiment with a specific number of ions.    <br>
Table 19-2.  Micro-PIXE Metal Determination Showing Ca and Zn Contents for NprE Native and Variants<br>
.     (Table Removed ) <br>
Consistent with other well-characterized calcium and zinc dependent neutral proteases such as thermolysin or thermolysin-like proteases (TLPsX-Se* e.g., Dahlquist et ol., Biochem., 15:1103-1111 [1976]; Srpingman et al, (1995) Biochemistry 34,15713-15720 [1995]; and Willenbrock el al., (1995) FEES Lett. 358:189-192 [1995]), NprE was found to contain at least two tightly bound calcium ions and one zinc ion per molecule. A potential third calcium binding site is proposed but expected to be very weak. Since all samples were desalted to remove any exogenous metals, these weakly bounding calcium sites are expected to be unoccupied.<br>
EXAMPLE 20 <br>
Stabilizing NprE with Calcium Formate in TIDE Compact HDL Detergent<br>
In this Example, experiments conducted to develop means to stabilize NprE in TIDE compact HDL detergent are described. In these experiments, means to stabilize NprE by increasing the calcium formate level at a fixed citrate concentration while lowering DTPA content in experimental TIDE compact formulation ('TIDE 2x") were investigated. A statistical design of experiments (DOE) methodology was used in order to simplify the . experiments as well as data analyses. It was shown that DTPA present in TIDE adversely affects NprE stability, while addition of calcium formate helps overcome the detrimental effect in the full strength TIDE compact formulation.<br>
A full central composite response surface model with duplicate center points was used as a DOE method. A total of 16 unique formulations varying four components were pre-made according to the composition variations listed in Table 19.1. LAS was varied from 0-6% (w/w) with DTPA (0 - 0.25%) and calcium-formate (0 - 0.2%) at a fixed concentration of citric acid (1.9%). All other components of the TIDE detergent were held constant. The component concentration boundary conditions were determined based on phase stability of the various mixes. The protein stability tests were conducted with 780 ppm nprE in the full strength (~ 100%) formulation mixes and incubated at 32 C. Inactivation was measured up to 24 hours. All assays were done using red fluorescent labeled casein assay kit (Invitrogen) with 0.5 ppm protein concentration. Rates of NprE inactivation were measured in three independent experiments. DOE data were analyzed using DOE Fusion Pro (S-Matrix).<br>
Table 20.1. Composition of the  16 TIDE Formulations Used for DOE Studies<br>
.     (Table Removed )  <br>
Table 20.2 and Figure 31 show the results of NprE stability measurements in various formulation mixes. Average rates and the standard deviation were the averaged NprE inactivation rate (hour"1) from three independent measurements. Qualitatively, fonnulations with low DTPA content with high calcium load tend to be more stable in the full strength compact TIDE. As an example, Formulation #5, with no addition of DTPA and high calcium formate level showed the lowest inactivation rate, indicating high NprE stability. In contrast, Formulation #9, with high DTPA concentration with no added calcium formate showed lowest stability. In Table 20.2, the ranking is based on measured stability (Le., averaged rates).  Runs are from three independent stability experiments.<br>
Table 20.2. NprE Inactivation Rates in 16 Formulation Mixes<br>
.     (Table Removed )  <br>
Figure 33 shows NprE inactivation effects by DTPA at varying levels of fixed calcium formate concentration. Panel A shows rate of NprE inactivation by DTPA without any added calcium formate. The correlation shows that DTPA has significant detrimental effect. Panel B shows some decreased effect of DTPA with 0.1% calcium formate. Panel C shows significantly decreased effect of DTPA with 0.2% calcium formate.<br>
Figure 34 shows DOE analysis software (Fusion Pro) generated prediction profile of DTPA and calcium formate composition based on response goal (decay rate) of less than 0.5 hr*1<br>
(Panel A), 0.25 hf' (Panel B) and 0.05 hr'1 (Panel C). The shaded areas indicate DTPA and<br>
calcium formate composition ratios that are predicted to show stability with decay rate below the set goal. For example, 0.16% calcium formate in the presence of 0.04% DTPA would provide NprE stability with decay rate of less than 0.25 hour"1 as shown in Panel B of Figure 34. On the other hand, 0.08% calcium formate cannot sustain NprE stability with decay rate of at least 0.25 hour"1 in the presence of 0.16% DTPA.<br>
EXAMPLE 21<br>
Identification of the Citrate-induced Autolvtic Sites for B. amyloliquefaciens Neutral<br>
Metalloprotease NprE<br>
In this Example, methods used to assess the citrate-induced autolysis of wild-type and recombinant variant nprE (e.g., B. subtilis variant) are described.  In these experiments, autolysis of the neutral metalloprotease from B. amyloliquefaciens (natural and the recombinant variant expressed in B. subtilis') was induced using sodium citrate (Sigma). The autolysis process was controlled by performing the reaction at 4C in 25 mM MES, pH 6.5, buffer. In these experiments, the autolysis of 0.4 mg/ml NprE was optimized by varying either: (a) the time of incubation (10-100 minutes) in 10 mM citrate; or (b) the citrate concentration (10-100 mM) over 100 minutes. A control of neutral metalloprotease diluted in buffer alone (i.e., no citrate) was incubated under similar conditions. The autolytic reactions were terminated by addition of an equal volume of IN HC1, the samples were precipitated using TCA and the pellet was washed and dried using acetone. The resultant pellet was resuspended in 20 uL buffer, pH 6.5, and 4X LDS sample buffer (NuPage, Invitrogen)<br>
The autolytic fragments were resolved by 10 % (w/v) SDS-PAGE and electroblotted onto a PVDF membrane. The first 10 amino acid residues were sequenced by Edman degradation (Argo Bioanalytica). The partial amino acid sequences of the autolytic fragments were determined using trypsin in-gel digestion and analyzed using LCQ-MS (Agilent). The in-gel digestion process involved macerating the gel piece that contained the protein, removal of the Coomassie blue stain followed by re-hydration of the gel pieces in 25 mM NH^COs containing 2 M urea. Trypsin was added to the re-hydrated gel pieces for approx. 6 hours at 37 C. Following the digestion, the peptides were extracted using acetonitrile and TCA. The peptides were separated on a C4-hydrophobic column (Vydac) using an acetonitrile- water gradient The<br>
resultant peptide maps were searched with the SEQUEST database search program against a database containing Genencor enzymes.<br>
The amino acid sequences of the first 10 amino acids of each of the fragments were compared with the known amino acid sequence for B. amyloliquefaciens NprE. This enabled the identification of the amino acid at the N-termini and hence the cleavage site(s).<br>
The generation of the citrate-induced fragments and their resolution was shown on 10 % SDS-PAGE. The sizes of the fragments were identified using a standard molecular weight marker from Invitrogen. In the presence of 10 mM citrate, two fragments in addition to remaining intact NprE were observed over the 100 minute time range. The two fragments formed at the low citrate concentration were found to be 24 kDa and 9 kDa in size. The intact nprE is 32 kDa. The 100-minute time range results in a good proportion of cleaved protein (i.e., the primary autolysis fragments). No additional fragments were observed or detected under these conditions. A study over 100 minutes in the presence of increasing citrate was performed to obtain the secondary autolytic fragments. In this experiment, when concentrations between 10 -30 mM citrate were used, the two fragments described above were observed. At 40 mM citrate, less of the larger 24-kDa fragments were apparent however a 15-kDa fragment was also apparent Between 50 -100 mM citrate, the 24 kDa fragment and the'9-kDa fragments were no longer detected but three other fragments, of sizes 21 kDa, 15 kDa and 11 kDa, were observed.<br>
The identity of the N-termini of the 24 kDa, 9 kDa (first two fragments), and the 21 kDa, 15 kDa and 11 kDa (the next autolytic fragments) were determined using Edman degradation (Argo Bioanalytica).  <br>
Table 21. N-Termina) Sequence of Fragments     .     (Table Removed )  <br>
Bands Al, A 3 and A4 have the native N-terminal sequence that matches the N-terminus for the intact NprE. The sequencing report for Band A2 showed three fragments where the least intense sequence appeared to be identical to the more intense sequence, except that it was two residues and one residue shorter than the more intense sequences, respectively. This was .consistent with a fraying of that particular protein fragment. The pattern and the sizes of the gel fragments suggest that the 15 kDa (Band A4) may be derived from the 21-kDa fragment (Band A3) and hence the C-terminus is deduced to be at or near position 198. However, it is not intended that the present invention be limited to this particular embodiment.<br>
Figure 35 provides the amino acid sequences for the various fragments (1-5 or Al-5 for N-terminal sequencing purposes). Fragment 1 (Al) has the N-terminal residues equivalent to that for the intact native protein (SEQ ID NO:222), fragment 2 (Ad2) N-terminus starts at or near D220 (SEQ ID NO:223). The following two amino acid residues (A221 and G222) are also highlighted because this fragment was identified as being frayed. Fragment 3 (A3) (SEQ ID NO:224) and fragment 4 (A4) (SEQ ID NO:225) have the N-terminus of the intact protein, and fragment 5 (A5) (SEQ I DNO:226) starts at LI 98, The C-terminus of fragment 4 is likely to be at or near Ml 38 (based on the size difference between A3 and A4). The corresponding fragment for A3 was not detected.<br>
Trypsin digestion followed by LCQ-MS of the peptide maps for fragments 1 through 5 positively identified several amino acid peptides within the respective fragments. These are highlighted in Figure 35. The LCQ-MS provided a positive control for the identity of the fragments.<br>
Based on the N-terminal and LCQ-MS analysis of the cleavage fragments, primary cleavage sites were identified at amino acid positions D220, A221, G222, Ml 38, and L198. These sites were targeted using site-directed mutagenesis and site-evaluation libraries of D220, A221, G222, L198, and M138, D139 were created. The mutant proteins were screened for increasing stability in detergent and for BMI-wash performance, as indicated hereia In some instances, the amino acids alongside these sites were also selected for protein engineering, in  order to ensure that the clip site was indeed targeted.<br>
The protein engineering results clearly indicated that amino acid substitutions of either Pro or GIu at D220 generated an NprE molecule that is more stable in detergent In addition, additional stability was afforded to the NprE molecule by replacing G222 with Cys, and M138 with either He or Leu. In general, these specific amino acid substitutions provided the NprE with detergent stability advantages without the BMI-wash performance being compromised. Thus, these experiments provide important mapping data for the citrate-induced autolysis sites, facilitating the identification of key amino acid residues that alter and affect the overall stability of NprE. Citrate (a builder added to detergent matrices) destabilizes and autolyses NprE and is suggested to do so by cbelating the essential calcium-bound atoms. The application of NprE in extreme detergent conditions requires that a more stable NprE molecule be used in these settings. In these experiments, substitutions of one or more of the autolytic sites of NprE have resulted in a more detergent-stable nprE molecule for use in these extreme detergents.<br>
EXAMPLE 22 Liquid Laundry Detergent Compositions<br>
In this Example, various formulations for liquid laundry detergent compositions are provided. The following liquid laundry detergent compositions of the present invention are prepared:     <br>
.     (Table Removed )  <br>
: Add IN HCI aq. soln to adjust the neat pH of the formula in the range from about 3 to about<br>
5.<br>
The pH of Examples above 22(I)-(II) is about 5 to about 7, and of 22(III)-(V) is about 7.5 to about 8.5.<br>
EXAMPLE 23 <br>
Hand Dish Liquid Detergent Compositions<br>
In this Example, various hand dish liquid detergent formulations are provided. The following hand dish liquid detergent compositions of the present invention:      <br><br>
.     (Table Removed )  <br>
The pH of Examples 23(I)-(VI) is about 8 to about 11<br>
EXAMPLE 24 Liquid Automatic Dishwashing Detergent Compositions<br>
In this Example, various liquid automatic dishwashing detergent formulations are provided. The following hand dish liquid detergent compositions of the present invention:<br>
.     (Table Removed )   <br>
EXAMPLE 2S  Granular and/or Tablet Laundry Compositions<br>
This Example provides various formulations for granular and/or tablet laundry detergents.  The following laundry compositions of present invention, which may be in the form of granules or tablet, are prepared.<br>
.     (Table Removed )  <br>
Balance to 100% Moisture and/or Minors* '<br>
* Perfume, dye, brightener / SRPl / Na carboxymethylcellulose/ photobleach / MgSO4 / PVPVI/ suds suppressor /high molecular PEG/clay.<br>
EXAMPLE 26 <br>
Liquid Laundry Detergents<br>
This Example provides various formulations for liquid laundry detergents. The following liquid laundry detergent formulations of the present invention are prepared:  <br>
.     (Table Removed )   <br>
EXAMPLE 27 High Density Dishwashing Detergents<br>
This Example provides various formulations for high density dishwashing detergents.<br>
The following compact high density dishwashing detergents of the present invention are<br>
prepared:<br>
.     (Table Removed )  <br>
 Balance to 100% Moisture and/or Minors*<br>
*Brightener / dye / SRPl / Na carboxymethylcellulose/ photobleach / MgSO4 / PVPVI/ suds suppressor /high molecular PEG/clay.<br>
The pH of Examples 27(1) through (VI) is from about 9.6 tb about 11.3.<br><br>
EXAMPLE 28 Tablet Detergent Compositions<br>
This Example provides various tablet detergent formulations. The following tablet detergent compositions of the present invention are prepared by compression of a granular<br>
dishwashing detergent composition at a pressure of 13KN/cm2 using a standard 12 head rotary press:    <br><br>
      .     (Table Removed )  <br>
*Brightener / SRP1 / Na carboxymethylcellulose/ photobleach / MgSO4 / PVPVI/ suds suppressor /high molecular PEG/clay.<br>
The pH of Examples 28(1) through 28(VII) is from about 10 to about 11.5; pH of 15(Vni) is from 8-10. The tablet weight of Examples 280) through 28(VIH) is from about 20 grams to about 30 grams.<br>
EXAMPLE 29 Liquid Hard Surface Cleaning Detergents<br>
This Example provides various formulations for liquid hard surface cleaning detergents, The following liquid hard surface cleaning detergent compositions of the present invention are prepared:    <br>
.     (Table Removed )  <br>
The pH of Examples 29(1) through (VII) is from	about 7.4 to about 9.5. <br>
While particular embodiments of the present invention have been illustrated and described, it will be apparent to those skilled in the art that various other changes and modifications can be made without departing from the spirit and scope of the invention. It is therefore intended to cover in the appended claims all such changes and modifications that are within the scope of this invention.<br>
All patents and publications mentioned in the specification are indicative of the levels of those skilled in the art to which the invention pertains. All patents and publications are herein incorporated by reference to the same extent as if each individual publication was specifically and individually indicated to be incorporated by reference.<br>
Having described the preferred embodiments of the present invention, it will appear to those ordinarily skilled in the art that various modifications may be made to the disclosed embodiments, and that such modifications are intended to be within the scope of the present invention.<br>
Those of skill in the art readily .appreciate that the present invention is well adapted to carry out the objects and obtain the ends and advantages mentioned, as well as those inherent therein. The compositions and methods described herein are representative of preferred embodiments, are exemplary, and are not intended as limitations on the scope of the invention. It is readily apparent to one skilled in the art that varying substitutions and modifications may be made to the invention disclosed herein without departing from the scope and spirit of the invention.<br>
The invention illustratively described herein suitably may be practiced in the absence of any element or elements, limitation or limitations which is not specifically disclosed herein. The terms and expressions which have been employed are used as terms of description and not of limitation, and there is no intention that in the use of such terms and expressions of excluding any equivalents of the features shown and described or portions thereof, but it is recognized that various modifications are possible within the scope of the invention claimed. Thus, it should be understood that although the present invention has been specifically disclosed by preferred embodiments and optional features, modification and variation of the concepts herein disclosed may be resorted to by those skilled in the art, and that such modifications and variations are considered to be within the scope of this invention as defined by the appended claims. The invention has been described broadly and generically herein. Each of the narrower species and subgeneric groupings falling within the generic disclosure also form part of the invention. This includes the generic description of the invention with a proviso or negative limitation removing any subject matter from the genus, regardless of whether or not the excised material is specifically recited herein.<br><br><br><br>
CLAIMS<br>
We Claim:<br>
1.	An isolated neutral metalloprotease having improved storage stability, wherein<br>
said neutral metalloprotease is a Bacillus neutral metalloprotease.<br>
2.	The neutral metall oprotease of Claim 1, wherein said Bacillus is B.<br>
amyloliquefaciens.<br>
3.	The neutral metalloprotease of Claim 1, wherein said neutral metalloprotease has<br>
at least 45% amino acid identity with the neutral metalloprotease comprising SEQ ID NO:3 or<br>
SEQ ID NO:18.<br>
4.	The neutral metalloprotease of Claim 1, wherein said neutral metalloprotease<br>
comprises the amino acid sequence set forth in SEQ ID NO:3 or SEQ ID NO: 18.<br>
5.	A nucleotide sequence encoding at least a portion of the neutral metalloprotease<br>
of Claim 1, wherein said nucleotide sequence is selected from of SEQ ID NOS: 1,2,12, and 13.<br>
6.	An expression vector comprising a nucleotide sequence encoding the amino acid<br>
sequence set forth in SEQ ID NO:3.<br>
7.	A host cell comprising the expression vector of Claim 6.<br>
8.	A storage-stable neutral metalloprotease obtained from said host cell of Claim 7,<br>
wherein said neutral metalloprotease is encoded by said expression vector.<br>
9.	An isolated neutral metalloprotease having immunological cross-reactivity with<br>
the neutral metalloprotease of Claim 1.<br>
10.	The amino acid sequence set forth in SEQ ID N0:18.<br>
11.	The amino acid sequence set forth in SEQ ID N0:3.<br>
12.	The amino acid sequence of Claim 10, wherein said sequence comprises at least<br>
one substitution of an amino acid made at a position equivalent to a position in a neutral<br>
metalloprotease comprising the amino acid sequence set forth in SEQ ID NO: 18, wherein said<br>
positions are selected from positions 1, 3,4,5,6,11,12,13,14,16,21,23,24,25,31,32, 33,<br>
35,36, 38,44,45,46,47,48,49, 50,51,54, 55,58,59,60,61,62,63,65, 66, 69,70,76,85, 86,<br>
87,88, 90,91, 92,96,97, 98, 99,100,102,109,110, 111, 112,113,115,117,119,127,128,<br>
129,130,132,135,136,137,138,139, 140,146,148,151,152,153,154,155,157,158,159,<br>
161,162,169,173,178,179,180,181,183,184,186,190,191,192,196,198,199,200,202,<br>
203,204,205,210,211,214,215,216,217,218, 219,220,221,222,223, 224,228,229, 237,<br>
239,240, 243,244, 245, 248,252,253,260,261, 263,264,265,267,269, 270,273,277, 280,<br>
282,283, 284,285, 286,288, 289,290,292,293,296,297, and 299.<br>
13.	An isolated neutral metalloprotease variant having an amino acid sequence<br>
comprising at least one substitution of an amino acid made at a position equivalent to a position<br>
in a neutral metaJloprotease comprising the amino acid sequence set forth in SEQ ID NO: 18.<br>
14.	The isolated neutral metaloprotease variant of Claim 13, wherein said<br>
substitutions are made at positions equivalent to positions 1,3,4,5,6,11,12,13,14,16,21,23,<br>
24,25,31,32,33,35,36, 38, 44,45,46,47,48,49, 50, 51, 54,55,58,59, 60, 61,62,63,65,66,<br>
69,70,76,85,86, 87, 88, 90, 91, 92,96,97,98,99,100,102,109,110, 111, 112,113,115,117,<br>
119,127,128,129,130,132,135,136,137,138,139,140,146,148,151,152,153,154,155,<br>
157,158,159,161,162,169,173,178,179,180,181,183,184,186,190,191,192,196,198,<br>
199,200,202,203,204,205, 210,211,214,215, 216,217, 218, 219, 220,221,222,223,224,<br>
228,229,237,239,240,243, 244,245,248,252,253,260, 261,263, 264, 265,267,269,270,<br>
R110Q,R110S,R110Y,D119E,D119H,D119I,D119L,D119Q,D119R,D119S,D119T, D119V, D119W, G128C, G128F, G128H, G128K, G128L, G128M, G128N, G128Q, GJ28R, G128W, G128Y, S129A, S129C, S129D, S129F, S129G, S129H, S129I, S129K, S129L, S129M, S129Q, S129R, S129T, S129V, S129W, S129Y, F130I. F130K, F130L, F130M, F130Q, F130R, F130T, F130V, F130Y, S135P, G136I, G136L, G136P, G136V, G136W, G136Y, S137A, M138I, M138K, M138L, M138Q, M138V, D139A, D139C, D139E, D139G, D139H, D139I, D139K, D139L, D139M, D139P, D139R, D139S, D139V, D139W, D139Y, V140C, Q151I, E152A, E152C, E152D, E152F, E152G, E152H, E152L, E152M, E152N, E152R, E152S, E152W, N155D, N155K, N155Q, N155R, D178A, D178C, D178G, D178H, D178K, D178L, D178M, D178N, D178P, D178Q, D178R, D178S, D178T, D178V, D178W, D178Y, T179A, T179F, T179H, T179I, T179K, T179L, T179M, T179N, T179P, T179Q, T179R, T179S, T179V, T179W, T179Y, E186A, E186C, E186D, E186G, E186H, E186K, E186L, E186M, E186N, E186P, E186Q, E186R, E186S, E186T, E186V, E186W, E186Y, V190H, V190I, V190K, V190L, V190Q, V190R, S191F, S191G, S191H, S191I, S191K, S191L, S191N, S191Q, S191R, S191W, L198M, L198V, S199C, S199D, S199E, S199F, S199I, S199K, S199L, S199N, S199Q, S199R, S199V, Y204H, Y204T, G205F, G205H, G205L, G205M, G205N, G205R, G205S, G205Y, K211A, K211C, K211D, K211G, K211M, K211N, K211Q, K211R, K211S, K211T, K211V, K214A, K214C, K214E, K214I, K214L, K214M, K214N, K214Q, K214R, K214S, K214V, L216A, L216C, L216F, L216H, L216Q, L216R, L216S, L216Y, N218K, N218P, T219D, D220A JD220E, D220H, D220K, D220N, D220P, A221D, A221E, A221F, A221I, A221K, A221L, A221M, A221N, A221S, A221V, A221Y, G222C, G222H, G222N, G222R, Y224F, Y224H, Y224N, Y224R, T243C, T243G, T243H, T243I, T243K, T243L, T243Q, T243R, T243W, T243Y, K244A, K244C, K244D, K244E, K244F, K244G, K244L, K244M, K244N, K244Q, K244S, K244T, K244V, K244W, K244Y, V260A, V260D, V260E, V260G, V260H, V260I, V260K, V260L.V260M, V260P, V260Q, V260R V260S, V260T, V260W, V260Y, Y261C, Y261F, Y261I, Y261L, T263E, T263F, T263H, T263I, T263L, T263M, T263Q, T263V, T263W, T263Y, S265A, S265C, S265D, S265E, S265K, S265N, S265P, S265Q, S265R.S265T, S265V, S265W, K269E, K269F, K269G, K269H, K269I, K269L, K269M, K269N, K269P, K269Q, K269S, K269T, K269V, K269W, K269Y, A273C, A273D, A273H, A273I, A273K, A273L, A273N, A273Q, A273R, A273Y, R280A, R280C, R280D, R280E,<br>
R280F, R280G, R280H, R280K, R280L, R280M, R280S, R280T, R280V, R280W, R280Y, L282F, L282G, L282H, L282I, L282K, L282M, L282N, L282Q, L282R, L282V, L282Y, S285A, S285C, S285D, S285E, S285K, S285P, S285Q, S285R, S285W, Q286A, Q286D, Q286E, Q286K, Q286P, Q286R, A289C, A289D, A289E, A289K, A289L, A289R, A293C, A293R, N296C, N296D, N296E, N296K, N296R, N296V, A297C, A297K, A297N, A297Q, A297R, and G299N.<br>
16.       The isolated variant neutral metalloprotease of Claim 13, wherein said protease comprises multiple mutations selected from S023W/G024M, T004V/S023W/G024W, S023W/G024Y/A288V, T004L/S023W/G024Y, N046Q/N050F/T054L, N050Y/T059R/S129Q, S023W/G024W, A273H/S285P/E292G, S023Y/G024Y, S023Y/G024W, T004S/S023Y/G024W, N046Q/T054K, S023W/G024Y, T004V/S023W, T059K/S066N, N046Q/N050W/T054Hm53A, T004V/S023W/G024Y, L282M/Q286P/A289R, N046Q/R047K/N050Y/T054K, L044Q/T263W/S285R, T004L/S023W/G024W, R047K7N050F/T034K, A273H/S285R, N050Y/T059K/S066Q, T054K/Q192K, N046Q/N050W, L282M/Q286K, T059K/S066Q, T004S/S023W, L282M/Q286R/A289R/K011N, L282M/A289R, N046Q/N050W/T054H, T059K/S129Q, T004S/S023N/G024Y/F210L,     -T004V/S023W/G024M/A289V, L282M/Q286K/A289R/S132T, N050W/T054H, L282M/Q286R, L282F/Q286K/A289R, T059R/S066Q, R047K/N050W/T054H, S265P/L282M/Q286K/A289R, L282M/Q286R/T229S, L282F/Q286K, T263W/S285R, S265P/L282M/Q286K, T263H/A273H/S285R, T059R/S129V, S032T/T263H/A273H/S285R, T059R/S066Q/S129Q, T004S/G024W, T004V/S023W/G024M, T059K/S066Q/S129Q, L282M/Q286K/A289RA253V, T004V/S023Y/G024W, T059R/S066N/S129Q, N050F/T054L, T004S/S023N/G024W, T059R/S066N, T059R/S066N/S129V, Q286R/A289R, N046Q/R047K/N050F/T054K, S265P/L282M/Q286P/A289R, S265P/L282M/Q286R/A289R Q062K/S066Q/S129I, S023N/G024W, N046Q/R047K7N050W/T054H, R047K/T054K, T004L/G024W, T014MT059R/S129V, T059R/S066Q/N092S/S129I, R047K/N050Wm)54K, T004V/G024W, N047Km050F/T054K, S265P/L282F7Q286KyN061Y, L282F/Q286K/E159V, T004V/S023Y/G024M, S265P/L282F/A289R/T065S, T059K/F063L/S066N/S129V, T004US023W, N050F/T054H, T059RVS066Q/S129V, V190I/D220E/S265W/L282F,<br>
T004S/S023Y/G024M, T004L/S023N/G024Y, T059K/S066N/S129I, T059R/S066N/S129I, L282M/Q286R/A289R/P162S, N046Q/N050F/T179N, T059K/Y082C/S129V, T059K/S129I, N050Y/T054K, T059K/S066Q/V102A7S129Q, T059R/S066Q/S129I, T059W/S066N/S129V/S290R, T059R/S129I, T059K/S066Q/S129I, T059K/S066Q/S129V, S265P/L282M/Q286R/A289R/T202S/K203N, T004V/S023N/G024W, S265P/Q286K, S265P/L282F/A289R, D220P/S265W, LQ55F/T059W/S129V, T059R/S129Q/S191R, N050W/T054K, T004S/S023W/G024M, R047K/N050F/T054H, T059K/S066N/K088E, T059K/S066Q/S129I/V291L, L282M/Q286R/A289R, T059R/S066N/F085S/S129I, L282F/Q286P/A289R, L282F/Q286R/A289R, G099D/S265P/L282F/Q286K/A289R, N046Q/N050F, N050Y/T059W/'S066N/S129V, T009I/D220P/S265N, V190F/D220P/S265W, Nl 57Y/T263W/A273H/S285R, T263W/A273HyS2S5R, T263W/S285W, T004V/S023Y, N046Q/R047KyN050W, N050W/T054L, N200Y/S265P/L282F/Q286P/A289R, T059R/S066Q/P264Q, T004V/G024Y, T004L/G024Y, N050Y/S191I, N050Y/T054L, T004L/S023W/G024Y/N155K, F169I/L282F/Q286R/A289R, L282M/Q286K/A289R, Fl 3017M138L/E152W/D183N, N046Q/R047K/N050Y/T054H, T004V/G024M, N050Y/T059W/S066Q/S129V, S023N/G024Y, T054H/P162Q, T004S/S023W/G024Y, N050Y/T054H, L282F/Q286R/A289R/F169I, R047K/N050W, V190F/D220P, L282M/F173Y, T004I7S023Y, N050W/A288D, V190I/D220P/S265Q, S265P/L282F/Q286P/A289R, . S265P/L282F/Q286R/A289R, N046Q/N050Y/T054K, T059W/S066Q, T263W/A273H/S285W T263W/A273H/S285P, S023Y/G024M, T004L/S023N/G024W, T004V/S023N/G024Y, T059W/S066N/S129Q, T004S/S023Y, T004S/S023N/G024M, T059W/S066N/A070T, T059W/S066Q/S129Q, T263W/A273H, A273H^85P, N046Q/R047KyN050Y/T054L, N046Q/R047K/N050Y, R047K/N050Y, T263H/S285W, R047K/N050F, N046Q/R047K/N050F/T054H, S023N/G024M, T004S/G024Y, R047K/N050Y/T054H, T059W/S066N/S1291, R047KH-054L, T004S/S023W/G024W, M13817E152F/T146S,. D220P/S265N, T004S/G024M, T004V/S023N, N046Q/N050F/T054K, N046Q/N050Y/T054H, Q062H/S066Q/S129Q, T059W/S129Q, T059W/S129V, N050F/T054K, R047K/N050F/T054L, V190I/D220P/S265W, N112LT263H/A273H/S285R, T059W/S066N/S129V, T059W/S066Q/S129I, T059W/S129I, T263W/S285P, V1901/D220P, A289V/r263H/A273H, T263H/A273H/S285P, N90S/A273H/S285P, R047K/N050Y/T054L,<br>
T004S/S023N, T059R/S129Q, N046Q/R047K/T054H, T059W/S066Q/S129V, E152Wm79P, N050Y/S066Q/S129V, T202S/T263W/A273H, T263W/A273H/S285P, M138L/E152W/T179P, N046Q/R047K, N046Q/T054H/F176L, T004L/G024M, T004S/L282M, T263H/A273H, T263H/A273H/S285W, T004L/S023Y/G024M, L282F/Q286P, T004V/S023Y/G024Y, V190F/S265W, M138L/E152F, V190F/D220E/S265W, N046Q/N050F/T054H, N157Y/S285W, T004F/S023YA3024M, T004V/S023N/G024M, L198I/D220E/S265Q,<br>
N046Q/N050Y/T054K/A154T, S016L/D220E/S265W, D220E/S265W, D220E/A237S/S265W, S066Q/S129Q, V190F/D220E/S265Q/T267I, L282M/F173Y/T219S, E152F/T179P, V190I/S265W, M138L/S066Q, M138L/E152W, T059W/S066Q/A070T/S129I, V190F/D220E/S265N, V190F/S265N, N046Q/N050Y, and M138UE152F/T179P.<br>
17.       The isolated variant neutral metalloprotease of Claim 13, wherein said protease comprises multiple mutations selected from V190I/D220P, V190I/D220P/S265Q, V190L/D220E, V1901/D220E/S265Q, V1901/D220E/S265W/L282F, V190UD220E/S265Q, V190LTD220E/S265W, V190UD220E/S265N, T059R/S066Q/S129I, V190I/D220E/S265N, V190L/D220E/S265W, V190I/D220E, T059W/S066N/S129V, T059K/S066Q/S129V, 1059X^0820/8129V, T059RyS066N/S129I, S066Q/S129V, T059R/S066Q/S129V, T059R/S129I, N050Y/T059W/S066N/S129V, D220P/S265N, S066Q/S129I, T059W/S066Q/S129V, T059K/S066Q/S1291, T059R/S129V, N050Y/S066Q/S129V, T059W/S066Q/S129I, N050Y/T059W/S066Q/S129V, T059K/S129I, D220P/S265W, F1301JM138iyri79P, S066N/S129I, T059R/S066N/S129V, F130VM138LTT179P, T059R/S066Q/N092S/S129I, S066N/S129V, D220E/S265Q, F130L/M138UE152W/T179P, T059W/S129V, S265P/L282M/Q286R/A289R, S265P/L282F/Q286R/A289R, T059W/S066N/S129I, V1901/D220P/S265W, F130L/E152W/T179P, F130L/M138UE152F/T179P, Q062K/S066Q/S129I, T059K/S066N/S129I, E152H/T179P, S265P/L282M/Q286K/A289R, F130IJM138I7E152H/T179P, T263W/A273H/S285R, D220E/S265N, Fl 301/M138L/E152H/T179P, F130V/M13817E152W/T179P, F130iyM138LyE152W/ri79P, T059W/S129I, D220E/S265W, F130V/M138IJT179P, F130L/E152V/T179P, T059R/S129Q, T263W/S285P, F130I/M138L/E152F/T179P,<br>
E152W/T179P, V190L/S265Q, F130L/E152F/T179P, L282M/Q286R/A289R/P162S, D220P/S265Q, M138L/E152F/T179P, F130I/E152H/T179P, M13817E152W/T179P, F130UT179P, F130L/M138L/E152W/T179P/Q286H, F130L/M138L/E152H, T263W/A273H/S285W, S265P/Q286K, T059W/S066Q/S129Q, T263W/S285R, T059W/S066N/S129Q, T263W/S285W, T059R/S066N/S129Q, S265P/L282M/Q286R/A289R/T202S/K203N, T059W/S129Q, Q062H/S066Q/S129Q, L282M/Q286R/A289R, V190L/D220E/S265N/V291I, V190L/S265N, F130L/M138L/E152W, N050Ym)59R/S129Q, F130I/T179P, T059K/S066Q/S129Q, T059K/S129Q, S265P/L282M/Q286P/A289R, S265P/L282F/Q286P/A289R, T263W/A273H/S285P, S265P/L282MyQ286K, S01617D220E/S265W, S066Q/S129Q, S265P/L282M/Q286P, L282F/Q286R/A289R, F130V/E152W/T179P, L044Q/T263W/S285R LX)55F/T059W/S129V, V190L/S265W, Q286R/A289R, G99D/S265P/L282F/Q286K/A289R, F130L/M138L/E152F, T059R/S066Q/S129Q, F130L/E152H, S066N/S129Q, T004S/S023N/G024M/K269N, S265P/L282M, E152F/T179P, T059W/S066N/S129V/S290R, L282F/Q286K/A289R, F130I^M138L, F130LM138iyE152W, S265P/L282F, F130I/M138UE152H, F130V/M138L/E152H, V190I/S265Q, M138IJE152M, S265P/L282F/Q286P, M138L/E152H, T059K/S066N/K088E, V190I/S265W, F1301/E152W, L282M/Q286K/A289R, L282M/Q286K/A289R/I253V, T263W/A273H, V190I/S265N, M138L/E152W, A273H/S285R, F130I/M138L, F13017E352F, F130V/M138L/E152W, T059K/S066QA^102A/S129Q, F130V/E152H/T179P, F1301/M138L/E152F, F130V/M138L/E152F, M138L/E152F.. L282M/Q286R, F130I/EI52H, S265P/L282F/A289R/TO65S, T263H/A273H/S285R, F130V/M138L, T014MT059R/S129V, L282MyQ286R/A289R/KllN, A273H/S285P, L282M/Q286K/A289R/S132T, T263H/A273H/S285W, F130V/E152W, S265P/L282F/Q286K^f061Y, F130I/E152W, L198M&gt;220E/S265Q, V190I/S265L, T263H/S285W, S265P/L282F/A289R, M138L/S066Q, F130I/E152F, N90S/A273H/S285P, S032T/T263H/A273H/S285R, L282F/Q286P/A289R, N157Y/T263W/A273H/S285R, V105A/S129V, T263H/A273H/S285P, S129Q/L282H, T059W/S066Q, F130V/E152H, S023W/G024Y, T004V/S023N, T059R/S066Q, N050W/T054L, L282M/Q286P/A289R, A115VA^190L/S265W, L282M/Q286K, T059R/S066N, L282F/Q286P, T004V/S023W/G024M, S265P/L282F/Q286R/L78H, L2S2FyQ286K, T004V/S023W/G024Y, S023W/G024M,<br>
T059R/R256S, F130V/E152F, T004V/G024W, N050W/T054K, S023Y/G024M, T004V/S023Y, T004V/S023Y/G024M, N050Y/T054H, S023W/G024W, T004V/S023Y/G024Y, T004V/S023N/G024W,F130L/M138L/E152F/T179PA^291I,N050Y/T059K/S066Q, T004V/S023Y/G024W, T059KyS066N, T004V/S023N/G024Y, S023Y/G024W, N050FflX)54L, R047K/T054K, S023N/G024W, L282M/A289R, S023Y/G024Y, T004V/G024M, R047K/N050F/T054K, N050F/T054K, T059K/S066Q, S023N/G024M, S023N/G024Y, T004L/S023N, R047K7N050W/T054H, T004L/S023W/G024Y, T004S/S023W, N046Q/N050W/T054H/A142T, T004L/S023Y, T004V/S023W, N050W/T054H, T004S/S023N, T004S/L282M, T004L/S023W, N050F/T054H, N050Y/T054L, and R047K/N050W/T054K.<br>
18.       The isolated variant neutral metalloprotease of Claim 13, wherein said protease comprises multiple mutations selected from S066Q/S129V, S066Q/S129I, N050Y/S066Q/S129V, S066N/S129I, T059K/S066Q/S129V, S066N/S129V, F130L/E152W/T179P, S265P/L282M/Q286R/A289R, F130L/E152V/T179P, T059K/S066Q/S129I, T263W/S285P, T059K/S066N/S129I, T263W/A273H/S285P, S265P/L282F/Q286R/A289R, F130V/E152W/T179P, T263W/A273H/S285R, V190I^&gt;220P;S265W, F130UE152H, S066N/S129Q, S265P/L282M/Q286K/A289R, V190I/D220E, T059R/S066N/S129I, V190yD220E/S265W, T059K/S129I, T059R/S066Q/S129I, F1301/M138L/E152H/T179P, F130in-179P, T263W/A273H/S285W, S016L/D220E/S265W, S066Q/S129Q, V190I/D220E/S265Q, T059R/S066Q/S129V, D220E/S265N, V190IJD220E, D220E/S265W, V190I/D220P, V190M)220E/S265N, L044Q/T263W/S285R, S265P/L282M/Q286P/A289R, F130UM138UE152H/T179P, T263W/S285R, L282M/Q286R;A289R, T263W/S285W, F130I/E152H/T179P, V190MD220E/S265N, V190UD220E/S265W, V190I/D220P/S265Q, T059R/S066N/S129V, V190L/D220E/S265Q, E152H/T179P, F130L/M138iyE152F/T179P, Q062H/S066QAS129Q, T059R/S129V, V190I/D220E/S265W/L282F, V1901/S265Q, F130L/E152F/T179P, D220E/S265Q, E152W/T179P, T059K/S066Q/S129Q, F130L/M138L/T179P, F130I/M138L/E152F/T179P, F130UMl38iyE152W^179P, N050Y/T059W/S066Q/S129V, S265P/L282M/Q286K, T059R/S129I, F130V/E152H/T179P, D220P/S265N, S265P/L282M/Q286P, F130I/E152H, T059R/S066Q/N092S/S129I, F130L/T179P,<br>
G99D/S265P/L282F/Q286K/A289R, T263W/A273H, V190I/S265N, D220P/S265W, F130L/E152W, F130L/M138L/E152H, S265P/L282M, V190I/S265Q, F130L/E152F, T059K/S129Q, Q286R/A289R, M138L/E152W/T179P, F130I/M138L/E152H, D220P/S265Q, V190L/S265N, F130I/M138UE152W, S265P/Q286K, V190US265Q, V190I/S265W, F130L/M138L/E152F, F130V/E152H, E152F/T179P, N050Y/T059W/S066N/S129V, T059R/S066N/S129Q, Fl 30I/E152W, F130V/E152W, T059R/S066Q/S129Q, T263H/A273H/S285P, N90S/A273H/S285P, V190L/D220E/S265N/V291I, T059R/S129Q, A273H/S285P, F130I/M138L/E152W/T179P, F130V/M138L/E152F, N050Y/T059R/S129Q, T059W/S066Q/S129I, F130VM138L/T179P, F130V/M138L/E152W/T179P, V190I7S265W, F130V/M138L/E152W, T059W/S066Q/S129V, V190I/S265Q, F130V/M138L/E152H, F130I/E152F, N157Y/T263W/A273H/S285R, T263H/S285W, M138L/E152F/T179P, Al 15V/V190L/S265W, M138L/E152M, T263H/A273H/S285W, F130L/M138L/E152W, T059K/S066N/K088E, Fl 301/M138L/E152F, F130I/M138L/T179P, T004V/S023N, T059K/S066QA'302A/S129Q, F130L/M138L, N047KyN050F/T054K, T263H/A273H/S285R, F130L/M138L/E152W/T179P/Q286H, M138L/E152H, M138L/S066Q, L282M/Q286R/A289R/P162S, L282F/Q286R/A289R, Q062KyS066Q/S129I, A273H/S285R, S265P/L282F/Q286P, S265P/L282F/Q286P/A289R,<br>
S265P/L282M/Q286R/A289RA-202S/K203N, T059W/S066N/S1291, V190I/S265L, T059W/S066N/S129V, F130IM138L, L282M/Q286K/A289R^253V, R047K/N050F/T054K, M138UE152F, N050W/T054K, L198yD220E/S265Q, L282F/Q286K/A289R, N050F/T054K, L282M/Q286R, M138L/E152W, S265P/L282F, F130V/E152F, T059W/S066N/S129Q, F130V/M138L, T263H/A273H, L282M/Q286K/A289R, N046Q/N050W/T054H/A142T, T059W/S066Q/S129Q, S265P/L282F/A289RA-065S, N050F/T054H, S129Q/L282H, L282M/Q286K/A289R/S132T, L282M/Q286R/A289R/K1 IN, T059K/S066N, R047K/N050W/T054K, T059K/S066Q, T004V/S023Y, T059W/S066N/S129V/S290R, N050Y/T059K/S066Q, and R047K/N050Y.<br>
19.       A variant neutral metalloprotease having improved performance as compared to wild-type B. amyloliquefaciens neutral metalloprotease.<br>
20.	The variant neutral metalloprotease of Claim 19, wherein said improved<br>
performance comprises improved thermostability, as compared to wild-type B. amyloliquefaciens<br>
neutral metalloprotease.<br>
21.	The variant neutral metalloprotease of Claim 19, wherein said improved<br>
performance comprises improved performance under lower or higher pH conditions, as compared<br>
to wild-type B. amyloliquefaciens neutral metalloprotease.<br>
22.	The variant neutral metalloprotease of Claim 19, wherein said improved<br>
performance comprises improved autolytic stability, as compared to wild-type B.<br>
amyloliquefaciens neutral metalloprotease.		<br>
23.	A method for producing an enzyme having neutral metalloprotease activity,<br>
comprising: transforming a host cell with an expression vector comprising a polynucleotide<br>
having at least 70% sequence identity to SEQ ED NO:3; and cultivating said transformed host cell<br>
under conditions suitable for the production of said neutral metalloprotease.<br>
24.	The method of Claim 23, further comprising the step of harvesting said produced<br>
neutral metalloprotease.<br>
25.	The method of Claim 23, wherein said host cell is a Bacillus species.<br>
26.	An isolated nucleic acid probe comprising 4 to 150 nucleotide sequence<br>
substantially identical to a corresponding fragment of SEQ ID NOS:1,2,12 and/or 13.<br>
27.	The isolated nucleic acid probe of Claim 26, wherein said probe is used to detect<br>
a nucleic acid sequence coding for an enzyme having metalloproteolytic activity.<br>
28.	The nucleic acid sequence of Claim 27, wherein said nucleic acid sequence is<br>
obtained from a Bacillus sp.<br>
29.	A composition comprising at least one neutral metalloprotease obtained from B.<br>
amylolujuefaciens, wherein said composition further comprises at least one ion calcium and/or<br>
zinc ion.	.<br>
30.	A composition comprising at least one neutral metalloprotease obtained from B,<br>
amyloliquefaciens, wherein said composition further comprises at least one stabilizer.<br>
31.	The composition of Claim 30, wherein said stabilizer is selected from the group<br>
consisting of borax, glycerol, zinc ions, calcium ions, and calcium formate.<br>
32.	The composition of Claim 30, wherein said stabilizer is at least one competitive<br>
inhibitor that stabilizes said at least one neutral metalloprotease in the presence of an anionic<br>
surfactant.<br>
33.	A cleaning composition comprising the isolated neutral metalloprotease of Claim<br>
1.<br>
34.	The cleaning composition of Claim 33, wherein said cleaning composition is a<br>
detergent.<br>
35.	The cleaning composition of Claim 33, further comprising at least one additional<br>
enzymes or enzyme derivatives selected from proteases, amylases, lipases, mannanases,<br>
pectinases, cutinases, oxidoreductases, hemicellulases, and cellulases<br>
36.	The cleaning composition of Claim 34, wherein said composition comprises at<br>
least about 0.0001 weight percent of said neutral metalloprotease.<br>
37.	The cleaning composition of Claim 36, wherein said composition comprises<br>
from about 0.001 to about 0.5 weight percent of said neutral metalloprotease.<br>
38.	The cleaning composition of Claim 36, wherein said composition further<br>
comprises at least one adjunct ingredient.<br>
39.	The cleaning composition of Claim 31, further comprising a sufficient amount of<br>
a pH modifier to provide the composition with a neat pH of from" about 3 to about 5, the<br>
composition being essentially free of materials that hydrolyze at a pH of from about pH 3 to<br>
about pH 5.<br>
40.	The cleaning composition of Claim 39, wherein said materials that hydrolyze at a<br>
pH of from about pH 3 to about pH 5 comprise at least one surfactant.<br>
41.	The cleaning composition of Claim 40, wherein said surfactant is a sodium alkyl<br>
sulfate surfactant comprising an ethylene oxide moiety.<br>
42.	The cleaning composition of Claim 33, wherein said composition is a liquid.<br>
43.	The cleaning composition of Claim 33, further comprising at least one acid-<br>
stable enzyme.<br>
44.	A method of cleaning, comprising the step of contacting a surface and/or ah<br>
article comprising a fabric with the cleaning composition of Claim 33.<br>
45.	The method of Claim 44, further comprising the step of rinsing said surface<br>
and/or material after contacting said surface or material with said cleaning composition.<br>
46.	The method of Claim 44, wherein said surface and/or an article comprising a<br>
fabric comprises a grass stain and said step of contacting comprises said grass stain with said<br>
cleaning composition.<br>
47.	An animal feed composition comprising the isolated neutral metalloprotease of<br>
Claim 1.<br>
48.	A textile processing composition comprising the isolated neutral jnetalloprotease<br>
of Claim 1..<br>
49.	A leather processing composition comprising the isolated neutral<br>
metalloprotease of Claim 1.                            <br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUFic3RyYWN0LSgyMi0wNy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Abstract-(22-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUFic3RyYWN0LSgzMC0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Abstract-(30-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNsYWltcy0oMjItMDctMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Claims-(22-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(03-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(09-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(14-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTQtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(14-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMDctMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(22-07-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(26-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondence Others-(30-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNyZSBPdGhlcnMtKDMwLTA0LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Correspondencre Others-(30-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LURyYXdpbmdzLSgzMC0wOS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Drawings-(30-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUZvcm0tMi0oMzAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Form-2-(30-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUZvcm0tMy0oMzAtMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Form-3-(30-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LUdQQS0oMTQtMDMtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-GPA-(14-03-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcxMS1kZWxucC0yMDA4LVBldGl0aW9uLTEzNy0oMzAtMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">2711-delnp-2008-Petition-137-(30-04-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="263562-hybrid-components-containing-reactive-hotmelt-adhesives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263564-reduced-number-of-channels-decoding.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263563</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2711/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>45/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>07-Nov-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Oct-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Apr-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENENCOR INTERNATIONAL , INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>925 PAGE MILL ROAD, PALO ALTO, CA 94304 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SHAW ANDREW</td>
											<td>2560 HYDE STREET, SAN FRANCISCO, CA 94109 USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WALLACE LOUSE</td>
											<td>2275 SHARON ROAD #302, MENLO PARK, CALIFORNIA 94025 USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ESTELL DAVID A.</td>
											<td>248 WOODBRIDGE CIRCLE, SAN MATEO, CALIFORNIA 94403, USA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HOMESH RONALD</td>
											<td>HOGERWOERDSTRAAT 54, 2023 VD HAARLEM, NL-2023 HAARLEM THE NETHERLAND.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LEE SANG-KYU</td>
											<td>2521 COWPER STREET, PALO ALTO, CALIFORNIA 94301 USA.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>SADLOWSKI EUGENE STEVEN</td>
											<td>9980 PEBBLEKNOLL SRIVE, CINCINNATI, OHIO 45252 USA.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>OH HIROSHI</td>
											<td>8541 MEADOW BLUFF COURT, CINCINNATI, 45246 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/43</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/040695</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-10-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/726,448</td>
									<td>2005-10-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263563-variant-neutral-metalloproteases-having-improved-thermostability-wash-performance-or-autolytic-stability by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:12:46 GMT -->
</html>
