<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/263715-gram-positive-fusion-flagellin-protein-producing-bacteria by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:25:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 263715:GRAM POSITIVE FUSION FLAGELLIN PROTEIN PRODUCING BACTERIA</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">GRAM POSITIVE FUSION FLAGELLIN PROTEIN PRODUCING BACTERIA</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides f lagellin-based fusion proteins (FBFP) useful for a variety of purposes, in bioremediation to remove metal ions from a liquid, to express an enzyme or an immunogen, nucleic acids encoding the FBFP, vectors containing the nucleic acids, and host cells harboring the vectors. Furthermore, the invention provides methods for obtaining over-expression and surface display of heterologous polypeptides in Gram-positive bacterial cells in Bacillus halodurans in particular. In addition, the invention features gene-disrupted bacteria useful for expressing the recombinant FBFP on their surfaces. Also included in the invention are genetic constructs useful for making FBFP and methods of using the FBFP.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>1<br>
GRAM POSITIVE RECOMBINANT PROTEIN PRODUCING BACTERIA<br>
TECHNICAL FIELD<br>
This invention relates to recombinant proteins, and more particularly to recombinant<br>
proteins produced by gram positive bacteria.<br>
BACKGROUND<br>
Bacterial recombinant protein production is most commonly performed in gram negative<br>
bacteria and, in particular, Escherichia coli bacteria. There is a need to develop methods of<br>
producing recombinant proteins and peptides in Gram positive bacteria.<br>
SUMMARY<br>
The inventors have developed strains of gram positive Bacillus halodurans ALK36<br>
bacteria deposited under accession number NCIMB41348 transformed to produce and express<br>
on their surfaces fusion proteins composed of flagellin and, within the flagellin amino acid<br>
sequence, any of a variety of heterologous polypeptides. These recombinant bacteria produce<br>
high levels of stable and soluble recombinant protein on the surface of the recombinant bacterial<br>
cells. They have also developed genetic constructs containing flagellin-encoding nucleotide<br>
sequences and sites within the flagellin-encoding nucleotide sequences into which nucleotide<br>
sequences encoding heterologous polypeptides can be inserted. The invention features<br>
modified bacterial lines suitable for producing the flagellin-based fusion proteins (FBFP) of the<br>
invention, constructs useful for making the fusion proteins, FBFP, nucleic acids encoding FBFP,<br>
vectors containing the nucleic acids, cells containing the vectors, transformed bacterial lines<br>
expressing FBFP, and methods of making and using the FBFP.<br>
More specifically, the invention features a substantially pure culture of bacterial cells, a<br>
substantial number of which comprise a disrupted flagellin gene, the disruption preventing<br>
expression of functional flagellin. The invention also includes an isolated bacterial cell that<br>
contains a disrupted flagellin gene, the disruption preventing expression of functional flagellin.<br>
In both cases, the disruption can be by replacement of the endogenous gene in the substantial<br>
number of the bacterial cells with a DNA sequence encoding no polypeptide or a non-functional<br>
flagellin polypeptide. The bacterial cells can be gram positive bacterial cells, e.g., Bacillus cells<br>
such as B. halodurans cells. The cell may be of the strain BhFCO.1 ( hag). The non-functional<br>
flagellin polypeptide can lack amino acids 14 to 226 of SEQ  ID NO:2.<br><br>
2<br>
In addition to comprising a disrupted flagellin gene, at least one cell wall protease gene<br>
of a substantial number of the bacterial cells or the isolated bacterial cell can be disrupted and<br>
the disruption can be by replacement of the endogenous gene in the substantial number of<br>
bacterial cells with a DNA sequence encoding either no polypeptide or a non-functional cell wall<br>
protease polypeptide. The bacterial cell(s) can be those listed above. The at least one cell wall<br>
protease gene can be a wrpA gene and the disruption can include deletion of the entire coding<br>
sequence of cell wall protease gene. The cells can be BhFC04 ( hag, wprA) cells deposited<br>
under Accession Number	 at the NCIMB on 23 November 2005.<br>
Another aspect of the invention is a method of genetically engineering a bacterium of the<br>
genus Bacillus. The method includes disrupting the hag gene in the chromosome of the<br>
bacterium, the disruption preventing expression of functional flagellin by the gene. The Bacillus<br>
bacterium can be a Bacillus halodurans bacterium.<br>
The method may further comprise disrupting one or more genes encoding one or more<br>
cell wall proteases, the disruption preventing expression of the one or more functional cell wall<br>
proteases by the one or more genes. The one or more cell wall protease genes can include the<br>
wrpA gene.<br>
Also embraced by the invention is a fusion protein that contains: all or part of bacterial<br>
flagellin protein, the part of the flagellin protein including the N-terminal and C-terminal<br>
conserved regions of the flagellin protein; and a heterologous polypeptide within, or replacing,<br>
the variable region of the flagellin protein. The fusion protein has the ability, if made by a<br>
bacterial cell, to be expressed on the surface of the bacterial cell. The heterologous polypeptide<br>
can a polypeptide having the ability to bind to a metal ion. The metal ion may be nickel, copper,<br>
cadmium, platinum, palladium, titanium, silver, or gold, and the heterologous polypeptide may<br>
be a polyhistidine sequence. The polyhistidine sequence may contain six histidine residues.<br>
In addition, the heterologous polypeptide may be an enzyme or a functional fragment of<br>
an enzyme. The enzyme may be a lipase enzyme, e.g., G. thermoleovorans lipase A. The<br>
enzyme may be a hydrolytic enzyme, e.g., an amylase, a protease, an esterase, or a cellulase.<br>
Moreover, the heterologous polypeptide may be an immunogen.<br>
The fusion protein may further include one to fifteen linker residues N-terminal of the N-<br>
terminus of the heterologous polypeptide and/or one to fifteen linker residues C-terminal of the<br>
C-terminus of the heterologous polypeptide.   It may also include cleavable sites N-terminal of<br><br>
3<br>
the N-terminus of the heterologous polypeptide and C-terminal of the C-terminus of the<br>
heterologous polypeptide.<br>
The invention also provides: a nucleic acid encoding the above-described fusion protein;<br>
a vector including the nucleic acid sequence, e.g., a vector in which nucleic acid sequence is<br>
operably linked to a transcriptional regulatory element (TRE); and an isolated cell containing the<br>
vector. The cell may be a prokaryotic cell, e.g., a bacterial cell. The bacterial cell may be a<br>
gram positive bacterial cell such as a cell of the Bacillus genus, e.g., a cell of the B. halodurans<br>
species. The cell may be of the strain BhFC04 ( hag, wprA) deposited under Accession<br>
Number	at the NCIMB on 23 November 2005.<br>
In another aspect the invention embraces a method of making a fusion protein. The<br>
method may include culturing a cell containing a vector that includes a nucleic acid encoding<br>
the above-described fusion protein, the nucleic acid being operably linked to a TRE, and<br>
obtaining the fusion protein from the culture.<br>
Also featured by the invention is a DNA construct that contains: all or part of the coding<br>
sequence for a bacterial flagellin polypeptide, the part of the coding sequence including<br>
nucleotides encoding the N-terminal and C-terminal conserved regions of the flagellin protein;<br>
and, inserted into, or replacing, the sequence encoding the variable region of the flagellin<br>
polypeptide, a nucleotide sequence that includes at least one restriction enzyme site. The<br>
bacterial flagellin polypeptide may be a Bacillus flagellin polypeptide, e.g., B. halodurans<br>
flagellin (SEQ ID NO:1). In the construct, the nucleotide sequence may be inserted immediately<br>
after any nucleotide between nucleotide 162 and nucleotide 606 of SEQ ID NO:1; immediately<br>
after any nucleotide between nucleotide 441 and nucleotide 570 of SEQ ID NO:1; or<br>
immediately after any nucleotide between nucleotide 459 and nucleotide 540 of SEQ ID NO:1.<br>
Also provided by the invention is a method of removing one or more metal ions from a<br>
liquid. The method may involve: contacting a liquid containing one or more metal ions with a<br>
fusion protein of the invention, the heterologous polypeptide in the fusion protein being a<br>
polypeptide that binds to one or more of said metal ions. The liquid may be contacted with a<br>
bacterial cell expressing the fusion protein. Alternatively, the fusion protein may be a cell-free<br>
polypeptide.<br>
Another method of the invention is one for isolating one or more metal ions from a liquid<br>
containing the one or more metal ions. The method may include: contacting a liquid containing<br>
one or more metal ions with a fusion protein of the invention, the heterologous polypeptide in<br><br>
4<br>
the fusion protein being a polypeptide that binds to the one or more metal ions, the contacting<br>
resulting in binding of the one more metal ions to the fusion protein; and separating the one or<br>
more metal ions from the fusion protein.<br>
Another aspect of the invention is method of converting a substrate to a product, which<br>
method includes: contacting an enzyme substrate with a fusion protein of the invention, the<br>
heterologous polypeptide in the fusion protein being an enzyme or a functional fragment of the<br>
enzyme for the enzyme substrate thereby converting said substrate into said product.<br>
The invention also extends to use of a fusion protein of the invention in the manufacture<br>
of a preparation for generating an immune response in a mammalian subject.<br>
The invention extends further to a substance or composition for use in a method of<br>
generating an immune response in a mammalian subject, said substance or composition<br>
comprising a fusion protein of the invention, and said method comprising administering an<br>
effective amount of said substance or composition to said mammalian subject.<br>
Also embraced by the invention is a method of generating an immune response. The<br>
method may include administering the fusion protein of the invention to a mammalian subject,<br>
the heterologous polypeptide in the fusion protein being an immunogen. The mammalian<br>
subject may be, for example, a human.<br>
The invention also provides a kit that contains an expression vector that contains a DNA<br>
construct of the invention. The kit may further include: at least one restriction enzyme; a host<br>
cell, the host cell being a cell in which the vector is capable of replicating; and/or instructions for<br>
inserting a nucleic acid sequence encoding a heterologous polypeptide into the DNA construct.<br>
"Polypeptide" and "protein" are used interchangeably and mean any peptide-linked chain<br>
of amino acids, regardless of length or post-translational modification.<br>
As used herein, the term "isolated" as applied to a polypeptide refers to a polypeptide<br>
which either has no naturally-occurring counterpart or has been separated or purified from<br>
components which naturally accompany it, e.g., microorganism cellular components such as<br>
bacterial cell cellular components. Typically, the polypeptide is considered "isolated" when it is<br>
at least 70%, by dry weight, free from the proteins and other naturally-occurring organic<br>
molecules with which it is naturally associated. Preferably, a preparation of a polypeptide (or<br>
peptide fragment thereof) of the invention is at least 80%, more preferably at least 90%, and<br>
most preferably at least 99%, by dry weight, the polypeptide (or the peptide fragment thereof),<br><br>
5<br>
respectively, of the invention. Thus, for example, a preparation of polypeptide x is at least 80%,<br>
more preferably at least 90%, and most preferably at least 99%, by dry weight, polypeptide x.<br>
Since a polypeptide that is chemically synthesized is, by its nature, separated from the<br>
components that naturally accompany it, the synthetic polypeptide is "isolated." Moreover,<br>
since fusion proteins such as the FBFP of the invention do not exist in nature, they are always<br>
"isolated."<br>
An isolated polypeptide of the invention may be obtained, for example, by expression of<br>
a recombinant nucleic acid encoding the polypeptide or by chemical synthesis. A polypeptide<br>
that is produced in a cellular system different from the source from which it naturally originates<br>
is "isolated," because it will necessarily be free of components which naturally accompany it.<br>
The degree of isolation or purity can be measured by any appropriate method, e.g., column<br>
chromatography, polyacrylamide gel electrophoresis, or HPLC analysis.<br>
An "isolated nucleic acid", such as an isolated DNA, is either (1) a nucleic acid that<br>
contains sequence not identical to that of any naturally occurring sequence, or (2), in the<br>
context of a nucleic acid with a naturally-occurring sequence (e.g., a cDNA or genomic DNA), a<br>
nucleic acid free of at least one of the genes that flank the gene containing the nucleic acid of<br>
interest in the genome of the organism in which the gene containing the nucleic acid of interest<br>
naturally occurs. The term therefore includes a recombinant nucleic incorporated into a vector,<br>
into an autonomously replicating plasmid or virus, or into the genomic DNA of a prokaryote or<br>
eukaryote. The term also includes a separate molecule such as: a cDNA where the<br>
corresponding genomic DNA may include introns and therefore can have a different sequence;<br>
a genomic fragment that lacks at least one of the flanking genes; a fragment of cDNA or<br>
genomic DNA produced by polymerase chain reaction (PCR) and that lacks at least one of the<br>
flanking genes; a restriction fragment that lacks at least one of the flanking genes; a nucleic acid<br>
encoding a non-naturally occurring protein such as a fusion protein, mutein, or fragment of a<br>
given protein; and a nucleic acid which is a degenerate variant of a cDNA or a naturally<br>
occurring nucleic acid. In addition, it includes a recombinant nucleotide sequence that is part of<br>
a hybrid gene, i.e., a gene encoding a non-naturally occurring fusion protein. It will be apparent<br>
from the foregoing that isolated DNA does not mean a DNA present among hundreds to millions<br>
of other DNA molecules within, for example, cDNA or genomic DNA libraries or genomic DNA<br>
restriction digests in, for example, a restriction digest reaction mixture or an electrophoretic gel<br>
slice.<br><br>
6<br>
As used herein, "operably linked" means incorporated into a genetic construct so that an<br>
expression control sequence (transcriptional or translational regulatory element) effectively<br>
controls expression of a coding sequence of interest.<br>
The term "endogenous" as used herein with reference to nucleic acids or genes and a<br>
particular cell refers to any nucleic acid or gene that does occur in (and can be obtained from)<br>
that particular cell as found in nature.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same<br>
meaning as commonly understood by one of ordinary skill in the art to which this invention<br>
pertains. In case of conflict, the present document, including definitions, will control. Preferred<br>
methods and materials are described below, although methods and materials similar or<br>
equivalent to those described herein can be used in the practice or testing of the present<br>
invention. All publications, patent applications, patents and other references mentioned herein<br>
are incorporated by reference in their entirety. The materials, methods, and examples disclosed<br>
herein are illustrative only and not intended to be limiting.<br>
Other features and advantages of the invention, e.g., FBFP useful for bioremediation or<br>
biomining, and as immunogens, will be apparent from the following description, from the<br>
drawings and from the claims.<br>
DESCRIPTION OF DRAWINGS<br>
Fig. 1 is a photograph of a Coomassie blue-stained SDS-PAGE (sodium dodecyl sulfate<br>
polyacrylamide electrophoresis) gel showing protein profiles of extracellular (EX) and cell bound<br>
(CB) proteins produced by B. halodurans Alk 36 sampled during different stages of growth.<br>
Lane 1, CB (6 hours); lane 2 , CB (24 h); lane 3, CB (48 h); lane 4, (72 h); lane 5, molecular<br>
weight markers; lane 6, EX (24 h); lane 7, EX (72 h). The positions of various molecular weight<br>
markers (in kDa) are indicated on the right of the photograph. The arrow on the left of the<br>
photograph indicates the position of an over-expressed 34 kDa protein.<br>
Fig. 2 is a depiction of the first 22 N-terminal amino acids of the -34 kDa B. halodurans<br>
Alk36 protein (SEQ ID NO:3) aligned with the corresponding first 22 amino acids of the flagellin<br>
protein from the alkalophilic Bacillus sp. C-125 (SEQ ID NO:4). Positions of identity are<br>
indicated by + signs as well as letters corresponding to the relevant amino acids.<br><br>
7<br>
Fig. 3 is a depiction of a nucleotide sequence alignment of the putative hag coding<br>
region cloned from B. halodurans Alk36 (SEQ ID NO:1) with the corresponding coding region of<br>
the hag gene from Bacillus sp. C-125 (SEQ ID NO:1).<br>
Fig. 4 is photograph of DIG-labeled Southern blot of B. halodurans Alk36 chromosomal<br>
DNA digested with various restriction enzymes. The blot was probed with the 800 bp 6.<br>
halodurans Alk36 hag coding fragment labeled with DIG. Lane 1, 800 bp B. halodurans Alk36<br>
hag gene fragment digested with Xtoal/BamHI; Lanes 2-7, B. halodurans Alk36 chromosomal<br>
DNA digested with different restriction endonucleases: lane 2, Acd; lane 3, EcoRI; lane 4,<br>
/-//ndlll; lane 5, Psfl; lane 6, C/al; lane 7, Pvu\. Lane 8, molecular weight markers. The positions<br>
of size markers (in Kb) are indicated on the right of the photograph and the positions on the blot<br>
of fragments of 1.70 Kb and 1.47 Kb are indicated by arrows on the left of the photograph.<br>
Fig. 5 is a photograph of an ethidium bromide-stained agarose (1%) electrophoretic gel<br>
showing inverse-PCR products obtained from B. halodurans Alk36 chromosomal digests. Lane<br>
1, size markers; lane 2, Acd digest (10 I); lane 3, HindII digest (10 I); lane 4, positive control<br>
(plasmid containing flagellin DNA fragment, Scal digested). The positions of size markers (in<br>
Kb) are indicated on the left side of the photograph and the positions on the blot of fragments of<br>
1.157 Kb and 0.925 Kb are indicated by arrows on the right of the photograph.<br>
Fig. 6 is a depiction of the complete nucleotide sequence of the B. halodurans Alk36 hag<br>
gene including the regions flanking the coding sequence (SEQ ID NO:5). The boxed<br>
nucleotides represent the putative promoter regions and potential ribosome binding site. The<br>
white nucleotides are different to those of the hag gene from Bacillus halodurans C125. The<br>
shaded nucleotides show all the new sequence obtained from the iPCR. The coding region of<br>
the gene (SEQ ID NO:1) is shown in bold.<br>
Fig. 7 is a depiction of the nucleotide sequence of the B. halodurans Alk36 hag gene<br>
including up- and downstream regions (SEQ ID NO:5). The boxed nucleotides represent the<br>
putative promoter regions. The underlined nucleotides represent a potential ribosome binding<br>
(Shine-Dalgarno) site. Also shown is the complete amino acid sequence (SEQ ID NO:2) of B.<br>
halodurans Alk36 flagellin.<br>
Fig. 8A is a depiction of the pSE194 (4.313 kb) plasmid map.<br>
Fig. 8B is a depiction of the pSEC194 (5.496 kb) plasmid map.<br><br>
8<br>
Fig. 9A is a depiction of the nucleotide sequence of the B. halodurans Alk36 hag gene<br>
with its up- and down-stream regions (SEQ ID NO:6) indicating how the defective hag gene was<br>
constructed. The black sequence indicates the deleted region of the hag open reading frame<br>
and the grey shaded sequences indicate the two fragments, N- terminal and C-terminal (<br>
containing bits of coding region as well as up- and downstream sequence) which were obtained<br>
from PCR reactions and ligated together to create the defective hag gene used in the<br>
construction of pSECFIg-. Sequences corresponding to PCR primers are white.<br>
The sequence indicates the deleted region of the hag open reading frame and sequence<br>
the remainder of the open reading frame in the construction of pSECFIg-. Primers used are<br>
underlined.<br>
Fig. 9B is a depiction of the pSEC194F1g- plasm id map.<br>
Fig. 10 is a photograph of an ethidium bromide-stained agarose (0.8%) electrophoretic<br>
gel showing the PCR-determined results of integration events of pSEC194Flg- in the 6.<br>
halodurans Alk36 chromosome. Different primer sets (A, B, C and D) were used to test the<br>
different integration events. Lane 1, sco, mutant 49; lane 2, sco, mutant 50; lane 3, dco, non-<br>
motile mutant - BhFC01 (derived from 49); and lane 4, B. halodurans Alk36 chromosomal DNA.<br>
The lane between primer sets B and C contains size markers. The positions of size (in Kb)<br>
markers are indicated on the right side of the photograph.<br>
Fig. 11 is a depiction of a comparison of the secondary structure of the proteins encoded<br>
by NC1, NC2, NC3, NC5, and NC6 as predicted by PSI-Pred software. This program was used<br>
to generate a 3-state prediction for each protein. The secondary structures a-helix (H),<br>
extended p-strand (E) and coil (C) are indicated. Amino acid insertions are indicated in bold<br>
and grey blocks.<br>
Fig. 12A is a depiction of the amino acid sequence (SEQ ID NO:7) of the NC2 peptide<br>
and the nucleotide sequence (SEQ ID NO:8) encoding it.<br>
Fig. 12B is a depiction of the amino acid sequence (SEQ ID NO:9) of the NC3 peptide<br>
and the nucleotide sequence (SEQ ID NO:10) encoding it<br>
Fig. 12C is a depiction of the amino acid sequence (SEQ ID NO:11) of the NC5 peptide<br>
and the nucleotide sequence (SEQ ID NO:12) encoding it.<br><br>
9<br>
Fig. 12D is a depiction of the amino acid sequence (SEQ ID NO:13) of the NC6 peptide<br>
and the nucleotide sequence (SEQ ID NO:14) encoding it.<br>
Fig. 13A is photograph of a Coomassie blue-stained SDS-PAGE gel containing cell-<br>
surface protein fractions. Lanes 1 - 7, S. halodurans strain BhFCOl transformed with various<br>
genetic constructs. Lane 1, NC1; lane 2, NC2; lane 3, NC3; lane 4, NC3sco; lane 5, NC5; lane<br>
6, FliC; lane 7, BhFCOl; lane 8, B. halodurans Alk3 WT (wild-type); lane 9, molecular weight<br>
markers. The positions of various molecular weight (in kDa) markers are indicated on the right<br>
of the photograph. The arrow on the right of the photograph indicates the position of an over-<br>
expressed 34 kDa protein.<br>
Fig. 13B is photograph of a Coomassie blue-stained SDS-PAGE gel containing CS proteins<br>
extracted from: lane 1, B. halodurans Alk36; lane 2, strain BhFCOl pSEC194NC3; lane 3, strain<br>
BhFCOl pSEC194NC6; and lane 4, molecular weight markers. The positions of various molecular<br>
weight (in kDa) markers are indicated on the right of the photograph.<br>
Fig. 13C is a photograph of a Western blot of an SDS-PAGE gel of protein fractions as<br>
described to Fig. 13A. The Western blot was stained with a polyclonal flagellin-specific antibody.<br>
The positions of various molecular weight (in kDa) markers are indicated on the left of the<br>
photograph. The arrow on the left of the photograph indicates the position of an over-expressed<br>
34 kDa protein.<br>
Fig. 14 is photographs of motility plates of various B. halodurans Alk36 strains. A: colony (1),<br>
B. halodurans A\k36WT; colony (2) BhFCOl ( hag); colony (3), BhFCOl ( hag) + pSEC194FliC. B:<br>
colony (1), 6. halodurans Alk36 WT; colonies 2-6 BhFCOl transformed with: (2) pSECNCI, (3)<br>
pSECNC2, (4) pSECNC3, (5) pSECNC5, (6) pSECNC6.<br>
Fig. 15A is a schematic depiction of the complete wprA- coding region. The white region in A<br>
shows the area of the coding region which has been deleted in construction of pSECwprA- depicted<br>
in B. This construct was then used for the knockout of the wprk gene on the chromosome of B.<br>
halodurans BhFCOl<br>
Fig. 15B is a depiction of the pSEC194nprA- plasmid map.<br>
Fig. 16 is a photograph of Coomassie-blue-stained SDS-PAGE gels analyzing proteases in<br>
the extracellular (EX) and cell surface (CS) fractions of the strains BhFCOl and BhFCOA Aliquots of<br>
the EX and CS fractions were separated by SDS-PAGE gel electrophoresis and stained with<br><br>
10<br>
Coomassie blue to vizualize the protein bands (lanes 2-5) or for protease activity after renaturation<br>
treatment (lanes 7-10). Lanes 1 and 6, molecular weight markers; lanes 2 and 7, BhFC04<br>
extracellular fraction; lanes 3 and 8, BhFC01 extracellular fraction; lanes 4 and 9, BhFC04 cell<br>
surface fraction; lanes 5 and 10, BhFC01 cell surface fraction. The positions of various molecular<br>
weight (in kDa) markers are indicated on the left of the photograph. The arrow indicates the deleted<br>
WprA protein band in the cell wall fraction of BhFC04 (lane 4). Successful elimination of protease<br>
activity in the same fraction is shown in lane 9.<br>
Fig. 17 is a depiction of the amino acid sequence of the poly-His peptide and the nucleotide<br>
sequence encoding it. The nucleotide sequence includes the part of the MCS remaining in the<br>
pSEC194NHisC6 construct to give the full size of the incorporated peptide. The total size of the<br>
inserted peptide was 13 amino acids.<br>
Fig. 18 is a photograph of a Coomassie blue-stained SDS-PAGE gel of CS fractions of the<br>
following strains: Lane 1, B. halodurans Alk36 WT; lane 2, BhFC04 containing pSEC194NC6; lane 3,<br>
BhFC04 containing pSEC194NHisC6; and lane 4, molecular weight markers. The positions of<br>
various molecular weight markers (in kDa) are indicated on the right of the photograph.<br>
Fig. 19 is a photograph of a Coomassie blue-stained SDS-PAGE gel showing metal binding<br>
of the displayed poly-His tag on the CS fraction of strain BhFC04 containing pSECi94NhisC6. Lane<br>
1, molecular weight markers; lane 2, pSECi94NHisC6 (eluted); lane 3, pSEC194NC6 (eluted); lane<br>
4, pSEC194NC6 (beads); lane 5, pSEC194NHisC6 (beads); lane 6, pSEC194NHisC6 (flow through);<br>
lane 7, pSEC194NC6 (flow through). The position of a molecular weight marker (in kDa) is indicated<br>
on the left of the photograph. The arrows indicate the positions of the flagellin fusion proteins. Note<br>
the successful binding of the pSEC194NHisC6 fusion protein (lane 2) due to the presence of the<br>
poly-His tag.<br>
Fig. 20 is a depiction of the amino acid sequence (SEQ ID NO:15) of the HIV gp120 peptide<br>
and the nucleotide sequence (SEQ ID NO:16) encoding it. The nucleotide sequence includes the<br>
part of the MCS remaining in the pSEC194NHivC6 construct.<br>
Fig. 21 is photograph of a Coomassie blue-stained 10% SDS-PAGE gel of CS fractions of<br>
various strains and showing the flagellin fusion protein carrying the HIV gp120 antigenic peptide.<br>
Lane 1, BhFC04 containing pSEC194HivNC6; lane 2, WT (wild-type) B. halodurans Alk36; and lane<br>
3, molecular weight markers. The positions of various molecular weight markers (in kDa) are<br>
indicated on the right of the photograph.<br><br>
11<br>
Fig. 22 is a photograph of a Western blot stained with a flagellin-specific polyclonal antibody.<br>
Lane 1, BhFC04 + pSECNHivC6; lane 2, BhFC04 +pSECNHisC6; lane 3, BhFC04 + pSECNC6; lane<br>
4, BhFC04; lane 5, B haloduranA\k36; lane 6, molecular weight marker. The position of a molecular<br>
weight (in kDa) marker is indicated on the right of the photograph. The arrow on the right of the<br>
photograph indicates the position of over-expressed flagellin (34 kDa) protein.<br>
Fig. 23 is a depiction of the nucleotide sequence (SEQ ID NO:17) of the region of the //pA<br>
gene of Geobacillus thermoleovorans encoding mature lipase and the amino acid sequence (SEQ ID<br>
NO: 18) of the mature lipase.<br>
Fig. 24 is a photograph of a zymogram of lipase activity in the various protein fractions<br>
isolated from BhFC04 carrying the pSEC194NLipC3 construct. The EX (lane 1), CS (lane 2 &amp; 3),<br>
CW (lane 4) and IC (lane 5) fractions are shown. Note that the EX fraction shows no lipase activity.<br>
Lane 6, molecular weight markers. The position of various molecular weight (in kDa) markers are<br>
indicated on the right of the photograph. The arrow on the left of the photograph indicates the<br>
positions of the flagellin (FliC):lipase (Lip) fusion protein having a molecular weight of approximately<br>
76 kDa.<br>
Fig. 25 is a bar graph showing extracellular and whole cell lipase activity (in terms of Units (U)<br>
per mg of protein) of BhFC04 cells containing pSECi94NC6 (lipase negative control) or<br>
pSECi94NLipC after 8, 24 and 48 hours of growth.<br>
Fig. 26 is a depiction of the amino acid sequence of the B. halodurans wprk cell wall protease<br>
(SEQ ID NO:19).<br>
DETAILED DESCRIPTION<br>
Various aspects of the invention are described below.<br>
Flagellin-based Fusion Proteins (FBFP)<br>
The invention features FBFP useful for a wide range or purposes. The FBFP contain all<br>
or part of a bacterial flagellin protein and a heterologous polypeptide sequence replacing, or<br>
within, the variable region of the flagellin protein. The FBFP has the ability to be transported to,<br>
and expressed on, the surface of a bacterial cell that makes it. As used herein, a protein that is<br>
"expressed on the surface of a bacterial cell" is a protein that is attached directly or indirectly to<br>
the bacterial cell wall (e.g., a protein that is a component of bacterial flagellum) or is lodged in<br>
the cell wall. In either case, all or part of the protein is exposed to the exterior of the bacterial<br>
cell and can interact with appropriate substances outside the bacterial cell, e.g., metal ions,<br>
enzyme substrates, or receptors on cells of the immune system (e.g., B lymphocytes, T<br><br>
12<br>
lymphocytes (CD4+ and/or CD8+), natural killer (NK) cells, or antigen presenting cells (APC)<br>
such as macrophages, monocytes, interdigitating dendritic cells (referred to herein as dendritic<br>
cells), and B lymphocytes.<br>
The flagellin portion of the FBFP can be any bacterial flagellin but is preferably a Gram<br>
positive bacterial flagellin. Gram positive bacterial genera of interest include, without limitation,<br>
Clostridium, Staphylococcus, Lactococcus, Lactobacillus, Streptococcus and Streptomyces. Of<br>
particular interest are bacteria of the genus Bacillus. Useful species include Bacillus subtilis,<br>
Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus brevis, Bacillus circulans, Bacillus<br>
clausii, Bacillus coagulans, Bacillus firmus, Bacillus lautus, Bacillus lentus, Bacillus<br>
licheniformis, Bacillus megaterium, Bacillus pumilus, Bacillus stearothermophilus or Bacillus<br>
thuringiensis; or a Streptomyces strain, e.g. Streptomyces lividans or a Lactococcus strain e.g.<br>
Lactococcus lactis<br>
A particularly useful species is B. halodurans and the flagellin can be that with an amino<br>
acid sequence (SEQ ID NO:2) shown in Fig. 7 and a coding sequence (SEQ ID NO:1) shown in<br>
Fig. 3. The flagellin portion of the FBFP can include the whole flagellin molecule or part of the<br>
flagellin molecule. As described in Example 2, flagellin molecules from different bacteria have<br>
regions of relatively high homology which include N-terminal and C-terminal conserved regions<br>
and an internal variable region. By aligning the amino acid sequence of any bacterial flagellin of<br>
interest with one or more of those referred to in Example 2 or to SEQ ID NO:2, one skilled in the<br>
art can determine where in the flagellin of interest these three regions are located.<br>
The FBFP can, for example, lack all or part of the variable region of the protein. In 6.<br>
halodurans flagellin (SEQ ID NO:2), a polypeptide of 272 amino acids, the variable region is<br>
located from amino acid 54 to amino acid 202, corresponding to nucleotides 162 to 606 of its<br>
coding sequence (SEQ ID NO:1) and the conserved N and C terminal regions are on either side<br>
of the variable region (i.e., amino acids 1 to 53 and amino acids 203 to 272 of SEQ ID NO:2,<br>
respectively, corresponding to nucleotides 1 to 161 and 607 to 816 of SEQ ID NO:1. In making<br>
these modifications to the flagellin sequence, all that is required is that the FBFP have the<br>
ability, if made in a bacterial cell, to be transported to and expressed on the surface of the<br>
bacterial cell.<br>
The heterologous polypeptide useful in the FBFP of the invention can be any<br>
polypeptide other than: (a) the particular flagellin from which the flagellin-derived part of the<br>
FBFP was obtained; or (b) a part of the particular flagellin. It can replace all or a part of the<br>
variable region of flagellin protein of interest or it can be inserted into an intact variable region.<br><br>
13<br>
When replacing part of the variable region, up to 148 (e.g., up to:145, 140, 135,130, 125,120,<br>
115, 110, 105, 100, 95, 90, 85; 80; 75; 65; 60; 55; 50; 45; 40; 35; 30; 25; 20; 15; 12; 10; 9; 8; 7;<br>
6; 5; 4; 3; or 2) of the variable region amino acids can be deleted. The heterologous<br>
polypeptide can be any length and is preferably from 5 - 450 (e.g., 5-450, 5-200, 5-150, 5-100,<br>
5-50, 5-40, 5-30, 5-20, 5-15, 5-10, 10-450, 10-200, 10-150, 10-100, 10-50, 10-40, 10-30, 10-<br>
20,10-15, 20-450, 20-200, 20-150, 20-100, 20-50, 20-40, or 20-30) amino acids in length.<br>
In addition, at either the N-terminal, C-terminal, or both ends of the heterologous<br>
polypeptide there can be "linker" amino acids separating the heterologous polypeptide from the<br>
flagellin-derived sequence. These amino acids can be inserted either as cleavable sites (see<br>
below), as moieties that allow the heterologous polypeptide to assume an appropriate three-<br>
dimensional structure, and/or they reflect the inclusion of appropriate restriction sites in a<br>
genetic construct used to make the FBFP recombinantly (see below). Such linkers can be 1 -<br>
20 (e.g., 1-15,1-12,1-10,1-9,1-8, 1-7,1-6,1-5,1-4,1-3, or 1-2) amino acids long.<br>
Cleavable sites on either end of the heterologous polypeptide in the FBFP would allow<br>
the heterologous polypeptide, optionally including residues from the linkers, to be excised from<br>
the FBFP and used in its excised form. Where the FBFP is made chemically, any number of<br>
cleavable cross-linkers (e.g., bifunctional cleavable cross-linkers) are known in the art and could<br>
be included. Alternatively, where the FBFP is made recombinantly, other cleavable sites can be<br>
"engineered" into the FBFP. An example of a chemical cleavage method makes use of a single<br>
methionine residue insertion at either end of the heterologous polypeptide (either immediately N<br>
and C terminal or separated by one or more linker residues from the termini). The heterologous<br>
polypeptide can then be easily cleaved using cyanogen bromide. A second example is the use<br>
of a single cysteine residue at either end of the heterologous polypeptide (located as described<br>
for the methionine residues) and subsequent cleavage using NTCB (2-nitro-5-<br>
thiocyanobenzoate). Enzymatic cleavage makes use of enzymatically active endoproteases<br>
able to recognise a specific amino acid sequence. An example is the Endo Arg C cysteine<br>
protease which recognises the amino acid sequence Arg-X where X can be any amino acid.<br>
This method would thus require the addition of two amino acids on either side of the<br>
heterologous polypeptide. Another example is the Endo Lys C serine protease which<br>
recognizes the sequence Lys-X where X can be any amino acid. Again two amino acids would<br>
need to be added at either end of the heterologous polypeptide. Different combinations of<br>
different enzymatic and chemical cleavage methodologies can be used to ensure all undesired<br>
amino acids are removed.<br><br>
14<br>
The heterologous polypeptide, or where one or two linkers are used the heterologous<br>
polypeptide and the linker(s), can be inserted immediately after any amino acid between amino<br>
acids 54-202 of SEQ ID NO:2, immediately after any amino acid between amino acids 147-190<br>
of SEQ ID NO:2, immediately after any amino acid between amino acids 150-184 of SEQ ID<br>
NO:2, or immediately after any amino acid between amino acids 153-180 of SEQ ID NO:2. For<br>
example, the heterologous polypeptide can be inserted immediately after amino acids 153 or<br>
180 of SEQ ID NO:2. Where a flagellin of interest has an amino acid sequence different from<br>
SEQ ID NO:2 (the amino acid sequence of B. halodurans), one of skill in the art, by aligning the<br>
relevant amino acid sequence with SEQ ID NO:2, would be able to predict amino acids in that<br>
flagellin corresponding to those listed above for SEQ ID NO: 2 and thus appropriate positions<br>
into which to insert a heterologous polypeptide, or where one or two linkers are used the<br>
heterologous polypeptide and the linker(s), of interest. Moreover, if less than the whole flagellin<br>
coding sequence is included in the construct, positions corresponding to those recited above for<br>
SEQ ID NO:2 can readily be ascertained by one of ordinary skill in the art.<br>
The invention also features FBFP with conservative substitutions in either the flagellin<br>
portion of the FBFP or in the heterologous polypeptide portion. Conservative substitutions<br>
typically include substitutions within the following groups: glycine and alanine; valine, isoleucine,<br>
and leucine; aspartic acid and glutamic acid; asparagine, glutamine, serine and threonine;<br>
lysine, histidine and arginine; and phenylalanine and tyrosine. Each portion contains no more<br>
than 30 (e.g., no more than: 25; 20; 15; 10; 9; 8; 7; 6; 5; 4; 3; 2; or 1) conservative<br>
substitution(s).<br>
As indicated above, in making any of the above modifications of sequence, all that is<br>
required that the FBFP have the ability, if made in a bacterial cell, to be transported to and<br>
expressed at the surface of the bacterial cell.<br>
Heterologous polypeptides of interest include those useful in, without limitation,<br>
bioremediation, biomining, enzyme-mediated substrate conversions, as immunogens for<br>
activating immune responses in any of a variety of mammals, for use in preparing immunogen<br>
preparations, as a substance or composition for use in methods of treatment or prevention and<br>
therapeutic applications.<br>
Bioremediation is a process whereby toxic substances (e.g., heavy metal ions or atoms)<br>
are removed from liquids (e.g., a potable water) by binding of the toxic substances to a solid<br>
substrate over which the liquid is passed so that the liquid is safe for animal (e.g., human)<br>
consumption.   Biomining involves binding to a solid substrate of useful and/or precious metal<br><br>
15<br>
ions or atoms in a liquid and the subsequent recovery of the metal atoms or ions from the solid<br>
substrate. For either purpose, certain metal-binding polypeptides can be used. These methods<br>
are described in greater detail below. Relevant polypeptides, which can be incorporated as<br>
heterologous polypeptides into the FBFP of the invention, include those known to bind to a<br>
variety of heavy, useful, precious, and/or toxic metal ions. Relevant metals include, without<br>
limitation, nickel (Ni), cadmium (Cd ), gold (Au), platinum (Pt), palladium (Pd), titanium (Ti),<br>
copper (Cu), and silver (Ag). Thus, for example, poly-histidine (poly-His; e.g., hexahistidine)<br>
polypeptides have been shown to bind Cd, Ni, and Cu. In addition, Ag has been shown to bind<br>
to a polypeptide with the amino acid sequence: NPSSLFTYLPSD (SEQ ID NO:20). Pd has<br>
been shown to bind to polypeptides with the amino sequences: CSVTQNKYC (SEQ ID NO:21);<br>
CSPHPGPYC (SEQ ID NO:22); and CHAPTPMLC (SEQ ID NO:23). Pt has been shown to<br>
bind polypeptides with the amino acid sequences: CDRTSTWRC (SEQ ID NO:24);<br>
CQSVTSTKC (SEQ ID NO:25); and CSSSHLNKC (SEQ ID NO:26). Ti has been shown to bind<br>
to a polypeptide with the amino acid sequence: RKLPDAPGMHTW (SEQ ID NO:27) (Kriplani<br>
and Kay (2005), Curr Opinion Biotechnol, 16:470-475). Thus, all these polypeptides can be<br>
used as heterologous polypeptides in the FBFP of the invention.<br>
A wide range of enzymes (or functional fragments of such enzymes) useful for high-<br>
volume conversion of certain substrates (in, for example, fermenters) to desired products can<br>
also be inserted as heterologous polypeptides into the FBFP of the invention. Relevant<br>
enzymes include, without limitation, hydrolytic enzymes such as amylases, proteases,<br>
esterases, and cellulases. Of particular interest are lipase enzymes such as described in<br>
Example 8. A "functional fragment" of an enzyme is a fragment of the enzyme that is shorter<br>
than the full-length mature enzyme and has at least 20% (e.g., at least: 30%; 40%; 50%; 60%;<br>
70%; 80%; 90%; 95%; 98%; 99%; 100%; or more) of the ability of the full-length, wild-type<br>
enzyme to convert its substrate to its relevant product.<br>
Immunogenic polypeptides useful for inducing immune responses in mammals (see<br>
below) can also be used as heterologous polypeptides. Such polypeptides can be full-length<br>
mature polypeptides or peptide fragments of such polypeptides. Immunogenic polypeptides can<br>
be derived from, for example, any of a wide variety of infectious (e.g., bacterial, fungal (including<br>
yeast), viral, or parasite (such as protozoan parasite)) microorganisms or cancer cells.<br>
Examples of relevant microorganisms include, without limitation, Mycobacteria tuberculosis,<br>
Salmonella enteriditis, Listeria monocytogenes, M. leprae, Staphylococcus aureus, Escherichia<br>
coll, Streptococcus pneumoniae, Borrelia burgdorferi, Actinobacillus pleuropneumoniae,<br>
Helicobacter pylori, Neisseria meningitidis, Yersinia enterocolitica, Bordetella pertussis,<br>
Porphyromonas gingivalis,  mycoplasma, Histoplasma capsulatum, Cryptococcus neoformans,<br><br>
16<br>
Chlamydia trachomatis, Candida albicans, Plasmodium falciparum, Entamoeba histolytica,<br>
Toxoplasma brucei, Toxoplasma gondii, Leishmania major, human immunodeficiency virus 1<br>
and 2, influenza virus, measles virus, rabies virus, hepatitis virus A, B, and C, rotaviruses,<br>
papilloma virus, respiratory syncytial virus, feline immunodeficiency virus, feline leukemia virus,<br>
and simian immunodeficiency virus. Examples of relevant microbial proteins include, without<br>
limitation, the B subunit of heat labile enterotoxin of E. coli [Konieczny et al. (2000) FEMS<br>
Immunol. Med. Microbiol. 27(4):321-332], heat-shock proteins, e.g., the Y. enterocolitica heat<br>
shock protein 60 [Konieczny et al. (2000) supra; Mertz et al. (2000) J. Immunol. 164(3):1529-<br>
1537] and M. tuberculosis heat-shock proteins hsp60 and hsp70, the Chlamydia trachomatis<br>
outer membrane protein [Ortiz et al. (2000) Infect. Immun. 68(3):1719-1723], the B. burgdorferi<br>
outer surface protein [Chen et al. (1999) Arthritis Rheum. 42(9):1813-1823], the L major GP63<br>
[White et al. (1999) Vaccine 17(17):2150-2161 (and published erratum in Vaccine 17(20-<br>
21):2755)], the N. meningitidis meningococcal serotype 15 PorB protein [Delvig et al. (1997)<br>
Clin. Immunol. Immunopathol. 85(2);134-142], the P. gingivalis 381 fimbrial protein [Ogawa,<br>
(1994) J. Med. Microbiol. 41(5):349-358], the E. coli outer membrane protein F [Williams et al.<br>
(2000) Infect. Immun. 68(5):2535-2545], influenza virus hemagglutinins and neuramindases,<br>
retroviral (e.g., HIV) surface glycoproteins (e.g., HIV gp160/120/41), or retroviral tat or gag<br>
proteins.<br>
Furthermore, tumor-associated antigens (TAA) or peptide fragments of TAA can be used<br>
as heterologous polypeptides. As used herein, a "TAA" is a molecule (e.g., a protein molecule)<br>
that is expressed by a tumor cell and either (a) differs qualitatively from its counterpart<br>
expressed in normal cells, or (b) is expressed at a higher level in tumor cells than in normal<br>
cells. Thus, a tumor antigen can differ (e.g., by one or more amino acid residues where the<br>
molecule is a protein) from, or it can be identical to, its counterpart expressed in normal cells. It<br>
is preferably not expressed by normal cells. Alternatively, it is expressed at a level at least two-<br>
fold higher (e.g., a two-fold, three-fold, five-fold, ten-fold, 20-fold, 40-fold, 100-fold, 500-fold,<br>
1,000-fold, 5,000-fold, or 15,000-fold higher) in a tumor cell than in the tumor cell's normal<br>
counterpart. Examples of relevant cancers include, without limitation, hematological cancers<br>
such as leukemias and lymphomas, neurological tumors such as astrocytomas or<br>
glioblastomas, melanoma, breast cancer, lung cancer, head and neck cancer, gastrointestinal<br>
tumors such as gastric or colon cancer, liver cancer, pancreatic cancer, genitourinary tumors<br>
such ovarian cancer, vaginal cancer, bladder cancer, testicular cancer, prostate cancer or penile<br>
cancer, bone tumors, and vascular tumors. Relevant TAA include, without limitation,<br>
carcinoembryonic antigen (CEA), prostate specific antigen (PSA), MAGE (melanoma antigen) 1-<br>
4, 6 and 12, MUC (mucin) (e.g., MUC-1, MUC-2, etc.), tyrosinase, MART (melanoma antigen),<br>
Pmel  17(gp100),  GnT-V  intron  V sequence  (N-acetylglucoaminyltransferase V intron  V<br><br>
17<br>
sequence), Prostate Ca psm, PRAME (melanoma antigen), (3-catenin, MUM-1-B (melanoma<br>
ubiquitous mutated gene product), GAGE (melanoma antigen) 1, BAGE (melanoma antigen) 2-<br>
10, C-ERB2 (Her2/neu), EBNA (Epstein-Barr Virus nuclear antigen) 1-6, gp75, human papilloma<br>
virus (HPV) E6 and E7, p53, lung resistance protein (LRP), Bc1-2, and Ki-67.<br>
Therapeutic polypeptides that can be incorporated as heterologous polypeptides into the<br>
FBFP of the invention include, without limitation, human growth hormone (HGH), anti-retroviral<br>
agents (e.g., FUZEON, NEUPOGEN), anti-microbial peptides (e.g., indolicidin, buforin, nisin and<br>
trichogin), polypeptides useful for treating cardiovascular disease (e.g., nesiritide), and<br>
polypeptides useful for treating diabetes (e.g., liraglutide and insulinotropin). Generally, but not<br>
necessarily, these heterologous polypeptides would be excised from the FBFP, and in certain<br>
circumstances formulated into a preparation, prior to administration to appropriate mammalian<br>
subjects (e.g., a human subjects or patients).<br>
The FBFP of the invention can be synthesized by standard chemical means known to<br>
those in the art. In addition, the FBFP can be produced by standard in vitro recombinant DNA<br>
techniques and in vivo transgenesis, using nucleotide sequences encoding the appropriate<br>
FBFP (e.g., those techniques described in the Examples below). Methods well-known to those<br>
skilled in the art can be used to construct expression vectors containing relevant coding<br>
sequences and appropriate transcriptional/translational control signals. See, for example, the<br>
techniques described in Sambrook et al., Molecular Cloning: A Laboratory Manual (2nd Ed.)<br>
[Cold Spring Harbor Laboratory, N.Y., 1989], and Ausubel et al., Current Protocols in Molecular<br>
Biology [Green Publishing Associates and Wiley Interscience, N.Y., 1989].<br>
FBFP of the invention also include those described above, but modified by the addition,<br>
at the amino- and/or carboxyl-terminal ends, of a blocking agent to facilitate survival of the<br>
relevant FBFP. This can be useful in those situations in which the peptide termini tend to be<br>
degraded by proteases (e.g. in a mammalian subject). Such blocking agents can include,<br>
without limitation, additional related or unrelated peptide sequences that can be attached to the<br>
amino and/or carboxyl terminal residues of the peptide to be administered. This can be done<br>
either chemically during the synthesis of the peptide or by recombinant DNA technology by<br>
methods familiar to artisans of average skill.<br>
Alternatively, blocking agents such as pyroglutamic acid or other molecules known in the<br>
art can be attached to the amino and/or carboxyl terminal residues, or the amino group at the<br>
amino terminus or carboxyl group at the carboxyl terminus can be replaced with a different<br><br>
18<br>
moiety. Likewise, the peptides can be covalently or non-covalently coupled to pharmaceutically<br>
acceptable "carrier" proteins prior to administration.<br>
Also of interest are peptidomimetic compounds that are designed based upon the amino<br>
acid sequences of the FBFP. Peptidomimetic compounds are synthetic compounds having a<br>
three-dimensional conformation (i.e., a "peptide motif") that is substantially the same as the<br>
three-dimensional conformation of a selected polypeptide. The polypeptide motif provides the<br>
peptidomimetic compound with the ability to function as required. Peptidomimetic compounds<br>
can have additional characteristics that enhance their therapeutic utility, such as increased cell<br>
permeability and prolonged biological half-life.<br>
The peptidomimetics typically have a backbone that is partially or completely non-<br>
peptide, but with side groups that are identical to the side groups of the amino acid residues that<br>
occur in the peptide on which the peptidomimetic is based. Several types of chemical bonds,<br>
e.g., ester, thioester, thioamide, retroamide, reduced carbonyl, dimethylene and ketomethylene<br>
bonds, are known in the art to be generally useful substitutes for peptide bonds in the<br>
construction of protease-resistant peptidomimetics.<br>
Nucleic Acid Molecules<br>
The invention also features nucleic molecules encoding the above-described FBFP of<br>
the invention. In these nucleic acid molecules, the sequences encoding the flagellin-derived<br>
portions of the FBFP, the sequences encoding the heterologous polypeptide portion, and those<br>
encoding optional linkers are assembled such that all the portions are in the appropriate reading<br>
frame such that, in the FBFP encoded by the nucleic acid molecule, each portion has an amino<br>
acid sequence corresponding to the section of the protein from which it was derived or such<br>
sequence but with any modifications deliberately included. One of skill in the art will know how<br>
to construct appropriate "in frame" nucleic acid molecules.<br>
The nucleic acid molecules of the invention also include genetic (DNA) constructs useful<br>
for making FBFP with any heterologous polypeptide of interest. The genetic construct contains<br>
all or part of the coding sequences of any of the flagellin described above and, inserted into the<br>
region of the coding sequence encoding the flagellin variable region (also referred to herein as<br>
the variable region of the flagellin coding sequence), a linker nucleic acid sequence.<br>
Particularly useful as linker nucleic acid linker sequences are multiple cloning sites (MCS)<br>
containing at least one (e.g., 2, 3, 4, 5, 6, 7,8, 9,10,11,12,15,17, 20, or even 30) consecutive<br>
and/or overlapping restriction enzyme recognition/cutting sites. Restriction enzymes and their<br>
recognition/cutting sequences are known in the art and include those described in the Examples<br><br>
19<br>
below. Such MCS are used for inserting heterologous polypeptide-encoding nucleotide<br>
sequences. It is understood that an MCS can be inserted such that a nucleotide sequence<br>
"downstream" of it is read in the appropriate frame or not. All that is required is that, after the<br>
insertion of the heterologous polypeptide-encoding sequence, the entire FBFP coding sequence<br>
is in the correct reading frame.<br>
Where the flagellin (or part of the flagellin) is B. halodurans flagellin, the linker nucleic<br>
acid sequences (including MCS) can be inserted immediately after any nucleotide between<br>
nucleotide 162 and nucleotide 606 of SEQ ID NO:1, e.g., immediately after any nucleotide<br>
between nucleotide 441 and nucleotide 570 of SEQ ID NO:1, immediately after any nucleotide<br>
between nucleotide 450 and nucleotide 552 of SEQ ID NO:1, or immediately after any<br>
nucleotide between nucleotide 459 and nucleotide 540 of SEQ ID NO:1. The linker nucleic acid<br>
sequences (including MCS) can be inserted, for example, immediately after nucleotide 459 or<br>
nucleotide 540 of SEQ ID NO:1. Where a flagellin of interest has an amino acid sequence<br>
different from SEQ ID NO:2 (the amino acid sequence of B. holodurans), one of skill in the art,<br>
by aligning the relevant amino acid sequence with SEQ ID NO:2, would be able to predict<br>
nucleotides in the coding sequence of that flagellin corresponding to those listed above for SEQ<br>
ID NO: 1 and thus appropriate positions into which to insert linker nucleic acid sequences of<br>
interest. Moreover, if less than the whole flagellin coding sequence is included in the construct,<br>
positions corresponding to those recited above for SEQ ID NO:1 can readily be ascertained by<br>
one of ordinary skill in the art.<br>
The invention also provides kits containing one or more (e.g., 2, 3, 4, 5, 6, 7, 8, 9,10, or<br>
more) of the genetic (DNA) constructs of the invention. The constructs are generally supplied<br>
as components incorporated into expression vectors. The kits can further contain one or more<br>
restriction enzymes useful for digesting the constructs in order to insert a heterologous<br>
polypeptide-encoding nucleotide sequence into the construct. The kit can also contain ancillary<br>
reagents such as buffers, salt solutions, and/or any other reagents for relevant restriction<br>
enzyme digest reactions. Moreover, the kit can contain a ligase enzyme (and the above-listed<br>
ancillary agents) for ligating a heterologous polypeptide-encoding nucleotide sequence into the<br>
construct. In addition, the kit can contain host (any of those recited herein) useful for<br>
expressing the genetic constructs. The kit can also include written instructions (on, for example,<br>
packaging material or a package insert) on how to digest the constructs with one or two<br>
appropriate restriction enzymes, how to ligate a heterologous polypeptide-encoding nucleotide<br>
sequence into the construct, and/or how to transform host cells with the genetic constructs after<br>
such ligation.<br><br>
20<br>
The FBFP-encoding nucleic acid molecules of the invention can be cDNA, genomic<br>
DNA, synthetic DNA, or RNA, and can be double-stranded or single-stranded (i.e., either a<br>
sense or an antisense strand). Segments of these molecules are also considered within the<br>
scope of the invention, and can be produced by, for example, the polymerase chain reaction<br>
(PCR) or generated by treatment with one or more restriction endonucleases. A ribonucleic<br>
acid (RNA) molecule can be produced by in vitro transcription. Preferably, the nucleic acid<br>
molecules encode polypeptides that, regardless of length, are soluble under normal<br>
physiological conditions.<br>
The nucleic acid molecules of the invention can contain naturally occurring sequences,<br>
or sequences that differ from those that occur naturally, but, due to the degeneracy of the<br>
genetic code, encode the same polypeptide. In addition, these nucleic acid molecules are<br>
not limited to coding sequences, e.g., they can include some or all of the non-coding<br>
sequences that lie upstream or downstream from a coding sequence.<br>
The nucleic acid molecules of the invention can be synthesized (for example, by<br>
phosphoramidite-based synthesis) or obtained from a biological cell, such as the cell of a a<br>
prokaryote (e.g., a bacterium such as a Bacillus bacterium or Escherichia coli). Combinations<br>
or modifications of the nucleotides within these types of nucleic acids are also encompassed.<br>
The isolated nucleic acid molecules of the invention encompass segments that are not<br>
found as such in the natural state. Thus, the invention encompasses recombinant nucleic acid<br>
molecules incorporated into a vector (for example, a plasmid or viral vector) or into the genome<br>
of a heterologous cell (or the genome of a homologous cell, at a position other than the natural<br>
chromosomal location). Recombinant nucleic acid molecules and uses therefore are discussed<br>
further below.<br>
The invention also encompasses: (a) vectors (see below) that contain any of the<br>
foregoing sequences (including coding sequence segments) and/or their complements (that is,<br>
"antisense" sequences); (b) expression vectors that contain any of the foregoing sequences<br>
(including coding sequence segments) operably linked to one or more transcriptional and/or<br>
translational regulatory elements (TRE; examples of which are given below) necessary to direct<br>
expression of the coding sequences; (c) expression vectors encoding, in addition to a FBFP, a<br>
sequence unrelated to the FBFP, such as a reporter, a marker, or a signal peptide fused to<br>
FBFP; and (d) genetically engineered host cells (see below) that contain any of the foregoing<br>
expression vectors and thereby express the nucleic acid molecules of the invention.<br><br>
21<br>
The TRE referred to above and further described below include but are not limited to<br>
inducible and non-inducible promoters, enhancers, operators and other elements that are<br>
known to those skilled in the art and that drive or otherwise regulate gene expression in<br>
prokaryotes. Such regulatory elements include but are not limited to a promoter fragment<br>
cloned from the Bacillus temperature phage SPO2 (Schoner et al., (1983) Gene 22: 47-57),<br>
sucrose inducible sacB promoter (Lee ef al., (2000), Appl Environ Microbiol 66: 476-480), vegl<br>
promoter from B. subtilis (Lam ef a/.,(1998), J Biotechnol 63: 167-177), aprE promoter (Olmos<br>
Soto and Contreras-Flores, (2003) Appl Microbiol Biotechnol 62: 369-373), AmyQ promoter<br>
from Bacillus amyloliquefaciens (Widner ef al., (2000), J Industrial Microbiol Biotechnol 25:204-<br>
212) and the temperature-sensitive C1 regulated promoter system (Scofield et al., (2003), Appl.<br>
Environ. Microbiol. 69 :3385-3392. Of particular interest is the D promoter that is the natural<br>
promoter of the B. halodurans flagellin (hag) gene [Sakamoto et al. (1992) J. Gen. Microbiol.<br>
138:2159-2166].<br>
Host bacterial cells can be of the same species or genus as the species or genus from<br>
which the flagellin-derived sequence of the FBFP expressed, or to be expressed, by the host<br>
cell was obtained, or not. The host cells and the flagellin-derived sequence will preferably be of<br>
the same genus, more preferably of the same species.<br>
Moreover the invention provides a substantially pure culture of a microorganism (e.g., a<br>
microbial cell such as a bacterial cell). As used herein, a "substantially pure culture" of a<br>
microorganism is a culture of that microorganism in which less than about 40% (i.e., less than<br>
about : 35%; 30%; 25%; 20%; 15%; 10%; 5%; 2%; 1%; 0.5%; 0.25%; 0.1%; 0.01%; 0.001%;<br>
0.0001%; or even less) of the total number of viable microbial (e.g., bacterial cells) in the culture<br>
are viable microbial cells other than the microorganism. The term "about" in this context means<br>
that the relevant percentage can be 15% percent of the specified percentage above or below<br>
the specified percentage. Thus, for example, about 20% can be 17% to 23%. Such a culture of<br>
microorganisms includes the microorganisms and a growth, storage, or transport medium.<br>
Media can be liquid, semi-solid (e.g., gelatinous media), or frozen. The culture includes the<br>
cells growing in the liquid or in/on the semi-solid medium or being stored or transported in a<br>
storage or transport medium, including a frozen storage or transport medium. The cultures are<br>
in a culture vessel or storage vessel or substrate (e.g., a culture dish, flask, or tube or a storage<br>
vial or tube).<br>
The microbial cells of the invention can be stored, for example, as frozen cell<br>
suspensions, e.g., in buffer containing a cryoprotectant such as glycerol or sucrose, as<br>
lyophilized cells.   Alternatively, they can be stored, for example, as dried cell preparations<br><br>
22<br>
obtained, e.g., by fluidised bed drying or spray drying, or any other suitable drying method.<br>
Similarly the enzyme preparations can be frozen, lyophilised, or immobilized and stored under<br>
appropriate conditions to retain activity.<br>
The invention also provides methods of making a FBFP of the invention. In such<br>
methods, a cell (any of those described herein) expressing an expression vector with which the<br>
cell is transformed, and in which the nucleotide sequence encoding is operably linked to one or<br>
more trancriptional and/or a translational regulatory elements, is cultured. The FBFP is then<br>
obtained (e.g., recovered from the culture). Obtaining or recovery from the culture can involve<br>
isolation of the FBFP from the cells or from the culture medium. The FBFP of the invention are<br>
actively secreted through flagellin channels from cells recombinantly expressing them and<br>
remain attached to the cell surface in chains up to 20 000 monomers where they may actively<br>
be separated from the cell surface into the culture medium, e.g. by mechanical shearing<br>
Gene-disrupted Cells<br>
In a substantial number of certain substantially pure cultures of cells of the invention,<br>
and in certain isolated cells, the endogenous flagellin gene has been disrupted such that it<br>
either encodes no flagellin protein or it encodes a non-functional flagellin. As used herein, a<br>
"substantial number of cells in a culture" is at least 60% (e.g., at least: 70%; 80%; 85%; 90%;<br>
95%; 98%; 99%; 99.5%; 99.8%; or even 100%) of the cells in the culture. As used herein, a<br>
"non-functional flagellin" is a flagellin that cannot be transported to the surface of the cell making<br>
it and/or cannot be expressed on the surface of a cell making it. An example of a non-functional<br>
flagellin polypeptide is a B. halodurans flagellin lacking amino acids 14 to 226 (see Example 4).<br>
Such cells are useful as host cells for producing and expressing on their surfaces an FBFP of<br>
the invention after transformation with an expression vector encoding the FBFP. Methods of<br>
disrupting endogenous genes are known in the art and generally involve homologous<br>
recombination. A particularly useful method is the forced integration method described in<br>
Example 4 or a variant of it.<br>
In addition to lacking a functional flagellin gene, a substantial number (see above) of the<br>
cells of the above-described cultures, and the isolated cells, can have one or more genes<br>
encoding cell wall proteases disrupted by the methods referred to above. Disruption of a cell<br>
wall protease gene results in the production of none of the relevant cell wall protease or in the<br>
production of non-functional cell wall protease. As used herein, a "non-functional cell wall<br>
protease" is one having less than 20% (e.g., less than: 10%; 5%; 2%; 1%; 0.5%; 0.2%; 0.1%;<br>
0.01%; or none) of the proteolytic activity of the relevant wild-type cell wall protease. By<br>
disrupting one or more cell wall protease genes, the level of expression on the surface of the<br><br>
23<br>
relevant disrupted cell of an FBFP encoded by an expression vector with which the cell is<br>
transformed can be increased. One such gene is the wrpA gene (see Example 7). Other<br>
protease genes include apr, alp, vpr, apr X among others. The cells can be bacterial cells of<br>
any of the genera, species, and strains recited herein. An example of bacterial cells lacking a<br>
functional flagellin gene and lacking the coding sequence of the cell wall protease gene is the<br>
BhFC04 strain of 6. halodurans (see Example 7).<br>
Methods of Using FBFP<br>
As pointed out above, the FBFP can be useful in, without limitation, bioremediation,<br>
biomining, enzyme-mediated substrate conversions, and as immunogens for activating immune<br>
responses in any of a variety of mammals. Thus, the invention features methods of performing<br>
bioremediation, biomining, enzyme-mediated substrate conversions, use in preparing<br>
immunogen preparations, as a substance or compostion for use in a method of treatment or<br>
prevention and activating immune responses in mammalian subjects.<br>
Bioremediation and Biomining<br>
As indicated above, both bioremediation and biomining involve exposing a fluid e.g.<br>
water or industrial toxic waste streams or run-off from mines such as gold and platinum mines to<br>
a solid substrate that has been adapted to bind to appropriate metal ions or atoms in the fluid.<br>
In the case of bioremediation, the solid substrate with metal atoms or ions bound to it is either<br>
discarded or processed for reuse by eluting off and discarding the metal atoms or ions. In the<br>
case of the biomining, the metal atoms or ions are separated from the solid substrate and<br>
further processed as appropriate.<br>
In the bioremediation and biomining processes of the invention, the fluid can be exposed<br>
to bacteria (any of those listed herein) expressing on their surfaces FBFP containing a metal-<br>
binding heterologous polypeptide. The bacteria can be live or dead (e.g., heat killed) and can<br>
be contained, for example, an appropriate filtering apparatus. Alternatively, the FBFP can be<br>
isolated from the appropriate recombinant bacteria and bound (e.g., covalently) to a solid<br>
substrate (e.g., metal, plastic, cellulose, agarose, or synthetic polymer such as nylon). The solid<br>
substrate can be in the form of, for example, a sheet, beads, particles, fibers, or threads. As yet<br>
another alternative, the heterologous polypeptide can be cleaved out of the FBFP, isolated, and<br>
bound to one of the above-listed solid substrates. The fluid is passed over or through, for<br>
example, a column or bed of the bacteria or substrate with FBFP (or heterologous polypeptide)<br>
bound to it, with the flow rate adjusted so that optimal binding of the metal atoms or ions to the<br>
FBFP occurs. The fluid can be contacted with the FBFP (or heterologous polypeptide) once or<br>
a plurality of times as required for optimal binding of the metal atoms or ions to the FBFP.<br><br>
24<br>
In bioremediation, the fluid is then used for whatever purpose it is intended, e.g., as<br>
drinking water or for an industrial process of interest. The FBFP source will generally be<br>
discarded or it can be regenerated by removal of the metal atoms or ions and reused.<br>
In the case of biomining, the metal atoms or ions are recovered from the FBFP source.<br>
This can be achieved by exposing bacteria expressing the FBFP or the isolated FBFP to<br>
hydrolysing conditions e.g., acidic or alkaline conditions or high concentrations of positively<br>
charged ions as EDTA such that the metals are dissociated from the peptides and can be re-<br>
isolated.<br>
Enzyme-mediated Substrate Conversions<br>
In these processes of the invention a substrate of interest could be esters or constituent<br>
carboxylic acids and alcohols. The constituent molecules may be aliphatic, aromatic in<br>
combinations thereof, and may comprise other functional groups. The constituent molecules<br>
may be larger or small and may be found in foodstuffs (e.g. fatty acids, menthol) "or in<br>
pharmaceutical compounds or intermediates (e.g. naproxen, ibuprofen). The lipases and<br>
esterases may be used in food and beverage processing, bioremediation or synthesis of fine<br>
chemicals and Pharmaceuticals. In the latter the enzymes may be used for chemo- regio- or<br>
stero- selectivity. In particular the lipases and esterases are often applied to the stero- selective<br>
synthesis or hydrolysis of esters thereby allowing for resolution desired chiral compounds from<br>
racemic mixtures.<br>
In this case, after a sufficient amount of product has been generated, the reaction can be<br>
stopped and the product extracted or isolated from the reaction mixture.<br>
In this case, after the reaction, the composition or mixture is processed as desired<br>
including, for example, inactivating the enzyme and/or separating the mixture or composition<br>
from the FBFP source.<br>
The reactions can be carried out in, for example, high volume fermenters. The FBFP (or<br>
the heterologous polypeptide enzyme (or fucleaved from the FBFP) can be used in the same<br>
forms as described above for bioremediation and biomining. Thus, live or dead bacteria<br>
expressing on their surfaces the appopriate FBFP, isolated FBFP, or heterologous polypeptide<br>
enzyme cleaved from the FBFP can be added directly to the reaction mixture containing the<br>
enzyme substrate. Alternatively, the isolated FBFP or the heterologous polypeptide enzyme<br>
cleaved from the FBFP, can be bound to one of the above-described solid substrates (e.g.,<br><br>
25<br>
agarose beads). Optionally and preferably, the reaction mixture is agitated or stirred. This is<br>
done, for example, where recombinant bacteria expressing the FBFP on their surfaces or solid<br>
substrate-bound agents are used as a source of the FBFP, in order to keep the bacteria or solid<br>
substrates suspended. At a time predetermined to be useful for product generation and/or<br>
substrate depletion, the product and/or composition or mixture depleted of substrate is<br>
separated from the FBFP source and processed as desired. Alternatively, the reaction can be<br>
monitored and once a desired level of product and/or substrate depletion is observed, the<br>
product and/or composition or mixture depleted of substrate is separated from the FBFP source<br>
and processed as desired. Prior to separation of the product and/or composition or mixture<br>
from the FBDP source, the enzyme reaction can optionally be stopped (e.g., by heat).<br>
Methods of Activating an Immune Response<br>
The invention features methods of activating mammalian immune responses in which<br>
cells of the immune system are exposed to one or more FBFP of the invention in which the<br>
heterologous polypeptide is an immunogenic polypeptide (see above) or to immunogenic<br>
heterologous polypeptides cleaved from FBFP. In the case of FBFP, the cells of the immune<br>
system can be contacted with isolated FBFP or recombinant bacteria expressing the FBFP on<br>
their surfaces. Such bacteria can be alive, dead, or attenuated. Immune responses that can be<br>
activated by these agents can be, for example, antibody-producing (B lymphocyte) responses.<br>
The FBFP (either isolated or on the surfaces of bacteria) are also useful for introducing peptide<br>
epitopes into antigen presenting cells (APC) in order to generate MHC (major histocompatibility<br>
complex) class I- or class II- restricted T cell responses. Such responses are typically only<br>
generated by recognition of peptide epitopes produced by processing of polypeptides<br>
synthesized within an appropriate APC. In addition, the FBFP of the invention can also be<br>
useful for sensitizing target cells for lysis by cytotoxic T lymphocytes (CTL) with specificity for a<br>
peptide epitope (as heterologous polypeptide) that the FBFP contains.<br>
The methods of the invention can be performed in vitro, in vivo, or ex vivo. In vitro<br>
application of the FBFP or cleaved immunogenic heterologous polypeptides can be useful, for<br>
example, in basic scientific studies of immune mechanisms or for production of activated T cells<br>
for use in either studies on T cell function or, for example, passive immunotherapy.<br>
In the in vitro methods of the invention, T cells (CD4+ and/or CD8+) obtained from a<br>
mammalian subject are cultured with a FBFP (isolated or expressed on the surface of bacteria,<br>
preferably dead bacteria) and APC, preferably, but not necessarily, obtained from the same<br>
individual as the T cells. Where the APC are obtained from a different individual, the donor of<br>
the T cells and the donor of the APC will preferably express at least one major histocompatibility<br><br>
26<br>
complex (MHC) molecule (e.g., a MHC class I molecule) in common. APC can be essentially<br>
any MHC molecule-expressing cell. Where it is desired to elicit a MHC class I restricted<br>
immune response, the APC will express MHC class I molecules (and optionally MHC class II<br>
molecules) and where it is desired to elicit an MHC class II restricted immune response, the<br>
APC will express MHC class II molecules (and optionally MHC class I molecules). The APC will<br>
optimally also express one or more co-stimulatory molecules, e.g., the B7 family of molecules.<br>
Thus APC can be, for example, dendritic cells (DC), macrophages, monocytes, B cells, or cell<br>
lines (clonal or non-clonal) derived from any of these cells. They can also be any cell type (e.g.,<br>
fibroblasts) transfected or transduced with and expressing a polynucleotide encoding an<br>
appropriate MHC molecule. Such cultures can also be supplemented with one or more<br>
cytokines or growth factors such as, without limitation, IL-1, IL-2, IL-3, IL-4, IL-6, IL-7, IL-12, IL-<br>
13, IL-15, IFN-, tumor necrosis factor- (TNF- ), granulocyte macrophage colony-stimulating<br>
factor (GM-CSF), or granulocyte-colony stimulating factor (G-CSF). The cultures can be<br>
"restimulated" as often as necessary with either the FBFP or cleaved immunogenic<br>
heterologous polypeptide. The cultures can also be monitored at various times to ascertain<br>
whether the desired level of immune reactivity (e.g., CTL activity) has been attained.<br>
The FBFP (and immunogenic heterologous polypeptides cleaved from FBFP) are<br>
generally useful for generating immune responses, as prophylactic vaccines or immune<br>
response-stimulating therapeutics, as a substance or compostion for us in methods of treatment<br>
or prevention, and for preparing immunogenic preparations for use in methods of treatment or<br>
prevention. Thus, they can be used, for example, as immunogenic preparations, vaccines or<br>
therapeutic agents against infectious diseases due to any of the pathogens listed herein. It is<br>
also possible to administer the immunogenic peptide as a fusion with the flagellin protein. This<br>
will enhance the immunogenic response of the target organism since the flagellin has the same<br>
effect as an adjuvant (An ingredient-as in a prescription or solution-that facilitates or modifies<br>
the action of the principal ingredient). Adjuvants have the ability to stimulate innate immunity<br>
and in turn activate the adaptive immune response. It has already been established that<br>
flagellin induces an inflammatory response through the activation of APC's (see above). An<br>
example is the successful creation and presentation of a flagellin enhanced green fluorescent<br>
protein (EGFP) fusion protein. The flagellin-EGFP fusion was capable of stimulating APC's and<br>
also specific anti EGFP T-cell responses. EGFP alone was unable to stimulate neither APC's<br>
nor specific T-cell responses (Cuadros et al., 2004, Inf. Immun. Vol 72, 2810-2816, McSorley et<br>
al., 2002, J. Immunol. Vol 169, 3914-3919). Other peptides inserted into the flagellin which<br>
induced an immune response include Cholera toxin subunit B, Hepatitus B epitopes,<br>
Streptococcus pyogenes M protein epitope, HIV epitopes (gp 41, gp120), influenza A<br>
hemagglutinin epitope and various cell surface antigens from Plasmodium sp., Rotavirus,<br><br>
27<br>
Corynebacterium diphtheriae and Miningococcal outer membrane protein (Stacker and Newton,<br>
1994, Intern. Rev. Immunol. Vol 2,167-178).<br>
In addition, the FBFP (and immunogenic heterologous polypeptides cleaved from FBFP)<br>
can be useful therapeutics for cancer (e.g., any of those recited above); in cases where a<br>
subject is at relatively high risk for a cancer (e.g., lung cancer in a tobacco smoker or melanoma<br>
in a subject with multiple nevi), appropriate fusion agents can be used as vaccines.<br>
As used herein, "prophylaxis" means complete prevention of the symptoms of a disease,<br>
a delay in onset of the symptoms of a disease, or a lessening in the severity of subsequently<br>
developed disease symptoms. As used herein, "therapy" means a complete abolishment of the<br>
symptoms of a disease or a decrease in the severity of the symptoms of the disease.<br>
It is understood that, while while the immune responses generated by the FBFP (and<br>
immunogenic heterologous polypeptides cleaved from FBFP) are preferably prophylactic and/or<br>
therapeutic, it is not required that they be. For example, the FBFP (and immunogenic<br>
heterologous polypeptides cleaved from FBFP) can be used in basic scientific studies on<br>
immune responses that are neither prophylactic nor therapeutic, for generating antibodies<br>
useful, for example, in detecting or purifying any of a variety of antigens, e.g., microbial antigens<br>
or TAA such as those recited herein.<br>
The methods of the invention can be applied to a wide range of species, e.g., humans,<br>
non-human primates (e.g., monkeys), horses, cattle, pigs, sheep, goats, dogs, cats, rabbits,<br>
guinea pigs, hamsters, rats, and mice.<br>
In Vivo Approaches<br>
In one in vivo approach, the FBFP itself, bacteria (e.g., commensal bacteria) expressing<br>
the FBFP on their surfaces, or immunogenic heterologous polypeptides cleaved from FBFP are<br>
administered to the subject. Generally, the fusion agents of the invention will be suspended in a<br>
pharmaceutically-acceptable carrier (e.g., physiological saline) and administered orally or<br>
transdermally or injected (or infused) intravenously, subcutaneously, intramuscularly,<br>
intraperitoneally, intrarectally, intravaginally, intranasally, intragastrically, intratracheally, or<br>
intrapulmonarily. They can be delivered directly to an appropriate lymphoid tissue (e.g. spleen,<br>
lymph node, or mucosal-associated lymphoid tissue (MALT)). The dosage required depends on<br>
the route of administration, the nature of the formulation, the nature of the patient's illness, the<br>
subject's size, weight, surface area, age, and sex, other drugs being administered, and the<br>
judgment of the attending physician.    Suitable dosages of isolated FBFP or immunogenic<br><br>
28<br>
heterologous polypeptides cleaved from FBFP are in the range of 0.001-10.0 mg/kg. Wide<br>
variations in the needed dosage are to be expected in view of the variety of FBFP (and<br>
immunogenic heterologous polypeptides cleaved from FBFP) available and the differing<br>
efficiencies of various routes of administration. For example, oral administration would be<br>
expected to require higher dosages than administration by i.v. injection. Variations in these<br>
dosage levels can be adjusted using standard empirical routines for optimization as is well<br>
understood in the art. Administrations can be single or multiple (e.g., 2- or 3-, 4-, 6-, 8-, 10-, 20-<br>
, 50-, 100-, 150-, or more fold). Encapsulation of the polypeptide in a suitable delivery vehicle<br>
(e.g., polymeric microparticles or implantable devices) may increase the efficiency of delivery,<br>
particularly for oral delivery.<br>
Alternatively, a polynucleotide containing a nucleic acid sequence encoding a FBFP of<br>
interest can be delivered to an appropriate cell of the animal. Expression of the coding<br>
sequence will preferably be directed to lymphoid tissue of the subject by, for example, delivery<br>
of the polynucleotide to the lymphoid tissue. This can be achieved by, for example, the use of a<br>
polymeric, biodegradable microparticle or microcapsule delivery vehicle, sized to optimize<br>
phagocytosis by phagocytic cells such as macrophages. For example, PLGA (poly-lacto-co-<br>
glycolide) microparticles approximately 1-10 m in diameter can be used. The polynucleotide is<br>
encapsulated in these microparticles, which are taken up by macrophages and gradually<br>
biodegraded within the cell, thereby releasing the polynucleotide. Once released, the DNA is<br>
expressed within the cell. A second type of microparticle is intended not to be taken up directly<br>
by cells, but rather to serve primarily as a slow-release reservoir of nucleic acid that is taken up<br>
by cells only upon release from the micro-particle through biodegradation. These polymeric<br>
particles should therefore be large enough to preclude phagocytosis (i.e., larger than 5 m and<br>
preferably larger than 20 m).<br>
Another way to achieve uptake of the nucleic acid is using liposomes, prepared by<br>
standard methods. The vectors can be incorporated alone into these delivery vehicles or co-<br>
incorporated with tissue-specific antibodies. Alternatively, one can prepare a molecular<br>
conjugate composed of a plasmid or other vector attached to poly-L-lysine by electrostatic or<br>
covalent forces. Poly-L-lysine binds to a ligand that can bind to a receptor on target cells<br>
[Cristiano et al. (1995), J. Mol. Med. 73, 479]. Alternatively, lymphoid tissue specific targeting<br>
can be achieved by the use of lymphoid tissue-specific transcriptional regulatory elements<br>
(TRE) such as a B lymphocyte, T lymphocyte, or dendritic cell specific TRE. Lymphoid tissue<br>
specific TRE are known [Thompson et al. (1992), Mol. Cell. Biol. 12, 1043-1053; Todd et al.<br>
(1993), J. Exp. Med. 177, 1663-1674; Penix et al. (1993), J. Exp. Med. 178, 1483-1496].<br><br>
29<br>
Delivery of "naked DNA" (i.e., without a delivery vehicle) to an intramuscular, intradermal, or<br>
subcutaneous site, is another means to achieve in vivo expression.<br>
In the relevant polynucleotides (e.g., expression vectors) the nucleic acid sequence<br>
encoding the FBFP of interest with an initiator methionine and optionally a targeting sequence is<br>
operatively linked to a promoter or enhancer-promoter combination.<br>
Polynucleotides can be administered in a pharmaceutically acceptable carrier.<br>
Pharmaceutically acceptable carriers are biologically compatible vehicles which are suitable for<br>
administration to a human or other mammalian subject, e.g., physiological saline. A<br>
therapeutically effective amount is an amount of the polynucleotide which is capable of<br>
producing a medically desirable result (e.g., a T cell response) in a treated animal. As is well<br>
known in the medical arts, the dosage for any one patient depends upon many factors, including<br>
the patient's size, body surface area, age, the particular compound to be administered, sex,<br>
time and route of administration, general health, and other drugs being administered<br>
concurrently. Dosages will vary, but a preferred dosage for administration of polynucleotide is<br>
from approximately 106 to 1012 copies of the polynucleotide molecule. This dose can be<br>
repeatedly administered, as needed. Routes of administration can be any of those listed above.<br>
Ex Vivo Approaches<br>
In one ex vivo approach, lymphoid cells, including T cells (CD4+ and/or CD8+ T cells),<br>
are isolated from the subject and exposed to the FBFP in vitro (see above). The lymphoid cells<br>
can be exposed once or multiply (e.g., 2, 3, 4, 6, 8, or 10 times). The level of immune activity<br>
(e.g., CTL activity) in the lymphoid cells can be tested after one or more exposures. Once the<br>
desired activity and level of that activity is attained, the cells are reintroduced into the subject via<br>
any of the routes listed herein. The therapeutic or prophylactic efficacy of this ex vivo approach<br>
is dependent on the ability of the ex vivo activated lymphocytes to exert, directly or indirectly, a<br>
neutralizing or cytotoxic effect on, for example, infectious microorganisms, host cells infected<br>
with microorganisms, or tumor cells.<br>
An alternative ex vivo strategy can involve transfecting or transducing cells obtained<br>
from the subject with a polynucleotide containing a FBFP-encoding nucleotide sequence. The<br>
transfected or transduced cells are then returned to the subject. While such cells would<br>
preferably be lymphoid cells, they could also be any of a wide range of types including, without<br>
limitation, fibroblasts, bone marrow cells, macrophages, monocytes, dendritic cells, epithelial<br>
cells, endothelial cells, keratinocytes, or muscle cells in which they act as a source of the fusion<br>
protein for as long as they survive in the subject.   In subjects with cancer, the cells can be<br><br>
30<br>
cancer cells, e.g., their own cancer cells or cells of the same cancer type but from another<br>
individual, preferably an individual having one or more (e.g., one, two, three, four, five, or six)<br>
MHC molecules in common with the subject. The use of lymphoid cells would be particularly<br>
advantageous in that such cells would be expected to home to lymphoid tissue (e.g., lymph<br>
nodes or spleen) and thus the FBFP would be produced in high concentration at the site where<br>
they exert their effect, i.e., activation of an immune response. By using this approach, as in the<br>
above-described in vivo approach using fusion agent-encoding polynucleotides, active in vivo<br>
immunization with the FBFP is achieved. The same genetic constructs and signal sequences<br>
described for the in vivo approach can be used for this ex vivo strategy.<br>
The ex vivo methods include the steps of harvesting cells from a subject, culturing the<br>
cells, transducing them with an expression vector, and maintaining the cells under conditions<br>
suitable for expression of the FBFP. These methods are known in the art of molecular biology.<br>
The transduction step is accomplished by any standard means used for ex vivo gene therapy,<br>
including calcium phosphate, lipofection, electroporation, viral infection, and biolistic gene<br>
transfer. Alternatively, liposomes or polymeric microparticles can be used. Cells that have<br>
been successfully transduced are then selected, for example, for expression of the FBFP or of a<br>
drug resistance gene. If desired, the cells can be treated with an agent (e.g., x- or -irradiation<br>
or mitomycin C) that inhibits cell proliferation; generally where the cells are cancer cells<br>
(particularly cancer cells from the subject or from an individual that is MHC identical to the<br>
subject) will be so treated. The cells are then injected or implanted into the patient.<br>
These methods of the invention can be applied to any of the diseases and species listed<br>
here. Methods to test whether a FBFP or immunogenic heterologous polypeptide cleaved from<br>
a FBFP is therapeutic for or prophylactic against a particular disease are known in the art.<br>
Where a therapeutic effect is being tested, a test population displaying symptoms of the disease<br>
(e.g., cancer patients) is treated with a test FBFP or an immunogenic heterologous polypeptide<br>
cleaved from a FBFP, using any of the above described strategies. A control population, also<br>
displaying symptoms of the disease, is treated, using the same methodology, with a placebo.<br>
Disappearance or a decrease of the disease symptoms in the test subjects would indicate that<br>
the FBFP or immunogenic heterologous polypeptide cleaved from a FBFP was an effective<br>
therapeutic agent.<br>
By applying the same strategies to subjects prior to onset of disease symptoms, FBFP<br>
or immunogenic heterologous polypeptides cleaved from FBFP can be tested for efficacy as<br>
prophylactic agents, i.e., vaccines. In this situation, prevention of onset of disease symptoms is<br>
tested.   Analogous strategies can be used to test for the efficacy of FBFP and immunogenic<br><br>
31<br>
heterologous polypeptides cleaved from FBFP in the prophylaxis of a wide variety of infectious<br>
diseases, e.g., those involving any of the microorganisms listed above.<br>
The following examples serve to illustrate, not limit, the invention.<br>
EXAMPLES<br>
Example 1. Isolation and Sequencing of B. halodurans Alk36 Flaqellin<br>
Aliquots of B. halodurans Alk36 bacteria deposited at the NCIMB Culture Collection<br>
(NCIMB Ltd, Ferguson Building, Craibstone Estate, Bucksburn, Aberdeen, AB21 9YA) under<br>
accession number NCIMB41348 were grown in Luria broth (Tryptone 10 g/l, Yeast extract 5 g/l,<br>
NaC110 g/l), pH8.5 at 42C for various times up to 72 hours. The cell-surface protein fractions<br>
of all samples were prepared by pelleting the cells (30 ml) by centrifugation at 7,000 rpm<br>
(revolutions per minute) in SS34 tubes for 10 min. Six ml of supernatant (extra cellular fraction)<br>
was mixed with 10% (w/v) trichloroacetic acid (TCA; 6 ml) and stirred for 1 hr. Proteins were<br>
pelleted by centrifugation at 7,000 rpm in glass Corex tubes for 20 min, resuspended in 100ul<br>
sample buffer (1X) boiled for 2 min, and loaded onto a 10% SDS-PAGE (sodium dodecyl sulfate<br>
polyacrylamide gel electrophoresis) gel.<br>
The cell pellets were resuspended in distilled H2O (3 ml) and 0.2 N NaOH (3 ml) and<br>
stirred for 30 min at room temperature. Cells were pelleted by centrifugation at 7,000 rpm for 10<br>
min, and the supernatant (6 ml) was mixed with 10% (7) trichloroacetic acid (TCA; 6 ml) and<br>
stirred for 1 hr. Precipitated cell surface proteins were pelleted by centrifugation at 7,000 rpm in<br>
glass Corex tubes for 20 min, resuspended in 100ul sample buffer (1x), boiled for 2 min, and<br>
loaded onto a 10% SDS-PAGE (sodium dodecyl sulfate polyacrylamide gel electrophoresis) gel<br>
(50  g of protein per well) and resolved using a standard procedure.<br>
A highly expressed protein resolved to a band corresponding to -34 kDa. This protein<br>
was present and stable for at least 72 hours (Fig. 1). The resolved protein fraction was<br>
subsequently blotted onto an Immobilon-P membrane (Millipore Corporation, Billerica, MA),<br>
stained for 2 min with Coomassie blue [0.1% R250 in 50% (v/v) methanol], and destained for 5<br>
min in destain solution (50% (v/v) methanol; 10% (v/v) ascetic acid). The ~34kDa protein band<br>
from Figure 1 lane 4 was cut out and the amino acid sequence of the blotted protein determined<br>
by N-terminal sequencing using a Perkin Elmer Applied Biosystems (Foster City, CA, U.S.A)<br>
Procise 491 Sequencer, according to manufacturer's instructions. The sequence of the first<br>
22 amino acids was obtained (Fig. 2).<br><br>
32<br>
Example 2. Bioinformatic Analysis of B. halodurans Alk36 Flaqellin<br>
The 22 amino acid peptide sequence obtained from the N-terminus of B. halodurans<br>
Alk36 flagellin was used to screen the Swiss-Protein database for homologous proteins. This<br>
peptide sequence demonstrated significant homology to the N-terminus of the hag product<br>
flagellin protein from the alkalophilic Bacillus sp. C-125 [Sakamoto et al. (1992), J. Gen.<br>
Microbiol. 138: 2159-2166], now renamed B. halodurans C-125 [Takami et al. (1999), Biosci.<br>
Biotechnol. Biochem. 5:943-945] (Fig. 3). The first 12 amino acids of the N-terminus of B.<br>
halodurans Alk36 flagellin demonstrated 92% identity with the N-terminus of B. halodurans C-<br>
125 flagellin [Sakamoto ef al., supra]. Since further comparison of the amino acid sequences of<br>
flagellin proteins from Escherichia coli, Salmonella typhimurium and Bacillus subtilis 168<br>
[LaVallie ef al. (1989) J. Bacteriol. 171:3085-3094] indicated that the N- and C-terminal regions<br>
of this family of flagellin proteins demonstrate significant conservation of sequence, the ~34kDa<br>
B. halodurans Alk36 protein was concluded to be a flagellin protein and therefore a product of<br>
the hag gene.<br>
Example 3. PCR Amplification and Cloning of the B. halodurans Alk36 Gene<br>
Cloning of the hag gene open reading frame (ORF)<br>
The ORF of the hag gene was amplified by a polymerase chain reaction (PCR) from<br>
chromosomal DNA using a forward primer (F-flag; 5' CTC CTG CAG AAT CAC AAT TTA CCA<br>
GCA 3' Tm=58.1 C) and a reverse primer (R-flag; 5' GGT TCG AAC ATC GCT TGA GAC GCT<br>
TC 3' Tm=61 C) based on the conserved sequences of the N- and C-terminal regions of the<br>
hag gene of B. halodurans C-125 (Sakamoto ef al., 1992). The PCR reaction included final<br>
concentrations of the template chromosomal DNA (100 ug/ul), F-flag (0.5 um/10 ul), R-flag (0.5<br>
ul/10 ul), dNTPs (deoxyribonucleotide triphosphates; 0.8 ul/10 ul) and 1ul Pwo DNA polymerase<br>
(5 u/ul, Roche) in 1X PCR Buffer with MgCI2 (2.0mM) to a final volume of 100 ul. The PCR<br>
reaction was incubated according to standard procedures with appropriate optimization. The<br>
PCR product was resolved by standard agarose (1.0%) electrophoresis using low-melting point<br>
agarose and yielded a fragment resolving to a band corresponding to 800 bp (base pairs). The<br>
800 bp band was excised under long-wave UV, the DNA extracted from the gel using the BIO<br>
101 (Irvine, CA, U.S.A.) Geneclean system, and cloned into the EcoRV site of pMOSBlue<br>
(blunt ended cloning kit from Amersham Pharmacia Biotech, Piscataway, NJ, U.S.A.) according<br>
to the manufacturer's instructions to create pMOSBIue(Flg).<br>
pMOSBIue(Flg) was electroporated into E. coli JM83 cells using a Bio-Rad (Hercules,<br>
CA, U.S.A) Gene Pulser (1.8 kV, 25 F, 200 ohm) according to the manufacturer's instructions<br>
and plated onto Luria agar plates containing ampicillin (50 jig/ml) and tetracycline (15 ng/ml).<br>
For blue/white selection, 35    I of X-gal (5-bromo-4-chloro-3-indolyl- -D-galactopyranoside)<br><br>
33<br>
(50mg/ml) and 20 I of IPTG (isopropyl- -D-thiogalactopyranoside) (100 mM) was spread onto<br>
plates. Plates were incubated overnight at 37
phenotype containing LacZ M15 is maintained and thus eliminates the background of non-<br>
recombinant white colonies which have lost the phenotype. Transformant colonies were picked<br>
into Luria broth containing ampicillin (50 jig/ml) and tetracycline (15 mglml), and incubated with<br>
shaking at 37C overnight. Plasmid DNA was extracted using a Qiagen (Hilden, Germany)<br>
plasmid purification kit and sequenced in both directions by the University of Cape Town<br>
(Microbiology Department; Cape Town, South Africa) DNA sequencing service using standard<br>
T7 and U-19 mer primers. On comparison, the B. halodurans Alk36 hag ORF demonstrated<br>
100% identity in nucleotide sequence to the 6. halodurans C-125 hag ORF (Fig. 4).<br>
Cloning the flanking regions of the hag gene fragment using inverse PCR.<br>
To complete the cloning of B. halodurans Alk36 hag, including its upstream and<br>
downstream regulatory regions, the ORF flanking regions were amplified using inverse PCR<br>
(iPCR). The method used for iPCR was adapted from the method by Ochman et al., [(1990)<br>
Amplification of flanking sequences by inverse PCR. In PCR Protocols: A Guide to Methods and<br>
Applications (ed. Innis M.A. ef a/.), p. 219-227. Academic Press, Inc.] B. halodurans Alk36<br>
chromosomal DNA was digested with a variety of restriction enzymes so as to identify restriction<br>
enzymes that "restrict" the core region only once and generate fragments of approximately 1 kb<br>
flanking the known 800 bp fragment which could be easily amplified. The digested chromosomal<br>
DNA was separated on a 1% agarose gel, blotted onto Immobulon-P membrane (Millipore) as<br>
described by Reed et al. [(1985) Nucleic Acids Res. 13:7207-7221] and probed with the DIG-<br>
labeled 800 bp fragment according to the manufacturer's specifications (Boehringer Mannheim,<br>
Mannheim, Germany). The DIG-labeled bands were detected using a chemiluminescent reaction<br>
(Boehringer Mannheim). From the blot (Fig. 5), it was concluded that the most promising digests<br>
were the HindIII (for the downstream region) and Acd (for the upstream region) digests.<br>
Protocol for inverse PCR (iPCR)<br>
B. holodurans Alk36 chromosomal digests (Hindlll and Accl) were separated on a 1%<br>
low-melting agarose gel. After electrophoresis, the region of the gel containing the appropriate<br>
sized fragments was excised with a razor blade. The gel slice was heated to 68C for 20 min to<br>
melt the agarose. The DNA fragments were re-ligated under conditions to favour the formation<br>
of monomeric circles in a total reaction volume of 50 I containing 10 I of the molten agarose, 5<br>
I of a 10x ligation buffer, 34 I of distilled H2O, and 1 Weiss unit of T4 DNA ligase. The<br>
reaction was incubated overnight at 15C and terminated by heating to 68C for 15 min. 10 I of<br>
a ligation mix was used in a 100   I PCR reaction using the following primers:<br><br>
34<br>
IF: 5' GCT GAG TCT CGT ATC CGT GAC (Tm - 56.2C)) (SEQ ID NO:30)<br>
IR: 5' CCT GCA GCA TCG TCT CCT GCA (Tm-58.1*C) (SEQ ID N0:31)<br>
The PCR reaction was set up as described above and incubated at 94C for 2 minutes,<br>
followed by a 3-step cycle of 949C for 1 minute, 50C for 1 minute, and 72C for 2 minutes,<br>
repeated a total of 35 cycles. A final extension step of 5 minutes at 72C was included.<br>
After completion of the iPCR, the reaction products were resolved on a 1% agarose gel<br>
and visualized by ethidium bromide staining. From the gel (Fig. 6) it can be seen that PCR<br>
products of approximately 1.15 kb and 0.925 kb were amplified with the Accl and HindIII digests<br>
respectively. The size of the PCR products correlated well with the results obtained from the<br>
southern blot.<br>
The PCR reactions were purified using the High Pure PCR purification kit (Boehringer<br>
Mannheim). The resulting DNA was used for sequencing with the same two primers as primers<br>
for the PCR reactions. DNA sequencing confirmed that both PCR fragments contained the<br>
correct up- and downstream regions. The two fragments were cloned into the pMOSBlue vector<br>
and again sequenced using the T7 and U19- mer primers. This confirmed the initial sequence<br>
of the iPCR sequenced samples. The two clones generated 977 bp of new sequence data<br>
upstream of the 800 bp core region and 796 bp downstream (Fig. 7). These sequences are<br>
sufficient to include both upstream and downstream regulatory regions. These regulatory<br>
region sequences were compared to those of B. halodurans C-125 and found to demonstrate<br>
significant identity (greater than 99.9%), with only four differing base pairs. The complete hag<br>
gene with its regulatory regions is shown in Fig. 8.<br>
Example 4.   Inactivation of the Endogenous B. halodurans Alk36 Chromosomal Flaqellin<br>
Gene<br>
The plasmid pE194 (DSMZ 4554) was obtained from the German culture collection<br>
DSMZ (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder weg<br>
1b, 38124 Braunschweig, Germany). Plasmid pE194 was isolated from Staphylococcus aureus<br>
bacteria, transformed into Bacillus subtilis bacteria, and shown to be stably maintained at<br>
temperatures up to 34C. However, at temperatures higher than 37C the copy number<br>
decreases with plasmid loss by segregation [Weisblum et al. (1979) J. Bacteriol. 137:635-643].<br>
Above 42C the plasmid cannot replicate and the only way the cell can survive on<br>
chloramphenicol is by integrating the plasmid into the chromosome. A shuttle-vector<br>
(pSEC194) was created using only the origin of replication from pE194, the chloramphenicol<br><br>
35<br>
gene from pC194, and the E.coli origin of replication as well as the ( -lactamase) gene from the<br>
Bluescript pSK vector from Stratagene (La Jolla, CA, U.S.A). Since pE194 is unstable at<br>
temperatures above 370 and is unable to replicate above 43C, this feature was used to force<br>
an integration event in B. halodurans Alk36.<br>
All B. halodurans Alk36 strains were transformed using the protoplast transformation<br>
method according to Chang and Cohen [(1979) Mol. Gen. Genet. 168:111-115].<br>
Construction of shuttle vector pSEC 194<br>
The Bacillus/E. coli shuttle vector (pSECi94) was constructed by replacing the<br>
erythromycin resistance gene on pE194 with a chloramphenicol resistance gene and ligating the<br>
pSK ori to the pE194 ori. Plasmid pE194 was digested with Taq\. The 1.35 kb band containing<br>
the ori was ligated to pSK digested with Clal (2.959 kb) and transformed into E. coli DH10B to<br>
create pSE194 (Fig. 9A). Plasmid pJM103 was digested with BglII I PvuII to obtain the<br>
chloramphenicol gene (1.2 kb) originally from pC194, [lordanescu et al. (1980) Plasmid 4:256-<br>
260]. The fragment containing the gene was ligated to pSE194 digested with BamHI / Smal<br>
and transformed into E coli DH10B to create pSEC194 (Fig. 9B). It was important to screen for<br>
transformants in E. coli on plates containing ampicillin since screening with chloramphenicol led<br>
to deletions in the construct. However, when pSEC194 was transformed into B. halodurans and<br>
chloramphenicol was used as a marker, clones obtained were stably maintained at 30C- 34C.<br>
This vector was used as a shuttle vector between E coli I B. halodurans Alk36 and B. subtilis<br>
and for the integration of genes of interest into the B. halodurans Alk36 chromosome.<br>
Construction of integration vector pSEC194Flg-<br>
A defective (endogenous gene-disrupting) hag gene was constructed using only the isl-<br>
and C-terminal ORF regions, as well as some of the upstream and downstream regulatory<br>
regions, ligated into pSEC194 and transformed into 8. halodurans Alk36. In the construction of<br>
pSECFIg- most of the internal region of the hag gene (Fig. 10A) has been deleted. The<br>
following primers were used to obtain the two PCR products needed for construction of the Fig-<br>
fragment:<br>
UPFor: 5' GC GGA TCC GTG TGG TGA CAT TTG AC 3' (BamHI) (SEQ ID NO:32)<br>
UPRev: 5' GC TCT AGA CGA TGC GCA TTC ATT GCT GG 3' (Xbal) (SEQ ID NO:33)<br>
DownFor: 5' GC TCT AGA GAG TCT CGT ATC CGT G 3' (Xbal) (SEQ ID NO:34)<br>
DownRev: 5' CG CTG CAG AAG AGG AAC GTA AAC G 3' (Pstl) (SEQ ID NO:35)<br><br>
36<br>
The PCR products were obtained using standard procedures with appropriate<br>
optimization and were ligated together into pBCKS (Stratagene), digested with BamHi/Psti in a<br>
3 way ligation to obtain the Fig- fragment. pBCFIg- was then digested with EcoR1 and cloned<br>
into pSEC194 digested with EcoR1 to obtain pSECFIg- (Fig. 10B).<br>
the protocol used for integration was a combination of two different methods described<br>
by Biswas et al. [(1993) J. Bacteriol. 175:3628-3635] and Poncet et al. [(1997) Appl. and<br>
Environ. Microbiol. 63:4413-4420]. A first crossover (single crossover (sco)) event was forced<br>
by incubation of the B. halodurans Alk36 transformants containing pSEC194Flg- picked into 25<br>
ml LB (pH 8.5) with chloramphenicol (10 mg/ml) at 52C for 16-24 hours. Serial dilutions of the<br>
resulting cell suspension were made and plated onto LA Luria agar (pH 8.5, 10ml/ml<br>
chloramphenicol) and LA (pH 8.5) plates and incubated overnight at 52C. Growth on LA (pH<br>
8.5, 10 ml/ml chloramphenicol) and LA (pH 8.5) plates was compared to determine integration<br>
efficiencies. The sco event was determined with colony PCR (Fig. 11). Two chloramphenicol<br>
resistant clones, 49 and 50, were identified and clone 49 was then used to create a second<br>
cross-over (double crossover (dco)) event. This was achieved by subsequent incubation of<br>
clone 49 in 25 ml LB Luria broth (pH 8.5) for 2-2V2 hrs (log phase) at 30C in the absence of<br>
chloramphenicol. The culture was diluted and plated onto LA (pH 8.5) plates, and incubated at<br>
30cC overnight. Colonies were picked in duplicate onto LA (pH 8.5, 10 uJ/ml chloramphenicol)<br>
and LA (pH 8.5) plates. Colonies in which gene replacement had occurred (dco) were<br>
chloramphenicol sensitive and non-motile. Chloramphenicol sensitive colonies were picked<br>
onto motility assessment plates (Luria plates, pH 8.5, 0.4% agar + 0.8% gelatin) to screen for<br>
non-motile mutants. A non-motile dco mutant was screened for a dco event by colony PCR<br>
(Fig. 10). This mutated non-motile culture was renamed as Strain BhFC01. The double<br>
crossover was essential in order to ensure the loss of the chloramphenicol gene, which was<br>
also indicative of loss of plasmid sequences, thereby producing a B. halodurans Alk36 mutant<br>
strain BhFC01 ( hag) having a defective hag gene, and demonstrating a non-motile phenotype<br>
(motility plates; Fig. 15A).<br>
Different primer sets were used to demonstrate the dco event:<br>
The SigDF/lnvR primer combination was predicted to give a PCR product of 342 bp if an<br>
intact hag gene is present on the chromosome.<br>
The SigDF/FMCR primer set was predicted to amplify both the defective (400 bp) and<br>
intact hag (1.150) kb genes present on the chromosome. Only the defective band was shown<br>
to be present in the two sco events (Fig. 11, Primer Set B, lanes 1 and 2) even though an intact<br><br>
37<br>
copy was shown to be present using the SigD/lnvR primer set (Fig. 11, primer set B, lane 1). It<br>
seemed likely that the primers preferentially amplify the defective copy. An analogous result<br>
was reported by Aquino de Muro et al. [(2000) Res. Microbiol. 151:547-555]; in that study a very<br>
prominent band was observed for one copy and a very faint band for the other.<br>
The SigDF/M13R primer set was predicted to amplify only the hag gene if there was<br>
plasmid DNA present on the chromosome, which had been obtained from a sco event. After<br>
the dco, all plasmid DNA should be looped out and no PCR product should be obtained. This is<br>
what was observed (Fig. 11, primer set C, lanes 3 and 4). The size of the PCR product (sco)<br>
was expected to be 2.507kb (defective hag) or 3.178 (intact hag) depending on whether or not<br>
there has been an N-terminal or C-terminal cross-over. Appropriate sized bands were<br>
observed.<br>
The UpFor/DownRev primer set was expected to amplify either the defective or<br>
complete hag gene depending on the location of the cross-over. As predicted, an N-terminal<br>
cross-over (defective hag) yielded a PCR product of 1.735 kb (Fig. 11, primer set D, lanes 1 - 3)<br>
and a C-terminal cross-over (intact hag) a product of 2.406 kb (Fig, 11, primer set D, lane 4).<br>
After the dco only the smaller band was amplified.<br>
The results obtained from the PCR profiles showed that a dco event was obtained<br>
resulting in a flagellin- mutant BhFC01. The next step was to confirm that the mutation was in<br>
the hag gene by complementation of the mutation with an intact copy of the hag gene on<br>
PSEC194.<br>
Example 5. Complementation Studies Demonstrated Restoration of Flaaellin Expression<br>
in Strain BhFCOl<br>
Primers used for these experiments were:<br>
FliCR:	5' CAA CAA AGT AAC GGT TGA GCG 3' (SEQ ID NO:36)<br>
InvR:	5' CCT GCA GCA TCG TCT CCT GCA 3' (SEQ ID NO:37)<br>
SigDF:	5' CTC GGT ACC CTC GCG TTA CGC TCT TTC TGT 3' (Kpn\) (SEQ ID<br>
NO:38)<br>
UPFor:	5' GC GGA TCC GTG TGG TGA CAT TTG AC 3" (BamHI) )   (SEQ ID<br>
NO:32)<br>
DownRev:	5' CG CTG CAG AAG AGG AAC GTA AAC G 3' (Pstl) (SEQ ID NO:35)<br>
M13R	51 GGA AAC AGC TAT GAC CAT G 31 (SEQ ID NO:39)<br><br>
38<br>
The complete hag gene containing the D promoter as well as the coding sequence was<br>
cloned into pSEC194 and transformed into Strain BhFCOl. Plasmid pSEC194FliC (containing<br>
the intact flagellin promoter and structural gene), was introduced into BhFCOl by protoplast<br>
transformation to assess the ability to compliment the genomic hag deletion with multiple copies<br>
of the gene. Transformants were toothpicked onto motility assessment plates and showed clear<br>
movement of strain BhFCOl containing pSEC194FliC (Fig 15). A cell-surface protein extract<br>
was obtained of the Strain BhFCOl as in Example 1 and resolved by SDS-PAGE. A protein<br>
resolving to a band corresponding to -34 kDa demonstrated recovery of flagellin protein<br>
expression in Strain BhFCOl transformants (Fig. 14A, lane 6).<br>
Example 6. Protein Modeling of the Variable Region of B. halodurans Alk36 Flaaellin and<br>
Construction of Fusion Protein Expression Vectors<br>
3D-PSSM (three-dimensional, position specific scoring matrix) is a fast, Web-based<br>
method for protein fold recognition using 1D and 3D sequence profiles coupled with secondary<br>
structure and solvation potential information. A summary of the protocol can be found in the<br>
Journal of Molecular Biology, 299:501-522 (Kelley et al. 2000) the disclosure of which is<br>
incorporated herein by reference in its entirety. The putative variable region of B. halodurans<br>
Alk36 flagellin was modeled using 3D-PSSM-software. From the protein model five sites were<br>
chosen within the variable region which would allow for peptide insertions. All these sites are<br>
situated or involve the manipulation of the externally exposed extended -strands and coils of<br>
the variable region of the B. halodurans FliC protein (Fig. 12). Five constructs were created,<br>
using the vector pSEC194 Kpnl IHincII as the backbone.<br>
The NC1 construct contained a deletion of the flagellin variable region and an insertion<br>
of a nine amino acid peptide, while the NC2, NC3, NC5 and NC6 constructs contained<br>
insertions of peptides of various sizes.<br>
Construction of pSEC194NC1<br>
pSEC194FliC was used as template to obtain the N and C terminal regions by PCR<br>
amplification according to standard procedures with appropriate optimization for the<br>
construction of the truncated hag gene (pSEC194NC1). pSECFliC was digested with<br>
Kpnl/EcoFH to obtain the N-terminal fragment (566 bp). PCR amplification using the FliCR and<br>
CterF primers was used to obtain the C-terminal fragment (232 bp).<br>
FliCR: 5' CAA CAA AGT AAC GGT TGA GCG 3' (SEQ ID NO:36)<br>
CterF: 5' CGC GAATTC CTA GGA GCT ATG CAA AAC C 3' (EcoRI) (SEQ ID NO:40)<br><br>
39<br>
The C-terminal PCR fragment was digested with EcoRI/Dral and ligated with the N-<br>
terminal fragment (KpnVEcoFH) to pSEC194 digested with Kpn\/Dra\ in a 3-way ligation. The<br>
resulting truncated flagellin (FliC) protein has a deletion of 89 amino acids from amino acids<br>
114-202. This deletion spans a large area of the variable region of the FliC protein.<br>
Construction ofpSEC194NC2<br>
pSEC194NC2(NC2) was obtained by inserting a 15 bp (corresponding to 5 amino acids)<br>
multiple cloning site (MCS) insert after nucleotide (nt) 606 of the open reading frame (Fig. 13A).<br>
The N-terminal region was PCR amplified using the primers SigDKpn and FliN-terRev (see<br>
below) and the C-terminal region was PCR amplified using the primers CterF2 and DownRev<br>
(see below) using standard procedures with appropriate optimization. B. halodurans Alk36<br>
genomic DNA was used as the template for all PCR reactions.<br>
FliN-terRev: 5'CTC CTC GAG CGA CCT TCT GAA ACA GC 3' (XhoI) ) (SEQ ID<br>
NO:41)<br>
SigDKpn: 5' CTC GGT ACC CTC GCG TTA CGC TCT TTC TGT (Kpnl) (SEQ ID<br>
NO:42)<br>
CterF2: 5' CAC GAA TTC TCG AGC CCG GGA TCC TCT TCA CTA GGA GCT ATG<br>
CAA AAC 3' (EcoR\, XhoI, SmaI, BamHI) (SEQ ID NO:43)<br>
DownRev: 5' CGCJGCAGA AGA GGA ACG TAA ACG 3' (PstI) (SEQ ID NO:35)<br>
The N-terminal fragment was digested with Xho\/Kpn\ and the C-terminal fragment with<br>
XhoVSspI. pSEC194 was digested with KpnIIHincII and a 3-way ligation resulted in<br>
PSEC194NC2.<br>
Construction of pSEC194NC3<br>
pSEC194NC3 differed from pSEC194NC2 in the position of the insert (after nt 459 of<br>
rather than nt 606) and the size of the insert (27 bp corresponding to 9 amino acids) (Fig. 13B).<br>
The N-terminal region was PCR amplified using the primers SigDKpn and VNR2 (see below)<br>
and the C-terminal region was amplified using the primers VCF and DownRev (see below)<br>
according to standard procedures with appropriate optimization. The template for both<br>
reactions was B. halodurans Alk36 genomic DNA.<br>
SigDKpn: 5' CTC GGT ACC CTC GCG TTA CGC TCT TTC TGT (Kpnl) ) (SEQ ID<br>
NO:42)<br><br>
40<br>
VNR2: 5' CGG CAG CTG TTC ACC AGA ATT AGC ACC AAC 31 (PvuII) (SEQ ID<br>
NO:44)<br>
VCF: 51 CAC GTC GAC TCG AGC CCG GGA TCC TTA ATT GAA CTT GAT TTA ACA<br>
AAA G 3' (Sall, XhoI, SmaI, SamHI) (SEQ ID NO:45)<br>
DownRev: 5' CGC TGC AGA AGA GGA ACG TAA ACG 3' (PstI) (SEQ ID NO:35)<br>
The C-terminal PCR fragment was digested with Sa/I and Psti and ligated into the pSK<br>
vector that had been digested with the same two restriction enzymes to create pSKCter. The N-<br>
terminal fragment was digested only with Kpn\ (the other end was left blunt) and then ligated to<br>
pSKCter digested with Hincll and Kpnl to create pSKNC3. This construct was digested with<br>
KpnI and Sspl to liberate the NC3 fragment which was then ligated to pSEC194 digested with<br>
KpnI and Hindl to obtain pSEC194NC3.<br>
Construction of pSEC194NC5<br>
pSEC194NC5 was generated by inserting 21 nucleotides (corresponding to 7 amino<br>
acids after nt 387 of (Fig. 13C). The N-terminal region was PCR amplified using primers NC5R<br>
and SigDKpn (see below) and the C-terminal was amplified using primers NC5F and DownRev<br>
(see below) using standard procedures with appropriate optimization. B. halodurans genomic<br>
DNA was used as the template for all PCR reactions.<br>
NC5F: 5' CAC GTC GAC TCG AGC CCG GGA TCC TTT AAT ACG CAA AAA TTA<br>
CTC 3' (Sa/I, Xhol, Smal, BamHI) (SEQ ID NO:46)<br>
SigDKpn: 5' CTC GGT ACC CTC GCG TTA CGC TCT TTC TGT (Kpnl) (SEQ ID<br>
NO:42)<br>
DownRev: 5' CGC TGC AGA AGA GGA ACG TAA ACG 3' (Pstl) (SEQ ID NO:35)<br>
NC5R: 51 CAC CTC GAG TGA GTTGTA TCT TTG ATT C 3' {Xhol) (SEQ ID NO:47)<br>
The N-terminal PCR fragment was digested with Kpnl and Xhol. The C-terminal was<br>
generated using pwo DNA polymerase (Roche Diagnostics, Basel, Switzerland) and this<br>
resulted in a blunt ended PCR product. The C-terminal PCR fragment was then restricted with<br>
Xhol and used in a 3-way ligation resulting in pSEC194NC5.<br>
Construction of pSEC194NC6<br>
pSEC194NC6 was generated by inserting 27 bp (corresponding to 9 amino acids after nt<br>
540 of (Fig. 13D). The N-terminal (767 bp) region was PCR amplified using primers SigDKpn<br>
and VNR6 and the C-terminal (349 bp), with primers VCF6 and DownRev. The C-terminal<br>
product was restricted with Sall/Pstl and ligated into pSK restricted with Sall/Pstl to obtain<br><br>
41<br>
pSKCter2. Plasmid pSKCter2 as well as the N-terminal PCR product was restricted with<br>
Kpnl/Sall and ligated to produce the plasmid pSKNC6. The NC6 fragment from pSKNC6 was<br>
PCR amplified with PWO taq using primers SigDKpn and FliCR to obtain a blunt ended PCR<br>
product, which was then restricted with Kpnl and ligated into pSEC194 (Kpnl/Hincll) to obtain<br>
pSEC194NC6.<br>
FliCR:	5' CM CAA AGT AAC GGT TGA GCG 3' (SEQ ID NO:36)<br>
SigDKpn:	5' CTC GGT ACC CTC GCG TTA CGC TCT TTC TGT 3' (Kpnl) (SEQ ID<br>
NO:42)<br>
VNR6:	5' GAC GTC GAC AGT GTG GTC AGT AAT ATC CTC 3'(Safl) (SEQ ID<br>
NO:48)<br>
VCF6:	51 CAC GTC GAC TCG AGC CCG GGA TGG ATC CAG AAT GCA CAA<br>
TCA GCT ATT GAC 3' (Sall) (SEQ ID NO:49)<br>
DownRev:	5' CGC TGC AGA AGA GGA ACG TAA ACG 3' (Pstl) (SEQ ID NO:35)<br>
In all cases the insert was designed to carry a multiple cloning site (MCS) for the<br>
addition of sequences encoding heterologous peptides and proteins. All the NC constructs<br>
were generated to identify the functional insertion sites within the FliC protein, determine the<br>
level of expression, and observe phenotypic (restoration of motility in B. halodurans BhFCOl)<br>
characteristics after transformation. Cell-surface protein (CS) extracts were produced from B.<br>
halodurans Alk36 cultures transformed with NC1, NC2, NC3, NC5 and NC6 respectively, and<br>
the protein extracts resolved by SDS-PAGE as before. BhFCOl containing the NC3 and NC6<br>
constructs were found to over-express a protein resolving to a band corresponding to that of the<br>
FliC protein. (Fig, 14A, lanes 1-5 and Fig. 14B, lanes 2-3). From the electrophoretic gel it is<br>
clear that the levels of the modified FliC protein produced by the BhFC04 (NC6) strain<br>
compared well with that of WT (wild-type) bacteria. NC6 was the only construct that restored<br>
motility to 8. halodurans non-motile mutants (Fig. 15).<br>
The -34 kDa protein band in the cell surface (CS) fraction was confirmed to be flagellin,<br>
or modified flagellin, by Western Blot analysis (Fig. 14C) with FliC-specific antibodies.<br>
Example 7.    Directed Inactivation of the Cell Wall Protease-encoding wprA gene on the<br>
Chromosome of B halodurans strain BhFCOl using the PSEC194 Integration Vector<br>
The wprA gene encodes a cell wall protease that could decrease levels of fusion proteins<br>
expressed on the surface of relevant recombinant bacteria. A strategy was devised to delete the<br>
wprA gene in BhFCOl bacteria.<br><br>
42<br>
The plasmid pSECwprA- was constructed by deleting 1056 bp of the internal region of the<br>
wprA gene (Fig. 16A). This region included the entire wprA coding sequence. Fig. 16B shows the<br>
amino acid sequence of the wprA protein. The following primers were used to obtain the N- and C-<br>
terminal PCR products needed for construction of the wprA- fragment.<br>
N-For: 5' GC GAG CTC TGC AGC GTA CTA CAA CCA 3' (Sad) (SEQ ID NO:50)<br>
N-Rev: 5' GC GGA TCC AGC TGA TAA CGC TAC GTA 3' (BamHI) (SEQ ID NO:51)<br>
C-For: 5' GC GGA TCC TAG CGG ACC TGT AGA TGC TA 3' (BamHI) (SEQ ID NO:52)<br>
C-Rev: 5' GG TCT AGA TGC CTT GTC CTT CGC TGT A 3' (Xbsl) (SEQ ID NO:53)<br>
The PCR products were digested and ligated together into pSEC194 restricted with Sacl/Xbal<br>
to obtain pSEC194wprA- (Fig. 16B). The plasmid pSEC194wprA- was transformed as before into<br>
strain BhFC01. A transformant containing a sco was used to force a dco event as described in<br>
Example 4. The strain obtained was named BhFC04 ( hag, wprA). The extracellular and cell<br>
surface proteins of B. halodurans strains BhFCOl and BhFC04 were extracted and protein and<br>
protease profiles were obtained on SDS-PAGE and gelatin-SDS-PAGE gels (Fig. 17).<br>
Results obtained from a reporter gene in both strains showed improvement of protein<br>
production and stability in the different fractions of BhFC04 (results not shown). The plasmids<br>
pSEC194NC3 and pSEC194NC6 were then transformed into strain BhFC04 to improve production<br>
and stability of the modified FliC proteins on the surface.<br>
Applicants deposited under the Budapest Treaty the BhFC04 ( hag, AwprA) B. halodurans<br>
strain with the NCIMB Culture Collection (NCIMB Ltd, Ferguson Building, Craibstone Estate,<br>
Bucksburn, Aberdeen, AB21 9YA) under accession number NCIMB on 23 Nov 2006. The BhFCO4<br>
strain was assigned the NCIMB accession no.	. The strain deposited with the NCIMB<br>
culture collection was taken from a deposit maintained by the Council for Industrial and Scientific<br>
Research (CSIR) since prior to the priority date of this application. The deposits of the strain will be<br>
maintained without restriction in the NCIMB depository for a period of 30 years, or five years after the<br>
most recent request, or for the effective life of the patent, whichever is the longer, and will be<br>
replaced if the deposit becomes non-viable during that period.<br>
Example 8. Surface Display of Fusion Proteins<br>
Bioremediation and bio-mining: surface display of a poly-His tag peptide<br>
Construction of pSEC194NHisC6<br><br>
43<br>
pSEC194NHisC6 was constructed as an example of display of a metal binding peptide (poly-<br>
Histag) on the surface of strain BhFC04 using the NC6 insertion site. The poly-His tag (containing<br>
six histidine residues) has previously been shown to bind Cadmium, Nickel and Copper [Sousa et al.<br>
(1996) Nature Biotech. 14:1017-1020]. The tag was thus suitable to demonstrate the use of the<br>
flagellin display system for bioremediation or bio-mining of precious metals in the mining industry.<br>
pSEC194NHisC6 was constructed by restricting 3 g pSEC194NC6 with Xho\ and SamHI.<br>
The poly-His tag was generated by annealing two complimentary oligonucleotides (HisF3 and HisR3;<br>
see below) using a method described by IDT (Integrated DNA Technologies, Coralville, IA, U.S.A.).<br>
HisF3 5' TCG AGA CAT CAT CAT CAT CAT CAC AG (SEQ ID NO:54)<br>
HisR3 5' GAT CCT GTG ATG ATG ATG ATG ATG TC (SEQ ID NO:55)<br>
Briefly, the two oligonucleotides (oligos) were diluted separately in STE buffer (10mM Tris pH<br>
8, 50mM NaCI, 1mM EDTA) at a final concentration of 50 M. An equal volume of each oligo solution<br>
was mixed together and diluted to a final concentration of 5 M. The mixture of oligos was heated to<br>
1006C in a boiling waterbath and then allowed to cool slowly to room temperature by switching off the<br>
waterbath. The annealed oligos were purified using the Geneclean III Kit (BIO 101). The annealed<br>
oligos contained a Xhol and SamHI site at the 5' and 3' ends respectively.<br>
5'   TCG AGA CAT CAT CAT CAT CAT CAC AG   BamHI (SEQ ID NO:56)<br>
XhcA CT GTA GTA GTA GTA GTA GTG TCC TAG   3'     (SEQ ID NO:57)<br>
Ligation of the annealed poly-His tag to the restricted pSEC194NC6 was done according to<br>
the Fast-Link DNA ligation kit (Epicentre, Madison, Wl) to give pSECi94NHisC6. 20ng of<br>
PSEC194NC6 and 30ng annealed His oligo were used in the ligation reaction. The reaction was<br>
stopped by heat inactivation at 70C for 15 minutes. Two I of the ligation mix was transformed into<br>
E. coli DH10B cells by electroporation (25 F, 200 Ohms, 1.6KV). Clones were screened using<br>
colony PCR analysis (25 I) using primers HisF3 and FliCR (51 CAA CAA AGT AAC GGT TGA GCG<br>
3'). Clones were picked into 50 I sterile water and boiled for 5 minutes. Boiled colonies were<br>
centrifuged at 12,000 rpm for 30 seconds to pellet cell debris. 5 I of boiled lysate was used in each<br>
PCR reaction. Reaction mixtures were as follows, 2.5 I 10X Buffer, 2.5 I 8mM dNTP's, 0.75 I 50<br>
mM MgCI, 1.25 I 5 M each primer, 0.2 I Taq polymerase (1 unit) and sterile PCR water to 25 I total<br>
volume. PCR parameters were: one cycle at 94C for 4 min, 35 cycles at 94C for 1 minute, 56QC<br>
for 1 minute and 72C for 1 Vz minutes with a final extension of one cycle at 72C for 4 minutes. A<br><br>
44<br>
single positive clone was grown up and plasmid DNA was isolated from the resulting culture. The<br>
amino acid sequence (and its encoding nucleotide sequence) of the poly-His peptide and amino<br>
acids encoded by parts of the MCS that form the inserted heterologous polypeptide are shown in Fig.<br>
18. The full inserted peptide was 13 amino acids long.<br>
Strain BhFC04 was transformed as described above with pSEC194NhisC6 to<br>
determine the metal binding capabilities of this construct. Transformants were confirmed to carry<br>
pSEC194NhisC6 with colony PCR's. BhFC04 clones carrying the pSEC194NhisC6 construct were<br>
grown in 25ml LB broth pH 8.5 to determine both the successful display of the peptide but also its<br>
ability to bind to MagneHis Ni-Particles.<br>
Production and functionality of NHisC6 protein<br>
The locations of the displayed peptide in the bacterial cells were determined by isolating the<br>
extracellular (Ex), cell surface (CS), cell wall (CW) and intracellular (IC) protein fractions and<br>
separating the protein samples on a 10% SDS-PAGE gel. Fractions were isolated as follows.<br>
Cells were grown at 30C in 25ml Luria broth (LB) pH 8.5 to stationary phase (16 hours).<br>
Cells were pelleted and resuspended in 2.5ml sterile phosphate buffer pH 7.5 and the cell pellet was<br>
used for the CS fraction (see next paragraph). The supernatant was precipitated by adding an equal<br>
volume of 5% trichloroacetic acid (TCA) and incubated for 30 min at room temperature. The<br>
precipitated proteins were pelleted by centrifugation at 17,000xg for 30 minutes and dried at 37fiC for<br>
30 minutes. The protein pellet was resuspended in 300^1 of phosphate buffer pH 7.5 to produce the<br>
EX protein fraction.<br>
An equal volume of 0.2M NaOH was added to the resuspended cell pellet and stirred<br>
vigorously with a stirrer bar for 30 minutes on ice. The cells were pelleted by centrifugation at<br>
8,000xg for 10 minutes to remove cellular material and obtain the supernatant. An equal volume of<br>
5% TCA (trichloroacteic acid) was added to the supernatant and the resulting mixture was incubated<br>
at room temperature with shaking for 30 minutes. Cell surface (CS) proteins were pelleted by<br>
centrifugation at 17,000xg for 30 minutes. The pellet was rinsed with sterile water and dried at 379C<br>
for 30 minutes. The CS proteins were then resuspended in 300nl phosphate buffer pH 7.5. The cell<br>
pellet obtained after "stripping" with NaOH was resuspended in 5ml phosphate buffer and sonicated<br>
for 30 minutes at full strength using a Sonopuls Ultrasonic homogenizer (Bandelin, Berlin, Germany).<br>
Lysed cells were centrifuged at 10,000xg for 10 minutes to precipitate the CW fraction. The<br>
supernatant (IC) was removed and placed in 2ml eppendorf tubes. The CW pellet was dried at 379C<br>
for 30 minutes and resuspended in 500 I phosphate buffer pH 7.5.   Protein concentrations were<br><br>
45<br>
determined by the Bradford method (Bradford, 1976, Anal Biochem 72:248-259). CS fractions were<br>
separated on a 10% SDS-PAGE gel and stained using Coomassie stain (Fig. 19).<br>
In order to determine the metal binding capacity of the displayed poly-His tag, the<br>
MagneHis protein purification system (Promega, Madison, Wl, U.S.A.) was used. This kit makes<br>
use of the His-tags ability to bind to nickel particles which can then be isolated using a magnet.<br>
Proteins can then be cleaved from the nickel particles using imidazole. Crude cell extracts were<br>
obtained by pelleting 6ml of overnight cultures of pSEC194NHisC6 and pSEC194NC6 transformed<br>
bacteria (-ve control), and resuspending the pellets in 5 ml phosphate buffer. Each sample was<br>
sonicated as described for the IC extraction above. Proteins were precipitated with an equal volume<br>
of 5% TCA and resuspended in 600 I of lysis buffer (MagneHis Kit). All subsequent steps were<br>
performed according to the manufacturer's instructions under non-denaturing conditions. Proteins<br>
were eluted in a final volume of 100 I.<br>
Successful binding of the His-tag to the MagneHis beads (nickel) were visualized by<br>
separating the samples on a 10% SDS-PAGE gel and staining with Coomassie blue (Fig. 20). Three<br>
fractions, including unbound proteins (lanes 6 and 7), the bead-bound proteins which were not eluted<br>
(lanes 4 and 5), and the bead-bound and eluted proteins (lanes 2 and 3), were included. The fusion<br>
of the poly-His tag to flagellin resulted in the successful display of this peptide in a functional form.<br>
The fact that the preparation was carried out under non-denaturing conditions also demonstrates that<br>
the peptide is displayed on the exposed domain of the flagellin as predicted by sequence alignment<br>
studies.<br>
Surface expression of antigens: HIV (human immunodeficiency virus) peptide<br>
Construction of pSEC194NHivC6<br>
pSEC194NHivC6 was constructed to display a foreign antigenic epitope on the surface of B.<br>
halodurans strain BhFC04. Recombinant organisms such as these can be used as immunogens for<br>
generating immune responses of a variety of types and against antigens from a wide variety of<br>
sources. The V3 loop portion of the HIV isolate HIV-1 was chosen as an example. The antigenic<br>
motif of the gp120 (V3 loop) was found to be conserved (Gly-Pro-Gly-Arg-Ala-Phe) and induces the<br>
production of neutralising antibodies. This epitope was also shown to activate both T-helper and<br>
cytotoxic T-lymphocyte responses [Goudsmit et al. (1988) Proc. Natl. Acad. Sci. USA. 85: 4478-<br>
4482; Javaherian et al. (1989) Proc. Natl. Acad. Sci. USA 86:6788-6772].<br>
pSEC194NHivC6 was constructed by restricting pSEC194NC6 with Xhol and BamHI. The<br>
HIV gp120 antigenic epitope was generated by annealing two complimentary oligonucleotides<br>
(sequences below) as described above.<br><br>
46<br>
Xhol<br>
HivF3B 5' CAC CTC GAG ACG TTC ATT ATC ATA TGG ACC AGG ACG TGC<br>
ATT TCG TAC GCG TTG GAT CCA CAG 3' (SEQ ID NO:58)<br>
BamHI<br>
BamHI<br>
HivR3B 5' CTG TGG ATC CAA CGC GTA CGA AAT GCA CGT CCT GGT CCA<br>
TAT GAT AAT GAA CGT CTC GAG GTG 3'(SEQ ID NO:59)<br>
Xhol<br>
The annealed oligo's were restricted with Xho\ and BamHI and purified as described above.<br>
5' - TC GAG ACG TTC ATT ATC ATA TGG ACC AGG ACG TGC ATT TCG TAC GCG TTG<br>
BamHI (SEQ ID NO:60)<br>
Xhol C TGC AAG TAA TAG TAT ACC TGG TCC TGC ACG TAA AGC ATG CGC AAC CAT<br>
G-3' (SEQIDNO:61)<br>
Ligation of the HIV peptide to the restricted pSEC194NC6 was done using the Fast-Link DNA<br>
Ligation Kit (Epicentre) to give pSEC194NHivC6. 20 ng of pSEC194NC6 and 30ng of the annealed<br>
HIV oligo were used in the ligation reaction. The ligation reaction was heat inactivated at 702C for 15<br>
minutes.<br>
Two I of the ligation mix was transformed into E. coli DH10B cells by electroporation<br>
(25 F, 200 Ohms, 1.6Kv). Clones were screened by colony PCR analysis (25 I) using primers<br>
NC5F and FliN-terRev. PCRs were done as described above with the annealing time being reduced<br>
from 1 1/2 minutes to 1 minute. All samples were analysed on a 1.5% TAE Agarose gel. A single<br>
positive clone grown up and plasmid DNA was isolated from the resulting culture. The amino acid<br>
sequence (and its encoding nucleotide sequence) of the HIV V3 loop peptide and amino acids<br>
encoded by parts of the MCS that form the inserted heterologous polypeptide are shown in Fig. 21.<br>
Sequences were confirmed to be correct and subsequently transformed into B. halodurans BhFC04<br>
(   hag,    wprA).<br>
NC5F: 5' CAC GTC GAC TCG AGC CCG GGA TCC TTT AAT ACG CAA AAA TTA CTC 31<br>
(Sail) (SEQ ID NO:46)<br><br>
47<br>
FliN-terRev: 5' CTC CTC GAG CGA CCT TCT GAA ACA GC 3' (Xho\) (SEQ ID N0:41)<br>
Strain BhFC04 was transformed as described above. Positive transformants were confirmed<br>
with colony PCR using primers FliN-terRev and NC5F. A single positive clone was chosen to isolate<br>
protein fractions as described above. 10 g of CS, 20 g of EX, 40 g of CW and 40 g of IC fractions<br>
were loaded onto 10% SDS-PAGE gels and stained using Coomassie blue stain.<br>
On the SDS-PAGE gel, only the CS fraction gave a band resembling the correct size of the<br>
flagella-Hiv peptide fusion (Fig. 22). The inclusion of the peptide resulted in the insertion of a 21 aa<br>
peptide in the FliC protein and an increase in size of the flagella chimera of approximately 2.3KDa<br>
that is clearly visible on the gel.<br>
Western Blot analysis using FliC-specific antibodies showed that the bands obtained on the<br>
SDS-PAGE gels were either FliC or functional FliC fusion proteins (Fig. 23).<br>
Biotransformation<br>
Construction of pSEC194NMLipC<br>
pSEC194NMLipC3 was constructed to demonstrate the use of the FliC display system for<br>
biotransformation. The examples described above focused on small peptides which can be used for<br>
bioremediation or antigenic determinants in vaccine development. Ezaki et al. [(1998) J. Ferm.<br>
Bioeng. 86:500-503] and Tanskanen [(2000) Appl. Environ. Micro. 66:4152-4156] both<br>
demonstrated that large polypeptides (471 and 302 amino acids in length could also be displayed<br>
successfully using E. coli flagellin. Lipases have been well characterised and play a role in a number<br>
of compositions and processes including, without limitation, detergents, glycerolysis of fats and oils,<br>
direct esterif ication, chiral resolution and acylate synthesis [Litthauer et al. (2002) Enz. Micro. Tech.<br>
30:209-215]. The use of lipases immobilized on the cell surface of Staphylococcus camosus has<br>
been demonstrated using the fibronectin binding protein B fused to the Staphylococcus hyicus lipase<br>
[Strauss et al. (1996) Mol. Microbiol. 21:491-500].<br>
pSEC194NLipC was constructed by restricting pSEC194NC3 with SamHI and Xhol. The<br>
mature lipase was PCR amplified from G thermoleovorans chromosomal DNA (LipA) so that the<br>
signal sequence was not present (Fig. 24). The primers used were LipFSD and LipR (see below).<br>
The PCR products were purified and restricted with BamHl and Xhol.<br>
LipFSD: 5' GTC CTC GAG GCT TCG CGA GCC AAC GAT G 3' (Xho\) (SEQ ID NO:62)<br>
LipR :     5' GTC GGA TCC AGG CCC GAA GCT CGC CA -3' (SamHI) (SEQ ID NO:63)<br><br>
48<br>
Ligation of the lipase polypeptide to the restricted pSEC194NC3 was done using the Fast-<br>
Link DNA Ligation Kit (Epicentre) to give pSEC194NLipC3. 20ng of vector DNA and 60ng of lipase<br>
were used for the ligation reaction. The ligation reaction was heat inactivated at 70C for 15 minutes.<br>
2 I of the ligation mix was transformed into E. coli DH10B cells as described above. A single<br>
positive clone was selected for further analysis.<br>
Transformation of B. halodurans BhFC04 with pSECNLipC and determination of lipase activity<br>
B. halodurans strain BhFC04 transformation was carried as described above. Clones were<br>
screened using colony PCR with primers M13F (5'-GTA AAA CGA CGG CCA GT-3') and LipR. 50<br>
ml LB containing chloramphenicol (10 g/ml) was inoculated from an overnight culture and grown to<br>
an OD540 of 1.2-1.6. Protein fractions were isolated as described above except that 5ml LiCI (5M),<br>
rather than NaOH, was used to strip the CS proteins from the cell surface. This allowed for<br>
maximum lipase activity as NaOH inactivated the lipase enzyme. The EX and CS fractions were<br>
TCA precipitated even though this method reduced lipase activity; sufficient activity remained to<br>
visualize on a zymogram (activity gel) (Fig. 25). However, this method naturally could not be used to<br>
accurately quantify the activity in the EX and CS fractions. Nevertheless, the activity gel is a useful<br>
tool for determining the actual size and stability of the fusion protein and performed as described by<br>
Takahashi et al. [(1998) J. Ferm. Bioeng. 86:164-168]. Samples were prepared by adding non-<br>
denaturing loading dye and incubation for 30 min at 37C. SDS-PAGE gels were run at a constant<br>
current of 30mA until the dye front reached the end of the gel. The gels were incubated overnight in<br>
25mM Tris pH 7.5 and 2.5% Triton X-100 to remove the SDS, transferred to equilibration buffer<br>
(25mM Tris pH 7.5) for 30 min, and then stained for lipase activity using 0.1% -naphthyl acetate and<br>
0.2% Fast Red TR Salt in equilibration buffer. The reaction was stopped with 2 washes of TE buffer<br>
(1 OmMTris, pH 8,1 mM EDTA) once the bands could be clearly seen.<br>
Lipase activity was observed in the CS, CW and intracellular fractions but not in the<br>
extracellular fraction. These findings indicated that the FliC-lipase fusion remains firmly bound to the<br>
CW and CS fractions and are exposed on the cell surface. The reduced activity in the CS fraction<br>
was possibly due to the TCA which inactivates lipase. These results also indicate that the fusion<br>
protein is very stable.<br>
The next step was to quantify the lipase activity produced in liquid cultures (Fig. 26).<br>
Overnight cultures of BhFCO4 transformed with pSEC194NC6 (control) or pSEC194NLipC<br>
constructs were grown in Luria Broth pH 8.5, chloramphenicol 10 n.g/ml at 30 C. Two flasks<br>
containing Luria Broth pH 8.5 (60 ml) containing chloramphenicol 10 mg/ml were inoculated from the<br>
ON cultures to give a starting OD54o of 0.1. The flasks were incubated at 30 C and samples taken at<br>
8, 24 and 48 hours.  Whole cell and extracellular samples were used for lipase assays.  Lipolytic<br><br>
49<br>
activity was determined essentially according to Vorderwulbecke et al. [(1992) Enzym Microb.<br>
Technol. 14: 631-639], by a spectrophotometric assay using p-nitrophenyl-palmitate as substrate.<br>
Assay solution 1 was prepared by dissolving 90 mg p-nitophenyl-palmiate in 30 ml propan-2-ol.<br>
Assay solution 2 contained Na-deoxycholate (2g) and gum Arabic (0.5g) dissolved in 450 ml Tris-HCI<br>
buffer pH8. Emulsion solution was prepared by adding 1ml of assay solution 1 to 9 ml of assay<br>
solution 2. Under standard conditions, the assays were performed by incubating 600 nl of emulsion<br>
solution and 25 u.l enzyme preparation.<br>
The reaction was performed at 65 C and the absorbance measured at 410nm. Lipase<br>
activity was calculated as U/ml (nmol fatty acids/min/ml enzyme). The extinction coefficient of<br>
p-nitrophenol at 410 nm (pH8) is 15 (1x nrnol-1 x cm-1 = ml x cm"1). Protein concentrations of<br>
enzyme samples were determined by the method of Bradford (1976) using bovine serum<br>
albumin dilutions as standards. Final activity was expressed as U/mg total protein as shown in<br>
Fig. 26. This data shows that it is possible to insert an enzyme into the FliC sandwich and<br>
retain activity. The FliC/Lipase fusion protein forms a very stable complex within the B<br>
halodurans cell wall with lipase activity available to the cell surface. Enzyme activity was shown<br>
to be relatively strong and sustained over a long period of time. Very little enzyme activity was<br>
detected in the supernatant, both favorable characteristics for bio-transformation.<br>
Example 9 immunoqenic peptide as a fusion with flaqellin protein.<br>
Adjuvants have the ability to stimulate innate immunity and in turn activate the adaptive immune<br>
response. It has already been established that flagellin induces an inflammatory response<br>
through the activation of APC's (see above). An example is the successful creation and<br>
presentation of a flagellin enhanced green fluorescent protein (EGFP) fusion protein. The<br>
flagellin-EGFP fusion was capable of stimulating APC's and also specific anti EGFP T-cell<br>
responses. EGFP alone was unable to stimulate neither APC's nor specific T-cell responses<br>
(Cuadros et al., 2004, Inf. Immun. Vol 72, 2810-2816, McSorley et al., 2002, J. Immunol. Vol<br>
169, 3914-3919). Other peptides inserted into the flagellin which induced an immune response<br>
include Cholera toxin subunit B, Hepatitus B epitopes, Streptococcus pyogenes M protein<br>
epitope, HIV epitopes (gp 41, gp120), influenza A hemagglutinin epitope and various cell<br>
surface antigens from Plasmodium sp., Rotavirus, Corynebacterium diphtheriae and<br>
Miningococcal outer membrane protein (Stocker and Newton, 1994, Intern. Rev. Immunol. Vol<br>
2,167-178). Newton etal., (1995) Res. Microbiol. 146: 203-216.<br><br>
50<br>
A number of embodiments of the invention have been described. Nevertheless, it will be<br>
understood that various modifications may be made without departing from the spirit and scope<br>
of the invention. Accordingly, other embodiments are within the scope of the following claims.<br><br>
51<br>
WHAT IS CLAIMED IS:<br>
1.	A fusion protein comprising:<br>
all or part of bacterial flagellin protein, wherein the part of the flagellin protein comprises<br>
the N-terminal and C-terminal conserved regions of the flagellin protein; and<br>
a heterologous polypeptide sequence within, or replacing, the variable region of the<br>
flagellin protein,<br>
wherein the fusion protein has the ability, if made by a Gram positive bacterial cell, to be<br>
over-expressed on the surface of the Gram positive bacterial cell.<br>
2.	The fusion protein of claim 1, wherein the heterologous polypeptide is a polypeptide<br>
having the ability to bind to a metal ion.<br>
3.	The fusion protein of claim 2, wherein the metal ion is selected from the group consisting<br>
of nickel, copper, cadmium, platinum, palladium, titanium, silver, and gold.<br>
4.	The fusion protein of any one of claims 1 to 3, wherein the heterologous polypeptide is a<br>
polyhistidine sequence.<br>
5.	The fusion protein of claim 4, wherein the polyhistidine sequence comprises six histidine<br>
residues.<br>
6.	The fusion protein of one of claims 1 to 3, wherein the heterologous polypeptide is an<br>
enzyme or a functional fragment of an enzyme.<br>
7.	The fusion protein of claim 6, wherein the enzyme is a lipase enzyme.<br>
8.	The fusion protein of claim 7, wherein the lipase enzyme is G. thermoleovorans lipase A.<br>
9.	The fusion protein of claim 6, wherein the enzyme is a hydrolytic enzyme.<br>
10.	The fusion protein of claim 6, wherein the enzyme is selected from the group consisting<br>
of amylases, proteases, esterases, and cellulases.<br>
11.	The fusion protein of claim 1, wherein the heterologous polypeptide is an immunogen.<br>
12.	The fusion protein of any one of claims 1 to 11, further comprising one to fifteen linker<br>
residues N-terminal of the N-terminus of the heterologous polypeptide.<br><br>
52<br>
13.	The fusion protein of any one of claims 1 to 11, further comprising one to fifteen linker<br>
residues C-terminal of the C-terminus of the heterologous polypeptide.<br>
14.	The fusion protein of any one of claims 1 to 11, further comprising cleavable sites N-<br>
terminal of the N-terminus of the heterologous polypeptide and C-terminal of the C-terminus of<br>
the heterologous polypeptide.<br>
15.	A nucleic acid encoding the fusion protein of claim 1.<br>
16.	A vector comprising the nucleic acid sequence of claim 15.<br>
17.	The vector of claim 16, wherein the nucleic acid sequence is operably linked to a<br>
transcriptional regulatory element (TRE).<br>
18.	An isolated cell comprising the vector of claim 17.<br>
19.	The cell of claim 18, wherein the cell is a prokaryotic cell.<br>
20.	The cell of claim 19, wherein the cell is a bacterial cell.<br>
21.	The cell of claim 20, wherein the cell is a gram positive bacterial cell.<br>
22.	The cell of claim 21, wherein the cell is of the Bacillus genus.<br>
23.	The cell of claim 22, wherein the cell is of the B. halodurans species.<br>
24.	The cell of claim 23, wherein the cell is of the strain BhFC04 deposited under Accession<br>
Number	at the NCIMB on 23 November 2005.<br>
25.	A method of making a fusion protein, the method comprising culturing the cell of claim<br>
18 and obtaining the fusion protein from the culture.<br>
26.	A DNA construct comprising:<br>
all or part of the coding sequence for a bacterial flagellin polypeptide, wherein the part of<br>
the coding sequence comprises nucleotides encoding the N-terminal and C-terminal conserved<br>
regions of the flagellin protein; and,<br><br>
53<br>
inserted into, or replacing, the sequence encoding the variable region of the flagellin<br>
polypeptide, a nucleotide sequence comprising at least one restriction enzyme site.<br>
27.	The construct of claim 26, wherein the bacterial flagellin is a Bacillus flagellin.<br>
28.	The construct of claim 26, wherein the Bacillus flagellin is B. halodurans flagellin (SEQ<br>
IDN0:1).<br>
29.	The construct of any one of claims 26 to 28, wherein the nucleotide sequence is inserted<br>
immediately after any nucleotide between nucleotide 162 and nucleotide 606 of SEQ ID NO:1.<br>
30.	The construct of any one of claims 26 to 28, wherein the nucleotide sequence is inserted<br>
immediately after any nucleotide between nucleotide 441 and nucleotide 570 of SEQ ID NO:1.<br>
31.	The construct of any one of claims 26 to 28, wherein the nucleotide sequence is inserted<br>
immediately after any nucleotide between nucleotide 459 and nucleotide 540 of SEQ ID NO:1.<br>
32.	The construct of any one of claims 26 to 28, wherein the nucleotide sequence is inserted<br>
immediately after nucleotide 459 of SEQ ID NO:1.<br>
33.	The construct of any one of claims 26 to 28, wherein the nucleotide sequence is inserted<br>
immediately after nucleotide 540 of SEQ ID NO:1.<br>
34.	A substantially pure culture of Gram positive bacterial cells, a substantial number of<br>
which comprise a disrupted flagellin gene, wherein the disruption prevents expression of<br>
functional flagellin.<br>
35.	The culture of claim 34, wherein the disruption is by replacement of the endogenous<br>
gene in the substantial number of bacterial cells with a DNA sequence encoding no polypeptide<br>
or a non-functional flagellin polypeptide.<br>
36.	The culture of claim 34 or claim 35, wherein the bacterial cells are gram positive<br>
bacterial cells.<br>
37.	The culture of claim 36, wherein the gram positive bacterial cells are Bacillus cells.<br>
38.	The culture of claim 37, wherein the Bacillus cells are B. halodurans cells.<br><br>
54<br>
39.	The culture of claim 38, wherein the B. halodurans cells are selected from BhFC04(<br>
hag,     wprA )cells deposited under Accession Number	at the NCIMB on 23 November<br>
2005 and BhFC01( hag) cells.<br>
40.	The culture of any one of claims 35 to 39, wherein the non-functional flagellin<br>
polypeptide lacks amino acids 14 to 226 of SEQ ID NO:2.<br>
41.	The culture of any one of claims 34 to 40, wherein, in addition to comprising a disrupted<br>
flagellin gene, at least one cell wall protease gene of a substantial number of the bacterial cells<br>
is disrupted.<br>
42.	The culture of claim 41, wherein the disruption of the at least one cell wall protease gene<br>
is by replacement of the endogenous gene in the substantial number of bacterial cells with a<br>
DNA sequence encoding either no polypeptide or a non-functional cell wall  protease<br>
polypeptide.<br>
43.	The culture of claim 41, wherein the at least one cell wall protease gene is the wrpA<br>
gene.<br>
44.	The culture of claim 41 or claim 43, wherein the disruption comprises deletion of the<br>
entire coding sequence of cell wall protease gene.<br>
45.	An isolated Gram positive bacterial cell comprising a disrupted flagellin gene, wherein<br>
the disruption prevents expression of functional flagellin.<br>
46.	A method of genetically engineering a bacterium of the genus Bacillus, the method<br>
comprising disrupting the hag gene in the chromosome of the bacterium, wherein the disruption<br>
prevents expression of functional flagellin by the gene.<br>
47.	The method of claim 46, wherein bacterium is a Bacillus holodurans bacterium strain<br>
BhFC04 (   hag,    wprA).<br>
48.	The method of claim 46, further comprising disrupting one or more genes encoding one<br>
or more proteases, wherein the disruption prevents expression of the one or more functional<br>
proteases by the one or more genes.<br><br>
55<br>
49.	The method of claim 48, wherein the one or more cell wall protease genes comprise the<br>
wrpA gene.<br>
50.	A method of removing one or more metal ions from a liquid, the method comprising:<br>
contacting a liquid comprising one or more metal ions with the fusion protein of claim 1,<br>
wherein the heterologous polypeptide is a polypeptide that binds to the one more metal ions.<br>
51.	The method of claim 50, wherein the liquid is contacted with a bacterial cell expressing<br>
the fusion protein on its surface.<br>
52.	The method of claim 50 or claim 51, wherein the fusion protein is a cell-surface<br>
polypeptide.<br>
53.	A method of isolating one or more metal ions from a liquid containing the one or more<br>
metal ions, the method comprising:<br>
contacting a liquid comprising one or more metal ions with the fusion protein of claim 1,<br>
wherein the heterologous polypeptide is a polypeptide that binds to the one or more metal ions,<br>
the contacting resulting in binding of the one more metal ions to the fusion protein; and<br>
separating the one or more metal ions from the fusion protein.<br>
54.	A method of converting a substrate to a product, the method comprising:<br>
contacting an enzyme substrate with the fusion protein of claim 1, wherein the<br>
heterologous polypeptide is the enzyme or a functional fragment of the enzyme.<br>
55.	Use of the fusion protein of claim 1, wherein the heterologous polypeptide is an immunogen,<br>
in the manufacture of a preparation for generating an immune response in a mammalian<br>
subject.<br>
56.	A substance or composition for use in a method of generating an immune response in a<br>
mammalian subject, said substance or composition comprising the fusion protein of claim 1,<br>
wherein the  heterologous  polypeptide is an  immunogen,  and said  method comprising<br>
administering an effective amount of said substance or composition to said mammalian subject.<br>
57.	A method of generating an immune response, the method comprising administering the<br>
fusion protein of claim 1 to a mammalian subject, wherein the heterologous polypeptide is an<br>
immunogen.<br><br>
56<br>
58.	The method of claim 57, wherein the mammalian subject is a human.<br>
59.	A kit comprising an expression vector that includes DNA construct of claim 26.<br>
60.	The kit of claim 59, further comprising at least one restriction enzyme.<br>
61.	The kit of claim 59 or claim 60, further comprising a host cell, wherein the host cell is a<br>
cell in which the expression vector is capable of replicating.<br>
62.	The kit of any one of claims 59 to 61, further comprising instructions for inserting a<br>
nucleic acid sequence encoding a heterologous polypeptide into the DNA construct.<br>
63.	A method of producing polypetides attached to the cell surface of Gram-positive bacteria,<br>
the method comprising growing a host cell which has a nucleic acid encoding the fusion protein<br>
of claim 1 and thereby allowing production of said polypeptides to take place.<br><br>
The invention provides f lagellin-based fusion proteins (FBFP) useful for a variety of purposes, in bioremediation<br>
to remove metal ions from a liquid, to express an enzyme or an immunogen, nucleic acids encoding the FBFP, vectors containing<br>
the nucleic acids, and host cells harboring the vectors. Furthermore, the invention provides methods for obtaining over-expression<br>
and surface display of heterologous polypeptides in Gram-positive bacterial cells in Bacillus halodurans in particular. In addition,<br>
the invention features gene-disrupted bacteria useful for expressing the recombinant FBFP on their surfaces. Also included in the<br>
invention are genetic constructs useful for making FBFP and methods of using the FBFP.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-correspondence others 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4yLnBkZg==" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-correspondence others 1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4zLnBkZg==" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-correspondence others 1.3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIGV4bSByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-international exm report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI0MDQta29sbnAtMjAwNy1wcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">02404-kolnp-2007-priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgwNy0wOS0yMDExKS1FWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(07-09-2011)-EXAMINATION REPORT REPLY RECEIVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgwNy0wOS0yMDExKS1GT1JNIDMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(07-09-2011)-FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgwNy0wOS0yMDExKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(07-09-2011)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxMy0wOC0yMDE0KS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(13-08-2014)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1DRVJUSUZJRUQgQ09QSUVTKE9USEVSIENPVU5UUklFUykucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-CERTIFIED COPIES(OTHER COUNTRIES).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1ERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1GT1JNLTIucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-FORM-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1GT1JNLTMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-FORM-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgxNS0wMy0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(15-03-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyMi0wMy0yMDEzKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(22-03-2013)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNC0wOS0yMDE0KS1DTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(24-09-2014)-CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNC0wOS0yMDE0KS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(24-09-2014)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNC0wOS0yMDE0KS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(24-09-2014)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNC0xMS0yMDE0KS0oSVBPKS1ERUNJU0lPTiBMRVRURVIucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(24-11-2014)-(IPO)-DECISION LETTER.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNi0wNi0yMDE0KS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(26-06-2014)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNy0wNC0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(27-04-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LSgyNy0wNC0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-(27-04-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFIDEuNC5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-CORRESPONDENCE 1.4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1LT0xOUC0yMDA3LUNPUlJFU1BPTkRFTkNFLTEuNS5wZGY=" target="_blank" style="word-wrap:break-word;">2404-KOLNP-2007-CORRESPONDENCE-1.5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1rb2xucC0yMDA3LUNPUlJFU1BPTkRFTkNFLnBkZg==" target="_blank" style="word-wrap:break-word;">2404-kolnp-2007-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNC1rb2xucC0yMDA3LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">2404-kolnp-2007-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QtMjQwNC1rb2xucC0yMDA3LmpwZw==" target="_blank" style="word-wrap:break-word;">abstract-2404-kolnp-2007.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="263714-power-regulation-for-field-instruments.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="263716-a-method-and-apparatus-for-providing-network-support-for-a-wireless-emergency-call.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>263715</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2404/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Nov-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Jun-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CSIR</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SCIENTIA, 0002 PRETORIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BERGER, ELDIE</td>
											<td>93 BRAMPTON ROAD, LYNNWOOD MANOR, 0081 PRETORIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CRAMPTON, MICHAEL, CRAIG</td>
											<td>669 CORINNE STREET, 0060 GARSFONTEIN EAST</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LOUW, MAUREEN, ELIZABETH</td>
											<td>262A, JUPITER STREET, WATERKLOOF RIDGE, 0181 PRETORIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2005/054022</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2004/9786</td>
									<td>2004-12-02</td>
								    <td>South Africa</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/263715-gram-positive-fusion-flagellin-protein-producing-bacteria by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:25:58 GMT -->
</html>
